<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-13 09:27:15 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="The tumor microenvironment (TME) is a complex network of interactions between malignant and host cells, yet its orchestration in advanced high-grade serous ovarian carcinoma (HGSC) remains poorly understood. We present a comprehensive single-cell spatial atlas of 280 metastatic HGSCs, integrating high-dimensional imaging, genomics, and transcriptomics. Using 929 single-cell maps, we identify distinct spatial domains associated with phenotypically heterogeneous cellular compositions, and demonstrate that immune cell co-infiltration at the tumor-stroma interface significantly influences clinical outcomes. To uncover the key drivers of the tumor ecosystem, we developed CEFIIRA (Cell Feature Importance Identification by RAndom forest), which identified tumor cell-intrinsic MHC-II expression as a critical predictor of prolonged survival, independent of clinicomolecular profiles. Validation with external datasets confirmed that MHC-II-expressing cancer cells drive immune infiltration and orchestrate spatial tumor-immune interactions. Our atlas offers novel insights into immune surveillance mechanisms across HGSC clinicomolecular groups, paving the way for improved therapeutic strategies and patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/684d76f91d1a8b1bc3e208feca47d927bec6a2ec" target='_blank'>
              Single-cell spatial atlas of high-grade serous ovarian cancer unveils MHC class II as a key driver of spatial tumor ecosystems and clinical outcomes
              </a>
            </td>
          <td>
            Fernando Pérez-Villatoro, Lilian van Wagensveld, Aleksandra Shabanova, Ada Junquera, Ziqi Kang, Iga Niemiec, Matias M. Falco, Ella Anttila, J. Casado, Eric Marcus, Essi Kahelin, Foteini Chamchougia, M. Salko, Saundarya Shah, Salvatore Russo, Jacopo Chiaro, Mikaela Grönholm, G. Sonke, K. K. van de Vijver, Rutgerus Fpm Kruitwagen, Maaike Avan der Aa, A. Virtanen, Vincenzo Cerullo, Anna Vähärautio, P. Sorger, Hugo M. Horlings, A. Färkkilä
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Neuroendocrine and tuft cells are rare, chemosensory epithelial lineages defined by expression of ASCL1 and POU2F3 transcription factors, respectively1,2. Neuroendocrine cancers, including small cell lung cancer (SCLC), frequently display tuft-like subsets, a feature linked to poor patient outcomes3–13. The mechanisms driving neuroendocrine–tuft tumour heterogeneity, and the origins of tuft-like cancers are unknown. Using multiple genetically-engineered animal models of SCLC, we demonstrate that a basal cell of origin (but not the accepted neuroendocrine origin) generates neuroendocrine–tuft-like tumours that highly recapitulate human SCLC. Single-cell clonal analyses of basal-derived SCLC further uncovers unexpected transcriptional states and lineage trajectories underlying neuroendocrine–tuft plasticity. Uniquely in basal cells, introduction of genetic alterations enriched in human tuft-like SCLC, including high MYC, PTEN loss, and ASCL1 suppression, cooperate to promote tuft-like tumours. Transcriptomics of 944 human SCLCs reveal a basal-like subset and a tuft-ionocyte-like state that altogether demonstrate remarkable conservation between cancer states and normal basal cell injury response mechanisms14–18. Together, these data suggest that the basal cell is a plausible origin for SCLC and other neuroendocrine-tuft cancers that can explain neuroendocrine–tuft heterogeneity—offering new insights for targeting lineage plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92a9f4695f47fd01e98ecf709150d28f3a1f7f97" target='_blank'>
              Basal cell of origin resolves neuroendocrine–tuft lineage plasticity in cancer
              </a>
            </td>
          <td>
            A. Ireland, Sarah B. Hawgood, Daniel A. Xie, Margaret W. Barbier, Scarlett Lucas-Randolph, Darren R Tyson, Lisa Y. Zuo, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C. Moser, Sonam Puri, Charles M. Rudin, J. Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Embryogenesis is commonly viewed through a tree model of cell differentiation, which does not adequately represent the spatiotemporal modulation of cell multipotency underlying morphogenesis. Here we develop an integrated approach, combining in vivo single-cell high-throughput clonal lineage tracing with machine learning, to systematically decompose continuous spectra of clonal fate biases in mouse embryos traced from neurulation until mid-gestation. The reconstructed patterns of clonal variation uncovered gene programs driving dynamic positional biasing of clonal composition during axial skeletogenesis and peripheral neurogenesis. Mosaic combinatorial perturbations targeting multiple receptors, including the Hedgehog pathway, led to novel clone types, which has implications for engineering custom cell type assemblages from well-defined progenitors in vivo. Altogether, our work demonstrates an effective practical approach for interrogating programs guiding lineage specification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5676798f4b41e9176c4597c7d9b91163b9f17a43" target='_blank'>
              Unbiased profiling of multipotency landscapes reveals spatial modulators of clonal fate biases
              </a>
            </td>
          <td>
            A. Erickson, Sergey Isaev, Artem Artemov, Jingyan He, Bettina Semsch, Aliia Murtazina, Jia Sun, Katrin Mangold, Anthi Chalou, Jonas Frisén, Michael Ratz, E. Andersson, P. V. Kharchenko, I. Adameyko
          </td>
          <td>2024-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="In bladder cancer (BLCA), the cellular expression of lineage markers with predictive biomarker function can have important implications for tumor progression, treatment response, and survival. However, tumor heterogeneity and the coexistence of distinct tumor subpopulations can complicate the utility of using a primary, clinically assigned tumor signature. In this report, we have applied the reference atlas, Tabula Sapiens, and deep learning model, UniCell, to conduct in-depth transcriptional analysis of unique lineage marker-defined cell clusters in a carcinogen (BBN) induced bladder cancer model. UniCell deconvolution has identified tumor populations, including urothelial, adenocarcinoma, squamous cell carcinoma, and mesenchymal tumor populations, each with cell-intrinsic gene expression signatures relevant to human BLCA progression. The identified tumor clusters contain uniquely basal, luminal, stromal, or hybrid cells in cell lineage marker expression. To understand the significance of these populations during progression, we used trajectory and pseudo-time analysis to show that cells uniquely basal are plastic in lineage identity and can evolve to form tumor populations composed of other lineage marker-defined signatures. Finally, pathway and drug enrichment analysis of tumor cell clusters were used to identify therapeutics that may preferentially target the identified tumor cell populations. These data collectively define a molecular template that may uniquely profile molecular plasticity occurring during progression and response to therapy with important implications for human disease. Significance Tumor heterogeneity is a mechanism for treatment resistance. Our study defines unique tumor subpopulations having differential therapeutic sensitivities and potential for lineage plasticity. Our modeling may impact the treatment of BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e51b8c18f48b163ace1388c13e342c3f5124f692" target='_blank'>
              Mapping of Multilineage Tumor Cell Populations in Mouse Bladder Cancer
              </a>
            </td>
          <td>
            Nadine Schrode, Yang Hu, Haocheng Yu, M. Galsky, J. Sfakianos, R. Sebra, Kristin G Beaumont, David J Mulholland
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) is evolving rapidly as a pivotal technology in studying the biology of tumors and their associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. In this study, we used serial 5-m sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma tumor samples in tissue microarrays to compare the performance of the single cell ST platforms CosMx, MERFISH, and Xenium (uni/multi-modal) platforms in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx Digital Spatial Profiler, and hematoxylin and eosin staining data for the same samples. In addition to objective assessment of automatic cell segmentation and phenotyping, we performed pixel-resolution manual evaluation of phenotyping to carry out pathologically meaningful comparison between ST platforms. Our study detailed the intricate differences between the ST platforms, revealed the importance of parameters such as tissue age and probe design in determining the data quality, and suggested reliable workflows for accurate spatial profiling and molecular discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17805d448b42a38f2a1b50b0eb2fd17aaa264ce4" target='_blank'>
              Comparison of imaging-based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed, paraffin-embedded tumor samples
              </a>
            </td>
          <td>
            Nejla Ozirmak Lermi, Max Molina Ayala, Sharia D Hernandez, Wei Lu, Khaja B. Khan, Alejandra Serrano, Idania Lubo, Leticia Hamana, K. Tomczak, Sean Barnes, Jinzhuang Dou, Qingnan Liang, Rti Team, M. Gabriela Raso, Ximing Tang, Mei Jiang, B. Sanchez-Espiridion, A. Weissferdt, J. Heymach, Jianjun Zhang, B. Sepesi, T. Cascone, Anne Tsao, Mehmet Altan, Reza J Mehran, Don L Gibbons, Ignacio I. Wistuba, C. Haymaker, Ken Chen, L. S. Solis Soto
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Recent studies have identified a unique subtype of cancer-associated fibroblasts (CAFs) termed antigen-presenting CAFs (apCAFs), which remain the least understood CAF subtype. To gain a comprehensive understanding of the origin and function apCAFs, we construct a fibroblast molecular atlas across 14 types of solid tumors. Our integration study unexpectedly reveals two distinct apCAF lineages present in most cancer types: one associated with mesothelial-like cells and the other with fibrocytes. Using a high-resolution single-cell spatial imaging platform, we characterize the spatial niches of these apCAF lineages. We find that mesothelial-like apCAFs are located near cancer cells, while fibrocyte-like apCAFs are associated with tertiary lymphoid structures. Additionally, we discover that both apCAF lineages can up-regulate the secreted protein SPP1, which facilitates primary tumor formation and peritoneal metastasis. Taken together, this study offers an unprecedented resolution in analyzing apCAF lineages and their spatial niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fa0ff2ff7820fdabd0fa0f4d3421c72dfbfc1" target='_blank'>
              Single-cell resolution spatial analysis of antigen-presenting cancer-associated fibroblast niches
              </a>
            </td>
          <td>
            Xiongfeng Chen, Zhuan Zhou, Zeynep Yazgan, Luyu Xie, Francesca Rossi, Yang Liu, Bo Zhang, Patricio M. Polanco, Herbert J. Zeh, Alex C. Kim, Huocong Huang
          </td>
          <td>2024-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), a distinct subpopulation within tumors, are pivotal in driving treatment resistance and tumor recurrence, posing substantial challenges to conventional therapeutic strategies. Precise quantification and profiling of these cells are essential for improving cancer treatment outcomes. We present ACSCeND, an advanced deep neural network model accompanied by a robust workflow, specifically developed to quantify cellular compositions from bulk RNA-seq data, enabling accurate CSC profiling. By integrating bulk RNA-seq data with insights derived from single-cell RNA-seq datasets, ACSCeND effectively captures the diversity and hierarchical organization of tumor-resident cell states, alongside cell-specific gene expression profiles (GEPs). Compared to current tissue deconvolution models, ACSCeND exhibits superior performance, achieving significantly higher Concordance Correlation Coefficient (CCC) values and lower Root Mean Square Error (RMSE) across various pseudobulk and real-world bulk tissue samples. Application of ACSCeND to TCGA and PRECOG datasets reveals a strong association between CSC abundance and poorer disease-free survival outcomes, underscoring the clinical relevance of CSCs in cancer progression. Furthermore, cell-specific GEPs for distinct CSC states unveil novel molecular signatures and illuminate the origins of CSC-driven tumor heterogeneity. In summary, ACSCeND provides a powerful, scalable platform for high-throughput quantification of cellular compositions and distinct potency states within normal tissues as well as highly heterogeneous tissues, such as tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b3b0a8bb89725a847aae6e2af230a0666ba9d6" target='_blank'>
              Comprehensive Enumeration of Cancer Stem-like Cell Heterogeneity Using Deep Neural Network
              </a>
            </td>
          <td>
            Debojyoti Chowdhury, Shreyansh Priyadarshi, Sayan Biswas, Bhavesh Neekhra, Debayan Gupta, Shubhasis Haldar
          </td>
          <td>2024-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="The organizational principles of nephronal segments are based on longstanding anatomical and physiological attributes that are closely linked to the homeostatic functions of the kidney. Novel molecular approaches have recently uncovered layers of deeper signatures and states in tubular cells that arise at various timepoints on the spectrum between health and disease. For example, a dedifferentiated state of proximal tubular cells with mesenchymal stemness markers is frequently seen after injury. The persistence of such a state is associated with failed repair. Here, we introduce a novel analytical pipeline applied to highly multiplexed spatial protein imaging to characterize proximal tubular subpopulations and neighborhoods in reference and disease human kidney tissue. The results were validated and extended through integration with spatial and single cell transcriptomics. We demonstrate that, in reference tissue, a large proportion of S1 and S2 proximal tubular epithelial cells express THY1, a mesenchymal stromal and stem cell marker that regulates differentiation. Kidney disease is associated with loss of THY1 and transition towards expression of PROM1, another stem cell marker shown recently to be linked to failed repair. We demonstrate that the trajectory of proximal tubular cells to THY1 expression is clearly distinct from that of PROM1, and that a state with PROM1 expression is associated with niches of inflammation. Our data support a model in which the interplay between THY1 and PROM1 expression in proximal tubules associates with their regenerative potential and marks the timeline of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e12887f03204d9141acb86ce8374e99f9dcbad" target='_blank'>
              Integration of spatial multiplexed protein imaging and transcriptomics in the human kidney tracks the regenerative potential timeline of proximal tubules
              </a>
            </td>
          <td>
            Mahla Asghari, Angela R. Sabo, Daria Barwinska, R. M. Ferreira, M. Ferkowicz, William S. Bowen, Ying-Hua Cheng, D. Gisch, Connor J. Gulbronson, Carrie L. Phillips, Katherine J. Kelly, T. Sutton, James C Williams, Miguel Vazquez, John O’Toole, Paul M. Palevsky, Sylvia E. Rosas, S. Waikar, Krzysztof Kiryluk, C. Parikh, Jeff Hodgins, P. Sarder, Ian H. de Boer, Jonathan Himmelfarb, Matthias Kretzler, Sanjay Jain, Michael T. Eadon, S. Winfree, Tarek M. El-Achkar, P. Dagher
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4258aac43fd62290da08aca5103948ba5a168475" target='_blank'>
              Multimodal single cell-resolved spatial proteomics reveal pancreatic tumor heterogeneity
              </a>
            </td>
          <td>
            Yanfen Xu, Xi Wang, Yuan Li, Yiheng Mao, Yiran Su, Yize Mao, Yun Yang, Weina Gao, Changying Fu, Wendong Chen, Xueting Ye, Fuchao Liang, Panzhu Bai, Ying Sun, Shengping Li, Ruilian Xu, Ruijun Tian
          </td>
          <td>2024-11-21</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="High-risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single-cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this gap, we profiled untreated and chemotherapy-treated high-risk neuroblastomas from 2 patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p-gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells and we experimentally demonstrated that CCL18 increases neuroblastoma cell migration. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical-like cells expressed the ALK ligand ALKAL2 and were predicted to communicate with neighboring ALK expressing cancer cells. We demonstrated a unique developmental pattern of adrenal medulla-specific expression of ALK and adrenocortical-specific expression of ALKAL2, suggesting a role of these signaling interactions in neuroblastoma carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cacb0dcc803b6c58db82e59f6d5ec533dad1808" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment unveils novel paracrine interactions
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlson, A. Claeys, Eva Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third leading cause of cancer mortality in the United States. Familial adenomatous polyposis (FAP) is a hereditary syndrome that raises the risk of developing CRC, with total colectomy as the only effective prevention. Even though FAP is rare (0.5% of all CRC cases), this disease model is well suited for studying the early stages of malignant transformation as patients form many polyps reflective of pre-cancer states. In order to spatially profile and analyze the pre-cancer and tumor microenvironment, we have performed single-cell multiplexed imaging for 52 samples: 12 normal mucosa,16 FAP mucosa,18 FAP polyps, 2 FAP adenocarcinoma, and 4 sporadic colorectal cancer (CRCs) using Co-detection by Indexing (CODEX) imaging platform. The data revealed significant changes in cell type composition occurring in early stage polyps and during the malignant transformation of polyps to CRC. We observe a decrease in CD4+/CD8+ T cell ratio and M1/M2 macrophage ratio along the FAP disease continuum. Advanced dysplastic polyps show a higher population of cancer associated fibroblasts (CAFs), which likely alter the pre-cancer microenvironment. Within polyps and CRCs, we observe strong nuclear expression of beta-catenin and higher number neo-angiogenesis events, unlike FAP mucosa and normal colon counterparts. We identify an increase in cancer stem cells (CSCs) within the glandular crypts of the FAP polyps and also detect Tregs, tumor associated macrophages (TAMs) and vascular endothelial cells supporting CSC survival and proliferation. We detect a potential immunosuppressive microenvironment within the tumor ‘nest’ of FAP adenocarcinoma samples, where tumor cells tend to segregate and remain distant from the invading immune cells. TAMs were found to infiltrate the tumor area, along with angiogenesis and tumor proliferation. CAFs were found to be enriched near the inflammatory region within polyps and CRCs and may have several roles in supporting tumor growth. Neighborhood analyses between adjacent FAP mucosa and FAP polyps show significant differences in spatial location of cells based on functionality. For example, in FAP mucosa, naive CD4+ T cells alone tend to localize near the fibroblast within the stromal compartment. However, in FAP polyp, CD4+T cells colocalize with the macrophages for T cell activation. Our data are expected to serve as a useful resource for understanding the early stages of neogenesis and the pre-cancer microenvironment, which may benefit early detection, therapeutic intervention and future prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f34b5e9edbfa348b570295378a92ebadba11703" target='_blank'>
              Single-cell spatial mapping reveals alteration of cell type composition and tissue microenvironment during early colorectal cancer formation
              </a>
            </td>
          <td>
            Tuhin K. Guha, Edward D. Esplin, Aaron M. Horning, R. Chiu, Kristina Paul, Annika K. Weimer, Winston R. Becker, Rozelle Laquindanum, M. Mills, D. G. Esplin, Jeanne Shen, Emma Monte, Shannon M. White, Thomas V. Karathanos, Daniel L. Cotter, Joanna Bi, U. Ladabaum, Teri A. Longacre, Christina Curtis, William J. Greenleaf, James M Ford, Michael P. Snyder
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Thyroid cancer progression from curable well-differentiated thyroid carcinoma to highly lethal anaplastic thyroid carcinoma is distinguished by tumor cell de-differentiation and recruitment of a robust stromal infiltrate. Combining an integrated thyroid cancer single-cell sequencing atlas with spatial transcriptomics and bulk RNA-sequencing, we define stromal cell subpopulations and tumor-stromal cross-talk occurring across the histologic and mutational spectrum of thyroid cancer. We identify distinct inflammatory and myofibroblastic cancer-associated fibroblast (iCAF and myCAF) populations and perivascular-like populations. The myCAF population is only found in malignant samples and is associated with tumor cell invasion, BRAFV600E mutation, lymph node metastasis, and disease progression. Tumor-adjacent myCAFs abut invasive tumor cells with a partial epithelial-to-mesenchymal phenotype. Tumor-distant iCAFs infiltrate inflammatory autoimmune thyroid lesions and anaplastic tumors. In summary, our study provides an integrated atlas of thyroid cancer fibroblast subtypes and spatial characterization at sites of tumor invasion and de-differentiation, defining the stromal reorganization central to disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b786943f0e4090faa02f89896c2ed59c657e0d0" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations
              </a>
            </td>
          <td>
            Matthew A. Loberg, George J. Xu, , , Claudia C. Wahoski, Kailey Caroland, Megan L. Tigue, H. Hartmann, J. Gallant, Courtney J. Phifer, Andres Ocampo, Dayle K. Wang, Reilly G. Fankhauser, Kirti A. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H Belcher, Ken S. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER.
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Carcinogenesis is characterized not only by the uncontrolled growth of malignant cells but also by the disruption of the normal balance of cellular processes and intercellular interactions in the microenvironment that overcome the constraints of tissue homeostasis and support malignant growth. We profiled benign pulmonary dysplasia, carcinoma in situ, and invasive lung carcinomas at single-cell resolution to identify composite changes in cellular processes, signaling, and interactions among tumor-immune-stromal cells in the microenvironment in progressively advanced disease stages. We developed OncoTerrain, a hyperparameter-tuned model that captured synergistic multimodal signatures of an increasingly perturbed microenvironment in malignant disease stages and identified composite microenvironmental changes that supported cancer hallmarks. Key cancer-related changes in transcriptional states, cellular processes, and intercellular interactions involving immune, fibroblast, and stromal cell types preceded tumor initiation and were often synergistic. The microenvironment of increasingly malignant tissues was characterized by immune avoidance, ECM remodeling, and altered cell mobility. There were changes in cell states in fibroblasts, macrophages, and their inter-cellular interactions with other cell types, whereas T-cell activation occurred late. The in-situ carcinomas showed variations in the composite microenvironmental states that corroborated their pathology, which was not apparent at the genome level. A subset of those harbored populations of tumor and non-tumor cells with aggressive characteristics in some but not all aspects of hallmarks of carcinogenesis in the lung. We suspect that the variation in the coordination of microenvironmental cues may influence why some but not all in-situ tumors progress to the advanced stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/437dc7b34021d21e799bbd0ca24c7b8aa0ec942c" target='_blank'>
              Disruptive changes in tissue microenvironment prime oncogenic processes at different stages of carcinogenesis in lung
              </a>
            </td>
          <td>
            Vignesh Venkat, Xiaoju Hu, Antara Biswas, Ankit Saxena, Jyoti Malhotra, Gregory Reidlinger, Subhajyoti De
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In this study, we delved into the intricate cellular landscape and interactions within hepatocellular carcinoma (HCC) using single-cell RNA sequencing (scRNA-seq). Our analysis uncovered a diverse array of cell types present in HCC tumors, including endothelial cells, fibroblasts, hepatocytes, Kupffer cells, macrophages, neutrophils, NK cells, pericytes, and proliferating cells. By examining the expression profiles of key marker genes, we elucidated the distinct transcriptional signatures associated with each cell type. Notably, we observed upregulation of certain genes associated with tumor progression and metastasis, highlighting potential therapeutic targets. Additionally, our study revealed the relative proportions of each cell type within HCC tumors, shedding light on the complex cellular composition of these malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/516f83808b51104c5136da0299de6bce74fe34c4" target='_blank'>
              Single-Cell RNA Sequencing: Exploring the Cellular Landscape and Interactions in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Wenyi Wei
          </td>
          <td>2024-11-25</td>
          <td>Advances in Engineering Technology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Although immune checkpoint inhibitors (ICIs) have revolutionized melanoma treatment, many individuals either show no response or eventually develop acquired resistance. This variability could be linked to differences in the tumor immune microenvironment, pre-existing drug-resistant cells or treatment induced changes in melanoma cell states. Using single cell RNA sequencing (scRNA-seq) coupled with heritable barcodes, this study aims to lineage trace changes in melanoma transcriptional cell states and map changes in the tumor immune microenvironment to uncover if preexisting and/or therapy-induced melanoma transcriptional cell states lead to resistance to ICIs. In this study, we used YUMMER1.7 mouse melanoma cells which were transduced with barcodes under conditions that allow delivery of a single unique barcode to each cell, this served as a cell lineage tag. Each barcode is heritable and stably transcribed into RNA molecules so that individual barcoded cells can be matched with a gene expression profile from scRNA-seq outputs. The barcoded cells were subcutaneously injected into C57BL/6 mice, and once tumors were established, mice were treated with 3 cycles of anti-PD-1 and anti-CTLA-4 therapy. Lineage tracing and scRNA-seq analyses were performed on tumors harvested prior to treatment, early on treatment (Day 6), during minimal residual disease (Day 13) and upon relapse. Early on treatment (Day 6)- two tumour groups were harvested, those that responded well and others that only partially responded. Our results showed that during the early on treatment phase (Day 6) tumors that responded well and those that only partially responded displayed distinct barcodes. Additionally, prevalent barcoded cells identified at the minimal residual disease phase were also dominant upon relapse. Analysis of the transcriptional states is underway to determine if ICI therapy induces transcriptional heterogeneity in sensitive and tolerant cells and if a particular or several transcriptional states lead to relapse. This study will potentially identify predictive response biomarkers and vulnerabilities of ICI naïve and resistant cells that could be targeted to improve outcomes for melanoma patients. We will also identify neoadjuvant approaches, to remove the inherent heterogeneity within the tumor cell population, facilitating a more uniform sensitivity to ICIs.
 Citation Format: Reem Saleh, Riyaben Patel, Dane Vassiliadis, Fayrouz Hammal, Benjamin Blyth, Xin Du, Katie Fennell, Mark A. Dawson, Grant A. McArthur, Karen Sheppard. Single cell RNA-sequencing coupled with lineage tracing identifies novel clonal populations associated with immunotherapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2046528ce162da3c656f0facbde8bbcbfa6faf03" target='_blank'>
              Abstract C044: Single cell RNA-sequencing coupled with lineage tracing identifies novel clonal populations associated with immunotherapy resistance
              </a>
            </td>
          <td>
            Reem Saleh, Riyaben P. Patel, Dane Vassiliadis, Fayrouz Hammal, Benjamin Blyth, Xin Du, Katie Fennell, Mark A. Dawson, Grant A. McArthur, Karen E. Sheppard
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="T cell therapies, such as chimeric antigen receptor (CAR) T cells and T cell receptor (TCR) T cells, are a growing class of anti-cancer treatments. However, expansion to novel indications and beyond last-line treatment requires engineering cells’ dynamic population behaviors. Here we develop the tools for cellular behavior analysis of T cells from live-cell imaging, a common and inexpensive experimental setup used to evaluate engineered T cells. We first develop a state-of-the-art segmentation and tracking pipeline, Caliban, based on human-in-the-loop deep learning. We then build the Occident pipeline to collect a catalog of phenotypes that characterize cell populations, morphology, movement, and interactions in co-cultures of modified T cells and antigen-presenting tumor cells. We use Caliban and Occident to interrogate how interactions between T cells and cancer cells differ when beneficial knock-outs of RASA2 and CUL5 are introduced into TCR T cells. We apply spatiotemporal models to quantify T cell recruitment and proliferation after interactions with cancer cells. We discover that, compared to a safe harbor knockout control, RASA2 knockout T cells have longer interaction times with cancer cells leading to greater T cell activation and killing efficacy, while CUL5 knockout T cells have increased proliferation rates leading to greater numbers of T cells for hunting. Together, segmentation and tracking from Caliban and phenotype quantification from Occident enable cellular behavior analysis to better engineer T cell therapies for improved cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e025cf1fccd7422ab1c920631722b6d643225bf" target='_blank'>
              Cellular behavior analysis from live-cell imaging of TCR T cell–cancer cell interactions
              </a>
            </td>
          <td>
            Archit Verma, Changhua Yu, Stefanie Bachl, Ivan Lopez, Morgan Schwartz, Erick Moen, Nupura Kale, Carter Ching, Geneva Miller, Tom Dougherty, Edward Pao, William Graf, Carl Ward, Siddhartha Jena, Alex Marson, J. Carnevale, David Van Valen, Barbara E. Engelhardt
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Melanoma, a highly metastatic skin cancer, exhibits variations in prognosis and response to therapy based on the site of metastasis. Despite the success of immunotherapy and targeted therapies in melanoma, over half of metastatic melanoma patients will experience disease progression due to therapy resistance. The heterogeneity and plasticity of melanoma cells contribute to metastatic dissemination and therapy resistance. Our aim is to determine whether distinct clones and/or their transcriptional cell states can predict the formation of tumors in various organs and assess how these clones change over time. To identify melanoma clones across different metastatic sites, NOD scid gamma (NSG) mice and C57BL/6 mice were injected subcutaneously, intravenously or intracranially with the same pool of cells of barcoded luciferase expressing YUMMER1.7 murine melanoma cells. Transduction conditions ensured that one DNA barcode integrated into cell genomes at one barcode per cell, serving as a lineage tag. Bioluminescence imaging was performed once a week to monitor tumor growth of mice injected intravenously and intracranially. Subcutaneous tumors were measured by calliper. Mice were euthanized at different time points; day 8, 15, 22, 29 post-implantation and at ethical endpoints. Tumors were harvested and DNA sequencing was performed to identify barcodes expressed by the tumor cells. All mice developed tumors, with 100% penetrance in NSG mice. However, in C57BL/6 mice, 10% of mice intravenously injected and 27% of subcutaneously implanted mice showed complete lesion regression, suggesting that the immune system may be responsible for tumor regression. Analysis of barcodes allowed us to assess the heterogeneity of melanoma tumors at different metastatic sites and their evolution over time. Lineage tracing and clonal heterogeneity will be evaluated using the state of art technology, SPLINTR (Single-cell Profiling and LINeage Tracing), enabling us to match a cells evolution with changes in transcriptional states. Barcode analysis performed before implanting the cells and, at different timepoints in subcutaneous and lung tumors showed that dominant subclones at the baseline were also dominant in subcutaneous and lung tumors in immunocompromised mice. In contrast, dominant subclones in immunocompetent mice was those present in lower proportion at baseline. Additionally, greater variability of subclones was observed especially in lung and brain tumors, across immunocompetent mice, likely as a mechanism of resistance, enabling these tumors to overcome immunoediting. Understanding the variability of clonality between different metastatic sites and over time will improve our comprehension of the role of different subclones in organ- specific metastasis and their transcriptional cell states.
 Citation Format: Veronica L. Aedo Lopez, Reem Saleh, Benjamin Blyth, Xin Du, Dane Vassiliadis, Katie Fennell, Davide Moi, Roberta Mazzieri, Riccardo Dolcetti, Karen E. Sheppard, Grant A. McArthur. Intra- and inter-tumoral heterogeneity of melanoma across different metastatic sites [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd114bc5004ce77e46072db707d1b3ec32d44c22" target='_blank'>
              Abstract B001: Intra- and inter-tumoral heterogeneity of melanoma across different metastatic sites
              </a>
            </td>
          <td>
            Veronica L. Aedo Lopez, Reem Saleh, Benjamin Blyth, Xin Du, Dane Vassiliadis, Katie Fennell, D. Moi, Roberta Mazzieri, Riccardo Dolcetti, Karen E. Sheppard, Grant A. McArthur
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract High-dimensional, spatial single-cell technologies, such as CyTOF imaging mass cytometry (IMC), provide detailed information regarding locations of a large variety of cancer and immune cells in microscopic scales in tumor microarray slides obtained from patients prior to immune checkpoint inhibitor (ICI) therapy. An important question is how the initial spatial organization of these cells in the tumor microenvironment (TME) changes with time and regulates tumor growth and eventually outcomes as patients undergo ICI therapy. Utilizing IMC data of melanomas of patients who later underwent ICI therapy, we develop a spatially resolved interacting cell system model that is calibrated against patient response data to address the above question. We find that the tumor fate in these patients is determined by the spatial organization of activated CD8+ T cells, macrophages, and melanoma cells and the interplay between these cells that regulate exhaustion of CD8+ T cells. We find that fencing of tumor cell boundaries by exhausted CD8+ T cells is dynamically generated from the initial conditions that can play a protumor role. Furthermore, we find that specific spatial features such as co-clustering of activated CD8+ T cells and macrophages in the pretreatment samples determine the fate of the tumor progression, despite stochastic fluctuations and changes over the treatment course. Our framework enables the determination of mechanisms of interplay between a key subset of tumor and immune cells in the TME that regulate clinical response to ICIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9225e8a7e492a347e0673b0bb1568c995b67d250" target='_blank'>
              Spatial organization and stochastic fluctuations of immune cells impact clinical responsiveness to immunotherapy in melanoma patients
              </a>
            </td>
          <td>
            Giuseppe Giuliani, William Stewart, Zihai Li, Ciriyam Jayaprakash, Jayajit Das
          </td>
          <td>2024-11-26</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Inadequate response to androgen deprivation therapy (ADT) frequently arises in prostate cancer, driven by cellular mechanisms that remain poorly understood. Here, we integrated single-cell RNA sequencing, single-cell multiomics, and spatial transcriptomics to define the transcriptional, epigenetic, and spatial basis of cell identity and castration response in the mouse prostate. Leveraging these data along with a meta-analysis of human prostates and prostate cancer, we identified cellular orthologs and key determinants of ADT response and resistance. Our findings reveal that mouse prostates harbor lobe-specific luminal epithelial cell types distinguished by unique gene regulatory modules and anatomically defined androgen-responsive transcriptional programs, indicative of divergent developmental origins. Androgen-insensitive, stem-like epithelial populations - resembling human club and hillock cells - are notably enriched in the urethra and ventral prostate but are rare in other lobes. Within the ventral prostate, we also uncovered two additional androgen-responsive luminal epithelial cell types, marked by Pbsn or Spink1 expression, which align with human luminal subsets and may define the origin of distinct prostate cancer subtypes. Castration profoundly reshaped luminal epithelial transcriptomes, with castration-resistant luminal epithelial cells activating stress-responsive and stemness programs. These transcriptional signatures are enriched in tumor cells from ADT-treated and castration-resistant prostate cancer patients, underscoring their likely role in driving treatment resistance. Collectively, our comprehensive cellular atlas of the mouse prostate illuminates the importance of lobe-specific contexts for prostate cancer modeling and reveals potential therapeutic targets to counter castration resistance. Significance Statement Androgen deprivation therapy is a mainstay in prostate cancer treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer new directions for prostate cancer modeling and pave the way toward novel therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47e36c2b5c9a5688d815f8fbf4d281fd5d7c15cd" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>167</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23b3d699dd705e1b450a74d7c903ab854deff777" target='_blank'>
              Increased spatial coupling of integrin and collagen IV in the immunoresistant clear-cell renal-cell carcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, Taylor Peak, Oscar E. Ospina, N. Chakiryan, Anders E. Berglund, Paul A Stewart, Jonathan V Nguyen, C. M. Segura, Natasha L. Francis, P. R. Echevarria, J. Chahoud, Roger Li, Kenneth Y. Tsai, Jodi A. Balasi, Yamila Caraballo Peres, J. Dhillon, Lindsey A. Martinez, Warren E. Gloria, Nathan Schurman, Seang-Jae Kim, M. Gregory, James J Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-12-05</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Immune cells determine the role of the tumor microenvironment during tumor progression, either suppressing tumor formation or promoting tumorigenesis. We analyzed the profile of immune cells in the tumor microenvironment of control mouse skins and skin tumors at the single-cell level. We identified 15 CD45+ immune cell clusters, which broadly represent the most functionally characterized immune cell types including macrophages, Langerhans cells (LC), conventional type 1 dendritic cells (cDC1), conventional type 2 dendritic cells (cDC2), migratory/mature dendritic cells (mDC), dendritic epidermal T cells (DETC), dermal γδ T cells (γδT), T cells, regulatory T cells (Tregs), natural killer cells (NK), type 2 innate lymphoid cells (ILC2), neutrophils (Neu), mast cells (Mast), and two proliferating populations (Prolif.1 and Prolif.2). Skin tumor progression reprogramed immune cells and led to a marked increase in the relative percentages of macrophages, cDC2, mDC, Tregs, and Neu. Macrophages, the largest cell cluster of immune cells in skin tumors. In addition, macrophages emerged as the predominant communication ‘hub’ in skin tumors, highlighting the importance of macrophages during skin tumor progression. In contrast, other immune cell clusters decreased during skin tumor progression, including DETC, γδT, ILC2, and LC. In addition, skin tumor progression dramatically upregulated Jak2/Stat3 expression and the interferon response across various immune cell clusters. Further, skin tumor progression activated T cells and NK cells indicated by elevated expression of IFN-γ and Granzyme B in skin tumors. Meanwhile, a pronounced infiltration of M2-macrophages and Tregs in skin tumors created an immunosuppressive microenvironment, consistent with the elevated expression of the Stat3 pathway in skin tumors. In summary, our study elucidates the immune cell landscape of epidermal neoplasms, offering a comprehensive understanding of the immune response during skin tumor progression and providing new insights into cancer immune evasion mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e55f00e717854abc6b07c45a9ea29834680a1b82" target='_blank'>
              Single-Cell Profiling Reveals Global Immune Responses during the Progression of Murine Epidermal Neoplasms
              </a>
            </td>
          <td>
            Xiying Fan, Tonya M. Brunetti, Kelsey Jackson, Dennis R. Roop
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7e21f7327e57fd7a5664b9a5fd090102737fcd" target='_blank'>
              scStateDynamics: deciphering the drug-responsive tumor cell state dynamics by modeling single-cell level expression changes
              </a>
            </td>
          <td>
            Wenbo Guo, Xinqi Li, Dongfang Wang, Nan Yan, Qifan Hu, Fan Yang, Xuegong Zhang, Jianhua Yao, Jin Gu
          </td>
          <td>2024-11-21</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The oral cavity is a critical barrier with immunosurveillance capabilities. A detailed understanding of its cellular, molecular, and spatial architecture is essential for advancing precision medicine across aerodigestive tissues. Here, we present the first integrated atlas of human adult oral and craniofacial tissues, derived from single-cell RNA sequencing of ∼250,000 cells from 70 samples across 13 niches, including salivary glands and oral mucosae. Complementary spatial atlases using a 40-plex protein and 300-plex transcriptomics panels were developed, covering >2 million cells. Using our new AstroSuite package, we identified 8 tissue cellular neighborhoods (TCNs) and 17 multicellular interaction modules (MCIMs) across gland and mucosae in health. Diverse fibrovascular TCNs emerged as centralized hubs, coordinating immune cell migration and activation. Mucosal hubs within stroma displayed higher immune cell density, diversity, and activation than glands, underscoring differences in functional architecture. In keratinized and nonkeratinized mucosae, common fibrovascular hubs, and MCIMs revealed a specialized biogeography of immunoregulation. Under chronic inflammation, mucosal TCNs were fragmented, and MCIMs involving fibroblasts and immune cells were reorganized and expanded. Focused analysis of tertiary lymphoid structures (TLS) and fibroblasts identified TLS-associated MCIMs and drug candidates targeting these spatial interactions, offering a foundational framework for broadly dissecting mucosal immunity to guide precision medicine approaches. KEY TERMS AstroSuite: A suite of bioinformatics tools, including TACIT, Constellation, STARComm, and Astrograph, used for identifying and analyzing spatial features, cell types, and interactions in tissues. Astrograph: A visualization tool within AstroSuite for exploring cellular neighborhoods and spatial relationships within tissues. Cell Ontology: A structured, hierarchical classification system that provides a controlled vocabulary for cell types based on distinct properties like morphology, lineage, function, and molecular markers. Cell Recipe: A pre-defined set of markers used to specify cell identities and states by combining proteins or transcripts. Constellation: An integration and clustering algorithm within AstroSuite that identifies Tissue Cellular Neighborhoods by dividing tissue into subregions, aiding in scalable whole-slide imaging analyses. Fibrovascular Hubs: Centralized regions where fibroblasts and vascular endothelial cells within defined Tissue Cellular Neighborhoods coordinate with immune cells. Heterotypic Interactions: Predicted receptor-ligand interactions occurring between different cell types, such as immune cells and structural cells. Homotypic Interactions: Predicted receptor-ligand interactions between cells of the same type. Multicellular Interaction Modules (MCIMs): Spatial clusters where multiple cell types engage in receptor-ligand interactions—conceptually akin to Tissue Cellular Neighborhoods. Receptor-Ligand Density Scoring: A method in STARComm used to measure the density of receptor-ligand interactions within tissue regions, predicting areas of high cell-cell communication. Spatial Drug Targeting: The approach of identifying druggable targets within specific tissue regions, guided by spatial analysis of receptor-ligand interactions within identified regions of interest. STARComm: A computational tool in AstroSuite for analyzing spatial data, focusing on quantifying receptor-ligand interactions and communication densities in tissues to discover single or Multicellular Interaction Modules. Structural Immunity: The concept that structural cells, like fibroblasts, contribute to immune regulation through interactions with immune cells to maintain homeostasis. TACIT: A tool within AstroSuite used to identify cell types and interactions in spatial data, supporting neighborhood analysis and visualization. Tiered Cell Annotation: A classification system that organizes cells into broad groups, refining to specific cell types for in-depth tissue analysis—i.e., higher tiered ontologies are more generally labeled. Tissue Cellular Neighborhoods (TCNs): Spatially organized groups of cells within a tissue that are predicted to interact with one another in specific patterns or proximities. These neighborhoods represent the local microenvironments where different cell types coordinate biological processes— conceptually akin to or Multicellular Interaction Modules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ac67a4d64356ce3ad6ba3d954b4b1f09ee77215" target='_blank'>
              The Immunoregulatory Architecture of the Adult Oral Cavity
              </a>
            </td>
          <td>
            B. Matuck, Khoa L. A. Huynh, Diana Pereira, XiuYu Zhang, Meik Kunz, Nikhil Kumar, Q. Easter, Alexandre Fernandes, Ameer N. Ghodke, Alexander V. Predeus, Lili Szabó, Nadja Harnischfeger, Z. Khavandgar, M. Beach, Paola Perez, Benedikt Nilges, Maria M. Moreno, Kang I. Ko, Sarah A. Teichmann, A. Kimple, Sarah Pringle, Kai Kretzschmar, Blake M. Warner, Inês Sequeira, Jinze Liu, K. Byrd
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Glioma is an aggressive brain tumor with a poor prognosis. Establishing an in vitro culture model that closely replicates the cellular composition and microenvironment of the original tumor has been challenging, limiting its clinical applications. Here, we present a novel approach to generate glioma organoids with a microenvironment (GlioME) from patient-derived glioma tissue. These organoids maintain the genetic and epigenetic characteristics of the primary tumor and preserve cell-to-cell interactions within the tumor microenvironment, including resident immune cells. Bulk RNA sequencing, whole exome sequencing, and DNA methylation analysis were used to confirm the molecular similarities between the organoids and primary glioma tissues. Immunofluorescence and flow cytometry were used to assess immune cell viability, comparing GlioME with floating glioma organoids. GlioME exhibited high responsiveness to chemotherapy and targeted therapy, demonstrating its potential for therapeutic screening applications. Notably, GlioME accurately predicted patient response to the recently approved MET inhibitor, vebreltinib. Thus, this organoid model provides a reliable in vitro platform for glioma microenvironment-related research and clinical drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076fe0fc7a2a2145464370151771e2e7246f87bc" target='_blank'>
              GlioME: A Novel Organoid Model Retaining the Glioma Microenvironment for Personalized Drug Screening and Therapeutic Evaluation
              </a>
            </td>
          <td>
            Chengjun Zheng, Qiaodong Chen, Peng Wang, Delong Zhang, Zheng Fang, Yutong Feng, Jie Chen, Jiahong Chen, Yiwen Fu, Bao Yang, Qian Zhao, Fei Sun, Ying Zhang, Tao Jiang, Lemin Zheng, Zhaoshi Bao
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Dysregulation of intracellular signaling networks underpins cancer. However, a systems-level elucidation of how signaling networks within distinct cell subpopulations drive cancer progression in vivo has been unattainable due to technical limitations. We developed INSIGHT (INvestigating SIGnaling network of specific cell subpopulation in Heterogeneous Tissue), a new platform technology combining fluorescence-activated cell sorting with ultra-sensitive mass spectrometry to enable phosphoproteomic characterization of rare and discrete cell subpopulations from fixed tissues. We demonstrated the broad utility of INSIGHT by analyzing the oligodendroglial cell-specific signaling network in the mouse brain. We then applied INSIGHT to investigate the rare, disseminated tumor cell subpopulation in glioblastoma patient-derived xenograft models. INSIGHT uncovered a global rewiring of signaling networks with tumor cell dissemination, marked by a transition from proliferation-associated signaling in the primary tumor cells to signaling associated with postsynapse, neuronal migration, and ion homeostasis in disseminated tumor cells. We reveal interconnections between signaling circuitries within the networks, with numerous proteins, including GluA2, exhibiting altered phosphorylation without protein expression changes, emphasizing the role of post-translational modifications in glioblastoma dissemination. We validated key phosphorylation changes and inferred differentially active kinases with tumor spread to offer new systems-level insights into glioblastoma dissemination mechanisms in vivo. INSIGHT is generally applicable to a wide range of biological systems without genetic engineering and provides quantitative phosphorylation and protein expression data for selected cell subpopulations from heterogeneous tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/decaf507fd537acca35393bd50f3bedca402f1f6" target='_blank'>
              Uncovering the signaling networks of disseminated glioblastoma cells in vivo with INSIGHT
              </a>
            </td>
          <td>
            Ryuhjin Ahn, Alicia D. D’Souza, Lawrence Long, Yufei Cui, D. Burgenske, K. Bakken, Lauren L Ott, Brett L. Carlson, Grace Zhou, Tomer M. Yaron-Barir, Ishwar N. Kohale, Charles A. Whittaker, Cameron T. Flower, Jeffrey Wyckoff, A. Tuma, Jared L. Johnson, Jann Sarkaria, Forest M. White
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1054dfecee5e5157b77728bea8bb0a9b8823603" target='_blank'>
              Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype
              </a>
            </td>
          <td>
            Aino-Maija Leppä, Karen Grimes, Hyobin Jeong, Frank Y Huang, Alvaro Andrades, A. Waclawiczek, T. Boch, Anna Jauch, S. Renders, Patrick Stelmach, Carsten Müller-Tidow, Darja Karpova, Markus Sohn, F. Grünschläger, Patrick Hasenfeld, Eva Benito Garagorri, Vera Thiel, A. Dolnik, Bernardo Rodriguez-Martin, L. Bullinger, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Alwin Krämer, A. Sanders, J. Korbel, A. Trumpp
          </td>
          <td>2024-11-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="DICER1 syndrome predisposes children and young adults to tumor development across various organs. Many of these cancers are sarcomas, which uniquely express the RNase IIIb domain-deficient form of DICER1 and exhibit consistent histological and molecular similarities regardless of their anatomical origins. To uncover their cellular origin and developmental hierarchy, we established a lineage-traceable genetically engineered mouse model that allows for precise activation of Dicer1 mutations in Hic1+ mesenchymal stromal cells. This model resulted in the development of renal tumors closely mirroring human DICER1 sarcoma histologically and molecularly. Single-cell transcriptomics coupled with targeted spatial gene expression analysis revealed a Hic1+ progenitor population marked by Pdgfra, Dpt, and Mfap4, corresponding to universal fibroblasts of steady-state kidneys. These fibroblastic progenitors exhibit the capacity to undergo rhabdomyoblastic differentiation or transition to highly proliferative anaplastic sarcoma. Investigation of patient samples identified analogous cell states. This study uncovers a fibroblastic origin for DICER1 sarcoma and provides a faithful model for mechanistic investigation and therapeutic development for tumors within the rhabdomyosarcoma spectrum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7612b5f96fea7c628834a8ae0728263c8ee1372" target='_blank'>
              Spatial single cell transcriptomic analysis of a novel DICER1 Syndrome GEMM informs the cellular origin and developmental hierarchy of associated sarcomas
              </a>
            </td>
          <td>
            F. Kommoss, Joyce Zhang, Branden Lynch, S. Chen, J. Senz, Yana Moscovitz, Lesley A. Hill, Wilder Scott, Jonathan Bush, Kenneth S. Chen, Andreas von Deimling, William D. Foulkes, Gregg B. Morin, T. M. Underhill, Yemin Wang, D. Huntsman
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d97774290c1f515233beb8849c220d76ed9e57" target='_blank'>
              Unsupervised clustering reveals noncanonical myeloid cell subsets in the brain tumor microenvironment
              </a>
            </td>
          <td>
            Ismaïl Hermelo, Tuomo Virtanen, I. Salonen, R. Nätkin, Sofia Keitaanniemi, A. Tiihonen, Suvi Lehtipuro, Laura Kummola, Ella Raulamo, K. Nordfors, H. Haapasalo, M. Rauhala, J. Kesseli, M. Nykter, J. Haapasalo, K. Rautajoki
          </td>
          <td>2025-01-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The spatial dissimilarity method is a new statistical tool to uncover complex bivariate relationships in single cells and spatial transcriptomics data, thereby addressing unresolved issues such as alternative splicing and allele-specific gene expression. By applying our spatial dissimilarity analysis method on datasets of neurons, tumor, and normal cells, we identified thousands of alternatively spliced genes and instances of allele-specific gene expression. This is particularly evident in the analysis of quiescent and transitioning cell states, where our method reveals the nuanced gene expression dynamics associated with these states. Notably, our findings highlight how allele-specific genetic variants can provide insights into the subclone architecture of normal cells and cancer cells, offering a more comprehensive understanding of cellular heterogeneity and a new insight to understand gene function at specific cell types. The spatial dissimilarity analysis method deepens our comprehension of cellular complexity and gene expression dynamics during cell state transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4172a5da9c9bc681c952ca59d27b42c8796b34e" target='_blank'>
              Spatial dissimilarity analysis in single cell transcriptomics
              </a>
            </td>
          <td>
            Quan Shi, Karsten Kristiansen
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90bace8a9791e95b3eff7c16fd892d053802f89b" target='_blank'>
              Single-cell transcriptomes of dissecting the intra-tumoral heterogeneity of breast cancer microenvironment
              </a>
            </td>
          <td>
            Peixian Chen, Kaifeng Liang, Xiaofan Mao, Qiuyuan Wu, Zhiyan Chen, Yabin Jin, Kairong Lin, Tiancheng He, Shuqing Yang, Huiqi Huang, Guolin Ye, Juntao Gao, Dan Zhou, Zhihao Zeng
          </td>
          <td>2024-12-26</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Dysregulation of enhancer-promoter communication in the context of the three-dimensional (3D) nucleus is increasingly recognized as a potential driver of oncogenic programs. Here, we profiled the 3D enhancer-promoter networks of primary patient-derived glioblastoma stem cells (GSCs) in comparison with neuronal stem cells (NSCs) to identify potential central nodes and vulnerabilities in the regulatory logic of this devastating cancer. Specifically, we focused on hyperconnected 3D regulatory hubs and demonstrated that hub-interacting genes exhibit high and coordinated expression at the single-cell level and strong association with oncogenic programs that distinguish IDH-wt glioblastoma patients from low-grade glioma. Epigenetic silencing of a recurrent 3D enhancer hub—with an uncharacterized role in glioblastoma—was sufficient to cause concordant downregulation of multiple hub-connected genes along with significant shifts in transcriptional states and reduced clonogenicity. By integrating published datasets from other cancer types, we also identified both universal and cancer type-specific 3D regulatory hubs which enrich for varying oncogenic programs and nominate specific factors associated with worse outcomes. Genetic alterations, such as focal duplications, could explain only a small fraction of the detected hyperconnected hubs and their increased activity. Overall, our study provides computational and experimental support for the potential central role of 3D regulatory hubs in controlling oncogenic programs and properties. HIGHLIGHTS - 3D regulatory “hubs” in glioblastoma enrich for highly coregulated genes at a single-cell level and expand oncogenic regulatory networks. - Targeted perturbation of a highly recurrent 3D regulatory hub in GSCs results in altered transcriptional states and cellular properties. - 3D regulatory hubs across cancer types associate with tumor-specific and universal oncogenic programs and worse outcomes. - The majority of hyperconnected hubs do not overlap with structural variants, suggesting epigenetic mechanisms. eTOC Here we profile the 3D enhancer connectomes of primary patient-derived human glioblastoma stem cells (GSCs), identify hyperconnected 3D regulatory “hubs”, and examine the impact of 3D hub perturbation on the transcriptional program and oncogenic properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef46a4bf29419cbc8b345c6cf8f3bc9f4287626" target='_blank'>
              Three-dimensional regulatory hubs support oncogenic programs in glioblastoma
              </a>
            </td>
          <td>
            Sarah L. Breves, Dafne Campigli Di Giammartino, James Nicholson, S. Cirigliano, Syed Raza Mahmood, UkJin Lee, Alexander Martínez-Fundichely, Johannes Jungverdorben, R. Singhania, Sandy Rajkumar, Raphael Kirou, L. Studer, Ekta Khurana, A. Polyzos, Howard A. Fine, E. Apostolou
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Tumor morphological features from histology images are a cornerstone of clinical pathology, diagnostic biomarkers, and basic cancer biology research. Spatial transcriptomics, which provides spatially resolved gene expression profiles overlaid on histology images, offers a unique opportunity to integrate morphological and expression features, thereby deepening our understanding of tumor biology. However, spatial transcriptomics experiments with patient samples in either clinical trials or clinical care are costly and challenging, whereas histology images are generated routinely and available for many legacy prospective cohorts of disease progression and outcomes in well-annotated cohorts. Inferring spatial transcriptomics profiles computationally from these histology images would significantly expand our understanding of tumor biology, but paired data for training multi-modal spatial-histology models remains limited. Here, we tackle this challenge by incorporating performant foundation models pre-trained on massive datasets of pathology images and single-cell RNA-Seq, respectively, which provide useful embeddings to underpin multi-modal models. To this end, we developed PathOmCLIP, a model trained with contrastive loss to create a joint-embedding space between a histopathology foundation model and a single-cell RNA-seq foundation model. We incorporate a set transformer to gather localized neighborhood tumor architecture following contrastive training, which further enhances performance and is necessary to obtain robust results. We validate PathOmCLIP across five tumor types and achieve significant performance improvements in gene expression prediction tasks over other methods. PathOmCLIP can be applied to many archived histology images, unlocking valuable clinical information and facilitating new biomarker discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0858f170113a2543c26ceee12ee7ba475e2efa6" target='_blank'>
              PathOmCLIP: Connecting tumor histology with spatial gene expression via locally enhanced contrastive learning of Pathology and Single-cell foundation model
              </a>
            </td>
          <td>
            Yongju Lee, Xinhao Liu, Minsheng Hao, Tianyu Liu, Aviv Regev
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: Melanoma, a malignant neoplasm originating from melanocytes, is a form of skin cancer with rapidly increasing global incidence, often exacerbated by UV radiation[1]. Particularly, acral melanoma, characterized by its swift metastasis and poor prognosis, underscores the significance of further research into its heterogeneity. Single-cell sequencing has been widely utilized in the study of tumor heterogeneity; however, research related to melanoma remains to be further refined. Materials and Methods: We employed single-cell RNA sequencing (scRNA-seq) transcriptomic analysis to delve into the melanoma cells from six samples of melanoma patients. This approach enabled the identification of critical melanoma cell subpopulations and their roles in melanoma progression. Subsequently, we examined the interactions among these subpopulations and analyzed their interactions with other cell types. Results: Our analysis identified C3 ID2+ melanoma cells as an early-stage subpopulation and C4 PCLAF+ cells as a late-stage subpopulation in melanoma evolution. Through our analysis, we identified C4 PCLAF+ Melanoma cells as a significant subpopulation in acral melanoma (AM), playing a pivotal role in the differentiation and development of AM. Further analysis of transcription factors, enriched pathways, cell stemness, and cell trajectories highlighted the significant role of C4 PCLAF+ melanoma cells in acral melanoma (AM) proliferation. Conclusion: This study identifies new factors influencing melanoma progression, providing a foundation for subsequent research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/229e51ffb3ee08536170b8fa38a3a390a50b76d0" target='_blank'>
              Single-cell analysis unveils cell subtypes of acral melanoma cells at the early and late differentiation stages
              </a>
            </td>
          <td>
            Mengyuan Hou, Zhijie Zhao, Shuxiao Li, Ziwei Zhang, Xin Li, Yichi Zhang, Wenyi Huang, Li Li, Wenjing Xi, Feiteng Liang, Li Lin, Yan Zhang, G. Chai
          </td>
          <td>2025-01-01</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The ability of cancer cells to alter their identity, known as lineage plasticity, is crucial for tumor progression and therapy resistance. In lung adenocarcinoma (LUAD), tumor progression is characterized by a gradual loss of lineage fidelity and the emergence of non-pulmonary identity programs. This can lead to hybrid-identity (hybrid-ID) states in which developmentally incompatible identity programs are co-activated within individual cells. However, the molecular mechanisms underlying these identity shifts remain incompletely understood. Here, we identify the gastrointestinal (GI) transcriptional regulator HNF4α as a critical driver of tumor growth and proliferation in KRAS-driven LUAD. In LUAD cells that express the lung lineage specifier NKX2-1, HNF4α can induce a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also forms an aberrant protein complex with NKX2-1, which disrupts NKX2-1 localization and dampens pulmonary identity within hybrid-ID LUAD. Sustained signaling through the RAS/MEK pathway is critical for maintaining the hybrid-ID state. Moreover, RAS/MEK inhibition augments NKX2-1 chromatin binding at pulmonary-specific genes and induces resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion enhances sensitivity to pharmacologic KRASG12D inhibition. Collectively, our data show that co-expression of opposing lineage specifiers leads to a hybrid identity state that can drive tumor progression and dictate response to targeted therapy in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207f2a257f041c211d79bb19f39ca01a49097c42" target='_blank'>
              Opposing lineage specifiers induce a pro-tumor hybrid-identity state in lung adenocarcinoma
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Rushmeen Tariq, Anna Waters, Kayla T. O'Toole, Eric L. Snyder
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma (GBM) cells are highly heterogeneous and invasive, leading to treatment resistance and relapse. However, the molecular regulation in and distal to tumors remains elusive. Here, we collected paired tissues from the tumor core (TC) and peritumoral brain (PTB) for integrated snRNA-seq and snATAC-seq analyses. Tumor cells infiltrating PTB from TC behave more like oligodendrocyte progenitor cells than astrocytes at the transcriptome level. Dual-omics analyses further suggest that the distal regulatory regions in the tumor genome and specific transcription factors are potential determinants of regional heterogeneity. Notably, while activator protein 1 (AP-1) is active in all GBM states, its activity declines from TC to PTB, with another transcription factor, BACH1, showing the opposite trend. Combined inhibition of AP-1 and BACH1 more efficiently attenuates the tumor progression in mice and prolongs survival than either single-target treatment. Together, our work reveals marked molecular alterations of infiltrated GBM cells and a synergy of combination therapy targeting intratumor heterogeneity in and distal to GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c193d462d0c2dbf4aee7a761498e3e53b84808" target='_blank'>
              Single-cell multi-omics sequencing uncovers region-specific plasticity of glioblastoma for complementary therapeutic targeting
              </a>
            </td>
          <td>
            Xin Wang, Qian Sun, Tianbin Liu, Haoran Lu, Xuyi Lin, Weiwen Wang, Yang Liu, Yunting Huang, Guodong Huang, Haixi Sun, Qianxue Chen, Junmin Wang, Daofeng Tian, Fan-En Yuan, Longqi Liu, Bo Wang, Ying Gu, Baohui Liu, Liang Chen
          </td>
          <td>2024-11-22</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCCs) constitute one of the few cancer indications for which mortality rates continue to rise. While Notch signaling dictates a key progenitor lineage choice during development, its role in HCC has remained controversial. Using therapeutic antibodies targeting Notch ligands and receptors to screen over 40 patient-derived xenograft models, we here identify progenitor-like HCCs that crucially depend on a tumor-intrinsic JAG1-NOTCH2 signal. Inhibiting this signal induces tumor regressions by triggering progenitor-to-hepatocyte differentiation, the same cell fate-switch that Notch controls during development. Transcriptomic analysis places the responsive tumors within the well-characterized progenitor subclass, a poor prognostic group of highly proliferative tumors, providing a diagnostic method to enrich for Notch-dependent HCCs. Furthermore, single-cell RNA sequencing uncovers a heterogeneous population of tumor cells and reveals how Notch inhibition shifts cells from a mixed cholangiocyte-hepatocyte lineage to one resembling mature hepatocytes. Analyzing the underlying transcriptional programs brings molecular detail to this process by showing that Notch inhibition de-represses expression of CEBPA, which enables the activity of HNF4α, a hepatocyte lineage factor that is otherwise quiescent. We thus describe a compelling and targetable dependency in a poor-prognosis class of HCCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3764bfcf8bf124425bfb62c588b7e271cb0186df" target='_blank'>
              Notch signaling maintains a progenitor-like subclass of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kerstin Seidel, Robert Piskol, Thi Thu Thao Nguyen, Amy Shelton, Charisa L. Cottonham, Cecile C. de la Cruz, Joseph Castillo, Jesse Garcia, Udi Segal, Mark Merchant, Yeqing Angela Yang, Jasmine Chen, Musa Ahmed, Alexis Scherl, Rajesh Vij, Lluc Mosteiro, Yan Wu, Z. Modrusan, C. Metcalfe, Christopher W. Siebel
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Spatial transcriptomics offers unprecedented insights into the complex cellular landscapes of tissues, particularly in cancer research where understanding the tumor microenvironment is crucial. However, interpreting the vast and intricate data generated by this technology remains a significant challenge. This study explores the potential of Large Language Models (LLMs) to assist in the analysis and interpretation of spatial transcriptomic data from a murine melanoma tumor model. We first evaluated the performance of multiple LLM models in describing and quantifying spatial gene expression patterns. Our benchmarking revealed that spatial transcriptomics data interpretation proved challenging for most models, with only a few demonstrating sufficient capability for this complex task. Using Claude 3.5 Sonnet, which showed the highest accuracy in spot quantification and pattern recognition, we developed a systematic workflow for analyzing the tumor immune landscape. The model first assisted in identifying and prioritizing panels of M1 and M2 macrophage-associated markers through knowledge-driven scoring. It then demonstrated remarkable ability to integrate spatial expression data with extensive immunological knowledge, providing sophisticated interpretation of local immune organization. When analyzing individual tumor regions, the model identified coordinated immunosuppressive mechanisms including metabolic barriers and disrupted pro-inflammatory signaling cascades, findings that both aligned with and extended current understanding of tumor immunology. This study highlights the potential of LLMs as powerful assistive tools in spatial transcriptomics analysis, capable of combining advanced pattern recognition with extensive knowledge integration to enhance data interpretation. While significant development work remains to make such workflows scalable, our proof of concept demonstrates that LLMs can help accelerate the translation of spatial transcriptomics data into biological insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1287bc0c305d5950e4e6f2901503d04b32465b5e" target='_blank'>
              Tackling the Complexity of Spatial Transcriptomics Data Interpretation with Large Language Models
              </a>
            </td>
          <td>
            Taushif Khan, Colleen M. Farley, John J. Wilson, Chih-Hao Chang, Damien Chaussabel
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lymph nodes (LNs) are secondary lymphoid organs where lymphocytes interact with antigen presenting cells to initiate adaptive immune responses within microenvironments established by resident stromal cells. LNs are also the major site of growth of follicular lymphoma (FL) and chronic lymphocytic leukemia (CLL), B cell neoplasms that alter the stromal architecture of LNs in highly stereotypic ways. To characterize FL and CLL cells within the LN microenvironment, we developed a pipeline for single-cell RNA sequencing of all resident LN cells. We observed that proliferation of FL and CLL cells within specialized niches commences with transient upregulation of MYC, subsequent downregulation of which may act to limit the growth potential of these indolent neoplasms. Proliferating FL cells within neoplastic follicles co-localized with follicular dendritic cells, whereas proliferating CLL cells were spatially associated with a distinct set of fibroblasts expressing CCL19 that localized to proliferation centers. We used informatic analyses and microscopy to identify and validate interacting sets of ligand-receptor pairs between proliferating neoplastic B cells, immune cells and stromal fibroblasts, including interactions involving CD74-MIF, TNFRSF13C (BAFF receptor), immunomodulatory factors such as CD55 and Galectin-9 (Gal9), and adhesion molecules. Our analyses highlight common features of these two microenvironment-dependent neoplasms and provide a roadmap for identifying vulnerabilities and new therapeutic strategies. Key points Nodal FL and CLL cells have similar proliferative programs and predicted ligand-receptor interactions with immune and stromal cells. Proliferation of FL and CLL cells occurs in distinct lymph node niches defined by different types of fibroblasts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927c055d766ad9be825511f77a8550176d0b49cd" target='_blank'>
              The follicular lymphoma and chronic lymphocytic leukemia proliferative microenvironment at single-cell resolution
              </a>
            </td>
          <td>
            Antonio Ferreira, Shumei Wang, Laryssa Poluben, Joshua D. Brandstadter, Eric Perkey, Steven Sotirakos, Mai Drew, Li Pan, Madeleine E. Lemieux, David M. Dorfman, Brent Shoji, Ivan Maillard, S. Blacklow, Jon C. Aster
          </td>
          <td>2024-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background Low-grade endometrial stromal sarcoma (LG-ESS) is a rare uterine malignancy characterized by its complex tumor microenvironment (TME) and high recurrence rates, posing challenges to accurate prognosis and effective treatment. Identifying prognostic biomarkers is essential for improving patient stratification and guiding therapeutic strategies. Methods Using single-cell transcriptome analysis combined with H&E and multiplex immunofluorescence staining, we identified a subpopulation of tumor cells in LG-ESS and further validated the association of this subpopulation and its characteristic genes with LG-ESS prognosis by molecular characterization and bulk transcriptome data. Results Our analysis reveals multiple cellular subpopulations within the tumor tissue, particularly a tumor cell subpopulation among them which is associated with poor prognosis. Originating from normal stromal fibroblasts, this subpopulation appears to play a crucial role in TME remodeling, smooth muscle cell behavior, and potentially in tumorigenesis and metastasis. Of particular interest in this subpopulation is the highly expressed FGF12 gene, which is significantly associated with a shortened survival in ESS, highlighting its potential as a prognostic biomarker. Conclusion Our study reveals the complexity of TME within the LG-ESS and highlights the role that tumor cell subpopulations play in disease progression and patient prognosis. The identification of FGF12 as a prognostic biomarker suggests a new approach for the personalized treatment and prognosis monitoring of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e49616fef8441ab9c968f5c960e3e63e7204d31e" target='_blank'>
              Single-cell RNA-seq reveals FGF12 as a prognostic biomarker in low-grade endometrial stromal sarcoma
              </a>
            </td>
          <td>
            Yu Miao, Meng Dong, Qiyin Zhou, Julia Thiel, Na Li, Ying Cai, Dan Yuan, Haitao Wang, Su-Han Jin, Hua Yang, Jinjing Wang, Benjamin Frey, Udo S. Gaipl, Hu Ma, Jianghu Zhou
          </td>
          <td>2024-11-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Computerized chest tomography (CT)-guided screening in populations at risk for lung cancer has increased the detection of preinvasive subsolid nodules, which progress to solid invasive adenocarcinoma. Despite the clinical significance, there is a lack of effective therapies for intercepting the progression of preinvasive to invasive adenocarcinoma. To uncover determinants of early disease emergence and progression, we used integrated single-cell approaches, including scRNA-seq, multiplexed imaging mass cytometry and spatial transcriptomics, to construct the first high-resolution map of the composition, lineage/functional states, developmental trajectories and multicellular crosstalk networks from microdissected non-solid (preinvasive) and solid compartments (invasive) of individual part-solid nodules. We found that early disease initiation and subsequent progression are associated with the evolution of immune-suppressive cellular phenotypes characterized by decreased cytotoxic CD8 T and NK cells, increased T cell exhaustion and accumulation of immunosuppressive regulatory T cells (Tregs) and M2-like macrophages expressing TREM2. Within Tregs, we identified a unique population of 4-1BB+ Treg subset enriched for the IL2-STAT5 suppressive pathway with transcription profiles supporting discrete metabolic alterations. Spatial analysis showed increased density of suppressive immune cells around tumor cells, increased exhaustion phenotype of both CD4 and CD8 T cells expressing chemokine CXCL13, and spatial micro-complex of endothelial and lymphocyte interactions within tertiary lymphoid structures. The single-cell architecture identifies determinants of early disease emergence and progression, which may be developed not only as diagnostic/prognostic biomarkers but also as targets for disease interception. Additionally, our dataset constitutes a valuable resource for the preinvasive lung cancer research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f08ee8028223311574c20ea0561992916e40482b" target='_blank'>
              Acquisition of discrete immune suppressive barriers contributes to the initiation and progression of preinvasive to invasive human lung cancer
              </a>
            </td>
          <td>
            L. Yoffe, B. Bhinder, Sung Wook Kang, Haoran Zhang, Arshdeep Singh, H. Ravichandran, G. Markowitz, Mitchell T. Martin, Junbum Kim, Chen Zhang, O. Elemento, Wesley Tansey, Stewart Bates, T. McGraw, Alain Borczuk, Hyun-Sung Lee, Nasser K Altorki, Vivek Mittal
          </td>
          <td>2025-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background The adherent-to-suspension transition (AST) describes how solid tumor cells reprograms their anchorage dependency via defined hematopoietic transcriptional regulators to disseminate into circulating tumor cells (CTCs). It remains unclear, however, whether the dissemination and colonization processes require dynamic plasticity of AST factor expression in CTCs to permit the completion of the metastatic cascade. Methods We investigated AST factor-mediated adherent-suspension plasticity (ASP) in a newly developed cell-based dissemination assay. Here we identified a rare population of CTC-like cells spontaneously suspended from highly confluent breast cancer cells under chronic stress conditions that mimic the tumor microenvironment (TME). We investigated the oscillatory dynamics of AST factor expression in these CTC-like cells and their role in dissemination and colonization. Furthermore, we performed a single-cell transcriptomic analysis of matched primary tumors, CTCs, and metastatic lesions from newly enrolled de novo metastatic breast cancer patients, focusing on the plasticity of AST factor expression during the metastatic cascade in vivo. Results Using data obtained from the dissemination assay and de novo metastatic breast cancer patients, we have demonstrated the dynamic regulation of AST factor expression during the dissemination and colonization of CTCs both in vitro and in vivo. From our dissemination assay of breast cancer cell lines, we show that AST factor induction is required for the reprogramming of anchorage dependency and spontaneous detachment of CTC-like cells. We also show that subsequent AST factor suppression is required for the reattachment of CTCs during colonization. Single-cell transcriptome analyses in matched specimens of primary tumor, CTC, and metastatic lesions obtained from de novo metastatic breast cancer patients confirmed the oscillatory dynamics of AST factors and related gene signatures related to the adherent-suspension plasticity. Conclusion Our results show that the reprogramming of anchorage dependency in solid tumor cells via AST factor-mediated ASP plays a critical role in the dissemination and colonization of CTCs. These findings highlight the potential of targeting AST factors to develop effective anti-metastatic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24dbe2635c48e91654f5538698e4e2b76cbd216e" target='_blank'>
              Adherent-suspension plasticity promotes the dissemination and colonization of circulating tumor cells
              </a>
            </td>
          <td>
            Hyunbin D. Huh, Yujin Sub, Jee Hung Kim, Hannah Lee, Jongwook Oh, Dong Ki Lee, Jong Hwa Byun, Woo Yang Pyun, S. Bae, Sehyung Pak, Hai-Chon Lee, Yu Suk Choi, Junjeong Choi, Joon Jeong, H. Gee, Hyun Woo Park
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Immune checkpoint inhibitors such as anti-PD-1 antibodies (aPD1) can be effective in treating advanced cancers. However, many patients do not respond and the mechanisms underlying these differences remain incompletely understood. In this study, we profile a cohort of patients with locally-advanced or metastatic basal cell carcinoma undergoing aPD-1 therapy using single-cell RNA sequencing, high-definition spatial transcriptomics in tumors and draining lymph nodes, and spatial immunoreceptor profiling, with long-term clinical follow-up. We find that successful responses to PD-1 inhibition are characterized by an induction of B-cell receptor (BCR) clonal diversity after treatment initiation. These induced BCR clones spatially co-localize with T-cell clones, facilitate their activation, and traffic alongside them between tumor and draining lymph nodes to enhance tumor clearance. Furthermore, we validated aPD1-induced BCR diversity as a predictor of clinical response in a larger cohort of glioblastoma, melanoma, and head and neck squamous cell carcinoma patients, suggesting that this is a generalizable predictor of treatment response across many types of cancers. We discover that pre-treatment tumors harbor a characteristic gene expression signature that portends a higher probability of inducing BCR clonal diversity after aPD-1 therapy, and we develop a machine learning model that predicts PD-1-induced BCR clonal diversity from baseline tumor RNA sequencing. These findings underscore a dynamic role of B cell diversity during immunotherapy, highlighting its importance as a prognostic marker and a potential target for intervention in non-responders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c13f381588d31078af3cc3cb23453357caf14bf3" target='_blank'>
              Induced B-Cell Receptor Diversity Predicts PD-1 Blockade Immunotherapy Response
              </a>
            </td>
          <td>
            Yonglu Che, Jinwoo Lee, Farah Abou-Taleb, Kerri E. Rieger, Ansuman T. Satpathy, A. L. S. Chang, Howard Y. Chang
          </td>
          <td>2024-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/332b50c1b783afb4c65a813ff9aedb52f38aaf88" target='_blank'>
              Spatial tumor immune heterogeneity facilitates subtype co-existence and therapy response in pancreatic cancer
              </a>
            </td>
          <td>
            Lukas Klein, M. Tu, Niklas Krebs, Laura Urbach, Daniela Grimm, Muhammad Umair Latif, Frederike Penz, Anna Blandau, Xueyan Wu, Rebecca Diya Samuel, Stefan Küffer, F. Wegwitz, N. Chan, Kazeera Aliar, F. Vyas, Uday Kishore, E. Hessmann, Andreas Trumpp, Elisa Espinet, A. Papantonis, Rama Khokha, V. Ellenrieder, Barbara T. Grünwald, Shiv K. Singh
          </td>
          <td>2025-01-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e01bd46496bf7ca9aab2045ddff751c6da9e8f4" target='_blank'>
              Tumor microenvironment remodeling after neoadjuvant chemoradiotherapy in local advanced rectal cancer revealed by single-cell RNA sequencing
              </a>
            </td>
          <td>
            Wen-jing Su, Yuhang Ling, Xiaodong Yang, Yong Wu, Chungen Xing
          </td>
          <td>2024-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Understanding the phenotypic transitions of cancer cells is crucial for elucidating tumor progression mechanisms, particularly the transition from a non-invasive spheroid phenotype to an invasive network phenotype. We developed an agent-based model (ABM) using Compucell3D, an open-source biological simulation software, to investigate how varying biophysical and biochemical parameters influence emerging properties of cellular communities, including cell growth, division, and migration. Our focus was on cell-cell contact adhesion and matrix remodeling effects on cancer cell migration. We simplified enzymatic remodeling of the extracellular matrix and the subsequent enhancements to cellular chemotaxis or durotaxis as a combined effect of localized cellular secretion of a chemoattractant. By varying the chemoattractant secretion rate and contact adhesion energy, we simulated their effects on cellular behavior and driving the transition from a spheroid phenotype to a network phenotype. The model serves as a digital twin for 3D cancer cell culture, simulating cancer cell growth, division, and invasion over 1 week, validated against published data. The simulations track the emergent morphological and collective phenotype changes using key metrics such as cell circularity and invasion. Our findings indicate that increased chemoattractant secretion enhances the invasiveness of the collective cells, promoting the transition to a network phenotype. Additionally, changing cell-cell contact energy from a strong cell-cell adhesion to a weak cell-cell adhesion affects the compactness of the spheroids, resulting in lower circularity and increased collective cell invasion. Our work advances the understanding of tumor progression by providing insights into the biophysical mechanisms behind invasive cancer cell phenotypic transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6e2c13239072839fce8a0eaeadba76fadb7701" target='_blank'>
              Agent-based modeling reveals impacts of cell adhesion and matrix remodeling on cancer collective cell migration phenotypes
              </a>
            </td>
          <td>
            Temitope O. Benson, Mohammad Aminul Islam, Kailei Liu, Ashlee N. Ford Versypt
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 The tumor microenvironment (TME) comprises diverse cell types whose interactions are crucial for cancer progression. Traditional bulk tissue analyses mask cell-type-specific epigenetic changes that may drive disease progression. We used DNA methylation data and a reference- based cell-type deconvolution algorithm — hierarchical tumor immune microenvironment epigenetic deconvolution (HiTIMED) — to quantify cell types in the TME. Then, we employed CellDMC, a statistical interaction testing framework that can use cell type proportions in conjunction with differential methylation analysis to identify cell-specific differential methylation. Genome-scale DNA methylation profiles from breast cancer tissue (n=609) and normal breast tissue (n=230) samples were accessed in GEO and GTEx. Our novel approach leverages bulk DNA methylation data to infer cell-type-specific epigenetic landscapes, circumventing challenges and limitations associated with direct single-cell methylation profiling, such as high costs and technical variability. Reference-based computational tools allow us to dissect complex tissue compositions and reveal distinct methylation patterns and pathways that may be unique to individual cell types, providing insights that are not discernible from traditional bulk tumor analysis approaches. Compared with nontumor normal samples, we identified significant DNA methylation alterations (FDR≤0.05) across tumor, stromal, endothelial, epithelial, lymphocytes, and myeloid cells in breast tumors. Tumor cells exhibited the highest number, 8,510, differentially methylated cytosines (DMCs) with 1,966 of these unique to tumor cells. Stromal, endothelial, and epithelial cells also showed considerable methylation alterations, with a total of 530, 440, 339 DMCs, respectively, with 5 DMCs unique to epithelial cells. Despite lower immune cell proportions in our samples, we identified 104 lymphocyte specific DMCs and 185 myeloid specific DMCs, with 14 and 20 DMCs unique to these cell types respectively. The unique DMCs identified in each cell type suggest distinct epigenetic landscapes that may drive specific cellular functions and interactions within the TME. Importantly, Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analysis of the annotated cell-type-specific DMCs revealed distinct pathways that were not apparent in the pathway analysis of bulk tumor versus normal tissue, underscoring the value of examining epigenetic alterations at the cell-type level. Our findings highlight the opportunity to leverage DNA-based cell typing with DNA methylation data to discern cell-specific molecular alterations. By focusing on cell-type-specific alterations, we can better understand the epigenetic mechanisms driving TME crosstalk, such as the activation of stromal cells and the modulation of immune responses by tumor cells. Our findings underscore the need for targeted therapeutic strategies that consider the unique epigenetic landscapes of individual cell types within the TME, offering new avenues for improving cancer treatment outcomes.
 Citation Format: Barbara Karakyriakou, Brock C. Christensen, Lucas A. Salas. Identification of cell-specific DNA methylation alterations in the breast tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae2a87bb3a64a66fe890b4d4d581317faf06571" target='_blank'>
              Abstract B033: Identification of cell-specific DNA methylation alterations in the breast tumor microenvironment
              </a>
            </td>
          <td>
            Barbara Karakyriakou, Brock C. Christensen, Lucas A Salas
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e6a9e6b43769b80db465f9dd9f678e6c40ef5c" target='_blank'>
              Single cell and spatial transcriptomics highlight the interaction of club-like cells with immunosuppressive myeloid cells in prostate cancer
              </a>
            </td>
          <td>
            A. Kiviaho, Sini K. Eerola, H. Kallio, M. K. Andersen, Miina Hoikka, A. Tiihonen, I. Salonen, Xander Spotbeen, A. Giesen, C. T. Parker, S. Taavitsainen, Olli Hantula, Mikael Marttinen, Ismaïl Hermelo, Mazlina Ismail, Elise Midtbust, M. Wess, W. Devlies, Abhibhav Sharma, S. Krossa, Tomi Häkkinen, Ebrahim Afyounian, K. Vandereyken, Sam Kint, J. Kesseli, T. Tolonen, T.L.J. Tammela, T. Viset, Øystein Størkersen, G. Giskeødegård, M. B. Rye, T. Murtola, A. Erickson, Leena Latonen, G. Bova, Ian G. Mills, S. Joniau, J. Swinnen, Thierry Voet, T. Mirtti, Gerhardt Attard, Frank Claessens, T. Visakorpi, K. Rautajoki, May-Britt Tessem, Alfonso Urbanucci, M. Nykter
          </td>
          <td>2024-11-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Cell tracking in chimeric models is essential yet challenging, particularly in developmental biology, regenerative medicine, and transplantation studies. Existing methods, such as fluorescent labeling and genetic barcoding, are technically demanding, costly, and often impractical for dynamic, heterogeneous tissues. To address these limitations, we propose a computational framework that leverages sex as a surrogate marker for cell tracking. Our approach uses a machine learning model trained on single-cell transcriptomic data to predict cell sex with high accuracy, enabling clear distinction between donor (male) and recipient (female) cells in sex-mismatched chimeric models. The model identifies specific genes critical for sex prediction and has been validated using public datasets and experimental flow sorting, confirming the biological relevance of the identified cell populations. Applied to skeletal muscle macrophages, our method revealed distinct transcriptional profiles associated with cellular origins. This pipeline offers a robust, cost-effective solution for cell tracking in chimeric models, advancing research in regenerative medicine and immunology by providing precise insights into cellular origins and therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2607809a828288d92a6fadd5cfee754b3bcdb55" target='_blank'>
              CellSexID: Sex-Based Computational Tracking of Cellular Origins in Chimeric Models
              </a>
            </td>
          <td>
            Huilin Tai, Qian Li, Jingtao Wang, Jiahui Tan, Ryann Lang, Basil J. Petrof, Jun Ding
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Identifying spatial domains is critical for understanding breast cancer tissue heterogeneity and providing insights into tumor progression. However, dropout events introduces computational challenges and the lack of transparency in methods such as graph neural networks limits their interpretability. This study aimed to decipher disease progression-related spatial domains in breast cancer spatial transcriptomics by developing the three graph regularized non-negative matrix factorization (TGR-NMF). A unitization strategy was proposed to mitigate the impact of dropout events on the computational process, enabling utilization of the complete gene expression count data. By integrating one gene expression neighbor topology and two spatial position neighbor topologies, TGR-NMF was developed for constructing an interpretable low-dimensional representation of spatial transcriptomic data. The progressive lesion area that can reveal the progression of breast cancer was uncovered through heterogeneity analysis. Moreover, several related pathogenic genes and signal pathways on this area were identified by using gene enrichment and cell communication analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744a8472d128321ec52aed7c2a2480cf5c10946b" target='_blank'>
              Deciphering progressive lesion areas in breast cancer spatial transcriptomics via TGR-NMF
              </a>
            </td>
          <td>
            Juntao Li, Shan Xiang, Dongqing Wei
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. A better understanding of the temporal dynamics and specific pathways leading into and out of the persister state is needed to identify strategies to prevent treatment failure. Using spatial transcriptomics in patient-derived xenograft models, we captured clonal lineage evolution during treatment. The persister state showed increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients. Phylogenetic tracing identified intrinsic and acquired resistance mechanisms (e.g., dual specific phosphatases, reticulon-4, and CDK2) and suggested specific temporal windows of potential therapeutic susceptibility. Deep learning-enabled analysis of histopathological slides revealed morphological features correlating with specific cell states, demonstrating that juxtaposition of transcriptomics and histological data enabled identification of phenotypically distinct populations from using imaging data alone. In summary, this study defined state change and lineage selection during melanoma treatment with spatiotemporal resolution, elucidating how choice and timing of therapeutic agents will impact the ability to eradicate resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d694b41718972dca1b5b8aa26a4498984d65d307" target='_blank'>
              Spatiotemporal Profiling Defines Persistence and Resistance Dynamics During Targeted Treatment of Melanoma.
              </a>
            </td>
          <td>
            Jill C. Rubinstein, Sergii Domanskyi, Todd B Sheridan, Brian Sanderson, SungHee Park, Jessica Kaster, Haiyin Li, Olga Anczuków, M. Herlyn, Jeffrey H. Chuang
          </td>
          <td>2024-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Background Single-cell sequencing can provide novel insights into the understanding and treatment of diseases. In cancer, for example, intratumor heterogeneity is a major cause of treatment resistance and relapse. Although technological progress has substantially increased the throughput of sequenced cells, single-cell sequencing remains cost and labor-intensive. Multiplexing, i.e., the pooling and subsequent joint preparation and sequencing of samples, followed by a demultiplexing step, is a common practice to reduce expenses and confounding batch effects, especially in single-cell RNA sequencing. Results Here, we introduce demoTape, a computational demultiplexing method for targeted single-cell DNA sequencing (scDNA-seq) data based on a distance metric between individual cells at single-nucleotide polymorphisms loci. To validate demoTape, we sequence three B-cell lymphoma patients separately and multiplexed on the Tapestri platform. We find similar genotypes, clones, and evolutionary histories in all three samples when comparing the individual with the demultiplexed samples. Using the three individually sequenced samples, we simulate multiplexed ground truth data and show that demoTape outperforms state-of-the-art demultiplexing methods designed for RNA sequencing data. Additionally, we demonstrate through downsampling that the inferred clonal composition remained largely stable for samples with fewer cells despite the inevitable loss in resolution of low-frequency clones. Conclusions Multiplexing and subsequent genotype-based demultiplexing of scDNA-seq will reduce costs and workload, eventually allowing the sequencing of more samples. This will open new possibilities and accelerate the investigation of biological questions where cellular heterogeneity on the genomic level plays a crucial role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854d13588a800839a9fb4c222c2fbfc27f0a2f60" target='_blank'>
              DemoTape: Computational demultiplexing of targeted single-cell sequencing data
              </a>
            </td>
          <td>
            Nico Borgsmüller, Jack Kuipers, Johannes Gawron, Marco Roncador, Marcel Pohly, Erkin Acar, Thi Huong Lan Do, Stefanie Reisenauer, Mirjam Judith Feldkamp, C. Beisel, Thorsten Zenz, A. Moor, N. Beerenwinkel
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af37e793e7291e5ec4163cfb7481755bd374769" target='_blank'>
              Spatial transcriptomics reveals unique metabolic profile and key oncogenic regulators of cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Limin Zhou, Jiejie Liu, Peipei Yao, Xing Liu, Fei Chen, Yu Chen, Li Zhou, Chao Shen, You Zhou, Xin Du, Junbo Hu
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Epithelial-to-mesenchymal transition (EMT) is a reversible process during which epithelial cells lose their epithelial characteristics and acquire mesenchymal features, that are important for tumor initiation, progression, stemness, metastasis, and therapy resistance. Different EMT states, each with distinct functional properties, have been identified across various tumor models. These EMT states reside within specific physical niches, characterized by unique stromal cell populations. Understanding the cellular interactions within the tumor microenvironment (TME) and their influence on EMT could lead to novel therapeutic strategies. To understand how the TME regulates the distinct tumor states, we performed single-cell RNA sequencing of the tumor cells and their TME in mouse cancer models presenting EMT. We identified distinct subsets of cancer-associated fibroblasts (CAFs), macrophages, neutrophils, endothelial cells, pericytes, adipocytes, dendritic cells, mast cells, Schwann cells, and T cells in different tumor types. To unravel how TME-tumor cell interactions regulates EMT, we used different bioinformatic approaches to identify ligand-receptor pairs that are likely to organize the tumor niche and to regulate tumor transition states. In vitro treatment of tumor cells with some of these ligands demonstrated their EMT-promoting effects. These findings provide novel mechanistic insights on the crosstalk between tumor cells and the surrounding TME, that may have important implications for anti-cancer therapy.
 Citation Format: Cédric Blanpain. Regulation of EMT tumor states by stromal cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr IA019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ded367efe26987fbf64cb06b553ba3a3b5b7a19" target='_blank'>
              Abstract IA019: Regulation of EMT tumor states by stromal cells
              </a>
            </td>
          <td>
            Cédric Blanpain
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cells, immune cells, and stromal cells within the tumor microenvironment (TME) collaboratively influence disease progression and therapeutic responses. The nutrient-limited conditions of the TME, particularly the scarcity of glucose, amino acids, and lipids, challenge cancer cell survival1–4. However, the metabolic constraints faced by immune and stromal cells in comparison to cancer cells, and how these limitations affect therapeutic outcomes, remain poorly understood. Here, we introduce Dual Ribosome Profiling (DualRP), a method that allows for simultaneous analysis of translation and identification of ribosome stalling, revealing amino acid shortages in different cell types within tumors. Using DualRP, we uncover that interactions between cancer cells and fibroblasts trigger an inflammatory response, mitigating amino acid limitations during glucose starvation. In immunocompetent mouse models, we observe that immune checkpoint blockade therapy induces serine and glycine restrictions specifically in T cells, but not in cancer cells. We further demonstrate that these amino acids are essential for optimal T cell function both in vitro and in vivo, highlighting their critical role in effective immunotherapy. Our findings show that therapeutic interventions create distinct metabolic demands across different tumor cell types, with nutrient availability significantly influencing the success of immunotherapy. DualRP’s ability to explore cell type-specific metabolic vulnerabilities offers a promising tool for advancing our understanding of tumor biology and improving therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cb1f538143f57c46c3d2552aff719b62450477f" target='_blank'>
              Dual Ribosome Profiling reveals metabolic limitations of cancer and stromal cells in the tumor microenvironment
              </a>
            </td>
          <td>
            Daniela Aviles-Huerta, Del Pizzo Rossella, Alexander Kowar, Ali Hyder Baig, Giuliana Palazzo, Ekaterina Stepanova, C. C. Amaya Ramirez, Sara D’Agostino, Edoardo Ratto, Catarina Pechincha, Nora Siefert, Helena Engel, Shangce Du, Silvia Cadenas-De-Miguel, Beiping Miao, Victor Cruz Vilchez, Karin Müller-Decker, Ilaria Elia, Chong Sun, Wilhelm Palm, Fabricio Loayza-Puch
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de8ac06c638611539d33f6ef5fdb316eabe9f02c" target='_blank'>
              Spatio-temporal transcriptomics of chromothriptic SHH-medulloblastoma identifies multiple genetic clones that resist treatment and drive relapse
              </a>
            </td>
          <td>
            Ilia Kats, Milena Simovic-Lorenz, Hannah Sophia Schreiber, Pooja Sant, Jan-Philipp Mallm, Verena Körber, Albert Li, Pravin Velmurugan, Sophie Heuer, Luisa Kües, Frauke Devens, M. Sill, M. Jugold, Mahmoud Moustafa, Amir Abdollahi, Frank Winkler, A. Korshunov, Stefan M Pfister, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220a065de9e40d9837ef9539b8c30437e0e009d" target='_blank'>
              Advanced single-cell and spatial analysis with high-multiplex characterization of circulating tumor cells and tumor tissue in prostate cancer: Unveiling resistance mechanisms with the CoDuCo in situ assay
              </a>
            </td>
          <td>
            Lilli Bonstingl, Margret Zinnegger, Katja Sallinger, Karin Pankratz, Christin-Therese Müller, Elisabeth Pritz, Corinna Odar, C. Skofler, Christine Ulz, Lisa Oberauner-Wappis, Anatol Borrás-Cherrier, Višnja Somođi, Ellen Heitzer, T. Kroneis, T. Bauernhofer, Amin El-Heliebi
          </td>
          <td>2024-11-16</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly disease. Over recent decades, a better understanding of the biological mechanisms implicated in its pathogenesis has led to the development of targeted therapies and immunotherapy, resulting in improvements in patient outcomes. To enhance our understanding of lung cancer tumor biology and advance towards precision oncology, we need a comprehensive tumor profile. In recent years, novel in situ spatial multiomics approaches have emerged, offering a more detailed view of the spatial location of tumor and tumor microenvironment cells. These advancements in in situ profiling may unveil further molecular and immune mechanisms in tumor biology that will lead to the discovery of biomarkers for treatment prediction and prognosis. In this review, we provide an overview of current and emerging pathology-based approaches for spatial immune profiling in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55e58f728c302c9eea20cd5a44be684287b92352" target='_blank'>
              Novel Spatial Approaches to Dissect the Lung Cancer Immune Microenvironment
              </a>
            </td>
          <td>
            Idania Lubo, Sharia D Hernandez, I. Wistuba, L. S. Solis Soto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Breast cancer is one of the most frequently diagnosed cancers and the second leading cause of women’s cancer death. Although the five-year survival rate for women in the United States with nonmetastatic invasive breast cancer is 90%, this rate drops to less than 30% for those with tumors that have metastasized to distant organs. Epithelial-to-Mesenchymal Transition (EMT) plays a critical role in tumor metastasis by enabling the loss of epithelial characteristics and the gain of mesenchymal traits, leading to increased cell motility, invasion, and resistance to therapy. Consequently, targeting the EMT process may prevent tumor cell migration and improve patient outcomes. Rather than being a binary switch from an epithelial to a mesenchymal state, cells undergoing EMT can assume a spectrum of intermediate states, some of which may reshape the tumor microenvironment by recruiting pro-tumor immunosuppressive cells, such as regulatory T cells, M2-like macrophages, and myeloid-derived suppressor cells. However, the impact of EMT on the immune composition of the tumor microenvironment and the causative mechanisms remain poorly understood. Using multi-omics approaches in a mouse model of triple-negative breast cancer, we assessed the interaction between tumor and immune cells in the tumor microenvironment driven by the EMT. A heterogeneous mouse breast cancer cell line (4T1) was used to isolate five distinct single-cell-derived clonal populations residing in distinct states within the EMT spectrum. These EMT clones, along with the parental 4T1 cell line, were orthotopically transplanted into the mammary fat pads of BALB/c mice. Primary tumors were subsequently harvested and dissociated for single-cell RNA-sequencing (scRNA-seq) and DNA methylation analysis. Lungs were histologically evaluated for metastases. Tumors derived from an intermediate EMT clone exhibited increased tumor growth kinetics and a higher incidence of metastasis compared to epithelial or mesenchymal clones. In scRNA-seq data, tumors derived from different EMT clones largely retained their original EMT states in vivo and recruited distinct immune cell profiles. DNA methylation analysis further revealed that these tumors displayed unique methylation patterns in EMT-related CpGs. This study introduced a model that captures the complexity of the tumor microenvironment across the EMT spectrum. This was accomplished through immunophenotyping using scRNA-seq and identifying unique epigenetic patterns through DNA methylation. Our findings highlight the potential for distinct immune cell populations to be recruited to tumors in different EMT states, thereby influencing survival and metastasis outcomes. Furthermore, our results open new opportunities for developing cancer combination treatment strategies that target specific tumor-immune cell interactions.
 Citation Format: Hanxu Lu, Meisam Bagheri, Todd Miller, Diwakar Pattabiraman, Brock Christensen. Investigating the impact of tumor EMT states on immune cell infiltration in breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c944095e7498b092052ea979a0c030386cafd59" target='_blank'>
              Abstract B040: Investigating the impact of tumor EMT states on immune cell infiltration in breast cancer
              </a>
            </td>
          <td>
            Hanxu Lu, Meisam Bagheri, Todd W. Miller, D. Pattabiraman, Brock C. Christensen
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Breast tumors comprise several sub-populations of cells, so-called “clones”, with specific cellular and molecular characteristics. Some cancer clones can be more ‘aggressive’ than others in particular tumor microenvironments (TMEs), contributing to metastatic progression and drug resistance. Understanding the specific genes/pathways deregulated in these aggressive cancer cells is essential for developing new tailored treatment strategies. To accomplish this, we optimized the use of a lentiviral-based barcoding technique (Berthelet, J., et.al., Sci Adv 2021, Serrano, A., et.al., Nat Rev Cancer 2022) where cancer cells from cell lines and patient-derived xenografts are tagged with optical barcodes and transplanted into the mammary fat pads of immunodeficient mice. Upon resection of tumors and collection of metastatic organs at the experimental endpoint, we analyzed clonal fate and fitness using flow cytometry and imaging. The subsequent molecular characterization of individual clones was determined by bulk and single-cell RNA sequencing. We also leverage the use of these barcoded models to investigate the mechanisms of innate and acquired resistance to Sacituzumab Govitecan (SG), a drug recently approved for the treatment of patients with advanced breast cancer. Our studies revealed unique gene signatures and pathways associated with preferential organ tropism (in the lungs, liver and brain TMEs) and SG resistance. This comprehensive analysis, currently validated using autopsy samples and patient databases, will support the design of new predictive markers and targeted therapies for metastatic breast cancer.
 Citation Format: Sreeja Reddy Gadipally, Jean Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, Caroline Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, Charafe Jauffret Emmanuelle, Melissa Davis, Robin L. Anderson, Luciano Martelotto, Belinda Yeo, Delphine Merino. Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528d6539434045b291c3a0faa67fa6d3c29fbc80" target='_blank'>
              Abstract B005: Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance
              </a>
            </td>
          <td>
            Sreeja Gadipally, J. Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, C. Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, C. Emmanuelle, Melissa J Davis, Robin L Anderson, Luciano Martelotto, B. Yeo, Delphine Merino
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Introduction Effective treatment of breast cancer remains a formidable challenge, partly due to our limited understanding of the complex microenvironmental factors that contribute to disease pathology. Among these factors are tissue-resident perivascular cells, which play crucial roles in shaping vascular basement membranes, maintaining vessel integrity, and communicating with adjacent endothelial cells. Despite their essential functions, perivascular cells have been relatively overlooked. Identifying them by immunostaining has been challenging due to their low abundance, inherent heterogeneity, and shared marker expression with other cell types. These challenges have hindered efforts to purify pericytes and generate primary cell models for studying their biology. Methods Using a recently developed FACS method, we successfully identified and purified each cell type from breast tissues, allowing us to deep-sequence their transcriptomes and generate primary cell models of each cell type—including pericytes. Here, we used these data to analyze cell-type-specific gene expression in tumors, which revealed a strong association between pericyte-specific genes and breast cancer patient mortality. To explore this association, we defined the heterogeneity of breast pericytes using single-cell RNA sequencing and identified a broad marker for visualizing perivascular cells in breast tumors. Results Remarkably, we discovered perivascular cells dissociated from vessels and emerged as a dominant mesenchymal cell type in a subset of breast tumors that contrasted with their normal perivascular location. Moreover, when we purified pericytes from the breast and cultured them alongside breast tumor cells, we discovered that they induced rapid tumor cell growth significantly greater than isogenic fibroblast controls. Discussion These findings identify perivascular cells as a key microenvironmental factor in breast cancer, highlighting the critical need for further research to explore their biology and identify specific stimulatory mechanisms that could be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920638149619a80dd215db038826c980f92e7f53" target='_blank'>
              Breast pericytes: a newly identified driver of tumor cell proliferation
              </a>
            </td>
          <td>
            Katelyn Del Toro, Y. Licon-Munoz, William Crabtree, Tristan Oper, Christine Robbins, William C. Hines
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f233ae172c7106f684ef62640304f7d51d73186" target='_blank'>
              Integrative mapping of human CD8+ T cells in inflammation and cancer.
              </a>
            </td>
          <td>
            Ziwei Xue, Lize Wu, Ruonan Tian, Bing Gao, Yu Zhao, Bing He, Di Sun, Bingkang Zhao, Yicheng Li, Kaixiang Zhu, Lie Wang, Jianhua Yao, Wanlu Liu, Linrong Lu
          </td>
          <td>2024-11-29</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Copy number variations (CNVs), the gain or loss of genomic regions, are associated with different diseases and cancer types, where they are related to tumor progression and treatment outcome. Single cell technologies offer new possibilities to measure CNVs in individual cells, allowing to assess population heterogeneity and to delineate subclonal structures. Single cell whole-genome sequencing is considered the gold-standard for the quantification of CNVs in single cells. However, the majority of existing single cell datasets interrogate gene expression, using scRNA-seq. Consequently, several computational approaches have been developed to identify CNVs from that data modality. Nevertheless, an independent benchmarking of these methods is lacking. We used 15 scRNA-seq datasets and evaluated six popular computational methods in their ability to recover the ground truth CNVs using a large set of performance metrics. Additionally, we explored whether they could correctly identify euploid cells, especially also in fully diploid samples, and subclonal structures in heterogeneous tumor samples. We discovered several dataset-specific factors that influence the performance of the methods, such as the dataset size and the number and type of CNVs in the analyzed sample. We found that the choice of the reference dataset can have a large impact on the performance. Methods which included additional allelic information from the scRNA-seq reads performed more robustly across scenarios, but at the cost of higher runtime. Furthermore, the methods differed substantially in their additional functionalities and resource requirements. We offer a benchmarking pipeline to help identify the optimal CNV calling method for newly generated scRNA-seq datasets, and to benchmark and improve new methods performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa3f23333379961a358cfa4029d6bc3607c70188" target='_blank'>
              Benchmarking scRNA-seq copy number variation callers
              </a>
            </td>
          <td>
            Katharina T. Schmid, Aikaterini Symeonidi, Dmytro Hlushchenko, M. Richter, Maria Colomé-Tatché
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Background & Objective: Prostate cancer (PCa) cells frequently disseminate to bone where they can remain dormant for extended periods. The precise cellular and molecular mechanisms responsible remain largely unknown. The advent of spatial transcriptomics provides a unique opportunity to study key bone microenvironment-dormant PCa cell interactions in situ. Here, we developed a novel immunocompetent in vivo model of PCa dormancy in bone and employed the technology to address this major gap in knowledge. Methods: We established an in vitro protocol to induce dormancy in multiple PCa cell lines, including the C57BL/6 murine PCa cell line, RM1. Isogenic active and dormant PCA cells underwent bulk RNA Seq. We used an intra- iliac artery (IIA) approach to allow niche-specific seeding of dormant PCa within the bone. Spatial transcriptomics (10X Visium) was performed on tissue sections and R packages (Seurat & CellChat) were used for data analysis. Results: We validated in vitro PCa dormancy via cell cycle analysis, membrane dye retention and ERK/p38 ratio. RNASeq analysis of dormant PCa cells (vs. active) revealed upregulation of immunomodulatory genes such as, GDF15, BDNF, A2M, VGF, NLRP1, and CD55. C57BL/6 immunocompetent mice were inoculated with active or dormant RM1 cells via IIA. Majority of active RM1 mice reached endpoint within 15 days. In contrast, mice bearing dormant RM1 (D-RM1) had extended survival and were removed from study after 30 days. Histological analysis revealed solitary D-RM1 (panCK+/Ki67-/CC3-) on endosteal surfaces compared to macrometastases (high Ki67+) in the active RM1 group. Using IIA PBS injected tumor naïve bones as a control, we applied spatial transcriptomics (10X Visium) to interrogate the microenvironment surrounding active and dormant RM1 PCa cells. Bioinformatic analyses demonstrated higher expression of B cell markers (CD19, CD79a, CD79b, Ighm, and Pax5) and enhanced B cell activation/antigen phagocytosis signature in the D- RM1 group compared to other groups. We validated the presence of CD19 B-cells in proximity to the D-RM1 cells in subsequent tissue sections. In contrast, B cells were located less frequently and to the periphery of active RM1 metastases. Inference analysis using CellChat also identified a significant and distinct signaling pattern between CD55 and CD97 in the B cell enriched clusters in D-RM1 group. CD55/CD97 interaction is a noted mediator of B cell homing. These data suggest a potential role for distinct populations of B cells during the establishment of PCa dormancy in bone. Conclusion: Spatial transcriptomic analysis of the dormant PCa bone microenvironment reveals an enrichment of CD19+ B cells with a distinct signaling pattern. Given their known roles in immune tolerance we posit that infiltrating B cells may play a role in protecting and sustaining newly disseminated dormant PCa cells from immune mediated elimination.
 Citation Format: Mostafa M. Nasr, Ryan T. Bishop, Haley du Bois, Tao Li, Jeremy S. Frieling, Conor C. Lynch. Spatial transcriptomics reveals a distinct B cell signature in the dormant prostate cancer bone microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1da73f9c20af7ebfb3e86b110c704de45b40046e" target='_blank'>
              Abstract B044: Spatial transcriptomics reveals a distinct B cell signature in the dormant prostate cancer bone microenvironment
              </a>
            </td>
          <td>
            Mostafa M. Nasr, Ryan T Bishop, Haley du Bois, Tao Li, Jeremy S. Frieling, Conor C Lynch
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary Cancer initiation and progression are controlled not only by intrinsic factors, such as genetic mutations and mesenchymal trait acquisition, but also by dynamic spatiotemporal shifts within the tumor microenvironment. These elements induce plasticity, and enhance invasiveness, metastasis, and treatment resistance, while promoting immune evasion through the reprogramming of the immune microenvironment. However, the molecular mechanisms driving this reprogramming remain unclear. In this review, we summarize the recent pivotal findings on pancreatic cancer, which is a highly malignant disease, and overview the roles of plasticity, metabolic reprogramming, and the creation of an immunosuppressive microenvironment in disease progression. We also present our research on two crucial signaling pathways, i.e., the dynamic interplay between mutational load and the tumor microenvironment, and the acquisition of cancer plasticity via inflammatory responses and metabolic reprogramming. A comprehensive understanding of these processes is expected to transform therapeutic strategies and improve the outcomes of pancreatic cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d6b2b31d56874111eca289fe4cec7a0df598ab" target='_blank'>
              Plasticity and Tumor Microenvironment in Pancreatic Cancer: Genetic, Metabolic, and Immune Perspectives
              </a>
            </td>
          <td>
            A. Hashimoto, S. Hashimoto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Existing single-cell clustering methods are based on gene expressions that are susceptible to dropout events in single-cell RNA sequencing (scRNA-seq) data. To overcome this limitation, we proposed a pathway-based clustering method for single cells (scPathClus). scPathClus first transforms single-cell gene expression matrix into pathway enrichment matrix and generates its latent feature matrix. Based on the latent feature matrix, scPathClus clusters single cells using the method of community detection. Applying scPathClus to PDAC scRNA-seq datasets, we identified two types of cancer-associated fibroblasts (CAFs), termed csCAFs and gapCAFs, which highly expressed complement system and gap junction-related pathways, respectively. Spatial transcriptome analysis revealed that gapCAFs and csCAFs are located at cancer and non-cancer regions, respectively. Pseudotime analysis suggest a potential differentiation trajectory from csCAFs to gapCAFs. Bulk transcriptome analysis showed that gapCAFs-enriched tumors are more endowed with tumor-promoting characteristics and worse clinical outcomes, while csCAFs-enriched tumors confront stronger antitumor immune responses. Compared to established CAF subtyping methods, this method displays better prognostic relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e5196bb420146fb22bc069bed5ff6f7bc8606d" target='_blank'>
              Pathway-based clustering identifies two subtypes of cancer-associated fibroblasts associated with distinct molecular and clinical features in pancreatic ductal carcinoma
              </a>
            </td>
          <td>
            Hongjing Ai, R. Nie, Xiaosheng Wang
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancers arising from dysregulation of generally operative signaling pathways are often tissue specific, but the mechanisms underlying this paradox are poorly understood. Based on striking cell-type specificity, we postulated that these mechanisms must operate early in cancer development and set out to study them in a model of von Hippel Lindau (VHL) disease. Biallelic mutation of the VHL ubiquitin ligase leads to constitutive activation of hypoxia inducible factors HIF1A and HIF2A and is generally a truncal event in clear cell renal carcinoma. We used an oncogenic tagging strategy in which VHL-mutant cells are marked by tdTomato, enabling their observation, retrieval, and analysis early after VHL-inactivation. Our findings reveal markedly different consequences of HIF1A and HIF2A activation, but that both contribute to renal cell-type specific consequences of VHL-inactivation in the kidney. Early involvement of HIF2A in promoting proliferation within proximal tubular epithelium supports therapeutic targeting of HIF2A early in VHL disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be97f9c0f0b00dada5534b4bd4b5d705fdb68b10" target='_blank'>
              HIFα isoform specific activities drive cell-type specificity of VHL-associated oncogenesis
              </a>
            </td>
          <td>
            Joana Darc Carola Correia Lima, Madeleine Hooker, Ran Li, Ayslan B. Barros, N. Masson, Christopher W. Pugh, D. Mole, Julie Adam, Peter J. Ratcliffe, Samvid Kurlekar
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa9e49413db565400c66d264638af2b81a7c480" target='_blank'>
              Single cell-spatial transcriptomics and bulk multi-omics analysis of heterogeneity and ecosystems in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jiazhou Ye, Yan Lin, Zhiling Liao, Xing Gao, Cheng Lu, Lu Lu, Julu Huang, Xi Huang, Shilin Huang, Hongping Yu, Tao Bai, Jie Chen, Xiaobo Wang, Mingzhi Xie, Min Luo, Jinyan Zhang, Feixiang Wu, Guo-bin Wu, Liang Ma, Bangde Xiang, Lequn Li, Yongqiang Li, Xiaoling Luo, R. Liang
          </td>
          <td>2024-11-15</td>
          <td>NPJ Precision Oncology</td>
          <td>2</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69505c7dd30d328b855b8e26bb2a91fa4c9f9efa" target='_blank'>
              Computer-aided analysis reveals metallothionein-positive cancer-associated fibroblasts promote angiogenesis in gastric adenocarcinoma
              </a>
            </td>
          <td>
            Xiaolong Jin, Yu Tian, Haoran Zhu, Yuewen Sun, Zhenxing Zhang
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="We develop Spatial Perturb-Seq, an in vivo CRISPR technology that interrogates multiple genes within single cells of intact tissues. We apply Spatial Perturb-Seq to screen risk genes for neurodegenerative diseases in the mouse brain, uncovering cell autonomous and cell-cell microenvironmental effects within the spatially intact tissue. Spatial Perturb-Seq functionally screens multiple genes in situ and in vivo, identifying candidate genes underlying dysregulated neuronal intercellular communication pathways in Lrp1 signalling and ephrin-Eph receptor interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/463cf687c456c72f8cdd5ea5f55255ae0d4d529a" target='_blank'>
              Spatial Perturb-Seq: Single-Cell Functional Genomics within Intact Tissue Architecture
              </a>
            </td>
          <td>
            Kimberle Shen, W. Seow, Choong Tat Keng, Daryl Lim Shern, Ke Guo, Amine Meliani, Irfan Muhammad Bin Hajis, Kok Hao Chen, W. Chew
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="T-cell activation and clonal expansion are essential for the efficacy of immunotherapy in non-small cell lung cancer (NSCLC) patients. Since the distribution of T-cell clones might provide insights into immunogenic mechanisms, we determined the α/β TCR clonality using RNA-sequencing from frozen tumor tissue of 182 NSCLC patients and paired the results with extensive in situ image and sequence analyses of the immune microenvironment of NSCLC. TCR clonality (Gini index) patterns ranged from high T-cell clone diversity with high evenness (Gini index low) to clonal dominance with low evenness (Gini index high). TCR clonality in cancer tissue was lower than in matched normal lung (p=0.021). High Gini index correlated strongly with distinct mutations (EGFR, P53), tumor mutation burden (p<0.001), and inflamed tumor phenotypes (PRF1, GZMA, GZMB, INFG) with exhaustion signatures (LAG3, TIGIT, IDO1, PD-1, PD-L1). Correspondingly, PD-1+, CD3+, CD8A+, CD163+, and CD138+ immune cells infiltrated cancer tissue with high TCR clonality. In situ sequencing revealed that dominant T-cell clones were more often of CD8-subtype and tended to approximate the tumor cell compartment (p<0.03). In a checkpoint inhibitor-treated NSCLC patient cohort, high TCR clonality was associated with therapy response (p=0.016) and prolonged survival (p=0.003, median survival 13.8 vs 2.9 months). Our robust analysis pipeline revealed diverse TCR repertoires related to genotypes and immune phenotypes. The in situ positioning of expanded T-cell clones indicated functional impact, which was clinically confirmed in NSCLC patients receiving immunotherapy. One sentence summary T-cell clone expansion in NSCLC is associated with genetic mutations, immune phenotypes, immunotherapy response, and patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d627cf9e5819664ceb7a72e57e39ad1a7a4de3d3" target='_blank'>
              TCR clonality and TCR clonal expansion in the in situ microenvironment of non-small cell lung cancer
              </a>
            </td>
          <td>
            Hui Yu, Anastasia Magoulopoulou, Rose-Marie Amini, Maria Chatzinikolaou, Masafumi Horie, A. Lindberg, Artur Mezheyeuski, M. Backman, Andreas Metousis, Hans Brunström, Millaray Marincevic Zuniga, J. Botling, J. Mattsson, K. Kärre, Karin Leandersson, Mats Nilsson, Carina Strell, P. Micke
          </td>
          <td>2024-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a leading cause of cancer-related deaths globally. The heterogeneity of the tumor microenvironment significantly influences patient prognosis, while the diversity of tumor cells shapes its unique characteristics. A comprehensive analysis of the molecular profile of tumor cells is crucial for identifying novel molecular targets for drug sensitivity analysis and for uncovering the pathophysiological mechanisms underlying CRC. Methods We utilized single-cell RNA sequencing technology to analyze 13 tissue samples from 4 CRC patients, identifying key cell types within the tumor microenvironment. Intercellular communication was assessed using CellChat, and a risk score model was developed based on eight prognostic genes to enhance patient stratification for immunotherapeutic approaches. Additionally, in vitro experiments were performed on DLX2, a gene strongly associated with poor prognosis, to validate its potential role as a therapeutic target in CRC progression. Results Eight major cell types were identified across the tissue samples. Within the tumor cell population, seven distinct subtypes were recognized, with the C0 FXYD5+ tumor cells subtype being significantly linked to cancer progression and poor prognosis. CellChat analysis indicated extensive communication among tumor cells, fibroblasts, and immune cells, underscoring the complexity of the tumor microenvironment. The risk score model demonstrated high accuracy in predicting 1-, 3-, and 5-year survival rates in CRC patients. Enrichment analysis revealed that the C0 FXYD5+ tumor cell subtype exhibited increased energy metabolism, protein synthesis, and oxidative phosphorylation, contributing to its aggressive behavior. In vitro experiments confirmed DLX2 as a critical gene associated with poor prognosis, suggesting its viability as a target for improving drug sensitivity. Conclusion In summary, this study advances our understanding of CRC progression by identifying critical tumor subtypes, molecular pathways, and prognostic markers that can inform innovative strategies for predicting and enhancing drug sensitivity. These findings hold promise for optimizing immunotherapeutic approaches and developing new targeted therapies, ultimately aiming to improve patient outcomes in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8647de1e0192ab64820f770599e15f3edf4689c3" target='_blank'>
              Innovative strategies to optimise colorectal cancer immunotherapy through molecular mechanism insights
              </a>
            </td>
          <td>
            Quanjun Lin, Zhiqiang Wang, Jue Wang, Ming Xu, Xinyi Zhang, Peng Sun, Yihang Yuan
          </td>
          <td>2024-12-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The cascade of events leading to tumor formation includes induction of a tumor supporting neovasculature, as a primary hallmark of cancer. Developing vasculature is difficult to evaluate in vivo but can be captured using microfluidic chip technology and patient derived cells. Herein, we established an on chip approach to investigate the mechanisms promoting tumor vascularization and vascular targeted therapies via co-culture of cancer spheroids and endothelial cells in a three dimensional environment. Methods: We investigated both, tumor neovascularization and therapy, via co-culture of human derived endothelial cells and adjacently localized metastatic renal cell carcinoma spheroids on a commercially available microfluidic chip system. Metastatic renal cell carcinoma spheroids adjacent to primary vessels model tumor, and induce vessels to sprout neovasculature towards the tumor. We monitored real time changes in vessel formation, probed the interactions of tumor and endothelial cells, and evaluated the role of important effectors in tumor vasculature. In addition to wild type endothelial cells, we evaluated endothelial cells that overexpress Prostate Specific Membrane Antigen (PSMA), that has emerged as a marker of tumor associated neovasculature. We characterized the process of neovascularization on the microfluidic chip stimulated by enhanced culture medium and the investigated metastatic renal cell carcinomas, and assessed endothelial cells responses to vascular targeted therapy with bevacizumab via confocal microscopy imaging. To emphasize the potential clinical relevance of metastatic renal cell carcinomas on chip, we compared therapy with bevacizumab on chip with an in vivo model of the same tumor. Results: Our model permitted real-time, high-resolution observation and assessment of tumor-induced angiogenesis, where endothelial cells sprouted towards the tumor and mimicked a vascular network. Bevacizumab, an antiangiogenic agent, disrupted interactions between vessels and tumors, destroying the vascular network. The on chip approach enabled assessment of endothelial cell biology, vessel's functionality, drug delivery, and molecular expression of PSMA. Conclusion: Observations in the vascularized tumor on chip permitted direct and conclusive quantification of vascular targeted therapies in weeks as opposed to months in a comparable animal model, and bridged the gap between in vitro and in vivo models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0cee76c44f4581ac79061d246894e755a88b77d" target='_blank'>
              Vascularized tumor on a microfluidic chip to study mechanisms promoting tumor neovascularization and vascular targeted therapies
              </a>
            </td>
          <td>
            Magdalena Skubal, B. M. Larney, N. Phung, Juan Carlos Desmaras, Abdul Vehab Dozic, A. Volpe, Anuja Ogirala, Camila Longo Machado, Jakob Djibankov, V. Ponomarev, J. Grimm
          </td>
          <td>2025-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background Cellular plasticity, involving loss of lineage determination and emergence of hybrid cell states, plays a pivotal role in non-small cell lung cancer (NSCLC) disease progression and therapy resistance. However, the full spectrum of atypical states generated in human NSCLC and the pathways that regulate them are yet to be fully elucidated. Here we examine the role of developmental programmes, alveogenesis and branching morphogenesis (BM), in regulating phenotypic diversity in NSCLC. Methods Transcriptomic analysis of epithelial cells isolated from murine lungs at different stages of organogenesis were used to derive gene signatures for developmental programmes. Bulk tissue transcriptomic datasets from human NSCLC and non-neoplastic control samples were used to identify whether developmental programmes were associated with molecular, morphological, and clinical parameters. Single-cell RNA-sequencing was used to identify malignant cell states in human NSCLC (n = 16,621 epithelial cells from 72 samples) and protein level validation of these states was carried out using multiplexed immunohistochemistry (n = 40). Results Mutually antagonistic regulation of alveogenesis and BM was found to account for a significant proportion of transcriptomic variance in human NSCLC bulk tissue datasets. BM activation was associated with poor overall survival rates in five independent lung adenocarcinoma (LUAD) cohorts (p=2.04e-13); and was significantly prognostic for resistance to tyrosine kinase inhibitors (TKIs; p=0.003) and immune checkpoint blockade (ICBs; p=0.014), in pre-treatment biopsies. Single-cell RNA-sequencing analysis revealed that malignant LUAD cells with loss of alveolar lineage fidelity predominantly acquired inflamed or basal-like cellular states, which were variably persistent in samples from TKI and ICB recurrence. Conclusions Our results show LUAD tumours undergo reversion from an alveogenic to branching morphogenic phenotype during disease progression, generating inflamed or basal-like cell states that are variably persistent following TKI or ICB treatments. These findings identify prognostic biomarkers for therapy response and underscore the role of different cell states in resistance to multiple treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/390cd095a97603dc458c49ab8aaa4b76e0155529" target='_blank'>
              Developmental programmes drive cellular plasticity, disease progression and therapy resistance in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kamila J Bienkowska, Stephany Gallardo Y, Nur S Zainal, Matthew Ellis, Maria-Antoinette Lopez, Judith Austine, Sai Pittla, S. Chee, A. Alzetani, Emily C Shaw, Christian H Ottensmeier, Gareth J Thomas, Christopher Hanley
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Purpose
In glioblastoma, the therapeutically intractable and resistant phenotypes can be derived from glioma stem cells, which often have different underlying mechanisms from non-stem glioma cells. Aberrant signaling across the EGFR-PTEN-AKT-mTOR pathways have been shown as common drivers of glioblastoma. Revealing the inter and intra-cellular heterogeneity within glioma stem cell populations in relations to signaling patterns through these pathways may be key to precision diagnostic and therapeutic targeting of these cells.


Materials and Methods
Single cell parallel proteomic heterogeneity profiling of the EGFR-PTEN-AKT-mTOR pathways was conducted in a panel of fifteen glioma stem cell models derived from patient glioblastoma biopsies.


Results
The analysis included 59,464 data points from 14,866 cells and identified forty-nine molecularly distinct signaling phenotypes. High content bioinformatics resolved two unique patient clusters diverging on EGFR expression and AKT/TORC1 activation. Phenotypic validation indicated drug responsive phenotypes to EGFR blocking in the high EGFR expressing cluster with lower tumor initiating potential in comparison to the AKT/TORC1 activated cluster. High EGFR expression trended with improved patient prognosis while AKT/TORC1 activated samples trended with poorer patient outcomes. Genetic heterogeneity was observed in both clusters with proneural, classical and mesenchymal subtypes observed.


Conclusion
Quantitative single cell heterogeneity profiling reveals divergent EGFR-PTEN-AKT-mTOR pathways of patient derived glioma stem cells, which would inform future research and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0984500cc5c038ee7f954e1d13785419da86e8c6" target='_blank'>
              Single-Cell Heterogeneity of EGFR Pathway Is Linked to Unique Signatures of Drug Response and Malignancy in Patient Derived Glioblastoma Stem Cells.
              </a>
            </td>
          <td>
            Michael D. Masterman-Smith, Nicholas A. Graham, E. Panosyan, J. Mottahedeh, Eric R. Samuels, Araceli Nunez, Tiffany Phillips, Meeryo Choe, T. Cloughesy, Jorge A. Lazareff, Linda M. Liau, William H. Yong, Thomas G. Graeber, H. Kornblum, Ming-Fei Lang, Yanzhao Li, Jing Sun
          </td>
          <td>2025-01-07</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="PURPOSE
Pancreatic ductal adenocarcinoma (PDAC) patients with tumors enriched for the basal-like molecular subtype exhibit enhanced resistance to standard of care treatments and have significantly worse overall survival (OS) compared to patients with classical subtype enriched tumors. It is important to develop genomic resources, enabling identification of novel putative targets in a statistically rigorous manner.


EXPERIMENTAL DESIGN
We compiled a single cell RNA sequencing (scRNAseq) atlas of the human pancreas with 229 patient samples, aggregated from publicly available raw data. We mapped cell-type specific scRNAseq gene signatures in bulk RNAseq (n=744) and spatial transcriptomics (ST) (n=22) and performed validation using multiplex immunostaining.


RESULTS
Analysis of tumor cells from our scRNAseq atlas revealed nine distinct populations, two of which aligned with the basal subtype, correlating with worse OS in bulk RNAseq. Deconvolution identified one of the basal populations to be the predominant tumor subtype in non-dissociated ST tissues and in vitro tumor cell and patient-derived organoid lines. We discovered a novel enrichment and spatial association of CXCL10+ cancer associated fibroblasts (CAFs) with basal tumor cells. We identified that besides immune cells, ductal cells also express CXCR3, the receptor for CXCL10, suggesting a relationship between these cell types in PDAC tumor microenvironment.


CONCLUSIONS
We show that our scRNAseq atlas (700,000 cells), integrated with ST data, has increased statistical power and is a powerful resource, allowing for expansion of current subtyping paradigms in PDAC. We uncovered a novel signaling niche marked by CXCL10+ CAFs and basal tumor cells that could be explored for future targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07269a9cb871b60ea57ca5b2a01a49e35b66cc9b" target='_blank'>
              Human pancreatic cancer single cell atlas reveals association of CXCL10+ fibroblasts and basal subtype tumor cells.
              </a>
            </td>
          <td>
            Ian M Loveless, Samantha B. Kemp, Kailee M Hartway, Jacob T. Mitchell, Yuesong Wu, Samuel D Zwernik, D. Salas-Escabillas, Sydney Brender, Madison George, Yetunde Makinwa, Thais Stockdale, K. Gartrelle, Rohit G Reddy, Daniel W Long, Allison Wombwell, Julie M Clark, Albert M Levin, David Kwon, Ling Huang, R. Francescone, D. Vendramini-Costa, Ben Z. Stanger, Adam Alessio, Andrew M. Waters, Yuehua Cui, E. Fertig, L. T. Kagohara, Brian Theisen, Howard C Crawford, Nina G. Steele
          </td>
          <td>2024-12-05</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Muscle-invasive bladder cancer (MIBC) is a prevalent cancer characterized by molecular and clinical heterogeneity. Assessing the spatial heterogeneity of the MIBC microenvironment is crucial to understand its clinical significance. Methods In this study, we used imaging mass cytometry (IMC) to assess the spatial heterogeneity of MIBC microenvironment across 185 regions of interest in 40 tissue samples. We focused on three primary parameters: tumor (T), leading-edge (L), and nontumor (N). Cell gating was performed using the Cytobank platform. We calculated the Euclidean distances between cells to determine cellular interactions and performed single-cell RNA sequencing (scRNA-seq) to explore the molecular characteristics and mechanisms underlying specific fibroblast (FB) clusters. scRNA-seq combined with spatial transcriptomics (ST) facilitated the identification of ligand–receptor (L–R) pairs that mediate interactions between specific FB clusters and endothelial cells. Machine learning algorithms were used to construct a prognostic gene signature. Results The microenvironments in the N, L, and T regions of MIBC exhibited spatial heterogeneity and regional diversity in their components. A distinct FB cluster located in the L region—identified as S3—is strongly associated with poor prognosis. IMC analyses demonstrated a close spatial association between S3 and endothelial cells, with S3-positive tumors exhibiting increased blood vessel density and altered vascular morphology. The expression of vascular endothelial growth factor receptor and active vascular sprouting were significant in S3-positive tumors. scRNA-seq and ST analyses indicated that the genes upregulated in S3 were associated with angiogenesis. NOTCH1–JAG2 signaling pathway was identified as a significant L–R pair specific to S3 and endothelial cell interactions. Further analysis indicated that YAP1 was a potential regulator of S3. Machine learning algorithms and Gene Set Variation Analysis were used to establish an S3-related gene signature that was associated with the poor prognosis of tumors including MIBC, mesothelioma, glioblastoma multiforme, lower-grade glioma, stomach adenocarcinoma, uveal melanoma, kidney renal clear cell carcinoma, kidney renal papillary cell carcinoma, and lung squamous cell carcinoma. Conclusions We assessed the spatial landscape of the MIBC microenvironment and revealed a specific FB cluster with prognostic potential. These findings offer novel insights into the spatial heterogeneity of the MIBC microenvironment and highlight its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fcfdce6314523f58922cfd8b6a6e220595a1f7" target='_blank'>
              Spatially-resolved analyses of muscle invasive bladder cancer microenvironment unveil a distinct fibroblast cluster associated with prognosis
              </a>
            </td>
          <td>
            Chao Feng, Yaobang Wang, Wuyue Song, Tao Liu, Han Mo, Hui Liu, Shulin Wu, Zezu Qin, Zhenxing Wang, Yuting Tao, Liangyu He, Shaomei Tang, Yuanliang Xie, Qiuyan Wang, Tianyu Li
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Immune checkpoint inhibitors have revolutionized cancer therapy by harnessing the body’s immune system to eliminate tumor cells. However, only a subset of patients responds to treatment. Understanding why only some patients respond remains a critical challenge in cancer research due to the complexity and variability of cellular composition within the tumor microenvironment. Our study presents PRECISE (Predicting therapy Response through Extraction of Cells and genes from Immune Single-cell Expression data), a pipeline that leverages single-cell RNA sequencing data and machine learning techniques to predict ICI responses, while maintaining the richness of single-cell information and ensuring interpretability of the results. Results The PRECISE pipeline implements gene and cell-filtering approaches for optimizing treatment prediction. Utilizing the XGBoost algorithm for predicting patient response to ICI on a dataset of melanoma-infiltrated immune cell achieved an initial Area Under the Curve (AUC) of 0.84. This signal is further improved to 0.89 after Boruta feature selection, revealing an 11-gene predictive signature. Investigation of these genes through SHAP values identified various gene-pair interactions with non-linear conditional effects on predictions. Furthermore, a novel reinforcement learning framework implemented in PRECISE reveals non-predictive single cells that detracts the model’s performance. Altogether, the identified gene- and cell-based signatures demonstrates high prediction power across independent datasets, including lung, breast, brain, and skin cancers. Conclusions Our approach demonstrates the potential of supervised machine learning and reinforcement learning to enhance the understanding of cancer immunity and improve the prediction of treatment responses using single-cell data. Understanding patients’ response to immune checkpoint inhibitors (ICIs) is a critical challenge in cancer research, given the complexity and variability of immune interactions within the tumor microenvironment. Our study leverages single-cell RNA sequencing data and aims to utilize machine learning techniques to predict immunotherapy responses, while maintaining the richness of single-cell information and ensuring interpretability of the results. Using a dataset of melanoma immune cells, we conducted thorough preprocessing and applied the XGBoost algorithm in a leave-one-out cross-validation fashion to predict sample response. By labeling cells according to their sample’s response and aggregating predictions, we achieved an initial AUC score of 0.84. This score was improved to 0.89 with the application of Boruta feature selection, which identified key predictive genes, leading to an 11-gene predictive signature. Further analysis pinpointed T cell clusters as significant contributors to immune response. Utilizing SHAP values provided deeper insights into gene behaviors, interactions, and their effects on the model’s predictions. Additionally, a novel reinforcement learning model was developed for single-cell level prediction and characterization, allowing us to identify and analyze the most predictive cells for response and non-response. Our approach demonstrates the potential of sophisticated computational methods to enhance our understanding of cancer immunity and improve the prediction of treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdff0e265c13d348620c08428ea6a43d804a4e06" target='_blank'>
              Uncovering Predictive Gene and Cellular Signatures for Checkpoint Immunotherapy Response through Machine Learning Analysis of Immune Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Reduced therapy response in breast cancer has been correlated with heterogeneity in biomarker composition, expression level, and spatial distribution of cancer cells within a patient tumor. Thus, there is a need for models to replicate cell-cell, cell-stromal, and cell-microenvironment interactions during cancer progression. Traditional two-dimensional (2D) cell culture models are convenient but cannot adequately represent tumor microenvironment histological organization, in vivo 3D spatial/cellular context, and physiological relevance. Recently, three-dimensional (3D) in vitro tumor models have been shown to provide an improved platform for incorporating compositional and spatial heterogeneity and to better mimic the biological characteristics of patient tumors to assess drug response. Advances in 3D bioprinting have allowed the creation of more complex models with improved physiologic representation while controlling for reproducibility and accuracy. This review aims to summarize the advantages and challenges of current 3D in vitro models for evaluating therapy response in breast cancer, with a particular emphasis on 3D bioprinting, and addresses several key issues for future model development as well as their application to other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ccd5bcde9ada94a158d2c3401e2be4034200cf" target='_blank'>
              Modeling intratumor heterogeneity in breast cancer.
              </a>
            </td>
          <td>
            E. McDonough, M. Barroso, Fiona Ginty, David T Corr
          </td>
          <td>2024-12-06</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Despite surgical resection, rigorous endoscopic surveillance, and immunotherapy with the Bacillus Calmette–Guérin (BCG) vaccine, 30% of high-risk bladder cancers recur, and 10% result in fatal outcomes within two years of diagnosis. The global shortage of BCG underscores the urgent need for alternative or complementary therapeutic strategies. To address this, we integrated transcriptomic profiling and targeted genomic sequencing to identify four consensus intrinsic subtypes of bladder cancer. Initially derived from bulk RNA profiling, these subtypes were further validated at the cellular and tissue-compartment levels using single-cell RNA sequencing and spatial transcriptomics. Notably, we identified a subtype of inflamed tumors with enhanced endogenous retroelement expression and increased commensal bacterial presence, which showed the highest responsiveness to BCG therapy. Additionally, we developed a machine learning-based model incorporating composite molecular features to predict recurrence risk, achieving a high accuracy (AUC = 0.90). Our findings establish a molecular precision framework for bladder cancer and nominate novel therapeutic targets to reduce reliance on BCG immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f0d9394677cb2bb295a5f2fe96a5d0c0fc0b84" target='_blank'>
              Genomic and transcriptomic profiling of high-risk bladder cancer reveals diverse molecular and microenvironment ecosystems
              </a>
            </td>
          <td>
            K. Meghani, Yanni Yu, N. Frydenlund, Erik Z. Li, B. Choy, S. Abdulkadir, J. Meeks
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) tumor cells and their interactions with the immune microenvironment, particularly at the leading-edge area, have been underexplored. This study employs single-cell RNA sequencing (scRNA-seq) and spatial transcriptome (ST) analysis on samples from the tumor core, adjacent non-tumorous tissue, and the leading-edge area of nine ICC patients. These findings indicate that tumor cells at the leading-edge area demonstrate enhanced proliferation and are tightly associated with the stroma, including endothelial cells and POSTN+ FAP+ fibroblasts. Notably, CD8+ T cells in this region exhibit a naive phenotype with low cytotoxicity and signs of exhaustion, likely due to compromised antigen presentation by antigen-presenting cells (APCs). The predominant CD8+ T cell subset, mucosal-associated invariant T (MAIT) cells, recruits SPP1+ macrophages within the stroma. This interaction, along with the presence of POSTN+ cancer-associated fibroblasts (CAFs) and endothelial cells, forms a unique "triad structure" that fosters tumor growth and ICC progression. The research highlights the intricate characteristics and interactions of ICC tumor cells in the leading-edge area, offering insights into potential therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4692f475931cf20c24dd17b389c955de7ede39" target='_blank'>
              Single-Cell and Spatial Transcriptomics Delineate the Microstructure and Immune Landscape of Intrahepatic Cholangiocarcinoma in the Leading-Edge Area.
              </a>
            </td>
          <td>
            Li Zuyin, Li Zhao, Cheng Qian, Changkun Zhang, Ma Delin, Jialing Hao, Chen Zhuomiaoyu, Li Yuzi, Zheng Jiaxi, Gao Jie, Zhu Jiye
          </td>
          <td>2024-12-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Immunoediting is the process where developing tumors are shaped by cells of the immune system. Tumors with low immunogenicity are likely to emerge after neoantigen-specific T cells eliminate the most immunogenic cell populations. However, direct interrogation of mechanisms by which this occurs has not possible due to the pre-emergent nature before the disease is verifiably established. By modifying a mouse model of soft-tissue sarcoma where emergent tumors are consistently edited, we developed a system to examine mechanisms underlying how T cells interact with pre-emergent tumor cell populations to suppress tumor emergence. This system involves production of neoantigen-expressing autochthonous tumors with a fluorescent reporter to define neoantigen-expression status of individual cells. Tumors emerge with lower penetrance and delayed onset in T cell-sufficient mice, while every tumor from T cell-deficient mice contain a significant fraction of neoantigen-negative tumor cells. Yet, ∼53% of T cell-sufficient mice never developed, which suggests that neoantigen silencing was necessary, but not sufficient for tumor emergence. Consistent with this, genetic removal of the neoantigen after malignant transformation shows that tumor development is only rescued when neoantigen loss occurs prior to the peak T cell response, occurring around 8 days post transformation. Moreover, clonality studies reveal the presence of T cells reduces the number clones that contribute to tumor emergence down to 1-2 clones per emerged tumor. Mechanistically, blockade of the T cell effector cytokine, IFNgamma, rescues the neoantigen-negative, but not neoantigen-expressing tumor cells, demonstrating that T cells control early tumors via at least two distinct mechanisms: IFNgamma-dependent suppression of neoantigen-negative tumors cells and IFNgamma-independent suppression of neoantigen-expressing cells. Thus, our findings support that neoantigen-negative cells are subjected to cytokine-mediated suppression from T cells responding to the neoantigen-positive population in the pre-emergent tumor microenvironment. This bystander suppression can prevent tumor emergence despite non-homogeneity in neoantigen expression.
 Citation Format: Brian G Hunt, Julie F. Cheung, Srividhya Venkatesan, Jennifer L. Loza, Kelli A. Connolly, Shudipto Wahed, Elaine Cheng, Ishan Bansal, Emily Borr, Wei Wei, Ivana William, Brittany Fitzgerald, Nikhil Joshi. Productive T-cell-mediated immunoediting of developing sarcomas by distinct mechanisms [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B030.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef0c90f5cb6aad90d4fae187c6f95d4947bb2c3" target='_blank'>
              Abstract B030: Productive T-cell-mediated immunoediting of developing sarcomas by distinct mechanisms
              </a>
            </td>
          <td>
            Brian G. Hunt, Julie F. Cheung, Srividhya Venkatesan, Jennifer L. Loza, Kelli A. Connolly, Shudipto Wahed, Elaine Cheng, Ishan Bansal, Emily Borr, Wei Wei, I. William, Brittany Fitzgerald, Nikhil S. Joshi
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, , Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Epithelial-mesenchymal plasticity (EMP) is a cell-fate switching program that enables cells to adopt a spectrum of phenotypes ranging from epithelial (E) to mesenchymal (M) and hybrid E/M states. Hybrid E/M phenotypes are considered optimal for cancer metastasis due to their association with metastatic initiation and traits such as stemness, drug resistance, and collective migration. Boolean models of gene regulatory networks (GRNs) underlying EMP have provided valuable insights into the dynamics of E and M phenotypes. However, these models fail to capture hybrid phenotypes effectively, as Boolean frameworks allow only binary gene expression, while hybrid E/M cells exhibit partial expression of both E and M genes. To address this limitation, we modified a threshold-based Boolean formalism to incorporate intermediate gene expression levels. The resulting multilevel model reveals new hybrid steady states with partial expression of both E and M genes, expanding the phenotypic landscape beyond that captured by traditional Boolean models. These hybrid states exhibit lower frustration and higher abundance, indicating enhanced stability. By resolving degeneracy in the dynamics, we further demonstrate that hybrid states identified by the multilevel model are more stable. These findings suggest that adding minimal complexity to Boolean models can uncover previously hidden qualitative features of phenotypic landscapes governed by GRNs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29922a69bbdff28fdd118c9e80f2fc16659ff8c" target='_blank'>
              A multi-level formalism to capture the characteristics of hybrid phenotypes in Epithelial-Mesenchymal Plasticity
              </a>
            </td>
          <td>
            K. Hari, Shubham Tripathi, Vaibhav Anand, M. Jolly, Herbert Levine
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The adaptation of metastatic cells to their host tissue critically determines the pathogenicity of a cancer and therefore patient survival. Yet, it remains elusive to what extent the host environment drives gene expression programs in metastatic cells. Here we identify adaptive mechanisms that enable metastases to establish themselves in a novel tissue context. We performed single-cell RNA-sequencing on malignant and benign tissue samples from untreated donors with colorectal adenocarcinoma and liver metastasis to deduce tissue adaptive expression patterns. A novel computational approach identified genes and pathways that consistently adapted to the host tissue at the transition from the primary tumor to the paired metastasis across donors. This analysis revealed that the majority of expression changes in the metastasis reflect an expression signature reminiscent of benign liver epithelial cells. Cellular processes adapting to the liver environment include basic cellular functions such as energy metabolism, as well as tissue-specific pathways such as the regulation of lipid metabolism by PPAR-α. These adaptations potentially increase the pathogenicity of the metastatic cells and may provide new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189b4d35fab487ddefe0f36a861c9c03b2e18b5a" target='_blank'>
              Gene expression adaptation of metastases to their host tissue
              </a>
            </td>
          <td>
            L. Nagel, Marten C. Wenzel, Sascha Hoppe, Patrick S. Plum, Mohammad Karimpour, Marek Franitza, Roger Wahba, Marc Bludau, Christiane J. Bruns, Alexander Quaas, A. Beyer, Axel M. Hillmer
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) account for 0.01 to 2% of the total tumor mass; however, they play a key role in tumor progression, metastasis and resistance to current cancer therapies. The generation and maintenance of CSCs are usually linked to the epithelial–mesenchymal transition (EMT), a dynamic process involved in reprogramming cancer cells towards a more aggressive and motile phenotype with increased stemness potential. Cells that undergo an EMT process have shown to be more resistant to conventional chemo/radiotherapies. In this context, aldehyde dehydrogenase (ALDH) enzymes, known for their role in the cellular detoxification of aldehydes and enhancement of cell survival, are often upregulated in cancer cells, promoting their resistance to conventional cancer treatments. Indeed, high ALDH levels have become a hallmark biomarker of CSCs and are often used to isolate this sub-population from the more abundant cancer cell populations. Herein, we isolated human breast cancer epithelial cells with higher ALDH abundance (ALDHHigh) and compared them to those with low ALDH abundance (ALDHLow). ALDHHigh sub-populations exhibited more characteristic EMT biomarkers by adopting a more mesenchymal phenotype with increased stemness and enhanced migratory potential. Furthermore, ALDHHigh sub-populations displayed elevated senescent markers. Moreover, these cells also demonstrated higher levels of mitochondria DNA/mass, as well as greater mitochondrial and glycolytic metabolic function. Conversely, ALDHLow sub-populations showed a higher efficiency of mammosphere/colony formation and an increased proliferative capacity. Therefore, we demonstrated that these ALDH sub-populations have distinct characteristics, underscoring their role in EMT, the formation of tumors and the mechanisms of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c656cc9a0663a21dc7738f32f6803ef9c5938a5" target='_blank'>
              ALDHHigh Breast Cancer Stem Cells Exhibit a Mesenchymal–Senescent Hybrid Phenotype, with Elevated Metabolic and Migratory Activities
              </a>
            </td>
          <td>
            Luis Larrea Murillo, Conor J. Sugden, Bela Ozsvari, Zahra Moftakhar, Ghada S. Hassan, F. Sotgia, Michael P Lisanti
          </td>
          <td>2024-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="
 Background: Breast cancer is the most frequently diagnosed cancer, and the second leading cause of cancer-related death in women. Most breast cancer related deaths are due to metastasis to distant organs. The primary tumor and secondary site microenvironments provide cues that activate cancer cell plasticity programs such as those associated with invasion and stemness, which are needed for dissemination and the establishment of metastatic foci. We demonstrated that in the primary tumor >50% of cancer stem cells (CSCs) co-express the invasive phenotype marker, MenaINV. Given that ∼80% of newly arriving circulating tumor cells to the lung are CSCs, we questioned if extravasation of CSCs to the lung is mediated solely by MenaINV or if other compensatory pathways are at play. We describe here our discovery that CSCs require MenaINV for extravasation to the lung parenchyma. Methods: To visualize breast cancer stem cell extravasation into the lung parenchyma, we expressed the stemness biosensor, SORE6, in MDA-MB-231 breast cancer cells. SORE6 is composed of 6 tandem repeats of the Sox2 and Oct4 response elements placed upstream of destabilized Green Fluorescent Protein (GFP). To distinguish CSCs (green) from non-CSCs, all cells were also labeled with red tracker dye. Breast cancer cells transduced with lenti-SORE6 were FACS sorted into low SORE6-GFP cells (non- CSCs) and high SORE6-GFP (CSCs). Activation of the stem program in FACS sorted cells was validated by qRT-PCR analysis of the widely used stem cell markers CD44, CD133, P63, ALDH1, Sox9, Sox2, and Oct4. The extravasation efficiency of non-CSCs vs. CSCs was assessed in vitro by the extravasation directed transendothelial migration (eTEM) assay, which mimics extravasation by allowing migration of cancer cells through endothelial cells cultured on a porous membrane, as well as in vivo, by intravital imaging. The impact of MenaINV on CSC extravasation was evaluated through CRISPR/Cas9 mediated knockout of MenaINV. In in vivo experiments, a 50/50 mixture of non-CSCs and CSCs expressing either wild type or knocked-out MenaINV were injected into the vasculature of Rag2-/- MacBlue mice. in vivo visualization of cancer cell extravasation was accomplished by implantation of an indwelling lung imaging window prior to the injection of cancer cells, followed by high-resolution intravital imaging. Results: CSCs exhibit significantly higher expression of all stem cell markers and express more MenaINV compared to non-CSCs. Moreover, CSCs show significantly better transendothelial migration in eTME assay (p=0.03) and lung extravasation (p<0.0001) in intravital imaging, compared to non-CSCs. Finally, knocking out MenaINV reduced lung extravasation to the levels comparable to non-CSCs. Conclusions: This study highlights the crucial role of MenaINV in facilitating lung metastasis by CSCs. Thus, targeting MenaINV may represent a promising therapeutic strategy to reduce metastatic burden in breast cancer patients.
 Citation Format: Mohd Nauman, Yookyung Jung, Burcu Karadal-Ferrena, Camille Duran, Madeline Friedman-DeLuca, Nicole Barth, Prachiben Patel, John Condeelis, David Entenberg, Maja H. Oktay. Requirement of MenaINV for metastatic colonization of the lungs by breast cancer stem cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B045.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7690eaf236a5e768439bef697a6037261a3a64ae" target='_blank'>
              Abstract B045: Requirement of MenaINV for metastatic colonization of the lungs by breast cancer stem cells
              </a>
            </td>
          <td>
            Mohd Nauman, Yookyung Jung, Burcu Karadal-Ferrena, Camille L. Duran, Madeline Friedman-DeLuca, Nicole Barth, Prachiben Patel, J. Condeelis, D. Entenberg, M. Oktay
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Even seemingly homogeneous populations of cells can express phenotypic diversity in response to environmental changes. Thus, the effects of X-ray irradiation on tissues composed of diverse cell types is likely to be complex. We have used single-cell RNA sequencing to study the effects of X-ray radiation on the wing-imaginal disc of Drosophila, a relatively simple tissue that is mostly composed of epithelial cells. Transcriptomic clustering of cells collected from the wing disc generates clusters that are mainly grouped based on cell location in the proximodistal axis. To quantify heterogeneity of gene expression among clusters, we adapted a metric used to study market concentration, the Herfindahl-Hirschman Index. We show that genes involved in DNA damage repair, alleviation of reactive oxygen species, cell-cycle progression, and apoptosis are expressed relatively uniformly. In contrast, genes encoding a subset of ligands, notably cytokines that activate the JAK/STAT pathway, transcription factors implicated in regeneration such as Ets21C, and some signaling proteins are expressed in more restricted territories. Several of these genes are still expressed in a p53-dependent manner indicating that regional and radiation-induced factors combine to regulate their expression. We similarly examined heterogeneity within territories by using a clustering approach based on cell-cycle gene expression. Using this method, we identified a subpopulation characterized by high levels of tribbles expression that is mostly found in irradiated discs. Remarkably, this subpopulation accounts for a considerable fraction of radiation-induced gene expression, indicating that cellular responses are non-uniform even within territories. Thus, both inter-regional and intra-regional heterogeneity are important features of tissue responses to X-ray radiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0cdf2959d2975e5b26ebbd9d5583e8560966205" target='_blank'>
              Single cell transcriptomics of X-ray irradiated Drosophila wing discs reveals heterogeneity related to cell-cycle status and cell location
              </a>
            </td>
          <td>
            Joyner Cruz, Willam Y. Sun, Alexandra Verbeke, I. Hariharan
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d907c1f70a7e162db16222c621c3aec6a63202a" target='_blank'>
              Integrated single-cell analysis reveals heterogeneity and therapeutic insights in osteosarcoma
              </a>
            </td>
          <td>
            Dongan He, Xiaoqian Che, Haiming Zhang, Jiandong Guo, Lei Cai, Jian Li, Jinxi Zhang, Xin Jin, Jianfeng Wang
          </td>
          <td>2024-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The complex interplay between metal abundance, transport mechanisms, cell distribution, and tumor progression-related biological pathways (e.g., metabolism, collagen remodeling) remains poorly understood. Traditionally, genes and metals have been studied in isolation, limiting insights into their interactions. Recent advances in spatial transcriptomics and elemental profiling now enable comprehensive exploration of tissue-wide metal-gene interactions, though integration remains challenging. In this proof-of-concept study, we investigated metal-dependent signaling within the tumor microenvironment of a unique colorectal cancer (CRC) tumor. We implemented a spatial multimodal workflow which integrated elemental imaging, gene expression, cellular composition, and histopathological features to uncover metals-related pathways through spatially resolved differential expression analysis. Preliminary findings revealed significant associations, for instance: elevated iron correlated with mesenchymal phenotypes located at the tumor's proliferative front, reflecting epithelial-to-mesenchymal transition pathways, and extracellular matrix remodeling. High concentrations of copper were predominantly localized in regions of active tumor growth and associated with upregulation of immune response genes. This proof-of-concept workflow demonstrates the feasibility of integrating elemental imaging with spatial transcriptomics to identify metals-based gene correlates. Future application of this workflow to larger patient cohorts will pave the way for expansive comparisons across the metallome and transcriptome, ultimately identifying novel targets for tumor progression biomarkers and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b7bf64840757cf5e603b33283a2afe6a77fb5a3" target='_blank'>
              Integration of Elemental Imaging and Spatial Transcriptomic Profiling for Proof-of-Concept Metals-Based Pathway Analysis of Colon Tumor Microenvironment
              </a>
            </td>
          <td>
            A. Srivastava, N. Shaik, Y. Lu, M. Chan, A. B. Diallo, S. Han, T. Punshon, B. P. Jackson, L. Vahdat, X. Liu, V. Mittal, K. S. Lau, J. Gui, L. Vaickus, J. Hoopes, F. Kolling, L. Perreard, J. Marotti, J. Levy
          </td>
          <td>2024-12-10</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Cryoablation induces antitumor immune responses. Spatial transcriptomic landscape technology has been used to determine the micron-level panoramic transcriptomics of tissue slices in situ.


METHODS
The effects of cryoablation on the immune microenvironment in non-small cell lung cancer (NSCLC) were explored by comparing the Whole Transcriptome Atlas (WTA) panel of immune cells before and after cryoablation using the spatial transcriptomic landscape.


RESULTS
The bioinformatics analysis showed that cryoablation significantly affected the WTA of immune cells, particularly genes related to cellular components, biological processes, molecular functions, proliferation and migration, and cytokine-cytokine receptor interaction signaling pathways.


CONCLUSIONS
The findings of this study suggest that cryoablation significantly impacts the biological functions of immune cells in the tumor microenvironment of NSCLC through multiple mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3a0230d879f0812d1beffd1bcd78bcdc486ebd" target='_blank'>
              Effect of cryoablation on the spatial transcriptomic landscape of the immune microenvironment in non-small cell lung cancer.
              </a>
            </td>
          <td>
            Jun Zhou, Shengxi Wang, Ming Liu, Zhaopei Li
          </td>
          <td>2024-12-01</td>
          <td>Journal of cancer research and therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d052ec02be986508482ae1f68379c54cb0ed96e8" target='_blank'>
              Cell-type specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tissue homeostasis relies on a precise balance of fate choices between renewal and differentiation, which is dysregulated during tumor initiation. Although much progress has been done over recent years to characterize the dynamics of cellular fate choices at the single cell level, their underlying mechanistic basis often remains unclear. In particular, although physical forces are increasingly characterized as regulators of cell behaviors, a unifying description of how global tissue mechanics interplays with local cellular fate choices is missing. Concentrating on skin epidermis as a paradigm for multilayered tissues with complex fate choices, we develop a 3D vertex-based model with proliferation restrained in the basal layer, showing that mechanics and competition for space naturally gives rise to homeostasis and neutral drift dynamics that are seen experimentally. We then explore the effect of introducing mechanical inhomogeneities, whereby subpopulations have differential tensions. We uncover that relatively small mechanical disparities can be sufficient to heavily tilt cellular towards symmetric renewal and exponential growth. Importantly, the simulations predict that such mechanical inhomogeneities are reflected by distinct morphological changes in single-cell shapes. This led us to derive a master relationship between two very different experimentally measurable parameters, cell shape and long-term clonal dynamics, which we validated using a model of basal cell carcinoma (BCC) consisting in clonal Smoothened overexpression in mouse tail epidermis. Altogether, we propose a theoretical framework to link mechanical forces, quantitative cellular morphologies and cellular fate outcomes in complex tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/215257d8ed053f55e3fe3327a06377e6d6a6e712" target='_blank'>
              Biomechanics of stem cell fate decisions in multilayered tissues
              </a>
            </td>
          <td>
            Preeti Sahu, Sara Canato, Raquel Soares, Adriana Sánchez-Danés, É. Hannezo
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Background Pericytes are crucial for the development, stabilization, and functional regulation of microvasculature, especially in the retina. In diabetic retinopathy (DR), early loss of pericytes is a key event that drives microvascular dysfunction. Despite their critical role, the mechanisms underlying the functional heterogeneity of pericytes in DR remain poorly understood, impeding the development of effective therapeutic strategies. Methods We employed single-cell RNA sequencing to construct a comprehensive single- cell atlas of non-diabetic and diabetic retinas. Using bioinformatic clustering and subcluster analysis, we identified a specific pericyte subcluster associated with diabetic microvascular complications. Differential gene expression analysis and immunofluorescence validation highlighted PTTG1 as a potential key regulator of pericyte dysfunction. To investigate its functional role, we emplyed CRISPR/Cas9 and adenoviral vectors to modulate PTTG1 expression in vitro and in vivo. Combined transcriptomic and metabolomic approaches were used to explore the mechanistic pathways through which PTTG1 influences pericyte biology and vascular function. Results We identified a novel pericyte subcluster characterized by elevated expression of PTTG1, which was strongly correlated with diabetic microvascular dysfunction. Silencing PTTG1 using CRISPR/Cas9 and siRNA in vitro mitigated pericyte dysfunction under high- glucose conditions. Targeted knockdown of PTTG1 using viral vectors improved retinal vascular integrity and reduced neovascularization in diabetic mice. Transcriptomic and untargeted metabolomic analyses revealed that PTTG1 knockdown reprogrammed pericyte energy metabolism by modulating glycolysis pathway genes, reducing oxidative stress, and restoring pericyte function, ultimately alleviating microvascular dysfunction in DR. Conclusions PTTG1 plays a critical role in regulating pericyte dysfunction and maintaining vascular homeostasis in diabetic retinopathy. By modulating key metabolic pathways and pericyte phenotypes, PTTG1 represents a promising therapeutic target for treating diabetic microvascular complications. These insights offer a novel molecular framework for developing targeted therapies aimed at restoring retinal vascular health in diabetic patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d2cfb264dd76b8c3dd1f5b9db0dce030fd7e6a3" target='_blank'>
              PTTG1-Mediated Pericyte Dysfunction Drives Diabetes-Induced Microvascular Dysfunction
              </a>
            </td>
          <td>
            Linyu Zhang, Ling Ren, Jingyue Zhang, Min Xia, Xiaosa Li, Mudi Yao, Fanfei Ma, Chang Jiang, Jin Yao, Biao Yan
          </td>
          <td>2024-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The metastasis of solid tumors hinges on cancer cells navigating through complex three-dimensional tissue environments, characterized by mechanical heterogeneity and biological diversity. This process is closely linked to the dynamic migration behavior exhibited by cancer cells, which dictates the invasiveness of tumors. In our study, we investigate tumor spheroids composed of breast cancer cells embedded in three-dimensional (3D) collagen matrices. Through a combination of quantitative experiments, artificial-intelligence-driven image processing, and mathematical modeling, we uncover rapid transitions in cell phenotypes and phenotype-dependent motility among disseminating cells originating from tumor spheroids. Persistent invasion leads to continuous remodeling of the extracellular matrix surrounding the spheroids, altering the landscape of migration phenotypes. Consequently, filopodial cells emerge as the predominant phenotype across diverse extracellular matrix conditions. Our findings unveil the complex mesoscale dynamics of invading tumor spheroids, shedding light on the complex interplay between migration phenotype plasticity, microenvironment remodeling, and cell motility within 3D extracellular matrices.




 Published by the American Physical Society
 2024


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afedef8827a246c2a67d180498da9a907634ab61" target='_blank'>
              Coupled Dynamics in Phenotype and Tissue Spaces Shape the Three-Dimensional Cancer Invasion
              </a>
            </td>
          <td>
            Austin Naylor, Maximilian Libmann, Izabel Raab, W.-J. Rappel, Bo Sun
          </td>
          <td>2024-12-24</td>
          <td>PRX Life</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5361264ba7fa5b03a05e9caae598d056bebbf71c" target='_blank'>
              Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small-cell lung cancer.
              </a>
            </td>
          <td>
            Yilv Yan, Dongqing Sun, Junjie Hu, Yue Chen, Liangdong Sun, Huansha Yu, Yicheng Xiong, Zhida Huang, Haoran Xia, Xinsheng Zhu, D. Bian, Fenghuan Sun, Likun Hou, Chunyan Wu, Orion R Fan, Haiyang Hu, An Zeng, Lele Zhang, Yi Eve Sun, Chenfei Wang, Peng Zhang
          </td>
          <td>2024-12-10</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND
Growth hormone-secreting pituitary adenomas (GHPA) display diverse biological behaviors and clinical outcomes, necessitating the identification of tumor heterogeneity and prognostically relevant markers.


METHODS
In this study, we performed single-cell RNA sequencing (scRNA-seq) on 10 GHPA samples, four of which also underwent spatial transcriptome sequencing, and used scRNA-seq data from four normal pituitary samples as controls. Cell subtype characterization in GHPA was analyzed using multiple algorithms to identify malignant bias regulators, which were then validated using a clinical cohort.


RESULTS
We constructed the first single-cell and spatial transcriptome profiles of GHPA, which contained 87,862 cells and revealed 16 tumor cell subtypes. Among the tumor cells, we identified distinct developmental trajectories and three malignant-biased subtypes (PIT1_C05, PIT1_C06, and PIT1_C10). The spatial distribution characteristics of these malignant-biased cells may influence the growth characteristics and prognosis of GHPA. We screened specific regulatory transcription factors, including FOXO1, GTF2IRD1, and MAX. Clinical cohort validation indicated that FOXO1 might be associated with tumor invasion and progression, while high expression of MAX could result in poor endocrine outcomes.


CONCLUSION
GHPA exhibits rich heterogeneity and diverse cell subtypes, with specific transcription factors potentially regulating cell malignant bias, thereby influencing tumor characteristics and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a105784b2d156549c656072efae7d8dac904126" target='_blank'>
              Single-cell and spatial transcriptomic analyses reveal heterogeneity characteristics and specific cell subtype regulators in growth hormone-secreting pituitary adenomas.
              </a>
            </td>
          <td>
            Yu Zhang, Lifeng Wang, Xingcheng Yi, Xinyu Ma, Hongyu Wu, Mingzhao Zhang, Zhenqi Yang, Lizhen Ma, Zenghua Mi, Wei-jia Zhi, Cong Fu, Pinan Liu, Zhijun Yang
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Metastatic disease is the common cause of death in colorectal cancer. To better understand the cellular landscape of colorectal metastasis, we profiled primary tumors and liver, lung, and peritoneal metastases using single-cell RNA sequencing. Our analysis revealed a subset of tumor epithelial cells with a highly plastic transcription profile and, inversely, low expression of stemness markers. Tumor cells with plasticity markers were spatially enriched at the tumor- stroma interface with exposure to tissue microenvironmental factors. Cell-free malignant ascites, which recapitulates the tumor microenvironment, impaired expression of proliferation and stem- like markers in favor of plasticity markers and enhanced migration of tumor cells. Furthermore, Co-culture of colorectal tumor cells with CD11b+ macrophages isolated from malignant ascites from patients was sufficient to recapitulate tumor cell plasticity. Profiling of tumor-associated macrophages identified enrichment for a lipid-handling features and high spatial correlation with plastic tumor cells. Taken together, we demonstrate that tumor-associated macrophages contribute to disease progression through development of pro-malignant features.
 Citation Format: Matthew A. Cottam, Muhammad B. Mirza, Emily N. Arner, Marium Siddiqui, Clare Lipscombe, Zaid Hatem, Katy E. Beckermann, Kamran Idrees. Regulatory macrophages contribute to tumor cell plasticity in colorectal cancer metastases [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38b2321f9b6c18e29ebc7faf6bc4bebdd5ef49f8" target='_blank'>
              Abstract B019: Regulatory macrophages contribute to tumor cell plasticity in colorectal cancer metastases
              </a>
            </td>
          <td>
            Matthew A. Cottam, M. Mirza, Emily N. Arner, Marium Siddiqui, Clare Lipscombe, Zaid Hatem, Katy E. Beckermann, K. Idrees
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="A comprehensive understanding of tumor heterogeneity, tumor microenvironment and the mechanisms of drug resistance is fundamental to advancing breast cancer research. While single-cell RNA sequencing has resolved the issue of “temporal dynamic expression” of genes at the single-cell level, the lack of spatial information still prevents us from gaining a comprehensive understanding of breast cancer. The introduction and application of spatial transcriptomics addresses this limitation. As the annual technical method of 2020, spatial transcriptomics preserves the spatial location of tissues and resolves RNA-seq data to help localize and differentiate the active expression of functional genes within a specific tissue region, enabling the study of spatial location attributes of gene locations and cellular tissue environments. In the context of breast cancer, spatial transcriptomics can assist in the identification of novel breast cancer subtypes and spatially discriminative features that show promise for individualized precise treatment. This article summarized the key technical approaches, recent advances in spatial transcriptomics and its applications in breast cancer, and discusses the limitations of current spatial transcriptomics methods and the prospects for future development, with a view to advancing the application of this technology in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9484c611ac13b7e4460cc6b1b5eb2c6b25e4a9" target='_blank'>
              Spatial transcriptomics in breast cancer: providing insight into tumor heterogeneity and promoting individualized therapy
              </a>
            </td>
          <td>
            Junsha An, Yajie Lu, Yuxi Chen, Yuling Chen, Zhaokai Zhou, Jianping Chen, Cheng Peng, Ruizhen Huang, Fu Peng
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Cancer-associated fibroblasts (CAFs) are attractive therapeutic targets for cancer due to their abundance in the tumor microenvironment (TME) and extensive pro- and antitumorigenic functions. However, formulating CAF-targeting strategies remains challenging due to the complexity of CAF biology. CAFs specialize into phenotypically and functionally distinct subtypes that may respond differently to specific treatments. Thus, designing CAF-targeting therapies necessitates a thorough understanding of CAF heterogeneity and preclinical model systems that can reliably prototype CAF biology observed in tumors. However, CAF heterogeneity is still not well-understood for many solid tumors, including the stroma-rich pancreatic ductal adenocarcinoma (PDAC), and the translatability of stroma models has not been comprehensively evaluated. In this work, we developed in vitro cell lines of tumor-CAF cocultures using primary CAFs derived from PDAC and used single-cell transcriptomics and CRISPR perturbation to investigate CAF heterogeneity and subtype-specific responses. We found that coculturing primary CAFs with the BxPC3 PDAC cell line could capture the tumor-CAF crosstalks underlying CAF activation. We characterized eight subtypes, including myofibroblastic, inflammatory, extracellular matrix (ECM)-like, and interferon CAFs, which could be preserved with immortalization. We also delineated the trajectories of CAF subtype interconvertibility, highlighting inflammatory CAFs as the likely initial diseased state and ECM CAFs as a “reservoir” that seeded the later states. By correlating the in vitro subtypes with clinical data of immunotherapy responses, we established that the in vitro cell lines could model the immuno-modulatory and immunoresistant phenotypes of CAFs. Finally, using an immortalized tumor-CAF coculture line, we performed Perturb-seq single-cell perturbation to perturb several stroma genes of interest, including TGFBR1, SPATS2L, AEBP1, and PTGS1, and evaluated the subtype-specific effects of the perturbations. We found that perturbing TGFBR1 resulted in the expected eradication of myofibroblastic CAFs, while the other perturbations resulted in subtype-specific modulations of metabolic and extracellular matrix related pathways. Overall, our findings underscored that in vitro tumor-CAF coculture models could serve as a translatable system that effectively captured CAF biology and heterogeneity observed in tumors and highlighted some similarities between PDAC CAFs and previously characterized breast cancer CAFs. We also demonstrated that single-cell perturbation could be a useful approach for interrogating the distinct responses of different CAF subtypes to perturbations.
 Citation Format: Elysia Saputra, Shamsudheen Karuthedath Vellarikkal, Lixia Li, Hong Sun, Suchitra Natarajan, Federica Piccioni, Alex M. Tamburino, Xin Yu, Aleksandra K. Olow. Characterization and CRISPR perturbation of cancer-associated fibroblast heterogeneity in vitro models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d13335b76ec58b7c353073a2851e7fff51991a" target='_blank'>
              Abstract B009: Characterization and CRISPR perturbation of cancer-associated fibroblast heterogeneity in vitro models
              </a>
            </td>
          <td>
            Elysia Saputra, S. Vellarikkal, Lixia Li, Hong Sun, Suchitra Natarajan, Federica Piccioni, Alex M. Tamburino, Xin Yu, Aleksandra K. Olow
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Cancer progression poses a clinical hurdle due to tumor heterogeneity that contributes to treatment resistance and immune evasion. While the adaptive immune system can effectively eliminate cancer in certain instances, tumor escape remains challenging. The immune microenvironment plays an important role in this process. One unanswered question is the role of ECM geometry, also known as Tumor Associated Collagen Signatures (TACS), and its effects on tumor immune recognition. In solid malignancies, ECM geometry has been associated with disease stage and T cell infiltration. Specifically, empirically observed ECM topologies are frequently categorized based on fiber arrangement: random fibers (TACS1), circumferentially aligned fibers (TACS2), and radially arranged fibers (TACS3). Even though a clear negative correlation between TACS and patient survival has been established, the specific roles and extent of TACS in T cell-driven cancer evolution remain uncharacterized. The precise mechanisms underlying how TACS influences cell movement are still not fully elucidated, with divergent opinions on whether and how the ECM mediates immune cell infiltration. At present, we currently lack a physical model relating the impact of TACS on the spatial co-evolution between an adaptive immune repertoire and a heterogeneous population of evading cancer cells. Here, we developed the EVO-ACT (EVOlutionary Agent-based Cancer T cell interaction) model, which is based on the Gillespie algorithm, to study the effects of TACS on tumor evolution and dynamical tumor-T cell interactions. Our analysis of tumor-T cell interactions across three TACS types reveals distinct migration patterns, with TACS3 showing the highest efficiency, TACS1 intermediate, and TACS2 the lowest. TACS impacts T cells more than tumors, with stronger chemokine gradients aiding T cell infiltration. Our model suggests that despite TACS2's limited efficiency, it cannot fully impede T cell infiltration. Varied migration efficiencies lead to diverse T cell infiltration patterns and immunoediting levels. TACS3 shows a benefit to immune recognition and higher survival, contrary to lower survival in TACS3 patients observed clinically in breast cancer. Our results indicate that only after introducing phenotypic changes, such as Epithelial-Mesenchymal Transition (EMT), and noting that EMT occurs exclusively after the TACS2-TACS3 progression, can our findings successfully explain the clinically observed trends. Considering mesenchymal tumor cells' ability to upregulate PD-1/PD-L1, we explore TACS-specific tumor evolution's impact on PD-1/PD-L1 inhibitor responses and patient survival. We observe higher tumor escape rates and lower survival in TACS3 compared to TACS2. Thus, TACS3 alone does not account for lower survival in breast cancer; TACS3-associated late-stage cancer byproducts like EMT and elevated checkpoint expression significantly decrease survival.
 Citation Format: Yijia Fan, Jason Tomas George. ECM architecture drives distinct evolutionary patterns of tumor immune escape and elimination in breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A031.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52d9e3eca63d450bccf177de70595f8faf5e6be" target='_blank'>
              Abstract A031: ECM architecture drives distinct evolutionary patterns of tumor immune escape and elimination in breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Jason Tomas George
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc23d3d27f11aa0ad92c216de43831771b9dae4" target='_blank'>
              Differential cell architecture and microenvironmental responses of pretumoral and tumoral cellular models exposed to coverslip-induced hypoxia.
              </a>
            </td>
          <td>
            Magdalena Millán, Felipe Parietti, Florencia Lamela, M. C. De Rossi, Belén Benítez, Valeria Levi, Manoela Domingues, R. Bologna-Molina, M. Arocena, J. Hochmann
          </td>
          <td>2025-01-03</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer evolution follows complex trajectories involving diverse genomic alterations and clonal dynamics, making it challenging to validate computational methods for single-cell DNA sequencing analysis. Here we present cancerSimCraft, a comprehensive framework for simulating cancer genome data at both clonal and single-cell resolution. cancerSimCraft combines deterministic rules with stochastic processes to model various genomic events including CNVs, SNVs, and WGDs. The framework enables integration of real cancer genome patterns with user-defined parameters, supporting customizable simulation designs that reflect both empirical data and theoretical models. Through systematic benchmarking, we demonstrate cancerSimCraft’s utility in evaluating computational methods under various conditions, particularly focusing on the impact of dataset size and parameter sensitivity. We further demonstrated its application in exploring clonal evolution and mutation patterns through controlled in silico experiments. These comprehensive simulation capabilities make cancerSimCraft a valuable resource for both computational method development and theoretical studies in cancer genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617e592c6c24284ee3c33b23e1a58a1555ca1d22" target='_blank'>
              cancerSimCraft: A Multi-resolution Cancer Genome Simulator with Comprehensive Ground Truth Tracking
              </a>
            </td>
          <td>
            Haijing Jin, Nicholas Navin, Ken Chen
          </td>
          <td>2024-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Tumor is a complex and dynamic ecosystem formed by the interaction of numerous diverse cells types and the microenvironments they inhabit. Determining how cellular states change and develop distinct cellular communities in response to the tumor microenvironment is critical to understanding cancer progression. Tumour-associated macrophages (TAMs) are an important component of the tumour microenvironment and play a crucial role in cancer progression. This study was designed to identify cell-state-specific M2 macrophage markers associated with gastric cancer (GC) prognosis through integrative analysis of single-cell RNA sequencing (scRNA-seq) and bulk RNA-seq data using a machine learning framework named EcoTyper. Results The results showed that TAMs were classified into M1 macrophages, M2 macrophages, monocytes, undefined macrophages and dendritic cells, with M2 macrophages predominating. EcoTyper assigned macrophages to different cell states and ecotypes. A total of 168 cell-state-specific M2 macrophage markers were obtained by integrative analysis of scRNA-seq and bulk RNA-seq data. These markers could categorize GC patients into two clusters (clusters A and B) with different survival and M2 macrophages infiltration abundance. Cell adhesion molecules, cytokine-cytokine receptor interaction, JAK/STAT pathway, MAPK pathway were significantly enriched in cluster A, which had worse survival and higher M2 macrophages infiltration. Conclusion In conclusion, this study profiles a single-cell atlas of intratumor heterogeneity and defines the cell states and ecotypes of TAMs in GC. Furthermore, we have identified prognostically relevant cell-state-specific M2 macrophage markers. These findings provide novel insights into the tumor ecosystem and cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d8a2fb2bd7548dde928bcb962ded6e0380b6c3" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA-Sequencing Based on EcoTyper Machine Learning Framework Identifies Cell-State-Specific M2 Macrophage Markers Associated with Gastric Cancer Prognosis
              </a>
            </td>
          <td>
            A-Kao Zhu, Guang-Yao Li, Fang-Ci Chen, Jia-Qi Shan, Yu-Qiang Shan, Chen-Xi Lv, Zhi-Qiang Zhu, Yi-Ren He, Lu-Lu Zhai
          </td>
          <td>2024-12-01</td>
          <td>ImmunoTargets and Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d643827078776ad2a6449f6a7b6358707684af74" target='_blank'>
              Cell-cell interactions mediating primary and metastatic breast cancer dormancy.
              </a>
            </td>
          <td>
            Nicholas A Lenart, Shreyas S Rao
          </td>
          <td>2024-11-25</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The development of T cell receptor-engineered T cells (TCR-T) targeting intracellular antigens is a promising strategy for treating solid tumors; however, the mechanisms underlying their effectiveness remain poorly understood. In this study, we employed advanced techniques to investigate the functional state of T cells engineered with retroviral vectors to express a TCR specific for the NY-ESO-1 157-165 peptide in the HLA-A*02:01 context. Flow cytometry revealed a predominance of naïve T cells. Gene expression profiling using NanoString technology revealed upregulation of genes encoding chemokine receptors CCR2 and CCR5, indicating enhanced migration towards tumor sites. In the SK-Mel-37 xenograft model, these transduced T cells achieved complete tumor eradication. Furthermore, single-cell RNA sequencing (scRNA-seq) conducted 14 days post-TCR T cell infusion provided a comprehensive analysis of the in vivo adaptation of these cells, identifying a distinct subset of CD8+ effector T cells with an NK cell-like gene expression profile. Our findings indicate that NY-ESO-1 TCR-transduced T cells have the potential to mediate dual antitumor effects through both antigen-independent NK-like and antigen-specific CTL-like responses. This study underscores the potential of NY-ESO-1 TCR-T cells as potent tumor-eradicating agents, highlighting the importance of harnessing their versatile functional capabilities to refine and enhance therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c35da049d73d1a38187e30a25bcb51a1113dcd7" target='_blank'>
              Decoding NY-ESO-1 TCR T cells: transcriptomic insights reveal dual mechanisms of tumor targeting in a melanoma murine xenograft model
              </a>
            </td>
          <td>
            A. Alsalloum, S. Alrhmoun, O. Perik-Zavosdkaia, M. Fisher, M. Volynets, J. Lopatnikova, R. Perik-Zavodskii, J. Shevchenko, Julia Philippova, Olga Solovieva, Evgenii Zavjalov, V. Kurilin, H. Shiku, A. Silkov, S. Sennikov
          </td>
          <td>2024-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa47d77b71a7337ad0ecaef200be807283fa4101" target='_blank'>
              Heterogeneity-driven phenotypic plasticity and treatment response in branched-organoid models of pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            A. Papargyriou, Mulham Najajreh, David P Cook, Carlo Maurer, S. Bärthel, H. Messal, Sakthi K Ravichandran, Till Richter, Moritz Knolle, Thomas Metzler, Akul R Shastri, R. Öllinger, Jacob Jasper, Laura Schmidleitner, Surui Wang, C. Schneeweis, H. Ishikawa-Ankerhold, T. Engleitner, Laura Mataite, Mariia Semina, Hussein Trabulssi, S. Lange, Aashreya Ravichandra, Maximilian Schuster, S. Mueller, K. Peschke, A. Schäfer, S. Dobiasch, S.E. Combs, Roland M. Schmid, Andreas R. Bausch, R. Braren, I. Heid, C. Scheel, Günter Schneider, Anja Zeigerer, Malte Luecken, K. Steiger, G. Kaissis, J. van Rheenen, F. Theis, Dieter Saur, R. Rad, Maximilian Reichert
          </td>
          <td>2024-12-10</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Despite recent therapeutic breakthroughs, cancer patients continue to face high recurrence and mortality rates due to treatment resistance. Cancer stem cells (CSCs), a subpopulation with self-renewal capabilities, are key drivers of refractive disease. This review explores the application of molecular imaging techniques, such as PET and SPECT, for the noninvasive detection of CSCs. By providing real-time monitoring of CSCs, these imaging methods have the potential to predict therapy resistance and guide personalized treatment approaches. Here, we cover the biological characteristics of CSCs, mechanisms of therapy resistance, and the identification and targeting of CSC-specific biomarkers with molecular imaging. Additionally, we address the challenges and opportunities for the clinical translation of CSC imaging, highlighting strategies where CSC imaging can be used to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c598784fb4bf91a45e00e3bbc55f45f0d4b538" target='_blank'>
              Molecular Imaging of Cancer Stem Cells and Their Role in Therapy Resistance
              </a>
            </td>
          <td>
            Sofia N Dos Santos, T. Witney
          </td>
          <td>2025-01-01</td>
          <td>Journal of Nuclear Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, Kevin Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88ba70c1f7c9e0c6817db8a69bc543b13dfed9df" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics of esophageal squamous cell carcinoma with lymph node metastases.
              </a>
            </td>
          <td>
            Wei Guo, Bolun Zhou, Lizhou Dou, Lei Guo, Yong Li, J. Qin, Zhen Wang, Qilin Huai, Xuemin Xue, Yin Li, Jianming Ying, Qi Xue, Shugeng Gao, Jie He
          </td>
          <td>2025-01-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="ABSTRACT Macrophages are pivotal in driving breast tumor development, progression, and resistance to treatment, particularly in estrogen receptor-positive (ER+) tumors, where they infiltrate the tumor microenvironment (TME) influenced by cancer cell-secreted factors. By analyzing single-cell RNA sequencing data from 25 ER+ tumors, we elucidated interactions between cancer cells and macrophages, correlating macrophage density with epithelial cancer cell density. We identified that S100A11, a previously unexplored factor in macrophage-cancer crosstalk, predicts high macrophage density and poor outcomes in ER+ tumors. We found that recombinant S100A11 enhances macrophage infiltration and migration in a dose-dependent manner. Additionally, in a 3D matrix using a panel of three ER+ breast cancer cell lines, we showed that secreted S100A11 levels from cancer cells were associated with increased monocyte infiltration that subsequently differentiation toward macrophages. Genetic silencing of S100A11 in the S100A11-high T47D cancer cells reduced monocyte infiltration, consistent with results using a S100A11 blocking antibody in T47D cancer cells and in a clinically relevant patient-derived organoid model. Phenotypic analysis of macrophages cocultured with T47D cancer cells following S100A11 knockdown revealed lower expression of the immunosuppressive marker CD206, further underscoring the role of S100A11 as a paracrine regulator of pro-tumorigenic cancer-macrophage crosstalk. This study offers novel insights into the interplay between macrophages and cancer cells in ER+ breast tumors, highlighting S100A11 as a potential therapeutic target to modulate the macrophage-rich tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bd1881c70cca02440b0e461e40eba12027d0287" target='_blank'>
              Cancer-cell derived S100A11 promotes macrophage recruitment in ER+ breast cancer
              </a>
            </td>
          <td>
            Sanghoon Lee, Youngbin Cho, Yiting Li, Ruxuan Li, Angela Wong Lau, Matthew S Laird, Daniel B Brown, Priscilla McAuliffe, Adrian V. Lee, S. Oesterreich, Ioannis K. Zervantonakis, H. Osmanbeyoglu
          </td>
          <td>2024-11-26</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Spatial transcriptomics is revolutionizing our understanding of cellular dynamics and interactions in the tumor microenvironment (TME). By mapping distinct cellular populations and their interactions, we can gain valuable insights into tumor behavior. To investigate these dynamics in breast cancer, we conducted Xenium In-situ spatial transcriptomics on twenty formalin-fixed paraffin-embedded (FFPE) samples taken from eight patients undergoing neoadjuvant chemotherapy. These samples have been taken pre-treatment, mid-treatment and post-treatment from four cases with pathological complete response (pCR), and four cases with residual disease (RD) at time of surgery. Our analyses have found distinct cellular composition changes between pCR and RD cases. Patients with RD demonstrated an increase in endothelial, perivascular-like (PVL) and epithelial cells from pre- to post-treatment. This was accompanied by a decrease in myeloid and malignant epithelial cells at mid-treatment. In contrast, patients with pCR demonstrated increases in cancer-associated fibroblasts (CAFs) and myeloid cells, with reductions in plasmablasts and cancer epithelial cells at mid-treatment. This suggests that an increase in CAF and myeloid cells observed between pre-treatment and mid-treatment samples may be predictive of response. We performed ligand-receptor (LR) analyses, which demonstrated diverse myeloid interactions in RD cases and diverse CAF interactions in pCR cases in pre-treatment samples. Mid-treatment samples exhibited a decrease in interactions for CAFs, myeloid and T cells in pCR and RD cases. The most frequently identified CAF LR interactions, as senders, were between TGFB1/TGFB3 and TGFBR1/TGFBR2, while the most frequently identified myeloid LR interactions were between PDCD1LG2 - PDCD1 (encoding PDL2 & PD1), CCL8 - ACKR1 (encoding Atypical Chemokine Receptor 1) and CCL2 - ACKR1. Our findings indicate that the dynamic changes in cellular composition and interactions within the TME differs between responders and non-responders. To further explore these differences, we plan to perform spatially constrained analyses to identify unique cellular niches that may serve as potential biomarkers for informing novel therapeutic strategies.
 Citation Format: Jeremy Mo, Hanyun Zhang, Kate Harvey, Alexander Swarbrick. Spatial transcriptomics unveils disparities in tumor microenvironment dynamics between breast cancer responders and non-responders [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00d85f0bd80cb01173f30a8b1b70f6e0a4a64eb5" target='_blank'>
              Abstract C040: Spatial transcriptomics unveils disparities in tumor microenvironment dynamics between breast cancer responders and non-responders
              </a>
            </td>
          <td>
            Jeremy Mo, Hanyun Zhang, K. Harvey, Alexander Swarbrick
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer heterogeneity has been well understood based solely on tumor components. However, the tumor microenvironment, which plays a crucial role in cancer by regulating tumor cells’ differentiation, maturation, and malignant potential, remains under-explored. This study conducted high-throughput RNA-Seq analysis on fresh breast cancer tissues from each molecular subtype and a pure population of cancer-associated fibroblasts (CAFs) isolated from cancer tissue and adjacent normal parts. Based on their functions, we identified three populations of CAFs: a) immune response-related, b) ECM remodeling, and c) calcium/protein binding. Validation of differentially expressed genes among these CAF populations revealed subtype-specific correlations. For instance, CAFs expressing immune response-related genes were significantly enriched in Luminal A/B and TNBC (log +1-fold; p =0.00047), compared to Her-2 positive (log -2.3-fold; p =0.0026). ECM remodeling genes exhibited greater expression in Her-2 positive and TNBC (log +4 to +11-fold; p =7.87E-08) when compared to Luminal subtypes (log +1.5-fold to +6-fold; p =8.66E-05). Calcium binding genes displayed overall similar expression across breast cancer subtypes. Thus, this study identified and underscored the functional heterogeneity of CAFs within the tumor microenvironment across the molecular subtypes of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e870b934402b9a2b10afe0f5e6b9ec8fa60b89" target='_blank'>
              Deciphering Functional Heterogeneity of Cancer-Associated Fibroblasts Across Molecular Subtypes of Breast Cancer
              </a>
            </td>
          <td>
            D. Kashyap
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="CDK4/6 inhibitors such as palbociclib block cell cycle progression and improve outcomes for many ER+/HER2- breast cancer patients. Unfortunately, many patients are initially resistant to the drug or develop resistance over time in part due to heterogeneity among individual tumor cells. To better understand these mechanisms of resistance, we used multiplex, single-cell imaging to profile cell cycle proteins in ER+ breast tumor cells under increasing palbociclib concentrations. We then applied spherical principal component analysis (SPCA), a dimensionality reduction method that leverages the inherently cyclical nature of the high-dimensional imaging data, to look for changes in cell cycle behavior in resistant cells. SPCA characterizes data as a hypersphere and provides a framework for visualizing and quantifying differences in cell cycles across treatment-induced perturbations. The hypersphere representations revealed shifts in the mean cell state and population heterogeneity. SPCA validated expected trends of CDK4/6 inhibitor response such as decreased expression of proliferation markers (Ki67, pRB), but also revealed potential mechanisms of resistance including increased expression of cyclin D1 and CDK2. Understanding the molecular mechanisms that allow treated tumor cells to evade arrest is critical for identifying targets of future therapies. Ultimately, we seek to further SPCA as a tool of precision medicine, targeting treatments by individual tumors, and extending this computational framework to interpret other cyclical biological processes represented by high-dimensional data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57de25c3330f7fce9f46a524989a6ef4e9398a4c" target='_blank'>
              Spherical Manifolds Capture Drug-Induced Changes in Tumor Cell Cycle Behavior
              </a>
            </td>
          <td>
            Olivia Wen, Samuel C Wolff, Wayne Stallaert, Didong Li, Jeremy E. Purvis, Tarek M. Zikry
          </td>
          <td>2024-11-21</td>
          <td>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor heterogeneity is a significant factor influencing cancer treatment effectiveness and can arise from genetic, epigenetic, and phenotypic variations among cancer cells. Understanding how tumor heterogeneity impacts tumor evolution and therapy response can lead to more effective treatments and improved patient outcomes. Traditional bulk genomic approaches fail to provide insights into cellular-level events, whereas single-cell RNA sequencing (scRNA-seq) offers transcriptomic analysis at the individual cell level, advancing our understanding of tumor growth, progression, and drug response. However, implementing single-cell approaches in clinical trials involves challenges, such as obtaining high-quality cells, technical variability, and the need for complex computational analysis. Effective implementation of single-cell genomics in clinical trials requires a collaborative “Team Medicine” approach, leveraging shared resources, expertise, and workflows. Here, we describe key technical considerations in implementing the collection of research biopsies and lessons learned from integrating scRNA-seq into City of Hope’s clinical trial design, highlighting collaborative efforts between computational and clinical teams across breast, brain, and ovarian cancer studies to understand the composition, phenotypic state, and underlying resistance mechanisms within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8e501cb60571ffc07f7277f4aa067130d5278a3" target='_blank'>
              Single-Cell Transcriptomics Sheds Light on Tumor Evolution: Perspectives from City of Hope’s Clinical Trial Teams
              </a>
            </td>
          <td>
            Patrick A. Cosgrove, Andrea H. Bild, T. Dellinger, B. Badie, J. Portnow, Aritro Nath
          </td>
          <td>2024-12-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Breast cancer has the highest global incidence among all cancers, affecting more than 2 million individuals annually. Despite the availability of new drugs and novel treatment combinations, it is postulated that the incidence and mortality of breast cancer will rise by 40.8% and 51.9% respectively by 2040. Such dire statistics are associated with the clonal evolution of cancer cells that leads to therapeutic resistance and consequent relapse in breast cancer patients. On the other hand, the tumor microenvironment (TME) comprising of tumor cells, cancer-associated immune cells, re-programmed stromal cells, and the extracellular matrix (ECM) creates an immunosuppressive niche facilitating immune evasion. This review focuses on a critical cellular component of the tumor microenvironment, the tumor-associated macrophages (TAMs) in breast cancer immunotherapy. Macrophages are inherently plastic and can convert from an anti-tumor M1 phenotype to a pro-tumor M2 phenotype based on microenvironmental cues. Cancer cells facilitate these cues, allowing the tumor-associated macrophages to gain M2 phenotype and mediate immune evasion. Therefore, knowledge of the distinct role of tumor-associated macrophages in immune evasion can help design therapeutics such as engineered macrophages, M2 targeting drugs, and novel macrophage-mediated drug delivery strategies for long-term survival in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee6e4305fc222b5a36786a300bfa76f97eae0c28" target='_blank'>
              Reprogramming the tumor microenvironment – macrophages emerge as key players in breast cancer immunotherapy
              </a>
            </td>
          <td>
            Ana Sami, Afsheen Raza
          </td>
          <td>2024-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumor cells to adapt to and exploit their microenvironment for sustained growth. The liver is a common site of metastasis, but the interactions between tumor cells and hepatocytes remain poorly understood. In the context of liver metastasis, these interactions play a crucial role in promoting tumor survival and progression. This study leverages multiomics coverage of the microenvironment via liquid chromatography and high-resolution, high-mass accuracy mass spectrometry-based untargeted metabolomics, 13C-stable isotope tracing, and RNA sequencing to uncover the metabolic impact of co-localized primary hepatocytes and a colon adenocarcinoma cell line, SW480, using a 2D co-culture model. Metabolic profiling revealed disrupted Warburg metabolism with an 80% decrease in glucose consumption and 94% decrease in lactate production by hepatocyte-SW480 co-cultures relative to SW480 control cultures. Decreased glucose consumption was coupled with alterations in glutamine and ketone body metabolism, suggesting a possible fuel switch upon co-culturing. Further, integrated multiomic analysis indicates that disruptions in metabolic pathways, including nucleoside biosynthesis, amino acids, and TCA cycle, correlate with altered SW480 transcriptional profiles and highlight the importance of redox homeostasis in tumor adaptation. Finally, these findings were replicated in 3-dimensional microtissue organoids. Taken together, these studies support a bioinformatic approach to study metabolic crosstalk and discovery of potential therapeutic targets in preclinical models of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8536f19d4a9f802b3a9ad739c338954491fd6245" target='_blank'>
              Deciphering Colorectal Cancer-Hepatocyte Interactions: A Multiomic Platform for Interrogation of Metabolic Crosstalk in the Liver-Tumor Microenvironment
              </a>
            </td>
          <td>
            Alisa B. Nelson, Lyndsay E. Reese, Elizabeth Rono, Eric D. Queathem, Yinjie Qiu, Braedan M. McCluskey, Alexandra Crampton, Eric Conniff, Katherine A Cummins, Ella Boytim, Senali Dansou, Justin H. Hwang, S. Safo, Patrycja Puchalska, David K. Wood, K. Schwertfeger, Peter A. Crawford
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81af71d49f35f4ca08b391cf52780be4b4e726a9" target='_blank'>
              Integrated single-cell analysis reveals distinct epigenetic-regulated cancer cell states and a heterogeneity-guided core signature in tamoxifen-resistant breast cancer
              </a>
            </td>
          <td>
            Kun Fang, A. Ohihoin, Tianxiang Liu, Lavanya Choppavarapu, Bakhtiyor Nosirov, Qianben Wang, Xue-Zhong Yu, S. Kamaraju, Gustavo W. Leone, Victor X Jin
          </td>
          <td>2024-11-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e7cfabf5bb0faa15589259ab10869fb550df73" target='_blank'>
              Divergent WNT signaling and drug sensitivity profiles within hepatoblastoma tumors and organoids
              </a>
            </td>
          <td>
            T. A. Kluiver, Yuyan Lu, Stephanie A Schubert, Lianne Kraaier, F. Ringnalda, P. Lijnzaad, Jeff DeMartino, Wouter L Megchelenbrink, Vicky Amo-Addae, Selma Eising, F. D. de Faria, Daniel Münter, M. van de Wetering, K. Kerl, E. Duiker, Marius C van den Heuvel, Vincent E. de Meijer, R. D. de Kleine, J. J. Molenaar, T. Margaritis, Hendrik G. Stunnenberg, R. R. de Krijger, J. Zsiros, H. Clevers, Weng Chuan Peng
          </td>
          <td>2024-11-20</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0c7c17d016e29c5fa8d5c2c935484d2301a05e" target='_blank'>
              Anaplastic thyroid carcinoma: vimentin segregates at the invasive front of tumors in a murine xenograft model.
              </a>
            </td>
          <td>
            Alessandro Miraglia, L. Giannotti, F. De Nuccio, A. S. Treglia, Michele Maffia, D. Lofrumento, Bruno Di Jeso, G. Nicolardi
          </td>
          <td>2024-11-18</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background: Microvascular invasion (MVI) is a critical prognostic factor in intrahepatic cholangiocarcinoma (ICC), strongly associated with postoperative recurrence. However, the phenotypic features and spatial organization of MVI remain inadequately understood. Methods: We performed a spatial transcriptomic analysis on 29,632 spots from six ICC samples, manually delineating MVI clusters using the cloupe software. Key biomarkers were identified and validated in an independent cohort of 135 ICC patients. Functional and survival analyses were conducted to assess clinical relevance, and cell-cell communication pathways were investigated. Results: MVI regions exhibited heightened proliferation, angiogenesis, and epithelial-mesenchymal transition, driven by increased expression of transcription factors SOX10, ZEB1, and SNAI2. CXCL5 and SLC6A14 were identified as potential MVI biomarkers and showed high expression in tumor-invasive areas. Serum CXCL5 demonstrated strong predictive power for vascular invasion (AUC = 0.92) and intrahepatic metastasis (AUC = 0.96). High expression of both CXCL5 and SLC6A14 was associated with the worst survival outcomes. MVI regions were enriched with immunosuppressive MRC1+ macrophages and exhibited elevated immune checkpoint expression, including HAVCR2 and TIGHT, indicative of immune resistance. Cell-cell communication analysis revealed CXCL5-CXCR2 and LGALS9-HAVCR2 as key ligand-receptor pairs contributing to the immunosuppressive microenvironment. Conclusions: This study identifies CXCL5 and SLC6A14 as key biomarkers of MVI, highlighting their roles in tumor proliferation, immune resistance, and poor clinical outcomes. These findings provide valuable insights into the spatial organization of MVI and its contribution to ICC progression, offering potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80f1aeb6ba61c9f3ee11c5dafb6bfea9349b09d" target='_blank'>
              Spatial analyses revealed CXCL5 and SLC6A14 as the markers of microvascular invasion in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            G. Fan, L. Dai, Tongji Xie, Lin Li, Le Tang, Xiaohong Han, Yuankai Shi
          </td>
          <td>2024-12-11</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Inherited genetic variants contribute to Barrett’s esophagus (BE) and esophageal adenocarcinoma (EAC) but it is unknown which cell types are involved in this process. We performed single cell RNA-sequencing of BE, EAC and paired normal tissues and integrated data of a genome-wide association study to determine cell type-specific genetic risk and cellular processes that contribute to BE and EAC. The analysis revealed that EAC development is driven to a greater extent by local cellular processes than BE development. One cell type of BE origin (BE-EAC) and cellular processes that control the differentiation of columnar cells are of particular relevance for EAC development. Further, specific subtypes of fibroblasts and endothelial cells contribute to BE and EAC development, while dendritic cells and CD4+ memory T cells contribute exclusively to BE development. The diagnostic use of markers characterizing the identified cell types and cellular processes should be explored in future for EAC prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cab9925c507d28dd37c045256d8f4a2182543fb5" target='_blank'>
              Single cell analysis of Barrett’s esophagus and carcinoma reveals cell types conferring risk via genetic predisposition
              </a>
            </td>
          <td>
            Marten C. Wenzel, Ann-Sophie Giel, Patrick S. Plum, Sascha Hoppe, Marek Franitza, Christoph Jonas, Pouria Dasmeh, René Thieme, Yue Zhao, Dominik Heider, C. Palles, R. Fitzgerald, Christiane J Bruns, Reinhard Buettner, Alexander Quaas, Ines Gockel, Carlo Maj, Seung-Hun Chon, Johannes Schumacher, Axel M. Hillmer
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Cancer cells exhibit a remarkable resilience to cytotoxic stress, often adapting through transcriptional changes linked to alterations in chromatin structure. In several types of cancer, these adaptations involve epigenetic modifications and restructuring of topologically associating domains (TADs). However, the underlying principles by which chromatin architecture facilitates such adaptability across different cancers remain poorly understood. To investigate the role of chromatin in this process, we developed a physics-based mechanistic model that connects chromatin organization to cell fate decisions, specifically survival following chemotherapy. Our model builds on the observation that chromatin forms packing domains, which influence transcriptional efficiency through macromolecular crowding. The model accurately predicts chemoevasion in vitro, suggesting that changes in packing domains affect the likelihood of survival. Consistent results across diverse cancer types indicate that the model captures fundamental principles of chromatin-mediated adaptation, independent of the specific cancer or chemotherapy mechanisms involved. Based on these insights, we hypothesized that compounds capable of modulating packing domains, termed Transcriptional Plasticity Regulators (TPRs), could prevent cellular adaptation to chemotherapy. Using live-cell chromatin imaging, we conducted a compound screen that identified several TPRs which synergistically enhanced chemotherapyinduced cell death. The most effective TPR significantly improved therapeutic outcomes in a patient-derived xenograft (PDX) model of ovarian cancer. These findings underscore the central role of chromatin in cellular adaptation to cytotoxic stress and present a novel framework for enhancing cancer therapies, with broad potential across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f35f61f79981a14cfb46f8ab43779aba13110922" target='_blank'>
              Leveraging chromatin packing domains to target chemoevasion in vivo
              </a>
            </td>
          <td>
            J. Frederick, R. Virk, I. C. Ye, L. Almassalha, Greta Wodarcyk, D. VanDerway, Paola Carrillo Gonzalez, Rikkert J. Nap, Vasundhara Agrawal, Nicholas M. Anthony, John Carinato, Wing Shun Li, Cody L Dunton, Karla I Medina, Rivaan Kakkaramadam, Surbhi Jain, Shohreh Shahabi, Guillermo Ameer, I. Szleifer, Vadim Backman
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Uveal melanoma (UM), arising from melanocytes in the choroid, accounts for 3% to 5% of all melanocytic tumors and over 70% of intraocular malignancies. Despite effective local treatments, metastasis remains a significant challenge, with more than half of patients developing metastatic disease within ten years. Conventional therapies often yield poor outcomes, highlighting the urgent need for novel therapeutic strategies to enhance survival and prognosis for UM patients. Methods We conducted a detailed analysis of the GSE139829 dataset, focusing on scRNA-seq data from eight primary UM patients and three with metastatic disease. Through clustering and marker gene expression analyses, we identified distinct subtypes of UM tumor cells and examined their transcriptional, metabolic, and intercellular communication profiles. We developed a novel prognostic model, PCOLCE TCs Risk Score (PTRS), centered on the C5 PCOLCE+ tumor cells, which was validated through in vitro functional assays. Additionally, we performed immune infiltration and metabolic pathway analyses to elucidate tumor-immune interactions and their clinical significance. Results We identified eight distinct cell types in UM and classified tumor subpopulations into six subgroups. The C5 PCOLCE+ TCs subpopulation was highlighted as crucial in UM malignancy, demonstrating high differentiation potential and a significant role in tumor progression. CellChat analysis revealed substantial communication between C5 PCOLCE+ TCs and fibroblasts, suggesting their involvement in tumor growth and extracellular matrix remodeling. Metabolic pathway analysis indicated enhanced oxidative phosphorylation and glutathione metabolism in this subpopulation. Additionally, we developed a PTRS model based on C5 PCOLCE+ TCs, identifying CITED1 as a high-risk gene that promotes UM cell proliferation, invasion, and migration in vitro. Conclusion This study provides insights into UM metastasis via single-cell analysis, identifying C5 PCOLCE+ TCs as key malignancy drivers associated with oxidative phosphorylation and immune interactions. Our PTRS model highlights CITED1 as a high-risk gene that promotes UM cell proliferation, paving the way for new prognostic models and therapeutic targets to enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960495014c1bd91ad32e09912185659c46495106" target='_blank'>
              Single-cell RNA sequencing reveals key molecular drivers and immune landscape in uveal melanoma: implications for targeted therapy and prognostic modeling
              </a>
            </td>
          <td>
            Zeyu Song, Wenwen Shao, Zhikai Xiahou, Yue Xu, Xiaofeng Zhang
          </td>
          <td>2024-11-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="While transcription factors (TFs) provide essential cues for directing and redirecting cell fate, TFs alone are insufficient to drive cells to adopt alternative fates. Rather, transcription factors rely on receptive cell states to induce novel identities. Cell state emerges from and is shaped by cellular history and the activity of diverse processes. Here, we define the cellular and molecular properties of a highly receptive state amenable to transcription factor-mediated direct conversion from fibroblasts to induced motor neurons. Using a well-defined model of direct conversion to a post-mitotic fate, we identify the highly proliferative, receptive state that transiently emerges during conversion. Through examining chromatin accessibility, histone marks, and nuclear features, we find that cells reprogram from a state characterized by global reductions in nuclear size and transcriptional activity. Supported by globally increased levels of H3K27me3, cells enter a quiescent-like state of reduced RNA metabolism and elevated expression of REST and p27, markers of quiescent neural stem cells. From this transient state, cells convert to neurons at high rates. Inhibition of Ezh2, the catalytic subunit of PRC2 that deposits H3K27me3, abolishes conversion. Our work offers a roadmap to identify global changes in cellular processes that define cells with different conversion potentials that may generalize to other cell-fate transitions. Highlights Proliferation drives cells to a compact nuclear state that is receptive to TF-mediated conversion. Increased receptivity to TFs corresponds to reduced nuclear volumes. Reprogrammable cells display global, genome-wide increases in H3K27me3. High levels of H3K27me3 support cells’ transits through a state of altered RNA metabolism. Inhibition of Ezh2 increases nuclear size, reduces the expression of the quiescence marker p27. Acute inhibition of Ezh2 abolishes motor neuron conversion. One Sentence Summary Cells transit through a quiescent-like state characterized by global reductions in nuclear size and transcriptional activity to convert to neurons at high rates. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a19355f17f418661bc24e1cda5ae80bdeb54b6c2" target='_blank'>
              Cells transit through a quiescent-like state to convert to neurons at high rates
              </a>
            </td>
          <td>
            A. Beitz, Jmy Teves, C. Oakes, C. Johnstone, N. Wang, JM Brickman, Kate E. Galloway
          </td>
          <td>2024-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Despite the ability to induce strong clinical responses, therapies that target mutational drivers of oncogenic signaling are not curative in advanced cancers, as a subset of cancer cells is capable of surviving therapy and evolving resistance. While therapy resistance is commonly viewed as a cell-intrinsic phenomenon, multiple factors produced by tumors can also confer strong therapy resistance. In contrast to the detailed elucidation of molecular mediators of cell-intrinsic and stroma-mediated (SM) resistance, the relative importance of cell-intrinsic and SM resistance to in vivo therapeutic responses remains poorly defined. To address this gap of knowledge, we used a well-characterized experimental model of ALK+ lung cancer H3122 cell line that exemplifies a duality observed across common models of targetable cancers. Under standard stroma-free in vitro cultures, H3122 cells can survive under therapeutically relevant concentrations of ALK inhibitors (ALKi), eventually acquiring cell-intrinsic therapy resistance. At the same time, co-culture with stromal fibroblasts drastically reduces the sensitivity of H3122 cells to ALKi, indicating the relevance of SM resistance. To understand the impact of SM resistance on in vivo responses, we sought to identify the mechanism(s) responsible for the therapy-protective effects of stromal fibroblasts, subsequently interrogating the impact of the disruption of this mechanism on in vivo therapeutic responses. Our in vitro studies with a large panel of primary stromal fibroblast isolates identified the HGF-cMET axis as the major mechanism responsible for ALKi desensitization. Surprisingly, xenograft validation studies demonstrated a weak effect of HGF-cMET modulation. Histological analyses of tumor tissues revealed that this relative weakness is attributable to a strong, spatially limited HGF-independent component of SM resistance. Mechanistic follow-up demonstrated that the HGF-independent component of SM resistance integrates the effect of multiple well-known juxtacrine and paracrine-acting mechanisms. Whereas the multifactorial nature of stroma-mediated resistance prevented a clear assessment of its relative contribution to in vivo therapy responses, our spatial analyses indicate that SM resistance dominates the ability of tumors to avoid therapeutic elimination. Surprisingly, we found that SM resistance was also a substantial contributor to tumor relapse, indicating that in vivo therapy resistance can integrate both cell-intrinsic and cell-extrinsic effects. This duality of therapy resistance and the multifactorial underpinning of both cell-intrinsic and stroma-mediated resistance present a challenge to therapeutic strategies focused on identifying and targeting specific resistance mechanisms. Our studies suggest that this limitation can be overcome by shifting the therapeutic focus towards shared orthogonal therapeutic sensitivities of tumor cells within residual disease.
 Citation Format: Bina Desai, Tatiana Miti, Sandhya Prabhakaran, Daria Miroshnychenko, Menkara Henry, Viktoriya Marusyk, Chandler Gatenbee, Marilyn Bui, Jacob Scott, Philipp M. Altrock, Eric Haura, Alexander R.A. Anderson, David Basanta, Andriy Marusyk. Spatially limited stroma-mediated resistance potentiates targeted therapy resistance through an integration of multiple juxtacrine and paracrine mechanisms [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C035.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943bc0d90e7f0f13d1f5d2038bd76c206372d768" target='_blank'>
              Abstract C035: Spatially limited stroma-mediated resistance potentiates targeted therapy resistance through an integration of multiple juxtacrine and paracrine mechanisms
              </a>
            </td>
          <td>
            Bina Desai, Tatiana Miti, Sandhya Prabhakaran, Daria Miroshnychenko, Menkara-Chinua N Henry, Viktoriya Marusyk, Chandler D. Gatenbee, Marilyn M Bui, Jacob G. Scott, Philipp M. Altrock, E. Haura, Alexander R. A. Anderson, D. Basanta, A. Marusyk
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Triple-negative breast cancer (TNBC) is associated with a high risk of distant metastasis, particularly to the lung and liver. Both intrinsic characteristics of cancer cells and the tumor microenvironment (TME) influence TNBC growth and metastasis. The TME, comprising stromal, immune, and endothelial cells, exhibits heterogeneity both between and within cell types. Identifying the molecular characteristics of TME cells that affect cancer progression is crucial. We aim to investigate this using single-cell RNA sequencing of patient-derived xenograft (PDX) models to explore TME cells and genes involved in TNBC metastasis. We established PDX models by transplanting tumor tissues from 26 TNBC patients into immunodeficient mice. These models were categorized based on their metastatic potential and patient outcomes: non-metastatic (n=5) and metastatic (n=4). Single-cell RNA sequencing of nine PDX tumors revealed significant differences in gene expression of murine stromal cells between metastatic and non-metastatic models. We identified ten genes (Serpinb2, Spp1, Tnc, Thbs1, Timp1, Il11, Mt2, Crabp1, Cck, Mt1) that were highly expressed in the stromal cells of metastatic PDX models. To test if TNBC cells influence these fibroblast genes, we exposed NIH3T3 fibroblasts to conditioned media from the 4T1 TNBC cell line. In three independent experiments, only Crabp1 expression in NIH3T3 cells was consistently increased by 4T1- conditioned media. CRABP1, a retinoic acid-binding protein, is known for its role in differentiation and proliferation by regulating MAPK signaling. Although Crabp1-positive cancer-associated fibroblasts have been noted, its role in TNBC metastasis was previously unexplored. We investigated CRABP1 function using a CRABP1 knockdown NIH3T3 cell line. CRABP1 knockdown led to reduced invasion and migration of 4T1 cells in trans-well assays and decreased invasiveness of 4T1 spheroids in a collagen matrix. Additionally, CRABP1 knockdown NIH3T3 cells showed reduced proliferation in vitro and fewer alpha-SMA positive cancer-associated fibroblasts in co-injected 4T1 tumors in BALB/C mice. In summary, CRABP1, identified through single-cell RNA sequencing of stromal cells in TNBC PDX models, is a potential regulator of TNBC metastasis, affecting cancer cell migration, invasion, and fibroblast proliferation.
 Citation Format: Woohang Heo, Yujeong Her, Sieun Yang, Dakyung Lee, Rokhyun Kim, Jong-Il Kim, Hyeong-Gon Moon. Single cell RNA sequencing of triple negative breast cancer patient-derived xenograft model identifies CRABP1 of cancer-associated fibroblast as a key regulator of breast cancer metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f55fcf53482b3293e557961f04bd5df55bfd4945" target='_blank'>
              Abstract A034: Single cell RNA sequencing of triple negative breast cancer patient-derived xenograft model identifies CRABP1 of cancer-associated fibroblast as a key regulator of breast cancer metastasis
              </a>
            </td>
          <td>
            W. Heo, Y. Her, Sieun Yang, Dakyung Lee, Rokhyun Kim, Jong-Il Kim, H. Moon
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="In 2024, the United States was projected to experience 2 million new cancer diagnoses and approximately 611,720 cancer-related deaths, reflecting a broader global trend in which cancer cases are anticipated to exceed 35 million by 2050. This increasing burden highlights ongoing challenges in cancer treatment despite significant advances that have reduced cancer mortality by 31% since 1991. Key obstacles include the disease’s inherent heterogeneity and complexity, such as treatment resistance, cancer stem cells, and the multifaceted tumor microenvironment (TME). The TME—comprising various tumor and immune cells, blood vessels, and biochemical factors—plays a crucial role in tumor growth and resistance to therapies. Recent innovations in cancer treatment, particularly in the field of immuno-oncology, have leveraged insights into TME interactions. An emerging example is the FDA-approved therapy using tumor-infiltrating lymphocytes (TILs), demonstrating the potential of cell-based approaches in solid tumors. However, TIL therapy is just one of many strategies being explored. This review provides a comprehensive overview of the emerging field of immuno-oncology, focusing on how novel therapies targeting or harnessing components of the TME could enhance treatment efficacy and address persistent challenges in cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69efe2b80b4ba6460d216813b5cdf5408b15b18" target='_blank'>
              Targeting Cancer: Microenvironment and Immunotherapy Innovations
              </a>
            </td>
          <td>
            Irena Padzińska-Pruszyńska, B. Taciak, Łukasz Kiraga, A. Smolarska, M. Górczak, P. Kucharzewska, M. Kubiak, J. Szeliga, Agata Matejuk, M. Król
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Lung Adenocarcinoma (LUAD) leads in cancer deaths worldwide. Immune Checkpoint blockade (ICB) therapy has improved patient survival of some subtypes (like Kras oncogene-driven; loss of tumor-suppressor p53– “KP”), but not others (Kras oncogene-driven; loss of LKB1– “KL”). Some studies suggest the poorly inflamed KL tumor environment reduces T cell infiltration. However, the impact of other immune mechanisms/cell types on overall ICB-responsiveness remains understudied. Tertiary lymphoid structures (TLSs) are T/B cell aggregates organized by various cell types that form within inflamed non-lymphoid organs. Recent studies have strongly correlated tumor-associated TLSs (TA-TLSs) with positive clinical outcomes and are predictive of patient responders to immunotherapy in many solid tumor types. Thus, understanding the mechanistic factors that sustain mature TA-TLSs will provide more clarity on what molecularly entails an optimal immune response within an ICB-treated patient. To that end, a major advancement has been our knowledge of TA-TLS phenotypic markers driven through developments in multiplex immunofluorescence imaging of 2D tissue sections. However, tissue sectioning destroys the organ architecture, providing limited spatial information over a small field of view. Therefore, non-destructive 3D fluorescent imaging of whole organs, and lightsheet fluorescence microscopy (LSFM) can provide a novel and broader global view of multicellular structures, like TA-TLSs within the whole organ. By using the Blaze ultramicroscope, we imaged whole lung lobes from our novel autochthonous neoantigen-induced lung adenocarcinoma cancer models of KP and KL mice based on our previously published immunogenic KP x NINJA (inversion-induced joined neoantigen) model, which relies on Cre- recombinase activity dependent activation of KrasLox-Stop-Lox (LSL)G12D and deletion of p53fl/fl. Then, doxycycline and tamoxifen administration lead to lung-specific promoter [club cell secretory protein (CCSP)-rtTA] dependent expression of NINJA. We have also developed novel image segmentation methods using machine learning tools in Imaris 10.2 to identify tumors and immune cell clusters, enabling the quantification of the distance and volume of observable tumors and TLSs in the entire organ, which to our knowledge has not been previously attempted. Using this novel imaging and quantification pipeline, we recently discovered that while KP-NINJA lungs produce many TA-TLSs, the KL tumors (compared to KP) lack organized TA-TLSs, and that intriguingly, B-cells are sparsely sequestered near the main bronchus, away from tumors, resembling small bronchi-associated lymphoid structures (BALTs) that typically form in inflamed lungs. We are concurrently doing 2D multiplex cyclic-immunofluorescence imaging to characterize the variety of other cell types present in TLSs of KP-NINJA vs KL-NINJA and more closely understand the molecular cues sustaining TA-TLSs, providing clarity on what entails an “optimal” immune response within an ICB-treated patient.
 Citation Format: Advait Jeevanandam, Kelli Connolly, Nikhil Joshi. Spatial study of tertiary lymphoid structures in lung adenocarcinoma using 3D lightsheet whole-organ Imaging and machine learning-based quantification [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B032.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93e250b90708e399aa85957d7d601a70aaa9b96c" target='_blank'>
              Abstract B032: Spatial study of tertiary lymphoid structures in lung adenocarcinoma using 3D lightsheet whole-organ Imaging and machine learning-based quantification
              </a>
            </td>
          <td>
            Advait Jeevanandam, Kelli A. Connolly, Nikhil S. Joshi
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary Melanoma is an aggressive malignancy defined by significant intratumoral heterogeneity, driving its capacity for therapeutic resistance and recurrence. This study adopts a systems-level approach to dissect the melanoma microenvironment, focusing on intricate interactions between malignant cells and immune infiltrates. We identified critical regulatory networks and intercellular communication pathways that appear to influence disease progression. These findings highlight the dynamic interplay between tumor-intrinsic factors and the surrounding microenvironment, revealing potential mechanisms underlying immune evasion and therapy resistance. By mapping these complex interactions, the present study builds on our foundation for precision-based therapeutic strategies tailored to the unique biological landscape of melanoma, offering promise for improved clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f383353b5d7dd45f21551edcc7dd626f8651941" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA Data Reveals Complexity and Significance of the Melanoma Interactome
              </a>
            </td>
          <td>
            Michael J Diaz, J. T. Tran, Arthur M. Samia, M. Forouzandeh, Jane M. Grant-Kels, Marjorie E. Montanez-Wiscovich
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185d03d5b622173898f4a312b550c66a7f8243e6" target='_blank'>
              Soft tissue sarcomas at the single-cell and spatial resolution: new markers and targets.
              </a>
            </td>
          <td>
            M. Menyailo, Elena E. Kopantseva, A. Khozyainova, Anastasia A. Korobeynikova, E. Denisov
          </td>
          <td>2024-11-24</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaacb86999aeed003c707b4000adb56d67bb6547" target='_blank'>
              Spatial transcriptome profiling identifies DTX3L and BST2 as key biomarkers in esophageal squamous cell carcinoma tumorigenesis
              </a>
            </td>
          <td>
            Rutao Li, Na Li, Qianqian Yang, Xing Tong, Wei Wang, Chang Li, Jun Zhao, Dong Jiang, Haitao Huang, Chen Fang, Kai Xie, Jiamin Yuan, Shaomu Chen, Guangbin Li, Haitao Luo, Zhibo Gao, Dongfang Wu, Xiaoli Cui, Wei Jiang, Lingchuan Guo, Haitao Ma, Yu Feng
          </td>
          <td>2024-12-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The resistance of tumors to immune checkpoint inhibitors (ICI) may be intricately linked to cellular senescence, although definitive clinical validation remains elusive. In this study, comprehensive pan-cancer scRNA-seq analyses identify fibroblasts as exhibiting the most pronounced levels of cellular senescence among tumor-associated cell populations. To elucidate this phenomenon, a fibroblast senescence-associated transcriptomic signature (FSS), which correlated strongly with protumorigenic signaling pathways and immune dysregulation that fosters tumor progression, is developed. Leveraging the FSS, the machine learning (ML) framework demonstrates exceptional accuracy in predicting ICI response and survival outcomes, achieving superior area under curve (AUC) values across validation, testing, and in-house cohorts. Strikingly, FSS consistently outperforms established signatures in predictive robustness across diverse cancer subtypes. From an integrative analysis of 17 CRISPR/Cas9 libraries, CDC6 emerges as a pivotal biomarker for pan-cancer ICI response and prognostic stratification. Mechanistically, experimental evidence reveals that CDC6 in tumor cells orchestrates fibroblast senescence via TGF-β1 secretion and oxidative stress, subsequently reprogramming the tumor microenvironment and modulating ICI response. These findings underscore the translational potential of targeting fibroblast senescence as a novel therapeutic strategy to mitigate immune resistance and enhance antitumor efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83eb29659bd661ae208e4642398f8b3b7069e404" target='_blank'>
              A Distinguished Roadmap of Fibroblast Senescence in Predicting Immunotherapy Response and Prognosis Across Human Cancers.
              </a>
            </td>
          <td>
            Dongjie Chen, Pengyi Liu, Jiayu Lin, Longjun Zang, Yihao Liu, Shuyu Zhai, Xiongxiong Lu, Yuanchi Weng, Hongzhe Li
          </td>
          <td>2024-12-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Anaplastic Thyroid Cancer (ATC) is an aggressive form of cancer with poor prognosis, heavily influenced by its tumor immune microenvironment (TIME). Understanding the cellular and gene expression dynamics within the TIME is crucial for developing targeted therapies. This study analyzes the immune microenvironment of ATC and Papillary Thyroid Cancer (PTC) using single-cell RNA sequencing (scRNA-seq). We performed a comprehensive scRNA-seq analysis on ATC and PTC samples, incorporating cell type annotation, marker gene identification, and clustering based on gene expression. A specific focus was on the prevalence and biomarkers of Pre-Exhausted CD8+ T cells in ATC, utilizing the single-cell tumor immune atlas for immune cell characterization. The scRNA-seq analysis identified distinct immune cell populations and differentially expressed genes in ATC and PTC samples. Notably, Pre-Exhausted CD8+ T cells were found to be prevalent in ATC datasets. Additional immunofluorescence staining and co-culture experiments with the ATC cell line identified GNLY, a member of the saposin-like protein family as a potential biomarker for Pre-Exhausted CD8+ T cells in ATC. This study provides valuable insights into the immune landscape of ATC, emphasizing the prevalence of Pre-Exhausted CD8+ T cells and identifying GNLY as a potential biomarker. Understanding the TIME composition and the role of specific immune cells in cancer progression can inform the development of targeted immunotherapies for ATC. Future research should explore the functional implications of GNLY and other identified biomarkers in modulating the immune response in thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bee3b0c483d956dde8850d164e4f812a4b30008" target='_blank'>
              Persistent pre-exhausted CD8+ T cells shape the tumor immune microenvironment in anaplastic thyroid cancer.
              </a>
            </td>
          <td>
            X. Ruan, Mei Tao, Yuanxing Dong, Linfei Hu, Guangwei Xu, Chuanxiang Hu, Yue Huang, Yuqi Wang, Jialong Yu, Wei Luo, Ming Gao, Min Zhao, Xiangqian Zheng
          </td>
          <td>2025-01-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is the most common and aggressive type of primary brain cancer, known for its rapid growth, invasion into surrounding brain tissue, and strong resistance to current therapies. Despite intensive treatment efforts, patients have a median survival of just 15 months. One major treatment challenge is the tumor’s complex vascular network which causes the limited success of antiangiogenic therapies in extending patients’ survival. Pericytes are cells essential for maintaining the integrity of this vascular network. Additional to their structural functions, they play diverse roles that support tumor progression and treatment resistance. This review aims to understand the multifaced functions of pericytes in the glioblastoma microenvironment; the comprehension of those roles may guide new therapeutic strategies to improve outcomes for patients affected with this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/835c71d63c5567604bb678c8f9634fc408771476" target='_blank'>
              Pericytes in Glioblastoma: Hidden Regulators of Tumor Vasculature and Therapy Resistance
              </a>
            </td>
          <td>
            Irene Salazar-Saura, María Pinilla-Sala, J. Megías, Lara Navarro, Esther Roselló-Sastre, T. San-Miguel
          </td>
          <td>2024-12-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Mortality from cancer is almost exclusively due to metastasis to distant organs. Therefore, understanding the biology of tumor metastasis is the most significant challenge in cancer research today. The tumor microenvironment is central in supporting tumor growth. However, the role of the metastatic microenvironment in the multistage process of metastasis is largely unresolved. Since each metastatic microenvironment exerts specific functions that support or oppose colonization by disseminated tumor, understanding distinct organ-specific mechanisms that enable metastatic growth is of crucial importance. Utilizing mouse models of breast cancer metastasis to lungs, brain and bone, combined with clinical data from patients with metastatic disease, we uncover the mechanisms that underlie the early events of metastasis to provide the mechanistic basis for therapeutic targeting to reduce metastatic relapse. Our findings reveal organ-specific co-evolution of stromal and immune cells at distinct metastatic niches, that lead to the formation of an immunosuppressive microenvironment, permissive to metastatic growth. Analysis of human metastasis reveal that these pathways are also operative in human disease. We show that genetic and/or pharmacological targeting of specific nodes in the interactions between stromal and immune cells results in attenuated metastasis and improved survival. Our findings imply that combinatorial therapeutic strategies to target the crosstalk between stromal and immune cells at the metastatic microenvironment may be a promising treatment strategy to inhibit metastatic relapse.
 Citation Format: Neta Erez. Stromal and immune plasticity shape the metastatic microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr IA013.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c052feaff0e9ba6aedea07bd40ffbb3e7c16342f" target='_blank'>
              Abstract IA013: Stromal and immune plasticity shape the metastatic microenvironment
              </a>
            </td>
          <td>
            Neta Erez
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Lung adenocarcinoma (LUAD) is the leading cause of cancer-related mortality, largely due to its high prevalence rate and limited therapeutic options. Immunotherapy remains one of the most promising therapeutic avenues, yet further research is necessary to enhance therapy effectiveness for a broader range of patients. Central to the mechanism of immunotherapy is the detection of tumor neoantigens by cytotoxic T cells, which trigger the elimination of tumor cells. However, it is largely unknown how the early interaction between T cells and neoantigen-expressing tumor cells affects the acquisition of cancer hallmarks and tumor progression. To study this, our lab has developed the iNversion-INduced Joined neoAntigen (NINJA) genetically engineered mouse model in which we can initiate expression of a neoantigen in Kras-mutant, Trp53-null, tdTomato-expressing lung adenocarcinoma cells (KPT-NINJA). This system enables us to analyze tumor initiation, progression, and advanced LUAD development over 20 weeks in both neoantigen-expressing and non-expressing tumors. We defined cancer hallmark acquisition and tumor progression in our model by using a combination of multiplex immunofluorescence and computational algorithms to quantify tumor size, immune infiltration, metastasis, and expression of cancer molecular markers p-ERK and Ki67. We isolated tumor cell and immune cell subsets using neural networks to segment cells and tumor structures and used the colocalization of molecular markers to confirm the identity of various cell types. Our initial data validated that within immunogenic tumors there is an increase of T cell infiltration in tumors and have found immunogenic tumors are associated with overall smaller tumors at early time points. Interestingly, these tumors are associated with a decrease in p-ERK expression, suggesting immune infiltration is negatively correlated with p-ERK expression and tumor size. Together, these data indicate that increased immunogenic pressure may be disrupting early tumor proliferation and hallmark progression, shaping the trajectory of development at a very early stage. We are also correlating other molecular markers of tumor progression with immune infiltration, such as Ki67 expression, which we will colocalize with the other findings to construct a model of how tumors acquire hallmark capabilities from an immunogenic perspective. By isolating the mechanism of early tumor progression in immunogenic contexts, we can begin to predict disease trajectory. This will contribute to a better understanding of the mechanisms behind immune evasion and may uncover crucial insights to broaden immune therapy efficacy.
 Citation Format: Ishan Bansal, Jennifer Loza, Brian G. Hunt, Kelli Connolly, Srividhya Venkatesan, Nikhil Joshi. Hallmarks of progression in early neoantigen-expressing lung adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B013.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6754e5bab1083c1c562c8d6c1058076e47cfd9e" target='_blank'>
              Abstract B013: Hallmarks of progression in early neoantigen-expressing lung adenocarcinoma
              </a>
            </td>
          <td>
            Ishan Bansal, Jennifer L. Loza, Brian G. Hunt, Kelli A. Connolly, Srividhya Venkatesan, Nikhil S. Joshi
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Despite the pivotal role of cytotoxic T lymphocytes (CTLs) in anti-tumor immunity, a substantial proportion of CTL-rich hepatocellular carcinoma (HCC) patients experience early relapse or immunotherapy resistance. However, spatial immune variations impacting the heterogeneous clinical outcomes of CTL-rich HCCs remain poorly understood. Here, we compared the single-cell and spatial landscapes of 20 CTL-rich HCCs with distinct prognoses using multiplexed in situ staining and validated the prognostic value of myeloid spatial patterns in a cohort of 386 patients. Random forest and Cox regression models identified macrophage aggregation as a distinctive spatial pattern characterizing a subset of CTL-rich HCCs with an immunosuppressive microenvironment and poor prognosis. Integrated analysis of single-cell and spatial transcriptomics, combined with in situ staining validation, revealed that spatial aggregation enhanced pro-tumoral macrophage reprogramming in HCCs, marked by lipid metabolism orientation, M2-like polarization, and increased adjacent CTL exhaustion. This spatial effect on macrophage reprogramming was replicated in HCC-conditioned human macrophage cultures, which showed an enhanced capability to suppress CTLs. Notably, increased macrophage aggregation was associated with higher response rates to anti-PD-1 immunotherapy. These findings suggest that the spatial distribution of macrophages is a biomarker of their functional diversities and microenvironment status, which holds prognostic and therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0a3e561fbd74941c09e71f727074ba04c903f0f" target='_blank'>
              Spatial Organization of Macrophages in CTL Rich Hepatocellular Carcinoma Influences CTL Antitumor Activity.
              </a>
            </td>
          <td>
            Yulan Weng, Lu Wang, Yuting Wang, Junyu Xu, Xiaoli Fan, Shufeng Luo, Qiaomin Hua, Jing Xu, Gaoteng Liu, , Chang-An Zhao, Dong-Ming Kuang, Chong Wu, Limin Zheng
          </td>
          <td>2025-01-02</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286075cb8f68cf0d587bedcc36d60116c411594e" target='_blank'>
              Spatially resolved gene expression profiling of tumor microenvironment reveals key steps of lung adenocarcinoma development
              </a>
            </td>
          <td>
            Yuma Takano, Jun Suzuki, Kotaro Nomura, Gento Fujii, Junko Zenkoh, Hitomi Kawai, Yuta Kuze, Y. Kashima, Satoi Nagasawa, Yuka Nakamura, M. Kojima, K. Tsuchihara, Masahide Seki, A. Kanai, D. Matsubara, Takashi Kohno, Masayuki Noguchi, Akihiro Nakaya, Masahiro Tsuboi, G. Ishii, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-12-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Lineage plasticity and histologic transformation from prostate adenocarcinoma to neuroendocrine prostate cancer (NEPC) occurs in up to 15-20% of patients with castration-resistant prostate cancer (CRPC) as mechanism of treatment resistance and is associated with aggressive disease and poor prognosis. NEPC tumors typically display small cell carcinoma morphology with loss of androgen receptor (AR) expression and gain of neuroendocrine (NE) lineage markers. However, there is a spectrum of phenotypes that are observed during the lineage plasticity process, and the clinical significance of mixed histologies or those that co-express AR and NE markers or lack all markers is not well defined. Translational research studies investigating NEPC have used variable definitions making clinical trial design challenging. Here we discuss the diagnostic workup of metastatic biopsies to help guide the reproducible classification of phenotypic CRPC subtypes. We recommend classifying CRPC tumors based on histomorphology (adenocarcinoma, small cell carcinoma, poorly differentiated carcinoma, other morphologic variant, or mixed morphology) and immunohistochemical markers with a priority for AR, NKX3.1, INSM1, synaptophysin and cell proliferation based on Ki-67 positivity, with additional markers to be considered based on the clinical context. Ultimately, a unified workup of metastatic CRPC biopsies can improve clinical trial design and eventually practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ba90573d033c02c8e63a055556c5045dc3b6fc" target='_blank'>
              Framework for the Pathology Workup of Metastatic Castration-Resistant Prostate Cancer Biopsies.
              </a>
            </td>
          <td>
            Michael C Haffner, Michael J. Morris, C. C. Ding, Erolcan Sayar, Rohit Mehra, Brian D. Robinson, Lawrence D. True, Martin Gleave, Tamara L. Lotan, Rahul Aggarwal, Jiaoti Huang, Massimo Loda, Peter S Nelson, Mark A. Rubin, H. Beltran
          </td>
          <td>2024-11-26</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Though somatic mutations play a critical role in driving cancer initiation and progression, the systems-level functional impacts of these mutations—particularly, how they alter expression across the genome and give rise to cancer hallmarks—are not yet well-understood, even for well-studied cancer driver genes. To address this, we designed an integrative machine learning model, Dyscovr, that leverages mutation, gene expression, copy number alteration (CNA), methylation, and clinical data to uncover putative relationships between nonsynonymous mutations in key cancer driver genes and transcriptional changes across the genome. We applied Dyscovr pan-cancer and within 19 individual cancer types, finding both broadly relevant and cancer type-specific links between driver genes and putative targets, including a subset we further identify as exhibiting negative genetic relationships. Our work newly implicates–and validates in cell lines–KBTBD2 and mutant PIK3CA as putative synthetic lethals in breast cancer, suggesting a novel combinatorial treatment approach. HIGHLIGHTS Integrative framework Dyscovr links mutations within cancer drivers to downstream expression changes Dyscovr uncovers known and novel targets of cancer-driver genes Dyscovr reveals clinically important negative genetic interaction pairings Web platform to explore uncovered driver gene-target relationships eTOC BLURB An integrative computational framework, Dyscovr, links mutated cancer driver genes to expression changes in putative target genes within and across 19 TCGA cancer types. Dyscovr’s results include experimentally verifiable synthetic lethal driver-target pairings. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a48c5eb14f1890839d95663dc9da6929e284d0cf" target='_blank'>
              Integrative Computational Framework, Dyscovr, Links Mutated Driver Genes to Expression Dysregulation Across 19 Cancer Types
              </a>
            </td>
          <td>
            Sara Geraghty, Jacob A. Boyer, Mahya Fazel-Zarandi, Nibal Arzouni, Rolf-Peter Ryseck, Matthew J McBride, Lance R. Parsons, Joshua D. Rabinowitz, Mona Singh
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda6b8df14f4e01a65f3a2aa9bb495dcc561cdf0" target='_blank'>
              Transcript-specific enrichment enables profiling of rare cell states via single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Tsion Abay, Robert R. Stickels, Meril T. Takizawa, Benan N. Nalbant, Yu-Hsin Hsieh, Sidney Hwang, Catherine Snopkowski, K. Yu, Z. Abou-Mrad, Viviane Tabar, B. Howitt, Leif S. Ludwig, R. Chaligné, Ansuman T. Satpathy, Caleb A. Lareau
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/602ca4326619720e872bb3125e86fd5844dc2d44" target='_blank'>
              A multiscale model of immune surveillance in micrometastases gives insights on cancer patient digital twins
              </a>
            </td>
          <td>
            Heber L Rocha, Boris Aguilar, Michael Getz, I. Shmulevich, P. Macklin
          </td>
          <td>2024-12-04</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>2</td>
          <td>83</td>
        </tr>

        <tr id="
 Although various effective anti-cancer treatments have become available over the last decades, resistance to all available therapies remains the major cause of death of cancer patients with disseminated tumors. Striking examples are patients with triple-negative breast cancer (TNBC), which are frequently defective in the repair of DNA double strand breaks, e.g. due to loss of BRCA1 function. Because of this defect, the patients initially respond very well to DNA damage-inducing chemotherapy. Unfortunately, disseminated tumors are usually not eradicated and resistant tumor cells are eventually selected from residual primary or metastatic tumor sites. It is therefore crucial to understand the molecular mechanisms underlying the drug tolerance of the residual tumor cells. To study residual disease, we used the K14cre;Brca1 F/F ;p53 F/F (KB1P) mouse model for hereditary breast cancer, which provides the unique opportunity to explore and target those mechanisms in a fully immunocompetent model. The mammary tumors that spontaneously develop highly resemble their human counterparts, both morphologically and in their therapy response. For example, tumors shrink in response to poly(ADP-ribose) polymerase inhibition, platinum-based treatment or the commonly used doxorubicin, docetaxel and cyclophosphamide (TAC) combination therapy. But despite repeated drug sensitivity, the KB1P mammary tumors are not eradicated, not even by a frequent dosing schedule. By combining single-cell RNA sequencing, spatial transcriptomics and imaging mass cytometry, we dissected the intratumoral heterogeneity and alterations in the tumor microenvironment of residual disease. We identified specific subpopulations of tumor cells that have a survival benefit after treatment, and these occurred together with an altered microenvironment characterized by a highly reactive stroma infiltrated with both anti-inflammatory and pro-tumoral immune cells. Interestingly, those structural and transcriptional changes are reversed in the relapsed tumors, highlighting the plasticity of drug tolerance. To investigate the mechanisms of the altered tumor-stroma interactions that are relevant for drug tolerance, we have designed a custom-made CRISPR/Cas9 library based on differential gene expression of the residual tumor cells. This library enables us to functionally test relevant mechanisms of drug tolerance in vivo and we expect that these analyses will provide useful insights into the tumor-stroma interactions that contribute to residual disease. Taken together, our detailed analyses demonstrate the substantial remodeling of tumor cells in their microenvironment upon treatment. To develop new therapeutic approaches to eradicate drug-tolerant tumor cells and thereby circumvent tumor relapse, it is essential to better understand the heterogeneous cellular composition of residual tumors in its spatial context. With this project, we hope to provide comprehensive data to develop better therapeutic strategies that target the tumor-stroma interaction and eradicate residual tumors.
 Citation Format: Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, Ismar Klebic, Sven Rottenberg. Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecf546e6bc5a8fa717c4c87fcb379e8ed3399d7" target='_blank'>
              Abstract C034: Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors
              </a>
            </td>
          <td>
            Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, I. Klebic, Sven Rottenberg
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive subtype of breast cancer, with the worst prognosis among all subtypes. The impact of distinct cell subpopulations within the tumor microenvironment (TME) on TNBC patient prognosis has yet to be clarified. Methods Utilizing single-cell RNA sequencing (scRNA-seq) integrated with bulk RNA sequencing (bulk RNA-seq), we applied Cox regression models to compute hazard ratios, and cross-validated prognostic scoring using a GLMNET-based Cox model. Cell communication analysis was used to elucidate the potential mechanisms of CPVL and MSR1. Ultimately, RNA interference-mediated gene knockdown was utilized to validate the impact of specific genes on the polarization of tumor-associated macrophages (TAMs). Results Our findings revealed that the function of immune cells is more pivotal in prognosis, with TAMs showing the strongest correlation with TNBC patient outcomes, compared with other immune cells. Additionally, we identified CPVL and MSR1 as critical prognostic genes within TAMs, with CPVL expression positively correlated with favorable outcomes and MSR1 expression associated with poorer prognosis. Mechanistically, CPVL may contribute to favorable prognosis by inhibiting the SPP1-CD44 ligand-receptor and promoting CXCL9-CXCR3, C3-C3AR1 ligand-receptor, through which TAMs interact with other cells such as monocytes, neutrophils, and T cells. Moreover, cytokines including IL-18, IFNγR1, CCL20, and CCL2, along with complement-related gene like TREM2 and complement component CFD, may participate in the process of CPVL or MSR1 regulating macrophage polarization. Furthermore, RT-PCR experiments confirmed that CPVL is positively associated with M1-like TAM polarization, while MSR1 is linked to M2-like TAM polarization. Finally, the prognostic significance of these two genes is also validated in HER2-positive breast cancer subtypes. Conclusions CPVL and MSR1 are potential biomarkers for macrophage-mediated TNBC prognosis, suggesting the therapeutic potential of macrophage targeting in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af61ce193245e3b400806d4fcea4eed6dfe425f5" target='_blank'>
              Single-cell Atlas reveals core function of CPVL/MSR1 expressing macrophages in the prognosis of triple-negative breast cancer
              </a>
            </td>
          <td>
            Xinan Wang, Li Lin, Xue Zhang, Minghui Zhang, Zhuo Sun, Yichen Yang, Xiuna Zhang, Yonghui Yuan, Yong Zhang, Hao Chen, Ti Wen
          </td>
          <td>2024-12-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
A spatially resolved, niche-level analysis of tumour microenvironments (TME) can provide insights into cellular interactions and their functional impacts in gastric cancers (GC).


OBJECTIVE
Our goal was to translate the spatial organisation of GC ecosystems into a functional landscape of cellular interactions involving malignant, stromal and immune cells.


DESIGN
We performed spatial transcriptomics on nine primary GC samples using the Visium platform to delineate the transcriptional landscape and dynamics of malignant, stromal and immune cells within the GC tissue architecture, highlighting cellular crosstalks and their functional consequences in the TME.


RESULTS
GC spatial transcriptomes with substantial cellular heterogeneity were delineated into six regional compartments. Specifically, the fibroblast-enriched TME upregulates epithelial-to-mesenchymal transformation and immunosuppressive response in malignant and TME cells, respectively. Cell type-specific transcriptional dynamics revealed that malignant and endothelial cells promote the cellular proliferations of TME cells, whereas the fibroblasts and immune cells are associated with procancer and anticancer immunity, respectively. Ligand-receptor analysis revealed that CCL2-expressing fibroblasts promote the tumour progression via JAK-STAT3 signalling and inflammatory response in tumour-infiltrated macrophages. CCL2+ fibroblasts and STAT3-activated macrophages are co-localised and their co-abundance was associated with unfavourable prognosis. We experimentally validated that CCL2+ fibroblasts recruit myeloid cells and stimulate STAT3 activation in recruited macrophages. The development of immunosuppressive TME by CCL2+ fibroblasts were also validated in syngeneic mouse models.


CONCLUSION
GC spatial transcriptomes revealed functional cellular crosstalk involving multiple cell types among which the interaction between CCL2+ fibroblasts and STAT3-activated macrophages plays roles in establishing immune-suppressive GC TME with potential clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe72f4555eff64ae7e607a98dd2b2dd8cb471c6e" target='_blank'>
              Spatial dissection of tumour microenvironments in gastric cancers reveals the immunosuppressive crosstalk between CCL2+ fibroblasts and STAT3-activated macrophages.
              </a>
            </td>
          <td>
            Sung Hak Lee, Dagyeong Lee, Junyong Choi, Hye Jeong Oh, In-Hye Ham, Daeun Ryu, Seo-Yeong Lee, Dong-Jin Han, Sunmin Kim, Youngbeen Moon, In-Hye Song, K. Song, Hyeseong Lee, Seungho Lee, Hoon Hur, Tae-Min Kim
          </td>
          <td>2024-11-23</td>
          <td>Gut</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Cancer arises through an evolutionary process in which somatic mutations, including single nucleotide variants (SNVs) and copy number aberrations (CNAs), drive the development of a malignant, heterogeneous tumor. Reconstructing this evolutionary history from sequencing data is critical for understanding the order in which mutations are acquired and the dynamic interplay between different types of alterations. Advances in modern whole genome single-cell sequencing now enable the accurate inference of copy number profiles in individual cells. However, the low sequencing coverage of these low pass sequencing technologies poses a challenge for reliably inferring the presence or absence of SNVs within tumor cells, limiting the ability to simultaneously study the evolutionary relationships between SNVs and CNAs. In this work, we introduce a novel tumor phylogeny inference method, Pharming, that jointly infers the evolutionary histories of SNVs and CNAs. Our key insight is to leverage the high accuracy of copy number inference methods and the fact that SNVs co-occur in regions with CNAs in order to enable more precise tumor phylogeny reconstruction for both alteration types. We demonstrate via simulations that Pharming outperforms state-of-the-art single-modality tumor phylogeny inference methods. Additionally, we apply Pharming to a triple-negative breast cancer case, achieving high-resolution, joint reconstruction of CNA and SNV evolution, including the de novo detection of a clonal whole-genome duplication event. Thus, Pharming offers the potential for more comprehensive and detailed tumor phylogeny inference for high-throughput, low-coverage single-cell DNA sequencing technologies compared to existing approaches. Availability https://github.com/elkebir-group/Pharming">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d64f3b951403db46ae5a215e58c613919dd9b93c" target='_blank'>
              Pharming: Joint Clonal Tree Reconstruction of SNV and CNA Evolution from Single-cell DNA Sequencing of Tumors
              </a>
            </td>
          <td>
            Leah L. Weber, Anna Hart, Idoia Ochoa, M. El-Kebir
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72ace1bf5ccc7bc1a835b1fd333db7f3f67e0b96" target='_blank'>
              Lessons learned from spatial transcriptomic analyses in clear-cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Jesper Jespersen, Cecilie Lindgaard, Laura Iisager, J. Ahrenfeldt, I. Lyskjær
          </td>
          <td>2025-01-09</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Introduction: Tumors systemically remodel the immune system during metastasis. Developing anti-metastatic immunotherapies that target these tumor-immune interactions to make the metastatic niche hostile colonization would represent a paradigm shift in cancer treatment. However, traditional high-throughput screening (HTS) platforms that use simple read-outs and contrived in vitro systems are ill-suited to identify such therapies. In contrast, HTS platforms that accurately model the metastatic niche and use high-dimensional read-outs such as multiplexed single-cell RNA-sequencing (scRNA-seq) can accurately profile nuanced perturbation responses in individual immune cell types and therefore have the potential to discover anti-metastatic immunotherapies. In this study, we describe the development of an ex vivo lung tissue culture HTS platform that preserves immune gene expression signatures observed in the in vivo metastatic niche and is amenable to single-cell chemical transcriptomic analysis using MULTI- seq (McGinnis et al., Nature Methods, 2019). Inspired by our recent description of myeloid cell TLR-NFκB inflammation during breast cancer lung metastasis (McGinnis et al., Cancer Cell, 2024), we then performed the first anti-metastatic immunotherapy drug screen and identified TLR-NFκB inhibitors that effectively operate on all desired myeloid cell types in the lung metastatic niche. Methods and Results: Building upon existing techniques for culturing precision-cut lung slices (Wu et al., Journal of Experimental Medicine, 2024) and patient- derived tumor fragments (Voabil et al., Nature Medicine, 2021), we established and optimized an ex vivo lung tissue culture system that enables 48-hour culturing of lung slices isolated from 4T1 tumor-bearing mice. scRNA-seq profiling revealed successful capture of all relevant immune cell types and retention of metastasis-associated gene expression programs (e.g., myeloid TLR-NFκB inflammation and neutrophil degranulation) following culture. We then scaled our platform to address how myeloid cells in the lung metastatic niche respond to perturbations targeting every component of the canonical TLR-NFκB signaling cascade. Cell-type-specific perturbation modeling and quantification of TLR-NFκB signaling inhibition identified compounds that optimally perturb all intended myeloid cell types (e.g., tissue-resident macrophages, neutrophils, and monocytes) and represent prioritized candidates for future in vivo validation studies to assess impact on metastatic disease progression. Conclusions: In this study, we describe the development and application of a novel HTS platform that couples ex vivo lung tissue cultures with single-cell chemical transcriptomics to identify anti-metastatic immunotherapy candidates. We demonstrate how our platform can identify TLR-NFκB signaling inhibitors that optimally operate in the lung metastatic niche, paving the way for future interrogation of additional metastasis-associated immune cell signaling pathways in different disease and tissue backgrounds.
 Citation Format: Chris McGinnis, Winnie Yao, Ansuman Satpathy. Anti-metastatic immunotherapy discovery using ex vivo lung tissue cultures and high-throughput single-cell chemical transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr PR017.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abfcae45218f099638f6a7c32802b5781110ec64" target='_blank'>
              Abstract PR017: Anti-metastatic immunotherapy discovery using ex vivo lung tissue cultures and high-throughput single-cell chemical transcriptomics
              </a>
            </td>
          <td>
            Chris McGinnis, Winnie Yao, Ansuman T. Satpathy
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Dendritic cells (DCs) are promising targets for cancer immunotherapies because of their central role in the initiation and control of immune responses. The rare cDC1 population is of particular interest because of its remarkable ability to cross-present antigens (Ag) to CD8+ T cells, to promote Th1 cell polarization and NK cell activation and recruitment. However, the spatial organization and specific functions of cDC1s in response to immunotherapy remain to be clearly characterized in human tumors. Here, we implemented a multiplexed immunofluorescence analysis pipeline coupled with computational image analysis to determine the spatial organization of the cDC1 subset in a cohort of skin lesions from advanced melanoma patients treated with immune checkpoint inhibitors (ICI). For this, we performed a whole-slide image analysis of cDC1 infiltration and distribution as well as their spatial interactions with key immune partners such as CD8+ T cells and pDC according to the response of patients to ICI. We also analyzed LAMP3+-DC, which correspond to a mature subset of tumor-infiltrating DCs. Distance and cell network analyses demonstrated that cDC1s exhibited a scattered distribution compared to tumor-infiltrating pDCs and LAMP3+-DCs, which were preferentially organized in dense areas with high homotypic connections. Interestingly, the proximity and interactions between CD8+ T cells and cDC1s were positively associated with the response to ICI. In conclusion, our study unravels the complex spatial organization of cDC1s and their interactions with CD8+ T cells in melanoma patient lesions, shedding light on their pivotal role in shaping the response to ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f10d121483bbab07523097b63187cdc94830056d" target='_blank'>
              The spatial organization of cDC1 with CD8+ T cells is critical for the response to immune checkpoint inhibitors in melanoma patients.
              </a>
            </td>
          <td>
            Elisa Gobbini, Margaux Hubert, A. Doffin, Anaïs Eberhardt, Léo Hermet, Danlin Li, Pierre Duplouye, Sarah Ghamry-Barrin, Justine Berthet, Valentin Benboubker, Maxime Grimont, C. Sakref, Jimmy Perrot, Garance Tondeur, Olivier Harou, Jonathan Lopez, Bertrand Dubois, Stéphane Dalle, Christophe Caux, J. Caramel, J. Valladeau-Guilemond
          </td>
          <td>2025-01-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Lung cancer is the leading cause of cancer mortality, and despite improvements in treatment, tumors typically respond incompletely and resume growth after acquisition of drug resistance. Recent completion of a neoadjuvant osimertinib Phase II trial treating patients with surgically resectable stage I-IIIA EGFR-mutated non-small cell lung cancer (EGFRm NSCLC) (NCT03433469) (Blakely et. Al, JCO 2024) has highlighted the importance of further identifying non-genomic mechanisms of persistence and resistance to targeted therapy. Here we analyzed these patient samples to identify transcriptionally regulated signaling patterns enriched at Residual Disease (RD) after neoadjuvant osimertinib treatment compared to treatment naïve samples (TN), as well as signaling patterns enriched at RD in patients who ultimately develop disease recurrence after surgical resection compared to those who remain disease-free. Spatially resolved transcriptomic sequencing was performed, using the 10X Genomics Visium platform, on 36 tissue sections (n=9 TN, n=19 RD, n=4 Progressive Disease (PD), n=4 Tumor Adjacent Normal), from 25 patients. TN and PD samples were from standard-of-care surgical resections. 18 of 19 RD samples were from NCT03433469. The Visium array spots that tile the sequencing capture area are 55 µm in diameter, and thus can contain an estimated 1-10 cells within each array spot. After quality control, 91,582 array spots remained for downstream analysis. We identified high confidence tumor array spots annotated both as “Cancer” by a board-certified pathologist and with tumor characteristic copy number variations. Through dimensionality reduction clustering, we identified 10 major array spot clusters, including the high confidence tumor array spot cluster. These clusters were further subset into 44 high resolution clusters of array spot niches, 32 of which were shared across at least nine patients. Because array spots are not single-cell resolution, these niches represent spots with similar mixed cell type composition and transcriptional signatures. We identified significant increases in the CCL14+ Endothelial Cell Niche, CXCL2+ Endothelial Cell Niche, Lymphocytes + Endothelial Niche, and CXCL12+ Fibroblast Niche in RD samples compared to TN. We identified a cell signaling program defined by spatially co-expressed ligand-receptor pairs associated with focal adhesion, enriched in PD samples. This same pattern was enriched in RD samples from patients who had recurrent disease after surgical resection compared to RD samples from patients who remained disease free. These data describe an increase in immune trafficking in RD samples, as well as identify an EGFR- alternative signaling pathway correlating with osimertinib resistance. Identification of spatially resolved, differential architectural organization and signaling patterns in RD and PD states will expand our understanding of targeted therapy resistance in EGFRm NSCLC, with the potential to identify additional clinically actionable targets.
 Citation Format: Whitney Tamaki, Daniel L. Kerr, Wei Wu, Grant Eilers, Anatoly Urisman, Yu-Ting Chou, Philippe Gui, Shigeki Nanjo, Johannes R. Kratz, David M. Jablons, Trever G. Bivona, Collin M. Blakely. Identification of cell-cell signaling programs across the tumor-tumor microenvironment ecosystem and associated with clinical status in neoadjuvant osimertinib treated patient samples revealed by spatial profiling [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00831e7253b0b6e70e43b596e9e7ff252fbadae" target='_blank'>
              Abstract C025: Identification of cell-cell signaling programs across the tumor-tumor microenvironment ecosystem and associated with clinical status in neoadjuvant osimertinib treated patient samples revealed by spatial profiling
              </a>
            </td>
          <td>
            W. Tamaki, D. Kerr, Wei Wu, Grant Eilers, A. Urisman, Yu-Ting Chou, Philippe Gui, Shigeki Nanjo, J. Kratz, D. Jablons, T. Bivona, Collin M Blakely
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80cae72455d0f13cb986669bd3a06f266bb4aab" target='_blank'>
              Spatial transcriptomics reveals substantial heterogeneity in triple-negative breast cancer with potential clinical implications
              </a>
            </td>
          <td>
            Xiaoxiao Wang, D. Venet, F. Lifrange, D. Larsimont, Mattia Rediti, L. Stenbeck, F. Dupont, G. Rouas, Andrea Joaquin Garcia, L. Craciun, L. Buisseret, Michail Ignatiadis, M. Carausu, N. Bhalla, Yuvarani Masarapu, Eva Gracia Villacampa, L. Franzén, Sami Saarenpää, L. Kvastad, K. Thrane, J. Lundeberg, F. Rothé, Christos Sotiriou
          </td>
          <td>2024-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Abstract Background Perioperative chemotherapy is the standard of care for patients with locally advanced gastric and gastroesophageal junction cancer. Recent evidence demonstrated the addition of programmed cell death protein 1 (PD-1) inhibitors enhanced therapeutic efficacy. However, the mechanisms of response and resistance remain largely undefined. A detailed multiomic investigation is essential to elucidate these mechanisms. Methods We performed whole-exome sequencing, whole-transcriptome sequencing, multiplex immunofluorescence and single-cell RNA sequencing on matched pretreatment and post-treatment samples from 30 patients enrolled in an investigator-initiated Phase 2 clinical trial (NCT04908566). All patients received neoadjuvant PD-1 inhibitors in combination with chemotherapy. A major pathologic response (MPR) was defined as the presence of no more than 10% residual viable tumor cells following treatment. Results Before treatment, the positive ratio of CD3+T cells in both the tumor parenchyma and stroma was significantly higher in the non-MPR group compared with the MPR group (p=0.042 and p=0.013, respectively). Least absolute shrinkage and selection operator regression was employed for feature gene selection and 13 genes were ultimately used to construct a predictive model for identifying MPR after surgery. The model exhibited a perfect area under curve (AUC) of 1.000 (95% CI: 1.000 to 1.000, p<0.001). Post-treatment analysis revealed a significant increase in CD3+T cells, CD8+T cells and NK cells in the tumor stroma of MPR patients. In the tumor parenchyma, aside from a marked increase in CD8+T cells and NK cells, a notable reduction in macrophage was also observed (all p<0.05). Importantly, forkheadbox protein 3 (FOXP3), the principal marker for regulatory T cells (Treg) cells, showed a significant decrease during treatment in MPR patients. FOXP3 expression in the non-MPR group was significantly higher than in the MPR group (p=0.0056) after treatment. Furthermore, single-cell RNA sequencing analysis confirmed that nearly all Treg cells were derived from the non-MPR group. Conclusions Our study highlights the critical role of dynamic changes within the tumor immune microenvironment in predicting the efficacy of neoadjuvant combined immunochemotherapy. We examined the disparities between MPR/non-MPR groups, shedding light on potential mechanisms of immune response and suppression. In addition to bolstering cytotoxic immune responses, specifically targeting Treg cells may be crucial for enhancing treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd7e41ce76cedb5c64fc740c519b9826d4d1968" target='_blank'>
              Multiomics reveals tumor microenvironment remodeling in locally advanced gastric and gastroesophageal junction cancer following neoadjuvant immunotherapy and chemotherapy
              </a>
            </td>
          <td>
            Z. Ji, Xia Wang, Jiaqi Xin, Lijun Ma, Duo Zuo, Hong-li Li, Lan Su, Xinze Lv, S. Ge, Le Zhang, Yong Liu, Yanhui Zhang, Tingting Ding, Ting Deng, Y. Ba, Rui Liu
          </td>
          <td>2024-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Most malignant hepatocellular tumors in children are classified as either hepatoblastoma (HB) or hepatocellular carcinoma (HCC), but some tumors demonstrate features of both HB and HCC1–3. These tumors have been recognized under a provisional diagnostic category by the World Health Organization and are distinguished from HB and HCC by a combination of histological, immunohistochemical, and molecular features4–6. Their outcomes and cellular composition remain an open question7–9. The heterogeneous histological and molecular profiles of hepatoblastomas with carcinoma features (HBCs)4 may result from cells with combined HB and HCC characteristics (HBC cells) or from mixtures of cells displaying either HB or HCC signatures. We used multiomics profiling to show that HBCs are mixtures of HB, HBC, and HCC cell types. HBC cells are more chemoresistant than HB cells, and their chemoresistance—a driver of poor outcomes10–12—is determined by their cell types, genetic alterations, and embryonic differentiation stages. We showed that the prognosis of HBCs is significantly worse than that of HBs. We also showed that HBC cells are derived from HB cells at early hepatoblast differentiation stages, that aberrant activation of WNT-signaling initiates HBC transformation, and that WNT inhibition promotes differentiation and increases sensitivity to chemotherapy. Furthermore, our analysis revealed that each HBC is the product of multiple HB-to-HBC and HBC-to-HCC transitions. Thus, multiomics profiling of HBCs provided key insights into their biology and resolved major questions regarding the etiology of these childhood liver tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d21a6337dd63078471fed4a0a2ddbb3e3a1500b" target='_blank'>
              Asynchronous Transitions from Hepatoblastoma to Carcinoma in High-Risk Pediatric Tumors
              </a>
            </td>
          <td>
            Xinjian Yu, S. Sarabia, Martin Urbicain, Sonal Somvanshi, R. Patel, Tuan M Tran, Yen-Ping Yeh, Keng-Shih Chang, Yi-Tzu Lo, Jessica Epps, Kathleen A. Scorsone, Hua-Sheng Chiu, E. F. Hollingsworth, Cintia R. Perez, M. J. Najaf Panah, B. Zorman, Milton Finegold, John A. Goss, R. Alaggio, Angshumoy Roy, Kevin E. Fisher, Andras Heczey, Sarah E. Woodfield, Sanjeev A. Vasudevan, Kalyani A. Patel, Ting-Wen Chen, Dolores Lopez-Terrada, Pavel Sumazin
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Precision medicine for paediatric and adult cancers that includes drug sensitivity profiling, can identify effective therapies for individual patients. However, obtaining adequate biopsy samples for high-throughput (HTP) screening remains challenging, with tumours needing to be expanded in culture or patient-derived xenografts – this is time-consuming and often unsuccessful. Herein, we have developed paediatric patient-derived tumour models using an engineered extracellular matrix (ECM) tissue mimic hydrogel system and HTP 3D bioprinting. Gene expression analysis from neuroblastoma and sarcoma patients identified key components of the ECM in these tumour types. Engineered hydrogels with ECM-mimic peptides were used to create patient-specific tumour organoids, modelling tumour growth conditions. Expanded tumour organoids recapitulated the genetic and phenotypic characteristics of the original tumours and retained tumourgenicity. Screening of these models identified individualised drug sensitivities. Our approach offers a timely and clinically relevant technology platform for precision medicine in paediatric cancers, potentially transforming preclinical testing across cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea0198b891b0eff96e2184508189a30fce8af925" target='_blank'>
              Engineered paediatric tumours retain maintains tumour genotype and phenotype for precision medicine
              </a>
            </td>
          <td>
            MoonSun Jung, Valentina Poltavets, Joanna Skhinas, G. Tax, Alvin Kamili, Angela (Jinhan) Xie, Sarah Ghamrawi, Philipp Graber, Jie Mao, Marie Wong-Erasmus, Louise Cui, Kathleen Kimpton, Pooja Venkat, Chelsea Mayoh, Emmy D G Fleuren, Ashleigh M. Fordham, Zara A Barger, John Grady, David M. Thomas, Eric Y Du, Mark J. Cowley, Andrew J. Gifford, Jamie I. Fletcher, Loretta M S Lau, M. E. M. Dolman, J. J. Gooding, Maria Kavallaris
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Introduction The extracellular matrix (ECM) supports tumor progression by influencing tumor cell migration and invasion. This study examines the link between peritumoral ECM morphology and five-year recurrence risk in TNM Stage II colon cancer, using quantitative whole-slide ECM imaging. We hypothesize that loose ECM regions are associated with increased recurrence risk due to enhanced tumor budding (TB) or poorly differentiated clusters (PDC). Methods In a case-control study of 100 TNM Stage II colon cancer patients (25 with recurrence and 75 controls matched by lymph node sampling and tumor extent), Picrosirius red-stained sections were imaged to quantify ten ECM parameters across 798 regions of interest (ROIs). Conditional logistic regression assessed associations between ECM morphologies, TB, PDCs, and recurrence. Results Unsupervised clustering identified three ECM morphologies: dense fibrous, loose sparse, and complex tortuous. Dense fibrous ECM correlated strongly with recurrence (aOR 9.43, 95% CI 3.29-29.30, p < 0.001), while loose sparse and complex tortuous ECMs were associated with reduced recurrence risk (aOR 0.33, 95% CI 0.11-0.91, p = 0.040, and aOR 0.14, 95% CI 0.02-0.53, p = 0.012, respectively). TB was highest in loose sparse ECM (mean 9.3), and PDCs were highest in dense fibrous ECM (mean 5.5). Discussion Our findings suggest that ECM morphology, particularly dense fibrous ECM, predicts recurrence in Stage II colon cancer, highlighting ECM profiling as a promising tool for patient stratification beyond traditional staging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10cd804b89bbca0cdd6f6ca0c5c57995b707a846" target='_blank'>
              Whole-Slide ECM Imaging Reveals Dense Fibrous Matrix as a High-Risk Factor for Recurrence in Stage II Colon Cancer
              </a>
            </td>
          <td>
            C. Ravensbergen, V.S. Colaco, H. Putter, A. Crobach, J. Boonstra, J.H.N. Lindeman, R. Tollenaar, W. Mesker
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Significance Our study provides functional evidence that metabolic reprogramming can directly impact cancer phenotypes and establishes proliferator-activated receptor gamma coactivator 1-alpha (PGC-1α)-induced mitochondrial metabolism as a driver of small cell neuroendocrine cancer (SCNC) progression and lineage determination. These mechanistic insights reveal common metabolic vulnerabilities across SCNCs originating from multiple tissues, opening additional avenues for pan-SCN cancer therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a586b3dcd3d3eae51970eccbac4cd9bc30b11e95" target='_blank'>
              PGC-1α drives small cell neuroendocrine cancer progression toward an ASCL1-expressing subtype with increased mitochondrial capacity
              </a>
            </td>
          <td>
            Grigor Varuzhanyan, Chia-Chun Chen, Jack Freeland, Tian He, Wendy Tran, Kai Song, Liang Wang, Donghui Cheng, Shili Xu, Gabriella A. Dibernardo, Favour N Esedebe, Vipul Bhatia, Mingqi Han, Evan R. Abt, Jung Wook Park, Sanaz Memarzadeh, David B. Shackelford, John K. Lee, Thomas G Graeber, O. Shirihai, Owen N. Witte
          </td>
          <td>2024-11-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="
 Introduction: Properties of tumor angiogenesis impact on tumor growth and composition of the tumor microenvironment. Tumor vessels are composed of pericytes and other mural cells. Detailed characterization of the cellular composition of the microvasculature can suggest functional vessel and case properties relevant for outcome, response to treatment and immune surveillance. Results: Based on single cell RNA-seq from three human stage II/III colon cancers and pilot in-situ multi-antibody staining of six tumors two main clusters of perivascular cells were identified; B1 (MCAM+, MYH11-) and B2 (MCAM+, MYH11+) cells. B1 cells displayed pericyte features including expression of RGS5, whereas B2 cells showed high expression of smooth muscle cell markers like ACTG2 and Desmin. Desmin-high B2 cells were predominantly located in muscular layers, whereas Desmin-low/MYH11-high B2 cells showed perivascular enrichment. An extended novel analyses workflow of digital image analysis was used to profile peri-endothelial composition in 19 intervals; 11 1-µm intervals from 0 to 10 µm and 8 5-µm intervals from 10 to 50 µm from endothelial areas. Each of these expansion areas was quantitated regarding density of PDGFRB+/- cells including B1/B2 cells, and a MYH11+/MCAM- cells. Density of macrophage subsets (CD68-positive subsets defined by CD163 and CD11c) were also quantitated. In general, B1 cells showed stronger spatial association with endothelial cells than B2 cells, and this spatial association was stronger for the PDGFRB+ B1 subset. Survival analysis indicated that prognostic relevance was strongest when PDGFRB+ (favorable prognosis) cells were closest to endothelial cells. Regarding perivascular macrophage density, M0 and transit M1/M2 macrophages were predominantly enriched in the peri-endothelial regions, whereas density of M1 and M2 increased gradually as distance from vessels increased. M2 density was overall associated with poor prognosis, whereas high density of other macrophage subsets overall was associated with good prognosis. Ongoing studies are using a clustering-based approach to identify vessel subsets, and prognosis associations are being consolidated in additional cohorts. Summary: This study provides a novel approach for high resolution profiling of perivascular status in CRC, and possibly other tumor types, that could be of specific importance to identify tumor features associated with response to angiogenesis-directed therapies.
 Citation Format: Linglong( 凌 龙 ) Huang( 黄 ), Mercedes Herrera, Jonas Sjölund, Vladimir Chocoloff, Simon Joost, Rasul M. Tabiev, Lina Wik Leiss, Carina Strell, Luis Nunes, Artur Mezheyeuski, David Edler, Anna Martling, Fredrik Pontén, Bengt Glimelius, Tobias Sjöblom, Maria Kasper, Kristian Pietras, Arne Östman. Mapping the vascular walls of colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B029.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a41b99632bcf5e6a6bbd94c1d54966c1ecf742fe" target='_blank'>
              Abstract B029: Mapping the vascular walls of colorectal cancer
              </a>
            </td>
          <td>
            Linglong(凌龙) Huang( 黄 ), Mercedes Herrera, Jonas Sjölund, Vladimir Chocoloff, S. Joost, Rasul M. Tabiev, Lina Wik Leiss, Carina Strell, Luis Nunes, A. Mezheyeuski, D. Edler, A. Martling, Fredrik Pontén, B. Glimelius, T. Sjöblom, Maria Kasper, Kristian Pietras, Arne Östman
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eeb1135b7c75ad3978d67f0a6864e002ffcff1b" target='_blank'>
              Comprehensive single-cell aging atlas of healthy mammary tissues reveals shared epigenomic and transcriptomic signatures of aging and cancer.
              </a>
            </td>
          <td>
            B. Angarola, Siddhartha Sharma, Neerja Katiyar, Hyeon Gu Kang, Djamel Nehar-Belaid, SungHee Park, Rachel Gott, Giray N Eryilmaz, Mark A. LaBarge, Karolina Palucka, Jeffrey H. Chuang, Ron Korstanje, D. Ucar, Olga Anczukόw
          </td>
          <td>2024-11-25</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Despite promising results with immune checkpoint blockade (ICB) therapy, brain metastases (BrM) remain a deadly complication for cancer patients. Understanding the highly specialized brain metastatic tumor microenvironment (BrTME) is crucial for identifying determinants of response, however, progress has been hindered by limited access to patient samples and scarcity of relevant preclinical models. Here, we developed two mouse melanoma BrM models by intracardiac injection of cells derived from our previously established M4-B2905 melanoma mouse cell line. We characterized their mutational landscape and performed single-cell phenotypic and transcriptomic analysis, demonstrating that these models recapitulate the cellular and molecular features of human melanoma BrMs. Comparative and interactome analysis of our models revealed key factors contributing to ICB response and resistance. We found that the responsive model (BR1) was characterized by tumor cells that polarize microglia toward reactive states via inflammatory programs and response to IFN signaling. These microglia express high levels of antigen presentation and immunostimulatory molecules, triggering T cell recruitment and activation and promoting ICB therapy response. In contrast, in the resistant model (BR3), the tumor cells express neurological molecular signatures and ligands that sustain microglia homeostasis. This results in poor T cell infiltration predominantly composed of naïve cells, evading immune reaction and promoting ICB resistance. While we showed that systemic ICB therapy induced BrTME changes in both models, differences between responder and resistant models were maintained, emphasizing the importance of tumor cell-intrinsic programs in dictating the BrTME. We validated the translational relevance of our findings and demonstrated that BR1 signatures positively correlate with T cell infiltration and are associated with improved patient outcomes, whereas BR3 signatures negatively correlate with T cells and are found in patients with worse prognosis. Here we address an important challenge in the field by providing clinically relevant BrM models that recapitulate both the cellular and molecular features of human disease and the variability of ICB response seen in patients. We further reveal mechanistic insights into BrM ICB response and identify potential therapeutic targets to modulate the BrTME.
 Citation Format: Amelie Daugherty-Lopes, Eva Perez-Guijarro, Vishaka Gopalan, Jessica Rappaport, Quanyi Chen, April Huang, Khiem C. Lam, Sung Chin, Jessica Ebersole, Emily Wu, Gabriel Needle, Isabella Church, George Kyriakopoulos, Shaojun Xie, Yongmei Zaho, Charli Gruen, Antonella Sassano, Romina E Araya, Andres Thorkelsson, Cari Smith, Maxwell P. Lee, Sridhar Hannenhalli, Chi-Ping Day, Glenn Merlino, Romina S. Goldszmid, Shaojun Xie. Dissecting the brain metastatic microenvironment to uncover immune and molecular correlates of response to immunotherapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83bdd8b5e937692077f180146aa529c30245669c" target='_blank'>
              Abstract A002: Dissecting the brain metastatic microenvironment to uncover immune and molecular correlates of response to immunotherapy
              </a>
            </td>
          <td>
            Amélie Daugherty-Lopès, Eva Pérez-Guijarro, Vishaka Gopalan, Jessica Rappaport, Quanyi Chen, April Huang, Khiem C. Lam, Sung Chin, Jessica Ebersole, Emily Wu, Gabe Needle, Isabella Church, George Kyriakopoulos, Shaojun Xie, Yongmei Zaho, Charli Gruen, Antonella Sassano, Romina E. Araya, Andres Thorkelsson, Cari Smith, Maxwell P. Lee, Sridhar Hannenhalli, Chi-Ping Day, Glenn Merlino, R. Goldszmid, Shaojun Xie
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Introduction Metastatic cancer affects millions of people worldwide annually and is the leading cause of cancer-related deaths. Most patients with metastatic disease are not eligible for surgical resection, and current therapeutic regimens have varying success rates, some with 5-year survival rates below 5%. Here, we test the hypothesis that metastatic cancer can be genetically targeted by exploiting single base substitution mutations unique to individual cells that occur as part of normal aging prior to transformation. These mutations are targetable because ~10% of them form novel tumor-specific “NGG” protospacer adjacent motif (PAM) sites targetable by CRISPR-Cas9. Methods Whole genome sequencing was performed on five rapid autopsy cases of patient-matched primary tumor, normal and metastatic tissue from pancreatic ductal adenocarcinoma decedents. CRISPR-Cas9 PAM targets were determined by bioinformatic tumor-normal subtraction for each patient and verified in metastatic samples by high-depth capture-based sequencing. Results We found that 90% of PAM targets were maintained between primary carcinomas and metastases overall. We identified rules that predict PAM loss or retention, where PAMs located in heterozygous regions in the primary tumor can be lost in metastases (private LOH), but PAMs occurring in regions of loss of heterozygosity (LOH) in the primary tumor were universally conserved in metastases. Conclusions Regions of truncal LOH are strongly retained in the presence of genetic instability and, therefore, represent genetic vulnerabilities in pancreatic adenocarcinomas. A CRISPR-based gene therapy approach targeting these regions may be a novel way to genetically target metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada3070ea15ff6901b235b693417e61df449621f" target='_blank'>
              Islands of genomic stability in the face of genetically unstable metastatic cancer
              </a>
            </td>
          <td>
            Kirsten Bowland, Jiaying Lai, Alyza M. Skaist, Yan Zhang, S. Teh, Nicholas J. Roberts, Elizabeth Thompson, Sarah Wheelan, Ralph H. Hruban, Rachel Karchin, Matthew H Bailey, Christine A. Iacobuzio-Donahue, J. Eshleman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="BACKGROUND
Prostate cancer's complex interplay with the immune microenvironment prompted an investigation into immune-related pathogenic mechanisms and potential therapeutic targets.


METHODS
Within the GSE176031 data set, Seurat meticulously dissected single-cell profiles from radical prostatectomy patients. Leveraging CellMarker and SingleR cell identities were precisely annotated. Then, monocle traced pseudotime trajectories, illuminating cellular paths, complemented by CellChat's insights into intricate intercellular communications. Furthermore, mendelian randomization (MR) robustly substantiated causal associations within prostate cancer contexts.


RESULTS
Employing single-cell analysis on intraoperative tumor and normal tissue, we identified 15 distinct cell types, notably observing a significant T cell reduction in tumor samples. Intercellular communication analysis revealed multiple pathways between epithelial cells and T cells, highlighting interleukin (IL)-IL7R-IL2RG interactions. IL7R, crucial in T cell apoptosis, showed differential expression across T cell development stages. Patients with IL7 amplification had poorer outcomes (p < 0.05), supported by MR in two cohorts (ieu-b-4809 cohort: odds ratio [OR] = 1.005, p = 0.002, 95% confidence interval [CI] [1.002-1.008]; ebi-a-GCST90018905: OR = 1.063, p = 0.032, 95% CI [1.005-1.125]), confirming IL7 as a prostate cancer risk factor.


CONCLUSIONS
These findings suggest T cell depletion via IL7-IL7R signaling may drive prostate cancer progression, offering novel therapeutic insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bfc520d1668fd03c8682647e804ec854987049b" target='_blank'>
              IL7 as a Risk Factor for Prostate Cancer: Implications for T Cell Apoptosis and Infiltration in the Tumor Microenvironment.
              </a>
            </td>
          <td>
            Enyang He, Yaowen Li, Rui Zhao, Qinyan Kong, Yi Shao, Cong Wang, Baoqun Liu, Yvhang Jiang, Qian Liu, Hualei Cui
          </td>
          <td>2024-11-26</td>
          <td>The Prostate</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 In the tumor microenvironment (TME) cancer cells engage dynamically with the stromal component in promoting tumor progression and metastasis formation that are accompanied by intratumor fibrosis. Extensive extracellular matrix (ECM) remodeling and stiffness boost fibrosis which co-occurs with poor immune infiltration. Very little is known about the cross-interactions between tumor cells and stroma. A better understanding of this dynamic relationship may advance the development of new therapies. Transforming growth factor (TGF)-β signaling in both cancer epithelial cells and stroma is crucial for the establishment of intratumor fibrosis that enhances tumor progression. We previously discovered that in cancer cells, TGF-β signaling synergizes with RAS signal to regulate classical EMT transcription factors (Snail, Zeb1) and fibrogenic factors (Il11, Pdgf, Ctgfb, Wisp and others). This synergistic program is required for metastasis formation in pancreatic and lung adenocarcinoma (LUAD) models. However, how this dual program is affecting the stromal population in vivo remains unknown. Additionally, the role of each TGF-β dependent fibrogenic factor in regulating the stroma is still unclear. Using metastasis in vivo assays, we tested the metastatic ability of LUAD cells wild-type in comparison to Knock-Out cells targeting each one of the TGF-β dependent fibrogenic factors. We found that these factors are drivers of metastasis. More interestingly, these fibrogenic factors have a preferential effect depending on the metastatic site, suggesting that survival of the cancer cells in a certain tissue is dependent on the stromal niche within the site, which will act in favor or not of their growth. We also deeply characterize the stroma of micro-metastatic lesions by using high-throughput multiplex immunofluorescence to analyze the whole tissue TME at single cell resolution and found that lack of any of these fibrogenic factors compromise the establishment of a robust ECM at the very beginning of the metastatic colonization, to favor immune infiltration and consequently lead to cancer cell clearance. Together, these findings shed light on the potential role of these specific fibrogenic factors in metastasis, how their absence impacts the TME and how targeting these factors could be beneficial in metastatic disease.
 Citation Format: Elena Spina, Liangji Li, Junho Lee, Joan Massaguè. Deciphering the crosstalk between cancer cells and stroma that drives EMT- coupled fibrosis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A041.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46415627eb768278db1174a5ce4aee2f0c671b58" target='_blank'>
              Abstract A041: Deciphering the crosstalk between cancer cells and stroma that drives EMT- coupled fibrosis
              </a>
            </td>
          <td>
            Elena Spina, Liangji Li, Junho Lee, Joan Massagué
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/150e626daa762108097ae88ebcf26c0674670dd5" target='_blank'>
              Isolation and identification of patient-derived liver cancer stem cells and development of personalized treatment strategies
              </a>
            </td>
          <td>
            Tingting Guo, Shuai Zhang, Weiping Zeng, Yan Liang, Jinghe Xie, Shoupei Liu, Yaqi Qiu, Ying Fu, Y. Ou, Keqiang Ma, Bailin Wang, Weili Gu, Yuyou Duan
          </td>
          <td>2024-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Background: Prostate cancer is a prevalent malignancy in men, with treatment resistance posing a significant challenge to successful cure. Acquired Resistance to Therapy (ART) arises from two main components: (1) Tumor Microenvironment (TME) and (2) mutations or epigenetic changes at gene level within the tumor cells themselves. While most cancer therapies target the intrinsic biology of cancer cells, growing evidence indicates that these treatments can also harm the stromal cells within TME, potentially driving ART. The biology of ART may vary based on the types of treatments, cell types, and organ locations. Thus, this study aims to dissect ART components driven by TME-mediated cellular processes in a comprehensive manner. Hypothesis: We propose that cancer therapies induce stress responses in non-cancerous cells within the TME. These stress responses involve the secretion of cytokines, mediators, and growth factors, which we term Stress Response Secretory Programs (SRSPs). We further hypothesize that targeting SRSPs could overcome ART that helps to improve treatment specificity and efficacy. Materials and Methods: We employed comprehensive single-cell profiling of common metastatic sites included liver, prostate, lung, bone, lymph node and spleen in preclinical models to identify secreted proteins and dysregulated pathways caused by genotoxic therapies. In vitro, we evaluated treatment resistance in murine prostate epithelial cell lines (Myc-CAP) by exposing them to conditioned media (CM) from irradiated murine fibroblasts (F4M2), using cell growth assays. Results: Single-cell RNA sequencing of organ samples revealed significant expressions of SRSPs and increased inflammatory responses in liver and prostate fibroblasts as well as other stromal cells treated with carboplatin. In vitro, CM from irradiated F4M2 cells enhanced cell proliferation and survival of Myc-CAP cells in response to docetaxel treatment. Conclusion: Our findings highlight the crucial role of SRSPs in mediating ART through interactions within the TME, which alter tumor cell phenotypes. Targeting these SRSPs holds potential for overcoming treatment resistance and improving therapeutic outcomes, underscoring the need to consider TME complexity in cancer treatment strategies.
 Citation Format: Tony Lok Heng Chu, Armand Bankhead, Ilsa Coleman, Sander Frank, Peter S. Nelson, Tarana Arman. Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c940709273c954da8cb4bd8042d98f46649ce8ee" target='_blank'>
              Abstract C033: Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment
              </a>
            </td>
          <td>
            Tony Lok Heng Chu, Armand Bankhead, Ilsa M Coleman, Sander Frank, Peter S. Nelson, Tarana Arman
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Over the past century, colorectal cancer (CRC) has become one of the most devastating cancers impacting the human population. To gain a deeper understanding of the molecular mechanisms driving this solid tumor, researchers have increasingly turned their attention to the tumor microenvironment (TME). Spatial transcriptomics and proteomics have emerged as a particularly powerful technology for deciphering the complexity of CRC tumors, given that the TME and its spatial organization are critical determinants of disease progression and treatment response. Spatial transcriptomics enables high-resolution mapping of the whole transcriptome. While spatial proteomics maps protein expression and function across tissue sections. Together, they provide a detailed view of the molecular landscape and cellular interactions within the TME. In this review, we delve into recent advances in spatial biology technologies applied to CRC research, highlighting both the methodologies and the challenges associated with their use, such as the substantial tissue heterogeneity characteristic of CRC. We also discuss the limitations of current approaches and the need for novel computational tools to manage and interpret these complex datasets. To conclude, we emphasize the importance of further developing and integrating spatial transcriptomics into CRC precision medicine strategies to enhance therapeutic targeting and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b317770be242dc222a3004766249a5b9de43dcc" target='_blank'>
              Current state and future prospects of spatial biology in colorectal cancer
              </a>
            </td>
          <td>
            Francisco G. Carranza, Fernando C. Diaz, Maria Ninova, Enrique Velazquez-Villarreal
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Melanoma, a highly aggressive and life-threatening cancer, poses a significant therapeutic challenge, particularly for patients with distant metastasis, who face a 5-year survival rate of only approximately 22.5%. This underscores the urgent need for innovative therapeutic approaches. Emerging evidence suggests that sensory and parasympathetic innervation within the tumor microenvironment (TME) plays a crucial role in driving metastasis and immune evasion, partly through the promotion of M2 macrophages and cancer stem cells. To uncover the mechanisms behind tumor-specific innervation in metastatic melanoma, we employed a systems biology approach integrating bulk RNAseq from diverse melanoma cell lines, TCGA-SKCM data analysis, single-cell RNAseq, and spatial transcriptomics. This comprehensive analysis led us to identify a previously uncharacterized gene regulatory network (GRN) orchestrating tumor innervation. This GRN is driven by axon guidance cues (Netrin, Slit, Semaphorin), their receptors (DCC, Robo, Neuropilin, ITGB1), and transcription factors (RAS, RND1, TFAP2, CAMKII). We further discovered that the pioneer factor FOXC2, known for its association with metastasis, plays a critical role in opening the promoters of innervation-specific genes. Additionally, we identified Ephrin and CXCR4-mediated crosstalk between neurons and tumor cells. Machine learning models validated the accuracy of this GRN in predicting melanoma metastasis, and a neural network-based model confirmed the enrichment of nerve cells near metastatic tumor cells within the TME in TCGA immunohistochemistry data. The functional relevance of this network was further supported by CRISPR-mediated knockout studies, which disrupted tumor-specific signaling pathways. Notably, we also observed the enrichment of genes linked to mRNA processing bodies and neuron-specific membrane-less condensates in metastatic melanoma. These findings highlight the critical role of neuronal infiltration in melanoma metastasis. Currently, therapeutic strategies targeting this aspect of the TME remain limited. Our identified gene set offers potential as a metastatic predictor, and targeting the neural component of the TME represents a promising new therapeutic avenue for melanoma.
 Citation Format: Anjana Goli, Subhajit Dutta. Deciphering the neurogenic tumor microenvironment: A novel gene network driving innervation in metastatic melanoma offers prognostic and therapeutic insights [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a5d79cdb084a3fb556d9c17280ebbf7e5cef77" target='_blank'>
              Abstract B024: Deciphering the neurogenic tumor microenvironment: A novel gene network driving innervation in metastatic melanoma offers prognostic and therapeutic insights
              </a>
            </td>
          <td>
            Anjana Goli, Subhajit Dutta
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Desmosomes are transmembrane protein complexes critical for cell-cell adhesion in epithelial tissues and other cell types. In skin cancers like melanoma, epithelial cells called keratinocytes are the predominant cells in the primary melanoma microenvironment, yet it remains largely unknown whether this cell type is subject to genetic alterations during melanoma development and its role in early melanoma progression. In this study, using an integrative analysis of tumor mutations and protein biophysical interactions, we identify a high frequency of genetic alterations in desmosome-related genes in human cancers, with the highest occurrences in cutaneous melanoma. Notably, over 70% of cutaneous melanoma cases exhibit mutations in desmosome genes and desmosome gene mutations are associated with lowered expression of the desmosome complex in melanoma. Using spatial transcriptomics and immunohistochemistry analyses, we find that the decrease in desmosome gene expression predominantly occurs in keratinocytes within the tumor microenvironment rather than in the melanoma cells themselves. To study the functional significance of this alteration in melanoma development, we used a human melanoma/keratinocyte co-culture system using primary melanoma cell lines. Our studies show that knockdown of desmosome genes in keratinocytes markedly increased proliferation of adjacent melanoma cells in melanoma/keratinocyte co-cultures. Additionally, melanoma cell proliferation is enhanced when exposed to media preconditioned by desmosome-deficient keratinocytes. These observations suggest that the gradual accumulation of mutations in desmosome genes within neighboring keratinocytes may create a conducive environment that primes melanoma cells for neoplastic transformation. This study underscores the critical role of genetic alterations in the tumor microenvironment in melanoma progression and highlights the potential for targeting desmosome-related pathways in therapeutic strategies against melanoma.
 Citation Format: Mohita Tagore. Desmosome mutations in the microenvironment regulate melanoma proliferation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A042.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cede4aac61d054a3a470265c4e2db5a458ea7c19" target='_blank'>
              Abstract A042: Desmosome mutations in the microenvironment regulate melanoma proliferation
              </a>
            </td>
          <td>
            Mohita M. Tagore
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Motivation In recent years, developments in single-cell next-generation sequencing technology and computational methodology have made it possible to reconstruct, with increasing precision, the evolutionary history of tumors and their cell phylogenies, represented as mutation trees. Many mutation tree inference tools exist, but they do not support detailed visual tree inspection, nor tree comparisons or analysis at the cohort level, an important task in computational oncology. Results We developed oncotreeVIS, an interactive graphical user interface for visualizing mutation tree cohorts and tree posterior distributions obtained from mutation tree inference tools. Oncotree-VIS can display mutation trees that encode single or joint genetic events, such as point mutations and copy number changes, and highlight matching subclones, conserved trajectories and drug-gene interactions at cohort level. OncotreeVIS facilitates the visual inspection of mutation tree clusters and pairwise tree distances. It is available both as a JavaScript library that can be used locally or as a web application that can be accessed online. It includes seven default datasets of public mutation tree cohorts for visualization, while new mutation trees are provided in a predefined JSON format. Availability and implementation https://github.com/cbg-ethz/oncotreeVIS. Web application https://cbg-ethz.github.io/oncotreeVIS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/150fa13503ea6ff864d87ce3a7008a8cc31edc24" target='_blank'>
              OncotreeVIS – an interactive graphical user interface for visualizing mutation tree cohorts
              </a>
            </td>
          <td>
            Monica-Andreea Baciu-Drăgan, N. Beerenwinkel
          </td>
          <td>2024-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Cellular activity is spatially organized across different organelles. While several structures are well-characterized, many organelles have unknown roles. Profiling biomolecular composition is key to understanding function but is difficult to achieve in the context of small, dynamic structures. Photoproximity labeling has emerged as a powerful tool for mapping these interaction networks, yet maximizing catalyst localization and reducing toxicity remains challenging in live cell applications. Here, we disclose a new intracellular photocatalyst with minimal cytotoxicity and off-target binding, and we utilize this catalyst for HaloTag-based microenvironment-mapping (μMap) to spatially catalog subnuclear condensates in living cells. We also specifically develop a novel RNA-focused workflow (μMap-seq) to enable parallel transcriptomic and proteomic profiling of these structures. After validating the accuracy of our approach, we generate a spatial map across the nucleolus, nuclear lamina, Cajal bodies, paraspeckles, and PML bodies. These results provide potential new insights into RNA metabolism and gene regulation while significantly expanding the μMap platform for improved live-cell proximity labeling in biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cedc48a841755a5a6c7dc0077077c40278c4804" target='_blank'>
              Parallel Proteomic and Transcriptomic Microenvironment Mapping (μMap) of Nuclear Condensates in Living Cells.
              </a>
            </td>
          <td>
            Steve D. Knutson, Chenmengxiao Roderick Pan, Niels Bisballe, Brandon J Bloomer, Philip Raftopolous, Iakovos Saridakis, David W. C. MacMillan
          </td>
          <td>2024-12-21</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND
Clear cell renal cell carcinoma (ccRCC) is the most common histologic type of RCC. However, the spatial and functional heterogeneity of immunosuppressive cells and the mechanisms by which their interactions promote immunosuppression in the ccRCC have not been thoroughly investigated.


METHODS
To further investigate the cellular and regional heterogeneity of ccRCC, we analyzed single-cell and spatial transcriptome RNA sequencing data from four patients, which were obtained from samples from multiple regions, including the tumor core, tumor-normal interface, and distal normal tissue. On the basis, the findings were investigated in vitro using tissue and blood samples from 15 patients with ccRCC and validated in the broader samples on tissue microarrays.


RESULTS
In this study, we revealed previously unreported subsets of both stromal and immune cells, as well as mapped their spatial location at finer resolution. In addition, we validated the clusters of tumor cells after removing batch effects according to six characterized gene sets, including epithelial-mesenchymal transitionhigh clusters, metastatic clusters and proximal tubulehigh clusters. Importantly, we identified a special regulatory T (Treg) cell subpopulation that has the molecular characteristics of terminal effector Treg cells but expresses multiple cytokines, such as interleukin (IL)-1β and IL-18. This group of Treg cells has stronger immunosuppressive function and was associated with a worse prognosis in ccRCC cohorts. They were colocalized with MRC1 + FOLR2 + tumor-associated macrophages (TAMs) at the tumor-normal interface to form a positive feedback loop, maintaining a synergistic procarcinogenic effect. In addition, we traced the origin of IL-1β+ Treg cells and revealed that IL-18 can induce the expression of IL-1β in Treg cells via the ERK/NF-κB pathway.


CONCLUSIONS
We demonstrated a novel cancer-promoting Treg cell subset and its interactions with MRC1 + FOLR2 +TAMs, which provides new insight into Treg cell heterogeneity and potential therapeutic targets for ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2214f0f0d02f003fb8cfcf936f4f2873d564786b" target='_blank'>
              Spatial and single-cell transcriptomics reveal cellular heterogeneity and a novel cancer-promoting Treg cell subset in human clear-cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Xiyu Song, Yumeng Zhu, Wenwen Geng, J. Jiao, Hongjiao Liu, Ruo Chen, Qian He, Lijuan Wang, Xiuxuan Sun, Weijun Qin, Jiejie Geng, Zhinan Chen
          </td>
          <td>2025-01-01</td>
          <td>Journal for immunotherapy of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The role of cancer-associated pericytes (CAPs) in tumor microenvironment (TME) suggests that they are potential targets for cancer treatment. The mechanism of CAP heterogeneity in esophageal squamous cell carcinoma (ESCC) remains unclear, which has limited the development of treatments for tumors through CAPs. Therefore, a comprehensive understanding of the classification, function, cellular communication and spatial distribution of CAP subpopulations in ESCC is urgently needed.This study used large-sample single-cell transcriptome sequencing (scRNA-seq) data to investigate pericytes’ subpopulation characteristics, functions, upstream and downstream regulation and interactions with other components of the TME in the ESCC, and analyzed prognostically in conjunction with Bulk RNA-seq data. In addition, pericyte subpopulations were validated and their spatial distribution in the ESCC TME was observed by multiplex immunofluorescence. Drug prediction and molecular docking was further used to validate the medicinal value of drug targets.CAPs in the ESCC TME were found to be highly heterogeneous, and we identified six pericyte subtypes: c1_ARHGDIB, c2_BCAM, c3_LUM, c4_SOD2, c5_TYMS, and c6_KRT17, which have commonality in a part of their functions, and each of them has a major function to play, by having different strengths of interaction with different components in the TME. In addition, we found that c4_SOD2 was negatively correlated with prognosis, conversely, c5_TYMS was positively correlated with prognosis. The drug with a better effect on c5_TYMS was docetaxel (binding energy = -8.1, -8.7 kcal/mol); raloxifene may be more effective against c4_SOD2, although raloxifene has a slightly lower binding energy to SOD2 (-6.4 kcal/mol), it has a higher binding energy to PDGFRβ (-8.1 kcal/mol).The present study identified and discovered pericyte subpopulations that were significantly associated with prognosis, which provides new biomarkers for predicting patient prognosis and adds usable targets for immunotherapy, and it is also important for gaining insights into the composition of the TME in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e295485d04626d24ad4217e848a133499f9df967" target='_blank'>
              Single-cell sequencing analysis reveals cancer-associated pericyte subgroup in esophageal squamous cell carcinoma to predict prognosis
              </a>
            </td>
          <td>
            Kai Xiong, Bing Pan, Hao Fang, Ziyou Tao
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Defining consequential differences in intestinal epithelial stem cells in healthy humans versus those with inflammatory bowel disease (Crohn’s disease and ulcerative colitis) is essential for the development of much needed therapies to restore the epithelial barrier and maintain its fidelity. Employing single cell transcriptomic/epigenomic approaches and colonoid models from children and adults with Crohn’s disease led us to identify an inflammatory secretory progenitor (ISP) cell state present almost exclusively in patients with Crohn’s disease compared to control subjects. ISPs exhibit gene expression profiles consistent with normal secretory progenitor cells but concomitantly express a suite of distinguishing pro-inflammatory genes. Mechanistically, ISPs exhibit open chromatin and gene expression of ISP signature genes. While these ISP-specific genes are not expressed in intestinal stem cells, their chromatin is accessible in Crohn’s disease stem cells suggesting that ISP genes are epigenetically poised in stem cells and are transcriptionally activated in ISP cells in the presence of inflammatory stimuli. Consistently, Crohn’s disease colonoids exhibit sustained ISP gene expression that can be elicited further with pro-inflammatory cytokines or via co-culture with pro-inflammatory macrophages. In summary, we define differences in the epithelial stem and progenitor compartment of patients with Crohn’s disease suggesting aberrant stem cell differentiation and inflammatory gene expression arises during disease. HIGHLIGHTS We identify an inflammatory secretory progenitor cell state in endoscopically non-inflamed tissue from pediatric and adult patients with Crohn’s disease. Crohn’s disease epithelial stem and progenitor cells display increased chromatin accessibility preceding the emergence of inflammatory secretory progenitor cell states. Crohn’s disease colonoids from non-inflamed regions contain inflammatory secretory progenitor cells in the absence of inflammatory stimuli Inflammatory secretory progenitor cells expand in colonoid culture in response to cytokine stimulation or co-culture with pro-inflammatory macrophages">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1c97ff6cf153588b934b86c33ad5324b1f6d33" target='_blank'>
              An epigenetic basis for sustained inflammatory epithelial progenitor cell states in Crohn’s disease
              </a>
            </td>
          <td>
            Tatiana A. Karakasheva, Clara Morral Martinez, Yusen Zhou, Jingya Qui, Xinyi E. Chen, Gloria E. Soto, Shaneice K. Nettleford, Olivia T. Hix, Daana M. Roach, Alyssa M. Laguerta, Anusha Thadi, Rachael M. Edwards, Daniel Aleynick, Sarah Weinbrom, Elizaveta Borodyanskaya, Oliver Pickering, Marykate Fulton, Chia- Chen, Bella V. Peterson, Erik B. Hagen, Ian P. Yannuzzi, Zainab Haider, Zvi Cramer, Maire C Conrad, Ning Li, Meenakshi Bewtra, Yasin Uzun, Kai Tan, J. Kelsen, Andy J. Minn, Christopher Joachim Lengner, Kathryn E. Hamilton
          </td>
          <td>2024-11-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5daf8fc78edec4dc67cc0ac87030a2ee471c5078" target='_blank'>
              Cancer-associated fibroblasts: heterogeneity, tumorigenicity and therapeutic targets
              </a>
            </td>
          <td>
            Keke Lv, Tianlin He
          </td>
          <td>2024-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Fibroblasts are critical cells that shape the architecture and cellular ecosystems in multiple tissues. Understanding fibroblast heterogeneity and their spatial context in health and disease has enormous clinical relevance. In this study, we constructed a spatially-resolved atlas of human skin fibroblasts from healthy skin and 23 skin disorders. We define 6 major skin fibroblast populations in health and a further three skin disease-specific fibroblast subtypes, and demonstrate the fibroblast composition in different types of skin disease. We characterise a human-specific fibroblastic reticular cell (FRC)-like subtype in the skin perivascular niche and postulate their origin from prenatal skin lymphoid tissue organiser (LTo)-like cells. We also show that inflammatory myofibroblasts (IL11+MMP1+CXCL5+IL7R+) are a conserved fibroblast subtype in inflammatory disorders and cancers across multiple human tissues. We provide a harmonised nomenclature for skin fibroblasts that integrates previous findings from human skin and other tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8105c94d4aa143c539f0558370814becb076a8c6" target='_blank'>
              A single cell and spatial genomics atlas of human skin fibroblasts in health and disease
              </a>
            </td>
          <td>
            Lloyd Steele, Chloe Admane, Keerthi Priya Chakala, A. Foster, N. Gopee, Simon Koplev, Pavel Mazin, B. Olabi, Kenny Roberts, Catherine Tudor, E. Winheim, Karl Annusver, D. Correa-Gallegos, A. Forsthuber, L. Francis, S. Frech, Clarisse Gânier, T. Layton, Yingzi Liu, Hao Yuan, J. Gudjonsson, B. Lichtenberger, S. Mahil, J. Nanchahal, Edel A O'Toole, Maksim Plikus, Y. Rinkevich, Emanuel Rognoni, Catherine Smith, Sarah A. Teichmann, Maria Kasper, M. Lotfollahi, M. Haniffa
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="
 Ewing sarcoma (EwS) is an aggressive bone and soft tissue tumor and EwS patients often succumb to their disease years after initial treatment due to relapses at metastatic sites. Novel therapeutic strategies are needed that can successfully eliminate microscopic residual disease foci that persist at the end of primary therapy. Extracellular matrix (ECM) proteins in the tumor microenvironment (TME) provide critical pro-survival and pro-invasion signals to tumor cells. Our published and unpublished data show that the glycoprotein tenascin-C (TNC) is enriched in EwS metastases and that TNC and other pro-tumorigenic ECM proteins are deposited by subpopulations of CAF-like tumor cells that are activated in response to tumor and TME-derived TGF-beta ligands. TNC is abundantly produced by multiple human tumors of epithelial and non-epithelial lineage but is otherwise rarely expressed outside of development and wound healing. Importantly, TNC is specifically enriched in metastatic lesions where it has been implicated as a mediator of metastatic competence and treatment resistance. In the current work we have tested whether the TNC-rich TME of disseminated EwS tumor foci could be leveraged to direct therapies to sites of residual micrometastatic disease. To achieve this, we generated monovalent and bivalent anti-human TNC VHHs (hTNC-VHH) using a mammalian expression platform. Camelid-derived hTNC-VHH sequences were sourced from published literature. VHHs are single domain heavy chain only antibody fragments with higher stability and tissue penetration than conventional monoclonal antibodies. Using EwS tumor spheroids in collagen-rich 3D culture, hTNC-VHH penetrated dense ECM matrices and bound through multiple cell layers in under 10 minutes. hTNC-VHH accumulated in TNC-positive but not TNC-knockout EwS spheroids and were retained beyond 96 hours. To assess their potential to drive protein and immune therapies to TNC-rich TMEs, we fused hTNC-VHH to immune modulating CD3- and CD28-activating VHHs. These hTNC-VHH-CD3/CD28-conjugates promoted immune cell activation and tumor cell death in PBMC-tumor co-culture assays as determined by CD25 flow cytometry and fluorescence microscopy, respectively. To assess in vivo localization, fluorescently-tagged hTNC-VHH were intravenously injected into mice that had been xenografted with EwS cells. Tissue microscopy confirmed selective binding of hTNC-VHH to small lung and liver EwS micrometastases in less than 4 hours. VHHs were not retained in non-tumor bearing organs including the brain and heart or in tumor-free lung and liver regions. Experiments are ongoing to assess anti-tumor efficacy and pharmacodynamics of hTNC-VHH conjugates in EwS xenografts. Together these findings suggest that the ECM-remodeling properties of CAF-like EwS cells can be exploited to recruit novel ECM-targeting protein therapeutics to micrometastases. If successful, this innovative approach could eradicate microscopic residual disease and prevent metastatic EwS recurrence.
 Citation Format: Emma D. Wrenn, Jason P. Price, Raymond O. Ruff, Nicolas M. Garcia, James M. Olson, Elizabeth R. Lawlor. Leveraging ECM deposition by CAF-like Ewing sarcoma tumor cells to target micrometastases with matrix-binding VHHs [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C050.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9942d5319f0b477046d50c6a8b105f9899b2930d" target='_blank'>
              Abstract C050: Leveraging ECM deposition by CAF-like Ewing sarcoma tumor cells to target micrometastases with matrix-binding VHHs
              </a>
            </td>
          <td>
            Emma D. Wrenn, Jason P. Price, Raymond O. Ruff, Nicolas M Garcia, James M. Olson, Elizabeth R. Lawlor
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) play pivotal roles in solid tumor initiation, growth, and immune evasion. However, the optimal biomimetic modeling conditions remain elusive. In this study, we investigated the effects of 2D and 3D culturing conditions on human primary CAFs integrated into a modular tumor microenvironment (TME). Using single-nucleus RNA sequencing (snRNAseq) and Proteomics’ Proximity Extension Assays, we characterized CAF transcriptomic profiles and cytokine levels. Remarkably, when cultured in 2D, CAFs exhibited a myofibroblast (myCAF) subtype, whereas in 3D tumor spheroid cultures, CAFs displayed a more inflammatory (iCAF) pathological state. By integrating single-cell gene expression data with functional interrogations of critical TME-related processes [natural killer (NK)-mediated tumor killing, monocyte migration, and macrophage differentiation], we were able to reconcile form with function. In 3D TME spheroid models, CAFs enhance cancer cell growth and immunologically shield cells from NK cell-mediated cytotoxicity, in striking contrast with their 2D TME counterparts. Notably, 3D CAF-secreted proteins manifest a more immunosuppressive profile by enhancing monocyte transendothelial migration and differentiation into M2-like tumor-associated macrophages (TAMs). Our findings reveal a more immunosuppressive and clinically relevant desmoplastic TME model that can be employed in industrial drug discovery campaigns to expand the cellular target range of chemotherapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43b61425d19bdd4e9908dd233e505b35ad2127a3" target='_blank'>
              Unveiling immune cell response disparities in human primary cancer-associated fibroblasts between two- and three-dimensional cultures
              </a>
            </td>
          <td>
            Jian-Ping Yang, Nikhil Nitin Kulkarni, Masashi Yamaji, Tsubasa Shiraishi, Thang Pham, Han Do, Nicole Aiello, Michael Shaw, Toshihiro Nakamura, Akiko Abiru, N. Gavva, Shane R. Horman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="ABSTRACT High-risk non-muscle-invasive bladder cancer (NMIBC) presents high recurrence and progression rates. Despite the use of Bacillus Calmette-Guérin gold-standard immunotherapy and the recent irruption of anti-PD-1/PD-L1 drugs, we are missing a comprehensive understanding of the tumor microenvironment (TME) that may help us find biomarkers associated to treatment outcome. Here, we prospectively analyzed TME composition and PD-L1 expression of tumor and non-tumoral tissue biopsies from 73 NMIBC patients and used scRNA-seq, transcriptomic cohorts and tissue micro-array to validate the prognostic value of cell types of interest. Compared to non-tumoral tissue, NMIBC presented microvascular alterations, increased cancer-associated fibroblast (CAF) and myofibroblast (myoCAF) presence, and varied immune cell distribution, such as increased macrophage infiltration. Heterogeneous PD-L1 expression was observed across subsets, with macrophages showing the highest expression levels, but cancer cells as the primary potential anti-PD-L1 binding targets. Unbiased analysis revealed that myoCAF and M2-like macrophages are specifically enriched in high-grade NMIBC tumors. The topological distribution of these two cell types changed as NMIBC progresses, as shown by immunofluorescence. Only myoCAFs were associated with higher rates of progression and recurrence in three independent cohorts (888 total patients), reaching prediction values comparable to transcriptomic classes, which we further validated using tissue micro-array. Our study provides a roadmap to establish the landscape of the NMIBC TME, highlighting myoCAFs as potential prognostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe97543b0d1de2e394926985ac37960e50a2bea" target='_blank'>
              Characterisation of the tumour microenvironment and PD-L1 granularity reveals the prognostic value of cancer-associated myofibroblasts in non-invasive bladder cancer
              </a>
            </td>
          <td>
            Carmen G. Cañizo, F. Guerrero-Ramos, M. Perez Escavy, I. Lodewijk, C. Suárez-Cabrera, L. Morales, Sandra P Nunes, E. Munera-Maravilla, Carolina Rubio, Rebeca Sánchez, Marta Rodríguez-Izquierdo, Jaime Martínez de Villarreal, Francisco X. Real, D. Castellano, Cristina Martín-Arriscado, David Lora Pablos, A. Rodríguez Antolín, M. Dueñas, J. Paramio, Victor G. Martínez
          </td>
          <td>2024-12-19</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="CAR-T therapies utilizing T cells engineered with chimeric antigen receptors (CARs) have revolutionized the treatment of hematologic and immune-related malignancies. However, the anti-tumor capability of engineered CAR-T cells are often not persistent, which greatly dampen its clinical efficacy. To address this limitation, massive parallel genetic screens were widely used to identified novel targets and regulators that enhance T cell anti-tumor capability and persistence in tumor microenvironment. We hypothesized that by combining the pooled screen data from multiple independent genetic screens we could provide a systematic, comprehensive, and robust analysis of the effect of gene perturbation on T-cell based immunotherapies. After collecting data from previously published T cell screens, including CRISPR-based and ORF-based screens, through Gene Expression Omnibus (GEO), we reprocessed the gene hits summary and conducting a pathway enrichment analysis. A T cell screen perturbation score (TPS) was employed to quantifies the impact of the gene on T cell function. Additionally, gene expression data (both bulk RNA level and single-cell RNA level) from autoimmune disease and T cell-derived cancers were analyzed to pick up gene perturbations that potentially augment T cell proliferation. We integrated all data and analysis on 27 T-cells screens into our state-of-the-art T cell perturbation genomics database (TCPGdb), which is made easily accessible through our webserver and allows users to interactively explore the impact of query genes on T cell function based on prior screen data and our TPS scoring module. TCPGdb is publicly accessible at http://tcpgdb.sidichenlab.org/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a099e229f2c35d38ee6b66c8422aec7d483d003c" target='_blank'>
              TCPGdb: A comprehensive T cell perturbation genomics database for identification of critical T cell regulators
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Kaiyuan Tang, Nipun Verma, Xinxin Zhu, Di Feng, James Cai, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Aging is a multifaceted process that manifests differently in various cell types and within the same cell type. Therefore, understanding age-related changes with single-cell resolution may illuminate the mechanistic details of tissue aging and offer targets for intervention. We utilized high-throughput single-cell RNA-sequencing (scRNA-seq) to analyze senescence-associated changes within the CD4 T cell compartment. First, we sequenced total human CD4 T cells from multiple donors, revealing that naïve CD4 T cells exhibited a less senescent state compared to memory cells. We focused on naïve CD4 T cells to determine if individual cells differ in expression of aging-associated markers. By sorting cells based on senescence-associated beta-galactosidase (B-gal) activity and applying scRNA-seq, we identified transcriptional signatures associated with more youthful and senescent states. Surprisingly, we discovered cells with lower B-gal expression comprised two distinct populations: one marked by high expression of HELIOS and low expression of SESN3, and another marked by high expression of BACH2 and low expression of TC2N. In comparison, 2 youthful subpopulation and senescence subpopulations differ in markers of immunological activation We then investigated if the composition of naive CD4 compartment changes with age, finding that the abundance of youthful subpopulations was reduced in the older donors’ CD4 populations. Thus, aging in the naïve CD4 compartment is associated with a shifted distribution between more youthful and more senescent cells. Moving forward, we will attempt to manipulate naïve CD4 T cells into a more youthful state by altering expression of the identified transcriptional signatures associated with youthful and senescent states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bab6ece44b0e3064c9cd597254aa9a3e1045d71" target='_blank'>
              DISTINCT CELL SUBTYPES WITHIN NAÏVE CD4 T CELLS COMPARTMENT SHAPE ITS AGING AND SENESCENCE
              </a>
            </td>
          <td>
            C. J. Kelly, Ashley Jeon, Jacob Rodriguez, Dina Prosser, Morgan Diegel, Nikolay Burnaevskiy
          </td>
          <td>2024-12-01</td>
          <td>Innovation in Aging</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55057507874b2d1db614aaeb55e2719a4aaa942d" target='_blank'>
              Metabolic rewiring in skin epidermis drives tolerance to oncogenic mutations.
              </a>
            </td>
          <td>
            Anupama Hemalatha, Zong-Hu Li, David G. Gonzalez, C. Matte-Martone, Karen Tai, Elizabeth Lathrop, Daniel Gil, S. Ganesan, Lauren E Gonzalez, Melissa C. Skala, Rachel J. Perry, Valentina Greco
          </td>
          <td>2025-01-06</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Ambiguous understanding of tumors and the tumor microenvironments (TMEs) hinders accurate diagnosis and available treatment for multifocal hepatocellular carcinoma (HCC) covering intrahepatic metastasis (IM) and multicentric occurrence (MO). Here, we characterized the diverse TMEs of IM and MO identified by whole-exome sequencing (WES) at single-cell resolution.


APPROACH AND RESULTS
We performed parallel WES and single-cell RNA sequencing (scRNA-seq) on twenty-three samples from seven patients to profile their TMEs when major results were validated by immunohistochemistry in the additional cohort. Integrative analysis of WES and scRNA-seq found that malignant cells in IM showed higher intra-tumor heterogeneity, stemness and more activated metabolism than those in MO. Tumors from IM shared similar TMEs while distinct TMEs were noticed in those from MO. Furthermore, CD20+ B cells, plasma cells and conventional type II dendritic cells (cDC2s) were decreased in IM relative to MO while T cells in IM exhibited a more terminally exhausted capacity with a higher proportion of proliferative/exhausted T cells than that in MO. Both CD20 and CD1C correlated with better prognosis in multifocal HCC. Additionally, MMP9+ tumor-associated macrophages were enriched across IM and MO which formed cellular niches with regulatory T cells (Tregs) and proliferative/exhausted T cells.


CONCLUSIONS
Our findings deeply decipher the heterogeneous TMEs between IM and MO, which provide a comprehensive landscape of multifocal HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2c7a45eee2228edb84e8693a219534a9ab548a" target='_blank'>
              Genomic and the tumor microenvironment heterogeneity in multifocal hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yongheng Yang, Qingqiang Ni, Hongguang Li, Jiuzheng Sun, Xia Zhou, Lingxin Qu, Liyuan Wang, Chuanzong Zhao, Xiaolu Zhang
          </td>
          <td>2024-12-12</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25ea97844e3ac53280202c1ab1e25a03d1cb6ccc" target='_blank'>
              Tailoring glioblastoma treatment based on longitudinal analysis of post-surgical tumor microenvironment
              </a>
            </td>
          <td>
            C. Bastiancich, E. Snacel-Fazy, Samantha Fernandez, Stéphane Robert, Roberta Stacchini, Léa Plantureux, Sébastien Boissonneau, Benoit Testud, Benjamin Guillet, F. Debarbieux, H. Luche, D. Figarella-Branger, Marie-Anne Estève, E. Tabouret, A. Tchoghandjian
          </td>
          <td>2024-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4bbd7ae49e10933acda8e20cbd7e3bb473c50d" target='_blank'>
              Endosomal pH is an evolutionarily conserved driver of phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            Hari Prasad, H. Bv, A. R. Subbalakshmi, S. Mandal, M. Jolly, Sandhya S. Visweswariah
          </td>
          <td>2024-12-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 Breast cancer ranks as the most commonly diagnosed cancer in women with 90% of associated deaths caused by distant metastasis. The metastatic lesions in brain, lung, liver, bone and lymph nodes in human breast cancer are not arbitrary, but rather are a complex mechanism of extrinsic and intrinsic regulatory factors controlling organ-specific colonization. Specialized mechanisms driving bone, brain and lung cancer have been described. However, the studies of liver and lymph node metastatic drivers are limited despite the prevalence of metastases in these organs. A significant limitation is the lack of genetically engineered mouse models that develop mammary tumors that spontaneously metastasize to liver and lymph nodes. Our previous research identified the role for E2F5 in both mammary gland development and breast cancer. Loss of E2F5 resulted in tumorigenesis after a latency of one year and 70% of mice showed metastatic lesions in lung, lymph node and liver. Transplantation of these spontaneous tumors into FVB MMTV Cre mice recapitulated the metastatic pattern observed in the primary tumors. We propose that E2F5 regulates drivers of metastasis and potentiates organ tropism to lymph nodes and the liver. To address this hypothesis, we conducted serial transplantations of tumors originating from the lymph nodes and liver back into the mammary gland. This enrichment study led to a significant increase in organ-specific metastasis, with 60% and 77% of penetrance observed in two distinct mouse lineages with liver (LVM) and lymph node tropism (LNM). Immunohistochemical studies confirmed a complete epithelial-mesenchymal transition (EMT) phenotype in LVM that is conserved in the cell lines generated. LNM showed a partial EMT with a more epithelial phenotype in cell lines. We observed the presence of an immune enriched environment in primary and metastatic tumors in both lineages. Bulk-RNA sequencing studies uncovered the presence of unique organotropic pathways associated with liver and lymph node metastasis. We observed enriched pathways from the primary tumor as major component of our organotropic drivers with a clear distinction between liver and lymph node enriched lineages. Additionally, a gradual acquisition of distant organ-specific markers and signaling pathways suggests that phenotypic mimicry and tumor plasticity are key for successful events of colonization to liver and lymph nodes in our model. With these results, we unravel potential mechanisms responsible for the organotropic metastases in breast cancer under the absence of the transcription factor E2F5.
 Citation Format: Jesus A. Garcia Lerena, Briana To, Jing-Ru Jhan, Eran Andrechek. E2F5 conditional knockout in mammary epithelium drives organotropic metastasis in breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/858b1038bd0045570bb917d2e42ad23af4f814e4" target='_blank'>
              Abstract A004: E2F5 conditional knockout in mammary epithelium drives organotropic metastasis in breast cancer
              </a>
            </td>
          <td>
            Jesus A. Garcia Lerena, Briana To, Jing-Ru Jhan, E. Andrechek
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Hepatic arterial infusion chemotherapy (HAIC) has emerged as a promising treatment strategy for hepatocellular carcinoma (HCC), but a detailed understanding of the multicellular ecosystem after HAIC treatment is lacking. Here, we collected tumor samples from treatment-naïve primary and post-HAIC HCC, and integrated single-nucleus RNA sequencing with spatial transcriptomics to characterize the tumor ecosystem in the post-HAIC HCC. Increased fractions and enhanced cellular communication of CD4+ T, CD20+ B, and dendritic cell subtypes were identified in post-HAIC tumors. Moreover, it is substantiated that HAIC promoted tertiary lymphoid structures (TLS) formation, and addressed the roles of TLSs as spatial niches of cellular communication. Specifically, intermediate exhausted CD8+ T cells expressing Granzyme-K and PD-1 (PD-1+CD8+ Tex-int) expanded following HAIC and exhibited a functionally antitumor phenotype. PD-1+CD8+ Tex-int accumulated in the TLS vicinity and disseminated throughout the tumor microenvironment, demonstrating potential as an effective biomarker for HAIC-based treatment in HCC. This study provides valuable resources and biological insights in the cellular underpinnings of HAIC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62624d4b04f46d42e65710fe0c98f6d6afaa46d" target='_blank'>
              Single-Nucleus and Spatial Transcriptome Profiling Delineates the Multicellular Ecosystem in Hepatocellular Carcinoma After Hepatic Arterial Infusion Chemotherapy.
              </a>
            </td>
          <td>
            YeXing Huang, Zefeng Du, Zhicheng Lai, DongSheng Wen, Lichang Huang, Minke He, ZiChao Wu, Huifang Li, Hanyue Ouyang, Wenchao Wu, Anna Kan, Ming Shi
          </td>
          <td>2024-12-16</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b26feb24c74a70e25f404994931c40c73dd0fe64" target='_blank'>
              Deciphering molecular landscape of breast cancer progression and insights from functional genomics and therapeutic explorations followed by in vitro validation
              </a>
            </td>
          <td>
            Bushra Khan, Rowaid M. Qahwaji, Mashael S. Alfaifi, Tanwir Athar, Abdullah Khan, Mohammad Mobashir, I. Ashankyty, K. Imtiyaz, Areej Alahmadi, M. M. Rizvi
          </td>
          <td>2024-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aaea8ef96bcf757be9b672ce6a26fbe4a6d0c91" target='_blank'>
              Integration of single-cell sequencing and drug sensitivity profiling reveals an 11-gene prognostic model for liver cancer
              </a>
            </td>
          <td>
            Qunfang Zhou, Jingqiang Wu, Jiaxin Bei, Zixuan Zhai, Xiuzhen Chen, Wei Liang, Jing Meng, Mingyu Liu
          </td>
          <td>2024-11-25</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d49c3b89849fb97fd894e63e495575849c7b7545" target='_blank'>
              A multilineage screen identifies actionable synthetic lethal interactions in human cancers.
              </a>
            </td>
          <td>
            Samson H. Fong, Brent M Kuenzi, Nicole M Mattson, John Lee, Kyle S. Sanchez, A. Bojorquez-Gomez, Kyle Ford, Brenton P. Munson, Katherine Licon, Sarah Bergendahl, John Paul Shen, J. Kreisberg, Prashant Mali, Jeffrey H Hager, Michael A White, T. Ideker
          </td>
          <td>2024-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is the third leading cause of cancer-related deaths with a 5-year survival rate of 13%. Surgical resection remains the only curative option as systemic therapies offer limited benefit. Poor response to chemotherapy and immunotherapy is due, in part, to the dense stroma and heterogeneous tumor microenvironment (TME). Opportunities to target the PDAC stroma may increase the effectiveness of existing or novel therapies. Current strategies targeting the stromal compartment within the PDAC TME primarily focus on degrading extracellular matrix or inhibiting stromal cell activity, angiogenesis, or hypoxic responses. In addition, extensive work has attempted to use immune targeting strategies to improve clinical outcomes. Preclinically, these strategies show promise, especially with the ability to alter the tumor ecosystem; however, when translated to the clinic, most of these trials have failed to improve overall patient outcomes. In this review, we catalog the heterogenous elements of the TME and discuss the potential of combination therapies that target the heterogeneity observed in the TME between patients and how molecular stratification could improve responses to targeted and combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254631c9190ee22fb9be4694a816eb9553b6afce" target='_blank'>
              Challenges and Opportunities in Targeting the Complex Pancreatic Tumor Microenvironment
              </a>
            </td>
          <td>
            Jennifer M. Finan, Yifei Guo, Shaun M Goodyear, Jonathan R. Brody
          </td>
          <td>2024-12-01</td>
          <td>JCO Oncology Advances</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor microenvironment (TME) plays a crucial role in the oncogenesis, early distant metastasis, and limited immunotherapeutic responses of pancreatic ductal adenocarcinoma (PDAC). Myeloid cell populations, particularly tumor-associated macrophages (TAMs), located within this immunosuppressive niche, are pivotal in these processes. With aid of single cell RNA sequencing (scRNA-seq), multiple studies have successfully validated distinct macrophage subtypes pathological functions in different cancer types, yet the correlation of TAMs in hepatic metastatic and primary PDAC lesions remains insufficiently understood. In this study, we reanalyzed samples from both primary and hepatic metastatic PDACs to elucidate the functional conservation and variability of TAMs. Pseudotime trajectory inference, together with pseudobulk analysis based on subtype level, manifests TAMs undergo similar metabolism reprogramming and pro-inflammatory style transitions with specific signaling pathway activations. Compared to primary tumors, hypoxic conditions are alleviated, which largely depends on the physiological context and the preliminary ligand-receptor interactive network within the metastatic niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a56bbf6854596a35b27cf40482bd3f5c23fafab" target='_blank'>
              In-depth analysis of pathological function of tumor-associated macrophage populations in hepatic metastatic and primary PDAC
              </a>
            </td>
          <td>
            Fanyu Meng
          </td>
          <td>2024-12-12</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Metastasis in breast cancer is a major challenge for prognosis and effective treatment, primarily because cancer cells can create an immune-suppressive microenvironment that supports their progression and spread. The physical properties of cancer cells significantly influence their metastatic potential and their interaction with the environment. Identifying targets that impact both the physical characteristics of cancer cells, and their microenvironment is crucial for tackling metastatic disease. Interferon-stimulated gene 15 (ISG15), a ubiquitin-like protein, is often overexpressed in breast cancer tissues and is associated with poor prognosis, higher tumor grade, and increased invasiveness. Despite these associations, the precise role of ISG15 and its post-translational modification, ISGylation, in cancer cell behavior and interactions with their microenvironment remains unclear. To explore this, we interrogated the biophysical role of ISG15 by using mouse modeling and in vitro loss of function approaches. We utilized whole- body ISG15-deficient mice and conducted a series of experiments to assess the impact of ISG15 on cancer cell properties and their surrounding environment. Our preliminary results indicate that cells lacking ISG15 exhibit altered physical traits, such as increased stiffness and size, which may influence their metastatic behavior. Additionally, targeting ISG15 or disrupting ISGylation enhances the susceptibility of cancer cells to immune cell-mediated destruction. In ISG15- deficient mice, we observed significantly reduced tumor growth and metastasis compared to wild-type controls. These findings suggest that ISG15 modulates critical aspects of cancer cell behavior and their interactions with the microenvironment, contributing to immune evasion and metastatic spread. Our research highlights ISG15 as a promising therapeutic target and provides a foundation for developing new strategies to prevent and treat metastatic cancer
 Citation Format: Mohamed Haloul, Priya Shah, Vinay Pai, Faruk Hossen, James James Lee, Ekrem Emrah Er. ISG15 modulate cancer cell mechanics and metastasis in breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B027.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a28aa9d49c850dc48c11b47ab53305430122a856" target='_blank'>
              Abstract B027: ISG15 modulate cancer cell mechanics and metastasis in breast cancer
              </a>
            </td>
          <td>
            Mohamed Haloul, Priya Shah, Vinay Pai, Faruk Hossen, James James Lee, E. E. Er
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 The awakening of dormant disseminated cancer cells is likely responsible for the clinical relapses of patients whose primary tumors have been cured months and even years earlier. In the present study, we demonstrate that dormant breast cancer cells lodged in the lungs reside in a highly mesenchymal, non-proliferative phenotypic state. The awakening of these cells does not occur by a cancer cell-autonomous process. Instead, inflammation and wound healing of the surrounding tissue microenvironment causes them to shift from a highly mesenchymal to a quasi-mesenchymal phenotypic state in which they acquire stemness and proliferative ability. Once awakened, these cells can stably reside in this quasi-mesenchymal state and maintain their stemness, doing so without ongoing heterotypic signaling from the lung microenvironment. EGFR ligands released by the cells of the injured tissue microenvironment, including notably M2 type macrophages, promote dormant cancer cells to move toward this quasi-mesenchymal state, a transition that is essential for the awakening process. An understanding of the mechanisms of metastatic awakening may lead in the future to treatment strategies designed to prevent such awakening and resulting metastatic relapse.
 Citation Format: Jingwei Zhang, Robert Weinberg. Awakened dormant cancer cells undergo highly mesenchymal to quasi- mesenchymal transition and acquire stemness [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8beb7f3a092f66cbe81343c9f59c0b804bcf8a" target='_blank'>
              Abstract C018: Awakened dormant cancer cells undergo highly mesenchymal to quasi- mesenchymal transition and acquire stemness
              </a>
            </td>
          <td>
            Jingwei Zhang, Robert Weinberg
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed731d817cd7c367b8f1abe8184ac2fe55d110b6" target='_blank'>
              Mapping the developmental trajectory of human astrocytes reveals divergence in glioblastoma.
              </a>
            </td>
          <td>
            Caitlin Sojka, Hsiao-Lin V. Wang, Tarun N. Bhatia, Yangping Li, Pankaj Chopra, Anson Sing, Anna Voss, Alexia King, Feng Wang, Kevin Joseph, V. Ravi, Jeffrey Olson, Kimberly Hoang, E. Nduom, Victor G. Corces, Bing Yao, Steven A. Sloan
          </td>
          <td>2025-01-08</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In this study, we delve into the intrinsic mechanisms of cell communication in hepatocellular carcinoma (HCC). Initially, employing single-cell sequencing, we analyze multiple malignant cell subpopulations and cancer-associated fibroblast (CAF) subpopulations, revealing their interplay through receptor-ligand interactions, with a particular focus on SPP1. Subsequently, employing unsupervised clustering analysis, we delineate two clusters, C1 and C2, and compare their infiltration characteristics using various tools and metrics, uncovering heightened cytotoxicity and overall invasion abundance in C1. Furthermore, our gene risk scoring model indicates heightened activity of the immune therapeutic pathway in C1. Lastly, employing a formulated scoring system, we stratify patients into high and low-risk groups, revealing notably poorer outcomes in the high-risk cohort on Kaplan-Meier curves. Risk scores exhibit a negative correlation with model genes and immune cell infiltration scores, indicating poor prognosis in the high-risk group. Further characterization elucidates the regulatory landscape of the high and low-risk groups across various signaling pathways. In addition, we used wet lab experiments to prove that ABCA1 plays a pro-oncogenic role in hepatocellular carcinoma cells by promoting proliferation, invasion, migration, and reducing apoptosis. In summary, these findings provide crucial insights, offering valuable clues and references for understanding HCC pathogenesis and patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ebd5bca77500ddcf823c450cf7ed50074d3c6d" target='_blank'>
              ABCA1 promote tumor environment heterogeneity via epithelial mesenchymal transition in Huh7 and HepG2 liver cancer cell
              </a>
            </td>
          <td>
            Dinglai Yu, Fang Guo, Qiyu Zhang, Huajun Yu, Wenmin Wang, Yunzhi Chen
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) is characterized by recurrent somatic mutations in epigenetic regulators, which stratify patients into clinically significant subgroups with distinct prognoses and treatment responses. However, the cell type-specific epigenetic landscape of RCC—broadly and in the context of these mutations—is incompletely understood. To investigate these open questions, we integrated single nucleus ATAC sequencing data from RCC tumors across four independent cohorts. In clear cell RCC tumors, we identified four shared malignant epigenetic programs related to angiogenesis, proximal tubule-like features, interferon (IFN) signaling, and one that lacked distinct genomic regions with increased accessibility. Among the mutated epigenetic regulators, BAP1 mutation exhibited the most significant impact on chromatin accessibility in tumor cells, and the associated epigenetic changes were linked to IFN response. We identify multiple potential sources of elevated IFN signaling in these lesions, such as increased immune infiltration and increased accessibility and expression of an IFN-associated ERV, ERV3-16A3_LTR. We find that the expression of ERV3-16A3_LTR may itself be a negative prognostic biomarker in ccRCC. Our findings highlight the convergence of malignant epigenetic programs across ccRCC tumors and suggest that BAP1 loss, potentially through ERV3-16A3_LTR dysregulation, is associated with an IFN response-high epigenetic program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f60a362700f5a511019adc2819b352bb15feca9" target='_blank'>
              Single-cell epigenetic profiling reveals an interferon response-high program associated with BAP1 deficiency in kidney cancer
              </a>
            </td>
          <td>
            Sabrina Y. Camp, M. X. He, Michael S. Cuoco, E. Saad, Erica Pimenta, Kevin Meli, Z. Bakouny, C. Labaki, Breanna M. Titchen, Yun Jee Kang, Jack Horst, Rachel Trowbridge, Erin Shannon, K. Helvie, A. Thorner, S. Vigneau, Angie Mayorga, Jahnavi Kodali, Hannah Lachmayr, Meredith Bemus, Jihye Park, T. Choueiri, Kevin Bi, E. V. Van Allen
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f22cb74c78b53ff77c895d167785c4abf72ee807" target='_blank'>
              Colorectal cancer patients-derived immunity-organoid platform unveils cancer-specific tissue markers associated with immunotherapy resistance
              </a>
            </td>
          <td>
            A. Esposito, A. Agostini, G. Quero, G. Piro, L. Priori, A. Caggiano, G. Scaglione, A. Battaglia, M. Calegari, L. Salvatore, M. Bensi, M. Maratta, A. Ceccarelli, G. Trovato, G. Genovese, E. Gurreri, S. Ascrizzi, M. Martini, C. Fiorillo, A. Fattorossi, F. De Sanctis, S. Ugel, V. Corbo, S. Alfieri, G. Tortora, C. Carbone
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Mucinous colorectal carcinoma (MUC CRC) dissemination into the tumor stroma and metastasis to multiple organs, including the peritoneum, is associated with poor prognosis. Disseminating MUC CRCs exhibit either a conventional ‘apical-in’ or an inverted ‘apical-out’ polarity phenotype that influence patient outcome. Identifying the mechanisms controlling MUC CRC polarity is critical to understand disease progression. Here, we analyze patient-derived MUC CRC xenografts, with apical-in or apical-out polarity, ex vivo or within collagen gels to mimic the peritumoral stroma. Single-cell analyses reveal α2β1-integrin as a key collagen-binding receptor in these models. Collagen–α2β1-integrin interaction activates Src and ERK/MAPK signaling and upregulates the expression of SorLA, an endosomal sorting receptor. SorLA supports apical-in polarity and carcinoma-stroma interactions by promoting integrin recycling to the plasma membrane and HER2/HER3 expression through a positive feedback mechanism. Accordingly, we observe positive correlation between HER2, HER3 and SorLA in patient samples with the highest HER2 expression in apical-in-presenting tissues. Treatment of tumor spheres with clinically relevant HER2/HER3-targeting antibodies reverts sphere polarity and impedes collagen remodeling and adhesion to mouse peritoneum. This SorLA—integrin—HER2/HER3 signaling axis may represent a basis for MUC CRC-patient stratification and shed light on other carcinomas with similar apical-out phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/284740ed86720e8dd61c88e27e9b05faf5479ca5" target='_blank'>
              Signaling downstream of tumor-stroma interaction regulates mucinous colorectal adenocarcinoma apicobasal polarity
              </a>
            </td>
          <td>
            Nicolas Pasquier, A. Isomursu, Hellyeh Hamidi, J. Mathieu, Jouni Härkönen, Gautier Follain, Christophe Desterke, Zoé Fusilier, Junel Solis, Irina Belaya, Pasi Kankaanpää, Valeria Barresi, J. Cartry, Sabrina Bedja, Fanny Jaulin, Johanna Ivaska
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer stem cells aggregate to form clusters, which have enhanced stem-like properties and metastasis potential. However, the molecular mechanisms underlying the formation of cancer stem cell cluster-like structures with acquisition of stronger invasion and metastasis abilities remain unclear. Micropapillary carcinoma (MPC) is a subpopulation of small, merulioid, inverted, nonfibrous vascular clusters floating in the stroma present in a range of solid malignant tumors and characterized by frequent vascular/lymphatic vessel invasion and lymph node metastasis. Our results showed that these cell clusters exhibit a stem cell phenotype, supporting the premise that MPC may serve as a promising solid tumor model for studying invasion and metastasis of cancer stem cell clusters. In this review, we discuss the latest advances in MPC research and targeted therapy, focusing on analysis of their stem-like characteristics, mapping their multiomics characteristics, and elucidating the vascular and immune microenvironment of MPC. The existing MPC organoid model was employed to explore potential breakthroughs in targeted therapy and immunotherapy for cancer stem cell clusters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/465f3c6afb9220f958e9f34e5f679f5bbb6538be" target='_blank'>
              Micropapillary structure: A natural tumor collective invasion model with enhanced stem-like properties.
              </a>
            </td>
          <td>
            Sisi Li, Shuangshu Gao, Ling Qin, Caixia Ding, Jinghui Qu, Yifei Cui, Lixia Qiang, Shengjie Yin, Xiaoyu Zheng, Hongxue Meng
          </td>
          <td>2024-11-20</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Ovarian cancer is one of the deadliest gynecologic cancers affecting the female reproductive tract. This is largely attributed to frequent recurrence and development of resistance to the platinum-based drugs cisplatin and carboplatin. One of the major contributing factors to increased cancer progression and resistance to chemotherapy is the tumor microenvironment (TME). Extracellular signaling from cells within the microenvironment heavily influences progression and drug resistance in ovarian cancer. This is frequently done through metabolic reprogramming, the process where cancer cells switch between biochemical pathways to increase their chances of survival and proliferation. Here, we focus on how crosstalk between components of the TME and the tumor promotes resistance to platinum-based chemotherapy. We highlight the role of cancer-associated fibroblasts, immune cells, adipocytes, and endothelial cells in ovarian tumor progression, invasion, metastasis, and chemoresistance. We also highlight recent advancements in targeting components of the TME as a novel therapeutic avenue to combat chemoresistance in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/269e750ab86f78d7891a1cfc60747cc01af21929" target='_blank'>
              Ovarian tumor microenvironment contributes to tumor progression and chemoresistance
              </a>
            </td>
          <td>
            Adriana L. Ponton-Almodovar, Samuel Sanderson, Ramandeep Rattan, Jamie J. Bernard, S. Horibata
          </td>
          <td>2024-12-17</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Introduction: Survival rates in advanced head and neck cancer (HNC) remain poor with no clinically approved biomarkers to predict treatment efficacy. HNC is a highly heterogenous disease and the lack of available models that capture patient heterogeneity and the complexity of the tumor microenvironment (TME) contributes to these poor patient outcomes. Microphysiological systems (MPS) are complex multicellular in vitro models that can recapitulate three-dimensional organ structure and function. The low cell number requirements for MPS permits the creation of patient-specific TME models, where all the cell types can be isolated from a single piece of tumor tissue. These patient-specific MPS represent a promising strategy for elucidating the contribution of the TME to treatment resistance, predicting the best treatment for each patient and improving patient outcomes in HNC. Here, we report a patient-specific MPS that contains multiple TME compartments and recapitulates patient heterogeneity. Methods: Tumor tissue was collected from patients with oral cavity carcinoma undergoing standard of care surgery followed by radiation or chemoradiation. Tissue from each patient was digested and tumor infiltrating leukocytes (TIL) were isolated using CD45 magnetic beads, while epithelial cells and fibroblasts were expanded in culture using specific media formulations. Tissue was also snap frozen for single cell RNA sequencing analysis. To build the MPS, a custom designed microfluidic device was filled with a hydrogel matrix containing patient-matched TIL, fibroblasts and epithelial spheroids to mimic the solid tumor. Lumens molded from the hydrogel were seeded with blood or lymphatic endothelial cells to create vasculature. Multiple replicate MPS can be built from each patient, which are cultured in an optimized media formulation to support the multiple primary cell types. Patient-specific MPS were treated with radiation or cisplatin + radiation to mimic the treatment the patient received. Multiple orthogonal endpoints were measured including tumor cell migration and viability, cytokine secretion and angiogenesis. Single cell sequencing was also performed on the original tumor tissue and matched MPS. Results: We have successfully created patient-specific models of the HNC TME. Single cell RNA sequencing analysis of the MPS suggests that cellular heterogeneity is being retained. Patient characteristics varied between MPS with some patient TME MPS showing highly migratory tumor cells or angiogenic responses. Treatment responses were measured for each patient-specific MPS and ongoing work will characterize the contributions of the TME to treatment resistance. These data demonstrate the feasibility of using patient-specific MPS to investigate the contribution of the TME to treatment response and resistance and lays important groundwork for using MPS to identify patient treatment responses.
 Citation Format: Adeel Ahmed, Marcos Lares, Fauzan Ahmed, Adam R. Burr, Paul M. Harari, David J. Beebe, Sheena C. Kerr. Engineering the tumor microenvironment: Patient-specific microphysiological systems of the head and neck cancer tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5867639a61561e0c0780d2ce0be172eff5b37091" target='_blank'>
              Abstract B034: Engineering the tumor microenvironment: Patient-specific microphysiological systems of the head and neck cancer tumor microenvironment
              </a>
            </td>
          <td>
            Adeel Ahmed, Marcos Lares, Fauzan Ahmed, Adam R. Burr, Paul M. Harari, David J. Beebe, S. Kerr
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde0e9abf6b50983ec6394f87b77c188d773d609" target='_blank'>
              YAP-driven malignant reprogramming of oral epithelial stem cells at single cell resolution
              </a>
            </td>
          <td>
            F. Faraji, Sydney I. Ramirez, Lauren M. Clubb, Kuniaki Sato, Valeria Burghi, Thomas S. Hoang, Adam Officer, Paola Y. Anguiano Quiroz, William M. G. Galloway, Zbigniew Mikulski, Kate Medetgul-Ernar, P. Marangoni, Kyle B Jones, Yuwei Cao, A. Molinolo, Kenneth Kim, Kanako Sakaguchi, Joseph A. Califano, Quinton Smith, Alon Goren, Ophir D. Klein, Pablo Tamayo, J. S. Gutkind
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="
 Disseminated cancer cells (DCCs) arriving to a new organ may either proliferate and form metastasis, get eliminated by immune cells, or enter a quiescent stage. The factors that determine the DCCs’ fate, including getting quiescent DCCs to re-initiate proliferation and successfully evade the immune system, remain poorly understood, in part due to the lack of robust mouse models. We established a pancreatic ductal adenocarcinoma (PDAC) mouse model of dormant DCCs in the liver. An “immunization protocol” was used to mimic PDAC resection in patients: KPC1199 PDAC cells were injected to establish primary tumors and allow an adaptive immune response to develop against the cancer cells; the tumors were surgically resected; and new KPC1199 cells were intrasplenically injected to establish experimental liver metastases. The overwhelming majority of the KPC1199 cells were eliminated. No metastases spontaneously formed during 20 months of observation, but about 3-8 sporadic, single DCCs were found per liver cross section, and 94% of these DCCs were negative for proliferation markers. Although the model is based on entraining an adaptive immune response, T cell depletion resulted in only 25-30% of mice developing just 1-3 metastases each. This suggests that T cell depletion results in metastasis from the small percentage of DCCs that were proliferating. The DCC-hosting mice therefore represents of model to identify factors that can trigger further DCC proliferation and metastatic recurrence. Patients with PDAC often experience significant stress, leading to excess of endogenous glucocorticoids (GCs). Additionally, synthetic GCs are used during e.g., chemotherapy to manage side effects. To mimic elevated GC levels in patients, we treated DCC-hosting mice with synthetic GCs resulting in an increased percentage of proliferating DCCs (increasing from <6% of the examined cells in control mice to about 36%). Yet, no metastases formed. GC treatment also decreased T cell and increased neutrophil liver infiltration. Neutrophils can form protease-containing neutrophil extracellular traps (NETs), which we previously showed trigger DCC proliferation by proteolytic remodeling of the extracellular matrix protein laminin. We found that neutrophils from GC-treated mice spontaneously formed more NETs, and additionally, NET-generated laminin remodeling was observed in the liver metastasis. Together, the data suggest, that GC treatment drive quiescent DCCs to proliferate via neutrophils but alone this is insufficient to cause metastases. Consistent with this interpretation, T cell depletion combined with GC treatment resulted in multiple metastases in all mice examined. Of note, deleting the GC receptor in the DCCs did not reduce metastases, supporting that GCs act on the host and not the cancer cells. In conclusion, our data suggest that for quiescent DCCs to form metastases, they need signals that trigger proliferation and simultaneously, they must overcome immune surveillance. Elevated GC levels or GC treatment in immune suppressed individuals represents such a scenario.
 Citation Format: Xiao Han, Xueyan He, Yuan Gao, Sicheng Pan, Mikala Egeblad. Glucocorticoids establish a metastatic-promoting tumor microenvironment in a PDAC mouse model [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5174ee47dc358614c322b5604295e8de3a8bcc9a" target='_blank'>
              Abstract C004: Glucocorticoids establish a metastatic-promoting tumor microenvironment in a PDAC mouse model
              </a>
            </td>
          <td>
            Xiao Han, Xueyan He, Yuan Gao, Sicheng Pan, M. Egeblad
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="ABSTRACT Background Metastasis is the major cause of cancer‐related mortality. The premetastatic niche is a promising target for its prevention. However, the generality and cellular dynamics in premetastatic niche formation have remained unclear. Aims This study aimed to elucidate the generality and cellular dynamics in premetastatic niche formation. Materials and Methods We performed comprehensive flow cytometric analysis of lung and peripheral immune cells at three time points (early premetastatic, late premetastatic, and micrometastatic phases) for mice with subcutaneous implants of three types of cancer cells (breast cancer, lung cancer, or melanoma cells). The immuno‐cell profiles were then used to predict the metastatic phase by machine learning. Results We found a common pattern of changes in both lung and peripheral immune cell profiles across the three cancer types, including a decrease in the proportion of eosinophils in the early premetastatic phase, an increase in that of regulatory T cells in the late premetastatic phase, and an increase in that of polymorphonuclear myeloid‐derived suppressor cells and a decrease in that of B cells in the micrometastatic phase. Machine learning using immune cell profiles could predict the metastatic phase with approximately 75% accuracy. Discussion Validation of our findings in humans will require data on the presence or absence of micrometastases in patients and the accumulation of comprehensive and temporal information on immune cells. In addition, blood proteins, extracellular vesicles, DNA, RNA, or metabolites may be useful for more accurate prediction. Conclusion The discovery of generalities in premetastatic niche formation allow prediction of metastatic phase and provide a basis for the development of methods for early detection and prevention of cancer metastasis in a cancer type‐independent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f02612a5b53055050e50dcce6e173f615e1f3275" target='_blank'>
              Immune Cell Profiling Reveals a Common Pattern in Premetastatic Niche Formation Across Various Cancer Types
              </a>
            </td>
          <td>
            Shigeaki Sugiyama, Kanae Yumimoto, Keiichi I Nakayama
          </td>
          <td>2024-12-30</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a multifactorial and heterogeneous disease, ranking among the most prevalent malignancies in men. In 2020, there were 1,414,259 new cases of PCa worldwide, accounting for 7.3% of all malignant tumors. The incidence rate of PCa ranks third, following breast cancer and lung cancer. Patients diagnosed with high-grade PCa frequently present with existing or developing metastases, complicating their treatment and resulting in poorer prognoses, particularly for those with bone metastases. Utilizing single-cell RNA sequencing (scRNA-seq), we identified specific malignant cell subtypes that are closely linked to high-grade PCa. By investigating the mechanisms that govern interactions within the tumor microenvironment (TME), we aim to offer new theoretical insights that can enhance the prevention, diagnosis, and treatment of PCa, ultimately striving to improve patient outcomes and quality of life. Methods Data on scRNA-seq was obtained from the GEO database. The gene ontology and gene set enrichment analysis were employed to analyze differential expression genes. Using inferCNV analysis to identify malignant epithelial cells. We subsequently employed Monocle, Cytotrace, and Slingshot packages to infer subtype differentiation trajectories. The cellular communication between malignant cell subtypes and other cells was predicted using the CellChat package. Furthermore, we employed pySCENIC to analyze and identify the regulatory networks of transcription factors (TFs) in malignant cell subtypes. The MDA PCa 2b and VCap cell lines were employed to validate the analysis results through cellular functional experiments. In addition, a risk scoring model was developed to assess the variation in clinical characteristics, prognosis, immune infiltration, immune checkpoint, and drug sensitivity. Results A malignant cell subtype in PCa with high expression of NEFH was identified through scRNA-seq analysis. This subtype was situated at the differentiation terminal, exhibited a higher level of malignancy, and exhibited characteristics that were more prone to advanced tumor lesions. In addition, our research underscored the intricate interactions that exist within the TME, particularly the interaction between PTN secreted by this subtype and fibroblasts via the NCL receptor. This interaction may be closely associated with cancer-associated fibroblasts and tumor progression. Subsequently, we determined that the NEFH+ malignant cell subtype was significantly correlated with the TF IRX4. This TF is linked to a worse prognosis in PCa and may affect disease progression by regulating gene transcription. Our conclusions were additionally verified through cellular experiments. Furthermore, the prognostic model we developed demonstrated satisfactory predictive performance, with gene sets from the high NmRS group facilitating tumor progression and deterioration. The analysis of immune infiltration was instrumental in the development of clinical intervention strategies and patient prognosis. Conclusion By examining the cellular heterogeneity of a unique NEFH+ malignant cell subtype within the PCa microenvironment, we were able to disclose their reciprocal interaction with disease progression. This offers a novel viewpoint on the diagnosis and treatment of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/850edd1c421faf9a184e4ab73c61663fd59239af" target='_blank'>
              Unveiling the NEFH+ malignant cell subtype: Insights from single-cell RNA sequencing in prostate cancer progression and tumor microenvironment interactions
              </a>
            </td>
          <td>
            Jieya Wang, Fu Zhao, Qiang Zhang, Zhoujie Sun, Zhikai Xiahou, Changzhong Wang, Yan Liu, Zongze Yu
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Epithelial cancers are typically heterogeneous with primary prostate cancer being a typical example of histological and genomic variation. Prior studies of primary prostate cancer tumour genetics revealed extensive inter and intra-patient genomic tumour heterogeneity. Recent advances in machine learning have enabled the inference of ground-truth genomic single-nucleotide and copy number variant status from transcript data. While these inferred SNV and CNV states can be used to resolve clonal phylogenies, however, it is still unknown how faithfully transcript-based tumour phylogenies reconstruct ground truth DNA-based tumour phylogenies. We sought to study the accuracy of inferred-transcript to recapitulate DNA-based tumour phylogenies. We first performed in-silico comparisons of inferred and directly resolved SNV and CNV status, from single cancer cells, from three different cell lines. We found that inferred SNV phylogenies accurately recapitulate DNA phylogenies (entanglement = 0.097). We observed similar results in iCNV and CNV based phylogenies (entanglement = 0.11). Analysis of published prostate cancer DNA phylogenies and inferred CNV, SNV and transcript based phylogenies demonstrated phylogenetic concordance. Finally, a comparison of pseudo-bulked spatial transcriptomic data to adjacent sections with WGS data also demonstrated recapitulation of ground truth (entanglement = 0.35). These results suggest that transcript-based inferred phylogenies recapitulate conventional genomic phylogenies. Further work will need to be done to increase accuracy, genomic, and spatial resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79d56f0a1a843a737c66bbddbc1e662cc71b163e" target='_blank'>
              Clonal phylogenies inferred from bulk, single cell, and spatial transcriptomic analysis of epithelial cancers
              </a>
            </td>
          <td>
            Andrew Erickson, Sandy Figiel, Timothy Rajakumar, Srinivasa Rao, Wencheng Yin, D. Doultsinos, Anette Magnussen, Reema Singh, Ninu Poulose, Richard J. Bryant, Olivier Cussenot, Freddie C Hamdy, Dan Woodcock, I. Mills, Alastair D. Lamb
          </td>
          <td>2025-01-03</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55719070e9f1cff38fbdc568fedbec8bc7d69e98" target='_blank'>
              Single-cell transcriptomic and spatial analysis reveal the immunosuppressive microenvironment in relapsed/refractory angioimmunoblastic T-cell lymphoma
              </a>
            </td>
          <td>
            Mengyan Zhu, Ning Li, Lei Fan, Rongrong Wu, Lei Cao, Yimin Ren, Chuanyang Lu, Lishen Zhang, Yun Cai, Yuzhu Shi, Zihan Lin, Xueying Lu, Jiayan Leng, Shiyang Zhong, Xingfei Hu, Bin Huang, Runheng Huang, Wanting Zhou, Diru Yao, Lingxiang Wu, Wei Wu, Quanzhong Liu, Peng Xia, Ruize Chen, W. Shi, Ruohan Zhang, Sali Lv, Chunling Wang, Liang Yu, Jianyong Li, Qianghu Wang, Kening Li, H. Jin
          </td>
          <td>2024-12-01</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/456d5e0f44d3a56861aef434239933c23d1853a0" target='_blank'>
              Single-cell RNA sequencing analysis reveals the dynamic changes in the tumor microenvironment during NMIBC recurrence.
              </a>
            </td>
          <td>
            Ziang Chen, Tianxiang Zhang, Weijian Li, Jiawei Hu, Y. Ou, Fangdie Ye, Jinhao Zhang, Haowen Jiang, Shenghua Liu
          </td>
          <td>2024-12-04</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Mouse models are the basis of preclinical and translational research in solid tumors. Multiple methods exist to induce tumor formation in mice, including genetically engineered mouse models, chemotoxic agents, injection of tumor cells, and xenograft approaches. Synovial sarcoma, a rare highly malignant soft-tissue tumor, poses a serious threat to patients due to its aggressive nature and associated high rates of mortality and morbidity. This disease's underlying cause and development remain poorly comprehended, and existing treatment approaches have yielded unsatisfactory outcomes. In the last two decades, numerous effective animal models of soft tissue sarcoma (STS) have been created, primarily utilizing genetically engineered mice and zebrafish, however long latency of tumor development for several months in these models creates a barrier for practical in-vivo experiments for drug testing or manipulating the cellular composition of the tumor microenvironment. In studying cancer immunotherapy, it is important to consider aspects of antitumor immune responses and to produce a model that mimics the complexity of the immune system We have generated a new mouse model of synovial sarcoma of exceptionally fast tumor development. Intramuscular delivery of Cre recombinase protein in mice which drives conditional knockdown in PTEN, stabilization of β-catenin, and formation of fusion oncogenes of SS18-SSX2 was sufficient to initiate high-grade synovial sarcomas with myofibroblastic differentiation in 3 weeks. As immunotherapy is increasingly applied to sarcomas, our mouse model will serve as a great tool for preclinical data. Flow cytometry-based analysis of peripheral blood shows significant changes in immune cell populations as early as five days after Tat-Cre injection and continues throughout the development of synovial sarcoma. These results suggest a significant systemic phenotypic alteration of circulating immune cells, which may impact other major organs in this genetic mouse model of synovial sarcoma. Single- cell RNA sequencing of the CD45+ leukocytes in the tumor microenvironment reveals subsets that may promote tumor progression. Our work promises to significantly enhance our understanding of the pathogenesis of this disease and establish a reliable preclinical platform for devising and assessing therapeutic interventions.
 Citation Format: Hesham Mohei, Donn Van Deren, Irene S. Molina, Malay Haldar. New insights from a novel mouse model of synovial sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758631257ad8cc801e48894909b984af2231852b" target='_blank'>
              Abstract C008: New insights from a novel mouse model of synovial sarcoma
              </a>
            </td>
          <td>
            Hesham Mohei, Donn Van Deren, Irene S. Molina, Malay Haldar
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epithelial-to-mesenchymal transition (EMT) is known to play roles in orchestrating cellular plasticity across many physiological and pathological contexts. Partial EMT, wherein cells maintain both epithelial and mesenchymal features, is gaining recognition for its functional importance in cancer in recent years. There are many factors regulating both partial and full EMT, and the precise mechanisms underlying these processes vary depending on the biological context. Furthermore, how different EMT states cooperate to create a heterogeneous tumor population and promote different pro-malignant features remains largely undefined. In a recent study published in Nature Cancer, Youssef and colleagues described how two disparate EMT programs, active in either organ fibrosis or embryonic development, are utilized within different cells within the same murine mammary tumor model. This work provides mechanistic insight into the development of intratumoral heterogeneity, providing evidence for the cooperation between the two EMT trajectories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/977baba3595719140a67fa21b126b2dbe5492ad6" target='_blank'>
              Tumor heterogeneity and cooperating cancer hallmarks driven by divergent EMT programs.
              </a>
            </td>
          <td>
            Phoebe Carter, Yibin Kang
          </td>
          <td>2024-11-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a common malignant tumor of the digestive system with a high incidence that seriously threatens patients’ lives and health. However, with the rise and application of new treatments, such as immunotherapy, there are still some restrictions in the treatment and diagnosis of HCC, and the therapeutic effects on patients are not ideal.Two single-cell RNA sequencing (scRNA-seq) datasets from HCC patients, encompassing 25,189 cells, were analyzed in the study. We utilized non-negative matrix factorization (NMF) clustering to identify mitophagy patterns in HCC TME cells, including cancer-associated fibroblasts (CAFs), T cells, B cells, and tumor-associated macrophages (TAMs). Cell-to-cell communication was analyzed using the CellChat package, and pseudotime trajectory analysis was performed using the Monocle package. Gene regulatory networks were investigated with the SCENIC package, and survival analyses were conducted with mitophagy-related signatures.HCC samples analysis identified 22 clusters, including 7 principal cell types. Complex cell communications were observed among these cell types. Mitophagy-related CAFs, TAMs, CD8+ T cells, and B cells were identified. These subtypes had different biological states, cell-cell communications, and metabolic pathways. Mitophagy levels were elevated in tumor samples. Changes in mitophagy-related genes within specific cell subtypes were associated with different overall survival rates. However, mitophagy did not seem to affect the effectiveness of immunotherapy.This study provides evidence that mitophagy within the HCC TME modulates intercellular communication, influencing tumor progression and patient prognosis. Targeting mitophagy may offer a promising approach to improve the long-term prognosis of HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/689de3b101ffbbea61436b63812b3f0a30320588" target='_blank'>
              Single-cell mitophagy patterns within the tumor microenvironment modulate intercellular communication, impacting the progression and prognosis of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zhengyan Li, Wei Chen, Shu Yao, Zuxiang Peng, Hongming Liu, Yongliang Tang, Yi Feng
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Breast cancer is a heterogeneous malignant disease with a varying prognosis and is classified into four molecular subtypes. It remains one of the most prevalent cancers globally, with the tumor microenvironment playing a critical role in disease progression and patient outcomes. Methods: This study evaluated tumor samples from 40 female patients with luminal A and B breast cancer, utilizing flow cytometry to phenotypically characterize the immune cells and tumor cells present within the tumor tissue. Results: The luminal B-like tumor samples exhibited increased infiltration of CD4+ cells, regulatory T cells (Tregs), and Th17 cells and decreased levels of NK cells, γδ T cells, Th1 cells, and follicular T cells, which is indicative of a more immunosuppressive tumor microenvironment. Conclusions: These findings suggest that luminal B-like tumors have a microenvironment that is less supportive of effective anti-tumor immune responses compared to luminal A tumors. This study enhances the understanding of the immunological differences between luminal subtypes of breast cancer and identifies potential new therapeutic targets and biomarkers that could drive advancements in precision medicine for breast cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbe73bb00ad46a49a657de990032b43dbbcf89b6" target='_blank'>
              Early-Stage Luminal B-like Breast Cancer Exhibits a More Immunosuppressive Tumor Microenvironment than Luminal A-like Breast Cancer
              </a>
            </td>
          <td>
            Tânia Moura, Olga Caramelo, Isabel Silva, Sandra Silva, Manuela Gonçalo, M. A. Portilha, João N. Moreira, Ana M. Gil, P. Laranjeira, Artur Paiva
          </td>
          <td>2025-01-07</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ef6a160eddfbbad312082d26af8e03b167d27f" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance.
              </a>
            </td>
          <td>
            Michael J Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, Sarah R. Amend, James B. Hicks
          </td>
          <td>2024-11-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2461a2661414e1e6ff8b37117c56d2cfd9c5c7b" target='_blank'>
              Cell hiding in colorectal cancer: correlation with response to chemotherapy in vitro and in vivo
              </a>
            </td>
          <td>
            I. Druzhkova, A. Potapov, N. Ignatova, M. Bugrova, I. Shchechkin, M. Lukina, L. Shimolina, E. Kolesnikova, M. Shirmanova, E. Zagaynova
          </td>
          <td>2024-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77ea6b7a2dc14678d80892bdad28eb8662925eda" target='_blank'>
              Impact of pan-cancer analysis of the exportins family on prognosis, the tumor microenvironment and its potential therapeutic efficacy
              </a>
            </td>
          <td>
            Ying Peng, Youheng Li, Lingmei Wang, Shenglai Lin, Hong Xu
          </td>
          <td>2024-12-21</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer, a prominent global health concern, involves the uncontrolled growth and dissemination of abnormal cells, forming tumors that can infiltrate nearby tissues and metastasize to distant sites. Early and precise tumor detection and treatments are crucial to impede their spread and mitigate disease progression. This study presents the development of a scoring system that harnesses single-cell RNA sequencing (scRNA-seq) data to predict the suitability of solid tumors, specifically glioblastoma (GBM) and lymphoma, for advanced cell therapies such as CAR-T, NK cell, and TIL therapies. By integrating key cellular factorsincluding tumor-infiltrating cells, immunosuppressive cells, and immune escape markersthis system provides a quantitative assessment of a tumors potential responsiveness to these therapies. Validation with patient-derived samples has demonstrated the system's potential to improve precision in therapy selection, offering a promising tool for personalized oncology. This research not only aims to enhance the effectiveness of cell therapies for cancer patients but also lays the groundwork for applying similar methodologies to other cancer types, advancing the integration of precision medicine in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa40086c8127d386e515ff58d140655d2ef491d" target='_blank'>
              A Scoring System that Leverages Single Cell RNA-Sequencing Data to Predict Cell Therapy Outcome in Solid Tumor Treatment
              </a>
            </td>
          <td>
            Yuwen Li
          </td>
          <td>2024-12-26</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8645f518787a0ca20fba12896526944d30b5806b" target='_blank'>
              Development of patient-derived lymphomoids with preserved tumor architecture for lymphoma therapy screening
              </a>
            </td>
          <td>
            A. Santamaria-Martínez, J. Epiney, Divyanshu Srivastava, Daniele Tavernari, Marco Varrone, Dina Milowich, Igor Letovanec, Thorsten Krueger, Rafael Duran, Giovanni Ciriello, A. Cairoli, E. Oricchio
          </td>
          <td>2024-12-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cellular senescence is a state of irreversible cell cycle arrest accompanied by a distinctive inflammatory secretory profile known as the senescence-associated secretory phenotype (SASP). While various biomarkers, such as senescence-associated beta-galactosidase (SA-βgal), EdU incorporation, P21, and P16, are used to identify senescent cells, no single biomarker universally defines cellular senescence, and current methods often fail to address heterogeneity in biomarker expression levels. This study leverages single-cell fluorescence imaging to assess multiple senescence markers including SA-βgal enzymatic activity, P21 and IL-6 expression, and nuclear and cell area, in chemotherapy-induced (mitomycin C) and oxidative stress-induced (D-galactose) senescence models in human fibroblasts. Our findings reveal significant heterogeneity in SA-βgal activity and distinct sub-populations within senescent cells. Nuclear and cell area measurements emerged as robust indicators of cellular senescence, displaying similar variability across individual cells. Importantly, we identified specific nuclear area sub-populations that strongly correlate with IL-6 expression levels, demonstrating a relationship between the heterogeneous expression of senescence biomarkers and the SASP. To address this heterogeneity, we introduced an induction threshold method to more accurately quantify the percentage of cells expressing senescence biomarkers. Furthermore, in both senescence models, we observed that rapamycin, a well-known senomorphic agent, selectively targets specific biomarker-expressing sub-populations. This study underscores the value of assessing cellular heterogeneity in senescence research and provides an improved approach for analysing senescence markers in diverse cellular contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/628d7d0deaaee786490438495d7bcdca1881906e" target='_blank'>
              Single-cell fluorescence imaging reveals heterogeneity in senescence biomarkers and identifies rapamycin-responsive sub-populations
              </a>
            </td>
          <td>
            Vijayraghavan Seshadri, Charmaine Chng, Joel Tyler, Kaveh Baghaei, Yan Wang, Nuri Gueven, Iman Azimi
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d9fe7b7420dce6cba1ac7caf17cba80570e76f1" target='_blank'>
              Single-cell and spatial transcriptomics identify COL6A3 as a prognostic biomarker in undifferentiated pleomorphic sarcoma
              </a>
            </td>
          <td>
            J.C. Klein, Lei Wang, Douglas Strand, Chewlan Lastufka, G. Hosler, G. Hon
          </td>
          <td>2024-11-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer spheroids are spherical, three-dimensional (3D), in vitro assemblies of cancer cells, which are gaining importance as a useful model in cancer behavior studies. Designed to simulate key features of the in vivo tumor microenvironment, spheroids offer reliable insights for drug screening and testing applications. We observed contrasting phenotypes in 3D cervical cancer (CC) cultures. Thus, in this study, we compared the proteomes of 3D and traditional two-dimensional (2D) cultures of CC cell lines, HeLa, SiHa, and C33A. When cultured in in-house poly-(2-hydroxyethyl methacrylate)-coated plates under conditions suitable for 3D spheroid formation, these CC cell lines yielded spheroids exhibiting different features. Proteomic analysis of cells cultured in 2D and 3D cultures revealed similar protein profiles but remarkable differences in the expression levels of some proteins. In SiHa and C33A cells, the upregulation of key proteins required for spheroid formation was insufficient for the formation of compact spheroids. In contrast, HeLa cells could form compact spheroids because they upregulated the proteins, including cadherin-binding, cytoskeleton, and adhesion proteins, necessary for spheroid formation during the remodeling process. Overall, this study unravels the mechanisms underlying the formation of spheroids in the commonly used CC cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/815cc2d468c5e900a469f3a75fe62c8fb5f3a378" target='_blank'>
              Proteomic Analysis Reveals Cadherin, Actin, and Focal Adhesion Molecule-Mediated Formation of Cervical Cancer Spheroids
              </a>
            </td>
          <td>
            P. Molika, K. Leetanaporn, W. Chiangjong, Pongsakorn Choochuen, R. Navakanitworakul
          </td>
          <td>2024-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4843e60a35dd8061e986640d8a344bfa3d97b50" target='_blank'>
              Cancer Clones Revised
              </a>
            </td>
          <td>
            L. Laplane
          </td>
          <td>2024-12-23</td>
          <td>Biological Theory</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da415d9954dd04158cfcfaa83dc1880ca38c0523" target='_blank'>
              Single-cell and spatial transcriptomics reveal SPP1-CD44 signaling drives primary resistance to immune checkpoint inhibitors in RCC
              </a>
            </td>
          <td>
            Junfeng Zhang, Qingyan Peng, Jin Fan, Fuzhong Liu, Hongbo Chen, X. Bi, Shuai Yuan, Wei Jiang, Ting Pan, Kailing Li, Sihai Tan, Peng Chen
          </td>
          <td>2024-12-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c214c85284c0f7c6d4f85e6f9096b58eba34d42" target='_blank'>
              Dissecting tumor transcriptional heterogeneity from single-cell RNA-seq data by generalized binary covariance decomposition.
              </a>
            </td>
          <td>
            Yusha Liu, P. Carbonetto, Jason Willwerscheid, Scott A. Oakes, Kay F Macleod, Matthew Stephens
          </td>
          <td>2025-01-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background: Targeted therapies (e.g., ibrutinib) have markedly improved chronic lymphocytic leukemia (CLL) management; however, ~20% of patients experience disease relapse, suggesting the inadequate depth and durability of these front-line strategies. Moreover, immunotherapeutic success in CLL has been stifled by its pro-tumor microenvironment milieu and low mutational burden, cultivating poor antigenicity and limited ability to generate anti-tumor immunity through adaptive immune cell engagement. Previously, we have demonstrated how a three-carbon-linker spirocyclic dimer (SpiD3) promotes futile activation of the unfolded protein response (UPR) in CLL cells through immense misfolded-protein mimicry, culminating in insurmountable ER stress and programmed CLL cell death. Method: Herein, we used flow cytometry and cell-based assays to capture the kinetics and magnitude of SpiD3-induced damage-associated molecular patterns (DAMPs) in CLL cell lines and primary samples. Result: SpiD3 treatment, in vitro and in vivo, demonstrated the capacity to propagate immunogenic cell death through emissions of classically immunogenic DAMPs (CALR, ATP, HMGB1) and establish a chemotactic gradient for bone marrow-derived dendritic cells. Conclusions: Thus, this study supports future investigation into the relationship between novel therapeutics, manners of cancer cell death, and their contributions to adaptive immune cell engagement as a means for improving anti-cancer therapy in CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6bb6049f43b4446b0fae7febaccea77debb8ea" target='_blank'>
              Immunogenic Cell Death Traits Emitted from Chronic Lymphocytic Leukemia Cells Following Treatment with a Novel Anti-Cancer Agent, SpiD3
              </a>
            </td>
          <td>
            Elizabeth Schmitz, Abigail Ridout, Audrey L. Smith, Alexandria P. Eiken, Sydney A. Skupa, Erin M. Drengler, Sarbjit Singh, Sandeep Rana, A. Natarajan, Dalia El-Gamal
          </td>
          <td>2024-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="NPM1-mutated AML is one of the largest entities in international classification systems of myeloid neoplasms, which are based on integrating morphologic and clinical data with genomic data. Previous research, however, indicates that bulk transcriptomics-based subtyping may improve prognostication and therapy guidance. Here, we characterized the heterogeneity in NPM1-mutated AML by performing single-cell RNA-sequencing and spectral flow cytometry on 16 AML belonging to three distinct subtypes previously identified by bulk transcriptomics. Using single-cell expression profiling we generated a comprehensive atlas of NPM1-mutated AML, collectively reconstituting complete myelopoiesis. The three NPM1-mutated transcriptional subtypes showed consistent differences in the proportions of myeloid cell clusters with distinct patterns in lineage commitment and maturational arrest. In all samples, malignant cells were detected across different myeloid cell clusters, indicating that NPM1-mutated AML are heavily skewed but not fully arrested in myelopoiesis. Same-sample multi-color spectral flow cytometry recapitulated these skewing patterns, indicating that the three NPM1-mutated subtypes can be consistently identified across platforms. Moreover, our analyses highlighted differences in the abundance of rare hematopoietic stem cells suggesting that skewing occurs early in myelopoiesis. To conclude, by harnessing single-cell RNA-sequencing and spectral flow cytometry, we provide a detailed description of three distinct and reproducible patterns in lineage skewing in NPM1-mutated AML that may have potential relevance for prognosis and treatment of patients with NPM1-mutated AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d2c7d3f716516ca564f7acecb6aa3ff7231897" target='_blank'>
              Resolving inter- and intra-patient heterogeneity in NPM1-mutated AML at single-cell resolution
              </a>
            </td>
          <td>
            E. O. Karakaslar, E. Argiro, N. E. Struckman, R. H. Shirali, J. Severens, W. Honders, S. L. Kloet, H. Veelken, M. Reinders, M. Griffioen, E. B. van den Akker
          </td>
          <td>2024-12-11</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Spatial transcriptomics, proteomics, and epigenomics are innovative technologies which offer an unparalleled resolution and wealth of data in understanding and the interpretation of cellular functions and interactions. These techniques allow researchers to investigate gene and protein expressions at an individual cell level, revealing cellular heterogeneity within, for instance, bioengineered tissues and classifying novel and rare cell populations that could be essential for the function of the tissues and in disease processes. It is possible to analyze thousands of cells simultaneously, which gives thorough insights into the transcriptomic view of complex tissues. Spatial transcriptomics combines gene expressions with spatial information, conserving tissue architecture and making the mapping of gene activity across different tissue regions possible. Despite recent advancements in these technologies, they face certain limitations. Single-cell transcriptomics can suffer from technical noise and dropout events, leading to incomplete data. Its applicability has been limited by the complexity of data integration and interpretation, although better resolution and tools for the interpretation of data are developing fast. Spatial proteomics and spatial epigenomics provide data on the distribution of proteins and the gene regulatory aspects in tissues, respectively. The disadvantages of these approaches include rather costly and time-consuming analyses. Nevertheless, combining these techniques promises a more comprehensive understanding of cell function and tissue organization, which can be predicted to be useful in achieving better knowledge of cell guidance in tissue-engineered constructs and a higher quality of tissue technology products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b910cb889d8cf7dd3b82978f6e3916ced3b58d85" target='_blank'>
              Spatial Transcriptomics, Proteomics, and Epigenomics as Tools in Tissue Engineering and Regenerative Medicine
              </a>
            </td>
          <td>
            M. Lammi, C. Qu
          </td>
          <td>2024-12-01</td>
          <td>Bioengineering</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Stromal cells are non‐hematopoietic cells that consist of endothelial cells and various mesenchymal cell populations. The composition of the stromal cell compartment is diverse in different organs. Numerous recent studies demonstrated that the lung environment contains heterogeneous mesenchymal stromal cell populations with distinctive genomic signatures and location preferences. Besides their role in supporting organ structure and remodeling tissue, mesenchymal stromal cells fulfill critical immune functions. These stromal cells show alterations during lung fibrosis and infectious disorders like COVID‐19 or flu infection. To date, their identification and isolation were challenging, and most information about their heterogeneity was derived from scRNAseq data. In this protocol, we describe an isolation, comprehensive flow cytometry assessment, and purification strategy for murine lung stromal cells. The described method is optimized for minimizing cell death while keeping a high level of cell purity. This protocol can be also used for ex‐vivo analysis of these cells in downstream functional assays. © 2024 The Author(s). Current Protocols published by Wiley Periodicals LLC. Basic Protocol 1: Isolation of stromal cells from murine lung tissue Basic Protocol 2: Flow cytometry assessment of lung stromal populations Basic Protocol 3: Purification of lung fibroblastic stromal cells Alternate Protocol: Positive selection of fibroblastic stromal cells">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768210ba3f920bfb05c8ef8764e7bd739b49227" target='_blank'>
              Isolation, Purification, and Comprehensive Flow Cytometry Assessment of Lung Stromal Cells
              </a>
            </td>
          <td>
            Sophia Rottmann, Veronika Lukacs-Kornek
          </td>
          <td>2024-12-01</td>
          <td>Current Protocols</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59e5b41f611d8bd26c1558060c6ef330913ca3b" target='_blank'>
              Pan-cancer drivers of metastasis
              </a>
            </td>
          <td>
            Ryan Lusby, Engin Demirdizen, Mohammed Inayatullah, Paramita Kundu, O. Maiques, Ziyi Zhang, Mikkel G Terp, Victoria Sanz-Moreno, Vijay K Tiwari
          </td>
          <td>2025-01-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62546c0d2b5f34aba7fb009609c59c863f6606f" target='_blank'>
              Immunophenotypic analysis on circulating T cells for early diagnosis of lung cancer
              </a>
            </td>
          <td>
            Sung-Woo Lee, Young Ju Kim, Kyung Na Rho, Saei Jeong, Jeong Eun Noh, Hee-Ok Kim, Hyun-Ju Cho, Ju Sik Yun, In-Jae Oh, Jae-Ho Cho
          </td>
          <td>2024-12-26</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT
Tumor-infiltrating lymphocytes (TILs) are key components of the tumor microenvironment (TME) and serve as prognostic markers for breast cancer. Patients with high TIL infiltration generally experience better clinical outcomes and extended survival compared to those with low TIL infiltration. However, as the TME is highly complex and TIL subtypes perform distinct biological functions, TILs may only provide an approximate indication of tumor immune status, potentially leading to biased prognostic results. Therefore, we reviewed the interactions between immune-infiltrating subtypes and tumor cells throughout the entire TME. By examining the antitumor or protumor effects of each TIL subtype, we aimed to better characterize the tumor immune landscape, offering more accurate and comprehensive insights for guiding triple-negative breast cancer (TNBC) treatment. In addition, this approach could lead to the development of new therapeutic targets, enabling tailored treatment strategies and precision medicine. Accumulating evidence suggests that the intestinal microbiome and its metabolites influence antitumor responses by modulating innate and adaptive immunity, with specific bacteria potentially serving as biomarkers for predicting clinical responses. Various studies have identified microorganisms in breast tissue, previously considered sterile, revealing differences in breast microbial composition between patients with breast cancer and controls, as well as associations between specific breast microorganisms and clinicopathologic features, including immune correlations. The aim of this review was to provide a more comprehensive set of prognostic markers for TNBC and to tap into potential-specific therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e0ddc5858a8a27ce545faa706c8e06b464e3d7" target='_blank'>
              Prognostic value of tumor-infiltrating lymphocyte subtypes and microorganisms in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Yating Shi, Zhi Guo, Qiang Wang, Huan Deng
          </td>
          <td>2024-12-01</td>
          <td>Journal of cancer research and therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The molecular features determining the risk of metachronous metastases in clear cell renal cell carcinoma (ccRCC) are poorly defined. Using a systematic tumor transcriptome deconvolution approach, we investigated the genomic and transcriptomic profiles of 192 ccRCC primary tumors with extended clinical follow-up to identify cancer and stromal cell molecular features associated with metastatic risk. At the genomic level, we identified a significantly higher frequency of copy number loss at 1p31-36 in primary tumors that later progressed with metastases. Tumor transcriptome deconvolution identified significant down-regulation of epithelial cell polarity, including PATJ (1p31), and fatty acid metabolism, including CYP4A11 (1p33), in cancer cells of tumors that developed metastatic progression. We developed and benchmarked a compact 5-feature predictive model (5G) that demonstrated improved accuracy over existing ccRCC gene signatures in the prediction of metachronous metastasis risk. Overall, our study highlights convergent genomic and transcriptomic alterations in chromosome 1p, driving dysregulation of epithelial cell polarity and fatty acid metabolism, as putative novel risk factors of metachronous metastasis in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b85898b20987df3b296ae9761d05ca0764e6d312" target='_blank'>
              Convergent genomic and molecular features predict risk of metachronous metastasis in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            M. Naeini, Mengyuan Pang, Neha Rohatgi, Sinem Kadioglu, Umesh Ghoshdastider, R. DiNatale, Roy Mano, A. Hakimi, A. Skanderup
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Lung adenocarcinoma (LUAD) stands as a formidable challenge in oncology, despite the emergence of immunotherapy as a promising treatment avenue. The dynamic interplay between the immune system and cancer plays a pivotal role in shaping cancer development and progression. Central to this process are neoantigens, tumor-specific antigens recognized by cytotoxic T cells, which trigger the elimination of tumor cells. It has been implicated that the presence of T cells also decreases tumor grade. However, the precise impact of neoantigen expression on LUAD development and progression remains poorly understood. Our lab previously developed a genetically engineered mouse model called the iNversion-INduced Joined neoAntigen (NINJA) system with the ability to induce expression of a fluorescent neoantigen in oncogenic KRAS, P53-null, tdTomato-expressing lung adenocarcinoma cells (KPT-NINJA). Using this system, we track tumor initiation, progression, and advanced LUAD development across 20 weeks in both immune-deficient and -sufficient settings. Further, tumors initiated in this system give rise to heterogeneous (neoantigen+ and -) tumors, enabling the study of T cell-driven tumor selection. Through multiplexed immunofluorescence, we identified that neoantigen+ tumors experience increased immune infiltration (CD3+, CD4+, CD8+, B220+) and reduced volume, ultimately leading to prolonged animal survival. Surprisingly, advanced lung tumors maintain neoantigen+ cells, indicating that endogenous lung T cells are ineffective at clearing neoantigen-expressing tumors. Yet, we detected that infiltrated tumors exhibit reduced p-ERK expression, suggesting that T cells may slow tumor progression through a non-cytotoxic mechanism. Transferring stimulated neoantigen-specific T cells to tumor-bearing mice largely eliminated neoantigen+ tumor cells. All these data together suggest complex mechanisms for the role of endogenous T cells in controlling lung tumors. Through molecular characterization of tumor progression and T cell-mediated tumor suppression, this study reveals specific pathways that LUAD cells undergo during the process of development and progression, providing valuable insights into the immune response against neoantigen-expressing tumors. The characterization of early neoantigen-expressing tumor cells is pivotal in validating its utility for studying cancer immunotherapy and identifying key molecular changes during tumor development. Ultimately, by unraveling the intricate interplay between neoantigen expression, immune surveillance, and tumor progression, this project aims to contribute significantly to our understanding of how neoantigens shape the dynamics of tumor biology.
 Citation Format: Jennifer L. Loza, Ishan Bansal, Brian Hunt, Kelli Connolly, Advait Jeevanandam, Srividhya Venkatesan, Nikhil Joshi. Exploring the impact of neoantigen expression on lung tumor development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B039.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8e042869028a24910b9faaf22e8b12fd3c0b8b" target='_blank'>
              Abstract B039: Exploring the impact of neoantigen expression on lung tumor development
              </a>
            </td>
          <td>
            Jennifer L. Loza, Ishan Bansal, Brian G. Hunt, Kelli A. Connolly, Advait Jeevanandam, Srividhya Venkatesan, Nikhil S. Joshi
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT Background Metastatic colorectal cancer (mCRC) is the main cause of CRC mortality, with limited treatment options. Although immunotherapy has benefited some cancer patients, mCRC typically lacks the molecular features that respond to this treatment. However, recent studies indicate that the immune microenvironment of mCRC may be modified to enhance the effect of immune checkpoint inhibitors. This study aimed to explore the metastatic tumor microenvironment (TME) by comparing cell populations in colorectal liver (CLM), lung (mLu), and peritoneal (PM) metastases. Methods RNA isolated from 20 CLM, 15 mLu, and 35 PM samples was subjected to mRNA sequencing and explored through TME deconvolution tools, consensus molecular subtyping (CMS), and differential gene expression and gene set enrichment analysis, with respect to the metastatic sites. Clinical data and KRAS/BRAF hotspot mutation status were also obtained for all the cases. Results The cell type fractions in the TME were relatively similar between the metastatic sites, except for cancer‐associated fibroblasts (CAFs), B cells, endothelial cells, and CD4+ T cells. Notably, PM showed enrichment for CAFs and endothelial cells, consistent with distinct pathways associated with metastatic growth and progression in the peritoneal cavity. PM with the mesenchymal subtype, CMS4, had increased CAFs, endothelial cells, and macrophages, along with up‐regulated genes related to TNF‐α signaling via NF‐κB, EMT, and angiogenesis. Conclusions Tumor samples from different metastatic sites exhibited a broadly similar TME in terms of immune cell composition, with some intriguing differences. Targeting CAF‐associated pathways, macrophages, and TNF‐α signaling through NR4A could represent potential novel therapeutic approaches in CMS4 PM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e496a24f62076b578b328b6a82488e3686ba6b6" target='_blank'>
              Enrichment of Cancer‐Associated Fibroblasts, Macrophages, and Up‐Regulated TNF‐α Signaling in the Tumor Microenvironment of CMS4 Colorectal Peritoneal Metastasis
              </a>
            </td>
          <td>
            Eirik Høye, Chakravarthi Kanduri, A. Torgunrud, Susanne Lorenz, Bjørn Edwin, S. Larsen, Å. Fretland, V. Dagenborg, K. Flatmark, Christin Lund-Andersen
          </td>
          <td>2024-12-30</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Crosstalk between cancer and the nervous system regulates tumor progression. These interactions occur at both the local (i.e., within the tumor parenchyma) and systemic (e.g., endocrine signaling) levels. More recent work has determined that cancer cells not only interact with their local environment via secreted molecules, but in some cases, via acquiring proteins or organelles from other cells within the tumor microenvironment (TME). Here, we demonstrate that breast cancer cells acquire mitochondria from neurons. Breast cancer cells with higher metastatic potential acquire more mitochondria in NGF dependent manner compared to the cell line with lower metastatic potential. These results suggest that in addition to their known roles in electrochemical and paracrine signaling, tumor-innervating nerves can also enhance cancer cell fitness via transfer of mitochondria. We are now characterizing the mechanisms underlying this transfer, determining the fate of transferred mitochondria in recipient cells, and defining how transferred mitochondria drive cancer development, progression, and metastasis. Further, sensory neurons at the tumor microenvironment dictate the intracellular mitochondria dynamics affecting morphology and function. The change in mitochondria dynamics enhances cancer cell survival and metastasis, we are uncovering how sensory neuron induced mitochondria dynamics drives tumor progression.
 Citation Format: Ankit Tiwari, Yue Wu, Jeremy C. Borniger. Sensory neurons regulate intracellular and intercellular mitochondrial dynamics in breast cancer cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B011.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91bf02ff593af4565e19edc1a02a36729c4d623a" target='_blank'>
              Abstract B011: Sensory neurons regulate intracellular and intercellular mitochondrial dynamics in breast cancer cells
              </a>
            </td>
          <td>
            Ankit Tiwari, Yue Wu, Jeremy C. Borniger
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor growth and progression involve coordinated regulation by internal, microenvironmental, and systemic signals and often display conspicuous sexual dimorphism. The mechanisms governing the integration and coordination of these signals, along with their sex-based differences, remain largely unknown. Using a Drosophila tumor model originating from nonreproductive tissue, we show that female-biased tumor growth involves multifaceted communications among tumor cells, hemocytes, and neuroendocrine insulin-producing cells (IPCs). Notch-active tumor cells recruit hemocytes carrying the tumor necrosis factor–α (TNF-α) homolog Eiger to the tumor microenvironment (TME), activating the c-Jun N-terminal kinase (JNK) pathway in tumor cells, instigating the sexually dimorphic up-regulation of cytokine Unpaired 2 (Upd2). Upd2, in turn, exerts a distal influence by modulating the release of a Drosophila insulin-like peptide (Dilp2) from IPCs. Dilp2 then activates the insulin signaling in the tumor, thereby fostering sexual-dimorphic tumor growth. Together, these findings reveal a relay mechanism involving the TME and systemic signals that collectively control the sexual dimorphism of tumor growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4504e9cb20bc72b2973f3610b4d0ff83ae5829a" target='_blank'>
              Sex-dimorphic tumor growth is regulated by tumor microenvironmental and systemic signals
              </a>
            </td>
          <td>
            Xianfeng Wang, Hongcun Bao, Yi-Chun Huang, Anindita Barua, Chun-Ming Lai, Jie Sun, Youfang Zhou, Fei Cong, Shangyu Gong, Chih-Hsuan Chang, Wu-Min Deng
          </td>
          <td>2024-12-06</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9558944d48747d281f714408c8c756ade188a3" target='_blank'>
              Genetic and epigenetic characterization of sarcoma stem cells across subtypes identifies EZH2 as a therapeutic target
              </a>
            </td>
          <td>
            Edmond O'Donnell, Maria Muñoz, Ryan Davis, Jessica Bergonio, R. L. Randall, Clifford Tepper, Janai R. Carr-Ascher
          </td>
          <td>2025-01-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c426370b1fe4fbd062526de11c77d9b155f25f" target='_blank'>
              Single-cell RNA sequencing reveals immune microenvironment niche transitions during the invasive and metastatic processes of ground-glass nodules and part-solid nodules in lung adenocarcinoma
              </a>
            </td>
          <td>
            Yi-Feng Ren, Qiong Ma, Xiao Zeng, Chun-Xia Huang, Jia-Li Ren, Fang Li, Jia-Jing Tong, Jia-Wei He, Yang Zhong, Shi-Yan Tan, Hua Jiang, Long-Fei Zhang, Heng-zhou Lai, Ping Xiao, Zhuang Xiang, Peng Wu, Li-Ting You, Wei Shi, Xi Fu, Chuan Zheng, Feng-ming You
          </td>
          <td>2024-11-23</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62a0b0314fc694a0d5ac1d3e0908ca758f9f880c" target='_blank'>
              Cancer-associated fibroblasts, tumor and radiotherapy: interactions in the tumor micro-environment
              </a>
            </td>
          <td>
            Kris Raaijmakers, G. Adema, Johan Bussink, M. Ansems
          </td>
          <td>2024-12-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcd5c512a1a95d505528decdc0fed35eccbfc51" target='_blank'>
              Insights from a multiscale framework on metabolic rate variation driving glioblastoma multiforme growth and invasion
              </a>
            </td>
          <td>
            M. Amereh, S. Shojaei, A. Seyfoori, Tavia Walsh, P. Dogra, V. Cristini, B. Nadler, Mohsen Akbari
          </td>
          <td>2024-11-25</td>
          <td>Communications Engineering</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The vast majority of cancers exhibit Somatic Copy Number Alterations (SCNAs)—gains and losses of variable regions of DNA. SCNAs can shape the phenotype of cancer cells, e.g. by increasing their proliferation rates, removing tumor suppressor genes, or immortalizing cells. While many SCNAs are unique to a patient, certain recurring patterns emerge as a result of shared selectional constraints or common mutational processes. To discover such patterns in a robust way, the size of the dataset is essential, which necessitates combining SCNA profiles from different cohorts, a non-trivial task. To achieve this, we developed CNSistent, a Python package for imputation, filtering, consistent segmentation, feature extraction, and visualization of cancer copy number profiles from heterogeneous datasets. We demonstrate the utility of CNSistent by applying it to the publicly available TCGA, PCAWG, and TRACERx cohorts. We compare different segmentation and aggregation strategies on cancer type and subtype classification tasks using deep convolutional neural networks. We demonstrate an increase in accuracy over training on individual cohorts and efficient transfer learning between cohorts. Using integrated gradients we investigate lung cancer classification results, highlighting SOX2 amplifications as the dominant copy number alteration in lung squamous cell carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f94ed57d2a2d247c1baada02015edbbd8a64f902" target='_blank'>
              CNSistent integration and feature extraction from somatic copy number profiles
              </a>
            </td>
          <td>
            Adam Streck, Roland F. Schwarz
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neuroblastoma (NB) remains associated with high mortality and low initial response rate, especially for high-risk patients, thus warranting exploration of molecular markers for precision risk classifiers. Through integrating multiomics profiling, we identified a range of hub genes involved in cell cycle and associated with dismal prognosis and malignant cells. Single-cell transcriptome sequencing revealed that a subset of malignant cells, subcluster 1, characterized by high proliferation and dedifferentiation, was strongly correlated with the hub gene signature and orchestrated an immunosuppressive tumor microenvironment (TME). Furthermore, we constructed a robust malignant subcluster 1 related signature (MSRS), which was an independent prognostic factor and superior to other clinical characteristics and published signatures. Besides, TME differences conferred remarkably distinct therapeutic responses between high and low MSRS groups. Notably, polo-like kinase-1 (PLK1) was one of the most crucial contributors to MSRS and remarkably correlated with malignant subcluster 1, and PLK1 inhibition was effective for NB treatment as demonstrated by in silico analysis and in vitro experiments. Overall, our study constructs a novel molecular model to further guide the clinical classification and individualized treatment of NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33949844ac45f5e41b1ac34112dc1705f6129d2" target='_blank'>
              Multiomics Analysis Reveals Neuroblastoma Molecular Signature Predicting Risk Stratification and Tumor Microenvironment Differences.
              </a>
            </td>
          <td>
            Xing Zhou, Zhaokai Zhou, Xiaohan Qin, Jian Cheng, Yongcheng Fu, Yuanyuan Wang, Jingyue Wang, Pan Qin, Da Zhang
          </td>
          <td>2025-01-06</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="During development, cells gradually assume specialized fates via changes of transcriptional dynamics in thousands of genes. Terminal cell identities are then stabilized through the convergence of gene regulatory network dynamics and the accumulation of epigenetic DNA modifications. “Gene-free” (or geometric) modeling approaches for cell fate acquisition, which abstract from the underlying gene regulatory landscape and reason in phenotypic space, have been remarkably successful in explaining terminal fate outcomes. However, their implications for cellular dynamics during fate acquisition processes have so far not been tested in vivo. To do so, here we combine gene-free mathematical modeling of cell fate acquisition during C. elegans vulval development with temporally controlled perturbations of in vivo signaling dynamics using temperature sensitive (ts) mutant alleles of the EGF/Ras/MAPK and Notch signaling pathways. We show that gene-free modeling can quantitatively predict non-intuitive fate outcomes in a variety of ts genetic backgrounds, including pathway epistasis effects. In addition, we use gene-free modeling to infer how cell fate transitions can be guided towards specific outcomes through timed pulses of signaling activity and verify these model predictions quantitatively with temporally controlled signaling perturbations via temperature shifts in ts backgrounds. Our results highlight the predictive power of gene-free models beyond terminal fate outcomes and illustrate a new approach to quantitatively guide cell fate acquisition in a developmental context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a8eb424dfb3d6b3acca350cb9c990cdfc9fbd3" target='_blank'>
              Quantitative guiding of developmental cell fate transitions using gene-free modelling
              </a>
            </td>
          <td>
            Ismail Hajji, Eric D. Siggia, F. Corson, Wolfgang Keil
          </td>
          <td>2024-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4f1962ef1598f22c4d709373f72cc9202dbf33" target='_blank'>
              Mesenchymal stromal cells promote the formation of lung cancer organoids via Kindlin-2
              </a>
            </td>
          <td>
            Zhi-lin Sui, Xianxian Wu, Jiaxin Wang, ShihJye Tan, Chao Zhao, Zhentao Yu, Chuanyue Wu, Xiaoxia Wang, Ling Guo
          </td>
          <td>2025-01-09</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Nongenetic plasticity has emerged as a key driver of cancer drug resistance, yet its precise origins, nature, and consequences are not fully clarified. This review examines technological, computational, and conceptual developments in the nongenetic determinants of drug resistance. We begin by proposing refined definitions of cellular state, fate, and plasticity. We subsequently contextualize the findings from multimodal approaches to investigate plasticity, highlighting how new single-cell lineage-tracing methods provide opportunities for quantitatively capturing state transitions, their timescales and heritability, and how they contribute to resistance mechanisms. We also draw parallels with concepts from developmental biology and microbial persistence research. Next, we cover the role that computational approaches have played in revealing the otherwise latent patterns of heterogeneity that underlie plasticity from complex datasets. We conclude by emphasizing the need for standardized terminology in this rapidly evolving field and the path from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38fadd15ca85570a489d3058bda72d96573ce1d" target='_blank'>
              Navigating Nongenetic Plasticity in Cancer Drug Resistance
              </a>
            </td>
          <td>
            Hanxiao Sun, Nitu Kumari, Madeline E. Melzer, Vadim Backman, Yogesh Goyal
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
The pathogenesis of psoriasis, an inflammatory skin disease, is incompletely understood. Growing evidence substantiates the involvement of stromal cells in the inflammatory process.


OBJECTIVES
To investigate the roles of stromal cells, including fibroblasts, vascular endothelial cells (VECs) and smooth muscle cells (VSMCs), in the psoriatic inflammatory microenvironment and the possible underlying mechanisms involved.


METHODS
This study employed combination of single-cell, spatial transcriptome and bulk RNA sequencing using lesional and nonlesional skin samples from patients with psoriasis vulgaris (PV) and healthy skin samples from unaffected individuals.


RESULTS
Through the analysis of transcriptome from 364,098 single cells, we uncovered WNT5A+ fibroblasts, ITIH5+ VECs and VCAN+ VSMCs with the significantly increased cell proportions in the papillary dermis of lesional skin. We defined eight unique subclusters of fibroblasts in the skin and observed a shift of WIF1+ fibroblasts towards WNT5A+ fibroblasts, with abnormal activation of the non-canonical Wnt signaling pathway and increased capabilities of angiogenesis and pro-inflammatory. For the microvascular cells, VSMCs could undergo phenotypic transformation from a contractile phenotype to a synthetic phenotype in the development of psoriatic inflammation. ITIH5+ VECs and VCAN+ VSMCs were identified with an essential role in regulating angiogenesis and vascular remodeling involved in the mechanism of psoriatic pathological changes. Ligand receptor analyses demonstrated WNT5A+ fibroblasts were extensively implicated in interactions with various cell types in skin, especially with ITIH5+ VECs and VCAN+ VSMCs within the papillary dermis.


CONCLUSIONS
Interactions of stromal cells in the papillary dermis were identified as possible pathogenic elements in psoriasis vulgaris. Improving the inflammatory microenvironment by targeting stromal cells might be a potential treatment strategy for psoriasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1eb8edc16b87584906a5c6fdfa4b1295d5ed3a" target='_blank'>
              Multitranscriptome analysis revealed that stromal cells in the papillary dermis promote angiogenesis in psoriasis vulgaris.
              </a>
            </td>
          <td>
            Bo Zhang, Junpu Mei, Qijun Liao, Shan Zhou, He Huang, Hui Liu, Xiaoli Xu, Yafen Yu, Chao Wu, Wenjun Wang, Weining Hu, Tingting Zhu, Yin Zhang, Mengyun Chen, Caihong Zhu, Mengjun Yu, Jinping Gao, Xianfa Tang, Xiawei Liu, Ze Guo, Xiaodong Zheng, Zhuang Wen, Gang Chen, Lili Tang, Xiaoyan Ding, Hui Cheng, Yang Li, Hongyan Wang, Hui Li, Yangrui Zhang, Xing Fan, Rouxi Chen, Zherou Rong, Ping Liu, Shengxiu Liu, Zhen Yue, Peiguang Wang, Zhiming Cai, Min Gao, Zaixing Wang, Xiaodong Fang, Fusheng Zhou, Huayang Tang
          </td>
          <td>2024-11-21</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="BACKGROUND
Despite recent advances in the biology of IDH-wildtype glioblastoma, it remains a devastating disease with median survival of less than 2 years. However, the molecular underpinnings of the heterogeneous response to the current standard-of-care treatment regimen consisting of maximal safe resection, adjuvant radiation, and chemotherapy with temozolomide remain unknown.


METHODS
Comprehensive histopathologic, genomic, and epigenomic evaluation of paired initial and recurrent glioblastoma specimens from 106 patients was performed to investigate the molecular evolution and cellular phenotypes underlying differential treatment responses.


RESULTS
While TERT promoter mutation and CDKN2A homozygous deletion were early events during gliomagenesis shared by initial and recurrent tumors, most other recurrent genetic alterations (eg, EGFR, PTEN, and NF1) were commonly private to initial or recurrent tumors indicating acquisition later during clonal evolution. Furthermore, glioblastomas exhibited heterogeneous epigenomic evolution with subsets becoming more globally hypermethylated, hypomethylated, or remaining stable. Glioblastoma that underwent sarcomatous transformation had shorter interval to recurrence and were significantly enriched in NF1, TP53, and RB1 alterations and the mesenchymal epigenetic class. Patients who developed somatic hypermutation following temozolomide treatment had significantly longer interval to disease recurrence and prolonged overall survival, and increased methylation at 4 specific CpG sites in the promoter region of MGMT was significantly associated with this development of hypermutation. Finally, an epigenomic evolution signature incorporating change in DNA methylation levels across 347 critical CpG sites was developed that significantly correlated with clinical outcomes.


CONCLUSIONS
Glioblastoma undergoes heterogeneous genetic, epigenetic, and cellular evolution that underlies prognostically different treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8c70c5b87e0a8ebd1138c27ed60e44214d7859" target='_blank'>
              Longitudinal multimodal profiling of IDH-wildtype glioblastoma reveals the molecular evolution and cellular phenotypes underlying prognostically different treatment responses.
              </a>
            </td>
          <td>
            Calixto-Hope G Lucas, N. Al-Adli, Jacob S. Young, Rohit Gupta, R. Morshed, Jasper Wu, Ajay Ravindranathan, Anny Shai, N. Oberheim Bush, Jennie W Taylor, John de Groot, J. Villanueva‐Meyer, M. Pekmezci, Arie Perry, Andrew W Bollen, P. Theodosopoulos, M. Aghi, E. Chang, Shawn L Hervey-Jumper, D. Raleigh, Annette M Molinaro, Joseph F Costello, Aaron A. Diaz, Jennifer L. Clarke, N. Butowski, Joanna J. Phillips, Susan M Chang, Mitchell S. Berger, D. A. Solomon
          </td>
          <td>2024-11-19</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="The study by Grossmann and colleagues uses single-nucleus RNA sequencing in a cohort of matched high-risk neuroblastoma primary tumor samples, obtained from the same patient at diagnosis and definitive surgery, to identify persister cells that survive induction chemotherapy. These persister cells utilize mechanisms of chemoresistance that are both tumor-intrinsic and tumor-extrinsic, are highly dependent on the original genetic profile of the tumor, and represent novel, patient-specific targets to precisely inhibit chemoresistance and disease recurrence. See related article by Grossmann et.al., p. 2387.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17960d164122a139c932c5adc28d8d07f55f0399" target='_blank'>
              Characterization of Persister Cells Provides Insights into Mechanisms of Therapy Resistance in Neuroblastoma.
              </a>
            </td>
          <td>
            Amber B. Wolf, C. P. Reynolds, Eveline Barbieri
          </td>
          <td>2024-12-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Intrapatient tumor heterogeneity contributes to immunotherapy and targeted therapy resistance in cutaneous metastatic melanoma patients. More than 50% of melanoma patients develop brain metastases (MBM) during disease progressionand these patients have a very poor prognosis with a median overall survival of 4 months. To assess how tumors persist through treatments and adapt to and rely on organ-specific microenvironments, we utilized samples from a patient who had received anti-CTLA4, BRAFi/MEKi, and BRAFi/MEKi+CDK4/6i treatments during the LOGIC2 clinical trial. Eight short-term melanoma cultures (STMCs) were established from different metastatic site biopsies at chronological time points throughout the patient’s treatment and subsequent post-mortem resection of the bone, ovary, and brain metastatic sites. While the initial treatment naïve metastatic tumor in the breast was BRAF V600E mutant, NRAS Q61 mutations were acquired after treatment with CDK4/6i in the bone and ovary tumors. We performed single-cell RNA sequencing and identified that each STMC consists of 2-10 different subpopulations. By cell viability assay, we determined all STMCs were resistant to CDK4/6i, however had varying degrees of response to BRAFi and MEKi in vitro. Two STMCs (MM150604, MM150859) were sensitive to both BRAFi and MEKi. We identified HER3 was upregulated in the sensitive STMCs and more specifically, HER3 expression was elevated in the MBM STMC and in other MBM patient tumors in publicly available datasets. In publicly available data of the MBM tumor microenvironment (TME), neurons produce NRG1 and NRG2 while astrocytes and pericytes produce NRG2, exogenous sources of NRGs in the MBM TME that act as ligands for the HER3 receptor. Treatment with recombinant NRG1 partially rescued the growth of MBM cells and restored MAPK and AKT-mTOR signaling in the presence of BRAFi/MEKi suggesting that the brain microenvironment may play a role in drug tolerance. This novel finding demonstrates a unique role of HER3 in MBM and targeting the NRG-HER3 signaling axis could have therapeutic potential in melanoma patients with active brain metastases.
 Citation Format: Haley P. Wilson, Glenn L. Mersky, Jelan I. Haj, Jessica L.F. Teh, Phil F. Cheng, Signe Caksa, Manal Mustafa, Isabella Trachtenberg, Francis J. Waltrich Jr, Casey D. Stefanski, Vivian Chua, Dan A. Erkes, Mitchell P. Levesque, Reinhard Dummer, Timothy J. Purwin, Claudia Capparelli, Andrew E. Aplin. Intrapatient heterogeneity and targeting microenvironment protection mechanisms in melanoma metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C048.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44bfe1a0d7f48e3ce31e054b1efd8507580f7dfc" target='_blank'>
              Abstract C048: Intrapatient heterogeneity and targeting microenvironment protection mechanisms in melanoma metastasis
              </a>
            </td>
          <td>
            Haley P Wilson, Glenn L Mersky, Jelan I Haj, J. Teh, Phil F Cheng, Signe Caksa, Manal U Mustafa, Isabella V Trachtenberg, Francis J. Waltrich, Casey D Stefanski, Vivian Chua, Dan A Erkes, Mitchell Levesque, R. Dummer, Timothy J. Purwin, Claudia Capparelli, Andrew E. Aplin
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cervical cancer is the fourth most common cancer in women globally, and the main cause of the disease has been found to be ongoing HPV infection. Cervical cancer remains the primary cause of cancer-related death despite major improvements in screening and treatment approaches, especially in low- and middle-income nations. Therefore, it is crucial to investigate the tumor microenvironment in advanced cervical cancer in order to identify possible treatment targets.In order to better understand malignant cervical cancer epithelial cells (EPCs), this study used bulk RNA-seq data from UCSC in conjunction with single-cell RNA sequencing data from the ArrayExpress database. After putting quality control procedures into place, cell type identification and clustering analysis using the Seurat software were carried out. To clarify functional pathways, enrichment analysis and differential gene expression were carried out. The CIBERSORT and ESTIMATE R packages were used to evaluate the immune microenvironment characteristics, and univariate and multivariate Cox regression analyses were used to extract prognostic features. Furthermore, assessments of drug sensitivity and functional enrichment were carried out.Eight cell types were identified, with EPCs showing high proliferative and stemness features. Five EPC subpopulations were defined, with C1 NNMT+ CAEPCs driving tumor differentiation. A NNMT CAEPCs Risk Score (NCRS) model was developed, revealing a correlation between elevated NCRS scores and adverse patient outcomes characterized by immune evasion. In vitro experiments validated that the prognostic gene PLOD2 significantly enhances proliferation, migration, and invasion of cervical cancer cells.This investigation delineated eight cell types and five subpopulations of malignant EPCs in cervical cancer, establishing the C1 NNMT+ CAEPCs as a crucial therapeutic target. The NCRS model demonstrated its prognostic capability, indicating that higher scores are associated with poorer clinical outcomes. The validation of PLOD2 as a prognostic gene highlights its therapeutic potential, underscoring the critical need for integrating immunotherapy and targeted treatment strategies to enhance diagnostic and therapeutic approaches in cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a0fda708b879e8e016206b2e92375962d615799" target='_blank'>
              Single-cell RNA sequencing and immune microenvironment analysis reveal PLOD2-driven malignant transformation in cervical cancer
              </a>
            </td>
          <td>
            Zhiheng Lin, Fengxin Wang, Renwu Yin, Shengnan Li, Yuquan Bai, Baofang Zhang, Chenlin Sui, Hengjie Cao, Dune Su, Lianwei Xu, Honghong Wang
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaa35d32ecdf58360fe2158aa64cfa3b17ec1726" target='_blank'>
              Evolving cell states and oncogenic drivers during the progression of IDH-mutant gliomas.
              </a>
            </td>
          <td>
            Jingyi Wu, L. G. Gonzalez Castro, S. Battaglia, C. E. El Farran, J. D’Antonio, Tyler E. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-11-21</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e14091bb37f5209a2b92498963038115c6336e4" target='_blank'>
              Epigenetic dysregulation in glioblastoma: potential pathways to precision medicine.
              </a>
            </td>
          <td>
            Vijeta Prakash, R. Gabrani
          </td>
          <td>2024-11-25</td>
          <td>Neurogenetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="KRAS is the most frequently mutated oncogene in lung adenocarcinoma, with G12C and G12V being the most predominant forms. Recent breakthroughs in KRASG12C inhibitors have transformed the clinical management of patients with G12C mutation and advanced our understanding of its function. However, little is known about the targeted disruption of KRASG12V, partly due to a lack of specific inhibitors. Here, we leverage the degradation tag (dTAG) system to develop a KRASG12V transgenic mouse model. We explore the therapeutic potential of KRASG12V degradation and characterize its impact on the tumor microenvironment (TME). Our study reveals that degrading KRASG12V abolishes lung and pancreatic tumors in mice and causes a robust inhibition of KRAS-regulated cancer intrinsic signaling. Importantly, targeted degradation of KRASG12V reprograms the TME towards a stimulatory milieu and drives antitumor immunity, elicited mainly by effector and cytotoxic CD8+ T cells. Our work provides important insights into the impact of degrading KRASG12V on both tumor progression and immune response, highlighting degraders as a powerful strategy for targeting KRAS mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7557f24725afcb0b1e2df48764b1f6cd0b9ab54" target='_blank'>
              Targeted degradation of oncogenic KRASG12V triggers antitumor immunity in lung cancer models.
              </a>
            </td>
          <td>
            Dezhi Li, Ke Geng, Yuan Hao, Jiajia Gu, Saurav Kumar, Annabel T Olson, Christina C. Kuismi, Hye Mi Kim, Yuanwang Pan, Fiona Sherman, Asia M. Williams, Yiting Li, Fei Li, Ting Chen, C. Thakurdin, Michela Ranieri, Mary Meynardie, Daniel S. Levin, Janaye Stephens, Alison Chafitz, Joy Chen, Mia S Donald-Paladino, Jaylen M Powell, Ze-Yan Zhang, Wei Chen, Magdalena Ploszaj, Han Han, S. Gu, Tinghu Zhang, Baoli Hu, B. Nacev, Medard Ernest Kaiza, Alice H. Berger, Xuerui Wang, Jing Li, Xuejiao Sun, Yang Liu, Xiaoyang Zhang, T. Bruno, N. Gray, Behnam Nabet, Kwok-Kin Wong, Hua Zhang
          </td>
          <td>2024-12-24</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Transformative technologies to sequence tumor genomes at large scale and single-cell resolution have exposed the repertoire of genetic alterations that are present in leukemia genomes, the timing of their acquisition and patterns of their co-occurrence. In parallel, single-cell multi-omics technologies are allowing us to map the differentiation paths and hierarchical structures of malignant cells and giving us a glimpse into hematopoietic development in prenatal life. We propose that interrogating how the genetic evolution, differentiation hierarchy and ontogeny of malignant myeloid cells intersect with each other, using new experimental systems and multimodal technologies, will fuel the next generation of research breakthroughs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f504f6d5de23edcc7a44b61924d8a2be55a1c9e" target='_blank'>
              The crossroads of clonal evolution, differentiation hierarchy and ontogeny in leukemia development.
              </a>
            </td>
          <td>
            Christopher M Sturgeon, Elvin Wagenblast, Franco Izzo, E. Papapetrou
          </td>
          <td>2024-12-09</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 The heterogeneity of ovarian mesenchymal/stromal cells has just been revealed in both mice and humans. However, it remains unclear about the cellular development trace and the intercellular communication network in the whole life of the ovary. In the study, we integrated ours and published single-cell RNA sequencing data from E11.5 (embryonic day 11.5) until M12 (12-month-old) ovaries to show the dynamics of somatic cells along the developmental timeline. The intercellular crosstalk among somatic cell types was depicted with collagen signaling pathway as the most outgoing signals from stromal cells. We identified mesenchymal progenitor cells (CD24+) as the origin of stromal cells. Although their numbers decreased significantly in adults, the cells served as the major signal sender until ovarian senescence. Moreover, the ovarian injury could activate these stem cells and induce stroma remodeling in the aged ovary. Thus, mesenchymal progenitor cells may represent a new strategy to delay ovarian aging in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/738761dd66266e63f55d4f08fe8f7233527bdb3d" target='_blank'>
              Single-cell RNA sequencing reveals the intercellular crosstalk and the regulatory landscape of stromal cells during the whole life of the mouse ovary
              </a>
            </td>
          <td>
            Wan Jiang, Wenya Sun, Yue Peng, Hao Xu, Haonan Fan, Xin Jin, Yue Xiao, Yuxiang Wang, Pin Yang, Wenjie Shu, Jing Li
          </td>
          <td>2024-12-01</td>
          <td>Life Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background
Sarcomatoid hepatocellular carcinoma (HCC) is a rare histological subtype of HCC characterized by extremely poor prognosis; however, its molecular characterization has not been elucidated.


Methods
In this study, we conducted an integrated multiomics study of whole-exome sequencing, RNA-seq, spatial transcriptome, and immunohistochemical analyses of 28 paired sarcomatoid tumor components and conventional HCC components from 10 patients with sarcomatoid HCC, in order to identify frequently altered genes, infer the tumor subclonal architectures, track the genomic evolution, and delineate the transcriptional characteristics of sarcomatoid HCCs.


Results
Our results showed that the sarcomatoid HCCs had poor prognosis. The sarcomatoid tumor components and the conventional HCC components were derived from common ancestors, mostly accessing similar mutational processes. Clonal phylogenies demonstrated branched tumor evolution during sarcomatoid HCC development and progression. TP53 mutation commonly occurred at tumor initiation, whereas ARID2 mutation often occurred later. Transcriptome analyses revealed the epithelial-mesenchymal transition (EMT) and hypoxic phenotype in sarcomatoid tumor components, which were confirmed by immunohistochemical staining. Moreover, we identified ARID2 mutations in 70% (7/10) of patients with sarcomatoid HCC but only 1-5% of patients with non-sarcomatoid HCC. Biofunctional investigations revealed that inactivating mutation of ARID2 contributes to HCC growth and metastasis and induces EMT in a hypoxic microenvironment.


Conclusions
We offer a comprehensive description of the molecular basis for sarcomatoid HCC, and identify genomic alteration (ARID2 mutation) together with the tumor microenvironment (hypoxic microenvironment), that may contribute to the formation of the sarcomatoid tumor component through EMT, leading to sarcomatoid HCC development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/043615905cf4026d2c0bac526bd7ad774bf375ad" target='_blank'>
              Integrated Molecular Characterization of Sarcomatoid Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Rongrong Sun, Yu-Hang Ye, Ye Xu, Bo Wang, Si-yuan Pan, Ning Li, Long Chen, Jing-Yue Pan, Zhi-qiang Hu, Jia Fan, Zheng-jun Zhou, Jian Zhou, Chengli Song, Shaolai Zhou
          </td>
          <td>2024-12-10</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Oral squamous cell carcinoma (OSCC) is a prevalent type of head and neck cancer with a high mortality rate, partly due to late diagnosis often accompanied by metastasis. Cachexia, a complex syndrome marked by severe weight loss and muscle wasting, frequently accompanies advanced cancers including OSCC. Notably, 50-70% of OSCC patients exhibit cachexia at diagnosis, which persists post-treatment, hinting at a metastatic contribution to this syndrome. One significant challenge in studying cancer cachexia is the lack of well-characterized pre-clinical models that accurately reflect human disease. Traditional mouse models often involve subcutaneously transplanted cells, which fail to mimic the natural tumor-host interactions and metastatic progression seen in patients. These models do not typically exhibit metastasis and differ significantly from the genetic and molecular landscape of human cancers. To address these limitations, we developed a syngeneic mouse model by orthotopically transplanting MOC2 murine OSCC cells into the tongues of B6J mice, thereby closely mimicking the tumor microenvironment and metastatic behavior of human OSCC. By enhancing metastasis with a palm-enriched high-fat diet, we observed progressive tumor growth, metastasis, and body weight loss, independent of food intake reduction, indicating metabolic dysregulation rather than anorexia. Comprehensive analyses, including single-cell RNA sequencing (scRNAseq) of immune cells and transcriptomic and metabolomic profiling of tumor and host tissues, identified significant gene expression changes associated with metastasis and cachexia. Enhanced metastasis correlated with lipid metabolism changes in lung metastases, suggesting that metastatic cells exploit lipids for growth. Further, immune profiling revealed neutrophils' metabolic adaptation in metastasis, with implications for both tumor progression and cachexia. Targeting key metabolic and inflammatory pathways in tumor and immune cells holds potential for mitigating metastasis and cachexia. Early-stage serum metabolomics indicated systemic metabolic shifts, emphasizing the need to understand the interplay between tumor metastasis and host metabolism. These findings underscore the complex mechanisms underlying OSCC-induced cachexia, and the need for new targeted therapeutic strategies to disrupt this lethal synergy between cancer progression and metabolic deterioration of the host.
 Citation Format: Blanca Majem, Josefina Martin, Liudmila Shevkova, Claudia Bigas, Gloria Pascual, Salvador Aznar-Benitah. Cross-talk between metastatic cells and host systems: Neutrophil metabolic adaptation, immune profiling, and systemic metabolic shifts in tumor progression and cachexia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr PR008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c1eb32f102c37cc14f504819a073149f1e255e" target='_blank'>
              Abstract PR008: Cross-talk between metastatic cells and host systems: Neutrophil metabolic adaptation, immune profiling, and systemic metabolic shifts in tumor progression and cachexia
              </a>
            </td>
          <td>
            Blanca Majem, Joseﬁna Martin, Liudmila Shevkova, Claudia Bigas, Gloria Pascual, Salvador Aznar-Benitah
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="BACKGROUND
Gastric intestinal metaplasia (IM) is a precancerous stage spanning a morphological spectrum that is poorly represented by human cell line models.


OBJECTIVE
We aim to establish and characterise human IM cell models to better understand IM progression along the cancer spectrum.


DESIGN
A large human gastric IM organoid (IMO) cohort (n=28), their clonal derivatives and normal gastric organoids (n=42) for comparison were established. Comprehensive multi-omics profiling and functional characterisation were performed.


RESULTS
Single-cell transcriptomes revealed IMO cells spanning a spectrum from hybrid gastric/intestinal to advanced intestinal differentiation. Their lineage trajectories connected different cycling and quiescent stem and progenitors, highlighting differences in gastric to IM transition and the potential origin of IM from STMN1 cycling isthmus stem cells. Hybrid IMOs showed impaired differentiation potential, high lineage plasticity beyond gastric or intestinal fates and reactivation of a fetal gene programme.Cell populations in gastric IM and cancer tissues were highly similar to those derived from IMOs and exhibited a fetal signature. Genomically, IMOs showed elevated mutation burden, frequent chromosome 20 gain and epigenetic deregulation of many intestinal and gastric genes. Functionally, IMOs were FGF10 independent and showed downregulated FGFR2. Several IMOs exhibited a cell-matrix adhesion independent subpopulation that displayed chromosome 20 gain but lacked key cancer driver mutations, potentially representing the earliest neoplastic precursor of IM-induced gastric cancer.


CONCLUSIONS
Overall, our IMO biobank captured the heterogeneous nature of IM, revealing mechanistic insights on IM pathogenesis and progression, offering an ideal platform for studying early gastric neoplastic transformation and chemoprevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10191e1bc78f541e4ae8577c359e64a33ed296da" target='_blank'>
              Divergent lineage trajectories and genetic landscapes in human gastric intestinal metaplasia organoids associated with early neoplastic progression.
              </a>
            </td>
          <td>
            Sarah S K Yue, Yin Tong, H. Siu, S. Ho, S. Law, W. Tsui, D. Chan, Yuanhua Huang, Annie S Y Chan, Shui Wa Yun, Ho Sang Hui, Jee-Eun Choi, Matthew S S Hsu, Frank P L Lai, April S Chan, S. Yuen, Hans Clevers, Suet Yi Leung, H. Y. Yan
          </td>
          <td>2024-11-20</td>
          <td>Gut</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Metastatic disease represents a complex series of interaction between tumors and their surrounding microenvironment. Here we utilize model systems to define organ specific transcriptional signatures present in metastatic seeding. We have identified unique epigenetic programing necessary for the colonization of the liver or the lung by colorectal cancer. We have shown that manipulation of master regulators of these signatures in vitro and in vivo changes the metastatic potential of the tumor. In specific, we have found that native cytokines within the liver and lung are engaged by the metastatic tumor cells to promote this reprograming. In the case of the liver, we have identified hepatocyte released CCL2 acts on metastasizing tumors to promote colonization. Through in vitro and in vivo models, we showed that CCL2 engagement drives upregulation of the transcription factor TCF7 and subsequent enhancer reprograming to promote survival of the metastatic cells in the liver. In the lung, we have identified additional, unique secreted cytokines which lead to the upregulation of other TCF family members. Together this data identifies an important signaling nodes in colorectal cancer. Here we propose a model in which secreted factors by the host organ, such as CCL2, reprogram genes within the tumor cell to promote growth and survival of the metastatic cell in foreign microenvironments such as the lung or liver. Targeting these proteins may kill existing metastatic lesions as well as block new metastases from forming which will be essential in improving the outcomes of patients with oligometastatic and premetastatic disease.
 Citation Format: Charlie Niesen, Nathan Wu, Jonathan Rennhack. Microenvironmentally released cytokines mediate organ specific transcriptional profiles for metastatic colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c74823f81b8e3a451e0d9b48e3071c6de71312d7" target='_blank'>
              Abstract A008: Microenvironmentally released cytokines mediate organ specific transcriptional profiles for metastatic colorectal cancer
              </a>
            </td>
          <td>
            Charlie Niesen, Nathan Wu, Jonathan Rennhack
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Cancers perpetuate immunosuppressive niches that fuel progression and hinder the clinical utility of immunotherapies. Pre-clinical validation of immunotherapies often relies on 2D models that lack spatial complexity, or mouse models that do not fully recapitulate the human tumor microenvironment (TME). We recently developed a human induced pluripotent stem cell (iPSC)-derived bone marrow organoid platform, creating a robust system to interrogate cancer-stroma cross talk. Here, we further develop the model to create the first in vitro system for evaluation of blood cancer immunotherapies in a human tissue environment. We focused on targeting myeloproliferative neoplasms (MPNs) – chronic blood cancers affecting ∼400,000 individuals in the US that are currently largely incurable. 1/3 of MPNs are driven by a mutation in calreticulin (mutCALR), and the mutCALR oncoprotein is displayed on the cell surface of disease-initiating stem cells and fibrosis-driving megakaryocytes. MPNs are characterized by pronounced inflammation and fibrosis (“myelofibrosis”). Therapeutic antibodies have entered first-in-human trials, and we recently presented pre-clinical data for the first mutCALR-directed CAR-T cell therapy. However, it is unclear how the TME may limit the efficiency of these agents. To address this, we first tested the utility of the organoid platform to evaluate target killing of leukemic cell lines and cells from patients using the anti-CD33 antibody-drug conjugate (ADC) gemtuzumab. The ADC showed good penetration and potent target cell killing in organoids, with only minimal reduction in efficacy in TGFβ-rich/fibrotic microenvironments. We then developed more sophisticated ‘chimeroids’ comprising the iPSC organoid scaffold engrafted with malignant stem cells plus CAR-Ts. Addition of mutCALR-directed CAR-Ts induced highly selective killing of mutCALR+ stem/progenitor cells with minimal toxicity to non-malignant cells including niche cells (n=5 donors), replicating the highly specific and robust killing seen in 2D cultures. To evaluate the impact of TME perturbations on CAR-T phenotype and function, we generated a scRNAseq dataset of 161,970 cells from ‘healthy’ and ‘myelofibrotic’ organoids engrafted with patient-derived mutCALR+ cells plus CAR-Ts, pre-stimulating organoids with two immunoregulatory cytokines highly abundant in myelofibrosis -TGFβ and Galectin-1. CAR- Ts exposed to mutCALR+ cells showed a 2x expansion of CD8+ effector memory cells, with significant enrichment of IFNγ, TNFα and IL-2 signaling gene-sets. The proportion of immunosuppressive Tregs increased in a high TGFβ TME, but not with Galectin-1. Ongoing analyses are further interrogating crosstalk between CAR-T, target cells, and niche components, and exploring the pro-inflammatory impact of CAR-T killing on the marrow niche – a potential mediator of delayed hematopoietic recovery. These studies demonstrate the utility of human organoids to evaluate blood cancer-targeting immunotherapies within a relevant human TME, with broad relevance for mechanistic and translational studies.
 Citation Format: Zoë C. Wong, Yuqi Shen, Alex Rampotas, Isaac Gannon, Charlotte Brierley, Camelia Benlabiod, Nawshad Hayder, Eleanor Murphy, Aude-Anaïs Olijnik, Claire Roddie, Martin Pulé, Adam J. Mead, Abdullah O. Khan, Bethan Psaila. Implementing human bone marrow organoids to interrogate microenvironmental influences on the efficacy of blood cancer immunotherapies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C049.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aee8c91681fcd641874626a899d8a0762b404e9" target='_blank'>
              Abstract C049: Implementing human bone marrow organoids to interrogate microenvironmental influences on the efficacy of blood cancer immunotherapies
              </a>
            </td>
          <td>
            Zoë C. Wong, Yuqi Shen, Alexandros Rampotas, Isaac Gannon, Charlotte K Brierley, Camelia Benlabiod, Nawshad Hayder, Eleanor Murphy, Aude-Anais Olijnik, C. Roddie, Martin Pule, A. Mead, Abdullah O Khan, B. Psaila
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad57ccef8d61a81007e75712476b9503ceed1d2e" target='_blank'>
              GPRC5A promotes lung colonization of esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            H. Zhou, Licheng Tan, Baifeng Zhang, D. Kwong, Ching Ngar Wong, Yu Zhang, Beibei Ru, Yingchen Lyu, Kin To Hugo Siu, Jie Luo, Yu-Ma Yang, Qin Liu, Yixin Chen, Weiguang Zhang, Chaohui He, Peng Jiang, Y. Qin, Beilei Liu, Xin-Yuan Guan
          </td>
          <td>2024-11-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Cancers exhibit a remarkable ability to develop resistance to a range of treatments, often resulting in relapse following first-line therapies and significantly worse outcomes for subsequent treatments. While our understanding of the mechanisms and dynamics of the emergence of resistance during cancer therapy continues to advance, many questions remain about which treatment strategies can minimize the probability that resistance will evolve, thereby improving long-term patient outcomes. In this study, we present an evolutionary simulation model of a clonal population of cells that can acquire resistance mutations to one or more treatments. We then leverage this model to examine the efficacy of a two-strike “extinction therapy” protocol—in which two treatments are applied sequentially in an effort to first contract the population to a vulnerable state and then push it to extinction—in comparison to that of a combination therapy protocol. We investigate the impact of parameters such as the timing of the switch between the two strikes, the rate of emergence of resistant mutations, the dose of the applied drugs, the presence of cross-resistance, and whether resistance is a binary or a quantitative trait. Our results indicate that the timing of switching from the first to the second strike has a marked effect on the likelihood of driving the population to extinction, and that extinction therapy outperforms combination therapy when cross-resistance is present. We conduct an in silico trial that reveals more detailed insight into when and why a second strike will succeed or fail. Finally, we demonstrate that modeling resistance as a quantitative rather than binary trait does not change our overall conclusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f738c3dddcab46b26ff2c97c3888fefb1e9d2858" target='_blank'>
              Evolutionary rescue model informs strategies for driving cancer cell populations to extinction
              </a>
            </td>
          <td>
            Amjad Dabi, Joel S. Brown, Robert A Gatenby, Corbin D. Jones, Daniel R. Schrider
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="BACKGROUND
The majority of patients with prostate cancer (PCa) exhibit intrinsic resistance to immune checkpoint blockade (ICB) following radiotherapy (RT). This resistance is generally attributed to the limited antigen presentation of heterogeneous cells within tumors. Here, we aimed to isolate and characterize these diverse subgroups of tumor post-RT to understand the molecular mechanisms of their resistance to ICB.


METHODS
Single-cell RNA-sequencing (scRNA-seq) was used to profile senescent cancer cell clusters induced by RT in LNCaP cells. The expression and phosphorylation levels of ataxia telangiectasia and Rad3-related protein (ATR) were assessed by immunohistochemistry in clinical samples from patients with or without RT. Co-immunoprecipitation, mutagenesis, and Western blotting were used to measure the interactions between proteins. Xenograft experiments were performed to assess the tumor immune response in the mice.


RESULTS
We identified a subset of PCa cells that exhibited resistance to RT, characterized by a reduced antigen presentation capability, which enhanced their ability to evade immune detection and resist cytotoxic T-lymphocyte-associated protein 4 (CTLA-4) blockade. scRNA-seq revealed that the senescent state was a transient phase of PCa cells post-RT, particularly in CTLA-4 blockade treatment-resistant cells. This state was marked by increased cytosolic ATR level. Cytosolic ATR phosphorylated CD86 in its cytosolic domain and enhanced the interaction between CD86 and its E3 ligase MARCH1 through electrostatic attraction. Depletion or inhibition of Atr increased the sensitivity to immune attack and improved responses to anti-Ctla-4 antibody treatment in a mouse model.


CONCLUSIONS
Our findings indicate that the activation of cytosolic ATR, which is associated with cellular senescence, impedes the effectiveness of combined RT and ICB treatments. This discovery may provide valuable insights for improving the efficacy of combined RT and ICB therapies in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54111804377a9c0bbc1679bc34e9106d5b8ef9e5" target='_blank'>
              Radiotherapy-resistant prostate cancer cells escape immune checkpoint blockade through the senescence-related ataxia telangiectasia and Rad3-related protein.
              </a>
            </td>
          <td>
            Chenyi Shao, Yingyi Zhang, Hang Li, Jiajia Chen, Ting Huang, Jiaze Li, Simeng Wen, Sen Wang, Saijun Fan, Yu Zhao
          </td>
          <td>2024-12-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Introduction Macrophages and T cells play crucial roles in liver physiology, but their functional diversity in hepatocellular carcinoma (HCC) remains largely unknown. Methods Two bulk RNA-sequencing (RNA-seq) cohorts for HCC were analyzed using gene co-expression network analysis. Key gene modules and networks were mapped to single-cell RNA-sequencing (scRNA-seq) data of HCC. Cell type fraction of bulk RNA-seq data was estimated by deconvolution approach using single-cell RNA-sequencing data as a reference. Survival analysis was carried out to estimate the prognosis of different immune cell types in bulk RNA-seq cohorts. Cell-cell interaction analysis was performed to identify potential links between immune cell types in HCC. Results In this study, we analyzed RNA-seq data from two large-scale HCC cohorts, revealing a major and consensus gene co-expression cluster with significant implications for immunosuppression. Notably, these genes exhibited higher enrichment in liver macrophages than T cells, as confirmed by scRNA-seq data from HCC patients. Integrative analysis of bulk and single-cell RNA-seq data pinpointed SPP1 + macrophages as an unfavorable cell type, while VCAN + macrophages, C1QA + macrophages, and CD8 + T cells were associated with a more favorable prognosis for HCC patients. Subsequent scRNA-seq investigations and in vitro experiments elucidated that SPP1, predominantly secreted by SPP1 + macrophages, inhibits CD8 + T cell proliferation. Finally, targeting SPP1 in tumor-associated macrophages through inhibition led to a shift towards a favorable phenotype. Discussion This study underpins the potential of SPP1 as a translational target in immunotherapy for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d7601947ef3a8c3102650d556bda64571da461a" target='_blank'>
              Identification of SPP1 + macrophages as an immune suppressor in hepatocellular carcinoma using single-cell and bulk transcriptomics
              </a>
            </td>
          <td>
            Han Jin, Woonghee Kim, Meng Yuan, Xiangyu Li, Hong Yang, Mengzhen Li, Mengnan Shi, Hasan Turkez, Mathias Uhlén, Cheng Zhang, Adil Mardinoglu
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Introduction Cancer-associated fibroblasts (CAFs) are a diverse group of cells that significantly contribute to reshaping the tumor microenvironment (TME), and no research has systematically explored the molecular landscapes of senescence related CAFs (senes CAF) in NB. Methods We utilized pan-cancer single cell and spatial transcriptomics analysis to identify the subpopulation of senes CAFs via senescence related genes, exploring its spatial distribution characteristics. Harnessing the maker genes with prognostic significance, we delineated the molecular landscapes of senes CAFs in bulk-seq data. We established the senes CAFs related signature (SCRS) by amalgamating 12 and 10 distinct machine learning (ML) algorithms to precisely diagnose stage 4 NB and to predict prognosis in NB. Based on risk scores calculated by prognostic SCRS, patients were categorized into high and low risk groups according to median risk score. We conducted comprehensive analysis between two risk groups, in terms of clinical applications, immune microenvironment, somatic mutations, immunotherapy, chemotherapy and single cell level. Ultimately, we explore the biological function of the hub gene JAK1 in pan-cancer multi-omics landscape. Results Through integrated analysis of pan-cancer spatial and single-cell transcriptomics data, we identified distinct functional subgroups of CAFs and characterized their spatial distribution patterns. With marker genes of senes CAF and leave-one-out cross-validation, we selected RF algorithm to establish diagnostic SCRS, and SuperPC algorithm to develop prognostic SCRS. SCRS demonstrated a stable predictive capability, outperforming the previously published NB signatures and clinic variables. We stratified NB patients into high and low risk group, which showed the low-risk group with a superior survival outcome, an abundant immune infiltration, a different mutation landscape, and an enhanced sensitivity to immunotherapy. Single cell analysis reveals biologically cellular variations underlying model genes of SCRS. Spatial transcriptomics delineated the molecular variant expressions of hub gene JAK1 in malignant cells across cancers, while immunohistochemistry validated the differential protein levels of JAK1 in NB. Conclusion Based on multi-omics analysis and ML algorithms, we successfully developed the SCRS to enable accurate diagnosis and prognostic stratification in NB, which shed light on molecular landscapes of senes CAF and clinical utilization of SCRS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/900ec2c5a181f8ad61d932465735fe6366505e3c" target='_blank'>
              Pan-cancer single cell and spatial transcriptomics analysis deciphers the molecular landscapes of senescence related cancer-associated fibroblasts and reveals its predictive value in neuroblastoma via integrated multi-omics analysis and machine learning
              </a>
            </td>
          <td>
            Shan Li, Junyi Luo, Junhong Liu, Dawei He
          </td>
          <td>2024-12-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Langerhans cell histiocytosis (LCH) is a rare inflammatory hematological neoplasm, characterized by the accumulation of LCH cells in various tissues and organs. These LCH cells originate from myeloid progenitors that carry a MAPK-pathway mutation. Once these mutated cells enter the tissue from the periphery, they differentiate into so-called LCH cells. In addition to LCH cells, the tumor microenvironment (TME) consists of a wide variety of immune cells. However, the interactions between these cells within the TME and their contribution to disease progression remain poorly understood. In this study, we focus on understanding the role of lymphocytes in LCH lesions. We used single-cell RNA sequencing to comprehensively profile the immune populations in LCH lesions on the transcriptomic level, which will be supplemented with spatial profiling. Our analysis identified a wide variety of infiltrating immune cells within LCH lesions with high cellular heterogeneity among patients. LCH- and T cells are the most prevalent cell types, suggesting a key role in lesion development and maintenance. Furthermore, the immune landscape includes several dendritic cell subsets – such as plasmacytoid DCs (pDCs), cDC1, and mature regulatory dendritic cells (mregDCs) – alongside tumor-associated macrophages (TAMs), monocytes, NK cells, B cells, and plasmablasts. Among T cells, regulatory T cells (Tregs) are particularly abundant and appear to be heavily influenced by LCH cells and other myeloid cells via the TNF-TNFR2 and CD86-CTLA4 pathways. Additionally, LCH cells express high levels of TGFβ – a known regulator of T cell activity – and MMP9 – which might hamper cytotoxic T cell infiltration. This suggests that LCH cells actively contribute to creating an immunosuppressive environment. Surprisingly, pDCs seem to be a major immune regulator within lesions and are likely to interact with T cells via TNFSF9 and Granzyme B. These insights into the immunosuppressive environment within LCH lesions highlight the critical role of Tregs as central regulators within the tumor microenvironment. The implications of these findings will be further explored and validated through in-vitro experiments and high-plex spatial protein profiling. Ultimately, our work may pave the way for developing targeted therapies aimed at modulating the TME, potentially enhancing anti-tumor immunity, and offering new strategies for treating LCH.
 Citation Format: Wouter van Midden, Raphaela Schwentner, Sebastian Eder, Philipp Ben Soussia-Weiss, Giulio Abagnale, Caroline Hutter. Exploring the composition of and interaction of the tumor microenvironment in Langerhans cell histiocytosis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C017.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d3ab0716138e1560234499a1766590bdc83e39" target='_blank'>
              Abstract C017: Exploring the composition of and interaction of the tumor microenvironment in Langerhans cell histiocytosis
              </a>
            </td>
          <td>
            Wouter van Midden, Raphaela Schwentner, S. Eder, Philipp Ben Soussia-Weiss, Giulio Abagnale, C. Hutter
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/364473dc218628b02eb817b8257429a67fd0ac17" target='_blank'>
              Current biological implications and clinical relevance of metastatic circulating tumor cells
              </a>
            </td>
          <td>
            R. Shahhosseini, S. Pakmehr, Anis Elhami, Maha Noori Shakir, Abdullah Ali Alzahrani, M. Al-Hamdani, Munther Abosoda, Ali Alsalamy, Majid Mohammadi-Dehcheshmeh, Tahereh Ezazi Maleki, Hossein Saffarfar, Payam Ali-Khiavi
          </td>
          <td>2024-11-15</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The incidence of melanoma, a highly aggressive skin cancer, continues to increase worldwide, particularly among populations with lighter skin tones. The diagnostic challenge of melanoma lies in the absence of a distinctive clinical presentation, as its characteristics vary based on anatomical location, growth type, and histopathology. The melanoma-associated antigen (MAGE) gene family is differentially expressed in various human cancers, including melanoma. In this study, we explored the association between human MAGEA2 (hMAGEA2) expression and melanoma. Using a human melanoma tissue array, we confirmed that hMAGEA2 expression was higher in melanoma and metastatic melanoma than in normal tissues. Additionally, we used SK-MEL-5 and SK-MEL-28 cell lines to investigate the cellular and molecular mechanisms underlying melanoma progression and invasiveness. In SK-MEL-5 and SK-MEL-28 cells, hMAGEA2 overexpression accelerated cell proliferation. Conversely, the knockdown of hMAEGA2 reduced cell proliferation, colony formation, and migration significantly and induced arrest at the G2/M phase of the cell cycle. With respect to the molecular mechanism, the knockdown of hMAGEA2 decreased the phosphorylation of Akt, JNK, and p38 MAPK. Additionally, hMAGEA2 knockdown reduced tumor formation significantly at the in vivo level. Collectively, the robust correlation between hMAGEA2 and melanoma metastasis supports the potential utility of hMAGEA2 as both a diagnostic marker and novel therapeutic target for patients with melanoma metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2f210e7fd413579bc370d616c2d10236a1bb97e" target='_blank'>
              hMAGEA2 as a potential diagnostic and therapeutic target for melanoma progression and metastasis.
              </a>
            </td>
          <td>
            Junkoo Yi, C. Kim, Hyeonjin Kim, Eun Gyung Kim, Do Yoon Kim, Yonghun Sung, Kang Dong Liu, Seong-Kyoon Choi, Zaeyoung Ryoo, Song Park, Myoung-Ok Kim
          </td>
          <td>2024-11-24</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract We performed comprehensive histopathologic, genomic, and epigenomic evaluation of paired initial and recurrent glioblastoma specimens from 106 patients to investigate the molecular evolution and cellular phenotypes underlying differential treatment responses. While TERT promoter mutation and CDKN2A homozygous deletion were early events during gliomagenesis shared by initial and recurrent tumors, most other recurrent genetic alterations (e.g., EGFR, PTEN, NF1) were commonly private to initial or recurrent tumors indicating acquisition later during clonal evolution. Further, glioblastomas exhibited heterogeneous epigenomic evolution with subsets becoming more globally hypermethylated, hypomethylated, or remaining stable. Glioblastoma that underwent sarcomatous transformation at recurrence had inferior survival and were significantly enriched in NF1, TP53, and RB1 alterations and the Mesenchymal epigenetic class. Patients who developed somatic hypermutation following temozolomide treatment had significantly longer interval to disease recurrence and prolonged overall survival, and hypermethylation at four specific CpG sites in the promoter region of MGMT was significantly associated with this development of hypermutation. Finally, an epigenomic evolution signature incorporating change in DNA methylation levels across 347 critical CpG sites was developed that significantly correlated with clinical outcomes. This talk will describe how GBM undergoes heterogeneous genetic, epigenetic, and cellular evolution that underlies prognostically different treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/704f3941c3c62c19e62495b9dd1824a7c49e3146" target='_blank'>
              IL-1 IDHWT GLIOBLASTOMA LONGITUDINAL MOLECULAR EVOLUTION
              </a>
            </td>
          <td>
            C. G. Lucas, N. Al-Adli, J. S. Young, R. Gupta, R. Morshed, J. Wu, A. Ravindranthan, A. Shai, N. Oberheim Bush, J. W. Taylor, J. de Groot, J. Villanueva-Meyer, M. Pekmezci, A. Perry, A. W. Bolen, P. Theodosopoulos, M. Aghi, E. Chang, S. Hervey-Jumper, D. R. Raleigh, A. M. Molinaro, J. Costello, A. A. Diaz, J. Clarke, N. Butowski, J. J. Phillips, S. M. Chang, M. S. Berger, D. A. Solomon
          </td>
          <td>2024-11-29</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Age is the biggest risk factor for developing most forms of cancer. In breast cancer, over 50% of patients are over 60 years old; however, less than 25% of patients enrolled in clinical trials are in this age bracket. Furthermore, most mouse modeling is performed in young mice, meaning that from preclinical through clinical stages patients ages are not accurately reflected in trials designs. Triple negative breast cancer (TNBC) is an aggressive subtype of breast cancer with no targeted therapies, so many treatment strategies utilize chemotherapy in combination with immunotherapy – which relies on a functional immune response. It is well known that immune function declines as we age, and thus therapeutic responses may differ with age. It is yet unclear how the tumor immune microenvironment changes with age, and how this may impact therapeutic response. Here, we utilize a mouse model which mimics hallmarks of human immune aging to understand how age impacts TNBC immune responses to dictate disease progression and therapeutic response. Immunologically young (8 weeks old) and aged (∼12 months old) FVB mice were orthotopically injected with syngeneic Met1 TNBC cells. Once tumors became palpable, mice were treated with paclitaxel chemotherapy or vehicle control and/or anti-PDL1 immunotherapy to mimic first-line therapeutic strategies. Tumors grew significantly faster in young control mice compared to aged, but young mice had a more robust response to therapy. Notable, only young mice had a significant benefit of combo therapy over monotherapy alone. Despite aged mice having more circulating CD8+ and CD4+ T cells, tumors in aged mice had significantly more infiltration of CD8+ cytotoxic T cells and FOXP3+ inhibitory T-regulatory cells compared to young control mice, yet only tumors in young mice exhibited an increase in T cell infiltration with therapy. Additionally, the ratio of CD8+ T cells and FOXP3+ Tregs was unchanged with age or treatment. Depletion of either CD8+ or CD4+ T cells led to a complete loss of age-associated differences in tumor growth. RNA sequencing analysis suggests that infiltrating T cells in aged mice may be exhausted due to a chronic interferon (IFN) phenotype, indicated by enrichment of IFN-alpha, IFN-gamma, and IL2 pathways, as well as increased expression of immune exhaustion markers PD1, CTLA4, TIM3 and LAG3. Neutralization of IFN-gamma blocks therapeutic response to anti-PDL1 in young mice but has no impact on aged mice. Analysis of published single cell RNAseq data indicated the same IFN response pathways enriched with age in human TNBC. These results demonstrate that the aging tumor immune microenvironment significantly impacts tumor progression and therapeutic response and suggest that targeting IFN signaling may lead to improved therapeutic responses. Together these findings establish a significant impact of immune aging on TNBC disease progression and response to therapy, highlighting the importance of developing age- stratified treatment strategies to improve responses in patients of all ages.
 Citation Format: Milos Spasic, Adrienne Parsons, Abigail Recko, Elizabeth Mittendorf, Peter van Galen, Sandra McAllister. Age dictates immune response within the tumor microenvironment of triple negative breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/114a0f540faf10e2e02da3fe7f40b293c8d733bd" target='_blank'>
              Abstract C016: Age dictates immune response within the tumor microenvironment of triple negative breast cancer
              </a>
            </td>
          <td>
            M. Spasic, Adrienne M Parsons, Abigail Recko, Elizabeth Mittendorf, Peter van Galen, Sandra McAllister
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Age is the single largest risk factor for the development of cancer, but how age impacts the molecular mechanisms that drive cancer remain poorly understood. While it is clear that age-related accumulation of cell autonomous mutations contributes to tumorigenesis, the central role age-related changes in the tumor microenvironment play in the transformation process is becoming more fully appreciated. Underscoring the importance of an aged microenvironment in cancer development are findings that senescent fibroblasts, which accumulate with age, directly stimulate preneoplastic and neoplastic cell growth and tumor progression. Investigations into how senescent fibroblasts promote tumorigenesis revealed that they express a plethora of growth factors, extracellular matrix remodeling enzymes, chemokines, and cytokines collectively referred to as the senescence associated secretory phenotype (SASP). Chemotherapy induces similar changes that can negatively impact a patient’s quality of life. We will discuss how these changes impact tumor progression and therapy-induced bone loss.
 Citation Format: Sheila A. Stewart. Age-related stromal changes drive breast cancer tumor progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr IA027.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a25dfba7fd100b42c0d41c8f8cf69167a46b14" target='_blank'>
              Abstract IA027: Age-related stromal changes drive breast cancer tumor progression
              </a>
            </td>
          <td>
            Sheila A Stewart
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="According to the principles of Evolutionary Cancer Cell Biology (ECCB), cancer stem cells (CSCs) do not derive from normal stem cells; rather, they originate from a distinct functional phenotype of germline cells characterized by asymmetric cell division (ACD). This phenotype proliferates through ACD, producing self-renewing cells alongside non-proliferating daughter cells with CSC qualities. ECCB posits that CSCs do not proliferate. Similar to protists, there exists a close reciprocal relationship between sister cells that collectively form a germline and stem cells. These sister cells perform complementary roles: proliferating cancer germline cells generate stem cells, whereas the non-proliferating CSCs give rise to progenitor cells for the formation of new germline clones. ECCB distinguishes between primary CSCs, which are associated with carcinogenesis and primary tumors, and secondary CSCs, which are linked to metastases. This unicellular stem cell system is homologous to that of parasitic protists, such as amebae. Both CSCs and amebae stem cells are produced by an oxygen-sensitive germline and are vulnerable to damage when oxygen levels exceed 6.0% (germline hyperoxia), as elevated oxygen concentrations can harm the germline genome. Germline cells that lose their stemness quality continue to cycle through defective symmetric cell divisions (DSCD). However, to restore functionality, the DSCD genome must be repaired through hyperpolyploidization. This process occurs in native polyploid giant cancer cells, which are homologous to the multinucleated genome repair structures found in protists.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22fd9b25c7727535490357470b16f77d5c7c81c2" target='_blank'>
              Reevaluating cancer stem cells and polyploid giant cancer cells from the evolutionary cancer cell biology perspective
              </a>
            </td>
          <td>
            V. Niculescu
          </td>
          <td>2024-11-21</td>
          <td>Cancer Plus</td>
          <td>0</td>
          <td>8</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [9, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>