<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-05-26 09:52:49 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="We conducted a spatial analysis of stage III colorectal adenocarcinomas using Hyperion Imaging Mass Cytometry, examining 52 tumors to assess the tumor microenvironment at the single-cell level. This approach identified 10 distinct cell phenotypes in the tumor microenvironment, including stromal and immune cells, with a subset showing a proliferative phenotype. By focusing on spatial neighborhood interactions and tissue niches, particularly regions with tumor-infiltrating lymphocytes, we investigated how cellular organization relates to clinicopathological and molecular features such as microsatellite instability (MSI) and recurrence. We determined that microsatellite stable (MSS) colorectal cancers had an increased risk of recurrence if they had the following features: 1) a low level of stromal tumor-infiltrating lymphocytes, and 2) low interactions between CD4 + T cells and stromal cells. Our results point to the utility of spatial single-cell interaction analysis in defining novel features of the tumor immune microenvironments and providing useful clinical cell-related spatial biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a63f7d3dd84bf14bb59b2621b8ec0ed0c881cf8" target='_blank'>
              The single-cell spatial landscape of stage III colorectal cancers
              </a>
            </td>
          <td>
            Andrew Su, Hojoon Lee, Minh Tran, Richard C Dela Cruz, A. Sathe, Xiangqi Bai, Ignacio A Wichmann, Lance Pflieger, Bryce Moulton, Tyler Barker, D. Haslem, David Jones, Lincoln Nadauld, Quan H Nguyen, Hanlee P. Ji, Terence Rhodes
          </td>
          <td>2025-04-07</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Tumor microenvironments (TMEs) are spatially complex and dynamic systems shaped by evolutionary pressures, tissue architecture, and cellular interactions. To capture this complexity, we developed the Tumor Landscape Analysis (TLA) pipeline, a computational framework applying principles from landscape ecology and spatial statistics to quantitatively characterize tumor spatial heterogeneity. TLA integrates spatially resolved pathology data, including whole-cell, point-based, and region-level formats, and computes multiscale metrics to assess cell distributions, neighborhood relationships, and tissue-level organization. The framework leverages ecological indices such as the Morisita-Horn index, Ripley’s H function, and Shannon diversity to quantify intercellular proximity, spatial clustering, and cellular diversity. It also uses the concept of local microenvironments (LMEs), data-driven ecological niches defined by local cell-type abundance and spatial uniformity, enabling unsupervised and reproducible classification of tumor regions. The use of fragmentation metrics, including patch density, shape complexity, and interspersion, provide further insight into spatial disorganization and emergent tissue architecture. TLA is agnostic to imaging modality and biological context, supporting broad applicability across tumor types and sample formats. By translating complex tissue architectures into interpretable spatial metrics, the pipeline enables integrative analyses that link spatial ecology to clinical and molecular phenotypes. This approach facilitates a deeper understanding of how spatial features contribute to tumor progression, therapeutic resistance, and clinical outcomes, offering new opportunities for precision oncology rooted in spatial systems biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/718b2bbad37dfa5c06ac139aad6c2d7117e031ab" target='_blank'>
              Tumor Landscape Analysis: An Ecologically Informed Framework to Understand Tumor Microenvironments
              </a>
            </td>
          <td>
            Luis Cisneros, M. D. Toruner, Martin E Fernandez-Zapico, Carlo C. Maley, Ryan M. Carr
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e484c7784673f4b5393c2f4419927f4ebb20abef" target='_blank'>
              Mapping the spatial architecture of glioblastoma from core to edge delineates niche-specific tumor cell states and intercellular interactions
              </a>
            </td>
          <td>
            Saad M. Khan, Anthony Z. Wang, Rupen Desai, Colin R. McCornack, Rui Sun, Sonika M. Dahiya, Jennifer A. Foltz, Ngima D Sherpa, Lydia A. Leavitt, T. West, Alexander F. Wang, Aleksandar Krbanjevic, Bryan D. Choi, Eric C. Leuthardt, B. Patel, Al Charest, Albert H. Kim, Gavin P. Dunn, Allegra A. Petti
          </td>
          <td>2025-04-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae3a20489c547fc5b01d13afcfc1a7edcf2969d" target='_blank'>
              An image analysis pipeline to quantify the spatial distribution of cell markers in stroma-rich tumors
              </a>
            </td>
          <td>
            Antoine A. Ruzette, Nina Kozlova, Kayla A. Cruz, Taru Muranen, S. Nørrelykke
          </td>
          <td>2025-05-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b426c5f702b08645ac569d89b1da888e84c491" target='_blank'>
              Deciphering the cellular tumor microenvironment landscape in salivary gland carcinomas using multiplexed imaging mass cytometry
              </a>
            </td>
          <td>
            M. Mayer, L. Nachtsheim, L. Jansen, P. Wolber, M. Schmiel, A. Quaas, J. P. Klussmann, C. Arolt
          </td>
          <td>2025-05-13</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a complex ecosystem where malignant and non-malignant cells cooperate and interact determining cancer progression. Cell abundance, phenotype and localization within the TME vary over tumor development and in response to therapeutic interventions. Therefore, increasing our knowledge of the spatiotemporal changes in the tumor ecosystem architecture is of importance to better understand the etiologic development of the neoplastic diseases. Imaging Mass Cytometry (IMC) represents the elective multiplexed imaging technology enabling the in-situ analysis of up to 43 different protein markers for in-depth phenotypic and spatial investigation of cells in their preserved microenvironment. IMC is currently applied in cancer research to define the composition of the cellular landscape and to identify biomarkers of predictive and prognostic significance with relevance in mechanisms of drug resistance. Herein, we describe the general principles and experimental workflow of IMC raising the informative potential in preclinical and clinical cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/091e898b367848be40c66f493aec8ad58485d477" target='_blank'>
              From surfing to diving into the tumor microenvironment through multiparametric imaging mass cytometry
              </a>
            </td>
          <td>
            M. Erreni, M. R. Fumagalli, Matteo Marozzi, Roberto Leone, Raffaella Parente, Raffaella D’Anna, Andrea Doni
          </td>
          <td>2025-04-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL) therapy, recently approved by the FDA for melanoma, is an emerging modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors such as glioblastoma remains limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas using spectral flow cytometry, TCR sequencing, single-cell RNA-seq, Xenium in situ transcriptomics, and CODEX spatial proteomics. Comparative analysis of TIL-generating (TIL+) versus non-generating (TIL−) tumors revealed that IL7Rexpression, structured perivascular immune clustering, and tumor-intrinsic metabolic programs such as ACSS3 were associated with successful TIL expansion. In contrast, TIL−tumors were enriched for neuronal lineage signatures, immunosuppressive transcripts including TOX and FERMT1, and tumor-connected macrophages. This study defines spatial and molecular correlates of TIL manufacturing success and establishes a genomics-enabled selection platform for adoptive T cell therapy. The profiling approach is now being prospectively implemented in the GIANT clinical trial (NCT06816927), supporting its translational relevance and scalability across glioblastoma and other immune-excluded cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7799f3cb909d60657cf83c09eaecb6b9bad79c3" target='_blank'>
              A Spatial Multi-Omic Framework Identifies Gliomas Permissive to TIL Expansion
              </a>
            </td>
          <td>
            Mustafa Khasraw, K. Hotchkiss, Kenan Zhang, Anna M Corcoran, Elizabeth Owens, P. Noldner, C. Railton, Kyra Van Batavia, Ying Zhou, Jodie Jepson, Kirit Singh, R. McLendon, Kristen Batich, Anoop P Patel, Katayoun Ayasoufi, Michael Brown, Evan Calabrese, Jichun Xie, Jose R Conejo-Garcia, B. Shaz, John Hickey
          </td>
          <td>2025-04-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Motivation Spatial transcriptomic (ST) technologies, such as GeoMx Digital Spatial Profiler, are increasingly utilized to investigate the role of diverse tumor microenvironment components, particularly in relation to cancer progression, treatment response, and therapeutic resistance. However, in many ST studies, the spatial information obtained from immunofluorescence imaging is primarily used for identifying regions of interest (ROIs) rather than as an integral part of downstream transcriptomic data analysis and interpretation. Results We developed ROICellTrack, a deep learning-based framework that better integrates cellular imaging with spatial transcriptomic profiling. By analyzing 56 ROIs from urothelial carcinoma of the bladder and upper tract urothelial carcinoma, ROICellTrack identified distinct cancer–immune cell mixtures, characterized by specific transcriptomic and morphological signatures and receptor–ligand interactions linked to tumor content and immune infiltrations. Our findings demonstrate the value of integrating imaging with transcriptomics to analyze spatial omics data, improving our understanding of tumor heterogeneity and its relevance to personalized and targeted therapies. Availability and implementation ROICellTrack is publicly available at https://github.com/wanglab1/ROICellTrack.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af060e4cc93259c4d19b62b9cc2136ad5699039" target='_blank'>
              ROICellTrack: a deep learning framework for integrating cellular imaging modalities in subcellular spatial transcriptomic profiling of tumor tissues
              </a>
            </td>
          <td>
            Xiaofei Song, Xiaoqing Yu, Carlos M. Morán-Segura, Hongzhi Xu, Tingyi Li, Joshua T Davis, A. Vosoughi, G. D. Grass, Roger Li, Xuefeng Wang
          </td>
          <td>2025-04-08</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL) therapy, recently approved by the FDA for melanoma, is an emerging modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors such as glioblastoma remains limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas using spectral flow cytometry, TCR sequencing, single-cell RNA-seq, Xenium in situ transcriptomics, and CODEX spatial proteomics. Comparative analysis of TIL-generating (TIL⁺) versus non-generating (TIL⁻) tumors revealed that IL7R expression, structured perivascular immune clustering, and tumor-intrinsic metabolic programs such as ACSS3 were associated with successful TIL expansion. In contrast, TIL⁻ tumors were enriched for neuronal lineage signatures, immunosuppressive transcripts including TOX and FERMT1, and tumor-connected macrophages. This study defines spatial and molecular correlates of TIL manufacturing success and establishes a genomics-enabled selection platform for adoptive T cell therapy. The profiling approach is now being prospectively implemented in the GIANT clinical trial (NCT06816927), supporting its translational relevance and scalability across glioblastoma and other immune-excluded cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/945f91b27ecd7bfb796d6743f706c6451c1e2314" target='_blank'>
              A Spatial Multi-Omic Framework Identifies Gliomas Permissive to TIL Expansion
              </a>
            </td>
          <td>
            K. Hotchkiss, Kenan Zhang, Anna M Corcoran, Elizabeth Owens, P. Noldner, C. Railton, Kyra Van Batavia, Ying Zhou, Jodie Jepson, Kirit Singh, R. McLendon, Kristen Batich, Anoop P Patel, Katayoun Ayasoufi, Michael Brown, Evan Calabrese, Jichun Xie, Jose R Conejo-Garcia, B. Shaz, John W. Hickey, Mustafa Khasraw
          </td>
          <td>2025-04-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d556d25908485193c4e548584bab10bfa70daa1" target='_blank'>
              Oncogenic drivers shape the tumor microenvironment in human gliomas
              </a>
            </td>
          <td>
            Bo Zhao, Chae Yun Cho, Lingqun Ye, Tula Keal, Thomas Mitchell, Inés Martín-Barrio, C. A. Faraj, Mesut Unal, Jessica Alsing, Nora M Lawson, Ellie Rahm Kim, Francisco Chavez, Kristy Mendoza Rangel, Sanjay K Singh, Joseph H. McCarthy, D.M. Ashley, W. Yung, Vinay K. Puduvalli, Guy Nir, Jeffrey S. Weinberg, Linghua Wang, Keith L. Ligon, G. Genovese, Peter Van Loo, P. Futreal, Jason T. Huse, Jesse R. Dixon, Frederick F. Lang, Kadir C. Akdemir
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc7a8f38a5d53281f1873999c12438505ab4d573" target='_blank'>
              Spatial epigenomic niches underlie glioblastoma cell state plasticity
              </a>
            </td>
          <td>
            Sam Kint, S. T. Younes, Shuozhen Bao, Gretchen Long, David Wouters, Erin Stephenson, Xing Lou, Mei Zhong, Di Zhang, Graham Su, Archibald Enninful, Mingyu Yang, Huey-Miin Chen, K. Ellestad, Colleen Anderson, Jennifer Moliterno, John J. Kelly, Jennifer A. Chan, Alejandro Sifrim, Jiangbing Zhou, Ana Nikolic, Rong Fan, Marco Gallo
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd408ac5f53831aa4c4d85811f6f5969702d07e" target='_blank'>
              High-Dimensional Protein Analysis Uncovers Distinct Immunological and Stromal Signatures Between Primary and Metastatic Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Emily Greene, Natalie K. Horvat, D. Doxie, Vaunita C. Parihar, Jayden Kim, Cameron J. Herting, Erin E. Grundy, Ayana T. Ruffin, Alyssa Krasinskas, Shishir Maithel, Juan M. Sarmiento, Mihir M. Shah, Mohammad Y Zaidi, Maria Diab, O. Alese, Kavita Dhodapkar, H. Kissick, Bassel F. El-Rayes, C. Paulos, Gregory B. Lesinski
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="In breast cancer related to the BRCA1 mutation, luminal progenitor cells are believed to be the cells of origin, yet how these cells transform into invasive cancer cells remain poorly understood. Here, we combine single-cell epigenomic and transcriptomic data to reconstitute sequences of events in luminal cells that lead to tumorigenesis. Upon deletion of Trp53 and Brca1, we find that luminal progenitors display an extensive epigenomic disorder associated with a loss of cell identity. These cells then progress to tumor formation through a partial epithelial-to-mesenchymal transition, orchestrated by Snail and the timely activation of immunosuppressive and FGF signaling with their microenvironment. In human samples, pre-tumoral changes can be detected in early stage, basal-like tumors, which rarely recur, as well as in normal-like mammary glands of BRCA1 mutation carriers who have had cancer. Our study fills critical gaps in our understanding of BRCA1-driven tumorigenesis, opening perspectives for the early monitoring of individuals with high cancer risk. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02331-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f69d9b899da12be47c2084b992747c9f0654fa" target='_blank'>
              Epigenomic disorder and partial EMT impair luminal progenitor integrity in Brca1-associated breast tumorigenesis
              </a>
            </td>
          <td>
            Camille Landragin, Mélissa Saichi, Marthe Laisné, Adeline Durand, Pacôme Prompsy, Renaud Leclère, Jérémy Mesple, Kyra Borgman, Amandine Trouchet, Marisa M Faraldo, A. Chiche, A. Vincent-Salomon, Hélène Salmon, Céline Vallot
          </td>
          <td>2025-04-28</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b28a229fd99f6a9c5521983f68e3ec9d09a032a" target='_blank'>
              Abstract 6604: Characterizing the tumor microenvironment using spatially barcoded archival FFPE tissue: converting single-nucleus RNA-seq into spatial transcriptomics
              </a>
            </td>
          <td>
            Cedric R. Uytingco, Julie Wilhelmy, Bryant Ngo, Jonathan Sakkos, Wanxin Wang, Christina Chang, Christina Fan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abfdae0d9888d7167ae3464de1089da14e882abe" target='_blank'>
              Multi-cellular phenotypic dynamics during the progression of breast tumors
              </a>
            </td>
          <td>
            Louise Gsell, Spencer S. Watson, Iva Sutevski, Matteo Massara, Klara Soukup, Alper Eroglu, Jeff E Mold, Antony Cougnoux, Johanna A. Joyce, Jean Hausser
          </td>
          <td>2025-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffdda1a300bd205aab897ffa9f139e98ec525b59" target='_blank'>
              Abstract 155: Spatial analysis of tumor-immune-stromal cellular phenotypes and microenvironments in pancreatic cancer
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Elizabeth Sunnucks, Sibyl Drissler, Tiak Ju Tan, Chengxin Yu, Jennifer L. Gorman, Michael J Geuenich, Sheng-Ben Liang, Edward L. Chen, Golnaz Abazari, Barbara Gruenwald, A. Borgida, Cassandra J. Wong, Brendon Seale, Zhen Yuan Lin, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Anne-Claude Gringas, Grianne M. O'Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/411bf5452c2dd87027a9ed52038adc0d9888da4c" target='_blank'>
              HRCHY-CytoCommunity identifies hierarchical tissue organization in cell-type spatial maps
              </a>
            </td>
          <td>
            Zekun Wang, Yafei Xu, Yuxuan Hu, Lin Gao
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf4d0e7843bbae99e8106885d03d265c924f51c" target='_blank'>
              A unified eIF1A+ luminal cells-centered hypoxic and “cold” tumor microenvironment promotes PCa progression among different subtypes
              </a>
            </td>
          <td>
            Lilin Wan, Enyao Huang, Zicheng Xu, Bing Zheng, Xuefei Ding, Chunyan Chu, Tiange Wu, Shengrong Chen, Junyong Zhuang, Ming Chen, Rong Na, Shenghong Ju, Dingxiao Zhang, Wenchao Li, Yifei Cheng, Bin Xu
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Objectives: Bone metastasis is a frequent and life-threatening event in advanced cancers, affecting up to 70–85% of prostate cancer patients. Understanding the cellular and molecular mechanisms underlying bone metastasis is essential for developing targeted therapies. This study aimed to systematically characterize the heterogeneity and microenvironmental adaptation of prostate cancer bone metastases using single-cell transcriptomics. Methods: We integrated the largest single-cell transcriptome dataset to date, encompassing 124 samples from primary prostate tumors, various bone metastatic sites, and non-malignant tissues (e.g., benign prostatic hyperplasia, normal bone marrow). After quality control, 602,497 high-quality single-cell transcriptomes were analyzed. We employed unsupervised clustering, gene expression profiling, mutation analysis, and metabolic pathway reconstruction to characterize cancer cell subtypes and tumor microenvironmental remodeling. Results: Cancer epithelial cells dominated the tumor microenvironment but exhibited pronounced heterogeneity, posing challenges for conventional clustering methods. By integrating genetic and metabolic features, we revealed key evolutionary trajectories of epithelial cancer cells during metastasis. Notably, we identified a novel epithelial subpopulation, NEndoCs, characterized by unique differentiation patterns and distinct spatial distribution across metastatic niches. We also observed significant metabolic reprogramming and recurrent mutations linked to prostate-to-bone microenvironmental transitions. Conclusions: This study comprehensively elucidates the mutation patterns, metabolic reprogramming, and microenvironment adaptation mechanisms of bone metastasis in prostate cancer, providing key molecular targets and clinical strategies for the precise treatment of bone metastatic prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64291da16ef528fb561e1af0f4b9ad8a8ddde042" target='_blank'>
              Comprehensive Integrated Analysis Reveals the Spatiotemporal Microevolution of Cancer Cells in Patients with Bone-Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Yinghua Feng, Xiuli Zhang, Guangpeng Wang, Fei-ya Yang, Ruifang Li, Lu Yin, Dong Chen, Wenkuan Wang, Mingshuai Wang, Zhiyuan Hu, Yuan Sh, N. Xing
          </td>
          <td>2025-04-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd09b27d0d8fe6fa80c372bc8924f459f7a4d33" target='_blank'>
              MOSAIC: Intra-tumoral heterogeneity characterization through large-scale spatial and cell-resolved multi-omics profiling
              </a>
            </td>
          <td>
            Caroline Hoffmann
          </td>
          <td>2025-05-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178f28f36aaf5c0de57a93091858bdbe9ce93e72" target='_blank'>
              Tumor Microenvironment in Ovarian Cancer through Spatial Transcriptomics and Identification of Key Gene Expression Profiles
              </a>
            </td>
          <td>
            K. K. Kırboğa, E. Küçüksille, M. Rudrapal, E. Aktaş, R. Achar, Gouri Deshpande, V. Stupin, E. Silina
          </td>
          <td>2025-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a1265fddcea3943ae47df40af80fa7b2b32f84" target='_blank'>
              Abstract 2663: Mapping clonal heterogeneity and stromal dynamics in prostate cancer
              </a>
            </td>
          <td>
            S. Figiel, Mengxiao He, Wencheng Yin, Emmanouela Perisynaki, Renuka Teague, T. Anbarasan, Eleanor M O'Roberts, N. Ranasinha, Sophia M. Abusamra, Trishna A. Desai, D. Doultsinos, Andrew Erickson, Srinivasa Rao, Clare Verrill, Richard Colling, P. Gill, T. Mirtti, Ibrahim Kulac, K. Smith-Byrne, Ruth C. Travis, Charlotte Stadler, Richard J. Bryant, Freddie C Hamdy, Dan Woodcock, Ian G. Mills, J. Lundeberg, Alastair D. Lamb
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Glioblastoma, one of the most aggressive and heterogeneous malignant tumors, presents significant challenges for clinical management due to its cellular and metabolic complexity. This review integrates recent advancements in single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to elucidate glioblastoma’s cellular heterogeneity and metabolic reprogramming. Diverse cellular subpopulations, including malignant proliferative cells, stem-like cells, mesenchymal-like cells, and immune-related cells, contribute to tumor progression, treatment resistance, and microenvironmental interactions. Spatial transcriptomics has further revealed distinct spatial distributions of these subpopulations, highlighting differences in metabolic activities between the tumor core and periphery. Key metabolic adaptations, such as enhanced glycolysis, fatty acid oxidation, and glutamine metabolism, play critical roles in supporting tumor growth, immune evasion, and therapeutic resistance. Targeting these metabolic pathways, especially in combination with immunotherapy, represents a promising avenue for glioblastoma treatment. This review emphasizes the importance of integrating single-cell and spatial multi-omics technologies to decode glioblastoma’s metabolic landscape and explore novel therapeutic strategies. By addressing current challenges, such as metabolic redundancy and spatiotemporal dynamics, this work provides insights into advancing precision medicine for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea52e3e92b1a5c4c3caddeca60f128feaaaf952b" target='_blank'>
              Single-cell and spatial transcriptomic insights into glioma cellular heterogeneity and metabolic adaptations
              </a>
            </td>
          <td>
            Yixin Fu, Yong Yi, Yongxiang Shao, Jingcheng Jiang, Qingshan Deng
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73634c31cfc63813b66fedfb8ecff6a08ba232af" target='_blank'>
              Spatiotemporal organisation of residual disease in mouse and human BRCA1-deficient mammary tumours and breast cancer
              </a>
            </td>
          <td>
            Demeter Túrós, Morgane Decollogny, Astrid Chanfon, Myriam Siffert, L. Romanens, J. Tille, O. Trédan, I. Labidi-Galy, Alberto Valdeolivas, Sven Rottenberg
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f029b0348e3538dc977fb4d11569d99d619e9e1" target='_blank'>
              Abstract 2089: Spatial profiling of exhausted T cells using high-plex imaging mass cytometry
              </a>
            </td>
          <td>
            Thomas D. Pfister, J. Y. Chwee, Qanber Raza, Nikesh Parsotam, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462e90afb96bda692d274dd22748c2e27453a930" target='_blank'>
              Diving Deep: Profiling Exhausted T Cells in the Tumor Microenvironment Using Spectral Flow Cytometry
              </a>
            </td>
          <td>
            Karen W Teng, Weng Hua Khoo, Nicholas Ching Wei Ho, S. Jasemine Yang, , Edmond Chua, Shu Wen Samantha Ho
          </td>
          <td>2025-04-01</td>
          <td>Cytometry Part A</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb74e2d2b03a62ef097600aaf5d84f08dfc469c4" target='_blank'>
              Prognostically favorable immune responses to ovarian cancer are distinguished by self-reactive intra-epithelial plasma cells
              </a>
            </td>
          <td>
            Allyson C. Banville, Céline M. Laumont, Karanvir Singh, Breeze Gladwin, Jaden Dedora, Liam Mitchell, Jen Wang, A. Miranda, Bridget Mateyko, E. Chavez, G. Negri, Se Wing Grace Cheng, Julian Smazynski, K. Milne, Kylee Wright, Malia Lampard, Nicole S. Gierc, P. Hamilton, Sandra E. Spencer, Shreena Kalaria, Talen Oostenbroek, Victor Negrea Puskas, Christian Steidl, G. Morin, F. Nathoo, Brad H Nelson
          </td>
          <td>2025-05-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/613002ca1a9de7d4f7ae92b0493c480e7bbe4c62" target='_blank'>
              Abstract 5323: Spatial heterogeneity of tumor and stromal cells in desmoplastic small round cell tumors
              </a>
            </td>
          <td>
            Jiaqian Fan, Adewale A. Adebayo, Diana Shamsutdinova, D. Ingram, Clement Agyemang, R. Cárdenas-Zúñiga, Alexander J. Lazar, Danh D. Truong, Joseph A Ludwig
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="It has been reported that microsatellite instable (MSI) tumors in colorectal cancer (CRC) exhibit stronger anti-tumor responses than microsatellite stable (MSS) tumors. Analyzing scRNA-seq data from 185 CRC patients we found that immune, structural, and cancer cells in MSS tumors with high numbers of tumor-infiltrating CD8 T cells and macrophages (TAMs; CD8hiTAMhi) and CD8lowTAMhi tumors were enriched for pathways associated with anti- and pro-tumor responses, respectively. In CD8hiTAMhi tumors, TAMs expressed an IFN-induced phenotype (e.g. GBP1, CXCL9, IFITM3) and high infiltration of GBP1+ TAMs was associated with better overall survival (n=941). High-resolution spatial transcriptomics (Visium HD) revealed that GBP1+ TAMs clustered with CXCL13+IFNG+PDCD1+ CD8 T cells in tumoral niches, suggesting that GBP1+ TAMs were involved in recruitment and activation of tumor-reactive CD8 T cells. Together, we uncovered coordinated cellular programs across cell types in the microenvironment of MSS tumors, based on simple classification variables, that were strongly associated with prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5dfde3e4f7d39011a34eb5a718ac54f39aa6ee4" target='_blank'>
              Single cell spatial transcriptomics identifies coordinated cellular programs associated with good prognosis in microsatellite stable colorectal cancer
              </a>
            </td>
          <td>
            Victoria T. Karlsen, Umair Majid, Håvard T. Lindholm, S. U. Umu, Karoline Rapp Vander-Elst, A-C Røberg Beitnes, Marianne A. Merok, Kjersti Thorvaldsen Hagen, H. R. Nilsen, Sheraz Yaqub, E. Bækkevold, Diana Domanska, F. Jahnsen
          </td>
          <td>2025-05-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Objective This study examines the cellular heterogeneity of epithelial cells within pancreatic ductal adenocarcinoma (PDAC) and their contributions to tumor progression, metastasis, and immunosuppressive interactions using single-cell RNA sequencing. Methods Single-cell RNA-sequencing data from two datasets (GSE154778 and GSE158356) were integrated using the Harmony algorithm, followed by quality control, clustering, and differential gene expression analysis. Distinct subpopulations of epithelial cells were identified, and their gene expression profiles were analyzed. To assess the malignancy of these subpopulations, single-cell copy number variation (CNV) analysis and trajectory analysis were conducted. Additionally, intercellular communication was examined using the CellChat platform. Results The analysis revealed pronounced heterogeneity among PDAC epithelial cells, with specific subpopulations exhibiting distinct roles in tumor proliferation, extracellular matrix remodeling, and metastatic dissemination. Subpopulations 4 and 6 were characterized by increased CNV levels and a more malignant phenotype, suggesting an enhanced capacity for metastasis. Single-cell trajectory analysis, along with CellChat, mapped the temporal evolution of epithelial cells, identifying key regulatory genes such as DCBLD2 and JUN. A prognostic model incorporating five key genes, including KLF6, was developed and demonstrated strong predictive accuracy for patient outcomes. Notably, KLF6 emerged as a critical prognostic marker associated with immune modulation, particularly through interactions with M2 macrophages. Conclusion The study highlights the pronounced heterogeneity of epithelial cells in PDAC and their distinct contributions to tumor progression, metastasis, and immune modulation. Through single-cell transcriptomic and CNV analyses, we identified epithelial subpopulations with varying malignant potentials and distinct interactions with the tumor microenvironment. Among these, KLF6 emerged as a key regulator associated with immune modulation and metastasis. Our findings emphasize the significance of epithelial cell heterogeneity in shaping pancreatic cancer progression. These insights provide a foundation for future investigations into novel prognostic markers and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12957-025-03793-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c02e096347df4014f9175b24f19fbd3cae4aa9d0" target='_blank'>
              Deciphering the heterogeneity of epithelial cells in pancreatic ductal adenocarcinoma: implications for metastasis and immune evasion
              </a>
            </td>
          <td>
            Jie Liu, Hui Li, Xiuyun Lin, Jiani Xiong, Guangfeng Wu, Lingyan Ding, Bin Lin
          </td>
          <td>2025-04-16</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328ab9ab4a1194c8da958fc1fd33834df82031cb" target='_blank'>
              Context-dependent spatial multicellular network motifs for single-cell spatial biology
              </a>
            </td>
          <td>
            Amos Zamir, Yael Amitay, Yuval Tamir, Leeat Keren, A. Zaritsky
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35597888b2f30575949d9afcb81bdfc42dbe699b" target='_blank'>
              Multiscale Spatial Mapping of Microbial Communities for Biotherapeutic Development
              </a>
            </td>
          <td>
            Prateek Sehgal, Andrea G. Shaw, Kevin Cutler, Bridget Letson, Gregory T. Booth, Gaetano J. Scuderi, Anjali Doshi, Peter Diebold, Hannah Bronson, Maya Trisolini, Unnati Dharaiya, Kevin Keomanee-Dizon, Seyi Jung, Jennifer Bagin, James Berleman, Aditya Bhalla, Kyle Jacoby, Steve Kujawa, Jackson Mintz, P. Murugkar, Marguerite Prior, Keiramarie Robertson, Ashlie L. Burkart, Jason Pontin, Lee R. Swem, I. Vlaminck, M. P. Cheng, Philip Burnham, Hao Shi
          </td>
          <td>2025-03-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1db37f2f209f5470571ed3506376b29858f863c2" target='_blank'>
              Tumor cell heterogeneity drives spatial organization of the intratumoral immune response.
              </a>
            </td>
          <td>
            Miho Tanaka, Lotus Lum, Kenneth H Hu, Piyush Chaudhary, Savannah Hughes, Cecilia Ledezma-Soto, B. Samad, Daphne Superville, Kenneth Ng, Arun Chumber, Ciara Benson, Zoe N Adams, Kelly Kersten, Oscar A. Aguilar, Lawrence Fong, Alexis J. Combes, Matthew F. Krummel, Melissa Q Reeves
          </td>
          <td>2025-06-02</td>
          <td>The Journal of experimental medicine</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13f1f42b49de112548e7d274daf610e71f0196dd" target='_blank'>
              Abstract 2067: Single-cell spatial whole transcriptome reveals tumor heterogeneity and stromal dynamics in invasive ductal carcinoma of the breast
              </a>
            </td>
          <td>
            Liang Zhang, Shanshan He, Michael Patrick, Stefan Rogers, Haiyan Zhai, John S Lyssand, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb7304b0f5fc3d1e6eaf91e29f30f258a3c3f94f" target='_blank'>
              Abstract 5263: The spatial landscape of tumor-infiltrating lymphocytes in stage III colorectal cancers by single-cell spatial analysis
              </a>
            </td>
          <td>
            Hojoon Lee, Andrew Su, Minh Tran, Richard Delacruz, A. Sathe, Xiangqi Bai, Ignacio A Wichmann, Lance Pflieger, Bryce Moulton, Tyler Barker, D. Haslem, David Jones, Lincoln Nadauld, Quan H Nguyen, Hanlee P. Ji, Terence Rhodes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de704d9a88b35d65b9fbe36b0e36b4cae7473ce" target='_blank'>
              Abstract 161: High-resolution spatial proteomics characterizes colorectal cancer consensus molecular subtypes
              </a>
            </td>
          <td>
            Batuhan Kısakol, Anna Matveeva, Sanghee Cho, Mohammedreza Azimi, A. Lindner, M. Salvucci, E. McDonough, Joanna Fay, Tony O’Grady, N. McCawley, John P Burke, Deborah A. McNamara, John Graf, Simon McDade, Daniel B. Longley, Fiona Ginty, Jochen H M Prehn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7516851ad82e74ba0e5d8528ea22ebc6695c2d0" target='_blank'>
              Abstract 2584: In situ hybridization-based validation of transcriptionally-defined stromal clusters in the prostate cancer microenvironment
              </a>
            </td>
          <td>
            Filippo Pederzoli, H. Pakula, Wikum Dinalankara, Jeanne Jiang, Mohamed Omar, Silvia Rodrigues, P. Nuzzo, Fabio Socciarelli, G. Fanelli, L. Marchionni, Massimo Loda
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b2e8b55d68338e1058f7b78eb076c9fe5aa970" target='_blank'>
              Abstract 166: Spatial subtypes and cellular interactions of cancer-associated fibroblasts revealed by single-cell spatial omics
              </a>
            </td>
          <td>
            Yunhe Liu, Ansam Sinjab, Jimin Min, Guangchun Han, Francesca Paradiso, Yuanyuan Zhang, Yang Liu, Ruiping Wang, Guangsheng Pei, K. S. Cho, Enyu Dai, Akshay V. Basi, Jared K. Burks, K. Rajapakshe, Yanshuo Chu, Jiahui Jiang, Daiwei Zhang, Xinmiao Yan, Paola A. Guerrero, Alejandra G Serrano, Mingyao Li, Tae Hyun Hwang, Andrew Futreal, Jaffer A Ajani, Luisa M. Solis Soto, A. Jazaeri, H. Kadara, Anirban Maitra, Linghua Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) enables gene expression analysis within spatial context of tissues. Since most data are limited to multicellular resolution, current computational methods can only estimate cell proportions. To address this gap, we introduce SpatioCell, a framework that integrates imaging data-derived morphological features with transcriptomic measurements for precise single-cell annotation. SpatioCell combines automated adaptive prompting with an optimized vision model for accurate cell segmentation and morphological analysis. Dynamic programming then optimizes cell-type assignments by integrating H&E-derived morphology and transcriptomic constraints, improving resolution within and beyond ST spots. Extensive benchmarking on histopathological, simulated, and real ST data from five distinct cancer types shows that SpatioCell outperforms state-of-the-art methods in both nuclear segmentation and cell annotation accuracy. Notably, SpatioCell reveals tumor microenvironment details, such as tumor boundaries, blood vessel structures, and immune infiltration, that were missed by other methods, while also showing strong potential in correcting deconvolution errors to further increase annotation accuracy. By redefining single-cell spatial mapping with unprecedented accuracy and resolution, SpatioCell enables precise analysis of tissue heterogeneity, offering new insights into cancer and tissue biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9557247c42d57e2f9ecb59190ef45f17e986bf3" target='_blank'>
              SpatioCell: A Deep Learning Algorithm for High-resolution Single-cell Mapping through Deep Integration of Histology Image and Sequencing Data
              </a>
            </td>
          <td>
            Naiqiao Hou, Zhaorun Wu, Yue Yu, Yanqi Zhang, Jiayun Wu, Zhixing Zhong, Fangfang Xu, Zeyu Wang, Chaoyong Yang, Weihong Tan, Jia Song
          </td>
          <td>2025-04-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e2b0637dd6be9b8d44c6621977a5ed5666d298" target='_blank'>
              Abstract 2078: Single-cell spatial transcriptomics in colon adenocarcinoma reveals tumor heterogeneity and immune microenvironmental shifts
              </a>
            </td>
          <td>
            Margaret L Hoang, Yi Cui, Shanshen He, Michael Patrick, Megan Vandenberg, Stefan Rogers, Evelyn Metzger, Haiyan Zhai, Martin Shelton, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1326a43bcf280179de51c8de12b664e06eaa82" target='_blank'>
              Single cell resolution of an epigenetic signature of persister tumor cells
              </a>
            </td>
          <td>
            Mihai Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohamed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Breast cancers present intricate microenvironments comprising heterotypic cellular interactions, yet a comprehensive spatial map remained to be established. Here, we employed the DNA nanoball-based genome-wide in situ sequencing (Stereo-seq) to visualize the geospatial architecture of 30 primary breast tumors and metastatic lymph nodes across different molecular subtypes. This unprecedented high-resolution atlas unveils the fine structure of the tumor vasculature, highlighting heterogeneity in phenotype, spatial distribution, and intercellular communication within both endothelial and perivascular cells. In particular, venular smooth muscle cells are identified as the primary source of CCL21/CCL19 within the microenvironment. In collaboration with ACKR1-positive endothelial cells, they create a chemokine-rich venular niche to synergistically promote lymphocyte extravasation into tumors. High venule density predicts increased immune infiltration and improved clinical outcomes. This study provides a detailed spatial landscape of human breast cancer, offering key insights into the venular regulation of tumor immune infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3853b930721fb6df8f03ce40a70213e79309b5e" target='_blank'>
              Spatially resolved atlas of breast cancer uncovers intercellular machinery of venular niche governing lymphocyte extravasation
              </a>
            </td>
          <td>
            Xin Wang, Zhanyu Wang, Qijun Liao, Pei Yuan, Junpu Mei, Yin Zhang, Chao Wu, Xiyu Kang, Sufei Zheng, Chen-En Yang, Jiaxiang Liu, Qingyao Shang, Jiangtao Li, Bingning Wang, Liangyu Li, Hui Liu, Weining Hu, Zhensheng Dong, Jie Zhao, Linying Wang, Tao Liu, Yusheng Den, Chengrui Wang, Lijuan Han, Qianjun Chen, Huanming Yang, Xun Xu, Jie He, Zhen Yue, Nan Sun, Xiaodong Fang, Jianming Ying
          </td>
          <td>2025-04-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="PURPOSE
To identify the specific intratumoral and microenvironmental heterogeneity of acral (AM) and mucosal melanoma (MM), we aimed to delineate their distinct cellular compositions, evolutionary trajectories, and subtype-specific therapeutic strategies.


EXPERIMENTAL DESIGN
Single-cell transcriptomic and genomic landscapes were analyzed across 42 melanoma samples (28 AM, 11 MM, and 3 non-acral cutaneous melanoma [CM]), supplemented by in vitro and in vivo validation. Tumor and stromal cells were profiled using scRNA-seq, whole-exome sequencing, and functional assays, including transwell migration, co-culture systems, and xenograft models.


RESULTS
Tumor cells exhibited divergent evolutionary routes, with MM dominated by MGP⁺/PCOLCE⁺ subpopulations showing high epithelial-to-mesenchymal transition (EMT) potential. MM displayed elevated neutrophil infiltration and CXCL3⁺ tumor-associated macrophages, while AM was enriched with PI16⁺ cancer-associated fibroblasts (CAFs) promoting tumor proliferation. Molecular classification revealed MM subtypes: an antigen-presenting subtype linked to favorable outcomes and a proliferative subtype associated with recurrence. TIGIT⁺ Treg cells were enriched in AM, suggesting targeted inhibition potential. Genomic analysis connected BRAF/NRAS mutations to ALDOA⁺ stem-like tumor cells and identified PTGDS as a therapeutic target in triple-WT melanomas.


CONCLUSIONS
Our study provides a comprehensive comparison of AM and MM, uncovering subtype-specific stromal-immune interactions and molecular programs. The findings highlight actionable targets (e.g., TIGIT in AM, CXCL3⁺ macrophages in MM) and propose a framework for precision therapies, biomarker-driven trials, and risk stratification to improve outcomes in these aggressive melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d1f1c177fc4d5a9d6abeefba72d4bbfa28d55fc" target='_blank'>
              Single-cell transcriptomic landscape deciphers intratumoral heterogeneity and subtypes of acral and mucosal melanoma.
              </a>
            </td>
          <td>
            Yunyan Li, Ziyang Cui, Xiaole Song, Yeqing Chen, Cang Li, Junfeng Shi, Wenkang Qian, Guoxin Ren, Jiang Zhou, Chunpu Li, Xiaoqing Ma, Yifan Chen, Dongdong Jia, Yongli Zhang, Zhiling Zhang, Ronghao Zhang, Zhaotian Zhang, Yong Chen, Zhixiang Xu, Wantao Chen, X. Miao, Hongmeng Yu, Jianxin Chen, Kai Wang, Colin R. Goding, Zhi Wei, Tao Li, Rutao Cui
          </td>
          <td>2025-04-07</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an exceedingly aggressive primary brain tumor defined by rapid growth, extensive infiltration, and resistance to standard therapies. A central factor driving these malignancies is the subpopulation of glioblastoma stem cells (GSCs), which possess self-renewal capacity, multipotency, and the ability to regenerate tumor heterogeneity. GSCs contribute to key hallmarks of GBM pathobiology, including relentless progression, resistance to chemotherapy and radiotherapy, and inevitable recurrence. GSCs exhibit distinct molecular signatures, enhanced DNA repair, and metabolic adaptations that protect them against conventional treatments. Moreover, they reside within specialized niches—such as perivascular or hypoxic microenvironments—that sustain stemness, promote immunosuppression, and facilitate angiogenesis. Recent discoveries highlight signaling pathways like Notch, Wnt/β-catenin, Hedgehog, STAT3-PARN, and factors such as TFPI2 and HML-2 as critical regulators of GSC maintenance, plasticity, and immune evasion. These findings underscore the complexity of GSC biology and their pivotal role in driving GBM heterogeneity and therapeutic failure. Emerging therapeutic strategies aim to target GSCs through multiple avenues, including surface markers, immunotherapeutics (e.g., CAR T cells), metabolic vulnerabilities, and combination regimens. Advances in patient-derived organoids, single-cell omics, and 3D co-culture models enable more accurate representation of the tumor ecosystem and personalized therapeutic approaches. Ultimately, improved understanding of GSC-specific targets and the tumor microenvironment promises more effective interventions, paving the way toward better clinical outcomes for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e8f7c834c39bbf5240bb0afeda29b66fdae65e" target='_blank'>
              Glioblastoma Stem Cells at the Nexus of Tumor Heterogeneity, Immune Evasion, and Therapeutic Resistance
              </a>
            </td>
          <td>
            Justin Tang, Md Al Amin, J. Campian
          </td>
          <td>2025-04-01</td>
          <td>Cells</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a50452a86dbb7c37a7b8f94f2df738ddd72d94" target='_blank'>
              3D vascularized microtumors unveil aberrant ccRCC vasculature and differential sensitivity to targeted treatments
              </a>
            </td>
          <td>
            Noémie Brassard-Jollive, Y. Atlas, Camille LM Compère, Corinne Ardidie-Robouant, Philippe Mailly, Morad El Bouchtaoui, Virginie Lelarge, Guillaume Blot, Nathalie Josseaume, Stéphanie De Oliveira, C. Hélary, Christophe Leboeuf, Isabelle Cremer, Mathilde Sibony, Guilhem Bousquet, S. Germain, L. Muller, C. Monnot
          </td>
          <td>2025-03-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) often exhibits poor response rates to immune checkpoint inhibitor (ICI) therapies, largely owing to the intricate composition and spatial organization of immune cells within the tumor-immune microenvironment (TIME). The diversity of immune cell populations, their spatial relationships, and dynamic interactions significantly influence the immunosuppressive nature of the TIME, thereby limiting the efficacy of immunotherapy. To address these challenges and enhance the therapeutic potential of ICIs in HNSCC, a comprehensive analysis of the TIME is essential. Spatial transcriptomics (ST), a cutting-edge technology, enables high-resolution mapping of gene expression within the spatial context of the tumor, providing critical insights into the functional roles and interactions of immune cells in the TIME. This review highlights the importance of ST in uncovering the complexities of the TIME in HNSCC and proposes strategies for leveraging these insights to develop more effective immunotherapeutic approaches. By integrating spatial and molecular information, this review aims to pave the way for personalized and precision-based treatments in HNSCC, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef3b95effc04f0811082363795fb3dc9666b0f8" target='_blank'>
              Mapping immunotherapy potential: spatial transcriptomics in the unraveling of tumor-immune microenvironments in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Seo-Won Choi, Jeong Heon Kim, Jisu Hong, Minsu Kwon
          </td>
          <td>2025-04-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd78a6499d1079983203268939cd1b90e4adb3f" target='_blank'>
              Abstract 154: Single-cell spatial transcriptomics unravels cell states and ecosystems associated with clinical response to immunotherapy
              </a>
            </td>
          <td>
            Z. Abdulrahman, Roderick C. Slieker, Daniel McGuire, M. J. Welters, M. V. van Poelgeest, S. H. van der Burg
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/623dee1630864493156f7c52b0b73a1aacf3c1e1" target='_blank'>
              Abstract 756: Unveiling the spatial landscape of tumor and stroma heterogeneity in colorectal cancer with spatial transcriptomics and deep learning
              </a>
            </td>
          <td>
            Jie Zhou, Todd B Sheridan, Sergii Domanskyi, Stephanie L. Cowles, SungHee Park, Ilham Putra, Olga Anczuków, Jeffrey H. Chuang, Jill C. Rubinstein
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39cd802b620e803fe61dfbfe7d21f2c3b02ece22" target='_blank'>
              Spatial mapping of transcriptomic plasticity in metastatic pancreatic cancer.
              </a>
            </td>
          <td>
            Guangsheng Pei, Jimin Min, K. Rajapakshe, Vittorio Branchi, Yunhe Liu, B. Selvanesan, F. Thege, Dorsay Sadeghian, Daiwei Zhang, K. S. Cho, Yanshuo Chu, Enyu Dai, Guangchun Han, Mingyao Li, Cassian Yee, Kazuki Takahashi, Bharti Garg, Hervé Tiriac, Vincent Bernard, A. Semaan, Jean L. Grem, T. Caffrey, Jared K. Burks, A. Lowy, Andrew J. Aguirre, P. Grandgenett, Michael A. Hollingsworth, Paola A. Guerrero, Linghua Wang, Anirban Maitra
          </td>
          <td>2025-04-23</td>
          <td>Nature</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Motivation Spatial transcriptomics holds great promise for revolutionizing biology and medicine by providing gene expression profiles with spatial information. Until recently, spatial resolution has been limited, but advances in high-throughput in situ imaging technologies now offer new opportunities by covering thousands of genes at a single-cell or even subcellular resolution, necessitating databases dedicated to comprehensive coverage and analysis with user-friendly intefaces. Results We introduce the HISSTA database, which facilitates the archival and analysis of in situ transcriptome data at single-cell resolution from various human tissues. We have collected and annotated spatial transcriptome data generated by MERFISH, CosMx SMI, and Xenium techniques, encompassing 112 samples and 28 million cells across 16 tissue types from 63 studies. To decipher spatial contexts, we have implemented advanced tools for cell type annotation, spatial colocalization, spatial cellular communication, and niche analyses. Notably, all datasets and annotations are interactively accessible through Vitessce, allowing users to focus on regions of interest and examine gene expression in detail. HISSTA is a unique database designed to manage the rapidly growing dataset of in situ transcriptomes at single-cell resolution. Given its comprehensive data content and advanced analysis tools with interactive visualizations, HISSTA is poised to significantly impact cancer diagnosis, precision medicine, and digital pathology. Availability and implementation HISSTA is freely accessible at https://kbds.re.kr/hissta/. The source code is available at https://doi.org/10.5281/zenodo.14904523.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8847bc6bc6ac698124acae8ba2d55144bc394fe" target='_blank'>
              HISSTA: a human in situ single-cell transcriptome atlas
              </a>
            </td>
          <td>
            Jiwon Yu, Jiwoo Moon, Minseo Kim, Gyeol Han, Insu Jang, Jinyoung Lim, Seungmook Lee, Seok-Hwan Yoon, Woong-Yang Park, Byungwook Lee, Sanghyuk Lee
          </td>
          <td>2025-03-31</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89052d3ed2d82ce2288e348ab845b3ae781f2935" target='_blank'>
              Abstract LB265: The spatial landscape of cancer hallmarks reveals patterns of tumor ecological dynamics and drug sensitivity
              </a>
            </td>
          <td>
            E. Porta-Pardo, Mustafa Sibai, Sergi Cervilla, D. Grases, M. Esteller
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25613f3a85e6691e66bbf8705e07ffcebf8c6cbf" target='_blank'>
              Drug and Single-Cell Gene Expression Integration Identifies Sensitive and Resistant Glioblastoma Cell Populations
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, , S. Williams, Michael E. Ivan, Ricardo J Komotar, M. I. De La Fuente, Gregory Stein, Alexander Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, N. Ayad
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is a highly lethal malignant intracranial tumor, distinguished from low-grade glioma by histopathological hallmarks such as pseudopalisading cells around necrosis (PAN) and microvascular proliferation (MVP). To date the spatial organization of the molecular and cellular components of these specific histopathological features has not been fully elucidated. Methods Here, using bulk RNA sequencing, spatial transcriptomic and single cell RNA sequencing (scRNA-seq) data of GBM patients, we identified niche-specific transcriptional programs and characterized the differences in molecular expression and cellular organization between PAN and MVP. Results Notably, we discovered spatially distinct domains within the tumor core and identified niche-specific signatures: NDRG1 and EPAS1, specifically expressed in the PAN and MVP regions. The clustering results showed two distinct phenotypes of endothelial cells (ECs) were enriched in the MVP and PAN regions, respectively. PAN-associated endothelial cells exhibit copy number variations similar to those in GBM cells. Single cell trajectory analysis reveals a pseudotime trajectory, indicating the differentiation of glioblastoma stem cells (GSCs) toward ECs. Conclusions Necrosis cores which are surrounded by hypoxic and perivascular niches and microvascular proliferation area within the glioblastoma tumor microenvironment, have been considered as standardized morphological indicators of aggressive GBM. Our findings provide a cellular and molecular insights into GBM progression. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06185-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75320d960d6bc41d40bd96ad524b1fc28b650a0b" target='_blank'>
              Multi-transcriptomics reveals niche-specific expression programs and endothelial cells in glioblastoma
              </a>
            </td>
          <td>
            Jiukun Hu, Xiaohan Sa, Yue Yang, Yuwen Han, Jie Wu, Minxuan Sun, S. Shafi, Nafees Ahmad, Sami Siraj, Jiao Yang, Yuanshuai Zhou
          </td>
          <td>2025-04-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) represents the most common pathological type of lung cancer, and the combination of neoadjuvant immunotherapy with chemotherapy has emerged as the first-line treatment for NSCLC. Nevertheless, the efficacy of this therapeutic approach remains variable. The present study aims to examine the impact of chemoimmunotherapy in NSCLC patients, with a view to identifying key molecules, critical cell subpopulations, communication patterns and spatial distributions that potentially correlate with therapeutic sensitivity. A total of 16 lung cancer tissue samples were collected from a cohort of 12 NSCLC patients and subjected to single-cell RNA and spatial transcriptome sequencing. Our data demonstrated that the distribution of CD4 + Treg T cells and mCAFs indicated an immunosuppressive tumor microenvironment, while the accumulation of CD4 + Th17 T cells and iCAFs could act as a positive marker for the sensitivity to chemoimmunotherapy. Furthermore, a significant high level of SELENOP-macrophages was observed in tissues from positive responders, and a strong co-localization between SELENOP-macrophages and antigen-presenting cancer associated fibroblasts (CAFs) in the tumor boundaries was identified, indicating the cooperative roles of these two cell types in response to combined therapy. Moreover, SELENOP-macrophages were observed to be accumulated in tertiary lymphoid structures, which further suggested its critical role in recruiting lymphocytes. Furthermore, analysis of cell–cell communication, based on spatial transcriptomics, suggests that the interactions between SELENOP-macrophages, apCAFs, CD4 + and CD8 + T cells were significantly enhanced in responders. In addition, SELENOP-macrophages recruited CD4 + Naïve, Helper and CD8 + Naïve T cells through pathways such as the cholesterol, interleukin, chemokine and HLA when responding to combined therapy. The present study further unveils the dynamic spatial and transcriptional changes in the tumor microenvironment of non-small cell lung cancer in response to combination therapy. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02287-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86da4dd75d392abc980cae7b3d81379fef557c28" target='_blank'>
              Single-cell and spatial transcriptomic analyses revealing tumor microenvironment remodeling after neoadjuvant chemoimmunotherapy in non-small cell lung cancer
              </a>
            </td>
          <td>
            Xiaolu Cui, Siyuan Liu, He Song, Jingjing Xu, Yanbin Sun
          </td>
          <td>2025-04-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Nasopharyngeal carcinoma (NPC) is a common malignancy with a complex pathogenesis and diverse cellular composition. This study aimed to systematically analyze the transcriptional characteristics of different cell types in NPC and their roles in tumor development using single-cell transcriptomics. Methods The research team collected NPC and normal tissue samples, and performed in-depth single-cell sequencing analysis of the transcriptomes. Single-cell RNA sequencing was performed on the collected samples to obtain high-resolution transcriptional profiles of individual cells. This allowed the researchers to identify and characterize the diverse cell populations present within the NPC tumor microenvironment. Results The results showed that NPC samples contained multiple distinct cell subpopulations, including epithelial cells, immune cells (such as macrophages and T cells), endothelial cells, and stromal cells. These cell types exhibited marked differences in spatial distribution and transcriptional profiles, reflecting the high degree of heterogeneity within the tumor microenvironment. Further functional analysis revealed significant dysregulation of mitochondrial-related pathways, extracellular matrix-receptor interactions, as well as Wnt, Notch, and other signaling cascades in NPC. Conclusion This study employed single-cell transcriptomics to comprehensively elucidate the complexity of the NPC tumor microenvironment, providing new insights into the underlying mechanisms of disease pathogenesis and suggesting potential therapeutic targets. These findings lay the groundwork for the development of precision medicine approaches for NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9793611f5914170a344f624509f87b1fa27aa37" target='_blank'>
              Single-cell transcriptomic landscape reveals complex cellular heterogeneity and dysregulated signaling pathways in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Xiaoyan Zhang, Zuojia Guan, Zhan Xu
          </td>
          <td>2025-05-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy characterized by early liver metastasis and high mortality. The tumor microenvironment plays a pivotal role in tumor progression; however, the immune microenvironment's involvement in PDAC liver metastasis remains poorly understood. Methods: This study investigates cellular heterogeneity in primary tumor (PT) and liver metastasis (LM) tissues of PDAC using single-nucleus RNA sequencing and spatial transcriptomics. Intra-tumor heterogeneity and cell interactions were elucidated through deconvolution, intercellular signalling, pseudotime analysis, and immune infiltration profiling. The spatial distribution of immune cells was assessed by multiplexed immunofluorescence staining, and prognostic models were developed and validated through immunohistochemistry (IHC). Analyzing the regulatory role of CITED4 in the invasion and metastasis of pancreatic cancer cells through transwell assay and scratch wound healing assay. Results: A total of 62,326 cells were sequenced, with metastatic dissemination cells showing significant upregulation of epithelial-mesenchymal transition (EMT)-related genes during liver metastasis. Spatial transcriptomics revealed the enrichment of metastatic dissemination cells and FOXP3-related Treg cells at the tumor front in PT tissues. In comparison to LM tissues, the tumor front in PT tissues fosters an immunosuppressive microenvironment through the accumulation of Treg cells. Interaction analysis identified the SPP1 pathway as a key promoter of this immunosuppressive environment. Furthermore, prognostic models highlighted CITED4 as critical biomarkers in PDAC. Elevated CITED4 expression is correlated with liver metastasis and poor prognosis in patients with PDAC. siRNA-mediated knockdown of CITED4 suppresses the invasion and metastasis of pancreatic cancer cells. Conclusions: In summary, this study revealed that Treg cell alterations, mediated by metastatic dissemination cells within the immune microenvironment, significantly contribute to PDAC liver metastasis, and that CITED4 enhances the metastatic potential of metastatic dissemination cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7cfcf741ce365a1a3bfe1ca1a5f5a71b8d0a0d3" target='_blank'>
              Single-nucleus RNA sequencing and spatial transcriptomics reveal an immunosuppressive tumor microenvironment related to metastatic dissemination during pancreatic cancer liver metastasis
              </a>
            </td>
          <td>
            Hongsen Liu, Mengting Chen, Bo Hong, Yufei Xiao, Qianming Chen, Yun Qian
          </td>
          <td>2025-04-13</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4356e915cd9638ea3a28d0766da31c4fd8473bcf" target='_blank'>
              Abstract 3703: Cell identity revealed by precise cell cycle state mapping links data modalities
              </a>
            </td>
          <td>
            Saeed S. Alahmari, Andrew Schultz, Jordan Albrecht, Vural Tagal, Zaid Siddiqui, Sandhya Prabhakaran, I. El Naqa, Alexander Anderson, Laura Heiser, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/947de9f123e944f865748b3a75364432746d6f15" target='_blank'>
              Abstract 6257: Spatial and multi-omic profiling of metastatic colorectal cancer reveals key cellular ecosystems and drivers of heterogeneity
              </a>
            </td>
          <td>
            Chien-Wei Peng, Michael D. Iglesia, Adrienne Visani, Xiang Li, Xiangwei Fang, R. Jayasinghe, Jingxian Liu, John M Herndon, Jacqueline L Mudd, Andrew Houston, Preet Lal, Siqi Chen, Ambrose Plante, A. T. D. Targino da Costa, Kapur B. Dhami, J. Ponce, Robert S Fulton, Michael Heinz, M. Chheda, Feng Chen, Ryan C. Fields, Li Ding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e55caa0c96eda1ece0ec721a1f932b608bbc992" target='_blank'>
              Spatial heterogeneity, stromal phenotypes, and therapeutic vulnerabilities in colorectal cancer peritoneal metastasis.
              </a>
            </td>
          <td>
            J. C. Ong, Joseph J Zhao, Ying Liu, Supriya Srivastava, D. Chia, Ying En Quek, Xiaonan Fan, Haoran Ma, Kiekyong Huang, T. Sheng, Qiu Xuan Tan, Gillian Ng, J. W. Tan, J. J. Lee, L. Loo, Li Yen Chong, Xue Wen Ong, S. Tay, T. Hagihara, Angie Tan, Craig Ryan Cecil Joseph, Melissa C C Teo, Josephine Hendrikson, C. Chong, Wanyu Guo, C. Chia, Jolene S M Wong, Chin Jin Seo, Mingzhe Cai, Yvonne Tay, Kevin M S Sim, R. Tay, Robert Walsh, M. Guaglio, F. Morano, Ming Teh, Huey Yew Jeffrey Lum, Tony K H Lim, Louis Vermeulen, M. Bijlsma, K. Lenos, S. Klempner, J. Yeong, W. Yong, F. Pietrantonio, P. Tan, R. Sundar
          </td>
          <td>2025-04-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a1ac1da292bdfa7cb083f4691cc613492b11ea" target='_blank'>
              Pan-cancer Analysis Identified Ectopic RUNX1T1 Associated with Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, I. Kim, Su Deng, Ping Mu
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad68c593de3480a301b38845ef8182dbdcc71b6" target='_blank'>
              Abstract 6481: Multiplexed single-cell pharmacotranscriptomics and spatial tumor biology reveals targetable reciprocal cancer-stromal interactions in prostate cancer
              </a>
            </td>
          <td>
            A. Murumägi, Frida Rantanen, Alice Dini, Mariliina Arjama, K. Välimäki, Elina Multamäki, T. Mirtti, Antti S. Rannikko, T. Pellinen, Daniela Ungureanu, Olli Kallioniemi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0444f2eee8d5a4470978d8e6c693faf1ee9e687e" target='_blank'>
              A Phenotype-Driven Multi-Omic Atlas of Glioblastoma Invasion
              </a>
            </td>
          <td>
            C. Krona, Anders Sundström, Emil Rosén, Soumi Kundu, H. Mangukiya, H. Babačić, Irem Uppman, Madeleine Skeppås, Ida Larsson, Ludmila Elfineh, Xiaofang Cao, Rashmi Ramachandra, Ramy Elgendy, Katrin Rut Halldorsdottir, Zankruti Dave, Milena Doroszko, K. Olausson, Johan Wikström, M. Pernemalm, Sven Nelander
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6af959f7143f19a656c32e73ddcf95ebee2a8fbb" target='_blank'>
              Abstract 6463: Interrogating the tumor microenvironment of nasopharyngeal carcinoma by single-cell spatial transcriptomics
              </a>
            </td>
          <td>
            Chee Yit Lim, Yi Ren, W. K. Teo, Chor Hiang Siow, K. Loh, Raymond KY Tsang, Bingcheng Wu, Joshua K. Tay
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddc6a274f48d59a3fed08cd82e2b131cfa0c1209" target='_blank'>
              Abstract 2779: Whole transcriptome spatial exploration of cultured tumor cells at subcellular resolution
              </a>
            </td>
          <td>
            Yi Cui, Margaret L Hoang, Shanshan He, Rachel Liu, Tushar Rane, R. Khafizov, Haiyan Zhai, Christine Kang, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6ce39bb3cec413d591d56991415df20b385ea43" target='_blank'>
              Abstract 2705: Spatial profiling of tumor microenvironment uncovers potential novel therapeutic approach in pediatric neuroblastoma
              </a>
            </td>
          <td>
            Cui Tu, Chin Wee Tan, J. Monkman, Ana Clara Almeida, Gabrielle Antonio-Carreon, Natacha Omer, Giaan Hull, Kimberly Chung, Ahmed M Mehdi, Antonietta Salerno, Orazio Vittorio, Aaron Mayer, Cleber Machado de Souza, Fernanda de Almeida Brehm Pinhati, Selene Elifio-Esposito, Wayne Nicholls, Lucia de Noronha, A. Kulasinghe, Fernando Souza-Fonseca-Guimaraes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e024d7323084416b4b6e84dca254aca71e425ab" target='_blank'>
              Learning the cellular origins of cancer using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Bruno Giotti, Wooseung Lee, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract We developed a deep learning Pathomics image analysis workflow to generate spatial Tumor-TIL maps to visualize and quantify the abundance and spatial distribution of tumor infiltrating lymphocytes (TILs) in colon cancer. Colon cancer and lymphocyte detection in hematoxylin and eosin (H&E) stained whole slide images (WSIs) has revealed complex immuno-oncologic interactions that form TIL-rich and TIL-poor tumor habitats, which are unique in each patient sample. We compute Tumor%, total lymphocyte%, and TILs% as the proportion of the colon cancer microenvironment occupied by intratumoral lymphocytes for each WSI. Kaplan-Meier survival analyses and multivariate Cox regression were utilized to evaluate the prognostic significance of TILs% as a Pathomics biomarker. High TILs% was associated with improved overall survival (OS) and progression-free interval (PFI) in localized and metastatic colon cancer and other clinicopathologic variables, supporting the routine use of Pathomics Tumor-TIL mapping in biomedical research, clinical trials, laboratory medicine, and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b4f8d94ce78fc5fa82e89d6b995817192c3e592" target='_blank'>
              Pathomics Image Analysis of Tumor Infiltrating Lymphocytes (TILs) in Colon Cancer
              </a>
            </td>
          <td>
            Yuwei Zhang, Shahira Abousamra, Mahmudul Hasan, Luke A. Torre-Healy, S. Krichevsky, Sampurna Shrestha, Erich Bremer, Derek A. Oldridge, Andrew J. Rech, E. E. Furth, Therese J. Bocklage, Justin S. Levens, Isaac Hands, Erich B. Durbin, D. Samaras, T. Kurç, J. Saltz, Rajarsi Gupta
          </td>
          <td>2025-04-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/527fb3619bec4cdd8432c9c09f5972cd49b69aac" target='_blank'>
              Abstract 5276: Comprehensive spatial mapping uncovers immune ecotypes linked to tumor heterogeneity in HCC lung metastasis
              </a>
            </td>
          <td>
            Jinghan Zhu, Guangzhen Cai, Bin Guo, Xiaoping Chen, Huilan Zhang, Hong Zhu, Wanguang Zhang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1368b434a243d669a84298d854a992ef89d9558b" target='_blank'>
              Atlas-scale metabolic activities inferred from single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Erick Armingol, James Ashcroft, Magda Marečková, M. Prete, Valentina Lorenzi, C. Mazzeo, Jimmy Tsz Hang Lee, Marie Moullet, O. Bayraktar, Christian M Becker, K. Zondervan, L. Garcia-Alonso, Nathan E. Lewis, R. Vento-Tormo
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) has transformed our understanding of tissue architecture and cellular interactions, but integrating ST data across platforms remains challenging due to differences in gene panels, data sparsity, and technical variability. Here, we introduce Lloki, a novel framework for integrating imaging-based ST data from diverse platforms without requiring shared gene panels. Lloki addresses ST integration through two key alignment tasks: feature alignment across technologies and batch alignment across datasets. Optimal transport-guided feature propagation adjusts data sparsity to match scRNA-seq references through graph-based imputation, enabling single-cell foundation models such as scGPT to generate unified features. Batch alignment then refines scGPT-transformed embeddings, mitigating batch effects while preserving biological variability. Evaluations on mouse brain samples from five different technologies demonstrate that Lloki outperforms existing methods and is effective for cross-technology spatial gene program identification and tissue slice alignment. Applying Lloki to five ovarian cancer datasets, we identify an integrated gene program indicative of tumor-infiltrating T cells across gene panels. Together, Lloki provides a robust foundation for cross-platform ST studies, with the potential to scale to large atlas datasets, enabling deeper insights into cellular organization and tissue environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e90b432ee960f45325b36f2ea9a08b0d4152a5" target='_blank'>
              Unified integration of spatial transcriptomics across platforms
              </a>
            </td>
          <td>
            Ellie Haber, Ajinkya Deshpande, Jian Ma, Spencer Krieger
          </td>
          <td>2025-04-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483a65f81564c748a7e6be963230e929d2717f09" target='_blank'>
              Abstract 153: High-resolution profiling of the tumor microenvironment with spatial ecotypes
              </a>
            </td>
          <td>
            Wubing Zhang, Erin L. Brown, A. Usmani, N. Earland, Hyun Soo Jeon, I. Alahi, N. Semenkovich, Janella C. Schwab, Chloé B. Steen, Chloe M. Sachs, Bilge Gungoren, F. Qaium, P. Harris, Jacqueline L Mudd, Qingyuan Cai, A. Bacchiocchi, Ryan C. Fields, Mario Sznol, R. Halaban, David Y. Chen, A. Chaudhuri, Aaron M. Newman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcbbcac8a736304b9cab15ef0da632985a22219d" target='_blank'>
              Abstract 2666: Mapping intratumoral heterogeneity in colorectal cancer: Spatially resolved subclones and tumor microenvironment
              </a>
            </td>
          <td>
            YeonJu Lee, Woong-Yang Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Gallbladder cancer (GBC) is a rare but aggressive malignancy, often detected late due to early asymptomatic stages. Understanding cellular and molecular changes from normal tissue to high-grade intraepithelial neoplasia (HGIN) and invasive GBC is vital for identifying early biomarkers and therapeutic targets. Methods: We performed single-cell RNA sequencing on 98,113 cells derived from 2 normal adjacent tissues (NAT), 2 HGIN, and 6 GBC samples. The cellular diversity and heterogeneity, particularly within epithelial and immune cell populations in NAT–HGIN–GBC, were investigated utilizing single-cell RNA sequencing, bulk RNA sequencing (bulk RNA-seq), and 10 machine learning methodologies. Furthermore, the intercellular crosstalk between epithelial cells and tumor immune microenvironment cells was examined and validated through multiplex immunofluorescence staining. Results: The constructed cell atlas elucidated alterations in the immune landscape across various states of NAT–HGIN–GBC, highlighting a more pronounced inhibitory immune microenvironment in GBC. The epithelial subtype TOP2A+ Epi is markedly elevated in GBC and is correlated with a poor prognosis. Key genes associated with this subtype may include GMNN, CYTOR, KLK6, and BIRC5. Similarly, immunosuppressive macrophages, identified as TOP2A+ Macro, also increase along the NAT–HGIN–GBC sequence and are linked to reduced patient survival. Furthermore, TOP2A+ Macro and CD8+ exhausted T cells (CD8+ Tex) engage in intercellular communication with epithelial TOP2A+Epi cells via the TWEAK/FN14 signaling pathway, thereby promoting tumor progression and immune evasion in GBC. The findings were further corroborated through multiplex immunofluorescence staining conducted on specimens from patients. Conclusions: This study elucidates significant alteration in the cellular ecosystems and intercellular signaling within the tumor immune microenvironment across the NAT–HGIN–GBC sequence. It identifies TOP2A, TWEAK, and FN14 as potential biomarkers and therapeutic targets for GBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f8ecb5a2d68898a868d59ee319a91eaeca48bb" target='_blank'>
              Single-cell analyses unravel ecosystem dynamics and intercellular crosstalk during gallbladder cancer malignant transformation
              </a>
            </td>
          <td>
            Zhaobin He, Jianqiang Cao, Xiqiang Wang, Shengbiao Yang, Huijie Gao, Yongzhe Yu, Zequn Di, Cheng Peng
          </td>
          <td>2025-05-16</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f29ad2e19f93fc33c2a089984de394bfc868d5e" target='_blank'>
              Abstract 3962: Pan-cancer spatial characterization of tertiary lymphoid structures
              </a>
            </td>
          <td>
            K. S. Cho, Yunhe Liu, Guangsheng Pei, Jianfeng Chen, Tieling Zhou, Khalida M Wani, Alejandra G Serrano, Sharia D Hernandez, Akshaya S. Jadhav, Daiwei Zhang, Hannah C. Beird, Jiahui Jiang, Diana Shamsutdinova, Xinmiao Yan, Yibo Dai, Enyu Dai, Ashley L. Wilson, Anne Waldrep, F. Peng, Wei Lu, Yang Liu, Jimin Min, Paola A. Guerrero, Idania C. Julio, Tong Yin, Shin Hsu Shi, Sabitha Prabhakaran, S. Kopetz, D. Maru, Alexander J. Lazar, A. Jazaeri, Michael Green, Mingyao Li, C. Sautès-Fridman, Wolf H. Fridman, Andrew Futreal, L. Soto, H. Kadara, Anirban Maitra, Suet Yi Leung, Jianjun Gao, Linghua Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1ca3901ac698b79cb29b7a3967fd687d4414a1" target='_blank'>
              Multiomic profiling of hypoxic glioblastoma stem cells reveals expansion of subpopulations with distinct epigenetic and CNV profiles
              </a>
            </td>
          <td>
            Adrianne Corseri, Travis Moore, Nicole Szczepanski, Hyeyeon Hwang, Alan Zdon, Gurkan Yardimci Galip, Nikos Tapinos
          </td>
          <td>2025-05-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35acfa591bf2d48662dc328bde4741026068f8bb" target='_blank'>
              Neoantigens and Stochastic Fluctuations Regulate T Cell Proliferation in Primary and Metastatic Malignant Brain Tumors
              </a>
            </td>
          <td>
            Maheshwor Poudel, William Stewart, Ciriyam Jayaprakash, Jayajit Das
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f160a16e8e12a7fc63680546f0557a806342875" target='_blank'>
              Charting the Single-Cell and Spatial Landscape of IDH-Wildtype Glioblastoma with GBmap.
              </a>
            </td>
          <td>
            Cristian Ruiz-Moreno, Sergio Marco Salas, Erik R Samuelsson, Mariia Minaeva, Ignacio L Ibarra, Marco Grillo, S. Brandner, Ananya Roy, K. Forsberg-Nilsson, M. Kranendonk, Fabian J. Theis, Mats Nilsson, H. Stunnenberg
          </td>
          <td>2025-05-02</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8315f0147f2ad2f550ec4867f837baf7efd724be" target='_blank'>
              Abstract 156: Tumor microenvironment profiling of multiplexed images in head and neck cancer reveals spatial features associated with immunotherapy response
              </a>
            </td>
          <td>
            E. Markovits, J. Monkman, Aaron Kilgallon, T. Aung, David L Rimm, A. Kulasinghe
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1bafbb3ba9f42ba8adab0b213f43d4265484aab" target='_blank'>
              Abstract 2100: Integrative spatial analysis reveals tumor heterogeneity and immune colony niche related to clinical-outcomes in small cell lung cancer
              </a>
            </td>
          <td>
            C. Deng, Changbin Zhu, Tao Hu, Jing Wang, Haiquan Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/486a09b849707de5f94be47aaa6a9ceac4469f38" target='_blank'>
              Joint Modeling of Spatial Dependencies Across Multiple Subjects in Multiplexed Tissue Imaging
              </a>
            </td>
          <td>
            Joel Eliason, Arvind Rao, Timothy L. Frankel, Michele Peruzzi
          </td>
          <td>2025-04-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2a1c8fd12302b7cec59cd6c236cab65714760b" target='_blank'>
              Abstract 770: ccRCC metastasis rewires tumor microenvironment to increase immunosuppression and fibrosis
              </a>
            </td>
          <td>
            Yufei Wang, Jae-Won Cho, Y. Laimon, Wenxin Xu, A. B. Sheshdeh, Nithyassree Murugan, Hsien-Chi Yuan, Jon Wee, David Braun, Toni K. Choueiri, Catherine J. Wu, S. Signoretti, Gordon Freeman, M. Hemberg, Wayne A. Marasco
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d351bbaebd26c85c6704dd8dccdfca1b91ad8729" target='_blank'>
              Single cell spatial transcriptomics integration deciphers the morphological heterogeneity of atherosclerotic carotid arteries
              </a>
            </td>
          <td>
            J. Pauli, Daniel Garger, Fatemeh Peymani, J. Wettich, N. Sachs, Johannes Wirth, Katja Steiger, Hanrui Zhang, Ira Tabas, Alan Tall, Mingyao Li, Muredach P. Reilly, D. Branzan, Holger Prokisch, M. Menden, Lars Maegdefessel
          </td>
          <td>2025-04-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast cancer (BRCA) is a complex disease influenced by the tumor microenvironment, where interactions between immune cells and cancer cells play a crucial role in tumor progression and response to therapy. Understanding the intricacies of these interactions requires detailed analysis at the single-cell level, enabling the identification of specific immune cell subpopulations and their functional roles within the tumor milieu. This study comprehensively analyzed immune cell subpopulations and macrophage subtypes in BRCA using single-cell RNA sequencing technology and various computational tools. Initially, Sc-Type software accurately identified and annotated immune cell subpopulations, followed by CNV analysis using infercnv software, revealing significant CNV variations in epithelial cells. Subsequently, macrophages were re-clustered into 5 clusters, and their biological significance and functional features were assessed. CellChat analysis elucidated potential interactions between macrophage subtypes and BRCA cells, primarily through SPP1-CD44 and LGALS9-CD44 signaling networks. Additionally, CytoTRACE and Monocle were employed to analyze cellular plasticity and differentiation trajectories of macrophage subtypes. Furthermore, efferocytosis-related gene set scoring, transcription factor analysis, and risk score development were conducted, followed by immune infiltration and tumor mutation burden analysis, revealing increased immune infiltration and higher TMB levels in the high-risk group. These findings offer crucial insights into the interaction mechanisms of immune cells and macrophage subtypes within the BRCA tumor microenvironment, aiding in the understanding of tumor progression and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61362329e16982a536f86bb6af1a06a1e1ff5d77" target='_blank'>
              Mapping immune cell dynamics and macrophage plasticity in breast cancer tumor microenvironment through single-cell analysis
              </a>
            </td>
          <td>
            Wang Chen, Siyu Zeng, Junyong Zhong, Jian Zou, Yanli Lei, Xiaohan Chen, Qinghua Mei, Qianhua Luo
          </td>
          <td>2025-04-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1477b05d2464a1bcee44f14957311d44000b130" target='_blank'>
              scTransMIL Bridges Patient-Level Phenotypes and Single-Cell Transcriptomics for Cancer Screening and Heterogeneity Inference
              </a>
            </td>
          <td>
            Zhenchao Tang, Fang Wang, Fan Yang, Jiangning Song, C. Chen, Jianhua Yao
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Background Gastric cancer (GC) exhibits high heterogeneity that relies on the oncogenic properties of cancer cells and multicellular interactions in the tumour microenvironment. However, the heterogeneity of GC and their molecular characteristics are still largely unexplored. Methods We employed single‐cell and spatial transcriptomics to comprehensively map the intra‐tumoural heterogeneity within GC. Additionally, in vitro experiments, clinical sample analyses, and patient‐derived organoid models (PDOs) were conducted to validate the key interaction patterns between tumor cells and stromal cells. Results Seven robust meta‐programs (MP1–MP7) in GC were defined with distinct biological significance and spatial distributions. MP3 and MP4 were intimately associated with distinct CD8 T cells skewed toward a cytotoxic or exhaustion state, while MP7, characterised by the highest degree of malignancy, harboured an immune lockdown microenvironment around it and spatially associated with myofibroblasts (myCAFs). Notably, we clarified the interplay between the MP7 and myCAFs, where MP7 induces the chemotactic migration of fibroblasts and promoting their transformation into myCAFs via GDF15/TGFBR2, and in turn, myCAFs‐derived RSPO3 up‐regulates EGR1 to promote the transformation to MP7 in GC cells and human PDOs. Ultimately, the accumulation of myCAFs around MP7 led to fewer infiltration of CD8 T cells, resulting an immune‐deprived microenvironment and the diminished efficacy of immunotherapy. Additionally, based on the gene expression signatures of MP7 GC cells, we predicted specific drugs and verified more potent inhibitory effects of Taselisib and Lapatinib for MP7 GC cells than conventional drugs at the same concentration. Conclusion Taken together, these results deepened the understanding of GC heterogeneity and paved the way for novel therapeutic strategies by targeting MP7 GC cells and their interaction loop with myCAFs in GC treatment. Key points Seven robust meta‐programs (MP1‐MP7) were identified in gastric cancer. MP7 was strongly correlated with cancer metastasis and poor survival of gastric cancer patients. MP7 promoted fibroblast transformation into myCAFs via GDF15/TGFBR2, creating an immune lockdown microenvironment. MyCAFs induced MP7 transformation via the RSPO3/EGR1 pathway, promoting gastric cancer cell migration. Taselisib and Lapatinib were potent inhibitors of MP7 GC cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6915321c1b17b2ab2500b930b7287f803a2304b" target='_blank'>
              Deciphering of intra‐tumoural heterogeneity and the interplay between metastasis‐associated meta‐program and myofibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xiong-yan Wu, Zhijian Jin, Baolong Li, Yifan Lu, Junyi Hou, Lizhong Yao, Zhenjia Yu, Qingqing Sang, Beiqin Yu, Jian-fang Li, Chen Li, Chao Yan, Zhenggang Zhu, Kaiwen Tang, Bingya Liu, Liping Su
          </td>
          <td>2025-04-28</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1eb1624eaa05c3beff7c464020ea5f0d816374c" target='_blank'>
              Spatial proteomics of the tumor microenvironment in melanoma: current insights and future directions
              </a>
            </td>
          <td>
            Chiara Bungaro, Michele Guida, Benedetta Apollonio
          </td>
          <td>2025-05-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c0b646451e8f6a513c46aa0a01996017a71aac6" target='_blank'>
              Abstract 5322: Spatial profiling of the tumor microenvironment in colorectal cancer: Insights into immune, tumor, and stromal interactions using high-dimensional protein analysis
              </a>
            </td>
          <td>
            Enrique Velazquez Villarreal, E.-W. Yang, Fernando C. Diaz, Maria Ninova
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d545d97bc8751905973e21664bea8353ccb95ea" target='_blank'>
              Abstract 7475: A comprehensive single-cell atlas of gastric cancer reveals malignant cell state dynamics and plasticity in tumor progression and metastasis
              </a>
            </td>
          <td>
            Enyu Dai, Yanshuo Chu, Yang Liu, Ruiping Wang, Yibo Dai, Jaffer A Ajani, Linghua Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Neoadjuvant chemo-immunotherapy has significantly improved the treatment landscape for patients with ALK/EGFR wt resectable non-small cell lung cancer (NSCLC), offering novel opportunities for translational and clinical investigations. By leveraging deep molecular profiling through spatial transcriptomics integrated with single-cell RNA-sequencing from public atlases, and serum proteomic profiling, we report a case of a patient with resectable NSCLC and a solitary synchronous brain metastasis showing a complete pathologic response after chemo-immunotherapy, and a durable event-free survival. The deep profiling of the tumor immune microenvironment of both pre-treatment brain metastasis and post-treatment primary lung tumor tissues unveiled a key role of B lymphocytes at different maturation state, spanning from naïve to plasma cells, in mediating tumor eradication. Notably, the formation of several tertiary lymphoid structures in the regression bed of post-treatment primary tumor was observed, suggesting the in situ generation of high-affinity antibody and specific immune memory response. This multimodal approach paves the way for the discovery of novel biomarkers at both tissue and systemic levels, fostering improved patient stratification and guiding clinical decisions on post-surgical treatment escalation or de-escalation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ca3e849d66be330813305765da5566fa9efc94" target='_blank'>
              Spatial transcriptomics highlights B cells as key contributors to a complete and durable response to chemo-immunotherapy in a patient with resectable NSCLC
              </a>
            </td>
          <td>
            Nicla Porciello, F. Gallina, G. Frisullo, Francesca Fusco, L. D’Ambrosio, V. Balzano, F. de Nicola, Paolo Visca, L. Landi, F. Cappuzzo, Paola Nisticò
          </td>
          <td>2025-05-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8623aa3b6277db74a5df00d2e59a3be79b68bfd2" target='_blank'>
              Abstract 2418: CellClique: Dissecting tumor microenvironments at the single cell level using generative AI and spatial transcriptomics
              </a>
            </td>
          <td>
            Sachini Weerasekara, Natasha Darras, Nicolas Fernandez, Melinda Chen, Alina Ainbinder, Colles Price
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca7ecdbac1d09bcfe33d7f9c228aa944ed8e3ef5" target='_blank'>
              Abstract 6484: Single-cell transcriptomic analysis of hepatocellular carcinoma: elucidating intratumoral heterogeneity and immune microenvironment crosstalk
              </a>
            </td>
          <td>
            Qingyang Zhang, Y. Tsui, Vanilla Xin Zhang, Daniel Wai-Hung Ho, Irene Oi-Lin Ng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Gliomas, including both glioblastoma multiforme (GBM) and lower-grade glioma (LGG), present a substantial challenge in neuro-oncology because of genetic heterogeneity and unsatisfactory prognosis. This study aimed to conduct a comprehensive multi-omics analysis of gliomas using various bioinformatics approaches to identify potential therapeutic targets and prognostic markers. A comprehensive analysis was conducted on 1327 sequencing data samples alongside their relevant clinical information sourced from The Cancer Genome Atlas (TCGA) pertaining to glioblastoma (GBM), low-grade glioma (LGG), the Chinese Glioma Genome Atlas (CCGA) and University of California Santa Cruz Xena (UCSC Xena) datasets. These tools were employed for gene expression profiling, survival analysis, and cell communication mapping. Spatial transcriptomics revealed the localization of mesenchymal (MES)-like malignant tumors, and drug sensitivity analysis was performed to evaluate responses to quinpirole and meropenem. Additionally, the Tumor Immune Dysfunction and Exclusion (TIDE) framework was utilized to gauge the responsiveness to immunotherapy. The MES-like malignant and monocyte/macrophage (mono/macro) cell subsets showed high hallmark scores, playing key roles in the tumor microenvironment. MES-like malignant marker gene scores correlated with overall survival across datasets, whereas mono/macro marker gene scores were significant in the TCGA-LGG and CCGA datasets. Key interactions between these cell types were found, especially with CD14-ITGB2, LGALS1-CD69, and APOE-TREM2. The mono/macro cell subset demonstrated better immune therapy responsiveness, as indicated by lower TIDE scores. Spatial transcriptomics revealed that MES-like malignant tumors are predominantly localized in four distinct regions, with the marker genes CHI3L1 and ADM confirming these locations. Drug sensitivity analysis revealed differential responses of the MES-like malignant cell subset to quinpirole and meropenem. Our results offer fresh perspectives on the differential roles of MES-like malignant and monocyte/macrophage cell subsets in tumor progression and immune modulation, providing novel insights into glioma biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ece6067255a1933f97765246f767fae55de1c3a5" target='_blank'>
              A spatial transcriptomics study of MES-like and mono/macro cells in gliomas
              </a>
            </td>
          <td>
            Boyan Zhao, Jianing Wu, Tiehui Zhang, Mingyang Han, Cheng Zhang, Xuan Rong, Ruotian Zhang, Xin Chen, Fei Peng, Jin Jin, Shiya Liu, Xingli Dong, Shiguang Zhao
          </td>
          <td>2025-04-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afbfafb6e752991df103b290fb3be622c923aa5e" target='_blank'>
              Immune Spatial Organization Predicts Metastasis Risk in Aggressive Localized Prostate Cancer
              </a>
            </td>
          <td>
            David D Yang, Aya Abdelnaser, Jeremiah Wala, Alexander J Haas, Alfred A Barney, Eddy Saad, Jett Crowdis, Cora A. Ricker, Jihye Park, Martin T King, P. L. Nguyen, T. Choueiri, A. Tewari, Keyan Salari, M. Taplin, Chin-Lee Wu, E. V. Van Allen
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a major health concern, with over 150,000 new diagnoses and more than 50,000 deaths annually in the United States, underscoring an urgent need for improved screening, prognostication, disease management, and therapeutic approaches. The tumor microenvironment (TME)- comprising cancerous and immune cells interacting within the tumor's spatial architecture- plays a critical role in disease progression and treatment outcomes, reinforcing its importance as a prognostic marker for metastasis and recurrence risk. However, traditional methods for TME characterization, such as bulk transcriptomics and multiplex protein assays, lack sufficient spatial resolution. Although spatial transcriptomics (ST) allows for the high-resolution mapping of whole transcriptomes at near-cellular resolution, current ST technologies (e.g., Visium, Xenium) are limited by high costs, low throughput, and issues with reproducibility, preventing their widespread application in large-scale molecular epidemiology studies. In this study, we refined and implemented Virtual RNA Inference (VRI) to derive ST-level molecular information directly from hematoxylin and eosin (H&E)-stained tissue images. Our VRI models were trained on the largest matched CRC ST dataset to date, comprising 45 patients and more than 300,000 Visium spots from primary tumors. Using state-of-the-art architectures (UNI, ResNet-50, ViT, and VMamba), we achieved a median Spearman's correlation coefficient of 0.546 between predicted and measured spot-level expression. As validation, VRI-derived gene signatures linked to specific tissue regions (tumor, interface, submucosa, stroma, serosa, muscularis, inflammation) showed strong concordance with signatures generated via direct ST, and VRI performed accurately in estimating cell-type proportions spatially from H&E slides. In an expanded CRC cohort controlling for tumor invasiveness and clinical factors, we further identified VRI-derived gene signatures significantly associated with key prognostic outcomes, including metastasis status. Although certain tumor-related pathways are not fully captured by histology alone, our findings highlight the ability of VRI to infer a wide range of "histology-associated" biological pathways at near-cellular resolution without requiring ST profiling. Future efforts will extend this framework to expand TME phenotyping from standard H&E tissue images, with the potential to accelerate translational CRC research at scale.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9c463795082212654b2fe4f8e1d20c9792788e" target='_blank'>
              Histology-Based Virtual RNA Inference Identifies Pathways Associated with Metastasis Risk in Colorectal Cancer
              </a>
            </td>
          <td>
            Gokul Srinivasan, Minh-Khang Le, Zarif L. Azher, Xiaoying Liu, Louis J Vaickus, Harsimran Kaur, F. Kolling, Scott M. Palisoul, L. Perreard, Ken S. Lau, Keluo Yao, Joshua Levy
          </td>
          <td>2025-04-23</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Spatial transcriptomics has revolutionized our understanding of tissue organization by simultaneously capturing gene expression and spatial localization within intact tissues. However, analyzing these increasingly complex datasets requires specialized expertise across computational biology, statistics, and biological context. To address this challenge, we introduce the Spatial Transcriptomics AI Agent (STAgent), an autonomous multimodal agentic AI that integrates multimodal large language models (LLMs) with specialized computational tools to transform weeks-long analysis tasks into minutes of automated processing. Unlike conventional machine learning approaches that are limited to narrow, predefined tasks, STAgent leverages the emergent capabilities of multimodal LLMs – such as flexible reasoning, contextual understanding, and cross-modal integration – which allow it to adapt to novel data, execute multi-step analyses, and generate biologically meaningful insights with minimal human input. STAgent enables autonomous deep research through integrated capabilities, including dynamic code generation for complex analytical workflows, visual reasoning for interpreting spatial patterns, real-time retrieval of relevant peer-reviewd scientific literature, and synthesis of comprehensive, actionable reports. We applied STAgent to investigate the in vivo maturation of human stem cell-derived pancreatic cells (SC-pancreas) transplanted into immunodeficient mice. We generated single-cell spatial transcriptomics data spanning multiple developmental timepoints. STAgent autonomously (1) identified the maturation of initially scattered endocrine cells into well-defined islet-like structures, with predominantly peripheral α-cells surrounding β-cell cores supported by an expanding mesenchymal network; (2) revealed strengthening endocrine-endocrine cell interactions over time and, through context-aware gene set analysis, uncovered spatially resolved biological processes driving maturation; (3) unlike traditional analytical approaches, STAgent offers mechanistic explanations of spatial patterns, contextualizing findings with relevant literatures and developing cohesive insights into human pancreatic development. This agentic approach establishes a new paradigm in spatial transcriptomics analysis by substantially lowering the expertise barrier and reducing analysis time, accelerating biological and biomedical discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c45bfa74d72bc131f7254ae28e5d7ee1abd425f" target='_blank'>
              Spatial transcriptomics AI agent charts hPSC-pancreas maturation in vivo
              </a>
            </td>
          <td>
            Zuwan Lin, Wenbo Wang, Arnau Marin-Llobet, Qiang Li, Samuel D. Pollock, Xin Sui, Almir Aljović, Jaeyong Lee, Jongmin Baek, Ningyue Liang, Xinhe Zhang, Connie Kangni Wang, Jiahao Huang, Mai Liu, Zihan Gao, Hao Sheng, Jin Du, Stephen J. Lee, Brandon Wang, Yichun He, Jie Ding, Xiao Wang, Juan R. Alvarez-Dominguez, Jia Liu
          </td>
          <td>2025-04-04</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99969b328faea200d4bc4333f10cd0e2095990a" target='_blank'>
              Abstract 5302: Mapping the tumor microenvironment at single-cell resolution with Stereo-seq v1.3
              </a>
            </td>
          <td>
            Jiajun Zhang, Xing Liu, Zexian Zeng, Rui Zhang, Tibby Tianbi Duan, Jing Guo, Liang Wu, Sha Liao, Ao Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195e5c9d321296053fe8059a5c1ae5b1f465a2aa" target='_blank'>
              Abstract 5054: Understanding the genetic heterogeneity within clonal spheroids and its interaction with the tumor microenvironment
              </a>
            </td>
          <td>
            Anshu Yadav, Rupesh Chaturvedi, Rana P. Singh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Computed microtomography (micro-CT) is a non-destructive imaging technique that enables 3D visualization of tumor progression at a resolution comparable to light microscopy without the artifacts common to conventional histology. In this study, we performed contrast-enhanced micro-CT imaging of esophageal cancer specimens via a murine orthotopic xenograft model. We visualized tumor growth, invasion, and metastasis at high resolution, preserving the natural tissue microarchitecture. We highlighted the powerful application of micro-CT in capturing the complexity of tumor invasion and dissemination, uncovering tissue-remodeling strategies. Additionally, our findings suggest that collectively migrating tumor cells exhibit organized behavior reminiscent of early multicellular organisms, reflecting the adaptive capabilities of tumor cells and their potential evolutionary origins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba801a47122ebd0ac02a5f7a4da48677fdc3e738" target='_blank'>
              Unveiling Another Dimension: Advanced Visualization of Cancer Invasion and Metastasis via Micro-CT Imaging
              </a>
            </td>
          <td>
            S. Tkachev, Vladimir Brosalov, Oleg Kit, Alexey Maksimov, Anna Goncharova, Evgeniy Sadyrin, Alexandra Dalina, E. Popova, Anton Osipenko, M. Voloshin, Nikolay Karnaukhov, Peter Timashev
          </td>
          <td>2025-03-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c5186055b7098bddfe8ece68d797911822c432e" target='_blank'>
              Abstract 6290: sc2DAT: workflow for targeting tumor subpopulations of single cells
              </a>
            </td>
          <td>
            Giacomo B. Marino, Nasheath Ahmed, Daniel J. B. Clarke, Avi Ma’ayan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f3af6173f4bd387223a85f81701f7e3b9e33817" target='_blank'>
              Abstract 1241: Deciphering heterogeneity and tumor evolution in bladder cancer using multi-omics and liquid biopsies from rapid autopsies
              </a>
            </td>
          <td>
            P. Itagi, Samantha L. Schuster, Sonali Arora, Patricia Galipeau, Thomas W Persse, F. Vakar‐López, John K. Lee, P. Grivas, R. Montgomery, Jonathan L. Wright, Hung-Ming Lam, Andrew C. Hsieh, Gavin Ha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Purpose The aim of this study is to explore the pathological niche of cancer metastasis and the site-specific interactions between tumor cells and the microenvironment, understand the mechanisms driving metastasis progression and identify potential therapeutic targets. Methods Data from four lung cancer metastasis datasets (GSE123902, GSE131907, GSE148071, and GSE186344) were downloaded and subjected to stringent quality control and filtering. Cell types were identified using canonical markers, and pseudotime trajectory analysis was performed to evaluate cell differentiation. Functional and pathway enrichment analyses, including ssGSEA and GO/KEGG, were conducted. CellphoneDB was used to analyze intercellular communication, ranking receptor-ligand interactions based on communication strength. Results Eleven cell types were identified after quality control, revealing significant heterogeneity and site-specific functionality in lung cancer metastases. CTLs showed notable activity in antigen presentation and T-cell differentiation pathways, with DNAJB1⁺ CTLs playing a dominant role in cytotoxicity and immune regulation. B cells, myeloid cells, and CAFs were involved in immune modulation, defense, and matrix remodeling through specific signaling pathways. Tumor cell subclusters drove proliferation, migration, and immune evasion via immune-regulatory, Hippo, and TGF-beta pathways. No overlapping pathways were observed across metastatic sites. Cell communication analysis identified PPIA-BSG and APP-CD74 as key axes in brain and lymph node metastases, while FN1-Integrin and CTLA4-CD86 dominated in bone and adrenal metastases, respectively. Conclusions In summary, this study highlights the functional heterogeneity and site-specific interactions of cells in lung cancer metastases, providing insights into the mechanisms shaping metastatic niches and potential therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02269-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c4f954c571841166277c9ca7ec19c26848339cb" target='_blank'>
              Single-cell profiling uncovers the intricate pathological niche diversity in brain, lymph node, bone, and adrenal metastases of lung cancer
              </a>
            </td>
          <td>
            Le Liu, Yuan Zhou, Zhenjun Ye, Zhiyong Chen, Benchao Yuan, Liyi Guo, Haiyan Zhang, Yuanyuan Xu
          </td>
          <td>2025-04-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1870d444cf3f0fec43c774decd3ef3d91c4b6138" target='_blank'>
              Abstract 6622: Dissecting in situ states of invading glioblastoma cells using single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Toshiro Hara
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c16f756ddb575cdf26b92c703a6e6c80ae8425f" target='_blank'>
              Abstract 6358: Single cell transcriptional dynamics in the HCMI cancer model collection
              </a>
            </td>
          <td>
            Dina Elharouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, T. Hinoue, Sean A. Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W. Kyi, Julyann Pérez-Mayoral, Megan J. Stine, Eva Y. Tonsing-Carter, James M. Clinton, The Hcmi Network, Peter W. Laird, Calvin J. Kuo, O. Elemento, David L. Spector, Andrew D Cherniack, Kyle Elrott, M. Ferguson, R. Beroukhim, Katie Hoadley, N. Robine, Andrew McPherson, M. Garnett, D. Tuveson, Andrea Califano, Paul T. Spellman, Keith L. Ligon, Daniela S. Gerhard, Louis M. Staudt, Jesse S. Boehm
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>149</td>
        </tr>

        <tr id="Malignant pleural mesothelioma (MPM) is a rare yet highly aggressive malignancy with a severe prognosis. Compounded by the lack of effective treatment modalities, MPM remains a formidable health challenge. Therefore, the identification of actionable liabilities is critical for advancing precision medicine to combat this lethal disease. Here, we exploit an unbiased genome-wide CRISPR screen, integrating and cross-comparing three MPM cell lines with nonmalignant mesothelial cells, to selectively map the gene targets whose depletion indicates a common dependency in MPM cells. This systematic approach unveils a cohort of verifiable genes, among which BUB1, a mitotic checkpoint serine/threonine kinase, emerges as a high-confidence hit in cancer cells. Cellular and molecular studies demonstrate that genetic depletion or pharmacological inhibition of BUB1 profoundly impairs MPM cell survival and growth while inducing G2/M cell cycle arrest, cellular senescence, and apoptosis, and attenuating functional hallmarks of aggressive cancer cells. Transcriptomic profiling of BUB1-depleted cells discloses differential gene expression signatures congruent with cell fate phenotypes, including the reprogramming of mitotic network genes. Mechanistically, BUB1 is indispensable for the proper localization of essential mitotic regulators MAD1, MAD2, and Shugoshin (SGO1), thereby ensuring the functionality of the spindle assembly checkpoint (SAC). Furthermore, BUB1 ablation leads to cytokinesis failure and multinucleation, a phenotype characterized by the downregulation of CDC20, Cyclin A, and Cyclin B, and a reciprocal upregulation of the cyclin-dependent kinase inhibitor p21. Clinically, MPM tumors exhibit elevated levels of BUB1, and high BUB1 expression is associated with shorter patient survival. Our novel findings accentuate comparative CRISPR screens as a powerful platform to explore tumor cell-selective gene essentiality and propose BUB1 kinase as a potential marker and druggable vulnerability with therapeutic implications for MPM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147c280cc9a37e6a39ce87d2c82cecf0b12eda26" target='_blank'>
              Genome-wide CRISPR screen identifies BUB1 kinase as a druggable vulnerability in malignant pleural mesothelioma
              </a>
            </td>
          <td>
            E. Cakiroglu, Sude Eris, Ozden Oz, G. Karakülah, Serif Senturk
          </td>
          <td>2025-04-03</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d379694e8835fc398019d9f1f787e108dd62b53d" target='_blank'>
              Abstract 1896: Spatial proteomics and tumor-immune phenotyping identifies a diverse immune landscape in syngeneic murine tumor models
              </a>
            </td>
          <td>
            Arindam Bose, E. Alonzo, Sophie Struble, Jeremy Fisher, R. Heil-Chapdelaine, Natasha F. Diaz Granados, Vasundhara Agrawal
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683616b1a667ef7717c017121e0ce36b95c50974" target='_blank'>
              Spatial multi-omics profiling of breast cancer oligo-recurrent lung metastasis.
              </a>
            </td>
          <td>
            Yang Gao, Bin Li, Yuzhi Jin, Jinlin Cheng, Weihong Tian, Lixiong Ying, Libing Hong, Shan Xin, Bo Lin, Chuan Liu, Xuqi Sun, Jun Zhang, Haibo Zhang, Jindong Xie, X. Deng, Xiaomeng Dai, Lulu Liu, Yi Zheng, Peng Zhao, Guangchuang Yu, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-04-15</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef350217417b22617b2248d742f15859d0838a2a" target='_blank'>
              Abstract 752: Hypoxia-driven spatial dynamics of epigenetic and transcriptional cell states in glioblastoma
              </a>
            </td>
          <td>
            Phoebe Lombard, M. Zaidi, Ronald Wu, S. Mansouri, Gelareh Zadeh, B. Wouters
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Approximately 15-20% of head and neck cancer squamous cell carcinoma (HNSCC) patients respond favorably to immune checkpoint blockade (ICB). Previous single-cell RNA-Seq (scRNA-Seq) studies identified immune features, including macrophage subset ratios and T-cell subtypes, in HNSCC ICB response. However, the spatial features of HNSCC-infiltrated immune cells in response to ICB treatment need to be better characterized. Methods Here, we perform a systematic evaluation of cell interactions between immune cell types within the tumor microenvironment using spatial omics data using complementary techniques from both 10X Visium spot-based spatial transcriptomics and Nanostring CosMx single-cell spatial omics with RNA gene panel including 435 ligands and receptors. In this study, we used integrated bioinformatics analyses to identify cellular neighborhoods of co-localizing cell types in single-cell spatial transcriptomics and proteomics data. In addition, we used both publicly available scRNA-Seq and in-house spatial RNA-Seq data to identify spatially constrained Ligand-Receptor interactions in Responder patients. Results With 522,399 single cells profiled with both RNA and protein from 26 patients, in addition to spot-resolved spatial RNA-Seq from 8 patients treated with ICB together with bioinformatics analysis of publicly available single-cell and bulk RNA-Seq, we have identified a spatial and cell-type specific context-dependency of myeloid and T cell interaction difference between Responders and Non-Responders. We defined further cellular neighborhood and the sources of chemokine CXCL9/10-CXCR3 interactions in Responders, emerging targets in ICB, as well as CXCL16-CXCR6, CCL4/5-CCR5, and other underappreciated and potential markers and targets for ICB response in HNSCC. In addition, we have contributed a rich data resource of cell-cell Ligand Receptor interactions for the immunotherapy and HNSCC research community. Discussion Our work provides a comprehensive single-cell and spatial atlas of immune cell interactions that correlate with response to ICB in HNSCC. We showcase how integrating multiple technologies and bioinformatics approaches can provide new insights into potential immune-based biomarkers of ICB response. Our results suggested refining future studies using preclinical animal models in a more context-specific manner to elucidate potential underlying mechanisms that lead to improved ICB responses. What is already known on this topic Most cancer patients still do not experience clinical benefits from immune checkpoint blockade (ICB), necessitating the development of response biomarkers and new immunotherapeutic targets. What this study adds Here, we use integrated high-dimensional omics and bioinformatics approaches to identify immune cell-cell interaction markers associated with ICB response in patients with Head and neck squamous cell carcinoma. How this study might affect research, practice or policy We identified spatial and cell-type specificity of Ligand-Receptor interactions between myeloid and T cells in ICB Responder patients that may help inform further mechanistic studies and biomarker development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ece16324064b70206dfb8fd687bfc5db3089e8d7" target='_blank'>
              Integrated single-cell and spatial analysis identifies context-dependent myeloid-T cell interactions in head and neck cancer immune checkpoint blockade response
              </a>
            </td>
          <td>
            Athena E. Golfinos-Owens, Taja Ložar, Parth Khatri, Rong Hu, Paul M. Harari, Paul F. Lambert, Megan B Fitzpatrick, Huy Q. Dinh
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9cb5b1581890bc09659329e338f756aca67f340" target='_blank'>
              Abstract 2090: Integrative spatial and single-cell analysis elucidates tumor microenvironment heterogeneity in NSCLC
              </a>
            </td>
          <td>
            Qiming Zhou, Jiadong Zhou, Beixi Wang, Jingwen Shi, Han Yan, Yun Zhang, Peilin Zhang, Zhirong Shen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dfbdf3e1bc24e3e89e9ac8874b0428cc6cc6092" target='_blank'>
              Targeting CAFs-Mediated Stromal Signaling in a Patient-Derived Organotypic Colorectal Tumor Model.
              </a>
            </td>
          <td>
            Astha Lamichhane, Prasiddha Guragain, Jacob Heiss, Pouria Rafsanjani Nejad, Anju Rana Magar, Nicholas Ciavattone, Seema Agarwal, G. Luker, Hossein Tavana
          </td>
          <td>2025-04-14</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is one of the most lethal cancers for humans. HCC is highly heterogeneous. In this study, we performed ultra-depth (∼1 million reads per spot) spatial sequencing on a case of HCC. Sixteen distinct spatial expression clusters were identified. Each of these clusters was spatially contiguous and had distinct gene expression patterns. In contrast, benign liver tissues showed minimal heterogeneity in terms of gene expression. Numerous immune cell-enriched spots were identified in both HCC and benign liver regions. Cells adjacent to these immune cell-enriched spots showed significant alterations in their gene expression patterns. Interestingly, the responses of HCC cells to the nearby immune cells were significantly more intense and broader, while the responses of benign liver cells to immune cells were somewhat narrow and muted, suggesting an innate difference in immune cell activities towards HCC cells in comparison with benign liver cells. When standard-depth sequencing was performed, significant numbers of genes and pathways that were associated with these changes disappeared. Qualitative differences in some pathways were also found. These results suggest that deep spatial sequencing may help to uncover previously unidentified mechanisms of liver cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae5e28e42c1316f5f0d8b1613ea993c57374ea44" target='_blank'>
              Deep Spatial Sequencing Revealing Differential Immune Responses in Human Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Yan-Ping Yu, Silvia Liu, Caroline Obert, Baoguo Ren, Marielle Krivet, Kyle Metcalfe, Jia-Jun Liu, Tuval Ben-Yehezkel, Jian-Hua Luo
          </td>
          <td>2025-03-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf8353686e8bd9d104e11c8e9f95fa9e551e49d0" target='_blank'>
              Abstract 6265: Cross-tissue atlas of human disease identifies tumor-specific components in tumor microenvironment
              </a>
            </td>
          <td>
            Seong-Kun Kim, Sungmin Cheong, Suho Lee, Yeju Kim, Jong-Eun Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab07960659fe1e798a089508fbf87b3452b9a753" target='_blank'>
              Abstract 3116: High-definition spatial transcriptomics reveals microenvironment features associated with resistance to anti-PD1 therapies in lung cancer
              </a>
            </td>
          <td>
            Corinne Ramos, Sandra Delebecq, Richard Ruez, Victor Senechal
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a511df75119fba0ee72cb0fb6f464cde16c9487e" target='_blank'>
              Abstract 2439: Spatial transcriptomics-informed latent diffusion models generate histopathological images reflective of tumor microenvironment
              </a>
            </td>
          <td>
            Gokul Srinivasan, Minh-Khang Le, Zarif L. Azher, Harsimran Kaur, F. Kolling, Keluo Yao, Ken S. Lau, Xiaoying Liu, Louis J Vaickus, Joshua Levy
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e6b61389a616ff13ee13899ada0ff35cea9d234" target='_blank'>
              Abstract LB161: Bioprinting 3D spatially resolved tumor avatars to mimic the native tumor microenvironment
              </a>
            </td>
          <td>
            Haylie R. Helms, A. Tahayeri, Ellen M. Langer, Alexander Davies, L. Bertassoni
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e5ba5dfba618dfefbeb55cda17b746756b7211" target='_blank'>
              Abstract 2570: Deciphering local immunosuppression in pancreatic cancer: targeting CAF-myeloid cell crosstalk to enhance immune response
              </a>
            </td>
          <td>
            Marie C. Hasselluhn, Lukas J. Vlahos, Yuanqing Xue, Dafydd Thomas, Quin T. Waterbury, Allison C. Hess, Tanner C. Dalton, Amanda R. Decker-Farrell, Á. C. García, Stephen A. Sastra, Carmine F. Palermo, Timothy C Wang, Andrea Califano, K. Olive
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The spatial interactions between malignant and immune cells in the tumor microenvironment (TME) play a crucial role in cancer biology and treatment response. Understanding these interactions is critical for predicting prognosis and assessing immunotherapy effectiveness. Conventional methods, which focus on local spatial features, often struggle to achieve robust analysis due to the complex and heterogeneous cellular distributions. We propose a Topological Data Analysis (TDA)-based framework using both global and local spatial features between malignant and immune cells. For the global aspects, we introduce Topological Malignant Clusters (TopMC), a method utilizing persistence diagrams in TDA to capture the global spatial shapes of malignant cell distributions. It quantifies tumor-immune cell infiltration at a global scale by computing distances from the boundaries of the TopMC. Local interactions are evaluated through the density of malignant cells. Using high-resolution multiplex immunofluorescence (mIF) imaging in Diffuse Large B Cell Lymphoma, we integrate distance and density measures into a distance-density space to analyze patterns of malignant-immune cell interactions in TME. This study shows the robustness of the proposed approach to variations in cell distribution, enabling consistent analysis irrespective of whether images are acquired from malignant-enriched or border regions of the tumors. Furthermore, we elucidate the correlation between spatial patterns of immune phenotypes and patient survival probability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f549327f511c89a0763403d23ea2ac5417a3579" target='_blank'>
              Spatial Analysis of Malignant-Immune Cell Interactions in the Tumor Microenvironment Using Topological Data Analysis
              </a>
            </td>
          <td>
            Seol Ah Park, Yongsoo Kim, Paweł Dłotko, Davide Gurnari, Jooyoung Hahn
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f34cbb1aaff4c24b87ad94662408c71987a782" target='_blank'>
              Abstract 159: Spatial multimodal understanding of disease heterogeneity and aggressiveness in renal cell carcinoma
              </a>
            </td>
          <td>
            Yize Li, Wagma Caravan, Yuwei Zhang, Andrew Houston, Siqi Chen, Chia-Kuei Mo, Preet Lal, Xiangwei Fang, Xiyi Wei, Ruiyang Liu, Ilya Strunilin, Yizhe Song, Song Cao, Cody Weimholt, Hui Zhang, Feng Chen, Li Ding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Objectives The tumor microenvironment (TME) plays a critical role in the progression of lung adenocarcinoma (LUAD). This study aims to investigate the cellular composition of the TME in LUAD and assess the role of the MDK-NCL signaling pathway. Methods We employed a multi-omics strategy to investigate LUAD, combining single-cell RNA sequencing (scRNA-seq), spatial transcriptomics (ST), and bulk RNA-seq datasets. Publicly available scRNA-seq data and ST data were utilized. scRNA-seq data underwent quality control, dimensionality reduction, and clustering to characterize cell populations and identify malignant epithelial subtypes using the Seurat and inferCNV packages. Spatial transcriptomics data facilitated the identification of distinct tumor niches, while immune infiltration and ligand-receptor interactions were analyzed using MCPcounter and Niches. Experimental validation was performed via real-time PCR and western blotting on paired LUAD and adjacent normal tissue samples. Results scRNA-seq revealed the presence of multiple immune and stromal cell populations, with malignant epithelial cells being subdivided into six clusters. The MDK-NCL axis demonstrated high activity in malignant cells, showing strong interactions with immune and stromal components. Spatial transcriptomics revealed nine distinct tumor niches, with MDK-NCL signaling notably upregulated at the tumor-immune interface, highlighting its role in establishing an immunosuppressive microenvironment. In both the TCGA-LUAD cohort and in-house cohort, MDK and NCL were significantly upregulated at the mRNA and protein levels in tumor samples compared to normal tissues. High MDK-NCL expression in the TCGA-LUAD cohort correlated with increased TMB, MSI, and reduced immune cell infiltration. Elevated levels of immune checkpoint genes, including PD-1 and CTLA-4, in patients with high MDK-NCL expression suggested a potential resistance to immune checkpoint inhibitors. Moreover, patients with high MDK-NCL expression exhibited poorer survival outcomes, underscoring the pathway’s role in tumor progression and immune evasion. Conclusion Our findings reveal that LUAD cells use the MDK-NCL signaling pathway to shape the TME, suppressing immune activity and promoting malignancy in epithelial cells. This study highlights the MDK-NCL axis as a potential therapeutic target for LUAD, particularly for patients with high MDK-NCL expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b567f020873734871af5baa9880f442a2f4775bd" target='_blank'>
              Integrative single-cell and spatial transcriptomics analysis reveals MDK-NCL pathway’s role in shaping the immunosuppressive environment of lung adenocarcinoma
              </a>
            </td>
          <td>
            Yu Fu, Song Li, Yikang Zhao, Xiran Zhang, Xiaolu Mao, Ran Xu
          </td>
          <td>2025-05-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3bcf08a317f0fb50f59a7b481b2ba31a08d66c" target='_blank'>
              Abstract 7493: Spatially resolved insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, Padma Kadiyala, Alexander Bray, Jamie N. Mills, Hannah R Watkoske, Arvind Rao, J. Okoye, Atul Deshpande, Jiaqi Shi, E. Fertig, Filip Bednar, Yaqing Zhang, Eileen S. Carpenter, Timothy L. Frankel, M. Pasca di Magliano
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183f97a84615b6dbc6366f5d2cb1c5539c88c0f6" target='_blank'>
              Abstract 7466: Advancing tumor characterization and immune phenotyping through multiomic in situ profiling of RNA and proteins
              </a>
            </td>
          <td>
            Thanutra Zhang, Smritee Dadhwal, Joshua Thao, Anatalia Robles, Cheyenne Christopherson, Suyog Pathare, Kristen Pham, Minkyung Park, Hsin Wen Chang, Janine Hensel, Cedric Espenel, Hayato Ikoma, Gabriele Girelli, Paul Ryvkin, Veronica Gonzalez Muñoz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a32f7726bad7de5bd6ab6b676111c1867233e6" target='_blank'>
              Precancerous Cells Initiate Glioblastoma Evolution and Contribute to Intratumoral Heterogeneity.
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, Jong Hee Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, J. H. Lee
          </td>
          <td>2025-04-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background The differentiation characteristics of neutrophils within the gastric cancer (GC) tumor microenvironment (TME) and their interactions with malignant gastric epithelial cells require further investigation. Furthermore, the therapeutic potential of tumor-associated neutrophils (TANs) in immunotherapy remains inadequately explored. Methods We integrated two single-cell transcriptome datasets comprising 12 samples, including gastric primary tumors, non-tumor tissues, and metastatic tumors, to profile the epithelial cells and TANs atlas within the TME and examine their interaction modules. In addition, these data were integrated with the bulk transcriptomic including the Cancer Genome Atlas - Stomach Adenocarcinoma (TCGA-STAD) and Asian Cancer Research Group (ACRG) datasets to analyze the expression levels of neutrophil-associated genes across the tumor-associated neutrophil subsets. Results We analyzed 3,118 gastric epithelial cells and 2,365 TANs from all samples. Epithelial cells were classified into ten subclusters, while TANs were grouped into five subclusters. In gastric primary tumors, epithelial cell subtypes included primarily MUC16 + and stem-like populations. In metastatic tumors, the epithelial cell subset with high CXCL5 expression was a characteristic subtype. TANs mainly interacted with epithelial cells via the LGALS9-CD45 and CD46-JAG1 pathways. And RGS2 was highly expressed in N4, a tumor-associated neutrophils subcluster characterized by high MMP9 expression, highlighting its potential as an immunotherapy target. Conclusion TANs exhibit robust interactions with gastric malignant epithelial cell subsets. Furthermore, RGS2, which is highly expressed in N4, could serve as a promising target for immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12876-025-03920-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7e6ebcd454b68a72cccf0bcd29fab2640349a" target='_blank'>
              Unveiling the heterogeneity and immunotherapy potency of tumor-associated neutrophils in the tumor microenvironment of gastric cancer
              </a>
            </td>
          <td>
            Tong-tong Qi, Si-jiang Zhou, Zhu Yu, Yong Li, Jun-qiang Chen
          </td>
          <td>2025-04-28</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cac8b257d411c793fd0016b4f5f926fd3d98de50" target='_blank'>
              Abstract 5253: Immune landscape analysis of solid tumors via spatially guided cell isolation for enhanced immunotherapeutic development
              </a>
            </td>
          <td>
            Elie Abi Khalil, Yichao Wu, Jingtian Zheng, Joshua Plank, Alexander Lippert, Steve Seung‐Young Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Cell migration plays a key role in normal developmental programs and in disease, including immune responses, tissue repair, and metastasis. Unlike other cell functions, such as proliferation which can be studied using high-throughput assays, cell migration requires more sophisticated instruments and analysis, which decreases throughput and has led to more limited mechanistic advances in our understanding of cell migration. Current assays either preclude single-cell level analysis, require tedious manual tracking, or use fluorescently labeled cells, which greatly limit the number of extracellular conditions and molecular manipulations that can be studied in a reasonable amount of time. Using the migration of cancer cells as a testbed, we established a workflow that images large numbers of cells in real time, using a 96-well plate format. We developed and validated a machine-vision and deep-learning analysis method, DeepBIT, to automatically detect and track the migration of individual cells from time-lapsed videos without cell labeling and user bias. We demonstrate that our assay can examine cancer cell motility behavior in many conditions, using different small-molecule inhibitors of known and potential regulators of migration, different extracellular conditions such as different contents in extracellular matrix and growth factors, and different CRISPR-mediated knockouts. About 1500 cells per well were tracked in 840 different conditions, for a total of ~1.3M tracked cells, in 70h (5 min per condition). Manual tracking of these cells by a trained user would take ~5.5 years. This demonstration reveals previously unidentified molecular regulators of cancer cell migration and suggests that collagen content can change the sign of how cytoskeletal molecules can regulate cell migration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7619e646f8100faec8365bd603f016e6bb430960" target='_blank'>
              Accelerated Discovery of Cell Migration Regulators Using Label-Free Deep Learning-Based Automated Tracking
              </a>
            </td>
          <td>
            Tiffany Chu, Yeongseo Lim, Yufei Sun, Denis Wirtz, Pei-Hsun Wu
          </td>
          <td>2025-04-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed241f7ab2b6bafb6e0e97061f34d2c422a2e850" target='_blank'>
              Abstract 5076: Unravelling melanoma brain metastasis tumor microenvironment characteristics using spatial transcriptomics
              </a>
            </td>
          <td>
            Clemence J. Belle, S. Brosda, V. Bonazzi, Victor Bulteau, Thomas Stuart, Zherui Xiong, Duncan Lambie, Peter A. Johansson, L. Aoude, Kalpana Patel, Samantha J Stehbens, Mitchell S. Stark, N. Haass, Wen Xu, M. Pinkham, Matthew C. Foote, Sarah Olson, Victoria Atkinson, A. Kulasinghe, Quan H. Nguyen, Andrew Barbour
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f30684782f70f49100b5a4c8e6aa8a01fed878d3" target='_blank'>
              Single-cell dissection of the genotype-immunophenotype relationship in glioblastoma.
              </a>
            </td>
          <td>
            N. Soni, Kavita Rawat, Zhihong Chen, Angela DiMauro, Bruno Giotti, Dolores Hambardzumyan, A. Tsankov
          </td>
          <td>2025-04-11</td>
          <td>Brain : a journal of neurology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Schwannomas are nerve sheath tumors arising at cranial and peripheral nerves, either sporadically or in patients with a schwannomatosis-predisposition syndrome. There is limited understanding of the transcriptional heterogeneity of schwannomas across genetic backgrounds and anatomic locations. Methods Here, we prospectively profile by single-cell full-length transcriptomics tumors from 22 patients with NF2-related schwannomatosis, non-NF2-related schwannomatosis, and sporadic schwannomas, resected from cranial and peripheral nerves. We profiled 11,373 cells (after QC), including neoplastic cells, fibroblasts, T cells, endothelial cells, myeloid cells, and pericytes. Results We characterize the intra-tumoral genetic and transcriptional heterogeneity of schwannoma, identifying six distinct transcriptional metaprograms, with gene signatures related to stress, myelin production, antigen presentation, interferon signaling, glycolysis, and extracellular matrix. We demonstrate the robustness of our findings with analysis of an independent cohort. Conclusions Overall, our atlas describes the spectrum of gene expression across schwannoma entities at the single-cell level and will serve as an important resource for the community. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01462-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a8868eb66d6c72488d55b6daa0e1d4183e3adcb" target='_blank'>
              A single-cell atlas of Schwannoma across genetic backgrounds and anatomic locations
              </a>
            </td>
          <td>
            L. G. Gonzalez Castro, Avishai Gavish, L. Bussema, Christopher W Mount, Cyril Neftel, M. Nomura, E. Chiocca, W. Bi, Omar Arnaout, Fred G. Barker, Justin M. Brown, Justin T Jordan, Tracy T. Batchelor, A. Stemmer-Rachamimov, Scott Plotkin, I. Tirosh, M. Suvà
          </td>
          <td>2025-04-11</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Osteosarcoma (OS), a prevalent malignant bone tumor, has seen limited progress in treatment efficacy and patient outcomes over decades. Recent insights into the tumor microenvironment (TME) have revealed its crucial role in tumor progression and therapeutic resistance, particularly in OS. This review offers a comprehensive exploration of the OS microenvironment, meticulously dissecting its crucial components: the mesenchymal stromal TME, the immune microenvironment, hypoxia-induced adaptations, and the impact of the physical microenvironment. By demonstrating how these elements collectively drive tumor proliferation, immune evasion, and invasion, this review explores the intricate molecular and cellular dynamics at play. Furthermore, innovative approaches targeting the OS microenvironment, such as immunotherapies, are presented. This review highlights the importance of the TME in OS progression and its potential as a source of novel therapeutic strategies, offering new hope for improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66a91e822690718ee9aef6926172cee6b0fe48f6" target='_blank'>
              Microenvironment matters: insights from the FOSTER consortium on microenvironment-driven approaches to osteosarcoma therapy
              </a>
            </td>
          <td>
            Aurélie Dutour, M. Pasello, Luke Farrow, Mahetab Amer, N. Entz-Werlé, Michaela Nathrath, Katia Scotlandi, Sibylle Mittnacht, Anne Gomez-Mascard
          </td>
          <td>2025-04-10</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09be66602da590319fde64cb1853d326259197c5" target='_blank'>
              Abstract 3100: Spatially resolved effects of oligonucleotide therapeutics on cellular phenotypes in cancer tissues
              </a>
            </td>
          <td>
            Corinne Ramos, Sandra Boute, Amandine Gerstenberg, Richard Ruez
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e688452c1e795984dc287acda606201c118b68e6" target='_blank'>
              Abstract 5241: Mapping spatial 3D genome landscape of breast cancer cells in different tumor immune microenvironments using onco-immune co-culture as a model
              </a>
            </td>
          <td>
            Jude Dunne, Huy Nguyen, Shyamtanu Chattoraj
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202c6e4f5a1bfae884ce6819555bba6dc3de7227" target='_blank'>
              Amplification of extrachromosomal MYC paralogs shapes immunosuppressive tumor microenvironment in small cell lung cancer.
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Haodong Lin, Jiaming Deng, Yan Ju, Xueyan Hu, Jianqi She, Zhijian Liang, Kongxu Dai, Mantang Qiu, Kunkun Sun, Jun Wang, Fan Yang, Jian Chen, Ence Yang, Xiao Li
          </td>
          <td>2025-04-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c127048e0d0f262eff5e6fdada1e6625367af0dd" target='_blank'>
              Abstract 5227: Extrachromosomal MYC-paralogs amplification shapes suppressive tumor microenvironment in small cell lung cancer
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Ence Yang, Xiao Li
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma is the most aggressive and lethal cancer of the central nervous system, presenting substantial treatment challenges. The current standard treatment, which includes surgical resection followed by temozolomide and radiation, offers limited success. While immunotherapies, such as immune checkpoint inhibitors, have proven effective in other cancers, they have not demonstrated significant efficacy in GBM. Emerging research highlights the pivotal role of tumor-associated macrophages (TAMs) in supporting tumor growth, fostering treatment resistance, and shaping an immunosuppressive microenvironment. Preclinical studies show promising results for therapies targeting TAMs, suggesting potential in overcoming these barriers. TAMs consist of brain-resident microglia and bone marrow-derived macrophages, both exhibiting diverse phenotypes and functions within the tumor microenvironment. This review delves into the origin, heterogeneity, and functional roles of TAMs in GBM, underscoring their dual roles in tumor promotion and suppression. It also summarizes recent progress in TAM-targeted therapies, which may, in combination with other treatments like immunotherapy, pave the way for more effective and personalized strategies against this aggressive malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f77ce2f24a7dfd7badc0d732e7d82ccdd3d1f004" target='_blank'>
              Exploring tumor-associated macrophages in glioblastoma: from diversity to therapy
              </a>
            </td>
          <td>
            Wenwen Zhao, Zhi Zhang, Mingyuan Xie, Feng Ding, Xiangrong Zheng, Shicheng Sun, Jianyang Du
          </td>
          <td>2025-05-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7803e29b219e38d64e2cdad32375f3e68aa64a3" target='_blank'>
              Abstract 5318: Identifying immune cell phenotypes and their function by using high throughput spatial multiomics
              </a>
            </td>
          <td>
            Debia Wakhloo, Julia S. Yu, Sonali A Deshpande, Donald Allen, Anne Hellebust, A. Dikshit
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Microwave thermochemotherapy (MTC) has been applied to treat lip squamous cell carcinoma (LSCC), but a deeper understanding of its therapeutic mechanisms and molecular biology is needed. To address this, we used single-cell transcriptomics (scRNA-seq) and spatial transcriptomics (ST) to highlight the pivotal role of tumor-associated neutrophils (TANs) among tumor-infiltrating immune cells and their therapeutic response to MTC. MNDA+ TANs with anti-tumor activity (N1-phenotype) are found to be abundantly infiltrated by MTC with benefit of increased blood perfusion, and these TANs are characterized by enhanced cytotoxicity, ameliorated hypoxia, and upregulated IL1B, activating T&NK cells and fibroblasts via IL1B-IL1R. In this highly anti-tumor immunogenic and hypoxia-reversed microenvironment under MTC, fibroblasts accumulated in the tumor front (TF) can recruit N1-TANs via CXCL2-CXCR2 and clear N2-TANs (pro-tumor phenotype) via CXCL12-CXCR4, which results in the aggregation of N1-TANs and extracellular matrix (ECM) deposition. In addition, we construct an N1-TANs marker, MX2, which positively correlates with better prognosis in LSCC patients, and employ deep learning techniques to predict expression of MX2 from hematoxylin-eosin (H&E)-stained images so as to conveniently guide decision making in clinical practice. Collectively, our findings demonstrate that the N1-TANs/fibroblasts defense wall formed in response to MTC effectively combat LSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2560362b63b6e3eb203f753fb2eef7cc7ed8926f" target='_blank'>
              Single-cell and spatial transcriptomics reveals an anti-tumor neutrophil subgroup in microwave thermochemotherapy-treated lip cancer
              </a>
            </td>
          <td>
            Bingjun Chen, Huayang Fan, Xin Pang, Zeliang Shen, Rui Gao, Haofan Wang, Zhenwei Yu, Tianjiao Li, Mao Li, Yaling Tang, Xinhua Liang
          </td>
          <td>2025-05-13</td>
          <td>International Journal of Oral Science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaadba98f7ce64b7b0c38d9f04394bc2ded990c7" target='_blank'>
              Spatial Transcriptomics Unveils the Blueprint of Mammalian Lung Development
              </a>
            </td>
          <td>
            Mingyue Chen, Junjie Lv, Qiao Zhang, Qian Gong, Ting Zhao, Zhenping Chen, Haishen Xu, Nan Zhou, Shan Jiang, Jian Du, Xuepeng Chen, Yuwen Ke
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5d1f1c40a5059d6ba0fa61469f0ea3ea998322e" target='_blank'>
              A Morpho-Proteomic Atlas of Mitosis at Sub-Minute Resolution
              </a>
            </td>
          <td>
            Ede Migh, Vivien Miczán, Frederik Post, K. Koos, A. Beleon, Dávid Kókai, Zsanett Zsófia Iván, István Grexa, Nikita Moshkov, Réka Hollandi, David Csikos, Nora Hapek, F. Kaptás, Ferenc Kovács, A. Kriston, Diana Mahdessian, Ulrika Axelsson, Csaba Pál, Emma Lundberg, M. Manczinger, A. Mund, M. Mann, Peter Horvath
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Background The heterogeneity of cancer‐associated fibroblasts (CAFs) has become a crucial focus in understanding cancer biology and treatment response, revealing distinct subpopulations with specific roles in tumor pathobiology. CAFs have also been shown to promote resistance in lung cancer cells to epidermal growth factor receptor‐tyrosine kinase inhibitors (EGFR‐TKIs). However, the specific CAF subsets responsible for driving tumor advancement and resistance to EGFR‐TKIs in lung adenocarcinoma (LUAD) remain poorly understood. Methods We integrate multiple scRNA‐seq datasets to identify cell subclusters most relevant to tumor stage, patient survival, and EGFR–TKIs response. Additionally, in vitro and in vivo experiments, clinical tissue sample immunohistochemistry and patient plasma ELISA experiments are performed to validate key findings in independent LUAD cohorts. Results By analyzing multiple scRNA‐seq and spatial transcriptomic datasets, we identified a unique subset of CXCL14+ myofibroblastic CAFs (myCAFs), emerging during the early differentiation phase of pan‐cancer invasiveness‐associated THBS2⁺ POSTN⁺ COL11A1⁺ myCAFs. Notably, plasma levels of CXCL14 in LUAD patients correlate significantly with tumor stage. Mechanistically, this subset enhances tumor aggressiveness through epithelial‐to‐mesenchymal transition and angiogenesis. Among standard treatment regimens, transitional CXCL14+ myCAFs specifically confer resistance to EGFR‐TKIs, while showing no significant impact on chemotherapy or immunotherapy outcomes. Through a pharmacological screen of FDA‐approved drugs, we identified Filgotinib as an effective agent to counteract the EGFR‐TKIs resistance conferred by this CAF subset. Conclusions In summary, our study highlights the role of the differentiated stage from transitional CXCL14+ myCAFs to invasiveness‐associated myCAFs in driving tumor progression and therapy resistance in LUAD, positioning Filgotinib as a promising targeted therapy for this process. These insights may enhance patient stratification and inform precision treatment strategies in LUAD. Key points Single‐cell analysis identifies transitional CXCL14+ myofibroblastic cancer‐associated fibroblasts (myCAFs) predominantly exist in the advanced‐stage lung adenocarcinoma (LUAD). Transitional CXCL14+ myCAFs fuel metastasis by promoting epithelial–mesenchymal transition (EMT) and angiogenesis on the spatial level. CXCL14 is a potential diagnostic marker for LUAD patients and predict the occurrence of metastasis. Transitional CXCL14+ myCAFs induce the resistance to epidermal growth factor receptor‐tyrosine kinase inhibitors (EGFR‐TKIs) and JAK1 inhibitor, filgotinib could reverse the effect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483d0a285e7fa54ac097ea53f3a31d2cd466049d" target='_blank'>
              Transitional CXCL14+ cancer‐associated fibroblasts enhance tumour metastasis and confer resistance to EGFR‐TKIs, revealing therapeutic vulnerability to filgotinib in lung adenocarcinoma
              </a>
            </td>
          <td>
            Weijiao Xu, Haitang Yang, Ke Xu, Anshun Zhu, S. Hall, Yunxuan Jia, Baicheng Zhao, Enshuo Zhang, Gang Liu, Jianlin Xu, T. M. Marti, R. Peng, Patrick Dorn, Yongliang Niu, Xufeng Pan, Yajuan Zhang, Feng Yao
          </td>
          <td>2025-03-31</td>
          <td>Clinical and Translational Medicine</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/938952a283824cbae8acde9bd23dfb3cd1a44259" target='_blank'>
              Abstract 5309: Single-cell-based analysis of tumor microenvironment heterogeneity in human MASLD/MASH-associated HCC: Classifications by inter-cellular network variations
              </a>
            </td>
          <td>
            Naoko Ohtani, Yoshiki Nonaka, Kanae Echizen, Yi Cheng, Takahiro Kodama
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d748941bf68f55dce06ac739e5c0e7d9fe6de07" target='_blank'>
              A gel-top model for characterizing mesenchymal features of glioblastoma cells
              </a>
            </td>
          <td>
            Yuanzhong Pan, Priscilla Chan, Jeremy N. Rich, Steve A. Kay, Jinseok Park
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b35fc24142fa9e46cb8827397a0b39028b4c9b9" target='_blank'>
              Abstract 6471: Spatial multiomics reveals T-cell activation and exhaustion states in the tumor microenvironment
              </a>
            </td>
          <td>
            Steve Zhou, Yifan Wang, Debia Wakhloo, A. Dikshit, Li-chong Wang, Maithreyan Srinivasan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ed535cc83be9bb714c00d9bb8580fac8efc104e" target='_blank'>
              Abstract 908: Decoding the role of intrinsic pancreatic cellular signaling in shaping the immunosuppressive tumor microenvironment
              </a>
            </td>
          <td>
            Bushangqing Liu, Ethan S Agritelley, Junping Wei, Gangjun Lei, Tao Wang, Jason McBane, Daniel Nussbaum, Z. Hartman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af476913bede9308f97c975da143461f67e3a4d" target='_blank'>
              Abstract LB466: Visium-HD spatial RNAseq analysis of aggressive prostatic adenocarcinoma identifies distinct RNA signatures in metastatic and non-metastatic disease
              </a>
            </td>
          <td>
            Rakesh Shiradkar, Tilak Pathak, Cynthia L. Winham, I. Lawal, O. Abiodun-Ojo, Martin G Sanda, David M. Schuster, A. Osunkoya, Carlos S. Moreno
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258df61f2a4c963aeb2c62a2f4585ccce0f62b1b" target='_blank'>
              Abstract 6464: Spatial mapping of the tumor microenvironment for combination therapy design
              </a>
            </td>
          <td>
            Arindam Bose, Susan Keezer, E. Alonzo, Sophie Struble, Jeremy Fisher, Matthew Norton, R. Heil-Chapdelaine, Natasha F. Diaz Granados, Vasundhara Agrawal
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Single-cell analysis has refined our understanding of cellular heterogeneity in glioma, yet RNA alternative splicing (AS)—a critical layer of transcriptome regulation—remains underexplored at single-cell resolution. Here, we present a pan-glioma single-cell AS analysis in both tumor and immune cells through integrating seven SMART-seq2 datasets of human gliomas. Our analysis reveals lineage-specific AS across glioma cellular states, with the most divergent AS landscapes between mesenchymal- and neuronal-like glioma cells, exemplified by AS in TCF12 and PTBP2. Comparison between core and peripheral glioma cells highlights AS-redox co-regulation of cytoskeleton organization. Further analysis of glioma-infiltrating immune cells reveals potential isoform-level regulation of protein glycosylation in regulatory T cells and a link between MS4A7 AS in macrophages and clinical response to anti-PD-1 therapy. This study emphasizes the role of AS in glioma cellular heterogeneity, highlighting the importance of an isoform-centric approach to better understand the complex biological processes driving tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad0ab2b92026794b33c50be07f43cb4158d402cd" target='_blank'>
              A Single-Cell Atlas of RNA Alternative Splicing in the Glioma-Immune Ecosystem
              </a>
            </td>
          <td>
            Xiao Song, D. Tiek, Minghui Lu, Xiaozhou Yu, Runxin Wu, Maya Walker, Qiu He, Derek Sisbarro, Bo Hu, Shi-Yuan Cheng
          </td>
          <td>2025-03-30</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Glioblastoma (GBM), or grade 4 glioma, is the most common and aggressive primary brain tumor in adults with a median survival of 15 months. Increasing evidence suggests that GBM’s aggressiveness, invasiveness, and therapy resistance are driven by glioma stem cells (GSCs), a subpopulation of tumor cells that share molecular and functional characteristics with neural stem cells (NSCs). GSCs are heterogeneous and highly plastic. They evade conventional treatments by shifting their state and entering in quiescence, where they become metabolically inactive and resistant to radiotherapy and chemotherapy. GSCs can exit quiescence and be reactivated to divide into highly proliferative tumor cells which contributes to recurrence. Understanding the molecular mechanisms regulating the biology of GSCs, their plasticity, and the switch between quiescence and mitotic activity is essential to shape new therapeutic strategies. This review examines the latest evidence on GSC biology, their role in glioblastoma progression and recurrence, emerging therapeutic approaches aimed at disrupting their proliferation and survival, and the mechanisms underlying their resistance to therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1418264d0938e38785a30378512e83f1e451aa8" target='_blank'>
              Targeting Glioma Stem Cells: Therapeutic Opportunities and Challenges
              </a>
            </td>
          <td>
            Asma Mahdi, Mohamed Aittaleb, Fadel Tissir
          </td>
          <td>2025-05-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e4a4169aaf7d2eba5de69f52ef7120fb04fd31" target='_blank'>
              Personalized CRISPR Knock-In Cytokine Gene Therapy to Remodel the Tumor Microenvironment and Enhance CAR T Cell Therapy in Solid Tumors
              </a>
            </td>
          <td>
            Michael Launspach, Julia Macos, Shoaib Afzal, Janik Hohmann, Stefanie Beez, Maximilian Pilgram, Emily Ohlendorf, Andrii Yevchuk, C. F. T. van der Ven, Chahrazad Lachiheb, Lena Andersch, Marvin Jens, Marc Lukas Appis, Felix Zirngibl, Jonas Kath, M. Stecklum, K. Anders, D. Wagner, Ralf Kühn, Angelika Eggert, A. Künkele
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9d10ba962b31f50d67da52456440cdb7f8eb0c" target='_blank'>
              Abstract 3880: Functional dissection of the highly plastic basal cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Kate Ryan, Jung Yun Kim, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G. Christensen, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78e3d7344b2788e380616083356db8cc35b78722" target='_blank'>
              Abstract 5289: Mechanisms of metastasis revealed through single-cell spatial transcriptome analysis in patients with liver metastatic colorectal cancer
              </a>
            </td>
          <td>
            Jongwon Lee, Yeseul Kim, Hyo Seon Ryu, Jongmin Sim, Chungyeul Kim, Hyun Woo Kwon, Hyun Je Kim, Jong Min Park, Ah-reum Lim, Jung Sun Kim, Hwa Jung Sung, Xingyi Guo, Jungmin Choi, Jungyoon Choi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3bfe2b136df9484a23fdf6a59f1b40397fdc9bc" target='_blank'>
              Abstract 87: Targeting POU5F1-expressing circulating tumor cells to prevent metastasis and recurrence in colorectal cancer
              </a>
            </td>
          <td>
            S. Fujino, Aya Ito, Rie Hayashi, M. Yasui, M. Ohue, N. Miyoshi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background The methylation of N6-methyladenosine (m6A) RNA plays a crucial role in the genetic regulation of various cancers. While m6A modifications have been extensively studied in the tumor microenvironment (TME) of several malignancies, their role in cutaneous melanoma (CM) remains unexplored. Methods Using Non-negative matrix factorization (NMF) analysis on single-cell RNA-seq data (GSE215121) from three CM samples obtained from public databases, 26 m6A RNA methylation regulators were utilized to determine TME subclusters, their expression, and function. Results Six distinct TME cell types were identified and NMF clustering further revealed unique m6A-based subpopulations of cancer-associated fibroblasts and T cells. The prognostic model demonstrated strong predictive capabilities, particularly for fibroblast and T cell m6A clusters, and highlighted COL3A1 as a critical regulator of melanoma-fibroblast interactions. Conclusion Highlighting the COL3A1 gene as a critical link and potential therapeutic target in melanoma could offer new avenues for targeted therapies and improve prognostic assessments in cutaneous melanoma. Supplementary Information The online version contains supplementary material available at 10.1007/s00432-025-06176-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e20d76052c3fcce67fbc8337ada951bc0b9da27" target='_blank'>
              Revealing tumor microenvironment communication through m6A single-cell analysis and elucidating immunotherapeutic potentials for cutaneous melanoma (CM)
              </a>
            </td>
          <td>
            Lun Liu, Maxwell A Kishengere, Xueming Xu, Zhanghui Yue
          </td>
          <td>2025-04-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db705f60f289f122950733aa7588b5a6d8c6c985" target='_blank'>
              Abstract 5317: Multimodal spatial analysis of colon cancer tissue reveals emergence of an immunosuppressive tumor maintenance mechanism
              </a>
            </td>
          <td>
            Gaurav N Joshi, Nicholas Sciascia, Michael T Yang, B. Falkenstein, R. Yan, S. Quinn, Brenna Fearey, Junya Yoshioka, Arif Burak Tosun, S. Chennubhotla, F. Pullara, Fumiki Yanagawa
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3e9453e43454e615e5c1280d7017de163bb8905" target='_blank'>
              Abstract 6460: Spatiotemporal multi-omics uncover tumor ecosystem dynamics driving metastatic colonization
              </a>
            </td>
          <td>
            Liang Wu, Yu Zhong, Shang Liu, Chunqing Wang, Zefan Zhang, Yunfan Sun, Jian Zhou, Jia Fan, Xun Xu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cbc83941350f959c8dff2b5403f5812073037ec" target='_blank'>
              Abstract 777: Revealing the transcriptional and proteomic spatial neighborhoods of early prostate cancer and the tumor microenvironment
              </a>
            </td>
          <td>
            A. Lee, M. Hopkins, L. Liao, Vida Talebian, Tamara Jamaspishvili, David M. Berman, Melanie Spears, Jane Bayani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Advances in tissue labeling, imaging, and automated cell identification now enable the visualization of immune cell types in human tumors. However, a framework for analyzing spatial patterns within the tumor microenvironment (TME) is still lacking. To address this, we develop Spatiopath, a null-hypothesis framework that distinguishes statistically significant immune cell associations from random distributions. Using embedding functions to map cell contours and tumor regions, Spatiopath extends Ripley’s K function to analyze both cell-cell and cell-tumor interactions. We validate the method with synthetic simulations and apply it to multi-color images of lung tumor sections, revealing significant spatial patterns such as mast cells accumulating near T cells and the tumor epithelium. These patterns highlight differences in spatial organization, with mast cells clustering near the epithelium and T cells positioned farther away. Spatiopath enables a better understanding of immune responses and may help identify biomarkers for patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80949e1640c0e5682964239d082757c3241b697f" target='_blank'>
              Statistical analysis of spatial patterns in tumor microenvironment images
              </a>
            </td>
          <td>
            Mohamed M Benimam, V. Meas-Yedid, Suvadip Mukherjee, Astri Frafjord, Alexandre Corthay, T. Lagache, Jean-Christophe Olivo-Marin
          </td>
          <td>2025-03-31</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37bbf9a9279b9ea882191a6ab180a2f1195c66ee" target='_blank'>
              Abstract 1246: Driver and signaling cascade in therapy pressure driven cancer cell plasticity
              </a>
            </td>
          <td>
            Poorvi Subramanian, Sreenidhi Mohanvelu, Sivasubramani Narayanan, Afsana Parveen Jahir Hussain, Sabir Salim, Madhi Oli Ramamurthy, Loganayaki Periyasamy, N. Aravindan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/446717528f9ab0573ae7634a1587a6e025056fd4" target='_blank'>
              Abstract 2676: Single cell spatial profiling reveals ERBB2 and EGFR co-expression in primary esophageal adenocarcinoma and brain metastasis tumors
              </a>
            </td>
          <td>
            Nora M Lawson, Lingqun Ye, Chae Yun Cho, Bo Zhao, Thomas Mitchell, Ines Martín, Andy Futreal, Kadir C Akdemir
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A major challenge in spatial transcriptomics (ST) is resolving cellular composition, especially in technologies lacking single-cell resolution. The mixture of transcriptional signals within spatial spots complicates deconvolution and downstream analyses. To uncover the spatial heterogeneity of tissues, we introduce SvdRFCTD, a reference-free spatial transcriptomics deconvolution method, which estimates the cell type proportions at each spot on the tissue. To fully capture the heterogeneity in the ST data, we combine SvdRFCTD with a Bayesian hierarchical negative binomial model with spatial effects incorporated in both the mean and dispersion of the gene expression, which is used to explicitly model the generative mechanism of cell type proportions. By integrating spatial information and leveraging marker gene information, SvdRFCTD accurately estimates cell type proportions and uncovers complex spatial patterns. We demonstrate the ability of SvdRFCTD to identify cell types on simulated datasets. By applying SvdRFCTD to mouse brain and human pancreatic ductal adenocarcinomas datasets, we observe significant cellular heterogeneity within the tissue sections and successfully identify regions with high proportions of aggregated cell types, along with the spatial relationships between different cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cae43b7eec7648572af744389adb1d08ec29dc2d" target='_blank'>
              A Bayesian hierarchical model with spatially varying dispersion for reference-free cell type deconvolution in spatial transcriptomics
              </a>
            </td>
          <td>
            Xuan Li, Yincai Tang, Jingsi Ming, Xingjie Shi
          </td>
          <td>2025-04-03</td>
          <td>Statistical Theory and Related Fields</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Giant cell tumor of bone (GCTB) represents a group of tumors that characterized by their heterogeneity and the presence of multiple cell types. It is essential to understand their molecular mechanisms for better designing therapeutic approaches. Methods We applied single cell RNA-sequencing technology to investigate differentiation process during GCTBs formation. DAVID was used to perform a pathway enrichment and Gene Ontology (GO) analyses on signaling pathways and biological functions. Cell–cell interactions and signaling networks are depicted with network diagrams and usages of heatmaps. Results The analysis identified numerous significant pathways, including Pathways in Parkinson’s disease, Prion diseases and COVID-19. ViBe identified a wide range of biological process, cellular component and molecular function associated with tumorigenesis. Furthermore, we validate that the SPP1 signaling pathway is essential for cell–cell crosstalk, specifically functioning as a positive feedback loop between cancer-associated fibroblasts and macrophages. Conclusion Our study revealed comprehensive molecular and intercellular interaction networks in GCTB, which may advance the understanding of the pathogenic mechanisms in GCTB and contribute essential information for its treatment. Our findings make confution in the complex functional dynamics in the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a986e75a7d696dd9113448b0b025717241d3db24" target='_blank'>
              Single-cell RNA sequencing reveals key signaling pathways and biological functions in giant cell tumors of bone
              </a>
            </td>
          <td>
            Zuozhuan Lin, Zuoxi Chen
          </td>
          <td>2025-04-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d66d16b13f8aa6c453618719a245bd84fb380319" target='_blank'>
              Abstract 759: Spatial transcriptomics identifies unique tumor and microenvironment pathomic programs that are associated with the lung premalignancy and adenocarcinoma continuum
              </a>
            </td>
          <td>
            Yibo Dai, F. Peng, Ansam Sinjab, Sujuan Yang, Minyue Chen, Warapen Treekitkarnmongkol, Lorena I. Gomez Bolanos, Tieling Zhou, Alejandra G Serrano, Jianlong Liao, Guangsheng Pei, Yunhe Liu, Yang Liu, Jiahui Jiang, K. S. Cho, Yanshuo Chu, Kai Yu, Ruiping Wang, Jiping Feng, Zahraa Rahal, Guangchun Han, N. Jimbo, Takuo Hayashi, S. Kishikawa, K. Takamochi, Akshay V. Basi, Avrum E. Spira, Stephen Dubinett, Tomokazu Itoh, Takashi Yao, Kenji Suzuki, Luisa M. Solis, S. Swisher, Mingyao Li, Junya Fujimoto, I. Wistuba, Jared K. Burks, Kadir C Akdemir, Hind Refai, K. Rezvani, Jeffrey Myers, H. Kadara, Linghua Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f55f59e35cc571551e6bd44fc89c978612969a99" target='_blank'>
              Abstract 4735: In silico reconstruction of primary and metastatic estrogen receptor positive breast tumor architecture using geographic information science-augmented spatial transcriptomics
              </a>
            </td>
          <td>
            Jin Young Yoo, Sabrina Akter, Qianying Zuo, Mahima Goel, Audrey Lam, Debapriya Dutta, Betsy Barnick, Maria Grosse-Perdekamp, Aiman Soliman, Zeynep Madak-Erdogan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa2642424dfa7bd96d090083343b61ad5cc5439" target='_blank'>
              Development of a Single-Cell Spatial Metabolomics Method for the Characterization of Cell-Cell Metabolic Interactions.
              </a>
            </td>
          <td>
            Yaqi Zhang, Panpan Chen, Haoyuan Geng, Min Li, Shiping Chen, Bangzhen Ma, Yan Ma, Jianjun Lai, Xiaoqing Cui, Wei Chong, Hao Chen, Xiao Wang, Chenglong Sun
          </td>
          <td>2025-04-04</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c29d3f2a7e2965363c1aee5aae3a7a970978045e" target='_blank'>
              Tumour-intrinsic alterations and stromal matrix remodelling promote Wnt-niche independence during diffuse-type gastric cancer progression
              </a>
            </td>
          <td>
            Lars J S Kemp, Jooske L. Monster, Colin S Wood, Martijn Moers, M. Vliem, Antoine A Khalil, Nigel B Jamieson, L. Brosens, L. L. Kodach, J. V. van Dieren, Tanya M Bisseling, Rachel S van der Post, M. Gloerich
          </td>
          <td>2025-04-01</td>
          <td>Gut</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, J. Cook, Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96a94c903e0937bcd16fe7b5fa29e52d24143783" target='_blank'>
              Abstract 2181: Establishing a multi-modal, cell therapy characterization workflow to identify cell subsets with heightened cytotoxicity
              </a>
            </td>
          <td>
            Richard Yau, Shreya Deshmukh, Jangkeun Kim, Jiwoon Park, Yanping Yang, Makenzie Sacca, Shan Sabri, Teresa Ai, P. F. Gherardini, Gary Schroth, Christopher E. Mason, Moonsoo M. Jin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/339780e1e052f73bd7a155d38cbe11ed7059d53c" target='_blank'>
              Abstract 1253: Single-cell analysis reveals intricate subclonal heterogeneity and evolution in response to standard of care treatments in glioblastoma
              </a>
            </td>
          <td>
            Yueshan Zhao, Xiaoyu Lu, Verah Nyarige, Junfei Zhao, Gonzalo Lopez, Ann Forslund, Maria Ortiz-Estevez, Jorge Benitez-Hernandez, Wei Zhang, C. Fontanillo, Kai Wang, Ha Dang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99aa7af8e7e4a84d84918f1a59bfdf70d970aa43" target='_blank'>
              Abstract 5303: Temporal dynamics of transcriptional and spatial landscape in tumor microenvironment of esophageal squamous cell carcinoma during chemotherapy treatment
              </a>
            </td>
          <td>
            Hiroki Yamashita, Chiaki Mashima, Maiko Narahara, R. Kamata, Kenji Watanabe, R. Nakamura, Sumie Muramatsu, Ayumi Tezuka, Eri Otsuka, Chisa Wada, Yoko Ishimoto, D. Kotani, Takashi Kojima, Naoya Wada, Hiroyuki Sumi, Tomonori Yano, Gensuke Takayama, Akihiro Ohashi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019b74728f4c34b438fa0bc6a482738d055e0044" target='_blank'>
              Abstract 6266: Spatial profiling of prognostic differences in the tumor microenvironment between primary and metastatic ovarian cancer using multiplexed protein imaging
              </a>
            </td>
          <td>
            Raymond M Moore, Brenna Novotny, Chathu L. Atherton, Matthew S. Block, Bryan M Mccauley, Stacey Winham, E. Goode, Ray Guo, Yajue Huang, Svetomir N Markovic, Chen Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0640112dbbdf90099adbfa03bf4452da14a83a22" target='_blank'>
              Abstract 2664: Spatial and single-cell transcriptomics reveal molecular heterogeneity in pulmonary pleomorphic carcinoma
              </a>
            </td>
          <td>
            Atsushi Matsuoka, K. Shien, Shuta Tomida, Hirofumi Inoue, Kazuya Hisamatsu, Ryota Fujiwara, Kousei Ishimura, Ryunosuke Fujii, Ryo Yoshichika, Tomoaki Higashihara, Naohiro Hayashi, Kazuhiro Okada, Fumiaki Mukohara, Mao Yoshikawa, K. Suzawa, Hidetaka Yamamoto, Shinichi Toyooka
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f10bf47822156d7b368733d148999f0f14fb4c81" target='_blank'>
              Spatial Transcriptomic Landscape of Brain Metastases from Triple-Negative Breast Cancer: Comparison of Primary Tumor and Brain Metastases Using Spatial Analysis.
              </a>
            </td>
          <td>
            Jihwan Yoo, Inho Park, Hyun Jung Kim, Hun Ho Park, Sora Lee, Jee Hung Kim, Yoon Jin Cha
          </td>
          <td>2025-04-15</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6d37955c551700a448a3cd5642f0bb308460197" target='_blank'>
              Abstract 7479: Single-cell OncoTreat: Novel therapeutic candidates by high resolution inference and validation of drug sensitivities in heterogeneous cholangiocarcinoma tumors
              </a>
            </td>
          <td>
            Aleksandar Z Obradovic, Lorenzo Tomassoni, Daoqi You, Charles G. Drake, F. D. Dela Cruz, Andrew L Kung, Andrea Califano
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0adc8a52f0d44cf8d5d6b4f5680ab473639625" target='_blank'>
              Model-based tumour subclonal deconvolution accounting for spatio-temporal sampling biases
              </a>
            </td>
          <td>
            Elena Rivaroli, Elena Buscaroli, Salvatore Milite, Alberto Casagrande, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/700ea58a4c1d36566c271fc146a1aaff828a43d8" target='_blank'>
              Abstract 3652: Celleporter: a foundation model of cell and tissue biology with application to patient stratification and target discovery
              </a>
            </td>
          <td>
            Yubin Xie, Eshed Margalit, Tyler Van Hensbergen, Dexter Antonio, Jake Schmidt, Yu (Phoebe) Guo, Jérémie Decalf, Maxime Dhainaut, Lucas Cavalcante, Hargita Kaplan, Rodney Collins, Francis Fernandez, Rob Schiemann, Eric Siefkas, Michela Meister, Joy Tea, Carl Ebeling, Anastasia Mavropoulos, Nicole Snell, Shafique Virani, Ronald Alfa, Lacey J Padrón, Jacob Rinaldi, Daniel Bear
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e55ff70454b1bfef937caf2e04dc247b0607a96" target='_blank'>
              Modeling Tumor Microenvironment Complexity In Vitro: Spheroids as Physiologically Relevant Tumor Models and Strategies for Their Analysis
              </a>
            </td>
          <td>
            Shrey Shah, Gerard G. M. D’Souza
          </td>
          <td>2025-05-17</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb25978847c5553a988089ffa28339e8dff51f1" target='_blank'>
              Single-Cell profiling reveals targetable malignant T-cell subtypes and immune evasion pathways in Sezary Syndrome
              </a>
            </td>
          <td>
            M.S.F Beth A. Childs, Ph.D Eslam A. Elghonaimy, Praveen, Ramakrishnan Geethakumari, Kiran A. Kumar, Joseph F. Merola, M. D. Heather W. Goff, M. Todd A Aguilera, Ph.D Todd
          </td>
          <td>2025-05-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b881f06a481256ab1b5e06c0de554804f026d184" target='_blank'>
              Abstract 162: Mapping the complex interplay between vasculogenic mimicry and immune response in lung cancer using digital spatial profiling
              </a>
            </td>
          <td>
            Chun-Hui Lee, Chun-Hua Hung, Shu-Hsien Wang, Wei-Pang Chung, Yu-Min Yeh, Peng-Chan Lin, Wu-Chou Su
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In the tumor context, platelets play a significant role in primary tumor progression, dissemination and metastasis. Analysis of this interaction in various cancers, such as non-small cell lung cancer (NSCLC), demonstrate that platelets can both transfer and receive biomolecules (e.g. RNA and proteins) to and from the tumor at different stages, becoming tumor-educated platelets. To investigate how platelets are able to transfer oncogenic material, we developed in vitro platelet-like particles (PLPs), from a differentiated MEG-01 cell line, that stably carry RNA and protein of the KRASG12D oncogene in fusion with GFP. We co-cultured these PLPs with NSCLC H1975 tumor cells to assess their ability to transfer this material. We observed that the generated platelets were capable of stably expressing the oncogene and transferring both its RNA and protein forms to tumor cells using qPCR and imaging techniques. Additionally, we found that coculturing PLPs loaded with GFP-KRASG12D with tumor cells increased their proliferative capacity at specific PLP concentrations. In conclusion, our study successfully engineered an MEG-01 cell line to produce PLPs carrying oncogenic GFP-KRASG12D simulating the tumor microenvironment, demonstrating the efficient transfer of this oncogene to tumor cells and its significant impact on enhancing proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91b51d4673aaea8a53878437c3a918ca1f1fbfbf" target='_blank'>
              Oncogenic KRASG12D Transfer from Platelet-like Particles Enhances Proliferation and Survival in Non-Small Cell Lung Cancer Cells
              </a>
            </td>
          <td>
            J. Ceron-Hernandez, Gonzalo Martínez-Navajas, Jose Manuel Sanchez-Manas, María Pilar Molina, Jiajun Xie, Inés Aznar-Peralta, Abel Garcia-Diaz, Sonia Perales, Carolina Torres, M. J. Serrano, Pedro J. Real
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb00b6391ad2db5e3f6a7c46e5b2dee1a20fa631" target='_blank'>
              Abstract 1072: MOSAIC Window: Spatial intra-tumoral heterogeneity characterization through large-scale spatial and single-nuclei multi-omics profiling
              </a>
            </td>
          <td>

          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer is the most prevalent malignancy in women and exhibits significant heterogeneity. The tumor microenvironment (TME) plays a critical role in tumorigenesis, progression, and response to therapy. However, its impact on the prognosis and immunotherapy responses is incompletely understood. Using public databases, we conducted a comprehensive investigation of transcriptome and single-cell sequencing data. After performing immune infiltration analysis, we conducted consensus clustering, weighted gene co-expression network analysis (WGCNA), Cox regression, and least absolute shrinkage and selection operator (Lasso) regression to identify independent prognostic genes in breast cancer. Subsequently, we developed a prognostic model for patients with breast cancer. Tumor Immune Dysfunction and Exclusion (TIDE) values were used to assess patient’s responsiveness to breast cancer. Based on single-cell RNA-sequencing data, we identified various cell types through cluster analysis and investigated the expression of prognostic model genes in each cell type. The drug sensitivity of targeted therapeutic agents for breast cancer treatment was analyzed in different cell types. We identified 12 independent prognostic genes associated with breast cancer and used these genes to construct a prognostic model. The prognostic model accurately discriminated between patients classified as high- and low-risk, providing precise prognostic predictions for individual patients. Additionally, our model exhibited a robust capacity to predict the immunotherapeutic response in breast cancer patients. Our investigation revealed a notable association between the proportion of endothelial cells (ECs) and patient prognosis in breast cancer. A prognostic model for breast cancer was formulated that showed close associations between prognosis and response to immunotherapy. For patients predicted by our model to not respond effectively to immunotherapeutic agents, it may be considered to combine immunotherapeutic agents with targeted therapeutic agents identified through our drug sensitivity analysis, which could potentially enhance treatment efficacy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-97784-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d3bf29828fea11eaaea62401b9c917be2125fbb" target='_blank'>
              Exploring the tumor microenvironment of breast cancer to develop a prognostic model and predict immunotherapy responses
              </a>
            </td>
          <td>
            Ye Tian, Yong Yang, Lei He, Xiaocheng Yu, Hu Zhou, Juan Wang
          </td>
          <td>2025-04-12</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29faac1f752106215ff7dadafa3504675aecf0f8" target='_blank'>
              Abstract LB076: Spatial transcriptomics reveals differences in the tumor and immune microenvironment of high-grade serous ovarian cancers with differing responses to immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Jing Qian, Lynda D. Roman, Rania Bassiouni, Hanim I Ozkizilkaya, Kathy Kwock, Yuxin Jin, Javier Arias-Stella, Joseph W. Carlson, John D. Carpten
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db10e8dfa73f80517f73e1a05e405d99f33973b4" target='_blank'>
              Abstract 5290: Spatial multi-omics defines a shared glioblastoma infiltrative signature at the resection margin
              </a>
            </td>
          <td>
            Balagopal Pai, S. Ramos, W. Cheng, T. Joshi, Gabrielle Price, Jessica Tomé-García, Kristin Beaumont, Robert P Sebra, Raymund Yong, E. Zaslavsky, N. Tsankova
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Biological systems exhibit remarkable heterogeneity, characterized by intricate interplay among diverse cell types. Resolving the regulatory processes of specific cell types is crucial for delineating developmental mechanisms and disease etiologies. While single-cell sequencing methods such as scRNA-seq and scATAC-seq have revolutionized our understanding of individual cellular functions, adapting bulk genome-wide assays to achieve single-cell resolution of other genomic features remains a significant technical challenge. Here, we introduce Deep-learning-based DEconvolution of Tissue profiles with Accurate Interpretation of Locus-specific Signals (DeepDETAILS), a novel quasi-supervised framework to reconstruct cell-type-specific genomic signals with base-pair precision. DeepDETAILS’ core innovation lies in its ability to perform cross-modality deconvolution using scATAC-seq reference libraries for other bulk datasets, benefiting from the affordability and availability of scATAC-seq data. DeepDETAILS enables high-resolution mapping of genomic signals across diverse cell types, with great versatility for various omics datasets, including nascent transcript sequencing (such as PRO-cap and PRO-seq) and ChIP-seq for chromatin modifications. Our results demonstrate that DeepDETAILS significantly outperformed traditional statistical deconvolution methods. Using DeepDETAILS, we developed a comprehensive compendium of high-resolution nascent transcription and histone modification signals across 39 diverse human tissues and 86 distinct cell types. Furthermore, we applied our compendium to fine-map risk variants associated with Primary Sclerosing Cholangitis (PSC), a progressive cholestatic liver disorder, and revealed a potential etiology of the disease. Our tool and compendium provide invaluable insights into cellular complexity, opening new avenues for studying biological processes in various contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/275a4074dd5de5430ad371087a262308cca6fd4d" target='_blank'>
              High-resolution reconstruction of cell-type specific transcriptional regulatory processes from bulk sequencing samples
              </a>
            </td>
          <td>
            Li Yao, Sagar R. Shah, A. Ozer, Junke Zhang, Xiuqi Pan, Tianyu Xia, Vrushali D. Fangal, A. K. Leung, Meihan Wei, John T. Lis, Haiyuan Yu
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417776fcb75ac4a7513d08a7fa65829e73ccec03" target='_blank'>
              Abstract 5205: ECM-free patient-derived organoids preserve diverse prostate cancer lineages and uncover in vitro-enriched cell types
              </a>
            </td>
          <td>
            Robin Dolgos, R. Parmentier, Jing Wang, Raphaëlle Servant, Arnoud J. Templeton, T. Zellweger, Alastair D Lamb, Kirsten Mertz, S. Subotic, T. Vlajnic, Helge Seifert, A. Mortezavi, Cyril Rentsch, Lukas Bubendorf, C. Le Magnen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149ae335dfeb331ab7d81fc0ac4db62e86f66444" target='_blank'>
              Abstract 4536: Dissecting tumor-immune microenvironment in response and resistance to immune checkpoint blockade in metastatic melanoma
              </a>
            </td>
          <td>
            Xinyu Cui, Giuseppe Tarantino, Yiwen He, Priyanka Solanky, K. Pfaff, A. Thorner, Tyler J. Aprati, Boyang Zhang, Timothy R Blosser, Emily J. Robitschek, Jiajia Chen, Junko Tsuji, Elliot Boblitt, Allison Frangieh, H. Faulkner, Marta M. Holovatska, Aleigha Lawless, M. Manos, K. Helvie, T. Sharova, Dennie T. Frederick, J. L. Fahey, Diego Villamarin, Sachi Krishna, Chanell Mangum, A. J. Nirmal, Domenic Abbondanza, Cai McCann, B. Johnson, Alex K. Shalek, E. Van Allen, Xiaowei Zhuang, Ryan J. Sullivan, Barbara Engelhardt, Samouil L. Farhi, Scott J Rodig, F. S. Hodi, Genevieve M. Boland, David Liu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c4c0df9de0d71dd67f18dd27bf1bf7b9c3857af" target='_blank'>
              Abstract 3759: Single cell transcriptomics on a bioprinted 3D neurovascular unit glioblastoma model identifies pharmacological interventions that selectively target tumor cells
              </a>
            </td>
          <td>
            Yen-Ting Tung, M. J. Song, Marc Ferrer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d9f71c97b02d4a1d072a400c199093fb87572c0" target='_blank'>
              Abstract 3458: Antagonism between tumors and lymphocytes drives coevolutionary processes throughout metastasis
              </a>
            </td>
          <td>
            Cort B Breuer, Marcos Labrado, Nathan E. Reticker-Flynn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Accumulating research suggests that Epstein-Barr Virus-positive Diffuse Large B-cell Lymphoma (EBV+DLBCL) is associated with immune dysfunction and tumor microenvironment (TME) heterogeneity. While the prognostic role of the TME in EBV-DLBCL is established, its impact on EBV+DLBCL survival remains unclear. Here, we integrated 10X Visium spatial transcriptomics (ST) with single-cell RNA sequencing (scRNA-seq) to map TME heterogeneity in EBV+DLBCL. Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses identified PD-1/PD-L1 signaling as a hallmark of EBV+DLBCL’s immunosuppressive TME. Functional validation using the PD-1/PD-L1 inhibitor BMS202 revealed dose-dependent suppression of proliferation and enhanced apoptosis in EBV+Farage cells, with TLR4 emerging as a downstream effector showing EBV-status-dependent regulation. These findings not only link TME-driven PD-1/PD-L1 activation to EBV+DLBCL’s poor prognosis but also provide preclinical evidence for PD-1/PD-L1 blockade as a therapeutic strategy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00410-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1796a1ec9b1ce6b656d31aa53096b6765637e098" target='_blank'>
              Spatial transcriptomics reveals tumor microenvironment heterogeneity in EBV positive diffuse large B cell lymphoma
              </a>
            </td>
          <td>
            Mei-Yao He, Meng Liu, Jiayin Yuan, Jin Lv, Wei Li, Qianwen Yan, Yujiao Tang, Luyi Wang, Li Guo, Fang Liu
          </td>
          <td>2025-05-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose Bladder cancer (BC) is characterized by high heterogeneity, with non-muscle-invasive (NMIBC) and muscle-invasive (MIBC) stages differing significantly in clinical behavior and outcomes. The transition from NMIBC to MIBC involves extensive tumor microenvironment (TME) remodeling, particularly in endothelial cells (ECs), which drive angiogenesis and modulate immune and extracellular matrix (ECM) interactions. However, the precise roles of ECs in this progression remain poorly defined. Methods Public single-cell RNA sequencing (scRNA-seq) datasets from 47 BC patients were analyzed to characterize endothelial cell heterogeneity and functional states across NMIBC and MIBC. Computational tools such as CellChat were applied to reconstruct cell–cell communication networks, focusing on pathways related to angiogenesis, immune crosstalk, and ECM remodeling. Results Twelve major cell types were identified, with endothelial cells exhibiting distinct transcriptional profiles between NMIBC and MIBC. NMIBC-associated ECs promoted adhesion and migration through HMGB1 and CXCL12 signaling. In contrast, MIBC was enriched in an ADAM10+ endothelial subset associated with vascular remodeling and activation of Wnt signaling via CTNNB1. Key ligand-receptor interactions highlighted the dynamic roles of ECs in TME modulation during BC progression. Conclusions This study reveals stage-specific endothelial cell phenotypes and signaling networks in BC. The identification of an MIBC-specific ADAM10+ endothelial subset underscores its potential role in driving tumor progression and highlights opportunities for stage-adapted vascular-targeted therapies. These findings advance our understanding of BC pathogenesis and provide the foundation for novel therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02297-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7505578630274eaf54930e9821b19f4ea81e3ca3" target='_blank'>
              A single-cell atlas of bladder cancer unveils dynamic cellular composition and endothelial functional shifts during progression
              </a>
            </td>
          <td>
            Hongjian Song, Guixiang Xie, Yaowei Li, Xiaowei Hu, Zongzheng Yang, Yubo Zhao, Qing Shi, Haonan Li, Ziyi Liu, Zhihao Yin, Ziqi Wang, Zhichao Tong, Wanhai Xu
          </td>
          <td>2025-04-09</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827f74e9a451d846709a409162aebd5ef66250d5" target='_blank'>
              Abstract LB468: Histological subtypes of metastatic bladder cancer have distinct tumor microenvironments and high intratumoral heterogeneity
              </a>
            </td>
          <td>
            Samantha L. Schuster, P. Itagi, Sonali Arora, Thomas W Persse, Patricia Galipeau, John K. Lee, R. Montgomery, Jonathan L. Wright, P. Grivas, F. Vakar‐López, Andrew C. Hsieh, Hung-Ming Lam, Gavin Ha
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8ac6134f8fc2dc9bc5d1f05d773b9d8559310fb" target='_blank'>
              Abstract 7506: A gene program association study (GPAS) in prostate cancer reveals novel gene modules associated with plasticity and metastasis
              </a>
            </td>
          <td>
            Hanbing Song, Junxiang Xu, Paul Allegakoen, Yih-An Chen, H. N. Weinstein, Arda Demirci, Isaac Salzman, Keila Velazquez-Arcelay, Margaret Tsui, Jon Akutagawa, Julia H. Pham, Peter R. Carroll, M. Cooperberg, Ezequiel Goldschmidt, Nancy Y. Greenland, Joshua D. Campbell, Franklin W. Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29fa9279efe44a23f2837544c024f634f88198ac" target='_blank'>
              Abstract 5666: Akos: A novel CAF-targeted therapy to reprogramming the tumor microenvironment to reverse metastatic traits in all solid tumors
              </a>
            </td>
          <td>
            J. Cerda-Infante, Benjamín Prieto, Valentina Cerda, M. Nuñez
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86c2e0fe302cf21b65e16778aa8ab9c33ba4b41f" target='_blank'>
              Abstract 3971: Single-cell transcriptomic insights into tumor and immune dynamics driving resistance to sitravatinib, nivolumab, and ipilimumab in advanced clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Kai Yu, Ying Yuan, Jianfeng Chen, Xinmiao Yan, Menuka Karki, Fei Duan, Rahul A Sheth, Priya Rao, K. Sircar, A. Shah, A. Zurita-Saavedra, Giannincola Genovesa, Min Li, Chih-Chen Yeh, M. Dang, Guangchun Han, Yanshuo Chu, Max Hallin, Peter Olson, Rui Yang, Daniela Slavin, H. Der-Torossian, Curtis Chin, N. Tannir, Linghua Wang, Jianjun Gao, P. Msaouel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/531426f3c35aa5ebc6ad2f9c61722dfa98d7556d" target='_blank'>
              Abstract 5128: Role of adherent-to-suspension transition in melanoma metastatic dissemination and circulating tumor cell dynamics
              </a>
            </td>
          <td>
            Dong Ki Lee, Jongwook Oh, Soyeon Lee, Hyun Woo Park, H. Gee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d4da2d95c6bade92c76a9deae19dad9fbd73c0" target='_blank'>
              Abstract 6362: Characterization of therapeutic effects in glioblastoma through integrated spatial single-cell multiome profiling of longitudinal samples: Unveiling clinically relevant subtypes of pathological changes
              </a>
            </td>
          <td>
            M. Vanmechelen, Pouya Nazari, J. Beckervordersandforth, D. Leunissen, B. Cole, B. Decraene, Y. De Visser, Gautam Shankar, M. Verduin, Daniele Pantano, C. B. González-Blas, Chiara Caprioli, Sien Bevers, Tom Moors, Jayesh Telang, Ivey Sebasstian, J. Messiaen, Y. Van Herck, Emma Geens, Danielle Eekers, A. Claeys, M. Derweduwe, A. Z. Hausen, Francesca M Bosisio, Frank Weyns, T. Daenekindt, Peter Van Eyken, Mieke Govers, Ferdinand Mennens, Koos E. Hovinga, S. De Vleeschouwer, Paul Clement, M. Broen, Marc Vooijs, R. Sciot, A. Antoranz, Jon Pey, Ernst-Jan Speel, A. Hoeben, F. De Smet
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfe55329ddb31e27a6a2e90cca8f16e695b75bc6" target='_blank'>
              Abstract 5216: RPS19-C5AR1 axis in tumor-associated macrophages drives glioblastoma progression: Insights from single-cell transcriptomics
              </a>
            </td>
          <td>
            Chan-Chuan Liu, G. Anuraga, Cheng-Lin Wu, J. Chuang, Chih-Yang Wang, Pin-Yuan Chen, K. Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Pancreatic cancer, a highly malignant tumor of the digestive system, is characterized by a tumor microenvironment with a high degree of immunosuppression. This immunosuppressive property poses significant challenges, as it hampers the effective infiltration of immune cells and impairs their ability to exert cytotoxic effects. The metabolic process of taurine has emerged as a crucial factor in modulating the functions and activities of immune cells. Intervening in taurine metabolism holds the potential to reshape the tumor immune microenvironment, thereby enhancing the ability of immune cells to recognize and eliminate tumor cells. To explore the potential therapeutic relationship between taurine metabolism disorders and pancreatic cancer immunotherapy, we employed multiple software packages, including “Seurat”, “DoubletFinder”, “Harmony”, “GSVA”, and “CellChat” to analyze single-cell data and spatial transcriptomic data of pancreatic cancer. In the present study, four distinct tumor cell subsets, namely RPS4Y1+ tumor cells, LYZ+ tumor cells, CPE+ tumor cells, and MKI67+ tumor cells, were identified for the first time. The CNV score and taurine metabolism score highlighted the significant role of RPS4Y1+ tumor cells within the immunosuppressive microenvironment of pancreatic cancer. Through cell-communication analysis, the crosstalk among fibroblasts, CD8+ T cells, and RPS4Y1+ tumor cells was identified, offering novel insights into immunotherapy strategies, which was strengthened by the co-localization analysis of spatial transcriptomics. Furthermore, by conducting a combined analysis of survival data, we identified LY6D as a potential therapeutic target. Through co-culture experiments with fibroblasts, we uncovered the underlying mechanism of LY6D in regulating taurine metabolism imbalance within the immunosuppressive microenvironment of pancreatic cancer. The establishment of the “taurine-immune crosstalk” criteria in this study effectively paves the way for pancreatic cancer immunotherapy. In conclusion, the current research underscores the significance of taurine metabolism in the immunosuppressive microenvironment of pancreatic cancer. Targeting taurine metabolism may represent a crucial approach for reversing the “stiff-cancer” characteristics of pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a13ae09eea863d7c5e219d0cb61e3e79fc49de40" target='_blank'>
              Multidimensional transcriptomics based to illuminate the mechanisms of taurine metabolism in immune resistance of pancreatic cancer
              </a>
            </td>
          <td>
            Zongshuai Qin, Guixiang Huang, Jian Xu, Lujuan Pan, Chaojun Lan, Yuhuan Yang, Yixia Yin, Yueqiu Qin
          </td>
          <td>2025-03-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3403a48f5780117716d0b711f94672490029280d" target='_blank'>
              Universal Dual-Target CAR-γδT Cells Targeting B7-H3 and IL13Rα2 for Glioblastoma Therapy
              </a>
            </td>
          <td>
            Yang Xu, Yunhao Ye, Yi Wang, Guangna Liu
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2fdb51de20c5dbd0fcd0301755454ca3bb28a10" target='_blank'>
              Abstract 5078: Single-cell characterization of a carcinosarcoma patient-derived xenograft model
              </a>
            </td>
          <td>
            Alida Palmisano, Yuri Kotliarov, Parimal Kumar, Ting-Chia Chang, Li Chen, B. Fullmer, Alyssa Chapman, A. Peach, Biswajit Das, T. Vilimas, Thomas D. Forbes, S. Jiwani, M. Hollingshead, Yvonne A. Evrard, Sarah J Shin, Hari Sankaran, Julia Krushkal, Alice P. Chen, Yingdong Zhao, C. Karlovich, Mickey Williams, Lisa M. McShane, James H. Doroshow
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are the most abundant non-cancerous cell type in glioblastoma (GBM) and heavily influence GBM biology, contributing to tumor progression, therapeutic resistance, immune evasion, and neovascularization. Current in vitro models that utilize IL-4/IL-13 stimulation fail to capture the transcriptional and functional heterogeneity of TAMs observed in vivo. In this study, we utilize a serum-free indirect co-culture model with patient-derived xenolines to polarize primary human macrophages and characterize their molecular and functional phenotypes. We demonstrate that xenoline-polarized macrophages diverge from classical M1/M2 states and instead adopt transcriptional signatures reflective of TAM subsets identified from patients. Notably, macrophages polarized with the radiation-therapy selected xenoline, JX14P-RT, exhibited gene expression patterns enriched for interferon response and hypoxia, mirroring recurrent GBM samples. In contrast, JX14P TAMs showed enrichment in phagocytic gene sets. Functional validation of these phenotypes revealed discrepancies between the transcriptionally predicted and observed phenotypes, emphasizing the importance of integrating phenotypic validation in sequencing studies. Altogether, our findings establish xenoline-polarized macrophages as a more physiologically relevant alternative to traditional models, offering a useful model for studying tumor-immune interaction in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e59e91c3e7e604dd7d80dd0333a4e13f653a5b4e" target='_blank'>
              Xenoline-polarized macrophages as a physiologically relevant in vitro model of tumor-associated macrophages in glioblastoma
              </a>
            </td>
          <td>
            Hasan Alrefai, Benjamin Lin, Amr Elkohly, Manoj Kumar, Taylor L. Schanel, Kevin J. Lee, Patricia H Hicks, Joshua C Anderson, Gao Guo, Eun-Young Erin Ahn, C. R. Miller, Christopher Willey
          </td>
          <td>2025-05-09</td>
          <td>Research Square</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4a49cf0a193cdeef05a09af604560eb6cfc60d0" target='_blank'>
              A novel Tc17 population recruited by tumor cells promotes tumor progression in gastric cancer
              </a>
            </td>
          <td>
            Yichao Wang, Hao Yuan, Yayun Tan, Li Wu, Xinchen Zhao, Zeyuan Yang, Shaoxing Dai, Naixue Yang
          </td>
          <td>2025-05-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are key players in cancer development and therapy, and they exhibit multifaceted roles in the tumor microenvironment (TME). From their diverse cellular origins, CAFs undergo phenotypic and functional transformation upon interacting with tumor cells and their presence can adversely influence treatment outcomes and the severity of the cancer. Emerging evidence from single-cell RNA sequencing (scRNA-seq) studies have highlighted the heterogeneity and plasticity of CAFs, with subtypes identifiable through distinct gene expression profiles and functional properties. CAFs influence cancer development through multiple mechanisms, including regulation of extracellular matrix (ECM) remodeling, direct promotion of tumor growth through provision of metabolic support, promoting epithelial-mesenchymal transition (EMT) to enhance cancer invasiveness and growth, as well as stimulating cancer stem cell properties within the tumor. Moreover, CAFs can induce an immunosuppressive TME and contribute to therapeutic resistance. In this review, we summarize the fundamental knowledge and recent advances regarding CAFs, focusing on their sophisticated roles in cancer development and potential as therapeutic targets. We discuss various strategies to target CAFs, including ECM modulation, direct elimination, interruption of CAF-TME crosstalk, and CAF normalization, as approaches to developing more effective treatments. An improved understanding of the complex interplay between CAFs and TME is crucial for developing new and effective targeted therapies for cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a333ac0501e7572a5af1f1975c3f74e5d8b822" target='_blank'>
              Cancer associated fibroblasts in cancer development and therapy
              </a>
            </td>
          <td>
            Hongyuan Jia, Xingmin Chen, Linling Zhang, Meihua Chen
          </td>
          <td>2025-03-28</td>
          <td>Journal of Hematology & Oncology</td>
          <td>2</td>
          <td>1</td>
        </tr>

        <tr id="Background Adoptive transfer of tumor-infiltrating lymphocytes (TIL) is now a Food and Drug Administration (FDA)-approved treatment for melanoma. While this is a major milestone, there is room for improvement to increase clinical response rates and to further optimize the manufacturing of TIL products. In this study, we characterized the association of tumor-infiltrating B-cells (TIL-B) and tertiary lymphoid structures (TLSs) with clinical response to TIL therapy and tested whether the presence of B-cells in the tumor can be leveraged to optimize TIL manufacture. Methods Tumor sections from TIL responders (R, n=9) and non-responders (NR, n=11) were analyzed by RNA sequencing, and immune cell content was estimated in silico. To study the association between B-cells and TIL expansion, we quantified B-cell subsets and TIL phenotype by flow cytometry. CD40L-induced effects on melanoma-infiltrating B-cells were analyzed by flow cytometry and scRNA-sequencing. Results Tumors from TIL clinical responders had greater abundance of class-switched B-cells (p=0.007) and a greater TLS score (p=0.03) than those of NRs. In addition, greater abundance of B-cells (p≤0.05) and switched memory B-cells (CD27+ IgD−, p≤0.05) in the tumors were associated with greater TIL expansion. Stimulation of TIL-B through addition of CD40L during TIL ex vivo culture improved their expansion success rate from 33% to 67% (p=0.03). Similarly, the addition of CD40L to non-small cell lung cancer (NSCLC) TIL cultures shortened the manufacturing period by 1 week. Moreover, CD40L-enhanced TIL showed more stem-like T-cells (CD39− CD69−, p≤0.05) and an enrichment of neoantigen-reactive T-cell clones in NSCLC TIL. Gene expression analysis showed that CD40L induced gene expression changes in TIL-B after 48 hours in culture (126 differentially expressed genes (DEGs)), with minimal to no changes observed in other immune cell types (including 12 DEG in macrophages, 10 DEG in dendritic cells, and none in monocytes). B-cell DEGs included upregulated co-stimulatory ligands (CD83, CD58), chemokines (CCL22, CCL17), among others. CD40L-induced upregulation of CD58 by melanoma infiltrating B-cells was associated with successful TIL expansion. Conclusions Our results show that CD40L-stimulated B-cells can be leveraged to enhance the quality and quantity of TIL. Clinical trial NCT05681780 is currently testing this concept applied to NSCLC TIL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c235df4e575b67f2bc02fb98db70223032fa501" target='_blank'>
              CD40L stimulates tumor-infiltrating B-cells and improves ex vivo TIL expansion
              </a>
            </td>
          <td>
            R. Rossetti, L. Tordesillas, Matthew S Beatty, Junior Cianne, Elena Martinez Planes, Dongliang Du, Sebastian Snedal, Chao Wang, Bradford A Perez, Anders E Berglund, Yian Chen, A. Sarnaik, James J Mulé, Ben C Creelan, S. Pilon-Thomas, Daniel Abate-Daga
          </td>
          <td>2025-04-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8492f82ea4a9bedb9da34190b6e586980dd59c" target='_blank'>
              Abstract 5313: Spatial transcriptomics reveals clone-specific features of primary and metastatic tumors in triple-negative breast cancer
              </a>
            </td>
          <td>
            Yu-Ting Kang, Tzu-Hung Hsiao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/433969bcfd881b817be3dbffcc239a3ea8a4b05e" target='_blank'>
              Abstract 179: Spatial and multiomics analysis reveals immune interactions as key drivers of immunotherapy outcomes in melanoma patients with in-transit metastases
              </a>
            </td>
          <td>
            Xinyu Bai, C. Quek, G. Al-Eryani, Sonny Ramkomuth, Louise Baldwin, Jiabao Tian, Nurudeen A. Adegoke, Ismael A Vergara, F. Newell, K. Harvey, John Reeves, A. Spillane, Nicola Waddell, Serigne N. Lo, Alexander M. Menzies, I. D. da Silva, U. Palendira, Georgina V Long, R. Scolyer, Alex Swarbrick, J. Wilmott
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Despite significant advances in cancer biology research and treatment, clinical outcomes for patients with liver cancer remain unsatisfactory. The biological and molecular mechanisms underlying the bidirectional signaling between tumor cells and the tumor microenvironment (TME), which promotes tumor progression in the liver, remain to be elucidated. Fibroblasts are crucial regulators of tumor progression and response to therapy; however, our understanding of their roles remains limited. Here, we integrated single-cell RNA sequencing and spatial transcriptomic data of pan-liver cancers to characterize the different subtypes of cancer-associated fibroblasts (CAFs). siRNA transfection was used for knockdown the expression of LAMA4. Western blot assay was used for gene expression analysis. Flow cytometry was used to detect proliferation, toxicity and cytolytic capacity of CD8+ T cells. To establish a spontaneous murine hepatocellular carcinoma (HCC) model, a combined DEN and CCL4 approach was performed. Notably, we identified CD90+ extracellular matrix CAFs (eCAFs) associated with poor prognosis. These CD90+ eCAFs, located distal to the tumor nest, overlapped with the distribution of CD8+ T cells. Functional experiments demonstrated that CD90+ eCAFs recruited CD8+ T cells and inhibited their function through secretion of LAMA4. Further investigation revealed that LAMA4 induced the CD8+ T cell senescence through a DNA damage signaling pathway mediated by the receptor ITGA6. In a mouse model of spontaneous HCC, targeting LAMA4 can inhibit the progression of malignant transformation and synergize with anti-PD-1 therapy. Our study reveals the function of specific CAFs subtypes and highlights the importance of interactions with the immune system. Graphical Abstract CD90+ eCAFs recruited and impaired the anti-tumor activity of CD8+ T cells through LAMA4. LAMA4 induced DNA damage signaling and senescence in CD8+ T cells via binding ITGA6. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02162-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4995a6400cdf1b6395f0b8478b1c2b4dc332edd" target='_blank'>
              LAMA4+ CD90+ eCAFs provide immunosuppressive microenvironment for liver cancer through induction of CD8+ T cell senescence
              </a>
            </td>
          <td>
            Jianlei Zhang, Zhihui Li, Qiong Zhang, Wen Ma, Weina Fan, Jing Dong, Jingjie Tian, Hongfan Liao, Junzhe Guo, Yabing Cao, Jiang Yin, G. Zheng, Nan Li
          </td>
          <td>2025-04-28</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/485b5a5466734bd061c3fdf3a59fcfb2350632cb" target='_blank'>
              Differentiation Responsive Architected Macrophage (DREAM) Selectively Modulate Tumor Microenvironment for Cancer Immunotherapy
              </a>
            </td>
          <td>
            Haoting Chen, Ding Yang, Dongyue He, Zechuan Li, Gengyou Li, Jing Ma, Jiaqing Wan, Meifen Luo, Qi Chen, Yansong Liu, Qihan Zhang, Yanwen Liang, Ying‐Shi Sun, Weisheng Guo, Lu Liu
          </td>
          <td>2025-05-20</td>
          <td>Advanced Functional Materials</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fd0e67d45bf4c5f69a873c6cb3c79e9be60dadd" target='_blank'>
              Abstract 7480: Spatial Groups provide a pan-tumor spatial transcriptomic biomarker of immunotherapy response
              </a>
            </td>
          <td>
            Vivek Behera, Hannah Giba, Ue-yu Pen, A. Di Lello, Benjamin A. Doran, Alessandra Esposito, A. Pezeshk, Christine Bestvina, Justin Kline, M. Garassino, Arjun S. Raman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fa2a70e190a923893596d261326d79e9e7ada3b" target='_blank'>
              Abstract LB082: Disrupting lipid metabolism in GBM: A systems biology approach to targeting treatment-resistant persister cells and tumor associated myeloid cell-mediated support
              </a>
            </td>
          <td>
            Avirup Chakraborty, Changlin Yang, Diana Feier, Aryeh Silver, Nyla T. Searl, Miruna Anica, Avinash Pittu, Illena West, Olusegun O. Sobanjo, Ethan D. Hodge, Mia K. Engelbart, Srabasti Sengupta, Nagheme Thomas, C. V. von Roemeling, Maryam Rahman, Matthew R. Sarkisian, Jianping Huang, Jeffrey Harrison, Duane A Mitchell, L. Deleyrolle
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e0df805bc31950b58610c00bf890d8057b9c31" target='_blank'>
              Abstract 165: Spatially resolved mechanostress effects on tumor microenvironments: Tumor cells and fibroblasts in focus
              </a>
            </td>
          <td>
            Wooseok Kim, HyunSeok Kim, Minki Kim, Zhangxian Lan, Hyung-Soon Park, Mi Ryung Roh, Joon Kim, Jong-Eun Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human complex in vitro models (CIVMs) have demonstrated remarkable potential to study tissue development, physiology and disease at high-throughput. To effectively employ these miniaturized systems in translational preclinical research, their in-depth benchmarking is pivotal. Histology has been the core of tissue characterization for centuries and the foundation of spatial phenotyping. However, standard histology workflows are inherently low-throughput and centered on large tissue pieces. This does not match the high sample volumes and small sample sizes in CIVM research. Here, we introduce a holistic ‘histo-workflow’, utilizing 3D-printed histomolds that facilitate co-planar embedding of CIVMs at high-throughput, resulting in up to 48 samples in one section. We developed a variety of model-specific histomold designs that enable spatially controlled histological sectioning and downstream analyses. We describe these workflows, including mold generation, highplex staining and image analysis, and exemplify their application to histological analyses of various CIVMs. Altogether, the histomolds introduced here afford opportunities for CIVM processing and analysis, while significantly reducing labor and reagent resources, thereby democratizing high-throughput CIVM in histopathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1aa7001ef0a4f67578ea7c5375be8b50760ab64" target='_blank'>
              High-throughput histopathology for complex in vitro models
              </a>
            </td>
          <td>
            Marius F. Harter, Elisa D’Arcangelo, Julien Aubert, Barbora Lavickova, C. Havnar, Bilgenaz Stoll, Irineja Cubela, Adrian M. Filip, Laura Gaspa-Toneu, Julian Scherer, Evdoxia Karagkiozi, Jean-Samuel Dupré, Rupén López-Sandoval, Rebecca Hsia, Leah M Norona, Giovanna Brancati, Ryo Okuda, J. G. Camp, Eliah R. Shamir, J. Garcia-Cordero, Iago Pereiro, N. Gjorevski, N. Gjorevski
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937e7d64004dc2ecd2a28936483a775c209f1f11" target='_blank'>
              Proteomic Remodeling During Tumor Cell-Induced Platelet Aggregation Unveils Metastatic Drivers in Colorectal Cancer
              </a>
            </td>
          <td>
            Thorben Sauer, Caroline Gruner, Katharina Kern, Antje Rackisch, Lea Tischner, Katharina Schulz, Jasmin Ostermann, Lena Cohrs, Michael Kohl, A. Verschoor, Timo Gemoll
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The spatial organization of cells in tissues underlies biological function, and recent advances in spatial profiling technologies have enhanced our ability to analyze such arrangements to study biological processes and disease progression. We propose MESA (multiomics and ecological spatial analysis), a framework drawing inspiration from ecological concepts to delineate functional and spatial shifts across tissue states. MESA introduces metrics to systematically quantify spatial diversity and identify hot spots, linking spatial patterns to phenotypic outcomes, including disease progression. Furthermore, MESA integrates spatial and single-cell multiomics data to facilitate an in-depth, molecular understanding of cellular neighborhoods and their spatial interactions within tissue microenvironments. Applying MESA to diverse datasets demonstrates additional insights it brings over prior methods, including newly identified spatial structures and key cell populations linked to disease states. Available as a Python package, MESA offers a versatile framework for quantitative decoding of tissue architectures in spatial omics across health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162d6ff6e3cedba6aedd63bc146790643b33531d" target='_blank'>
              Quantitative characterization of tissue states using multiomics and ecological spatial analysis
              </a>
            </td>
          <td>
            Daisy Yi Ding, Zeyu Tang, Bokai Zhu, Hongyu Ren, Alex K. Shalek, Robert Tibshirani, GP Nolan
          </td>
          <td>2025-04-01</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8ac2ba6fe27ed44bff66e7ee62b13701829278a" target='_blank'>
              Abstract 6437: Mutational patterns in normal tissue hold the key to understanding the cellular origins of cancer
              </a>
            </td>
          <td>
            Kirsten Kübler, R. Karlić, Mendy Miller, C. Stewart, William D. Foulkes, P. Polak, Peter F Arndt, G. Getz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/653b80a32b949cb3e328b878333ade7e654abf52" target='_blank'>
              Abstract 6283: Performance evaluation of tumor microenvironment deconvolution and gene prediction methods in lung cancer
              </a>
            </td>
          <td>
            Fanying Tang, Dean Lee, Lexiang Ji, Jincheng Wu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c1afd31db17576044e08fba4a3cda5db3c4dd10" target='_blank'>
              Abstract 2417: N-Orbit: An interpretable, distance-based framework for comparison of tumor microenvironments and beyond
              </a>
            </td>
          <td>
            Barbara Xiong, Yuxuan Hu, Kai Tan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The tumor microenvironment (TME) encompasses various cell types, blood and lymphatic vessels, and noncellular constituents like extracellular matrix (ECM) and cytokines. These intricate interactions between cellular and noncellular components contribute to the development of a malignant TME, such as immunosuppressive, desmoplastic, angiogenic conditions, and the formation of a niche for cancer stem cells, but there is limited understanding of the specific subtypes of stromal cells involved in this process. Here, we utilized p16-CreERT2-tdTomato mouse models to investigate the signaling networks established by senescent cancer stromal cells, contributing to the development of a malignant TME. In pancreatic ductal adenocarcinoma (PDAC) allograft models, these senescent cells were found to promote cancer fibrosis, enhance angiogenesis, and suppress cancer immune surveillance. Notably, the selective elimination of senescent cancer stromal cells improves the malignant TME, subsequently reducing tumor progression in PDAC. This highlights the antitumor efficacy of senolytic treatment alone and its synergistic effect when combined with conventional chemotherapy. Taken together, our findings suggest that the signaling crosstalk among senescent cancer stromal cells plays a key role in the progression of PDAC and may be a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06c45a25e4aa78209216dcd2827d3d116f51fb9d" target='_blank'>
              Signaling networks in cancer stromal senescent cells establish malignant microenvironment.
              </a>
            </td>
          <td>
            Yue Zhang, Teh-Wei Wang, Maho Tamatani, Xinyi Zeng, Lindo Nakamura, Satotaka Omori, Kiyoshi Yamaguchi, Seira Hatakeyama, Eigo Shimizu, Satoshi Yamazaki, Yoichi Furukawa, S. Imoto, Yoshikazu Johmura, M. Nakanishi
          </td>
          <td>2025-04-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2aaaf6ecddf1942a365097ba44e7686790d4a91" target='_blank'>
              Abstract 2068: Mapping the spatial whole transcriptome from normal to tumor tissue in renal clear cell carcinoma: tumorigenesis and microenvironmental shifts at single-cell resolution
              </a>
            </td>
          <td>
            Shanshan He, Liang Zhang, Michael Patrick, Sierra McKenzie, Haiyan Zhai, Christine Kang, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb18bd3e4cb1302dcaf3faa0469e23f342caf15" target='_blank'>
              Abstract 5315: Mapping tumor infiltrating lymphocytes in whole prostatectomy specimens to visualize and quantify the immune microenvironment of prostate cancer
              </a>
            </td>
          <td>
            Yuwei Zhang, Rajarsi Gupta, J. Saltz, T. Kurç, Jakub R. Kaczmarzyk, Erich Bremer, Veronica Wallaengen, Amanda Galvez, Abigail Bartley, Leonard Salcedo, Alan Pollack, S. Punnen, Oleksandr N. Kryvenko, R. Stoyanova, Sandra M Gaston
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Tumor heterogeneity is predicted to confer inferior clinical outcomes with precision-based strategies, however, modeling heterogeneity in a manner that still represents the tumor of origin remains a formidable challenge. Sequencing technologies are limited in their ability to identify rare subclonal populations and predict response to treatments for patients. Patient-derived organotypic cultures have significantly improved the modeling of cancer biology by faithfully representing the molecular features of primary malignant tissues. Patient-derived cancer organoid (PCO) cultures contain subclonal populations with the potential to recapitulate heterogeneity, although treatment response assessments commonly ignore diversity in the molecular profile or treatment response. Here, we demonstrate the advantage of evaluating individual PCO heterogeneity to enhance the sensitivity of these assays for predicting clinical response. Additionally, organoid subcultures identify subclonal populations with altered treatment response. Finally, dose escalation studies of PCOs to targeted anti-EGFR therapy are utilized which reveal divergent pathway expression when compared to pretreatment cultures. Overall, these studies demonstrate the importance of population-based organoid response assessments, the use of PCOs to identify molecular heterogeneity not observed with bulk tumor sequencing, and PCO heterogeneity for understanding therapeutic resistance mechanisms. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-96204-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d73b21d8a323278ef745932d923e8c41351e22f0" target='_blank'>
              Subclonal response heterogeneity to define cancer organoid therapeutic sensitivity
              </a>
            </td>
          <td>
            J.D. Kratz, Shujah Rehman, Katherine A. Johnson, A. Gillette, Aishwarya Sunil, Peter F Favreau, Cheri A. Pasch, Devon D Miller, Lucas Zarling, Austin H. Yeung, L. Clipson, Samantha J. Anderson, Alyssa K Steimle, Carley M. Sprackling, K. Lemmon, Daniel E Abbott, M. Burkard, Michael F. Bassetti, J. Eickhoff, Eugene F Foley, Charles P Heise, Randall J Kimple, Elise H Lawson, N. LoConte, S. Lubner, Daniel L Mulkerin, Kristina A Matkowskyj, Cristina B. Sanger, Nataliya V. Uboha, Sean J Mcilwain, Irene M Ong, Evie H Carchman, Melissa C Skala, Dustin A. Deming
          </td>
          <td>2025-04-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Oligodendrogliomas are initially slow-growing brain tumors that are prone to malignant transformation despite surgery and cytotoxic therapy. Understanding of oligodendroglioma evolution and new treatments for patients have been encumbered by a paucity of patient-matched newly diagnosed and recurrent tumor samples for multiplatform analyses, and by a lack of preclinical models for interrogation of therapeutic vulnerabilities that drive oligodendroglioma growth. Here we integrate spatial and functional analyses of tumor samples and patient-derived organoid co-cultures to show that synaptic connectivity is a hallmark of oligodendroglioma evolution and recurrence. We find that patient-matched recurrent oligodendrogliomas are enriched in synaptic gene expression programs irrespective of previous therapy or histologic grade. Analyses of spatial, single-cell, and clinical data reveal epigenetic misactivation of synaptic genes that are concentrated in regions of cortical infiltration and can be used to predict eventual oligodendroglioma recurrence. To translate these findings to patients, we show that local field potentials from tumor-infiltrated cortex at the time of resection and neuronal hyperexcitability and synchrony in patient-derived organoid co-cultures are associated with oligodendroglioma proliferation and recurrence. In preclinical models, we find that neurophysiologic drugs block oligodendroglioma growth and pathologic electrophysiology. These results elucidate mechanisms underlying oligodendroglioma evolution from an indolent tumor to a fatal disease and shed light on new biomarkers and new treatments for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f52e9325187b8f1b82e5b11265e77c3c484b9d39" target='_blank'>
              Spatial synaptic connectivity underlies oligodendroglioma evolution and recurrence
              </a>
            </td>
          <td>
            D. Raleigh, K. Mirchia, S. Oten, T. Picart, Minh Nguyen, Vardhaan S. Ambati, H. Vasudevan, Jacob Young, Jennie W Taylor, S. Krishna, D. Brang, Joanna Phillips, Arie Perry, M. Berger, Susan Chang, John de Groot, Shawn Hervey-Jumper
          </td>
          <td>2025-04-04</td>
          <td>Research Square</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0577e3703eceb9dd16f950f2b2b372372b328f4f" target='_blank'>
              Abstract 5304: Profiling the tumor- immune landscape with a same-section multiomics assay
              </a>
            </td>
          <td>
            A. Dikshit, Cansaran Saygili Demir, Rose Delvillar, Emerald Doolittle, Alice Comberlato, Arec Manoukian, M. Procopio, Pino Bordignon, S. Brajkovic, Maithreyan Srinivasan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b565f8797f4fa651d27cbd6107a1a5335c33485f" target='_blank'>
              Abstract 2852: Mapping single cell spatial 3D genome landscape of localized and metastatic breast cancer sub-types
              </a>
            </td>
          <td>
            Shyamtanu Chattoraj, Huy Nguyen, Jude Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb874fc1b76423af08fc061b2412437123e6829d" target='_blank'>
              Abstract 6258: Inter- and intra-tumoral heterogeneity of TNBC determined by spatial gene transcriptomics
              </a>
            </td>
          <td>
            Fengyuan Huang, Deepa Bedi, Clayton C. Yates
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782f6f0e92847d06cb3d67cdcc10afaa17e2f75c" target='_blank'>
              Abstract 1823: Spatial transcriptome profiling shows a decrease in heterogeneity of GBM xenografts that regrow after irradiation and suggests novel therapeutic targets
              </a>
            </td>
          <td>
            Charlotte M. Degorre
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite remarkable advancements in therapeutic strategies, a considerable proportion of patients with bladder cancer (BC) still experience disease progression and unfavorable prognosis. The heterogeneity and biological functions of tumor endothelial cells (ECs) during BC progression remain poorly understood. We collected scRNA-seq data from BC samples and identified two EC subpopulations through hierarchical clustering analysis. The activity of signaling pathways in distinct EC subpopulations was assessed utilizing AUCell analysis. Gene regulatory networks (GRN) were constructed and analyzed for different EC subpopulations using the pySCENIC algorithm. Additionally, we investigated the association between the abundance of EC subpopulations and both clinical prognosis and immune cell infiltration. The biological effects of ESM1 protein on BC cells were further validated through EdU and Transwell assays. We analyzed 7,519 CD45-negative single cells from BC tissues and discerned two distinct EC subpopulations. The two subpopulations were characterized by high expression of ESM1 (S1 ECs) and CXCL2 (S2 ECs), respectively. In S1 ECs, we observed significant activation of signaling pathways involved in tumor promotion, including angiogenesis and cell proliferation. Additionally, our GRN analysis uncovered notable differences in transcription factor activity between S1 and S2 ECs. Moreover, ESM1 protein promoted proliferation and migration of BC cells. Patients with higher abundance of the S1 EC subpopulation exhibited more unfavorable clinical outcomes and increased infiltration of inhibitory immune cells. Our findings elucidate the transcriptional profiles and biological roles of the high ESM1-expression endothelial cell subpopulation in BC. This subpopulation is associated with poor prognosis and immunosuppressive tumor microenvironment. Accordingly, targeting endothelial cells with high ESM1 expression may offer a novel therapeutic strategy for patients with BC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-95731-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4283029ca72adeb8c04b30a9f72593fd7af08134" target='_blank'>
              Single-cell sequencing uncovers a high ESM1-expression endothelial cell subpopulation associated with bladder cancer progression and the immunosuppressive microenvironment
              </a>
            </td>
          <td>
            Yifeng Qiu, Yuhan Wang, Jiahe Liu, Baohua Liu, Kai Sun, Qi Hou
          </td>
          <td>2025-03-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e947118fcfc484004bcc041ad01d4d6f777a678" target='_blank'>
              Abstract 5307: New developments in spatial transcriptomics unlock insights into the tumor microenvironment
              </a>
            </td>
          <td>
            Manisha Ray, Justin He, Bin Wang, Bing Yang, Sudhir Tattikota, Timothy Wiggin, Hao Wang, Jichuan Zhang, Lizi Maziashvili, Alexander Genshaft, Peter Reinhold, Brittany Auclair, Robert Mathis, Shawn Wang, Jiang He, George Emanuel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528f741dd4b2dd6cad4ba0b800a0b727e027e98c" target='_blank'>
              Abstract 2583: A T cell engager targeting cancer associated fibroblasts inhibits tumor growth and modulates the tumor microenvironment
              </a>
            </td>
          <td>
            Joshua R Tagore, Robert J Norgard, Pratha Budhani, Sai Charan Penikalapati, Jessica N Egan, Di Feng, Leticia Corrales-Pecino, Anna Bachmayr-Heyda, Claudia Reichel-Voda, Abhishek Kashyap, Youli Xia, Sarah A O'Brien
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdedfede7608cb2fb7f1fa7b1c417998bb4fd21" target='_blank'>
              Spatially resolved transcriptomic analysis of the adult human prostate.
              </a>
            </td>
          <td>
            Junyi Hu, Fei Liu, Jing Zhang, Lei Yin, Wanli Cao, Weidong Xu, Yifan Chang, Ye Wang, Jian Wang, Yaxin Hou, Lilong Liu, Sujun Chen, Guanghui Zhu, Junhui Jiang, Zixian Wang, Gong-Hong Wei, H. H. He, Di Gu, Ke Chen, Shancheng Ren
          </td>
          <td>2025-04-01</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4eb756e3a5aa845f598bc861225d3c569477eee" target='_blank'>
              Integrating construction and spatial imaging of organoid for tumor metabolic heterogeneity and interactions
              </a>
            </td>
          <td>
            Ling Guo, Zhaohua Xia, Jiaxin Wang, Xiaoling Yu, Chuanyue Wu, Qian Luo, Chao Zhao
          </td>
          <td>2025-05-04</td>
          <td>VIEW</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a plasma cell (PC) malignancy that is preceded by monoclonal gammopathy of undetermined significance (MGUS) and/or smoldering multiple myeloma (SMM). MGUS and SMM PCs exhibit the same primary oncogenic abnormalities as MM but lack the end-organ damage that defines proliferative disease, suggesting that clonal PCs in these precursor conditions could exhibit senescence or senescence-like growth arrest. Herein we identified monoclonal gammopathy patient-derived PCs that exhibit senescence features and found that senescent PCs were significantly increased in MGUS patients compared to SMM or MM. Spatial analysis of senescent PCs in stable MGUS and SMM patient biopsies demonstrated the activation of local paracrine senescence in the bone marrow microenvironment. Stable MGUS and SMM patients also exhibited disease-specific senescence-associated secretory phenotype (SASP) signatures that significantly correlated with PC burden and clonal antibody. In contrast, progressing MGUS, SMM, and new MM patients lacked local paracrine senescence responses and robust activation of disease specific SASP signatures. Overall, these data suggest that failure to activate tumor-specific paracrine senescence responses is key to disease progression in monoclonal gammopathies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4be49f177072ce83aa060cab8f4aa0783c0889c" target='_blank'>
              Senescence profiling of monoclonal gammopathies reveals paracrine senescence as a crucial defense against disease progression
              </a>
            </td>
          <td>
            Gabriel Alvares Borges, Marta Diaz-delCastillo, A. Guilatco, B. M. El-Masri, Fatima Mustapha, M. Gundesen, Maja Hinge, Thomas Lund, Nadine Abdallah, Linda B. Baughn, Ming Xu, A. Gingery, Tamar Tchkonia, James L. Kirkland, T. Kourelis, Matthew T. Drake, T. Andersen, Megan M Weivoda
          </td>
          <td>2025-03-31</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94a034a47df46d31c3bc02070b210d92025cd351" target='_blank'>
              Abstract 5283: Decoding and disrupting the coupled cellular plasticity and myeloid cell instigation in metastatic prostate cancer
              </a>
            </td>
          <td>
            Tianhe Huang, Yaoyin Li, Jun Wan, Liang Cheng, Xin Lu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff150fb08f88fc51a25fd22d924b89d76aa03ba" target='_blank'>
              Cell Confluency Affects p53 Dynamics in Response to DNA Damage.
              </a>
            </td>
          <td>
            Alina Simerzin, Emily E Ackerman, Kotaro Fujimaki, Rainer H. Kohler, Yoshiko Iwamoto, Mathias S. Heltberg, A. Jambhekar, Ralph Weissleder, G. Lahav
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a60e85c34b84d3ee7bcc4641260225bbd7766477" target='_blank'>
              Abstract 6350: Exploring genetic and non-genetic evolution of glioblastoma by integrating single cell DNA and RNA sequencing
              </a>
            </td>
          <td>
            F. D’Angelo, A. Ko, L. Garofano, Pedro Davila, Luigi Ferraro, Benjamin Currall, S. Williams, S. Gultekin, Ashish Shah, Ricardo J Komotar, Michael Ivan, Corneliu Sologon, A. Laurenge, A. Picca, Amel Dridi-Aloulou, B. Mathon, Franck Bielle, Stephan Zuchner, M. D. L. Fuente, M. Eoli, Chul-Kee Park, Michele Ceccarelli, Marc Sanson, A. Lasorella, A. Iavarone
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d6445efc526771002f7bfac5c1cb947546e45a" target='_blank'>
              Abstract 5456: Development of human antibodies targeting a novel cancer-associated fibroblast protein identified from spatial transcriptomics
              </a>
            </td>
          <td>
            Jinyeoung Choi, Shengjun Li, Minkyu Kim, Yeonjae Jung, Jae Eun Lee, D. Lee, Hongyoon Choi, Hyung-Jun Im
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT The correlation between the microbiota found in tumors and tumor development is being progressively understood, specifically regarding its involvement in the initiation and advancement of tumors. We examined a total of 102 samples, examining the microbial composition at the species level in each person unveiled significant variations in both the microbial makeup and tumor proportions among individuals, examining the fluctuating alterations in the microbial profile during breast cancer advancement and progression. The levels of expression for Pseudoalteromonas elyakovii were notably elevated in the tumor groups when compared to the para-cancer normal group, aligning with the results obtained from qRT-PCR analysis. The relationship between tumor immunity and microorganisms within the tumor was investigated using double immunofluorescence staining combined with SweAMI probe in situ hybridization and scRNA-seq, allowing for an in-depth analysis of intratumoral microorganisms. Experiments have demonstrated that the supernatant derived from P. elyakovii displayed a significant ability to promote tumor growth and stimulation. In summary, we describe the characteristics of the intratumoral microbiota and the tumor-promoting effects of P. elyakovii supernatant within a small dose range in ductal carcinoma of the breast and characterize the potential clinical application value of intratumoural microorganisms in the progression of cancer and immunotherapy. IMPORTANCE Despite the existing studies, the specific microbial factors that influence the occurrence and progression of breast cancer still remain unclear. Researchers have clarified the distinctive microbial profile related to ductal carcinoma, a common histological type of breast cancer, in order to identify tumor-specific microbes and their roles in tumorigenesis. With the tumor microbiome as the focus, the enrichment of Pseudoalteromonas elyakovii features accelerates the disease progression in patients with ductal carcinoma of the breast. This study reveals the initial role relationship and innovative findings between Pseudoalteromonas elyakovii and ductal carcinoma in the breast. Despite the existing studies, the specific microbial factors that influence the occurrence and progression of breast cancer still remain unclear. Researchers have clarified the distinctive microbial profile related to ductal carcinoma, a common histological type of breast cancer, in order to identify tumor-specific microbes and their roles in tumorigenesis. With the tumor microbiome as the focus, the enrichment of Pseudoalteromonas elyakovii features accelerates the disease progression in patients with ductal carcinoma of the breast. This study reveals the initial role relationship and innovative findings between Pseudoalteromonas elyakovii and ductal carcinoma in the breast.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78eb760dac971fc0c9d9c921e3630cb4941aa8c4" target='_blank'>
              Tumor-colonizing Pseudoalteromonas elyakovii metabolically reprograms the tumor microenvironment and promotes breast ductal carcinoma
              </a>
            </td>
          <td>
            Shuyan Liu, Youpeng Pan, Chaopeng Zheng, Qinghui Zheng, Yaoqiang Du, Yajuan Zheng, Hongchao Tang, Xiaozhen Liu, Jiancheng Mou, Xin Zeng, Zhuotao Yang, Wenjuan Gui, Yuning Tang, Mingxing Xu, Zhihao Ye, Haotian Su, Qiuran Xu, Xuli Meng
          </td>
          <td>2025-04-07</td>
          <td>mBio</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e56cfb98ead022484cef458f223d5c6fb21104d" target='_blank'>
              Abstract 6469: Defining architectural tissue features and cell composition of IDH-mutant gliomas
              </a>
            </td>
          <td>
            Elisabet Rodríguez Tomàs, L. Leiss, Rasul M. Tabiev, Fredrik Nysjö, Anna Klemm, C. Wählby, T. O. Bontell, A. Dénes, A. Jakola, Anja Smits, Arne Östman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976e22baff59fe9d920ebd8e3c77016a68c58b9" target='_blank'>
              Single-Cell Transcriptomics of Multi-Site Cell Therapy in Osteoarthritis: Tissue-Specific Traits and Treatment Correlations
              </a>
            </td>
          <td>
            P. Chatterjee, H. Stevens, L. Kippner, C. Yeago, H. Drissi, K. Mautner, S. Boden, G. Gibson, K. Roy
          </td>
          <td>2025-04-16</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f05272abf51c9f38dd50672d47db4690dbc1bb3" target='_blank'>
              Spatial Profiling Identifies Regionally Distinct Microenvironments and Targetable Immunosuppressive Mechanisms in Pediatric Osteosarcoma Pulmonary Metastases.
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice S Pereira, Demond Williams, Douglass W Morris-Ii, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael C. Kelly, A. LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-04-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a70d8ffb57c8661082ae6d08c258b2d7875bf6f" target='_blank'>
              Abstract 3894: Three-dimensional whole tumor genomic analysis redefine intratumor heterogeneity of gliomas, with implications for early tumorigenesis
              </a>
            </td>
          <td>
            Lee H. Chen, Nicholas O Stevers, Chibo Hong, Ivan V. Smirnov, Nadia R. Grishanina, Joshua S. Wu, Anny Shai, Susan Marina Chang, Mitchell S. Berger, Shawn L. Hervey-Jumper, Joanna J Phillips, Joseph F Costello
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbc8689bc43dfd14400580dddce77b1d4a41ac2f" target='_blank'>
              Abstract LB047: Resolving tumor heterogeneity by uncovering novel genomic and transcriptomic events with a new scaled and automated workflow
              </a>
            </td>
          <td>
            Shuwen Chen, Peng Xu, Xuan Li, Joseph Liu, Yana Ryan, Kazuo Tori, Hima Anbunathan, Alan Du, Mike Covington, Raymond Mendoza, Samantha Leong, Tomoya Uchiyama, Mohammad Fallahi, Xuan Qu, Xiaoyun Xing, Bryan Bell, Patricio Espinoza, Ting Wang, Yue Yun, Andrew Farmer
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e120967c6039c9b8a8507c569dbb9140d2239a00" target='_blank'>
              Abstract 5320: Integrative spatial omics reveals distinct tumor-promoting multicellular niches and immunosuppressive mechanisms in African American and European American patients with TNBC
              </a>
            </td>
          <td>
            Qian Zhu, Akhila Balasubramanian, Jaya Ruth Asirvatham, Badrajee Piyarathna, Jaspreet Kaur, Nada Mohamed, Ling Wu, Megha Chatterjee, Stacy Wang, Niloufar Pourfarrokh, Uttam Rasaily, Yitian Xu, Junjun Zheng, D. Jebakumar, Arundathi Rao, Shu-Hsia Chen, Yi Li, Eric Chang, Xiaoxian Li, R. Aneja, X. Zhang, Arun Sreekumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08487b7db3c7e22e3795c18f3a3ea30e842f09b5" target='_blank'>
              Abstract 5770: Single-cell Raman-based omics of the melanoma tumor-immune microenvironment for precision therapy
              </a>
            </td>
          <td>
            Kai Chang, Mamatha Serasanambati, Baba Ogunlade, Ariel Stiber, Fareeha Safir, H. Delgado, Amanda R. Kirane, Jennifer A. Dionne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a complex, dynamic ecosystem where tumor cells interact with diverse immune and stromal cell types. This review provides an overview of the TME’s evolving composition, emphasizing its transition from an early pro-inflammatory, immune-promoting state to a later immunosuppressive milieu characterized by metabolic reprogramming and hypoxia. It highlights the dual roles of key immunocytes—including T lymphocytes, natural killer cells, macrophages, dendritic cells, and myeloid-derived suppressor cells—which can either inhibit or support tumor progression based on their phenotypic polarization and local metabolic conditions. The article further elucidates mechanisms of immune cell plasticity, such as the M1/M2 macrophage switch and the balance between effector T cells and regulatory T cells, underscoring their impact on tumor growth and metastasis. Additionally, emerging therapeutic strategies, including checkpoint inhibitors and chimeric antigen receptor (CAR) T and NK cell therapies, as well as approaches targeting metabolic pathways, are discussed as promising avenues to reinvigorate antitumor immunity. By integrating recent molecular insights and clinical advancements, the review underscores the importance of deciphering the interplay between immunocytes and the TME to develop more effective cancer immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11004abebcf7dd56b12c4dc898da1ed0acccbe13" target='_blank'>
              Immunocytes in the tumor microenvironment: recent updates and interconnections
              </a>
            </td>
          <td>
            Jiyao Yu, Li Fu, Rui Wu, Linyi Che, Guodong Liu, Qinwen Ran, Zhiwei Xia, Xisong Liang, Guanjian Zhao
          </td>
          <td>2025-04-14</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc725ddd29c73799b619a4c59fece243a3bbe0ad" target='_blank'>
              Abstract 2578: GDF15 reprograms the microenvironment to drive the development of uveal melanoma liver metastases
              </a>
            </td>
          <td>
            S. Sriramareddy, Navya Siddarajappa, Thanh Nguyen, Jeffim Kunetsov, Stefan Kurtenbach, J. Dollar, Jonathan D. Licht, Ann Y. Chen, William J. Harbour, K. Smalley
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Transcription factors play a central role in cancer growth, progression, and metastasis, and contribute to intratumor phenotypic plasticity which enable drug tolerance and cancer relapse. Changes in the regulatory activities of transcription factors in cancer may not always be detected by investigating mutational signatures or differential expression of the transcription factor, as done in traditional analysis. In addition, past studies have focused on the activities of transcription factors in tumors as a whole and thus, have not fully captured the heterogeneity in gene regulation among different cell types within the tumor microenvironment. In this work, through an analysis of the transitions in regulatory network architecture and gene regulation dynamics, we identify the central transcription factors associated with lung adenocarcinoma progression. The gene NR2F1, associated with neurodevelopment and cancer dormancy, emerge as a key transcription factor in the progression of lung adenocarcinoma. We further identify transcription factors that are only active in cancer samples and uncover how changes in gene regulation dynamics influence intratumor heterogeneity. Taken together, our work elucidates the transitions in gene regulatory network during cancer progression, identifies central transcription factors in this process, and reveals the complex regulatory changes cooccurring in different cell types within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a902ef3e0da1388bf4b1861117a26aab25af6066" target='_blank'>
              Gene regulatory network transitions reveal the central transcription factors in lung adenocarcinoma progression
              </a>
            </td>
          <td>
            Upasana Ray, Adarsh Singh, Debabrata Samanta, Riddhiman Dhar
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d80f820ef529b42735b0a0485a4a2c9e277fa1" target='_blank'>
              Abstract 6309: CliPP-on-Web: a dynamic platform for clonal structure identification through penalizing pairwise differences in cancers
              </a>
            </td>
          <td>
            Aaron Wu, Quang Tran, Matthew D. Montierth, Yujie Jiang, Jessica C. Lai, Shuangxi Ji, Wenyi Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daad54b0fcb7c5c704adea069e134ecf7c7a856b" target='_blank'>
              Abstract 1252: Evolution of transcriptomic and epigenomic intra-tumor heterogeneity in high-grade serous ovarian cancer with chemotherapy
              </a>
            </td>
          <td>
            Yuna Landais, Juliette Bertorello, Marta Puerto, Baptiste Simon, Amélie Roehrig, Adeline Durand, Marceau Quatredeniers, C. Lamy, Fabrice Lécuru, Christophe Le Tourneau, Céline Vallot
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Coupling single-cell transcriptomics with tumor-initiating cell enrichment identified IFN response gene expression not previously reported in bulk RNA-sequencing–derived signatures and proposed IFN/STAT1 signaling as a candidate therapeutic target in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49032c731c7f43ad3396d95549c8d805e477fd3" target='_blank'>
              Lineage Tracing and Single-Cell RNA Sequencing Reveal a Common Transcriptional State in Breast Cancer Tumor–Initiating Cells Characterized by IFN/STAT1 Activity
              </a>
            </td>
          <td>
            Eric P Souto, Ping Gong, John D. Landua, R. Srinivasan, Abhinaya Ganesan, L. Dobrolecki, S. C. Purdy, Xingxin Pan, Michael Zeosky, Anna Chung, S. S. Yi, H. Ford, Michael T. Lewis, Connie Cepko, Dr. Kenneth L. Scott, Dr. Michael Davidson
          </td>
          <td>2025-02-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bbd04b9885757e3389b1d8ac08711c046f83506" target='_blank'>
              Abstract 5068: Spatial architecture and epigenetic programs in IDH-mutant and wildtype glioma
              </a>
            </td>
          <td>
            Jingxian Liu, Song Cao, Wagma Caravan, Xiangwei Fang, Preet Lal, Andrew Houston, Yuting Zhao, F. Simin, Michael C. Jin, J. Ponce, Robert S Fulton, Michael Heinz, Yang E. Li, Sonika M. Dahiya, D. Mathios, Tao Liu, Albert H. Kim, M. Chheda, Li Ding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/086b2a8f8a0217ab42193f943b683f5fc349b5af" target='_blank'>
              Abstract 2749: Single-cell 3D genome structure analysis reveals clonal evolution and heterogeneity in acute myeloid leukemia
              </a>
            </td>
          <td>
            Yu Luan, Ye Hou, Yutong Lei, Qiushi Jin, Jie Xu, Bei Jia, Siwei Xu, Ping Wang, Alok Swaroop, Yihao Fu, Juan Wang, Qixuan Wang, Hong Zheng, Jing Zhang, Feng Yue
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/743a748682182612c504b30793e3105a56193c50" target='_blank'>
              TIGIT expression dictates the immunosuppressive reprogramming of myeloid cells in glioblastoma
              </a>
            </td>
          <td>
            Mohammad Asad, Julio Inocencio, Stefan Mitrasinovic, Minori Aoki, C. Crisman, Patrick Lasala, Emad Eskandar, Chandan Guha, Xingxing Zang, I. Parney, B. Himes
          </td>
          <td>2025-05-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87978b2579ac3a33bf3369fd95388496b8a42e2" target='_blank'>
              Genomic and Single-Cell Analyses Characterize Patient-Derived Tumor Organoids to Enable Personalized Therapy for Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Jung Hyun Um, Yueyuan Zheng, Qiong Mao, Chehyun Nam, Hua Zhao, Yoon Woo Koh, Su-Jin Shin, Y. Park, De-Chen Lin
          </td>
          <td>2025-04-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18fc5411928817db2e4ae82f8b06dabe1ec683eb" target='_blank'>
              Abstract 2672: Eco-evolutionary profiling of the UV-driven clonal dynamics and tumor initiation
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0432bb6f94ff8d24b24c3d2aa77012cd1f35bed1" target='_blank'>
              Abstract 2081: Tissue-niche-based and cell-type-selective proteomics
              </a>
            </td>
          <td>
            Yichao Wu, Elie Abi Khalil, Aditi Upadhye, Jalees Rehman, N. Kady, Ryan Wilcox, Steve Seung‐Young Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="HOX (homeobox) genes are virtually absent in healthy adult brains but are detected in malignant brain tumors, particularly gliomas. In 2021, the World Health Organization (WHO) classified adult-type diffuse gliomas into three distinct categories: astrocytomas (isocitrate dehydrogenase [IDH]-mutated), oligodendrogliomas (IDH-mutated and 1p/19q-deleted), and glioblastomas, IDH-wildtype (GBM). GBM is the most common and aggressive primary malignant tumor of the Central Nervous System (CNS), characterized by its high recurrence rate and rapid growth. Dysregulation of HOX genes is a well-established phenomenon in both solid and liquid malignancies, playing crucial roles in various fundamental characteristics of cancer, including GBM. In recent years, HOX genes have gained recognition not only as key regulators of tumor progression but also as potential biomarkers for predicting disease outcomes and as promising therapeutic targets for GBM. This review compiles the latest research on HOX genes in GBM, encompassing studies published before and after the 2021 WHO classification of CNS tumors. Our goal is to provide a comprehensive overview of key findings on the role of HOX gene clusters, which are groups of genes involved in regulating the development of the body plan along the anterior–posterior axis, in GBM initiation, progression, prognosis, and treatment response. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02684-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e4a53ead8582c3ade2d72aeb49d5ea960a31a96" target='_blank'>
              HOX gene dysregulation in glioblastoma: a narrative review of current advances
              </a>
            </td>
          <td>
            Suleyman Hekim, S. Bunda, Shaliman Dilibaerguli, Maierdan Palihati, Yasin Mamatjan
          </td>
          <td>2025-05-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a34629f096bf89244c75b2f93047a0ddf47ecc20" target='_blank'>
              Single-Cell Atlas Reveals Tumorigenic Profiles and Immune Dynamics of Adrenal Incidentalomas.
              </a>
            </td>
          <td>
            Meng Wang, Guangmin Zheng, Xiaoyong Hu, Feng Tian, Tuo Li, Zheng Zhang, Kan Gong, Shiwei Chen, Lin Yuan, Yu Qi, Lin Li, Daofu Cheng, Liu Liu, Fuqiang Liu, Yujing Sun, Xiangdong Fang, Ruxing Zhao, Bing Liu, Chao Zhang
          </td>
          <td>2025-04-07</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0c39aa857bbfd9543aa84644d12df58e1a71f0a" target='_blank'>
              Single-cell and spatial RNA sequencing identify divergent microenvironments and progression signatures in early- versus late-onset prostate cancer.
              </a>
            </td>
          <td>
            Yifei Cheng, Bingxin Liu, Junyi Xin, Xiaobin Wu, Wenchao Li, Jinwei Shang, Jiajin Wu, Zhengdong Zhang, Bin Xu, Mulong Du, Gong Cheng, Meilin Wang
          </td>
          <td>2025-04-10</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5253c5eabd95acbc435d8663b104faf3baf67ce" target='_blank'>
              Abstract 1454: Investigating the expression of PTPRN2 and MT1X genes in glioblastoma cell migration using hydrogel-based models
              </a>
            </td>
          <td>
            Ananya Bharadwaj, Talia M. Sanazzaro, S. Seidlits
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Introduction Immune checkpoint inhibitors (ICIs) have significantly improved survival for patients with metastatic melanoma, yet many experienceresistance due to immunosuppressive mechanisms within the tumor immune microenvironment (TIME). Understanding how the spatial architecture of immune and inflammatory components changes across disease stages may reveal novel prognostic biomarkers and therapeutic targets. Methods We performed high-dimensional spatial profiling of two melanoma tissue microarrays (TMAs), representing Stage III (n = 157) and Stage IV (n = 248) metastatic tumors. Using imaging mass cytometry (IMC) and multiplex immunofluorescence (mIF), we characterized the phenotypic, functional, and spatial properties of the TIME. Cellular neighborhoods were defined by inflammatory marker expression, and spatial interactions between immune and tumor cells were quantified using nearest-neighbor functions (G-cross). Associations with survival were assessed using Cox proportional hazards models with robust variance estimation. Results Stage IV tumors exhibited a distinct immune landscape, with increased CD74- and MIF-enriched inflammatory neighborhoods and reduced iNOS-associated regions compared to Stage III. Cytotoxic T lymphocytes (CTLs) and tumor cells were more prevalent in Stage IV TIME, while B cells and NK cells were depleted. Spatial analysis revealed that CTL–Th cell, NK–T cell, and B–NK cell interactions were linked to improved survival, whereas macrophage aggregation and excessive B–Th cell clustering in inflammatory regions correlated with worse outcomes. Organ-specific analyses showed that CTL infiltration near tumor cells predicted survival in gastrointestinal metastases, while NK–T cell interactions were prognostic in lymph node and skin metastases. Discussion Our results reveal stage-specific shifts in immune composition and spatial organization within the melanoma TIME. In advanced disease, immunosuppressive neighborhoods emerge alongside changes in immune cell localization, with spatial patterns of immune coordination—particularly involving CTLs, NK cells, and B cells—strongly predicting survival. These findings highlight spatial biomarkers that may refine patient stratification and guide combination immunotherapy strategies targeting the inflammatory architecture of the TIME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26ea85ba1c4980ace36f2d1785fed9a6143d63d8" target='_blank'>
              Characterizing spatial immune architecture in metastatic melanoma using high-dimensional multiplex imaging
              </a>
            </td>
          <td>
            Joel Eliason, S. Krishnan, Yasunari Fukuda, Matias A. Bustos, Daniel G. Winkowski, Sungnam Cho, Akshay V. Basi, Regan Baird, Elizabeth A. Grimm, Michael A. Davies, Dave S. B. Hoon, Arvind Rao, Jared K. Burks, S. Ekmekcioglu
          </td>
          <td>2025-04-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c5fa7d80af5b43992f24ebd211378eba4c94dea" target='_blank'>
              Abstract 503: Tracking evolution of transcriptional landscapes during colorectal cancer development and progression
              </a>
            </td>
          <td>
            Sudhir B. Malla, Natalie Fisher, Raheleh Amirkhah, J. Edwards, Owen J. Sansom, Philip D Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce63b80e10f58d9373f4a7b7bf33e6cdea133649" target='_blank'>
              Abstract 6355: Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy
              </a>
            </td>
          <td>
            Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen E. Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea1a8d6967192526fb7f6ecd11f9a23af479e446" target='_blank'>
              Abstract 7499: Driver mutations propel subclonal evolution and spatial patterns of tumor growth in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Evit R. John, Pooja Shukla, Anish J Jain, Alexandria Lau, T. Lesluyes, Huw Ogilvie, M. Tarabichi, J. Roszik, Yinyin Yuan, P. Van Loo, Lawrence Kwong, Yun Shin Chun
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77c67aedf7052778400f6b8258e4d803910f72b0" target='_blank'>
              Abstract 2668: Spatial-temporal diversity of extrachromosomal DNA shapes urothelial carcinoma evolution and tumor-immune microenvironment
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, J. Luebeck, Huiying Tao, Yanfen Zhu, X. Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xiang Xi, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B. Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) continues to present a therapeutic challenge due to the fact that by definition, these cancer cells lack the expression of targetable receptors. Current treatment options include cytotoxic chemotherapy, antibody–drug conjugates (ADC), and the PD-1 checkpoint inhibitor, pembrolizumab. Due to high rates of recurrence, current guidelines for early-stage TNBC recommend either multi-agent chemotherapy or chemo–immunotherapy in all patients other than those with node-negative tumors < 0.5 cm. This approach can lead to significant long-term effects for TNBC survivors, driving a growing interest in de-escalating therapy where appropriate. Tumor infiltrating lymphocytes (TILs) represent a promising prognostic and predictive biomarker for TNBC. These diverse immune cells are present in the tumor microenvironment and within the tumor itself, and multiple retrospective studies have demonstrated that a higher number of TILs in early-stage TNBC portends a favorable prognosis. Research has also explored the potential of TIL scores to predict the response to immunotherapy. However, several barriers to the widespread use of TILs in clinical practice remain, including logistical and technical challenges with the scoring of TILs and lack of prospective trials to validate the trends seen in retrospective studies. This review will present the current understanding of the role of TILs in TNBC and discuss the future directions of TIL research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179f3f7188fc55a777be68aae10786476ade4158" target='_blank'>
              Harnessing Tumor-Infiltrating Lymphocytes in Triple-Negative Breast Cancer: Opportunities and Barriers to Clinical Integration
              </a>
            </td>
          <td>
            Cara Coleman, Tharakeswari Selvakumar, A. Thurlapati, Kevin Graf, Sushma Pavuluri, Shikhar Mehrotra, Ozgur Sahin, Abirami Sivapiragasam
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b20de9f45ab191e47d9cf9a94c04226356537fe" target='_blank'>
              The Role of Myeloid-derived Suppressor Cells and Non-tumor Cells in the Glioma Microenvironment
              </a>
            </td>
          <td>
            O. Beylerli, Elmar Musaev, Sergey Roumiantsev, Vladimir Chekhonin, Valentin Pavlov, I. Gareev
          </td>
          <td>2025-04-08</td>
          <td>Current Gene Therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumor dissemination is a key event in tumor progression. During this event, a main role is played by circulating tumor cells (CTCs), immune cells, and their interaction. How the immune system supports the survival and proliferation of CTCs is not fully elucidated. In this study we established an in-vitro co-culture system consisting of immune cells and CTCs from the same patient, which increased the success rate in the establishment of CTC-derived long-term cell cultures. In this system, we characterized the immune cells of successful co-cultures and the signals they exchange with cancer cells, including cytokines and extracellular vesicle (EV) content. Using this protocol, we stabilized four CTC-derived cell lines from patients with metastatic gastroesophageal cancer, which were cultured for over a year and characterized from a genetic and molecular point of view. The four cell lines harbor shared chromosomal aberrations including the amplification at 8q24.21 containing MYC and deletion 9p21.3 containing CDKN2A/B and the IFN type I cluster. The transcriptomic profile of CTC cell lines is distinct from primary tumors, and we detected the activation of E2F, G2M and MYC pathways and the downregulation of interferon response pathway. Each cell line shows a degree of invasiveness in zebrafish in-vivo, and the most invasive ones share the same mutation in RAB14 gene. In addition, the four cell lines secrete cell-line specific EVs containing microRNAs that target YAP, BRG1-AKT1, TCF8-HDAC pathways. Overall, we highlight how the immune system plays a key role in the proliferation of CTCs through EV signaling, and how CTC cell line genomic and transcriptomic alterations make these cells less visible from the immune system and likely responsible for the survival advantage in sites distant from the microenvironment of origin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fac3e4fbae2c7ba5a9e39e780e9dfb2e2cf33f" target='_blank'>
              Gastroesophageal circulating tumor cell crosstalk with peripheral immune system guides CTC survival and proliferation
              </a>
            </td>
          <td>
            T. Rossi, Martina Valgiusti, Maurizio Puccetti, Giacomo Miserocchi, Michele Zanoni, D. Angeli, Chiara Arienti, Ilaria Pace, C. Bassi, Ivan Vannini, M. Melloni, E. Bandini, Milena Urbini, Massimo Negrini, Massimiliano Bonafé, Manuela Ferracin, Giulia Gallerani
          </td>
          <td>2025-03-29</td>
          <td>Cell Death & Disease</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba2b2e41c178b6dbd66cbfc9ba10d448bf6866f2" target='_blank'>
              Spatial multi-omics reveals cell-type-specific nuclear compartments.
              </a>
            </td>
          <td>
            Yodai Takei, Yujing Yang, Jonathan White, Isabel N. Goronzy, Jina Yun, Meera Prasad, Lincoln J Ombelets, Simon Schindler, Prashant Bhat, Mitchell Guttman, L. Cai
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>2</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63a435551ab128bd453aa26cc898c619184a9dc" target='_blank'>
              Abstract 6238: Interactions between ploidy and resource availability shape clonal interference in glioblastoma
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, Konstantin Maksin, Parag Katira, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Mesenchymal stromal cells belong to a diverse collection of cells in different states that are poorly characterized in soft-tissue sarcomas. In this study, we explored tumor growth-regulatory signaling between differentially educated non-malignant mesenchymal stromal cells and malignant cells in pediatric rhabdomyosarcoma (RMS). Methods Xenograft experiments demonstrated that non-malignant stromal cells influence tumor behavior. Gene expression analysis identified deregulated genes, which were further studied using cell culture assays and patient data. Clinicopathological correlations were made in a discovery cohort (N = 147) and a validation cohort (N = 101). Results The results revealed transiently suppressive paracrine effects of orthotopic stromal cells derived from skeletal muscle. These effects were lost when the stromal cells were exposed to RMS cells, either short-term in vitro, or long-term in hindlimb muscle in vivo. High resolution microarray-based Clariom D gene expression analysis identified insulin-like growth factor binding protein 5 (IGFBP5) as the top upregulated gene in RMS cells exposed to naïve stromal cells, and effects on growth arrest, caspase 3/7 activation, and myogenic cell identity were demonstrated in functional assays. Furthermore, IGFBP5 associated with the caspase 3 substrate growth arrest specific protein 2 (GAS2), lower disease stage and favorable survival in patient cohorts. Conclusions This study uses functional modeling and omics approaches to identify IGFBP5 as a candidate mediator of anti-tumor growth mechanisms originating from tumor-neighboring mesenchymal stromal cells. Tumors of mesenchymal origin, such as RMS, are known for their heterogeneity, and this could potentially pose a limitation to the study. However, a clinical relevance is emphasized by consistent findings across patient cohorts. These insights pave the way for novel therapeutic strategies modulating activities of stromal cell subsets at primary and metastatic sites in RMS. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02171-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e41dcbfd236451c27b7bc073e4993ea2fbfdf1" target='_blank'>
              Myogenic IGFBP5 levels in rhabdomyosarcoma are nourished by mesenchymal stromal cells and regulate growth arrest and apoptosis
              </a>
            </td>
          <td>
            Yue Zhang, Karim Katkhada, Liu Zhen Meng, Binbin Zhao, Shanlin Tong, Wiem Chaabane, Aditi Kallai, Nicholas P Tobin, Arne Östman, Alessandro Mega, M. Ehnman
          </td>
          <td>2025-04-15</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b54e815fc1726aef84f4ee9a37487834e0596053" target='_blank'>
              Abstract 6615: Uncovering the landscape of cellular heterogeneity in low- and high-grade endometrial cancer using single cell profiling
              </a>
            </td>
          <td>
            Sabina Swierczek, E. Garfinkle, Sophia A. Paxton, Katherine E. Miller, David Doo, Leslie Andriani, Shatovisha Dey, Steven Sieber, Alpana Dholakia, Syeda Rashid, Linus Chuang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4707d957a214c6885ebfa8cd85de26bb43f4b999" target='_blank'>
              Abstract 5097: Single cell analysis reveals upregulation of immune stimulatory pathways in quiescent metastatic castration resistant prostate cancer cells
              </a>
            </td>
          <td>
            K. G. Ibrahim, Julie Boerner, Steven Zielske, F. Cackowski
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Significance The spatial organization of cells within tissues plays a critical role in maintaining normal function and is often disrupted in disease. Understanding how cells are organized in complex tissues has been advanced by technologies that analyze proteins, RNA, and DNA in their spatial contexts. However, existing spatial multiomics methods often rely on specialized equipment and reagents, limiting their accessibility. To address this, we developed SPatial assay for Accessible chromatin, Cell lineages, and gene Expression with sequencing (SPACE-seq), a technique that enables the simultaneous analysis of multiple molecular features-gene expression, chromatin accessibility, and mitochondrial DNA mutations-using a widely available spatial transcriptomics platform. SPACE-seq offers a unified molecular approach, providing a versatile solution for studying the multiomics landscape of complex tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea7c3fb7a2a3d176707a9c763bcbadbf43d28332" target='_blank'>
              Unified molecular approach for spatial epigenome, transcriptome, and cell lineages
              </a>
            </td>
          <td>
            Yung-Hsin Huang, J. Belk, Ruochi Zhang, Natasha E Weiser, Zachary D. Chiang, Matthew G. Jones, P. Mischel, Jason D. Buenrostro, Howard Y. Chang
          </td>
          <td>2025-04-18</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9784b91d41ea5a8973638da603158b697d7d3373" target='_blank'>
              Abstract 1110: Longitudinal and multi-regional analysis of cancer phenotypes in hepatocellular carcinoma and intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Javier Villela Castrejon, Dongya Jia, J. T. George
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d000b5dbafb928d866718b2a6f1d1452be0aec5" target='_blank'>
              Intercellular communication between FAP+ fibroblasts and SPP1+ macrophages in prostate cancer via multi-omics
              </a>
            </td>
          <td>
            Tingting Wu, Xinyu Li, Fei Zheng, Hanchao Liu, Yang Yu
          </td>
          <td>2025-05-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b2192d47a9207dc2a6bdb7a97c977402cac1b23" target='_blank'>
              Abstract 7048: Molecular and immune landscape of extracranial malignant rhabdoid tumors (eMRT) and atypical teratoid rhabdoid tumors (ATRT): Insights into disease mechanisms
              </a>
            </td>
          <td>
            Maria D. Hernandez Ramirez, Marlena Mucha, Victoria E. Fincke, M. Hasselblatt, C. Vokuhl, M. Frühwald, Pascal D. Johann
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce16964e484673e282fc2269d3a4d1174c03a3ea" target='_blank'>
              Abstract 758: Spatial profiling of matched primary and recurrent glioblastoma samples reveals changes in cellular states and the immune microenvironment
              </a>
            </td>
          <td>
            S. Artzi, Marc N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, N. Bager, N. M. Hammouda, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Structural variants (SVs) are increasingly recognized as important contributors to oncogenesis through their effects on 3D genome folding. Recent advances in whole-genome sequencing have enabled large-scale profiling of SVs across diverse tumors, yet experimental characterization of their individual impact on genome folding remains infeasible. Here, we leveraged a convolutional neural network, Akita, to predict disruptions in genome folding caused by somatic SVs identified in 61 tumor types from the Children’s Brain Tumor Network dataset. Our analysis reveals significant variability in SV-induced disruptions across tumor types, with the most disruptive SVs coming from lymphomas and sarcomas, metastatic tumors, and germline cell tumors. Dimensionality reduction of disruption scores identified five recurrently disrupted regions enriched for high-impact SVs across multiple tumors. Some of these regions are highly disrupted despite not being highly mutated, and harbor tumor-associated genes and transcriptional regulators. To further interpret the functional relevance of high-scoring SVs, we integrated epigenetic data and developed a modified Activity-by-Contact scoring approach to prioritize SVs with disrupted genome contacts at active enhancers. This method highlighted highly disruptive SVs near key oncogenes, as well as novel candidate loci potentially implicated in tumorigenesis. These findings highlight the utility of machine learning for identifying novel SVs, loci, and genetic mechanisms contributing to pediatric cancers. This framework provides a foundation for future studies linking SV-driven regulatory changes to cancer pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e24a79ed206cbada1bc6b51069582d272e00ef" target='_blank'>
              Machine learning-predicted chromatin organization landscape across pediatric tumors
              </a>
            </td>
          <td>
            Ketrin Gjoni, Shu Zhang, Rachel E. Yan, Bo Zhang, Daniel Miller, Adam C. Resnick, Nadia Dahmane, K. Pollard
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d67b2668a358ce85c979198ea57d81c3f6890b" target='_blank'>
              Abstract 5013: The interplay of genomic aberrations, transcriptional states, and therapeutic responses in lung cancer
              </a>
            </td>
          <td>
            Han-Byoul Cho, Noshad Hosseini, Yi-Wan Hsiao, R. Mannan, Chandan Kumar-Sinha, Yuping Zhang, Gabriel Cruz, Yamei Deng, Taiwan Cancer Moonshot Program, Clinical Proteomic Tumor Analysis Consortium, Natalie Clark, S. Satpathy, Gilbert S. Omenn, Yu-Ju Chen, Michael Gillette, Marcin P. Cieslik, S. Dhanasekaran, Alexey I. Nesvizhskii, A. Chinnaiyan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>168</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c41366c3fd4fa6147862f4f28b5b22377077f441" target='_blank'>
              Abstract 2579: Integrated sequencing analysis reveals prognostic thyroid cancer-associated fibroblast subpopulations
              </a>
            </td>
          <td>
            Matthew A. Loberg, George J. Xu, Claudia C. Wahoski, Kailey P. Caroland, Megan L. Tigue, Sheau-Chiann Chen, Hua-Chang Chen, Quanhu Sheng, Eric C. Huang, Fei Ye, Ryan H. Belcher, Ethan Lee, V. Weiss
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b227688b3630d892a146eef43f17b0e1088321e" target='_blank'>
              Abstract 5019: Genomic landscape, tumor heterogeneity and immune-related post-translational regulation in cutaneous melanoma
              </a>
            </td>
          <td>
            Jianhui Gong, Gabriel Cruz, Yamei Deng, Maria Virginia Ruiz Cuevas, Chandan Kumar-Sinha, R. Mannan, Yi-Wan Hsiao, F. Petralia, Weiping Ma, Shrabanti Chowdhury, Noshad Hosseini, Cptac Clinical Proteomic Tumor Analysis Consortium, Gilbert S. Omenn, A. Chinnaiyan, Paul Harms, Raghav Jain, Jennifer Kyle, Avi Ma’ayan, S. Dhanasekaran, K. Clauser, Hui Zhang, Alexey I. Nesvizhskii, S. Carr, Tao Liu, Pei Wang, Marcin P. Cieslik
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>168</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f499ed27ee797b62ddf192d5f361132033db2e" target='_blank'>
              Abstract 5080: Investigation of exome-wide tumor heterogeneity on colorectal tissue-based single cells
              </a>
            </td>
          <td>
            Nikolett Szakállas, Alexandra Kalmár, B. Barták, Béla Molnár, István Takács
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ddc0ae23b848c6a2bead56e11b48f6e63657e8f" target='_blank'>
              Abstract 3900: Integrating single-molecule localization microscopy and computational pipelines to elucidate ecDNA dynamics
              </a>
            </td>
          <td>
            Mesut Unal, Jessica Alsing, Gabrielle S. Dewson, Guy Nir, Kadir C Akdemir
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef9248da70f170d3d4f13da4454da93eec256be3" target='_blank'>
              Abstract 6557: Human 16p11.2 microdeletion suppresses malignant progression of colon cancer through dysregulation of the tumor microenvironment
              </a>
            </td>
          <td>
            Jacob L. Billingsley, Joshua R. Haebe, Chris Bergin, Tanguy Fenouil, Baptiste Lacoste, Yannick D Benoit
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a10eeef1d1c839000cc335b995c3c2a9ec510a" target='_blank'>
              Multimodal cell maps as a foundation for structural and functional genomics.
              </a>
            </td>
          <td>
            Leah V. Schaffer, Mengzhou Hu, Gege Qian, Kyung-Mee Moon, Abantika Pal, Neelesh Soni, Andrew P Latham, Laura Pontano Vaites, Dorothy Tsai, Nicole M Mattson, Katherine Licon, Robin Bachelder, Anthony J. Cesnik, Ishan Gaur, Trang Le, William D. Leineweber, Ajikarunia Palar, Ernst H. Pulido, Yue Qin, Xiaoyu Zhao, Christopher Churas, Joanna Lenkiewicz, Jing Chen, K. Ono, Dexter Pratt, Peter Zage, Ignacia Echeverria, Andrej Šali, J. W. Harper, Steven P. Gygi, L. Foster, Edward L. Huttlin, Emma Lundberg, T. Ideker
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>2</td>
          <td>82</td>
        </tr>

        <tr id="The dynamic three-dimensional spatial conformations of chromosomes demonstrate complex structural variations across single cells, which plays pivotal roles in modulating single-cell specific transcription and epigenetics landscapes. The high rates of missing contacts in single-cell chromatin contact maps impose significant challenges to reconstruct high-resolution spatial chromatin configurations. We develop a data-driven algorithm, Tensor-FLAMINGO, based on a low-rank tensor completion strategy. Implemented on a diverse panel of single-cell chromatin datasets, Tensor-FLAMINGO generates 10kb- and 30kb-resolution spatial chromosomal architectures across individual cells. Tensor-FLAMINGO achieves superior accuracy in reconstructing 3D chromatin structures, recovering missing contacts, and delineating cell clusters. The unprecedented high-resolution characterization of single-cell genome folding enables expanded identification of single-cell specific long-range chromatin interactions, multi-way spatial hubs, and the mechanisms of disease-associated GWAS variants. Beyond the sparse 2D contact maps, the complete 3D chromatin conformations promote an avenue to understand the dynamics of spatially coordinated molecular processes across different cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c71d200c8ec0d904fef250777d437b306a20443a" target='_blank'>
              Tensor-FLAMINGO unravels the complexity of single-cell spatial architectures of genomes at high-resolution
              </a>
            </td>
          <td>
            Hao Wang, Jiaxin Yang, Xinrui Yu, Yu Zhang, Jianliang Qian, Jianrong Wang
          </td>
          <td>2025-04-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/153ee449d128aed80c5d24be6809e6ba7cd3c213" target='_blank'>
              Precision Spatial Control of Tumor‐Stroma Interactions in Cancer Models via 3D Bioprinting for Advanced Research and Therapy
              </a>
            </td>
          <td>
            Yan Li, Haijun Cui, Haitao Cui
          </td>
          <td>2025-04-24</td>
          <td>Advanced Functional Materials</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Given their rarity and diversity, a fundamental understanding of the genomic underpinnings for many sarcoma subtypes is still lacking. To better define the molecular landscape of this group of diseases, we perform matched whole exome sequencing and RNA sequencing on a cohort of 1340 sarcoma tumor specimens. We identify recurrent somatic mutations and observe an increased mutational burden in metastatic vs. primary samples (p < 0.001). We observe frequent copy number alterations including whole genome doubling, with this feature being more common in metastatic tumors (p = 0.026). Estimation of immune cell abundances followed by hierarchical clustering identifies five immune subtypes ranging from low to high and we observe inferior overall survival in immune deplete clusters compared to immune enriched (p < 0.01). Interestingly, GIST predominantly form a distinct “immune intermediate” cluster that is marked by a specific enrichment for NK cells (FDR < 0.01).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09dedff6b3501b2376796d9aaef10264491ab2ba" target='_blank'>
              Genomic, transcriptomic, and immunogenomic landscape of over 1300 sarcomas of diverse histology subtypes
              </a>
            </td>
          <td>
            Alex C. Soupir, Oscar E. Ospina, Oliver Hampton, Michelle L Churchman, Michael Radmacher, D. Hedges, David M. McKean, Phaedra Agius, Saman Zeeshan, Nathan D Seligson, Raphael Pollock, D. Liebner, James L. Chen, G. Tinoco, B. Salhia, Martin D McCarter, B. Wilky, Benjamin J. Miller, M. Cavnar, John S. Groundland, Bryan P. Schneider, Gregory Riedlinger, Stephen B Edge, C. Moskaluk, Kenneth Cardona, A. R. Naqash, Ricardo J. Gonzalez, J. Mullinax, David M Joyce, O. Binitie, G. Douglas Letson, A. Naghavi, M. Druta, Damon R Reed, E. Siegel, Jamie K Teer, Brooke L. Fridley, Andrew S Brohl
          </td>
          <td>2025-05-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c597bafcc8146535ec15c83ac7b10f9d86be5a1" target='_blank'>
              Abstract 2187: Functional classification of tumor-associated macrophages and changes in TAM subpopulations in response to various immunotherapies
              </a>
            </td>
          <td>
            Bryan Weselman, S. Noonepalle, Manasa Suresh, Marie Durr, Nithya Gajendran, Alejandro Villagra, Xintang Li
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e1b2442ed827fe07315db2d2daebd9063c39b" target='_blank'>
              Abstract LB048: Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47975e1afc850e6c92f5d1873544c03b447f35c7" target='_blank'>
              Abstract 3698: A longitudinal investigation of temporal/intratumoral heterogeneity by plasma NGS profiling of mutation clones in metastatic colorectal cancer
              </a>
            </td>
          <td>
            Petra Kroupova, Barbora Minarikova, Andrea Chmelarova, Marek Minarik
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [20],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>