<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-09-15 09:29:58 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Understanding tissue development and intra-tissue evolution requires the ability to trace clones in intact tissues coupled with high-plex molecular profiling preserving spatial context. However, current lineage tracing tools are incompatible with spatial omics. Here, we present SUMMIT (Spatially Unveiling Mitochondrial Mutations In Tissues), a spatially-resolved lineage tracing technology that integrates gene expression profiling with mitochondrial mutation-based clone identification. Unlike synthetic lineage recording methods, SUMMIT relies only on endogenous mutations and thus can be applied to human tissues. To address the compositional mixing of cell types within spatial spots, SUMMIT includes a rigorous statistical framework to confidently assign variants to specific cell subpopulations and achieves high power for spatially localized clones by pooling information across neighboring spots. We validated SUMMIT using a controlled model in which we mixed two cancer cell lines in a mouse tumor, then demonstrated it on multiple human tissues including Barrett’s esophagus, gastric cardia, small bowel, and colorectal cancer. Across these samples, we distinguished between global mutations and mutations marking locally restricted clones. The coupled transcriptomic data allowed us to characterize the cell type composition within each clone and delineate their spatial configuration. This integrated approach provides a framework to understand spatially-defined clonal evolution in preserved human tissue.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7b03288f2508b41200f4c7766764bb4883ed895" target='_blank'>
              Mitochondrial clone tracing within spatially intact human tissues
              </a>
            </td>
          <td>
            Sydney A. Bracht, Jiazhen Rong, Rodrigo A. Gier, M. Demarshall, Hailey Golden, Diya Dhakal, Jayne C McDevitt, Feiyan Mo, Emma E. Furth, Alexandra Strauss Starling, Amanda B. Muir, Gary W. Falk, Bryson W. Katona, Ben Z. Stanger, Nancy R. Zhang, Sydney M. Shaffer
          </td>
          <td>2025-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background: Breast cancer, the most prevalent malignancy among women worldwide, exhibits significant heterogeneity, particularly in the tumor microenvironment (TME), which poses challenges for treatment. Spatial transcriptomics (ST) has emerged as a transformative technology, enabling gene expression analysis while preserving tissue spatial architecture. This provides unprecedented insights into tumor heterogeneity, cellular interactions, and disease mechanisms, offering a powerful tool for advancing breast cancer research and therapy. This review aims to synthesize the applications of ST in breast cancer research, focusing on its role in decoding tumor heterogeneity, characterizing the TME, elucidating progression and metastasis dynamics, and predicting therapeutic responses. We also explore how ST can bridge molecular profiling with clinical translation to enhance precision therapy. The key scientific concepts of review included the following: We summarize the technological advancements in ST, including imaging-based and sequencing-based methods, and their applications in breast cancer. Key findings highlight how ST resolves spatial heterogeneity across molecular subtypes and histological variants. ST reveals the dynamic interplay between tumor cells, immune cells, and stromal components, uncovering mechanisms of immune evasion, metabolic reprogramming, and therapeutic resistance. Additionally, ST identifies spatial prognostic markers and predicts responses to chemotherapy, targeted therapy, and immunotherapy. We propose that ST serves as a hub for integrating multi-omics data, offering a roadmap for precision oncology and personalized treatment strategies in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbbaea20f0037aedc680b32a7558f9b67f94b665" target='_blank'>
              Spatial Transcriptomics Decodes Breast Cancer Microenvironment Heterogeneity: From Multidimensional Dynamic Profiling to Precision Therapy Blueprint Construction
              </a>
            </td>
          <td>
            A. Ma, L. Xiang, Jingping Yuan, Qianwen Wang, Lina Zhao, Honglin Yan
          </td>
          <td>2025-07-24</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Summary Intratumoral heterogeneity, originating from genetic, epigenetic, and phenotypic cellular diversity, is pervasive in cancer. As these heterogeneous states employ diverse mechanisms to promote tumor progression, metastasis, and therapy resistance, emergence of cancer heterogeneity is one of the most significant barriers to curative treatment. Here we leverage deep learning approaches to develop a high-throughput image-analysis paradigm with subcellular resolution that quantifies and predicts colorectal cancer (CRC) patient outcome based on tissue architecture and nuclear morphology. We further combine this approach with spatial transcriptomics, multiplex immunohistochemistry, and patient-derived organoids to uncover a dynamic, co-evolutionary relationship between tumor architecture and cell states. We identify clinically relevant architectural interfaces in CRC tissue that diversify cellular identities by favoring distinct cancer stem cell states and thus promote evolution of tumor heterogeneity. Specifically, tissue fragmentation and associated compressive forces promote loss of classic stem cell signature and acquisition of fetal/regenerative stem cell states, which initially emerges as a hybrid state with features of epithelial-to-mesenchyme (EMT) transition. Additional tumor stroma communication then diversifies these states into distinct stem cell and EMT states at the invasive margins of tumors. Reciprocally, Wnt signaling state of tumor cells tunes their responsiveness to tissue architecture. Collectively, this work uncovers a feedback loop between cell states and tissue architecture that drives cancer heterogeneity and cell state diversification. Machine learning-based analyses harness this co-dependency, independently of mutational status or cancer stage, to predict patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8d83b8caf4ac682c803318ebb577d7836c261e" target='_blank'>
              Colorectal cancer heterogeneity co-evolves with tumor architecture to determine disease outcome
              </a>
            </td>
          <td>
            Fabien Bertillot, Solène Hervé, Nelly Drobjazko, Noemi Kedei, Mona El Gendi, Maria O Hernandez, Kai Kruse, E. Erkan, Mirjam I. Binner, Karolina Punovuori, T. Pellinen, M. Ahtiainen, Jan Böhm, J. Mecklin, Caj Haglund, Jaana Hagström, Mike Nikolaev, N. Gjorevski, Matthias Lütolf, Toni T Seppälä, Sara A. Wickström, Yekaterina A. Miroshnikova
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Intrinsic genetic alterations and dynamic transcriptional changes contribute to the heterogeneity of solid tumors. Lung adenocarcinoma (LUAD) is characterized by its significant histological, cellular and molecular heterogeneity. The present study aimed to study the spatial transcriptomics of primary LUAD with initial hopes to decipher molecular characteristics of subtype transitions in LUAD progression, offering new insights for novel therapeutic strategies. Spatial transcriptomics libraries were first generated from tumor samples collected from patients with LUAD who underwent surgical resection in The Fourth Hospital of Hebei Medical University in 2022 and were sequenced using Illumina NovaSeq 6000 system. The processed data were analyzed for differential gene expressions and networks, and were annotated according to cell type, spatial ligand-receptor interaction and trajectory inference. Our analysis revealed 34 annotated cell types, with cancer-associated fibroblasts (CAFs) being the most abundant, playing a crucial role in tumor microenvironment remodeling and prognosis. We noted significant spatial correlations between various immune cells and found that different histological subtypes displayed unique cell composition profiles, particularly in the micropapillary subtype, which exhibited higher macrophage proportions and distinct gene expression pathways related to extracellular matrix organization and receptor tyrosine kinase signaling. Additionally, we explored the dedifferentiation states within these subtypes, identifying that region with higher dedifferentiation scores corresponded to increased tumor invasiveness and potential drug resistance. Our findings demonstrate dynamic biological changes and dedifferentiation states of tumor subtypes during the progression process. This study reveals important biological processes in tumor development and may offer valuable guidance for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a252a3c50c2377c8fca1733740eaa8f4897e893" target='_blank'>
              Spatial transcriptomics uncovers immune-cell plasticity and dedifferentiation signatures in aggressive lung adenocarcinoma subtypes
              </a>
            </td>
          <td>
            Huiyan Deng, Qingyi Liu, Ziqiang Tian, Guan-Yuan Liu, Shize Wang, Di Liang, Yun Wang, Yaqing Han, Shaonan Xie
          </td>
          <td>2025-08-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Racial disparities in the clinical outcomes of triple-negative breast cancer (TNBC) have been well-documented, but the underlying biological mechanisms remain poorly understood. To investigate these disparities, we employed a multi-omic approach integrating imaging mass cytometry and spatial transcriptomics to characterize the tumor microenvironment (TME) in self-identified Black American (BA) and White American (WA) TNBC patients. Our analysis revealed that the TME in BA patients is marked by a network of endothelial cells, macrophages, and mesenchymal-like cells, which correlates with reduced patient survival. In contrast, the WA TNBC microenvironment is enriched in T-cells and neutrophils, indicative of T-cell exhaustion and suppressed immune responses. Ligand-receptor and pathway analyses further demonstrated that BA TNBC tumors exhibit a relatively “immune-cold” profile, while WA TNBC tumors display features of an “inflamed” TME, suggesting the evolution of a unique immunosuppressive mechanism. These findings provide insight into racially distinct tumor-promoting and immunosuppressive microenvironments, which may contribute to the observed differences in clinical outcomes among BA and WA TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2139c15b76d963542dd090ab4bb098c3eb9f6a" target='_blank'>
              Integrative spatial omics reveals distinct tumor-promoting multicellular niches and immunosuppressive mechanisms in Black American and White American patients with TNBC
              </a>
            </td>
          <td>
            Qian Zhu, Akhila Balasubramanian, Jaya Ruth Asirvatham, Megha Chatterjee, Badrajee Piyarathna, Jaspreet Kaur, Nada Mohamed, Ling Wu, Stacy Wang, Niloufar Pourfarrokh, P. D. Binsol, Mahak Bhargava, Uttam Rasaily, Yitian Xu, Junjun Zheng, D. Jebakumar, Arundhati Rao, Carolina Gutierrez, A. Omilian, Carl D. Morrison, Gokul M. Das, C. Ambrosone, Erin H. Seeley, Shu-hsia Chen, Yi Li, Eric Chang, Xiaoxian Li, Elizabeth Baker, Ritu Aneja, X. Zhang, Arun Sreekumar
          </td>
          <td>2025-07-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Tumor relapse remains a significant obstacle to successful therapy. Preclinical animal models that accurately reflect tumor relapse in patients are urgently needed. Here, we employed a dual recombinase-mediated genetic system to genetically trace and ablate proliferating cells in a polyomavirus middle T antigen (PyMT)-induced spontaneous murine breast cancer model. This system enabled the acute ablation of cells that had undergone proliferation within a defined time window, resulting in a drastic tumor shrinkage, followed by a gradual tumor relapse due to the presence of residual low-cycling cells. We then applied single-cell RNA sequencing (scRNA-seq) to unbiasedly compare the tumor ecosystems of the primary and relapsed PyMT tumors. Compared with the primary tumors, the relapsed tumors exhibited a higher proportion of cancer stem cells and pro-tumor γδ T cells, as well as co-expression of Spp1 and Vegfa in multiple myeloid cell populations – features that predict poor therapeutic response and unfavorable outcomes in human breast cancer patients. Collectively, this proliferation tracing and ablation model emulates chemotherapies that preferentially eliminate proliferating cancer cells, serving as a robust tool and a valuable resource for testing novel therapeutic strategies in relapsed tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95fdfebff6c6c3226be581651d876e8c2d584753" target='_blank'>
              Modeling tumor relapse using proliferation tracing and ablation transgenic mouse
              </a>
            </td>
          <td>
            Chuang Zhao, Xin-nan Zheng, Han-Ying Huang, Lin Tian
          </td>
          <td>2025-07-17</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Xenium, a new spatial transcriptomics platform, enables subcellular-resolution profiling of complex tumor tissues. Despite the rich morphological information in histology images, extracting robust cell-level features and integrating them with spatial transcriptomics data remains a critical challenge. We introduce CellSymphony, a flexible multimodal framework that leverages foundation model-derived embeddings from both Xenium transcriptomic profiles and histology images at true single-cell resolution. By learning joint representations that fuse spatial gene expression with morphological context, CellSymphony achieves accurate cell type annotation and uncovers distinct microenvironmental niches across three cancer types. This work highlights the potential of foundation models and multimodal fusion for deciphering the physiological and phenotypic orchestration of cells within complex tissue ecosystems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0df7c5843203625e193cd9c5f8b65c48433cc0" target='_blank'>
              CellSymphony: Deciphering the molecular and phenotypic orchestration of cells with single-cell pathomics
              </a>
            </td>
          <td>
            Paul H. Acosta, Pingjun Chen, Simon P. Castillo, M. E. Salvatierra, Yinyin Yuan, Xiaoxi Pan
          </td>
          <td>2025-08-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="SUMMARY Metastasis is an emergent continuum, driven by evolving reciprocal adaptations between continuously disseminating tumor cells (DTCs) and the specialized metastatic niches of distant organs. The interplay between intrinsic and niche-driven mechanisms that enables DTCs to survive and home to distant organs remains incompletely understood. Here, using MetTag, a single-cell barcoding and transcriptome profiling approach with time-stamped batch identifiers (BC.IDs), we mapped temporal, clonal dynamics of DTCs and the immune cell landscape across ovarian cancer metastatic niches. Deep sequencing of barcodes revealed preferred enrichment of early-disseminated clones across metastatic niches. Mechanistically, single-cell RNA sequencing (scRNA-seq) coupled with velocity analyses in ascites and metastasis-bearing omenta uncovered an emergent, distinct interferon-gamma (IFNγ) centric transcriptional trajectory, enriched among early seeding clones. Moreover, in vivo CRISPR/Cas9 screening of metastatic niche-specific signatures demonstrated that genes belonging to the ascites IFNγ signature, including Marco, Gbp2b, and Slfn1, are functionally important for peritoneal metastasis. Knockout of IFNγ receptor 1 (Ifngr1) in tumor cells significantly reduced metastatic burden and extended survival, underscoring the importance of tumor cell intrinsic IFNγ signaling in ovarian cancer metastasis. Furthermore, we identified that the tumor intrinsic IFNγ response and ascites-derived tumor-associated macrophages (TAMs) protect cancer cells from anoikis-mediated death within the IFNγ-rich ascites environment. Our study resolves temporal dynamics of disseminating tumor cells and highlights an ascites-driven, IFNγ program as a necessary pro-metastatic adaptation in the ovarian metastasis cascade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95e0d9496ced48f75294cbc20541bbba656bbf" target='_blank'>
              Temporal Clonal Tracing and Functional Perturbation Reveal Niche-Adaptive and Tumor-Intrinsic IFNγ Dependencies Driving Ovarian Cancer Metastasis
              </a>
            </td>
          <td>
            Emilija Aleksandrovic, Shaneann Fross, Samantha M. Golomb, Xiyu Liu, Zhuo Zhao, Nikitha M. Das, T. Reese, Wei Ma, Jacqueline Lopez, M. S. Stack, Min Zhao, Siyuan Zhang
          </td>
          <td>2025-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Objective To illustrate a new barcoded orthotopic patient‐derived xenograft (PDX) mouse model where one can investigate phenotypic effects of single‐cell level gene manipulation in a pooled format. To address some concerns of current PDX mouse models of head and neck squamous cell carcinoma (HNSCC): (1) genomic evolution with passage by generating high‐purity cancer cells, which can also be utilized for other downstream applications, including cell culture‐based studies, and (2) cost‐effectiveness of current PDX models. Methods Two‐millimeter tumor cubes from nine patients were implanted into immunodeficient mouse flanks subcutaneously. Purified tumor cells were obtained from subcutaneous xenografts. Various numbers of purified tumor cells were then injected into the lingual tissue of immunodeficient mice, and the lowest amount of cells needed to achieve a 100% orthotopic engraftment rate were identified. Clonal stability was tested using a lentiviral barcoding system. The orthotopic PDXs' genetic landscapes were characterized using whole exome sequencing. Results This approach yielded an overall engraftment rate of 88.9%. The purification process increased cancer cell purity from 34% to 92%. Lingual injection of 100,000 purified tumor cells achieved a 100% orthotopic engraftment rate from purified subcutaneous PDX tumor cells while maintaining clonal and genetic stability. Conclusion Our study presents a barcoded orthotopic patient‐derived xenograft model for head and neck squamous cell carcinoma with clonal stability. This model provides a way to study phenotypic effects of single cell level gene manipulation in a pooled format. The method can be adapted for in vitro work as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/770f65963b9ed4389a6e68906eb989805afcd015" target='_blank'>
              Barcoded Orthotopic Patient‐Derived Head & Neck Squamous Cell Carcinoma Model Demonstrating Clonal Stability and Maintenance of Cancer Driver Mutational Landscape
              </a>
            </td>
          <td>
            Peiran Zhou, Claire B Mills, Zhao Ming Dong, Brittany R Barber, Slobodan Beronja
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Meningioma is a common primary intracranial tumor with high recurrence and metastasis rate. Delineating the pathological ecosystem at the brain-tumor interface (BTI) of meningioma is critical for understanding the mechanisms of tumor metastasis and developing effective new therapies. Methods To identify biomarkers of early metastasis and discover potential therapeutic targets, we integrated single-cell transcriptome datasets of meningioma, and identified the cell populations and molecular signatures uniquely present at the BTI. Results A specific BTI-enriched tumor cell population with a pro-EMT (epithelial mesenchymal transition) characteristics was associated with invasion and metastasis, and ANXA2 and COL5A1 were detected as the biomarkers for these BTI-enriched tumor cells. Additionally, we characterized the BTI-specific immunosuppressive microenvironment composed of SPP1+ tumor-associated macrophages, as well as specific endothelial cells (ACKR1high) and pericytes (THY1high) promoting the highly malignant invasive state of angiogenesis. Conclusions Collectively, BTI in meningioma is a metastatic and immunosuppressive zone. We have discovered potential biomarkers that help detect early metastasis and recurrence of meningioma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06935-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dea39ae7cd89d7f388a32faad30c0efc42673529" target='_blank'>
              Single-cell transcriptomic analysis reveals metastatic and immunosuppressive characteristics in meningioma brain-tumor interface
              </a>
            </td>
          <td>
            Boya Huang, Jinlian Liang, Xiaogen Tang, Yue Zhu, Lijuan Gao, Dongwei Fu, Cheng Wei, Yifang Li, Chao Ke, Hongyi Zhang, Oscar Junhong Luo
          </td>
          <td>2025-08-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Despite advances in cancer therapies, treatment responses remain highly variable due to the complexity and heterogeneity of the tumor microenvironment. The tumor microenvironment comprises malignant, immune, stromal, and endothelial cells, along with extracellular matrix and soluble factors, all organized into spatially distinct communities that evolve dynamically throughout tumor progression and therapy. These spatial structures orchestrate tumor behavior, immune evasion, and drug resistance. Recent breakthroughs in spatial omics technologies, including spatial transcriptomics and spatial proteomics, have enabled high-resolution, multiplexed mapping of tissue architecture and molecular characteristics. These technologies provide valuable insights into how the spatial organization of cells and signaling networks within the tumor microenvironment influences therapeutic efficacy. Notably, specific structures, such as tertiary lymphoid structures, fibroblast-mediated stromal barriers, and vascular heterogeneity have been identified as spatial determinants of treatment response. By delineating cellular communities and their interactions, spatial omics technologies can reduce intratumoral complexity into clinically interpretable modules. This review summarizes the diversity of these spatial structures and their relationships with treatment outcomes in immunotherapy, chemotherapy, radiotherapy, and targeted therapy. In addition, it highlights present challenges in data integration, analytical standardization, and functional validation, and discusses future directions for incorporating spatial omics technologies into precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7572818b2484643c196593baace3df887c52839e" target='_blank'>
              Structural cell communities in the tumor microenvironment: Spatial determinants of therapeutic response
              </a>
            </td>
          <td>
            Xuan Cui, Y. Ni, Xia Lei, Lan Zhao, Cheng Qian, Juanjuan Shan
          </td>
          <td>2025-08-14</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Spatial transcriptomics links gene expression with tissue morphology, however, current tools often prioritize genomic analysis, lacking integrated image interpretation. To address this, we present Thor, a comprehensive platform for cell-level analysis of spatial transcriptomics and histological images. Thor employs an anti-shrinking Markov diffusion method to infer single-cell spatial transcriptome from spot-level data, effectively combining gene expression and cell morphology. The platform includes 10 modular tools for genomic and image-based analysis, and is paired with Mjolnir, a web-based interface for interactive exploration of gigapixel images. Thor is validated on simulated data and multiple spatial platforms (ISH, MERFISH, Xenium, Stereo-seq). Thor characterizes regenerative signatures in heart failure, screens breast cancer hallmarks, resolves fine layers in mouse olfactory bulb, and annotates fibrotic heart tissue. In high-resolution Visium HD data, it enhances spatial gene patterns aligned with histology. By bridging transcriptomic and histological analysis, Thor enables holistic tissue interpretation in spatial biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9035239a51ee30cf1cc4c50f6531e17ae4f012f8" target='_blank'>
              Thor: a platform for cell-level investigation of spatial transcriptomics and histology
              </a>
            </td>
          <td>
            Pengzhi Zhang, Weiqing Chen, Tu N Tran, Minghao Zhou, Kaylee N Carter, Ibrahem Kandel, Shengyu Li, Xen Ping Hoi, Yuxing Sun, Li Lai, Keith A Youker, Qianqian Song, Yu Yang, Fotis Nikolos, Zejuan Li, K. S. Chan, John P Cooke, Guangyu Wang
          </td>
          <td>2025-08-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality, with tumor heterogeneity and drug resistance posing significant challenges to treatment. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq deconvolution to analyze BRCA samples. Our analysis identified 15 major cell clusters, including neoplastic epithelial, immune, stromal, and endothelial populations. Notably, low-grade tumors showed enriched subtypes, such as CXCR4+ fibroblasts, IGKC+ myeloid cells, and CLU+ endothelial cells, with distinct spatial localization and immune-modulatory functions. These subtypes were paradoxically linked to reduced immunotherapy responsiveness, despite their association with favorable clinical features. High-grade tumors exhibited reprogrammed intercellular communication, with expanded MDK and Galectin signaling. Bulk RNA-seq deconvolution further supported the prognostic significance of low-grade-enriched subtypes. Our findings highlight the heterogeneity of the tumor microenvironment and provide new insights into immune evasion and therapeutic resistance in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57925849e8c34466fe3132e9855985812e402540" target='_blank'>
              Machine learning analysis reveals tumor heterogeneity and stromal-immune niches in breast cancer
              </a>
            </td>
          <td>
            Junjie Kuang, Guofang Zhong, Linfeng Zhao, Xia Yuan, Yundong Zhou, Jun Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cell-in-cell (CIC) structures, in which one cell is entirely engulfed by another, have been associated with poor outcomes in cancers. However, the mechanisms underlying this association remain poorly understood. We performed multiplex imaging of 56 cell identity, cell ‘state’ and cancer ‘hallmark’ proteins to characterise CICs, map their spatial interactions, and assess clinical associations across 444 tumour cores from 148 colorectal cancer patients, which contained over one million spatially resolved cells. We found that tumour regions containing CICs were associated with lower levels of cytotoxic T cells. We identified upregulated glucose metabolism as a consistent metabolic hallmark of CICs independent of cell type. Spatial analyses revealed that T cells adjacent to CICs underwent selective remodeling with distinct apoptotic and metabolic signatures. Finally, the presence of T cells within CIC neighbourhoods identified a subset of patients with improved survival. Our findings suggest that CICs may be a feature of metabolically competitive niches and a potential factor contributing to T-cell exclusion in tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bc73acfaddb58e731cdec2701f17feb2d562ab2" target='_blank'>
              Multiplex analysis of colorectal cancer tissue describes the composition, cell biology and spatial effects of cell-in-cell events and identifies a T cell-dependent prognostic signature
              </a>
            </td>
          <td>
            E. Bozkurt, Batuhan Kısakol, Mohammadreza Azimi, H. Dussmann, Sanghee Cho, E. McDonough, Joana Fay, Tony O’Grady, John P Burke, N. McCawley, Deborah A. McNamara, Daniel B. Longley, Fiona Ginty, Jochen H. M. Prehn
          </td>
          <td>2025-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary This study explores how combined PET/MRI imaging can track early biological changes within HER2-positive breast tumors treated with trastuzumab, an anti-HER2 therapy. A preclinical mouse model and multiple imaging techniques were used to map tumor features like cell density, blood supply, metabolism, and proliferation. Assessing single imaging measures or their changes could not detect treatment effects early on (day 4). However, by clustering imaging data into five distinct “tumor habitats” with unique biological properties, early treatment responses were observed before tumor shrinkage. For example, the hypoxic responding cluster decreased significantly in treated tumors by day 4. The imaging-derived habitats matched well with tissue-based measurements, confirming their biological relevance. Overall, this approach may improve early assessment of treatment effectiveness, support tumor stratification, and advance personalized cancer therapy by capturing subtle tumor microenvironment alterations before structural changes appear.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79e42ecaf90dad52ebaab60bcb03c8cf88648481" target='_blank'>
              Multiparametric Analysis of PET and Quantitative MRI for Identifying Intratumoral Habitats and Characterizing Trastuzumab-Induced Alterations
              </a>
            </td>
          <td>
            Ameer Mansur, Carlos Gallegos, Andrew Burns, Lily Watts, Seth Lee, Patrick Song, Yun Lu, Anna Sorace
          </td>
          <td>2025-07-22</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Background Cutaneous squamous cell carcinoma (cSCC) and basal cell carcinoma (BCC) are the most prevalent types of nonmelanoma skin cancer (NMSC) and exhibit significant inter‐ and intra‐tumor heterogeneity. cSCC has a higher metastatic potential than BCC, accompanied by a considerable mortality rate. However, the detailed mechanisms of tumor evolution in cSCC have not yet been described. Methods We performed single‐cell RNA sequencing (scRNA‐seq) and T cell receptor (TCR) clonal analysis of skin biopsies from five BCCs, three squamous cell carcinomas in situ (SCCIS), and two invasive squamous cell carcinomas (SCC). Independent SCC specimens were used for spatial transcriptomic (ST) analysis using GeoMx Digital Spatial Profiler (DSP). Result Using scRNA‐seq, we analyzed a total of 117,663 cells. We distinguished cancer cells using copy number variation and identified SCC‐specific genes that potentially contribute to tumor progression. Analysis of tumor clones revealed SCC‐specific COL6A1+/ITGA5+ carcinoma cells which produce CXCL16. We also annotated CXCR6+ regulatory T cells (Tregs) which potentially move toward the tumor site by CXCL16, shaping the immunosuppressive TME. ST analysis supported these clones were located at the invasion site of SCC. Conclusion We suggest COL6A1 and ITGA5 promote the invasive and metastatic property of SCC. We also uncovered how SCC recruits Tregs via the CXCL16/CXCR6 axis to create a TME favorable for its survival. These molecules can be used as potential therapeutic targets for treatment of SCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10a0e1cd92069735154742961ad751b08397d911" target='_blank'>
              CXCL16 Producing Tumor Clones Are Shaping Immunosuppressive Microenvironment in Squamous Cell Carcinoma via CXCR6 Regulatory T Cell
              </a>
            </td>
          <td>
            Hyun Seung Choi, Sunyoung Jung, Ki-Myo Kim, Mihyun Lee, Jun Ho Park, Sanha Hwang, Seung-Min Cha, Jade N Young, D. Poplausky, Hyunsung Nam, Nicholas Gulati, Chung-Gyu Park, Hyun Je Kim, Ji-Ung Park
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Repeated oncogenic mutations and polyclonal proliferation are evident in cancers. However, little is known about the polyclonal principles governing the systemic cancerous lineage during immunotherapy. Here, we examine a unique autopsy case of metastatic urothelial carcinoma that exhibits different treatment responses to anti-PD-1 therapy at each tumor site. By performing in-depth analyses of different multiregional bulk tumor masses, we reveal that subsets of subclones acquire potential driver mutations under treatment selection pressure. Spatial transcriptomics analysis reveals that subclones resistant to immunotherapy form distinct immunosuppressive environments consistent with their habitats. Furthermore, different cancer hallmarks are identified in each of the subclones that expand under immunotherapy at single-cell level; for example, one subclone is more proliferative, and another is more stem-cell-like. In summary, this study provides an overall picture of the polyclonal competition and changes in the immune microenvironment that are related to resistance to immunotherapy in patients with malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f357f9c7e57b9e95e8ee48a6378f3d690f3275bd" target='_blank'>
              Clonal diversity shapes the tumour microenvironment leading to distinct immunotherapy responses in metastatic urothelial carcinoma
              </a>
            </td>
          <td>
            Takashi Kamatani, K. Umeda, T. Iwasawa, Fuyuki Miya, K. Matsumoto, S. Mikami, Kensuke Hara, Masayuki Shimoda, Yutaka Suzuki, Jo Nishino, Mamoru Kato, Kazuhiro Kakimi, Nobuyuki Tanaka, M. Oya, T. Tsunoda
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease with diverse outcomes, and intra-tumoral heterogeneity plays a significant role in both diagnosis and treatment. Despite its importance, the spatial distribution of intra-tumoral heterogeneity is not fully elucidated. Spatial transcriptomics has emerged as a promising tool to study the molecular mechanisms behind many diseases. It offers accurate measurements of RNA abundance, providing powerful tools to correlate the morphologies of cellular neighborhoods with localized gene expression patterns. However, the spot-based spatial transcriptomic tools, including the most widely used platform, Visium, do not achieve single-cell resolution readouts, which hinders data interpretability. In this study, we present a computational pathology image analysis pipeline (i.e., computational tissue annotation, CTA) that utilizes machine learning algorithms to accurately map tumor, stroma, and immune compartments within Visium-assayed tumor sections. Using a cohort of 23 breast tumor sections from four patients, we demonstrate that CTA can provide high-resolution annotations on the hematoxylin-and-eosin-stained images alongside the paired sequencing data, support the evaluation of deconvolution methods, deepen insights into intra-tumoral heterogeneity by increasing data analysis resolution, assist with spatially resolved intrinsic subtyping, and enhance the visualization of lymphocyte clones at single-cell resolution. The proposed pipeline provides valuable insights into the complex spatial architecture of breast cancer, contributing to more personalized diagnostics and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5fa100fe1b095f296661cac47143cc93bae15b" target='_blank'>
              Computational pathology annotation enhances the resolution and interpretation of breast cancer spatial transcriptomics data
              </a>
            </td>
          <td>
            Tianyi Li, Qiao Yang, B. Ács, E. Sifakis, Hosein Toosi, Camilla Engblom, K. Thrane, Qirong Lin, Jeff E Mold, Wenwen Sun, Ceren Boyaci, Sanna Steen, Jonas Frisén, Jens Lagergren, J. Lundeberg, Xinsong Chen, Johan Hartman
          </td>
          <td>2025-09-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Tumor heterogeneity in metastatic prostate cancer (mPC) is well established, but comprehensive characterization using routine sampling remains challenging. Autopsy-based research addresses this obstacle by enabling broad tissue collection within individual patients after treatment. In this issue of the JCI, Roudier et al. analyzed samples from a mPC research autopsy cohort, revealing extensive inter- and intratumor heterogeneity across patients and at the cellular level. The authors associated this variability with genomic, phenotypic, and clinical features and explored the importance of tumors expressing both androgen receptor and neuroendocrine markers. Their findings demonstrate heterogeneity across metastatic sites that may influence treatment response and clinical outcomes, informing future therapeutic strategies in mPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1162fb7a103df269563289c6dd21cc5f28aaf11" target='_blank'>
              Uncovering phenotypic heterogeneity through research autopsy in lethal prostate cancer
              </a>
            </td>
          <td>
            Sylvie S.W. Chan, Osvaldas Vainauskas, Gerhardt Attard
          </td>
          <td>2025-08-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The organizational principles of nephronal segments are based on anatomical and physiological attributes that are linked to the homeostatic functions of the kidney. Recent molecular approaches have uncovered layers of deeper signatures and states in tubular cells that arise at various time points on the disease trajectory. Here, we introduce an analytical pipeline of multiplexed spatial protein imaging integrated with RNA expression to characterize proximal tubular subpopulations and neighborhoods in human kidney tissue. We demonstrate that, in reference tissue, a large proportion of S1 proximal tubular epithelial cells expresses thymus antigen 1 (THY1), a mesenchymal stromal and stem cell marker that regulates differentiation. Kidney disease is associated with loss of THY1 and transition toward expression of prominin 1 (PROM1), another stem cell marker recently linked to failed repair. Our data support a model in which the interplay between THY1 and PROM1 expression in proximal tubules associates with their regenerative potential and marks the timeline of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1923d134d598f27cb8990ba3881ccadfe262fa12" target='_blank'>
              Integration of spatial protein imaging and transcriptomics in the human kidney tracks the regenerative potential of proximal tubules
              </a>
            </td>
          <td>
            Mahla Asghari, Angela R. Sabo, Daria Barwinska, Ricardo Melo Ferreira, M. Ferkowicz, William S. Bowen, Ying-Hua Cheng, D. Gisch, Connor J. Gulbronson, Carrie L. Phillips, Katherine J. Kelly, T. Sutton, James C Williams, Miguel A Vazquez, John O'Toole, Paul M. Palevsky, Sylvia E. Rosas, S. Waikar, Krzysztof Kiryluk, C. Parikh, Jeffrey B. Hodgin, Pinaki Sarder, Ian H. de Boer, Jonathan Himmelfarb, Matthias Kretzler, Sanjay Jain, Michael T. Eadon, S. Winfree, T. El-Achkar, P. Dagher
          </td>
          <td>2025-08-15</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Background Lymph node metastasis markedly worsens prognosis in lung adenocarcinoma (LUAD); however, the evolutionary dynamics and regulatory mechanisms underlying the heterogeneity of malignant epithelial cells during this process remain poorly understood and warrant comprehensive investigation. Methods We performed a comprehensive single-cell transcriptomic analysis of epithelial cells from 18 samples comprising normal lung tissue and lymph node metastases. Malignant epithelial cells were identified via inferred copy number variation (CNV) profiles. Key malignant subpopulations were further characterized through trajectory inference, cell–cell communication mapping, gene set variation analysis (GSVA), and reconstruction of transcription factor regulatory networks. To assess clinical relevance, we developed and validated a prognostic model—termed the EAS score—based on the transcriptional signatures of malignant epithelial subsets, using integrated data from multiple TCGA and GEO cohorts. The functional role of the hub gene SELENBP1 was experimentally validated through quantitative PCR (qPCR), Western blotting, immunohistochemistry (IHC), Transwell migration assays, colony formation assays, flow cytometry, ROS quantification, and subcutaneous tumorigenesis assays in vivo. Results Single-cell transcriptomic analysis identified four distinct malignant epithelial subtypes (Clusters 0–3), each characterized by unique patterns of CNV. Leveraging these defined cellular subpopulations, we constructed a highly accurate model for prognostication in LUAD, enabling reliable classification of patients based on clinical outcomes. Through detailed comparisons between groups with divergent prognostic risks, the study revealed notable differences across the tumor microenvironment (TME), including alterations in pathway activity, gene enrichment distributions, mutation profiles, and anticipated responses to immune checkpoint blockade. In addition, functional validation experiments confirmed that SELENBP1 plays a tumor-suppressive role, further supporting its relevance as a potential intervention target in LUAD. Conclusion This research provides insights into the evolutionary complexity and heterogeneity of malignant epithelial populations in lymph node metastatic sites of LUAD. It also presents a scoring system based on prognostic indicators, which serves as a reliable tool for forecasting patient survival outcomes. Moreover, the discovery of SELENBP1 as a candidate tumor suppressor emphasizes its importance in guiding both clinical risk categorization and the design of personalized treatment strategies for individuals classified as high-risk LUAD cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6a2796c042204c38d000db3a271f2db16f9cb8" target='_blank'>
              Single-cell transcriptomic profiling reveals the heterogeneity of epithelial cells in lung adenocarcinoma lymph node metastasis and develops a prognostic signature
              </a>
            </td>
          <td>
            Qiuqiao Mu, Han Zhang, Ying Shi, Mengli Xue, Jingxian Wang, Yun Ding, Lin Tan, Hui Yuan, Xin Li, Daqiang Sun
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cutaneous malignant melanoma is an extraordinarily aggressive and heterogeneous cancer that contains a small subpopulation of tumor stem cells (CSCs) responsible for tumor initiation, metastasis, and recurrence. Identification and characterization of CSCs in melanoma is challenging due to tumor heterogeneity and the lack of specific markers (CD271, ABCB5, ALDH, Nanog) and the ability of cells to dynamically change their phenotype. Phenotype-maintaining signaling pathways (Wnt/β-catenin, Notch, Hedgehog, HIF-1) promote self-renewal, treatment resistance, and epithelial–mesenchymal transitions. Tumor plasticity reflects the ability of differentiated cells to acquire stem-like traits and phenotypic flexibility under stress conditions. The interaction of CSCs with the tumor microenvironment accelerates disease progression: they induce the formation of cancer-associated fibroblasts (CAFs) and neo-angiogenesis, extracellular matrix remodeling, and recruitment of immunosuppressive cells, facilitating immune evasion. Emerging therapeutic strategies include immunotherapy (immune checkpoint inhibitors), epigenetic inhibitors, and nanotechnologies (targeted nanoparticles) for delivery of chemotherapeutic agents. Understanding the role of CSCs and tumor plasticity paves the way for more effective innovative therapies against melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624525e56a22d9cf1d979cf203e09ae4124349ea" target='_blank'>
              Cancer Stem Cells in Melanoma: Drivers of Tumor Plasticity and Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Adriana Sabău, A. Tinca, Raluca Niculescu, I. Cocuz, Andreea Raluca Cozac-Szöke, B. Lazar, D. Chiorean, C. Budin, O. S. Cotoi
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dce7c8d7d28d1cfa3f77ec4f74044ee737c8a1da" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment in archived FFPE samples unveils novel paracrine interactions.
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlsson, A. Claeys, E. Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2025-08-08</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Spatially mapping the cellular positions and their microenvironments with spatial transcriptomics (ST) shows great potential to illustrate key factors and mechanisms driving complex tissue organizations. The spatial data require specialized handling with different statistical and inferential considerations. Here, we develop SPECTRUM (Spatial Pattern Enhanced Cellular and Tissue Recognition Unified Method), which combines inclusive prior known cell-type-specific markers and spatial weighting for cell-type identification and spatial community detection. Comprehensive benchmarks demonstrate the superior performance of SPECTRUM. Applying SPECTRUM on real ST datasets with various spatial patterns demonstrates its capability in correctly mapping region-specific cell types and functional spatial communities. With that, we uncovered that context-dependent communication supports the functional plasticity of cells in spatial communities in human limb development. In summary, SPECTRUM is a unified tool for ST data analysis that deepens our insights into spatial organization at molecular, cellular, and community levels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c9b8b9ddfe632ccb15127d057b72c71a1262dac" target='_blank'>
              Spatial pattern enhanced cellular and tissue recognition for spatial transcriptomics
              </a>
            </td>
          <td>
            Yucen Wang, Zhuoyu Zhang, Guoqiang Li
          </td>
          <td>2025-07-04</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Summary The organization of diverse mesenchymal populations during human intestinal development is critical for tissue architecture and function yet remains poorly defined. To construct a comprehensive, tissue-scale map of the developing human small intestine, we leveraged single-cell RNA-sequencing data to build a custom Xenium spatial transcriptomics gene panel covering the diversity of cell types in the human intestine. Analysis was focused on the developing mesenchyme populations (also referred to as fibroblasts or stroma) given the lack of spatiotemporal information about these cell populations. We defined 5 broad mesenchymal populations occupying discrete anatomical locations within the lamina propria and submucosa – the subepithelial cells (SEC), lamina propria fibroblasts (LPF), submucosal fibroblasts (SMF), smooth muscle cells (SMC), and CXCL13+ fibroblasts. Our data reveal dynamic spatial remodeling of fibroblast communities during development and establish molecular markers to distinguish these populations. We leverage this high-resolution atlas to benchmark pluripotent stem cell-derived human intestinal organoids and to demonstrate how this foundational resource can be used to dissect intestinal stromal signaling in a spatial manner, with broad implications for modeling development, regeneration, and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5302016abe1b00dedbc64fd49546e523b7362240" target='_blank'>
              Mapping mesenchymal diversity in the developing human intestine and organoids
              </a>
            </td>
          <td>
            Kelli F. Johnson, Xiangning Dong, Yu-Hwai Tsai, Angeline Wu, Sydney G Clark, Sha Huang, Rachel Zwick, Ian A. Glass, Katherine D Walton, Ophir D. Klein, Jason R. Spence
          </td>
          <td>2025-07-22</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="High-grade serous carcinoma (HGSC) is the most common ovarian cancer subtype, typically diagnosed at late stages with poor prognosis. Understanding early molecular events driving HGSC progression is crucial for timely detection and development of effective treatment strategies. We performed and integrated spatial cell-type resolved proteomics and paired transcriptomics across 25 women with precursor lesions of the fallopian tube and/or HGSC. Epithelial cell signatures revealed early activation of SUMOylation machinery, increased ATR and Wnt signaling, and enhanced MHC-I antigen presentation along the disease trajectory. The stroma exhibited extracellular matrix remodeling and interferon-mediated inflammation. Serous tubal intraepithelial carcinomas (STICs) in cancer patients contained a pro-coagulative signature and reduced APOA1/2 compared to STICs in individuals without cancer. We functionally established important roles of epithelial-derived TRIP13 and SUMOylation, and cancer-associated fibroblast-derived SULF1 and BGN in HGSC progression. These findings provide unique molecular insights into HGSC pathogenesis and identify potential new therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44488614a89cae7d4f29ffc136362eb7dd32e0db" target='_blank'>
              Integration of cell-type resolved spatial proteomics and transcriptomics reveals novel mechanisms in early ovarian cancer
              </a>
            </td>
          <td>
            Andreas Metousis, Hilary A Kenny, Aasa Shimizu, L. Schweizer, S. Ben-Moshe, A. Bilecz, Rahul Krishnan, Jingwen Zhang, Isabel Alcazar, Lucy Kelliher, Mallika Ravi, Tejas Samantaray, Sabrina Richter, Yan Li, Jiying Wang, Sophia Steigerwald, F. Theis, Florian A. Rosenberger, T. Nordmann, S. Yamada, Ricardo R. Lastra, M. Mann, E. Lengyel
          </td>
          <td>2025-08-28</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the third most common cancer in the world. Within the tumor microenvironment (TME), cancer cells are surrounded by diverse cell types including cancer-associated fibroblasts (CAFs) and immune cells. These neighboring cells can have a profound effect on prognosis and disease progression by providing local signals that impact tumor growth and metastasis. Prior work in the lab has identified via single cell RNA sequencing novel subtypes of cancer epithelial cells and enrichment of specific CAFs within the CRC TME. However, the locations and signaling interactions of these cells within the tumors remain poorly understood due to the loss of spatial information. In this study, we adopt a multi-facet approach to try to understand the CRC TME by utilizing a combination of single cell RNA sequencing and spatial omics technology. We first comprehensively characterized all the major cell types present in the CRC TME using single cell RNA sequencing, and with the knowledge gained, we set out to investigate the spatial localization of these cell types of interests and their interactions within the tumor architecture using the 10x Xenium Analyzer and merFISH. From our data, we identified subtypes of tumor epithelial cells that are uniquely enriched in CRC patients and observed distinct spatial organization of these cells within colonic glands across patient subgroups. In addition, we observed an enrichment of CAFs and immune cell populations at the tumor invasive margin, often accompanying regions of tumor budding. Further study of these tumor epithelial, CAF and immune subtypes and their spatial interactions may reveal new insights into CRC tumor biology and assist in the development of targeted therapies for CRC.



 Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, Nirmala Arul Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar. Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2896c139d399fbdf03bf5e11753408c9bfebdc83" target='_blank'>
              Abstract P19: Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics
              </a>
            </td>
          <td>
            Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, N. A. Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Charting the spatiotemporal dynamics of cell fate determination in development and disease is a long-standing objective in biology. Here we present the design, development, and extensive validation of PEtracer, a prime editing-based, evolving lineage tracing technology compatible with both single-cell sequencing and multimodal imaging methodologies to jointly profile cell state and lineage in dissociated cells or while preserving cellular context in tissues with high spatial resolution. Using PEtracer coupled with MERFISH spatial transcriptomic profiling in a syngeneic mouse model of tumor metastasis, we reconstruct the growth of individually-seeded tumors in vivo and uncover distinct modules of cell-intrinsic and cell-extrinsic factors that coordinate tumor growth. More generally, PEtracer enables systematic characterization of cell state and lineage relationships in intact tissues over biologically-relevant temporal and spatial scales.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18c22f75fc7e1ad65362ee0eac19d559b599a24d" target='_blank'>
              High-resolution spatial mapping of cell state and lineage dynamics in vivo with PEtracer.
              </a>
            </td>
          <td>
            Luke W. Koblan, Kathryn E. Yost, Pu Zheng, William N. Colgan, Matthew G Jones, Dian Yang, Arhan Kumar, Jaspreet Sandhu, Alexandra Schnell, Dawei Sun, Can Ergen, R. Saunders, Xiaowei Zhuang, William E. Allen, N. Yosef, Jonathan S. Weissman
          </td>
          <td>2025-07-24</td>
          <td>Science</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Hepatoblastoma is the most common pediatric liver cancer, with the fastest rising incidence among childhood malignancies. Early genomic studies revealed that hepatoblastoma has the lowest mutational burden of any human cancer, however, recent advances in single-cell RNA-seq, multiomics, spatial transcriptomics, and functional genomics screenings have revealed substantial complexity. Diverse cellular subpopulations, divergent WNT signaling, key developmental pathways, and intricate interactions between the tumor cells and tumor immune microenvironment (TME) collectively shape tumor heterogeneity, disease progression, therapeutic responses, and genetic dependencies. Mosaic embryonic loss of heterozygosity (LOH) at chromosome 11p15.5 may be a hepatoblastoma-initiating event, as clonal expansion of 11p15.5 LOH occurs in adjacent normal liver tissue. A cholangiocyte-like subpopulation expresses FGF19, in a SOX4-dependent, paracrine manner, to drive the proliferation of neighboring embryonal hepatoblastoma cells. WNT-signaling dependent MDK promotes the immunosuppressive TME, which impairs immune cell infiltration. The TME may also be driven by islands of erythroblasts, which influence treatment resistance. Plasticity driven by changes in chromatin accessibility enables differentiation transition between hepatocytic and liver progenitor cell types, which is associated with treatment resistance. Here, we review recent findings in pediatric hepatoblastoma cells, tumor-associated cell types, and genetic dependencies that will serve to advance hepatoblastoma therapy. Clinical trial number: Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02405-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361b9ebd3fa588c3f28e25ba36c91c45f4b07fb6" target='_blank'>
              From preneoplastic lesion to heterogenous tumor: recent insights into hepatoblastoma biology and therapeutic opportunities
              </a>
            </td>
          <td>
            Jun Yang, A. Davidoff, Andrew J. Murphy
          </td>
          <td>2025-07-19</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinoma (cSCC), basal cell carcinoma (BCC), and melanoma, the three major types of skin cancer, account for over 70% of all cancer cases. Despite their prevalence, the skin cancer microenvironment remains poorly characterized, both in the outer skin layer where the cancer originates and at the deeper junctional and dermal layers into which it progresses. To address this, we integrated 12 complementary spatial single-cell technologies to construct orthogonally-validated cell signatures, spatial maps, and interactomes for cSCC, BCC, and melanoma. We comprehensively compared and integrated these spatial methods and provided practical guidelines on experimental design. Integrating four spatial transcriptomics platforms, we found keratinocyte cancer signatures, including six consistently validated gene markers. Spatial integration of transcriptomics, proteomics, and glycomics uncovered cancer communities enriched in melanocyte–fibroblast–T-cell colocalization with altered tyrosine and pyrimidine metabolism. Ligand-receptor analysis across >700 cell-type combinations and >1.5 million interactions highlighted key roles for CD44, integrins, and collagens, with CD44-FGF2 emerging as a potential therapeutic target. We consistently found differential interactions of melanocytes with fibroblasts and T-cells. We validated these interactions using Opal Polaris, RNAScope, and Proximal Ligation Assay. To integrate population-scale data, genetic association mapping in >500,000 individuals suggested SNPs enriched for spatial domains containing melanocytes, dysplastic keratinocytes, and fibroblasts, shedding light on functional mechanisms linking genetic heritability to cells within cancer tissue. This publicly available multiomics resource offers insights into the initiation and progression of the most lethal skin cancer (melanoma) and the most common forms (cSCC and BCC) and can be explored interactively at https://skincanceratlas.com.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87a9509055032cdb5b434f194c71651a4b05c7db" target='_blank'>
              Integrating 12 Spatial and Single Cell Technologies to Characterise Tumour Neighbourhoods and Cellular Interactions in three Skin Cancer Types
              </a>
            </td>
          <td>
            P. Prakrithi, Laura F Grice, Feng Zhang, Levi Hockey, Samuel X. Tan, Xiao Tan, Zherui Xiong, Onkar Mulay, Andrew Causer, Andrew Newman, Duy Pham, Guiyan Ni, Kelvin Tuong, Xinnan Jin, Eunju Kim, Minh Tran, Hani Vu, Nicholas M Muller, E. Killingbeck, M. Gregory, S. Teoh, Tuan Vo, Min Zhang, M. T. Landi, Kevin M. Brown, M. Iles, Zachary Reitz, K. Devitt, Liuliu Pan, A. Kulasinghe, Y. Kao, Michael Leon, Sarah Murphy, Hiromi Sato, J. G. Cruz, Snehlata Kumari, H. N. Luu, Sarah Warren, Chris L. D. McMillan, J. Henricson, Chris Anderson, David A Muller, Arun Everest-Dass, Blake O’Brien, M. Seviiri, Matthew H. Law, H. P. Soyer, Ian H. Frazer, Youngmi Kim, Mitchell S. Stark, Kiarash Khosrotehrani, Quan Nguyen
          </td>
          <td>2025-07-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296f18d57dd6ad5d3ff3f75155cdc24fcd7506bf" target='_blank'>
              Cross-species analysis identifies genotype-driven vulnerabilities in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Angeliki Karatza, Maayan Pour, Jaylen M Powell, Yuan Hao, Dimitris Nasias, Yi Jer Tan, Alfonso Lopez, M. Ciampricotti, Chengwei Peng, Álvaro Quintanal Villalonga, Yeuan Ting Lee, Ruveyda Ayasun, Muhammad Elnaggar, Priyanka Sahu, Igor Dolgalev, Hai Hu, Luis Chiriboga, Cynthia Loomis, Daniel Andrussier, Magdalena A Ploszaj, A. Tsirigos, Peter Meyn, Fred P. Davis, Nicolas Stransky, Gromoslaw A. Smolen, John T. Poirier, Jonathan So, Mark Philips, Harvey Pass, Dafna Bar Sagi, Itai Yanai, 
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most aggressive and treatment-resistant brain tumors, largely due to its profound intratumoral heterogeneity and immunosuppressive microenvironment. Various classifications of GBM subtypes were created based on transcriptional and methylation profiles. This effort, followed by the development of new technology such as single-nuclei sequencing (snRNAseq) and spatial transcriptomics, led to a better understanding of the glioma cells’ plasticity and their ability to transition between diverse cellular states. GBM cells can mimic neurodevelopmental programs to resemble oligodendrocyte or neural progenitor behavior and hitchhike the local neuronal network to support their growth. The tumor microenvironment, especially under hypoxic conditions, drives the tumor cell clonal selection, which then reshapes the immune cells’ functions. These adaptations contribute to immune evasion by progressively disabling T cell and myeloid cell functions, ultimately establishing a highly immunosuppressive tumor milieu. This complex and metabolically constrained environment poses a major barrier to effective antitumor immunity and limits the success of conventional therapies. Understanding the dynamic interactions between glioma cells and their microenvironment is essential for the development of more effective immunotherapies and rational combination strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0dda0986360e9c65e565d1dc755ad16cfd3932d" target='_blank'>
              Unraveling Glioblastoma Heterogeneity: Advancing Immunological Insights and Therapeutic Innovations
              </a>
            </td>
          <td>
            Joshua H. Liu, Maksym Horiachok, Santosh Guru, Cecile L. Maire
          </td>
          <td>2025-08-01</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) remains a lethal malignancy. Although immunochemotherapy regimens have improved patient survival rates, drug resistance still occurs in a significant subset of patients, highlighting the importance of elucidating the mechanisms within the tumor microenvironment. Here, we applied spatial single-cell transcriptomics to investigate the spatial characteristics of SCLC and their associations with immunochemotherapy resistance. By analyzing samples from 18 patients with extensive-stage SCLC, we identified two distinct epithelial cell subtypes: Epi-I and Epi-II. Epi-I exhibited high proliferative activity and was associated with treatment resistance and poor survival outcomes. In contrast, Epi-II showed more spatial contact with immune cells and was associated with treatment sensitivity. Further analysis uncovered a fascinating cellular transition paradigm, wherein Epi-I may be derived from Epi-II, with myeloid cells playing a facilitatory role in this transformation cascade. Specifically, within the spatial zone that was enriched with the Epi-II, epithelial cells may secrete MIF gene, which promoted the polarization of myeloid cells towards the M2 macrophages. The M2-polarized myeloid cells subsequently upregulated the expression of SPP1 that in turn triggered the activation of the PI3K-AKT signaling pathway in the adjacent epithelial cells, driving the conversion of Epi-II to Epi-I cells. Our findings revealed that the intricate crosstalk between epithelial and myeloid cells constitutes a pivotal resistance mechanism in SCLC, and targeting the SPP1/MIF pathway emerged as a promising strategy with the potential to enhance the treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0302ec6123f8cd6d62c8c99e2e99dcc7087718b" target='_blank'>
              Spatial transcriptomics reveals macrophage domestication by epithelial cells promotes immunotherapy resistance in small cell lung cancer
              </a>
            </td>
          <td>
            Yu Sun, Minghui Zhang, Yanbin Zhao, Yubo Yan, Lei Wang, Xuhui Liu, Songsong Xia, Bingbing Wang, Xiaoxin Zhang, Yan Wang
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is one of the most aggressive types of brain cancer, known for spreading quickly into nearby healthy brain tissue. This makes it challenging to treat, and therefore, patient survival is extremely low. To better understand how this cancer spreads and to find more effective treatments, researchers use animal models that closely mimic how the disease behaves in humans. This review focuses on how these models help to study the movement and invasion of cancer cells in the brain. It also highlights how advanced imaging tools allow researchers to watch tumor growth in real time. By comparing animal models with lab-based systems, this review shows how animal models offer unique insights that can lead to better therapies. This knowledge could help bridge the gap between basic science and real-world treatments for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3d2bea10beea9c35e13ee76767026c6d58245d" target='_blank'>
              Choice of Animal Models to Investigate Cell Migration and Invasion in Glioblastoma
              </a>
            </td>
          <td>
            Piyanka Hettiarachchi, Taeju Park
          </td>
          <td>2025-08-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Natural killer cells (NK cells) are important immune cells within the tumor microenvironment (TME) and are considered the first line of defense in tumor immunity. Although many studies have focused on the role of NK cells in tumor therapy, the heterogeneity of NK cells complicates the investigation of the complex mechanisms within the tumor microenvironment. Single-cell sequencing technology, with its high-resolution capability, reveals the gene expression profiles of individual NK cells, highlighting their heterogeneity and providing more accurate information for NK cell therapy. This article begins by addressing the mechanisms underlying the formation of NK cell heterogeneity, emphasizing the significance of differentiation, development, and tissue residency in establishing this heterogeneity. It also summarizes the advances in the study of NK cell heterogeneity under physiological conditions and in tumor environments using single-cell sequencing technology. Finally, it analyzes the dynamic changes of NK cells within the tumor microenvironment under various therapeutic approaches to explore drug effects and resistance mechanisms, as well as to optimize therapeutic options. Investigating the mechanisms of tumor progression and drug intervention at the single-cell level will provide new perspectives for personalized treatment strategies centered around NK cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60507aa9211c6d7d18d4e28dd353dc3d06e0a4a9" target='_blank'>
              Unraveling NK cell heterogeneity through single-cell sequencing: insights from physiological and tumor contexts for clinical applications
              </a>
            </td>
          <td>
            Mingxin Shen, Yutong Liu, Liang Shao, Meng Qu, Shixin Song, Wei Sun, Hao Zhang
          </td>
          <td>2025-07-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Spatial transcriptomics (ST) has emerged as an efficient technology for mapping gene expression within tissue sections, offering informative spatial context for gene activities. However, most current ST techniques suffer from low spatial resolution, where each spatial location often contains cells of various types. Deconvolution methods are used to resolve the cell mixture within the spots, but conventional approaches rely on spot-by-spot analyses, which are limited by low gene expression levels and disregard spatial relationships between spots, ultimately reducing performance. Here, we introduce DECLUST, a cluster-based deconvolution method to accurately estimate the cell-type composition in ST data. The method identifies spatial clusters of spots using both gene expression and spatial coordinates, hence preserving the spatial structure of the tissue. Deconvolution is subsequently performed on the aggregated gene expression of individual clusters, mitigating the challenges associated with low expression levels in individual spots. We evaluate DECLUST on simulated ST datasets from a human breast cancer tissue and two real ST datasets from human ovarian cancer and mouse brain. We compare DECLUST to current methods including CARD, GraphST, Cell2location, and Tangram. The results indicate that DECLUST not only maintains the spatial integrity of tissues but also outperforms existing methods in terms of robustness and accuracy. In conclusion, DECLUST provides an effective and reliable approach for identifying cell-type compositions in ST data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a849f6608a52c27ce54815b06c38254be7c33009" target='_blank'>
              A cluster-based cell-type deconvolution of spatial transcriptomic data
              </a>
            </td>
          <td>
            Qingyue Wang, Parth Khatri, Huy Q. Dinh, Jian Huang, Yudi Pawitan, T. Vu
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary Brain metastasis is a serious and often early complication in lung adenocarcinoma, the most common type of lung cancer. While many studies have examined treated or advanced cases, little is known about the tumor environment at initial diagnosis. This study analyzed tissue from five patients whose brain metastases were the first signs of lung adenocarcinoma. Using spatial proteomic profiling, we compared protein expression between matched brain and lung tumors. Brain metastases showed reduced immune activity and increased markers of cell growth and immune evasion. These findings highlight the distinct biology of brain lesions even before treatment begins and suggest early immune escape. Understanding these early changes may support future efforts to develop site-specific strategies targeting brain metastases at initial presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b71912171aef6f8867b2853c36b6318ebf2376ae" target='_blank'>
              Spatial Omics Profiling of Treatment-Naïve Lung Adenocarcinoma with Brain Metastasis as the Initial Presentation
              </a>
            </td>
          <td>
            Seoyeon Gwon, Inju Cho, Jieun Lee, Seung Yun Lee, Kyue-Hee Choi, Taejung Kim
          </td>
          <td>2025-07-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The intrinsic complexity of biological processes often hides the role of dynamic microenvironmental cues in the development of pathological states. Microphysiological systems (MPSs) are emerging technological platforms that model in vitro dynamics of tissue-specific microenvironments, enabling a holistic understanding of pathophysiology. In our previous works, we engineered and used breast tumor MPS differing in matrix stiffness, pH, and fluid flow mimicking normal and tumor breast tissue. High-dimensional data using two distinctive human breast cell lines (i.e., MDA-MB-231, MCF-7), investigating cell proliferation, epithelial-to-mesenchymal transition (EMT), and breast cancer stem cell markers (B-CSC), were obtained from breast-specific microenvironments. Recognizing that the widespread adoption of MPS requires tailoring its complexity to application demands, we herein report an innovative machine-learning (ML)-based approach to analyze MPS data. This approach uses unsupervised k-means clustering and feature extraction to inform on key markers and specific microenvironments that distinguish invasive from non-invasive breast cell phenotypes. This data-driven approach streamlines future experimental design and emphasizes the translational potential of integrating MPS-derived insights with ML to refine prognostic tools and personalize therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d04f6b70ba36ca6bdecf1ece254a7da2cd8bdc7c" target='_blank'>
              Data-Driven Sustainable In Vitro Campaigns to Decipher Invasive Breast Cancer Features
              </a>
            </td>
          <td>
            Lekha Shah, V. Breschi, Annalisa Tirella
          </td>
          <td>2025-07-25</td>
          <td>ACS Biomaterials Science & Engineering</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of breast cancer is a complex ecosystem, in which cancer-associated fibroblasts (CAFs), as the most abundant stromal cell type, meticulously construct an ecological niche that supports tumor growth through mechanisms including extracellular matrix (ECM) remodeling, secretion of bioactive factors, and interactions with neighboring cells. High-resolution technologies, including single-cell sequencing and spatial transcriptomics, have revealed the high heterogeneity, functional diversity, and spatial distribution within the CAF population. Significant differences exist in the interactions between distinct CAF subpopulations and immune cells. Through complex crosstalk with the immune system, they collaboratively establish an immunosuppressive network, becoming a core driving force for tumor immune escape. This review focuses on the latest research advances in heterogeneous subpopulations of CAFs within the breast cancer microenvironment, delves into how the complex bidirectional crosstalk between different CAF subpopulations and immune cells collaboratively shapes the tumor immune microenvironment (TIME), and summarizes various CAF-based therapeutic strategies for breast cancer, aiming to provide critical theoretical basis and novel therapeutic perspectives for the clinical translation of CAF heterogeneity research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e74899699742a2ae95fa97ca3627d6bdb366035" target='_blank'>
              Crosstalk between heterogeneous cancer-associated fibroblast subpopulations and the immune system in breast cancer: key players and promising therapeutic targets
              </a>
            </td>
          <td>
            Lu Liu, Weijun Wan, Yilin Chang, Luoquan Ao, Yan Xu, Xiang Xu
          </td>
          <td>2025-09-01</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="BACKGROUND
T-cell activation and clonal expansion are essential to effective immunotherapy responses in non-small cell lung cancer (NSCLC). The distribution of T-cell clones may offer insights into immunogenic mechanisms and imply potential prognostic and predictive information.


METHODS
We analyzed α/β T-cell receptor (TCR) clonality using RNA-sequencing of bulk frozen tumor tissue from 182 patients with NSCLC. The data was integrated with molecular and clinical characteristics, extensive in situ imaging, and spatial sequencing of the tumor immune microenvironment. TCR clonality was also determined in an independent cohort of nine patients with immune checkpoint-treated NSCLC.


RESULTS
TCR clonality (Gini index) patterns ranged from high T-cell clone diversity with high evenness (low Gini index) to clonal dominance with low evenness (high Gini index). Generally, TCR clonality in cancer was lower than in matched normal lung parenchyma distant from the tumor (p=0.021). The TCR clonality distribution between adenocarcinoma and squamous cell carcinoma was similar; however, smokers showed a higher Gini index. While in the operated patient with NSCLC cohort, TCR clonality was not prognostic, in an immune checkpoint inhibitor-treated cohort, high TCR clonality was associated with better therapy response (p=0.016) and prolonged survival (p=0.003, median survival 13.8 vs 2.9 months). On the genomic level, a higher Gini index correlated strongly with a lower frequency of epidermal growth factor receptor (EGFR) and adenomatous polypsis coli (APC) gene mutations, but a higher frequency of P53 mutations, and a higher tumor mutation burden. In-depth characterization of the tumor tissue revealed that high TCR clonality was associated with an activated, inflamed tumor phenotype (PRF1, GZMA, GZMB, INFG) with exhaustion signatures (LAG3, TIGIT, IDO1, PD-1, PD-L1). Correspondingly, PD-1+, CD3+, CD8A+, CD163+, and CD138+immune cells infiltrated cancer tissue with high TCR clonality. In situ sequencing recovered single dominant T-cell clones within the patient tumor tissue, which were predominantly of the CD8 subtype and localized closer to tumor cells.


CONCLUSION
Our robust analysis pipeline characterized diverse TCR repertoires linked to distinct genotypes and immunologic tumor phenotypes. The spatial clustering of expanded T-cell clones and their association with immunological activation underscores a functional, clinically relevant immune response, particularly in patients with NSCLC treated with checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10cb95082ff80184987c55803cfe4d35d428a707" target='_blank'>
              Spatial TCR clonality and clonal expansion in the in situ microenvironment of non-small cell lung cancer.
              </a>
            </td>
          <td>
            Hui Yu, Anastasia Magoulopoulou, Rose-Marie Amini, Maria Chatzinikolaou, Masafumi Horie, A. Lindberg, Artur Mezheyeuski, M. Backman, Andreas Metousis, H. Brunnström, M. Marincevic, J. Botling, J. Mattsson, K. Kärre, Karin Leandersson, Mats Nilsson, Carina Strell, P. Micke
          </td>
          <td>2025-08-01</td>
          <td>Journal for immunotherapy of cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background Neoadjuvant therapy (NAT) has transformed cancer treatment by improving surgical outcomes and survival rates, yet resistance mechanisms across multiple cancer types remain unclear. This study aimed to decipher tumor ecosystem dynamics during NAT using cross-cancer single-cell sequencing data, focusing on identifying key mediators of immunosuppression and treatment resistance. Methods Single-cell RNA-sequencing (scRNA-seq) datasets from five solid tumors (esophageal squamous cell carcinoma, esophagogastric junction carcinoma, colorectal cancer, cervical cancer, and triple-negative breast cancer) were integrated. The data from these five cancer types underwent a rigorous process to standardize cell types across all datasets. Cell-cell communication analysis, Meta-Programs (MPs) via non-negative matrix factorization, and functional enrichment were performed. Immunohistochemistry (IHC) and Western blot validated S100A4 expression and PD-L1 induction in vitro. Results We constructed a single-cell map across cancer types and systematically characterized dynamic changes in tumor cells and diverse microenvironmental cell populations following neoadjuvant therapy, along with thier gene expression and pathway alterations. Our findings highlight that crosstalk between cancer-associated fibroblasts (CAFs) and tumor cells represents a critical determinant of neoadjuvant therapy resistance. Fibroblasts underwent significant state transitions post-treatment, marked by hypoxia-associated gene upregulation (e.g., S100A4) and immunosuppressive pathways. Meta-Programs (MPs) analysis identified a hypoxia-driven fibroblast state (MP5) containing S100A4 that correlated with treatment resistance. In vitro experiments, S100A4 co-localized with α-SMA + fibroblasts and directly induced PD-L1 expression in tumor cells, linking CAFs secreted S100A4 to immunosuppressive PD-L1 upregulation. Conclusion This cross-cancer single-cell atlas reveals S100A4, secreted by CAFs, as a conserved mediator of PD-L1 upregulation in tumor cells, driving immunosuppression and resistance to nICT. The atlas and mechanistic findings provide a rationale for targeting S100A4 to enhance treatment efficacy, pending validation in larger cohorts and mechanistic studies. This resource also supports the development of personalized, cross-cancer neoadjuvant strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/875f07738539c0aaf3118d5091d4cb483e59f11d" target='_blank'>
              Deciphering the tumor ecosystem dynamics undergoing immunochemotherapy therapy across multiple cancer types unveils the immunosuppressive role of S100A4 in fibroblasts by promoting PD-L1 expression in tumor cells
              </a>
            </td>
          <td>
            Bo Yang, Ruiji Chen, Mali Zu, Jie Yao, Hong Ren, Yi‐Shung Lin, Bo Zhang, Tianjiao Ji, Yang Liu
          </td>
          <td>2025-07-23</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metabolic crosstalk among diverse cellular populations contributes to shaping a competitive and symbiotic tumor microenvironment (TME) to influence cancer progression and immune responses, highlighting vulnerabilities that can be exploited for cancer therapy. Using a spatial multiomics platform to study the cell-specific metabolic spectrum in hepatocellular carcinoma (HCC), we map the metabolic interactions between different cells in the HCC TME and identify a unique tumor-immune-cancer-associated fibroblast (CAF) "interface" zone, where cell-cell interactions are enhanced and accompanied by significant upregulation of lactic acid and long-chain polyunsaturated fatty acids. Further combining single-cell mass spectrometry imaging of patient-derived tumor organoids, cocultured CAFs, and macrophages, we demonstrate that CAFs increase glycolysis and secrete lactic acid to the surrounding microenvironment to drive immunosuppressive macrophage M2 polarization. These findings facilitate the understanding of cancer-associated metabolic interactions in complex TME and provide clues for targeted clinical therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b292d4bb0787fa6280b9910112e99765dee292d" target='_blank'>
              Integrating spatial omics and single-cell mass spectrometry imaging reveals tumor-host metabolic interplay in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Panpan Chen, Haoyuan Geng, Bangzhen Ma, Yaqi Zhang, Zihan Zhu, Min Li, Shiping Chen, Xiao Wang, Chenglong Sun
          </td>
          <td>2025-07-29</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background Lung cancer, a leading cause of cancer death, displays profound histologic and molecular heterogeneity across adenocarcinoma, squamous, and small‐cell types. Clinically, tumours can shift between these states, reflecting lineage plasticity—the reprogramming of differentiated cells to alternate identities. Pre‐existing genomic/epigenomic diversity and microenvironmental cues supply the substrates and pressures for plasticity from disease onset. This review anchors plasticity within normal lung development to clarify how fate programs are co‐opted to drive progression, immune escape, therapy resistance, and invasion. Main text Focusing on the intricate interplay between lineage dysregulation and tumour progression in lung cancer, this review integrates insights from lung tissue development to explore the pivotal molecules and mechanisms driving lineage plasticity, alterations and migration during lung carcinogenesis and progression. Recent research findings on lung cancer lineage plasticity are synthesised, shedding light on the role of transcriptional and epigenetic regulators in disrupting tumour lineages. Particular emphasis is placed on how tumour microenvironmental factors, such as hypoxia, stromal cells and immune cells, reshape tumour cellular profiles by modulating the epigenomic landscape. Furthermore, this review specifically discusses the impact of epidermal growth factor receptor (EGFR) and KRAS mutations on lung cancer progression and the consequent immune escape mechanisms they engender. Importantly, we highlight that lineage regulation persists throughout tumour development, from the early onset of lung adenocarcinoma (LUAD) to its progression through late‐stage dedifferentiation and metastasis. We evaluate the implications of these factors on treatment resistance in lung cancer and focus on innovative therapeutic strategies targeting lineage plasticity. Conclusions Lineage plasticity spans the entire course of lung cancer, from early tumorigenesis through metastasis to treatment resistance. Lineage transitions that occur during tumour progression arise from specific combinations of genomic and epigenetic alterations and are further shaped by microenvironmental forces such as hypoxia, stromal remodeling, and immune pressure. By summarising current research advancements, we aim to provide new insights for future lung cancer research and to promote the development of more effective therapeutic interventions. Key points Lineage plasticity runs through the entire process of lung cancer progression and drug resistance, and drives early tumorigenesis via lineage imbalance. Certain driver mutations have lineage‐restricted tumorigenic potential, requiring lineage reprogramming for tumor initiation. Lineage transitions in lung cancer require specific genomic and epigenetic alterations. Lineage plasticity insights provide a mechanistic framework linking lung cancer origin, evolution, and therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d07aa8707388affcb2b558f5da5a8327303f46a8" target='_blank'>
              Persistent lineage plasticity driving lung cancer development and progression
              </a>
            </td>
          <td>
            Fanchen Meng, Jianyu Li, Zhijun Xia, Qinglin Wang, Qinhong Sun, Siwei Wang, Lin Xu, Rong Yin
          </td>
          <td>2025-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer remains a formidable global health burden owing to its persistently high incidence and mortality, highlighting the need to elucidate its underlying biological mechanisms. Such insights are crucial for refining diagnostic precision and therapeutic efficacy. Notably, human tumors represent highly heterogeneous ecosystems, comprising a dynamic interplay between malignant cells and non-malignant components, such as immune infiltrates and stromal elements. This cellular heterogeneity constitutes a major obstacle in tumor research. Recent technological advancements have enabled the development and application of novel tools for investigating tumor heterogeneity. In this context, single-cell sequencing technologies emerged as transformative tools for dissecting tumor architecture at a cellular resolution, offering unprecedented insights into the cellular diversity and molecular underpinnings of cancer. In particular, single-cell RNA sequencing (scRNA-seq) has been widely used to analyze the tumor microenvironment, particularly its immune components, thereby providing a valuable framework for understanding tumor-immune system interactions. However, the intrinsic heterogeneity of tumor cells has received comparatively less attention, with limited studies focusing on these malignant populations. Comprehensive profiling of tumor cells has become increasingly feasible as scRNA-seq technologies continue to evolve, offering higher throughput, increased cell capture efficiency, and more advanced analytical pipelines. Despite these advancements, current applications of scRNA-seq remain primarily focused on immune and stromal cell populations. Therefore, a paradigm shift is warranted, redirecting the investigative focus toward tumor cells and gaining deeper insights into tumorigenesis and therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c3598e7911b47f7291bb4a9f0c4b83bd05a9bc" target='_blank'>
              Exploration and Application of Malignant Cell Heterogeneity Analysis with Single-Cell Transcriptome Sequencing Technology.
              </a>
            </td>
          <td>
            Wen Zhang, Tianliang Liu, Ting Liu, Yu Zhang, Zhen-guo Zhao, Xinxin Zhang, Xiaoyang Li, Jiasheng Zhong, Zhicheng Li, Shifu Chen, Libin Xu
          </td>
          <td>2025-07-25</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary Circulating tumor cells (CTCs) are the primary mediators of metastatic disease. After leaving the tumor, they can travel through the blood alone or with cluster partners to aid them on their way to distant metastatic sites. While CTCs are critical diagnostic tools for monitoring cancer progression, the biology underlying their behavior is obscured by their rarity and heterogeneity. To unravel the complexity of their actions, we undertook a large-scale longitudinal study to determine the factors that affect their presence, clustering, and their impacts on disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ef013e40043bfc40c0608d2cc2f97f3380a6366" target='_blank'>
              Comprehensive Longitudinal Linear Mixed Modeling of CTCs Illuminates the Role of Trop2, EpCAM, and CD45 in CTC Clustering and Metastasis
              </a>
            </td>
          <td>
            Seth D. Merkley, Huining Kang, U. Brown-Glaberman, Dario Marchetti
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background and objective Melanoma exhibits profound biological complexity, driven by immune evasion, phenotypic plasticity, and resistance to therapy. While programmed cell death (PCD) shapes tumor–immune interactions, its mechanistic landscape in melanoma remains incompletely defined. This study aims to comprehensively characterize PCD-related signatures and their associations with tumor heterogeneity, prognosis, and immunotherapeutic outcomes. Methods Single-cell RNA sequencing data from melanoma cohorts (cutaneous and acral subtypes) were used to assess PCD activity via AUCell-based scoring across major cell types. Cell-type–specific analyses examined heterogeneity, metabolic dependencies, and pathway correlations. Intercellular communication was analyzed using CellChat. Bulk RNA sequencing data were then integrated to identify PCD-related gene signatures, and machine learning models (LASSO, Ridge, XGBoost) were applied to develop a prognostic model. Immune infiltration, immunogenomic correlations, and immunotherapy responses were further evaluated using ESTIMATE, CIBERSORT, TMB, IPS, and external ICB-treated cohorts. Results Among all cell types, melanoma cells exhibited the highest PCD activation, with disulfidptosis, immunogenic cell death (ICD), and autosis being the most prominent. High PCD activity was linked to advanced clinical stage, lymphatic metastasis, and poor prognosis. Melanoma subpopulations with hyperactivated PCD displayed elevated copy number variation (CNV) burden, enhanced fibroblast/endothelial interactions, and invasive transcriptional profiles. A 15-gene prognostic signature was developed, effectively stratifying survival and immunotherapy response across multiple cohorts. Low-risk tumors demonstrated favorable immune infiltration (CD8+ T cells, M1 macrophages), higher tumor mutational burden (TMB), and greater immunogenicity, while high-risk tumors exhibited immune exclusion, cancer-associated fibroblast (CAF) enrichment, and adverse mutations. Conclusion This study highlights the functional and clinical significance of PCD heterogeneity in melanoma and provides a validated prognostic model for patient stratification and therapeutic decision-making. These findings underscore the potential of targeting PCD dynamics as a novel approach in melanoma management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2d7ead94be26e504d9669468ee3ae02d2938b23" target='_blank'>
              Programmed cell death-driven remodeling of the melanoma microenvironment enables prognostic stratification and therapeutic prediction
              </a>
            </td>
          <td>
            Bo Hu, Shengnan Chai, Xuan Li, Qiang Zhang, Mei Jin, Long Zhang
          </td>
          <td>2025-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most aggressive brain tumor in adults, is characterized by its infiltrative growth along the perivascular space. Mural cells (MCs), encompassing pericytes and smooth muscle cells, are multifunctional perivascular cells implicated in GBM progression. MCs not only facilitate vascular co‐option but have also been suggested to contribute to the immunosuppressive tumor microenvironment, promoting tumor growth and migration. However, whether MC interactions with immune cells differ based on their proximity to the tumor remains unclear. Using single‐cell RNA sequencing, we analyzed MC transcriptome profiles across distinct regions relative to the tumor mass in mouse and human GBM samples. Tumor‐residing MCs exhibited profound phenotypic changes, showing upregulated gene expression and enhanced signaling activity toward immune cells, with region‐specific ligand–receptor interactions. Conversely, border‐residing MCs, despite their abundance, showed reduced activation and lacked distinct transcriptional profiles. These findings reveal spatially defined transcriptional heterogeneity in MCs within the GBM microenvironment, underscoring their dynamic role in the GBM microenvironment. This study provides novel insights into MC responses in GBM, identifying potential avenues for targeting MC–immune‐cell interactions in therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34269e7ccaff0901b281158ca70f685dbde49243" target='_blank'>
              Pericytes change function depending on glioblastoma vicinity: emphasis on immune regulation
              </a>
            </td>
          <td>
            C. Buizza, Robert Carlsson, Coralie Gamper, Gayatri Chitale, J. Bengzon, Gesine Paul
          </td>
          <td>2025-07-17</td>
          <td>Molecular Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Mixed neuroendocrine–nonneuroendocrine neoplasms (MiNEN) are usually highly aggressive tumors characterized by marked histological heterogeneity, most commonly represented by mixed adenocarcinoma and poorly differentiated neuroendocrine carcinoma (NEC). However, beyond morphological observations, the biological basis and implications of this heterogeneity remain incompletely understood. In this study, we combined component-specific next-generation sequencing and spatial transcriptomics to investigate three mixed adenocarcinoma-NEC cases from different anatomical sites (ileocecal, ovarian, gastric), tracing tumor progression from precursor lesions to invasive NEC. Genomic analyses revealed a shared trunk of driver mutations across all tumor compartments, confirming their clonal origin, while also uncovering additional compartment-specific alterations. Spatial transcriptomics, together with gene set enrichment analysis (GSEA), revealed distinct transcriptional profiles aligned with histologically annotated compartments (e.g., adenocarcinoma, NEC, precursor). In NECs, GSEA consistently showed downregulation of immune-related pathways and upregulation of proliferation-associated pathways compared to non-neuroendocrine tumor areas. Moreover, distinct transcriptomic subclusters were identified within morphologically homogeneous NEC regions in two of the three cases. These subclusters exhibited significant differences in immune regulation, proliferation signaling, and cell-cycle control, and were associated with divergent predicted chemotherapy-response signatures, suggesting clinically relevant implications for treatment sensitivity and resistance. In summary, our findings indicate that despite a shared clonal origin, MiNEN develop distinct genetic and transcriptomic features across tumor compartments. The inconsistent presence of transcriptomic subclusters within morphologically similar regions underscores the complexity of intratumoral heterogeneity in these aggressive neoplasms. By connecting morphological and molecular layers of tumor architecture, spatial profiling may aid in translating biological complexity into more targeted clinical strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12022-025-09869-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d940265ab2a62d1a11097a7fc51ef43cc25274" target='_blank'>
              Exploring Intratumoral Heterogeneity in Mixed Neuroendocrine-Nonneuroendocrine Neoplasms with Spatial Transcriptomics: Even More Diverse Than Anticipated
              </a>
            </td>
          <td>
            Annika Weiß, Julia Teply-Szymanski, Maxime Schmitt, S. Foersch, P. Jank, Joscha Griger, Uwe Wagner, Detlef K Bartsch, Carsten Denkert, Moritz Jesinghaus
          </td>
          <td>2025-08-20</td>
          <td>Endocrine Pathology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="The key advantage of spatial transcriptomics (ST) technologies lies in the spatial domain: these techniques not only offer an unprecedented opportunity to interrogate intact biological samples in a spatially informed manner, but also set the stage for integration with other imaging-based modalities. However, how to most effectively exploit spatial context and integrate ST with imaging-based modalities that capture morphological insight remains an open and heavily investigated question. To address this, particularly under real-world experimental constraints such as limited dataset size, class imbalance, and bounding-box-based segmentation, we used a publicly available murine ileum Multiplexed Error-Robust Fluorescence In Situ Hybridization (MERFISH) dataset to evaluate whether a minimally tuned variational autoencoder (VAE) could extract informative low-dimensional representations from cell crops of spot counts, nuclear stain, membrane stain, or a combination thereof. We assessed the resulting embeddings through PERMANOVA, cross-validated classification, and unsupervised Leiden clustering, and compared them to classical image-based feature vectors extracted via CellProfiler. While transcript counts (TC) generally outperformed other feature spaces, the VAE-derived latent spaces (LSs) captured meaningful biological variation and enabled improved label recovery for specific cell types. LS2, in particular, trained solely on morphological input, also exhibited moderate predictive power for a handful of genes in a ridge regression model. Notably, combining TC with LSs through multiplex clustering led to consistent gains in cluster homogeneity, a trend that also held when augmenting only subsets of TC with the stain-derived LS2. In contrast, CellProfiler-derived features failed to match the performance of the LSs, highlighting the advantage of learned representations over hand-crafted features. Collectively, these findings demonstrate that even under constrained conditions, VAEs can extract biologically meaningful signals from imaging data and constitute a promising strategy for multi-modal integration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/831421dc91eca6d1f6b01729afc1f1ce61dd5d5b" target='_blank'>
              Evaluating Integrative Strategies for Incorporating Phenotypic Features in Spatial Transcriptomics
              </a>
            </td>
          <td>
            Levin M Moser, Ahmad Kamal Hamid, Esteban Miglietta, Nodar Gogoberidze, Beth A. Cimini
          </td>
          <td>2025-07-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Multipotency refers to the ability of a cell to differentiate into multiple, yet limited as opposed to pluripotency, number of cell types within a specific lineage or tissue. Studies using transgenic mouse models of the mammary gland have revealed a cellular hierarchy in which both luminal and basal lineages are replenished by unipotent progenitor cells. Hence, despite the existence of bipotent stem cells, normal mammary gland homeostasis is intimately linked with unipotency. However, recent literature revealed that under specific physiological or experimental conditions, lineage-restricted mammary cells can reacquire multipotency and undergo a lineage switch, challenging the traditional unidirectional model of cell differentiation. This reactivation of multipotency has been observed, for instance, in response to pregnancy, lineage ablation or oncogenic stimuli, indicating a certain level of plasticity that may have consequences in the context of tumorigenesis. Understanding the molecular mechanisms governing this phenomenon could provide valuable insights into mammary gland cellular hierarchy and breast cancer progression. Indeed, reactivation of multipotency is a result of developed cell plasticity, which can drive tumor heterogeneity, promote disease aggressiveness and hamper diagnosis. This review provides an overview of models that have inferred reactivation of multipotency, discusses the underlying molecular and cellular mechanisms and proposes future perspectives for research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c4f8296e9963ebb8925de4fecc8f34d988678e" target='_blank'>
              Reactivation of Multipotency in the Mammary Gland – a Ripple in the Pond and a Turn of the Tide
              </a>
            </td>
          <td>
            C. Hager, C. Jehanno, M. Bentires-Alj
          </td>
          <td>2025-07-17</td>
          <td>Journal of Mammary Gland Biology and Neoplasia</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Histopathological analysis has been transformed by serial section-based methods, advancing beyond traditional 2D histology to enable volumetric and microstructural insights in oncology and inflammatory disease diagnostics. This review outlines key developments in specimen preparation and high-throughput imaging that support these innovations. Computational workflows are categorized into multimodal image co-registration, 3D histoarchitecture reconstruction, multiplexed immunohistochemical correlation, and cross-scale data fusion. These approaches exploit serial section-derived spatial concordance to enhance resolution in microenvironmental and molecular profiling. Despite progress, challenges remain in harmonizing heterogeneous datasets, optimizing large-scale registration, and ensuring interpretability. Future directions include spatial transcriptomics, and applications in developmental biology and neuroscience in AI integration, establishing serial section analytics as central to precision histopathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86af6520d3a1a379b187f872dae8a73b0dd5b112" target='_blank'>
              Evolutionary Paradigms in Histopathology Serial Sections technology
              </a>
            </td>
          <td>
            Zhenfeng Zhuang, Min Cen, Lei Jiang, Qiong Peng, Yihuang Hu, Hong-Yu Zhou, Liansheng Wang
          </td>
          <td>2025-08-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Osteosarcoma (OS) is the most common primary malignant bone tumour of childhood, yet five-year survival has plateaued at ~60–70% for localised disease and plunges below 30% once metastasis emerges. Formerly viewed as a cell-intrinsic neoplasm entombed in mineralised bone, OS is now understood as a spatially stratified ecosystem whose immune-evasion niches choreograph progression. Three-dimensional spatial transcriptomics (3-D ST) fuses barcode-based transcript capture with volumetric reconstruction, preserving x-, y- and z-axis context and exposing concentric C1QC necrotic belts, MCAM (melanoma cell-adhesion molecule, CD146) peri-vascular corridors, hypoxic glycolytic rims and therapy-induced tertiary-lymphoid islets that collectively sequester cytotoxic lymphocytes. Pre-clinical atlases link PD-L1 high SOX9 stem-like cells, LGALS3 macrophages and VEGFA-driven endothelial tips to chemoresistance and immune-checkpoint failure, while ligand–receptor inference highlights VEGFA–VEGFR2, CXCL12–CXCR4 and complement–CSF1R axes as actionable bottlenecks. Translational efforts already echo these insights: dual MCAM/VEGFR blockade collapses vascular gates, C1s or CSF1R antagonists dismantle necrotic-core “cold pockets”, and MCT1–POSTN combinations target lactate-stiffened stromal shells. By weaving methodological advances with emergent biology, this review crystallises how 3-D ST redefines OS immunopathology, sharpens biomarker discovery and accelerates spatially guided combination therapies. We aim to expose diagnostic blind spots, spotlight niche-directed interventions and chart a roadmap toward lifting the long-standing therapeutic ceiling in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c84eb34c63a5fd31899b587fca53cec9b872be8" target='_blank'>
              Deciphering spatially confined immune evasion niches in osteosarcoma with 3-D spatial transcriptomics: a literature review
              </a>
            </td>
          <td>
            Guangqiang Qiu, Yongcheng Tang, Junhui Zuo, Heng Wu, Yongxian Wan
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f3009a3214b83a250848d6ef5905a4fb9e1a3db" target='_blank'>
              Unraveling the single cell spatial landscapes of melanoma brain metastases
              </a>
            </td>
          <td>
            A. Manukyan, K. Peters, H. Radbruch, F. Roßner, A. Lehmann, K. Kleo, T. Conrad, J. Altmueller, I. Plumbon, A. Akalin, K. Juerchott, E. Wyler, M. Landthaler, J. Radke, T. Redmer
          </td>
          <td>2025-09-02</td>
          <td>None</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT Continuous passaging of cancer cell lines can drive phenotypic and genotypic divergence, potentially compromising the reliability of such models. In this study, we show that two late-passage strains (S1 and S2) of ovarian cancer cell line SKOV3, although authenticated via short tandem repeat (STR) profiling as identical, exhibit substantial differences in morphology, transcriptomic signatures, ability to form 3D cultures and chemotherapeutic responses. Notably, S1 formed compact 3D spheroids and exhibited enhanced epithelial-mesenchymal transition (EMT) pathway activity, whereas S2 displayed a more proliferative, MYC-driven phenotype with larger spheroid structures requiring higher seeding densities. Transcriptomic analysis revealed pathways associated with hypoxia, EMT and angiogenesis in 3D culture, highlighting the complexity introduced by dimensionality in tumour modelling. Critically, S1 showed higher sensitivity to doxorubicin than S2 (IC50 of 0.12 µM versus 1.28 µM, P=0.0001), indicating how clonal evolution can confound drug-response assays. Ultimately, our findings suggest that although STR profiling remains essential for cell line authentication, functionally distinct subpopulations can arise and coexist within the same culture, and their isolation may reveal divergent phenotypes that compromise reproducibility in preclinical cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85b19ad530ecaca77ba40f9ce705c8944938d900" target='_blank'>
              Extended passaging of the SKOV3 ovarian cancer cell line leads to two phenotypically different strains
              </a>
            </td>
          <td>
            Eglė Žymantaitė, M. Gabrielaitė, V. Pašukonienė, A. Mlynska
          </td>
          <td>2025-08-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Spread through air spaces (STAS) represents a novel invasion mechanism in adenocarcinoma that considerably influences lung cancer clinical outcomes; however, studies of its mechanisms at the spatial level are lacking. Methods We used the NanoString GeoMx digital spatial profiling (DSP) technology to conduct a spatial transcriptomic analysis of surgically resected tissues from non-small-cell lung cancer (NSCLC) patients with or without STAS. Results Compared with tumor nests in non-STAS patients, HLA-DRB5 and RASGRF1 were significantly less expressed in compartments of STAS, suggesting their inhibitory roles in the occurrence of STAS. Meanwhile, an increase in CD4 T memory cells and a decrease in B cells were observed in the tumor immune microenvironment of STAS. Furthermore, distinct molecular profiles were observed between tumor cells in tumor nests and in air spaces in STAS patients, which was highlighted by the elevated ITGA2 expression in the air spaces. These results were validated in an independent cohort by multiplex immunofluorescence stainings. Conclusion This study is the first to use DSP to analyze spatial transcriptomic profiles of NSCLC tumor nests and air space tumors, and it identifies potential module features that may be used for STAS identification and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/570168f77732c756800b064b7d546cfa69566936" target='_blank'>
              Spatial deciphering of the transcriptomic heterogeneity of tumor spread through air spaces in lung cancer
              </a>
            </td>
          <td>
            Wenhao Wang, Wenhao Zhou, Jingli Fan, Tao Jiang, Guang Yang, Congcong Song, Siwei Xu, Haitao Luo, Huining Liu
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="A whole pathology section contains approximately 1,000,000 cells of various types, this large-scale heterogeneity of cells and non-cellular constituents constructs a mutually competitive community. Conventional pixel-based visual processing techniques are insufficient to accurately capture the complexities inherent with cell-entity deployment and formation strategy. Here, we conquered segmentation and classification of all cells on the whole pathology sections from 387 hepatocellular carcinoma (HCC) patients across six cohorts with 57 pathologists assisted. Further, an AI system called Hybrid Graph Neural Network-Transformer system (HGTs) was proposed. It precisely predicted local recurrence of postoperative HCC by analyzing cell interactions across multiple scales, from cell-to-cell, cell-community, to tissue-level interactions. The proposed HGTs outperformed existing SOTA methods, with the C-index improving by 23.1% to reach 0.823, by further integrating multimodal data, including clinical information and immunohistochemical markers. A set of spatial relational biomarkers influencing tumor prognosis was discovered and quantitatively validated. They include the frequency of tumor-lymphocyte and tumor-tumor interactions, the distribution and sparsity of key cellular communities, and the degree of fibrosis in adjacent peritumoral tissues. Utilizing the anti-tumor potential of this marker set, we’re developing therapies to enhance the immune system’s fight against cancer. All cell semantic segmentation datasets and code are publicly available: https://github.com/Yuan1z0825/HGTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2cd5bfdebf32b51dd7e8525e95b35b9df810fb6" target='_blank'>
              Cell graph analysis in hepatocellular carcinoma: predicting local recurrence and identifying spatial relationship biomarkers
              </a>
            </td>
          <td>
            Yizhe Yuan, Ziyin Zhao, Xin Fang, Qing Zhang, Wenqing Zhong, Midie Xu, Gongqi Li, Rushi Jiao, Heng Yu, Ruoxi Wang, Shuyu Liu, Weitao Zu, Bingsen Xue, Yuze Chen, Chengxiang Wang, Ya Zhang, Minghui Liang, Bing Han, Cheng Jin
          </td>
          <td>2025-07-25</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Gliosarcoma (GS) is a rare variant of glioblastoma (GBM), characterized by biphasic glial and sarcomatous histology and poor prognosis. Despite its distinct clinical features, GS remains underrepresented in glioma research due to the lack of biologically faithful and experimentally tractable models. Methods We established patient-derived gliosarcoma organoids (GSOs) from freshly resected tumors using a suspension culture system without enzymatic dissociation. Histological, molecular, and functional properties were evaluated using H&E staining, immunohistochemistry, whole-exome sequencing, 3D invasion assays, and single-cell RNA sequencing (scRNA-seq). Drug response assays were performed using temozolomide and ANA-12 (NTRK2 inhibitor). Results GSOs preserved key histological features, genetic alterations, and tumor microenvironmental cell populations from the original tumors. They stable growth and viability over extended culture periods and maintained integrity following cryopreservation and recovery, supporting their long-term utility. 3D invasion assays further revealed infiltrative behavior, consistent with the aggressive nature of GS. Histological and molecular analyses revealed that GSOs retained glial and mesenchymal differentiation, diverse non-malignant stromal cells, and case-specific somatic alterations. Comparative scRNA-seq revealed distinct transcriptional programs: GSOs were enriched for fibroblast-like and oligodendrocyte progenitor-like states, while glioblastoma organoids (GBOs) displayed astrocyte-like differentiation and high connectivity signatures. Functional assays confirmed consistent sensitivity of GSOs to temozolomide, and selective therapeutic response to NTRK2 inhibition was observed in the GSO harboring an NTRK2 alteration, supporting the utility for genomic-context-guided therapy. Conclusion Together, GSOs constitute a tractable and translationally relevant model that faithfully reflects the cellular complexity, genetic landscape, and tumor-specific heterogeneity. This model addresses a critical gap in GS biology and supports its integration into precision oncology. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06952-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6998dea80dedd424ac624050d4245e9964a32790" target='_blank'>
              Tumor microenvironment-preserving gliosarcoma organoids as an in vitro preclinical platform: a comparative analysis with glioblastoma models
              </a>
            </td>
          <td>
            Junseong Park, Dokyeong Kim, Hyeon-Chun Park, Min-Gyoung Park, Songzi Zhang, Okcho Na, Youn Soo Lee, Minho Lee, Stephan Ahn, Yeun-Jun Chung
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Sacrococcygeal teratomas (SCTs) are the most common neonatal tumors, yet their cellular origins, clinical stratification, and sex bias–occurring three times more in XX than XY individuals—remain poorly understood. To address these gaps, we examined six postnatal (one male and five female) and two prenatal (both female) SCTs by single nuclei RNA-seq and spatial transcriptomics. We identified five broad cellular lineages in SCTs: stroma, epithelia, endothelia, neuroectoderm, and immune. The transcriptomes and lineage compositions showed significant heterogeneity, which offer a framework for future molecular stratification. SCTs are thought to originate from and be propagated by pluripotent cells, notably however, we did not detect these populations. Among female tumors, a subset of cells exhibited biallelic expression of X-linked genes, consistent with X-inactivation failure or reactivation of the once inactivated X-chromosome. These biallelic cells were enriched for developmental and neuronal programs, whereas cells with single-allelic X-chromosome preferentially expressed immune-related genes. Biallelic X-chromosome activation, which can occur only in female cells, may result in transcriptomic features that favor survival of tumor cells, contributing to the sex bias of SCTs. Our findings reveal a link between X-chromosome inactivation and SCT cell identity, suggesting that X-dosage dysregulation may influence SCT heterogeneity and immune landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee83296f6240c6d4cef47d40571146d063d815c6" target='_blank'>
              Integrated single-nuclei and spatial transcriptomic profiling of human sacrococcygeal teratomas reveals heterogeneity in cellular composition and X-chromosome inactivation
              </a>
            </td>
          <td>
            Ernesto J. Rojas, K. Giannikou, Benjamin J. Huang, Soo-Jin Cho, Marco Cordero, Deion Pena, Lan Vu, Aditya Bagrodia, S. Derderian, T. Mackenzie, D. Laird
          </td>
          <td>2025-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Current approaches to estimating cell trajectories, tumor progression dynamics, and cell population diversity of tumor microenvironment often depend on single-cell RNA sequencing, which is costly and resource intensive. To address this limitation, we developed an artificial intelligence (AI) model that leverages cell morphology features and histological spatial organization to classify tumor cell differentiation status, infer cell dynamic trajectories, and quantify tumor progression from hematoxylin and eosin (H&E)–stained whole-slide images. In three independent lung adenocarcinoma cohorts, our AI-based model accurately predicted cell differential status and provided quantifiable measures of tumor progression that were prognostic of patient survival. Spatial transcriptomic integrative analyses revealed cell components and gene signatures enriched in different cell differentiation statuses. Bulk transcriptomic analyses revealed that fast-progressing tumors exhibit up-regulated cell cycle pathways, while slow-progressing tumors retain characteristics of normal lung epithelium. This cost-effective method enables large-scale analysis of tumor progression dynamics using routinely collected pathology slides and provides insights into intratumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ccd7a11dd44adb135af6900f42639f7a9eb70bd" target='_blank'>
              Image-based inference of tumor cell trajectories enables large-scale cancer progression analysis
              </a>
            </td>
          <td>
            Yang Liu, Ling Cai, Ruichen Rong, Shidan Wang, Liwei Jia, Peiran Quan, Qin Zhou, Guan‑Di Xiao, Yang Xie
          </td>
          <td>2025-07-18</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="SUMMARY Ovarian high-grade serous carcinoma (HGSC) is characterized by extensive intra-peritoneal dissemination and tumor heterogeneity. In the metastatic cascade, tumors utilize several transcriptional programs to translocate and survive in distant tissues. Here, we analyzed multi-modal, real-world data from 350 tumor samples across 160 patients with HGSC to identify transcriptional programs that drive intra-peritoneal metastasis and heterogeneity. We identified nine transcriptional programs, including those regulating epithelial-mesenchymal transition and immune modulation and cytoskeletal reorganization, which shape distinct metastatic trajectories to solid and ascitic environments and are associated to treatment response. Our results reveal pronounced intra-patient transcriptional heterogeneity, which in some cases surpassed inter-patient heterogeneity, highlighting the importance of multi-site sampling for accurate prognostication and combinatorial treatments in HGSC. Our extensive characterization offers novel insights into intra-peritoneal metastasis with significant prognostic implications, reveals histomorphological biomarkers for patient stratification and paves the way for innovative therapeutic strategies aimed at impairing cancer cell adaptability and limiting metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32c6dc18825437bb12a5a64bda09f91c2c6d329" target='_blank'>
              Multi-modal characterization of transcriptional programs that drive metastatic cascades to solid sites and ascites in ovarian cancer
              </a>
            </td>
          <td>
            Kaiyang Zhang, Essi Kahelin, G. Marchi, Oskari Lehtonen, S. Salloum, K. Lavikka, Yilin Li, Felix Dietlein, A. Lahtinen, J. Oikkonen, S. Hietanen, J. Hynninen, Antti Häkkinen, A. Virtanen, S. Hautaniemi
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Introduction Recent advancements in transcriptomic analysis, combined with single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics, have deepened our understanding of the tumor microenvironment and cellular heterogeneity, laying the groundwork for personalized therapies. The aim of this research is to explore the heterogeneity of tumor cells in colorectal cancer (CRC) and evaluate their prognostic value in different therapeutic contexts, emphasizing the impact of tumor cell heterogeneity on disease progression. Methods scRNA-seq alongside spatial transcriptomics was employed to analyze the heterogeneity of tumor cells in CRC, the spatial distribution of tumor cells, and their interactions with the microenvironment. Results We identified nine distinct tumor cell subtypes, with MLXIPL + neoplasm prevalent in advanced CRC, while ADH1C + and MUC2 + neoplasms were more common in early-stage CRC. MLXIPL + neoplasm was significantly associated with chemotherapy and targeted therapy efficacy. Analysis of spatial transcriptomics indicated that MLXIPL + neoplasm is located in the core area of the tumor cells. We developed a 13-gene prognostic signature (PS) using machine learning algorithm (StepCox backward), which predicts the prognosis of CRC patients. Furthermore, the patients with low PS score demonstrated higher immune cell infiltration and immune regulatory factors, suggesting enhanced immune surveillance and treatment response. Conclusions The findings highlight the critical role of tumor cell heterogeneity in CRC progression and treatment response, suggesting the need for personalized therapeutic strategies targeting different subpopulations. The constructed PS demonstrates significant clinical application value in predicting patient prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00805-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8065bcb7e9315d28e942e5a0a0b33a35e0d6f402" target='_blank'>
              Spatial transcriptomics and scRNA-seq: decoding tumor complexity and constructing prognostic models in colorectal cancer
              </a>
            </td>
          <td>
            Wei Song, Yatao Wang, Min Zhou, Fengqin Guo, Yanliang Liu
          </td>
          <td>2025-08-13</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Innate immune cells constitute the majority of the tumor microenvironment (TME), where they mediate both natural anti-tumor immunity and immunotherapy responses. While single-cell T- and B-cell receptor sequencing has provided fundamental insights into the clonal dynamics of human adaptive immunity, the lack of appropriate tools has precluded similar analysis of innate immune cells. Here, we describe a method that leverages somatic mitochondrial DNA (mtDNA) mutations to reconstruct clonal lineage relationships between single cells across cell types in native human tissues. We jointly sequenced single-cell transposase-accessible chromatin and mtDNA to profile n=124,958 cells from matched tumor, non-involved lung tissue (NILT), and peripheral blood of early-stage non-small cell lung cancer (NSCLC) patients, as well as n=93,757 cells from matched tumor and peripheral blood of ovarian cancer patients. Single-cell concomitant profiling of lineage and cell states of thousands of immune cells resolved clonality across cell types, tissue sites, and malignancies. Clonal tracing of innate immune cells demonstrates that TME-resident myeloid subsets, including macrophages and type 3 dendritic cells (DC3), are clonally linked to both circulating and tissue-infiltrating monocytes. Further, we identify distinct DC-biased and macrophage-biased myeloid clones, enriched in the tumor and NILT, respectively, and find that their circulating monocyte precursors exhibit distinct epigenetic profiles, suggesting that myeloid differentiation fate may be predetermined before TME infiltration. These results delineate the clonal pathways of intratumoral myeloid cell recruitment and differentiation in human cancer and suggest that remodeling of the tumor myeloid compartment may be peripherally programmed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3aaf4843ae398b98669a86ffe5560f59ef43af0" target='_blank'>
              Clonal lineage tracing of innate immune cells in human cancer
              </a>
            </td>
          <td>
            Vincent Liu, Katalin D. Sandor, Patrick K. Yan, Max Miao, Yajie Yin, Robert R. Stickels, Andy Y. Chen, Kamir J. Hiam-Galvez, Jacob C. Gutierrez, Wenxi Zhang, Sairaj Sajjath, Raeline Valbuena, Steven Wang, Bence Dániel, Leif S. Ludwig, Brooke E Howitt, Caleb A. Lareau, Ansu Satpathy
          </td>
          <td>2025-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Simple Summary Meningiomas are common brain tumors, but a subset behaves aggressively and resists current treatments. This review proposes a developmentally informed framework showing that where a meningioma arises in the skull mirrors its embryologic origin (neural crest vs. mesoderm) and imprints distinct genetic, epigenetic, and immune programs that shape tumor behavior and treatment response. This lineage “stamp” helps explain why convexity and skull-base tumors differ and highlights actionable circuits, NF2/YAP–TAZ, PI3K/AKT, TRAF7-linked pathways. We integrate spatial genomics and single-cell data to map tumor location and molecular class to rational therapies, including CDK4/6, PI3K/AKT, YAP/TEAD, and epigenetic inhibitors, with emerging strategies such as synthetic-lethal combinations, CRISPR-guided target discovery, and degrader (PROTAC) approaches for high-grade and recurrent disease. Our goal is a practical roadmap that links anatomy and biology to precision treatment and trial design, improving outcomes by matching the right therapy to the right meningioma at the right site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3274f2ff22ffc0fdf55125037e12d854830b1a18" target='_blank'>
              Spatially Encoded Oncogenesis and Transcriptional Plasticity in Meningioma: Drivers of Therapeutic Resistance and Opportunities for Targeted Intervention
              </a>
            </td>
          <td>
            Matthew A Abikenari, Amit Regev, Brandon Bergsneider, Vratko Himič, Shreyas Annagiri, Lily H Kim, Ravi Medikonda, John Choi, S. Jeyaretna, Daniel M. Fountain, Michael Lim
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer characterized by a lack of hormone receptors, making it challenging to treat.We generated a comprehensive spatial cell atlas of TNBC using a multi-omics integration approach that combined single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics. This integration allowed us to characterize the spatial microenvironment and map the cell-type-specific distributions in TNBC tissues.Our analysis revealed significant heterogeneity in cell types and spatial distribution, with normal regions enriched in insulin resistance functions, whereas cancerous regions displayed diverse cell populations, including immune cells, cancer-associated fibroblasts (CAFs), and mesenchymal cells. By constructing transcription factor (TF) regulatory networks, we identified TFF3, RARG, GRHL1, RORC, and KLF5 as critical regulators of epithelial cells, whereas EMX2, TWIST1, TWIST2, NFATC4, and HOXC6 were found to play essential roles in mesenchymal cells. Immunohistochemical validation supported the involvement of these TFs in TNBC. Further analysis of receptor-ligand interactions highlighted the roles of KNG1_BDKRB2 and NRG1_ERBB4 signaling in promoting tumor aggression, suggesting potential therapeutic targets. GO enrichment analysis revealed overlapping pathways between epithelial and mesenchymal cells, focusing on migration, signaling, and development, indicating that the shared regulatory mechanisms contribute to cancer progression.Our findings provide new insights into the TNBC microenvironment, emphasizing the complex spatial interactions between different cell types and highlighting key regulatory pathways that could be targeted for future therapeutic interventions. This spatial cell atlas lays the foundation for further exploration of tumor microenvironment dynamics and precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8697e1d4f457ce1a990f5f037a0dd30fbd923ee5" target='_blank'>
              Multi-omics integration uncovers key transcriptional regulators in triple-negative breast cancer spatial heterogeneity
              </a>
            </td>
          <td>
            Ning Zhang, Ye Zhang, Rui-Fei Yang, Hai-Qing Chu, Abdur Rehman, Lu-Yu Yang, Ya-Yu Li, Xin Zhou, Shou-Ping Gong, Hui-Ling Cao
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42c69db52b1c0603dccbe1767f18803f881e000a" target='_blank'>
              Accurately Predicting Cell Type Abundance from Spatial Histology Image Through HPCell.
              </a>
            </td>
          <td>
            Yongkang Zhao, Youyang Li, Weijiang Yu, Hongyu Zhang, Zheng Wang, Yuedong Yang, Yuansong Zeng
          </td>
          <td>2025-09-03</td>
          <td>Interdisciplinary sciences, computational life sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d320162041d03e1b05ee82602e945a36e317508" target='_blank'>
              Hemocytes facilitate interclonal cooperation-induced tumor malignancy by hijacking the innate immune system in Drosophila.
              </a>
            </td>
          <td>
            Sihua Zhao, Yifan Guo, Xiaoyu Kuang, Xiaoqin Li, Chenxi Wu, Peng Lin, Qi Xie, Zongzhao Zhai, D. Kong, Xianjue Ma
          </td>
          <td>2025-08-22</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0c307663da680bc20dfe3893c95285de2664ca" target='_blank'>
              Spatiotemporal analyses of the pan-cancer single-cell landscape reveal widespread profibrotic ecotypes associated with tumor immunity.
              </a>
            </td>
          <td>
            Ya Han, Lele Zhang, Dongqing Sun, Guang-xu Cao, Yuting Wang, Jiali Yue, Junjie Hu, Zhonghua Dong, Fang Li, Taiwen Li, Peng Zhang, Qiu Wu, Chenfei Wang
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) was considered the most aggressive type of primary brain tumor, marked by poor clinical outcomes and a high tendency to relapse. The therapeutic efficacy of GBM was significantly compromised by tumor heterogeneity, dysregulated metabolic pathways, the formation of an immunosuppressive microenvironment, and treatment resistance. Therefore, multi-dimensional therapeutic strategies targeting GBM-specific molecular features, its intrinsic properties, and microenvironmental regulatory networks were considered to potentially provide new breakthroughs for overcoming treatment resistance in GBM. Methods We analyzed single-cell RNA sequencing (scRNA-seq) data processed with the Seurat package to accurately identify cell types. Spatial transcriptomics integrated Multimodal Intersection Analysis, TransferData, and Robust Cell Type Decomposition techniques to characterize the spatial distribution patterns of key cell subtypes. CellChat was employed to assess intercellular communication networks. Furthermore, in vitro experiments confirmed the main regulatory role of YEATS4 (key transcription factor of C2 PCLAF+ subtype) in GBM malignant progression. Results Through scRNA-seq, we identified the C2 PCLAF+ subtype in GBM and analyzed its molecular characteristics and functional role in tumor progression. This subtype exhibited a unique malignant phenotype, marked by significant proliferative activity, characteristic metabolic reprogramming, and dysregulated cell death regulation mechanisms. Spatial transcriptomics revealed its preferential localization within specific tumor niches. Furthermore, the C2 PCLAF+ subtype established a specific interaction with fibroblasts through the MDK-LRP1 ligand–receptor pair. Critically, silencing YEATS4 in vitro significantly inhibited GBM malignancy. Additionally, the prognostic risk score model based on the C2 PCLAF+ subtype demonstrated significant clinical translational value. Conclusion Our study systematically elucidated the malignant characteristics of the C2 PCLAF+ subtype and its molecular mechanisms driving GBM progression. This subtype promoted therapeutic resistance through unique metabolic reprogramming, MDK-LRP1-mediated microenvironmental interactions, and immunosuppressive properties. YEATS4 knockdown effectively suppressed malignant tumor behaviors, highlighting its therapeutic potential. These findings provided novel targeted intervention strategies to address GBM heterogeneity and treatment resistance, offering promising avenues for overcoming current therapeutic limitations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d8a996fe1a4f26a1455e98643e994edaf34d934" target='_blank'>
              Single-cell and spatial atlas of glioblastoma heterogeneity: characterizing the PCLAF+ subtype and YEATS4’s oncogenic role
              </a>
            </td>
          <td>
            Siqi Ma, Yuwei Sun, Shaowei Zheng, Yilong Fu, Liangyu Wang, Dun Liu, Henan Jiao, Xuqiang Zhu, Xueyuan Li, Dongming Yan, Di Chen, Zi Ye
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic cholangiocarcinoma (ICC) have not yet been fully characterized. In this study, we analyzed publicly available single-cell RNA sequencing (scRNA-seq) data derived from paired samples of two intrahepatic cholangiocarcinoma (ICC) tissues and two adjacent normal tissues, thoroughly examining their cellular composition. InferCNV analysis was employed to compare tumor cells and normal cells, and pseudotime analysis was used to identify the growth and differentiation trajectories of the cells. Additionally, intercellular communication analysis was conducted to elucidate the communication networks between cells. Our analysis delineated the cellular ecosystem of ICC, identifying cell subclusters with shared characteristics between ICC and normal tissues. Notably, we characterized a distinct C7-E-T subcluster that exhibited high expression of CXCR4 and BPTF, markers associated with cancer stem cells (CSCs). Further investigation revealed that the MIF intercellular signaling pathway promotes the progression of ICC by activating intracellular signals in the MYC pathway. This study highlights the dysregulation of intercellular signaling pathways within tumor clusters, which influences the onset and progression of ICC. The cancer stem cell subpopulation (CXCR4hiBPTFhiE-T) exerts a significant influence on ICC progression by secreting relevant signaling molecules via the MIF signaling pathway. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17102-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28caecb5ae60429cc1067831e07cb74217728059" target='_blank'>
              Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Xiaowei Gu, Fuzhong Liu, Junfeng Zhang, Nigeerayi Nuermaimaiti, Wenjia Guo, Xiaogang Dong
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of kidney renal clear cell carcinoma (KIRC) exhibits complex dynamics among immune, stromal, and malignant cells that drive immune escape (IE) mechanisms and influence clinical outcomes. Through single-cell RNA sequencing and high-dimensional weighted gene co-expression network analysis (hdWGCNA), we identified a crucial IE-related gene module most strongly associated with KIRC progression. Partitioning Around Medoids (PAM) clustering delineated two distinct IE patterns, with pattern one demonstrating prolonged patient survival. Employing advanced machine learning (ML) algorithms, we identified Cathepsin S (CTSS) as the most pivotal tumor suppressor, with elevated CTSS expression consistently predicting improved survival across multiple independent cohorts. Functional analyses revealed significant enrichment of CTSS in immune-regulatory pathways, including B/T cell activation and inflammatory activity. Mutation profiling uncovered distinct genomic alterations in 5q35.3 among CTSS-high tumors, while drug response prediction identified eight potential therapeutic agents (e.g., Navitoclax, Ibrutinib) exhibiting enhanced efficacy in these patients. Notably, CTSS expression strongly correlated with immune cell infiltration and established immunotherapy biomarkers, supporting its dual role as both a prognostic indicator and predictor of immune response. This study provides mechanistic insights into IE in KIRC and positions CTSS as a promising biomarker and therapeutic target for precision immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03267-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6364c60b20a894510d01361e9f698e8acfc1b2ab" target='_blank'>
              CTSS in the tumor microenvironment links immune escape and immunotherapy sensitivity in kidney renal clear cell carcinoma
              </a>
            </td>
          <td>
            Hanjing Zhou, Jun Ying, Xuchun Xu, Jian Huang
          </td>
          <td>2025-07-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Uterine leiomyosarcoma (ULSA) is a highly aggressive gynecologic malignancy characterized by early metastasis, profound immunosuppression, and resistance to conventional therapies, including immune checkpoint blockade (ICB). The intricate tumor microenvironment (TME) and cellular heterogeneity driving its progression and therapy resistance remain poorly defined. Methods We performed single-cell RNA sequencing (scRNA-seq) on metastatic lesions (pelvic cavity, rectum, peritoneum, bladder) from a treatment-naïve ULSA patient and compared them to normal uterine myometrium, MMM (n=5). Integrated analyses included cellular composition mapping, copy number variation (CNV) assessment, pseudotemporal trajectory reconstruction, cell-cell communication inference, functional enrichment, and validation via multiplex immunofluorescence (mpIF). Survival correlations were assessed using the TCGA-SARC cohort. Results In this study, the main finding is that the tumor microenvironment (TME) has a strong immunosuppressive effect. Firstly, its characteristic is exhausted CD8+T cells. This study found that as time progresses, the initial cell markers (CCR7, MAL) gradually disappear, while the exhaustion markers (LAG3, HAVCR2, TIGIT) are enriched. This is associated with poor prognosis. Secondly, the M2-polarized macrophages are mainly composed of M2-like tumor-associated macrophages (TAMs) with tumor-promoting characteristics (CD163, FTH1, FTL, TIMP1), and there is a polarization from M1 to M2. Finally, the immature, tumor-promoting N2 neutrophils (CD15+EDARADD+) enriched in the metastatic foci are associated with poor prognosis. The cell communication involves the interaction of MIF-(CD74+CD44) between T/B cells, as well as the role of the CXCL8 signaling axis in promoting angiogenesis, TAM polarization, and immunosuppression. Conclusion For the first time, a comprehensive single-cell map of ULSA was constructed, depicting a metastasis-susceptible cell subset (U11-EDARADD) and an extremely immunosuppressed tumor microenvironment dominated by depleted CD8+T cells, M2 macrophages and N2 neutrophils. These features shed light on the underlying mechanisms of chemotherapy resistance and immunotherapy failure. The biomarkers identified here (EDARADD, CLDN10, TMIGD2) as well as the dysregulated pathways (TGF-β, angiogenesis, MIF signaling) provide possible targets for future development of combined immunotherapy strategies against this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15b01f016db1d6adc66dea5efac6b72384978f8f" target='_blank'>
              Single-cell profiling delineates the tumor microenvironment and immunological networks in patient-derived uterine leiomyosarcoma
              </a>
            </td>
          <td>
            Yi Guo, Dongsheng Shen, Yuhang Xiao, Chenghao Wu, Meiyi Chen, Lina Yang, Huaifang Li, Xiaowen Tong, Rujun Chen, Fang Li
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Summary/Abstract Glioblastoma stem cell (GSC) cultures are initiated from glioblastoma (GBM) surgical resection tissue. They can capture and propagate key GBM primary tumor molecular and cellular features. We have deeply characterized four isocitrate dehydrogenase (IDH)-expressing (or IDH+) GSC cultures from unrelated adults to serve as cellular models for the majority of adult primary GBM. We demonstrate that GSC cultures can be continuously propagated in defined, serum-free media and 5% oxygen without requiring specialized growth substrates; have well-defined genomic and mtDNA variants and gene/protein expression profiles; and highly reproducible dose-survival curves when treated with the GBM standard-of-care therapies of ionizing radiation (IR) and temozolomide (TMZ). We also illustrate how expressed lentiviral barcodes, mtDNA variants and single cell gene expression profiling can be used to define and track cellular heterogeneity over 40 days after IR treatment. These well-characterized IDH+ GSC cultures can support many high throughput in vitro assay formats, including xenograft, organoid and other GBM disease modeling protocols. They should prove a useful resource to better understand GBM biology, and to identify new and more effective GBM therapies and treatment regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b852883dfd3891f1795ac6382d7546d2b8963f5d" target='_blank'>
              Cellular heterogeneity and therapeutic response profiling of human IDH+ glioma stem cell cultures
              </a>
            </td>
          <td>
            N. Chambwe, Scott R. Kennedy, Brendan F. Kohrn, Pavlo Lazarchuk, Mario Leutert, Guangrong Qin, Bahar Tercan, M. Sanchez-Contreras, Weiliang Tang, Jerome J. Graber, Patrick J. Paddison, Judit Villén, Ilya Shmulevich, Raymond J Monnat
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Single-cell transcriptomics has significantly advanced our ability to uncover the cellular heterogeneity of tumors. A key challenge in single-cell transcriptomics is identifying cancer cells and, in particular, distinguishing them from non-malignant cells of the same cell lineage. Focusing on features that can be measured by single-cell transcriptomics, this review explores the molecular aberrations of cancer cells and their observable readouts at the RNA level. Identification of bona fide cancer cells typically relies on three main features, alone or in combination: i) expression of cell-of-origin marker genes; ii) inter-patient tumor heterogeneity; iii) inferred copy-number alterations. Depending on the cancer type, however, alternative or additional features may be necessary for accurate classification, such as single-nucleotide mutations, gene fusions, increased cell proliferation, and altered activation of signaling pathways. We summarize computational approaches commonly applied in single-cell analysis of tumoral samples, as well as less explored features that may aid the identification of malignant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86abedac67c16f4a560937fab77dcbb2e53869a5" target='_blank'>
              Identification of malignant cells in single-cell transcriptomics data
              </a>
            </td>
          <td>
            M. Andreatta, J. Garnica, Santiago J. Carmona
          </td>
          <td>2025-08-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="High-grade serous tubo-ovarian cancer (HGSC) is marked by substantial inter- and intra-tumor heterogeneity. The tumor microenvironments (TME) of HGSC show pronounced variability in cellular make-up across metastatic sites, which is linked to poorer patient outcomes. The influence of cellular composition on therapy sensitivity, including chemotherapy and targeted treatments, has not been thoroughly investigated. In this study, we examined the premise that the variations in cellular composition can forecast drug efficacy. Using a high-throughput 3D in vitro tumoroid model, we assessed the drug responses of 23 distinct cellular configurations of tumoroids comprised of OVCAR3 HGSC cells, mesenchymal stem cells, HUVEC endothelial cells, and U937 monocytes to an assortment of five therapeutic agents, including carboplatin and paclitaxel. We identified that the overall pooled viability in response to these five drugs was highest among tumoroid compositions that contained a large number of myeloid cells, whereas the most sensitive tumoroids to these agents were comprised of only cancer cells. Additionally, we found that the “mesenchymal tumoroids” containing 400 or more mesenchymal stem cells were more sensitive to carboplatin than paclitaxel. By amalgamating our experimental findings with random forest machine learning algorithms, we assessed the influence of TME cellular composition on treatment reactions. Our findings reveal notable disparities in drug responses correlated with tumoroid composition, underscoring the significance of cellular diversity within the TME as a predictor of therapeutic outcomes. This research establishes a foundation for employing human tumoroids with varied cellular composition as a method to delve into the roles of stromal, immune, and other TME cell types in enhancing cancer cell susceptibility to various treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49e1984d98981fa778612856dadda5f531075bc3" target='_blank'>
              Cracking the code: predicting tumor microenvironment enabled chemoresistance with machine learning in the human tumoroid models
              </a>
            </td>
          <td>
            Michael Bregenzer, P. Mehta, Kathleen M Burkhard, Geeta Mehta
          </td>
          <td>2025-08-19</td>
          <td>Npj Biomedical Innovations</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Significance Androgen deprivation therapy (ADT) is a mainstay in prostate cancer (PCa) treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer directions for PCa modeling and pave the way toward therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b171b1d175becacb6af2e7a071be6d1fa5ee752f" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2025-08-25</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a tumor microenvironment (TME) composed of a dense extracellular matrix, cancer-associated fibroblasts (CAFs), vasculature, neural elements, and immune cell populations. This complex network promotes tumor proliferation, invasion, metastasis, and resistance to immunotherapy and chemotherapy. The microenvironmental characteristics of the various PDAC subtypes are discussed in this review. And we examines the role of cancer cells in the TME, highlighting their ability to manipulate stromal components to serve as collaborators in tumor progression. Furthermore, we explored the formation mechanism of the immunosuppressive microenvironment in PDAC, paying attention on Inflammation and intrinsic genetic alterations, the regulatory effect of metabolic reprogramming, the contribution of CAFs and the role of immune cells in cancer cell metastasis. This review shows the role of soluble molecules and exosomes in facilitating PDAC progression and immune evasion within the microenvironment. In conclusion, we outline the novel therapeutic strategies that focus on the interaction between cancer cells and their microenvironment, with the objective of offering new insights for future precision medical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e8295ec7fa4434bd00f48ee0333dca0ffe9c46" target='_blank'>
              Domestication and feedback: bidirectional hijacking in pancreatic ductal adenocarcinoma microenvironment
              </a>
            </td>
          <td>
            Yuxi Qiao, Haobo Yin, Yong Zhang, Jingdong Zhang, Qian Dong
          </td>
          <td>2025-08-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Malignant rhabdoid tumors (MRTs) are aggressive pediatric cancers with poor outcomes. MRTs exhibit low tumor mutational burden, yet recent studies reported immune cell infiltration. Here, we used spatial transcriptomics and multi-omic profiling to study immune cell infiltration in MRT samples. We observed a diverse repertoire of candidate tumor antigens (TAs), IRF1 signaling activity and elevated expression of antigen processing and presentation genes, strongly associated with a “hot” tumor immune microenvironment (TIME). Upregulation of factors involved in skeletal muscle development was observed in some MRTs with higher CD8 + T cell infiltration and the ratio of M1/M2 macrophages was positively correlated with cytotoxic lymphocyte infiltration. We identified genes, such as SPP1, preferentially expressed in tumor-associated macrophages (TAMs) and noted the MITF regulatory network appeared active in the M2-like TAMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/248c19c84a73e2abe2ce78fd660b38edcf7c75f9" target='_blank'>
              Spatial and multi-omic profiling reveals genes and pathways associated with cytotoxic lymphocyte infiltration in malignant rhabdoid tumor
              </a>
            </td>
          <td>
            Marco A. Marra, Dan Jin, Joshua Dubland, Elizabeth Mullen, James Geller, J. Bush
          </td>
          <td>2025-08-19</td>
          <td>Research Square</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) progression is driven by a complex interplay of immunosuppressive and inflammatory mechanisms involving various cell types, functional states, and extracellular milieu. To unravel this intricate relationship, we took a systems biology approach using high resolution spatial distributions of multiple key markers in clinically diverse CRC tumors. We employed Multiplexed Ion Beam Imaging (MIBI-TOF) to capture 65 protein markers across 3,700 fields of view from 250 CRC tumors. This allowed us to deeply characterize the phenotypes and functional states of 5.7 million cells. To incorporate extracellular geography information, we developed a computational approach named CellExt. This quantified and modeled the gradated abundance of key structural and functional proteins found extracellularly, such as fibronectin, granzymeB and vimentin, from each cell boundary to a 40-micron distance in pixelwise steps. CellExt detected a distinct immunosuppressive modality characterized by extracellular PDL1, TNF-related apoptosis-inducing ligand (TRAIL), and matrix metalloproteinase-9 (MMP9). Investigating the cell types involved led us to map the potential receivers and emitters by linking extracellular trends to cellular enrichment and gene expression. We found that Pankeratin-high epithelial cells were acting as TRAIL receivers with downstream activation of non-canonical pro-tumorigenic signaling through S6 phosphorylation. In parallel, we found that T cell suppressive CD40+ antigen-presenting B cells with PD1 expression were acting as MMP9 emitters. Additionally, CellExt was able to identify two spatially segregated inflammatory modalities: IFNgamma cytokine with mucin protein, MUC5AC and prostaglandin synthase, cyclooxygenase-2 with extracellular matrix component, Collagen1A1. This suggests a strong inflammatory role for Collagen1A1 in CRC. We then contextualized our identified modalities by genetic background, tumor stage and CMS subtype of CRC tumors. Taken together, our approach revealed novel axes of immunosuppression and inflammation and puts forward potential extracellular targets for therapeutic intervention in specific CRC patient subsets.



 Reema Baskar, Vairavan Lakshmanan, Audrey Lee, Wilfred Wong, Dane Bagaoisan, Yuhan Peng, Marc Bosse, Ferda Filiz, Christine Fullaway, Sean Bendall, Iain Beehuat Tan, Shyam Prabhakar. Spatial Profiling of Colorectal Cancer Extracellular Milieu Reveals Novel Axes of Immunosuppression and Inflammation [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P38.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18eea1ae1312b735cfb9925fbcce433bb07c964b" target='_blank'>
              Abstract P38: Spatial Profiling of Colorectal Cancer Extracellular Milieu Reveals Novel Axes of Immunosuppression and Inflammation
              </a>
            </td>
          <td>
            Reema Baskar, Vairavan Lakshmanan, Audrey Lee, Wilfred Wong, Dane Bagaoisan, Yuhan Peng, Marc Bosse, Ferda Filiz, C. Fullaway, Sean C. Bendall, Iain Beehuat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The tumor microenvironment (TME) consists of tumor cells, stromal cells, infiltrating immune cells and non-cellular components such as extracellular matrix, blood vessels and a wide variety of secreted proteins. Evidence shows that beyond supporting tumor growth, the TME also promotes tumor cell proliferation and invasion and contributes to treatment resistance, ultimately affecting patient prognosis. Cell-to-cell communication within the TME is driven by secreted proteins such as cytokines, chemokines, growth factors and interferons, which are produced not only by tumor cells but also by various stromal cells and immune cells. These proteins form a complex signaling network that promotes tumor cell proliferation and invasion and enables tumors to evade innate and adaptive immune responses. Antibody arrays are a technology that can simultaneously screen hundreds of secreted proteins in complex biological samples, aiding in the exploration of this complex signaling network. By combining high-throughput multiplex immunoassays such as antibody arrays with cellular and molecular biology techniques, researchers have uncovered complex regulatory mechanisms of cytokine networks within the TME. The present review summarized recent findings on the communication between tumor cells and the TME, as well as key secreted proteins essential for tumor progression and the development of therapeutic resistance. In addition, it discusses how high-throughput antibody arrays contribute to our understanding of regulatory networks of secreted proteins in the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b84b1200ba5d18e4e2abf23d107f5f3c2ed9e0f6" target='_blank'>
              Unraveling tumor cell-tumor microenvironment crosstalk through antibody array technologies (Review)
              </a>
            </td>
          <td>
            Yanlin Wang, Shuhong Luo, Hua Dong, Ruo-Pan Huang
          </td>
          <td>2025-08-18</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Retinoblastoma is the most common intraocular malignant tumor in childhood. Although current treatments offer a high survival rate, treatment toxicity, tumor relapse, and treatment resistance require a deeper understanding of the disease mechanisms to develop adapted therapies. Microscopically, this tumor is characterized by different states of differentiation and proliferation, ranging from poorly differentiated to well-differentiated retinoblastoma. Retinocytoma, on the other hand, is a benign non-proliferative tumor. Recent next-generation multi-omics analyses classified retinoblastoma tumors in subtypes 1 or 2, subtype 2 presenting a later age of onset, more genetic alterations, and higher metastatic potential. In parallel, several single-cell transcriptomics studies demonstrated intratumoral heterogeneity. However, mapping the different cell populations directly on the tumor and comparing histological features and molecular subtypes remains an unmet need. Methods Spatial transcriptomics was used to characterize a primary enucleated case with two histologically distinct areas. The whole transcriptomic profile of sixteen regions of interest, covering the two differentiation patterns of the tumor and the non-tumoral retina, was analyzed. Results The clustering of the regions of interest based on whole transcriptome data correlated with the histological description: cluster 1 (6 regions of interest) corresponded to highly differentiated areas and cluster 2 (6 regions of interest) to poorly differentiated areas. They showed enrichment for phototransduction and proliferation respectively, confirmed by immunohistochemistry for markers of these pathways. The publicly available molecular signatures of the two retinoblastoma subtypes categorized our regions of interest into two groups, which correlated perfectly with histological observation and transcriptomic profiles. Further analysis of the expression of specific senescence markers in the well-differentiated area did not support the diagnosis of retinocytoma as it did not confirm the expected up-regulation seen in this tumor type. Conclusions This study demonstrates a strong correlation between histological observation and molecular profiling, representing the first mapping of retinoblastoma composed of adjacent molecular subtypes 1 and 2. It also highlights the diagnostic ambiguity between retinocytoma and well-differentiated non-proliferative subtype 1 retinoblastoma and emphasizes the need for specific biomarkers to differentiate these two tumor types. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14814-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/599058c9510122a1b04d9c9b60c21f33e8817b4e" target='_blank'>
              Spatial transcriptomics of retinoblastoma: a visual window on intra-patient heterogeneity
              </a>
            </td>
          <td>
            A. Moulin, J. Thevenet, L. Mazzeo, S. Tissot, C. Stathopoulos, F. L. Munier, A. Berger
          </td>
          <td>2025-09-02</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumorigenesis is a microevolutionary process in which heterogeneous tumor cells adapt within a complex microenvironment. Current tumor omics analyses often focus exclusively on tumor samples, overlooking the valuable insights that normal tissues can provide. To address this gap, we introduce VaDTN (Variational Autoencoder–Derived Tumor-to-Normal), a pan-cancer framework that integrates transcriptomic data from both tumor and normal samples into a unified latent space. By measuring each tumor’s “distance” from a normal reference within this latent space, VaDTN reveals subtle molecular shifts linked to tumor evolution and heterogeneity. We applied VaDTN to six representative cancers (SKCM, BRCA, LIHC, LUSC, STAD, and PAAD), identifying distinct subtypes characterized by unique transcriptional profiles. Notably, four cancer types (SKCM, BRCA, LIHC, and STAD) displayed significant survival stratification based on these subtype groupings, underscoring the clinical relevance of the distance-based approach. This reference-centered perspective thus provides a refined lens for dissecting intra-tumor diversity and guiding potential precision oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dc636c3386b69302e143969833b154f54e564be" target='_blank'>
              Novel cancer subtyping method guided by tumor-normal sample in latent space of transcriptomic variational autoencoder
              </a>
            </td>
          <td>
            Hongzhi Wang, Yu Zhang, Dandan Zhang, Min Luo
          </td>
          <td>2025-07-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Trachea squamous cell carcinoma (TSCC) is a subtype of lung cancer. A thorough investigation of the tumor microenvironment of TSCC is crucial for the development of cancer therapeutics and predicting clinical responses. In this study, we utilized single-cell RNA sequencing to analyze seven TSCC samples (including five malignant and two non-malignant samples) and obtained 70,682 high-quality cells. Based on the expression levels of marker genes, we identified 7 major cell types within the samples. By comparing malignant samples that received chemotherapy with those that did not, we identified critical transcriptional regulators responsible for T cell state transition in response to chemotherapy. Additionally, we found specific transcriptional regulators and differentially expressed genes between malignant and non-malignant groups. We identified more particularly abundant specifical intercellular communication in the malignant sample group and that may significantly influence the progression and spread of cancerous cells. Overall, our study provides the first single-cell atlas that comprehensively explains TSCC development and chemotherapy effects, thereby laying a new molecular foundation for therapeutic research in TSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f4d5fafc26e824adc29e95477c2a4bfd6ebf074" target='_blank'>
              Comprehensive single-cell sequencing reveals the tumor microenvironment and tumor-specific characteristics in trachea squamous cell carcinoma
              </a>
            </td>
          <td>
            HongWu Wang, Hongli Li, Heng Zou, Han Meng, Yan Liu, Chengjun Ban, Weixia Yu, Miao Cheng, Jun Teng
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b8aa36b9705d21beef4a1a46329e47825192017" target='_blank'>
              Comparative single-cell lineage bias in human and murine hematopoietic stem cells.
              </a>
            </td>
          <td>
            I. Shamie, Meghan Bliss-Moreau, J. Lee, Ronald Mathieu, Harold M. Hoffman, Bob Geng, Nathan E Lewis, Y. P. Zhu, Ben A Croker
          </td>
          <td>2025-07-17</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Patient-derived tumor xenograft (PDX) models serve as powerful tools in oncology research owing to their ability to capture patient-specific tumor heterogeneity and clinical behavior. However, the conventional matrices derived from murine tumors, commonly used to generate PDX models, suffer from key limitations such as lack of tissue specificity, high production costs, and inconsistent batch quality. In response, our study investigates the use of decellularized liver extracellular matrix (Liver ECM) as a biomimetic alternative that more accurately recapitulates the native hepatic microenvironment. We demonstrate that Liver ECM, enriched with liver-specific biochemical cues, enables robust engraftment, growth, and metastasis of patient-derived hepatocellular carcinoma cells in both subcutaneous and orthotopic PDX models. Notably, orthotopic models established with Liver ECM exhibited enhanced metastatic behavior, particularly to the intestine, compared to those formed using conventional matrices. Transcriptomic analysis further revealed activation of key pathways associated with cancer progression, including angiogenesis, apoptosis, migration, and inflammation. Additionally, we extend the application of Liver ECM to patient-derived organoid xenografts, which showed improved tumorigenicity and retained pathophysiological features of the original tumor tissue. Together, these findings underscore the potential of liver-specific ECM as a superior platform for generating physiologically relevant PDX models and enhancing the translational relevance of preclinical cancer studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77773de20ab5035f09de4e444e6e5d2f39f36813" target='_blank'>
              Liver-Specific Extracellular Matrix Enables High-Fidelity Patient-Derived Hepatocellular Carcinoma Xenograft Models
              </a>
            </td>
          <td>
            Su Kyeom Kim, Jungho Bae, Mi Jeong Lee, Dai Hoon Han, Seung-Woo Cho
          </td>
          <td>2025-07-23</td>
          <td>Biomaterials Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) continues to be a leading cause of cancer-related deaths globally, primarily due to its late diagnosis and the complex nature of its tumor microenvironment (TME). Within this environment, cancer-associated fibroblasts (CAFs) play a crucial role in regulating NSCLC progression and therapeutic resistance. Recent advancements in single-cell and spatial technologies have uncovered significant heterogeneity among CAFs, revealing distinct subpopulations with varying cellular origins, phenotypes, and functions. Besides, the role of small extracellular vesicles (sEVs) in facilitating bidirectional communication highlights functional importance of CAF derived sEVs in mediating the crosstalk between cancer and TME, implicating the potential of CAF-sEVs to be un-invasive diagnostic biomarkers for NSCLC. Despite such new insights, challenges remain exist, particularly concerning the mechanisms that drive CAF plasticity, the interactions between specific CAF subsets with cancer cells or immune cells, and the translational significance of CAF-sEVs. In this review, we summarize the latest advances in our understanding of CAF heterogeneity, the functional roles and clinical relevance of CAFs and their secreted sEVs in driving NSCLC, and the translational landscape of CAF-targeted strategies. By critically evaluating the most recent evidence, this review provides clinically relevant insights and highlights future directions for overcoming CAF-mediated barriers to therapy. Our aim is to facilitate the development of more precise, biomarker-driven, and safe approaches for targeting CAFs, ultimately improving personalized treatment and outcomes for patients with NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82d1bb104cbbe26ccaae9f632691df643466beac" target='_blank'>
              Hidden forces: the impact of cancer-associated fibroblasts on non-small cell lung cancer development and therapy
              </a>
            </td>
          <td>
            Ziheng Wu, Meilin Luo, Huiyi Hu, Zhijun Jiang, Yinan Lu, Zhi-Jie Xiao
          </td>
          <td>2025-07-25</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0289ee010aab237793e43a5fa9c9591075ed88ec" target='_blank'>
              A clinically relevant morpho-molecular classification of lung neuroendocrine tumours
              </a>
            </td>
          <td>
            A. Sexton-Oates, Emilie Mathian, Noah Candeli, Yuliya Lim, C. Voegele, A. Génova, Laurane Mangé, Zhaozhi Li, Tijmen van Weert, L. Hillen, R. Blázquez-Encinas, A. González-Pérez, Maike L. Morrison, Eleonora Lauricella, L. Mangiante, Lisa Bonheme, L. Moonen, G. Absenger, Janine Altmuller, Cyril Dégletagne, Vincent Cahais, G. Centonze, Amélie Chabrier, C. Cuenin, F. Damiola, V. Montpréville, J. Deleuze, Anne-Marie C. Dingemans, É. Fadel, N. Gadot, A. Ghantous, Paolo Graziano, Paul Hofman, V. Hofman, A. Ibáñez-Costa, S. Lacomme, Núria López-Bigas, M. Lund-Iversen, Massimo Milione, L. Muscarella, S. Pedraza-Arévalo, C. Perrin, G. Planchard, H. Popper, L. Roz, A. Sparaneo, W. Buikhuisen, José van den Berg, M. Tesselaar, Jaehee Kim, Ernst Jan, M. Speel, S. Tabone-Eglinger, Thomas Walter, Gavin M. Wright, Justo P Castaño, Lara Chalabreysse, Liming Chen, Christophe Caux, Marco Volante, Nicolas Girard, J. Vignaud, Esther Conde, A. Mansuet-Lupo, Luka Brčić, Giuseppe Pelosi, Mauro Papotti, Sylvie Lantuéjoul, J. Derks, Talya Dayton, N. Alcala, M. Foll, L. Fernandez-Cuesta
          </td>
          <td>2025-07-18</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Simple Summary Cutaneous T-cell Lymphomas (CTCLs) refer to a group of rare cancers that affect immune cells in the skin and often the blood. Their exact cause is not entirely understood, and they are difficult to diagnose because they can appear similar to common, non-malignant skin conditions. Recent advances in single-cell RNA sequencing have made it possible for researchers to explore these tumors at an unprecedented level of detail, revealing important differences between CTCL subtypes in how they interact with the immune system. By analyzing these patterns, researchers can begin to explain differences in disease behavior and identify targets for new therapies. This summary highlights how single-cell technologies are transforming our understanding of CTCL and may pave the way for more precise diagnosis and treatment strategies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/504bc3b3f8515fbfaaeba0942215a380977f71da" target='_blank'>
              An Update on Single-Cell RNA Sequencing in Illuminating Disease Mechanisms of Cutaneous T-Cell Lymphoma
              </a>
            </td>
          <td>
            Sara Suhl, Alexander Kaminsky, Caroline Chen, B. Lapolla, Maggie H. Zhou, Joshua Kent, Abigail Marx, I. Nebo, G. Ramush, Sophia Luyten, Yoni Sacknovitz, Julie Sung, Christina M. Bear, Celine M. Schreidah, Alejandro A. Gru, L. J. Geskin
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have revolutionized cancer therapy, yet most patients fail to achieve durable responses. To better understand the tumor microenvironment (TME), we analyze single-cell RNA-seq (~189 K cells) from 36 metastatic melanoma samples, defining 14 cell types, 55 subtypes, and 15 transcriptional hallmarks of malignant cells. Correlations between cell subtype proportions reveal six distinct clusters, with a mature dendritic cell subtype enriched in immunoregulatory molecules (mregDC) linked to naive T and B cells. Importantly, mregDC abundance predicts progression-free survival (PFS) with ICIs and other therapies, especially when combined with the TCF7 + /– CD8 T cell ratio. Analysis of an independent cohort (n = 318) validates mregDC as a predictive biomarker for anti-CTLA-4 plus anti-PD-1 therapies. Further characterization of mregDCs versus conventional dendritic cells (cDC1/cDC2) highlights their unique transcriptional, epigenetic (single-nucleus ATAC-seq data for cDCs from 14 matched samples), and interaction profiles, offering new insights for improving immunotherapy response and guiding future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/065ebdb359d600d41a58e73e81b2ef0cb4692c57" target='_blank'>
              Mature and migratory dendritic cells promote immune infiltration and response to anti-PD-1 checkpoint blockade in metastatic melanoma
              </a>
            </td>
          <td>
            Jie-Ying Yang, Cassia Wang, Doris Fu, Li-Lun Ho, K. Galani, Lee Chen, Jose Gonzalez, Jolene Fu, Amy Y. Huang, Dennie T. Frederick, Liang He, Mukta Asnani, Rahul Tacke, Emily J. Robitschek, Sandeep K Yadav, Wentao Deng, Kelly P. Burke, T. Sharova, Ryan J. Sullivan, Sarah A. Weiss, Kunal Rai, David Liu, Genevieve M. Boland, M. Kellis
          </td>
          <td>2025-09-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Gliomas represent the most frequent primary brain tumors in adults and are marked by considerable molecular and clinical heterogeneity. The 2021 WHO classification of central nervous system tumors identifies three major adult-type diffuse glioma subtypes: astrocytoma IDH-mutant, oligodendroglioma IDHmutant with 1p/19q-codeletion, and glioblastoma IDH-wildtype.1 Despite ongoing advances, treatment resistance and poor prognosis remain major challenges, largely due to intratumoral heterogeneity and the complexity of the tumor microenvironment.2 A promising yet underexplored avenue for patient-centered glioma profiling is the tumor secretome, which includes soluble factors, extracellular vesicles, and metabolites released by both neoplastic and stromal cells.3 Existing studies rely on in vitro models that fail to recapitulate the complexity of tumor-host interactions. To bridge this gap, we established an innovative, standardized ex vivo protocol for secretome isolation directly from glioma tissue. The protocol preserves microenvironmental cues from the operating room to laboratory analysis, ensuring spontaneous secretome release under controlled conditions. Feasibility and performance were assessed in a preliminary cohort of seven adult diffuse glioma patients. Transmission electron microscopy confirmed tissue structural integrity. Proteomic profiling identified 84 cancer-associated proteins within the secretome. Additionally, metabolomic analysis revealed a distinctive signature of small molecules. Notably, glioblastoma IDH-wildtype samples exhibited increased lactate levels, consistent with hypoxia-driven metabolism, and decreased N-acetylaspartate, indicating neuronal loss. Overall, this ex vivo secretome protocol offers a robust and physiologically relevant platform for glioma characterization, with potential applications in biomarker discovery, patient stratification, and the development of personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea6096ea6e8da3b1feda45c80168b298c559e43" target='_blank'>
              P64 | EX VIVO EXTRACTION OF GLIOMA SECRETOME: A NOVEL MICROENVIRONMENT-BASED PROTOCOL FOR METABOLIC AND MOLECULAR PROFILING
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Gliomas are aggressive brain tumors whose infiltrative growth is mediated by intercellular crosstalk. Exosomes, small extracellular vesicles, play a key role in cell–cell communication but are difficult to visualize using conventional microscopy. Performing immunostaining for CD63, a known exosome marker, and using STED microscopy, we demonstrate exosome secretion in primary glioma cells. Applying mathematical deconvolution, we enhance the contrast and resolution for in-depth analysis of STED images. We identify CD63-positive cellular footprints and exosome deposits in the extracellular space. Quantitative analysis shows CD63-positive exosomes ranging 36.55-157.06 nm in size. CD63/actin co-staining demonstrates different actin polymerization states associated with exosomes. In conclusion, STED microscopy coupled with immunostaining allows exosome primary characterization at the single-vesicle level in the cellular spatial context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19403b747c6c798bf26375a3c2009af4ac5fca44" target='_blank'>
              Super-resolution microscopy reveals glioma cell footprints and exosome deposits
              </a>
            </td>
          <td>
            Stefania Petrini, Frédéric Eghiaian, Valentina Apollonio, Giulia Pericoli, Maria Vinci
          </td>
          <td>2025-07-24</td>
          <td>Cell Adhesion & Migration</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b07b858c6fd1cb8139cba6229ad20ebbdfab9d" target='_blank'>
              Multicellular tumor-stromal interactions recapitulate aspects of therapeutic response and human oncogenic signaling in a 3D disease model for H3K27M-altered DIPG.
              </a>
            </td>
          <td>
            Meenakshi Upreti, Astgik Petrosyan, M. Thornton, A. Hovsepyan, G. E. Fernandez, David S. Koos, S. Byrum, Samuel G Mackintosh, Jacob K Al-Husseini, Tania Porras, Joseph Ha, A. Tackett, Miqin Zhang, M. Johal, Anat Erdreich-Epstein, Susan Durham, Mark D Krieger, Ashley S Margol, Brendan H. Grubbs, T. Chambers, S. Asgharzadeh, Rex A. Moats, Peter A Chiarelli
          </td>
          <td>2025-09-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Recent studies have explored the composition of tumor microenvironment (TME) in diffuse large B-cell lymphoma (DLBCL) However, cell-to-cell interactions, along with the spatial organization of DLBCL TME and their impact on patient outcomes, have remained poorly characterized. We ap-plied multiplexed immunofluorescence, cell phenotyping, and neighborhood analysis to investigate 1,218,756 single cells in 99 samples from patients with primary DLBCL. We identified 17 cell pheno-types and 10 recurrent cellular neighborhoods (RCNs) across samples, subdividing DLBCLs into immune-poor areas and areas with diverse immune cell infiltrates. Avoidance of B cells and PD-1+ T cells was associated with less aggressive clinical characteristics and favorable survival. Like-wise, the proximity of CD8+ T cell-rich and immune-poor RCNs translated to favorable, and the proximity of PD-L1+ B cell-rich and CD8+ T cell-rich RCNs to unfavorable patient outcomes. Our findings provide insights into the spatial interactions and organization of DLBCL TME with implica-tions for patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8a36543383d1dc42cdb78f6b491ab4c213fbf1a" target='_blank'>
              Spatial Analysis of the Tumor Microenvironment in Diffuse Large B-Cell Lymphoma Reveals Clinically Relevant Cell Interactions and Recurrent Cellular Neighborhoods.
              </a>
            </td>
          <td>
            M. Autio, S. Leivonen, L. Meriranta, M. Karjalainen-Lindsberg, T. Pellinen, S. Leppä
          </td>
          <td>2025-08-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) tumours are classified by pathology into several distinct subtypes. Gene expression profiling has revealed transcriptional heterogeneity across NEPC, but this is rarely considered in the context of variation between pathologies. Diagnosis typically relies on immunohistochemical markers (CHGA, SYP, NCAM1) and genomic alterations in RB1, PTEN and TP53. We hypothesized that NEPC pathologies have unique transcriptional features. Single-cell RNA sequencing of 18 632 tumour cells from nine patient-derived xenograft models representing five pathologies (small-cell and large-cell neuroendocrine carcinomas, focal neuroendocrine differentiation (Focal NED), low-grade neuroendocrine and amphicrine) demonstrated pathway-specific enrichment. Focal NED and amphicrine tumours exhibited cellular subpopulations enriched for KRAS, IL2-STAT5 and TNF signalling pathways, absent in small- and large-cell carcinomas, which were instead enriched for Myc and E2F pathways. Furthermore, focal NED cells exhibited minimal clonal divergence from adjacent adenocarcinoma cells, while small cell carcinoma cells were clonally distinct. These data underscore significant transcriptional variation among NEPC pathologies, highlighting focal NED's unique biological context and its clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c9094a51a0844d2931b4655b974ca910515f91f" target='_blank'>
              Single-cell transcriptomics redefines focal neuroendocrine differentiation as a distinct prostate cancer pathology.
              </a>
            </td>
          <td>
            Rosalía Quezada Urban, S. Keerthikumar, A. Clark, Hong Wang, B. Phipson, Andrew Bakshi, Andrew Ryan, Heather Thorne, R. Taylor, Mitchell G. Lawrence, G. Risbridger, Roxanne Toivanen, D. L. Goode
          </td>
          <td>2025-07-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Spatial transcriptomics has advanced our understanding of cellular heterogeneity at single-cell resolution. Here, we assess the suitability of in situ sequencing (ISS) for analyzing formalin-fixed, paraffin-embedded (FFPE) postmortem human brain tissue. A key challenge in ISS data analysis is optimizing transcript allocation while minimizing misallocation, particularly in the morphologically complex central nervous system (CNS). We compared geospatial methods using nuclear and expanded nuclear boundaries for segmentation and transcript allocation. While overall cell-type proportions remained comparable, transcript allocation methods affected specific cell types, including microglia, neurons, and neurovascular cells. To enhance specificity, we integrated fluorescent imaging data targeting 18S RNA and IBA1 protein to direct transcript allocation toward RNA-rich cells (e.g., neurons) and microglia, respectively. We demonstrate how this approach, paired with secondary allocation of transcripts outside imaging masks, improved both the number of microglia detected and the specificity of microglial transcripts assigned. Our method offers a flexible and efficient strategy for targeted transcript allocation based on cellular morphology, optimizing CNS cell segmentation in FFPE-preserved human brain tissue.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/865ca3f5d716d0f571d46550eae97052916e2053" target='_blank'>
              Beyond the nuclear border: single-cell analysis of in situ sequenced human brain tissue using cellular features
              </a>
            </td>
          <td>
            J. Kotah, Thomas Rust, H. R. V. van Weering, Janneke Bosma, Amber L. Woudstra, S. Kooistra, B. Eggen
          </td>
          <td>2025-07-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Pediatric high grade gliomas are lethal evolutionary disorders with stalled developmental trajectories and disrupted differentiation hierarchies. We integrate transcriptional and algorithmic network complexity based perturbation analysis to elucidate gene expression patterns and molecular divergence between Diffuse Midline Gliomas and glioblastoma, revealing shared developmental programs steering cell fate decision making. Our complex systems approach supports the emerging paradigm that pediatric high grade gliomas are neurodevelopmental disorders with hybrid lineage identities and disrupted patterning. We identify dysregulated neurodevelopmental and morphogenetic signatures, alongside bioelectric and neurotransmitter signaling programs that alter synaptic organization, neuronal fate commitment, and phenotypic plasticity, regulating glioma phenotypic switching. Causal drivers and regulators of plasticity were predicted as both biomarkers and therapeutic targets, reinforcing the view that pediatric gliomas are disorders of cell fate decisions and collective cellular identity. Decoded plasticity signatures indicate a teleonomic bias toward neural progenitor or neuron like identities, while synaptic transmission gene enrichment supports neuron glioma interactions shaping fate trajectories. These findings advance differentiation therapy as a systems medicine strategy, discovering plasticity regulators to reprogram malignant fates toward stable lineages, offering complex systems based targets for cancer reversion and predictive, preventive, precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/788cce7ac9b2484c7dc3d6df46715fc3b5981d51" target='_blank'>
              Decoding Neurodevelopmental Dynamics in Pediatric High Grade Gliomas: Divergences in Transcriptomic Network Complexity from Healthy Neural Stem Cell Signatures
              </a>
            </td>
          <td>
            Abicumaran Uthamacumaran, Hector Zenil
          </td>
          <td>2025-08-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Metastatic breast cancer remains largely incurable, and the mechanisms driving the transition from primary to metastatic breast cancer remain elusive. We analyzed the complex landscape of estrogen receptor (ER)-positive breast cancer primary and metastatic tumors using scRNA-seq data from twenty-three female patients with either primary or metastatic disease. By employing single-cell transcriptional profiling of unpaired patient samples, we sought to elucidate the genetic and molecular mechanisms underlying changes in the metastatic tumor ecosystem. We identified specific subtypes of stromal and immune cells critical to forming a pro-tumor microenvironment in metastatic lesions, including CCL2+ macrophages, exhausted cytotoxic T cells, and FOXP3+ regulatory T cells. Analysis of cell-cell communication highlights a marked decrease in tumor-immune cell interactions in metastatic tissues, likely contributing to an immunosuppressive microenvironment. In contrast, primary breast cancer samples displayed increased activation of the TNF-α signaling pathway via NF-kB, indicating a potential therapeutic target. Our study comprehensively characterizes the transcriptional landscape encompassing primary and metastatic breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c30789b26912d007696198e3cb05def7ea34fbf8" target='_blank'>
              Single-cell RNA sequencing reveals different cellular states in malignant cells and the tumor microenvironment in primary and metastatic ER-positive breast cancer
              </a>
            </td>
          <td>
            F. Ozmen, T. Ozmen, A. Ors, M. Janghorban, M. Rames, Xi Li, A. Doe, F. Behbod, Gordon B. Mills, H. Mohammed
          </td>
          <td>2025-08-26</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Despite the availability of numerous approved immunotherapies for various cancers, durable progression-free survival remains relatively uncommon among patients with advanced cancer. As research into immunotherapy intensifies, the heterogeneity and complexity of the tumor microenvironment (TME) have emerged as critical determinants of treatment response and a major obstacle to understanding tumor resistance mechanisms. Recent advances in spatially resolved transcriptomics (SRT) enable transcriptome-wide measurement of gene expression while preserving essential spatial information, which supports the characterization of the features of the TME. This review outlines key challenges in cancer immunotherapy and emphasizes the importance of the TME in determining therapeutic efficacy. SRT strategies suitable is systematically introduced for TME profiling, comparing their relative advantages and limitations. Additionally, bioinformatics tools and methods used to reconstruct spatial tissue patterns are discussed from SRT data. Furthermore, how SRT is explored can enhance the understanding of the complexity and heterogeneity of the TME, facilitate the identification of novel therapeutic targets, and advance biomarker discovery to refine personalized treatment strategies for diverse cancer patients. Finally, future perspectives on the application of SRT in cancer immunotherapy and addressing the associated challenges are provided.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53e788cfc2c6f7fa7f419ffa32b647da6da2a8f3" target='_blank'>
              Spatially Resolved Transcriptomics: Revealing Tumor Microenvironment Heterogeneity to Advance Cancer Immunotherapy.
              </a>
            </td>
          <td>
            Dan Liu, Lu Xiao, Yuyuan Wu, Chensirong Yue, Mengmeng Li, Yuanpei Sun, Chao Yang
          </td>
          <td>2025-09-04</td>
          <td>Small methods</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Lung cancer remains a significant public health burden. One of the most personalized treatments uses a patient’s own tumor infiltrating lymphocytes (TILs), and TIL activity is also essential for immune checkpoint inhibitor (ICI) effectiveness. Responses to immunotherapies vary due to immune-suppressive tumor microenvironments (TMEs) and limited antigen presentation. In this study, we computationally examine cell-cell signaling and transcriptional activity using single-cell RNA sequencing of lung cancer treated by inhibiting methyltransferase EZH2. We show that EZH2 inhibition shifts the TME to immunogenic signaling patterns conducive to increased T cell response, including antigen presentation and homing. T cells also showed more stem-like phenotypes. Transcriptional activity was quieter with EZH2 inhibition but revealed better interferon response, altered myeloid and B cell differentiation, and apoptotic markers. Importantly, inferred EZH2 activity showed it could perform non-methyltransferase duties vital for T cell differentiation. These results indicate that EZH2 inhibition could improve immunotherapies for lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6d5b3471193acbd21fb76beb7e0f92d1e7a78da" target='_blank'>
              Single-cell profiling of EZH2-mediated immune signaling perturbations in NSCLC
              </a>
            </td>
          <td>
            Daniel R. Plaugher, Avery R Childress, Christian M. Gosser, Dave-Preston Esoe, Jinpeng Liu, C. Brainson
          </td>
          <td>2025-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is an aggressive and heterogeneous liver cancer with restricted therapy selections and poor diagnosis. Although there have been great advances in genomics, the molecular mechanisms essential to HCC progression are not yet fully implicit, particularly at the single-cell stage. This research utilized single-cell RNA sequencing technology to evaluate transcriptional heterogeneity, immune cell infiltration, and potential therapeutic targets in HCC. A detailed bioinformatics pipeline used in the experiment included quality control, feature selection, dimensionality reduction using Principal Component Analysis (PCA), Uniform Manifold Approximation and Projection (UMAP), and t-distributed stochastic neighbor embedding (t-SNE), clustering, differential gene expression, pseudotime trajectory inference, and immune cell profiling with GSEA and survival analysis examining potential biomarkers of survival. Key findings include the identification of 1178 differentially expressed genes (DEGs), with macrophage infiltration contributing to immune evasion. Notably, APOE and ALB are linked to a better prognosis, while XIST and FTL are associated with poor survival. The potential drug candidates include IGMESINE in the case of SERPINA1 and PKR-A/MITZ for APOA2 in the gene-drug interaction analysis. Graph Neural Network (GNN) is used to predict drug-gene interactions and rank potential therapeutic candidates. The model shows robust predictive performance (R²: 0.9867, MSE: 0.0581) and identifies important drug candidates, such as Gadobenate Dimeglumine and Fluvastatin, and describes repurposing opportunities in network analysis, enhancing computational drug discovery for novel treatments. This research sheds new light on HCC tumor evolution, immune suppression, and the potential drug target based on the viewpoint of the importance of single-cell approaches in liver cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e827b59658da5d3c8017bf844efafdcd1183fd" target='_blank'>
              Integrating single-cell RNA sequencing and artificial intelligence for multitargeted drug design for combating resistance in liver cancer
              </a>
            </td>
          <td>
            Houhong Wang, Youyuan Yang, Junfeng Zhang, Wenli Chen, Jingyou Dai, Changquan Li, Qing Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Spatial genomics has revolutionized breast cancer research by mapping gene expression within the tumor microenvironment, offering unprecedented insights into the molecular and cellular interactions that drive tumor progression and therapeutic response across luminal A, luminal B, HER2-positive, and triple-negative subtypes. Unlike traditional genomic methods, spatial genomics preserves tissue architecture, revealing subtype-specific TME features, such as immune-rich regions in triple-negative breast cancer and stromal-dense environments in HER2-positive tumors. This review explores spatial genomics technologies, including Visium, GeoMx, and MERFISH, which enable precise TME analysis, highlighting their role in identifying biomarkers like PD-L1 and TGF-β for targeted therapies and immunotherapies. By elucidating TME-driven resistance mechanisms, such as hypoxia in triple-negative tumors or stromal remodeling in HER2-positive cases, spatial genomics informs clinical trials and combination therapies, enhancing patient stratification and treatment efficacy. Despite its promise, challenges including limited resolution, complex data analysis, and high costs hinder clinical adoption. Emerging technologies, artificial intelligence, and multi-omics integration offer solutions, promising higher precision and scalability. This review underscores spatial genomics’ transformative potential to bridge research and clinical practice, paving the way for personalized breast cancer therapies tailored to the dynamic TME, with future advancements poised to redefine treatment paradigms and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5822524fe77c324fb45f43e4adc21fca11863f06" target='_blank'>
              Unraveling tumor microenvironments with spatial genomics: Implications for precision breast cancer therapy
              </a>
            </td>
          <td>
            Amina Suleiman Kamsalem, Obianujunwa Mercy Ubah, Tobiloba Philip Olatokun, Regina Omodolapo Oluwaniyi, Olivia Ajifa Philips, Damilola Samuel Ojo-Omoniyi, Omolola Lolade Oluwasakinju
          </td>
          <td>2025-07-30</td>
          <td>International Journal of Life Science Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive and heterogeneous brain tumor. Glioma stem-like cells (GSCs) play a central role in tumor progression, therapeutic resistance, and recurrence. Although immune cells are known to shape the GBM microenvironment, the impact of antigen-presenting-cell (APC) signature genes on tumor-intrinsic phenotypes remains underexplored. We analyzed both bulk- and single-cell RNA sequencing datasets of GBM to investigate the association between APC gene expression and tumor-cell states, including stemness and metabolic reprogramming. Signature scores were computed using curated gene sets related to APC activity, KEGG metabolic pathways, and cancer hallmark pathways. Protein–protein interaction (PPI) networks were constructed to examine the links between immune regulators and metabolic programs. The high expression of APC-related genes, such as HLA-DRA, CD74, CD80, CD86, and CIITA, was associated with lower stemness signatures and enhanced inflammatory signaling. These APC-high states (mean difference = –0.43, adjusted p < 0.001) also showed a shift in metabolic activity, with decreased oxidative phosphorylation and increased lipid and steroid metabolism. This pattern suggests coordinated changes in immune activity and metabolic status. Furthermore, TNF-α and other inflammatory markers were more highly expressed in the less stem-like tumor cells, indicating a possible role of inflammation in promoting differentiation. Our findings revealed that elevated APC gene signatures are associated with more differentiated and metabolically specialized GBM cell states. These transcriptional features may also reflect greater immunogenicity and inflammation sensitivity. The APC metabolic signature may serve as a useful biomarker to identify GBM subpopulations with reduced stemness and increased immune engagement, offering potential therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaf0983476db2efab2251dc274b40344cf2a61c3" target='_blank'>
              Elevated Antigen-Presenting-Cell Signature Genes Predict Stemness and Metabolic Reprogramming States in Glioblastoma
              </a>
            </td>
          <td>
            Ji-Yong Sung, K. Hwang
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The dysregulated and fibrotic tumor microenvironment of hepatocellular carcinoma (HCC) delays diagnosis and presents many complex signals that drive disease progression. To better recapitulate this microenvironment, we have enhanced our established protein microarray platform by integrating design of experiments (DoE) methodology with high-throughput cell microarray screening. This innovative approach systematically interrogates the intricate roles of matrix stiffness (spanning healthy and fibrotic conditions), extracellular matrix (ECM) composition, and protein concentration, while simultaneously examining their interdependent interactions. By leveraging DoE principles, we were able to explore 117 unique microenvironments on a single microscope slide, ultimately generating a comprehensive dataset of 234 different microenvironments without compromising statistical rigor. Our enhanced screening system enabled the identification of unique microenvironmental interactions critically significant in dictating cellular responses, including adhesion, survival, proliferation, epithelial-to-mesenchymal transition, and drug resistance markers. Utilizing advanced statistical techniques such as linear models and principal component analysis, we characterized phenotypic clusters defined by precise microenvironmental cues. This work presents a robust, high-throughput microarray screening system that comprehensively explores the contributions of 9 physiologically relevant extracellular matrix proteins and matrix stiffness in modulating cellular behavior and disease progression through a methodologically sophisticated and statistically sound approach. .">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d6d43b14b77f59b187769ee40b324e60d7e33b" target='_blank'>
              Enhanced combinatorial analysis of tumor cell-ECM interactions using design-of-experiment optimized microarrays.
              </a>
            </td>
          <td>
            H. Kimmel, Allison L. Paxhia, Zahra Adamji, Gregory H. Underhill
          </td>
          <td>2025-07-24</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy characterized by poor prognosis, strong resistance to therapy, and a dense immunosuppressive tumor microenvironment (TME). A small subset of cells known as cancer stem cells (CSCs), or tumor-initiating cells (TICs), are increasingly recognized as key contributors to tumor initiation, metastasis, immune evasion, and treatment failure. These cells are defined by their self-renewal capacity, plasticity, and resistance to chemotherapeutic and targeted therapies. Pancreatic cancer stem cells (PaCSCs) are maintained by specific surface markers (CD44, CD133, EpCAM, ALDH1A1) and regulated by stemness-associated signaling pathways such as Wnt/β-catenin, Notch, Hedgehog, and TGF-β. Their survival is further enhanced by metabolic reprogramming, including shifts between glycolysis and oxidative phosphorylation and the activation of ROS-detoxifying enzymes. Importantly, PaCSCs reside in specialized niches formed by hypoxia, cancer-associated fibroblasts (CAFs), tumor-associated macrophages (TAMs), and extracellular matrix (ECM) components that together shield them from immune clearance and promote therapeutic resistance. This review outlines the molecular features and functional roles of PaCSCs, their interaction with the TME, and recent advances in targeting this CSC-stroma network. Promising therapeutic strategies, such as CAR-T/NK cell therapies, epigenetic inhibitors, and combination regimens with checkpoint blockade or stromal modulators, are discussed in the context of ongoing clinical trials. Targeting both CSCs and their supportive microenvironment is emerging as a necessary strategy to overcome resistance and improve clinical outcomes in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0765b9737d7d7042e865a1f38d718a7df419afbe" target='_blank'>
              Deciphering the secrets of tumor-initiating cells in pancreatic ductal adenocarcinoma microenvironment: mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Jun Zhou, Jiajun Li, Haoyi Lu, Yanggang Hong
          </td>
          <td>2025-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Pancreatic ductal adenocarcinoma (PDAC) is a highly chemoresistant malignancy with a dismal 11% 5-year survival rate. PDAC tumors are composed of a dense desmoplastic stroma, and the interaction of this collagen-rich tumor microenvironment with PDAC cells promotes their aggressive growth and metastatic spread. In this study, we investigated the role of collagen XVII, a unique transmembrane collagen that connects the extracellular matrix to cytoplasmic signaling complexes. Examination of PDAC tumors revealed that its expression is an independent prognostic biomarker associated with worsened oncological outcomes. Xenograft tumor studies and in vitro coculture experiments with PDAC cell lines and immortalized cancer-associated fibroblasts defined a critical role for the tumor microenvironment in mediating the expression of collagen XVII in the cancer compartment of PDAC tumors. Genetic depletion of collagen XVII reduces the viability and migration of PDAC cells in vitro and orthotopic tumor formation in vivo, underscoring its critical role in PDAC tumor biology. Tumors formed by collagen XVII–depleted PDAC cells exhibit a dramatic downregulation of phosphoinositide 3-kinase regulatory subunit 5, a regulatory subunit of the class I phosphatidylinositol 3-kinase (PI3K) γ complex, and decreased levels of downstream activated AKT. Taken together, these findings reveal collagen XVII signaling as a novel target for PDAC treatment and provide insight into the mechanisms by which it regulates PI3K activity. Significance: Our study highlights the importance of cross-talk between the tumor microenvironment and PDAC cells in mediating the expression of a key signaling pathway responsible for PI3K activity in PDAC cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2df20c2b15e39948b89a698100c7ff74fc7269b" target='_blank'>
              Collagen XVII Promotes Pancreatic Ductal Adenocarcinoma Tumor Growth through Regulation of PIK3R5
              </a>
            </td>
          <td>
            T. Hank, Annie Li, L. Bolm, Marta Sandini, Taisuke Baba, N. Petruch, Julia Strässer, Kenzui Taniue, Jon M Harrison, M. Mino-Kenudson, David T Ting, Keith D. Lillemoe, Andrew L Warshaw, C. F. Fernandez-Del Castillo, Kim C. Honselmann, Andrew S Liss
          </td>
          <td>2025-07-29</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Abstract Background Penile squamous cell carcinoma (PSCC) is a rare yet potentially lethal malignancy, often resulting in devastating disfigurement, with a 5-year survival rate of only ∼50%. Human papillomavirus (HPV) infection is implicated in approximately half of PSCC cases and is associated with improved clinical outcomes; however, the underlying mechanisms remain poorly understood. Methods To elucidate HPV-associated differences in the tumor microenvironment, we performed single-cell RNA sequencing on tumors from 11 treatment-naïve PSCC patients, analyzing a total of 52 980 single cells. Unsupervised clustering identified 49 distinct cellular clusters across immune and stromal compartments. Results HPV-positive tumors exhibited an increased abundance of mast cells and a reduction in the proliferative macrophages subpopulation compared to HPV-negative tumors. Notably, CD8+ T cells in HPV-positive PSCC expressed lower levels of immune checkpoint molecules, suggesting a less exhausted immune state. Conversely, TIGIT and its ligands were significantly enriched in HPV-negative tumors, potentially fostering an immunosuppressive niche. Conclusion Collectively, our study delineates the single-cell landscape of PSCC and highlights distinct tumor microenvironment remodeling associated with HPV status, suggesting that the reduced immunosuppression in HPV-positive tumors may underlie their more favorable prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d170eff0ce2ddd0035f0702533123e38640014" target='_blank'>
              Single-cell transcriptomic profiling reveals distinct tumor microenvironments in HPV-associated penile squamous cell carcinoma
              </a>
            </td>
          <td>
            Lianbang Zhu, Deyun Shen, Jiahao Zhou, Chen Cheng, Zhiyao Xu, Yong Liang, Wen Pan, Tao Tao
          </td>
          <td>2025-09-01</td>
          <td>Precision Clinical Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Endothelial cells (ECs) play complex roles across tissues and vessel types. Yet, systematic investigations of EC heterogeneity in the combined context of vessel type and tissue microenvironment are still largely lacking. We integrated over three million single cells of scRNA-seq datasets in 15 human tissues and found that ECs in some tissues (e.g., heart and kidney) exhibited greater tissue specificity, while others displayed more substantial vessel specificity. We developed a computational pipeline to analyze cell-cell communications (CCC) mediated by metabolites or proteins to explore microenvironmental regulation. Interestingly, our results showed that CCC events involving ECs varied vastly across tissues, highlighting tissue-specific EC interactions. Using topic modeling, we identified CCC patterns, termed CCC topics, representing metabolite- and protein-mediated interactions between ECs and other tissue-resident cells. Most CCC topics exhibited high tissue specificity, potentially explaining microenvironmental regulations for EC heterogeneity. The work systematically investigates EC heterogeneity and provides insights into how EC heterogeneity was regulated across diverse tissue microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c584c5af5601779d13578366342256b73ed6c81" target='_blank'>
              Human single-cell atlas analysis reveals heterogeneous endothelial signaling
              </a>
            </td>
          <td>
            Zimo Zhu, Rongbin Zheng, Yang Yu, Lili Zhang, Kaifu Chen
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor heterogeneity encompasses genetic, epigenetic, and phenotypic diversity, impacting treatment response and resistance. Spatial heterogeneity occurs both inter- and intra-lesionally, while temporal heterogeneity results from clonal evolution. High-throughput technologies like next-generation sequencing (NGS) enhance tumor characterization, but conventional biopsies still do not adequately capture genetic heterogeneity. Liquid biopsy (LBx), analyzing circulating tumor DNA (ctDNA), provides a minimally invasive alternative, offering real-time tumor evolution insights and identifying resistance mutations overlooked by tissue biopsies. This study evaluates the capability of LBx to capture tumor heterogeneity by comparing genetic profiles from multiple metastatic lesions and LBx samples. Eight patients from the Augsburger Longitudinal Plasma Study with various types of cancer provided 56 postmortem tissue samples, which were compared against pre-mortem LBx-derived circulating-free DNA sequenced by NGS. Tissue analyses revealed significant mutational diversity (4–12 mutations per patient, VAFs: 1.5–71.4%), with distinct intra- and inter-lesional heterogeneity. LBx identified 51 variants (4–17 per patient, VAFs: 0.2–31.1%), which overlapped with mutations from the tissue samples by 33–92%. Notably, 22 tissue variants were absent in LBx, whereas 18 LBx-exclusive variants were detected (VAFs: 0.2–2.8%). LBx effectively captures tumor heterogeneity, but should be used in conjunction with tissue biopsies for comprehensive genetic profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99ae66d468d6885d0bee2286ab25cb4f8d1e150" target='_blank'>
              Dissecting Tumor Heterogeneity by Liquid Biopsy—A Comparative Analysis of Post-Mortem Tissue and Pre-Mortem Liquid Biopsies in Solid Neoplasias
              </a>
            </td>
          <td>
            Tatiana Mögele, Kathrin Hildebrand, Aziz Sultan, Sebastian Sommer, Lukas Rentschler, Maria Kling, Irmengard Sax, Matthias Schlesner, B. Märkl, M. Trepel, Maximilian Schmutz, Rainer Claus
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Systems biology approaches have been applied to prostate cancer to model how individual cellular and molecular components interact to influence cancer development, progression, and treatment responses. The integration of multi-omic experimental data with computational models has provided insights into the molecular characteristics of prostate cancer and emerging treatment strategies that have the potential to improve patient outcomes. Here, we highlight recent advancements that have emerged from systems modeling in prostate cancer. These include descriptions of the molecular landscape of prostate cancer and how genomic alterations inform computational models of disease progression, how evolutionary processes give rise to mechanisms of therapeutic resistance, and the development of innovative treatment strategies such as adaptive therapy. We also highlight current challenges in prostate cancer that can be addressed through systems biology approaches. These include tumor heterogeneity, poor immunotherapy response, a paucity of experimental model systems, and the ongoing translation of computational models for clinical decision making. Leveraging systems biology approaches has the potential to lead to a better understanding of the disease and better patient outcomes in the treatment of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88aeffc4990ae5d3cf0820d80475621089e3b441" target='_blank'>
              Systems biology successes and areas for opportunity in prostate cancer
              </a>
            </td>
          <td>
            Michael V Orman, Laura S Graham, Scott D. Cramer, James C. Costello
          </td>
          <td>2025-08-01</td>
          <td>Endocrine-Related Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Midkine (MDK), a multifunctional growth factor, has been implicated in promoting tumor progression, yet its role in glioblastoma (GBM) remains insufficiently characterized. To investigate MDK’s function in glioma, we integrated four RNA-Seq datasets into a harmonized cohort of 1,017 adult gliomas, including 256 GBM samples. We complemented this with freshly collected human GBM tissues and matched primary cell cultures to evaluate MDK expression and secretion patterns, further contextualized using single-cell RNA-Seq. Finally, we tested the impact of GBM-derived MDK on macrophage secretome composition to validate our in silico observations. We found that MDK expression increases with tumor grade in IDHwildtype gliomas, accompanied by a shift in isoform proportions favoring the canonical MDK transcript. High MDK expression was associated with poor prognosis specifically in GBM, where the MDKhigh subgroup comprised 75% of cases. MDKhigh GBMs exhibited a distinctive multiomic signature, including elevated chemokine and cytokine expression. Functionally, GBM-derived MDK induced macrophages to secrete multiple cytokines and chemokines, suggesting its role in reshaping the tumor microenvironment. Our findings reveal MDK’s previously underappreciated role in GBM aggressiveness and immune modulation, underscoring its potential as a biomarker and actionable therapeutic target for most GBM patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-16253-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5822ef3a00105861887610dcb7da2c42fc7f87" target='_blank'>
              Coordinated changes in midkine expression and midkine-associated multiomic profile in glioma microenvironment
              </a>
            </td>
          <td>
            Mieszko Lachota, K. Zielniok, Agata Gozdz, Patrycja Szpak, Ilona Kalaszczyńska, Radosław Zagożdżon
          </td>
          <td>2025-08-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Gastric cancer (GC) metastasis remains a major clinical challenge due to insufficient understanding of tumor microenvironment (TME) dynamics. While B cells are implicated in GC progression, their subset-specific roles in metastatic niches are poorly defined. Methods We analyzed gastric cancer (GC) single-cell RNA-seq data from the GEO database (GSE163558), complemented by bulk RNA-seq analysis of TCGA-STAD cohorts. Meta_B cells were identified through Seurat clustering and validated in colorectal cancer metastases (GSE166555). And we constructed a prognostic model via hdWGCNA and LASSO-Cox regression. Functional analyses included GSEA, pseudotime trajectory (Monocle2) and cell-cell communication (CellChat). Results We identified meta_B cells, a metastasis-enriched B cell subset, characterized by CLEC2B/YBX3 overexpression. Functional analyses suggested a potential immunosuppressive role associated with computational inference of BTLA-TNFRSF14 pathway activation, correlating with interactions with macrophages and other immune cells. A machine learning-derived 10-gene prognostic model effectively stratified high-risk patients with stromal-rich tumor microenvironments and predicted potential enhanced chemosensitivity to axitinib, dasatinib, olaparib, rapamycin, and ribociclib. Conclusions Meta_B cells may represent a novel B cell subset computationally associated with immunosuppression and GC metastasis potentially mediated by the BTLA axis. Our integrative transcriptomic framework provides hypothesis-generating insights into metastatic TME remodeling and a clinically actionable tool for prognostic prediction. Targeting meta_B cells can be explored as a strategy to potentially overcome immunotherapy resistance. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03356-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e483f9e1efa543fa750c8d4c16a75f93d5ae3ef4" target='_blank'>
              Meta_B cells: a computationally identified candidate immunosuppressive driver of gastric cancer metastasis revealed by single-cell analysis and machine learning
              </a>
            </td>
          <td>
            Tianchi Lei, Yiwen Jiang, Kexin Yang, Chuqi Meng, Yue An
          </td>
          <td>2025-08-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Tumor‐associated macrophages (TAMs) are prominent constituents of solid tumors, and their prevalence is often associated with poor clinical outcomes. These highly adaptable immune cells undergo dynamic functional changes within the immunosuppressive tumor microenvironment (TME), engaging in reciprocal interactions with malignant cells. This bidirectional communication facilitates concurrent phenotypic transformation: tumor cells shift toward invasive mesenchymal states, whereas TAMs develop immunosuppressive, pro‐tumorigenic traits. Increasing evidence highlights metabolic reprogramming, characterized by dysregulation of lipid metabolism, amino acid utilization, and glycolytic activity, as the fundamental molecular basis orchestrating this pathological symbiosis. However, a comprehensive understanding of how metabolic reprogramming specifically coordinates the mutual polarization of tumor cells and TAMs is lacking. This review thoroughly examines the molecular mechanisms governing this co‐polarization process, detailing critical transcriptional regulators, essential signaling pathways, and the maintenance of adaptive phenotypes within the TME. Furthermore, this review critically assesses promising therapeutic strategies aimed at disrupting this alliance, including the use of metabolically targeted agents, engineered chimeric antigen receptor macrophages, and TAM‐selective nanoparticle delivery systems. These insights provide a crucial foundation for the development of next‐generation cancer immunotherapies focused on reprogramming pathological polarization dynamics to overcome treatment resistance and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7946cf3691363a365260b427222e670775d573c5" target='_blank'>
              Polarization of Tumor Cells and Tumor‐Associated Macrophages: Molecular Mechanisms and Therapeutic Targets
              </a>
            </td>
          <td>
            Guohao Wei, Bin Li, Mengyang Huang, Mengyao Lv, Zihui Liang, Chuandong Zhu, Lilin Ge, Jing Chen
          </td>
          <td>2025-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f63086cbccde700980caecaca85ce93b724792a" target='_blank'>
              Neoadjuvant immunotherapy promotes the formation of mature tertiary lymphoid structures in a remodeled pancreatic tumor microenvironment.
              </a>
            </td>
          <td>
            Dimitri N. Sidiropoulos, Sarah M. Shin, Meredith Wetzel, Alexander Girgis, Daniel Bergman, Ludmila Danilova, Susheel Perikala, Daniel H. Shu, Janelle M. Montagne, Atul Deshpande, J. Leatherman, Lucie Dequiedt, Victoria Jacobs, Aleksandra Ogurtsova, Guanglan Mo, Xuan Yuan, Dmitrijs Lvovs, G. Stein-O’Brien, M. Yarchoan, Qingfeng Zhu, Elizabeth I. Harper, A. Weeraratna, A. Kiemen, E. Jaffee, Lei Zheng, W.J. Ho, Robert A. Anders, E. Fertig, L. T. Kagohara
          </td>
          <td>2025-08-15</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) has emerged as a powerful tool to map gene expression patterns to the local tissue structure in cancer, enabling unprecedented insights into cellular heterogeneity and tumour microenvironments. As the technology matures, developing new, spatially informed analytical frameworks will be essential to fully leverage its potential to elucidate the complex organisation and emerging properties of cancer tissues. Here, we highlight key challenges in cancer spatial transcriptomics, focusing on three emerging topics: (a) defining cell states, (b) delineating cellular niches and (c) integrating spatial data with other modalities that can pave the way towards clinical translation. We discuss multiple analytical approaches that are currently implemented or could be adapted in the future in order to tackle these challenges, including classical biostatistics methods as well as methods inherited from geospatial analytics or artificial intelligence. In the rapidly expanding landscape of ST, such methodologies lay the foundation for biological discoveries that conceptualise cancer as an evolving system of interconnected niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9031fde1ffd2ce70d2c223e1b5d79202f9418532" target='_blank'>
              Decrypting cancer's spatial code: from single cells to tissue niches.
              </a>
            </td>
          <td>
            Cenk Celik, Shi Pan, Eloise Withnell, Hou Wang Lam, M. Secrier
          </td>
          <td>2025-07-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Prostate cancer (PCa) is an extremely heterogeneous cancer and is highly prevalent in the older male population. Since intra‐tumour heterogeneity (ITH) commonly results in PCa chemotherapy resistance and recurrence, it is critical to explore its effects on tumour behaviour. Prognostic genes related to ITH were identified, and a signature was constructed using Cox regression analyses and multiple machine learning algorithms. Single‐cell RNA sequencing data extracted from PCa and CRPC samples were analysed via sub‐clustering, pseudotime, cell communication and drug sensitivity approaches to elucidate their function. The oncogenic potential of hub genes was confirmed by immunohistochemistry and cell proliferation assays. An 11‐gene signature underlying a prostate cancer meta‐program (PCMP) was generated by selecting an optimal combination of machine learning methods. Survival assays and multivariate Cox regression analyses conducted in multiple cohorts revealed the superior prognostic value of the PCMP signature. Functional enrichment analyses indicated that it dysregulates the cell cycle. Using trajectory and cell–cell communication analyses, we illustrated that PCMP genes exert oncogenic effects by enhancing the proliferation and oxidative phosphorylation of epithelial cells. Intra‐cellular assays also demonstrated that CENPA and CKS1B had promising malignant potential. In summary, our research not only establishes the association between the PCMP signature and reveals its malignant characteristics, but also deepens our understanding of the mechanisms underlying PCa progression and ITH. It holds promise for the development of targeted therapeutic interventions, thereby offering clinical benefits to patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/045ffd0cd32bcb5869978af9b2509598525fb2d8" target='_blank'>
              Single‐Cell and Bulk RNA Sequencing Highlights Intra‐Tumoral Heterogeneity and Malignant Progression Mechanisms in Prostate Cancer
              </a>
            </td>
          <td>
            Junchao Wu, Ziqi Chen, Wentian Wu, Jiaxuan Qin, Rongfang Zhong, Jialin Meng, Yu Yin, Peng Guo, Song Fan
          </td>
          <td>2025-08-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Dendritic cells (DCs) are pivotal orchestrators of anti-tumor immunity. DC-based anti-tumor treatments are being actively developed, but effective clinical responses have not yet been achieved. Further exploration of DC heterogeneity in the tumor microenvironment (TME) and across cancer types could provide insights for developing DC-based immunotherapies. Here, we integrated single-cell RNA sequencing data of DCs from over 2,500 samples across 33 cancer types and established a comprehensive blueprint of human DCs. Several rare subsets of DCs infiltrated the tumors, including AXL+SIGLEC6+ (AS) DCs and Langerhans cell (LC)-like DCs, and displayed functional potentials marked with distinct transcriptomic characteristics. Computational analyses demonstrated that the LC-like subset could be an additional cellular origin of tumor-enriched LAMP3+ DCs and that distinct cellular origins are associated with the pleiotropic functional potentials of LAMP3+ DCs. Furthermore, this DC atlas enabled development of a machine learning model to guide DC annotation for subsequent single-cell analysis and prioritization of a valuable target for enhancing anti-tumor DC vaccination. This integrative resource provides a panoramic view to unravel the complexity of tumor-infiltrating DCs and offers valuable insights for developing therapies targeting DCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20198842496886b53a9ff7adf340fb98f1bc673" target='_blank'>
              Pan-cancer Analyses Refine the Single-Cell Portrait of Tumor-Infiltrating Dendritic Cells.
              </a>
            </td>
          <td>
            Tianyi Ma, Xiaojing Chu, Jinyu Wang, Xiangjie Li, Yu Zhang, Dan Tong, Wenbin Xu, Guohui Dang, Lu Qi, Yuhui Miao, Zemin Zhang, Si Cheng
          </td>
          <td>2025-07-31</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Intratumor heterogeneity is a hallmark of cancer, enabling subpopulations of cells to evade therapy, adapt to immune attack, and thrive in diverse microenvironments. While retrospective genomic and epigenomic analyses have mapped the large-scale histories of tumor evolution, they cannot capture the rapid, dynamic changes in cell state that occur as individual cells divide. Panagopoulos and colleagues recently developed a cellular platform to monitor the role of transient replication stress in real time, tracking sister cells as they divide and replicate. The authors use these techniques to show that daughter and granddaughter cells can inherit very different states, often leading to further cellular instability. This work broadens our understanding of how diverse cell states arise from oncogenic stress and how cellular heterogeneity emerges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beadadba41d03604ec2c3cef748b8a0844e1482c" target='_blank'>
              Generations in Flux: Tracking the Birth of Cellular Heterogeneity in Cancer.
              </a>
            </td>
          <td>
            Aaron McKenna
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Meningiomas are the only primary tumors that undergo hyperostosis in the tumor microenvironment. Clinical observations suggest TRAF7short somatic variants (SSVs) are enriched in skull-base meningiomas, and that SSVs targeting the TRAF7 WD40-domain may be associated with meningioma-induced hyperostosis (MIH). Here, we integrate single-cell RNA sequencing and lineage tracing data with mechanistic and functional studies across re- gionally distinct primary meningioma cell cultures. We ﬁnd that TRAF7 WD40-domain mutations drive menin- gioma cell transdifferentiation via PRRX1 into an osteoblast-like state that is associated with mineralization in vitro and MIH in vivo.



 Single-cell RNA sequencing,mitochondrial alteration enrichment from single-cell transcriptomes to establish re- latedness (MAESTER) lineage tracing, RNA velocity and pseudotime analyses were used to deﬁne cell types, cell states, and evolutionary trajectories from 8 human samples. Regionally distinct primary meningioma cell cul- tures (n=6) from 3 meningiomas with MIH were derived for mechanistic and functional studies. Targeted next- generation DNA sequencing, QPCR, immunoﬂuorescence, cell proliferation, immunoprecipitation with western blot, and mineralization assays with alizarin red stains were used to deﬁne how TRAF7 WD40-domain mutations inﬂuence protein function and cell state.



 Single-cell transcriptomics and lineage tracing identiﬁed a differentiation event from meningioma origin clus- ter to an osteoblast-like meningioma cell cluster that was enriched for bone-associated genes, driven by genes associated with PRRX1. Primary meningioma cell cultures were validated using DNA methylation proﬁling and SSTR2A expression. Only cell cultures from intraosseous regions expressed RUNX2, SP7, or ALPL, and under- went mineralization in vitro. Suppression of mutant TRAF7 using siRNAs blocked in vitro mineralization, and media from meningioma cell cultures from intraosseous regions induced mineralization of TRAF7 wildtype meningioma cell cultures in vitro.



 These data demonstrate that TRAF7 WD40-domain mutations drive meningioma cell differentiation and intra- tumor hyperostosis via expression of PRRX1, and shed light on fundamental mechanisms underlying lamellar bone formation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/190662e8dee7fcce899ad86ef23836987311c4dd" target='_blank'>
              TRAF7 WD40-DOMAIN MUTATIONS DRIVE MENINGIOMA TRANSDIFFERENTIATION AND INTRATUMOR HYPEROSTOSIS
              </a>
            </td>
          <td>
            C. Eaton, E. Goldschmidt, Ayush Aggarwal, Zhenhong Chen, David R Raleigh
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cell type markers have been instrumental to physiological and molecular investigation of the human brain and remain essential for annotating cell type clusters in single-cell expression data and for targeted validation studies. However, expression of canonical markers in the target cell type (which we termed as the expression ‘fidelity’) as well as expression in unrelated cell types (which we termed as the ‘background expression’) across cortical regions remains poorly characterized. Using nearly 500,000 high-quality single-nucleus profiles from 19 studies, we quantified marker fidelity, revealing substantial regional variability. We developed a statistical framework that aggregates annotated barcodes into pseudo-bulk profiles, applied rigorous performance metrics, and identified markers with improved fidelity, reduced background, and consistent expression across regions. This approach extended the canonical marker set for six major brain cell types and yielded superior subtype-specific markers. The resulting marker lists, and a user-friendly analysis interface, provide a valuable resource for cell type annotation and validation in neurological research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40d3defdba63314b1d51e6d4e42495e28376f453" target='_blank'>
              Expanding canonical cortical cell type markers in the era of single-cell transcriptomics
              </a>
            </td>
          <td>
            Dennis Joshy, Soojin V Yi
          </td>
          <td>2025-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Endometriosis is often diagnosed late and presents significant challenges in clinical treatment. A comprehensive investigation of the cellular classification and composition of endometriosis is essential for studying its diagnosis and treatment. Methods This study utilized the Gene Expression Omnibus (GEO) public database and referenced single-cell RNA sequencing (scRNA-seq) atlases. The CIBERSORTx algorithm was applied to perform deconvolution on the samples and estimate the proportions of endometrial cell subtypes. A random forest model was constructed to predict the diagnosis of endometriosis. Additionally, immunohistochemical validation was performed on the marker genes of MUC5B+ epithelial cells and dStromal late mesenchymal cells, which showed high diagnostic contribution. Results Endometriosis consists of 5 major cell types, further classified into 52 distinct cell subtypes. Compared to healthy controls, these subtypes exhibited varying degrees of alterations, with MUC5B+ epithelial cells, dStromal late mesenchymal cells, and M2 macrophages showing an increasing trend. Enriched signaling pathways were primarily associated with epithelial-mesenchymal transition (EMT), cell migration, and inflammatory responses. A random forest model, based on cell-type proportions, has been shown to achieve excellent diagnostic performance (AUC = 0.932), with MUC5B+ epithelial cells identified as the top predictive feature. Immunohistochemical validation confirmed high expression of the marker genes MUC5B and TFF3. Conclusion By integrating single-cell and bulk transcriptomics, we identified MUC5B+ epithelial cells and dStromal-late mesenchymal cells as dual drivers of fibrosis and inflammation in endometriosis. Our findings revealed that MUC5B+ epithelial cells may serve as the top factor for the diagnosis of endometriosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99fd62628ca18755a361b8c7c59b2f9cf5de62a" target='_blank'>
              Integrated analysis of single-cell and bulk transcriptomic data reveals altered cellular composition and predictive cell types in ectopic endometriosis
              </a>
            </td>
          <td>
            Meihong Chen, Liqun Wang, Yuanting Chen, Ting Wang, Guanqun Jiang, Qi Chen
          </td>
          <td>2025-07-18</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common primary brain cancer. It causes death mainly by local invasion via several routes, including infiltration of white matter tracts and penetration of perivascular spaces. However, the pathways that mediate these invasion routes are only partly known. Here, we conduct an integrative study to identify cell states and central drivers of route-specific invasion in GBM. Combining single-cell profiling and spatial protein detection in patient-derived xenograft models and clinical tumor samples, we demonstrate a close association between the differentiation state of GBM cells and their choice of invasion route. Computational modeling identifies ANXA1 as a driver of perivascular involvement in GBM cells with mesenchymal differentiation and the transcription factors RFX4 and HOPX as orchestrators of growth and differentiation in diffusely invading GBM cells. Ablation of these targets in tumor cells alters their invasion route, redistributes the cell states, and extends survival in xenografted mice. Our results define a close association between GBM cell differentiation states and invasion routes, identify functional biomarkers of route-specific invasion, and point toward targeted modulation of specific invasive cell states as a therapeutic strategy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bd579d99eaa69724a2c2c47d55f912d2748b0a9" target='_blank'>
              The invasion phenotypes of glioblastoma depend on plastic and reprogrammable cell states
              </a>
            </td>
          <td>
            Milena Doroszko, Rebecka Stockgard, Irem Uppman, Josephine Heinold, Faidra Voukelatou, H. Mangukiya, T. Millner, Madeleine Skeppås, Mar Ballester Bravo, Ramy Elgendy, Maria Berglund, Ludmila Elfineh, C. Krona, Soumi Kundu, Katarzyna Koltowska, Silvia Marino, Ida Larsson, Sven Nelander
          </td>
          <td>2025-07-19</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="It is well established from in vitro studies of immune cells that stimulation by a wide range of potential signals leads to motility and morphology changes. How these physical behaviors manifest inside a living animal remains unclear due to limitations of conventional imaging and analysis approaches. Here, we establish a quantitative framework for imaging and tracking neutrophil and macrophage dynamics in larval zebrafish, spanning a large fraction of the animal for multi-hour timescales with few-minute temporal resolution. We focus especially on the gut, examining innate immune responses to different preparations of the intestinal microbiome. Using light sheet fluorescence microscopy and trajectory analysis of hundreds of individual cells, we characterize speeds, directional persistence measures, and cellular morphology to reveal distinct population behaviors. Individual immune cells exhibit stable motility phenotypes, favoring predominantly motile or non-motile states rather than frequent transitions between them. Gut architecture constrains migration patterns as demonstrated by preferential anterior-posterior movement and a high probability of cells remaining in the vicinity of the gut throughout the imaging duration. Macrophages display significantly reduced sphericity during motile periods compared to non-motile periods, providing a morphological signature that may enable inference of dynamic behavior from static snapshots. Surprisingly, migration patterns remain consistent across diverse microbial conditions – germ-free, conventionally reared, and colonized by two strains of a zebrafish-native Vibrio species – indicating that tissue structure exerts a stronger influence than bacterial stimuli on immune surveillance dynamics. Previously observed tissue damage by the wild-type Vibrio strain, and the resulting recruitment of immune cells towards the damage site, provided the only microbe-specific cellular behavior. These findings reveal innate immune surveillance as a stereotyped process whose characteristics reflect both cellular decision-making and larger-scale anatomical structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1591d467917ceed9ad4c3b39956f5d44b060babf" target='_blank'>
              Characterizing trajectories of innate immune cells in larval zebrafish
              </a>
            </td>
          <td>
            Piyush Amitabh, Raghuveer Parthasarathy
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="MYCN amplification without concurrent RB1 mutations characterizes a rare yet highly aggressive subtype of retinoblastoma; however, its precise developmental origins and therapeutic vulnerabilities remain incompletely understood. Here, we modeled this subtype by lentiviral-mediated MYCN overexpression in human pluripotent stem cell-derived retinal organoids, revealing a discrete developmental window (days 70–120) during which retinal progenitors showed heightened susceptibility to transformation. Tumors arising in this period exhibited robust proliferation, expressed SOX2, and lacked CRX, consistent with origin from primitive retinal progenitors. MYCN-overexpressing organoids generated stable cell lines that reproducibly gave rise to MYCN-driven tumors when xenografted into immunodeficient mice. Transcriptomic profiling demonstrated that MYCN-overexpressing organoids closely recapitulated molecular features of patient-derived MYCN-amplified retinoblastomas, particularly through activation of MYC/E2F and mTORC1 signaling pathways. Pharmacological screening further identified distinct therapeutic vulnerabilities, demonstrating distinct subtype-specific sensitivity of MYCN-driven cells to transcriptional inhibitors (THZ1, Flavopiridol) and the cell-cycle inhibitor Volasertib, indicative of a unique oncogene-addicted state compared to RB1-deficient retinoblastoma cells. Collectively, our study elucidates the developmental and molecular mechanisms underpinning MYCN-driven retinoblastoma, establishes a robust and clinically relevant human retinal organoid platform, and highlights targeted transcriptional inhibition as a promising therapeutic approach for this aggressive pediatric cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7456982cd84b09cab50502a3b5ab5c8e7a74c4cd" target='_blank'>
              Human Retinal Organoid Modeling Defines Developmental Window and Therapeutic Vulnerabilities in MYCN-Amplified Retinoblastoma
              </a>
            </td>
          <td>
            Jinkyu Park, Gang Cui, Jiyun Hong, H. Jeong, Minseok Han, Min Seok Choi, Jeong Ah Lim, Sanguk Han, Christopher Seungkyu Lee, Min Kim, Sangwoo Kim, Junwon Lee, S. Byeon
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cancer cells orchestrate the surrounding tumor microenvironment (TME) to strike a fine balance between tissue regeneration providing them with nutrients, and tissue destruction triggered by immunogenic alarm signals. At steady state, the tenuous balance favors cancer growth. Therapies aimed at enhancing the immunogenic properties of cancer cells or the reacting immune responses can, however, revert the equilibrium to clear the host of cancer. The understanding of factors that affect this balance is progressing rapidly due to advances in high throughput technologies disclosing from previously uncharted territories new biologies referred to as “dark matter”. These advances are critical for the understanding of the true mechanisms leading to immune-mediated cancer rejection. This review focuses on cancer genetic, epigenetic and metabolic derangements that approximate those caused by intra-cellular pathogen infection, a phenomenon referred to as “viral mimicry” (VM) and other aspects of cancer/host cells interactions unexplored in the past that enhance the VM effects. On the cancer side, VM prompts alterations of cancer cell metabolism leading to the generation of aberrant cellular products recognized as foreign by the host’s immune system. The latter are defined as “dark matter” to emphasize the powerful effects exerted by these obscure bioproducts on the TME as the mass of invisible particles can dictate the rotational period of galaxies. On the other side, a myriad of previously unappreciated factors can influence the host responses. Thus, here we propose an extended definition of dark matter beyond the limits of cancer cell-intrinsic biology, to a broader interpretation encompassing elements that influence the cellular networks within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a42b4f37f1c7313faa25f37bb036aac0202e1c7" target='_blank'>
              The dark matter in cancer immunology: beyond the visible– unveiling multiomics pathways to breakthrough therapies
              </a>
            </td>
          <td>
            Salvatore De Rosis, Gianni Monaco, Joyce Hu, Erick Hett, R. Lappano, Francesco M. Marincola, A. Asadi, M. Maggiolini
          </td>
          <td>2025-07-22</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>65</td>
        </tr>

        <tr id="BACKGROUND
Atypical teratoid rhabdoid tumors (ATRTs) are highly aggressive pediatric central nervous system tumors defined by the inactivation of the SMARCB1 gene. Despite the identification of three distinct molecular subtypes, each defined by unique clinical and molecular characteristics, no subtype-specific therapeutic strategies are currently available. This highlights an urgent need to deepen our understanding of the cellular heterogeneity and developmental origins of ATRTs.


METHODS
We generated a comprehensive single-nucleus transcriptomic atlas of ATRT samples, integrated it with single-nucleus ATAC-seq and spatial transcriptomics data, and validated our findings experimentally using patient-derived ATRT tumoroid models.


RESULTS
Our analyses revealed distinct subtype-specific differentiation trajectories, each resembling different brain progenitor lineages. We identified key transcription factors that appear to drive these developmental pathways. Furthermore, a shared cycling, intermediate precursor cell (IPC)-like cell population, interspersed throughout tumors, was consistently present within all ATRT samples. We demonstrate that these subtype-specific differentiation pathways can be pharmacologically manipulated in patient-derived ATRT tumoroids. By directing tumor cells along their respective subtype-specific trajectories, we were able to induce a shift toward more differentiated, non-proliferative states.


CONCLUSIONS
Collectively, our findings show that ATRTs recapitulate fetal brain signaling programs in a subtype-specific manner. This work provides a framework for understanding ATRT heterogeneity and supports the feasibility of maturation-based therapeutic strategies tailored to the molecular subtype of the tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f089b7cc6f0aae306ce7c5e4e459d6bc20cd73eb" target='_blank'>
              A cycling, progenitor-like cell population at the base of atypical teratoid rhabdoid tumor subtype differentiation trajectories.
              </a>
            </td>
          <td>
            Enrique Blanco-Carmona, Irene Paassen, Jiayou He, Jeff DeMartino, Annette Büllesbach, Nadia Anderson, Juliane L Buhl, A. Federico, Monika Mauermann, Mariël Brok, Karin Straathof, S. Behjati, Rajeev Vibhakar, A. Donson, Nicholas K Foreman, McKenzie L. Shaw, M. Frühwald, A. Korshunov, M. Hasselblatt, C. Thomas, Niels Franke, M. Kranendonk, E. Hoving, N. Jäger, P. Johann, Stefan M Pfister, Mariella G. Filbin, M. Kool, J. Drost
          </td>
          <td>2025-07-28</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="ABSTRACT Background Bladder cancer (BC) ranks as the 10th leading cause of cancer‐related deaths in the United States. Despite significant advancements in managing BC, including antibody‐drug conjugates and immune checkpoint blockers, a significant percentage of patients fail these treatments. This underscores the pressing need to develop methods for identifying novel therapeutic targets, predicting therapy responses, and optimizing personalized treatment strategies for patients. Patient‐derived xenograft (PDX) models have the potential to address these challenges. However, these models suffer from limitations, including a consistent rate of tumor acceptance, extended time for tumor growth, and the absence of the donor's immune system. Methods This study aimed to address these challenges through a comprehensive comparison of different PDX implantation methods, culminating in the establishment of a PDX cell line for utilization in in vivo BC preclinical models. Flow cytometry analysis and total RNA sequencing were used for comparative analysis of original tumor tissue, PDX tumor, and PDX257S cell line. Results Tumor tissue processing techniques were optimized to enhance the rate of PDX tumor acceptance and shorten the time to tumor development. Furthermore, a PDX cell line, PDX257S, was successfully established and confirmed to exhibit more aggressive and tumorigenic characteristics compared to the original donor tumor tissue. The PDX257S cell line demonstrated faster tumor growth, higher expression of human epithelial cell adhesion molecule, increased mutation burden in BC‐associated genes, and significant alterations in BC‐related gene expression, compared to the original tumor. The PDX257S cell line was also used to establish a double humanized BC murine model that minimized graft‐versus‐host disease and is a potential platform for testing novel immune therapeutics. Conclusions In summary, this study offers an optimized protocol for the consistent establishment of BC PDX tumors. Furthermore, it has also established a novel double‐humanized model, demonstrating the potential for drug screening and clinical prognosis in the context of BC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b467e1c34bd6787c0785c1b91954f032c9c82d9" target='_blank'>
              A Double‐Humanized Murine Model in Bladder Cancer: A Novel Preclinical Model for Cancer Immunology Research
              </a>
            </td>
          <td>
            Niannian Ji, Zaineb Hassouneh, Shaun Trecarten, Zhen-Ju Shu, Jaime Furman, Tyler J. Curiel, Robert S. Svatek, Neelam Mukherjee
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract BACKGROUND A major barrier in our understanding of glioblastoma (GBM) is the difficulty in tracking tumour development in real-time. We adopted a novel approach that combines the principles of classic cellular barcoding with CRISPR/Cas-9 technology and single-cell RNA sequencing known as continuous lineage tracing to apply a phylogenetic approach to studying tumour development. METHODS Patient derived glioma initiating cell lines were engineered with expressed DNA barcodes with CRISPR/Cas-9 targets and subsequently implanted to create an intracranial xenograft model. Tumors were sent for single cell RNA sequencing; clonal relationships were surmised through identification of expressed barcodes, and cells were characterized by their transcriptional profiles. Phylogenetic lineage trees were created utilizing lineage reconstructive algorithms to define cell fitness and expansion. RESULTS Our work has revealed a significant amount of intra-clonal cell state heterogeneity, suggesting that tumour cells engage in phenotype switching prior to therapeutic intervention. We defined a consistent transcriptional pattern for tumour engraftment and in vivo clonal advantage. Phylogenetic lineage trees allowed us to define gene signatures of both cell fitness and expansion, which correlate strongly with published neural-mesenchymal and developmental-injury response phenotypes. GBMs exist along a transcriptional gradient between undifferentiated but “high-fit” cells and terminally differentiated, “low-fit” cells: cells with highly fitness appear to represent cells undergoing cell state transition. CONCLUSION We successfully employed a novel lineage tracing technique in GBM, creating a powerful tool for real-time tracing of tumour growth through the analysis of a unique set of highly detailed transcriptional data with associated clonal and phylogenetic relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51825ee39f0efc909dc234d3cc75f48f8c3aa8f6" target='_blank'>
              56 SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS AND EXPANSION AXES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Ajisebutu, F. Fan, Chloe Gui, Y. Ellenbogen, A. Landry, Jeff Liu, Q. Wei, G. Zadeh
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Aims
Patient-derived organoids (PDOs) are promising preclinical models that replicate critical tumor features. However, intratumoral heterogeneity challenges the clinical utility of PDOs, especially in capturing diverse tumor cell subpopulations.


Methods
Single-cell transcriptomics was used to analyze PDOs from distinct sites within a single gastric cancer tumor, aiming to assess their ability to reflect intratumoral heterogeneity.


Results
The PDOs displayed similarities in gene expression but also exhibited distinct profiles. Single-cell analysis of PDOs revealed upregulation of markers for neuroendocrine tumors, which was validated via immunohistochemistry staining of neuron-specific enolase in the primary tumor. Notably, heat shock proteins showed significant variability among the PDOs, impacting immune responses. Tumors with abundant heat shock proteins are reported to have increased cytotoxic T cell activity.


Conclusions
Intratumoral heterogeneity poses challenges for PDO-based models, highlighting the need for comprehensive assessment. Despite their limitations, PDOs offer valuable insights into precision medicine for patients with gastric cancer, aiding in the development of therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9154970e5ea83aed48cb2a2d344b23752aade163" target='_blank'>
              Patient-Derived Organoids from Multiple Sites of a Single Tumor Recapitulates Intratumoral Heterogeneity in Patients with Gastric Cancer.
              </a>
            </td>
          <td>
            Bo Kyung Yoon, Y. Bae, Yeonjin Je, Seyeon Joo, Yuna Kim, Su-Jin Shin, Sungsoon Fang, Jie-Hyun Kim
          </td>
          <td>2025-09-05</td>
          <td>Gut and liver</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor characterized by an immunosuppressive microenvironment that importantly contributes to treatment resistance. Monocyte-derived macrophages (MDMs), which comprise approximately 50% of the cellular population within the GBM microenvironment, represent a major subset of tumor-associated macrophages. These cells drive tumor progression by promoting angiogenesis, immune evasion, and the phenotypic transformation of tumor cells. MDM infiltration is mediated by specific signaling pathways and regulated by the disruption of the blood–brain barrier and tumor-associated hypoxia. Recent technological advances have uncovered substantial heterogeneity among macrophages, including hypoxia-induced, lipid-metabolizing, phagocytic, and interferon-activated subtypes. This functional diversity is shaped by tumor-specific genetic alterations and metabolic reprogramming. Therapeutic approaches focusing on MDMs include inhibiting their recruitment, enhancing phagocytic activity, employing genetically engineered macrophage, and modulating metabolic pathways. While preclinical studies suggest that these approaches may improve efficacy when combined with immune checkpoint inhibitors, the dynamic spatiotemporal heterogeneity and adaptability of macrophages within the tumor microenvironment remain substantial therapeutic challenges. Future development in combination therapies, integrating single-cell multi-omics, spatial metabolic profiling, and targeted interventions, will be critical to precisely modulate MDMs, overcome immune tolerance, and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e73dae7727564fcd760697c5cdc2cc02393ac584" target='_blank'>
              New Insights into Monocyte-Derived Macrophages in Glioblastoma
              </a>
            </td>
          <td>
            Xuetong Li, Wei Gao, Xinmiao Long, Minghua Wu
          </td>
          <td>2025-07-23</td>
          <td>Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are a critical component of the glioma microenvi-ronment and play essential roles in tumor progression and resistance to immunotherapy. To comprehensively characterize CAF heterogeneity and their interactions with immune cells, we conducted an integrative multi-omics analysis incorporating single-cell and bulk RNA sequencing, spatial transcriptomics, and multiplex immunofluorescence. This approach identified nine distinct CAF subtypes with phenotypic and functional diversity, including tumor-like CAFs (tCAFs), myofibroblast-like CAFs (myCAFs), vascular CAFs (vCAFs), metabolic CAFs (meCAFs), proliferative CAFs (pCAFs), antigen-presenting CAFs (apCAFs), interferon-responsive CAFs (infCAFs), inflammatory CAFs (iCAFs), and a group of CAFs with unknown identity. Several subtypes were significantly associated with poor clinical outcomes. Notably, apCAFs engaged in extensive crosstalk with M2-polarized macrophages via TGF-β signaling pathways. Spatial transcriptomic pro-filing and immunofluorescence imaging revealed the co-localization of apCAFs and M2 macrophages at the tumor periphery, indicating the formation of an immunosuppressive niche. Moreover, AQP4 was identified as a specific marker of apCAFs, and its expression was significantly correlated with poor prognosis and resistance to immunotherapy. These findings offer a comprehensive atlas of CAF heterogeneity in glioma and highlight the therapeutic promise of targeting apCAF–M2 macrophage interactions or AQP4 to over-come immune resistance and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01ea0f5facb6a5cedd9f4ccccded98f2155bac61" target='_blank'>
              A multi-omics atlas of CAF subtypes reveals apCAF–M2 macrophage interactions driving immune resistance in glioma
              </a>
            </td>
          <td>
            Yubo Ren, Deng-Feng Lu, Fei Wang, Zixuan Wang, Jinfeng Li, Run Huang, Yue Lu, A-Li Duan, Renjie Shou, Jiangang Liu, Zhouqing Chen, Zhong Wang, Xiaoou Sun
          </td>
          <td>2025-08-11</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Recent biological research has highlighted the relevance of myeloid-cell populations in glioma growth, with a particular role played by tumor-associated macrophages (TAMs), which comprise resident microglia and monocyte-derived macrophages. Additionally, radiation therapy, the most common treatment for gliomas, significantly alters the tumor microenvironment, affecting TAMs and contributing to tumor recurrence. Promising preclinical studies have identified and developed drugs targeting TAMs. The development and combined deployment of these therapies require in silico techniques that enable us to optimize their outcomes. To do so, we need mathematical models of glioma growth and therapy response that explicitly incorporate TAMs—an often overlooked component in existing models. Here, we present a dynamical model of glioma growth driven by tumor-immune interactions. The model was parametrized using published data from mice experiments, including responses to ionizing radiation. We used this model to investigate glioma progression under radiotherapy combined with three treatments targeting distinct aspects of TAM biology. Simulations revealed that anti-CD47 enhanced the otherwise weak phagocytic activity, extending the upper tail of the survival curve. α-CD49d, which limits monocyte trafficking after irradiation, offered consistent survival benefits across digital twins of mice. Finally, CSF-1R inhibitors, which block the primary growth factor regulating TAM function, resulted in the largest overall survival improvement in silico. Our results aligned well with experimental evidence, suggesting that the model may help inform the optimization of myeloid cell-targeted immunotherapies, including their timing, dosage, and combination with radiation therapy, with potential relevance for improving glioma treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92c930ffd6201d9f5f23b5cf400a233231307968" target='_blank'>
              Mathematical model of tumor-macrophage dynamics in glioma to advance myeloid-targeted therapies
              </a>
            </td>
          <td>
            Jesús J. Bosque, Jordan Martínez, José García Otero, Guim Aguadé-Gorgorió, Javier E. Sánchez-Galán, J. Belmonte-Beitia
          </td>
          <td>2025-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708353dcfdc8ca7c4e2b434fced2427c746e282e" target='_blank'>
              Tracing the evolution of single-cell 3D genomes in Kras-driven cancers.
              </a>
            </td>
          <td>
            Miao Liu, Shengyan Jin, Sherry S Agabiti, Tyler B Jensen, Tianqi Yang, Jonathan S. D. Radda, Christian F. Ruiz, Gabriel Baldissera, Moein Rajaei, Fangyong Li, Jeffrey P. Townsend, M. Muzumdar, Siyuan Wang
          </td>
          <td>2025-08-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Understanding the roles of myeloid cells in the tumor microenvironment (TME) has emerged as a promising strategy to identify novel targets to counteract the immunosuppressive barriers protecting multiple myeloma (MM). Neutrophils are a new cancer research focus due to their potential to reduce the efficacy of immune-based therapies. This study aimed to deepen understanding of neutrophil function in MM by analyzing freshly isolated myeloid cells from paired focal lesions (FL) and bone marrow (BM) using single-cell RNA sequencing (scRNA-seq), immunofluorescence imaging, and functional assays. We describe three distinct CXCR2+ mature neutrophil subsets: TREM1+CD10+, RETN+LCN2+, and TNFAIP3+CXCL8+, each exhibiting unique phenotypes within the TME. Notably, the TREM1+CD10+ subset was highly prevalent, particularly in FL, demonstrating potent immunosuppressive effects on T cells. This subset's gene signature was correlated with shorter overall survival (OS) in a large MM patient dataset, underscoring its clinical significance. Targeted inhibition of neutrophil activity through CXCR2 blockade, alone or combined with standard anti-MM therapies, significantly reduced tumor burden, improving OS in preclinical MM models. These insights into neutrophil-mediated immunosuppression in MM provide valuable knowledge regarding mechanisms driving immune evasion and reveal new therapeutic approaches to enhance the efficacy of MM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa3b405dd150d9c3f5e25a26a609ad2e8d28a1ca" target='_blank'>
              Deciphering neutrophil dynamics in the focal lesion tumor microenvironment to overcome immunosuppression in multiple myeloma.
              </a>
            </td>
          <td>
            Joshua Rivera, Qi Yan, Saeed Daneshmandi, R. Lannes, Eriko Katsuta, Jeeeun Choi, Prashant K Singh, A. Belal, Ronald Alberico, Ian Lund, Magan Schaefer, Hamza Hassan, Sarah Parker, Kenneth C. Anderson, Nikhil C Munshi, M. Samur, Philip L McCarthy, Jens Hillengass, H. Mohammadpour
          </td>
          <td>2025-08-04</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Neuroblastomas encompass malignant cells with varying degrees of differentiation, ranging from adrenergic (adr) cells resembling the sympathoadrenal lineage to undifferentiated, stem-cell-like mesenchymal (mes) cancer cells. Relapsed neuroblastomas, which often have mesenchymal features, have a poor prognosis and respond less to anticancer therapies, necessitating the development of novel treatment strategies. To identify novel treatment options, we analyzed the sensitivity of 91 pediatric cell models, including patient-derived tumoroid cultures, to a drug library of 76 anti-cancer drugs at clinically relevant concentrations. This included 24 three-dimensionally cultured neuroblastoma cell lines representing the range of mesenchymal to adrenergic subtypes. High-throughput ATP-based luminescence measurements were compared to high-content confocal imaging. With machine learning-supported imaging analysis, we focused on changes in the lysosomal compartment as a marker for therapy-induced senescence and assessed the basal lysosomal levels in a subset of untreated mesenchymal versus adrenergic cells. We correlated these findings with pathway activity signatures based on bulk RNA and scRNAseq. Comprehensive image-based synergy screens with spheroid cultures validated the combined effects of selected drugs on proliferation and cytotoxicity. Mesenchymal models presented high basal lysosomal levels correlating with senescence-associated secretory phenotype (SASP) and sphingolipid metabolism pathways. Chemotherapy treatment further increased lysosome numbers, indicative of therapy-induced senescence. Furthermore, the mesenchymal subtypes correlated with MAPK activity and sensitivity to MAPK pathway inhibitors. Lysosomal and SASP signaling is druggable by inhibitors of lysosomal acid sphingomyelinase (SLMi) or senolytics, including BCL2-family inhibitors. Especially the sequential combination of MEK inhibitors (MEKi) with BCL2-family inhibitors was the most effective on relapsed neuroblastoma cell lines. Gene expression analysis of 223 patient samples, drug sensitivity profiling of five patient-derived fresh tissue cultures, and in vivo zebrafish embryo neuroblastoma xenograft models confirmed these findings. Inhibition of MAPK signaling in combination with BCL2-family inhibitors is a novel treatment option for patients suffering from relapsed neuroblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0577cfd3155a7a66d89cff76645a273955a12c7e" target='_blank'>
              High content-imaging drug synergy screening identifies specific senescence-related vulnerabilities of mesenchymal neuroblastomas
              </a>
            </td>
          <td>
            Sonja Herter, Marta Emperador, Kyriaki Smyrilli, Daniela Kocher, Simay Celikyürekli, Constantia Zeiser, Xenia F. Gerloff, Sina Kreth, K. Henrich, Kendra K. Maaß, Johanna Rettenmeier, T. G. Grünewald, H. Peterziel, Frank Westermann, A. Hamacher-Brady, Olaf Witt, Ina Oehme
          </td>
          <td>2025-08-25</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Metastasis is a hallmark of cancer and is responsible for the majority of cancer-related deaths. Evidence suggests that even a single cancer cell can spread and seed a secondary tumour. However, not all circulating tumour cells have this ability, which implies that dissemination and distal growth require adaptative mechanisms during circulation. Here we report that constriction during microcapillary transit will trigger reprogramming of melanoma cells to a tumorigenic cancer stem cell-like state. Using a microfluidic device mimicking physiological flow rates and gradual capillary narrowing, we showed that compression through narrow channels lead to cell and nuclear deformation, rapid changes in chromatin state and increased calcium handling through the mechanosensor PIEZO1. Within minutes of microcapillary transit, cells show increased regulation of transcripts associated with metabolic reprogramming and metastatic processes, which culminates in the adoption of cancer stem cell-like properties. Squeezed cells displayed elevated melanoma stem cell markers, increased propensity for trans-endothelium invasion, and characteristics of enhanced tumorigenicity in vitro and in vivo. Pharmacological disruption of channel activity inhibited the stem cell-like state, while the selective PIEZO1 activator Yoda1 primed this state irrespective of constriction. Finally, deletion of PIEZO1 led to complete abrogation of the constriction-induced stem cell-like state. Together, this work demonstrates that compressive forces during circulation can reprogram circulating cancer cells to tumorigenic stem cell-like states that are primed for extravasation and metastatic colonization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3810403f83f02e75d7eba1f1e769e5b2b7eed299" target='_blank'>
              Capillary constrictions prime cancer cell tumorigenicity through PIEZO1
              </a>
            </td>
          <td>
            G. Silvani, C. Kopecky, S. Romanazzo, V. Rodríguez, A Das, E. Pandzic, J. G. Lock, C. Chaffer, K. Poole, K. Kilian
          </td>
          <td>2025-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Classical Hodgkin lymphoma (cHL) is a biologically and clinically unique malignancy characterized by rare Hodgkin and Reed–Sternberg (HRS) cells surrounded by a dense and diverse inflammatory infiltrate. These malignant cells actively reshape the tumor microenvironment (TME) through metabolic reprogramming and immune evasion strategies. This review synthesizes current knowledge on how metabolic alterations contribute to tumor survival, immune dysfunction, and therapeutic resistance in cHL. We discuss novel therapeutic approaches aimed at disrupting these processes and examine the potential of combining metabolic interventions with immune-based strategies—such as immune checkpoint inhibitors (CPIs), epigenetic modulators, bispecific antibodies, and CAR-T/CAR-NK cell therapies—which may help overcome resistance and enhance anti-tumor responses. Several agents are currently under investigation for their ability to modulate immune cell metabolism and restore effective immune surveillance. Altogether, targeting metabolic vulnerabilities within both tumor and immune compartments offers a promising, multifaceted strategy to improve clinical outcomes in patients with relapsed or refractory cHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84b70765d93f8c830e92294e485ce37dc955d43b" target='_blank'>
              Metabolic Interactions in the Tumor Microenvironment of Classical Hodgkin Lymphoma: Implications for Targeted Therapy
              </a>
            </td>
          <td>
            M. Kurlapski, Alicja Braczko, Paweł Dubiela, Iga Walczak, B. Kutryb-Zając, J. Zaucha
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT Multiple myeloma (MM) orchestrates a profound disruption of immune balance within the bone marrow (BM) microenvironment, driving disease progression and therapeutic resistance. To better understand these complex immune dynamics, we used high-dimensional mass cytometry (CyTOF) profiling to comprehensively characterize the immune landscape of the BM across different stages of myeloma progression, including MGUS (n = 16), smoldering MM (SMM; n = 25), and active MM, both newly diagnosed (n = 43) and relapsed/refractory (n = 104). Our analysis revealed substantial immune remodeling during disease progression, characterized by adaptive immune suppression and extensive infiltration of innate immune populations. Transformation from MGUS to SMM was marked by significant alterations in central and effector memory T helper cells, effector cytotoxic T cells, and an enrichment of monocytic and neutrophil subsets. Active MM stages were further distinguished by increased expansion of myeloid and monocytic lineages, alongside a pronounced reduction in progenitor and transitional B cells. Correspondence analysis demonstrated that specific immune profiles were significantly associated with clinical outcomes, including progression-free survival and overall survival. This study highlights the potential of CyTOF-based molecular profiling to unravel the intricate immune dynamics of the BM microenvironment across MM disease stages, enhancing our understanding of MM pathogenesis and providing a foundation for identifying prognostic biomarkers and tailoring precision immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7c28c48facee0fbd69c0cc410926fbb87e89302" target='_blank'>
              CyTOF profiling of bone marrow immune dynamics across myeloma stages
              </a>
            </td>
          <td>
            Danka Cholujová, G. Beke, Ľuboš Kľučár, Ľ. Drgoňa, Zuzana Valuskova, M. Leiba, E. Kastritis, David M. Dorfman, Kenneth C Anderson, J. Jakubikova
          </td>
          <td>2025-08-18</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC), a prevalent and highly lethal malignancy, is notorious for its aggressive nature and inherent tendency to metastasize, posing significant challenges in clinical management and prognosis. Hypoxia, a pivotal characteristic of the tumor microenvironment (TME) in hepatocellular HCC, is intimately linked to disease progression and unfavorable patient outcomes, underscoring its critical role in shaping the malignant behavior of this cancer. Methods Our research leveraged single-cell RNA sequencing technology to dissect the heterogeneity of the HCC TME, focusing on hypoxia-related genes. To probe the effects of hypoxia on HCC invasion, we developed the Cell Hypoxia-Related Prognostic Feature (CHPF). We analyzed transcriptome data from the The Cancer Genome Atlas (TCGA) database and the GSE149614 dataset, employing computational methods such as UMAP, Weighted correlation network analysis (WGCNA), and CellChat to identify hypoxia cells, characterize cell subsets, and elucidate intercellular communications. Results Our analysis revealed significant heterogeneity in hypoxia cell populations within the HCC TME, with distinct expression patterns of hypoxia-related genes in neoplastic and immune cells. Our analysis revealed distinct hypoxia subpopulations within HCC, with significant overexpression of genes like MEG3, KLF6 and JUN in hypoxia cells. We identified a unique hypoxia subpopulation with high invasive potential and constructed a prognostic model based on H2-specific transcription factors including LRP10、MED8、NOL10、NOP58 and REXO4. The model demonstrated significant predictive value for lifespan of patients as verified in the TCGA dataset and an external validation group. Conclusion Key transcription factors like NOP58, MED8 play pivotal roles in hypoxia-induced HCC invasion and metastasis, and a predictive model based on these factors forecasts HCC survival. Our findings provide novel molecular markers and therapeutic targets for HCC, highlighting the importance of considering the hypoxia TME in diagnostic and treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03201-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85919f351fa84b022b25ebecfed4ffc0c988bac6" target='_blank'>
              Deciphering hypoxia’s role in hepatocellular carcinoma prognosis with single-cell approaches
              </a>
            </td>
          <td>
            Jun Liang, Di Chen, Huiyu Liang
          </td>
          <td>2025-07-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Advances in treating primary breast tumors and extracranial metastases have inadvertently led to a rise in brain metastases, which are severe and carry bleak prognostic outcomes. Thus, there is an urgent need to investigate the cellular and molecular drivers of metastatic central nervous system (CNS) tumors. The tumor microenvironment (TME) is critically important for metastasis, as metastatic cells must alter their new environment to survive and grow. Our laboratory's recent studies have pinpointed platelet-derived growth factor receptor beta (PDGFRβ) signaling as a critical pathway in promoting breast cancer-associated brain metastases (BCBM). Key findings from this published work involved data emanating from a genetically engineered mouse model (GEMM) with hyperactive PDGFRβ-D849V in the mesenchymal lineage driven by Fsp1-cre. These mice exhibit increased BCBM when injected with murine mammary tumor cells secreting the ligand PDGFB, making them an effective model for studying BCBM. However, hyperactivity of PDGFRβ in these mice occurs in all mesenchymal cells. To investigate cell-specific PDGFRβ functions in the CNS, we analyzed publicly available single-cell RNA-seq data of the adult murine brain. This analysis identified astrocyte populations expressing both Pdgfrb and the Fsp1/S100A4 mesenchymal lineage marker. To probe the role of PDGFRβ in reactive astrocytes, we developed a Gfap-cre GEMM model crossed with our mutant PDGFRβ hyperactivation allele (Gfap-cre;Pdgfrb-D849V). These mice exhibit increased GFAP expression in the absence of tumor and decreased overall survival upon intracardiac injection of PDGFB secreting breast cancer cells. Furthermore, flow cytometry at day 10 post intracardiac injection indicates potential differences in the immune response of the brain TME compared to controls. Current work is focused on identifying specific transcriptomic changes in astrocyte populations due to both PDGFRβ hyperactivation as well as metastatic breast cancer colonization of the brain. Overall, this work aims to identify potential therapeutic targets to treat BCBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/065fae0e35727b10f37e95370e754abb9b46417c" target='_blank'>
              BSBM-08 ASTROCYTE PDGFRβ HYPERACTIVATION IN BREAST CANCER BRAIN METASTASIS
              </a>
            </td>
          <td>
            Nathaniel Grabinski, Johnathon Scheibel, Jesse Reardon, Paul Lee, Jonathan Godbout, S. Sizemore, Jessica Winter, Gina M. Sizemore
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background: Lung adenocarcinoma (LUAD) continues to pose a major challenge in cancer treatment, characterized by low survival rates, particularly in metastatic instances. Although immunotherapy has become a common approach for treating LUAD, its success rate is merely 20%, hindered by the absence of effective biomarkers. This limitation is likely influenced by the tumor's immune microenvironment. CD40LG, an important immune molecule, has emerged as a crucial factor in modulating the tumor immune environment and affecting the outcomes of immunotherapy. Nevertheless, its exact mechanisms in LUAD remain unclear, requiring more research. Methods: Through a correlation analysis of prognosis and clinical traits across multiple cancers, we determined that CD40LG is significant in LUAD. This role was confirmed using various public datasets. We evaluated CD40LG protein expression in 94 LUAD and nearby non-cancerous tissues via immunohistochemistry. To gauge immune cell infiltration, we employed multiplex immunofluorescence staining on tissue microarrays. Additionally, in vitro assays were carried out to explore the effects of CD40LG modulation on the behavior of LUAD cells. Results: Various cancers, including LUAD, exhibited down-regulation of CD40LG, which correlated with a worse prognosis. In LUAD tissues, higher CD40LG expression was associated with longer Progression-Free Survival (PFS) and Overall Survival (OS). Moreover, CD40LG expression negatively correlated with the TNM stage and T stage in LUAD. Elevated CD40LG levels were linked to increased infiltration of CD8+ T cells. In vitro studies showed that modulating CD40LG affected LUAD cell metastasis and proliferation. Conclusion: Our study demonstrates the pivotal role of CD40LG in LUAD, proposing its potential utility as a biomarker for prognosis and immunotherapy. The correlation between CD40LG expression, immune cell infiltration, and clinical outcomes emphasizes its importance in tumor-immune dynamics and the efficacy of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6101ef3e2ce499e361f51adb84fd48dc1ec645f4" target='_blank'>
              CD40LG Downregulation in Lung Adenocarcinoma: A Prognostic Biomarker Linked to Immune Cell Infiltration and Survival Outcomes
              </a>
            </td>
          <td>
            Peng Tang, Ruihao Zhang, Yuqi Zhang, Yuan Ao, Yanan Wang, Guannan Wang, Wenjun Wang, Junmei Xu, Fang Hu, Guangsheng Zhu
          </td>
          <td>2025-08-22</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Patient-derived cancer cells (PDCCs) have emerged as a key strategy for advancing personalized cancer treatment. Unlike traditional cancer cell lines, PDCCs retain the genetic and phenotypic characteristics of the patient’s original tumor and can more accurately reflect tumor biology. This review explores recent advances in methods for culturing PDCCs, highlighting the role of these models in drug discovery and high-throughput screening of personalized therapeutic options. By establishing living models directly from patient tumors, PDCCs can more faithfully recapitulate tumor heterogeneity and microenvironmental features than traditional cell lines. These cultures bridge laboratory research and clinical reality, allowing functional testing of patients' cancer cells. Despite the promise of PDCCs, their culture remains fraught with challenges, including the extremely low number of cancer cells that can be obtained, difficulty maintaining tumor heterogeneity, low culture initiation success rates, and ethical considerations for using patient tissues. In addition, controversy remains regarding the reproducibility of results between different laboratories and patient samples. By examining the field’s current state, this review identifies gaps in the application of PDCCs, such as limited modeling capabilities for specific tumor types and the lack of comprehensive, scalable protocols for broad clinical use. This article discusses future directions, including integration with advanced microengineering and AI-driven analysis, which have the potential to overcome existing limitations and optimize PDCCs-based therapeutic strategies. PDCCs are expected to transform the future of cancer treatment as they ultimately provide more accurate drug testing and personalized medicine models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/350a1202d8af5d4a9d6b419293e7569e23ebec7c" target='_blank'>
              Advancements and challenges in culturing patient-derived cancer cells for personalized therapeutics
              </a>
            </td>
          <td>
            Yatian Fu, B. L. Khoo, Chwee Teck Lim
          </td>
          <td>2025-09-08</td>
          <td>Frontiers in Lab on a Chip Technologies</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70d735b44b6ee6328d790d08642f35e3daac2ffe" target='_blank'>
              Integrated in vivo combinatorial functional genomics and spatial transcriptomics of tumours to decode genotype-to-phenotype relationships.
              </a>
            </td>
          <td>
            Marco Breinig, Artem Lomakin, Elyas Heidari, M. Ritter, G. Rukhovich, Lio Böse, Luise Butthof, Lena Wendler-Link, Hendrik Wiethoff, Tanja Poth, Felix Sahm, Peter Schirmacher, Oliver Stegle, M. Gerstung, Darjus F. Tschaharganeh
          </td>
          <td>2025-07-28</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="In this review, we explore the complexities of objectively assessing the response to immunotherapy in mycosis fungoides (MF), a prevalent form of cutaneous T-cell lymphoma. The core challenge lies in distinguishing between reactive and malignant lymphocytes amidst treatment, particularly given the absence of uniform pathological biomarkers for MF. We highlight the vital role of emerging histological technologies, such as multispectral imaging and spatial transcriptomics, in offering a more profound insight into the tumor microenvironment (TME) and its dynamic response to immunomodulatory therapies. Drawing on parallels with melanoma—another immunogenic skin cancer—our review suggests that methodologies and insights from melanoma could be instrumental in refining the approach to MF. We specifically focus on the prognostic implications of various TME cell types, including CD8+ tumor-infiltrating lymphocytes, natural killer (NK) cells, and histiocytes, in predicting therapy responses. The review culminates in a discussion about adapting and evolving treatment response quantification strategies from melanoma research to the distinct context of MF, advocating for the implementation of novel techniques like high-throughput T-cell receptor gene rearrangement analysis. This exploration underscores the urgent need for continued innovation and standardization in evaluating responses to immunotherapies in MF, a field rapidly evolving with new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/547dbbe3b1a147ee04b2d5b3f9a0b851da1e8162" target='_blank'>
              Dermatopathological Challenges in Objectively Characterizing Immunotherapy Response in Mycosis Fungoides
              </a>
            </td>
          <td>
            Amy Xiao, A. Karunamurthy, Oleg E. Akilov
          </td>
          <td>2025-07-29</td>
          <td>Dermatopathology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly heterogeneous malignancy, characterized by low tumor cellularity, a dense stromal response, and intricate cellular and molecular interactions within the tumor microenvironment (TME). Although bulk omics technologies have enhanced our understanding of the molecular landscape of PDAC, the specific contributions of non-malignant immune and stromal components to tumor progression and therapeutic response remain poorly understood. Methods: We explored genome-wide DNA methylation and transcriptomic data from the Cancer Genome Atlas Pancreatic Adenocarcinoma cohort (TCGA-PAAD) to profile the immune composition of the TME and uncover gene co-expression networks. Bioinformatic analyses included DNA methylation profiling followed by hierarchical deconvolution, epigenetic age estimation, and a weighted gene co-expression network analysis (WGCNA). Results: The unsupervised clustering of methylation profiles identified two major tumor groups, with Group 2 (n = 98) exhibiting higher tumor purity and a greater frequency of KRAS mutations compared to Group 1 (n = 87) (p < 0.0001). The hierarchical deconvolution of DNA methylation data revealed three distinct TME subtypes, termed hypo-inflamed (immune-deserted), myeloid-enriched, and lymphoid-enriched (notably T-cell predominant). These immune clusters were further supported by co-expression modules identified via WGCNA, which were enriched in immune regulatory and signaling pathways. Conclusions: This integrative epigenomic–transcriptomic analysis offers a robust framework for stratifying PDAC patients based on the tumor immune microenvironment (TIME), providing valuable insights for biomarker discovery and the development of precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec3422e79e1fa464a39c9ceade9afd8d95e43c98" target='_blank'>
              Deciphering the Heterogeneity of Pancreatic Cancer: DNA Methylation-Based Cell Type Deconvolution Unveils Distinct Subgroups and Immune Landscapes
              </a>
            </td>
          <td>
            Barbara Mitsuyasu Barbosa, Alexandre Todorovic Fabro, Roberto da Silva Gomes, C. Rainho
          </td>
          <td>2025-09-05</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive blood cancer in which disease initiation and relapse are driven by leukemic cells with stem-like properties, known as leukemic stem cells (LSCs). The LSC compartment is highly heterogenous and this contributes to differences in therapy response. This heterogeneity is determined by genetic and nongenetic factors including somatic mutations, the cell of origin, transcriptional and epigenetic states as well as phenotypic plasticity. While this complicates the identification and eradication of LSCs, it also presents an opportunity to tailor therapeutic strategies to the phenotypic and functional states of LSCs present in a patient, exploiting their specific vulnerabilities. The emergence of single-cell multiomics technologies has transformed our ability to dissect cellular heterogeneity in AML, enabling simultaneous interrogation of genomic, transcriptomic, epigenomic and proteomic layers and providing high-resolution molecular snapshots of individual cells. In this review, we discuss causes and consequences of LSC heterogeneity, highlight advances in single-cell multiomics technologies to resolve it and outline how they can address shortcomings in our understanding of LSC heterogeneity and plasticity to revolutionize diagnostics and disease monitoring of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07654c74db7467e7588089114a2ee5e54a7d1bf9" target='_blank'>
              Resolving leukemic stem cell heterogeneity and plasticity with single-cell multiomics.
              </a>
            </td>
          <td>
            Frank Y Huang, Andreas Trumpp, Patrick Stelmach
          </td>
          <td>2025-08-01</td>
          <td>Seminars in hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) is a global health challenge shaped by a complex tumor microenvironment (TME), where myeloid‐driven inflammatory and immunosuppressive processes play central roles. Understanding the spatial and dynamic interactions within the TME is essential for uncovering novel regulatory mechanisms. We integrated spatial transcriptomics, single‐cell RNA‐sequencing, and network pharmacology to develop a comprehensive analytical framework for deconstructing cellular and molecular interactions across spatial domains within the TME. Using this approach, we investigated the potential mechanisms by which the Sparganii Rhizoma‐Curcumae Rhizoma (SCP) Chinese medicine (TCM) herbal pair may influence primary CRC and liver metastases. Our analysis suggests that in primary CRC, SCP may affect neutrophil‐enriched domains through modulation of CEACAM‐related signaling, while in liver metastases, it may influence macrophage‐dominated domains associated with APP signaling. These findings highlight possible domain‐specific molecular interactions and suggest that SCP may regulate intra‐ and inter‐domain signaling patterns relevant to TME remodeling. This study establishes a novel integrative methodology to investigate spatial domain‐specific mechanisms within the TME. Our findings reveal the multi‐dimensional effects of SCP, providing a precision‐targeted, TME‐centric therapeutic strategy for CRC. The framework aligns with TCM principles and offers new avenues for personalized cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64108e2457f88f143ed78e3504c4067d48fd77bb" target='_blank'>
              Spatial Domain‐Based Approach to Analyze the Mechanism of Sparganii Rhizoma‐Curcumae Rhizoma Pair in the Treatment of Colorectal Cancer
              </a>
            </td>
          <td>
            Yuan-Jie Liu, Qian-wen Ye, Yi‐lan Jin, Qian Zhang, Xi Zou, Jie-pin Li
          </td>
          <td>2025-08-01</td>
          <td>Food Science & Nutrition</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Functional cellular heterogeneity in tumours often underlies incomplete response to therapy and relapse. Previously, we demonstrated that the growth of the paediatric brain malignancy, sonic hedgehog subgroup medulloblastoma, is rooted in a dysregulated developmental hierarchy, the apex of which is defined by characteristically quiescent SOX2+ stem-like cells. Integrating gene expression and chromatin accessibility patterns in distinct cellular compartments, we identify the transcription factor Olig2 as regulating the stem cell fate transition from quiescence to activation, driving the generation of downstream neoplastic progenitors. Inactivation of Olig2 blocks stem cell activation and tumour output. Targeting this rare OLIG2-driven proliferative programme with a small molecule inhibitor, CT-179, dramatically attenuates early tumour formation and tumour regrowth post-therapy, and significantly increases median survival in vivo. We demonstrate that targeting transition from quiescence to proliferation at the level of the tumorigenic cell could be a pivotal medulloblastoma treatment strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27908f257548bd97b59f7a3c88e807b37fb0d43b" target='_blank'>
              38 OLIG2 MEDIATES A RARE TARGETABLE STEM CELL FATE TRANSITION IN SONIC HEDGEHOG MEDULLOBLASTOMA
              </a>
            </td>
          <td>
            Kinjal Desai, Siyi Wanggou, Erika Luis, Heather Whetstone, Chunying Yu, Robert J. Vanner, Hayden J. Selvadurai, Lilian Lee, Jinchu Vijay, J. E. Jaramillo, Jerry Fan, P. Guilhamon, M. Kushida, Xuejun Li, Gregory Stein, Santosh Kesari, Benjamin D. Simons, Xi Huang, Peter B. Dirks
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Colorectal cancer (CRC) arises in the colorectal tissue driven by genetic disorder or the accumulation of somatic mutations, leading to abnormal epithelial cell growth. In this study, we employed single-nucleus multi-omics analysis, including single-nucleus RNA-seq and single-nucleus ATAC-seq, on over 100,000 high-quality nuclei to investigate the molecular landscape of both primary tissue and patient-derived organoids (PDOs). Our analysis showed that normal PDOs (N-PDOs) derived from tissue adjacent to tumors replicate the cellular composition and differentiation trajectory of colorectal crypts. In contrast, tumor PDOs (T-PDOs) showed patient-specific transcriptomic and epigenomic heterogeneity yet consistently maintained a stem cell-like state. T-PDOs retained the somatic mutation profile of the primary tumor while also exhibiting de novo mutations not detected in either the primary tumor or N-PDOs. Notably, inferred cell-cell interaction analysis highlighted the activin signaling pathway as a potential unique feature of fibroblast-epithelial interactions within the tumor microenvironment. This study provides a comprehensive view of the transition from normal to malignant colorectal epithelium and underscores the utility of PDOs as a faithful model for capturing both conserved and patient-specific features of colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/044d9b62ee3fa5808d22e4c1cdf93a59f76f395e" target='_blank'>
              Single-cell multi-omics characterize colorectal tumors, adjacent healthy tissue and matched (tumor) organoids identifying CRC-unique features.
              </a>
            </td>
          <td>
            Zhijun Yu, Merel Derksen, Brigit M Te Pas, Sabrina Ladstätter, Rene Overmeer, Peter Brazda, Marc van de Wetering, F. Pourfarzad, Robert G. J. Vries, Wout Megchelenbrink, Christoph Bock, L. Altucci, H. Stunnenberg
          </td>
          <td>2025-08-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract The presence of a single metastatic lesion significantly decreases overall survival in patients with head and neck squamous cell carcinoma (HNSCC), and invasion of malignant keratinocytes is one of the initial steps required for HNSCC metastasis. Histological grading of tumor cell invasion predicts outcome in HNSCC, yet the molecular factors that determine the extent of invasion, and subsequent grading are not fully understood. Using a 3D organ culture model and multiple patient‐derived HNSCC keratinocytes representing all major anatomical subsites of the disease, we identified a range of cell states that represent a continuum of epithelial‐to‐mesenchymal (EMT) characteristics. We also demonstrated how these cell states change in response to TGF‐beta stimulation and co‐culture with cancer‐associated fibroblasts in organ cultures. Using 3D culture models that recapitulate the pattern of invasion seen in primary tumors from which the keratinocytes were derived, we identified distinct clusters of partial‐EMT marker expression in individual patient HNSCC keratinocyte populations. Partial‐EMT transcription factors were correlated with separate invasive characteristics, and we demonstrated that ZEB2 (a known EMT driver) and HIC1 (a novel EMT driver) are central nodes in HNSCC keratinocyte invasion. Collectively, our findings refine the concepts of partial‐EMT and tumor cell invasion, and identify potential therapeutic targets for future development. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57d784069c4c2d528e9675a4a5572b4390059179" target='_blank'>
              Partial‐EMT cell state correlates with single cell pattern of invasion in head and neck SCC keratinocytes
              </a>
            </td>
          <td>
            Pyung Hun Park, Lauren E Israel, Michael H Alexander, Grace Tartaglia, Harvey G South, Suhao Han, Joseph M. Curry, Adam J Luginbuhl, Andrew P. South
          </td>
          <td>2025-07-30</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Background SPOCK1, a matricellular glycoprotein, has been implicated in tumor progression, metastasis, and the tumor immune microenvironment, yet its specific roles in breast cancer (BRCA) remain unclear. This study aimed to systematically explore the expression pattern, prognostic significance, mutation landscape, immune association, and spatial localization of SPOCK1 in breast cancer through integrated multi-omics analyses. Methods Transcriptomic, genomic, and clinical data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) were utilized. Bulk RNA sequencing and single-cell RNA sequencing (scRNA-seq) analyses were conducted, including functional enrichment, immune infiltration assessments, mutation profiling, and transcription factor activity analysis. Multiplex immunohistochemistry (mIHC) was performed to validate the spatial distribution of SPOCK1+ cancer-associated fibroblasts (CAFs) within the tumor microenvironment. Statistical analyses were performed using R and GraphPad Prism. Results SPOCK1 was broadly overexpressed in multiple cancer types and significantly associated with poor prognosis in BRCA. High SPOCK1 expression correlated with immune checkpoint activation, enhanced immune infiltration, and enriched metastasis-related pathways such as epithelial–mesenchymal transition (EMT) and TGF-β signaling. Single-cell analysis identified CAFs as the primary cell population expressing SPOCK1, with spatial mIHC confirming their close proximity to tumor cells. Furthermore, SPOCK1-high CAFs exhibited stronger intercellular communications with malignant cells via collagen, fibronectin, and IGFBP signaling pathways, alongside distinct transcription factor and metabolic profiles. In breast cancer CAF cell lines with knockdown of ANXA2 we found that the expression of both SPOCK1 and IGF1 was reduced. Conclusion SPOCK1 serves as a critical regulator of breast cancer progression, influencing tumor metastasis and reshaping the immune microenvironment via CAF-mediated mechanisms. These findings suggest that targeting SPOCK1+ CAFs could offer new therapeutic opportunities for breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6ec4d3a2efd20e6cfad0b54c715da9bed1f6fb" target='_blank'>
              SPOCK1 promotes the progression of breast cancer by modulating cancer-associated fibroblasts and exerts a synergistic effect with ANXA2
              </a>
            </td>
          <td>
            Yuan Jie, Xin Fei, Meng Fan
          </td>
          <td>2025-08-05</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Brain metastases (BM) critically reduce breast cancer (BC) patients’ survival. Extravasation is pivotal for BM development, but the underlying events at the blood-brain barrier (BBB) remain elusive. We aimed to unravel the players and mechanisms governing BC cells (BCCs)-BBB interaction. For that, mixed cultures of human brain microvascular endothelial cells (HBMECs), mimicking the BBB, and brain-tropic triple-negative BCCs (MDA-MB-231 Br4), or non-brain-tropic (MDA-MB-231) or non-metastatic cells (MCF-7) were established. Temporal and spatial analysis of BCCs-BBB interactions (live-cell imaging automated microscopy), and assessments of endothelial-to-mesenchymal transition (EndMT) markers, transcription factors, cytoskeletal proteins, and morphology (immunocytochemistry) were performed. BBB integrity (permeability, transendothelial electrical resistance) and endothelial migration (wound-healing) were also assessed. Our results revealed that contrasting with non-metastatic and non-brain-tropic cells, BCCs quickly developed an invasive, migratory phenotype, characterized by invadopodium formation and reduced roundness. Spatial analysis showed different positioning of BCCs relative to the BBB endothelium over time, with 14% of BCCs transmigrated after 3 h, compromising BBB integrity through endothelial holes, reduced tightness, and increased permeability. Prior to transmigration, alterations in adhesion markers (E-selectin, ICAM-1, CD24, CD34, β3-integrin, Sialyl-Lewis X) were observed. EndMT was also evident by decreased endothelial (β-catenin and pan cytokeratin) and increased mesenchymal (vimentin, neuronal-cadherin, Slug, ZEB1) markers, elongation (RhoA, α-SMA), nuclear deformation, and migratory capacity. Caveolin-1 silencing in HBMEC decreased BCCs transmigration. This study reveals significant BBB phenotypic and structural changes, facilitating both paracellular and transcellular BCCs transmigration. These findings provide advanced understanding of BCCs trafficking across the BBB, aiding strategy development to prevent extravasation and BM. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-01985-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58375842dd28bad8d863cc4ae660178921a26a5b" target='_blank'>
              Unveiling the intricate dynamics of the interplay between triple-negative breast cancer cells and the blood-brain barrier endothelium
              </a>
            </td>
          <td>
            Ana Rita Garcia, Ana Rita Vaz, Rui Malhó, Hugo M. Botelho, Inês Figueira, M. A. Brito
          </td>
          <td>2025-08-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b1a443d05c1334b2d0a0dff8460a523dfdb2d5" target='_blank'>
              A three-dimensional ex vivo model recapitulates in vivo features and drug resistance phenotypes in childhood acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Magdalini Kanari, Iria Jimenez Garcia, Fabio Steffen, L. Krattiger, Charles Bataclan, Wangjie Liu, Benjamin R. Simona, Bart Deplancke, O. Naveiras, M. Ehrbar, B. Bornhauser, Jean-Pierre Bourquin
          </td>
          <td>2025-09-10</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Aberrantly elevated Ras signals, triggered by various distinct genetic mutations, are frequent features in myeloid leukemias. Normal hematopoiesis requires perpetual and balanced production of different blood cell lineages by multipotent hematopoietic stem cells (HSC). Stem- and progenitor- cells combine dormancy with proliferative drive and require finely tuned metabolism and protein translation rates. Due to the scarcity of stem cells, it has remained largely unknown how aberrantly elevated Ras signals may impact frequency, lineage potential, and quiescent metabolism in rare HSC. Using single cell proteomics and computational analyses, we characterized the effects of induced oncogenic mutant KRasG12D or overexpression of the Ras activator RasGRP1, compared to normal native hematopoiesis. The two Ras pathway lesions drive shared profound skewing towards and expansion of mature myeloid cells. The resolution of CyTOF unmasked opposing patterns for the HSC- and progenitor- compartments: Overexpression of RasGRP1 induced expansion of both subsets, whereas KRASG12D resulted in depletion. By combining spectral flow with SCENITHÔ, a method to quantitate protein translation as a proxy for metabolic state, we first corroborated that immature cells display low metabolic SCENITHÔ rates. Both RasGRP1 and KRASG12D drive elevated, mean SCENITHÔ signals in immature hematopoietic cells. However, RasGRP1-overexpressing stem cells retain a metabolically quiescent cell-fraction, whereas this fraction is incompatible with KRASG12D. Our temporal proteomics and metabolomics datasets provide mechanistic insights into altered hematopoiesis at single cell resolution and support the idea that the exact identity and duration of signals from Ras lesions has profound impacts on stem cell maintenance and lineage-potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8833491efb56cc3a2592617d3966eb43f088b40f" target='_blank'>
              Single cell proteomics characterization of bone marrow hematopoiesis with distinct Ras pathway lesions.
              </a>
            </td>
          <td>
            Laila Karra, Anna-Marie Finger, Lauren A Shechtman, M. Krush, Rita Meng-Yao Huang, Morgan Prinz, Iliana Tennvooren, Kriti Bahl, Lisiena Hysienaj, Paulina Garcia-Gonzalez, Alexis J. Combes, Hugo Gonzalez, R. Argüello, Mathew Spitzer, Jeroen P. Roose
          </td>
          <td>2025-07-30</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The major event that leads to death from breast cancer (BrCa) is the emergence of micrometastases into lethal growing metastases. While it is still uncertain what regulates the cell fate decision between remaining in dormancy and aggressive proliferative progression, accumulating evidence demonstrates a major role for the metastatic microenvironment. One area of interest is that of tissue and circulating mesenchymal stem cells (MSCs), which have been shown to alter the proliferative and metastatic potential of BrCa. Herein, we investigate how these cells impact the phenotype of metastatic BrCa. As the disseminated BrCa cells initially adopt an epithelial phenotype in ectopic organs, one that is dormant in having limited proliferation and being immune-silent, interactions that revert the disseminated metastatic BrCa to aggressive mesenchymal phenotypes, would be a driver of metastatic progression. BrCa cells exhibited phenotypic changes including increased E-cadherin expression, altered proliferation, and differential sensitivity to TRAIL-induced apoptosis when directly co-cultured with immortalized human MSCs, compared to the BrCa cells not co-cultured. These regulatory effects were dependent upon the BrCa cell’s epithelial–mesenchymal status and involved distinct juxtacrine and paracrine signaling mechanisms, as evidenced by differing responses in direct co-culture, conditioned medium, and Transwell systems. Our findings highlight the complex and context-dependent roles of MSCs in BrCa progression, improving our understanding of tumor-stroma interactions and laying groundwork for future therapeutic exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa88923bf15cd98418c8a15636d1fa54e1c47607" target='_blank'>
              Context-Dependent Modulation of Breast Cancer Cell E-Cadherin Expression, Mitogenesis, and Immuno-Sensitivity by Immortalized Human Mesenchymal Stem Cells In Vitro
              </a>
            </td>
          <td>
            Bei Dai, Neha Atale, A. Clark, A.J. Wells
          </td>
          <td>2025-08-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions that drive progression and pathobiology. The aim of this study was to understand how these spatial patterns and interactions evolve with treatment.


METHODS
To explore these relationships, we employed imaging mass cytometry to measure the expression of 34 protein markers, enabling the identification of GBM-specific cell types and their interactions at single-cell protein level in paired primary (pre-treatment) and recurrent (post-treatment) GBM samples from five patients.


RESULTS
We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes.


CONCLUSIONS
These findings provide insight into treatment-induced shifts in GBM's cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fcab07cdc7f1fdc819d108cb3fdc13fdcffb099" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment.
              </a>
            </td>
          <td>
            Shoaib Ajaib, Joshua Winter-Luke, Richard J Digby, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F Stead
          </td>
          <td>2025-09-01</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Introduction Gallbladder cancer (GBC) is a highly lethal cancer with a poor prognosis. The adenoma-carcinoma sequence is a recognized model for GBC development, but its underlying mechanisms are not well understood. Methods Human specimens were collected from Shengjing Hospital of China Medical University. Single-cell isolation and sequencing were conducted on cells from four GBC and four gallbladder adenomatous lesions (GBA) samples, and the raw gene expression matrices were analyzed using R software with the Seurat package. This included cell type annotation, differential gene expression analysis, functional enrichment, and gene set score calculation. Additional analyses such as protein-protein interaction network, immune infiltrate analysis, high-dimensional weighted gene co-expression network analysis, and cell-cell communication analysis were also performed. Results The study revealed that epithelial-mesenchymal transition (EMT) plays a key role in the malignant transformation of epithelial cells from GBA to GBC. The immune landscape of GBC is predominantly immunosuppressive compared to the inflammatory environment within GBA. A specific subset of fibroblasts with extracellular matrix remodeling capabilities appears to be a major driver of the TME differences between GBC and GBA, potentially acting through COL1A2-mediated cell communication. Discussion This work highlights the distinct roles of various cell types in the TME of GBA and GBC, and emphasizes the importance of understanding the mechanisms of malignant transformation from adenomatous lesion to carcinoma in the gallbladder. The findings pave the way for further research into the mechanisms underlying the adenoma-carcinoma sequence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c69e4ae92e5b5b54d3287bb06a6bb410ddac5b0" target='_blank'>
              Extracellular matrix remodeling fibroblasts govern the tumor microenvironment disparity between adenomatous lesions and adenocarcinoma in gallbladder
              </a>
            </td>
          <td>
            Chuhan Ma, Huixin Hu, Hanrong Li, Bing Han, Chao Lv, Yu Tian
          </td>
          <td>2025-07-18</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94d39a50858ad986c811d60a70891ccb3b663404" target='_blank'>
              Single-cell spatial analysis with Xenium reveals anti-tumour responses of CXCL13 + T and CXCL9+ cells after radiotherapy combined with anti-PD-L1 therapy
              </a>
            </td>
          <td>
            Shunsuke A. Sakai, Hidekazu Oyoshi, M. Nakamura, T. Taki, Kotaro Nomura, H. Hojo, H. Hirata, A. Motegi, Yuka Nakamura, Junko Zenkoh, K. Aokage, Akira Hamada, M. Kojima, Takeshi Kuwata, K. Tsuchihara, T. Akimoto, J. Soh, Tetsuya Mitsudomi, Masahiro Tsuboi, Genichiro Ishii, Yutaka Suzuki, Ayako Suzuki, Riu Yamashita, S. Kageyama
          </td>
          <td>2025-07-16</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) are ectopic lymphoid aggregates often found in chronic inflammatory conditions, including cancer. These structures, which share many cellular and functional features with secondary lymphoid organs, can profoundly influence the tumor microenvironment by promoting local anti-tumor immune activation. TLSs have been observed in various cancers, including melanoma, and are associated with improved responses to immunotherapy and clinical outcomes. However, our understanding of the molecular mechanisms underlying TLS formation and function remains incomplete. This review summarizes the current findings on TLSs in human melanoma, drawing from multiple studies to provide an updated overview. We discuss the cellular composition, spatial distribution, and genetic signatures of TLSs at different stages of melanoma pathogenesis and in subtypes including acral, uveal, and desmoplastic melanoma. Additionally, we examine the influence of tumor mutational burden (TMB) and complement activation on TLS formation, as well as the role of TLSs in immune checkpoint inhibitor therapy. We also highlight the potential of TLSs as indicators for disease progression and treatment response, and review preclinical strategies aimed at inducing TLSs to improve therapeutic outcomes. This synthesis aims to support ongoing research into the role of TLSs in melanoma immunobiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cd5d327c6ce7e444f0890b04b9badc06dcf384c" target='_blank'>
              Tertiary Lymphoid Structures in Human Melanoma: Molecular Mechanisms and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Gelare Ghajar-Rahimi, Ishika Patel, Nabiha Yusuf
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The expression and location of proteins in tissues represent key determinants of health and disease. Although recent advances in multiplexed imaging have expanded the number of spatially accessible proteins1–3, the integration of biological layers (that is, cell structure, subcellular domains and signalling activity) remains challenging. This is due to limitations in the compositions of antibody panels and image resolution, which together restrict the scope of image analysis. Here we present pathology-oriented multiplexing (PathoPlex), a scalable, quality-controlled and interpretable framework. It combines highly multiplexed imaging at subcellular resolution with a software package to extract and interpret protein co-expression patterns (clusters) across biological layers. PathoPlex was optimized to map more than 140 commercial antibodies at 80 nm per pixel across 95 iterative imaging cycles and provides pragmatic solutions to enable the simultaneous processing of at least 40 archival biopsy specimens. In a proof-of-concept experiment, we identified epithelial JUN activity as a key switch in immune-mediated kidney disease, thereby demonstrating that clusters can capture relevant pathological features. PathoPlex was then used to analyse human diabetic kidney disease. The framework linked patient-level clusters to organ disfunction and identified disease traits with therapeutic potential (that is, calcium-mediated tubular stress). Finally, PathoPlex was used to reveal renal stress-related clusters in individuals with type 2 diabetes without histological kidney disease. Moreover, tissue-based readouts were generated to assess responses to inhibitors of the glucose cotransporter SGLT2. In summary, PathoPlex paves the way towards democratizing multiplexed imaging and establishing integrative image analysis tools in complex tissues to support the development of next-generation pathology atlases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23c2b08a570600ff687ba656919842c350bb0fc7" target='_blank'>
              Pathology-oriented multiplexing enables integrative disease mapping
              </a>
            </td>
          <td>
            Malte Kuehl, Y. Okabayashi, Milagros N Wong, Lukas Gernhold, Gabriele Gut, Nico Kaiser, Maria Schwerk, Stefanie K Gräfe, Frank Y Ma, J. Tanevski, P. S. L. Schäfer, Sam Mezher, Jacobo Sarabia del Castillo, Thiago Goldbeck-Strieder, O. Zolotareva, Michael Hartung, F. M. Delgado Chaves, Lukas Klinkert, Ann-Christin Gnirck, M. Spehr, David Fleck, Mehdi Joodaki, Victor Parra, Mina Shaigan, M. Diebold, M. Prinz, Jennifer Kranz, Johan M. Kux, Fabian Braun, Oliver Kretz, Hui Wu, Florian Grahammer, Sven Heins, Marina Zimmermann, Fabian Haas, Dominik Kylies, Nicola Wanner, Jan Czogalla, Bernhard Dumoulin, Nikolay Zolotarev, Maja Lindenmeyer, Pall Karlson, J. R. Nyengaard, M. Sebode, S. Weidemann, Thorsten Wiech, Hermann-Josef GroÈne, N. Tomas, Catherine Meyer-Schwesinger, C. Kuppe, R. Kramann, Alexandre Karras, Patrick Bruneval, P. Tharaux, D. Pastene, Benito A Yard, Jennifer A. Schaub, Phillip J. McCown, Laura Pyle, Y. Choi, Takashi Yokoo, Jan Baumbach, Pablo J. Sáez, Ivan G. Costa, J. Turner, Jeffrey B. Hodgin, Julio Saez-Rodriguez, T. Huber, Petter Bjornstad, Matthias Kretzler, O. Lenoir, D. Nikolic-Paterson, L. Pelkmans, Stefan Bonn, Victor G Puelles
          </td>
          <td>2025-07-18</td>
          <td>Nature</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are cells shed from tumors into the bloodstream, providing a unique source of information on tumor biology. CTCs can be collected through liquid biopsy, offering a noninvasive option to track cancer progression and treatment responses. Whereas ctDNA gives valuable insights into genomic alterations, CTCs allow for the direct analysis of cellular protein and RNA expression, essential for identifying targetable markers and resistance mechanisms. Resistance to cancer therapies is often due to genetic and phenotypic adaptations within tumor cells. CTC analysis can help track these adaptations, providing real-time insights into evolving resistance mechanisms and suggesting alternative therapies.


SIGNIFICANCE
Enumeration and molecular characterization of CTCs provide unique insights into the biology of cancer metastasis as well as clinically relevant information on tumor evolution, risk assessment, and therapy response in individual patients with solid malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80509a0d0dd4086ca6871a4474226159831e914e" target='_blank'>
              From Discovery to Diagnosis: A Perspective for Circulating Tumor Cells in Personalized Oncology.
              </a>
            </td>
          <td>
            C. Alix-Panabières, Klaus Pantel
          </td>
          <td>2025-08-18</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a09d2a9d5ee2e4cda19d7cf627d28deb93a6522c" target='_blank'>
              Multi-modal analysis of pediatric pilocytic astrocytomas reveals tumor location-associated cellular and transcriptional heterogeneity.
              </a>
            </td>
          <td>
            J. Lammers, Jacob S Rozowsky, F. Calkoen, Thijs J M van den Broek, J. Meesters-Ensing, Tiago Carvalheiro, Tiago P J Rijsdijk, Robyn D Gartrell, D. V. van Vuurden, H. Stunnenberg, E. Hoving, P. Wesseling, J. van der Lugt, M. Kranendonk, Lennart A. Kester
          </td>
          <td>2025-07-28</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive and lethal brain tumor with very poor prognosis despite recent progress in multimodal treatments. Within glioma tissue, various niche cells such as macrophages and neutrophils form a unique glioma immune microenvironment (GIME) by interacting with heterogenous cancer cells, and this has been implicated in disease progression and responsiveness to immunomodulatory therapies. This study explores novel potential prognostic markers associated with the GIME using integrated bioinformatics analyses, including single-cell RNA-sequencing (scRNA-seq), and spatial transcriptome (ST) datasets of clinical GBM specimens. We first identified 42 genes as being associated with poor prognosis in GBM from 5 different cohorts, GBM vs. nontumor tissue, grade IV vs. grade II gliomas, isocitrate dehydrogenase (IDH)-wild-type vs. IDH-mutant variants, mesenchymal vs. proneural and classical subtypes, and hazard ratio for overall survival. Among these, 32 genes were positively correlated with ESTIMATEScore, infiltration of various immune cell types, expression of known immune-related genes, and representative immune-associated biological signals. On scRNA-seq analysis, 7 genes were relatively concentrated in tumor-associated macrophages rather than in malignant cells. ST analysis revealed that Collagen beta(1-O)galactosyltransferase 1 (COLGALT1), Integrin subunit beta 2 (ITGB2), and Myosin light chain 12A (MYL12A) were distributed in the interface between the tumor and the peritumoral area, overlapping with the expression of representative immune-related genes. These findings support the potential of COLGALT1, ITGB2 and MYL12A as biomarkers for predicting the prognosis and immune responses of GBM, which can help in the development of potential immunotherapeutic strategies for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76a83bcafe86bab51686dcebc0d98499aabea491" target='_blank'>
              Potential Prognostic Markers for Glioblastoma Associated with the Glioma Immune Microenvironment.
              </a>
            </td>
          <td>
            Kazuya Tokumura, E. Hinoi
          </td>
          <td>2025-09-01</td>
          <td>Biological & pharmaceutical bulletin</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="ABSTRACT Background Colorectal cancer (CRC) remains a major challenge in oncology, necessitating the identification of new therapeutic targets. This study aimed to explore the CRC microenvironment and uncover potential targets using single‐cell RNA sequencing (scRNA‐seq). Methods Single‐cell RNA sequencing data from GEO (GSE164522, GSE132465, and GSE144735) were integrated and stratified based on CRC clinical stages and tumor grades. Cell clustering analysis identified distinct cellular subsets, while survival and functional enrichment analyses evaluated the prognostic significance and biological activity of differentially expressed genes. Additionally, transcription factor activity and pseudotime trajectory analyses elucidated cellular dynamics during CRC progression. Cell–cell communication analysis identified key ligand‐receptor interactions involved in immune regulation. Finally, multiplex immunofluorescence was used to validate the expression of key markers and cells in CRC tissues from patients. Results In the low differentiated colorectal adenocarcinoma (LDCA) grade of CRC, the proportion of T cells significantly increased, accompanied by the enrichment of Treg cells and the upregulation of genes associated with immunosuppression, proliferation, and differentiation. Notably, TGFβ1+ Tregs significantly expanded in LDCA, accumulating in late tumor grades with enhanced immunosuppressive capacity and self‐proliferation. Pseudotime analysis confirmed their dominance in late CRC, reinforcing immunosuppression through signal amplification and sustained expansion. Concurrently, CD8+ T cells in the LDCA grade exhibited a progressive loss of cytotoxic function, transitioning into an exhausted state while concurrently activating apoptotic pathways, leading to a profound impairment of antitumor immunity. Cell–cell communication analysis further demonstrated that TGFβ1+ Treg exhibits the strongest interactions with CD8+ T cells, with KLRB1‐CLEC2D, LGALS9‐HAVCR2, and TNFSF10‐TNFRSF10B emerging as pivotal ligand‐receptor pairs displaying significantly enhanced signaling in the LDCA grade. Conclusion Collectively, TGFβ1+ Treg may mediate CD8+ T cell dysfunction through these ligand‐receptor interactions, accelerating T cell exhaustion and apoptosis, thereby fostering a profoundly immunosuppressive tumor microenvironment that ultimately drives CRC immune evasion and malignant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b126f27787b7f75c899e3f95b02d5186b9c098e5" target='_blank'>
              Integrated Single‐Cell RNA‐Seq Reveals Immunosuppressive Mechanisms of Treg Cell Differentiation and Tumor Microenvironment Interactions in Colorectal Cancer
              </a>
            </td>
          <td>
            Dongwei Li, Jingya Liu, Lixian Yu, Xina Luo, Xiaomei Wu, Ronghui Chen
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Glioblastoma multiforme is one of the most aggressive brain tumors, with limited treatment success due to its ability to suppress the immune system and resist conventional therapies. This review explores the emerging role of a unique subset of immune cells, known as γδ T cells, which can identify and destroy tumor cells without relying on traditional antigen presentation pathways. The authors aim to examine how these cells function within the glioblastoma environment and evaluate their potential as a novel immunotherapeutic approach. By highlighting both the opportunities and challenges associated with γδ T cell-based strategies, this research may support the development of innovative therapies that improve outcomes for patients with glioblastoma and potentially other solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/194395a11a9c2161120f21f2463344f5a6fdda44" target='_blank'>
              γδ T Cells in Glioblastoma Multiforme: Novel Roles and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Costanza Dieli, R. Maugeri, A. M. Corsale, Marta Di Simone, Claudia Avellone, Francesco Dieli, D. G. Iacopino, Lara Brunasso, Alessandra Cannarozzo, R. Costanzo, Silvana Tumbiolo, S. Meraviglia
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Metaplastic breast cancer (MpBC) is defined by the presence of various morphological elements, typically biphasic, with epithelial (e.g. no-special type (NST), squamous) and mesenchymal (e.g. spindle, chondroid, osteoid) components. The established clonality of the different components favours an evolution model encompassing either a multipotent progenitor, or a linear metaplastic conversion. We used methylation profiling and showed that different morphologies have specific methylation profiles. Furthermore, our spatial transcriptomic approach, using 10× Genomics Visium and trajectory analysis, evidenced that spindle cells form a transition between the originating carcinoma of no-special type (NST) and pleomorphic regions, with osteoid differentiation likely to be an end-stage fate of the chondroid growth pattern, supporting the conversion model of lineage differentiation. We have also identified a series of master transcription factors likely to regulate these processes, and are significantly associated with metaplastic-like clinical features. This data further supports the conversion model of metaplasia and warrants functional analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077285caa17e07f895159bb1ece8c70f06837f91" target='_blank'>
              Integrated profiling of metaplastic breast cancer identifies putative master regulators of intratumoral heterogeneity
              </a>
            </td>
          <td>
            Yufan Feng, Albert Xiong, Onkar Mulay, Anna Sokolova, M. Lim, Benjamin Van Haeringen, Natasha McGuire, Xavier M. de Luca, Peter T Simpson, Quan Nguyen, S. Lakhani, A. M. McCart Reed
          </td>
          <td>2025-08-11</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4146020be8bb663490d3534c28c485f753647c10" target='_blank'>
              Machine learning integration of bulk and single-cell RNA-seq data reveals glycolytic heterogeneity in colorectal cancer.
              </a>
            </td>
          <td>
            Yuanyuan Du, Zefeng Miao, Peng Li, Dan Feng, Mulin Liu, Aifang Ji, Shijun Li
          </td>
          <td>2025-08-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Glioblastoma is a highly aggressive and devastating brain malignancy with dismal prognosis and extremely limited therapeutic options. Identification of prognostic biomarkers and therapeutic targets from multi‐omics data is critical for improving patient outcomes. In this study, we investigated the clinical significance of cellular heterogeneity and super‐enhancer‐driven regulatory networks, which are critically implicated in glioblastoma progression and treatment resistance. We first performed scRNA‐seq to dissect tumour microenvironment heterogeneity, identifying 16 distinct cell clusters, including astrocytes, macrophages, and CD8+ T cells. CellChat analysis revealed key intercellular signalling pathways, with astrocytes and macrophages acting as central communication hubs. To integrate bulk RNA sequencing data, we applied the Scissor algorithm to identify survival‐associated cell states. By combining single‐cell and bulk transcriptomic data, we uncovered 642 survival‐related genes, including QKI and RBM47, which robustly predicted patient survival and immunotherapy response. Furthermore, WGCNA analysis identified seven co‐expression modules and super enhancer‐regulated networks orchestrated by transcription factors (RFX2, RFX4) and hub genes (NEAT1, CFLAR). These networks stratified patients into high‐ and low‐risk groups with significant survival differences. Collectively, our findings elucidate the intricate interplay between cellular heterogeneity and super enhancer‐driven gene regulation in glioblastoma, providing a translational framework for targeting oncogenic hubs and modulating microenvironment interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7616587b7549f42937faeb12de6e3dcf343225e7" target='_blank'>
              Integration of Single‐Cell RNA and Bulk RNA Sequencing Reveals Cellular Heterogeneity and Identifies Survival‐Associated Regulatory Networks in Glioblastoma
              </a>
            </td>
          <td>
            Zijun Xu, Bohan Xi, Jiaming Huang, Liqiang Zhang, Sifu Cui, Xianwei Wang, Dong Chen, Shupeng Li
          </td>
          <td>2025-08-13</td>
          <td>IET Systems Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT In humans, cell-cell communication orchestrates tissue organization, immune coordination, and repair, yet spatially mapping these interactions remains a challenge for biology. We introduce STARComm, a scalable-interpretable computational method that identifies Multicellular Communication Interaction Modules (MCIMs) by detecting spatially co-located receptor-ligand activity from high-plex spatial transcriptomics in 2D and 3D. Applied to an atlas of >14million cells across 8 cancers, STARComm revealed 24 conserved and tumor-specific MCIMs, including a fibro-immune module with targetable axes linked to immune exclusion and immunotherapy resistance. In chronic graft-versus-host disease, STARComm identified three salivary gland MCIMs predictive of patient death and two druggable axes ( CXCL12-CXCR4 , CCL5-SDC4 ), both with FDA-approved therapeutics. STARComm demonstrated that peripheral tissue profiling can forecast fatality nearly 3 years in advance using minor salivary glands. By enabling scalable biomarker discovery, drug targeting, and spatially resolved precision profiling, STARComm bridges the gap between spatial biology and clinical translation, advancing the field of spatial medicine. SUMMARY Despite major advances in spatial biology, no framework has yet linked spatially resolved intercellular communication networks, independent of cell types, to clinical outcomes in human disease. Here, we present STARComm, a scalable method that identifies Multicellular Interaction MCIMs (MCIMs). Applying STARComm to minor salivary gland biopsies from patients with chronic graft-versus-host disease (GVHD), we identify MCIMs that not only distinguish healthy from diseased tissue but also stratify patient survival. High-risk MCIMs are enriched for actionable immune and stromal pathways, including those targetable with existing therapies. These findings establish the first outcome-linked spatial communication framework in any human disease and highlight the translational potential of oral tissues as minimally invasive platforms for real-time immune diagnostics, prognostic modeling, and therapeutic screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a093b13aff7d0f5c2b6d575cc2d04f2b86e47ba" target='_blank'>
              STARComm Scalably Detects Emergent Modules of Spatial Cell-Cell Communication in Inflammation and Cancer
              </a>
            </td>
          <td>
            Khoa Huynh, B. Matuck, Deiziane de Souza, XiuYu Zhang, Giancarlo Fatobene, Luiz Alberto Valente Soares Júnior, L. F. Ferraz da Silva, V. Rocha, K. Byrd, Jinze Liu
          </td>
          <td>2025-08-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8261c73bf75d788913526d1e389ee8b4f740f28c" target='_blank'>
              Inhibition of ACVR1 in Cancer-Associated Fibroblasts Suppresses Colorectal Cancer Cell Growth.
              </a>
            </td>
          <td>
            Shinya Kato, N. Miyoshi, S. Fujino, Masafumi Horie, Shinichi Yachida, Mitsunobu Takeda, Y. Sekido, Tsuyoshi Hata, Atsushi Hamabe, T. Ogino, Mamoru Uemura, Hirofumi Yamamoto, Masayoshi Yasui, M. Ohue, Y. Doki, H. Eguchi
          </td>
          <td>2025-07-31</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Glioma intratumoral heterogeneity remains a critical barrier to effective treatment, driving recurrence and resistance to therapy. Emerging evidence suggests that glioma cells acquire neural-like features through paracrine and synaptic communication with neural cells, fostering functional diversity within tumors. While neuro-cancer interactions are implicated in glioma heterogeneity, their precise roles remain incompletely synthesized. Here, by consolidating these discoveries, a conceptual framework is proposed for understanding the glioma intratumoral heterogeneity shaped by neuro-cancer interactions: cellular phenotypes, spanning neurogliomal synapses, glioma networks, and neuronal-like motility; and transcriptional states, which exhibit remarkable resemblance to neural cells. These diverse cellular phenotypes and transcriptional states synergistically fuel glioma progression, invasion, and resistance. By emphasizing the converging phenotypical and transcriptional evidence with spatial context within them, an underexplored but critical role of neuro-cancer interactions are proposed in glioma intratumoral heterogeneity. This provides potential strategies to explore and disrupt these neuro-cancer interactions, offering new insights to address glioma intratumoral heterogeneity for improved therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7eca8bd6e6e1b76e76ba8a64aea76418740245" target='_blank'>
              Neuro-Cancer Interactions Shape Glioma Intratumoral Heterogeneity.
              </a>
            </td>
          <td>
            Tong Wu, Peiran Zhao, Zheng Chen, Tianlei Ying, Zhihong Nie, Wei Hua, Huiyan Li, Ying Mao, Wenbo Bu
          </td>
          <td>2025-08-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we demonstrate that EZH2 regulates multilineage cell states dependent on the RNA binding protein Tristetraprolin (TTP) that mediates RNA stability and activation of translation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration or enzalutamide. Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1223372b66c898bfc66493d1d8cf7781f04f51" target='_blank'>
              EZH2-TTP-mTORC1 Axis Drives Phenotypic Plasticity and Therapeutic Vulnerability in Lethal Prostate Cancer
              </a>
            </td>
          <td>
            Leigh Ellis, Beatriz German, Katherine L. Morel, Teia Noel, Nadia Boufaied, Deborah L. Burkhart, Sujun Chen, Felipe Dezem, Xintao Qiu, H. Long, Stefan DiFazio, S. Baca, Ayesha Shafi, Matthew L Freedman, Himisha Beltran, Christopher J Sweeney, H. He, Myles Brown, Jasmine T Plummer, Simon Knott, David Labbé
          </td>
          <td>2025-08-21</td>
          <td>Research Square</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Glioma represents the most prevalent primary tumors of the central nervous system, originating from glial cells. Cancer stem cells have the ability to extensively proliferate, self-renew, and form colonies, which contribute to tumorigenesis. Studies have found a population of cells within glioblastoma exhibiting characteristics similar to those of cancer cells, termed glioma stem cells (GSCs). GSCs have two distinct phenotypes: mesenchymal subtype (MES) and proneural subtype (PN). Despite the vital role of these subtypes in glioma biology, there is a significant lack of comprehensive reviews focusing on the regulatory mechanisms underlying each phenotype.This review integrates emerging insights into the regulatory mechanisms underlying GSCs plasticity, with dedicated analysis of novel pathways governing PN and MES phenotypes and their dynamic transition. By examining these critical elements, we aim to contribute to the development of novel therapeutic strategies in the ongoing fight against gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50da6e5130555aeaeba55f2b83f42650df5fbf68" target='_blank'>
              Phenotypic variations in glioma stem cells: regulatory mechanisms and implications for therapeutic strategies
              </a>
            </td>
          <td>
            Guofeng Tian, Yifu Song, Yaochuan Zhang, Liang Kan, Ana Hou, Sheng Han
          </td>
          <td>2025-09-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The spatial heterogeneity of gene expression has driven the development of diverse spatial transcriptomics technologies. Here, we present photocleavage and ligation sequencing (PCL-seq), a spatial indexing method utilizing a light-controlled DNA labeling strategy applied to tissue sections. PCL-seq employs photocleavable oligonucleotides and ligation adapters to construct transcriptional profiles of specific regions of interest (ROIs) designated via microscopically controlled photo-illumination. In frozen mouse embryos, PCL-seq generates spatially aligned gene expression matrices and produces high-quality data, detecting approximately 170,000 unique molecular identifiers (UMIs) and 8600 genes (illumination diameter = 100 µm). Moreover, PCL-seq is compatible with formalin-fixed paraffin-embedded (FFPE) tissues, successfully identifying thousands of differentially enriched transcripts in the digits and vertebrae of mouse embryo FFPE sections. Additionally, PCL-seq achieves subcellular resolution, as demonstrated by differential expression profiling between nuclear and cytoplasmic compartments. These characteristics establish PCL-seq as an accessible and versatile workflow for spatial transcriptomic analyses in both frozen and FFPE tissues with subcellular resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57261101fdad09ceeb7cf3dea95fc95b91515b80" target='_blank'>
              High-resolution spatial transcriptomics in fixed tissue using a cost-effective PCL-seq workflow.
              </a>
            </td>
          <td>
            Xue Dong, Mengzhu Hu, Xiaonan Cui, Wenjian Zhou, Jingtao Cai, Guangyao Mao, Weiyang Shi
          </td>
          <td>2025-08-05</td>
          <td>Genome research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfc80059f8767909d0c8bd930a6c33c0803bbb" target='_blank'>
              Preservation and Clonal Behavior of Extrachromosomal DNA in Patient-Derived Xenograft Models of Childhood Cancers.
              </a>
            </td>
          <td>
            Rishaan Kenkre, Jon D. Larson, Owen S. Chapman, J. Luebeck, Yan Yuen Lo, Megan Paul, Wenshu Zhang, V. Bafna, Robert J Wechsler-Reya, Lukas Chavez
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Recent advancements in spatial transcriptomics have been largely triggered by two high-resolution technologies: Visium-HD and Xenium in-situ. While sequencing-based Visium HD features a refined bin size of 2 µm and transcriptome wide coverage, Xenium in-situ is a targeted imaging-based detection technology with sub-micron resolution. Herein we use a publicly available lung dataset which contains Visium-HD and Xenium-5K data generated on identical tissue slides to make a bona-fide technical comparison aligned with thorough pathological annotations. Whilst Visium-HD offers a broader gene coverage for detection and likely detects more tumor subclones, Xenium-5K achieves comparable results when robust clustering algorithms are applied. Importantly, from the pathological point of view, the single-cell segmentation accuracy is essential when analyzing irregularly shaped cells, where Xenium may be in favor. At the opposite side, although Xenium-5K based cell segmentation to delineate immune cells, normal lung, and vasculature at cell resolution is decent, it relies on fluorescent signals for transcript detection, which is challenging in heavily pigmented tissues such as melanoma or dust-laden alveolar macrophages, an application scenario for which Visium HD may stand out. From this perspective, pathological derived factors are the prior consideration for selecting an appropriate ST approach under difference research settings including cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03479-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3b1c5ce3ce737368b581b0a77a6e7fdc9558842" target='_blank'>
              Comparing Xenium 5K and Visium HD data from identical tissue slide at a pathological perspective
              </a>
            </td>
          <td>
            Mengping Long, Taobo Hu, Weixin Wang, Junshun Gao, Nan Wang, Mats Nilsson
          </td>
          <td>2025-07-26</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Stromal cells are found in all tissues of the body. Among them, lymphoid stromal cells (LSCs) correspond to the cell subsets found in secondary and tertiary lymphoid organs. LSC heterogeneity has been characterized in depth in mice based on cell-fate mapping, high-resolution imaging and single-cell RNAseq analysis, and more recently in humans despite the difficulty of accessing these rare cell populations. At steady-state, LSCs organize discrete anatomical niches in lymphoid organs and orchestrate adaptive immune response. Studies of LSCs at the single cell level have identified a wide role for these cells in various pathological conditions, including solid tumors, autoimmune diseases, and lymphomas. In this review, we will discuss the diversity and plasticity of LSCs and LSC-like cells as well as their functions in pathological settings, with a focus on cancer and autoimmune diseases. Altogether, it highlights the importance of increasing our understanding of these cells, to use them as a target for novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae385fc73dd8ff4540908af0f79aa0db93c9021" target='_blank'>
              Lymphoid stroma in all its states
              </a>
            </td>
          <td>
            Nicolas Barbier, Valentin Isen, Karin Tarte, David Roulois
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24a4c1ebff0edca5f86402765983e14a517f49f7" target='_blank'>
              Form Follows Function: A New Paradigm of Pancreatic Cancer Progression.
              </a>
            </td>
          <td>
            William Gasper, Giada Pontecorvi, Zhikai Chi, S. Enrico, Lily C. Goodman, Haneen Amarneh, Evelyn Mazzarelli, Joshua Thomas, Lucia Zhang, Rachel Handloser, Seth LeCates, A. L. Tomescu, Francesca Rossi, Myla Sykes, Anjali Bhatt, Nicholas Fazio, Alessandro Bifolco, Nafeesah Fatimah, Austin Seamann, Cheryl M Lewis, Shelby Melton, Herbert J Zeh, Megan B. Wachsmann, Dario Ghersi, Matteo Ligorio
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Hypoxia is a hallmark of the colorectal cancer (CRC) tumor microenvironment (TME) that drives malignant progression, chemoresistance, and immune evasion. However, the cellular heterogeneity underpinning hypoxic responses in CRC and its impact on prognosis remain incompletely understood. Methods We integrated single-cell RNA sequencing data from 15 CRC samples (GSE166555 and GSE221575) to delineate hypoxic and normoxic cell populations and identify hypoxia-related genes (HRGs). Weighted gene co-expression network analysis (WGCNA) and enrichment profiling elucidated key biological processes associated with hypoxia. Cell–cell communication networks were inferred using CellChat, and transcription factor regulatory modules were reconstructed via SCENIC and GRNBoost2. A hypoxia-based prognostic signature was developed from unique H3 cluster genes using univariate Cox and Lasso regression on The Cancer Genome Atlas (TCGA; n = 606) and validated in GSE39582 (n = 579). Drug sensitivity correlations were derived from the GDSCv2 database. Finally, in vitro assays assessed the functional role of GIPC2, a model gene, in CRC cell lines. Results Single-cell profiling uncovered eight distinct hypoxic clusters, with H3 exhibiting the highest hypoxia scores and enrichment in glycoprotein metabolic and angiogenesis pathways. The eight-gene prognostic model stratified patients into high- and low-risk groups with significantly different overall survival in both TCGA (P = 0.0026) and validation cohorts (P = 0.011). Drug analysis highlighted associations of model genes with PI3K/MTOR and apoptosis pathways. GIPC2 knockdown in LS180 and HT-29 cells markedly inhibited proliferation, migration, and invasion, while inducing apoptosis and reversing EMT phenotypes. Conclusions We present a robust hypoxia-related gene signature that accurately predicts CRC patient prognosis and nominate GIPC2 as a potential biomarker and therapeutic target, offering new insights into hypoxia-driven CRC biology and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8575947e15fbc35fb6bf68728fce7870efdbff1" target='_blank'>
              Decoding the hypoxic tumor microenvironment in colorectal cancer for prognostic modeling and therapeutic target discovery
              </a>
            </td>
          <td>
            Xiao-Cui Duan, Yan Zhou, Fan Feng, Hai-Bo Jiang, Mei-Lin Wang, Zhe Han, Hong-Fei Pang, Yu-Hang Liu, Heng-Zhe Jia, Meng- He, Hongpan Xu, Yuan-Yuan Wang
          </td>
          <td>2025-08-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Recent advances in single-cell studies have unveiled the extensive transcriptomic heterogeneity of cancer-associated fibroblasts (CAFs), with each subset potentially playing unique roles in the tumor microenvironment. Despite these insights, it remains unclear how to culture different subsets of CAFs in vitro while preserving their in vivo phenotype. The inherent plasticity of CAFs, characterized by their ability to dynamically alter their phenotype in response to various environmental stimuli, presents significant challenges in inducing and maintaining specific CAF states in vitro. In this study, we investigated the effects of cell shape and confinement on two-dimensional culture substrates on the transcriptomic profile and phenotype of cultured CAFs. By inducing cytoskeletal rearrangement and activating mechanotransductory pathways through micropatterning of polyacrylamide hydrogels, we observed significant shape- and confinement-dependent changes in cell morphology. Our results showed that micropatterned CAFs exhibited phenotypic shifts towards more myofibroblastic, desmoplastic and inflammatory subsets. Additionally, micropatterning allowed for control over a range of CAF-specific markers and pathways, significantly altering their transcriptome landscape and behavior. Lastly, we demonstrated that micropatterned and non-micropatterned CAFs responded differently to anti-cancer drugs, underscoring the importance of phenotype-oriented therapy that accounts for CAF plasticity and regulatory networks. Control over CAF morphology through micropatterning presents a unique opportunity to establish highly robust CAF phenotypes in vitro, thereby facilitating a deeper understanding of CAF plasticity and heterogeneity. This approach also holds promise for the development of novel therapeutic targets, as it allows for the precise manipulation of CAF states to better model and study their interactions within the tumor microenvironment.



 Aleksandr Mitriashkin, Josephine Yu Yan Yap , N. Gopalakrishna Iyer , Gianluca Grenci, Eliza Li Shan Fong. Cell confinement by micropatterning induces phenotypic changes in cancer-associated fibroblasts [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr LT03.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7eb0c539e2fb60deec8ba647c172fbc76cfd279" target='_blank'>
              Abstract LT03: Cell confinement by micropatterning induces phenotypic changes in cancer-associated fibroblasts
              </a>
            </td>
          <td>
            Aleksandr Mitriashkin, Josephine Yu Yan Yap, N. G. Iyer, Gianluca Grenci, Eliza Li Shan Fong
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Clonally established tumor cell lines often do not recapitulate the behavior of cells in tumors. The sequencing of a whole tumor tissue may not uncover transcriptome profiles induced by the interactions of all different cell types within a tumor. Interferons for instance have a vast number of binding sites in their target genes. Access to the DNA binding sites is determined by the epigenomic state of each different cell type within a tumor mass. To understand how genes such as interferons appear to have both tumor-promoting and tumor-inhibiting functions, single-cell transcript analysis was performed in the breast cancer tissue of HER2+ (epidermal growth factor receptor 2) patients. We identified that potential antagonistic oncogenic activities of cells can be due to diverse expression patterns of genes with pleiotropic functions. Molecular pathways both known and novel were identified and were similar with those previously identified for patients with rheumatoid arthritis. Our study demonstrates the efficacy in using single-cell transcript analysis to gain insight into genes with apparent contradictory or paradoxical roles in oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261a79861db7f4bc06df15b1b4671db2e425c476" target='_blank'>
              Single-Cell mRNA Analysis for the Identification of Molecular Pathways of IRF1 in HER2+ Breast Cancer
              </a>
            </td>
          <td>
            Laura Vilardo, P. Pelucchi, Antonia Brindisi, E. Abeni, E. Piscitelli, E. Mosca, G. Bertalot, Mira Palizban, Theodoros Karnavas, A. Gritzapis, Ioannis Misitzis, Martin Götte, Ileana Zucchi, R. Reinbold
          </td>
          <td>2025-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Effective identification of oncogenic mutations is essential for diagnosis, forecasting resistance, and metastasis in remission. It is required for an optimal drug regimen. We develop a framework to discover mutations that co-exist in different oncoproteins, and those that are excluded, likely encoding oncogene-induced senescence. First, mapping the proteins onto pathways assists combinatorial drug selections and helps to detect metastases. Second, it provides the molecular basis for synthetic lethality, to date investigated at the genome level. Our pan-cancer profiles of ~60,000 tumor sequences, detect 3424 co-existing tumor-specific mutations. Mapping them onto pathways indicates that they preferentially promote specific primary tumors. We uncover metastatic mutations and provide metastatic breast-cancer markers. This work not only clarifies the mechanistic basis of intratumor mutational diversity but usefully reveals markers for metastasis in patients’ genomes and introduces a novel computational framework for detecting metastasis based on tumor mutational profiles. Mapping the mutations onto pathways provides an invaluable metastasis-targeting resource, guiding drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8dbee9272fd01bd4b589963df2977d50caa0e8" target='_blank'>
              Mutations in tumor signaling, metastases, and synthetic lethality establish distinct patterns
              </a>
            </td>
          <td>
            B. R. Yavuz, Uğur Şahin, Hyunbum Jang, Ruth Nussinov, Nurcan Tunçbağ
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background The N-formyl peptide receptor family (FPRs) is implicated in the progression of diverse cancer types, yet studies specifically exploring their roles in breast cancer remain scarce. Methods A comprehensive analysis integrating bulk RNA-seq transcriptomics, methylomics, single-cell transcriptomics, and spatial single-cell transcriptomics data was conducted to elucidate the distinctive characteristics of FPRs in breast cancer. This study particularly focused on delineating the e xpression profiles of FPR3 across distinct breast cancer subtypes, while systematically investigating its prognostic implications and association with macrophage polarization patterns in breast cancer patients. Furthermore, molecular docking analysis was performed to screen potential therapeutic compounds targeting FPR3, providing insights into its druggability. Results Notably, FPR3 was found to be highly expressed in macrophages within breast cancer tissues, with a notably elevated level in HER2-positive and triple-negative breast cancer (TNBC) subtypes, both of which are associated with poor prognosis. FPR3 expression inversely correlates with promoter methylation levels. Further analysis of pan-cancer immune infiltration patterns uncovered a striking association between FPR3 and macrophage infiltration, as well as their polarization status. Knockdown of FPR3 expression in macrophages markedly enhanced the expression of IL6, TNF-α, and TGF-β, while significantly reducing IL10 levels, indicative of a shift towards an M1-like macrophage phenotype. Through computational molecular docking analyses, Otamixaban and Rivaroxaban emerged as promising candidate inhibitors of FPR3. Conclusions These findings underscore the profound infiltration of FPR3 + macrophages in breast cancer patients with adverse prognoses, highlighting FPR3 as a potential therapeutic target for intervening breast cancer aggressiveness. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03942-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18854fe793c2d29227b32ba5f47639fad4edcf8c" target='_blank'>
              FPR3 orchestrates macrophage polarization in breast cancer: multi-omics dissection of prognostic relevance and therapeutic targeting
              </a>
            </td>
          <td>
            Ying Chen, Xin Tang, Liran Zhu, Yi Wang, Gaopeng Li, Wulin Yang
          </td>
          <td>2025-08-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) enables gene expression profiling while preserving the spatial architecture of intact tissue. Analyzing ST data often proceeds by first extracting cell-level information, typically through cell segmentation or cell-type deconvolution. However, they can fail to capture important expression patterns, for example from fragile or underrepresented cell types, subcellular structures like neurites, and extracellular expression. These omissions can result in biased analyses and incorrect or incomplete biological interpretations. We describe a new computational method, RESCUE, that can recover the idiosyncratic spatial expression patterns missed by existing ST analysis methods and enable robust inference even when reference data are incomplete. We validate RESCUE using MERFISH data from the honey bee brain and apply it to multiple ST datasets to demonstrate how it can reveal novel insights into complex tissue biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a368ce21a0742d813d27a95e0edba6cadc1ec63a" target='_blank'>
              RESCUE: recovery of idiosyncratic expression patterns in spatial transcriptomics
              </a>
            </td>
          <td>
            Young Joo Lee, Seokjin Yeo, Alex W. Schrader, Ian M. Traniello, A. C. Ahmed, Gene E. Robinson, Hee-Sun Han, S. Zhao
          </td>
          <td>2025-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) is a highly aggressive brain tumor characterized by aberrant angiogenesis and an immunosuppressive microenvironment. Pericytes are aberrantly recruited but their spatiotemporal roles and molecular changes remain unclear. This study investigated platelet-derived growth factor receptor beta-positive (Pdgfrb+) pericyte dynamics and reprogramming in GBM vasculature.


METHODS
We generated GL261-Luc and GL261-CFP glioblastoma cells via lentiviral transduction and established two transgenic models. (1) For pericyte labeling, Ai14 reporter mice was crossed with PDGFRβ-P2A-CreERT2 mice for tdTomato-specific lineage tracing (PT mice). (2) For conditional ablation, we generated inducible Pdgfrb-expressing cell ablation models (PT mice was crossed with ROSA-DTA mice). An intravital imaging platform (FITC-dextran/CFP/tdTomato + two-photon microscopy) tracked pericytes, vessels, and tumor cells, while FACS-sorted Pdgfrb+ cells from GBM and normal brain were analyzed by LC-MS/MS proteomics.


RESULTS
Cre-mediated ablation of Pdgfrb-expressing cells revealed stage-dependent effects on GBM growth: early ablation inhibited progression while late ablation promoted it. Pericytes undergo dual spatial reorganization in GBM: regional enrichment with pre-sprouting accumulation at the tumor-brain interface, and focal positioning with preferential localization at vascular branch points. Concurrently, GBM vasculature displayed simplified branching, dilation, and pericyte remodeling (shorter processes, higher density). Proteomics revealed 1426 altered proteins, with upregulated proliferation pathways (e.g., matrix metallopeptidase 14 [Mmp14], lysyl oxidase like 2 [Loxl2]) and downregulated homeostasis functions (e.g., transforming growth factor beta 1 [Tgfb1]), validated by scRNA-seq in human GBM.


CONCLUSIONS
This study demonstrates that during early GBM progression, pericytes actively drive tumor angiogenesis through molecular reprogramming toward proliferative and pro-angiogenic phenotypes, with the integrated imaging-proteomics framework revealing potential therapeutic targets for disrupting pericyte-mediated vascular remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/531e6a56e9b350ddd48811a1f650e717592a9882" target='_blank'>
              Insight into pericytes in glioblastoma angiogenesis: In vivo tracking by two-photon microscopy and proteomic profiling.
              </a>
            </td>
          <td>
            Qinghong Wang, Chengyan Ma, Xinpei Wang, Mengyuan Li, Xingjiu Yang, Ran Gao
          </td>
          <td>2025-08-07</td>
          <td>Animal models and experimental medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Monocytes are recruited to tumors and undergo transcriptional reprogramming resulting in tumor-promoting functions. Epigenomic features, such as post-translational modification of histones and chromatin accessibility, are key determinants of transcription factor binding and thereby play an important role in controlling transcriptional responses to the tissue environment. It remains unknown whether systemic tumor-associated signals could alter the epigenomic landscape of peripheral monocytes before they reach the tumor, thus shaping their subsequent response to the tumor microenvironment. Methods We used a combination of genome-wide assays for chromatin accessibility and multiple histone modifications (H3K4me1, H3K4me3, H3K27ac) in a mouse tumor model to investigate changes in the epigenomic landscape of peripheral monocytes. We then integrated these epigenomic data with transcriptomic data to link altered regulatory elements to gene expression changes in monocytes occurring in the periphery or during tumor infiltration. Results We found that tumor-induced systemic inflammation was associated with transcriptional and epigenomic preconditioning of peripheral monocytes. The distal tumor caused extensive remodeling of both H3K4me3+ promoters and H3K4me1+ enhancers. Specifically, this involved the repression of interferon-responsive regulatory elements as well as the establishment of enhancers harboring binding motifs for transcription factor families downstream of pro-inflammatory signaling, such as C/EBP, AP-1, and STAT. Reprogrammed enhancers in peripheral monocytes were linked to sustained gene expression changes that persisted after tumor infiltration. In addition, key pro-tumor genes upregulated in tumor-infiltrating monocytes showed epigenetic priming already in the circulation. Conclusions These results suggest that cancer-associated remodeling of the epigenomic landscape in peripheral monocytes can shape the gene expression programs they acquire in the tumor, highlighting the role of the epigenome in redirecting monocyte function to support cancer progression. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01511-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b6b814b8af78e271c9f391688a7c45b00d193ce" target='_blank'>
              Epigenomic preconditioning of peripheral monocytes determines their transcriptional response to the tumor microenvironment
              </a>
            </td>
          <td>
            Máté Kiss, László Halász, Eva Hadadi, Wilhelm K. Berger, Petros Tzerpos, S. Póliska, Daliya Kancheva, Aurélie Gabriel, Romina Mora Barthelmess, Ayla Debraekeleer, Jan Brughmans, Y. Elkrim, L. Martens, Yvan Saeys, Bence Daniel, Zsolt Czimmerer, Damya Laoui, László Nagy, J. V. Van Ginderachter
          </td>
          <td>2025-07-23</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Simple Summary Luminal and basal molecular subtypes were first described in breast cancer, where they show distinct biological properties and clinical outcomes. Recent research has revealed that these subtypes are also present in many other epithelial cancers, beyond their tissue of origin. Basal tumors are often linked to aggressive behavior, poorer prognosis, and potential sensitivity to treatments such as DNA-damaging agents or immunotherapy. Luminal tumors usually have more favorable outcomes and may respond to hormone-related therapies. In this review, we summarize studies published between 2010 and 2024 that have applied luminal/basal subtyping across different cancer types and within single tumor types. Our aim is to provide readers with a clear overview of how these molecular programs are defined, where they occur, and why recognizing them could help improve cancer classification, prognosis, and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e1cd92bd5684128dd1291ba7c9bf8b475416dcc" target='_blank'>
              Luminal and Basal Subtypes Across Carcinomas: Molecular Programs Beyond Tissue of Origin
              </a>
            </td>
          <td>
            Celia Gaona-Romero, M. E. Domínguez-Recio, I. Comino-Méndez, M. V. Ortega-Jiménez, R. Lavado-Valenzuela, Emilio Alba
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The transition from acute kidney injury to chronic kidney disease is characterized by significant changes in the cellular composition and molecular interactions within the kidney. Utilizing high-resolution Xenium and whole transcriptome Visium spatial transcriptomics platforms, we analyze over a million cells on 12 male mouse kidneys across six stages of renal injury and repair. We define and validate 20 major kidney cell populations and delineate distinct cellular neighborhoods through this multimodal spatial analysis. We further reveal a specific fibro-inflammatory niche enriched in failed-repair proximal tubule cells, fibroblasts, and immune cells, with conserved neighborhood gene signatures across mouse and human. Within this niche, we predict Runx2 as a key upstream regulator, along with platelet-derived growth factor and integrin beta-2 signaling pathways shaping the fibrogenic microenvironment. Altogether, our study provides deep insights into the cellular and molecular dynamics during kidney injury and repair and establishes a comprehensive multimodal analytical framework applicable to other spatial omics studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80e9ed63af964434bd5c50df588ce36470352d4" target='_blank'>
              Multimodal spatial transcriptomic characterization of mouse kidney injury and repair
              </a>
            </td>
          <td>
            Xuanyuan Qiao, Haojia Wu, Hemalatha Sundaramoorthi, Pierre Isnard, Changfeng Chen, Waleed Rahmani, Benjamin D. Humphreys
          </td>
          <td>2025-08-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Gliomas are the most common primary malignant tumors of the central nervous system (CNS), and despite progress in molecular diagnostics and targeted therapies, their prognosis remains poor. In recent years, immunotherapy has emerged as a promising treatment modality in cancer therapy. However, the inevitable immune evasion by tumor cells is a key barrier affecting therapeutic efficacy. Epigenetic regulation, such as DNA methylation, histone modification, and non-coding RNA expression, plays a crucial role in the occurrence, development, and immune evasion of gliomas. These modifications can dynamically regulate gene expression, leading to the silencing of tumor-associated antigens, dysregulation of pro-inflammatory cytokines, and dynamic modulation of immune checkpoints (such as PD-L1). This review systematically elucidates the key mechanisms by which epigenetic regulation promotes immune evasion in gliomas and details three interconnected mechanisms: 1) epigenetic silencing of tumor-associated antigens and antigen-presenting machinery; 2) dysregulation of pro-inflammatory cytokine secretion; and 3) dynamic modulation of PD-L1 expression through chromatin remodeling. We emphasize the potential of combining epigenetic therapies with immunotherapies to enhance anti-tumor immune responses and overcome treatment resistance in gliomas. Future research should focus on developing biomarker-driven epigenetic immunotherapies and exploring the complex interplay between epigenetic modifications, glioma cells, and the tumor immune microenvironment to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd05063be53a8d164f1a96c934eb33dcd3607ec2" target='_blank'>
              Unraveling epigenetic drivers of immune evasion in gliomas: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Dan Wu, Dongen Ju, Yujia Zhao, Wenna Liu, Qingqing Liu, Ying Liang
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a central role in melanoma growth, resistance to treatment, and relapse, however, their dynamic regulatory behavior remains poorly understood. Vibrational spectroscopy offers a unique, label-free approach to investigate cellular heterogeneity at the molecular level. Here, we explored the biochemical and regulatory dynamics of CSCs identified by using a time-course design, integrating infrared and Raman spectroscopies with cell cycle analysis and immunocytochemistry targeting the checkpoint proteins p16 and p21. CSCs, non-cancer stem cells (NCSCs), and bulk CHL-1 melanoma cells were monitored at 11, 24, 48, and 72 h. CSCs showed a steady S-phase with an early rise in p16 followed by a subsequent increase in p21 expression, indicating a dynamic state of cell cycle checkpoints. In contrast, NCSCs and CHL-1 cells showed more transient p16/p21 expression and CHL-1 exhibited a marked p16 increase at 24 h. Spectroscopic analysis revealed that CSCs exhibited distinct vibrational profiles, predominantly in the nucleic acid-, protein- and lipid-associated regions. These differences were further supported by principal component and hierarchical clustering analyses, which consistently distinguished CSCs from NCSCs. Our findings underline the potential of vibrational spectroscopy to sensitively detect CSC-specific regulatory patterns and support its use in detecting new therapeutic targets in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23a07ebb80708549e93a2a342c6077ba0b56ea8" target='_blank'>
              Vibrational spectroscopy unveils distinct cell cycle features of cancer stem cells in melanoma
              </a>
            </td>
          <td>
            Bensu Rüya Uslu, Berrin Ozdil, E. Tarhan, Serdar Özçelik, H. Aktug, Günnur Güler
          </td>
          <td>2025-08-05</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Cyclin dependent kinases 4 and 6 inhibitors have brought great improvements in the treatment of luminal breast cancer, but resistance is a major clinical hurdle. Multiple biomarkers of resistance have been proposed, but none is currently utilized in clinical practice. By performing single-cell RNA sequencing of seven palbociclib-naïve luminal breast cancer cell lines and palbociclib-resistant derivatives, we show that established biomarkers and pathways related to CDK4/6i resistance present marked intra- and inter- cell-line heterogeneity. Transcriptional features of resistance could be already observed in naïve cells correlating with levels of sensitivity (IC50) to palbociclib. Resistant derivatives showed transcriptional clusters that significantly varied for proliferative, estrogen response signatures or MYC targets. This marked heterogeneity was validated in the FELINE trial where, compared to the sensitive ones, ribociclib-resistant tumors developed higher clonal diversity at genetic level and showed greater trascriptional variability for genes associated with resistance. A potential signature of resistance inferred from the cell-line models, positively enriched for MYC targets and negatively enriched for estrogen response markers, was probed on the FELINE trial, separating sensitive from resistant tumors and revealing higher heterogeneity in resistant versus sensitive cells. These data suggest that heterogeneity for CDK4/6 inhibitors resistant markers might facilitate the development of resistance and challenge the validation of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab65930181b447a2afea51540e3bba84721ca231" target='_blank'>
              Single-cell transcriptomics reveals biomarker heterogeneity linked to CDK4/6 Inhibitor resistance in breast cancer cell lines
              </a>
            </td>
          <td>
            I. Migliaccio, M. Bonechi, Dario Romagnoli, Giulia Boccalini, F. Galardi, C. Guarducci, A. Nardone, R. Schiff, L. Biganzoli, L. Malorni, M. Benelli
          </td>
          <td>2025-07-31</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="OBJECTIVE
Adamantinomatous craniopharyngioma (ACP) is an epithelial tumor that occurs in the sellar region of the human brain. Despite resection, relapse is frequent with poor prognosis. To facilitate the development of new therapy for ACP, the authors examined the spatial distribution, cell of origin, and potential biological functions of leukemia inhibitory factor (LIF), an important stem cell self-renewal regulator, in a series of ACP tumors.


METHODS
The transcriptional sites of LIF and LIF receptor (LIFR) were determined by single-cell sequencing and space transcriptome analysis. LIF and LIFR distribution characteristics in different histopathological regions were detected with immunohistochemistry and immunofluorescence analysis. The relationships between the regional distributions of different tissues and tumor imaging characteristics, tumor cell stemness, cell proliferation, LIF expression, and patient prognosis were analyzed.


RESULTS
The authors' analysis of 39 ACPs detected LIF overexpression that was selectively enriched in cell clusters. In addition to the discovery of the stromal cells in the interstitial region, palisade epithelium, and stellate reticulum as the source cells of LIF production, the authors also revealed that LIFR was primarily generated by the cell clusters. Examination of differentially expressed genes between LIF-high and LIF-low ACP tumors indicated that the binding of LIF to LIFR may lead to the activation of the PI3K-AKT signaling pathway. Further analysis showed enrichment of LIF expression in β-catenin-positive cell clusters expressing stem cell markers of CD44, supporting its role in stem cell self-renewal. Integrated analysis with diagnostic imaging found higher level expression of LIF in cystic tumors than in solid tumors, displaying a trend toward poorer prognosis.


CONCLUSIONS
This study confirmed for the first time that LIF in ACP mainly originates from tumor microenvironment stroma. The authors' data suggest that future efforts should also include tumor stromal cells as a novel cellular and/or molecular target when developing new anticancer therapies against ACP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90e38d490dae9950f240b893ac7de6f57947b7a0" target='_blank'>
              Use of integrated spatial transcriptomics and histopathological analysis in adamantinomatous craniopharyngiomas to identify stromal cells as a new cellular source of leukemia inhibitory factor.
              </a>
            </td>
          <td>
            Wenxin Hu, Chuan Zhao, Wenrong Zheng, Yi Lin, Ning Luo, Hongxing Liu, Xingfu Wang, Xueling Qi, Xianlong Wang, Xiao-Nan Li, Zhixiong Lin
          </td>
          <td>2025-08-01</td>
          <td>Journal of neurosurgery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Melanoma brain metastasis (MBM) leads to significant morbidity and mortality. Building on previous findings that WNT5A regulates melanoma metastasis and dormancy in extracranial niches, we hypothesize that WNT5A signaling facilitates melanoma invasion across the blood-brain barrier, while its downregulation drives intracranial metastatic proliferation. To explore this, we conducted functional studies using human melanoma cell lines and an organotypic mouse brain xenograft system. Overexpression or pharmacologic induction of WNT5A increased melanoma migration and invasion toward brain explant conditioned media in transwell assays. Wholemount confocal imaging of brain explant cultures revealed distinct tumor colonization patterns mirroring clinical dormancy and proliferative outgrowth depending on WNT5A level: melanoma overexpressing WNT5A formed single-cell dispersions perivascularly, whereas silencing WNT5A in syngeneic cells produced proliferative clusters. Exogenous recombinant WNT5A also induced G1 cell cycle arrest in FUCCI-transduced melanoma cells, supporting its role in maintaining dormancy. Conversely, the WNT5A antagonist SFRP1 drove melanoma toward increased tumor size and accelerated tumor development. Interestingly, immunohistochemical analysis of postmortem human brain tissue demonstrated robust SFRP1 expression in cerebral endothelium and neurons, indicating a potential source promoting metastatic outgrowth. Integrin profiling revealed elevated expression of adhesion proteins in invasive melanoma compared to WNT5A-silenced syngeneic lines, suggesting an association with enhanced metastatic potential and tumor extravasation. WNT5A overexpression resulted in greater endothelial and extracellular matrix engagement by adhesion assay and live-cell tracking. These interactions were recapitulated in ex vivo xenografts, with WNT5A-high melanoma adhering to endothelial basement laminin. Together, these findings highlight the importance of WNT5A regulation through the metastatic cascade, from endothelial adhesion and establishment of a perivascular dormancy niche to metastatic outgrowth. Further integration of transcriptomics and proteomics on mouse models and clinical MBM samples aim to uncover key modulators of WNT signaling and associated pathways in MBM progression which might be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a305149b00eaf3bc4069c98fdc0c3ac940583e00" target='_blank'>
              BSBM-02 WNT5A MEDIATES MELANOMA BRAIN METASTASIS
              </a>
            </td>
          <td>
            Kevin Zhang, Denys Balandin, M. R. Rocha, Vania Wang, Fan Huang, Cheyenne Palm, Alexis E Carey, Ritvik Koya, Jordina Rincon-Torroella, C. Eberhart, Calixto-Hope G. Lucas, A. Weeraratna
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Emerging evidence underscores the pivotal role of M2-polarized tumor-associated macrophages (M2-TAMs) in orchestrating immunosuppressive tumor microenvironments that fuel metastatic dissemination in prostate cancer (PCa), yet the fundamental mechanisms governing M2-TAM trafficking in lethal PCa progression remain poorly understood. Methods Multi-cohort transcriptomic analyses were performed to identify metastasis-associated genes, with CTSZ prioritized as a key cathepsin linked to prostate cancer progression. Circulating tumor and bone metastatic mouse models were employed to investigate CTSZ-driven M2-TAM infiltration and metastatic behavior. Mechanistic studies included proteasomal degradation assays, IL32 pre-mRNA splicing analysis, and IL-32 binding experiments using RGD motif-dependent interactions. Therapeutic efficacy was tested with the ITGA5 inhibitor GLPG0187 in preclinical models. Results Elevated CTSZ expression shows strong clinical association with advanced pathological progression. CTSZ overexpression in PCa cells drives lung metastasis dissemination and bone metastatic in vivo model but fails to enhance cell-intrinsic oncogenic behaviors in vitro systems. Overexpression of CTSZ promotes M2-TAM infiltration and metastasis by inducing TRA2A degradation via the proteasome pathway, which alleviates TRA2A-mediated suppression of IL32 alternative splicing. Enhanced IL-32 secretion facilitates M2-TAM recruitment through binding to macrophage integrin ITGA5. Pharmacological inhibition of ITGA5 with GLPG0187 significantly reduced metastatic burden and M2-TAM infiltration in vivo. Conclusions The CTSZ/TRA2A/IL-32/ITGA5 axis orchestrates protumoral immunity in PCa metastasis by driving M2-TAM recruitment. Targeting this pathway, particularly through ITGA5 blockade, represents a promising therapeutic strategy to inhibit metastatic progression and remodel the immunosuppressive tumor microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06865-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb3c4538516e4490955f050ba378f4d185bb8971" target='_blank'>
              The CTSZ-TRA2A-IL32 axis defines a targetable macrophage-dependent pathway in metastatic prostate cancer
              </a>
            </td>
          <td>
            Saipeng Chen, Bingqian Deng, Youxin Liu, Langlang Xie, Jie Xu, Ruimin Hu, Fu-Cheng Zhao, Guojing Song, Rongrong Ni, Heting Liu, Zhansong Zhou, Gang Huang, Wenhao Shen
          </td>
          <td>2025-08-05</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Developing patient-derived organoids (PDOs) from metastatic breast cancers offers opportunities for functional drug screening and biomarker discovery, yet their establishment remains technically demanding. To test this methodology for generating PDOs from CNS breast cancer metastases, we successfully derived a PDO from a solid CNS metastasis. The PDOs continued to express proliferative markers and Estrogen Receptor during passaging, demonstrating the feasibility of this approach. We next sought to generate PDOs from breast cancer leptomeningeal disease (LMD), isolating metastatic breast cancer cells from the cerebrospinal fluid (CSF) of affected patients who consented to donate tissue for research. We compared previously reported Type 1 and Type 2 medium formulations for growing breast cancer organoids, as well as a recently described Ocello medium that incorporates conditioned medium from leptomeningeal cells. In our initial experience, we found that LMD organoids could survive in Ocello medium but exhibited slow or no expansion in culture. For comparison, we also tested the growth of metastatic cancers that had spread to other fluid-filled compartments, leading to malignant effusions in the pleural and peritoneal spaces. In contrast to LMD, we found that PDOs from pleural effusions and ascites exhibited rapid expansion of epithelial components for the majority of cases, expressed expected breast cancer markers, and maintained intratumoral heterogeneity. We also demonstrate the ability to isolate T cells from malignant fluid for co-culturing with PDOs to test immunotherapy agents. These results underscore both the potential and limitations of current ex vivo approaches in modeling metastatic disease within fluid compartments. While PDOs from pleural effusions and ascites show promise, refining culture conditions for LMD remains critical. Our future studies will aim to optimize ex vivo expansion by testing additional modifications to culture conditions and continuing the slow expansion of existing viable cells. We will also incorporate in vivo growth efforts using mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dbee1e52c3cc4078fa31acf650940fe9ad8392a" target='_blank'>
              PRCA-03 PATIENT-DERIVED ORGANOIDS FOR MODELING CENTRAL NERVOUS SYSTEM (CNS) METASTATIC BREAST CANCER
              </a>
            </td>
          <td>
            M. Ebadi, Lisa Ni, David P Sheppard, Molly H Blau, Erica L Peters, Indira Dastan, Larissa Muller, Sara Wade, Fei Gao, S. Braunstein, Jonathan T Yang, V. Venur, L. M. Halasz
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In prostate cancer, it is recognized that adenocarcinoma can transdifferentiate into neuroendocrine prostate cancer (NEPC) owing to lineage plasticity; however, transdifferentiation into other histological types remains uncertain. We present a case of a patient who underwent surgery for adenocarcinoma, which later recurred as prostate carcinosarcoma. Genomic analysis revealed a TMPRSS2-ERG fusion, confirming a common clonal origin and transdifferentiation from adenocarcinoma to carcinosarcoma. Additionally, we identified a frameshift mutation in TP53 and the loss of PTEN and RB1. Transcriptome analysis revealed enriched epithelial-mesenchymal transition and immune-related pathways, a pattern distinct from both adenocarcinoma and NEPC. To our knowledge, this is the first report that comprehensively evaluated the clonal origin of the rare prostate carcinosarcoma and characterized it using genomic and transcriptomic sequencing. It enhances our understanding of prostate cancer lineage plasticity and highlights the importance of developing novel therapies specifically targeted at prostate carcinosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc3dc165d184c74305476d6dbd5e449b98486ca" target='_blank'>
              Case Report: Genomic insights into prostate adenocarcinoma transdifferentiation to carcinosarcoma due to lineage plasticity
              </a>
            </td>
          <td>
            Tomohiro Fukui, Arinobu Fukunaga, Yuki Teramoto, Maki Fujiwara, Kensuke Hikami, T. Sunada, Kei Mizuno, Yuki Kita, Takayuki Sumiyoshi, Takayuki Goto, R. Saito, Takashi Kobayashi, S. Akamatsu
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The growth of a tumor is tightly linked to the distribution of its cells along a continuum of activation states. Here, we systematically decode the activation state architecture (ASA) in a glioblastoma (GBM) patient cohort through comparison to adult murine neural stem cells. Modelling of these data forecasts how tumor cells organize to sustain growth and identifies the rate of activation as the main predictor of growth. Accordingly, patients with a higher quiescence fraction exhibit improved outcomes. Further, DNA methylation arrays enable ASA-related patient stratification. Comparison of healthy and malignant gene expression dynamics reveals dysregulation of the Wnt-antagonist SFRP1 at the quiescence to activation transition. SFRP1 overexpression renders GBM quiescent and increases the overall survival of tumor-bearing mice. Surprisingly, it does so through reprogramming the tumor’s stem-like methylome into an astrocyte-like one. Our findings offer a framework for patient stratification with prognostic value, biomarker identification, and therapeutic avenues to halt GBM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dee763118a03533523bd5d7836ea98324ad3594" target='_blank'>
              Cross-species comparison reveals therapeutic vulnerabilities halting glioblastoma progression
              </a>
            </td>
          <td>
            Leo Carl Foerster, Oguzhan Kaya, Valentin Wüst, Diana-Patricia Danciu, Vuslat Akcay, Milica Bekavac, Kevin Chris Ziegler, Nina Stinchcombe, Anna Tang, S. Kleber, Joceyln Tang, Jan Brunken, Irene Lois-Bermejo, Noelia Gesteira-Perez, Xiujian Ma, Ahmed Sadik, P. Le, Kevin Petrecca, Christiane A Opitz, Haikun Liu, C. R. Wirtz, Angela Goncalves, A. Marciniak-Czochra, Simon Anders, A. Martin-Villalba
          </td>
          <td>2025-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Tumor tissues exhibit cell-specific genetic variations and gene expression profiles critical to disease diagnosis, progression, and treatment outcomes. Single cell sequencing offers detailed insights into the heterogeneity of cancer cells which is pivotal for developing personalized treatment strategies. However, current high-throughput methods primarily detect gene expression levels, lacking information on genetic variants such as mutations or gene fusions. Understanding tumor responses to therapy requires linking cell-type specific mutations with gene expression profiles to understand cancer development and progression.



 We developed FocuSCOPE, a high-throughput single cell sequencing solution that simultaneously measures gene expression and detects genetic variantion from individual cells. Single cell partitioning is performed using SCOPE-chip®, a portable microwell chip the size of a microscopy slide. The technology utilizes barcoding microbeads conjugated with oligos to efficiently capture mRNA and target sequences. The optimized chemistry facilitates amplification and library construction of both mRNA and target regions, including point mutations, fusion genes, and non-polyadenylated RNAs such as viral RNAs.



 NB4, CCRF, and K562 cell lines were mixed in equal proportions. Libraries were prepared using the FocuSCOPE Blood Cancer Panel. Experiments demonstrated the specific detection of KRAS and TP53 mutations in mixed cell lines (NB4, CCRF, and K562) using the FocuSCOPE Blood Cancer Panel. The NB4 cell line contained KRAS (A18D) and TP53 (R248Q) mutations, CCRF contained KRAS (G12D) and TP53 (R248Q, R175H), and K562 contained only the BCR-ABL1 fusion gene. Additionally, FocuSCOPE effectively captured PML-RARA and BCR-ABL1 fusion genes with high sensitivity.



 FocuSCOPE enables comprehensive analysis of gene expression profiles and targeted genetic sequences, capturing driver mutations, fusion genes, and non-polyA sequences such as viral sequences with high sensitivity, thus advancing our understanding of tumor biology and therapeutic responses. This makes it an invaluable tool for personalized medicine, guiding therapeutic decisions and improving patient outcomes.



 Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick. FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P17.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafbd9d94e3c2b4f9a27806a0df8f1e19c248cc6" target='_blank'>
              Abstract P17: FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia
              </a>
            </td>
          <td>
            Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="KRAS is mutationally activated in 45–50% of colorectal cancer (CRC) cases, and while KRAS-targeted therapies have shown some clinical promise, upfront and acquired resistance limit their efficacy. To explore the acute response and mechanisms underlying KRAS inhibitor resistance, we used targeted exome sequencing and single-cell spatial transcriptomics to analyze patient-matched pre-treatment, on-treatment, and progression biopsies from patients treated with combined KRASG12C and EGFR inhibition. Acquired genetic events were identified in most patients at progression but were often subclonal and coexisted with transcriptional adaptive states. Mesenchymal, YAP, and fetal-like transcriptional signatures predominated in resistant tumors, while tumor cell-intrinsic inflammatory programs were induced in the early treatment phase. Single-cell spatial analysis revealed significant intratumoral heterogeneity, with diverse adaptive states predominating in different zones of individual tumors. Using human and murine organoid models, we show that these drug-induced inflammatory programs are cancer-cell autonomous and precede the emergence of regenerative fetal-like programs associated with drug resistance. We uncover TBK1 as a promising target to abrogate the early inflammatory adaptive phase and enhance responses to KRAS inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75a7967138a21f862a2f0477898f16663439bb11" target='_blank'>
              Concurrent genetic and non-genetic resistance mechanisms to KRAS inhibition in CRC
              </a>
            </td>
          <td>
            Salvador Alonso, Kevan Chu, Marie J Parsons, Elizabeth Granowsky, H. Gunasinghe, J. Shia, R. Yaeger, Lukas E. Dow
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is characterized by pervasive genomic instability and high inter- and intra-tumor heterogeneity. Approximately half of HGSC tumors harbor homologous recombination deficiency (HRD), rendering them vulnerable to PARP inhibitors and platinum-based chemotherapy. In contrast, patients lacking HRD (HR-proficient, HRP) generally respond poorly to the current therapies. To overcome heterogeneity and identify relevant HGSC subtypes, we characterized the genomic landscape of 640 tumors from 243 patients using whole-genome sequencing. Our chromosomal instability (CIN) signature-based analysis characterized the structural variation landscape and revealed five HGSC subtypes, validated in an independent dataset. Two HRD subtypes, associated with BRCA1- or BRCA2-driven alterations, demonstrated favorable treatment responses. Strikingly, three HRP subtypes emerged, marked by unique structural alterations and gene expression patterns, tumor microenvironment interactions, and different chemotherapy responses. Finally, organoid experiments showed subtype-specific sensitivity to CHK1 inhibition, suggesting prexasertib as a potential targeted treatment for most currently untreatable HRP patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e62a189c25cb112512c18bbcd2dda397ed67ef" target='_blank'>
              Decoding the Genomic and Functional Landscape of Emerging Subtypes in Ovarian Cancer.
              </a>
            </td>
          <td>
            Giulia Micoli, K. Lavikka, Yilin Li, A. Pirttikoski, D. Afenteva, Wojciech Senkowski, G. Marchi, Anna Vaharautio, T. Muranen, Titta Joutsiniemi, S. Hietanen, A. Virtanen, K. Wennerberg, J. Hynninen, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-07-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Cancer remains one of the leading causes of morbidity and mortality globally, driven by genetic alterations, uncontrolled cell proliferation, and metabolic reprogramming. The tumor microenvironment (TME) is a highly dynamic and heterogeneous system composed of tumor cells, immune cells, stromal cells, and extracellular matrix (ECM) components, which influence cancer progression. Tumor-associated macrophages (TAMs), especially those polarized into the M2 phenotype, play a critical role in modulating this environment. M2 macrophages promote tumor progression through mechanisms such as immune suppression, angiogenesis, and metastasis. This polarization is heavily influenced by the altered metabolic landscape of tumors, where the Warburg effect leads to excessive lactate production, which in turn drives M2 polarization through G protein-coupled receptor 132 (GPR132). M2 macrophages secrete cytokines like IL-10, transforming growth factor β (TGF-β), and vascular endothelial growth factor (VEGF), which contribute to immune escape, tumor growth, and metastasis. The metabolic shifts within TAMs, especially the transition from oxidative phosphorylation to glycolysis, further support the pro-tumoral functions of these cells. This review explores the intricate relationship between M2 macrophage polarization bias, tumor metabolism, and the resulting impact on cancer progression, highlighting the potential of targeting these pathways for therapeutic strategies. The findings suggest that M2 macrophage polarization could serve as a key prognostic factor for cancer outcomes and provide a basis for future research into therapeutic interventions that target macrophage polarization and the tumor metabolic milieu.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/805f3bfa800e64033b539fe1d60f9bfbb1355460" target='_blank'>
              Unraveling the connection: M2 macrophage polarization and cancer metabolism
              </a>
            </td>
          <td>
            José Daniel Moreno-Ocampo, Miguel Angel Pardiño-Vega, N. E. Herrera-González
          </td>
          <td>2025-08-05</td>
          <td>Exploration of Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3e7f27fab3fec0c82e9314d5c7fe1bca9a3ba3" target='_blank'>
              A Vascularized Human Organ Chip Reveals SARS-CoV-2 Susceptibility in Developmentally Guided Tissue Maturation
              </a>
            </td>
          <td>
            Titilola D. Kalejaiye, Rohan Bhattacharya, Samira Musah
          </td>
          <td>2025-07-22</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4144c066603d461af67a1609df3300fe3c044e71" target='_blank'>
              Single-cell and spatial transcriptomics reveal mechanisms of radioresistance and immune escape in recurrent nasopharyngeal carcinoma.
              </a>
            </td>
          <td>
            Rui You, Qunlun Shen, Chao Lin, Kangning Dong, Xiao Liu, Hanshi Xu, Wanming Hu, Yulong Xie, Ruoqi Xie, Xiaoyi Song, Chunliu Huang, Jinhui Wu, Tao Yu, Huifeng Li, Zining Wang, Keming Chen, Xiong Zou, Peiyu Huang, Yijun Hua, Youping Liu, Tianliang Xia, Shihua Zhang, Mingyuan Chen
          </td>
          <td>2025-07-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Emerging evidence links alterations in the tumor microbiome to therapeutic responses to immunotherapy. Alterations in β-catenin are among the most frequently observed oncogenic drivers of hepatocarcinogenesis and are associated with T-cell exclusion. However, their effect on the immune cell environment and microbiome in hepatocellular cancer is not well understood. We hypothesized that β-catenin could modulate the immune microenvironment through alterations in the secretome and release of extracellular vesicles (EV) that mediate tumor and immune cell interactions and increase tumor growth within regions with attenuated immune activity and reduced microbial diversity. Methods We used a synthetic transgenic murine model of β-catenin-driven hepatocarcinogenesis to analyze microbiome composition, diversity, and immune cell profiles in vivo. Tumor and stool samples were collected from mice with early- or late-stage hepatocellular carcinoma and were used for profiling. Results The microbiome associated with intrahepatic tumors differs from that in non-tumoral regions, normal liver tissues, and gut tissues. Constitutive β-catenin expression modulates lipopolysaccharide-mediated signaling in macrophages and alters the secretion of immunomodulatory chemokines and cytokines. Tumoral immune cell profiles differed from those in hepatic tissues. EV-mediated signaling between immune cells and epithelial cells with mutant β-catenin and immune cells modulates immune cell populations in vitro and in vivo. Conclusion Mutations in β-catenin can drive immune responses through EV-based tumor cell-immune cell interactions to modulate both the tumor microflora and immune microenvironment. A potential strategy to augment responses to immunotherapy for hepatocellular cancer could target these interactions to restore microbial diversity or immune cell infiltration within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d73e045384f38112d9502014ce3fbc6f907c47" target='_blank'>
              Beta-Catenin Mutations Can Impact on the Interplay Between Tumor and Immune Cells and Hepatic Microbiota in Hepatocellular Cancer
              </a>
            </td>
          <td>
            Yu Ota, Julia Driscoll, Anneliese Hill, Irene K Yan, Tushar Patel
          </td>
          <td>2025-08-01</td>
          <td>Journal of Hepatocellular Carcinoma</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background: Glioblastoma (GBM) is one of the most challenging malignancies in all of neoplasms. These malignancies are associated with unfavorable clinical outcomes and significantly compromised patient wellbeing. The immunological landscape within the tumor microenvironment (TME) plays a critical role in determining GBM prognosis. By mining data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) databases and correlating them with immune responses in the TME, genes associated with the immune microenvironment with potential prognostic value were obtained. Method: We selected GSE16011 as the training set. Gene expression profiles were substrates scored by both ESTIMATE and xCell, and immune cell subpopulations in GBM were analyzed by CIBERSORT. Gene expression profiles associated with low immune scores were performed by lasso regression, Cox analysis and random forest (RF) to identify a prognostic model for the multiple genes associated with immune infiltration in GBM. Then we constructed a nomogram to optimize the prognostic model using GSE7696 and TCGA-GBM as validation sets and evaluated these data for gene mutation and gene enrichment analysis. Result: The prognostic correlation between the six genes (MEOX2, PHYHIP, RBBP8, ST18, TCF12, and THRB) and GBM was finally found by lasso regression, Cox regression, and RF, and the online database obtained that all six genes were differentially expressed in GBM. Therefore, a prognostic correlation model was constructed based on the six genes. Kaplan–Meier (KM) survival analysis showed that this prognostic model had excellent prognostic ability. Conclusions: Prognostic models based on tumor microenvironment and immune score stratification and the construction of related genes have potential applications for prognostic analysis of GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/370e018abc182e267e84e93eb970da89cb35c965" target='_blank'>
              Construction of a Genetic Prognostic Model in the Glioblastoma Tumor Microenvironment
              </a>
            </td>
          <td>
            Wenhui Wu, Wenhao Liu, Zhonghua Liu, Xin Li
          </td>
          <td>2025-07-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) display remarkable functional heterogeneity, yet the molecular mechanisms driving their diverse phenotypes remain elusive. Using CRISPR screens in primary macrophages, we identified tumor-derived factors, including lactic acid, PGE2, and GM-CSF, as key modulators of TAM polarization. These factors interact synergistically and antagonistically to shape distinct TAM phenotypes that are highly conserved across human cancers. Mechanistically, lactic acid and PGE2 jointly induce angiogenic gene programs while suppressing GM-CSF-driven MHC-II expression at the chromatin level, creating mutually exclusive distributions of proangiogenic and MHC-II+ TAMs, which are differentially localized to specific spatial niches in the tumor microenvironment. Furthermore, we showed that shifting TAMs to an interferon-responsive phenotype, triggered by Adar inactivation, significantly promotes the infiltration of effector CD8+ T cells through specific receptor-ligand interactions. These findings uncover a conserved mechanism of TAM polarization and offer insights into therapeutic strategies for TAM reprogramming to potentiate cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cb6bdb7d04c92080704c1d38a2b724c54141d29" target='_blank'>
              Functional genetic screens reveal key pathways instructing the molecular phenotypes of tumor-associated macrophages.
              </a>
            </td>
          <td>
            Youxue Lu, Ce Luo, Lanxiang Huang, Genyi Wu, Lihan Zhong, Jieyu Chu, Fubing Wang, Zexian Zeng, Deng Pan
          </td>
          <td>2025-09-04</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: Heterogeneous circulating tumor cells (CTCs) have been implicated in the formation of new metastases. However, circulating cells expressing both tumor and immune cell proteins are often dismissed as insignificant findings in CTC studies. Methods: Two non-contemporaneous blood samples from a metastatic breast cancer patient were analyzed using an enrichment-free platform to identify canonical, epithelial-only CTCs (CD45-/cytokeratin + , epi.CTCs) and CD45 + /cytokeratin+ immune-like CTCs (im.CTCs). Single cells from both samples were subjected to copy number and protein expression profiling. A cohort of 36 metastatic breast cancer patients was then analyzed to search for additional cases with im.CTCs. Results: Here, we identified and characterized a population of CTCs exhibiting an immune-like state. In two samples from an index patient, im.CTCs outnumbered epi.CTCs, comprising >97% of the CTC population. Single-cell copy number analysis of 43 im.CTCs and 30 epi.CTCs revealed clonal alterations across both populations, confirming a shared tumor origin. Furthermore, im.CTCs contained pseudo-diploid profiles that did not reflect dilution from the addition of a normal diploid genome, indicating that they were unlikely to have originated from tumor-immune cell fusion. Protein expression analysis showed that im.CTCs express CD45 as well as other immune-related markers, such as CD3 and CD4, and the cancer stemness marker, CD44. Subsequent analysis of a metastatic breast cancer cohort identified an additional patient harboring im.CTCs with the same tumor-derived, non-fusion genome as in the index case. Conclusions: Collectively, these genomic and proteomic features distinguish im.CTCs from previously reported circulating cells may represent a novel form of tumor cell plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aef97fcd405edd04a5788c8bfb045e76aa5a9c2" target='_blank'>
              Simultaneous expression of epithelial and immune cell markers in circulating tumor cells identified in patients with stage 4 breast cancer
              </a>
            </td>
          <td>
            Nikki Higa, Audrey Limb, Valerie Hennes, Andrés Rivera, Rafael Nevarez, A. Kolatkar, Carol Kaplan Tweed, Adam I Riker, Young Lee, Lorraine Tafra, Jeremy G Perkins, Craig D Shriver, Peter Kuhn, James B. Hicks
          </td>
          <td>2025-07-24</td>
          <td>Communications Medicine</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) occurs as the third most common cancer with high mortality across the world. Understanding the intratumoral immune cell heterogeneity and their responses to various therapies is crucial for enhancing patient outcomes. This study aimed to characterise and evaluate the immune microenvironment landscapes of CRC shaped by different therapies including CD73 inhibitor, PD‐1 blockade and photothermal therapy (PTT). Our investigation revealed that three therapies could commonly modulate the down‐regulation of Treg, M2 macrophage and Ptprj+ G4 granulocyte, up‐regulation of effector/memory T cell, M1 macorphage and Hilpda+ G1 granulocyte. Moreover, we identified the uniquely dis‐regulated cell types and pathway activities response to each therapy, such as CD73 inhibitor enriched more Cd8+ memory and central memory (CM) cell, PD‐1 blockade with more Cd8+ CTL and Cxcl3+ G2 granulocyte, and PTT with more Cd8+ effector memory and Rethlg+ G3 granulocyte cell. These responses disordered the glycolysis, angiogenesis, phagocytosis functions and cellular communication to reshape the CRC tumour immune microenvironment. We provide the detail insights into the intratumoral immunomodulation preferences of CRC mice treated with CD73 inhibitor, PD‐1 blockade and PTT therapies, which might contribute to the ongoing development of more effective anticancer strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ef48ff72788aaf2bb81620f30c069f2cf46018" target='_blank'>
              Characterising and Evaluating the Immune Microenvironment Landscapes of Colorectal Cancer Shaped by Different Therapies
              </a>
            </td>
          <td>
            Chen Zhou, Yifan Wang, Yuanyuan Li, Weitao Zhang, Yunmeng Bai
          </td>
          <td>2025-07-16</td>
          <td>IET Systems Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background The interactions among cell subgroups in the cervical cancer immune microenvironment play crucial roles in tumor development, but their causal relationships remain unclear. Methods This study employed Mendelian randomization to analyze causal associations between immune cell subgroups and cervical cancer. Multiple statistical methods, including inverse variance weighted, weighted median, and simple mode approaches, were used to evaluate effect sizes. Hierarchical clustering, UMAP, and t-SNE were applied for cell subgroup classification, combined with MIF signaling pathway analysis for cell-cell interaction networks. Results Most immune cell subgroups showed effect estimates close to 1.000 (95%CI: 0.997–1.002) with statistical significance (p < 0.05). Hierarchical clustering analysis revealed eight major cell populations: regulatory T cells, T cells, epithelial cells, natural killer cells, monocytes, ciliated epithelial cells, B cells, and fibroblasts. Cell-cell interaction network analysis demonstrated extensive connectivity among immune cells and between immune and epithelial cells, with particularly strong interactions between monocytes and other immune cells. MIF signaling pathway analysis further confirmed the close relationship between regulatory T cells and T cells. Conclusion This study systematically revealed the causal associations among cell subgroups in the cervical cancer immune microenvironment using Mendelian randomization, providing new insights into understanding tumor immune microenvironment regulation mechanisms and potentially offering theoretical basis for optimizing cervical cancer immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d095e06d9259f527b20902d8cf1fde0afa31bb99" target='_blank'>
              Association and interaction network analysis of immune cell subgroups in cervical cancer microenvironment based on Mendelian randomization
              </a>
            </td>
          <td>
            Yan Cai, Nanxing Jiang, Yan Wang, Min Ji, Wenzhe Shen, Qi-Ming Wang
          </td>
          <td>2025-07-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/034c8133be4ba672c54b97e9a66fb7c40de89b73" target='_blank'>
              Single-cell transcriptomic analysis reveals a metastasis-associated PCSK1+ neuroendocrine subpopulation in pancreatic neuroendocrine tumors.
              </a>
            </td>
          <td>
            X. Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-08-16</td>
          <td>Journal of neuroendocrinology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9095797e23ad8944a80f33c6e2aa42554f1acad9" target='_blank'>
              Uncovering Cellular Interactome Drivers of Immune Checkpoint Inhibitor Response in Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Shay Ladd, Anne M. Talkington, Mary O’Sullivan, R. Barnes, Remziye E. Wessel, Gabriel F. Hanson, Sepideh Dolatshahi
          </td>
          <td>2025-09-08</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Brain metastases are a significant complication of non-small-cell lung cancer (NSCLC), contributing to high morbidity and mortality rates. The introduction of immune checkpoint inhibitors (ICIs) has opened new therapeutic avenues for patients with NSCLC, including those with brain metastases. However, the distinct microenvironment of the brain presents unique challenges to the effectiveness of these treatments. This review examines the mechanisms by which ICIs impact brain metastases from NSCLC, with particular focus on immune cell trafficking across the blood–brain barrier (BBB), tumor microenvironment modulation, and transcriptomic evolution of brain-tropic tumor clones. Unlike prior reviews, we integrate emerging data from single-cell and spatial transcriptomic studies, BBB disruption mechanisms, and the tumor-supportive role of brain-resident glia. We also critically evaluate key clinical trials and real-world evidence, highlighting differences in ICI efficacy across patient subgroups and therapeutic contexts. Additionally, we address the evolving role of surgical resection, stereotactic radiosurgery, and cerebrospinal-fluid-based biomarkers in optimizing ICI-based treatment strategies. This synthesis provides a comprehensive, mechanistic, and clinically relevant framework for improving outcomes in patients with NSCLC brain metastases treated with immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c6ed65bc2f6ca5f07b82a4924c158eebc0c9a9" target='_blank'>
              Immune Checkpoint Inhibition in Patients with Brain Metastases from Non-Small-Cell Lung Cancer: Emerging Mechanisms and Personalized Clinical Strategies
              </a>
            </td>
          <td>
            Nicola J. Nasser, K. Sindhu, Loor Nasser, Z. Shafaee, Joshua Li, Lucas Resende Salgado, Baoqing Li
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb4c318da38745c1257f8f87c4b1e84917526c2d" target='_blank'>
              Spatial-in-Spatial: The Utility of Combining Fluorescence-Guided Multiple Sampling with Spatial-Omics in Human Glioblastoma.
              </a>
            </td>
          <td>
            Shubhang Bhalla, Bethsabe Romero, Yusor Al-Nuaimy, Felix Toussaint, Sina Zoghi, Niel Pacheco-Barrios, Stefan T. Prvulovic, C. A. Bowers, S. Piccirillo
          </td>
          <td>2025-07-25</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is the most prevalent subtype of lung cancer, presenting significant challenges in treatment and prognostic prediction. Despite advancements in therapeutic approaches in recent years, personalized medicine has not yet achieved a notable breakthrough. Given the poor prognosis of patients, there is an urgent need to enhance the ability for precise prediction. Mitochondria play a crucial role in the metabolism and energy production of cancer cells, yet their specific impact in lung adenocarcinoma warrants further investigation. This study leveraged data from the TCGA and GEO databases to stratify 515 lung adenocarcinoma patients into two distinct subtypes based on mitochondrial-related genes. We systematically evaluated survival outcomes and biological pathway activities between subtypes, characterized their immune infiltration profiles, and developed a prognostic model using subtype-specific differentially expressed genes. Drug sensitivity disparities were further assessed. Single-cell RNA sequencing data were analyzed using an XGBoost classifier to delineate cell-type heterogeneity across subtypes at single-cell resolution. In LUAD, we identified two distinct subtypes. One subtype exhibited active mitochondrial metabolism, which was associated with poor prognosis and higher tumor purity. Moreover, this subtype showed greater sensitivity to Osimertinib. Further single-cell analysis revealed that this subtype was characterized by substantial macrophage infiltration, potentially promoting tumor progression through the NF-κB signaling pathway. Overall, our study identified novel LUAD subtypes and provided new insights into the clinical treatment of LUAD. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07982-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08aa1cb8c92b7c61ac76cf9ab60fdfeab09318a6" target='_blank'>
              Mitochondria-related gene-based molecular subtypes of lung adenocarcinoma and their prognostic implications
              </a>
            </td>
          <td>
            Ziyi Zhanghuang, Fei Xie, Xuemei Ma, Jinfeng Chen
          </td>
          <td>2025-07-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) prognosis continues to be challenging due to tumor heterogeneity and dynamic immunosuppressive microenvironments. Although pyroptosis plays a critical role in tumor-immune interactions, its prognostic significance in HCC at single-cell resolution has not been systematically investigated. Methods We analyzed a publicly available single-cell RNA sequencing (scRNA-seq) data from 10 HCC tumors and paired adjacent tissue samples (60,496 cells) to elucidate pyroptosis-related gene (PRG) profiles. Differential expression and functional pathway analyses revealed PRG expression dynamics across cell subtypes. A LASSO-Cox prognostic model was developed using data from the liver hepatocellular carcinoma (LIHC) cohort of The Cancer Genome Atlas (TCGA) (n=365); the model was externally validated with International Cancer Genome Consortium (ICGC) datasets (n=231). Biological validation comprised reverse transcription quantitative polymerase chain reaction (RT-PCR) in HCC cell lines and immunohistochemical analysis of clinical specimens. Results The scRNA-seq atlas identified 10 cellular clusters with enriched expression of 29 PRGs, primarily in natural killer cells, T lymphocytes, monocytes, and macrophages. The prognostic model developed in this study stratified patients into high-risk and low-risk categories based on eight significant genes, achieving area under the curve (AUC) values of 0.73, 0.65, and 0.69 for overall survival at one-year, two-year, and three-year intervals, respectively. Furthermore, external validation using data from the ICGC confirmed the prognostic model’s discriminative ability. Notably, high-risk patients demonstrated enhanced sensitivity to immunotherapy, as indicated by decreased tumor immune dysfunction and exclusion (TIDE) scores and increased expression of the immune checkpoints PD-1 and CTLA4. Conclusions This study established a scRNA-seq-derived prognostic model based on PRGs, which offers insights into HCC immune landscape remodeling. The risk score and nomogram integrate tumor stages and pyroptosis-associated signatures, providing a clinical tool for personalized prognosis and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9a40d89071454b5b94dd84d93e8f8f5e26d978" target='_blank'>
              Prognostic model construction and immune microenvironment analysis of pyroptosis-related genes in hepatocellular carcinoma based on single-cell RNA sequencing
              </a>
            </td>
          <td>
            Haoming Shen, Yizhi Peng, Qingqing Xie, Yuxi Ren, Junping Hu, Peifang Qin, Yuanxiong Chen, Haoyu Zeng, Yifan Sun
          </td>
          <td>2025-08-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Interstitial lung diseases (ILDs) encompass a diverse group of pulmonary disorders, with progressive fibrosis leading to poor prognosis. Here we aimed to identify key molecules involved in progressive fibrosis across various ILDs, using spatial transcriptomics (ST). ST analysis (Visium) was performed on lung cryobiopsy specimens from five patients with various ILDs. Two cases, rich in young fibrotic lesions, as defined by fibroblastic foci and destructive alveolar organization, were selected for spatial high‐dimensional weighted gene coexpression network analysis (hdWGCNA) to identify key gene networks with biological significance in active fibrosis. We utilized public single‐cell RNA sequencing datasets of various ILDs, performed enrichment analysis and trajectory‐based differential expression analysis, and quantified cell–cell communication to evaluate the involvement of the spatially extracted module in fibrosis. Immunohistochemical staining of the extracted molecules was performed. Using hdWGCNA, we identified a distinct gene module (the SM2 module) enriched in young fibrotic lesions. The SM2 module was characterized by distinct features of fibroblast activation that were represented across various lesions. Key hub genes within this module, including COL1A2, COL3A1, COL1A1, and SPARC, formed a robust coexpression network. Immunohistochemical staining showed that SPARC, a component of the SM2 module, was highly expressed in young fibrotic lesions, but not in old scarring lesions, across various ILDs. To assess the prognostic significance of SPARC immunohistochemical expression, we extended our analysis to a cohort of 71 patients with unclassifiable ILDs (uILDs), a particularly heterogeneous subtype with unclear pathogenesis and limited treatment options. Higher SPARC levels in the upper, lower, or both lung lobes in uILD were significantly associated with poor overall survival. In summary, an integrated cross‐disease approach using ST revealed key gene expression patterns central to active fibrosis and successfully identified SPARC as a potentially beneficial prognostic marker. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9a55a6327413c2017bbf7b10baf6a8049b2a4f0" target='_blank'>
              Spatial transcriptomics identifies SPARC as a prognostic marker in interstitial lung diseases
              </a>
            </td>
          <td>
            T. Niitsu, T. Kataoka, K. Fukushima, D. Motooka, Shigeyuki Shichino, Yayoi Natsume-Kitatani, H. Kitamura, Takashi Niwa, T. Baba, Daisuke Okuzaki, Atsushi Kumanogoh, Shizuo Akira, K. Okudela, T. Ogura
          </td>
          <td>2025-07-28</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) techniques are emerging to revolutionize modern biomedical sciences by providing a detailed landscape of individual cells. However, these methods often lack crucial spatial localization information. To address this gap, spatial transcriptomic technologies have developed, enabling gene expression profiling while mapping cells spatial information. Yet, the gene throughput in spatial transcriptomic technologies makes it challenging to characterize whole-transcriptome-level data for single cells in space. In this context, approaches for predicting the spatial distribution of genes are still under development. Here, we present GCNgene, a novel method to predict the spatial distribution of the undetected RNA transcripts, through integrating spatial and scRNA-seq datasets. GCNgene leverages a graph convolutional network to embed spatial transcriptomics data and then applies a learned rule to reconstruct gene expression by combining the reference single-cell data with the calculated cell-type proportions. Ultimately, this learned paradigm enables accurate predictions of gene expression levels. The source code is freely available at: https://github.com/zhangying-njust/GCNgene/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83dd531344de2a14af03e0de1241e2b5631b13d3" target='_blank'>
              Integrating Graph Convolutional Networks for Missing Gene Expression Imputation.
              </a>
            </td>
          <td>
            Ying Zhang, Hong-Jin Yu, Zi-Hao Yan, Tong Pan, Yiwen Zhang, Yan Liu, Shanshan Li, Yuming Guo, Jiangning Song, Dong-Jun Yu
          </td>
          <td>2025-09-02</td>
          <td>IEEE transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Purpose The heterogeneity of immune cells is a critical manifestation of gastric cancer (GC) heterogeneity and significantly contributes to immune therapy resistance. Although previous studies have focused on the roles of specific myeloid cells and exhausted CD8+ T cells in immune resistance, the immune cell interaction network and its spatiotemporal distribution in GC immune resistance remain underexplored. Methods This study integrated multiple GC single-cell RNA sequencing, spatial transcriptomics, bulk-RNA sequencing, and single-cell immunotherapy datasets of our cohort (NFHGC Cohort). Methods such as single-cell subpopulation identification, transcriptomic analysis, spatial colocalization, cell communication network analysis and tissue immunofluorescence of gastric cancer were employed to investigate immune cell interactions and their molecular mechanisms in immune resistance. Results By leveraging a comprehensive approach that integrates single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq profiles, we identified 20 immune subsets with potential prognostic and therapeutic implications. Our findings suggest a stromal immunosuppressive network orchestrated by Macro_SPP1/C1QC macrophages and CD8_Tex_C1 T cells, which may form a barrier impeding antitumor immunity. Macrophage-derived MIF signaling appears to drive immunosuppression via the MIF-CD74/CXCR4/CD44 axis. Based on these observations, we developed a preliminary TME classification system using a gene signature derived from barrier-associated immune cell markers and unsupervised clustering. Conclusions Our study identified a potential stromal immunosuppressive barrier in gastric cancer, driven by Macro_SPP1/C1QC macrophages and CD8_Tex_C1 T cells, which may contribute to immune dysfunction and therapy resistance. Molecular subtyping based on this barrier’s presence could inform personalized immune therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1cf3d75b61cafecd69ad5d518d7d286742cb1e5" target='_blank'>
              Identification of a stromal immunosuppressive barrier orchestrated by SPP1+/C1QC+ macrophages and CD8+ exhausted T cells driving gastric cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Guichuang Ma, Xiaohan Liu, Qinrui Jiang, Shaowei Li, Qijing Wu, B. Liang, F. Sun, Chunhui Gu, Wangjun Liao, Zhi-Huan Zhang, Min Shi, Qiong Huang
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In most solid tumors, hypoxia constitutes a defining microenvironmental feature that reprograms malignant cells into a highly metastatic state by driving cellular plasticity and exacerbating chromosomal instability (CIN). However, the mechanisms by which cancer cells concurrently co-opt these elements of hypoxic adaptation to promote metastasis remains poorly understood. Here, we report that hypoxia promotes metastasis by suppressing the JmjC-containing histone lysine demethylase Kdm8. CRISPR/Cas9-mediated targeting of Kdm8 in a Kras;Trp53-driven mouse model of pancreatic ductal adenocarcinoma (PDA) robustly rewires the malignant cell transcriptomic programs, leading to a profound loss of the epithelial morphology and widespread metastatic disease. In PDA patients, a high KDM8-induced gene signature is associated with reduced metastatic burden and better survival in advanced disease. Notably, Kdm8 suppression in normoxia recapitulates key aspects of the global epigenetic and transcriptomic rewiring, mitotic spindle defects, and CIN induced by hypoxia. Moreover, disruption of Kdm8’s demethylase activity phenocopies Kdm8 loss, whereas expression of hypermorphic Kdm8 variants resistant to hypoxic suppression markedly reduces metastasis beyond the levels achieved by the wildtype protein. Through the suppression of Kdm8 demethylase function, hypoxia unleashes a potent metastatic program by simultaneously advancing cellular plasticity and CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ead3df1c0ef44aa86f0839ae810b8a1fead329" target='_blank'>
              Hypoxia-induced metastatic heterogeneity in pancreatic cancer
              </a>
            </td>
          <td>
            Pradeep Moon Gunasekaran, Qianqian Wang, Yoke-Chen Chang, Polina Guseva, Rajika Chauhan, Alexander Kley, Gene Lee, Siddharth Ghosh Roy, Yousef Masoudpoor, Arthur Roberts, Kelly Watkins Walton, Lucyann Franciosa, Shafiq Bhat, Emmanuel Zachariah, Kishan Patel, Zhongren Zhou, Wenjin Chen, J. Ni, S. Gu, Cristina Montagna, Shin-Heng Chiou
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Lu and colleagues identified profound microbial heterogeneity across individual tumor nodules in multifocal hepatocellular carcinoma, revealing that specific bacterial consortia enrich in metastatic clones and drive epithelial-mesenchymal transition and immune suppression. Their study places the intratumoral microbiome alongside genomic and transcriptomic diversity as a key modulator of intrahepatic tumor evolution. See related article by Lu et al., p. 1630.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3eb39fed6f77266430fee7056b5354580c4710f" target='_blank'>
              A New Dimension in Tumor Heterogeneity: Intratumoral Bacteria Impact Multifocal Hepatocellular Carcinoma Development.
              </a>
            </td>
          <td>
            A. Dzutsev, Giorgio Trinchieri
          </td>
          <td>2025-08-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="ABSTRACT Neuron–glioma interactions are critical drivers of glioma progression, with neuronal activity promoting tumor growth and invasion through paracrine signaling and direct synaptic input. Beyond well‐established glutamatergic synapses, recent discoveries revealed that GABAergic interactions also contribute to glioma proliferation. Here, we focus on how glioma cells decode neuronal cues via epigenetic mechanisms, including enhancer reprogramming, chromatin remodeling, and rewiring of 3D genome organization, with transcriptions factors such as SMAD3 and PITX1 orchestrating transcriptional programs that sustain neuron‐to‐glioma communication. Additionally, recent integration of multi‐omics data highlights gene regulatory networks linked to GABAergic signaling as contributors to glioblastoma (GB) pathogenesis. We also underscore the distinct roles of GABAergic signaling across glioma subtypes, noting that, in GB, GABA‐related metabolic and paracrine mechanisms, rather than synaptic input, may drive tumor progression. Understanding how epigenetic reprogramming facilitates glioma integration into neural circuits opens new avenues to disrupt these malignant neuron–glioma interactions by targeting the epigenetic machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9563d3051732361233db886a9fcc7b7509bf9ab" target='_blank'>
              What Epigenetics Teaches Us About Neuron–Glioma Interactions
              </a>
            </td>
          <td>
            Chaitali Chakraborty, Itzel Nissen, Silvia Remeseiro
          </td>
          <td>2025-07-20</td>
          <td>Bioessays</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Soft tissue sarcomas (STSs) are rare cancers that respond poorly to current treatments. As the immune system is increasingly recognized as a key player in cancer control, we investigate a specific type of immune cell—natural killer (NK) cells. We analyzed NK cells from blood samples of STS patients and healthy individuals. NK cells from STS patients were less functional, showing reduced degranulation and lower production of IFNγ, both essential for killing cancer cells. We also found alterations in key surface markers, which may explain their impaired function. Among these, CD27 and NKp44 were notably different between patients and healthy individuals and showed strong potential to distinguish between these groups. Higher CD27 levels were associated with NK cells that are less effective at killing cancer cells. Overall, our findings suggest that NK cells from STS patients may have a reduced capacity to fight cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14bb47c660c210aea5cd2c30473df6b270829f44" target='_blank'>
              Comprehensive Receptor Repertoire and Functional Analysis of Peripheral NK Cells in Soft Tissue Sarcoma Patients
              </a>
            </td>
          <td>
            L. M. Sousa, J. Almeida, Tânia Fortes-Andrade, Patrícia Couceiro, Joana Rodrigues, Rúben Fonseca, M. Santos-Rosa, Paulo Freitas-Tavares, J. Casanova, Paulo Rodrigues-Santos
          </td>
          <td>2025-07-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Tumor evolution is shaped by selective pressures imposed by physiological factors as the tumor naturally progresses to colonize local and distant tissues, as well as by therapy. However, the distinction between these two types of pressures and their impact on tumor evolution remain elusive, mainly, due to extensive intra-tumor heterogeneity. To disentangle the effects of these selective pressures, we analyze data from diverse cohorts of patients, of both treated and untreated cancers. We find that, despite the wide variation across patients, the selection strength on tumor genomes in individual patients is stable and largely unaffected by tumor progression in the primary settings, with some cancer-specific signatures detectable in the progression to metastases. However, we identify a nearly universal shift toward neutral evolution in tumors that resist treatment and demonstrate that this regime is associated with worse prognosis. We validate these findings on both published and original datasets. We suggest that monitoring the selection regime during cancer treatment can assist clinical decision-making in many cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95ee20056cad563b0d086b8e70065c4daacf4f" target='_blank'>
              Genome-level selection in tumors as a universal marker of resistance to therapy
              </a>
            </td>
          <td>
            Erez Persi, P. Sudalagunta, Yuri I. Wolf, Rafael R. Canevarolo, Mehdi Damaghi, K. Shain, A. Silva, Eugene V. Koonin
          </td>
          <td>2025-07-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The immortal murine hepatic stellate cell line Col-GFP HSC was comprehensively characterized using genetic and molecular approaches. Short tandem repeat (STR) profiling and karyotyping combined with multiplex fluorescence in situ hybridization (M-FISH) confirmed the identity of the cell line and revealed no contamination. Col-GFP HSCs showed a near tetraploid karyotype. Additionally, next-generation sequencing (NGS) data, quantitative reverse transcription PCR, and Western blot analyses demonstrated robust expression of genes and proteins associated with hepatic stellate cells, including those involved in extracellular matrix remodeling and fibrogenic pathways. Phalloidin staining revealed filamentous actin patterns characteristic of stellate cells, providing additional support for their cytoskeletal organization and functional status. These findings provide strong evidence that the Col-GFP HSC cell line originates from hepatic stellate cells and can serve as a reliable in vitro model to study stellate cell biology and related pathophysiological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5a1db386ef6175f68c4642250f680b7fcd1495" target='_blank'>
              Comprehensive Genetic and Molecular Characterization Confirms Hepatic Stellate Cell Origin of the Immortal Col-GFP HSC Line
              </a>
            </td>
          <td>
            Larissa F. Buitkamp, Thomas Liehr, Stefanie Kankel, E. Buhl, Katharina S. Hardt, Diandra T. Keller, Sarah K Schröder-Lange, Ralf Weiskirchen
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is a frequently observed phenomenon in aging individuals without apparent illness and exhibits an increased prevalence in cancer patients. Mechanistic studies indicate that mutant immune cells alter the tumor microenvironment, leading to increased inflammation, blood vessel formation, and immune cell exhaustion. Paradoxically, these changes also preserve stem-like T-cell pools that can be utilized by immunotherapy. CH may be incidentally detected in patients whose solid tumors are profiled by next-generation sequencing. Clinically, CH confers higher risks of therapy-related myeloid neoplasms, cardiovascular and inflammatory toxicities, and context-specific changes in treatment efficacy. Moreover, tumorinfiltrating CH independently shortens survival. Two validated risk scores can inform the risk for myeloid malignancy, yet surveillance, cardiometabolic management, and regimen selection still primarily rely on expert consensus. Because CH may be discovered incidentally, rigorous confirmation of variant origin when CH is suspected is essential to avoid misdirected therapy. We propose a pragmatic approach: confirm CH with paired blood sequencing when feasible; integrate high-risk features into risk stratification, counseling, and monitoring for cytopenias and cardiovascular events; and prefer less genotoxic regimens when the oncologic benefit is comparable. Early trials blocking interleukin-1β suggest that targeting inflammation driven by CH may improve outcomes in patients with solid tumors. Prospective studies informed by mutation analysis and tracking clonal changes and inflammatory markers are needed to determine if routine CH assessment can be integrated into precision oncology to improve outcomes for patients with solid tumors and CH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48a6d6136853e6a95a267ec620d529e97551fcc6" target='_blank'>
              What's hidden in plain sight? Impact of clonal hematopoiesis on the risk and progression of non-hematologic cancers.
              </a>
            </td>
          <td>
            José C Martínez, Catherine C Coombs
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Fusobacterium nucleatum (Fn) is commonly enriched in colorectal cancer (CRC) and associated with poor outcomes, though its mechanisms remain unclear. Our study investigated how Fn affects the tumor microenvironment through single-cell transcriptomic analyses of 42 CRC patient tissues, comparing Fn-positive and Fn-negative tumors. We discovered that Fn impairs IgA plasma cell development and secretory IgA (sIgA) production by disrupting communication with tumor-associated macrophages. Additional experiments in germ-free mice, together with our re-analysis of a publicly available single-cell RNA-seq data set from a CRC mouse model with an intact gut microbiome–both models having been orally gavaged with Fn–jointly validated the causal role of Fn in impairing sIgA induction. We identified a dysregulated IgA maturation (IGAM) module in Fn-positive patients, indicating compromised mucosal immunity and increased bacterial infiltration. This IGAM signature effectively stratified Fn-positive patients, suggesting potential for targeted therapeutic approaches. Our findings reveal that Fn disrupts sIgA production, increasing tumor microbial burden and worsening prognosis through chronic inflammation in Fn-positive CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b983088c09af7c7c8c02b6b79dd03932133b600d" target='_blank'>
              Secretory IgA dysfunction underlies poor prognosis in Fusobacterium-infected colorectal cancer
              </a>
            </td>
          <td>
            Ilseok Choi, Kyung-A Kim, Sang Cheol Kim, Donghwan Park, Ki Taek Nam, Junha Cha, S. Baek, Junha Cha, Hye-Yeong Jo, Minsun Jung, Melody Y. Zeng, Irina Matei, Susan Bullman, Joong Bae Ahn, Yoon Dae Han, Han Sang Kim, Insuk Lee
          </td>
          <td>2025-07-16</td>
          <td>Gut Microbes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background/Objectives: Medulloblastoma is the most common malignant brain tumor in children and comprises four molecular subtypes—WNT, SHH, Group 3, and Group 4—each with distinct genetic, epigenetic, and metabolic features. Increasing evidence highlights the critical role of metabolic reprogramming and epigenetic alterations in driving tumor progression, therapy resistance, and clinical outcomes. This review aims to explore the interplay between metabolic and epigenetic mechanisms in medulloblastoma, with a focus on their functional roles and therapeutic implications. Methods: A comprehensive literature review was conducted using PubMed and relevant databases, focusing on recent studies examining metabolic pathways and epigenetic regulation in medulloblastoma subtypes. Particular attention was given to experimental findings from in vitro and in vivo models, as well as emerging preclinical therapeutic strategies targeting these pathways. Results: Medulloblastoma exhibits metabolic adaptations such as increased glycolysis, lipid biosynthesis, and altered amino acid metabolism. These changes support rapid cell proliferation and interact with the tumor microenvironment. Concurrently, epigenetic mechanisms—including DNA methylation, histone modification, chromatin remodeling, and non-coding RNA regulation—contribute to tumor aggressiveness and treatment resistance. Notably, metabolic intermediates often serve as cofactors for epigenetic enzymes, creating feedback loops that reinforce oncogenic states. Preclinical studies suggest that targeting metabolic vulnerabilities or epigenetic regulators—and particularly their combination—can suppress tumor growth and overcome resistance mechanisms. Conclusions: The metabolic–epigenetic crosstalk in medulloblastoma represents a promising area for therapeutic innovation. Understanding subtype-specific dependencies and integrating biomarkers for patient stratification could facilitate the development of precision medicine approaches that improve outcomes and reduce long-term treatment-related toxicity in pediatric patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733445d34f0ee82899b45eeeaeabd1e9ff5c7ba8" target='_blank'>
              Deciphering Medulloblastoma: Epigenetic and Metabolic Changes Driving Tumorigenesis and Treatment Outcomes
              </a>
            </td>
          <td>
            Jenny Bonifacio-Mundaca, Sandro Casavilca-Zambrano, Christophe Desterke, Íñigo Casafont, Jorge Mata-Garrido
          </td>
          <td>2025-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1887242a48e17cb79a4dc75cbcfd6cfcffc02606" target='_blank'>
              Distinct malignant cell states and myeloid glutamate signaling associated with aggressive pancreatic neuroendocrine tumors.
              </a>
            </td>
          <td>
            Jeanna M Arbesfeld-Qiu, Jae-Won Cho, Phuong T T Nguyen, N. Lester, J. Su, Jimmy A. Guo, H. Hoffman, C. Shiau, Nicholas J Caldwell, Shugo Muratani, Miranda Galvan, Jessica E Proctor, Zack Ely, Steven Wang, M. Ganci, Ruben Dries, Theodore Hong, Jennifer Wo, Genevieve Boland, C. Fernández‐del Castillo, Cristina R Ferrone, Christopher M. Heaphy, M. L. Zhang, Mari Mino-Kenudson, M. Hemberg, William L. Hwang
          </td>
          <td>2025-07-19</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7804c5fd70a0346e6b8d695aa48f935c1a89590" target='_blank'>
              Topological velocity inference from spatial transcriptomic data.
              </a>
            </td>
          <td>
            Yichen Gu, Jialin Liu, Kun H Lee, Chen Li, Lu Lu, Jaimee Moline, Renxiang Guan, Joshua D. Welch
          </td>
          <td>2025-07-16</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb8107340798d9682d8c6977b5d27a13471e822" target='_blank'>
              Integrated single-cell and spatial transcriptomics uncover the prognostic, epigenetic, and immunological roles of FANCC in low-grade glioma.
              </a>
            </td>
          <td>
            Zongye Zhang, Zhi Sha, Han Liu, Yusheng Chen, Zhendong Liu, Xingbo Cheng, Sujie Gu, Yanzheng Gao
          </td>
          <td>2025-09-09</td>
          <td>Neurological research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Mitotic catastrophe (MC) is a well-recognized endogenous mechanism of tumor cell death, characterized as a delayed cell death process associated with aberrant mitosis. However, its prognostic significance in the context of intratumoral heterogeneity in esophageal squamous cell carcinoma (ESCC) remains largely unexplored. Methods We performed an in-depth analysis of single-cell RNA sequencing (scRNA-seq) data from ESCC obtained from the Gene Expression Omnibus (GEO) database. MC scores for individual cells were calculated using the AddModuleScore function, and T cell specific gene modules were identified via the high-dimensional weighted gene co-expression network analysis (hdWGCNA) framework. To further elucidate the developmental trajectories and intercellular interactions of T cells, pseudotime analysis and cell-cell communication inference were conducted. A prognostic risk model was then constructed using three machine learning algorithms combined with multivariate Cox regression analysis. Following risk stratification, we performed immune infiltration profiling, drug sensitivity analysis, and molecular docking to comprehensively assess the functional implications of the risk model in ESCC. Based on preliminary results from quantitative Real-time PCR (qRT-PCR) and Western blotting (WB), we selected the hub gene SLF2 for functional validation using wound healing, Cell Counting Kit-8 (CCK-8) assay, Transwell, and colony formation assays. Results Based on T cell mitotic catastrophe associated genes (MCAGs) and utilizing machine learning algorithms, we established a robust prognostic risk model for ESCC. The model demonstrated excellent stratification capability in predicting patient outcomes and effectively revealed the heterogeneity of the tumor immune microenvironment (TIME) and drug sensitivity. Furthermore, functional experiments confirmed that knockdown of the hub gene SLF2 significantly inhibited the migration, invasion, and proliferation of ESCC cells. Conclusion The prognostic model based on MCAGs we developed serves as an effective tool for predicting outcomes in ESCC.T cell-specific MCAGs drive intratumoral heterogeneity in ESCC, serving as potential prognostic biomarkers and therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00815-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4482ff5484bb99ce98963d648871eb4d8f4fb2bb" target='_blank'>
              Integrative single-cell and bulk transcriptomic analysis reveals the landscape of T cell mitotic catastrophe associated genes in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Shuang Li, Zheng Tao, Nan Wang, Yazhou Liu, Kai Xie, Haitao Ma
          </td>
          <td>2025-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tissue function emerges from coordinated interactions among diverse cell types, but how these interactions are structured and rewired in disease remains unclear. In a recent study, Shi and colleagues introduce CoVarNet, a computational framework that maps reproducible multicellular modules (CMs) across 35 human tissues using single-cell and spatial transcriptomics. These CMs, spanning immune, stromal, and endothelial cells, exhibit functional organization across tissue systems and dynamically respond to biological transitions such as aging and menopause. Importantly, cancer progression is marked by a breakdown of tissue-specific CMs and the emergence of a convergent cancer-associated ecosystem, cCM02. This rewiring reflects a fundamental reorganization of tissue architecture during malignancy and provides new opportunities for diagnostics and therapeutic targeting. The study signifies a conceptual advance from cell-centric to ecosystem-level biology and offers a generalizable framework for integrating multimodal data to dissect tissue-level coordination. Here, we discuss how CoVarNet redefines our understanding of tissue organization, its translational implications in oncology, and unresolved questions in modular tissue biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77233b071e8184bbc674d8e5cc53c013d3b3c380" target='_blank'>
              From Harmony to Discord: Multicellular Coordination in Tissues and Its Rewiring in Cancer.
              </a>
            </td>
          <td>
            Merve Dede, Vakul Mohanty, Ken Chen
          </td>
          <td>2025-07-23</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The tumor ecosystem heterogeneity of papillary thyroid carcinoma (PTC) is poorly characterized in children and young adult patients (CAYA-PTC). In this study, single-cell RNA sequencing is used to profile transcriptomes from the paratumor and tumor tissues of 11 patients. Compared to adult, CD4T_Tfh and CD8T_Tex cells are significantly more prevalent in CAYA-PTC patients. Three phenotypes are identified within the thyrocytes through differentiation trajectory analysis, including normal, BRAF-like, and Fusion-like. Notably, the data reveal that CAYA-PTC patients lack the "mild-state (BRAF-like)" malignant thyrocyte population. This variation in differentiation states indicates that PTC cells in CAYA patients rapidly develop into invasive and metastatic forms, whereas in adult patients, this progression occurs gradually over a longer period. Additionally, extracellular matrix cancer-associated fibroblasts (emCAFs_LAMP5) interact with endothelial cells and thyrocytes, promoting tumor angiogenesis and metastasis more prominently in CAYA patients. Fibroblast activation protein (FAP) expression is high in emCAF_LAMP5 and positively correlated with LAMP5 in CAYA-PTC tissues. Consequently, 68Ga-FAPI-PET emerges as a promising diagnostic method for CAYA patients who are not effectively diagnosed by traditional 18F-FDG-PET. Collectively, the findings provide insight into the CAYA-PTC ecosystem that suggests distinct diagnostic, prognostic, and therapeutic implications compared to adults.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce5c40a8e8a8fbc86d0cfd5509e3eb16f2ff46d" target='_blank'>
              Single-Cell RNA Sequencing Reveals the Heterogeneity in Differentiation Trajectory and Tumor Microenvironment Leading to More Aggressive Phenotypes of Papillary Thyroid Cancer in Children and Young Adult Patients.
              </a>
            </td>
          <td>
            K. Guo, Lingyi Yang, Zirui Huang, Kai Qian, Yuan Shi, Mengjia Fei, Yuan Feng, Zixian Chen, Ming Qiu, Linglin Tang, Anqi Li, Zhibao Lv, Jiangbin Liu, Zhou Chen, Kuiran Dong, Zhuoying Wang
          </td>
          <td>2025-07-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background/Objectives: Myelodysplastic syndrome (MDS) is a heterogeneous clonal hematopoietic disorder characterized by ineffective hematopoiesis and leukemic transformation risk. Current therapies show limited efficacy, with ~50% of patients failing hypomethylating agents. This review aims to synthesize recent discoveries through an integrated network model and examine translation into precision therapeutic approaches. Methods: We reviewed breakthrough discoveries from the past three years, analyzing single-cell multi-omics technologies, epitranscriptomics, stem cell architecture analysis, and precision medicine approaches. We examined cell-type-specific splicing aberrations, distinct stem cell architectures, epitranscriptomic modifications, and microenvironmental alterations in MDS pathogenesis. Results: Four interconnected mechanisms drive MDS: genetic alterations (splicing factor mutations), aberrant stem cell architecture (CMP-pattern vs. GMP-pattern), epitranscriptomic dysregulation involving pseudouridine-modified tRNA-derived fragments, and microenvironmental changes. Splicing aberrations show cell-type specificity, with SF3B1 mutations preferentially affecting erythroid lineages. Stem cell architectures predict therapeutic responses, with CMP-pattern MDS achieving superior venetoclax response rates (>70%) versus GMP-pattern MDS (<30%). Epitranscriptomic alterations provide independent prognostic information, while microenvironmental changes mediate treatment resistance. Conclusions: These advances represent a paradigm shift toward personalized MDS medicine, moving from single-biomarker to comprehensive molecular profiling guiding multi-target strategies. While challenges remain in standardizing molecular profiling and developing clinical decision algorithms, this systems-level understanding provides a foundation for precision oncology implementation and overcoming current therapeutic limitations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e501707362379bd5b49ca68342c5a79da7724f2" target='_blank'>
              A Genetically-Informed Network Model of Myelodysplastic Syndrome: From Splicing Aberrations to Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Sanghyeon Yu, Junghyun Kim, Man S. Kim
          </td>
          <td>2025-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3685ee163d6c268edbf6962e0dbfe3fe91dd59" target='_blank'>
              Tumor microenvironment differences between diagnostic and relapsed classic Hodgkin lymphoma revealed by scRNA-seq.
              </a>
            </td>
          <td>
            Yifan Yin, Shinya Rai, A. Jiang, Alexander M Xu, Luke O'Brien, A. Telenius, J. Delabie, L. Chong, S. Hung, Akil A Merchant, David W Scott, K. Savage, Tomohiro Aoki, Christian Steidl
          </td>
          <td>2025-09-09</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Glioblastoma multiforme (GBM), the most prevalent and lethal type of brain cancer, is characterized by a poor prognosis despite advancements in comprehensive treatments, including surgery, chemotherapy, and radiotherapy. Lactylation, an emerging epigenetic modification, has been shown to influence the biological behavior of tumor cells; however, its role in GBM remains to be further elucidated. Methods In this study, we analyzed the relationship between lactylation-related genes (LRGs) and malignant biological behavior, temozolomide resistance, and the immune microenvironment of GBM using scRNA-seq data from public databases. Subsequently, we identified temozolomide-resistant lactylation-related genes (TMZR-LRGs) through differential gene expression analysis. Based on these genes, we proceeded to classify GBM subtypes and establish a risk prediction model to assess patient prognosis and treatment response. Finally, we validated the impact of lactylation on TMZ resistance and malignant biological behavior of GBM both in vivo and in vitro by knocking out UBE2E1 to increase cellular lactylation levels. Result ScRNA-seq analysis and in vivo and in vitro experiments both demonstrated that lactylation was significantly up-regulated in GBM cells. In the GBM subtype, MES-like cells have the highest lactylation level. Furthermore, an increase in lactylation levels enhanced the malignant proliferation and temozolomide resistance of GBM cells. The risk model based on TMZR-LRGs effectively predicted the prognosis and immune characteristics of GBM patients and had the potential to accurately identify targeted therapeutic drugs for GBM. Conclusion Lactylation is critical for malignant progression, temozolomide resistance and the establishment of an immunosuppressive microenvironment of GBM. The risk model based on lactylation-related genes is an effective tool for assessing the prognosis and treatment response of GBM patients. LRGs have potential as therapeutic targets for GBM, providing a new direction for improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03933-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e001ce4b8474d29cf49bb90b32acbe158c68ab2e" target='_blank'>
              Unlocking the secret of glioblastoma multiforme: the role of lactylation in tumor progression, drug resistance and immune microenvironment
              </a>
            </td>
          <td>
            Yifei Xiao, Ruipeng Zheng, Fengjun Lv, Guang Yang, Haitao Ge, Mingchun Yang, Kan Wang, Yu Cheng
          </td>
          <td>2025-08-07</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5af84640a910b5c7e72c72fa38163e38d00a66c4" target='_blank'>
              Morphological map of under- and overexpression of genes in human cells.
              </a>
            </td>
          <td>
            S. Chandrasekaran, Eric Alix, John Arevalo, Adriana Borowa, Patrick J. Byrne, William G. Charles, Zitong S. Chen, B. Cimini, Boxiong Deng, John G Doench, Jessica D. Ewald, Briana Fritchman, Colin J. Fuller, Jedidiah Gaetz, Amy Goodale, Marzieh Haghighi, Yu Han, Zahra Hanifehlou, Holger Hennig, Desiree Hernandez, Christina B. Jacob, Tim James, Tomasz Jetka, Alexandr A. Kalinin, B. Komalo, Maria Kost-Alimova, Tomasz Krawiec, Brittany A. Marion, Glynn Martin, Nicola Jane McCarthy, Lisa Miller, Arne Monsees, Nikita Moshkov, Al'an F. Munoz, Arnaud Ogier, Magdalena Otrocka, Krzysztof Rataj, D. Root, Francesco Rubbo, Simon Scrace, Douglas W Selinger, Rebecca Senft, Peter Sommer, A. Thibaudeau, Sarah Trisorus, Rahul Valiya Veettil, William J. Van Trump, Sui Wang, Michał Warchoł, Erin Weisbart, A. Weiss, Michael Wiest, Agata Zaremba, Andrei Zinovyev, Shantanu Singh, A. Carpenter
          </td>
          <td>2025-08-01</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Brain metastasis (BM) is one of the common ways of tumor metastasis and has a poor prognosis. This study aims to identify potential biomarkers from the perspective of somatic mutations, providing a basis for the prognosis evaluation and immunogenicity prediction of BM patients. Methods This study collected the somatic mutation profiles and clinical information of a total of 421 patients with BM in Memorial Sloan Kettering Cancer Center (MSKCC). Non-negative matrix factorization was employed to extract the mutational process signatures operating in the genome. Consensus clustering analysis was utilized to identify mutation-related molecular subtypes. Through a comprehensive analysis of genomic mutations and copy number variations (CNV), biomarkers associated with outcomes and tumor immunogenicity were screened. Results Non-small cell lung cancer, melanoma, and breast cancer were common primary tumors of BM, and these three tumor types exhibited better prognosis compared to other types. This study found that a higher tumor mutation burden (TMB) was significantly associated with a better prognosis of BM. A total of four mutational process signatures were extracted, and among them, a signature featured by C > T mutations and related to DNA damage repair was proven to be linked with an inferior outcome and a lower TMB. Through integrated genomic mutation analysis, PTPRT mutation was determined to associate with improved prognosis of BM. More importantly, patients carrying this mutation also harbored a better response to immunotherapy. CNV analysis indicated that PTEN deletion and DUSP4 deletion were respectively associated with poorer and better outcomes in patients with BM. Conclusions By integrating the somatic mutation data of patients with BM, this study identified molecular biomarkers related to outcomes and immunogenicity from three perspectives: mutational process signatures, molecular subtypes, and genomic variations. Our findings provide clues for prognosis evaluation in BM patients. They also establish a theoretical basis for predicting immunotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0ed9fa3e17f4338cc659e3a71e5d9e5dff2308" target='_blank'>
              A mutational process signature and genomic alterations associated with outcome and immunogenicity in cancers with brain metastasis
              </a>
            </td>
          <td>
            Wanli Sun, Xueying Wang, Yixin Xu, Yanfeng Ren, Wenjing Zhang, Qinghua Wang, Yingzhi Chong
          </td>
          <td>2025-07-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background: Anoikis is a form of programmed cell death triggered by the detachment of cells from the extracellular matrix. Anoikis resistance represents a critical factor in tumor metastasis, and elucidating the mechanisms by which epithelial cancer cells evade this process may provide a molecular insight for effectively targeting metastatic progression. Methods: Presented here are an experimental workflow and a detailed protocol to examine anoikis sensitivity in tumor cells both in vitro and in vivo. We described a detachment-induced anoikis model, a three-dimensional spheroid culture system, and an in vivo circulating tumor cell assay, by using the human lung carcinoma cell line A549 as a model system. We detailed the cell culture conditions, materials, and sample preparation, and the evaluation and quantification of anoikis. Together, these methods provide a comprehensive approach for investigating anoikis resistance. Conclusion: This protocol offers valuable insights into the mechanisms underlying anoikis resistance and may facilitate the identification of novel therapeutic targets for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44845c475c780a2f717aad7ae39727b62d7d7bbc" target='_blank'>
              An experimental workflow for investigating anoikis resistance in cancer metastasis
              </a>
            </td>
          <td>
            Xue Han, Yipan Zheng, Xiaohui Si, Zhe Liu
          </td>
          <td>2025-08-05</td>
          <td>Journal of Biological Methods</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff04e163a7aa41f773b08a8fee437665a57ee718" target='_blank'>
              CB2R-induced differentiation epigenetically restrains cancer plasticity enabling adaptive therapy.
              </a>
            </td>
          <td>
            Nuria G. Martínez-Illescas, Gemma Pérez-García, María Rubert-Hernández, Camila Quezada-Gutiérrez, Estrella Martín-Zapater, Luis M Alonso-Colmenar, Carmen Hernández, Ángela Zarco-Cuadrillero, Eduardo Caleiras, Miguel Muñoz-Ruiz, Ana Payo-Payo, Laura Frías-Aldeguer, Cristina Sánchez, Mercedes Balcells-Camps, E R Edelman, María Salazar-Roa
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Comprehensive analysis of single-cell transcriptome and chromatin accessibility will contribute to interpret the heterogeneity of acute myeloid leukemia (AML). We hypothesize that integrating single-cell transcriptomic and chromatin accessibility landscapes underlying t(8;21) AML will provide valuable insights into its heterogeneous cellular properties and gene regulatory programs. Methods Here, we conducted paired single-cell RNA-sequencing (scRNA-seq) and single-cell ATAC sequencing on bone marrow samples from newly diagnosed t(8;21) AML patients and healthy controls. Genetic signatures extracted from scRNA-seq were built and validated across three independent cohorts (German AMLCG1999, GSE106291 and TCGA LAML). Results We identified TCF12, a core component of AML1-ETO-containing transcription factor complex (AETFC), as the most active transcription factor in blast cells, driving a universally repressed chromatin state. Furthermore, we delineated two functionally distinct T cell subsets, revealing that EOMES-mediated transcriptional regulation promotes the expansion of a cytotoxic T cell population (T cells_2; high GNLY, NKG7 and GZMB expression), with an increased clonality and a tendency for drug resistance. In addition, we discovered a novel leukemic CMP-like cluster characterized by high TPSAB1, HPGD and FCER1A expression. Leveraging machine learning-based integration of multi-omic profiles, we identified a robust 9-gene prognostic signature, which demonstrated significant predictive value for AML outcomes across three independent cohorts. Conclusions This multi-omics study provides unprecedented insights into the transcriptional and epigenetic heterogeneity of t(8;21) AML, providing a potential actionable tool for clinical risk stratification. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06659-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb6d672d2579a9875bb647a603a272b4cc733a93" target='_blank'>
              A multi-omic single-cell landscape reveals transcription and epigenetic regulatory features of t(8;21) AML
              </a>
            </td>
          <td>
            Xue-Ping Li, Yan Gao, Bai-Tian Zhao, Yujun Dai, J. Mao, Yang Liang, Lu Jiang
          </td>
          <td>2025-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary Head and neck cancers advance rapidly by establishing dense nerve networks that not only promote tumor growth but also enable immune evasion. Despite their critical role, the specific molecules driving this nerve–immune communication—and representing promising therapeutic targets—remain largely unidentified. In this study, we hypothesized that tumor-secreted neuropeptides act as pivotal mediators within the neuroimmune axis, orchestrating interactions that facilitate cancer progression and treatment resistance. Through comprehensive analysis of existing data, we identified key neuropeptides strongly associated with factors driving nerve density, suppressed immune activity, and resistance to therapy. These molecules represent promising, targetable “messengers” that link neural and immune pathways within the tumor microenvironment. Our findings lay a crucial foundation for future mechanistic research aimed at disrupting nerve–immune crosstalk, opening new avenues for innovative therapies to improve outcomes for patients battling head and neck cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bcff21bff06935172b69d5d4aebbc7f928d775a" target='_blank'>
              Bioinformatics Screening of Tumor-Derived Neuropeptides Mediating Neuroimmune Axis of Head and Neck Cancer
              </a>
            </td>
          <td>
            Ravi Kishan, Gao Zhang, Weifa Yang, Yuxiong Su
          </td>
          <td>2025-07-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background A significant challenge in bladder cancer treatment is primary chemoresistance, in which cancer-associated fibroblasts (CAFs) in the tumor microenvironment (TME) play a pivotal role. While the contributions of CAFs to tumor progression and drug resistance are well established, the precise molecular mechanisms by which they induce chemoresistance remain unclear. A comprehensive understanding of the effect of TME modulation—particularly through CAFs—on the chemotherapeutic response is crucial for developing effective strategies to overcome chemoresistance and improve patient survival. Methods Primary fibroblasts were isolated from paired clinical samples of bladder cancer tissues and adjacent normal tissues to identify key CAF-derived secretory factors. Bioinformatics analysis, semiquantitative RT‒qPCR, and dual-luciferase reporter assays were subsequently used to investigate the functional role and mechanistic basis of CXCL14 in chemoresistance. The therapeutic relevance of these findings was further evaluated through in vitro and in vivo models, including ex vivo patient-derived organoid (PDO) models, by assessing cisplatin sensitivity and validating therapeutic targeting of the CXCL14-CCR7-STAT3 axis with small molecule inhibitors. Results Compared to normal fibroblasts and CAFs from nonchemoresistance groups, CAFs derived from cisplatin-resistant patients demonstrated significantly greater paracrine-mediated induction of chemoresistance. Mechanistically, CAF-secreted CXCL14 engaged CCR7 on bladder cancer cells, triggering STAT3 phosphorylation and consequently upregulating the DNA repair gene ERCC4 to promote cisplatin resistance. In vivo validation confirmed that pharmacological CCR7 or STAT3 inhibition markedly reversed chemoresistance and potentiated cisplatin-induced tumor cell death. Notably, STAT3 activation mediated the overexpression of the glycolytic enzymes HK2 and LDHA, resulting in greater glycolytic flux in resistant cells. This metabolic reprogramming further facilitated the transdifferentiation of normal fibroblasts into CXCL14-secreting CAFs, establishing a self-reinforcing feedback loop that sustains chemoresistance. Conclusion The CXCL14/CCR7/STAT3 axis critically mediates cisplatin resistance in bladder cancer through dual modulation of DNA repair and glycolytic metabolism. Therapeutic cotargeting of this pathway with CCR7 or STAT3 inhibitors combined with cisplatin represents a promising strategy to overcome chemoresistance and improve clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03487-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e39eae34be692eb8887b6d2262ac0b898c0b1af" target='_blank'>
              Cancer-associated fibroblast derived CXCL14 drives cisplatin chemoresistance by enhancing nucleotide excision repair in bladder cancer
              </a>
            </td>
          <td>
            Tinghao Li, Kunyao Zhu, Hang Tong, Yan Sun, Junlong Zhu, Zijia Qin, Junrui Chen, Linfeng Wu, Xiaoyu Zhang, Aimin Wang, Xin Gou, Hubin Yin, Weiyang He
          </td>
          <td>2025-09-02</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Pancreatic cancer stem cells (CSCs) are known culprits of recurrent pancreatic ductal adenocarcinoma (PDAC). The disputable presence of CSCs in continuous cell lines has led to an extensive application of patient-derived CSC culture for experimentation. However, obtaining patient biopsies remains a challenge for many laboratories. Here, this study aimed to investigate the presence of CSCs in KRAS-driven PDAC cell lines.



 Cell viability assays evaluated the cytotoxicity of gemcitabine and vismodegib in PDAC cell lines. Flow cytometry was used to analyse the expression of pancreatic CSC surface markers. Tumoursphere-forming ability was investigated by culturing cells in serum-free and non-adherent conditions in vitro, while the tumourigenicity of PDAC cells was assessed in immunocompromised mice. Molecular events were delineated by Western blotting, and SOX2-expressing cells in tumour xenografts were detected by immunohistochemistry.



 PANC-1 and Capan-2 cell lines showed exceptional chemoresistance, with Capan-2 containing a notable number of putative CSCs that co-expressed CD44, CD24, and CD133 (3.7 % of the total population) and sustained the cytotoxicity of gemcitabine. The Capan-2 cell line harbours tumoursphere-forming cells, of which self-renewal was inhibited by foetal bovine serum. The presence of these cells is consistent with Capan-2 being highly tumourigenic in immunocompromised mice. MAPK, PI3K-AKT, NF-κB, and WNT pathways were activated, and SOX2 expression was upregulated during the self-renewal of putative CSCs. SOX2-expressing cells were correspondingly present in xenografts of highly tumourigenic Capan-2 cell lines.



 The Capan-2 cell line contains a subset of putative CSCs, whose self-renewal is driven by MAPK, PI3K-AKT, NF-κB, and WNT pathways.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/436e454e3e012d3c6079c5e033e5730ae1927ad5" target='_blank'>
              KRAS-driven pancreatic ductal adenocarcinoma cell lines harbour putative cancer stem cells
              </a>
            </td>
          <td>
            Yuan Han Teh, Rui Jing, Rajesh Ramasamy, K. L. Ho, S. Sagineedu, Johnson Stanslas
          </td>
          <td>2025-07-30</td>
          <td>Oncologie</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eef5e91b5687385920939ef78adecfab45682045" target='_blank'>
              Genetically encoded assembly recorder temporally resolves cellular histories in cellulo and in vivo.
              </a>
            </td>
          <td>
            Yuqing Yan, Jiaxi Lu, Zhe Li, Zuohan Zhao, Timothy F. Shay, Shunzhi Wang, Yaping Lei, Yimei Wang, Wei Chen, Patrick Parker, Hongru Yang, Aileen Qi, Yongzhi Sun, Dwight E Bergles, David Baker, Dingchang Lin
          </td>
          <td>2025-07-18</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Intratumoral heterogeneity remains a major barrier to durable cancer therapies, largely driven by the persistence of cancer stem cells (CSCs). In this study, we employed an integrated, multi-scale approach to investigate how melanoma CSCs respond to siRNA-mediated silencing of three key regulatory genes: KLF4, SHH, and HIF1α. Using a combination of morphological, molecular, spectroscopic, and elemental analyses, we explored structural and biochemical consequences of gene knockdown. Gene silencing resulted in significant changes in cell shape and size, reduced F-actin organization, and decreased PFN1 expression, indicating a loss of stem-like properties. ATR-FTIR spectroscopy revealed shifts in biomolecular composition, notably a reduction in amide III intensity and an increase in lipid ester content. SEM-EDS point-based elemental analysis revealed SEM-EDS point-based elemental analysis revealed relative differences in carbon and nitrogen levels between selected central and peripheral regions of silenced and control cells, at the micron-scale working depth, reflecting broader elemental distribution trends rather than precise subcellular compartmentalization. XPS analysis further confirmed these differences, providing additional insights into the elemental composition of the cellular surface. The integration of FTIR spectroscopy into this study highlights the potential of infrared spectroscopy as a powerful tool in cancer research. These findings demonstrate that targeting critical regulatory pathways induces cytoskeletal and biochemical remodelling in melanoma CSCs, offering a multi-dimensional perspective on cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5835a6c3f340e01657d8039c9562c5a0b45cfc" target='_blank'>
              Integrated spectroscopic and morphological analyses reveal cellular shifts in gene-silenced melanoma CSCs
              </a>
            </td>
          <td>
            Berrin Ozdil, Günnur Güler, E. Ataman, H. Aktug
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Malate dehydrogenase 1 (MDH1), an NAD(H)-dependent isoenzyme, is a key component of the malate-aspartate shuttle (MAS). A significant association has been observed between MDH1 expression and various characteristics of the tumor microenvironment across different cancer types. Methods This study provides comprehensive pan-cancer analyses exploring the expression patterns, clinical and pathological correlations, genetic alterations, immunogenomic profiles, single-cell dynamics, alternative splicing signatures, and pharmacological sensitivities related to MDH1. Drug sensitivity profiling and molecular docking techniques have been employed to identify potential anti-cancer compounds targeting MDH1. Experiments have also been conducted to investigate the biological function of MDH1 in lung adenocarcinoma (LUAD) and to confirm the interaction between MDH1 and macrophages using immunofluorescence assays. Results MDH1 expression levels are elevated across a wide range of malignancies, and overexpression of MDH1 was consistently linked to poor prognosis in multiple cancer subtypes. Moreover, MDH1 expression shows complex correlations with various immune cell populations, particularly macrophages, and cohort analysis of both bulk and pan-cancer single-cell immunotherapy data suggest that MDH1 could serve as a predictive marker for immunotherapy responses. Moreover, knockdown of MDH1 suppresses macrophage invasion. To investigate the role of MDH1 in LUAD cells, a potential inhibitor of MDH1 was identified, BI-2536, and has been confirmed to impact MDH1 activity and impede the growth of LUAD cells. Conclusion Our findings indicate that MDH1 may serve as a potential prognostic marker and a promising target for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3176f0dff692eaebf853d9454461d636ffecd8e7" target='_blank'>
              Panoramic view of MDH1: driving cancer progression and shaping the tumor immune microenvironment
              </a>
            </td>
          <td>
            Yunchen Lou, Yunwei Lou, Yao Cheng, Beining Xu, Hanbin Chen, Yinwei Dai
          </td>
          <td>2025-08-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) is a highly vascularized and immunogenic malignancy with a complex tumor microenvironment (TME) that shapes disease progression and therapeutic resistance. Despite advances in immune checkpoint inhibitors (ICIs) and targeted therapies, clinical responses remain heterogeneous, underscoring the need for a deeper understanding of RCC immunobiology. This review comprehensively examines the immunosuppressive TME of RCC, emphasizing the roles of cytotoxic and immunosuppressive immune cells, carcinoma-associated fibroblasts (CAFs), abnormal vasculature, and extracellular matrix (ECM) remodeling in fostering immune evasion. This review summarized emerging biomarkers—including PD-L1 expression, tumor mutational burden (TMB), gene mutations, and immune-based subtypes—that may predict ICI response. Furthermore, we evaluate current immunotherapeutic strategies, such as ICIs, combination therapies, and novel approaches targeting immunosuppressive cells and metabolic pathways. While combination therapies have improved outcomes, challenges like toxicity and resistance persist, necessitating biomarker-driven patient stratification and optimized treatment sequencing. Future directions should focus on deciphering TME heterogeneity and developing precision immunotherapy strategies to enhance clinical efficacy in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c635f92429f0a0ad81f2c837971a966305d3f506" target='_blank'>
              Characteristics of the tumor microenvironment and potential immunotherapy strategies in renal cell carcinoma
              </a>
            </td>
          <td>
            Hui Wen, Shi Zheng, Xiaoqin Zhu, Ling Wang, Dongping Chen
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor stroma can be actively shaped by tumor cells to adopt pro-tumorigenic properties, making it a focal point of cancer research. Among the stromal components, cancer-associated fibroblasts (CAFs) -- often the most abundant cell type in the tumor microenvironment (TME) -- play a pivotal role in promoting cancer progression through their interactions with tumor cells and other stromal components. Elucidating the mechanisms behind these interactions requires robust methods for isolating and characterizing CAFs, which can also lay the foundation for developing novel CAF-targeted cancer therapies. This study presents a method for isolating fibroblasts from both the tumor and the adjacent normal regions of radical prostatectomy specimens obtained from prostate cancer (PCa) patients. This approach enables the concurrent isolation of CAFs and their normal counterparts (NFs) from the same individual, providing a paired experimental system. Detailed protocols are provided for the maintenance of primary CAFs and NFs for downstream in vitro analyses, as well as for their immortalization to facilitate long-term studies. We also describe functional assays designed to compare the effects of CAFs and NFs on cancer cell behaviors, including proliferation, migration, and anchorage-independent growth. Two complementary approaches are detailed: treatment of cancer cells with fibroblast-derived conditioned media (CM), and direct co-culture of fibroblasts with tumor cells. Together, these methodologies represent a comprehensive toolkit for investigating the dynamic interplay between CAFs and tumor cells, not only in PCa but potentially across a range of human cancers. Moreover, molecular characterization of these interactions may reveal key mediators of CAF-driven oncogenesis, offering promising targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2d8dc58b820e48882f34f8df6b8e5bd76919a2" target='_blank'>
              Isolation, Culture, and Characterization of Prostate Cancer-Associated Fibroblasts.
              </a>
            </td>
          <td>
            Somayeh Mirzaaghaei, L. Avalle, Chiara Verrengia, Jahnavi Srivatsa, Laura Conti, Marta Gai, Chiara Fiameni, Paolo Gontero, Umberto Mortara, Luca Molinaro, Mauro Papotti, Valeria Poli
          </td>
          <td>2025-08-01</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="This review provides a comprehensive synthesis of current knowledge on immunotherapy resistance in non-small cell lung cancer (NSCLC), a disease that accounts for approximately 85% of all lung cancer cases and remains the leading cause of cancer-related death worldwide. Although immune checkpoint inhibitors (ICIs) have significantly improved survival for a subset of patients with advanced NSCLC, over 70% of cases ultimately exhibit primary or acquired resistance, underscoring the urgent need to understand the underlying mechanisms. The review categorizes resistance into tumor-intrinsic and tumor-extrinsic processes and provides an in-depth mechanistic analysis of how factors such as tumor antigen loss, impaired antigen presentation, cGAS-STING pathway dysregulation, metabolic reprogramming in tumor microenvironment (TME), immune cell exhaustion, and microbiomes collectively contribute to immune escape. In parallel, the influence of the lung and gut microbiome on shaping immunotherapy responses is discussed, with emphasis on microbial dysbiosis, immunosuppressive metabolite production, and TME remodeling. Therapeutic strategies to overcome resistance are also discussed, including combination approaches involving chemotherapy, radiotherapy, and antiangiogenic agents, as well as epigenetic modulators (HDAC and BET inhibitors). Moreover, the review explores bispecific antibodies, antibody-drug conjugates, and small-molecule agents that enhance T cell function or disrupt immunosuppressive signaling networks. By integrating insights from preclinical models and clinical trials, the review underscores the necessity of biomarker-guided patient stratification, combination immunotherapy approaches, and interventions that restore tumor immunogenicity. It concludes that a multipronged therapeutic strategy, one that addresses both immune evasion and TME-induced suppression, holds the greatest promise for improving response durability and advancing personalized immunotherapy for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df76f6889b4c3ea7f132198d5c6301465fae771" target='_blank'>
              Immunotherapy resistance in non-small cell lung cancer: from mechanisms to therapeutic opportunities
              </a>
            </td>
          <td>
            Huiyu Wang, Xiaomin Niu, Zhenning Jin, Shaoxing Zhang, Rong Fan, Hua Xiao, Shen S. Hu
          </td>
          <td>2025-08-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary Medulloblastoma is a type of brain tumor that mainly affects children and often has serious outcomes. In this study, we examined how certain proteins—neuropilin-1 and glial fibrillary acidic protein (GFAP)—are distributed in tumor tissue and how they relate to blood vessels, immune cells, and tumor behavior. By studying different types of medulloblastoma, we found that these proteins show specific patterns depending on the tumor subtype. Our results suggest that abnormal blood vessel formation and the presence of certain cell markers may be linked to tumor growth and recurrence. These findings help us better understand how the tumor develops and may eventually lead to new ways to predict disease progression or create targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f7433e73c47939f837be266f6f9479fc5798fee" target='_blank'>
              NRP1 and GFAP Expression in the Medulloblastoma Microenvironment: Implications for Angiogenesis and Tumor Progression
              </a>
            </td>
          <td>
            Margarita Belem Santana-Bejarano, María Paulina Reyes-Mata, J. J. Guerrero-García, Daniel Ortuño-Sahagún, M. Godínez-Rubí
          </td>
          <td>2025-07-22</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Neuroblastoma (NB), the most prevalent extracranial solid malignancy in children, poses significant therapeutic challenges, particularly concerning high-risk subtypes characterized by an immunologically “cold” phenotype. These tumors evade immune surveillance through mechanisms such as impaired antigen presentation and immunosuppressive microenvironment formation. Despite the incorporation of immunotherapies (e.g., GD2 monoclonal antibodies) into international clinical guidelines, the 5-year survival rate of patients with NB persistently remains lower than 50%. Small-molecule targeted agents, distinguished by their low molecular weight and superior chemical stability, offer advantages over macromolecular antibody therapies by effectively penetrating cell membranes to engage intracellular targets. Epigenetic regulation, a DNA sequence-independent gene expression modulation system, plays a pivotal role in cell fate determination via dynamic DNA methylation, histone covalent modifications, chromatin spatial reorganization, and non-coding RNA-mediated pathways. Emerging evidence has highlighted a strong correlation between epigenetic dysregulation and NB progression, establishing a molecular rationale for novel therapeutic strategies. Current epigenetic research in NB primarily focuses on histone deacetylase inhibitors and DNA methyltransferase inhibitors. These drugs exhibit unique translational potential because of their accelerated development timelines and cost-effective production, significantly enhancing therapeutic accessibility. This review systematically examines the current landscape of epigenetic modulators in NB treatment and discusses their transformative potential in improving outcomes for high-risk patients with NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2ed0206bc0a5ccb2cde86f23fd7b8f058a1635a" target='_blank'>
              Emerging frontiers in epigenetic-targeted therapeutics for pediatric neuroblastoma
              </a>
            </td>
          <td>
            Xiaokang Wang, Gengrui Xu, Hongyan Ma, Xiaoyan Deng, Guiping Ma
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA), a major focal oncogene amplification mode found across cancer, has recently regained attention as an emerging cancer hallmark, with a pervasive presence across cancers. With technical advancements such as high-coverage sequencing and live-cell genome imaging, we can now investigate the behaviors and functions of ecDNA. However, we still lack an understanding of how to eliminate ecDNA. We observed depletion of cells containing ecDNA during lentiviral but not transposon-based transduction, whereas we sought to investigate the mechanism of ecDNA behavior. This discovery may provide critical information on utilizing a lentiviral system in emerging ecDNA research. Additionally, this observation suggests specific sensitivities for cells with ecDNA. Significance: ecDNA is an essential factor in cancer progression. We found that a group of cancer cells with ecDNA is selectively depleted after lentiviral infection. This finding provides promise for ecDNA-specific targeting, suggests the need for caution in using lentivirus, and offers alternative ways to study ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acb47b6253f4e8e1db726d8195373534a90d452c" target='_blank'>
              Selective Depletion of Cancer Cells with Extrachromosomal DNA via Lentiviral Infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, Emmanuel E Korsah, Noah A Dusseau, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Background Human endogenous retroviruses (HERVs) are remnants of ancient viral infections and comprise 6–8% of the human genome. Their biological functions in cancer remain poorly understood, especially in glioblastoma, the most common and deadly primary brain cancer in adults. Prior studies on HERV expression in glioblastoma have yielded conflicting results. Here, we employed orthogonal computational pipelines to address these limitations. Results Locus-specific analysis revealed marked heterogeneity of transcription among HERVs within the same clade. We found that individual HERV proviruses are more than twice as likely to be underexpressed in glioblastoma than overexpressed and that most differentially expressed HERVs are exonized within transcripts. These HERVs are enriched in the 3’-terminal exons of transcripts, associated with alternative polyadenylation and contribute conserved polyadenylation signals. We identified HERV expression patterns associated with glioblastoma subtypes and anatomic features. We also identified three proviruses or proviral fragments of particular interest including one associated with survival and one exonized within a currently unannotated cancer-specific transcript. Among the most recently integrated clade of HERVs, excluding solo-LTRs, only the HML-2 provirus at 1q22 is overexpressed in glioblastoma. We report previously undescribed transcripts incorporating this provirus that may encode several proteins. Conclusions This study represents the first systematic exploration of the heterogeneity of HERV expression between anatomic features of any cancer. It shows that exaptation of HERV polyadenylation signals and HERV-associated APA are widespread in the human transcriptome and identifies critical structural information regarding the HML-2 1q22 provirus transcript, which has been the focus of several recent analyses. Our findings underscore the importance of locus-specific and anatomic feature-specific HERV analysis and suggest structural and functional roles for HERVs in glioblastoma-associated transcripts. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00365-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae712fda8b06db59cbe0784641111444d4db678e" target='_blank'>
              In-depth analysis of endogenous retrovirus expression in glioblastoma
              </a>
            </td>
          <td>
            Daniel A. Murimi-Worstell, Irene B. Murimi-Worstell, Farrah M. Roy, A. Burn, Michael Freeman, Ralph B. Puchalski, John M. Coffin
          </td>
          <td>2025-07-16</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Glioblastoma multiforme (GBM) represents one of the most aggressive and lethal primary brain malignancies, characterized by rapid proliferation, extensive invasiveness, and a dismal prognosis. Emerging evidence implicates nucleotide-binding oligomerization domain-containing protein 2 (NOD2), an intracellular pattern recognition receptor, as a potential driver of GBM progression. This study investigates NOD2’s role in promoting glioblastoma through its effects on the epithelial–mesenchymal transition (EMT) and cancer stem cell (CSC) markers. Methods: NOD2 expression levels and survival outcomes were assessed using TCGA data from GBM tumor samples (n = 153) and normal brain tissues (n = 5). NOD2 protein expression was validated in glioma cell lines using Western blot and immunofluorescence analyses. Functional studies employed siRNA-mediated NOD2 knockdown to evaluate effects on cellular proliferation, migration, invasion, and colony formation, while correlations between NOD2 and EMT/CSC markers were assessed. Results: The analysis of TCGA data revealed a significantly elevated NOD2 expression in GBM tumors compared to normal brain tissue, with a high NOD2 expression correlating with a reduced disease-free survival in GBM patients. All tested glioma cell lines demonstrated robust NOD2 expression. Functional analyses demonstrated that NOD2 depletion substantially impaired cellular proliferation, migration, invasion, and the colony-forming capacity. Mechanistically, siRNA-mediated NOD2 knockdown significantly decreased the expression of EMT (Snail, SLUG, Vimentin) and CSC markers (CD44, CD133) at both protein and mRNA levels. Conclusions: Our results indicate that NOD2 contributes to GBM progression by influencing EMT and CSC pathways. These findings suggest NOD2’s potential as a therapeutic target in glioblastoma, highlighting the need for further mechanistic studies and therapeutic exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63a96821ffcc57176d3a659b94af662202504257" target='_blank'>
              NOD2 Promotes Glioblastoma Progression Through Effects on Epithelial–Mesenchymal Transition and Cancer Stemness
              </a>
            </td>
          <td>
            Eshrat Jahan, Shubhash Chandra Chaudhary, Abdus Salam, Eun-Jung Ahn, Nah Ihm Kim, Tae-Young Jung, Jong-Hwan Park, Sung Sun Kim, Ji Young Lee, Kyung-Hwa Lee, Kyung-Sub Moon
          </td>
          <td>2025-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background To explore the differences in tumor immune microenvironment between different KRAS mutation subtypes, and to provide a new direction for clinical immunotherapy of different subtypes of colorectal cancer. Methods We examined the spatial distribution of common inflammatory cell markers of CD8 + T cells, CD4 + T cells, natural killer cells, and B cells and macrophages in 55 colorectal cancer patients with different KRAS gene phenotypes using immunohistochemistry and panoramic scanning. We analyzed the relationship between inflammatory cells and clinical information. TCGA and String online databases were used to analyze the relationship between KRAS mutation subtypes and inflammatory cell markers. Results We observed that significant differences in the spatial distribution of the tumor invasion front and the immune cell infiltration within the tumor. Colorectal cancer patients with different KRAS mutations showed different immune infiltration, and the main cells with differences were FOXP3 regulatory T cells and M2 macrophages. Furthermore, these two cell types were strongly associated with the prognosis of KRAS wild-type and KRAS mutant colorectal cancer. Conclusions The complexity of the immune microenvironment cannot be explained by infiltration of specific cell types alone, but may arise from cell-to-cell interactions or changes in the proportion of different immune cells. However, the infiltration of FOXP3 cells and M2 macrophages probably accounts for the differences between KRAS mutant subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03be9a5c418e895db04c9d1a928c80c7d3a288ec" target='_blank'>
              Analysis of the immune microenvironment in colorectal cancer with different KRAS gene subtypes
              </a>
            </td>
          <td>
            Jing Huang, Qian Gong, Qingshu Li, M. Xiao, Ming Li, Shuxian Zhang, Yalan Wang, Yi Tang
          </td>
          <td>2025-08-04</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Despite advances in understanding the morphological disruptions that lead to defects in palate formation, the precise perturbations within the signaling microenvironment of palatal clefts remain poorly understood. To explore in greater depth the genomic basis of palatal clefts, we designed and implemented the first single cell spatial RNA-sequencing study in a cleft palate model, utilizing the Pax9−/− murine model at multiple developmental timepoints, which exhibits a consistent cleft palate defect. Visium HD, an emerging platform for true single-cell resolution spatially resolved transcriptomics, was employed using custom bins of 2 × 2 μm spatial gene expression data. Validation of spatial gene expression was then validated using custom designed Xenium In Situ mRNA spatial profiling and RNAscope Multiplex assays. Functional enrichment analysis revealed a palate cell-specific perturbation in Wnt signaling effector function in tandem with disrupted expression of extracellular matrix genes in developing mesenchyme. As a key step toward laying the framework for identifying key molecular targets these data can be used for translational studies aimed at developing effective therapies for human palatal clefts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-14807-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a33f260de07738fe1e1b7eb3fbbd247023eb604" target='_blank'>
              Single cell spatial transcriptomics links Wnt signaling disruption to extracellular matrix development in a cleft palate model
              </a>
            </td>
          <td>
            J. O. Piña, R. Raju, Evan Stipano, A. Myo, Ziyi Wang, M. Ono, P. Chattaraj, M. Furukawa, Rena N. D’Souza
          </td>
          <td>2025-08-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Wilms tumor (WT), the most common pediatric renal malignancy, exhibits a relatively low mutational burden compared to adult cancers, which hinders the development of targeted therapies. To elucidate the molecular landscape of WT, we perform integrative proteomic, phosphoproteomic, transcriptomic, and whole-exome sequencing analyses of WT and normal kidney tissue adjacent to tumor. Our multi-omics approach uncovers prognostic genetic alterations, distinct molecular subgroups, immune microenvironment features, and potential biomarkers and therapeutic targets. Proteome- and transcriptome-based stratification identifies three molecular subgroups with unique signatures, correlating with different histopathological subtypes and putative cellular origins at different stages of embryonic kidney development. Notably, we identify EHMT2 as a promising prognostic biomarker and therapeutic target associated with epigenetic regulation and Wnt/β-catenin pathway. In this work, we provide a comprehensive molecular characterization of WT, offering valuable insights into its pathogenesis and a foundational resource for future therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3be681b81ddbc1d74bdc72b4011124ef4e90944" target='_blank'>
              Integrative proteogenomic characterization of Wilms tumor
              </a>
            </td>
          <td>
            Cheng Cheng, Li Zhang, Xiaofeng Chang, Kai Chen, Tian He, Jia Shi, Fan Lv, Lijia Pan, Yangkun Wu, Qianqian Cheng, Dong Ren, Y. Guo, Weiping Zhang, Huanmin Wang, Tieliu Shi, Jing Li, Xinxin Ni, Yeming Wu, Yaqiong Jin, Zhixiang Wu
          </td>
          <td>2025-08-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) was a major cause of cancer-related mortality globally. Despite advancements in immunotherapy and targeted therapies, clinical outcomes were still limited by tumor heterogeneity and treatment resistance. The transcription factor (TF) FOS, a key component of the AP-1 complex, was linked to tumor progression and therapy resistance in various cancers, but its precise mechanisms remained unclear, and its role in lung adenocarcinoma (LUAD) was unknown. We investigated the tumor microenvironment (TME) of LUAD using single-cell RNA sequencing (scRNA-seq) to identify potential therapeutic vulnerabilities and FOS-driven mechanisms. Methods We identified fourteen cell types by analyzing scRNA-seq data from LUAD samples (GSE164789) using Seurat (v4.4.0) and Harmony for batch correction. InferCNV was utilized to characterize the tumor cell subtypes after they were clustered using marker genes. CytoTRACE and Monocle were used to create pseudotime trajectories in order to map differentiation states. CellChat revealed intercellular communication networks, while SCENIC identified TF regulatory modules. The CCK-8, Edu, Transwell, and wound healing assays showed that FOS knockdown functionally validated A549 and NCI-H1975 cells. Furthermore, a prognostic model was developed. Results We discovered that invasive LUAD was dominated by a highly stem-like C0 MAFF+ tumor cell subtype that produced chemokines and activated lipid metabolism. These cells stimulated immunosuppression and tumor-associated macrophage (TAM) differentiation by interacting with macrophages via MIF-(CD74+CD44) signaling. Experiments using FOS knockdown demonstrated its role in maintaining invasion, migration, and proliferation. Using the MTRS model, patients were categorized into high- and low-risk cohorts, high-risk patients exhibited unique drug sensitivities. Immunoprofile analysis revealed higher M1 macrophages in high-risk patients, suggesting that FOS inhibition could repolarize TAMs and enhance immunotherapies. Conclusion Our studies show that FOS is a main regulator of C0 MAFF+ TCs in LUAD, polarizing macrophages via MIF and rewiring lipid metabolism to support cancer. The MTRS model offers clinical value for risk assessment even if FOS inhibition shows promise as a therapeutic approach to raise immunotherapy efficacy. Targeting the FOS could cause TME immunosuppression to be disrupted, thus LUAD presents a fresh precision oncology approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2ef348cf0a48b9b9c178cc598309bdb56456d8" target='_blank'>
              Targeted and personalized immunotherapy in lung adenocarcinoma: single-cell RNA sequencing of MAFF+ tumor cells and the therapeutic potential of FOS
              </a>
            </td>
          <td>
            Xiangsong Cheng, Shu Chen, Yilong Fu, Runze Jiang, Yanlong Jing, Bizhu Zhao, Dong Guo, Liangyu Wang, Zi Ye, Yumeng Li, Xianliang Chen
          </td>
          <td>2025-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Serous tubal intraepithelial carcinomas (lesions) in the human fallopian tube epithelium (hFTE) are theorized to give rise to high-grade serous ovarian cancers. Small extracellular vesicles (sEV) are known to mediate key signaling in both normal and cancerous tissues, but few ex vivo systems exist for studying the impact of sEV on hFTE tissue. In this study, we present a microfluidic tissue culture platform with combined spatial transcriptomic and proteomic readouts that allows us to profile dual responses in tissue exposed to sEV “messages”—capturing both short-term transcriptomic shifts in the tissue and long-term changes in protein cargo of secreted EVs (the “reply”). Using spatial transcriptomics, we show that the short-term 1-day exposure to ovarian cancer–derived sEVs alters expression of 68 transcripts in secretory cells, the progenitor of high-grade serous ovarian cancer, notably upregulating immune-related mRNA, including CXCL family chemokines, VCAM1, and pro-inflammatory mediators (NFKB1, IL1B, and IFNA7/17). Additionally, we observed that the long-term 14-day exposure to sEVs alters the expression of seven transcripts and 25 EV cargo proteins of fallopian tube–derived EVs (“secondary release EVs”) following stimulus from cancer EVs. Together, tissue transcriptomics and tissue-derived EV proteomics indicate that ovarian cancer–derived sEVs rewire target cell signaling to modify the tubal immune landscape. This study provides insights into the early molecular changes associated with the pathogenesis of ovarian cancer in its tissue of origin, providing a platform to study EV–tissue interactions and identify how sEVs drive cell signaling reprogramming in hFTE. Significance: We model the fallopian tube preneoplastic landscape using a microfluidic platform to study EV-induced stress and show that cancer EVs promote immune signaling changes representing the earliest stages of ovarian cancer pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7b9b7316e20845062da859010e1284d4df77437" target='_blank'>
              Defining the Ovarian Cancer Precancerous Landscape through Modeling Fallopian Tube Epithelium Reprogramming Driven by Extracellular Vesicles
              </a>
            </td>
          <td>
            Jared Sipes, Didi Zha, Sagar Rayamajhi, Leonidas E Bantis, Rashna Madan, Amrita Mitra, Rajni V Puri, M. Rahman, Foyez Ahmmed, H. Pathak, Angela Russo, Mihaela Sardiu, Brett C Isenberg, B. Cain, J. Coppeta, Pamoda M Galhenage, S. Pathania, Shannon MacLaughlan David, Joanna E. Burdette, Andrew K. Godwin
          </td>
          <td>2025-07-21</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
This review highlights recent advances in single cell "-omics" technologies that have transformed our understanding of the HIV reservoir in the central nervous system (CNS) and cerebrospinal fluid (CSF).


RECENT FINDINGS
Recent studies have applied single cell and single nucleus RNA-seq, ATAC-seq, CITE-seq, AIRR-seq, multiomic platforms, and spatial transcriptomics to postmortem brain tissues and CSF. These analyses have revealed that HIV persists in rare subsets of CNS-resident microglia and trafficking CD4+ T cells despite ART. Infected microglia often display inflammatory transcriptional states, while clonal T cell populations harboring HIV can migrate between blood and CSF. Spatial and multimodal approaches are uncovering both the tissue localization and epigenetic regulation of infected cells, offering unprecedented insight into reservoir biology and neuropathogenesis.


SUMMARY
Single cell studies have established the CNS as a transcriptionally active and clonally maintained reservoir of HIV during ART. These findings underscore the need for cure strategies that penetrate the brain, target both lymphoid and myeloid reservoirs, and consider the transcriptional, epigenetic and spatial context of HIV-infected cells. Ongoing technological advances will further illuminate the dynamics of the CNS reservoir and guide the design of diagnostic, prognostic and therapeutic biomarkers and CNS-penetrant therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db22db47d2559ef04a4bd2fef782115f04291fd8" target='_blank'>
              Single cell analyses of the HIV reservoir in the CNS and CSF: recent insights and implications.
              </a>
            </td>
          <td>
            P. Filippidis, Michael J Corley
          </td>
          <td>2025-07-16</td>
          <td>Current opinion in HIV and AIDS</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Immune checkpoint inhibitors (ICIs) have become an integral part of cancer therapy, but only a minority of patients experience durable responsiveness. Response rates vary greatly and are often unpredictable, highlighting the urgent need for predictive biomarkers to guide treatment decisions. Methods We investigated immune- and tumor-specific expression and secretion profiles in peripheral blood and tumor samples derived from patients with head and neck squamous cell carcinoma (HNSCC). We combined flow cytometry, LEGENDplex™ immune profiling, and preoperative/postoperative serum cytokine analyses to determine checkpoint molecules (e.g., PD-1, TIM-3, LAG-3), immune cell profiles, as well as key markers on tumor cells (CD44, PD-L1, MHC class I/II). In addition, a 3D co-culture model using tumor slices and autologous mononuclear cells from selected HNSCC patients were analyzed upon atezolizumab and pembrolizumab treatment. Results Co-expression of PD-1 and TIM-3 on a subset of CD8+ tumor-infiltrating T cells was frequently observed, alongside a pronounced infiltration of myeloid cells in the tumor microenvironment. In the peripheral blood, we detected elevated levels of soluble CD27 in patients compared to controls and distinct preoperative cytokine profiles (e.g., reduced IFN-γ, CCL3, CCL20; elevated IL-15/IL-16). Postoperatively, most cytokines showed lower levels compared to healthy controls but significantly higher CCL2 levels. Furthermore, tumor–immune co-cultures from selected patients showed a stronger apoptotic response and phenotypic differences (e.g., increased PD-1 and CD137 expression) upon atezolizumab treatment. Individual changes in soluble factor release (e.g., Gal-9, sPD-L1, sCD25, and sTIM-3) was noticeable upon co-culture under immune checkpoint therapy. Conclusions This study provides proof-of-principle data suggesting that a combined multiplexed marker profiling and a functional 3D co-culture assay may help to explore predictive ICI response for HNSCC patients in the future. However, extensive studies with larger cohorts are warranted to validate and refine this approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44e906eb68c4d9ae7390358edb5e291e5b366aa8" target='_blank'>
              Multiplexed immune profiling and 3D co-culture assays to assess the individual checkpoint therapy response in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Verena Schweihofer, D. Schulz, Raquel Blazquez, G. Brockhoff, T. Ettl, Mathias Fiedler, Sina Heimer, Juliane Schikora, Richard J Bauer, A. Wege
          </td>
          <td>2025-08-08</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are malignant cells that detach from primary or metastatic tumors and enter the bloodstream. Organoids, as three‐dimensional in vitro models, can mimic the tumor microenvironment and histopathological characteristics, thereby serving as valuable tools in tumor research. CTC‐derived organoids retain tumor heterogeneity and metastatic potential, which provides a unique model for the study of metastatic cascade mechanisms, individualized drug screening, and precision therapy. However, the current research on CTC‐derived organoids faces challenges, such as the scarcity of CTCs and the high technical difficulty in their isolation and enrichment, which leads to a low success rate in constructing organoid models. Moreover, most existing studies focus on a single cancer type and lack systematic integration of full‐process standardization as well as cross‐cancer applicability. In this paper, we review the isolation and enrichment strategies of CTC‐derived organoids along with the techniques for optimizing in vitro culture systems, and discusses their potential applications. This review summarizes the existing results, analyzes the technical bottlenecks, and provides a theoretical basis for the standardized construction and application of CTC‐derived organoids, while promoting their application in tumor precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/971e10876cd9eda4f902688d2d69a778549269d1" target='_blank'>
              Circulating Tumor Cell‐Derived Organoids: Current Progress, Applications, and Future
              </a>
            </td>
          <td>
            Tiantian Li, Biao Deng, Shenglong Li, Yanxia Wu, Zhenghao Lu, Zhu Liang
          </td>
          <td>2025-09-01</td>
          <td>MedComm – Future Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background The unique stress response state of T cells (Tstr) is found in various cancers and correlates with altered Lung Adenocarcinoma (LUAD) tumor microenvironment and immunotherapy outcomes. However, fewer studies have been described on the relationship between Tstr and LUAD, and its mechanism of action in the pathogenesis of LUAD needs to be further investigated. Methods The categorization and validation of cluster types for Tstr signature genes were systematically carried out using transcriptomic data sourced from the TCGA and GEO databases. The focus was on elucidating distinct pharmacological characteristics, tumor microenvironmental traits, and immunotherapeutic benefits across various subtypes. Moreover, a predictive model was developed through Least Absolute Shrinkage and Selection Operator (LASSO) regression analysis, and subsequent cellular experiments confirmed the pivotal role of CREG2 in driving LUAD cell proliferation and migration. Results Tstr signature genes represented by the heat shock protein family are highly expressed at the end of T-cell differentiation and affect cellular communication. Patients with the C1 subtype have a poorer prognosis and high expression of HSPA1A and HSPA1B, which exhibit cold tumor characteristics and define them as important Tstr subtypes. CREG2 is positioned as a promising biological indicator as it plays an important part in enhancing the migratory and proliferative capacity of LUAD cells. Discussion This study reveals the characteristics of different Tstr types. A predictive model was built to forecast the prognosis of LUAD patients based on the DEGs of the subtype. The exploration delved into the bio-function of CREG2 in the progression of LUAD. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03170-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77dda53b77f8e92a63f02ef3e1d805a06740b60e" target='_blank'>
              Integration of single cell and bulk transcriptomes identifies T cell stress subtypes in LUAD
              </a>
            </td>
          <td>
            Shengping Min, Linfeng Pan, Xinyu Zhang, Huili Chen, Lixuan Qiu, Xinyu Wang, Yiluo Xie, Kai Zhang, Qiang Zhang, Chaoqun Lian, Jing Zhang
          </td>
          <td>2025-07-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, , Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Thyroid cancer (TC) is now among the fastest-growing solid tumours, yet therapeutic gains remain limited for poorly differentiated, anaplastic and medullary variants whose median survivals are measured in months. Once guided chiefly by histology and single-gene assays, immunotherapy is being reshaped by single-cell profiling, which exposes the cellular mosaics that arbitrate response and resistance. Dissection of more than 150–000 tumour- and immune-cell transcriptomes has uncovered follicular-like, partial EMT-like and dedifferentiated thyrocyte states embedded within ‘hot’ (CD8hi IFN-γhi), ‘cold’ (CD8lo) and ‘excluded’ (stroma-walled) immune niches; these phenotypes correlate with PD-1/LAG-3 expression, macrophage polarisation and radio-iodine refractoriness. Functional studies reveal that SPP1–CD44 and GAS6–AXL crosstalk licenses epithelial–mesenchymal transition while VSIG4+ macrophages blunt cytotoxic T-cell activity, collectively undermining checkpoint blockade. Spatial transcriptomics corroborates these insights, mapping PD-L1-high tumour islets millimetres from CXCL13-rich tertiary lymphoid structures, whereas CITE-seq quantifies actionable checkpoints and cytokine receptors across patient biopsies. Emerging therapeutics mirror this granular knowledge: combinatorial PD-1 + LAG-3 inhibition, CSF-1R-directed macrophage re-programming and TSH-receptor-targeted CAR-T cells are advancing through early-phase trials, while ex-vivo single-cell pharmacotyping aligns drug cocktails with an individual’s tumour ecosystem. Early lenvatinib-pembrolizumab or selpercatinib-nivolumab trials show ~40% ORR but grade-3 hypertension >60%, prompting staggered-start designs. These advances sharpen pathogenetic resolution, refine patient selection and accelerate translational pipeline design. By integrating single-cell biology, immunology and endocrine oncology, this review identifies diagnostic blind spots, spotlights drug-repurposing opportunities and charts a roadmap toward personalised immunotherapeutic strategies capable of improving outcomes across the diverse spectrum of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6d6beff9a786418bc13757a09189c54eb993ef" target='_blank'>
              Personalised immunotherapy strategies informed by single cell profiling in thyroid cancer: a mini review
              </a>
            </td>
          <td>
            Ruyu Chen, Zhimin Wang, Xueying Chen, Yangling Huang, Fengnuan Zhang, Tianshu Gao
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d43ce8f81cf0d90a92d73349eebd57717f9bf249" target='_blank'>
              Decoding adenomyosis pathogenesis using an assembloid model.
              </a>
            </td>
          <td>
            Yiliang Xu, Tao Cheng, Jianzhang Wang, Zhiruo Qiu, Xiaohong Guan, Yayuan Yu, Jiacheng Shen, Fangfang Xu, Xiaohong Jiang, Dandan Bai, Mingzhu Wang, Shuyan Mei, Hong Wang, Xiaocui Xu, Li Wu, Shaorong Gao, Xuan Che
          </td>
          <td>2025-08-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Macrophage polarization plays a pivotal role in shaping the tumor microenvironment and influencing rectal cancer progression. However, the metabolic and prognostic regulators governing this process remain largely undefined. Methods We constructed a macrophage polarization gene signature (MPGS) by integrating weighted gene co-expression network analysis (WGCNA) with multiple machine learning algorithms across two independent cohorts: 363 rectal cancer samples from GSE87211 and 177 samples from The Cancer Genome Atlas (TCGA). The prognostic performance of MPGS was evaluated across rectal and multiple other cancer types. Functional analyses, single-cell RNA sequencing, immunohistochemistry of clinical specimens, and in vitro cellular assays were employed to investigate the role of the MPGS hub gene, PYGM, in tumor biology and immune modulation. Results The MPGS exhibited robust prognostic capability and effectively predicted responses to immunotherapy and various chemotherapeutic agents. Both MPGS and its central metabolic component, PYGM, were closely linked to M2 macrophage infiltration, immunosuppressive tumor microenvironments, and poor clinical outcomes in rectal adenocarcinoma. Single-cell transcriptomic analysis revealed that malignant epithelial cells with elevated PYGM expression are metabolically active and closely interact with M2 macrophages. Clinical tissue analyses and functional assays confirmed that PYGM is upregulated in rectal cancer and promotes tumor cell proliferation, migration, and M2 macrophage polarization. Conclusions This study firstly highlights PYGM as a key metabolic and immunological regulator in rectal cancer, with significant prognostic and therapeutic implications. MPGS and PYGM may serve as novel biomarkers for risk stratification and guide personalized treatment strategies in patients with rectal adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8198c80f5a26115add8dc0453b94b75563056daf" target='_blank'>
              Machine learning identifies PYGM as a macrophage polarization–linked metabolic biomarker in rectal cancer prognosis
              </a>
            </td>
          <td>
            Chengyuan Xu, Siqi Zhang, Bin Sun, Zicheng Yu, Hailong Liu
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48a5df41c5ba0be3a9d26dc85ce353b479996b4" target='_blank'>
              A patient-derived orthotopic xenograft model unveils metastatic dynamics in head and neck squamous cell carcinoma.
              </a>
            </td>
          <td>
            Yuchen Bai, Benjamin Blyth, Jarryd M Boath, Qiji Deng, Ruihong Huang, C. Darido
          </td>
          <td>2025-08-11</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Current molecular quantitative trait locus catalogs are mostly at bulk resolution and centered on Europeans. Here, we constructed an immune cell atlas with single-cell transcriptomics of >1.5 million peripheral blood mononuclear cells, host genetics, plasma proteomics and gut metagenomics from 235 Japanese persons, including patients with coronavirus disease 2019 (COVID-19) and healthy individuals. We mapped germline genetic effects on gene expression within immune cell types and across cell states. We elucidated cell type- and context-specific human leukocyte antigen (HLA) and genome-wide associations with T and B cell receptor repertoires. Colocalization using dynamic genetic regulation provided better understanding of genome-wide association signals. Differential gene and protein expression analyses depicted cell type- and context-specific effects of polygenic risks. Various somatic mutations including mosaic chromosomal alterations, loss of Y chromosome and mitochondrial DNA (mtDNA) heteroplasmy were projected into single-cell resolution. We identified immune features specific to somatically mutated cells. Overall, immune cells are dynamically regulated in a cell state-dependent manner characterized with multiomic profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/485aaa6c6ae7c10944eff4bdd9fd536b93adfcd9" target='_blank'>
              Deciphering state-dependent immune features from multi-layer omics data at single-cell resolution
              </a>
            </td>
          <td>
            R. Edahiro, Go Sato, Tatsuhiko Naito, Y. Shirai, R. Saiki, K. Sonehara, Yoshihiko Tomofuji, Kenichi Yamamoto, S. Namba, Noa Sasa, Genta Nagao, Qingbo S. Wang, Yugo Takahashi, Takanori Hasegawa, T. Kishikawa, Ken Suzuki, Yu-Chen Liu, D. Motooka, Ayako Takuwa, Hiromu Tanaka, S. Azekawa, H. Namkoong, R. Koike, Akinori Kimura, S. Imoto, Satoru Miyano, Takanori Kanai, K. Fukunaga, Mamoru Uemura, T. Morita, Yasuhiro Kato, H. Hirata, Y. Takeda, Y. Doki, H. Eguchi, Daisuke Okuzaki, Shuhei Sakakibara, Seishi Ogawa, Atsushi Kumanogoh, Yukinori Okada
          </td>
          <td>2025-07-28</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>54</td>
        </tr>

        <tr id="Accurate determination of lineage or evolutionary pathways is critical for understanding the dynamic developmental and temporal progression patterns observed in single cell omics data. In this study, we present a computational approach designed to infer progression patterns in cell populations that are actively evolving, differentiating or progressing along dynamic pathways. Our method works at single cell resolution using single cell RNA-Seq data. We determine the optimal cell order based on the cellular transcriptional profiles to uncover the progression of cell populations along differentiation, signaling, or tumor evolution trajectories. To achieve this, we developed a seriation-based method for progression pattern inference, leveraging optimally reordered hierarchies. We provide advanced visualization tools using principal curves to represent the inferred pathways in a three-dimensional latent space derived from scRNA-Seq data. Furthermore, we propose new metrics for evaluating the accuracy of the reconstructed order and identified pathways. The effectiveness of our approach is demonstrated through applying the analysis to real human colon samples transcriptomics data that we generated specifically for this study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/004d6d59d63d9011d09efaebc9853e9f6c9a5089" target='_blank'>
              Inference of Tumor Progression Patterns in Colon Cancer using Optimal Cell Order Analysis in Single Cell Resolution.
              </a>
            </td>
          <td>
            Marmar Moussa, Charles H. Street
          </td>
          <td>2025-07-24</td>
          <td>IEEE transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Myelodysplastic syndromes (MDS) are heterogeneous hematopoietic stem cell disorders defined by ineffective hematopoiesis, multilineage dysplasia, and risk of progression to acute myeloid leukemia. Improvements have been made to identify recurrent genetic mutations and their functional roles, but translating this into preclinical models is still difficult. Traditional murine systems lack the human-specific cytokine support and microenvironmental support that is necessary to reproduce MDS pathophysiology. Humanized mouse models, particularly those incorporating human cytokines (e.g., MISTRG, NSG-SGM3, NOG-EXL), immunodeficient backgrounds, and co-transplantation strategies, have improved the engraftment and differentiation of human hematopoietic stem and progenitor cells. These models allow the study of clonal evolution, mutation-specific disease dynamics, and response to therapies in vivo. However, difficulties persist, such as limited long-term engraftment, incomplete immune reconstruction, and limited possibilities of modeling early-stage or low-risk MDS. This review presents an overview of current humanized and genetically engineered mouse models suitable for studying MDS, evaluating their capacity to replicate disease complexity, preserve clonal architecture, and support translational research. We highlight the need to develop new approaches to improve the actual methodologies and propose future directions for standardization and improved clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15f36c2241b27aeba19116a964e555c1a089cad1" target='_blank'>
              Humanized mouse models in MDS
              </a>
            </td>
          <td>
            R. Munteanu, Diana Gulei, C. Moldovan, Emanuele Azzoni, Laura Belver, Richard-Ionut Feder, Simina Pîrv, A. Buzoianu, Hermann Einsele, Moshe Mittelman, Gabriel Ghiaur, Robert P Hasserjian, C. Tomuleasa
          </td>
          <td>2025-07-17</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Osteosarcoma, a prevalent malignant bone tumor in pediatric and adolescent populations, is characterized by a markedly diverse tumor microenvironment (TME) that complicates therapeutic intervention. Among the key components of the TME, M2 macrophages contribute to immune suppression, tumor progression, and resistance to therapy. This study aims to explore the immune microenvironment of osteosarcoma and identify key immunosuppressive components using integrated transcriptomic analysis.



 We integrated scRNA-seq (GSE152048) and bulk RNA-seq (TARGET) data to characterize the osteosarcoma tumor microenvironment. Key analyses included cell clustering and annotation (Seurat), pseudotime trajectory (Monocle 2), cell-cell communication (CellChat), and M2 macrophage deconvolution (CIBERSORT, BayesPrism). Gene modules were identified via WGCNA, and drug sensitivity was predicted using oncoPredict. Key genes were validated by qRT-PCR and Western blot.



 Our integrated analysis revealed that M2 macrophage infiltration was significantly associated with an immunosuppressive tumor microenvironment in osteosarcoma. Weighted gene co-expression network analysis (WGCNA) identified gene modules highly correlated with M2 macrophage abundance and immune evasion signatures. Reduced M2 infiltration was linked to altered transcriptional patterns and increased predicted sensitivity to Etoposide and Doxorubicin. Two immune-related genes, PTK2B and PTPRC, were significantly upregulated in osteosarcoma samples and validated by RT-qPCR and Western blotting in U2OS cells.



 This study demonstrates that M2 macrophage infiltration is a defining feature of the osteosarcoma immune microenvironment and is associated with immune suppression. Through integrative transcriptomic analysis, PTK2B and PTPRC were identified as key immune-related genes linked to M2 macrophage abundance and patient prognosis. These results establish a transcriptional basis for the immunosuppressive phenotype in osteosarcoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c9aeb21260637fca67e1110200fbde1ea1d4d" target='_blank'>
              Integrated RNA sequencing reveals tumor microenvironment heterogeneity and immunosuppressive role of M2 macrophages in osteosarcoma
              </a>
            </td>
          <td>
            Zhijun Ding, Yi Li, Cheng Zhong
          </td>
          <td>2025-08-01</td>
          <td>Oncologie</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The process of wound healing is regulated by complex interactions between different cell types across space and time. Through the profiling of individual cells within their complex environment, single-cell transcriptomics methods enable the investigation of cellular heterogeneity, cell communication networks, and cell-cell interactions involved in the wound healing process. However, many single-cell analysis tools are run within a computer coding environment, and their more widespread use by wound healing scientists is thwarted by the apparent lack of bioinformatics expertise. Therefore, a step-by-step workflow is presented showing how to use a graphical coding environment called RStudio to perform a basic single-cell analysis of a temporal mouse excisional skin wound healing dataset. This visual and guided protocol will enable scientists with no bioinformatics background to download a previously published wound healing dataset, perform critical quality control steps, run a standard single-cell analysis workflow including dataset visualizations and cell type annotations using Seurat, run cell subtype analyses, run module scoring analyses, run cell-cell interaction analyses using CellChat, and perform integrative analyses of multiple datasets using Seurat. Narrative explanations are provided for each step in the protocol and graphical results from every line of code are presented to safely guide the user through the workflow. The goal of this visual introduction to a single-cell analysis pipeline is to enable more wound healing scientists to use bioinformatics tools directly in their own laboratories in order to facilitate deeper analyses of their own single-cell datasets as well as more widespread re-analyses of previously published single-cell datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01aaa9ce25d59a7c9eba1c49987e0404e67db6a5" target='_blank'>
              Using R, Seurat, and CellChat to Analyze a Single-Cell Transcriptomics Dataset of Mouse Skin Wound Healing.
              </a>
            </td>
          <td>
            Shalyn Keiser, Nichole Botello, Ethan Cruz, Mateusz S. Wietecha
          </td>
          <td>2025-08-01</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Identifying accurate cell markers in single-cell RNA-seq data is crucial for understanding cellular diversity and function. Localized Marker Detector (LMD) is a novel tool to identify “localized genes”—genes exclusively expressed in groups of highly similar cells—thereby characterizing cellular diversity in a multi-resolution and fine-grained manner. LMD constructs a cell-cell affinity graph, diffuses the gene expression value across the cell graph, and assigns a score to each gene based on its diffusion dynamics. LMD’s candidate markers can be grouped into functional gene modules, which accurately reflect cell types, subtypes, and other sources of variation such as cell cycle status. We apply LMD to mouse bone marrow and hair follicle dermal condensate datasets, where it facilitates cross-sample comparisons by identifying shared and sample-specific gene signatures and novel cell populations, without requiring batch effect correction or integration. We also assess the performance of LMD across ten single-cell RNA sequencing datasets, compare it to eight existing methods with similar objectives, and find that LMD outperforms the other methods evaluated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdcbdb187baf4d5d7e985679ce461af8d3a10bd" target='_blank'>
              Cluster-independent multiscale marker identification in single-cell RNA-seq data using localized marker detector (LMD)
              </a>
            </td>
          <td>
            Ruiqi Li, Rihao Qu, Fabio Parisi, Francesco Strino, Hainan Lam, Jay S. Stanley, Xiuyuan Cheng, Peggy Myung, Y. Kluger
          </td>
          <td>2025-07-16</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Background Glioma, a tumor derived from neuroglial cells of the central nervous system, is characterized by its high malignancy, frequent recurrence, and high mortality rate. Recent advances in exosome research related to gliomas have propelled this emerging field as a pivotal area of clinical neuroscience investigation. Despite extensive focus on glioma exosome research, a systematic visual analysis of this field remains lacking. Using bibliometric analysis, this study maps the scientific landscape of glioma exosome research to identify current hotspots and emerging trends in neuro-oncology. Methods This bibliometric analysis extracted glioma exosome publications from the Web of Science Core Collection(WoSCC) spanning 2005–2025. Following systematic screening, 209 articles constituted the analytic corpus. Using VOSviewer(VOS), Scimago Graphica, and CiteSpace, we mapped contributions across geographic, institutional, authorship, and conceptual domains. This visualization delineates the field’s intellectual structure, identifies current research foci, and projects emerging trends in neuro-oncology. Results Annual publication volume on glioma exosomes has increased significantly since 2019. China and the United States are the predominant contributors, collectively driving field advancement. Leading institutions include Shandong University, Shandong Provincial Key Laboratory of Brain Function Reconstruction, and Capital Medical University, whose productivity metrics reflect substantial influence. Notably, Chinese institutions comprise 90% of the top 10 productive entities. Key investigators Li Gang and Xue Hao demonstrate exceptional scholarly output, advancing the field through high-impact research. NEURO-ONCOLOGY emerges as the premier journal, evidenced by its dominant citation frequency. Thematic analysis reveals three conceptual domains: Exosome biogenesis in gliomas, Glioblastoma extracellular vesicle signatures, and Exosome mediated drug delivery systems-representing current priorities and evolving research frontiers. Conclusion Exosomes have attracted considerable attention in glioma research, establishing themselves as a cutting edge thematic focus. This study applies bibliometric rigor to map research trajectories and conceptual evolution in glioma exosomes. The analysis establishes foundational frameworks that prioritize future investigation domains while objectively quantifying the field’s translational potential. These evidence-based insights will accelerate mechanistic discovery and clinical translation in neuro-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce5f890f1f8b8ce61524a09398cf2f3457fe7ab4" target='_blank'>
              Evolution of glioma exosome research: emerging trends and global collaborations (2005–2025)
              </a>
            </td>
          <td>
            Yijun Zeng, Ting Liu, Shixue Jing, Yongfeng Wang, Jihong He, Hui Cao, Xinlian Liu, Lushun Zhang
          </td>
          <td>2025-07-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Immune evasion and suppression lead to unchecked tumor growth in glioblastoma. Cytomegalovirus (CMV) has been implicated in tumor progression and modulation in glioblastoma. To investigate this potential connection, CMV-associated changes in the glioblastoma immune landscape were characterized in vitro and in a murine glioblastoma model. Infection of mouse glioblastoma cells (GL261Luc2) with mCMV resulted in a short period of viral replication. MHC-I cell surface expression was reduced after mCMV infection by approximately 40% compared with non-infected tumor cells (p < 0.0001). Viral regulators of antigen presentation (vRAP) were shown to be responsible for MHC-I downregulation using a recombinant mCMV (ΔvRAP) lacking the known immune evasion genes. RNA sequencing of mCMV infected GL261Luc cells revealed 2711 differentially expressed genes (p < 0.005). Of particular interest was the downregulation of MHC-I-associated genes H2-Q1-10 and Tap1 fter CMV infection. In vivo, the mCMV immediate early gene (IE1) was detected in brains of mCMV + animals after tumor implantation and increased during tumor growth. mCMV + mice had significantly shorter survival than controls, depending on initial tumor size (P < 0.001). Tumor immune infiltrates in mCMV infection were characterized by B cell infiltrates and low levels of NK cell infiltration. Here, the landscape of immune cell infiltrates is shifted toward B cell infiltration and reduced numbers of NK cells. CMV leads to immune evasion mediated MHC-I downregulation in murine glioblastoma. Thus, CMV infection in glioblastoma may contribute to unchecked tumor growth in glioblastoma by increasing immune evasion. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10107-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c26a9db53ace46c31e39ed6d28fed21d09c975" target='_blank'>
              Cytomegalovirus-induced oncomodulation drives immune escape in glioblastoma
              </a>
            </td>
          <td>
            Harald Krenzlin, F. Corr, Deepak Ailani, P. Einheuser, Thomas Bukur, Thomas Rößler, Alina Henrich, Raja Hollnagel, Alice Dauth, Libo Hu, Leon Schmidt, M. Griessl, Michael Gutknecht, Noe Mercado, Beat Alessandri, Charles H. Cook, Florian A Ringel, S. Lawler, Niels A. Lemmermann, N. Keric
          </td>
          <td>2025-07-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a prevalent and malignant tumor of the digestive system, characterized by high incidence and mortality rates. This study aimed to investigate the heterogeneity of the tumor microenvironment (TME) and the involvement of immune cells in CRC. Single-cell RNA sequencing (scRNA-seq) data obtained from the Gene Expression Omnibus (GEO) database were used to analyze and identify six major cell types across normal, core, and border tumor samples. A total of 27414 cells from various regions of patients with CRC were selected for subsequent analyses. Cellular interaction analysis revealed that differential signaling pathways between the TME and normal tissues, with several pathways involving interactions between myeloid cells and epithelial cells. Myeloid cells were extracted and classified into six subtypes based on markers identified in the literature. Monocle3 revealed the trajectory of tumor-associated macrophages (TAMs) and identified genes associated with pseudotime. Single-Cell ENrichment analysis for Interpreting Cellular Heterogeneity (SCENIC) analysis identified specific regulons and target genes associated with TAMs. This study reanalyzed single-cell RNA-sequencing data and provided insights into the heterogeneity of the TME, particularly in relation to the role of TAMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e37ff9c0af32829265ad92932c051ab5305d1fd" target='_blank'>
              Single-cell RNA sequencing analysis reveals the heterogeneity and effect of TAMs in colorectal cancer.
              </a>
            </td>
          <td>
            Chengang Wang, Ying Qian, Mingsheng Zhang
          </td>
          <td>2025-08-20</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Developing advanced preclinical models and targeted therapies is essential for reducing cancer-related deaths in children with solid tumors. Patient-derived xenografts (PDX) have the potential to replicate key elements of the original tumor, including morphology, genetic alterations, and microenvironment, making them valuable tools for studying tumor biology and drug response. We implanted 124 pediatric solid tumor samples, collected for 1 y, into NOD/SCID/IL2Rg (NSG) mice. Tumor fragments were placed subcutaneously, and the animals were monitored for up to 1 y. Histopathology, Short Tandem Repeat (STR) profiling, RT-PCR and/or RNA-sequencing were performed to confirm tumor identity and detect driver fusions. Fifty-five xenografts were successfully established (44.35% of implanted samples), representing 19 tumor types. Sarcomas, notably osteosarcoma, Ewing sarcoma, synovial sarcoma, and rhabdomyosarcoma, displayed first-generation engraftment rates above 55%. Central nervous system tumors had lower success, reflecting unique microenvironmental requirements. Histopathology and STR concordances were 85.45% and 81.1%, respectively, while 92.6% of sarcoma PDXs retained original fusion genes. Second-generation xenografts showed faster growth, suggesting adaptation to the murine host. Sporadic discrepancies, such as new fusions or lymphoproliferative expansions, indicated the need for ongoing molecular validation parallel to other techniques. A pediatric PDX biobank can effectively capture key tumor features while facilitating the study of therapeutic responses and tumor evolution. Our models confirm the feasibility of achieving stable histological and molecular profiles, offering a valuable resource for precision oncology research. Ultimately, these pediatric PDXs could accelerate the discovery of targeted therapy and significantly improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e2beb1824c0da93aca0d94e158f83285bfd918e" target='_blank'>
              Establishing a pediatric solid tumor PDX biobank for precision oncology research
              </a>
            </td>
          <td>
            Larissa Akemi Kido, Milena Rodrigues Marusco, Ellen Aparecida da Silva, Laís Do Carmo, Ana Beatriz Teodoro Borges, Felipe Luz Torres Silva, Juliana Silveira Ruas, Dieila Giomo de Lima, Larissa de Abreu Fernandes, Camila Maia Martin Daiggi, Izilda Aparecida Cardinalli, Mayara Ferreira Euzébio, Patricia Yoshioka Jotta, Mariana Maschietto, Priscila Pini Zenatti
          </td>
          <td>2025-08-13</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a578f7d230f21ac2bdf4f8d637eac1581c89d9f7" target='_blank'>
              CXCL13 as a Prognostic Biomarker and Modulator of the Tumor Microenvironment in Colorectal Cancer.
              </a>
            </td>
          <td>
            Fang Fang Guo, Yu Qing Yan, Wei Chen, Yao Cheng, Rui Zhang, Chao Qin Shen, Yun Cui, Yan Shen Peng, Hao Yan Chen, Lin Hua Ji, Bao Qin Xuan, Xiao Qiang Zhu
          </td>
          <td>2025-08-13</td>
          <td>Journal of digestive diseases</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Patient-derived xenografts (PDXs) are frequently used as preclinical models, but their recapitulation of tumour metabolism in patients has not been closely examined. We developed a parallel workflow to analyse [U-13C]glucose tracing and metabolomics data from patient melanomas and matched PDXs. Melanomas from patients have substantial TCA cycle labelling, similar to levels in human brain tumours. Although levels of TCA cycle labelling in PDXs were similar to those in the original patient tumours, PDXs had higher labelling in glycolytic metabolites. Through metabolomics, we observed consistent alterations of 100 metabolites among PDXs and patient tumours that reflected species-specific differences in diet, host physiology and microbiota. Despite these differences, most of nearly 200 PDXs retained a ‘metabolic fingerprint’ largely durable over six passages and often traceable back to the patient tumour of origin. This study identifies both high- and low-fidelity metabolites in the PDX model system, providing a resource for cancer metabolism researchers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b72e5e72726ae899a2f43f50766abe84e03ec90" target='_blank'>
              Conservation and divergence of metabolic phenotypes between patient tumours and matched xenografts
              </a>
            </td>
          <td>
            A. Rao, Ling Cai, Marelize Snyman, R. E. Walsdorf, Xiangyi Li, Sophia N Wix, Gabrielle Gard, Ariel B Brown, Juliana H. Kim, Joao Santos Patricio, Sarah Muh, Misty Martin Sandoval, Lauren G. Zacharias, Kristen A Heimdal, Wen Gu, Jade Homsi, Brittny Tillman, Rohit Sharma, Travis Vandergriff, Ashley Solmonson, B. Faubert, Thomas P. Mathews, Sean J. Morrison, R. DeBerardinis, Jennifer G Gill
          </td>
          <td>2025-07-29</td>
          <td>Nature Metabolism</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) remains a daunting global health concern. It is marked by aggressive progression and poor survival. While immunotherapy has emerged as a promising treatment modality, both primary and acquired resistance continue to limit its clinical impact, leaving many patients without durable benefits (e.g., CheckMate-648, ESCORT-1st). This review explains resistance mechanisms and suggests new strategies to improve outcomes. These mechanisms include immunosuppressive cells (Treg cells, myeloid-derived suppressor cells), inhibitory cytokines, molecular alterations involving programmed death 1/programmed death-ligand 1 signaling, and impaired antigen presentation. We also highlight key clinical trials—for example, CheckMate-648 and ESCORT-1st—that reveal both the potential and pitfalls of current immune checkpoint blockade strategies, underscoring the need for robust predictive biomarkers. Moreover, we examine cutting-edge tactics to overcome resistance, including combination regimens, tumor microenvironment remodeling, and tailored treatment approaches rooted in the patient’s unique genomic and immunologic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8755d6f455eb60021c79724f1c8204130606cf4" target='_blank'>
              Unmasking immune checkpoint resistance in esophageal squamous cell carcinoma: Insights into the tumor microenvironment and biomarker landscape
              </a>
            </td>
          <td>
            Zhe Wang, Rui-Ying Zhang, Yi-Fan Xu, Bingtong Yue, Jia-Yi Zhang, Feng Wang
          </td>
          <td>2025-08-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Steatotic liver disease-related hepatocellular carcinoma (SLD-HCC), a rising global challenge, is characterised by unique tumour microenvironment (TME) adaptations.


OBJECTIVE
This study investigates the immune microenvironment and interactions driving immunosuppression and potential resistance to immunotherapy in SLD-HCC.


DESIGN
We employed single-cell transcriptomics, cytometry by time-of-flight (CyTOF) and two independent spatial transcriptomics platforms to study the TME of 22 SLD-HCC and 31 non-SLD-HCC cases. Findings were further validated using multiplex immunohistochemistry in an independent cohort of 103 patients, an HCC model and an immunotherapy-treated patient cohort to evaluate clinical relevance.


RESULTS
Our findings revealed significant alterations in immune and lipid metabolism pathways, particularly in regulatory T cells (Tregs) and cancer-associated fibroblasts (CAFs), suggesting distinct cellular adaptations to a high-fat TME and general immunosuppression. CyTOF revealed a cold, immunosuppressive TME with reduced CD8+ T cells and increased Tregs. Spatial transcriptomics further highlighted distinct Treg-CAF clusters localised at tumour margins, suggesting a spatially organised immunosuppressive niche. Mechanistically, tumour necrosis factor superfamily member 14 (TNFSF14)-tumour necrosis factor receptor superfamily member 14 (TNFRSF14)-mediated Treg-CAF interaction was identified as a critical driver of immunotherapy resistance in SLD-HCC. Blocking TNFRSF14 in an HCC model fed with a high-fat diet resulted in reduced Tregs, increased active CD8+ and memory CD4+ T cells, and a synergistic effect with anti-programmed cell death protein 1 therapy to enhance antitumour immunity and overcome immunotherapy resistance in SLD-HCC.


CONCLUSION
This study uncovers critical immune and metabolic adaptations in SLD-HCC, identifying TNFSF14-TNFRSF14 signalling as a key driver of immunotherapy resistance. Targeting this signalling axis enhances antitumour immunity and improves immunotherapy efficacy, offering a promising therapeutic strategy for SLD-HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8945e27ac60921d9362c3d051e5dc0848a2a4ff0" target='_blank'>
              Targeting Treg-fibroblast interaction to enhance immunotherapy in steatotic liver disease-related hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Aldo Prawira, Hang Xu, Yu Mei, N. J. M. Nasir, Marie J Y Reolo, Masayuki Otsuka, W. Leow, Mohammad Rahbari, Ziyao Chen, M. Weerasooriya, Liyana Bte Abdullah, Jiawei Wu, S. Hazirah, Martin Wasser, A. Chung, Brian K. P. Goh, Pierce Chow, Salvatore Albani, Joycelyn Lee, T. Lim, Weiwei Zhai, Y. Dan, George Bb Goh, Mathias F Heikenwälder, Yongliang Zhang, Ramanuj Dasgupta, Wai Fei David Tai, Haiyan Liu, Jinmiao Chen, Valerie Chew
          </td>
          <td>2025-07-22</td>
          <td>Gut</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Single-cell transcriptomics (scRNA-seq) has facilitated the characterization of cell state heterogeneity and recapitulation of differentiation trajectories. However, the exclusive use of mRNA measurements comes at the risk of missing important biological information. Here we leveraged recent technological advances in single-cell proteomics by Mass Spectrometry (scp-MS) to generate an scp-MS dataset of an in vivo differentiation hierarchy encompassing over 2500 human CD34+ hematopoietic stem and progenitor cells. Through integration with scRNA-seq, we identified proteins that are important for stem cell function, which were not indicated by their mRNA transcripts. Further, we showed that modeling translation dynamics can infer cell progression during differentiation and explain substantially more protein variation from mRNA than linear correlation. Our work offers a framework for single-cell multi-omics studies across biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6bb52db76c97923f9e0dc8a3325cf71c60438a0" target='_blank'>
              Mapping early human blood cell differentiation using single-cell proteomics and transcriptomics.
              </a>
            </td>
          <td>
            B. Furtwängler, Nil Üresin, Sabrina Richter, M. Schuster, Despoina Barmpouri, Henrietta Holze, Anne Wenzel, Kirsten Grønbæk, Kim Theilgaard-Mönch, F. Theis, E. Schoof, Bo T Porse
          </td>
          <td>2025-08-21</td>
          <td>Science</td>
          <td>0</td>
          <td>16</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>