<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-11 09:35:10 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal cancer, with liver metastases significantly worsening outcomes. However, distinct features of the tumor microenvironment (TME) between primary and metastatic sites remain poorly defined. Cellular neighborhoods within the TME are recognized as functional units that influence tumor behavior. Conventional spatial methods, which assign equal weights to all cells in a region, fail to capture the nuances of cellular interactions. To address this, we developed Functional Cellular Neighborhood (FunCN) quantification, which integrates both the proportion and proximity of surrounding cells. Applying FunCN to PDAC imaging mass cytometry data, we identified neutrophil-enriched interactions in liver metastases compared to primary tumors, correlating with elevated VISTA expression by tumor cells. Additionally, FunCN clusters around CD8+ T cells in pancreas and liver were associated with higher TIGIT and LAG3, respectively. These findings demonstrate the importance of spatial immune landscapes in PDAC and identify potential therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c056a7bc97d38004639e645962dc9ff9c3cfad" target='_blank'>
              Modeling cellular influence delineates functionally relevant cellular neighborhoods in primary and metastatic pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yeonju Cho, Jae W. Lee, Sarah M. Shin, Alexei Hernandez, Xuan Yuan, Jowaly Schneider, Jody E. Hooper, Laura D. Wood, Elizabeth M. Jaffee, Atul Deshpande, W. Ho
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is characterized by high malignancy and early propensity for metastasis, and modest response to immunotherapy due to the immunosuppressive microenvironment. Surgical intervention has shown benefits in treating early-stage SCLC. However, most patients experience recurrence after surgery. The factors associated with relapse free survival in these patients remain unclear. We collected operation specimens from ten early-stage SCLC patients (N0M0), conducted long-term follow-up, and grouped them based on disease status. Subsequently, we performed a retrospective analysis using single-cell spatial imaging mass cytometry to explore the characteristics of tumor cells and differences in the tumor microenvironment, especially the single-cell constitute of immune cells, between the two groups. We found that, in early-stage SCLC, tumor cells display pronounced heterogeneity, both intra-group and inter-group. Patients with early recurrence are characterized by a distinct subpopulation of tumor cells with high Ki-67 expression. Non-relapse patients demonstrate better infiltration of M1 macrophages and stromal cells. Neighborhood analysis suggested that positive interactions between macrophages, stromal cells, and T cells with tumor cells may benefit patient prognosis. Additionally, recurrent tumor cells might enhance their metastatic capacity and remodel the microenvironment through upregulation of GranzymeB or reduction of c-Myc expression. In conclusion, SCLC tumor cells demonstrate tumor heterogeneity and microenvironmental changes in the early clinical stages. A higher proportion of M1 macrophages is associated with prolonged postoperative survival in early-stage SCLC patients. This research provides novel insights and evidence for treating and preventing postoperative recurrence in SCLC. Supplementary Information The online version contains supplementary material available at 10.1007/s10565-025-10056-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1755c9b716477f70da2c740f51d015cee2a36b" target='_blank'>
              Single-cell spatial proteomics of non-relapse small cell lung cancer identifies tumor microenvironment determinants of survival
              </a>
            </td>
          <td>
            Yin Li, L. Xia, Hui Wang, Xinghao Ai, Ying Wang, Qingquan Luo, Yuchen Han, Shun Lu, Xinghua Cheng
          </td>
          <td>2025-06-17</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Early detection of melanoma through skin surveillance is critical for preventing metastatic progression. Primary cutaneous melanomas at early stage offer a unique opportunity to uncover fundamental mechanisms of tumor initiation, progression, and immune surveillance, but detailed spatial profiling of early disease remains limited. Here we integrate high-plex cyclic immunofluorescence (CyCIF) imaging, spatial transcriptomics, and conventional histology to identify factors associated with de-differentiation and dermal invasion in early-stage melanomas. We demonstrate a high level of variability from one primary cancer to the next, from one 100-300 cell microregion to the next within a single cancer, and from one cell to the next within a microregion. Intra-tumoral heterogeneity is influenced by local features of the microenvironment including proximity to T and myeloid cells and to perivascular environments. Thus, tumor plasticity and spatial heterogeneity arise early in melanoma development, potentially allowing for competition among multiple tumor states during the emergence of invasive disease. STATEMENT OF SIGNIFICANCE This study shows that tumor cell dedifferentiation and plasticity are locally patterned in primary human melanomas under the influence of microenvironment components such as myeloid cells and vasculature. As a result, neighborhoods in a single tumor can be as different from each other as in different tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bc89e4de14d8f34f2814e7cf735899a3c934813" target='_blank'>
              Spatial determinants of tumor cell dedifferentiation and plasticity in primary cutaneous melanoma
              </a>
            </td>
          <td>
            Tuulia Vallius, Yingxiao Shi, Edward Novikov, Shishir M Pant, R. Pelletier, Yu-An Chen, Juliann B. Tefft, Ajit Nirmal Johnson, Z. Maliga, Guihong Wan, George F. Murphy, S. Santagata, Yevgeniy R. Semenov, David Liu, C. G. Lian, P. Sorger
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a7baab96b7e0361b2e10254d2ab098fe969a9" target='_blank'>
              Tumor Cells Enriched for Interferon and Inflammatory Programs Pre-Exist in High Grade Serous Ovarian Cancer and are Proportionately Significantly Increased Post Chemotherapy
              </a>
            </td>
          <td>
            B. Winterhoff, Nomeda Girnius, Muyi Liu, Jaeyoung Kang, Mihir Shetty, M. A. Martinez-Gakidis, Weiyi Xu, Mayura Thomas, Linna Lahmadi, Aylin Z. Henstridge, Pilar Baldominos, Shobhana Talukdaer, Z. Chang, Jordan Mattson, Morgan Gruner, Sally A. Mullany, Peter Argenta, Gottfried E. Konecny, Dennis J. Slamon, Ronny Drapkin, Andrew C. Nelson, Timothy K. Starr, Joan S Brugge
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary While subcutaneous colorectal cancer (CRC) models are commonly employed in biological and preclinical studies, their cell morphological organization and molecular landscape are not yet fully understood. Using spatial transcriptomics, we identified two spatially distinct tumor regions demarcated by cancer-associated fibroblasts (CAFs), each region exhibiting unique molecular signatures and immune niche. Moreover, we uncovered specific cancer–stroma cell interactions that actively remodel the tumor microenvironment to facilitate immune evasion. These findings uncover the previously unrecognized spatial heterogeneity of the CRC mouse model and highlight the critical role of specialized tumor niches in driving tumor development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acb09b0c3fbf7dabf3a2ff1e9a531d5a3277e04" target='_blank'>
              Cancer-Associated Fibroblasts Establish Spatially Distinct Prognostic Niches in Subcutaneous Colorectal Cancer Mouse Model
              </a>
            </td>
          <td>
            Zhixian Lin, Jinmeng Wang, Yixin Ma, Yanan Zhu, Yuhan Li, Zhengtao Xiao, Wei Zhao
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6cfe1f46dc56e476a3d0be0d700257cc9ae2152" target='_blank'>
              Single-Cell and Spatial Profiling of Tumor Microenvironment Heterogeneity in Human Osteosarcomas
              </a>
            </td>
          <td>
            Xuejing zheng, Jin Yuan, Peng Jin, Xuejun Wang, Runlei Xu, Helin Feng, Xinxin Zhang, Yongjie Xie, Wence Wu
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumor microenvironment is heterogeneous, structurally complex, and continually evolving, making it difficult to fully capture. Common dissociative techniques thoroughly characterize the heterogeneity of cellular populations but lack structural context. The recent boom in spatial analyses has exponentially accelerated our understanding of the structural complexity of these cellular populations. However, to understand the dynamics of cancer pathogenesis, we must assess this heterogeneity across space and time. In this review, we provide an overview of current dissociative, spatial, and temporal analysis strategies in addition to existing and prospective spatiotemporal techniques to illustrate how understanding the tumor microenvironment, focusing on dynamic immune-cancer cell interactions, across four dimensions will advance cancer research and its diagnostic and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1f3339cce25c83c98dd45cea90c67e2a48bc606" target='_blank'>
              The tumor microenvironment across four dimensions: assessing space and time in cancer biology
              </a>
            </td>
          <td>
            Christina R. Larson, Ayushi Mandloi, Satwik Acharyya, Julienne L. Carstens
          </td>
          <td>2025-06-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Transcriptome profiling of bladder cancer has revealed distinct basal-like and luminal-like molecular subtypes, which may be correlated with pathological subtypes of different patient outcomes. However, whether these molecular subtypes originate from the corresponding cell types in the normal urothelium and whether different cells of origin influence bladder cancer progression remain unclear. Here, we conducted cell-type-specific lineage tracing in CRISPR/Cas9-induced mouse bladder cancer models of Pten and Trp53 targeting. We show that although basal, intermediate, and superficial umbrella cells can all serve as the cell of origin for bladder cancer, transformed umbrella cells were gradually displaced by tumor cells from inner layers, particularly transformed basal cells, which had highest stemness. Histological and single cell RNA-sequencing data comparing basal- and intermediate-cell-induced bladder tumors revealed that basal-induced tumors displayed higher heterogeneity, and contained unique cell clusters including Krt14+Ki67+ highly proliferative basal cells, squamous cell carcinoma, and transitioning cells towards the Gata3+ luminal subtype. Trajectory analysis confirmed the cell lineage differentiation hierarchy uncovered in lineage tracing. Moreover, human bladder cancer molecular subtype signatures were highly enriched in mouse tumor cell clusters of the corresponding cell of origin, and a gene signature derived from the unique basal-induced clusters is predictive of worse patient outcome. Overall, our results support that the basal and luminal molecular subtypes of bladder cancer have the corresponding cells of origin as their basis, and that urothelial basal cells are intrinsically more competitive than intermediate and umbrella cells in generating aggressive bladder cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c74441168c668a9994b2523834450566ba1b30" target='_blank'>
              Intrinsic higher potency of basal urothelial cells to intermediate and umbrella cells as the cell of origin for bladder cancer
              </a>
            </td>
          <td>
            Chuan Yu, Nicholas Chu, Alexandra Aguirre, Jonathan Green, Qing Xie, Beatrice Knudsen, Zhu A. Wang
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b6a64a6d2ad28ecff7ab0f31663d8f3fd11b6a" target='_blank'>
              A spatial atlas of chemoradiation therapy in pancreatic cancer identifies cellular and microenvironmental determinants of persister populations
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, Ariana Acevedo-Diaz, K. Rajapakshe, Galia Jacobson, Daniela Tovar, Jimin Min, Guangsheng Pei, Candise Tat, Ayush Suresh, Ching-Wei Tzeng, Matthew HG Katz, M. Bhutani, Huaming Wang, Robert A. Wolff, C. Haymaker, E. Ludmir, Huocong Huang, Xiongfeng Chen, Liang Li, Albert C Koong, Linghua Wang, N. Navin, Dadi Jiang, Ziyi Li, Anirban Maitra, Eugene J. Koay
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease, with tumors capable of adapting to shifting conditions, making the development of effective personalized therapies particularly challenging. Patient-derived models, such as patient-derived organoids (PDOs) and circulating tumor cell (CTC) cultures, have emerged as powerful tools for investigating intra- and inter-tumor heterogeneity. These models largely retain the genetic, phenotypic, and microenvironmental features of the original tumors, providing valuable insights into disease progression, drug response, and resistance mechanisms. Furthermore, by enabling tumors’ spatiotemporal molecular profiling, PDOs and CTCs offer a dynamic approach to assess treatment efficacy over time. However, to fully capture the complexity of breast cancer heterogeneity, it is required to develop models from multiple tumor and blood samples collected throughout the course of treatment. This review explores the potential of integrating PDOs and CTC models to better understand intra-tumor heterogeneity while addressing key challenges in developing patient-derived models that accurately recapitulate patients’ tumors to advance personalized care. The integration of PDOs and CTCs could represent a paradigm shift in the personalized management of metastatic breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a82abf6024e151676efff540be9bde6be013c6" target='_blank'>
              Deciphering Breast Tumor Heterogeneity Through Patient-Derived Organoids and Circulating Tumor Cells
              </a>
            </td>
          <td>
            Benedetta Policastro, Nikoline Nissen, Carla L. Alves
          </td>
          <td>2025-06-25</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83adb5995a58165fece89d2aea822be3362a5535" target='_blank'>
              Epigenetic Subtypes of High-Grade T1 Bladder Cancer Reveal Intra-Tumor Heterogeneity and Distinct Interactions with Tumor Microenvironment
              </a>
            </td>
          <td>
            J. Bellmunt, Yingtian Xie, N. Juanpere, M. Munoz, Shweta Kukreja, Sonsoles Liria Veiga, Rong Li, Xintao Qiu, Yijia Jiang, Alba Font-Tello, Marie Nunez Duarte, Ilana B. Epstein, Silvia Hernández-Llodrà, Marta Lorenzo, Silvia Menendez, T. Choueiri, Myles Brown, H. Long, Paloma Cejas
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Melanoma exhibits high intratumoral heterogeneity, characterized by a diverse population of cells undergoing dynamic transitions between cellular states. These adaptive changes enable melanoma cells to survive in the harsh tumor microenvironment, acquire drug resistance, and metastasize. One such state, quiescence, has been linked to both relapse and drug resistance, but its underlying biology and molecular mechanisms remain poorly understood. Our study challenges the conventional understanding of melanoma quiescence. Contrary to the notion of a rare, unique subpopulation, we demonstrate that quiescence is a highly dynamic state accessible to most, if not all, melanoma cells. This state is exquisitely sensitive to microenvironmental cues. We identify GPNMB as a marker of quiescence, that is expressed in both primary and metastatic tumors. GPNMB-positive cells exhibit a pro-metastatic phenotype and are enriched in metastatic sites, suggesting a potential role for quiescence in tumor dissemination. Our findings position GPNMB as a valuable marker for isolating quiescent melanoma cells and as a potential therapeutic target to tackle metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a119955e8e93a780087f38a6e6b1bd83175ea168" target='_blank'>
              GPNMB marks a quiescent cell population in melanoma and promotes metastasis formation
              </a>
            </td>
          <td>
            F. Lotti, Marine Melixetian, T. Vlachou, M. S. Nobile, Leone Bacciu, M. Malferrari, Nicolò Quaresima, Stefania Rapino, F. Marocchi, Massimo Barberis, C. Soriani, Barbara Gallo, Velia Mollo, Ilaria Ferrarotto, Daniela Bossi, P. Ferrucci, PierGiuseppe Pelicci, L. Luzi, L. Lanfrancone
          </td>
          <td>2025-06-17</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe2aa8fe44537c3fe83f62a4d4acf8f8b14517" target='_blank'>
              STHELAR, a multi-tissue dataset linking spatial transcriptomics and histology for cell type annotation
              </a>
            </td>
          <td>
            Félicie Giraud-Sauveur, Quentin Blampey, Hakim Benkirane, A. Marinello, PH Cournède, Stergios Christodoulidis
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) has a poor survival rate due to late detection. PDAC arises from precursor microscopic lesions, termed pancreatic intraepithelial neoplasia (PanIN), that develop at least a decade before overt disease; this provides an opportunity to intercept PanIN-to-PDAC progression. However, immune interception strategies require full understanding of PanIN and PDAC cellular architecture. Surgical specimens containing PanIN and PDAC lesions from a unique cohort of 5 treatment-naive patients with PDAC were surveyed using spatial omics (proteomic and transcriptomic). Findings were corroborated by spatial proteomics of PanIN and PDAC from tamoxifen-inducible KPC mice. We uncovered the organization of lymphoid cells into tertiary lymphoid structures (TLSs) adjacent to PanIN lesions. These TLSs lacked CD21+CD23+ B cells compared with more mature TLSs near the PDAC border. PanINs harbored mostly CD4+ T cells, with fewer Tregs and exhausted T cells than PDAC. Peritumoral space was enriched with naive CD4+ and central memory T cells. These observations highlight the opportunity to modulate the immune microenvironment in PanINs before immune exclusion and immunosuppression emerge during progression into PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6b6b93e0faedffa7b598bebfd58275c143f057" target='_blank'>
              Spatial proteomics and transcriptomics reveal early immune cell organization in pancreatic intraepithelial neoplasia
              </a>
            </td>
          <td>
            Melissa R Lyman, Jacob T. Mitchell, Sidharth Raghavan, Luciane T. Kagohara, Amanda L. Huff, Saurav D Haldar, Sarah M. Shin, Samantha Guinn, Benjamin Barrett, Gabriella Longway, Alexei Hernandez, Erin M Coyne, Xuan Yuan, Lalitya Andaloori, Jiaying Lai, Yun Zhou Liu, Rachel Karchin, Anuj Gupta, A. Kiemen, André Forjaz, Denis Wirtz, Pei-Hsun Wu, Atul Deshpande, Jae W. Lee, T. Armstrong, Nilofer S. Azad, Jacquelyn W. Zimmerman, Laura D. Wood, Robert A Anders, Elizabeth D. Thompson, Elizabeth M. Jaffee, E. Fertig, W. Ho, N. Zaidi
          </td>
          <td>2025-06-26</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91860355badc3254098a406329f26559d939957e" target='_blank'>
              Optimised dissociation and multimodal profiling of prostate cancer stroma reveal fibromuscular cell heterogeneity with clinical correlates
              </a>
            </td>
          <td>
            E. Damisch, Elena Brunner, Lukas Nommensen, Lucy Neumann, Georgios Fotakis, Z. Trajanoski, S. Sopper, Georg Schäfer, M. Puhr, Isabel Heidegger, Marianna Kruithof-de Julio, Natalie Sampson
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a dense extracellular matrix (ECM) that sustains an immunosuppressive tumor microenvironment (TME). While this protective niche has been described, the molecular determinants orchestrating its formation and dictating its immune interactions are not well defined. Using Perturb-map, we determine how dozens of different gene perturbations shape the growth and cellular environments of PDAC clones through space and time. Our study reveals dynamic, gene-specific adaptations of immune neighborhoods during clonal selection. We identified Serpinb2 (PAI2) and Serpine1 (PAI1) as key cancer-derived mediators of TME remodeling and immune evasion. These factors promote the deposition of a fibrin-rich ECM that shapes immune cell composition, locally retains and polarizes immunosuppressive macrophages and excludes cytotoxic T cells. Deletion of either Serpinb2 or Serpine1 greatly enhanced tumor response to anti-PD1 immunotherapy in an aggressive PDAC model. Transcriptomic analysis further linked their expression to distinct PDAC subtypes and poor patient survival. Our findings demonstrate that Serpinb2 and Serpine1 establish a permissive niche for tumor progression and show how PDAC cells exploit components of the fibrinolysis pathway to remodel the ECM, alter macrophage composition, and protect themselves from immune editing, ultimately reinforcing the role of extracellular factors in shaping an immune-privileged tumor niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35c21056b3d1d6cc6bc17075f4a894aa4c5791ca" target='_blank'>
              Plasminogen activator inhibitors orchestrate the immunosuppressive tumor microenvironment in pancreatic cancer
              </a>
            </td>
          <td>
            Chiara Falcomatà, Sebastian R. Nielsen, Maximilian M. Schaefer, Bhavya Singh, Alexander Tepper, Divya Chhamalwan, Hunter T. Potak, Maxime Dhainaut, G. Mollaoglu, Matthew D. Park, Miriam Merad, Alessia Baccarini, Brian D. Brown
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/963ead743bf88aafb98699a954ce89dbd9be99ad" target='_blank'>
              Decoding Lymphangioleiomyomatosis (LAM) Niche Environment via Integrative Analysis of Single Cell Multiomics and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ken Chen, Shuyang Zhao, M. Guo, Hasan Al Reza, Andrew Wagner, Adnan Cihan Cakar, Cheng Jiang, Erik Zhang, Jenna Green, Emily Martin, Kathryn A. Wikenheiser-Brokamp, A. Perl, Debora Sinner, Jane J. Yu, Yan Xu
          </td>
          <td>2025-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Tumor relapse remains a significant obstacle to successful therapy. Preclinical animal models that accurately reflect tumor relapse in patients are urgently needed. Here, we employed a dual recombinase-mediated genetic system to genetically trace and ablate proliferating cells in a polyomavirus middle T antigen (PyMT)-induced spontaneous murine breast cancer model. This system enabled the acute ablation of cells that had undergone proliferation within a defined time window, resulting in a drastic tumor shrinkage, followed by a gradual tumor relapse due to the presence of residual low-cycling cells. We then applied single-cell RNA sequencing (scRNA-seq) to unbiasedly compare the tumor ecosystems of the primary and relapsed PyMT tumors. Compared with the primary tumors, the relapsed tumors exhibited a higher proportion of cancer stem cells and pro-tumor γδ T cells, as well as co-expression of Spp1 and Vegfa in multiple myeloid cell populations – features that predict poor therapeutic response and unfavorable outcomes in human breast cancer patients. Collectively, this proliferation tracing and ablation model emulates chemotherapies that preferentially eliminate proliferating cancer cells, serving as a robust tool and a valuable resource for testing novel therapeutic strategies in relapsed tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95fdfebff6c6c3226be581651d876e8c2d584753" target='_blank'>
              Modeling tumor relapse using proliferation tracing and ablation transgenic mouse
              </a>
            </td>
          <td>
            Chuang Zhao, Xin-nan Zheng, Han-Ying Huang, Lin Tian
          </td>
          <td>2025-07-17</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47dca4509610b2f00715550942002b3b04fd28df" target='_blank'>
              Tracing colorectal malignancy transformation from cell to tissue scale
              </a>
            </td>
          <td>
            Helena L. Crowell, Irene Ruano, Zedong Hu, Yourae Hong, G. Caratù, Hubert Piessevaux, Ashley Heck, Rachel Liu, Max Walter, Megan Vandenberg, Kimberly Young, Dan McGuire, Evelyn Metzger, Margaret L Hoang, Joseph M. Beechem, Sabine Tejpar, A. Pascual-Reguant, Holger Heyn
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Single-cell RNA-sequencing (scRNA-seq) coupled with robust computational analysis facilitates the characterization of phenotypic heterogeneity within tumors. Current scRNA-seq analysis pipelines are capable of identifying a myriad of malignant and non-malignant cell subtypes from single-cell profiling of tumors. However, given the extent of intra-tumoral heterogeneity, it is challenging to assess the risk associated with individual cell subpopulations, primarily due to the complexity of the cancer phenotype space and the lack of clinical annotations associated with tumor scRNA-seq studies. To this end, we introduce SCellBOW, a scRNA-seq analysis framework inspired by document embedding techniques from the domain of Natural Language Processing (NLP). SCellBOW is a novel computational approach that facilitates effective identification and high-quality visualization of single-cell subpopulations. We compared SCellBOW with existing best practice methods for its ability to precisely represent phenotypically divergent cell types across multiple scRNA-seq datasets, including our in-house generated human splenocyte and matched peripheral blood mononuclear cell (PBMC) dataset. For tumor cells, SCellBOW estimates the relative risk associated with each cluster and stratifies them based on their aggressiveness. This is achieved by simulating how the presence or absence of a specific cell subpopulation influences disease prognosis. Using SCellBOW, we identified a hitherto unknown and pervasive AR−/NElow (androgen-receptor-negative, neuroendocrine-low) malignant subpopulation in metastatic prostate cancer with conspicuously high aggressiveness. Overall, the risk-stratification capabilities of SCellBOW hold promise for formulating tailored therapeutic interventions by identifying clinically relevant tumor subpopulations and their impact on prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7e092ade35e70219652d97def0ea91976347a8d" target='_blank'>
              Artificial intelligence driven tumor risk stratification from single-cell transcriptomics using phenotype algebra
              </a>
            </td>
          <td>
            Namrata Bhattacharya, A. Rockstroh, S. Deshpande, S. Thomas, Anunay Yadav, Chitrita Goswami, Smriti Chawla, Pierre Solomon, Cynthia Fourgeux, Gaurav Ahuja, Brett Hollier, Himanshu Kumar, A. Roquilly, Jérémie Poschmann, M. Lehman, Colleen C. Nelson, Debarka Sengupta
          </td>
          <td>2025-06-13</td>
          <td>eLife</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a critical focus for biomarker discovery and therapeutic targeting in cancer. However, widespread clinical adoption of TME profiling is hindered by the high cost and technical complexity of current platforms such as spatial transcriptomics and proteomics. Artificial Intelligence (AI)-based analysis of the TME from routine Hematoxylin & Eosin (H&E)-stained pathology slides presents a promising alternative. Yet, most existing deep learning approaches depend on extensive high-quality single-cell or patch-level annotations, which are labor-intensive and costly to generate. To address these limitations, we previously introduced HistoTME, a weakly supervised deep learning framework that predicts the activity of cell type-specific transcriptomic signatures directly from whole slide H&E images of non-small cell lung cancer. This enables rapid, high throughput analysis of the TME composition from whole slide H&E images (WSI) without the need for segmenting and classifying individual cells. In this work, we present HistoTME-v2, a pan-cancer extension of HistoTME, applied across 25 solid tumor types, substantially broadening the scope of prior efforts. HistoTME-v2 demonstrates high accuracy for predicting cell type-specific transcriptomic signature activity from H&E images, achieving a median Pearson correlation of 0.61 with ground truth measurements in internal cross- validation on The Cancer Genome Atlas (TCGA), encompassing 7,586 WSIs, 6,901 patients, and 24 cancer types, and a median Pearson correlation of 0.53 on external validation datasets spanning 5,657 WSIs, 1,775 patients and 9 cancer types. Furthermore, HistoTME- v2 resolves the spatial distribution of key immune and stromal cell types, exhibiting strong spatial concordance with single-cell measurements derived from multiplex imaging (CODEX, IHC) as well as Visium spatial transcriptomics, spanning 259 WSI, 154 patients, and 7 cancer types. Overall, across both bulk and spatial settings, HistoTME-v2 significantly outperforms baselines, positioning it as a robust, interpretable and cost-efficient tool for TME profiling and advancing the integration of spatial biology into routine pathology workflows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3867ee374678a732e8d29ff1143462f5a27ee092" target='_blank'>
              Towards interpretable molecular and spatial analysis of the tumor microenvironment from digital histopathology images with HistoTME-v2
              </a>
            </td>
          <td>
            Sushant Patkar, Timothy R. Rosean, Palak Patel, Stephanie A. Harmon, Peter L. Choyke, Tamara Jamaspishvili, B. Turkbey
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="While pathology foundation models have transformed cancer image analysis, they often lack integration with molecular data at single-cell resolution, limiting their utility for precision oncology. Here, we present PAST, a pan-cancer single-cell foundation model trained on 20 million paired histopathology images and single-cell transcriptomes spanning multiple tumor types and tissue contexts. By jointly encoding cellular morphology and gene expression, PAST learns unified cross-modal representations that capture both spatial and molecular heterogeneity at the cellular level. This approach enables accurate prediction of single-cell gene expression, virtual molecular staining, and multimodal survival analysis directly from routine pathology slides. Across diverse cancers and downstream tasks, PAST consistently exceeds the performance of existing approaches, demonstrating robust generalizability and scalability. Our work establishes a new paradigm for pathology foundation models, providing a versatile tool for high-resolution spatial omics, mechanistic discovery, and precision cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9399a2bf1b84832c86d75a49ae3c9a7107309283" target='_blank'>
              PAST: A multimodal single-cell foundation model for histopathology and spatial transcriptomics in cancer
              </a>
            </td>
          <td>
            Changchun Yang, Haoyang Li, Yushuai Wu, Yilan Zhang, Yifeng Jiao, Yu Zhang, Rihan Huang, Yuan Cheng, Yuan Qi, Xin Guo, Xin Gao
          </td>
          <td>2025-07-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a2fe9a625c4ba7b1dd23966948293739db34623" target='_blank'>
              In Situ Inference of Copy Number Variations in Image-Based Spatial Transcriptomics
              </a>
            </td>
          <td>
            Augusta Jensen, Helena L. Crowell, Anna Pascual Reguant, Irene Ruano, Sabine Tejpar, Holger Heyn, Mats Nilsson, Sergio Marco Salas
          </td>
          <td>2025-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary The tumor microenvironment (TME) is known to influence both disease progression and treatment outcomes in cancer. In this study, we analyzed high-dimensional imaging mass cytometry data from lung adenocarcinoma and triple-negative breast cancer to examine how the spatial positioning of immune cells relates to clinical outcomes. We developed a relative distance (RD) score that quantifies the proximity of immune cell pairs to cancer cells. This metric showed stronger associations with patient survival and treatment response than conventional features such as cell fraction or density. Our results suggest that spatial immune features may serve as robust biomarkers for patient stratification and personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1da098f6c41405a345327975831d3ae20b4e8e98" target='_blank'>
              Spatial Proximity of Immune Cell Pairs to Cancer Cells in the Tumor Microenvironment as Biomarkers for Patient Stratification
              </a>
            </td>
          <td>
            Jian-Rong Li, Xingxin Pan, Yupei Lin, Yanding Zhao, Yanhong Liu, Yong Li, Christopher I Amos, Chao Cheng
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f703b1371a985c2aa75a6f16b3b87971f363e77b" target='_blank'>
              Spatial analysis reveals a novel inflammatory tumor transition state which promotes a macrophage-driven induction of sarcomatoid renal cell carcinoma
              </a>
            </td>
          <td>
            Allison M. May, Suguru Kadomoto, Claire Williams, Alex C. Soupir, Stephanie The, J. McGue, Tyler Robinson, G. Shelley, Mitchell T Hayes, Brooke L. Fridley, Jodi A. Balasi, Paolo M. Ramos Echevarria, J. Dhillon, S. Nallandhighal, Satwik Acharyya, Liying Chen, Jessica Aldous, Nathan Schurman, Veera Baladandayuthapani, Timothy L. Frankel, S. Salami, Brandon J. Manley, R. Mehra, Aaron M. Udager, Evan T. Keller
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6bec286991f2affb3f9522a83843f9469ba7b89" target='_blank'>
              Developmental Plasticity and Stromal Co-option Shape a Pituitary Neuroendocrine Tumor Transcriptional Continuum
              </a>
            </td>
          <td>
            Robert C. Osorio, Jun Y. Oh, Jangham Jung, Alexander J. Ehrenberg, A. Saha, Meeki K. Lad, Harmon Khela, Nicole Brennick, P. Giannikopoulos, William W. Seeley, Lea T. Grinberg, Aaron Diaz, M. Aghi
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Innate immune cells constitute the majority of the tumor microenvironment (TME), where they mediate both natural anti-tumor immunity and immunotherapy responses. While single-cell T- and B-cell receptor sequencing has provided fundamental insights into the clonal dynamics of human adaptive immunity, the lack of appropriate tools has precluded similar analysis of innate immune cells. Here, we describe a method that leverages somatic mitochondrial DNA (mtDNA) mutations to reconstruct clonal lineage relationships between single cells across cell types in native human tissues. We jointly sequenced single-cell transposase-accessible chromatin and mtDNA to profile n =124,958 cells from matched tumor, non-involved lung tissue (NILT), and peripheral blood of early-stage non-small cell lung cancer (NSCLC) patients, as well as n =93,757 cells from matched tumor and peripheral blood of ovarian cancer patients. Single-cell concomitant profiling of lineage and cell states of thousands of immune cells resolved clonality across cell types, tissue sites, and malignancies. Clonal tracing of innate immune cells demonstrates that TME-resident myeloid subsets, including macrophages and type 3 dendritic cells (DC3), are clonally linked to both circulating and tissue-infiltrating monocytes. Further, we identify distinct DC-biased and macrophage-biased myeloid clones, enriched in the tumor and NILT, respectively, and find that their circulating monocyte precursors exhibit distinct epigenetic profiles, suggesting that myeloid differentiation fate may be predetermined before TME infiltration. These results delineate the clonal pathways of intratumoral myeloid cell recruitment and differentiation in human cancer and suggest that remodeling of the tumor myeloid compartment may be peripherally programmed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ba9df6c66740c3cece720fa35456bef19fb17f8" target='_blank'>
              Clonal lineage tracing of innate immune cells in human cancer
              </a>
            </td>
          <td>
            Vincent Liu, Katalin D. Sandor, Patrick K. Yan, Max Miao, Yajie Yin, Robert R. Stickels, , Kamir J. Hiam-Galvez, Jacob Gutierrez, Wenxi Zhang, Sairaj Sajjath, Raeline Valbuena, Steven Wang, Bence Dániel, Leif S. Ludwig, Brooke E Howitt, Caleb A. Lareau, Ansu Satpathy
          </td>
          <td>2025-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="ABSTRACT Background Cutaneous squamous cell carcinoma (cSCC) and basal cell carcinoma (BCC) are the most prevalent types of nonmelanoma skin cancer (NMSC) and exhibit significant inter‐ and intra‐tumor heterogeneity. cSCC has a higher metastatic potential than BCC, accompanied by a considerable mortality rate. However, the detailed mechanisms of tumor evolution in cSCC have not yet been described. Methods We performed single‐cell RNA sequencing (scRNA‐seq) and T cell receptor (TCR) clonal analysis of skin biopsies from five BCCs, three squamous cell carcinomas in situ (SCCIS), and two invasive squamous cell carcinomas (SCC). Independent SCC specimens were used for spatial transcriptomic (ST) analysis using GeoMx Digital Spatial Profiler (DSP). Result Using scRNA‐seq, we analyzed a total of 117,663 cells. We distinguished cancer cells using copy number variation and identified SCC‐specific genes that potentially contribute to tumor progression. Analysis of tumor clones revealed SCC‐specific COL6A1+/ITGA5+ carcinoma cells which produce CXCL16. We also annotated CXCR6+ regulatory T cells (Tregs) which potentially move toward the tumor site by CXCL16, shaping the immunosuppressive TME. ST analysis supported these clones were located at the invasion site of SCC. Conclusion We suggest COL6A1 and ITGA5 promote the invasive and metastatic property of SCC. We also uncovered how SCC recruits Tregs via the CXCL16/CXCR6 axis to create a TME favorable for its survival. These molecules can be used as potential therapeutic targets for treatment of SCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10a0e1cd92069735154742961ad751b08397d911" target='_blank'>
              CXCL16 Producing Tumor Clones Are Shaping Immunosuppressive Microenvironment in Squamous Cell Carcinoma via CXCR6 Regulatory T Cell
              </a>
            </td>
          <td>
            H. Choi, Sunyoung Jung, Ki-Myo Kim, Mihyun Lee, Jun Ho Park, Sanha Hwang, Seung-Min Cha, Jade N Young, D. Poplausky, Hyunsung Nam, Nicholas Gulati, Chung-Gyu Park, Hyun Je Kim, Ji-Ung Park
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90f49812ba9bf54efb86922eb7c7255926faf91" target='_blank'>
              Mapping and reprogramming microenvironment-induced cell states in human disease using generative AI
              </a>
            </td>
          <td>
            Amir Akbarnejad, Lloyd Steele, D. Jafree, Sebastian Birk, M. Sallese, K. Rademaker, Adam Boxall, Ben Rumney, Catherine Tudor, Minal Patel, M. Prete, Stanislaw Makarchuk, Tong Li, Heather Stanley, A. Foster, Kenny Roberts, Andrew L. Trinh, C. E. Villa, G. Testa, S. Mahil, Arash Mehrjou, Catherine Smith, Sattar Vakili, M. Clatworthy, Thomas J. Mitchell, O. Bayraktar, M. Haniffa, M. Lotfollahi
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains among the most lethal cancers, with metastasis as the primary driver of mortality. While metastatic mechanisms are shared across malignancies, PDAC metastasis poses unique therapeutic challenges due to the presence of extensive tumor heterogeneity, desmoplasia, and immunosuppression — features that enable diverse migratory behaviors and therapeutic resistance. Recent advances have shown that metastatic progression in PDAC emerges from dynamic interactions between tumor cell–intrinsic and microenvironmental factors, each adapting to evolving stressors throughout the metastatic cascade. In the primary tumor, genomic instability and epigenetic reprogramming generate subclones with heightened invasive potential, while dense stromal reactions and myeloid-dominated immune suppression facilitate escape. During circulation, PDAC cells employ distinctive survival strategies through homotypic clustering and heterotypic interactions with blood components. At distant sites, PDAC cells adapt to organ-specific microenvironments through context-dependent metabolic and immune modulation, resulting in phenotypes that diverge from the primary tumor. In this Review, we examine how tumor-stroma crosstalk mechanisms shape metastatic progression in PDAC, provide a framework for understanding why conventional therapies often fail against metastatic disease, and highlight emerging opportunities for stage- and site-specific therapeutic interventions that target these unique adaptations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48f2d47124253bedbc482deeccc073d8dd0b354b" target='_blank'>
              Metastatic heterogeneity in pancreatic cancer: mechanisms and opportunities for targeted intervention
              </a>
            </td>
          <td>
            R. Maddipati
          </td>
          <td>2025-07-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Cancer therapy resistance (CTR) remains a significant challenge in oncology. Traditional methods like imaging, liquid biopsies and conventional omics analyses provide valuable insights, but lack the spatial resolution to fully characterise heterogeneity of tumour and the tumour microenvironment (TME). Recent advancements in spatial omics technologies offer unprecedented insights into the spatial organisation of tumours and TME. In this review, we summarise current methodologies for CTR research and highlight how spatial omics technologies and computational methods are revolutionising our understanding of CTR mechanisms. We also summarise recent studies leveraging spatial omics to uncover novel insights into CTR across various cancer types and therapies and discuss future opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baf8daf1a76c566edbbdf0a752cb4ca39d6eea13" target='_blank'>
              Cancer therapy resistance from a spatial‐omics perspective
              </a>
            </td>
          <td>
            Yinghao Zhang, Cheng Yang, Xi Chen, Liang Wu, Zhiyuan Yuan, Fan Zhang, Bin-Zhi Qian
          </td>
          <td>2025-07-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background: Lung adenocarcinoma (LUAD) remains a leading cause of cancer-related mortality despite advances in treatments, necessitating more effective therapeutic strategies. Single-cell RNA sequencing (scRNA-seq) technology has revolutionized our ability to dissect the cellular complexity of cancers, which is often obscured in conventional bulk transcriptomic experiments. Methods: In this study, we performed an integrative analysis of scRNA-seq data from multiple LUAD patient cohorts to investigate cell-type-specific transcriptomic changes across disease stages. Clustering, lineage trajectory analysis, and transcriptional regulatory network reconstruction were employed to identify stage-specific gene markers and their upstream regulators. Additionally, we constructed intercellular communication networks to evaluate signaling changes within the tumor microenvironment (TME) during LUAD progression. Results: Our analysis revealed that epithelial cells from stage IV tumors exhibited a distinct transcriptional profile compared to earlier stages, a separation not observed in immune or stromal cell populations. We identified a panel of gene markers that differentiated epithelial cells across disease stages and effectively stratified patients into subgroups with distinct survival outcomes and TME compositions. Regulatory network analysis uncovered key transcription factors, including ATF3, ATF4, HSF1, KLF4, and NFIC, as potential upstream regulators of these stage-specific genes. Moreover, cell–cell communication analysis revealed a significant increase in signaling originating from epithelial cells and a concomitant decrease in immune-derived signals in late-stage LUAD. We identified several signaling pathways enriched in stage-specific crosstalk, including Wnt, PTN, and PDGF pathways, which may play critical roles in LUAD progression. Conclusions: This study provides a comprehensive single-cell resolution map of LUAD progression, highlighting epithelial-driven regulatory programs and dynamic intercellular communication within the TME. Our findings uncover novel molecular markers and regulatory mechanisms with potential prognostic and therapeutic value for more precise treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ecbc245a8fbf12aaba1f14070430ba2f845cb23" target='_blank'>
              Single-Cell Transcriptomic Analysis Unveils Key Regulators and Signaling Pathways in Lung Adenocarcinoma Progression
              </a>
            </td>
          <td>
            Jialu Ma, Caleb McQuay, John Talburt, Amit K. Tiwari, Mary Qu Yang
          </td>
          <td>2025-06-30</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The Spatial Atlas of Human Anatomy (SAHA) represents the first multimodal, subcellular-resolution reference of healthy adult human tissues across multiple organ systems. Integrating spatial transcriptomics, proteomics, and histological features across over 15 million cells from more than 100 donors, SAHA maps conserved and organ-specific cellular niches in gastrointestinal and immune tissues. High-resolution profiling using CosMx SMI, 10x Xenium, RNAscope, GeoMx DSP, and single-nucleus RNA-seq reveals spatially organized cell states, rare adaptive immune populations, and tissue-specific cell-cell interactions. Comparative analyses with colorectal cancer and inflammatory bowel disease demonstrate the power of SAHA to detect disease-associated spatial disruptions, including crypt dedifferentiation, perineural invasion, and therapy-resistant immune remodeling. All data are openly accessible through a FAIR-compliant interactive portal to support exploration, benchmarking, and machine learning model training. Through SAHA, we provide a foundational framework for spatial diagnostics and next-generation precision medicine grounded in a comprehensive human tissue atlas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a6700430cd96bd0ff586b384b4fa3931ebab91" target='_blank'>
              The Spatial Atlas of Human Anatomy (SAHA): A Multimodal Subcellular-Resolution Reference Across Human Organs
              </a>
            </td>
          <td>
            Jiwoon Park, Roberto De Gregorio, Erika Hissong, Elif Ozcelik, Nicholas Bartelo, F. Dezem, Luke Zhang, Maycon Marção, Hannah Chasteen, Yimin Zheng, Ernesto Abila, Junbum Kim, J. Proszynski, Akua A. Agyemang, Mohith Reddy Arikatla, Evelyn Metzger, Stefan Rogers, P. Divakar, P. Dulai, Jason Reeves, Yan Liang, Liuliu Pan, Sayani Bhattacharjee, Kimberly Young, Ashley Heck, Mithra Korukonda, Dan McGuire, Lidan Wu, Aster Wardhani, Joseph M. Beechem, George M. Church, Steven M Lipkin, Sanjay Patel, F. Socciarelli, Sébastien Monette, Brian Robinson, M. Loda, O. Elemento, Luciano G. Martelotto, Jasmine T. Plummer, André F. Rendeiro, Alicia Alonso, R. Schwartz, S. Houlihan, Christopher E. Mason
          </td>
          <td>2025-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Lymph node metastasis markedly worsens prognosis in lung adenocarcinoma (LUAD); however, the evolutionary dynamics and regulatory mechanisms underlying the heterogeneity of malignant epithelial cells during this process remain poorly understood and warrant comprehensive investigation. Methods We performed a comprehensive single-cell transcriptomic analysis of epithelial cells from 18 samples comprising normal lung tissue and lymph node metastases. Malignant epithelial cells were identified via inferred copy number variation (CNV) profiles. Key malignant subpopulations were further characterized through trajectory inference, cell–cell communication mapping, gene set variation analysis (GSVA), and reconstruction of transcription factor regulatory networks. To assess clinical relevance, we developed and validated a prognostic model—termed the EAS score—based on the transcriptional signatures of malignant epithelial subsets, using integrated data from multiple TCGA and GEO cohorts. The functional role of the hub gene SELENBP1 was experimentally validated through quantitative PCR (qPCR), Western blotting, immunohistochemistry (IHC), Transwell migration assays, colony formation assays, flow cytometry, ROS quantification, and subcutaneous tumorigenesis assays in vivo. Results Single-cell transcriptomic analysis identified four distinct malignant epithelial subtypes (Clusters 0–3), each characterized by unique patterns of CNV. Leveraging these defined cellular subpopulations, we constructed a highly accurate model for prognostication in LUAD, enabling reliable classification of patients based on clinical outcomes. Through detailed comparisons between groups with divergent prognostic risks, the study revealed notable differences across the tumor microenvironment (TME), including alterations in pathway activity, gene enrichment distributions, mutation profiles, and anticipated responses to immune checkpoint blockade. In addition, functional validation experiments confirmed that SELENBP1 plays a tumor-suppressive role, further supporting its relevance as a potential intervention target in LUAD. Conclusion This research provides insights into the evolutionary complexity and heterogeneity of malignant epithelial populations in lymph node metastatic sites of LUAD. It also presents a scoring system based on prognostic indicators, which serves as a reliable tool for forecasting patient survival outcomes. Moreover, the discovery of SELENBP1 as a candidate tumor suppressor emphasizes its importance in guiding both clinical risk categorization and the design of personalized treatment strategies for individuals classified as high-risk LUAD cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6a2796c042204c38d000db3a271f2db16f9cb8" target='_blank'>
              Single-cell transcriptomic profiling reveals the heterogeneity of epithelial cells in lung adenocarcinoma lymph node metastasis and develops a prognostic signature
              </a>
            </td>
          <td>
            Qiuqiao Mu, Han Zhang, Ying Shi, Mengli Xue, Jingxian Wang, Yun Ding, Lin Tan, Hui Yuan, Xin Li, Daqiang Sun
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High-risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single-cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this, we profiled untreated and chemotherapy-treated high-risk neuroblastomas from archived, formalin-fixed, paraffin-embedded (FFPE) tissues from two patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p-gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical-like cells expressed several oncogenic ligand-encoding genes (e.g. ALKAL2 and NRTN), which were predicted to communicate with neighboring cancer cells that expressed the corresponding receptors (e.g. ALK, RET). Several of these interactions were further validated experimentally and were shown to be clinically relevant. Collectively, our spatial analysis identifies multiple previously unrecognized signaling axes that may offer novel therapeutic options in neuroblastoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dce7c8d7d28d1cfa3f77ec4f74044ee737c8a1da" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment in archived FFPE samples unveils novel paracrine interactions.
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlsson, A. Claeys, E. Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2025-08-08</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cce1fa72c7e2d6491d95989929c304afa244428" target='_blank'>
              Epithelial–mesenchymal cell state heterogeneity predetermines differential phospho-signaling responses to EGF stimulation
              </a>
            </td>
          <td>
            F. Kohane, Chantelle Johnstone, Daniel P Neumann, Ihuan Gunawan, Tim Huang, Fatemeh Vafaee, Christine L Chaffer, John G Lock
          </td>
          <td>2025-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa179eb3cd2fd6b3471b1b5ec6d1e223709a7a51" target='_blank'>
              Predicting Spatial Transcriptomics from H&E Image by Pretrained Contrastive Alignment Learning
              </a>
            </td>
          <td>
            Jiawei Zou, Kai Xiao, Zexi Chen, Jiazheng Pei, Jing Xu, Tao Chen, Likun Hou, Chunyan Wu, Y. She, Zhiyuan Yuan, Luonan Chen
          </td>
          <td>2025-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631d6fa6d4de4b757d78ca3c32e8b7a61d4aff26" target='_blank'>
              Single cell functional immunogenomics of the fallopian tube reveals a precursor immune surveillance network for ovarian cancer prevention
              </a>
            </td>
          <td>
            Luyao Wang, Breeshey Roskams-Hieter, Nosheen Hussain, Joel Nulsen, Aneesh Aggarwal, May Sallam, Lili Wang, Lena Rai, Amro Ahmed-Ebbiary, Aws Al-Deka, Takashi Takeda, Mara Artibani, H. Soleymani majd, Jason Yap, Christopher Yau, Nancy Zaarour, A. Ahmed
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 The tumor microenvironment (TME) actively contribute to tumor development and treatment response. The interplay between tumor cells, immune cells, fibroblasts and blood vessels contribute to immune escape and drug resistance. Prior to treatment, higher tumor infiltrating lymphocytes correlate with better survival, while greater stromal content is linked to poor survival. Studying the composition and dynamics of the TME is essential for improving patient stratification, however a scalable tool for addressing this question is still lacking. Spatially resolved omics technologies allow for charting tissue architecture at the individual cell level, though large-scale studies remain challenging due to high expenses. In contrast, hematoxylin and eosin (H&E) slides are a cost-effective modality that provide rich morphological information for studying spatial biology. However, their use relies on pathologist interpretation. A key area of research in digital pathology has been automating cell (type) identification, predicting nuclei location and cell type in H&E slides. Existing deep learning models are limited by the quantity and diversity of training data, which requires pathologists to carefully annotate the location and identity of large volume of cells. To date, the largest dataset comprises approximately 200,000 cells, annotated with four cell types across 19 cancer types. We propose a novel approach of automated “molecular annotation”, where cell types and location on H&E slides are annotated with the aid of spatial proteomics, in place of pathologist annotation. Specifically, samples were profiled with both modalities. From the spatial omics modality, pixels were first segmented into cells, followed by cell clustering and cluster annotation based on molecular features of the cells. The location and identity of all cells on the tissue were then identified. These information were subsequently transferred to the H&E image by alignment at single-cell resolution, forming a dataset annotated with molecular ground truth. With a spatial omics dataset of two spatial proteomics slides from colorectal cancer patients, we obtained 160,000 annotated cells including 50,000 immune cells, 25,000 tumor cells, 12,600 stroma, 7,900 endothelial cells, among others. The size of this dataset is close to the largest annotated dataset publicly available to date. We then use this dataset to benchmark existing state-of-the-art deep-learning based cell type predictions models, as well as to fine-tune existing models for predicting cells in the colorectal tumor microenvironment. This proof-of-concept study aims to demonstrate the feasibility of molecular annotation approach. By including more spatial omics data, this approach can boost the performance of existing pre-trained models and enhance generalizability to specific tumor types. It opens up the opportunity to harness millions of cells for deep learning models to predict cell types on H&E slides, make AI models a cost-effective option for studying the TME.



 Siao-Han Wong, Benedikt Brors, Sonja Loges. Empowering AI-driven prediction of the tumor microenvironment from histopathology images via molecular annotation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B053.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62b30dcab5fc423419dfc7696f7849978493fc18" target='_blank'>
              Abstract B053: Empowering AI-driven prediction of the tumor microenvironment from histopathology images via molecular annotation
              </a>
            </td>
          <td>
            Siao-Han Wong, B. Brors, Sonja Loges
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Given the graft-versus-leukemia effect observed with allogeneic hematopoietic stem cell transplantation in refractory or relapsed acute myeloid leukemia (AML), immunotherapies have been explored in nontransplant settings. We applied a multiomic approach to examine bone marrow interactions in patients with AML treated with pembrolizumab and decitabine. Using extensively trained nuclear and membrane segmentation models, we achieved precise transcript assignment and deep learning–based image analysis. To address read-depth limitations, we integrated single-cell RNA sequencing with single-cell spatial transcriptomics from the same sample. Quantifying cell-cell distances at the edge level enabled more accurate tumor microenvironment analysis, revealing global and local immune cell enrichment near leukemia cells postpembrolizumab treatment, potentially linked to clinical response. Furthermore, ligand-receptor analysis indicated potential alterations in specific signaling pathways between leukemia and immune cells following immunotherapy treatment. These findings provide insights into immune interactions in AML and may inform therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0066efa7bc3f52352d23a57bb535219a1adb0cf1" target='_blank'>
              Single-cell spatial transcriptomics reveals immunotherapy-driven bone marrow niche remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R Herzog, Abigail Wong-Rolle, Laura W. Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann F Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine R Calvo, Christopher S. Hourigan, Kasper D Hansen, Chen Zhao
          </td>
          <td>2025-07-09</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the third most common cancer in the world. Within the tumor microenvironment (TME), cancer cells are surrounded by diverse cell types including cancer-associated fibroblasts (CAFs) and immune cells. These neighboring cells can have a profound effect on prognosis and disease progression by providing local signals that impact tumor growth and metastasis. Prior work in the lab has identified via single cell RNA sequencing novel subtypes of cancer epithelial cells and enrichment of specific CAFs within the CRC TME. However, the locations and signaling interactions of these cells within the tumors remain poorly understood due to the loss of spatial information. In this study, we adopt a multi-facet approach to try to understand the CRC TME by utilizing a combination of single cell RNA sequencing and spatial omics technology. We first comprehensively characterized all the major cell types present in the CRC TME using single cell RNA sequencing, and with the knowledge gained, we set out to investigate the spatial localization of these cell types of interests and their interactions within the tumor architecture using the 10x Xenium Analyzer and merFISH. From our data, we identified subtypes of tumor epithelial cells that are uniquely enriched in CRC patients and observed distinct spatial organization of these cells within colonic glands across patient subgroups. In addition, we observed an enrichment of CAFs and immune cell populations at the tumor invasive margin, often accompanying regions of tumor budding. Further study of these tumor epithelial, CAF and immune subtypes and their spatial interactions may reveal new insights into CRC tumor biology and assist in the development of targeted therapies for CRC.



 Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, Nirmala Arul Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar. Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2896c139d399fbdf03bf5e11753408c9bfebdc83" target='_blank'>
              Abstract P19: Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics
              </a>
            </td>
          <td>
            Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, N. A. Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Individual cells within a given population exhibit striking variability in viral susceptibility, but it remains unknown whether this heterogeneity reflects memories encoded into the cellular lineage or true probabilistic variability. We used multi-color lineage tracing in a human primary organotypic skin model to reveal that viral resistance is encoded within specific cellular lineages. These lineages create distinct boundaries that block viral spread. Our lineage analyses in vitro confirmed that viral susceptibility exhibits strong heritability across cell generations, with siblings and cousins displaying remarkably similar infection outcomes. ATAC and proteomics profiling of resistant and susceptible clones revealed distinct epigenomic and proteomic states, with the transcription factor AP-1 emerging as a potential central regulator of lineage-encoded viral resistance. Inducing AP-1 activity with PMA rendered cells resistant to viral infection, suggesting a causative role in mediating resistance memory. Our findings demonstrate that antiviral resistance in human skin cells is encoded within cellular lineages and preserved through cell divisions, revealing how cell memory may shape infection dynamics and viral containment in tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb2bae8a5c0fba39a07b5a8201b6d06e217381d" target='_blank'>
              Lineage memory shapes viral resistance barriers in human skin
              </a>
            </td>
          <td>
            Laura C. Van Eyndhoven, Grant Kinsler, Jingchao Zhang, Aoife O’Farrell, Rim Abderrahim, Apurv Srivastav, Catherine G. Triandafillou, Todd M. Greco, Kenneth S. Zaret, I. Cristea, Megan H. Orzalli, N. Drayman, Abhyudai Singh, Arjun Raj
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Introduction Spatial transcriptomic analysis has proposed valuable insights into the behavior of tongue cancer. However, the specific cell types involved in chemically-induced carcinogenesis and the process of tumor development remain elusive. Methods We leveraged artificial intelligence (AI) algorithms and spatial transcriptomic sequencing to meticulously characterize the spatial and temporal evolution of 4-nitroquinoline-1-oxide (4NQO)-induced tongue carcinogenesis and intratumor heterogeneity. Results An AI classifier effectively categorized dysplastic tongue tissue into 13 distinct groups. Spatial transcriptomics identified 13 corresponding cellular subgroups with unique features within the lesion. Both methods successfully distinguished subtle muscle phenotype and genetic lineage variations induced by 4NQO, despite limited morphological differences. Evolutionary tree analysis revealed the dynamic appearance and disappearance of functionally and genetically diverse cell subgroups during the progression from epithelial dysplasia to in situ carcinoma and invasive cancer. Key findings include the identification of specific switch genes associated with tumor invasion and the revelation of significant intratumor heterogeneity. Discussion This spatial transcriptomic analysis of 4NQO-induced tongue cancer provides a detailed characterization of tumor evolution and heterogeneity. It elucidates critical aspects of tongue cancer cell behavior and identifies potential therapeutic targets (switch genes). These findings offer novel insights for improving the diagnosis and treatment of tongue cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679c71fb0ae1356ab18cbbcb3f0859a89370b05e" target='_blank'>
              Spatial transcriptomic analysis of 4NQO-induced tongue cancer revealed cellular lineage diversity and evolutionary trajectory
              </a>
            </td>
          <td>
            Feng Liu, Xiaojun Wu, Shaoqing Yu, Deqiang Cheng, Jun Pan, Xiaodong Wang, Guanzhen Yu, Chaofu Li
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Integrating spatial transcriptomic data with immunofluorescence image data is challenging using existing tools due to their differences in spatial resolution. Immunofluorescence provides information about protein expression at the cellular or subcellular level, whereas spatial transcriptomic platforms typically rely on multicellular “spots” for RNA profiling. Our study coupled spatial transcriptomics of irradiated glioblastoma tissues with immunofluorescence for γH2AX, a marker of DNA damage within the nuclei of cells. We then compared gene expression in γH2AX-positive and negative regions within the tissue. There was significant interobserver variability in manual annotation of γH2AX positivity in multicellular spots by three different researchers (Kappa statistic = 0.345), despite all of them being familiar with γH2AX immunofluorescence and having predefined imaging parameters for annotation. This variability led to different researchers nominating different genes as being associated with DNA repair. To overcome this problem, we have developed a new tool using MATLAB. This tool performs “spot”-wise image analysis and uses researcher-defined parameters such as immunofluorescent marker intensity threshold and number of positive cells to annotate the “spots” as γH2AX positive or negative. The tissue with the most variability in manual annotation was annotated reproducibly by our MATLAB tool, leading to reproducible downstream analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36da1cf7d69ccada17068ecf953f32e30ee56bd" target='_blank'>
              A New Tool to Decrease Interobserver Variability in Biomarker Annotation in Solid Tumor Tissue for Spatial Transcriptomic Analysis
              </a>
            </td>
          <td>
            Sravya Palavalasa, Emily Baker, Jack Freeman, Aditri Gokul, Weihua Zhou, Dafydd Thomas, W. Al-Holou, Meredith A. Morgan, Theodore S. Lawrence, Daniel R Wahl
          </td>
          <td>2025-07-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Vestibular schwannoma (VS) is a benign tumor that can result in significant neurological and otological complications. The mechanisms underlying its development and spatial heterogeneity remain poorly understood. In this study, we analyzed single-cell RNA sequencing (scRNA-seq) data from three previously published vestibular schwannomas (VS1-3), along with spatial transcriptomics data from two additional specimens (VS_S1-2). Our results identified a VEGFA-enriched Schwann cell (SC) subtype in scRNA-seq data, which was validated by spatial transcriptomics. This subtype also exhibited a significant positive correlation with NOVhi SCs expression. These cells were centrally localized within tumor tissue. Furthermore, spatial analysis provided new insights into SC-stromal cell interactions, and we constructed a preliminary cellular atlas of VS tissues, enhancing our understanding of tumor growth dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7a7b00297ec42d2e5217a894e786882ec23b539" target='_blank'>
              Integrating single-cell and spatial transcriptomics reveals the cellular heterogeneity of vestibular schwannoma
              </a>
            </td>
          <td>
            Wenqi Dong, Yuchen Jin, Lingkang Dong, Yumeng Jiang, Zhuangzhuang Li, Maoxiang Xu, Jingjing Wang, Feng Liu, Dongzhen Yu
          </td>
          <td>2025-07-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background Lung adenocarcinoma (LUAD) exhibits significant cellular heterogeneity, yet the precise interactions between epithelial and stromal cells remain unclear. This study integrates single-cell and spatial transcriptomics to delineate tumor microenvironment dynamics, aiming to uncover key cellular subpopulations and their roles in LUAD progression. Methods We analyzed single-cell RNA sequencing (scRNA-seq) data from 21 LUAD patients and performed spatial transcriptomic deconvolution. Epithelial and fibroblast subpopulations were identified using Seurat and Harmony. Cell-cell communication was inferred via CellChat, while metabolic interactions were assessed using MEBOCOST. Copy number variation (CNV) analysis distinguished malignant cells, and trajectory inference mapped differentiation states. Spatial colocalization was examined via CellTrek. Prognostic signatures were derived from Cox regression, and a six-gene MCI score was validated using survival analysis. Results We identified eight epithelial (e.g., MUC21 + Epi, ASCL1 + Epi) and nine fibroblast subpopulations (e.g., Fb_IGFBP4, Fb_COL11A1), with tumor-enriched subsets showing elevated CNVs and metabolic crosstalk. Fb_IGFBP4 correlated with poor prognosis, while MUC21 + Epi exhibited amplified COL1A1/SDC4-mediated interactions with fibroblasts. Pathway analysis highlighted tumor-specific MK and collagen signaling between fibroblasts and epithelial cells, suggesting stromal-epithelial synergy drives progression. Spatial analysis revealed colocalization of epithelial and fibroblast subclusters in tumors, contrasting with normal tissue. The MCI score, derived from six genes (e.g., ADAM10, MARVELD1), independently predicted survival and stratified high-risk patients (AUC > 0.6). Conclusion This study identifies key stromal-epithelial subset interactions in LUAD, proposing prognostic biomarkers and therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1007/s00432-025-06250-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e782303b020428f38f4b87de805bc2e91d348a09" target='_blank'>
              Decoding epithelial–fibroblast interactions in lung adenocarcinoma through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Jiajin Yang, Qiuping Xu, Yanjun Lu
          </td>
          <td>2025-07-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatoblastoma is the most common pediatric liver cancer, with the fastest rising incidence among childhood malignancies. Early genomic studies revealed that hepatoblastoma has the lowest mutational burden of any human cancer, however, recent advances in single-cell RNA-seq, multiomics, spatial transcriptomics, and functional genomics screenings have revealed substantial complexity. Diverse cellular subpopulations, divergent WNT signaling, key developmental pathways, and intricate interactions between the tumor cells and tumor immune microenvironment (TME) collectively shape tumor heterogeneity, disease progression, therapeutic responses, and genetic dependencies. Mosaic embryonic loss of heterozygosity (LOH) at chromosome 11p15.5 may be a hepatoblastoma-initiating event, as clonal expansion of 11p15.5 LOH occurs in adjacent normal liver tissue. A cholangiocyte-like subpopulation expresses FGF19, in a SOX4-dependent, paracrine manner, to drive the proliferation of neighboring embryonal hepatoblastoma cells. WNT-signaling dependent MDK promotes the immunosuppressive TME, which impairs immune cell infiltration. The TME may also be driven by islands of erythroblasts, which influence treatment resistance. Plasticity driven by changes in chromatin accessibility enables differentiation transition between hepatocytic and liver progenitor cell types, which is associated with treatment resistance. Here, we review recent findings in pediatric hepatoblastoma cells, tumor-associated cell types, and genetic dependencies that will serve to advance hepatoblastoma therapy. Clinical trial number: Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02405-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361b9ebd3fa588c3f28e25ba36c91c45f4b07fb6" target='_blank'>
              From preneoplastic lesion to heterogenous tumor: recent insights into hepatoblastoma biology and therapeutic opportunities
              </a>
            </td>
          <td>
            Jun Yang, A. Davidoff, Andrew J. Murphy
          </td>
          <td>2025-07-19</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 The tumor microenvironment (TME) plays a pivotal role in shaping therapeutic outcomes in lung cancer, with cancer-associated fibroblasts (CAFs) emerging as key mediators of drug resistance. In situ sequencing enables spatial resolution of tumor architecture, offering insight into heterogeneity and the influence of CAFs on tumor cell biology. However, directly translating these microenvironmental effects on tumor cell behavior into drug response predictions remains a major challenge. In this work, we leveraged spatial transcriptomics and in situ single-cell RNA sequencing to predict spatially resolved drug responses in non-small cell lung cancer (NSCLC) patient samples and investigated the impact of fibroblast density on therapeutic efficacy. We adapted our recently published scIDUC modeling approach—which integrates bulk RNA-seq from cancer cell lines with single-cell RNA-seq data to predict drug responses—for use with in situ data from eight NSCLC patients. Our spatial analysis revealed that tumor cells in high fibroblast-density regions exhibited distinct predicted drug sensitivities compared to those in low-fibroblast contexts. Notably, lapatinib—a dual EGFR/HER2 inhibitor—was consistently predicted to be less effective in tumor cells residing within fibroblast-rich niches. These predictions were validated experimentally using a co-culture model of CALU-3 lung cancer cells and IMR-90 fibroblasts. CALU-3 cells grown with fibroblasts, either in direct contact or separated by a transwell, showed significantly increased resistance to lapatinib compared to monoculture conditions, confirming our computational findings. Our study highlights the role of CAF-rich microenvironments in therapeutic response and proposes a CAF-informed framework for refining therapeutic strategies in NSCLC. By integrating spatial transcriptomics with predictive modeling, we provide a pathway to directly link microenvironmental context with functional drug response in tumors.



 Robert F. Gruener, Lilin Wang, Adam Lee, Weijie Zhang, R. Stephanie Huang. Spatially resolved drug response predictions reveal fibroblast-associated resistances in in situ lung cancer samples [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd4918af0e969655d588afdfb7b613e252a4d08" target='_blank'>
              Abstract A023: Spatially resolved drug response predictions reveal fibroblast-associated resistances in in situ lung cancer samples
              </a>
            </td>
          <td>
            Robert F. Gruener, Lilin Wang, Adam Lee, Weijie Zhang, R. S. Huang
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinoma (cSCC), basal cell carcinoma (BCC), and melanoma, the three major types of skin cancer, account for over 70% of all cancer cases. Despite their prevalence, the skin cancer microenvironment remains poorly characterized, both in the outer skin layer where the cancer originates and at the deeper junctional and dermal layers into which it progresses. To address this, we integrated 12 complementary spatial single-cell technologies to construct orthogonally-validated cell signatures, spatial maps, and interactomes for cSCC, BCC, and melanoma. We comprehensively compared and integrated these spatial methods and provided practical guidelines on experimental design. Integrating four spatial transcriptomics platforms, we found keratinocyte cancer signatures, including six consistently validated gene markers. Spatial integration of transcriptomics, proteomics, and glycomics uncovered cancer communities enriched in melanocyte–fibroblast–T-cell colocalization with altered tyrosine and pyrimidine metabolism. Ligand-receptor analysis across >700 cell-type combinations and >1.5 million interactions highlighted key roles for CD44, integrins, and collagens, with CD44-FGF2 emerging as a potential therapeutic target. We consistently found differential interactions of melanocytes with fibroblasts and T-cells. We validated these interactions using Opal Polaris, RNAScope, and Proximal Ligation Assay. To integrate population-scale data, genetic association mapping in >500,000 individuals suggested SNPs enriched for spatial domains containing melanocytes, dysplastic keratinocytes, and fibroblasts, shedding light on functional mechanisms linking genetic heritability to cells within cancer tissue. This publicly available multiomics resource offers insights into the initiation and progression of the most lethal skin cancer (melanoma) and the most common forms (cSCC and BCC) and can be explored interactively at https://skincanceratlas.com.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87a9509055032cdb5b434f194c71651a4b05c7db" target='_blank'>
              Integrating 12 Spatial and Single Cell Technologies to Characterise Tumour Neighbourhoods and Cellular Interactions in three Skin Cancer Types
              </a>
            </td>
          <td>
            P. Prakrithi, Laura F Grice, Feng Zhang, Levi Hockey, Samuel X. Tan, Xiao Tan, Zherui Xiong, Onkar Mulay, Andrew Causer, Andrew Newman, Duy Pham, Guiyan Ni, Kelvin Tuong, Xinnan Jin, Eunju Kim, Minh Tran, Hani Vu, Nicholas M Muller, E. Killingbeck, Mark T. Gregory, S. Teoh, Tuan Vo, Min Zhang, M. T. Landi, Kevin M. Brown, M. Iles, Zachary Reitz, K. Devitt, Liuliu Pan, A. Kulasinghe, Y. Kao, Michael Leon, Sarah Murphy, Hiromi Sato, J. G. Cruz, Snehlata Kumari, Hung N. Luu, Sarah E. Warren, Chris McMillan, J. Henricson, Chris Anderson, David Muller, Arun Everest-Dass, Blake O’Brien, M. Seviiri, Matthew H. Law, H. P. Soyer, Ian Frazer, Youngmi Kim, Mitchell S. Stark, Kiarash Khosrotehrani, Quan Nguyen
          </td>
          <td>2025-07-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="DNA sequencing technologies are widely used to study tumor evolution within a cancer patient. However, analyses require various computational methods, including those to infer clone sequences (genotypes of cancer cell populations), clone frequencies within each tumor sample, clone phylogeny, mutational tree, dynamics of mutational signatures, and metastatic cell migration events. Therefore, we developed GenoPath, a streamlined pipeline of existing tools to perform tumor evolution analysis. We also developed and added tools to visualize results to assist interpretation and derive biological insights. We have illustrated GenoPath’s utility through a case study of tumor evolution using metastatic prostate cancer data. By reducing computational barriers, GenoPath broadens access to tumor evolution analysis. The software is available at https://github.com/SayakaMiura/GP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e039aed15ffae5155d36f38b8ec85975e0960dc5" target='_blank'>
              GenoPath: a pipeline to infer tumor clone composition, mutational history, and metastatic cell migration events from tumor DNA sequencing data
              </a>
            </td>
          <td>
            Ryan M. Tobin, Shikha Singh, Sudhir Kumar, Sayaka Miura
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) remains a lethal malignancy. Although immunochemotherapy regimens have improved patient survival rates, drug resistance still occurs in a significant subset of patients, highlighting the importance of elucidating the mechanisms within the tumor microenvironment. Here, we applied spatial single-cell transcriptomics to investigate the spatial characteristics of SCLC and their associations with immunochemotherapy resistance. By analyzing samples from 18 patients with extensive-stage SCLC, we identified two distinct epithelial cell subtypes: Epi-I and Epi-II. Epi-I exhibited high proliferative activity and was associated with treatment resistance and poor survival outcomes. In contrast, Epi-II showed more spatial contact with immune cells and was associated with treatment sensitivity. Further analysis uncovered a fascinating cellular transition paradigm, wherein Epi-I may be derived from Epi-II, with myeloid cells playing a facilitatory role in this transformation cascade. Specifically, within the spatial zone that was enriched with the Epi-II, epithelial cells may secrete MIF gene, which promoted the polarization of myeloid cells towards the M2 macrophages. The M2-polarized myeloid cells subsequently upregulated the expression of SPP1 that in turn triggered the activation of the PI3K-AKT signaling pathway in the adjacent epithelial cells, driving the conversion of Epi-II to Epi-I cells. Our findings revealed that the intricate crosstalk between epithelial and myeloid cells constitutes a pivotal resistance mechanism in SCLC, and targeting the SPP1/MIF pathway emerged as a promising strategy with the potential to enhance the treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0302ec6123f8cd6d62c8c99e2e99dcc7087718b" target='_blank'>
              Spatial transcriptomics reveals macrophage domestication by epithelial cells promotes immunotherapy resistance in small cell lung cancer
              </a>
            </td>
          <td>
            Yu Sun, Minghui Zhang, Yanbin Zhao, Yubo Yan, Lei Wang, Xuhui Liu, Songsong Xia, Bingbing Wang, Xiaoxin Zhang, Yan Wang
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Cholangiocarcinoma (CCA) is a highly heterogeneous malignancy, primarily comprising intrahepatic (iCCA) and extrahepatic (eCCA) subtypes. Reconciling the variability between iCCAs and eCCAs in clinical trials remains a challenge, largely due to the inadequate understanding of their shared and subtype-specific cellular heterogeneity. We aim to address this issue using single-cell and spatially resolved transcriptomic approaches. Methods We performed comprehensive single-cell RNA sequencing (scRNA-seq) by profiling 109,071 single cells from 28 samples, including chronic biliary inflammatory conditions (n = 7) and CCAs from different anatomical sites (n = 21). Findings were validated using external multi-omics datasets, tissue microarray cohort, spatial RNA in situ sequencing, CCA patient-derived organoids (PDOs), and mouse models. Results iCCAs and eCCAs exhibited distinct tumor ecosystems, with notable differences in cellular composition, diversity, and abundance across various cell types. Non-malignant epithelial cells displayed divergent precancer hallmarks from different biliary sites, with inflammatory extrahepatic bile ducts exhibiting early hijacking of the gastrointestinal metaplastic process. We identified seven meta-programs within cancer cells, mapped into four major subtypes. This subtyping was validated using external CCA cohorts and PDO models, distinguishing patients based on clinical outcomes and drug vulnerabilities. Specifically, iCCAs were associated with a senescent program, while eCCAs were enriched in an IFN-responsive program linked to adverse clinical outcomes and increased drug resistance. We identified a basal-like LY6D+ cancer cell subpopulation specific to eCCAs, which displayed significant stemness, drug resistance, and IFN-responsive features. This subpopulation was closely associated with an interferon-stimulated gene 15 (ISG15)-enriched mesenchymal and immune microenvironment. Functional assays demonstrated that ISG15 stimulation significantly boosted stemness, basal-like features, and drug resistance in eCCA cells, highlighting its pivotal role in sustaining the LY6D+ progenitor niches. Conclusion We present a comprehensive single-cell landscape of CCAs, uncovering the molecular heterogeneity between iCCA and eCCA subtypes. Transcriptomic subtyping of CCA cancer cells offers implications for clinical stratification and functional precision oncology. We identify basal-like epithelial progenitors and characterize their associated ISG15-enriched microenvironment in eCCAs. These findings hold significant promise for the development of novel prognostic biomarkers, therapeutic targets, and treatment strategies for CCAs. Supplementary Information The online version contains supplementary material available at 10.1186/s13045-025-01716-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fd09acd4eb4bccd6f0a7c5305db1cb0247e6ead" target='_blank'>
              Deciphering cholangiocarcinoma heterogeneity and specific progenitor cell niche of extrahepatic cholangiocarcinoma at single-cell resolution
              </a>
            </td>
          <td>
            Chunliang Liu, Xiang Wang, Erdong Liu, Yali Zong, Wenlong Yu, Youhai Jiang, Jianan Chen, Mingye Gu, Zhengyuan Meng, Jingfeng Li, Yang Liu, Yongjie Zhang, Jing Tang, Hongyang Wang, Jing Fu
          </td>
          <td>2025-06-23</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Background Neoadjuvant immune checkpoint blockade (nICB) has revolutionized cancer treatment, yet the underlying mechanisms of resistance in bladder cancer remain to be explored. Methods We conducted single-cell RNA sequencing (scRNA-seq) on peripheral blood mononuclear cells, tumor tissues, adjacent normal tissues, and metastatic lymph nodes from 2 nICB-naïve and 10 nICB-treated patients with bladder cancer (5 responders and 5 non-responders). Spatial RNA sequencing was performed on tumor slides from two responders and four non-responders. Findings were validated by multiplex immunohistochemistry, mice orthotopic bladder cancer model, and flow cytometry assays. Results nICB remodeled the tumor microenvironment of bladder cancer from both single-cell and spatial perspectives. scRNA-seq analysis revealed a significant increase in MYBL2hi cancer stem cells (CSCs) among non-responders. Analysis of the myeloid population showed that SPP1+ macrophages associated with angiogenesis were linked to CD8+ T cell exclusion. Further investigation into cell–cell communication revealed a propensity for bidirectional crosstalk between MYBL2hi CSCs and SPP1+ macrophages in non-responders. MYBL2hi CSCs derived CCL15, which bound to CCR1 and induced SPP1 upregulation in macrophages which reciprocally enhanced bladder cancer stemness and resistance to nICB through the SPP1-ITGα9β1 axis. Additionally, we identified an aged CCL3+ neutrophil population that interacted with SPP1+ macrophages through a positive feedback loop, contributing to nICB resistance. Finally, in vivo studies demonstrated that combined MYBL2 knockdown and SPP1 targeting synergistically enhanced ICB efficacy in bladder cancer. Conclusions Our research reveals transcriptomic characteristics associated with distinct therapeutic responses to nICB treatment, offering a foundation for optimizing personalized neoadjuvant strategies in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f561fa737d2808a2b787b34e9d1e658c2219648a" target='_blank'>
              Malevolent alliance of MYBL2hi cancer stem cell and SPP1+ macrophage confers resistance to neoadjuvant immunotherapy in bladder cancer
              </a>
            </td>
          <td>
            Hualin Chen, Zhaoheng Jin, Yueqiang Peng, Yingjie Li, Ziyi Li, Xuebin Zhang, Yi Xie, Jie Dong, Lin Ma, Zhigang Ji
          </td>
          <td>2025-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is characterized by high incidence and recurrence rates, presenting as an immune ‘cold’ tumor that exhibits a poor response to immunotherapy. The mechanisms underlying immune suppression and evasion within the tumor microenvironment (TME) of PCa remain inadequately understood. Methods A comprehensive analysis of the immune environment in PCa was conducted using combined single-cell and spatial transcriptomic approaches, encompassing samples from healthy tissue, adjacent normal tissue, and localized tumors. Cell abundance and polarization state analyses were performed to identify pivotal cellular populations. Spatial deconvolution techniques were employed to elucidate cell composition within its spatial context. Additionally, cell niche and spatial colocalization analyses were conducted to evaluate potential cellular interactions. Immune response enrichment analysis was utilized to assess cellular response states. In vivo and in vitro experiments were conducted to validate hypotheses. Results Data indicated a prevalent immunosuppressive state among CD8 T cells, accompanied by variations in cell abundance. Macrophages emerged as key regulators in recruiting CD8+ effector T cells and regulatory T cells (Tregs) into the TME, mediated by the CXCL12/CXCR4 axis. A spatial proximity relationship was established between CD8+ effector T cells and Tregs, suggesting Tregs directly influence CD8+ T cell function. Immune cell state analysis revealed interleukin-2 (IL-2) as a critical cytokine in reshaping the immune microenvironment, with Tregs competitively depleting IL-2 and mediating IL-2/STAT5 signaling to induce CD8+ effector T cell exhaustion. Treatment with CXCR4 inhibitor and IL-2 demonstrated significant antitumor effects and reversed immune dysfunction in both in vivo and in vitro experiments, with combined treatment exhibiting superior efficacy. Conclusion These findings elucidate the role of macrophages in mediating the CXCL12/CXCR4 axis to aggregate CD8+ effector T cells and Tregs, thereby influencing the TME. Furthermore, Tregs competitively deplete IL-2 and mediate IL-2/STAT5 signaling, leading to CD8+ effector T cells exhaustion and the establishment of an immunosuppressive microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/363b3add731d9a14c149f45989beacb3b8971f64" target='_blank'>
              CXCL12/CXCR4 axis governs Treg spatial dominance over CD8+ T cells via IL-2 sequestration: a dual therapeutic target in prostate cancer
              </a>
            </td>
          <td>
            Junyi Li, Long Zhang, Ruoyang Liu, Changwen Xu, Huihui Tang, Yunfei Zou, Qingfei Cao, Weichao Huang
          </td>
          <td>2025-07-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Neoadjuvant therapy (NAT) has transformed cancer treatment by improving surgical outcomes and survival rates, yet resistance mechanisms across multiple cancer types remain unclear. This study aimed to decipher tumor ecosystem dynamics during NAT using cross-cancer single-cell sequencing data, focusing on identifying key mediators of immunosuppression and treatment resistance. Methods Single-cell RNA-sequencing (scRNA-seq) datasets from five solid tumors (esophageal squamous cell carcinoma, esophagogastric junction carcinoma, colorectal cancer, cervical cancer, and triple-negative breast cancer) were integrated. The data from these five cancer types underwent a rigorous process to standardize cell types across all datasets. Cell-cell communication analysis, Meta-Programs (MPs) via non-negative matrix factorization, and functional enrichment were performed. Immunohistochemistry (IHC) and Western blot validated S100A4 expression and PD-L1 induction in vitro. Results We constructed a single-cell map across cancer types and systematically characterized dynamic changes in tumor cells and diverse microenvironmental cell populations following neoadjuvant therapy, along with thier gene expression and pathway alterations. Our findings highlight that crosstalk between cancer-associated fibroblasts (CAFs) and tumor cells represents a critical determinant of neoadjuvant therapy resistance. Fibroblasts underwent significant state transitions post-treatment, marked by hypoxia-associated gene upregulation (e.g., S100A4) and immunosuppressive pathways. Meta-Programs (MPs) analysis identified a hypoxia-driven fibroblast state (MP5) containing S100A4 that correlated with treatment resistance. In vitro experiments, S100A4 co-localized with α-SMA + fibroblasts and directly induced PD-L1 expression in tumor cells, linking CAFs secreted S100A4 to immunosuppressive PD-L1 upregulation. Conclusion This cross-cancer single-cell atlas reveals S100A4, secreted by CAFs, as a conserved mediator of PD-L1 upregulation in tumor cells, driving immunosuppression and resistance to nICT. The atlas and mechanistic findings provide a rationale for targeting S100A4 to enhance treatment efficacy, pending validation in larger cohorts and mechanistic studies. This resource also supports the development of personalized, cross-cancer neoadjuvant strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/875f07738539c0aaf3118d5091d4cb483e59f11d" target='_blank'>
              Deciphering the tumor ecosystem dynamics undergoing immunochemotherapy therapy across multiple cancer types unveils the immunosuppressive role of S100A4 in fibroblasts by promoting PD-L1 expression in tumor cells
              </a>
            </td>
          <td>
            Bo Yang, Ruiji Chen, Mali Zu, Jie Yao, Hong Ren, Yi‐Shung Lin, Bo Zhang, Tianjiao Ji, Yang Liu
          </td>
          <td>2025-07-23</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer remains a formidable global health burden owing to its persistently high incidence and mortality, highlighting the need to elucidate its underlying biological mechanisms. Such insights are crucial for refining diagnostic precision and therapeutic efficacy. Notably, human tumors represent highly heterogeneous ecosystems, comprising a dynamic interplay between malignant cells and non-malignant components, such as immune infiltrates and stromal elements. This cellular heterogeneity constitutes a major obstacle in tumor research. Recent technological advancements have enabled the development and application of novel tools for investigating tumor heterogeneity. In this context, single-cell sequencing technologies emerged as transformative tools for dissecting tumor architecture at a cellular resolution, offering unprecedented insights into the cellular diversity and molecular underpinnings of cancer. In particular, single-cell RNA sequencing (scRNA-seq) has been widely used to analyze the tumor microenvironment, particularly its immune components, thereby providing a valuable framework for understanding tumor-immune system interactions. However, the intrinsic heterogeneity of tumor cells has received comparatively less attention, with limited studies focusing on these malignant populations. Comprehensive profiling of tumor cells has become increasingly feasible as scRNA-seq technologies continue to evolve, offering higher throughput, increased cell capture efficiency, and more advanced analytical pipelines. Despite these advancements, current applications of scRNA-seq remain primarily focused on immune and stromal cell populations. Therefore, a paradigm shift is warranted, redirecting the investigative focus toward tumor cells and gaining deeper insights into tumorigenesis and therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c3598e7911b47f7291bb4a9f0c4b83bd05a9bc" target='_blank'>
              Exploration and Application of Malignant Cell Heterogeneity Analysis with Single-Cell Transcriptome Sequencing Technology.
              </a>
            </td>
          <td>
            Wen Zhang, Tianliang Liu, Ting Liu, Yu Zhang, Zhenguo Zhao, Xinxin Zhang, Xiaoyang Li, Jiasheng Zhong, Zhicheng Li, Shifu Chen, Libin Xu
          </td>
          <td>2025-07-25</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) exhibits substantial intertumoral heterogeneity, largely attributable to multiple tumor stem‐like cell populations, whose molecular identities and clinical significance remain incompletely defined. This study delineates tumor‐intrinsic stem‐like cell diversity and its prognostic implications through single‐cell transcriptomic profiling of 171,906 tumor epithelial cells (n = 152), integrated with bulk transcriptomic (n = 1389) and genomic (n = 1077) datasets. Functional validation was conducted via in vitro assays and multiplex immunofluorescence. A previously unrecognized lysosome‐associated transmembrane protein 4B‐positive (LAPTM4B+) stem‐like cell cluster was identified, distinct from the classical leucine‐rich repeat‐containing G‐protein coupled receptor 5‐positive (LGR5+) population. LAPTM4B+ cells exhibited MYC pathway activation and 8q chromosomal gains, with preferential enrichment in microsatellite‐stable, POLE wild‐type, and left‐sided tumors. Stratification based on LAPTM4B+/LGR5+ stem‐like cell ratios defined four CRC stem‐like subtypes (CSS), with CSS2 (LAPTM4B+‐dominant) associated with the poorest prognosis (HR = 2.31, p < 0.001). The combined expression of LAPTM4B and LGR5 demonstrated superior predictive power for CRC progression compared to either marker alone (AUC = 0.820 vs. 0.715/0.699), underscoring the synergistic influence of distinct stem‐like cell populations on patient outcomes. These findings provide novel insights into CRC heterogeneity and cooperative interactions among diverse stem‐like populations shaping disease outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/870d87b3057664fbe8a51f6ca3867c7fe15a2488" target='_blank'>
              Novel Lysosomal‐Associated Transmembrane Protein 4B‐Positive Stem‐Like Cell Subpopulation Characterizes High‐Risk Colorectal Cancer Subtypes
              </a>
            </td>
          <td>
            Yangyang Fang, Tian-Hao Fu, Ziqing Xiong, Qian Zhang, Wei Liu, Kuai Yu, Aiping Le
          </td>
          <td>2025-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The most common cause of death in neurofibromatosis type 1 (NF1) is the development of malignant peripheral nerve sheath tumor (MPNST), a deadly sarcoma that can transform from benign plexiform neurofibromas (PN) or premalignant atypical neurofibromas (AN). We built a single-cell dataset of 55 NF1-associated PN, AN, and MPNST to define cellular changes in neurofibroma at-risk of malignant transformation. Integrative analysis of changes in the tumor microenvironment revealed the emergence of malignant tumor cells, regulatory T cells, and loss of activated macrophages in MPNST. Using this reference dataset, we validated findings using anchor-based label transfer in an additional 19 NF1 nerve sheath tumors profiled with single cell sequencing, and public datasets. We then defined protein biomarkers of malignant transformation from high-throughput proteomic analysis of plasma samples collected from 45 NF1 patients that correlated to mRNAs specific to MPNST cell populations. Fifty plasma proteins accurately and non-invasively distinguished patients with MPNST from those with premalignant tumors. These markers should improve the ability to identify high-risk neurofibromas for improved cancer surveillance and enable early detection of malignant transformation in NF1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea5d7e2095827b87208bdc18119d43a501ce00d" target='_blank'>
              Single-cell tumor microenvironment profiling informs a circulating proteome test for the interception of malignant transformation in NF1 nerve sheath tumors
              </a>
            </td>
          <td>
            Jack Shern
          </td>
          <td>2025-06-18</td>
          <td>Research Square</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most aggressive brain tumor in adults, is characterized by its infiltrative growth along the perivascular space. Mural cells (MCs), encompassing pericytes and smooth muscle cells, are multifunctional perivascular cells implicated in GBM progression. MCs not only facilitate vascular co-option but have also been suggested to contribute to the immunosuppressive tumor microenvironment, promoting tumor growth and migration. However, whether MC interactions with immune cells differ based on their proximity to the tumor remains unclear. Using single-cell RNA sequencing, we analyzed MC transcriptome profiles across distinct regions relative to the tumor mass in mouse and human GBM samples. Tumor-residing MCs exhibited profound phenotypic changes, showing upregulated gene expression and enhanced signaling activity toward immune cells, with region-specific ligand-receptor interactions. Conversely, border-residing MCs, despite their abundance, showed reduced activation and lacked distinct transcriptional profiles. These findings reveal spatially defined transcriptional heterogeneity in MCs within the GBM microenvironment, underscoring their dynamic role in the GBM microenvironment. This study provides novel insights into MC responses in GBM, identifying potential avenues for targeting MC-immune-cell interactions in therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34269e7ccaff0901b281158ca70f685dbde49243" target='_blank'>
              Pericytes change function depending on glioblastoma vicinity: emphasis on immune regulation.
              </a>
            </td>
          <td>
            C. Buizza, Robert Carlsson, Coralie Gamper, Gayatri Chitale, J. Bengzon, Gesine Paul
          </td>
          <td>2025-07-17</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Organoid models often fail to capture the complex niche of patient-specific tumors. In this study, we present a novel methodology for generating gastric cancer assembloids composed of matched tumor organoids and stromal cell subpopulations, which closely recapitulate the cellular heterogeneity and microenvironment of primary tumors. The inclusion of autologous stromal cell subpopulations significantly influences gene expression and drug response sensitivity. By incorporating diverse stromal cell populations derived from the same tumor tissue as the organoids, these assembloids enable a more comprehensive investigation of individual tumor biology, biomarker expression, transcriptomic profiles, and cell–cell interactions. The model also supports personalized drug screening and the optimization of combination therapies. Altogether, the integration of patient-specific stromal cell subsets enhances the physiological relevance of preclinical testing, providing insights into resistance mechanisms and ultimately contributing to the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a497a2c2ec986f0ccfd70ff27fc0f7edf51baf58" target='_blank'>
              Patient-Derived Gastric Cancer Assembloid Model Integrating Matched Tumor Organoids and Stromal Cell Subpopulations
              </a>
            </td>
          <td>
            Irit Shapira-Netanelov, Olga Furman, Dikla Rogachevsky, G. Luboshits, Yael Maizels, Dmitry Rodin, Igor Koman, Gabriela A. Rozic
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14d1b8aeaba5a454b04d6690efe6420aea37c43e" target='_blank'>
              Stochastic modelling of prostate progenitor architecture
              </a>
            </td>
          <td>
            Christo Morison, Esther Baena, Weini Huang
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is an aggressive gynecological malignancy characterized by intraperitoneal spread and chemotherapy resistance. Chemotherapies have demonstrated limited effectiveness in HGSOC, underscoring the urgent need to evaluate how the tumor microenvironment (TME) was reshaped by chemotherapy in different sites of tumor foci. In this study, we performed single-cell transcriptomic analysis to explore the TME in samples obtained from various sites of tumor foci, with or without the history of Neoadjuvant chemotherapy (NACT). We discovered that chemotherapy reshaped the tumor immune microenvironment, evident through the reduction in human leukocyte antigen (HLA) diversity and the increase in PDCD1/CD274 in CD8_ANXA1, LAMP3+ dendritic cell (DC_LAMP3), and EREG+ monocytes (mono_EREG). Moreover, cancer.cell.2, cancer-associated C3+ fibroblasts (CAF_C3), and Fibrocyte_CD34, which are prone to accumulate in the metastatic site and post-NACT group, harbored poor clinical outcome, reflected in the immune exclusion and tumor progression signaling. Cell–cell communication identified a stronger interaction between cancer.cell.2 and CAF_C3, as well as Fibrocyte_CD34, in post-NACT samples, indicating that chemotherapy reshapes pre-existing cell clusters in a site-dependent manner. Our findings suggest that chemotherapy and sites of foci were critical for the transcriptional reprogramming of pre-existed cell clusters. Our study offers a single-cell phenotype data substrate from which to develop a personalized combination of chemotherapy and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facdb0c7f366e37f598e4ee1a1925a639be9766f" target='_blank'>
              Chemoresistance Evolution in Ovarian Cancer Delineated by Single-Cell RNA Sequencing
              </a>
            </td>
          <td>
            Yuanmei Wang, Zongfu Tang, Haoyu Li, Run Zhou, Hao Wu, Xiaoping Cen, Yi Zhang, Wei Dong, Huanming Yang
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The effectiveness of colorectal cancer (CRC) therapy is limited owing to the absence of treatments targeting drug-tolerant residual cancer cells. Although neoadjuvant therapy is effective, pathological examination of residual tumors has revealed the presence of small clusters of LGR5-positive cancer cells in the fibrous tissue. Here, we established a colorectal cancer cell line-derived organoid (CCD-organoid) regrowth model using a patient-derived cell line with cancer stem cell properties and demonstrated that it displayed the morphological characteristics of small clusters in clinical tissues. Time course analysis of single-cell RNA sequencing of the CCD-organoid regrowth model revealed various states and dynamic alterations within non-cycling cells. We identified subpopulations highly expressing protein translation-related genes RPL17 and EEF1G. To identify key signals for the transition of residual cancer cells to regrowth, we evaluated inhibitors targeting pathways such as the Wnt pathway, reactive oxygen species pathway, and RNA polymerase I pathway, highlighted in the single-cell RNA sequencing analysis. Only the polymerase I-inhibitor BMH-21 significantly reduced tumor growth both in vitro and in vivo, indicating the critical cell subpopulation driving recurrence. Our results demonstrate the possibility of a unique therapeutic target for CRC treatment targeting drug-tolerant residual cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa9b58cf763b8a32b666aa1599214c1a009f1d2" target='_blank'>
              Colorectal cancer cell line-derived organoid model with stem cell properties captures the regrowing state of residual cancer cells after neoadjuvant chemotherapy
              </a>
            </td>
          <td>
            Kiyotaka Nakano, E. Oki, Masaki Yamazaki, Masami Suzuki, S. Kawai, T. Fujita, Atsuhiko Kato, Y. Zaitsu, Tomoko Jogo, Chie Kato, Takeshi Watanabe, Eri Hashimoto, Chiyoko Nishime, E. Fujii, K. Ando, G. Nagae, N. Harimoto, Mitsuhiko Ota, H. Saeki, Hiroyuki Aburatani, Yoshihiko Maehara, Tatsumi Yamazaki
          </td>
          <td>2025-06-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Globally, gastric cancer (GC) stands as the fifth most prevalent form of malignant neoplasm and represents a significant contributor to mortality associated with oncological conditions. Despite advancements in therapeutic strategies for GC, the outcomes for patients with advanced stages of the disease continue to be unfavorable, largely due to tumor heterogeneity and the challenges posed by resistance to therapeutic agents. Metabolic reprogramming is pivotal in driving the advancement of GC, contributing to the development of resistance to pharmacological treatments and facilitating the cancer’s ability to evade immune surveillance. Developing multi-target comprehensive treatment strategies by integrating tumor microenvironment (TME) modulation holds promise for significantly improving therapeutic efficacy. Methods The study analyzed GC and identified key cell subtypes by integrating data derived from single-cell RNA-sequencing (scRNA-seq) alongside spatial transcriptomics information. Cell type identification was accomplished using the tool of Seurat, and the spatial distribution of cell types was revealed through the Robust Cell Type Decomposition technique. CellChat was used to analyze the interactions between key cell subtypes and other cells, and the “StLearn” package was employed to investigate spatial cell communication in depth. Additionally, the functional role of the key molecule ELK4 was validated through in vitro experiments. Results This research utilized scRNA-seq combined with spatial transcriptomics to comprehensively analyze GC, identifying the C1 NDUFAB1+ subtype, which exhibited high proliferative activity, metabolic reprogramming capabilities, and immune evasion properties. It was found that the C1 NDUFAB1+ subtype closely interacted with fibroblasts and pericytes via the PARs signaling pathway. Additionally, in vitro experiments confirmed that knockdown of ELK4 substantially curbed tumor cell proliferation, migration, and invasion. Conclusion This study revealed the main significance of the C1 NDUFAB1+ subtype in GC, elucidating its core mechanisms in tumor progression, metabolic reprogramming, and immune evasion. ELK4 was identified as a key regulatory factor that markedly enhanced the proliferation, migratory capacity, and invasive potential of tumor cells, while changes in the TME were a driving force behind immune suppression and drug resistance. The findings underscored the importance of developing specific therapeutic targets, targeting metabolic reprogramming, and overcoming immune evasion, providing new theoretical foundations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94360dc80f5310630bd93066b812e27d59cfa9fb" target='_blank'>
              Integrative single-cell and spatial transcriptomics uncover ELK4-mediated mechanisms in NDUFAB1+ tumor cells driving gastric cancer progression, metabolic reprogramming, and immune evasion
              </a>
            </td>
          <td>
            Yuwei Sun, Wenyang Nie, Zhikai Xiahou, Xiaojing Wang, Wenjia Liu, Zongkai Liu, Zhiheng Lin, Zhaidong Liu
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Osteosarcoma (OS) is the most common primary malignant bone tumour of childhood, yet five-year survival has plateaued at ~60–70% for localised disease and plunges below 30% once metastasis emerges. Formerly viewed as a cell-intrinsic neoplasm entombed in mineralised bone, OS is now understood as a spatially stratified ecosystem whose immune-evasion niches choreograph progression. Three-dimensional spatial transcriptomics (3-D ST) fuses barcode-based transcript capture with volumetric reconstruction, preserving x-, y- and z-axis context and exposing concentric C1QC necrotic belts, MCAM (melanoma cell-adhesion molecule, CD146) peri-vascular corridors, hypoxic glycolytic rims and therapy-induced tertiary-lymphoid islets that collectively sequester cytotoxic lymphocytes. Pre-clinical atlases link PD-L1 high SOX9 stem-like cells, LGALS3 macrophages and VEGFA-driven endothelial tips to chemoresistance and immune-checkpoint failure, while ligand–receptor inference highlights VEGFA–VEGFR2, CXCL12–CXCR4 and complement–CSF1R axes as actionable bottlenecks. Translational efforts already echo these insights: dual MCAM/VEGFR blockade collapses vascular gates, C1s or CSF1R antagonists dismantle necrotic-core “cold pockets”, and MCT1–POSTN combinations target lactate-stiffened stromal shells. By weaving methodological advances with emergent biology, this review crystallises how 3-D ST redefines OS immunopathology, sharpens biomarker discovery and accelerates spatially guided combination therapies. We aim to expose diagnostic blind spots, spotlight niche-directed interventions and chart a roadmap toward lifting the long-standing therapeutic ceiling in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c84eb34c63a5fd31899b587fca53cec9b872be8" target='_blank'>
              Deciphering spatially confined immune evasion niches in osteosarcoma with 3-D spatial transcriptomics: a literature review
              </a>
            </td>
          <td>
            Guangqiang Qiu, Yongcheng Tang, Junhui Zuo, Heng Wu, Yongxian Wan
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Triple-negative breast cancer is a highly aggressive disease subtype with limited treatment options. Despite the approval of targeted therapies such as PARP inhibitors, metastatic progression remains inevitable. Additionally, spatial arrangements, including immune exclusion zones and immune–stromal crosstalk, can profoundly influence therapeutic response. These observations underscore the need for more effective, individualized treatments that both target malignant cells and promote anti-tumor microenvironments. Toward that goal, we have developed a coordinated experimental-computational approach designed to enable identification of effective treatment strategies that consider the functional relationships between different cell types. First, to identify adaptive responses that may serve as therapeutic vulnerabilities, we treated C3TAg genetically engineered mouse model with a panel of clinically approved therapies then subjected these samples to single-cell RNAseq and multiplex tissue imaging. We observed that relatively short treatment durations lead to changes in the composition and spatial organization of tumors, suggesting adaptive responses that may be leveraged in combination treatment strategies. Notably, we observed changes in several macrophage populations, including changes to co-localization of M2-like macrophage and cancer-associated fibroblasts, highlighting stromal responses to therapeutic pressure. Second, we are using the resultant data to guide development and parameterization of an agent-based model that considers the functional relationships between different tumor components and can be used to predict effective therapeutic treatment strategies. We have developed analytical approaches to leverage our single-cell RNAseq data to estimate key model parameters operable in prioritized cell populations, including those related to proliferation, motility, antigen presentation, and cell-cell signaling. Initial model simulations suggest that spatial cell patterns are often not steady: immune cells dynamically aggregate, disperse, migrate, and reaggregate at new locations; macrophages infiltrated hypoxic and necrotic regions, where they polarized from an anti-tumor M1-like state to a pro-tumor M2-like state. These observations lay the groundwork for leveraging agent-based models for in silico testing of therapeutic combinations that can be prioritized for experimental validation. In total, our approach has the potential to rationally identify effective therapeutic strategies that lead to long-term durable control of tumors.



 Laura M. Heiser, Young Hwan Chang, Paul Macklin. Multiscale systems approach to target tumor ecosystem responses for therapeutic benefit [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3f02bb56a8eb490bc79542639fae113efb7474f" target='_blank'>
              Abstract A027: Multiscale systems approach to target tumor ecosystem responses for therapeutic benefit
              </a>
            </td>
          <td>
            Laura M. Heiser, Young Hwan Chang, Paul Macklin
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d71cf07bbb7f7d458f445b935e53d33e0b0d78bd" target='_blank'>
              Single-Cell Spatial Mapping of Human Kidney Development Reveals the Critical Role of the Local Microenvironment in Cell Fate Decisions
              </a>
            </td>
          <td>
            Jonathan Levinsohn, Samuel Grindel, Bernhard Dumoulin, Amin Abedini, A. Huang, Rena Levin-Klein, Boaz Weicz, Grace Rabinowitz, Andi M. Bergeson, Eunji Ha, Konstantin A Klötzer, Nancy R. Zhang, Paul M. Titchenell, Mingyao Li, Joo-Seop Park, Laura S. Finn, Kotaro Sasaki, P. Beckerman, Oren Pleniceanu, Alex J. Hughes, Katalin Susztak
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="While cervical cancer (CC) prognosis depends on tumor staging, the spatiotemporal evolution of tumor microenvironment (TME) heterogeneity during metastatic progression remains poorly characterized at single-cell resolution. We employed an integrative multi-omics approach, combining single-cell RNA sequencing (scRNA-seq; n = 11), spatial transcriptomics (ST), and bulk RNA-seq data from the TCGA-CESC cohort (n = 304), to systematically map TME remodeling across CC progression stages. scRNA-seq was performed on primary lesions from patients with localized (n = 3), regional (n = 4), and metastatic (n = 4) diseases, with in-depth analyses focusing on cellular characteristics, cell type composition alterations, functional changes, differentiation trajectories, and cell-cell interaction networks. These findings were further validated using spatial transcriptomics, bulk RNA-seq data, and multiple immunohistochemistry (mIHC) experiments. ScRNA-seq data revealed that the TME of the metastatic group displayed a distinct immunosuppressive phenotype. Three key subclusters closely linked to TME remodeling in this group were identified. Notably, a novel metastasis-associated epithelial subpopulation (Epi0_AGR2), characterized by both epithelial-mesenchymal transition (EMT) and chemokine secretory phenotypes, was discovered. Gene Set Variation Analysis (GSVA) revealed that transforming growth factor β (TGF-β) signaling activation served as its primary transcriptional driver. Additionally, a neutrophil subset with pro-tumor and immunosuppressive properties, as well as a cancer-associated fibroblasts (CAFs) subset that promoted angiogenesis, were enriched in the metastatic group. Cell-cell interaction analysis and spatial mapping further revealed the formation of coordinated Epi0-neutrophil-CAFs niches, which established TGF-β-CXCL1/2/8-OSM/OSMR feedforward loops. Importantly, a computational model derived from the TME metastatic niche signature demonstrated significant prognostic stratification in the TCGA cohort (HR = 2.5179, p = 0.0144). In all, this study provides the first comprehensive delineation of stage-specific TME dynamics in CC, revealing TGF-β-driven cellular cooperativity as a metastatic switch. The joint framework establishes a potential clinically translatable tool for precision prognosis and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac624640fa9f989701717857751510daeec92896" target='_blank'>
              Multi-Omics Analysis Reveals the transforming growth factor-β Signaling-Driven Multicellular Interactions with Prognostic Relevance in Cervical Cancer Progression
              </a>
            </td>
          <td>
            Yuhan Wang, Guang-xu Cao, Huimin Zeng, Yong Zhi, Meng-Xi Xu, Ying Wang, Min Liu, Yetian Ruan, Ka Yu Tse, Qingfeng Zhang, Jinli Gao, Zhiqiang Han, Fang Li
          </td>
          <td>2025-06-20</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Spread through air spaces (STAS) represents a novel invasion mechanism in adenocarcinoma that considerably influences lung cancer clinical outcomes; however, studies of its mechanisms at the spatial level are lacking. Methods We used the NanoString GeoMx digital spatial profiling (DSP) technology to conduct a spatial transcriptomic analysis of surgically resected tissues from non-small-cell lung cancer (NSCLC) patients with or without STAS. Results Compared with tumor nests in non-STAS patients, HLA-DRB5 and RASGRF1 were significantly less expressed in compartments of STAS, suggesting their inhibitory roles in the occurrence of STAS. Meanwhile, an increase in CD4 T memory cells and a decrease in B cells were observed in the tumor immune microenvironment of STAS. Furthermore, distinct molecular profiles were observed between tumor cells in tumor nests and in air spaces in STAS patients, which was highlighted by the elevated ITGA2 expression in the air spaces. These results were validated in an independent cohort by multiplex immunofluorescence stainings. Conclusion This study is the first to use DSP to analyze spatial transcriptomic profiles of NSCLC tumor nests and air space tumors, and it identifies potential module features that may be used for STAS identification and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/570168f77732c756800b064b7d546cfa69566936" target='_blank'>
              Spatial deciphering of the transcriptomic heterogeneity of tumor spread through air spaces in lung cancer
              </a>
            </td>
          <td>
            Wenhao Wang, Wenhao Zhou, Jingli Fan, Tao Jiang, Guang Yang, Congcong Song, Siwei Xu, Haitao Luo, Huining Liu
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Nuclear protein in testis (NUT) carcinomas (NCs) are rare, clinically aggressive tumors with characteristic translocation involving NUTM1 and BRD4 genes. While NCs are generally characterized by undifferentiated basaloid cells with focal/abrupt squamous differentiation, tumoral heterogeneity remains unexplored. This may have therapeutic implications as NC frequently develops drug resistance and metastasis. Spatial transcriptomics (ST) sequencing technology emerged as an approach to elucidate tumoral heterogeneity by integrating cellular transcriptome and spatial distributions within tissues, providing insights into the interactions among different cell types and overall tissue genomic architecture.


METHODS
Three cases of sinonasal NC with formalin-fixed paraffin embedded tissue formed the study material. A representative region of each NC was subjected to 10X Genomics Visium Spatial Gene Expression analysis. Resulting data were analyzed in 10X Genomics Loupe Browser.


RESULTS
All cases demonstrated statistically distinct (graph-neural network based) transcriptomic clusters. These clusters were correlated with histologic characteristics by parsing all spots by region types based on tissue distribution including tumor, limited stroma, and normal epithelium. Cluster constitution and transcriptomic pattern were consistent with the morphologic features of each tissue region. NC#3 had a relatively low number of tumor clusters in comparison to NC#1 and NC#2.


CONCLUSIONS
Spatial transcriptomic profiling corresponds to histopathologic features in NC and yet highlights cell type diversity across tumors and in the case of NC#2, within a given tumor. Behavioral correlates are speculative, but NC#3 for which the patient had a rapidly lethal outcome showed a preponderance of neural gene expression which may be of interest in tumor cell progression/evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8f96bf0cc82c0a5a49b8e030eb8a17bb4a9e111" target='_blank'>
              Magnifying Glass on Sinonasal NUT Carcinoma Heterogeneity via Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Diana Bell, Garret W. Choby, M. Afkhami, E. Maghami, R. Ferrarotto, Carl Snyderman, Eric Wang, Ehab Y. Hanna, Raja Seethala
          </td>
          <td>2025-07-07</td>
          <td>Head & neck</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c2fd63e082e26fba9172752b056b5282ca5810" target='_blank'>
              Barcoded viral tracing identifies immunosuppressive astrocyte-glioma interactions.
              </a>
            </td>
          <td>
            Brian M Andersen, Camilo Faust Akl, Michael A. Wheeler, Zhaorong Li, M. Diebold, Michael Kilian, Joseph M Rone, Aditya Misra, Jessica E. Kenison, Joon-Hyuk Lee, Hong-Gyun Lee, C. M. Polonio, David Merrell, J. Weiss, Lillie Godinez, Gavin Piester, Tomer Illouz, Jessica J Ye, Arianna Ghia, Jazmin Martinez, Elizabeth N. Chung, Léna Srun, Daniel Farrenkopf, Lucas E. Flausino, Anton M Schüle, L. Sanmarco, Federico Giovannoni, Luca Fehrenbacher, M. Charabati, Cristina Gutiérrez-Vázquez, Margaret M Cusick, Prem Prabhakar, Connor C Bossi, Emily Lapinskas, Roni Nowarski, G. Getz, Keith L. Ligon, M. Prinz, E. Chiocca, David A. Reardon, Francisco J. Quintana
          </td>
          <td>2025-06-25</td>
          <td>Nature</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Current approaches to estimating cell trajectories, tumor progression dynamics, and cell population diversity of tumor microenvironment often depend on single-cell RNA sequencing, which is costly and resource intensive. To address this limitation, we developed an artificial intelligence (AI) model that leverages cell morphology features and histological spatial organization to classify tumor cell differentiation status, infer cell dynamic trajectories, and quantify tumor progression from hematoxylin and eosin (H&E)–stained whole-slide images. In three independent lung adenocarcinoma cohorts, our AI-based model accurately predicted cell differential status and provided quantifiable measures of tumor progression that were prognostic of patient survival. Spatial transcriptomic integrative analyses revealed cell components and gene signatures enriched in different cell differentiation statuses. Bulk transcriptomic analyses revealed that fast-progressing tumors exhibit up-regulated cell cycle pathways, while slow-progressing tumors retain characteristics of normal lung epithelium. This cost-effective method enables large-scale analysis of tumor progression dynamics using routinely collected pathology slides and provides insights into intratumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ccd7a11dd44adb135af6900f42639f7a9eb70bd" target='_blank'>
              Image-based inference of tumor cell trajectories enables large-scale cancer progression analysis
              </a>
            </td>
          <td>
            Yang Liu, Ling Cai, Ruichen Rong, Shidan Wang, Liwei Jia, Peiran Quan, Qin Zhou, Guan‑Di Xiao, Yang Xie
          </td>
          <td>2025-07-18</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J. Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Histologic variant (HV) subtypes of bladder cancer are clinically aggressive tumors that are more resistant to standard therapy compared to conventional urothelial carcinoma (UC). Little is known about the transcriptional programs that account for their biological differences. Here we show using single cell analysis that HVs harbor a tumor cell state characterized by expression of MUC16 (CA125), MUC4, and KRT24. This cell state is enriched in metastases, predicted to be highly resistant to chemotherapy, and linked with poor survival. We also find enriched expression of TM4SF1, a transmembrane protein, in HV tumor cells. Chimeric antigen receptor (CAR) T cells engineered against TM4SF1 protein demonstrated in vitro and in vivo activity against bladder cancer cell lines in a TM4SF1 expression-dependent manner, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/548ce9ebb79018ca6cae79de3ec30d90694da33a" target='_blank'>
              Bladder cancer variants share aggressive features including a CA125+ cell state and targetable TM4SF1 expression
              </a>
            </td>
          <td>
            Heiko Yang, Hanbing Song, Elizabeth Yip, Tim Gilpatrick, Kevin Chang, Paul Allegakoen, Kevin L. Lu, Keliana Hui, Julia H. Pham, Corynn Kasap, Vipul Kumar, Janae Gayle, B. Stohr, Chien-Kuang Cornelia Ding, Arun P Wiita, Maxwell V. Meng, Jonathan Chou, S. Porten, Franklin W. Huang
          </td>
          <td>2025-06-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) lacks targeted therapies, leading to a poor prognosis. Younger patients with TNBC often present with aggressive disease and exhibit distinct tumor microenvironments (TME). We performed spatial profiling to understand the influence of menopausal status on the immune environment, tumor progression, and therapy response. Methods Eleven treatment-naive TNBC tumors were examined in epithelial and non-epithelial areas using digital spatial profiling to identify differentially expressed markers and pathways. Deconvolution methods were employed to identify immune cell subtypes, and the protein expression of T and B cells was confirmed. The prognostic utility of the identified genes and pathways was validated using the METABRIC and SCAN-B datasets. The expression of target genes was analyzed in the I-SPY 2 trial data to understand their influence on the response to specific therapies. Results Premenopausal tumors formed a distinct cluster characterized by the upregulation of cell activation and antigen presentation pathways. In contrast, T cell checkpoint, cancer antigen, and PI3K-AKT pathways were downregulated. External datasets validated these findings, showing a lower hazard and better prognosis for genes upregulated in premenopausal tumors. An enrichment of CD8 + T cells, endothelial cells, and monocytes was observed, along with increased intratumoral protein expression of CD8, CD4, and CD20. Premenopausal tumors demonstrated better responses to PARP and HSP90 inhibitors but showed lower sensitivity to Pembrolizumab and PI3K-AKT inhibitors compared to postmenopausal tumors in the I-SPY 2 trial data. Conclusion Our study underscores the importance of menopausal status in shaping the TME within TNBC, revealing distinct immune landscapes and therapy responses. These findings highlight the need for larger prospective studies to validate differential treatment strategies in younger patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06786-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c6a6cdaccc22fd9e9450a555636943aa5ed9adb" target='_blank'>
              Spatial profiling reveals unique immune microenvironment in premenopausal triple-negative breast cancer associated with therapy response
              </a>
            </td>
          <td>
            V. P. Nimbalkar, V. Snijesh, S. Rajarajan, C. Anupama, S. Mahalakshmi, Annie Alexander, Deshica Dechamma, Manju Moorthy, Gopalakrishna Ramaswamy, Rakesh Ramesh, B. Srinath, J. Prabhu
          </td>
          <td>2025-07-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Identifying functionally important cell states and structure within heterogeneous tumors remains a significant biological and computational challenge. Current clustering or trajectory-based models are ill-equipped to address the notion that cancer cells reside along a phenotypic continuum. We present Archetypal Analysis network (AAnet), a neural network that learns archetypal states within a phenotypic continuum in single-cell data. Unlike traditional archetypal analysis, AAnet learns archetypes in simplex-shaped neural network latent space. Using pre-clinical models and clinical breast cancers, AAnet resolves distinct cell states and processes, including cell proliferation, hypoxia, metabolism and immune interactions. Primary tumor archetypes are recapitulated in matched liver, lung and lymph node metastases. Spatial transcriptomics reveal archetypal organization within the tumor, and, intra-archetypal mirroring between cancer and adjacent stromal cells. AAnet identifies GLUT3 within the hypoxic archetype that proves critical for tumor growth and metastasis. AAnet is a powerful tool, capturing complex, functional cell states from multimodal data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4f4efe9bf4b85978ed408342a79a79722d40fd" target='_blank'>
              AAnet resolves a continuum of spatially-localized cell states to unveil intratumoral heterogeneity.
              </a>
            </td>
          <td>
            Aarthi Venkat, Scott E. Youlten, Beatriz P. San Juan, Carley A Purcell, Shabarni Gupta, Matthew Amodio, Daniel P Neumann, John G Lock, Anton E Westacott, Cerys S McCool, Daniel B. Burkhardt, Andrew Benz, A. Mollbrink, J. Lundeberg, David van Dijk, Jeff Holst, Leonard D Goldstein, Sarah Kummerfeld, Smita Krishnaswamy, Christine L Chaffer
          </td>
          <td>2025-06-24</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) was considered the most aggressive type of primary brain tumor, marked by poor clinical outcomes and a high tendency to relapse. The therapeutic efficacy of GBM was significantly compromised by tumor heterogeneity, dysregulated metabolic pathways, the formation of an immunosuppressive microenvironment, and treatment resistance. Therefore, multi-dimensional therapeutic strategies targeting GBM-specific molecular features, its intrinsic properties, and microenvironmental regulatory networks were considered to potentially provide new breakthroughs for overcoming treatment resistance in GBM. Methods We analyzed single-cell RNA sequencing (scRNA-seq) data processed with the Seurat package to accurately identify cell types. Spatial transcriptomics integrated Multimodal Intersection Analysis, TransferData, and Robust Cell Type Decomposition techniques to characterize the spatial distribution patterns of key cell subtypes. CellChat was employed to assess intercellular communication networks. Furthermore, in vitro experiments confirmed the main regulatory role of YEATS4 (key transcription factor of C2 PCLAF+ subtype) in GBM malignant progression. Results Through scRNA-seq, we identified the C2 PCLAF+ subtype in GBM and analyzed its molecular characteristics and functional role in tumor progression. This subtype exhibited a unique malignant phenotype, marked by significant proliferative activity, characteristic metabolic reprogramming, and dysregulated cell death regulation mechanisms. Spatial transcriptomics revealed its preferential localization within specific tumor niches. Furthermore, the C2 PCLAF+ subtype established a specific interaction with fibroblasts through the MDK-LRP1 ligand–receptor pair. Critically, silencing YEATS4 in vitro significantly inhibited GBM malignancy. Additionally, the prognostic risk score model based on the C2 PCLAF+ subtype demonstrated significant clinical translational value. Conclusion Our study systematically elucidated the malignant characteristics of the C2 PCLAF+ subtype and its molecular mechanisms driving GBM progression. This subtype promoted therapeutic resistance through unique metabolic reprogramming, MDK-LRP1-mediated microenvironmental interactions, and immunosuppressive properties. YEATS4 knockdown effectively suppressed malignant tumor behaviors, highlighting its therapeutic potential. These findings provided novel targeted intervention strategies to address GBM heterogeneity and treatment resistance, offering promising avenues for overcoming current therapeutic limitations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d8a996fe1a4f26a1455e98643e994edaf34d934" target='_blank'>
              Single-cell and spatial atlas of glioblastoma heterogeneity: characterizing the PCLAF+ subtype and YEATS4’s oncogenic role
              </a>
            </td>
          <td>
            Siqi Ma, Yuwei Sun, Shaowei Zheng, Yilong Fu, Liangyu Wang, Dun Liu, Henan Jiao, Xuqiang Zhu, Xueyuan Li, Dongming Yan, Di Chen, Zi Ye
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Spatial multiomic profiling has been transforming the understanding of local tumor ecosystems. Yet, the spatial analyses of tumor-immune interactions at systemic levels, such as in liquid biopsies, are challenging. Within the last 10 years, we have longitudinally collected nearly 3,000 patient blood samples for multiplexing imaging of circulating tumor cells (CTCs) and their interactions with white blood cells (WBCs). Multicellular CTC clusters exhibit enhanced metastatic potential. The detection of CTCs and characterization of tumor immune ecosystems are constrained by (1) low frequency of CTCs in blood samples; (2) specific lineages of immune cells are not recognized by limited channels of current imaging methods, (3) reliance on labor-intensive manual analysis slows down the discovery of biomarkers for predicting therapy response and survival in cancer patients. We hypothesize that an AI-powered platform will accelerate the lineage and spatial characterization of tumor immune ecosystems for prognostic evaluations.



 Leveraging FDA-approved CellSearch technology, we collected 2,853 blood specimens longitudinally from 1358 patients with advanced cancer (breast, prostate, etc. Integrating machine learning and deep learning tools, we developed a novel platform -identification of Cell Types and Clusters (iCTC) - to automate the detection and identification of CTCs, immune cell types , and their interactions. Using machine-learning image analysis, we extracted over 270 cellular and nuclear features of cytokeratin, CD45, and DAPI expression patterns, enabling precise characterization of CTCs and WBCs including differentiation of sub-types of WBCs, CTC clusters, and their intercellular interactions with one another.



 The iCTC platform enabled automated identification of CTCs and WBCs (granulocytes, T cells, monocytes, B cells, NK) at specificity and sensitivity >0.97-1.0. It also recognized homotypic CTC clusters, heterogenous CTC-WBC clusters, and immune cell clusters, providing insights into cell morphology and spatial organization within hours for 50+ millions cells from nearly 3,000 blood tests. These features correlated with patient survival, disease progression, and treatment response. Our findings highlight the clinical significance of CTC–immune cell interactions and dynamic alterations of CTCs (singles and clusters) and underscore their potential in stratifying patients into distinct risk categories.



 This study demonstrates the transformative potential of machine learning in accelerating and enabling large-scale spatial data analyses of tumor immune ecosystem in blood biopsies and can readily extend the method to other imaging data analyses, integrating imaging data with large cohorts of patient data. By automating and enhancing the analysis of CTC-immune cell interactions, we present a robust framework for developing predictive models with direct clinical relevance. This work opens avenues for personalized treatment strategies, underscoring the impact of AI in advancing precision oncology.



 Joshua R. Squires, Yuanfei Sun, Andrew D. Hoffmann, Youbin Zhang, Allegra C. Minor, Anmol Singh, David Scholten, Hannah Ding, Chengsheng Mao, Leonidas C. Platanias, Yuan Luo, Deyu Fang, William J. Gradishar, Massimo Cristofanilli, Carsen Stringer, Huiping Liu. Deep Learning Enables Identification of Cell Types and Clusters (iCTC) in Immune Tumor Ecosystems for Prognostic Assessment in Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B049.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ce333a82d0235963171eb486a28d772f6c20a47" target='_blank'>
              Abstract B049: Deep Learning Enables Identification of Cell Types and Clusters (iCTC) in Immune Tumor Ecosystems for Prognostic Assessment in Cancer
              </a>
            </td>
          <td>
            Joshua R. Squires, Yuanfei Sun, Andrew D. Hoffmann, Youbin Zhang, Allegra C. Minor, Anmol Singh, David Scholten, Hannah Ding, Chengsheng Mao, Leonidas C. Platanias, Yuan Luo, Deyu Fang, W. Gradishar, Massimo Cristofanilli, C. Stringer, Huiping Liu
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Rationale: Breast cancer recurrences and treatment failures can be attributed to intra-tumoral heterogeneity (ITH), which is characterize by the coexistence of diverse cellular states, including cancer stem cells (CSCs), within a single tumor.Recent insights suggest that ITH arises from non-genetic dynamics, enabling tumors to adapt and evolve into a therapy-tolerant state under treatment pressure. The aim of this work is to decipher the origin of persistent radiation tolerant cells (RTP) in breast tumors and to understand their mechanisms in order to find new strategies to avoid radiation resistance. Methods: To this aim,we developed a lineage tracing system and engineeredvarious breast cancer cell lines and patient-derived xenografts totracked radiation-induced cell plasticity. We combined lineage tracing with a unique RNAi screen under irradiation to identify and functionally validate the regulators of radio-induced cell plasticity. Results: We discovered that RTP cells, which possess CSC properties, emerge from radiotherapy-induced reprogramming of non-CSCs. From the combinatorial approach of the lineage tracing and the RNAi screen under irradiation, we then identified and functionally validated the LRP4/YAP axis as a crucial regulator of radio-induced cell plasticity. We further demonstrate that overexpression of LRP4 is common in residual disease post-treatment and is associated with breast tumors of poor prognosis. Conclusions: This work has demonstrated that the LRP4/YAP axis drives radioresistance by promoting the emergence of RTP cells through radiation-induced plasticity, and that modulation of the LRP4/YAP axis is a promising strategy for sensitizing breast cancers to radiotherapy, opening up a new avenue for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ee60b3470960d374218748b117281dcd5ba1d2a" target='_blank'>
              The LRP4/YAP axis drives the radiation-tolerant persister (RTP) cell state in breast cancer
              </a>
            </td>
          <td>
            Violaine Forissier, Julien Wicinski, Martin Castagné, Guillaume Pinna, Shuheng Lin, Anaïs Grandon, Caroline Bonnet, M. Macario, Remy Castellano, Simon Valdenaire, Julien Darréon, Agnès Tallet, C. Ginestier, E. Charafe-Jauffret
          </td>
          <td>2025-06-23</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Background Small cell lung cancer (SCLC) is a lethal lung malignancy and patients are often diagnosed with distant metastasis. Nearly all patients suffer from disease relapsing with inherent chemoresistance. Lack of targeted SCLC therapies further worsens disease outcomes, making it highly desirable to identify novel and effective therapeutic targets. Methods To search for potential therapeutic targets in SCLC, we analyzed publicly available single-cell and bulk RNA-sequencing (RNA-seq) data from normal, lung adenocarcinoma, and SCLC tumor tissues. To assess the targeting potential of FOXM1, we developed various in vitro models, including DOX-On-shFOXM1 (Tet-ON) inducible stable knockdown systems. Cisplatin resistant human and murine SCLC cell lines were generated to assess the role of FOXM1 in chemotherapy resistance. Immunoblotting, immunohistochemistry (IHC), and immuno-fluorescence were used to analyze the expression of FOXM1 and target proteins. ChIP-assay was used to study protein-gene interactions. Further, multicolor flow cytometry was employed to study the effect of FOXM1 inhibition on human T cells activation and differentiation. Subcutaneous xenograft and SCLC spontaneous (RPM: RBfl/fl;TP53fl/fl;LSL-MYCT58A) mouse models were used to evaluate the efficacy of FOXM1 inhibitors. Results Single-cell as well as bulk RNA-seq data revealed that FOXM1, an oncogenic transcription factor, is overexpressed in SCLC, and it was recapitulated in human and murine SCLC tissues and cell lines. Interestingly, chemo-resistant (CR) SCLC showed a substantially higher FOXM1 expression than naïve SCLC. Silencing FOXM1 genetically or pharmacologically by FOXM1 inhibitors revealed a marked reduction in cell viability, colony formation, migration and sphere formation in naïve and CR SCLC cells. Moreover, FOXM1 inhibition induced apoptosis and cell cycle arrest in SCLC cells. Furthermore, FOXM1 inhibition in combination with first-line platinum-based chemotherapy showed synergistic anticancer effects in both xenograft and RPM mouse models of SCLC. Our RNA-seq analysis revealed that FOXM1 inhibition altered the Aurora Kinase B (AURKB) signaling pathway, which is dysregulated in SCLC. Moreover, we found FOXM1 inhibition enhanced T cell activation and supported the differentiation of CD8 + cytotoxic T cells, and T cell-mediated killing of cancer cells. Conclusions Our study demonstrates that FOXM1 targeting using small molecule inhibitors has the potential to be a novel therapeutic strategy to combat SCLC progression including chemotherapeutic resistance and reshaping the anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cf5514bc8f1dd3db6ac08b065580f4ed8890ea4" target='_blank'>
              FOXM1 targeting alters AURKB activity and reshapes antitumor immunity to curb the progression of small cell lung cancer
              </a>
            </td>
          <td>
            Md Arafat Khan, Parvez Khan, Mahek Fatima, Asad Ur Rehman, Laiba Anwar, Z. Alsafwani, Aatiya Ahmad, Mohammad Ali Abbas Zaidi, Jesse L. Cox, Areem Zahid, Sameer Mohiuddin, Sung Hoon Kim, Juan A. Santamaria-Barria, Imayavaramban Lakshmanan, B. Katzenellenbogen, John A. Katzenellenbogen, A. Ganti, Surinder K. Batra, M. Nasser
          </td>
          <td>2025-07-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of kidney renal clear cell carcinoma (KIRC) exhibits complex dynamics among immune, stromal, and malignant cells that drive immune escape (IE) mechanisms and influence clinical outcomes. Through single-cell RNA sequencing and high-dimensional weighted gene co-expression network analysis (hdWGCNA), we identified a crucial IE-related gene module most strongly associated with KIRC progression. Partitioning Around Medoids (PAM) clustering delineated two distinct IE patterns, with pattern one demonstrating prolonged patient survival. Employing advanced machine learning (ML) algorithms, we identified Cathepsin S (CTSS) as the most pivotal tumor suppressor, with elevated CTSS expression consistently predicting improved survival across multiple independent cohorts. Functional analyses revealed significant enrichment of CTSS in immune-regulatory pathways, including B/T cell activation and inflammatory activity. Mutation profiling uncovered distinct genomic alterations in 5q35.3 among CTSS-high tumors, while drug response prediction identified eight potential therapeutic agents (e.g., Navitoclax, Ibrutinib) exhibiting enhanced efficacy in these patients. Notably, CTSS expression strongly correlated with immune cell infiltration and established immunotherapy biomarkers, supporting its dual role as both a prognostic indicator and predictor of immune response. This study provides mechanistic insights into IE in KIRC and positions CTSS as a promising biomarker and therapeutic target for precision immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03267-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6364c60b20a894510d01361e9f698e8acfc1b2ab" target='_blank'>
              CTSS in the tumor microenvironment links immune escape and immunotherapy sensitivity in kidney renal clear cell carcinoma
              </a>
            </td>
          <td>
            Hanjing Zhou, Jun Ying, Xuchun Xu, Jian Huang
          </td>
          <td>2025-07-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Cervical cancer (CC) remains a significant global health challenge despite advancements in screening, HPV vaccination, and therapeutic strategies. Tumor heterogeneity, driven by epigenetic modifications, affects immune evasion, metastasis, and treatment response. Cancer-associated fibroblasts (CAFs) play a crucial role in CC progression and therapy resistance. Single-cell sequencing offers new insights but remains underutilized in CC research. This study integrates single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and deconvolution analysis to identify key genes and immunotherapy targets. By constructing a prognostic model and exploring the immune microenvironment, we aim to provide novel insights into CC pathogenesis and potential therapeutic strategies. Methods We utilized scRNA-seq, spatial transcriptomics, deconvolution analysis, and pseudotime trajectory mapping to delineate fibroblast subtypes within the tumor immune microenvironment (TIME) of CC. Functional annotations, differential gene expression profiling, cell–cell communication pathways, and transcription factor networks were systematically analyzed. A prognostic model based on bulk RNA-seq data was constructed and validated through survival analysis, with correlations to immune microenvironment characteristics. Functional experiments investigated the role of SDC1, a critical mediator of fibroblast-tumor crosstalk. Additionally, Fibroblast–tumor cell co-culture systems and functional assays were employed to investigate the paracrine role of SDC1. The CAF MYH11⁺ subpopulation was isolated via fluorescence-activated cell sorting (FACS). Multiplex immunofluorescence and immunohistochemical analyses were performed on both cultured cells and human cervical cancer tissue samples to characterize the spatial distribution and dynamic remodeling of MYH11 during stromal reorganization. Results Six distinct fibroblast subtypes were identified, including the C0 MYH11 + fibroblasts, which exhibited unique roles in stemness maintenance, metabolic activity, and immune regulation. Spatial and functional analyses revealed that the C0 subtype is central to tumor-fibroblast interactions, particularly through the MDK-SDC1 signaling axis. The prognostic model incorporating fibroblast-specific markers demonstrated robust predictive power for patient survival outcomes. Additionally, in vitro SDC1 knockdown significantly inhibited CC cell proliferation, migration, and invasion. Fibroblasts show spatially regulated heterogeneity, with activation markers enriched in the tumor zone and MYH11 highest in normal zones, indicating dynamic stromal remodeling. C0 MYH11 + CAF Promotes Tumor Cell Proliferation, Migration, and Inhibits Apoptosis via Soluble SDC1. Conclusion Our results illustrate, in some ways, the possible immunomodulatory and tumor supporting roles of CAFs in CC TIME and highlight the possibility that the MDK-SDC1 pathway is a promising therapeutic target. This study not only promotes a partially new understanding of temporal heterogeneity in CC, but also provides a possible reference base for the development of new biomarkers and immunotherapy approaches to improve clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03432-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d1387a936b1ef6d7e045f6c7bcff60d568a6f7" target='_blank'>
              Deciphering the tumor immune microenvironment: single-cell and spatial transcriptomic insights into cervical cancer fibroblasts
              </a>
            </td>
          <td>
            Zhiheng Lin, Youwei Zhou, Zhenran Liu, Wenyang Nie, Hengjie Cao, Shengnan Li, Xuanling Li, Lijun Zhu, Guangyao Lin, Yanyu Ding, Yi Jiang, Zuxi Gu, Lianwei Xu, Zhijie Zhao, Hua-Bao Cai
          </td>
          <td>2025-07-05</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Summary/Abstract Glioblastoma stem cell (GSC) cultures are initiated from glioblastoma (GBM) surgical resection tissue. They can capture and propagate key GBM primary tumor molecular and cellular features. We have deeply characterized four isocitrate dehydrogenase (IDH)-expressing (or IDH+) GSC cultures from unrelated adults to serve as cellular models for the majority of adult primary GBM. We demonstrate that GSC cultures can be continuously propagated in defined, serum-free media and 5% oxygen without requiring specialized growth substrates; have well-defined genomic and mtDNA variants and gene/protein expression profiles; and highly reproducible dose-survival curves when treated with the GBM standard-of-care therapies of ionizing radiation (IR) and temozolomide (TMZ). We also illustrate how expressed lentiviral barcodes, mtDNA variants and single cell gene expression profiling can be used to define and track cellular heterogeneity over 40 days after IR treatment. These well-characterized IDH+ GSC cultures can support many high throughput in vitro assay formats, including xenograft, organoid and other GBM disease modeling protocols. They should prove a useful resource to better understand GBM biology, and to identify new and more effective GBM therapies and treatment regimens. Highlights Glioblastoma (GBM)-derived IDH-expressing Glioma Stem Cell (GSC) cultures can capture and propagate GBM genomic variants, gene and protein expression programs. Mitochondrial DNA (mtDNA) variants identified by high accuracy Duplex DNA sequencing are abundant and sub-clonally organized in GSC cultures. GSC cultures have highly reproducible dose-survival curves for ionizing radiation and temozolomide, the GBM standard-of-care therapies. GSC culture cellular heterogeneity can be captured, characterized and tracked by using expressed lentiviral barcodes, mtDNA variants and scRNA sequencing. eTOC blurb IDH-expressing glioma stem cell cultures (GSCs) are experimentally tractable and versatile glioblastoma (GBM) cellular disease models. Deeply characterized GSC cultures can enable new work to understand GBM biology, and help identify new and more effective GBM therapies and treatment regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b852883dfd3891f1795ac6382d7546d2b8963f5d" target='_blank'>
              Cellular heterogeneity and therapeutic response profiling of human IDH+ glioma stem cell cultures
              </a>
            </td>
          <td>
            N. Chambwe, Scott R Kennedy, Brendan F. Kohrn, Pavlo Lazarchuk, Mario Leutert, Guangrong Qin, Bahar Tercan, M. Sanchez-Contreras, Weiliang Tang, Jerome J. Graber, Patrick J. Paddison, Judit Villén, Ilya Shmulevich, Raymond J Monnat
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Introduction Pancreatic cancer is a highly malignant tumor of the digestive system with a dismal prognosis. Despite advances in diagnosis and treatment, overall survival remains extremely low. Early diagnostic markers and an improved understanding of tumor-microenvironment interactions are essential for developing more effective therapies. Methods We analyzed 74 single-cell RNA sequencing (scRNA-seq) samples, performing unsupervised clustering and marker-gene expression profiling to define major cell types. Large-scale chromosomal copy-number variation (CNV) analysis distinguished malignant from non-malignant ductal cells. Non-negative matrix factorization (NMF) identified stage-associated gene modules, which were integrated with TCGA bulk-RNA data and machine-learning feature selection to pinpoint candidate prognostic genes. Two independent cohorts were used for validation. Regulatory network inference (pySCENIC) and ligand–receptor interaction analysis (CellPhoneDB) explored cross-talk between malignant cells and macrophages. Finally, in vitro knockdown of CTSV assessed its functional role in pancreatic cancer (PAC) cell proliferation and migration. Results Three prognosis-related genes—ANLN, NT5E, and CTSV—were selected based on their strong association with clinical stage and validated in external datasets. High expression of these genes correlated with poorer overall survival and an increased infiltration of M0 macrophages. CellPhoneDB predicted significant interactions between high-expression malignant ductal cells and M0 macrophages via CXCL14–CXCR4 and IL1RAP–PTPRF axes, with SPI1 identified as an upstream regulator of IL1RAP. In vitro CTSV knockdown significantly inhibited PAC cell proliferation and migration. Discussion Our integrative single-cell and bulk-RNA workflow identifies ANLN, NT5E, and CTSV as novel prognostic biomarkers in pancreatic cancer and highlights a pro-tumorigenic interaction between malignant ductal cells and macrophages. Targeting CTSV or disrupting CXCL14–CXCR4 and IL1RAP–PTPRF signaling may offer new therapeutic avenues for PAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50ce5676a35dfc57b1b078e8894ee25f419175d" target='_blank'>
              Integration of single-cell RNA and bulk RNA sequencing revealed malignant ductal cell heterogeneity and prognosis signatures in pancreatic cancer
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Tumor-associated macrophages (TAMs) are critical regulators of the hepatocellular carcinoma (HCC) microenvironment, yet their epigenetic heterogeneity and regulatory programs remain poorly defined. Methods: We performed integrative analysis on single-cell RNA-seq and ATAC-seq profiling of HCC patients to dissect TAM subtypes at high resolution. By correlating chromatin accessibility with gene expression, we identified cell-type-specific candidate cis-regulatory elements (CREs). TAM subsets with prognostic significance were determined through integration with HCC clinical cohorts. Pseudotime and multi-regional analyses were used to uncover regulatory trajectories underlying macrophage phenotypic transitions. The identification framework of a super-enhancer (SE) was constructed, and potential therapeutic targets were prioritized using drug–gene interaction data. Results: We delineated the regulatory landscape of TAMs in HCC, revealing cell-type-specific chromatin accessibility patterns underlying TAM heterogeneity. The 65,342 CREs linked to gene expression were identified, with distal CREs contributing most to cell-type-specific regulation. Notably, SPP1+ TAMs were found to be enriched in tumor cores and associated with poor prognosis in HCC. Liver-resident Kupffer cells showed progressive loss of the core transcription factors SPIC and MAFB, suggesting a potential transition into SPP1+ TAMs under tumor pressure. We identified 133 SPP1+ TAM-specific SEs and constructed a TF–SE–target gene regulatory network. Notably, 13 target genes showed higher drug–gene interaction effects, highlighting their therapeutic potential. Conclusions: This study provides the chromatin accessibility map of TAMs in HCC and reveals how distal CRE-driven transcriptional programs shape TAM states. Our findings lay the foundation for understanding the epigenetic regulation of TAM heterogeneity and nominate potential targets for TAM-directed immunotherapy in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e02327a10ea72222cca675d378cb3f0e78ed3b3" target='_blank'>
              Integrated Single-Cell Analysis Dissects Regulatory Mechanisms Underlying Tumor-Associated Macrophage Plasticity in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Yu Gu, Wenyong Zhu, Zhihui Zhang, Huiling Shu, Hao Huang, Xiao Sun
          </td>
          <td>2025-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background The dynamics of single-nucleotide variants (SNVs) play a critical role in understanding tumor development, yet their influence on shaping tumor microenvironments remains largely unexplored. Spatial transcriptomics offers an opportunity to map SNVs within the tumor context, potentially uncovering new insights into tumor microenvironment dynamics. Results This study developed SpatialSNV for identifying effective SNVs across tumor sections using multiple spatial transcriptomics platforms. The analysis revealed that SNVs reflect regional tumor evolutionary traces and extend beyond RNA expression changes. The tumor margins exhibited a distinct mutational profile, with novel SNVs diminishing in a distance-dependent manner from the tumor boundary. These mutations were significantly linked to inflammatory and hypoxic microenvironments. Furthermore, spatially correlated SNV groups were identified, exhibiting distinct spatial patterns and implicating specific roles in tumor–immune system crosstalk. Among these, critical SNVs such as S100A11L40P in colorectal cancer were identified as tumor region–specific mutations. This mutation, located within exonic nonsynonymous regions, may produce neoantigens presented by HLAs, marking it as a potential therapeutic target. Conclusions SpatialSNV represents a promising framework for unraveling the mechanisms underlying tumor–immune crosstalk within the tumor microenvironment by leveraging spatial transcriptomics and SNV-based tissue domain characterization. This approach is designed to be scalable, integrative, and adaptable, making it accessible to researchers aiming to explore tumor heterogeneity and identify therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32eb52bad2a391e12368299093f1326ca08dc55e" target='_blank'>
              SpatialSNV: A novel method for identifying and analyzing spatially resolved SNVs in tumor microenvironments
              </a>
            </td>
          <td>
            Yi Liu, Fan Zhu, Xinxing Li, Xiangyu Guan, Yong Hou, Yu Feng, Xuan Dong, Young Li
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background Pancreatic angiosarcoma is a rare and highly aggressive tumor originating from lymphatic or vascular endothelial cells, with poor prognosis and few effective treatments. In this study, we aimed to characterize the tumor ecosystem of metastatic pancreatic angiosarcoma, along with its potential treatment strategies. Methods Single-cell RNA-sequencing and bioinformatics analysis were performed on samples obtained from one patient, including at total of 16,841 cells from pancreatic angiosarcoma liver metastasis and adjacent normal liver tissue. Results Pancreatic angiosarcoma cells exhibited marked upregulation of nuclear factor kappa-B (NF-κB), hypoxia-inducible factor 1 (HIF-1), and myelocytomatosis oncogene (MYC) proto-oncogene signaling pathways, while presenting limited upregulation of actionable therapeutic targets except for cyclin-dependent kinase 4 (CDK4) and epidermal growth factor receptor. Several immune checkpoint genes, including cytotoxic T-lymphocyte-associated protein 4 (CTLA4), lymphocyte-activation gene 3 (LAG3), programmed cell death protein 1 (PDCD1), and cluster of differentiation 86 (CD86), were upregulated in tumor-infiltrating T cells, natural killer (NK) cells, and myeloid cells. Furthermore, intercellular interaction profiling demonstrated enhanced activity of the programmed death-ligand 1 (PD-L1) and CD86 signaling pathways within the tumor microenvironment. The gene-set scores of T/NK-cell exhaustion, regulatory T cell, and macrophage angiogenesis were significantly higher in tumor tissues compared with adjacent normal tissues. However, the phagocytosis scores of macrophages within the tumor-infiltrating region were significantly lower than those in the adjacent normal tissues. Conclusions Our findings outlined an immunosuppressive and angiogenic tumor ecosystem in pancreatic angiosarcoma liver metastasis, suggesting that pancreatic angiosarcoma may be insensitive to most targeted therapies. Conversely, immunotherapies targeting LAG3, PD-L1, and CD86 (e.g. isatuximab, Opdualag, and abatacept) and anti-angiogenic agents may be therapeutically effective and worthy of subsequent exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fffbe9dc01181c5c05c578e4d7eb55a7b801a4e" target='_blank'>
              Single-cell transcriptomic landscape indicates the potential role of immunotherapy in metastatic pancreatic angiosarcoma
              </a>
            </td>
          <td>
            Yizhen Yang, Luohai Chen, Man Liu, Xiaoxuan Lin, Sui Peng, Yubin Xie, Zhirong Zeng, Minhu Chen, Ning Zhang
          </td>
          <td>2025-06-16</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Insufficient infiltration of cytotoxic T cells into solid tumors remains a critical obstacle in cancer immunotherapy. Despite extensive efforts to comprehend the mechanisms governing this limited infiltration, few studies have focused on the evolution of T cell motility behavior after co-culture. In this study, we combined quantitative cell trajectory analysis, computational modeling, and bulk/single-cell RNA sequencing to systematically characterize the impact of cell interactions. We reveal that in a 2.5D co-culture system with multiple cancer-cell clusters, cancer-specific T cells exhibit increased directional persistence, which facilitates their efficient searching of cancer-cell clusters. Additionally, these T cells form prolonged interactions with cancer cells, which is the most crucial factor for their accumulation on cancer-cell clusters. Furthermore, post-interaction, a cancer-cell subpopulation displays immunosuppressive traits, reducing T cell attractant expression, and undergoing epithelial-to-mesenchymal transition. These findings offer valuable insights into improving immunotherapy efficacy and tackling T cell infiltration challenges in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0a380330210bd455d4b1fb482cbeaabdb64b1d" target='_blank'>
              Deciphering antigen-specific T cell navigation tactics and cancer immune evasion in co-cultures
              </a>
            </td>
          <td>
            Xinyue Li, Taoli Jin, Lisha Wang, Senbin Wu, Yerong Liu, Ming Li, Weijing Han, Xuefei Li
          </td>
          <td>2025-07-31</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) play dual roles in cancer, either promoting or suppressing tumor progression, complicating therapeutic approaches. TAMs include recruited macrophages (recMacs), derived from circulating monocytes, and tissue-resident interstitial macrophages (IMs). We recently identified a heterogeneous population of chemokine-expressing IMs, including subsets that support tertiary lymphoid structure (TLS) formation during lung inflammation. Here, we show that IMs can be either pro- or anti-tumorigenic, depending on the subset. Using Pf4 Cx3cr1 mice to deplete CD206hi IMs expressing Cxcl13, Cxcl9, and Cxcl10, we demonstrate their essential role in TLS formation, lymphocyte recruitment, and tumor suppression in melanoma and lung adenocarcinoma. In contrast, Ccl2-expressing IMs promote tumor growth by recruiting pro-tumorigenic recMacs. Spatial transcriptomics confirmed the distinct localization and chemokine profiles of these subsets. Finally, CCR5 blockade with the FDA-approved inhibitor Maraviroc during neoantigen vaccination improved tumor control by preventing the migration of immunosuppressive, antigen-presenting recMacs (moDCs). These findings support the development of macrophage-targeted therapies by identifying pro-tumorigenic subsets and recMac trafficking as actionable targets, while preserving macrophage populations that sustain anti-tumor immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/313d8c71525c97f65bdf0888b072625e1176bbdb" target='_blank'>
              The Dichotomy of Tumor Control by Recruited and Resident Tumor-Associated Macrophages
              </a>
            </td>
          <td>
            C. Jakubzick, Soubhik Ghosh, Xin Li, Kavita Rawat, Aishwarya Dighal, Stephanie Kalinowski, F. Iv, Carol Ringelberg
          </td>
          <td>2025-07-04</td>
          <td>Research Square</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) progression is driven by a complex interplay of immunosuppressive and inflammatory mechanisms involving various cell types, functional states, and extracellular milieu. To unravel this intricate relationship, we took a systems biology approach using high resolution spatial distributions of multiple key markers in clinically diverse CRC tumors. We employed Multiplexed Ion Beam Imaging (MIBI-TOF) to capture 65 protein markers across 3,700 fields of view from 250 CRC tumors. This allowed us to deeply characterize the phenotypes and functional states of 5.7 million cells. To incorporate extracellular geography information, we developed a computational approach named CellExt. This quantified and modeled the gradated abundance of key structural and functional proteins found extracellularly, such as fibronectin, granzymeB and vimentin, from each cell boundary to a 40-micron distance in pixelwise steps. CellExt detected a distinct immunosuppressive modality characterized by extracellular PDL1, TNF-related apoptosis-inducing ligand (TRAIL), and matrix metalloproteinase-9 (MMP9). Investigating the cell types involved led us to map the potential receivers and emitters by linking extracellular trends to cellular enrichment and gene expression. We found that Pankeratin-high epithelial cells were acting as TRAIL receivers with downstream activation of non-canonical pro-tumorigenic signaling through S6 phosphorylation. In parallel, we found that T cell suppressive CD40+ antigen-presenting B cells with PD1 expression were acting as MMP9 emitters. Additionally, CellExt was able to identify two spatially segregated inflammatory modalities: IFNgamma cytokine with mucin protein, MUC5AC and prostaglandin synthase, cyclooxygenase-2 with extracellular matrix component, Collagen1A1. This suggests a strong inflammatory role for Collagen1A1 in CRC. We then contextualized our identified modalities by genetic background, tumor stage and CMS subtype of CRC tumors. Taken together, our approach revealed novel axes of immunosuppression and inflammation and puts forward potential extracellular targets for therapeutic intervention in specific CRC patient subsets.



 Reema Baskar, Vairavan Lakshmanan, Audrey Lee, Wilfred Wong, Dane Bagaoisan, Yuhan Peng, Marc Bosse, Ferda Filiz, Christine Fullaway, Sean Bendall, Iain Beehuat Tan, Shyam Prabhakar. Spatial Profiling of Colorectal Cancer Extracellular Milieu Reveals Novel Axes of Immunosuppression and Inflammation [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P38.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18eea1ae1312b735cfb9925fbcce433bb07c964b" target='_blank'>
              Abstract P38: Spatial Profiling of Colorectal Cancer Extracellular Milieu Reveals Novel Axes of Immunosuppression and Inflammation
              </a>
            </td>
          <td>
            Reema Baskar, Vairavan Lakshmanan, Audrey Lee, Wilfred Wong, Dane Bagaoisan, Yuhan Peng, Marc Bosse, Ferda Filiz, C. Fullaway, Sean C. Bendall, Iain Beehuat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Tumormicroenvironment (TME) is a complex milieu of multiple cell types mainly including tumor cells, immune cells, endothelial cells and stromal cells. Immune cell profile within the TME can determine success of immunotherapies, indicate therapeutic efficacy and determine any potential therapeutic toxicity in various malignancies including breast cancer. Spatially interrogating these immune cells is crucial in studying immune-immune and immune-tumor interactions. Characterizing immune cells require assessing cell marker expression using protein detection and activation markers using RNA detection. To address this, we have developed a high throughput RNAscope™ assay on the BOND RX to spatially visualize RNA and protein markers on the same slide. We utilized the new RNAscope™ Multiomic LS assay that can detect up to 6 RNA and protein targets in breast cancer FFPE tissue. The TSA-based amplification strategy offers signal boost for both RNA and protein targets. To optimize protein detection, the workflow is completely protease-free there by preserving antigen integrity and tissue morphology. The assay can be customized to include any target RNA and proteins of interest. Here, we demonstrate the use of our pre-conjugated antibody panel which includes CD8, CD4, FoxP3 and PanCK to visualize tumor infiltrating lymphocytes (TILs). We also utilized unconjugated primary antibodies for CD68 and CD163 to detect tumor macrophages in multiple tumor tissues. Infiltrating Cytotoxic T cell lymphocytes were detected using CD8, GZMB and IFNG co-expression. Regulatory T cells were detected using CD4, FOXP3 co-expression. PanCK was used as a tumor marker to delineate tumor from stroma. Similarly, tumor associated macrophages were detected using CD163, CD68 , IL10 and IL-1B expression. We also identified distinct M1 and M2 macrophage populations in the breast tumor sample.
 Citation Format: Anushka Dikshit, Debia Wakhloo, Julia Yu, Paul Liu, Sonali Deshpande. Spatial mulitomic assay for assessing immune cell phenotype and function in the tumor microenvironment [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7cfc6bee0167aa11eb70e6990a48ce515a4adb5" target='_blank'>
              Abstract P3-06-19: Spatial mulitomic assay for assessing immune cell phenotype and function in the tumor microenvironment
              </a>
            </td>
          <td>
            A. Dikshit, Debia Wakhloo, Julia S. Yu, Paul Liu, Sonali A Deshpande
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Invasive lobular breast cancer (ILC) represents 10-15% of all breast cancer cases but remains understudied. Lack of e-cadherin is thought to be the basis for the distinct single-file growth pattern of ILC. ILC exhibits late recurrence, commonly metastasizing to the peritoneum and bone with poor outcome. Primary ER+ ILC tumor cells exhibit wide variation in nuclear GR expression, ranging from negative to strongly positive. We previously reported better patient outcome in ER+ breast cancers with high glucocorticoid receptor (GR) expression (Pan et al 2011). We hypothesize that GR expression may also impact outcome in ILC. Crosstalk between tumor and stromal cells leads to evolution of a tumor microenvironment (TME) favorable for growth and/or invasion. Because of the characteristic histology of ILC, we expected the TME to be unique compared to other breast cancers. We recently reported that GR+ ILC shows reduced proliferation to GR- but exhibit an enhanced metastatic phenotype (Porter et al 2023). Hence, we predicted that tumor cell-intrinsic GR expression will impact the ILC TME. We used an integrated approach to explore in-depth the molecular profile of primary ILC and to study interactions between tumor and neighboring cells. We measured average gene and protein expression in regions of interest on primary ILC tissue sections using nanoString GeoMx DSP with whole transcriptome (WTA), protein modules and IO Proteome Atlas. We also performed single-cell spatial transcriptomics for a subset of those patients leveraging the CosMx platform. GR+ ILC tumor cells exhibited distinct transcriptional profiles with enrichment of RNA processing, translation pathways and oxidative phosphorylation. MHC class II antigen presentation pathways were downregulated. The most significantly enriched pathways in GR- tumor cells and TME were collagen synthesis and extracellular matrix (ECM) remodeling. Sflomos et al previously reported that ILC cells contribute to their own extracellular matrix, but we were surprised to observe the magnitude of increased expression in cells without GR. Gene expression in the TME strongly suggested of a much greater presence of cancer-associated fibroblasts (CAFs) surrounding GR- ILC. Proteome assays supported the WTA data, showing more fibronectin and type I collagen in GR- tumors. Also consistent with transcriptomic data, protein DSP indicated suppression of antigen presentation in GR+ ILC evidenced by lower expression of type I and type II MHC proteins. Next, CosMx single cell in situ transcriptomics allowed us to reaffirm observations from DSP, and to examine expression at the single cell level, assign cell types and examine interactions between cells. GR+ and GR- tumor cells were transcriptionally distinct, as shown by clustering, and the abundance of certain cell types in their surrounding microenvironment were notably different. We confirmed a higher presence of CAFs transcriptionally similar to myofibroblasts that were in close proximity to GR- tumor cells. We also observed downregulation of MHC class I and II genes at the single cell level in GR+ compared to GR- tumor cells and low representation of antigen presenting cells in the stroma. Our combined data supports repression of antigen presentation pathways in GR+ ILC accompanied by a relative exclusion of CAFs. Evidence suggests that tumor cells as well as stromal cells contribute to the modeling of the ECM, and crosstalk between them leads to the establishment of a favorable environment to support ILC tumor cell survival, growth and invasion.
 Citation Format: Lynda Bennett, Sunati Sahoo, Cheryl M. Lewis, Indu Raman, Guanchun Chen, Chengsong Zhu and Suzanne D. Conzen. GeoMx DSP and CosMx single cell spatial transcriptomics for molecular characterization of invasive lobular breast cancer cells and their microenvironment [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS18-03.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1096f0e514cebfcc24fba97cddd90feca48a8b9" target='_blank'>
              Abstract PS18-03: GeoMx DSP and CosMx single cell spatial transcriptomics for molecular characterization of invasive lobular breast cancer cells and their microenvironment
              </a>
            </td>
          <td>
            Lynda Bennett, S. Sahoo, Cheryl Lewis, Indu Raman, Guanchun Chen, Chengsong Zhu, Suzanne D. Conzen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Tumorigenesis is a microevolutionary process in which heterogeneous tumor cells adapt within a complex microenvironment. Current tumor omics analyses often focus exclusively on tumor samples, overlooking the valuable insights that normal tissues can provide. To address this gap, we introduce VaDTN (Variational Autoencoder–Derived Tumor-to-Normal), a pan-cancer framework that integrates transcriptomic data from both tumor and normal samples into a unified latent space. By measuring each tumor’s “distance” from a normal reference within this latent space, VaDTN reveals subtle molecular shifts linked to tumor evolution and heterogeneity. We applied VaDTN to six representative cancers (SKCM, BRCA, LIHC, LUSC, STAD, and PAAD), identifying distinct subtypes characterized by unique transcriptional profiles. Notably, four cancer types (SKCM, BRCA, LIHC, and STAD) displayed significant survival stratification based on these subtype groupings, underscoring the clinical relevance of the distance-based approach. This reference-centered perspective thus provides a refined lens for dissecting intra-tumor diversity and guiding potential precision oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dc636c3386b69302e143969833b154f54e564be" target='_blank'>
              Novel cancer subtyping method guided by tumor-normal sample in latent space of transcriptomic variational autoencoder
              </a>
            </td>
          <td>
            Hongzhi Wang, Yu Zhang, Dandan Zhang, Min Luo
          </td>
          <td>2025-07-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Globally, endometrial cancer continues to impact a significant number of women. Immunotherapy provides those suffering from advanced or relapsed disease hope, but an important barrier is still the absence of trustworthy predictive biomarkers. To tackle this challenge, single-cell sequencing and spatial transcriptomics (ST) are increasingly applied. In cervical cancers of the no specific molecular profile (NSMP) subtype accompanied by p53 mutations. In many cases, the tumor microenvironment (TME) in endometrial cancer exhibits strong immunosuppression or poor immune cell infiltration, often leading to worse clinical outcomes. Single-cell sequencing reveals cellular heterogeneity and helps identify potential therapeutic targets and predict treatment responses. Conversely, ST assists in determining biomarkers that influence the effectiveness of immunotherapy by capturing the spatial organization of tumors. When combined, these technologies allow for integrated multi-omics analysis that aids in the development of immunotherapies, prognostication, and diagnosis. But there are still moral and legal issues. Clinicians may be able to improve outcomes for patients who don’t respond well to current immunotherapies by utilizing these combined approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f087dd53dee17003d5211dac4cb4360cce4847" target='_blank'>
              Single-cell technologies and spatial transcriptomics: decoding immune low - response states in endometrial cancer
              </a>
            </td>
          <td>
            Yumeng Li, Hua Qiu, Zhenzhen Zhao, Fanghua Qi, Pingping Cai
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Cancer development and progression are driven by the accumulation of somatic genetic alterations, which occur in a specific temporal order. However, how the order of mutations impacts cancer phenotypes of solid tumors remains poorly understood. To address this, we developed a novel computational framework, IMOP-Cancer (Identifying Mutation Order Pairs in Cancer), to identify mutation gene pairs whose order influences cancer phenotypes. We applied IMOP-Cancer to The Cancer Genome Atlas-Lung Adenocarcinoma (TCGA-LUAD) cohort and identified 446 key mutation order pairs, with 34 pairs significantly associated with prognosis. Mutation order impacts cancer phenotypes, as demonstrated by CSMD3 and PTPRD (tumor proliferation) and TP53 and NAV3 (immune modulation), with effects validated in four independent datasets. We further presented the impact of mutation pairs on cancer phenotypes through case studies in the TCGA cohorts of bladder urothelial carcinoma (BLCA), and colon adenocarcinoma (COAD). We extended this analysis to 33 cancer cohorts from TCGA portal, identifying 106 034 critical mutation pairs across 17 cancers, with 3036 pairs co-occurring in multiple cancers. Shared mutation pairs across cancers also showed distinct effects on cancer phenotype. Our study highlights the importance of mutation order in cancer progression and diversity, offering new insights into the temporal dynamics of co-occurring mutations and paving the way for personalized treatment strategies and improved diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348ae5004fc1076373867ea177acfe998124027" target='_blank'>
              IMOP-Cancer: identifying mutation order pairs impacting cancer phenotypes
              </a>
            </td>
          <td>
            Yijing Zhang, Shaobo Kang, Renjie Dou, Wanmei Zhang, Yuanyuan Liu, Yang Wu, Dongxue Li, Fangfang Fan, Yanyan Ping
          </td>
          <td>2025-07-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) continues to be a leading cause of cancer-related deaths globally, primarily due to its late diagnosis and the complex nature of its tumor microenvironment (TME). Within this environment, cancer-associated fibroblasts (CAFs) play a crucial role in regulating NSCLC progression and therapeutic resistance. Recent advancements in single-cell and spatial technologies have uncovered significant heterogeneity among CAFs, revealing distinct subpopulations with varying cellular origins, phenotypes, and functions. Besides, the role of small extracellular vesicles (sEVs) in facilitating bidirectional communication highlights functional importance of CAF derived sEVs in mediating the crosstalk between cancer and TME, implicating the potential of CAF-sEVs to be un-invasive diagnostic biomarkers for NSCLC. Despite such new insights, challenges remain exist, particularly concerning the mechanisms that drive CAF plasticity, the interactions between specific CAF subsets with cancer cells or immune cells, and the translational significance of CAF-sEVs. In this review, we summarize the latest advances in our understanding of CAF heterogeneity, the functional roles and clinical relevance of CAFs and their secreted sEVs in driving NSCLC, and the translational landscape of CAF-targeted strategies. By critically evaluating the most recent evidence, this review provides clinically relevant insights and highlights future directions for overcoming CAF-mediated barriers to therapy. Our aim is to facilitate the development of more precise, biomarker-driven, and safe approaches for targeting CAFs, ultimately improving personalized treatment and outcomes for patients with NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82d1bb104cbbe26ccaae9f632691df643466beac" target='_blank'>
              Hidden forces: the impact of cancer-associated fibroblasts on non-small cell lung cancer development and therapy
              </a>
            </td>
          <td>
            Ziheng Wu, Meilin Luo, Huiyi Hu, Zhijun Jiang, Yinan Lu, Zhi-Jie Xiao
          </td>
          <td>2025-07-25</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Salivary carcinoma ex pleomorphic adenoma (CXPA) is a carcinoma that develops from benign pleomorphic adenoma, and it is characterized by aggressiveness and poor prognosis. Recent advancements in cancer stem cell (CSC) research have identified CSCs as critical contributors to tumor carcinogenesis due to their unlimited self-renewal ability. Single-cell RNA sequencing has enabled a comprehensive analysis of CSC heterogeneity. However, CSCs in CXPA remain largely unexplored. In this study, we constructed a comprehensive atlas of CXPA at single-cell resolution and demonstrated that CXPA exhibited intratumoral heterogeneity. We identified four tumor epithelial cell subgroups characterized by epithelial development, cell cycle, extracellular matrix remodeling, and salivary secretion signatures. ECT2+ tumor cells in the cell cycle cluster were identified as CSCs in CXPA. Analysis of tumor tissues from patients with CXPA revealed that ECT2 expression was strongly associated with invasiveness and prognosis. Targeting the ECT2 gene significantly inhibited tumor sphere formation and aggressiveness in CXPA cell lines. Our findings provide new insights into CXPA diversity and origin, with substantial clinical implications for targeting the ECT2 gene in CXPA treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061f8c8957e80cfaf42fd19577841c361d71fc6b" target='_blank'>
              ECT2+ cell group acts as cancer stem cell in malignant pleomorphic adenoma
              </a>
            </td>
          <td>
            Wan-qun Chen, Yang Wang, Zhengduo Yao, T. Gu, Yuhua Hu, Nannan Leng, Chunye Zhang, Zhen Tian, Jiang Li
          </td>
          <td>2025-06-17</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Microglial spatial heterogeneity remains a crucial yet not fully answered question in the context of potential cell-directed therapies for Alzheimer’s disease (AD). There is an unclear understanding of the dynamics of distinct microglia states adjacent to or far from amyloid-beta (Aβ) plaques and their contributions to neurodegenerative diseases. Here we combine multicolor fluorescence cell fate mapping, single-cell transcriptional analysis, epigenetic profiling, immunohistochemistry and computational modeling to comprehensively characterize the relation of plaque-associated microglia (PAM) and non-plaque-associated microglia (non-PAM) in a mouse model of AD. We show that non-PAM are a distinct and highly dynamic microglial state, transitioning to PAM after Aβ plaque deposition in female mice. Non-PAM modulate the cell population expansion in response to amyloid deposition and rapidly respond to environmental cues. Indeed, Csf1 signaling modulates non-PAM-to-PAM transition during disease progression. Our data suggest that microglia states and their dynamics between each other can have distinct contributions to disease, and they may be targeted for the treatment of AD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cef9a3fe9de367ad9376482f5aa956c82f8113ee" target='_blank'>
              Response of spatially defined microglia states with distinct chromatin accessibility in a mouse model of Alzheimer’s disease
              </a>
            </td>
          <td>
            A. Ardura-Fabregat, Lance Fredrick Pahutan Bosch, Emile Wogram, Omar S. Mossad, R. Sankowski, Philipp Aktories, Lina Kieger, James Cook, Dilara Hasavci, Hatice Ulupinar, Daniel Brock, Fang Wang, Nicola Iovino, Samuel Wald, Sebastian Preissl, Bahtiyar Yilmaz, Daniel Schnepf, Andrew J. Macpherson, Thomas Blank, Katrin Kierdorf, M. Prinz
          </td>
          <td>2025-07-14</td>
          <td>Nature Neuroscience</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Up until present day, chimeric antigen receptor (CAR)-T cell therapy has only been approved for hematological malignancies, as CAR-T cells do not show comparable efficacy in solid tumors. Therefore, understanding the features of the tumor microenvironment (TME), is key to improve efficacy of adoptive cell therapies (ACTs) against solid tumors. In this context, robust workflows, which dissect the complex interactions between CAR-T cells and the TME are still lacking. To address this need, we have established an ex vivo workflow co-culturing tissue slices from patient tumor resections with CAR-T cells. The workflow is composed of assessing several complementary attributes, such as cytokine release via flow cytometry, quantification of cell infiltration into the tumor and assessment of the regions of the tissue slice the CAR-T cell infiltrate into by using the MACSima™ imaging cyclic staining technology. Using this workflow it is possible to observe the behavior of CAR-T cells within the tumor and its TME, their infiltration into distinct tumor compartments, as well as to dissect the underlying molecular mechanisms that drive T cell migration, thanks to MACSima™ multiplexing technology and its ability to image several markers at the same time. Assessment of ovarian carcinoma tissue slices revealed substantial release of specific cytokines and increased infiltration of T cells in the tumor areas when CAR-T cells were added to the tissue slices as compared to non-engineered T cells. The establishment of this novel approach will enable researchers to better characterize the interaction between CAR-T cells and the TME. Tissue slices present an intrinsic heterogeneity, which is indeed an advantage compared to other in vitro models but can turn itself into complex results interpretation. Therefore, we recommend that any conclusion derived from this assay should be verified with complementary models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e80f902b452985b81dfcba4e7339eba4450d5adf" target='_blank'>
              Precision-cut tumor tissue slices, a novel tool to study the tumor microenvironment interactions with chimeric antigen receptor (CAR) T cells
              </a>
            </td>
          <td>
            Valeria Durante, Alina Wittwer, Benjamin Theek, Manuel Martinez-Osuna, E. Donnadieu, O. Hardt, Dominik Eckardt, A. Bosio, Sonja Schallenberg, Christoph Herbel
          </td>
          <td>2025-08-08</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Pulmonary endothelial cells (PECs) are indispensable for sustaining lung microenvironmental homeostasis and exert significant influence across a spectrum of pulmonary pathologies. Single-cell RNA sequencing (scRNA-seq) has fundamentally transformed conventional paradigms surrounding PECs, unveiling novel perspectives on their roles in both physiological and pathological lung conditions. This technology provides critical insights into the phenotypic diversity and distinct molecular signatures of PECs, underscoring their substantial heterogeneity in structure, function and gene expression, which is contingent upon their spatial localization within the lung microenvironment. The advancements in scRNA-seq have catalyzed remarkable progress in the therapeutic management of pulmonary pathophysiology, facilitating breakthroughs in the identification of cellular subpopulations, functional characterization and discovery of innovative therapeutic targets. In this review, we systematically synthesize the markers and subclusters of PECs as delineated by scRNA-seq, elucidate their applications in normal and pathological lung contexts, and propose future directions regarding molecular mechanisms and therapeutic interventions targeting PECs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e54f05d284bc32e0555036f8e2514bea415da1f7" target='_blank'>
              Single-cell RNA sequencing: new insights for pulmonary endothelial cells
              </a>
            </td>
          <td>
            Ying Yang, Mengyuan Wang, Xiao Qiu, Rui Yang, Chengfang Yao
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) are functionally diverse stromal regulators that orchestrate tumor progression, metastasis, and therapy resistance through dynamic crosstalk within the tumor microenvironment (TME). Recent advances in single‐cell multiomics and spatial transcriptomics have identified conserved CAF subtypes with distinct molecular signatures, spatial distributions, and context‐dependent roles, highlighting their dual capacity to promote immunosuppression or restrain tumor growth. However, therapeutic strategies struggle to reconcile this functional duality, hindering clinical translation. This review systematically categorizes CAF subtypes by origin, biomarkers, and TME‐specific functions, focusing on their roles in chemoresistance, maintenance of stemness, and formation of immunosuppressive niches. We evaluate emerging targeting approaches, including selective depletion of tumor‐promoting subsets (e.g., fibroblast activation protein+ CAFs), epigenetic reprogramming toward antitumor phenotypes, and inhibition of CXCL12/CXCR4 or transforming growth factor‐beta signaling pathways. Spatial multiomics‐driven combinatorial therapies, such as the synergistic use of CAFs and immune checkpoint inhibitors, are highlighted as strategies to overcome microenvironment‐driven resistance. By integrating CAF biology with translational advances, this work provides a roadmap for developing subtype‐specific biomarkers and precision stromal therapies, directly informing efforts to disrupt tumor‐stroma coevolution. Key concepts include spatial transcriptomics, stromal reprogramming, and tumor‐stroma coevolution, offering actionable insights for both mechanistic research and clinical innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d8f3a8f9589a88ea165cf34934eda0905c9005" target='_blank'>
              Cancer‐Associated Fibroblasts: Heterogeneity, Cancer Pathogenesis, and Therapeutic Targets
              </a>
            </td>
          <td>
            Yutao Li, Qingyun Liu, Xilin Jing, Yuqi Wang, Xiaohua Jia, Xing Yang, Kezhong Chen
          </td>
          <td>2025-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pediatric low-grade gliomas (pLGG) comprise 35% of all brain tumors. Despite favorable survival, patients experience significant morbidity from disease and treatments. A deeper understanding of pLGG biology is essential to identify novel, more effective, and less toxic therapies. We utilized single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and cytokine analyses to characterize and understand tumor and immune cell heterogeneity of pilocytic astrocytoma (PA) and ganglioglioma (GG). scRNA-seq revealed tumor and immune cells within the tumor microenvironment (TME). Tumor cell subsets include both progenitor and mature cell populations. Immune cells included myeloid and lymphocytic cells. There was a significant difference between the prevalence of two major myeloid subclusters between PA and GG. Bulk and single-cell cytokine analyses evaluated the immune cell signaling cascade with distinct immune phenotypes among tumor samples. KIAA1549-BRAF tumors appeared more immunogenic, secreting higher levels of immune cell activators and chemokines, compared to BRAF V600E tumors. Spatial transcriptomics revealed the differential gene expression of these chemokines and their location within the TME. A multi-pronged analysis demonstrated the complexity of the PA and GG TME and differences between genetic drivers that may influence their response to immunotherapy. Further investigation of immune cell infiltration and tumor-immune interactions is warranted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dc01e4fe40878b7885262fd79ea7216b56b949" target='_blank'>
              Multi-pronged analysis of pediatric low-grade glioma and ganglioglioma reveals a unique tumor microenvironment associated with BRAF alterations.
              </a>
            </td>
          <td>
            S. Zahedi, Kent Riemondy, Tian Liu, A. Griesinger, A. Donson, April A Apfelbaum, Rui Fu, Julian Grandvallet Contreras, M. Crespo, J. DeSisto, Madeline Groat, Emil Bratbak, Adam L Green, Todd C Hankinson, Michael Handler, Rajeev Vibhakar, Nicholas Willard, Nicholas K Foreman, Tzu Phang, Jean M. Mulcahy Levy
          </td>
          <td>2025-06-30</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ef3fe6c1c7dcea2e010f5575de7254a23fff3d7" target='_blank'>
              Spatial and single-cell transcriptomics landscape of adenomyosis
              </a>
            </td>
          <td>
            X. Yang, C. Li, J. He, B. Xie, L. Lv, J. Xu, Y. Lin, C. Zhou, M. Li, Z. Zeng, J. Tan, S. Chen, G. Cui, G. Liu, S. Suo, G. Peng, X. Liang
          </td>
          <td>2025-07-07</td>
          <td>None</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Precursor lesions like polyps and adenomas in the colon commonly precede colorectal cancer (CRC), advancing through the “normal-polyp-adenoma-carcinoma” sequence towards malignancy. Yet, the cellular heterogeneity and molecular mechanisms involved in CRC development remain inadequately characterized. Methods To understand the molecular mechanisms driving the onset and progression of CRC, we conducted a comprehensive analysis of ten clinical colorectal samples representing sequential pathological stages using single-cell RNA sequencing (scRNA-seq). Validation was performed through immunofluorescence and immunohistochemistry analyses in a separate human colorectal tissue cohort. Additional verification was carried out using bioinformatics analyses of public TCGA and GEO datasets. Results Our comprehensive analyses not only reveal the cellular diversity and transcriptomic differences throughout disease progression but also highlight the importance of leveraging ligand–receptor gene expression to distinguish various cell subtypes. Subsequent examination and validation with a larger sample cohort uncover the specific involvement of ligand–receptor genes, transcription factors, immunoglobulin genes, and heat shock genes in regulating immune responses and microenvironment changes during colorectal tumorigenesis. Conclusions Our extensive transcriptome dataset provides valuable insights and acts as a fundamental resource to deepen our understanding of the complex molecular landscape in CRC. This dataset facilitates improved diagnostic accuracy and the creation of more personalized therapeutic approaches. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06785-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e15cc889bc4d3cf181a1a8c15dbc72bc14a7b3" target='_blank'>
              A dynamic molecular landscape in colorectal cancer progression at single-cell resolution
              </a>
            </td>
          <td>
            Jianwen Sheng, Nianshuang Li, Shuai Li, Yuman Ye, Yuanhang Liao, Yupeng Zhang, Huan Chen, Mengxian Tu, Guping Zhong, Chaoliang Xiong, Pan Zheng, Yuting Lei, Zekun He, Xingxing He, Yao Zhang, Lei Wang, Xiaolin Gao, Yin Zhu, Jianping Liu, Huizhen Fan
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Traditional gene expression deconvolution methods assess a limited number of cell types, therefore do not capture the full complexity of the tumor microenvironment (TME). Here, we integrate nine deconvolution tools to assess 79 TME cell types in 10,592 tumors across 33 different cancer types, creating the most comprehensive analysis of the TME. In total, we found 41 patterns of immune infiltration and stroma profiles, identifying heterogeneous yet unique TME portraits for each cancer and several new findings. Our findings indicate that leukocytes play a major role in distinguishing various tumor types, and that a shared immune-rich TME cluster predicts better survival in bladder cancer for luminal and basal squamous subtypes, as well as in melanoma for RAS-hotspot subtypes. Our detailed deconvolution and mutational correlation analyses uncover 35 therapeutic target and candidate response biomarkers hypotheses (including CASP8 and RAS pathway genes).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6b1e17d9e760b6c7495c8a6aea0db799ec80899" target='_blank'>
              Pan-cancer immune and stromal deconvolution predicts clinical outcomes and mutation profiles
              </a>
            </td>
          <td>
            B. Bhinder, Verena Friedl, Sunantha Sethuraman, Davide Risso, Kami E. Chiotti, R. J. Mashl, Kyle Ellrott, Jordan A. Lee, Christopher K. Wong, K. Gyan, Aditya Deshpande, Marcin Imielinski, R. Bareja, Josh Stuart, Myron Peto, K. Hoadley, Alexander J. Lazar, A. Cherniack, Jingchun Zhu, S. Cao, Mark Rubin, Wenyi Wang, Oliver F. Bathe, N. Robine, Li-hua Ding, Peter W. Laird, Wanding Zhou, Hui Shen, V. Thorsson, Jen Jen Yeh, Matthew H. Bailey, D. Zhou, X. Peng, M. Goldman, Yongsheng Li, Anil Korkut, Nidhi Sahni, D. N. Hayes, Michael K. A. Mensah, Ina Felau, Anab Kemal, Samantha J. Caesar-Johnson, John A. Demchok, Liming Yang, M. Ferguson, R. Tarnuzzer, Zhining Wang, J. Zenklusen, Adam Alana Andrea Andrew Angela Benjamin Benjamin Brian Margolin Weinstein Sboner Blair Brooks Berman Raph, Adam Margolin, Alana Weinstein, A. Sboner, Andrew Blair, Angela Brooks, Benjamin Berman, Benjamin Raphael, Brian Craft, Dante S. Bortone, David Heimain, David L Gibbs, David Haan, D. Betel, Duncan McColl, Emek Demir, Faeze Brahman, F. Farshidfar, Gaddy Getz, G. Gao, Gordon Robertson, Huy Dinh, Hyo Young Choi, I. Shmulevich, Ioannis Anastopoulous, Jasmine Yang, Jeff Damrauer, Ken Chen, Lauren Sanders, Lee Cooper, Liang-Bo Wang, Matt Reyna, Mike Noble, Mohammed El-Kebir, Molly Zhang, Nicole Yeager, Paul Little, Richard Moffitt, S. Meier, Teresa Swatloski, T. Knijnenburg, Thomas Matthew, Vicky Chen, Vlado Uzunangelov, Xinghua Lu, Yige Wu, Yulia Newton, Zeya Wang, Paul T. Spellman, O. Elemento
          </td>
          <td>2025-07-04</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="Background Understanding the molecular interactions between cells, tissues or organs is key to understanding the functioning of a biological system as a whole. Results Here, we propose crossWGCNA: a co-expression-based method that identifies highly interacting genes unbiasedly and that we employ to study stroma-epithelium communication in breast cancer. CrossWGCNA can be applied to bulk, single cell and spatial transcriptomics data. We validate it both in silico and experimentally, and we provide a fully documented R package allowing users to employ it. Conclusions The wide applicability and agnostic nature of our tool make it complementary to existing methods overcoming the limitations arising from strong baseline assumptions. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11747-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b943ef5ac17c56205d2bc6ae7f9f20263cb8cd8" target='_blank'>
              Cross-tissue gene expression interactions from bulk, single cell and spatial transcriptomics with crossWGCNA
              </a>
            </td>
          <td>
            Aurora Savino, R. Iannuzzi, L. Avalle, Andrea Lobascio, Francesco Iorio, Paolo Provero, Valeria Poli
          </td>
          <td>2025-07-01</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Transcriptional Intratumoral heterogeneity (ITH) is a hallmark of aggressive cancers, yet how transcriptional ITH programs drive tumor metastasis and immune evasion in upper aerodigestive squamous cell carcinoma (UASCC) remains unclear. Through single-cell RNA sequencing analysis of UASCC cells and patient tumors, we uncovered a hybrid epithelial mesenchymal transition (hEMT) ITH program linked to metastatic dissemination. The transcription factor ETS1 was identified as a master regulator of the hEMT program, directly activating pro-metastatic genes and promoting distant spread in vivo. Unexpectedly, ETS1 also orchestrated an immune-cold tumor microenvironment by transcriptionally activating both STAT1 and PD-L1 (CD274) genes, suppressing T lymphocyte infiltration, and elevating immune checkpoint molecules. Clinically, ETS1-high tumors strongly correlated with poor survival and resistance to immune checkpoint blockade across multiple cohorts. Leveraging drug screens, we discovered that ETS1-high cancers are vulnerable to HSP90 inhibitors (e.g., Alvespimycin), which suppress ETS1 by disrupting HIF1A-mediated transcriptional activation. Together, our work reveals ETS1 as a dual driver of tumor distal metastasis and immune evasion in UASCC, while nominating HSP90 inhibition as a tailored treatment strategy for ETS1-driven tumors. These findings provide a roadmap for targeting aggressive ITH subsets and overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e629750fa34c7548f170b4ee72868f72fd5026a6" target='_blank'>
              ETS1 Orchestrates a Hybrid EMT Program Driving in vivo Metastasis and Immune Evasion
              </a>
            </td>
          <td>
            Benjamin Ziman, Talia Wenger, Chehyun Nam, Qian Yang, Daniel Arnaudov, Megha Sheth, Zhixuan Jing, Yuhao Pan, Joe Vargas, Yong Teng, Uttam K. Sinha, Y. Park, De-Chen Lin
          </td>
          <td>2025-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background: Breast cancer, the most prevalent malignancy among women worldwide, exhibits significant heterogeneity, particularly in the tumor microenvironment (TME), which poses challenges for treatment. Spatial transcriptomics (ST) has emerged as a transformative technology, enabling gene expression analysis while preserving tissue spatial architecture. This provides unprecedented insights into tumor heterogeneity, cellular interactions, and disease mechanisms, offering a powerful tool for advancing breast cancer research and therapy. This review aims to synthesize the applications of ST in breast cancer research, focusing on its role in decoding tumor heterogeneity, characterizing the TME, elucidating progression and metastasis dynamics, and predicting therapeutic responses. We also explore how ST can bridge molecular profiling with clinical translation to enhance precision therapy. The key scientific concepts of review included the following: We summarize the technological advancements in ST, including imaging-based and sequencing-based methods, and their applications in breast cancer. Key findings highlight how ST resolves spatial heterogeneity across molecular subtypes and histological variants. ST reveals the dynamic interplay between tumor cells, immune cells, and stromal components, uncovering mechanisms of immune evasion, metabolic reprogramming, and therapeutic resistance. Additionally, ST identifies spatial prognostic markers and predicts responses to chemotherapy, targeted therapy, and immunotherapy. We propose that ST serves as a hub for integrating multi-omics data, offering a roadmap for precision oncology and personalized treatment strategies in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbbaea20f0037aedc680b32a7558f9b67f94b665" target='_blank'>
              Spatial Transcriptomics Decodes Breast Cancer Microenvironment Heterogeneity: From Multidimensional Dynamic Profiling to Precision Therapy Blueprint Construction
              </a>
            </td>
          <td>
            A. Ma, L. Xiang, Jingping Yuan, Qianwen Wang, Lina Zhao, Honglin Yan
          </td>
          <td>2025-07-24</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca80c0d5c8118b6afa306ae5816ee0be1c3263a" target='_blank'>
              Single-cell transcriptomics reveals hypoxia-driven iCAF_PLAU is associated with stemness and immunosuppression in anorectal malignant melanoma.
              </a>
            </td>
          <td>
            Hao Zhang, Lin Gan, Xiaofei Duan, Baojing Tuo, Hao Zhang, Senbo Liu, Yugui Lian, Enjie Liu, Zhenqiang Sun
          </td>
          <td>2025-07-04</td>
          <td>Journal of gastroenterology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are the most abundant non-cancerous cell type in glioblastoma (GBM) and heavily influence GBM biology, contributing to tumor progression, therapeutic resistance, immune evasion, and neovascularization. Current in vitro models that utilize IL-4/IL-13 stimulation fail to capture the transcriptional and functional heterogeneity of TAMs observed in vivo. In this study, we utilize a serum-free indirect co-culture model with patient-derived xenolines to polarize primary human macrophages and characterize their molecular and functional phenotypes. We demonstrate that xenoline-polarized macrophages diverge from classical M1/M2 states and instead adopt transcriptional signatures reflective of TAM subsets identified from patients. Notably, macrophages polarized with the radiation-therapy selected xenoline, JX14P-RT, exhibited gene expression patterns enriched for interferon response and hypoxia, mirroring recurrent GBM samples. In contrast, JX14P TAMs showed enrichment in phagocytic gene sets. Functional validation of these phenotypes revealed discrepancies between the transcriptionally predicted and observed phenotypes, emphasizing the importance of integrating phenotypic validation in sequencing studies. Altogether, our findings establish xenoline-polarized macrophages as a useful alternative to traditional models that can be used to study immune-interactions in vitro. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02057-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f15f733124e62cedabe127db49de61b0e9f489" target='_blank'>
              Xenoline-polarized macrophages as an alternative in vitro model of tumor-associated macrophages in glioblastoma
              </a>
            </td>
          <td>
            Acta Neuropathologica Communications, Christopher D. Willey, Hasan Alrefai, Benjamin Lin, Amr Elkholy, Manoj Kumar, Taylor L. Schanel, Kevin J. Lee, Patricia H. Hicks, Joshua C. Anderson, Gao Guo, Eun-Young Erin Ahn, Ryan Miller
          </td>
          <td>2025-06-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Multi-omics profiling of glioblastoma (GBM) has unravelled two aspects fundamental to its aggressiveness and lethality that is molecular heterogeneity inherent to the tumour and cellular plasticity modulated by microenvironment. Yet, empirical validation to identify causal factors for these complex mechanisms is rather scarce. Here, we report our endeavour in establishing Glioportal, a GBM tumour biobank with derivative preclinical models and molecular information that we leverage for basic and translational research on precision therapies.


METHODS
Bulk transcriptome and single-cell-based deconvolution analyses highlighted key features of distinct GBM subtypes and ligand-receptor pairs predicted to regulate malignant cell states plasticity. Synthetic genetic tracing tool and target genes/proteins expression analyses validated ligands-induced mesenchymal transition. This was further corroborated with phenotypic invasion/migration assays and cell-based assays using inhibitors, functional antibodies and gene silencing approaches. Proof-of-concept animal experiment was conducted using orthotopic xenograft carrying gene knockdown. Clinical relevance was assessed through immunohistochemical assay.


RESULTS
Our transcriptomic analysis highlights the integral roles of STAT3 and NF-κB pathways in maintaining intrinsic mesenchymal identity and enabling myeloid-induced plasticity towards mesenchymal phenotype. One critical ligand, TNF, confers mesenchymal adaptation and cellular invasiveness that is mitigated by TNFRSF1A, but not TNFRSF1B, loss of function. TNFRSF1A silencing significantly improves survival in vivo.


CONCLUSION
Glioportal makes a valuable resource for identifying therapeutic vulnerabilities in molecularly stratified GBM. Here, we underscore GBM dependency on myeloid-derived ligands to acquire mesenchymal traits that has clinical implications in therapeutic response and recurrence. Such reliance warrants treatment strategies targeting ligand-receptor pairs to mitigate interactions with tumour ecosystem.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be58b788b3b08ac114aabcdc35465e8eea100aee" target='_blank'>
              Glioportal: a comprehensive transcriptomic resource unveiling ligand-mediated mesenchymal transition in glioblastoma.
              </a>
            </td>
          <td>
            Qing You Pang, Wisna Novera, L. Koh, Y. Chong, S. Lim, Ngak-Leng Sim, Simone Rizzetto, Jinyue Liu, Xuling Lin, Samantha Ya Lyn Ang, J. Ker, Kai-Rui Wan, David Chyi Yeu Low, Marija Cvijovic, Wilson Wen Bin Goh, Huilin Shao, Nguan Soon Tan, Stephen Yip, A. Skanderup, Patrick Tan, Carol Tang, B. Ang
          </td>
          <td>2025-06-16</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Radiomics is undergoing a paradigm shift from single-omics to multi-omics, from single-temporal to multi-temporal analysis, and from global to subregional analysis. These transformations have shown great potential in addressing key challenges related to imaging changes before and after neoadjuvant chemotherapy (NAC) in breast cancer. Furthermore, radiomics has achieved remarkable progress in tasks such as exploring tumor heterogeneity and uncovering underlying biological mechanisms. Integrating imaging data with gene data offers novel perspectives for understanding imaging changes driven by specific genetic alterations. However, current radiomics studies on neoadjuvant chemotherapy for breast cancer have not yet achieved a close integration of imaging changes with underlying biological mechanisms. They are largely limited to simple associations between models and genomic data, without in-depth interpretation of the biological significance inherent in imaging features, which is essential to directly link these features with the dynamic progression of the disease. This review seeks to explore the spatial-temporal heterogeneity of imaging alterations observed during NAC for breast cancer, while assessing their biological implications using established analytical approaches. It highlights the distinct advantages of spatial-temporal radiomics in predictive model development and examines potential correlations between imaging dynamics and gene expression profiles before and after NAC. Additionally, we critically examines previous radiogenomics studies, providing theoretical insights into their limitations. Finally, the review proposes future directions and innovative approaches for applying spatial-temporal radiogenomics in NAC for breast cancer, serving as a valuable reference and roadmap for researchers and clinical practitioners in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1413564d0b7a7765ec86c504011b8f276395913" target='_blank'>
              Spatial-temporal radiogenomics in predicting neoadjuvant chemotherapy efficacy for breast cancer: a comprehensive review
              </a>
            </td>
          <td>
            Tingfeng Zhang, Liang Zhao, Tingting Cui, Yi Zhou, Peifen Li, Chuqiao Luo, Junkang Wei, Hong Hu
          </td>
          <td>2025-06-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/938950b1958bbfaef501d56fa236f86c6e2c6141" target='_blank'>
              Multi-omic analysis of gallbladder cancer identifies distinct tumor microenvironments associated with disease progression.
              </a>
            </td>
          <td>
            Tao Zhou, Yanhong Wu, Shuai Li, Xinyao Qiu, Erdong Liu, Zhihua Xie, Xuebing Shi, Yani Zhang, Guosheng Ma, Wenbo Guo, Xiang Wang, Kaiting Wang, Xiaomeng Yao, Ji Hu, Siyun Shen, Shuai Yang, Xiaoqing Jiang, Jing Fu, Hongyang Wang, Jin Gu, Lei Chen
          </td>
          <td>2025-06-26</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>12</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is a leading cause of cancer-related death, and reliable biomarkers for prognosis and treatment guidance remain limited. Membrane proteins play key roles in tumor progression and therapeutic response, yet their clinical utility in LUAD remains underexplored. We integrated scRNA-seq, spatial transcriptomics, and bulk RNA-seq datasets from multiple LUAD cohorts to identify cancer-specific membrane proteins derived from epithelial subpopulations. Based on these results, we constructed a prognostic signature, LCaMPS, and evaluated its predictive performance using multiple datasets. The expression of model genes was confirmed at both the bulk RNA and protein levels. Associations with the tumor microenvironment (TME) and drug sensitivity were further analyzed. A distinct LUAD-enriched epithelial cluster (Epi_c0) exhibiting hypoxic and EMT signatures was identified. 35 cancer-specific membrane proteins were defined, several of which, including TSPAN8, BACE2, and COX16, showed strong spatial localization within the tumor regions. LCaMPS, a 9-membrane gene-based prognostic model, stratified patient prognosis and predicted 5- and 10-year survival rates with high accuracy. High LCaMPS scores were associated with increased infiltration of neutrophils, endothelial cells, and fibroblasts in the TME and predicted higher sensitivity to 66 chemotherapeutic agents, including Gemcitabine and Sorafenib. Low-risk patients were predicted to respond better to drugs, such as Cisplatin and Parthenolide. This study highlights the importance of membrane expression patterns in LUAD at single-cell and spatial resolution. The LCaMPS model provides a robust prognostic and therapeutic stratification tool with potential applications in personalized cancer management. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-11105-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82b97812f84c01b026525655a678dbe43cf10957" target='_blank'>
              A novel cancer-associated membrane signature predicts prognosis and therapeutic response for lung adenocarcinoma
              </a>
            </td>
          <td>
            Biao Tu, Jun Wu, Wei Zhang, Haitao Tang, Tenghui Dai, Bingfeng Xie
          </td>
          <td>2025-07-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Spatial transcriptomics (ST) enables high-resolution mapping of gene expression across tissue architecture and can reveal the organization of cellular states in complex tissues, such as the tumor microenvironment. Using ST data from glioblastoma (GBM) patients, a large recent study1 identified robust gene programs in a spatially oblivious manner and used them to assign cell states to spatial spots and assess cell-state spatial organization. The results characterized GBM tumors as generally spatially heterogeneous, with some hypoxia-associated structured regions. However, recent computational methods support analyses that more tightly integrate spatial context and gene expression to identify continuous gradients and rare niches in ST data. We hypothesized that applying such an approach may reveal additional key aspects of the spatial organization of transcriptional programs in GBM tumors.



 Based on our previous successes with traditional topic modeling in transcriptomic analysis, we focused on the recently published method Spatial Transcriptomics Analysis with topic Modeling to uncover spatial Patterns (STAMP)2. STAMP uses a deep generative model to perform spatially aware topic modeling that can account for replicates. Using STAMP, we reanalyzed the 26 ST samples (10x Visium, 60,000+ spots) from 17 patients in the GBM dataset1 to infer spatial topics or gene programs, which we then interpreted in the context of previous studies. We also extended the analysis framework to enable quantification and statistical assessment of the spatial coherence of continuous gene program weights.



 Consistent with previous studies, we identified gene programs associated with a hypoxic niche, as well as with precursor and differentiated endothelial, oligodendrocyte and neuronal cell types. However, our study also revealed a number of new gene programs with strongly spatially organized patterns of expression occurring in multiple patient samples. These findings indicate additional spatial axes that may capture important variation in and across GBM tumor microenvironments.



 Complementing a previous hypoxia-associated model of GBM spatial organization, our study characterized additional, spatially resolved functional processes, which may be clinically and therapeutically relevant. The results suggest that a spatially aware, non-categorical approach to ST analysis may drive discovery of previously hidden, spatially coherent, recurrent gene programs in the tumor microenvironment. This framework is broadly applicable for dissecting the transcriptionally governed spatial architecture of solid tumors.



 1Greenwald, A.C. et al. Integrative spatial analysis reveals a multi-layered organization of glioblastoma. Cell 187, 2485–2501.e26 (2024). 2ZHong, C., Ang, K.S. & Chen, J. Interpretable spatially aware dimension reduction of spatial transcriptomics with STAMP. Nat. Methods 21, 2072–2083 (2024).



 Joseph J. Sifakis, Sophia Madejski, Samantha J. Riesenfeld. Spatially aware transcriptomic topic modeling reveals novel signals of spatial organization in glioblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/953df0c913ecb019b9b2578d60ef3030541a85b4" target='_blank'>
              Abstract B050: Spatially aware transcriptomic topic modeling reveals novel signals of spatial organization in glioblastoma
              </a>
            </td>
          <td>
            Joseph J. Sifakis, Sophia Madejski, Samantha J Riesenfeld
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Recent studies have explored the composition of tumor microenvironment (TME) in diffuse large B-cell lymphoma (DLBCL) However, cell-to-cell interactions, along with the spatial organization of DLBCL TME and their impact on patient outcomes, have remained poorly characterized. We ap-plied multiplexed immunofluorescence, cell phenotyping, and neighborhood analysis to investigate 1,218,756 single cells in 99 samples from patients with primary DLBCL. We identified 17 cell pheno-types and 10 recurrent cellular neighborhoods (RCNs) across samples, subdividing DLBCLs into immune-poor areas and areas with diverse immune cell infiltrates. Avoidance of B cells and PD-1+ T cells was associated with less aggressive clinical characteristics and favorable survival. Like-wise, the proximity of CD8+ T cell-rich and immune-poor RCNs translated to favorable, and the proximity of PD-L1+ B cell-rich and CD8+ T cell-rich RCNs to unfavorable patient outcomes. Our findings provide insights into the spatial interactions and organization of DLBCL TME with implica-tions for patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8a36543383d1dc42cdb78f6b491ab4c213fbf1a" target='_blank'>
              Spatial Analysis of the Tumor Microenvironment in Diffuse Large B-Cell Lymphoma Reveals Clinically Relevant Cell Interactions and Recurrent Cellular Neighborhoods.
              </a>
            </td>
          <td>
            M. Autio, S. Leivonen, L. Meriranta, M. Karjalainen-Lindsberg, T. Pellinen, S. Leppä
          </td>
          <td>2025-08-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) tumours are classified by pathology into several distinct subtypes. Gene expression profiling has revealed transcriptional heterogeneity across NEPC, but this is rarely considered in the context of variation between pathologies. Diagnosis typically relies on immunohistochemical markers (CHGA, SYP, NCAM1) and genomic alterations in RB1, PTEN and TP53. We hypothesized that NEPC pathologies have unique transcriptional features. Single-cell RNA sequencing of 18 632 tumour cells from nine patient-derived xenograft models representing five pathologies (small-cell and large-cell neuroendocrine carcinomas, focal neuroendocrine differentiation (Focal NED), low-grade neuroendocrine and amphicrine) demonstrated pathway-specific enrichment. Focal NED and amphicrine tumours exhibited cellular subpopulations enriched for KRAS, IL2-STAT5 and TNF signalling pathways, absent in small- and large-cell carcinomas, which were instead enriched for Myc and E2F pathways. Furthermore, focal NED cells exhibited minimal clonal divergence from adjacent adenocarcinoma cells, while small cell carcinoma cells were clonally distinct. These data underscore significant transcriptional variation among NEPC pathologies, highlighting focal NED's unique biological context and its clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c9094a51a0844d2931b4655b974ca910515f91f" target='_blank'>
              Single-cell transcriptomics redefines focal neuroendocrine differentiation as a distinct prostate cancer pathology.
              </a>
            </td>
          <td>
            Rosalía Quezada Urban, S. Keerthikumar, A. Clark, Hong Wang, B. Phipson, Andrew Bakshi, Andrew Ryan, Heather Thorne, R. Taylor, Mitchell G. Lawrence, G. Risbridger, Roxanne Toivanen, D. L. Goode
          </td>
          <td>2025-07-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Tumor drug resistance represents a major challenge in contemporary cancer therapeutics, significantly compromising the clinical efficacy of chemotherapy, targeted therapy, and immunotherapy. While existing research has elucidated the critical role of tumor cell-intrinsic mechanisms in drug resistance—including genomic instability, persistent activation of signaling pathways and aberrant epigenetic modifications—emerging evidence highlights the crucial involvement of dynamic remodeling within the tumor microenvironment (TME) in driving therapeutic resistance. The TME fosters drug resistance through dynamic remodeling, creating hypoxic conditions, immunosuppressive networks, and metabolic stress, which collectively impair treatment response and promote therapeutic escape. Advances in multi-omics technologies now enable a comprehensive, multi-dimensional analysis of these interactions, integrating genomic, epigenomic, transcriptomic, proteomic, and metabolomic data to uncover critical molecular networks and vulnerabilities. In this review, we explore the key mechanisms by which the TME influences drug resistance, discuss how multi-omics approaches enhance our understanding of these processes and evaluate emerging therapeutic strategies aimed at reprogramming the TME to overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0679c6b5fd0da9cf7502220f6fe761b9cdab9b5" target='_blank'>
              Multi-omics dissection of tumor microenvironment-mediated drug resistance: mechanisms and therapeutic reprogramming
              </a>
            </td>
          <td>
            Fanghua Chen, Yuandong Fu, Gaigai Bai, J. Qiu, Keqin Hua
          </td>
          <td>2025-07-07</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastasis causes about 90% of cancer deaths, but the anticipation of its occurrence from primary tumor biopsies remains an intractable clinical dilemma. Spatial organization and biochemical heterogeneity of TME are regulators of metastatic formation but are usually not interrogated by typical diagnostic approaches. To overcome this deficiency, we established a hybrid modeling approach that unites biopsy-derived proteomics, spatial statistical analysis, and machine learning to predict metastatic potential. Employing multiplexed proteomic imaging technologies like imaging mass cytometry, we profiled in situ protein expression patterns in the patient's biopsy tissue. These data were used to calculate spatial features like cell–cell interaction graphs, spatial proximity scores, and neighborhood-based immune signatures. Machine learning classifiers—graph neural networks and random forests—were trained to predict these spatial-biochemical signatures from recognized metastatic endpoints. Our model performed with excellent prediction capability (AUC ≈ 0.90) in a retrovalidation set across various solid cancers. Notably, we also found specific TME signatures that correlate with metastasis: dense foci of PD-L1⁺ macrophages and FOXP3⁺ regulatory T cells at the invasive edge were correlated with a higher risk of metastasis, and abundant CD8⁺ T cell infiltration at the invasive edge with lower progression. This integrated strategy provides a clinically feasible method for predicting metastasis from standard biopsy samples, which could potentially lead to earlier treatment and therapy stratification. Our results highlight the translational value of the combination of spatial proteomics and computational modeling towards improved prognostic precision in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc1c3111d9dc4060886bfbec6b57329f90c0fbb" target='_blank'>
              BIOCHEMICAL AND SPATIAL MODELING OF TUMOR MICROENVIRONMENT FROM BIOPSY PROTEOMICS TO PREDICT METASTATIC PROGRESSION
              </a>
            </td>
          <td>
            PEACE CHINONYEREM IKE, CHIKA GLORIA OKWUOGORI, THEOPHILUS IGWEBUIKE ILOEGBU, ONYEKA MILICENT ASUMAH, ONWUKWE OLUCHI BUCHI, AFOLAYAN OLUWATOSIN SUNDAY
          </td>
          <td>2025-07-10</td>
          <td>International Journal of Medical Biological and Pharmaceutical Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86f5e782277174d9b03cd398747b9811afe28b1c" target='_blank'>
              Tissue architecture dynamics underlying immune development and decline in the thymus
              </a>
            </td>
          <td>
            Sophia Liu, Mirco J Friedrich, Ruth Raichur, Dennis Gong, Niklas Kehl, Qiyu Gong, Bingxian Chen, Karin Gustafsson, Larissa L. Ma, David T. Scadden, Rhiannon K. Macrae, Feng Zhang, Fei Chen
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Formalin-fixed paraffin-embedded (FFPE) samples represent a vast, untapped resource for epigenomic research, yet molecular tools for deep analysis of these specimens remain limited. We introduce spatial FFPE-ATAC-seq, an approach for in situ profiling chromatin accessibility within archived tissues. This approach overcomes formalin-induced crosslinking challenges, allowing high-resolution mapping of chromatin landscapes while preserving tissue architecture. Applying spatial FFPE-ATAC-seq to mouse and human tissues, including brain and thymus, reveals intricate spatial organization and distinct cell types in alignment with tissue morphology. Integration with single-cell RNA sequencing validates the precision of our chromatin profiles in identifying key cell types and regulatory elements. We further apply this method to human melanoma, comprehensively characterizing chromatin accessibility across both tumor and non-tumor regions. This method significantly expands the toolkit for epigenomic research, unlocking the potential of an extensive collection of archived FFPE samples for studying gene regulation and disease mechanisms with spatial context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9a8ea3c46aa7a27b26189991ac0f0ce423df34a" target='_blank'>
              Spatial profiling of chromatin accessibility in formalin-fixed paraffin-embedded tissues
              </a>
            </td>
          <td>
            Pengfei Guo, Yufan Chen, Liran Mao, Angelysia Cardilla, Chin Nien Lee, Yan Cui, Dengge Jin, Yucong Hua, Xiaowei Xu, Yanxiang Deng
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Spatial genomics has revolutionized breast cancer research by mapping gene expression within the tumor microenvironment, offering unprecedented insights into the molecular and cellular interactions that drive tumor progression and therapeutic response across luminal A, luminal B, HER2-positive, and triple-negative subtypes. Unlike traditional genomic methods, spatial genomics preserves tissue architecture, revealing subtype-specific TME features, such as immune-rich regions in triple-negative breast cancer and stromal-dense environments in HER2-positive tumors. This review explores spatial genomics technologies, including Visium, GeoMx, and MERFISH, which enable precise TME analysis, highlighting their role in identifying biomarkers like PD-L1 and TGF-β for targeted therapies and immunotherapies. By elucidating TME-driven resistance mechanisms, such as hypoxia in triple-negative tumors or stromal remodeling in HER2-positive cases, spatial genomics informs clinical trials and combination therapies, enhancing patient stratification and treatment efficacy. Despite its promise, challenges including limited resolution, complex data analysis, and high costs hinder clinical adoption. Emerging technologies, artificial intelligence, and multi-omics integration offer solutions, promising higher precision and scalability. This review underscores spatial genomics’ transformative potential to bridge research and clinical practice, paving the way for personalized breast cancer therapies tailored to the dynamic TME, with future advancements poised to redefine treatment paradigms and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5822524fe77c324fb45f43e4adc21fca11863f06" target='_blank'>
              Unraveling tumor microenvironments with spatial genomics: Implications for precision breast cancer therapy
              </a>
            </td>
          <td>
            Amina Suleiman Kamsalem, Obianujunwa Mercy Ubah, Tobiloba Philip Olatokun, Regina Omodolapo Oluwaniyi, Olivia Ajifa Philips, Damilola Samuel Ojo-Omoniyi, Omolola Lolade Oluwasakinju
          </td>
          <td>2025-07-30</td>
          <td>International Journal of Life Science Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug resistance poses a significant challenge in cancer therapy, contributing to rapid recurrence, disease progression, and high patient mortality. Despite its critical impact, few reliable predictors for cancer drug response or failure have been established for clinical application. Tumor heterogeneity and the tumor microenvironment (TME) are pivotal factors influencing cancer drug efficacy and resistance. Tumor heterogeneity leads to variable therapeutic responses among patients, while dynamic interactions between cancer cells and the TME enhance tumor survival and proliferation, underscoring the urgent need to identify cellular hallmarks for predicting drug response and resistance. Single-cell and spatial omics technologies provide high-resolution insights into gene expression at the individual cell level, capturing intercellular heterogeneity and revealing the underlying pathologies, mechanisms, and cellular interactions. This review delves into the principles, methodologies, and workflows of single-cell and spatial omics in cancer drug research, highlighting key hallmarks involving tumor heterogeneity, TME reprogramming, cell–cell interactions, metabolic modulation, and signaling pathway regulation in drug treatment at single-cell and spatial levels. Furthermore, we synthesize predictive cellular biomarkers for cancer drug response and resistance across 25 cancer types, paving the way for advancements in cancer precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af5a8672965af056452be6aaf8b56f1ff303b62" target='_blank'>
              Application of single-cell and spatial omics in deciphering cellular hallmarks of cancer drug response and resistance
              </a>
            </td>
          <td>
            Xiaoxia Cheng, Ting Peng, Tian Chu, Yiqun Yang, Jia Liu, Qinglei Gao, C. Cao, Juncheng Wei
          </td>
          <td>2025-07-02</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Pancreatic cancer (PC) is marked by extensive heterogeneity, posing significant challenges to effective treatment. The tumor microenvironment (TME), particularly cancer-associated fibroblasts (CAFs), plays a critical role in driving PC progression. However, the prognostic and functional contributions of distinct CAF subtypes remain inadequately understood. Here, we introduce a novel 7-gene risk model that not only robustly stratifies PC patients but also unveils the unique role of PHLDA1 as a key mediator in tumor-stroma crosstalk. Methods By integrating single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and bulk RNA sequencing data, we comprehensively characterized the heterogeneity of CAFs in PC. We identified five CAF subtypes and focused on matrix CAFs (mCAFs), which were strongly associated with poor prognosis. A 7-gene mCAF-associated risk model was constructed using advanced machine learning algorithms, and the biological significance of PHLDA1 was validated through co-culture experiments and pan-cancer analyses. Results Our multiomics analysis revealed that the novel 7-gene model (comprising USP36, KLF5, MT2A, KDM6B, PHLDA1, REL, and DDIT4) accurately predicts patient survival, immunotherapy response, and TME status. Notably, PHLDA1 was uniquely overexpressed in CAFs and correlated with the activation of key protumorigenic pathways, including EMT, KRAS, and TGF-β, underscoring its central role in modulating the crosstalk between CAFs and malignant ductal cells. Pan-cancer analysis further supported PHLDA1’s prognostic and immunomodulatory significance across multiple tumor types. Conclusion Our study presents a novel 7-gene prognostic model that significantly enhances risk stratification in PC and identifies PHLDA1+ CAFs as promising prognostic biomarkers and therapeutic targets. These findings provide new insights into the TME of PC and open avenues for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69308ca9c802c2244ef4487c21e12f41adc3c76e" target='_blank'>
              Integrated multiomics analysis identifies PHLDA1+ fibroblasts as prognostic biomarkers and mediators of biological functions in pancreatic cancer
              </a>
            </td>
          <td>
            Rui Wang, Guan-Hua Qin, Yifei Jiang, Fu-Xiang Chen, Zi-Han Wang, Linglin Ju, Lin Chen, Da Fu, En-Yu Liu, Su-Qing Zhang, Wei-Hua Cai
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Although the significance of intratumoral heterogeneity (ITH) has been widely acknowledged in various cancers, its role in pancreatic cancer (PC) remains underexplored and warrants further investigation. Methods Pancreatic cancer transcriptomic data were acquired from the TCGA and GEO databases. The DEPTH2 algorithm, in combination with differential expression analysis, was used to identify genes associated with intratumoral heterogeneity (ITH). We applied univariate Cox regression analysis and multiple machine learning techniques to establish a reliable prognostic model. Patients were then stratified according to their ITH scores, and differences between subgroups were examined through pathway enrichment analysis, immune cell infiltration profiling, and drug response prediction. Furthermore, we conducted subcellular localization and differential analysis of ITH using single-cell data, followed by cell-cell communication analysis to explore interaction relationships and identify key pathways. Results Patients exhibiting lower intratumoral heterogeneity (ITH) levels demonstrated poorer clinical outcomes. The constructed 11-gene signature successfully differentiated individuals into high- and low-risk categories with significant survival differences. Immune profiling revealed notable differences in immune cell composition between the two groups, with patients in the high ITH cohort exhibiting enhanced immune activation. Drug sensitivity analysis indicated a differential response to therapies, with high-risk patients more resistant to certain drugs. Single-cell RNA sequencing identified a greater ITH score in epithelial cells, highlighting key interactions, particularly involving Galectin signaling pathways. Conclusion Our results highlight intratumoral heterogeneity’s prognostic and therapeutic relevance in pancreatic cancer, suggesting its potential utility in guiding individualized treatment approaches. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03080-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c15cd4c65cc7acb2163700609aefaa2af8b33ee" target='_blank'>
              Integrating bulk and single-cell RNA sequencing reveals intratumor heterogeneity phenotypes and immune infiltration in pancreatic cancer
              </a>
            </td>
          <td>
            Li Zhao, Yao Huang, Jiang Li, Rui Xie, Kaiming Li
          </td>
          <td>2025-07-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Extracellular vesicles (EVs), serving as pivotal mediators of intercellular communication within the tumor microenvironment (TME), exert substantial regulatory influence on lung cancer progression and treatment resistance through their cargo of non-coding RNA (ncRNA). This comprehensive review systematically delineates the biogenesis mechanisms of EVs-ncRNA and their dualistic biological functions in lung carcinogenesis. Pro-tumoral ncRNA are selectively packaged into EVs through specialized sorting mechanisms, subsequently activating oncogenic pathways to potentiate tumor proliferation, invasion, and angiogenesis. Conversely, tumor-suppressive ncRNA are depleted intracellularly via EV-mediated export, thereby attenuating their regulatory control over tumor-suppressive pathways. Notably, EVs-ncRNA derived from tumor stromal components—CAFs, TAMs and BMSCs—orchestrate immunosuppressive reprogramming through cross-regulatory networks, facilitating M2 macrophage polarization, T-cell exhaustion, and consequent therapeutic resistance. Clinically, EVs-ncRNA hold substantial promise as multifaceted biomarkers, enabling early detection, prognostic stratification, and dynamic monitoring of therapy resistance in malignancies. Moreover, their emerging roles as therapeutic carriers or molecular targets highlight transformative potential in precision oncology. Nevertheless, critical challenges persist, including heterogeneity resolution among EVs-ncRNA subpopulations, standardization of cross-species engineered EV production, and establishment of multi-omics dynamic monitoring systems. This synthesis provides a molecular foundation and translational framework for developing innovative diagnostic and therapeutic strategies in lung cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48cdc83836bf0bd58b6ac84f9ccaab0fb3258ad2" target='_blank'>
              Harnessing EVs-ncRNA for Lung Cancer: From Oncogenic Pathways to Novel Diagnostic and Therapeutic Strategies
              </a>
            </td>
          <td>
            Yuqing Shi, Hong Chen, Wei Li, Song An, Linfang Li, Hongmei Gao
          </td>
          <td>2025-07-14</td>
          <td>International Journal of Nanomedicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4879912b1737fce64c962b382e72c481f6e838" target='_blank'>
              Spatial mapping of proteins and their activity states in cancer models by multiplex in situ PLA
              </a>
            </td>
          <td>
            Liza Löf, Bo Xu, Tanay Kumar Sinha, Caroline Dahlström, J. Vennberg, Tore Larsson Forssén, Axel Klaesson, C. Clausson, Xuan Wang, Ulla Strömberg-Olsson, M. Kamali-Moghaddam, C. Avenel, C. Wählby, Agata Zieba-Wicher, Ulf Landegren
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Renal cell carcinoma is characterized by a poor prognosis. Recently, renal cell carcinoma has been recognized as a metabolic disease associated with fatty acid metabolic reprogramming, although in-depth studies on this topic are still lacking. We found that fatty acid metabolism reprogramming in renal cell carcinoma is primarily characterized by high expression of FABP1. FABP1 + tumors significantly impact survival and display distinct differentiation trajectories compared to other tumor subclusters. They show elevated expression of angiogenesis and cell migration signals, with PLG-PLAT-mediated interactions with endothelial cells notably enhanced. Spatial transcriptomics show a prominent co-localization of FABP1 + tumors with endothelial cells, and their spatial distribution closely aligns with that of PLAT + endothelial cells. FABP1 + tumors exhibit a unique pattern in spatial transcriptomics, enriched in Extracellular Matrix and angiogenesis-related pathways. Through receptor-ligand interaction analysis, a novel PLG-PLAT functional axis was found between tumor epithelial cells and endothelial cells. Based on results of experiments, we infer that FABP1 + tumors can promote plasmin-related tumor angiogenesis by triggering the PLG-PLAT signaling axis. Finally, utilizing preclinical models, we suggest that targeting the FABP1-PLG-PLAT axis may serve as promising strategy enhancing the sensitivity of Tyrosine Kinase Inhibitor therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02377-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27626d5d72e69d9402b111fc64dabcbee4819f07" target='_blank'>
              Single-cell multi-omics reveals that FABP1 + renal cell carcinoma drive tumor angiogenesis through the PLG-PLAT axis under fatty acid reprogramming
              </a>
            </td>
          <td>
            Yiqiu Wang, Yingchun Liang, Min Li, Jiayi Lu, Sian Zhou, Yaoyu Yu, Changwei Yang, Xinhuang Hou
          </td>
          <td>2025-06-16</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The dysregulated and fibrotic tumor microenvironment of hepatocellular carcinoma (HCC) delays diagnosis and presents many complex signals that drive disease progression. To better recapitulate this microenvironment, we have enhanced our established protein microarray platform by integrating design of experiments (DoE) methodology with high-throughput cell microarray screening. This innovative approach systematically interrogates the intricate roles of matrix stiffness (spanning healthy and fibrotic conditions), extracellular matrix (ECM) composition, and protein concentration, while simultaneously examining their interdependent interactions. By leveraging DoE principles, we were able to explore 117 unique microenvironments on a single microscope slide, ultimately generating a comprehensive dataset of 234 different microenvironments without compromising statistical rigor. Our enhanced screening system enabled the identification of unique microenvironmental interactions critically significant in dictating cellular responses, including adhesion, survival, proliferation, epithelial-to-mesenchymal transition, and drug resistance markers. Utilizing advanced statistical techniques such as linear models and principal component analysis, we characterized phenotypic clusters defined by precise microenvironmental cues. This work presents a robust, high-throughput microarray screening system that comprehensively explores the contributions of 9 physiologically relevant extracellular matrix proteins and matrix stiffness in modulating cellular behavior and disease progression through a methodologically sophisticated and statistically sound approach. .">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d6d43b14b77f59b187769ee40b324e60d7e33b" target='_blank'>
              Enhanced combinatorial analysis of tumor cell-ECM interactions using design-of-experiment optimized microarrays.
              </a>
            </td>
          <td>
            Hannah R C Kimmel, Allison L Paxhia, Zahra Adamji, Gregory H Underhill
          </td>
          <td>2025-07-24</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/021065c3a42b66477bb5801231e24d3f3a77bd0a" target='_blank'>
              Decoding menopause-induced tissue fibrosis using pan-tissue network inference
              </a>
            </td>
          <td>
            Hirotaka Iijima, Atsushi Yamashita, J. Galloway, Nam Vo, Haksoo Choi, F. Ambrosio
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Ovarian cancer, the gynecological malignancy with the lowest survival rate, is significantly influenced by the tumor microenvironment. The mesenchymal subtype of high-grade serous carcinoma (HGSC) shows poor outcomes due to high stromal and low immune response. Single-cell RNA sequencing (scRNA-seq) of HGSC metastatic ascites has identified carcinoma-associated fibroblasts (CAFs), macrophages, and carcinoma-associated mesenchymal stem cells (CA-MSCs) as crucial drivers of immune exclusion, chemotherapy resistance, metastasis, and stem-like cell propagation. To explore this complex signaling, we developed heterogeneous tri-component tumoroids, incorporating HGSC cells (OVCAR3, OVCAR4, OVCAR8), primary MSCs, and U937-derived M2-like macrophages (M2-AAM) in defined ratios, each labeled with a fluorescent protein for distinct analysis. Upon a 48-hour treatment with carboplatin and/or paclitaxel, HGSC cells in tri-component tumoroids exhibited higher chemoresistance than HGSC-only spheroids. Flow cytometry revealed significant increases in cancer stem-like cell (CSC) markers CD44 and CD90 in the tri-component tumoroids. Conditioned medium from the tri-component tumoroids significantly enhanced HGSC cell migration compared to spheroids. Invasion assays further demonstrated that tri-component tumoroids penetrated monolayer of mCherry-labeled LP-9 mesothelial cells more effectively than spheroids. Additionally, scRNA-seq of tri-component tumoroids identified a unique cancer cell cluster enriched in epithelial-mesenchymal transition (EMT) and matrisome signatures, featuring a 14-gene signature linked to poor survival. MSCs in these tri-component tumoroids displayed a myofibroblastic-CAF signature, while macrophages indicated an ECM-associated and immunosuppressive phenotype. In conclusion, our 3D heterogenous tri-component tumoroids replicate key HGSC phenotypes, such as chemoresistance, CSC enrichment, migration, invasion, and EMT. This platform is invaluable for studying HGSC microenvironment interactions and preclinical testing of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7275362bc5403eae06b86a6f35a14cb4c65aa56" target='_blank'>
              Tumoroid model recreates clinically relevant phenotypes of high grade serous ovarian cancer (HGSC) cells, carcinoma associated fibroblasts, and macrophages
              </a>
            </td>
          <td>
            Kathleen M Burkhard, Ayush Semwal, Ben Johnson, Kristina Chu, Riley Kranick, Mihika Rayan, A. Difeo, Hui Shen, Geeta Mehta
          </td>
          <td>2025-06-19</td>
          <td>Research Square</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The high heterogeneity of tumor microenvironment (TME) is a key factor affecting immunotherapy. We integrated immunotherapy related scRNA-seq and bulk datasets to analyze the TME, mine key factors and dissect the mechanism in immunotherapy. Analyzing the cell composition of TME in cancer immunotherapy, we revealed key TME cell types, including B cells, CD8+ T cells and fibroblasts. Through cell subsets identification, pseudo-time sequence and functional status analysis of malignant cells, it was found that the functional status of malignant cell was heterogeneous under different immunotherapy conditions. Key cellular interactions in cancer immunotherapy were also revealed. Further, 16 marker genes and 6 marker cell types of cancer immunotherapy were identified, and then gene and cell model were constructed to predict the immunotherapy response of individuals, and the model were evaluated based on multiple independent validation datasets. This study can provide important guidance for improving the efficiency and understanding the mechanism of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e592eb99f369b31c7d4285663aa8ee963aaf98a9" target='_blank'>
              Single cell analysis of pan-cancer tumor microenvironment in immunotherapy.
              </a>
            </td>
          <td>
            Feng Li, Ke Xue, Liying Pei, Yu Sun, Zhe Chen, Yifang Zhang, Kaiyue Song, Wensong Liu, Quan Qi, Xue Gong, Jing Li, Yanjun Xu
          </td>
          <td>2025-06-19</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and aggressive primary malignant brain tumor in adults, with limited survival outcomes due to tumor recurrence, mainly driven by GBM cell invasion and therapy resistance. Although temozolomide (TMZ) remains the standard-of-care chemotherapeutic, its long-term efficacy is often compromised by rapid emergence of acquired resistance, largely mediated by the DNA repair enzyme, methylguanine methyltransferase (MGMT). To investigate the interplay between tumor heterogeneity, drug resistance, and the extracellular matrix (ECM) microenvironment, we adapted a 3D methacrylamide-functionalized gelatin (GelMA) hydrogel model to study the behavior of mixed populations of TMZ-sensitive and TMZ-resistant GBM cells. Using both single-cell distributions and multicellular spheroids, we report the impact of heterogeneous cell populations and TMZ dosing regimens, including physiological, supraphysiological, and metronomic TMZ schedules, on drug response and migration. We show that the combination therapy of TMZ with an MGMT inhibitor, lomeguatrib, can modulate TMZ resistance in vitro. This hydrogel model enables systematic investigation of GBM heterogeneity, “go-or-grow” phenotypic plasticity, and therapeutic resistance in an ECM-rich microenvironment, offering a valuable platform for future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbe93715d79528e499ebbccc2bfd3c31f1cbde8" target='_blank'>
              Multicellular Model of Temozolomide Resistance in Glioblastoma Reveals Phenotypic Shifts in Drug Response and Migratory Potential
              </a>
            </td>
          <td>
            Victoria Kriuchkovskaia, Ela K. Eames, Sydney A McKee, Paul J. Hergenrother, Rebecca B. Riggins, Brendan A. C. Harley
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Recently, contrary to attacking cancer cells, the tumor microenvironment (TME) with genomic stability and vulnerable nature has emerged as promising therapeutic targets in hepatocellular carcinoma (HCC). Within TME ecosystem, tumor-associated macrophages (TAMs) play a pivotal role in tumor evasion and progression of HCC. However, their clinical and therapeutic implications remain unexplored. Methods Utilizing a large-scale sc-RNA seq dataset, a landscape of HCC cellular ecosystem was depicted. Based on previous literature, an effectively differential TAMs subset classification was identified. Gene variations was extracted through trajectory analysis and then unsupervised clustering was conducted within RNA-seq data. Subsequently, survival analysis, specific pathway enrichment as well as hub regulatory network analysis were performed. Additionally, the immune cell infiltration and genomic variations were evaluated between clusters. Drug sensitivity and underlying therapeutic molecular were also explored. Through multiple immunofluorescence, our findings were verified. Results Herein, integrating single-cell RNA sequencing (scRNA-seq) and bulk RNA-seq data, we established a novel TAM classification system based on mutually exclusive SPP1 and FOLR2 signatures. According to the TAM trajectory genes, unsupervised clustering stratified HCC into three distinct clusters. Cluster 3 (C3), which is characterized by metabolic dysregulation and immunosuppressive TME, exhibited the poorest prognosis among the three groups. Hub network analysis of C3 further indicated its characteristic dysregulation of liver-specific metabolism. SPP1 was identified as a key signature of C3, which contributed to suppressing the infiltration of CD8+ T cells. Therapeutic evaluation revealed that C3 were sensitive to chemotherapy and tyrosine kinase inhibitors, while those C1 and C2 were more suitable for immunotherapy. Drug screening identified potential therapeutic compounds for each cluster. Conclusion This study redefines the heterogeneity of TAMs beyond the M1/M2 paradigm, linking the TAMs trajectory genes to HCC patient stratification. SPP1 blockade emerged as a strategy for counteracting immunosuppression, and cluster-specific therapies may optimize the management of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d85b5a9a5d2466c00caeea207b03b8145b6f159" target='_blank'>
              SPP1+ tumor-associated macrophages define a high-risk subgroup and inform personalized therapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Wei-Xuan Xu, Yanjun Ye, Jia-Lin Chen, Xin-Ying Guo, Chen Li, Juan Luo, Lin-Bin Lu, Xiong Chen
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) has emerged as a powerful tool to map gene expression patterns to the local tissue structure in cancer, enabling unprecedented insights into cellular heterogeneity and tumour microenvironments. As the technology matures, developing new, spatially informed analytical frameworks will be essential to fully leverage its potential to elucidate the complex organisation and emerging properties of cancer tissues. Here, we highlight key challenges in cancer spatial transcriptomics, focusing on three emerging topics: (a) defining cell states, (b) delineating cellular niches and (c) integrating spatial data with other modalities that can pave the way towards clinical translation. We discuss multiple analytical approaches that are currently implemented or could be adapted in the future in order to tackle these challenges, including classical biostatistics methods as well as methods inherited from geospatial analytics or artificial intelligence. In the rapidly expanding landscape of ST, such methodologies lay the foundation for biological discoveries that conceptualise cancer as an evolving system of interconnected niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9031fde1ffd2ce70d2c223e1b5d79202f9418532" target='_blank'>
              Decrypting cancer's spatial code: from single cells to tissue niches.
              </a>
            </td>
          <td>
            Cenk Celik, Shi Pan, Eloise Withnell, Hou Wang Lam, M. Secrier
          </td>
          <td>2025-07-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer cell morphology reflects the dynamic interplay between genetic, epigenetic, and environmental factors that
drive tumor progression. This review explores the morphological evolution of cancer cells as a phenotypic manifestation of
underlying molecular dynamics, drawing on recent advances in evolutionary biology, cytoskeletal remodeling, computational
pathology, and tumor metabolism. Clonal evolution, both gradual and punctuated, generates morphological diversity that is
shaped by selective pressures from the tumor microenvironment and therapeutic interventions. Studies highlights the structural
and mechanical properties, such as reduced cellular stiffness and altered actin organization, are linked to metastatic potential
and cytoskeletal deregulation. Oncogenic signaling pathways (e.g., PI3K/AKT, Ras/MAPK) and metabolic reprogramming
further modulate cell shape and behavior. High-throughput imaging and machine learning have enabled quantification of both
static and dynamic morphological traits, correlating them with chromosomal instability and clinical outcomes. Morphological
heterogeneity has emerged as a powerful diagnostic and prognostic biomarker, with deep learning-derived metrics providing
scalable tools for cancer classification and risk stratification. Collectively, these insights underscore the clinical utility of
integrating morphological analysis with molecular profiling, offering new directions for personalized cancer diagnosis and
therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055067ac5bdde412ce5a95b6486dff3c99db46a4" target='_blank'>
              Morphology Matters: A Multidimensional Perspective on Cancer Cell Biology
              </a>
            </td>
          <td>
            Rose Rani
          </td>
          <td>2025-06-30</td>
          <td>International Journal for Research in Applied Science and Engineering Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutated oncogenic Kirsten rat sarcoma virus (KRAS) antigen is expressed in a large variety of cancers, including pancreatic, colorectal, and pulmonary cancers. The oncogenic KRAS mutations cause malignancies and are usually ubiquitously expressed by all cells of a tumor. The KRAS amino acid substitutions at the positions 12 or 13 are among the most frequent mutations in human cancers. Here, we developed immuno-oncotherapeutic non-integrative lentiviral vectors encoding a segment encompassing KRASG12D, either alone, or associated with antigen carriers. These carriers can improve the intracellular antigen routing to major histocompatibility complex presentation machineries or provide universal helper CD4+ epitopes. Immunotherapy with one of these vectors resulted in significant immune control of tumor growth in colorectal or pulmonary preclinical cancer models, in several murine genetic backgrounds. The antitumor effect was correlated with increased proportions of intra-tumoral hematopoietic cells and notably CD8+ T cells. Although this effect was partial, it was robust, reproducible and advantageously combinable with conventional chemotherapies and immunotherapies to improve antitumor protection. Therefore, this approach shows promise as an immuno-oncotherapy against KRAS-mediated malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61951127f6ba1a1f776a4ed744c835e44ef0de6c" target='_blank'>
              A lentiviral vector targeting a KRAS neoepitope for cancer immunotherapy
              </a>
            </td>
          <td>
            Anastasia Goloudina, Fabien le Chevalier, P. Authié, Sylvain Ciret, K. Nemirov, Ingrid Fert, F. Moncoq, B. Vesin, A. Noirat, Catherine Blanc, Yu Wei, Pierre Charneau, L. Majlessi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Despite substantial progress in discovering novel therapeutic targets and compounds, cancer hyperproliferative cells (CHCs) play a central role in driving tumor progression in gastric cancer (GC). High-resolution single-cell RNA sequencing (scRNA-seq) integrated with spatial transcriptomics sequencing (ST-seq) elucidating the role of the CHC microenvironment in driving GC progression remains lacking. Methods Through integrated analysis of 40 scRNA-seq samples and 6 ST samples from human gastric tissues spanning tumor progression, we delineate the phenotypic plasticity of tumor epithelium and characterize the transcriptional trajectory from normal gastric cells to CHCs, a process associated with RRM2 overexpression. Finally, we validated the analysis results through cellular and animal experiments. Results Our findings demonstrate that RRM2 knockdown triggers ferroptosis in GC cells by promoting lipid peroxidation and intracellular iron accumulation. We further screen a compound and RRM2 inhibitor, osalmid, which effectively suppresses tumor growth in GC cell xenografts by triggering ferroptosis, thus offering a new therapeutic option for GC. Conclusions This study elucidates the critical role of RRM2 + CHCs in gastric cancer development and progression, providing novel insights for targeted therapies. The discovery of osalmid as a potential therapeutic agent offers a promising new treatment strategy for GC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06847-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad62270a5ce806049ffe5312712ee483297a8c1" target='_blank'>
              Integration of scRNA-seq and ST-seq identifies hyperproliferative RRM2+ cells features and therapeutic targets in gastric cancer
              </a>
            </td>
          <td>
            Shuai Ping, Xiong Jia, Yanan Tian
          </td>
          <td>2025-07-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is a debilitating disease that accounts for an estimated 890,000 new cases per year. Despite advancements in chemotherapy, radiotherapy, surgery and immunotherapy, the prognosis of HNSCC has remained relatively unchanged for more than a decade. Insight into the tumour microenvironment (TME) using spatially resolved approaches, and its association with clinical endpoints, may provide useful prognostic tools and refine current treatment outcomes. Here, we profiled 84 mucosal HNSCC tissue samples using next-generation ultra-high plex spatial protein profiling (580-proteins, Immuno-Oncology Proteome Atlas (IPA)) and spatial transcriptome mapping (18,000 mRNA, Whole Transcriptome Atlas (WTA)) from Bruker Spatial Biology. Samples were collected during tumour resection, after which patients went on to receive either chemotherapy and/or radiotherapy. Each sample was subdivided into tumour and stromal regions prior to digital spatial profiling. We found that patient survival outcomes were associated with anatomical subsite and tumour stage. Independent validation of key proteomic findings (including CD34 and CD44) was performed using single-cell protein profiling (PhenoCycler-Fusion, Akoya Biosciences). Harnessing the breadth of the 580-plex protein panel and WTA, we identified region-specific proteins and RNA that associate with patient survival. These findings include the expression of immune-specific proteins, CD3e and CXCR5, differentially expressed in the tumour compartment and indicate that the location of these immune signals is important for understanding disease progression. Taken together, this study provides a systematic workflow for the discovery and validation of high-plex protein and transcriptomic profiling in mucosal HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f4a98f332ffeb43581fc45c7b779e9477f23e65" target='_blank'>
              The development of a high-plex spatial proteomic methodology for the characterisation of the head and neck tumour microenvironment
              </a>
            </td>
          <td>
            Chin Wee Tan, Naomi Berrell, Meg L. Donovan, J. Monkman, Clara Lawler, H. Sadeghirad, V. Y. Naei, Jinjin Chen, Alyssa Rossenbloom, Shilah A Bonnett, Mark Conner, B. Filanoski, Christine Kang, Joseph M. Beechem, John F. Fraser, Catherine Barnett, Rahul Ladwa, Brett G M Hughes, A. Kulasinghe
          </td>
          <td>2025-06-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background/Objectives: Brain metastasis (BM) is a common and often early manifestation in lung adenocarcinoma (LUAD), yet its tumor microenvironment remains poorly defined at the time of initial diagnosis. This study aims to characterize early immune microenvironmental alterations in synchronous BM using spatial proteomic profiling. Methods: We performed digital spatial proteomic profiling using the NanoString GeoMx platform on formalin-fixed paraffin-embedded tissues from five treatment-naïve LUAD patients in whom BM was the initial presenting lesion. Paired primary lung and brain metastatic samples were analyzed across tumor and stromal compartments using 68 immune- and tumor-related protein markers. Results: Spatial profiling revealed distinct expression patterns between primary tumors and brain metastases. Immune regulatory proteins—including IDO-1, PD-1, PD-L1, STAT3, PTEN, and CD44—were significantly reduced in brain metastases (p < 0.01), whereas pS6, a marker of activation-induced T-cell death, was significantly upregulated (p < 0.01). These alterations were observed in both tumor and stromal regions, suggesting a more immunosuppressive and apoptotic microenvironment in brain lesions. Conclusions: This study provides one of the first spatially resolved proteomic characterizations of synchronous BM at initial LUAD diagnosis. Our findings highlight early immune escape mechanisms and suggest the need for site-specific immunotherapeutic strategies in patients with brain metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b71912171aef6f8867b2853c36b6318ebf2376ae" target='_blank'>
              Spatial Omics Profiling of Treatment-Naïve Lung Adenocarcinoma with Brain Metastasis as the Initial Presentation
              </a>
            </td>
          <td>
            Seoyeon Gwon, Inju Cho, Jieun Lee, Seung Yun Lee, Kyue-Hee Choi, Taejung Kim
          </td>
          <td>2025-07-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cutaneous melanoma is an aggressive cancer with an increasing incidence worldwide, highlighting the need for research into its pathogenesis. The tumor microenvironment (TME) plays a critical role in melanoma progression and consists of cellular components and an extracellular matrix (ECM) rich in cytokines and signaling molecules. The most abundant stromal cells within the TME are cancer-associated fibroblasts (CAFs), which remodel the ECM and modulate immune responses. Among immune cells, tumor-associated macrophages (TAMs) predominate, and their polarization toward the M2 phenotype supports tumor progression. Tumor-infiltrating lymphocytes (TILs) have diverse functions, including cytotoxic T-cells, helper T-cells that modulate immune response, B-cells forming tertiary lymphoid structures (TLS), and regulatory T-cells with immunosuppressive properties. Dendritic cells (DCs) also play a complex role in the TME. A notable subpopulation are mature regulatory dendritic cells (mregDCs), which contribute to immune evasion. All of these TME components may drive tumorigenesis. Advancements in melanoma treatment—including immunotherapy and targeted therapies—have significantly improved outcomes in advanced-stage disease. In parallel, emerging approaches targeting the tumor microenvironment and gut microbiome, as well as personalized strategies such as neoantigen vaccines and cell-based therapies, are under active investigation and may further enhance therapeutic efficacy in the near future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00d1c207a44eb08e471b7d10318dde99844e3005" target='_blank'>
              Tumor Microenvironment in Melanoma—Characteristic and Clinical Implications
              </a>
            </td>
          <td>
            Hubert Sikorski, M. Żmijewski, Anna Piotrowska
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Understanding the interactions between specific cell types within tissue environments is essential for elucidating key biological processes, such as immune responses, cancer progression, inflammation, and development, in both physiological and pathological studies. The predominant methods for analyzing cell–cell interactions (CCI) rely primarily on statistical inference using mapping or network-based techniques. However, these approaches often struggle to prioritize meaningful interactions owing to the high sparsity and heterogeneity inherent in single-cell RNA sequencing (scRNA-seq) data, where small but biologically important differences can be easily overlooked. To overcome these limitations, we developed PriorCCI, a deep-learning framework that leverages a convolutional neural network (CNN) alongside Grad-CAM++, an explainable artificial intelligence algorithm. This study aims to provide a scalable, interpretable, and biologically meaningful framework for systematically identifying and prioritizing key ligand–receptor interactions between defined cell-type pairs from single-cell RNA-seq data, particularly in complex environments such as tumors. PriorCCI effectively prioritizes interactions between cancer and other cell types within the tumor microenvironment and accurately identifies biologically significant interactions related to angiogenesis. By providing a visual interpretation of gene-pair contributions, our approach enables robust inference of gene–gene interactions across distinct cell types from scRNA-seq data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8d0cda95b391f509ed11c400e323a325085c153" target='_blank'>
              PriorCCI: Interpretable Deep Learning Framework for Identifying Key Ligand–Receptor Interactions Between Specific Cell Types from Single-Cell Transcriptomes
              </a>
            </td>
          <td>
            Hanbyeol Kim, Eunyoung Choi, Yujeong Shim, J. Kwon
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8a7efbbd0fffc709e336945d36a6f61377b28f" target='_blank'>
              Intratumoral heterogeneity in microsatellite instability at single cell resolution
              </a>
            </td>
          <td>
            Harrison Anthony, C. Seoighe
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc4d33cb171c95693c151dadaf1339c2b7a61c7" target='_blank'>
              SCLC-TumorMiner: A Directly Accessible Genomics Resource for Precision Oncology – Big Data for Small Cells
              </a>
            </td>
          <td>
            Fathi Elloumi, A. Dhall, Daiki Taniyama, Augustin Luna, Yasuhiro Arakawa, Sudhir Varma, Jeffrey Wang, Anisha Tehim, M. Raffeld, Kenneth J Aldape, Christophe E. Redon, Roshan Shresta, William C Reinhold, M. Aladjem, Jaydira Del Rivero, Nitin Roper, Anish Thomas, Y. Pommier
          </td>
          <td>2025-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common primary brain cancer. It causes death mainly by local invasion via several routes, including infiltration of white matter tracts and penetration of perivascular spaces. However, the pathways that mediate these invasion routes are only partly known. Here, we conduct an integrative study to identify cell states and central drivers of route-specific invasion in GBM. Combining single-cell profiling and spatial protein detection in patient-derived xenograft models and clinical tumor samples, we demonstrate a close association between the differentiation state of GBM cells and their choice of invasion route. Computational modeling identifies ANXA1 as a driver of perivascular involvement in GBM cells with mesenchymal differentiation and the transcription factors RFX4 and HOPX as orchestrators of growth and differentiation in diffusely invading GBM cells. Ablation of these targets in tumor cells alters their invasion route, redistributes the cell states, and extends survival in xenografted mice. Our results define a close association between GBM cell differentiation states and invasion routes, identify functional biomarkers of route-specific invasion, and point toward targeted modulation of specific invasive cell states as a therapeutic strategy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bd579d99eaa69724a2c2c47d55f912d2748b0a9" target='_blank'>
              The invasion phenotypes of glioblastoma depend on plastic and reprogrammable cell states
              </a>
            </td>
          <td>
            Milena Doroszko, Rebecka Stockgard, Irem Uppman, Josephine Heinold, Faidra Voukelatou, H. Mangukiya, T. Millner, Madeleine Skeppås, Mar Ballester Bravo, Ramy Elgendy, Maria Berglund, Ludmila Elfineh, C. Krona, Soumi Kundu, Katarzyna Koltowska, Silvia Marino, Ida Larsson, Sven Nelander
          </td>
          <td>2025-07-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract As drug and natural product libraries expand, assays for assessing mechanisms of action (MoA) are increasingly critical. Performing cytological profiling using the Cell Painting (CP) assay enables image-based profiling of cellular states upon treatment, yet many bioactive compounds remain uncharacterized due to undetectable cellular effects under standard conditions. To address this, we combined drug dosing with cell activation using the protein kinase C (PKC) agonist phorbol myristate acetate (PMA). Profiling A549 lung cancer cells treated with 8,387 compounds at two concentrations (1 and 10 µM) in both resting and PMA-activated states allowed us to detect phenotypic effects for up to 40% of all screened compounds, effectively illuminating new phenotypic “dark space”. Over 1,000 compounds exhibited phenotypes exclusively under PMA activation, establishing its advantage for MoA studies. We introduce novel quality control measures for CP screens and demonstrate that integrating phenotypic signatures enhances MoA discovery. Notably, 2-methoxycinnamaldehyde clustered with glucocorticoid receptor modulators and induced nuclear translocation, emphasizing the power of this approach in uncovering novel drug mechanisms and, therefore, aiding in improving therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a69a88bdf0eb86999887480ebc0a5b9adfd4cbb0" target='_blank'>
              Cell Painting in activated cells illuminates phenotypic dark space and uncovers novel drug mechanisms of action
              </a>
            </td>
          <td>
            Matylda Zietek, Akshar Lohith, Derfel Terciano, Beverley Marie Rabbitts, Aswad S. Khadilkar, John B MacMillan, Scott Lokey
          </td>
          <td>2025-06-23</td>
          <td>Research Square</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
Atypical teratoid rhabdoid tumors (ATRTs) are highly aggressive pediatric central nervous system tumors defined by the inactivation of the SMARCB1 gene. Despite the identification of three distinct molecular subtypes, each defined by unique clinical and molecular characteristics, no subtype-specific therapeutic strategies are currently available. This highlights an urgent need to deepen our understanding of the cellular heterogeneity and developmental origins of ATRTs.


METHODS
We generated a comprehensive single-nucleus transcriptomic atlas of ATRT samples, integrated it with single-nucleus ATAC-seq and spatial transcriptomics data, and validated our findings experimentally using patient-derived ATRT tumoroid models.


RESULTS
Our analyses revealed distinct subtype-specific differentiation trajectories, each resembling different brain progenitor lineages. We identified key transcription factors that appear to drive these developmental pathways. Furthermore, a shared cycling, intermediate precursor cell (IPC)-like cell population, interspersed throughout tumors, was consistently present within all ATRT samples. We demonstrate that these subtype-specific differentiation pathways can be pharmacologically manipulated in patient-derived ATRT tumoroids. By directing tumor cells along their respective subtype-specific trajectories, we were able to induce a shift toward more differentiated, non-proliferative states.


CONCLUSIONS
Collectively, our findings show that ATRTs recapitulate fetal brain signaling programs in a subtype-specific manner. This work provides a framework for understanding ATRT heterogeneity and supports the feasibility of maturation-based therapeutic strategies tailored to the molecular subtype of the tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f089b7cc6f0aae306ce7c5e4e459d6bc20cd73eb" target='_blank'>
              A cycling, progenitor-like cell population at the base of atypical teratoid rhabdoid tumor subtype differentiation trajectories.
              </a>
            </td>
          <td>
            Enrique Blanco-Carmona, Irene Paassen, Jiayou He, Jeff DeMartino, Annette Büllesbach, Nadia Anderson, Juliane L Buhl, A. Federico, Monika Mauermann, Mariël Brok, Karin Straathof, S. Behjati, Rajeev Vibhakar, A. Donson, Nicholas K Foreman, McKenzie L. Shaw, M. Frühwald, A. Korshunov, M. Hasselblatt, C. Thomas, Niels Franke, M. Kranendonk, E. Hoving, N. Jäger, P. Johann, Stefan M Pfister, Mariella G. Filbin, M. Kool, J. Drost
          </td>
          <td>2025-07-28</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Embryonal tumor with multilayered rosettes (ETMR) is a lethal embryonal brain tumor entity. To investigate the intratumoral heterogeneity and cellular communication in the tumor microenvironment (TME), we analyze in this work single-cell RNA sequencing of about 250,000 cells of primary human and murine ETMR, in vitro cultures, and a 3D forebrain organoid model of ETMR, supporting the main findings with immunohistochemistry and spatial transcriptomics of human tumors. We characterize three distinct malignant ETMR subpopulations - RG-like, NProg-like and NB-like - positioned within a putative neurodevelopmental hierarchy. We reveal PDGFRβ+ pericytes as key communication partners in the TME, contributing to stem cell signaling through extracellular matrix-mediated interactions with tumor cells. PDGF signaling is upregulated in chemoresistant RG-like cells in vivo and plays a role in recruiting pericytes to ETMR TME by finalizing a signaling cascade which promotes the differentiation of non-malignant radial glia cells, derived from our 3D model, into pericyte-like cells. Selective PDGFR-inhibition blocked the lineage differentiation into pericytes in vitro and reduced the tumor cell population in vivo. Targeting ETMR-pericyte interactions in the TME presents a promising therapeutic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/497d2ed06cfc62eb4cc72976940d7e3d7d08672f" target='_blank'>
              ETMR stem-like state and chemo-resistance are supported by perivascular cells at single-cell resolution
              </a>
            </td>
          <td>
            F. D. de Faria, Nicole C. Riedel, Daniel Münter, M. Interlandi, C. Göbel, L. Altendorf, Mathis Richter, V. Melcher, C. Thomas, Rajanya Roy, Melanie Schoof, Ivan Bedzhov, N. Moreno, M. Graf, M. Hotfilder, D. Holdhof, Wolfgang Hartmann, Ann-Katrin Bruns, A. Brentrup, F. Liesche-Starnecker, B. Maerkl, S. Sandmann, J. Varghese, M. Dugas, Pedro H Pinto, S. Balbach, I-Na Lu, Claudia Rossig, Oliver Soehnlein, Aysegül Canak, Martin Ebinger, Martin U Schuhmann, J. Schittenhelm, Michael F Frühwald, U. Schüller, T. Albert, Carolin Walter, Jan M Bruder, K. Kerl
          </td>
          <td>2025-06-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Immune checkpoint inhibitors show limited efficacy in tumors with low tumor mutational burden, partly due to insufficient neoantigen presentation. Methods We developed a novel approach for neoantigen identification using circulating tumor cells (CTCs) isolated via leukapheresis and flow cytometry. Peripheral blood mononuclear cells (PBMCs) were collected from 11 stage IV cancer patients and 2 healthy volunteers. CTCs were enriched by depleting CD45+ hematopoietic cells and selecting CD45−Vimentin+ cells, which were confirmed cytologically to contain malignant cells. Hematopoietic lineage analysis showed that over 50% of the CTC fraction consisted of non-hematopoietic cells. DNA extracted from both the CTC and normal hematopoietic fractions underwent exome sequencing. Neoantigens were identified using the Ancer® bioinformatics platform. Results In representative patients with gastric and salivary gland cancers, 94,636 and 46,423 CTCs were isolated, respectively. DNA yields were sufficient for exome sequencing without amplification or extensive cell culture. A total of 102 (patient with gastric cancer) and 108 (patient with salivary gland cancer) neoantigens were identified in each subject, including high-ranking T-cell epitopes derived from single nucleotide variants and frameshift mutations. According to the same procedures we could successfully identify a large number of neoantigens from the CTCs of all stage IV cancer patients. This confirms the feasibility of identifying individual patient-specific neoantigens from CTCs without requiring tumor biopsies. Conclusions This is the first study to demonstrate successful neoantigen identification using non-amplified CTCs isolated by apheresis and flow cytometry. The approach provides a minimally invasive, scalable alternative for neoantigen discovery and may better capture tumor heterogeneity compared to single-site biopsies. This method holds promise for enabling rapid, personalized immunotherapy strategies, including peptide vaccines, dendritic cell vaccines, and mRNA-based treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f2f611c004eda4f555a0f11de496ee09c9cb0f" target='_blank'>
              A method for identifying neoantigens through isolation of circulating tumor cells using apheresis among patients with advanced-stage cancer
              </a>
            </td>
          <td>
            Daiki Kobayashi, Takuya Kosumi, Queenie Lai Kwan Lam, Shigeharu Fujita, Yasuki Hijikata, Kaori Takeda, Tomoya Narita, Naomi Yamashita, Guilhem Richard, Anne S De Groot, Naohide Yamashita
          </td>
          <td>2025-07-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d94bb593ee184379443d2fb409c739871b84fe" target='_blank'>
              tugMedi: simulator of cancer-cell evolution for personalized medicine based on the genomic data of patients
              </a>
            </td>
          <td>
            Iurii S. Nagornov, E. Furukawa, M. Nagai, S. Yagishita, Tatsuhiro Shibata, M. Kato
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd9e961ad1f345af55769759a723ac6d14212120" target='_blank'>
              Integrated genetic screening reveals FEN1 as a driver of stemness and temozolomide resistance in glioblastoma
              </a>
            </td>
          <td>
            Benjamin A Brakel, D. Mckenna, Anish Puri, V. Shaikh, Manoj Singh, Ali Saleh, Abdo-jose Tomajian, N. Mikolajewicz, Marcello Beltrami, Alisha Anand, P. Miletic, Kevin R. Brown, D. Tieu, W. Maich, Sabra K. Salim, Yujin Suk, M. Subapanditha, Deena M. A. Gendoo, C. Venugopal, Jason Moffat, Sachin Katyal, C. Chokshi, Sheila K Singh
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a highly heterogeneous disease, with castration-resistant prostate cancer (CRPC) and neuroendocrine prostate cancer (NEPC) representing its most aggressive and therapy-resistant forms. Emerging evidence indicates that lineage plasticity—driven by key transcription factors such as Octamer Binding Factor 4 (OCT4)—plays a crucial role in therapeutic resistance and disease progression. OCT4, in coordination with SOX2 and NANOG, acts as a master regulator of stemness and is frequently upregulated in prostate cancer stem cells (PCSCs). This upregulation contributes to tumor initiation, metastasis, and resistance to both androgen deprivation therapy (ADT) and chemotherapy. In this review, we explore the role of OCT4 in mediating lineage plasticity in prostate cancer, with particular emphasis on its involvement in treatment resistance and neuroendocrine differentiation. We also examine therapeutic strategies aimed at targeting OCT4 directly, such as microRNA-mediated suppression, small-molecule inhibitors, and suicide gene therapy, as well as indirect approaches that modulate OCT4 expression via FGFR and NF-κB signaling pathways. While these strategies offer promising avenues, challenges such as adaptive resistance and the intricate signaling networks within PCSCs remain significant hurdles. A deeper understanding of the molecular mechanisms underlying OCT4-driven plasticity may pave the way for novel therapeutic approaches and improved outcomes in advanced prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a8ce283789b3c96575337b7647df8d11de9b2ba" target='_blank'>
              Unlocking the Role of OCT4 in Cancer Lineage Plasticity: A Cross-Cancer Perspective with an Emphasis on Prostate Cancer
              </a>
            </td>
          <td>
            Mohammad Esfini Farahani, Yanquan Zhang, A. O. Akinyemi, Fatemeh Seilani, Md Rakibul Alam, Xiaoqi Liu
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent advances in single-cell technologies have enabled the creation of comprehensive cell atlases, reference maps of various cell types within organisms. Here we specifically focus on T cell atlases, which offer a detailed catalog of the adaptive immune system at single-cell resolution. As such, they capture cellular diversity, functional states, and spatial dynamics across tissues, developmental stages, and disease conditions. Given the central role of T cells in orchestrating immune responses, their dysregulation underpins autoimmune disorders, cancer progression and failed immunotherapies. Therefore, a unified T cell atlas is critical for decoding such disease mechanisms, identifying therapeutic targets, and advancing personalized treatments. In this article, we explore the latest advances in T cell atlases, describing breakthroughs in multi-omics technologies, spatial profiling and computational frameworks that resolve transcriptional, epigenetic and proteomic heterogeneity. We also address persistent challenges and highlight strategies to address these gaps. Finally, we discuss emerging frontiers set to reshape our understanding of T cell dynamics in both health and diseases. Together, these insights underscore the transformative potential of T cell atlases in reconstructing precision immunology and accelerating therapeutic innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d0b0e5dda9a3845db88c70250149559b1315699" target='_blank'>
              Sketching T cell atlases in the single-cell era: challenges and recommendations.
              </a>
            </td>
          <td>
            Itana Bojović, António G. G. Sousa, S. Junttila, L. Elo
          </td>
          <td>2025-06-29</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d4e9d24e9c0c4b081e25cd4410c5971c24a9285" target='_blank'>
              Microenvironmental TGF-β is an early driver of NF1-associated tumour formation
              </a>
            </td>
          <td>
            Alex Power, Salome Stierli, Elizabeth Harford-Wright, Guillem Mòdol-Caballero, Emma Lloyd, Stella Kouloulia, Giulia Casal, Lara Krasinska, Omar Bouricha, Ilaria Napoli, Sara Ribeiro, C. Venturini, M. Clements, S. Parrinello, Alison C. Lloyd
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Liver diseases involve many distinct and functionally diverse cell types, but it remains unclear which specific cellular states contribute to inherited genetic risk. In this study, we integrated genetic data from several liver diseases with single-cell gene expression profiles from human liver tissue to identify disease-associated cells at single-cell resolution. By focusing on cellular heterogeneity, we found that genetic risk is not associated with broad cell types, but with distinct subpopulations defined by their functional states. In particular, liver sinusoidal endothelial cells and cholangiocytes showed disease associations only when in immune-activated or stress-responsive states. For example, autoimmune and viral liver diseases were linked to inflammatory subsets, not their metabolically active counterparts. These findings demonstrate that disease risk is shaped by specific cell states within heterogeneous populations. Our study underscores the need to consider cell-state diversity when studying disease mechanisms and developing targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/192d524a339350f75aa0e4bcaad1a36c461732e3" target='_blank'>
              Dissecting the Cellular Heterogeneity Underlying Liver Diseases Through the Integration of GWASs and Single-Cell RNA Sequencing
              </a>
            </td>
          <td>
            Miao Zhou, Meng Liu, Chao Xue
          </td>
          <td>2025-06-27</td>
          <td>Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Breast cancer exhibits heterogeneity at various levels, from subtype differences between patients (inter-patient heterogeneity) to diverse malignant cell compositions, variability in hormone receptor (HR) expression, and cellular makeup within single breast cancer samples (intra-tumour heterogeneity). Despite the availability of numerous effective therapies, many patients still experience incomplete treatment responses and subsequent relapse. These adverse outcomes are often due to the critical but frequently overlooked factor of cellular heterogeneity.
 To gain deeper insights into intra-tumour heterogeneity, we applied single-cell technologies to a cohort of 250 primary, untreated breast cancers. We optimized tissue cryopreservation methods, eliminating the need for fresh sample processing, and developed multiplex tissue profiling techniques. These cost-efficient processes improve handling of small tissue sizes, such as biopsies, and reduce batch effects. To ensure accurate and reliable data processing, we created a scalable computational workflow that includes benchmarked methods for sample SNP-demultiplexing, doublet detection, high-resolution cell annotation, and cellular integration. Additionally, we extended our existing methods to study the cellular heterogeneity of breast cancers.
 Our method, scSubtyper, examines the phenotypic differences between malignant cells within tumours by comparing each single cell to features associated with different molecular subtypes and assigning each cell to one of these subtypes. Our previous study and preliminary results from this project revealed that over 90% of samples exhibit a mix of malignant cells of different subtypes, and 50% of samples contain cells with characteristics of all subtypes, demonstrating that cellular heterogeneity exists not only between malignant cells but also within malignant cells of a tumour. Our second approach, known as ecotyping, assesses patterns of cell type frequencies across samples and groups them based on the similarity of these co-occurrences. Preliminary results have revealed the existence of five ecotypes that lack significant associations with clinical subtypes. Application of the same approach exclusively within the HR-positive samples identified four ecotypes characterized by distinct abundances of immune and stromal cells. This analysis revealed that ecotypes are not simple surrogates for clinical and molecular subtypes, but their presence could influence different responses to treatment.
 Together, our high-throughput tissue processing and computational approaches to study intra-tumour heterogeneity are now being applied to our large, well-annotated clinical cohort. Supported by preliminary results, we hypothesize that this study will play a vital role in optimizing breast cancer patient stratification to improve treatment management and outcomes.
 Citation Format: Beata Kiedik, Daniel L. Roden, Kate Harvey, Ghamdan Al-Eryani, Sunny Z. Wu, Mun N. Hui, Sandra O’Toole, Elgene Lim, Charles M. Perou, Alexander Swarbrick. Exploring cellular heterogeneity of localised breast cancers [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-09-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/355a116d2a0ae027b0d9081aa9023a8465a9971c" target='_blank'>
              Abstract P4-09-19: Exploring cellular heterogeneity of localised breast cancers
              </a>
            </td>
          <td>
            Beata Kiedik, D. Roden, K. Harvey, G. Al-Eryani, Sunny Z. Wu, M. Hui, Sandra O’Toole, Elgene Lim, C. Perou, Alex Swarbrick
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC), a prevalent and highly lethal malignancy, is notorious for its aggressive nature and inherent tendency to metastasize, posing significant challenges in clinical management and prognosis. Hypoxia, a pivotal characteristic of the tumor microenvironment (TME) in hepatocellular HCC, is intimately linked to disease progression and unfavorable patient outcomes, underscoring its critical role in shaping the malignant behavior of this cancer. Methods Our research leveraged single-cell RNA sequencing technology to dissect the heterogeneity of the HCC TME, focusing on hypoxia-related genes. To probe the effects of hypoxia on HCC invasion, we developed the Cell Hypoxia-Related Prognostic Feature (CHPF). We analyzed transcriptome data from the The Cancer Genome Atlas (TCGA) database and the GSE149614 dataset, employing computational methods such as UMAP, Weighted correlation network analysis (WGCNA), and CellChat to identify hypoxia cells, characterize cell subsets, and elucidate intercellular communications. Results Our analysis revealed significant heterogeneity in hypoxia cell populations within the HCC TME, with distinct expression patterns of hypoxia-related genes in neoplastic and immune cells. Our analysis revealed distinct hypoxia subpopulations within HCC, with significant overexpression of genes like MEG3, KLF6 and JUN in hypoxia cells. We identified a unique hypoxia subpopulation with high invasive potential and constructed a prognostic model based on H2-specific transcription factors including LRP10、MED8、NOL10、NOP58 and REXO4. The model demonstrated significant predictive value for lifespan of patients as verified in the TCGA dataset and an external validation group. Conclusion Key transcription factors like NOP58, MED8 play pivotal roles in hypoxia-induced HCC invasion and metastasis, and a predictive model based on these factors forecasts HCC survival. Our findings provide novel molecular markers and therapeutic targets for HCC, highlighting the importance of considering the hypoxia TME in diagnostic and treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03201-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85919f351fa84b022b25ebecfed4ffc0c988bac6" target='_blank'>
              Deciphering hypoxia’s role in hepatocellular carcinoma prognosis with single-cell approaches
              </a>
            </td>
          <td>
            Jun Liang, Di Chen, Huiyu Liang
          </td>
          <td>2025-07-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Evolution and outgrowth of drug-resistant cancer cells is a common cause of treatment failure. Patients with leukemic cutaneous T-cell lymphoma (L-CTCL) have a poor prognosis due to development of drug-resistance and severe bacterial infections. Here, we show that most L-CTCL patients harbor multiple genetically distinct subclones that express an identical clonal antigen receptor but display distinct phenotypes and functional properties. These co-existing malignant subclones exhibit differences in tissue homing, metabolism, and cytokine expression, and respond differently to extrinsic factors like Staphylococcus (S.)aureus and cancer drugs. Indeed, while S. aureus toxins selectively enhance activation and proliferation of certain subclones, these responsive subclones are also the most intrinsically sensitive to cancer drugs when the stimuli are removed. Consequently, although divergent evolution of malignant subclones drives aggressiveness, adaptability and drug-resistance, by removing extrinsic stimuli and mapping malignant subclones, we can expose inherent vulnerabilities that can be exploited in the treatment of these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9c75ce389008c94fe5703552a6c93e0ffef4c7" target='_blank'>
              Divergent Evolution of Malignant Subclones Maintains a Balance Between Induced Aggressiveness and Intrinsic Drug Resistance in T Cell Cancer.
              </a>
            </td>
          <td>
            T. B. Buus, C. K. Vadivel, Maria Gluud, Martin R J Namini, Ziao Zeng, S. Hedebo, Menghong Yin, A. Willerslev-Olsen, Emil M H Pallesen, Lang Yan, E. Blümel, E. U. Ewing, S. Ahmad, Lara P Sorrosal, Carsten Geisler, C. Bonefeld, A. Woetmann, M. Andersen, T. Mustelin, Claus Johansen, Marion Wobser, M. Kamstrup, E. Guenova, Jürgen C. Becker, S. Koralov, R. Bech, N. Ødum
          </td>
          <td>2025-06-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background: The existence of genetically, epigenetically, and phenotypically distinct cell populations inside and between tumors is known as tumor heterogeneity, and it is one of the main barriers to effective cancer treatment. This intricacy affects the likelihood of metastasis, therapeutic resistance, and disease recurrence, rendering single-omics methods and conventional diagnostics inadequate for whole-tumor profiling. As a result, multi-omics methods, which incorporate data from multiple biological layers, such as transcriptomics, proteomics, metabolomics, genomes, and epigenomics, have emerged as powerful tools for thoroughly examining intra- and inter-tumoral complexity. 
Methods: A comprehensive literature synthesis was conducted, emphasizing high-impact studies that illustrate technological innovation and translational impact in multi-omics applications. Key case studies in glioblastoma, non-small cell lung cancer, and breast cancer are highlighted to demonstrate real-world clinical relevance. 
Aims and Objectives: This paper explores the ways in which integrated multi-omics has transformed our understanding of clonal dynamics, tumor growth, and resistance mechanisms while charting a path toward precision oncology. 
Key Insights: The latest methods, such as single-cell multi-omics, spatial transcriptomics, and proteogenomics, were examined, along with computational frameworks including network-based models, probabilistic inference algorithms, and AI-driven tools that make it easier to integrate high-dimensional data. Along with discussing new technologies like in vivo biosensors, organoid-based modeling, and point-of-care omics, the function of the tumor microenvironment, lineage tracing, and liquid biopsies in monitoring the real-time progression of tumors was also discussed. Translational hurdles, including cost, complexity, and ethical issues were addressed while highlighting the importance of equity and worldwide access. 
Conclusion: Multi-omics has great promise for truly personalized oncology by providing the integrated insights required for dynamic monitoring, predictive diagnosis, and tailored therapy design in future cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/093106f85c13011bf6b7a0e38996483840456f08" target='_blank'>
              Decoding Tumor Heterogeneity through Multi Omics: Insights into Cancer Evolution, Microenvironment and Therapy Resistance
              </a>
            </td>
          <td>
            Paul-Miki Raluchukwu Ibekwe, Elizabeth Anuoluwa Akintayo, Cecilia Ndiuwem Okuku, Ismaila Muhammed, Fuhad M. Jeje, Okun, Oseghale, Kemiki Olalekan Ademola, M. D. Badru, Lydia Amarachi Onwuemelem
          </td>
          <td>2025-07-07</td>
          <td>Journal of Cancer and Tumor International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Mechanisms of HIV persistence in tissues are distinct from that in the blood. Spatial transcriptomic profiling examines HIV-infected cells, surrounding neighborhoods, and tissue microenvironment in unprecedented resolution. Spatial profiling captures cytokine gradients, distances between HIV-infected cells and immune effectors (and their function versus exhaustion), and cell-cell interactions. We present an overview of spatial transcriptomic platforms and a workflow of quality controls, sanity check, and bioinformatic analysis.


RECENT FINDINGS
The selection of spatial profiling methods should base on the research question, resolution, breadth of coverage, the expression level of RNA of interest, tissue quality, and tissue size. Advanced spatial transcriptomic profiling can capture RNA molecules at high resolution (<1 μm) and thus enable near-single cell profiling at genome-wide (~20 000 genes) breadth. Specifically, poly-A-based mRNA capture can identify previously unknown targets, while targeted RNA capture increases sensitivity in low-quality tissues. In targeted capture, however, the increase in target numbers frequently decreases sensitivity. Coupling ATAC-seq, protein capture, and T cell receptor sequencing to spatial platforms is ongoing.


SUMMARY
Spatial transcriptomic profiling uncovers mechanisms of HIV persistence in tissues and informs therapeutic strategies. Investigators should ensure the rigor of analysis, validate findings, and avoid reporting signatures with unknown biological significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f09209df407344118779f936014e334d9c123d1" target='_blank'>
              Roadmap for spatial transcriptomics of HIV in tissues.
              </a>
            </td>
          <td>
            Qijie Guan, Amare Eshetu, Ya-Chi Ho
          </td>
          <td>2025-07-07</td>
          <td>Current opinion in HIV and AIDS</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="INTRODUCTION
NPLOC4 (nuclear protein localization 4 homolog) is mainly involved in DNA damage, cell cycle, and ubiquitination promotion. Nonetheless, the role of NPLOC4 in the tumor immune microenvironment (TIME) and its potential as a promising tumor therapeutic target remains unclear.


METHODS
Therefore, analyses of NPLOC4 mRNA and protein expression, RNA subcellular localization, and patient prognosis associated with NPLOC4 expression were conducted across multiple tumor types. Additionally, the correlations between NPLOC4 and immune cells, non-immune cells, and immune molecules within the tumor immune microenvironment (TIME) were investigated. These analyses utilized data from various public resources, including the Genotype-Tissue Expression (GTEx) project, The Cancer Genome Atlas (TCGA), Cancer Cell Line Encyclopedia (CCLE), The Human Protein Atlas (HPA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), TIMER2.0, KM-Plotter, The University of Alabama at Birmingham Cancer Data Analysis Portal (UALCAN), and Tumor Immune Single-cell Hub 2 (TISCH2). Subsequently, we utilized hepatocellular carcinoma (HCC) patients' cancer and adjacent tissues plus tumor cell lines to verify the differential RNA and protein expression of NPLOC4 via qRT-PCR and immunohistochemistry (IHC). Then, the relationship of NPLOC4 expression level with immune infiltration score, infiltration of effector immune cells, suppressive immune cells, and several vital immune checkpoints was analyzed in HCC immune microenvironment. Furthermore, the distribution of expression of NPLOC4 in various cells in the HCC microenvironment was determined through single-cell sequencing analysis.


RESULTS
We discovered that NPLOC4 was up-regulated in a variety of tumors and was correlated with poor prognosis. NPLOC4 not only had the potential as a tumor prognostic marker and therapeutic target but also was strongly linked to immune cells, immune checkpoints, and immune-related molecules and pathways in HCC immune microenvironment.


CONCLUSION
In summary, NPLOC4 may serve as a promising target for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/613e1ad4e4291fbb905e3844227d3da94ff9ee44" target='_blank'>
              NPLOC4 constructs tumor immunosuppressive microenvironment in pan-cancer and hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Wanli Zhang, Chengdong Liu, Xiaohan Zhou
          </td>
          <td>2025-07-17</td>
          <td>Current medicinal chemistry</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Neuroblastomas are highly heterogeneous tumors originating from neural crest-derived cells destined to form the sympathetic nervous system. Nearly half of high-risk tumors present with amplification of the MYCN proto-oncogene. Here, we describe a Mycn-driven, transplantable, non-germline, genetically engineered mouse model (Mycn-nGEMM). Mycn-nGEMM tumors recapitulate the immune-evasive, macrophage-rich tumor microenvironment of high-risk, MYCN-amplified human neuroblastoma. Treatment of tumor-bearing mice with anti-PD-L1, but not anti-PD-1 or anti-CTLA-4, inhibited tumor growth, profoundly remodeling the tumor microenvironment by depleting anti-inflammatory macrophages and increasing T cell infiltration. Surprisingly, while tumor cells showed low expression of PD-L1, anti-inflammatory macrophages from both murine and human neuroblastoma expressed PD-L1. We identified cytokines, including macrophage migration inhibitory factor, secreted by the Mycn-nGEMM cancer cells that drive expression of PD-L1 on macrophages. Combining anti-PD-L1 with CD40 agonist antibodies further improved survival in Mycn-nGEMM mice, demonstrating the potential for myeloid-targeting immunotherapies to overcome inhibitory barriers in immune-evasive neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2aecad2de1b569da0f36a9e846ad42a57aa1628" target='_blank'>
              Myeloid-targeting immunotherapies overcome inhibitory barriers in immune-evasive neuroblastoma.
              </a>
            </td>
          <td>
            Marie Ménard, Hiroyuki Yoda, Nicole Nasholm, Megumi J Barata, Linyu Wang, Erin F. Simonds, Edbert D Lu, Shannon T. Wong-Michalak, Lauren McHenry, Alvin Farrel, Rebecca S. Kaufman, Vanessa Lopez, Rebekah J Kennedy, G. E. Fernandez, Hiroyuki Shimada, Liron D. Grossmann, S. Asgharzadeh, John M. Maris, W. Gustafson, William A Weiss
          </td>
          <td>2025-07-14</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Purpose Programmed cell death (PCD) mechanisms play crucial roles in cancer progression and treatment response. This study aims to develop a PCD scores prediction model to evaluate the prognosis of hepatocellular carcinoma (HCC) and elucidate the tumor microenvironment differences. Methods We analyzed transcriptomic data from 363 HCC patients in the TCGA database and 221 patients in the GEO database to develop a PCD prediction model. Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics sequencing (ST-seq) data from HCC patients were analyzed to investigate the tumor microenvironment and functional disparities. The oncogenic role of the key gene UBE2E1 in the model was explored in HCC through various in vitro experiments. Results Seventeen PCD-related genes were identified as significant prognostic indicators, forming the basis of our PCD prediction model. High-PCD scores correlated with poorer overall survival (OS) and exhibited significant predictive capabilities. scRNA-seq analysis revealed distinct tumor cell characteristics and immune microenvironment differences between high- and low-PCD groups. High-PCD tumors showed increased cell proliferation and malignancy-associated gene expression. T cells in high-PCD patients were more likely to be exhausted, with elevated expression of exhaustion markers. ST-seq data also confirmed these results. Among the genes associated with the PCD prognostic model, UBE2E1 was identified as a key oncogenic marker in HCC. Conclusions The PCD prediction model effectively predicts prognosis in HCC patients and reveals critical insights into the tumor microenvironment and immune cell exhaustion. This study underscores the potential of PCD-related biomarkers in guiding personalized treatment strategies for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88d93e2b7ad6d106b6fbcf29fd3ec02edc2e6591" target='_blank'>
              Integrative spatial and single-cell transcriptomics elucidate programmed cell death-driven tumor microenvironment dynamics in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kai Lei, Yutong Zhao, Shumin Li, Jiawei Liu, Wenhao Chen, Caihong Zhou, Yi Zhang, Jinmei Tan, Jian Wu, Qi Zhou, Jiehui Tan
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We recovered tumor microenvironmental features previously described to associate with therapy response. We subsequently identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.



 Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andreanne Gagne, Samantha Hoffman, Kevin Bi, Breanna Titchen, Connor Hennessey, Joshua Remland, Matthew Carnes, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, Eliezer M. Van Allen. Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b559d25313fad22976c28c00d01803263f1319a3" target='_blank'>
              Abstract A046: Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda E. Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor Hennessey, Joshua Remland, Matthew Carnes, Erin Shannon, Sabrina Y. Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve M Boland, A. Aguirre, Nilay S. Sethi, Valentina Boeva, Eliezer M Van Allen
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 DNA methylation, an essential epigenetic modification, plays a critical role in establishing and maintaining cellular identities by regulating gene expression. In cancer, aberrant DNA methylation patterns contribute to tumor development and progression, yet the cell-type-specific methylation alterations within the tumor microenvironment (TME) remain poorly understood. In breast cancer, a highly heterogeneous malignancy, endothelial cells within the TME (TECs) are key players in tumor angiogenesis and progression. Methylation changes detected in bulk tumor analyses, however, reflect signals from multiple cell types present in the TME, making it challenging to pinpoint cell-type-specific epigenetic changes. Recently, methods have been developed to using DNA methylation to deconvolute cell type proportions which can in turn be used to attribute methylation alterations to specific cell types in the TME. A reference-based cell-type deconvolution algorithm, the Hierarchical Tumor Immune Microenvironment Epigenetic Deconvolution (HiTIMED) method, allows accurate quantification of cell type proportions within the TME. We accessed DNA methylation array data from 609 breast tumors and 331 nontumor normal breast tissues from the Gene Expression Omnibus (GEO) and the Genotype-Tissue Expression (GTEx) portal. To these DNA methylation profiles, we applied HiTIMED and measured tumor and non-tumor cell proportions. Then, applying a statistical interaction model called CellDMC we leveraged the cell type proportion data to identify differential methylation specific to cell types in the breast cancer TME, including TECs and tumor cells. We identified 1897 tumor-cell-specific and 180 TEC-specific differentially methylated cytosines (DMCs) in cancer compared to normal breast tissue (FDR ≤ 1e-10). Importantly, in the cell-type unadjusted differential analysis, tumors exhibited hypermethylation compared to normal tissues; however, in the tumor-cell-specific analysis, tumor cells were hypomethylated relative to normal samples. By applying these advanced techniques to breast cancer samples, our study unveils cell-type-specific DNA methylation changes that are pivotal in driving the pro-tumorigenic environment. These findings not only enhance our understanding of the molecular mechanisms underlying breast cancer progression but also highlight potential therapeutic targets within the TME. The integration of HiTIMED and CellDMC represents a significant methodological advancement in cancer epigenetics, offering a more precise and insightful approach to studying the complex interplay between different cell types within the TME.
 Citation Format: Barbara Barbara, Lucas A. Salas, Brock C. Christensen. Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-29.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77ae964d40b279e042709ea22a740c1da93f0fc" target='_blank'>
              Abstract P5-06-29: Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution
              </a>
            </td>
          <td>
            Barbara Barbara, Lucas A Salas, Brock C. Christensen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Gliomas are highly aggressive, life-threatening tumors with poor prognosis, and remain a leading cause of mortality among brain cancers. Although the role of mitochondrial proteins in cancer has garnered increasing attention, their specific functions in the nervous system, particularly in gliomas, remain poorly understood. Methods We integrated single-cell RNA sequencing with cellular assays and flow cytometry to investigate the molecular characteristics and cellular interactions within glioblastoma subpopulations during tumor progression. Results Single-cell RNA sequencing revealed several differentially expressed genes (DEGs) within glioblastoma subpopulations. Trajectory analysis identified CHCHD2P9 as a pivotal marker for the terminal subpopulation. Moreover, elevated expression of CHCHD2P9 was found to correlate with poorer clinical outcomes. Subsequent cellular experiments further explored the underlying mechanisms driving these observations. Conclusions CHCHD2P9 is significantly overexpressed in glioma patients, and its differential expression plays a crucial role in regulating glioma cell proliferation and migration. A CHCHD2P9-based risk model holds promise as both a prognostic biomarker and a potential therapeutic target, providing novel insights into the pathogenesis of gliomas and opening avenues for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d625edcb2052943e2c1f19b8599a0238510719f1" target='_blank'>
              Molecular insights into glioblastoma progression: role of CHCHD2P9 in tumor heterogeneity and prognosis
              </a>
            </td>
          <td>
            Yanyu Ding, Lizhi Xiao, Xiaoling Zhou, Jiaxin Zhao, Jianli Ke, Hua-Bao Cai, Mengyu Zhao, Cun-Zhi Wang, Tian-Hang Yu, Zhijie Zhao, Yucai Wang, Jiyuan Ke
          </td>
          <td>2025-06-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer immunotherapy is an innovative treatment approach that leverages the immune system to combat tumors, demonstrating significant therapeutic potential. In the past few years, single‐cell RNA sequencing (scRNA‐seq) has made significant progress in the field of cancer immunotherapy, enabling us to understand the complexity of anti‐tumor immune processes with unprecedented depth and precision, thereby facilitating the design of more effective immunotherapy strategies. This review aims to summarize the recent applications of advanced scRNA‐seq technologies in tumor immunology. First, we outline the most representative scRNA‐seq technologies with different technical principles, with a particular focus on single‐cell T cell receptor sequencing. Next, we describe how scRNA‐seq technology is applied to identify the cellular composition and phenotypes within the tumor microenvironment for the construction of immune cell atlas, and uncover key cell types and molecular mechanisms underlying treatment responses for developing novel immunotherapies. Finally, we address the current challenges and future prospects of scRNA‐seq technology in tumor immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0caad9af0b2721a017882aaaa64cafb8d1119d63" target='_blank'>
              Advanced single‐cell RNA sequencing in tumor immunology
              </a>
            </td>
          <td>
            Yilong Liu, Xiyuan Yu, Zhipeng Han, Qiuyang Wei, Qi Niu, Chaoyong Yang, Jie P. Li, Zijian Guo, Huimin Zhang
          </td>
          <td>2025-06-24</td>
          <td>Interdisciplinary Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Despite extensive research efforts, glioblastoma (GBM) remains a deadly disease with poor prognosis. Although previous studies have identified various cell states within GBM tumors, the molecular mechanism underlying adaptive GBM cell plasticity induced by conventional therapy remains unclear.


METHODS
We used fluorescent reporters for proneural (PN) and mesenchymal (MES) subtypes to monitor GBM cell plasticity in real-time across multiple patient-derived cell lines. This approach revealed cells that concurrently expressed both proneural and mesenchymal markers. To investigate this unique hybrid population, we implemented a comprehensive methodological approach encompassing bulk and single-cell RNA sequencing, single-cell ChIP sequencing, nuclear proteomics, high-resolution imaging, orthotopic mouse models, clinical dataset analysis, and pharmacological and genetic techniques. This multifaceted strategy allowed us to gain functional and molecular insights into this distinct cellular population.


RESULTS
We showed that these hybrid cells are increased by conventional therapies, and are resistant to these therapies. At the molecular level, hybrid cells display significant alterations in chromatin structure and nuclear protein composition, elevated transcriptional activity, Myc activation, and improved transport between the nucleus and cytoplasm. Genetic and pharmaceutical inhibition of the nuclear import/export shuttling machinery, increased in hybrid cells, effectively suppressed adaptive GBM cell plasticity and hybrid identity, thereby enhancing the sensitivity of GBM cells to therapies.


CONCLUSION
Our results indicate that GBM hybrid cells play a crucial role in chemoradiation resistance. The nuclear transport machinery presents a potential therapeutic target for hybrid cells, offering a way to counteract the typical resistance to treatment observed in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0e616d882e7bd8159604ae82c04f4049ee875a" target='_blank'>
              Proneural-Mesenchymal hybrid glioblastoma cells are resistant to therapy and dependent on nuclear import.
              </a>
            </td>
          <td>
            G. Bourmeau, O. Anezo, Jeremy Raymond, A. Ballestín, C. Pichol-Thievend, J. Reveilles, Adrien Thomas, Lin Wang, Melanie Miranda, Eve Moutaux, S. Liva, V. Ribecco, L. Besse, F. Dingli, D. Loew, Céline Vallot, G. Gargiulo, V. Ravi, Kevin Joseph, G. Seano
          </td>
          <td>2025-07-08</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) are the predominant cell type in the stroma of many solid organ malignancies, including non-small cell lung cancer (NSCLC). They exhibit considerable phenotypic and functional heterogeneity and are widely used in functional assays and co-culture models. CAF research frequently involves the in vitro expansion and maintenance of CAFs to facilitate functional assays and co-culture studies. However, less is known about how in vitro culture temporally affects CAF phenotype. Methods We characterised the phenotype of CAFs from NSCLC patients compared to non-cancerous lung fibroblasts using conventional in vitro conditions by tracking changes in CAF subset marker expression levels by flow cytometry. Additional transcriptomic and functional analyses were performed to determine differences between CAFs and non-cancerous fibroblasts. Results We demonstrate that CAFs from NSCLC undergo phenotypic drift in culture, and that there is a convergence to a subset phenotype predominantly upregulated in non-cancerous lung. Additionally, we demonstrate the phenotype, transcriptome and function of fibroblasts converge between CAFs and fibroblasts from non-cancerous lung by the third culture passage, suggesting that in vitro conditions promote this phenotype. Conclusion We highlight the need to understand and monitor the culture phenotype during functional studies with CAFs, as the heterogeneity found in the tumour microenvironment is rapidly lost in cultured cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5599d9b2c874b1616cccf8809445b7ed868d0a94" target='_blank'>
              Patient derived cancer-associated fibroblasts from non-small cell lung cancer undergo phenotypic drift in culture
              </a>
            </td>
          <td>
            Layla Mathieson, Phoebe Jones, Lilian Koppensteiner, Liam Neilson, David A. Dorward, Richard A O'Connor, Ahsan R. Akram
          </td>
          <td>2025-07-10</td>
          <td>BJC Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Spatial transcriptomics (ST) is transforming our understanding of tumor heterogeneity by enabling high-resolution, location-specific mapping of gene expression across tumors and their microenvironment. However, the translational potential of spatial transcriptomics is still limited by its high cost, hindering the assembly of large patient cohorts needed for robust biomarker discovery. Here we present Path2Space, a deep learning approach that predicts spatial gene expression directly from histopathology slides. Trained on substantial breast cancer ST data, it robustly predicts the spatial expression of over 4,300 genes in independent validations, markedly outperforming existing ST predictors. Path2Space additionally accurately infers cell-type abundances in the tumor microenvironment (TME) based on the inferred ST data. Applied to more than a thousand breast tumor histopathology slides from the TCGA, Path2Space characterizes their TME on an unprecedented scale and identifies three new spatially-grounded breast cancer subgroups with distinct survival rates. Path2Space-inferred TME landscapes enable more accurate predictions of patients’ response to chemotherapy and trastuzumab directly from H&E slides than those obtained by existing established sequencing-based biomarkers. Path2Space thus offers a transformative, fast and cost-effective approach to robustly delineate the TME directly from their histopathology slides, facilitating the development of spatially-grounded biomarkers to advance precision oncology.



 Emma M. Campagnolo, Eldad D. Shulman, Roshan Lodha, Amos Stemmer, Peng Jiang, Carlos Caldas, Simon Knott, Danh-Tai Hoang, Kenneth Aldape, Eytan Ruppin. Path2Space: An AI approach for cancer biomarker discovery via histopathology inferred spatial transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B047.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d0f988faed3c86233b6f535df129f10c4acb357" target='_blank'>
              Abstract B047: Path2Space: An AI approach for cancer biomarker discovery via histopathology inferred spatial transcriptomics
              </a>
            </td>
          <td>
            Emma M. Campagnolo, E. Shulman, Roshan Lodha, Amos Stemmer, Peng Jiang, Carlos Caldas, Simon Knott, Danh-Tai Hoang, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Lymphatics provide a route for breast cancer cells to metastasize. Lymphatic endothelial cells (LECs), which form the structure of lymphatic vessels, play a key role in this process. Although LECs are pivotal in cancer progression, studies often rely on commercially available cell lines that may not accurately reflect the tumor microenvironment. Therefore, there is a pressing need to directly study patient-derived LECs to better understand their role in breast cancer. Methods This study developed a method to isolate and characterize LECs directly from human breast-to-axilla adipose tissue. We used magnetic cell separation to remove CD45 + leukocytes and fluorescence-activated cell sorting to isolate cells expressing CD31 and podoplanin. Isolated cells were cultured under conditions promoting endothelial cell growth and were characterized through various assays assessing proliferation, tube formation, and gene expression patterns. Results The sorted CD31 + /PDPN + /CD45 − cell populations exhibited marked increases in proliferation upon VEGF-C stimulation and formed tubule structures on BME-coated dishes, confirming their LEC properties. Notably, isolated LECs showed distinct gene expression patterns depending on the presence of lymph node metastasis and lymphatic invasion. Conclusions The ability to isolate and characterize patient-derived LECs from mammary adipose tissue offers new insights into the cellular mechanisms underlying breast cancer metastasis. Significant gene expression variability related to disease state highlights the potential of these cells as biomarkers and therapeutic targets. This study emphasizes the importance of using patient-derived cells to accurately assess the tumor microenvironment, potentially leading to more personalized therapeutic approaches. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02067-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79707a5035d75af2328bcdf0a3016ac796820a28" target='_blank'>
              Axillary adipose tissue–derived lymphatic endothelial cells exhibit distinct transcriptomic signatures reflecting lymphatic invasion status in breast cancer
              </a>
            </td>
          <td>
            A. Iesato, Jun Suzuka, K. Otsuji, Tomo Osako, Nami Yamashita, Yuka Inoue, Tetsuyo Maeda, Natsue Uehiro, K. Yoshida, Yoko Takahashi, Kohei Kumegawa, S. Saeki, Liying Yang, Ai Tsuchiya, Kana Sakiyama, Miwa Tanaka, Takehiko Sakai, Shinji Ohno, Tetsuo Noda, T. Ueno, R. Maruyama
          </td>
          <td>2025-06-17</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94d39a50858ad986c811d60a70891ccb3b663404" target='_blank'>
              Single-cell spatial analysis with Xenium reveals anti-tumour responses of CXCL13 + T and CXCL9+ cells after radiotherapy combined with anti-PD-L1 therapy
              </a>
            </td>
          <td>
            Shunsuke A. Sakai, Hidekazu Oyoshi, M. Nakamura, T. Taki, Kotaro Nomura, H. Hojo, H. Hirata, A. Motegi, Yuka Nakamura, Junko Zenkoh, K. Aokage, Akira Hamada, M. Kojima, Takeshi Kuwata, K. Tsuchihara, T. Akimoto, J. Soh, Tetsuya Mitsudomi, Masahiro Tsuboi, Genichiro Ishii, Yutaka Suzuki, Ayako Suzuki, Riu Yamashita, S. Kageyama
          </td>
          <td>2025-07-16</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background To discover immune response-associated gene expressions related to brain metastases of skin melanomas, we analyzed a group of fifteen primary skin melanomas consisting of five tumors from patients who did not develop systemic metastases (NM); ten from patients with systemic metastasis of which five with (BM), and five without brain involvement (OOM). Regions of interest (ROIs) harboring tumor cells only; immune cells only; or immune cell infiltration in tumor cell regions, were separately analyzed. Methods we profiled the tumor immune microenvironment (TIME) of primary skin melanomas by using a panel of 77 immune response-related oligo-nucleotide targets in the GeoMx Digital Spatial profiler (Nanostring Technologies). Results Melanomas from patients who developed brain metastases contained significantly higher levels of (CD45+) immune cells and overexpressed fibronectin and CTLA-4 in all ROIs compared to the melanomas that had metastasized to other sites than the brain (p<0.05). Distant metastases were accompanied by lower expression of CD137 (4-1BB). Downregulation of VISTA, IDO1 and ICOS was associated with melanomas that gave rise to distant metastases including brain. Conclusion Lower expression of CD137 (4-1BB) is linked with the formation of distant metastases and the expressions of fibronectin and CTLA-4 herald the formation of brain metastasis in particular.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f922ad0f88efcd474799b2ee7597e58e29a1a3" target='_blank'>
              Overexpression of CTLA-4 and fibronectin, and lower expression of CD137 (4-1BB) is associated with brain metastasis of primary skin melanomas. An analysis of local immune response by digital spatial profiling
              </a>
            </td>
          <td>
            S. Najjary, Nur al Amery, Mengqi Huang, Disha Vadgama, Alexander C. J. van Akkooi, H. J. Gilhuis, Zineb Belcaid, Dana A. M. Mustafa, J. Kros, Pedro Berraondo, Fer. Aranda Vega
          </td>
          <td>2025-06-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Background The high heterogeneity and multi-directional poor differentiation of tumor cells in mesothelioma (MESO) contributes to tumor growth and malignant biological behaviors. However, a molecular classification based on differentiated states of tumor cells remains void. Methods We performed dimensionality reduction analysis on the single-cell RNA sequencing profiles available from the GEO database, to visualize the cell types in MESO. Multi-omics analysis was done to supplement the plausibility of classification. We also constructed regulatory networks to detect the function of important tumor cell differential genes (TCDGs) in the MESO. Results Following twice dimensionality reduction analysis and clustering, eight malignant cell subtypes in the MESO were visualized. According to the expression of TCDGs, MESO was classified into three subtypes (Malignant differentiation-related MESO, Benign differentiation-related MESO, and Neutral differentiation-related MESO) with prognostic differences. The prediction model was built by 12 key TCDGs (ALDH2, HP, CASP1, RTP4, PDZK1IP1, TOP2A, LOXL2, CKS2, SPARC, TLCD3A, C6orf99, and SERPINH1) and validated with high accuracy. In the regulatory networks of MESO subtypes, RTP4, CASP1, MYO1B, SLC7A5, LOXL2, and GHR were labeled as key genes. A total of 14 potential inhibitors were predicted. Clinical specimens validated the reliability of the clinical subtyping of MESO patients. Conclusion The novel molecular classification system and the prognostic prediction model might benefit the management of MESO patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03816-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20d196862c5a6506ef6fe12760f39c1ad9e4ade5" target='_blank'>
              A novel mesothelioma molecular classification based on malignant cell differentiation
              </a>
            </td>
          <td>
            Jun Liu, Yifan Liu, Yuwei Lu, Wei Zhang, Jiale Yan, Bingnan Lu, Yuntao Yao, Shuyuan Xian, Donghao Lyu, Jiaying Shi, Yuanan Li, Xinru Wu, Chenguang Bai, Jie Zhang, Yuanzhi Zhang
          </td>
          <td>2025-06-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND
Pilocytic astrocytomas (PAs) are the most common pediatric central nervous system (CNS) tumors, which present with limited genetic but significant clinical heterogeneity. Current treatment strategies are partly effective, but tumors often progress, and patients experience long-term side effects, highlighting the need for additional novel therapeutic approaches. A promising alternative approach could be targeting the tumor immune microenvironment (TIME), however, a comprehensive overview of the TIME of PAs across different anatomical tumor locations is currently lacking. The aim of this study was to comprehensively characterize the cellular and transcriptional landscape of pediatric PAs.


METHODS
We employed a multi-modal, integrative approach using single-nucleus and bulk RNA-sequencing alongside high-dimensional immunofluorescence imaging and flow cytometry to elucidate the transcriptional landscape and cellular composition of pediatric PAs across tumor locations.


RESULTS
Our analysis uncovered neoplastic cell transcriptional heterogeneity reflective of aberrant glial differentiation. Moreover, we provided a detailed characterization of the TIME, revealing pro-inflammatory brain-resident microglia and abundant activated monocyte-derived macrophages and T cells. Deconvolution of bulk RNA-sequencing data revealed variation in tumor and TIME composition across anatomical tumor locations. Suprasellar tumors additionally exhibited increased expression of immune-related genes compared to tumors arising in other anatomical locations.


CONCLUSIONS
Together, our multi-modal in-depth characterization provides detailed insights into the transcriptional and cellular heterogeneity of pediatric PAs across distinct anatomical tumor locations, which could aid in the development of novel (immuno)therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a09d2a9d5ee2e4cda19d7cf627d28deb93a6522c" target='_blank'>
              Multi-modal analysis of pediatric pilocytic astrocytomas reveals tumor location-associated cellular and transcriptional heterogeneity.
              </a>
            </td>
          <td>
            J. Lammers, Jacob S Rozowsky, F. Calkoen, Thijs J M van den Broek, J. Meesters-Ensing, Tiago Carvalheiro, Tiago P J Rijsdijk, Robyn D Gartrell, D. V. van Vuurden, H. Stunnenberg, E. Hoving, P. Wesseling, J. van der Lugt, M. Kranendonk, Lennart A. Kester
          </td>
          <td>2025-07-28</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c42ccef135de9d53a5d5b1a6abe8bae988944a3" target='_blank'>
              Histo-LCM-Hi-C reveals the 3D chromatin conformation of spatially localized rare cells in tissues at high resolution
              </a>
            </td>
          <td>
            Yixin Liu, Min Chen, Xin Liu, Zeqian Xu, Xinhui Li, Yan Guo, D. Czajkowsky, Zhifeng Shao
          </td>
          <td>2025-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Primary tumors constantly shed cancer cells into the circulation, yet only a fraction of these cells manages to give rise to metastatic tumors. Successful metastatic seeding and growth appears to depend on metabolic changes within cancer cells. Here, using a metabolism-focused CRISPR screen in a spontaneous metastasis model, we found that expression of the enzyme γ-butyrobetaine hydroxylase 1 (BBOX1) in a subpopulation of tumor cells in various carcinomas enables immune evasion. The metabolite carnitine produced by BBOX1 inhibited the small GTPase RhoA in natural killer (NK) cells, preventing immunological synapse formation and thereby protecting metastatic cells. Loss of BBOX1 in tumor cells promoted their destruction by NK cells in vitro and improved the efficacy of NK cell adoptive transfer therapy in vivo. These findings illustrate how BBOX1-positive tumor cells hijack carnitine production to evade immune surveillance during metastasis and propose BBOX1 as a potential metabolic checkpoint for anti-metastatic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83c565b424262f8f0a380c1e10b4fb97fa55b5b" target='_blank'>
              BBOX1 is a Metabolic Checkpoint that Mediates Metastatic Cancer Cell Evasion from Immunosurveillance by Natural Killer Cells.
              </a>
            </td>
          <td>
            Chuanjie Zhang, Hongchao He, Yi Gao, Jiawei Ding, Hai Huang, Yixun Liu, Hanqing Liu, Jun Xiao, Dan-Feng Xu, Zunguo Du, Zhenbo Zhang, Lechi Ye
          </td>
          <td>2025-07-31</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a high-mortality cancer characterized by its aggressive, treatment-resistant phenotype and a complex tumour microenvironment (TME) featuring significant hypoxia. Bulk transcriptomic analysis has identified the “classical” and “basal-like” transcriptional subtypes which have prognostic value; however, it is not well-established how microenvironmental heterogeneity contributes to these transcriptional signatures. Here, we exploited the TRACER platform to perform single cell transcriptome analysis of organoids at specific spatial locations to explore the effect of oxygen and other cell-generated microenvironmental gradients on organoid heterogeneity. We found that the microenvironmental gradients present in TRACER significantly impact the distribution of organoid transcriptional phenotypes and the enrichment of gene sets linked to cancer progression and treatment resistance. More significantly, we found that microenvironmental gradients, predominantly oxygen, drive changes in the expression of classical and basal-like transcriptional subtype gene signatures. This work suggests that hypoxia contributes to determining transcriptional subtypes in PDAC tumour cells independent of additional cells in the TME and broadly highlights the importance of considering microenvironmental gradients such as oxygen in organoid-based studies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-98344-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d7b936c3248bb437b55ed176049e4ec9a8d43e" target='_blank'>
              Analysis of an engineered organoid model of pancreatic cancer identifies hypoxia as a contributing factor in determining transcriptional subtypes
              </a>
            </td>
          <td>
            Natalie Landon-Brace, Simon Latour, Brendan T. Innes, Michelle Nurse, Jose L. Cadavid, I. Co, Cassidy M. Tan, Ferris Nowlan, Sibil Drissler, F. Notta, H. Jackson, Gary D Bader, A. McGuigan
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Objectives This study aims to elucidate the complex molecular and cellular landscape of pancreatic ductal adenocarcinoma (PDAC) by identifying key regulatory non-coding RNAs (ncRNAs), hub protein-coding genes, and Intercellular communication pathways that may serve as prognostic biomarkers and therapeutic targets. Background Pancreatic cancer remains one of the deadliest malignancies worldwide, characterized by late diagnosis, limited treatment response, and poor prognosis. Among its histological subtypes, PDAC accounts for over 80% of cases and is defined by a highly fibrotic and immunosuppressive tumor microenvironment (TME). Methods We performed a comprehensive bioinformatics analysis integrating multiple transcriptomic datasets from the NCBI Gene Expression Omnibus (GEO), including mRNA, miRNA, lncRNA, and circRNA profiles from PDAC and adjacent normal tissues. Differential expression analysis was conducted using GEO2R, followed by functional enrichment via DAVID. Hub genes were identified from protein–protein interaction (PPI) networks constructed using STRING and validated using GEPIA2. A competing endogenous RNA (ceRNA) network was developed to investigate regulatory ncRNA–mRNA axes. To refine these findings, single-cell RNA-seq (scRNA-seq) data were analyzed to resolve the cellular origin of hub genes and ncRNAs, and CellChat was employed to model intercellular communication within the TME. Results We identified several dysregulated genes and ncRNAs implicated in key oncogenic pathways, including ECM remodeling, inflammation, and immune evasion. The ceRNA network highlighted functional interactions between circRNAs, lncRNAs, and miRNAs regulating key hub genes. Single-cell analysis revealed cell-type-specific expression of hub genes—e.g., FN1 and COL11A1 in fibroblasts, CXCL8 in macrophages, and ITGA3 in ductal cells—and uncovered a macrophage–endothelial CXCL8–ACKR1 signaling axis potentially driving tumor-associated angiogenesis. Moreover, correlations with immune cell infiltration and drug sensitivity further underscored the translational relevance of the identified molecular targets. Conclusion Our analysis combining bulk and single-cell transcriptomics provides a multi-scale view of PDAC pathogenesis. The findings highlight the interplay between ncRNAs, hub genes, and cellular crosstalk in shaping the tumor ecosystem and suggest novel targets for precision therapeutic intervention and biomarker development. Supplementary Information The online version contains supplementary material available at 10.1007/s10238-025-01815-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/588c78ae7417c5606a00168a20889c995ded56f7" target='_blank'>
              Combined bulk and single-cell transcriptomic analysis reveals cell-type-specific inflammatory crosstalk in pancreatic cancer
              </a>
            </td>
          <td>
            Ahmad Golestanifar, Mana Zakeri, Sahar Gohari-Lasaki, Hengameh Khedri, Mohammadreza Saberiyan
          </td>
          <td>2025-07-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer (BC) remains a leading cause of cancer-related mortality in women, with complex mechanisms driving its initiation, progression, and resistance to therapy. In recent years, the tumor microenvironment (TME) has gained attention for its critical role in shaping tumor behavior, where small extracellular vesicles (small EVs) have emerged as key mediators of intercellular communication. These vesicles carry a diverse cargo of proteins, lipids, DNA, and various non-coding RNAs—such as miR-21, miR-155, and miR-1246—mirroring the molecular status of their originating cells. This review highlights the roles of small EVs in immune modulation, stromal remodelling, and metastatic niche formation, emphasizing their contribution to therapy resistance and immune evasion. We discuss recent updates on EV biogenesis, characterisation and isolation techniques, such as ultracentrifugation, immunoaffinity and microfluidic systems. We also critically evaluate their potential for clinical application and how well they conform to the MISEV2023 guidelines. Furthermore, we examine small EVs as diagnostic tools in liquid biopsies and compare them with conventional methods such as mammography and tissue biopsies. We also discuss organotropism mediated by small EV cargo (e.g., integrins α6β4, αvβ5) and the diagnostic potential of protein and lipid signatures (e.g., PD-L1, CD63, and exosomal lipidomics). Therapeutically, we explore engineered small EVs for drug delivery, gene modulation, and immune activation, addressing challenges of targeting efficiency, in vivo stability, immunogenicity, and clinical scalability. The review discusses ongoing clinical trials involving small EVs in BC and highlights key translational gaps between preclinical advances and clinical implementation. Finally, we explores how integrating artificial intelligence, single-cell transcriptomics, and multi-omics approaches can help overcome major challenges such as small EV heterogeneity and tracking limitations. Crucially, this integration enables a more tailored understanding of each patient’s tumor biology, reducing therapeutic failures by guiding more personalized and effective treatment strategies. Overall, small EVs represent a transformative tool in precision oncology, contingent on resolving key challenges in their clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbe25e9e7298ee7befa5cf726da413903d9b701b" target='_blank'>
              Small extracellular vesicles: messengers at the service of breast cancer agenda in the primary and distant microenvironments
              </a>
            </td>
          <td>
            Angela Galardi, Valentina Fogazzi, Claudia Tottone, M. Giussani, S. Pupa, G. Cosentino, M. Iorio
          </td>
          <td>2025-07-21</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7be43f99c39d72183ee247e245dd3934afb3150" target='_blank'>
              Decoding meningioma prognosis with multi-omics: macrophage diversity, immune-CNV interplay, and novel SPP1-targeted strategies.
              </a>
            </td>
          <td>
            Hailang Fan, Xiaojie Li, Yaqian Zhao, Lairong Song, Lanlan Sun, Zhen Wu, Junting Zhang, Haikun Zhang, Feifan Zhang, Yingdi Huang, Dake Zhang, Liang Wang
          </td>
          <td>2025-06-16</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Oncogene amplification is a key driver of cancer pathogenesis and is often mediated by extrachromosomal DNA (ecDNA). EcDNA amplifications are associated with increased pathogenicity of cancer and poorer outcomes for patients. EcDNA can be detected accurately using fluorescence in situ hybridization (FISH) when cells are arrested in metaphase. However, the majority of cancer cells are non-mitotic and must be analyzed in interphase, where it is difficult to discern extrachromosomal amplifications from chromosomal amplifications. Thus, there is a need for methods that accurately predict oncogene amplification status from interphase cells. Here, we present interSeg, a deep learning-based tool to cytogenetically determine the amplification status as EC-amp, HSR-amp, or not amplified from interphase FISH images. We trained and validated interSeg on 652 images (40,446 nuclei). Tests on 215 cultured cell and tissue model images (9,733 nuclei) showed 89% and 97% accuracy at the nuclear and sample levels, respectively. The neuroblastoma patient tissue hold-out set (67 samples and 1,937 nuclei) also revealed 97% accuracy at the sample level in detecting the presence of focal amplification. In experimentally and computationally mixed images, interSeg accurately predicted the level of heterogeneity. The results showcase interSeg as an important method for analyzing oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a539ca1a35b8049ad46a136d79392bb325111564" target='_blank'>
              Accurate Prediction of ecDNA in Interphase Cancer Cells using Deep Neural Networks
              </a>
            </td>
          <td>
            Utkrisht Rajkumar, Gino Prasad, Ellis J Curtis, I. Wong, Xiaowei Yan, Shu Zhang, Lotte Brückner, Kristen Turner, Julie Wiese, Justin Wahl, Sihan Wu, J. Theißen, Matthias Fischer, Howard Y. Chang, A. Henssen, P. Mischel, V. Bafna
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8261c73bf75d788913526d1e389ee8b4f740f28c" target='_blank'>
              Inhibition of ACVR1 in Cancer-Associated Fibroblasts Suppresses Colorectal Cancer Cell Growth.
              </a>
            </td>
          <td>
            Shinya Kato, N. Miyoshi, S. Fujino, Masafumi Horie, Shinichi Yachida, Mitsunobu Takeda, Y. Sekido, Tsuyoshi Hata, Atsushi Hamabe, T. Ogino, Mamoru Uemura, Hirofumi Yamamoto, Masayoshi Yasui, M. Ohue, Y. Doki, H. Eguchi
          </td>
          <td>2025-07-31</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="ABSTRACT Introduction Lung adenocarcinoma, the most common subtype of non‐small cell lung cancer, faces challenges such as drug resistance and tumor heterogeneity. N4‐acetylcytidine (ac4C) is an important RNA modification involved in cancer progression, but its role in lung adenocarcinoma remains unclear. Methods This study analyzed transcriptomic and single‐cell RNA sequencing data from public databases to investigate the expression and clinical significance of ac4C‐related genes in lung adenocarcinoma. Ten machine learning algorithms were applied to develop and validate an ac4C‐related gene signature (ARGSig) for prognosis prediction across multiple independent cohorts. Results Cells with high ac4C activity showed increased intercellular communication and activation of tumor‐associated pathways. The ARGSig model effectively stratified patients by survival outcomes and predicted sensitivity to immune checkpoint inhibitors and chemotherapy agents. Conclusion ac4C modification and its related genes play a critical role in lung adenocarcinoma development. The ARGSig model provides a promising molecular tool for prognosis evaluation and personalized treatment guidance in lung adenocarcinoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b6722e2c14c1989978a512d01d29289da459bd" target='_blank'>
              Integrating Single‐Cell Transcriptomics and Machine Learning to Define an ac4C Gene Signature in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Yuan Wang, Wei Su, Guangyao Zhou, Yijie Wang, Chunnuan Wu, Pengpeng Zhang, Lianmin Zhang
          </td>
          <td>2025-08-01</td>
          <td>Thoracic Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction Skin cutaneous melanoma (SKCM) is a highly aggressive form of cancer with poor prognosis, characterized by significant molecular and immune heterogeneity. The activation of KRAS signaling pathways is implicated in melanoma progression, yet its role in shaping the tumor microenvironment, particularly in macrophage infiltration, remains poorly understood. Methods A comprehensive multi-platform approach was employed, analyzing gene expression data from the Gene Expression Omnibus (GEO) and The Cancer Genome Atlas (TCGA) databases. Gene set enrichment analysis (GSEA) was utilized to characterize the molecular pathways associated with KRAS signaling. Single-cell RNA sequencing (scRNA-seq) was leveraged to investigate the cellular heterogeneity within the SKCM tumor microenvironment, and macrophage populations were categorized using the Monocle2 algorithm. A KRAS-Macrophage Prognostic Associated Gene (KMPAG) signature was developed by integrating these findings, followed by validation using a least absolute shrinkage and selection operator (LASSO) regression model. The prognostic value of the KMPAG signature was assessed through its correlation with clinical outcomes, immune cell infiltration patterns, response to therapy, drug sensitivity, and miRNA-gene regulatory interactions. Cell-cell communication within the SKCM microenvironment was explored using the “CellChat” tool. Experimental validation of gene expression was performed via immunohistochemistry (IHC) and functional assays in gene-modified melanoma cell lines. Results Twenty-two genes involved in KRAS signaling were identified as critical for patient survival. Single-cell analysis revealed nine distinct cell populations within the SKCM microenvironment, leading to the construction of the KMPAG risk model, which incorporated three key genes—CLEC4A, CXCL10, and LAT2. This signature effectively reclassified macrophage subsets, offering improved diagnostic and prognostic capabilities. Furthermore, the KMPAG signature correlated with a range of clinical parameters, including immune infiltration levels, tumor stage, and therapy response. The model also provided insights into the immune landscape of SKCM, facilitating the prediction of responses to immunotherapy. Functional assays demonstrated that downregulation of CLEC4A significantly promoted melanoma cell proliferation, migration, and invasion. Conclusion This study highlights the importance of KRAS signaling and macrophage infiltration in melanoma prognosis. The KMPAG gene signature presents a novel prognostic tool, offering insights into personalized treatment strategies and predictive biomarkers for immunotherapy in SKCM. Further exploration of CLEC4A’s role in melanoma progression may provide new therapeutic avenues for targeted intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c090f6061dd7c65360b39576ed1484f2fed629c" target='_blank'>
              Identification and validation of a KRAS-macrophage-associated gene signature as prognostic biomarkers and potential therapeutic targets in melanoma
              </a>
            </td>
          <td>
            Beichen Cai, Qian Lin, Ruonan Ke, Jiaqi Yu, Lu Chen, Xuejun Ni, Hekun Liu, X. Shan, Biao Wang
          </td>
          <td>2025-06-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Invasive lobular carcinoma (ILC) is the second most common histological subtype of breast cancer after invasive ductal carcinoma of no special type (NST), accounting for 10–15% of diagnoses. Despite the myriad molecular, histological and clinical differences between ILC and NST tumors, patients are treated in the same way, and although prognosis initially is good, ILC patients have poorer long-term outcomes. Understanding the differences between these two subtypes and identifying ILC-enriched therapeutic targets is necessary to improve patient care. Methods Human and mouse cancer-associated fibroblasts (CAFs), ILC cell lines and patient-derived organoids were used for in vitro and in vivo studies, including western blotting, migration, organotypic invasion assays and dissemination in zebrafish embryos. RNASeq was used to identify CAF and interleukin-6 (IL-6)-derived gene signatures. Bioinformatic analysis of public databases and immunohistochemical of human tumor microarrays was carried out. Results We identified IL-6 as a paracrine CAF-derived factor that activates Signal-Transducer-and-Activator-of-Transcription-3 (STAT3) in human and mouse ILC models. Analysis of human breast tumors showed that the IL-6/JAK/STAT3 pathway is enriched in ER + ILC compared to ER + NST. A 42-gene CAF dependent IL-6 gene signature and 64-gene consensus IL-6 gene signature were generated and were significantly enriched in ER + ILC, with many of the genes overexpressed in ILC tumors. IL-6 treatment suppressed downstream estrogen signaling and also led to the acquisition of a more mesenchymal-like phenotype associated with increased migration and invasion. Finally, IL-6 treatment significantly increased ILC cell dissemination following injection into zebrafish embryos. Conclusions CAF-derived IL-6 drives paracrine activation of the IL6/JAK/STAT3 signaling pathway which is enriched in ILC. This leads to the acquisition of pro-tumorigenic phenotypes, highlighting the pathway as a potential therapeutic target in ILC. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02074-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ff0735a08b68cfe9b8184d5f4b41c0bb6234a9" target='_blank'>
              Cancer-associated fibroblast driven paracrine IL-6/STAT3 signaling promotes migration and dissemination in invasive lobular carcinoma
              </a>
            </td>
          <td>
            Esme Bullock, Aleksandra Rozyczko, Sana Shabbir, Ifigenia Tsoupi, Adelaide I J Young, J. Travnickova, Laura Gómez-Cuadrado, Zeanap Mabruk, Giovana Carrasco, Elizabeth Morrow, K. Pennel, Pim Kloosterman, J. Houthuijzen, Jos Jonkers, L. Avalle, Valeria Poli, R. Iggo, Xue Xiao, Jingjing Guo, Xuan Zhu, Elizabeth Mallon, Joanne Edwards, E. E. Patton, Valerie G. Brunton
          </td>
          <td>2025-07-01</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Many studies have shown that pancreatic cancer is one of the cancers with extremely high mortality. The poor prognosis and lack of early diagnostic methods remain major challenges in the treatment of this cancer. In this article, in order to better detect the occurrence of pancreatic cancer, tools such as monocle3, singleR, harmony in R, and scanpy in Python were used to analyze the cells and genes of pancreatic cancer tissues in mice. By analyzing the data, the following results were obtained: T-cells in healthy PBMCs exhibited broader spatial dispersion than in PDAC tissues, suggesting tumor-driven immune surveillance impairment, while PDAC-associated macrophages displayed expanded distribution linked to pro-tumorigenic functions such as COL1A1-mediated ECM remodeling; Pseudotemporal trajectory analysis revealed myeloid progenitor bifurcation into monocytes/macrophages, with PDAC macrophages showing epigenetically silenced cytotoxic pathways such as suppressed GZMA/NKG7 and enhanced ribosomal biogenesis; Tissue-specific markers such as LCN2 in healthy and CTRB1/AMY2A in PDAC) and spatial co-localization of macrophages/tumor cells highlighted NOP53 as a dual-function hubinhibiting PI3K-AKT while activating p53and SPP1 as a paradoxical regulator of metastasis and antitumor immunity; Differential expression and GO enrichment analyses identified ribosomal biogenesis and cytoplasmic translation as PDAC-enriched pathways, contrasting with suppressed stress responses. Our spatial transcriptomic profiling further resolved elevated NOP53, CFB, and SPP1 expression gradients in PDAC tissues, proposing these as diagnostic biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab94aa8cc2272f4a29c06caf0dcfe1338e28a0a0" target='_blank'>
              Spatiotemporal Transcriptomic Dissection of Tumor-Associated Macrophage Heterogeneity and Dual-Function Molecular Nodes in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Mengfei Li
          </td>
          <td>2025-06-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In the ever-evolving landscape of cancer therapy, while cancer treatments such as chemotherapy, radiotherapy, and targeted therapy aim to eradicate malignant cells, they also inadvertently trigger cellular senescence in both cancerous and microenvironmental tissues. Therapy-induced senescence (TIS) can act as a barrier against tumor growth by halting cell proliferation in the short term, but the long-term persistence of therapy-induced senescent (TISnt) cells may pose a significant challenge in cancer management. Their distinct characteristics, like senescence-associated secretory phenotype (SASP), metabolic dysregulation, and immune evasion, make them exhibit remarkable heterogeneity to orchestrate the tumor microenvironment (TME), resulting in therapy resistance. However, how these TISnt cells functioning differently in cancer progression, and the intricate mechanisms by which they remodel the senescence-associated immunosuppressive microenvironment present challenges for improving anticancer therapy. Therefore, this review summarizes the heterogeneous TISnt cell phenotypes contributing to an accumulated senescent state, outlines their multidimensional interactions in the senescent microenvironment, and discusses current senescence-targeting strategies. Building on the current understanding of TIS, we propose potential avenues for improving TIS-targeting methodologies in the context of head and neck cancer, a representative heterogeneous malignancy, which can substantially enhance the efficacy of the “one-two punch” sequential treatment approach for head and neck cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/378a355476ab29ee4164f4bc25cdfced24706c11" target='_blank'>
              Persistent accumulation of therapy-induced senescent cells: an obstacle to long-term cancer treatment efficacy
              </a>
            </td>
          <td>
            Jingjing Luo, Tongxu Sun, Zhenghui Liu, Yangfan Liu, Junjiang Liu, Shimeng Wang, Xueke Shi, Hongmei Zhou
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Oral Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="For years, valuable clinical samples preserved in formalin-fixed paraffin-embedded tissues were underutilized. However, with advanced spatial multiomics profiling tools, crucial information has become increasingly accessible. Integrating genomic data with spatial information has unveiled crucial insights into cellular activities, enhancing our comprehension of biology. Measuring cellular gene expression while capturing spatial context – including morphology and intercellular relationships – is vital for understanding both normal and diseased biological processes. To date, this approach has illuminated the mechanisms of complex diseases, such as cancer and has facilitated the discovery of biomarkers for early disease detection and new therapeutic targets, accelerating progress in cancer immunotherapies. Cutting-edge single-cell analysis tools are rapidly emerging as the gold standard for investigating intricate biological systems and medical specimens, fueling a multi-billion-dollar industry. Single-cell spatial research, in particular, is inherently cross-disciplinary and addresses questions that remain hidden when focusing solely on the genome or transcriptome of large cell populations. Leveraging advances in single-cell spatial profiling can offer insights into improving cancer immunotherapy and other modern medical treatments. This review will delve into the diverse applications of spatial profiling technology, showcasing examples that demonstrate its ability to provide a detailed picture of the underlying molecular and cellular mechanisms within cells. As a comprehensive reference, this review empowers researchers and industry leaders to harness single-cell and spatial omics for breakthroughs in biomedicine and translational science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33ed8c77341ba16d0b5a75b9927877b08daea9ef" target='_blank'>
              In situ spatial profiling: Gaining molecular and cellular insights within content and context
              </a>
            </td>
          <td>
            Shu-Ti Lin, Ian Molyneaux, Chen Yeh
          </td>
          <td>2025-07-10</td>
          <td>Gene &amp; Protein in Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Spatial multi-slice multi-omics (SMSMO) integration has transformed our understanding of cellular niches, particularly in tumors. However, challenges like data scale and diversity, disease heterogeneity, and limited sample population size, impede the derivation of clinical insights. Here, we propose stClinic, a dynamic graph model that integrates SMSMO and phenotype data to uncover clinically relevant niches. stClinic aggregates information from evolving neighboring nodes with similar-profiles across slices, aided by a Mixture-of-Gaussians prior on latent features. Furthermore, stClinic directly links niches to clinical manifestations by characterizing each slice with attention-based geometric statistical measures, relative to the population. In cancer studies, stClinic uses survival time to assess niche malignancy, identifying aggressive niches enriched with tumor-associated macrophages, alongside favorable prognostic niches abundant in B and plasma cells. Additionally, stClinic identifies a niche abundant in SPP1+ MTRNR2L12+ myeloid cells and cancer-associated fibroblasts driving colorectal cancer cell adaptation and invasion in healthy liver tissue. These findings are supported by independent functional and clinical data. Notably, stClinic excels in label annotation through zero-shot learning and facilitates multi-omics integration by relying on other tools for latent feature initialization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/623e00baaa4b0886d492ad3269f4640463357088" target='_blank'>
              stClinic dissects clinically relevant niches by integrating spatial multi-slice multi-omics data in dynamic graphs
              </a>
            </td>
          <td>
            Chunman Zuo, Junjie Xia, Yupeng Xu, Ying Xu, Pingting Gao, Jing Zhang, Yan Wang, Luonan Chen
          </td>
          <td>2025-06-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) is a leading cause of cancer-related deaths worldwide. Despite advancements in treatment, prognosis for patients with advanced stages remains poor. Metabolic reprogramming in the tumor microenvironment, particularly abnormal glycolysis, plays a crucial role in immune evasion and treatment response. We collected nine single-cell datasets to create a single-cell atlas of CD8 + T cells from 89 NSCLC patients, revealing ten distinct states of these cells. We employed a multimodal data analysis approach, integrating bulk transcriptomics, single-cell transcriptomics, spatial transcriptomics, and proteomics. Using 117 machine learning models, we identified key genes associated with NSCLC metastasis. Notably, the StepCox[forward] + Lasso model was instrumental in pinpointing key genes that significantly impact disease prognosis. Our analysis revealed that LTB + LDHA + CD8 + T cells have a distinct metabolic and immune phenotype, characterized by enhanced glycolysis and elevated lactate production. This not only facilitates tumor cell migration and invasion but also impairs the cytotoxic function of CD8 + T cells. Furthermore, our machine learning models identified four key genes significantly associated with NSCLC metastasis: TBCD, PTPRC, LDHA, and ACTR2. Of these, high LDHA expression was strongly linked to poorer responses to immunotherapy and a higher risk of therapy resistance. LTB + LDHA + CD8 + T cells also reduced antitumor immune responses by inhibiting the secretion of effector molecules like GNLY. Additionally, elevated LDHA expression was associated with reduced CD8 + T cell infiltration, which further promotes tumor immune evasion. This study highlights the heterogeneity of CD8 + T cells in NSCLC, emphasizing the unique role of the LTB + CD8 + Tn subpopulation in metastasis. LDHA is identified as a critical key gene with a significant impact on immunotherapy outcomes, presenting a potential therapeutic target. These insights offer new biomarkers and targeted strategies for personalized immune therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e078cc67ce9052ada65f40295ba6296864dfb9f6" target='_blank'>
              Metabolic alterations driven by LDHA in CD8 + T cells promote immune evasion and therapy resistance in NSCLC
              </a>
            </td>
          <td>
            Hao Wen, Panpan Zhang, Juan Zhao, Yakui Liu, Lei Wan, Haoran Li, Jun Yi, Xinqiang Li
          </td>
          <td>2025-07-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
The effectiveness of PD-1/PD-L1 immune checkpoint blockade therapy in glioblastoma (GBM) is limited due to the tumor immunosuppressive microenvironment (TIME). Therefore, strategies of reprogramming TIME to a proinflammatory state offers a promising therapeutic approach.


METHODS
We applied bioinformatics analysis of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (stRNA-seq) to identify a significant accumulation of a cancer-associated fibroblasts (CAFs) subcluster with elevated LRRC15 expression in the nonresponders to anti-PD-1 therapy. Molecular mechanism of LRRC15 were functionally validated in vitro and in vivo.


RESULTS
These CAFs subcluster drive the infiltration of macrophages (Mφ) into the tumor microenvironment and promote their polarization toward the M2 phenotype. Deletion of Lrrc15 in CAFs significantly restrained tumor growth and prolonged survival in mouse models. Mechanistically, LRRC15 in CAFs promotes IL8 expression by activating the downstream FAK/SRC/NF-κB pathways, leading to Mφ migration and M2-like polarization. In turn, M2-like Mφs secrete TGF-β, which induces LRRC15 expression in CAFs via SMAD2-dependent transcriptional activation. Targeting CAFs subcluster with elevated LRRC15 expression in combination with anti-PD-1 treatment enhanced antitumor efficacy.


CONCLUSIONS
Our findings suggest that targeting LRRC15 may provide a novel strategy to augment anti-PD-1 therapy and overcome immunotherapy resistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1506f6c672d0a05f16addb76763bcd00e0bd9a3b" target='_blank'>
              CAF-derived LRRC15 orchestrates macrophage polarization and limits PD-1 immunotherapy efficacy in glioblastoma.
              </a>
            </td>
          <td>
            Feifei Luo, Y. Mei, Yanwen Li, Jiahao Yang, Shaoyan Xi, Endong Cao, Cong Shen, Dexiang Zhou, Peng Wang, Dong Zhou, Haiping Cai
          </td>
          <td>2025-06-26</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung cancer, predominantly non-small cell lung cancer (NSCLC), remains a principal driver of cancer-related morbidity and mortality worldwide. Despite advancements in surgery, radiotherapy, chemotherapy, and targeted treatments, outcomes remain poor in advanced NSCLC. The tumor microenvironment (TME) exerts a critical influence on therapy responses. Within the TME, immune cells such as T and B lymphocytes, dendritic cells, myeloid-derived suppressor cells, tumor-associated macrophages, neutrophils, and natural killer cells can drive both pro- and anti-tumor processes. This review integrates their classification, phenotypic plasticity, and roles in NSCLC, highlighting key preclinical and clinical evidence while discussing pathogenesis, prognostic significance, and therapeutic potential. We also summarize the current immunotherapeutic strategies for advanced NSCLC, including first- or second-line regimens with immune checkpoint inhibitors alone or combined with chemotherapy, anti-angiogenic agents, or additional checkpoint inhibitors, and future directions. By elucidating the interplay between the NSCLC immune microenvironment and emerging immunotherapies, this review emphasizes the need for novel combination regimens and robust predictive biomarkers to improve clinical outcomes and extend survival in advanced NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d594d55b67aa26998978e57ef768648090cd4cd" target='_blank'>
              Tumor-infiltrating lymphocytes in NSCLC: from immune surveillance to immunotherapy
              </a>
            </td>
          <td>
            Haiyi Xue, Yilan Fan, Yi Li, Qian Zhao, Xuelu Zhang, Pei Zhao, Zhenjun Liu
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a lethal brain cancer with no effective treatment; understanding how GBM cells respond to tumor microenvironment remains challenging as conventional cell cultures lack proper cytoarchitecture while in vivo animal models present complexity all at once. Developing a culture system to bridge the gap is thus crucial. Here, a multicellular approach is employed using human glia and vascular cells to optimize a 3D brain vascular niche model that enabled not only long-term culture of patient derived GBM cells but also recapitulation of key features of GBM heterogeneity, in particular invasion behavior and vascular association. Comparative transcriptomics of identical patient derived GBM cells in 3D and in vivo xenotransplants models revealed that glia-vascular contact induced genes concerning neural/glia development, synaptic regulation, as well as immune suppression. This gene signature displayed region specific enrichment in the leading edge and microvascular proliferation zones in human GBM and predicted poor prognosis. Gene variance analysis also uncovered histone demethylation and xylosyltransferase activity as main themes for gene adaption of GBM cells in vivo. Furthermore, the 3D model also demonstrated the capacity to provide a quiescence and a protective niche against chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58339daec87fc88b9de1cfb8daf1fd79e18b6ecc" target='_blank'>
              3D Brain Vascular Niche Model Captures Glioblastoma Infiltration, Dormancy, and Gene Signatures.
              </a>
            </td>
          <td>
            Vivian K. Lee, R. Tejero, Nathaniel Silvia, Anirudh Sattiraju, Aarthi Ramakrishnan, Li Shen, Alexandre Wojcinski, Santosh Kesari, R. Friedel, Hongyan Zou, Guohao Dai
          </td>
          <td>2025-06-19</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Lung cancer is the leading cause of cancer‐related death globally. To better understand the biology of lung cancer, mouse models have been developed using either tail vein‐injected tumour cell lines or genetically modified mice. The current gold‐standard models typically present with multiple lung foci. However, although these models are widely used, their correlation with human disease are limited, as early‐stage human lung cancer usually presents as a single lesion rather than multiple foci. Additionally, a major challenge of using multifocal lung tumour models is the difficulty in distinguishing primary lung tumours from intrathoracic metastasis and lethal levels of lung congestion before distant metastases develop. Here, we present a refined and detailed surgical method in which murine tumour cells [Lewis lung carcinoma (LLC), alveogenic lung carcinoma (CMT), or Kras/Trp53‐KP mutant cells] were injected directly into the left lung lobe of C57BL/6 mice, or, alternatively, adenoviral‐Cre or adenoviral‐FlpO was administered directly into the left lung lobe of Kras LSL‐G12D ;Trp53 fl/fl or Kras FSF‐G12D ;Trp53 frt/frt (KP) mice, respectively. This method generated unifocal primary left lung lobe tumours with traceable spread to local and distant sites. A cross‐comparison of the unifocal models described commonalties and differences between LLC, CMT, KP cells, and adenoviral‐Cre or ‐FlpO methods in terms of timings for primary lung tumour growth and traceable spread to local and distant sites, histological analysis of CD3 and CD11b immune cell infiltration, and Picrosirius Red analysis of extracellular matrix complexity. Lastly, the frequency of clinical histopathological features typical of human lung cancer were assessed across the unifocal mouse models to provide a direct comparison with human lung cancer. Overall, this study details a refined and reproducible protocol for intralobular lung injection to generate unifocal lung cancer models that resemble key features of human lung cancer. This approach can be applied to other lung cancer initiation strategies. The cross‐comparative histological analysis across the models tested here offers a valuable resource to aid researchers in selecting the most appropriate next‐generation unifocal lung cancer models for their specific research needs. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31cd240c200a61f4f9520128185c37cce77c02eb" target='_blank'>
              Development and characterisation of improved unifocal primary mouse lung cancer models with metastatic potential
              </a>
            </td>
          <td>
            Ana-Rita Pedrosa, Alejandro Castillo-Kauil, Yuliia Kravchuk, Louise Reynolds, Bruce Williams, David Moore, Cameron Lang, Srinivas Allanki, Eleni Maniati, A. Hardas, Jozafina Haj, Rebecca Drake, Julie Cleaver, Julie Foster, Jana Kim, Ester Stern, Jane Sosabowski, Gilbert O Fruhwirth, E. Sahai, Ori Wald, K. Hodivala-Dilke
          </td>
          <td>2025-06-18</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="ABSTRACT Breast cancer (BRCA) is characterised by complex cellular heterogeneity and differentiation hierarchies, which play a crucial role in bone metastasis and therapeutic resistance. However, existing classification systems remain inadequate in capturing these complexities, limiting their effectiveness in guiding treatment strategies. To address this gap, we integrated single‐cell RNA‐seq profiles, spatial transcriptomes, along with 1097 bulk RNA‐seq profiles of TCGA‐BRCA cohort to dissect the molecular landscape of BRCA. By performing UMAP analysis, we identified nine tumour clusters and three spatially distinct spot types (immune, stromal and malignant spots) and further delineated 11 differentiation states from 2493 malignant spots. Through clustering, monocle 2 pseudo‐time and prognostic analyses, we identified the prognostic BRCA cell fate‐related genes, then constructed a novel BRCA stratification system (four subtypes) with differential prognosis, biological plausibility and clinical significance. Also, least absolute shrinkage and selection operator (LASSO) regression analysis was performed for the BRCA cell fate‐related genes in constructing a prognostic model. The model has modest accuracy and accordance (AUC = 0.708), which could distinguish BRCA patients into high‐risk or low groups. With correlation analysis, regulation networks were constructed for different subtypes based on the key cell fate‐related genes, transcription factors, metastasis‐related pathways, immune components and so on, to investigate the regulatory relationships between primary BRCA and BRCA bone metastasis. Afterwards, we identified the most significant inhibitors (puromycin, MS‐275, megestrol, aesculetin) for bone metastatic BRCA, which might have potential translational significance. In all, we developed a novel molecular stratification system for BRCA based on the cell fate‐related markers of malignant cells, which offered strong translational potential for diagnosis, prognosis and personalised therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1838f6be81f1e41cf0fcf9935ae12d48d0ddb8c" target='_blank'>
              Molecular Subtypes and Risk Prediction Model Based on Malignant Cell Differentiation Trajectories in Breast Cancer
              </a>
            </td>
          <td>
            Penghui Yan, Hanlin Sun, Siqiao Wang, Runzhi Huang, Chaofeng Shi, Qihang Yang, Yibo Qiao, Haonan Wang, Deqian Kong, Jiwen Zhu, Yunqing Yang, Zongqiang Huang
          </td>
          <td>2025-08-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The mitotic inhibitor docetaxel (DTX) is often used to treat endocrine-refractory metastatic breast cancer, but initial responses are mitigated as patients eventually have disease progression. Using a cohort of ex vivo cultures of circulating tumor cells (CTCs) from patients with heavily pretreated breast cancer (n=18), we find two distinct patterns of DTX susceptibility, independent of clinical treatment history. In CTCs cultured from some patients, treatment with a single dose of DTX results in complete cell killing, associated with accumulation of non-viable polyploid (≥8N) cells arising from endomitosis. In others, a transient viable drug-tolerant persister (DTP) population emerges, ultimately enabling renewed proliferation of CTCs with preserved parental cell ploidy and DTX sensitivity. In these CTC cultures, efficient cell cycle exit generates a ≤4N drug-tolerant state dependent on CDKN1B (p27Kip1). Exposure to DTX triggers stabilization of CDKN1B through AKT-mediated phosphorylation at serine 10. Suppression of CDKN1B reduces the number of persister CTCs, increases ≥8N mitotic cells and abrogates regrowth after DTX exposure. Thus, CDKN1B-mediated suppression of endomitosis contributes to a reversible persister state following mitotic inhibitors in patient-derived treatment refractory breast cancer cells. Summary in bullets Transient DTX tolerant persister cells emerge in some patient-derived cultured CTCs. DTX-tolerant persisters restrict endoreduplication and polyploidy through CDKN1 (p27kip1). DTX exposure induces CDKN1B stabilization through AKT mediated phosphorylation at serine 10. Suppression of polyploidy underlies a drug tolerant persister state specific to mitotic inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c80ac0aa94dfd6f02a8336f8c8d7fde16ba668bb" target='_blank'>
              CDKN1B (p27/kip1) enhances drug-tolerant persister CTCs by restricting polyploidy following mitotic inhibitors
              </a>
            </td>
          <td>
            Elad Horwitz, Taronish D. Dubash, Annamaria Szabolcs, Ben S. Wittner, J. Kreuzer, Robert Morris, Aditya Bardia, Brian Chirn, Devon F. Wiley, Dante Che, Hunter C. Russel, Xcanda Ixchel Herrera Lopez, Douglas B. Fox, Ezgi Antmen, David T. Ting, Wilhelm Haas, Moshe Sade-Feldman, S. Maheswaran, Daniel A Haber
          </td>
          <td>2024-02-22</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Cyclin dependent kinases 4 and 6 inhibitors have brought great improvements in the treatment of luminal breast cancer, but resistance is a major clinical hurdle. Multiple biomarkers of resistance have been proposed, but none is currently utilized in clinical practice. By performing single-cell RNA sequencing of seven palbociclib-naïve luminal breast cancer cell lines and palbociclib-resistant derivatives, we show that established biomarkers and pathways related to CDK4/6i resistance present marked intra- and inter- cell-line heterogeneity. Transcriptional features of resistance could be already observed in naïve cells correlating with levels of sensitivity (IC50) to palbociclib. Resistant derivatives showed transcriptional clusters that significantly varied for proliferative, estrogen response signatures or MYC targets. This marked heterogeneity was validated in the FELINE trial where, compared to the sensitive ones, ribociclib-resistant tumors developed higher clonal diversity at genetic level and showed greater trascriptional variability for genes associated with resistance. A potential signature of resistance inferred from the cell-line models, positively enriched for MYC targets and negatively enriched for estrogen response markers, was probed on the FELINE trial, separating sensitive from resistant tumors and revealing higher heterogeneity in resistant versus sensitive cells. These data suggest that heterogeneity for CDK4/6 inhibitors resistant markers might facilitate the development of resistance and challenge the validation of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab65930181b447a2afea51540e3bba84721ca231" target='_blank'>
              Single-cell transcriptomics reveals biomarker heterogeneity linked to CDK4/6 Inhibitor resistance in breast cancer cell lines
              </a>
            </td>
          <td>
            I. Migliaccio, M. Bonechi, Dario Romagnoli, Giulia Boccalini, F. Galardi, C. Guarducci, A. Nardone, R. Schiff, L. Biganzoli, L. Malorni, M. Benelli
          </td>
          <td>2025-07-31</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="OBJECTIVE
Adamantinomatous craniopharyngioma (ACP) is an epithelial tumor that occurs in the sellar region of the human brain. Despite resection, relapse is frequent with poor prognosis. To facilitate the development of new therapy for ACP, the authors examined the spatial distribution, cell of origin, and potential biological functions of leukemia inhibitory factor (LIF), an important stem cell self-renewal regulator, in a series of ACP tumors.


METHODS
The transcriptional sites of LIF and LIF receptor (LIFR) were determined by single-cell sequencing and space transcriptome analysis. LIF and LIFR distribution characteristics in different histopathological regions were detected with immunohistochemistry and immunofluorescence analysis. The relationships between the regional distributions of different tissues and tumor imaging characteristics, tumor cell stemness, cell proliferation, LIF expression, and patient prognosis were analyzed.


RESULTS
The authors' analysis of 39 ACPs detected LIF overexpression that was selectively enriched in cell clusters. In addition to the discovery of the stromal cells in the interstitial region, palisade epithelium, and stellate reticulum as the source cells of LIF production, the authors also revealed that LIFR was primarily generated by the cell clusters. Examination of differentially expressed genes between LIF-high and LIF-low ACP tumors indicated that the binding of LIF to LIFR may lead to the activation of the PI3K-AKT signaling pathway. Further analysis showed enrichment of LIF expression in β-catenin-positive cell clusters expressing stem cell markers of CD44, supporting its role in stem cell self-renewal. Integrated analysis with diagnostic imaging found higher level expression of LIF in cystic tumors than in solid tumors, displaying a trend toward poorer prognosis.


CONCLUSIONS
This study confirmed for the first time that LIF in ACP mainly originates from tumor microenvironment stroma. The authors' data suggest that future efforts should also include tumor stromal cells as a novel cellular and/or molecular target when developing new anticancer therapies against ACP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90e38d490dae9950f240b893ac7de6f57947b7a0" target='_blank'>
              Use of integrated spatial transcriptomics and histopathological analysis in adamantinomatous craniopharyngiomas to identify stromal cells as a new cellular source of leukemia inhibitory factor.
              </a>
            </td>
          <td>
            Wenxin Hu, Chuan Zhao, Wenrong Zheng, Yi Lin, Ning Luo, Hongxing Liu, Xingfu Wang, Xueling Qi, Xianlong Wang, Xiao-Nan Li, Zhixiong Lin
          </td>
          <td>2025-08-01</td>
          <td>Journal of neurosurgery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Human endogenous retroviruses (HERVs) harbor accessory proteins that influence cellular processes and have been linked to a wide variety of diseases, including cancer. This study investigates locus-specific HERV expression and its association with gene dysregulation in hepatocellular carcinoma (HCC), a highly prevalent and deadly form of liver cancer worldwide. Methods We analyzed RNASeq data from 424 HCC samples from The Cancer Genome Atlas (TCGA), which comprised 371 tumor and 50 matched normal tissues from a total of 371 hepatocellular carcinoma participants. We employed Telescope to identify and quantify HERV expression across the total RNA sequencing data. Results The majority of differentially expressed HERVs exhibited reduced expression in tumor tissue (166 downregulated vs. 50 upregulated), suggesting a potential functional role of HERV expression patterns in shaping the pathophysiological landscape of HCC. Specifically, the suppression of HERV-H family members, which are known to regulate cellular differentiation, may contribute to tumor dedifferentiation, increased plasticity, and enhanced metastatic potential. This loss of differentiation control and increased adaptability may play a critical role in driving the progression of liver cancer. Discussion Our study highlights a significant association of HERV expression with HCC, highlighting the differential regulation of specific HERV families in tumor tissue. For example, HERVH and ERVLE families showed consistent downregulation in tumor samples, while HERVE and HERV9 were more commonly upregulated. These shifts may reflect underlying changes in transcriptional regulation or chromatin structure between normal and malignant tissues. Rather than indicating a singular functional role, the observed expression patterns likely reflect a multifaceted relationship between HERVs and tumor biology. Further studies will be needed to determine whether these expression differences contribute to, or result from, tumor progression and to explore their potential as biomarkers or therapeutic targets. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00367-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06da0045d446b88205ce15609079cd44c93959a4" target='_blank'>
              Locus-specific HERV expression associated with hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hui Wen, M. Pérez‐Losada, Lopa Mishra, Keith A. Crandall
          </td>
          <td>2025-07-16</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="BACKGROUND AND OBJECTIVES
Field cancerisation is the process that results in a group of cells acquiring some of the phenotypic changes of cancer prior to transformation into cancer. Clinically, an important challenge remains the ability to distinguish clonal lineages and microenvironments within cancerised fields that will remain indolent from those that will progress to malignant transformation. Spatial 'omics' can help us investigate genetic, epigenetic, transcriptomic, proteomic, and cellular microenvironments that transform normal cells into a cancerised field, and subsequently into cancer. In this review, we will discuss how spatial omics techniques have expanded our understanding of field cancerisation in prostate and bladder cancer, and the challenges associated with this research.


METHODS
We identified key articles relating to field cancerisation in bladder and prostate cancer. Special emphasis was placed on studies that used modern spatial profiling technologies and studies that were designed to investigate changes within normal tissue rather than simply using it as a control for tumour tissue.


RESULTS
Spatial omics research into field cancerisation has identified interesting early findings that have informed our understanding of: transformation of the benign epithelium and mechanisms of intra-prostatic clonal expansion for prostate cancer; clonal expansion within the normal urothelium; mutations that are unique to cancerised fields within the bladder; and how field cancerisation may prime the urothelium for cancer transformation.


CONCLUSIONS
Spatial omics profiling of field cancerisation can inform risk stratification and personalised treatment options. However, there are a number of challenges associated with the technologies that must be overcome before the potential of spatial omics can be fully realised in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/957e9305f80e5b98aef1432b61f0ded66b1ee007" target='_blank'>
              What spatial omics is teaching us about field cancerisation in prostate and bladder cancer.
              </a>
            </td>
          <td>
            Matthew H V Byrne, T. Anbarasan, Lisa Browning, Dan Woodcock
          </td>
          <td>2025-06-25</td>
          <td>BJU international</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Natural killer (NK) cells are innate lymphocytes endowed with potent cytotoxic activity. The presence of tumor-associated NK cells has been correlated with better prognosis in several solid tumors including colorectal cancer (CRC). This malignant disease is the second cause of cancer death worldwide and is in urgent need for novel approaches to improve current immunotherapies. Since CRC microenvironment can induce NK cell dysfunction and hinder cancer control, understanding tumor-associated NK cell features is mandatory to fully unlock their immunotherapeutic potential. Purpose Our study aims at elucidating the molecular and functional characteristics of tumor-associated NK cells in CRC focusing on the expression of immune checkpoints that critically regulate NK cell function. We performed an in-depth cytofluorimetric analysis of tumor-associated NK cells obtained by tissue dissociation of samples derived from 80 CRC patients comparing tumor with matched tumor-free tissue and peripheral blood, stratifying patients by tumor stage or MSI/MSS condition. Tumor tissue was also analyzed by immunohistochemistry. Results NK cells expressing immune checkpoints (i.e., KIR, NKG2A and TIM-3) were significantly enriched in tumor compared to tumor-free tissue, and an increase in PD-1+ NK cells was observed in tumors compared to peripheral blood and tumor-free tissue, indicating TME-induced modulation. Notably, tumor-associated PD-1+ NK cells characterized MSI rather than MSS CRC. In addition, tumor-associated NK cells also expressed tissue residency markers (CD103 and/or CD49a) and displayed a distinct profile also including the PD-1+ NK cell subset in MSI CRC, possibly representing NK cells recruited from circulation, retained in tumors, and reconfigured by TME signals. Importantly, tissue resident NK cells adequately expressed activating NK receptors and cytotoxic molecules. Conclusions These results suggest, together with an increased PD-L1 expression on MSI tumor cells, that the efficacy of immunotherapies in MSI CRC based on PD-1/PD-L1 blockade could also rely on a superior anti-tumor potential of PD-1+ NK cells. Conversely, MSS CRC, in which tumor-associated PD-1+ NK cells are scarce, could benefit more from immunotherapies blocking NKG2A and/or KIRs. Thus, novel approaches based on NK cell features related to CRC type, fully exploiting circulating and resident NK cell anti-tumor activity, could be key to next-generation therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d24c3ead36cebff454d826553750607699c7935" target='_blank'>
              Colorectal cancer-infiltrating NK cell landscape analysis unravels tissue-resident PD-1+ NK cells in microsatellite instability tumors
              </a>
            </td>
          <td>
            Valentina Obino, Chiara Giordano, S. Carlomagno, Chiara Setti, Marco Greppi, Matteo Bozzo, Silvia Pesce, Elisa Ferretti, Simona Candiani, Letizia Muccio, Enrico Ciferri, Tania Buttiron Webber, Agnese Solari, Fulvia Ortolani, L. Paleari, Matteo Clavarezza, Andrea Barberis, M. Filauro, N. Provinciali, M. Rutigliani, E. Marcenaro, A. DeCensi, M. Della Chiesa, S. Sivori
          </td>
          <td>2025-06-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The growth of a tumor is tightly linked to the distribution of its cells along a continuum of activation states. Here, we systematically decode the activation state architecture (ASA) in a glioblastoma (GBM) patient cohort through comparison to adult murine neural stem cells. Modelling of these data forecasts how tumor cells organize to sustain growth and identifies the rate of activation as the main predictor of growth. Accordingly, patients with a higher quiescence fraction exhibit improved outcomes. Further, DNA methylation arrays enable ASA-related patient stratification. Comparison of healthy and malignant gene expression dynamics reveals dysregulation of the Wnt-antagonist SFRP1 at the quiescence to activation transition. SFRP1 overexpression renders GBM quiescent and increases the overall survival of tumor-bearing mice. Surprisingly, it does so through reprogramming the tumor’s stem-like methylome into an astrocyte-like one. Our findings offer a framework for patient stratification with prognostic value, biomarker identification, and therapeutic avenues to halt GBM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dee763118a03533523bd5d7836ea98324ad3594" target='_blank'>
              Cross-species comparison reveals therapeutic vulnerabilities halting glioblastoma progression
              </a>
            </td>
          <td>
            Leo Carl Foerster, Oguzhan Kaya, Valentin Wüst, Diana-Patricia Danciu, Vuslat Akcay, Milica Bekavac, Kevin Chris Ziegler, Nina Stinchcombe, Anna Tang, S. Kleber, Joceyln Tang, Jan Brunken, Irene Lois-Bermejo, Noelia Gesteira-Perez, Xiujian Ma, Ahmed Sadik, P. Le, Kevin Petrecca, Christiane A Opitz, Haikun Liu, C. R. Wirtz, Angela Goncalves, A. Marciniak-Czochra, Simon Anders, A. Martin-Villalba
          </td>
          <td>2025-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related mortality globally, with non-small cell lung cancer (NSCLC) representing 85% of cases. Advances in treatment modalities, including stereotactic radiation therapy, have improved outcomes. However, possible synergistic effects of these therapies remain underexplored at the molecular level. This study investigated high-dose radiation-induced proteomic changes in lung adenocarcinoma cell line HCC-44 grown adherently and cell line A549, grown as adherent cells or spheroids. Our hypothesis was that proteins upregulated by 10 Gy irradiation serve as resistance drivers in cancerous cells and can thus represent potential therapeutic targets. The label-free mass spectrometry revealed distinct proteomic responses to 10 Gy irradiation, varying by cell line and culturing conditions. Differentially expressed proteins elevated in the irradiated samples included ephrin type-A receptor 2 (EPHA2) in adherent cells and insulin-like growth factor 2 receptor, tetraspanin 3 as well as cathepsin D (CTSD) in spheroids. The validation of these targets was carried out via Western blot, immunofluorescence, viability assay and spheroid formation assay which demonstrated that irradiation sensitized adherent NSCLC cells to EPHA2 and CTSD inhibitors. These findings underscore the potential of integrating radiation and targeted therapies in NSCLC treatment, and highlight EPHA2 as a promising candidate for future therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-09285-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b394ae32bb3fba2e6d32961ffafe24b8e9f20a6" target='_blank'>
              Proteome changes associated with effect of high dose single-fractionation radiation on lung adenocarcinoma cell lines
              </a>
            </td>
          <td>
            Elina Leis, Vijayachitra Modhukur, H. Lust, M. Vardja, Merilin Saarma, Ivar Ilves, Margaret Pütsepp, Sergei Kopanchuk, Jana Jaal, D. Lavogina
          </td>
          <td>2025-07-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Cervical cancer’s tumor microenvironment (TME) was composed of a diverse array of immune cells that significantly influence tumor progression and response to treatment. Recent advancements in multi-omics and single-cell sequencing had provided valuable insights into the cellular heterogeneity and immune landscape of the TME, revealing critical interactions that shape tumor behavior and therapy outcomes. Method This study used multi-omics and single-cell sequencing to explore the immune landscape, cellular heterogeneity, and drug sensitivity in cervical cancer, focused on tumor subtypes and their interactions with immune cells, and aimed to understand therapy responses. Results The research presented a thorough single-cell analysis of cervical cancer, identified distinct tumor epithelial cell (EPC) subtypes, and explored their roles in tumor progression, immune evasion, and therapeutic response. It underscored the potential of tumor EPCs as valuable biomarkers for prognosis and as targets for personalized treatment approaches. Conclusion The immune landscape of cervical cancer and its interaction with tumor endothelial progenitor cells played crucial roles in determining the tumor’s progression and response to therapy. The classification of tumor subtypes based on immune characteristics and drug sensitivity was critical for personalized treatment. The identification of TSPAN1 as key biomarkers provided insight into tumor biology and potential therapeutic targets. Our findings emphasized the need for combining immune checkpoint modulation with precise drug sensitivity analysis to optimize treatment strategies, particularly in advanced cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db53e3e78ba135d282fa510797c202842bdafc26" target='_blank'>
              Revolutionizing cervical cancer treatment: single-cell sequencing of TSPAN1+ tumor EPCs and immune checkpoints to assess drug sensitivity and optimize therapy
              </a>
            </td>
          <td>
            Yumeng Li, Zhiheng Lin, Guangyao Lin, Zhijie Zhao, Zhikai Xiahou, Pingping Cai
          </td>
          <td>2025-07-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Tumor tissues exhibit cell-specific genetic variations and gene expression profiles critical to disease diagnosis, progression, and treatment outcomes. Single cell sequencing offers detailed insights into the heterogeneity of cancer cells which is pivotal for developing personalized treatment strategies. However, current high-throughput methods primarily detect gene expression levels, lacking information on genetic variants such as mutations or gene fusions. Understanding tumor responses to therapy requires linking cell-type specific mutations with gene expression profiles to understand cancer development and progression.



 We developed FocuSCOPE, a high-throughput single cell sequencing solution that simultaneously measures gene expression and detects genetic variantion from individual cells. Single cell partitioning is performed using SCOPE-chip®, a portable microwell chip the size of a microscopy slide. The technology utilizes barcoding microbeads conjugated with oligos to efficiently capture mRNA and target sequences. The optimized chemistry facilitates amplification and library construction of both mRNA and target regions, including point mutations, fusion genes, and non-polyadenylated RNAs such as viral RNAs.



 NB4, CCRF, and K562 cell lines were mixed in equal proportions. Libraries were prepared using the FocuSCOPE Blood Cancer Panel. Experiments demonstrated the specific detection of KRAS and TP53 mutations in mixed cell lines (NB4, CCRF, and K562) using the FocuSCOPE Blood Cancer Panel. The NB4 cell line contained KRAS (A18D) and TP53 (R248Q) mutations, CCRF contained KRAS (G12D) and TP53 (R248Q, R175H), and K562 contained only the BCR-ABL1 fusion gene. Additionally, FocuSCOPE effectively captured PML-RARA and BCR-ABL1 fusion genes with high sensitivity.



 FocuSCOPE enables comprehensive analysis of gene expression profiles and targeted genetic sequences, capturing driver mutations, fusion genes, and non-polyA sequences such as viral sequences with high sensitivity, thus advancing our understanding of tumor biology and therapeutic responses. This makes it an invaluable tool for personalized medicine, guiding therapeutic decisions and improving patient outcomes.



 Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick. FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P17.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafbd9d94e3c2b4f9a27806a0df8f1e19c248cc6" target='_blank'>
              Abstract P17: FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia
              </a>
            </td>
          <td>
            Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is characterized by pervasive genomic instability and high inter- and intra-tumor heterogeneity. Approximately half of HGSC tumors harbor homologous recombination deficiency (HRD), rendering them vulnerable to PARP inhibitors and platinum-based chemotherapy. In contrast, patients lacking HRD (HR-proficient, HRP) generally respond poorly to the current therapies. To overcome heterogeneity and identify relevant HGSC subtypes, we characterized the genomic landscape of 640 tumors from 243 patients using whole-genome sequencing. Our chromosomal instability (CIN) signature-based analysis characterized the structural variation landscape and revealed five HGSC subtypes, validated in an independent dataset. Two HRD subtypes, associated with BRCA1- or BRCA2-driven alterations, demonstrated favorable treatment responses. Strikingly, three HRP subtypes emerged, marked by unique structural alterations and gene expression patterns, tumor microenvironment interactions, and different chemotherapy responses. Finally, organoid experiments showed subtype-specific sensitivity to CHK1 inhibition, suggesting prexasertib as a potential targeted treatment for most currently untreatable HRP patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e62a189c25cb112512c18bbcd2dda397ed67ef" target='_blank'>
              Decoding the Genomic and Functional Landscape of Emerging Subtypes in Ovarian Cancer.
              </a>
            </td>
          <td>
            Giulia Micoli, K. Lavikka, Yilin Li, A. Pirttikoski, D. Afenteva, Wojciech Senkowski, G. Marchi, Anna Vaharautio, T. Muranen, Titta Joutsiniemi, S. Hietanen, A. Virtanen, K. Wennerberg, J. Hynninen, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-07-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="BACKGROUND
Efferocytosis (ER) plays a crucial role in the programmed clearance of dead cells, a process that is mediated by phagocytic immune cells. However, further exploration is needed to determine the full extent of its impact on the progression of pancreatic ductal adenocarcinoma (PDAC), particularly through interactions among tumor cells, stromal cells, and immune cells within the tumor microenvironment (TME).


METHODOLOGY AND RESULTS
In this study, we comprehensively analyzed the Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) database, as well as additional databases from multiple bioinformatics websites, utilizing 167 ER features derived from the integration of single-cell RNA sequencing (scRNA-seq) and bulk transcriptomic data. A set of 14 ER-associated prognostic signatures, referred to as the "14-gene panel" genes, was identified based on overall survival (OS)/disease-free survival (DFS) data, Pearson correlation coefficients, and multivariate Cox regression analyses. The model pathways enriched by the four-gene combination represented by "LEAF" and the 14-gene combination represented by the "14-gene panel" presented a high degree of similarity, including among the adhesion, mitotic, G2/M checkpoint, and epithelial‒mesenchymal transition (EMT) signaling pathways. Least absolute shrinkage and selection operator (LASSO) regression was subsequently employed to construct an ER risk scoring system using deep learning, based on the following formula: LGALS3, EMP1, ASPH, and FNDC3B, collectively termed the "LEAF" panel. Additionally, random survival forest (RSF) algorithms facilitated the identification of a key panel of genes, designated "LEAP" genes, including LGALS3, EREG, ASPH, and PLS3; three of which genes (ASPH, LGALS3, and EREG) were identified as key factors influencing the behaviors of PDAC tumors, tumor-associated stroma, and macrophages. Finally, we utilized experimental methods, including Boyden chamber analyses, immunohistochemical staining, and cell cycle analyses, to demonstrate that interference with ASPH suppresses the malignant properties of tumors, including proliferation and migration. Multiplex immunofluorescence staining was employed to identify EREG as highly relevant to the M2 macrophage subpopulation.


CONCLUSION
Our findings underscore the importance of considering a novel prognostic signature comprising 14 ER genes in the context of the TME when investigating the biology of PDAC. Future studies may explore how modulating these interactions could lead to novel therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59c81897cb1c769e789b52358a323bc255ac9a02" target='_blank'>
              A Novel 14-Gene Panel Associated With Efferocytosis for Predicting Pancreatic Cancer Prognosis Through Bulk and Single-Cell Databases.
              </a>
            </td>
          <td>
            Yaheng Wu, Lin Zhao, Dingyan Yi, Zhihua Tian, Bin Dong, Chunxiang Ye, Jingtao Liu, Huachong Ma, Wei Zhao
          </td>
          <td>2025-07-30</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Background: Multiplex immunofluorescence (mIF) captures spatial relationships within the tumor microenvironment and predicts response to therapy. However, cost and complexity limit real-world clinical application. We evaluated generative artificial intelligence approaches to recreate virtual mIF images from routine hematoxylin and eosin (H&E) stained pathology. Methods: Core-needle biopsies from a cohort of triple-negative breast cancer underwent staining and mIF imaging. H&E and a composite mIF image for DAPI, pan-CK, CD3, CD4, CD8, and CD20 were aligned. We designed a pipeline, multiplex Synthetic Immunofluorescence Generated through H&E Translation (mSIGHT), which integrates a trainable registration network to overcome alignment issues in pairs of H&E and mIF images. Pairs of image tiles were extracted to train the mSIGHT pipeline, and the results were compared to Pix2Pix and CycleGAN image translation approaches. The mean squared error (MSE) between the six image channels was compared across generators using a paired t-test in the validation cohort, and Frechet Inception Distance (FID) was also compared. Cells were segmented and clustered into tumor cells, B-cells, CD4+ T-cells, and CD8+ T-cells. The accuracy of cell classification was measured using Spearman’s correlation coefficient between real and generated classifications, as well as the area under the receiver operating characteristic curve (AUROC) for the prediction of real cell density in the top tertile of images from the validation set. The association of predicted immune cell subsets with pathologic complete response (pCR) was evaluated using an adjusted logistic regression model in a retrospective cohort of patients receiving neoadjuvant therapy, controlling for hormone receptor status and grade. Results: Fifteen core needle biopsies yielding 418 image tiles were used for training, and two biopsies yielding 160 images were used for validation of image generation and cell classification. The mSIGHT pipeline (MSE 0.11, 95% CI 0.10 - 0.12, FID 125.7) outperformed Pix2Pix (MSE 0.31, 95% CI 0.28 - 0.33, p < 0.001 vs mSIGHT; FID 322.0) and CycleGAN (MSE 0.14, 95% CI 0.13 - 0.15, p = 0.019 vs mSIGHT; FID 167.0) in average pixel level accuracy across all channels. Generated images provided significant correlations for all immune cell subtypes, although accuracy ranged from Spearman rho 0.49 (95% CI 0.37 – 0.60), AUROC 0.77 (95% CI 0.69 – 0.85) for CD4+ T-cells to Spearman rho 0.20 (95% CI 0.05 – 0.35), AUROC 0.61 (95% CI 0.51 – 0.71) for CD8+ T-cells. In a cohort of 310 cases treated with neoadjuvant therapy with digital H&E images available, the predicted density of CD8+ T-cells was significantly associated with pCR (adjusted odds ratio [aOR] 1.34, 95% CI 1.16 – 1.62), whereas among spatial metrics derived from virtual mIF, proximity of CD4+ T-cells within 20 microns of B-cells was negatively associated with pCR (aOR 0.63, 95% CI 0.44 – 0.91). Conclusions: The mSIGHT pipeline may be used to gain insight into clinically relevant spatial immune features directly from H&E images. Compared to other image translation pipelines, mSIGHT demonstrated superior predictive accuracy for identification of immune cell subsets. Additional study of this approach using larger datasets is ongoing.
 Citation Format: Anran Li, Madeleine Torcasso, Anna Woodard, Hanna Hieromnimon, James Dolezal, Rebecca Abraham, Marcus R. Clark, Yuanyuan Zha, Alexander T. Pearson, Frederick M. Howard. Virtual Multiplex Immunofluorescence Identifies Lymphocyte Subsets Predictive of Response to Neoadjuvant Therapy [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80d0625b901a1ed5f429740699e4c822f7f73f9" target='_blank'>
              Abstract P2-02-19: Virtual Multiplex Immunofluorescence Identifies Lymphocyte Subsets Predictive of Response to Neoadjuvant Therapy
              </a>
            </td>
          <td>
            Anran Li, Madeleine S. Torcasso, Anna Woodard, Hanna M. Hieromnimon, James Dolezal, Rebecca Abraham, Marcus R. Clark, Yuanyuan Zha, Alexander T Pearson, Frederick M. Howard
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4144c066603d461af67a1609df3300fe3c044e71" target='_blank'>
              Single-cell and spatial transcriptomics reveal mechanisms of radioresistance and immune escape in recurrent nasopharyngeal carcinoma.
              </a>
            </td>
          <td>
            Rui You, Qunlun Shen, Chao Lin, Kangning Dong, Xiao Liu, Hanshi Xu, Wanming Hu, Yulong Xie, Ruoqi Xie, Xiaoyi Song, Chunliu Huang, Jinhui Wu, Tao Yu, Huifeng Li, Zining Wang, Keming Chen, Xiong Zou, Peiyu Huang, Yijun Hua, Youping Liu, Tianliang Xia, Shihua Zhang, Mingyuan Chen
          </td>
          <td>2025-07-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Gastric cancer (GC) is characterized by heterogeneous tumor microenvironment (TME) with various cell types contributing to disease progression and patient outcomes. This study aims to dissect the single-cell transcriptomic landscape of GC, highlighting the role of tumor-associated macrophages (TAMs) and establishing a novel prognostic signature based on high oxidative phosphorylation (OXPHOS) macrophages. Methods Single-cell sequencing data from paired GC and normal stomach tissues, obtained from the GEO database (GSE184198), were processed to reveal cellular heterogeneity and identify TAM subsets with high OXPHOS activity. Using the TCGA STAD dataset, survival analyses were conducted on 435 GC patients to establish a high-OXPHOS-macrophage-related prognostic signature. Results We identified eight distinct cell types within the GC TME, indicating significant cellular heterogeneity. Macrophages, particularly TAMs, were found in greater numbers in tumor tissue, with the C3 macrophage subset exhibiting the highest OXPHOS score. A 19-gene high-OXPHOS-macrophage-related prognostic signature was constructed, stratifying patients into different risk categories with significant survival differences (P<0.05). NPC2, LY96, and TPP1 were identified as key macrophage-expressed markers, correlating with prognosis. Cell communication analysis revealed increased interaction in tumor tissues, especially involving NPC2, LY96, and TPP1 positive macrophages, which facilitated tumorigenesis and immune evasion. Conclusion The high-OXPHOS-macrophage-related prognostic signature derived from scRNA-seq data provides valuable insights into GC patient stratification. NPC2, LY96, and TPP1, highly expressed in TAMs, were implicated in promoting tumor growth and immune escape, offering potential targets for novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f9b92dd366657dc4590a10fb71fe958b8515ec7" target='_blank'>
              Prognostic implications of high- OXPHOS macrophages in gastric cancer: a single-cell transcriptomics and tumor microenvironment communication study
              </a>
            </td>
          <td>
            Ziyuan Lin, Yunyu Xu, Xiaohe Xu, Xinjun Lin, Lin Cheng, Qifeng Zhao
          </td>
          <td>2025-06-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Melanoma metastasis, driven by tumor microenvironment (TME)-mediated crosstalk facilitated by extracellular vesicles (EVs), remains a major therapeutic challenge. A critical barrier to clinical translation is the overlap in protein cargo between tumor-derived and healthy cell EVs. Objective: To address this, we developed Scaffold-free Functional Deconvolution (SFD), a novel computational approach that leverages a comprehensive healthy cell EV protein database to deconvolute non-oncogenic background signals. Methods: Beginning with 1915 proteins (identified by MS/MS analysis on an Orbitrap Fusion Lumos Mass Spectrometer using the IonStar workflow) from melanoma EVs isolated using REIUS, SFD applies four sequential filters: exclusion of normal melanocyte EV proteins, prioritization of metastasis-linked entries (HCMDB), refinement via melanocyte-specific databases, and validation against TCGA survival data. Results: This workflow identified 21 high-confidence targets implicated in metabolic-associated acidification, immune modulation, and oncogenesis, and were analyzed for reduced disease-free and overall survival. SFD’s versatility was further demonstrated by surfaceome profiling, confirming enrichment of H7-B3 (CD276), ICAM1, and MIC-1 (GDF-15) in metastatic melanoma EV via Western blot and flow cytometry. Meta-analysis using Vesiclepedia and STRING categorized these targets into metabolic, immune, and oncogenic drivers, revealing a dense interaction network. Conclusions: Our results highlight SFD as a powerful tool for identifying clinically relevant biomarkers and therapeutic targets within melanoma EVs, with potential applications in drug development and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6953d309a58e875cd28384fa7e281232e73ceb" target='_blank'>
              Scaffold-Free Functional Deconvolution Identifies Clinically Relevant Metastatic Melanoma EV Biomarkers
              </a>
            </td>
          <td>
            S. Shu, Shawna Benjamin-Davalos, Xue Wang, Eriko Katsuta, M. Fitzgerald, Marina Koroleva, Cheryl L. Allen, Flora Qu, Gyorgy Paragh, Hans Minderman, Pawel Kalinski, K. Takabe, Marc S. Ernstoff
          </td>
          <td>2025-07-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Background: Heterogeneous circulating tumor cells (CTCs) have been implicated in the formation of new metastases. However, circulating cells expressing both tumor and immune cell proteins are often dismissed as insignificant findings in CTC studies. Methods: Two non-contemporaneous blood samples from a metastatic breast cancer patient were analyzed using an enrichment-free platform to identify canonical, epithelial-only CTCs (CD45-/cytokeratin + , epi.CTCs) and CD45 + /cytokeratin+ immune-like CTCs (im.CTCs). Single cells from both samples were subjected to copy number and protein expression profiling. A cohort of 36 metastatic breast cancer patients was then analyzed to search for additional cases with im.CTCs. Results: Here, we identified and characterized a population of CTCs exhibiting an immune-like state. In two samples from an index patient, im.CTCs outnumbered epi.CTCs, comprising >97% of the CTC population. Single-cell copy number analysis of 43 im.CTCs and 30 epi.CTCs revealed clonal alterations across both populations, confirming a shared tumor origin. Furthermore, im.CTCs contained pseudo-diploid profiles that did not reflect dilution from the addition of a normal diploid genome, indicating that they were unlikely to have originated from tumor-immune cell fusion. Protein expression analysis showed that im.CTCs express CD45 as well as other immune-related markers, such as CD3 and CD4, and the cancer stemness marker, CD44. Subsequent analysis of a metastatic breast cancer cohort identified an additional patient harboring im.CTCs with the same tumor-derived, non-fusion genome as in the index case. Conclusions: Collectively, these genomic and proteomic features distinguish im.CTCs from previously reported circulating cells may represent a novel form of tumor cell plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aef97fcd405edd04a5788c8bfb045e76aa5a9c2" target='_blank'>
              Simultaneous expression of epithelial and immune cell markers in circulating tumor cells identified in patients with stage 4 breast cancer
              </a>
            </td>
          <td>
            Nikki Higa, Audrey Limb, Valerie Hennes, Andrés Rivera, Rafael Nevarez, A. Kolatkar, Carol Kaplan Tweed, Adam I Riker, Young Lee, Lorraine Tafra, Jeremy G Perkins, Craig D Shriver, Peter Kuhn, James B. Hicks
          </td>
          <td>2025-07-24</td>
          <td>Communications Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) occurs as the third most common cancer with high mortality across the world. Understanding the intratumoral immune cell heterogeneity and their responses to various therapies is crucial for enhancing patient outcomes. This study aimed to characterise and evaluate the immune microenvironment landscapes of CRC shaped by different therapies including CD73 inhibitor, PD‐1 blockade and photothermal therapy (PTT). Our investigation revealed that three therapies could commonly modulate the down‐regulation of Treg, M2 macrophage and Ptprj+ G4 granulocyte, up‐regulation of effector/memory T cell, M1 macorphage and Hilpda+ G1 granulocyte. Moreover, we identified the uniquely dis‐regulated cell types and pathway activities response to each therapy, such as CD73 inhibitor enriched more Cd8+ memory and central memory (CM) cell, PD‐1 blockade with more Cd8+ CTL and Cxcl3+ G2 granulocyte, and PTT with more Cd8+ effector memory and Rethlg+ G3 granulocyte cell. These responses disordered the glycolysis, angiogenesis, phagocytosis functions and cellular communication to reshape the CRC tumour immune microenvironment. We provide the detail insights into the intratumoral immunomodulation preferences of CRC mice treated with CD73 inhibitor, PD‐1 blockade and PTT therapies, which might contribute to the ongoing development of more effective anticancer strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ef48ff72788aaf2bb81620f30c069f2cf46018" target='_blank'>
              Characterising and Evaluating the Immune Microenvironment Landscapes of Colorectal Cancer Shaped by Different Therapies
              </a>
            </td>
          <td>
            Chen Zhou, Yifan Wang, Yuanyuan Li, Weitao Zhang, Yunmeng Bai
          </td>
          <td>2025-07-16</td>
          <td>IET Systems Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Sporadic colorectal cancer (CRC), the third leading cause of cancer-related death globally, arises through a continuum from normal tissue to adenomas, progressing from low-grade (LGD) to high-grade dysplasia (HGD); yet, the early epigenetic drivers of this transition remain unclear. To investigate these events, we profiled LGD and HGD adenomas using EM-seq, and identified a consensus differential methylation signature (DMS) of 626 regions through two independent bioinformatics pipelines. This signature effectively distinguished LGD from HGD in both tissue and plasma-derived cell-free DNA (cfDNA), highlighting specific methylation patterns. Functional annotation indicated enrichment for regulatory elements associated with transcription factor activity and cell signaling. Applying the DMS to the TCGA CRC dataset revealed three tumor subtypes with increasing hypermethylation and one normal cluster. The most hypermethylated subtype exhibited poor survival, high mutation burden, and disrupted transcriptional networks. While overlapping with classical CpG Island Methylator Phenotype (CIMP) categories, the DMS captured a broader spectrum of methylation alterations. These findings suggest that the DMS captures functionally relevant, antecedent epigenetic alterations in CRC progression, enabling the robust stratification of dysplasia severity and tumor subtypes. This signature holds promise for enhancing preclinical detection and molecular classification, and warrants further evaluation in larger prospective cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c27b0ef917c9876876a5f66fb35ed8c368fc721c" target='_blank'>
              Multiomics Signature Reveals Network Regulatory Mechanisms in a CRC Continuum
              </a>
            </td>
          <td>
            Juan Carlos Higareda-Almaraz, Francesco Mattia Mancuso, P. Canal-Noguer, Kristi Kruusmaa, Arianna Bertossi
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Endometrial carcinoma (EC), the most common gynecologic cancer type in developed countries, encompasses four molecular subtypes (POLEmut, MMRd, p53abn, and NSMP) that have prognostic values and guide treatment decisions. Additionally, dual loss of ARID1A and ARID1B (referred to as ARID1A/B) characterizes a significant portion of dedifferentiated/undifferentiated EC (DD/UDEC), a rare but highly aggressive subtype of EC. To advance the translational research for ECs, we analyzed the genomic features of a panel of 39 EC cell lines, leading to the identification of cell lines representing each of these EC molecular subtype. Histologic and immunohistochemical analyses of xenografted tumors from these cell lines confirmed their resemblance of cognate primary EC molecular subtypes. Further investigation of the publicly available genome-wide CRISPR screen data for EC cell lines identified multiple specific genetic dependencies in MMRd, p53abn, and ARID1A/B-dual deficient EC cell lines. Particularly, ARID1A/B-dual deficient DD/UDEC cells selectively rely on mitochondrial oxidative phosphorylation in vitro and in vivo. Therefore, through molecular subtyping of EC cell lines and subsequent characterization of molecular subtype-specific genetic dependencies, our study provides a framework that guides the utility of the EC cell line models for accelerating translational research in EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e1f3b25336fe4feba408a071575d0aa7bd9be8e" target='_blank'>
              Molecular subtyping of endometrial carcinoma cell lines uncovers subtype-specific targetable vulnerabilities
              </a>
            </td>
          <td>
            Eunice Li, Rebecca Ho, Ran Tao, Yannes Wai Yan Choi, Chae Young Shin, Shary Chen, Bengul Gokbayrak, J. Senz, Betty Yao, Liam Johnston, Spencer D Martin, Eric Yang, Mark S Carey, Bryan T. Hennessy, D. Huntsman, Ramon I Klein Geltink, Lynn Hoang, Yemin Wang
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="A compelling body of research has confirmed the critical role of the tumor microenvironment (TME) in the growth and aggressiveness of breast cancer (BC). Cancer-associated fibroblasts (CAF), the predominant cell component of the TME, significantly influences its behavior through complex crosstalk with cancer and other stromal cells in response to tumor-derived activating signals. Murine models are extensively used to investigate the mechanisms underlying CAF's role in BC progression, as they allow for the evaluation of the endogenous TME and the assessment of immune system involvement using syngeneic xenograft studies. Here, we present a set of methods for culturing BC-derived CAFs, manipulating gene expression, and evaluating their pro-tumorigenic functions using both in vitro and in vivo approaches. We also describe protocols for CAF gene expression profiling and transcriptomic analysis, alongside a computational procedure for inferring transcriptional signatures in primary tumors using publicly available bulk RNA-seq data. Additionally, we introduce a user-friendly, web-based tool for analyzing the transcriptomes of stromal and epithelial components from laser capture micro-dissected BC tumors. Ultimately, these methods can provide valuable insights into the molecular mechanisms driving CAF-tumor interactions and hold potential for identifying novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f94a97738d3f7846e0a2b6fcec7f408e4d3b367d" target='_blank'>
              Cancer-Associated Fibroblasts from Mouse Mammary Tumors as Tools for Molecular and Computational Studies.
              </a>
            </td>
          <td>
            Daniele Viavattene, Aurora Savino, Somayeh Mirzaaghaei, Andrea Lobascio, Valeria Poli, L. Avalle
          </td>
          <td>2025-07-03</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Merkel cell carcinoma (MCC) is a rare but lethal skin neoplasm, caused, in approximately 80% of cases, by the genomic integration of Merkel cell polyomavirus (MCPyV) and the expression of viral oncoproteins small T (sT) and large T (LT) antigens. Virus-negative MCCs exhibit extensive UV-induced mutations. Although there is a growing understanding of MCC pathogenesis, the cellular origin of MCC remains a topic of intense investigation and debate. In this perspective, we will provide a description and discussion of the current theories regarding the cellular ancestry of MCC. The most recent findings point in favor of a potential epithelial origin of MCC. MCPyV integration likely occurs in an epithelial precursor cell prior to MCPyV-driven clonal expansion, while the same originating cell type may undergo a specific molecular switch that drives neuroendocrine differentiation, leading to UV-mutated, virus-negative MCCs. Identifying the cellular origin of MCC is crucial for developing accurate pre-clinical models and advancing clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4673ac623d6244ccbb4d7742c720647c8c4f5a0" target='_blank'>
              The complex conundrum of Merkel cell carcinoma cellular ancestry
              </a>
            </td>
          <td>
            C. Mazziotta, Fernanda Martini, J. Rotondo
          </td>
          <td>2025-07-29</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="ABSTRACT In most cancers, T lymphocytes comprise an essential cellular component of the non-neoplastic microenvironment, where they have the capacity to both suppress and support tumor growth. One specialized T lymphocyte population is the CD8+ exhausted T cell, which has been intensely studied as an actionable therapeutic target. Unfortunately, there is currently no uniformly accepted classification scheme for these specialized T cells. To provide a potential model for classifying CD8+ exhausted T cells, we leveraged single cell transcriptomic analysis of a diverse collection of both human (n = 8) and mouse (n = 4) cancers to identify unique subpopulations shared across tumor types and species. By integrating data from both human and mouse cancer studies, as well as previously described CD8+ exhausted T cell subsets, we provide an integrated framework to characterize the heterogeneity of exhausted CD8+ T cells. As such, one of these subpopulations (cluster C1) increases following immune checkpoint inhibitor treatment in the setting of cancer in mice and patients. Taken together, this proposed classification scheme may be useful for the design and interpretation of current and future immune-based therapy studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9bb9b37e719dff3c4a062c99fe8116dd65c7d0" target='_blank'>
              Pan-cancer single cell transcriptomic clustering reveals heterogeneous CD8+ exhausted T cell populations with different immune checkpoint inhibitor responses
              </a>
            </td>
          <td>
            Rui Mu, Rasha Barakat, David H. Gutmann
          </td>
          <td>2025-08-03</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer (BC) is commonly labeled a “cold tumor” due to its dense population of immunosuppressive cells, particularly M2-like macrophages, which contribute to its resistance to therapy. Thus, there is a pressing need to shift the macrophage polarization towards M1 and revitalize the tumor immune microenvironment (TIME) to improve BC prognosis. In this study, we leveraged published RNA-sequencing data and performed multiplex immunohistochemistry on clinical specimens to identify NR4A3 as a promising biomarker for favorable outcomes in BC. High NR4A3 expression correlates with an inflamed TIME, characterized by heightened T-cell infiltration and activation. NR4A3 was preferentially expressed in macrophages and fostered M1-like macrophage polarization through direct binding to p65, thereby enhancing NF-κB transcriptional activity. Overexpression of Nr4a3 in tumor-infiltrating macrophages significantly inhibited the growth of E0771 tumors in a syngeneic mouse model, accompanied by increased T-cell infiltration and elevated production of functional cytokines. Conversely, suppression of Nr4a3 in macrophages compromised T-cell recruitment and diminished their anti-tumor capabilities. Consistent with these findings, co-culture experiments involving human T cells and NR4A3-overexpressing THP1 cells further demonstrated enhanced T-cell functionality. Collectively, our findings uncover a novel role for NR4A3 in macrophage polarization and TIME remodeling, offering a potential biomarker for favorable BC prognosis and a therapeutic target to enhance immunotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce686251c087adfd1cc06d0b58d3463e9ccf7c9f" target='_blank'>
              NR4A3 potentials M1-like macrophage polarization to facilitate anti-tumor immune responses in breast cancer
              </a>
            </td>
          <td>
            Yi-yu Qian, Ning Jin, Shan-shan Rao, Ya Wang, Xin Li, Wen Pan, Pu Huang, Si-yuan Wang, Ping-fei Li, Yan-kai Lv, Qing-lei Gao, Yu Xia
          </td>
          <td>2025-07-07</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d135048b3a2d9e0d80a5651add7d7f82d46f7f4a" target='_blank'>
              Innate immunity and the NF-κB pathway control prostate stem cell plasticity, reprogramming and tumor initiation.
              </a>
            </td>
          <td>
            Chen Jiang, Yura Song, S. Rorive, Justine Allard, Elisavet Tika, Zahra Zahedi, C. Dubois, Isabelle Salmon, Alejandro Sifrim, Cédric Blanpain
          </td>
          <td>2025-06-23</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are responsible for colorectal cancer (CRC) chemoresistance, recurrence, and metastasis. Therefore, identifying molecular stemness targets that are involved in tumor growth is crucial for effective treatment. Here, we performed an extensive in vitro and in vivo molecular and functional characterization, revealing the pivotal role of the lysine methyltransferase SET and MYND Domain Containing 3 (SMYD3) in colorectal cancer stem cell (CRC-SC) biology. Specifically, we showed that SMYD3 interacts with and methylates c-MYC at K158 and K163, thereby modulating its transcriptional activity, which is implicated in stemness and colorectal malignancy. Our in vitro data suggest that SMYD3 pharmacological inhibition or its stable genetic ablation affects the clonogenic and self-renewal potential of patient-derived CRC-SCs and organoids by altering their molecular signature. Moreover, we found that SMYD3 stable knock-out or pharmacological inhibition drastically reduces CRC tumorigenicity in vivo and CRC-SC metastatic potential. Overall, our findings identify SMYD3 as a promising therapeutic target acting directly on c-MYC, with potential implications for countering CRC-SC proliferation and metastatic dissemination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/508ee1e83935ec4b0c34a584bea5279ee9131213" target='_blank'>
              Tailoring a novel colorectal cancer stem cell-targeted therapy by inhibiting the SMYD3/c-MYC axis
              </a>
            </td>
          <td>
            Martina Lepore Signorile, Elisabetta Di Nicola, G. Forte, P. Sanese, C. Fasano, Vittoria Disciglio, Katia De Marco, Marialaura Latrofa, Loris De Cecco, Marica Ficorilli, Marta Lucchetta, E. Torchia, Chiara Dossena, Giusy Bianco, Vito Spilotro, C. Ferroni, Nicoletta Labarile, R. Armentano, Francesco Albano, A. Mestice, Gianluigi Gigante, V. Lantone, G. Lantone, Leonardo Vincenti, Alberto Del Rio, G. Varchi, V. Grossi, Cristiano Simone
          </td>
          <td>2025-06-30</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is the most prevalent subtype of lung cancer, presenting significant challenges in treatment and prognostic prediction. Despite advancements in therapeutic approaches in recent years, personalized medicine has not yet achieved a notable breakthrough. Given the poor prognosis of patients, there is an urgent need to enhance the ability for precise prediction. Mitochondria play a crucial role in the metabolism and energy production of cancer cells, yet their specific impact in lung adenocarcinoma warrants further investigation. This study leveraged data from the TCGA and GEO databases to stratify 515 lung adenocarcinoma patients into two distinct subtypes based on mitochondrial-related genes. We systematically evaluated survival outcomes and biological pathway activities between subtypes, characterized their immune infiltration profiles, and developed a prognostic model using subtype-specific differentially expressed genes. Drug sensitivity disparities were further assessed. Single-cell RNA sequencing data were analyzed using an XGBoost classifier to delineate cell-type heterogeneity across subtypes at single-cell resolution. In LUAD, we identified two distinct subtypes. One subtype exhibited active mitochondrial metabolism, which was associated with poor prognosis and higher tumor purity. Moreover, this subtype showed greater sensitivity to Osimertinib. Further single-cell analysis revealed that this subtype was characterized by substantial macrophage infiltration, potentially promoting tumor progression through the NF-κB signaling pathway. Overall, our study identified novel LUAD subtypes and provided new insights into the clinical treatment of LUAD. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07982-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08aa1cb8c92b7c61ac76cf9ab60fdfeab09318a6" target='_blank'>
              Mitochondria-related gene-based molecular subtypes of lung adenocarcinoma and their prognostic implications
              </a>
            </td>
          <td>
            Ziyi Zhanghuang, Fei Xie, Xuemei Ma, Jinfeng Chen
          </td>
          <td>2025-07-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Background Triple‐negative breast cancer (TNBC) poses significant challenges due to its aggressive nature and lack of targeted therapies. Understanding the cellular behaviors of TNBC is crucial for developing effective treatments. Aims This study aims to compare the morphological characteristics of non‐tumorigenic MCF10A and aggressive MDA‐MB‐231 TNBC cell lines using advanced analytical techniques. Methods and Results Advanced techniques such as Principal Component Analysis (PCA), t‐Distributed Stochastic Neighbor Embedding (t‐SNE), and digital holographic microscopy were utilized. Cellular features such as area, migration, motility, irregularity, and optical thickness were thoroughly analyzed over time. Our results revealed significant morphological differences between the MCF10A and MDA‐MB‐231 cell lines. Specifically, MDA‐MB‐231 cells displayed enhanced motility and a smaller, more variable size, attributes that may facilitate their invasive potential. In contrast, MCF10A cells exhibited larger sizes and more regular migration patterns, suggesting stability in structured tissue environments. Additionally, temporal analysis highlighted consistent phenotypic behaviors over time, with MDA‐MB‐231 cells demonstrating higher optical thickness and irregularity, indicating potential structural complexities associated with malignant transformation. Correlative analysis further confirmed these results by revealing connections between cell size, motility, and optical properties crucial for understanding cell behavior within their microenvironment. Conclusion The profound differences in cellular dynamics between MCF10A and MDA‐MB‐231 cell lines underscore the unique adaptive mechanisms of TNBC cells. Our study provides valuable insights into the cellular foundations of TNBC aggressiveness, offering a foundation for future research aimed at understanding the mechanistic underpinnings of TNBC progression and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9f97f859a9a924b82dc01461c45eed54b60c098" target='_blank'>
              Dissecting Morphological and Functional Dynamics of Non‐Tumorigenic and Triple‐Negative Breast Cancer Cell Lines Using PCA and t‐SNE Analysis
              </a>
            </td>
          <td>
            Yazan A. Almahdi, Eva R Schwark, Aidan J. McLaughlin, Besa Xhabija
          </td>
          <td>2025-06-27</td>
          <td>Cancer Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Metastases are the leading cause of cancer-related deaths. The spread of neoplasms involves multiple mechanisms, with metastatic tumors exhibiting molecular behaviors distinct from their primary counterparts. The key hallmarks of metastatic lesions include chromosomal instability, copy number alterations (CNAs), and a reduced degree of subclonality. Furthermore, metabolic adaptations such as enhanced glycogen synthesis and storage, as well as increased fatty acid oxidation (FAO), play a critical role in sustaining energy supply in metastases and contributing to chemoresistance. FAO promotes the infiltration of macrophages into the tumor, where they polarize to the M2 phenotype, which is associated with immune suppression and tissue remodeling. Additionally, the tumor microbiome and the action of cytotoxic drugs trigger neutrophil extravasation through inflammatory pathways. Chemoresistant neutrophils in the tumor microenvironment can suppress effector lymphocyte activation and facilitate the formation of neutrophil extracellular traps (NETs), which are linked to drug resistance. This article examines the genomic features of metastatic tumors, along with the metabolic and immunological dynamics within the metastatic tumor microenvironment, and their contribution to drug resistance. It also discusses the molecular mechanisms underlying resistance to chemotherapeutic agents commonly used in the treatment of metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7adea99fe5f3be60db7314e38e07d8d5a4381e12" target='_blank'>
              Analyzing the Blueprint: Exploring the Molecular Profile of Metastasis and Therapeutic Resistance
              </a>
            </td>
          <td>
            G. Avalos-Navarro, M. Gallegos-Arreola, Emmanuel Reyes-Uribe, Luis Felipe Jave Suárez, Gildardo Rivera-Sánchez, Héctor Rangel-Villalobos, Ana Luisa Madriz-Elisondo, Itzae Adonai Gutierrez Hurtado, J. J. Varela-Hernández, R. Ramírez-Patiño
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cervical cancer (CC) is the fourth commonest malignant tumor among women worldwide and is characterized by high heterogeneity and a complex ecosystem. A comprehensive understanding of the heterogeneity of tumors and the tumor microenvironment (TME) is crucial for effective CC management. Single-cell RNA sequencing (scRNA-seq) is a powerful tool that can be employed to unveil the heterogeneity of tumors and the TME, as well as to elucidate the evolutionary trajectories of tumors. Spatial transcriptomics (ST) technology, on the other hand, can address the complexity and diversity of the spatial microenvironment of tumors, thereby compensating for the limitations of scRNA-seq. As emerging technologies, both scRNA-seq and ST are increasingly being utilized in CC research. In this review, we summarized the latest advancements in scRNA-seq and ST for CC, with a focus on investigating tumor heterogeneity, the TME, tumor evolutionary trajectories, treatment resistance mechanisms, and potential therapeutic targets. These insights collectively contribute to the development of more effective treatment and prevention strategies for CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f5b3526e206bb4dcf755c55607c83c389e4ccb" target='_blank'>
              Applications and Prospects of Single-Cell RNA Sequencing and Spatial Transcriptomics in Cervical Cancer
              </a>
            </td>
          <td>
            Yifu Wang, Li Yang, Yunzhi Liu, Huangrong Ma, Miaoying Cai, Chunyu Liang, Li Zhang, Zezhuo Su, Zhiyuan Xu
          </td>
          <td>2025-06-26</td>
          <td>BioMed Research International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Stem cell plasticity is crucial for rapidly proliferating tissues, such as the intestine, to cope with microenvironmental stress and maintain homeostasis. This framework may be employed by cancer stem cells (CSCs) to interact with the tumor microenvironment (TME) and give rise to drug resistance, tumor recurrence, and metastasis. Here, we consider colorectal cancer (CRC) as an example to review the technological and conceptual advances in microenvironmental regulation of cancer stemness. Some unique features of CRC, including reprogramming of stromal cells, bacterial niche alteration, and metabolic interplay, were summarized to illustrate how the TME affects the plasticity of CSCs. The key pathways and molecular mechanisms involved in metastasis and drug resistance are reviewed to obtain a deeper understanding of how CSCs may be manipulated for better outcomes in CRC treatment. These findings advance our knowledge of tumor heterogeneity and may provide insights into the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e85fc5484d5fb7be8f2f1af589f1c3b9e327f305" target='_blank'>
              Microenvironmental Regulation of Colorectal Cancer Stemness: Technological Advances, Molecular Basis, and Therapeutic Targets
              </a>
            </td>
          <td>
            Hui Zhao, Qilin Li, Jichao Qin
          </td>
          <td>2025-06-26</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose The heterogeneity of immune cells is a critical manifestation of gastric cancer (GC) heterogeneity and significantly contributes to immune therapy resistance. Although previous studies have focused on the roles of specific myeloid cells and exhausted CD8+ T cells in immune resistance, the immune cell interaction network and its spatiotemporal distribution in GC immune resistance remain underexplored. Methods This study integrated multiple GC single-cell RNA sequencing, spatial transcriptomics, bulk-RNA sequencing, and single-cell immunotherapy datasets of our cohort (NFHGC Cohort). Methods such as single-cell subpopulation identification, transcriptomic analysis, spatial colocalization, cell communication network analysis and tissue immunofluorescence of gastric cancer were employed to investigate immune cell interactions and their molecular mechanisms in immune resistance. Results By leveraging a comprehensive approach that integrates single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq profiles, we identified 20 immune subsets with potential prognostic and therapeutic implications. Our findings suggest a stromal immunosuppressive network orchestrated by Macro_SPP1/C1QC macrophages and CD8_Tex_C1 T cells, which may form a barrier impeding antitumor immunity. Macrophage-derived MIF signaling appears to drive immunosuppression via the MIF-CD74/CXCR4/CD44 axis. Based on these observations, we developed a preliminary TME classification system using a gene signature derived from barrier-associated immune cell markers and unsupervised clustering. Conclusions Our study identified a potential stromal immunosuppressive barrier in gastric cancer, driven by Macro_SPP1/C1QC macrophages and CD8_Tex_C1 T cells, which may contribute to immune dysfunction and therapy resistance. Molecular subtyping based on this barrier’s presence could inform personalized immune therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1cf3d75b61cafecd69ad5d518d7d286742cb1e5" target='_blank'>
              Identification of a stromal immunosuppressive barrier orchestrated by SPP1+/C1QC+ macrophages and CD8+ exhausted T cells driving gastric cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Guichuang Ma, Xiaohan Liu, Qinrui Jiang, Shaowei Li, Qijing Wu, B. Liang, F. Sun, Chunhui Gu, Wangjun Liao, Zhi-Huan Zhang, Min Shi, Qiong Huang
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumor evolution is shaped by selective pressures imposed by physiological factors as the tumor naturally progresses to colonize local and distant tissues, as well as by therapy. However, the distinction between these two types of pressures and their impact on tumor evolution remain elusive, mainly, due to extensive intra-tumor heterogeneity. To disentangle the effects of these selective pressures, we analyze data from diverse cohorts of patients, of both treated and untreated cancers. We find that, despite the wide variation across patients, the selection strength on tumor genomes in individual patients is stable and largely unaffected by tumor progression in the primary settings, with some cancer-specific signatures detectable in the progression to metastases. However, we identify a nearly universal shift toward neutral evolution in tumors that resist treatment and demonstrate that this regime is associated with worse prognosis. We validate these findings on both published and original datasets. We suggest that monitoring the selection regime during cancer treatment can assist clinical decision-making in many cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95ee20056cad563b0d086b8e70065c4daacf4f" target='_blank'>
              Genome-level selection in tumors as a universal marker of resistance to therapy
              </a>
            </td>
          <td>
            Erez Persi, P. Sudalagunta, Yuri I. Wolf, Rafael R. Canevarolo, Mehdi Damaghi, K. Shain, A. Silva, Eugene V. Koonin
          </td>
          <td>2025-07-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Background: Metastatic breast cancer (MBC) is considered an incurable disease and understanding the molecular mechanisms at the basis of tumor progression has become mandatory. Circulating tumor cells (CTCs) constitute a population of rare cells that are shed in the bloodstream by primary and metastatic tumors and migrate towards distant organs. The molecular characterization of CTCs is a powerful tool to better investigate the mechanisms underlying metastasis. In this study, we combine phenotypic and transcriptional profiling of CTCs isolated from MBC patients to infer the mechanisms involved in their biology, also in relation to the metastatic site and their phenotypic features. Methods: Blood samples for CTC analysis were collected from luminal MBC patients recruited in the KENDO study (Protocol code: IRST174.19); CTCs were enriched using the RosetteSep CTC Enrichment Cocktail and frozen until downstream analyses as viable cells. To validate the antibody staining for phenotypic analysis, tests were conducted using the cancer cell line SKBR-3. In brief, SKBR-3 cells were spiked in peripheral blood of a healthy volunteer and enriched using RosetteSep CTC Enrichment Cocktail. Enriched SKBR-3 and patients’ CTCs were incubated with antibodies targeting CD45 (leukocyte marker) and epithelial markers (E-tag; EpCAM, E-cadherin) for phenotypic analysis using the DEPArray NxT platform. For transcriptomic profiling, single CTCs were isolated as single, viable cells. Libraries were prepared using the QIAseq UPX 3’ Transcriptome Kit and sequenced using MiSeq on V3-150 cycles cartridges. Bioinformatic and statistical analyses were carried out using the CLC Genomic Workbench. Results: Through feasibility analysis conducted on SKBR-3 cells, we demonstrate that our workflow involving phenotypic analysis on DEPArray NxT is an efficient tool to identify cancer cells (CD45-/E-tag+) and white blood cells (WBCs; CD45+/E-tag-). In MBC samples, we detected different cell populations: WBCs (CD45+/E-tag-), CTCs (CD45-/E-tag+), and dual-positive CTCs (CD45+/E-tag+; dpCTCs). Transcriptomic profiling was successfully performed on 37 single CTCs isolated from 7 patients. The transcriptional profile of CTCs is consistent with breast malignancy for different databases including ClinVar2019, DisGeNet, and Human Phenotype Ontology. Moreover, Gene Set Enrichment analysis highlighted pathways associated with synapse organization and calcium channel activity, suggesting the implication of the interplay between cancer cells and neurons in metastasis. Compared to a commercial RNA, CTCs express at high levels transcripts associated with cancer dissemination and genome architecture (i.e., STAG2 and H2AFZ). In CTCs from patients with bone metastasis we detected the expression of genes associated with metastatic osteotropism (i.e., S100A4, VAPA, HMGB1), while CTCs from a single patient with gastric metastasis expressed genes associated with gastric cancer metastasis and cell redox homeostasis. Transcriptional profiling of dpCTCs revealed the expression of transcripts encoding for members of the CD47/SIRPα axis, supporting their macrophage-cancer cell fusion origin, and increased glycogen biosynthetic and metabolic activity. Conclusions: Our data suggest the potential of CTC based transcriptomics in providing new insights on the clinical-biological evaluation of MBC and its mechanisms underlying metastatic cascade and organotropism. In addition, our data provide hints for the characterization of dpCTCs, a poorly studied population whose role in metastasis is still far to be fully elucidated.
 Citation Format: Michela Palleschi, Tania Rossi, Sara Bandini, Michele Zanoni, Michela Cortesi, Erika Bandini, Andrea Rocca, Giulia Gallerani, Ivan Vannini, Meropi Plousiou, Lorenzo Gerratana, Giovanni Tallini, Giovanni Martinelli, Ugo De Giorgi, Paola Ulivi. Transcriptomic and phenotypic profiling of circulating tumor cells from metastatic breast cancer patients [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-01-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48cb1aa4534473e2256f4e5d0f28171830899d25" target='_blank'>
              Abstract P3-01-17: Transcriptomic and phenotypic profiling of circulating tumor cells from metastatic breast cancer patients
              </a>
            </td>
          <td>
            M. Palleschi, T. Rossi, Sara Bandini, Michele Zanoni, M. Cortesi, E. Bandini, Andrea Rocca, G. Gallerani, Ivan Vannini, Meropi Plousiou, L. Gerratana, Giovanni Tallini, Giovanni Martinelli, U. De Giorgi, Paola Ulivi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Fusobacterium nucleatum (Fn) is commonly enriched in colorectal cancer (CRC) and associated with poor outcomes, though its mechanisms remain unclear. Our study investigated how Fn affects the tumor microenvironment through single-cell transcriptomic analyses of 42 CRC patient tissues, comparing Fn-positive and Fn-negative tumors. We discovered that Fn impairs IgA plasma cell development and secretory IgA (sIgA) production by disrupting communication with tumor-associated macrophages. Additional experiments in germ-free mice, together with our re-analysis of a publicly available single-cell RNA-seq data set from a CRC mouse model with an intact gut microbiome–both models having been orally gavaged with Fn–jointly validated the causal role of Fn in impairing sIgA induction. We identified a dysregulated IgA maturation (IGAM) module in Fn-positive patients, indicating compromised mucosal immunity and increased bacterial infiltration. This IGAM signature effectively stratified Fn-positive patients, suggesting potential for targeted therapeutic approaches. Our findings reveal that Fn disrupts sIgA production, increasing tumor microbial burden and worsening prognosis through chronic inflammation in Fn-positive CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b983088c09af7c7c8c02b6b79dd03932133b600d" target='_blank'>
              Secretory IgA dysfunction underlies poor prognosis in Fusobacterium-infected colorectal cancer
              </a>
            </td>
          <td>
            Ilseok Choi, Kyung-A Kim, Sang Cheol Kim, Donghwan Park, Ki Taek Nam, Junha Cha, S. Baek, Junha Cha, Hye-Yeong Jo, Minsun Jung, Melody Y. Zeng, Irina Matei, Susan Bullman, Joong Bae Ahn, Yoon Dae Han, Han Sang Kim, Insuk Lee
          </td>
          <td>2025-07-16</td>
          <td>Gut Microbes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Lung cancer remains the leading cause of cancer-related deaths globally. Emerging evidence indicates a potential link between blood cell perturbation responses and various diseases. Further investigation into the connection between these responses and lung cancer could provide valuable insights into its biological behavior and improve strategies for risk stratification. Methods This study employs two-sample mendelian randomization, incorporating lung cancer data from the IEU OpenGWAS project, as well as hematopoietic perturbation response data, to investigate the causal relationships between hematopoietic perturbation responses and lung cancer. This study also differentiated between subtypes: small cell lung carcinoma, adenocarcinoma, and squamous cell carcinoma. The primary analytical method was the inverse variance weighted (IVW) approach. Egger intercept analysis, abnormal MR-PRESSO test, Cochran Q, and leave one out analysis were also employed as multiple sensitivity analyses to assess the robustness of the results. Results 18 blood cell perturbation responses were significantly associated with lung cancer and its subtypes, including 10 protective factors and 8 risk factors. In addition, reverse Mendelian randomization analysis identified 12 blood cells with reverse causal relationships with cancer, comprising 11 inhibitory factors and 1 promoting factor. Conclusions Our findings demonstrate a potential causal relationship between lung cancer and blood cell perturbation responses, providing a new perspective for diagnosing and treating lung cancer. However, further studies are needed to elucidate the underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be6e652601f3c7d95cff2da1933b80ecaee483ff" target='_blank'>
              Unraveling the role of blood cell perturbation responses in lung cancer by Mendelian randomization
              </a>
            </td>
          <td>
            Yichao Huang, Xinjing Lou, Ziqing Han, Linyu Wu, Chen Gao
          </td>
          <td>2025-06-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Natural killer (NK) cells and T cells play crucial roles in liver metastatic cancer, particularly through their cytotoxic effects on tumor cells. Although existing evidence suggests a functional network of mutual regulation between NK and T cells, the nature of their interaction and its role in liver metastasis remain elusive. In this study, we analyzed single-cell transcriptomics of liver metastases and adjacent tissues from nasopharyngeal carcinoma (NPC), thyroid carcinoma (THCA), breast cancer (BC), colorectal cancer (CRC) and cervical cancer (CESC) to uncover the NK-T cell interaction network and its functional implications. In adjacent tissues, we observed increased infiltration of CD8+ NKT-like cells, γδT cells, and NK cells. The interaction between CD8+ NKT-like cells, CD8+ T cells, γδT cells, and NK cells were enhanced, with stronger signals associated with T and NK cell functions. Notably, CD8+ NKT-like cells, CD8+ T cells, and γδT cells exhibited an increased capacity to activate NK cells. These T cell subsets promoted NK cell anti-tumor activity via CD48-CD244 and TNF-TNFR signaling pathways, which in turn activated the ERK, JNK, and MAPK cascades. Our findings provide valuable insights into the NK-T cell interaction network in liver metastatic cancer, highlighting its potential as a therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-06241-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d630bd4e5bebe82c7ea2d9533a6c99b8e57bfc4" target='_blank'>
              Single-cell transcriptomics analysis reveals a disrupted NK-T cell interaction network in liver metastatic cancer
              </a>
            </td>
          <td>
            Xiaoshuang Wang, Zhongen Wu, Yan Zhou, Dehua Yang, Qingtong Zhou, Di Zhu, Mingwei Wang, Lu Wang
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Objective: Increasing evidence suggests that the tumor microenvironment (TME) of patients with triple negative breast cancer (TNBC) is hallmarked by extensive inter and intra-tumor heterogeneity. Deeper insights into the biological basis of this heterogeneity are needed to elucidate the aggressive nature of TNBC. Early attempts to stratify TNBC patients using bulk or single-cell transcriptomics have shown promise to understand differences in clinical outcomes and provide a personalized medicine approach. However, most examples limit themselves to a single 2D tissue slice per patient. Unfortunately, such an approach represents only a fraction of a 3D core needle biopsy, which is the only available tissue source to examine the treatment naïve TNBC microenvironment. Recently, spatial omics technologies demonstrated superior results for determining heterogeneity as these tools simultaneously profile the intact TME architecture and spatially resolved transcriptomes. Constructing 3D datasets by sectioning tissue and profiling each slice (n > 40) is feasible, but cost-prohibitive and impractical. An innovative digitization workflow that infers transcript localization across 3D datasets without probing each tissue slice would reduce costs and maximize tissue usage, whilst subsequently enhancing our understanding of tissue composition in the treatment naïve TNBC biopsy. Our goal is to facilitate building a comprehensive and large-scale map depicting TNBC heterogeneity from difficult-to-access tissue sources that could shed the much-needed light on TNBC. Method: We sectioned and imaged a treatment-naive core needle biopsy of TNBC into 48 serial slices, with 8 for spatial transcriptomics (Visium, 10X Genomics) equally spaced and 40 for H&E staining. Image coloration was normalized in a semi-automatic manner, and sections were registered on the associated H&E images using VALIS to create a 3D representation of the tissue. Transcripts from spatial transcriptomic sections were enhanced at super-resolution and mapped across serial sections using iSTAR. 3D spatial transcriptomic data analysis and quantitative comparisons with in-silico 2D slices was performed using the Giotto Suite ecosystem. Results: We generated a fully digital and super-enhanced (20 µm pixels) 3D spatial transcriptomics dataset from a single core needle TNBC biopsy by integrating image registration with spatial transcriptomics. Transcripts were projected across tissue sections using spatial interpolation and deep learning methods that learn the spatial and histology relationships for genes, thereby providing a comprehensive and super-resolved 3D representation. We observed that specific 3D structures, such as stromal plasma infiltrates and tertiary lymphoid structures, can be more accurately identified from the full 3D digital biopsy. We also illustrate how spatial gene expression patterns, key signaling pathways, and cellular architecture are organized in three dimensions, which cannot be easily inferred from 2D datasets alone. Importantly, systematic analysis comparing 3D versus 2D in-silico generated slices provides a quantitative framework that can be used to determine the limitations and power of spatial transcriptomic data analysis in 2D. Conclusion: This study demonstrates the feasibility of incorporating spatial transcriptomics and 3D registration methods into standard spatial analyses, enabling a detailed understanding of the spatial organization within a core needle TNBC biopsy and leading to informed insights on tissue architecture and function. Our approach facilitates a data-driven approach to better understand the strengths and limitations of profiling tumor heterogeneity in core needle biopsies of breast cancer. Finally, our approach enables researchers to make more informed research designs for large-scale projects, maximizing the usage of precious and often difficult-to-obtain patient samples.
 Citation Format: Jeffrey Sheridan, Jaiji George Chen, Junxiang Xu, Christina Ennis, Emma Kelley, Neal Kewalramani, Wonyl Choi, Jason Weiss, Gerald Denis, Naomi Ko, Ruben Dries. Accurate and cost-effective reconstruction of TNBC heterogeneity through 3D digitization of core needle biopsies [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-11-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f6a0b0ec31bb881078f32de3e375f20a2b2a55c" target='_blank'>
              Abstract P2-11-05: Accurate and cost-effective reconstruction of TNBC heterogeneity through 3D digitization of core needle biopsies
              </a>
            </td>
          <td>
            Jeffrey P. Sheridan, Jaiji George Chen, Junxiang Xu, Christina Ennis, Emma Kelley, Neal Kewalramani, Wonyl Choi, Jason Weiss, Gerald Denis, Naomi Ko, Ruben Dries
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Cell type-specific transcriptional heterogeneity in embryonic mouse skin is well-documented, but few studies have investigated the regulatory mechanisms. Here, we present high-throughput single-cell chromatin accessibility and transcriptome sequencing (HT-scCAT-seq), a method that simultaneously profiles transcriptome and chromatin accessibility. We utilized HT-scCAT-seq to dissect the gene regulatory mechanism governing epidermal stratification, periderm terminal differentiation, and fibroblast specification. Results By linking chromatin accessibility to gene expression, we identify candidate cis-regulatory elements (cCREs) and their target genes which are crucial for dermal and epidermal development. We describe cells with similar gene expression profiles that exhibit distinct chromatin accessibility statuses during periderm terminal differentiation. Finally, we characterize the underlying lineage-determining transcription factors and demonstrate that ALX4 and RUNX2 are candidate transcription factors regulators of the dermal papilla lineage development through in silico perturbation analysis and CUT&Tag experiment. Conclusions Overall, HT-scCAT-seq represents a powerful tool for unraveling the spatiotemporal dynamics of gene regulation in single cells. Our results advance the understanding of embryonic skin development while providing a scalable framework for investigating regulatory mechanisms across diverse biological systems and disease contexts. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03652-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/747c145f394c82a8a3bbf273f6afa4e007611092" target='_blank'>
              Deciphering gene regulatory programs in mouse embryonic skin through single-cell multiomics analysis
              </a>
            </td>
          <td>
            Q. Deng, Pengfei Cai, Yingjie Luo, Zhongjin Zhang, Wen Ma, Zijie Huang, Xiaoya Chen, Shijie Hao, Weiguang Ma, Jiangshan Xu, Chunqing Wang, Mengnan Cheng, Xiumei Lin, Ru Zhou, Shanshan Duan, Junjie Chen, Ronghai Li, Xuyang Shi, Chang Liu, Liang Wu, Peng Gao, Jianting Li, Xiao Yang, Xiangdong Wang, Jun Xie, Longqi Liu, Yue Yuan, Chuanyu Liu
          </td>
          <td>2025-07-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Non-melanoma skin cancer (NMSC), comprising basal cell carcinoma (BCC) and cutaneous squamous cell carcinoma (cSCC), represents the most common type of cancer worldwide, particularly among Caucasians. While BCC is locally invasive with minimal metastatic potential, cSCC is a highly aggressive tumor with a significant potential for metastasis, particularly in elderly populations. Tumor development and progression and the metastasis of cSCC are influenced by a complex interplay between tumor cells and the tumor microenvironment. Recent research highlights the importance of various immune cell subsets, including T cells, tumor-associated macrophages (TAMs), and dendritic cells, in influencing tumor progression, immune evasion, and treatment resistance. This review outlines key regulatory mechanisms in the immune tumor microenvironment (TME) of cSCC and explores the role of cytokines, immune checkpoints, and stromal interactions. We further discuss the relevance of three-dimensional (3D) in vitro models such as spheroids, organoids, and tumor-on-chip systems as tools to mimic immune–tumor interactions with higher physiological relevance, such as macrophage activation and polarization against cSCC cells. Globally, 3D models offer new opportunities for immunotherapy screening and mechanistic studies. Understanding the immune landscape in cSCC through advanced modeling techniques holds strong clinical potential for improving diagnostic and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/423c3e60a81f05412f1191c356ba2acb90d2d9a5" target='_blank'>
              Exploring the Complexity of Cutaneous Squamous CellCarcinoma Microenvironment: Focus on Immune Cell Roles by Novel 3D In Vitro Models
              </a>
            </td>
          <td>
            M. Quadri, Marco Iuliano, Paolo Rosa, G. Mangino, Elisabetta Palazzo
          </td>
          <td>2025-07-23</td>
          <td>Life</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Background/Aim: Tongue squamous cell carcinoma (TSCC), a highly aggressive subtype of head and neck cancers, is characterized by frequent lymphatic metastasis and poor prognosis. Recently, we showed that lymphatic vessel endothelial hyaluronan receptor 1 (LYVE-1) is involved in TSCC progression, yet the underlying molecular mechanisms remain unclear. Materials and Methods: CRISPR/Cas9 gene editing was employed to generate LYVE1 knockout (KO) TSCC cell lines. Single-cell clones were isolated, screened, and validated through sequencing and Inference of CRISPR Edits (ICE) analysis and qRT-PCR. RNA sequencing was performed on LYVE1 KO and wild-type (WT) cells to identify differentially expressed genes (DEGs). Bioinformatic analyses, including Gene Ontology (GO) enrichment and protein-protein interaction (PPI) network mapping, were conducted to explore affected pathways. Finally, network topology was examined using NetworkAnalyzer and cytoHubba plugins. Results: Transcriptomic analysis revealed significant down-regulation of pro-metastatic pathways, including epithelial-mesenchymal transition (EMT), extracellular matrix remodeling, and immune modulation. DEG analysis identified 263 genes, with key down-regulated targets such as WNT5A, TGFB2, and MMP2, and up-regulation of tumor-suppressive genes including PTGS2. GO and PPI analyses highlighted LYVE1’s pivotal role in regulating cell adhesion, migration, and immune response. Conclusion: LYVE1 KO reduces TSCC invasive potential by disrupting EMT and tumor-stroma interactions, aligning with previous experimental findings. These results suggest LYVE1 as a critical driver of metastasis, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff3c633bd83b5225f79493e53059643bbd3c2790" target='_blank'>
              CRISPR/Cas9-mediated Knockout of LYVE1 In Human Tongue Cancer Cells Reveals Transcriptomic Changes in Metastasis-associated Pathways
              </a>
            </td>
          <td>
            Sini Karinen, Johanna FORERO-RODRÍGUEZ, Annika Järvinen, Kari K Eklund, Gonçalo Barreto, A. Salem
          </td>
          <td>2025-06-26</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) are essential components of the tumor microenvironment (TME), contributing to tumorigenesis, progression, and resistance to therapy. However, the functional diversity of CAF subpopulations and their role in tumor progression and patient prognosis remain poorly understood. This study aims to explore CAF heterogeneity and their functional roles in the TME using single-cell RNA sequencing (scRNA-seq) and multi-omics data analysis. Methods scRNA-seq data were analyzed to cluster CAF subpopulations in the TME, with key genes identified through functional annotation. Differentially expressed genes were analyzed, and prognostic genes were selected via Cox and LASSO regression. A risk score model (RiskScore) was developed for survival prediction and immune therapy sensitivity evaluation. Core CAF genes were examined using siRNA interference, qPCR, and Western blotting. Drug sensitivity was assessed to explore the clinical relevance of these genes. Results Four CAF subpopulations (CAF-0, CAF-1, CAF-2, CAF-3) were identified, revealing differences in key tumor-associated signaling pathways (e.g., MYC, WNT, TGF-β). Thirteen core genes related to prognosis were identified, and a RiskScore model was developed, showing significantly worse survival rates for high-risk patients (p < 0.001) and features of immune suppression, including increased M0 macrophage infiltration. Drug sensitivity analysis indicated that core genes (e.g., KLRB1, MAP1B) were linked to drug sensitivity, suggesting potential biomarkers for targeted therapy. Experimental validation showed that knockdown of the HIP1R gene significantly reduced tumor cell expression, confirming its critical role in tumor development. Conclusion This study offers a comprehensive analysis of CAF heterogeneity and its impact on TME, patient prognosis, and drug sensitivity. The developed RiskScore model provides theoretical support for personalized treatment based on CAF-related genes, offering new insights into CAF-driven tumor progression and potential targets for precision oncology and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5c1ce9d5dedbb8252961d9100d080e0187631a3" target='_blank'>
              Therapeutic implications of cancer-associated fibroblast heterogeneity: insights from single-cell and multi-omics analysis
              </a>
            </td>
          <td>
            Yumin Wang, Yi Ding, Haolin Liu, Zhongyou Xia, Guoqiang Liao, Shicheng Fan, JunXiong Li, JingBo Qin, PinYao Liang, Peng Gu, Xiaodong Liu, Runan Dong
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background: Several studies have demonstrated the occurrence of secondary tumors as a rare but significant complication of chimeric antigen receptor T (CAR-T) cell therapy, underscoring the need for a detailed investigation. Given the limited variety of secondary tumor types reported to date, a comprehensive characterization of the various secondary tumors arising after CAR-T therapy is essential to understand the associated risks and to define the role of the immune microenvironment in malignant transformation. This study aims to characterize the immune microenvironment of a newly identified secondary tumor post-CAR-T therapy, to clarify its pathogenesis and potential therapeutic targets. Methods: In this study, the bone marrow (BM) samples were collected by aspiration from the primary and secondary tumors before and after CD19 CAR-T treatment. The CD45+ BM cells were enriched with human CD45 microbeads. The CD45+ cells were then sent for 10× genomics single-cell RNA sequencing (scRNA-seq) to identify cell populations. The Cell Ranger pipeline and CellChat were used for detailed analysis. Results: In this study, a rare type of secondary chronic myelomonocytic leukemia (CMML) were reported in a patient with diffuse large B-cell lymphoma (DLBCL) who had previously received CD19 CAR-T therapy. The scRNA-seq analysis revealed increased inflammatory cytokines, chemokines, and an immunosuppressive state of monocytes/macrophages, which may impair cytotoxic activity in both T and natural killer (NK) cells in secondary CMML before treatment. In contrast, their cytotoxicity was restored in secondary CMML after treatment. Conclusions: This finding delineates a previously unrecognized type of secondary tumor, CMML, after CAR-T therapy and provide a framework for defining the immune microenvironment of secondary tumor occurrence after CAR-T therapy. In addition, the results provide a rationale for targeting macrophages to improve treatment strategies for CMML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f62036ee9eb9eb457e3406f8e167f834148a9998" target='_blank'>
              Decoding the immune microenvironment of secondary chronic myelomonocytic leukemia due to diffuse large B-cell lymphoma with CD19 CAR-T failure by single-cell RNA-sequencing
              </a>
            </td>
          <td>
            Xudong Li, Hong Huang, Fang Wang, Mengjia Li, Binglei Zhang, Jianxiang Shi, Yuke Liu, Mengya Gao, Mingxia Sun, Haixia Cao, Danfeng Zhang, Na Shen, Weijie Cao, Zhilei Bian, Haizhou Xing, Wei Li, Linping Xu, Shiyu Zuo, Yongping Song
          </td>
          <td>2025-07-07</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Temozolomide (TMZ) is an important chemotherapeutic agent for glioma treatment. However, the emergence of drug resistance impedes its application. Traditional population-level studies are limited in elucidating resistance mechanisms. Advances in single-cell and spatial transcriptomics technologies provide feasible resolution for studying the cellular composition and dynamics of tumors. In this study, we investigated the heterogeneity of gliomas associated with TMZ resistance at the single-cell and spatial transcriptome levels to identify the resistance mechanisms and potential therapeutic strategies. Methods Single cell sequencing technology was utilized to identify the cellular clusters of gliomas. Drug perturbation analysis and cellular propensity analysis revealed key cluster responding to TMZ. Enrichment analysis was preformed to explore the function of clusters. Transcription factor activity analysis revealed key transcription factors contributing to tumor resistance. Spatial transcriptome data and bulk RNA-seq data validates the role of key transcription factors. Downstream targets of key transcription factors were predicted and validated using gene regulation assays. Drug sensitivity analyses were used to seek viable strategies to overcome drug resistance. Results Glioma cells from before and after temozolomide treatment samples were classified into six clusters: NPC-like cluster, OPC-like cluster, MES-like cluster, AC-like cluster, OC, and Neuron. NPC-like clusters exhibited strong stemness and DNA repair capacity. The activity of MAZ in NPC-like cluster was significantly enhanced after TMZ treatment. The proportion of MAZ( +)_NPC-like cluster was higher in TMZ treated samples. Patients with high proportion of MAZ( +)_NPC-like cluster had poorer survival. Upregulation of MAZ is able to enhance drug resistance in glioma cells, but this phenomenon disappeared when FoxM1 expression was further silenced. The combination of paclitaxel and Trametinib is a promising strategy to overcome resistance. Conclusions NPC-like cluster is prevalent in recurrent and drug-resistant gliomas. MAZ transcription factors are critical regulators that promote the development of drug resistance in NPC-like clusters by enhancing the capacity of DNA repair and stemness. Patients with high proportions of MAZ( +)_NPC-like clusters have poor TMZ sensitivity and prognosis. MAZ enhances stemness and drug resistance in glioma cells by upregulating FOXM1 expression. The combination of paclitaxel and paclitaxel is a promising therapeutic strategy for treating gliomas and overcoming drug resistance. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06706-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c31c53340b2d02203f651d970425a0fc3e3628" target='_blank'>
              Single-cell and spatial transcriptome analyses reveal MAZ(+) NPC-like clusters as key role contributing to glioma recurrence and drug resistance
              </a>
            </td>
          <td>
            Qiang Gu, Kan Wang, Tingming Lu, Yifei Xiao, Yaqing Wu, Hang Zhou, Kaijia Zhou
          </td>
          <td>2025-06-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Lung cancer stands as one of the most prevalent malignancies globally, with its high morbidity and mortality intimately associated with its inconspicuous early symptoms, the advanced stages at diagnosis, and resistance to conventional therapeutic interventions. Exosomes, serving as pivotal mediators of intercellular communication, carry biomolecules that play crucial roles in tumorigenesis, progression, and metastasis, holding promise as novel targets for early diagnosis, prognostic evaluation, and treatment of lung cancer. Methods In this study, lung cancer-related datasets were obtained from the GEO database and TCGA. Through differential gene analysis, enrichment analysis, immune infiltration analysis, and drug regulatory analysis, exosome-associated genes pertinent to lung cancer were screened and identified. Results The research revealed significant downregulation of CRYAB, CAV1, HYAL1, and TUBB6 genes in lung cancer tissues, whereas SERINC2, PAICS, SLC2A1, and BIRC5 genes were markedly upregulated. These genes were predominantly enriched in biological processes such as cell migration, oxidative stress response, and cell cycle regulation, as well as in KEGG pathways like the IL-17 signaling pathway. Immune infiltration analysis demonstrated a high correlation between these genes and the infiltration levels of various immune cells. Furthermore, through drug-gene enrichment analysis and molecular docking experiments, significant correlations were found between drugs such as celecoxib and some exosome-related genes, with interaction targets existing between these drugs and CAV1, SLC2A1, and BIRC5 genes. Conclusion This study unveils the expression characteristics and biological significance of exosome-associated genes in lung cancer. The differential expression of these genes not only offers potential biomarkers for early diagnosis of lung cancer but also lays a foundation for further research into their biological functions in this disease. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03173-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34b7ac84cc89c2118e1e8ef700dcc6c2b9b5300f" target='_blank'>
              Expression characteristics and biological significance of exosome-related genes in lung cancer
              </a>
            </td>
          <td>
            Qixiang Zhong, Yujie Zhao
          </td>
          <td>2025-07-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Cancer is a leading cause of death worldwide, and researchers are continuously working to uncover its drivers and new therapeutic targets. In this review, we explain the role of the oncogenic protein CREPT that is overexpressed in the vast majority of cancers, leading to poor overall survival. CREPT ablation has shown sustained tumor regression in vitro and in mouse models. CREPT activation mediates cancer progression by enhancing the cell cycle by promoting chromatin loop formation. Thus, selectively targeting CREPT expression may provide alternative strategies for inhibiting abnormal cell division, metastasis, and tumor growth, and may improve the efficacy of the currently available therapies in combination, and will be key to reversing cancerous growth in patients with CREPT-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c17bae866b8a91dd27eb2fc5ec68639a8ba51279" target='_blank'>
              Disrupting Cell Cycle Machinery: CREPT Is an Emerging Target in Cancer Therapy
              </a>
            </td>
          <td>
            Umar Farooq, Jun Li, Zhijie Chang
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The advent of single-cell sequencing has revolutionized the study of cellular dynamics, providing unprecedented resolution into the molecular states and heterogeneity of individual cells. However, the rich potential of exon-level information and junction reads within single cells remains underutilized. Conventional gene-count methods overlook critical exon and junction data, limiting the quality of cell representation and downstream analyses such as subpopulation identification and alternative splicing detection. We introduce DOLPHIN, a deep learning method that integrates exon-level and junction read data, representing genes as graph structures. These graphs are processed by a variational graph autoencoder to improve cell embeddings. DOLPHIN not only demonstrates superior performance in cell clustering, biomarker discovery, and alternative splicing detection but also provides a distinct capability to detect subtle transcriptomic differences at the exon level that are often masked in gene-level analyses. By examining cellular dynamics with enhanced resolution, DOLPHIN provides new insights into disease mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1272bec04f01bf689ca7892996faa12e4060673f" target='_blank'>
              DOLPHIN advances single-cell transcriptomics beyond gene level by leveraging exon and junction reads
              </a>
            </td>
          <td>
            Kailu Song, Yumin Zheng, Bowen Zhao, D.H. Eidelman, Jian Tang, Jun Ding
          </td>
          <td>2025-07-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Many patients harbor minimal residual disease (MRD)—small clusters of residual tumor cells that survive therapy and evade conventional detection but drive recurrence. Although advances in molecular and computational methods have improved circulating tumor DNA (ctDNA)-based MRD detection, these approaches face challenges: ctDNA shedding fluctuates widely across tumor types, disease stages, and histological features. Additionally, low levels of driver mutations originating from healthy tissues can create background noise, complicating the accurate identification of bona fide tumor-specific signals. These limitations underscore the need for refined technologies to further enhance MRD detection beyond DNA sequences in solid malignancies. Methods: Profiling circulating cell-free mRNA (cfmRNA), which is hyperactive in tumor and non-tumor microenvironments, could address these limitations to inform postoperative surveillance and treatment strategies. This study reported the development of OncoMRD BREAST, a customized, gene signature-informed cfmRNA assay for residual disease monitoring in breast cancer. OncoMRD BREAST introduces several advanced technologies that distinguish it from the existing ctDNA-MRD tests. It builds on the patient-derived gene signature for capturing tumor activities while introducing significant upgrades to its liquid biopsy transcriptomic profiling, digital scoring systems, and tracking capabilities. Results: The OncoMRD BREAST test processes inputs from multiple cutting-edge biomarkers—tumor and non-tumor microenvironment—to provide enhanced awareness of tumor activities in real time. By fusing data from these diverse intra- and inter-cellular networks, OncoMRD BREAST significantly improves the sensitivity and reliability of MRD detection and prognosis analysis, even under challenging and complex conditions. In a proof-of-concept real-world pilot trial, OncoMRD BREAST’s rapid quantification of potential tumor activity helped reduce the risk of incorrect treatment strategies, while advanced predictive analytics contributed to the overall benefits and improved outcomes of patients. Conclusions: By tailoring the assay to individual tumor profiles, we aimed to enhance early identification of residual disease and optimize therapeutic decision-making. OncoMRD BREAST is the world’s first and only gene signature-powered test for monitoring residual disease in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86278e0a31f84ae70e8ec42ddfdd96b706c9c797" target='_blank'>
              Revolutionizing Detection of Minimal Residual Disease in Breast Cancer Using Patient-Derived Gene Signature
              </a>
            </td>
          <td>
            Chen Yeh, Hung-Chih Lai, Nathan Grabbe, Xavier Willett, Shu-Ti Lin
          </td>
          <td>2025-07-12</td>
          <td>Onco</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Phosphodiesterase 10A (PDE10A), located on cytoband 6q27, acts as a haploinsufficient tumor suppressor in glioblastoma, where its loss drives aggressive tumor behavior and is associated with a proneural-to-mesenchymal (PN-MES) shift. Leveraging bioinformatic analyses of The Cancer Genome Atlas (TCGA) pan-cancer dataset, we identified gastrointestinal carcinomas, specifically stomach adenocarcinoma and colon adenocarcinoma, as cancers where 6q27/PDE10A loss is prognostic. In both stomach adenocarcinoma and colon adenocarcinoma, PDE10A loss was associated with reduced overall survival (OS) and progression-free survival (PFS). Additionally, PDE10A loss correlated with increased lymphatic invasion and higher AJCC pathologic nodal stage in colon adenocarcinoma. In an independent institutional cohort, immunohistochemical analysis showed lower PDE10A expression in metastatic colon adenocarcinomas compared to primary tumors. Furthermore, PDE10A loss and decreased protein expression corresponded with an epithelial-to-mesenchymal transition (EMT) phenotype in both cohorts, consistent with the known PN-MES shift observed in glioblastoma. In summary, PDE10A loss in colon adenocarcinoma is associated with decreased OS and PFS, more frequent lymphatic invasion, higher pathologic nodal stage, metastatic disease, and an EMT phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bf65f5c71154818abc6f02f53ec10e9e90dbcc7" target='_blank'>
              Loss of PDE10A is associated with metastasis of colon adenocarcinoma.
              </a>
            </td>
          <td>
            Sumaita Arowa, Nicholas Nuechterlein, Vinny Ha, Allison Shelbourn, Ashis Chowdhury, Y. Banasavadi-Siddegowda, J. Lasota, Markku Miettinen, Patrick J Cimino
          </td>
          <td>2025-06-16</td>
          <td>Histology and histopathology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fdc708d82395eddf5ef9f62e5ba5f1c540963b1" target='_blank'>
              Single-cell and bulk transcriptome analysis identifies B-cell subpopulations and associated cancer subtypes with distinct clinical and molecular characteristics.
              </a>
            </td>
          <td>
            Yin A He, Li Zhao, Yufen Zheng, Xiaosheng Wang
          </td>
          <td>2025-06-17</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae991021621725bf1b34e349540819c402a22de9" target='_blank'>
              A Ligand-Centered Framework for γδ T Cell Activation in Colorectal Cancer Revealed by Single-Cell and Transformer-Based Perturbation
              </a>
            </td>
          <td>
            Ran Ran, Douglas K. Brubaker
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a subset of cells within tumors that exhibit stem cell-like characteristics, including the ability to self-renew and differentiate. CSCs are the cause of carcinogenesis and tumorigenesis. The expression of cell surface markers, which varies linked to the kind of tumor, is utilized to recognize CSCs. An essential part of tumor invasion and metastasis is played by CSCs. Numerous investigations have been carried out to find distinguished markers and different phenotypes of CSCs, which are especially crucial for identifying and separating this subset of cells. It was discovered that the regulation of CSCs involves a multitude of signaling pathways. These cells are determined by their ability to self-renewal pathways such as Wnt/β-catenin, JAK/STAT3, PTEN/PI3-K/Akt, and Hedgehog, their surface biomarkers, and their resistance to many drugs. Aberrant activation of these signaling pathways is associated with cell growth. Thus, focusing on CSCs is seen to be a viable anti-cancer treatment approach. It is encouraging that CSCs' self-renewal pathways present a viable target for changing their survival tactics and limiting their capacity to proliferate tumors. This study highlights the characterization and investigation of CSC self-renewal pathways, also discusses potential targeted therapy for CSC, and gives a summary of the significant factors and pathways that adjust CSC formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/441bf7be356d1c2d3ba019982154a866827815ef" target='_blank'>
              Navigating the Molecular Signaling: Deciphering Cancer Stem Cell Self-Renewal Pathways
              </a>
            </td>
          <td>
            Seyed Nasser Hoseinian, Mohammad Saeedi, Mohammad Erfan Saravani, Sepideh Zenoozi, F. Mehranfar, Amin Pouyan
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular and Cellular Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor adaptability relies on the ability of cancer cells to dedifferentiate and acquire stem-like features, fueling therapeutic resistance and metastasis. Differentiation therapy aims to reprogram tumor cells into more mature, less aggressive states to counteract this plasticity. Here, we identify cannabinoid receptor 2 (CB2R) as a novel therapeutic target that promotes sustained differentiation in breast cancer. Using tumor-derived organoids from both mouse models and patient biopsies, we show that brief, low-dose exposure to phytogenic or synthetic CB2R ligands induces a basal-to-luminal switch, suppresses stemness, and reduces invasiveness and self-renewal. These phenotypic changes are associated with decreased tumor initiation and aggressiveness in vivo . Transcriptomic profiling reveals that CB2R activation initiates transient chromatin remodeling and epigenetic reprogramming, resulting in a stably differentiated state. Importantly, CB2R-driven differentiation sensitizes tumor cells to tamoxifen, enabling lower therapeutic doses with improved efficacy-supporting the principles of adaptive therapy aimed at long-term disease control. Our findings position CB2R modulation as a promising non-cytotoxic strategy to restrict cancer plasticity and enhance the effectiveness of existing breast cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff04e163a7aa41f773b08a8fee437665a57ee718" target='_blank'>
              CB2R-induced differentiation epigenetically restrains cancer plasticity enabling adaptive therapy.
              </a>
            </td>
          <td>
            Nuria G. Martínez-Illescas, Gemma Pérez-García, María Rubert-Hernández, Camila Quezada-Gutiérrez, Estrella Martín-Zapater, Luis M Alonso-Colmenar, Carmen Hernández, Ángela Zarco-Cuadrillero, Eduardo Caleiras, Miguel Muñoz-Ruiz, Ana Payo-Payo, Laura Frías-Aldeguer, Cristina Sánchez, Mercedes Balcells-Camps, E R Edelman, María Salazar-Roa
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
Ferroptosis has been implicated in the regulation of the tumor immune environment; however, its precise effect on immune checkpoint inhibitors remains contradictory.


OBJECTIVE
To elucidate the "double-edged sword effect" of a key ferroptosis-related factor in regulating the immune microenvironment.


METHODS
This study utilized single-cell RNA sequencing (scRNA-seq) analysis to characterize the tumor microenvironment in ovarian cancer samples from immunotherapy cohorts. Following quality control and variable gene screening, data from The Cancer Genome Atlas (TCGA), Genotype-Tissue Expression (GTEx), GENE EXPRESSION OMNIBUS (GEO), bulk, and spatial transcriptome databases were analyzed. The AddModuleScore_UCell function was employed for gene set scoring by evaluating the expression patterns of specific gene features in single-cell datasets, which were found to correlate with interactions between tumor cells and stromal cells, recognized as key contributors in the immunosuppressive milieu. Immunohistochemistry, western blot, and multiplex immunohistochemistry (mIHC) analyses were employed to explore the HMOX1/TGF-β1/PI3K/AKT/NF-κB(p65) signaling pathways. In vitro findings were further validated in a mouse model. The correlation between risk factors and progression-free survival (PFS) was analyzed using Cox regression and Kaplan-Meier methods.


RESULTS
We demonstrated decreased expression of the ferroptosis-activating gene HMOX1 in ovarian cancer epithelial cells, while being upregulated in macrophages. Ovarian cancer (OV) epithelial cells with HMOX1 inhibition could secrete TGF-β1 to activate three macrophage subtypes: SPP1+, FOLR2+ and C1QC+ via the PI3K/AKT/NF-κB (p65) pathway. The up-regulation of HMOX1 in macrophages also activated these three macrophage subtypes via the NF-κB pathway. Both pathways simultaneously inhibited Cytotoxic T Lymphocyte (CTL) activation and contributed to the immunosuppressive microenvironment of ovarian cancer, as demonstrated in both in vitro and in vivo models. Targeting HMOX1 alone, whether through activation or inhibition, was only effective in modulating a single pathway while simultaneously inducing negative feedback on the opposing pathway, demonstrating the "double-edged sword effect" of HMOX1 in regulating the immune microenvironment.


CONCLUSION
Overall, we proposed and validated two strategies targeting HMOX1 to improve the efficacy of PD-1 inhibitors, and confirmed that HMOX1, TGF-β1, SPP1, FOLR2, and C1QC could be used to construct models predicting the efficacy of immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7f01291927dba9ec25f0b861c5f37f0048388aa" target='_blank'>
              Distinct roles of HMOX1 on tumor epithelium and macrophage for regulation of immune microenvironment in ovarian cancer.
              </a>
            </td>
          <td>
            Yi Liu, Li-Jun Jiang, Hong-Fang Liu, Li Chen, Lei Guo, Jun Ge, Xin-Yi Zhang, Jing Li, Wei Gong
          </td>
          <td>2025-07-09</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Pancreatic cancer stem cells (CSCs) are known culprits of recurrent pancreatic ductal adenocarcinoma (PDAC). The disputable presence of CSCs in continuous cell lines has led to an extensive application of patient-derived CSC culture for experimentation. However, obtaining patient biopsies remains a challenge for many laboratories. Here, this study aimed to investigate the presence of CSCs in KRAS-driven PDAC cell lines.



 Cell viability assays evaluated the cytotoxicity of gemcitabine and vismodegib in PDAC cell lines. Flow cytometry was used to analyse the expression of pancreatic CSC surface markers. Tumoursphere-forming ability was investigated by culturing cells in serum-free and non-adherent conditions in vitro, while the tumourigenicity of PDAC cells was assessed in immunocompromised mice. Molecular events were delineated by Western blotting, and SOX2-expressing cells in tumour xenografts were detected by immunohistochemistry.



 PANC-1 and Capan-2 cell lines showed exceptional chemoresistance, with Capan-2 containing a notable number of putative CSCs that co-expressed CD44, CD24, and CD133 (3.7 % of the total population) and sustained the cytotoxicity of gemcitabine. The Capan-2 cell line harbours tumoursphere-forming cells, of which self-renewal was inhibited by foetal bovine serum. The presence of these cells is consistent with Capan-2 being highly tumourigenic in immunocompromised mice. MAPK, PI3K-AKT, NF-κB, and WNT pathways were activated, and SOX2 expression was upregulated during the self-renewal of putative CSCs. SOX2-expressing cells were correspondingly present in xenografts of highly tumourigenic Capan-2 cell lines.



 The Capan-2 cell line contains a subset of putative CSCs, whose self-renewal is driven by MAPK, PI3K-AKT, NF-κB, and WNT pathways.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/436e454e3e012d3c6079c5e033e5730ae1927ad5" target='_blank'>
              KRAS-driven pancreatic ductal adenocarcinoma cell lines harbour putative cancer stem cells
              </a>
            </td>
          <td>
            Yuan Han Teh, Rui Jing, Rajesh Ramasamy, Kok Lian Ho, S. Sagineedu, Johnson Stanslas
          </td>
          <td>2025-07-30</td>
          <td>Oncologie</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Existing approaches to identifying cancer genes rely overwhelmingly on DNA sequencing data. Here, we introduce RVdriver, a computational tool that leverages paired bulk genomic and transcriptomic data to classify RNA variant allele frequencies (VAFs) of non-synonymous mutations relative to a synonymous mutation background. We analyze 7882 paired exomes and transcriptomes from 31 cancer types and identify novel, as well as known, cancer genes, complementing other DNA-based approaches. Furthermore, RNA VAFs of individual mutations are able to distinguish “driver” from “passenger” mutations within established cancer genes. This approach highlights the value of multi-omic approaches for cancer gene discovery. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03557-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fff6bf27033e3de68705c3b36b17d1f26265844a" target='_blank'>
              Cancer gene identification from RNA variant allelic frequencies using RVdriver
              </a>
            </td>
          <td>
            James R. M. Black, Thomas P. Jones, Carlos Martínez-Ruiz, Maria Litovchenko, C. Puttick, C. Swanton, N. Mcgranahan
          </td>
          <td>2025-06-13</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="INTRODUCTION
. The ability of cancer cells to disseminate from the primary tumor and form metastatic lesions frequently leads to fatal outcomes. Recently, however, it has been recognized that this process is driven by complex interactions between the cancer and the neighboring cells, and, overall, made possible by a supportive tumor microenvironment. The emergence of high-throughput technologies is expected to bring much needed clarity to unraveling the players and intricate communication pathways that promote metastatic progression.


AREAS COVERED
In this report, the impact of mass spectrometry and proteomic technologies on deciphering the cross-talk between cancer and tumor microenvironment cells is discussed. Focus is placed on the role of cell-membrane and secretome proteins as the main enablers of this cross-talk, and on the challenges presented by metastatic tumors that evolve in the brain. Future prospects are assessed in the context of recent biology, technology, and data analysis breakthroughs.


EXPERT OPINION
Advancements in high-throughput proteomic technologies, complemented by the exciting potential of new disease model systems and data processing abilities of artificial intelligence, are expected to bring groundbreaking progress in deciphering the fundamental biological mechanisms that support cancer behavior and metastatic development, revealing novel therapeutic targets, and guiding innovative intervention approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad606f697b154e1d0963d77bb80a6ca95fd8151f" target='_blank'>
              Proteomic technologies for profiling cell-membrane/secretome interactions in brain metastatic cancer progression.
              </a>
            </td>
          <td>
            I. Lazar
          </td>
          <td>2025-07-17</td>
          <td>Expert review of proteomics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Ovarian cancer (OV) is the deadliest gynecologic malignancy owing to its late diagnosis and high metastatic propensity. Current biomarkers lack sufficient sensitivity and specificity for the detection of early-stage cancer. To address this gap, we integrated single‐cell transcriptomic profiling of tumor tissues with analysis of circulating exosomal RNA, aiming to uncover candidate markers that reflect tumor heterogeneity and metastatic potential and that may serve as sensitive, non‐invasive diagnostics. Methods We integrated single-cell RNA sequencing (scRNA-seq) data from primary tumors and metastatic lesions with bulk tissue transcriptomes and plasma-derived exosomal RNA sequencing (RNA-seq). Differentially expressed genes (DEGs) shared across tumor cells, metastatic subpopulations, and exosomes were identified through intersection analysis. Candidate genes were validated in clinical specimens using qPCR and immunohistochemistry. We then applied ten machine learning algorithm to exosomal transcriptomic data to evaluate diagnostic performance and identify the optimal classifier. Tumor cell differentiation states were evaluated using CytoTRACE, and intercellular communication was analyzed with CellChat. Results Intersection analysis highlighted 52 overlapping DEGs, of which SCNN1A and EFNA1 emerged as the top prognostic indicators. Both genes were significantly upregulated in tumor tissues, metastatic foci, and plasma exosomes (P < 0.01). The exosome-based Adaboost model had an area under the curve of 0.955 in an independent test cohort. Single-cell subcluster analyses revealed high SCNN1A/EFNA1 expression correlated with stem-like differentiation states and enriched pathways associated with immune evasion and adhesion. CellChat analysis demonstrated that highly differentiated tumor cells extensively engaged with fibroblasts and endothelial cells, implying their role in niche formation. Conclusions By coupling single-cell, bulk tissue, and exosomal transcriptomics, we elucidated the key molecular drivers of OV metastasis and established SCNN1A and EFNA1 as promising non-invasive biomarkers. This multi-omics framework provides an effective strategy for early detection and a better understanding of metastatic progression in OV.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0c71607fd05b63fb6eb088e5cf81be9d264179" target='_blank'>
              Integrative single-cell and exosomal multi-omics uncovers SCNN1A and EFNA1 as non-invasive biomarkers and drivers of ovarian cancer metastasis
              </a>
            </td>
          <td>
            Liping Tang, Dong Pang, Chengbang Wang, Jiali Lin, Shaohua Chen, Jiangchun Wu, Junqi Cui
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tumor microenvironment (TME) plays an important role in tumorigenesis, development, metastasis and drug sensitivity, but little is known about it in lung squamous cell carcinoma (LUSC). Here, the RNA-sequencing data, clinical and survival data of patients with LUSC in The Cancer Genome Atlas and six independent datasets were collected. Based on the unsupervised clustering of knowledge-based functional gene expression signatures, LUSC was classified into four subtypes. Cluster1 and cluster3 exhibited substantial tumor immune infiltration, suggesting a better response to immunotherapy. Relatively worse survival was observed in cluster4, probably due to higher angiogenesis. Besides, differentially expressed genes in cluster1, cluster2 and cluster3 were prominently enriched in immune-related pathways, while extracellular matrix-related pathways were enriched for cluster4. Genomic data analyses showed significant variations in tumor mutational burden and mutational frequency of several genes, such as TP53, among the four subtypes. Additionally, the four subtypes exhibited heterogeneity in the sensitivity of commonly used chemotherapy drugs for lung cancer and the intratumor microbiome profile. Finally, a prognostic model was developed and its performance and generalization ability were independently validated in multiple datasets. Overall, our study advances the understanding of the TME in LUSC and proposes a prognostic model that facilitates clinical decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad5b9219c70b502bb25fefa3777b3f5d8d780eb" target='_blank'>
              Multi-dimensional characterization of the tumor microenvironment profiles in lung squamous cell carcinoma.
              </a>
            </td>
          <td>
            Yi Zhou, Lixun Chai, Yuyao Wang, Hongguang Zhang
          </td>
          <td>2025-06-25</td>
          <td>Physiological genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A comprehensive pan-cancer analysis of HES6 was conducted to investigate the differential expression patterns, clinical implications, and prognostic significance of HES6 across multiple cancer types, while also aiming to elucidate the underlying biological mechanisms of HES6 in oncogenesis. HES6 plays a pivotal regulatory role in the progression of numerous malignant tumors, profoundly influencing the survival duration of patients with solid tumors and demonstrating a strong correlation with the immune microenvironment. Notably, HES6 is generally overexpressed in a wide range of cancer types, exhibiting significant divergence from its expression levels in normal tissues. Furthermore, HES6 has been found to serve as a predictive indicator of survival outcomes in patients with various solid tumors. The gene expression profile of HES6 and its associated genes is predominantly enriched in CD8 + T cells and macrophages. Additionally, HES6 and its cognate genes are significantly enriched within critical signaling cascades, including the “Epithelial-Mesenchymal Transition Pathway,” “NOTCH Signaling,” and the “Wnt signaling activation,” among others. HES6 may exert its influence on cellular function and oncogenic behaviors through interactions with HES1 and other proteins. It also demonstrates a robust association with a multitude of inflammation-related genes, immunomodulatory genes, and receptor genes across diverse cancer types. Based on these findings, it is hypothesized that HES6 may emerge as a potential clinical biomarker for solid tumors, particularly gliomas. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02852-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/115e3734b88452f7e416ba93c5a59ce8c5bc1a74" target='_blank'>
              Pan-cancer analysis reveals dual roles of HES6 in cancer immunity and prognosis
              </a>
            </td>
          <td>
            Ying Sun, Peiru Liu, Dongyang Lv, Defu Yang, Shiyu Qiao, Jing Liu, Ying Yan, Wenping Wang
          </td>
          <td>2025-06-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Trastuzumab deruxtecan (T-DXd) is the standard of care for HER2+ or HER2 low metastatic breast cancer (mBC). However, approximately 20% of patients exhibit resistance to T-DXd therapy. As the development of more antibody-drug conjugate (ADC) therapies progresses, understanding ADC resistance mechanisms is critical. Our study aims to elucidate the etiology of ADC resistance, such as characterizing molecular changes and identifying novel biomarkers in tumor-immune interactions between T-DXd responsive (R) and non-responsive (NR) patients with mBC. We employed Nanostring’s GeoMX spatial proteomics and transcriptomics profiling technology to analyze 29 metastatic tissue sections (15R, 14NR) from heavily pre-treated patients with HER2+/low mBC, prior to T-DXd treatment. For each section, 6-8 circular regions of interest (ROIs) with a diameter of 300µm were chosen by researchers blinded to T-DXd R vs. NR outcomes and HER2 scores. ROIs were selected to capture spatially diverse tumor-dense and tumor-sparse regions. Immune markers, stromal markers, and HER2 expression levels were quantified for each ROI. Consistent immune and stromal features related to T-DXd resistance were identified across various metastatic sites, including brain, bone, liver, lymph node, chest wall, and lung. NR tumors were characterized by relative immunosuppression and a disorganized immune response, with consistent upregulation of fibronectin and, surprisingly, granzyme B. Local immune response significantly varied based on HER2 expression. Organ-specific characteristics were also observed: Bone metastases showed an increased correlation between various immune markers and fibronectin in NR, suggesting that fibronectin may modulate immune infiltration and promote a supportive microenvironment for tumor growth. In contrast, chest wall metastases showed loss of correlation between various immune markers and smooth muscle actin in NR. This suggests that a remodeled, less dense smooth muscle environment might reduce physical barriers, potentially allowing better immune cell infiltration and activity in responsive tumors. Furthermore, a subset of these patient samples was selected for spatial whole transcriptomic analysis to gain deeper insights.
 Citation Format: Glori Das, Matthew Vasquez, Jeffrey Zhang, Ji-Hoon Lee, Yuan Gao, Jilun Zhang, Jenny Chang, Xiaoxian Li, Hong Zhao, Stephen T.C. Wong. Spatial Omics Analysis Uncovers Microenvironmental Remodeling and Immune Dynamics in T-DXd Resistant Metastatic Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS4-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c6cc9418499478f1051264c562f281d5a33003b" target='_blank'>
              Abstract PS4-09: Spatial Omics Analysis Uncovers Microenvironmental Remodeling and Immune Dynamics in T-DXd Resistant Metastatic Breast Cancer
              </a>
            </td>
          <td>
            Glori Das, Matthew Vasquez, Jeffrey Zhang, Ji-hoon Lee, Yuan Gao, Jilun Zhang, Jenny Chang, Xiaoxian Li, Hong Zhao, Stephen Wong
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common malignant tumors with high incidence and mortality. The challenge remains to construct reliable prognostic prediction models and to further elucidate the key molecular mechanisms of tumor progression. To address this, we performed WGCNA based on 120 immune cell expression profiles from GEO sources to obtain a collection of monocytes/macrophages-related genes. The prognostic model was constructed by univariate survival analysis and LASSO regression analysis. Then, the prognostic model was validated by Multivariate Cox regression, Kaplan-Meier survival analysis and ROC analysis. In this prognostic model, we identified that PLIN2 has a potential value for CRC prognosis. PLIN2 expression in monocytes/macrophages was verified by scRNA-seq datasets and spatial transcriptome datasets, and PLIN2 was found to promote macrophage transformation to M2 subtype. Clinical specimens and tissue microarrays confirmed the differential expression and prognostic value of PLIN2 in CRC patients. Functional experiments demonstrated that PLIN2 gene overexpression promoted the proliferation, migration and invasion of CRC cells and significantly facilitated tumor growth in vivo. Mechanistically, we revealed that CD36 is a potential downstream target gene of PLIN2. The CD36 inhibitor Sulfo-N-succinimidyl Oleate significantly reversed PLIN2-induced proliferation, migration, invasion, and EMT activity of CRC cells in vitro and in vivo. Immunoprecipitation and immunofluorescence experiments confirmed that PLIN2 could interact with CD36. PLIN2 stabilized CD36 protein expression by inhibiting the proteasomal degradation pathway, thereby promoting CD36-mediated EMT activity. Overall, our study highlights that the PLIN2/CD36 axis regulates EMT activity and CRC progression, suggesting that interventions in this signaling pathway may offer a promising therapeutic approach to CRC progression. Schematic diagram elucidating the role of PLIN2 in CRC by Figdraw. FA is transported into the cell via CD36-mediated endocytosis. In CRC cells, PLIN2 promotes stability of CD36 and interacts with CD36 to activate the EMT process. However, the CD36 inhibitor SSO inhibits the binding of FAs to CD36 and attenuates its endocytosis, thereby reversing the PLIN2-mediated EMT process. Ultimately, the PLIN2-induced enhancement of CRC cell proliferation, migration, and invasion is attenuated by the CD36 inhibitor SSO.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e0a7e18e5633e2450668612a5920001ab2297dc" target='_blank'>
              PLIN2 promotes colorectal cancer progression through CD36-mediated epithelial-mesenchymal transition
              </a>
            </td>
          <td>
            Fan Yang, Ying Li, Xue Shang, Yun Zhu, Wenting Hou, Yi Liu, Qing Hua, Zhirong Sun
          </td>
          <td>2025-07-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains one of the most lethal malignancies, largely due to its pronounced cellular heterogeneity, therapy resistance, and metastatic propensity, traits driven by cancer stem cells (CSCs). Once thought to be a rare, static subpopulation, CSCs are now recognized as a dynamic, plastic cell state influenced by both intrinsic and extrinsic factors. In this review, we explore the “inner” (genetic, cellular, and metabolic) and “outer” (microenvironmental and macroenvironmental) determinants that govern CSC identity, emergence, and evolution in PDAC. We focus on how acinar and ductal cell plasticity, in combination with oncogenic mutations (e.g., KRAS, TP53), contribute to tumor initiation and CSC generation, how genomic instability further fuels CSC heterogeneity and adaptability, and we highlight how micro and macro TME properties, metabolism and external factors regulate CSC behavior afterwards. Altogether, CSCs exist at the nexus of genetic chaos and environmental cues. Understanding this complex interplay is key to developing therapies that not only eradicate CSCs but also prevent their reemergence. While still far from clinical application, disrupting CSC-supportive networks should be a research and clinical focus for improving outcomes for PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b08e2a3f4aedf4a7111928c29cee984777f2e319" target='_blank'>
              Inner and Outer Determinants of Pancreatic Cancer Stem Cells
              </a>
            </td>
          <td>
            J. C. López-Gil, Diego Navarro, B. Sainz
          </td>
          <td>2025-06-16</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor heterogeneity and the dynamic evolution of tumor immune microenvironment (TIME) contribute to therapeutic resistance and poor clinical prognosis. To elucidate this mechanism, we first established a murine tumor evolution model (TEM) and systematically identified evolutionary core genes demonstrating progressive alterations during evolution. Subsequently, we developed a single-cell TEM through integrative analysis of hepatocellular carcinoma (HCC) clinical specimens (n=10) with external cohorts (n=11), enabling dynamic characterization of tumor-immune interactions during evolution, while addressing ethical challenges associated with obtaining tumor tissues from multiple stages in a single patient. Through TEMs analyses, we identified a contrasting glucose metabolism pattern between malignant cells and CD8+ T cells during tumor evolution. Mechanistically, glucose metabolic dominance triggers NSUN2 upregulation in tumor cells, where this functional RNA methyltransferase stabilizes key glycolytic transcripts (GLUT1, HK2, PFKM) through mRNA methylation. The NSUN2-mediated GLUT1 stabilization enhances the competitive advantage of tumor cells in glucose acquisition, creating a positive feedback loop that accelerates malignancy and exacerbates CD8+ T cell dysfunction. Building on these insights, we designed a dual-targeting strategy combining GLUT1/NSUN2 axis inhibitor WZB117 with PD-L1 blockade, which synergistically suppressed tumor evolution and reversed immunosuppression in preclinical models, suggesting a novel synergistic therapeutic strategy for treatment-resistant HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b96376e6c37993b534f4d68956c9aa4ba9723b03" target='_blank'>
              The glucose sensor NSUN2-m5C modification regulates tumor-immune glucose metabolism reprogramming to drive hepatocellular carcinoma evolution
              </a>
            </td>
          <td>
            Jing He, Boqiang Liu, Weijun Zhao, Hao Shen, Yi Wang, Weiqi Li, Chenqi Jin, Yifan Wang, Xiujun Cai, Liang Shi
          </td>
          <td>2025-07-11</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Canine oral squamous cell carcinoma (COSCC) is the second most common oral tumor in dogs and the most relevant for comparative human trials as a spontaneous large animal model of disease. Historical genomic work has focused primarily on bulk sequencing. The present study describes the complete transcriptomic landscape of COSCC with spatial distinction between the surface tumor, deep invasive tumor, peritumoral dysplastic epithelium, and tumor microenvironment compared to matched normal oral samples. Each region demonstrated distinct molecular signatures. Genes related to epithelial growth factor (EGFR) and epithelial-mesenchymal transformation (EMT) were upregulated in both peritumoral dysplasia and surface cancer. Additionally, the KRAS gene set, KRT17, and SSP1 were enriched in cancer. We identified five genes that represent dysplastic lesion with high potential for malignant transformation (FZD4, GAS1, HACD2, NOG, and SLC39A6). Also, three genes, SFRP4, FZD1, and IL34 represented a specific signature of the invasive portion of the COSCC that should be explored for prognostic value as a biomarker of malignancy. Lastly, we verified the immunomodulatory tumor microenvironment detecting an increase in macrophages and an abundance of IL-10 secretion. The other predominant leukocytes were T-cells, with CD4+ T-cells being the most prevalent. CD4+ T cells expressed transcripts for both stimulatory (Inducible T-cell Co-Stimulator (ICOS) and inhibitory molecules (CTLA4). The observed high CTLA4 suggests that this inhibitory signal may be preventing a robust antitumor immune response. Taken together, this study identified multiple targets to be explored for biomarkers of malignancy, prediction of tumor behavior, and potential targets for development of novel therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8c8aa8e67efb4b761fc68addeac746847b989f" target='_blank'>
              Spatial Transcriptomic Landscape of Canine Oral Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Stephanie L Goldschmidt, Clifford G Tepper, Jack Goon, M. Soltero-Rivera, Robert B Rebhun, Andrew C. Birkeland, Xiao-Jing Wang, Ryan R Davis, Stephenie Y Liu, Iris Rivas, Brian Murphy, Natalis Vapniarsky
          </td>
          <td>2025-06-17</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer is the most common malignancy in females and remains the leading cause of cancer-related deaths for women worldwide. The cellular and molecular basis of breast tumorigenesis is not completely understood partly due to the lack of human research models which simulate the development of breast cancer. Here, we developed a method for generating functional mammary-like cells (MCs) from human-induced pluripotent stem cells (iPSCs). The iPSC-MCs closely resemble human primary MCs at cellular, transcriptional, and functional levels. Using this method, a breast cancer model was generated using patient-derived iPSCs harboring germline BRCA1 mutation. The patient iPSC-MCs recapitulated the transcriptome, clinical genomic alteration, and tumorigenic ability of breast cancer cells. We also identified S100P as an oncogene downstream of mutated BRCA1 that promotes cancer cell stemness and tumorigenesis. Our study establishes a promising system of breast cancer for studying the mechanism of tumorigenesis and identifying potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38bb1e01fa606965d2a0d3911a39dbf8ba8c4b7" target='_blank'>
              Human iPSC-based breast cancer model identifies S100P-dependent cancer stemness induced by BRCA1 mutation
              </a>
            </td>
          <td>
            Jingxin Liu, Cai Zhao, Jiahao Chen, Pengguihang Zeng, Qingjian Li, Ranran Dai, Xingqiang Lai, Wenqian Song, Jianing Chen, Xixi Zhu, Xinyi Liu, Jun Sun, Jia Wang, P. Fang, Tengfei Wang, Wenjie Chen, D. Guallar, Nan Cao, Jianli Zhao, Shicheng Su, Andy Peng Xiang, Y. A. Zeng, Jie Li, J. Cai, Dung-Fang Lee, Jinxin Bei, Yongliang Huo, Hai Hu, Shengbao Suo, Dong-feng Huang, Jin Bai, Junjun Ding
          </td>
          <td>2025-07-25</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Background: Personalized medicine in breast cancer requires a detailed understanding of the tumor and its microenvironment at the molecular level. Current diagnostic strategies, primarily based on 2D imaging methods, are limited by the lack of spatial molecular 3D mapping. Our study aims to bridge this gap by developing a 3D imaging methodology for simultaneously profiling RNA and protein within breast cancer tissues, providing a holistic view of tumor biology. Methods: We developed a novel 3D imaging method to achieve high-resolution 3D mapping of RNA and protein expressions in breast cancer tissue samples. This method integrates 3D in situ hybridization chain reaction, 3D immunostaining, and a non-toxic tissue clearing technique. We applied this method to intact triple-negative breast cancer (TNBC) samples, performing multiplex labeling for ErbB-2 (mRNA), HER2 (protein), and CD34 for co-mapping tumor cells with the vascular network. Machine-learning-based topological analysis was utilized to characterize spatial blood vessel organization across samples. Various visualization techniques, including scatter plots, histograms and cumulative plots, were employed to investigate the interactions between tumor cells and their nearest vessels. Results: Our approach successfully generated 3D maps of RNA and protein distribution within breast cancer sample blocks. Topological analysis of the vascular network identified metrics such as vessel length density (VLD), vessel volume density (VVD), connection density (CD), mean angle (MA), mean diameter (MD), and straightness. The results indicated that VLD and VVD positively correlated with prognosis, while CD and MA showed a tendency towards negative correlation. Notably, a subset of cells with ErbB2-mediate expression and HER2-negative expression (ErbB2++/HER2-) was prevalent in many TNBC cases. By comparing these findings with simulated random cell distribution experiments, we observed that the attraction distance between ErbB2++/HER2- cells and blood vessels in TNBC with lymph node metastasis was 58.77μm, significantly smaller than the 401.20μm observed in TNBC without lymph node metastasis. This implies a closer association of ErbB2++/HER2- cells with blood vessels in metastatic TNBC. The analysis revealed a statistically significant increase in the proportion of perivascular ErbB2++/HER2- cells in TNBC samples associated with lymph node metastasis compared to those without lymph node involvement (P<0.05). Conclusion: The simultaneous 3D mapping of RNA and protein provides unprecedented insights into the molecular architecture of breast cancer. Our findings underscore the critical importance of spatial context in molecular profiling. This newly developed method offers a powerful tool for identifying spatially therapeutic targets, redefining our understanding of tumor biology, and guiding clinical decisions.
 Citation Format: Yue Li, Shigeaki Kanatani, Per Uhlén. Simultaneous 3D Mapping of RNA and Protein to Advance Personalized Medicine in Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-09-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cca86c99502724279e660618f3ccf5d434b2655" target='_blank'>
              Abstract P1-09-05: Simultaneous 3D Mapping of RNA and Protein to Advance Personalized Medicine in Breast Cancer
              </a>
            </td>
          <td>
            Yue Li, Shigeaki Kanatani, Per Uhlén
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Serous tubal intraepithelial carcinomas (lesions) in the human fallopian tube epithelium (hFTE) are theorized to give rise to high-grade serous ovarian cancers. Small extracellular vesicles (sEV) are known to mediate key signaling in both normal and cancerous tissues, but few ex vivo systems exist for studying the impact of sEV on hFTE tissue. In this study, we present a microfluidic tissue culture platform with combined spatial transcriptomic and proteomic readouts that allows us to profile dual responses in tissue exposed to sEV “messages”—capturing both short-term transcriptomic shifts in the tissue and long-term changes in protein cargo of secreted EVs (the “reply”). Using spatial transcriptomics, we show that the short-term 1-day exposure to ovarian cancer–derived sEVs alters expression of 68 transcripts in secretory cells, the progenitor of high-grade serous ovarian cancer, notably upregulating immune-related mRNA, including CXCL family chemokines, VCAM1, and pro-inflammatory mediators (NFKB1, IL1B, and IFNA7/17). Additionally, we observed that the long-term 14-day exposure to sEVs alters the expression of seven transcripts and 25 EV cargo proteins of fallopian tube–derived EVs (“secondary release EVs”) following stimulus from cancer EVs. Together, tissue transcriptomics and tissue-derived EV proteomics indicate that ovarian cancer–derived sEVs rewire target cell signaling to modify the tubal immune landscape. This study provides insights into the early molecular changes associated with the pathogenesis of ovarian cancer in its tissue of origin, providing a platform to study EV–tissue interactions and identify how sEVs drive cell signaling reprogramming in hFTE. Significance: We model the fallopian tube preneoplastic landscape using a microfluidic platform to study EV-induced stress and show that cancer EVs promote immune signaling changes representing the earliest stages of ovarian cancer pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7b9b7316e20845062da859010e1284d4df77437" target='_blank'>
              Defining the Ovarian Cancer Precancerous Landscape through Modeling Fallopian Tube Epithelium Reprogramming Driven by Extracellular Vesicles
              </a>
            </td>
          <td>
            Jared Sipes, Didi Zha, Sagar Rayamajhi, Leonidas E Bantis, Rashna Madan, Amrita Mitra, Rajni V Puri, M. Rahman, Foyez Ahmmed, H. Pathak, Angela Russo, Mihaela Sardiu, Brett C Isenberg, B. P. Cain, J. Coppeta, Pamoda M Galhenage, S. Pathania, Shannon MacLaughlan David, Joanna E. Burdette, Andrew K. Godwin
          </td>
          <td>2025-07-21</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Introduction Hepatocellular carcinoma (HCC) is the most common primary liver cancer and a leading cause of cancer-related mortality worldwide. While the Triggering Receptor Expressed on Myeloid Cells 1 (TREM1) is well-known for its role in amplifying inflammation within the tumor microenvironment (TME), its tumor-intrinsic role remains poorly defined. Liver cancer stem-like cells (LCSLCs), charecerized by expression of CD133 and EpCAM, are critical for HCC initiation, metastasis, recurrence, and therapy resistance. Methods We used flow cytometry to assess TREM1 expression in LCSLCs and employed CRISPR-Cas9 gene editing to knock out TREM1 in HCC cell lines. Functional assays, including proliferation, migration, apoptosis, clonogenicity, and spheroid formation, were performed. Cell line-derived xenograft (CDX) models were used to evaluate in vivo tumorigenicity. Transcriptomic profiling was conducted to explore downstream effects of TREM1 deletion. Additionally, a pharmacological inhibitor of TREM1 (VJDT) was used to validate the therapeutic potential of targeting TREM1 in vivo. Results TREM1 was highly expressed in CD133+EpCAM+ LCSLCs. Knockout of TREM1 significantly impaired proliferation and migration while promoting apoptosis in HCC cells. In LCSLCs, TREM1 silencing reduced clonogenic ability and spheroid formation, indicating loss of self-renewal and stemness. In CDX models, TREM1-deficient LCSLCs exhibited markedly reduced tumorigenicity. Transcriptomic analysis revealed distinct, context-dependent gene expression changes in nuclear and extracellular signaling pathways following TREM1 loss. Pharmacologic inhibition of TREM1 with VJDT recapitulated the tumor-suppressive effects observed in genetic models. Discussion Our findings establish TREM1 as a critical tumor-intrinsic regulator of LCSLC survival and tumorigenic potential, independent of its known immunomodulatory role in the TME. Targeting TREM1 may therefore represent a promising dual-action therapeutic strategy to disrupt both cancer stem-like cell function and the pro-inflammatory tumor milieu in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a00d49907f1e3afafac72e30d20166f4b46251b4" target='_blank'>
              TREM1 is essential for maintaining stemness of liver cancer stem-like cells in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Arsha Sreekumar, Ashwin Ajith, K. Mamouni, D. Horuzsko, Anatolij Horuzsko
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="INTRODUCTION
Osteosarcoma, a highly aggressive bone tumor, continues to pose significant treatment challenges despite advances in molecular research. Traditional therapeutic strategies have largely relied on targeting genetic alterations of tumor genes or signaling pathways, but these approaches have been less effective in clinical settings due to the complex biology.


AREAS COVERED
Recent insights into the molecular landscape of osteosarcoma have revealed key mechanisms of therapeutic resistance, including tumor plasticity, immune evasion, and metabolic reprogramming. The interaction between the tumor and its microenvironment, such as mechanical stress, hypoxia, and extracellular matrix composition, leads to spatially distinct regions with varying drug sensitivities.


EXPERT OPINION
We highlight three key shifts in understanding osteosarcoma biology: 'target plasticity' driven by evolving tumor dynamics, the importance of mechanical signaling at the tumor-bone interface, and the potential of multi-omics platforms for real-time monitoring and personalized treatment. We propose a new therapeutic framework that integrates these advances to overcome resistance mechanisms. By focusing on epigenetic reprogramming, immune resetting, and mechanopharmacological approaches, we envision a more comprehensive strategy for osteosarcoma treatment that goes beyond traditional single-target therapies. The success of these strategies will depend on integrating spatially informed, time-resolved treatment regimens, guided by advanced molecular and computational technologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ffde0bbfbd90bb0bf5e018161ce107f39f7740f" target='_blank'>
              Assessing the current molecular understanding of therapeutic targets in osteosarcoma.
              </a>
            </td>
          <td>
            Chao Zhang, Ji-Lan Yang
          </td>
          <td>2025-08-07</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background The unique stress response state of T cells (Tstr) is found in various cancers and correlates with altered Lung Adenocarcinoma (LUAD) tumor microenvironment and immunotherapy outcomes. However, fewer studies have been described on the relationship between Tstr and LUAD, and its mechanism of action in the pathogenesis of LUAD needs to be further investigated. Methods The categorization and validation of cluster types for Tstr signature genes were systematically carried out using transcriptomic data sourced from the TCGA and GEO databases. The focus was on elucidating distinct pharmacological characteristics, tumor microenvironmental traits, and immunotherapeutic benefits across various subtypes. Moreover, a predictive model was developed through Least Absolute Shrinkage and Selection Operator (LASSO) regression analysis, and subsequent cellular experiments confirmed the pivotal role of CREG2 in driving LUAD cell proliferation and migration. Results Tstr signature genes represented by the heat shock protein family are highly expressed at the end of T-cell differentiation and affect cellular communication. Patients with the C1 subtype have a poorer prognosis and high expression of HSPA1A and HSPA1B, which exhibit cold tumor characteristics and define them as important Tstr subtypes. CREG2 is positioned as a promising biological indicator as it plays an important part in enhancing the migratory and proliferative capacity of LUAD cells. Discussion This study reveals the characteristics of different Tstr types. A predictive model was built to forecast the prognosis of LUAD patients based on the DEGs of the subtype. The exploration delved into the bio-function of CREG2 in the progression of LUAD. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03170-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77dda53b77f8e92a63f02ef3e1d805a06740b60e" target='_blank'>
              Integration of single cell and bulk transcriptomes identifies T cell stress subtypes in LUAD
              </a>
            </td>
          <td>
            Shengping Min, Linfeng Pan, Xinyu Zhang, Huili Chen, Lixuan Qiu, Xinyu Wang, Yiluo Xie, Kai Zhang, Qiang Zhang, Chaoqun Lian, Jing Zhang
          </td>
          <td>2025-07-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The bone marrow microenvironment is intimately linked to the biology that underpins the development and progression of multiple myeloma. However, the complex cellular and molecular features that form bone marrow niches are poorly defined. Here, we used subcellular spatial transcriptomics to profile the expression of 5,001 genes in human bone marrow in the context of multiple myeloma. Using this approach, we explored the plasma cell and stroma ecosystem in bone marrow trephines from 21 individuals, including 7 with pre-malignant disease and 10 with newly diagnosed multiple myeloma. Using spatial transcriptomics in conjunction with an optimised trephine biobanking methodology, we could resolve major components of the human bone marrow microenvironment and reliably characterise distinct plasma cell populations in samples from healthy, pre-malignant disease and active myeloma. When plasma cells were visualised in the context of location, we detected spatially restricted subpopulations of plasma cells in five of ten newly diagnosed myeloma trephines. Surprisingly, the composition of haematopoietic and stromal microenvironments varied significantly between newly diagnosed myeloma trephines. Furthermore, these differences in microenvironments were also observed within trephines that had spatially restricted plasma cell subpopulations. Thus, these data are not consistent with the hypothesis that a universal bone marrow microenvironment supports the expansion of malignant plasma cells in myeloma. Instead, we propose that myeloma subpopulations form distinct microenvironments and can vary between both patients and spatial location.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb6467d36212cc2446a47ddf3e54593d16e09483" target='_blank'>
              Profiling the spatial architecture of multiple myeloma in human bone marrow trephine biopsy specimens with spatial transcriptomics.
              </a>
            </td>
          <td>
            R. K. Yip, Jeremy Er, Lei Qin, Quoc Hoang Nguyen, Allan Motyer, Joel S. Rimes, Amanda Light, Ruvimbo D. Mishi, Ling Ling, Casey J. A. Anttila, Ellen Tsui, D. Amann-Zalcenstein, Mark R. Dowling, K. Rogers, R. Bowden, Yunshun Chen, Simon J Harrison, Edwin D Hawkins
          </td>
          <td>2025-07-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Abstract Malignant glioma is a highly aggressive brain tumor characterized by frequent recurrence, poor prognosis, and limited responsiveness to standard therapies. Glioblastoma multiforme, the most common and aggressive subtype, further complicates treatment due to its infiltrative nature, genetic heterogeneity, the protective blood-brain barrier, and an immunosuppressive microenvironment. Despite aggressive treatment strategies such as surgical resection combined with chemoradiotherapy, the median survival for malignant glioma patients remains low, highlighting the urgent need for more effective therapeutic approaches. Oncolytic virotherapy (OVT), a dual-modality approach that combines immunotherapy and biotherapy, has emerged as a promising alternative. Oncolytic viruses (OVs) can replicate continuously, disseminate within the tumor, and stimulate anti-tumor immunity, offering distinct advantages in targeting invasive and immunologically “cold” malignant glioma. However, the efficacy of OVT in clinical trials remains unsatisfactory, particularly in single-agent regimens. This limitation is primarily attributed to the viruses’ limited replication efficiency, suboptimal immune induction, premature clearance by antiviral immune responses, and the blood-brain barrier, which impedes effective intracranial delivery. Thus, further optimization of viral modifications, delivery systems, and treatment regimens is critical to enhancing therapeutic potency before OVT can become a standard therapy for malignant glioma. This review systematically summarizes current strategies for enhancing OVs, including genetic engineering, chemical functionalization, and carrier-based delivery. Furthermore, it highlights combination therapies that aim to synergistically enhance therapeutic efficacy through chemotherapy, radiotherapy, and immunotherapy. Finally, the review emphasizes recent clinical trials leveraging these enhancement strategies, aiming to offer novel insights for translating OVs from research to clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f823dd6016cb2ab73e24869fff2ae51d94c3fc49" target='_blank'>
              Optimizing Oncolytic Virotherapy for Malignant Glioma: From Bench to Bedside
              </a>
            </td>
          <td>
            Weihua Jiang, Yeqing Tian, Huafen Gu, Wenqing Guan
          </td>
          <td>2025-08-04</td>
          <td>Cancer Management and Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immune evasion represents a significant challenge in oncology. It allows tumors to evade immune surveillance and destruction, thereby complicating therapeutic interventions and contributing to suboptimal patient outcomes. This review addresses the critical need to understand how cancers evade immune surveillance. It aims to provide a comprehensive overview of strategies of tumors to escape immune detection by examining tumor-induced immune suppression, immune checkpoint regulation, and genetic and epigenetic influences. Moreover, it explores the dynamic role of the tumor microenvironment (TME) in fostering immune resistance and highlights the impact of metabolic reprogramming on immune suppression. Additionally, this review focuses on how tumor heterogeneity influences immune evasion and discusses the limitations of current immunotherapies. The role of key signaling pathways, including programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte–associated antigen 4 (CTLA-4), transforming growth factor-β (TGF-β), nuclear factor kappa-light-chain-enhancer of activated B cells (NF-κB), and cyclic GMP-AMP synthase-stimulator of interferon genes (cGAS–STING) is analyzed to elucidate their contributions to immune escape. Emphasizing the complexities of immune evasion, this review underscores the importance of personalized approaches and the integration of multi-omics data to combat therapeutic resistance. Furthermore, it discusses novel and emerging therapeutic strategies, such as bispecific antibodies, oncolytic viruses, and nanotechnology-driven immunotherapies, showcasing innovative avenues in cancer treatment. The significance of this review lies in its potential to guide future research and innovations in immunotherapy, ultimately improving patient outcomes and advancing our understanding of cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b9bd056fe3296a46c5f79e0e0f8c410fd536528" target='_blank'>
              Immune evasion in cancer: mechanisms and cutting-edge therapeutic approaches
              </a>
            </td>
          <td>
            Muhammad Tufail, Canhua Jiang, Ning Li
          </td>
          <td>2025-07-31</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Oxaliplatin-based neoadjuvant chemotherapy (NAC) is the standard treatment for advanced colorectal cancer (CRC), yet resistance to NAC poses a significant clinical challenge. Methods To investigate the mechanisms of chemoresistance, we analyzed single-cell RNA sequencing (scRNA-seq) data from CRC patients undergoing NAC. Comprehensive analyses, including InferCNV, differentially expressed genes analysis, pathway enrichment, cell communication, and SCENIC were performed. High-throughput drug screening identified potential therapeutic candidates targeting chemoresistance pathways, and the efficacy of targeting the KLF5/PI3K/AKT axis in combination with oxaliplatin was explored in animal models. Results NAC effectively reduced tumor burden and enhanced T_NK cell infiltration in responsive tumors. Notably, NAC-resistant cell clusters exhibited activation of fatty acid-related metabolic pathways and demonstrated limited immune infiltration. Transcriptional analysis identified KLF5 as a potential driver of chemotherapy resistance. Based on these findings, we developed a KLF5 regulon-associated risk score model with significant potential for predicting CRC patient prognosis. Mechanistically, KLF5 activation of the PI3K/AKT pathway conferred chemoresistance in CRC cells. Through high-throughput screening, GDC-0941, a PI3K/AKT inhibitor, emerged as a promising therapeutic agent that synergistically enhanced oxaliplatin efficacy and overcame resistance in preclinical models. Conclusions Targeting the KLF5/PI3K/AKT axis may enhance chemotherapy efficacy and overcome drug resistance in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c89694e65cc06e36cf7753e13edc503c27ae0798" target='_blank'>
              Targeting the KLF5/PI3K/AKT axis as a therapeutic strategy to overcome neoadjuvant chemoresistance in colorectal cancer
              </a>
            </td>
          <td>
            Meiling Gao, Jiahao Qian, Ping Xia, Wancheng Liu, Yunjia Jiang, Yuchang Xia, Xin Yao, Q. Jiao, Minggang Wei
          </td>
          <td>2025-07-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Macrophages often pervade solid tumors, and clusters of macrophages sometimes associate with longer survival of patients. However, clustering mechanisms and impacts on key functions such as phagocytosis remain obscure. Here, under conditions that maximize cancer cell phagocytosis within cohesive tumors, we uncover pathways that favor dynamic clusters and find a colocalization of tumor-intrusive pseudopodia which we term "intrudopodia." Cluster formation over hours on low-adhesion substrates occurs after macrophage induction to a state colloquially referred to as M1 after exposure to interferons and T cell-derived cytokines. Clusters prove fluid on timescales of minutes and also sort from interleukin-4-treated, so-called M2 macrophages that tend to disperse. M1 macrophages upregulate specific cell-cell adhesion receptors but suppress actomyosin contractility, with both pathways contributing to cluster formation. Decreased cortical tension was not only reflected in a low level of nuclear lamin-A that downregulates cytoskeletal targets of serum response factor and tends to soften the nucleus but was also predicted to unleash pseudopodia. Macrophage neighbors in tumor spheroids indeed coextend intrudopodia between cancer cell junctions-at least when phagocytosis conditions are maximized. Intrudopodia from neighbors help detach and individualize cancer cells for rapid engulfment. Juxtaposition of a macrophage cluster with tumor cell nests defines a broad interface that minimizes cancer cell nearest neighbor interactions and maximizes coordination of macrophage intrudopodia. Cooperative phagocytosis thus overcomes solid tumor cohesion-and might explain why the macrophage clustering factor ITGAL associates with patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a55b2dafa57fa725379d7dc71b70c66988362113" target='_blank'>
              Clustered macrophages cooperate to eliminate tumors via coordinated intrudopodia.
              </a>
            </td>
          <td>
            L. Dooling, Alişya A Anlaş, Michael P. Tobin, Nicholas M. Ontko, Tristan Marchena, Maximilian Wang, Jason C. Andrechak, D.E. Discher
          </td>
          <td>2025-07-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Osteosarcoma, a prevalent malignant bone tumor in pediatric and adolescent populations, is characterized by a markedly diverse tumor microenvironment (TME) that complicates therapeutic intervention. Among the key components of the TME, M2 macrophages contribute to immune suppression, tumor progression, and resistance to therapy. This study aims to explore the immune microenvironment of osteosarcoma and identify key immunosuppressive components using integrated transcriptomic analysis.



 We integrated scRNA-seq (GSE152048) and bulk RNA-seq (TARGET) data to characterize the osteosarcoma tumor microenvironment. Key analyses included cell clustering and annotation (Seurat), pseudotime trajectory (Monocle 2), cell-cell communication (CellChat), and M2 macrophage deconvolution (CIBERSORT, BayesPrism). Gene modules were identified via WGCNA, and drug sensitivity was predicted using oncoPredict. Key genes were validated by qRT-PCR and Western blot.



 Our integrated analysis revealed that M2 macrophage infiltration was significantly associated with an immunosuppressive tumor microenvironment in osteosarcoma. Weighted gene co-expression network analysis (WGCNA) identified gene modules highly correlated with M2 macrophage abundance and immune evasion signatures. Reduced M2 infiltration was linked to altered transcriptional patterns and increased predicted sensitivity to Etoposide and Doxorubicin. Two immune-related genes, PTK2B and PTPRC, were significantly upregulated in osteosarcoma samples and validated by RT-qPCR and Western blotting in U2OS cells.



 This study demonstrates that M2 macrophage infiltration is a defining feature of the osteosarcoma immune microenvironment and is associated with immune suppression. Through integrative transcriptomic analysis, PTK2B and PTPRC were identified as key immune-related genes linked to M2 macrophage abundance and patient prognosis. These results establish a transcriptional basis for the immunosuppressive phenotype in osteosarcoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c9aeb21260637fca67e1110200fbde1ea1d4d" target='_blank'>
              Integrated RNA sequencing reveals tumor microenvironment heterogeneity and immunosuppressive role of M2 macrophages in osteosarcoma
              </a>
            </td>
          <td>
            Zhijun Ding, Yi Li, Cheng Zhong
          </td>
          <td>2025-08-01</td>
          <td>Oncologie</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background The progression of cancer cells is influenced by the tumor microenvironment (TME); however, the molecular mechanisms driving the progression of esophageal squamous cell carcinoma (ESCC) remain unclear. Therefore, we aimed to investigate the TME of ESCC and construct a risk signature based on apoptosis-related genes to identify prognosis-related genes in ESCC. Methods We integrated a total of 92,714 cells from 18 samples across three single-cell datasets to analyze the differences in cellular landscapes between primary tumor tissues and adjacent normal tissues. Subsequently, univariate COX regression analysis was employed to construct an apoptosis-related prognostic risk model. The expression of key risk genes was elucidated using immunohistochemistry (IHC). Additionally, the effects of CTSC knockdown on ESCC cell behavior were validated through in vitro and in vivo experiments. Results We identified three malignant cell subtypes (Malig1, Malig2, and Malig4) associated with worse prognosis, which were enriched in apoptosis-related pathways. Pseudotime analysis revealed that the expression scores of apoptosis-related pathways increased along the inferred pseudotime, indicating that apoptosis plays a critical regulatory role in the differentiation of malignant epithelial cells. Furthermore, analysis of the TME demonstrated that immune cells and cancer-associated fibroblasts (CAFs) were significantly more abundant in tumor tissues compared to non-tumor tissues. Additionally, we identified eight apoptosis-related genes associated with prognosis, among which the expression of CTSC was closely correlated with resistance outcomes in patients receiving neoadjuvant immunotherapy. In vitro experiments showed that knockdown of CTSC inhibited the proliferation, migration, and other processes of ESCC cells. In vitro experiments showed that knockdown of CTSC inhibited tumor growth and expression of fibroblast markers. Conclusions CTSC plays a crucial role in driving TME remodeling and the progression of drug resistance in ESCC, making it a potential target for clinical therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/805313b37a5d587c6d3293d14d669cfdf04f5520" target='_blank'>
              Identification of CTSC-driven progression in ESCC by single-cell sequencing and experimental validation
              </a>
            </td>
          <td>
            Xin Sui, Yong-xu Jia, Jing Li, Jiayao Xu, Wenjia Wang, Yanru Qin
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tumor Treating Fields (TTFields) therapy is an approved cancer treatment modality, based on non-invasive application of electric fields to the tumor region. Proteomic and cell biology methods revealed a versatile mechanism of action to be involved in the response to TTFields. In the current research we performed whole transcriptome analysis across tumor types to identify pan-cancer responses to TTFields. For this we collected samples from control and TTFields-treated human cancer cell lines of gastric cancer, pancreatic cancer, ovarian cancer, non-small cell lung carcinoma, pleural mesothelioma, and glioblastoma. The transcriptomic analysis supported previous reported effects: downregulation of pathways associated with cell cycle, cell growth, and proliferation; downregulation of DNA replication and the FA-BRCA DNA repair pathway; and upregulation of cellular responses to stress—senescence, autophagy, and apoptosis. Notably, previously unrecognized downstream effects of TTFields were revealed on cellular metabolism, with downregulation of protein and RNA metabolism, and upregulation of steroid biosynthesis. Additional DNA repair pathways were also found to be downregulated, including nucleotide excision repair, base excision repair, and mismatch repair. In conclusion, this study revealed similar response patterns to TTFields across different tumor types, re-enforcing some already pinpointed mechanisms, while revealing new mechanisms. Unlocking these new mechanisms may allow identification of potential new cancer treatments for application together with TTFields based on mechanistical compatibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1627b07ec04103438850967d9453d926d5ec1e3d" target='_blank'>
              The transcriptomic fingerprint of cancer response to Tumor Treating Fields (TTFields)
              </a>
            </td>
          <td>
            K. Wainer-Katsir, A. Haber, H. Fishman, L. Ding, Michael D. Story, Renfei Du, U. Kahlert, L. Mannarino, F. Mirimao, M. Lupi, Maurizio D’Incalci, G. Lavy-Shahaf, Hila Ene, R. Frechtel-Gerzi, Zeina Drawshy, A. Martinez-Conde, E. Dor-On, Y. Porat, M. Giladi, U. Weinberg, Y. Palti
          </td>
          <td>2025-07-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b31f4399101f9b6167477a72fd46a9e859a5855" target='_blank'>
              Novel Computational Analysis Identifies Cytotoxic Lymphocyte-to-Monocyte Balance in Tumors as a Predictor of Recurrence-Free Survival in Colorectal Carcinoma.
              </a>
            </td>
          <td>
            Manuel Fernandez, L. Todeschini, Bridget P Keenan, David Rosenberg, Sophia Hernandez, M. Zampese, Guilin Qiao, T. Pollini, A. Maker
          </td>
          <td>2025-06-21</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Drug resistance significantly impairs the prognosis of patients with gastric cancer (GC). As a key player in the acquisition of drug resistance, understanding the detailed evolution of the GC tumor microenvironment (TME) is crucial for improving therapeutic effectiveness. The clinical significance of markers related to cancer-associated fibroblast (CAF) proliferation and extracellular matrix remodeling were analyzed using public databases and immunohistochemistry staining on an in-house cohort of patients with GC. Combining this with single-cell RNA sequencing data revealed the expression patterns of all candidate markers, highlighting POSTN due to its pronounced upregulation in CAFs and its strong correlation with poor prognosis in patients with GC. Mechanistically, POSTN is directly regulated by the YAP1/TEAD1 co-transcription factor and is demonstrated to play significant roles in fibroblast proliferation and migration processes. The present study underscores POSTN as a promising marker for predicting GC prognosis and a powerful regulator that can augment the tumorigenic phenotypes of CAFs, providing a potential target to mitigate CAF-originated drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3a85ca4b39083cd7faf716f5d06621406b8862a" target='_blank'>
              POSTN is exclusively activated in cancer-associated fibroblasts and leads to unfavorable prognosis of patients with gastric cancer
              </a>
            </td>
          <td>
            Ching Hei To, Fuda Xie, Peiyao Yu, Jialin Wu, Yang Lyu, Bonan Chen, Tiejun Feng, Hoi Wing Leung, Wei Kang
          </td>
          <td>2025-07-08</td>
          <td>Molecular and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36b470342bf0dd2e7ecb0177374c341bedf3ddd" target='_blank'>
              Epithelial tumor cells utilize mast cell-derived histamine to regulate perineural invasion
              </a>
            </td>
          <td>
            Ankit Srivastava, Tomas Bencomo, Chaw-Ning Lee, Angela M. Mah, Jasmine M Garcia, L. Seow, Isoline M. Donohue, Aiko J. Tan, Audrey Nguyen, Tiffany Jiang, S. Gombar, Lilian Phu, Pankaj Dwivedi, Christopher M. Rose, Ryanne Brown, Carolyn S. Lee
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Advanced sarcomas have dismal outcomes in children and adults, with limited therapeutic options. While chimeric antigen receptor T cells (CAR-T) hold promise for treating advanced sarcomas, it is constrained by a paucity of effectual targets. Our previous clinical study identified endoglin (ENG/CD105), a TGF-β co-receptor, as a target of the endogenous immune response in a sarcoma patient who exhibited an exceptional response to HER2 CAR-T therapy. ENG is expressed on various sarcomas, cancer-associated fibroblasts, and neoangiogenic vessels - offering comprehensive tumor targeting. Further, ENG knockout in sarcoma cells reduced their invasiveness, highlighting its potential as a therapeutic target. Accordingly, we designed a second-generation human ENG targeting CAR molecule signaling through the CD28 endodomain and retrovirally transduced primary human T cells. The ENG CAR-T exhibited strong antigen-specific cytokine release, robust proliferation, memory formation, and cytotoxic function against various sarcoma cell lines. Their cytotoxicity remained unaffected by the presence of soluble ENG or its natural ligand bone morphogenetic protein-9. Further, ENG CAR-T disrupted multicellular tumor spheroids in vitro, overcoming tumor compactness and the stromal barrier created by cancer-associated fibroblasts - critical challenges in sarcoma CAR-T therapy. In orthotopic murine models of sarcomas, ENG CAR-T treatment resulted in control of tumor growth and metastasis leading to survival extension. In summary, our study describes the involvement of ENG in sarcoma metastasis and validates our human ENG CAR-T as a potential therapeutic for advanced sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25759f5ebd4df60c994a7d268106ebfd826f4a9d" target='_blank'>
              Endoglin-directed CAR-T cells Comprehensively Target Tumors in Advanced Sarcomas.
              </a>
            </td>
          <td>
            Harrison Berger, Malina Maharana, Jeneffer Mirabal, L. Kurenbekova, A. Delaidelli, Atreyi Dasgupta, A. Gad, Mohamed F Sheha, Sybrina S Kerr, Ada I Ozcan, Jessica S Morris, Angela M Major, M. J. Hicks, Mary K McKenna, Ben K Seon, Matthew L. Baker, Poul H. Sorensen, M. Hegde, Jason Yustein, Nabil Ahmed, S. Joseph
          </td>
          <td>2025-08-01</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9d6d4cae3e0a9086b4188f54060ec83ad93a702" target='_blank'>
              Integrating scRNA-seq and spatial transcriptomics to explore the implication of G6PD on immune microenvironment in lymphatic metastasis of breast cancer.
              </a>
            </td>
          <td>
            Hongsen Liu, Mengting Chen, Bo Hong, Ruijin Liang, Lijie Fan, Yun Qian
          </td>
          <td>2025-07-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Background Epigenetic regulatory genes (epiRG) are pivotal in the epigenetic regulation of the human genome, primarily through DNA and histone modifications. These genes are frequently mutated in human cancers, particularly bladder cancer (BC). However, the functional impact of epiRG mutations on patient outcomes remains poorly understood. Methods In this study, we developed gene signatures for the most frequent genomic aberrations of epiRG using The Cancer Genome Atlas Bladder Carcinoma (TCGA‐BLCA) dataset and validated these signatures with independent tumor expression profiles for prognostic relevance. Furthermore, we evaluated the role of these signature scores in the immune system within the tumor microenvironment (TME). Finally, we assessed the correlation between epiRG and global DNA methylation. Results Our results indicated that the inferred aberration‐specific signature scores were more predictive of patient stratification than the genomic aberrations. Notably, certain signature scores were significantly associated with patient progression, whereas others correlated with the tumor immune microenvironment via interactions with the immune system. Patients with mutations had high signature scores in CREBBP‐mut and EP300‐mut, which revealed poor overall survival. Conversely, KDM6A‐mut signatures showed an opposite trend, with low scores linking to favorable prognosis through enhanced immune activity. Also, other epiRG signature scores were strongly correlated with the immune system in TME and successfully predicted patients who responded to immunotherapy. Global methylation analysis revealed that high signature scores of KDM6A‐mut are associated with hypomethylation. Conclusions These findings collectively establish epiRG signature scores as powerful biomarkers that integrate genomic, epigenetic, and immune microenvironment features for improved prognostic prediction in bladder cancer. This integrative approach not only advances our understanding of epigenetic mechanisms in BC but also offers potential for developing innovative prognostic tools and therapeutic strategies tailored to personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21c006331fc69988a65728ddc6ae59549e19f926" target='_blank'>
              Integrative Analysis Reveals the Prognostic Effects of Epigenetic Regulators in Bladder Cancer
              </a>
            </td>
          <td>
            Venugopalareddy Mekala, Yupei Lin, Xiang Wang, Naail Chowdhury, Jianrong Li, Chao Cheng
          </td>
          <td>2025-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Immunogenic cell death (ICD) is crucial for cancer development and enhances the effectiveness of immunotherapy by triggering an adaptive immune response. However, the impact of ICD-related genes (ICDRGs) on iCCA progression and patient prognosis remains unclear. iCCA samples from The Cancer Genome Atlas (TCGA) were categorized into two ICD-associated subtypes by consensus clustering, and patient prognosis in these subgroups was assessed. Single-cell RNA sequencing was conducted on viable cells from 8 iCCA patients to understand their transcriptomic profiles, heterogeneity, and immune microenvironment. Principal component analysis (PCA) and uniform manifold approximation and projection (UMAP) analysis were applied to identified 13 cell subpopulations. The gene set enrichment analysis (GSEA) was used to investigate pathway heterogeneity among these subpopulations. CellChat was employed for intercellular communication analysis. Pseudotime trajectory analyses with CytoTRACE and Monocle2 were used to explore differentiation trajectories and functional differences across various states. 30,485 single cells from human iCCA and normal tissues were sequenced, revealing two ICD-associated subtypes. The ICD-low subtype correlates with better clinical outcomes, whereas the ICD-high subtype is linked to increased immune cell infiltration and immune response signaling activity. Distinct biological functions and metabolic characteristics were observed in iCCA between high- and low-ICD groups. High infiltration of ICD-low subtype in iCCA is associated with poor prognosis, potentially through the ANXA-FPR pathway affecting macrophage proliferation and differentiation. Our study constructed an immunogenic cell death-related signature that can promise prognosis prediction and personalized medicine in iCCA. This finding might help guide clinicians for iCCA patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08602-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e45d84f628dac0dca7f3bfd929def33d5b1582" target='_blank'>
              Single cell RNA sequencing to identify an Immunogenic cell death related prognostic signature in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Bowen Sha, Fei Tong, Qinghua Shu, Miao Wu, Liang Zhao, Yufeng Zhang
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal cancers, with an average survival time of only six months following diagnosis, even with currently available therapies. Thus, PDAC represents a significant therapeutic challenge, necessitating a deeper understanding of its biology and tumor microenvironment (TME) to develop more effective treatments and improve patient outcomes. Here, we report that the expression of Eukaryotic Elongation Factor-2 Kinase (eEF2K) is associated with shorter patient survival and demonstrate that eEF2K signaling is critical for the PDAC tumor growth and regulated by the TME. Furthermore, in vivo targeted genetic inhibition of eEF2K suppressed tumor growth in two different PDAC mouse models, reduced tumor-associated macrophages (TAMs), and induced marked apoptosis in tumor tissues without any signs of toxicity. Our data suggest that eEF2K knockdown diminishes the activity of the AXL receptor tyrosine kinase and reduces the expression of macrophage-derived factors, such as Monocyte Chemoattractant Protein-1 (MCP1), along with the Gas6/AXL signaling pathway in PDAC cells. Additionally, analysis of the NCI-TCGA PDAC patient database further showed that eEF2K expression, in the presence of TAM markers, correlates with even shorter patient survival. TAM-released factors, such as MCP1, Gas6, and exosomes, induce eEF2K expression in PDAC cells, as well as the activity of AXL, SRC, VEGF, Snail, and MMP2, contributing to epithelial-to-mesenchymal transition (EMT), invasion, metastasis, and angiogenesis. In conclusion, our findings reveal for the first time that eEF2K is a critical oncogenic driver of PDAC tumor growth and thus targeting eEF2K represents a promising and novel therapeutic strategy for PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca2a458a943c8611c08480c527e7133171db725" target='_blank'>
              eEF2K is a poor prognostic factor and novel molecular target in pancreatic cancer: regulating tumor growth and progression via the tumor microenvironment
              </a>
            </td>
          <td>
            Didem Karakaş, Ahmed Ashour, H. Mokhlis, N. Kahraman, R. Bayraktar, S. Dilmac, N. Kabil, M. Erdoğan, E. Dere, E. Ulukaya, B. Ozpolat
          </td>
          <td>2025-07-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The presence of a single metastatic lesion significantly decreases overall survival in patients with head and neck squamous cell carcinoma (HNSCC), and invasion of malignant keratinocytes is one of the initial steps required for HNSCC metastasis. Histological grading of tumor cell invasion predicts outcome in HNSCC, yet the molecular factors that determine the extent of invasion, and subsequent grading are not fully understood. Using a 3D organ culture model and multiple patient-derived HNSCC keratinocytes representing all major anatomical subsites of the disease, we identified a range of cell states that represent a continuum of epithelial-to-mesenchymal (EMT) characteristics. We also demonstrated how these cell states change in response to TGF-beta stimulation and co-culture with cancer-associated fibroblasts in organ cultures. Using 3D culture models that recapitulate the pattern of invasion seen in primary tumors from which the keratinocytes were derived, we identified distinct clusters of partial-EMT marker expression in individual patient HNSCC keratinocyte populations. Partial-EMT transcription factors were correlated with separate invasive characteristics, and we demonstrated that ZEB2 (a known EMT driver) and HIC1 (a novel EMT driver) are central nodes in HNSCC keratinocyte invasion. Collectively, our findings refine the concepts of partial-EMT and tumor cell invasion, and identify potential therapeutic targets for future development. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57d784069c4c2d528e9675a4a5572b4390059179" target='_blank'>
              Partial-EMT cell state correlates with single cell pattern of invasion in head and neck SCC keratinocytes.
              </a>
            </td>
          <td>
            Pyung Hun Park, Lauren E Israel, Michael H Alexander, Grace Tartaglia, Harvey G South, Suhao Han, Joseph M. Curry, Adam J. Luginbuhl, Andrew P. South
          </td>
          <td>2025-07-30</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Sex differences in breast cancer (BC) lead to distinct clinical and molecular characteristics. Sex-associated heterogeneity relies intricately on the oncogenic properties of cancer cells and multicellular interactions in tumor microenvironments. Here we conducted a multiomic analysis encompassing single-cell RNA sequencing, spatial transcriptomics and large-scale histological analysis, combined with genetic and pharmacological perturbations to systematically develop a high-resolution and spatially resolved map of intratumoral expression heterogeneity in male breast cancer (MBC). Among the malignant cells, we identify 9 consensus meta-programs (MPs, for example, estrogen response and interferon response), each consisting of dozens of genes that are coordinately upregulated in subpopulations of cells within MBC. Of these, MP5 exhibits significant male bias, characterized by aberrant activation of the estrogen receptor α (ERα) signaling pathway, inducing endocrine therapy resistance in MBC. Bidirectional interactions between MP5 malignant cells and immunosuppressive cancer-associated fibroblasts form an immune exclusionary microenvironment mediated by LRBA. Collectively, our study highlights the endocrine therapy resistant and immunosuppressive role of LRBA in MBC, and these findings may provide insights into the molecular features of MBC and facilitate the development of sex-specifical treatment strategies.
 Citation Format: Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, Jingyan Xue, Jiong Wu. Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df96734a9b8fda8be55365f44c01373b9990e46" target='_blank'>
              Abstract P3-02-26: Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer
              </a>
            </td>
          <td>
            Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, J. Xue, Jiong Wu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains a deadly cancer with non-curative upfront treatment of radiation, resection, and chemotherapy. Not only has the standard of care for GBM patients not improved significantly over the past decade, life expectancy is less than 18 months, with no standard second-line therapy. We previously developed a 2nd generation 4-1BB co-stimulated chimeric antigen receptor (CAR) targeting tumor-specific variant of the epidermal growth factor receptor (EGFRvIII) for treating patients with GBM. This CAR T was used in Phase 1 clinical trials, and demonstrated that CAR T cells rapidly trafficked to tumors and showed initial anti-tumor activity upon encountering EGFRvIII-bearing tumor cells. However, the CAR T cells rapidly became exhausted, losing anti-tumor function, with no durable objective tumor responses.Here, we evaluated the GBM immune environment in a syngeneic implantable GL261 murine model. Prior to tumor implantation, brain-resident immune cells were mostly absent. Following tumor engraftment, there was a pronounced increase in immune cell infiltration over time and with GBM size. Immune infiltrates were intitally comprised of early-arriving lymphocytes including T, NK, and B cells, later this shifted towards increased presence of macrophages and myeloid-derived suppressor cells. Evaluating both fresh and archival GBM samples from patients, we found similarly high levels of infiltrating immune cells, and PDL1 expression on both tumor and immune cells. PD1/PDL1-antibody (Ab) mediated checkpoint inhibition (CPI) has been transformative in treating several types of solid tumors; however the localization of GBM behind the blood-brain barrier limits Ab access, and CPI trials have been unsuccessful in treating GBM. To deliver PD1/PDL1 checkpoint Ab for patients with GBM, we engineered our EGFRvIII-targeted CAR T cells to function as bio-factories, producing and secreting anti-PD1 mini-Abs in situ at the site of GBM.These Ab receptor-modified (ARMed) CAR T cells produced functional PD1 minibodies in vitro and demonstrated anti-tumor activity in vivo in a GBM xenograft model using NOD-Scid gammaC-null (NSG) mice. Delivered systemically, both soluble Ab plus CAR T, and ARMed CAR T cells improved subcutaneously implanted GBM treatment over CAR T alone, while treatment of orthotopic GBM treatment was only improved with ARMed CAR T therapy.These findings demonstrate that engineering EGFRvIII-directed CAR T cells to secrete checkpoint inhibitors locally can overcome immunosuppressive barriers in GBM and bypass the limitations of systemic antibody delivery. This strategy enhances CAR T cell functional persistence and holds strong translational potential for treating GBM and other CNS-localized disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24afeac4893ab682105e3432857369c444fee9b8" target='_blank'>
              Checkpoint antibody receptor modified ARMed CAR T circumvents the suppressive immunome in GBM
              </a>
            </td>
          <td>
            Danielle R. Cook, A. Boesteanu, Yibo Yin, Reiss Reid, L. Roccograndi, Nadia Dahmane, M. Martinez-Lage, Donald M. O’Rourke, Carl H. June, Laura A. Johnson
          </td>
          <td>2025-07-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Introduction Emerging research emphasizes the critical role of local microbiota in shaping the tumor microenvironment (TME) and influencing cancer progression. Lung adenocarcinoma (LUAD) is distinguished by unique bacterial communities that appear to regulate immune responses, gene expression, and patient outcomes. Methods We compiled microbiome profiles from several cancer types—including LUAD, lung squamous cell carcinoma (LUSC), breast carcinoma (BRCA), and thyroid carcinoma (THCA)—using public databases. Non-negative matrix factorization (NMF) was employed to categorize LUAD cases based on TME features, while DESeq2 was used to pinpoint bacterial taxa with differing abundance. Multi-omics networks were developed to integrate microbial, transcriptomic, and clinical data. For in vitro verification, we conducted siRNA-mediated knockdown of the long non-coding RNA LCIIAR and ISG15 in Lewis lung carcinoma cells, followed by proliferation assays. Results In contrast to LUSC, BRCA, and THCA, LUAD exhibited distinct microbial populations, with notable enrichment of Cylindrospermopsis, Cyanothece, and Sulfolobus. NMF clustering identified two LUAD subtypes with differing prognoses. One longer survival cluster, marked by reduced bacterial presence and stronger antitumor immunity—reflected in stronger immune response, increased effector T cells activity, and greater immune cell infiltration. A competing endogenous RNA (ceRNA) network analysis established a link between LCIIAR and ISG15, both overexpressed in LUAD and associated with worse survival outcomes. Knockdown LCIIAR or ISG15 through siRNA significantly inhibited lung cancer cell proliferation, pointing to their roles in tumor growth and ceRNA-mediated regulation. Conclusion LUAD features a distinctive microbiota that engages with inflammatory and ceRNA regulatory pathways. These observations underscore the value of targeting microbiome-influenced mechanisms, such as the LCIIAR–ISG15 axis, as a promising approach to enhance treatment outcomes in lung adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93a45784f057c67e4efb8c56c492f819f5fd7032" target='_blank'>
              Distinct Lung Adenocarcinoma-Associated Microbiota Are Associated with Inflammatory Immune Landscapes and Tumor Cell Proliferation via LCIIAR–ISG15 Regulatory Networks
              </a>
            </td>
          <td>
            Shipu Liu, Zijian Zhang
          </td>
          <td>2025-07-01</td>
          <td>Cancer Management and Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 6],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>