<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-19 09:44:49 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>103</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intertumoral heterogeneity in glioblastoma—driven by both genomic and transcriptomic variation—complicates our understanding of how different tumor cell populations contribute to disease progression. Infiltrating tumor cells, which invade surrounding brain tissue and evade surgical resection, are thought to play a central role in recurrence. To address this, we aimed to characterize the gene expression profiles and cellular states of infiltrative tumor cells in glioblastoma. We performed high-plex spatial transcriptomics using the CosMx Spatial Molecular Imager (NanoString) on tumor tissue from eight glioblastoma patients. Formalin-fixed paraffin-embedded samples were selected to capture both the tumor core and invasive margin. A targeted panel of 1,000 genes enabled spatially resolved gene expression profiling at single-cell resolution, allowing precise identification and localization of malignant and non-malignant cell states. We show that malignant cells can be distinguished from non-malignant populations by using patient-specific clustering. Based on this annotation, we identified several known malignant states—including AC-, OPC-, NPC-, and MES-like cells—as well as a recently characterized glial-progenitor (GPC)-like state. This population co-expressed genes associated with both astrocytic and oligodendrocyte progenitor lineages and was found to be more proliferative than the traditional AC-like state. The GPC-like state was most enriched in the classical glioblastoma subtype and was strongly associated with EGFR amplification or mutation. Spatial analyses investigating malignant differences between tumor and infiltrated tissue showed heterogeneous infiltration patterns across patients. In the most extreme case, the dominant GPC-like population in the tumor core gave way to increased proportions of AC-like cells in infiltrated regions. Our study highlights diverging infiltration patterns across glioblastoma tumors, with indications of a GPC-like to AC-like transition occurring in classical-subtyped tumors. This shift is associated with a decrease in cell proliferation and may have implications for clinical treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300bf1ce4707bea01a2973482ed7fc72841204e2" target='_blank'>
              Genomic heterogeneity drives distinct infiltration patterns in glioblastoma
              </a>
            </td>
          <td>
            D. Harwood, S. Artzi, V. Pedersen, A. Locallo, M. Lü, David Scheie, D. Nørøxe, N. Hammouda, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-11-29</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e69c4c3aa7ac04b1dca84c51fe6263d52efa060" target='_blank'>
              Breast cancer through the lens of whole transcriptome spatial imaging
              </a>
            </td>
          <td>
            Claire Williams, Yi Cui, Michael Patrick, Giang T Ong, Terence Theisen, Megan Vandenberg, Joseph M. Beechem, Patrick Danaher
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Melanoma outcomes have dramatically improved over the past decade, but some patients still experience disease recurrence, particularly those who present at later stage of disease. Prior studies identified an altered immune microenvironment in the sentinel lymph node (SLN) including the presence of dysfunctional CD8 T cells and CD4 T regulatory cells (Tregs), but the spatial organization of the SLN and the interactions of individual cell types have not been extensively studied. To understand how spatial organization of immune cells in the SLN is related to outcomes, we performed spatial proteomic profiling of Stage I and II melanoma SLN to generate an atlas of cell types. Following multiplexed immunofluorescence imaging, deep learning-based segmentation and clustering, we identified 33 subsets of T cells, B cells and Tumor cells. Lymphoid region analyses established a foundational spatial map of the melanoma SLN, revealing higher regional frequencies of activated and memory CD4 T cells in Stage I SLN and consistent positioning of T cells at functional spatial locations. To better understand cellular interactions, we evaluated the immediate neighbors around each index cell using Effect Size Interaction mapping (ESI-map), a novel computational toolkit for understanding spatial interactions. Nearest-neighbor analyses across biological replicates revealed rewiring in cell-cell interacting pairs across disease progression and patient outcomes, particularly with respect to exhausted TOX+ CD8 T cells. Stage II patients who experienced disease recurrence had notable enrichment of cellular interactions between Tregs and TOX+ CD8 T cells that was associated with reduced expression of granzyme B and interferon-γ that was most pronounced in exhausted CD8 T cells. Together, these data demonstrate that the specific spatial interactions between Tregs and exhausted CD8 T cells in the SLN provide a novel immune signature for understanding melanoma outcomes. One Line Summary Treg-CD8 T cell interactions in the sentinel lymph node predict recurrence outcomes in Stage II SLN+ melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927407a38e9474aa6152c45daf9e0743472dc6ee" target='_blank'>
              Cellular interactions in the sentinel lymph node predict melanoma recurrence
              </a>
            </td>
          <td>
            Sabrina M. Solis, Yang Yang, Yee Hoon Foong, Anushka Dheer, K. Ma, M. S. Farooq, Shira G. Rosenberg, Sophie L. Gray-Gaillard, G. Karakousis, Xiaowei Xu, Rami S. Vanguri, Alexander C. Huang, R. Herati
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d2c494a1cd657524f51f9577b21d3baa396ea7" target='_blank'>
              Spatiotemporal regulation of cell cycle states within the complex tumor microenvironment
              </a>
            </td>
          <td>
            Gianni Zanardelli, Olivier Tassy, Maulik K. Nariya, Nacho Molina
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, I. Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatial transcriptomics has emerged as a transformative technology in biomedical research, offering unprecedented insights into gene and protein expression within their native tissue context. Unlike conventional bulk or single-cell sequencing approaches, spatial omics has the advantage of preserving the spatial structure of tissues, allowing researchers to directly map molecular information onto histological structures. This review provides an overview of the current state of spatial omics technologies, highlighting their application in cancer research. Spatial omics has enabled detailed characterization of the tumor microenvironment (TME), revealing spatial heterogeneity, immune cell infiltration patterns, and complex mechanisms of tumor progression and therapy resistance across various cancer types. The review covers future directions, including artificial intelligence–driven analytics, improved standardization, and cost reduction to accelerate clinical translation. Ultimately, spatial omics is poised to play a central role in precision oncology, enabling a deeper understanding of tumor biology and informing more effective individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a77ce136fd5fd93d240b66bc2895fcb841db1de" target='_blank'>
              Spatial omics: applications and utility in profiling the tumor microenvironment
              </a>
            </td>
          <td>
            Ji-Eun See, Sarah Barlow, Wani Arjumand, Hannah DuBose, Felipe Segato Dezem, Jasmine T. Plummer
          </td>
          <td>2025-12-01</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad9383618813c7fb20bfa893bf5368770766ebe" target='_blank'>
              Analysis of intratumoral immune heterogeneity reveals spatial organisation of immunosuppression in breast cancer
              </a>
            </td>
          <td>
            Christina Metoikidou, P. Bonté, Vadim K. Karnaukhov, Laëtitia Lesage, L. Djerroudi, André Nicolas, P. Sirven, A. Vincent-Salomon, Olivier Lantz, Emanuela Romano, Sebastián Amigorena
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d63cfbf90a2e544c2d24c8ab605e5d44e8ba24" target='_blank'>
              A Pan-Cancer Single-Cell Compendium of Intratumoural Heterogeneity
              </a>
            </td>
          <td>
            Ido Nofech-Mozes, Ashton Cook, Tom W. Ouellette, Sara Hafezi-Bakhtiari, Philip Awadalla, Sagi Abelson
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Protein spatial imaging, rooted in spatial proteomics, enables the localization of proteins within their native environment, thus preserving structural and functional context. Its integration into cancer research has provided insights into tumor progression, especially in breast cancer, where ductal carcinoma in situ (DCIS) may transition into invasive breast cancer (IBC). Multiplexed ion beam imaging by time-of-flight (MIBI-TOF) has emerged as a powerful method to visualize protein distribution, tumor–stroma interactions, and immune cell dynamics. Evidence indicates that the tumor microenvironment (TME)—including stromal architecture, extracellular matrix (ECM), and immune cell infiltration—plays a central role in malignant progression. Here, we highlight recent advances in protein spatial imaging, discuss optimization strategies, and examine its implications for breast cancer progression and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a7eef8cc6668ecc3d6de49f721002ee2756192" target='_blank'>
              Spatial Proteomics Imaging to Decode Disease Progression: A Perspective on Tumor Microenvironments
              </a>
            </td>
          <td>
            Haonan Zhang
          </td>
          <td>2025-11-30</td>
          <td>Clinical Medicine and Pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="While advances in single-cell genomics have helped to chart the cellular components of tumor ecosystems, it has been more challenging to characterize their specific spatial organization and functional interactions. Here, we combine single-cell RNA-seq, spatial transcriptomics by Slide-seq, and in situ multiplex RNA analysis to create a detailed spatial map of healthy and dysplastic colon cellular ecosystems and their association with disease progression. We profiled inducible genetic CRC mouse models that recapitulate key features of human CRC, assigned cell types and epithelial expression programs to spatial tissue locations in tumors, and computationally used them to identify the regional features spanning different cells in the same spatial niche. We find that tumors were organized in cellular neighborhoods, each with a distinct composition of cell subtypes, expression programs, and local cellular interactions. Comparing to scRNA-seq and bulk RNA-seq data from human CRC, we find that both cell composition and layout features were conserved between the species, with mouse neighborhoods correlating with malignancy and clinical outcome in human patient tumors, highlighting the relevance of our findings to human disease. Our work offers a comprehensive framework that is applicable across various tissues, tumors, and disease conditions, with tools for the extrapolation of findings from experimental mouse models to human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af74f8f0f1a2428bab07b6671a3e436ab63d622f" target='_blank'>
              Spatially defined multicellular functional units in colorectal cancer revealed from single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            I. Avraham-Davidi, Simon Mages, J. Klughammer, Noa Moriel, Shinya Imada, Matan Hofree, Evan Murray, Jonathan Chen, Karin Pelka, Arnav Mehta, Genevieve M Boland, T. Delorey, Leah Caplan, Danielle Dionne, Robert Strasser, Jana Lalakova, Anežka Niesnerová, Hao Xu, Morgane Rouault, I. Tirosh, Hacohen Nir, Fei Chen, Omer Yilmaz, Jatin Roper, O. Rozenblatt-Rosen, Mor Nitzan, A. Regev
          </td>
          <td>2025-12-12</td>
          <td>eLife</td>
          <td>0</td>
          <td>198</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7a3ed4eb95927059b414292cf9cce743a10321" target='_blank'>
              Spatiotemporal Mapping of Tertiary Lymphoid Structure Heterogeneity Shapes Immune Niches and Clinical Outcomes in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Liaoliao Gao, Jie Mei, Libing Hong, Yuzhi Jin, Jinlin Cheng, Xuqi Sun, Chuan Liu, Bin Li, Xiaomeng Dai, Bo Lin, Yajie Sun, Peng Zhao, Minshan Chen, Rongping Guo, Shan Xin, Jingping Yun, Inmaculada Martínez-Reyes, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9ae85bd79cada9f97140c874baf065a8524c49a" target='_blank'>
              NF1 Loss Remodels Tumor Niches for Immune Evasion
              </a>
            </td>
          <td>
            Milad Ibrahim, Irineu Illa-Bochaca, Tara Muijlwijk, Ines Delclaux, K. S. Ventre, G. Jour, Paola Angulo Salgado, Shi Qiu, Agrima Dutt, Amanda W. Lund, Iman Osman, Markus Schober
          </td>
          <td>2026-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Greater understanding of differential therapeutic sensitivity, specifically to immunotherapy, in small-cell lung cancer (SCLC) is required. We explored SCLC heterogeneity through integrated molecular characterization of tumor tissue samples from 159 treatment-naive patients, utilizing genetic, epigenetic, transcriptional, and proteomic profiling, immunohistochemistry staining for multiple biologically relevant markers including transcriptional subtype-defining proteins, and spatial immune profiling using multiplex immunofluorescence. Multi-omics analysis confirmed high heterogeneity across/within neuroendocrine and non-neuroendocrine subtypes. Methylomics analysis identified four methylome clusters that may enhance subtype prediction, prognosis, and longitudinal monitoring of subtype evolution. Immunohistochemistry analysis showed high MHC-I expression in non-neuroendocrine subtypes, which have greatest potential benefit from adding immunotherapy to chemotherapy; high DLL3 expression associated with neuroendocrine subtypes and an immune-cold tumor microenvironment. Multiplex immunofluorescence demonstrated associations of MHC-I with spatial arrangement and phenotypic features of immune cells in the tumor microenvironment of high-MHC-I-expressing SCLC, providing mechanistic rationale for MHC-I as a potential biomarker of immunotherapy response. This multimodal profiling analysis provides further insights into the biologic complexity of SCLC and highlights potential therapeutic vulnerabilities of distinct disease subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7c241dbb9b8047ecee9d971f81cea0a1e193e3" target='_blank'>
              Multi-omic profiling provides insights into the heterogeneity, microenvironmental features, and biomarker landscape of small-cell lung cancer
              </a>
            </td>
          <td>
            Mingchao Xie, Miljenka Vuko, Shashank Saran, Siyu Liu, Andrew G Chambers, Hana Baakza, Helen K Angell, Felicia Ng, C. Gay, R. Cardnell, F. Segerer, Alma Andoni, Jaime Rodriguez-Canales, Paul M Waring, Markus Schick, J. C. Barrett, L. Byers, Giulia Fabbri
          </td>
          <td>2025-12-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Background : In contrast to most solid tumors, high immune cell infiltration in clear cell renal cell carcinoma (ccRCC) is associated with poor patient prognosis. The biological mechanisms underlying this paradox remain unclear, particularly regarding tumor cell– microenvironment interactions promoting local invasion and recurrence. This study aimed to identify spatially resolved tumor, immune, and stromal features that define aggressive phenotypes in localized ccRCC. Methods : Multiplex immunofluorescence was performed using a 33-marker panel on 1,728 multi-region tissue cores from 435 surgically treated patients with localized ccRCC. Samples systematically included tumor centers, invasive borders, and adjacent benign tissue. Single-cell analyses quantified immune, stromal, endothelial, and epithelial cell populations within their spatial context. Results : Spatially resolved profiling uncovered a highly aggressive tumor subtype distinguished by fibroblast activation protein (FAP) expression on tumor epithelial cells, a marker typically associated with stromal cells. Tumor-cell-specific FAP expression characterized an epithelial-to-mesenchymal transition (EMT)-like state and was spatially associated with profound immunosuppression, marked by enrichment of regulatory T cells, exhausted CD8+ T cells, and M2-like macrophages, particularly at the invasive border. Tumor-cell FAP promoted invasion and independently predicted significantly poorer recurrence-free survival (RFS), even in early-stage disease (multivariable Cox p = 0.022 for pT1–2), surpassing established biomarkers such as PD-L1 in capturing aggressive biological features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42470b1512d22d2033f3b6dc6829a3e316db8519" target='_blank'>
              Tumor cell FAP orchestrates EMT and immune suppression in aggressive localized ccRCC
              </a>
            </td>
          <td>
            T. Pellinen, L. Luomala, K. Mattila, A. Hemmes, K. Välimäki, Mariliina Arjama, Oscar Brück, L. Paavolainen, Elisa Kankkunen, H. Nísen, P. Järvinen, Leticia Castillon, Sakari Vanharanta, P. Vainio, Olli Kallioniemi, Panu M. Jaakkola, T. Mirtti
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background: Intratumour heterogeneity (ITH) is one of the key characteristics of cancer and is closely associated with patient prognosis, treatment resistance, and tumor metastasis. Nevertheless, the study of ITH in hepatocellular carcinoma (HCC) remains limited. Methods: The present study elucidated the influence of ITH on the tumor microenvironment (TME) in HCC. We applied Non-negative Matrix Factorization (NMF) analysis to a cohort of 78 single-cell RNA sequencing (scRNA-seq) HCC samples to systematically characterize ITH. Furthermore, by integrating spatial transcriptomics (ST) data from five HCC patients, we comprehensively analyzed the spatial organization and functional properties of distinct niches within HCC. We conducted a detailed analysis of the cell-type co-localization relationships within the TME and constructed a comprehensive atlas of HCC spatial organization. Results: We observed a co-localization relationship between hypoxia tumor cells, plasmalemma vesicle-associated protein (PLVAP+) endothelial cells (EC), and vascular endothelial growth factor A (VEGFA+) cancer-associated fibroblasts (CAF), suggesting a key role for hypoxia tumor cells in VEGFA+ CAF transformation and tumor angiogenesis. We identified a unique boundary region enriched with dendritic cells1 (DC1), interferon-expressing tumor cells, lymphatic EC, C–X–C Motif Chemokine Ligand 10 (CXCL10+) macrophages (Mac), and secreted phosphoprotein 1 (SPP1+) Mac located between the tumor-infiltrating immune cells and tumor regions. Furthermore, we found that CXCL10+ Mac and SPP1+ Mac, despite co-localizing in the boundary region, exhibit distinct functions, which may be attributed to their unique spatial locations, with the former being closer to the immune-infiltrated region and the latter more proximal to the tumor area. Conclusions: Our study highlights the critical role of spatial interactions between tumor cells and the microenvironment in HCC. The findings offer new insights into ITH and underscore the importance of spatial organization in understanding cancer biology and designing future precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e826ac6739dc6ba19b8b336aa7292a375efd8d3e" target='_blank'>
              Integrative Analysis of Single-Cell and Spatial Transcriptomics Reveals Intratumor Heterogeneity Shaping the Tumor Microenvironment in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-12-01</td>
          <td>Livers</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cervical cancer (CC), a leading cause of cancer-related deaths among women worldwide, is primarily driven by high-risk human papillomavirus (HPV) infections and comprises two major histological subtypes: adenocarcinoma (AC) and squamous cell carcinoma (SCC). Despite advances in prevention and treatment, the molecular and cellular heterogeneity of these subtypes poses significant challenges to achieving optimal clinical outcomes. Here, we integrate single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) to dissect the cellular and spatial heterogeneity of AC and SCC, uncovering distinct tumor microenvironment (TME) dynamics that underlie their divergent clinical behaviors. Our scRNA-seq analysis reveals that AC is enriched in epithelial cells, while SCC exhibits a more immunogenic TME with elevated plasma cells and NK/T cells. Spatial transcriptomics further highlights robust interactions between CD8 + T cells and epithelial subtypes in SCC, contrasting with the stromal-rich, immune-cold phenotype of AC. We identify subtype-specific immune and stromal features, including ICOS+ Tregs, IDO1+ cancer-associated fibroblasts (CAFs), and PLVAP+ endothelial cells, which may drive immune evasion, angiogenesis, and metastasis. These findings provide a comprehensive framework for understanding CC heterogeneity and offer actionable insights for developing subtype-specific therapeutic strategies, such as combining immune checkpoint inhibitors with stromal-targeting agents. This study underscores the potential of spatial multi-omics technologies to advance precision oncology and improve outcomes for cervical cancer patients. Single-cell and spatial transcriptomics uncovered a stark contrast in the TMEs of cervical cancer subtypes: squamous cell carcinoma is more immunogenic, whereas adenocarcinoma is stromal-rich and immune-cold.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b0c553f54a8eda4d3dc1abe42f3db69a41cfe1" target='_blank'>
              Single-cell and spatial transcriptomic profiling reveals distinct tumor microenvironment dynamics in cervical adenocarcinoma and squamous cell carcinoma
              </a>
            </td>
          <td>
            Yaqin Wu, Bei Wei, Zhentong Wei, Hongyan Wang, Yunyun Liu, Dongdong Xu, Xiaokang Li, Liyuan Guo, Huaiwu Lu, Huiling Shang
          </td>
          <td>2025-12-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ea01efdafe26d09e76545f3f6eadf396cb683a" target='_blank'>
              Integrative Single-cell and Spatial Transcriptomic Analysis of Osteosarcoma Reveals Conserved and Distinct Ecosystems Across Sites and Species
              </a>
            </td>
          <td>
            Yogesh Budhathoki, Matthew V. Cannon, T. A. McEachron, Anand G. Patel, Matthew J Gust, Jaime F Modiano, D.T. Ammons, Kathryn E Cronise, M. Macchietto, A. Sarver, Ruben Dries, B. Gryder, Daniel P. Regan, Heather L. Gardner, Ryan D. Roberts
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern, and a high relapse rate. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, where malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, we identify pericytes as the most active paracrine signaling hub within the tumor parenchyma. Their depletion through genetic engineering results in accelerated tumor progression and shortened survival. Mechanistic studies reveal that pericyte deficiency remodels the endothelium and impacts the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, the pericyte-deprived endothelium recruits perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages express Hepatocyte Growth Factor, which reinforces activation of its receptor tyrosine kinase MET on GBM cells harboring a pronounced mesenchymal subtype driven by the key phenotypic regulator Fosl1. Indeed, orthotopic implantation of MET-expressing GBM cells corroborates their superior tumor-initiating and invasive capabilities. Thus, pericytes represent critical modulators of GBM development by orchestrating a tumor-suppressive microenvironment, highlighting the importance of their preservation in therapy. Improved understanding of the tumor ecosystem in glioblastoma is critical for developing new treatment strategies for the disease. Here, the authors identify pericytes as an active paracrine signaling hub within the tumor parenchyma that orchestrates a tumor-suppressive microenvironment, suggesting pericyte preservation as a key feature of future therapeutic regimens for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c09a05790107e087a632df06714ceec3176e20b4" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, Mehrnaz Safaee Talkhoncheh, B. Phung, E. Cordero, R. Rosberg, E. Johansson, G. Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumors are composed of cellular populations with distinct genotypes and phenotypes, which dynamically evolve over time and during treatment. This process is known as clonal evolution, and it is difficult to reveal fine-scale clonal structure with traditional bulk sequencing. Although single-cell genome sequencing could enable reconstruction of tumor clonal evolution, it remains technically challenging and the number of single cells profiled is generally insufficient due to high cost. To address this issue, we developed scClone, a computational toolkit that integrates variant detection and genotype inference for single-cell RNA-seq (scRNA-seq) and spatial transcriptomic data. It further provides interactive visualization of clonal structure and dynamic evolution. scClone addresses key limitations inherent to scRNA-seq, such as expression drop-out and allelic imbalance, and incorporates cell type or state annotation with mutational signature analysis to enable comprehensive profiling of tumor heterogeneity. scClone demonstrated robust performance across multiple datasets—generated from both full-length and fragmented RNA sequencing—by accurately reproducing mutation profiles and resolving clonal mixtures in myeloma, hepatocellular carcinoma and pancreatic cancer. Additionally, scClone has been applied to spatial transcriptomics, enabling the delineation of clonal structures within histological sections from ovarian cancer and cutaneous squamous cell carcinoma. In summary, our results demonstrate that scClone can extract genetic information from scRNA-seq datasets, thereby successfully establishing genotype–phenotype associations at the single-cell level and providing insights into the clonal evolution of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d470f131086238e06c10a786c9bfc7fb86a41116" target='_blank'>
              De Novo Detection of Clonal Structure and Evolution in Single-Cell and Spatial Transcriptomes
              </a>
            </td>
          <td>
            Shihao Bai, Xianbin Su, Ziyao Chen, Ze-Guang Han
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafddf4eb7b4f74e9d7699b13e08df2afba97c1e" target='_blank'>
              Pan-cancer tumor classification by a holistic tumor microenvironment atlas
              </a>
            </td>
          <td>
            Shishang Qin, Xiao Du, Jinhu Li, Nan Jiang, Tian Diao, Yufei Bo, Qinhang Gao, Liangtao Zheng, Xinnan Ling, Qianqian Gao, Xiangjie Li, Sen Gao, Fei Tang, Wenjie Zhang, Chenwei Li, Peihong Fang, Linna Zhu, Dongfang Wang, Zemin Zhang
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Tumor heterogeneity drives drug resistance and relapse, influencing immune evasion and tumor progression. While intratumor heterogeneity has been extensively studied at the genomic level, its functional outcomes and interactions with the tumor microenvironment remain underexplored. In contrast, the functional outcome of heterogeneity and the interplay with the tumor microenvironment have not been addressed. In this study, we integrate multi-region spatial MS-based proteomics of 280 tumor regions, exome sequencing, and imaging to investigate spatial proteomic heterogeneity in breast cancer. Our findings reveal increased proteomic heterogeneity with tumor progression, independent of genomic heterogeneity but closely associated with microenvironmental differences. Integration with immune and stromal imaging highlighted a dynamic interplay where low-grade tumors exhibit constrained immune infiltration, and upon progression to higher grades, macrophages and T cells infiltrate. However, anti-inflammatory pathways involving kynurenine and prostaglandins are more highly expressed in infiltrated regions, suggesting that anti-tumorigenic activities are inhibited. Integration with the global protein network provides potential targetable mediators of immune evasion in breast cancer that can serve as the basis for future development of personalized breast cancer therapies. Intratumor heterogeneity (ITH) has been mainly studied at the genomic level. Here, the authors integrate multi-region proteomics of 280 tumor regions from 33 patients, exome sequencing, and imaging-based immune and stromal profiling to investigate functional spatial proteomic ITH in breast cancer and the interactions of cancer cells and the tumor microenvironment during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d48d5d89f33d4d9425895ba820536d951749d4" target='_blank'>
              Integrated spatial proteomic analysis of breast cancer heterogeneity unravels cancer cell phenotypic plasticity
              </a>
            </td>
          <td>
            M. Mardamshina, Shiri Karagach, Vishnu Mohan, Gali Arad, Daniela Necula, O. Golani, Liat Fellus-Alyagor, Anjana Shenoy, Kateryna Krol, Daniel Pirak, Nitay Itzhacky, I. Marin, Bruria Shalmon, Y. Addadi, Roded Sharan, E. Gal-Yam, Iris Barshack, Tamar Geiger
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Neuroblastoma (NB), the most common extracranial solid tumor in children, is characterized by remarkable cellular heterogeneity and clinical variability ranging from spontaneous regression to aggressive progression and relapse. Despite advances in multimodal therapies, including surgery, chemotherapy, radiotherapy, differentiation therapy, and immunotherapy—treatment resistance remains the principal barrier to improving survival in high-risk patients. Recent single-cell and spatial multi-omics studies have revolutionized our understanding of NB by revealing its developmental origins, lineage hierarchy, and adaptive evolution under therapeutic pressure. These technologies have delineated distinct cellular states along an adrenergic–mesenchymal continuum and uncovered the dynamic interplay between tumor cells and their microenvironment. Genetic instability, epigenetic reprogramming, and metabolic plasticity cooperate with immune and stromal remodeling to drive tumor persistence and relapse. At the molecular level, mechanisms such as MYCN-driven chromatin remodeling, super-enhancer reorganization, bypass signaling activation, quiescent persister programs, immune checkpoint engagement, and metabolic rewiring collectively enable therapeutic escape. Importantly, these processes are reversible, highlighting tumor plasticity as both a hallmark and a potential vulnerability of NB. Integrating single-cell transcriptomics, epigenomics, and spatial profiling provides an unprecedented framework to map resistance evolution, identify lineage-specific vulnerabilities, and guide rational combination strategies. Targeting epigenetic regulators, metabolic checkpoints, and immune suppressive networks in a temporally coordinated manner holds promise for converting NB from an adaptive to a controllable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance
              </a>
            </td>
          <td>
            Guoqian He, Sijia He, Xiao-yu Jing, Yi-Ling Dai, Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be57704ac8a67d3c2570360452cb391401b265a" target='_blank'>
              A Comprehensive Treatment-Induced Resistance Atlas of Glioblastoma Reveals a Fibrotic Niche Shielding the Tumor from Immunotherapy
              </a>
            </td>
          <td>
            Fei Wang, Run Huang, Hongyi Ling, Yuhan Bai, Chen Yang, Guozhen Zhao, Xin Wu, Wenqian Cao, Yue Lu, Yining Zhang, Deng-Feng Lu, Youjia Qiu, Juyi Zhang, Bixi Gao, Chao Ma, Hanhan Dang, Yulian Zhang, Yanbo Yang, Ting Sun, Zhouqing Chen, Zhong Wang
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/862b682dc4ca7fafbd8a3e9ea15acd9f6c5dbbca" target='_blank'>
              Single-cell phenomics through integrated imaging and molecular profiling
              </a>
            </td>
          <td>
            Johannes Bues, Joern Pezoldt, C. L. Lambert, B. D. Hale, Elisa Bugani, Ramon Vinas Torne, Tim Ferrari, Nadia Grenningloh, V. Gardeux, Shuo Wen, Caroline Wandinger, Maximilian Kohnen, Romina Augustin, Katharina Eckstein, Assia Ouanaya, Jillian R Love, Sarthak Saha, Amirhossein Saba, Aviv Huttner, Maria Vittoria Impagliazzo, Jose Antonio Vasquez Porto Viso, Angel de Jesus Corria Osorio, D. Psaltis, Wouter R Karthaus, B. Snijder, Maria Brbić, Bart Deplancke
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bdb3fc96051ec080fed6bcf88bde02391e145c9" target='_blank'>
              Integrative spatial multi-omics reveals prognostic tumor niches in female genital tumors
              </a>
            </td>
          <td>
            Tian Xu, Siyu Lin, Yuling Wu, Yufeng He, Zongxu Zhang, Yanping Zhao, Peng Zhang, Luyang Zhao, Wen Yang, M. Ye, Rui Zhang, Liang Wen, Pengfei Ren, Ce Luo, Zhe Zhang, Zexian Zeng, Yuanguang Meng
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb72dd83254c43d1127b19ea69b553e5fd51c4d" target='_blank'>
              TMPO promotes cellular dissemination and metastasis in circulating tumor cells
              </a>
            </td>
          <td>
            A. Giacobbe, Aleksandar Z Obradovic, Jinqiu Lu, Soonbum Park, Carlos Pedraz-Valdunciel, G. Fanelli, Aunika Zheng, Jaime Y Kim, Maya Stella Dixon, Jung Seung Nam, Florencia Picech, Caroline J. Laplaca, R. Virk, Matteo Di Bernardo, Alexander Chui, J. M. Arriaga, Stephanie N Afari, Francisca Nunes de Almeida, Min Zou, Helen Garcia, Brian D. Robinson, Hongshan Guo, S. Maheswaran, D. Haber, D. Miyamoto, D. Nanus, S. Tagawa, Tian Zheng, Massimo Loda, I. I. C. Chio, Michael M. Shen, P. Giannakakou, Andrea Califano, Peter A. Sims, Cory Abate-Shen
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Identifying drivers of cancer progression to guide treatment selection is hindered by our limited understanding of tumor heterogeneity and its impact on tumor evolution. Here, we delineate the phenotypic variability across ~300,000 cells collected from multiple tumor loci in primary prostate and matched locoregional metastases using single-cell chromatin accessibility and gene expression sequencing. We find inter-patient heterogeneity to be confined to malignant populations. Within individual tumor loci, we see phenotypic heterogeneity among malignant cell populations despite a shared clonal genotypic architecture. We also observe that malignant cell populations disseminating to locoregional lymph nodes mirror the clonal architecture and phenotypic heterogeneity across primary tumor loci, while shifting from canonical prostate-cancer states to non-canonical inflammatory-like states. Our findings suggest a bottleneck imposed during the dissemination process, funneling prostate cancer cells toward an inflammatory-like cell state. These insights into the interplay between phenotypic identity and clonal architecture refine our understanding of prostate cancer progression and suggest that convergence of cancer cells towards an inflammatory-like state underlies dissemination to lymph nodes, offering a critical framework for future studies into prostate cancer metastatic potential. Understanding tumor heterogeneity and its impact on prostate cancer progression remains elusive. Here, single nucleus snATAC and snRNA sequencing of a multi-loci sampled cohort of advanced prostate cancer patients identifies an inflammatory-like state underlying metastatic dissemination to the lymph nodes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b9d84066de3071bee8ea7d75cdd27aba04e0e02" target='_blank'>
              Prostate cancer cells converge to an inflammatory-like state upon metastatic dissemination
              </a>
            </td>
          <td>
            Tina Keshavarzian, K. Furlano, Giacomo Grillo, Lisanne Mout, C. Arlidge, Faizan Hasan, Ankita Nand, Migle Mikutenaite, Evdoxia Karadoulama, Ashish Goyal, Elisabeth L Pezzuto, Jessica Heilmann, Jakub Sykora, S. Minner, Thorsten Schlomm, G. Sauter, R. Simon, H. He, Joachim Weischenfeldt, C. Plass, Clarissa Gerhäuser, M. Lupien
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb4486dfc086ad91b487a1c5269e719293310a3" target='_blank'>
              Machine perfusion and single-cell spatial transcriptome mapping identifies novel immune escape mechanisms in colorectal cancer liver metastasis
              </a>
            </td>
          <td>
            Peter Kok-Ting Wan, Ranchu Cheng, David Johnson, Carl Lee, Shihong Wu, Areeb Mian, Ahmet Hazini, Sorayya Moradi, Kate Friesen, Flurin Caviezel, Syed Hussain Abbas, Hatem Sadik, A. Lakha, Keaton Jones, Girishkumar Kumaran, M. Bottomley, Rob Jones, Mark C. Coles, R. Bashford-Rogers, Constantin C. Coussios, Len Seymour, Robert Carlisle, Kerry Fisher, Alex Gordon-Weeks
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, M.-R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICI) have demonstrated prolonged efficacy in certain melanoma patients, yet a significant portion of patients do not experience clinical improvement, with the mechanisms underlying this resistance still not fully understood. Using established cell markers, we partitioned the single‐cell transcriptome into clusters, finding a notable link between NK cells and patient response to immunotherapy. We further identified four distinct subpopulations of NK cells, profiling marker gene sets and unique biological functions associated with each subpopulation. This analysis provides insights into the trajectory of NK cell development and differentiation, along with identifying the transcription factors driving these processes. The study pinpointed NK cluster 01 as pivotal in influencing patient sensitivity and prognosis during immunotherapy. Single‐cell transcriptome and spatial transcriptomics (ST) analysis revealed the proximity of NK cluster 01 cells to melanoma cells, hinting at a potential regulation of cell‐cell interaction via the IFN‐II signaling pathway network. ST analysis revealed the spatial arrangement and interaction of NK cluster 01 cells with melanoma cells. This study explores the feasibility of targeting NK cluster 01 cells with small molecule drugs via molecular docking, offering a promising approach to bolster the clinical utility of NK cell therapy. We comprehensively analyze the heterogeneity of NK cells within melanoma, elucidate the potential regulatory interactions between NK cells and other microenvironmental components, and establish a basis for the future clinical utilization of distinct NK cell subsets as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fbdc1aaba32b7600dd3221ae61c5e31ee96bd3" target='_blank'>
              Integrating Single‐Cell and Spatial Transcriptomics Reveals NK Cell Subpopulations Associated With Immunotherapy for Melanoma
              </a>
            </td>
          <td>
            Zhicheng Hu, Yongfei Chen, Hao Yang, Qiuming Pan, Hongrui Li, Shuting Li, Junxi Wang, Yudi Huang, Guanglong Huang, Shanqiang Qu
          </td>
          <td>2025-12-01</td>
          <td>Smart Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Bladder cancer (BCa) is a prevalent malignancy and major cause of cancer-related mortality in men, with clinical outcomes still varying despite advances in personalized treatments. High inter- and intra-tumor heterogeneity significantly contributes to this variability. While traditional high-throughput sequencing has provided insights into BCa mechanisms, driver genes, and clinical strategies, it falls short in completely elucidating cellular heterogeneity. Recently, single-cell sequencing (SCS) technologies have substantially enhanced the detection of tumor heterogeneity by improving sensitivity, accuracy, and efficiency. Single-cell transcriptome sequencing offers unbiased, high-resolution analysis of gene expression patterns at the single-cell level, offering essential insights into BCa pathogenesis. This article reviews advancements in SCS technology and its applications in evaluating tumor heterogeneity, the tumor microenvironment, metastasis, and treatment resistance, offering new perspectives for future BCa research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/128cca3d1edd8a7877ad22a89e79521ff40bea75" target='_blank'>
              Single-cell sequencing in bladder cancer: new insights from tumor cell diversity to individualized treatment strategies
              </a>
            </td>
          <td>
            Cen Liufu, Shuai Ye, Cong Yin, Tao Zhu, Jiahao Jiang, Yan Wang, Bentao Shi
          </td>
          <td>2025-12-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction Breast cancer (BC) prognosis is fundamentally dictated by its pronounced molecular and cellular heterogeneity. While single-cell RNA sequencing (scRNA-seq) has detailed the tumor microenvironment (TME) cellular composition, a critical knowledge gap persists regarding the vascular heterogeneity of tumor endothelial cells (TECs) across different BC subtypes. Current TEC classification remains conventional, obscuring whether subtype-specific TECs possess unique transcriptomic signatures that influence treatment response, particularly to anti-angiogenic therapies. Given the pivotal role of tumor vasculature in proliferation and metastasis, and the existing research bias towards Triple-Negative BC (TNBC), a comprehensive investigation into the complexity of the vascular system is warranted. Furthermore, as lymph node metastasis (LNMT) is a major determinant of cancer mortality, distinguishing the endothelial and immune cell subtypes within LNMT microenvironments from primary tumors is essential for clarifying pathogenesis. Methods To address this, we integrated scRNA-seq data from primary BC tumors, ER_LN and healthy tissues, generating a robust dataset of 98,000 cells (~ 12 samples). Employing single-cell transcriptomics, spatial transcriptomics, and immunohistochemistry, we precisely delineated the endothelial cell (EC) heterogeneity within matched tumor and peri-tumoral tissues and scrutinized their intricate interactions with immune populations. Results We identified two previously uncharacterized, tumor-enriched endothelial cell subtypes, designated EC4 and EC5, which demonstrate subtype-specific functional adaptations and prognostic significance. EC4 cells, highly prevalent across BC, are principally characterized by antigen presentation, immune cell recruitment, and pro-inflammatory signaling. Conversely, EC5 cells, also enriched in BC, exhibit robust extracellular matrix (ECM) remodeling and potent tumor angiogenesis. We further characterized the functional divergence of EC4 and EC5 within ER tumors relative to HER2 tumors and ER_LN metastases. Our analysis reveals conserved endothelial programming mechanisms across BC subtypes, coexisting with distinct TME-driven transcriptional adaptations. Importantly, interactome analysis highlighted novel and subtype-specific communications between these novel EC subsets and immune cells, particularly CD8+T cells and macrophages. Experimental validation demonstrates that ECs overexpressing APP can mediate the M2 polarization of macrophages, underscoring diverse immunomodulatory roles for EC subsets across different BC contexts. Conclusions These findings offer critical and granular insights into the complex interplay between novel EC subtypes and the immune microenvironment in BC progression and metastasis. We establish that ECs are active and heterogeneous modulators of the TME, identifying specific therapeutic vulnerabilities within the tumor vasculature and providing a foundational blueprint for developing future precision immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67936270be24af1f65608417190d3a4e352d6fe3" target='_blank'>
              Single-cell profiling identifies heterogeneity of the immune microenvironment in healthy, primary and lymph node metastatic BC
              </a>
            </td>
          <td>
            Chuyu Liu, Feng Zeng, Shanshan Gao, Nengying Zhang, Yulan Cai
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, Christin-Therese Müller, Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faf55c0c7d18000c8ba092357d0dd95271e7c35c" target='_blank'>
              Spatial dissection of ADC/RPT targets defines therapeutic opportunities in rhabdoid tumors
              </a>
            </td>
          <td>
            N. Reitsam, V. Fincke, Maria Daniela Hernandez Ramirez, M. Mucha, Eva Sipos, Lisa Siebenhüter, J. Enke, Maurice Loßner, C. Vokuhl, Constantin Lapa, M. Hasselblatt, Michael C. Frühwald, B. Märkl, Pascal D. Johann
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the leading causes of cancer-related morbidity and mortality worldwide. Tumor sidedness, distinguishing left- (LCRC) and right-sided (RCRC) cancers, has emerged as a critical clinical determinant, influencing patient prognosis and therapeutic response. However, the cellular and molecular mechanisms underlying these differences remain poorly understood. Recent advances in single-cell RNA sequencing (sc-seq) provide high-resolution insights into CRC heterogeneity, revealing distinct tumor, immune, and stromal cell populations and their context-specific interactions. In this review, we synthesize sc-seq studies that dissect the molecular programs driving progression, therapy resistance, and metastasis in CRC. We highlight malignant subclusters characterized by metabolic reprogramming and spatially organized oncogenic signaling; specialized immune cell states, including macrophage subsets, exhausted T cells, and mast cells, that shape tumor immunity; and stromal elements such as cancer-associated fibroblasts and endothelial tip cells that remodel the extracellular matrix, promote angiogenesis, and foster immune evasion. Importantly, sc-seq demonstrates that LCRC and RCRC represent distinct multicellular ecosystems with differential immune recruitment and stromal signaling, underscoring the need for sidedness-informed therapeutic strategies. We propose that future interventions should target cell-cell communication networks and spatially defined tumor–microenvironment interactions to overcome heterogeneity and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03b9e3bca233f1317bcd2e2e43fb69a5fedfb984" target='_blank'>
              Sidedness matters: single-cell perspectives on left- and right-sided colorectal cancer
              </a>
            </td>
          <td>
            M. Yanova, Diana Maltseva, Alexander Tonevitsky
          </td>
          <td>2025-12-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor cell plasticity plays a key role in the progression of malignant neoplasms and therapy resistance, yet its spatial dynamics remain poorly understood. This study investigates the spatial distribution of epithelial–mesenchymal transition (EMT) and stemness, and their prognostic relevance in lung adenocarcinoma (LUAD). Single‐cell RNA sequencing was performed on 10 tumor fragments from three regions (core, core‐adjacent, and edge) of a single LUAD using a SURFSeq 5000 platform. Stemness potential was identified by CytoTRACE2, EMT by UCell and the Hallmark_epithelial_mesenchymal_transition signature, and tumor cells associated with disease prognosis by Scissor. LUAD displayed pronounced spatial heterogeneity of tumor cell plasticity. Stemness potential was heightened at the edge, while EMT was most prominent in the core. The correlation between EMT and stemness was slightly higher at the edge, though it remained negligible (R = 0.17, p = 5.5e−7). Higher EMT scores were observed in tumor cells associated with poor overall survival (r = 0.433, p < 2e−16), whereas stemness was prevalent in tumor cells associated with poor progression‐free survival (r = 0.251, p = 7.15e−6). Tumor cells associated with poor prognosis were enriched with the MYC‐targets v1 signature with PCBP1 and PA2G4 as the top contributing genes. Immunohistochemical data showed that PCBP1 and PA2G4 proteins predominantly localized within tumor cells of LUAD. Taken together, these findings highlight the spatial heterogeneity of tumor cell plasticity features in LUAD and uncover biomarkers associated with poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badf7545e441ad3330c93dcdb325456db2158597" target='_blank'>
              Spatial Dynamics of Tumor Cell Plasticity in Lung Adenocarcinoma Revealed by Region‐Specific Single Cell Transcriptomics
              </a>
            </td>
          <td>
            A. Khozyainova, Maxim E. Menyailo, Vera G. Subrakova, Dmitry M. Loos, P. Iamshchikov, Anton A Fedorov, M. S. Tretyakova, Ustinia A Bokova, Evgeny O Rodionov, Sergey V Miller, Evgeny B Topolnitsky, D. V. Piankov, Sergei A Korostelev, Evgeny V. Denisov
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a heterogeneous malignancy shaped by genetic alterations and immune microenvironmental interactions. While single-cell RNA sequencing (scRNA-seq) has uncovered diverse epithelial and T cell subsets, the spatial architecture of their crosstalk remains poorly understood. We integrated scRNA-seq and spatial transcriptomics data from 36 CRC patients to comprehensively characterize epithelial and T cell heterogeneity, differentiation dynamics, and intercellular communication networks. Data preprocessing and integration were conducted using Seurat and Harmony. Cell trajectory inference was performed via CytoTRACE and Slingshot. Ligand-receptor signaling was assessed using CellChat, while spatial mapping was achieved using CellTrek. Nine epithelial and eight T cell subpopulations were identified, each exhibiting distinct transcriptional states, CNV burdens, and pseudotime trajectories. CD8⁺ T_GZMK cells displayed tumor-specific activation signatures and engaged in enriched communication with epithelial subsets through CEACAM, APP, and MIF signaling pathways. Spatial transcriptomics confirmed the in situ colocalization of CD8⁺ T_GZMK cells with PTP4A3⁺ epithelial cells in tumor regions, revealing spatially organized epithelial–immune niches. These interactions suggest potential mechanisms of immune modulation and tumor progression. This study provides an integrative single-cell and spatial atlas of CRC, revealing structured epithelial–T cell communication and spatial architecture within the tumor microenvironment. Our findings offer novel insights into immune-epithelial crosstalk and identify signaling pathways that may serve as therapeutic targets or biomarkers for CRC precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6c4f2ec4971acec7d7b9addcd8348e71b0ec1ff" target='_blank'>
              Decoding epithelial-T cell interactions in colorectal cancer through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Rentao Yu, Xinyu Lu, Zhuangzhuang Yue
          </td>
          <td>2025-11-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Single-cell sequencing (scRNA-seq) has emerged as a pivotal technology for deciphering the complex cellular heterogeneity and tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC), positioning it as a critical tool for informing novel therapeutic strategies. This review explores how scRNA-seq reveals diverse cellular subpopulations and their functional roles within the PDAC TME, including malignant epithelial cells with transitional phenotypes, heterogeneous cancer-associated fibroblasts (CAFs), functionally distinct immune cells such as tumor-associated neutrophils (TANs) and macrophages (TAMs), and actively participating neural components like Schwann cells. These cellular constituents form specialized functional units that drive tumor progression, immune evasion, neural invasion, and therapy resistance through metabolic reprogramming, immunosuppressive signaling, and cellular plasticity. The review further examines technological advances in single-cell sequencing from 2023 to 2025, focusing on sample preprocessing innovations, multi-omics integration (combining transcriptomics with epigenomics and proteomics), spatial resolution enhancements, and customized computational tools that address PDAC-specific challenges. Clinically, single-cell sequencing enables precise cellular subtyping, identification of novel biomarkers, and development of personalized therapeutic approaches, including combination therapies targeting specific cellular subpopulations and their interactions. Despite these advances, significant challenges remain in standardizing clinical applications such as liquid biopsy for early detection and tumor microenvironment assessment for diagnostic staging, validating biomarkers like CLIC4, GAS2L1, Cytokeratins, Vimentin and N-cadherin in circulating tumor cells, and comprehensively integrating multi-omics data. Future research focusing on both technology refinement and biological validation will be essential for translating single-cell insights into improved diagnostic and therapeutic outcomes for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9576dbb6fe77bb5f04f2ad67b474589557936" target='_blank'>
              Single-Cell Sequencing Unravels Pancreatic Cancer: Novel Technologies Reveal Novel Aspects of Cellular Heterogeneity and Inform Therapeutic Strategies
              </a>
            </td>
          <td>
            Keran Chen, Zeyu Chen, Jinai Wang, Mo Zhou, Yun Liu, Bin Xu, Zhi-Jing Yu, Yiming Li, Guanhu Yang, Tiancheng Xu
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The global incidence of basal cell carcinoma (BCC) is steadily increasing. Although immunotherapy has emerged as a treatment option, its efficacy remains limited. This underscores the need for further investigation of immune remodeling mechanisms in BCC. Here, we aimed to characterize fibroblast subsets and their interactions with other components of the tumor microenvironment in BCC. We performed single-cell RNA sequencing on nine surgical specimens from six patients, including BCC tumors and adjacent normal tissues. Key findings were further validated using spatial transcriptomics and multiplex immunohistochemistry. We identified the C01_TNC fibroblast cluster as a population of cancer-associated fibroblasts (CAFs). These cells exhibited prominent peritumoral infiltration and were characterized by matrix-associated and inflammatory gene expression profiles. We propose that C01_TNC CAFs contribute to tumor progression, possibly by upregulating collagens that engage integrins on tumor cells, thereby facilitating extracellular matrix remodeling and invasion. Furthermore, C01_TNC CAFs were spatially associated with Tregs and expressed ligands that interact with Treg receptors, suggesting a role in enhancing immunosuppression. Notably, this subset was significantly enriched in peritumoral regions and could establish an immune barrier favoring tumor progression. Our findings shed light on the role of fibroblasts in orchestrating interactions between tumor and immune cells in BCC. This enhanced understanding of the tumor microenvironment may inform novel therapeutic strategies to improve treatment outcomes for patients with BCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254816fb1662c33060d795673833d2ef5dfef42d" target='_blank'>
              Single-cell and spatial transcriptomics reveal TNC-positive cancer-associated fibroblasts that mediate immunosuppression and promote tumor progression in basal cell carcinoma
              </a>
            </td>
          <td>
            Min Luo, Wei Tian, Qianwei Zhuo, Zhenjian Zou, Jinger Lin, Lihang Lin, Rong Li, Yue Han
          </td>
          <td>2025-11-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a05de91b68f189c5a7a8fad28c43d535bcdc834" target='_blank'>
              A spatial atlas of colorectal cancer reveals the influence of stromal niches on tumour differentiation
              </a>
            </td>
          <td>
            Andrew D. Pattison, Rebekah M. Engel, Wing Hei Chan, Spencer Greatorex, David Nickless, Liam Skinner, Camilla Cohen, Julie Hickey, Christine Georges, Anne L. Fletcher, P. McMurrick, Thierry Jardé, Lochlan Fennell, Helen E. Abud
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Introduction Glioblastoma (GBM) is a highly aggressive brain tumor with a median survival of only 15 months. A major challenge in GBM management is the pronounced inter- and intratumoral heterogeneity, which complicates prognosis and therapy. Radiomics, the quantitative extraction of features from medical images, can capture this heterogeneity across the entire tumor volume, but the biological basis of radiographic phenotypes remains poorly understood. Methods We integrated preoperative MRI-based radiomic stratification with multi-platform transcriptomics (bulk RNA-seq, single-cell RNA-seq, and spatial transcriptomics) in IDH-wildtype GBM patients. Unsupervised clustering of radiomic features identified four imaging subtypes. Results Group 4 emerged as a high-risk subtype associated with significantly worse survival and a distinctive MRI pattern of peripheral contrast enhancement. Transcriptomic analyses revealed that Group 4 tumors were enriched in cell-cycle and proliferation markers and exhibited neural stem cell–like gene expression signatures. Single-cell profiling confirmed an elevated proportion of stem-like malignant cells in this subtype. Spatial transcriptomics further demonstrated that these proliferative, stem-like programs were localized predominantly to the tumor periphery, corresponding to the rim-enhancing regions on MRI. Finally, we identified the developmental transcription factor VAX2 as a candidate driver of the Group 4 gene network; functional assays showed that VAX2 promotes GBM cell proliferation in vitro. Discussion Our findings link a radiomics-defined MRI phenotype to specific molecular programs and cell populations in GBM, suggesting that radiomic subtypes can serve as noninvasive biomarkers of tumor biology and highlighting potential therapeutic targets in aggressive, stem-like tumor cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63325a32d6106e3ca9f5bff7f21292a347c5da7b" target='_blank'>
              MRI-based radiomic clustering identifies a glioblastoma subtype enriched for neural stemness and proliferative programs
              </a>
            </td>
          <td>
            Zhongyi Zhang, Yang Liu, Zhicong Zhang, Tian Gui, Youhui Chen, Qian Chen, Xiufu Wu, Li Sun, Sujie Li, Shuyang Wei
          </td>
          <td>2025-11-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cellular plasticity, the ability of cells to dynamically alter their phenotypes, is a key driver of tumor evolution. This process is a hallmark of cancer which enables the acquisition of malignant traits, leading to metastasis, progression, and therapy resistance. It is governed by cell‐intrinsic factors, such as genomic instability and epigenetic reprogramming, and extrinsic stimuli from the tumor microenvironment. However, a unified framework is still needed to position plasticity as the central process that links these drivers to diverse cancer hallmarks. In this review, we first explore how plasticity enables key steps of tumor evolution, including tumorigenesis, metastasis driven by epithelial–mesenchymal plasticity (EMP), therapy resistance, and cancer stem cell (CSC) dynamics. We then summarize the intrinsic and extrinsic mechanisms that govern this adaptability. Finally, we discuss clinical advances in monitoring and targeting plasticity and highlight how new spatiotemporal technologies can address current research challenges. This review provides a framework positioning cellular plasticity as a central mechanism in cancer evolution, connecting its fundamental drivers to clinical translation. By synthesizing the latest advances, we offer perspectives for developing therapies that integrate prediction, monitoring, and targeting of plasticity to proactively guide cancer evolution toward manageable outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683e6aa0ee59731275431238ec93cc43ffca8292" target='_blank'>
              Tumor Cell Plasticity in Cancer: Signaling Pathways and Pharmaceutical Interventions
              </a>
            </td>
          <td>
            Shangwei Sun, Yunwei Sun, Ling Lan, S. Luan, Jin Zhou, Jiehui Deng, Yong Yuan, Zhong Wu
          </td>
          <td>2025-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The application of single-cell transcriptomic approaches has deepened our understanding of tumor heterogeneity, immune dynamics, and molecular programs underlying therapy response. The recent development of fixation-compatible single-cell platforms, such as 10x Genomics Flex, offers the opportunity to profile archived formalin-fixed, paraffin-embedded (FFPE) specimens, expanding access to clinically valuable samples. However, most benchmarking studies of recent single-cell RNA sequencing (scRNA-seq) technologies have relied on peripheral blood mononuclear cells, limiting their relevance to human tissues. Here, we compared three 10x single-cell RNA profiling methods, fresh tumor scRNA-seq, flash-frozen single-nucleus Multiome, and FFPE single-nucleus Flex (snFlex), using clear cell renal cell carcinoma (ccRCC) biopsies. Across methods, we observed broadly consistent cell type-specific transcriptional profiles among major ccRCC cell populations. Despite lower gene and UMI counts, snFlex reliably identified fine-grained states within CD8+ T cells, tumor-associated macrophages, and tumor compartments, comparable to those detected by scRNA-seq. Together, these findings highlight the distinct advantages of each technology depending on sample preservation type and study design, providing practical guidance for single-cell RNA profiling technology selection in translational studies using human tumor biopsies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0ba0d723e0604e91072fa9745f53e634a7a5814" target='_blank'>
              Evaluation of single-cell RNA profiling technologies using FFPE, fresh, and frozen ccRCC tumor specimens
              </a>
            </td>
          <td>
            Sherin Xirenayi, Sabrina Y. Camp, Amanda E. Garza, Yashika Rustagi, Erica Pimenta, Erin Shannon, Anwesha Nag, Adam Betherman, Anthony Anselmo, Rachel Trowbridge, Aaron R. Thorner, Jihye Park, Kevin Bi, E. V. Van Allen
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Thyroid cancer, the most common endocrine malignancy, is characterized by a unique and complex tumor microenvironment (TME). To unravel the high tumor heterogeneity and molecular mechanisms driving cancer progression, we performed single-cell RNA sequencing (scRNA-seq) analysis, enabling a comprehensive exploration of cellular diversity and molecular dynamics at single-cell resolution. We employed Principal Component Analysis (PCA) and Uniform Manifold Approximation and Projection (UMAP) for dimensionality reduction and subsequent identification of cellular clusters. Differential gene expression analysis across subclusters was conducted using the FindAllMarkers function, while the DoHeatmap function was utilized to visualize the distribution of differentially expressed genes. The AUCell algorithm was applied to evaluate pathway enrichment within specific cell subtypes. To decipher cellular communication networks, we integrated the CellChat and NicheNet algorithms, which revealed intricate intercellular signaling interactions. Finally, multiplex immunohistochemistry (mIHC) was performed to validate key cellular interactions identified in silico. By analyzing 405,077 single cells from 50 thyroid cancer samples (including papillary, anaplastic, and metastatic tumors) and 14 normal thyroid tissues, we identified four major cellular subpopulations through unbiased clustering based on gene expression patterns and representative cellular markers. The TME was found to encompass diverse immune, endothelial, and mesenchymal cell subtypes, including novel populations such as CD4 + HSPA1A + T cells. Functional pathway enrichment analysis highlighted the roles of abundant cell types in tumor progression. Cell-cell communication analysis uncovered potential immunotherapeutic targets and revealed critical crosstalk among hub niche cells, including APOE+ macrophages, EMT-like cancer-associated fibroblasts (CAFs), and RBP7+ endothelial cells. These findings were further validated by multiplex immunohistochemistry, confirming the spatial organization and interactions of these cell populations within the TME. Our study provides a comprehensive single-cell transcriptomic atlas of thyroid cancer, offering profound insights into tumor heterogeneity, the functional roles of key niche cells, and potential biomarkers for anticancer therapy. These findings not only enhance our understanding of thyroid cancer biology but also pave the way for the development of novel therapeutic strategies targeting the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b009366b4ed17ce1b669fc0314c882a610b52822" target='_blank'>
              Comprehensive single-cell RNA analysis reveals intertumoral microenvironment heterogeneity and hub niche of carcinogenesis in thyroid cancer
              </a>
            </td>
          <td>
            Hao Xu, Junjie Ma, Nanjun Li, Haoxiang Xuan, Shijie Lin, Yunuo Shen, Lu Yin, Ping Wang, Qianhui Xu
          </td>
          <td>2025-11-26</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumor heterogeneity is a fundamental property of leukemia and lymphoma that drives disease progression, relapse, and therapy resistance. For decades, bulk genomic and transcriptomic analyses have provided a foundational understanding of these malignancies but have obscured the cellular diversity within the tumor ecosystem. The advent of high-throughput single-cell technologies, particularly single-cell RNA sequencing (scRNA-seq), has revolutionized our capacity to deconvolute this complexity. This review synthesizes how single-cell analyses are reshaping our understanding of tumor heterogeneity in hematological cancers. We explore the role of intra-clonal genetic diversity in Acute Myeloid Leukemia (AML) and B-cell Acute Lymphoblastic Leukemia (B-ALL), the functional and transcriptional plasticity of subpopulations, and the dynamic interplay between malignant cells and the immune microenvironment in both leukemia and lymphoma. We further highlight how the tumor microenvironment in lymphomas fosters heterogeneity and immune evasion. Furthermore, the integration of single-cell data with spatial transcriptomics and computational models is beginning to predict evolutionary trajectories, offering a window into future clinical behaviors. Critically, we discuss how single-cell approaches have identified pre-existing or emergent resistant subclones under therapeutic pressure, providing a mechanistic basis for treatment failure. The clinical implications are profound, paving the way for single-cell diagnostics for minimal residual disease (MRD) monitoring and the identification of novel therapeutic targets. We conclude that single-cell multi-omics is an indispensable tool for unraveling the multifaceted nature of tumor heterogeneity, with the potential to propel the field toward more precise and effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0baf5679635e6fcbb86a03a048b12087069db02" target='_blank'>
              Tumor Heterogeneity in Leukemia and Lymphoma: Insights from Single-Cell Analyses
              </a>
            </td>
          <td>
            Pang Hou, Wen
          </td>
          <td>2025-11-21</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND Abdominal aortic aneurysm (AAA) is characterized by sterile inflammation, immune cell infiltration, and stromal remodeling that progressively weaken the aortic wall, leading to life-threatening aortic rupture. The molecular mechanisms and spatial organization of immune–stromal interactions in human tissue are poorly understood, limiting the potential to develop effective pharmacological therapy for AAA. METHODS In this observational cross-sectional study, formalin-fixed, paraffin-embedded tissues from 11 AAA patients and 12 controls were analyzed by Xenium spatial transcriptomics. Cellular states and localization within tissue architecture were mapped to identify cellular neighborhoods and infer cell–cell communication. RESULTS We generated a high-resolution spatial transcriptomics atlas of 581,664 cells in 26 clusters. AAAs showed a significant loss of contractile smooth muscle cells, expansion of pro-angiogenic endothelial subsets, and broad infiltration of immune cells. These inflammatory changes were accompanied by expansion of activated, universal, and CXCL12⁺ adventitial fibroblasts. Spatial transcriptomic analysis revealed fibroblast–immune colocalization and adventitial tertiary lymphoid organs. Inferred signaling pathway analysis identified increased interactions between CXCL12⁺ fibroblasts and CXCR4⁺ T and B cells in the adventitia of AAAs. Fibroblasts that expressed CXCL12 had significantly more immune cell neighbors than fibroblasts that did not, suggesting that they serve as stromal hubs for adaptive immune clustering. Genome-wide association analysis linked AAA heritability to fibroblasts, modulated smooth muscle cells, and foamy macrophages. CONCLUSION Our novel high-resolution spatial transcriptomic atlas of human AAAs revealed coordinated pathogenic reprogramming of stromal and immune cells, defined by smooth muscle cell depletion, fibroblast activation, endothelial remodeling, and disproportionate expansion of immune cells. Through CXCR4 signaling, CXCL12⁺ fibroblasts serve as central organizers of immune niches, suggesting stromal–immune crosstalk as a therapeutic target in AAA. CLINICAL PERSPECTIVES What Is New? We generated the first subcellular-resolution spatial transcriptomic atlas of human abdominal aortic aneurysm (AAA), with >580,000 cells identified from aortic tissue sections We identified CXCL12+ fibroblasts as central stromal hubs that organize adaptive immune niches through CXCR4-mediated crosstalk with B and T cells We discovered that stromal populations carry the strongest genetic enrichment for AAA risk, notably fibroblast and modulated smooth muscle cell populations What Are The Clinical Implications? These findings position stromal-immune interactions, particularly the CXCL12-CXCR4 axis, as a potential therapeutic target to slow AAA progression The spatial atlas provides a framework for mechanistic studies and drug-discovery efforts, guiding future interventions aimed at modifying the microenvironment that destabilizes the aneurysmal aortic wall">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dc4c75342cbb8a240b73ba078f4ae6edca40173" target='_blank'>
              Spatial Transcriptomics Reveals CXCL12+ Fibroblasts as Central Immune Organizers through CXCR4 Signaling in Abdominal Aortic Aneurysm
              </a>
            </td>
          <td>
            Dina Levy-Lambert, Joel L. Ramirez, Saba Shaikh, Cesar De Jeronimo Diaz, A. G. Huang, Alexis Combes, G. Fragiadakis, Trevor P. Fidler, Adam Oskowitz
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive adult brain tumor, marked by intratumoral heterogeneity and therapy resistance. Metabolic reprogramming through histone lactylation has been linked to tumor progression and immune suppression. However, the spatial and single-cell landscape of lactylation in GBM and its prognostic significance remain poorly understood. We employed a multi-omics approach integrating bulk RNA sequencing, single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics to investigate lactylation-related signatures in GBM. Differential expression and pathway analyses were performed using GEO and TCGA datasets. Cell clustering, SCENIC transcriptional network inference, CellChat intercellular communication modeling, and pseudotime analysis were conducted. A prognostic risk model was constructed using LASSO-Cox regression based on lactylation-associated genes. Experimental validation was performed using western blotting, immunohistochemistry, and functional assays in GBM cell lines. Lactylation-related genes were significantly upregulated in GBM and associated with poor prognosis and immunosuppressive tumor microenvironments. Single-cell analysis revealed high-lactylation malignant subpopulations enriched in hypoxic tumor cores, exhibiting metabolic reprogramming and enhanced immune evasion. Spatial transcriptomics confirmed the localization of S100A6-high-lactylation GBM cells in aggressive tumor regions. A nine-gene lactylation-based risk model stratified patients into high- and low-risk groups with significantly different survival outcomes (AUC: 0.77–0.87). Experimental knockdown of S100A6 reduced GBM cell proliferation, migration, and invasion. Lactylation defines distinct tumor cell clusters in GBM that are spatially localized, metabolically reprogrammed, and immunosuppressive. The S100A6-associated lactylation signature serves as a robust prognostic biomarker and potential therapeutic target in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5bf5750b51fd180863b176de6d4f229a2cae4e" target='_blank'>
              Single-cell and spatial transcriptomics reveal lactylation-associated tumor cell clusters and define a prognostic risk model in glioblastoma
              </a>
            </td>
          <td>
            Rui Han, Guangfan Chi, Dongjie Sun, Ziran Xu, Liangfu Zhou, Kan Xu
          </td>
          <td>2025-11-22</td>
          <td>BMC Cancer</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Intratumoral heterogeneity (ITH) is a major driver of mortality in breast cancer (BC) patients and a critical factor in the variable therapeutic outcomes observed in BC treatment. Understanding the mechanisms underlying ITH is essential for advancing both clinical and basic BC research. Chromatin accessibility is critical for regulation of gene expression and cellular identity and plays a central role in shaping ITH and tumor evolution. However, studying chromatin accessibility in situ has been challenging due to the availability of technical platforms. Here, we leveraged the spatial ATAC-seq platform to profile the chromatin accessibility landscape of tumors from six BC patients. Our analyses revealed prominent heterogeneity within tumor regulatory modules and spatial variations in immune cell composition and stromal structures, offering a framework for investigation of the molecular architecture underlying ITH. Moreover, we identified two tumor subclones with potential common origin but distinct immune infiltration conferred by regulatory cascades, suggesting that epigenetic regulation may further contribute to the divergent tumor microenvironments and phenotypic diversity of these subclones. Our study provides novel insights into the molecular mechanisms driving ITH and opens up potential avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b8d2e84c5991207c662ff2b4c48eb4734a447e" target='_blank'>
              Spatial Chromatin Accessibility Analysis of Intratumor Heterogeneity in Breast Cancer.
              </a>
            </td>
          <td>
            Yingying Qian, Miao Zhu, Chongyang Ren, Yeyong Zhou, Jian Xu, Liang Dong, Guangyu Zhang, Cheukfai Li, Jiaoyi Lv, Qiaorui Xing, Guochun Zhang, Guangdun Peng, Ning Liao
          </td>
          <td>2026-01-12</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Recent progress in single-cell RNA sequencing has led to mechanistic and clinically actionable insight into genetic heterogeneity and tumor progression
 via
 transcriptome profiling at single-cell level, applied to identification of cell types, gene expression patterns, and signaling pathways involved in cancer development. In this work, we review the use of single-cell RNA sequencing (scRNA-seq) applications to gain insights into tumor molecular and cellular characteristics, such as cellular heterogeneity, rare cell populations, characteristic pathogenic cell populations, cells of the immune tumor microenvironment, and information regarding clonal evolution, none of which can be observed using bulk RNA-seq. We describe how this set of methods facilitates a better understanding of tumor heterogeneity, interactions between the tumor cells and the cells of the tumor microenvironment (TME), and can elucidate potential therapeutic targets. From the applied clinical perspective, we summarise the ability of scRNA-seq data to identify molecular indicators for diagnosis, outcome, and prediction of response to therapy. This is particularly relevant due to the low response rate to therapy of non-small cell lung cancer (NSCLC) and acquired resistance, including in immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d91b19b2652e7118bd597f31b2a5c69fb2aa994e" target='_blank'>
              Tumor heterogeneity assessment using single-cell RNA sequencing (scRNA-seq): applications in lung cancer for diagnosis and treatment
              </a>
            </td>
          <td>
            C. Bica, Oana Zanoaga, L. Pop, C. Ciocan, L. Raduly, A. Nutu, I. Berindan‐Neagoe, Andreas Bender
          </td>
          <td>2026-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background KRAS is one of the most frequently mutated genes in colorectal cancer (CRC) and plays a crucial role in tumorigenesis, progression, immune evasion, and treatment resistance. The pronounced heterogeneity within KRAS-mutant CRC highlights the urgent need for more precise and personalized therapeutic approaches. Methods To investigate this heterogeneity, we employed single-cell RNA sequencing and spatial transcriptomics to comprehensively characterize the tumor microenvironment of KRAS-mutant CRC. Data preprocessing and clustering were performed using Scanpy. Spatial cell-type deconvolution was conducted via Cell2location, whereas intercellular communication and spatial dependencies were analyzed using CellChat, MISTy, and stLearn. Results Our analyses revealed that KRAS-mutant tumor epithelial cells recruit Mono_S100A8 monocytes via the MDK_SDC4 signaling axis. Concurrently, surrounding Fib_CTHRC1 fibroblasts secrete collagen, which interacts with integrin receptors on KRAS-mutant epithelial cells and contributes to the exclusion of lymphocyte infiltration. Conclusion These cellular components collaboratively established an immunosuppressive spatial niche. These findings offer novel theoretical insights and potential targets for the development of immunoregulatory strategies tailored to KRAS-mutant CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059c9dfc013c2c370152c3fa0d39b4efc600e01e" target='_blank'>
              Single-cell and spatial transcriptome profiling identifies the immunosuppressive spatial niche in KRAS-mutant colorectal cancer
              </a>
            </td>
          <td>
            Sheng‐Li Yang, Chao Gu, Xinsheng Miao, Hao Zuo, Wei Xu, Yan Zhang, Wei Tang, Jianhua Zhu, Zheng Yuan, Xinhua Gu, Chenyi Zhong, Yueming Sun, Jiahui Zhou
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ffa550baca76e714d9b657ce67086b4b77a9db" target='_blank'>
              Deciphering Tumor Microenvironment Dynamics in Tumorigenesis and Lymph Node Metastasis of Esophageal Squamous Cell Carcinoma using Single-cell RNA Sequencing
              </a>
            </td>
          <td>
            Hansoll Na, T. Hong, Chung Lee, Y. Yang, Ha Eun Kim, Byung Jo Park, Hyun Ki Kim, Dae Joon Kim
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/465e60c37fcf50d018cbfc326971ebe12576816e" target='_blank'>
              Single-cell analysis reveals a universal pericyte signature associated with poor clinical outcome and immune T cell dysfunction in thyroid cancer and other cancers
              </a>
            </td>
          <td>
            Aarav Bhasin, Yubin Na, Jack Lawler, C. Nucera
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e234e293312dfb9093dad9f87bd40ef8d26f2e28" target='_blank'>
              Spatial transcriptomics reveals molecular heterogeneity and subtype-specific therapeutic targets in small cell lung cancer.
              </a>
            </td>
          <td>
            Tongji Xie, Le Tang, G. Fan, Xiaohong Han, Yuankai Shi
          </td>
          <td>2026-01-16</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background: Despite the use of immunotherapy in esophageal squamous cell carcinoma (ESCC), treatment failure occurs occasionally in patients, yet the underlying mechanisms remain poorly understood. Methods: We conducted large-scale single-cell RNA sequencing (scRNA-seq) data analysis, which integrated seven independent datasets from 192 ESCC patients to yield over 440,000 high-quality single cells, to systematically characterize the tumor microenvironment (TME) landscape during ESCC progression and immunotherapy response. Additionally, we performed high-resolution spatial transcriptomics (stRNA-seq) using the 10x Visium HD platform on paired pre- and post-treatment tissues from two patients (one immunotherapy responder and one non-responder), which enhanced the findings from the scRNA-seq data and mapped therapy-induced TME at the spatial level. Multiplex immunohistochemistry was employed based on seven patients to confirm distinct patterns of intercellular crosstalk underlying differential therapeutic outcomes. Results: In scRNA-seq data, we found that B lineage cells were reduced during ESCC progression but were enriched in immunotherapy-resistant patients. Further analysis of malignant ESCC cells suggested that immunotherapy resistance might be associated with a subpopulation of tumor cells exhibiting aberrantly elevated cholesterol biosynthesis. Cell communication analysis of scRNA-seq and stRNA-seq data collectively revealed that immunotherapy resistance was linked to cellular crosstalk between cholesterol-biosynthetic tumor cells and germinal center (GC) B cells within tertiary lymphoid structures. Notably, single-cell, spatial data, and multiplex immunohistochemistry demonstrated that cholesterol biosynthesis-associated ESCC cells express elevated levels of MIF. This disrupts GC reactions by competing with the CXCL12-CXCR4 signaling axis via MIF-CXCR4 interactions, thereby impairing B cell-mediated immunity. Conclusions: MIF+ tumor cells in GCs may be a biomarker for predicting immunotherapy resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8a1ee6ff72d48f3e16c31d484505ab587cac16" target='_blank'>
              MIF-expressing tumor cells mediate immunotherapeutic resistance in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Jing Song, Xiaomei Song, Yue Xie, Hong Guo, Yupeng Cun
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Spatial Transcriptomics (spTx) offers unprecedented insights into the spatial arrangement of the tumor microenvironment, tumor initiation/progression and identification of new therapeutic target candidates. However, spTx remains complex and unlikely to be routinely used in the near future. Hematoxylin and eosin (H&E) stained histological slides, on the other hand, are routinely generated for a large fraction of cancer patients. Here, we present a novel deep learning-based approach for multiscale integration of spTx with tumor morphology (MISO). We trained MISO to predict spTx from H&E on a new unpublished dataset of 72 10X Genomics Visium samples, and derived a novel estimate of the upper bound on the achievable performance. We demonstrate that MISO enables near single-cell-resolution, spatially-resolved gene expression prediction from H&E. In addition, MISO provides an effective patient representation framework that enables downstream predictive tasks such as molecular phenotyping or MSI prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b66b910856826f08a930be56d475ed3036cfdbc" target='_blank'>
              A deep learning-based multiscale integration of spatial omics with tumor morphology
              </a>
            </td>
          <td>
            B. Schmauch, L. Herpin, Antoine Olivier, Thomas Duboudin, Rémy Dubois, Lucie Gillet, Jean-Baptiste Schiratti, Valentina Di Proietto, Delphine Le Corre, Alexandre Bourgoin, Pr. Julien Taïeb, Pr. Jean-François Emile, Pr. Wolf H. Fridman, Elodie Pronier, P. Laurent-Puig, Eric Y. Durand
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Head and neck squamous cell carcinoma (HNSCC) is the sixth most common cancer worldwide. HPV-negative HNSCC, arising in diverse upper airway mucosal niches, is particularly aggressive, with poor 5-y survival and a limited response to immune checkpoint inhibitors. A deeper understanding of the tumor-localized immune landscape is essential to uncover actionable immunotherapeutic targets. Here, we integrated two single-cell RNA sequencing (scRNA-seq) datasets from 29 samples totaling nearly 300,000 immune cells to dissect immune rewiring during tumor progression and lymph node metastasis in HPV-negative HNSCC. We identified distinct shifts in adaptive immune cell populations across 14 peripheral blood mononuclear cell (PBMC) and 21 tumor-infiltrating immune cell (TIC) states. Notably, TICs exhibited enriched interferon response and immunomodulatory gene signatures, in contrast to PBMCs, indicating tumor-specific immune imprinting. Ligand–receptor analysis revealed that immunosuppressive crosstalk between macrophages and cytotoxic cells was associated with advanced disease. To spatially validate these transcriptional states, we conducted multiplexed immunofluorescence profiling on nine locally invasive HPV-negative HNSCCs, all from the ventrolateral tongue mucosa. Spatial proteomics confirmed peritumoral enrichment of activated (CD107a+, ICOS+) NK and CD8+ T cells and intratumoral accumulation of exhausted (PD-1+, PD-L1+) phenotypes, mirroring pseudotime trajectories inferred from scRNA-seq. These findings highlight spatially localized cytotoxic cell exhaustion as a key immune evasion mechanism in HPV-negative HNSCC and underscore the value of integrating spatial and single-cell data to reveal therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e184d6ff5ae8303f9ef6f76673a4a076479b44fb" target='_blank'>
              Single-cell landscape of peripheral and tumor-infiltrating immune cells in HPV-negative HNSCC
              </a>
            </td>
          <td>
            Rômulo Gonçalves Agostinho Galvani, Adolfo Alexis Rojas Hidalgo, Carlos Alberto Biagi-Junior, B. Matuck, Jelte Martinus Maria Krol, Brittany T. Rupp, Nikhil V. Kumar, Khoa L. A. Huynh, Jinze Liu, Siddharth Sheth, Vinicius Maracaja-Coutinho, K. Byrd, Patricia Severino
          </td>
          <td>2025-12-30</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9542f45171a12fa01fc5a868a4f227534778e8" target='_blank'>
              Iron-Associated Mesenchymal Plasticity and Tumor Invasion in Glioblastoma
              </a>
            </td>
          <td>
            C. Petry, A. Boecke, A. Mohammed, N. Ghariwala, V. Jariwala, Mohammad Al Shhab, O. Buchholz, I. Vasilikos, J. M. Nakagawa, Andreas Vlachos, J. Grauvogel, M. Shah, Roland Rölz, Marco Prinz, U. G. Hofmann, Jürgen Beck, Kevin Joseph, V. Ravi
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is an aggressive brain tumor with limited treatment options, and the tumor microenvironment plays a major role in its progression. Among stromal cells, cancer-associated fibroblasts (CAFs) are known to influence tumor growth and immune responses, but their diversity in glioblastoma has not been well understood. In this study, we analyzed both bulk and single-cell RNA sequencing data to identify CAF subtype-associated patterns within glioblastoma. We discovered four CAF populations—immune, inflammatory, antigen-presenting, and myofibroblastic—each associated with poor patient survival and unique immune and metabolic features. We also identified Spleen Tyrosine Kinase (SYK) as a key regulatory gene shared across these CAF subtypes. Our findings highlight the importance of stromal heterogeneity in glioblastoma and suggest that targeting CAF-related signaling, including SYK, may offer new therapeutic opportunities for this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77eebcce441ee2b3c1609920849b71d1de5f2a30" target='_blank'>
              Dissecting CAF Heterogeneity in Glioblastoma Reveals Prognostic Subtypes and a Central Regulatory Role for Spleen Tyrosine Kinase (SYK)
              </a>
            </td>
          <td>
            Ji-Yong Sung, K. Hwang
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PAAD) metastasis is driven by complex tumor–microenvironment interactions. Here, we integrated single-cell and bulk transcriptomic analyses of 104,855 cells from 10 patients to delineate the cellular and molecular landscape of primary versus metastatic PAAD. We identified metastasis-associated epithelial (LMO7⁺, TOP2A⁺, PIGR⁺), fibroblast (IGKC⁺, RGS5⁺), and M2-like macrophage (APOE⁺, CD14⁺, FOLR2⁺, SPP1⁺) subpopulations, validated via bulk deconvolution. Functional analyses revealed upregulated Wnt signaling, epithelial–mesenchymal transition, and angiogenesis in metastatic epithelial and fibroblast compartments. Intercellular communication analysis highlighted SPP1-mediated macrophage–epithelial/fibroblast crosstalk involving key receptor–ligand pairs, contributing to immune suppression and metastatic niche formation. Integrating gene expression and cell proportions, we developed a prognostic model with high predictive accuracy (C-index > 0.85), stratifying patients into risk groups with distinct immune landscapes. Furthermore, PTK6 was identified as a driver of PAAD proliferation, migration, and invasion. Collectively, our study elucidates TME-driven mechanisms of PAAD metastasis, identifies prognostic and therapeutic targets, and provides a framework for precision intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/759c03413ef4b1106fa2a25c75713885026e6884" target='_blank'>
              Single-cell transcriptomic analysis reveals key cell types and pathogenic genes associated with metastasis in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Shijun Shen, Zhiqiang Li, Hong Yang, Xinwei Zhang, Gang Chen, Chengzhou Pa
          </td>
          <td>2025-11-29</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c1adf877d1b31f72e6e7bf9d982961bd5b1573" target='_blank'>
              Digital Spatial Pathway Mapping Reveals Prognostic Tumor States in Head and Neck Cancer
              </a>
            </td>
          <td>
            Julius Hense, M. J. Idaji, Laure Ciernik, Jonas Dippel, Fatma Ersan, Maximilian Knebel, Ada Pusztai, Andrea Sendelhofert, Oliver Buchstab, S. Fröhling, Sven Otto, Jochen Hess, P. Liokatis, Frederick Klauschen, Klaus-Robert Müller, A. Mock
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0135229fffb59fe03ba95bebb58be45a4b3470c" target='_blank'>
              De novo H3.3K27M-altered Diffuse Midline Glioma in human brainstem organoids to dissect GD2 CAR T cell function
              </a>
            </td>
          <td>
            N. Bessler, A. Wezenaar, H. Ariese, C. Honhoff, E. J. Wehrens, N. Dommann, C. Ruiz Moreno, Thijs J M van den Broek, R. Collot, D.J. Kloosterman, F. Keramati, M. Roosen, S. de Blank, Esmée J. van Vliet, M. Barrera Román, L. C. Gatti, Ali Ertürk, J. Kuball, Z. Sebestyén, M. Kool, S. Patrizi, E. Miele, A. Künkele, M. Kranendonk, Annelisa M. Cornel, Stefan Nierkens, Christian Mayer, H. Stunnenberg, A. Alemany, Maria Alieva, A. Rios
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a highly heterogeneous malignancy with poor prognosis, in which tumor metastasis represents a critical event driving disease progression and treatment failure. Understanding the cellular drivers of ICC metastasis and establishing effective risk assessment models are essential for improving patient outcomes. We analyzed the GSE201425 single-cell RNA sequencing dataset to explore the cellular origins and mechanisms of ICC metastasis. Copy number variation (CNV) profiling and clonal evolution analysis were performed to identify malignant epithelial cells. Pseudotime trajectory analysis revealed metastasis-associated epithelial cells (MAECs). Key biomarkers within MAECs were screened, and a Metastasis Index (Met-Index) based on one-class logistic regression was constructed to assess metastasis risk in bulk RNA-seq data from the TCGA-CHOL cohort. Multiplex immunofluorescence staining was performed in 34 clinical ICC specimens. Functional validation was conducted in HuCCT1 cells using siRNA-mediated gene silencing, followed by CCK-8 proliferation assays and wound healing assays. MAECs were identified as a distinct epithelial subpopulation enriched in metastatic ICC samples. Biomarkers including MMP7, FXYD2, and PTHLH were upregulated in MAECs and strongly correlated with metastatic status. The Met-Index effectively stratified patients into high- and low-risk groups, with the high Met-Index group exhibiting significantly worse overall survival (p = 0.005) and progression-free survival (p = 0.015). Immunofluorescence validation confirmed increased expression of MMP7, FXYD2, and PTHLH in metastatic tumors, which correlated with advanced clinicopathological features. Functional assays demonstrated that knockdown of these biomarkers inhibited HuCCT1 cell proliferation and suppressed cell migration capacity. This study delineates the cellular origin and molecular drivers of ICC metastasis at the single-cell level, identifies MAECs as a metastasis-related subpopulation, and establishes a clinically relevant Met-Index with prognostic value. These findings provide potential biomarkers and therapeutic targets for the early identification and management of ICC patients at high risk of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27d855cd27f4e8545b1968d216eb341e556b791" target='_blank'>
              Single-cell multi-omics analysis reveals drivers of intrahepatic cholangiocarcinoma metastasis
              </a>
            </td>
          <td>
            Zetian Zhang, Hao Dou, Shangyumeng Zhao, Jiaqi Wang, Meiru Chen, Xiaoli Xie, Xiangcong Tian, Dongqiang Zhao
          </td>
          <td>2025-11-24</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e493ac7b6aab5ae1ab712cb247fef1395ea0b1a" target='_blank'>
              Quantitative profiling of intratumor immune heterogeneity identifies loss of immune diversity as a hallmark of cancer progression.
              </a>
            </td>
          <td>
            Qiqi Lu, Jian-qin Luo, Chia-Hao Tung, Xiaosheng Wang, Zhongming Zhao
          </td>
          <td>2025-12-16</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5bdb6dd873fe4b6bf8ca46144bcdfa66e477ac0" target='_blank'>
              Beyond one-size-fits-all: single-cell transcriptomic signatures predict drug efficacy and reveal responder subgroups in endometriosis
              </a>
            </td>
          <td>
            R. Pérez-Moraga, C. Bafligil, S. Harden, Santiago García‐Martín, M. J. Jiménez-Santos, Ioanna Tiniakou, Sophie Ribeiro-Volturo, Line Gies, María Teresa Pérez Zaballos, Cristina Fernández-Molina
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Glioblastoma is a lethal primary brain tumor with poor prognosis. Tumor cells exhibit substantial phenotypic variation, complicating treatment. As functional diversity is driven by underlying transcriptional states, characterizing tumor cell gene expression is essential for understanding tumor biology and therapeutic response. The GL261 tumor cell line is a common pre-clinical model system for investigating glioblastoma pathobiology. However, global gene expression patterns in this model are unknown. Here we describe the use of single-cell RNA sequencing (scRNA-Seq) to investigate transcriptional profiles of 5764 adherent and 4951 neurosphere GL261 cells, generating 133,442,221 sequenced reads. Following Principal Component Analysis (PCA) for dimensionality reduction, we applied Uniform Manifold Approximation and Projection (UMAP) to visualize transcriptionally distinct subpopulations (clusters) of GL261 cells grown adherently or as neurospheres. Highly expressed and differentially expressed genes were identified. Because the neurosphere phenotype is known to be more tumorigenic, we further examined differentially expressed genes with gene ontology expression analysis. We found that upregulated genes in neurosphere cells are associated with angiogenesis, cell adhesion, and cell signaling pathways. In addition, we specifically examined gene expression patterns of matrix metalloproteinases and purinergic receptors, glioblastoma drug targets known to be important for promoting tumor infiltration into adjacent healthy tissue. We found that P2RX7, MMP15 and MMP16 are upregulated in neurosphere cells, indicating a potential role for these genes in tumor formation. Together these results reveal global transcriptional profiles of GL261 cells, establish a resource for further scRNA-Seq-based analyses, and give insight into gene expression changes relevant to glioblastoma tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b536ff0acd69dbca3af6bd3122c752115f69d4a" target='_blank'>
              Single-Cell Transcriptomic Profiling of GL261 Glioblastoma Cells Reveals Gene Expression Signatures Underlying Tumorigenicity
              </a>
            </td>
          <td>
            Colton E Troxel, Ruby A Olvera, Emily A Freko, Suely S C Soeiro, Thandiswa Mdluli, Richard L Daniels
          </td>
          <td>2025-12-16</td>
          <td>Cellular and Molecular Neurobiology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Epidermal growth factor receptor (EGFR) mutation is a key oncogenic driver in lung adenocarcinoma (LUAD), but its impact on the tumor immune microenvironment (TIME) remains unclear. By integrating single-cell transcriptomes from 153 LUAD samples using machine learning, we generated an atlas of over one million cells that delineates immune heterogeneity. EGFR-mutant tumors exhibited enrichment of TIGIT+regulatory T cells, neutrophils, and macrophages, whereas wild-type tumors contained abundant ZNF683+CD8+tissue-resident memory T cells, diverse memory B cells, and FGFBP2+CD16high natural killer cells, reflecting an immune-active TIME. Non-negative matrix factorization defined five TIME subtypes, with EGFR-mutant patients clustering into immunosuppressive profiles linked to poor prognosis. Flow cytometry and mouse models confirmed the cytotoxic and PD-1 blockade-enhancing functions of FGFBP2+NK cells. These findings reveal distinct TIME landscapes in EGFR-mutant LUAD and illustrate the potential of machine learning-based immunogenomic analysis to inform precision immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad6284204cf8fc0e7c47d8fa667f142a64e34e5" target='_blank'>
              Machine learning identifies TIME subtypes linking EGFR mutations and immune states in lung adenocarcinoma
              </a>
            </td>
          <td>
            Zetian Gong, Mingjun Du, Ying Li, Bicheng Ye, Yuming Huang, Hui Gong, Wei Wang, Liang Chen, Zongli Ding, Pengpeng Zhang
          </td>
          <td>2025-11-26</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Osteosarcoma (OS) exhibits substantial genomic complexity and inter-patient heterogeneity, necessitating longitudinal, patient-matched analyses to understand acquired features of tumor evolution. However, most published OS data is limited to primary tumor samples, limiting insight into patient-specific resistance mechanisms. To address this, we characterized the genomic landscape of paired primary and metastatic tumor samples from dogs with spontaneous OS. Whole-genome and single-cell RNA sequencing reveal mutation and gene expression profiles that are predominantly organized by patient identity. Mutational burden and pathway alterations such as those involving PI3K, NOTCH, TP53, MAPK, RAS and epigenetic regulation differ between primary and metastatic samples. Variants present in tumor tissue are readily detectable in paired cfDNA samples, demonstrating the utility of this assay for identifying tumor-specific alterations associated with treatment resistance. Analysis of bulk RNA-seq data to estimate cell-type composition shows greater immune cell representation in metastases, underscoring the importance of immune signaling pathways in OS. These findings exemplify the presence of patient-specific alterations in genomic architecture over the course of tumor progression, linking CNV amplification, pathway reprogramming, and immune evasion in metastatic OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cervical cancer, as one of the most common types of malignant tumors in women, poses a significant threat to the health and survival of women worldwide. Tumor drug resistance has emerged as a critical bottleneck severely limiting the efficacy of current cervical cancer treatment strategies, making it imperative to delve deeper into the underlying mechanisms for improved therapeutic outcomes. Spatial transcriptomics and single-cell sequencing technologies were employed. Single-cell sequencing was used to uncover the heterogeneity in cervical cancer development. Pseudotime analysis, exploration of cell communication pathways, and spatial transcriptomics were further applied to investigate the roles of relevant molecules. This study reveals an abundance of PI3 + neoplastic (NEO) in tumors and SLC40A1 + NEO cells in HSIL samples. These cell populations may play crucial roles in the development of drug resistance, as they could potentially alter the tumor microenvironment and interfere with the efficacy of therapeutic agents. Pseudotime analysis, cell communication pathways, and spatial transcriptomics highlight the pivotal role of LGALS9 and its ligands, offering new insights for early diagnosis, molecular typing, prognosis evaluation, and personalized treatment of cervical cancer. Our findings reveal that high expression of LGALS9 suppresses T cell function, fosters a strongly immunosuppressive tumor microenvironment, and is associated with chemotherapy resistance, ineffective immunotherapy, and poor prognosis. LGALS9 may serve as a biomarker for predicting immunotherapy response. This knowledge has the potential to facilitate early diagnosis, precise molecular typing, accurate prognosis evaluation, and the development of personalized treatment strategies that can overcome tumor drug resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f662f4b181b3395cd096885302fa365b2511567e" target='_blank'>
              Exploring tumor heterogeneity and drug resistance in cervical cancer through single-cell and bulk transcriptomics
              </a>
            </td>
          <td>
            Yinxia Yang, Sirong Cheng, Yuxuan Zhong, Pengyu Sun, Xiaoru Deng, Bohui Zhou
          </td>
          <td>2025-11-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc11040a019ecb4b6c2065cd731c8ff500837ec3" target='_blank'>
              CancerSEA-X: A Single-cell Resource for Tumor Microenvironment Cell States Across over 30 Cancer Types.
              </a>
            </td>
          <td>
            Lantian Deng, Wei Liu, Jinyuan Xu, Bo Zhang, Shengyuan He, Kun Liu, Xinxin Zhang, Huating Yuan, Fei Quan, Yun Xiao
          </td>
          <td>2026-01-16</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cf48b273167de19bd485ee182b5df8f720d9f6b" target='_blank'>
              Spatially distinct chromatin compaction states predict neoadjuvant chemotherapy resistance in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Reut Mealem, T. A. Phillips, Leor Ariel Rose, S. Marcotti, M. Parsons, A. Zaritsky
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cervical cancer (CC) is one of the most prevalent malignancies that affects women worldwide. It is strongly influenced by lifestyle and environmental factors, most notably persistent HPV infection and smoking. A defining feature of CC is its high degree of heterogeneity, which poses significant challenges for conventional sequencing and diagnostic approaches in fully capturing its oncogenetic landscape. The advent of single-cell RNA sequencing (scRNA-seq) has transformed this field, offering an unprecedented ability to dissect the tumor microenvironment at a single-cell resolution. By mapping cell–cell communication networks and characterizing gene expression across diverse cellular populations, scRNA-seq enables the identification of biomarkers with strong diagnostic and prognostic potential. Beyond biomarker discovery, this technology also facilitates the recognition of novel therapeutic targets, paving the way for more precise and effective treatment strategies. This review summarizes the applications of scRNA-seq in cervical cancer research, with a focus on methodological advances, key findings from recent studies, and their implications for improving diagnosis, prognosis, and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e029b6816cfdfef9fa928cd92a7a3e4aaf306a" target='_blank'>
              Implications of Single-Cell RNA Sequencing in Cervical Cancer: Unravelling the Molecular Landscape
              </a>
            </td>
          <td>
            Blessy Kiruba, Raja Rishi Raghavendar Raja Karthikeyan, S. Anilkumar, Vino Sundararajan, S. Lulu S
          </td>
          <td>2025-12-02</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pulmonary pleomorphic carcinoma (PPC) is a rare subtype of lung cancer that comprises both epithelial and sarcomatoid components. The molecular basis of PPC, including the cellular dynamics of its components, remains largely unknown. To elucidate potential therapeutic targets for PPC, we perform a multi-omics analysis incorporating digital spatial profiling and single-cell RNA sequencing (scRNA-seq). PPC exhibits diverse driver gene alterations, including MET exon 14 skipping mutation (METex14) and ALK fusion. In spatial transcriptomics, MET gene and protein are overexpressed exclusively within the epithelial component and not in the sarcomatoid component, even in patients harboring METex14. Epithelial-mesenchymal transition (EMT)-related transcriptional changes, along with extracellular matrix (ECM) remodeling between the epithelial and sarcomatoid components, are observed. scRNA-seq identifies cell populations within the epithelial component that contribute to the malignant transformation and differentiation of the sarcomatoid component. They are characterized by an intermediate EMT state with ECM remodeling signature, suggesting their potential as novel therapeutic targets for PPC. Single-cell and spatial transcriptomic analysis of pulmonary pleomorphic carcinoma reveal epithelial–mesenchymal transitions and extracellular matrix remodeling that shape the distinct architecture of this rare lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f421d642675df6e6bdc9b51c7d9b23970c7bec20" target='_blank'>
              Single-cell and spatial transcriptomic characterization of pulmonary pleomorphic carcinoma
              </a>
            </td>
          <td>
            Atsushi Matsuoka, K. Shien, Shuta Tomida, Masayoshi Ohki, Kazuya Hisamatsu, Ryota Fujiwara, Kosei Ishimura, Ryunosuke Fujii, Tomoaki Higashihara, Naohiro Hayashi, Kazuhiro Okada, Ryo Yoshichika, Fumiaki Mukohara, Mao Yoshikawa, Yuma Fukumoto, K. Suzawa, Y. Tomioka, Shin Tanaka, Kentaroh Miyoshi, Mikio Okazaki, S. Sugimoto, Yusuke Otani, Atsushi Tanaka, Hirofumi Inoue, Yosuke Togashi, Hidetaka Yamamoto, D. Ennishi, Shinichi Toyooka
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Remodeling of the tumor microenvironment (TME) under therapeutic pressure is a critical determinant of treatment response and resistance in hepatocellular carcinoma (HCC). Triple-combination therapy integrating targeted agents, immune checkpoint inhibitors, and radiotherapy (T+I+R) has shown potential synergistic effects in intermediate to advanced HCC, particularly in patients with portal vein tumor thrombus (PVTT), yet the spatial and cellular mechanisms underlying its efficacy remain largely unknown. In this retrospective clinical cohort study, we compared T+I+R with targeted therapy plus radiotherapy (T+R) in advanced HCC, and further employed single-cell spatial transcriptomics and spatial proteomics to generate an integrated multi-omics atlas mapping tumor and stromal compartments, cellular compositions, and intercellular interactions with spatial resolution. Clinically, T+I+R achieved superior tumor shrinkage and disease control compared with T+R. Spatial multi-omics revealed marked region-specific remodeling, with myofibroblastic cancer-associated fibroblasts, angiogenic tip endothelial cells, and conventional dendritic cells enriched at the invasive margin and associated with therapeutic resistance, while CD8+ effector T cells were redistributed away from immunosuppressive niches, a spatial configuration correlating with enhanced response. These findings identify spatial segregation between cytotoxic and suppressive immune elements as a potential hallmark of effective therapy, providing a high-resolution spatial framework for understanding T+I+R induced TME remodeling and offering mechanistic insights to guide biomarker discovery and the optimization of combination strategies in advanced HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d6c3edb9469068b18d4bde88276d91593b43ce6" target='_blank'>
              Spatial multi-omics mapping of tumor microanatomy dynamics following radiotherapy combined with targeted-immunotherapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Fansen Ji, Haoming Xia, Ying Xiao, Jiawei Zhang, Hao Chen, Bingjun Tang, Huan Li, Hao Liu, Boyang Wu, Xiaojuan Wang, Shizhong Yang, Jiahong Dong
          </td>
          <td>2026-01-08</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="To identify the cell types that are infected with KSHV and the immune interactions in Kaposi sarcoma (KS) lesions, we performed spatial transcriptomics with seven KS skin tumors. We used a single‐cell RNA‐sequencing reference data set from healthy skin donors with a method to conduct spatially informed cell‐type deconvolution for spatial transcriptomics. This allowed us to predict the relative amounts of each cell type within the patient sample sections. We included custom probes for five KSHV genes that allowed us to measure human and KSHV expression patterns at the same time. We then compared the spatial gene expression data of KS skin samples with six normal skin samples and found higher expression of marker genes corresponding to macrophages/dendritic cells, lymphatic endothelial cells, and vascular endothelial cells in the KS skin lesions when compared to normal skin samples. Our spatial transcriptomic results from thousands of spots across multiple KS tumors indicated a correlation between high levels of STC1 and decreased expression of macrophage markers. Together, these analyses offer potential mechanisms by which KSHV infection may remodel skin tissue, inhibit immune responses against KSHV infection, and confer resistance to anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0e2360131ded34f6dbf590cb05f764fc0e85cc" target='_blank'>
              Spatial Transcriptomics of Patients With Kaposi Sarcoma Identifies Mechanisms of Immune Evasion
              </a>
            </td>
          <td>
            Bahman Afsari, R. Ramaswami, K. Lurain, Takanobu Tagawa, Daphne Knudsen-Palmer, Guruswamy Mahesh, Ameera Mungale, R. Yarchoan, Joseph Ziegelbauer
          </td>
          <td>2025-12-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is a lethal subtype of prostate cancer characterized by its androgen independence, resistance to chemotherapy, and display of neuroendocrine features which can emerge either de novo or via transformation after a prior diagnosis of adenocarcinoma. The poor clinical outcomes in patients with AVPC are associated with its profound molecular heterogeneity. In this study, we analyzed 23 consecutive AVPC cases treated at a dedicated small-cell clinic (2017-2025) using clinicogenomic and transcriptomic profiling. Transformed AVPC exhibited significantly shorter overall survival times than de novo AVPC (11.8 vs 26.0 months, P < 0.001). Integrative genomic analyses identified residual androgen signaling in subsets of cases harboring neuroendocrine lineage programs, highlighting a decoupling of lineage identity and morphology. To facilitate mechanistic and pharmacologic studies, we established NCI-LYM-1, a patient-derived organoid/PDX from an AR-negative, ASCL1+/SYP+ lymph node metastasis, which faithfully recapitulates the donor tumor's molecular and phenotypic features. Short- and long-read whole-genome sequencing combined with optical genome mapping identified biallelic inactivation of PTEN, TP53, RB1 and BRCA2 as potential drivers, demonstrating clonal concordance with circulating tumor DNA from the original patient donor. Pathway and perturbation analyses suggested that NCI-LYM-1 harbored a strong dependency on apoptotic pathways, which was confirmed by in vitro organoid testing with the BCL-2/BCL-xL inhibitor navitoclax (IC50: 0.27 M) and the MCL-1 inhibitor AZD-5991 (IC50: 0.060 M). Overall, NCI&-LYM-1 recapitulates the clinical aggressiveness and heterogeneity of AVPC, providing a tractable platform to identify novel precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e71e79208c8d37fdc48221a6ff9b64fb28fe818c" target='_blank'>
              Molecular decoupling of lineage identity and morphology in aggressive variant prostate cancer
              </a>
            </td>
          <td>
            C. Li, , M. L. Abel, D. S. Vargas Solivan, K. Bhadresha, S. Kartal, S. Nichols, K. Parmar, J. Twohig, T. M. Truong, C. H. Chau, K. Kelly, W. Figg, A. Thomas, A. Sowalsky
          </td>
          <td>2026-01-09</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="In cancer, extrachromosomal DNA (ecDNA) contributes to tumor heterogeneity and is associated with poor prognosis, but studies on patient-derived ecDNA are relatively limited at single-cell resolution. Here, we introduce scCirclehunter, a framework designed to identify ecDNA from scATAC-seq data and assign ecDNA to specific cell populations. Leveraging scCirclehunter and available glioblastoma (GBM) datasets, we uncover the inter-cellular heterogeneity of ecDNA-carrying cells across GBM patients and trace the trajectories of malignant cells within a single patient that harbors multiple ecDNAs. By integrating scRNA-seq data, we use ecNR2E1 as an example to demonstrate that ecDNA drives tumor progression in GBM through several mechanisms. Additionally, our findings suggest a potential link between ecDNA and increased mitochondrial transfer frequency. Overall, scCirclehunter provides a novel framework for analyzing patient-specific ecDNAs with single-cell precision, offering insights into the role of ecDNA-carrying cells in driving GBM heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c481a654d2e51f6263c828b5c2a73535459ef5" target='_blank'>
              scCirclehunter delineates ecDNA-containing cells using single-cell ATAC-seq, with a focus on glioblastoma
              </a>
            </td>
          <td>
            Rong Jiang, Zhengmao Lu, Fang Li, Yibei Zhu, Manqiu Yang, Shufan Zhang, Ping Wu, Chengliang Gong, Yiyuan Fei, Yonghua Sang, Yulun Huang, Jiong Jiong Guo, Moli Huang
          </td>
          <td>2025-12-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The spatial organization of the tumor microenvironment (TME) encodes the selective pressures that drive cancer evolution, but many tools focus on local cell contacts or single-modality readouts. We developed Tumor Landscape Analysis (TLA), a modular framework that borrows from landscape ecology and spatial statistics to quantify heterogeneity, fragmentation, and diversity across scales, from single cells to whole-slide ecosystems.



 TLA ingests whole-cell or nuclear centroids and region-level segmentations to compute (i) point statistics such as nearest-neighbor distance, Ripley’s H, and Morisita-Horn colocalization; (ii) kernel-smoothed local spatial profiles including Getis-Ord Gi* hot and cold spots; and (iii) fragmentation, shape, and diversity metrics such as patch density, contagion, shape index, and Shannon entropy. We also introduce Local Microenvironments (LMEs), unsupervised ecological niches defined by joint cell-type abundance and spatial mixing. Pilot analyses were performed in ductal carcinoma in situ (DCIS; n=353 samples, 175 patients) with cell-level maps of epithelial, lymphocyte, and fibroblast populations, and in pancreatic ductal adenocarcinoma (PDAC; n=203 resected cases after neoadjuvant therapy) using cancer and stroma segmentations.



 In DCIS, spatial relationships rather than raw abundance associated with recurrence risk. Shorter lymphocyte-to-fibroblast minimum nearest-neighbor distance and lower Ripley’s H, indicating greater intermixing, correlated with earlier recurrence independent of cell counts. This highlights immune–stromal proximity as a candidate eco-evolutionary driver of persistence. In PDAC, tissue-level fragmentation captured prognostic architecture after therapy. Two shape-based signatures emerged: (1) stromal uniformity, where lower standard deviation of stromal patch shape index associated with improved disease-free survival, consistent with a more constrained, less reactive stroma; and (2) tumor geometric complexity, where higher mean cancer patch shape index, reflecting more irregular or dendritic nests, associated with lower recurrence risk, whereas compact nests predicted worse outcomes. Across datasets, LME maps localized niche phenotypes, such as mixed-immune or segmented tumor habitats, that align with hypothesized selection landscapes and provide inputs for fragmentation analysis.



 TLA operationalizes an eco-evolutionary view of cancer by unifying interpretable, multiscale spatial metrics in a single workflow that works with any rasterized data input. By quantifying how habitats, edges, and mixing patterns are organized, TLA reveals niche structures and architectural motifs linked to clinical outcomes and complements foundation models and modality-specific pipelines with theory-grounded, clinically translatable descriptors. Ongoing work integrates TLA features with molecular data and outcomes to derive robust biomarkers and to inform adaptive, niche-targeted strategies in precision oncology.



 Ryan M. Carr, Luis Cisneros, Merih Deniz. Toruner, Martin E. Fernandez-Zapico, Carlo Maley. Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f5d8ed56157756f83dcf8a730cae9e8b632c46" target='_blank'>
              Abstract A015: Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments
              </a>
            </td>
          <td>
            Ryan M. Carr, L. Cisneros, M. D. Toruner, Martín E Fernandez-Zapico, Carlo C. Maley
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor heterogeneity is one of the key factors leading to diagnostic errors, difficulty in identifying subtle cellular differences, and the occurrence of immune escape. Although single-cell RNA sequencing (scRNA-seq) has elucidated the molecular characteristics and heterogeneity at the single-cell level, it fails to retain spatial location information. In contrast, spatial transcriptomics (ST) technology can simultaneously acquire gene expression information and cellular localization, providing a unique perspective for studying the tumor microenvironment, cell-cell interactions, and other aspects. The integrated application of scRNA-seq and ST technologies enables the analysis of tumor heterogeneity from both molecular and spatial dimensions, offering comprehensive information support for precision medicine. The complementary advantages of these two technologies not only overcome the limitations of traditional methods but also promote the research on complex mechanisms such as tumor heterogeneity and the tumor microenvironment, accelerating the translational process of clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9514d53fae21da5a677d774e552caceeac1e73" target='_blank'>
              Research Progress in Single-Cell RNA Sequencing and Spatial Transcriptomics for Precision Oncology
              </a>
            </td>
          <td>
            Minxin Mo
          </td>
          <td>2025-12-18</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a084b4c5d54edf23648139fa8024008066585a8" target='_blank'>
              Beyond genomics: 3D microtumor assays for rapid, clinically relevant functional drug testing
              </a>
            </td>
          <td>
            Marina Chan, Taranjit S. Gujral
          </td>
          <td>2025-11-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="BACKGROUND
Immune cells critically influence pituitary neuroendocrine tumor (PitNET) progression and therapeutic responses. However, their composition and functional dynamics remain unclear. This study aimed to delineate the immune heterogeneity and intercellular communication networks within PitNETs.


METHODS
This study conducted single-cell RNA sequencing (scRNA-seq) on 22 fresh PitNET samples obtained from surgical patients at Beijing Tiantan Hospital between September 2023 and January 2024, with classification based on the expression of key transcription factors: pituitary-specific transcription factor 1 (PIT1), steroidogenic factor 1 (SF1), and T-box transcription factor 19 (TPIT). Following cell type identification, cell-cell communication analysis revealed specific intercellular interactions, which were validated by multiplex immunohistochemistry (mIHC), flow cytometry, and in vitro co-culture assays.


RESULTS
The scRNA-seq analysis revealed significant cellular heterogeneity and a lineage-specific immune landscape, including cancer-associated fibroblasts (CAFs), neutrophils, T cells, natural killer cells, and myeloid cells. Specifically, CAF subset distribution was highly lineage-dependent. Collagen-expressing CAF3 was significantly enriched in the PIT1 lineage compared to both TPIT and SF1 lineages. In contrast, inflammatory CAF5 was predominantly found in the TPIT lineage relative to the PIT1 and SF1 lineages. Furthermore, CD4+ regulatory T (Treg) and T follicular helper (Tfh) cells were significantly enriched in the PIT1 lineage relative to the TPIT and SF1 lineages, as validated by both flow cytometry and mIHC data. Cellular communication analysis revealed notable interactions between Treg/Tfh cells and macrophages/microglia that were mediated by the cytotoxic T-lymphocyte-associated protein 4 (CTLA4)-CD86 pathway. Subsequent mIHC assays confirmed spatial colocalization of Treg cells with macrophages/microglia. Complementing these findings, in vitro co-culture assays demonstrated functional CTLA4-CD86 signaling specifically between Treg cells and macrophages, providing deeper insights into the complex cellular crosstalk within the PitNET tumor microenvironment (TME).


CONCLUSIONS
The three major PitNET lineages exhibited distinct immune cell frequencies, with the PIT1 lineage notably enriched in Treg and Tfh cells, as well as collagen-producing CAF3. Furthermore, a novel immunosuppressive interaction between Treg cells and macrophages, mediated by the CTLA4-CD86 pathway, was identified, suggesting its potential regulatory role within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d53bbb9dbcd32760a571bdb944453b1d4eebc82" target='_blank'>
              Single-cell transcriptomics reveals the landscape of immune phenotypes in pituitary neuroendocrine tumors.
              </a>
            </td>
          <td>
            Zhe Zhang, Hong Su, Jie Yin, Junhua He, Yanan Hu, Peng Li, Pinan Liu
          </td>
          <td>2026-01-09</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity and plasticity. GBM tumors contain neoplastic cells within a fluid spectrum of diverse transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we develop scFOCAL (Single-Cell Framework for -Omics Connectivity and Analysis via L1000), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identify compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states, and leverage this capability to predict combination synergy. We validate the significance of these findings in vitro, ex vivo, and in vivo, and identify a combination of an OLIG2 inhibitor and Depatux-M for the treatment of GBM. Our studies suggest that scFOCAL identifies cell states that are sensitive and resistant to targeted therapies in GBM using a measure of cell and drug connectivity, which can be applied to identify new synergistic combinations. Glioblastoma is characterised by high levels of intratumoural heterogeneity and plasticity, hindering treatment. Here, the authors develop an analytical framework, scFOCAL, to predict the sensitivity of glioblastoma cell subpopulations to therapies based on reversal of disease transcriptional signatures to identify synergistic therapeutic combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef55e3b53c83b7969434910c5b4838c55ce9672" target='_blank'>
              Drug and single-cell gene expression integration identifies sensitive and resistant glioblastoma cell populations
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, R. Komotar, Macarena I. de la Fuente, Gregory Stein, Alexandre Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, Nagi G. Ayad
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdc2f4fd014e0601edcc3f78406fd8afaf2448ea" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors.
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, D. Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson A. Weir, O. Ashenberg, Geraldine S Pinkus, Scott J. Rodig, Caroline Uhler, Evan Z. Macosko, Margaret Shipp, A. Louissaint, Fei Chen, Todd R Golub
          </td>
          <td>2025-12-20</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Molecular heterogeneity in head and neck squamous cell carcinoma (HNSCC) is well recognized, yet existing subtype frameworks remain largely descriptive and have not translated into therapeutic decision-making. Here, we establish a mechanistic platform that converts transcriptomic diversity into drug-actionable tumor states. Integrating multi-cohort RNA-seq from 727 tumors across five independent datasets, genome-scale CRISPR dependency maps, and pharmacologic screening, we define distinct tumor survival circuits across HPV-negative HNSCC and nominate subtype-matched therapeutic strategies. These circuits encompass a proliferative axis (MYC, MET/FAK, inflammatory and translational programs), an epithelial-differentiated/adhesion program, an EMT-like state with stromal activation, and mitochondrial/oxidative metabolic states, each mapping to selective liabilities (e.g., mitotic/autophagy control, ERBB/PI3K and cadherin signaling, OXPHOS/mitochondrial translation, and G2/M-integrin-Notch pathways, respectively). We then develop a transcriptomic predictor of EGFR-inhibitor response using machine learning and validate it in prospectively collected, fresh patient-derived 3D microtumors. The resulting 13-gene signature identifies erlotinib-responsive tumors (R = 0.93) and maps biologically to an epithelial-differentiated state, outperforming EGFR expression alone. Our study establishes a subtype-to-dependency-to-therapy framework, enabling precision stratification and providing a clinically feasible path for prospective biomarker deployment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29377070a0d6fb125a75d4b7cf89275cb55faab" target='_blank'>
              Subtype-Specific Dependencies and Drug Vulnerabilities Enable Precision Therapeutics in Head and Neck Cancer
              </a>
            </td>
          <td>
            Joel M Vaz, Songli Zhu, Mateo Useche, Lauren Shih, Emily Marchiano, Slobodan Beronja, Bruce E Clurman, Cristina Rodriguez, Brittany Barber, Marina Chan, Taranjit S. Gujral
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gastric cancer (GC), of which cases with peritoneal metastasis are particularly challenging, retains its position of being highly complex and remarkably resistant to therapy. Understanding the spatial heterogeneity and leveraging recent technologies such as machine learning to uncover explanatory patterns remains critical to truly understanding this disease. Here, we conducted spatial transcriptomics analysis to identify distinct niches within GC tissues. Among these, a niche enriched with fibroblasts and macrophages exhibited a striking spatial co-infiltration pattern with tumor cells dominant niches. Further validation by multiplex immunofluorescence highlighted the coordinated cellular interactions that characterize the TME. Through integration of sc-RNA with bulk RNA sequencing, we identified DAB2⁺ TAMs and ACTA2⁺ myCAFs as the main contributors to this co-infiltration pattern. NicheNet analysis further revealed that the PLAU-PLAUR signaling axis holds a central regulatory role in the communication between macrophages, fibroblasts and tumor cells. Given the prognostic value of this spatial pattern, we additionally applied transfer learning based on an ImageNet pre-trained ResNet-50 model to develop a machine learning framework that can accurately recognize the macrophage-fibroblast-malignant cell co-infiltration pattern, called Gastric-Discovery. Potentially, Gastric-Discovery could be a tool for precise patient stratification and provides novel insights into the dynamic architecture of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ca98a4e613e1447cb2daf8bdbf25c8434e308a3" target='_blank'>
              Multi-dimensional omics integrated machine learning framework identifies macrophage-fibroblast-tumor co-infiltration patterns to predict prognosis in gastric cancer
              </a>
            </td>
          <td>
            Qi Wang, Yuan Ni, Sheng Lu, Benyan Zhang, J. Ji, Qu Cai, Chao Yan, Feng Qi, M. Shi, Jun Zhang
          </td>
          <td>2025-12-09</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: Hepatocellular carcinoma (HCC) harbors a dynamic tumor microenvironment (TME) in which macrophages are highly abundant and plastic. Under physiological conditions, macrophages switch between inflammatory and resolution/tissue-repair programs to maintain homeostasis; however, during hepatocarcinogenesis these programs are reprogrammed into tumor-associated macrophage (TAM) states that foster immune suppression, angiogenesis, and tumor progression. Purpose: To summarize macrophage heterogeneity and polarization mechanisms in HCC, and to highlight omics-informed therapeutic opportunities for targeting TAMs and improving precision immunotherapy. Research Design: This review summarizes physiological macrophage polarization and the mechanistic basis of macrophage reprogramming in the hepatocellular carcinoma immune microenvironment, integrating evidence from recent advances in single-cell sequencing, multi-omics, and spatial transcriptomics, with a focus on macrophage subset diversity, key regulatory pathways governing polarization and function, and emerging macrophage-targeted interventions and biomarkers. Results: Recent single-cell and spatial multi-omics studies reveal substantial TAM heterogeneity and plasticity in HCC. Macrophage-targeted strategies—including TAM depletion, phenotypic reprogramming, and exosome-mediated drug delivery—show encouraging preclinical efficacy. Macrophage-associated prognostic models and biomarkers may support individualized immunotherapeutic approaches. Conclusions: Macrophage polarization in HCC represents a dynamic continuum that is essential for homeostasis but is co-opted by tumors to drive immunosuppression and tissue remodeling. Advances in single-cell and spatial multi-omics are redefining TAM subsets and actionable pathways, enabling more rational macrophage-targeted therapies. However, challenges remain in standardizing TAM definitions, identifying robust predictive biomarkers, minimizing off-target effects, and optimizing combinations with immunotherapy. Integrating longitudinal multi-omics with AI-based modeling may help predict macrophage state transitions, guide patient-specific regimens, and advance precision medicine in HCC. Plain language summary Hepatocellular carcinoma is a highly aggressive cancer. Studies have found that immune cells within tumors—particularly macrophages—play a crucial role in cancer development and therapeutic response. Macrophages have two main “personalities”: the M1 type, which can kill cancer cells, and the M2 type, which supports cancer growth. They shift between these states in response to signals from the tumor microenvironment. Both liver-resident Kupffer cells and infiltrating macrophages are involved in this process. Various molecules, such as cytokines, enzymes, and exosomes, influence their polarization. This polarized state of macrophages, in turn, affects key tumor processes such as stemness, proliferation, metastasis, and angiogenesis. In recent years, scientists have been exploring drugs that specifically target macrophage polarization to suppress liver cancer, offering new directions for future therapy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b9c3d3369daa7c8bd747ff210bf91cca4945570" target='_blank'>
              Macrophage Polarization in the Tumor Microenvironment of Hepatocellular Carcinoma: From Mechanistic Insights to Translational Therapies
              </a>
            </td>
          <td>
            Xiaoqing Fu, Mingquan Pang, Zhixin Wang, Haijiu Wang
          </td>
          <td>2025-11-01</td>
          <td>Cancer Control: Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Angioimmunoblastic T-cell lymphoma (AITL) represents the second most prevalent subtype of peripheral T-cell lymphoma, characterized by a dismal prognosis. However, a systematic exploration of tumor microenvironment (TME) features and their prognostic significance in AITL remains notably deficient. To address this knowledge gap, we conducted spatial transcriptome sequencing (ST-SEQ) and whole-exome sequencing in four AITLs and two noncancerous lymph nodes for discovery purposes, complemented by immunohistochemistry analyses on 37 AITL cases for validation. We identified 14 ST clusters, including five neoplastic clusters, wherein a global shift in B-cell phenotypes and enrichment of myeloid cells were observed. These findings underscore a hallmark of exacerbated inflammation and immune dysregulation within the neoplastic TME. Among the 4 ST-sequenced AITLs, 3 expressed high CD40-CD40LG activity, accompanied by the upregulation of immune-suppressive-associated genes, such as CCL17 and PDCD1. Conversely, the remaining patient displayed an uncommon absence of CD40-CD40LG activity but harbored a phagocytosis-associated tumor-associated macrophage (TAM)-enriched TME, which correlated with significantly reduced relapse rates and longer event-free survival (EFS), highlighting the critical value of precise TME stratification in tailoring AITL therapeutic strategies. Finally, trajectory analysis unveiled a distinct trajectory of molecular evolution within this TME landscape. Collectively, our findings illuminate the heterogeneity and prognostic implications of the TME in AITL, providing a robust foundation for the rational design of targeted immunotherapeutic approaches. These insights may substantially advance the development of personalized treatment strategies for AITL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47887a6371994feccf99a507b06079fcceafb4cb" target='_blank'>
              Revealing tumor microenvironmental heterogeneity and prognostic value in angioimmunoblastic T-cell lymphoma via spatial transcriptome sequencing
              </a>
            </td>
          <td>
            Xiang Zhang, Yong Sun, Duo Wu, Fang Yu, Hanjin Yang, X. Ye, Juying Wei, Xuewu Zhang, Yanan Zhu, Yunfei Lv, Zijing Xu, Yuxiang Chen, Hongyan Tong, Jie Jin, Xiaofei Ye, Wenjuan Yu
          </td>
          <td>2026-01-09</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) enables simultaneous mapping of tissue morphology and spatially resolved gene expression, offering unique opportunities to study tumor microenvironment heterogeneity. Here, we introduce a computational framework that predicts spatial pathway activity directly from hematoxylin-and-eosin-stained histology images at microscale resolution 55 and 100 um. Using image features derived from a computational pathology foundation model, we found that TGFb signaling was the most accurately predicted pathway across three independent breast and lung cancer ST datasets. In 87-88% of reliably predicted cases, the resulting spatial TGFb activity maps reflected the expected contrast between tumor and adjacent non-tumor regions, consistent with the known role of TGFb in regulating interactions within the tumor microenvironment. Notably, linear and nonlinear predictive models performed similarly, suggesting that image features may relate to pathway activity in a predominantly linear fashion or that nonlinear structure is small relative to measurement noise. These findings demonstrate that features extracted from routine histopathology may recover spatially coherent and biologically interpretable pathway patterns, offering a scalable strategy for integrating image-based inference with ST information in tumor microenvironment studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29b8964f22d869e7cd99fbb783574fbfe821242b" target='_blank'>
              Digital Modeling of Spatial Pathway Activity from Histology Reveals Tumor Microenvironment Heterogeneity
              </a>
            </td>
          <td>
            Ling Liao, Changhuei Yang, Maxim N. Artyomov, Mark Watson, Adam Kepecs, Haowen Zhou, Alexey Sergushichev, R. Cote
          </td>
          <td>2025-12-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Prostate cancer is a highly heterogeneous disease, driven by genomic and transcriptional changes that impact disease progression and metastatic potential. The interplay between clonal evolution, transcriptional plasticity, and tumour microenvironment is, however, poorly understood. Here, we leverage and integrate single-nuclei RNA sequencing and whole-genome sequencing from 43 spatially distinct tumour samples from five patients with locally advanced prostate cancer to reconstruct clonal evolution trajectories and transcriptional changes driving metastasis at single-cell resolution. We find extensive clonal heterogeneity, including both monophyletic and polyphyletic metastatic dissemination, and ongoing clonal evolution in the primary tumour after metastatic spread. Metastatic seeding converges on disease trajectories involving both genomic and transcriptional changes, including androgen receptor independence and activation of estrogen-, WNT- and JAK-STAT- pathway activity, in spatially distinct areas. Our findings suggest an intricate interplay between clonal evolution and cellular plasticity driving metastatic seeding and point toward more integrative prognostic markers for improved patient management. The impact of tumour heterogeneity on metastatic potential in prostate cancer remains poorly understood. Here, the analysis of single nuclei RNA sequencing and whole-genome sequencing from samples from five patients suggests an interplay between clonal evolution and cellular plasticity driving metastatic seeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea384ac37d8c4c2acd2bd5fc9ab62cffd7ea81c" target='_blank'>
              Clonal evolution and transcriptional plasticity shape metastatic dissemination routes in prostate cancer
              </a>
            </td>
          <td>
            Migle Mikutenaite, Evdoxia Karadoulama, F. Favero, A. Locallo, Francisco German Rodriguez Gonzalez, Daria Kiriy, Tina Keshavarzian, K. Furlano, M. Graefen, Robert G. Bristow, Clarissa Gerhäuser, C. Plass, G. Sauter, R. Simon, M. Lupien, S. Minner, Thorsten Schlomm, Joachim Weischenfeldt
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is a highly prevalent malignancy with poor prognosis, largely driven by lymph node metastasis (LNM). Despite its clinical significance, the underlying mechanisms of LNM remain elusive. In this study, we used single-cell transcriptomic data to dissect the cellular and molecular interactions within metastatic lymph nodes (MET). Specifically, we analyzed Single-cell RNA sequencing (scRNA-seq) data from GSE195655, GSE140042, GSE227156, and GSE159929 (n = 41) to delineate cellular heterogeneity, intercellular communication networks, and functional enrichment in primary tumors (PT), MET, and non-metastatic lymph nodes (LN). Our analysis identified a subpopulation of tumor-associated macrophages (TAMs) distinctly enriched in MET, characterized by high expression of SPP1. Functional analysis revealed that this TAM subpopulation promotes angiogenesis through specific ligand-receptor interactions with endothelial cells (ECs), involving the SPP1-ITGa9b1 and FN1-ITGa2b1 signaling axes. Furthermore, we leveraged bulk RNA-seq for prognostic research. Immunohistochemistry (IHC) confirmed the increased density and number of blood and lymphatic vessels in MET. Spatial analysis via multiplex immunohistochemistry (mIHC) confirmed the preferential localization of SPP1+ TAMs near ECs in MET. These results suggest a potential communication between TAMs and ECs that contributes to LNM in HNSCC, providing critical insights for HNSCC prognosis and precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa35cd24399d837bd70f92cab5dbd70391a0453c" target='_blank'>
              Single-cell identification of an endothelial cell proximal SPP1+ macrophage population defines the metastatic vascular niche in lymph nodes
              </a>
            </td>
          <td>
            Zhentao Lao, Rukeng Tan, Mailikanmuhan Maitikuerban, Xiaofang Xu, Sien Zhang, Jingxin Ma, Dikan Wang, Guiqing Liao, Le Yang
          </td>
          <td>2025-12-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d59ad194fb04ebf384288815f79948f2c11cf9f" target='_blank'>
              Humanized glioblastoma patient-derived orthotopic xenografts recreate a locally immunosuppressed human immune ecosystem amenable to immunotherapeutic modulation
              </a>
            </td>
          <td>
            P. M. Moreno-Sanchez, A. Oudin, Batuhan Kısakol, Heiko Dussmann, E. Klein, V. Baus, Camille Rolin, Carole Seguin-Devaux, Mahsa Rezaeipour, Aurélie Poli, A. Michelucci, J. Paggetti, E. Moussay, Jochen H. M. Prehn, Simone P. Niclou, A. Gołȩbiewska
          </td>
          <td>2025-11-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Interactions among tumor, immune, and vascular niches play major roles in glioblastoma (GBM) malignancy and treatment responses. The composition and heterogeneity of extracellular core matrix proteins (CMPs) that mediate such interactions are not well understood. Here, we present an analysis of the clinical relevance of CMP expression in GBM at bulk, single-cell, and spatial anatomical resolution. We show that CMP enrichment is associated with worse patient survival, specific driver oncogenic alterations, mesenchymal state, pro-tumor immune infiltration, and immune checkpoint expression. Matrisome expression is enriched in vascular and leading edge/infiltrative niches that are known to harbor glioma stem cells. Finally, we identify a 17-gene CMP signature, termed Matrisome 17 (M17), which is a stronger prognostic factor compared to MGMT promoter methylation status as well as canonical subtypes, and importantly, may predict responses to PD1 blockade. Patient stratification based on matrisome profiles can contribute to the selection and optimization of treatment strategies. A comprehensive genomic analysis of glioblastoma tumors reveals matrisome enrichment associated with mesenchymal phenotypes and pro-tumor immune signatures, a matrisome-based classification of GBM patients, and a prognostic 17-gene matrisome signature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984fecbf9126f6a0596ab278d28e0d91cbca56fc" target='_blank'>
              A prognostic matrix gene expression signature defines functional glioblastoma phenotypes and niches
              </a>
            </td>
          <td>
            Monika Vishnoi, Zeynep Dereli, Zheng Yin, Elisabeth Kong, Meric Kinali, Kisan Thapa, O. Babur, Kyuson Yun, Nourhan Abdelfattah, Xubin Li, B. Bozorgui, M. Farach‐Carson, Robert C. Rostomily, Anil Korkut
          </td>
          <td>2026-01-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6aac86da3cbb3bd85a8ebe57c0ff220d334a71" target='_blank'>
              Circulating immune profiling reveals impaired monocyte states and trajectories driving immunosuppression in glioblastoma
              </a>
            </td>
          <td>
            A. Scafidi, T. Kaoma, Claudia Cerella, Eleonora Campus, K. Grzyb, Bakhtiyor Nosirov, Frida Lind-Holm Mogensen, E. Klein, Raul da Costa, A. Skupin, Frank Hertel, G. Berchem, Michel Mittelbronn, A. Cosma, Beatrice Melin, Simone P. Niclou, Petr V Nazarov, A. Gołȩbiewska, Aurélie Poli, A. Michelucci
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Pleomorphic rhabdomyosarcoma is a rare soft-tissue tumor that occupies an uncertain middle ground between rhabdomyosarcoma and undifferentiated pleomorphic sarcoma. With its relative rarity, aggressiveness, and lack of detailed characterization, it presents a challenging task for therapeutic treatment. In this case study, we use single-cell transcriptomics to investigate the heterogeneous landscape of pRMS and the tumor microenvironment. We demonstrate that the tumor populations in pRMS have a clear division into myogenic and non-myogenic clusters, with the non-myogenic clusters having more numerous communication links with the immune populations. All pRMS tumor clusters use the MIF-CD74 pathway to suppress the immune response, while APP, PTN, and CXCL12 signaling are employed predominantly by the non-myogenic tumor clusters. The cytotoxic T cells in pRMS bear markers of exhaustion (LAG3, HAVCR2, EOMES), and the macrophages express myeloid checkpoint-related genes (SIGLEC1, SIRPA, CSF1R, HAVCR2). This transcriptomic data suggests that targeting MIF and APP signaling in pRMS may have therapeutic potential; however, studies on multiple-patient cohorts, protein verification, and in vitro and in vivo validation are still needed for clinical actionability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/247589e7166c26c19a2c0ff799f9b8b5c5af60e4" target='_blank'>
              Single-Cell Sequencing Reveals Novel Tumor Populations and Their Interplay with the Immune Microenvironment in a Pleomorphic Rhabdomyosarcoma
              </a>
            </td>
          <td>
            Elena E. Kopantseva, A. Ikonnikov, Maxim E. Menyailo, Timur I. Fetisov, A. Korobeynikova, K. Kirsanov, A. Tararykova, B. Bokhyan, Nikolay A. Kozlov, M. Yakubovskaya, E. Denisov
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11151ef60d8479a0c0f55b160c123009b32bf93" target='_blank'>
              Single-cell transcriptomic analysis reveals the heterogeneity and functional characteristics of macrophage subpopulations in colon cancer.
              </a>
            </td>
          <td>
            Jiazheng Zhao, Baochun Wang, Xiao Li, Chunting Wei, Yi Min, Dayong Wang
          </td>
          <td>2025-11-23</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a powerful technique for understanding cellular diversity, but processing large patient cohorts to identify phenotype-associated cell populations remains challenging. Here, scPER (Estimating cell Proportions using single-cell RNA-seq Reference), a rigorous approach combining adversarial autoencoder and extreme gradient boosting to estimate tumor microenvironment cell compositions and identify phenotype-associated subclusters for bulk RNA-seq samples. Integrating scRNA-seq datasets from diverse studies, scPER constructed comprehensive reference panels and disentangled confounders from true signals. scPER achieved superior accuracy in cellular proportion estimation compared to CIBERSORTx, BayesPrism, Scaden, MuSiC, SCDC, DeSide and ReCIDE. It showed high accuracy in predicting metastatic melanoma immunotherapy response and identified a critical T cell subcluster expressing FCRL3 and SLAMF7. In metastatic urothelial cancer, scPER predicted TGFβ-mediated inhibition of CD4 naïve T cells to diminish PD-L1 checkpoint blockade efficacy. scPER enables robust integration of scRNA-seq datasets to estimate cellular proportions across tumors and identify clinically relevant cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8390e37b5363dcdab22134564264151d0e0926d0" target='_blank'>
              scPER: A Rigorous Computational Approach to Determine Cellular Subtypes in Tumors Aligned With Cancer Phenotypes From Total RNA Sequencing.
              </a>
            </td>
          <td>
            Bingrui Li, Xiaobo Zhou, R. Kalluri
          </td>
          <td>2025-11-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The tumor immune microenvironment (TIME) is closely involved in tumor initiation, malignant progression, immune escape, and response to immunotherapy. With the continued development of high-throughput sequencing technologies, transcriptomic approaches have become essential for examining the cellular and molecular features of the TIME. Bulk RNA sequencing offers tissue-level gene expression profiles and allows the estimation of immune cell composition through computational deconvolution. Single-cell RNA sequencing provides finer resolution, revealing cellular heterogeneity, lineage relationships, and functional states. Spatial transcriptomics (ST) retains the native anatomical context, making it possible to localize gene expression patterns and cell–cell interactions within intact tissues. These approaches, when considered together, have shifted TIME research from averaged measurements toward a more detailed and mechanistic understanding. This review summarizes the principles, applications and limitations of bulk, single-cell and spatial transcriptomic methods, highlighting emerging strategies for integrative analysis. Such multi-scale frameworks are increasingly important for studying immune dynamics and may contribute to the development of more precise biotechnological and immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a95188583167d2d5ed84b42177b48c6d850371a9" target='_blank'>
              Multi-Scale Transcriptomics Redefining the Tumor Immune Microenvironment
              </a>
            </td>
          <td>
            Jing Sun, Yingxue Xiao, Lingling Xie, Dan Qin, Yue Zou, Yingying Liu, Yitong Zhai, Minyi Zhang, Tong Li, Youjin Hao, Bo Li
          </td>
          <td>2026-01-15</td>
          <td>BioTech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cutaneous T-cell lymphoma (CTCL) is a type of lymphoma that affects the skin and has various subtypes with different features. It can be either slow-growing or aggressive and currently has limited treatment options, causing a significant impact on patients’ quality of life. However, more research is needed to fully understand the underlying causes of the disease. Single-cell sequencing technology shows promise in comprehending the complex genome and microenvironment of CTCL. By analyzing the molecular status of each tumor cell, this approach can provide insights into tumor heterogeneity, microenvironment composition, and cell state transitions, tailoring treatment strategies to individual patients. In this review, we highlight recent research progress in CTCL via single-cell sequencing, focusing on gene alterations and related biomarkers, disease origin and progression, and the tumor microenvironment. We aim to provide examples of single-cell studies that can help inform new drug discovery and contribute to improved treatment strategies for patients with CTCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dcc7ae8a9d6d7170ba10b8b26cd4ca7c7a1a2cc" target='_blank'>
              Single-cell sequencing reveals the complexity of cutaneous T-cell lymphoma
              </a>
            </td>
          <td>
            Jun Du, Hao-Ran Yuan, Zi-Han Lin, Yi-En Pan, Shu-Qi Zhang, Zi-Jin Ye, Jun-Ting Lyu
          </td>
          <td>2025-11-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background/Objectives: Achieving complete resection (R0) during secondary cytoreduction (SCR) is a pivotal prognostic factor for patients with platinum-sensitive relapsed ovarian cancer (PSROC). Methods: By integrating single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and functional assays on primary cells, we identified tumor microenvironment (TME) heterogeneity, particularly the presence of distinct subpopulations of cancer-associated fibroblasts (CAFs), as a critical determinant of surgical outcomes for SCR. Results: We characterized a previously unrecognized CAF-C7 subpopulation that was selectively enriched in tumors from non-R0 patients. Mechanistically, CAF-C7 promoted tumor cell migration, suppressed anoikis, and facilitated angiogenesis via the IGF1-IGF1R signaling axis, thereby contributing to multifocal recurrence and reducing the likelihood of complete resection. Importantly, the inhibition of the IGF1-IGF1R pathway effectively attenuated the pro-tumorigenic functions of CAF-C7. Conclusions: These findings uncover a novel cellular driver of unfavorable surgical outcomes in PSROC and suggest promising biomarkers and therapeutic targets for improving patient stratification and treatment in the context of SCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c69a7316901a4f26ec9560008414f1bd2cba29" target='_blank'>
              Single-Cell Analysis Links C7+ Cancer-Associated Fibroblasts with Incomplete Resection in Platinum-Sensitive Relapsed Ovarian Cancer
              </a>
            </td>
          <td>
            Longxia Li, Shangbing Gao, Yilizhati Maimaiti, Lifeng Lin, Xiaoxia Xing, Wei Wu, Yulian Chen, Mei-Chun Cai, Guanglei Zhuang, Rongyu Zang
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Y. Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, R. Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Mucosa-associated lymphoid tissue (MALT) initiates immune responses at mucosal entry sites. Within MALT, microfold (M) cells sample luminal antigens and deliver them to underlying immune cells. Despite their functional importance, few tools enable selective manipulation of M cells in vivo. Here we report the generation and characterization of a peptidoglycan recognition protein 1 (Pglyrp1)-Cre knock-in mouse designed to allow conditional genetic access to M cells. Using Rosa26-tdTomato reporter mice, we found strong Pglyrp1 promoter activity in gut epithelial cells, including goblet and M cells, whereas activity in nasal-associated lymphoid tissue (NALT) was more heterogeneous and skewed towards immune cells, particularly neutrophils. To functionally interrogate Pglyrp1-expressing cells, we performed Cre-mediated ablation using three DTA-based models. The Rosa26GFP-DTA line caused marked perinatal lethality in double-positive pups, suggesting essential roles for Pglyrp1-positive cells early in life. In contrast, Rosa26DTA and Rosa26iDTR crosses produced minimal depletion of mucosal populations, including M cells, even at the highest non-lethal diphtheria toxin dose. These findings demonstrate tissue-specific Pglyrp1 promoter activity and highlight challenges in achieving M cell-specific targeting. Although not M cell-restricted, the Pglyrp1-Cre mouse provides a useful tool for manipulating Pglyrp1-expressing lineages and probing their roles in mucosal homeostasis and immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/218a54897e416e59a97b965db206c22a26ae6f64" target='_blank'>
              Pglyrp1-Cre Marks Distinct Epithelial and Immune Lineages Across Mucosal Sites
              </a>
            </td>
          <td>
            Samuel Alvarez-Arguedas, M. Shiloh
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131d5075adfc8a892d5a30b581d5ddea0c0afd87" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular heterogeneity and prognostic subtypes in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhen Han, Yuan Gao, Yuxing Zhao, Jiahong Wu, Jia Chen, Jinling He, Hong Bai, Liang Qiao, Yali Ren, Lei Sun
          </td>
          <td>2025-12-22</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Colorectal carcinoma exhibits high heterogeneity, comprising subtypes that show poor efficacy of T cell-based immunotherapies, such as programmed cell death protein 1 (PD-1) checkpoint inhibitors. Although natural killer (NK) cells are considered a promising approach for cancer immunotherapy, it remains unclear what molecular mechanisms drive NK cell activation or suppression within the tumor microenvironment. Moreover, limitations in human tumor models that reflect the diversity of individual patient tumors hinder the ability to effectively select patients who would benefit most from NK cell-based therapies. Methods Here, we established a co-culture platform of genetically diverse colorectal cancer (CRC) patient-derived organoids (PDOs) with primary allogeneic NK cells. We performed bulk RNA sequencing analysis of sorted NK cells after exposure to PDOs and aligned gene expression signatures derived from our findings with publicly available single-cell RNA sequencing data of NK cells from peripheral blood and CRC tissues of patients. Moreover, we evaluated identified pathways using flow cytometry and IncuCyte live-cell imaging analysis to quantify phenotypic alterations and NK cell-mediated killing of PDOs over time, respectively. Ultimately, we tested CRISPR-Cas9-edited NK cells and PDOs, small molecule compounds, and clinically relevant monoclonal antibodies (mAbs) to increase NK cell potency. Results On co-culture, NK cells acquired common transcriptional signatures related to hypoxia and transforming growth factor-beta (TGF-β), similar to NK cells infiltrating CRC tissues of patients. In addition, we observed patient-specific differential PDO susceptibilities to NK cell-mediated lysis. Major histocompatibility complex class I deficiency and natural killer group 2, member D (NKG2D)-ligand expression on PDOs facilitated NK cell-mediated cytotoxicity, and induced phenotypic NK cell diversification related to activation and the acquisition of inflammation and tissue-residency-related transcriptional signatures. Genetic or pharmaceutical targeting of hypoxia-inducible factors HIF1A/EPAS1 or TGF-βR1, or the addition of anti-CEACAM1 mAbs, enhanced NK cell-mediated PDO killing or activation, respectively. Conclusions The NK cell/PDO co-culture platform allows the identification of both common and patient-specific impacts of the tumor microenvironment on NK cell function and can aid the development of patient-tailored immunotherapies. The majority of CRC (CMS2/CMS3) PDOs from our cohort were susceptible to NK cell-mediated killing and induced NK cell activation, highlighting the potential of NK cells for CRC immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfdbb13391ad7497b66effc8a4643668f2be84a" target='_blank'>
              Colorectal cancer organoids drive hypoxia, TGF-β, and patient-specific diversification of NK cell activation programs
              </a>
            </td>
          <td>
            Andreas von Kries, Irene Garcés-Lázaro, Bianca M Balzasch, Carsten Sticht, Indra A Shaltiel, Kim E. Boonekamp, Annika Sams, Alessia Triassi, Tomáš Hofman, Elke Burgermeister, J. Betge, Matthias P. Ebert, M. Boutros, Laura Helming, A. Stojanovic, A. Cerwenka
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Brain metastasis is a major contributor to melanoma-related mortality, arising from both genomic alterations and dynamic adaptations to the brain microenvironment. Tumor cells must cross the blood–brain barrier and exploit brain-specific signaling and metabolic landscapes to establish metastases. Brain metastases display molecular and phenotypic programs distinct from other organs, reflecting pre-existing and acquired mutations during progression. We analyzed primary melanoma, metastatic melanoma, and metastatic brain tumors genomic datasets to identify mutated proteins and signaling pathways that are common to or distinct among these tumor types. Our comprehensive computational analyses uncovered key pathways and proteins associated with brain metastasis. Brain metastases exhibit unique mutational landscapes and convergent alterations in neurotrophin, MAPK, and PI3K/AKT signaling compared with primary and metastatic melanoma. Proteins especially frequent in brain metastases pathways support survival, proliferation, and adaptation in the brain microenvironment. Pharmacology-wise, rational strategies such as combining c-KIT and VEGFR inhibitors can suppress converging oncogenic pathways and mitigate resistance in advanced melanoma. Co-targeting neurotrophin receptors with MAPK pathway members, particularly in combination with BRAF inhibitors, appears a promising approach for treating melanoma brain metastases. Our findings underscore the molecular heterogeneity of melanoma brain metastases, shaped by interconnected signaling networks, and reveal therapeutic targets that may diverge from those in other metastatic sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d4ecead949fc8a270a159656433a0bf5e5ad895" target='_blank'>
              Cellular Mechanisms Underlying Melanoma Brain Metastasis
              </a>
            </td>
          <td>
            B. R. Yavuz, Hyunbum Jang, Ruth Nussinov
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Abstract Background and objectives Recent molecular landscape studies characterizing lung cancer mostly utilize single tumor regions, ignoring tumor heterogeneity and evolutionary patterns. To address this issue, the TRAcking NSCLC Evolution through therapy (TRACERx) Consortium established a multiregion sampling of Western lung cancer to investigate its longitudinal evolutionary dynamics. Due to ancestral differences, there are still major gaps in understanding oriental lung cancer. Methods We have established a prospective cohort of patients with non-small cell lung cancer (NSCLC) named Chinese lung cancer evolution and microenvironment tracking under therapy (CLEVER) study. Discussion By acquiring comprehensive genetic variation data and clonal events from multiple regions of the tumor, we aim to decipher the molecular evolution code of Chinese lung cancer. The analysis of cell composition and spatial structure further characterizes the remodeling effects of tumor evolution and neoadjuvant therapy on the tumor microenvironment, helping to identify important markers that influence treatment effectiveness and recurrence. By monitoring mutations using a personalized minimal residual disease (MRD) panel from tumors at baseline, we can trace evolutionary significance and clonality, aiding in therapy decisionmaking. The CLEVER study significantly contributes to the development of personalized treatment strategies for lung cancer patients, enhancing outcomes and tailoring interventions to individual patient profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75f2f25eef8688466fb037f4fd6a8d16c5aeeb96" target='_blank'>
              Protocol for Chinese lung cancer evolution and microenvironment tracking under therapy study
              </a>
            </td>
          <td>
            Wenxiang Wang, Yue He, Xiaoqiu Yuan, Lin Weng, Jing Bai, Xu Liu, Li Han, Jun Xu, Shuai Wang, Shanshan Lu, Jian Bai, Hui Quan, Qingyun Liu, Xinrui Li, Yuxuan Xiao, Yun Wang, Qingna Zhang, Na Zhou, Yanyan Hou, Hao Li, Y. Li, Fan Yang, Jun Wang, Kezhong Chen
          </td>
          <td>2025-12-01</td>
          <td>Journal of Translational Internal Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Thymic epithelial tumors (TETs) exhibit marked heterogeneity that challenges clinical management and prognostic evaluation. By consensus clustering of immune-related gene expression from public datasets and an in-house validation cohort, we identified two immune subtypes: a lymphocyte-rich subtype (LRS, n = 86) characterized by strong T-cell activity and favorable prognosis, and a myeloid/stromal-rich subtype (MSRS, n = 33) defined by immunosuppressive features, stemness, oncogenic alterations, and poorer outcomes. An APC gene-based classification panel demonstrated robust performance in distinguishing these subtypes, as confirmed by ROC curve analysis and validated in the Ruijin cohort using FFPE-RNA sequencing and multiplex immunofluorescence. Single-cell transcriptomics further delineated subtype-specific tumor microenvironment features and revealed that MSRS tumors suppressed CD8+ T-cell function through MIF-mediated mechanisms. Collectively, these findings establish a clinically relevant immune classification of TETs that integrates molecular, immunological, and clinical features, providing insights into tumor heterogeneity and supporting the development of personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f6f13a57b79f0fdbdce54ed88ac9cc1b39fa61" target='_blank'>
              Immune landscape-driven subtyping reveals distinct microenvironment and prognostic profiles in thymic epithelial tumors
              </a>
            </td>
          <td>
            Yajie Zhang, Tong Lu, Yeke Huang, Xipeng Wang, Xiaoyan Chen, Wei Guo, Youqiong Ye, Hecheng Li
          </td>
          <td>2025-12-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) exhibits substantial genetic and functional heterogeneities. We generated clonal spheroids (CS) from HNSCC cell lines and patient tumor cells, uncovering distinct functional differences. Hyperproliferative spheroids (HRPS) showed increased proliferation and tumorigenic potential, whereas hypoproliferative spheroids (HOPS) demonstrated enhanced migration and resistance to cisplatin and radiation. In patient tumor-derived spheroids, spheroid size distribution analysis across various TNM stages indicated that early-stage tumors (T2N0M0) predominantly formed HRPS, while later-stage tumors (T4N2M0) exhibited a higher prevalence of HOPS. Single-cell RNA sequencing (scRNA-seq) identified six clusters in the HOPS and four in the HRPS, each defined by unique gene expression profiles and pathway enrichment. HOPS were enriched with fibroblast, mesenchymal, and keratinizing epithelial cells, indicating a metastatic phenotype. They also expressed genes associated with stemness (KRT15 and ALDH3A1). Conversely, HRPS was characterized by proliferative genes, including TOP2A, AURKA, CKS2, AREG, and ANLN, suggesting an advanced HNSCC proliferative signature. Despite being generated from only two cells/well, the various clusters in both spheroid types mirrored the diverse tumor microenvironment in HNSCC, including neoplastic, epithelial, basal, fibroblast, and mesenchymal cells. Notably, the gene expression profiles of Cluster 3 in the HOPS and Cluster 1 in the HRPS closely matched the expression patterns observed in the HNSCC-TCGA dataset. These clusters also displayed a high transcriptional correlation between patient tumors and their xenografts, reinforcing the clonal nature of HNSCC’s genetic and functional diversity of HNSCC. This study provides a deeper understanding of the genetic and functional diversity of HNSCC tumors, which can aid in the development of more precise diagnostic tools by identifying distinct tumor subtypes. Distinguishing between hyper- and hypoproliferative spheroids could improve prognostic accuracy, helping to predict tumor behavior and patient outcomes. Moreover, the identification of specific gene expression profiles associated with proliferation, metastasis, and treatment resistance may guide personalized therapeutic strategies, enabling targeted treatment based on the unique tumor characteristics of individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/942cca58b9f76ca4806f68d559d87541dfee33c1" target='_blank'>
              Uncovering functional divergence and cellular clusters with specific gene signatures in HNSCC clonal spheroids
              </a>
            </td>
          <td>
            Jyoti Pandey, Md. Zubbair Malik, Ritis K Shyanti, Palak Parashar, Praveen K. Kujur, Deepali Mishra, Dhanir Tailor, Jee Min Lee, T. Kataria, Deepak Gupta, Hitesh Verma, Anshu Yadav, S. Kateriya, Sanjay V. Malhotra, Vibha Tandon, Rupesh Chaturvedi, Rana P. Singh
          </td>
          <td>2025-12-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="

 The complexities of nucleotide metabolism in breast cancer (BC) cells are not yet fully understood. A deeper exploration of the various tumor subpopulations and their interactions with the tumor microenvironment (TME) could provide important insights into these clinically relevant signaling pathways.



 We integrated five distinct single-cell enrichment scoring methodologies to conduct a comprehensive enrichment analysis of BC cell populations. The analytical findings underwent subsequent validation using an independent single-cell cohort. Tumor cell clusters were categorized based on their average enrichment scores. Functional analyses were carried out using several tools, including CellChat, Monocle, CopyKAT, SCENIC, and CytoTRACE. The RCTD method was then employed to map the single-cell clusters onto spatial transcriptomics data, facilitating the evaluation of cellular dependencies and pathway activities to differentiate tumor cell subtypes. A prognostic framework was subsequently established using large-scale transcriptomic datasets, enabling prediction of immunotherapy responsiveness. Experimental validation further confirmed expression patterns of pivotal genes implicated in therapeutic outcomes.



 Tumor cells exhibit significantly upregulated nucleotide metabolic activity, enabling their classification into two distinct subgroups: NUhighepi and NUlowepi. Cells within the NUhighepi subgroup demonstrate pronounced malignant phenotypes. Intercellular communication analysis performed with the stLearn platform revealed robust interactions between NUhighepi cells and fibroblasts. Supporting this finding, spatial transcriptomic analysis via MISTy revealed a distinct dependency of NUhighepi on fibroblasts. A robust prognostic model, developed using various machine learning algorithms, was able to predict survival outcomes and responses to immunotherapy. Furthermore, targeted drugs were identified for both the high and low scoring groups. Experimental investigations confirmed the expression of core genes in different breast cancer cells.



 In conclusion, we developed a nucleotide metabolism-derived prognostic signature for BC, with DCTPP1 highlighted as a promising biomarker and therapeutic target. These findings provide a valuable framework for early clinical intervention and show promising potential for predicting responses to immunotherapy in BC patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502ca564395f021e293ba0e223c1f50472f85541" target='_blank'>
              Spatial transcriptome and single-cell sequencing reveal the role of nucleotide metabolism in breast cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yuejun Pan, Yong Xu, Ke Gong, Songlin Yuan
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) constitute the most abundant immune cell population in the glioblastoma (GBM) tumor microenvironment. These macrophages critically influence multiple pathological processes in GBM, including recurrence, drug resistance, and immune evasion. However, the role of TREM1 in mediating proneural-to-mesenchymal transition (PMT) in TAMs remains unclear. Methods Single-cell RNA sequencing analysis of 24,366 cells from seven GBM patients identified nine distinct cellular populations, including TAMs and malignant cells. Deconvolution analysis of TCGA mRNA-seq data complemented these findings. Differential expression analysis, weighted gene co-expression network analysis, and LASSO-Cox regression consistently identified TREM1 in TAMs as a PMT-associated biomarker. A co-culture system demonstrated TREM1’s influence on GBM cell malignancy and PMT progression in vitro, while intraperitoneal LP17 administration assessed its impact on tumor growth. Results High TAM infiltration correlated significantly with poor clinical outcomes in GBM patients. Integrated scRNA-seq and mRNA-seq analyses established TREM1 in TAMs as a key PMT biomarker. LP17-mediated TREM1 inhibition attenuated PMT through modulation of the TLR4/PI3K/AKT/mTOR pathway and reduced tumor growth in vivo. Conclusion TREM1 functions as an oncogenic biomarker in TAMs that drives GBM PMT via the TLR4/PI3K/AKT/mTOR axis, with inhibition of TREM1 demonstrating therapeutic potential in both ex vivo and in vivo models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/deea03500e24594a9c7271f07a79557e062e3249" target='_blank'>
              Unveiling the novel value of TREM1 in the proneural-mesenchymal transition of glioma via tumor-associated macrophages
              </a>
            </td>
          <td>
            Chao Zhang, Ben Hu, Chao Wang, Shiqiang Hou, Ning Lin
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Lung cancer is a highly heterogeneous disease, and the tumor microenvironment (TME) characteristics are closely related to disease progression and treatment outcomes. We elucidated the cell type-specific transcriptome landscape of cancer cells and the effect of the TME on lung adenocarcinoma (LUAD). Methods Single-cell RNA sequencing of the LUAD tissues and matched adjacent normal tissues of three patients in the early-stage via histopathological and immunohistochemical confirmation. Results The results revealed the landscape of 68,579 cells in LUAD microenvironment, and highly heterogeneous AT2 cells are crucial source of lung epithelial cell carcinogenesis. Genes KRT81, SPP1, PCDH7, SLC2A1, and TET1 were significantly upregulated in tumor tissues and associated with poor prognosis and survival, providing insights for exploring lung cancer biomarkers in future studies. Trajectory analysis identified ERBB4, SEMA4A, GCNT2 and SOX4 as key factors in AT2 cells that may promote cell proliferation, migration, or epithelial-mesenchymal transition (EMT) during the progression of lung adenocarcinoma (LUAD). We elucidated the transcription factor–target gene regulatory network involving NKX2-1 and TEAD1 in malignant tumor cells derived from AT2 cells. The findings indicated that malignant AT2 cells regulate communication between epithelial and immune cells in the TME by predicted FN1-CD44 and CADM1-CADM1 ligand-receptor interactions, which ultimately suppresses the host immune response. Discussion This comprehensive single-cell analysis increases our understanding of AT2 cells molecular and dynamics of metastatic lung cancer. In summary, single-cell RNA profiling of LUAD offers valuable prognostic insights based on AT2 cell types and identifies potential biomarkers for therapeutic responses, aiding the future development of LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68e15868fa220169dfea67c353cad3603481040a" target='_blank'>
              Single-cell RNA sequencing reveals heterogeneity among AT2 epithelial cells in the lung adenocarcinoma microenvironment
              </a>
            </td>
          <td>
            Zimeng Cao, Xuefeng Bai, Jing Li, Jinru Cai, Gang Cao, Yuncai Xiao, Wei Zhou
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) represents a significant global health challenge due to its molecular heterogeneity and the immunosuppressive nature of its tumor microenvironment (TME). Tumor-associated macrophages (TAMs), particularly the M2-polarized subset, are crucial in HCC progression, contributing to immune evasion, angiogenesis, and metastasis. However, the mechanisms driving TAM reprogramming and their metabolic interactions within the TME remain poorly understood. In this study, we applied single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to identify a novel metabolic-immune axis in HCC. Our findings revealed increased oxidative phosphorylation (OXPHOS) in tumor cells, which correlated with enhanced ribosomal activity in M2 macrophages, suggesting a metabolic coupling between the two. Tumor cells, under elevated OXPHOS, secrete factors that recruit macrophages and promote M2 polarization, thereby reinforcing immune suppression. Further analysis of The Cancer Genome Atlas (TCGA) dataset led to the development of a prognostic model based on OXPHOS-related genes, showing strong predictive power across multiple cohorts. These results suggest a novel OXPHOS-ribosome-M2 polarization axis and offer potential therapeutic targets for immunometabolic therapies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d98d705964e6108c751cf9dce67e7948e856f2d" target='_blank'>
              M2 macrophages and tumor cells engage in a metabolic feedback loop to drive HCC progression
              </a>
            </td>
          <td>
            Yimin Nong, Xiaolong Chen, Baoxin Zhang, Zhenying Lin, Wuriniletu, Chengyou Du, Qiang Liu
          </td>
          <td>2025-11-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is an aggressive thoracic disease characterized by rapid proliferation and early metastatic spread. The survival outcomes for SCLC patients remain notoriously poor, underlining that only modest improvements have been achieved in clinical settings to date. However, insights gained from human tumors and preclinical models in recent years have shed light on the heterogeneous molecular profile of SCLC. Numerous research groups have, therefore, begun to stratify SCLC into subgroups based on differential transcription factor expression, the tumor immune microenvironment, and other criteria. Since SCLC subtypes show major differences in their molecular landscape and biological behavior, they may offer unique therapeutic vulnerabilities and serve as a framework for future personalized clinical trials. Here, we summarize impactful classification attempts from the last ten years, highlighting discrepancies and connections between the nomenclature of each study, and expound upon relevant factors of SCLC biology influencing subtype composition and plasticity. This review delves into the implications of subgrouping for understanding and treating SCLC as well as potential future directions for SCLC research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc26cc7e8a1e8ed5ed66483e07fdc9dc2336ee3" target='_blank'>
              Small Cell Lung Cancer Classification: Unraveling Heterogeneity to Enable Personalized Treatments.
              </a>
            </td>
          <td>
            K. Boettiger, I. Kovács, Lilla Horvath, B. Ernhofer, K. Schelch, M. Pozonec, V. Teglas, Clemens Aigner, Evelyn Megyesfalvi, Ferenc Rényi-Vámos, K. Bogos, C. Lang, Abigail J. Deloria, C. Gay, L. Byers, J. Sage, F. Hirsch, Z. Megyesfalvi, B. Dome
          </td>
          <td>2025-12-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Phenotypic plasticity plays a key role in cancer progression and metastasis, enabling cancer cells to adapt and evolve, but precisely how distinct axes governing phenotypic plasticity interact to shape tumour progression and patient outcomes remains unclear. We investigated five major interconnected axes of plasticity in ER-positive (ER+) breast cancer: Metabolic Reprogramming, Epithelial-to-Mesenchymal Plasticity (EMP), Luminal–Basal (Lineage) Switching, Stemness, and Drug-resistance using network dynamics simulations, integrative bulk and single-cell transcriptomic analyses and patient survival analyses. We show that these axes are not independent but drive one another, forming two mutually inhibiting ‘teams’ of nodes enabling specified cellular behaviour. One team (favouring high glycolysis, stem-like, basal-like, mesenchymal/hybrid and tamoxifen-resistant phenotype) was found to be associated with aggressive progression and worse survival. On the other hand, the opposing team (favouring high oxidative phosphorylation, non-stem-like, luminal-like, epithelial and tamoxifen-sensitive phenotype) correlated with better outcomes. Importantly, altering one axis of plasticity often drove coordinated responses along other axes and vice versa. Our findings establish phenotypic plasticity in cancer as a coordinated, multi-axis dynamical process, thus suggesting novel strategies to disrupt systems-level reprogramming enabling metastasis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acaf4b9f2872701e5c44aad69b0ee83629b9228" target='_blank'>
              Interconnected axes of phenotypic plasticity drive coordinated cellular behaviour and worse clinical outcomes in breast cancer
              </a>
            </td>
          <td>
            Ritesh Kumar Meena, Yijia Fan, Soundharya Ramu, Anjaney J. Pandey, Abhinav Kannan, Yashvita Subramanian, Yukta Subramanian, J. T. George, M. Jolly
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Although well-differentiated thyroid carcinoma (WDTC) is characterized by a robust treatment response, aggressive subtypes, such as anaplastic thyroid carcinoma (ATC), remain highly lethal. To understand thyroid cancer evolution in both children and adults, we analyzed single-cell transcriptomes of 423,733 cells from 81 samples and spatially resolved key tumor and microenvironment populations across 28 tumors with spatial transcriptomics, including rare and unique composite WDTC/ATC tumors and pediatric diffuse sclerosing thyroid carcinomas. Additionally, we identified gene signatures of stromal cell populations in 5 large thyroid cancer bulk RNA-sequencing cohorts. Through this multi-institutional effort, we defined a population of POSTN+ myofibroblast cancer-associated fibroblasts (myCAFs) that are intimately associated with invasive tumor cells and correlate with poor prognosis, lymph node metastasis, and disease progression in thyroid carcinoma. We also revealed a population of inflammatory CAFs that are distant to tumor cells and are found in the inflammatory stromal microenvironment of autoimmune thyroiditis. Together, our study provides spatial profiling of thyroid cancer evolution in samples with mixed WDTC/ATC histopathology and identifies a prognostic myCAF subtype with potential clinical utility in predicting aggressive disease in both children and adults.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9674219bbd66cbdd5e33b2f92b59cb54d20e94cd" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations.
              </a>
            </td>
          <td>
            M. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey P. Caroland, Megan L. Tigue, H. Hartmann, Jean-Nicolas Gallant, Courtney J. Phifer, Andres A Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H. Belcher, K. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2026-01-09</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has revolutionized transcriptomic research, enabling the creation of detailed tissue, organ, and species-level atlases for model organisms. In Hydractinia, a cnidarian model for stem cell and regeneration studies, recent atlases have revealed key insights into cell types and developmental processes. However, these atlases remain limited in cell numbers and transcriptomic depth and cell type assignments were largely made in silico. Here, we present an updated Hydractinia single-cell atlas by integrating new datasets from fixed cells with previously published live-cell data. This expanded atlas captures over 47,000 cells from feeding polyps and stolon tissue, recovering and refining major somatic cell lineages including cnidocytes, neurons, gland cells, epithelial cells, and stem cells (i-cells), as well as identifying a novel population of putative immune cells. We investigated the spatial expression patterns of selected marker genes and validated all major cell types and several cell states. Our analyses uncovered a previously undescribed neural subtype, two spatially distinct gland cell populations, a stolon-specific cell type, and a putative immune cell cluster. Additionally, we recovered and explored a complete Hydractinia cnidocyte trajectory with two distinct endpoints, supported by spatial marker gene expression that reflects the developmental progression of cnidoblasts as they mature and migrate towards the tentacles. Subclustering of somatic i-cells revealed putative progenitor states and a potential population of true stem cells. Together, this atlas significantly advances our understanding of Hydractinia cellular diversity and dynamics, allowing us to generate new hypotheses and provide a valuable resource for the cnidarian research community and beyond.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93a014f75f68d9221cc5d7f83183c6a8717b5d32" target='_blank'>
              An updated and spatially validated somatic single-cell atlas of Hydractinia symbiolongicarpus
              </a>
            </td>
          <td>
            Jingwei Song, Danielle de Jong, Justin Waletich, A. D. Baxevanis, C. Schnitzler
          </td>
          <td>2025-12-01</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Mechanosensitive ion channel PIEZO2 translates microenvironmental forces into Ca²⁺-dependent signals that shape tumor behavior. Across cancers, PIEZO2 exhibits context-dependent roles, with evidence for both pro- and anti-tumor functions, depending on tissue type, molecular subtype, disease stage, and stromal context. This review synthesizes emerging insights into the multifaceted roles of PIEZO2 in cancer biology, summarize clinical correlations, highlight areas with concordant directionality (e.g., colorectal cancer, medulloblastoma) versus paradoxical findings (e.g., breast cancer across subtypes, gastric cancer by differentiation status), and provide a pragmatic framework for precision, context aware targeting. While PIEZO2 remains attractive for combination strategies (anti-angiogenesis, barrier modulation, dormancy escape, radio/chemo sensitization) and cancer pain interfaces, the field lacks truly PIEZO2 selective small molecule modulators. We propose feasible future priorities encompassing both pharmacologic and non-pharmacologic strategies to accelerate translational applications. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2298c7009cdc540e541e43d7dcb3fb98a9de240" target='_blank'>
              PIEZO2 in tumors: from mechanobiological switches to activity-targeted therapies
              </a>
            </td>
          <td>
            Dan-Xia Huang, Qiu-Zhi Zhou, Hong-Mei Luo, M. Nasb, Yi-Zhou Liu, Yu-Jie Yang, Hong Chen
          </td>
          <td>2025-12-01</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="OBJECTIVE
Glioma recurrence severely impacts patient prognosis, with current treatments showing limited efficacy. Traditional methods struggle to analyze recurrence mechanisms due to challenges in assessing tumor heterogeneity, spatial dynamics, and gene networks. Single-cell combined spatial transcriptomics (ST) offers innovative solutions.


METHODS
We analyzed glioma mRNA data from TCGA and single-cell and ST data from GEO. Following quality control, dimensionality reduction, clustering, and cell annotation of single-cell sequencing data, we identified cell types exhibiting significantly aberrant distributions between primary and recurrent samples by analyzing the deviation degree of Ro/e values. Fibroblasts demonstrating the greatest intergroup differences were subsequently selected as the key cellular population for further investigation. Key differentially expressed genes (DEGs) were identified via random survival forest analysis. Drug sensitivity was assessed using GDSC. Deconvolution algorithms mapped cellular spatial distribution, while PROGENy quantified pathway activity. MISTy modeling revealed cell-cell interactions.


RESULTS
Fibroblasts were the primary recurrence-associated subpopulation, with marker genes enriched in extracellular matrix and adhesion pathways. AEBP1, ZNF708, and TSHZ2 were identified as key genes: AEBP1/TSHZ2 correlated with poor prognosis, while ZNF708 showed an inverse trend. These genes were linked to chemosensitivity (Irinotecan, Carmustine, Vincristine, and Cisplatin). Recurrent tumors exhibited increased plasma cell infiltration, with key genes regulating IL-17, Notch, and Toll-like receptor pathways. Spatial analysis highlighted oligodendrocyte-astrocyte interactions in the tumor microenvironment.


INTERPRETATION
Fibroblasts drive glioma recurrence, with AEBP1, ZNF708, and TSHZ2 predicting recurrence and chemoresistance. These genes promote immune suppression (via plasma cells) and activate recurrence pathways. Oligodendrocyte-astrocyte interactions shape the recurrent microenvironment, suggesting new therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec7c524a9377c361243faed8c72c15c26a0c97d" target='_blank'>
              Unraveling the Molecular Mechanisms of Glioma Recurrence: A Study Integrating Single-Cell and Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Lei Qiu, Yinjiao Fei, Jiaxuan Ding, Kexin Shi, Jinyan Luo, Yuchen Zhu, Xingjian Sun, Gefei Jiang, Yuandong Cao, Weilin Xu, Shu Zhou
          </td>
          <td>2026-01-06</td>
          <td>Annals of clinical and translational neurology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive primary malignant tumor of the central nervous system (CNS), characterized by marked molecular heterogeneity, rapid proliferation, extensive invasiveness, and strong resistance to conventional therapies. Despite current standard treatments–surgery, radiotherapy, and temozolomide–median patient survival remains below 15 months. Key contributors to GBM progression include activated glial cells and remodeling of the extracellular matrix. In this study, we investigated the cellular and molecular dynamics of the tumor microenvironment, focusing on astrocytic and microglial responses, and on the modulation of glial markers such as GFAP, Iba1, Connexin 43 (Cx43), and the purinergic receptor P2X4R. Acute and organotypic brain slice cultures were prepared from C57BL/6J mice, into which GL261 glioblastoma cells were injected into the cortex. Tissues were collected 3 and 7 days post-injection to analyze tumor progression. GBM growth was monitored through the proliferation marker Ki67. Histological analysis, confocal immunofluorescence, and western blotting were performed to assess marker expression and localization. GL261 cells engaged with the surrounding neuroglial network, migrating in fascicle-like patterns that extended even into the contralateral hemisphere. This dynamic infiltration was associated with pronounced peritumoral astrocyte activation (increased GFAP) and a robust microglial response, evidenced by morphological changes in Iba1-positive cells. Cx43 expression was altered by tumor invasion, suggesting disrupted intercellular communication, while P2X4R expression dynamics indicated modulation of the local immune response. Consistently, elevated Ki67 levels confirmed the highly proliferative and aggressive behavior of the tumor. Organotypic slice cultures represent a fast and reliable ex vivo platform to investigate GBM invasiveness, migration, and neuroglial interactions, revealing spatiotemporal remodeling of the tumor microenvironment during disease progression. Region-specific glial activation and time-dependent modulation of Cx43 and P2X4R highlight the complex and evolving nature of glial–tumor crosstalk. These findings may help identify potential therapeutic windows and support the development of molecularly targeted treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7731564feaeab85e0322e27b0f7ea6db2c5e59" target='_blank'>
              UNRAVELING GLIAL REMODELING IN SPACE AND TIME DURING GLIOBLASTOMA EVOLUTION
              </a>
            </td>
          <td>

          </td>
          <td>2025-12-12</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma is a lethal brain cancer marked by functional plasticity driven by tumor cell-intrinsic mutations and their interplay with developmental programs. To investigate how canonical glioblastoma mutations promote functional plasticity, we have developed an isogenic human neural stem cell (NSC) model of glioblastoma by sequential addition of TERT promoter, TP53, and PDGFRA point mutations. TP53 loss-of-function increases TERT expression during serial mutagenesis, but only triple mutant NSCs reliably form lethal brain tumors in vivo that recapitulate glioblastoma. Tumor cell evolution triggers stress-related metabolic changes and transitions toward a neuronal progenitor network driven by transcription factor INSM1. INSM1 is highly expressed in human glioblastoma tumors and, during cortical development, in intermediate progenitor cells, which give rise to neurons. Remarkably, INSM1 knockdown in triple mutant NSCs and primary glioblastoma cells disrupts oncogenic gene expression and function and inhibits the in vivo tumorigenicity of triple mutant NSCs, highlighting the functional importance of an intermediate progenitor cell-like cell state in glioblastoma pathogenesis. Glioblastoma (GBM) is characterized by a high degree of heterogeneity and plasticity due to interplay with neural developmental programs. Here, the authors develop a model of GBM by introducing sequential oncogenic mutations in human neural stem cells and using this, identify INSM1 as a driver of a neural progenitor gene network promoting tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ee12cd01ff0dc670471af7bea5ea93e2f7413e0" target='_blank'>
              INSM1 governs a neuronal progenitor state that drives glioblastoma in a human stem cell model
              </a>
            </td>
          <td>
            Patrick A. DeSouza, Matthew Ishahak, Xuan Qu, Colin R. McCornack, Devi Annamalai, Diane D. Mao, Rajanikanth Vangipurapu, Yiwei Fu, Alexandre T. Vessoni, Ryan Cleary, Rowland H Han, Punn Augsornworawat, Timothy Woodiwiss, Darby Agovino, Braxton Sizemore, Jessica Kline, Maryam Borhani-Haghighi, Hao Chen, Sangami Pugazenthi, Hiroko Yano, Ting Wang, Luis F Z Batista, J. Millman, Albert H. Kim
          </td>
          <td>2025-12-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="BACKGROUND
Meningioma brain invasion encumbers surgical resection and increases the risk of tumor recurrence, but the molecular mechanisms underlying this process are poorly understood.


METHODS
To identify molecular and cellular features of brain-invasive meningiomas, we (1) analyzed bulk RNA sequencing data from 199 meningiomas, including 33 brain-invasive tumors, (2) analyzed patient-matched single-cell RNA sequencing data of spatially mapped meningioma samples from the tumor core or brain-tumor interface (BTI), and (3) performed spatial transcriptomic sequencing of brain-invasive meningioma samples. Multiplexed immunofluorescence (IF) was used to validate bioinformatic spatial expression patterns. Functional interactions between meningioma cells and neurons were studied in meningioma/neuron co-cultures using confocal microscopy, multi-electrode array recordings, and live cell calcium imaging.


RESULTS
Transcriptomic analyses showed conserved enrichment of TGM2, S100A11, ZYX, and PDGFRA at the BTI across bulk, single-cell, and spatial RNA sequencing datasets. The expression of these genes at the BTI was confirmed using multiplexed IF, and single-cell bioinformatic and microscopy analyses further demonstrated enrichment of macrophages at the BTI. Co-culture assays showed neuronal hyperexcitability and increased proliferation of meningioma cells, suggesting functional communication between meningioma cells and the tumor microenvironment may contribute to meningioma growth in cases with brain invasion.


CONCLUSIONS
Meningioma brain invasion is defined by molecular remodeling of tumor cells and functional interactions within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4352a0cf42108bac27d918f350130e856ef324b" target='_blank'>
              Meningioma cell reprogramming and microenvironment interactions underlie brain invasion.
              </a>
            </td>
          <td>
            Ayush Aggarwal, Mark W. Youngblood, T. Picart, H. Najem, S. Oten, M. A. Cady, Stephen T. Magill, Craig M Horbinkski, James P. Chandler, A. Heimberger, K. Mirchia, S. Hervey-Jumper, David R. Raleigh
          </td>
          <td>2025-12-31</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Metastatic prostate cancer (mPC) is characterized by molecular and phenotypic heterogeneity. With increasing guideline‐driven use of metastatic biopsies, more mPC specimens are being evaluated in surgical pathology. However, unlike localized prostate cancer, no standardized framework currently exists to guide the diagnostic workup of metastatic biopsies or reliably determine phenotypic subtypes. While many mPCs retain conventional acinar features, a growing subset exhibits phenotypic plasticity – including loss of prostate epithelial identity and emergence of neuroendocrine or other divergent lineages. This phenotypic diversity often occurs in castration‐resistant prostate cancer as a mechanism of resistance to chronic androgen receptor pathway inhibition and is characterized by genomic alterations and epigenetic reprogramming. This review outlines the histologic and molecular spectrum of mPC and proposes a practical, pathology‐informed diagnostic approach integrating morphologic assessment and immunohistochemistry. Adoption of a standardized diagnostic framework and multidisciplinary integration will be useful for employing precision oncology in advanced mPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de38a46f132d3e7836729ee1ba6aae01604d174" target='_blank'>
              Molecular subtypes of metastatic prostate cancer: from pathophysiology to diagnosis
              </a>
            </td>
          <td>
            J. M. Dsouza, Erolcan Sayar, M. Schweizer, Stephanie Harmon, C. Morrissey, H. Beltran, Peter S. Nelson, Liang Cheng, C. C. Ding, Michael C. Haffner
          </td>
          <td>2025-12-12</td>
          <td>Histopathology</td>
          <td>1</td>
          <td>58</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven by heterogeneous tumor cell populations and microenvironmental interactions. However, a comprehensive single-cell atlas across patients that captures this heterogeneity, and its clinical implications has been lacking. Such an atlas could reveal rare tumor subpopulations that underpin disease aggressiveness and offer new prognostic biomarkers or therapeutic targets. We integrated two high-quality single-cell RNA sequencing datasets from 29 CRC patients to construct a cellular atlas encompassing immune, stromal and epithelial compartments. Malignant epithelial cells were distinguished via inferCNV-based copy number alteration analysis and reclustered, yielding seven transcriptionally distinct malignant subpopulations. One malignant epithelial cluster, marked by high expression of the long non-coding RNA ELFN1-AS1, exhibited the highest stemness signature. From this stem-like cluster, we derived a four-gene prognostic model (RPL21, GAL, ELFN1-AS1, AIF1L). In the TCGA-COAD cohort, this model stratified patients into high-risk and low-risk groups with significantly different survival outcomes, with the high-risk group experiencing significantly worse survival. High-risk tumors were enriched for metabolic and translational pathways and displayed distinct immune and genomic features. AIF1L was identified as a hub gene within the signature, and its knockdown in CRC cells enhanced migration and invasion, functionally validating its role in tumor progression. Our study provides a high-resolution single-cell atlas of CRC and identifies a previously unrecognized stem-like tumor cell subpopulation with prognostic significance. These findings highlight novel prognostic biomarkers and suggest potential therapeutic targets (such as AIF1L) that could inform patient stratification and the development of targeted therapies for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3d35184693bc48dd0fe85be6bfc55a03d6e4e7" target='_blank'>
              Single-cell RNA sequencing defines prognostic subtypes and identifies AIF1L as a therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Gao Li, Dingxue Wang, Guo Chen
          </td>
          <td>2025-12-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common type of primary liver cancer. Tumor heterogeneity is a major obstacle to effective treatment and is poorly understood using traditional bulk sequencing methods. This review highlights the transformative role of single-cell and multi-omics technologies in determining the cellular and molecular complexities of HCC. We summarize recent advances in single-cell transcriptomics, epigenomics, multi-omics, and spatial transcriptomics platforms, emphasizing their applications in characterizing tumor subclones, cancer-associated fibroblast-immune interactions, circulating tumor cells, and immune-resistant phenotypes. Spatial approaches have revealed the architecture of cancer stem cell niches and tertiary lymphoid structures, providing unprecedented insights into tumor organization and microenvironmental crosstalk. Although still in their early stages, clinical trials have begun to incorporate these technologies, underscoring their translational potential. Single-cell and spatial omics have reshaped HCC research by enabling high-resolution profiling of tumor ecosystems and driving the discovery of biomarkers, therapeutic targets, and strategies for patient stratification. However, high cost, technical expertise, and limited accessibility, particularly in resource-constrained settings, are major barriers to its widespread adoption. Addressing these challenges is critical for translating these powerful approaches into clinical practice and for advancing precision medicine for the treatment of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0abad77c5fcae33c11d7d851ad3eb337d3881a5" target='_blank'>
              Exploring Single-Cell and Multi-Omics Technologies and Their Role in Unravelling Tumor Heterogeneity of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Charmi Jyotishi, Suresh Prajapati, Mansi Patel, Reeshu Gupta
          </td>
          <td>2025-12-17</td>
          <td>Journal of liver cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Aggrephagy plays a pivotal role in ovarian cancer (OC) progression by clearing misfolded proteins. However, the functional role of aggrephagy within the tumor microenvironment (TME) remains elusive. Non-negative matrix factorization (NMF) was employed to analyze 43 aggrephagy-related genes (ARGs) on 65,820 single cells from scRNA-seq data of 12 samples (7 OC tissues and 5 normal tissues). The OC cohort from 9 public databases and TIDE-based immunotherapy scores were used to identify prognosis and immune response patterns of these TME cell clusters. Furthermore, the association between ARGs and CD8+ T cells was validated by the ID8 ovarian cancer mouse subcutaneous tumor model. Fibroblasts, T cells, macrophages and B cells identified from scRNA-seq of OC samples were subjected to biological process enrichment and classified based on ARGs expression patterns. Additionally, it revealed that ARGs may be significantly associated with clinical and biological characteristics of OC and pseudotime trajectories of major TME cell types. Bulk-seq analysis revealed that these aggrephagy-associated TME cell subsets were closely linked to the prognosis of OC patients and exhibited significant heterogeneity in predicting response to immunotherapy. Among them, CAF and CD8+ T cells showed particularly significant immune responsiveness. Notably, CellChat analysis revealed that aggrephagy-associated subtypes of TME cells engage in diverse and extensive interactions with tumor epithelial cells. Meanwhile, significant co-localization of TUBA1B with CD8+ T cells was observed in OC tissues from the mice model. This study is the first to delineate the role of aggrephagy regulators-mediated intercellular communication of the TME in regulating OC growth and antitumor immunomodulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f5dbf13eb9670595afcc13d733cac2504d66e7a" target='_blank'>
              Single-cell aggrephagy landscape delineates intercellular communication of tumor microenvironment associated with ovarian cancer progression and immunotherapy
              </a>
            </td>
          <td>
            Feng Fang, Jiemiao Chen, Xinran Li, Cao Zhou, Yilan Dai, Yuanyuan Chen, Xin Yang, Yong Xie, Fujie Qin, Huijiao Fu
          </td>
          <td>2025-11-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Sinonasal squamous cell carcinoma (SNSCC) is a rare malignancy with poorly understood molecular drivers. Consequently, its cellular composition and tumor microenvironment (TME) remain largely undefined. Here, we performed integrated bulk and single-nucleus multi-omic analyses to map the SNSCC ecosystem. Within the malignant compartment, we identified five distinct populations, with hypoxic (TC1) and proliferative (TC2) subtypes associated with adverse clinical outcomes. Functionally, TC1 cells orchestrate a hypoxia-driven angiogenic program via coordinated secretion of adrenomedullin (ADM), MIF, and VEGFA, promoting endothelial tip cell (EC1) differentiation. Integrative analysis revealed these transcriptional programs are underpinned by tumor-specific chromatin accessibility and DNA hypomethylation, particularly at AP-1-enriched regulatory elements. Mechanistically, in vitro studies confirmed that this response depends on cooperative AP-1 and HIF1A signaling. Furthermore, histological analysis of patient tissues demonstrated spatial co-localization of GLUT1-expressing TC1 cells with DLL4-positive EC1 cells. These findings elucidate the epigenetic landscape underlying tumor-stromal interactions and establish the ADM/VEGFA axis as a critical therapeutic target to disrupt epigenetically controlled angiogenesis in SNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bae1ad73527bcabf41b52f28eb9547020179867e" target='_blank'>
              Single-Nucleus Multi-Omics Reveals Hypoxia-Driven Angiogenic Programs and Their Epigenetic Control in Sinonasal Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, J. Noh, Jaehyun Lee, Geunho Kwon, Myeong S. Yu, , Seung-Jun Lee, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2026-01-07</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most prevalent malignant neoplasms worldwide, characterized by a high incidence of recurrence and metastasis, which substantially diminishes patient survival rates. This underscores the urgent need to identify novel biomarkers and therapeutic targets. Transcribed ultraconserved regions (T-UCRs), a category of non-coding RNAs with significant evolutionary conservation, are crucial to various biological processes. Recent studies have shown that T-UCRs play a pivotal role in tumorigenesis and tumor progression. A growing body of evidence indicates that T-UCRs significantly influence CRC development by modulating critical mechanisms, including cell proliferation, apoptosis, invasion, and metastasis. This review systematically explores the functions of T-UCRs in tumorigenesis, focusing on their regulatory roles, underlying molecular mechanisms, and potential clinical applications in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ca72dace2c26f564f76a90eafc66dbf14b419b5" target='_blank'>
              The transcribed ultraconserved regions: emerging players in colorectal cancer biology and therapy
              </a>
            </td>
          <td>
            Yi Zhang, Zichen Wei, Xin Wang, Pan Wang, Lei Pang, Hongliang Dong, Han Xu
          </td>
          <td>2026-01-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Highlights What are the main findings? Patient-derived organoids (PDOs) faithfully recapitulate the genetic heterogeneity and histological architecture of original tumors, serving as a superior platform for modeling intrinsic and acquired resistance mechanisms, particularly the reversible state of drug-tolerant persisters (DTPs). Despite their utility, current PDO models face significant challenges, including the absence of tumor microenvironment (TME) components, the lack of standardized drug sensitivity cutoff values, and the scarcity of acquired resistance models in lung cancer due to re-biopsy limitations. What is the implication of the main finding? Integrating PDOs with advanced technologies—such as single-cell RNA sequencing, spatial transcriptomics, and immune co-culture systems—enables the precise mapping of resistance evolutionary trajectories and the evaluation of novel therapeutics like antibody-drug conjugates (ADCs). Establishing standardized culture protocols and large-scale biobanks will transition PDOs from experimental tools to robust “pharmacotyping” platforms, facilitating functional precision medicine and improved treatment strategies for relapsed lung cancer patients. Abstract Background: Chemotherapy, targeted therapy and radiotherapy are the cornerstones of cancer treatment. However, therapeutic resistance—not only to these classic modalities but also to novel therapeutics like immune checkpoint inhibitors (ICIs) and antibody-drug conjugates—remains a major hurdle. Resistance significantly limits efficacy and increases recurrence rates. A deep understanding of the molecular mechanisms driving this resistance is critical for developing personalized therapeutic strategies and improving patient outcomes. Recent Advances: Patient-derived cancer organoids have emerged as a powerful preclinical platform that faithfully recapitulates the genetic, phenotypic, and histological characteristics of original tumors. Consequently, PDOs are being widely utilized to evaluate drug responses, investigate resistance mechanisms, and discover novel therapeutic targets for a range of therapies. Limitations: While organoid models have been instrumental in studying resistance, significant limitations persist. First, standard organoid-only models lack key tumor microenvironment components, such as immune cells, limiting immunotherapy research. Second, there is a significant lack of research on acquired resistance, particularly in lung cancer. This gap is largely driven by the clinical infeasibility of rebiopsy in patients with progressive diseases. Third, the absence of standardized protocols for generating and validating resistance models hinders reproducibility and complicates clinical translation. Conclusions: This review summarizes recent advances in using organoid models to study resistance to chemotherapy, radiotherapy, and novel therapeutics (ICIs and ADCs). We emphasize the critical need for standardization in resistance organoid research. We also propose future directions to overcome existing challenges, including the integration of co-culture systems (to include the TME) and advanced technologies (e.g., scRNA-seq, Spatial Transcriptomics). Our specific focus is on advancing lung cancer resistance modeling to enable functional precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02de70fc862ddc88f8c5994f14a3861da00f3847" target='_blank'>
              Modeling Drug and Radiation Resistance with Patient-Derived Organoids: Recent Progress, Unmet Needs, and Future Directions for Lung Cancer
              </a>
            </td>
          <td>
            Dahye Lee, Yoonjoo Kim, D. Kang, Chaeuk Chung
          </td>
          <td>2025-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Introduction Understanding CD8+ tissue-resident memory T cells (TRM) spatial characteristics in hepatocellular carcinoma (HCC) is challenging, and clarifying the spatial feature changes following immunotherapy represents an urgent research gap. Methods This study employs a multi-omics approach to analyze the spatial distribution and intercellular interactions of TRM cells in HCC tissues using radiomics, single-cell sequencing, and multiplex immunofluorescence histochemistry (m-IHC). Results Our results show that the number of CD8+ TRM cells in HCC increases following immunotherapy. Furthermore, after dividing tumor tissues into the tumor core (TC), invasion margin (IM), and normal tissue (N), a increase in CD8+ TRM cells from the IM to the TC can be observed. Consistent with the results of single-cell sequencing analysis, this change in spatial characteristics may be associated with the interactions between CD8+ TRM cells and CD68+ cells. Discussion Immunotherapy can modify the spatial characteristics of CD8+ TRM cells via regulating their crosstalk with other immune cells, and the spatial distribution of CD8+ TRM cells in the HCC tumor microenvironment (TME) correlates with immune checkpoint blockade (ICB) therapeutic efficacy. Clarifying the mechanisms of action of immunotherapeutic drugs and developing a non-invasive radiomics model to predict CD8⁺ TRM cell dynamics will facilitate the clinical management of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/384dbc51e0f9efcd6dc85b035d8ba5b7b014c96f" target='_blank'>
              Multi-omics spatial characteristics of CD8+TRM cells in hepatocellular carcinoma and immunotherapy response prediction
              </a>
            </td>
          <td>
            Shipeng Li, Ju Ma, Xinqiang Li, Bingbing Qiao, Liancai Wang, Deyu Li, Hua Zhou, Haixu Xu, Jinzhen Cai
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The development of novel therapeutics for deadly diseases such as glioblastoma (GBM) is bottlenecked by poor preclinical models. GBM is the most common and deadliest primary brain tumor in adults, with an average prognosis of 12–15 months, primarily due to its high cellular heterogeneity and treatment resistance from GBM stem cells. The advancement of in vitro models into organoids, three-dimensional tissue-like modeling systems, has been a promising approach to improving translational medicine for GBM. However, the critical tradeoff between technical convenience and physiological relevance threatens the integrity and reproducibility of GBM organoid (GBO) biomanufacturing. This comprehensive review breaks down and discusses the key features of GBM tumor microenvironment (TME), traces the advancement of in vitro models from two-dimensional cultures to three-dimensional stem cell-derived GBOs, evaluates the process through an engineering perspective (genetic, biochemical, biophysical, and process engineering), and addresses critical translational gaps. Reviewing trends over the last fifteen years in biomanufacturing approaches to GBOs revealed fundamental oversights that address previous review focuses on the limitations of organoids (i.e., maturity, vasculature, and immune defense). To summarize, GBO’s translational gap and reproducibility challenges are rooted in the prioritization of technical convenience over physiological relevance. To achieve clinical relevance, future GBO development must focus on transitioning to fully defined components (excluding animal-derived ECM), developing sufficiently large-sized constructs to recapitulate the full TME, and integrating non-destructive and enhanced functional readouts of the GBOs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e994f4d6a33fd45263cff1ae3c39866677c30f73" target='_blank'>
              Bioengineering Stem Cell-Derived Glioblastoma Organoids: A Comprehensive Review
              </a>
            </td>
          <td>
            Alexandra D. Avera, Yonghyun Kim
          </td>
          <td>2025-12-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract While cancer immunotherapies have primarily focused on activation of cytotoxic CD8 cells, CD4 T cell activity is also associated with survival and immunotherapeutic response in numerous cancers. We applied integrated single-cell RNA sequencing and multiplexed protein epitope profiling to breast cancer samples to resolve the complexity of immune cell states within the tumor microenvironment. This approach enhanced phenotypic resolution, identifying three distinct states within the CD4 T follicular helper-like (Tfh) cell cluster. A CXCR4high progenitor state gave rise to two differentiated states: an IGFL2high subset resembling conventional Tfh cells and localised to B cell-rich lymphoid aggregates, and a CD103+ subset, exhibiting features of tissue residency, exhaustion, and cytotoxicity, which co-localised with tumor foci. CD103+ Tfh-like cells were found to interact with CXCL10+ macrophages through production of CCL chemokines and CSF1. A higher CD103+ Tfh to IGFL2high Tfh ratio, together with the selective clonal expansion of the CD103+ subset, was strongly associated with improved tumour immunity and superior responses to anti-PD-1 checkpoint blockade, surpassing the predictive value of exhausted CD8 T cells. These findings integrate Tfh and CD4 with cytotoxic potential in breast cancer, offering new insight into anti-tumor immunity and response to checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/570ecff53156bbe18de4ac45af83e6cec5c103da" target='_blank'>
              Proteotranscriptomic Dissection of Breast Cancer T Cell States Identifies CD103+ Tfh-derived Cytotoxic Cells Linked to Immunotherapy Response
              </a>
            </td>
          <td>
            Alexander Swarbrick, G. Al-Eryani, Sophie van der Leij, Etienne Masle-Farquhar, , K. Harvey, Sunny Z. Wu, D. Roden, Taopeng Wang, John Reeves, Bertrand Yeung, Christopher Goodnow, Cindy Ma, Charles Perou, N. Hacohen, Aziz Al’Khafaji, Mats Nilsson, J. Lundeberg, Marcel Batten, S. Junankar
          </td>
          <td>2026-01-07</td>
          <td>Research Square</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="Dogs play a major role in sustaining transmission of Leishmania infantum to people, thus prevention and treatment of canine leishmaniosis (CanL) to reduce transmission represents an unmet public health need. Although advances have been made in understanding how immunopathology correlates to infectiousness as CanL progresses, the immune mechanisms underlying transition between disease stages and terminal decline remain ill-defined. To address this knowledge gap, we generated a comprehensive atlas of peripheral immune cells from control dogs and naturally-infected dogs using single-cell RNA sequencing. The LeishDog Atlas captures the cellular and transcriptional complexity underlying CanL, tracing shifts in immune composition and gene expression across progressive, well-defined disease stages. Notably, we identified distinct myeloid, CD4+ and CD8+ T cell phenotypes and transcriptional states associated with disease progression. This resource provides a valuable framework for understanding systemic immune dysregulation in CanL, including determinants of T cell exhaustion and myeloid cell activation, and establishes a foundation for comparative, translational, and mechanistic studies of CanL immunopathology and transmissibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8412687f28e64e4d6b2558529c494e2ee4440eca" target='_blank'>
              A single-cell transcriptomic atlas of peripheral blood immune cells spanning progressive canine leishmaniosis
              </a>
            </td>
          <td>
            Danielle P. Uhl, Daniel J. Holbrook, Max C Waugh, S. Dey, Najmeeyah Brown, Karen I. Cyndari, Jacob J. Oleson, Paul M. Kaye, Christine A. Petersen
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Bone sarcomas remain lethal despite multimodal therapy, primarily because the mineralized, immunosuppressive tumor microenvironment (TME) promotes chemo‑ and immune‑resistance. Integrating single‑cell and spatial omics across osteosarcoma, Ewing sarcoma and chondrosarcoma delineates subtype‑specific TME archetypes dominated by M2 macrophages, exhausted T cells and a stiff extracellular matrix. Mechanistic dissection reveals tractable vulnerabilities, myeloid reprogramming, extracellular matrix modulation and metabolic and epigenetic checkpoints, that can be targeted with bone‑selective delivery systems and biomarker‑driven combination trials to convert therapeutic failure into durable remission. Therefore, the aim of the present review is to synthesize the latest single‑cell, spatial and functional data to map bone‑sarcoma TME heterogeneity, dissect resistance mechanisms and propose integrated, biomarker‑guided therapeutic strategies that can be translated into treatments..">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a42d68e5855e4884a22e5af558a0428be6a50022" target='_blank'>
              Tumor microenvironment in bone sarcomas: Implications for immunotherapy and emerging therapeutic vulnerabilities (Review).
              </a>
            </td>
          <td>
            Wentao Li, Lijun Lv, Yibin Jin, Xin Yuan
          </td>
          <td>2026-01-16</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epithelial compartments play a central role in the progression and metastasis of colorectal cancer (CRC), yet the mechanisms underlying their transcriptional reprogramming across disease stages remain incompletely understood. We constructed comprehensive single-cell and spatial transcriptomic atlases encompassing normal colorectal tissue, primary CRC, and liver metastases (LMs), profiling over 200,000 high-quality cells. Specific coexpression modules and transcriptional regulators were identified and spatially mapped. Additionally, the clinical and functional roles of SCAND1 were investigated using western blot, RT-qPCR, and immunohistochemistry in clinical samples, as well as functional assays in CRC cell lines. Distinct coexpression modules were selectively enriched in normal epithelial cells and associated with vesicle transport, ion homeostasis, and barrier function, whereas LM-specific modules were linked to Wnt signalling, ribosome biogenesis, and cell cycle processes. High-CNV epithelial cells exhibited progressive activation of key transcription factors such as CEBPB, E2F, and the AP-1 family, with concurrent suppression of ARGLU1 and MTNR2L8, indicating coordinated remodelling of transcriptional and metabolic programs during metastasis. These changes were spatially validated and correlated with alterations in epithelial composition within the tumour core. Consistently, multiplex immunofluorescence (mIHC) confirmed the expression patterns of CEBPB, ARGLU1, S100A4 and FOSL1, across normal epithelium, CRC, and LM. Notably, SCAND1 was identified as a potential therapeutic target for malignant epithelial plasticity. We demonstrated for the first time that SCAND1 expression is significantly upregulated in CRC patients and is associated with superior diagnostic and prognostic value. Functional experiments revealed that SCAND1 promotes tumour cell proliferation, inhibits apoptosis, and enhances metastatic potential. Furthermore, co-culture of SCAND1-overexpressing tumour cells with T cells showed that SCAND1 overexpression attenuates T cell–mediated cytotoxicity, supporting an additional immune-evasive role. Our study elucidates the molecular logic of epithelial state transitions during CRC metastasis and underscores the promise of SCAND1 as a novel biomarker and therapeutic target in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/914a9adf8fcce69d1ec7290317d881a8d12ed794" target='_blank'>
              Dynamic remodelling of epithelial plasticity in colorectal cancer from single-cell and spatially resolved perspectives
              </a>
            </td>
          <td>
            Chengyuan Xu, Yun Pan, Siqi Zhang, Moubin Lin, Hailong Liu, Zicheng Yu
          </td>
          <td>2025-11-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Although single-cell analyses have advanced our understanding of pancreatic ductal adenocarcinoma (PDAC), most studies to date have focused on primary and metastatic tumors. Here, we map cell composition, phenotypic plasticity, and microenvironmental remodeling from human normal pancreas through preneoplastic lesions to PDAC, with the preneoplastic phase recognized as a critical window for carcinogenesis. We pinpoint genes that are persistently dysregulated throughout malignant transformation and are associated with a poor prognosis. Focusing on ductal and acinar cells as the principal origins of PDAC, we delineate malignant preneoplastic cell clusters that exhibit strong carcinogenic potential. Immune profiling reveals marked expansion and functional reprogramming of macrophages during disease progression. Integrative analysis with human PDAC bulk transcriptomic cohorts identifies candidate compounds, such as Brefeldin A, with potential for intervention in preneoplastic disease. Together, our study elucidates dynamic molecular and cellular mechanisms underlying PDAC carcinogenesis and provides actionable insights for early intervention and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34c71ca9f3afc515fdc9293246a8c35d1406ec06" target='_blank'>
              Elucidating Pancreatic Ductal Adenocarcinoma Carcinogenesis at Single-Cell Resolution and Identifying Subtype Specific Drug Candidates
              </a>
            </td>
          <td>
            Jing Chen, Hui Jiang, Hui Chen, Kuan Yang, Kaiyue Yang, Mingyao Sun, Na Lv, Bolin Ren, Xinyi Lin, Xia Li, Yunpeng Zhang, Congxue Hu
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Intratumoral heterogeneity supports cancer progression and metastasis while limiting therapeutic efficacy. Comprehensive biological characterization of heterogeneous subclones in breast cancer may hold promise in battling against this deadly disease. Using single-cell transcriptomic analysis and in situ profiling of primary tumors, metastases, and circulating tumor cells from multiple patients with breast cancer, we identified a unique EPCAM+ CD68+ TREM2+ tumor subpopulation, likely resulting from the fusion of tumor cells and lipid-associated macrophages (LAM). The presence of these tumor-LAM fusion cells in the blood or in distinct metastatic sites was significantly correlated with metastatic progression. Stable fusion clonal lines established in vitro exhibited substantially enhanced proliferation, tumor initiation, and metastasis formation in mice. Integrative molecular and functional analyses revealed a critical role for SNX10 in mediating tumor-LAM fusion. Mechanistically, SNX10 physically interacted with the phospholipid scramblase ANO6 and maintained its protein stability by suppressing proteasome-mediated degradation. Furthermore, the transcription factor CBX3 directly targeted SNX10 promoter and modulated its expression. Fusion cells accumulated abundant intracellular lipid droplets and were highly sensitive to simvastatin treatment in vitro and in vivo. Together, this study uncovered that CBX3-SNX10-ANO6 signaling facilitates generation of an aggressive tumor-LAM fusion cell subpopulation that promotes metastasis, revealing an alternative metastatic mechanism and exposing putative therapeutic vulnerabilities.


SIGNIFICANCE
Single-cell transcriptomic profiling combined with functional and clinical validation identifies fusion of tumor cells and lipid-associated macrophages mediated by the CBX3-SNX10-ANO6 axis as a potentially targetable mechanism driving cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4271e1aeed14ff3c28d665da96c160262a128c0a" target='_blank'>
              Fusion of Tumor Cells with Lipid-Associated Macrophages Drives Metastatic Progression of Breast Cancer.
              </a>
            </td>
          <td>
            Yixin Cheng, Guanyin Huang, Xuefei Liu, Chun Wu, Jingru Lian, Jing Cao, Lianhui Duan, Boxi Zhao, Yufan Yang, Siqi Chen, Guanqiao Li, Liping Wu, Shuqian Zheng, Wenjing Wang, Tianyu Zha, Yang Yang, Yilin Lu, Jian Zhang, Hao Yu, Jun Tan, Feiqiu Wen, Zhenyu He, Xin Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Secreted phosphoprotein 1 (SPP1+) macrophages are a recurrent and functionally critical immune cell subset across multiple cancer types. They drive adverse clinical outcomes by promoting immunosuppression, tumor invasion, metastasis, and therapy resistance. Given their prevalence and pivotal role, SPP1+ macrophages have become a major focus in cancer immunology and a promising target for therapeutic development. SPP1+ macrophages have been identified in a wide range of human malignancies through single-cell RNA sequencing and spatial transcriptomics studies. Their differentiation and maintenance are strongly influenced by reciprocal cellular interactions and hypoxic conditions within the tumor microenvironment (TME). Within the tumor microenvironment (TME), SPP1+ macrophages promote tumor progression by interacting with cancer-associated fibroblasts (CAFs) and helping to form a physical barrier that restricts immune cell infiltration into the tumor core. Specifically, they impair the recruitment of CD8+ T cells and promote T cell exhaustion (TEX). In this review, we focus on recent advances in understanding the differentiation of SPP1 macrophages in hypoxic tumor microenvironment and the role of SPP1+ macrophages in immunosuppression and their therapeutic implications in cancer. Targeting this subset of macrophages has emerged as a highly promising therapeutic strategy, with several approaches demonstrating encouraging results in preclinical models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7c7489149a391c5b3e442dad42f10334718f488" target='_blank'>
              SPP1+ macrophages in tumor immunosuppression: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Juanjuan Wang, Ya Wang, Yuqing Liu, Rongcun Yang
          </td>
          <td>2025-12-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background and objective Cancer-associated fibroblasts (CAFs) promote hepatocellular carcinoma (HCC) progression. Identifying their cellular origin and molecular determinants may help to inform molecular targeted therapy for HCC. Identifying the tissue distribution of CAFs in HCC to indirectly reflect their cellular origin may provide a histological basis for further exploration of their cellular origin and molecular determinants. Methods and results The tissue distribution of CAFs in 57 HCC tissues was determined by immunohistochemical staining of α-SMA, and α-SMA-positive CAFs were found to be widely located along the capillary network. Immunofluorescent double staining was used to examine the co-expression of CD31 in α-SMA-positive CAFs to identify whether the endothelial-to-mesenchymal transition (EndoMT) is involved in the origin, and obvious colocalization was observed. Visium and Visium HD spatial transcriptomics further revealed endothelial cells (ECs) exhibited remarkable co-expression of CAF-specific marker genes and revealed inferred developmental trajectories to CAFs; molecular determinants, including TIMP1, IGFBP7, THBS2, CD74, COL4A1, COL4A2, AEBP1, S100A6, KCTD12, CALD1, IGHG1, SERPINE1, MCL1, MGP, GSTP1, TAGLN, THBS1, and CTGF, were positively correlated with the spatial developmental trajectories of ECs to CAFs; and CTGF exhibited extensive interactions with other common positively correlated molecular determinants and was a highly connected node in the interaction network. Conclusion ECs that undergo EndoMT may be one of the potential cellular and mechanical origins of CAFs in HCC, and the development of EndoMT may be associated with CTGF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46aefa9029373673496a5e998482842f777dedef" target='_blank'>
              Identification of cancer-associated fibroblasts and analysis of the association of origin with endothelial-to-mesenchymal transition in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zeli Yin, Qiuxiang Wang, Zhe Xu, Kun Guo, Liming Wang
          </td>
          <td>2025-11-30</td>
          <td>Scandinavian Journal of Gastroenterology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c60ea4aa70ca5fc55ee1757c090687b18cbeab5" target='_blank'>
              Construction of the cancer cell continuum reveals hybrid EMT state driving lung adenocarcinoma aggression.
              </a>
            </td>
          <td>
            Qiuhao Qu, Yixin Ma, Chao Huang, Botao Fa, Zhenni Liu, Yuhan Li, Shigao Yang, Zhengtao Xiao
          </td>
          <td>2025-11-25</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Pancreatic neuroendocrine tumors (panNETs) undergo high-grade (HG) transformation often, with the liver being a common site for metastasis​. Surgical resection being considered as long-term outcomes for selected patients. However, recurrence and grade transformation remain potential significant challenges, with recurrence rates approaching 50–95%.1,2 Despite extensive whole exome analysis and construction of phylogenetic trees, we were unable to identify either a consistent genetic driver or any specific patterns of tumor transformation.2 We are also concerned that some of our treatments may inadvertently contribute to grade transformation. To assess the treatment effects, we performed pre- and post- trans arterial embolization bulk-transcriptome analysis, which revealed cell population shifts in panNETs that may be linked to grade transformation.4To explore cellular shifts within the liver tumor microenvironment during panNET progression, we performed single-cell RNA sequencing. The resulting data was processed using Seurat.v5. The clusters of panNET cells, fibroblasts, hepatocytes, and immune cells were annotated by known markers. Additionally, we distinguished malignant cells from a reference of immune cells, fibroblasts, and hepatocytes by InferCNV. To understand the tumor progression, we performed trajectory analysis using monocle3. Differential metabolic activity between the panNETs grades were assessed using the scMetabolism package. Cell-cell communication and related ligand-receptor pairs were inferred using CellChatDB. Our results revealed distinct transcriptional trajectories, with pseudotime analysis demonstrating a trajectory of tumor evolution from low-grade (LG) to HG states. HG tumors were enriched for proliferative and aggressive subclusters compared to LG cells. Metabolic profiling further highlighted grade-specific adaptations, with HG tumors showing enhanced oxidative phosphorylation and TCA cycle across tumor cells, whereas LG tumors displayed relatively lower metabolic activity. Notably, immune cells were found metabolically active in LG via glycolysis. Conversely, HG panNETs exhibit glucose depletion in immune cells and less active microenvironment. The CellChat analysis revealed distinct cellular communication patterns in panNET grade. For instance, in HG tumors, signaling was primarily mediated between fibroblasts and panNET cells, with Glu-(SLC1A2+GLS)-GRM7 as the dominant ligand–receptor pair. In LG tumors, major interactions occurred among immune cells, hepatocytes, and panNET cells, with Glu-(SLC1A3+GLS)-GRIK2 representing the highest contributor. The results suggest that metabolic rewiring, altered intercellular signaling, and transcriptional plasticity underlie progression from LG to HG disease.
 Reference:
 1. Spolverato etal J Gastrointest Surg. 2017, 21(12):2039-2047
 2. Mayo etal Ann Surg Onc. 2011, 18(13):3657-65.
 3. Singh et al. Early ... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium. 2023.
 4. Singh et al. Post... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium.2023



 Himanshu N. Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jungeui Hong, Nitya Raj, Olca Basturk, Zeynep Tarcan, Christine A. Iacobuzio, Etay Ziv. Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f19a2d46016dbf4a94424ac84c3f39682a28b2d" target='_blank'>
              Abstract B025: Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors
              </a>
            </td>
          <td>
            Himanshu Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jung-Pyo Hong, Nitya Raj, O. Basturk, Z. Tarcan, Christine A. Iacobuzio, E. Ziv
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Tumor budding is positively associated with colorectal cancer (CRC) metastasis. In this study, we integrated a single-cell transcriptomic dataset of 287 CRC samples to comprehensively illustrate the transcriptomic landscape of metastatic CRC and identified a unique subcluster of tumor epithelial cells associated with tumor budding. This subcluster exhibited high mesothelin (MSLN) expression and was located at the invasive front of CRC. MSLN was confirmed to promote CRC growth and metastasis by in vitro and in vivo models. Also, POSTN+ fibroblasts in the CRC microenvironment showed enhanced expression of genes in epithelial-mesenchymal transition and angiogenesis signaling pathways, which wrapped around MSLN+ tumor budding cells in the invasive front of CRC. POSTN+ fibroblasts may interact with MSLN+ budding-potential cells through the ligand-receptor pair POSTN-ITGB5 to promote tumor metastasis. In conclusion, our findings identified the transcriptomic feature of budding-potential cells and revealed the role of crosstalk between MSLN+ budding-potential cells and POSTN+ fibroblasts in CRC metastasis, which provide new insights into targeting cancer-associated fibroblasts and tumor budding cells in CRC therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a0a32fa1e03245ef6c4c1569e60e598b59ede0" target='_blank'>
              Comprehensive single-cell transcriptome landscape of metastatic colorectal cancer identifies budding-potential cells and their interactions with cancer-associated fibroblasts
              </a>
            </td>
          <td>
            Yao Ma, Lijian Wang, Ziyue Zhang, Yifei Zhao, Zimin Zhao, Renjie Luo, Junwei Wang, Limei Guo, Wei Fu, Kai Miao, Xin Zhou
          </td>
          <td>2025-11-26</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer is genetically and histologically heterogenous, and is influenced by a variety of factors, including the tumor microenvironment (TME). The PAM50 subtypes; Luminal A, Luminal B, Normal-like, Basal-like and Her2-enriched, are associated with different tumor phenotypes and overall survival. The quantity and quality of immune cell infiltration in breast tumors play a key role in cancer development and progression and are associated with survival and treatment response. We used multiplex immunohistochemistry, single-cell RNA sequencing, and two deconvolution algorithms to explore the immune landscape across PAM50 subtypes. Immunostaining of CD3+ T cells, tryptase+ mast cells, CD20+ B cells, CD68+ CD163+ macrophages, and CD66b+ granulocytes revealed marked differences in tumor immune infiltrates according to breast cancer subtypes. Luminal tumors were relatively deprived of T cells and B cells, while exhibiting sparse to moderate amounts of macrophage infiltration. In contrast, Her2-enriched tumors exhibited a moderate immune presence, with sparse to moderate T cell infiltration and moderate infiltration of B cells and macrophages. At the other end of the spectrum, Basal-like tumors stood out for their strikingly rich immune environment and are heavily infiltrated by T cells, B cells, and macrophages. The results from the single-cell and deconvolution analyses confirmed subtype-specific immune microenvironments, which also allowed us to observe increased levels of natural killer (NK) and CD8+ T cells in Her2-enriched and Basal-like subtypes. In conclusion, our findings demonstrate significant differences in the immune tumor microenvironment between the established breast cancer molecular subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c99c23b4dfb7da23c9b0f3a691c05aec9a1b68" target='_blank'>
              Multi-platform analysis of the tumor immune microenvironment associated with breast cancer subtypes.
              </a>
            </td>
          <td>
            Lilly Anne Torland, Jürgen Geisler, Y. Azimzade, Astri Frafjord, Hogne Røed Nilsen, Linn Buer, J. Lømo, Ø. Garred, Andliena Tahiri, M. Lyngra, Inger Øynebråten, Alexandre Corthay, Kristine Sahlberg, Vessela N. Kristensen, Xavier Tekpli
          </td>
          <td>2026-01-13</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 225


 Background:
 Colorectal cancer (CRC) is the second leading cause of cancer-related deaths in the United States. Although immune checkpoint blockade benefits ~15% of patients with high mutational burden and microsatellite instability, the majority of CRC remains refractory, with cytotoxic chemotherapy as the standard of care. Identifying mechanisms that sustain cancer cell survival and proliferation is therefore critical to uncovering vulnerabilities that may enhance therapeutic efficacy. The complement peptide C5a, a potent effector generated by proteolytic activation of the complement cascade, signals through its receptor C5AR1 to drive cell survival, cytokine production, and chemotaxis. While historically regarded as a systemic pathway, recent evidence suggests that complement activation can also occur locally. We hypothesized that CRC tumors locally generate C5a, with cancer cell-intrinsic C5AR1 driving survival and chemoresistance.
 Methods:
 CRC tissue microarrays (TMAs) were generated from de-identified specimens (Hackensack Meridian
 Health
 ,
 N
 = 28; commercial TMA,
 N
 = 17). Preliminary spatial transcriptomics was performed on the commercial TMA using the Nanostring CosMx Spatial Molecular Imager. Publicly available single-cell transcriptomes from two independent CRC cohorts (
 N
 = 63 and
 N
 = 62) were reanalyzed. Multiplexed immunofluorescence (24-plex Cyclic IF) was applied to detect complement fragments (C3, C3b/iC3b, C5a) and C5AR1, and to map their cell type-specific expression in situ. Cancer cell-intrinsic C5AR1 expression was validated in two human CRC cell lines and four patient-derived organoids. Functional relevance of C5AR1 was assessed using in vitro cytotoxicity assays combining chemotherapy with pharmacologic C5AR1 inhibition.
 Results:
 Spatial transcriptomics revealed induction of complement pathway transcripts in CRC tumors, further confirmed by reanalysis of two independent CRC single-cell datasets. Multiplexed CycIF revealed an enriched localization of complement ligands (C3, C3b/iC3b, C5a) to tumor-adjacent regions, whereas C5AR1 was enriched in the tumor core. Furthermore, CycIF demonstrated robust C5AR1 protein expression in cytokeratin-positive cancer cells and infiltrating myeloid phagocytes; cancer cell-specific C5AR1 expression was further validated in two human CRC cell-lines and patient-derived organoids. Functionally, cancer cell-intrinsic C5AR1 proved essential for survival, and its pharmacologic inhibition synergistically enhanced 5-fluorouracil cytotoxicity.
 Conclusions:
 Complement activation is spatially organized in CRC, with cancer cell-intrinsic C5AR1 supporting survival. Pharmacologic C5AR1 inhibition synergizes with chemotherapy, positioning the C5a-C5AR1 axis as a tractable therapeutic vulnerability. Ongoing studies aim to define how C5AR1 supports cancer cell survival and mediates chemoresistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a950393c39eca04ec13c0ab1c5dcbbbb94f802" target='_blank'>
              Role of cell-intrinsic C5AR1 in colorectal cancer survival and chemoresistance.
              </a>
            </td>
          <td>
            Ashley Carlo, Raju Shivarathri, Adela Karuli, Richa Mandrekar, Anastasia Zenkevich, Chen-Hua Ma, Kar Chow, Alvin Makohon-Moore, Kevin Tong, Jigar V. Desai
          </td>
          <td>2026-01-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background Colorectal cancer (CRC) continues to represent a major cause of cancer-related mortality worldwide, with its progression and therapeutic outcomes strongly shaped by the complexity and heterogeneity of the tumor immune microenvironment (TME). This review critically examines the cellular and molecular mechanisms driving immune evasion in CRC, emphasizing the dual roles of immune cell populations—including tumor-associated macrophages, neutrophils, dendritic cells, T cells, B cells, and natural killer cells—as well as non-cellular elements such as the extracellular matrix and extracellular vesicles. Objective A key objective is to evaluate recent developments in immunotherapeutic approaches, including immune checkpoint inhibitors, tumor vaccines, adoptive cell transfer, and novel combinatorial regimens, while addressing their therapeutic promise and inherent limitations, especially in microsatellite-stable (MSS) tumors that exhibit primary resistance to standard immunotherapies. Further analysis integrates perspectives on metabolic reprogramming within the TME, epigenetic alterations, and advances in engineered cellular therapies, thereby providing a comprehensive framework for overcoming immunosuppressive mechanisms. Discussion and conclusion Special consideration is directed toward the translational value of targeting immune-metabolic interactions and spatial dynamics within the TME. Ultimately, this work synthesizes current knowledge and outlines forward-looking strategies to advance personalized, multi-target immunotherapy, with the potential to reshape clinical paradigms in CRC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03fb3a1b140db8c8ae9afa2ff97c1f40f6b8cd5d" target='_blank'>
              Cellular characteristics of the immune microenvironment of colorectal cancer and progress in immunotherapy research
              </a>
            </td>
          <td>
            Chunbaixue Yang, Jianchun Fan, Yixuan Zhang, Yixiao Zhang, Lei Xia, Xinran Cao, Xueliang Wu
          </td>
          <td>2025-11-30</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa68ef0a898a9de3aed5f44e700f118ea58e14a" target='_blank'>
              A cellular epigenetic classification system for glioblastoma.
              </a>
            </td>
          <td>
            Dana Silverbush, Liv Jürgensen, Nelson F. Freeburg, Channing Pooley, Fabio Boniolo, Federico Gaiti, M. Suvà, V. Hovestadt
          </td>
          <td>2026-01-07</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a988a1774cf7ed3f81f590a732823cd8a46b829" target='_blank'>
              Wnt ligands in breast cancer metastasis: unraveling their dual roles and therapeutic potential
              </a>
            </td>
          <td>
            Azita Asadi, Mehran Molavand, Mostafa Vaghari -Tabari, Bahman Yousefi
          </td>
          <td>2026-01-12</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The aim of this study was to characterize the transcriptional landscape of tuberous sclerosis complex-associated renal angiomyolipomas (TSC-AMLs) using spatial transcriptomics and to compare it with sporadic AMLs and renal cell carcinoma (RCC) to identify TSC-specific molecular features. Spatial gene expression analysis (CytAssist Visium) was performed on formalin-fixed, paraffin-embedded sections from one case each of TSC-AML, sporadic AML, and RCC. Unsupervised clustering revealed that TSC-AML and sporadic AML, unlike RCC, displayed triphasic tumor structures with vascular, smooth muscle, and adipose components expressing canonical perivascular epithelioid cell markers such as PMEL, MLANA, ACTB, and DES. TSC1 and TSC2 were downregulated, and mild upregulation of MTOR, MLST8, and RPTOR was observed, consistent with mTORC1 hyperactivation, whereas RCC showed broader pathway activation. Spatial pathway analysis indicated shared differentiation programs in smooth muscle regions but unique angiogenesis, extracellular matrix remodeling (Wnt and NABA matrisome), and lipid metabolic (PPAR signaling) enrichment in TSC-AML. Comparative profiling identified 42 genes uniquely upregulated in TSC-AML, including GRIA2, FABP4, and IL33, defining candidate TSC-AML–enriched features. Despite the single-case design, this pilot study provides hypothesis-generating insights into TSC-associated renal tumorigenesis and highlights potential biomarkers for future investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca795c8f856b060495a340d41a11434aceebd201" target='_blank'>
              Spatial gene expression profiling reveals the distinct microenvironment of tuberous sclerosis complex-associated angiomyolipoma
              </a>
            </td>
          <td>
            R. Watanabe, Tetsuya Fukumoto, N. Miura, M. Kurata, R. Kitazawa, T. Kikugawa, Takashi Saika
          </td>
          <td>2025-11-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) have emerged as critical prognostic and immunotherapeutic indicators in cancer, with their clinical significance modulated by spatial distribution patterns and density. Here, we performed integrated single-cell and spatial transcriptomic profiling of 30 gastric cancer (GC) specimens stratified by TLS spatial localization patterns. Comparative analysis shows pronounced enrichment of CXCL13+ T lymphocytes (TLCs), CXCR5+ germinal center B lymphocytes (gc_B cells), LAMP3+CD80+ activated dendritic cells (DCs), and SELP+ACKR1+ high endothelial venule (HEV) cells within intratumoral-TLS (iTLS) rich tumors compared to peritumoral-TLS (pTLS) and desert-TLS (dTLS) tumor subtypes. Multimodal cell–cell interaction analysis and functional experiments demonstrate that HEV expressed VCAM1 and ICAM1 recruits and activates CXCL13+ TLC through the CXCL13-ACKR1 pathway, which promotes TLS formation via CXCL13-CXCR5-dependent crosstalk with B lymphocytes. We further develop a single-cell/spatial TLS signature that captures the cellular ecosystem of iTLS-containing tumor, demonstrating predictive value for immunotherapy outcomes in GC patients. Tertiary lymphoid structures (TLS) within the tumour microenvironment are important for prognostic and immunotherapeutic indications in cancer. Here the authors use a spatial and single cell transcriptomics analysis of gastric cancer (GC) samples and show transcriptomic differences grouped by types of TLS and generate a prognostic transcriptomic signature to help predict outcome in GC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f70b23aaaa2341fd42d68d4bef2be6eb1c78ea" target='_blank'>
              Single-cell and spatial transcriptomics implicate a prognostic function of tertiary lymphoid structures in gastric cancer
              </a>
            </td>
          <td>
            Yanchun Wang, Guangyu Zhang, Xin Zhang, Gaoke Liu, Lihua Zhang, Lei Chen, Shuli Sang, Sheng Yao, Yang Fei, Zhichao Tian, Yu Zang, Hongwei Wang, Yiping Dong, Chunjie Liu, Xizhao Chen, Yan Liu, Min Zhang
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4d4eea61e2f155251ae235dac3fc5c5c43d18" target='_blank'>
              A pan-cancer single cell landscape reveals heterogeneity and functional diversity of double-negative T cells.
              </a>
            </td>
          <td>
            Qing Hao, Tao Zhou, Huayun Yan, Zhixiang Ren, Wei Mao, Li-Bin Huang, Yue-Yun Chen, Yuelan Wang, Minqin Xiao, Yajiao He, Xuyang Xia, Jing Wu, Wei-Han Zhang, Hai-Ning Chen, Maoxiang Qian, Feng Zhang, Li Yang, Lunzhi Dai, Huiyuan Zhang, Canhua Huang, Yang Shu, Yu Xu, Yang Luo, Yong Peng, Heng Xu
          </td>
          <td>2026-01-03</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 The introduction of bowel cancer screening has led to an increase in the detection and removal of pre-malignant adenomas, however many of these would never progress to colorectal cancer (CRC) within a patient’s lifetime. This implies a key evolutionary bottleneck at the transition from adenoma to carcinoma, with immune surveillance likely playing a central role in suppressing the outgrowth of invasive cells. Here we analysed a cohort of “ca-in-ads” (adenomas with a small focus of cancer) to provide a unique snapshot of the invasive transition. We combined genomics, spatial transcriptomics, digital pathology and multiplex imaging, concurrently deriving the distribution of tumour clones, their gene expression and their interplay with the immune microenvironment. We performed multi-region low-coverage whole genome sequencing (lcWGS) and T-cell receptor sequencing (using FUME-TCRseq, a novel method developed in our lab) on archival ca-in-ads from 40 patients. For a subset of cases we performed spatial transcriptomics using the 10x Genomics Visium platform, including custom probes to detect T-cell clonotypes of interest. We applied a deep learning cell classifier to haematoxylin and eosin (H&E) stained sections to determine the abundance and spatial distribution of eight cell types, with detailed immunophenotyping on selected cases (using a 50-marker panel on Akoya’s PhenoCycler-Fusion platform). We found a significant increase in copy number alterations in carcinoma regions relative to adenomas, and this was accompanied by a large shift in the composition of the local T-cell repertoire. TCR sequencing revealed expanded T-cell clonotypes that were adenoma-specific or carcinoma-specific, and we validated the spatial exclusivity of these using a custom Visium assay. Although overall lymphocyte density remained unchanged, digital pathology and cyclic immunofluorescence revealed significantly increased infiltration of macrophage and neutrophil populations in regions of invasive cancer. Taken together our analysis suggests a striking reshaping of tumour-immune dynamics at the transition to an invasive phenotype in CRC. Understanding what governs this invasive shift could highlight new avenues for cancer prevention or identify individuals at risk of CRC.



 Ann-Marie Baker, Eleanor Smith, Chirine Sakr, Konstantin Bräutigam, Nick Trahearn, Holly Eggington, Miangela M. Lacle, Manuel Rodriguez-Justo, Simon J. Leedham, Trevor A. Graham. Multi-omic spatial analysis reveals reshaping of tumour-immune dynamics at the transition to invasive colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr PR004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b4fa90cafae68daf434680b6837e68209ae59d6" target='_blank'>
              Abstract PR004: Multi-omic spatial analysis reveals reshaping of tumour-immune dynamics at the transition to invasive colorectal cancer
              </a>
            </td>
          <td>
            Ann-Marie Baker, Eleanor Smith, Chirine Sakr, Konstantin Bräutigam, N. Trahearn, Holly Eggington, M. Laclé, M. Rodriguez-Justo, Simon J. Leedham, Trevor A. Graham
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="NSCLC accounts for approximately 85% of lung cancers, and Kirsten rat sarcoma (KRAS) is the most commonly mutated gene in NSCLC. Although management of NSCLC has advanced markedly in recent years, its heterogeneity and plasticity remain incompletely understood. Single-cell RNA sequencing (scRNA-seq) has become an essential tool for dissecting cellular heterogeneity in complex tissues such as the lung. However, generating high-quality single-cell suspensions from tumor-bearing lung tissue is challenging, particularly during tumor formation when the tissue becomes fibrotic and heterogeneous. A key requirement for successful scRNA-seq is the preparation of viable single cells with minimal processing-induced stress or damage. Here, we present a detailed step-by-step protocol for isolating high-quality single-cell suspensions from both healthy and tumor-bearing mouse lung tissue in a KrasG12D; tdTomato reporter mouse model. The procedure includes tissue perfusion, controlled enzymatic digestion, gentle mechanical dissociation, red blood cell lysis, filtration, and viability assessment, and achieves >85% viable cells prior to scRNA-seq. Finally, the tdTomato+ tumor cells are efficiently isolated by FACS and can be detected as distinct clusters by scRNA-seq using this protocol. SUMMARY We present a detailed protocol for isolating high-quality single-cell suspensions from both healthy and tumor-bearing murine lung tissue. This protocol can be applied to scRNA-seq, flow cytometry analysis, and primary cell isolation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05709be045bd3ac955b0ab18505a25294de0610f" target='_blank'>
              Optimized protocol for processing murine tumor-bearing lung tissue for flow cytometry and single cell RNA-sequencing
              </a>
            </td>
          <td>
            Sicong Wang, Anastasiia Ivanova, Helen P. Makarenkova, Katja A. Lamia
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a key factor influencing carcinogenesis, disease progression, and therapeutic outcomes. Tumor-associated endothelial cells (TECs), integral components of the TME, form the structural and functional foundation of the vasculature, while also modulating tumor immune responses. TECs regulate immune cell activity and distribution through multiple molecular mechanisms, thereby influencing tumor immunity. Recent evidence highlights TEC-targeted interventions as a promising strategy to enhance immunotherapy efficacy. This review examines the mechanisms through which TECs drive tumorigenesis, progression, and metastasis, and explores the therapeutic potential of TEC-targeted approaches in cancer treatment. It also provides a systematic analysis of TECs’ impact on the immune microenvironment, assessing the potential of TEC-directed therapies to overcome chemoresistance and immunotherapy resistance, thus offering new perspectives for drug development targeting TECs. Finally, the review critically evaluates potential molecular targets within TECs and recent advancements in genetic research for cancer therapy, emphasizing key research priorities and pressing clinical challenges that require urgent focus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5e2322afb5c6ffa7a0363f83b269479ee8db97" target='_blank'>
              Tumor-associated endothelial cells in tumor immune escape and immunotherapy: multifaceted roles and treatment approaches
              </a>
            </td>
          <td>
            Tianyu Zheng, Xinran Yu, Caihong Yu, Wangting Xu, Zhuoyang Fan, Yongjie Zhou, Changyu Li, Juncheng Wan, Chaoqiao Jin, Xuran Jin, Wen Zhang, Zhiping Yan, Peng Luo, Bufu Tang, Xudong Qu
          </td>
          <td>2025-12-24</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Background The most aggressive type of skin cancer, melanoma, has a high prevalence of metastases and a poor prognosis. Despite advancements in treatment, drug resistance and tumor microenvironment heterogeneity, especially involving neuro-immune interactions, continue to exist. The goal of this study is to uncover the cellular heterogeneity of melanoma in order to pinpoint molecular targets and tumor-promoting subtypes. Methods Melanoma single-cell RNA sequencing data came from GEO. Twelve cell types were discovered after Harmony batch effect adjustment and Seurat quality control; melanoma cells were subtyped. Functional expression of differential genes was examined using gene ontology and gene set enrichment. Cytotrace measured subtype differentiation potency. PySCENIC revealed transcription factor regulatory networks, and CellChat predicted intercellular communication between malignant cell subtypes and other cell types. Functional experiments with A375 and MEWo cell lines—lentiviral shRNA knockdown, CCK-8 proliferation, wound healing, transwell migration, and flow cytometry apoptotic assays—confirmed the analysis. Results Single-cell RNA sequencing was utilized to analyze melanoma cell subtypes and their interactions in the tumor microenvironment. C2 IGFBP3+, which had the lowest CytoTRACE2 score and was enriched in late tumor stages, affected melanoma development. This subtype expressed prominent immunomodulatory pathways. The C2 subtype mediate signaling to fibroblasts/pericytes via the PROS pathway and myeloid/plasmacytoid dendritic cells via the MHC-II pathway. The C2 subtype was strongly linked with FOSL1 expression, which may affect gene transcription and illness progression. FOSL1 knockdown significantly increased apoptosis and decreased melanoma cell motility and proliferation in vitro. Conclusion We identified immunoregulatory C2 IGFBP3+ melanoma cell subtypes in our investigation, and FOSL1 was a crucial transcription factor that aided in cell migration, proliferation, and survival. Because the C2 subtype involves the MHC-II and PROS signaling pathways, which have been shown to have roles in neuroimmunology, neuroinflammation, and pain regulation, it may serve as a hub for neuro-immune interactions in the tumor microenvironment. Precision treatments for melanoma may be advanced by focusing on the FOSL1 or C2 subtype pathways, which may assist in overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34e116063cb6903b41d6e715ba8448ec0e1c090" target='_blank'>
              Single-cell transcriptomics identifies FOSL1-regulated IGFBP3+ melanoma subtype as a neuro-immunoregulatory signaling hub facilitating tumor progression
              </a>
            </td>
          <td>
            Wenjia Ge, Ziwei Zhang, Wenjie Chen, Zhijie Zhao, Hua-Bao Cai, Yantao Ding, Jin Xu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND & AIMS
CRISPR-Cas9 screening is a powerful tool for the in vivo discovery of cancer dependencies. The aim of this study was to perform in vivo CRISPR knockout screening for gastric tumor suppressors using gastric murine organoids in a subcutaneous as well as a surgical model of orthotopic tumor growth.


METHODS
In vivo screening was performed using a custom library targeting 49 putative gastric tumor suppressor genes, as well as a "cancer genome-wide" library targeting 5000 genes, in immunocompetent and -deficient mice, and in the presence or absence of the gastric pathogen Helicobacter pylori. The top hits were selected for individual validation and mechanistic follow-up.


RESULTS
Our custom library knockout screens revealed single-guide RNAs targeting Pten, Fbxw7, and genes encoding several components of the transforming growth factor-ß signaling pathway (Smad4, Tgfbr1, Tgfbr2, and Acvr2a) to be recurrently enriched both in subcutaneously and orthotopically growing tumors. The same, and several additional genes were identified by cancer genome-wide CRISPR screening. Ten of our top hits could be validated individually in vivo. Pten inactivation resulted in large tumors characterized by increased neo-angiogenesis, neutrophil recruitment, and T-cell exclusion. Inactivation of Smad4, Tgfbr1, or Acvr2a all produced phenotypes that were reminiscent of early gastric cancer precursor lesions such as intestinal Alcian blue-positive metaplasia and compensatory hyperplasia. Helicobacter pylori infection failed to affect the mutational landscape of tumors; rather, we found that H pylori modulates the tumor microenvironment and recruits large numbers of tumor-promoting SiglecF+ neutrophils.


CONCLUSIONS
In summary, we describe here a versatile model of gastric carcinogenesis that uncouples the genetics of the tumor and the host, and that faithfully recapitulates key risk factors of the malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a500e77069da9fdfddc1529d0df87aec7f46164" target='_blank'>
              Gastric Organoid-Based Ectopic and Orthotopic In Vivo CRISPR Screening for Tumor Suppressors in Gastric Cancer.
              </a>
            </td>
          <td>
            Jiazhuo He, Giovanni Papa, Flora Azizi, Lydia Kirsche, Mariela Artola-Borán, Rafaela Ferreira Cassio, A. Hotz, Gavin Geiger, Biel Francas, Achim Weber, A. Tzankov, Zacharias Kontarakis, Peter Leary, Anne Müller
          </td>
          <td>2025-11-01</td>
          <td>Gastroenterology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Purpose
Colorectal cancer (CRC) lung metastases exhibit high recurrence rates after resection, underscoring the need for improved therapeutic strategies. This study aimed to characterize the tumor microenvironment (TME) of CRC lung metastases and identify the factors associated with recurrence.


Materials and Methods
Fifteen CRC patients who underwent lung metastasectomy were enrolled. Multiplex immunohistochemistry (IHC), whole exome sequencing, transcriptome profiling, and single-cell RNA sequencing (scRNA-seq) were conducted on matched tumor, adjacent and distant normal lung tissues. Immune cell populations and gene expression profiles were analyzed and correlated with clinical recurrence outcomes.


Results
Exome and transcriptome analyses revealed frequent TP53, KRAS, and APC mutations. Most tumors corresponded to consensus molecular subtypes 2 and 4, characterized by immune-depleted and fibrotic features. Tumors showed downregulation of effector T and NK cell signatures. IHC revealed reduced density and increased distance of CD8+ T cells and macrophages from the epithelial cells. scRNA-seq demonstrated increased regulatory T cells and decreased NK and effector T cells in tumor. Tumor-associated macrophages (TAMs), particularly SPP1 (osteopontin)-expressing subsets, were markedly enriched in tumor and correlated with suppressed effector T cella activity. High SPP1 expression was associated with early recurrence and poor overall survival. Patients with recurrence had higher proportion of PD-1+ CD8+ T cells in adjacent normal tissues.


Conclusion
Immunosuppressive features including enrichment of SPP1+ TAMs and depletion of effector T and NK cells contribute to recurrence after CRC lung metastasectomy. Therapeutic strategies targeting both TAMs and T cells may enhance clinical outcomes in this patient population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae972bf762bae15538dd4da3de79b673b4d58b2d" target='_blank'>
              Immunosuppressive Tumor Microenvironment in Colorectal Cancer Lung Metastases: Implications for Recurrence After Metastasectomy.
              </a>
            </td>
          <td>
            Minsuk Kwon, Minkyue Shin, M. An, Y.J. Jeon, T. Hong, Jung Kyong Shin, S. Lim, Yoonah Park, Y. Cho, Seung-Tae Kim, Y. Choi, Jeeyun Lee
          </td>
          <td>2025-12-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Introduction Oropharyngeal squamous cell carcinoma (OPC) associated with high-risk human papillomavirus (HPV), particularly HPV-16, generally shows favorable outcomes yet paradoxically exhibits a high incidence of early lymphatic metastasis. The immune mechanisms underlying this phenomenon remain unclear. Methods We conducted an exploratory spatial transcriptomic analysis using the GeoMx Digital Spatial Profiler on formalin-fixed, paraffin-embedded samples from six patients with palatine tonsil-derived OPC. Tumor tissue regions (TTRs) and lymphoid follicular regions (LFRs) were compared according to HPV status and nodal involvement. Results In HPV-positive LFRs, pathways related to B-cell apoptosis appeared downregulated, suggesting prolonged B-cell survival and antigen presentation. Metastasis-negative HPV-positive cases displayed a Th2-skewed immune profile in LFRs, with increased naïve B cells, plasma cells, eosinophils, M2 macrophages, and activated mast cells. In contrast, metastasis-positive cases showed increased T cell activation in LFRs and reduced proliferation-related signaling in TTRs. Pathways involving estrogen signaling and bile acid metabolism were also associated with metastatic behavior. Conclusion These exploratory findings suggest that peritumoral Th2-biased immunity within lymphoid structures may contribute to restraining lymphatic metastasis in HPV-positive OPC. Spatial transcriptomics may provide a high-resolution framework for investigating tumor–immune interactions and generating hypotheses for future mechanistic and clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb3a9ec6ff5bf8bd62696a72523c1ce32b123f2" target='_blank'>
              Exploratory spatial transcriptomic profiling of peritumoral Th2 immune polarization in HPV-positive oropharyngeal cancer
              </a>
            </td>
          <td>
            N. Wakisaka, M. Moriyama-Kita, Satoru Kondo, E. Kobayashi, Takayoshi Ueno, Y. Nakanishi, Kazuhira Endo, H. Sugimoto, T. Yoshizaki
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Cervical cancer is among the prevalent gynecological malignancies affecting women worldwide. The diverse cellular populations and their intricate communication patterns within the tumor environment significantly influence cancer development and metastasis. Our research focused on elucidating the cellular diversity, intercellular signaling networks, and underlying molecular processes in cervical cancer tissues via single-cell transcriptomic approaches. Publicly available single-cell RNA sequencing (scRNA-seq) data for cervical cancer (GSE236738) were re-analysed to delineate the cellular composition and transcriptional programs of the tumour microenvironment. We applied UMAP and t-SNE algorithms for dimensionality reduction, alongside intercellular communication mapping, MIF pathway analysis, temporal trajectory modeling, and pathway enrichment studies, to comprehensively characterize the cellular architecture and molecular profiles within cervical cancer tissues. In addition, paired tumour and adjacent non-tumour cervical tissues from 20 patients were subjected to quantitative real-time PCR and enzyme-linked immunosorbent assay to confirm the transcriptional and protein-level expression of key genes identified by scRNA-seq. Analysis revealed substantial cellular and molecular diversity within cervical cancer specimens. Ten distinct cellular populations were characterized, including NK lymphocytes, epithelial components, and macrophage subsets, indicating pronounced cellular heterogeneity. Intercellular communication mapping revealed that stromal and epithelial populations are the primary coordinators of cell-to-cell signaling. MIF pathway activity peaked in cancer-associated fibroblasts, which exhibited distinct cell type distributions. Temporal analysis revealed dynamic cellular states and pinpointed critical regulatory elements, including ISG15. Pathway analysis emphasized the importance of neutrophil activity, mitochondrial respiration, and cell adhesion mechanisms in cancer progression. qPCR confirmed that ISG15 and TNFRSF18 were significantly upregulated in tumor tissues, while SDF4 was downregulated and NOC2L showed a moderate increase. ELISA results were consistent with transcript-level findings, demonstrating significant protein overexpression of ISG15 and TNFRSF18 in tumor tissues compared with controls. Our investigation comprehensively characterized the cellular diversity and molecular features of the cervical cancer microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7329b5a877ad7165acf8e0004372f128c5c7552" target='_blank'>
              Mapping cellular interactions and communication landscapes in cervical cancer via single-cell transcriptomics
              </a>
            </td>
          <td>
            Shaoli Zhuang, Xiaoxiao Deng, Furong Fu, Zhang Zhang
          </td>
          <td>2025-11-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077d8d4a45150f5dcadd79e304aadcb9aba01396" target='_blank'>
              YAP1 Is a Crucial Nexus in the Tumor Microenvironment.
              </a>
            </td>
          <td>
            Raghavan Narasimhan, Anshula Narayanasamy, Jaya Padmanabhan, Srikumar Chellappan, Durairaj Mohan Kumar
          </td>
          <td>2026-01-13</td>
          <td>Current medical science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is characterized by a high incidence of lymphatic metastasis and poor clinical prognosis. Cancer-associated fibroblasts (CAFs), as major component of the tumor stroma, play pivotal roles in shaping the tumor microenvironment. With recent advances of single-cell omics and spatial transcriptomics, increasing evidence has revealed the phenotypic and functional heterogeneity of CAFs and their dynamic interactions with both tumor and stromal cells. These interactions contribute significantly to lymphatic metastasis through promoting lymphangiogenesis, remodeling the extracellular matrix (ECM), and modulating immune responses. Despite the growing recognition of CAFs as key mediators of metastasis, the precise mechanisms through which CAFs drive lymphatic spread in HNSCC remain largely unknown. This gap hampers the development of effective therapeutic strategies specifically targeting CAF-driven lymphatic dissemination. This review comprehensively summarizes the current knowledge on the mechanistic roles of CAFs in lymphatic metastasis in HNSCC, encompassing lymphangiogenesis, pre-metastatic niche formation, lymph node metastasis (LNM), and extranodal extension (ENE). Furthermore, we discuss emerging therapeutic strategies targeting CAFs in other malignant tumors and their translational potential implications for mitigating lymphatic metastasis in HNSCC. Understanding the complex interactions between CAFs and the lymphatic metastatic cascade may provide new opportunities for developing targeted interventions to improve clinical outcomes in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6a6e0275ca05e00beeb61610d1e15cc6653360" target='_blank'>
              Unraveling the impact of cancer-associated fibroblasts on lymphatic metastasis of head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Zunxuan Xie, Boyang Gao, Han Wu, Fang Zheng, Qinglong Song, Cangwei Liu, Ce Shi
          </td>
          <td>2025-12-29</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy characterised by remarkable cellular heterogeneity, which emerges early from the interplay of oncogenic KRAS signalling and inflammatory injury. However, the transcriptional, metabolic and functional properties of these pre-malignant cell states that initiate and drive PDAC progression remain elusive.


OBJECTIVE
This study aimed to identify and functionally characterise the critical premalignant cell states that arise from this heterogeneity, to define novel biomarkers and targets for early intervention.


DESIGN
Public and in-house scRNA-seq data of pancreatic tumour models were analysed to identify key subpopulations in early cellular heterogeneity. Genetic perturbation in KrasG12D-driven models was performed to assess functional impact. Mechanistic studies used TurboID proximity proteomics, epigenetic profiling and metabolic assays. Clinical relevance was validated in human PDAC cohorts.


RESULTS
We identified LY6D as a marker of a distinct, gastric-like cell state that emerges early and persists throughout tumourigenesis. The LY6D+ population exhibits conserved stemness and a unique, pan-stage dependency on oxidative phosphorylation (OXPHOS). Genetic ablation of Ly6d specifically impaired the gastric lineage and delayed tumourigenesis, while its overexpression enhanced tumourigenic and metastatic potential. Mechanistically, the glycosylphosphatidylinositol (GPI)-anchored LY6D protein scaffolds a lipid raft-associated kinase network that drives FOSL1-dependent epigenetic-transcriptional reprogramming. In human PDAC, LY6D+ cells harbour stemness and Epithelial-Mesenchymal Transition (EMT) signatures, and high LY6D expression is an independent prognostic marker of poor survival.


CONCLUSION
Our work defines the LY6D+ gastric-like cell state as a key driver linking early pre-malignant heterogeneity to PDAC initiation and progression. LY6D represents a pan-stage therapeutic target and a candidate biomarker for early detection and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f42670138ad6a4407b9d3c018c7d78a840fd287d" target='_blank'>
              LY6D identifies persistent stem-like cells driving pancreatic tumourigenesis.
              </a>
            </td>
          <td>
            Juanjuan Shi, Xian Wang, Yingying Tang, Shixin Meng, Zhengyan Zhang, Ping Lu, Junyi Xu, Feier Yu, Xueni Wang, Zheng Wang, Yongwei Sun, Jing Xue
          </td>
          <td>2026-01-16</td>
          <td>Gut</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In breast cancer patients, metastasis is the stage of disease where prognosis significantly worsens. However, the timing at which metastasis initiates and the location of metastatic lesions in an organ are stochastic, limiting the timely identification of disease and the administration of treatments. Herein, we employ a synthetic metastatic niche comprised of a microporous scaffold to investigate the dynamic immune processes associated with metastatic progression. Upon implantation, the porous scaffold is infiltrated with immune cells and recruits tumor cells. We have previously reported stable tumor cell numbers in the scaffold, suggesting a state of metastatic dormancy. Towards understanding dormancy, we investigated the immune cell dynamics at the scaffold, including neutrophils, monocytes, and dendritic cells, and compared these changes to the lungs, the native metastatic niche in this model. The cell phenotypes within the scaffold microenvironment are initially polarized toward an anti-tumor phenotype and become progressively more pro-tumor with disease progression, similar to the lung microenvironment. However, the phenotypes at the scaffold are consistently less pro-tumor than the phenotypes in the lung, consistent with the lung supporting tumor cell expansion and the scaffold exhibiting dormancy. Signaling pathways identified from the analysis are consistent with the changing innate cell phenotypes, with macrophages having a significant role during the early responses and neutrophils dominating the latter stages of disease. Collectively, the scaffold captures the immune dynamics during disease progression and the signaling that underlies stable tumor cell numbers, providing a tool for investigating the mechanisms of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75a583c884e0287dc2a64d6b8d5b49487f66ecab" target='_blank'>
              Dynamic Immune Cell Composition, Phenotypes, and Signaling in an Engineered Metastatic Niche.
              </a>
            </td>
          <td>
            Rebecca S. Pereles, Jyotirmoy Roy, Michael D. Brooks, Max S. Wicha, J.S. Jeruss, Lonnie D. Shea, Sophia M. Orbach
          </td>
          <td>2026-01-10</td>
          <td>Biotechnology and bioengineering</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent epithelial malignancy. Immune-checkpoint inhibitors have reshaped first-line therapy for recurrent/metastatic disease; yet durable benefit is confined to a subset, reflecting myeloid-centric mechanisms—SPP1+ TAM barriers, cDC1/IL-12 insufficiency, and CXCL8–CXCR1/2–driven neutrophil trafficking—distinct from, and complementary to, classical lymphoid exhaustion. In this review we summarize advances from single-cell RNA and ATAC profiling and spatial transcriptomics that resolve macrophage, dendritic-cell and neutrophil programs, and appraise translational opportunities spanning myeloid reprogramming, innate–adaptive combinations and spatial biomarkers. We also discuss enduring challenges—including HPV-status heterogeneity, limited assay standardization and a scarcity of predictive metrics—that temper implementation. By integrating myeloid-informed readouts (e.g., SPP1–TAM burden, cDC1 competency, serum IL-8) with PD-1–based regimens, EGFR-directed antibodies and myeloid checkpoints (CD47–SIRPα, PI3Kγ, CXCR1/2), emerging strategies aim to restore antigen presentation, improve lymphocyte trafficking and remodel tumor–stroma interfaces. Our synthesis provides an appraisal of the evolving landscape of myeloid-informed precision immuno-oncology in HNSCC and outlines pragmatic standards and avenues for clinical translation. We hope these insights will assist researchers and clinicians as they endeavor to implement more effective, individualized regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1e89f3eb622744a2cb356a1f52faae72322d69" target='_blank'>
              Myeloid-driven immunosuppression in head and neck cancer: single-cell ATAC/RNA and spatial transcriptomic perspectives
              </a>
            </td>
          <td>
            Rui Luo, Jianzheng Yang, Zimeng Cao, Bing Li
          </td>
          <td>2025-12-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61684838fc3c53a5cd2329af114682e03471ba75" target='_blank'>
              Single-Cell Mendelian Randomization Identifies Cell-Type Specific Genetic Drivers of Lung Cancer Subtypes
              </a>
            </td>
          <td>
            Q. Yang, C. Yan, X.-f. Wang
          </td>
          <td>2025-12-02</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 190


 Background:
 Microsatellite stable colorectal cancer (MSS-CRC) in advanced-stage disease remains a major therapeutic challenge due to its molecular heterogeneity. While classification frameworks such as CMS (Consensus Molecular Subtypes) and IMF (Intrinsic CMS, MSI, Fibrosis) have improved understanding of localized CRC, they offer limited insight into the biology of metastatic tumors. To address this gap, we implemented a multi-omic study of a cohort of treatment-naïve MSS-CRC patients with synchronous liver metastases (CRLM) and a large MSS-CRC Real World Dataset (RWD).
 Methods:
 Paired primary CRC and CRLM specimens from 20 patient underwent comprehensive profiling, including Whole Exome, Transcriptome: bulk and single-nucleus RNA sequencing (snRNA-seq) analysis. The intrinsic CMS framework (iCMS) was applied to characterize tumor-intrinsic states across primary CRC and CRLM. iCMS scores were inferred using previously published templates. Key findings were validated in a real-world dataset (TEMPUS MSS-CRC) profiled using multi-omics approaches.
 Results:
 Analysis of 388,018 single nuclei revealed 6 epithelial clusters associated with specific iCMS subtypes, supported by concordant bulk RNA-seq calls. Subtype composition at snRNA-seq level revealed that approximately one-third of tumors exhibited a hybrid phenotype with admixture of iCMS2 and iCMS3 epithelial cells. Tumors that were iCMS-unclassified in bulkRNAseq consistently displayed hybrid composition in snRNA-seq. Similarity metrics (cosine distance) from bulk data correlated with admixture proportions in snRNAseq. Genomic analysis revealed mutations in
 APC
 and
 KRAS
 as the predominant genetic

 events in non-hybrid iCMS2 and iCMS3 tumors, aligned with their differential WNT and MAPK pathway activity, respectively. In contrast, iCMS-hybrid tumors were significantly enriched for clonal co-mutations in WNT and MAPK drivers, with overrepresentation of specific
 KRAS
 variants (G12D, G13D). These iCMS-hybrid tumors exhibited intermediate WNT-MAPK co-activation. In the metastatic setting, iCMS2 and iCMS3 tumors maintained their subtype identity, whereas iCMS-hybrid primary tumors showed a shift toward iCMS2 dominance in liver, suggesting that the liver microenvironment promotes expansion of the iCMS2 state.
 Conclusions:
 Our study identifies a novel iCMS-hybrid phenotype with transcriptomic plasticity and co-activation of WNT and MAPK pathways, offering refined molecular resolution and potential to guide precision therapy in advanced MSS-CRC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7dd62e9da37801a1d7c35e801c417c4239af39d" target='_blank'>
              Genetic configurations and liver niche drive tumor-intrinsic state plasticity in metastatic MSS-CRC.
              </a>
            </td>
          <td>
            Ruben Ferrer Luna, Bogdan-Alexandru Luca, Carlos Mackintosh, Rachel Webster, Micaela Tatman, Nives Rombini, Belen Rivero, Benjamin Chen, S. Tejpar, Gonzalo Lopez, María Ortiz Estévez, Ruslan Novosiadly
          </td>
          <td>2026-01-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Programmed cell death protein-1 (PD-1) blockade has shown promising clinical efficacy in hepatocellular carcinoma (HCC), yet the underlying immunological mechanisms governing response and resistance remain unclear. This study aimed to delineate the temporal and spatial dynamics of T-cell clonotypes and their relationship with pathological response in neoadjuvant PD-1 blockade therapy in HCC. By integrating T-cell receptor (TCR) repertoire analysis with transcriptomic profiling, we sought to elucidate the immune landscape alterations associated with treatment outcomes. Methods We analyzed longitudinal, matched tumor and blood samples from our previous clinical trial (NCT05471674), encompassing 114 specimens, including pre-nivolumab tumor biopsies, post-treatment resected tumors, adjacent non-tumorous livers, and peripheral blood collected before, after, and during follow-up from 19 treatment-naïve HCC. TCR clonality, diversity, and transcriptomic signatures were assessed to characterize immune responses. Results Elevated clonality of intratumoral TCR clonotypes (ITCs) post-treatment correlated with increased pathological tumor necrosis, predominantly driven by the most abundant top 1% ITCs. These dominant clonotypes occupied a significantly larger clonal space and demonstrated increased spatial sharing across tumor, non-tumorous liver, and peripheral blood compartments in nivolumab responders (nivo-Rs). Shared clonotypes were positively associated with tumor necrosis extent and cytolytic activity, suggesting active immune engagement. Post-treatment, shared clonotypes exhibited peripheral expansion and greater intratumoral dominance in nivo-Rs, whereas tumor unique intratumoral TCR clonotypes (ITCs) remained prevalent in non-responders (nivo-NRs). These divergent patterns were linked to higher chemokine expression within the tumor microenvironment of nivo-Rs and impaired human leukocyte antigen (HLA) class II antigen presentation in nivo-NRs, indicating distinct immune landscape configurations influencing therapeutic response. Conclusions Our findings demonstrate that the peripheral infiltration and expansion of high-frequency intratumoral T-cell clonotypes are critical drivers of pathological response to neoadjuvant PD-1 blockade in HCC, highlighting potential immune biomarkers and therapeutic targets to enhance immunotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/901935af9edd3945285033fa6917d3b60bbe154f" target='_blank'>
              Spatiotemporal dynamics of T cells in peripheral blood and tumor underlying differential responses to neoadjuvant PD-1 blockade in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yajing Zhang, Y. Tsui, J. Lee, T. Cheung, DanielWai-Hung Ho, I. Ng
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9990df240d9cf9afd4d976bdcb6753a231a07876" target='_blank'>
              CA9+ cancer-associated fibroblasts cooperate with SPP1+ tumor-associated macrophages driving immune resistance in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Qin Ma, Jing Wang, Qian Jiang
          </td>
          <td>2026-01-09</td>
          <td>Cellular and molecular life sciences : CMLS</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Neoadjuvant therapy (NAT) is increasingly used for pancreatic ductal adenocarcinoma (PDAC); yet most patients only achieve partial response. Pathological treatment response grading focuses on assessing residual tumor burden, often overlooking changes in tumor microenvironment (TME). To address this gap, we compared tumor cells and TME of 13 NAT-naïve and 23 post-NAT PDACs using integrated spatial pathomics and transcriptomics, with validation in an independent single-cell spatial dataset. NAT significantly reduced tumor burden (14.7%-6.2%, p = 0.004), but systemic comparison of 13 cytomorphometric features of tumor cells alone did not reliably distinguish between naïve and NAT cases. In contrast, NAT profoundly remodeled TME by increasing cancer-associated fibroblast (CAF) and CD8+ T cell densities, promoting CD8+ T cell-tumor cell proximity and fibrosis, reducing tumor-associated neutrophils, and redistributing tertiary lymphoid structures (TLSs). Spatial transcriptomics shows NAT induced apoptosis, DNA-damage response, and AGC-kinase (S_TK_X) signaling in tumor cells, and upregulated complement pathway, p53 signaling, and cellular senescence program in TME. Cross-platform single-cell spatial analysis revealed decreased regulatory T cells (Treg) and a shift from myofibroblastic (mCAF) to inflammatory CAF (iCAF). Importantly, post-NAT patients with more fibrosis had longer overall survival (p = 0.02), and higher B-cell density showed a favorable trend (p = 0.06). Together, these results suggest that beyond tumor debulking, NAT induces a coordinated TME remodeling characterized by fibroblast reprogramming, matrix fibrosis, and immune spatial reorganization. Incorporating assessment of NAT-induced stromal and immune changes into TRG may improve prognostication and guide more precise therapy in post-NAT PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc4f312a97dc39d70b704d780122da11f4cc3af9" target='_blank'>
              Neoadjuvant Therapy-Induced Remodeling in Pancreatic Ductal Adenocarcinoma: Multimodal Spatial Analysis and Prognosis.
              </a>
            </td>
          <td>
            Xiaofei Zhang, Ruoxin Lan, Danting Li, Yongjun Liu, Sonu Kalyan, Momin Iqbal, Nancy Liu, Jerry Zhang, Iman Hanna, Mala Gupta, C. Zhao, Weiguo Liu, Jonathan Melamed, Michael Shusterman, Jessica Widmer, John Allendorf, Yao-zhong Liu
          </td>
          <td>2026-01-13</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive tumor type with poor patient outcomes. Most patients present with metastatic disease, which generally has reduced immune infiltration compared to primary tumors. Further work to elucidate the specific cellular features of metastatic PDAC is needed to guide the development of future immunotherapy strategies. Here, we investigated the hypothesis that PDAC tumors harbor distinct immunologic and stromal features depending on their anatomical site. Multiplex immunohistochemistry (mIHC), spatial analysis, and single-cell mass cytometry (CyTOF) uncovered dominant immune and stromal cell populations in tumors derived from 27 primary and 26 liver metastases. Metastatic liver tumors from PDAC patients contained reduced T cell infiltration, fibroblast populations, and collagen accumulation than primary lesions, while CD68+ cells, often co-expressing CCR2, were more abundant. Spatial analyses revealed distinct immune cell communities in primary and metastatic PDAC, whereby CK19+ cells clustered differentially with α-SMA+, CD3+, and CD68+ cells, depending on the tumor site. When comparing tumor-associated regions, the proportion of peritumoral CK19- cells remained consistent, but their composition varied by disease site. CD8+ T cells were significantly less frequent in metastatic tumors, while both CD4+ and CD8+ T cells present in primary tumors expressed more transcription factors (TFs) associated with suppressive properties, including FoxP3 and RORγt. CyTOF revealed that T cells co-expressed multiple inhibitory checkpoint receptors, most notably LAG-3 and PD-1. This report reveals that primary and metastatic tumors from PDAC patients harbor vastly distinct immunologic and stromal features at the protein level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bea286cd97f7f8064af812daefa94a7da6b7d73" target='_blank'>
              High-Dimensional Protein Analysis Uncovers Distinct Immunological and Stromal Features in Primary and Metastatic Pancreatic Ductal Adenocarcinoma.
              </a>
            </td>
          <td>
            E. Greene, N. Horvat, D. Doxie, V. Parihar, Jayden Kim, Cameron J Herting, Erin E. Grundy, Ayana T Ruffin, Alyssa M Krasinskas, Shishir K Maithel, Juan M. Sarmiento, Mihir M. Shah, Mohammad Y Zaidi, Maria Diab, O. Alese, Kavita M. Dhodapkar, H. Kissick, B. El-Rayes, Chrystal M. Paulos, G. Lesinski
          </td>
          <td>2025-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d360e6a196e9de4b8313a49f5e3b66fe3b338c6" target='_blank'>
              Integrative single-cell and spatial transcriptomics with explainable AI reveal lethal prognostic axis in prostate cancer.
              </a>
            </td>
          <td>
            Qintao Ge, Zhenda Wang, Yangyun Wang, Tonghui Chu, Zhongyuan Wang, Wenkai Zhu, Youzhao Zhang, Yonghao Chen, Dingwei Ye, Wenhao Xu, Zhongyuan Wang, Mierxiati Abudurexiti
          </td>
          <td>2026-01-06</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Rationale: Tertiary lymphoid structures (TLSs) play a key role in the adaptive immune response within the local tumor immune microenvironment (TIME) and can predict the clinical outcome of several solid tumors. However, the clinical relevance of TLSs and their formation mechanism in gastrointestinal stromal tumors (GISTs) remain unclear. Methods: Integration analysis was performed on a single-cell RNA sequencing (scRNA-seq) cohort (n = 8), a public transcriptome cohort (n = 65), a public scRNA-seq cohort (n = 7), a tissue cohort (n = 197) and a serum cohort (n = 169) to decode the characteristics of the immunological microenvironment of GIST. The multiplex immunohistochemistry (mIHC) staining, in vitro cell culture, chemotaxis assays and antibody-dependent cellular phagocytosis (ADCP) experiments were used to validate the results of the bioinformatics analysis. Results: Preoperative imatinib targeted therapy significantly enhanced TLS formation in GIST tissues, predicting improved therapeutic efficacy and favorable prognosis. Mechanistically, imatinib remodeled the local microenvironment via tumour-associated macrophages, recruiting B cells via the CXCR4-CXCL12 axis to drive TLS development. Functionally, TLS dominated germinal centre (GC) B-cell differentiation and the formation of IgG+ plasma cells (PCs), which preferentially enhanced the adaptive immune response through the ADCP effect. From a clinical perspective, we identified three distinct GIST immune classes (GICs A-C). GIC-A tumors featured abundant CD20+ B cells and TLS, as well as a favorable prognosis. They were accompanied by enhanced antigen presentation, accumulation of IgG+ PCs, increased immunosuppressive properties and a high frequency of KIT exon 11 mutations. These mutations potentially serve as a predictive biomarker for future targeted and immunotherapies. Furthermore, patients with high serum IgG levels experienced significant therapeutic benefits. Conclusions: Our data show that local adaptive immunity dominated by TLS is a key factor in the efficacy of targeted therapy, and suggest that inducing IgG could be a feasible strategy for improving the prognosis of patients with GIST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/485f7a996f0585ddffcf4dd3ac8a1112abd5a345" target='_blank'>
              Enhanced formation of tertiary lymphoid structures shapes the anti-tumor microenvironment in gastrointestinal stromal tumors after imatinib targeted therapy
              </a>
            </td>
          <td>
            Ping Tao, Jiongyuan Wang, Peidang Fan, Wen-Shuai Liu, Wei-Qi Lu, Yujie Hu, Q. Lu, Lijie Ma, Hanxing Tong, Yong Zhang
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Patient-derived organoids (PDOs) have emerged as transformative preclinical models in non-small cell lung cancer (NSCLC), offering high-fidelity recapitulation of tumor heterogeneity and drug responses. Compared to traditional cell lines and xenografts, PDOs preserve the genetic, phenotypic, and functional features of parental tumors, enabling precise drug sensitivity testing for chemotherapy, targeted therapy, and immunotherapy, particularly through optimized culture protocols, genetic engineering techniques, and cryopreservation methods, have significantly enhanced their scalability and clinical relevance. PDOs have proven instrumental in elucidating key resistance mechanisms such as EGFR-TKI resistance mediated through DCLK1-dependent Wnt signaling activation, while simultaneously identifying novel therapeutic synergies for clinical translatio. However, challenges remain in modeling the tumor immune microenvironment and standardizing clinical translation. This review systematically outlines the advancements and challenges in establishing NSCLC PDOs, highlights the potential of PDOs to guide personalized NSCLC therapy while addressing current limitations to bridge the gap between research and clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a0a3f8f865fb80834ba9f16e69c07a2097e28f2" target='_blank'>
              Patient-derived organoids in non-small cell lung cancer: advances in drug sensitivity testing
              </a>
            </td>
          <td>
            Wanyu Tang, Xudong Tian
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is the most aggressive primary brain tumor in adults, marked by rapid progression and poor prognosis. GBM may also harbor a complex virome, defined here as the total repertoire of viral elements within tumors, encompassing both exogenous viruses and reactivated endogenous retroviruses. While viral signatures have been observed in other cancers, their transcriptional activity in GBM and their distribution across tumor cells and extracellular compartments remain largely unexplored. Methods We analyzed 45 long- and small-RNA libraries from glioma stem-like cells, their extracellular fractions (exosomes, microvesicles, ribonucleoprotein complexes), and normal brain controls (BioProject PRJNA360129). Taxonomic profiling and functional annotation were performed using established pipelines (MetaPhlAn4, BLASTn, UniProtKB, InterPro), with fresh media controls to exclude background contamination. To validate discriminative power, viral RNA profiles were classified using machine learning -Random Forest models trained on nine species identified as glioblastoma-enriched by LEfSe. Results We detected ~3.41 million viral reads, with >99% from GBM cells and extracellular compartments and <1% from normal brain. Reads spanned 36 species across 11 phyla. Nine species were consistently enriched in glioblastoma: Saimiriine gammaherpesvirus 2, Human endogenous retrovirus K, Camelpox virus, Pestivirus A, Macacine gammaherpesvirus 5, Avian myelocytomatosis virus, Rabbit fibroma virus, Omegapapillomavirus 1, and Cyprinid herpesvirus 2 with genes mapping to oncogenic pathways. Random Forest models showed long-RNA profiles perfectly discriminated against tumor from normal samples (AUC = 1.0), whereas small-RNA profiles carried limited signal (AUC = 0.5). Tumor–extracellular comparisons revealed strong compartment-specific long-RNA signatures (AUC = 1.0), with partial overlap between tumor cells and exosomes in small-RNA data (AUC = 0.75). Conclusion Glioblastoma contains a distinct viral transcriptome; long-RNA profiles show strong diagnostic power, and compartment-specific signatures suggest biomarker potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f03aa5d91d0d223768b844afb632280b82ecb5f9" target='_blank'>
              The glioblastoma virome across tumor and extracellular compartments
              </a>
            </td>
          <td>
            M. Ghorbani, H. Karimi, Farnoosh Solati, Nazanin Anbarestani, Nooshin Khoshdoozmasouleh
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The Tracking Cancer Evolution Through Therapy (TRACERx) program represents the most comprehensive effort to characterize tumor evolution in real time. Through longitudinal, multiregion, and multiomic profiling of tumors—and particularly of non-small-cell lung cancer and clear cell renal cell carcinoma—TRACERx has illuminated the dynamic interplay between genetic, nongenetic, and (micro)environmental factors that drive cancer progression, immune evasion, and therapeutic resistance. A central insight from TRACERx has been that not all tumor evolution is genomic: Transcriptomic diversity, epigenetic alterations, RNA editing, and changes in cell–cell interactions also drive adaptation. Methodological innovations—including tumor-informed and ultrasensitive circulating tumor DNA assays, representative sequencing, and integrative immune–genomic analyses—have yielded biomarkers resistant to sampling bias and/or predictive of recurrence, metastasis, and treatment response. By demonstrating that intratumor heterogeneity is a key determinant of clinical outcome and revealing its molecular, transcriptional, and ecosystem-level drivers, TRACERx has established a framework for linking evolutionary dynamics to patient care. As both a scientific framework and a clinical paradigm, TRACERx demonstrates how adaptive, iterative research can refine evolutionary models, improve patient risk stratification, and inspire next-generation cancer evolution studies across malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a59431b9259c4928178579f6c460cf8e369f4f1" target='_blank'>
              TRACERx as an Optimized Paradigm for Understanding Cancer Evolution
              </a>
            </td>
          <td>
            Naomi Iris van den Berg, C. Swanton, S. Turajlic
          </td>
          <td>2025-12-24</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Single-cell analysis has refined our understanding of cellular heterogeneity in glioma, yet RNA alternative splicing, a critical layer of transcriptome regulation, remains underexplored at single-cell resolution. Here, we present a pan-glioma single-cell alternative splicing analysis in both tumor and immune cells through integrating seven SMART-seq2 datasets of human gliomas to uncover overlooked isoform-level regulations shaping glioma progression and immune responses. Our analysis reveals lineage-specific alternative splicing regulation among glioma cellular states, with the most divergent alternative splicing landscapes observed between mesenchymal and neuronal-like glioma cells. Notably, this includes events in TCF12 exon 15 and PTBP2 exon 10, two key regulators of mesenchymal and neuronal transcriptomes. Inducing TCF12 exon 15 inclusion in glioma cells with a dCasRx-RBM25 system promotes neuronal gene expression, suppresses genes related to extracellular matrix organization, and enhances sensitivity to radiotherapy. Comparison of core and peripheral glioma cells highlights alternative splicing redox co-regulation of TPM1 and ACTN4, key genes involved in cytoskeletal organization. Further analysis of glioma-infiltrating immune cells reveals altered first exon usage of UGP2 in regulatory T cells and an association between MS4A7 alternative splicing in macrophages and clinical response to anti-PD-1 therapy. This study emphasizes the role of alternative splicing in glioma cellular heterogeneity, highlighting the importance of an isoform-centric approach to better understand the complex biological processes driving tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0607c4921b5c1be62ed81a8c96b4eebea1605a52" target='_blank'>
              A single-cell atlas of RNA alternative splicing in the glioma-immune ecosystem
              </a>
            </td>
          <td>
            Xiao Song, D. Tiek, Minghui Lu, Xiaozhou Yu, Runxin Wu, Maya Walker, Qiu He, Derek Sisbarro, Bo Hu, Shi-Yuan Cheng
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology</td>
          <td>2</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (iCCA) is a highly metastatic tumor type, and iCCA with intrahepatic metastasis is associated with poor prognosis. Previous studies have confirmed that the tumor microenvironment (TME) is closely related to tumor metastasis. However, the state of the TME during iCCA liver metastasis remains unknown.


APPROACH AND RESULTS
We profiled 28 specimens from 16 patients with iCCA (with and without intrahepatic metastasis) using integrated single-cell and spatial transcriptomics, bulk RNA sequencing, whole-exome sequencing, and in vivo and in vitro functional experiments to characterize the specific TME changes during iCCA liver metastasis. we identified the metastatic tumor cells, COL3A1+ epithelial cells, and revealed that COL3A1+ epithelial cells-endothelial-to-mesenchymal transition cells crosstalk promoted tumor invasion by inducing mesenchymal transformation of endothelial cells, while COL3A1+ epithelial cells-VEGFA+ CCL4+ neutrophils crosstalk promoted tumor colonization by forming neutrophil extracellular traps.


CONCLUSIONS
We revealed the key biological mechanisms involved in the invasion and colonization phases of iCCA liver metastasis. Moreover, we provide valuable data for understanding iCCA liver metastasis and a possible avenue for the treatment of advanced iCCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd684325d487cd1d69724768d37142d24e7c6b41" target='_blank'>
              Spatiotemporal landscape of intrahepatic cholangiocarcinoma liver metastasis at the single-cell level.
              </a>
            </td>
          <td>
            Ziyu Xun, Hao Wang, Zhengfeng Xuan, Kexu Xiong, Na Zhang, J. Long, Huishan Sun, Yiran Li, Chengpei Zhu, M. Piao, Ting Zhang, Longhao Zhang, Shuofeng Li, Chengjie Li, Jiongyuan Li, Boyu Sun, Zi-Qi Zhou, Shanshan Wang, Ziyue Huang, Kai Liu, Yang Tan, Xiaohua Shi, Xiaobo Yang, Hanping Wang, Ling Lu, Haitao Zhao
          </td>
          <td>2025-12-10</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="AbstractAbstract Background The intra- and inter-tumoral heterogeneity of glioblastoma (GBM) represents a significant therapeutic challenge and barrier to the generation of reliable and accurate models for in vitro studies. Three-dimensional models, such as patient-derived neurospheres (PDN) and organoids (PDO), recapitulate complex cellular states. However, the direct comparison of models derived in parallel from the same primary tumor tissue has never been performed. The aim of this study was to determine the tumor cell composition of PDN and PDO models relative to matched primary GBM tissue. Methods Four GBM surgical samples were used to establish matched PDO and PDN models, which were genomically verified using single nucleotide polymorphism array. DNA methylation, histology, and transcriptome were examined for intra-tumoral heterogeneity correlating with the matched GBM tissue. PDN lines were used to investigate the tumor cell composition response to temozolomide chemotherapy over time, maximizing their utility. Results We find that both patient-derived models recapitulate the genomic, epigenomic, and tumor cell heterogeneity of the primary tissue at similar capacity. Furthermore, single-nuclei RNA sequencing revealed a subset of organoids containing small numbers of non-malignant cells from neuron and immune cell lineages. Harnessing the intra-tumoral heterogeneity of PDN models, we reveal the impact of temozolomide chemotherapy on individual cell states, altering composition of tumors over time in response to treatment. Conclusions Our data confirms that both PDN and PDO patient-derived models recapitulate patient intra-tumoral heterogeneity providing a platform for tumor cell state refined therapeutic studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5585413e6addae1bd1c700e1b837828da5ca47a" target='_blank'>
              Patient-derived organoids and neurospheres recapitulate salient features of primary tumor heterogeneity
              </a>
            </td>
          <td>
            Zachery Moore, Claire Storey, Daniel V Brown, A. Valkovic, Montana Spiteri, Jasmine F. Pignatelli, Shannon J. Oliver, A. Fakhri, Katharine J Drummond, Jessica Hart, L. Harris, Seth Malinowski, Keith L Ligon, Oliver M Sieber, J. Whittle, S. Freytag, Sarah A. Best
          </td>
          <td>2025-11-22</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic cancer, recognized as a refractory tumor, has an overall survival rate of less than 10%, and its mortality rate continues to rise. Due to the low immune activity induced by its unique tumor microenvironment, pancreatic cancer is classified as a “cold” tumor and is insensitive to current immunotherapies. However, little is still known about the identification and functional mechanisms of key regulatory molecules in the formation of “cold” tumors. In this study, we identified the pancreatic ductal cell niche through single-cell sequencing. By calculating CNV scores using inferCNV to distinguish malignant from non-malignant cells and analyzing differences in transcriptional levels, we constructed a 15-gene model. The inhibitory effect of UBE2H on the immune microenvironment was studied through single-cell sequencing analysis, including the inhibition of T cell function, impairment of antigen presentation in macrophages and neutrophils, and activation of neutrophil extracellular traps (NETs). Our results revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche, primarily observed in pancreatic cancer samples. Notably, single-cell sequencing analysis of UBE2H overexpression revealed an enhanced T cell PD-L1 interaction axis and a downregulation of antigen presentation pathways in macrophages and neutrophils (antigen processing and presentation, macrophage activation, and neutrophil activation). Further in vivo experiments confirmed that high UBE2H expression promotes tumor progression, leading to increased T cell exhaustion (PD1) and decreased activation (CD69, GZMB, IFNR) in the immune microenvironment, impaired antigen presentation of macrophages and neutrophils (H2Kb, I-Ab, I-A/E), and increased neutrophil extracellular trap formation (MPO, NE, CITH3). This study revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche and analyzed the inhibition of UBE2H on the immune microenvironment of pancreatic cancer through single-cell sequencing and in vivo experiments, providing important clues for the formation of “cold” tumors in pancreatic cancer and opening new directions for exploring new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ecc8251593af0fcef99c0ebc48949b3c7fba9f" target='_blank'>
              Integrated scRNA-seq and transcriptome analyses uncover the effects of UBE2H on the immune microenvironment regulation in pancreatic cancer
              </a>
            </td>
          <td>
            Fuxin Huang, Zhongyan Zhang, Jike Fang, Yue Chen, Jinhui Wei, Quanzhang Li, Chuanzhao Zhang, Shanzhou Huang, B. Hou
          </td>
          <td>2025-11-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Brain metastases are frequent and often lethal complications of advanced cancers. Microglia, resident immune cells of the brain, are known to exert both anti-tumor and pro-tumor functions in late-stage metastases; however, their response during the initial outgrowth of metastatic lesions is not well characterized. Understanding how heterogeneous microglial subgroups are regulated in the developing tumor microenvironment could pave the way for therapeutic strategies to eliminate metastatic tumors at an early stage. In this study, we used a combination of in vivo fate map imaging, single-cell RNA sequencing, and a holographic photoconversion-based technique (Opto-omics) to track tumor fate and early microglial responses over time in the same animals during colonization of disseminated tumor cells. The microglial population was transcriptionally and morphologically heterogeneous, comprising both pro- and anti-tumor subsets. Genetic and pharmacological perturbations revealed that microglial phenotypes could be shifted by inhibiting TGF-β signaling or by deleting the tumor cell surface antigens CD24a and CD47. These findings reveal targetable plasticity in early-stage microglial responses to brain metastasis and suggest that harnessing pro-phagocytic microglial states may offer a therapeutic window before systemic immunosuppression becomes dominant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1778e4e2ee1302e3145504143572e313e56713fc" target='_blank'>
              Microglia Display Heterogeneous Initial Responses to Disseminated Tumor Cells.
              </a>
            </td>
          <td>
            T. Tsuji, Haruka Hirose, Daisuke Sugiyama, Mariko Shindo, Rahadian Yudo Hartantyo, Yutaro Saito, Tsuyako Tatematsu, Shouta Sugio, M. Sanbo, Masumi Hirabayashi, Yasuhiro Kojima, Jun Koseki, K. Hosoya, Hiroshi Yoshida, Tatsuya Ogimoto, Y. Yasuda, Kentaro Hashimoto, Hitomi Ajimizu, Yuichi Sakamori, H. Yoshida, Noritaka Sano, Masahiro Tanji, Hiroaki Ito, Kazuhiro Terada, M. Hamaji, T. Menju, Hiroyuki Konishi, S. Kumagai, Cyrus M. Ghajar, Daisuke Kato, Hiroshi Date, Akihiko Yoshizawa, Yoshiki Arakawa, H. Ozasa, A. Moorhouse, Teppei Shimamura, H. Nishikawa, Toyohiro Hirai, Hiroaki Wake
          </td>
          <td>2025-12-10</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9eea39a3d2bb28b39f9d4aacd093433e9784cc7" target='_blank'>
              High-Throughput, Multiplex Immunofluorescence for the Computer-automated Immunophenotyping of Mycosis Fungoides
              </a>
            </td>
          <td>
            Patrick McMullan, Marc R. Benoit, Nathan Gasek, Sydney Riddick, Joseph Masison, Katalin Ferenczi, David Rowe, Gillian Weston
          </td>
          <td>2026-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Due to their intrinsic heterogeneity and plasticity, pediatric brain tumors present highly complex clinical challenges. To provide insight into the molecular scene of these tumors, we generated spatial transcriptomics data from 19 distinct patients, 7 of which suffered a relapse of the disease. In this cohort, spanning 59 tissue sections across 8 diagnoses and 4 tumor grades, we recovered diagnosis-specific gene expression patterns that could be linked to processes characteristic of developmental stages. We also identified between 2 to 4 spatial archetypal niches per section, which could then be related to 11 main biological themes, and to distinct celltype-like transcriptomic signatures. For example, we noted strong spatial correlations between developmental archetypes and oligodendrocyte lineage signal in pilocytic astrocytoma. Lastly, we utilised spatially inferred copy-number variants for the profiling of relapse-associated tentative tumor clones. These clones were then related via spatial geographical analysis to regions with strong blood vessel signatures. Overall, these results provide vital details into the progression and maintenance of pediatric brain tumors, offering novel edges to be exploited in personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f69864dd9511e02d9a815817c172642be7190a7d" target='_blank'>
              Surveying the molecular landscape of pediatric brain tumors via spatially resolved transcriptomics
              </a>
            </td>
          <td>
            Javier Escudero Morlanes, Timo-Pekka K Lehto, L. Larsson, L. A. Galicia, A. Mollbrink, Alia Shamikh, Elisa Basmaci, Gabriela Prochazka, T. de Ståhl, Johanna Sandgren, F. Taylan, Bianca Tesi, Ann Nordgren, Andrew Erickson, Alastair D Lamb, Klas Blomgren, Monica Nistér, J. Lundeberg, R. Mirzazadeh, L. Kvastad
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Gastric cancer (GC) exhibits molecular heterogeneity and diverse immune cell infiltration patterns closely associated with patient prognosis. However, a comprehensive understanding of the variations in immune cell phenotypes among different patient subgroups still needs to be improved. In this study, we performed a detailed analysis of the tumor microenvironment in GC by integrating 200,466 single cells from 72 patients across six datasets. We classified patients into immune-deserted, B, T, and myeloid cell subtypes. Using genomic and clinical data from TCGA samples, we identified cellular components associated with tumor histology and genotypes. GC patients were stratified into immune-deserted, B cell, T cell, and myeloid cell subtypes, and we described the pathway and transcription factor activity characteristics of different microenvironment subtypes. Integration of bulk RNA-seq data reveals that fibroblasts and endothelial cells were associated with adverse patient outcomes whereas NK and T cells were notably correlated with improved prognosis. Subsequently, we focused on characterizing cancer-associated fibroblasts (CAFs) and discovered that they acquire new functional properties within the tissue microenvironment, providing evidence of CAF plasticity. We constructed a novel four-gene CAF signature including SPARC, EFEMP1, RGS5 and SERPINE1 which may enhance patient stratification and prognostic prediction of GC patients. qPCR analysis revealed that the significant expressions of SPARC, EFEMP1, RGS5 and SERPINE1 were significantly upregulated in gastric cancer tissues compared to the normal tissues. Our study provides insights into the composition of the tumor microenvironment and construction of a four-gene CAF signature associated with clinical prognosis, offering new perspectives for the clinical management of gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6b22dc4e8c3bd98a01915b46ab3333a465c3c7e" target='_blank'>
              Integrative single-cell atlas unveils heterogeneity and prognostic value of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Ziyu Jiang, Xiaomei Zhuang, Junkui Guo, Minyi Zhu, Chunhong Hong, Chunhui Sun, Kaiming Wu, Haofan Yin, Cuncan Deng, P. Jiang
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/517f57f9e6c335d60cb5a3e0fde4a76539819f12" target='_blank'>
              A Mechanophenotyping chip for high-throughput detection of metastatic bacteria-infected circulating tumor cells.
              </a>
            </td>
          <td>
            Wenxin Luo, Yanfeng Gao, Shujun Feng, Bangshun He, Jingjing Li, Jingjing Yang, Yi Yin, Meng Wang, Bin Xue, Yi Cao, Tony Y Hu, Yujun Song
          </td>
          <td>2026-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The immune microenvironment profoundly shapes the progression and therapeutic response of thyroid carcinoma. Through comprehensive analysis of single-cell RNA sequencing data, this review delineates the immune landscapes of papillary thyroid carcinoma (PTC), poorly differentiated thyroid carcinomas (PDTC), and anaplastic thyroid carcinomas (ATC), revealing a differentiation dependent trajectory of tumor immune microenvironment remodeling—from immune activation suppression coexistence in PTC, to immune exclusion in PDTC, and terminal exhaustion in ATC. This single-cell based approach enables high resolution dissection of cellular heterogeneity, immune crosstalk, and spatial organization that are often masked in bulk analyses. Such insights provide a scientific basis for precision immunotherapy, offering guidance for differentiation tailored strategies to overcome immune escape and improve clinical outcomes in thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/662dd84def56730e4ddb7a83c80225823dda796d" target='_blank'>
              Single-cell sequencing reveals the tumor immune microenvironment in thyroid cancer: a narrow review
              </a>
            </td>
          <td>
            Kangcong Liang, Ziyu Wang, Zhiqiang Zhang, Gengluan Liu, Xidi Wang, Heng Cao, Ming Zhong, Liping Ye, Xin Zhong, Jingyu Xun, Kefeng Lei, Ningning Li
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d749d1664733485d97fc50139d1df5bec7e20cae" target='_blank'>
              Reconstructing the metastatic journey: functional circulating tumor cells and disseminated tumor cells based models for translational oncology.
              </a>
            </td>
          <td>
            Caroline Sanglier, L. Cayrefourcq, C. Alix‐Panabières
          </td>
          <td>2026-01-14</td>
          <td>Journal of experimental & clinical cancer research : CR</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Spatial transcriptomics, which deals with tissue architecture in genetic investigation, is an innovative technique for examining cell heterogeneity and tissue organization. This review emphasizes major approaches, include spatially resolved transcriptome methods, immunohistochemistry as well as in situ hybridization, all of which permit the mapping of RNA molecules in their native tissue environment. These methods have proven essential in achieving our understanding of biological events such as tumor evolution, progression of cancer, and cancer tumor stem cell detection. Spatial transcriptomics, the study of patterns of gene expression in space, reveals the intricate nature of the tumor microenvironment (TME) and its effect on cancer biology. Although it delivers insight on the cellular connections that underlie disease, the significance of spatial transcriptomics in multiple organs has expanded.

Although its immense potential, there are still difficulties to be conquered, particularly within the areas of analysis of data, spatial resolution, and integration with other omics data. To be able to fully comprehend the complexities of tissues biology and ailments, this review additionally tackles future potential avenues, including the necessity for greater multiplexing, enhanced resolution, and the combination of functional genomics. With this synthesis, we intend provide an extensive summary of the state of spatial transcriptomics currently and demonstrate that it possesses the potential to improve precision medicine, cancer research, and our understanding of broader biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579b9e172c2769b4ad99d0da42fa24b04f1ee980" target='_blank'>
              Spatial Transcriptomics in Cancer Research: Methods and Applications
              </a>
            </td>
          <td>
            P. K. Kumar, K. M. Rao, Syed Mohiuddin Quadri, Bagadam Aparna, V. U. M. rao
          </td>
          <td>2025-12-18</td>
          <td>International Journal of Current Science Research and Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Hepatocellular carcinoma (HCC) exhibits pronounced intratumoral heterogeneity and a complex immune microenvironment, which together limit therapeutic efficacy. Identifying actionable biomarkers and mechanisms of immune modulation remains critical for improving patient outcomes.



 We integrated single-cell RNA sequencing data with bulk transcriptomic datasets to comprehensively characterize tumor cell heterogeneity and immune landscape features in HCC. Associations between RDH16 expression and clinicopathological characteristics were evaluated, and in vitro functional assays were conducted. Mendelian randomization analyses were performed to assess causal relationships.



 RDH16 expression was significantly reduced in HCC tissues compared with non-tumor liver tissues and was associated with vascular invasion, elevated alpha-fetoprotein levels, poor tumor differentiation, and advanced T stage. Higher RDH16 expression was consistently associated with improved overall prognosis. Functional assays indicated that RDH16 did not directly affect tumor cell proliferation, migration, or invasion. Notably, RDH16 expression was inversely correlated with infiltration of CD163⁺ M2 macrophages, suggesting a potential immunomodulatory role in the tumor microenvironment. Mendelian randomization analyses further supported a protective effect of higher RDH16 expression against HCC risk.



 These findings identify RDH16 as an immune-modulating biomarker in HCC, highlighting its potential role in shaping the tumor immune microenvironment and suggesting new avenues for personalized immunotherapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/444ab6526b7dd129322dafa940c094f836b618fb" target='_blank'>
              Single-cell transcriptomics reveals a novel mechanism of RDH16 regulating immune infiltration in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zhenzhen Zhang, Rui Fan, Jing Ma, Dehui Li, Yan Jiang, Fahui Liu, Qiming Gong
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="High-throughput"pathomic"analysis of Whole Slide Images (WSIs) offers new opportunities to study tissue characteristics and for biomarker discovery. However, the clinical relevance of the tissue characteristics at the micro- and macro-environment level is limited by the lack of tools that facilitate the measurement of the spatial interaction of individual structure characteristics and their association with clinical parameters. To address these challenges, we introduce HistoWAS (Histology-Wide Association Study), a computational framework designed to link tissue spatial organization to clinical outcomes. Specifically, HistoWAS implements (1) a feature space that augments conventional metrics with 30 topological and spatial features, adapted from Geographic Information Systems (GIS) point pattern analysis, to quantify tissue micro-architecture; and (2) an association study engine, inspired by Phenome-Wide Association Studies (PheWAS), that performs mass univariate regression for each feature with statistical correction. As a proof of concept, we applied HistoWAS to analyze a total of 102 features (72 conventional object-level features and our 30 spatial features) using 385 PAS-stained WSIs from 206 participants in the Kidney Precision Medicine Project (KPMP). The code and data have been released to https://github.com/hrlblab/histoWAS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407dd7f4b75161f7922e517e4737dcf219aed8fa" target='_blank'>
              HistoWAS: A Pathomics Framework for Large-Scale Feature-Wide Association Studies of Tissue Topology and Patient Outcomes
              </a>
            </td>
          <td>
            Yuechen Yang, Junlin Guo, Yanfan Zhu, Jialin Yue, Junchao Zhu, Yu Wang, Shilin Zhao, Haichun Yang, Xingyi Guo, Jovan Tanevski, Laura Barisoni, Avi Rosenberg, Yuankai Huo
          </td>
          <td>2025-12-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b2473c436968a0964a051f1901617b06af4fa0b" target='_blank'>
              Systemic activation and tissue infiltration of CD8 + CX3CR1 + T cells in non-small cell lung cancer treated with neoadjuvant immune checkpoint blockade.
              </a>
            </td>
          <td>
            Sisi Dai, Yongcheng Liu, Tingting Guo, Hong Luo, Guanglei Yang, Songliang Yu, Sifan Zhang, Lili Jiang, Jie Liu, Ye Wang, Daxing Zhu, Xuyu Cai
          </td>
          <td>2025-11-22</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Diffuse large B-cell lymphoma (DLBCL) is an aggressive cancer with profound heterogeneity. While standard immunochemotherapy is curative for the majority, treatment failure in 30–40% of patients due to refractory or relapsed disease remains a significant clinical challenge, emphasizing the need for better prognostic markers. We investigated the interplay between tumor glycolytic metabolism and the tumor immune microenvironment. We identified four distinct glycolysis-related patterns that were associated with three immune subtypes (known as immune-inflamed, immune-excluded, and immune-desert). Importantly, these patterns stratified clinical outcomes and predicted chemotherapy sensitivity. For example, tumors with an immune-inflamed subtype were more sensitive to doxorubicin. Collectively, our findings propose a metabolism- and immune-based classification framework for DLBCL that may facilitate more precise risk stratification and treatment optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e894f2d6a04501c694424859815514794b12befc" target='_blank'>
              Glycolysis-Driven Immune Subtypes in the Tumor Microenvironment Determine Clinical Outcomes in Diffuse Large B-Cell Lymphoma
              </a>
            </td>
          <td>
            Ying Du, Zhenyu Liu, Chuanyan Liu, Yining Yuan, Mengkun Fang, Tong Zhou, Mengge Pan, Weilu Xu, Xu Liu, Peipei Xu
          </td>
          <td>2025-12-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4393b9168f5ebc4a80f332a0eba92fab9bfeef3" target='_blank'>
              Single-Cell Multiomic Profiling Uncovers Radiation Dosage-Sensitive, Cluster-Specific Regulatory Dynamics in Glioblastoma
              </a>
            </td>
          <td>
            Khoi Huynh, Sara Nalina Barcik Weissman, Cheol Park, Maria L.G. Quiloan, Blake Sun Chang, Eric L Chang, Kelly Street, David Tran, A. Modrek
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary This review was conducted to address the complexities of breast cancer research, which is often limited by two-dimensional cell cultures that fail to accurately represent the tumor microenvironment. Although three-dimensional (3D) models are generally superior, the absence of standardized methodologies has resulted in inconsistent findings. The objective of this review was to systematically analyze how different 3D culture methods influence the epigenetic reprogramming of tumor cells, focusing on changes in DNA methylation, histone modifications, and chromatin organization, as well as the differential expression of various types of non-coding RNA. Our findings demonstrate that different 3D culture methods are not equivalent and yield distinct epigenetic signatures. This underscores the urgent need to standardize protocols and provide detailed reports on the properties of culture media to enhance reproducibility, which is crucial for identifying new therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/933fb504bd5970ab97b7de6c042e05e852c76571" target='_blank'>
              Epigenetic and Transcriptional Reprogramming in 3D Culture Models in Breast Cancer
              </a>
            </td>
          <td>
            Laura Cecilia Flores-García, Karla Rubio, Eloísa Ibarra-Sierra, M. B. Silva-Cázares, C. Palma-Flores, C. López-Camarillo
          </td>
          <td>2025-11-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9d84cab81cd323269c1a623f592cc3353c4b7c" target='_blank'>
              Organoid Pharmacotyping of Pancreatic Cancer Enables Functional Precision Oncology and Drug Repurposing
              </a>
            </td>
          <td>
            He Dong, F. Vizeacoumar, Yue Zhang, Nicholas Jette, J. Price, Vincent Maranda, Lihui Gong, T. Freywald, J. P. Vizeacoumar, M. Lazell-Wright, Saruul Uuganbayar, Jaylynn Quail, Alain Morejon Morales, Ayaan Rizvi, Rani Kanthan, Yuliang Wu, Anand Krishnan, Kathleen Felton, Bilal Marwa, Laura Hopkins, Gary Groot, J. Shaw, Gavin Beck, Yigang Luo, Maurice Ogaick, Mike Moser, A. Freywald, Shahid Ahmed, A. Zaidi, Franco J Vizeacoumar
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The field effect describes the phenomena where environmental exposures, infection, and genetic predisposition result in molecular changes in cells that predispose them to developing cancer. Though this is a well-established concept in pathology, it remains underexplored in the context of high-resolution omics. We utilized the Diagnostic Evidence Gauge of Single Cells (DEGAS) deep transfer learning framework to analyze prostate cancer spatial transcriptomics to identify cells and tissues that are highly associated with cancer progression. DEGAS highlighted morphologically benign glands that had reduced expression of MSMB, a differentiation marker that is decreased in aggressive tumors. These glands have upregulated genes associated with antigen presentation and aggressive neoplasms. Integration of single-cell transcriptomics and deep learning image analysis separately revealed altered immune-cell infiltration, suggesting a complex interplay in the tumor environment facilitating aggressiveness. We used immunohistochemistry to quantify the MSMB protein (PSP-94) expression on morphologically normal and tumor tissues from patients with and without 5-year distant metastasis. Samples from patients who developed metastasis consistently showed lower fractions of positively stained cells, indicating a subtle yet significant "field effect" in seemingly benign regions. These proteomic results validate the transcriptomic findings and further underscore that inflammatory or immune-related changes in ostensibly normal tissue may contribute to aggressive disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68d7d78a2fef9048f9bcafbf03164f9002e0042" target='_blank'>
              Deep Transfer Learning Links Benign Glands to Prostate Cancer Progression via Transcriptomics.
              </a>
            </td>
          <td>
            Justin L Couetil, Ziyuan Liu, Chao Chen, A. Alomari, Kun Huang, Jie Zhang, Travis S. Johnson
          </td>
          <td>2025-11-29</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progression involves dynamic remodeling of the tumor microenvironment (TME). However, the stage-specific dynamics of immune and stromal cell remodeling throughout LUAD progression remain incompletely understood. Here, the study systematically profiles the cellular composition and transcriptional states across multiple LUAD stages, integrating early-stage patient specimens with publicly available datasets encompassing advanced-stage disease. The analysis reveals a marked stage-dependent shift from a proliferative and immune-activated microenvironment in early LUAD to a hypoxia-enriched and immunosuppressive landscape in advanced disease. A distinct hypoxia-adapted epithelial tumor cell subpopulation (C5), exhibiting transcriptional features of metastasis, invasion, and hypoxia, and poor prognosis, is identified. Advanced LUAD featured immunosuppressive LGMN⁺ macrophages and STAT1-driven exhausted CD8⁺ T cells. FKBP11⁺ plasma B cells exhibited exhaustion-linked metabolic changes. POSTN⁺ CAFs emerged as central mediators of extracellular matrix (ECM) remodeling and immune exclusion. Collectively, the findings reveal a model of hypoxia-driven functional convergence, in which distinct TME components co-evolve toward phenotypes that collectively promote immune evasion, matrix remodeling, and tumor progression. These findings may provide insights into stage-specific cellular dynamics and highlight promising therapeutic targets for precision immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8876e01109cf8f024442bf59e35e9ed246f97f" target='_blank'>
              Characterizing Stage-Specific Cellular Dynamics and Microenvironmental Remodeling in Lung Adenocarcinoma by Single-Cell RNA Sequencing.
              </a>
            </td>
          <td>
            Bomiao Qing, Xiaolan Li, Xiang He, Junyi Wang, Yi Yang, Manling Jiang, Bingbing Yan, Lei Zhang, Anying Xiong, Q. Ran, Guoping Li
          </td>
          <td>2025-11-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Precision medicine has evolved through distinct phases, from the origins of the Human Genome Project to mutation‐based targeted therapies. This editorial posits that “stereological cell biomedicine” could be a new approach promoting the development of the next generation of precision medicine. This emerging discipline transitions the focus from genomic data to the multi‐dimensional and spatiotemporal complexity of single cells. Driven by advances in Stereo single‐cell multi‐omics (Stereo Cell‐seq), spatial transcriptomics (Stereo‐seq), and single‐cell surfaceomics (sc‐surfaceome), this approach aims to capture the stereologically dynamic interactions between organelles within a cell and between cells in the tissue. We argue that understanding the spatiotemporal location of molecules, particularly protein interactions at organelle interfaces and on the cell surface, is as critical as their abundance for defining cellular function in health and disease. Integrating these high‐resolution measurements with artificial intelligence and computational modelling will bridge the gap between advanced omics and pathology. Initiatives such as the newly established European Stereo Cell Center (ESCC) signal a global shift towards this new paradigm, which promises to unlock novel diagnostic biomarkers and therapeutic targets for truly multi‐factorial and dynamic precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acdcd69ccf472862bb486c6c2a16a93191b0f96" target='_blank'>
              Stereo cell: A new approach to the next generation of clinical precision medicine
              </a>
            </td>
          <td>
            Wanxin Duan, Mingjie Wang, Yifei Liu, Celine Desoyer, Christian Baumgartner, Xiangdong Wang
          </td>
          <td>2025-11-28</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ade8a8b9df4408ea89e61a0e1bd36268d586bf5" target='_blank'>
              Decoding osteosarcoma from heterogeneity to precision therapy.
              </a>
            </td>
          <td>
            Yu Xu, Yanfu Bai, Fan Yang, Binbin Yang, Zhisheng Huang, Lingling Su, Kuntao Peng
          </td>
          <td>2025-12-11</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e26bc66a34e745a67c0425c16be84d04301c9194" target='_blank'>
              Multimodal analysis reveals potential association of CDH13 with endothelial cells and its overexpression in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Ke-Jun Wu, Li-Hua Yang, Dong-Ming Li, Rong-Quan He, Di-Yuan Qin, Shi-De Li, Jian-Di Li, Y. Dang, Ming-Jie Li, Qi Li, Jian-jun Li, Lu Zhang, Han He, Ji-Feng He, Gang Chen, Xiao-Bo Jiang
          </td>
          <td>2025-12-12</td>
          <td>European journal of medical research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays an important role in the development, progression, and treatment response of pediatric cancers, yet remains less elucidated compared to adult malignancies. Pediatric tumors are unique with a low mutational burden, an immature immune landscape, and unique stromal interactions. The resultant “cold” immune microenvironments limits the effectiveness of conventional immunotherapies. This review summarizes the key cellular and non-cellular components of the pediatric TME—including T cells, NK cells, tumor-associated macrophages, cancer-associated fibroblasts, extracellular matrix remodeling, angiogenesis, and hypoxia—and describes how these elements shape tumor behavior and therapy resistance. The role of TME in common pediatric cancers like leukemia, lymphoma, neuroblastoma, brain tumors, renal tumors, and sarcomas is discussed. Emerging therapeutic strategies targeting immune checkpoints, macrophage polarization, angiogenic pathways, and stromal barriers are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34cc7bfcda14da44bca02f277e5096c55bbdde0" target='_blank'>
              Tumor Microenvironment: Current Understanding and Therapeutic Implications in Adult and Pediatric Cancers
              </a>
            </td>
          <td>
            Satyendra Batra, Prashant Prabhakar, Debabrata Mohapatra, Noreen Grace George, Neha Goel, Bhavika Rishi, Aroonima Misra, Amitabh Singh
          </td>
          <td>2025-12-25</td>
          <td>Onco</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ebcdfb94400220dc54cbd6a2cfd578841de975f" target='_blank'>
              Integrated transcriptomic and single-cell analysis reveals cell cycle dysregulation and cellular heterogeneity in lung cancer.
              </a>
            </td>
          <td>
            Yuzi Zhang, Wencheng Che, Qingchuan Li, Wei Huang
          </td>
          <td>2025-12-30</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Prostate cancer commonly presents as multifocal disease, with distinct tumours that are often genetically independent. Metastases, however, typically derive from a single dominant clone, highlighting the clinical challenge of identifying which lesion drives progression. Focal therapy, an emerging treatment strategy, aims to target only the most aggressive tumour focus or foci, sparing normal prostate structures and reducing morbidity. Yet, the lack of consistent genomic drivers in primary tumours complicates efforts to identify the focus most likely to progress. While prostate cancers are known to harbour widespread methylation alterations, the extent to which these epigenetic changes track with clonal origin remains poorly understood. To address this, we performed multiregional epigenomic sequencing of 189 prostate tissue samples, including 109 tumour foci and 80 non-tumour regions from 44 patients (2-8 tumour samples per case). Differential methylation analysis revealed widespread alterations distinguishing benign from tumour tissue, including a core ‘epigenetic trunk’ of >20,000 recurrently altered CpGs shared across cases. DNA variants called from non-methylation sites of the epigenomic sequencing closely matched variants from WGS in cases with WGS available. Phylogenetic trees constructed from DNA variants and phyloepigenetic trees from methylation sequencing revealed striking patterns. Some cases lacked shared DNA variants across all tumour foci, indicating a polyclonal tumour origin, yet these genetically unrelated foci shared extensive methylation alterations. This suggests widespread methylation convergence occurs in clonally distinct tumour foci. To dissect clonal relationships further, we applied two orthogonal lineage tracing approaches using the methylation sequencing data: fluctuating CpGs (sites that change stochastically over time) and highly entropic 8-mer ‘methylation barcodes’ within the protocadherin gene cluster. Fluctuating CpGs showed concordant states across tumours that shared DNA sequence variants, while the methylation barcodes likewise revealed high similarity among such tumours. Together, these methods support the independent evolutionary origins of polyclonal prostate tumour foci. Pathway analysis of recurrently altered methylation loci highlighted consistent enrichment in epithelial–mesenchymal transition, MYC signalling, DNA damage response, and hormonal signalling across cases, suggesting functional convergence at the pathway level despite divergent genetic origins, consistent with selective pressures acting on shared programmes. Together, these findings demonstrate that DNA methylation alterations represent fundamental and recurrent events in prostate cancer, more consistent than DNA sequence variants across multifocal disease. Lineage-tracing analyses further highlight that methylation processes record tumour evolutionary history, encoding both clonal relationships and convergent biology. These observations may inform methylation-based biomarkers and the rational selection of lesions for focal therapy.



 Tamsin J. Robb, Melissa Cheung, Rajbir Batra, Henson Lee Yu, John C. Thomas, Anne Warren, Andrew Lynch, Daniel Brewer, David Wedge, CRUK-ICGC Prostate Cancer Group, Charles Massie, Harveer Dev. Widespread methylation convergence in clonally distinct foci of multifocal prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90717932598b7c081fea25f61e12e100d9757059" target='_blank'>
              Abstract B003: Widespread methylation convergence in clonally distinct foci of multifocal prostate cancer
              </a>
            </td>
          <td>
            Tamsin J. Robb, Melissa Cheung, Rajbir Batra, Henson Lee Yu, John C. Thomas, Anne Y Warren, Andrew Lynch, Daniel Brewer, D. Wedge, Charles Massie, H. Dev
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a self-renewing population capable of fueling long-term tumor growth. In gastric cancer, the identity of CSC populations remains unclear. In this study, we established a gastric CSC population marked by the water channel protein aquaporin-5 (AQP5), which resides in human and mouse pyloric tumors. Using multiple organoid and mouse models, we found a requirement for AQP5+ CSCs in both initiating and sustaining cancer progression and demonstrated that AQP5 expression also directly promotes tumor growth and invasion in a WNT, PI3K (phosphatidylinositol 3-kinase), and MAPK (mitogen-activated protein kinase)-dependent manner. Beyond primary cancers, AQP5 further enriches for a functional CSC population in metastatic tumors. Together, our findings support a CSC model in gastric tumors that may have application for therapeutic strategies targeting CSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b857f221b4120247f60ae689da7ba8ea312dc26b" target='_blank'>
              AQP5: A functional gastric cancer stem cell marker in mouse and human tumors.
              </a>
            </td>
          <td>
            H. Lim, Swathi Yada, Kazuhiro Murakami, B. Lee, Sowmya Sagiraju, Phyllis Phuah, Tanysha Chi-Ying Chen, Fidelia B Alvina, S. Tan, Kaushal Krishna Kaslikar, Nur Syahirah Binte Ruhazat, Menaka Priyadharsani Rajapakse, Katzrin Bte Ahmed Murad, Snezhina Kancheva, Liang Thing Tan, Seri Mustafah, Jimmy Bok Yan So, Nick Barker
          </td>
          <td>2025-12-18</td>
          <td>Science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tumor-Associated Macrophages play a significant role in the tumor microenvironment of breast cancer, influencing its growth and progression. Their presence is associated with key processes such as angiogenesis and immunomodulation, impacting the immune system response and tumor aggressiveness. This study employed various methods to analyze scRNA-seq and spatial transcriptome data. The scRNA-seq data underwent thorough processing and analysis, with a focus on breast cancer subtypes. The CellChat R package was employed to analyze and visualize cell-cell communication networks within the scRNA-seq data. hdWGCNA, conducted in R, involved multiple steps such as data preprocessing, gene network construction, module identification, module preservation analysis, and functional enrichment analysis. To analyze the pseudotime dynamics of endothelial cells, Monocle3 was utilized. The spatial data for normal skin and leprosy were obtained and analyzed using the SPATA2 and semla R packages, facilitating the integration of spatial and single-cell data. The study utilized scRNA-seq and spatial transcriptome data to comprehensively analyze 40 breast cancer samples, including ER+, HER2+, and TNBC subtypes, alongside normal mammary tissue. Distinct cellular clusters were identified, and myeloid cells were characterized into four subtypes. Comparative analyses across breast cancer subtypes highlighted variations in the expression of the CXCL10 gene, and hdWGCNA provided insights into molecular landscapes. Trajectory inference analysis revealed pseudotime dynamics in myeloid cells, with specific gene expression changes. Spatial transcriptome profiling in TNBC and HER2+ breast cancer showed unique cellular clusters and gene expression patterns within the tumor microenvironment. Spatial and single-cell data integration emphasized the distribution of cell types and spatial enrichment of gene expression signatures.  TAMs emerge as key players engaging in multifaceted interactions with neighboring cell types within the intricate context of breast cancer. These identified genes, particularly CXCL10, assume central roles in regulating various facets of myeloid cell physiology and modulating immune responses. CXCL10 expression increases in TNBC macrophages suggest a potential association with T lymphocyte activity and myeloid cell chemotaxis, implicating its significance in the intricate immune response dynamics within the TNBC microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/666c93cdc1f48635e7eb409847dc8e7a85718b81" target='_blank'>
              Spatial profiling of CXCL10 in breast cancer: tumor-associated macrophages' mediation of immune response
              </a>
            </td>
          <td>
            Natália Cordeiro de Albuquerque D’Alva, Ana Kelly da Silva Fernandes Duarte, Leonardo Vinícius da Silva Lima, Eduardo Paulo da Silva Lima, Caio Cesar Fernandes Nobre Porto, Maria Nathália de Menezes, Luis Filipe Pereira Rocha, Leonardo Prudêncio Coutinho de Almeida, Rafael de Oliveira Calaça Farias, Leonardo Barbosa da Costa, Salomão Belfort Sparapan de Melo, Raimundo Rodrigues de França Júnior, Rodolfo Tibério Ferreira Silva, Alberson Maylson Ramos DA Silva, Taina Santos Bezerra
          </td>
          <td>2025-12-04</td>
          <td>CONTRIBUCIONES A LAS CIENCIAS SOCIALES</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits profound cellular heterogeneity, the understanding of which is critical for improving prognosis and therapy. Using single-cell RNA sequencing of 32,247 cells from human HCC samples, we characterized the tumor ecosystem and identified five malignant hepatocyte subpopulations with distinct molecular profiles and stage-specific enrichment. Among these, the S100A6⁺ C1 and S100A9⁺ C4 subpopulations were predominantly associated with advanced tumors and actively remodeled the tumor microenvironment through enhanced signaling pathways such as MDK and MIF. We further identified PGAM2 as a key transcriptional regulator in early-stage tumors, whose activity correlated with sialylation—a process linked to immune evasion. Based on these findings, we developed a prognostic model integrating PGAM2 and sialylation-related genes, which robustly stratified patients into high- and low-risk groups with significantly different survival outcomes, immune contextures, and predicted therapeutic responses. Functional experiments validated AGRN, a component of the signature, as a functional driver of HCC proliferation and invasion. Collectively, our results decode the cellular and molecular heterogeneity of HCC, provide a clinically relevant prognostic tool, and highlight potential targets for further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d560ef3a66a46cb286a615fdcaac30d8c4ef51e5" target='_blank'>
              Machine learning-driven comprehensive profiling of tumor heterogeneity and sialylation in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kaiqiang Tang, Lu Han, Junlin Li, Kang Li
          </td>
          <td>2025-12-17</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor cell–vasculature interactions play a crucial role in lung cancer progression and metastasis; however, traditional cell culture models rarely capture these complex dynamics. We developed a vascularized lung cancer organoid (VLCO) model by coculturing lung cancer organoids (LCOs) with blood vessel organoids (BVOs) to recreate the tumor–vasculature niche in a three-dimensional in vitro system. During coculture, LCOs migrated along BVO-formed vascular networks and invaded vessel-like structures, mimicking intravasation. The organoid types interacted via secretory factors, including TGF-β and PDGF-BB, inducing epithelial–mesenchymal transition (EMT) in LCOs and pericyte-to-fibroblast transition in BVOs. We also developed a scalable drug response assay for vasculature-induced EMT, enabling targeted therapeutic screening. The VLCO model provides a robust platform for studying tumor–endothelial crosstalk and testing antimetastatic therapies targeting tumor–vasculature interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb6f6bbc13e8f32592986de52884782ace9253f1" target='_blank'>
              Modeling the metastatic tumor microenvironment: a co-culture platform of lung cancer and blood vessel organoids for drug evaluation
              </a>
            </td>
          <td>
            Mingyun Lee, Yeojoon Yoon, Ji Yeon Lee, Mina Hwang, Sewon Park, Mingyun Lee, Sang-Won Um, Byoung Chul Cho, J. Rheey, Toshiro Sato, Yong-Soo Lee
          </td>
          <td>2025-11-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Bone metastases are a common and devastating complication of advanced prostate cancer (PCa), with up to 80–90% of men with castration-resistant PCa (CRPC) developing metastases to the bone. The development of bone metastasis serves as a critical indicator of worse long-term clinical outcomes. Understanding the factors driving PCa metastasis to the bone, including through myeloid cell reprogramming within the tumor microenvironment (TME), is critical. Our study examined published single-cell RNA sequencing data from 7 PCa patients with bone metastasis and 7 benign donors, focusing on myeloid cells across tumor, involved marrow, distal marrow, and benign bone factions. We used manual annotation methods and scANVI for whole integrated data annotation. Myeloid cells were sub-classed and manually annotated for classical monocytes (Mono1), intermediate monocytes (Mono2), non-classical monocytes (Mono3), myeloid dendritic cells (mDCs), tumor-infiltrating monocytes (TIMs), tumor-associated macrophages (TAMs), and osteoclasts. We applied pseudotime and used CellRank's Random-Walk Simulation for trajectory inference to determine final terminal states for the myeloid cells, i.e., tumor-associated macrophages (TAMs), myeloid dendritic cells (mDCs), and osteoclasts. Ranked driver gene lists for every sample type (benign, distal, involved, tumor) and cell type (Mono1/2/3, TIM) that were associated with the final terminal states were generated. QIAGEN Ingenuity Pathway Analysis (IPA) and Gene Set Enrichment Analysis (GSEA) were performed on driver gene lists across sample types and cell types, thus identifying upstream regulators influencing terminal state formation. IPA and GSEA identified ATF3, a stress-response regulator, and NF-κB, a central inflammatory mediator, to be significantly associated with TAM differentiation driver genes and were shared across extremes of sample types (benign & tumor) and cell types (Mono1 & TIM). CellOracle's in-silico perturbations were performed to assess the significance of ATF3 in context of the PCa bone TME while visualizing impacts on cell development. Through siRNA suppression of ATF3 in-vitro, followed by flow cytometry and qPCR for knockdown validation, our results show reduced M2/TAM differentiation with decreased associated markers, suggesting a role for ATF3 in myeloid reprogramming that may contribute to TAM-associated immunosuppression and the metastatic disease state



 Azimullah Rifai, Safiullah Rifai, Zumar Meher, Tausif Khan, Wei Guang, Linbo Wang, Mohamad Khan, Arif Hussain. Rewiring the tumor microenvironment: Uncovering lineage drivers of immunosuppressive tumor-associated macrophages in prostate cancer bone metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b3000b262dc2818edcc8efcb35668c9389013e" target='_blank'>
              Abstract A019: Rewiring the tumor microenvironment: Uncovering lineage drivers of immunosuppressive tumor-associated macrophages in prostate cancer bone metastasis
              </a>
            </td>
          <td>
            Azimullah Rifai, Safiullah Rifai, Zumar Meher, Tausif Khan, Wei Guang, Linbo Wang, Mohamad Khan, Arif Hussain
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinomas (cSCCs) arise from keratinocytes in the skin, but the molecular changes driving this transformation remain unclear. To better understand this process, we perform multi-omic profiling of keratinocytes, actinic keratoses, and cSCCs. Single-cell mutational analyses reveal that most keratinocytes have remarkably low mutation burdens; however, keratinocytes with TP53 or NOTCH1 mutations exhibit substantially higher burdens. These findings suggest that keratinocytes can withstand high dosages of cumulative ultraviolet radiation, but certain pathogenic mutations break these adaptive mechanisms, inducing a mutator phenotype. Mutational profiling of cSCCs adjacent to actinic keratoses reveals TERT promoter and CDKN2A mutations emerge in actinic keratoses, whereas additional mutations that inactivate ARID2 and activate the mitogen-activated protein kinase pathway delineate the transition to cSCC. Surprisingly, actinic keratoses are often not related to their neighboring cSCC. Spatial analyses reveal gene expression heterogeneity, including checkpoint molecule enrichment at invasive fronts, highlighting tumor and immune cell interactions. The evolution of cutaneous squamous cell carcinoma (cSCC) remains poorly understood. Here, the authors employ multi-omics and multi-scale analyses to explore the genetic evolution of keratinocytes to cSCC, finding key pathogenic mutations that break the resistance to ultraviolet radiation as well as spatial heterogeneity patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ace63c32efba22479251c352c77035a045dd5b73" target='_blank'>
              Genetic evolution of keratinocytes to cutaneous squamous cell carcinoma
              </a>
            </td>
          <td>
            Bishal Tandukar, Delahny Deivendran, Limin Chen, A. K. Bandari, Noel Cruz-Pacheco, H. Sharma, Meng Wang, Albert Xu, Daniel B. Chen, Christopher D. George, Annika L. Marty, Raymond J. Cho, Jeffrey B. Cheng, Drew K Saylor, P. Gerami, I. Yeh, Sarah T. Arron, Boris C. Bastian, A. H. Shain
          </td>
          <td>2025-11-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a90c7d87d145743b8f163f5d40e3c8d84fc50c6" target='_blank'>
              Single-cell phylodynamics reveal rapid late-stage colorectal cancer expansions
              </a>
            </td>
          <td>
            J. M. Alves, Kylie Chen, Sonia Prado-López, N. Estévez-Gómez, P. Alvariño, Juana Alonso, Monica V. Valecha, Laura Tomás, A. Zwaans, Ugnė Stolz, Tanja Stadler, Débora Chantada, J. Cameselle-Teijeiro, Alexei Drummond, D. Posada
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Primary tracheobronchial tumors (PTBTs) can often be successfully treated by surgery; however, more effective strategies are still warranted for advance tumor cases. The present study aimed to provide a comprehensive understanding of the molecular and cellular features of PTBTs, along with the tumor microenvironment (TME). By performing single-cell RNA, we revealed a distinct TME landscape of PTBTs containing 52,739 single cells. For instance, these three pathological types were found have heterogeneous fibroblast populations, immune cells were more activated in inflammatory myofibroblastic tumor (IMT). Notably, a subpopulation of BIRC5+ cells was identified in primary tracheal squamous cell carcinoma (SCC) and adenoid cystic carcinoma (ACC) samples, which may have similar properties to cancer stem cells (CSCs). Moreover, IMT was characterized by the infiltration of pro-inflammatory immune cells, and might be a benign tumor. Importantly, based on ligand-receptor analysis, the PPIA/CD147 pair was found to be strongly upregulated during tumor and CD8+ T cell interactions. By applying multiplex immunofluorescence for selected markers, we validated the presence of the separate cell entities. Our findings provide deeper insights into the biology of PTBTs, which may be beneficial for advancing the diagnosis and therapy of PTBTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc41a4f5248143a0c3cf35d4c57326e220116bee" target='_blank'>
              A single-cell atlas reveals the tumorigenesis and microenvironment status of primary tracheobronchial tumors
              </a>
            </td>
          <td>
            Chen Shu, Xiyang Tang, Kaifu Zheng, Runze Zhang, Yulong Zhou, Mengchao Li, Weiguang Du, Yuquan Bai, Yang Shen, Yujian Liu, Haitang Yang, Feng Yao, Xiaolong Yan, Nan Ma, Jinbo Zhao
          </td>
          <td>2025-12-13</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Most breast biopsies show non-cancerous (benign) changes, yet some women later develop breast cancer. Predicting who is at higher risk from these small tissue samples is difficult because the relevant structures are tiny and easily exhausted by testing. We evaluated whether a multiplex technology called digital spatial profiling can measure many proteins at once on tissue microarrays built from benign breast lobules, nearby tissue next to cancers, and cancers themselves. We also compared the results to standard lab tests. The approach worked well overall and reproduced known biology, especially in cancer tissue. Signals in benign lobules were more variable and did not, on their own, distinguish women who later developed cancer from matched controls in this pilot. These findings show that high-plex, tissue-conserving profiling is feasible and biologically coherent, and they outline concrete ways to improve sampling and marker panels for future risk-prediction studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4db98f919a3b2d7e640d624464895a0f412d05ad" target='_blank'>
              Tissue Microarray-Based Digital Spatial Profiling of Benign Breast Lobules and Breast Cancers: Feasibility, Biological Coherence, and Cross-Platform Benchmarks
              </a>
            </td>
          <td>
            Cristian Scatena, Carmine De Angelis, Mark E. Sherman, Jodi C. Carter, Rob Vierkant, M. Stallings-Mann, Laura M Pacheco-Spann, Stacey J. Winham, Celine M Vachon, Chen Wang, M. Jensen, M. Troester, Amy C. Degnim, E. A. Thompson, Jennifer M. Kachergus, Ji Shi, Derek C. Radisky
          </td>
          <td>2025-11-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background: Atherosclerosis, a leading cause of cardiovascular disease, involves complex interactions between vascular and immune cells. The role of lymphatic endothelial cells (LECs) in this process remains incompletely characterized, limiting our understanding of disease mechanisms. This study aimed to delineate the phenotypic and functional dynamics of LECs during atherosclerosis progression. Methods: We performed single-cell RNA sequencing on aortic cells from ApoE-/- mice on a high-fat diet at baseline, 8 weeks (early disease), and 16 weeks (late disease). Bioinformatic analyses, including clustering, differential expression, trajectory inference, and cell-cell communication analysis, were applied to characterize LEC subpopulations and their transcriptional reprogramming. Results: Our analysis identified two LEC subpopulations that exhibited a biphasic numerical response: expansion at the early stage followed by a decline by the late stage. Early-disease LECs displayed altered immunomodulatory capacity, with features of reduced T cell tolerance and enhanced activation via IL-7/IL-7R signaling, coupled with a downregulation of key lipid-handling genes (Ldlr, Abca1). Trajectory analysis suggested multiple cellular origins, including a conventional but delayed differentiation path from vascular endothelial cells and an atherosclerosis-specific transdifferentiation path from fibroblasts observed only in early disease. Conclusions: Our findings indicate that LECs undergo substantial phenotypic and functional alterations during atherosclerosis. The maladaptive differentiation and acquired dysfunction in lipid transport and immune regulation may contribute to disease progression. This study provides a foundational transcriptional atlas for understanding lymphatic involvement in vascular disease and highlights potential contexts for therapeutic modulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/846d22c5db232fdb36562bbd0bfea5b322f4411d" target='_blank'>
              Dysfunction and Pathological Origins of Lymphatic Endothelial Cells in Atherosclerosis Revealed by Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Qinhang Shen, Guangchao Gu, Dan Yang, Yuehong Zheng
          </td>
          <td>2025-11-21</td>
          <td>Genes</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) often occurs in combination with sarcopenia (SP), which affects the prognosis of CRC, but the histopathological mechanisms underlying this process are unclear. Single-cell transcriptome technology provides a tool for resolving multi-organ pathological interactions, and previous studies have lacked direct molecular evidence of a trans-tissue microenvironment. To address this, we applied an integrated analytical approach. The integration of single-cell transcriptome data from 16 CRC tumor tissues (scRNA-seq) and 17 SP muscle tissues (snRNA-seq) was used to systematically resolve heterogeneity across tissue microenvironments using Harmony to correct for batch effects and combining cellular annotations, differential expression analyses, and CellChat communication modeling. Our analysis revealed that CRC and SP share 3 core cell types, namely, endothelial cells, fibroblasts and monocytes, but they show distinct transcriptional and functional differentiation in different tissues. CRC cells are enriched in the inflammatory, angiogenic and matrix remodeling pathways, accompanied by the upregulation of hypoxia-associated genes, whereas SP is biased toward muscle structural maintenance, metabolic regulation and repair functions. The MIF-CD74, COLLAGEN and THBS-CD47 pathways were active in CRC, whereas SDC4 and CD36-related signals were dominant in SP. VWF, PDGFRA and FCN1 were stably expressed in both groups, suggesting the existence of a conserved homeostatic regulatory network. In conclusion, this study reveals the cross-tissue functional heterogeneity of cell types shared by CRC and SP, suggests a set of potential pervasive regulators, and provides new perspectives for understanding the systemic effects of tumors and the microenvironmental mechanisms of sarcopenia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28aac551ab88d08b24ed6a70bb80afdcde46ff4f" target='_blank'>
              Single-cell transcriptome analysis elucidates the microenvironmental interactions between colorectal cancer and sarcopenia
              </a>
            </td>
          <td>
            Tihong Qiu, Xiaoying Wang, Jie Li, Yiping Guan, Hongyou Wen
          </td>
          <td>2025-11-28</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5407cdb757b46038485ad236efb4e8ad5a8a93" target='_blank'>
              Tumor-associated macrophages promote chemoresistance to Paclitaxel via activating NOTCH2-JAG1 juxtacrine signaling.
              </a>
            </td>
          <td>
            Fazhi Yu, Qin Zhou, Weiqiang Yu, Tong Zhou, Cheng Cao, Yijia Xie, Peng Zhang, Hanyuan Liu, Wei He, Aoxing Cheng, Xiaopeng Ma, Qingfa Wu, Qi Zhao, Jing Guo, Kaiguang Zhang, Ying Zhou, Jue Shi, Zhenye Yang
          </td>
          <td>2026-01-10</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="In the growing tumor, hypoxia may lead to tumor necrosis which is associated with more aggressive tumor features and reduced patient survival. However, there are shortcomings in our understanding of biological features and clinical significance linked to tumor necrosis. We therefore analyzed transcriptome mRNA expression profiles from primary breast tumors from TCGA and METABRIC (n = 2289) including TCGA mutational data (n = 409). We employed bioinformatics analysis and independent literature-based signatures for validation to identify alterations unique to necrosis. Necrosis was associated with aggressive tumor features and strongly predicted the basal-like breast cancer phenotype. Increased tumor cell proliferation, hypoxia, stemness, and epithelial-to-mesenchymal transition (EMT) were observed in tumors with necrosis. From gene expression data, we constructed a novel Breast Cancer Necrosis Signature (BCNS) score that was a strong predictor of the basal-like phenotype but also conveyed information relevant to the luminal subtypes of breast cancer, including prognosis. Mutational profiling pointed to enrichment of TP53 and PIK3CA mutations in tumors with and without necrosis, respectively. This study confirms the association of breast cancer necrosis with aggressive tumor features and reduced survival, and points to the BCNS score as a potential biomarker that should be further explored and validated to improve breast cancer diagnosis and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/067acf02eaac0d3a5661ec642434a729418ce808" target='_blank'>
              Tumor necrosis associates with aggressive breast cancer features, increased hypoxia signaling and reduced patient survival
              </a>
            </td>
          <td>
            Astrid A Syrtveit, L. M. Ingebriktsen, Amalie F Tegnander, L. Akslen, E. Wik, E. Hoivik
          </td>
          <td>2025-11-27</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Introduction The analysis of tumor-infiltrating lymphocytes often requires techniques that expand their numbers, potentially introducing bias. To address this, we performed a detailed analysis of minimally cultured TILs to evaluate whether this approach better preserves their characteristics. Methods The TIL culture method was based solely on tumor tissue with low IL-2 supplementation to minimize artificial alterations. The validity of this approach was confirmed by the correlation between CD3+ T cell percentages in cultures and infiltration patterns observed by immunohistochemistry. Immunophenotyping, cytokine release, and TCR repertoire analysis were used to characterize CD4+ and CD8+ T cell subsets and their molecular features during minimal expansions. Results High TIL infiltration areas did not consistently correspond to an increased presence of any T cell subset; both CD4+ and CD8+ T cells frequently coexisted in these regions. In contrast, low TIL infiltration sections often displayed a higher proportion of CD4+ T cells. An inverse correlation between CD4+ T cell percentages and cytotoxic molecules was observed, indicating reduced cytotoxic activity in low-TIL sections with abundant CD4+ T cells. TCR repertoire analysis revealed differences between T cell subsets: CD4+ T cells were associated with longer TRA CDR3 nt and shorter TRB N(D)N nt lengths, along with lower diversity, while CD8+ T cells did not exhibit significant correlation with any TCR feature. Discussion This study highlights the distinct biological features of CD4⁺ and CD8⁺ TIL populations within the tumor microenvironment that can be preserved using a minimally expanded TIL approach. The observed associations between IHC patterns, T cell subset composition, cytotoxic potential, and TCR repertoire diversity help identify which biopsy regions yield TILs with greater therapeutic potential, thus providing guidance for TIL selection in immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c988d227329c53bf2d2653e81969a85f3a08fee" target='_blank'>
              Minimally expanded breast cancer tumor-infiltrating-lymphocytes provide guidance for therapeutic selection
              </a>
            </td>
          <td>
            A. Aran, Gonzalo Lázaro, Vicente Marco, V. Peg, Maitane Faus, L. Garrigós, J. Pérez-García, Javier Cortés, Mercè Martí
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background: Clear cell renal cell carcinoma (ccRCC) is the most common subtype of kidney cancer and is marked by pronounced intra-tumoural heterogeneity that complicates therapeutic response. Patient-derived organoids offer a physiologically relevant model to capture this diversity and evaluate treatment effects. When integrated with spatial transcriptomics, they might enable the mapping of spatially resolved transcriptional and isoform-level changes within the tumour microenvironment. Methods: We established a robust workflow for generating patient-derived ccRCC organoids, that are not passaged and retain original cellular components. These retain key features of the original tumours, including cancer cell, stromal, and immune components. Results: Spatial transcriptomic profiling revealed multiple transcriptionally distinct regions within and across organoids, reflecting the intrinsic heterogeneity of ccRCC. Isoform-level analysis identified spatially variable expression of glutaminase (GLS) isoforms, with heterogeneous distributions of both the GAC and KGA variants. Treatment with NUC-7738, a phosphoramidate derivative of 3′-deoxyadenosine, induced marked transcriptional remodelling of organoids, including alterations in ribosomal and mitochondrial gene expression. Conclusions: This study demonstrates that combining long-read spatial transcriptomics with patient-derived organoid models provides a powerful and scalable approach for dissecting gene and isoform-level heterogeneity in ccRCC and for elucidating spatially resolved transcriptional responses to novel therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11ea98f536949abee4bc7391c3123466548631ed" target='_blank'>
              Long-Read Spatial Transcriptomics of Patient-Derived Clear Cell Renal Cell Carcinoma Organoids Identifies Heterogeneity and Transcriptional Remodelling Following NUC-7738 Treatment
              </a>
            </td>
          <td>
            Hazem Abdullah, Ying Zhang, Kathryn Kirkwood, Alexander Laird, Peter Mullen, David J. Harrison, Mustafa Elshani
          </td>
          <td>2026-01-14</td>
          <td>Cancers</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Traditionally, scientists tend to approach cancer research in a reductionistic way: aiming at uncovering underlying, separate components in malignant processes. And indeed, great progress has been made by reducing the development of a tumor to single, specific genes and mutations. For instance, familial adenomatous polyposis (FAP) could be reduced to a germline mutation in the Adenomatous Polyposis Coli (APC) gene. The escape of tumor cells from immune surveillance could be reduced to the tumor expression of immune checkpoints, resulting in new approaches in tumor therapy by applying immune checkpoint inhibitors. However, a germline mutation in APC is not 1:1 related to colorectal cancer (CRC), and only some patients respond to immune checkpoint inhibitors. The point here is that biological systems, also comprising cancer, have properties that cannot be reduced to single components. The cooperation of the single components results in new, emergent properties. The outcome of an interaction in a complex network, like the immune system, depends on the many cell types involved and the numerous molecules that interact and activate or inhibit pathways. The way the composing elements are organized is a causal factor in itself for any emergent property. The rise of genomic analysis at the end of the previous century, enabling us to sequence a full genome at the DNA and RNA levels, has initiated an awareness of the need for ‘systems biology’: to consider a full system and how it is organized, in all of its aspects, to understand biological pathways and their outcomes. In this review, we outline the prospects and limitations of systems biology in cancer research and propose a causal framework that integrates upward and downward causation and multiple realizability to understand the emergent properties of tumors that determine the dynamics of tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bad26730d321f49013a748603a1b42d130f54226" target='_blank'>
              The Need for a Systems Biology Approach in Cancer Explained
              </a>
            </td>
          <td>
            Hehuan Zhu, Xi Zhang, E. Nazemalhosseini-Mojarad, J. Roelands, L. D. A. N. de Muynck, C. Ravensbergen, Rachel Hoorntje, I. Stouten, Marianne Hokland, A. L. Vahrmeijer, Rob A. E. M. Tollenaar, Edwin Koster, Peter J K Kuppen
          </td>
          <td>2025-12-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V. Semenov, Irina S. Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-08</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Despite the emergence of new therapies such as, immunotherapy, the treatment and diagnosis of Hepatocellular Carcinoma (HCC) still face many challenges, and the therapeutic outcomes for patients remain less than satisfactory. The study identified differentially expressed genes in HCC through differential analysis and then used univariate cox regression analysis to identify genes associated with prognosis. The intersection of these two sets of genes was used to obtain differentially expressed prognostic genes in HCC, which were then subjected to enrichment analysis. We analyzed two single-cell RNA sequencing (scRNA-seq) datasets from HCC patients, comprising 24,637 cells. Non-negative Matrix Factorization (NMF) clustering was used to identify cell cycle regulation in HCC tumor microenvironment (TME) cells, including three cell subpopulations: proliferating cells (PC), dendritic cells (DC), and macrophages (MAC). We employed the CellChat package to analyze cell–cell communication, the Monocle package for pseudotime trajectory analysis, and the SCENIC software package to study gene regulatory networks. Survival analysis was also performed using cell cycle-related features. A total of 26 clusters, including 15 major cell types, were identified in the HCC samples. Complex cell–cell communication networks were observed among these cell types. Enrichment analysis revealed that these cells were mainly enriched in pathways related to the cell cycle. The expression of cell cycle-related genes was elevated in tumor samples, and changes in cell cycle-related genes in specific subtypes were associated with different overall survival rates. The study focused on single-cell level data analysis of the cell cycle. The bubble plot results showed that the cell cycle scores were significantly upregulated in the PC, DC, and MAC subpopulations. Further subtyping revealed that these subtypes exhibited distinct biological states, cell–cell communication, and metabolic pathways. This study demonstrates that cell cycle regulation and cell–cell communication within the HCC tumor microenvironment impact tumor progression and patient prognosis. Cell cycle dysregulation in TME cells correlates with poor prognosis and immunotherapy efficacy, suggesting cell cycle targeting as a therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc3531834465d59ab5f37ade029201ae0f13d144" target='_blank'>
              Impact of single-cell cell cycle regulation of intercellular communication on the prognosis of hepatocellular carcinoma in the tumor microenvironment
              </a>
            </td>
          <td>
            Cong Hu, Rui Deng, Shuxiong Nong, Xinglang Mou
          </td>
          <td>2025-11-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This study addresses the limited mechanistic understanding behind medical imaging for tumor microenvironment (TME) assessment. We developed a novel framework that analyzes tumor imaging heterogeneity index (TIHI)-correlated genes to uncover underlying TME biology and therapeutic vulnerabilities. DCE-MRI and mRNA data from 987 high-risk breast cancer patients in the I-SPY2 trial, together with mRNA data from 508 patients in GSE25066, were analyzed. TIHI-associated genes were identified via Pearson correlation, clustered via weighted gene co-expression network analysis (WGCNA), and subgroups were defined via non-negative matrix factorization (NMF). The clinical relevance of the image-to-gene comprehensive (I2G-C) subtype defined by subgroups was assessed using logistic regression and Cox analysis. I2G-C comprised four clusters with distinct immune and replication/repair functions. It further stratified receptor, PAM50, and RPS5 subtypes. The “immune+/replication+” was more likely to achieve pathological complete response (pCR) (OR = 2.587, p < 0.001), while the “immune−/replication−” was the least likely to achieve pCR (OR = 0.402, p < 0.001). The “immune+/replication+” showed sensitivity to pembrolizumab (OR = 10.192, p < 0.001) and veliparib/carboplatin (OR = 5.184, p = 0.006), while “immune-/replication-” responded poorly to pembrolizumab (OR = 0.086, p < 0.001). Additionally, “immune+/replication-” had the best distant recurrence-free survival (DRFS), whereas “immune-/replication+” had the worst (log-rank p = 6 × 10−4, HR = 5.45). By linking imaging heterogeneity directly to molecular subtypes and therapeutic response, this framework provides a robust, non-invasive surrogate for genomic profiling and a strategic tool for personalized neoadjuvant therapy selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaac41c7947552aff84e1e324fe7769979665178" target='_blank'>
              Tumor Imaging Heterogeneity Index-Inspired Insights into the Unveiling Tumor Microenvironment of Breast Cancer
              </a>
            </td>
          <td>
            Qingpei Lai, Xinzhi Teng, Jiang Zhang, Xinyu Zhang, Yufeng Jiang, Yao Pu, Peixin Yu, Wen Li, Tian Li, J. Cai, Ge Ren
          </td>
          <td>2025-11-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Endometriosis involves ectopic growth of endometrial-like tissue, yet the spatial transcriptomic and metabolic landscape of ovarian endometriomas remains poorly understood. This investigation presents a comprehensive multi-omics analysis of ovarian endometriomas incorporating single-cell RNA sequencing in conjunction with Digital Spatial Profiler-Whole Transcriptome Atlas for spatial transcriptomics, and non-targeted Matrix-Assisted Laser Desorption/Ionization-Mass Spectrometry Imaging for spatially resolved metabolomics. Our integrated analysis confirms the importance of cell adhesion, ECM-receptor interaction, and focal adhesion pathways in disease context. We identify XBP1, VCAN, and CLDN7 as key markers in epithelial cells, and THBS1 in perivascular cells. Spatially resolved metabolomics further reveals altered activity of cytochrome P450 enzymes, lipoprotein particles, and cholesterol metabolism in mesenchymal regions, along with several undefined metabolites enriched in epithelial areas of endometriomas compared to ovarian cortex controls. These findings reveal potential markers and metabolic pathways linked to ovarian endometriomas, offering a foundation for future diagnostic and therapeutic strategies. Ovarian endometriomas, with distinct microenvironment and heightened hormonal sensitivity, are recognized as precursors of ovarian carcinomas. This study decodes ovarian endometriomas by integrating single-cell and spatial transcriptomics with spatial metabolomics to reveal key markers and altered pathways, offering new avenues for diagnosis and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b76d5485f1fbcc4c032549f15e895ea9152fd283" target='_blank'>
              Single-cell and spatially resolved omics reveal transcriptional and metabolic signatures of ovarian endometriomas
              </a>
            </td>
          <td>
            Yujuan Qi, Xia Chen, Sen Zheng, Tiantian Wu, Zhenbei Li, Jie Cheng, Xinhui Yang, Wei Tao, Qiuru Huang, Juan Gu, Qingqing Sun, Ning Chen, Xiangnan Cao, Jiaxin Li, Lei He, Chenyu Wang, Xinda Wang, Qingqing Hu, Qiushi Xia, Yi Zhang, Jiangming Reng, Weiyi Qian, Lingyi Kong, Yuqi Huang, Yanting Wang, Chen Qiao, Xinyuan Zhao, Ying Zheng, Mei Xu, Bo Zheng, Yijuan Cao, Jun Yu
          </td>
          <td>2025-11-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Testicular germ cell tumours (TGCTs) are amongst the most common malignancies in young men, and their incidence is increasing worldwide. Tissue heterogeneity hampers efforts to understand how TGCT precursors (termed germ cell neoplasia in situ; GCNIS) emerge and progress, restricting elucidation of new strategies for diagnosis and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69b2668308b974c3ecddec26d79d42dd71656fcf" target='_blank'>
              Spatial transcriptomics mapping of immune cell and TGFβ signalling pathway heterogeneity in testicular germ cell tumours
              </a>
            </td>
          <td>
            Sarah C. Moody, D. Fietz, Benedict Nathaniel, M. Frydenberg, Ben Tran, H. Schuppe, Kate L. Loveland
          </td>
          <td>2025-07-22</td>
          <td>Andrology</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="Computational pathology enables the automatic tissue analysis of Whole Slide Images (WSIs), offering unmatched possibilities to capture quantitative tumor microenvironment (TME) characteristics that are essential for patients' prognosis and therapy response. The existing clinical and digital biomarkers do not encompass the morphometric features and spatial interactions between vascular networks and immunological compartment in the TME. To address this challenge, this work presents a high- throughput quantitative framework for automatic segmentation and assessment of aberrant phenotypes of blood vessels and immune cell clusters in hematoxylin & eosin-stained WSIs, to construct the Vascular-Immune Pathomic (VIPath) biomarker. For our study, we utilized three public datasets of Colon Adenocarcinoma (COAD) and Stomach Adenocarcinoma (STAD) from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) projects: TCGA-COAD, TCGA-STAD, and CPTAC-COAD. Additionally, we collected two in-house gastric cancer cohorts of 80 and 51 patients : DBGC-M0 and DBGC-M1. The VIPath biomarker was incorporated in a Cox proportional hazards model trained on the TCGA-COAD. Then, it was validated for predicting Overall Survival (OS) in TCGA-STAD, DBGC-M0, and DBGC-M1, Disease Free Survival in CPTAC-COAD and second-line therapy Progression-free Survival (PFS-2) in DBGC-M1. VIPath encompasses features from both vascular and immunological compartments interacting in the TME. Results proved that VIPath was capable to significantly stratify risk groups for OS TCGA-STAD (p=0.018), OS DBGC-M0 (p=0.029), OS DBGC-M1 (p=0.014), and PFS-2 DBGC-M1 (p$< $0.005). Furthermore, when inserted in a Cox model, it led to an improvement of C-index and R2 over all other considered prognostic factors, i.e., p-TNM, ECOG, MSI, HER2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f526aa6f7bc4b1f2859a4decf8cee5c58634b7e" target='_blank'>
              Automated Pathomic Analysis of Angiogenesis and Immune Profiles Unveils an Interpretable Prognostic Biomarker in Colon and Gastric Cancers.
              </a>
            </td>
          <td>
            Michela Prunella, Nicola Altini, Rosalba D'Alessandro, Annalisa Schirizzi, Giampiero De Leonardis, Graziana Arborea, M. Savino, Anna Maria Valentini, R. Armentano, Angela Dalia Ricci, Claudio Lotesoriere, Raffaele Carli, M. Dotoli, Gianluigi Giannelli, Vitoantonio Bevilacqua
          </td>
          <td>2025-11-24</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Lung cancer is one of the most prevalent malignant tumors with high morbidity and mortality rates worldwide. Extensive multi-omics analyses have revealed significant intratumoral heterogeneity even within the same histopathological subtype. However, a database that systematically integrates multi-omics data for lung cancer research has long been lacking. Here, we developed LOSTdb, a molecular subtype annotation system for lung cancer that integrates multi-omics data and metadata. LOSTdb comprises 295 multi-omics datasets, including bulk RNA-seq, genomic, proteomic, methylation, and scRNA-seq data, with over 10,000 manually curated metadata entries. This resource encompasses high-quality clinical specimens, mouse models, and cell lines, totaling 34,393 samples and more than 1.2 million single cells. Each omics sample was annotated with both literature-based classical subtypes and NMF-derived meta-program (MP) subtypes. The platform supports cross-searching of omics and metadata at the gene and dataset levels, offers multiple visualization and analysis methods, and includes five tool modules, enabling functions such as integrated analysis, significance analysis between metadata as well as between genes and metadata, and target prediction for lung cancer molecular subtypes, serving as an essential tool for lung cancer precision medicine. LOSTdb is a user-friendly interactive database freely accessible at http://lostdbcancer.com:8080.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d313178158ab200d4da77fbc83c8812b7dd83e" target='_blank'>
              LOSTdb: a manually curated multi-omics database for lung cancer research
              </a>
            </td>
          <td>
            Hao Luo, Yunhao Yang, Zhipeng Gong, Lunxu Liu, Yaohui Chen
          </td>
          <td>2025-12-01</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cervical cancer is a prevalent malignancy among women, yet the involvement of neutrophils in its tumor microenvironment remains insufficiently explored. This study utilized single-cell RNA sequencing (scRNA-seq) to delineate neutrophil subsets and elucidate their roles in disease progression and prognosis. Analysis of 20 cervical biopsy samples across different disease stages identified five neutrophil subsets (N0-N4), among which the N4 subset exhibited a marked increase during disease advancement. Spatial transcriptomics and tissue microarray analyses revealed that N4 neutrophils are enriched in tumor regions and are associated with genes implicated in proliferation, metastasis, and immune evasion. Functional characterization demonstrated that N4 promotes tumor progression via activation of the Wnt signaling pathway and extracellular matrix remodeling. A neutrophil infiltration-based risk model was established and validated through multi-omics approaches, highlighting its potential in prognostic prediction. These findings underscore the pivotal role of N4 neutrophils in cervical cancer and provide valuable insights for the development of targeted immunotherapies and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9eca7bc06f3fa7d3102669c9c6d4eefb56b347" target='_blank'>
              Single-Cell and Multiomic Analysis Reveals Neutrophil Heterogeneity and Prognostic Value in Cervical Lesions.
              </a>
            </td>
          <td>
            Ze Wang, Zhuang Liang, Shiyi Liu, C. Cao, Shimin Chen, Nan Yu, Xiaoyuan Huang, Tao Zhang
          </td>
          <td>2025-12-12</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Urothelial carcinoma (UC), encompassing both bladder cancer and upper tract UC, is a biologically heterogeneous malignancy that has long complicated patient stratification and therapeutic prediction. Recent genomic and transcriptomic analyses, however, have delineated reproducible molecular subtypes across non-muscle-invasive and muscle-invasive disease. These classification frameworks have refined biological understanding and improved both risk stratification and treatment selection. Emerging evidence also indicates that genomic alterations, such as FGFR3 mutations, may confer distinct immunologic phenotypes within specific transcriptomic contexts, thereby challenging prior assumptions. The advent of immune checkpoint inhibitors has transformed UC management, yet variable response rates underscore the complexity of the tumor microenvironment (TME). Single-cell and spatial analyses have identified immune-inflamed, -desert, and -excluded TME subtypes, with cancer-associated fibroblasts playing a key role in immune exclusion and therapeutic resistance. Liquid biopsy, particularly circulating tumor DNA, has demonstrated utility for real-time disease monitoring, minimal residual disease detection, and treatment stratification, as highlighted in trials such as IMvigor010 and IMvigor011. Urinary extracellular vesicles also represent promising non-invasive biomarkers, though further standardization is required. In addition, novel therapeutic strategies-including antibody-drug conjugates, gene therapies, and sustained-release delivery systems-are broadening treatment options across disease stages. Collectively, these translational advances mark a paradigm shift toward precision immuno-oncology in UC. Nonetheless, successful clinical implementation will require prospective validation, optimized analytical platforms, and sustained physician-scientist engagement to translate these discoveries into improved patient outcomes. This review provides a comprehensive overview of recent progress in UC translational research, with a focus on genomic and transcriptomic insights, the evolving role of immunotherapy, advances in liquid biopsy, and the development of novel therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e44cd39c3a6847a9cce1aefa2b7bb98a79ab335f" target='_blank'>
              Translational Advances in Urothelial Carcinoma: From Bench to Bedside.
              </a>
            </td>
          <td>
            T. Iwasawa, T. Tsujino, S. Tsukahara, Hiroshi Fukushima, Daisuke Ito, E. Tomiyama, Fumihiko Urabe
          </td>
          <td>2026-01-01</td>
          <td>International journal of urology : official journal of the Japanese Urological Association</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Objective This study aimed to investigate cell-type-specific RNA methylation patterns in primary breast cancers and metastatic lymph nodes and explore their roles in cancer progression and immune regulation. Methods Using single-cell RNA sequencing data from five breast cancer patients (primary cancers and matched metastatic lymph nodes), we calculated RNA methylation scores per cell using the AUCell method. Cell types were annotated using Seurat. Differentially expressed gene analysis, pseudotime trajectory assessment, copy number variation analysis, and cell–cell communication analyses were performed between high- and low-methylation groups. Validation was conducted using bulk RNA sequencing data from The Cancer Genome Atlas. Results Metastatic lymph nodes showed significantly lower RNA methylation scores than primary cancers, particularly in T cells and B cells. Low methylation was correlated with enhanced cell differentiation, increased malignancy in epithelial cells, and reduced immune cell communication. Monocytes exhibited opposite trends. Differentially expressed genes from methylation groups stratified The Cancer Genome Atlas patients into two clusters with distinct survival and immune infiltration profiles. Conclusions RNA methylation is suppressed during breast cancer metastasis, affecting cell differentiation, immune communication, and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3a57d08895a4d132dadd9681e650758bc323c10" target='_blank'>
              Single-cell analysis reveal cell-type-specific RNA methylation patterns in breast cancer
              </a>
            </td>
          <td>
            Shengtao Huang, Yichen Lin, Xiaobin Liu, Kun Zhang, Qiuju Han, Ruilei Liu, Yong Zhuang
          </td>
          <td>2025-12-01</td>
          <td>The Journal of International Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The incidence of prostate cancer continues to increase, making it the second most common malignant tumor among men worldwide. Immunotherapy has emerged as a key therapeutic strategy for treating tumors. Numerous studies have established that the efficacy of tumor immunotherapy is closely associated with the tumor microenvironment and T cell subsets. However, the specific functions of certain T cell subsets in prostate cancer remain incompletely characterized. Therefore, this study aimed to systematically investigate the distribution patterns of T cell subsets within the tumor microenvironment of prostate cancer patients and their correlations with clinicopathological parameters. Therefore, we investigated the impact of T cells on the tumor microenvironment of prostate cancer at the single-cell level. We employed a variety of analytical methods to reveal the functions of T cells, including cell interaction analysis, time-series analysis, enrichment analysis, immune infiltration analysis, and other analytical approaches. By integrating bulk RNA-seq data, we constructed and validated a prognostic risk model based on T cell marker genes. Finally, we utilized the ssGSEA and ESTIMATE algorithms to explore the relationship between the prognostic risk model and immunotherapy. After quality control, 16,999 cells from the single-cell data were retained for downstream analysis. Our study focused on T cells, revealing the communication between various cell types and T cells. Pseudotime analysis showed that different T cell marker genes exhibited differential expression at various time points, corresponding to distinct biological processes. Enrichment analysis indicated that T cell marker genes were enriched in several immune-related pathways. From our analysis, BCAS2, EIF2S2, RIOK3, and ATP6V1E1 were ultimately identified as prognostic markers. Immune infiltration analysis revealed that high-risk patients had lower immune scores, stromal scores, and ESTIMATE scores and greater tumor purity compared to low-risk patients. We analyzed the mechanisms involving T cells in prostate cancer from multiple perspectives, constructed a prognostic model, and conducted immune infiltration analysis. Our findings contribute to the understanding of prostate cancer and its prognosis, providing valuable insights for future research and prognostic assessments in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94839f7e60978966361da4e926a8390bd98bb805" target='_blank'>
              Exploring the key functions of T cells and the regulation of the immune microenvironment in prostate cancer using single-cell RNA sequencing and bulk RNA sequencing.
              </a>
            </td>
          <td>
            Zhidu Wang, Yan Xing, Dongmei Shang, Xuefei Jin
          </td>
          <td>2025-12-31</td>
          <td>Autoimmunity</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Lung cancer symptoms typically do not appear until advanced stages, leading to late diagnoses. This delay is a major contributor to its poor prognosis, resulting in lung cancer being the leading cause of all cancer-related deaths worldwide. Essential biological understanding of how lung cancers arise and progress is still lacking. Studying premalignant lesions that eventually develop into invasive carcinomas helps to bridge this knowledge gap. We are focusing on adenocarcinomas (ACs), the most common subtype of lung cancer, and especially, activating mutations in the Kirsten rat sarcoma virus oncogene (KRAS), which is among the most common oncogenic mutations and has previously been reported in premalignant lung lesions. Given that KRAS signaling is growth-promoting, it is reasonable to assume that KRAS mut promotes tumor formation and progression. However, the extent of its influence, the timing of when it arises, and particularly its connection to histologic and phenotypic changes driving tumor evolution, remain unclear.



 We have collected a cohort from biopsy material, initially collected for diagnostic purposes from AC patients at varying stages of tumor evolution, with some matched ‘normal-histology’ samples when available. In total, tissue from 38 patients with AC premalignancies is available for investigation. To map KRAS mutations in situ, we employed a BaseScope assay with commercial detection probes against the different KRAS mut transcripts. To map KRAS statuses to cellular phenotypes in the microenvironment, we are running the 6K discovery panel on the CosMx platform (Bruker) on a subset of patients, which is a highly sensitive spatial transcriptomics technique at single-cell resolution. Spatial Mass Spectrometry (MS) tracking metabolic features on the timsTOF flex MALDI-2 (Bruker) is also being performed on a subset of patients.



 Initial tests of the BaseScope assay have revealed specific, yet not sensitive, signals of KRAS mut transcripts in premalignant lesions. While advanced cancers display distinct signals, premalignant lesions have fewer, if any, signals. This might be due to the lower expression levels of KRAS at the earlier stages. To address this, we are now applying a BaseScope assay with signal amplification to be able to track KRAS expressed at lower levels. Preliminary analyses of the Spatial MS matrix have revealed highly individual metabolomics features, with great inter-patient heterogeneity. This pattern is also reflected in bulk RNA-seq data, revealing expression profiles more similar across, e.g., Adenocarcinoma in Situ (AiS) and invasive AC from the same patient, than between the AiSs from different patients.



 By applying this state-of-the-art multi-omics approach to premalignant adenocarcinomas, we hope to identify key regulators of early lung cancer progression and to translate these findings to clinically relevant basic molecular diagnostics.



 Amanda Lindberg, Bartosz Sobocki, Patrick Micke, Carina Strell. In situ KRAS-mapping and spatial-omics characterization in lung premalignancy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A002.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23b558a4354aae9f9807afda6b85aec2e424e2cc" target='_blank'>
              Abstract A002:
 In situ
 KRAS-mapping and spatial-omics characterization in lung premalignancy
              </a>
            </td>
          <td>
            A. Lindberg, B. Sobocki, P. Micke, C. Strell
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Lymph nodes, brain, bone, and liver are recognized as the four most common metastatic sites for lung adenocarcinoma (LUAD). Metastasis to these locations exhibits some common features, such as immune suppression, and distinct tumor microenvironment (TME) heterogeneity involving differentiation of immune cells, impacting treatment efficacy and prognosis. Lymph node metastases are characterized by immune suppression with exhausted CD8+ T cells, expanded regulated T cell (Tregs), M2-polarized macrophages, and high programmed death ligand-1 (PD-L1) expression. Brain metastases display an “immune desert” phenotype due to blood–brain barrier constraints, reduced T-cell infiltration, and microglia-mediated immunosuppression. Bone metastases involve osteoclast activation, RANKL/OPG pathway dysregulation, and metabolic reprogramming, while liver metastases show Kupffer cell-driven PD-L1/ programmed death 1(PD-1) axis suppression and elevated Treg infiltration. Key biomarkers across all types of metastases include PD-L1, cytokine profiles, immune cell ratios, and metabolic markers. Therapeutic strategies focus on combination therapies such as immune checkpoint inhibitors (ICIs) with metabolic modulators, localized drug delivery, and biomarker-guided approaches. Challenges in this field encompass spatial heterogeneity, dynamic TME evolution, and clinical translation barriers. Future research directions highlight spatial transcriptomics, microbiome interactions, and organoid models to optimize personalized immunotherapy. This article aims to provide a comprehensive review of regarding TME alterations across these four main metastatic locations of LUAD. It will also discuss relevant TME biomarkers and their clinical significance on therapeutic response and prognosis. We expect this article to serve as a source of evidence and inspiration for the future development of treatment strategies based on LUAD TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eac0fc7ba7c67303e9257f2a2d91aba7b28ddab" target='_blank'>
              Unraveling the alterations and biomarkers in the tumor microenvironment in lung adenocarcinoma metastases and their indications for therapeutic response and prognosis
              </a>
            </td>
          <td>
            Chao Guan, Xiangmin Li, Xiaoshan Zeng, Zhenguang Du, Zhicheng Zhou, Jungang Zhao, Lele Song, Li Yu
          </td>
          <td>2025-12-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAM) exert both pro- and anti-tumoral functions that influence cancer progression and patient prognosis. However, single-cell RNA sequencing (scRNA-seq) studies have revealed that TAM heterogeneity remains incompletely characterized. By performing an unbiased, integrated in silico analyses of publicly available scRNA-seq datasets, comprising samples from blood, tumor, and non-tumoral mammary tissue (NTMT) from both breast cancer (BRCA) patients and healthy individuals, we identified seven transcriptional signatures corresponding to distinct TAM subsets, exhibiting unique functional profiles, including heightened interferon responses, scavenging, and matrix remodeling, the latter two being characteristic of tissue repair. Notably, none of these subsets aligned with the M1/M2 classification of macrophage (Mφ) polarization. Interferon-associated genes were predominantly enriched in blood monocytes, whereas tissue-repair-associated-signatures were more abundant in tissue resident Mφ, suggesting that TAMs bearing these signatures resemble monocyte-derived or tissue-resident Mφs, respectively. Importantly, TAM subsets expressing interferon-associated genes were associated with improved survival compared to tissue-repairing TAMs in a BRCA cohort from The Cancer Genome Atlas (TCGA). Additionally, one signature was heightened in peripheral monocytes from BRCA patients compared with healthy individuals, which was experimentally validated in a pilot study of Mexican BRCA patients. We concluded that these signatures are a closer description of TAM heterogeneity in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769e9d0c6f0718581121627c404c58103ae137e7" target='_blank'>
              Single cell analysis identifies inflammatory and tissue remodeling tumor associated macrophages distinct from M1/M2 paradigm.
              </a>
            </td>
          <td>
            Eric López-Huerta, Diana P Portales-Pérez, M. E. Martínez-Leija, Vianney Ortiz-Navarrete, E. Fuentes-Pananá
          </td>
          <td>2025-12-05</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Simple Summary Breast cancer is a highly heterogeneous disease, and about 15–30% of cases are driven by the HER2 receptor, a protein that can be specifically targeted by modern therapies. Traditionally, cancer cells are grown in flat layers on plastic, but this does not reproduce the complexity of a real tumor. Three-dimensional (3D) culture systems, where cells grow as spherical clusters called “spheroids,” provide a more realistic environment that mimics how cells interact in the body. In this study, we generated 3D spheroids from two commonly used HER2-positive breast cancer cell lines, SKBR3 and BT474. Although both cell types formed viable spheroids, their shapes and internal organization were very different: SKBR3 spheroids were loose and irregular, while BT474 spheroids were compact and highly spherical. We examined how HER2 was distributed within these structures and found that its signal decreased toward the spheroid core, especially in BT474. We also analyzed signaling proteins and cellular structures, discovering differences in cell–cell adhesion, mitochondrial features, and key molecular pathways. Overall, our results show that the 3D architecture of breast cancer spheroids strongly influences how cells organize, signal, and maintain HER2 expression. This system provides a powerful tool to study drug responses and to improve strategies for personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d155c78e81cc0e3e7e461d59374a103c979710df" target='_blank'>
              3D Breast Cancer Spheroids Reveal Architecture-Dependent HER2 Expression and Signaling
              </a>
            </td>
          <td>
            Pietro Arnaldi, Valentina Delli Zotti, G. Bellese, M. Gagliani, Paola Orecchia, P. Castagnola, Katia Cortese
          </td>
          <td>2025-11-24</td>
          <td>Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Tissue-resident memory T (TRM) cells, memory T cells that stably occupy tissues and contribute to immunosurveillance, induce favorable survival outcomes in solid tumors. While TRM cells have been observed in lymph nodes, their phenotype and prognostic significance in diffuse large B-cell lymphoma (DLBCL) remains uncharacterized. In this study, CD103+ T cells were quantified in DLBCL samples by immunofluorescence (IF) staining of tissue biopsies and flow cytometry of cell disaggregates, and linked with clinical outcomes. Across two patient cohorts, CD103+ T cells were identified in both nodal and extranodal DLBCL, and higher CD103+ T cell levels correlated with superior clinical outcomes. Single-cell RNA sequencing (scRNAseq) revealed ITGAE expressing T cells in both malignant and reactive lymph node (rLN) samples. However, transcriptional profiles differed, as a canonical TRM population was observed in the malignant setting. This TRM cluster was enriched for genes associated with cytotoxicity and activation and was validated in an external CITEseq dataset. Flow cytometry additionally confirmed protein expression of TRM markers (CXCR6, CD39, and PD-1) on CD69+CD103+ T cells. We assessed functional activity in co-culture experiments of CD103+ versus CD103- T cells with autologous CD20+ B cells, where CD103+ T cells displayed enhanced killing. CD103+ TRM cells in DLBCL represent a prognostically favorable population with an activated/cytotoxic T cell phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75e4cde340cefbbda9de07c5cdbbe7ec6ec4213" target='_blank'>
              Phenotypic characterization and prognostic impact of CD103+ tissue-resident memory T cells in diffuse large B cell lymphoma.
              </a>
            </td>
          <td>
            Gillian Savage, K. Milne, B. Collinge, Yifan Yin, Pauline Loos, Mary E Warren, Céline M. Laumont, Danielle Keith, Brad H Nelson, Leandro Venturutti, Andrew P. Weng, D. Scott, Christian Steidl, L. Evgin
          </td>
          <td>2025-11-21</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Breast cancer is the most common cancer among women and a leading contributor to disease-related mortality. While advancements in diagnostic techniques and the widespread dissemination of medical knowledge have improved outcomes, immunotherapy targeting the tumor microenvironment offers a promising approach to further reducing the disease burden. Methods We first explored cellular heterogeneity within breast cancer tumors using single-cell RNA sequencing and subsequently analyzed the association between the infiltration levels of various cell types and survival prognosis using a deconvolution algorithm combined with bulk RNA sequencing. Our analysis identified CXCL9 + macrophages as being associated with improved survival outcomes. Next, WGCNA was used to identify genes related to CXCL9 + macrophage infiltration, and a survival risk model was constructed based on these genes. Finally, experiments were conducted to validate the effects of CXCL9 + macrophages on breast cancer cells. Results We found that CXCL9 + macrophage primarily interacted with cytokines and activated T and NK cells through the CXCL9/10/11-CXCR3 axis. Increased infiltration, as observed in bulk RNA sequencing, was associated with improved patient survival. The risk model constructed based on CXCL9 + macrophage infiltration-related genes demonstrated strong efficacy in predicting patient survival outcomes. Finally, in vitro experiments confirmed that CXCL9 + macrophages inhibited the viability of breast cancer cells. Conclusion CXCL9 + macrophages could play a significant role in inhibiting breast cancer, and their infiltration into tumor tissues is associated with improved survival in breast cancer patients. Immunotherapy targeting CXCL9 + macrophages hold great potential as a therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c633a0c3546d68c439ee716b36ed43ccdccd0f56" target='_blank'>
              Infiltration of CXCL9+ macrophages confers a favorable prognosis in breast cancer: Insights from an integrated single-cell RNA and bulk RNA sequencing study
              </a>
            </td>
          <td>
            Bin Liang, Zhicai Duan, Siyu Long, Peng Zhou
          </td>
          <td>2025-12-01</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/457e45e428bfba1eae8a9669bae5386f0ae9da25" target='_blank'>
              Integrated single-cell multi-omics profiling reveals a senescence-associated hematopoietic landscape and regulatory network in aging bone marrow
              </a>
            </td>
          <td>
            Guoyang Zhang, Weixuan Lin, Linghuan Guo, Ziyun Li, Yu Xiang, Luo Wang, Xiaoyu Yan
          </td>
          <td>2025-11-24</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c362676e5f313e967da9a02b557ed5fbcda507b0" target='_blank'>
              Single-cell and spatial transcriptome-based metabolism-immunity interaction network and therapeutic target discovery of matrine in cervical cancer.
              </a>
            </td>
          <td>
            Hong Zou, Wang Xiong, Yumei He
          </td>
          <td>2025-11-24</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with a poor prognosis and limited response to current therapies. Cancer stem cells (CSCs) contribute to this heterogeneity, driving tumor progression, immune evasion, and therapeutic resistance. Epigenetic regulators have emerged as pivotal modulators of CSC phenotypes. The histone methyltransferase SETDB1, a key stemness factor in both embryonic and adult stem cells, catalyzes H3K9 trimethylation (H3K9me3) and acts as a major oncogenic driver in a wide range of solid and hematological malignancies. Yet, its specific role in maintaining stemness and contributing to therapeutic resistance in HCC remains poorly defined. Integrative analyses of bulk and single-cell transcriptomic datasets from public HCC cohorts were performed. Tumors were stratified by SETDB1 expression, followed by pathway enrichment, immune deconvolution, and stemness scoring. Associations with clinical outcomes, immune phenotypes, drug resistance signatures, and chromatin-binding partners (DNMT3A, TRIM28, HDAC1) were also explored. The role of SETDB1 was then examined in vitro, where a stem-like phenotype was induced in HCC cell lines under hypoxic conditions, and patterns of SETDB1 expression and stemness marker levels were assessed following paclitaxel treatment. SETDB1-low tumors were enriched in well-differentiated, immunologically active subclasses and were associated with increased predicted responsiveness to immune checkpoint blockade (ICB), sorafenib, and transarterial chemoembolization. Conversely, SETDB1-high tumors exhibited transcriptional features of dedifferentiation, elevated stemness scores, and enrichment in aggressive molecular subtypes. These tumors also displayed higher TP53 mutation rates, reduced immune infiltration, immune exclusion signatures, and activation of pathways linked to ferroptosis and ABC transporter dysfunction, consistent with resistance to ICB and conventional therapies. SETDB1 expression was also associated with hypoxia-related pathways, suggesting a role in maintaining CSC niches. In vitro, hypoxia-induced stemness coincided with increased SETDB1 levels, whereas paclitaxel treatment decreased both SETDB1 expression and stemness markers. A chromatin risk score incorporating SETDB1 and its partners predicted poor disease-free survival independently of clinical parameters. SETDB1 defines a stemness-enriched, immune-resistant HCC subtype associated with poor outcomes and therapeutic failure. Correlations between SETDB1 expression and stem-like features in vitro suggest a potential role in maintaining CSC phenotypes, supporting its relevance as a biomarker and candidate for epigenetic- and CSC-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c393716a837e5b1ef0091b6385eb92c64be4ab4" target='_blank'>
              The epigenetic regulator SETDB1 as a key component of cancer stem cells and drug resistance in primary liver cancer
              </a>
            </td>
          <td>
            Maël Padelli, Christophe Desterke, Aurore Devocelle, Georges Uzan, Antoinette Lemoine, Julien Giron-Michel
          </td>
          <td>2026-01-06</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Why and how does cancer start? Building from a Symposium at the 2025 Society of Toxicology meeting, we convened a group of international experts to answer this seemingly simple question. As experimental evidence has evolved, perspectives on cancers' origins have shifted from the accumulation of DNA mutations in single cells to complex processes involving signals from an altered tissue microenvironment which promote tumorigenesis. Carcinogen exposures impact the biology of the microenvironment in complex and tissue-specific ways. These changes can include the infiltration of inflammatory cells that produce growth factors, neo-angiogenesis, morphological changes, and immune tolerance that avoids immune-mediated elimination. In this in-depth review, we discuss the evidence linking chemical-driven microenvironmental changes in the development of a range of solid and liquid tumors. We discuss specific phenotypic alterations, such as selection pressure driving clonal expansion and cellular plasticity and reacquisition of stem cell states, linked to carcinogen-induced changes in the microenvironment. We describe assays and biomarkers which can allow us to experimentally assess links between chemical exposures, the microenvironment, and cancer phenotypes. We end by discussing how understanding the role of the microenvironment and malignancy in toxicology is essential for accurate cancer hazard evaluation, development of next-generation risk assessment frameworks, identifying new strategies for cancer prevention, and improving patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c64691a327a7b8227d277fcb6bbfe9d59f21fba7" target='_blank'>
              Rethinking the Microenvironment's Role in Chemical-Induced Malignancy.
              </a>
            </td>
          <td>
            William H. Bisson, Richard A Currie, Emilia L Lim, Coraline Mlynarczyk, Erik J Tokar, Annamaria Colacci, Justin A. Colacino
          </td>
          <td>2025-12-17</td>
          <td>Toxicological sciences : an official journal of the Society of Toxicology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Angioimmunoblastic T-cell lymphoma (AITL) is a subtype of peripheral T-cell lymphoma originating from follicular helper T cells and characterized by profound clinical, pathological, and molecular heterogeneity. Despite advances in classification and diagnosis, therapeutic outcomes remain suboptimal, particularly in relapsed or refractory settings. In recent years, the evolution of risk stratification models and a deeper understanding of AITL’s molecular pathogenesis, including its unique pattern of clonal evolution, have paved the way for precision medicine. Notably, targeted treatment approaches, such as immunomodulatory agents, epigenetic therapies, kinase inhibitors, and immune checkpoint blockade, are demonstrating promising clinical efficacy, especially in patients harboring specific molecular aberrations. This review comprehensively summarizes the multilayered heterogeneity of AITL, emphasizes the role of immune and molecular profiling in informing therapeutic decisions, and outlines future directions for personalized, multi-agent treatment strategies aimed at overcoming resistance and improving survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1f48636814c0a428af30d1bb9e6df8d06e9d0f" target='_blank'>
              Unraveling the many faces of angioimmunoblastic T-cell lymphoma: clinical, pathological, and molecular heterogeneity
              </a>
            </td>
          <td>
            Jia-Ping Chen, Peng Ke, Xiao Qiu, Qiu-Yuan Xian, Jing-Song Wu, Chun Feng, Hui-Jun Li, Yang Wang, Ji-Hao Zhou
          </td>
          <td>2026-01-01</td>
          <td>Annals of Hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia is a genetically and cellularly heterogeneous disease. We characterize 120 AMLs using genomic and transcriptomic analyses, including single-cell RNA sequencing. Our results reveal an extensive cellular heterogeneity that distorts the bulk transcriptomic profiles. Selective examination of the transcriptional signatures of >90,000 immature AML cells identifies four main clusters, thereby extending current genomic classification of AML. Notably, NPM1-mutated AML can be stratified into two clinically relevant classes, with NPM1class I associated with downregulation of MHC class II and excellent survival following hematopoietic stem cell transplantation. NPM1class II is instead associated with resistance to allogeneic T cells in an ex vivo co-culture assay, and importantly, dismal survival following hematopoietic stem cell transplantation. These findings provide insights into the cellular state space of AML, define diagnostic entities, and highlight potential therapeutic intervention points. The clinical outcomes and treatment responses of acute myeloid leukemia (AML) patients are heterogeneous. Here, the authors use bulk and single-cell sequencing approaches and identify two transcriptomic subtypes within NPM1-mutated AML with distinct immune evasion properties and responses to hematopoietic stem cell transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0674b44d7df57e733fca27fc81a60b0eeaf92970" target='_blank'>
              The AML cellular state space unveils NPM1 immune evasion subtypes with distinct clinical outcomes
              </a>
            </td>
          <td>
            H. Lilljebjörn, Pablo Peña-Martínez, Hanna Thorsson, Rasmus Henningsson, M. Rissler, N. Landberg, Noelia Puente-Moncada, Sofia von Palffy, Vendela Rissler, Petr Stanek, Jonathan Desponds, Xiangfu Zhong, Gunnar Juliusson, Vladimir Lazarevic, Sören Lehmann, Magnus Fontes, H. Ågerstam, Carl Sandén, Christina Orsmark-Pietras, T. Fioretos
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Adamantinomatous craniopharyngioma (ACP), a benign yet highly recurrent and therapy-resistant intracranial tumor, remains a considerable clinical challenge because of its complex pathological structure, infiltrative growth, and limited treatment options. Here, integrated spatially resolved multiomics is employed-including single-cell spatial transcriptomics via CosMx SMI and spatially resolved metabolomics via AFADESI-MSI, accompanied by bulk metabolomics and functional validation-to unravel the driving factors of ACP progression and recurrence. Analysis results reveal three interdependent biological hallmarks: first, the spatial segregation and molecular heterogeneity of 10 distinct tumor epithelial cell subpopulations within the ACP, each of which presents unique transcriptional signatures; second, in tumor regions and recurrent tumor epithelium tissues, stronger transporter-mediated choline/ethanolamine uptake from cystic fluid and significant upregulation of phosphatidylcholine (PC) and phosphatidylethanolamine (PE) synthesis is observed, creating the enhanced "cystic fluid-tumor cell" and "choline/ethanolamine-PC/PE" metabolic axis, and demonstrating the spatial metabolic remodeling of ACP; and third, this metabolic axis directly couples to autophagy activation of corresponding regions in ACP tissue, which is validated by multi-immunohistochemistry for Beclin1 and GABARAP. Together, these findings reveal metabolic remodeling and autophagic activation as critical drivers of ACP progression and recurrence and provide an opportunity for precise biomarker-driven treatment of this intractable tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4bf376edd94fbdf6f86d786c026d6877b062546" target='_blank'>
              Spatially Resolved Multiomics Reveals Metabolic Remodeling and Autophagy Activation in Adamantinomatous Craniopharyngiomas.
              </a>
            </td>
          <td>
            Dongting Chen, Yahui Gao, Yulin Wang, Ting Lei, Zheng Qu, Yuhan An, Jiaxu Fu, Xin Li, Fangjun Liu, Yan Li
          </td>
          <td>2026-01-05</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gene-specific expansion patterns were evident among the most frequent CH lesions, with DNMT3A-mutant clones exhibiting impaired hematopoietic differentiation and TET2-mutant clones showing multi-lineage propagation. Notably, identical CH clones were detected in 41% of corresponding lymphomas, displaying distinct clonal dynamics: tumor-promoting CH (expansion in B-NHL; 10/16 clones; mainly TP53) and tumor-infiltrating CH (no expansion; mainly DNMT3A). Moreover, we identified lymphoma-associated mutations in flow-sorted hematopoietic progenitors from patients with indolent but not aggressive B-NHL and observed a stepwise accumulation of mutations along the lymphoid differentiation path. Single-cell genotyping confirmed the presence of mutated progenitors in 3 follicular, 2 mantle cell and 2 marginal zone lymphoma patients, providing direct evidence of a pre-neoplastic state in disease pathogenesis. Our findings offer novel insight into the cellular origin of nodal B-NHLs and highlight a previously underappreciated role for early clonal events involving the stem/progenitor cell compartment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ec5b65b719f8d9e5db3ee2069637056c360c3b" target='_blank'>
              Clonal Hematopoiesis and Lymphoma-Associated Mutations in Hematopoietic Progenitors in B-Cell Non-Hodgkin Lymphoma.
              </a>
            </td>
          <td>
            Laura Wiegand, Patricia Silva, D. Noerenberg, F. Christen, Klara Kopp, B. Locher, P. Löwe, Marlon Tilgner, Robert Altwasser, Vanessa Storzer, Catarina M Stein, F. Briest, C. M. Arends, Mareike Frick, J. Ihlow, A. Dolnik, N. Ishaque, U. Keller, Il-Kang Na, L. Penter, Lars Bullinger, Raphael Hablesreiter, Frederik Damm
          </td>
          <td>2026-01-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) with highly expressed secreted phosphoprotein 1 (SPP1) carry immunosuppressive property as a potential target for tumor metastasis. However, the mechanisms regulating SPP1 + TAMs in head and neck squamous cell carcinoma (HNSCC) remain poorly understood. This study employs a combination of single-cell and bulk RNA sequencing bioinformatics analysis to confirm the impact of TAMs with high levels of SPP1 on patient prognosis. Additionally, Key genes linked to SPP1 + macrophages were identified using weighted gene co-expression network analysis. A prognostic model was built using the Random Forest algorithm. Here we show, P4HA1 is strongly correlated with SPP1 + macrophages and holds significant value in predicting patient prognosis and diagnosis. In vitro experiments demonstrated that TAMs educated by HNSCC cells with knockdown P4HA1 expressed lower SPP1 level compared to the control group. Furthermore, Gene Set Variation Analysis and Gene Set Enrichment Analysis indicated that P4HA1 mediates the hypoxia pathway in HNSCC. In xenografts model, P4HA1 knockdown effectively suppressed tumor malignant progress, confirming that P4HA1 was positive correlation with SPP1 + TAMs and could mediate tumor hypoxia pathways. Overall, this study identified P4HA1 as a key gene involved in regulating SPP1 + TAMs through modulating hypoxia, providing a potential macrophage-centered therapeutic target in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a63e2de7b9c3162e04da7d6e387827ec9ac28c" target='_blank'>
              P4HA1 facilitates SPP1+ tumor-associated macrophages by activating the hypoxia pathway in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Yiding Liu, Bing Yan, Hui Dong, Yi Li, Yuying Zhang, Yi Wang, Ni Kou, Lu Gao
          </td>
          <td>2025-12-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Hepatocellular carcinoma (HCC) is one of the leading causes of cancer-related deaths worldwide, and many patients do not respond well to immunotherapy. In this study, we identified a unique group of macrophages—called NFKBIZ+ M0 macrophages—that are enriched in patients non-responsive to anti-PD-1 treatment. These macrophages are activated by hypoxia and release factors such as VEGFA and HBEGF that promote new blood vessel formation and tumor growth. They also produce inflammatory molecules that suppress the immune system, helping the tumor evade immune attack. Further analysis revealed that specific signaling pathways (FOSB–VEGFA and FOS–HBEGF) drive this harmful macrophage behavior. Our findings uncover a new mechanism linking hypoxia, angiogenesis, and immune evasion to treatment resistance in HCC and suggest potential therapeutic targets to improve immunotherapy outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce50b60fe1db2dee500e6f910be7f3a05fceb0c3" target='_blank'>
              Single-Cell Transcriptomic Profiling Reveals That Macrophage-Induced Angiogenesis Contributes to Immunotherapy Resistance in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Xinyu Pan, Baolin Liao, Zhijie Hu, Yuanyan Xiong
          </td>
          <td>2026-01-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND: The spatial organization of immune cells within tumors is an important area of investigation in studies of interactions between tumor cells and the tumor microenvironment. This is particularly relevant because some immune cells function through direct contact with their target cells, whereas others communicate over a distance via paracrine signaling involving cytokines. Thus, the topography of tumor cells and microenvironmental cells may determine the possibility and nature of intercellular interactions and thereby influence the functional effects of immune cells. AIM: This study aimed to compare the spatial transcriptomic profiles of tumor and stromal regions enriched in macrophages in triple-negative breast cancer. METHODS: Eight patients with triple-negative breast cancer were included. Spatial transcriptomic analysis was performed on formalin-fixed, paraffin-embedded tissue sections using high-throughput RNA sequencing with the 10x Visium platform. The annotated spots enriched in intraepithelial and stromal macrophages were used for downstream bioinformatic analysis. RESULTS: A total of 437 differentially expressed genes were identified between the two groups of spots containing macrophages with distinct spatial localization. Spots with intraepithelial macrophages were characterized by activation of processes related to cytokine and chemokine signaling, regulation of regulatory T-cell differentiation, organization of cell – cell contacts, wound healing, and inhibition of viral activity. Spots enriched in stromal macrophages demonstrated activation of biological processes associated with the regulation of angiogenesis, cell migration and recruitment, cell adhesion, and stromal remodeling. CONCLUSION: Macrophage topography within primary tumors of triple-negative breast cancer is associated with their functional characteristics. These fundamental findings may be useful for developing prognostic criteria and therapeutic approaches aimed at modulating the tumor microenvironment to improve long-term outcomes in patients with triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1996d432009ab729b97a00ec19d18afda8c70949" target='_blank'>
              Spatial Characterization of Macrophage-Enriched Tumor Regions in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            I. Patskan, A. Kalinchuk, Elisaveta Tsarenkova, Evgeniia Grigoryeva, I. Larionova, Natalia Popova, L. Tashireva
          </td>
          <td>2025-12-02</td>
          <td>Genes &amp; Cells</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80783a5ab29ea43b8d392154d14bbcb3414ea2c" target='_blank'>
              A Syngeneic Immunocompetent Mouse Model of Gallbladder Cancer Reveals Tumorigenesis and Therapeutic Response
              </a>
            </td>
          <td>
            Wenqing Qiu, Xiaojian Ni, Fuqing Xiang, Weida Meng, Rongyi Qin, Jiakang Ma, Dandan Zhang, Bosen Li, Shijie Xu, Changcheng Wang, Lingxi Nan, Xiaobo Bo, Yun Liu, Li Wang, Yueqi Wang, Houbao Liu
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Summary SISTEM is a software package and mathematical framework for simulating tumor evolution and cell migrations at single-cell resolution. Unlike existing frameworks which simulate cancer cell populations under the neutral coalescent or using simple birth–death models, SISTEM simulates tumor populations under somatic clonal selection using an agent-based framework. SISTEM can generate mutation profiles, read counts, and DNA sequencing reads along with ground truth cell lineages and migration graphs under a number of easily customizable mutation and selection models. For improved realism, SISTEM allows for cell fitness to be driven by genomic events of various scales including single nucleotide variants, segmental gains and losses, whole-chromosomal and chromosome-arm aberrations, and whole-genome duplications. SISTEM also includes numerous migration models to simulate metastatic cancers, facilitating the exploration and evaluation of diverse migration patterns. Availability and implementation SISTEM is written in Python and is freely available open-source under GNU GPLv3 from: https://github.com/samsonweiner/sistem">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f6ea45c6bd4d361ab9e67581ccc6b02aecffc37" target='_blank'>
              SISTEM: simulation of tumor evolution, metastasis, and DNA-seq data under genotype-driven selection
              </a>
            </td>
          <td>
            Samson Weiner, M. Bansal
          </td>
          <td>2025-11-23</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common primary liver carcinoma with high lethality. Both of hepatitis B virus (HBV) and Clonorchis sinensis (C. sinensis) are critical infectious contributors to HCC development. However, the inter-tumor heterogeneity and tumor microenvironment (TME) of HCC patients with different infectious background remain largely unknown. We compiled a cohort of 269 primary HCC patients to assess the clinical impact of C. sinensis and HBV infections on patient prognosis. Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomic (ST-seq) analyses were performed on tumor and adjacent normal samples from C. sinensis-associated HCC (CP), and double-infection HCC (DP) patients. Additionally, we integrated publicly available scRNA-seq and ST-seq datasets from HBV-associated (HP) patients. Immunofluorescence, immunohistochemistry and in vitro experiments were conducted to validate inter-tumor heterogeneity among the three HCC subtypes. C. sinensis infection is significantly associated with poorer prognosis in HCC patients. Multi-omics analyses revealed distinct inter-tumor heterogeneity in epithelial, immune, and stromal compartments across different HCC subtypes. Tumor cells in the DP group exhibited more malignant marker expression, higher copy number variation scores, increased activation of p53 pathway, and worse survival outcomes. Compared with other HCC subtypes, the TME in DP samples was enriched with SPP1+ macrophages, exhausted CD8+ T cells and COL1A1+ fibroblasts. In contrast, the CP and HP groups showed higher proportions of M2-like macrophages and ENPP2+ liver vascular endothelial cells, respectively. These findings decipher the cellular signatures and their interactions within the TME, shedding light on the inter-tumoral heterogeneity driven by different infections, and the development of targeted therapies for infectious HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce3fb4e883deefe93c32ac0665db0506da077e80" target='_blank'>
              Integrating single cell- and spatial- resolved transcriptomics unravels the inter-tumor heterogeneity and immunosuppressive landscape in HBV- and Clonorchis sinensis-associated hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jiayun Chen, Wenmin Lu, Yanni Lou, Jing Liu, Xiwen Liao, Yunmeng Bai, Guangqing Cheng, Guangzhi Zhu, Ji Feng, Junqi Liu, Zhaoji Liu, Liqun Jia, Jing Zhou, Tao Peng, Guo-Dong Lu, Jigang Wang
          </td>
          <td>2026-01-05</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5632b562f5d8cd55ed3ab2b552edc3104ca3f554" target='_blank'>
              ONCOchannelome: A computational framework for investigating altered ion channels across tumor types
              </a>
            </td>
          <td>
            K. Parthasarathi, Soundharya Ramu, M. Jolly, Jyoti Sharma
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent a paradigm of cancer evolution, where spatial and temporal heterogeneity creates dynamic ecosystems that rapidly adapt to therapeutic pressures. Through multi-region multi-omics profiling of 45 patients across primary, irradiated, and recurrent tumors, we reconstructed evolutionary trajectories using an integrated analytical framework. DNA methylation (RRBS) was processed via Bismark, with CAMDAC deconvolution distinguishing cell type-specific signals from copy number alterations. Transcriptomic data were analyzed using Salmon and edgeR, while copy number landscapes were inferred through TitanCNA and InferCNV. Evolutionary dynamics were reconstructed using PRISM, enabling methylation-based phyloepigenome reconstruction to trace subclonal epigenetic evolution. Our analysis reveals that STS evolution follows a branching phylogenetic pattern, with early “trunk” alterations establishing the foundational epigenome and subsequent “branch” events driving radiation adaptation. PRISM-based methylation phylogenies demonstrated conserved evolutionary trajectories across patients, with epigenetic divergence increasing through treatment. At diagnosis, widespread methylome remodeling systematically rewires cellular identity, with convergent hypermethylation-mediated silencing of tumor suppressive networks (SMARCA4, DICER1, FOXO3) and hypomethylation-driven activation of oncogenic pathways (CCND1/2, CDK6). Across patients, these events were recurrent and enriched for functional gene expression changes, representing positive selection for survival advantages. Under radiotherapy selection pressure, the evolutionary dynamics shift from broad remodeling to targeted adaptation. The epigenome stabilizes, focusing on DNA repair (DDB2), stemness (ZBTB16), and proliferation (CCND1) pathways—evidence of directional selection for resistance mechanisms. Single-cell phylogenetics of >150,000 cells reveals that radiation selects for pre-existing subclones rather than generating novel populations, with overall ecosystem architecture maintained despite cellular turnover. Spatial-temporal mapping demonstrates that evolutionary trajectories vary dramatically both between patients and within individual tumors, generating multi-focal ecosystems where distinct subclones coexist, compete, and occasionally converge on shared adaptive solutions. This evolutionary framework explains radiotherapy resistance as the inevitable outcome of selection acting upon pre-existing heterogeneity. The convergent emergence of DNA repair enhancement, stem-like properties, and immunosuppressive niches across patients represents parallel evolution under common selective pressures. Together, these findings argue for the development of evolution-informed therapeutic strategies that anticipate and intercept adaptive trajectories, with the potential to overcome the formidable challenge of heterogeneity in treatment-resistant STS.



 Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, Raffi Avedian, Robert Steffner, David Mohler, Anusha Kalbasi, Matt van de Rijn, Gregory Charville, Everett Moding. Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbc7c928e0902865ecc73b6cf6d1b8b370047f" target='_blank'>
              Abstract B018: Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation
              </a>
            </td>
          <td>
            Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey C L Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, R. Avedian, R. Steffner, David Mohler, A. Kalbasi, Matt van de Rijn, Gregory Charville, E. Moding
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The lung is a structurally and immunologically complex organ, constantly exposed to airborne microbes, allergens, and pollutants. Understanding how diverse pulmonary immune cells respond to these challenges is critical for advancing respiratory disease research and identifying appropriate therapeutic interventions. Flow cytometry remains a cornerstone of immune profiling, and advances in high‐parameter spectral cytometry have significantly expanded its analytical capabilities. However, challenges, such as poor tissue dissociation, spectral overlap, and loss of spatial information, can hinder the comprehensive interpretation of the lung environment. To address these limitations, we developed an optimized spectral flow cytometry platform, utilizing 5‐laser Cytek Aurora spectral cytometers, for deep immunophenotyping of murine lung tissue. Our approach integrates in vivo CD45 antibody labeling—administered intravenously and oropharyngeally—to distinguish circulating, airway, and interstitial immune populations, preserving spatial context in single‐cell suspensions. We utilize complementary 25+ parameter panels targeting myeloid and lymphoid compartments, built on a shared backbone to enable consistent classification across datasets. Refined tissue processing protocols support optimal recovery of representative lung cell populations, and overnight intracellular staining enhances marker resolution. Using this platform, we reliably resolve stromal, endothelial, and epithelial cells alongside immune subsets—including macrophages, monocytes, dendritic cells, eosinophils, neutrophils, T and B cells, innate lymphoid cells (ILCs), and natural killer (NK) cells—subclassified by activation, function, and tissue residency. Validation with an influenza A virus model confirmed expected dynamic immune responses and revealed previously unrecognized populations. This spatially informed approach enables high‐resolution interrogation of pulmonary immunity in health and disease. © 2025 Wiley Periodicals LLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91eea359003311f7d1a7abfa2e8821468360d814" target='_blank'>
              Spatially Resolved Immune Profiling of the Murine Lung Using In Vivo Labeling and Spectral Cytometry
              </a>
            </td>
          <td>
            Rebecca Palmer, Katherine R Pilkington, Kit Moloney-Geany, Kathryn E Hally, K. Hilligan
          </td>
          <td>2025-12-01</td>
          <td>Current Protocols</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are critical contributors to cancer progression and are often recruited by cancer cells to support a pro-tumorigenic microenvironment. Integrin αvβ3 is a known driver and marker of cancer stem-like properties, including tumor initiation, in various epithelial cancers. We have previously shown a positive correlation between αvβ3 expression and TAM infiltration across multiple tumor types; however, the mechanistic link remains undefined. Here, we demonstrated that integrin αvβ3 expression in non-small cell lung cancer (NSCLC) cells is both necessary and sufficient to drive TAM accumulation. In orthotopic murine and human NSCLC models, ectopic integrin αvβ3 expression increased TAM infiltration independently of T cells, whereas genetic deletion of integrin β3 significantly reduced TAM numbers and tumor burden. Mechanistically, integrin αvβ3 promotes glutamine secretion from NSCLC cells, which enhances the survival and/or differentiation of bone marrow-derived macrophages. Functionally, TAMs are essential for the elevated tumor-initiating capacity of αvβ3+ cancer cells, as macrophage depletion abolished this effect. Together, our findings uncover a novel mechanism by which NSCLC cells remodel the tumor microenvironment via αvβ3-mediated glutamine secretion, promoting TAM enrichment and tumor initiation. Targeting this axis may offer therapeutic benefits in αvβ3-expressing cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2997389ed2e2cf70cef149cc43a915aa9c061e0" target='_blank'>
              Lung Cancer Cells Secrete Glutamine to Accumulate Tumor-Associated Macrophages.
              </a>
            </td>
          <td>
            Joshua P. Reddy, Rebecca A Clague, Beatriz P Peixoto, Sara Bernstein, Emma E Lauth, Clara Y Takanohashi, Sophie C Kim, Hiromi I. Wettersten
          </td>
          <td>2026-01-07</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The presence of bone marrow (BM) disseminated tumor cells (DTCbm) identifies early-stage breast cancer patients at increased risk of recurrence and poorer overall survival. However, limitations in detecting DTCbm by standard immunohistochemical approaches have hampered clinical application. To address this gap, we developed a flow cytometry-based method, DTC-Flow, that enables the sensitive and efficient detection and molecular characterization of breast cancer DTCbm. Our analysis identified HER2 as a sensitive marker for detecting breast cancer cells, including those lacking HER2 amplification are claudin-low. DTC-Flow using a HER2/EpCAM/CD45 marker panel enabled >90% cancer cell recovery and sensitivity of one cancer cell per million nucleated BM cells across a range of breast cancer subtypes. Molecular analyses of DTC-Flow-sorted DTCbm from metastatic patients suggested a quiescent state and demonstrated their close genomic relationship to primary/metastatic tumors, as well as continued genetic evolution. In early-stage breast cancer patients, DTC-Flow detected DTCbm with greater sensitivity than cytokeratin-based immunohistochemical approaches. Our data support the development of DTC-Flow as a sensitive and specific platform to identify breast cancer patients harboring DTCbm and better understand the biology of minimal residual disease. Ultimately, this platform could enable the selection of personalized therapeutic approaches based on molecular features of DTCbm, monitoring of DTCbm to assess the efficacy of such therapies, and the development of novel therapeutic approaches targeting unique biological vulnerabilities of DTCs in order to eradicate these cells before they can give rise to lethal recurrent cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c7c6d01c7a20bdb4e978ee96e3d22d609e2fa2a" target='_blank'>
              DTC-Flow: a flow cytometry-based detection platform for characterizing bone marrow disseminated tumor cells in breast cancer
              </a>
            </td>
          <td>
            Elizabeth M. Chislock, Tien-Chi Pan, D. Pant, Yan Chen, George Woodfield, Jewell Graves, Matthew R Lawrence-Paul, G. Belka, Jianping Wang, Lauren Bayne, Tatiana Blanchard, Meaghan Smith, Xiaodan Ji, Natalie N.C. Shih, Danielle Soucier-Ernst, Noah Goodman, I. Nivar, B. Goodspeed, S. Deluca, J. Ndicu, Candace Clark, Melissa Langer, S. William Stavropoulos, William DeMuth, Andrew Morschauser, William Murphy, Anupma Nayak, Michael Feldman, Amy Clark, Liping Yu, Kevin Judge, S. Bornheimer, C. Pletcher, Jonni S Moore, Angela Demichele, Lewis A. Chodosh
          </td>
          <td>2025-11-24</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer is increasingly recognized not as a fixed genetic condition but as a dynamic and plastic disease state driven by reversible epigenetic, transcriptional and microenvironmental cues. This evolving understanding supports a therapeutic paradigm shift: from eradicating malignant cells to reprogramming them toward quiescence, differentiation or functional normalization. This review explores diverse strategies for redirecting cancer cell fate, including differentiation therapy, epigenetic remodeling, lineage reprogramming, senescence induction and tumor microenvironment resetting. These approaches exploit intrinsic cellular plasticity and contextual adaptability, offering novel avenues to contain malignancy and overcome resistance. By reframing cancer as a potentially reversible phenotype, this therapeutic strategy demands redefinition of clinical endpoints, incorporation of dynamic biomarkers, and development of integrative treatment frameworks. Ultimately, reprogramming‑based oncology holds the promise of transforming aggressive malignancies into manageable conditions while minimizing the collateral damage associated with conventional cytotoxic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99bfdfe1c6957af978b8fb4415d04fd96bdf7b85" target='_blank'>
              Beyond eradication: Therapeutic reprogramming strategies for cancer normalization (Review).
              </a>
            </td>
          <td>
            Aliasgar F. Shahiwala
          </td>
          <td>2025-12-04</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="

 Inflammatory breast cancer (IBC) is a rare invasive tumor and characterized by the formation of tumor emboli within dermal lymphatic vessels. The tumor microenvironment (TME) is a key factor in IBC aggressiveness, but its heterogeneity and intercellular role remain incompletely understood.



 Weighted gene coexpression network analysis (WGCNA) was performed to identify immune infiltration-associated genes. We also depicted cellular communication networks and specific ligand-receptor signaling pathways using the single-cell transcriptomics analysis of IBC and non-IBC samples. Finally, we verified the expression of key proteins by immunohistochemistry.




 Through the intersection of WGCNA module genes and differentially expressed genes between pathological complete response and residual disease samples,
 GZMB
 was identified as hub gene which is related to immune infiltration and efficacy of neoadjuvant therapy in IBC. In IBC cohort, patients with high expression of
 GZMB
 harbored more immunosuppressive cells thus showed unfavorable prognosis compared with
 GZMB
 -low expression group (p<0.05). Subsequent to dimension reduction and clustering, 12 clusters were identified to construct the single-cell atlas between IBC and non-IBC samples. Cellular communication analysis unveiled the heterogeneity of cell communication in IBC. The proportion of immune cells was significantly lower than that of malignant epithelial cells in the cellular composition of IBC. Moreover, it indicated that
 SPP1
 and plasmacytoid dendritic cells were specific in IBC and associated with an immunosuppressive microenvironment in IBC. Immunohistochemical analysis suggested that protein levels of
 GZMB
 and
 SPP1
 tended to be higher in the samples from patients with residual disease compared to the patient achieving pathological complete response, though this observation is based on an extremely small sample size.




 This study identifies GZMB and SPP1 as potential immunosuppressive-related prognostic biomarkers in IBC patients, reveals the key role of plasmacytoid dendritic cells in remodeling of immunosuppressive microenvironment in IBC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/282d00d57fceb6947f3f7237a1980645189f7d26" target='_blank'>
              Integrative analysis of bulk and single-cell transcriptomics identifies factors related to immunosuppressive microenvironment to predict unfavorable prognosis in inflammatory breast cancer
              </a>
            </td>
          <td>
            Kexuan Feng, Xiaoduo Li, Xiaoqian Li, Junjie Liu, Rui Zhang, Heyan Chen, Yunpeng Zhang, Jianjun He, Huimin Zhang
          </td>
          <td>2026-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b954d4c894a7ba2af75533050256820b25f647" target='_blank'>
              Heterocellular crosstalk and architecture of the pancreatic tumour microenvironment.
              </a>
            </td>
          <td>
            F. Arnold, Annachiara Del Vecchio, Zainab Hussain, Mara H. Sherman
          </td>
          <td>2026-01-16</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, A. Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, C. Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, Stanley W. K. Ng, S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Li-Fraumeni Syndrome (LFS), caused by pathogenic germline Tp53 mutations, is a pediatric cancer predisposition disorder associated with markedly elevated risk of early-onset tumors, particularly rhabdomyosarcoma (RMS). Our study examined how developmental disruption may prime the tissues of LFS individuals for tumorigenesis. Single-nuclei RNA sequencing (snRNA-seq) was performed on RMS tumors, distal muscle from mice matched to RMS tumors, and cancer-free WT and LFS mouse muscle to define lineage-specific transcriptomic signatures of LFS-RMS. Embryonic limb buds were collected at embryonic days 10, 12, 14, and 16, subjected to snRNA-seq, and used to reconstruct developmental trajectories. These lineage-specific signatures were mapped back onto both WT and LFS embryonic lineages to determine whether oncogenic programs were detectable in utero and to identify the timing of their emergence. A stress-primed transcriptional program was identified as a defining feature of LFS tissues. This program distinguished precancer from postcancer states, becoming amplified in both RMS tumors and distal muscle matched to tumors. In LFS embryos, stage-specific alterations were observed in cell type proportions and maturation, including the emergence of unique myogenic and mesenchymal subpopulations absent in both WT embryos and postnatal muscle, reflecting developmental plasticity restricted to the LFS embryonic context. A lag in the transition to mature myogenic cells was detected at E14, preceded by a spike in the LFS stress-primed signature at E12 during windows of active lineage specification and proliferation. Pathway analyses of lineage-specific signatures indicated widespread dysregulation, with particularly strong changes in RNA-associated and developmental programs, further defining this stress-sensitive transcriptional state. These findings provide the first single-nuclei resolution map of embryogenesis in LFS and provide evidence that LFS-associated oncogenic programs are prenatally encoded. By linking altered lineage dynamics, developmental plasticity, and stress-primed transcriptional states to the emergence of RMS, this work establishes a developmental framework for understanding pediatric cancer predisposition. The results highlight potential windows for early detection and suggest opportunities to target embryonically primed programs for intervention in LFS.



 Ashby Kissoondoyal, Paula R. Quaglietta, Brianne Laverty, David Malkin. Single-nuclei profiling of LFS development reveals tumour susceptibility [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4bd18cd0e2b5ff6a30ceebfb643ff450b1c37a" target='_blank'>
              Abstract B033: Single-nuclei profiling of LFS development reveals tumour susceptibility
              </a>
            </td>
          <td>
            Ashby Kissoondoyal, Paula R. Quaglietta, B. Laverty, D. Malkin
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9c32a29d95e98ff8be740eed1b959dae12abb15" target='_blank'>
              Drosophila midgut tumor-induced insulin resistance systemically remodels lymph gland hematopoiesis during cancer cachexia
              </a>
            </td>
          <td>
            Ujjayita Chowdhury, Gauri Panzade, Priya Neelesh Karnik, Pushkar Birwadkar, Tashu Tashu, R. Khadilkar
          </td>
          <td>2025-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The tumor microenvironment (TME) represents a highly dynamic and multifaceted ecosystem composed of both cellular and noncellular components that collectively modulate tumor initiation, progression, and metastatic potential. Within this intricate milieu, neuron-tumor interactions have recently emerged as critical determinants of cancer behavior, especially in malignancies of the nervous system, where neuronal activity appears to sustain and even accelerate tumor growth. Increasing evidence supports the existence of a bidirectional communication between neurons and cancer cells, involving the exchange of soluble factors, neurotransmitters, and vesicular signals that influence cell survival, proliferation, and morphology. To dissect the molecular mechanisms underlying this neuron-tumor crosstalk, we established and optimized a series of in vitro co-culture models. Primary cortical neurons derived from C57BL/6 mice and neurons differentiated from human induced pluripotent stem cells (hiPSCs) were co-cultured with U87 glioblastoma cells to reproduce the neural-tumor interface. Both transwell-based co-culture systems and conditioned media (CM) approaches were implemented to evaluate the impact of glioma-secreted components on neuronal viability and morphology. Neuronal dynamics were monitored in real time using the Incucyte live-cell imaging system equipped with the Neurotrack module. Exposure to glioma-conditioned medium led to pronounced neurotoxic effects in murine cells, including a 36% reduction in neurite length (p=0.0055), a 44% decrease in cell number (p=0.0024), and a significant increase in cytotoxicity after 48 h (p=0.0116). These findings highlight the detrimental influence of glioblastoma-derived soluble factors on neuronal structure and function. Complementary Western blot analyses confirmed this phenotype, revealing a marked decrease in βIII-tubulin (-60%) and synapsin (-80%) expression in CM-treated murine neurons, indicating impaired cytoskeletal stability and synaptic integrity. Current efforts focus on extending these observations to hiPSCderived neuronal cultures and enhancing the physiological relevance of the model by integrating advanced microfluidic co-culture systems. Such platforms will allow spatiotemporal control of cellular interactions, providing a more faithful in vitro representation of the brain tumor microenvironment and enabling the identification of molecular targets capable of disrupting the pro-tumor neuronal support network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47801036babe68c650156cad23ef9951afb3627d" target='_blank'>
              EXPLORING NEURON–GLIOMA SIGNALING THROUGH IN VITRO CO-CULTURES
              </a>
            </td>
          <td>

          </td>
          <td>2025-12-12</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 CD47 is a key innate immune checkpoint that enables tumor cells to evade macrophage-mediated clearance.



 To systematically identify genetic regulators of CD47 surface expression, we performed FACS-based genome-wide CRISPR screens in three murine cancer cell lines B16 (melanoma), MC38 (colon adenocarcinoma), and EMT6 (breast carcinoma).



 Comparative analysis of cells with high or low CD47 surface expression using DrugZ revealed CD47 itself as the top hit, validating the screens. Notably, DNAJC13 emerged as a consistent and robust regulator of CD47 expression across all three cell lines. Functional validation using DNAJC13-knockout cells confirmed a significant reduction in CD47 surface levels. Furthermore, in co-culture assays with macrophages, DNAJC13-deficient tumor cells exhibited increased susceptibility to phagocytosis, supporting a functional role for DNAJC13 in innate immune evasion. Finally, we verify that DNAJC13-knockout decrease tumor burden when treated with CD47 blockade.



 Overall, this study highlights a previously unrecognized regulator of CD47 and demonstrates the utility of high-throughput FACS-based CRISPR screening to uncover modulators of key immune checkpoint pathways.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/333becc09fbebbb08ac6ada5d4961c0def934201" target='_blank'>
              FACS-based genome-wide CRISPR screening platform identifies modulators of CD47
              </a>
            </td>
          <td>
            Ling Yin, Wei He, Yifan Wang, Huimin Zhang, Min Huang, Yuelong Yan, Siting Li, Xu Feng, Francisco Saenz, Jie Zhang, Dandan Zhu, Chang Yang, Tiantian Ma, Jialing Fu, Junjie Chen
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b89d2858599b4236f22769f3a4af65ec9fd4eb7" target='_blank'>
              Deciphering a common code: Unitary gene profile factor enabling Spontaneous Tumor Regression
              </a>
            </td>
          <td>
            Arushi Misra, B. Kumari, Prasun K. Roy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c936d3b24a47f917e24b5388752187347f78add" target='_blank'>
              Advancing Precision Medicine in Malignant Pleural Mesothelioma: The Emerging Role of Patient-Derived Organoids
              </a>
            </td>
          <td>
            F. Gonnelli, F. Mei
          </td>
          <td>2025-12-01</td>
          <td>Current Pulmonology Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cholesterol metabolism (CM) plays a critical role in the progression of colorectal cancer (CRC), yet its molecular and immunological implications remain incompletely understood. Therefore, we aimed to identify CRC subtypes according to CM-related genes and reveal their distinct characteristics. Based on CM-related genes, we applied unsupervised clustering to classify CRC into two subtypes using transcriptomic data from TCGA and comprehensively compared their transcriptomic, genomic and clinical characteristics. We utilized single-cell RNA sequencing data and classified the samples into two subtypes and investigated the distinctions in the tumor microenvironment (TME) between these subtypes. Two distinct CM subtypes were identified: Subtype A, characterized by cholesterol esterification and storage, was associated with inflammatory activation and cellular senescence. This subtype exhibited a poor prognosis and reduced predicted response to chemotherapy and immunotherapy. Tumor cells in Subtype A exhibited characteristics of epithelial-mesenchymal transition and angiogenesis. The TME in Subtype A contained higher infiltration of myeloid cells, fibroblasts, and pericytes, with dominant immunosuppressive tumor-associated macrophages (TAMs), especially TAM_SPP1, which interacted closely with Fibro_IL32, promoting immune exclusion. In contrast, Subtype B was marked by enhanced cholesterol catabolism and regulation. Tumor cells in this subtype displayed features of proliferation and stem-like properties. It showed a more active immune microenvironment with increased plasma cell infiltration and fewer immunosuppressive TAMs. Finally, we constructed a prognostic signature and validated its performance across multiple datasets. These findings provide comprehensive insights into CM subtypes in CRC, highlighting their clinical significance and potential therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/706999d9315dc3820db3be850be9b31661068035" target='_blank'>
              Multi-omics analysis reveals different cholesterol metabolism subtypes in colorectal cancer
              </a>
            </td>
          <td>
            Yu Sun, Deyang Kong, Huiru Zhang, Jialiang Fan, Shuaibing Lu, Wenjing Yang, Renshen Xiang, Qi Zhang, Wei Pei, Lin Feng, Haizeng Zhang
          </td>
          <td>2025-11-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Emerging evidence suggests that oral squamous cell carcinoma (OSCC) harbors distinct microbial communities, yet their influence on tumor immunobiology remains unclear. This study investigated the prognostic value of intratumoral microbiota and their role in modulating CD8⁺ T-cell function. Using TCGA datasets, a 19-microorganism prognostic signature was constructed via Cox and LASSO regression and validated with Kaplan–Meier survival analysis. Comprehensive bioinformatics analyses, including GSEA/GSVA, CIBERSORT, ESTIMATE, pharmacogenomics, and mutational profiling, were performed to explore underlying mechanisms. The microbial signature demonstrated strong predictive performance, with higher risk scores significantly associated with reduced overall survival (p < 0.001). High-risk tumors exhibited enrichment of PI3K/AKT/mTOR signaling, metabolic reprogramming, and epithelial–mesenchymal transition, alongside an immunosuppressive microenvironment characterized by CD8⁺ T-cell depletion, M0 macrophage infiltration, and upregulation of immunosuppressive markers including CD276 and TGF-β1. Conversely, immune-activating checkpoints such as PD-1 and CTLA-4 were elevated in low-risk tumors. Notably, periodontal pathogens negatively correlated with immune effector activity, and TP53 mutations were more frequent in high-risk cases (82% vs. 67%). This study identified intratumoral microbial signatures as independent prognostic biomarkers and validated their reproducibility in an external cohort. Our findings support a microbiota–immune axis contributing to immune evasion in OSCC, offering novel avenues for prognostic stratification and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea87f1d3cad21e45f622b4777a7389fe481a0a6c" target='_blank'>
              Intratumoral microbiota drive immune evasion and disease progression in oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaolong Zang, Xiaoxia Li, Rongxin Sun, Xiaojie Zhang, Zhiyong Li, Zijian Cheng
          </td>
          <td>2025-12-01</td>
          <td>BDJ Open</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6172dabf52906e17a9170089cda881b60914418f" target='_blank'>
              A human multilineage gut organoid model for Parkinson disease
              </a>
            </td>
          <td>
            Mahya Hosseini Bondarabadi, Minqian Xu, Jim de Leeuw, S. Slingerland, S. van der Zee, Iris E. C. Sommer, H. Harmsen, T. van Laar, Sven C D van IJzendoorn
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c2a1ea73f6fab9f1966b90164d74f94fad3f917" target='_blank'>
              Large scale quantification of natural killer cell-induced apoptosis in patient-derived organoids reveals intratumoral response heterogeneity.
              </a>
            </td>
          <td>
            Marisa Mercadante, Armin Scheben, Jacob Estrada, Jan Savas-Carstens, William Sullivan, Nicholas Housel, Maria Sapar, Tatiana Volpari, Jax Hebner, Tomasz Rusielewicz, Frederick J. Monsma, Stefan Semrau, Yinan Wang, Laura A. Martin
          </td>
          <td>2025-12-26</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Nowadays, chronic benign and malignant prostatic diseases are prevalent, costly, and impose a significant burden. Benign prostatic hyperplasia (BPH), a common condition in the aging population, often coexists with localized prostate cancer (PCa). These diseases likely share underlying molecular mechanisms, which remain poorly understood. The exploration of novel cell subpopulations and specific biomarkers for accurate diagnosis and treatment of prostatic diseases is ongoing and holds great clinical promise. Prostate cell proliferation and immune inflammation are key contributors to the progression of BPH and PCa, involving various prostate and immune cell subpopulations. This raises important questions about how specific cell types drive phenotypic heterogeneity. Advanced single-cell RNA sequencing (scRNA-seq), a cutting-edge technology, offers unparalleled insights at the single-cell level. Similar to a microscope that identifies cell types within tissue samples, scRNA-seq elucidates cellular heterogeneity and diversity within single cell populations, positioning itself as a future-leading sequencing technology. Considering that BPH and PCa share androgen-dependent growth, chronic inflammation and specific microenvironmental changes, this review discusses recent discoveries of novel cell subpopulations and molecular signatures in BPH and PCa that can be dissected by scRNA-seq. It aims to help researchers better understand the molecular pathogenesis of these conditions while offering new therapeutic possibilities for clinical management of benign and malignant prostatic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d65bfb63670355f27eb34fa433fbe18e5d3f994e" target='_blank'>
              Advances in the identification of novel cell signatures in benign prostatic hyperplasia and prostate cancer using single-cell RNA sequencing
              </a>
            </td>
          <td>
            Yu Pan, Qingqing Song, B. Lai, He Ma
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastasis is still the leading cause of cancer-related death. It happens when disseminated tumor cells (DTCs) successfully navigate a series of steps and adapt to the unique conditions of distant organs. In this review, key molecular and immune mechanisms that shape metastatic spread, long-term survival, and eventual outgrowth are examined, with a focus on how tumor-intrinsic programs interact with extracellular matrix (ECM) remodeling, angiogenesis, and immune regulation. Gene networks that sustain tumor-cell plasticity and invasion are described, including EMT-linked transcription factors such as SNAIL and TWIST, as well as broader transcriptional regulators like SP1. Also, how epigenetic mechanisms, such as EZH2 activity, DNA methylation, chromatin remodeling, and noncoding RNAs, lock in pro-metastatic states and support adaptation under therapeutic pressure. Finally, proteases and matrix-modifying enzymes that physically and biochemically reshape tissues, including MMPs, uPA, cathepsins, LOX/LOXL2, and heparinase, are discussed for their roles in releasing stored growth signals and building permissive niches that enable seeding and colonization. In parallel, immune-evasion strategies that protect circulating and newly seeded tumor cells are discussed, including platelet-mediated shielding, suppressive myeloid populations, checkpoint signaling, and stromal barriers that exclude effector lymphocytes. A major focus is metastatic dormancy, cellular, angiogenic, and immune-mediated, framed as a reversible survival state regulated by stress signaling, adhesion cues, metabolic rewiring, and niche constraints, and as a key determinant of late relapse. Tumor-specific metastatic programs across mesenchymal malignancies (osteosarcoma, chondrosarcoma, and liposarcoma) and selected high-burden cancers (melanoma, hepatocellular carcinoma, glioblastoma, and breast cancer) are highlighted, emphasizing shared principles and divergent organotropisms. Emerging therapeutic strategies that target both the “seed” and the “soil” are also discussed, including immunotherapy combinations, stromal/ECM normalization, chemokine-axis inhibition, epigenetic reprogramming, and liquid-biopsy-enabled minimal residual disease monitoring, to prevent reactivation and improve durable control of metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a5ad40c8765430fb4e35e8df4878d707b5a6318" target='_blank'>
              Molecular and Immune Mechanisms Governing Cancer Metastasis, Including Dormancy, Microenvironmental Niches, and Tumor-Specific Programs
              </a>
            </td>
          <td>
            Dae Joong Kim
          </td>
          <td>2026-01-15</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c6b3a4672d8a49cdbb7af4005dce769ab547c41" target='_blank'>
              Prognostic impact of spatial niches in prostate cancer.
              </a>
            </td>
          <td>
            Felix Schneider, Sarah Böning, Beatriz Coelho Antunes, A. Kaczorowski, M. Görtz, V. Schütz, Johannes Huber, Albrecht Stenzinger, Markus Hohenfellner, S. Duensing, A. Duensing
          </td>
          <td>2026-01-17</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492f9dec710186d0bb398ff8467a9fedaf4e7eda" target='_blank'>
              Three-dimensional bioprinting of patient-derived Gastrointestinal stromal tumor: a novel platform for precision oncology and drug response profiling.
              </a>
            </td>
          <td>
            Liwei Du, Zicheng Zheng, Yanan Wang, Kai Zhang, Yuce Lu, Minghao Sun, M. Pang, Shangze Jiang, Yixuan He, Shunda Du, Haitao Zhao, Yilei Mao, Huayu Yang, Weiming Kang
          </td>
          <td>2025-12-18</td>
          <td>Journal of nanobiotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Metastasis is a leading cause of mortality in breast cancer patients, yet the signaling promoting metastatic dissemination is not completely understood. Prior literature implicates neuronal innervation in tumor progression, including recent studies in breast cancer progression with the 4T1 and PyMT cell line orthotopic injection models. Our experiments address the immune limitations of these studies with an alternative model to elucidate neuronal control of metastatic breast cancer by using a MMTV-PyMT transplant model and resiniferatoxin (RTX) for denervation. To this end, we generated a robust array of spontaneous MMTV-PyMT tumors with various histological subtypes. These tumors were transplanted into RTX challenged MMTV-Cre mice. In contrast to previous literature, denervation did not impact survival or tumor growth. Interestingly, we noticed a slight reduction in the percent of the solid poorly differentiated tumors with a corresponding increase in tumors that contained a mixed pathology in RTX challenged mice. Strikingly, and consistent with prior work, we noted a reduction in metastasis with denervation. Together, these data suggest neuronal innervation promotes metastasis without impacting tumor growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a7eaed2f0618eb9d5a5d855a2c066e128ecb44" target='_blank'>
              Neuronal innervation of breast cancer promotes metastatic dissemination
              </a>
            </td>
          <td>
            Anthony J. Schulte, E. Andrechek
          </td>
          <td>2025-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer develops through the interactions between cancer stem cells and the components of tumor microenvironment (TME). To model in vivo cancer stem cell/TME interactions and elucidate their functional consequences, we focused on myelofibrosis (MF), a stem cell driven myeloproliferative neoplasm. We co-cultured MF hematopoietic stem and progenitor cells (HSPCs) with normal donor endothelial cells (ECs) and mesenchymal stromal cells (MSCs) to investigate the consequences of interactions between malignant MF HSPCs and non-malignant microenvironmental cells. This tri-cultivation system proved to be a simple and reproducible platform, which promoted malignant clone dominance and the persistence of MF HSPCs that recapitulate the MF phenotype upon transplantation into immunodeficient mice, including splenomegaly and marrow fibrosis. Transcriptional profiling revealed extensive reprogramming of not only the co-cultured MF HSPCs, but also MSCs and ECs. Although numerous disease-relevant pathways were upregulated, the pro-inflammatory response stood out as a key consequence of MF HSPC/TME interactions. We validated these findings through quantitation of pro-inflammatory transcript upregulation and cytokine production. This human multicellular model system has proven useful in demonstrating the multidirectional interactions of MF HSPCs with TME cells that are essential for sustaining fully functional MF stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/205e3d2dddeef25776bcf5fd8943a758cf086b36" target='_blank'>
              Microenvironmental cell interactions are essential for sustaining functionality of myelofibrosis malignant stem cells.
              </a>
            </td>
          <td>
            Min Lu, Md Babu Mia, Lijuan Xia, Gohar Mosoyan, Momina Hayat, Christoph Schaniel, Ronald Hoffman
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Studies show that only 20–30% of untreated ductal carcinoma in situ (DCIS) cases will evolve into invasive breast cancer. Thus, many patients are subject to overtreatment and its associated morbidities. The ecological context of the tumor microenvironment may impact how DCIS evolves. We applied landscape ecology statistics to hematoxylin and eosin (H&E) images from DCIS to test whether they predict recurrence of DCIS or progression to invasive disease. The ability to make such predictions is key for optimizing treatment decisions, reducing associated risks, and improving the efficiency of healthcare resource allocation.



 We conducted a retrospective cohort study comprising 158 patients with a DCIS diagnosis, which included 57 cases of DCIS recurrence (2–228 months) and 45 cases of progression to invasive disease (12–180 months). Of these, 21 patients received lumpectomy, 92 had lumpectomy followed by radiation therapy, and 45 had total mastectomy. We generated a digital pathology library of whole-slide images (WSIs) from H&E tumor sections and performed machine learning guided nuclear segmentation and classification. A comprehensive set of spatial statistics was calculated for each WSI to characterize the spatial architecture and cellular interactions within the microenvironment. Cross-validation using a linear spline-based hazard regression model (HARE) was applied to a training subset of the data. The resulting survival model was validated on the test subset to predict disease-free survival (DFS) of recurrence and progression.



 We found that the type of DCIS treatment significantly stratified patients based on DFS (log-rank test p<0.0001). Further analysis of patients who received lumpectomy + radiation or mastectomy revealed that decreased mean nearest neighbor distance between tumor cells and lymphocytes, and between fibroblasts and lymphocytes, was significantly associated with improved recurrence-free survival (concordance=0.68). Stratification of samples based on the median risk score derived from our model was discriminatory with a p-value <0.015. Additionally, a decreased nearest neighbor distance between lymphocytes and fibroblasts showed a marginal association with an increased time to invasive disease (p<0.06).



 Integrating spatial ecological statistics derived from digital pathology provides enhanced risk stratification for DCIS, beyond the established effects of initial treatment. Our findings underscore the importance of the tumor microenvironment's ecological dynamics in DCIS recurrence and progression. The predictive risk associated with specific spatial interactions, particularly involving lymphocytes and fibroblasts, suggests that previously unappreciated ecological mechanisms may play a role in the development of early micro-metastases, influencing outcomes even after standard therapies such as mastectomy or lumpectomy with radiation. These insights highlight novel targets for improved prognostic tools and more personalized therapeutic strategies.



 Luis H. Cisneros. Microenvironmental drivers of DCIS recurrence and progression to breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1296ebbd19523c1df1d8c9459fc242375494458" target='_blank'>
              Abstract A016: Microenvironmental drivers of DCIS recurrence and progression to breast cancer
              </a>
            </td>
          <td>
            L. Cisneros
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is an indolent malignancy with modest proliferation in the lymph nodes and accumulation of quiescent B cells in the peripheral blood. Targeted agents, including BTK inhibitors such as ibrutinib and the BCL2 antagonist venetoclax, have transformed therapy by disrupting proliferation, survival, and lymph node retention of CLL cells, yet CLL remains incurable. Recent studies reveal that CLL cells exist along a spectrum of proliferating, activated, and quiescent states, with dynamic transitions that shape intraclonal behavior. Whilst proliferation occurs mainly in lymph nodes, most emigrant cells in the peripheral blood become quiescent, with only a minority remaining activated. Quiescent, activated, and proliferating fractions display distinct phenotypes and CXCR4 and CD5 levels can be used to distinguish these states in the CLL life cycle. While proliferating and activated cells are more susceptible to BTK inhibition, quiescent subsets show greater sensitivity to BCL2 blockade. These functional differences, together with emerging evidence that phenotypic markers may correlate with residual disease activity, point to potential translational significance. Understanding how CLL cells switch between proliferative, activated and quiescent states will be important to uncover novel vulnerabilities and inform rational treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a77b3d2236234d56896cf6069ee3a5f691d6d65" target='_blank'>
              Functional Intraclonal Heterogeneity in Chronic Lymphocytic Leukemia: Proliferation vs. Quiescence
              </a>
            </td>
          <td>
            Daniel Friedman, P. Patten, Robbert Hoogeboom
          </td>
          <td>2025-12-17</td>
          <td>Lymphatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="

 Hepatocellular carcinoma (HCC) remains a leading cause of cancer-related mortality, with limited efficacy of current therapies in advanced cases. As a key risk factor for HCC, liver fibrosis may influence tumor progression and immune responses. However, fibrosis-related therapeutic targets remain poorly defined. This study aimed to identify fibrosis-related genes in HCC tumor microenvironment (TME).



 Our research integrated single-cell RNA sequencing (GSE149614), spatial transcriptomics (GSE245908), and bulk RNA-seq data to identify fibrosis-related prognostic genes in HCC. The genes were selected via the Random Survival Forest algorithm. Additionally, bioinformatics analyses were conducted to explore gene expression patterns, immune infiltration, and spatial localization. Key genes were further validated through in EDU incorporation assay, Transwell migration assay, and CCK-8 proliferation assay.



 Firstly, single-cell analysis identified endothelial cells as key fibrosis-associated cluster in HCC. Three fibrosis-related prognostic genes, LUC7L3, CREB1, and YIPF4, were further identified and validated to patient survival, immune infiltration, and metabolic activity. In addition, enrichment and drug sensitivity analyses linked key genes to tumor-related pathways and chemotherapy response. Spatial transcriptomics then confirmed the spatial distribution and interactions of these genes. Lastly, cellular assays showed that YIPF4 promoted proliferation and migration of HCC cells.



 In this study, we identified fibrosis-related prognostic genes in HCC, including LUC7L3, CREB1, and YIPF4. The roles of these genes in TME were further explored through relevant analyses, potentially providing clinical evidence to support decision-making in HCC management.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05045962e54fadb4e8932c963a89c2eda496945e" target='_blank'>
              Integrating single-cell RNA sequencing with spatial transcriptomics reveal the fibrosis-related genes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Wenying Qiao, Lei Li, Ronghua Jin, Caixia Hu
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Central nervous system (CNS) metastasis is a major driver of morbidity in metastatic breast cancer, yet the molecular determinants of CNS tropism remain incompletely defined. LYN, a Src-family kinase integrating receptor tyrosine kinase and integrin signaling, is a biologically plausible mediator of metastatic traits. We performed a retrospective, multi-study analysis of publicly available breast cancer cohorts aggregated in cBioPortal. After harmonization and de-duplication, LYN status was determinable in 5,947 invasive carcinoma of no special type (NST) tumors across 29 studies. The primary endpoint was CNS metastasis at any time (Yes/No), harmonized via a prespecified controlled vocabulary (case-insensitive substring mapping). Somatic LYN variants (coding SNVs/indels) were collapsed to patient-level classes (missense-only; truncating if any nonsense/frameshift/splice). Variants with resolvable positions were mapped to Src-family modules (SH4/Unique, SH3, SH2, SH2-kinase linker, kinase). Two-group comparisons used two-sided Fisher’s exact tests with exact 95% CIs; domain screens used omnibus χ² and Benjamini–Hochberg FDR control. A prespecified Firth logistic model evaluated truncating vs. missense within LYN-mutant tumors. Public cancer genomics repositories (cBioPortal); multi-institutional cohorts. 5,947 tumors across multi-study cohorts with LYN status available. None. Primary: ever-CNS metastasis (yes/no). Secondary: distribution of LYN variant classes and domains (SH4/Unique, SH3, SH2, linker, kinase). CNS metastasis occurred in 5/46 (10.9%) LYN-mutated tumors vs. 110/5,901 (1.9%) LYN wild-type tumors (odds ratio (OR) = 6.42; 95% confidence interval (CI), 2.49–16.56; p = 0.0018). The endpoint was captured as ever vs. never CNS involvement (event dates unavailable), precluding time-to-event inference. Within LYN-mutant cases, an exploratory domain analysis indicated that distributions differed by CNS status (omnibus χ² p ≈ 0.014); a one-versus-rest signal at the SH4/Unique N-terminus was nominally significant and borderline after false discovery rate (FDR) (unadjusted p ≈ 0.010; q ≈ 0.052; small in-domain n = 3). By mutation class, truncating vs. missense showed a higher CNS-positive proportion (28.6% vs. 7.9%) but did not reach significance (Fisher p = 0.166; alternatively framed OR = 4.22; exact 95% CI, 0.58–30.75; p = 0.182). Firth estimates were directionally consistent with wide profile CIs under sparse counts. Across pooled cohorts, LYN mutation is associated with increased odds of CNS metastasis, and domain context appears informative, with a small-sample, FDR-borderline enrichment at the SH4/Unique N-terminus. The truncating-class signal is exploratory given limited power. Signals by domain (notably SH4/Unique) are exploratory and require independent validation in larger, uniformly annotated datasets. Given small mutant denominators and ever-CNS endpoint capture, findings are hypothesis-generating and not actionable for risk stratification or treatment selection. Results motivate domain-aware annotation in future validation studies and mechanistic work. What is already known: CNS metastasis is a major clinical problem in breast cancer; molecular determinants of brain tropism remain incompletely defined. What this study adds: Across pooled cohorts, any LYN mutation is associated with higher odds of CNS metastasis; domain context (notably SH4/Unique) may matter, albeit with small counts and FDR-borderline evidence. How this might affect research/practice/policy: Encourages domain-aware LYN annotation and targeted mechanistic work on membrane targeting/BBB traversal; not ready for risk stratification or treatment selection. What is already known: CNS metastasis is a major clinical problem in breast cancer; molecular determinants of brain tropism remain incompletely defined. What this study adds: Across pooled cohorts, any LYN mutation is associated with higher odds of CNS metastasis; domain context (notably SH4/Unique) may matter, albeit with small counts and FDR-borderline evidence. How this might affect research/practice/policy: Encourages domain-aware LYN annotation and targeted mechanistic work on membrane targeting/BBB traversal; not ready for risk stratification or treatment selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e375e78de0e51d3450906c5e0322f113edd318" target='_blank'>
              LYN mutations in breast cancer: hypothesis-generating evidence for an association with central nervous system metastasis and domain-level insights
              </a>
            </td>
          <td>
            Elif Kardelen Çağdaş, Berkay Çağdaş
          </td>
          <td>2025-12-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Myxofibrosarcoma (MFS) is a rare and aggressive soft tissue sarcoma characterized by high genomic instability, resulting in high local recurrence rates and limited effective therapeutic options in advanced stages. Recent progress in cancer immunology research has encouraged investigation into the Tumor Microenvironment (TME) of sarcomas, including MFS, to identify immune-related biomarkers of prognostic and therapeutic relevance. Although data remain limited in MFS, existing evidence suggests a heterogeneous immune landscape, including: i) variable expression of immune checkpoint molecules such as Programmed Cell Death Protein 1 (PD-1) and Programmed Death-Ligand 1 (PD-L1), ii) presence of tumor-infiltrating lymphocytes, iii) alterations in antigen presentation pathways, and iv) a pronounced angiogenic signature. These findings underscore the potential role of immune biomarkers for patients’ clinical stratification and the consequent possibility of developing new immunotherapeutic strategies. This review will focus on the cellular and molecular architecture of immune infiltration, vascular remodeling, and lymphoid neogenesis, assessing their prognostic and predictive value as potential biomarkers. Finally, we will present ongoing clinical trials aimed at modulating the immune-vascular niche to inform innovative therapeutic strategies for this challenging sarcoma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f542fc2b8b6a3c1d5def7187e73b0d930cab87" target='_blank'>
              Unveiling the myxofibrosarcoma tumor microenvironment: implications for immunotherapy
              </a>
            </td>
          <td>
            Cecilia Profumo, Massimiliano Grassi, Valentina Rigo, Matteo Mascherini, Paola Monti, Giorgia Arcovito, Giorgia Anselmi, Laura Damele, G. Timon, Danila Comandini, Michela Croce
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions (doi.org/10.1101/2024.12.28.623235). We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. We thus document cancer initiation in its entirety and 1) show how the earliest step presages the acquisition of driver mutations, 2) identify the tiny subset of post-UV exposure clades (<1%) prone to acquire more mutations and spawn lesions, and 3) provide an explanation for why cancers are rare in relation to the degree of somatic mosaicism present.



 Stanislav Avdieiev, Leticia Tordesillas, Karol Prieto, Omar Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, Luiza Simoes, Y. Ann. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai. Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr PR001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3d807e01a5b2ecf2c9924475e458271bb925483" target='_blank'>
              Abstract PR001: Goliath clades and
 in vivo
 tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, Omar Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. Simoes, Y. A. Chen, Noemi Andor, R. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y Tsai
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The reproducibility of scientific research has been increasingly challenged in recent years. While extrinsic factors (e.g., cross-contamination, mycoplasma infection, serum variability) are well-studied, the role of intrinsic attributes like cell passage number remains underexplored. Using two tumor cell lines (ACHN and Renca), we employed RNA sequencing to analyze transcriptomic dynamics across passages (P3 to P39). Results revealed a nonlinear transcriptomic shift: mid-passage cells (P10/P11, P17) showed heightened activity in cell cycle, metabolism, and stress response, whereas low (P3) and high passages (P24/P39) exhibited stable and similar profiles. KEGG analysis indicated significant alterations in signaling, immune, and metabolic pathways during passaging. This study demonstrates that passage number shapes transcriptional landscapes and impacts experimental reproducibility. We advocate for strict passage control and explicit reporting of passage information to enhance reliability. These findings underscore the need for standardized cell culture practices to improve research reproducibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf829c80ee16d8593d7d0383b2985d0175ddbfa6" target='_blank'>
              Cell passage number drives transcriptomic drift as an overlooked factor in experimental reproducibility
              </a>
            </td>
          <td>
            Shuai Liu, Hai-juan Peng, Junyan Meng, Liyuan Zhuo, Shaoping Fu, Wei Yang
          </td>
          <td>2025-11-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Solid pseudopapillary neoplasm (SPN) of the pancreas is a rare but distinct disease that remains poorly understood, especially at proteome level. We report comprehensive mass spectrometry-based proteomic analyses of SPN (n = 13) and characterize differences from other pancreatic neoplasms, pancreatic ductal adenocarcinoma (n = 11) and neuroendocrine tumor (n = 10). We discovered that the SPN proteome is uniquely distinct from that of other pancreatic neoplasms. Lysosome-related proteins are enriched and upstream lysosomal processes transcriptional regulators, MITF and TFE3, are overexpressed in SPN. MITF protein expression is more specific for SPN than TFE3, previously considered the most specific immunohistochemical marker. Since lysosomal-related processes are connected to biological energy generation processes, we profiled metabolic pathways and found that SPN is characterized by higher fatty acid oxidation and lower glycolysis than PDAC and high proteasome pathway activity with many proteasomal proteins upregulated, suggesting a possible link to metabolic adaptation mechanisms in low-nutrient environments. Proteomics characterizes SPN as an immune-cold tumor with low MHC class I expression. Proteome-based receptor tyrosine kinase (RTK) pathway profiling suggests PDGFRA and ERBB2 (HER2) as potential candidates for targeted therapy. Our results provide unique proteomic contribution to the understanding of SPN biology and highlight differences from other pancreatic tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef74a81b52ffee19b8bfc39c72b4301805c6d4c3" target='_blank'>
              Deep proteogenomic characterization of pancreatic solid pseudopapillary neoplasm reveals unique features distinct from other pancreatic tumors
              </a>
            </td>
          <td>
            Atsushi Tanaka, Yusuke Otani, David S Klimstra, O. Basturk, Monika M Vyas, Julia Y. Wang, Michael H. Roehrl
          </td>
          <td>2025-11-27</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Osteosarcoma arises within heterogeneous tumor–immune ecosystems in which impaired antigen visibility—shaped by chromatin programs—limits immune surveillance and blunts responses to immunotherapy. Beyond structural defects in the antigen-processing pathway, Polycomb-mediated repression, DNA hypermethylation, and state-specific enhancer closure converge on the HLA class I/NLRC5/interferon axis to diminish peptide display. These constraints are context dependent, varying across malignant clones, differentiation states, and myeloid and T-cell niches. Traditional bulk assays obscure this complexity; single-cell ATAC-seq, integrated with single-cell and spatial transcriptomics, now resolves promoter–enhancer accessibility at HLA, NLRC5, and antigen-processing genes, distinguishes reversible repression from fixed lesions, and links microenvironmental stress to interferon competence. Translationally, epigenetic reprogramming—targeting Polycomb repressive complex 2 (PRC2), DNA methyltransferases (DNMTs), and complementary regulators (for example, LSD1, BET, CDK4/6, YAP/TEAD)—offers biomarker-guided avenues to restore antigen presentation, provided ecosystem-aware pharmacodynamic readouts track chromatin opening and antigen-presentation recovery across compartments. Despite encouraging preclinical evidence, efficacy will depend on clone selection, scheduling that preserves interferon signaling, and rational combinations with innate agonists and checkpoint blockade. This mini-review synthesizes epigenetic mechanisms of antigen-presentation failure in osteosarcoma and outlines how single-cell chromatin profiling can guide strategies to reinstate tumor antigen visibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f51ccd00bee57595f98289280452a3ec7548b15" target='_blank'>
              Epigenetic control of antigen presentation failure in osteosarcoma: from single-cell chromatin maps to therapeutic strategies
              </a>
            </td>
          <td>
            Yan He, Heng Wu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Mismatch repair-deficient (dMMR) colorectal cancers (CRCs) exhibit variable clinical responses to immune checkpoint inhibitors (ICIs) despite their high immunogenicity.



 To investigate the molecular and spatial determinants of this heterogeneity, we analyzed five metastatic sites (colon, liver, peritoneum, left ovary, and right ovary) from a dMMR CRC patient treated with pembrolizumab using bulk and single-cell RNA sequencing combined with multiplex immunohistochemistry.



 Responsive lesions were characterized by enriched cytotoxic and interferon-γ signatures, greater CD8⁺ T cell infiltration, and close spatial proximity between PD-L1⁺ tumor cells and M1-like macrophages. In contrast, resistant lesions demonstrated reduced effector cell presence and were enriched in immunosuppressive programs, including SPP1⁺ and CD163⁺ macrophages, noncanonical WNT5A/B signaling, and TGF-β–mediated matrix remodeling.



 Collectively, these findings highlight the importance of spatial immune architecture and macrophage polarization in shaping ICI responses in dMMR CRC and underscore the need for spatial profiling to guide immunotherapy strategies in metastatic disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7dee5edb3a5c46063fed969eabd6424ae6c3c71" target='_blank'>
              Comprehensive Spatial and Immune Profiling of Metastatic Mismatch Repair–Deficient Colorectal Cancer Reveals Response to Immunotherapy Spatial Immune Profiling in Metastatic dMMR CRC
              </a>
            </td>
          <td>
            Ilkyu Park, Hari Kang, Se-Hee Kim, YunJae Jung, Won-Suk Lee
          </td>
          <td>2026-01-09</td>
          <td>Immunotherapy Advances</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d45e59a2aedee088f21461bab887db43573ed97" target='_blank'>
              Transcriptomic profiling reveals prognostic gene signatures and immune landscape alterations in canine diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            L. A. Anaí, A. B. Cavalca, P. Xavier, L. Semolin, P. Jark, I. Senhorello, H. Fukumasu, Thiago Demarchi Munhoz, Marxa L. Figueiredo, Camila Cesário Fernandes, M. T. Costa, P. Kobayashi, Áureo Evangelista Santana, C. Fonseca‐Alves
          </td>
          <td>2026-01-07</td>
          <td>Veterinary Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="C1orf50 encodes a small, evolutionarily conserved protein, the function of which remains unclear. Its significance across various human cancers, particularly its specific role in ovarian cancer within an immunogenomic context, is not yet fully understood. Utilizing The Cancer Genome Atlas and single-cell RNA sequencing (scRNA-seq) public datasets, we conducted a comprehensive profiling of C1orf50 across multiple cancer types, with a particular focus on ovarian cancer, to investigate its associations with copy-number status, genomic instability, tumor programs, and the immune microenvironment. Across cancer types, copy-number gain or amplification of C1orf50 was most frequent in ovarian cancer and closely tracked with higher messenger RNA levels. Higher C1orf50 expression was associated with a greater tumor mutational burden and homologous recombination deficiency, as indicated by gene-set patterns that suggested heightened cell-cycle and cellular stress responses accompanied by reduced oxidative phosphorylation, enrichment of regulatory T cells, and depletion of resting memory CD4 T cells. In ovarian cancer, focal events at chromosome 1p34.2 were accompanied by stepwise increases in C1orf50 expression by clinical stage and were linked to higher tumor mutational burden, homologous recombination deficiency, and greater loss of heterozygosity, together with more frequent gene alterations in BRCA1 or BRCA2. Immune composition clustered into profiles consistent with an immunosuppressive context in tumors with higher C1orf50 expression. The scRNA-seq data further revealed that cancer cells enhanced immune-suppressive interactions with various immune cell populations and diminished antigen-presentation signals. Analyses of genomic instability in ovarian cancer suggested mutational processes compatible with base-substitution patterns associated with cytidine deaminase activity and with insertion-deletion patterns characteristic of homologous recombination failure, while transcript-level patterns pointed to a broad downshift of canonical DNA repair activity with apparent compensatory adjustments in related pathways rather than a uniform change in any single pathway. The overexpression of C1orf50 characterizes an aggressive immunogenomic phenotype in ovarian cancer, distinguished by genomic instability, impaired DNA repair mechanisms, and extensive immunosuppression. These findings indicate that C1orf50 warrants consideration as a potential biomarker and a prospective target for therapeutic investigation. Furthermore, they advocate for the progression to prospective validation and functional studies to ascertain its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214214b3f4bf431d9df6a0363a7ab05464474373" target='_blank'>
              Tight junction-high and CDH17-positive cell population is the source of colorectal cancer liver metastases.
              </a>
            </td>
          <td>
            Daniel Álvarez-Villanueva, Mariano R Maqueda, Dina Harti, E. Cantón, Evelyn Andrades, Joan Bertran, M. Martínez-Iniesta, Laura Solé, Violeta García‐Hernández, Ángela Montoto, T. Lobo-Jarne, J. Alonso-Marañón, Patricia Herrero-Molinero, Monica Larrubia-Loring, Anna Tramuns, Kieran Wynne, Indrani Bera, D. Matallanas, Alberto Villanueva, A. Bigas, Mar Iglesias, Lluís Espinosa
          </td>
          <td>2026-01-03</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background/Objectives: Head and neck squamous cell carcinoma (HNSCC) is a biologically heterogeneous malignancy with poor outcomes in advanced disease. Increasing evidence indicates that the tumor microenvironment, particularly cancer-associated fibroblasts (CAFs), plays an important role in tumor progression and immune regulation. However, the diversity of CAF subsets and their clinical relevance in HNSCC remain incompletely understood. This study aimed to characterize CAF heterogeneity and assess the prognostic significance of CAF subset-specific transcriptional programs. Methods: Single-cell RNA sequencing data from HNSCC tumors were analyzed to identify CAF subsets based on differentially expressed genes. CAF subset-specific gene signatures were used to construct prognostic risk models for overall survival (OS) and progression-free survival (PFS) in The Cancer Genome Atlas HNSCC cohort, with validation in an independent dataset. CAF-driven prognostic groups were defined, and their immune landscapes and biological pathways were evaluated. Bulk RNA sequencing of primary CAF cultures was performed for validation. Results: Six CAF subsets were identified, including myofibroblastic (myCAF), inflammatory (iCAF), antigen-presenting, and extracellular matrix-related CAFs. Risk scores derived from inflammatory CAF subsets consistently predicted shorter OS across independent cohorts, whereas PFS prediction showed greater cohort dependency. CAF-based stratification identified patient subgroups with distinct immune profiles and pathway enrichment patterns. These results were supported by validation analyses and by bulk RNA sequencing of primary CAFs, demonstrating preservation of myCAF- and iCAF-like transcriptional programs ex vivo. Conclusions: CAF heterogeneity has important prognostic and immunological implications in HNSCC. Inflammatory CAF-related transcriptional programs represent robust markers of patient survival and may complement tumor-intrinsic biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39c29db87141bfec7ddb84a086ca849e34133944" target='_blank'>
              Cancer-Associated Fibroblast Heterogeneity Shapes Prognosis and Immune Landscapes in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Hideyuki Takahashi, Hiroyuki Hagiwara, Hiroe Tada, Miho Uchida, T. Matsuyama, K. Chikamatsu
          </td>
          <td>2026-01-09</td>
          <td>Cancers</td>
          <td>0</td>
          <td>33</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025', '2026'],
    y: [0, 12, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>