<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-22 09:37:27 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>103</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/527b71b36a5f66efe4b636291f9add0899d1e6f5" target='_blank'>
              ROSIE-Enabled Spatial Mapping Reveals Architectural Fragmentation and Immune Reprogramming in Lung Adenocarcinoma Evolution
              </a>
            </td>
          <td>
            Beibei Huang, Bo Zhu
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f38657311790b6bfe59be5cdd1d26d9f9bbb5cc7" target='_blank'>
              Advancing small cell lung cancer metastasis research: innovations in preclinical mouse models
              </a>
            </td>
          <td>
            Qiqi Zhao, Liang Hu, Hongbin Ji
          </td>
          <td>2025-11-29</td>
          <td>Cancer and Metastasis Reviews</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927407a38e9474aa6152c45daf9e0743472dc6ee" target='_blank'>
              Cellular interactions in the sentinel lymph node predict melanoma recurrence
              </a>
            </td>
          <td>
            Sabrina M. Solis, Yang Yang, Yee Hoon Foong, Anushka Dheer, , M. S. Farooq, Shira G. Rosenberg, Sophie L. Gray-Gaillard, G. Karakousis, Xiaowei Xu, Rami S. Vanguri, , R. Herati
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d2c494a1cd657524f51f9577b21d3baa396ea7" target='_blank'>
              Spatiotemporal regulation of cell cycle states within the complex tumor microenvironment
              </a>
            </td>
          <td>
            Gianni Zanardelli, Olivier Tassy, Maulik K. Nariya, Nacho Molina
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, Ivan Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Spatial transcriptomics has emerged as a transformative technology in biomedical research, offering unprecedented insights into gene and protein expression within their native tissue context. Unlike conventional bulk or single-cell sequencing approaches, spatial omics has the advantage of preserving the spatial structure of tissues, allowing researchers to directly map molecular information onto histological structures. This review provides an overview of the current state of spatial omics technologies, highlighting their application in cancer research. Spatial omics has enabled detailed characterization of the tumor microenvironment (TME), revealing spatial heterogeneity, immune cell infiltration patterns, and complex mechanisms of tumor progression and therapy resistance across various cancer types. The review covers future directions, including artificial intelligence–driven analytics, improved standardization, and cost reduction to accelerate clinical translation. Ultimately, spatial omics is poised to play a central role in precision oncology, enabling a deeper understanding of tumor biology and informing more effective individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a77ce136fd5fd93d240b66bc2895fcb841db1de" target='_blank'>
              Spatial omics: applications and utility in profiling the tumor microenvironment
              </a>
            </td>
          <td>
            Ji-Eun See, Sarah Barlow, Wani Arjumand, Hannah DuBose, Felipe Segato Dezem, Jasmine T. Plummer
          </td>
          <td>2025-12-01</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="While advances in single-cell genomics have helped to chart the cellular components of tumor ecosystems, it has been more challenging to characterize their specific spatial organization and functional interactions. Here, we combine single-cell RNA-seq, spatial transcriptomics by Slide-seq, and in situ multiplex RNA analysis to create a detailed spatial map of healthy and dysplastic colon cellular ecosystems and their association with disease progression. We profiled inducible genetic CRC mouse models that recapitulate key features of human CRC, assigned cell types and epithelial expression programs to spatial tissue locations in tumors, and computationally used them to identify the regional features spanning different cells in the same spatial niche. We find that tumors were organized in cellular neighborhoods, each with a distinct composition of cell subtypes, expression programs, and local cellular interactions. Comparing to scRNA-seq and bulk RNA-seq data from human CRC, we find that both cell composition and layout features were conserved between the species, with mouse neighborhoods correlating with malignancy and clinical outcome in human patient tumors, highlighting the relevance of our findings to human disease. Our work offers a comprehensive framework that is applicable across various tissues, tumors, and disease conditions, with tools for the extrapolation of findings from experimental mouse models to human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af74f8f0f1a2428bab07b6671a3e436ab63d622f" target='_blank'>
              Spatially defined multicellular functional units in colorectal cancer revealed from single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            I. Avraham-Davidi, Simon Mages, J. Klughammer, Noa Moriel, Shinya Imada, Matan Hofree, Evan Murray, Jonathan Chen, Karin Pelka, Arnav Mehta, Genevieve M Boland, T. Delorey, Leah Caplan, Danielle Dionne, Robert Strasser, Jana Lalakova, Anežka Niesnerová, Hao Xu, Morgane Rouault, I. Tirosh, Hacohen Nir, Fei Chen, Omer Yilmaz, Jatin Roper, O. Rozenblatt-Rosen, Mor Nitzan, A. Regev
          </td>
          <td>2025-12-12</td>
          <td>eLife</td>
          <td>0</td>
          <td>197</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f7dcdd3510de0385797859e77f339a9cbb0192e" target='_blank'>
              Multidimensional Single-Cell Transcriptomic Profiling of Uterine Leiomyosarcomas Identifies Molecular Subtypes with Distinct Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Maria Korah, J. Agolia, Biren Reddy, R. Flojo, Amanda Gonçalves, Lilin Wang, Wesley Bobst, Brooke Liang, Kaylin A. Yip, Rosyli Reveron-Thornton, Chuner Guo, Robert Gruener, Andrea E. Delitto, Peter Y. Xie, Angela D. Tabora, Beatrice J. Sun, Sara K. Daniel, Babak Litkouhi, Oliver Dorigo, Nam Q. Bui, Minggui Pan, E. Moding, A. Kalbasi, Amanda R. Kirane, George A. Poultsides, Jeffrey A. Norton, Aaron M. Newman, G. Charville, K. Ganjoo, Deshka S. Foster, R. Stephanie Huang, Byrne Lee, Michael T. Longaker, Daniel J. Delitto
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Integration of scRNA-seq data from millions of cells revealed a high diversity of cell types in the healthy and diseased human lung. In a large and complex organ, constantly exposed to external agents, it is crucial to understand the influence of lung tissue topography or external factors on gene expression variability within cell types. Here, we apply three spatial transcriptomics approaches, to: (i) localize the majority of lung cell types, including rare epithelial cells within the tissue topography, (ii) describe consistent anatomical and regional gene expression variability within and across cell types, and (iii) reveal distinct cellular neighborhoods in specific anatomical regions and examine gene expression variations in them. We thus provide a spatially resolved tissue reference atlas in three representative regions of the healthy human lung. We further demonstrate its utility by defining previously unknown imbalances of epithelial cell type compositions in chronic obstructive pulmonary disease lungs. Our topographic atlas enables a precise description of characteristic regional cellular responses upon experimental perturbations or during disease progression. Using multiplexed spatially resolved transcriptomic approaches, the authors generate a topographic atlas of the healthy adult lung with location-related gene expression variability or neighbourhood changes as exemplified by COPD patient sample analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe80d855be30beaf44c2810f98505d59172a3d0" target='_blank'>
              Spatial single-cell atlas reveals regional variations in healthy and diseased human lung
              </a>
            </td>
          <td>
            A. Firsova, Sergio Marco Salas, L. B. Kuemmerle, X. Abalo, A. Sountoulidis, L. Larsson, Krishnaa T. Mahbubani, Jonas Theelke, Žaneta Andrusivová, Leire Alonso Galicia, A. Liontos, Tamás Balassa, Ferenc Kovács, Peter Horvath, Yuexin Chen, Janine Gote-Schniering, M. Stoleriu, Jürgen Behr, Kerstin B. Meyer, W. Timens, Herbert B. Schiller, Malte D. Luecken, F. Theis, J. Lundeberg, Mats Nilsson, M. Nawijn, Christos Samakovlis
          </td>
          <td>2025-11-05</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf21fbcae37403e9935121738032d46b9b3bab3c" target='_blank'>
              ProMPt: A modular preclinical platform for functional modelling of prostate cancer heterogeneity and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Nicole Pandell, Jacob Househam, Matteo Tartagni, Archana Thankamony, Florian Gabel, Roberto Rota, Elizabeth Flittner, B. Gurel, I. Figueiredo, G. Seed, A. Neeb, Mary Chol, W. Yuan, J. Clohessy, C. Tape, A. Sharp, J. D. De Bono, Marco Bezzi
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076f7d290241ef13795e81746d2f0f634b36afeb" target='_blank'>
              Star-Motifs: Revealing Single-Cell Spatiotypes from Routine Histology Using Star-Convex Neighborhoods
              </a>
            </td>
          <td>
            G. N. Gunesli, Muhammad Dawood, Lawrence S. Young, F. Minhas, S. E. A. Raza, N. Rajpoot
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background: Intratumour heterogeneity (ITH) is one of the key characteristics of cancer and is closely associated with patient prognosis, treatment resistance, and tumor metastasis. Nevertheless, the study of ITH in hepatocellular carcinoma (HCC) remains limited. Methods: The present study elucidated the influence of ITH on the tumor microenvironment (TME) in HCC. We applied Non-negative Matrix Factorization (NMF) analysis to a cohort of 78 single-cell RNA sequencing (scRNA-seq) HCC samples to systematically characterize ITH. Furthermore, by integrating spatial transcriptomics (ST) data from five HCC patients, we comprehensively analyzed the spatial organization and functional properties of distinct niches within HCC. We conducted a detailed analysis of the cell-type co-localization relationships within the TME and constructed a comprehensive atlas of HCC spatial organization. Results: We observed a co-localization relationship between hypoxia tumor cells, plasmalemma vesicle-associated protein (PLVAP+) endothelial cells (EC), and vascular endothelial growth factor A (VEGFA+) cancer-associated fibroblasts (CAF), suggesting a key role for hypoxia tumor cells in VEGFA+ CAF transformation and tumor angiogenesis. We identified a unique boundary region enriched with dendritic cells1 (DC1), interferon-expressing tumor cells, lymphatic EC, C–X–C Motif Chemokine Ligand 10 (CXCL10+) macrophages (Mac), and secreted phosphoprotein 1 (SPP1+) Mac located between the tumor-infiltrating immune cells and tumor regions. Furthermore, we found that CXCL10+ Mac and SPP1+ Mac, despite co-localizing in the boundary region, exhibit distinct functions, which may be attributed to their unique spatial locations, with the former being closer to the immune-infiltrated region and the latter more proximal to the tumor area. Conclusions: Our study highlights the critical role of spatial interactions between tumor cells and the microenvironment in HCC. The findings offer new insights into ITH and underscore the importance of spatial organization in understanding cancer biology and designing future precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e826ac6739dc6ba19b8b336aa7292a375efd8d3e" target='_blank'>
              Integrative Analysis of Single-Cell and Spatial Transcriptomics Reveals Intratumor Heterogeneity Shaping the Tumor Microenvironment in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-12-01</td>
          <td>Livers</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes TP53 and RB1. However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yueli Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Risk stratification remains a critical challenge in non-small cell lung cancer patients for optimal therapy selection. In this study, we develop an artificial intelligence-powered spatial cellomics approach that combines histology, multiplex immunofluorescence imaging and multimodal machine learning to characterize the complex cellular relationships of 43 cell phenotypes in the tumor microenvironment in a real-world retrospective cohort of 1168 non-small cell lung cancer patients from two large German cancer centers. The model identifies cell niches associated with survival and achieves a 14% and 47% improvement in risk stratification in the two main non-small cell lung cancer subtypes, lung adenocarcinoma and squamous cell carcinoma, respectively, combining niche patterns with conventional cancer staging. Our results show that complex immune cell niche patterns identify potentially undertreated high-risk patients qualifying for adjuvant therapy. Our approach highlights the potential of artificial intelligence powered multiplex imaging analyses to better understand the contribution of the tumor microenvironment to cancer progression and to improve risk stratification and treatment selection in non-small cell lung cancer. Risk stratification in non-small cell lung cancer (NSCLC) remains challenging. By combining multiplex immunofluorescence, H&E histology, and AI, the study identifies spatial “cell-niche” patterns that enhance survival prediction beyond UICC8 staging. These patterns reclassify many stage I patients as high risk, revealing potentially undertreated cases and establishing spatial tumor microenvironment features as clinically actionable biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53bd1a5c6021c6b38d3074a4cf442e836ada177" target='_blank'>
              AI-powered spatial cell phenomics enhances risk stratification in non-small cell lung cancer
              </a>
            </td>
          <td>
            Simon Schallenberg, Gabriel Dernbach, Sharon Ruane, Philipp Jurmeister, Cornelius Böhm, Kai Standvoss, Sandip Ghosh, M. Frentsch, M. Dragomir, Philipp Keyl, Corinna Friedrich, Il-Kang Na, Sabine Merkelbach-Bruse, A. Quaas, N. Frost, Kyrill Boschung, Winfried Randerath, G. Schlachtenberger, Matthias Heldwein, Ulrich Keilholz, Khosro Hekmat, Jens Rückert, R. Buettner, Angela Vasaturo, David Horst, Lukas Ruff, M. Alber, Klaus-Robert Müller, Frederick Klauschen
          </td>
          <td>2025-11-03</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b8f75faab2c4feb9da5df93770ed23963d387e7" target='_blank'>
              Nuclear Irregularity as a Universal Diagnostic Tool in Solid Tumors
              </a>
            </td>
          <td>
            Francesco Hamilton, Kane Foster, I. Ghobrial
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern, and a high relapse rate. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, where malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, we identify pericytes as the most active paracrine signaling hub within the tumor parenchyma. Their depletion through genetic engineering results in accelerated tumor progression and shortened survival. Mechanistic studies reveal that pericyte deficiency remodels the endothelium and impacts the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, the pericyte-deprived endothelium recruits perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages express Hepatocyte Growth Factor, which reinforces activation of its receptor tyrosine kinase MET on GBM cells harboring a pronounced mesenchymal subtype driven by the key phenotypic regulator Fosl1. Indeed, orthotopic implantation of MET-expressing GBM cells corroborates their superior tumor-initiating and invasive capabilities. Thus, pericytes represent critical modulators of GBM development by orchestrating a tumor-suppressive microenvironment, highlighting the importance of their preservation in therapy. Improved understanding of the tumor ecosystem in glioblastoma is critical for developing new treatment strategies for the disease. Here, the authors identify pericytes as an active paracrine signaling hub within the tumor parenchyma that orchestrates a tumor-suppressive microenvironment, suggesting pericyte preservation as a key feature of future therapeutic regimens for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c09a05790107e087a632df06714ceec3176e20b4" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, Mehrnaz Safaee Talkhoncheh, B. Phung, E. Cordero, R. Rosberg, E. Johansson, G. Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumors are composed of cellular populations with distinct genotypes and phenotypes, which dynamically evolve over time and during treatment. This process is known as clonal evolution, and it is difficult to reveal fine-scale clonal structure with traditional bulk sequencing. Although single-cell genome sequencing could enable reconstruction of tumor clonal evolution, it remains technically challenging and the number of single cells profiled is generally insufficient due to high cost. To address this issue, we developed scClone, a computational toolkit that integrates variant detection and genotype inference for single-cell RNA-seq (scRNA-seq) and spatial transcriptomic data. It further provides interactive visualization of clonal structure and dynamic evolution. scClone addresses key limitations inherent to scRNA-seq, such as expression drop-out and allelic imbalance, and incorporates cell type or state annotation with mutational signature analysis to enable comprehensive profiling of tumor heterogeneity. scClone demonstrated robust performance across multiple datasets—generated from both full-length and fragmented RNA sequencing—by accurately reproducing mutation profiles and resolving clonal mixtures in myeloma, hepatocellular carcinoma and pancreatic cancer. Additionally, scClone has been applied to spatial transcriptomics, enabling the delineation of clonal structures within histological sections from ovarian cancer and cutaneous squamous cell carcinoma. In summary, our results demonstrate that scClone can extract genetic information from scRNA-seq datasets, thereby successfully establishing genotype–phenotype associations at the single-cell level and providing insights into the clonal evolution of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d470f131086238e06c10a786c9bfc7fb86a41116" target='_blank'>
              De Novo Detection of Clonal Structure and Evolution in Single-Cell and Spatial Transcriptomes
              </a>
            </td>
          <td>
            Shihao Bai, Xianbin Su, Ziyao Chen, Ze-Guang Han
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Mouse models are pivotal for translational research, enabling the detection of therapeutic-induced immune modifications within the tumor microenvironment (TME). We utilized a 40-marker panel targeting tumor and immune markers to investigate the spatial protein composition of the whole mouse embryo, mouse brain and mouse glioblastoma (GBM) tumor sections. Imaging Mass Cytometry™ (IMC™) technology was employed to capture the expression of 40 markers simultaneously while preserving tissue integrity and enabling a flexible workflow without autofluorescence artifacts. Data analysis revealed distinct spatial patterns of tumor and immune markers across the GBM, identifying key features such as necrotic cores, regions of high immune infiltration, proliferating tumor cells and diverse populations of infiltrating immune cells. The wide detection range of expression provided by IMC technology enabled visualization and analysis of clusters exhibiting varying levels of key GBM markers, suggesting a probable dual origin of the GBM, coexistence of pro- and anti-tumorigenic immune responses and overall extreme heterogeneity that can only be visualized with high-plex antibody panels. This approach demonstrates the capability of IMC technology to reveal the complex cellular and structural spatial landscape of the TME in GBM, providing valuable insights into tumor biology and potential therapeutic targets for patient stratification. For Research Use Only. Not for use in diagnostic procedures.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d885a41b02f8da25eb39b11abd7a0e0030ab5b6" target='_blank'>
              Characterization of cellular diversity and immune landscape in Mouse Glioblastoma using whole slide imaging modes of high-plex Imaging Mass Cytometry 2619
              </a>
            </td>
          <td>
            Qanber Raza, Nick Zabinyakov, T. Pfister, Nikesh Parsotam, David Howell, Liang Lim, James Pemberton, Yun-Jyh Chwee, C. Loh
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34b247d052e44baf977776483c7e044dbe0ae158" target='_blank'>
              Mechanisms of tumor persistence in metastatic melanoma following successful immunotherapy
              </a>
            </td>
          <td>
            Yingxiao Shi, Z. Maliga, Tuulia Vallius, Shishir M. Pant, R. Pelletier, Brigette Kobs, Solanky Priyanka, Yiwen He, Eliezer Van Allen, S. Santagata, P. Ott, C. G. Lian, Elizabeth I. Buchbinder, David Liu, P. Sorger
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2a93ffa8be7e0912bd5d9f29072cbe15c529994" target='_blank'>
              Inferring cellular communication through mapping cells in space using Tangram 2
              </a>
            </td>
          <td>
            Hejin Huang, Alma Andersson, Sunny Z. Wu, Gabriele Scalia, Jenna Collier, K. H. Hoi, Sören Müller, Graham Heimberg, Hector Corrada-Bravo, Shreya Gaddam, Jan-Christian Huetter, Shannon J. Turley, David Richmond, Aicha Bentaieb, Tommaso Biancalani
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 A major barrier in understanding adult diffuse gliomas lies in the inability to track tumor evolution in real time. To address this, we employed a novel method known as continuous lineage tracing, which integrates CRISPR/Cas9-based expressed DNA barcoding with single-cell RNA sequencing, enabling a phylogenetic approach to studying tumor development. Patient-derived glioma-initiating cell lines were engineered with expressed barcodes targeted by CRISPR/Cas9 and implanted into mice to create intracranial xenografts. Tumors underwent single-cell RNA sequencing; expressed barcodes were used to infer clonal relationships, and transcriptomic profiles enabled cell state classification. Phylogenetic lineage trees were reconstructed using lineage inference algorithms to characterize cell fitness, expansion, and plasticity. Our analysis uncovered extensive intra-clonal cell state heterogeneity, indicating active phenotype switching prior to therapy. We identified consistent transcriptional programs associated with tumor engraftment and in vivo clonal advantage. Lineage tracing revealed gene expression signatures linked to key phenotypes: fitness, enriched for neural-mesenchymal and injury-response pathways; expansion, associated with RNA splicing; and plasticity, correlated with cell cycle and DNA repair programs. Glioma stem cells appeared to span a transcriptional continuum from undifferentiated, high-fitness states to more differentiated, low-fitness states, with high-fitness cells potentially representing transitional phenotypes. We then validated these findings in a cohort of 185 surgically resected gliomas with matched bulk RNA and proteomic data. Phylogenetic gene signatures differed markedly between IDH-mutant and IDH-wildtype tumors. When stratified by tumor type, both fitness and expansion signatures were significantly associated with overall survival in GBM (median 12.6 months, p=0.041) and oligodendrogliomas (median 66 months, p=0.027). This study demonstrates the utility of continuous lineage tracing to reconstruct tumor evolution and identify transcriptional programs linked to tumor growth and prognosis. Our approach provides a powerful framework for dissecting glioma biology and identifying potential therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d67c2559a40c369b3a39bc8fe0aad62196164a" target='_blank'>
              STEM-24. SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS, EXPANSION AND PLASTICITY AXES IN ADULT DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Ajisebutu, Xiaojun Fan, Chloe Gui, Jeff Liu, Vikas Patil, A. Landry, Y. Ellenbogen, L. Yefet, Phooja Persaud, F. Nassiri, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor heterogeneity drives drug resistance and relapse, influencing immune evasion and tumor progression. While intratumor heterogeneity has been extensively studied at the genomic level, its functional outcomes and interactions with the tumor microenvironment remain underexplored. In contrast, the functional outcome of heterogeneity and the interplay with the tumor microenvironment have not been addressed. In this study, we integrate multi-region spatial MS-based proteomics of 280 tumor regions, exome sequencing, and imaging to investigate spatial proteomic heterogeneity in breast cancer. Our findings reveal increased proteomic heterogeneity with tumor progression, independent of genomic heterogeneity but closely associated with microenvironmental differences. Integration with immune and stromal imaging highlighted a dynamic interplay where low-grade tumors exhibit constrained immune infiltration, and upon progression to higher grades, macrophages and T cells infiltrate. However, anti-inflammatory pathways involving kynurenine and prostaglandins are more highly expressed in infiltrated regions, suggesting that anti-tumorigenic activities are inhibited. Integration with the global protein network provides potential targetable mediators of immune evasion in breast cancer that can serve as the basis for future development of personalized breast cancer therapies. Intratumor heterogeneity (ITH) has been mainly studied at the genomic level. Here, the authors integrate multi-region proteomics of 280 tumor regions from 33 patients, exome sequencing, and imaging-based immune and stromal profiling to investigate functional spatial proteomic ITH in breast cancer and the interactions of cancer cells and the tumor microenvironment during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d48d5d89f33d4d9425895ba820536d951749d4" target='_blank'>
              Integrated spatial proteomic analysis of breast cancer heterogeneity unravels cancer cell phenotypic plasticity
              </a>
            </td>
          <td>
            M. Mardamshina, Shiri Karagach, Vishnu Mohan, Gali Arad, Daniela Necula, O. Golani, Liat Fellus-Alyagor, Anjana Shenoy, Kateryna Krol, Daniel Pirak, Nitay Itzhacky, I. Marin, Bruria Shalmon, Y. Addadi, Roded Sharan, E. Gal-Yam, Iris Barshack, Tamar Geiger
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Spatial transcriptomic technologies are transforming our ability to profile gene expression, by mapping it to the native architecture of the tissues. However, the inherent complexity and multimodality of the problem demand robust analytical computational tools. In this study, we benchmarked seven state-of-the art clustering algorithms that cover different methodological approaches-conventional statistics, Machine Learning, and Deep Learning-based on their capacity to integrate gene expression, histology, and spatial proximity to identify spatial domains in spatial transcriptomic data. Our comparative analysis implemented on a wellannotated human brain reference dataset shows that algorithms that integrate spatial adjacency (GraphST and BayesSpace) significantly outperform the others, with GraphST achieving the best performance. As a proof of principle, we applied GraphST to a highgrade serous ovarian cancer sample. The model revealed transcriptionally distinct subregions within tumor and stromal compartments not distinguishable by histological assessment alone. Furthermore, by integrating cell-type deconvolution with spatial T-cell receptor (TCR) clonotyping, we discovered heterogeneous immune niches, characterized by domain-specific clonal expansion. Tumor cores exhibited highly diverse TCR repertoires, while stromal margins were dominated by a few expanded clones, suggesting differential immune-selection pressure. These findings highlight the potential of spatially informed Deep Learning methods to dissect the tumor microenvironment and identify actionable targets for immune-based anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef571e385b61839fb9ba7057c5643bbdab41a16c" target='_blank'>
              Deep Learning Models that Integrate Transcriptomic and Spatial Information, Allow Efficient Reconstruction and Clonal TCR Analysis of the Tumor Microenvironment
              </a>
            </td>
          <td>
            Pedro Castillo-Rosique, Enric Vercher, Angel M. Mártinez, Iván Cortés-Domínguez, Sandra Hervás-Stubbs, Carlos Ortiz-de-Solórzano
          </td>
          <td>2025-11-06</td>
          <td>2025 IEEE 25th International Conference on Bioinformatics and Bioengineering (BIBE)</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bdb3fc96051ec080fed6bcf88bde02391e145c9" target='_blank'>
              Integrative spatial multi-omics reveals prognostic tumor niches in female genital tumors
              </a>
            </td>
          <td>
            Tian Xu, Siyu Lin, Yuling Wu, Yufeng He, Zongxu Zhang, Yanping Zhao, Peng Zhang, Luyang Zhao, Wen Yang, M. Ye, Rui Zhang, Liang Wen, Pengfei Ren, Ce Luo, Zhe Zhang, Zexian Zeng, Yuanguang Meng
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fa5a90466cca365ac23cb5111624676a9f4bea9" target='_blank'>
              Deciphering Cellular Ecosystems Driving Tumor Progression and Immune Escape from Spatial Transcriptomics and Single-Cell with COMPOTES
              </a>
            </td>
          <td>
            L. Herpin, Anaïs Chossegros, Roberta Codato, Josep Monserrat Sanchez, Jean El Khoury, Simon Grouard, V. Ducret, Alex Cornish, Baptiste Gross, Elodie Pronier, C. Hoffmann, Alberto Romagnoni, Eric Y. Durand, Almudena Espin Perez, Quentin Bayard
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Spatial context is a fundamental driver of intra-tumor heterogeneity in glioblastoma (GBM), shaping the molecular landscape, cellular composition, and therapeutic vulnerabilities. While single-cell profiling has illuminated the diversity of GBM cell states, a critical gap remains in understanding how local microenvironments orchestrate gene expression programs and cellular plasticity within intact tumors. To address this, we analyzed spatial transcriptomics data from 44 GBM specimens, comprising over 128,000 tissue spots, to systematically map the biological programs and cellular architecture of distinct tumor niches. Leveraging machine learning, we derived a unified batch-adjusted data embedding that captured the local tissue context of each spot across all slides, allowing the study of spatially continuous rewiring of molecular programs. This approach enabled us to identify ligand–receptor interactions and transcription factors (TF) with spatially localized activity across tissue slides. Using cross-correlation metrics, we identified multiple TF modules specifically active in certain environments. Notably, the perivascular region exhibited the most diverse transcriptional activity, with multiple TF modules associated with tumor growth, division, and migration processes. Additional modules were uniquely enriched at the leading edge and hypoxic regions, as well as at the tumor–necrosis interface. Importantly, we identified a unique module active at the tumor core–leading edge interface, with corresponding spots enriched for alternative splicing processes, suggesting this could be a core driver of the invasive tumor phenotype. Dynamic gene network inference along a continuous hypoxic gradient uncovered context-specific hub genes and TFs that facilitate tumor metabolic adaptation and cell state shifts. These findings underscore the immense spatial heterogeneity of GBM, characterized by context-specific molecular rewiring of tumor cells driven by complex cellular interactions. Our study not only reveals new vulnerabilities but also highlights the need for spatially targeted combination therapies to disrupt the adaptive spectrum of cancer cell states in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fddeac5d22fe2de68ff3fa47ac318949e6c8ee0" target='_blank'>
              TMIC-118. Identifying spatially-localized biological programs and molecular vulnerabilities in glioblastomas
              </a>
            </td>
          <td>
            V. Ravikumar, Reva Kulkarni, A. Maddox, Arushi Tripathy, Emre Kiziltug, Dah-Luen Huang, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cervical cancer ranks as the fourth most common malignancy among women worldwide, with approximately 604,000 new cases and 342,000 deaths annually. Despite advances in HPV vaccination and screening programs, significant challenges remain in diagnostic accuracy, therapeutic efficacy, and prognostic assessment. Traditional bulk RNA sequencing methods average gene expression data across entire tissue specimens, masking cellular diversity and spatial architectural patterns that influence tumor progression. Telomere maintenance mechanisms play crucial roles in cervical cancer development, with over 90% of cervical cancers exhibiting telomerase reactivation. This study integrated single-cell RNA sequencing and spatial transcriptomics technologies to construct the first comprehensive spatial molecular atlas of cervical cancer. Single-cell RNA sequencing analysis utilized public datasets from the GEO database with systematic quality control through a multidimensional assessment framework. Spatial transcriptomics analysis employed the 10x Genomics Visium platform, successfully identifying 38 distinct cellular neighborhoods (N1-N38). Spatial expression patterns of 10 key genes were analyzed, including epithelial markers (MUC1, CDH1, KRT16), stromal markers (COL1A1, COMP, DCN), and immune markers (CD3G, FCGR1A). In vitro validation was performed using HeLa and SiHa cell lines. Multiple distinct cell subpopulations and 38 cellular neighborhoods with unique molecular characteristics were successfully identified. MKI67 demonstrated spatial heterogeneity with proliferative “hotspots”; COMP was primarily expressed in stromal regions and participated in tumor-stroma interactions; KRT16 exhibited patterns reflecting epithelial differentiation gradients. Different neighborhoods showed enrichment for various cell types, including immune hotspots, stromal-rich regions, and epithelial-dominant areas. Immunoglobulin-related genes (IGLC2, IGHG1, IGHG2, etc.) displayed unique spatial expression characteristics. Quantitative PCR validation confirmed differential expression patterns between cell lines. This study established the first comprehensive spatial transcriptomic atlas of cervical cancer, revealing unprecedented insights into tumor microenvironment organization and cellular spatial relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a6a98dd06253f9d1bf4fd4787fb52f0b0c3d2f1" target='_blank'>
              Spatial transcriptomic landscape and cellular neighborhood heterogeneity in cervical cancer: integrative single-cell and spatial RNA sequencing analysis
              </a>
            </td>
          <td>
            Dieyi Mo, Huana Cao, Shaojuan Lai, Jinding Luo, Weijie Ye
          </td>
          <td>2025-11-05</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Immune checkpoint inhibitors (ICI) have demonstrated prolonged efficacy in certain melanoma patients, yet a significant portion of patients do not experience clinical improvement, with the mechanisms underlying this resistance still not fully understood. Using established cell markers, we partitioned the single‐cell transcriptome into clusters, finding a notable link between NK cells and patient response to immunotherapy. We further identified four distinct subpopulations of NK cells, profiling marker gene sets and unique biological functions associated with each subpopulation. This analysis provides insights into the trajectory of NK cell development and differentiation, along with identifying the transcription factors driving these processes. The study pinpointed NK cluster 01 as pivotal in influencing patient sensitivity and prognosis during immunotherapy. Single‐cell transcriptome and spatial transcriptomics (ST) analysis revealed the proximity of NK cluster 01 cells to melanoma cells, hinting at a potential regulation of cell‐cell interaction via the IFN‐II signaling pathway network. ST analysis revealed the spatial arrangement and interaction of NK cluster 01 cells with melanoma cells. This study explores the feasibility of targeting NK cluster 01 cells with small molecule drugs via molecular docking, offering a promising approach to bolster the clinical utility of NK cell therapy. We comprehensively analyze the heterogeneity of NK cells within melanoma, elucidate the potential regulatory interactions between NK cells and other microenvironmental components, and establish a basis for the future clinical utilization of distinct NK cell subsets as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fbdc1aaba32b7600dd3221ae61c5e31ee96bd3" target='_blank'>
              Integrating Single‐Cell and Spatial Transcriptomics Reveals NK Cell Subpopulations Associated With Immunotherapy for Melanoma
              </a>
            </td>
          <td>
            Zhicheng Hu, Yongfei Chen, Hao Yang, Qiuming Pan, Hongrui Li, Shuting Li, Junxi Wang, Yudi Huang, Guanglong Huang, Shanqiang Qu
          </td>
          <td>2025-12-01</td>
          <td>Smart Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, Christin-Therese Müller, Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, M.-R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Introduction Breast cancer (BC) prognosis is fundamentally dictated by its pronounced molecular and cellular heterogeneity. While single-cell RNA sequencing (scRNA-seq) has detailed the tumor microenvironment (TME) cellular composition, a critical knowledge gap persists regarding the vascular heterogeneity of tumor endothelial cells (TECs) across different BC subtypes. Current TEC classification remains conventional, obscuring whether subtype-specific TECs possess unique transcriptomic signatures that influence treatment response, particularly to anti-angiogenic therapies. Given the pivotal role of tumor vasculature in proliferation and metastasis, and the existing research bias towards Triple-Negative BC (TNBC), a comprehensive investigation into the complexity of the vascular system is warranted. Furthermore, as lymph node metastasis (LNMT) is a major determinant of cancer mortality, distinguishing the endothelial and immune cell subtypes within LNMT microenvironments from primary tumors is essential for clarifying pathogenesis. Methods To address this, we integrated scRNA-seq data from primary BC tumors, ER_LN and healthy tissues, generating a robust dataset of 98,000 cells (~ 12 samples). Employing single-cell transcriptomics, spatial transcriptomics, and immunohistochemistry, we precisely delineated the endothelial cell (EC) heterogeneity within matched tumor and peri-tumoral tissues and scrutinized their intricate interactions with immune populations. Results We identified two previously uncharacterized, tumor-enriched endothelial cell subtypes, designated EC4 and EC5, which demonstrate subtype-specific functional adaptations and prognostic significance. EC4 cells, highly prevalent across BC, are principally characterized by antigen presentation, immune cell recruitment, and pro-inflammatory signaling. Conversely, EC5 cells, also enriched in BC, exhibit robust extracellular matrix (ECM) remodeling and potent tumor angiogenesis. We further characterized the functional divergence of EC4 and EC5 within ER tumors relative to HER2 tumors and ER_LN metastases. Our analysis reveals conserved endothelial programming mechanisms across BC subtypes, coexisting with distinct TME-driven transcriptional adaptations. Importantly, interactome analysis highlighted novel and subtype-specific communications between these novel EC subsets and immune cells, particularly CD8+T cells and macrophages. Experimental validation demonstrates that ECs overexpressing APP can mediate the M2 polarization of macrophages, underscoring diverse immunomodulatory roles for EC subsets across different BC contexts. Conclusions These findings offer critical and granular insights into the complex interplay between novel EC subtypes and the immune microenvironment in BC progression and metastasis. We establish that ECs are active and heterogeneous modulators of the TME, identifying specific therapeutic vulnerabilities within the tumor vasculature and providing a foundational blueprint for developing future precision immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67936270be24af1f65608417190d3a4e352d6fe3" target='_blank'>
              Single-cell profiling identifies heterogeneity of the immune microenvironment in healthy, primary and lymph node metastatic BC
              </a>
            </td>
          <td>
            Chuyu Liu, Feng Zeng, Shanshan Gao, Nengying Zhang, Yulan Cai
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Cancer is not composed of a single cell type but rather a complex community of cancer cells, immune cells, and supporting stromal cells that communicate with each other. These cellular conversations shape how each cancer grows, spreads, and responds to treatment. In this study, we compared single-cell sequencing data from seven different human cancers to explore how these cell interactions differ among tumor types. We found that pancreatic cancer contains many neutrophils, a type of immune cell that interacts mainly with other immune cells, while liver cancer lacks the usual fibroblast support cells. In contrast, esophageal and breast cancers were rich in fibroblasts that send growth signals to tumor cells, and thyroid cancer retained genes that may slow tumor progression. These differences help explain why some cancers behave more aggressively than others. Our findings provide a clearer picture of how the tumor environment varies among cancers and may guide the development of new strategies to treat solid tumors by targeting their surrounding cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95ce9b5897ed438ba40418ac1eb1c9c140e7a59a" target='_blank'>
              Comparative Oncology: Cross-Sectional Single-Cell Transcriptomic Profiling of the Tumor Microenvironment Across Seven Human Cancers
              </a>
            </td>
          <td>
            Riku Okamoto, Kota Okuno, Akiko Watanabe, Kanako Naito, Hiroyuki Minoura, Shumpei Shibaki, Kyonosuke Ikemura, Keiko Oki, Yu Kuroda, Shiori Fujino, Yusuke Nie, Nobuyuki Nishizawa, Eiichiro Watanabe, Mariko Kikuchi, Koshi Kumagai, Takahiro Yamanashi, Hiroshi Katoh, Hajime Takayasu, Takeo Sato, Takafumi Sangai, Yusuke Kumamoto, T. Naitoh, Naoki Hiki, Keishi Yamashita
          </td>
          <td>2025-10-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a05de91b68f189c5a7a8fad28c43d535bcdc834" target='_blank'>
              A spatial atlas of colorectal cancer reveals the influence of stromal niches on tumour differentiation
              </a>
            </td>
          <td>
            Andrew D. Pattison, Rebekah M. Engel, Wing Hei Chan, Spencer Greatorex, David Nickless, Liam Skinner, Camilla Cohen, Julie Hickey, Christine Georges, Anne L. Fletcher, P. McMurrick, Thierry Jardé, Lochlan Fennell, Helen E. Abud
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) remains a leading cause of cancer mortality, with liver metastasis being the principal determinant of poor prognosis, but the spatial mechanisms orchestrating metastatic niches remain elusive. Method To dissect the molecular and spatial dynamics of CRC progression, we constructed an integrative atlas using 35 single-cell RNA-seq datasets and spatial transcriptomics from primary tumors, liver metastases, and matched normal tissues. Malignant epithelial subpopulations were stratified via inferCNV and CytoTRACE analyses. Stromal-tumor interactions were dissected using CellChat and NicheNet, with functional validation through in vitro co-culture and immunohistochemistry. Result We identified a transcriptionally distinct epithelial subpopulation, termed high-malignancy CRC (High-M CRC), enriched in metastatic lesions and characterized by enhanced stemness, MYC-driven transcriptional activity, and glycolytic reprogramming. Stromal-tumor interaction analyses revealed that cancer-associated fibroblasts (CAFs), particularly matrix CAFs (mCAFs), promote malignant progression via the HGF-MET-MYC signaling axis. Spatial transcriptomic mapping confirmed the physical proximity and molecular co-localization of High-M CRC cells and mCAFs, along with enriched glycolysis and MYC expression at the cell-cell interface. In vitro functional validation demonstrated that CAF-derived HGF activates MET-MYC signaling in CRC cells, enhancing their invasion and proliferation—effects reversible by MET knockdown. Conclusion We unveil a spatially organized metabolic niche driven by stromal-tumor HGF-MET-MYC signaling. These findings offer novel insights into the stromal-tumor interaction and suggest actionable targets for therapeutic intervention in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dc67b8fc38aa04a6e00b35ef87b6cb65aaaec9" target='_blank'>
              A spatially resolved single-cell landscape of colorectal cancer liver metastasis reveals a stromal-tumor glycolytic signaling interaction
              </a>
            </td>
          <td>
            Jiahui Chen, Zukai Wang, Bingwang Zhu, Guoxian Guan
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT The metabolic programs of immune cells influence their activation, differentiation, and effector functions. While much of immunometabolism has focused on cell‐intrinsic regulation, it is now clear that metabolic activity is profoundly influenced by the surrounding tissue environment. In tumors and other inflammatory settings, immune cells are shaped by nutrient gradients, hypoxia, and immunoregulatory metabolites, factors that are spatially heterogeneous and often poorly captured by traditional methods. This review highlights recent technological advances that enable spatially resolved analysis of immune metabolism, with an emphasis on multimodal integration and cancer as a model system. Mass spectrometry imaging (MALDI, DESI), high‐resolution platforms like SIMS, and vibrational imaging approaches such as Raman microscopy enable direct visualization of metabolites in tissue. Transcriptomic and proteomic data can be used to infer metabolic states, and computational models are being developed to integrate these diverse data layers. Together, these technologies are transforming the study of immunometabolism from dissociated cells to the intact tissue context. Key challenges remain in resolution, annotation, and data integration, but spatial immunometabolism holds particular promise for illuminating mechanisms of immune regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4ef9dd08ead570796d0b86fc9dc291d59a2835" target='_blank'>
              Spatial Immunometabolism: Integrating Technologies to Decode Cellular Metabolism in Tissues
              </a>
            </td>
          <td>
            F. Hartmann
          </td>
          <td>2025-11-01</td>
          <td>European Journal of Immunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor vascular endothelial cells are essential constituents of the tumor microenvironment, responsible for delivering oxygen and nutrients that are vital for tumor growth and proliferation. As a hallmark of cancer progression, abnormal tumor vasculature contributes to tumor development through multiple mechanisms. Although anti-angiogenic therapies are widely used in the treatment of various cancers, the intrinsic heterogeneity of endothelial cells poses significant challenges regarding therapeutic efficacy. Therefore, further investigation into the heterogeneity of tumor endothelial cells is of paramount importance. The rapid advancement of single-cell sequencing technologies in recent years has facilitated the detailed characterization of heterogeneity among tumor endothelial cells at the single-cell level, thereby fostering a more precise understanding of the functional roles of individual cells within the tumor microenvironment. This technology has become an indispensable tool for investigating the heterogeneity of tumor endothelial cells, offering insights that could inform the refinement of future cancer treatments. In this review, we synthesize findings from the field of single-cell omics to elucidate the heterogeneous characteristics of tumor endothelial cells. We analyze recent advancements in single-cell technology used in the study of tumor cell heterogeneity in terms of both commonalities and distinctive features, covering aspects at the gene and cellular levels. In this review, we provide an overview of recent applications of single-cell sequencing technology in analyzing tumor endothelial cell heterogeneity, offering insights into the development of precise tumor therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a50144e46bbcc6cadd8fab9df5c73d57f4ff4f3" target='_blank'>
              Research Progress Regarding the Use of Single-Cell Sequencing Technology in Analyzing Tumor Endothelial Cell Pathophysiology
              </a>
            </td>
          <td>
            Shu Zhao, S. Liu, Wenxin Shao, Dong Liu
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Thyroid cancer, the most common endocrine malignancy, is characterized by a unique and complex tumor microenvironment (TME). To unravel the high tumor heterogeneity and molecular mechanisms driving cancer progression, we performed single-cell RNA sequencing (scRNA-seq) analysis, enabling a comprehensive exploration of cellular diversity and molecular dynamics at single-cell resolution. We employed Principal Component Analysis (PCA) and Uniform Manifold Approximation and Projection (UMAP) for dimensionality reduction and subsequent identification of cellular clusters. Differential gene expression analysis across subclusters was conducted using the FindAllMarkers function, while the DoHeatmap function was utilized to visualize the distribution of differentially expressed genes. The AUCell algorithm was applied to evaluate pathway enrichment within specific cell subtypes. To decipher cellular communication networks, we integrated the CellChat and NicheNet algorithms, which revealed intricate intercellular signaling interactions. Finally, multiplex immunohistochemistry (mIHC) was performed to validate key cellular interactions identified in silico. By analyzing 405,077 single cells from 50 thyroid cancer samples (including papillary, anaplastic, and metastatic tumors) and 14 normal thyroid tissues, we identified four major cellular subpopulations through unbiased clustering based on gene expression patterns and representative cellular markers. The TME was found to encompass diverse immune, endothelial, and mesenchymal cell subtypes, including novel populations such as CD4 + HSPA1A + T cells. Functional pathway enrichment analysis highlighted the roles of abundant cell types in tumor progression. Cell-cell communication analysis uncovered potential immunotherapeutic targets and revealed critical crosstalk among hub niche cells, including APOE+ macrophages, EMT-like cancer-associated fibroblasts (CAFs), and RBP7+ endothelial cells. These findings were further validated by multiplex immunohistochemistry, confirming the spatial organization and interactions of these cell populations within the TME. Our study provides a comprehensive single-cell transcriptomic atlas of thyroid cancer, offering profound insights into tumor heterogeneity, the functional roles of key niche cells, and potential biomarkers for anticancer therapy. These findings not only enhance our understanding of thyroid cancer biology but also pave the way for the development of novel therapeutic strategies targeting the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b009366b4ed17ce1b669fc0314c882a610b52822" target='_blank'>
              Comprehensive single-cell RNA analysis reveals intertumoral microenvironment heterogeneity and hub niche of carcinogenesis in thyroid cancer
              </a>
            </td>
          <td>
            Hao Xu, Junjie Ma, Nanjun Li, Haoxiang Xuan, Shijie Lin, Yunuo Shen, Lu Yin, Ping Wang, Qianhui Xu
          </td>
          <td>2025-11-26</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Recent single-cell studies implicate the glutamatergic lineage of the rhombic lip (RL) as the origin of group (G) 3/4 medulloblastomas (MBs) and a stalled differentiation program as underlying their malignant transformation. It is unknown, however, if G3/4 MBs exhibit cellular plasticity as has been observed in glioma. The effect of therapy on cellular hierarchies in G3/4 MBs is unknown. We profiled N=57 G3/4 MB frozen specimens obtained from the University of California, San Francisco Brain Tumor Center and the Children’s Brain Tumor Network biobanks via single-nucleus RNA and epigenetic sequencing. This cohort included 43 diagnostic and 14 patient-matched recurrent specimens. We profiled N=32 FFPE specimens via Xenium single-cell spatial transcriptomics. We performed genetic lineage tracing in intracranial implants of G3 MB cell lines derived from primary and metastatic disease using engineered barcodes, and in clinical specimens using endogenous mutations. Remarkably, xenografts from primary tumors produced similar cell types, in similar proportions to those found in treatment-naive human MBs. Moreover, the distribution of cell types found in xenografts was conserved at the level of individual clones, indicating cell-intrinsic regulation of tumor composition. Single-cell phylogenetic analysis demonstrated significant rates of dedifferentiation events from unipolar brush-like cells to RL-progenitor phenotypes. This plasticity was increased in recurrent specimens and in metastatic models compared to models and samples derived from primary disease. Samples with higher plasticity exhibited greater spatial heterogeneity, while samples with lower rates of dedifferentiation had more structured spatial organization. These studies show that G3/4 MB cellular composition is homeostatic in treatment-naive disease and conserved at the level of individual clones. Yet, these tumors exhibit plasticity that is exacerbated by therapy and affects spatial organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7150b972b545d28e4de15c0584606a43a9a4dc" target='_blank'>
              STEM-18. GENETIC LINEAGE TRACING OF MEDULLOBLASTOMA UNDER THERAPY WITH SPATIAL CONTEXT
              </a>
            </td>
          <td>
            Bohyeon Yu, Jangham Jung, Abhinav Jain, Emanuele Rosatti, Joanna Phillips, Aaron A. Diaz
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor heterogeneity is a fundamental property of leukemia and lymphoma that drives disease progression, relapse, and therapy resistance. For decades, bulk genomic and transcriptomic analyses have provided a foundational understanding of these malignancies but have obscured the cellular diversity within the tumor ecosystem. The advent of high-throughput single-cell technologies, particularly single-cell RNA sequencing (scRNA-seq), has revolutionized our capacity to deconvolute this complexity. This review synthesizes how single-cell analyses are reshaping our understanding of tumor heterogeneity in hematological cancers. We explore the role of intra-clonal genetic diversity in Acute Myeloid Leukemia (AML) and B-cell Acute Lymphoblastic Leukemia (B-ALL), the functional and transcriptional plasticity of subpopulations, and the dynamic interplay between malignant cells and the immune microenvironment in both leukemia and lymphoma. We further highlight how the tumor microenvironment in lymphomas fosters heterogeneity and immune evasion. Furthermore, the integration of single-cell data with spatial transcriptomics and computational models is beginning to predict evolutionary trajectories, offering a window into future clinical behaviors. Critically, we discuss how single-cell approaches have identified pre-existing or emergent resistant subclones under therapeutic pressure, providing a mechanistic basis for treatment failure. The clinical implications are profound, paving the way for single-cell diagnostics for minimal residual disease (MRD) monitoring and the identification of novel therapeutic targets. We conclude that single-cell multi-omics is an indispensable tool for unraveling the multifaceted nature of tumor heterogeneity, with the potential to propel the field toward more precise and effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0baf5679635e6fcbb86a03a048b12087069db02" target='_blank'>
              Tumor Heterogeneity in Leukemia and Lymphoma: Insights from Single-Cell Analyses
              </a>
            </td>
          <td>
            Pang Hou, Wen
          </td>
          <td>2025-11-21</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Deciphering the spatial biology of the tumor microenvironment (TME) can inform researchers about the likelihood of therapeutic success. Of particular interest is the spatial organization of immune cells, especially T cells, which can undergo exhaustion resulting in reduced effector functions due to expression of inhibitory markers such as PD-1. Imaging Mass Cytometry™ (IMC™) technology is a quantitative multiplexed spatial imaging technique for assessment of 40-plus biomarkers simultaneously on the same tissue slide without signal amplification or autofluorescence interference. We used IMC technology to investigate the spatial distribution and phenotypic characteristics of immune cells, focusing on exhausted T cells in the TME of multiple cancers. Various IMC imaging modes revealed striking heterogeneity of the TME, including clusters of lymphocytes at the tumor margins, indicative of immune surveillance. Pixel-clustering and single-cell analyses delineated distinct tumor areas based on extent of T cell exhaustion, immune cell density, cell proliferation and stromal components. These findings illustrate the power of IMC technology to elucidate the complex spatial landscape of the TME, which may aid in developing prognostic assessments and guide personalized therapeutic strategies for cancer treatments. For Research Use Only. Not for use in diagnostic procedures.



 Tumor Immunology: Checkpoints, Prevention, and Treatment (TIPT)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b22f9c7f1b4911c7dde0700548e2e19989a2ee" target='_blank'>
              High-plex spatial analysis of T cell exhaustion using Imaging Mass Cytomerty for immuno-oncology studies 2622
              </a>
            </td>
          <td>
            Qanber Raza, T. Pfister, Yun-Jyh Chwee, Nikesh Parsotam, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastases represent one of the hardest obstacles in cancer treatment, accounting for many cancer-related deaths. Understanding the mechanisms that drive disease progression is essential to improve patient outcomes and develop more effective therapeutic strategies. This phenomenon appears to be elicited by dormant cancer cells (DCCs), which can persist undetected for extended periods of time, entering a non-proliferative, hibernation-like state that confers resistance to conventional therapies and facilitates immune evasion. Owing to their status as a rare and energy-restricted population, combined with the limitations of current medical imaging, DCCs often evade early detection, hindering timely intervention and effective clinical management. Consequently, a critical need exists to develop high-resolution detection systems and identify specific DCC targetable biomarkers. Here, a comprehensive overview of the current understanding of DCCs is reported, with a focus on recent advancements in experimental strategies for their identification and tracking, as well as therapeutic approaches currently under clinical investigation aimed at targeting these elusive cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9d13b10d52e65e6fb3d5e89c185ee227dee670" target='_blank'>
              Decoding cancer dormancy: integrative genomic, phenotypic and live-cell imaging analysis to reveal the hidden cancer cell reservoir
              </a>
            </td>
          <td>
            G. Porcelli, Caterina D’Accardo, Francesca Angeloro, Martina Cucchiara, Paola Bianca, Vincenzo Davide Pantina, Narges Roozafzay, Chiara Modica, Miriam Gaggianesi, Sebastiano Di Bella, G. Stassi, Alice Turdo, M. Todaro
          </td>
          <td>2025-11-12</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Cell state plasticity drives metastasis and therapy resistance in cancers. In melanoma, these behaviors map onto a melanocytic-to-mesenchymal-like continuum regulated by AP-1 transcription factors. However, how the AP-1 network encodes a limited set of discrete states, why their distributions vary across tumors, and what drives phenotypically consequential AP-1 state transitions remain unclear. We develop a mechanistic ODE model of the AP-1 network capturing their dimerization-controlled, co-regulated, competitive interactions. Calibrated to heterogeneous single-cell data across genetically diverse melanoma populations and combined with statistical learning, the model reveals network features explaining population-specific AP-1 state distributions. These features correlate with MAPK activity across tumor lines and with variability within clones, linking MAPK signaling to AP-1 states. The model predicts and experiments validate adaptive AP-1 reconfiguration under MAPK inhibition, inducing a dedifferentiated, therapy-resistant state that can be blocked by model-guided AP-1 perturbations. These results establish AP-1 as a configurable network and provide a computational framework for predicting and modulating AP-1 driven cell state plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80053bfbfe501e1acc3e16cb2b130e2f5342e9b1" target='_blank'>
              Cell state plasticity emerging from co-regulated, competitive, and configurable interactions within the AP-1 network
              </a>
            </td>
          <td>
            Yonatan N. Degefu, Magda Bujnowska, Douglas G. Baumann, Mohammad Fallahi-Sichani
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The spontaneous regression of cancer lesions demonstrates the potential of immune surveillance; yet, these transient events have remained inaccessible to systematic study. Using longitudinal intravital microscopy in a carcinogen-induced model of head and neck cancer, we tracked premalignant lesions within the same animals for 24 weeks at single-cell resolution. This approach revealed three trajectories: progression, stability, or regression, and enabled dissection of the immune dynamics underlying each fate. Lesion outcome was dictated by the spatial organization of myeloid-derived antigen-presenting cells: regressing lesions were characterized by dense clusters of myeloid-derived cells associated with CXCL9+/CXCL10+ expression and T cell recruitment, whereas progressing lesions displayed sparse, non-clustered infiltration. Remarkably, transient myeloid clusters arose prior to any detectable lesion formation and consistently marked regions that would later develop into premalignant lesions. These findings identify spatiotemporal myeloid organization as an early determinant of tumor fate and provide a mechanistic framework for predicting and intercepting cancer at its inception. Summary Sentence Early myeloid architecture dictates cancer fate: dense CXCL9⁺/CXCL10⁺ clusters with T-cell enrichment accompany regression, whereas sparse infiltration predicts progression. Transient pre-lesional myeloid clusters emerge at future tumor sites, revealing immune organization as an early determinant of malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59ee0ed38702dcef1f68ff3c8ff2d5ef5e4fc717" target='_blank'>
              Longitudinal Imaging of the Premalignant Tumor Microenvironment Reveals Transient Myeloid States Predictive of Tumor Fate
              </a>
            </td>
          <td>
            Thomas D. Madsen, Desu Chen, Maria Hernandez, Sarah M. Hammoudeh, Marco Heydecker, Emily Chen, Salma Abu-Elnaj, Noemi Kedei, Weiye Wang, Roberto Weigert
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is the 4th leading cause of cancer death in the United States as of 2024. Its tumor microenvironment (TME) contains heterogenous inflammatory and immunosuppressive areas due to varying immunophenotypes relating to immune cell infiltration and exhaustion. CD4+ T helper cells can contribute to these dynamics, but defined roles of some subsets in PDAC remain debatable. We hypothesize that effector CD4+ T cell subsets spatially influence levels of inflammation in the PDAC TME to promote tumor metastasis. To investigate these dynamics, we designed a novel sequential multiplex immunofluorescent panel for murine FFPE slides consisting of 50 markers to spatially capture immune cells with activation/exhaustion states, epithelial-to-mesenchymal phenotypes, vessel infiltration, and stromal deposition in the TME. Using FFPE slides also improves preservation of structural morphology and translation to human tumor samples. Here, we applied the panel to primary murine PDAC tumors and liver metastases compared to normal tissues, revealing heterogenous MHC-I presentation from primary tumor cells, variations in CD4+ subsets in the primary tumor, and increased lymphocyte infiltration in liver metastases, validating our panel with recent spatial and dissociative multimodal publications. Ultimately, we will apply the panel to mice deficient in specific CD4+ subsets to better elucidate the spatial role of effector T helper cells in PDAC metastases.



 Supported by UAB Hematology & Oncology Startup Fund; Hirshberg Foundation for Pancreatic Cancer Reasearch



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d46f7f6b06d7c343cfa41f8977a3f99ebd086c73" target='_blank'>
              High-dimensional spatial analysis of murine tumor microenvironment highlights the heterogeneity of primary and metastatic pancreatic cancer immune landscape 3822
              </a>
            </td>
          <td>
            Christina R Larson, Ayushi Mandloi, Jace Baines, Richard Laube, Alana M. Franceski, R. Welner, Julienne Carstens
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c38b6d1e04676ebe5c874a1af9317e2b5d3d46" target='_blank'>
              Single-cell Transcriptomic Profiling of Pancreatic Ductal Adenocarcinoma: Epithelial Reprogramming and Systemic Immune Exhaustion
              </a>
            </td>
          <td>
            Qinyu Xu
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/465e60c37fcf50d018cbfc326971ebe12576816e" target='_blank'>
              Single-cell analysis reveals a universal pericyte signature associated with poor clinical outcome and immune T cell dysfunction in thyroid cancer and other cancers
              </a>
            </td>
          <td>
            Aarav Bhasin, Yubin Na, Jack Lawler, C. Nucera
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background: Despite the use of immunotherapy in esophageal squamous cell carcinoma (ESCC), treatment failure occurs occasionally in patients, yet the underlying mechanisms remain poorly understood. Methods: We conducted large-scale single-cell RNA sequencing (scRNA-seq) data analysis, which integrated seven independent datasets from 192 ESCC patients to yield over 440,000 high-quality single cells, to systematically characterize the tumor microenvironment (TME) landscape during ESCC progression and immunotherapy response. Additionally, we performed high-resolution spatial transcriptomics (stRNA-seq) using the 10x Visium HD platform on paired pre- and post-treatment tissues from two patients (one immunotherapy responder and one non-responder), which enhanced the findings from the scRNA-seq data and mapped therapy-induced TME at the spatial level. Multiplex immunohistochemistry was employed based on seven patients to confirm distinct patterns of intercellular crosstalk underlying differential therapeutic outcomes. Results: In scRNA-seq data, we found that B lineage cells were reduced during ESCC progression but were enriched in immunotherapy-resistant patients. Further analysis of malignant ESCC cells suggested that immunotherapy resistance might be associated with a subpopulation of tumor cells exhibiting aberrantly elevated cholesterol biosynthesis. Cell communication analysis of scRNA-seq and stRNA-seq data collectively revealed that immunotherapy resistance was linked to cellular crosstalk between cholesterol-biosynthetic tumor cells and germinal center (GC) B cells within tertiary lymphoid structures. Notably, single-cell, spatial data, and multiplex immunohistochemistry demonstrated that cholesterol biosynthesis-associated ESCC cells express elevated levels of MIF. This disrupts GC reactions by competing with the CXCL12-CXCR4 signaling axis via MIF-CXCR4 interactions, thereby impairing B cell-mediated immunity. Conclusions: MIF+ tumor cells in GCs may be a biomarker for predicting immunotherapy resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8a1ee6ff72d48f3e16c31d484505ab587cac16" target='_blank'>
              MIF-expressing tumor cells mediate immunotherapeutic resistance in esophageal squamous cell carcinoma.
              </a>
            </td>
          <td>
            Jing Song, Xiaomei Song, Yue Xie, Hong Guo, Yupeng Cun
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) technologies can be used to align transcriptomes with histopathological morphology, presenting exciting new opportunities for biomolecular discovery. Using ST data, we construct a novel framework, GeneFlow, to map transcriptomics onto paired cellular images. By combining an attention-based RNA encoder with a conditional UNet guided by rectified flow, we generate high-resolution images with different staining methods (e.g. H&E, DAPI) to highlight various cellular/tissue structures. Rectified flow with high-order ODE solvers creates a continuous, bijective mapping between transcriptomics and image manifolds, addressing the many-to-one relationship inherent in this problem. Our method enables the generation of realistic cellular morphology features and spatially resolved intercellular interactions from observational gene expression profiles, provides potential to incorporate genetic/chemical perturbations, and enables disease diagnosis by revealing dysregulated patterns in imaging phenotypes. Our rectified flow-based method outperforms diffusion-based baseline method in all experiments. Code can be found at https://github.com/wangmengbo/GeneFlow.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a975ba742c61af322f7c44267c2667ff10a839d0" target='_blank'>
              GeneFlow: Translation of Single-cell Gene Expression to Histopathological Images via Rectified Flow
              </a>
            </td>
          <td>
            Mengbo Wang, Shourya Verma, Aditya Malusare, Luopin Wang, Yiyang Lu, Vaneet Aggarwal, Mario Sola, A. Grama, N. Lanman
          </td>
          <td>2025-10-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Somatic evolution drives cancer progression and therapy resistance. Underlying phenotypic progressions is the development of marked clonal heterogeneity in most cancers. However, deciphering the downstream molecular effects of clonal heterogeneity in primary cancer has been limited due to the inability to isolate subclones by standard methods. While single-cell RNA-sequencing (scRNA-seq) provides high-resolution mapping of heterogeneous cell states, it cannot link transcriptional programs to cancer-driving mutations in these individual cells. We and others developed methods to detect genotypes and their transcriptional outputs in single cells (e.g., Genotyping of Transcriptomes; Nature, 2019). However, these approaches have technical limitations to broadly profile numerous mutations, which is nonetheless required for clonally complex neoplasms. The existing approaches are also restricted to fresh or frozen samples, limiting the use of the widely available pathology formalin fixed paraffin embedded (FFPE) tissues.
 To address these challenges, we developed Genotyping of Transcriptomes for Multiple Targets and Sample Types, GoT-Multi, a next generation single-cell multi-omics method compatible with FFPE tissues that integrates multiplexed genotyping with transcriptomic profiling. GoT-Multi adapts a probe-based scRNA-seq method (Flex Fixed RNA Profiling, 10x Genomics) by including custom mutation-specific probes along with the standard transcriptional probes. Furthermore, we developed an ensemble-based machine learning pipeline, GoT-Multi-ML, that optimizes the genotyping calls by denoising technical artifacts. GoT-Multi-ML leverages multiple machine learning models to learn the non-linear patterns of the genotyping data features (e.g., reads per genotyping transcript, GC content of probes), agnostic to the whole transcriptomic data. We validated GoT-Multi via a cell line-mixing experiment using SK-BR-3 and MCF-7 cell lines. GoT-Multi detected the targeted mutations (HIST1H1C A24T and S100A10 A77V) in the expected cell line (MCF-7) with an average genotyping rate of 72% and accuracy of 97%.To define the impact of clonal evolution on phenotypic progression, we focused on a prototype of cancer evolution, the progression of chronic lymphocytic leukemia (CLL) to therapy-resistant large B-cell lymphoma (LBCL) called Richter Transformation (RT). We applied GoT-Multi to a unique cohort of primary nodal cryopreserved samples containing both CLL and LBCL components (n = 5). We targeted 18 mutations (1-6 targets per sample) with a 98% median accuracy. We observed heterogeneous cancer cell states, including proliferating, stress response and inflammatory (n = 51,465 cells). LBCL was enriched in proliferating and cyclin D2-elevated states, whereas CLL was associated with a stress response signature. GoT-Multi enabled us to reconstruct the clonal architectures, up to 4 distinct subclones per lymphoma, and their associated cell states. Differential gene expression between mutated and wildtype cells revealed that genetically heterogenous subclones, including those with SRRM2 and JUNB mutations and therapy resistance-associated mutations in PLCG2 and BTK, displayed enrichment in inflammatory cell states, upregulating TNF and interferon signaling genes while suppressing cell cycle-related genes. Conversely, other subclones with mutations in, e.g., IRF8, POU2F2, PRKDC were associated with MYC activation and/or enhanced proliferation. These findings suggested that heterogeneous genotypes may converge on similar downstream transcriptional cell states. In addition, TNF itself was overexpressed in the proliferative subclones, suggesting that TNF production from these subclones may enhance inflammatory signaling of the TNF-responsive subclones, indicative of crosstalk between subclones. Finally, we demonstrated the applicability of GoT-Multi to pathology archived FFPE tissues. We profiled 9 mutations with an accuracy of 98% in an FFPE RT sample (n = 2,140 nuclei), allowing us to detect 5 subclones and their associated cell states. In summary, co-mapping of clonal and cell state heterogeneity at single cell resolution through GoT-Multi suggested that heterogeneous subclonal genotypes may converge on similar downstream oncogenic pathways to enhance overall tumor fitness. We envision that the broad applicability of GoT-Multi may help uncover the molecular underpinnings of cancer progression and therapy-resistance across oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/954b694c778f5e54689dba8e5834b825525f28c2" target='_blank'>
              Co-mapping clonal and transcriptional heterogeneity in somatic evolution via got-multi.
              </a>
            </td>
          <td>
            Minwoo Pak, M. Saurty-Seerunghen, Kellie Wise, Tsega-Ab Abera, Chhiring Lama, Neelang Parghi, Xiaotian Sun, Qi Gao, Liming Bao, Mikhail Roshal, John N Allan, Richard Furman, Luciano G. Martelotto, Anna S. Nam
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The bone marrow (BM) is a complex and compartmentalized tissue where spatial context plays a critical role in regulating cell behavior, signaling, and disease progression. To capture these dynamics, we apply spatial transcriptomics using the Visium Spatial Gene Expression platform on formalin-fixed paraffin-embedded (FFPE) BM sections from both healthy and Multiple Myeloma (MM) mouse models, as well as MM patient samples. Overcoming the technical challenges of working with mineralized long bone tissue, we develop a custom analytical framework integrating spatial and single-cell transcriptomic data to map cellular composition and interactions in situ. This approach enables the spatial characterization of transcriptionally heterogeneous malignant plasma cells (MM-PC) and their surrounding microenvironments. We identify spatially distinct gene programs linked to MM pathogenesis, including signatures of NETosis and IL-17 signalling, which are reduced in MM-PC–rich regions. Additionally, a transition gradient from effector to exhausted T cell phenotype is associated with increased remoteness from MM-PC. These spatial patterns are identified in FFPE BM biopsies from MM patients with varying tumor burdens. In summary, our study demonstrates both the capabilities and limitations of Visium technology in characterizing spatially regulated mechanisms underlying MM pathogenesis. Visium Spatial technology enables the characterization of transcriptionally heterogeneous malignant plasma cells in multiple myeloma, and identify spatially distinct gene programs linked to MM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55f354b66d39731b9cc107b87037fbdf9278486f" target='_blank'>
              Characterization of the bone marrow architecture of multiple myeloma using spatial transcriptomics
              </a>
            </td>
          <td>
            E. Muiños-López, Ana Rosa López-Pérez, Laura Sudupe, A. Vilas-Zornoza, S. Sarvide, Purificación Ripalda-Cemboráin, P. Aguirre-Ruiz, P. S. Martín-Uriz, M. Larrayoz, L. Alvarez-Gigli, M. Abengozar-Muela, I. Cenzano, Miguel Cócera, Javier Ruiz, Ignacio Sancho, Azari Bantan, Aleksandra Kurowska, Jin Ye, Phillip T. Newton, B. Paiva, L. Tamariz-Amador, J. R. Rodríguez-Madoz, Vincenzo Lagani, Jesper N. Tegnér, Borja Saez, J. Martinez-Climent, Isabel A. Calvo, D. Gómez-Cabrero, F. Prósper
          </td>
          <td>2025-11-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8efc06cf9c217f5e70cbbc0a284aede3a11ce200" target='_blank'>
              Cancers modulate p53 truncal neoantigen display to evade T cell detection
              </a>
            </td>
          <td>
            Koji Haratani, Bruce Reinhold, J. Duke-Cohan, Caroline Fahey, Kemin Tan, R. J. Mallis, Alexander Gusev, K. Kehl, Jia Luo, Elizabeth L Holliday, Daniel J. Masi, Allyson Karmazyn, Katarzyna J. Zienkiewicz, Connor J Hennessey, Rafael B Blasco, Tran C. Thai, Grace M. Gibbons, Sophie Kivlehan, P. Lizotte, C. Paweletz, Andrew J. Aguirre, K. L. Ligon, Roberto Chiarle, Matthew J Lang, David A Barbie, Ellis L. Reinherz
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="


 One of the core issues surrounding solid tumor cancers is the inability to predict patient responsiveness to therapies. Although tumor biopsies are common, our ability to leverage these precious tissues for treatment plans has been technologically limited, as simple histological methods have lacked the resolution to elucidate the highly variable therapeutic response outcomes within similar patient groups. Deciphering these complexities will require quantitative measurement of cellular behaviors, interactions, and phenotypes within intact tissues. Our lab has pushed to bridge the gap between spatial immunoprofiling and spatial transcriptomics. To that end, we have successfully combined the IBEX (iterative bleaching extends multiplexity) platform with HiPlex RNAscope to visualize high-plex protein markers and high-plex RNA transcripts with single-cell resolution. “Bex-Plex” enables us to perform an unmodified multi-cycle IBEX protocol on fixed frozen murine tissues followed by a modified HiPlex RNAscope protocol (so far up to 12 probes), to ensure nuclear-level alignment. With Bex-Plex we can deeply profile both the transcriptional and functional state of all cells in the tumor microenvironment, while retaining their critical spatial information. The power to profile at this single-cell level can empower predictions about the nuanced cellular interactions that drive therapeutic responsiveness, potentially revolutionizing our approach to therapy-resistant solid tumors.



 Supported by NIH/NIGMS K99/R00 GM147841-01



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca97c1f5a8a88ed8f8a444e5de591397fc59dc55" target='_blank'>
              Leveraging spatial imaging technologies to decode cellular interactions in pancreatic cancer 3297
              </a>
            </td>
          <td>
            Catherine Murphy, Andrea Verhagen, Samantha Kimmel, Vivien I Maltez
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300bf1ce4707bea01a2973482ed7fc72841204e2" target='_blank'>
              Genomic heterogeneity drives distinct infiltration patterns in glioblastoma.
              </a>
            </td>
          <td>
            D. Harwood, S. Artzi, V. Pedersen, A. Locallo, M. Lü, David Scheie, D. Nørøxe, N. Hammouda, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-11-29</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Phenotypic plasticity is a major factor in tumor heterogeneity and treatment resistance. In particular, cancer stem cells (CSCs) represent a small subpopulation within tumors that possesses self-renewal and tumor-forming capabilities. Understanding reprogramming, maintenance, and lineage properties of CSCs requires dedicated tools to disentangle the respective influences of phenotypic inheritance and cell-cell interactions. Here, we set up ultra-wide field microscopy to image breast cancer cell lines expressing a stemness fluorescent reporter over several days. The fluorescent reporter distinguishes three phenotypes: CSCs, cancer differentiated cells (CDCs), and intermediate/transiting cancer cells (iCCs). Spatial statistics indicate significant zonation in which CSCs cluster together and are spatially separated from CDCs, forming patterns resembling niches. Surprisingly, single-cell time series reveal spontaneous reprogramming events from CDC to CSC even in unperturbed populations. We identify that such transitions are prone to arise during the cell cycle. Moreover, lineage analysis shows that the phenotype is partially inherited from ancestor cells. However, such heredity is not sufficient to explain the spatial properties of the cell population, which also depend on cell-cell interactions. Indeed, we find that phenotypic transitions of cancer cells are influenced by the phenotypic state of neighboring cells. Reprogramming into CSCs is respectively promoted and inhibited by the presence of CSCs and CDCs in the neighborhood. Altogether, our results disentangle how phenotypic inheritance and intercellular interactions orchestrate the spatio-temporal self-organization of cancer cell heterogeneity, maintaining a subpopulation of CSCs within niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0816aa85e620c96aaa89badbcea20ed0e370fd6d" target='_blank'>
              Spatial self-organization of cancer stem cell niches revealed by live single-cell imaging
              </a>
            </td>
          <td>
            Mathilde Brulé, Anaïs Horochowska, Emeline Fontaine, Raoul Torero-Ibad, Flavie Woesteland, Marie Denoulet, Jean Pesez, E. Adriaenssens, R. Toillon, X. Le Bourhis, Benjamin Pfeuty, Chann Lagadec, F. Anquez
          </td>
          <td>2025-10-27</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) invasion is shaped by dynamic clonal evolution within the spatial and cellular architecture of the brain. To investigate this process, we integrated multiparametric MRI, whole-exome sequencing (WES), bulk RNA-seq and single-nucleus RNA-seq (snRNA-seq) from 69 IDH-wildtype GBM patients, using spatially resolved biopsies from contrast-enhancing (CE) core tumor and non-enhancing (NE) invasive regions. Phylogenetic analysis revealed distinct clonal architectures. CE-enriched clones carried truncal mutations in conanical drivers of cell cycle and RTK signaling (TP53, PTEN, EGFR, PDGFRA/B), and private alterations in DNA repair genes (CDH11, PAX7, RAD51B). In contrast, NE-enriched clones exhibited private mutations in MAPK and Hippo pathways components (STAG2, BRAF, MAP2K3, CTNNA1), suggesting region-specific evolutionary trajectories. Furthermore, snRNAseq cell-cell communication analysis uncovered enriched ligand-receptor signaling in NE regions between malignant cells and brain-resident populations, including PTN—PTPRZ1 (myeloid), NRG-ERBB4 and SLIT-ROBO (neuronal, astrocytic, and oligodendrocytic). Bulk RNAseq deconvolution confirmed increased representation of neurons and glial cells in biopsies harboring NE-specific clones. These findings define a unique population of GBM clones localized at the infiltrative margin, marked by private MAPK/Hippo mutations and enhanced intracellular communication with the surrounding brain microenvironment. The recurrent engagement of neural-glial signaling pathways suggests that these NE-associated clones co-opt normal cellular networks to support invasion, immune evasion, and persistence following surgical resection. This study establishes a mechanistic link between spatial clonal evolution and tumor-host communication at the GBM infiltrative region and provide a framework for targeting cell-cell signaling circuits that sustain invasive tumor ecosystems and escape therapeutic treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee4b73c1f939dc48d12b3b70a8dbe5ab434d2210" target='_blank'>
              CSIG-31. REDEFINING THE GLIOBLASTOMA RESECTION MARGIN THROUGH SPATIAL CLONAL AND CELLULAR ECOSYSTEM ANALYSIS
              </a>
            </td>
          <td>
            Angad S. Beniwal, Junlin Wang, Xiaoling Lu, C. Sereduk, S. Migliozzi, L. Garofano, C. Plaisier, Leland Hu, R. Barajas, Junwen Wang, N. Tran
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Understanding the evolutionary dynamics of clonal populations is essential for uncovering the principles of development, disease progression, and therapeutic resistance. Recent advances in single-cell lineage tracing and transcriptomics enable such analyses by combining heritable barcodes with cell-state information. Here, we present SCOUT (single-cell Ornstein–Uhlenbeck trees), a framework that models gene expression dynamics along single-cell lineage trees using Ornstein–Uhlenbeck processes to distinguish neutral drift from selective pressure. Using simulations, we demonstrate that SCOUT accurately classifies genes based on their underlying evolutionary models. We further validate SCOUT in Caenorhabditis elegans development, identifying biological processes under selection across distinct developmental contexts. Finally, we apply SCOUT to a lung adenocarcinoma xenograft model, revealing key regulators of metastatic progression and tumor microenvironmental adaptation. By integrating lineage and transcriptomic data, SCOUT provides a powerful evolutionary lens for dissecting the forces that shape cell fate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b1a07c3c29336ff660b3f4e1f28115198c90579" target='_blank'>
              SCOUT: Ornstein–Uhlenbeck modelling of gene expression evolution on single-cell lineage trees
              </a>
            </td>
          <td>
            Hannah Stuart, Aaron McKenna
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Resistance to immune checkpoint blockade (ICB) remains a major challenge in lung adenocarcinoma (LUAD), with stromal mechanisms underlying CD8⁺ T cell exhaustion still poorly understood. By integrating single-cell, bulk, and spatial transcriptomic datasets using EcoTyper, we identified a distinct immunosuppressive ecotype, EC10, enriched for TGF-β signaling and epithelial–mesenchymal transition. EC10 exhibited spatial co-localization of fibroblasts, malignant epithelial cells, and exhausted CD8+ T cells, and was consistently associated with immune exclusion, poor progression-free survival, and elevated TIDE scores across four ICB-treated LUAD cohorts. Cell–cell communication analyses revealed a dominant TGFB1–SERPINE1 signaling axis originating from fibroblasts, linking stromal remodeling to T cell dysfunction. In contrast, EC12 represented an inflamed, ICB-responsive state enriched in interferon signaling. These findings define EC10 as a spatially organized, fibroblast-driven immunosuppressive ecosystem predictive of ICB resistance, and highlight the therapeutic potential of targeting the TGF-β axis in LUAD. This research identifies a fibroblast-driven immunosuppressive ecosystem in lung adenocarcinoma, linking TGF-β signaling to immune resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130525b051ffbdab87e5edbec0e75b95564de9c8" target='_blank'>
              Fibroblast TGF-β signaling defines spatial tumor ecosystems linked to immune checkpoint blockade resistance
              </a>
            </td>
          <td>
            Seo-Young Lee, Youngjoo Lee
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Background Angioimmunoblastic T cell lymphoma (AITL) is a subtype of peripheral T-cell lymphoma with a poor prognosis. Pathologically, AITL is characterized by the proliferation of diverse stromal cells (SCs), including blood endothelial cells (BECs) and follicular dendritic cells (FDCs). However, the tumor histology exhibits considerable heterogeneity, both between and within patients. It is hypothesized that these SCs form a supportive tumor microenvironment, and their histological heterogeneity may influence the clinical course. We previously reported that single-cell RNA sequencing (scRNA-seq) of SCs in lymph nodes (LNs) identified 30 distinct SC subclusters (Nat Cell Biol, 2022). However, the specific SC subclusters involved in AITL tumorigenesis remain undefined.
 Aims We aimed to characterize the SC heterogeneity in AITL microenvironments by employing scRNA-seq and spatial multi-omics at single-cell resolution.
 Methods Non-hematopoietic cells were isolated from 10 AITL and 5 normal LNs using magnetic- and fluorescence-activated cell sorting, followed by scRNA-seq. Based on the scRNA-seq data of SCs and immune cells, we designed a 289-gene panel for spatial transcriptomics (ST) using Xenium in situ, and a 50-protein panel for spatial proteomics (SP) using PhenoCycler fusion targeting immune cell and SC subclusters. These assays were applied to 27 formalin-fixed paraffin-embedded AITL samples. Whole exome sequencing (WES) was successfully performed on 26 of the 27 samples. We analyzed a bulk RNA-seq dataset of 97 AITL to validate the prognostic potential of gene expression patterns.
 Results First, we analyzed scRNA-seq data from 49,363 stroma cells and identified 30 SC subclusters consistent with our prior findings. Notably, transitional BECs between capillary BECs and activated high endothelial venules (C-aHEVs) expressed markers of both capillary BECs and HEVs, and were further subdivided into C-aHEV1 and C-aHEV2. Both tip cells, which contribute to angiogenesis, and C-aHEV2 were increased in AITL samples. Differentially expressed genes analysis revealed that both C-aHEV1 and C-aHEV2 highly expressed ACKR1. Notably, C-aHEV1 exhibited high expression of chemokines such as CXCL10 and CXCL12, whereas C-aHEV2 expressed genes encoding extracellular matrix proteins. Interaction analysis revealed that the CCL5-ACKR1 axis, which is known as a regulator of leukocyte migration, was activated between CD8-positive T cells and C-aHEVs. Furthermore, the COL15A1-integrin and SPARC-ENG axis, both of which have been reported to promote angiogenesis, were specifically enriched between C-aHEV2 and other HEVs.
 ST analysis of 2,020,192 cells identified 10 BEC subclusters, 5 non-endothelial SC subclusters, and 12 immune/tumor cell subclusters. Spatial proximity measurement revealed that C-aHEV1 was the BEC subcluster most closely associated with tumor cells. Spatial niche analysis revealed that C-aHEV1, together with FDCs, formed a distinct tumor-associated niche (CFT niche). On the other hand, C-aHEV2 was localized farther from the tumor than C-aHEV1, and formed a niche in conjunction with other HEV subclusters (HEV niche). The CFT niche-dense area and the HEV niche-dense area were often in close proximity, and the densities of CXCR4, encoding areceptor for CXCL12, and LAG3 and CXCR6, serving as T-cell exhaustion markers, were high in the CFT niche.
 SP analysis detected 6,172,072 cells and identified SCs containing 5 BEC components (artery, capillary, C-aHEV, HEV, and vein), tumor cells, and immune cell subtypes. C-aHEVs tended to be located closer to tumor cells than other BECs and formed a unified vascular niche in conjunction with HEV.
 WES analysis revealed the recurrent G17V RHOA mutations in 19 samples. Notably, samples harboring G17V RHOA mutations exhibited significantly higher densities of C-aHEV2 cells.
 Bulk RNA-seq analysis revealed that patients with high expression of C-aHEVs and other HEV-related signature exhibited poorer survival outcomes.
 Summary/ Conclusion By integrating scRNA-seq and spatial data, we identified C-aHEVs as key components in shaping the AITL microenvironment through the recruitment of immune cells. As the gene expression level, C-aHEVs can be subdivided into C-aHEV1 and C-aHEV2, which exhibited distinct properties. Further investigation is needed to validate the mechanisms by which these BEC clusters interact with tumor cells and contribute to tumorigenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/087d4af2ea5c2629eb9fea9276a3b1bcc2d2dd51" target='_blank'>
              Distinct blood endothelial cells shape spatial niches in angioimmunoblastic T-cell lymphoma
              </a>
            </td>
          <td>
            Yuki Shimizu, Manabu Fujisawa, Yoshiaki Abe, K. Makishima, S. Suma, Y. Suehara, K. Hattori, Tatsuhiro Sakamoto, Kyohei Sano, A. Tsubosaka, Miwako Kakiuchi, D. Kaji, Ayako Suzuki, Kensuke Usuki, Kosei Matsue, Yasunori Ota, Shigeru Chiba, S. Ishikawa, Yutaka Suzuki, Christian Steidl, M. Sakata-Yanagimoto
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8015c113e13d1290a915b03f174abb55236e475" target='_blank'>
              Longitudinal analysis of matched patient biospecimens reveals Neural Reprogramming of Cancer-Associated Fibroblasts Following Chemotherapy in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Aylin Z. Henstridge, Nandini Arya, Ahmed M. Elhossiny, Padma Kadiyala, Georgina Branch, Vaibhav Sahai, Nicole Peterson, Jorge D. Machicado, Richard S. Kwon, Allison Schulman, Erik Wamsteker, George M. Philips, Stacy Menees, Jonathan Xia, Arvind Rao, Jiaqi Shi, Timothy L. Frankel, Filip Bednar, Marina Pasca di Magliano, C. Lyssiotis, M. Truty, Martin Fernandez-Zapico, Eileen S. Carpenter
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Single-cell sequencing (scRNA-seq) has emerged as a pivotal technology for deciphering the complex cellular heterogeneity and tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC), positioning it as a critical tool for informing novel therapeutic strategies. This review explores how scRNA-seq reveals diverse cellular subpopulations and their functional roles within the PDAC TME, including malignant epithelial cells with transitional phenotypes, heterogeneous cancer-associated fibroblasts (CAFs), functionally distinct immune cells such as tumor-associated neutrophils (TANs) and macrophages (TAMs), and actively participating neural components like Schwann cells. These cellular constituents form specialized functional units that drive tumor progression, immune evasion, neural invasion, and therapy resistance through metabolic reprogramming, immunosuppressive signaling, and cellular plasticity. The review further examines technological advances in single-cell sequencing from 2023 to 2025, focusing on sample preprocessing innovations, multi-omics integration (combining transcriptomics with epigenomics and proteomics), spatial resolution enhancements, and customized computational tools that address PDAC-specific challenges. Clinically, single-cell sequencing enables precise cellular subtyping, identification of novel biomarkers, and development of personalized therapeutic approaches, including combination therapies targeting specific cellular subpopulations and their interactions. Despite these advances, significant challenges remain in standardizing clinical applications such as liquid biopsy for early detection and tumor microenvironment assessment for diagnostic staging, validating biomarkers like CLIC4, GAS2L1, Cytokeratins, Vimentin and N-cadherin in circulating tumor cells, and comprehensively integrating multi-omics data. Future research focusing on both technology refinement and biological validation will be essential for translating single-cell insights into improved diagnostic and therapeutic outcomes for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9576dbb6fe77bb5f04f2ad67b474589557936" target='_blank'>
              Single-Cell Sequencing Unravels Pancreatic Cancer: Novel Technologies Reveal Novel Aspects of Cellular Heterogeneity and Inform Therapeutic Strategies
              </a>
            </td>
          <td>
            Keran Chen, Zeyu Chen, Jinai Wang, Mo Zhou, Yun Liu, Bin Xu, Zhi-Jing Yu, Yiming Li, Guanhu Yang, Tiancheng Xu
          </td>
          <td>2025-12-10</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="HPV status is a key determinant of prognosis and treatment response in head and neck squamous cell carcinoma (HNSCC). To investigate how HPV influences the tumor-immune-stromal landscape, we performed high-dimensional spatial profiling, including its impact on spatial organization, tertiary lymphoid structures (TLSs), and spatially organized cellular neighborhoods. Tumor biopsies from HNSCC patients (n = 16; 7 HPV-positive, 9 HPV-negative) were stained with a multiplex immunofluorescence (mIF) panel focused on immune profiling. A deep learning-based analysis pipeline enabled the identification and phenotypic state profiling of 14 cell types. Tissues were segmented into four distinct tumor regions, and spatial neighborhoods and TLSs were identified and analyzed for differential cellular composition, activation states, and spatial interactions between HPV-positive and HPV-negative tumors. HPV-positive and HPV-negative tumors differ in their tumor microenvironment (TME) composition, tumor cell state and spatial organization. The TME of HPV-positive tumors exhibited a greater abundance of activated lymphocytes, B- and T-cell-enriched spatial neighborhoods, and PD-1–PD-L1 interactions within the tumor area, whereas HPV-negative tumors were dominated by fibroblast- and macrophage-rich niches. T- cells in HPV-positive tumors showed greater activation across neighborhoods and areas, while in HPV-negative tumors T- cells demonstrated enrichment of exhaustion and terminal differentiation markers such as PD-1 and CD57. HPV-positive tumor cells had increased IDO1, HLA-DR, and Ki67 positivity, whereas HPV-negative tumor cells were more frequently CD44 positive, reflecting a more stem-like phenotype. Importantly, TLSs in HPV-positive tumors were located closer to the tumor area and enriched in activated immune cells, including ICOS+ CD4 T- cells, memory T- cells, and CD21+ B- cells. In contrast, TLSs in HPV-negative tumors were more distant and enriched for immunosuppressive populations such as PD-1+/PD-L1+ Tregs and macrophages. HPV status defines distinct spatial immune architectures in HNSCC. HPV-positive tumors harbor immune-activating TLSs and cellular neighborhoods that support antitumor immunity, whereas HPV-negative tumors exhibit suppressive niches and stromal dominance. These findings highlight TLSs, particularly their proximity and composition, as key features of the HPV-stratified TME and potential biomarkers for immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec5a252b8e081f817dffff9e37e6b264cbbcf5" target='_blank'>
              Spatial profiling of HPV-stratified head and neck squamous cell carcinoma reveals distinct immune niches and microenvironmental architectures
              </a>
            </td>
          <td>
            E. Markovits, Dmytro Klymyshyn, Roni Froumine, Hailing Zong, M. Mints, Sangeetha Mahadevan, Kenneth Bloom, Jamie G. Bates, Gareth J. Thomas, Lauri Diehl, Oscar Puig, Abhishek Aggarwal
          </td>
          <td>2025-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The tumor microenvironment (TME) has emerged as a promising source of prognostic biomarkers. To fully leverage its potential, analysis methods must capture complex interactions between different cell types. We propose HiGINE -- a hierarchical graph-based approach to predict patient survival (short vs. long) from TME characterization in multiplex immunofluorescence (mIF) images and enhance risk stratification in lung cancer. Our model encodes both local and global inter-relations in cell neighborhoods, incorporating information about cell types and morphology. Multimodal fusion, aggregating cancer stage with mIF-derived features, further boosts performance. We validate HiGINE on two public datasets, demonstrating improved risk stratification, robustness, and generalizability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d207f0757528c944b801713e062bc7358ff14382" target='_blank'>
              From Cells to Survival: Hierarchical Analysis of Cell Inter-Relations in Multiplex Microscopy for Lung Cancer Prognosis
              </a>
            </td>
          <td>
            Olle Edgren Schullerqvist, Jens Baumann, Joakim Lindblad, Love Nordling, Artur Mezheyeuski, Patrick Micke, Natavsa Sladoje
          </td>
          <td>2025-12-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Human breast cancer cells exhibit organotropism during metastasis, showing preferential homing to certain organs such as bone, lung, liver, and brain. Spatial genome organization plays a crucial role in oncogenic transformation and progression, but the extent to which chromosome architecture contributes to organ-specific metastatic traits is unclear. This work characterizes chromosome architecture changes associated with organotropic metastatic traits. By comparing a collection of human genomic data from different subtypes of localized and lung metastatic breast cancer cells with both normal and cancerous lung cells, we find important trends of genomic reorganization. The most striking differences in 3D genome compartments segregate cell types according to their epithelial vs. mesenchymal status. This epithelial–mesenchymal transition (EMT) compartment signature occurs at genomic regions distinct from transcription-defined EMT signatures, suggesting a separate layer of regulation. Specifically querying organotropism, we find 3D genome changes consistent with adaptations needed to survive in a new microenvironment, with lung metastatic breast cancer cells exhibiting compartment switch signatures that shift the genome architecture to a lung cell-like conformation and brain metastatic prostate cancer cells showing compartment shifts toward a brain-like state. TCGA patient data reveals gene expression changes concordant with these organ-permissive compartment changes. These results suggest that genome architecture provides an additional level of cell fate specification informing organotropism and enabling survival at the metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d470a8773036dd5ad3cf2b42848d5f3ff555882" target='_blank'>
              Rearrangement of 3D genome organization in breast cancer epithelial to mesenchymal transition and metastasis organotropism
              </a>
            </td>
          <td>
            Priyojit Das, Rebeca San Martin, Tian Hong, R. P. McCord
          </td>
          <td>2025-11-12</td>
          <td>eLife</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 H3K27 altered diffuse midline gliomas (DMG) are aggressive brain tumors that arise centrally in the brain. These tumors are not amenable to surgery and no curative therapy exists. DMG exhibits profound heterogeneity, with tumor cells displaying dynamic and interchangeable cellular states that recapitulate aspects of normal development. In parallel, the tumor microenvironment (TME) is highly diverse, further contributing to the complexity of the disease. A better understanding of the DMG ecosystem is needed for therapeutic target identification. In this study, publicly available single cell RNA seq data of DMG (Jessa et al. Nat. Genet. 2022) were analyzed to define key molecular characteristics of DMG. Xenium spatial transcriptomics and multiplex immunohistochemistry were performed on FFPE patient samples collected at diagnosis to study the tumor and TME cells in their native spatial context. In DMG, both the glioma cellular composition and the heterogeneity of the TME varied based on the anatomical location of the tumor. Tumors located in the pons exhibited a high proportion of astrocyte-like glioma cells. In contrast, thalamic DMG showed a greater abundance of oligodendrocyte-like tumor cells, suggesting a regional bias in cellular differentiation states. Furthermore, the TME in thalamic tumors was notably enriched with immunotolerant macrophage populations, expressing molecules associated with an anti-inflammatory, pro-tumoral phenotype. With spatial profiling, distinct tumor niches were identified, showing heterogeneity among different samples and underscoring the complex and spatially organized nature of both tumor cell states and the surrounding microenvironment in DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/408581e61f93671e8e7b1c9fc7f6b7a6669a068a" target='_blank'>
              EPCO-24. DISSECTING THE ECOSYSTEM OF PEDIATRIC H3K27 ALTERED DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            Julie Morscio, Chiara Caprioli, C. González-Blas, Pierluigi Calò, K. Vandereyken, Thierry Voet, R. Sciot, Sandra Jacobs, F. De Smet
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a23889452eb3a744641d2c79b511978437ffd39c" target='_blank'>
              A new genetically engineered transplant model of glioma recapitulates key phenotypes of low- and high-grade gliomas
              </a>
            </td>
          <td>
            D. Forsythe, Shannon J. Oliver, A. Valkovic, Danielle Lewthwaite, Germaine Uys, Brittany Koning, David Eccles, L. Satgunaseelan, I. Hermans, S. Freytag, Jim R Whittle, Sarah A. Best, Melanie J. McConnell
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Deep learning and large language models can integrate complex datasets to uncover biological insights that are often undetectable through conventional analyses. With application to translational cancer research, these computational tools have positioned 3D patient-derived tumor avatars front and center as crucial data input sources. However, a major challenge remains: the lack of standardization in media composition in 3D patient-derived tumor models unpredictably affects cell behavior and limit the utility beyond predicting treatment responses. To address this unmet need, we developed a simple, reproducible perfusion circuit system to approximate in vivo physiology using autologous patient plasma. With peritoneal metastases and core needle biopsies across multiple tumor histologies, we demonstrate preservation of the tumor microenvironment for up to 48 hours using multi-modal interrogation techniques. With proof-of-concept experiments, we display the system’s ability to unveil complex drug-dependent biology within this time window. Standardizable, physiologically relevant platforms for 3D patient-derived tumor avatars will yield unprecedented insights through the integration of data from broad groups of patients and the use of an expanding armamentarium of artificial intelligence capabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7a2b5548999bc3e5693c3a46fe302fa33927e3b" target='_blank'>
              A simple circuit to sustain intact tumor microenvironments for complex drug interrogations
              </a>
            </td>
          <td>
            A. Sarvestani, Samantha Ruff, K. Remmert, Martha E Teke, Emily A Verbus, A. Saif, Marcial Garmendia-Cedillos, Alex Rossi, Ashley Rainey, K. Nomie, Surajit Sinha, Michael M. Wach, Carrie E Ryan, Stephanie N. Gregory, Priyanka P Desai, A. Panda, Emily Smith, Reed Ayabe, James D McDonald, Shreya Gupta, Tracey Pu, Tahsin M. Khan, Jacob T Lambdin, K. Luberice, Stephanie C Lux, Hanna Hong, Imani A Alexander, Sarfraz R. Akmal, Shahyan Rehman, Sophia Xiao, Jason Ho, Justine F. Burke, C. Hannah, Alexandra Livanova, Andrey Tyshevich, Stanislav Kurpe, Andrey Kravets, Basim Salem, Daniil Vibe, Arina Tkachukv, Anna Belozerova, A. Sarachakov, Evgeny Barykin, V. Kushnarev, E. Postovalova, A. Bagaev, Noemi Kedei, Maria Hernandez, Baktiar Karim, Donna Butcher, Jennifer Matta, Michael Kelly, Jatinder Singh, Todd D Prickett, James W Hodge, T. Andresson, Michael B Yaffe, Ethan M Shevach, David E. Kleiner, Jeremy L. Davis, Andrew M. Blakely, Jonathan M. Hernandez
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Patient-derived organoids (PDOs) preserve patient genotypes and 3D architecture, offering a useful platform to investigate mechanisms of radioresistance and test radiosensitizers. We outline an end-to-end workflow—model establishment, multi-omics profiling, pharmacologic screening, and in vivo confirmation—and spotlight immune-competent, vascularized, and organ-on-chip formats. PDOs reveal actionable mechanisms across DNA damage response, hypoxia–metabolic and immune remodeling, and radiation-induced senescence, enabling rational radiosensitizer selection. Paired tumor–normal organoids concurrently gauge efficacy and normal tissue toxicity, refining the therapeutic index. Remaining gaps (incomplete microenvironment, fractionation modeling, and standardization) are being addressed via reporting standards and co-clinical studies, positioning PDOs to support precision radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95eba7f430f22360f6d996dcc784b52bde1ba4ff" target='_blank'>
              Patient-Derived Organoids as a Platform to Decipher and Overcome Radioresistance: From the Tumor Microenvironment to Radiosensitizer Discovery
              </a>
            </td>
          <td>
            Dashan Yin, Xiujuan Hong, Xiaoqi Wang, Wenjia Ding, Chenli Wang, Jin Qian, Yi Zhou, Chuan Sun, Zhibing Wu
          </td>
          <td>2025-12-01</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance.
              </a>
            </td>
          <td>
            Guoqian He, Si-Jia He, Xiao-yu Jing, Yi-Ling Dai, Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Background Advances in single-cell and spatial transcriptomics have transformed our understanding of tumor cell states and lineage dynamics, enabling high-resolution investigation of cellular heterogeneity and evolution. Motivation Building on Schiffman et al.’s Markovian framework for inferring cell state transitions from single-cell lineage tracing, there is a need for models that integrate spatial context to better capture the dynamics of tumor evolution. Method and results We introduce StateSim, a spatial–temporal model that simulates tumor cell state transitions within tissue, incorporating spatial growth and mutation accumulation (Waclaw, et al. 2015). StateSim assumes that daughter cells are spatial neighbors, shaping the spatial organization of cell states, and models two cell states with distinct birth and death rates. To quantify spatial patterns, we developed StateMap, a network-based approach that maps tumor cell states and their spatial relationships, summarizing transition patterns through network statistics. Applying StateSim under various transition scenarios, we demonstrate that distinct transition modes yield unique spatial statistics, robustly correlated with transition rates (Pearson r = 0.9938, P = 6.87e-10). Using spatial transcriptomics data from glioma samples (Greenwald, et al. 2024), StateMap identified subgroups with distinct transition patterns, further validated by spatial statistics such as bivariate Moran’s I. Approximate Bayesian computation with StateSim enabled inference of posterior probabilities for cell state transition rates and phylogenetic trajectories. Conclusion Our framework enables inference of cell state transitions and in-silico lineage tracing from spatial data, advancing studies of tumor heterogeneity and evolution. Significance This work provides a novel computational approach for integrating spatial and temporal information to decipher the dynamics of cell state transitions and lineage development during cancer evolution, offering new insights into tumor heterogeneity and progression. References 1. Schiffman J.S., et al. ‘Defining heritability, plasticity, and transition dynamics of cellular phenotypes in somatic evolution.’ Nature Genetics 2024; 56(10):2174–2184. 2. Waclaw B., et al. ‘A spatial model predicts that dispersal and cell turnover limit intratumour heterogeneity.’ Nature 2015; 525(7568):261–264. 3. Greenwald A.C., et al. ‘Integrative spatial analysis reveals a multi-layered organization of glioblastoma.’ Cell 2024; 187(10):2485–2501.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c607caf6be05bb9a4600cf6079421f01ab3b8b" target='_blank'>
              Decipher the spatial dynamics of the cell state transition and lineage development during cancer evolution
              </a>
            </td>
          <td>
            Jiabao Li, Jiguang Wang
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 The diffusely infiltrative growth pattern of glioblastoma (GBM) is a major obstacle to effective therapy. A complex network of tumor intrinsic and microenvironmental features interact to shape the landscape of this growth pattern, but deciphering the molecular substrates of these interactions remains challenging. While single cell RNA sequencing has reshaped our understanding of GBM heterogeneity and uncovered cellular phenotypes of many cellular populations in these tumors, the spatial context of this heterogeneity could not be assessed with earlier technologies. By contrast, spatial transcriptomics platforms have the potential to map this transcriptome-wide information in situ. We recently reported the use of this technology to develop a layered model of spatial architecture in GBM and showed that hypoxia represents a key organizing feature of GBM heterogeneity. However, the limited resolution of this platform precluded assessment of whole-transcriptome features at truly cellular resolution. In this work, we now present our findings utilizing the next generation of this technology and showcase the ability to map whole transcriptome data in spatial context at true single cell resolution in primary patient GBM samples. We explore heterogeneous GBM phenotypes in the context of GBM spatial microarchitecture. Using copy number inference, we demonstrate the ability to assess clonal heterogeneity in situ and map the pattern of invasion of unique GBM clones. Finally, we discuss our progress to discover organizing principles underlying distinct patterns of GBM cell invasion. Taken together, we demonstrate the potential of single cell spatial transcriptomics to decipher the molecular heterogeneity of GBM microenvironment in situ and uncover the biology driving the aggressive infiltrative growth of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad815e1440b141545b8a8eee62737b92edef994" target='_blank'>
              TMIC-68. Deciphering the spatial architecture of transcriptional heterogeneity in glioblastoma at single cell resolution
              </a>
            </td>
          <td>
            C. Mount, Zebin Wen, Alissa C. Greenwald, S. Kovatsis, Jack Lu, D. Gerovasilis, I. Tirosh, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7be0b2cf86ead8f0005945c582738dd16581854f" target='_blank'>
              SPP1+Macrophages promote the growth and metastasis of intrahepatic cholangiocarcinoma via OSM and THBS1 signaling
              </a>
            </td>
          <td>
            Tingjie Wang, Ruitao Long, Zhongyu Cheng, Mi Zhang, Bing Wei, Cong Lyu, Cuiyun Zhang, Guangbo Tang, Zihang Li, Lei Li, Yida Li, Xiaofei Zhu, Chao Shi, Ke Yang, Chuanrui Xu
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e493ac7b6aab5ae1ab712cb247fef1395ea0b1a" target='_blank'>
              Quantitative profiling of intratumor immune heterogeneity identifies loss of immune diversity as a hallmark of cancer progression.
              </a>
            </td>
          <td>
            Qiqi Lu, Jian-qin Luo, Chia-Hao Tung, Xiaosheng Wang, Zhongming Zhao
          </td>
          <td>2025-12-16</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Single-cell and spatial transcriptomics have become pivotal tools for elucidating the phenotypic heterogeneity of immune cells and identifying spatially recurrent immune niche states. These technologies have produced extensive datasets containing whole-transcriptome profiles of dissociated single cells and spatially resolved tissues. However, both approaches are subject to significant technical and biological batch effects, which complicates the assignment of cell types from single-cell data to spatial transcriptomics datasets and hinders the comparative analysis of spatial niche and region labels across tissue slices. To overcome these limitations, we developed Synchronized Deconvolution and Integration (SDI), a deep learning model designed to integrate dissociated single-cell and spatial transcriptomics data across diverse platforms and spatial resolutions. Pretrained on over 5 million dissociated cells and more than 1 million spatial spots from pan-cancer tissues, SDI demonstrates the ability to accurately project cell type labels from single-cell to spatial datasets, with adaptability to novel datasets. We show that SDI facilitates batch-unbiased deconvolution of immune cell composition and provides consistent predictions of tertiary lymphoid structures (TLS) across over 400 tissue slices from more than 10 cancer types. In conclusion, SDI enables integrated analyses linking immune cell types from both single-cell and spatial transcriptomics.



 None declared



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5508adcf16a988490ac8e07619e7513b2c493ae0" target='_blank'>
              Linking large-scale pan-cancer single-cell and spatial transcriptomics profiles using SDI 9110
              </a>
            </td>
          <td>
            Ziwei Xue, Wanlu Liu
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Age is a dominant risk factor for all major breast cancer subtypes. However, the mechanisms by which aging influences tumor development remain unclear. Using a novel mouse model whereby breast cancer is induced in situ in young and old wild-type mice via intraductal delivery of a lentivirus encoding the HER2/neu oncogene, we found that old mice exhibited a higher oncogene-induced tumor burden than young mice. Old tumor cells showed reduced expression of interferon-related genes, particularly the T cell-recruiting chemokines Cxcl9 and Cxcl10, linked to their altered chromatin accessibility. CXCL9/10 expression also declined with age in human HER2+ tumors. Correspondingly, old tumors exhibited fewer T cells within tumor lesions. Targeted interventions showed that decreased expression of Cxcl9/10 is responsible for reduced T cell infiltration and weakened anti-tumor immunity. These results show how aged tumor cells are impaired in their recruitment of immune cells, leading to a defective anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38011df7fcbd774b6090832583a2578934863ea" target='_blank'>
              Aging alters tumor cell - T cell crosstalk to promote breast cancer progression
              </a>
            </td>
          <td>
            Shanshan Yin, Kay T Yeung, S. Mamde, Kate Lin, Armin Gandhi, Xue Lei, Andrew Davis, Rouven Arnold, Jing Yang, P. Adams
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a formidable clinical challenge due to therapy resistance, metastasis, and relapse. Central to these processes are colorectal cancer stem cells (CCSCs), a dynamic subpopulation endowed with self-renewal capacity, plasticity, and heterogeneity. This review synthesizes recent advancements in understanding how CCSCs orchestrate tumor progression through intricate bidirectional crosstalk with the tumor immune microenvironment (TIME). We begin by elucidating the cellular origins of CCSCs, their profound intratumoral heterogeneity, and their remarkable phenotypic plasticity—driven by genetic, epigenetic, and metabolic reprogramming—which collectively serve as the root cause of therapeutic failure. A significant portion of our discussion is dedicated to deconstructing the immunosuppressive niche co-opted by CCSCs. We detail mechanisms of immune evasion and tolerance, highlighting how CCSCs modulate innate and adaptive immune cells—including NK cells, Tregs, dendritic cells, macrophages, neutrophils, and myeloid-derived suppressor cells—to foster an environment that supports stemness and suppresses cytotoxic attack. This reciprocal interaction forms a vicious cycle that perpetuates tumor survival and progression. Finally, we critically evaluate emerging therapeutic strategies that concurrently target CCSC-specific vulnerabilities and counteract immunosuppression. We explore the limitations of conventional chemotherapy and the promise of targeted therapies (e.g., Wnt inhibitors), immunotherapies (e.g., CAR-T, bispecific antibodies), and combination regimens designed to remodel the TIME and eradicate the CCSC reservoir. By integrating insights from single-cell omics and spatial biology, this review provides a comprehensive framework for overcoming therapy resistance and proposes novel precision medicine approaches for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b2fada714432c2d56f0013d3edefb668b69269" target='_blank'>
              Colorectal cancer stem cells crosstalk in tumor immune microenvironment and targeted therapeutic strategies
              </a>
            </td>
          <td>
            Yuxuan Duan, Anshu Li, Daojia Miao, Diaoyi Tan, Keshan Wang, Jian Shi
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the leading causes of cancer-related morbidity and mortality worldwide. Tumor sidedness, distinguishing left- (LCRC) and right-sided (RCRC) cancers, has emerged as a critical clinical determinant, influencing patient prognosis and therapeutic response. However, the cellular and molecular mechanisms underlying these differences remain poorly understood. Recent advances in single-cell RNA sequencing (sc-seq) provide high-resolution insights into CRC heterogeneity, revealing distinct tumor, immune, and stromal cell populations and their context-specific interactions. In this review, we synthesize sc-seq studies that dissect the molecular programs driving progression, therapy resistance, and metastasis in CRC. We highlight malignant subclusters characterized by metabolic reprogramming and spatially organized oncogenic signaling; specialized immune cell states, including macrophage subsets, exhausted T cells, and mast cells, that shape tumor immunity; and stromal elements such as cancer-associated fibroblasts and endothelial tip cells that remodel the extracellular matrix, promote angiogenesis, and foster immune evasion. Importantly, sc-seq demonstrates that LCRC and RCRC represent distinct multicellular ecosystems with differential immune recruitment and stromal signaling, underscoring the need for sidedness-informed therapeutic strategies. We propose that future interventions should target cell-cell communication networks and spatially defined tumor–microenvironment interactions to overcome heterogeneity and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03b9e3bca233f1317bcd2e2e43fb69a5fedfb984" target='_blank'>
              Sidedness matters: single-cell perspectives on left- and right-sided colorectal cancer
              </a>
            </td>
          <td>
            M. Yanova, Diana Maltseva, Alexander Tonevitsky
          </td>
          <td>2025-12-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01df2c37dabe824163d0ae6c9689244efcafc8f6" target='_blank'>
              Multi-scale Network Topology Analysis Reveals Metabolic Reprogramming and Therapeutic Vulnerabilities in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Handan Cetin, P. Bogdan, S. Mumenthaler, Stacey D. Finley
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cancer‐associated fibroblasts (CAFs) are pivotal stromal components that shape the tumor microenvironment through their remarkable heterogeneity and dynamic functions. This review comprehensively examines the biology of CAFs, starting with their diverse cellular origins and induction mechanisms via key signaling pathways. We highlight the spatial‐temporal heterogeneity of CAF subsets and propose an updated classification system that encompasses eight functionally distinct subtypes based on recent single‐cell transcriptomic studies. The tumor‐promoting roles of CAFs are explored in depth, including their contributions to tumor behavior through novel mechanisms such as neutrophil extracellular traps formation, and multidrug resistance via metabolic reprogramming. A major focus is placed on CAF‐mediated immunotherapy resistance through immune checkpoint regulation, recruitment of immunosuppressive cells, and metabolite secretion. This review also highlights key technologies that are advancing CAF research, including patient‐derived 3D models, artificial intelligence‐enhanced spatial multiomics, clustered regularly interspaced short palindromic repeats (CRISPR)‐engineered mouse systems, and machine learning approaches for analyzing CAF heterogeneity and function. Finally, we evaluate therapeutic strategies targeting CAF activation, secreted factors, and immune crosstalk, while critically analyzing ongoing clinical trials. Overall, this review not only clarifies the biology of CAFs but also provides a translational framework for developing next‐generation stromal‐targeted therapies to overcome treatment resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1dae86be5576422570b1acdcb11b97738bdec4b" target='_blank'>
              Cancer‐Associated Fibroblasts: Origin, Classification, Tumorigenicity, and Targeting for Cancer Therapy
              </a>
            </td>
          <td>
            Chang Fan, Wenlong Zhu, Yuan Chen, Wanwan Zhu, Jin Ding
          </td>
          <td>2025-10-28</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy, poised to become the second leading cause of cancer-related deaths in North America. The paucity of neoantigen presentation in PDAC has been hypothesized to promote the development of an “immunologically-cold” tumor microenvironment (TME), driving poor clinical responses to immunotherapy. Despite the abundant immune infiltrate observed in PDAC, T cells display a limited cytotoxic capacity to kill malignant tumor cells which further diminishes throughout disease progression. Utilizing single cell RNA-sequencing coupled to T cell receptor profiling, we have characterized T cell populations in clinical specimens of inflammatory pre-malignant intraductal papillary mucinous neoplasia (IPMN) lesions and advanced stage PDAC tumors. Striking phenotypic and clonal dynamic differences were observed in both CD8 and CD4 T cell populations between IPMN and PDAC specimens which underly their distinct functional capabilities in the evolving TME. The spatial interactions and microenvironmental neighborhoods of neoantigen-reactive T cells in PDAC have yet to be explored in-depth and hold unexploited therapeutic potential. Utilizing the 10X Genomics Xenium platform with custom probes designed to hybridize with neo-antigen-specific T cell clones, we are interrogating the cellular interactions which drive T cell clonal expansion to identify therapeutically relevant vulnerabilities in PDAC malignant progression.



 Supported by National Insitutes of Health, Canadian Institutes of Health Research, and Princess Margaret Cancer Center Catalyst Grant.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f6400ccbc9c89325532880266eae31b90968879" target='_blank'>
              Defining molecular determinants of T cell spatial localization and clonal response throughout PDAC malignant progression 3030
              </a>
            </td>
          <td>
            Matthew Bianca, Sara Lamorte, Teresa Ciudad, Cristiane Naffah de Souza, Rene Quevedo, Tracy Mcgaha
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4726aa2358332b95021c3fdf2dd12bc437304fe5" target='_blank'>
              Targeting CREB remodels the immune microenvironment to enhance immunotherapy responses in pancreatic cancer
              </a>
            </td>
          <td>
            S. Mehra, Supriya Srinivasan, Sudhakar Jinka, Varunkumar Krishnamoorthy, Vineet Kumar Gupta, Vanessa Garrido, Anna Bianchi, Andrew Adams, Manan Patel, H. Amirian, Luis A Nivelo, Karthik Rajkumar, Rimpi Khurana, Edmond W Box, Yuguang Ban, Oliver G McDonald, J. Datta, Kathleen E. DelGiorno, Nipun B Merchant, Sangeeta Goswami, A. Dosch, Nagaraj S. Nagathihalli
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f0d1b1f44b4f72bd47f0ccdb51001e256798703" target='_blank'>
              A Unified Single-Cell Atlas of HNSCC: Uncovering HPV and Sex Variability in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Cristina Conde Lopez, Divyasree Marripati, M. J. Besso, Mareike Roscher, Rui Han, W. W. Hadiwikarta, M. Elkabets, Jochen Hess, Ina Kurth
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Understanding therapy resistance requires deconvolving heterogeneous cell populations and tracking clonal trajectories. While CRISPR-based cellular barcoding is powerful for lineage tracing, many platforms suffer from low efficiency and limited compatibility with single-cell transcriptomics. We developed Oligo-CALL (Oligonucleotide-inducible CRISPR transcriptional activator–Assisted Lineage Labeling), an advanced barcoding system enabling precise lineage tracing, live clone isolation, and seamless integration with single-cell RNA sequencing. Applied to lung cancer cells treated with a KRASG12C inhibitor, Oligo-CALL identified clones consistently enriched posttreatment, supporting a model of predestined resistance. Oligo-CALL achieved >95% efficiency in linking lineage identity to transcriptomes, uncovering diverse clone-specific pathways with underlying resistance. Paired analysis of barcode-matched clones from naïve and resistant populations revealed transient and fixed resistance phenotypes. Notably, DNA repair pathways are recurrently altered in resistant clones, and inhibition of poly(adenosine 5′-diphosphate–ribose) polymerase synergizes with KRAS G12C inhibition to overcome resistance. Together, Oligo-CALL provides a versatile platform for dissecting lineage evolution and molecular dynamics of targeted therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fc6bc5d7c1ef3b43ef612b4e6dbe8df9b69936e" target='_blank'>
              Oligo-CALL: A next-generation barcoding platform for studying resistance to targeted therapy
              </a>
            </td>
          <td>
            Yingzhuo Liu, Yi Ban, Dingcheng Gao
          </td>
          <td>2025-11-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Imaging biological samples in three dimensions across scales is essential for capturing the complex spatial relationships that govern cancer initiation, invasion, and therapeutic response. As biological inquiry shifts from isolated molecular measurements toward spatially contextualized, multiomic profiling, new strategies have emerged to reconstruct tissue architecture at the whole-organ scale and subcellular resolution. These advances offer more anatomically faithful representations of tissue organization and open doors to integrating morphology with deep multiomic profiling in spatially resolved formats. As a result, we are improving our understanding of inter- and intratumoral heterogeneity and the key role of rare events and minority cell populations in tumor progression. The primary techniques used for 3D imaging of tumors include intact tissue imaging for targeted visualization of biological processes and serial sectioning for integration of diverse, multiomic platforms. In this review, we survey the major technologies used to image tumors in three dimensions, highlighting key methodologies, trade-offs, and recent innovations that make these approaches increasingly central to modern cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ec1eecfebb643d8f7b0229bfefe0ab87342355e" target='_blank'>
              Lessons from 3D Reconstruction of Cancer
              </a>
            </td>
          <td>
            Ashley L. Kiemen, Laura D. Wood
          </td>
          <td>2025-11-26</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cervical cancer (CC) is one of the most prevalent malignancies that affects women worldwide. It is strongly influenced by lifestyle and environmental factors, most notably persistent HPV infection and smoking. A defining feature of CC is its high degree of heterogeneity, which poses significant challenges for conventional sequencing and diagnostic approaches in fully capturing its oncogenetic landscape. The advent of single-cell RNA sequencing (scRNA-seq) has transformed this field, offering an unprecedented ability to dissect the tumor microenvironment at a single-cell resolution. By mapping cell–cell communication networks and characterizing gene expression across diverse cellular populations, scRNA-seq enables the identification of biomarkers with strong diagnostic and prognostic potential. Beyond biomarker discovery, this technology also facilitates the recognition of novel therapeutic targets, paving the way for more precise and effective treatment strategies. This review summarizes the applications of scRNA-seq in cervical cancer research, with a focus on methodological advances, key findings from recent studies, and their implications for improving diagnosis, prognosis, and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e029b6816cfdfef9fa928cd92a7a3e4aaf306a" target='_blank'>
              Implications of Single-Cell RNA Sequencing in Cervical Cancer: Unravelling the Molecular Landscape
              </a>
            </td>
          <td>
            Blessy Kiruba, Raja Rishi Raghavendar Raja Karthikeyan, Saurav Ram Anilkumar, Vino Sundararajan, S. Lulu S
          </td>
          <td>2025-12-02</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b0c553f54a8eda4d3dc1abe42f3db69a41cfe1" target='_blank'>
              Single-cell and spatial transcriptomic profiling reveals distinct tumor microenvironment dynamics in cervical adenocarcinoma and squamous cell carcinoma.
              </a>
            </td>
          <td>
            Yaqin Wu, Bei Wei, Zhentong Wei, Hongyan Wang, Yunyun Liu, Dongdong Xu, Xiaokang Li, Liyuan Guo, Huaiwu Lu, Huiling Shang
          </td>
          <td>2025-12-19</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cell plasticity, the dynamic ability of tumor cells to switch phenotypic states, drives tumor progression, metastasis, and therapy resistance, posing a formidable challenge to effective cancer management. This review comprehensively explores the genetic and epigenetic mechanisms underlying plasticity, highlighting how oncogenic mutations, chromosomal instability, and pathways like PI3K/AKT and RAS/MAPK enable adaptive behaviors such as epithelial-mesenchymal transition and stemness. Epigenetic regulators, including DNA methylation, histone modifications, and non-coding RNAs, amplify this adaptability, allowing rapid responses to therapeutic pressures. We examine the profound clinical implications, with plasticity fueling relapse in breast, lung, and prostate cancers, exacerbated by tumor microenvironment interactions. Challenges in detecting plastic states due to intratumor heterogeneity underscore the need for advanced diagnostics like single-cell genomics. Emerging interventions, from CRISPR-based gene editing and epigenetic therapies to patient-derived organoids and AI-driven combination strategies, offer transformative potential to disrupt plasticity and overcome resistance. This review illuminates the intricate genetic landscape of cancer cell plasticity, advocating for integrated approaches to revolutionize personalized oncology and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56a74b7b33fc882b2d862736314940182017fc5" target='_blank'>
              The Genetic Landscape of Cancer Cell Plasticity: Mechanisms, Implications for Therapy Resistance and Emerging Interventions
              </a>
            </td>
          <td>
            I. N. Monago, Nyerovwo Charity Okei, Chibuike Stephen Nzereogu, Taiwo Emmanuel Olowodasa, Tamasanni Omeiza Salawu, Ahmad Muhammad Garba, Doris Ada Ojum
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Biological and Pharmaceutical Sciences Archive</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Super-enhancers (SEs) are clusters of enhancers with potent regulatory capabilities. They play a crucial role in shaping cellular identity and driving the progression of various diseases, including cancer. SEs exhibit significant heterogeneity across different cell types and cancer subtypes. Analysis of SE landscapes can recapitulate existing classification systems and unveil novel SE-driven epigenetic subtypes. In this review, we summarized the latest advancements in cancer subtype identification based on SE-related characteristics, outlined the typical analytical workflows adopted in such studies, and explored the biological and clinical significance of SE-driven subtypes. Furthermore, we discussed the field’s key challenges and emerging technologies to highlight future research directions. SE analysis provides a robust framework for dissecting cancer heterogeneity. This approach offers novel epigenetic perspectives and support for the realization of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/285d6877af1dc808910fb0a7890277cebd5d2976" target='_blank'>
              Decoding cancer heterogeneity through super-enhancer landscapes: from subtype discovery to therapeutic opportunity
              </a>
            </td>
          <td>
            Shuyang Cai, Zhenchang Wang, Xiao Sun, Hao Huang
          </td>
          <td>2025-11-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumor immune microenvironment plays a critical role in tumor progression and responses to immunotherapy. Nevertheless, its cellular complexity and heterogeneity remain incompletely understood. In this study, we employed high-resolution single-cell RNA sequencing on CD45+ immune cells isolated from ten syngeneic murine tumor models, representing seven distinct cancer types under treatment-naïve conditions, thereby enabling a comprehensive profiling of tumor-infiltrating immune cells. We identified seven principal immune cell populations and provided an in-depth characterization of T cells, NK/innate lymphoid cells, dendritic cells, monocytes/macrophages, and neutrophils. Cross-species analyses further delineated conserved immune cell states and transcriptomic features within the T cell and monocyte/macrophage compartments that are shared across syngeneic models and human tumors. To investigate the functional relevance of the predominant monocyte/macrophage compartment and the notable presence of neutrophils in syngeneic tumors, we evaluated responses to anti-PD-1 therapy across various models and analyzed the enrichment of monocyte/macrophage subsets in tumors that responded to treatment. Furthermore, we conducted neutrophil depletion experiments using anti-Ly6G antibodies, administered both as monotherapy and in combination with PD-1 blockade. Remarkably, an interferon-stimulated gene-high (ISGhigh) monocyte subset was significantly enriched in models responsive to anti-PD-1 therapy. Neutrophil depletion resulted in variable antitumor effects across models but failed to enhance the efficacy of PD-1 blockade. In summary, our single-cell profiling offered a detailed atlas of the immune microenvironment across multiple syngeneic mouse tumor models, thereby enabling rational model selection for immuno-oncology studies. We uncovered an ISGhigh monocyte subset enriched in anti-PD-1 responsive models, and showed the context-dependent effects of neutrophil depletion on tumor immunity and immunotherapy, underscoring the heterogeneity and functional divergence of immune cell sublineages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f925e5b4fe351a72672a38a0505d78e0be84e52" target='_blank'>
              Single-cell atlas of the tumor immune microenvironment across syngeneic murine models
              </a>
            </td>
          <td>
            Jia Wang, Bin Jiang, Minjuan Deng, Han Yan, Peilin Zhang, Wei Jin, Zhirong Shen
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05c188c0c6e2f128d8772ff9b6be6a1353be8fc1" target='_blank'>
              A Hybrid Multiscale Model for Predicting CAR-T Therapy Outcomes in Solid Tumors
              </a>
            </td>
          <td>
            M. Nikmaneshi, Lance L. Munn
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Endometrial cancer is regarded as one of the most prevalent malignancies in women globally. Despite the advancements brought by multiple therapeutic strategies, the efficacy and effectiveness of treatment still appear to be diminished. Therefore, this accentuates the importance of understanding the tumour microenvironment (TME). Conventional methods, including immunohistochemistry (IHC), along with recently emerged techniques like single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST), have rejuvenated the notion of deciphering TME. From deconstructing tumour heterogeneity, identifying cell populations that play significant roles in treatment response, to discovering key biomarkers and therapeutic targets, these technologies lay the base for innovations. Importantly, this is the first comprehensive review that brings together scRNA-seq, ST, and IHC in Endometrial cancer. Merging these methods is geared towards creating a better grasp of the dynamics and interactions between diverse cells and the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bef0a0a3a7a3aad177138a9aca3da36e10bf85b" target='_blank'>
              Navigating tumour microenvironment in endometrial carcinoma: a comprehensive review integrating immunohistochemistry, single-cell RNA-sequencing and spatial transcriptomics
              </a>
            </td>
          <td>
            Oishee Mondal, Blessy Kiruba, Sajitha Lulu Sudhakaran, V. Sundararajan
          </td>
          <td>2025-11-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The spatial organization of the tumor microenvironment (TME) encodes the selective pressures that drive cancer evolution, but many tools focus on local cell contacts or single-modality readouts. We developed Tumor Landscape Analysis (TLA), a modular framework that borrows from landscape ecology and spatial statistics to quantify heterogeneity, fragmentation, and diversity across scales, from single cells to whole-slide ecosystems.



 TLA ingests whole-cell or nuclear centroids and region-level segmentations to compute (i) point statistics such as nearest-neighbor distance, Ripley’s H, and Morisita-Horn colocalization; (ii) kernel-smoothed local spatial profiles including Getis-Ord Gi* hot and cold spots; and (iii) fragmentation, shape, and diversity metrics such as patch density, contagion, shape index, and Shannon entropy. We also introduce Local Microenvironments (LMEs), unsupervised ecological niches defined by joint cell-type abundance and spatial mixing. Pilot analyses were performed in ductal carcinoma in situ (DCIS; n=353 samples, 175 patients) with cell-level maps of epithelial, lymphocyte, and fibroblast populations, and in pancreatic ductal adenocarcinoma (PDAC; n=203 resected cases after neoadjuvant therapy) using cancer and stroma segmentations.



 In DCIS, spatial relationships rather than raw abundance associated with recurrence risk. Shorter lymphocyte-to-fibroblast minimum nearest-neighbor distance and lower Ripley’s H, indicating greater intermixing, correlated with earlier recurrence independent of cell counts. This highlights immune–stromal proximity as a candidate eco-evolutionary driver of persistence. In PDAC, tissue-level fragmentation captured prognostic architecture after therapy. Two shape-based signatures emerged: (1) stromal uniformity, where lower standard deviation of stromal patch shape index associated with improved disease-free survival, consistent with a more constrained, less reactive stroma; and (2) tumor geometric complexity, where higher mean cancer patch shape index, reflecting more irregular or dendritic nests, associated with lower recurrence risk, whereas compact nests predicted worse outcomes. Across datasets, LME maps localized niche phenotypes, such as mixed-immune or segmented tumor habitats, that align with hypothesized selection landscapes and provide inputs for fragmentation analysis.



 TLA operationalizes an eco-evolutionary view of cancer by unifying interpretable, multiscale spatial metrics in a single workflow that works with any rasterized data input. By quantifying how habitats, edges, and mixing patterns are organized, TLA reveals niche structures and architectural motifs linked to clinical outcomes and complements foundation models and modality-specific pipelines with theory-grounded, clinically translatable descriptors. Ongoing work integrates TLA features with molecular data and outcomes to derive robust biomarkers and to inform adaptive, niche-targeted strategies in precision oncology.



 Ryan M. Carr, Luis Cisneros, Merih Deniz. Toruner, Martin E. Fernandez-Zapico, Carlo Maley. Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f5d8ed56157756f83dcf8a730cae9e8b632c46" target='_blank'>
              Abstract A015: Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments
              </a>
            </td>
          <td>
            Ryan M. Carr, L. Cisneros, M. D. Toruner, Martín E Fernandez-Zapico, Carlo C. Maley
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Hepatocellular carcinoma (HCC) is a deadly and highly complex cancer. To better understand what makes it aggressive, we investigated the key genetic switches, known as transcription factors (TFs), that control tumor cell behavior. By analyzing vast amounts of genetic data from single cells, we identified a core network of nine TFs that drive HCC progression. We found that these TFs orchestrate a shift in tumor cells from a state of rapid growth to a more invasive and metabolically adapted state. This change also reshapes the tumor’s surroundings, creating an environment that suppresses the immune system, largely by recruiting and activating specific immune cells called SPP1+ macrophages. This coordinated action between tumor cells and immune cells promotes cancer growth and immune evasion. Our findings highlight this TF–macrophage axis as a promising new target for developing more effective HCC therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989aaab1ae1f0d6fa871b18d9e4cad44d18f7594" target='_blank'>
              Transcription Factor Networks Drive Tumor Progression and Immune Microenvironment Remodeling in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Sang Hoon Lee, Juwon Ahn, Wonbin Choi, Jina Kim, Joon Yeon Hwang, Jae-Hwan Kim, Hyaekang Kim, Woori Kwak
          </td>
          <td>2025-11-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Spatial Multiomics allows for context-based study of the tumor microenvironment (TME). TME consists of diverse cells that collectively influence tumor progression and immune activation. CD8 T-cells, essential for antitumor immunity, often become exhausted in the TME. PD1+ TCF1+ stem-like CD8 T-cells retains differentiation potential and can be rejuvenated through cytokine signaling and immune checkpoint modulation. Here we introduce a workflow for RNAscope™ Multiplex Fluorescent v2 assay combining RNA in-situ hybridization (ISH) and protein immunofluorescence to study cell marker (CD8, PD1) with RNA targets (TNFA, TCF7, IFNG) in a tumor microarray (TMA). The Manual Multiplex Protease-Free workflow enables simultaneous RNA and protein visualization without enzymatic disruption, preserving antigen integrity and tissue morphology. Using this Multiomics assay, we can study CD8 T-cell differentiation and activation. Co-detection of RNA transcripts and proteins provides insights into specific T cell phenotypes and their activation/exhaustion status. Data reveal robust co-expression patterns of these targets, particularly in the tumor-cell neighboring stromal and immune cells in breast, cervical and stomach cancers, reflecting immune activation and exhaustion across distinct TME niches. This methodology lays a foundation for refining immunotherapeutic strategies targeting PD1+ TCF7+ CD8 T-cells, enhancing effector functions, and advancing precision oncology interventions.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bea7d4ce3faf01c213d46ecb8b271a789ba794e6" target='_blank'>
              Spatial multiomics reveals T-cell activation and exhaustion states in the tumor microenvironment 3446
              </a>
            </td>
          <td>
            Steve Zhou, Yifan Wang, Debia Wakhloo, A. Dikshit, Li-chong Wang, Maithreyan Srinivasan
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Clear cell renal cell carcinoma (ccRCC) exhibits significant heterogeneity due to morphological changes and tumor microenvironment dynamics, influencing systemic therapy responses. While the role of high-mobility group AT-hook 2 (HMGA2) in tumor progression has been implicated in other cancers, its significance in ccRCC remains unclear. This study investigates the role of HMGA2 in these processes and its clinical impact. Methods Spatial transcriptomics (ST) was performed on primary ccRCC samples to investigate expression trajectories associated with HMGA2 expression and morphological evolution. In metastatic ccRCC cohorts treated with systemic therapy, immunohistochemistry and bulk RNA sequencing data were analyzed to evaluate molecular and clinical features in relation to HMGA2. Single-cell RNA sequencing (scRNA-seq) data were used to explore immune cell populations and their interactions. Based on these findings, multiplex immunohistochemistry (mIHC) assessed spatial distribution, cell–cell interactions, and pathological responses of key immune populations. Results HMGA2 expression was associated with aggressive morphological patterns, such as solid sheets and rhabdoid/sarcomatoid. ST revealed a progressive increase in HMGA2 expression along the morphological trajectory, marked by a shift from clear to eosinophilic cytoplasm, with eccentric nuclei and prominent nucleoli, and loss of vascular architecture. HMGA2-high tumors exhibited aggressive phenotypes driven by cell cycle, epithelial-mesenchymal transition, and inflammatory signaling pathways. Clinically, patients with high HMGA2 had worse progression-free survival but responded better to immune checkpoint inhibitor combination (Combo-ICI) therapy than to tyrosine kinase inhibitor monotherapy. To assess the immune landscape, scRNA-seq data revealed that HMGA2-high tumors were enriched with progenitor exhausted CD8+ T cells (Tpex), along with increased frequencies of conventional dendritic cell type 1 (cDC1) and inflammatory cDC type 2, which were found to interact with Tpex via ICAM-1. mIHC confirmed that Tpex were enriched among Combo-ICI responders in HMGA2-high tumors, with higher densities and closer proximity to ICAM-1+ cDC1. Conclusions These findings suggest that dynamic HMGA2 expression contributes to morphological evolution and modulates immune responses through enhanced Tpex–cDCs engagement, serving as a potential marker for systemic therapy response in ccRCC. However, additional experimental studies are required to validate these mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c12fc33545db9ae9db8d540caf1522acd61b8f02" target='_blank'>
              HMGA2 links morphological evolution and microenvironment dynamics to systemic therapy response in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            T. Nakamoto, Takashi Yoshida, C. Ohe, Yoshiki Yasukochi, Naho Atsumi, Takeshi Sano, K. Higasa, Katsunori Uchida, Koji Tsuta, Hidefumi Kinoshita
          </td>
          <td>2025-11-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Tumor progression is driven not only by the complex milieu of transcriptional cell states of distinct lineages that populate the tumor microenvironment (TME) but also by the interactions these states forge with one another. Systematically charting the landscape of such prognostic cell-state interactions (CSIs) remains difficult as single-cell datasets are small and rarely linked to clinical outcomes, whereas bulk profiles obscure cellular context. We developed CSI-TME, a computational pipeline that adapts the concept of synthetic lethality to infer prognostic cell-state interactions by directly utilizing the large cohorts of bulk transcriptomic datasets. We applied CSI-TME to IDH-mutant gliomas, deriving a highly reproducible cell-state interaction network (CSIN) that is predominantly pro-tumor and differentially activated in IDH-mut astrocytoma versus oligodendroglioma. Malignant cells within the CSIN resemble multiple neuronal lineages including, astrocyte-like and oligodendrocyte-progenitor-like program, and reveal key interactions between glioma stem cells and T cells. CSIN stratifies patient response to immune-checkpoint blockade therapy. Roughly 20% of interactions involve direct ligand-receptor communication, and their participating cell states significantly co-localize in spatial-transcriptomic datasets, most notably for a pro-tumorigenic interaction between tip-like endothelial cells and hypoxic malignant cells involving multiple ligand-receptor interactions. Integration with somatic-mutation profiles reveals that many interactions in CSIN exert anti-tumor effect early-on in glioma but shift toward pro-tumor roles as the disease progresses. Taken together, CSI-TME provides a unique way of identifying the prognostic and clinically relevant cell state interactions without the need for extensive single cell datasets and provides a framework for prioritizing therapeutic ligand-receptor targets, patient stratification, while providing new insight into how interactions among the cell states shape the TME in IDH-mutant glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3146cb53ad109ab0b9473e62c6d93d13f03288e3" target='_blank'>
              TMIC-77. Identification of Clinically Relevant Cell State Interactions in the Tumor Microenvironment of IDH-Mut Glioma
              </a>
            </td>
          <td>
            Arashdeep Singh, Bharati Mehani, Vishaka Gopalan, Ken Aldape, Sridhar Hannenhalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Cancers are characterized by genomic instability events such as aneuploidy, chromosomal arm imbalances, and segmental copy number changes. These genomic features frequently define different subclones within a tumor. Single-cell DNA sequencing (scDNA-seq) identifies these large-scale genomic alterations that define subclonal features. However, scDNA-seq does not provide biological phenotypic information on individual subclones. Single-cell RNA sequencing (scRNA-seq) offers biological information but is less accurate in discovering genomic instability events. We developed a computational framework, scAlign, for integrating scRNA-seq and scDNA-seq from the same specimen and define subclonal cellular phenotypes at the resolution of individual cells. Subclones were defined by aneuploidy and chromosomal arm imbalance among primary and metastatic cancers. Using the cells in the G0/G1 phase, the extensive cellular sampling from both assays characterized the subclonal architecture of these cancers. The scDNA-seq provided a ground truth for copy number-based subclones. From the scRNA-seq data, the epithelial cells in G0/G1 were identified and assigned to specific subclones by the scAlign based on gene dosage. Afterward, we determined the differential gene expression and biological pathway activities of specific clones. Overall, integrative multi-omics analysis of single-cell datasets is more informative than any individual genomic modality, provides deep insights into intratumoral heterogeneity and reveals subclonal biology. scAlign is available at https://github.com/XQBai/scAlign.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a91a097601e22d4ca33e4ce8206425906a9876da" target='_blank'>
              Single-cell aneuploidy and chromosomal arm imbalances define subclones with divergent transcriptomic phenotypes
              </a>
            </td>
          <td>
            Xiangqi Bai, Billy T. Lau, A. Sathe, Susan M. Grimes, Alison Almeda-Nostine, Hanlee P. Ji
          </td>
          <td>2025-10-31</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Molecular heterogeneity in head and neck squamous cell carcinoma (HNSCC) is well recognized, yet existing subtype frameworks remain largely descriptive and have not translated into therapeutic decision-making. Here, we establish a mechanistic platform that converts transcriptomic diversity into drug-actionable tumor states. Integrating multi-cohort RNA-seq from 727 tumors across five independent datasets, genome-scale CRISPR dependency maps, and pharmacologic screening, we define distinct tumor survival circuits across HPV-negative HNSCC and nominate subtype-matched therapeutic strategies. These circuits encompass a proliferative axis (MYC, MET/FAK, inflammatory and translational programs), an epithelial-differentiated/adhesion program, an EMT-like state with stromal activation, and mitochondrial/oxidative metabolic states, each mapping to selective liabilities (e.g., mitotic/autophagy control, ERBB/PI3K and cadherin signaling, OXPHOS/mitochondrial translation, and G2/M-integrin-Notch pathways, respectively). We then develop a transcriptomic predictor of EGFR-inhibitor response using machine learning and validate it in prospectively collected, fresh patient-derived 3D microtumors. The resulting 13-gene signature identifies erlotinib-responsive tumors (R = 0.93) and maps biologically to an epithelial-differentiated state, outperforming EGFR expression alone. Our study establishes a subtype-to-dependency-to-therapy framework, enabling precision stratification and providing a clinically feasible path for prospective biomarker deployment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29377070a0d6fb125a75d4b7cf89275cb55faab" target='_blank'>
              Subtype-Specific Dependencies and Drug Vulnerabilities Enable Precision Therapeutics in Head and Neck Cancer
              </a>
            </td>
          <td>
            Joel M Vaz, Songli Zhu, Mateo Useche, Lauren Shih, Emily Marchiano, Slobodan Beronja, Bruce E Clurman, Cristina Rodriguez, Brittany Barber, Marina Chan, Taranjit S. Gujral
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Aberrant tumor glycosylation can alter immune recognition; however, the specific influence of glycan-lectin interactions on tumor progression remains poorly understood. Here, we identify the C-type lectin receptor CD301b (encoded by Mgl2 ) as a regulator of immune activity within the breast tumor microenvironment (TME). Using a murine triple-negative breast cancer model, we demonstrate that tumors expressing the Tn glycoantigen grow more rapidly, and this growth is facilitated by CD301b⁺ immune cells. Depletion or genetic loss of CD301b markedly suppressed tumor growth, indicating that CD301b promotes tumor progression through myeloid-tumor interactions. Phenotypic analyses revealed that CD301b⁺ cells within tumors are type 2 conventional dendritic cells (cDC2s), a subset known to influence immune polarization. Single-cell RNA sequencing of human breast cancers showed that the human ortholog CLEC10A is expressed in cDC2-like dendritic cells and select macrophage subsets, suggesting a conserved role for CD301⁺ myeloid populations. Transcriptomic profiling of tumors developed in Mgl2 -deficient mice revealed a shift toward an inflammatory, immune-activated state consistent with enhanced antitumor immunity. Together, these findings establish a link between tumor glycosylation and lectin signaling of myeloid cells, highlighting CD301b as a potential target for reprogramming the tumor immune microenvironment in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401b567312697ac0756ae3899ebeedb84ca70dd0" target='_blank'>
              CD301b lectin expression in the breast tumor microenvironment augments tumor growth
              </a>
            </td>
          <td>
            A. Ozdilek, A. Paschall, Zahra Nawaz, Afaq M. M. Niyas, F. Avci
          </td>
          <td>2025-12-03</td>
          <td>Research Square</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 Understanding the immune microenvironment requires precise delineation of immune cell populations and their spatial interactions within tissues. Advanced spatial-omics technologies enable simultaneous analysis of cellular biomarkers, spatial contexts, and tissue architecture. However, current analytical tools fall short in integrating this multidimensional data, limiting insights into immune cell functionality and heterogeneity. To address this, we present CellSNAP (Cell Spatio- and Neighborhood-informed Annotation and Patterning), a self-supervised computational framework that integrates molecular expression, spatial neighborhood contexts, and local histological features using graph neural networks. CellSNAP enables the de novo discovery of fine-grained immune cell subpopulations, overcoming the constraints of predefined cell atlases. Applying CellSNAP to spatial-omics datasets across various tissues, including spleen and lymphoma, revealed immune subpopulations with distinct spatial distributions and functional states, such as tumor-associated macrophages and spatially stratified B and T cells. These findings underscore the utility of integrating spatial and molecular data to advance immunological research, offering a robust tool for unraveling immune cell dynamics in health and disease.



 1P01AI177687-01, 75N93019C00071, DP2AI171139, R01AI149672, U24CA224331, NSF 2345215, NSF 2245575.



 Technological Innovations in Immunology (TECH)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6caeae1dc09bc627a0349265ceed8d4f08e8abb" target='_blank'>
              Cross-domain information fusion for enhanced immune cell population delineation in single-cell spatial-omics data 2975
              </a>
            </td>
          <td>
            Bokai Zhu
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d4e23f0e2eb9b451cd70e81ca9b082a2f8b6469" target='_blank'>
              Advancements in single-cell sequencing for cervical cancer research.
              </a>
            </td>
          <td>
            Congli Pu, Biyuan Xing, Shujie Wang, Zhao Liu, Yingchao Zhao
          </td>
          <td>2025-11-06</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Low-grade gliomas (LGGs) exhibit low overall mutational burden and an immunosuppressive microenvironment, contributing to immunotherapy resistance. Intratumoral heterogeneity (ITH) further complicates the targeting of tumor-specific antigens (TSAs), yet its influence on the antigenic and immune landscapes in LGG remains understudied, particularly due to reliance on single-biopsy analyses.



 We performed exome and transcriptome sequencing on 70 spatially mapped biopsies from seven IDH-mutant Grade II astrocytoma patients. Tumor purity, immune deconvolution, and neoantigen prediction were conducted to assess regional immune variation and antigenic burden. In one patient (P375), neoantigen-specific CD8+ T cells were isolated using barcoded peptide-HLA multimers. Reactive T-cell receptor (TCR) clonotypes were identified via single-cell TCR sequencing and functionally validated in Jurkat76 and CD8+ T cells.



 Spatial profiling of whole tumors revealed most mutations were biopsy-specific, with a steep drop in mutations shared across multiple regions. RNA-seq–based unsupervised clustering with xCell and DESeq2 revealed two distinct immune microenvironments: immune-hot regions with elevated immune infiltration, and immune-cold regions with minimal immune presence. Notably, mutation-derived n-mer peptides from immune-hot regions showed enhanced predicted immunogenicity relative to those from immune-cold areas. In one patient (P375), neoantigen-specific CD8⁺ TCR clonotypes were identified targeting PRMT5 mutations found in immune-hot regions. A mutant PRMT5-specific TCR demonstrated robust, antigen-dependent activation and dose-dependent cytotoxicity in vitro. This TCR showed high specificity toward the mutant peptide with minimal cross-reactivity to the wild-type sequence. Furthermore, engineered CD8⁺ T cells effectively killed glioma cells endogenously expressing the full-length mutant PRMT5, supporting its candidacy as a viable immunotherapeutic target.



 Our findings reveal significant 3D spatial and immunologic heterogeneity in low-grade gliomas and demonstrate that neoantigens such as mutant PRMT5 can be selectively targeted by highly specific, cytotoxic TCR-engineered CD8⁺ T cells. These results support the potential of personalized TCR-based immunotherapy for low-grade glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fa3c48068f63d55f21f844232ecea3f66c8ffb8" target='_blank'>
              EXTH-52. Spatially Resolved Antigenic Landscapes In Low-grade Glioma Reveal Targets For Personalized Immunotherapy
              </a>
            </td>
          <td>
            Darwin W. Kwok, Michael Zhang, Cliff Wang, Maggie Colton Cove, Chibo Hong, Nicholas O. Stevers, Benjamin Lerman, S. Lakshmanachetty, Tyler Borrman, Zheng Pan, Benjamin Yuen, Songming Peng, Diana Nguyen, Michael Martin, S. Hilz, Joanna Phillips, Anny Shai, N. Bush, S. Hervey-Jumper, Michael McDermott, S. Mandl, H. Okada, Joseph F Costello
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Tumor fibrosis is recognized as a malignant hallmark in various solid tumors; however, the clinical importance and associated molecular characteristics of tumor fibrosis in liver metastases (LM) from colorectal cancer (CRLM) remain poorly understood. Here we show that patients with CRLM whose liver metastases (LM) exhibited tumor fibrosis (Fibrosis+ LM) had significantly worse progression-free survival (P = 0.025) and overall survival (P = 0.008). Single-cell RNA sequencing revealed that the tumor microenvironment of the Fibrosis+ LM was characterized by T cells with an exhausted phenotype, macrophages displaying a profibrotic and suppressive phenotype and fibrosis-promoting fibroblasts. Further investigation highlighted the pivotal role of VCAN_eCAF in remodeling the tumor fibrosis in the tumor microenvironment of Fibrosis+ LM, emphasizing potential targetable interactions such as FGF23 or FGF3-FGFR1. Validation through multiplex immunohistochemistry/immunofluorescence and spatial transcriptomics supported these findings. Here we present a comprehensive single-cell atlas of tumor fibrosis in LM, revealing the intricate multicellular environment and molecular features associated with it. These insights deepen our understanding of tumor fibrosis mechanisms and inform improved clinical diagnosis and treatment strategies. The liver is a common site for cancer spread, especially from colorectal cancer. This study looks at how fibrosis in liver metastases affects treatment and survival. Researchers studied 471 patients with liver metastases from colorectal cancer. They examined tissue samples to see how fibrosis impacts survival and used advanced techniques such as single-cell RNA sequencing to understand the tumor environment better. They found that patients with more fibrosis had worse survival rates. The study also revealed that fibrosis changes the tumor environment, making it harder for the immune system to fight cancer. This includes an increase in certain immune cells that suppress the body’s defense against tumors. The researchers suggest that targeting fibrosis could improve treatment outcomes. In the future, therapies that reduce fibrosis might help enhance the effectiveness of cancer treatments and improve patient survival. This summary was initially drafted using artificial intelligence, then revised and fact-checked by the author.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253b52aabc9f910e8e84f8a906624aeebc79b3e8" target='_blank'>
              Single-cell transcriptomic profiling reveals liver fibrosis in colorectal cancer liver metastasis
              </a>
            </td>
          <td>
            Yiqiao Deng, Chengyao Guo, Xiaomeng Liu, Xin Li, Jianmei Liu, Wenjie Liu, Jinghua Chen, Zhen Huang, Ye‐fan Zhang, Xinyu Bi, Jian‐jun Zhao, Jianguo Zhou, Zhi-yu Li, Hongliang Wu, Baocai Xing, Qichen Chen, Hong Zhao
          </td>
          <td>2025-11-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) enables simultaneous mapping of tissue morphology and spatially resolved gene expression, offering unique opportunities to study tumor microenvironment heterogeneity. Here, we introduce a computational framework that predicts spatial pathway activity directly from hematoxylin-and-eosin-stained histology images at microscale resolution 55 and 100 um. Using image features derived from a computational pathology foundation model, we found that TGFb signaling was the most accurately predicted pathway across three independent breast and lung cancer ST datasets. In 87-88% of reliably predicted cases, the resulting spatial TGFb activity maps reflected the expected contrast between tumor and adjacent non-tumor regions, consistent with the known role of TGFb in regulating interactions within the tumor microenvironment. Notably, linear and nonlinear predictive models performed similarly, suggesting that image features may relate to pathway activity in a predominantly linear fashion or that nonlinear structure is small relative to measurement noise. These findings demonstrate that features extracted from routine histopathology may recover spatially coherent and biologically interpretable pathway patterns, offering a scalable strategy for integrating image-based inference with ST information in tumor microenvironment studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29b8964f22d869e7cd99fbb783574fbfe821242b" target='_blank'>
              Digital Modeling of Spatial Pathway Activity from Histology Reveals Tumor Microenvironment Heterogeneity
              </a>
            </td>
          <td>
            Ling Liao, Changhuei Yang, Maxim Artyomov, Mark Watson, Adam Kepecs, Haowen Zhou, Alexey Sergushichev, R. Cote
          </td>
          <td>2025-12-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6c4f2ec4971acec7d7b9addcd8348e71b0ec1ff" target='_blank'>
              Decoding epithelial-T cell interactions in colorectal cancer through single-cell and spatial transcriptomics.
              </a>
            </td>
          <td>
            Rentao Yu, Xinyu Lu, Zhuangzhuang Yue
          </td>
          <td>2025-11-28</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72005e917b50c2e5c4b78fb42f30196a93fe8580" target='_blank'>
              Transcriptional Profiling of Commonly Used Liver Cancer Cell Lines Reveals Disease-Specific Modeling Potential and Authentication Concerns
              </a>
            </td>
          <td>
            Nima Sarfaraz, Gail Rosen, Trevor Winkler, Risha Patel, Srinivas Somarowthu, Michael Bouchard
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f6f13a57b79f0fdbdce54ed88ac9cc1b39fa61" target='_blank'>
              Immune landscape-driven subtyping reveals distinct microenvironment and prognostic profiles in thymic epithelial tumors.
              </a>
            </td>
          <td>
            Yajie Zhang, Tong Lu, Yeke Huang, Xipeng Wang, Xiaoyan Chen, Wei Guo, Youqiong Ye, Hecheng Li
          </td>
          <td>2025-12-06</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Triple-Negative Breast Cancer (TNBC) encompasses a biologically heterogeneous group of tumors, which can be classified into distinct molecular subtypes, namely basal-like 1 (BL1), basal-like 2 (BL2), immunomodulatory (IM), mesenchymal (M), mesenchymal stem-like (MSL), and luminal androgen receptor (LAR), with unique clinical and pathological characteristics. While immune features of these subtypes have been extensively characterized, the integration of non-immune stromal and structural components into our understanding of TNBC biology is only now being fully recognized. This narrative review synthesizes current evidence regarding differences in the non-immune microenvironment across TNBC molecular subtypes, with a focus on cancer-associated fibroblasts (CAFs), vascular features, extracellular matrix (ECM) dynamics, and epithelial–mesenchymal transition (EMT), along with metabolic–hypoxic reprogramming. Data from several studies are integrated to highlight subtype-specific signatures. Differences in stromal architecture and metabolic adaptations, potentially reflecting the underlying molecular heterogeneity, may hold prognostic or predictive significance and could inform personalized therapeutic strategies targeting the tumor–stroma interface.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/089f130c819509e80723565570647cc30d7f85ba" target='_blank'>
              Dissecting the Non-Immune Tumor Microenvironment in Triple-Negative Breast Cancer: Molecular Subtype-Specific Patterns and Prognostic Implications
              </a>
            </td>
          <td>
            A. Syrnioti, Eleni Timotheadou, Vasileios N. Papadopoulos, Georgia Syrnioti, T. Koletsa
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 The immune receptor repertoire captures the fingerprint of immune responses shaped by pathogens, malignancies, and autoimmunity. Advances in spatial transcriptomics have enabled the study of immune repertoires within their spatial context, linking cellular localization to functional diversity. To support this, we present CIRCUSS (Comprehensive Immune Receptor Curation and Spatial Sequencing), an open-source Python package that allows for the robust analysis and visualization of spatial immune cell repertories, quantifying T- and B-cell receptor clonal diversity, expansion, and spatial distribution within tissues. CIRCUSS provides tools for annotating immune repertoires, tracking clonotypes across geographies, and detecting immune cell spatial clustering relative to the tumor microenvironment. Its modular, user-friendly commands enable seamless integration of spatial immune repertoire data with transcriptomics and histological analyses, enhancing the biological interpretation of immune responses. We demonstrate CIRCUSS’s utility by analyzing publicly available spatial datasets from targeted and whole transcriptome detection approaches across tissue sections from human lymphoid tissue, brain metastases, renal cell carcinoma, and melanoma uncovering how immune receptor variation and spatial organization drive localized immune responses. CIRCUSS enables novel insights into immune expansion mechanisms and therapeutic resistance, fostering broader exploration of spatial immunology.



 Research funded by Lustgarten Foundation and Break Through Cancer foundation.



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93751fb379adef50114d2b78cf230f4c349805f1" target='_blank'>
              CIRCUSS, a spatial immune receptor computational analysis tool, enables the characterization of therapeutic resistance to cancer immunotherapies 4342
              </a>
            </td>
          <td>
            Benjamin F. Green, Janelle M. Montagne, Mulan Bell, J. Taube, Elizabeth M. Jaffee, Luciane T. Kagohara, E. Fertig, Dimitri N. Sidiropoulos
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Although non-muscle-invasive bladder cancer (NMIBC) frequently recurs and progresses despite BCG therapy, cellular mechanisms behind this remain unclear. Understanding the dynamics within the tumor microenvironment (TME) and identifying pathways associated with BCG resistance are crucial for improving NMIBC treatment. To elucidate transcriptional dynamics and key cellular interactions in the TME, we performed longitudinal single-nucleus RNA sequencing of 9 NMIBC cases with three pairs of treatment-naïve and disease progression samples. The resulting 58,037 nuclei were subject to cell type and subtyping annotations along with cell–cell interaction analyses. Our analysis revealed marked interpatient heterogeneity in malignant cells, accompanied by copy number alterations associated with disease progression. TGF-β signaling emerged as a key pathway, showing gradual enrichment from pre-treatment to post-progression samples. Within the tumor microenvironment, we identified distinct subtypes including LAMP3 + dendritic cells and iCAF fibroblasts that followed unique pseudotime trajectories linked to progression. Cell–cell interaction analysis further highlighted critical ligand–receptor pairs such as DSC2–DSG2 and ENG–BMPR2, which were associated with poor clinical outcomes. These interactions may represent hypothesis-generating candidates that warrant further validation. This study highlights key cellular and molecular mechanisms underlying bladder cancer progression Post-BCG treatment. Through single-nucleus RNA sequencing, we identified critical TME subtypes, TGF-β signaling involvement, and crucial ligand-receptor interactions such as DSC2-DSG2 and ENG-BMPR2, which may serve as preliminary biomarkers requiring confirmation in larger independent cohorts. These findings offer valuable insights into enhancement of treatment strategies and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b994449406e351724039c2b4ed65a466872b5c" target='_blank'>
              TGF-β signaling and tumor microenvironment dynamics in bladder cancer progression post-BCG therapy: a longitudinal single-nucleus RNA-seq study
              </a>
            </td>
          <td>
            Seo-Young Lee, Yun-Hee Lee, Tae-Min Kim, U-Syn Ha
          </td>
          <td>2025-11-10</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 MAPK pathway alterations occur in 10-20% of diffuse midline glioma, H3K27-altered (DMG-H3K27), resulting in a proposed novel subtype of the disease with distinct clinical features. In a previous co-clinical trial platform, we showed an increased sensitivity to MEK inhibitors in DMG-MAPK patient-derived models, though resistance occurred in vitro, in vivo and clinically. To understand the nature of these resistance mechanisms, we used sophisticated coupling of retrievable high-complexity barcoding and single cell RNA sequencing in DMG-MAPK cells. Lineage tracing conducted in several technical replicate experiments showed a subset of clones is consistently enriched across replicates when treated with a GI90 concentration of trametinib for 11 weeks. Interrogation of clonal dynamics using mathematical modelling showed barcodes commonly enriched across replicates exhibit significantly higher clonal fitness compared to replicate-specific enriched barcodes, suggesting a deterministic pattern of resistance. Serial single cell RNA sequencing analysis at weeks 5, 8 and 11 post-treatment initiation showed some resistant H3.3K27M/BRAF-mutant cells adopted astrocyte-like and neural progenitor-like metaprograms in which barcodes were commonly enriched across two cell states, suggesting a subset of clones circumvent trametinib treatment by phenotypic switching; simultaneously, a late-emerging resistant population of cells activate a distinct mesenchymal program. Furthermore, we determine the previously identified MAP2K1 (MEK1) mutations to arise specifically from a small population of radial glia-like stem cells that appear acutely predisposed to the acquisition of these mutations. Together, our data suggest two clonally and mechanistically distinct resistance mechanisms can be active in DMG-MAPK in response to MEK inhibition, involving both transcriptional plasticity and a progenitor-like state primed for the acquisition of genetic determinants which circumvent drug response. With trametinib-resistant cells being highly sensitive to selective SRC inhibition, evolutionary-guided combinatorial treatment schedules may provide for strategies to significantly lengthen time taken for resistance development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c54926c3fb10216fcf5c70e35738abc3a296f32" target='_blank'>
              DDDR-39. Untangling MEK inhibitor resistance in DMG-MAPK using coupled lineage tracing and single cell RNA sequencing
              </a>
            </td>
          <td>
            Molina Das, Y. Grabovska, Haider Tari, A. Mackay, Rebecca F Rogers, Jared Collins, Jiin Song, Laura Bevington, A. Burford, Drenusha Sejdiu, R. Beroukhim, P. Bandopadhayay, Chris Jones
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>152</td>
        </tr>

        <tr id="


 Despite the remarkable success of targeted therapy, the emergence of acquired resistance remains a major obstacle to cure acute myeloid leukemia (AML) patients. While existing studies have predominantly focused on genetic heterogeneity as a driver for therapy resistance, recent evidence suggests that non-genetic persister state is equally important, but such studies are lacking in AML. Here, we define the temporal transcriptome and chromatin landscape along with lineage trajectory of individual genetically identical AML persister clones to existing anti-AML therapeutics.
 To model persister state in AML, we utilized watermelon library (Oren et al., 2021), that expresses lineage barcode with inducible Histone-2B-mCherry to trace clonal evolution and cell division. We first evaluated whether slow-cycling, quiescent persisters emerge in AML lines after exposure to standard therapy. Our data showed the emergence of slow cycling persisters with cytarabine (AraC) in all 4 cell lines, and gilteritinib (GLT) in MOLM-13 and MV4-11 cells while BH3 mimetics (venetoclax (VEN), S63845 and navitoclax) had fast-cycling persisters. Integrating proliferative-index profiles with cell cycle profiles depicted extensive heterogeneity in AML persisters imposed by drug target rather than genetic state. Drug holiday experiments exhibited a reversible drug tolerance state in both slow and fast-cycling persisters. Longitudinal drug sensitivity and BH3 profiling assays showed that persisters emerged with adaptation to reduced mitochondrial priming that reversed after drug holiday. However, transitioning from persister to stable resistant cells resulted in a sustained loss in apoptotic priming, mirroring observations made in PDX models and primary tumors.
 To understand trajectory of persister emergence and underlying mechanisms, we performed single-cell multiomics on drug-naïve and persister populations (day 9) in MV4-11 (GLT), and OCI-AML2 (VEN and AraC). We expanded 10,000 unique barcoded cells and administered the drug treatment to derive persisters. Using lineage barcode analysis, we report significant reduction in the dominant lineage of day 0 and survival of multiple small lineages indicating an acquired state of persistence than selection of pre-existing resistant clone in both the lines. Further trajectory analysis defined three fates of the lineages as: expanding (log2FC>1), shrinking (log2FC <1) and stable. AraC persisters showed higher expanding and stable lineages, while VEN and GLT persisters resulted in higher shrinking and stable clones, revealing heterogeneity in lineage expansion. We next asked whether persisters resulting from different therapies arised from the same parental lineage. To test this, we performed lineage tracing on VEN and AraC persisters in OCI-AML2 and compared surviving lineages at Day 9 vs Day 0 cells. This analysis identified 52 parental lineages that were able to produce both VEN and AraC persisters in AML2, providing evidence of branched evolution of persisters from the same parental clone.
 We finally defined transcriptional and epigenetic heterogeneity in baseline and persister cells by performing UMAP on single-cell RNAseq and ATACseq data. We identified persisters to all 3 drugs clusters distinctly from their drug naïve counterparts at both transcriptome and chromatin accessibility. Next, we investigated differentially expressed pathways in persisters and observed higher inflammatory response pathways such as TNFα signaling in GLT, enrichment of MYC targets and oxidative phosphorylation in VEN and high mTORC1 signalling in AraC persisters. We observed higher chromatin accessibility in GLT and VEN whereas condensed chromatin in AraC persisters. We next asked which motifs are highly enriched in each persister and observed the enrichment of various KLF and SP family of motifs in each persister driving multiple gene signatures in each persister type. We also observed enrichment of LSC-primed and cDC like cells in GLT persisters but with AraC and VEN persisters a uniform pattern of all states was observed, indicated stemness may not be common mechanism to persister evolution.
 Collectively, our integrated single-cell multiomics and lineage tracing approach reveals that AML persisters emerge through stochastic drug-specific, non-genetic adaptive mechanisms characterized by distinct transcriptional and chromatin remodeling programs rather than deterministic Darwinian selection of pre-existing resistant cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b912c83b14f52bda36c8d81fafb30ac8e86a906c" target='_blank'>
              Dissection of non-genetic resistance mechanisms by lineage tracing and single cell multiomics reveals drug-specific persister programs in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            K. Bhatia, Xiao Xian Lin, Ankit Jana, Emanuelle I. Grody, Ryan Jeremy Lim Jing Wei, Shawn Koh Jia Jun, Elisha Teh Jia Yu, Cin thet Kyi, Prasad Iyer, Greg Kellogg, Yogesh Goyal, Lawrence Boise, Yaara Oren, Shruti Bhatt
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) exhibits early systemic dissemination that precedes clinical detection, challenging the classical model of metastasis as a late evolutionary event. Mounting evidence indicates that molecular and cellular programs enabling invasion and distant colonization emerge at preinvasive or early carcinoma stages. This mini-review synthesizes recent advances defining how tumor-intrinsic mechanisms, such as KRAS-driven basal extrusion, epithelial–mesenchymal plasticity and chromosomal instability, cooperate with microenvironmental cues to promote early metastatic competence. The desmoplastic stroma, rich in fibroblasts, inflammatory mediators and aligned extracellular matrix, provides both structural and biochemical support for tumor-cell escape. Additionally, neural and immune interactions, including chemokine and cytokine signaling, facilitate perineural invasion and systemic pre-metastatic niche (PMN) conditioning via extracellular vesicles and cytokine networks. Recognizing PDAC as a systemic disease from inception reframes therapeutic priorities: early systemic therapy and biomarker-guided patient stratification may intercept occult dissemination. We propose that integrating mechanistic insights on tumor–microenvironment crosstalk with perioperative and liquid-biopsy-driven clinical strategies could redefine early intervention in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92dd387af76159f510d779a0044bf468159245fa" target='_blank'>
              The molecular logic of early metastasis in pancreatic cancer: crosstalk between tumor and microenvironment
              </a>
            </td>
          <td>
            Andrea Costamagna, Noemi Ghiglione, M. Martini
          </td>
          <td>2025-11-28</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Pediatric cancers are often driven by unique alterations suggesting mechanistic ties to underlying developmental programs. Ependymoma (EPN) is an example of a brain cancer subtype driven by gene fusions involving ZFTA-RELA (ZR) that are exclusive to this disease. This leads to our hypothesis that specific chromatin states in developmental lineage programs are at risk of oncogenic transformation, and that these chromatin signatures persist during tumor development and govern tumor cell heterogeneity. To address this question, we constructed an integrated single-nucleus Multiome (snRNA-seq and snATAC-seq) atlas of the developing mouse forebrain and compared it with snMultiome analysis of ZR-driven mouse and human EPN. We identified specific developmental lineage programs present in glial progenitor cells enriched with the Plagl family transcription factors; the precise DNA binding sites of ZFTA fusion oncoproteins. In both mouse and human EPN, ZR expression maintains accessibility of developmental epigenomic programs leading to persistent cell proliferation and tumorigenicity. Cross-species analysis of mouse and human EPN revealed significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that established a putative tumor cell hierarchy. These findings were expanded by applying in vivo lineage barcode tracing methods in mouse EPN, which revealed dominant cell lineages that aggressively expanded and recapitulated the entire tumor cell diversity. Our findings unravel the intersection between developmental and oncogenic epigenomic states critical for ZR driven brain cancer. These findings shape our understanding of the earliest stages of epigenetic programs in brain cancer, and how cancer drivers such as ZR intersect with developmental programs to establish tumor cell heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76bc0f8a27b22de067c5d347cc54b71b6de69b8d" target='_blank'>
              EPCO-12. DOMINANT MALIGNANT LINEAGE LEVERAGE RESTRICTED EPIGENOMIC PROGRAMS TO DRIVE EPENDYMOMA DEVELOPMENT
              </a>
            </td>
          <td>
            Hua Sun, A. Kardian, S. Ippagunta, N. Laboe, Hsiao-Chi Chen, Jiangshan Zhan, Erik Emanus, S. Varadharajan, T. Zheng, B. Holcomb, Jon P. Connelly, Yong-Dong Wang, K. Lowe, Hyun Kyoung Lee, S. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background Gastric cancer progression involves complex interactions among tumor cells, immune components, and stromal elements within the tumor microenvironment. However, a comprehensive understanding of cellular heterogeneity, spatial organization, and cell-cell communication in gastric cancer remains incomplete. Methods Single-cell RNA sequencing was performed on 252, 399 cells from six tissue types, spanning gastritis, intestinal metaplasia, primary tumors, adjacent normal tissue, and metastatic lesions. Integration with spatial transcriptomics enabled spatial mapping of cellular interactions. Pseudotime, cell-cell communication, and transcriptional heterogeneity analyses were conducted. Tumor stage-associated gene modules were identified using Weighted Gene Co-expression Network Analysis (WGCNA) of The Cancer Genome Atlas (TCGA) data. Finally, a deep learning-based prognostic model was developed and externally validated. Results Our analysis revealed dynamic remodeling of the tumor microenvironment during gastric cancer progression, characterized by the expansion of dysfunctional CD8+ T cells, pro-tumorigenic fibroblasts (e.g., ITGBL1+, PI16+, and ITLN1+), and altered myeloid populations. Stromal-immune crosstalk, particularly fibroblast-driven immunosuppressive signaling, was prominent. Spatial transcriptomics revealed the colocalization of immune and stromal cells, supporting spatially organized cellular interactions. WGCNA identified a gene module (657 genes) associated with T cell, myeloid, and stromal alterations, as well as tumor stage. A deep learning model based on this gene set accurately stratified patients according to survival in both TCGA and independent validation cohorts. Risk scores were correlated with clinical features, including tumor stage and therapeutic response. Conclusions Our integrative single-cell, spatial, and computational analysis provides a high-resolution map of gastric cancer microenvironment remodeling. We identified key stromal and immune subpopulations, extensive cellular communication networks, and spatial structures that collectively drive tumor progression and metastasis. The derived gene signature and prognostic model have the potential for clinical risk stratification and therapeutic targeting in gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f147ce2dccf2934265aaef857f3e2b638aa838fb" target='_blank'>
              Integrative multi-omics analysis of gastric cancer evolution from precancerous lesions to metastasis identifies a deep learning-based prognostic model
              </a>
            </td>
          <td>
            Yulin Ren, Xiaoyan Zhang, Ke Li, Shuning Xu, L. Qiao, Qun Li, Cheng Zhang, Ying Liu
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11151ef60d8479a0c0f55b160c123009b32bf93" target='_blank'>
              Single-cell transcriptomic analysis reveals the heterogeneity and functional characteristics of macrophage subpopulations in colon cancer.
              </a>
            </td>
          <td>
            Jiazheng Zhao, Baochun Wang, Xiao Li, Chunting Wei, Yi Min, Dayong Wang
          </td>
          <td>2025-11-23</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Y. Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, R. Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Colorectal carcinoma exhibits high heterogeneity, comprising subtypes that show poor efficacy of T cell-based immunotherapies, such as programmed cell death protein 1 (PD-1) checkpoint inhibitors. Although natural killer (NK) cells are considered a promising approach for cancer immunotherapy, it remains unclear what molecular mechanisms drive NK cell activation or suppression within the tumor microenvironment. Moreover, limitations in human tumor models that reflect the diversity of individual patient tumors hinder the ability to effectively select patients who would benefit most from NK cell-based therapies. Methods Here, we established a co-culture platform of genetically diverse colorectal cancer (CRC) patient-derived organoids (PDOs) with primary allogeneic NK cells. We performed bulk RNA sequencing analysis of sorted NK cells after exposure to PDOs and aligned gene expression signatures derived from our findings with publicly available single-cell RNA sequencing data of NK cells from peripheral blood and CRC tissues of patients. Moreover, we evaluated identified pathways using flow cytometry and IncuCyte live-cell imaging analysis to quantify phenotypic alterations and NK cell-mediated killing of PDOs over time, respectively. Ultimately, we tested CRISPR-Cas9-edited NK cells and PDOs, small molecule compounds, and clinically relevant monoclonal antibodies (mAbs) to increase NK cell potency. Results On co-culture, NK cells acquired common transcriptional signatures related to hypoxia and transforming growth factor-beta (TGF-β), similar to NK cells infiltrating CRC tissues of patients. In addition, we observed patient-specific differential PDO susceptibilities to NK cell-mediated lysis. Major histocompatibility complex class I deficiency and natural killer group 2, member D (NKG2D)-ligand expression on PDOs facilitated NK cell-mediated cytotoxicity, and induced phenotypic NK cell diversification related to activation and the acquisition of inflammation and tissue-residency-related transcriptional signatures. Genetic or pharmaceutical targeting of hypoxia-inducible factors HIF1A/EPAS1 or TGF-βR1, or the addition of anti-CEACAM1 mAbs, enhanced NK cell-mediated PDO killing or activation, respectively. Conclusions The NK cell/PDO co-culture platform allows the identification of both common and patient-specific impacts of the tumor microenvironment on NK cell function and can aid the development of patient-tailored immunotherapies. The majority of CRC (CMS2/CMS3) PDOs from our cohort were susceptible to NK cell-mediated killing and induced NK cell activation, highlighting the potential of NK cells for CRC immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfdbb13391ad7497b66effc8a4643668f2be84a" target='_blank'>
              Colorectal cancer organoids drive hypoxia, TGF-β, and patient-specific diversification of NK cell activation programs
              </a>
            </td>
          <td>
            Andreas von Kries, Irene Garcés-Lázaro, Bianca M Balzasch, Carsten Sticht, Indra A Shaltiel, Kim E. Boonekamp, Annika Sams, Alessia Triassi, Tomáš Hofman, Elke Burgermeister, J. Betge, Matthias P. Ebert, M. Boutros, Laura Helming, A. Stojanovic, A. Cerwenka
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Cancer metastasis, the process by which tumor cells spread from the primary site to other organs, is one of the leading causes of death in cancer patients. However, due to its complexity and unpredictability, the study of cancer metastasis remains a significant challenge in medicine. In recent years, with the advancement of high-throughput sequencing technologies and single-cell transcriptomics, we could gain a deeper understanding of the molecular mechanisms and cellular heterogeneity underlying cancer metastasis. There is an accumulating volume of publicly available cancer metastasis research data. Nevertheless, these resources lack proper organization, hindering systematic analysis. In this study, we developed an integrated resource named Panmim (http://www.gdwk-bioinfo.com/pan_metastasis/home) to investigate the immune microenvironment of metastatic tumors. The database currently encompasses 90 single-cell RNA-seq datasets from metastatic cancers of diverse origins, encapsulating 14 distinct metastatic sites and 36 primary cancer types. Panmim facilitates a cellular-level comparison of similarities and differences between primary and metastatic cancers, encompassing pathway analysis, alterations in cellular metabolic pathways, cellular distribution preferences, and various aspects of intercellular communication. Panmim presents the analysis results through an intuitive interactive graphical interface, enabling users to explore and understand the complex biological phenomena during cancer metastasis more conveniently. This resource will become a valuable tool for biologists and bioinformaticians to study the mechanisms of cancer metastasis, providing important data support and scientific basis for the optimization of cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9f11b0d110baaaf542f6916b80b13cbf907d57" target='_blank'>
              Panmim: a resource of pan-cancer metastasis immune microenvironment
              </a>
            </td>
          <td>
            Xuelu Zhang, Simeng Hu, Huanjing Hu, Yongkun Li, Pengyin Nie, Xichen Hu, Ruidong Xue, Xiao Xiang, Lu Zheng
          </td>
          <td>2025-10-28</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Tertiary Lymphoid Structures (TLS) are organized aggregates of immune cells that form in non-lymphoid tissues in response to chronic inflammation, including cancer. In melanoma, TLS presence within the tumor microenvironment is associated with a pre-existing anti-tumor immunity and linked to improved immunotherapy responses. However, their role in predicting recurrence, particularly in early-stage melanoma, remains unclear.Methods: We conducted a retrospective analysis of archival primary tumor samples from 65 patients with stage I-II melanoma and negative sentinel lymph node (Table 1). Tumor immune infiltrates and TLS were initially assessed using four-marker immunohistochemistry (IHC) panels targeting CD20, CD4, CD8, and Podoplanin (PDPN), a marker of stromal cells. TLS were objectively identified using the HDBSCAN AI-based clustering algorithm. Tumor samples were further analyzed for high-dimensional phenotyping and spatial analysis using a 17 markers panel with hyperplexed immunofluorescence imaging (HIFI). Associations between TLS features and clinical outcomes were evaluated.Results: IHC revealed that 88% of patients had peritumoral T cell infiltration, and TLS were detected in 54% of cases. TLS-positive patients had improved disease-free survival (DFS, p=0.2) and overall survival (OS, p=0.2) (Figure 1A, B). To investigate further, HIFI was performed on 8 patients (4 recurrent, 4 non-recurrent), revealing 13 distinct cell types: stromal (n=6), tumor (n=2), and immune (n=5) (Figure 1C). Spatial analysis identified 9 recurrent spatial modules (SM), each representing a distinct pattern of interaction between cell types. One SM, enriched for PD-L1⁺ stromal cells, was significantly associated with recurrence (p=0.019) (Figure 1D).Conclusions: TLS presence associates with improved DFS and OS in early-stage melanoma. However, beyond their mere presence, TLS spatial organization and local stromal-immune context could influence clinical outcomes. Our data demonstrated that a specific spatial module—defined by the proximity of PD-L1⁺ stromal cells—is associated with disease recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/978d2823a740581a2a5f2a0fdbf9de6949d49e50" target='_blank'>
              13 | Spatial contexture of tertiary lymphoid structures as a predictor of recurrence in early-stage melanoma
              </a>
            </td>
          <td>
            Italian Melanoma Intergroup
          </td>
          <td>2025-12-05</td>
          <td>Dermatology Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer (BC) is the most common malignancy among women, with its progression and prognosis significantly influenced by the tumor microenvironment (TME). Age-related differences in TME composition lead to distinct tumor behaviors: young patients (≤ 40 years) exhibit aggressive tumors, while elderly patients (> 70 years) experience immunosenescence and reduced therapy responses. We performed single-cell RNA sequencing (scRNA-seq) analysis on tumors from 10 breast cancer patients (5 ≤ 40 years, 5 ≥ 70 years), encompassing 33,664 high-quality cells. After cell annotation and batch correction, malignant epithelial cells were identified using inferCNV. We applied pseudotime trajectory analysis, pathway enrichment, and cell–cell communication profiling to investigate age-specific TME dynamics. Survival relevance was assessed using a GEO cohort (GSE20685) of young breast cancer patients, and immunohistochemical staining was performed on clinical tumor and fibroadenoma tissues to validate protein-level expression of key ISGs. In young patients, malignant epithelial cells showed gradual upregulation of interferon-stimulated genes (ISGs) such as IFI44, IFI44L, IFIT1, and IFIT3 along the pseudotime trajectory, suggesting their involvement in early tumorigenesis. High expression of these ISGs was significantly associated with poor overall survival in a young BC cohort (GSE20685). Immunohistochemical validation further confirmed elevated IFIT3 protein levels in young tumor tissues. In contrast, elderly patients had a TME enriched in macrophages and fibroblasts, with activation of immunosuppressive pathways (e.g., SPP1, COMPLEMENT). Our integrative analysis identifies ISGs as key transcriptional drivers of tumorigenesis in young breast cancer, with potential prognostic and therapeutic value. Despite limited sample size, the combination of single-cell transcriptomics, clinical survival data, and protein-level validation provides robust evidence of age-specific TME remodeling. These findings support the development of age-tailored immunotherapy strategies targeting interferon signaling in young patients and immune checkpoint pathways (e.g., LAG3, CTLA4) in elderly individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81afa54ab4f065b06519a2062a7ca2a21beb87a4" target='_blank'>
              Single-cell RNA sequencing reveals age-related heterogeneity in the tumor microenvironment of breast cancer patients
              </a>
            </td>
          <td>
            Zihao Li, Ji Feng, Mengyang An, Shuming Kou, Zengren Zhao, Jingjing Yu, Bo Liu, Fangjian Shang
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Lymph node (LN) metastasis is a critical prognostic factor for lung cancer, but the influence of the LN immune microenvironment (IM) on metastasis remains unclear. We hypothesize that changes in the LN IM predict LN metastasis and indicate tumor progression in premetastatic early-stage non-small cell lung cancer (ES-NSCLC).



 Multi-modal single-cell sequencing was used to profile the transcriptome and proteome of 88,203 cells from LN and tumor biopsies of 10 ES-NSCLC patients. Imaging Mass Cytometry (IMC) spatially profiled 1,024,996 cells from FFPE slides of the same biopsies using 39 antibodies.



 Subtypes of exhausted CD8+ T cells were identified in N1-stationed LNs of ES-NSCLC patients and were enriched in corresponding patient tumors. These LNs were also enriched with mature regulatory dendritic cells (mregDCs), plasmacytoid dendritic cells (pDCs), plasma cells, B cell subsets, and regulatory T cells. IMC revealed spatial interactions among these cells, forming immune niches in the LNs of patients with more advanced disease.



 The observed immune dysregulation may suggest the formation of a “pre-metastatic immune niche” in specific LNs, increasing their susceptibility to tumor metastasis. This research has the potential to identify LN biomarkers, leading to earlier and more effective treatments for ES-NSCLC patients. Future work will address the limitation of a small patient cohort by enrolling additional patients for validation.



 Supported by NCI HTAN 1U2CCA233238-01; Elsa Pardee Foundation; J&J-BU Pilot Grant; CTSI Pilot Grant



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f54e5b78a99ae681ba305828e6be0afd66bcdc" target='_blank'>
              Multimodal and spatial single-cell profiling reveals immune-suppressed niches in tumor-adjacent lymph nodes of early-stage NSCLC patients 4608
              </a>
            </td>
          <td>
            Zhanhao Xi, Yusuke Koga
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) exhibits substantial genetic and functional heterogeneities. We generated clonal spheroids (CS) from HNSCC cell lines and patient tumor cells, uncovering distinct functional differences. Hyperproliferative spheroids (HRPS) showed increased proliferation and tumorigenic potential, whereas hypoproliferative spheroids (HOPS) demonstrated enhanced migration and resistance to cisplatin and radiation. In patient tumor-derived spheroids, spheroid size distribution analysis across various TNM stages indicated that early-stage tumors (T2N0M0) predominantly formed HRPS, while later-stage tumors (T4N2M0) exhibited a higher prevalence of HOPS. Single-cell RNA sequencing (scRNA-seq) identified six clusters in the HOPS and four in the HRPS, each defined by unique gene expression profiles and pathway enrichment. HOPS were enriched with fibroblast, mesenchymal, and keratinizing epithelial cells, indicating a metastatic phenotype. They also expressed genes associated with stemness (KRT15 and ALDH3A1). Conversely, HRPS was characterized by proliferative genes, including TOP2A, AURKA, CKS2, AREG, and ANLN, suggesting an advanced HNSCC proliferative signature. Despite being generated from only two cells/well, the various clusters in both spheroid types mirrored the diverse tumor microenvironment in HNSCC, including neoplastic, epithelial, basal, fibroblast, and mesenchymal cells. Notably, the gene expression profiles of Cluster 3 in the HOPS and Cluster 1 in the HRPS closely matched the expression patterns observed in the HNSCC-TCGA dataset. These clusters also displayed a high transcriptional correlation between patient tumors and their xenografts, reinforcing the clonal nature of HNSCC’s genetic and functional diversity of HNSCC. This study provides a deeper understanding of the genetic and functional diversity of HNSCC tumors, which can aid in the development of more precise diagnostic tools by identifying distinct tumor subtypes. Distinguishing between hyper- and hypoproliferative spheroids could improve prognostic accuracy, helping to predict tumor behavior and patient outcomes. Moreover, the identification of specific gene expression profiles associated with proliferation, metastasis, and treatment resistance may guide personalized therapeutic strategies, enabling targeted treatment based on the unique tumor characteristics of individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/942cca58b9f76ca4806f68d559d87541dfee33c1" target='_blank'>
              Uncovering functional divergence and cellular clusters with specific gene signatures in HNSCC clonal spheroids
              </a>
            </td>
          <td>
            Jyoti Pandey, Md. Zubbair Malik, Ritis K Shyanti, Palak Parashar, Praveen K. Kujur, Deepali Mishra, Dhanir Tailor, Jee Min Lee, T. Kataria, Deepak Gupta, Hitesh Verma, Anshu Yadav, S. Kateriya, Sanjay V. Malhotra, Vibha Tandon, Rupesh Chaturvedi, Rana P. Singh
          </td>
          <td>2025-12-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains one of the most fatal brain tumors, in part because its profound spatial heterogeneity undermines the effectiveness of targeted therapies. Despite extensive pursuit, EGFR and VEGF inhibitor monotherapies have shown limited clinical benefit, yielding only brief remissions. We hypothesized that this therapeutic failure stems from region-specific variations in drug target expression within the tumor microenvironment, limiting the reach of monotherapies. To investigate this, we performed spatial transcriptomic profiling on 17 regions from 8 human GBM specimens, mapping gene expression to distinct anatomical compartments defined by the Ivy Glioblastoma Atlas Project: Core Tumor (CT), Core Tumor with Pseudopalisading Necrosis (CTpan), Core Tumor with Microvascular Proliferation (CTmvp), and Leading Edge (LE). Our analysis revealed striking spatial compartmentalization of therapeutic targets: EGFR expression was largely confined to the proliferative CT; VEGF ligands (VEGFA, VEGFB) were enriched within the hypoxic, necrotic CTpan; and VEGF receptors (FLT1, FLT4, KDR) were mostly localized to CTmvp, consistent with their roles in angiogenesis. Notably, none of these targets were expressed at the infiltrative Leading Edge, a compartment strongly implicated in tumor recurrence. These findings provide mechanistic insight into the limited efficacy of EGFR and VEGF inhibitor monotherapies in GBM: the spatial segregation of targets prevents uniform drug action across all tumor regions. By leveraging spatial transcriptomics, our study highlights the inadequacy of one-size-fits-all regimens and underscores the need for spatially informed, multi-agent strategies tailored to the tumor’s regional biology. These results support a shift toward treatment approaches that better align with the evolving landscape of tumor heterogeneity in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1153b38bc03abdb3629caf32aaea054bbbcdd6" target='_blank'>
              EPCO-49. SPATIAL TRANSCRIPTOMICS ILLUMINATES MECHANISMS OF MONOTHERAPY FAILURE IN GLIOBLASTOMA BY MAPPING TARGET EXPRESSION ACROSS TUMOR REGIONS
              </a>
            </td>
          <td>
            Emre Kiziltug, V. Ravikumar, Reva Kulkarni, A. Maddox, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0127956ee657e81733c7622d2b48c0407a2cab1" target='_blank'>
              Cell heterogeneity contributes to the variable response of HIV-1 to latency reversing agents
              </a>
            </td>
          <td>
            Rachel Topno, Hussein Karaki, Flavia Mazzarda, Oriane Pourcelot, Manon Philippe, K. Zibara, Ovidiu Radulescu, Edouard Bertrand
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe18ec72d329e63e66769b08f8cae7347edc672" target='_blank'>
              Prognosis prediction using autophagy gene expression in osteosarcoma
              </a>
            </td>
          <td>
            Qing Liu, Cong Zhou, Ke Xu, Qikui Zhu, Junqin Lu, Chun Zhang, Wukun Xie, Guojiu Fang, Xue Zhou Chen, Dasheng Tian, Juehua Jing, Yize Li, Hongsheng Wang, Wenxiu Duan, Yihui Bi
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Introduction CAR-T cell therapy efficacy in B-cell lymphoma remains limited by treatment resistance. To address this fundamental challenge and elucidate underlying mechanisms, we undertook a comprehensive single-cell analysis of the CAR-T-tumor microenvironment interface. By leveraging five public scRNA-seq datasets spanning 1.48 million cells from CD19-targeted CAR-T patients, this study establishes an advanced computational framework to decipher the cellular ecosystem governing therapeutic resistance.
 Methods Methodologically, we developed three integrated innovations: (1) A biologically informed hierarchical graph convolutional network (GCN) modeling patient-specific tumor microenvironments as cell-cell correlation graphs for response prediction; (2) A novel Grad-CAM extension enabling quantitative interpretation of GCN outputs at single-cell resolution to establish gene-outcome relationships and resistance biomarkers; (3) GeneTransformer—a hybrid transformer architecture predicting transcriptional responses to genetic perturbations while resolving cross-platform heterogeneity via stochastic triple-masking during finetuning.
 Results Analysis revealed system-wide transcriptional dysregulation and extensive cell-subset-specific alterations distinguishing non-responders. Functional integration identified four core resistance modules: pro-survival signaling (BCL2/XIAP), metabolic rewiring (AMPK exhaustion), immune dysfunction (PD-1/TIGIT), and homeostatic collapse (Hippo/YAP-mediated stemness). Crucially, we exposed mechanistic crosstalk revealing actionable combinatorial targets—notably PI3K inhibition synergizing with anti-PD-1 blockade. Cross-population validation confirmed model robustness, with 84.65% (p<0.05) of reverse-engineered resistance genes exhibiting differential expression. The GCN achieved superior predictive performance (AUC=0.900).
 Conclusion This study establishes a transformative computational framework that deciphers the single-cell architecture of CAR-T resistance, providing a mechanism-guided blueprint for designing combinatorial therapeutic strategies. The GeneTransformer platform further enables optimization of individualized therapies through perturbation response prediction. Our findings illuminate fundamental resistance pathways while delivering clinically actionable targets for overcoming treatment failure in B-cell malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a8a729b90dea4325ceebae0fe0354baffbb43f1" target='_blank'>
              Deep learning-based tumor microenvironment profiling predicts CAR-t efficacy and guides combinatorial immunotherapy in B-cell lymphoma
              </a>
            </td>
          <td>
            Shaojie Wu, Jiatong Wang, Jiaxiu Li
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Current therapeutic strategies for endometrial cancer are mainly based on aggressive histological types and molecular subtypes. However, ignoring the spatial distribution of immune/stromal cells fails to account for the heterogeneity of the local tumour microenvironment, leading to biased prediction of treatment response. The goal of precision medicine is to delineate the biological characteristics of local functional units based on molecular labelling, which adequately reflects spatially adaptive changes during treatment or metastasis. Single-cell resolution analysis of 40 endometrial cancer cases across four molecular subtypes was performed using imaging mass cytometry (IMC) to quantify the frequency, spatial distribution, and intercellular crosstalk of distinct immune and stromal cell populations. These ecosystem-level features were systematically correlated with clinical features and outcomes, including treatment response and survival. We further identified CD90 + clusters as key regulators of macrophage polarization and T-cell infiltration dynamics, with flow cytometry used to validate their functional role in tumour subtype specification and microenvironmental remodelling. Finally, machine learning-based spatial phenotyping was employed to construct molecular subtype-specific signatures and a highly accurate recurrence prediction model for high-risk endometrial cancer. Single-cell profiling revealed that CD90 + clusters constitute a critical immunomodulatory component within the tumour microenvironment, demonstrating significant enrichment in macrophage differentiation pathways and serving as key mediators of intercellular signalling networks. Furthermore, computational models integrating functional molecular signatures with cell–cell interaction profiles demonstrated high predictive accuracy for both molecular subtyping and recurrence risk stratification in patients with endometrial carcinoma. Our study establishes a spatial eco-context framework for molecular subtypes of endometrial cancer by integrating single-cell spatial multiomics data. This approach enables high-resolution mapping of tumour-immune-stromal interaction networks and reveals novel targets for personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63a8a4865fa920849f9dc4e8439a41c6d062126a" target='_blank'>
              Spatial ecostructural modelling of endometrial cancer identifies the key role of CD90 + CD105 + endothelial cells in tumour heterogeneity and predicts disease recurrence
              </a>
            </td>
          <td>
            Di Wu, Cinian Lv, Zhifeng Yan, Luyang Zhao, Lian Li, M. Ye, Mingyang Wang, Qingzhi Zhai, Nan Wang, Zheng Wang, Yuanguang Meng, Mingxia Li
          </td>
          <td>2025-11-17</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 The spatial dynamics of B cells are crucial during immune response, shaping clonal evolution and affinity maturation. B cell evolution can by studied by reconstructing phylogenetic lineage trees, which represent B cell receptor (BCR) mutations among sampled cells. SpatialVDJ sequencing is a recently developed technique that pairs spatial RNA-sequencing with long-read, spatially resolved BCR data on a 2D slice of tissue. Currently, no methods exist to analyze the movement of B cell lineages across 2D coordinates. We introduce GIBLE (Geographic Inference of B Cell Lineage Evolution), a novel computational framework that reconstructs spatial locations of unobserved, ancestral B cells and quantifies movement across the tissue slice. We used GIBLE to understand B cell maturation in ectopic germinal centers in thymic tissue from patients with myasthenia gravis, an autoimmune disorder in which autoreactive B cells produce antibodies against the neuromuscular junction. GIBLE provides a robust and widely applicable framework for integrating spatial and phylogenetic data to trace B cell evolution during immune responses using SpatialVDJ data.



 Supported by NIH/NCI 5T32CA134286.



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171685d301f0d9a0ae908798444d8439b632feea" target='_blank'>
              Resolving B cell migration from SpatialVDJ sequencing with GIBLE 3612
              </a>
            </td>
          <td>
            Hunter J. Melton, Chinmay M. Vaidya, Jessie J. Fielding, Sherry Wu, Fred W. Kolling, Kevin C. O’Connor, Kenneth B. Hoehn
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8a1f1ccbeb7cda31923d2bd498d2b7cb3a690e5" target='_blank'>
              Metabolic zonation and characterization of tissue slices with spatial transcriptomics
              </a>
            </td>
          <td>
            Oier Etxezarreta Arrastoa, A. Pirona, Allon Wagner, N. Yosef
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) represents a significant global health challenge due to its molecular heterogeneity and the immunosuppressive nature of its tumor microenvironment (TME). Tumor-associated macrophages (TAMs), particularly the M2-polarized subset, are crucial in HCC progression, contributing to immune evasion, angiogenesis, and metastasis. However, the mechanisms driving TAM reprogramming and their metabolic interactions within the TME remain poorly understood. In this study, we applied single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to identify a novel metabolic-immune axis in HCC. Our findings revealed increased oxidative phosphorylation (OXPHOS) in tumor cells, which correlated with enhanced ribosomal activity in M2 macrophages, suggesting a metabolic coupling between the two. Tumor cells, under elevated OXPHOS, secrete factors that recruit macrophages and promote M2 polarization, thereby reinforcing immune suppression. Further analysis of The Cancer Genome Atlas (TCGA) dataset led to the development of a prognostic model based on OXPHOS-related genes, showing strong predictive power across multiple cohorts. These results suggest a novel OXPHOS-ribosome-M2 polarization axis and offer potential therapeutic targets for immunometabolic therapies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d98d705964e6108c751cf9dce67e7948e856f2d" target='_blank'>
              M2 macrophages and tumor cells engage in a metabolic feedback loop to drive HCC progression
              </a>
            </td>
          <td>
            Yimin Nong, Xiaolong Chen, Baoxin Zhang, Zhenying Lin, Wuriniletu, Chengyou Du, Qiang Liu
          </td>
          <td>2025-11-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f4df7fa2e4800fd55af9dfd1c2bbdecb131c8f" target='_blank'>
              A guide to transcriptomic deconvolution in cancer.
              </a>
            </td>
          <td>
            Yaoyi Dai, Shuai Guo, Yidan Pan, C. Castignani, Matthew D. Montierth, P. van Loo, Wenyi Wang
          </td>
          <td>2025-12-02</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Glioblastoma exhibits regional heterogeneity, with distinct cellular phenotypes and molecular features between contrast-enhancing (CE) solid tumor regions and non-enhancing (NE) infiltrative regions. Understanding functional repercussions of this heterogeneity is important.



 Image guided biopsies of CE and NE regions of newly diagnosed IDH wild-type glioblastoma were obtained. Tissues were analyzed using single cell sequencing to identify glioma stem cell populations using gene set enrichment analysis and established gene signatures. Cells were treated with temozolomide (TMZ, 1000 μM), paclitaxel (PTX, 50 μM), topotecan (TPT, 50 μM), vincristine (VCR, 50 μM), and doxorubicin (DOX, 20 μM) for 48 hours. Cell viability was measured using a luminescence-based assay to assess regional differences in chemotherapeutic response.



 Matched samples from 5 patients were sequenced and identified NE regions as harboring elevated levels of glioma stem-like cells using multiple established signatures. Multiple cell markers including several HOX genes, PROM1, F3, ID4, PAUPAR were significantly upregulated in NE tumor cells (FC ≥ 4.0; FDR < 0.05). Samples from NE regions grew slower (p<0.001) than CE regions at the 1-week mark, but were able to recapitulate confluent tumor in cortical organoid and xenograft models by 4 weeks. In matched samples from 3 patients, there was significantly reduced chemotherapeutic efficacy in NE regions (CE viability vs NE viability, p = 0.17) in the various chemotherapeutic agents tested.



 NE infiltrating regions of glioblastoma maintain high proportions of cells that have characteristics of glioma stem cells transcriptionally. These regions have slow initial growth in in vitro and in vivo models, but are able to recapitulate solid tumor properties. Likely related to this quiescent property, NE regions are more resistant to chemotherapeutic agents.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49847b7486480b2128b07d57d35417c0ebd3d28" target='_blank'>
              DDDR-05. Non-enhancing regions of Glioblastoma Harbor Quiescent Stem Like Cells that Contribute to Chemotherapy Resistance
              </a>
            </td>
          <td>
            Mahmoud I Youssef, Travis Perryman, Stuart Harper, Amani Carson, Lindsey A Dudley, Eliana S Oduro, Alondra Delgadillo, Shail Patel, Nish Sud, Harley Kornblum, Kunal S. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed06b707f53d2e1de85e352e5c2852fcf93afdb7" target='_blank'>
              Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer
              </a>
            </td>
          <td>
            Maria Vidal-Sabanés, R. Navaridas, Núria Bonifaci, Ada Gay-Rua, Damià Ortega-Peinado, Joaquim Egea, Mario Encinas, Xavier Matias-Guiu, David Llobet-Navás, Xavier Dolcet
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary Cancer stem cells (CSCs) have been suggested to be the subset of cancer cells that drive tumor growth and progression and fuel the tumor with heterogeneous populations of cells, including those resistant to therapy. Different proteins have been proposed as biomarkers of CSCs in oral squamous cell carcinoma (OSCC). In this study, we simultaneously investigated multiple putative biomarkers of CSCs that have been previously proposed in OSCC, with the aim of identifying a pure CSC subpopulation. Our results suggest that several stem cell subpopulations may co-exist in a tumor, each having an impact on different clinical parameters. The cell subpopulations identified by use of different CSC markers were found to be dynamic and able to switch their protein markers over time. The inter-tumoral diversity of cancer stem cells demonstrated here raises novel challenges in terms of their use as predictive biomarkers and as efficient therapeutic targets for OSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dc7fec448accd93f20cdf3e8cf88a44c781b603" target='_blank'>
              Identification of Phenotypically Distinct Cancer Stem Cell Subpopulations in Oral Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            T. Osman, Oddveig G. Rikardsen, M. Teh, D. Sapkota, Kristina Xiao Liang, E. Neppelberg, Adrian Biddle, Ian C. Mackenzie, Lars Uhlin-Hansen, A. C. Johannessen, D. E. Costea
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Gliomas are lethal malignancies composed of heterogeneous and dynamic subpopulations of cellular states that resemble both normal neurodevelopmental cell types and adaptive responses to stress within the tumor microenvironment (TME). While cellular plasticity enables gliomas to survive environmental pressures, the mechanisms driving glioma state dynamics remain unclear. To explore how glioma state heterogeneity functionally and metabolically interacts with the brain TME, we conducted a multi-omic analysis across 392 glioma specimens, including patient tumors, orthotopic xenografts and gliomasphere cultures using bulk RNA/whole-exome sequencing and lipidomic profiling. Single-cell transcriptome sequencing of a diverse panel of glioma tumors (n = 21) integrated with bulk tumor transcriptome profiling revealed a spectrum of cellular identities associated with distinct lipidomic profiles. Comparison of matched patient and derived models showed that the non-native environments, such as gliomasphere culture, restricted glioma state diversity and enriched for states characterized by elevated de novo fatty acid synthesis. Across environmental contexts, glioma state plasticity was coupled to lipid metabolic plasticity, enabling tumors to concordantly remodel their state composition and lipid profiles in response to environmental constraints. Of note, certain tumors exhibited limited state and metabolic plasticity – specifically those enriched for states within oligodendroglial and neuronal lineages. These gliomas demonstrated reduced de novo lipid synthesis capacity, and a concordant increased reliance on exogenous lipid scavenging within the brain microenvironment for survival. Together, these results link glioma heterogeneity and plasticity to lipid metabolic reprogramming, and highlight how distinct cellular state profiles result in environment-dependent metabolic liabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149479392e37d2670e1ddd1d4b3e1788684e5fcb" target='_blank'>
              EPCO-63. LIPID METABOLIC HETEROGENEITY ACROSS GLIOMA CELLULAR STATES REVEALS ENVIRONMENTAL DEPENDENCIES
              </a>
            </td>
          <td>
            Nicholas A. Bayley, Jenna K. Minami, Cassidy Andrasz, C. Tse, Jennifer Salinas, Dimitri Cadet, A. Bhaduri, T. Cloughesy, Thomas G. Graeber, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 Leukemia evolution is driven by a complex interplay between somatic mutations and epigenetic changes. However, our understanding of these processes is limited by a lack of scalable and cost-effective methods to co-assay genetic and epigenetic changes in single cells. Here, we present scGEM-seq (single-cell Genotyping and Methylome sequencing), a flexible multi-omics platform that permits simultaneous characterization of genetic variants and DNA methylation profiles from single cells. scGEM-seq combines three core innovations that can be accomplished at ~3% the cost ($0.03 per cell) of competing technologies (>$1 per cell in snmC-seq3 and SciMETv2): (1) single-cell capture with semi-permeable capsules (Atrandi Biosciences), enabling multi-step enzymatic reactions to be performed efficiently with minimal loss of DNA; (2) a novel split-pool strategy (HexaDecimal barcoding) that efficiently generates ~1 million unique cell barcodes that are compatible with bisulfite-like enzymatic conversions; and (3) flexible library preparation that can be adapted for genome-wide or targeted sequencing.
 In cell-mixing studies, we show that scGEM-seq accurately resolves cell populations based on DNA methylation profiles and genetics. At modest sequencing depth (0.001X per cell for whole-genome DNA libraries, 15734.4±12123.1 CpGs per cell for methylation libraries), scGEM-seq resolved mouse and human cell lines at expected ratios (1:1:1 ratio of GM12878, K562 and YAC1 cells) with low doublet rates (~2.4% using species genome alignment). scGEM-seq also resolved human cancer cell lines of distinct lineages (1:1:1 ratio of SW480, A375 and OCIAML3 cells) according to methylation and genetics (cell-specific SNVs and copy-number changes) with modest sequencing depth (0.002X per cell for whole-genome DNA libraries, 20242.6±12808.2 CpGs per cell for methylation libraries) and low doublet rates (2.88% using cell-specific SNVs).
 To enhance genotyping efficiency, we also combined scGEM-seq with two different target enrichment strategies: hybrid capture (Pan-cancer hybridization panel, IDT; n=127 genes) and amplicon sequencing. In mixed human cancer cell lines, scGEM-seq achieved consistent single-cell genotyping for known cancer genes and hotspot mutations, including a mean of 13.23±5.30% of cells for hybrid capture and >98% of cells for amplicon sequencing. To determine the sensitivity of scGEM-seq for genotyping of rare cells, we prepared a cell mixture containing varying proportions of human cancer cell lines (GM12878: 92.85%, SW480: 5%, A375: 1.5%, OCIAML3: 0.5%, K562, 0.15%). scGEM-seq coupled with amplicon sequencing for known cell-specific variants recovered the expected cell proportions down to ~0.1-0.2%, representing fewer than 2 malignant cells per 1000 (SW480: 302/6294 cells, 4.80%; A375: 88/6294 cells, 1.40%; OCIAML3: 30/6294 cells, 0.48%; K562: 11/6294 cells, 0.17%).
 Finally, we applied scGEM-seq to primary bone marrow samples from 4 patients with myeloid neoplasms (MDS and AML) representing a range of blast counts (47%, 52%, 10% and 16%). Here, scGEM-seq revealed distinct populations of normal, premalignant, and malignant cells, including subclonal leukemia cells with KIT D816V (47/777 cells, 6.05%) and T417_D419delinsL (72/700 cells, 10.29%) mutations in a patient with inv(16) AML, and clonal (premalignant) hematopoietic cells harboring a DNMT3A S770L mutation (251/558 cells, 44.98%) and subclonal malignant blasts with distinct KRAS G12C (92/385 cells, 23.9%) and G12D (4/381 cells, 1.05%) mutations.
 In conclusion, scGEM-seq provides a flexible, cost-effective, and high-resolution method for elucidating the complex interplay between epigenetic and genetic cell states in cancer cells, including within rare cell populations. We envision multiple applications for scGEM-seq for the study of leukemia and other blood cancers, including to characterize (epi)clonal heterogeneity and dynamics, identify minimal residual disease, and capture rare stem/progenitor populations or drug-resistant clones that emerge post therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42da31bde36d790d58830d2a574d2fd507a69d8f" target='_blank'>
              Single-cell genotyping and methylome sequencing (scGEM-seq) resolves cell-states and clonal hierarchies in leukemia
              </a>
            </td>
          <td>
            Kyung Lock Kim, Maria R. Capilla-Guerra, Evan C Chen, Andrew Lane, Gabriel Griffin
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Metastatic brain tumors (BrM) are associated with significant morbidity and mortality. Despite significant advances in understanding the role of microbiome in cancer and tumor immunity, the impact of microbiome in BrM remains largely unknown. In this study, we used a prospectively collected cohort of BrM tumors to explore the spatial distribution of microbial elements in BrM and determine their correlation with the tumor microenvironment. We employed pan-bacterial 16S fluorescent in situ hybridization and Small Molecular Imaging (SMI, CosMx® Platform, NanoString) and demonstrated that bacterial signals can be detected in BrM tumors, are intracellular and distributed across tumor and immune cells. We investigated the spatial association of these bacterial signals with specific niches in the tumor microenvironment using SMI and Digital Spatial Profiling (DSP, GeoMx® Platform, NanoString). We demonstrated that tumor regions with high bacterial signals were associated with in anti-bacterial signatures indicative of TLR pathway activation, enrichment of neutrophils and neutrophil enzymes, as well as distinct metabolic signatures. Collectively, our studies introduce intracellular bacterial signal as a novel component of the brain tumor microenvironment and demonstrate its correlation with distinct immune signatures. These findings can guide future mechanistic and translational studies to elucidate the functional role of the intra-tumoral bacterial signal in brain tumor biology.



 This study was supported by the National Cancer Institute (F32CA260769), the Andrew M. McDougall Brain Metastasis Research Award, and the Dr. Marnie Rose Foundation.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c5515b9b890be2829ade266a16baf6b7c61224" target='_blank'>
              Intra-tumoral bacterial signals are associated with distinct immune niches in metastatic brain tumors 3692
              </a>
            </td>
          <td>
            Golnaz Morad, Ashish V Damania, Brenda Melendez, Bharat Singh, Matthew Wong, P. Sahasrabhojane, Yasmine M. Hoballah, Fabiana J. Veguilla, Sarah B. Johnson, J. Losh, Susan Bullman, N. Ajami, Sherise Ferguson, Jennifer A. Wargo
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc26cc7e8a1e8ed5ed66483e07fdc9dc2336ee3" target='_blank'>
              Small Cell Lung Cancer Classification: Unraveling Heterogeneity to Enable Personalized Treatments.
              </a>
            </td>
          <td>
            K. Boettiger, I. Kovács, Lilla Horvath, B. Ernhofer, K. Schelch, M. Pozonec, V. Teglas, Clemens Aigner, Evelyn Megyesfalvi, Ferenc Rényi-Vámos, K. Bogos, C. Lang, Abigail J. Deloria, C. Gay, L. Byers, J. Sage, F. Hirsch, Z. Megyesfalvi, B. Dome
          </td>
          <td>2025-12-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45dd551579bb493b99c7ec5a7c44f7e95f00ebc0" target='_blank'>
              Single cell whole genome and transcriptome sequencing links somatic mutations to cell identity and ancestry
              </a>
            </td>
          <td>
            Abhiram Natu, Mrunal K. Dehankar, Reenal Pattni, M. Šuvakov, Dmitrii Olisov, Livia Tomasini, Yeongjun Jang, Yiling Huang, Eva Benito-Garragori, Patrick Hasenfeld, Jan O. Korbel, Alexander E. Urban, A. Abyzov, F. Vaccarino
          </td>
          <td>2025-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Phenotypic plasticity plays a key role in cancer progression and metastasis, enabling cancer cells to adapt and evolve, but precisely how distinct axes governing phenotypic plasticity interact to shape tumour progression and patient outcomes remains unclear. We investigated five major interconnected axes of plasticity in ER-positive (ER+) breast cancer: Metabolic Reprogramming, Epithelial-to-Mesenchymal Plasticity (EMP), Luminal–Basal (Lineage) Switching, Stemness, and Drug-resistance using network dynamics simulations, integrative bulk and single-cell transcriptomic analyses and patient survival analyses. We show that these axes are not independent but drive one another, forming two mutually inhibiting ‘teams’ of nodes enabling specified cellular behaviour. One team (favouring high glycolysis, stem-like, basal-like, mesenchymal/hybrid and tamoxifen-resistant phenotype) was found to be associated with aggressive progression and worse survival. On the other hand, the opposing team (favouring high oxidative phosphorylation, non-stem-like, luminal-like, epithelial and tamoxifen-sensitive phenotype) correlated with better outcomes. Importantly, altering one axis of plasticity often drove coordinated responses along other axes and vice versa. Our findings establish phenotypic plasticity in cancer as a coordinated, multi-axis dynamical process, thus suggesting novel strategies to disrupt systems-level reprogramming enabling metastasis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acaf4b9f2872701e5c44aad69b0ee83629b9228" target='_blank'>
              Interconnected axes of phenotypic plasticity drive coordinated cellular behaviour and worse clinical outcomes in breast cancer
              </a>
            </td>
          <td>
            Ritesh Kumar Meena, Yijia Fan, Soundharya Ramu, Anjaney J. Pandey, Abhinav Kannan, Yashvita Subramanian, Yukta Subramanian, Jason T. George, M. Jolly
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Metastatic outgrowth requires that cancer cells delaminate from the primary tumor, intravasate, survive in circulation, extravasate, migrate to, and proliferate at a distal site. Recurrent genetic drivers of metastasis remain elusive, suggesting that unlike the early steps of oncogenesis, metastasis drivers may be variable. We develop a framework for identifying metastasis regulators using CRISPR/Cas9-based screening in a genetically defined organoid model of colorectal adenocarcinoma. We conduct in vitro screens for invasion and migration alongside orthotopic, in vivo screens for gain of metastasis in a syngeneic mouse model. We identify CTNNA1 and BCL2L13 as bona fide metastasis-specific suppressors which do not confer any selective advantage in primary tumors. CTNNA1 loss promotes cell invasion and migration, and BCL2L13 loss promotes anchorage-independent survival and non-cell-autonomous changes to macrophage polarization. This study demonstrates proof of principle that large-scale genetic screening can be performed in tumor-organoid models in vivo and identifies novel regulators of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33c9f8937cc2d0d69608a2acde03a6c6afb3fab8" target='_blank'>
              Forward genetic screening in engineered colorectal cancer organoids identifies regulators of metastasis.
              </a>
            </td>
          <td>
            Xin Wang, Zvi Cramer, N. A. Leu, Keara Monaghan, K. Durning, S. Adams-Tzivelekidis, Joshua Rhoades, Jonathan Heintz, Yuhua Tian, Joshua Rico, Diego Mendez, R. Petroni, Austin King, Melissa S Kim, Rina Matsuda, Olivia Hanselman, Alice E. Shin, María F Carrera Rodríguez, Igor E. Brodsky, Anil Rustgi, Ning Li, Christopher Lengner, M. Andres Blanco
          </td>
          <td>2025-11-11</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have revolutionized cancer treatment and show notable success in some cancer types such as non-small cell lung cancer, melanoma and colorectal cancers, while they demonstrate relatively low response rate in others, such as esophageal cancers. Due to the heterogeneous nature of the tumor microenvironment and patient-to-patient variability, there remains a need to improve ICI response rates. Combining ICIs with therapies that can overcome resistance is a promising strategy. Compared to de novo drug development, drug repurposing offers a faster and more cost-effective approach to identifying such combination candidates. A variety of computational drug repurposing tools leverage genomics and/or transcriptomic data. As single-cell RNA sequencing (scRNA-seq) technology becomes available, it enables precise targeting of cancer-driving cellular components. In this review, we highlight current computational drug repurposing tools utilizing scRNA-seq data and demonstrate the application of two such tools, scDrug and scDrugPrio, on an esophageal squamous cell carcinoma dataset to identify potential drug candidates for combination with ICI therapy to enhance treatment response. scDrug focuses on predicting tumor cell-specific cytotoxicity, while scDrugPrio prioritizes drugs by reversing gene signatures associated with ICI non-responsiveness across diverse tumor microenvironment cell types. Together, this review underscores the importance of a multi-faceted approach in computational drug repurposing and highlights its potential for identifying drugs that enhance ICI treatment. Future work can expand the application of these strategies to multi-omics and spatial transcriptomics datasets, as well as personalized patient samples, to further refine drug repurposing involving ICI therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b33c93cda41e1064ce2cfb630c9ed87d2c20b17" target='_blank'>
              Harnessing Single-Cell RNA-Seq for Computational Drug Repurposing in Cancer Immunotherapy
              </a>
            </td>
          <td>
            Olivia J. Cheng, T.T.T. Tran, Y. A. Chen, A. Tan
          </td>
          <td>2025-11-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Genomic alterations in glioblastoma (GB) drive distinct transcriptional subtypes, yet how they shape the tumour microenvironment (TME) architecture remains poorly understood. Here, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss, and PDGFB overexpression in Tp53−/− Nestin-tv-a mice, that recapitulated classical, mesenchymal, and proneural transcriptomic GB subtypes. We integrated single-cell RNA sequencing with regional (NanoString GeoMx) and single cell (10x Visium HD) spatial transcriptomics and proteomics data to acquire an in-depth characterization of the TME organization. At single cell level, NF1- and EGFRvIII-driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human GBs that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1043a078420592595b843047c99776c531341d4b" target='_blank'>
              TMIC-04. Genetic drives promote distinct microenvironment organization in glioblastoma
              </a>
            </td>
          <td>
            Junyi Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Treatment-induced neuroendocrine prostate cancer (NEPC) represents a lethal evolution of prostate adenocarcinoma under androgen receptor pathway inhibition, posing a significant clinical challenge. In a recent landmark study, Wang et al. introduced an innovative internal Z-score based approach to comprehensively characterize the transcription factor (TF) landscape in prostate cancer progression, uncovering distinct TF profiles associated with adenocarcinoma and NEPC lineages. Notably, the study proposes a three-phase model of NEPC transdifferentiation—comprising de-differentiation, dormancy, and re-differentiation—revealing dynamic shifts in TF expression that underpin lineage plasticity and therapeutic resistance. This commentary critically evaluates the methodological advancements, the functional significance of the identified TF signatures, and the broader implications of these findings for developing novel therapeutic strategies. By delineating the molecular events driving the transition from androgen receptor (AR)-dependent adenocarcinoma to treatment-resistant NEPC, this work underscores the potential of targeting early and dormant phases of transdifferentiation to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f432a498dcbb709202897ddd04a1b59a52e28fa2" target='_blank'>
              Transcription Factor Dynamics in Neuroendocrine Prostate Cancer Development
              </a>
            </td>
          <td>
            Yu Wang, Yuzhuo Wang
          </td>
          <td>2025-10-24</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Understanding how small molecules affect cellular morphology is essential for exploring their mechanisms of action (MoA) and identifying new therapeutic candidates. In this study, the Cell Painting Assay (CPA) was used to profile 196 small molecules in HCT116 colorectal cancer cells. By applying t-distributed stochastic neighbor embedding (t-SNE) followed by density-based clustering, 18 distinct phenotypic clusters were identified based on similarities in the quantitative morphological profiles generated from Cell Painting data. Although it was initially hypothesized that clustering would reflect known MoAs, most clusters showed only partial overlap with target-based classifications. Instead, compounds from different MoA classes converged on similar cellular phenotypes, suggesting common downstream effects or shared stress responses. Notably, compounds affecting DNA replication, mitosis, or transcriptional control appeared across multiple clusters, indicating functional diversity within morphologically similar groups. Clusters enriched with mTOR/PI3K inhibitors, spindle poisons, or transcriptional CDK blockers exhibited well-defined phenotypes, supporting the robustness of the assay. In contrast, compounds inducing more subtle phenotypes formed distinct micro-clusters, highlighting the method’s sensitivity. Overall, this study demonstrates that CPA can capture convergent phenotypic signatures that extend beyond target-based classification. These findings underscore the value of phenotype-driven screening for the functional annotation of chemical compounds and may help uncover unexpected relationships among molecules with diverse biological activities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b69630fa6518b96d44a14fa030997f1e5bc439" target='_blank'>
              Phenotypic profiling of small molecules using cell painting assay in HCT116 colorectal cancer cells
              </a>
            </td>
          <td>
            Velemir Lavrinenko, Ilona Donskaya, Vladimir Popov, Ekaterina Litau, Varvara Petrova, Stanislav Tyazhelnikov
          </td>
          <td>2025-10-29</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) exhibits marked plasticity and intense microenvironmental crosstalk. We aimed to delineate mesenchymal programs with spatial resolution, clinical relevance, and mechanistic anchors. Methods We integrated single-cell RNA-seq, bulk transcriptomes, and Visium spatial data. After rigorous QC and Harmony integration, we annotated 12 cell states using canonical markers, decoupler-based ORA, and AUCell. Tumor boundaries were defined by inferCNV/CopyKAT; developmental potential by CytoTRACE2 and PHATE. Post-translational modification (PTM) axes were scored from curated gene sets. A cell type-aware GNN linked bulk expression to a patient-similarity graph for survival modeling and gene-level hazard attribution. Network convergence combined bulk WGCNA (TCGA/CGGA), single-cell hdWGCNA, BayesPrism deconvolution, and external GEO validation. Ligand–receptor (LR) signaling was inferred with LIANA+, embedded in a signed causal network, and mapped spatially. ARRDC3 expression was assessed in GBM tissues; U251 gain- and loss-of-function assays evaluated proliferation and migration. Results We resolved major GBM states, including two mesenchymal programs (MES1-like, MES2-like). CNV-high regions marked malignant cores, and CytoTRACE2 identified high-potency niches within MES2-like and Proliferation states along non-linear trajectories. PTM landscapes segregated by state; S-nitrosylation, glycosylation, and lactylation were enriched in mesenchymal programs. A GNN risk score stratified overall survival in TCGA (n=157) and generalized to CGGA-325 (n=85) and CGGA-693 (n=140). MES2-like abundance remained an independent adverse predictor (HR = 2.31; 95% CI, 1.04–5.10). MES2-high tumors upregulated EMT, TNFα/NF-κB, JAK/STAT, hypoxia, angiogenesis, and glycolysis; S-nitrosylation associated with increased hazard. Cross-modal convergence defined a conservative MES2 core enriched for ECM remodeling, collagen modification, focal adhesion, and TGF-β regulation. LR analysis prioritized a TAM-to-MES2 axis (e.g., GRN–TNFRSF1A, ADAM9/10/17–ITGB1, TGFB1–ITGB1/EGFR) converging on a CEBPD-centered module. Spatial mapping localized MES2 hotspots within CNV-defined territories and revealed a TNFRSF1A–CEBPD–ARRDC3 focus at an infiltrative rim. ARRDC3 was upregulated in GBM tissues; in U251 cells, knockdown promoted and overexpression suppressed proliferation and migration, indicating context-dependent roles. Conclusions MES2-like GBM is an ECM-driven, stress-adapted state with strong prognostic impact. We nominate CEBPD and TNFRSF1A/ITGB1 as actionable nodes and identify ARRDC3 as a spatially restricted effector with context-dependent tumor-modulatory functions warranting therapeutic exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8942e39c98213dbf5399f61de91b7075a89e9eb2" target='_blank'>
              Single-cell/spatial integration reveals an MES2-like glioblastoma program orchestrated by immune communication and regulatory networks
              </a>
            </td>
          <td>
            Chonghui Zhang, Lu Tan, Kaijian Zheng, Yifan Xu, Junshan Wan, Jinpeng Wu, Chao Wang, Pin Guo, Yugong Feng
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e28d12e48520ac375b3cbad04e1c6d130662c96" target='_blank'>
              Deciphering Cancer Evolution Through Genomic Profiling of Patient-Derived Xenograft Together with Matched Primary Gallbladder Cancer.
              </a>
            </td>
          <td>
            Vipin Yadav, R. Kilambi, Archana Rastogi, Deepti Sharma, Perumal Nagarajan, Varun Suroliya, Anupam Bhattacharya, Vaishali Yadav, Pramod Gautam, Manju Kashayp, M. K. Kashyap, Vanya Garg, Anupama Kumari, N. Trehanpati, G. Ramakrishna
          </td>
          <td>2025-11-10</td>
          <td>Digestive diseases and sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is marked by an immunologically inert tumor microenvironment (TME), which may explain why immunotherapies have generally failed to improve survival. While TME features have been linked to clinical outcomes, prior studies were limited by small sample sizes and narrow cell type resolution, precluding a full understanding of TME multicellular dynamics. To address this, we assembled 1,979 clinically annotated GBM RNA-seq profiles and benchmarked six deconvolution algorithms using pseudobulk data from a large GBM single-cell RNA-seq dataset. Bayesian deconvolution (InstaPrism) emerged as a top performer and was used to characterize the TME across 39 cell states. We then stratified tumors by their dominant malignant cell state (AC-like, MES-like, NPC-like, or OPC-like). We identified lymphoid-dominant and myeloid-dominant immune compositions within each GBM subtype. We demonstrate that lymphoid-dominant, immune-activated tumors associate with significantly improved overall survival within the aggressive MES-like group, but not within other groups. Immune activation was defined by a conserved stress-inflammatory module consisting of general inflammatory activation, tissue remodeling, and stress. In tumors dominated by MES-like cell states, immune cells within the TME were driven by robust interferon alpha and gamma gene signatures. Sobel mediation analysis pinpointed Stimulator of interferon genes (STING) upregulation as a potential upstream orchestrator within this group. Interestingly, STING activation showed no beneficial effect in AC-like or NPC-like tumors and was detrimental in OPC-like GBM, highlighting a group-specific biology. To our knowledge, this represents the most comprehensive analysis of the GBM TME to date in terms of patient number (n=1,979) and TME cell type plexy (n=39). The findings offer new understanding of GBM multicellular organization and how different cell types coordinate within the tumor. These data may help design rational combinations of immunomodulatory therapies and guide patient selection for next-generation clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91fc74608f0fb0d307e1af10e8fcfdf8bbe8b02e" target='_blank'>
              TMIC-63. Microenvironment archetypes in 1,979 glioblastomas reveal cell state-specific drivers of immune recruitment and survival benefit
              </a>
            </td>
          <td>
            Kevin J. Tu, Zachary J. Reitman
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Spatial studies of the tumor microenvironment are crucial for understanding the complex interactions between different cell types and how these interactions contribute to disease progression. This poster presents a streamlined approach to spatial analysis of the tumor microenvironment in brain tissues, focusing on astrocytoma and oligodendroglioma. Additionally, the methodology was applied to Parkinson’s Disease versus healthy brain tissue to observe disease-specific alterations. We employed primary antibody conjugates to perform immunohistochemistry (IHC) on human tissue samples, utilizing an 8-plex panel of markers. The study incorporated IHC-validated antibody conjugates supplemented by an antibody labeling kit for disease-specific markers, facilitating high spatial resolution imaging without the need for lengthy workflows commonly associated with secondary amplification or cyclic stripping methods. Our findings demonstrate a significant increase in astrocytes and oligodendrocytes in the tumor tissue, as revealed by the increased presence of GFAP and Olig2, respectively, compared to healthy controls. The use of the EVOS S1000 imaging system made image capture and analysis remarkably straightforward, further simplifying the workflow. This approach provides clear advantages over traditional methods, highlighting its potential for broad application in immunological and neuropathological studies. For Research Use Only. Not for use in diagnostic procedures.



 Thermo Fisher Scientific



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/957e86629e50682381d4aded4eed8bfc06834bb3" target='_blank'>
              Streamlined spatial analysis of the tumor microenvironment and associated disease states using multiplex immunohistochemistry 9329
              </a>
            </td>
          <td>
            Mae Voeun, C. Langsdorf
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2e088dcd4f395d6c7df482c6c35fcef0137dc41" target='_blank'>
              An epigenetic switch in vascular phenotype augments anti-tumor immunity
              </a>
            </td>
          <td>
            Dae Joong Kim, Mitchell T McGinty, S. Anandh, C. Riedstra, Yuvraj Sethi, Melanie R. Rutkowski, A. C. Dudley
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Over 95% of primary central nervous system lymphoma (PCNSL) cases are histologically classified as diffuse large B-cell lymphoma (DLBCL), but present markedly poorer prognosis than systemic DLBCL. This discrepancy may stem from its high molecular heterogeneity, immune privilege, inhibitory tumor microenvironment (TME) and dilemma in drug delivery. Current understanding in the tumor ecosystem of PCNSL remains limited. This study employed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomic (ST) analyses to decode tumor microenvironmental features of PCNSL.
 We performed scRNA-seq of PCNSL tumor samples and reactive lymphoid hyperplasia samples, and additionally conducted ST on PCNSL tumor samples. Unsupervised clustering of B cells identified 4 malignant B cell (mBc) subpopulations with apparently transcriptional and spatial discrepancy. Pseudotime trajectory analysis revealed that mBc3 represented the origin, which evolved into mBc4 and mBc1 and ultimately differentiated into mBc2. To assess the clinical relevance of these subsets, we applied BayesPrism for deconvolution of bulk RNA-seq data from DLBCL cohorts, presenting mBc2 subset was significantly correlated to poor prognosis (p< 0.001). Kyoto Encyclopedia of Genes and Genomes pathway analysis and scMetabolism analyses indicated that mBc2 exhibited a strong oxidative phosphorylation (OXPHOS) metabolic signature. ST analyses of tumor specimens further confirmed the overlap of mBc2 localization and the regions with elevated OXPHOS activity.
 Based on its high OXPHOS characteristics, we further explored its molecular expression features in an attempt to identify potential therapeutic targets. COX7B, a structural subunit of mitochondrial respiratory chain complex IV, was identified as an mBc2 marker through Venn analysis, specifically overexpressed in the mBc2 subset, the terminal stage of developmental trajectory. In vitroassays demonstrated that COX7B expression is upregulated in activated B-cell-like DLBCL cell lines, and its knockdown significantly reduced cell viability and induced apoptosis, supporting its potential as a therapeutic target.
 The previous studies have shown that clonally expanded CD8+T cells in PCNSL undergo a dynamic transition from a pre-exhausted state to an exhausted state, exhibiting more pronounced T cell exhaustion features compared to systemic DLBCL, suggesting the presence of an immunosuppressive TME in PCNSL. However, the specific mechanisms driving T cell exhaustion and immune suppression remain incompletely understood. Recognizing the critical role of T cells in the TME and in determining therapeutic response, we further investigated the immunosuppressive landscape of T cells and regulatory mechanisms mediated by tumor cells. T cell cluster analysis demonstrated a substantial enrichment of exhausted T cells (exhTc) and regulatory T cells (TregCD4) in PCNSL. Key exhausted molecules (TIGIT, HAVCR2/TIM-3, LAG3, CTLA4, and PDCD1/PD-1) were notably upregulated. ST confirmed high expression of HAVCR2/TIM-3 in T-cell-enriched regions. Importantly, the ligands for HAVCR2/TIM-3, CTLA4 and CEACAM1 were found to be upregulated in malignant B cells, with CD80 (CTLA4 ligands) particularly enriched in B-cell regions of spatial slices. These findings suggested active involvement of mBc in shaping the immunosuppressive TME. To explore intercellular signaling, we used CellChat to construct the communication network within the PCNSL microenvironment. mBc subsets showed stronger interactions with exhTc and TregCD4 compared to other TME cell types, primarily mediated by the macrophage migration inhibitory factor (MIF) signaling pathway. CD74+CXCR4 and CD74+CD44 were identified as key ligand-receptor pairs in this pathway. Analysis of spatial slices revealed elevated expression of CXCR4 and CD74 in tumor regions, further indicating widespread activation of the MIF pathway. These findings suggest that mBc subsets mediate the immunosuppression of exhTc and TregCD4 via MIF signaling.
 In conclusion, this study reveals the intratumoral heterogeneity and immunosuppressive microenvironment of PCNSL, identifies the high-risk COX7B⁺ mBc2 subset, and indicates COX7B as a potential therapeutic target. Additionally, we clarify the key role of MIF signaling in immune suppression, offering new directions for targeted and immune therapies in PCNSL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f0ba610cbed6b065d81ef84a88bd658adbfc55a" target='_blank'>
              Single-cell and spatial transcriptomics analyses decipher malignant B cell heterogeneity and immunosuppressive microenvironment in primary central nervous system lymphoma
              </a>
            </td>
          <td>
            Cong Wang, T. Lu, Xiangrong Zheng, Ran Kong, Mei Ding, Xiaohui Sui, Xiangxiang Zhou
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0abad77c5fcae33c11d7d851ad3eb337d3881a5" target='_blank'>
              Exploring Single-Cell and Multi-Omics Technologies and Their Role in Unravelling Tumor Heterogeneity of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Charmi Jyotishi, Suresh Prajapati, Mansi Patel, Reeshu Gupta
          </td>
          <td>2025-12-17</td>
          <td>Journal of liver cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 The World Health Organization provides clear criteria for diagnosing IDH1-wildtype glioblastoma (GBM), relying on traditional approaches to define pathoetiological features, genomic mutations, copy number variation, and aberrant expression of prognostic genes. While these methods guide diagnosis, they may lack the resolution to fully capture tumor heterogeneity, therapeutic response, or outcomes. To address this, we applied leading-edge technologies to 14 primary and recurrent tumors to add granularity to current diagnostic metrics and clinical classification. Aiming to better inform tumor subtyping and improve patient care, this integrative approach enabled us to identify malignant gene programs and localize prognostically relevant features at both single-cell and spatial levels. We analyzed 14 GBM tumors from 6 patients using spatial in situ sequencing (ISS) and single-nuclei RNA-sequencing. A custom ISS gene panel enabled spatial mapping of tumor-associated cell types and state markers, while matched snRNA-seq from adjacent tumor tissue provided transcriptome-wide profiles. One patient contributed 5 serially collected recurrent tumors, enabling longitudinal analysis of tumor evolution. Profiling these tumors using snRNA-seq identified canonical copy number alterations (+7/–10) and ligand-receptor interaction analysis showed an evocative ANXA1 signaling axis progress across recurrent tumors, which suggests possible therapeutic opportunities. Finally, the dataset enabled subcellular and spatial localization of the prognostic GBM biomarker, EGFRvIII, being enriched in oligodendrocyte progenitor (OPC)-like tumor cells, which provides exciting evidence of these cancer cell subtypes as the putative cell of origin for GBM. By extending beyond conventional diagnostic benchmarks, our molecular profiling reveals new insight into GBM biology that may support more refined tumor subtyping and prognostic assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb079e9e9eb8a8e1b502483557634499fe4e750" target='_blank'>
              BIOM-72. INTEGRATIVE SINGLE-NUCLEI SPATIAL TRANSCRIPTOMIC PROFILING OF PRIMARY AND RECURRENT HUMAN IDH-WILDTYPE GLIOBLASTOMA REVEALS CELL-SPECIFIC EGFRVIII EXPRESSION AND ANXA1 SIGNALING AXIS
              </a>
            </td>
          <td>
            Lauren Powers, Vaibhav Jain, Michael Aksu, Emily Hocke, Alan Smith, Stephanie Arvai, Ellora Haukefrers, Kevin Stevenson, Khooshbu Patel, Diane Satterfield, Elizabeth Thomas, Karen Abramson, Giselle López, R. McLendon, David M. Ashley, Simon Gregory
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Epithelial compartments play a central role in the progression and metastasis of colorectal cancer (CRC), yet the mechanisms underlying their transcriptional reprogramming across disease stages remain incompletely understood. We constructed comprehensive single-cell and spatial transcriptomic atlases encompassing normal colorectal tissue, primary CRC, and liver metastases (LMs), profiling over 200,000 high-quality cells. Specific coexpression modules and transcriptional regulators were identified and spatially mapped. Additionally, the clinical and functional roles of SCAND1 were investigated using western blot, RT-qPCR, and immunohistochemistry in clinical samples, as well as functional assays in CRC cell lines. Distinct coexpression modules were selectively enriched in normal epithelial cells and associated with vesicle transport, ion homeostasis, and barrier function, whereas LM-specific modules were linked to Wnt signalling, ribosome biogenesis, and cell cycle processes. High-CNV epithelial cells exhibited progressive activation of key transcription factors such as CEBPB, E2F, and the AP-1 family, with concurrent suppression of ARGLU1 and MTNR2L8, indicating coordinated remodelling of transcriptional and metabolic programs during metastasis. These changes were spatially validated and correlated with alterations in epithelial composition within the tumour core. Consistently, multiplex immunofluorescence (mIHC) confirmed the expression patterns of CEBPB, ARGLU1, S100A4 and FOSL1, across normal epithelium, CRC, and LM. Notably, SCAND1 was identified as a potential therapeutic target for malignant epithelial plasticity. We demonstrated for the first time that SCAND1 expression is significantly upregulated in CRC patients and is associated with superior diagnostic and prognostic value. Functional experiments revealed that SCAND1 promotes tumour cell proliferation, inhibits apoptosis, and enhances metastatic potential. Furthermore, co-culture of SCAND1-overexpressing tumour cells with T cells showed that SCAND1 overexpression attenuates T cell–mediated cytotoxicity, supporting an additional immune-evasive role. Our study elucidates the molecular logic of epithelial state transitions during CRC metastasis and underscores the promise of SCAND1 as a novel biomarker and therapeutic target in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/914a9adf8fcce69d1ec7290317d881a8d12ed794" target='_blank'>
              Dynamic remodelling of epithelial plasticity in colorectal cancer from single-cell and spatially resolved perspectives
              </a>
            </td>
          <td>
            Chengyuan Xu, Yun Pan, Siqi Zhang, Moubin Lin, Hailong Liu, Zicheng Yu
          </td>
          <td>2025-11-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Introduction
 Classical Hodgkin Lymphoma (cHL) is one of the most manageable human cancers. The early identification of patients at risk of relapse following front-line therapy, currently represents an unsolved clinical need. cHL is a complex ecosystem characterized by an intricate and mutual interaction between an extensively dominant tumor microenvironment (TME) and malignant Hodgkin Reed-Sternberg (HRS) cells; disease progression likely reflects some innate features of this organization. Nevertheless, even the most advanced prognostic models fail to fully account for the clinical heterogeneity of the disease. A major limitation lies in the inherent difficulty of profiling HRS cells, given their rarity and unique distribution within the tumor parenchyma. In this study, we employed an integrated multidimensional transcriptomic analysis to explore the peculiar architecture of the cHL tumor ecosystem and its potential relationship with clinical progression and patient outcomes.
 Methods
 Patient cohort: A retrospective training cohort of untreated cHL patients (N=155), age < 65 years and including all disease stages, was selected. Patients were classified as either relapsed/refractory (R/R+) (N=31) or non-relapsed (R/R-) (N=124) based on a minimum follow-up of 3 years.
 A morphology-guided spatial transcriptomics approach was used to analyze the transcriptional profile of HRS and TME components in surgical lymph node resections of a matched R/R-:R/R+ case-control subset of the cohort. A total of 118 Areas of Interest (AOIs) were collected, 56 from the TME (CD45+/CD30-) and 62 from HRS-rich regions (CD30+). Digital barcoding profiling was used to assess the expression of 770 immune-related genes across the entire training cohort. Cox regression analysis was used to identify genes significantly associated with progression-free survival (PFS). IHC was used to confirm the validity of the derived model.
 Results
 Morphology-guided profiling of HRS cells showed significant differences between R/R+ and R/R- cHLs. Unsupervised clustering based on the transcriptional profile of HRS cells strongly segregated AOIs according to clinical classification. Topographical organization of HRS within the tumor parenchyma, along with paracrine signaling, accounted for HRS differential properties in the two groups. The TME in R/R+ cHLs exhibited an immunosuppressed phenotype, defined by the exclusion of non-tumoral B-cells and overrepresentation of immunosuppressive populations (M2 Macrophages, Tregs and Mast cells). This phenotype was associated with altered immune-modulatory signaling by HRS cells, involving inhibition of stimulatory mediators such as chemotactic cytokines (CXCL9, CXCL10, etc) and IFN signaling and the upregulation of inhibitory molecules including CCL17 (encoding for TARC) and IL13. An inverse correlation was observed between the presence of non-tumoral B-cells and immunosuppressive populations. Digital barcoding gene expression analysis on the entire cohort (N=155) confirmed this model and identified a non-tumoral B-cell specific gene signature (N=18) that significantly correlated with improved PFS and stratified patients according to risk of progression (HR 0.23 95% CI 0.06-0.87; p=0.03)
 Conclusion
 This study provides, one of the first molecular characterization of HRS cells and of their interplay with the surrounding TME, uncovering mechanisms that drive immune escape and disease progression. Notably, our findings reveal a previously unrecognized protective role of non-malignant B cells residing in the TME of cHL, highlighting their potential utility as biomarkers for risk stratification in cHL. Finally, our data demonstrate that integrating high-resolution transcriptional profiling with morphology enables a more precise understanding of cHL complexity, capturing not only intrinsic variability but also the reciprocal spatial organization between cell populations, which significantly shape the biological properties of the entire tumor ecosystem.
 REFERENCES
 1. Scott, DW &Gascoyne, RD The tumour microenvironment in B cell lymphomas. Nat. Rev. Cancer 14, 517–534 (2014)
 2. Aoki, T. et al. Single-cell transcriptome analysis reveals disease-defining t-cell subsets in the tumor microenvironment of classic hodgkin lymphoma. Cancer Discov.10, 406–421 (2020)
 3. Aoki T. et al. Spatially Resolved Tumor Microenvironment Predicts Treatment Outcomes in Relapsed/Refractory Hodgkin Lymphoma. J Clin Oncol. doi: 10.1200/JCO.23.01115. (2024)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e25acab9e1dc7d742c876b8b8b809c99b9c1c2e9" target='_blank'>
              Morphology guided spatial transcriptomics to study topography of Hodgkin reed sternberg cells interaction with tumor microenvironment and its impact in defining risk of progression in classical Hodgkin lymphoma.
              </a>
            </td>
          <td>
            B. Donati, T. Lazic, Riccardo Valli, R. Durmo, Attilio Gennaro, Cristian Ascione, Pierluigi Di Chiaro, A. Ruffini, M. Gotti, Luca Arcaini, Erica Travaglino, Marco Paulli, Francesco Merli, A. Ciarrocchi, S. Luminari
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Plasticity, or the ability to rapidly and reversibly change phenotypes, may help explain how a single progenitor cell eventually generates a tumor with many different cell phenotypes. Normal colon plasticity is characterized by a conserved and broadly permissive epigenome, where expression and phenotype are determined by the microenvironment instead of epigenetic remodeling. To determine whether this stem-like plasticity is retained during progression, gene expression was measured with spatial transcriptomics and compared with gene-level DNA methylation in two colorectal cancers (CRCs). Like normal colon, genes that were differentially expressed between regions, subclones, and phenotypes (superficial, invasive, and metastatic) tended to have lower DNA methylation variability. We propose a quantitative signal of plasticity that correlates gene epigenetic variability with gene expression variability. In this framework, negative correlation implies phenotypic plasticity, as more variably expressed genes tend to have less epigenetic variability. We verify the presence of this signal in multiple external single-cell RNA-Seq datasets, in both normal colon and CRC samples. Therefore, the plasticity of normal colon appears to be retained during progression. A CRC progenitor with a preconfigured plastic phenotype is poised for rapid growth because it expresses, as needed, transcripts required for progression with minimal epigenetic remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86cc1dfed608d8a0144edef3180374f9ebee40c0" target='_blank'>
              Epigenetic conservation infers that colorectal cancer progenitors retain the phenotypic plasticity of normal colon
              </a>
            </td>
          <td>
            Kelly Street, Yifan Zhang, Kimberly Siegmund, Darryl Shibata
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Secreted phosphoprotein 1 (SPP1+) macrophages are a recurrent and functionally critical immune cell subset across multiple cancer types. They drive adverse clinical outcomes by promoting immunosuppression, tumor invasion, metastasis, and therapy resistance. Given their prevalence and pivotal role, SPP1+ macrophages have become a major focus in cancer immunology and a promising target for therapeutic development. SPP1+ macrophages have been identified in a wide range of human malignancies through single-cell RNA sequencing and spatial transcriptomics studies. Their differentiation and maintenance are strongly influenced by reciprocal cellular interactions and hypoxic conditions within the tumor microenvironment (TME). Within the tumor microenvironment (TME), SPP1+ macrophages promote tumor progression by interacting with cancer-associated fibroblasts (CAFs) and helping to form a physical barrier that restricts immune cell infiltration into the tumor core. Specifically, they impair the recruitment of CD8+ T cells and promote T cell exhaustion (TEX). In this review, we focus on recent advances in understanding the differentiation of SPP1 macrophages in hypoxic tumor microenvironment and the role of SPP1+ macrophages in immunosuppression and their therapeutic implications in cancer. Targeting this subset of macrophages has emerged as a highly promising therapeutic strategy, with several approaches demonstrating encouraging results in preclinical models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7c7489149a391c5b3e442dad42f10334718f488" target='_blank'>
              SPP1+ macrophages in tumor immunosuppression: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Juanjuan Wang, Ya Wang, Yuqing Liu, Rongcun Yang
          </td>
          <td>2025-12-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Intratumoral heterogeneity supports cancer progression and metastasis while limiting therapeutic efficacy. Comprehensive biological characterization of heterogeneous subclones in breast cancer may hold promise in battling against this deadly disease. Using single-cell transcriptomic analysis and in situ profiling of primary tumors, metastases, and circulating tumor cells from multiple patients with breast cancer, we identified a unique EPCAM+ CD68+ TREM2+ tumor subpopulation, likely resulting from the fusion of tumor cells and lipid-associated macrophages (LAM). The presence of these tumor-LAM fusion cells in the blood or in distinct metastatic sites was significantly correlated with metastatic progression. Stable fusion clonal lines established in vitro exhibited substantially enhanced proliferation, tumor initiation, and metastasis formation in mice. Integrative molecular and functional analyses revealed a critical role for SNX10 in mediating tumor-LAM fusion. Mechanistically, SNX10 physically interacted with the phospholipid scramblase ANO6 and maintained its protein stability by suppressing proteasome-mediated degradation. Furthermore, the transcription factor CBX3 directly targeted SNX10 promoter and modulated its expression. Fusion cells accumulated abundant intracellular lipid droplets and were highly sensitive to simvastatin treatment in vitro and in vivo. Together, this study uncovered that CBX3-SNX10-ANO6 signaling facilitates generation of an aggressive tumor-LAM fusion cell subpopulation that promotes metastasis, revealing an alternative metastatic mechanism and exposing putative therapeutic vulnerabilities.


SIGNIFICANCE
Single-cell transcriptomic profiling combined with functional and clinical validation identifies fusion of tumor cells and lipid-associated macrophages mediated by the CBX3-SNX10-ANO6 axis as a potentially targetable mechanism driving cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4271e1aeed14ff3c28d665da96c160262a128c0a" target='_blank'>
              Fusion of Tumor Cells with Lipid-Associated Macrophages Drives Metastatic Progression of Breast Cancer.
              </a>
            </td>
          <td>
            Yixin Cheng, Guanyin Huang, Xuefei Liu, Chun Wu, Jingru Lian, Jing Cao, Lianhui Duan, Boxi Zhao, Yufan Yang, Siqi Chen, Guanqiao Li, Liping Wu, Shuqian Zheng, Wenjing Wang, Tianyu Zha, Yang Yang, Yilin Lu, Jian Zhang, Hao Yu, Jun Tan, Feiqiu Wen, Zhenyu He, Xin Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5bf5750b51fd180863b176de6d4f229a2cae4e" target='_blank'>
              Single-cell and spatial transcriptomics reveal lactylation-associated tumor cell clusters and define a prognostic risk model in glioblastoma.
              </a>
            </td>
          <td>
            Rui Han, Guangfan Chi, Dongjie Sun, Ziran Xu, Liangfu Zhou, Kan Xu
          </td>
          <td>2025-11-22</td>
          <td>BMC cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce05a3966abc2d6663f2f9fc06215e6d78446ea" target='_blank'>
              Tumor microenvironment-mediated immune evasion and resistance in prostate cancer: mechanisms, cross-talk, and therapeutic opportunities.
              </a>
            </td>
          <td>
            Mingfeng Li, Louis Boafo Kwantwi, Cuimei Wang, Qin Xiao
          </td>
          <td>2025-11-26</td>
          <td>Clinical and experimental medicine</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba29f656998508469f49c53f9437fe122dadd98" target='_blank'>
              DYRK1A regulates cancer cell and cancer-associated fibroblast secretomes to foster an immunosuppressive microenvironment in pancreatic cancer
              </a>
            </td>
          <td>
            Silvia Pascual-Sabater, Sushmita Varhadi, Chiara Di Vona, Marta Celma, Cristina Fillat, S. de la Luna
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 The tumor microenvironment is highly heterogeneous, composed of diverse cell types with distinct phenotypes and characteristics. Gaining insights into these differences is essential for advancing cancer immunotherapy and research. Reliable results demand high-quality samples and standardized processing, however, generating single-cell suspensions from solid tumors is challenging due to variations in tumor type, size, texture, and developmental stage. Balancing enzymatic and mechanical dissociation is critical to preserve cell viability and yield. We developed the STEMprep™ Mouse Tumor Dissociation Kit for use with the STEMprep™ Automated Tissue Dissociation System, featuring a user-friendly instrument and specially designed sample tubes. This system includes an optimized enzyme formulation and a single protocol that works across diverse tumor types. Our findings demonstrate consistently high cell viability and yield (cells/mg tissue) across tumors: soft B16 melanoma (91.2 ± 3.3% and 1.5E4 ± 7.4E3 [n = 15]), medium-firm CT26 colon carcinoma (82.9 ± 6.1% and 2.6E4 ± 1.3E4 [n = 9]), and firm 4T1 mammary carcinoma (88.5 ± 4.0% and 2.7E4 ± 1.7E4 [n = 4]). This method maintains cell integrity and epitope availability for downstream analysis. Furthermore, STEMprep™-processed samples are compatible with EasySep™ magnetic cell isolation technology, and are functional in T cell suppression assays, enhancing tumor research workflows with efficiency, consistency, and reliability.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15528cd65eafa51df7aab4672196a35521811783" target='_blank'>
              Streamlined mouse tumor processing with STEMprep™: Automated, Efficient, and Reliable 3260
              </a>
            </td>
          <td>
            F. Antignano, G. Poon, So Ra Lee, Jenna Thompson, A. Chenery, Alice Liang, Payam Zachkani, Andrew Nobles, Bilal El yassem, Chris Ryan, Marc Delorme, Rodrigo Martins, Martin O’Keane, A. Kokaji, A. Eaves, Sharon Louis
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Gliomas and brain metastases are associated with poor prognosis, necessitating a deeper understanding of brain tumor biology and the development of effective therapeutic strategies. Although our group and others have demonstrated microbial presence in various tumors, recent controversies regarding cancer-type-specific intratumoral microbiota emphasize the importance of rigorous, orthogonal validation. This prospective, multi-institutional study included a total of 243 samples from 221 patients, comprising 168 glioma and brain metastases samples and 75 non-cancerous or tumor-adjacent tissues. Using stringent fluorescence in situ hybridization, immunohistochemistry and high-resolution spatial imaging, we detected intracellular bacterial 16S rRNA and lipopolysaccharides in both glioma and brain metastases samples, localized to tumor, immune and stromal cells. Custom 16S and metagenomic sequencing workflows identified taxa associated with intratumoral bacterial signals in the tumor microenvironment; however, standard culture methods did not yield readily cultivable microbiota. Spatial analyses revealed significant correlations between bacterial 16S signals and antimicrobial and immunometabolic signatures at regional, neighborhood and cellular levels. Furthermore, intratumoral 16S bacterial signals showed sequence overlap with matched oral and gut microbiota, suggesting a possible connection with distant communities. Together, these findings introduce microbial elements as a component of the brain tumor microenvironment and lay the foundation for future mechanistic and translational studies. A combination of high-resolution spatial imaging, spatial proteomics and transcriptional data reveals sparse and heterogeneous bacterial signals in gliomas and brain metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae0d1782181d2f2125dbd26294df4180d5bf9d82" target='_blank'>
              Microbial signals in primary and metastatic brain tumors
              </a>
            </td>
          <td>
            Golnaz Morad, Ashish V Damania, Brenda Melendez, Bharat Singh, Fabiana J. Veguilla, Rebecca A. Soto, Yasmine M. Hoballah, P. Sahasrabhojane, M. Wong, Mona M. Ahmed, Rene N. Rico, Kaitlyn N. Lewis, Khalida M Wani, D. Shamsutdinova, Rossana N. Lazcano Segura, D. Ingram, Eric A. Goethe, Abderrahman Day, Ivonne I Flores, Lauren K McDaniel, M. Chelvanambi, Sarah B. Johnson, Florentia Dimitriou, Pravesh Gupta, Shivangi Oberai, M. Zal, Phoebe Doss, Mohamed A Jamal, Eiko Hayase, Chetna Wathoo, Lisa M. Norberg, Stephanie L. Jenkins, Sara Nass, J. Gumin, L. Long, Jing Yang, Gina R. Bradley, M. Bekal, A. Dono, P. Pichardo-Rojas, Samuel W. Andrewes, Leomar Ballester, J. Losh, Jiyong Liang, Longfei Huo, Douglas Nielsen, Brittany C Parker Kerrigan, Priscilla K. Brastianos, N. Fowlkes, Chia-Chi Chang, Robert R. Jenq, C. Gomez-Manzano, Jason T Huse, Michael A. Davies, A. Lazar, Krishna P. Bhat, Nitin Tandon, Y. Esquenazi, Christine B. Peterson, Vinay K. Puduvalli, Frederick F. Lang, Christopher D. Johnston, Susan Bullman, N. Ajami, S. Ferguson, J. Wargo
          </td>
          <td>2025-11-01</td>
          <td>Nature Medicine</td>
          <td>1</td>
          <td>49</td>
        </tr>

        <tr id="


 Pancreatic neuroendocrine tumors (panNETs) undergo high-grade (HG) transformation often, with the liver being a common site for metastasis​. Surgical resection being considered as long-term outcomes for selected patients. However, recurrence and grade transformation remain potential significant challenges, with recurrence rates approaching 50–95%.1,2 Despite extensive whole exome analysis and construction of phylogenetic trees, we were unable to identify either a consistent genetic driver or any specific patterns of tumor transformation.2 We are also concerned that some of our treatments may inadvertently contribute to grade transformation. To assess the treatment effects, we performed pre- and post- trans arterial embolization bulk-transcriptome analysis, which revealed cell population shifts in panNETs that may be linked to grade transformation.4To explore cellular shifts within the liver tumor microenvironment during panNET progression, we performed single-cell RNA sequencing. The resulting data was processed using Seurat.v5. The clusters of panNET cells, fibroblasts, hepatocytes, and immune cells were annotated by known markers. Additionally, we distinguished malignant cells from a reference of immune cells, fibroblasts, and hepatocytes by InferCNV. To understand the tumor progression, we performed trajectory analysis using monocle3. Differential metabolic activity between the panNETs grades were assessed using the scMetabolism package. Cell-cell communication and related ligand-receptor pairs were inferred using CellChatDB. Our results revealed distinct transcriptional trajectories, with pseudotime analysis demonstrating a trajectory of tumor evolution from low-grade (LG) to HG states. HG tumors were enriched for proliferative and aggressive subclusters compared to LG cells. Metabolic profiling further highlighted grade-specific adaptations, with HG tumors showing enhanced oxidative phosphorylation and TCA cycle across tumor cells, whereas LG tumors displayed relatively lower metabolic activity. Notably, immune cells were found metabolically active in LG via glycolysis. Conversely, HG panNETs exhibit glucose depletion in immune cells and less active microenvironment. The CellChat analysis revealed distinct cellular communication patterns in panNET grade. For instance, in HG tumors, signaling was primarily mediated between fibroblasts and panNET cells, with Glu-(SLC1A2+GLS)-GRM7 as the dominant ligand–receptor pair. In LG tumors, major interactions occurred among immune cells, hepatocytes, and panNET cells, with Glu-(SLC1A3+GLS)-GRIK2 representing the highest contributor. The results suggest that metabolic rewiring, altered intercellular signaling, and transcriptional plasticity underlie progression from LG to HG disease.
 Reference:
 1. Spolverato etal J Gastrointest Surg. 2017, 21(12):2039-2047
 2. Mayo etal Ann Surg Onc. 2011, 18(13):3657-65.
 3. Singh et al. Early ... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium. 2023.
 4. Singh et al. Post... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium.2023



 Himanshu N. Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jungeui Hong, Nitya Raj, Olca Basturk, Zeynep Tarcan, Christine A. Iacobuzio, Etay Ziv. Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f19a2d46016dbf4a94424ac84c3f39682a28b2d" target='_blank'>
              Abstract B025: Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors
              </a>
            </td>
          <td>
            Himanshu Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jung-Pyo Hong, Nitya Raj, O. Basturk, Z. Tarcan, Christine A. Iacobuzio, E. Ziv
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa8b0639992b1a58eec77b9ba628d8cad1cb0d2" target='_blank'>
              Somatic copy number mutations contribute to fitness in transplantation models of spontaneous human breast cancer metastasis
              </a>
            </td>
          <td>
            Hoa Tran, Gurdeep Singh, Hakwoo Lee, Damian Boon Siew Yap, Eric Lee, William Daniels, Farhia Kabeer, C. O’Flanagan, Vinci Au, Michael Van Vliet, Daniel Lai, Elena Zaikova, Sean Beatty, Esther Kong, Shuyu Fan, Jessica Chan, Hoang Quan Dang, Viviana Cerda, Teresa Ruiz de Algaza, Andrew Roth, Sam Aparicio
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e555b898cb0281329d778a02e7fea08e0c7964" target='_blank'>
              Single cell transcriptional perturbome in pluripotent stem cell models.
              </a>
            </td>
          <td>
            Elisa Balmas, M. Ratto, Kirsten E. Snijders, Silvia Becca, Carla Liaci, Irene Ricca, Giorgio R Merlo, R. Calogero, Luca Alessandrì, Sasha Mendjan, Alessandro Bertero
          </td>
          <td>2025-12-10</td>
          <td>Molecular systems biology</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/759c03413ef4b1106fa2a25c75713885026e6884" target='_blank'>
              Single-cell transcriptomic analysis reveals key cell types and pathogenic genes associated with metastasis in pancreatic adenocarcinoma.
              </a>
            </td>
          <td>
            Shijun Shen, Zhiqiang Li, Hong Yang, Xinwei Zhang, Gang Chen, Chengzhou Pa
          </td>
          <td>2025-11-29</td>
          <td>Clinical and experimental medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0966480a39b8d44322c078389ea27b385628ca19" target='_blank'>
              Comprehensive characterization of early-stage Non-Small Cell Lung cancers with multi-modal data integration
              </a>
            </td>
          <td>
            D. Schulz, M. Morfouace, B. P. Fernandez, M. Możejko, S. Déglise, M. Daniel, N. Eling, H. Hong, S. Tissot, S. Rusakiewicz, T. Trefzer, J. Merlevede, F. Marzetta, N. de Souza, D. Wojtowicz, E. Szczurek, B. Besse, B. Bodenmiller
          </td>
          <td>2025-11-13</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Introduction : Chronic graft-versus-host disease (GVHD) is a leading cause of late non-relapse mortality following allogeneic hematopoietic cell transplantation, yet spatial biomarkers capable of predicting outcomes remain lacking. Minor salivary glands (MSGs) are anatomically accessible organs that can be easily accessed by fast and minor invasive biopsies and reflect systemic immune perturbations, making them ideal for spatial profiling of immune diseases. We applied STARComm, a spatial multiomics framework that identifies Multicellular Interaction Modules (MCIMs), which represent tissue microenvironments where multiple cell types engage in coordinated signaling. MCIMs capture clinically relevant hubs of immune communication and tissue remodeling, enabling spatial resolution of pathogenic processes in GVHD.
 Methods : Thirty-six MSG tissues were collected from patients enrolled between 2018 and 2022 at the Hospital das Clínicas, University of São Paulo. All patients had undergone allogeneic bone marrow transplantation and subsequently received MSG biopsies as part of clinical follow-up for chronic GVHD assessment. The cohort included healthy controls (n=8), GVHD survivors (n=18), and patients who died from GVHD-related complications (n=10), with detailed clinical metadata. Spatial multiomic profiling was performed on the same histological section using a dual-modality approach: spatial transcriptomics with Xenium (10x Genomics) using a targeted 280-gene panel, and spatial proteomics with Phenocycler Fusion (Akoya Biosciences) using a 36-antibody multiplex panel. MCIMs were computed via STARComm by integrating local communication density maps across spatial grids. Cell types were annotated with TACIT, and differences in MCIM composition were compared across groups. Prognostic associations were modeled using Cox regression. To prioritize therapeutic targets, we applied SpatialDrug2Cell, a single-cell-based pipeline for inferring druggable dependencies, to spatially segmented cell populations, adapting it for high-resolution in situ data.
 Results:A total of 842,154 spatially resolved cells were analyzed across all samples. Receptor–ligand interactions were inferred from 17 curated gene pairs selected for biological relevance to GVHD and compatibility with both transcriptomic and proteomic detection, based on expression specificity and spatial proximity. Nine MCIMs were identified. In healthy MSGs, MCIMs reflected structured stromal, epithelial, and vascular signaling. GVHD samples exhibited marked reorganization, with MCIMs-0, -2, and -6 significantly enriched (p<0.001), particularly within TLS-like periductal niches. MCIM-0, dominant in fatal cases, exhibited dense B and T cell, T and T cell, and stromal interactions, with high spatial density of CXCL12-CXCR4 and CCL5-SDC4 signaling, primarily transcribed by fibroblasts and pericytes. In contrast, MCIM-3, enriched in fibroblast–endothelial interactions and B cell regulatory networks, was more abundant in survivors. In multivariate analysis, MCIMs-0 and -6 were independently associated with increased mortality (HR=2.08 and 2.83), while MCIM-3 showed a protective trend (HR=0.51). SpatialDrug2Cell, applied to MCIM-localized cell populations and spatially restricted therapeutic targets, highlighting candidate agents such as rituximab, atezolizumab, and plerixafor.Conclusions:Our analysis establishes spatially resolved MCIMs as prognostic biomarkers in chronic GVHD. The emergence of high-risk communication modules in MSGs, particularly in patients with fatal outcomes, reveals tissue-localized immune architectures associated with disease progression. By integrating STARComm with adapted drug inference tools like SpatialDrug2Cell, we identify clinically relevant, spatially anchored therapeutic targets in routinely sampled tissues. This framework may enable mechanism-based stratification and therapeutic guidance in GVHD and other immune-mediated diseases
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1daff8734fed12743fa31dcf5eea76ecab05128e" target='_blank'>
              Spatial communication modules identify prognostic cell-cell signaling hubs in chronic graft-versus-host disease
              </a>
            </td>
          <td>
            Bruno Fernandes Matuck, Khoa L. A. Huynh, Deiziane de Souza, G. Fatobene, Luiz Valente Soares Junior, L. F. Ferraz da Silva, Marisa Dolhnikoff, V. Rocha, Jinze Liu, Kevin Byrd
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer is a highly heterogeneous disease, with many cancers containing multiple distinct subclones. While subclones are often seen as competitors (survival of the fittest), intratumor heterogeneity can also offer direct benefits to the tumor through cooperation between different clones. This has important clinical implications, as interdependent populations may present therapeutic vulnerabilities. Here, we review existing evidence for clonal cooperativity to address key questions and outline future developments based on six overarching principles: (a) secreted factors are important mediators of clonal cooperation; (b) (very) small subclones can significantly affect tumor behavior; (c) both genetic and nongenetic heterogeneity are substrates for cooperation; (d) nonmalignant cells from the tumor microenvironment can act as cooperating partners; (e) clonal cooperation occurs throughout different stages of cancer, from premalignancy to metastasis; and (f) clonal cooperation can promote therapy resistance by protecting otherwise sensitive populations. Together, these principles suggest clonal cooperation as an important mechanism in cancer. Lastly, we discuss how novel technological developments could address remaining gaps to open up new therapeutic strategies that exploit clonal cooperativity by targeting the tumor's weakest link.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb39ce21ba07d76bffe91c932b77c8c8a7e6988d" target='_blank'>
              Strength through diversity: how cancers thrive when clones cooperate.
              </a>
            </td>
          <td>
            Marije C. Kuiken, Maartje Witsen, E. Voest, K. Dijkstra
          </td>
          <td>2025-11-09</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Chromatin organization within the cell nucleus governs the epigenetic regulation of diverse cellular functions and is tightly controlled within the spatial context of tissues. In human non-Hodgkin B-cell lymphoma (B-NHL), epigenetic regulation is critical to disease pathogenesis, yet their underlying mechanisms remain poorly understood. While formalin-fixed paraffin-embedded (FFPE) samples are plentiful and invaluable for B-NHL translational research, effective methods for spatial profiling of chromatin accessibility and histone modifications in these tissues remain limited, hindering efforts to unravel epigenetic heterogeneity and regulatory mechanisms in B-NHL.
 Here, we introduce epi-Patho-DBiT, a novel platform integrating reverse crosslinking in FFPE tissues with spatially resolved epigenomic assays, including transposase-accessible chromatin using sequencing (spatial-FFPE-ATAC) and cleavage under targets and tagmentation (spatial-FFPE-CUT&Tag). We applied epi-Patho-DBiT to archival B-NHL FFPE samples to investigate tumor development, progression, and transformation.
 First, using spatial-FFPE-ATAC, we mapped genotype-specific epigenetic landscapes in human mucosa-associated lymphoid tissue (MALT) and follicular lymphoma (FL), identifying chromatin variants linked to B-cell malignancy. Spatially resolved chromatin accessibility profiles enabled inference of genomic copy number variations (CNVs) in MALT and FL lymphomas, resolving tissue heterogeneity, tumor karyotypes, and evolutionary relationships between tumor clones. We identified a chromosome 7 locus (7q21.12–7q31.31) with copy number amplifications (CNAs) in both MALT and FL tumor regions. In addition, we performed mitotic age inference analysis to trace tumor dynamics in MALT and FL, which revealed intra-to-extracellular migration of tumor cells, supported by multiomics velocity and pseudotemporal ordering analysis. Spatial mitotic age inference further identified a strong correlation between mitotic age and cholesterol metabolism gene activity, uncovering cholesterol-mediated cell cycle regulation in lymphoma development.
 Next, we performed spatial-FFPE-CUT&Tag targeting H3K4me3 and H3K27me3 histone modifications in low-grade FL, diffuse large B-cell lymphoma (DLBCL), and concurrent FL-DLBCL transformation samples, which revealed cell proliferation dynamics within the tumor microenvironment and identified DIP2C as a novel upregulated tumor marker. Unexpectedly, in the FL-DLBCL transformation sample, a chromosome 2 locus (2p15–2p16.1) exhibited high expression of multiple tumor-promoting genes alongside elevated levels of the repressive histone mark H3K27me3, presenting a paradox of “high gene expression; high repressor H3K27me3 levels”. This genomic locus covers genes including PAPOLG, REL, KIAA1841 and XPO1. Through whole-genome sequencing and CNV inference from spatial transcriptomics data on the same tissue sample, we demonstrated that this paradoxical observation was attributed to CNAs in transformed DLBCL, highlighting complex interplays between genomic and epigenomic regulation in large cell transformation.
 Overall, epi-Patho-DBiT enables high-resolution, spatially resolved epigenomic profiling of clinical archival FFPE samples, uncovering lymphoma tumor heterogeneity, chromatin variants, and genome-epigenome interactions. This approach provides a powerful tool for decoding the epigenetic mechanisms of lymphoma evolution and transformation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd9d77e4e9cb6ef100c918c54f53734736ee318c" target='_blank'>
              Spatially decoding genotype-associated epigenetic landscapes in human lymphoma FFPE tissues via epi-Patho-DBiT
              </a>
            </td>
          <td>
            Haikuo Li, Bo Tao, Archibald Enninful, Dingyao Zhang, Negin Farzad, Keyi Li, Zhiliang Bai, Xiaoyu Qin, Emily Hwang, Mingyu Yang, Mina Xu, Rong Fan
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) progression and recurrence are often driven by therapy-resistant cellular states emerging within hypoxic tumor niches. While hypoxia is a hallmark of GBM, the spatial transcriptomic heterogeneity that underpins resistance in these regions remains poorly characterized. Here, we employ pimonidazole labeling in conjunction with high-resolution spatial and single cell transcriptomics across 29 primary, treatment-naive human GBM samples, syngeneic mouse models, and patient-derived glioma-initiating cells exposed to hypoxia to define the hypoxia-specific tumor microenvironment and its associated cellular programs. We reveal spatially organized sub-niches within hypoxic regions, each supporting diverse cellular programs yet coexisting tumor lineage states with minimal transitions between cell states. Phylogenetic reconstruction based on inferred copy number variations demonstrates consistent evolutionary trajectories from non-hypoxic to hypoxic tumor loci. Specific copy number alterations were enriched in hypoxic regions and associated with immune infiltration and poor prognosis in MGMT-methylated tumors. Cross-species analyses confirmed the conservation of hypoxia-associated gene programs and spatial architectures. Together, our study delineates the cellular, evolutionary, and spatial complexity of hypoxia in GBM, identifies hypoxia-induced genomic alterations that may promote tumor fitness, and offers insights into new therapeutic vulnerabilities within hypoxic niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec812f9197d64f5c26991c527a5afc02eb1748" target='_blank'>
              TMIC-96. Integrated multi-omics reveals distinct hypoxic sub-niches and associated epigenomic and genomic alterations in glioblastoma
              </a>
            </td>
          <td>
            S. Mansouri, Jefferey Liu, Vikas Patil, Ruth Isserlin, Phoebe Lombard, Ronald Wu, Julio Sosa, S. Gandhi, Y. Ellenbogen, O. Singh, Hafsah Ali, C. Velásquez, Bradly Wouters, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Pineoblastoma (PB) is a clinically aggressive embryonal central nervous system (CNS) tumor composed of distinct molecular subgroups. The cellular and biological basis of PB remains poorly defined, limiting the development of more effective therapeutic strategies. To identify the cellular composition, origins, and drivers of PB, we derived single-cell transcriptomes from patient-derived pineal tumors (n=38) and integrated malignant cells with a novel transcriptional atlas of mouse pineal gland development (E13-P21). Application of a series of computational strategies pinpointed transient, cycling pinealocyte progenitors as the likely cellular origin of PB, irrespective of molecular subgroup. To functionally validate these findings, we generated lineage-specific genetically engineered mouse (GEM) models representing distinct PB subgroups with high temporal, anatomic, and phenotypic fidelity. Transcriptomic analysis further substantiated the accuracy of novel GEM models which converged on cell-cycle dysregulation, highlighting a core pathogenic mechanism across PB subgroups. Relative to the developing pineal gland, photoreceptor transcription factors were aberrantly activated in PB GEM models, suggesting photoreceptor involvement in tumor initiation and/or maintenance. Multi-omic analysis across a large range of diverse CNS cancers revealed an oncogenic photoreceptor program specific to PB, retinoblastoma (RETB), and Group 3 medulloblastoma (G3-MB). This shared transcriptional program was active within respective cellular origins, establishing a unified mechanistic basis across these anatomically distinct CNS tumors. CRISPR-based functional studies confirmed selective dependencies of photoreceptor transcription factors (NRL, CRX, OTX2) in PB, RETB and G3-MB, demonstrating the essentiality of this photoreceptor program. These discoveries not only resolve longstanding uncertainties regarding PB pathogenesis but also highlight conserved molecular signatures and therapeutic vulnerabilities spanning different pediatric CNS malignancies. Our results underscore the potential of targeting developmentally regulated programs and master transcription factors as novel therapeutic strategies for PB, RETB, and G3-MB, offering critical insights and preclinical models for future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cee7231cd5056fbbc7fcab0e2a1e377781c0c82" target='_blank'>
              EPCO-05. AN ONCOGENIC PHOTORECEPTOR PROGRAM UNIFIES DISTINCT CENTRAL NERVOUS SYSTEM TUMORS
              </a>
            </td>
          <td>
            B. Gudenas, S. Ahmad, Bernhard Englinger, Anthony P Y Liu, Miao-yun Zhao, Leena Paul, Jennifer L. Hadley, Hong Lin, Yiran Li, Melissa Batts, Priya Mittal, Stephanie C Wu, Sara Lewis, Katie Han, T. Soliman, Laure Janke, David Meredith, E. Pfaff, J. Gojo, Jennifer Cotter, P. Klimo, F. Boop, Amar Gajjar, G. Robinson, G. Rosén, S. Alexandrescu, David T. W. Jones, Brent A. Orr, F. Swartling, M. Filbin, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="


 Introduction:Plasma cell leukemia (PCL) is a rare yet highly aggressive form of plasma cell dyscrasia, defined by high tumor burden, frequent extramedullary dissemination, and rapid clinical progression. Compared with multiple myeloma (MM), PCL exhibits greater genomic instability, immune escape, and a more hostile bone marrow microenvironment, resulting in poor therapeutic responses and short survival. While prior bulk sequencing and cytogenetic studies have revealed select features of PCL biology, a high-resolution, single-cell dissection of its malignant and immune landscape remains lacking. In particular, how PCL reshapes its tumor microenvironment (TME) to support immune evasion and systemic dissemination has not been fully elucidated.
 Aim: This study aims to characterize the transcriptional, genomic, and immunologic features of PCL at single-cell resolution. We sought to identify PCL-specific oncogenic drivers, immune evasion mechanisms, and intercellular communication circuits that differentiate PCL from MM and healthy bone marrow, with a particular focus on the ERBB signaling axis and T/NK cell dysfunction.
 Methods:Bone marrow samples were collected from three newly diagnosed primary PCL patients and one MM patient from our center, These specimens were subjected to 10x Genomics 3' single-cell sequencing. Additionally, three normal bone marrow and three MM bone marrow samples retrieved from the Gene Expression Omnibus (GEO) database were incorporated for integrative analysis. Comparative analyses were conducted to assess clonal architecture, transcriptional programs, and immune landscape. Key findings were validated using qPCR and publicly available clinical and transcriptomic datasets.
 Results:After quality control filtering, we obtained single-cell transcriptome data from 49,059 high-quality mononuclear cells. Based on canonical marker gene expression, we identified eight major cell populations: plasma cells, myeloid cells, T cells, NK cells, NKT cells , B cells , dendritic cells, and megakaryocytes. This included 11,485 PCs, 3,466 B cells, and 33,603 other immune cells.
 PCL plasma cells demonstrated significantly higher CNV burden than MM, with recurrent gains on chr3q, chr9q, chr15q, and chr18q. Amplification of oncogenes including BCL2, MCL1, PIK3CA, KRAS, and BRAF suggested enhanced apoptotic resistance, proliferative signaling, and immune evasion.
 Transcriptomic profiling revealed PCL-specific upregulation of the ERBB signaling axis, including ERBB4, EPHA6, and downstream effectors KRAS, SOS1/2, and MAPK1/8. ERBB pathway enrichment was confirmed by GSEA and pseudotime analysis revealed persistent upregulation along the malignant differentiation trajectory. High expression of ERBB pathway genes correlated with significantly worse overall survival in the CoMMpass cohort. qRT-PCR validation in patient samples confirmed significant overexpression of ERBB-related genes in PCL.
 Analysis of over 23,000 T/NK lineage cells identified 21 subtypes. In PCL, cytotoxic subsets such as CD8⁺ naive, CD56dim NK, and NKT cells were notably reduced or exhibited elevated exhaustion scores. NKT cells showed elevated IFN-response genes but downregulation of cytotoxic effectors, indicating a paradoxical “hyperactivated-yet-dysfunctional” state.
 CellChat analysis revealed extensive reprogramming of ligand-receptor interactions in PCL. Compared to ND and MM, PCL plasma cells engaged in enriched immunosuppressive and adhesive signaling via MIF, TGF-β, CD86, NOTCH, and APRIL axes. Enhanced communication with CD8⁺ T and NKT cells may promote chronic stimulation and immune exhaustion, supporting tumor immune escape and dissemination.
 Conclusion: This study presents the first high-resolution single-cell atlas of primary PCL, revealing that enhanced ERBB signaling, elevated CNV burden, and profound immune dysfunction collectively drive its aggressive clinical behavior. Malignant plasma cells in PCL not only acquire stem-like features and oncogenic pathway activation but also actively remodel the immune microenvironment via suppressive signaling, leading to T/NK cell exhaustion and impaired surveillance. These findings uncover actionable targets such as ERBB4/PI3K/MAPK and immune exhaustion pathways, offering a molecular framework for future precision therapy in PCL. Further functional validation and clinical translation of these discoveries are warranted.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2c5eed7a02f279167bbebc35b4990c64fcc8584" target='_blank'>
              Single-cell transcriptomics reveals erbb-driven clonal evolution and immune evasion in plasma cell leukemia
              </a>
            </td>
          <td>
            Tingting Lei, Luyao Shi, Xinli Han, Chi Li, Zongyang Li, Xiaotong Zhang, Kairan Li, Saisai Sun, Lu Yang, Chong Wang, Zhilei Bian, Shujuan Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Intracranial germ cell tumors (IGCTs) encompass several histologic subtypes, each characterized by distinct morphologies and varying therapeutic outcomes, yet their immune landscape remains poorly understood. To elucidate the immune microenvironment associated with these tumors, we first obtained four samples with paired spatial transcriptomics (Visium HD) and proteomics (CODEX), as well as two additional samples with only spatial transcriptomics data to study the immune microenvironment in different histologic regions. Our observations revealed a diverse immune pattern across IGCT tissue types. Notably, we observed the remarkable presence of tertiary lymphoid structure (TLS) within the IGCTs. Specifically, intratumoral TLS was most abundant and mature in teratoma, germinoma, and their respective components in mixed GCT, whereas yolk sac tumor (YST) exhibited no intratumoral TLS, with only a few peripheral TLS noted. To confirm and extend these findings in additional cases, we analyzed IGCT bulk RNA-seq data, including mature teratomas, immature teratomas (IMT), germinomas, YSTs, and mixed GCTs. Our analysis indicated that germinoma, mixed GCT, and teratoma exhibited both the highest TLS signature scores and the highest abundance of T cells and B cells, while YST and IMT had the least. Additionally, we noticed that the high abundance of B cells and T cells was driven by multiple clones rather than oligoclonal expansion. These findings provide valuable insights into the immune landscape of IGCTs, highlighting the potential for targeted immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/460b9c1e95b60feb342540a44365c08c4e3019a4" target='_blank'>
              TMIC-56. Spatial multi-omics data reveals immune landscape and tertiary lymphoid structures in intracranial germ cell tumors
              </a>
            </td>
          <td>
            Ran Liu, Dong Song, Jihong Tang, Yimeng Qiao, Huiyuan Chen, R. Chai, Jiguang Wang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is characterized by its highly aggressive phenotype and dismal outcome. Despite the benefit of adding immune checkpoint blockade to standard chemotherapy, tumors acquire the ability to evade immunosurveillance and develop resistance. To investigate these underlying mechanisms, we perform high-dimensional profiling of human and murine SCLC specimens. In matched primary and metastatic human samples, we observe MHC-I loss in metastases, highlighting its role in immune evasion. Correspondingly, silencing MHC-I in SCLC cells drastically reduces immune infiltration and promotes metastasis in mice. Using mass spectrometry and phospho-tyrosine kinase analyses, we identify ERBB2 signaling as a suppressor of MHC-I and driver of immune-modulatory transcripts. Mechanistically, genetic and pharmacologic blockade of ERBB2 induces MHC-I in a STING-dependent manner and prevents immune evasion in autochthonous murine SCLC. Strikingly, combining ERBB2 inhibition with anti-PD-1 elicits profound synergistic responses in preclinical models, suggesting this combination for future clinical trials in SCLC patients. While the emergence of immune checkpoint inhibitors has improved outcomes in patients with small cell lung cancer (SCLC), tumour that develop means of immune evasion become resistant. Here, the authors report that ERBB2 signalling induces loss of MHC Class I expression and subsequently immune evasion in preclinical models of SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f612388efd2f0a9c1ebc8d3e60e6b396ae989beb" target='_blank'>
              ERBB2 signaling drives immune cell evasion and resistance against immunotherapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Meder, C. Orschel, C. Bouchez, R. Gholamipoorfard, C. V. Orschel, David Stahl, Christoph Kreer, Mirjam Koker, Marieke Nill, Ilayda G Kocak, Ka-Won Noh, Xinlei Zhao, Leon Ullrich, B. Häupl, Josefine Jakob, Marie-Lisa Eich, Alexandra Florin, Holger Grüll, J. Wolf, Filippo Beleggia, R. Buettner, T. Oellerich, Florian Klein, J. Brägelmann, H. C. Reinhardt, Nima Abedpour, R. Ullrich
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/999c9f7dc52086f050a2aa67ed25e42b93fae763" target='_blank'>
              Single-cell RNA sequencing in osteosarcoma: applications in diagnosis, prognosis, and treatment
              </a>
            </td>
          <td>
            Christèle Asmar, Guy Awad, Marc Boutros, Simon Daccache, A. Chebly, C. Alix‐Panabières, H. Kourie
          </td>
          <td>2025-11-13</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related mortality worldwide, with tumor recurrence and metastasis posing significant challenges despite advances in targeted therapies and immunotherapy. Cellular dormancy, a reversible, quiescent state marked by cell cycle arrest, has emerged as a key driver of therapeutic resistance and disease relapse, particularly in small-cell lung cancer (SCLC) and non-small cell lung cancer (NSCLC). Multiple mechanisms, including autophagy, stress-adaptive signaling, microenvironmental cues, and epigenetic dysregulation, have been implicated in the regulation of dormancy and long-term cell survival. Among these, epigenetic modifications such as DNA methylation, histone modifications, and non-coding RNAs (ncRNAs) play pivotal roles in maintaining dormancy by repressing proliferative gene expression programs. Increasing evidence suggests that dormant tumor cells harbor distinct epigenomic signatures, which may serve as predictive biomarkers for minimal residual disease (MRD) and relapse risk. This review summarizes current advances in understanding the epigenetic regulation of cellular dormancy in lung cancer, with a particular emphasis on the interplay between epigenetic modifiers and oncogenic signaling pathways. Furthermore, emerging molecular targets and associated therapeutic agents currently under clinical evaluation are presented, emphasizing how a deeper understanding of the epigenetic landscape governing dormancy may inform the development of novel interventions to improve long-term clinical outcomes in lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f78e8931704f8d1c6b0d604cdcde3728bfe1b4a" target='_blank'>
              Unlocking Lung Cancer Cell Dormancy: An Epigenetic Perspective
              </a>
            </td>
          <td>
            F. Fabrizio
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 The introduction of bowel cancer screening has led to an increase in the detection and removal of pre-malignant adenomas, however many of these would never progress to colorectal cancer (CRC) within a patient’s lifetime. This implies a key evolutionary bottleneck at the transition from adenoma to carcinoma, with immune surveillance likely playing a central role in suppressing the outgrowth of invasive cells. Here we analysed a cohort of “ca-in-ads” (adenomas with a small focus of cancer) to provide a unique snapshot of the invasive transition. We combined genomics, spatial transcriptomics, digital pathology and multiplex imaging, concurrently deriving the distribution of tumour clones, their gene expression and their interplay with the immune microenvironment. We performed multi-region low-coverage whole genome sequencing (lcWGS) and T-cell receptor sequencing (using FUME-TCRseq, a novel method developed in our lab) on archival ca-in-ads from 40 patients. For a subset of cases we performed spatial transcriptomics using the 10x Genomics Visium platform, including custom probes to detect T-cell clonotypes of interest. We applied a deep learning cell classifier to haematoxylin and eosin (H&E) stained sections to determine the abundance and spatial distribution of eight cell types, with detailed immunophenotyping on selected cases (using a 50-marker panel on Akoya’s PhenoCycler-Fusion platform). We found a significant increase in copy number alterations in carcinoma regions relative to adenomas, and this was accompanied by a large shift in the composition of the local T-cell repertoire. TCR sequencing revealed expanded T-cell clonotypes that were adenoma-specific or carcinoma-specific, and we validated the spatial exclusivity of these using a custom Visium assay. Although overall lymphocyte density remained unchanged, digital pathology and cyclic immunofluorescence revealed significantly increased infiltration of macrophage and neutrophil populations in regions of invasive cancer. Taken together our analysis suggests a striking reshaping of tumour-immune dynamics at the transition to an invasive phenotype in CRC. Understanding what governs this invasive shift could highlight new avenues for cancer prevention or identify individuals at risk of CRC.



 Ann-Marie Baker, Eleanor Smith, Chirine Sakr, Konstantin Bräutigam, Nick Trahearn, Holly Eggington, Miangela M. Lacle, Manuel Rodriguez-Justo, Simon J. Leedham, Trevor A. Graham. Multi-omic spatial analysis reveals reshaping of tumour-immune dynamics at the transition to invasive colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr PR004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b4fa90cafae68daf434680b6837e68209ae59d6" target='_blank'>
              Abstract PR004: Multi-omic spatial analysis reveals reshaping of tumour-immune dynamics at the transition to invasive colorectal cancer
              </a>
            </td>
          <td>
            Ann-Marie Baker, Eleanor Smith, Chirine Sakr, Konstantin Bräutigam, N. Trahearn, Holly Eggington, M. Laclé, M. Rodriguez-Justo, Simon J. Leedham, Trevor A. Graham
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) and diffuse midline glioma (DMG) are two of the most aggressive brain tumors, marked by significant intratumoral heterogeneity and transcriptional plasticity, which drive resistance to therapy. These tumors share similar transcriptional programs, though the specific genes associated with these programs differ. Transitions between transcriptional states, particularly to the mesenchymal state, which is associated with greater resistance to treatment, complicate treatment strategies. A significant challenge in glioma research is the inability to dynamically track these state transitions in vitro and in vivo. To address this challenge, we introduce CRISPR-TAG reverse genetics, a novel approach that enables real-time monitoring of key transcriptional state markers in glioblastoma and diffuse midline glioma cell lines. Using CRISPR-TAG, we insert a fluorescent reporter sequence at the 3’ end of target genes, linking the fluorescent signal to endogenous gene expression. This allows us to dynamically track changes in gene expression as cells transition between states. We have successfully tagged 30 key cell state markers in patient-derived GBM and DMG cell lines, providing a robust model to investigate the upstream regulators of cell state transitions. By integrating CRISPR-TAG with global CRISPR screens and high-content pharmacological screens, we uncover genetic dependencies that underlie glioma plasticity. For example, we identify modulators of the mesenchymal cell state, including ANXA1 expression, which serves as a critical marker of this resistant state. Additionally, we demonstrate how our method allows in vivo tracking and modulation of cell state transitions in explant cultures, with AI-assisted analysis providing detailed insights into these transitions. In conclusion, CRISPR-TAG reverse genetics lays a critical foundation for building a high-resolution map of glioma state regulators, paving the way for targeted therapeutic strategies. By enabling the precise modulation of cell states, this method offers the potential to complement and enhance existing treatment modalities for GBM and DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eaf38521e14ac09d468437f1dbddc6b4207f994" target='_blank'>
              CNSC-29. RESOLVING THE REGULATION OF GLIOBLASTOMA CELL STATES BY CRISPR-TAG REVERSE GENETICS
              </a>
            </td>
          <td>
            Josephine Heinold, Madeleine Skeppås, Filippa Kruse, Clara Nordquist, Irem Uppman, Sven Nelander
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) serve as a promising biomarker for tracking cancer metastasis, progression, and recurrence. Liquid biopsy techniques centered on CTC detection have demonstrated considerable potential due to their non-invasive nature and ability to provide real-time monitoring of tumor dynamics. However, conventional bulk CTC analyses fail to capture the intrinsic heterogeneity among CTC populations, obscuring crucial insights into tumor biology. Single-cell RNA sequencing (scRNA-seq) enables high-resolution characterization of CTC heterogeneity, offering new opportunities for precision oncology and mechanistic studies of tumor progression. Despite these advantages, the existing methodologies for single-CTC sequencing tend to suffer from inefficiencies, including low recovery rates, labor-intensive workflows, and contamination risks associated with multiple manual handling steps. To address these limitations, we present an integrated microfluidic protocol that consolidates CTC enrichment, purification, and single-cell sequencing into a unified workflow. The method employs dynamically controlled magnetic capture within a herringbone-structured chip, where vortex mixing and cumulative immunomagnetic bead binding enable robust, high-throughput CTC isolation with minimal cell damage. Subsequent purification using a leukocyte antibody-coated microfluidic chip effectively removes the non-target cells, further enhancing CTC purity through negative selection. Finally, a high-precision single-cell sequencing chip, designed based on differential flow resistance principles, facilitates efficient single-cell capture and pairing with uniquely barcoded microbeads. This novel platform overcomes the limitations of Poisson distribution-based methods, improving CTC utilization while minimizing microbead consumption and sequencing costs. Our integrated protocol significantly enhances CTC capture efficiency, purity, and single-cell sequencing throughput, making it well-suited for clinical applications and large-scale cancer research. By enabling a more precise and scalable analysis of CTC heterogeneity, this method has the potential to refine early cancer diagnosis, treatment monitoring, and mechanistic studies of metastasis, ultimately advancing the field of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9babafe8a10523678f47d2a96e859b16fafc3f98" target='_blank'>
              Microfluidics-based High-throughput Circulating Tumor Cell Sorting and Single-cell Sequencing Technology.
              </a>
            </td>
          <td>
            Zheyu Liu, Ye Tian, Zian Zhang, Lingling Wu, Liang Dong, Wei Xue
          </td>
          <td>2025-11-14</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background The most aggressive type of skin cancer, melanoma, has a high prevalence of metastases and a poor prognosis. Despite advancements in treatment, drug resistance and tumor microenvironment heterogeneity, especially involving neuro-immune interactions, continue to exist. The goal of this study is to uncover the cellular heterogeneity of melanoma in order to pinpoint molecular targets and tumor-promoting subtypes. Methods Melanoma single-cell RNA sequencing data came from GEO. Twelve cell types were discovered after Harmony batch effect adjustment and Seurat quality control; melanoma cells were subtyped. Functional expression of differential genes was examined using gene ontology and gene set enrichment. Cytotrace measured subtype differentiation potency. PySCENIC revealed transcription factor regulatory networks, and CellChat predicted intercellular communication between malignant cell subtypes and other cell types. Functional experiments with A375 and MEWo cell lines—lentiviral shRNA knockdown, CCK-8 proliferation, wound healing, transwell migration, and flow cytometry apoptotic assays—confirmed the analysis. Results Single-cell RNA sequencing was utilized to analyze melanoma cell subtypes and their interactions in the tumor microenvironment. C2 IGFBP3+, which had the lowest CytoTRACE2 score and was enriched in late tumor stages, affected melanoma development. This subtype expressed prominent immunomodulatory pathways. The C2 subtype mediate signaling to fibroblasts/pericytes via the PROS pathway and myeloid/plasmacytoid dendritic cells via the MHC-II pathway. The C2 subtype was strongly linked with FOSL1 expression, which may affect gene transcription and illness progression. FOSL1 knockdown significantly increased apoptosis and decreased melanoma cell motility and proliferation in vitro. Conclusion We identified immunoregulatory C2 IGFBP3+ melanoma cell subtypes in our investigation, and FOSL1 was a crucial transcription factor that aided in cell migration, proliferation, and survival. Because the C2 subtype involves the MHC-II and PROS signaling pathways, which have been shown to have roles in neuroimmunology, neuroinflammation, and pain regulation, it may serve as a hub for neuro-immune interactions in the tumor microenvironment. Precision treatments for melanoma may be advanced by focusing on the FOSL1 or C2 subtype pathways, which may assist in overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34e116063cb6903b41d6e715ba8448ec0e1c090" target='_blank'>
              Single-cell transcriptomics identifies FOSL1-regulated IGFBP3+ melanoma subtype as a neuro-immunoregulatory signaling hub facilitating tumor progression
              </a>
            </td>
          <td>
            Wenjia Ge, Ziwei Zhang, Wenjie Chen, Zhijie Zhao, Hua-Bao Cai, Yantao Ding, Jin Xu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Thyroid cancer is the most common endocrine malignancy, with an increasing incidence, particularly driven by differentiated thyroid cancers (DTC), including papillary thyroid carcinoma (PTC) and follicular thyroid carcinoma (FTC). Although DTC generally have excellent prognosis, dedifferentiated forms such as poorly differentiated thyroid cancer (PDTC) and anaplastic thyroid cancer (ATC) exhibit aggressive progression and poor outcomes. Understanding the tumor microenvironment (TME) is crucial for developing novel therapeutic strategies. We performed spatial transcriptomics (ST) on formalin-fixed paraffin-embedded (FFPE) tumor tissues from 12 patients with PTC, FTC, PDTC, or ATC using the Visium CytAssist platform. Cell type composition was inferred using the CellDART algorithm, and pathway activity was assessed via PROGENy and REACTOME analyses. Immunohistochemical (IHC) assays were conducted to validate key TME components. ST analysis revealed significant shifts in cell composition associated with thyroid cancer dedifferentiation. Specifically, there was an increase in myeloid cells and cancer-associated fibroblasts (CAFs), accompanied by a reduction in natural killer (NK) cells and endothelial cells. Intratumoral heterogeneity in the Thyroid Differentiation Score (TDS) showed a strong spatial correlation with myeloid cell density, and among myeloid subtypes, the MDSC score was negatively correlated with TDS. Pathway activity analysis further identified upregulation of the JAK-STAT and VEGF signaling pathways in dedifferentiated tumors, both of which were closely associated with increased myeloid cell infiltration. IHC validation confirmed the differential expression patterns of representative immune and stromal markers across subtypes. These findings suggest that dedifferentiation in thyroid cancer is associated with an immunosuppressive TME that promotes tumor progression. Therapeutic strategies targeting JAK-STAT and VEGF pathways, or enhancing NK cell activity, may hold promises for aggressive thyroid cancers. Further studies with larger cohorts are necessary to validate these findings and develop targeted therapies for aggressive thyroid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c190cffa58062152ca24006b8fcd82eeec4ea2bc" target='_blank'>
              Tumor microenvironment remodeling across thyroid cancer differentiation states revealed by spatial transcriptomics
              </a>
            </td>
          <td>
            Jungirl Seok, Hongyoon Choi, Eun-Kyung Lee, Chang Hwan Ryu, Dongjoo Lee, J. Ryu, Seog Yun Park, Yuh-Seog Jung
          </td>
          <td>2025-11-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Small-cell lung carcinoma (SCLC) remains one of the most aggressive lung cancers and continues to pose a major challenge for precision oncology. Despite its morphological uniformity, SCLC exhibits marked molecular heterogeneity with recurrent, potentially targetable genomic alterations. Comprehensive profiling is often hindered by limited tissue availability and the need for rapid therapeutic intervention. We performed genomic profiling of 55 primary and metastatic SCLC samples using a 324-gene hybrid-capture next-generation sequencing panel. Consistent with prior reports, nearly all tumors exhibited biallelic TP53 and RB1 inactivation. Recurrent alterations involved the PI3K/Akt/mTOR pathway (62%), chromatin regulators (42%), and NOTCH signaling genes (15%). PTEN mutations were enriched in brain metastases. Frequent copy-number gains affected SOX2, NKX2-1, MYC-family genes, and CCNE1. Two novel recurrent amplifications of potential clinical significance were identified: TYRO3 (33%) and SDHA (13%). TYRO3, a TAM family receptor tyrosine kinase, and SDHA, a mitochondrial enzyme involved in succinate metabolism, may contribute to tumor progression and represent emerging therapeutic vulnerabilities. These findings underscore the genomic diversity of SCLC and highlight the potential utility of broad next-generation sequencing in uncovering new molecular targets for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c257de36865932aa9b4ccc1af139b71a5ba02b7" target='_blank'>
              Comprehensive Genomic Profiling of Small-Cell Lung Cancer Reveals Frequent Potentially Targetable Alterations
              </a>
            </td>
          <td>
            Dániel Schmalz, Zoltán Krabóth, Veronika Czoma, Péter Urbán, Attila Gyenesei, István Ruzsics, Veronika Sárosi, Á. Boronkai, Emőke Papp, B. Kajtár
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 The tumor immune microenvironment (TiME) accommodates significant transcriptomic and proteomic heterogeneity, and understanding this heterogeneity is key to the development of cancer treatments. Most analytical approaches provide only one type of biological readout and/or lack spatial information needed to decipher TiME complexity. We developed a new spatial multi-omics toolset to comprehensively study the TiME transcriptomic and proteomic readouts with single cell resolution. We used the CellScape™ Precise Spatial Proteomics platform with enhanced photobleaching in cyclic immunofluorescence (EpicIF) technology to detect 12+ RNA targets via HCR™ Gold RNA-FISH and 24+ protein markers via multiplex immunofluorescence (mIF) on the same formalin-fixed paraffin-embedded (FFPE) tissue sample. RNA-FISH target probes were added to the tissue sample after antigen retrieval, and tissues were then mounted in CellScape fluidic chambers to enable on-instrument staining, imaging, and signal removal. The RNA-FISH assay was followed by a 24-plex mIF assay containing antibodies against common immune-oncology biomarkers. Multiomic data from human tonsil, colon and breast tissues showed high conformity between RNA-FISH and mIF assays on the CellScape platform. The combined assay revealed differential biomarker expression profiles associated with immune-regulatory processes in the tumor samples and provided detailed insight into immune and inflammatory processes of the TiME.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5daaa251cd6bb773ad5618931f13decbb4ef34a" target='_blank'>
              Spatial multi-omic analysis of the tumor microenvironment using high-plex HCR RNA and proteomic immune profiling on the CellScape platform 4057
              </a>
            </td>
          <td>
            Matthew H. Ingalls, J. Boog, Daniel Jimenez Sanchez, Adam Maddox, Aneesh Acharya, Cameron Earl, Harry Choi, A. Christians, O. Braubach
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de84f1d76b8dc759a28178cb80d35ff586603711" target='_blank'>
              Glioblastoma gene expression based subtypes have defined metabolomic states
              </a>
            </td>
          <td>
            Hélèna L. Denis, Jaëlle Méroné, Akhil Shukla, V. Watters, V. Fort, Gabriel Khelifi, Mikalie Lavoie, Line Berthiaume, Félix Rondeau, Émy Beaumont, Karine Michaud, M. D'Astous, S. Saikali, Marc-Étienne Huot, Étienne Audet-Walsh, Maxime Richer, Samer M. I. Hussein
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Introduction: Large B-cell lymphomas (LBCL) exhibit intra-patient biologic heterogeneity. Single-site biopsies limit comprehensive assessment of this complexity, which circulating tumor DNA (ctDNA) overcomes by detecting mutations from distinct anatomical sites. We have previously shown anatomical genomic heterogeneity can be quantified by integrating baseline ctDNA and tissue profiles (Goldstein, ASH 2024). Though aberrant somatic hypermutation (aSHM) drives lymphomagenesis, its contribution to anatomical heterogeneity, clonal evolution, and outcomes is poorly understood. We validate the clinical value of anatomical genomic heterogeneity in diverse aggressive lymphomas, delineate its relation to aSHM, and then propose a DLBCL clonal evolution model.
 Methods: We studied 588 specimens from 196 patients across 3 cohorts with comprehensive genotyping of baseline plasma and matched tumor tissues. The Discovery Cohort included 66 LBCL patients from a multinational cohort receiving front-line chemoimmunotherapy with pre-treatment tumor, germline and plasma genotyping by CAPP-Seq (Kurtz, JCO 2018). We developed the Metric Of Spatial Anatomic Intra-tumoral genomic Complexity (MOSAIC) as the root mean square error of a linear regression model of mutant allele frequencies (log[VAF]) in ctDNA versus those shared in tissue.
 After defining a MOSAIC threshold in our Discovery Cohort, two validation cohorts with pre-treatment tumor, germline and plasma samples and treated with front-line chemoimmunotherapy were used: 97 LBCL patients from the HOVON-902 study profiled by PhasED-seq (Wang, ASCO 2025) and 33 MCL patients from the LyMa trial profiled by CAPP-Seq (Tessoulin, 18-ICML).
 To elucidate clonal evolution, we explored the molecular associations and relationship of aberrant somatic hypermutation (aSHM) with anatomic heterogeneity and outcomes in LBCL. We defined aSHM rate as the fraction of ctDNA mutations occurring within canonical aSHM targets (Schmitz, NEJM 2017). We measured clonal architecture (sciclone) and molecular subtypes (LymphGen) and performed survival analysis (Kaplan-Meier; Cox models).
 Results: In our Discovery Cohort, 100% had SNVs unique to plasma, with median 15% of plasma mutations absent in paired tissue. The median MOSAIC was 0.235 (threshold for validation) with range 0.1-0.4. Cases with >1 subclone in plasma had higher MOSAIC (median 0.26 vs 0.19,p=0.002). MOSAIC did not correlate with age, COO, histological subtype, Stage, or IPI. However, high MOSAIC was associated with worse PFS (HR=4.3,p=0.01) and OS (HR=12.7,p=0.02). MOSAIC did not correlate with ctDNA burden and remained prognostic independent of ctDNA levels.
 In the MCL LyMa Validation Cohort, median MOSAIC was 0.28 (IQR 0.13-0.39). MOSAIC correlated with ctDNA burden (p=0.05) and LDH (p=0.01), but not with leukemic status, blastoid morphology, or Ki67. High MOSAIC predicted worse PFS (HR 13.6,p=0.01), independent of LDH and ctDNA level.
 In the LBCL HOVON-902 Validation Cohort, median MOSAIC was 0.21 (IQR 0.17-0.28) with 38% classified as high. BN2, EZB & A53 tumors by LymphGen had higher MOSAIC (median 0.26), while MCD had lower scores (median 0.18). High MOSAIC was again associated with inferior PFS (HR=2.3,p=0.04) and OS (HR=3.4,p=0.02) and higher end of treatment MRD+ rates (28% vs 12%).
 TP53 and P2RY8 mutations were associated with high MOSAIC patients. GCB tumors showed numerically higher aSHM rates, though no significant differences in aSHM rates by histology or stage were found. Lower aSHM rate correlated with higher MOSAIC (r= -0.4,p=0.001) and worse PFS (HR 4.4,p=0.02). Patients with both high MOSAIC and low aSHM rate (38%) had 5-year PFS 43% (HR 5.6,p=0.001) and OS 57% (HR 9.8,p=0.003), vs 87 and 93% respectively. In 80%of patients, median plasma VAF for aSHM exceeded non-aSHM mutations, indicating aSHM occurs early in lymphomagenesis. High MOSAIC patients had higher ratios of median aSHM to non-aSHM VAFs, consistent with ongoing subclonal evolution via non-aSHM mutations.
 Conclusions: We describe and validate a novel biomarker measuring anatomical genomic heterogeneity, integrating plasma and tissue profiles, that independently predicts prognosis for front-line chemoimmunotherapy. We show that aSHM aberrations occur early in clonal evolution and likely initiate lymphomagenesis. Low aSHM rates and high anatomical heterogeneity in B-cell lymphomas define a high-risk subgroup prone to treatment resistance via anatomical clonal divergence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166a4d4cafdabd9dbc2d4b4b470b779c06222e42" target='_blank'>
              Spatial anatomical genomic heterogeneity and aberrant somatic hypermutation define clonal evolution pathways that predict treatment resistance in aggressive B-cell lymphomas
              </a>
            </td>
          <td>
            J. Goldstein, Steven Wang, B. Tessoulin, C. Liu, F. Scherer, S. Le Gouill, Martine E D Chamuleau, M. Nijland, Maximilian Diehn, David Kurtz, A. Alizadeh
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="
 The H3K27M mutation inhibits PRC2 activity resulting in a global hypomethylation and concomitant increase in acetylation of H3K27 wildtype histones. Such epigenetic alterations force diffuse midline glioma (DMG) cells to retain a highly proliferative stem-like cell state. However, single cell transcriptomics revealed that some cancer cells, despite harboring the H3K27M mutation, can exit the cell cycle and acquire features of mature glial-like cells, providing rationale for differentiation therapies. Here, we sought to dissect the epigenetic landscape of these distinct cancer cell states within DMG to better understand how tumor cell identity might be modulated for therapeutic benefit. To do so, we profiled the histone posttranslational modifications landscape and genome accessibility in both stem- as well as glial-like DMG cells. We observed that the more mature glial-like cells overturn the oncogenic activity of the histone mutation on H3K27 marks, and consequently carry higher levels of the repressive histone mark H3K27me3 and lower levels of the activating mark H3K27ac. Next, we questioned whether H3K27 acetylation loss would reciprocally drive DMG cell differentiation. Intriguingly, our epigenetically-focused CRISPR screen identified the histone acetyltransferases responsible for acetylating H3K27, EP300/CBP, as essential genes for the continuous proliferation of stem-like DMG cells. In agreement, DMG cells are selectively sensitive to EP300/CBP inhibition or degradation, unlike human astrocytes and H3G34R/V high-grade glioma cells. Interestingly, targeting EP300/CBP reverts the epigenetic reprograming caused by H3K27M by not only depleting H3K27ac but also rescuing H3K27me3 levels. Finally, ATAC and single cell RNA sequencing revealed that the epigenetic changes caused by EP300/CBP targeting shift DMG stem-like cells towards more differentiated glial-like cell states. In summary, we show that distinct DMG transcriptomic cell states reflect underlying epigenetic heterogeneity which can be exploited for treatment. In particular, we demonstrate that EP300/CBP are promising therapeutic targets for differentiation therapy in this universally fatal glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38acecf50f0c665a725bfac7f8d20ecf3c8a39a3" target='_blank'>
              EPCO-64. EXPLOITING EPIGENETIC CELL STATE HETEROGENEITY TOWARDS DIFFERENTIATION THERAPY IN H3K27M-ALTERED DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            Joana G. Marques, Matthew Mullaly, C. B. Junior, C. Cascio, Andrezza Do Nascimento Silva, Qi Liu, Pruthvi Gowda, N. Danial, Jun Qi, M. Filbin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 Diffuse midline glioma (DMG) is an incurable pediatric brain tumor, with radiotherapy offering only transient benefit before inevitable recurrence. While recent studies have revealed the cellular complexity of DMG, the relationship between intratumoral heterogeneity and disease progression — particularly under therapeutic pressure — remains poorly understood. In this study, we integrate longitudinal single-cell- and spatial transcriptomics with single-cell chromatin accessibility profiling and radiomics to investigate how distinct DMG cell states and microenvironmental components evolve in response to treatment, contributing to resistance and the emergence of therapy-persistent niches. We applied this multi-omic approach to a unique cohort of 10 matched diagnostic and autopsy DMG samples obtained from children enrolled in the PNOC023 clinical trial. Spatial transcriptomic profiling revealed that post-treatment autopsy tissues showed consistent enrichment of oligodendrocyte progenitor cell-like malignant populations, suggesting that this stem-like subpopulation is selectively retained or expanded following radiotherapy in DMG. Moreover, integration of single-cell RNA and ATAC sequencing with cerebrospinal fluid microRNA analyses further revealed molecular signatures of resistance and uncovered minimally invasive biomarkers of treatment response. To complement our molecular analyses, we applied radiomic profiling of longitudinal MRI scans to non-invasively monitor changes in tumor heterogeneity over the course of treatment in this patient cohort. In parallel, we developed a data-driven mathematical model using patient-derived DMG xenografts to simulate tumor cell responses to radiotherapy and generate quantitative predictions of preclinical treatment outcomes. We employed this model to explore alternative radiation fractionation schemes and identify optimal dosing strategies that most effectively deplete therapy-resistant tumor cell populations in DMG. Together, these integrated spatial and molecular datasets provide a high-resolution framework for understanding how radiotherapy remodels the DMG tumor ecosystem. By identifying persistent cellular populations and spatial niches that survive treatment, this work lays the groundwork for future precision therapies aimed at targeting the cellular reservoirs that drive DMG recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4395d6f286b7b33609966b9d19ccc4d41894fd90" target='_blank'>
              RBIO-16. Mapping the Spatiotemporal Evolution of Diffuse Midline Glioma Under Radiotherapy
              </a>
            </td>
          <td>
            Costanza Lo Cascio, Shashank Katiyar, Simone Bruno, Carlos A O De Biagi Junior, Danielle Bitterman, Joana G. Marques, Sina Neyazi, Sharmistha Pal, David Kozono, Benjamin H. Kann, Charles Stiles, Sabine Mueller, Franziska Michor, D. Haas-Kogan, M. Filbin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a highly malignant pediatric brain tumor of the cerebellum characterized by extensive molecular, biological, and clinical heterogeneity. Current genetically engineered mouse models have only partially replicated this diversity, and harbor species-specific limitations owing to fundamental differences between mouse and human development. Therapeutic strategies that accommodate MB heterogeneity are required to improve patient outcomes but necessitate biologically faithful models that are predominantly lacking. To this end, we have established human pluripotent stem cell-derived cerebellar organoids that recapitulate early human cerebellar development and enable deep mechanistic studies into MB oncogenesis. Transcriptomic profiling and immunophenotyping of our organoids throughout developmental timepoints revealed cellular compositions and differentiation trajectories reminiscent of cerebellar development in vivo. As our cerebellar organoids faithfully reproduce putative lineages of origin for distinct MB subgroups, we’ve endogenously Cre-tagged stem cell lines and generated various expression constructs to manipulate known driver genes in specific cellular compartments to generate accurate human cerebellar organoid-based disease models. We furthered the utility of this ex vivo human system by establishing a cerebellar organoid/patient-derived xenograft (PDX) co-culture platform that fosters the sustainability of MB PDXs in culture. Upon co-culturing with cerebellar organoids, we observed sustained viability and proliferation of MB PDXs, both of which are unprecedented in standard in vitro cultures. Further, we observed differential electrophysiological properties in PDX-harboring organoids compared to standalone organoids and identified putative signaling pathways crucial for cellular crosstalk between tumor and their supporting microenvironment. Our ongoing multi-faceted MB modeling strategies leveraging cerebellar organoids engender new experimental opportunities otherwise incompatible with pre-existing approaches, providing an efficient platform for advancing disease biology and identifying therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9c6a6fee26a49ce1740bbf42bd5633cf13397a" target='_blank'>
              TMOD-13. Human pluripotent stem cell-derived cerebellar organoids as a robust platform for studying medulloblastoma pathogenesis
              </a>
            </td>
          <td>
            Stephanie C Wu, Rachel Xu, Yiran Li, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbb0b650183e1ab55b2143b3779619fab2e24dd" target='_blank'>
              aiAtlas: High-Fidelity Cell Simulations of Genetic Perturbations in Rare Diseases and Cancers
              </a>
            </td>
          <td>
            WR Danter
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Renal cell carcinoma (RCC) is responsive to immunotherapy despite having a low tumor mutational burden. Interestingly, the degree of CD8+ T cell infiltration in RCC negatively correlates with outcomes, suggesting that a deeper understanding of the tumor microenvironment may provide critical insight to help improve clinical outcomes.



 We conducted a multi-omics approach including flow cytometry, multiplex immunohistochemistry, and single-cell RNA-seq to compare immune cell composition, gene expression, spatial distribution, and cell-cell interactions among a cohort of primary breast cancer, head and neck squamous cell carcinoma, non-small cell lung cancer, and RCC specimens.



 Flow cytometry analysis of tumor-infiltrating lymphocytes (TILs) revealed increased exhausted CD8 + (Tex) and CD4+CD8+ double positive (DP) T cells and decreased CD4+ T cells including regulatory T (Treg) cells in RCC samples compared to other tumor types. scRNA-seq analysis of TILs confirmed these results and showed Tex cells as a major population of CD8+ T cells. Using biomarkers for tumor-reactive cells including 4-1BB, CD39, and CD103 along with a gene signature to identify neo-antigen reactive T cells, we found that the Tex population contained putative tumor-reactive T cells and were not merely bystander cells.



 Exhausted CD8+ and DP T cells in RCC contain tumor-reactive T cells and may represent a novel cell population capable of targeting RCC.



 Funding: Funding was provided by a sponsored research grant from GlaxoSmithKline, the Providence Portland Medical Foundation, and the M. J. Murdock Charitable Trust.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3623ddf9190f1765df9cf05747187e1bf4ab0a38" target='_blank'>
              Multi-cancer comparative multi-omics approach reveals a permissive tumor microenvironment in renal cell carcinoma through accumulation of exhausted CD8+ and CD4+CD8+ double positive T cells 2661
              </a>
            </td>
          <td>
            M. Janghorban, Y. Koguchi, Noah D Simons, Wesley Rosales, V. Rajamanickam, Tanisha L. Christie, William L. Miller, Iliana Gonzalez, Zhaoyu Sun, Yaping Wu, Lakhvir Kaur, Brendan D. Curti, N. Yanamandra, Johanna Kaufmann, B. Bernard, Thomas Duhen, Eric Tran, William L Redmond
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61684838fc3c53a5cd2329af114682e03471ba75" target='_blank'>
              Single-Cell Mendelian Randomization Identifies Cell-Type Specific Genetic Drivers of Lung Cancer Subtypes
              </a>
            </td>
          <td>
            Q. Yang, C. Yan, X.-f. Wang
          </td>
          <td>2025-12-02</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast Cancer is one of the most prevalent malignancies and the cause of mortality among women. Despite technological advancements, diseases have inherent heterogeneity, especially in the tumor microenvironment (TME), which poses a significant challenge for effective therapy. The study explores the spatial heterogeneity of two key molecular markers, EPCAM (Epithelial Cell Adhesion Molecule) and ERBB2 (HER2).  The research was conducted based on spatial transcriptomics and a graph-based clustering method. EPCAM plays a crucial role in maintaining epithelial integrity, and its dysregulation facilitates tumor invasion and metastasis. On the other hand, ERBB2 is a well-characterized oncogene that drives proliferation in HER2-positive breast cancer. The study leveraged 10x Genomics Visium data and a high-resolution histopathology image to analyze gene expression and spatially variable genes (SVGs) within the tissue. The study identified eight functional spatial domains. ERBB2 expression was localized to tumor cores, which reflects the role as an oncogenic driver. On the other hand, EPCAM exhibited a wider distribution, highlighting the involvement in tumor-stroma interactions and epithelial-mesenchymal transition (EMT). The findings demonstrate the utility of spatial transcriptomics in two aspects. The first is resolving tumor heterogeneity, and the second is to underscore the importance of spatial context in understanding the roles of EPCAM and ERBB2 for cancer progression. The insights contribute to the development of precision medicine approaches that target a specific tumor region for improving the therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b117f399cc68a98eef4707a09f6308f49ac08645" target='_blank'>
              Spatial Heterogeneity of EPCAM and ERBB2 Expression in Breast Cancer: Insights from Spatial Transcriptomics
              </a>
            </td>
          <td>
            Muhammad Afrizal Amrustian, Bita Parga Zen, Aminatus Sa'adah
          </td>
          <td>2025-10-27</td>
          <td>International Journal on Advanced Science, Engineering and Information Technology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is the most aggressive primary brain tumor in adults, marked by rapid progression and poor prognosis. GBM may also harbor a complex virome, defined here as the total repertoire of viral elements within tumors, encompassing both exogenous viruses and reactivated endogenous retroviruses. While viral signatures have been observed in other cancers, their transcriptional activity in GBM and their distribution across tumor cells and extracellular compartments remain largely unexplored. Methods We analyzed 45 long- and small-RNA libraries from glioma stem-like cells, their extracellular fractions (exosomes, microvesicles, ribonucleoprotein complexes), and normal brain controls (BioProject PRJNA360129). Taxonomic profiling and functional annotation were performed using established pipelines (MetaPhlAn4, BLASTn, UniProtKB, InterPro), with fresh media controls to exclude background contamination. To validate discriminative power, viral RNA profiles were classified using machine learning -Random Forest models trained on nine species identified as glioblastoma-enriched by LEfSe. Results We detected ~3.41 million viral reads, with >99% from GBM cells and extracellular compartments and <1% from normal brain. Reads spanned 36 species across 11 phyla. Nine species were consistently enriched in glioblastoma: Saimiriine gammaherpesvirus 2, Human endogenous retrovirus K, Camelpox virus, Pestivirus A, Macacine gammaherpesvirus 5, Avian myelocytomatosis virus, Rabbit fibroma virus, Omegapapillomavirus 1, and Cyprinid herpesvirus 2 with genes mapping to oncogenic pathways. Random Forest models showed long-RNA profiles perfectly discriminated against tumor from normal samples (AUC = 1.0), whereas small-RNA profiles carried limited signal (AUC = 0.5). Tumor–extracellular comparisons revealed strong compartment-specific long-RNA signatures (AUC = 1.0), with partial overlap between tumor cells and exosomes in small-RNA data (AUC = 0.75). Conclusion Glioblastoma contains a distinct viral transcriptome; long-RNA profiles show strong diagnostic power, and compartment-specific signatures suggest biomarker potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f03aa5d91d0d223768b844afb632280b82ecb5f9" target='_blank'>
              The glioblastoma virome across tumor and extracellular compartments
              </a>
            </td>
          <td>
            M. Ghorbani, H. Karimi, Farnoosh Solati, Nazanin Anbarestani, Nooshin Khoshdoozmasouleh
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most common and deadliest primary brain tumor, with no effective therapies and a longstanding record of failed phase 3 clinical trials—often without mechanistic insight into why promising approaches fall short. To address this critical gap, we implemented a novel on-therapy, longitudinal, and multi-regional tissue sampling strategy in patients treated with rQNestin34.5v2, an engineered oncolytic biologic based on herpes simplex virus 1, to explore how tumor and immune ecosystems evolve during treatment. Using high-resolution spatial transcriptomics (Xenium) and targeted spatial proteomics (CyCIF), we profiled over 80 spatially and temporally distinct samples from six patients, capturing over half a million immune, stromal, and malignant cells. This unique dataset enabled unprecedented resolution of the dynamic interactions between tumor and immune compartments in situ over the course of therapy. Our preliminary analysis revealed discrete, virus-associated lymphoid-like structures that formed early after treatment initiation. These structures were enriched in T cells, B cells, and myeloid cells, and were associated with localized reductions in tumor cell density. Over time, we observed a shift from an initially inflamed microenvironment to a more immunosuppressive state. By the time of radiographic progression, these lymphoid structures had largely dissipated—underscoring the importance of on-treatment sampling in capturing transient, treatment-responsive immune architectures. While these findings are preliminary, they suggest a transient immune structure linked to early therapeutic response, whose collapse may contribute to treatment resistance. Our study illustrates how integrative, on-therapy spatial profiling can begin to reveal mechanisms of response and resistance in GBM, providing a framework for refining future immunotherapeutic strategies in brain tumors and other treatment-refractory cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf60db76ebc26534935de49d68b0018aa310569" target='_blank'>
              IMMU-67. On-Treatment Spatiotemporal Profiling of Glioblastoma During Oncolytic ImmunoTherapy
              </a>
            </td>
          <td>
            Charles Couturier, Jennifer Gantchev, Gerard Baquer, Peter Zhang, Alicia D’Souza, Michael S Regan, Kaitlyn Bootz, J. Bonney, Md Amin Hossain, G. Teamlab, Atul Deshpande, Forest M. White, Kenny K. H. Yu, E. Chiocca, Nathalie Y R Agar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Clonal hematopoiesis of indeterminate potential (CHIP) is a precursor state characterized by expansion of blood clones carrying somatic mutations that originate in the hematopoietic stem cells (HSCs). Individuals with CHIP have a 10-fold increased risk of developing myeloid malignancies such as myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). Progression of CHIP to MDS/AML takes several decades and hence targeting the CHIP clones during this period provides a viable strategy to prevent MDS/AML. Despite its importance, we still don't understand the mechanisms of CHIP predisposition and its progression. Recent genome wide association studies (GWAS) have identified several non-coding genetic loci that are strongly associated with CHIP, but the causal variants remain unknown because most of the variants are found in non-coding genomic regions. We have previously shown thatmodulation of gene regulatory elements in HSCsunderlies genetic risk of myeloid malignancies; whether this is also true for premalignant state such as CHIP is currently unknown.
 We hypothesize that germline CHIP risk variants in non-coding loci affect enhancer elements in HSCs. In order to identify these variants we utilized a high throughput approach termed Massively Parallel Reporter Assay (MPRA) to simultaneously screen for regulatory effects of 1,374 non-coding variants within 24 GWAS loci from a comprehensive study of CHIP risk on ~600,000 individuals. In the MPRA approach, we synthesized pools of barcoded lentiviral constructs harboring non-coding variants (risk and non-risk alleles) and surrounding genomic sequence (250bp) upstream of a minimal promoter driving a reporter gene. Sequencing of the plasmid pool confirmed an uniform distribution of barcodes with a median of around 50 barcodes per variant. Since enhancers show cell-type specific activity, we next searched for the appropriate cell-type to perform the MPRA. As the disease relevant population in CHIP is the HSCs, which is rare, we searched for alternative cell types. We identified MUTZ-3, a AML cell line that maintains a self-renewing CD34+ population and is dependent on HSC transcription factors. We confirmed that MUTZ-3 cells have trans-acting factors similar to primary HSPCs, by testing several validated HSPC enhancers including PU.1 upstream regulatory element (URE) and GATA2 inv(3) enhancer using individual reporter assays.
 We performed our lentiviral MPRA screen using a library of ~73,000 constructs in CD34+ fraction of MUTZ-3 cells. 48 hours post-infection we lysed the cells for DNA and RNA and sequenced the barcodes by NextSeq. The experiments were highly reproducible and all 5 replicates correlated well with each other at both the DNA and RNA barcode level. When we measure the enhancer activity by ratio of RNA to DNA barcode read counts, positive controls in the pool (PU.1 URE, GATA2 enhancer) showed higher activity compared to negative ORF controls. The activity of test constructs that overlap with hematopoietic ATAC peaks was higher that non-overlapping constructs indicating MPRA can identify endogenous enhancers. We identified several hits that showed allele specific reporter activity based on CHIP variant (Ref and Alt allele).
 We prioritized rs17834140, a variant identified by the MPRA for further validation studies. The variant is located within the 4th intron of the MSI2 gene in an hematopoietic ATAC peak and MPRA and individual reporter assays indicate enhancer activity and the risk allele increases MSI2 expression. We further confirmed endogenous enhancer activity by both CRISPR/Cas9 deletions and CRISPR inactivation experiments in human hematopoietic cells. To test whether MSI2 overexpression can also contribute to clonal expansion of CHIP mutant HSCs, we infected LSK cells from TET2 knockout mice and wildtype controls with the MSI2 overexpressing and control lentiviral vectors and observed an increased in vitro serial colony replating ability of TET2 cells upon MSI2 upregulation.
 Our study paves the way to systematically assess non-coding risk variants associated with CHIP risk. We expect the MPRA approach will help to draw out key unknown myeloid neoplasm predisposition genes and their regulatory elements that are influenced by the germline variants.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/691b91de07b0de85cfcd28321c80391149dbd376" target='_blank'>
              Assessing germline noncoding risk variants impacting clonal hematopoiesis using a massively parallel reporter assay
              </a>
            </td>
          <td>
            Jessica Jeejan, Trieu Nguyen, S. Kales, Chie Yanase, Dominika Kwasniak, Aruna Shekhar Nannivala Chandra Shekhar, Gowri Poigaialwar, Richard Voit, Keisuke Ito, Ryan Tewhey, Satish Nandakumar
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Endometrial carcinoma (EC) is the most common form of uterine cancer, with increasing prevalence in developed countries due to population aging and rising obesity rates. Identifying biomarkers that may enable more personalized medicine or immunotherapy discoveries is important to create more treatment options as well as to tackle aggressive EC subtypes. To that end, we developed a 58-parameter mass cytometry (CyTOF™) panel that enables deep phenotyping of both immune and non-immune cells in tumors as well as in PBMC. This was achieved by combining pre-optimized panels for pan-immune cell phenotyping, T cell profiling and cytokine and cytotoxic mediator detection with nine drop-in antibodies selected for their relevance in EC as biomarkers or potential immunotherapy and drug targets. Cell proliferation was assessed using IdU incorporation and Ki-67 staining, while tumor cell and tumor-infiltrating leukocytes (TILs) were further defined with EpCAM, Her2, β-catenin, CD69, CD103, PD-L1 and FoxP3. Live-cell universal barcoding with a tellurium-based 7-choose-3 strategy allowed multiplexing of samples during cell staining and acquisition. This high-parameter panel allows for deep phenotypic and functional profiling of circulating immune cells, tissue and tumor cells and TILs. By utilizing a flexible and modular assay designed to characterize both immune and non-immune cells, this CyTOF panel can facilitate biomarker discovery and accelerate drug development in cancer therapeutics.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ed8c8be3b1ae0d535b8e10dcf3a65a15234fb4" target='_blank'>
              Flexible and comprehensive phenotyping of tumor and peripheral blood mononuclear cells in endometrial carcinoma 3011
              </a>
            </td>
          <td>
            D. Mahamed, G. Awong, Gary Impey
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 The immune surveillance hypothesis posits that the immune system controls pre-cancerous cells prior to tumor formation. However, direct visualization of the process has been nearly impossible, because pre-cancerous cells are difficult to identify in conventional cancer models. To address this technical gap, our lab established a unique mouse model called MADM (Mosaic Analysis with Double Markers) that generates rare mutant cells with unequivocal GFP labeling. In a MADM model for triple negative breast cancer, we have discovered the specific and prominent presence of T/B cell-dominated immune aggregates adjacent to premalignant mutant ducts. Characterization via flow cytometry and spatial staining has revealed these aggregates are heterogenous in size, composition, and organization, and tend to dissipate as malignant tumors form. Interestingly, a significant portion of T cells in these aggregates are CD4+ Foxp3+ Tregs, which typically play an important role in preventing excess inflammation. We hypothesize that Tregs regulate immune surveillance against premalignancy to prevent excess inflammation/autoimmunity, but in doing so also prevent eradication of premalignant cells. We are testing this hypothesis using Treg ablation during premalignancy and expect that Treg ablated mice will show increased T/B cell presence and activation in the mammary gland and decreased premalignant cell expansion.



 Supported by the Basser Center for BRCA; University of Virginia (UVA) Pinn Scholarship; NIH/NCI R01-CA256199; UVA Cancer Center Training Grant



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e28dd574c9b2b053c06b1da358808d0a549f6508" target='_blank'>
              Uncovering the role of Tregs in immune surveillance against cancer with a mouse model that reveals premalignancy at single cell resolution 3379
              </a>
            </td>
          <td>
            Elaina K. Ball, Xian Zhou, Jianhao Zeng, Alexys Riddick, Hui Zong
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f5dbf13eb9670595afcc13d733cac2504d66e7a" target='_blank'>
              Single-cell aggrephagy landscape delineates intercellular communication of tumor microenvironment associated with ovarian cancer progression and immunotherapy.
              </a>
            </td>
          <td>
            Feng Fang, Jiemiao Chen, Xinran Li, Cao Zhou, Yilan Dai, Yuanyuan Chen, Xin Yang, Yong Xie, Fujie Qin, Huijiao Fu
          </td>
          <td>2025-11-24</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 DNA methylation at CpG dinucleotides is a critical regulator of cell identity and epigenetic memory, and it becomes universally disrupted during hematopoietic aging, preleukemic clonal expansions and their progression towards malignancy. Therefore, understanding how hematopoietic identity is specified, inherited and disrupted at the single-cell level is essential to develop therapeutic strategies for age-related hematologic disorders. However, current single-cell DNA methylation profiling technologies are limited by either low throughput (hundreds of cells) or low capture rates (103-105 CpGs per cell; <0.3% of CpGs the human genome) and are thus insufficient to characterize heterogeneous populations in the hematopoietic system.
 To tackle this limitation, we developed DREAM-seq (Droplet-based Restriction Enzyme And Methylation sequencing), a high-throughput single-cell DNA methylation technology based on droplet microfluidics and MspI-based reduced representation chemistry. DREAM-seq profiles >5,000 single-cell methylomes per experiment, capturing 0.44M CpGs per cell (3.75-fold increase vs. existing methods) and 3.16 CpGs per read (4-fold increase), with each CpG detected in 1 out of 20 cells. This allows us to generate comprehensive datasets covering >50% of CpGs in the human genome, with marked improvements in clustering resolution and cell type identification.
 We leveraged DREAM-seq to sequence 51,208 single-cell methylomes from human peripheral blood, CD34-enriched and whole bone marrow (BM) cells, generating the largest tissue-specific atlas of hematopoietic differentiation at single-cell methylome resolution to date. Unbiased selection of 71,115 variably methylated CpG sites identified all major hematopoietic cell subtypes, recapitulating cell type specific methylation patterns (monocyte-specific CEBPA and MPO hypomethylation) and cell states (TCF7 gene body methylation in effector CD8+ T cells compared to naïve subsets). Cell-type defining CpGs were depleted from CpG islands and TSS but strongly enriched in distal enhancers and CTCF sites (72.2 %), highlighting their role in specifying hematopoietic cell fates. Interestingly, 27.8% of cell-identity CpGs were located in heterochromatin, where DNA methylation provides an additional regulatory layer not captured by chromatin accessibility methods. These data suggest a model in which DNA methylation regulates both enhancer activity and higher-order chromatin structure at heterochromatin regions, which together control hematopoietic cell identity.
 To gain insights into methylation-driven lineage specification, we computed enhancer methylation levels across hematopoietic populations. We discovered that >40% of CD34+ cells display hypomethylation of lineage-specific enhancers prior to commitment, suggesting methylation-based lineage priming at the HSPC level. We further traced the methylation patterns from HSPCs towards differentiated cells in myeloid, lymphoid and erythroid lineages, identifying early and late changes in the epigenome that occur upon lineage commitment. This highlights previously unrecognized roles of DNA methylation in lineage priming and hematopoietic differentiation dynamics.
 To understand how these methylation states are disrupted during hematopoietic aging and age-related clonal expansions, we analyzed 18,268 single-cell methylomes from aged BM donors, including 3 TET2-mutant clonal hematopoiesis samples. We first identified clonal populations by leveraging genetic variants as clonal markers, finding significant oligoclonality in aged BM, with 3-fold less clonal clusters than young BM. By characterizing the methylation state of these populations, we are defining “high-fitness” methylation states, which could be implicated in driving clonal expansions and might be reversed to constrain age-related clonal hematopoiesis.In summary, we present DREAM-seq, a new single-cell DNA methylation technology which we leverage to generate the largest single-cell methylome atlas of human hematopoiesis to date. By defining methylation-based cell states predictive of lineage output and clonal fitness, we identify epigenetic drivers of hematopoietic differentiation and their disruption during age-related clonal expansions. Our framework provides a foundational resource for understanding epigenetic drivers of hematopoietic malignancies, improving leukemia classification and predicting therapeutic responses.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35989f1212a0453a63a17b29c85e417f9e291e63" target='_blank'>
              High-throughput single-cell methylation resolves epigenetic states in hematopoietic differentiation and age-related clonal expansions
              </a>
            </td>
          <td>
            Alba Rodriguez-Meira, Sander Lambo, Carolina Varona-Arguelles, Rony Chanoch-Myers, Paul Kaegy, Salvador Casani-Galdon, Peter van Galen, Volker Hovestadt, Bradley Bernstein
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a prevalent malignancy with the efficacy of current treatments limited. Nitidine chloride (NC), a natural alkaloid, exhibits antitumor potential, but its mechanism in CRC remains unclear. To investigate the antitumor effects and mechanisms of NC in CRC for clinical therapeutic strategies. A Cell Counting Kit-8 was used to assess the viability of HCT116 cells. Xenograft mice received NC (2, 4, 8 mg/kg, intraperitoneal, once every other day for 2 weeks). Transmission electron microscopy and hematoxylin-eosin staining analyzed tumor ultrastructure and pathology. RNA sequencing, real-time quantitative polymerase chain reaction, and molecular dynamics simulation (MDS) examined gene expression and NC-budding uninhibited by benzimidazoles 1 (NC-BUB1) interaction. Single-cell RNA sequencing, spatial transcriptomics, immunohistochemistry, and chromatin immunoprecipitation sequencing explored BUB1 expression and RAD21 regulation. Immune correlation analysis was conducted using TISIDB, TIMER2.0, and deconvolution algorithms (CIBERSORT, CIBERSORT-ABS, EPIC, ESTIMATE, MCPcounter, quanTIseq, TIMER, xCell) to evaluate associations between BUB1 and immune factors or infiltration. NC inhibited HCT116 cells with time- and concentration-dependent half-maximal inhibitory concentration reduction. NC-treated tumors displayed apoptosis and reduced angiogenesis. Transcriptomics identified BUB1 as a core downregulated gene. Multi-omics confirmed BUB1 overexpression in CRC (1293 vs. 2299 samples; standardized mean difference = 1.77, area under the curve = 0.94) in proliferative/malignant regions. Immune analyses revealed that high BUB1 expression correlated negatively with antigen presentation molecules, chemokines/receptors, and immune cell infiltration (B cells, M2 macrophages, cancer-associated fibroblasts, endothelial cells), suggesting immune microenvironment suppression. BUB1 knockout inhibited CRC cells; NC reduced BUB1 expression. MDS showed a stable NC-BUB1 complex (− 8.8 kcal/mol). The RAD21–BUB1 axis exerted key regulatory roles. NC exerts anti-CRC effects by downregulating BUB1, disrupting the RAD21–BUB1 axis, and alleviating BUB1-mediated immune suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d703d4feecc793ab8f000bc5fc44a15216910320" target='_blank'>
              Nitidine chloride inhibits colorectal cancer by targeting BUB1: mechanistic insights from molecular dynamics simulation, spatial transcriptomics, and single-cell RNA sequencing
              </a>
            </td>
          <td>
            Ke-Jun Wu, Jin-Cheng Li, Li-Min Liu, Xiao-Jue Huang, Liu-Hui Mo, Jing-Wen Ling, Kun-Hua Xiong, Wei Zhang, Rong-Quan He, Hui Li, Gang Chen
          </td>
          <td>2025-11-19</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Extramedullary disease (EMD) in multiple myeloma (MM) represents an aggressive phenotype with poor outcomes. We performed single-cell RNA sequencing (scRNA-seq) on clonal plasma cells (CPCs) from EMD tissue, solitary plasmacytomas (SPM), and bone marrow aspirates (BMAs) from patients with MM to identify transcriptional states unique to EMD.



 We processed FFPE tumor samples from EMD and SPMs and cryopreserved CD138+ BMAs from MM, using the 10x Genomics FLEX platform. Cells/nuclei were fixed, hybridized with Human WTA probes, pooled, and sequenced (10,000–40,000 cells/library) on NovaSeq X. After removing low quality cells, data was normalized and integrated using RPCA in Seurat. Plasma cell identity was assigned via Cellarium and canonical markers. Clonality was inferred by light chain restrictions and refined via random forest modeling. Cell cycle effects were regressed out to reduce proliferation bias.



 We analyzed 128, 908 CPCs from 16 samples [8 EMD (62, 669 CPCs), 4 SPM (20, 607 CPCs), 4 BMA (45, 632 CPCs)] and identified eight transcriptionally distinct cell clusters. Differential expression and pathway comparisons were performed between EMD/SPM/BMA within each of these clusters. Integration of gene expression and pathway analyses revealed consistent upregulation of cell cycle and E2F pathways in EMD.
 We identified two highly proliferative cell cycle clusters (Cluster 2 with S-phase genes and Cluster 4 with a G2/M transcriptional program) that were expanded in EMD compared to BMA and SPM.
 Cluster 0 comprised a quiescent CPC population enriched for RNA processing pathways alongside downregulation of ER stress and autophagy. EMD cells demonstrated suppressed interleukin signaling [fold change (FC) –33.8), and class I MHC antigen processing (FC –23.5) relative to BMA, suggestive of an immune suppressive transcriptional program. Angiogenesis-related genes [VEGFA (log2FC 2.4), ADM2 (log2FC 2.2)] and EZH2 [log2FC 2.1; q<0.0001)] were upregulated in EMD. Compared to SPM, EMD plasma cells in this cluster demonstrated marked downregulation of CD74 (log2FC –2.6; q<0.0001), a key modulator of MHC class II antigen presentation.
 Cluster 1 comprised of metabolically stressed secretory CPCs and displayed downregulation of early growth response pathways in EMD compared to BMA, concurrent with upregulation of E2F signaling. Suppression of IL-2/STAT5, IL-6/JAK/STAT3, and TGF-β pathways in EMD suggests reduced reliance on bone marrow–derived cytokines and impaired immunoregulatory signaling. CD44 was upregulated in EMD versus BMA (log2FC 3.2), consistent with its role in EMD spread and marrow independence.
 Cluster 3 was enriched for extracellular matrix–interacting plasma cells (COL1A1, MMP2) with upregulated epithelial–mesenchymal transition (EMT) pathways in EMD compared to both BMA (FC 7.5) and SPM (FC 7.07). Cluster 6 was defined by a migratory, motility-primed plasma cell population (CD44, AHNAK, LMNA and S100A10). Notably, IL32 was overexpressed in EMD compared to BMA (log2FC 5.1; adj. p = 2.02×10⁻¹⁴), a finding linked to hypoxia-driven proliferation. Compared to SPM, EMD cells also had elevated MYC/E2F activity and DNA repair, hallmarks of a proliferative phenotype.
 Cluster 5 displayed a strong interferon-stimulated gene signature (ISG15, IFIT1, TNFSF10), with enrichment of type I IFN, JAK–STAT, and TNF pathways. Within this inflammatory cluster, EMD exhibited marked suppression of immune signaling, metabolism (TCA, OXPHOS), and TP53 programs, consistent with an immune-evasive, metabolically suppressed, and deregulated state.
 Finally, within Cluster 7, a smaller subset defined by NFκB–driven anti-apoptotic transcriptional activity, EMD exhibited significant downregulation of KLF2 (log2FC –3.7; adj. p = 1.77×10⁻²⁷), likely reflecting disruption of the KDM3A–KLF2–IRF4 axis, a transcriptional network critical for bone marrow homing and plasma cell survival.
 We consistently observed downregulation of the MAPK pathway in EMD compared to BMA, aligning with prior bulk RNA-seq data and contrasting with the near-universal presence of MAPK mutations, highlighting a disconnect between mutational status and pathway activation.



 In the largest yet reported single-cell transcriptional atlas of EMD and SPM, we demonstrate EMD to be highly proliferative with metabolic reprogramming, EMT, and downregulation of key immunologic signaling pathways, indicating immune escape and microenvironmental independence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e0a6922ae53649a447863637f01b1956b7523a5" target='_blank'>
              Single-cell transcriptomics reveals proliferative, immune-evasive, and bone marrow–independent transcriptional programs in extramedullary multiple myeloma.
              </a>
            </td>
          <td>
            S. Zanwar, Surendra Dasari, W. Gonsalves, T. Kourelis, M. Binder, D. Jevremovic, Michael D Howe, Jansen N. Seheult, Angela Dispenzieri, Mrinal Patnaik, S. V. Rajkumar, Shaji K Kumar
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323b823debf44456586f41e4e06b190055e610d9" target='_blank'>
              Genetic Ablation and Multi-Omics Profiling Reveal CEP55 as a Key Driver of Tumorigenesis in Diverse Cancer Models
              </a>
            </td>
          <td>
            Behnam Rashidieh, Yuanhao Yang, A. Bain, Phillip Theron, Catharina Vosloo, Parinaz Ahangar, Brydie Bowden, S. Tria, Ben B. Wang, Sandra Isenmann, Nicholas J. Westra van Holthe, Zherui Xiong, Hani Vu, Sowmya Sharma, A. Sengal, J. A. López, John Finnie, Antonella Papa, Pascal H. G. Duijf, Quan H. Nguyen, Pirjo M. Apaja, K. Khanna
          </td>
          <td>2025-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Pancreatic cancer, recognized as a refractory tumor, has an overall survival rate of less than 10%, and its mortality rate continues to rise. Due to the low immune activity induced by its unique tumor microenvironment, pancreatic cancer is classified as a “cold” tumor and is insensitive to current immunotherapies. However, little is still known about the identification and functional mechanisms of key regulatory molecules in the formation of “cold” tumors. In this study, we identified the pancreatic ductal cell niche through single-cell sequencing. By calculating CNV scores using inferCNV to distinguish malignant from non-malignant cells and analyzing differences in transcriptional levels, we constructed a 15-gene model. The inhibitory effect of UBE2H on the immune microenvironment was studied through single-cell sequencing analysis, including the inhibition of T cell function, impairment of antigen presentation in macrophages and neutrophils, and activation of neutrophil extracellular traps (NETs). Our results revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche, primarily observed in pancreatic cancer samples. Notably, single-cell sequencing analysis of UBE2H overexpression revealed an enhanced T cell PD-L1 interaction axis and a downregulation of antigen presentation pathways in macrophages and neutrophils (antigen processing and presentation, macrophage activation, and neutrophil activation). Further in vivo experiments confirmed that high UBE2H expression promotes tumor progression, leading to increased T cell exhaustion (PD1) and decreased activation (CD69, GZMB, IFNR) in the immune microenvironment, impaired antigen presentation of macrophages and neutrophils (H2Kb, I-Ab, I-A/E), and increased neutrophil extracellular trap formation (MPO, NE, CITH3). This study revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche and analyzed the inhibition of UBE2H on the immune microenvironment of pancreatic cancer through single-cell sequencing and in vivo experiments, providing important clues for the formation of “cold” tumors in pancreatic cancer and opening new directions for exploring new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ecc8251593af0fcef99c0ebc48949b3c7fba9f" target='_blank'>
              Integrated scRNA-seq and transcriptome analyses uncover the effects of UBE2H on the immune microenvironment regulation in pancreatic cancer
              </a>
            </td>
          <td>
            Fuxin Huang, Zhongyan Zhang, Jike Fang, Yue Chen, Jinhui Wei, Quanzhang Li, Chuanzhao Zhang, Shanzhou Huang, B. Hou
          </td>
          <td>2025-11-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Meningiomas represent the most prevalent primary intracranial tumors, with the majority classified as WHO grade 1, typically benign tumors exhibiting slow growth. In contrast, WHO grade 2 and 3 meningiomas display increased histopathological aggressiveness, characterized by increased growth, recurrence rates, invasiveness and poorer prognoses. Emerging evidence suggests that tumor microenvironment (TME) plays a critical role in tumor aggressiveness; however, its contribution to meningioma pathobiology is poorly understood. This study aims to establish a patient derived orthotopic xenograft (PDOX) meningioma model to elucidate the role of the TME in driving tumor aggressiveness and to identify novel therapeutic targets. To this end, primary tumor cells derived from meningiomas of various WHO grades were isolated and grown as monolayer cultures, spheroids, and organoids. Additionally, the immortalized meningioma WHO Grade 3 IOMM-Lee cell line was used to represent high-grade tumors. To investigate tumor formation and TME, these cells were implanted intracranially into NMRI nude mice. Tumor development was confirmed via MRI. Subsequently, whole brains were collected either 6-9 months post-inoculation or upon symptom onset, followed by sectioning and analysis through immunohistochemistry. Moreover, tissue samples are being collected for spatial transcriptomics to further investigate molecular profiles. Preliminary analysis of brains implanted with IOMM-Lee cells revealed a 100% tumor take rate, with high Ki67 expression, indicative of significant proliferation. A notable presence of microglia and macrophages (IBA1, CD204) was observed, suggesting involvement of the TME. Current efforts focus on expanding the cohort by including more patients with different grades of meningiomas. This model will provide valuable insights into the cellular composition, heterogeneity, and tumor-immune cell interactions. Ultimately, it will uncover potential therapeutic targets for meningioma and enable their functional validation. This project is supported by the European Union’s Horizon 2020 research and innovation programme under the Marie Sklodowska-Curie grant agreement No. 945322 (LEAD) and Lundbeck Foundation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3b2bddbecba83931920b06f1f448eed8ac49c4e" target='_blank'>
              TMIC-74. A patient-derived orthotopic meningioma model for interrogation of the tumor microenvironment
              </a>
            </td>
          <td>
            Pelin Gulizar Ersan, Claes Ottzen Laurentiussen, J. Haslund-Vinding, Josephine Budde, J. Lange, Mikael Håndstad, Shania Lashkari, N. Hammouda, Karina Stræde, Nadia Hajib, M. Andersen, Bo Halle, F. Poulsen, Andreas Kjaer, Frederik Vilhardt, T. Mathiesen, B. Kristensen
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (iCCA) is a highly metastatic tumor type, and iCCA with intrahepatic metastasis is associated with poor prognosis. Previous studies have confirmed that the tumor microenvironment (TME) is closely related to tumor metastasis. However, the state of the TME during iCCA liver metastasis remains unknown.


APPROACH AND RESULTS
We profiled 28 specimens from 16 patients with iCCA (with and without intrahepatic metastasis) using integrated single-cell and spatial transcriptomics, bulk RNA sequencing, whole-exome sequencing, and in vivo and in vitro functional experiments to characterize the specific TME changes during iCCA liver metastasis. we identified the metastatic tumor cells, COL3A1+ epithelial cells, and revealed that COL3A1+ epithelial cells-endothelial-to-mesenchymal transition cells crosstalk promoted tumor invasion by inducing mesenchymal transformation of endothelial cells, while COL3A1+ epithelial cells-VEGFA+ CCL4+ neutrophils crosstalk promoted tumor colonization by forming neutrophil extracellular traps.


CONCLUSIONS
We revealed the key biological mechanisms involved in the invasion and colonization phases of iCCA liver metastasis. Moreover, we provide valuable data for understanding iCCA liver metastasis and a possible avenue for the treatment of advanced iCCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd684325d487cd1d69724768d37142d24e7c6b41" target='_blank'>
              Spatiotemporal landscape of intrahepatic cholangiocarcinoma liver metastasis at the single-cell level.
              </a>
            </td>
          <td>
            Ziyu Xun, Hao Wang, Zhengfeng Xuan, Kexu Xiong, Na Zhang, J. Long, Huishan Sun, Yiran Li, Chengpei Zhu, M. Piao, Ting Zhang, Longhao Zhang, Shuofeng Li, Chengjie Li, Jiongyuan Li, Boyu Sun, Zi-Qi Zhou, Shanshan Wang, Ziyue Huang, Kai Liu, Yang Tan, Xiaohua Shi, Xiaobo Yang, Hanping Wang, Ling Lu, Haitao Zhao
          </td>
          <td>2025-12-10</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Cancer arises from genetic and epigenetic alterations that reprogram gene expression networks, driving unchecked proliferation. Glioblastoma (GBM), the most common and lethal primary brain tumor, has a median survival of about one year and a dismal 5% five-year survival rate. Despite extensive research, effective treatments remain elusive, underscoring the need for a deeper understanding of GBM biology. Although several genetic alterations have been implicated in gliomagenesis, driving genetic factors responsible for GBM are still obscure, and how these genetic changes interact with epigenetic dysregulation is poorly understood. Here, we identify CHD5, a gene located in the frequently deleted 1p36 chromosomal region, as recurrently deleted in over 20% of GBM cases and further downregulated epigenetically. To investigate the role of CHD5 in gliomagenesis, we generated a glioma-prone mouse model carrying conditional alleles of Pten, Trp53, and Chd5, and employed engineered neural stem cells (NSCs)—a major GBM cell-of-origin—to induce tumor formation. Loss of Chd5 impaired NSC differentiation, enhanced proliferation, and drove in vivo gliomagenesis. Mechanistically, Chd5 forms a NuRD complex that binds the Myc promoter and super-enhancers to regulate its expression. Chd5 loss leads to chromatin accessibility changes, Myc upregulation, and aggressive tumor development. Importantly, reactivating CHD5 in human GBM xenografts significantly extended survival. These findings establish CHD5 as a tumor suppressor in gliomagenesis, offer therapeutic insights for cancers with 1p36 deletions, and present a robust NSC-based platform for functional genetic studies and modeling GBM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccecf93321da1d6d66641061e1225be0f52f0874" target='_blank'>
              EPCO-68. CHD5 IS A TUMOR SUPPRESSOR IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Xueqin Sun, A. Mills
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Overtreatment of patients with ductal carcinoma in situ (DCIS) is driven by a lack of a reliable prognostic biomarker. Widespread mammographic screening has resulted in a substantial increase in women diagnosed with DCIS. To improve patient risk stratification, we investigate circulating tumor cells (CTCs) as a biomarker for DCIS patients’ biological aggressiveness and as an indicator of early dissemination. We apply microfluidics to enrich CTCs from 34 patients with DCIS and find a significantly higher concentration of CTCs compared to in healthy controls. We profile CTCs and matched DCIS tissues using single-cell RNA sequencing. We find that CTCs express higher clonal aberrations when compared to white blood cells from the same samples, and clonal comparisons between matched tissue and CTC samples provide evidence for an evolutionary bottleneck model. mRNA expression in CTC reveals EMT/MET and immunoregulatory pathway regulation with a suggestion of racial differences. Last, we provide support for early dissemination in DCIS using a Mouse IntraDuctal model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c47d4ebd8af791a497369853a1d6906389917524" target='_blank'>
              Circulating tumor cells as predictive biomarkers in the risk stratification of DCIS: Evidence of early dissemination
              </a>
            </td>
          <td>
            Neha Nagpal, B. Rupp, Yan Hong, Jamie Wagner, Kelsey Larson, Christa Balanoff, Allison Aripoli, Ashley I. Huppe, Onalisa D Winblad, Jessica Peterson, Molly Hill, Camron Smith, Carol Fabian, Andrew K. Godwin, Timothy Fields, Max S. Wicha, F. Behbod, S. Nagrath
          </td>
          <td>2025-10-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Over 75% of PTCL-NOS patients fail to achieve long-term control with current therapies, highlighting the need for novel therapeutic targets. Genetic and non-genetic drivers in cancer cells require external cues from other cancer cells and from the lymphoma microenvironment (LME), which enable cancer cell adaptation to metabolic stress, immune mediated stress and treatment exposure. The resulting signaling crosstalk propels the co-evolution of cancer and LME cells with emergence of diverse cancer-supporting cell states. Identifying these and their maintenance mechanisms may therefore reveal novel therapeutic targets.
 We characterized the LME of PTCL using transcriptional signature deconvolution and multiparametric imaging in 845 patient samples. We identified four biologically and clinically distinct categories named Depleted (DP), B-cell rich (BR), Mesenchymal (MS) and Inflammatory (IN) based on the proportion of different LME cells. RNA-seq analysis showed that SMA+ cancer associated fibroblasts (CAF) are present in DP, MS and IN LMEs each with distinct phenotypes. To functionally analyze the contribution of CAFs to PTCL maintenance, we established multiple patient-derived tumor xenografts (PDXs) that retain the same LME categories of primary tumors. We then tested lymphoma cell survival in co-cultures of CAFs and PTCL cells from matched (e.g., IN-CAF + IN-PTCL) and unmatched PDXs (e.g., IN-CAF + DP-PTCL). Matched co-cultures sustained lymphoma cell proliferation to a higher extent than unmatched, independently of LME subtype. This suggests that CAF are distinctly reprogrammed within each LME.
 We hypothesize that pro-tumoral CAF phenotypes require adaptive gene expression in coordination with lymphoma cells' phenotypes. RNA-Seq of matched IN-CAF co-cultured with IN-PTCL revealed activation of pathways like “Transcription”, “DNA replication” and “Translation” including the upregulation of CDK7. Although CDK7 plays a minor role in cell cycle progression, its phosphorylation of the RNA polymerase II as part of the TFIIH complex is critical in activating and coordinating “cell state”-defining transcription.
 To assess CDK7's role in establishing lymphoma-supportive CAF states, IN-PTCL PDX mice were treated with the covalent selective CDK7 inhibitor YKL-5-124 for 12 h followed by transcriptomic and proteomic profiling of IN-PTCL and IN-CAFs. In IN-CAFs, CDK7 inhibition altered gene and protein expression of several collagens and cytokines, including COL1A1, COL1A2, MMP8, and CXCL12. Ligand-receptor analysis identified CXCL12-CXCR4 as the key CDK7-dependent interaction between IN-CAFs and IN-PTCL cells. IN-PTCL cells showed activation of DNA damage repair pathways, including upregulation of ALYREF and EIF4E, likely as compensatory mechanisms for decreased CDK7-dependent transcription. Since these may represent acquired therapeutic vulnerabilities, we conducted viability screenings in co-cultures of IN-PTCL and IN-CAF isolated from YKL-5-124 (and vehicle) tretated PDX mice. Cells were subsequently exposed for 48 h, individually and in co-cultures, to a drug library containing 40 clinical-phase compounds. This approach led to the identification of 14 compounds with synergistic anti-lymphoma effects. Among these, ten drugs including the XPO1 inhibitor Selinexor, the HSP90 inhibitor PU-H71 and the translation inhibitor Omacetaxine exhibited enhanced anti-lymphoma activity specifically in the co-culture setting, whereas four compounds showed reduced efficacy. This was not seen in mismatched co-cultures and indicates a potential protective role of CAFs and, importantly, that this mechanism is CDK7 dependent.
 To address the limitations of PDX models established in immunodeficient mice with limited ability to capture the immune-mediated effects of drug treatments, we developed an IN-PTCL syngeneic model by intrasplenic implantation of the murine T-cell lymphoma cell line EL4. Treatment with YKL-5-124 significantly reduced the tumor burden compared to vehicles. Cytokine profiling (immunoblotting array) and ELISA of culture medium of YKL-5-124 treated EL4-CAFs showed reduced levels of pro-inflammatory cytokines such as CXCL12, IL6, CCL2, ICAM-1, and CXCL1 indicating that CDK7 is necessary in establishing pro-lymphomagenic IN-CAF phenotypes.
 In conclusion, CDK7 activity is required to sustain specific pro-tumoral crosstalk interactions of matched CAFs and PTCL cells, which can be exploited therapeutically.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/386e7a67f6fb5e2e6ca873d1aaf4d401efdcd89c" target='_blank'>
              CDK7 enables adaptive gene transcription required by stromal cells to support distinct T-cell lymphoma phenotypes and represents a novel microenvironment-directed therapeutic target
              </a>
            </td>
          <td>
            K. Kouidri, D. Fiore, M. Revuelta, N. Di Siervi, Federico Ruiz Moreno, Subhashini Joshi, C. Kayembe, Abigail Taylor, G. Inghirami, L. Cerchietti, N. Zamponi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Glioblastoma (GB), IDH-wildtype, exhibits increased incidence and poorer prognosis in elderly patients, yet the biological underpinnings of this age-related disparity remain poorly understood. Since previous studies have suggested only a limited increase in potential GB-driving genetic alterations in the human brain with age, we hypothesize that non-genetic factors may present aging-dependent contributors to GB pathobiology. In particular, age-associated microenvironmental drivers have been revealed in other cancer types, but their relevance in GB remains underexplored. Moreover, recent GB mouse model studies have suggested that poorer survival in older animals is driven by factors distant from the tumor bulk (“extratumoral”), but this has not been examined in humans due to limited analyses of matched non-tumor brain tissue. To address this gap, we performed single-cell RNA sequencing (scRNA-seq) on formalin-fixed paraffin-embedded tumor samples from 36 GB patients, comprising 18 elderly (65–88, median 75 years) and 18 younger (35–55, median 47 years) individuals. Cohorts were balanced by sex and by the three major epigenetic GB subtypes RTK I, RTK II, mesenchymal, as defined by DNA methylation profiling. Whole-exome sequencing was performed to control for mutational events. For half of the patients, tumor-adjacent normal-appearing cortex was histologically selected, and additionally profiled by scRNA-seq to assess potential age-dependent extratumoral changes. Our spatially stratified approach enabled the detection of diverse malignant cell populations within the tumor core, as well as enrichment of non-malignant cell types often underrepresented in tumor-core analyses, including astrocytes, neurons, and OPCs. Preliminary analyses indicate a region-specific increase in potentially tumor-promoting cellular signatures across immune and glial compartments in older patients, such as more reactive myeloid signatures at the tumor border. Ongoing analyses include integration of these findings with published normal brain datasets and correlation with clinical outcomes. Our study thereby provides a comprehensive framework for understanding aging-associated drivers in human GB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5116a94bf42e3504a7df9be18b8c861fc00afa09" target='_blank'>
              TMIC-76. Deciphering aging-associated drivers of glioblastoma biology at the tumor core and border
              </a>
            </td>
          <td>
            Laurens Zaschke, S. Schmid, D. Teichmann, Peggy Wolkenstein, Jakob Nückles, Christin Siewert, Brendan Osberg, Huy Philip Dao Trong, J. Onken, A. Koch, Matthias Endres, Leif Ludwig, David Capper, Ilon Liu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 For glioblastoma (GBM), like other solid tumors, intratumoral heterogeneity is likely a critical factor in promoting tumor growth, homeostasis, and therapeutic response. Recent applications of single-cell genomics techniques have revealed the scope and nature of GBM cell diversity. However, these techniques do not allow for the experimental manipulation of these same tumor subpopulations, raising a critical barrier to understanding tumor biology and developing effective therapies. To overcome this barrier, we developed endogenous gene reporter systems to enable the isolation, characterization, and manipulation of key GBM subpopulations directly in patient tumor isolates. This approach utilizes streamlined techniques for gene editing in GBM stem-like cells (GSCs), which eliminate the need for the cloning of individual cells. As proof of concept, we applied this approach to two genes in hGSC isolates: OLIG1 (GBM-OPC marker) and SOX9 (GSC/NSC stem cell homeostasis). We targeted the 3’ exon of each gene so that EGFP or RFP are in-frame with the upstream exon but separated by a 2A ribosome skip sequence. We then employed these GSC reporter isolates to comprehensively screen human transcription factors using a CRISPR-Cas9 library and LV-ORF library (1836 genes and 3548 isoforms) to identify transcription factors that activate or repress OLIG1 and SOX9. From these screens we identified 15 genes necessary and sufficient for regulating OLIG1 promoter activity, including known and novel regulators. These included SOX9 itself, which we find is a key factor linking OPC identify with self-renewal of GSCs. In addition, we identified dozens of transcription factor isoforms (validated by long read RNA-seq) that impact reporter activity, as well as ectopic factors not normally expressed in our GSCs which, nonetheless, significantly alter OLIG1 and SOX9 expression. At this meeting, we report results of these screens and present a functional genetic map of factors affecting the GBM-OPC cell identity, self-renewal, and plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9966806d724ae473e21ea19603f70a36860465" target='_blank'>
              STEM-26. COMPREHENSIVE FUNCTIONAL GENOMIC SCREENS REVEAL HUMAN TRANSCRIPTION FACTORS REQUIRED TO MAINTAIN GLIOBLASTOMA OLIGODENDROCYTE PROGENITOR CELL-LIKE IDENTITY
              </a>
            </td>
          <td>
            Anca B. Mihalas, Kelly Mitchell, Sonali Arora, Colette Felton, Mikey Witkowski, Aiden Abadilla, Daniel A. Kuppers, Angela Brooks, Alice Berger, Patrick J. Paddison
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Brain metastases are frequent and often lethal complications of advanced cancers. Microglia, resident immune cells of the brain, are known to exert both anti-tumor and pro-tumor functions in late-stage metastases; however, their response during the initial outgrowth of metastatic lesions is not well characterized. Understanding how heterogeneous microglial subgroups are regulated in the developing tumor microenvironment could pave the way for therapeutic strategies to eliminate metastatic tumors at an early stage. In this study, we used a combination of in vivo fate map imaging, single-cell RNA sequencing, and a holographic photoconversion-based technique (Opto-omics) to track tumor fate and early microglial responses over time in the same animals during colonization of disseminated tumor cells. The microglial population was transcriptionally and morphologically heterogeneous, comprising both pro- and anti-tumor subsets. Genetic and pharmacological perturbations revealed that microglial phenotypes could be shifted by inhibiting TGF-β signaling or by deleting the tumor cell surface antigens CD24a and CD47. These findings reveal targetable plasticity in early-stage microglial responses to brain metastasis and suggest that harnessing pro-phagocytic microglial states may offer a therapeutic window before systemic immunosuppression becomes dominant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1778e4e2ee1302e3145504143572e313e56713fc" target='_blank'>
              Microglia Display Heterogeneous Initial Responses to Disseminated Tumor Cells.
              </a>
            </td>
          <td>
            T. Tsuji, Haruka Hirose, Daisuke Sugiyama, Mariko Shindo, Rahadian Yudo Hartantyo, Yutaro Saito, Tsuyako Tatematsu, Shouta Sugio, M. Sanbo, Masumi Hirabayashi, Yasuhiro Kojima, Jun Koseki, K. Hosoya, Hiroshi Yoshida, Tatsuya Ogimoto, Y. Yasuda, Kentaro Hashimoto, Hitomi Ajimizu, Yuichi Sakamori, H. Yoshida, Noritaka Sano, Masahiro Tanji, Hiroaki Ito, Kazuhiro Terada, M. Hamaji, T. Menju, Hiroyuki Konishi, S. Kumagai, Cyrus M. Ghajar, Daisuke Kato, Hiroshi Date, Akihiko Yoshizawa, Yoshiki Arakawa, H. Ozasa, A. Moorhouse, Teppei Shimamura, H. Nishikawa, Toyohiro Hirai, Hiroaki Wake
          </td>
          <td>2025-12-10</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/128cca3d1edd8a7877ad22a89e79521ff40bea75" target='_blank'>
              Single-cell sequencing in bladder cancer: new insights from tumor cell diversity to individualized treatment strategies.
              </a>
            </td>
          <td>
            Cen Liufu, Shuai Ye, Cong Yin, Tao Zhu, Jiahao Jiang, Yan Wang, Bentao Shi
          </td>
          <td>2025-12-03</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background CD47 is a “don’t eat me” signal that is overexpressed in tumors to evade phagocytosis by tumor associated macrophages (TAM). Investigational agents targeting CD47, such as magrolimab, aim to induce phagocytosis of tumor cells by TAMs. Previously, two key TAM subsets have been identified: C1QC TAMs, which display pro-phagocytic activity, and SPP1 TAMs that express pro-angiogenic markers. We characterize CD47 expression and its relationship with tumor macrophages in solid tumor samples. Patients and methods Resectable tumors from head and neck squamous cell carcinoma (n=36) (HNSCC), breast cancer (n=37) (BC), and colorectal cancer (n=36) (CRC) were evaluated for CD47 expression by immunohistochemistry (IHC), two multiplex immunofluorescence panels were used to characterize TAM markers and T cell markers. RNA sequencing was also performed. Results CD47 protein expression was higher on tumor cells compared to stromal cells across tumor indications tested. Although CRC had the lowest prevalence for CD47 expression in primary tumors, we observed a marked increase in CD47 expression in CRC liver metastases. We developed an SPP1 TAM gene signature and validated a C1QC TAM gene signature to estimate TAM abundances from bulk RNA-Seq data. In the TAM mIF analysis, HNSCC had the highest macrophage density of the indications tested. We observed a positive correlation between a higher C1QC: SPP1 TAM ratio and macrophage phenotype and tumor T cell density. C1QC macrophage signatures correlate with tumor CD47 protein expression in BC and HNSCC samples, suggesting interplay between them. Conclusions We characterized CD47 expression across key solid tumor indications being evaluated clinically using anti-CD47 blockade agents: HNSCC, breast cancer and CRC. Using a CD47 IHC assay, we identified HNSCC as an indication with the highest CD47 expression. In addition, we quantified tumor macrophages using multiplex immunofluorescence (mIF) and determined that HNSCC had the highest density of TAMs. Compared to relatively low CD47 expression in primary CRC tumors, CRC liver metastases had very high CD47 expression. Quantification of TAM signatures and CD47 expression represent key biomarkers to monitor in patient samples during exploration of CD47-blockade agents in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31725c03ca6311073138a838325de55dcf4fe468" target='_blank'>
              CD47 expression in solid tumors correlates with phagocytic tumor-associated macrophage gene signature
              </a>
            </td>
          <td>
            Nicholas van Buuren, Mengshu Xu, Yi Zhang, Paola Correa, Shiva Zaboli, Azadeh Madjidi, Christina Moon, Szu-Wen Liu, Ruidong Li, Kai Sun, Shahed Iqbal, Abhishek Aggarwal, Min Wang, Li Li, Jared M Odegard, Kelli Boyd
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Somatic chromosome instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1, 2–3. Although CAs have been linked to mitotic errors resulting in the emergence of nuclear atypia4, 5, 6–7, the underlying processes and rates of spontaneous CA formation in human cells are underexplored. Here we introduce machine-learning-assisted genomics and imaging convergence (MAGIC)—an autonomously operated platform that integrates live-cell imaging of micronucleated cells, machine learning on-the-fly and single-cell genomics to systematically investigate CA formation. Applying MAGIC to near-diploid, non-transformed cell lines, we track de novo CAs over successive cell cycles, highlighting the common role of dicentric chromosomes as initiating events. We determine the baseline CA mutation rate, which approximately doubles in TP53-deficient cells, and observe that chromosome losses arise more frequently than gains. The targeted induction of DNA double-strand breaks along chromosome arms triggers distinct CA processes, revealing stable isochromosomes, coordinated segregation and amplification of isoacentric segments in multiples of two, as well as complex CA outcomes, influenced by the chromosomal break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The experimentation enabled by MAGIC advances the dissection of DNA rearrangement processes, shedding light on fundamental determinants of chromosomal instability. Coupling live-cell imaging, machine learning and genomic sequencing, the MAGIC platform enables investigation of the cellular context, mutation rates and triggers of spontaneous chromosomal abnormality formation, shedding light on fundamental determinants of chromosomal instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9becd4af94c7faf26e4bbaab5024e42c7ccb439f" target='_blank'>
              Origins of chromosome instability unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            Marco R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades, Nina Luisa Sautter, Marina Šimunović, M. Jendrusch, Sonia Zumalave, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Joshua Lucas Eigenmann, Thomas Weber, Patrick Hasenfeld, Eva Benito, Catherine Stober, I. Cortés-Ciriano, A. Kulozik, Rainer Pepperkok, Jan O. Korbel
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) is a highly aggressive malignancy with a heterogeneous molecular landscape. This study investigates the interplay between mRNA subtypes and the tumor immune microenvironment (TIME) in HGSOC. We analyzed transcriptomic data to characterize four mRNA subtypes: immunoreactive, differentiated, proliferative, and mesenchymal. Using DESeq2 and WGCNA, we identified gene modules associated with clinical traits and immune cell infiltration. The immunoreactive subtype exhibited significant enrichment of M1/M2 macrophages, Th1 cells, and B cells, along with LAIR-1, FCRL5, and IRF4 expression. The mesenchymal subtype was associated with fibroblasts, while the proliferative subtype showed strong ovarian stromal cell involvement. Our findings reveal distinct immune landscapes associated with each mRNA subtype, suggesting potential therapeutic targets and prognostic markers for HGSOC.



 NIH/NCI U54CA118638; Chan Zuckerberg Initiative CZIF2002-007045



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aa36312916ae96eb8236af900b44415c56ff2fa" target='_blank'>
              Transcriptomic subtypes and immune infiltration in high-grade serous ovarian cancer 4644
              </a>
            </td>
          <td>
            Kaylin M. Carey, C. Young, Alexis J Clark, Eric D Dammer, Rajesh Singh, J. Lillard
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) exhibits significant transcriptomic diversity, limiting treatment efficacy. This study aims to identify conserved molecular pathways that remain stable in heterogeneous TNBC, providing reliable therapeutic and prognostic targets. Genomic diversity was assessed via Mutant-Allele Tumor Heterogeneity (MATH) in TCGA-TNBC cohorts. Single-cell RNA sequencing from TNBC samples (GSE176078) underwent non-negative matrix factorization (NMF) to identify intratumoral heterogeneity (ITH) metaprograms. Transcriptomic diversity was quantified using PCA-based cell-cell Euclidean distances. CellChat analyzed cellular communication in high- and low-diversity groups. Conserved pathways were identified using KEGG enrichment, ssGSEA, and spatial transcriptomics. And we confirmed the feasibility of targeting sphingolipid metabolism through in vitro and in vivo experiments. TNBC showed elevated genomic diversity, inversely correlated with immune infiltration and pathway activity. Single-cell RNA sequencing revealed ITH metaprograms across malignant subtypes, with Cancer Cycling cells showing proliferative enrichment. High transcriptomic diversity tumors exhibited enhanced Cancer Basal SC interactions with CAFs, promoting stemness and EMT. Multi-region TNBC dataset analysis confirmed sphingolipid metabolism as the most conserved pathway (high interpatient and low intratumoral heterogeneity), enriched in malignant regions and negatively correlated with immune cells. A prognostic model incorporating sphingolipid genes (SPTLC2, SGMS1, SGPP2) stratified high-risk patients. Inhibiting SPHK1 via PF-543 induced dose-dependent cell death. Sphingolipid metabolism could be a transcriptionally conserved pathway in TNBC. Its inhibition may overcome heterogeneity-driven resistance, and its prognostic value enables precise patient stratification, offering a potential strategy against TNBC plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/273a42854b1bca75f9d7c92c18142aeda8983ebb" target='_blank'>
              Conserved sphingolipid metabolism under transcriptomic diversity: a prognostic and therapeutic target in triple-negative breast cancer
              </a>
            </td>
          <td>
            Jian Li, Rui Chen, Xiaowu Wang, Qing Zheng, Ruijie Dai
          </td>
          <td>2025-11-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a major cause of cancer mortality, with distant metastases presenting a significant clinical challenge. While epigenetic mechanisms like DNA methylation are known to influence TNBC progression, their specific role in driving metastasis remains underexplored. To address this gap, we performed comprehensive multi-omics analysis, integrating epigenomic and transcriptomic profiles from TNBC xenograft models and patient cohorts. Genome-wide DNA methylation profiling of primary tumors, lymph nodes, and lung metastases from xenografts revealed pronounced global hypomethylation in lung lesions, consistent with findings from the clinical cohort. Promoter-methylation changes were enriched in pathways linked to invasion and proliferation, and transcriptomic integration identified 22 epigenetically regulated genes. Among these, elevated AK1, SLC2A5, TPI1, and ZBTB17 expression correlated with a higher risk of lung dissemination. These findings highlight altered DNA methylation as a driver of TNBC lung colonization and identify candidate prognostic markers, emphasizing the importance of epigenetic reprogramming in organ-specific lung metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcba6d1c1c788aab2b1bb08a4a08046f1d4d0352" target='_blank'>
              Epigenetic programs shaping lung metastasis in triple-negative breast cancer
              </a>
            </td>
          <td>
            D. Marzese, Andrés F. Bedoya-López, Javier I. J. Orozco, Miquel Ensenyat-Mendez, Betsy J. Valdez, Sandra Íñiguez-Muñoz, Alexander D. Boiko, Huiwen Xie, Maggie L. DiNome, P. Llinàs-Arias
          </td>
          <td>2025-11-05</td>
          <td>Research Square</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Due to their intrinsic heterogeneity and plasticity, pediatric brain tumors present highly complex clinical challenges. To provide insight into the molecular scene of these tumors, we generated spatial transcriptomics data from 19 distinct patients, 7 of which suffered a relapse of the disease. In this cohort, spanning 59 tissue sections across 8 diagnoses and 4 tumor grades, we recovered diagnosis-specific gene expression patterns that could be linked to processes characteristic of developmental stages. We also identified between 2 to 4 spatial archetypal niches per section, which could then be related to 11 main biological themes, and to distinct celltype-like transcriptomic signatures. For example, we noted strong spatial correlations between developmental archetypes and oligodendrocyte lineage signal in pilocytic astrocytoma. Lastly, we utilised spatially inferred copy-number variants for the profiling of relapse-associated tentative tumor clones. These clones were then related via spatial geographical analysis to regions with strong blood vessel signatures. Overall, these results provide vital details into the progression and maintenance of pediatric brain tumors, offering novel edges to be exploited in personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f69864dd9511e02d9a815817c172642be7190a7d" target='_blank'>
              Surveying the molecular landscape of pediatric brain tumors via spatially resolved transcriptomics
              </a>
            </td>
          <td>
            Javier Escudero Morlanes, Timo-Pekka K Lehto, L. Larsson, L. A. Galicia, A. Mollbrink, Alia Shamikh, Elisa Basmaci, Gabriela Prochazka, T. de Ståhl, Johanna Sandgren, F. Taylan, Bianca Tesi, Ann Nordgren, Andrew Erickson, Alastair D Lamb, Klas Blomgren, Monica Nistér, J. Lundeberg, R. Mirzazadeh, L. Kvastad
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="


 Introduction: Subclonal diversity in multiple myeloma (MM)—defined by genetically distinct populations of malignant plasma cells—plays a pivotal role in disease progression, therapeutic resistance, and relapse. Characterizing subclone-specific transcriptomic profiles, copy number alterations, somatic variants, and their interactions with the immune microenvironment is critical for understanding tumor heterogeneity and identifying novel therapeutic targets. In this study, we analyzed smoldering multiple myeloma (SMM) and newly diagnosed multiple myeloma (NDMM) samples to explore the genomic landscape of subclones and their crosstalk with immune cells.
 Methods: CD138+ scRNA-seq and scATAC-seq multiomic data from SMM patients (n=10) and NDMM patients (n=21) were obtained from the published study dbGAP phs003220 (PMC11099140). Matched CD138- scRNA-seq data of immune cells were sourced from Sudha et al., IMS 2023. Subclones for each tumor sample were identified using inferCNV (v1.8.1) and a custom-built integration pipeline that utilized both scRNA-seq and scATAC-seq data. Cell-cell communications between subclones and immune cell types were identified using CellChat (v1.6.0) using the ligand-receptor interactions from CellChatDB. Copy Number Abnormality (CNA) profiles of subclones from single cell were compared with whole genome sequencing (WGS) data (phs003220) and similarity scores were computed. Due to differing technologies (RNA-seq vs WGS) and sequencing depth (single cell vs. bulk), this score reflects how well single-cell multiomics captures major chromosomal changes seen in WGS. Subclone-to-immune cell interactions were quantified to identify statistically significant chromosomal abnormalities that change their prevalence. Survival analysis was performed on the most common MM abnormalities to identify if subclonal co-occurrence influences prognosis. For each subclone, BAM files were generated, and somatic variants (SNVs + Indels) were called using Strelka2 with WGS control samples from the same patient.
 Results: We identified 23 subclones in SMM with a mean of 2.30 (min= 1; max=4); and 79 subclones in NDMM with a mean of 3.59 (min=1; max=8). Subclone and WGS CNAs were comparable to one another with mean Jaccard index (JI) of 0.57 for SMM and 0.69 for NDMM. Samples with low JI tended to have more subclones and more complex CNAs as measured through ploidy (correlation coefficient (r) = -0.069 in SMM and -0.177 in NDMM). NDMM subclones tended to have more CNAs than in SMM. From the IU cohort, Del13 was observed in 60% of the SMM and 45% of the NDMM patients. Gain1q was identified in 50% of the SMM and NDMM patients. Patients with co-occurring Gain1q and Del13q in the same subclone had significantly shorter progression-free survival (PFS) than patients with at least one of these alterations (median PFS 285 days vs 1194 days; P = 0.004). Subclones with Gain1q had a lower average percentage of interactions with CD8T cells (Gain1q=25.435, Norm1q=33.26; P=0.005). In NDMM subclones with Gain1q, there was a lower percentage of MHC-I pathway interactions with CD8T cells (P=0.001) and MHC-II pathway interactions with CD4T cells (P=0.001). Subclones with Del13q showed a higher percentage of MIF pathway (MIF-CD74/CXCR4) interactions in NDMM. SMM subclones with Del13p tended to have a lower percentage of MHC-I interactions in CD8T, CD4T and NK cells. Gain6p subclones had a higher percentage of MK pathway (MDK-NCL/ITGA4+ITGB1) interactions in SMM subclones. Somatic variants were identified for each subclone from the scATAC-seq. When compared with WGS somatic variants in NDMM, a median of 253 variants was found both subclones and WGS while 2752 were unique to subclones. Numerous pathogenic somatic variants were identified in the myeloma driver genes from subclones.Conclusions: The identification of subclone-specific immune interaction networks provides new opportunities for targeted therapeutic intervention. These results establish subclonal CNA profiling as both a prognostic tool and guide for precision immunotherapy in multiple myeloma, particularly for high-risk disease characterized by specific genomic alterations co-occurring in subclones. This study underscores the importance of single-cell analysis to fully capture the clinically relevant genomic heterogeneity that drives disease progression and treatment resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4387fcdd106b6a1f94375e3c3e641c2dbb1f9a" target='_blank'>
              Subclonal genomic heterogeneity shapes immune crosstalk and clinical outcomes in multiple myeloma
              </a>
            </td>
          <td>
            Parvathi Sudha, Sylvia Robertson, Sihong Li, Sean D. McCabe, Kelvin Lee, R. Abonour, Brian Walker, A. Suvannasankha, Travis S. Johnson
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Gastric cancer (GC) remains a major global health burden with persistently high mortality despite therapeutic advances. Accumulating evidence highlights the pivotal role of tumor-associated macrophages (TAMs) in orchestrating gastric tumor progression through immune suppression, angiogenesis, extracellular matrix remodeling, and metastasis. Within the tumor microenvironment (TME), TAMs exhibit functional plasticity, often polarizing toward an M2-like phenotype that promotes immunosuppression and tumorigenicity. These cells actively participate in immune evasion via immune checkpoint expression and cytokine-mediated T cell inhibition, while also facilitating lymphovascular invasion and chemoresistance through exosome-mediated crosstalk. The density and phenotype of TAMs have been associated with prognosis and therapeutic response in GC. Recent studies have proposed TAMs as promising targets for therapy, with strategies focusing on depleting M2 subsets, reprogramming toward M1 phenotypes, and blocking TAM-driven oncogenic signaling. Targeted interventions, including MENK, paclitaxel, and NF-κB inhibitors, have shown potential in preclinical models. This review comprehensively discusses the mechanistic roles of TAMs in GC and evaluates emerging TAM-targeted therapeutic strategies that may enhance the efficacy of immunotherapy and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/913bc3f4af4f9d7fed06d178714e6cfa582ea213" target='_blank'>
              Tumor associated macrophages in gastric cancer dual roles in immune evasion and clinical implications for targeted therapy
              </a>
            </td>
          <td>
            Chang Wang, Xu Fan, Xiaomeng Sun, Yi Xu, Yanna Sun, Jidong Liu
          </td>
          <td>2025-12-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) is a major challenge in oncology due to its high recurrence rate, highlighting the need to identify recurrence drivers. Based on the data provided by Zhang Zemin’s team in their research, we selected survival metadata from 119 patients with lung squamous cell carcinoma and data on the proportions of 51 immune cell types. We then combined machine learning methods to analyze the intrinsic tumor factors and tumor microenvironment (TME) in the recurrence of lung squamous cell carcinoma (LUSC). Chi-square tests identified immune cell subsets enriched in recurrent patients, such as high-MKI67 Tregs, low-FGFBP2 NK cells, low-FOXP3 Tregs, and low-KLRB1 CD8 + T cells. A random forest algorithm further pinpointed pathological response rate (PRR) as the primary predictive factor, with high-MKI67 Tregs as a key secondary contributor. Univariate and multivariate Cox regression analyses and Kaplan-Meier survival curves confirmed that pathological complete response (pCR) and low Treg_MKI67 expression were associated with better survival (p = 0.0055 and p = 0.011, respectively). Patients achieving pCR and with low Treg_MKI67 expression had superior recurrence-free survival (RFS; 2-year RFS: 96.7% vs. 57.9% for non-pCR and high Treg_MKI67, p = 0.00066). These findings underscore the prognostic value of PRR and TME markers. They also highlight the potential of using single-cell RNA sequencing and machine learning to guide personalized therapies and reduce recurrence risk by targeting Treg activity, laying the groundwork for integrating immune markers into clinical practice to improve LUSC prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07193-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771d9eb2ac87b50db8e745eeabbaa9aea5706912" target='_blank'>
              Which dominates recurrence: tumor or microenvironment?
              </a>
            </td>
          <td>
            Guoqing Lyu, Xin Deng, Man Yang, Yan Guo, Yuan Zhang, Lihua Wang, Yan Huang, Sun Wu, Yanting Liu, Wenting Lv, Jin-Cheng Guo
          </td>
          <td>2025-10-28</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Gliomas are highly heterogeneous tumors composed of dense cores and diffusely infiltrating margins, where glioma cells interact with non-neoplastic cells, including microglia—the brain’s resident immune cells. To investigate spatial microglial states, we isolated live microglia from paired patient glioma core and margin samples, and performed single-cell RNA sequencing. We found that margin-associated microglia displayed distinct transcriptional profiles from those in the core. Specifically, purinergic signaling pathways were upregulated at the margin, while chemokine signaling, interferon response, and antigen presentation pathways were enriched in the core. Using Monocle pseudotime analysis, we constructed microglial activation trajectories and identified P2RY12 as the gene most negatively correlated with inflammatory progression, suggesting it may play a role in early-stage activation at the tumor margin. To model this in vitro, we used human iPSC-derived microglia, which recapitulate microglial activation states seen in vivo. ATP stimulation of iPSC-microglia, mimicking increased levels of extracellular ATP (eATP) in the glioma microenvironment, significantly upregulated inflammatory genes seen in the late-stage pseudotime signatures, and downregulated genes in the early-stage pseudotime signature. This indicates that ATP stimulation induces a shift along the inflammatory trajectory towards later stage activation phenotypes. To probe therapeutic relevance, we treated microglia with the purinergic receptor inhibitor PPADS, which attenuated ATP-induced transcriptional programs, including interferon signaling. These results suggest that P2RY12 activation by eATP modulates early microglial activation. We are currently testing the effects of P2RY12 knockdown in the iPSC-microglia model to assess responses to ATP and glioma co-culture. Future directions include acute brain slices and in vivo mouse glioma models to evaluate how targeting P2RY12 affects microglial behavior and tumor progression, potentially informing novel immunomodulatory strategies for glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8293ff64f0c760324290f880f6d68ca70039fdf" target='_blank'>
              TMIC-105. P2RY12 as a potential therapeutic target for disrupting tumor microenvironment– microglia cross talk
              </a>
            </td>
          <td>
            Nkechime Ifediora, Nadine Khoury, Alexander R. Goldberg, A. Mahajan, N. Humala, J. Furnari, Clara Stucke, Kristin R Swanson, Gunnar Hargus, Andrew A. Sproul, Vilas Menon, Marta Olah, Peter Canoll
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Despite decades of concerted research and clinical efforts, patient outcomes in glioblastoma (GBM) remain dismal. While chimeric antigen receptor (CAR)- T cell therapies in GBM have produced individual reports of efficacy, they have yet to meaningfully alter standard of care. Tumor heterogeneity, both inter- and intra-patient, likely contributes to these variable responses. Single-cell RNA sequencing (scRNA-seq) conducted in our lab has revealed that this transcriptional heterogeneity is organized along distinct cellular state axes. We sought to overcome this heterogeneity by designing CAR-T cells directed against these cellular states. We hypothesize that CAR-T cells targeting a defined GBM cell state will lead to a depletion of those populations. We report the development of a panel of cell state-targeting CAR-T cell candidates engineered for target-specific activity in vitro. To model the three-dimensional tumor environment, we developed patient-derived GBM organoids that recapitulate the cell state landscape seen in primary tumors. We treated GBM organoids with these cell state-targeting CAR-T cells and assessed cell states by scRNA seq. Unexpectedly, we observed convergent transcriptome shifts to a new cell state with overlapping features of mesenchymal (MES) and interferon-induced signatures. To target this dynamic, we designed a novel CAR-T cell directed against this population and observed feedforward target enrichment in response to therapy. We anticipate that our findings will inform rational design of translational CAR-T cell candidates for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae77281b5ad33a2d2f93d13dc0c418dca38401a5" target='_blank'>
              EXTH-37. Evaluating cell state-targeting CAR-T cells in glioblastoma
              </a>
            </td>
          <td>
            S. Kovatsis, C. Mount, E. Boxer, Jun Zhong, S. Dumont, D. Gerovasilis, Jack Lu, I. Tirosh, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a high-grade glioma marked by high intratumoral molecular heterogeneity, resistance to therapy and poor prognosis. Leveraging a multisampling approach, we investigated the relation between genomic and physical distance, and whether this information was captured by imaging phenotypes and patient outcomes. We profiled 79 spatially distinct tumor regions from 24 GBM patients (F/M = 0.60; median age = 65.5 years) using deep whole-exome sequencing (>300x coverage). Genomic distance was quantified by pairwise Euclidean distance of somatic mutational profiles and integrated with 3D neuronavigation-based spatial coordinates and progression-free survival (PFS). Two distinct growth patterns emerged: expansive, defined by strong correlation between molecular and spatial distance (RE = 0.6), and stochastic, characterized by molecular divergence uncoupled from spatial proximity (RS = -0.2). High molecular distance correlated with reduced PFS (R = -0.5538, p = 0.026), and stochastic expansion predicted unfavorable outcome (p = 0.035), frequently localized to the frontal lobe. Radiomic analysis from contrast-enhanced T1-weighted MRI revealed that molecular distance was positively correlated with heterogeneous texture features (e.g., GLCM entropy, NGTDM complexity; p < 0.05). Stochastic tumors were enriched for radiomic heterogeneity (e.g.), while expansive tumors displayed homogeneous imaging textures, suggesting convergent phenotypic adaptation in genomically diverse tumors. Functional annotation using neuromaps demonstrated that tumors with high molecular distance exhibited positive correlations with differentiation- and angiogenesis-related metrics (SA_axis, genePC1, CBV), and negative associations with expression of neurotransmitter receptors (acetylcholine, serotonin, dopamine). Stochastic tumors correlated with increased Magnetoencephalography (MEG) derived timescales, implicating higher neural network engagement and spatial fluctuations. These findings suggest a link between type of clonal expansion, radiomic-based structural imaging phenotype, and functional neuroanatomy in GBM, providing a multidimensional framework to interpret tumor behavior and identify clinically relevant radiogenomic signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/111ccc1f311e4f7d5b04bf8582bc8726a2476d94" target='_blank'>
              CNSC-23. MULTIDIMENSIONAL SPATIAL AND MOLECULAR MODELING OF REFRACTORY GLIOBLASTOMA EVOLUTION
              </a>
            </td>
          <td>
            Youjin Song, Y. Miller-Michlits, K. Nenning, L. Greutter, B. Kiesel, D. Lötsch-Gojo, J. Gesperger, J. Breu, D. Nam, G. Widhalm, J. K. Sa, A. Woehrer
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Simple Summary Why does gastric cancer frequently recur despite aggressive treatment? The answer may lie in a small but stubborn population of cells hidden deep within the tumor, the cancer stem cells (CSCs). These cells possess self-renewal capacity, resist conventional therapies and drive relapse and metastasis, making them critical determinants of patient outcomes. Their persistence highlights the urgent need to understand the molecular mechanisms sustaining their survival. One emerging explanation comes from the field of epitranscriptomics, which investigates chemical modifications of RNA that regulate gene expression post-transcriptionally. Among these modifications, N6-methyladenosine (m6A) has attracted particular attention. It serves as a dynamic regulator of RNA metabolism, controlling stability, translation and splicing, and has been shown to promote stemness and aggressiveness in gastric cancer stem cells (GCSCs). Dissecting the m6A-dependent regulatory networks in CSCs paves the way for novel diagnostic biomarkers and therapeutic strategies designed to eliminate the root of gastric cancer recurrence and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57fa6c528cff78e5be882e6c5182cbc5cde84094" target='_blank'>
              The Epitranscriptomic Landscape of Gastric Cancer Stem Cells: The Emerging Role of m6A RNA Modifications
              </a>
            </td>
          <td>
            Diana Pádua, P. Mesquita, Raquel Almeida
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/027f986afc66cae82f4bb168ec0c6120d17b62d0" target='_blank'>
              Transgelin defines pro-tumorigenic cancer-associated fibroblasts in pancreatic cancer.
              </a>
            </td>
          <td>
            Xingxing Wang, Keiko Shinjo, Kohei Kumegawa, R. Maruyama, Shinji Mii, Yukihiro Shiraki, Tastunori Nishimura, Y. Murofushi, Miho M. Suzuki, T. Kabasawa, Mitsuru Futakuchi, A. Kanai, Yutaka Suzuki, Atsushi Enomoto, Yutaka Kondo
          </td>
          <td>2025-12-16</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Brain metastases from breast cancer (BCBM) are highly fatal malignancies with limited treatment options. Developing preclinical models that accurately replicate metastatic phenotypes is essential for advancing next-generation therapies. However, existing BCBM models often fail to capture patient heterogeneity or clinically relevant metastatic biology, limiting their translational potential. To address this gap, we established a cohort of clinically relevant BCBM models that reflect the heterogeneity and complexity of the disease. We collected 24 molecularly and clinically diverse BCBM samples, dissociated them into single cells, and intracranially injected them into CD1 nude mice. Half of these samples exhibited tumorigenicity, and the resulting xenografts were harvested, dissociated into single-cell suspensions, and cultured under conditions optimized for cancer stem cells (CSCs), potentially enriching for metastasis-initiating cells (MICs). MICs displayed key stem-like properties, including the expression of stem and breast cancer markers, self-renewal capacity, and tumorigenic potential in orthotopic models. Upon intracardiac or intranipple injection, MICs successfully colonized the brain and other organs, primarily the bones, recapitulating the metastatic cascade observed in patients. MIC-driven brain metastases closely mirrored patient tumor histology and cytoarchitecture. Brain tropism appeared to be mediated by enhanced MIC adhesion to brain endothelium and interactions with the brain microenvironment, which promoted MIC survival and outgrowth. To assess the fidelity of these models, we performed whole genome and RNA sequencing on matched patient samples, MICs, and MIC-driven brain metastases, revealing a strong similarity to the corresponding patient tumors. Notably, MICs adapted their transcriptomic profiles according to their specific metastatic niche—brain, bone, or mammary fat pad. Additionally, MICs demonstrated high chemoresistance, underscoring their potential as a reliable model for identifying novel therapeutic target">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acc75309cab187fde103a1ed93cd55a99d57736a" target='_blank'>
              TMOD-03. Metastasis-Initiating Cell Subpopulation for Modeling Breast Cancer Brain Metastases
              </a>
            </td>
          <td>
            M. Del Bene, D. Osti, S. Faletti, C. Richichi, M. Patanè, Bianca Pollo, G. Carone, F. DiMeco, G. Pelicci
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Medulloblastoma is the most common malignant brain tumor in children and includes several molecular subtypes with distinct clinical outcomes. Because genetic mutations are relatively rare in this tumor, our study focused on understanding how epigenetic regulation—the chemical and structural changes that control gene activity without altering DNA—contributes to disease progression. Using data from large patient cohorts and single-cell sequencing, we identified specific epigenetic factors that distinguish tumor subtypes and correlate with prognosis. We also developed an epigenetic score that predicts patient survival and is particularly elevated in aggressive Sonic Hedgehog (SHH) tumors. These results suggest that dysregulated epigenetic programs affecting RNA metabolism and cell proliferation play a key role in medulloblastoma aggressiveness, highlighting new potential targets for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733b0f36e6a075919ebc25ace84fd8a8035e0b6e" target='_blank'>
              Single-Cell Heterogeneity of Epigenetic Factor Regulation Deciphers Alteration of RNA Metabolism During Proliferative SHH-Medulloblastoma
              </a>
            </td>
          <td>
            R. Francés, Jenny Bonifacio-Mundaca, Íñigo Casafont, Christophe Desterke, Jorge Mata-Garrido
          </td>
          <td>2025-10-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Antiretroviral therapy (ART) has transformed HIV into a manageable chronic condition. Stopping ART causes rapid viral rebound, revealing complex and latent tissue reservoirs that hinder efforts for a cure. Understanding the biological, cellular, and tissue-specific factors involved is key to overcoming this challenge. In-situ spatial biology and multiplex histology technology can provide novel insights into the spatial immune and anatomical dysregulation driven by HIV. We performed in-situ multiplex-spatial proteomic analysis using the Nanostring CosMx Human 64-plex Immuno-Oncology Protein Panel on fresh-frozen spleen and gut tissues from two people with HIV (PWH) (detectable and undetectable) from the Last Gift Rapid Research Autopsy Cohort. Data quality control and normalization was processed with the AtoMx pipeline. Despite viremia status, we found similar total protein expression in the spleen and colon tissue. Cell type annotation using the CELESTA pipeline revealed that the viremic donor had more vascular smooth muscle cells in the colon, while the aviremic donor had more epithelial and fibroblast cells. Immune profiling showed higher lectin specific protein expressions in immune cells of the aviremic donor, particularly in dendritic and epithelial cells in the colon compared to the viremic donor. These preliminary findings highlight tissue-specific immune and cellular changes in PWH, with future studies planned to compare these changes to those in people without HIV.



 Supported by NIH/NIAID AI-164559



 Technological Innovations in Immunology (TECH)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5645976e541610bcd1dcd387cc992098aa48a1ae" target='_blank'>
              Spatial biology discovery approaches to dissect in situ modification of site-specific tissue HIV reservoirs 2542
              </a>
            </td>
          <td>
            Stephen T. Yeung, P. Moar, Vanessa Gomez-Moreno, Davey Smith, A. Chaillon, S. Gianella, L. Ndhlovu
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Immune system control is a principal hurdle in cancer evolution. The temporal dynamics of immune evasion remain incompletely characterized, and how immune-mediated selection interrelates with epigenome alteration is unclear. Here we infer the genome- and epigenome-driven evolutionary dynamics of tumor-immune coevolution within primary colorectal cancers (CRCs). We utilize a multiregion multiomic dataset of matched genome, transcriptome and chromatin accessibility profiling from 495 single glands (from 29 CRCs) supplemented with high-resolution spatially resolved neoantigen sequencing data and multiplexed imaging of the tumor microenvironment from 82 microbiopsies within 11 CRCs. Somatic chromatin accessibility alterations contribute to accessibility loss of antigen-presenting genes and silencing of neoantigens. Immune escape and exclusion occur at the outset of CRC formation, and later intratumoral differences in immuno-editing are negligible or exclusive to sites of invasion. Collectively, immune evasion in CRC follows a ‘Big Bang’ evolutionary pattern, whereby it is acquired close to transformation and defines subsequent cancer-immune evolution. This study nominates immune escape as an early event in colorectal cancer and shows how this can be driven through both genetic and epigenetic changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b298787dd108fa4c080b3a88642ab2a59626f9" target='_blank'>
              Epigenetically driven and early immune evasion in colorectal cancer evolution
              </a>
            </td>
          <td>
            E. Lakatos, V. Gunasri, L. Zapata, J. Househam, Timon Heide, Nick Trahearn, O. Swinyard, Luis H Cisneros, Claire Lynn, M. Mossner, C. Kimberley, Inma Spiteri, George D. Cresswell, Gerard Llibre-Palomar, M. Mitchison, Carlo C. Maley, Marnix Jansen, M. Rodriguez-Justo, John Bridgewater, Ann-Marie Baker, A. Sottoriva, Trevor A. Graham
          </td>
          <td>2025-11-05</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f662f4b181b3395cd096885302fa365b2511567e" target='_blank'>
              Exploring tumor heterogeneity and drug resistance in cervical cancer through single-cell and bulk transcriptomics.
              </a>
            </td>
          <td>
            Yinxia Yang, Sirong Cheng, Yuxuan Zhong, Pengyu Sun, Xiaoru Deng, Bohui Zhou
          </td>
          <td>2025-11-27</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The biological heterogeneity between left- and right-sided colorectal cancer (CRC) poses a significant clinical challenge and the underlying regulatory mechanisms remain elusive. As an emerging hallmark of cancer, the contribution of circadian rhythm disruption (CRD) to this side-specific heterogeneity is unclear. By integrating single-cell and spatial transcriptomic analyses, this study shows that tumor cells in right-sided CRC exhibit significantly higher CRD scores and identifies the emergence of a NONO-positive tumor-cell subpopulation (NONO⁺ TC) as a key molecular feature of this phenotype. Spatial analysis further confirms that this NONO⁺ TC forms a tightly co-localized microenvironment with fibroblasts in situ. Notably, cell–cell communication analyses indicate that NONO expression does not augment the signal-sending capacity of tumor cells but instead reprograms them into highly efficient signal receivers. These augmented incoming signals predominantly originate from the more pro-tumorigenic myofibroblastic cancer-associated fibroblast (myCAF) subtype, indicating that NONO⁺ TC are particularly sensitive to malignant stromal inputs. In conclusion, our study delineates the dual role of NONO as both a circadian regulator and a pro-tumorigenic signaling hub. By enhancing tumor-cell reception of CAF-derived signals, NONO links CRD to the formation of NONO⁺ TC niches and a malignant microenvironment in right-sided CRC, providing new mechanistic insight into the spatial heterogeneity of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241b9e1e30a34229f91641257f9f4e9436ac45df" target='_blank'>
              NONO links circadian rhythm disruption and enhanced tumor-fibroblast crosstalk in right-sided colorectal cancer
              </a>
            </td>
          <td>
            Zhi-Hao Shang, Qin-Chang Zhang, Song-yang Xi, Kai Chen, Shao-Bo Guo, Zhou Zhou, Xin-Zhuo Zhan, Yun-Xia Wu, Xin-Yi Li, Hai-Bo Cheng, Xue-Jun Song, Gui-Hua Tian
          </td>
          <td>2025-10-31</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86ecdda13a72d7b290cf8fe9dd99c8ce634923c0" target='_blank'>
              Galectin-9-driven immunosuppressive macrophage population shapes the lymph node metastatic microenvironment.
              </a>
            </td>
          <td>
            Yao Zhou, Xiaowei Liu, Guixiu Xiao, Genpeng Li, Xiujing He, Leyi Tang, Xueyan Wang, Yujia Lin, Jianyong Lei, Jing Jing, Hubing Shi
          </td>
          <td>2025-10-25</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progression involves dynamic remodeling of the tumor microenvironment (TME). However, the stage-specific dynamics of immune and stromal cell remodeling throughout LUAD progression remain incompletely understood. Here, the study systematically profiles the cellular composition and transcriptional states across multiple LUAD stages, integrating early-stage patient specimens with publicly available datasets encompassing advanced-stage disease. The analysis reveals a marked stage-dependent shift from a proliferative and immune-activated microenvironment in early LUAD to a hypoxia-enriched and immunosuppressive landscape in advanced disease. A distinct hypoxia-adapted epithelial tumor cell subpopulation (C5), exhibiting transcriptional features of metastasis, invasion, and hypoxia, and poor prognosis, is identified. Advanced LUAD featured immunosuppressive LGMN⁺ macrophages and STAT1-driven exhausted CD8⁺ T cells. FKBP11⁺ plasma B cells exhibited exhaustion-linked metabolic changes. POSTN⁺ CAFs emerged as central mediators of extracellular matrix (ECM) remodeling and immune exclusion. Collectively, the findings reveal a model of hypoxia-driven functional convergence, in which distinct TME components co-evolve toward phenotypes that collectively promote immune evasion, matrix remodeling, and tumor progression. These findings may provide insights into stage-specific cellular dynamics and highlight promising therapeutic targets for precision immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8876e01109cf8f024442bf59e35e9ed246f97f" target='_blank'>
              Characterizing Stage-Specific Cellular Dynamics and Microenvironmental Remodeling in Lung Adenocarcinoma by Single-Cell RNA Sequencing.
              </a>
            </td>
          <td>
            Bomiao Qing, Xiaolan Li, Xiang He, Junyi Wang, Yi Yang, Manling Jiang, Bingbing Yan, Lei Zhang, Anying Xiong, Q. Ran, Guoping Li
          </td>
          <td>2025-11-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e26bc66a34e745a67c0425c16be84d04301c9194" target='_blank'>
              Multimodal analysis reveals potential association of CDH13 with endothelial cells and its overexpression in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Ke-Jun Wu, Li-Hua Yang, Dong-Ming Li, Rong-Quan He, Di-Yuan Qin, Shi-De Li, Jian-Di Li, Y. Dang, Ming-Jie Li, Qi Li, Jian-jun Li, Lu Zhang, Han He, Ji-Feng He, Gang Chen, Xiao-Bo Jiang
          </td>
          <td>2025-12-12</td>
          <td>European journal of medical research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Understanding the mechanisms underlying key agricultural traits remains a central challenge in crop research, but recent advances in technologies are providing powerful tools to address this issue. Among these, single-cell and spatial transcriptomics have revealed tissue heterogeneity and spatial organization, offering unique insights into cellular gene expression dynamics and the coordinated activity of multiple cell types. These approaches help uncover how specific cell types contribute to agricultural traits and refine candidate loci lists through integration with trait-associated loci. Additionally, single-cell and spatial transcriptomics have the potential to serve as cell-level readout platforms integrating cellular perturbations, enabling high-throughput discovery of causal relationships between genotype and gene expression at the cellular level in plants. Successful implementation will accelerate the identification of key genetic variants for crop improvement. Here we review lessons learned from application of single-cell screening in mammalian cells, highlight major technical and biological barriers to its use in plants, and outline potential strategies to overcome these challenges. Together, the widespread application and integration of single-cell and spatial transcriptomics with other technologies enable not only the descriptive cataloging of cell states but also the causal interrogation of sequence functions and regulatory networks at cell-type resolution, ultimately advancing gene function studies and accelerating crop improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bff7551fbd396347f03bdd7b21419c8e91773900" target='_blank'>
              Decoding crops one cell at a time: from cell atlases to single-cell genetics.
              </a>
            </td>
          <td>
            Hao Zhang, Robert J. Schmitz
          </td>
          <td>2025-11-25</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) exhibits spatial and molecular heterogeneity. Understanding region-specific transcriptional programs within distinct tumor microenvironments is crucial for identifying niche-targeted therapeutic strategies. This study uses the Ivy Glioblastoma Atlas Project (Ivy GAP) to characterize transcriptional variation across histologically annotated tumor regions. We analyzed annotated bulk RNA-seq data from Ivy GAP, covering five tumor regions: cellular tumor (CT, n=111), infiltrating tumor (IT, n=24), leading edge (LE, n=19), microvascular proliferation (MVP, n=50), and perinecrotic zone (PNZ, n=65). Differential gene expression (DGE) analysis was performed using CT as reference. Enrichment analysis was conducted using Gene Ontology and KEGG databases. Prognostic significance of differentially expressed genes (DEG) was assessed using CGGA-GBM dataset for survival analysis. Finally, we performed a pseudo-spatial analysis by plotting normalized Fragments Per Kilobase of transcript per Million mapped fragments (FPKM) expression levels of DEG genes across tumor regions to infer expression dynamics across distinct regions. Each region exhibited distinct transcriptional profiles. IT and LE were enriched for synaptic transmission, neuronal signaling, calcium signaling and excitable membrane potential genes; MVP for angiogenesis and extracellular matrix remodeling; and PNZ for hypoxia-induced metabolic adaptation. Several genes, including CACNG3 (20.5 vs 13.6 m, p=0.03), HCN1 (19.8 vs 13.9 m, p=0.02), SULT4A1(19.2 vs 13.9 m, p=0.03), and SYCE1 (21.7 vs 13.6 m, p=0.0006), were strongly associated with survival in CGGA-GBM dataset. Pseudo-spatial mapping allowed visualization of region-specific gene expression gradients. This integrative analysis reveals spatially distinct molecular landscapes in GBM and identifies prognostic gene markers linked to specific tumoral microenvironments. Our findings define region-specific transcriptional programs in GBM, offering mechanistic insights into tumor microenvironment heterogeneity and nominating candidate genes for future functional validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eee57357cd7906869206ea1e359f6b55d3036c56" target='_blank'>
              BIOM-86. MAPPING REGION-SPECIFIC TRANSCRIPTOMIC PROGRAMS IN GLIOBLASTOMA FROM HISTOLOGICALLY ANNOTATED RNA-SEQ DATA
              </a>
            </td>
          <td>
            Biprojit Nath, S. Bobholz, A. Lowman, S. Duenweg, Benjamin Chao, F. Kyereme, Jennifer Connelly, E. K. Mrachak, Max O. Krucoff, Peter S LaViolette
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ade8a8b9df4408ea89e61a0e1bd36268d586bf5" target='_blank'>
              Decoding osteosarcoma from heterogeneity to precision therapy.
              </a>
            </td>
          <td>
            Yu Xu, Yanfu Bai, Fan Yang, Binbin Yang, Zhisheng Huang, Lingling Su, Kuntao Peng
          </td>
          <td>2025-12-11</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad6284204cf8fc0e7c47d8fa667f142a64e34e5" target='_blank'>
              Machine learning identifies TIME subtypes linking EGFR mutations and immune states in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Zetian Gong, Mingjun Du, Ying Li, Bicheng Ye, Yuming Huang, Hui Gong, Wei Wang, Liang Chen, Zongli Ding, Pengpeng Zhang
          </td>
          <td>2025-11-26</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Glioblastoma progression features extensive dissemination of cancer cells throughout the brain. Limited by neuroimaging resolution and lack of specific markers, malignant cell infiltration is clinically undetectable. This makes complete tumor removal impossible and relapse inevitable. During their exodus from the tumor core, glioblastoma cells are traversing distinct brain territories influencing their phenotypic cell state. To better understand the cellular and molecular dynamics of glioblastoma invasion, we profiled glioblastoma patient avatars and clinical samples using total, single-cell and spatial transcriptomics. In addition to gene expression data, active intracellular pathways of glioblastoma invasion were screened with high throughput activated kinase assays (Pamgene). We identified that human glioblastoma stem cells exhibiting proneural (PN) and neuron-progenitor like (NPC) transcriptomic traits in vitro formed highly infiltrative and connected xenografts in patient avatars. Consistent with this, PN/NPC glioblastoma stem cells cultured on primary hippocampal neurons exhibited higher connectivity and motility compared to mesenchymal-like cells. Leveraging single-cell and spatial transcriptome analytics of avatar and patient samples, we performed cell clustering, annotation, pseudotime and RNA velocity analyses to associate glioblastoma phenotypic cell states to cellular functions. We found that invasive glioblastoma cells re-activated genes and pathways related to neuron progenitor maturation when transitioning from the tumor core to the brain microenvironment. To verify whether invasive PN/NPC glioblastoma cell integration would alter brain functionality, we performed preclinical live cerebral blood volume recordings using noninvasive brain functional ultrasounds (fUS). As glioblastomas progressed, we detected a gradual alteration of brain functional connectivity. Modifications happened in brain areas featuring minimal, yet consistent glioblastoma cell colonization, identified by post-mortem pathology analyses. Altogether, our data shed light on the importance of glioblastoma cell adaptability and brain integration during the metastatic progression. They also provide a framework for detecting invisible clinical features such as tumor invasion, focusing on brain functional connectivity assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6de15ca728123a2849468be8f7f288fcf64e2b6b" target='_blank'>
              ANGI-14. GLIOBLASTOMA PROGRESSION FROM TRANSCRIPTOMIC REGULATION TO BRAIN FUNCTIONAL CONNECTIVITY
              </a>
            </td>
          <td>
            Madeleine Lackman, C. Bergström, Anaïs Virenque, Kato Herman, Aarti Venkatesan, Juli Udayani, Takashi Namba, Analiz Rodriguez, E. Castrén, P. Laakkonen, Sinem Karaman, V. Le Joncour
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ca98a4e613e1447cb2daf8bdbf25c8434e308a3" target='_blank'>
              Multi-dimensional omics integrated machine learning framework identifies macrophage-fibroblast-tumor co-infiltration patterns to predict prognosis in gastric cancer.
              </a>
            </td>
          <td>
            Qi Wang, Yuan Ni, Sheng Lu, Benyan Zhang, Jun Ji, Qu Cai, Chao Yan, Feng Qi, M. Shi, Jun Zhang
          </td>
          <td>2025-12-09</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Objectives FAM49B has been shown to promote proliferation and metastasis of colorectal cancer (CRC) by stabilizing MYC through phosphorylation of NEK9; however, its role in shaping the immune suppressive tumor microenvironment (TME), particularly in macrophage polarization, remains unclear. Methods We applied multi-omics approaches to study CRC by integrating 33 scRNA-seq samples from 16 CRC patients, 2 paired spatial transcriptomics (ST) samples, and bulk RNA data to characterize malignant epithelial cells (High_FAM49B_EP) and tumor-associated macrophages (TAMs). Functional validation of FAM49B was conducted via knockdown experiments and proteomics analysis. Results A High_FAM49B_EP subpopulation was identified in primary tumors (PT) and liver metastases (LM), exhibiting elevated MYC signaling and association with poor prognosis. TAMs showed spatial heterogeneity: M1-like CXCL3+ TAMs predominated in PT, whereas M2-like SPP1+ TAMs were enriched in LM. CellChat analysis revealed that High_FAM49B_EP activated macrophage polarization through the MDK–NCL signaling axis. Pseudotime trajectory analysis confirmed differentiation from CXCL3+ to SPP1+ TAMs driven by upregulation of NCL. Spatial mapping showed co-localization of MDK+ epithelial cells with NCL+ TAMs in the immunosuppressive microenvironment. FAM49B knockdown significantly inhibited MDK expression and disrupted ECM–receptor interactions. Conclusions FAM49B promotes immunosuppressive TME formation by mediating TAM polarization via the MDK–NCL axis, suggesting the FAM49B–MDK–NCL pathway as a potential therapeutic target for CRC metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8878570ce8b97d296e4580e17e3836990c936b1" target='_blank'>
              Single-cell and spatial transcriptomics integration reveals FAM49B promotes tumor-associated macrophages polarization in colorectal cancer via the MK pathway
              </a>
            </td>
          <td>
            Tianyu Liu, Quchen Ding, Jin Gou, Chen Lu, Xingming Lu, Jiatong Chen, Y. E, Lianhong Li, Chongguo Zhang, Xiaojuan Zhu, Chunzhao Yu, Xiagang Luo
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 The aberrant properties of brain tumor vasculature are increasingly recognized as barriers to efficient delivery and distribution of bloodborne therapies. These aberrant properties emerge from molecular abnormalities that can be identified through comparisons with normal human brain vasculature. However, vascular cells are difficult to isolate and capture, which leads to under-representation in single-cell datasets and challenges identifying reproducible molecular differences among patient populations. To address these challenges, we have developed a computational strategy to isolate transcriptional profiles of vascular cells through integrative gene coexpression analysis of bulk transcriptomes. We have applied this strategy to ~100 gene expression datasets representing >12K normal adult human brain and adult malignant glioma samples and identified robust markers of glioma vasculature that are not observed in normal adult human brain vasculature. These markers include ENPEP (aminopeptidase A), which encodes a cell-surface protein that is implicated in blood pressure regulation and angiogenesis. We confirmed histologically that ENPEP is profoundly upregulated in hyperplastic blood vessels of glioblastomas (GBMs) and to a lesser extent IDH-mut gliomas but is not detectable in vascular cells of non-neoplastic adult human brain. Furthermore, over 6503 normal adult human brain samples profiled with diverse technology platforms, we found that ENPEP is expressed in the 8th percentile genome-wide. To characterize the functional significance of ENPEP in GBM, we acquired firibastat, an oral prodrug that crosses the blood-brain barrier and selectively inhibits ENPEP. We used the GBM14 FL intracerebral tumor xenograft model to investigate the anti-tumor efficacy of firibastat in vivo. Log-rank survival analysis revealed that firibastat delivered daily by oral gavage (500mg/kg) significantly extended survival (P = 0.04 vs. vehicle). These results support further study of ENPEP and other novel glioma vascular markers for potential applications as biomarkers and therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67db9014a7c696a41912c8e5d1a1d46c4a0b8bd9" target='_blank'>
              TMIC-54. Identifying and exploiting molecular abnormalities of glioma vasculature
              </a>
            </td>
          <td>
            Patrick Schupp, Rebecca Eliscu, Nirupama Korada, Fabrice Balavoine, S. Beccari, T. Ozawa, D. Raleigh, Joanna Phillips, Michael Oldham
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 The relevance of the spatial context of the increasing number of potential biomarkers within the tumor microenvironment (TME) has resulted in a need for reliable and accurate multiplex immunohistochemical (mIHC) assays. The SignalStar mIHC assay from Cell Signaling Technology is a novel approach that was utilized to amplify and visualize up to 8 targets within the same formalin fixed, paraffin-embedded (FFPE) tissue section without the need for cycling of antibodies or fluorophore deposition. Here, we compare each target in an 8-plex panel consisting of T-cell exhaustion markers to the canonical chromogenic IHC assay. We also demonstrate the value of amplification when detecting lower abundance functional markers by comparing the percent positivity of SignalStar amplified targets to that achieved with direct and indirect immunofluorescence. Furthermore, we highlight the flexibility of this assay via the usage of a SignalStar secondary antibody, which allows for the incorporation of any IHC-validated rabbit antibody into a panel. Lastly, we also compare the SignalStar assay to other mIHC assays with respect to signal intensity and percent cell positivity. Ultimately, our data demonstrates that the SignalStar mIHC assay is a powerful tool that accurately characterizes the complex TME, allowing researchers to better understand immunotherapeutic approaches and other treatment strategies.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5388ac320b15789afa2366c0bfc8f1b720971bda" target='_blank'>
              Spatial analysis of the tumor microenvironment can be achieved via the novel SignalStarTM multiplex immunohistochemistry assay 3651
              </a>
            </td>
          <td>
            J. Ziello, Derek Papalegis, Lily Vu, Sasha Tkachev, Gabriella Spang, Sarah R Klein
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are increasingly recognized as critical contributors to the limited efficacy of immunotherapy, particularly through the establishment of immune-excluded tumor microenvironments. However, the functional heterogeneity and spatial ecological roles of CAFs in melanoma remain poorly characterized. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq data to systematically dissect CAF molecular programs in melanoma using weighted gene co-expression network analysis (WGCNA). Spatial distribution, CellChat, and ligand–receptor interaction analyses were applied to construct CAF-centered immune exclusion networks. A functional scoring model (riskScore) based on the CAF-M1 module was developed to evaluate clinical relevance. CAF was spatially co-localized with endothelial cells, and engaged in strong crosstalk with multiple immune cell types via pathways such as COLLAGEN–integrin and THBS1–CD47. The derived riskScore model demonstrated robust prognostic value across TCGA-SKCM cohort. Through spatial aggregation and multifaceted immune signaling, CAF constructs a multilayered immune-exclusion network in melanoma, linking stromal remodeling to immune evasion. These findings offer novel insights into CAF-driven immune resistance and may inform future stratified immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca83751e1c8ec5203b2f4b9c36b4cadfbfd1df19" target='_blank'>
              Exploration of the roles of CAFs in melanoma based on single-cell transcriptomics and spatial transcriptomics
              </a>
            </td>
          <td>
            Sheng Hong, Yuhan Zhao, Anni Hu, Yizhong Zhang, Yuming Li, Yan Rong, Zukai Li, Wenzhuo Su, Lude Zhu
          </td>
          <td>2025-11-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 As cancer therapies improve and patients live longer, the incidence of brain metastases (BrMets) continues to rise. Among women with metastatic breast cancer, 10-15% develop BrMets, with rates as high as 30% in HER2+ and 50% in triple-negative. BrMets, represent highly evolved, therapy-resistant tumors with a median survival of 10 months and accompanied by severe neurologic decline. BrMets often emerge years after initial diagnosis, suggesting that metastatic cells must acquire brain-specific traits to survive. This process may involve transcriptional convergence—where tumor cells adopt brain-like programs, and brain-resident cells exhibit tumor-associated changes. Despite its clinical relevance, the molecular basis of this convergence remains poorly defined. Decoding, and targeting, these shared gene programs could reveal new therapeutic vulnerabilities.



 We performed spatial transcriptomic profiling on 235 patient tissue cores (BrMets, adjacent normal brain, primary breast tumors, and non-cancer brain). Using NanoString GeoMx, DSP, we measured the expression of 18,677 RNAs across 450 spatially-defined regions enriched for tumor, immune, and brain-resident cells. To interrogate transcriptional convergence, we developed the Equivalent Expression Index, a novel statistical tool to detect genes with biologically meaningful expression similarity across distinct cell populations.



 Our integrated analysis revealed two prognostic gene signatures. The Metastasis-Induced Brain Shared (MIBS-9) signature, comprising brain-like genes upregulated in tumor cells, was associated with longer patient survival. Conversely, the Adjacent brain-Resident Cancer Shared (ARCS-81) signature, defined by tumor-like gene expression in brain-resident cells, correlated with shorter survival. These signatures were externally validated and enriched for pathways related to neuronal remodeling, immune evasion, cellular communication, and cancer/metastasis.



 By coupling spatial transcriptomics with a novel equivalence-based algorithm, we uncover distinct, clinically relevant gene programs that mediate bidirectional adaptations between tumor and brain-resident cells. These findings illuminate key mechanisms of metastatic outgrowth and offer new targets for disrupting the breast cancer BrMet microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4c6b99afe4369f11ea5994698b0e8b275a5df02" target='_blank'>
              TMIC-89. Spatial transcriptomics reveals tumor–brain expression convergence and identifies prognostic gene signatures in breast cancer brain metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, Christine Yeh, Pablo Nunez Perez, Giuseppe Barisano, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia Plevritis, M. Gephart
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fb5388d0bd0362161f91d2111550a5aa12010f" target='_blank'>
              Inference of clonal hematopoiesis using the collective behaviour of DNA methylation states
              </a>
            </td>
          <td>
            Samuel. J. C. Crofts, Caleb M. Grenko, Neil A. Robertson, Kristina Kirschner, Tamir Chandra
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represent a complex ecosystem of intertwined interconnections between various cell types. Through diverse mechanisms of intercellular communication, GBM cells can modulate the behavior of both malignant and non-tumor cells, promoting cancer progression. One of the key drivers of GBM plasticity and adaptability is extrachromosomal DNA (ecDNA). ecDNA allows GBM cells to rapidly and reversibly modify their genome and by thousands of folds increase the expression of critical oncogenes. However, virtually nothing is known about the role of ecDNA in intercellular communication. Using qPCR and immunofluorescent staining combined with fluorescent in situ hybridization, we demonstrated that ecDNA can be transported between different populations of GBM cells via tumor microtubes in vitro. The efficiency of this transfer substantially differs for various ecDNAs and for GBM cells derived from different patients. Our results suggest that recipient GBM cells can maintain exogenous ecDNA for up to five weeks. Further experiments with iPSC-derived cerebral organoids implanted with ecDNA-containing GBM cells revealed ecDNA transfer between GBM and non-tumor cells. To confirm that this process occurs in patients, we performed single-cell RNA sequencing and whole-genome sequencing of malignant and adjacent non-transformed brain tissues. Integrated analysis of mutations in DNA and RNA, copy number variations, and gene expression patterns confirmed that a substantial fraction of non-cancer cells within the tumor microenvironment harbor ecDNA. We hypothesize that ecDNA transfer between GBM cells can accelerate cancer evolution by disseminating the most beneficial ecDNA molecules within the tumor, thereby bypassing the bottleneck of clonal competition. On the other hand, ecDNA transfer from GBM to non-tumor cells can theoretically have a profound impact on the properties of recipient normal cells, altering their phenotypes to promote further tumor progression or even turning them into seeds for tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2992457556b240d36672234fd746fe0938a62d29" target='_blank'>
              TMIC-26. Intercellular transfer of extrachromosomal DNA in glioblastoma
              </a>
            </td>
          <td>
            Marat Pavlyukov, Anastasia Kazakova, Lindsey A Dudley, Yuji Nakano, Soniya Bastola, Kunal S. Patel, Harley Kornblum
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Immune surveillance is believed to eradicate pre-cancerous cells to guard against cancer. However, its mechanisms remain unclear, as pre-cancerous cells are difficult to detect. To gain a glimpse into immune surveillance, our lab developed a genetically engineered mouse model recapitulating human triple negative breast cancer (TNBC), which generates rare, p53-Brca1 mutant cells with unequivocal GFP labeling. Guided by the visualization of premalignant cells, we observed tertiary lymphoid structure (TLS)-like immune aggregates near mutant ducts long before tumor formation. GeoMx-based spatial profiling showed not only activation signatures in T and B cells but also elevated interferon response genes in mutant ducts associated with immune aggregates when compared to those free of immune infiltration. FTY720 treatment at premalignancy, which blocks T and B cell egress from lymph nodes, led to increased mutant cells expansion and shortened tumor latency, demonstrating the restraint of mutant cell growth by the immune system. To clarify the roles of specific immune cells, we plan to deplete CD4 T, CD8 T, and B cells at premalignancy. We will also conduct longitudinal studies on T and B cells along the tumor evolution process from premalignancy to malignancy to understand how immune surveillance eventually falters. Taken together, our studies are poised to elucidate key mechanisms of immune surveillance in the premalignant stage of TNBC.



 Supported by NIH R01-CA256199; Basser Center for BRCA; the Pinn Scholarship (UVA); UVA Cancer Center Spatial Biology Funding; UVA Cancer Center Training Grant to X.Z.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ee7201e884081fced81f5a6fa79a355f932afc" target='_blank'>
              First glimpse of immune surveillance during premalignant progression of triple negative breast cancer 3378
              </a>
            </td>
          <td>
            Xian Zhou, Jianhao Zeng, Elaina Kiral Ball, Alexys Riddick, Pat Pramoonjago, Victor Engelhard, Hui Zong
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Neutrophils exhibit a paradoxical role in cancer, either combating the tumor or facilitating its progression. Studies in mice have shown that polarizing neutrophils from a pro-tumor "N2" phenotype to an anti-tumor "N1" phenotype can serve as a strategy for immunotherapy. Understanding the behavioral differences between N1 and N2 human neutrophil subtypes in the solid tumor tissue is important for developing novel cancer immunotherapy. However, it is challenging to monitor neutrophil-cancer interactions in the human tumor tissue. In this study, we developed a human cell-based microphysiological system to quantify both real-time and end-point behaviors of N1-like and N2-like neutrophils during their interactions with pancreatic cancer in a 3D collagen matrix. We first validated that N1-like neutrophils showed higher levels of pro-inflammatory hallmarks including ROS production, IL-8 secretion, and NET release and 2D cancer cytotoxicity than N2-like neutrophils. We found that N2-like neutrophils showed greater migration to cancer cells and higher motility after migration than N1-like neutrophils. N1-like neutrophils engaged in longer contact with tumor spheroids and infiltrated greater into tumor spheroids than N2-like neutrophils. N1-like neutrophils also attenuated the progression of tumor spheroids in terms of invasion, proliferation (Ki-67 expression), and epithelial-mesenchymal transition (vimentin expression) compared to N2-like neutrophils, although tumor apoptosis was not affected. Our microphysiological system reveals the distinct "behavioromes" of anti-tumor and pro-tumor human neutrophil subtypes in pancreatic cancer, which may inform the design and development of novel cancer immunotherapies aiming to modulate specific neutrophil behaviors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d48c57940917155e64c44357483b6c2c6ba258c" target='_blank'>
              Behaviorome profiling of anti-tumor and pro-tumor human neutrophil subtypes in a microphysiological system.
              </a>
            </td>
          <td>
            Shuai Shao, Daniel Duncko, Caroline N Jones
          </td>
          <td>2025-10-24</td>
          <td>Lab on a chip</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The liver frequently serves as a site for metastasis in pancreatic ductal adenocarcinoma (PDAC), attributable to its extensive blood supply and supportive microenvironment, which fosters the formation of a pre-metastatic niche that facilitates tumor dissemination. Deciphering the mechanisms underlying niche formation has historically been challenging, requiring detailed investigation into the interactions between primary tumors and metastatic sites. Although these interactions were previously poorly understood, recent advances have elucidated key pathways involved in this process. These insights have been enabled by cutting-edge techniques, including spatial histological mapping, single-cell sequencing, and the identification of novel molecular markers in pancreatic and hepatic metastases. Furthermore, this review revisits and critically evaluates Paget’s ‘seed and soil’ hypothesis in light of current evidence from studies on liver metastatic niches. This review aims to deepen our understanding of liver metastasis in PDAC, with potential implications for the development of targeted therapies and improved clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63415326ba9f7296f1eee9d2e2aea54c5e8bf121" target='_blank'>
              Deciphering the liver’s role in pancreatic cancer metastasis: pathways and therapeutic approaches
              </a>
            </td>
          <td>
            José Inzunza, Andrea C. del Valle
          </td>
          <td>2025-12-01</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Introduction: The prevalence of epigenetic regulator mutations in clonal hematopoiesis and hematologic malignancies highlights the necessity of understanding how mutations in these regulators affect the fate potential of hematopoietic stem and progenitor cells (HSPCs) at a fundamental level. Utilizing single-cell multi-omics data, including paired transcriptome and chromatin accessibility (10X Multiome) and paired histone post-translational modification profiles and RNA polymerase II profiling (sciCUT&Tag + sciCUTAC) of human bone marrow mononuclear cells from healthy donors, we have identified regulatory dynamics of cell fate gain and loss unique to single lineages and quantify the loss of the stem cell program over a given fate transition.These tools will unravel the complex regulatory landscape of priming and lineage biases in healthy hematopoiesis and provide a pathway for investigating the specific mechanisms by which clinically observed epigenetic regulator mutations disrupt HSPC fate decisions in hematologic diseases.
 Methods: To construct a combined low-dimensional representation of chromatin accessibility scATAC-seq and sciCUTAC data were processed using shared genomic features using Seurat (Hao et al. 2021) and Signac (Stuart et al. 2021), then integrated using Seurat IntegrateData(). A shared pseudotime axis could then be computed using Palantir (Setty et al. 2019). To quantify epigenetic mark enrichment and chromatin accessibility at the gene level, gene scores were computed using ArchR (Granja et al. 2021). Using these scores, regulatory activity trends are fit for individual genes using Gaussian process regression (Otto et al. 2024).
 Results: Co-occurring H3K27me3 depletion, transcription, chromatin accessibility, and H3K27ac enrichment are restricted to the B cell lineage for PAX5 and EBF1, both critical master regulators of B lymphopoiesis. Comparable patterns of repression and de-repression at key transcription factor loci are detectable in the myeloid and erythroid lineages. The pseudotime ordering constructed from combined scATAC-seq and sciCUTAC data allows elucidation of the relative ordering of changes in histone mark enrichment, chromatin accessibility, and gene expression throughout hematopoiesis.
 Conclusions: Preliminary results from this work demonstrate the potential of co-analyzing more than two single-cell data modalities simultaneously, without the need for the development of novel techniques that can measure three or more modalities paired in single cells. The observed epigenetic activity at key master regulators, including PAX5 and EBF1 in B lymphopoiesis, defines a signature of lineage-specific gene activation during hematopoiesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f39fbc2f6b34120c4aaa60d8bbea50ff4cc6bd5" target='_blank'>
              Detecting lineage-specific regulatory activity and stemness retention in single-cell multiomics of human hematopoiesis
              </a>
            </td>
          <td>
            Cailin Jordan, Dominik J. Otto, Jacob Greene, S. Henikoff, Manu Setty
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Phenotypic plasticity plays a pivotal role in cancer, enabling tumor cells to adapt to environmental pressures and evade therapeutic interventions by transitioning between distinct cellular states. However, the contribution of phenotypic plasticity to adaptive drug resistance in glioblastoma (GBM), one of the most lethal of all cancers, remains poorly understood. In this study, we identify that GBM tumor-initiating cells resembling normal radial glia (RG), which occupy the apex of normal neurodevelopment, are driven by aberrant epidermal growth factor receptor (EGFR) signaling. Using a suite of patient-derived GBM models, we demonstrate through global proteomics and single-cell RNA sequencing that pharmacological inhibition of EGFR triggers a lineage transition toward neuronal and oligodendrocyte progenitor (OPC)-like states. This shift is accompanied by activation of oncogenic RAS-MAPK signaling – despite robust and durable inhibition of EGFR activation – and is further modulated by brain microenvironmental cues, including synaptic and calcium-mediated signaling programs. Dual inhibition of EGFR and RAS-MAPK with novel, tumor-selective small molecules blocks these phenotypic transitions and enhances GBM cell death in EGFR-mutated GBM models. To determine the subpopulation dynamics of RAS-MAPK signaling within GBM neurodevelopmental lineages, we develop DENALI (Dual-Expression Nuclear reporter of ERK Activity and Lineage Identity) – a novel, high-complexity barcoded lentiviral vector and integrative fluorescence reporter system. Using DENALI, we investigate the clonal mechanisms driving lineage plasticity in GBM following oncogenic EGFR inhibition and couple adaptive RAS signaling programs to the emergence of neuronal and OPC-like states under EGFRi therapy. Together, our findings establish neurodevelopmental lineage plasticity as a key driver of adaptive resistance in GBM and support dual-inhibition strategies to improve therapeutic outcomes in patients with GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a2a811b34aed9bceff625fc38dcd958e60df8" target='_blank'>
              STEM-23. GBM NEURODEVELOPMENTAL LINEAGE PLASTICITY DRIVES RAPID ADAPTATION TO ONCOGENE TARGETED THERAPY
              </a>
            </td>
          <td>
            Dimitri Cadet, Toby Harris, Elisa Fazzari, Cassidy Andrasz, Nicholas A. Bayley, Thomas G. Graeber, T. Cloughesy, A. Bhaduri, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95f4955996275b1364c480a5de0b28f26332655d" target='_blank'>
              Structural basis for constitutive activation of oncogenic gp130 mutants in human inflammatory hepatocellular adenomas
              </a>
            </td>
          <td>
            Yi Zhou, Jing Cao, T. Ramlall, Luke L. McGoldrick, Jennifer Jones, Annan S. I. Cook, Raymond Leidich, Mark W. Sleeman, William C. Olson, P. Stevis, Matthew C. Franklin
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72d62689c5dd46733c3aa071e7330253553893df" target='_blank'>
              IRF4 is a master regulator of multinucleated giant cell formation
              </a>
            </td>
          <td>
            M. Hofmann, G. Heller, M. Kieler, L. Musiejovsky, Lukas Kremp, Martina Kerndl, P. Ettel, Birgit Niederreiter, Lucia Quemada Garrido, Anahita Sedighi, Thomas Krausgruber, Leonhard X. Heinz, Axel Dietschmann, David Voehringer, Bart Everts, Thomas Weichhart, Omar Sharif, Stephan Blüml, Gernot Schabbauer
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae972bf762bae15538dd4da3de79b673b4d58b2d" target='_blank'>
              Immunosuppressive Tumor Microenvironment in Colorectal Cancer Lung Metastases: Implications for Recurrence After Metastasectomy.
              </a>
            </td>
          <td>
            Minsuk Kwon, Minkyue Shin, M. An, Y.J. Jeon, T. Hong, Jung Kyong Shin, Sung Hee Lim, Yoonah Park, Yong Beom Cho, Seung Tae Kim, Yong Soo Choi, Jeeyun Lee
          </td>
          <td>2025-12-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 The tumor microenvironment (TME) is a highly complex multicellular system and has been shown to play a fundamental role in dictating tumorigenesis. High expression of regulatory surface molecules, cytokines, and chemokines in fibroblast niches within lung tumor stroma, paired with their colocalization with Treg cells, suggests cancer-associated fibroblast (CAF) niches may alter the functional response of Treg cells and influence anti-tumor immunity. Here, utilizing a transcriptomic approach, we have uncovered a distinct CAF population – which we term “imCAFs” – that is predominantly localized to tumor-derived tissue samples and is enriched for immunomodulatory gene signatures, T cell ligands, and chemoattractants. This population expresses a unique set of cell surface molecules that permit their identification and isolation from lung tumor tissue. Interestingly, CXCL9, a well-established T cell chemoattractant, was among the top differentially expressed genes in the imCAF population. We were able to demonstrate that TME Treg cells preferentially migrate to CXCL9 gradients, and colocalize with CXCL9-producing imCAFs in lung tumors. CXCL9-producing imCAFs attract a unique subset of Treg cells that highly upregulate the CXCL9 receptor, CXCR3. This subset of Treg cells is functionally distinct from CXCR3-negative Treg cells. Our analyses suggest that imCAFs play a critical role in dictating the recruitment and function of a specific subset of Treg cells to the TME in lung tumors.



 Supported by NIH/NHLBI R01HL148718; NIH/NIAID T32AI106711.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/909bf82020ae6c71525db0b4290dae269c849d42" target='_blank'>
              Immunomodulatory cancer-associated fibroblasts influence Treg cell dynamics in lung cancer 2712
              </a>
            </td>
          <td>
            Nicholas Arpaia, Olivia R. Ringham
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Colorectal cancer (CRC) often occurs in combination with sarcopenia (SP), which affects the prognosis of CRC, but the histopathological mechanisms underlying this process are unclear. Single-cell transcriptome technology provides a tool for resolving multi-organ pathological interactions, and previous studies have lacked direct molecular evidence of a trans-tissue microenvironment. To address this, we applied an integrated analytical approach. The integration of single-cell transcriptome data from 16 CRC tumor tissues (scRNA-seq) and 17 SP muscle tissues (snRNA-seq) was used to systematically resolve heterogeneity across tissue microenvironments using Harmony to correct for batch effects and combining cellular annotations, differential expression analyses, and CellChat communication modeling. Our analysis revealed that CRC and SP share 3 core cell types, namely, endothelial cells, fibroblasts and monocytes, but they show distinct transcriptional and functional differentiation in different tissues. CRC cells are enriched in the inflammatory, angiogenic and matrix remodeling pathways, accompanied by the upregulation of hypoxia-associated genes, whereas SP is biased toward muscle structural maintenance, metabolic regulation and repair functions. The MIF-CD74, COLLAGEN and THBS-CD47 pathways were active in CRC, whereas SDC4 and CD36-related signals were dominant in SP. VWF, PDGFRA and FCN1 were stably expressed in both groups, suggesting the existence of a conserved homeostatic regulatory network. In conclusion, this study reveals the cross-tissue functional heterogeneity of cell types shared by CRC and SP, suggests a set of potential pervasive regulators, and provides new perspectives for understanding the systemic effects of tumors and the microenvironmental mechanisms of sarcopenia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28aac551ab88d08b24ed6a70bb80afdcde46ff4f" target='_blank'>
              Single-cell transcriptome analysis elucidates the microenvironmental interactions between colorectal cancer and sarcopenia
              </a>
            </td>
          <td>
            Tihong Qiu, Xiaoying Wang, Jie Li, Yiping Guan, Hongyou Wen
          </td>
          <td>2025-11-28</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) play crucial and important role in cancer dynamics by affecting homeostasis, immunosuppression, and angiogenesis within the tumor microenvironment. Using single-cell transcriptomics, we constructed a comprehensive atlas of myeloid cell populations across healthy and pan-cancer tissues that revealed heterogeneity. Our analysis suggested that TAMs may arise from two distinct origins: C1QC+ TAMs, which likely are derived from resident tissue macrophages, SPP1+ TAMs and ISG15+ TAMs, which appear to originate from circulating monocytes. Regarding immature myeloid-derived suppressor cells (MDSCs), we highlighted THBS1+ MDSCs and their descendants, SPP1+ TAMs, as key contributors to tumor progression, immunosuppression, and angiogenesis. We proposed a dichotomous model for TAMs, in which C1QC+ TAMs are associated with better patient outcomes, whereas the THBS1+ MDSCs - SPP1+ TAMs lineage correlates with poorer survival and unfavorable response to immunotherapy. This study offers insight into the complex interactions among monocyte-macrophage subtypes and sheds light on TAM heterogeneity and its implications for cancer progression and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a07f8d41863dfbb5389df0779f0c2e6e40c36fc" target='_blank'>
              Pan-cancer single-cell RNA sequencing analysis refines multi-origin monocyte and macrophage lineages.
              </a>
            </td>
          <td>
            Truc Do Thanh Nguyen, Andrew J Lee, Hyun Jung Park, Nameeta Shah, Bayrta Mandzhieva, Dong-Sup Lee, Inkyung Jung, Woong-Yang Park
          </td>
          <td>2025-11-13</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related deaths worldwide. Existing therapeutic options have limited efficacy, particularly for lung squamous cell carcinoma (LUSC), underscoring the critical need for the identification of new therapeutic targets. We previously demonstrated that the Transmembrane Serine Protease TMPRSS11B promotes the transformation of human bronchial epithelial cells and enhances lactate export from LUSC cells. Here, we evaluate the impact of TMPRSS11B activity on the host immune system and the tumor microenvironment (TME). Tmprss11b depletion significantly reduces tumor burden in immunocompetent mice and triggers an infiltration of immune cells. RNA FISH analysis and spatial transcriptomics in the autochthonous Rosa26-Sox2-Ires-GfpLSL/LSL; Nkx2-1fl/fl; Lkb1fl/fl (SNL) model reveal an enrichment of Tmprss11b expression in LUSC tumors, specifically in Krt13+ hillock-like cells. Furthermore, utilizing ultra-pH-sensitive nanoparticle imaging and metabolite analysis, we identify regions of acidification, elevated lactate, and enrichment of immunosuppressive (M2-like) macrophages in LUSC tumors. These results demonstrate that TMPRSS11B promotes an acidified and immunosuppressive TME and nominate this enzyme as a therapeutic target in LUSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edde3e6259df0a2e4e7c324d76c028ecfb0de34e" target='_blank'>
              TMPRSS11B promotes an acidified microenvironment and immune suppression in squamous lung cancer
              </a>
            </td>
          <td>
            Hari Shankar Sunil, Jean R. Clemenceau, Anthony Grichuk, Isabel Barnfather, S. R. Nakkireddy, Luke Izzo, Q. Feng, William Hartnett, Bret M. Evers, Lisa Thomas, I. Subramaniyan, Li Li, William C Putnam, Steven Hepensteil, Jingfei Zhu, Barrett L. Updegraff, John D. Minna, R. DeBerardinis, Tae Hyun Hwang, Jinming Gao, T. Oliver, Kathryn A. O’Donnell
          </td>
          <td>2025-11-10</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Understanding the complex molecular mechanisms driving drug resistance in glioblastoma (GBM) is crucial to develop effective therapeutic strategies. While prior studies have identified resistance mechanisms tied to specific drugs or pathways, a multi-modal molecular analysis of resistance across a range of drug classes is lacking. We identified highly drug-resistant (n = 5) and drug-sensitive (n = 4) cultures from a cohort of 32 patient-derived glioblastoma stem cell (GSC) cultures screened against a broad panel of ~ 500 anti-cancer drugs. To elucidate the key drivers of drug resistance, we performed integrative profiling of stemness, differentiation capacity, global gene expression, mutation profiles, and DNA methylation patterns between the two groups. Despite heterogeneous gene expression profiles, drug-resistant GSCs showed consistent upregulation of ATP-binding cassette (ABC) drug efflux transporters, stemness, and extracellular matrix (ECM)-related genes. Compared to drug-sensitive GSCs, drug-resistant GSCs exhibited more pronounced stem-like properties and reduced differentiation capacity. Notably, genes linked to axonogenesis displayed significant CpG island hypomethylation in drug-resistant GSCs. This study suggests a pivotal role for GSC plasticity, stemness maintenance, and ECM-mediated drug evasion in GBM treatment resistance. Our findings highlight the adaptive and dynamic nature of resistance mechanisms in GSCs, emphasizing the need for comprehensive molecular insights to inform targeted therapeutic strategies in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39de902ec6cfcaa0e52251d953e6e404ad13d38" target='_blank'>
              Drug resistant glioblastoma stem cells exhibit enriched stemness signatures and share extracellular matrix overexpression
              </a>
            </td>
          <td>
            Lance L Estabillo, E. Skaga, S. Halldorsson, Einar O Vik-Mo, C. Sandberg
          </td>
          <td>2025-10-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) progression is shaped by the tumor microenvironment, particularly tumor-associated macrophages (TAMs), which often adopt immunosuppressive functions. CD300e, a myeloid receptor involved in immune regulation, has an uncharacterized role in CRC. Methods Functional studies were conducted in azoxymethane/dextran sodium sulfate and MC38 murine CRC models using CD300e systemic and myeloid-specific CD300e knockout mice, and adoptive transfer experiments assessed macrophage-intrinsic effects. Human studies included analysis of CD300e expression in matched tumor and normal tissue from patients with CRC and in vitro co-culture of patient-derived colon tumor organoids with monocytes to study CD300e induction and TAM polarization. Results In vivo, CD300e deficiency led to reduced tumor burden, enhanced major histocompatibility complex expression on TAMs, and improved T-cell responses. CD300e-deficient macrophages exhibited increased phagocytic activity, antigen presentation, and support for T-cell proliferation and cytotoxicity. Adoptive transfer confirmed that macrophage-intrinsic CD300e expression is sufficient to suppress T-cell function and promote tumor growth. In patients with CRC, CD300e is selectively upregulated in tumor-infiltrating monocytes and macrophages, driving a suppressive phenotype marked by impaired antigen presentation. Tumor-derived signals in vitro induce CD300e expression and promote a protumorigenic macrophage profile. Conclusions Our findings identify CD300e as a critical regulator of macrophage-mediated immune suppression in CRC and a potential target for reprogramming TAMs to enhance immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c56c500ab6751223d28c46c0f7327dc521a283" target='_blank'>
              CD300e is a driver of the immunosuppressive tumor microenvironment and colorectal cancer progression via macrophage reprogramming
              </a>
            </td>
          <td>
            Annica Barizza, Stefania Vassallo, L. Masatti, Mattia Laffranchi, Sofia Giacometti, S. Lonardi, Mattia Bugatti, Sara Coletta, C. Della Bella, M. D’Elios, Simone Pizzini, A. Rosato, W. Vermi, Matteo Fassan, G. Spolverato, Silvano Sozzani, E. Calura, R. Sommaggio, Gaia Codolo
          </td>
          <td>2025-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Reconstructing and understanding intra-tumor heterogeneity, the coexistence of multiple genetically distinct subclones within the tumor of a patient, and tumor development is essential for resolving carcinogenesis and for identifying mechanisms of therapy resistance. While bulk sequencing can provide a broad view on tumoral complexity/heterogeneity of a patient, single-cell analysis remains essential to identify rare subclones that might drive chemotherapy resistance. In this study, we performed an integrated analysis of bulk and single-cell DNA sequencing data of core-binding factor acute myeloid leukemia patients, defined by the presence of a RUNX1::RUNX1T1 or CBFB::MYH11 fusion gene. By single-cell sequencing, we inferred tumor phylogenies for 8 patients at diagnosis including patient-specific somatic variants, somatic copy-number alterations and fusion genes, and studied clonal evolution under the pressure of chemotherapy for 3 patients. As a result, we developed an approach to reliably integrate subclonal somatic copy number alterations into phylogenetic trees and clonal evolution analysis, obtaining unprecedented resolution of intra-tumor heterogeneity in CBF AML. We were able to show that the fusion gene is among the earliest events of leukemogenesis at single-cell level. We identified remaining tumor clones in 6 patients with complete remission samples indicating incomplete eradication of the tumor clones. Here, we show that identifying the order of mutation acquisition can provide valuable insights into evolutionary history, offering a framework to improve drug selection in the era of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79dc8d9b0526060e06d2b2413c27f6af3f0f4c7" target='_blank'>
              Resolving intra-tumor heterogeneity and clonal evolution of core-binding factor acute myeloid leukemia patients with single-cell resolution
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina M Strzelecka, Klara Kopp, Natalia Estrada, A. Dolnik, Marlon Tilgner, Coral Fustero-Torre, Felicitas R. Thol, F. Heidel, M. Heuser, L. Haghverdi, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2025-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 INTRODUCTION Monoclonal gammopathies –including MGUS, smoldering multiple myeloma (SMM), and multiple myeloma (MM)– are driven by clonal expansion of malignant plasma cells. Understanding their clonal architecture and the genomic landscape at single-cell resolution is essential to unravel disease progression and guide treatment strategies, particularly in the context of novel immunotherapies. In this study, single-cell DNA sequencing, was combined with surface protein profiling, to comprehensively dissect somatic mutations, copy number variations (CNVs), clonotypes, and immunophenotypes across disease stages and treatment responses. This novel single cell multi-omics approach provides a powerful perspective on the genomic and functional heterogeneity of plasma cell disorders, offering key insights for precision medicine and therapeutic stratification.
 METHODS CD138+ were isolated from bone marrow aspirates of 11 patients (21 samples) with MGUS, SMM, or MM using AutoMACS sorting. Patient samples included diagnostic, progression, relapse, and treatment-resistant stages. Single-cell DNA sequencing was performed using a panel of 839 amplicons, combined with surface protein profiling via a 20-antibody cocktail. Clonal architecture was inferred by integrating somatic mutations, clonotyping and CNVs. Surface protein expression was used to correlate genotypic features with targetable phenotypes at single cell resolution.
 RESULTS Clonal complexity and genomic architecture showed substantial heterogeneity across disease stages and therapeutic contexts. In early disease, such as MGUS and SMM, some cases already exhibited complex CNV profiles or early driver mutations, indicating subclinical clonal evolution. Several patients displayed stable CNV landscapes over time, while mutational profiles fluctuated, suggesting a selection or suppression of cell-specific subclones rather than wholesale genomic shifts.
 In relapsed/refractory multiple myeloma, branched clonal evolution was frequently observed, with sequential acquisition of mutations affecting genes such as TP53, FRG1, and TRAF2 in specific cells. In some cases, dual-clone coexistence with distinct genomic and immunoglobulin rearrangement profiles was evident, reflecting convergent tumor evolution. Furthermore, certain patients exhibited a distinct pattern of surface antibody expression, indicating the presence of plasma cells co-expressing both major therapeutic targets in MM, GPRC5D and BCMA, while other plasma cell populations expressed only one or neither of these targets.
 A key finding emerged when integrating genomic data with clinical outcomes: patients harboring monoallelic deletions and/or mutations in GPRC5D or BCMA (TNFRSF17) consistently showed limited or no clinical response to corresponding targeted therapies. As an example, one patient with biallelic inactivation of GPRC5D in near 90% of the cells prior to TALQUETAMAB treatment, failed to respond, correlating with absent membrane expression of the target. Conversely, a separate patient who progressed on anti-GPRC5D therapy demonstrated a favorable clinical response to BCMA-directed CAR-T cells, consistent with preserved BCMA expression and copy number in all the cells.
 These findings indicate that the genomic integrity and surface expression of immunotherapeutic targets strongly influence treatment efficacy. Importantly, discrepancies between gene copy number and surface protein levels — such as cases with BCMA amplification but heterogeneous expression — highlight post-transcriptional regulation as an additional layer of resistance.
 CONCLUSIONS Single-cell DNA and protein co-profiling in monoclonal gammopathies provides critical insights into clonal evolution, therapeutic resistance, and the dynamic expression of immunotherapeutic targets. Specifically, GPRC5D genomic alterations and reduced protein expression may compromise response to targeted therapies, highlighting the need for functional biomarker screening. Our data support the clinical relevance of integrating single-cell genomics in the management of plasma cell dyscrasias to tailor therapy and anticipate resistance. Our results suggest that performingsingle-cell genomic and proteomic profiling prior to treatment initiation may improve therapeutic selection and predict response, particularly in the era of targeted immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b66dd32e338c623da193862d4d414630dc6f82e" target='_blank'>
              Single-cell DNA profiling unveils clonal evolution and identifies mechanisms of therapeutic resistance and progression in monoclonal gammopathies
              </a>
            </td>
          <td>
            B. Ariceta, P. Aguirre-Ruiz, Andrea Castro, D. Alignani, Saioa Vicente, A. Lopez, A. Alfonso-Piérola, L. Tamariz-Amador, Jesús F San-Miguel, P. Rodríguez-Otero, Felipe Prósper
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) progresses through the sequential accumulation of somatic mutations, with signaling transducer mutation typically preceding the acquisition of epigenetic alterations. However, recent single-cell DNA sequencing (scDNA-seq) studies have identified cases that do not follow this canonical mutational hierarchy. The prevalence of these non-classical mutation orders, and the biological mechanisms by which mutation timing drives phenotypic heterogeneity remain unclear.
 To address this, we analyzed publicly available scDNA-seq data from >150 newly diagnosed normal karyotype AML patients to reconstruct their evolutionary trajectories (Morita et al., 2020; Miles et al., 2020). We observed that 78% of the signaling mutation events—particularly FLT3-ITD, NRAS, and PTPN11—were terminal subclonal events in branch evolution. Surprisingly, 22% of the signaling mutations serve as initiating events, following a linear evolutionary trajectory and patients harboring these initiating signaling mutations exhibited significantly higher white blood cell and blast counts. These data suggest that the timing of signaling mutation acquisition is strongly linked to the resulting clonal evolution pattern, with FLT3-ITD emerging as the most prominent driver among all signaling events.
 To investigate the pathogenic mechanisms of initiating FLT3-ITD and the characteristics of the resultant AML, we therefore performed multimodal profiling (scDNA+protein-seq, scRNA-seq and patient-derived xenograft models) on 16 FLT3-ITD positive AML cases. Eight cases had FLT3-ITD as the initiating mutation (Initiating FLT3-ITD AML, IFA), and eight as subclonal events (Subclonal FLT3-ITD AML, SFA).
 Comparing tumor cell proportions identified by scRNA-seq with the mutant clone fractions from scDNA-seq in the same patient, we confirmed that leukemic transformation preceded acquisition of FLT3-ITD in SFA. Integrated genotype-immunophenotype analysis of paired samples—adjusted for co-mutations—revealed that initiating FLT3-ITD drove an expansion of hematopoietic stem and progenitor cells, whereas subclonal FLT3-ITD did not. Bayesian modeling further showed that initiating FLT3-ITD selectively upregulated leukemia stem cell (LSC) markers CD69 and CD123, resulting in a significantly larger CD123+CD69+ population (74% vs 35%, P=0.04). Consistently, in PDX models, IFA samples exhibited higher engraftment rates (5/8 vs 3/8). ScRNA-seq further revealed expansion of multipotent progenitor (MPP) and lymphoid-primed multipotent progenitor (LMPP) populations (29% vs 13%, P=0.04) and significant enrichment of LSC transcriptional signatures in IFA compared to SFA. Unsupervised clustering identified progenitor-enriched clusters in IFA characterized by downregulation of G2M checkpoint and E2F target pathways, indicative of a quiescent, stem-like state. Unexpectedly, canonical FLT3-ITD downstream pathways showed no significant differences between IFA, SFA, and FLT3-WT AML patients. Instead, in IFA, we observed significant upregulation of Hedgehog pathway genes (GLI2, ALDH1A1), known regulators of cell cycle and LSC quiescence.
 We observed frequent emergence of chr13q (FLT3 locus) uniparental disomy (UPD) in Initiating FLT3-ITD AML patients (15/23). Notably, we found the acquisition or expansion of UPD clones in IFA PDX models. Inferred from scRNA-seq, these UPD clones displayed increased proliferative capacity without losing LSC traits, potentially via RB1 inactivation. Drug sensitivity testing in PDX models revealed that IFA samples responded to chemotherapy and gilteritinib with distinct patterns, with gilteritinib selectively reducing leukemic blasts and inducing differentiation. However, single-cell genotyping of residual cells showed a marked rise in UPD clone post-gilteritinib. Longitudinal follow-up of these models confirmed that relapse was driven by the expansion of UPD clones. These data suggested chr13q is a required evolutionary event in the development and drug resistant progression of IFA.
 Our study demonstrates that initiating FLT3-ITD mutations reprograms leukemic cells to a quiescent, stem-like state, enforcing a linear evolutionary pattern, compared to subclonal FLT3-ITD. Clonal dominance is ensured by non-canonical FLT3 signaling and late-emerging chr13q UPD. These findings underscore the critical role of clonal timing in shaping AML biology and highlight new therapeutic vulnerabilities in initiating FLT3-ITD-driven AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c74975f6b634c7f7ca88b37dc7ce75122c37169" target='_blank'>
              Clonal timing of FLT3-ITD dictates evolutionary trajectory and phenotypic heterogeneity in acute myeloid leukemia
              </a>
            </td>
          <td>
            Xu-Dong He, Mengfang Xia, Ruoyang Shao, Qifa Liu, Qian-Fei Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 While single-cell profiling has been extensively explored in various tumor types, pituitary adenomas, particularly somatotroph tumors (ST), remain understudied. These growth hormone-secreting adenomas are clinically classified into two subtypes, Densely Granulated (DG) and Sparsely Granulated (SG), which have distinct prognostic implications. However, the cellular and molecular differences between these subtypes at the single-cell level are poorly understood. In this study, we performed single-cell RNA sequencing (scRNA-seq) on 12 somatotroph tumor samples, comprising 52,310 cells prospectively collected from acromegaly patients (10 DG and 2 SG). For the first time, we delineated tumor subclustering within ST, identifying two major populations: classic somatotroph (classic ST) and neuronal-like clusters (neuronal ST). A comparison of DG and SG subtypes revealed significant differences in immune cell composition, with DG tumors harboring higher proportions of myeloid and lymphoid lineage cells. Gene set enrichment analysis (GSEA) further highlighted the upregulation of immune-related pathways, including TNF-alpha signaling, in the DG subtype. These findings were validated by immunohistochemistry (IHC). Additionally, prolactin (PRL) expression was markedly elevated in DG tumors compared to SG tumors. A distinct population of cells co-expressing growth hormone (GH1) and PRL was identified, exclusively present in the DG subtype, and corroborated by IHC. Finally, copy number variation (CNV) analysis revealed substantial genomic instability within ST. Notably, most ST cells exhibited high CNV in the chromosome 20p arm within the classic ST cluster, while the neuronal ST cluster of the SG subtype displayed more prominent high CNV in the 20q arm compared to the DG subtype. This study represents the first single-cell transcriptomic analysis of somatotroph tumors, uncovering key cellular and molecular features that differentiate DG and SG subtypes. These findings provide new insights into tumor classification and hold significant potential for the development of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4981aca3364216be263083130840bd242b07fd" target='_blank'>
              PATH-46. Single-cell Transcriptomic Profiling of Somatotroph Tumors Reveals Subtype-Specific Cellular and Molecular Characteristics
              </a>
            </td>
          <td>
            Euihyun Kim, H. Oh, Cheol Ryoung Ku
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Metastasis is the leading cause of cancer deaths, with liver metastases in up to 60% of Triple Negative Breast Cancer (TNBC) cases contributing to the lowest survival rates among metastatic patients. In TNBC mouse models sensitive to immune-based therapy, B cells are essential for improved survival and prognostic of TNBC patient survival. Despite these data, the field has yet to understand the role and regulation of B cells in breast cancer liver metastasis. To overcome the limitations that the metastasis field currently faces regarding representative models, we have created a novel mouse model of TNBC-LM. Using this innovative tool, we have begun to characterize changes in the tumor microenvironment of the primary breast tumor, tumor draining lymph node, and metastatic liver associated with specific B cell phenotypes. Single-cell RNA sequencing data and confocal microscopy reveal changes in B cell subsets, clonal dynamics, and organization across tissue types and cancerous conditions. Longitudinal flow cytometry data describes changes in adaptive lymphocyte dynamics and status throughout metastatic progression. We hypothesize that TNBC cells can metastasize to the liver by disabling B cell activity through tolerance mechanisms and establishing an immune-suppressive tumor microenvironment. Understanding the mechanisms governing anti-metastatic B cell activity will inform development of therapeutics for metastasis; an urgently needed area given the poor prognosis for patients.



 Supported by T32GM133351



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af5e3c8cd45e745dfa06b4320ed3649a91cd4ae" target='_blank'>
              Investigating the role and regulation of B cells in Triple Negative Breast Cancer Liver Metastasis 3428
              </a>
            </td>
          <td>
            Monika J. Quackenbush, Haley Rodriguez, Georgia Lattanzi, Kendrick Nguyen, Daniel Hollern
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Formalin‐fixed paraffin‐embedded (FFPE) tissue specimens represent precious resources for clinical genomic profiling studies, especially when coupled with comprehensive medical records. Even though next‐generation sequencing (NGS) is an effective tool to detect somatic mutations and somatic copy number alterations (sCNA), the biggest challenges in unlocking clinically translatable genomic information from FFPE tissue are low DNA yields and degraded DNA, affected by variable formalin fixation. Another issue is that the proportion of carcinoma and other noncarcinoma cells is variable and can be confounded by intratumoral heterogeneity. To explore these challenges, we isolated pure carcinoma and stromal cells using the DEPArray™ NxT system, a microchip‐based digital sorter that allows isolation of pure, homogeneous subpopulations of cells from FFPE samples. We isolated pure carcinoma and stromal cell populations from 12 FFPE tissues, including tissues from nine primary and metastatic breast cancer and three primary ovarian high‐grade serous carcinomas. This was followed by downstream shallow whole‐genome sequencing (WGS) for copy number landscape profiling (10 samples) and/or a targeted panel for somatic mutation and sCNA analysis (seven samples), subject to cell availability. Seven out of 10 samples (even some with low tumour content or of old age) produced good‐quality genomic data, detecting sCNA in all carcinoma population samples but not in the stromal populations. Mutation analysis was performed successfully in 6/7 samples and somatic mutations were detected in all of them. Our workflow enabled the identification of clinically actionable targets, including PIK3CA, ERBB2, FGFR1/2, CDK6, CCNE1, KRAS amplifications and RB, BRCA1/2 losses in patients that would direct therapy. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/541e25768a5620a1211beabbcc9d8aaaa45c414b" target='_blank'>
              Identification of actionable targets using DEPArray‐based sorting of pure carcinoma and stromal populations from formalin‐fixed paraffin‐embedded tissues followed by shallow whole‐genome sequencing
              </a>
            </td>
          <td>
            Georgios Nteliopoulos, Evie Wren, A. Rushton, Marc K. Wadsley, D. Fernandez-Garcia, F. Manodoro, Otis Agbaimoni, Ritika Chauhan, Z. Cheng, D. Ennis, K. Page, R. Allsopp, Joel Bautista, Ignazio Puccio, Nik Matthews, Kelly L. T. Gleason, Rehman Farah, Laura Kenny, I. McNeish, Jacqueline A. Shaw, R. Coombes
          </td>
          <td>2025-10-27</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Purpose Monocytes and macrophages are recognized as predominant immune populations in human glioblastoma, where they play vital roles in tumor progression. Despite their established significance, the heterogeneity of these cells—particularly within the monocyte compartment—remains incompletely characterized. Methods We comprehensively used scRNA-seq, spatial transcriptome sequencing combined with immunofluorescence and T cell co-culture assays to illuminate the heterogeneity and function of monocyte in glioblastoma. Results In this study, from the perspective of ligand-receptor networks, we have identified and characterized three distinct glioblastoma subtypes. Single-cell RNA-seq analysis further revealed that the C3 subtype with bad prognosis exhibited a higher proportion of S100A9high monocytes. Spatial transcriptomics combined with immunofluorescence assays demonstrated that these S100A9high monocytes were spatially adjacent to M2 macrophages, exhausted CD8+ T cells, and endothelial cells. In vitro T cell co-culture assays revealed S100A9high monocyte produced elevated levels of the immunosuppressive cytokine IL-10, reactive oxygen species (ROS) and inducible nitric oxide synthase (iNOS), all of which might impair T cell function and immune response. Notably, elevated abundance of S100A9high monocyte correlated with poor patient prognosis. Conclusion In summary, our results deciphered the heterogeneity of monocytes in glioblastoma and identified a novel poor prognosis-associated monocyte subset, S100A9high monocytes, which foster an immunosuppressive, pro-tumorigenic microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b56f421b008f9fd171284dfd7fddcbd5bc0f429e" target='_blank'>
              Decoding the Prognosis-Related S100A9high Monocyte in Glioblastoma Using Single-Cell and Spatial Transcriptome Sequencing
              </a>
            </td>
          <td>
            Xiucan Li, Ying Qin, Pengfei Gao, Huixue Wang, Yanling Liu, Lian Ren, Dongdong Wu, Xueyuan Heng
          </td>
          <td>2025-11-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a highly dynamic network shaped by immune, vascular, and stromal interactions, further modulated by extracellular factors. The high tumor heterogeneity complicates treatment due to evolving resistance. Biodynamic imaging (BDI), a 3D coherence-gated holography technique, quantifies intracellular motion to assess phenotypic responses to therapy. We applied BDI to prostate cancer (PCa) treated with IL-27-based combinations to evaluate the dynamic responses of immunotherapy. This study is the first to integrate BDI with RNA-seq to correlate physiological changes with gene expression. Using a subcutaneous PCa model, tumors were treated ex vivo with chemotherapy and immunomodulatory agents. BDI enabled capturing motion frequency shifts (0.01–12.5 Hz), while RNA-seq revealed the molecular changes in the tumors. IL-27 showed potential to modulate the immunologically “cold” TME and enhance therapeutic efficacy. This integrative approach offers a novel platform for evaluating combination strategies in PCa and may inform more effective treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1439e864183bad47fa0679ec246c930f10dfa35f" target='_blank'>
              Exploring tumor dynamics and responses of prostate cancer to IL-27 based treatment combinations through biodynamic imaging and RNA sequencing analyses
              </a>
            </td>
          <td>
            Shreya Kumar, Dawith Lim, Harish Kothandaraman, Nadia Atallah-Lanman, David Nolte, John Turek, Marxa L. Figueiredo
          </td>
          <td>2025-11-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The efficacy of immunotherapy in bladder cancer (BCa) is highly influenced by tumor heterogeneity, underscoring the urgent need for mechanistic insights and precise molecular subtyping. Adenosine-to-inosine RNA editing (AIRE), a critical post-transcriptional mechanism driving protein diversity, offers a promising lens for decoding the immune microenvironment in BCa. In this study, we established an integrative framework combining molecular, imaging, and pathological data to identify and validate a novel immune-related BCa subtype. Based on multi-center transcriptomic datasets, we employed NMF, WGCNA, and Lasso algorithms to classify tumors into two distinct immune subtypes characterized by different levels of immune cell infiltration and divergent responses to immunotherapy. Mechanistic validation revealed that ANXA1 and RASAL2 may contribute to immunosuppressive niche formation via the TNF-NFκB-CXCL8 axis, potentially acting as molecular triggers for subtype differentiation. Furthermore, radiomic and histopathological analyses at the imaging and tissue levels supported the molecular findings, providing orthogonal validation of subtype-specific characteristics. Collectively, this study introduces an AIRE-based immune molecular subtype of BCa and systematically characterizes it across multiple dimensions. By achieving a closed-loop from identification to validation and clinical translation, our findings offer a new perspective for patients most likely to benefit from immunotherapy and advancing precision treatment strategies in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89065f12cff37f2644ca2e30f67834c8758ac184" target='_blank'>
              Integrative molecular–radiopathomic characterization of AIRE-driven immune subtypes in bladder cancer
              </a>
            </td>
          <td>
            Ji Liu, Shuwen Sun, Yadong Guo, Zhuoran Gu, Hong Wang, Yang Yu, Shiyu Mao, Yong Wei, Zongtai Zheng, Bing Shen, Xudong Yao
          </td>
          <td>2025-12-14</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="This study addresses the limited mechanistic understanding behind medical imaging for tumor microenvironment (TME) assessment. We developed a novel framework that analyzes tumor imaging heterogeneity index (TIHI)-correlated genes to uncover underlying TME biology and therapeutic vulnerabilities. DCE-MRI and mRNA data from 987 high-risk breast cancer patients in the I-SPY2 trial, together with mRNA data from 508 patients in GSE25066, were analyzed. TIHI-associated genes were identified via Pearson correlation, clustered via weighted gene co-expression network analysis (WGCNA), and subgroups were defined via non-negative matrix factorization (NMF). The clinical relevance of the image-to-gene comprehensive (I2G-C) subtype defined by subgroups was assessed using logistic regression and Cox analysis. I2G-C comprised four clusters with distinct immune and replication/repair functions. It further stratified receptor, PAM50, and RPS5 subtypes. The “immune+/replication+” was more likely to achieve pathological complete response (pCR) (OR = 2.587, p < 0.001), while the “immune−/replication−” was the least likely to achieve pCR (OR = 0.402, p < 0.001). The “immune+/replication+” showed sensitivity to pembrolizumab (OR = 10.192, p < 0.001) and veliparib/carboplatin (OR = 5.184, p = 0.006), while “immune-/replication-” responded poorly to pembrolizumab (OR = 0.086, p < 0.001). Additionally, “immune+/replication-” had the best distant recurrence-free survival (DRFS), whereas “immune-/replication+” had the worst (log-rank p = 6 × 10−4, HR = 5.45). By linking imaging heterogeneity directly to molecular subtypes and therapeutic response, this framework provides a robust, non-invasive surrogate for genomic profiling and a strategic tool for personalized neoadjuvant therapy selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaac41c7947552aff84e1e324fe7769979665178" target='_blank'>
              Tumor Imaging Heterogeneity Index-Inspired Insights into the Unveiling Tumor Microenvironment of Breast Cancer
              </a>
            </td>
          <td>
            Qingpei Lai, Xinzhi Teng, Jiang Zhang, Xinyu Zhang, Yufeng Jiang, Yao Pu, Peixin Yu, Wen Li, Tian Li, Jing Cai, Ge Ren
          </td>
          <td>2025-11-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly lethal malignancy with poor prognosis due to its resistance to conventional chemotherapy and limited number of targetable mutations. The recurrent loss-of-function mutation of the tumor suppressor ARID1A represents a key vulnerability potentially exploitable via synthetic lethality. To advance research in this area, we successfully established and characterized two novel CCA cell lines, SiSP-K01 and SiSP-K05, derived from moderately differentiated intrahepatic CCA. Both models were confirmed to carry ARID1A heterozygous loss and complete protein depletion, authentically reproducing the target genotype. Comprehensive molecular and phenotypic profiling (including karyotype, growth kinetics, 100% in vivo tumorigenesis, and tumor marker expression) confirmed their fidelity to the patient tissue and CCA lineage. Importantly, exome and transcriptome analysis not only validated the cell lines but also revealed an associated derangement of the PI3K/AKT signaling pathway. Based on this molecular finding, we performed drug screening and demonstrated that these ARID1A-deficient CCA cell lines are hypersensitive to inhibition by both PI3K/AKT and PARP inhibitors. The SiSP-K01 and SiSP-K05 cell lines are therefore critical new preclinical models suitable for functional studies of ARID1A deficiency and the development of targeted therapies for CCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef0e9dfbc0a68397e20306d980742627a07920fa" target='_blank'>
              Establishment of novel cholangiocarcinoma cell lines with ARID1A deficiency and preclinical validation of synthetic lethality therapies
              </a>
            </td>
          <td>
            Sunisa Prasopporn, G. Sittithumcharee, Jantappapa Chanthercrob, Somchai Limsrichamrern, Arada Hirankitti, Pimkanya More-krong, Sakda Sathirareuangchai, A. Chaiboonchoe, Somponnat Sampattavanich, Seiji Okada, Siwanon Jirawatnotai
          </td>
          <td>2025-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="We study the effect of intratumor heterogeneity in the likelihood of cancer cells moving from a primary tumor to other sites in the human body, generating a metastatic process. We model different scenarios of competition between tumor cells using a static evolutionary game in which cells compete for nutrients and oxygen and might choose to stay and proliferate in the primary tumor or opt to a motility strategy in order to find resources in a metastatic site. The theoretical results found in the evolutionarily equilibrium in the mathematical model are in line with the empirical results observed in oncology, namely, the coexistence of both primary and metastatic tumors and the conditions that favor a metastatic process. Particularly, the model finds mathematical support for what is empirically observed in punctuated and branching cancers for the specific case of clear cell renal cell carcinomas: motility of cells is larger in punctuated cancers if the proportion of BAP1 mutations remain below a given cell proportion threshold.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f15c5bee285c56f968dbaed3413f82133b75289e" target='_blank'>
              Effect of intratumor heterogeneity in managing the go-or-grow dichotomy of cancer cells: a game theory modeling to understand metastasis
              </a>
            </td>
          <td>
            André Rocha, C. Manini, Jos'e I L'opez, Annick Laruelle
          </td>
          <td>2025-10-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Lung cancer, particularly lung adenocarcinoma (LUAD), remains the leading cause of cancer-related deaths, with brain metastases complicating treatment and significantly reducing survival. Immune evasion is a critical factor in tumor progression, particularly in brain metastases, in which the immune microenvironment poses unique challenges to treatment. This study aimed to delineate the immune defense mechanisms that are shared between primary and brain metastatic LUAD, and to identify spatially organized programs and molecular interactions that may contribute to immune escape and therapeutic resistance. Methods We integrated single-cell RNA sequencing and spatial transcriptomics to investigate immune defense mechanisms shared between primary and brain metastatic LUAD tumors. Using non-negative matrix factorization (NMF) and signal distribution landscape analysis, we identified metaprogram MP1 in tumor cells, characterized by myeloid-related immunosuppressive features, significantly associated with poor prognosis. Results MP1 was significantly linked to poor prognosis and exhibited a distinct spatial enrichment at tumor borders in both primary and metastatic sites. Based on this, we propose a shared ‘tumor immune defense pattern’ (TIDP), which plays a central role in immune escape, tumor invasion, and intracranial colonization. We also identified ligand-receptor interactions—such as IL18RAP, CAMP (LL37), and other inflammatory or immunomodulatory molecules—that contribute to immune evasion and are upregulated in brain metastases. These may serve as biomarkers for predicting immunotherapy efficacy. MP1’s functional role further suggests that targeting this program could improve therapeutic outcomes. Conclusions This study highlights immune microenvironmental features shared between primary and metastatic tumors, particularly in the metastatic setting, and introduces TIDP as a potential framework for targeting immune evasion in brain metastases. Our findings provide a foundation for developing more targeted therapeutic strategies for patients with brain metastases from LUAD, warranting further investigation into TIDP’s clinical potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16a207bab4b47eb3cbe7a1dd26a7b91b72916199" target='_blank'>
              Single-cell and spatial transcriptome landscapes reveal the spatial immune defense pattern shared by primary and brain metastatic lung adenocarcinoma
              </a>
            </td>
          <td>
            Fanchen Meng, Yuxiang Sun, Jianyu Li, Lin Xu
          </td>
          <td>2025-11-01</td>
          <td>Journal of Thoracic Disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cervical cancer is a prevalent malignancy among women, yet the involvement of neutrophils in its tumor microenvironment remains insufficiently explored. This study utilized single-cell RNA sequencing (scRNA-seq) to delineate neutrophil subsets and elucidate their roles in disease progression and prognosis. Analysis of 20 cervical biopsy samples across different disease stages identified five neutrophil subsets (N0-N4), among which the N4 subset exhibited a marked increase during disease advancement. Spatial transcriptomics and tissue microarray analyses revealed that N4 neutrophils are enriched in tumor regions and are associated with genes implicated in proliferation, metastasis, and immune evasion. Functional characterization demonstrated that N4 promotes tumor progression via activation of the Wnt signaling pathway and extracellular matrix remodeling. A neutrophil infiltration-based risk model was established and validated through multi-omics approaches, highlighting its potential in prognostic prediction. These findings underscore the pivotal role of N4 neutrophils in cervical cancer and provide valuable insights for the development of targeted immunotherapies and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9eca7bc06f3fa7d3102669c9c6d4eefb56b347" target='_blank'>
              Single-Cell and Multiomic Analysis Reveals Neutrophil Heterogeneity and Prognostic Value in Cervical Lesions.
              </a>
            </td>
          <td>
            Ze Wang, Zhuang Liang, Shiyi Liu, C. Cao, Shimin Chen, Nan Yu, Xiaoyuan Huang, Tao Zhang
          </td>
          <td>2025-12-12</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Local relapse along the resection cavity is the most frequent mode of glioblastoma progression. Surgically induced molecular alterations and transformation of residual tumor ecosystems driving local tumor relapse remain elusive. Here, we applied MERFISH and Visium spatial transcriptomics to 203 specimens from 179 patients obtained at first surgery, early re-resection (2–40d) and late recurrence, and trained a spatial graph-attention network (sGAT) that predicted time-after-surgery from microenvironmental neighborhoods (AUC=0.83). Explainable-AI approaches (Integrated-gradient/attention) revealed a stereotyped cascade dominated by a hypoxia/HIF-1α-program with glycolytic switch and lactate-fueled-coupling between residual tumor cells, neurons and infiltrating myeloid cells within 2–10d at peri-cavity-rim. Between days 10–30 the niche transitions to TREM2⁺/P2RY12⁺ microglia, CXCL13⁺ T-cell influx and endothelial proliferation. Multiparametric MRI confirmed a stroke-like-hypoperfusion and blood-brain-barrier breach peaking at days 1–5 and resolving by days 60–90. To validate time-resolved tissue adaptation and causality, we performed spatial metabolomic and transcriptomic profiling on our murine-cortical-resection model. The sGAT classifier trained on human data assigned the exact post-operative status to murine samples (R² = 0.91). MALDI-imaging mass spectrometry confirmed the enhanced lactate consumption in proximity to the resection cavity enriched for neurons and altered microglia. Knockout of the lactate transporter MCT2 in neurons (rAAV:::hSyn:::shRNA-MCT2) demonstrated reduced tumor proliferation and lower-invasive growth (padj<0.05) with reduced tumor-neuron synaptic contacts in a rabies-ΔG-GFP retrograde tracing model using human neocortical slices (padj < 0.01). Similar reduction of tumor-neuron synaptic contacts was observed after depletion of microglia with PLX5622, supporting a co-operative metabolic-inflammatory-axis. Post-resection metabolic adaptation and tumor regrowth link was confirmed in vivo leveraging a LDH inhibitor or ROS scavenger demonstrating a significant decrease in tumor relapse. Our data delineate a narrow, druggable window immediately after surgery in which hypoxic-inflammatory-reprogramming metabolically and synoptically primes glioblastoma for recurrence, nominating peri-cavity lactate-shuttling or microglial activation as actionable adjuvant targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f6d4b8d16d4ee59080a74e2b9fd09c3dcb6928d" target='_blank'>
              TMIC-91. Deciphering spatio-temporal trajectories of tissue adaptation after surgical resection of glioblastoma
              </a>
            </td>
          <td>
            Y. Yabo, C. Larrieu, Mathis Pinglaut, Antonio Pagano Zottola, Giullia Villa, M. Larroquette, Maialen Arrieta, Yanis Zirem, E. Grabis, A. Davoust, Tao Peng, J. Benotmane, Junyi Zhang, D. Bomont, O. Schnell, A. Burban, M. Salzet, M. Nikolski, V. Venkataramani, F. Ricklefs, R. Sankowski, D. Heiland, T. Daubon
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [7, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>