<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-03 09:28:29 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="While recent innovations in spatial biology have driven new insights into how tissue organization is altered in disease, interpreting these datasets in a generalized and scalable fashion remains a challenge. Computational workflows for discovering condition-specific differences in tissue organization typically rely on pairwise comparisons or unsupervised clustering. In many cases, these approaches are computationally expensive, lack statistical rigor, and are insensitive to low-prevalence cellular niches that are nevertheless highly discriminative and predictive of patient outcomes. Here, we present QUICHE – an automated, scalable, and statistically robust method that can be used to discover cellular niches differentially enriched in spatial regions, longitudinal samples, or clinical patient groups. In contrast to existing methods, QUICHE combines local niche detection with interpretable statistical modeling using graph neighborhoods to detect differentially enriched cellular niches, even at low prevalence. Using in silico models and spatial proteomic imaging of human tissues, we demonstrate that QUICHE can accurately detect condition-specific cellular niches occurring at a frequency of 0.5% in fewer than 20% of patient samples, outperforming the next best method which required a patient prevalence of 60% for detection. To validate our approach and understand how tumor structure influences recurrence risk in triple negative breast cancer (TNBC), we used QUICHE to comprehensively profile the tumor microenvironment in a multi-center, spatial proteomics cohort consisting of primary surgical resections, analyzing over 2 million cells from 314 patients across 5 medical centers. We discovered cellular niches that were consistently enriched in key regions of the tumor microenvironment, including the tumor-immune border and extracellular matrix remodeling regions, as well as niches statistically-associated with patient outcomes, including recurrence status and recurrence-free survival. The majority of differential niches (74.2%) were specific to patients that did not relapse and formed a robust interconnected network enriched in monocytes, macrophages, APCs, and CD8T cells with tumor and stroma cells. In contrast, the interaction network for patients that relapsed was notably sparse and enriched in B cells, CD68 macrophages and neutrophils. We validated these findings using two independent cohorts, observing similar cellular interactions and predictive power. Collectively, these results suggest that salient, generalized profiles of productive anti-tumor immune responses are defined by a network of structural engagement between innate and adaptive immunity with tumor and stromal cells, rather than by any single specific cell population. We have made QUICHE freely available as a user-friendly open-source Python package at https://github.com/jranek/quiche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cda789c041e695c92206356373353eb906709b05" target='_blank'>
              QUICHE reveals structural definitions of anti-tumor responses in triple negative breast cancer
              </a>
            </td>
          <td>
            Jolene S Ranek, Noah F. Greenwald, Mako Goldston, C. Fullaway, Cameron Sowers, Alex Kong, S. Mourón, M. Quintela-Fandino, Robert West, Michael Angelo
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Immune checkpoint inhibition (ICI) has fundamentally changed cancer treatment. However, only a minority of patients with metastatic triple negative breast cancer (TNBC) benefit from ICI, and the determinants of response remain largely unknown. To better understand the factors influencing patient outcome, we assembled a longitudinal cohort with tissue from multiple timepoints, including primary tumor, pre-treatment metastatic tumor, and on-treatment metastatic tumor from 117 patients treated with ICI (nivolumab) in the phase II TONIC trial. We used highly multiplexed imaging to quantify the subcellular localization of 37 proteins in each tumor. To extract meaningful information from the imaging data, we developed SpaceCat, a computational pipeline that quantifies features from imaging data such as cell density, cell diversity, spatial structure, and functional marker expression. We applied SpaceCat to 678 images from 294 tumors, generating more than 800 distinct features per tumor. Spatial features were more predictive of patient outcome, including features like the degree of mixing between cancer and immune cells, the diversity of the neighboring immune cells surrounding cancer cells, and the degree of T cell infiltration at the tumor border. Non-spatial features, including the ratio between T cell subsets and cancer cells and PD-L1 levels on myeloid cells, were also associated with patient outcome. Surprisingly, we did not identify robust predictors of response in the primary tumors. In contrast, the metastatic tumors had numerous features which predicted response. Some of these features, such as the cellular diversity at the tumor border, were shared across timepoints, but many of the features, such as T cell infiltration at the tumor border, were predictive of response at only a single timepoint. We trained multivariate models on all of the features in the dataset, finding that we could accurately predict patient outcome from the pre-treatment metastatic tumors, with improved performance using the on-treatment tumors. We validated our findings in matched bulk RNA-seq data, finding the most informative features from the on-treatment samples. Our study highlights the importance of profiling sequential tumor biopsies to understand the evolution of the tumor microenvironment, elucidating the temporal and spatial dynamics underlying patient responses and underscoring the need for further research on the prognostic role of metastatic tissue and its utility in stratifying patients for ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1bdaa11baefcf7560def6afb49fb7f0020d09d" target='_blank'>
              Temporal and spatial composition of the tumor microenvironment predicts response to immune checkpoint inhibition
              </a>
            </td>
          <td>
            N. Greenwald, I. Nederlof, Cameron Sowers, Daisy Yi Ding, Seongyeol Park, Alex Kong, Kathleen E. Houlahan, Sricharan Reddy Varra, Manon de Graaf, Veerle Geurts, Candace C. Liu, Jolene S Ranek, L. Voorwerk, M. de Maaker, Adam Kagel, E. Mccaffrey, Aziz Khan, C. Yeh, C. Fullaway, Zumana Khair, Yunhao Bai, Hadeesha Piyadasa, Tyler Risom, A. Delmastro, Felix J. Hartmann, Lise Mangiante, Cristina Sotomayor-Vivas, Ton N. Schumacher, Zhicheng Ma, M. Bosse, Marc J van de Vijver, R. Tibshirani, Hugo M. Horlings, Christina Curtis, Marleen Kok, Michael Angelo
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>157</td>
        </tr>

        <tr id="Motivation Multiplexed imaging is increasingly used to study tissue architecture in health and disease. To investigate the cancer tumor microenvironment, typically either tissue micro-arrays or small patient cohorts are used to collect and process data. However, studies performed over the course of years, collecting data from thousands of samples are rare and require specialized workflows to ensure sample throughput and reproducibility for data production and processing. Here, we present two such workflows for multiplexed immunofluorescence and imaging mass cytometry of cancer tissues which are applied to a total of roughly 10’000 samples from 2500 patients over six years. Summary In cancer research, multiplexed imaging allows detailed characterization of the tumor microenvironment (TME) and its link to patient prognosis. The IMMUcan consortium collects multi-modal imaging data from thousands of cancer patients to perform broad molecular and cellular spatial profiling across five cancer indications. Here, we describe two workflows for multiplexed immunofluorescence (mIF) and imaging mass cytometry (IMC) developed within IMMUcan to enable analysis of thousands of cancer tissues. IFQuant supports web-based, user-friendly, and reproducible analysis of mIF data. High sample throughput for IMC is achieved by optimizing experimental protocols and developing a robotic arm for automated slide loading. We provide a resource of 350’000 manually labelled cells across 180 cancer samples to accurately annotate cell phenotypes in IMC. All major cell phenotypes and tissue structures correlate well between mIF and IMC. These pipelines form the basis for multiplexed image analysis within IMMUcan and provide computational tools for the larger community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb10125531b91dc5b90dd9e07dcfa4a252ff083e" target='_blank'>
              Multi-modal image analysis for large scale cancer tissue studies within IMMUcan
              </a>
            </td>
          <td>
            Nils Eling, J. Dorier, S. Rusakiewicz, Robin Liechti, P. Devanand, Michelle Daniel, J. Windhager, Bruno Palau Fernandez, Sophie Déglise, Lucie Despland, Abdelkader Benyagoub, Alexander Loboda, D. Sandkuijl, Nikesh Parsotam, Henoch S Hong, Marie Morfouace, Nicolas Guex, G. Coukos, B. Bodenmiller, S. Tissot, D. Schulz
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Small bowel adenocarcinoma (SBA) is a rare malignancy marked with a poor prognosis. The cellular and proteomic heterogeneity within the tumor immune microenvironment (TIME) of SBA is a likely driver of prognosis, disease progression and response to therapy. We have addressed a major gap in knowledge of the TIME in SBA using highly multiplexed, protein imaging of the SBA tumor-immune ecosystem generating a comprehensive, single-cell level, spatial, proteomic atlas of TIME in > 600,000 cells from 136 tumor and matched normal samples from clinically and genomically annotated SBA patients (N=37). The SBA TIME Atlas informs on spatial distribution and interactions of tumor-intrinsic processes, diverse immune cell types, immune checkpoints, and vascularization. Enrichment of proliferating epithelial tissue, stem cells, and likely pro-tumor immune signatures in the tumor niche is contrasted by the representation of naïve and early-effector T-cells in the epithelial compartments of adjacent normal tissues. Epithelial-stem-immune cell spatial architectures within tumor and matched-normal niches were strongly associated with patient survival, suggesting malignancy is driven by spatial architecture beyond the tumor microenvironment. The blueprints of therapeutically actionable immune checkpoints at the interface between epithelial and microenvironmental T-cells as well as macrophages have established a guideline for precision immunotherapies tailored to the TIME composition in SBA. We expect that this SBA atlas will contribute to a deeper understanding of the immune contexture in this rare disease as well as other gastrointestinal cancers and help guide future precision immune-oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2886703d3c5dcc191767cc197fd5d1a7d169072" target='_blank'>
              A spatially resolved single cell proteomic atlas of Small Bowel Adenocarcinoma
              </a>
            </td>
          <td>
            Z. Dereli, B. Bozorgui, M. Sanchez, Nicholas Hornstein, Guillaume Thibault, Huamin Wang, Gordon B. Mills, John N. Weinstein, M. Overman, Anil Korkut
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions driving progression and pathobiology. How these spatial patterns and interactions evolve with treatment remains unclear. Here, we apply imaging mass cytometry to analyse protein-level changes in paired pre- and post-treatment GBM samples from five patients. We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes. These findings provide insight into treatment-induced shifts in GBM’s cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets. Key points Spatial organisation in primary GBM consist of layers driven by the presence of hypoxia The layers in recurrent GBM appear are driven more by the presence of reactive astrocytes Increased cellular cross-talk in recurrent GBM presents novel therapeutic targets">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3061625679fbc2f861a2180efc0a648894c17487" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment
              </a>
            </td>
          <td>
            Shoaib Ajaib, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F. Stead
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-cell spatial transcriptomics reveals how cells organize in healthy and diseased tissues. From these data, tissue segmentation analysis defines discrete compartments that organize cells into functional multicellular units. Existing methods for automated tissue segmentation rely on spatial smoothing to define spatially coherent regions but often blur the boundaries between adjacent tissue compartments. We describe Tessera, an algorithm that approaches tissue segmentation through a novel approach, dividing the tissue into small multicellular tiles whose edges track with natural tissue boundaries. Tessera achieves this by incorporating successful tools from edge-preserving smoothing, topological data analysis, and morphology-aware agglomerative spatial clustering. We show that Tessera identifies a range of known anatomical structures, in healthy mouse brain and human lymph nodes, and novel disease-associated niches, in human brain and in lung cancer. Tessera is a general-purpose tool that returns spatially coherent spatial structures with accurate boundaries across a range of spatial transcriptomics and proteomics technologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfead60198d5f27b8c18dbe0ea5203b263942c83" target='_blank'>
              Accurate tiling of spatial single-cell data with Tessera
              </a>
            </td>
          <td>
            Daniel J. Stein, Miles Tran, I. Korsunsky
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="To date, few studies have detailed the tumor microenvironment (TME) of Ewing sarcoma (EwS). The TME has a vital role in cancer survival and progression with implications in drug resistance and immune escape. By performing spatially resolved transcriptomic analysis of primary treatment-naïve EwS samples, we discovered greater stromal enrichment in localized EwS tumors compared to metastatic EwS tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumorigenic role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling, suggesting a potential immune-evasive mechanism and immunotherapy target. Our findings provide insight into tumor cell/stromal cell interactions in EwS and serve as a valuable resource for further investigations in the tumor immune microenvironment of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245c7e47704099e340bfbb4047050883a6a92e6d" target='_blank'>
              Tumor-associated stroma shapes the spatial tumor immune microenvironment of primary Ewing sarcomas
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer characterized by profound desmoplasia and cellular heterogeneity, which cannot be fully resolved using traditional bulk sequencing approaches. To understand the contribution of this heterogeneity to PDAC biology, we analyzed a large cohort of primary human PDAC samples (n = 62), profiling 443,451 single cells and 53,236 spatial transcriptomic spots using a combined single-cell RNA sequencing and spatial transcriptomics approach. Our analysis revealed significant intratumoral heterogeneity, with multiple genetically distinct neoplastic clones co-existing within individual tumors. These clones exhibited diverse transcriptional states and subtype profiles, challenging the traditional binary classification of PDAC into basal and classical subtypes; instead, our findings support a transcriptional continuum influenced by clonal evolution and spatial organization. Additionally, these clones each interacted uniquely with surrounding cell types in the tumor microenvironment. Phylogenetic analysis uncovered a rare but consistent classical-to-basal clonal transition associated with MYC amplification and immune response depletion, which were validated experimentally, suggesting a mechanism driving the emergence of a more aggressive basal clonal phenotype. Spatial analyses further revealed dispersed clones enriched for epithelial-to-mesenchymal transition (EMT) activity and immune suppression, correlating with metastatic potential and colonization of lymph node niches. These dispersed clones tended to transition toward a basal phenotype, contributing to disease progression. Our findings highlight the critical role of clonal diversity, transcriptional plasticity, and TME interactions in shaping human PDAC biology. This work provides new insights into the molecular and spatial heterogeneity of PDAC and offers potential avenues for therapeutic intervention targeting clonal evolution and the mechanisms driving metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Spatial profiling of tissues promises to elucidate tumor-microenvironment interactions and generate prognostic and predictive biomarkers. We analyzed single-cell, spatial data from three multiplex imaging technologies: cyclic immunofluorescence (CycIF) data we generated from 102 breast cancer patients with clinical follow-up, and publicly available imaging mass cytometry and multiplex ion-beam imaging datasets. Similar single-cell phenotyping results across imaging platforms enabled combined analysis of epithelial phenotypes to delineate prognostic subtypes among estrogen-receptor positive (ER+) patients. We utilized discovery and validation cohorts to identify biomarkers with prognostic value. Increased lymphocyte infiltration was independently associated with longer survival in triple-negative (TN) and high-proliferation ER+ breast tumors. An assessment of ten spatial analysis methods revealed robust spatial biomarkers. In ER+ disease, quiescent stromal cells close to tumor were abundant in good prognosis tumors, while tumor cell neighborhoods containing mixed fibroblast phenotypes were enriched in poor prognosis tumors. In TN disease, macrophage/tumor and B/T lymphocyte neighbors were enriched and lymphocytes were dispersed in good prognosis tumors, while tumor cell neighborhoods containing vimentin-positive fibroblasts were enriched in poor prognosis tumors. In conclusion, we generated comparable single-cell spatial proteomic data from several clinical cohorts to enable prognostic spatial biomarker identification and validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e86a33275ed2b41ff7e598b0d0e6585ea48b22" target='_blank'>
              Highly multiplexed imaging reveals prognostic immune and stromal spatial biomarkers in breast cancer.
              </a>
            </td>
          <td>
            Jennifer Eng, Elmar Bucher, Zhi Hu, Cameron R Walker, Tyler Risom, Michael Angelo, Paula I. Gonzalez-Ericsson, M. E. Sanders, A. B. Chakravarthy, Jennifer A Pietenpol, Summer L. Gibbs, Rosalie C. Sears, Koei Chin
          </td>
          <td>2025-01-14</td>
          <td>JCI insight</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC), accounting for 90% of pancreatic neoplasms, is characterized by its poor prognosis, with a 5-year survival rate of only 12%. Most patients are diagnosed with metastatic or locally advanced disease, leaving only 15% eligible for curative resection. PDAC exhibits resistance to chemotherapy, targeted therapies, and immunotherapy, largely due to its highly heterogeneous tumor microenvironment (TME). In this study, we performed an integrative analysis of publicly available scRNA, spatial transcriptomics, and bulk RNA sequencing datasets to investigate the influence of TME composition and tumor architecture on PDAC progression, treatment response, and clinical outcomes. We identified TME subtypes with distinct cellular compositions, functional signatures, and immunomodulatory cell-cell interactions. Spatially distinct cellular niches and gene modules revealed heterogeneity across primary tumors and metastatic lesions. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications. These findings underscore the importance of integrating multi-omics approaches to unravel the complexity of the PDAC TME and highlight its potential to inform therapeutic strategies and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768379a4460d13ebf34f45a5af50c7d930ab8e2" target='_blank'>
              Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel F. Pozo de Mattos P., Marvin Paulo Lins, Julia Fontoura, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The impact of cancer driving mutations in regulating immunosurveillance throughout tumor development remains poorly understood. To better understand the contribution of tumor genotype to immunosurveillance, we generated and validated lentiviral vectors that create an epi-allelic series of increasingly immunogenic neoantigens. This vector system is compatible with autochthonous Cre-regulated cancer models, CRISPR/Cas9-mediated somatic genome editing, and tumor barcoding. Here, we show that in the context of KRAS-driven lung cancer and strong neoantigen expression, tumor suppressor genotype dictates the degree of immune cell recruitment, positive selection of tumors with neoantigen silencing, and tumor outgrowth. By quantifying the impact of 11 commonly inactivated tumor suppressor genes on tumor growth across neoantigenic contexts, we show that the growth promoting effects of tumor suppressor gene inactivation correlate with increasing sensitivity to immunosurveillance. Importantly, specific genotypes dramatically increase or decrease sensitivity to immunosurveillance independently of their growth promoting effects. We propose a model of immunoediting in which tumor suppressor gene inactivation works in tandem with neoantigen expression to shape tumor immunosurveillance and immunoediting such that the same neoantigens uniquely modulate tumor immunoediting depending on the genetic context. One Sentence Summary Here we uncover an under-appreciated role for tumor suppressor gene inactivation in shaping immunoediting upon neoantigen expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a2622766d9e8241bb1972eff5c441baddd14069" target='_blank'>
              Tumor suppressor genotype influences the extent and mode of immunosurveillance in lung cancer
              </a>
            </td>
          <td>
            Keren M. Adler, Haiqing Xu, Amy C. Gladstein, Valerie M. Irizarry-Negron, Maggie R. Robertson, Katherine R. Doerig, D. Petrov, M. Winslow, D. Feldser
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Thyroid cancer progression from curable well-differentiated thyroid carcinoma to highly lethal anaplastic thyroid carcinoma is distinguished by tumor cell de-differentiation and recruitment of a robust stromal infiltrate. Combining an integrated thyroid cancer single-cell sequencing atlas with spatial transcriptomics and bulk RNA-sequencing, we define stromal cell subpopulations and tumor-stromal cross-talk occurring across the histologic and mutational spectrum of thyroid cancer. We identify distinct inflammatory and myofibroblastic cancer-associated fibroblast (iCAF and myCAF) populations and perivascular-like populations. The myCAF population is only found in malignant samples and is associated with tumor cell invasion, BRAFV600E mutation, lymph node metastasis, and disease progression. Tumor-adjacent myCAFs abut invasive tumor cells with a partial epithelial-to-mesenchymal phenotype. Tumor-distant iCAFs infiltrate inflammatory autoimmune thyroid lesions and anaplastic tumors. In summary, our study provides an integrated atlas of thyroid cancer fibroblast subtypes and spatial characterization at sites of tumor invasion and de-differentiation, defining the stromal reorganization central to disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b786943f0e4090faa02f89896c2ed59c657e0d0" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations
              </a>
            </td>
          <td>
            Matthew A. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey Caroland, Megan L. Tigue, H. Hartmann, J. Gallant, Courtney J. Phifer, Andres Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H Belcher, Ken S. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Dedifferentiation programs are commonly enacted during breast cancer progression to enhance tumor cell fitness. Increased cellular plasticity within the neoplastic compartment of tumors correlates with disease aggressiveness, often culminating in greater resistance to cytotoxic therapies or augmented metastatic potential. Here we report that subpopulations of dedifferentiated neoplastic breast epithelial cells express canonical leukocyte cell surface receptor proteins and have thus named this cellular program “immune mimicry.” We document neoplastic cells engaging in immune mimicry within public human breast tumor single-cell RNA-seq datasets, histopathological breast tumor specimens, breast cancer cell lines, as well as in murine transgenic and cell line-derived mammary cancer models. Immune-mimicked neoplastic cells harbor hallmarks of dedifferentiation and are enriched in treatment-resistant and high-grade breast tumors. We corroborated these observations in aggressive breast cancer cell lines where anti-proliferative cytotoxic chemotherapies drove epithelial cells toward immune mimicry. Moreover, in subsequent proof-of-concept studies, we demonstrate that expression of the CD69 leukocyte activation protein by neoplastic cells confers a proliferative advantage that facilitates early tumor growth and therefore conclude that neoplastic breast epithelial cells upregulating leukocyte surface receptors potentiate malignancy. Moving forward, neoplastic immune mimicry should be evaluated for prognostic utility in breast cancer to determine stratification potential for patients with increased risks of tumor recurrence, metastasis, and therapeutic resistance. One Sentence Summary Neoplastic breast epithelial cells express surface receptors canonically attributed to leukocytes and are associated with therapy resistance and aggressive tumor behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e76ec59281378bbfe3f3e345e03279fc2e07162" target='_blank'>
              Neoplastic immune mimicry potentiates breast tumor progression
              </a>
            </td>
          <td>
            Eric B. Berens, Sokchea Khou, Elaine Huang, Amber Hoffman, Briana Johnson, Nell Kirchberger, Shamilene Sivagnanam, Nicholas L. Calistri, Daniel Derrick, T. Liby, Ian C. Mclean, Aryn A. Alanizi, F. Ozmen, T. Y. Ozmen, Gordon B. Mills, E. S. Hwang, P. Schedin, Hugo Gonzalez, Z. Werb, Laura M. Heiser, L. Coussens
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Osteosarcoma is the most common malignant bone tumor in young patients and remains a significant clinical challenge, particularly in the context of metastatic disease. Despite extensive documentation of genomic alterations in osteosarcoma, studies detailing the immunosuppressive mechanisms within the metastatic osteosarcoma microenvironment are lacking. Our objective was to characterize the spatial transcriptional landscape of metastatic osteosarcoma to reveal these immunosuppressive mechanisms and identify promising therapeutic targets. Here, we performed spatial transcriptional profiling on a cohort of osteosarcoma pulmonary metastases from pediatric patients. We reveal a conserved spatial gene expression pattern resembling a foreign body granuloma, characterized by peripheral inflammatory signaling, fibrocollagenous encapsulation, lymphocyte exclusion, and peritumoral macrophage accumulation. We also show that the intratumoral microenvironment of these lesions lack inflammatory signaling. Additionally, we identified CXCR4 as an actionable immunomodulatory target that bridges both the intratumoral and extratumoral microenvironments and highlights the spatial heterogeneity and complexity of this pathway. Collectively, this study reveals that metastatic osteosarcoma specimens are comprised of multiple regionally distinct immunosuppressive microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2ee7e019ae5fcae794fe1bb25f818cd218fc98" target='_blank'>
              Spatial profiling identifies regionally distinct microenvironments and targetable immunosuppressive mechanisms in pediatric osteosarcoma pulmonary metastases
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice Pereira, Douglass W Morris-II, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael Kelly, Amy K LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) features a complex tumor microenvironment (TME) that significantly influences patient outcomes. Understanding the TME’s cellular composition and interactions is crucial for identifying therapeutic targets to improve treatment. Methods We performed an integrative analysis combining 88 single-cell RNA sequencing (scRNA-seq) samples with 187,520 cells, 20 Visium spatial transcriptomics (ST-seq) samples with 67,933 spots, and 1383 bulk RNA-seq samples and 2 Xenium high-resolution ST-seq samples with 307,679 cells, to delineate and characterize distinct subpopulations of fibroblasts, macrophages, T/NK cells, and B/plasma cells within the PDAC microenvironment. Correlations among major cell types across 12 PDAC datasets were assessed through gene set variation analysis (GSVA). Kaplan-Meier survival analysis was used to evaluate the prognostic significance of cell subtypes. Furthermore, multiplex immunohistochemistry (mIHC) and the Xenium platform was utilized to validate cellular interactions in human PDAC tissues at both protein and RNA levels. Results Six fibroblast subtypes and eight macrophage/monocyte subtypes were identified. POSTN high fibroblasts and SPP1 high macrophages highly infiltrated tumor tissues and were associated with poor prognosis. Most immune cell subtypes mediate adverse prognoses, except for CCL4 high CD8+ T EFF cells and IGHG1 high plasma cells, which are linked to favorable outcomes. ST-seq revealed spatial colocalization of POSTN high Fibro and SPP1 high Macro cells, as well as CCL4 high CD8+ T EFF and IGHG1 high plasma cells. These findings were corroborated by mIHC and validated using Xenium spatial transcriptomics with single-cell resolution, which confirmed the expression and spatial proximity of these markers at both the gene and protein levels. Conclusion This integrative analysis of the PDAC TME underscores the prognostic importance of POSTN high fibroblasts and SPP1 high macrophages, while also highlighting the protective roles of CCL4 high CD8+ T EFF cells and IGHG1 high plasma cells. These insights could improve PDAC treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74fb81e29120e8309bf4f3e49e31706525768da7" target='_blank'>
              Integrating multi-modal transcriptomics identifies cellular subtypes with distinct roles in PDAC progression
              </a>
            </td>
          <td>
            Jun Wu, Tenghui Dai, Ziyue Li, Meng Pan, Wei Zhang, Hao Chen, Guansheng Zheng, Li Qiao, Qizhou Lian, Yang Liu, Jierong Chen
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="A key challenge in cancer research is to identify the secreted factors that contribute to tumor cell survival. Nowhere is this more evident than in Hodgkin lymphoma, where malignant Hodgkin Reed Sternberg (HRS) cells comprise only 1-5% of the tumor mass, the remainder being infiltrating immune cells that presumably are required for the survival of the HRS cells. Until now, there has been no way to characterize the complex Hodgkin lymphoma tumor microenvironment at genome scale. Here, we performed genome-wide transcriptional profiling with spatial and single-cell resolution. We show that the neighborhood surrounding HRS cells forms a distinct niche involving 31 immune and stromal cell types and is enriched in CD4+ T cells, myeloid and follicular dendritic cells, while being depleted of plasma cells. Moreover, we used machine learning to nominate ligand-receptor pairs enriched in the HRS cell niche. Specifically, we identified IL13 as a candidate survival factor. In support of this hypothesis, recombinant IL13 augmented the proliferation of HRS cells in vitro. In addition, genome-wide CRISPR/Cas9 loss-of-function studies across more than 1,000 human cancer cell lines showed that IL4R and IL13RA1, the heterodimeric partners that constitute the IL13 receptor, were uniquely required for the survival of HRS cells. Moreover, monoclonal antibodies targeting either IL4R or IL13R phenocopied the genetic loss of function studies. IL13-targeting antibodies are already FDA-approved for atopic dermatitis, suggesting that clinical trials testing such agents should be explored in patients with Hodgkin lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0453e5a36d23742e3f6af668adec8ef2f3f547e8" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, Daniel Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail A Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson Weir, Orr Ashenberg, Geraldine Pinkus, Scott J Rodig, Caroline Uhler, Evan Z. Macosko, Margaret M. Shipp, A. Louissaint, Fei Chen, Todd R. Golub
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Spatial transcriptomics aims to elucidate cell coordination within biological tissues by linking the state of the cell with its local tissue microenvironment. Imaging-based assays are particularly promising for exploring such interdependencies, as they can resolve molecular and cellular features with subcellular resolution in three dimensions. Quantification and analysis of cellular state in such data, however, ultimately depends on the ability to recognize which molecules belong to each cell. Despite computational and experimental progress, this cell segmentation task remains challenging. Here we re-analyze data from multiple tissues and platforms and find that segmentation errors currently confound most downstream analysis of cellular state, including analysis of differential expression, inference of neighboring cell influence, and ligand-receptor interactions. The extent to which mis-segmented molecules impact the results can be striking, often dominating the set of top hits. We show that factorization of molecular neighborhoods can be effective at isolating such molecular admixtures and minimizing their impact on downstream analysis, analogous to doublet filtering of scRNA-seq data. As applications of spatial transcriptomics assays become more widespread, we expect corrections for the confounding effect of segmentation errors to become increasingly important for being able to resolve molecular mechanisms of tissue biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca4acc410c4c078dfd09e8229953bf61e6a7e0c5" target='_blank'>
              Impact of Segmentation Errors in Analysis of Spatial Transcriptomics Data
              </a>
            </td>
          <td>
            Jonathan Mitchel, Teng Gao, Eli Cole, Viktor Petukhov, P. V. Kharchenko
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Understanding cell-cell communication and its dependence on spatial organization is critical for unraveling tissue complexity and organ function. This study integrates single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics (ST) to systematically assess how spatial niches influence gene expression and intercellular communication. Using breast cancer, brain cortex, and heart datasets, our analyses reveal limited global transcriptional changes in cells depending on their spatial microenvironment, with differential gene expression observed in less than half the samples explored. Moreover, cell-cell communication predictions, derived from ligand-receptor pairs, exhibit minimal correlation with spatial colocalization of cell types. Overall, our study underscores the limitations of using scRNA-seq data to capture niche-specific molecular interactions, even when spatial information is leveraged, and it highlights the need for novel strategies to refine our understanding of intercellular communication dynamics at molecular level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790c5632749fda7ad54ef1434d0ac14122b25196" target='_blank'>
              Systematic assessment of microenvironment-dependent transcriptional patterns and intercellular communication
              </a>
            </td>
          <td>
            Elena Pareja-Lorente, Patrick Aloy
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Malignant gliomas follow two distinct natural histories: de novo high grade tumors such as glioblastoma, or lower grade tumors with a propensity to transform into high grade disease. Despite differences in tumor genotype, both entities converge on a common histologically aggressive phenotype, and the basis for this progression is unknown. Glioma associated macrophages (GAM) have been implicated in this process, however GAMs are ontologically and transcriptionally diverse, rendering isolation of pathogenic subpopulations challenging. Since macrophage contextual gene programs are orchestrated by transcription factors acting on cis-acting promoters and enhancers in gene regulatory networks (GRN), we hypothesized that functional populations of GAMs can be resolved through GRN inference. Here we show via parallel single cell RNA and ATAC sequencing that a subpopulation of human GAMs can be defined by a GRN centered around the Activator Protein-1 transcription factor FOSL2 preferentially enriched in high grade tumors. Using this GRN we nominate ANXA1 and HMOX1 as surrogate cell surface markers for activation, thus permitting prospective isolation and functional validation in human GAMs. These cells, termed malignancy associated GAMs (mGAMs) are pro-invasive, pro-angiogenic, pro-proliferative, possess intact antigen presentation but skew T-cells towards a CD4+FOXP3+ phenotype under hypoxia. Ontologically, mGAMs share somatic mitochondrial mutations with peripheral blood monocytes, and their presence correlates with high grade disease irrespective of underlying tumor mutation status. Furthermore, spatio-temporally mGAMs occupy distinct metabolic niches; mGAMs directly induce proliferation and mesenchymal transition of low grade glioma cells and accelerate tumor growth in vivo upon co-culture. Finally mGAMs are preferentially enriched in patients with newly transformed regions in human gliomas, supporting the view that mGAMs play a pivotal role in glioma progression and may represent a plausible therapeutic target in human high-grade glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/190acfc8ea743ab5360d22a148d803b01bb31afc" target='_blank'>
              A pathogenic subpopulation of human glioma associated macrophages linked to glioma progression
              </a>
            </td>
          <td>
            Kenny Kwok Hei Yu, Z. Abou-Mrad, Kristof Törkenczy, Isabell Schulze, Jennifer Gantchev, Gerard Baquer, Kelsey Hopland, Evan D. Bander, Umberto Tosi, Cameron Brennan, Nelson S. Moss, Pierre-Jacques Hamard, R. Koche, Caleb Lareau, Nathalie Y.R. Agar, T. Merghoub, Viviane Tabar
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="The intra- and inter-tumoral heterogeneity of glioblastoma represents a significant therapeutic challenge, as well as difficulty in generating reliable models for in vitro studies. Historical 2D adherent cell lines do not recapitulate this complexity, whereas both patient-derived neurospheres (PDN) and organoids (PDO) demonstrate intra-tumoral heterogeneity. Here, we quantify the tumor cell composition from matched models established from the same primary tumor using a series of multi-omic interrogations. We find that both patient-derived models recapitulate the genomic, epigenomic and tumor cell heterogeneity of the primary tissue. Furthermore, single-nuclei RNA sequencing revealed a subset of organoids containing small numbers of non-malignant cells from neuron and immune cell lineages. Harnessing the intra-tumoral heterogeneity of PDN models, we reveal the impact of temozolomide chemotherapy on individual cell states, altering composition of tumors over time in response to treatment. Our data confirms that both patient-derived models recapitulate patient intra-tumoral heterogeneity providing a platform for tumor cell state refined therapeutic studies. Key Points Generation of matched patient-derived neurosphere and organoid models from resected GBM tissue Neurosphere models exhibit greater proliferative signatures Both patient-derived models recapitulate genomic and epigenomic features of the primary tissue Single-nuclei RNA sequencing reveals both models recapitulate intra-tumoral heterogeneity of the primary tissue Neurosphere models enable interrogation of therapeutic responses in the context of heterogeneity Importance of study Patient derived models can be powerful tools when they faithfully recapitulate the tumor tissue from which they are derived. In glioblastoma, patient derived neurospheres (PDN) and organoids (PDO) have both been used in studies, however the differences between these models and the recapitulation of patient heterogeneity remain to be fully characterized. To address this, we performed multi-omic profiling of PDN and PDO models generated from the same tumor tissue. We find that that across a range of data modalities, both model systems exhibit a high level of resemblance to tissue, and critically, maintain heterogeneity and tumor cell composition. The importance of modeling heterogeneity was demonstrated in PDN models, where temozolomide treatment specifically alters the abundance of MES-like and AC-like tumor cells. Our findings demonstrate that neurosphere and organoid models effectively preserve cellular heterogeneity, genomic alterations, methylation signatures and transcriptomic features, both highly suitable to model glioblastoma’s complex cellular landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c3caa2d109062b5ff4f7f9f68c99c0d6be111c" target='_blank'>
              Three-dimensional patient-derived models of glioblastoma retain intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            Zachery Moore, Claire Storey, Daniel V. Brown, A. Valkovic, Montana Spiteri, Jasmine F. Pignatelli, Shannon J Oliver, Alana Fakhri, Kate Drummond, Seth Malinowski, Keith L. Ligon, Oliver M. Sieber, J. Whittle, S. Freytag, Sarah A. Best
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors (TKIs) provide therapeutic benefit in EGFR mutation–positive non–small cell lung cancer, but some individuals develop early resistance. We performed spatial transcriptomics analysis of pre- and posttreatment tumor samples from the same patients to explore the underlying mechanisms of such early resistance. The proportion and activation of fibroblasts increased in association with the development of early resistance, whereas a distinct tumor cell cluster showed activation of tumor necrosis factor–α signaling via the NF-κB pathway even before treatment. Also in the early resistance sample, specific tumor cell clusters interacted with immune and stromal cells. Immature tertiary lymphoid structures (TLSs) were enriched in the early resistance sample, whereas mature TLSs were observed in the long-term response sample. These findings implicate tumor heterogeneity and an inflammatory tumor microenvironment in early EGFR-TKI resistance, providing insight into potential therapeutic strategies to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59cd540165dee7cba115de4f78c74d67fa9e26e" target='_blank'>
              Spatial dynamics of the tumor microenvironment associated with emerging resistance to targeted therapy in EGFR-mutated non–small cell lung cancer
              </a>
            </td>
          <td>
            Satoshi Nakamura, Daisuke Shibahara, Kentaro Tanaka, Yasuyuki Kishikawa, Mikiko Hashisako, K. Nakatomi, Noriaki Nakagaki, Mikihiro Kohno, Koichi Azuma, R. Ibusuki, K. Otsubo, Y. Yoneshima, E. Iwama, Yoshinao Oda, I. Okamoto
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 There is a growing appreciation for the role that the spatial organization of immune cells within the tumor microenvironment (TME) plays in shaping cellular function and response to therapy. However, the majority of studies have been limited to analyzing immune responses at a single time point, and so, little is known about the spatiotemporal changes that underlie successful tumor rejection after cancer immunotherapy. In this study, we used the CODEX multiplex imaging system with a 32-plex antibody panel to longitudinally characterize the spatial changes in the lymphoid and myeloid cell populations that were previously identified by scRNA-Seq and CyTOF to be associated with complete tumor rejection in mice treated with the combination of α-PD-1 and α-CTLA-4 (combo ICT) in our well-characterized mouse T3 MCA sarcoma model. T3 tumor-bearing mice were treated with control antibody or combo ICT, tumors were harvested at multiple time points (day 7, 9, 10, 11 and 13), fresh frozen, sectioned, and subjected to CODEX multiplex imaging. A total of 42 tumors were imaged with the full range of the tumor-immune boundary. A hierarchical cell clustering approach developed in-house was used to profile 13,051,156 cells resulting in the identification of 13 distinct cell types. Our results show that after ICT the decrease in T3 cells in the tumor tissue starts on day 10 and culminates in the complete elimination of tumor cells by day 13. This immune-mediated response is driven by an increase in CD4+ T cells, which allows them to more frequently co-localize with CD8+ T cells and type 1 conventional dendritic cells (cDC1), forming three-cell clusters with increased cell contacts. This change in cell-to-cell interactions is followed by a broader reorganization of the TME as noted by the expansion of a lymphoid-rich cellular neighborhood (CN) that harbors the majority of CD4+ and CD8+ T cells. Interestingly, this CN becomes a hub for T cell effector functions, characterized by an enrichment of GZM-B+ and KI67+ T cells, and exhibits spatial dynamics throughout tumor regression. While both GZM-B+ and KI67+ T cells are observed in proximity to blood vessels on days 7 and 9, they begin to spatially separate into distinct regions after day 10, indicating that cytotoxic T cells move towards areas with tumor cells, while proliferating T cells are more frequently observed at the periphery. This dynamic behavior was further confirmed by assessing the communication rules between CNs, which showed that the lymphoid-rich CN acts by eliminating layers of tumor cells at the tumor boundary and by modulating nearby myeloid populations. Additionally, integration of CODEX dataset with scRNA-seq revealed CD4+ T cells produce Interferon-gamma and a change in the molecular chemotactic pathways orchestrating lymphoid and myeloid communication. Overall, this study provides a detailed spatial view of the cellular events that result in successful ICT and allows us to propose a computational model of the actions of T cells in coordinating the antitumor response after α-CTLA-4/α-PD1.
 Citation Format: Ruan FV Medrano, Vladimir Sukhov, Maxim Artyomov, Robert Schreiber. Spatiotemporal analysis during tumor rejection reveals coordinated cellular dynamics underlying successful immune-checkpoint therapy [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B048.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8b6d1c2313c8e86b7cbcc29c94f8060fb9e7af4" target='_blank'>
              Abstract B048: Spatiotemporal analysis during tumor rejection reveals coordinated cellular dynamics underlying successful immune-checkpoint therapy
              </a>
            </td>
          <td>
            Ruan FV Medrano, Vladimir Sukhov, Maxim N Artyomov, Robert Schreiber
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background & Aims Epithelial tumors generally resemble the cellular architecture of their tissue of origin. However, this link remains largely unexplored in the pancreas. Methods Using Nanostring GeoMx DSP®, Resolve Molecular Cartography® and Nanostring CosMx®, and integration with single cell RNAseq datasets, we mapped the human pancreatic ductal epithelium in non-neoplastic pancreas, and compared it to pancreatic cancer subtypes. Results Groups of Keratin-5+ cells among the Pan-Cytokeratin+ cells in the duct have a gene signature reminiscent of stem cells and (supra)basal cells from other tissues. In spatial analysis at single cell resolution, the pancreatic duct manifests as a stratified epithelium comprising a basal and four luminal populations; In large ducts, KRT5+ basal (BAS) cells express ΔNp63 while KRT5+ luminal (LUM)-B cells reside supra-basally and are distinct from the common KRT5- LUM-A cells. LUM-C and LUM-D cells pertain to intercalated ducts and ductal glands, respectively. LUM-A and -C cells express gel-forming mucins while LUM-B cells have membrane-bound MUC4 and MUC16. In cancer, BAS and LUM-B signatures associate with the basal-like pancreatic ductal adenocarcinoma (PDAC) and correlate with lower survival but exhibit a mixed spatial pattern with a diffracted gene signature. In contrast, adenosquamous cancers of the pancreas (ASCP) preserve the normal spatially unmixed identity of LUM-B cells and BAS cells that is regulated by ΔNp63. Next to ΔNp63, conserved drug targets were identified for both populations. Conclusion This study offers a refined pancreatic tumor classification based on the native ductal architecture, with better preservation of the LUM-B and BAS cell population identity in ASCP than in PDAC, including the conserved drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a2f6f8e9909be212c547b03063b014e38a8855" target='_blank'>
              Spatial map of native duct cell populations in human pancreas and their representation in pancreatic cancers
              </a>
            </td>
          <td>
            JL Van den Bossche, M. Van der Vliet, E. Michiels, O. A. Senar, Z. Madran, K. Coolens, S. Van Lint, M. Nacher, T. Arsenijevic, N. Messaoudi, P. Lefesvre, E. Montanya, M. Rovira, J. Van Laethem, J. Baldan, I. Houbracken, I. Rooman
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Melanoma, a highly metastatic skin cancer, exhibits variations in prognosis and response to therapy based on the site of metastasis. Despite the success of immunotherapy and targeted therapies in melanoma, over half of metastatic melanoma patients will experience disease progression due to therapy resistance. The heterogeneity and plasticity of melanoma cells contribute to metastatic dissemination and therapy resistance. We aim to determine whether distinct clones and/or their transcriptional cell states can predict the formation of tumours in various organs. Lineage tracing and clonal heterogeneity were evaluated using the state-of-the-art technology, SPLINTR (Single-cell Profiling and LINeage Tracing), enabling us to match a cells evolution with changes in transcriptional states. To identify melanoma clones across different metastatic sites, specifically subcutaneous, lungs and brain, C57BL/6 mice were injected subcutaneously, intravenously or intracranially with the same pool of cells of barcoded luciferase expressing YUMMER1.7PV1 murine melanoma cells. Transduction conditions ensured that one DNA barcode integrated into cell genomes at one barcode per cell, serving as a lineage tag. Bioluminescence imaging was performed once a week to monitor the tumour growth of mice injected intravenously and intracranially. Subcutaneous tumours were measured by calliper. Mice were euthanized before ethical endpoints. Tumors were harvested and single cell analysis as well as DNA sequencing was performed to identify barcodes expressed by the tumor cells across different tumor-sites. Barcode analysis was conducted both before cell implantation and at tumor harvest. Clonality exhibited significant variability between sites. Single-cell RNA sequencing will unveil states associated with dominant clones present at different metastatic sites. This information will provide a new understanding of melanoma transcriptional phenotype behaviours and biological characteristics at different metastatic sites and in response to therapy.
 Citation Format: Veronica Aedo-Lopez. Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A120.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407e0ecf5209e24e1ad27b286f77ee0657cbbbda" target='_blank'>
              Abstract A120: Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis
              </a>
            </td>
          <td>
            Veronica Aedo-Lopez
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colonic stem cells have a key role in the continuous regeneration of a healthy intestinal epithelium. Despite considerable advances in single-cell omics technologies, the transcriptional heterogeneity of rare cell types such as colonic stem cells, as well as their functional states and niche-specific behaviors, remain poorly characterised. In this study, we leverage a meta-analysis of scRNA-seq and spatial transcriptomic datasets to comprehensively map the heterogeneity of colonic stem cells. We identify multiple, previously underappreciated stem cell states, including distinct quiescent subtypes marked by CDKN1A (P21), CDKN1B (P27), and CDKN1C (P57), proliferative populations defined by MKI67 (Ki67) and LRIG1, and a lineage-committed intermediate subpopulation expressing MUC2. Strikingly, we find that these states can be robustly identified solely by their extracellular matrix (ECM) gene expression signatures, revealing ECM composition as a critical determinant of stem cell identity. Notably, LAMA1 expression is highly specific to the P57+ quiescent population, linking laminin-mediated microenvironments to the active maintenance of deep quiescence, consistent with our recent findings associating LAMA1 with quiescent cell survival. By applying these ECM gene signatures, we delineate discrete “micro-niches” of quiescent stem cells in healthy tissue and provide evidence that analogous states persist in the colorectal cancer (CRC) environment. Extending our approach to an unrelated tissue, the pancreas, we detect parallel quiescent cell subtypes, illustrating the broader applicability of ECM-based signatures. Taken together, our findings redefine the concept of stem cell heterogeneity in the colon, establish ECM-driven gene signatures as a powerful tool for characterizing stem cell states, and offer new perspectives on the niche-dependent regulation of both healthy and cancerous stem cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b79223cf63c16f2be2c9879ace337f25901bae" target='_blank'>
              ECM Signatures Reveal Quiescent Stem Cell Diversity in the Colonic Niche
              </a>
            </td>
          <td>
            Séamus E. Hickey, M. Andreatta, Christina Enright, Emmanuel Boucrot, Patrick Kiely, Siobhán B. Cashman, Saintiago J. Carmona, Kieran McGourty
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Uncontrolled proliferation is a hallmark of cancer, yet tumour cells can enter G0 arrest by halting the cell cycle reversibly (quiescence) or irreversibly (senescence) to survive under stress and in hostile tumour microenvironments (TME). G0 arrested cells contribute to drug tolerance, metastasis and recurrence, but their identification remains challenging due to their rarity and elusive regulatory pathways. Here, we quantify G0 arrest and proliferation decisions in single-cell and spatially profiled breast primary tumours to unveil adaptive responses driving immune compartmentalisation. We identify a genomically-constrained and prolonged G0 arrest state resembling that of dormant precursors of cancer progression. This state featured adaptive transcriptional reprogramming, including unfolded protein response stress, reduced mitochondrial translation and epithelial-mesenchymal plasticity enabled by semaphorin signalling. Spatial transcriptomics revealed G0 arrest pockets encapsulated by APOE+ lipid-associated macrophages, myofibroblastic CAFs and immature perivascular cells, suggestive of an immunosuppressive niche contrasting with cytotoxic environments in proliferative hotspots and displaying distinct drug sensitivities. To facilitate future research, we provide a foundation model capturing G0 arrest and proliferation with 94% accuracy in single cell data, available at https://github.com/secrierlab/G0-LM. Our findings provide new insights into the spatial organisation of cell cycle arrest in breast cancer, highlighting the role of G0 states in tumour heterogeneity and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5d540e388314aece2599568858b5b0ad8004482" target='_blank'>
              Balancing tumour proliferation and sustained cell cycle arrest through proteostasis remodelling drives immune niche compartmentalisation in breast cancer
              </a>
            </td>
          <td>
            Cenk Celik, Eloise Withnell, Shi Pan, Tooki Chu, Johnathan Labbadia, M. Secrier
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Microbes have been found in various tumors and the research of tumor microbiome has been garnering increased attention. However, it remains challenge to investigate the microbiome in cancer at spatial resolution. The rapid advent of spatially resolved transcriptomics techniques has given rise to map transcripts at single-cell resolution in various types of cancer. Here, we constructed a comprehensive spatial meta-transcriptome resource by manually curating 203 fresh frozen (FF) slices from 20 cancers encompassing 334,253 spots and 1,908,646 cells. SMTdb (http://bio-bigdata.hrbmu.edu.cn/SMTdb/) was constructed to provide comprehensive insights into the abundance, distribution and enriched TME regions of 1218 microbiota in spatial tissue slices. SMTdb enables vast interactive data exploration of spatial distribution and expression of microbiota, host gene modules associated with certain microbiota and co-occurrence between microbiota and immune cells within tumor microenvironment. The atlas resource serves as a one-stop and time-effective platform to investigate the interactions among microbial ecosystems and hosts in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803a0dfe5a2e579dc8f160c9acd2466a2dcd43" target='_blank'>
              SMTdb: A comprehensive spatial meta-transcriptome resource in cancer
              </a>
            </td>
          <td>
            Weiwei Zhou, Qingyi Yang, Jiyu Guo, Si Li, Minghai Su, Feng Leng, Tingyu Rong, Jingyi Shi, Yueying Gao, Tiantongfei Jiang, Juan Xu, Yongsheng Li
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Bacterial colonization of cancer tumors is widespread. While the clinical significance of intratumor bacteria is widely investigated, the dynamics of tumor colonization remain underexplored. We monitored bacterial population dynamics in murine tumors after introducing genetically barcoded Escherichia coli cells by intravenous or intratumor injections. Barcode sequencing from resected tumors after intravenous injection suggested that narrow bottlenecks dominate infection and are followed by rapid, yet highly nonuniform growth. Within a day, bacteria reached a steady state and subsequently sustained load and clone diversity. Results from intratumor injections, circumventing the infection bottleneck, revealed that sizes of bacterial progenies followed a scale-free distribution matching Zipf’s law. Theory suggested that our observations are compatible with a growth model constrained by a local niche load, competition for a global resource, and noise. Our work provides the first dynamical model of tumor colonization and suggests that descriptive statistics may allow distinguishing genuine tumor microbiomes from contamination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ca2d699ba974ef22ac49a1f2990095215e2dd0" target='_blank'>
              Bacterial population dynamics during colonization of solid tumors
              </a>
            </td>
          <td>
            Serkan Sayin, Motasem ElGamel, Brittany Rosener, Michael Brehm, Andrew Mugler, Amir Mitchell
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cutaneous T-cell lymphoma (CTCL) remains a challenging disease due to its significant heterogeneity, therapy resistance, and relentless progression. Multi-omics technologies offer the potential to provide uniquely precise views of disease progression and response to therapy. We present here a comprehensive multi-omics view of CTCL clonal evolution, incorporating exome, whole genome, epigenome, bulk-, single cell (sc) VDJ-, and scRNA-sequencing of 114 clinically annotated serial skin, peripheral blood, and lymph node samples from 35 CTCL patients. We leveraged this extensive dataset to define the molecular underpinnings of CTCL progression in individual patients at single cell resolution with the goal of identifying clinically useful biomarkers and therapeutic targets. Our studies identified a large number of recurrent progression-associated clonal genomic alterations; we highlight mutation of CCR4, PI3K signaling, and PD-1 checkpoint pathways as evasion tactics deployed by malignant T cells. We also identified a gain of function mutation in STAT3 (D661Y) and demonstrated by CUT&RUN-seq that it enhances binding to transcription start sites of genes in Rho GTPase pathways, which we previously reported to have activated chromatin and increased expression in HDACi-resistant CTCL. These data provide further support for a previously unrecognized role for Rho GTPase pathway dysregulation in CTCL progression. A striking number of progression-associated mutations occurred in chromatin methylation modifiers, including EZH2, suggesting that EZH1/2 inhibition may also benefit patients with CTCL. Knowledge of these molecular changes should be leveraged for improved disease monitoring, biomarker-informed clinical trial design, and new therapeutic strategies in this challenging and incurable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam U. Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) tumors contain chaotic vasculature that limits immune surveillance and promotes early events in the metastatic cascade. However, current antiangiogenic therapies have failed in PDAC, and thus, it remains important to uncover mechanisms by which cancer cells signal to endothelial cells to increase angiogenesis. Our lab has shown that the tumor-intrinsic RNA-binding protein HuR (ELAVL1) plays an important role re-shaping the tumor microenvironment (TME) by regulating the stability and translation of cytokine encoding transcripts. Herein, we demonstrate that PDAC-intrinsic HuR influences endothelial cell function in the TME via extracellular vesicle (EV) signaling, an underexplored signaling axis in tumor progression. We found that HuR knockout (KO) tumors have impaired growth in an immunocompetent mouse model, and that administering purified wildtype (WT) EVs can increase tumor growth. Further, we observed that PDAC EVs contain HuR-dependent mRNA and protein cargoes relating to endothelial cell function and angiogenesis. Treatment of endothelial cells with HuR WT EVs strongly increased the expression of genes involved in barrier function and endothelial cell development, and directly increased their migratory and tube forming functions. In an immunocompetent orthotopic mouse model of PDAC, we showed that HuR increases endothelial cell presence and sprouting, while decreasing ICAM-1 expression. Importantly, we found utilizing a genetic EV reporter, that decreased ICAM-1 within WT tumors occurs in endothelial cells that have imported PDAC EVs, suggesting that this signaling axis is directly modulating endothelial cell behavior in vivo. Collectively, our data reveal a new role of HuR in EV signaling to endothelial cells, promoting angiogenesis while restricting endothelial cell leukocyte trafficking behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48dc9b4133295225413ef8a3cfd3cc3874847fd5" target='_blank'>
              Pancreatic cancer-intrinsic HuR regulates the pro-tumorigenic properties of extracellular vesicles
              </a>
            </td>
          <td>
            Jennifer M. Finan, Yifei Guo, Alexandra Q. Bartlett, Matthew Reyer, Kevin Hawthorne, Margaret Haerr, Hen Halamish, Olayinka Lamikanra, Valerie Calvert, Canping Chen, Zheng Xia, Emanuel F. Petricoin, Rosalie C. Sears, Katelyn T. Byrne, Jonathan R. Brody
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Triple-negative breast cancer is a prevalent breast cancer subtype with the lowest 5-year survival. Several factors contribute to its treatment response, but the inherent molecular and cellular tumor heterogeneity are increasingly acknowledged as crucial determinants. Methods Spatial transcriptomic profiling was performed on FFPE tissues from a retrospective, treatment-naive group of women with differential prognoses (17 with >15 years survival-good prognosis (GPx) and 15 with <3 years survival-poor prognosis (PPx)) using GeoMX® Digital Spatial Profiler. Regions of interest were segmented on pan-cytokeratin and analyzed for tumor and stromal components, probed using GeoMx human whole transcriptome atlas (WTA) panel. Data quality control, normalization, and differential analysis was performed in R using GeomxTools and linear mixed models. Additional analyses including cell-type deconvolution, spatial entropy, functional enrichment, TF-target / ligand-receptor analysis and convolution neural networks were employed to identify significant gene signatures contributing to differential prognosis. Results Here we report on the spatial and molecular heterogeneity underlying differential prognosis. We observe that the state of the epithelia and its microenvironment (TME) are transcriptionally distinct between the two groups. Invasive epithelia in GPx show a significant increase in immune transcripts with the TME exhibiting increased immune cell presence (via IF), while in PPx they are more metabolically and translationally active, with the TME being more mesenchymal/fibrotic. Specifically, pre-cancerous epithelia in PPx display a prescience of aggressiveness as evidenced by increased EMT-signaling. We identify distinct epithelial gene signatures for PPx and GPx, that can, with high accuracy, classify samples at the time of diagnosis and likely inform therapy. Conclusions To the best of our knowledge, this is the first study to leverage spatial transcriptomics for an in-depth delineation of the cellular and molecular underpinnings of differential prognosis in TNBC. Our study highlights the potential of spatial transcriptomics to not only uncover the molecular drivers of differential prognosis in TNBC but also to pave the way for precision diagnostics and tailored therapeutic strategies, transforming the clinical landscape for this aggressive breast cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31351362773ec3359f5dd1e2c1fcb9f73d1b49df" target='_blank'>
              Spatially distinct cellular and molecular landscapes define prognosis in triple negative breast cancer
              </a>
            </td>
          <td>
            Kavitha Mukund, Darya Veraksa, David Frankhouser, Lixin Yang, Jerneja Tomsic, Raju Pillai, Srijan Atti, Zahra Mesrizadeh, D. Schmolze, Xiao-Cheng Wu, Mary-Anne LeBlanc, Lucio Miele, Augusto Ochoa, Victoria L Seewaldt, Shankar Subramaniam
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell spatial omics analysis requires consideration of biological functions and mechanisms in a microenvironment. However, microenvironment analysis using bioinformatic methods is limited by the need to detect histological morphology and extend it to the surrounding area. In this study, we developed SpatialKNifeY (SKNY), an image-processing-based toolkit that detects spatial domains that potentially reflect histology and extends these domains to the microenvironment. Using spatial transcriptomic data from breast cancer, we applied the SKNY algorithm to identify tumor spatial domains, followed by clustering of the domains, trajectory estimation, and spatial extension to the tumor microenvironment (TME). The results of the trajectory estimation were consistent with the known mechanisms of cancer progression. We observed tumor vascularization and immunodeficiency at mid- and late-stage progression in TME. Furthermore, we applied the SKNY to integrate and cluster the spatial domains of 14 patients with metastatic colorectal cancer, and the clusters were divided based on the TME characteristics. In conclusion, the SKNY facilitates the determination of the functions and mechanisms in the microenvironment and cataloguing of the features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796429c918dab0f1b4d04958c50f7cf17d100c5e" target='_blank'>
              SpatialKNifeY (SKNY): Extending from spatial domain to surrounding area to identify microenvironment features with single-cell spatial omics data
              </a>
            </td>
          <td>
            Shunsuke A Sakai, Ryosuke Nomura, Satoi Nagasawa, SungGi Chi, Ayako Suzuki, Yutaka Suzuki, Mitsuho Imai, Y. Nakamura, Takayuki Yoshino, Shumpei Ishikawa, K. Tsuchihara, S. Kageyama, Riu Yamashita
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b78a9a8e3280a5c16b1080da5f7e2a241cd45990" target='_blank'>
              Diffusion Smart-seq3 of breast cancer spheroids to explore spatial tumor biology and test evolutionary principles of tumor heterogeneity
              </a>
            </td>
          <td>
            Antony Cougnoux, Loay Mahmoud, Per A Johnsson, Alper Eroglu, Louise Gsell, Jakob Rosenbauer, R. Sandberg, Jean Hausser
          </td>
          <td>2025-01-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) is the deadliest cancer worldwide. Therapeutic progress stagnate, highlighting the complexity to replicate NSCLC in preclinical models. Drug discovery studies rely mostly on cancer cells in two-dimension (2D), which poorly predict drug efficacy in patients. There is a growing interest in three-dimensional (3D) preclinical models, such as 3D spheroids, to better model tumor phenotype and improve therapeutic prediction. However, a comprehensive view of 3D culture methods impact on transcriptomes, epigenomes and pharmacological responses and their correlations to NSCLC tumors is still missing. Here, we demonstrate that NSCLC spheroids undergo time-dependent transcriptomic and epigenomic changes, which peak after 3 weeks of culture. While DNA methylome remained stable, chromatin methylation and acetylation marks gained features of advanced NSCLC in a time-dependent manner. Single-cell transcriptomic profiling of spheroids demonstrated that time of 3D culture improved the correlation to NSCLC tumors. Moreover, long-term culture of 3D spheroids increased drug screening predictability, by showing resistance to drugs that failed in NSCLC patients (such as HDAC inhibitors) while demonstrating novel pharmacological vulnerabilities and synergistic interactions (such as combination of PRMT and HDAC inhibitors). Strikingly, reverting 3D spheroids back to 2D culture rapidly reversed transcriptomic, epigenetic and pharmacological signatures acquired after 3 weeks of 3D culture, highlighting the critical impact of cell culture conditions on NSCLC phenotype. Collectively, our findings demonstrate that implementing a time-dependent maturation process into 3D spheroid culture induces chromatin and transcriptomic changes that enhance NSCLC preclinical modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f644ee3c95f246053a935ce676721b85861fe7f5" target='_blank'>
              Time-dependent chromatin maturation during 3D spheroid culture improves preclinical modeling of non-small cell lung cancer
              </a>
            </td>
          <td>
            Anaïs Darracq, Nicolas Sgarioto, Marielle Huot, Gabrielle McInnes, Antoine Meant, Alexandra Langford, Oscar D Villarreal, François Marois, Maxime Caron, P. Saint-Onge, S. Leclerc, G. Cagnone, Gregor Andelfinger, Stéphane Richard, Daniel Sinnett, Serge McGraw, Noël J-M Raynal
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The spatial dissimilarity method is a new statistical tool to uncover complex bivariate relationships in single cells and spatial transcriptomics data, thereby addressing unresolved issues such as alternative splicing and allele-specific gene expression. By applying our spatial dissimilarity analysis method on datasets of neurons, tumor, and normal cells, we identified thousands of alternatively spliced genes and instances of allele-specific gene expression. This is particularly evident in the analysis of quiescent and transitioning cell states, where our method reveals the nuanced gene expression dynamics associated with these states. Notably, our findings highlight how allele-specific genetic variants can provide insights into the subclone architecture of normal cells and cancer cells, offering a more comprehensive understanding of cellular heterogeneity and a new insight to understand gene function at specific cell types. The spatial dissimilarity analysis method deepens our comprehension of cellular complexity and gene expression dynamics during cell state transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4172a5da9c9bc681c952ca59d27b42c8796b34e" target='_blank'>
              Spatial dissimilarity analysis in single cell transcriptomics
              </a>
            </td>
          <td>
            Quan Shi, Karsten Kristiansen
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2410faffb025161d800cf5841a3c4049fde9d25" target='_blank'>
              Integration of single-cell and spatial transcriptomics reveals fibroblast subtypes in hepatocellular carcinoma: spatial distribution, differentiation trajectories, and therapeutic potential
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-02-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chronic inflammation is a well-established risk factor for cancer, but the underlying molecular mechanisms remain unclear. Using a mouse model of colitis, we demonstrate that colonic stem cells retain an epigenetic memory of inflammation following disease resolution, characterized by a cumulative gain of activator protein 1 (AP-1) transcription factor activity. Further, we develop SHARE-TRACE, a method that enables simultaneous profiling of gene expression, chromatin accessibility and clonal history in single cells, enabling high resolution tracking of epigenomic memory. This reveals that inflammatory memory is propagated cell-intrinsically and inherited through stem cell lineages, with certain clones demonstrating dramatically stronger memory than others. Finally, we show that colitis primes stem cells for amplified expression of regenerative gene programs following oncogenic mutation that accelerate tumor growth. This includes a subpopulation of tumors that have exceptionally high AP-1 activity and the additional upregulation of pro-oncogenic programs. Together, our findings provide a mechanistic link between chronic inflammation and malignancy, revealing how long-lived epigenetic alterations in regenerative tissues may contribute to disease susceptibility and suggesting potential therapeutic strategies to mitigate cancer risk in patients with chronic inflammatory conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f579a4d65bfde6d9ba50ff844febde4a92d36d5" target='_blank'>
              Clonal memory of colitis accumulates and promotes tumor growth
              </a>
            </td>
          <td>
            Surya Nagaraja, Lety Ojeda-Miron, Ruochi Zhang, Ena Oreskovic, Yan Hu, Daniel Zeve, Karina Sharma, Roni R. Hyman, Qiming Zhang, Andrew Castillo, David T. Breault, Ömer H Yilmaz, Jason D. Buenrostro
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor dissemination is increasingly recognized to begin early in tumor development. Although most of these early disseminated cells are cleared, some survive and persist below clinical detection, acting as reservoirs for metastatic relapse. Metastatic tumor cells often rely on interactions with local stromal cells to support their colonization. In this study, we propose that pericyte-tumor cell interactions promote dormancy induction in the early metastatic lung, enhancing disseminated tumor cell (DTC) persistence. Extravital imaging demonstrated that DTCs interact with pericytes upon extravasation into the lung. Co-culture experiments were used to assess DTC fate after pericyte contact and revealed that transient contact with pericytes reduced the proliferation of metastatic 4T1 breast cancer cells but had no effect on non-metastatic 67NR cells. In vivo, transient pericyte contact resulted in higher lung metastatic burden, driven by small, non-proliferative lesions (<6 cells), 10 days after intracardiac injection. These lesions exhibited reduced KI67 staining and EdU incorporation compared to those from monocultured cells. We further observed that primary lung pericytes transferred lyso-phospholipids (lyso-PLs) specifically to metastatic 4T1 cells through direct contact. Gene expression analysis indicated that transient pericyte contact activated pathways related to syncytium formation in metastatic cells. In normal physiology, pericytes act in a syncytium to regulate blood flow via mechanosensitive channels in response to blood pressure changes. We hypothesize that tumor cells exploit these mechanosensitive responses to trigger lyso-PL transfer from pericytes. Supporting this, calcium imaging showed higher calcium activity in pericytes co-cultured with 4T1 cells, and calcium channel inhibitors significantly reduced lyso-PL transfer. Pharmacological activation of pericyte calcium channels induced lyso-PL release, which was subsequently taken up by tumor cells. Conditioned medium from activated pericytes, containing free lyso-PLs, recapitulated the reduced proliferation observed in transient co-culture. Finally, we found our pericyte-induced dormancy signature to be associated with tumor dormancy and distant metastasis free survival latency in breast cancer patients. Together, these findings suggest that early DTCs may exploit pericyte signaling mechanisms to enter dormancy, facilitating their persistence at metastatic sites and contributing to future relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1fe686f97fcb46c56e3f6bfbef8e78fcca9cb41" target='_blank'>
              Pericyte-tumor crosstalk facilitates metastatic tumor cell latency through PIEZO1-activated lysophospholipid transfer
              </a>
            </td>
          <td>
            Tamara McErlain, Morgan J Glass, Elizabeth C McCulla, Caitlin A Madden, Lauren E Ziemer, Kirsten E Overdahl, Alan K. Jarmusch, Michael J. Kruhlak, A. Chesler, Howard H Yang, Maxwell P Lee, Cristina M Branco, Meera Murgai
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Single-cell RNA sequencing studies have revealed the heterogeneity of cell states present in the rheumatoid arthritis (RA) synovium. However, it remains unclear how these cell types interact with one another in situ and how synovial microenvironments shape observed cell states. Here, we use spatial transcriptomics (ST) to define stable microenvironments across eight synovial tissue samples from six RA patients and characterize the cellular composition of ectopic lymphoid structures (ELS). To identify disease-relevant cellular communities, we developed DeepTopics, a scalable reference-free deconvolution method based on a Dirichlet variational autoencoder architecture. DeepTopics identified 22 topics across tissue samples that were defined by specific cell types, activation states, and/or biological processes. Some topics were defined by multiple colocalizing cell types, such as CD34+ fibroblasts and LYVE1+ macrophages, suggesting functional interactions. Within ELS, we discovered two divergent cellular patterns that were stable across ELS in each patient and typified by the presence or absence of a “germinal-center-like” topic. DeepTopics is a versatile and computationally efficient method for identifying disease-relevant microenvironments from ST data, and our results highlight divergent cellular architectures in histologically similar RA synovial samples that have implications for disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5649472b0dc4cb2ee433d6b1131064e755b734d" target='_blank'>
              Deep topic modeling of spatial transcriptomics in the rheumatoid arthritis synovium identifies distinct classes of ectopic lymphoid structures
              </a>
            </td>
          <td>
            Preethi K. Periyakoil, Melanie H. Smith, Meghana Kshirsagar, Daniel Ramirez, Edward DiCarlo, Susan M. Goodman, Alexander Y. Rudensky, L. Donlin, Christina S. Leslie
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Spatially resolved transcriptomics (SRT) provides an invaluable avenue for examining cell-cell interactions within native tissue environments. The development and evaluation of analytical tools for SRT data necessitate tools for generating synthetic datasets with known ground truth of cell-cell interaction induced features. To address this gap, we introduce sCCIgen, a novel real-data-based simulator tailored to generate high-fidelity SRT data with a focus on cell-cell interactions. sCCIgen preserves transcriptomic and spatial characteristics in SRT data, while comprehensively models various cell-cell interaction features, including cell colocalization, spatial dependence among gene expressions, and gene-gene interactions between nearby cells. We implemented sCCIgen as an interactive, easy-to-use, realistic, reproducible, and well-documented tool for studying cellular interactions and spatial biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b742b7f80c26a09db2a4a0e4218062de306a242d" target='_blank'>
              sCCIgen: A high-fidelity spatially resolved transcriptomics data simulator for cell-cell interaction studies
              </a>
            </td>
          <td>
            Xiaoyu Song, J. C. Chávez‐Fuentes, Weiping Ma, Weijia Fu, Pei Wang, Guo-Cheng Yuan
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba81ba07e36306408c53ef4726b3e3e8f09de47b" target='_blank'>
              Spatial transcriptomic analysis of tumor microenvironment in esophageal squamous cell carcinoma with HIV infection
              </a>
            </td>
          <td>
            Yuanli Zuo, Yang Jin, Gang Li, Yue Ming, Ting Fan, Yitong Pan, Xiaojun Yao, Yong Peng
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Spatial transcriptomics has enabled the study of mRNA distributions within cells, a key aspect of cellular function. However, there is a dearth of tools that can identify and interpret functionally relevant spatial patterns of subcellular transcript distribution. To address this, we present CellSP, a computational framework for identifying, visualizing, and characterizing consistent subcellular spatial patterns of mRNA. CellSP introduces the concept of “gene-cell modules”, which are gene sets with non-random subcellular transcript distributions in many cells. It provides intuitive visualizations of the captured patterns and offers functional insights into each discovered module. We demonstrate that CellSP reliably identifies functionally significant modules across diverse tissues and technologies. We use the tool to discover subcellular spatial phenomena related to myelination, axonogenesis and synapse formation in the mouse brain. We find immune response-related modules that change between kidney cancer and healthy samples, and myelination-related modules specific to mouse models of Alzheimer’s Disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26c9d33690c61fab528b90b2d805b9244c8e0fba" target='_blank'>
              CellSP: Module discovery and visualization for subcellular spatial transcriptomics data
              </a>
            </td>
          <td>
            Bhavay Aggarwal, Saurabh Sinha
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Brain metastasis occurs in approximately 16% of metastatic breast cancer patients, and incidence is increasing. Patients with breast cancer brain metastases frequently develop neurological problems including cognitive impairments and seizures. We previously demonstrated accumulation of activated microglia around the tumour site in a breast cancer brain metastasis model, which co-localised with spontaneously occurring local field potential events that resembled interictal epileptic discharges. However, the mechanisms underlying these spatially restricted effects on neuronal excitability are poorly understood. Methods To better understand the underlying cellular changes and potential mechanisms, we used digital spatial profiling of brain cell subtype-specific markers to chart the spatial organisation of neurons, oligodendrocytes, astrocytes, and microglia in relation to metastatic lesion site. We developed a novel chordline-based approach to analyse these spatial changes in biomarker distribution. Results Several protein markers associated with proliferation (Ki67), astrocytes (GFAP) and microglia (CD11b, IBA1, CD45, MSR1) were upregulated in the tumour compared to healthy contralateral brain parenchyma. In contrast, some neuroglial markers (MAP2, Neurofilament light, NeuN, S100B and TMEM119) were lower in the lesion compared to normal tissue. Conclusions Overall, the protein marker changes in the lesion are indicative of neuronal cell loss and changes in the immune microenvironment of the metastatic lesion involving activated microglia, astrocytes and recruitment of peripheral immune cells. Such cellular changes may contribute to the changes in electrical activity resulting from brain metastases observed in mice and patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61dbeafc4e0030d90e60f5d024e3f1200854ddc2" target='_blank'>
              Spatial profiling of central nervous system cellular markers in murine breast cancer to brain metastases
              </a>
            </td>
          <td>
            Karen Hogg, Grant Calder, Alastair P. Droop, Ming Yang, Anna Simon, Mark J Hunt, Paul M Kaye, Miles A. Whittington, Sangeeta Chawla, W. Brackenbury
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Spatial transcriptomics provides insights into tissue architecture by linking gene expression with spatial localization. Current deconvolution methods rely heavily on single-cell RNA sequencing (scRNA-seq) references, which are costly and often unavailable, mainly if the tissue under evaluation is limited, such as in a core biopsy specimen. We present a novel tool, CITEgeist, that deconvolutes spatial transcriptomics data using antibody capture from the same slide as the reference, directly leveraging cell surface protein measurements from the same tissue section. This approach circumvents the limitations of scRNA-seq as a reference, offering a cost-effective and biologically grounded alternative. Our method employs mathematical optimization to estimate cell type proportions and gene expression profiles, incorporating sparsity constraints for robustness and interpretability. Benchmarks against state-of-the-art deconvolution methods show improved accuracy in cell type resolution, particularly in dense tumor microenvironments, while maintaining computational efficiency. This antibody-based tool advances spatial transcriptomics by providing a scalable, accurate, and reference-independent solution for deconvolution in complex tissues. We validate this tool by using a combined approach of simulated data and clinical samples by applying CITEgeist to translational pre-treatment and post-treatment ER+ breast tumors from an ongoing clinical trial, emphasizing the applicability and robustness of CITEgeist.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4308d97c43b1d495c969105587ab2b638fc0a951" target='_blank'>
              CITEgeist: Cellular Indexing of Transcriptomes and Epitopes for Guided Exploration of Intrinsic Spatial Trends
              </a>
            </td>
          <td>
            A. Chang, Brent T. Schlegel, N. Carleton, Priscilla F McAuliffe, S. Oesterreich, Russell Schwartz, Adrian V. Lee
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Brain metastases (BrMets), common for advanced-stage breast cancer patients, are associated with poor median survival and accompanied by severe neurologic decline. Halting the progression of breast cancer brain metastases (BCBMs) may require modulation of the tumor microenvironment (TME), yet little is known about the impact of the primary breast TME on brain tropism, or how, once there, metastatic breast cancer cells coexist with brain-resident cells (e.g., neurons and glia). Traditionally, studies in this space have focused on differential expression analysis, overlooking potential insights gained from investigating genes with equivalent expression between groups. This is particularly crucial in distant metastasis, where tumor cells may co-opt the transcriptional programs of the host organ (e.g., brain) to facilitate successful seeding and outgrowth. Prior to our work, no computational framework existed to determine biologically-relevant equivalent gene expression. To resolve molecular mechanisms of BCBM enabled by metastatic cancer cells and/or resident brain cells, we leveraged Nanostring GeoMx to perform spatially-resolved transcriptomic profiling on 235 patient-derived tissue cores from BCBM (including adjacent normal brain), primary invasive breast cancers, and normal (non-cancer) brain; analyzing 18,677 RNAs in 450 areas of interest (AOIs). We introduce the “Equivalent Expression Index” a highly specific and accurate algorithm that identifies statistically significant “Equivalently-Expressed Genes”. This method facilitated the identification of molecular remodeling and mimicry genes within tissue-specific TMEs. By integrating differential expression analysis with the Equivalent Expression Index, we discovered multiple novel gene signatures associated with BCBM and primary tumor brain-metastatic potential. We demonstrate that the Equivalent Expression Index is a powerful tool to uncover shared gene expression programs representing the adaptation of metastatic cells and brain-resident cells to the BCBM microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/015422dee0790a994280a3f1f594af35182af70e" target='_blank'>
              Spatial Profiling Reveals Equivalence-Derived Molecular Signatures of Brain Mimicry and Adaptation in Breast Cancer Brain Metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, C. Yeh, Bryanna Godfrey, Pablo Nunez Perez, Giuseppe Barisano, Sushama Varma, Saman Ahmadian, A. Toland, M. Granucci, Thy T. H. Trinh, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia K Plevritis, M. Gephart
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The ability of tumors to evolve and adapt by developing subclones in different genetic and epigenetic states is a major challenge in oncology. Traditional tools like multi-regional sequencing used to study tumor evolution and the resultant intra-tumor heterogeneity (ITH) are often impractical because of their resource-intensiveness and limited scalability. Here, we present MorphoITH, a novel framework that leverages histopathology slides to deconvolve molecular ITH through tissue morphology. MorphoITH integrates a self-supervised deep learning similarity measure to capture phenotypic variation across multiple dimensions (cytology, architecture, and microenvironment) with rigorous methods to eliminate spurious sources of variation. Using a prototype of ITH, clear cell renal cell carcinoma (ccRCC), we show that MorphoITH captures clinically-significant biological features, such as vascular architecture and nuclear grades. Furthermore, we find that MorphoITH recognizes differential biological states corresponding to subclonal changes in key driver genes (BAP1/PBRM1/SETD2). Finally, by applying MorphoITH to a multi-regional sequencing experiment, we postulate evolutionary trajectories that largely recapitulate genetic evolution. In summary, MorphoITH provides a scalable phenotypic lens that bridges the gap between histopathology and genomics, advancing precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51f0d69b3ba41ef9a508aa6acb9b650626ef267d" target='_blank'>
              MorphoITH: A Framework for Deconvolving Intra-Tumor Heterogeneity Using Tissue Morphology
              </a>
            </td>
          <td>
            Aleksandra Weronika Nielsen, H. E. Manoochehri, Hua Zhong, Vandana Panwar, V. Jarmale, Jay Jasti, Mehrdad Nourani, Dinesh Rakheja, J. Brugarolas, Payal Kapur, Satwik Rajaram
          </td>
          <td>2025-02-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Among the extensive genomic alterations in prostate cancer, the genomic deletion of PTEN stands out as one of the most consistently observed and confirmed alteration. PTEN loss in prostate tumors is primarily associated with cancer cell proliferation and survival through the activation of the PI3K-AKT-mTOR signalling pathway. However, its use as a robust biomarker in the clinical practice is hampered by the complex epigenetic, transcriptional and post-translational regulation. Here, we undertook an approach that combined in situ assessment of PTEN protein with transcriptional surrogates of its activity to gain insights into the downstream functional effects of PTEN loss in primary tumors. Our extensive bioinformatic analyses, including the integration with single-cell RNA-Seq approaches in a new clinical cohort, highlighted stroma remodeling as the major cancer cell-extrinsic process associated with PTEN loss. By applying similar computational strategy on the transcriptomic data generated from primary prostate tumors of genetically engineered Pten knock-out mouse models, we validated the causal role of Pten in the stromal reaction observed in clinical specimens. Mechanistically, we provide evidence for the activation of a paracrine program that encompasses enhanced TGF-β signalling and that is compatible with the secretome of PTEN-deficient senescent cancer cells. Our study provides relevant biological context to the cellular and molecular alterations unleashed upon PTEN protein loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2b98ad80d259dc7a130daab0e45ee7b8d7d990" target='_blank'>
              Transcriptional network analysis of PTEN protein-deficient prostate tumors reveals robust stromal reprogramming and signs of senescent paracrine communication
              </a>
            </td>
          <td>
            Ivana Rondon-Lorefice, Jose I. Lopez, Aitziber Ugalde-Olano, Maite Zufiaurre, I. Astobiza, Saioa Garcia-Longarte, Amaia Zabala, Sofía Rey, Aida Santos-Martín, Miguel Unda, Ana Loizaga-Iriarte, M. Graupera, Paolo Nuciforo, A. Carracedo, Isabel Mendizabal
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bec213fd287708367fee6bf0c6de9efb0e1c0cb" target='_blank'>
              Natural killer cells occupy unique spatial neighborhoods in HER2- and HER2+ human breast cancers
              </a>
            </td>
          <td>
            Femke A I Ehlers, Katie E. Blise, C. Betts, Shamilene Sivagnanam, L. Kooreman, E. S. Hwang, Gerard M. J. Bos, L. Wieten, L. Coussens
          </td>
          <td>2025-01-24</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Given the successful graft-versus-leukemia cell treatment effect observed with allogeneic hematopoietic stem cell transplant for patients with refractory or relapsed acute myeloid leukemia, immunotherapies have also been investigated in the nontransplant setting. Here, we use a multi-omic approach to investigate spatiotemporal interactions in the bone marrow niche between leukemia cells and immune cells in patients with refractory or relapsed acute myeloid leukemia treated with a combination of the immune checkpoint inhibitor pembrolizumab and hypomethylating agent decitabine. We derived precise segmentation data by extensively training nuclear and membrane cell segmentation models, which enabled accurate transcript assignment and deep learning-feature-based image analysis. To overcome read-depth limitations, we integrated the single-cell RNA sequencing data with single-cell-resolution spatial transcriptomic data from the same sample. Quantifying cell-cell distances between cell edges rather than cell centroids allowed us to conduct a more accurate downstream analysis of the tumor microenvironment, revealing that multiple cell types of interest had global enrichment or local enrichment proximal to leukemia cells after pembrolizumab treatment, which could be associated with their clinical responses. Furthermore, ligand-receptor analysis indicated a potential increase in TWEAK signaling between leukemia cells and immune cells after pembrolizumab treatment. Highlights Spatial transcriptomic analysis of R-AML bone marrow niches provides detailed information about intercellular interactions in the tumor microenvironment. Immunotherapy shifts the cell composition of the leukemia neighborhood. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79158e9c810aea3af6c5a5e81cece5524c3196f" target='_blank'>
              Single Cell Spatial Transcriptomics Reveals Immunotherapy-Driven Bone Marrow Niche Remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R. Herzog, Abigail Wong-Rolle, Laura W Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine Calvo, Christopher S Hourigan, Kasper Hansen, Chen Zhao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="During tumorigenesis, the extracellular matrix (ECM), which constitutes the structural scaffold of tissues, is profoundly remodeled. While the impact of such remodeling on tumor growth and invasion has been extensively investigated, much less is known on the consequences of ECM remodeling on tumor infiltration by immune cells. By combining tissue imaging and machine-learning, we here show that the localization of T lymphocytes and neutrophils, which orchestrate antitumor immune responses, can be predicted by defined topographical features of fibrillar collagen networks. We further show that these collagen topographies result from the activation of a fibrotic pathway controlled by the transcription factor Tcf4 upon depletion of tumor-associated macrophages at late tumor stages. This pathway promotes the deposition of collagen 3 by both tumor and stromal cells, resulting in intermingled collagen networks that favor intra-tumoral T cell and neutrophil localization. Importantly, analysis of human colorectal cancer public bulk RNAseq databases showed a strong correlation between Tcf4 and collagen 3, as well as between the expression of these genes and tumor infiltration by T lymphocytes and neutrophils, attesting the clinical relevance of our findings. This study highlights the key structural role of macrophages on the tumor extracellular matrix and identifies collagen network topographies as a major regulator of tumor infiltration by immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5894e6b84c18eb6409dfa2fcb90416bb17621a9" target='_blank'>
              Macrophages restrict tumor permissiveness to immune infiltration by controlling local collagen topography through a Tcf4-Collagen3 fibrotic axis
              </a>
            </td>
          <td>
            Zoé Fusilier, Franck Simon, Isabel Calvente, Lou Crestey, Alexandra Clément, Mathilde Mathieu, Roude Jean-Marie, Florence Piastra-Facon, Jeyani George Clément, Enola Lumineau, Mattia Tonani, Valeria Manriquez, Livia Lacerda, Perrine de Villemagne, Eliane Piaggio, Vincent Semetey, Sylvie Coscoy, Emanuele Martini, G. Scita, Jean-Christophe Gelly, Johanna Ivaska, Hervé Isambert, C. Goudot, Paolo Pierobon, A. Lennon-Duménil, Hélène D. Moreau
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Background Growing evidence supports the importance of characterizing the organizational patterns of various cellular constituents in the tumor microenvironment in precision oncology. Most existing data on immune cell infiltrates in tumors, which are based on immune cell counts or nearest neighbor-type analyses, have failed to fully capture the cellular organization and heterogeneity. Methods We introduce a computational algorithm, termed Tumor-Immune Partitioning and Clustering (TIPC), that jointly measures immune cell partitioning between tumor epithelial and stromal areas and immune cell clustering versus dispersion. As proof-of-principle, we applied TIPC to a prospective cohort incident tumor biobank containing 931 colorectal carcinoma cases. TIPC identified tumor subtypes with unique spatial patterns between tumor cells and T lymphocytes linked to certain molecular pathologic and prognostic features. T lymphocyte identification and phenotyping were achieved using multiplexed (multispectral) immunofluorescence. In a separate hepatocellular carcinoma cohort, we replaced the stromal component with specific immune cell types—CXCR3+CD68+ or CD8+—to profile their spatial relationships with CXCL9+CD68+ cells. Results Six unsupervised TIPC subtypes based on T lymphocyte distribution patterns were identified, comprising two cold and four hot subtypes. Three of the four hot subtypes were associated with significantly longer colorectal cancer (CRC)-specific survival compared to a reference cold subtype. Our analysis showed that variations in T-cell densities among the TIPC subtypes did not strictly correlate with prognostic benefits, underscoring the prognostic significance of immune cell spatial patterns. Additionally, TIPC revealed two spatially distinct and cell density-specific subtypes among microsatellite instability-high colorectal cancers, indicating its potential to upgrade tumor subtyping. TIPC was also applied to additional immune cell types, eosinophils and neutrophils, identified using morphology and supervised machine learning; here two tumor subtypes with similarly low densities, namely ‘cold, tumor-rich’ and ‘cold, stroma-rich’, exhibited differential prognostic associations. Lastly, we validated our methods and results using The Cancer Genome Atlas colon and rectal adenocarcinoma data (n = 570). Moreover, applying TIPC to hepatocellular carcinoma cases (n = 27) highlighted critical cell interactions like CXCL9-CXCR3 and CXCL9-CD8. Conclusions Unsupervised discoveries of microgeometric tissue organizational patterns and novel tumor subtypes using the TIPC algorithm can deepen our understanding of the tumor immune microenvironment and likely inform precision cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95d3eb1387144868bd261357332833886cbc29f0" target='_blank'>
              Tumor-immune partitioning and clustering algorithm for identifying tumor-immune cell spatial interaction signatures within the tumor microenvironment
              </a>
            </td>
          <td>
            M. Lau, J. Borowsky, J. Väyrynen, K. Haruki, Melissa Zhao, Andressa Dias Costa, Simeng Gu, Annacarolina da Silva, Tomotaka Ugai, Kota Arima, Minh N Nguyen, Yasutoshi Takashima, Joe Yeong, D. Tai, Tsuyoshi Hamada, Jochen K Lennerz, Charles S Fuchs, Catherine J Wu, J. Meyerhardt, S. Ogino, J. Nowak
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="Tissue regeneration relies on the ability of cells to undergo de novo patterning. While tissue patterning has been viewed as the transition from initially identical un-patterned cells to an arrangement of different cell types, recent evidence suggests that initial heterogeneities between cells modulate tissue-scale pattern formation. Yet, how such heterogeneities arise and, thereafter, regulate cell type emergence in a population of cells is poorly understood. Using in vivo and in vitro mouse regenerative systems, we identify a critical tissue density that is required to induce heterogeneous inactivation of the mechanosensor YAP1. Experimental and biophysical approaches demonstrate that YAP1 cell-to-cell heterogeneity pre-patterns the first cell fate decision, via both chromatin remodelling and a supracellular feedback between FOXA1 and Delta-Notch signalling. This feedback motif induces cell fate bistability endowing memory to the system and the maintenance of patterns during homeostasis. These findings reveal a generalisable framework in which transient cell-to-cell heterogeneity, regulated by tissue-scale properties, serves as a critical control parameter for the emergence of cell fate and stable patterning during regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4e71565210ea1e9031332a79fd8f2875c11cbbd" target='_blank'>
              Cell heterogeneity and fate bistability drive tissue patterning during intestinal regeneration
              </a>
            </td>
          <td>
            C. Schwayer, S. Barbiero, D. B. Brückner, C. Baader, N. A. Repina, O. E. Diaz, L. C. Meylan, V. Kalck, S. Suppinger, Q. Yang, J. Schnabl, U. Kilik, J. G. Camp, B. Stockinger, M. Bühler, M. B. Stadler, É. Hannezo, P. Liberali
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Breast cancer (BC) is the second leading cause of cancer-related death among women in the U.S. Organoid models of solid tumors have been shown to faithfully recapitulate aspects of cancer progression such as proliferation and invasion. Although patient-derived organoids (PDOs) and patient-derived xenograft organoids (PDXOs) are pathophysiologically relevant, they are costly to propagate, difficult to manipulate and comprised primarily of the most proliferative cell types within the tumor microenvironment (TME). These limitations prevent their use for elucidating cellular mechanisms of disease progression that depend upon tumor-associated stromal cells which are found within the TME and known to contribute to metastasis and therapy resistance. Here, we report on methods for cultivating epithelial-stromal multicellular 3D cultures. Advantages of these methods include a cost-effective system for rapidly generating organoid-like 3D cultures within scaffold-free environments that can be used to track invasion at single-cell resolution within hydrogel scaffolds. Specifically, we demonstrate how to generate these hetero-multicellular 3D cultures using BT-474 breast cancer cells in combination with fibroblasts (BJ-5ta), monocyte-like cells(THP-1) and/or endothelial cells (EA.hy926). Additionally, differential fluorescent labeling of cell populations enables time-lapse microscopy to define 3D culture assembly and invasion dynamics. Notably, the addition of any two stromal cell combinations to 3D cultures of BT-474 cells significantly reduces circularity of the 3D cultures, consistent of the presence of organoid-like or secondary spheroid structures. In tracker dye experiments, fibroblasts and endothelial cells co-localize in the peripheral organoid-like protrusions and are spatially segregated from the primary BT-474 spheroid. Finally, hetero-multicellular 3D cultures of BT-474 cells have increased hydrogel invasion capacity. Since we observed these protrusive structures in hetero-multicellular 3D cultures of both non-tumorigenic and tumorigenic breast epithelial cells, this work provides an efficient and reproducible method for generating organoid-like 3D cultures in a scaffold-free environment for subsequent analyses of phenotypes associated with solid tumor progression. SUMMARY There is a critical need for 3D cancer models that capture hetero-cellular crosstalk to study cancer metastasis. Our study presents the generation of hetero-multicellular stromal-epithelial in a scaffold and scaffold free environment that can be used to study invasion and cellular spatial distributions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a55555e37181d8e4de2e978898726ac03e16ca" target='_blank'>
              Hetero-multicellular stromal cells incorporate into scaffold-free 3D cultures of epithelial cancer cells to drive invasion
              </a>
            </td>
          <td>
            Elizabeth Ortiz, Kyaw Hsu Thway, Gabriela Ortiz-Soto, Paulina Yao, Jonathan A Kelber
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Constructing a comprehensive spatiotemporal map of tumor heterogeneity is essential for understanding tumor evolution, with copy number variation (CNV) as a significant feature. Existing studies often rely on tools originally developed for single-cell data, which fail to utilize spatial information. Our study aims to develop a model that fully leverages spatial omics data to elucidate spatio-temporal changes in tumor evolution. Here, we introduce SCOIGET (Spatial COpy number Inference by Graph on Evolution of Tumor), a novel framework using graph neural networks with graph attention layers to learn spatial neighborhood features of gene expression and infer copy number variations. This approach integrates spatial multi-omics features to create a comprehensive spatial map of tumor heterogeneity. Notably, SCOIGET achieves a substantial reduction in error metrics (e.g., mean squared error, cosine similarity, and distance measures) and produces superior clustering performance as indicated by higher Silhouette Scores compared to existing methods. Our model significantly enhances the efficiency and accuracy of tumor evolution depiction, capturing detailed spatial and temporal changes within the tumor microenvironment. It is versatile and applicable to various downstream tasks, demonstrating strong generalizability across different spatial omics technologies and cancer types. This robust performance improves research efficiency and provides valuable insights into tumor progression. In conclusion, SCOIGET offers an innovative solution by integrating multiple features and advanced algorithms, providing a detailed and accurate representation of tumor heterogeneity and evolution, aiding in the development of personalized cancer treatment strategies. Key Points 1. Innovative Framework for Spatially Contextualized CNV Inference Introduces SCOIGET, a graph-based model that integrates spatial omics data with graph neural networks and attention mechanisms to accurately infer spatial copy number features. 2. Detailed Tumor Heterogeneity Mapping Constructs representations of tumor evolution, capturing heterogeneity and clonal dynamics at both cellular and subcellular resolutions. 3. Versatile and Robust Performance Outperforms existing methods across multiple datasets, demonstrating reliability and adaptability to diverse spatial omics platforms and cancer types. 4. Broad Downstream Applications Enables tasks such as tumor subclone identification, clustering, and evolutionary trajectory analysis, facilitating precision oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5782e67aa86bd64459296bc264a422b250cba97" target='_blank'>
              SCOIGET: Predicting Spatial Tumor Evolution Pattern by Inferring Spatial Copy Number Variation Distributions
              </a>
            </td>
          <td>
            Yujia Zhang, Yitao Yang, Yan Kong, Bingxu Zhong, Kenta Nakai, Hui Lu
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, , Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/332b50c1b783afb4c65a813ff9aedb52f38aaf88" target='_blank'>
              Spatial tumor immune heterogeneity facilitates subtype co-existence and therapy response in pancreatic cancer
              </a>
            </td>
          <td>
            Lukas Klein, M. Tu, Niklas Krebs, Laura Urbach, Daniela Grimm, Muhammad Umair Latif, Frederike Penz, Anna Blandau, Xueyan Wu, Rebecca Diya Samuel, Stefan Küffer, F. Wegwitz, N. Chan, Kazeera Aliar, F. Vyas, Uday Kishore, E. Hessmann, Andreas Trumpp, Elisa Espinet, A. Papantonis, Rama Khokha, V. Ellenrieder, Barbara T. Grünwald, Shiv K. Singh
          </td>
          <td>2025-01-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76feb62f24d1b55375efafabff78309d558b9c49" target='_blank'>
              Spatial interaction mapping of PD-1/PD-L1 in head and neck cancer reveals the role of macrophage-tumour barriers associated with immunotherapy response
              </a>
            </td>
          <td>
            V. Y. Naei, Rafael Tubelleza, J. Monkman, Habib Sadeghirad, Meg L. Donovan, Tony Blick, A. Wicher, Sara Bodbin, Amelie Viratham, Robert Stad, Subham Basu, C. Cooper, Catherine Barnett, K. O’Byrne, Rahul Ladwa, M. Warkiani, Brett G M Hughes, A. Kulasinghe
          </td>
          <td>2025-02-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Lung cancer is a leading cause of cancer-related mortality worldwide, largely due to its heterogeneity and intrinsic drug resistance. Malignant pleural effusions (MPEs) provide diverse tumor cell populations ideal for studying these complexities. Although chemotherapy and targeted therapies can be initially effective, subpopulations of cancer cells with phenotypic plasticity often survive treatment, eventually developing resistance. Here, we integrated single-cell isolation and three-dimensional (3D) spheroid culture to dissect subclonal heterogeneity and drug responses, aiming to inform precision medicine approaches. Using A549 lung cancer cells, we established a cisplatin-resistant line and isolated three resistant subclones (Holoclone, Meroclone, Paraclone) via single-cell sorting. In 3D spheroids, Docetaxel and Alimta displayed higher IC50 values than in 2D cultures, suggesting that 3D models better reflect clinical dosing. Additionally, MPE-derived Holoclone and Paraclone subclones exhibited distinct sensitivities to Giotrif and Capmatinib, revealing their heterogeneous drug responses. Molecular analyses confirmed elevated ABCB1, ABCG2, cancer stem cell (CSC) markers (OCT4, SOX2, CD44, CD133), and epithelial–mesenchymal transition (EMT) markers (E-cadherin downregulation, increased Vimentin, N-cadherin, Twist) in resistant subclones, correlating with enhanced migration and invasion. This integrated approach clarifies the interplay between heterogeneity, CSC/EMT phenotypes, and drug resistance, providing a valuable tool for predicting therapeutic responses and guiding personalized, combination-based lung cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6813439ca1c219dd03651dca35df5520579d3204" target='_blank'>
              Application of an Integrated Single-Cell and Three-Dimensional Spheroid Culture Platform for Investigating Drug Resistance Heterogeneity and Epithelial–Mesenchymal Transition (EMT) in Lung Cancer Subclones
              </a>
            </td>
          <td>
            Shin-Hu Chen, Jian-Hong Yu, Yu-Chun Lin, Yi-Ming Chang, Nien-Tzu Liu, Su-Feng Chen
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Liver progenitor cells (LPCs) with bipotential differentiation capacities are essential for restoring liver homeostasis and hepatocyte population after damage. However, the low proportion and shared markers with epithelial cells make studying LPC heterogeneity difficult, especially in humans. To address this gap, we explored over 259,400 human liver single cells across 4 conditions (fetal, healthy, cirrhotic, and HCC-affected livers). Methods: Human liver tissue samples were analyzed using spatial transcriptomics sequencing technologies to describe the heterogeneity of LPCs. Liver tissue was characterized by LPC heterogeneity at single-cell resolution by employing cellular modules, differentiation trajectories, and gene co-expression patterns. Results: We annotated and identified 1 LPC cluster, 3 LPC subpopulations, and 4 distinct cellular modules, indicating the heterogeneity within LPC and the diversity between LPCs and epithelial cells. LPCs showed spatial colocalization with cholangiocytes and comprised a small proportion (2.95±1.91%) within the merged epithelial cells and LPC populations, exhibiting marked differences in marker expression patterns compared to those in mice. LPCs exhibited distinct cellular states in functional restoration, activation, proliferation, and cell transition. Additionally, the gene co-expression network of LPCs exhibited 3 unique modules, reflecting the distinct connectivity of genes encoding apolipoproteins and heat shock proteins in the gene co-expression network modules. Conclusions: Our study provides valuable insights into the multifaceted heterogeneity of human LPCs. Future studies focusing on spatial gene expression dynamics will contribute to our understanding of the spatial arrangement of liver regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/363fd866c375a553c02fda34c7798688e167a041" target='_blank'>
              Integrative single-cell and spatial transcriptome analysis reveals heterogeneity of human liver progenitor cells
              </a>
            </td>
          <td>
            Chuanjun Liu, Kai Wang, Junpu Mei, Ruizhen Zhao, Juan Shen, Wei Zhang, Liangyu Li, Bhaskar Roy, Xiaodong Fang
          </td>
          <td>2025-02-26</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3852f7c6358b758c264d481b7403f1d4972dcd97" target='_blank'>
              Immunotherapy in ovarian cancer: spatial functional genomics to unravel resistance mechanisms
              </a>
            </td>
          <td>
            Martina Rausch, Karlotta Bartels, Josef Leibold
          </td>
          <td>2025-01-22</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Diagnosis of ductal carcinoma in situ (DCIS) presents a challenge as we cannot yet distinguish between those lesions that remain dormant from cases that may progress to invasive ductal breast cancer (IDC) and require therapeutic intervention. Our overall interest is to develop biomimetic three-dimensional (3D) models that more accurately recapitulate the structure and characteristics of pre-invasive breast cancer in order to study the underlying mechanisms driving malignant progression. These models allow us to mimic the microenvironment to investigate many aspects of mammary cell biology, including the role of the extracellular matrix (ECM), the interaction between carcinoma-associated fibroblasts (CAFs) and epithelial cells, and the dynamics of cytoskeletal reorganization. In this review article, we outline the significance of 3D culture models as reliable pre-clinical tools that mimic the in vivo tumor microenvironment and facilitate the study of DCIS lesions as they progress to invasive breast cancer. We also discuss the role of CAFs and other stromal cells in DCIS transition as well as the clinical significance of emerging technologies like tumor-on-chip and co-culture models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c698a2edd8d5cc03e968580287676fc4d06fb974" target='_blank'>
              Three-Dimensional Models: Biomimetic Tools That Recapitulate Breast Tissue Architecture and Microenvironment to Study Ductal Carcinoma In Situ Transition to Invasive Ductal Breast Cancer
              </a>
            </td>
          <td>
            Seema Shah, Kingsley O. Osuala, Ethan J. Brock, Kyungmin Ji, Bonnie F. Sloane, Raymond R Mattingly
          </td>
          <td>2025-02-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Resistance to immune checkpoint inhibitors (ICIs) represents a major barrier to effectively treating patients with NSCLC. Increased tumor heterogeneity has been linked with diminished response to ICIs, but the mechanisms underlying such an effect remain underexplored, in part due to the lack of models for studying the impact of heterogeneity. To address this issue, we have developed a novel murine model of NSCLC that utilizes clonal cell lines to modulate tumor heterogeneity. Specifically, we isolated single cells from a murine NSCLC cell line harboring 3 common driver mutations (Kras G12D Tp53 +/- Lkb1 -/- , or “KPL”). We mutagenized KPL cells with N-methyl-N-nitrosourea in order to increase the tumor mutational burden, and then isolated single cells to derive clonal cell lines. Whole exome sequencing and subsequent phylogenetic analysis of tumor mutations yielded categorization of these clones into 5 clusters. Clones within the same cluster demonstrated comparable in vivo growth kinetics and similar response to anti-PD-1 immunotherapy. These clones exhibited unique tumor immune microenvironments that did not fully correlate with genomic factors such as tumor mutation burden. Notably, heterogeneous polyclonal tumors grew more aggressively than monoclonal tumors, and they exhibited response to anti-PD-1 therapy that differed from the response seen in the component single clones. Tracking of individual clones with fluorescent proteins and mutation tracing illustrated clonal shifts induced by ICI treatment. Our results validate this system as a model of NSCLC that can modify tumor heterogeneity and provide insights into the effects of heterogeneity on ICIs and other immunotherapies.
 Citation Format: Michael Oh, Jensen W Abascal, Linh M Tran, Ramin Salehi-Rad, Raymond J Lim, Camelia Dumitras, Cara Yean, Sunny Xiao, Bitta P Kahangi, William P Crosson, Edgar Perez Reyes, Nalani Coleman, Austin K Rennels, Kostyantyn Krysan, Bin Liu, Steven M Dubinett. A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B068.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64eb12bd7d9a78fd223bedc136a7a4d7a78d40" target='_blank'>
              Abstract B068: A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer
              </a>
            </td>
          <td>
            M. Oh, J. Abascal, L. Tran, R. Salehi-Rad, R. Lim, C. Dumitras, Cara E. Yean, Sunny Xiao, Bitta Kahangi, William P Crosson, E. P. Reyes, Nalani Coleman, Austin K Rennels, K. Krysan, Bin Liu, Steven M Dubinett
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="ABSTRACT Metabolic processes are crucial in immune regulation, yet the impact of metabolic heterogeneity on immunological functions remains unclear. Integrating metabolomics into immunology allows the exploration of the interactions of multilayered features in the biological system and the molecular regulatory mechanism of these features. To elucidate such insight in lung squamous cell carcinoma (LUSC), we analyzed 106 LUSC tumor tissues. We performed high-resolution matrix-assisted laser desorption/ionization (MALDI) mass spectrometry imaging (MSI) to obtain spatial metabolic profiles, and immunohistochemistry to detect tumor-infiltrating T lymphocytes (TILs). Unsupervised k-means clustering and Simpson’s diversity index were employed to assess metabolic heterogeneity, identifying five distinct metabolic tumor subpopulations. Our findings revealed that TILs are specifically associated with metabolite distributions, not randomly distributed. Integrating a validation cohort, we found that heterogeneity-correlated metabolites interact with CD8+ TIL-associated genes, affecting survival. High metabolic heterogeneity was linked to worse survival and lower TIL levels. Pathway enrichment analyses highlighted distinct metabolic pathways in each subpopulation and their potential responses to chemotherapy. This study uncovers the significant impact of metabolic heterogeneity on immune functions in LUSC, providing a foundation for tailoring therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b4c98251a8ac1956e7ccdfa63c51218dbe1751b" target='_blank'>
              Metabolic heterogeneity in tumor cells impacts immunology in lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Qian Wang, Na Sun, Chaoyang Zhang, Thomas Kunzke, P. Zens, A. Feuchtinger, Sabina Berezowska, Axel Walch
          </td>
          <td>2025-02-09</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 227


 Background:
 Colorectal cancer (CRC) is surgically resectable and eminently screenable yet remains a lethal entity with high affinity to metastasise synchronously and metachronously. It has been proposed that cellular and extracellular components of the microenvironment contribute to metastatic potential. Dissociative profiling techniques such as bulk transcriptomic and single-cell RNA sequencing have contributed insight but limited therapeutic progress. Spatial transcriptomic (ST) assessment allows molecular profiling of tissue while preserving tissue architecture. Here we employ ST approaches to interrogate tumor compartments of primary resectable and matched synchronous CRC demonstrating stromal signatures with distinct collagen expression associated with outcome.
 Methods:
 25 patients with primary resectable CRC and 4 patients with matched primary CRC and liver metastasis (CRLM) underwent single-cell spatial transcriptomics using the Nanostring CosMx Spatial Molecular Imager (SMI, 1000plex gene panel) (Discovery cohort). 71681 epithelial and 88806 stromal cells with intact spatial resolution were analysed. The spatial single-cell signatures were reconstituted in 89 patients using the Nanostring GeoMx Digital Spatial Profiler (Validation cohort). 3 GeoMx compartments were analysed: Epithelial (PanCK+); Stroma (aSMA+); Immune (PanCK-aSMA-).
 Results:
 CosMx demonstrated 2 distinct collagen signatures: COL1A1,COL1A2,COL3A1 associated with normal fibroblasts; COL9A2 associated with cancer-related stroma and lethal subtypes of epithelial cell. In the GeoMx validation cohort, 42 patients expressed the COL1A1 signature in the aSMA compartment and 47 did not (5 year recurrence free survival: 0.88 vs 0.49, p < 0.005) with different morphological patterns of each distinct aSMA group. In patients with favourable prognosis, chemokine high epithelial subtypes expressed CXCL5 and recruited COL9A2-, IL6+ neutrophils to the surrounding microenvironment. In contrast, epithelial cells in patients with poor prognosis expressed CXCL8 and recruited COL9A2+ neutrophils which were frequently in contact with SPP1+ macrophages.
 Conclusions:
 We have used ST approaches to interrogate cellular compartments of CRC demonstrating stromal subtypes that impact outcome. These insights could be used in the clinical setting to quantify COL9A2 expression in aSMA+ cells to prognosticate patients. Targeting of tumor collagen has been proposed to augment existing anti-cancer therapies and this work has demonstrated COL9A2 as a potential target in need of further investigation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ead468254f12c6dc5934a33a000887d505597fb4" target='_blank'>
              Spatial transcriptomics and lethality-associated stromal remodeling of colorectal cancer.
              </a>
            </td>
          <td>
            C. Wood, João Da Silva Filho, Tengyu Zhang, Campbell Roxburgh, Paul G. Horgan, J. Edwards, Matthias Marti, Colin W. Steele, Nigel B Jamieson
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 224


 Background:
 The tumor-immune microenvironment (TME) is crucial in the progression of prostate cancer (PCa), yet little is known about its spatial organization and immune composition in treatment-naïve, localized PCa. While immunotherapy has had limited success in metastatic PCa, its efficacy in localized disease remains largely unknown. This study aims to dissect tumor-immune interactions in primary PCa and explore differences in immune landscape between low- and high-Gleason PCa, thus providing biologic insights for immunotherapy actionability.
 Methods:
 We performed spatial profiling of radical prostatectomy (RP) specimens from 28 treatment-naïve prostate cancer patients. These included 14 patients with low-grade disease (Gleason Grade Group 1–2) and 14 patients with high-grade disease (Gleason Grade Group 4–5). Using multiplex cyclic immunofluorescence (CyCIF) and a 27-marker panel, we obtained single-cell resolution across whole-slide images. Tumor and stromal compartments were manually annotated, and immune cell densities and neighborhood were analyzed and compared between low- and high-grade tumors. Spatial analysis focused on identifying immune clusters (IC -suggestive of tertiary lymphoid structures -TLS) and quantified the degree of immune infiltration along with characterization of T cell exhaustion markers such as CD8+PD1+TCF1+ cells.
 Results:
 High-grade tumors exhibited significantly higher infiltration of CD8+ T cells and CD20+ B cells compared to low-grade tumors. Spatial profiling revealed organized ICs, especially in high-grade tumors, which exhibited characteristics consistent with TLS. These ICs were enriched with precursor-exhausted CD8+PD1+TCF1+ T cells (TPEX), suggesting a state of immune exhaustion that may be therapeutically reactivatable. The presence of TLS and immune exhaustion markers in high-grade tumors highlights a more active immune microenvironment than previously appreciated in primary prostate cancer.
 Conclusions:
 Our findings reveal that a subset of high-grade prostate cancers harbor an organized, immune-infiltrated tumor microenvironment, including adaptive immune features such as TLS and CD8+ T cell exhaustion. These immune landscapes, marked by significant CD8+ and CD20+ cell infiltration and spatial clustering, suggest that high-grade prostate tumors may be amenable to immune-based therapies, despite the failure of immune checkpoint blockade in metastatic PCa. These results raise the hypothesis that immunotherapy might be effective in treatment-naïve high-grade prostate cancer. Future efforts should explore whether these immune profiles can be leveraged for risk stratification or as biomarkers for response to immunotherapy in advanced disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12d9a96ad448410a624d4884f42c080b637c2227" target='_blank'>
              Whole-slide multiplexed tissue imaging of prostate adenocarcinoma to investigate immune niches in high- versus low-grade tumors.
              </a>
            </td>
          <td>
            Ali Amiryousefi, Jeremiah Wala, B. Labadie, Aishwarya Atmakuri, K. Chaudagar, Eamon Toye, Andréanne Gagné, Jia-Ren Lin, E. Van Allen, Akash Patnaik, P. Sorger
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a529afcdab161e3ac0972bc45b1449999ddb519" target='_blank'>
              EMT-driven plasticity prospectively increases cell–cell variability to promote therapeutic adaptation in breast cancer
              </a>
            </td>
          <td>
            Lauriane Muller, Frédérique Fauvet, Christelle Chassot, Francesca Angileri, Angèle Coutant, Cyril Dégletagne, Laurie Tonon, P. Saintigny, Alain Puisieux, A. Morel, Maria Ouzounova, Pierre Martinez
          </td>
          <td>2025-02-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Understanding how individual cancer cells adapt to drug treatment is a fundamental challenge limiting precision medicine cancer therapy strategies. While single-cell technologies have advanced our understanding of cellular heterogeneity, efforts to connect the behavior of individual cells to broader tumor drug responses and uncover global trends across diverse systems remain limited. There is a growing availability of single-cell and bulk omics data, but a lack of centralized tools and repositories makes it difficult to study drug response globally, especially at the level of single-cell adaptation. To address this, we present a multimodal framework that integrates bulk and single-cell treated and untreated transcriptomics data to identify drug responsive cell populations in triple-negative breast cancer (TNBC). Our framework leverages population-scale bulk transcriptomics data from TNBC samples to define seven main “identities”, each representing unique combinations of biologically relevant genes. These identities are dynamic and trackable, allowing us to map them onto single cells and uncover global patterns of how cell populations respond to drug treatment. Unlike static classifications, this approach captures the evolving nature of cellular states, revealing that a select few identities dominate and drive population-level responses during treatment. Crucially, our ability to decode these trends through the inherent noise of single-cell data provides a clearer picture of how heterogeneous cell populations adapt to therapy. By identifying the dominant identities and their dynamics, we can better predict how entire tumors respond to treatment. This insight is essential for designing precise combination therapies tailored to the unique heterogeneity of patient tumors, addressing the single-cell variations that ultimately determine therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d948793b1aa3c6db446f2356cc375541bc62bfa6" target='_blank'>
              A Multimodal Framework to Uncover Drug-Responsive Subpopulations in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Yue Wang, Santiago Haase, Austin A. Whitman, Adriana Beltran, Philip M. Spanheimer, Elizabeth Brunk
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="EMT converts epithelial (E) phenotypes to invasive mesenchymal (M) states. However, analyses of circulating tumor cells (CTCs) indicated that biphenotypic (E+M) CTCs better correlate with metastasis. Similarly, investigations of murine tumors undergoing EMT concluded that early E+M states posses the highest metastatic potential. To explore this, we selected in animals with breast cancer CTCs having progressively increasing intravasation abilities. This revealed that downregulation of arrestin Arrdc4 associates with CTC aggressiveness. In xenografts, depleting Arrdc4 accelerated tumor progression, whereas overexpression hindered progression in immunocompetent, but not in immunocompromised mice. Mechanistically, high Arrdc44 suppresses glucose uptake and enhances gasdermin E, triggering pyroptosis a type of pro-inflammatory cell death. Consistently, Arrdc4’s lowest levels characterize the most metastatic biphenotypic states. In patients, both epigenetic and chromosomal aberrations downregulate ARRDC4 and predict poor prognosis. In summary, the uncovered mechanism portrays pyroptosis of biphenotypic EMT cells as a rheostat of CTCs, which may resolve the controversy on the role played by EMT in metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad683c6747e49ef60cb44d3240bb40206d0d6b87" target='_blank'>
              Avoidance of pyroptosis accounts for the relatively high metastatic potential observed in early hybrid EMT states
              </a>
            </td>
          <td>
            Aakanksha Verma, Alessandro Genna, Yaron Vinik, N. Nataraj, Maha Abedrabbo, Boobash-Raj Selvadurai, Tithi Bhandari, Noa Aharoni, Palaniappan Ramesh, Boris Boguslavski, Diana Drago, T. Salame, Amir Prior, Yishai Levin, Eviatar Weizman, Rong Zhu, Carlos Caldas, O. Rueda, S. Lev, Yosef Yarden
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 Background: Spatial integration of metabolic pathways with single cell-level transcriptomic data in human tumor and normal tissue may elucidate mechanisms supporting tumorigenesis for therapeutic targeting. The suppressive adenosine signaling pathway is an emerging potential target for combination with radiotherapy. This study aims to create an in situ map of the pathway in non-small cell lung cancer (NSCLC) tumor and adjacent normal tissue. We seek to define how distinct cellular compartments cooperate and contribute to adenosine signaling within the tumor microenvironment (TME). Methods: Using Matrix-Assisted Laser Desorption/Ionization mass spectrometry imaging, we examined the spatial distribution of metabolites in paired human NSCLC compared to normal tissue from the same patient (n=15). In situ metabolite quantification and pathway enrichment analysis were performed. Second, we performed single nuclear RNA-sequencing (snRNA-seq) on NSCLC patient samples (n=4) to characterize the transcriptomic landscape of the tumors, including quantification of key members of the adenosine signaling pathway: ADORA2A (A2AR), ADORA2B (A2BR), ENTPD1 (CD39), NT5E (CD73), and ENPP1. Leveraging the average gene expression per cell subtype and pathologist-annotated H&E-stained tissue sections, we are integrating transcriptomic data with the spatial metabolomics to generate an in situ pathway map. Results: In NSCLC tissue, UMP, AMP, and ADP levels were significantly higher in tumor compared to normal tissue (p<0.0001), with mean percentage increases of 79%, 89%, and 73%, respectively. Metabolites downstream of adenosine, adenine and inosine, showed no significant differences (p>0.05). Single nuclear RNA-seq analyses revealed expression of the adenosine 2B receptor, ADORA2B, was highest in fibroblasts, lower in tumor, and absent in the immune compartment. ENTPD1 was expressed by all immune cells and tumor. NT5E was absent in T cells and ENPP1 was absent in macrophages, but expressed in all other immune compartments and tumor. Integration of snRNA-seq data with spatial metabolomics is underway to correlate metabolite abundance and gene expression within distinct cellular compartments to define cell type-specific contribution to adenosine signaling. Conclusion: Our study offers a multimodal approach to spatially resolve metabolic and transcriptomic data in situ relying on frozen NSCLC and normal lung tissue. This has significant translational implications, enabling analysis of biospecimens from multi-institutional clinical trials that precludes processing fresh tissue for single-cell analyses. Herein, we are able to localize key metabolites in untreated tumor and normal tissue and quantify the expression of pathway members at the cellular level. Through spatial integration of these data, we aim to decipher the relationship between distinct cell types within the TME and their contribution to adenosine-driven suppressive changes, then extend to radiotherapy-treated tumors.
 Citation Format: Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick J. McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z. Obradovic, Ramon C. Sun, Catherine S. Spina. {Abstract title} [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69f29862a6e18da7c70e3c9d22cd4b7cabc18a3" target='_blank'>
              Abstract B028: Multimodal approach to characterization of metabolism and immune modulation with insights from spatial metabolomics and transcriptomics
              </a>
            </td>
          <td>
            Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z Obradovic, Ramon C. Sun, Catherine S. Spina
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Myeloid cells in the tumor microenvironment (TME) exhibit diverse functions, from pro-tumor to anti-tumor. To elucidate myeloid heterogeneity, we developed a deep-learning model to annotate 20 myeloid subtypes in single-cell data. We identified two macrophage populations, macro_SPP1 and macro_C1QC, which were pan-cancer, enriched in tumors compared to normal tissues, and linked to clinical outcomes. Higher macro_SPP1 abundance correlated with IO non-responders, pre-IO treatment, and lower inflammation, while macro_C1QC showed the opposite trends. We developed and validated gene signatures to quantify these macrophages in bulk RNAseq. Applying these signatures to TCGA and real-world data, we confirmed that macro_SPP1 was associated with worse survival and poor IO response, whereas macro_C1QC was linked to better outcomes. Tumors with chromosome 9p21 deletions had higher macro_SPP1 and fibroblast signatures, suggesting a TME influence on macrophage heterogeneity. Cell-cell communication analysis in single-cell data revealed that fibroblasts sent the strongest signals to macrophages in lung and pancreatic cancers. Fibroblast-derived ligands such as FN1 and collagens exhibited stronger signaling to macro_SPP1 than macro_C1QC, and colocalized with macro_SPP1 spatially. Conversely, macro_SPP1 sent higher total signals to other cells in the TME than macro_C1QC, with SPP1 to fibroblasts and endothelial cells being the most differentiating signal. In summary, we identified two disease-critical macrophage populations: pro-tumor macro_SPP1 and anti-tumor macro_C1QC. We found interactions between macrophages and fibroblasts as key modulators of the TME.
 Citation Format: Weiwei Zhong, Mitra Ansariola, Micaela Tatman, Rachel Wester, Kim Van Naarden Braun, Shameek Biswas. Disease-critical macrophage populations interact with stroma to modulate the tumor microenvironment [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B086.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58e073c2b3967f3720af1b89697ce0e1aad33c4c" target='_blank'>
              Abstract B086: Disease-critical macrophage populations interact with stroma to modulate the tumor microenvironment
              </a>
            </td>
          <td>
            Weiwei Zhong, Mitra Ansariola, Micaela Tatman, Rachel Wester, Kim Van Naarden Braun, Shameek Biswas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Bone is a multifaceted tissue requiring orchestrated interplays of diverse cells within specialized microenvironments. Although significant progress has been made in understanding cellular and molecular mechanisms of component cells of bone, revealing their spatial organization and interactions in native bone tissue microenvironment is crucial for advancing precision medicine, as they govern fundamental signaling pathways and functional dependencies among various bone cells. In this study, we present the first integrative high-resolution map of human bone and bone marrow, using spatial and single-cell transcriptomics profiling from femoral tissue. This multi-modal approach discovered a novel bone formation-specialized niche enriched with osteoblastic lineage cells and fibroblasts and unveiled critical cell–cell communications and co-localization patterns between osteoblastic lineage cells and other cells. Furthermore, we discovered a novel spatial gradient of cellular composition, gene expression and signaling pathway activities radiating from the trabecular bone. This comprehensive atlas delineates the intricate bone cellular architecture and illuminates key molecular processes and dependencies among cells that coordinate bone metabolism. In sum, our study provides an essential reference for the field of bone biology and lays the foundation for advanced mechanistic studies and precision medicine approaches in bone-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de859f69ecf65ef803fd91ebf44b36784f4b9786" target='_blank'>
              Mapping the spatial atlas of the human bone tissue integrating spatial and single-cell transcriptomics
              </a>
            </td>
          <td>
            Weiqiang Lin, Yisu Li, Chuan Qiu, Binghao Zou, Yun Gong, Xiao Zhang, Di Tian, William Sherman, Fernando Sanchez, Di Wu, Kuan-Jui Su, Xinyi Xiao, Zhe Luo, Qing Tian, Yiping Chen, Hui Shen, Hongwen Deng
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Understanding how radiotherapy induces metabolic reprogramming and reshapes the cytoarchitecture of the tumor microenvironment (TME) is essential for improving treatment outcomes and overcoming resistance. Despite its importance, no studies have used computational tools to quantitatively assess these cytoarchitectural rearrangements in the TME following radiotherapy. In previous work, we developed a quantitative method called “colocatome analysis,” a spatial -omic approach that catalogs statistically significant colocalizations between pairs of cell subpopulations or states. Through a normalization process, this method enables direct comparisons of spatial features across different conditions, including comparisons between in vitro models and tissue samples. We applied colocatome analysis to multiplexed immunofluorescence images of lung adenocarcinoma specimens, examining differences between naïve and recurrent tumors following radiotherapy using a panel of markers that represent primary tumor compartments (endothelial, malignant, fibroblast, and immune) and include markers for the hexosamine biosynthesis pathway (HBP), glycans, key metabolic enzymes, proliferation, cell death, DNA damage, hypoxia, and EMT. Our preliminary results indicate significant cytoarchitectural differences: tumors that recurred after radiotherapy exhibited a lower epithelial-to-mesenchymal cancer cell ratio and were enriched in FAP+ cancer-associated fibroblasts (CAFs). Although the overall proportions of macrophages and other immune cells remained consistent across samples, their spatial organization differed. In naïve tumors, immune cells were largely segregated from cancer cells, while recurrent tumors showed a more homogeneous mixture of cancer and immune cells. Additionally, CAFs were markedly increased in recurrent tumors. Lastly, cancer cells in naïve tumors displayed high levels of O-GlcNAc, which significantly decreased in recurrent tumors. These findings provide insights into the impact of metabolic reprogramming towards HBP in the TME and highlight its potential role in mediating radioresistance.
 Citation Format: Gina Bouchard.Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P019">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b0ad7d9e2bbe3827fd09e473275b031d3cd8d0" target='_blank'>
              Abstract P019: Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy
              </a>
            </td>
          <td>
            Gina Bouchard
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 CD8+ T cell-tumor cell proximity correlates with immunotherapy response. However, it is unknown whether these cells can be captured as functional clusters from clinical samples. In defined co-cultures, tumor antigen-recognizing T cells outcompeted unmatched T cells in forming heterotypic clusters with tumor cells, showing >10-fold enrichment when comprising only 1% of the total T cell population, which was seen in across cancer cell lines. This prompted us to investigate whether this feature could be used to isolate tumor-reactive CD8+ T cells directly from cancer specimens. By regular and imaging flow cytometry, we could identify heterotypic clusters from 21/21 human melanoma metastases, comprising CD8+ T cells interacting with one or more tumor cells and/or antigen-presenting cells (APCs). These results prompted us to investigate whether these heterotypic T cell clusters could be used to better understand biology and whether they hold therapeutic potential. Regarding biology, we observed that CD8+ T cells from clusters isolated from clinical samples were significantly enriched for tumor-reactive and exhausted gene signatures. Integration with T cell receptor (TCR)-sequencing revealed increased clonality of clustered T cells, indicative of expansion. Whereas single CD8+ T cells showed enrichment of viral antigen-specific signatures, T cells from clusters were enriched for tumor antigen-reactivity signatures. In-depth single cell RNA analyses revealed conjugation of these T cells to tumor cells and different APC types. These were not random events, as both T cell-associated APCs and tumor cells were enriched for specific cell states and associated with differential cell-cell communication. CD8+ T cells carrying identical TCRs showed more exhaustion and co-inhibition when conjugated to APCs than to tumor cells. Regarding therapeutic opportunities, we found that upon rapid expansion (REP), CD8+ TIL from clusters exerted on average 9-fold increased melanoma-killing activity in ex vivo assays, accompanied by enhanced production of cytokines. Upon ACT, T cells from clusters showed superior in vivo killing of matched patient-derived xenografts (PDX), which was associated with more infiltration and activation than for ACT of single T cells. Together, these results demonstrate that tumor-reactive CD8+ T cells are enriched in functional clusters with tumor cells and/or APCs, and that they can be isolated and expanded from clinical samples. Being often excluded during sorting for single-cell studies, these distinct heterotypic CD8+ T cell clusters serve as a valuable source amenable to deciphering functional tumor-immune cell interactions, while they may also be therapeutically explored.
 Citation Format: Daniel S Peeper, Sofia Ibáñez-Molero, Johanna Veldman, Juan Simón-Nieto, Joleen Traets, Austin George, Kelly Hoefakker, Anita Karomi, Jonas Nilsson, John Haanen, Winan van Houdt. Heterotypic CD8 T cell clusters isolated from clinical samples are distinct and enriched for antitumor activity [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr PR007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac4b19bdc51a77041e08cca430b7dd7ca4fef49" target='_blank'>
              Abstract PR007: Heterotypic CD8 T cell clusters isolated from clinical samples are distinct and enriched for antitumor activity
              </a>
            </td>
          <td>
            D. Peeper, Sofía Ibáñez-Molero, Johanna Veldman, Juan Simon-Nieto, J. Traets, Austin George, K. Hoefakker, Anita Karomi, Jonas Nilsson, John Haanen, W. V. van Houdt
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) represent a central cell population of the tumor microenvironment (TME). Recently, single‐cell RNA‐sequencing (scRNA‐seq) analyses of primary tumors of different cancer entities yielded different classifications of CAF subsets underscoring the heterogeneity of CAFs within the TME. Here, we analyzed the transcriptional signatures of approximately 8400 CAFs and normal fibroblasts by scRNA‐seq and compared genetic profiles of CAFs from murine melanoma primary tumors to CAFs from corresponding melanoma lung metastases. This revealed distinct subsets for primary tumor and metastasis‐specific CAF populations, respectively. Combined with the spatial characterization of metastasis CAFs at the RNA and protein level, scRNA analyses indicate tumor‐dependent crosstalk between neutrophils and CAFs, mediated via SAA3 and IL1b‐related signaling pathways, which can be recapitulated in vitro. Analyzing tissue sections of human patient samples, this interaction was found to be present in human melanoma metastasis. Taken together, our data highlight unique characteristics of metastasis CAFs with potential therapeutic impact for melanoma metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae4d264150cbc5dcfaa11bf8a1b1f48e7dcf511" target='_blank'>
              A Unique Signature for Cancer‐Associated Fibroblasts in Melanoma Metastases
              </a>
            </td>
          <td>
            Saskia Tauch, Joschka Hey, Bettina Kast, N. Gengenbacher, Lena Weiß, Melanie Sator‐Schmitt, S. Lohr, Alexander Brobeil, P. Schirmacher, Jochen Utikal, H. Augustin, C. Plass, Peter Angel
          </td>
          <td>2025-02-09</td>
          <td>Pigment Cell & Melanoma Research</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="
 Purpose: The purpose of this study is to determine how T cell engagers (TCEs) impact the localization and phenotype of key T-cell subsets in preclinical solid tumor models. Background: TCEs are a class of immunotherapy that have demonstrated substantial benefit to patients with hematologic cancers and have shown evidence of clinical activity in solid tumor malignancies. Mechanisms of response to TCEs, particularly in solid tumors, are not well understood. Identification of diagnostic strategies to stratify patients that are more likely to respond is expected to enhance the clinical benefit of TCEs. Friedrich et al. (Cancer Cell 2023) found that effector CD8 T cell subsets significantly expanded in TCE responsive but not refractory patients in multiple myeloma using single cell RNA sequencing (scRNAseq). Furthermore, the increased abundance of exhausted T cells at baseline was associated with a worse prognosis. We hypothesize that analogous populations exist in preclinical solid tumor models, which have distinct localization and cellular interactions within the TCE treated tumor microenvironment. Methods: Either hCD3 transgenic C57BL/6J or wildtype FVB mice were implanted with either ID8-LyPD1 or Fo5-HER2 tumors. Mice were grouped at a tumor volume of ∼ 300mm^3 and treated with either 10mpk of LyPD1 TCE, 5mpk of HER2 TCE, or vehicle. scRNAseq data was produced at day 1 and 4 post-treatment according to the 10X gene expression kit protocol and was processed in R using Seurat. Flow cytometry data was generated using commercially available antibodies, acquired on a BD FACSymphony and analyzed using FlowJo. Xenium data was generated using a custom probe panel according to the 10X Xenium protocol and was processed in either Python or R using the Squidpy or Seurat packages. Results: In the ID8-LyPD1 TCE model, we identified proliferative effector (CD8.pe) and terminally differentiated (CD8.Tex) CD8 populations by scRNAseq. The abundance of CD8.pe increased 3-fold and CD8.Tex expanded 2-fold upon TCE treatment. We then validated the expansion of these CD8 subsets by flow cytometry. Notably, increases in the number of CD8.pe per gram of tumor significantly correlated with reductions in tumor volume. Due to the impact of TCE treatment on these CD8 subsets and the importance of CD8.pe in tumor outcomes, we profiled the localization and cellular interactions of CD8s in preclinical solid tumors. Cell-cell communication inference analysis suggested that CD8s were more frequently targets of cellular interactions in TCE treated tumors. Utilizing Xenium spatial transcriptomics we verified that CD8.pe and CD8.Tex were more abundant in TCE treated samples and that their interactions with other key cell subsets were enhanced by TCE treatment. Conclusions: In summary we found that key CD8 subsets were expanded by TCE treatment in preclinical solid tumor models, that the abundance of CD8.pe correlates with reduced tumor burden, and that TCE treatment promotes cellular interactions among CD8s and other key cell subsets in the tumor microenvironment.
 Citation Format: Billy Tomaszewski, Matthew Curtis, Conrad Foo, Gu Zhang, Patrick Chang, Robyn Clark, Thao Nguyen, Joshua Webster, Sandra Rost, Raj Jesudason, Robert Piskol, Klara Totpal, Lisa McGinnis, Kevin Walsh, Weilan Ye. T cell engager therapy affects the spatial distribution and phenotype of T cells in the tumor microenvironment [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr PR002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce737a0ad997a56048c24df55247b712e440df3d" target='_blank'>
              Abstract PR002: T cell engager therapy affects the spatial distribution and phenotype of T cells in the tumor microenvironment
              </a>
            </td>
          <td>
            Billy Tomaszewski, Matthew Curtis, Conrad Foo, Gu Zhang, Patrick S Chang, Robyn Clark, Thao Nguyen, Joshua Webster, Sandra Rost, Raj Jesudason, Robert Piskol, Klara Totpal, Lisa Mcginnis, Kevin B Walsh, Weilan Ye
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, this study develops engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples reveal the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f073ff296b054e7e4c285a38a9d240d687627e67" target='_blank'>
              Genetically Engineered Brain Organoids Recapitulate Spatial and Developmental States of Glioblastoma Progression.
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A DeSouza, Xuan Qu, Sonika Dahiya, Albert H Kim, J. Millman
          </td>
          <td>2025-01-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The link between the “stress phenotype”-a well-established hallmark of cancer-and its role in tumor progression and intratumor heterogeneity remains poorly defined. The integrated stress response (ISR) is a key adaptive pathway that enables tumor survival under oncogenic stress. While ISR has been implicated in promoting tumor growth, its precise role in driving tumor evolution and heterogeneity has not been elucidated. In this study, using a genetically engineered mouse models, we demonstrate that ISR activation—indicated by elevated levels of phosphorylated eIF2 (p-eIF2) and ATF4—is essential for the emergence of dedifferentiated, therapy-resistant cell states. ISR, through the coordinated actions of ATF4 and MYC, facilitates the development of tumor cell populations characterized by high plasticity, stemness, and an epithelial-mesenchymal transition (EMT)-prone phenotype. This process is driven by ISR-mediated expression of genes that maintain mitochondrial integrity and function, critical for sustaining tumor progression. Importantly, genetic, or pharmacological inhibition of the p-eIF2–ATF4 signaling axis leads to mitochondrial dysfunction and significantly impairs tumor growth in mouse models of lung adenocarcinoma (LUAD). Moreover, ISR-driven dedifferentiation is associated with poor prognosis and therapy resistance in advanced human LUAD, underscoring ISR inhibition as a promising therapeutic strategy to disrupt tumor evolution and counteract disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Epithelial-mesenchymal transition (EMT) occurs heterogeneously among malignant carcinoma cells to promote chemoresistance. Identifying the signaling pathways involved will nominate drug combinations to promote chemoresponse, but cell population-level studies are inherently fraught, and single-cell transcriptomics are limited to indirect ontology-based inferences. To understand EMT heterogeneity at a signaling protein level, we combined iterative indirect immunofluorescence imaging of pancreas cancer cells and tumors and mutual information (MI) modeling. Focusing first on MAP kinase pathways, MI predicted that cell-to-cell variation in ERK activity surprisingly dominated control of EMT heterogeneity in response to diverse growth factors and chemotherapeutics, but that JNK compensated when MEK was inhibited. Population-level models could not capture these experimentally validated MI predictions. The dominant role of ERK was predicted by MI even when analyzing seven potential EMT-regulating signaling nodes. More generally, this work provides an approach for studying highly multivariate signaling/phenotype relationships based on protein measurements in any setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a6514737cd599ba0d5ecbb0df4e4d91b74a1b1" target='_blank'>
              ERK plays a conserved dominant role in pancreas cancer cell EMT heterogeneity driven by diverse growth factors and chemotherapies
              </a>
            </td>
          <td>
            Michelle C. Barbeau, Brooke A. Brown, S. Adair, Todd W. Bauer, Matthew J Lazzara
          </td>
          <td>2025-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f453139c5abae56333acfe86ef24c64b6c8b8390" target='_blank'>
              A multimodal imaging pipeline to decipher cell-specific metabolic functions and tissue microenvironment dynamics.
              </a>
            </td>
          <td>
            S. Venkateswaran, Peter Kreuzaler, Catherine MacLachlan, Greg McMahon, Gina Greenidge, Lucy Collinson, Josephine Bunch, Mariia Yuneva
          </td>
          <td>2025-01-29</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastasis, the leading cause of cancer-related deaths, involves a complex cascade of events, including extravasation. Despite extensive research into metastasis, the mechanisms underlying extravasation remain unclear. Molecular targeted therapies have advanced cancer treatment, yet their efficacy is limited, prompting exploration into novel therapeutic targets. Here, we showed the association of polyploidy in MDA-MB-231 breast cancer cells and their extravasation, using microfluidic systems to reproduce the in vivo microvascular environment. We observed enhanced extravasation in polyploid cells alongside upregulated expression of genes involved in cell-substrate adhesion and cell mechanical dynamics. These findings offer insights into the relationship between polyploidy and extravasation, highlighting potential targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785941742bc350defbe67d5915862c497379ab77" target='_blank'>
              Polyploidy of MDA-MB-231 cells drives increased extravasation with enhanced cell-matrix adhesion
              </a>
            </td>
          <td>
            Satomi Hirose, Tatsuya Osaki, Roger D. Kamm
          </td>
          <td>2025-01-29</td>
          <td>APL Bioengineering</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Cancer stem cells (CSCs) are pivotal in tumor growth, recurrence, and resistance to standard therapies. This review delves into CSCs’ ability to self-renew and differentiate, fueling tumor heterogeneity and metastasis. It highlights critical signaling pathways like Wnt, Notch, and Hedgehog, which are essential for targeting CSCs. Emerging technologies, such as CRISPR/Cas9 and single-cell sequencing, promise to enhance CSC research and management. The review also explores the complex interaction between CSCs and the tumor microenvironment (TME), which influences therapeutic effectiveness. Addressing CSC-driven resistance, the article proposes advanced strategies for overcoming these challenges. It emphasizes the role of artificial intelligence in designing personalized, CSC-targeted therapies, ushering in a new era for precision medicine in oncology. This comprehensive approach aims to improve cancer treatment outcomes through innovative technologies and inter-disciplinary collaboration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99151138c6fc7abf05753313b772a7f6497b1128" target='_blank'>
              Deciphering the Role of Cancer Stem Cells: Drivers of Tumor Evolution, Therapeutic Resistance, and Precision Medicine Strategies
              </a>
            </td>
          <td>
            Mohamed El-Tanani, S. Rabbani, S. Satyam, I. Rangraze, A. Wali, Yahia El-Tanani, Alaa A. A. Aljabali
          </td>
          <td>2025-01-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Invasion is a prerequisite for metastasis formation. During tumor development, the extracellular matrix (ECM) is remodeled in part through overexpression of type I collagen, increasing tumor microenvironment stiffness, and facilitating cancer dissemination. During breast cancer cell migration, we observed membrane debris left behind, attached to the collagen fibrils, along the migration path. We named these structures collagen-tracks. These collagen-tracks can be deposited in 3D matrices in vitro and in vivo and their formation is stimulated by the interaction between the ECM and matrix receptors, such as the discoidin-domain receptor (DDR1). However, they are different from structures already known to be involved in cell-cell communication such as exosomes and migrasomes, due to their specific nucleic acid and protein contents. When deposited by highly invasive breast cancer cells, internalized collagen-tracks reprogram non-invasive cells into highly pro-metastatic ones by inducing a partial epithelial–mesenchymal transition (EMT). This cell reprogramming is dependent on specific miRNAs present in the collagen-tracks, that are necessary to promote ECM degradation, increase cell motility and invasiveness. Collagen-tracks thus represent a new form of cell-cell communication important for driving tumor invasion that could be targeted to prevent metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607709bacd540192f2a53b41a2c4eccacb7b266e" target='_blank'>
              Cancer cells transfer invasive properties through microRNAs contained in collagen-tracks
              </a>
            </td>
          <td>
            Léa Normand, L. Rouyer, Elodie Richard, Nathalie Allain, Julie Giraud, Sylvaine Di-Tommaso, C. Dourthe, A. Raymond, Jean-William Dupuy, Kevin Moreau, Richard Iggo, Gaetan McGrogan, Nathalie Dugot-Senant, Anthony Bouter, Sisareuth Tan, Alexandre Favereaux, Violaine Moreau, R. Luco, M. Ros, F. Saltel
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Many tumors evolve through cellular mutation and selection, where subpopulations of cells (subclones) with shared ancestry compete for dominance. Introduction of next generation sequencing enables subclone identification using small somatic. However, there are several advantages to marking subclones with structural variants: they have greater functional impact, play a crucial role in late-stage tumors, and provide a more complete view of genomic instability driving tumor evolution. Here, we present SVCFit, a scalable method to estimate the cellular prevalence of somatic deletions, duplications and inversions. We demonstrate that cellular prevalence estimation can be improved by incorporating distinct read patterns for each structural variant type. Additionally, this improvement is achieved without prior knowledge of tumor purity, which is often inaccurate. Using a combination of simulated data and patient-derived metastatic samples with known mixture proportions, we show that our algorithm achieves significantly greater accuracy than state-of-the-art in estimating the structural variants cellular prevalence (p<0.05). The speed of SVCFit estimation from cost-effective bulk whole-genome sequencing (WGS) makes it well-suited for analyzing large cohorts of sequenced tumor samples, enhancing the accessibility of SV-based clonal reconstruction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77e71e2d20d9e52879b8c2d8cc5014dd16b7a335" target='_blank'>
              SVCFit: Inferring structural variant cellular fraction in tumors
              </a>
            </td>
          <td>
            Yunzhou Liu, Jiaying Lai, Laura D. Wood, Rachel Karchin
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The epidermal basal layer consists of two distinct progenitor populations contributing differentially to skin homeostasis or wound healing. However, their resilience to oncogenic insults is unknown. Here, we developed mouse models allowing lineage tracing and oncogene expression in both populations. Our findings support the existence of two long-lived and functionally distinct epidermal progenitors and define them as tumour-primed and tumour-resistant. The tumour-primed progenitors drive cutaneous squamous cell carcinoma (cSCC), within days from oncogenic expression resembling the rapid onset of BRAF-inhibitor-driven cSCC. The tumour-resistant progenitors transform gradually, requiring SOX2 activation. Once transformed, they share a transcriptomic profile revealing molecular hallmarks of epidermal transformation independently of the cell-of-origin or the oncogenic driving mutations. Importantly, SOX2 is expressed in a subset of cSCC patients hinting at its cell-of-origin. These highlight the need for an in-depth understanding of the cell-specific vulnerabilities that apply in each tissue to open new avenues for rational treatment. STATEMENT OF SIGNIFICANCE Using in vivo models, we define tumour-primed and tumour-resistant epidermal progenitors and reveal SOX2-driven reprogramming as a driver of tumorigenesis in resistant cells. SOX2 expression in a subset of cSCC patients links to cell-of-origin, uncovering a cell-specific vulnerability that advances our understanding of cancer initiation and progression in epithelial tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8c6ee68b135f15e99b33463b50a273c96e7d816" target='_blank'>
              Epidermal tumour-resistant progenitors require SOX2-driven developmental reprogramming to drive tumorigenesis
              </a>
            </td>
          <td>
            Patricia P. Centeno, Christopher Chester, Catriona A. Ford, Rachel A Ridgway, P. Cammareri, Thomas Jamieson, Richard Marais, A. Campbell, Owen J. Sansom
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The spatial organization of adaptive immune cells within lymph nodes is critical for understanding immune responses during infection and disease. Here, we introduce AIR-SPACE, an integrative approach that combines high-resolution spatial transcriptomics with paired, high-fidelity long-read sequencing of T and B cell receptors. This method enables the simultaneous analysis of cellular transcriptomes and adaptive immune receptor (AIR) repertoires within their native spatial context. We applied AIR-SPACE to mouse popliteal lymph nodes at five distinct time points after Vaccinia virus footpad infection and constructed a comprehensive map of the developing adaptive immune response. Our analysis revealed heterogeneous activation niches, characterized by Interferon-gamma (IFN-γ) production, during the early stages of infection. At later stages, we delineated sub-anatomical structures within the germinal center (GC) and observed evidence that antibody-producing plasma cells differentiate and exit the GC through the dark zone. Furthermore, by combining clonotype data with spatial lineage tracing, we demonstrate that B cell clones are shared among multiple GCs within the same lymph node, reinforcing the concept of a dynamic, interconnected network of GCs. Overall, our study demonstrates how AIR-SPACE can be used to gain insight into the spatial dynamics of infection responses within lymphoid organs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ff2bb39af0926fb7ea6baef50f5c517a318f30" target='_blank'>
              A Temporal and Spatial Atlas of Adaptive Immune Responses in the Lymph Node Following Viral Infection
              </a>
            </td>
          <td>
            Shaowen Jiang, Madhav Mantri, Viviana I. Maymi, S. Leddon, Peter Schweitzer, Subash Bhandari, Chase Holdener, Ioannis Ntekas, Christopher Vollmers, A. Flyak, Deborah J. Fowell, Brian D. Rudd, I. De Vlaminck
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Quiescence, a reversible state of cell-cycle arrest, is an adaptive feature of many adult tissue stem cells, including those in the adult brain. In gliomas, brain tumour stem cells that reside in a quiescent state preferentially survive chemotherapy and radiotherapy, highlighting their critical role in therapy resistance and disease progression. To date, it remains unclear whether the molecular programs governing these states are functionally conserved between neural stem cells and brain tumour stem cells. Here, we establish novel in vitro models to study quiescence and find that glioma stem cells are markedly more resistant to entering quiescence than neural stem cells, suggesting that glioma stem cell quiescence more closely resembles a slow-cycling phenotype or shallow quiescence. Nonetheless, direct comparison of quiescent neural stem cells and quiescent/slow-cycling glioma stem cells, as they transition towards proliferation, reveals conserved gene expression trajectories, indicating shared molecular mechanisms. Furthermore, we find that pathways influencing quiescence in neural stem cells exert similar effects in glioma stem cells, underscoring the functional parallels between these populations. Finally, we identify that inhibition of TGF-β signalling might provide an avenue to improve current standard-of-care treatments by targeting quiescent glioma stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfaba977c1ae379d70e521eff155f6c1e15b6182" target='_blank'>
              Shared molecular regulation of quiescence in neural and glioma stem cells reveals therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Chandra Choudhury, Matthew Singleton, Stephanie Brauer, Dana Friess, Jessica Hart, N. Skarne, Karthik Pullela, Likai Mao, Bryan W Day, Lachlan Harris
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intra-tumor heterogeneity (ITH) is a compounding factor for cancer prognosis and treatment. Single-cell DNA sequencing (scDNA-seq) provides cellular resolution of the variations in a cell and has been widely used to study cancer progression and responses to drug and treatment. While the low coverage scDNA-seq technologies typically provides a large number of cells, accurate cell clustering is essential for effectively characterizing ITH. Existing cell clustering methods typically are based on either single nucleotide variations (SNV) or copy number alterations (CNA), without leveraging both signals together. Since both SNVs and CNAs are indicative of the cell subclonality, in this paper, we designed a robust cell clustering tool that integrates both signals using a graph autoencoder. Our model co-trains the graph autoencoder and a graph convolutional network (GCN) to guanrantee meaningful clustering results and to prevent all cells from collapsing into a single cluster. Given the low dimensional embedding generated by the autoencoder, we adopted a Gaussian Mixture Model to further cluster cells. We evaluated our method on eight simulated datasets and a real cancer sample. Our results demonstrate that our method consistently achieves higher V-measure scores compared to SBMClone, a SNV-based method, and a K-means method, which relies solely on CNA signals. These findings highlight the advantage of integrating both SNV and CNA signals within a graph autoencoder framework for accurate cell clustering. SCGclust is publicly available at https://github.com/compbio-mallory/cellClustering_GNN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e64812a9680cffb699fbd114fbd628509e7f6b4c" target='_blank'>
              SCGclust: Single Cell Graph clustering using graph autoencoders integrating SNVs and CNAs
              </a>
            </td>
          <td>
            Teja Potu, Yunfei Hu, Rituparna Khan, S. Dharani, Jingchao Ni, Liting Zhang, X. Zhou, Xian Mallory
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The CHI3L1 signaling pathway significantly influences glioma angiogenesis, but its role in the tumor microenvironment (TME) remains elusive. We propose a novel CHI3L1-associated vascular phenotype classification for glioma through integrative analyses of multiple datasets with bulk and single-cell transcriptome, genomics, digital pathology, and clinical data. We investigated the biological characteristics, genomic alterations, therapeutic vulnerabilities, and immune profiles within these phenotypes through a comprehensive multi-omics approach. We constructed the vascular-related risk (VR) score based on CHI3L1-associated vascular signatures (CAVS) identified by machine learning algorithms. Utilizing unsupervised consensus clustering, gliomas were stratified into three distinct vascular phenotypes: Cluster A, marked by high vascularization and stromal activation with a relatively low levels of tumor-infiltrating lymphocytes (TILs); Cluster B, characterized by moderate vascularization and stromal activity, coupled with a high density of TILs; and Cluster C, defined by low vascularization and sparse immune cell infiltration. We observed that the CAVS effectively indicated glioma-associated angiogenesis and immune suppression by single-cell RNA-seq analysis. Moreover, the high-VR-score group exhibited enhanced angiogenic activity, reduced immune response, resistance to immunotherapy, and poorer clinical outcomes. The VR score independently predicted glioma prognosis and, combined with a nomogram, provided a robust clinical decision-making tool. Potential drug prediction based on transcription factors for high-risk patients was also performed. Our study reveals that CHI3L1-associated vascular phenotypes shape distinct immune landscapes in gliomas, offering insights for optimizing therapeutic strategies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec81235338a787d945989660c2ddefa3be4c0bd" target='_blank'>
              Novel CHI3L1-Associated Angiogenic Phenotypes Define Glioma Microenvironments: Insights From Multi-Omics Integration.
              </a>
            </td>
          <td>
            Yu-Hang Zhao, Yu-Xiang Cai, , F. Tang, Chao Ma, Ze-Fen Wang, Gang Li, Hang Chang, Su-Fang Tian, Zhi-Qiang Li
          </td>
          <td>2025-02-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Purpose
In glioblastoma, the therapeutically intractable and resistant phenotypes can be derived from glioma stem cells, which often have different underlying mechanisms from non-stem glioma cells. Aberrant signaling across the EGFR-PTEN-AKT-mTOR pathways have been shown as common drivers of glioblastoma. Revealing the inter and intra-cellular heterogeneity within glioma stem cell populations in relations to signaling patterns through these pathways may be key to precision diagnostic and therapeutic targeting of these cells.


Materials and Methods
Single cell parallel proteomic heterogeneity profiling of the EGFR-PTEN-AKT-mTOR pathways was conducted in a panel of fifteen glioma stem cell models derived from patient glioblastoma biopsies.


Results
The analysis included 59,464 data points from 14,866 cells and identified forty-nine molecularly distinct signaling phenotypes. High content bioinformatics resolved two unique patient clusters diverging on EGFR expression and AKT/TORC1 activation. Phenotypic validation indicated drug responsive phenotypes to EGFR blocking in the high EGFR expressing cluster with lower tumor initiating potential in comparison to the AKT/TORC1 activated cluster. High EGFR expression trended with improved patient prognosis while AKT/TORC1 activated samples trended with poorer patient outcomes. Genetic heterogeneity was observed in both clusters with proneural, classical and mesenchymal subtypes observed.


Conclusion
Quantitative single cell heterogeneity profiling reveals divergent EGFR-PTEN-AKT-mTOR pathways of patient derived glioma stem cells, which would inform future research and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0984500cc5c038ee7f954e1d13785419da86e8c6" target='_blank'>
              Single-Cell Heterogeneity of EGFR Pathway Is Linked to Unique Signatures of Drug Response and Malignancy in Patient Derived Glioblastoma Stem Cells.
              </a>
            </td>
          <td>
            Michael D. Masterman-Smith, Nicholas A. Graham, E. Panosyan, J. Mottahedeh, Eric R. Samuels, Araceli Nunez, Tiffany Phillips, Meeryo Choe, T. Cloughesy, Jorge A. Lazareff, Linda M. Liau, William H. Yong, Thomas G. Graeber, H. Kornblum, Ming-Fei Lang, Yanzhao Li, Jing Sun
          </td>
          <td>2025-01-07</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Atherosclerosis involves complex interactions between lipids, immune cells, vascular smooth muscle cells (VSMCs), and fibroblasts within the arterial wall. While significant advances in single-cell technologies have shed light on the roles of immune cells and VSMCs in plaque development, fibroblasts remain underexplored, leaving critical gaps in understanding their contributions to disease progression and plaque stability. Comprehensive characterization of fibroblast phenotypes in atherosclerosis is essential to unravel their diverse functions and to distinguish between subsets that may play protective versus pathogenic roles in the disease process. Methods Here, we utilized CITE-seq (Cellular Indexing of Transcriptomes and Epitopes by Sequencing) to comprehensively profile fibroblast diversity in a mouse model of atherosclerosis. Mice were fed an atherogenic diet for 0, 8, 19, and 26 weeks, representing distinct stages of disease progression, enabling a detailed phenotypic characterization of fibroblasts throughout the course of atherosclerosis development. Results We identified four distinct fibroblast subpopulations, including a myofibroblast population closely resembling VSMC-derived chondromyocytes. The proportions of these fibroblast subsets exhibited a modest decline as atherosclerosis progressed. Through multimodal analysis, we identified CD26 as a highly expressed and specific marker for one of these fibroblast subpopulations, distinguishing it from other subsets. Using a combination of flow cytometry and immunohistochemistry, we demonstrated that CD26+ fibroblasts predominantly reside in the adventitia of healthy arteries. During atherosclerosis progression, these cells expand into the intima and primarily localize within the fibrous cap of the lesion. Conclusions Our multi-omic analysis highlights the phenotypic diversity and dynamic changes of fibroblasts during atherosclerosis progression. Among these, CD26+ fibroblasts emerge as a distinct subpopulation that expands within atherosclerotic lesions and may play a critical role in promoting plaque stability through their migration into the fibrous cap.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcefa069d1b01c8875c90b650f37b9a970f0c89d" target='_blank'>
              Single-Cell Multimodal Profiling Reveals a Novel CD26+ Fibroblast Subpopulation in Atherosclerosis
              </a>
            </td>
          <td>
            A. Bashore, Johana Coronel, Chenyi Xue, Lucie Y. Zhu, Muredach P. Reilly
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47aaf83b6567ca271ef456f540a98ac5c9738676" target='_blank'>
              Comparative analysis of pediatric SHH medulloblastoma DAOY spheres and adherent monolayers: implications for medulloblastoma research
              </a>
            </td>
          <td>
            Jana Klementová, Šárka Jarošová, Irina Danilová, Markéta Farníková, Josef Novotný, M. Davidkova, Martina Zíková
          </td>
          <td>2025-01-22</td>
          <td>Cancer Cell International</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC) is characterized by a limited infiltration of tumor-specific T cells and anti-tumor T cell activity. Extracellular factors in the PDAC TME have been widely reported to mediate immune suppression, but the contribution from tumor-intrinsic factors is not well understood. The RNA-binding protein, HuR (ELAVL1), is enriched in PDAC and negatively correlates with T cell infiltration. In an immunocompetent Kras-p53-Cre (KPC) orthotopic model of PDAC, we found that genetic disruption of HuR impaired tumor growth. Importantly, we found that HuR depletion in tumors enhanced both T cell number and activation states. Mechanistically, HuR mediated the stabilization of mTOR pathway transcripts, and inhibition of mTOR activity rescued the impaired function of local T cells. Phenotypically, we found that HuR induced T-cell suppression in PDAC, as HuR depletion sensitize PDAC tumors to immune checkpoint blockade, while isogenic, wildtype tumors are resistant. Our findings describe a novel role of HuR in facilitating tumor immune suppression in the PDAC TME by inhibiting T cell infiltration and function, and implicate HuR inhibition as a potential therapeutic combination with immunotherapy. SIGNIFICANCE This study identified a novel mechanism that HuR supports pancreatic tumor growth by restricting T cell infiltration, promoting immune evasion. Our work supports targeting strategies against HuR in PDAC with the goal of enhancing PDAC sensitivity to immune-based cancer therapies, such as checkpoint blockade and T cell transfer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5ca2919cbf47a322acf524c95727ed063316d71" target='_blank'>
              mRNA stability factor HuR promotes immune evasion in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yifei Guo, Jennifer M. Finan, Alexandra Q. Bartlett, Shamaline Sivagnanam, Katie E. Blise, Nell Kirchberger, Konjit Betre, Kevin Hawthorne, Canping Chen, Aaron J. Grossberg, Zheng Xia, L. Coussens, Rosalie C. Sears, Jonathan R. Brody, Robert Eil
          </td>
          <td>2025-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Gamma-delta (γδ) T cells are a crucial component of the tumor immune microenvironment which are considered a promising potential therapeutic strategy and target. Increasing evidence suggests that these unique immune cells play significant roles across various cancers. However, γδ T cells are often regarded as having dual roles in tumors, and their influence on lung adenocarcinoma (LUAD) remains controversial. In this research, we employed a wild-ranging approach using multi-omics data to investigate the function of γδ T cells in LUAD. The abundance of γδ T cell infiltration is linked to a positive prognosis in lung adenocarcinoma. The tumor-inhibiting role of γδ T cells was played through intrinsic lineage evolution, acquiring cytotoxic functions and engaging in signal transduction with antigen-presenting cells. Furthermore, patients with higher γδ T cells infiltration abundance might be more favorable for immunotherapy. Lastly, we established a predictive model using CT images based on radiomics, providing a non-invasive strategy to assess γδ T cells infiltration in LUAD patients. These findings offer new insights and perspectives the personalized therapies of γδ T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ada3c2d9795ca814751eeb983c8b55cf90854b2" target='_blank'>
              Integrating of Radiomics and Single-cell Transcriptomics Uncovers the Crucial Role of γδ T Cells in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Ziyi Chen, Changqing Yang, Jiajing Wang, Baichuan Wang, Long He, Zhaoyi Liu, Yingxi Li, Tingting Qin, Peng Chen
          </td>
          <td>2025-01-06</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Human genome sequencing efforts in healthy and diseased individuals continue to identify a broad spectrum of genetic variants associated with predisposition, progression, and therapeutic outcomes for diseases like cancer1–6. Insights derived from these studies have significant potential to guide clinical diagnoses and treatment decisions; however, the relative importance and functional impact of most genetic variants remain poorly understood. Precision genome editing technologies like base and prime editing can be used to systematically engineer and interrogate diverse types of endogenous genetic variants in their native context7–9. We and others have recently developed and applied scalable sensor-based screening approaches to engineer and measure the phenotypes produced by thousands of endogenous mutations in vitro10–12. However, the impact of most genetic variants in the physiological in vivo setting, including contextual differences depending on the tissue or microenvironment, remains unexplored. Here, we integrate new cross-species base editing sensor libraries with syngeneic cancer mouse models to develop a multiplexed in vivo platform for systematic functional analysis of endogenous genetic variants in primary and disseminated malignancies. We used this platform to screen 13,840 guide RNAs designed to engineer 7,783 human cancer-associated mutations mapping to 489 endogenous protein-coding genes, allowing us to construct a rich compendium of putative functional interactions between genes, mutations, and physiological contexts. Our findings suggest that the physiological in vivo environment and cellular organotropism are important contextual determinants of specific gene-variant phenotypes. We also show that many mutations and their in vivo effects fail to be detected with standard CRISPR-Cas9 nuclease approaches and often produce discordant phenotypes, potentially due to site-specific amino acid selection- or separation-of-function mechanisms. This versatile platform could be deployed to investigate how genetic variation impacts diverse in vivo phenotypes associated with cancer and other genetic diseases, as well as identify new potential therapeutic avenues to treat human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f71f3527a2a35f380d7482a2555f7e6eeef4ab" target='_blank'>
              Multiplexed in vivo base editing identifies functional gene-variant-context interactions
              </a>
            </td>
          <td>
            Jonuelle Acosta, Grace A. Johnson, Samuel I. Gould, Kexin Dong, Yovel Lendner, Diego Detrés, Ondine Atwa, Jari Bulkens, Samuel Gruber, Manuel E. Contreras, Alexandra N. Wuest, Varun K. Narendra, Michael T. Hemann, Francisco J. Sánchez-Rivera
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed580d09e5d8a50632bdd53ac296d7f45feff0d1" target='_blank'>
              The microenvironment cell index is a novel indicator for the prognosis and therapeutic regimen selection of cancers
              </a>
            </td>
          <td>
            Xian-Yan Yang, Nian Chen, Qian Wen, Yu Zhou, Tao Zhang, Ji Zhou, Cheng-Hui Liang, Li-Ping Han, Xiao-Ya Wang, Qingmei Kang, Xiaomei Zheng, Xue-Jia Zhai, Hong-Ying Jiang, Tian-Hua Shen, Jin-Wei Xiao, Yu-Xin Zou, Yun Deng, Shuang Lin, Jiang-jie Duan, Jun Wang, Shicang Yu
          </td>
          <td>2025-01-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43c4186232b1945af1f5d2a2e6f5e0453c760e7a" target='_blank'>
              Periostin-mediated NOTCH1 activation between tumor cells and HSCs crosstalk promotes liver metastasis of small cell lung cancer
              </a>
            </td>
          <td>
            Linlin Lou, Keren Peng, Shumin Ouyang, Wen Ding, Jian-shan Mo, Jiayu Yan, Xiaoxiao Gong, Guopin Liu, Jin-Jian Lu, Peibin Yue, Kai Zhang, Jian Zhang, Yan-dong Wang, Xiao-lei Zhang
          </td>
          <td>2025-01-07</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33631c265f68921d79fbd004eb59ad3a5ea83b81" target='_blank'>
              Spatially mapping the tumour immune microenvironments of non-small cell lung cancer
              </a>
            </td>
          <td>
            Lysanne Desharnais, Mark Sorin, M. Rezanejad, Bridget Liu, E. Karimi, Aline Atallah, Anikka M Swaby, Miranda W. Yu, Samuel Doré, Saskia Hartner, B. Fiset, Yuhong Wei, Baharak Kadang, R. Rayes, Philippe Joubert, Sophie Camilleri-Broët, P. Fiset, D. Quail, Jonathan D Spicer, L. Walsh
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183356cbf79c71bfdd985789fe82d5c12c548853" target='_blank'>
              Gemistocytic tumor cells programmed for glial scarring characterize T cell confinement in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            L. van Hijfte, M. Geurts, I. D. de Heer, S. Ghisai, H. Balcioglu, Y. Hoogstrate, Wies R Vallentgoed, R. Head, Rosa Luning, Thierry van den Bosch, Bart A Westerman, Pieter Wesseling, J. Joyce, P. French, R. Debets
          </td>
          <td>2025-01-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Metastatic cancer, a major cause of mortality, has been understudied compared to primary tumors, leaving gaps in our understanding of how cancer genes adapt between these states. We analyzed the association between mutations and copy-number alterations in 25,000 tumor samples from both primary and metastatic cancers. Our findings show that cancer genes display distinct interaction strengths across these states, with 27.45% of genes, including ARID1A, FBXW7, and SMARCA4, shifting between one-hit and two-hit drivers. Interaction strengths varied by cancer state and treatment conditions, revealing seven state-specific interactions. We also identified 38 primary-specific and 21 metastatic-specific high-order interactions, enriched in cancer hallmarks, indicating unique tumor progression mechanisms. These findings highlight dynamic tumor progression mechanisms and underscore the importance of considering cancer state in research and treatment strategies for precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968f72ea3f3d388ad8546b594ad1735637aaf86d" target='_blank'>
              Mapping cancer gene dynamics through state-specific interactions
              </a>
            </td>
          <td>
            Adrián Maqueda-Real, Laia Ollé-Monràs, Solip Park
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d2a730533f465c1760e5f39354ae03ee2bf1e3c" target='_blank'>
              Spatial expression of fibroblast activation protein-α in clear cell renal cell carcinomas revealed by multiplex immunoprofiling analysis of the tumor microenvironment
              </a>
            </td>
          <td>
            G. Larrinaga, M. Redrado, Ana Loizaga-Iriarte, Amparo Pérez-Fernández, Aida Santos-Martín, Javier C. Angulo, José A Fernández, Alfonso Calvo, José I. López
          </td>
          <td>2025-01-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Background: Spatial transcriptomics (ST) is revolutionizing our understanding of tumor heterogeneity by providing high-resolution, location-specific gene expression mapping across the tumor microenvironment (TME). Despite its transformative potential, ST’s high cost limits its widespread clinical application, particularly for large-scale biomarker discovery. Methods: To address these challenges, we developed Path2Space, a deep learning model trained on ST data, to predict spatial gene expression directly from histopathology slides, offering an accessible alternative to experimental ST assays. First, Path2Space is validated using three independent spatial transcriptomics (ST) breast cancer cohorts, demonstrating robust and consistent gene expression prediction across these datasets. Second, the model’s predictions allow for detailed TME characterization, including the accurate inference of cell-type abundances and immune cell distributions within spatial domains. Thirdly and most importantly, we applied Path2Space to the TCGA breast cancer cohort to assess the prognostic potential of immune and TME spatial patterns. Results: Path2Space achieves unprecedented accuracy and coverage in robustly predicting the spatial expression of 4,500 genes across three independent spatial transcriptomics (ST) cohorts, markedly outperforming current methods by a factor of 10. Applied to the TCGA breast cancer cohort, we identify three patient subsets with distinct immune tumor microenvironments (TME) having distinct survival outcomes. The immune TME of the poor-prognosis subset has dense cancer cell populations with low immune infiltration and marked inactivation of specific key immune interactions. In two trastuzumab-treated breast cancer cohorts, Path2Space identifies spatial domains strongly associated with treatment response, with high tumor-infiltrating lymphocytes (TIL) presence, activation of immune pathways, and robust immune-driven cell-cell interactions, enabling the prediction of patient response more accurately than that based on the measured gene expression. Conclusions: Path2Space represents a groundbreaking approach in spatially mapping the tumor microenvironment directly from histopathology slides, paving the way for the study of the immune TME of different cancer types on an unprecedented scale. By enabling the cost-effective inference of the TME, Path2Space holds potential to transform cancer care across diverse healthcare settings, facilitating the development of targeted treatments and improving patient survival globally.
 Citation Format: Roshan Lodha, Eldad D Shulman, Emma M Campagnolo, Thomas Cantore, Tom Hu, MacLean Nasrallah, Danh-Tai Hoang, Kenneth Aldape, Eytan Ruppin. Predicting spatial transcriptomics from histopathology with Path2Space: AI charts breast cancer patients tumor immune microenvironment on an unprecedented scale [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B105.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20c3f321acba286bf84a8f4af78044117acbf8c" target='_blank'>
              Abstract B105: Predicting spatial transcriptomics from histopathology with Path2Space: AI charts breast cancer patients tumor immune microenvironment on an unprecedented scale
              </a>
            </td>
          <td>
            Roshan Lodha, E. Shulman, Emma M. Campagnolo, Thomas Cantore, Tom Hu, M. Nasrallah, Danh-Tai Hoang, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and aggressively progressing malignancy that presents considerable clinical challenges.Although chemotherapy can effectively manage symptoms during the earlystages of SCCE, its long-term effectiveness is notably limited, with theunderlying mechanisms remaining largely undefined. In this study, weemployed single-cell RNA sequencing (scRNA-seq) to analyze SCCE samplesfrom a single patient both before and after chemotherapy treatment. Our analysisrevealed significant cellular plasticity and alterations in the tumormicroenvironment’s cellular composition. Notably, we observed an increase intumor cell diversity coupled with reductions in T cells, B cells, and myeloid-likecells. The pre-treatment samples predominantly featured carcinoma cells in amiddle transitional state, while post-treatment samples exhibited an expandedpresence of cells in terminal, initial-to-terminal (IniTerm), and universally alteredstates. Further analysis highlighted dynamic interactions between tumor cells andimmune cells, with significant changes detected in key signaling pathways, suchas TIGIT-PVR and MDK-SDC4. This study elucidates the complex dynamics of cellplasticity in SCCE following chemotherapy, providing new insights and identifyingpotential therapeutic targets to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae84357d27c8ef8315335bde0ededde5ffcf7d8" target='_blank'>
              Single-cell RNA sequencing elucidates cellular plasticity in esophageal small cell carcinoma following chemotherapy treatment
              </a>
            </td>
          <td>
            Qinkai Zhang, Ziyu Gao, Ru Qiu, Jizhao Cao, Chunxiao Zhang, Wei Qin, Meiling Yang, Xinyue Wang, Ciqiu Yang, Jie Li, Dongyang Yang
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb406f40baccfe449ee7910f107c5384df35a006" target='_blank'>
              Spatially resolved transcriptomics reveals gene expression characteristics in uveal melanoma
              </a>
            </td>
          <td>
            Jing-Ying Xiu, Yu-Ning Chen, Ya-Li Mao, Jingqiu Luo, Hao-Wen Li, Yang Li, Wenbin Wei
          </td>
          <td>2025-01-15</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Neuroblastoma is a heterogeneous disease with adrenergic (ADRN)- and therapy resistant mesenchymal (MES)-like cells driven by distinct transcription factor networks. Here, we investigate the expression of immunotherapeutic targets in each neuroblastoma subtype and propose pan-neuroblastoma and cell state specific targetable cell-surface proteins.


METHODS
We characterized cell lines, patient-derived xenografts, and patient samples as ADRN-dominant or MES-dominant to define subtype-specific and pan-neuroblastoma gene sets. Targets were validated with ChIP-sequencing, immunoblotting, and flow cytometry in neuroblastoma cell lines and isogenic ADRN-to-MES transition cell line models. Finally, we evaluated the activity of MES-specific agents in vivo and in vitro.


RESULTS
Most immunotherapeutic targets being developed for neuroblastoma showed significantly higher expression in the ADRN subtype with limited expression in MES-like tumor cells. In contrast, CD276 (B7-H3) and L1CAM maintained expression across both ADRN and MES states. We identified several receptor tyrosine kinases (RTKs) enriched in MES-dominant samples and showed that AXL targeting with ADCT-601 was potently cytotoxic in MES-dominant cell lines and showed specific anti-tumor activity in a MES cell line-derived xenograft.


CONCLUSIONS
Immunotherapeutic strategies for neuroblastoma must address the potential of epigenetic downregulation of antigen density as a mechanism for immune evasion. We identified several RTKs as candidate MES-specific immunotherapeutic target proteins for the elimination of therapy-resistant cells. We hypothesize that the phenomena of immune escape will be less likely when targeting pan-neuroblastoma cell surface proteins such as B7-H3 and L1CAM, and/or dual targeting strategies that consider both the ADRN- and MES-cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ce67e86c3ed1eb1595b4d5bed2ad026b261bd59" target='_blank'>
              Lineage-dependence of the neuroblastoma surfaceome defines tumor cell state-dependent and independent immunotherapeutic targets.
              </a>
            </td>
          <td>
            N. Kendsersky, Michal Odrobina, Nathaniel W. Mabe, Alvin Farrel, Liron D. Grossmann, M. Tsang, David Groff, A. Wolpaw, Alaa Narch, F. Zammarchi, P. V. van Berkel, C. V. Dang, Y. Mossé, K. Stegmaier, John M. Maris
          </td>
          <td>2025-01-18</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Disruptions in uterine tissue function contribute to disorders such as endometriosis, adenomyosis, endometrial cancer, and fibroids, which all significantly impact health and fertility. Advances in transcriptomics, particularly single-cell RNA sequencing, have revolutionized uterine biological research by revealing the cellular heterogeneity and molecular mechanisms underlying disease states. Single-cell RNA sequencing and spatial transcriptomics have mapped endometrial and myometrial cellular landscapes, which helped to identify critical cell types, signaling pathways, and phase-specific dynamics. Said transcriptomic technologies also identified stromal and immune cell dysfunctions, such as fibroblast-to-myofibroblast transitions and impaired macrophage activity, which drive fibrosis, chronic inflammation, and lesion persistence in endometriosis. For endometrial cancer, scRNA-seq uncovered tumor microenvironmental complexities, identifying cancer-associated fibroblast subtypes and immune cell profiles contributing to progression and therapeutic resistance. Similarly, studies on adenomyosis highlighted disrupted signaling pathways, including Wnt and VEGF, and novel progenitor cell populations linked to tissue invasion and neuroinflammation, while single-cell approaches characterized smooth muscle and fibroblast subpopulations in uterine fibroids, elucidating their roles in extracellular matrix remodeling and signaling pathways like ERK and mTOR. Despite challenges such as scalability and reproducibility, single-cell transcriptomic approaches may have potential applications in biomarker discovery, therapeutic target identification, and personalized medicine in gynecological disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d46c8d721f3ec856f042d24cd830de014f3393b" target='_blank'>
              Mapping Human Uterine Disorders Through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Sandra Boldu-Fernández, Carolina Lliberos, Carlos Simón, A. Mas
          </td>
          <td>2025-01-21</td>
          <td>Cells</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genetic defects in NOX2 can cause chronic granulomatous disease (CGD), characterized by increased susceptibility to infections and pronounced inflammatory responses that lead to granuloma formation. We developed a CGD model using Ncf2-/- mice through controlled environmental exposure. Unlike in specific pathogen-free environment, these mice spontaneously developed pulmonary granulomas under clean-grade conditions. Employing single-cell RNA sequencing, we observed an expansion of neutrophils and monocyte-derived macrophages (MDMs) within the lung tissue, identifying pro-inflammatory NOS2high-neutrophils and a distinct MDMs subset marked by NOS2 and ARG1. Spatial transcriptomics demonstrated that NOS2high-neutrophils were located at the core area, while the MDMs subset was positioned at the periphery, facilitating extracellular matrix remodeling. Pharmacological targeting of MIF, deletion of the pro-survival gene Morrbid, and elimination of Il1r1 all suppressed granuloma formation by mitigating inflammation. This study underscores how environmental control can establish a natural CGD model, elucidates the mechanisms of granuloma formation, and develops potent therapeutic interventions. Highlights Through installing a mutation in a NOX2 subunit gene Ncf2 and changing the husbandry environment, we generated a natural CGD mouse model with progressive pulmonary granuloma formation. NOS2high neutrophils located in the core region of CGD lung granulomas, exhibit a pro-inflammatory transcriptional profile. Monocyte-derived macrophages marked by Nos2 and Arg1 are located at the periphery of granulomas and exhibit a profibrotic transcriptional profile. The transcriptomic studies assist the development of three effective perturbations for treating the rare disease CGD via targeting Il1r1, Morrbid, and Mif respectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7ab17761f537ad25af8c4de8bd9c011f1caec1f" target='_blank'>
              Single-cell and spatial transcriptomics reveal the pathogenesis of chronic granulomatous disease in a natural model
              </a>
            </td>
          <td>
            Hanzhi Yu, Guorong Zhang, Yunxi Ma, Jiayu Ding, Jingjing Liu, Zhiliang Zhou, Shaozhuo Jiao, Ge Dong, Zhigang Cai
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The Imaging and Molecular Annotation of Xenografts and Tumors Cancer Grand Challenges team was set up with the objective of developing the "next generation" of pathology and cancer research by using a combination of single-cell and spatial omics tools to produce 3D molecularly annotated maps of tumors. Its activities overlapped, and in some cases catalyzed, a spatial revolution in biology that saw new technologies being deployed to investigate the roles of tumor heterogeneity and of the tumor micro-environment. See related article by Stratton et al., p. 22 See related article by Bhattacharjee et al., p. 28 See related article by Goodwin et al., p. 34.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9c9c61637383cf6c8f3f96d10f0ca3cac35ac4d" target='_blank'>
              Cancer Research in the Age of Spatial Omics: Lessons from IMAXT.
              </a>
            </td>
          <td>
            D. Bressan, Nicholas Walton, Gregory J. Hannon
          </td>
          <td>2025-01-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Postpartum mammary gland remodeling after a pregnancy/lactation cycle is characterized by mechanisms of immunosuppression. Here we show that SEMA7A promotes PD-L1 expression in immune cells of the mammary tissue during involution. These same phenotypes are mimicked in the microenvironment of SEMA7A-expressing tumors, which partially respond to αPD-1/αPD-L1 treatments in vivo. However, cells that remain after treatment are enriched for SEMA7A expression. Therefore, we tested a novel monoclonal antibody that directly targets SEMA7A-expressing tumors, in part, by reducing SEMA7A-mediated upregulation of PD-L1. In vivo, the SEMA7A monoclonal antibody also reduces tumor growth or promotes complete regression of mouse mammary tumors, reduces the immunosuppressive phenotypes in the tumor microenvironment and restores cytotoxic T cells suggesting that SEMA7A may be a candidate for a novel immune-based therapy for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd071ef275ca2e3552071b322ce26bff99f047b0" target='_blank'>
              Semaphorin7A and PD-L1 cooperatively drive immunosuppression during mammary involution and breast cancer
              </a>
            </td>
          <td>
            Alan M. Elder, Heather R Fairchild, Kelsey T Kines, Lauren M Cozzens, Alexandria R Becks, T. Lyons
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be893eec80147faf7d7cca1af35e6472e40e7cc" target='_blank'>
              Statistical identification of cell type-specific spatially variable genes in spatial transcriptomics
              </a>
            </td>
          <td>
            Lulu Shang, Peijun Wu, Xiang Zhou
          </td>
          <td>2025-01-26</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4de4d7796ed3b7e2389f61c7edf7191dc271c5b5" target='_blank'>
              State-of-the-Art Liver Cancer Organoids: Modeling Cancer Stem Cell Heterogeneity for Personalized Treatment.
              </a>
            </td>
          <td>
            Julien Giron-Michel, Maël Padelli, Estelle Oberlin, Hind Guenou, Jean-Charles Duclos-Vallée
          </td>
          <td>2025-01-18</td>
          <td>BioDrugs : clinical immunotherapeutics, biopharmaceuticals and gene therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc23d3d27f11aa0ad92c216de43831771b9dae4" target='_blank'>
              Differential cell architecture and microenvironmental responses of pretumoral and tumoral cellular models exposed to coverslip-induced hypoxia.
              </a>
            </td>
          <td>
            Magdalena Millán, Felipe Parietti, Florencia Lamela, M. C. De Rossi, Belén Benítez, Valeria Levi, Manoela Domingues, R. Bologna-Molina, M. Arocena, J. Hochmann
          </td>
          <td>2025-01-03</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly malignant tumor in women, characterized by high morbidity, mortality, and recurrence rates. Although surgical treatment, radiotherapy, and chemotherapy are the mainstays of current treatment methods, the high heterogeneity of TNBC results in unsatisfactory outcomes with low 5-year survival rates. Rapid advancements in omics technology have propelled the understanding of TNBC molecular biology. The emergence of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has significantly enhanced knowledge of tumor heterogeneity and the distribution, functionality, and intercellular interactions of various cell types within the tumor microenvironment, including tumor cells, T cells, B cells, macrophages, and fibroblasts. The present study provides an overview of the technical characteristics of scRNA-seq and ST, highlighting their applications in exploring TNBC heterogeneity, cell spatial distribution patterns, and intercellular interactions. This review aims to enhance the comprehension of TNBC at the cellular level for the development of effective therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db2a16a0771f525b51537059ff8768102022813" target='_blank'>
              Analysis of single-cell and spatial transcriptomics in TNBC cell-cell interactions
              </a>
            </td>
          <td>
            Yan Xin, Qiji Ma, Qiang Deng, Tielin Wang, Dongxu Wang, Gang Wang
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Attempts to activate an anti-tumor immune response in glioblastoma (GBM) have been met with many challenges due to its inherently immunosuppressive tumor microenvironment. The degree and mechanisms by which molecularly and phenotypically diverse tumor-propagating glioma stem cells (GSCs) contribute to this state are poorly defined. In this study, our multifaceted approach combining bioinformatics analyses of clinical and experimental datasets, single-cell sequencing, and molecular and pharmacologic manipulation of patient-derived cells identified GSCs expressing immunosuppressive effectors mimicking regulatory T cells (Tregs). We show that this Immunosuppressive Treg-Like (ITL) GSC state is specific to the mesenchymal GSC subset and is associated with and driven specifically by TGF-β type II receptor (TGFBR2) in contrast to TGFBR1. Transgenic TGFBR2 expression in patient-derived GBM neurospheres promoted a mesenchymal transition and induced a 6-gene ITL signature consisting of CD274 (PD-L1), NT5E (CD73), ENTPD1 (CD39), LGALS1 (galectin-1), PDCD1LG2 (PD-L2), and TGFB1. This TGFBR2-driven ITL signature was identified in clinical GBM specimens, patient-derived GSCs and systemic mesenchymal malignancies. TGFBR2High GSCs inhibited CD4+ and CD8+ T cell viability and their capacity to kill GBM cells, effects reversed by pharmacologic and shRNA-based TGFBR2 inhibition. Collectively, our data identify an immunosuppressive GSC state that is TGFBR2-dependent and susceptible to TGFBR2-targeted therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb7f721f28d28359511d7b452c6e33026ca71e4b" target='_blank'>
              TGFBR2High mesenchymal glioma stem cells phenocopy regulatory T cells to suppress CD4+ and CD8+ T cell function
              </a>
            </td>
          <td>
            Amanda L Johnson, Harmon Khela, Jack Korleski, Sophie Sall, Yunqing Li, Weiqiang Zhou, Karen Smith-Connor, H. Lopez-Bertoni, J. Laterra
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91820cda6d428b7441857912e71a4c3a105594b5" target='_blank'>
              Density and entropy of immune cells within the tumor microenvironment of primary tumors and matched brain metastases
              </a>
            </td>
          <td>
            M. Kleinberger, D. Cifci, C. Paiato, E. Tomasich, M. Mair, A. Steindl, Zoltán Spiró, Z. I. Carrero, L. Berchtold, J. Hainfellner, L. Müllauer, G. Heller, M. Preusser, Jakob N Kather, A. Berghoff
          </td>
          <td>2025-02-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2ac9b04d746f5cf36d236d7ac9a449435141cc" target='_blank'>
              Spatial transcriptomics reveals prognostically LYZ+ fibroblasts and colocalization with FN1+ macrophages in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            L. Dai, N. Lou, Liling Huang, Lin Li, Le Tang, Yuankai Shi, Xiaohong Han
          </td>
          <td>2025-02-25</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0116e059872731a0003137617d95a907ab32d870" target='_blank'>
              Molecular landscape of tumor-associated tissue-resident memory T cells in tumor microenvironment of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Mi-So Park, Hyeonbin Jo, Hyeree Kim, Ji Young Kim, Woong-Yang Park, Yong-Han Paik, Yeup Yoon, Wonseok Kang, Hong-Hee Won
          </td>
          <td>2025-02-11</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Several studies have attempted to find the initiating drivers of small intestinal neuroendocrine tumors (SI-NET) development and the molecular mechanisms driving progression and metastatic spread. For gene expression studies using bulk microarrays and RNA sequencing, researchers commonly use normal intestinal mucosa as a control. The intestine is made up of several different cell types, and using bulk RNA-seq may generate findings that reflect factors other than the tumor transformation. This could potentially contribute to the lack of discoverable treatments and prevention strategies for SI-NETs. We performed scRNA-seq on tissue from two patients that had tumor resection surgery and separated the EC cells from the normal intestinal mucosa and used specific markers to compare them against SI-NET tumors cells. We pinpointed new genes associated with chr18 haploinsufficiency but also others with loss or gain of expression that have not previously been associated with SI-NETs, and which could be potential targets for further functional genomic studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35da01c8609a7028b2e2ec26d0b8011e3cf58489" target='_blank'>
              A single-cell transcriptomic comparison between small intestinal neuroendocrine tumors and their progenitor
              </a>
            </td>
          <td>
            Fredrik Axling, Elham Barazeghi, Per Hellman, O. Norlén, S. Backman, P. Stålberg
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Single-cell analysis has revolutionized our understanding of cellular heterogeneity, yet current approaches face challenges in efficiency and interpretability. In this study, we present scKAN, a framework that leverages Kolmogorov-Arnold Networks for interpretable single-cell analysis through three key innovations: efficient knowledge transfer from large language models through a lightweight distillation strategy; systematic identification of cell-type-specific functional gene sets through KAN’s learned activation curves; and precise marker gene discovery enabled by KAN’s importance scores with potential for drug repurposing applications. The model achieves superior performance on cell-type annotation with a 6.63% improvement in macro F1 score compared to state-of-the-art methods. Furthermore, scKAN’s learned activation curves and importance scores provide interpretable insights into cell-type-specific gene patterns, facilitating both gene set identification and marker gene discovery. We demonstrate the practical utility of scKAN through a case study on pancreatic ductal adenocarcinoma, where it successfully identified novel therapeutic targets and potential drug candidates, including Doconexent as a promising repurposing candidate. Molecular dynamics simulations further validated the stability of the predicted drug-target complexes. Our approach offers a comprehensive framework for bridging single-cell analysis with drug discovery, accelerating the translation of single-cell insights into therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd09fda0d9476d37b17d6a7579efebe98202f811" target='_blank'>
              scKAN: Interpretable Single-cell Analysis for Cell-type-specific Gene Discovery and Drug Repurposing via Kolmogorov-Arnold Networks
              </a>
            </td>
          <td>
            Haohuai He, Zhenchao Tang, Guanxing Chen, Fan Xu, Yao Hu, Yinglan Feng, Jibin Wu, Yu-an Huang, Zhi-an Huang, Kay Chen Tan
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) is evolving rapidly as a pivotal technology in studying the biology of tumors and their associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. In this study, we used serial 5-μm sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma tumor samples in tissue microarrays to compare the performance of the single cell ST platforms CosMx, MERFISH, and Xenium (uni/multi-modal) platforms in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx Digital Spatial Profiler, and hematoxylin and eosin staining data for the same samples. In addition to objective assessment of automatic cell segmentation and phenotyping, we performed pixel-resolution manual evaluation of phenotyping to carry out pathologically meaningful comparison between ST platforms. Our study detailed the intricate differences between the ST platforms, revealed the importance of parameters such as tissue age and probe design in determining the data quality, and suggested reliable workflows for accurate spatial profiling and molecular discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1a6df87fa44ba62736b9ada1db0c1fd8750c04f" target='_blank'>
              Comparison of imaging-based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed, paraffin-embedded tumor samples
              </a>
            </td>
          <td>
            Luisa Solis Soto, N. O. Lermi, Max Molina Ayala, Sharia Hernandez Ruiz, Wei Lu, Khaja B. Khan, Alejandra Serrano, Idania Lubo, Leticia Hamana, K. Tomczak, Sean Barnes, Jinzhuang Dou, Qingnan Liang, Rti Team, M. Gabriela Raso, Ximing Tang, Mei Jiang, B. Sanchéz-Espiridion, A. Weissferdt, J. Heymach, Jianjun Zhang, B. Sepesi, T. Cascone, Anne Tsao, Mehmet Altan, Reza J Mehran, Don L Gibbons, Ignacio I. Wistuba, Cara Haymaker, Ken Chen
          </td>
          <td>2025-01-17</td>
          <td>Research Square</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76368b27f3c58a581bef9e47250d9059ca708920" target='_blank'>
              Multiplexed spatial mapping of chromatin features, transcriptome and proteins in tissues.
              </a>
            </td>
          <td>
            Pengfei Guo, Liran Mao, Yufan Chen, Chin Nien Lee, Angelysia Cardilla, Mingyao Li, Marek Bartošovič, Yanxiang Deng
          </td>
          <td>2025-01-27</td>
          <td>Nature methods</td>
          <td>2</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c89aac6a693cf7fc84e3a1fc45266835eb27a4d" target='_blank'>
              Fusion of tumor cells and mesenchymal stem/stroma cells: a source of tumor heterogeneity, evolution and recurrence.
              </a>
            </td>
          <td>
            Zhen Liu, Yihao Wang, Zesheng Peng, Hui Li, Haofei Wang, Yuyi Wu, Xiaobing Jiang, Peng Fu
          </td>
          <td>2025-01-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0c4d7e98a1c7e44d04ba5fa342a0635434e2cbe" target='_blank'>
              Tumor secretome shapes the immune landscape during cancer progression
              </a>
            </td>
          <td>
            Jianqiang Yang, Sijia Tang, Nabil F. Saba, C. Shay, Yong Teng
          </td>
          <td>2025-02-10</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background: Brain metastases are a leading cause of mortality in non-small cell lung cancer (NSCLC), yet their molecular mechanisms remain unclear. SEC61G, a subunit of the SEC61 translocon, has been implicated in tumor progression but its role in brain metastases is unknown. This study explores how SEC61G contributes to brain metastases by driving metabolic reprogramming and immune microenvironment remodeling. Methods: Brain-metastatic NSCLC cell lines were established through in vivo selection in a mouse model. SEC61G expression was analyzed via transcriptomics, immunohistochemistry, multiplex immunofluorescence, and patient datasets. Functional assays were used to assess SEC61G's role in glycolysis, TLS formation, and immune interactions, with a focus on the SEC61G-PGAM1 axis. Pharmacological inhibitors and co-culture systems were employed to validate findings. Results: SEC61G was identified as a key upregulated gene in brain metastases based on transcriptomic data from patient-derived samples and mouse models. Higher SEC61G expression in brain metastases correlated with advanced tumor stages and poor survival in NSCLC patients. Mechanistically, SEC61G promoted brain metastasis by stabilizing the key glycolytic enzyme PGAM1. This occurred through a novel mechanism of competitive inhibition of PGAM1 ubiquitination: SEC61G directly antagonized the E3 ubiquitin ligase UBE3C, preventing PGAM1 degradation via the proteasome pathway. Stabilized PGAM1 enhanced glycolysis and regulated oxidative phosphorylation, driving metabolic reprogramming that supported brain metastatic colonization. Moreover, SEC61G reshaped the tumor immune microenvironment by promoting microglial M2 polarization and suppressing M1 polarization, accompanied by increased secretion of IL-6 and IL-10. These immune effects were dependent on PGAM1, as its pharmacological inhibition reversed SEC61G-induced M2 polarization and restored CD8+ T cell infiltration. In vivo and clinical studies confirmed that high SEC61G expression in brain metastases correlated with excessive M2 microglia, reduced immune surveillance, and poor patient outcomes. Immunoprofiling revealed a striking gradient of SEC61G expression across tertiary lymphoid structures (TLS) maturation stages: SEC61G levels were highest in TLS-absent samples and CD206+ microglia infiltration, intermediate in samples with immature TLS, and lowest in those with mature TLS. Conclusion: In conclusion, this study identifies a novel mechanism in which SEC61G drives NSCLC brain metastases by competitively inhibiting UBE3C-mediated ubiquitination of PGAM1, stabilizing PGAM1 and enhancing glycolysis. In addition to metabolic reprogramming, SEC61G impairs TLS maturation, suppresses adaptive immune responses, and facilitates immune evasion, contributing to brain metastatic colonization. These findings highlight SEC61G as a key regulator of brain metastasis and a promising therapeutic target for NSCLC patients with brain metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ac5a1d0bb716b82421e404ccc941d9d657a5485" target='_blank'>
              SEC61G Facilitates Brain Metastases via Antagonizing PGAM1 Ubiquitination and Immune Microenvironment Remodeling in Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Changshuai Zhou, Yue-fei Yang, Huanhuan Cui, Sen Li, Zhisu Wang, Lei Chen, Mingtao Feng, Deheng Li, Xin Chen, Bin Hao, Xiaojun Wu, Yang Gao, Liangdong Li, Jiayan Chen, Yiqun Cao
          </td>
          <td>2025-01-27</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, K. Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Recent advances in single-cell RNA sequencing (scRNA-seq) technologies have enabled us to clarify gene regulatory networks and immune cell compositions. In this study, integration of large-scale bulk and single-cell datasets refined hepatocellular carcinoma (HCC) classification and shed light on the characteristics of its aggressive subtype.


APPROACH AND RESULTS
Single-cell analysis of 228,564 live cells from six scRNA-seq datasets identified five major clusters of HCC cells with high mitotic activity (Cluster 1), activated Wnt/β-catenin signaling (Cluster 2), elevated glycolysis (Cluster 3) and lipogenesis (Clusters 4 and 5). Aggressive HCC subtype defined in bulk RNA-seq analysis of 691 tumor samples comprised a combination of Cluster 1 with Clusters 3, 4 or 5. Gene regulatory network analysis and gene set enrichment analysis highlighted the essential roles of p53 and MYC in aggressive HCC/Cluster 1, and cell composition analysis elucidated T cell depletion as an immune resistance mechanism. In a syngeneic mouse model, Trp53 knockout and MYC overexpression caused highly mitotic, tumorigenic and metastatic phenotypes, characterized by a macrotrabecular (MT) pattern, vascular encapsulation (VE) and T cell exclusion. Angiogenesis inhibition disrupted MT/VE formation, resulting in T cell recruitment, and its combination with immune checkpoint blockade achieved remission.


CONCLUSIONS
Single-cell analysis has deepened our understanding of the molecular mechanism and tumor microenvironment in aggressive HCC. The combination of targeting tumor vasculature and blocking immune checkpoints represents a promising therapeutic strategy for this subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e969ce26ddc70310682b3f3db6a8aee97b5128" target='_blank'>
              Integrative transcriptome profiling elucidates molecular and immunovascular characteristics of macrotrabecular hepatocellular carcinoma.
              </a>
            </td>
          <td>
            T. Taniai, S. Shimada, Y. Akiyama, Megumi Hatano, Koya Yasukawa, Yosuke Igarashi, Shu Tsukihara, Yoshiaki Tanji, K. Kodera, Kohei Okazaki, K. Haruki, Atsushi Nara, Kohei Yagi, K. Akahoshi, D. Ban, Yasuhiro Asahina, Toru Ikegami, Minoru Tanabe, Shinji Tanaka
          </td>
          <td>2025-02-26</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f22ff573111b3740a2cb886527997d19806de8b" target='_blank'>
              Single-cell RNA sequencing reveals tumor heterogeneity in small cell neuroendocrine cervical carcinoma
              </a>
            </td>
          <td>
            Xuesong Xiang, X. Tao, Keqin Hua, Hua Jiang, Jingxin Ding
          </td>
          <td>2025-02-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are pivotal biomarkers in tumor metastasis, however, the underlying molecular mechanism of CTCs behavioral heterogeneity during metastasis remains unexplored. Here, an integrative workflow is developed to link behavior characteristics to metabolic profiling within individual CTCs, which simulates the metastatic process on a microfluidic system and combined with single-cell mass spectrometry (MS) detection. Spheroid-derived HCT116 cells are tracked and extracted via a temporary vascular system, revealing various arrest patterns under biomimetic vascular shear flow. Downstream MS analysis characterizes 17 cellular metabolites and associates metabolic profiles with de-adhesion behaviors of the same CTCs, identifying a potential high-metastatic subpopulation with enhanced arrest ability and evaluating critical metabolites involved in metastasis pathways. Additionally, the metastasis-inhibiting effect of anti-tumor drug 5-fluorouracil by reducing high-metastatic cells in spheroids is elucidated. This approach offers a valuable opportunity to dissect the interplay of the metastatic behavior and metabolic profiles of CTCs and foster insights into the molecular mechanisms underlying behavioral phenotypes in the tumor metastasis process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32d3e81bfd80ccc573739b0a4ae2229795ed81e7" target='_blank'>
              Linking Metastatic Behavior and Metabolic Heterogeneity of Circulating Tumor Cells at Single-Cell Level Using an Integrative Microfluidic System.
              </a>
            </td>
          <td>
            Ying Hou, Jiaxu Lin, Hongren Yao, Zengnan Wu, Yongning Lin, Jinming Lin
          </td>
          <td>2025-02-17</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The advent of spatial transcriptomics has dramatically expanded our ability to study the vast network of cell-cell interactions at the molecular level in tissue. Among current methods, sequencing-based approaches have great potential in discovering because of its unbiased capture. In the last couple of years, the spatial resolution for the capture addresses has been significantly improved from 100 μm to <1 μm, well below the size of a mammalian cell. However, the capture efficiency has always been a pain point, ∼one order of magnitude lower than that of single cell RNA sequencing (scRNA-seq). The low capture efficiency limits the depth and breadth of its applications in the study of complex biological systems and diseases. Here, we introduce Salus Spatial transcriptomic system (Salus-STS), which provides ∼1 μm capture resolution and a capture efficiency ∼1 order of magnitude higher than other current methods. Analysis with sub-cellular resolution becomes practical for sequencing-based spatial transcriptomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b6143f746de652c8a331460d4e5358837484378" target='_blank'>
              Sequencing-based Spatial Transcriptomics with scRNA-seq Sensitivity
              </a>
            </td>
          <td>
            Renjie Liao, Defeng Fu, Zaoxu Xu, Han Liang, Xiaoran Zhou, Yiling Chen, Xueqi Liu, Jiajun Cheng, Ruidong Guo, Chen Li, Huihua Xia, Gailing Li, Diewen Feng, Wei Chen, Yang Chen, Longchao Chen, Zihang Huang, Yang Zhou, Qingbin Chen, Dexia Chen, Shuguang Xuan, Changling Zhang, Yongyi Lu, Hui Wang, Taiqing Feng, Yuanye Bao, Luyang Zhao, Erkai Liu, Gufeng Wang
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer heterogeneity has been well understood based solely on tumor components. However, the tumor microenvironment, which plays a crucial role in cancer by regulating tumor cells’ differentiation, maturation, and malignant potential, remains under-explored. This study conducted high-throughput RNA-Seq analysis on fresh breast cancer tissues from each molecular subtype and a pure population of cancer-associated fibroblasts (CAFs) isolated from cancer tissue and adjacent normal parts. Based on their functions, we identified three populations of CAFs: a) immune response-related, b) ECM remodeling, and c) calcium/protein binding. Validation of differentially expressed genes among these CAF populations revealed subtype-specific correlations. For instance, CAFs expressing immune response-related genes were significantly enriched in Luminal A/B and TNBC (log +1-fold; p =0.00047), compared to Her-2 positive (log -2.3-fold; p =0.0026). ECM remodeling genes exhibited greater expression in Her-2 positive and TNBC (log +4 to +11-fold; p =7.87E-08) when compared to Luminal subtypes (log +1.5-fold to +6-fold; p =8.66E-05). Calcium binding genes displayed overall similar expression across breast cancer subtypes. Thus, this study identified and underscored the functional heterogeneity of CAFs within the tumor microenvironment across the molecular subtypes of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e870b934402b9a2b10afe0f5e6b9ec8fa60b89" target='_blank'>
              Deciphering Functional Heterogeneity of Cancer-Associated Fibroblasts Across Molecular Subtypes of Breast Cancer
              </a>
            </td>
          <td>
            D. Kashyap
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Increased stemness of cancer cells exacerbates radioresistance, thereby greatly limiting the efficacy of radiotherapy. In order to study the changes in cancer cell stemness during radiotherapy, we established a radioresistance model of human non-small cell lung cancer A549 cells and obtained A549 radioresistant cells (A549-RR). We sampled the cells at different time points during the modeling process and investigated the heterogeneity of each group of cells using single-cell sequencing. Cells in the early stages of fractionated irradiation were found to be significantly up-regulated in stemness, and a subpopulation of cells producing this response was screened and referred to as “radiation-induced stemness-responsive cancer cells”. They were undergoing stemness response, energy metabolism reprogramming, and progressively differentiating into cells with more diverse and malignant phenotypes in order to attenuate the killing effect of radiation. Furthermore, we demonstrated that such responses might be driven by the activation of the EGFR-Hippo signaling pathway axis, which also plays a crucial role in the development of radioresistance. Our study reveals the dynamic evolution of cell subpopulation in cancer cells during fractionated radiotherapy; the early stage of irradiation can determine the destiny of the radiation-induced stemness-responsive cancer cells. The activation of stemness-like phenotypes during the development of radioresistance is not the result of dose accumulation but occurs during the early stage of radiotherapy with relatively low-dose irradiation. The degree of the radiation-induced stemness response of cancer cells mediated by the EGFR-Hippo signaling pathway might be a potential predictor of the efficacy of radiotherapy and the development of radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6498ad18eccc13924e712513e028e4bc431ff4" target='_blank'>
              Single-Cell Sequencing Reveals the Role of Radiation-Induced Stemness-Responsive Cancer Cells in the Development of Radioresistance
              </a>
            </td>
          <td>
            Zheng Shi, Cuilan Hu, Jiadi Liu, Wei Cheng, Xiaohua Chen, Xiongxiong Liu, Yanyu Bao, Haidong Tian, Boyi Yu, Feifei Gao, Fei Ye, Xiaodong Jin, Chao Sun, Qiang Li
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND AIMS
The prognoses and therapeutic responses of patients with intrahepatic cholangiocarcinoma (iCCA) depend on spatial interactions among tumor microenvironment (TME) components. However, the spatial TME characteristics of iCCA remain poorly understood. The aim of this study was to generate a comprehensive spatial atlas of iCCA using artificial intelligence-assisted spatial multiomics patterns and to identify spatial features associated with prognosis and immunotherapy.


APPROACH RESULTS
Spatial multiomics, including imaging mass cytometry (IMC, n=155 in-house), spatial proteomics (n=155 in-house), spatial transcriptomics (n=4 in-house), multiplex immunofluorescence (mIF, n=20 in-house), single-cell RNA sequencing (scRNA-seq, n=9 in-house and n=34 public), bulk RNA-seq (n=244 public), and bulk proteomics (n=110 in-house and n=214 public), were employed to elucidate the spatial TME of iCCA. More than 1.06 million cells were resolved, and the findings revealed that spatial topology, including cellular deposition patterns, cellular communities, and intercellular communications, profoundly correlates with the prognosis of iCCA patients. Specifically, CD163hi M2-like resident-tissue macrophages suppress anti-tumor immunity by directly interacting with CD8+ T cells, resulting in poorer patient survival. Additionally, five spatial subtypes with distinct prognoses were identified, and potential therapeutic options were generated for these subtypes. Furthermore, a spatial TME deep learning system was developed to predict the prognosis of iCCA patients with high accuracy from a single 1-mm2 tumor sample.


CONCLUSIONS
This study offers preliminary insights into the spatial TME ecosystem of iCCA, providing valuable foundations for precise patient classification and the development of personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f973214ce129b073da8e0d3c1c3cc399902a2e8" target='_blank'>
              Spatial single-cell proteomics landscape decodes the tumor microenvironmental ecosystem of intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Libing Hong, Jie Mei, Xuqi Sun, Yifan Wu, Zhen Dong, Yuzhi Jin, Liaoliao Gao, Jinlin Cheng, Weihong Tian, Chuan Liu, Bin Li, Pingping Hu, Lulu Liu, Shan Xin, Xiaomeng Dai, Peng Zhao, Rongping Guo, Minshan Chen, Jingping Yun, Bo Lin, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-02-25</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Transgenic mice and organoid models, such as three-dimensional tumoroid cultures, have emerged as powerful tools for investigating cancer development and targeted therapies. Yet, the extent to which these preclinical models recapitulate the cellular identity of heterogeneous malignancies, like neuroblastoma (NB), remains to be validated. Here, we characterized the transcriptional landscape of TH-MYCN tumors by single-cell RNA sequencing (scRNA-seq) and developed ex vivo tumoroids. Integrated analysis with murine fetal adrenal samples confirmed that both TH-MYCN tumors and tumoroids closely mirror the cellular profiles of normal embryonic sympathoblasts and chromaffin cells. Comprehensive comparison between tumors from NB patients and TH-MYCN mice demonstrated similarities in adrenergic tumor cell composition. Ex vivo tumoroid cultures displayed histological resemblance and shared transcriptional profiles with the originating TH-MYCN tumors and human NB. Importantly, subpopulations within tumoroids exhibited gene expression associated with poor NB patient survival. Notably, recurrent observations of a low-proliferative chromaffin phenotype connected to the highly proliferative sympathetic phenotype suggested that pushing sympathoblasts into a chromaffin-like state may offer an interesting therapeutic strategy for NB. Together, this study not only deepens our understanding of a widely used transgenic mouse NB model but also introduces an ex vivo model that maintains critical adrenergic cell state identity, thereby enhancing its translational potential for NB research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5619b05ffd717eb9d8928b0e83ee66ca9cda8b" target='_blank'>
              Comparative Single-Cell Transcriptomics of Human Neuroblastoma and Preclinical Models Reveals Conservation of an Adrenergic Cell State.
              </a>
            </td>
          <td>
            Bethel Tesfai Embaie, Hirak Sarkar, A. Alchahin, Jörg Otte, T. Olsen, Conny Tümmler, Polina Kameneva, Artem V Artemov, N. Akkuratova, I. Adameyko, J. Stukenborg, Malin Wickström, P. Kogner, J. Johnsen, Shenglin Mei, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7453b499aa0f4f56b20054ed92fb6b1a0f0d81c" target='_blank'>
              Analysis of thyroid carcinoma composition and spatial architecture in the progression of dedifferentiation, lymphatic metastasis, and gastric metastasis
              </a>
            </td>
          <td>
            Di Wang, Ruichun Lu, Fenglian Yan, Yansong Lin, Hao Wang, Huabao Xiong
          </td>
          <td>2025-02-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Despite significant advancements in identifying novel therapeutic targets and compounds, cancer stem cells (CSCs) remain pivotal in driving therapeutic resistance and tumor progression in gastric cancer (GC). High-resolution knowledge of the transcriptional programs underlying the role of CSC niche in driving tumor stemness and progression is still lacking. Herein, spatial and single-cell RNA sequencing of 32 human gastric mucosa tissues at various stages of malignancy, illuminating the phenotypic plasticity of tumor epithelium and transcriptional trajectory from mature gastric chief cells to the CSC state, which is associated with activation of EGFR and WNT signaling pathways, is conducted. Moreover, the CSCs interact with not only the immunosuppressive CXCL13+ T cells and CCL18+ M2 macrophages to evade immune surveillance, but also the inflammatory cancer-associated fibroblasts (iCAFs) to promote tumorigenesis and maintain stemness, which construct the CSC niche leading to inferior prognosis. Notably, it is uncovered that amphiregulin (AREG) derived from iCAFs promotes tumor stemness by upregulating the expression of SOX9 in tumor cells, and contributes to drug resistance via the AREG-ERBB2 axis. This study provides valuable insight into the characteristics of CSC niche in driving tumor stemness and progression, offering novel perspective for designing effective strategies to overcome GC therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0d6cf84d15c0f17d3af8a79006d512aec0aa37" target='_blank'>
              Dissecting the Spatial and Single-Cell Transcriptomic Architecture of Cancer Stem Cell Niche Driving Tumor Progression in Gastric Cancer.
              </a>
            </td>
          <td>
            Guangyu Zhang, Xin Zhang, Wenting Pan, Xizhao Chen, Lingfei Wan, Chunjie Liu, Yuting Yong, Yue Zhao, Shuli Sang, Lihua Zhang, Sheng Yao, Yushu Guo, Mingmei Wang, Xinhui Wang, Guangdun Peng, Xinglong Yan, Yanchun Wang, Min Zhang
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Despite numerous studies on medulloblastoma (MB) cell heterogeneity, the spatial characteristics of cellular states remain unclear.


METHODS
We analyze single-nucleus and spatial transcriptomes and chromatin accessibility from human MB spanning four subgroups, to identify malignant cell populations and describe the spatial evolutionary trajectories. The spatial CNVs patterns and niches were analyzed to investigate the cellular interactions.


RESULTS
Three main malignant cell populations were identified, including progenitor-like, cycling and differentiated populations. Gene signatures of cell populations strongly correlate to clinical outcomes. These tumor cell populations are geographically organized as stem-like and mature regions, highlighting their spatially heterogeneous nature. Progenitor-like and cycling cells are mainly concentrated in stem-like regions, whereas various differentiated populations are primarily distributed in mature regions. By analyzing chromosomal alterations, we find that stem-like region typically harbors a single pattern of CNVs, reflecting high originality and uniformity, which is in stark contrast to mature region exhibiting multiple patterns with a broader range of biological functions. Projecting cellular state program onto spatial sections fully illustrates the evolution from stem-like region to various functional zones in mature region, which is correlated to microenvironmental components along the paths to maintain stemness or promote differentiation. Conclusions. This multi-omics database comprehensively facilitates the understanding of MB spatial evolutionary organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb3b0d5b454cf059f699fd40f05952bb5941c97" target='_blank'>
              Unveiling spatial heterogeneity in medulloblastoma: a multi-omics analysis of cellular state and geographical organization.
              </a>
            </td>
          <td>
            Jiankang Li, Hailong Liu, Ziwei Wang, Jiao Zhang, Xuan Chen, Craig Daniels, Xiaochong Wu, Olivier Saulnier, Hiromichi Suzuki, Pasqualino de Antonellis, Alexandra Rasnitsyn, Winnie W. Ong, Evan Y Wang, Liam D Hendrikse, Yu Su, Yu Tian, Dongming Han, Ruohan Wang, Jialin Mo, Fei Liu, Kaiwen Deng, Dongyang Wang, Zhaoyang Feng, Yifei Jiang, Yanong Li, Yuting Ma, Zijia Liu, Meiyu Li, Peiyi Tian, Yanfeng Shi, Yong Jiang, Tao Yang, Shouwei Li, Jianfeng Liang, Jingchuan Wu, Ying Wang, Wanjing Zou, Yina Jiang, Lusheng Wang, Fang Chen, Xin Jin, Shuaicheng Li, Xiaoguang Qiu, Chunde Li, Ya Gao, Yujie Tang, Michael D. Taylor, Tao Jiang
          </td>
          <td>2025-02-17</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Introduction: The molecular pathways involved in the response to radiation therapy (RT) in pancreatic ductal adenocarcinoma (PDAC) remain poorly understood. Attempts to combine RT with immunotherapies in PDAC have yielded inconsistent outcomes, highlighting a critical gap in our understanding of immune dynamics and mechanisms influencing RT response. Our study aims to uncover molecular pathways and cellular subtypes associated with radioresponse. To do so, we developed a novel cell-cell interaction platform, "SpaCCI", to map ligand-receptor (LR) interactions, essential for understanding cellular communication within the TME and its contribution to therapeutic resistance. Methods: Single-cell RNA sequencing (scRNA-seq) was performed on 16 matched pre- and post-treatment PDAC biopsies from a Phase I/II dose-escalation clinical trial of stereotactic body radiation therapy. Additionally, spatial transcriptomics (ST) was conducted on resected PDAC (20 treated with neoadjuvant chemoradiation, 8 untreated). We integrated scRNA-seq and ST datasets using cell type decomposition and developed SpaCCI to enhance LR interaction detection in tissues with complex cell mixtures. Results: ScRNA-seq identified five molecular subpopulations of cancer cells associated to RT response, with PDAC_3 enriched post-RT and marked by genes involved in lipid metabolism and ferroptosis defense. Post-RT depleted subpopulations, such as PDAC_0 and PDAC_prolif, saw upregulation of pathways related to rapid proliferation and oxidative phosphorylation. In a murine model, PDAC_3 was similarly enriched in post-RT tumors, with persister cell lines from these tumors showing overexpression of ferroptosis defense genes and resistance to ferroptosis inducers including Erastin and RSL3. ST analysis also revealed spatial immune changes post-RT, with increased cytotoxic and effector memory T cells balanced by regulatory T cells and M2-like macrophages, indicating immune escape. NK and memory B cells correlated with improved survival post-RT, while plasma IgG cells localized outside the tumor bed, suggesting compartmentalized immune responses. Using SpaCCI, we observed differential enrichment of ligand-receptor interactions in RT-treated tumors, particularly among lipid-associated tumor macrophages, cancer cells, myofibroblast-like cancer-associated fibroblasts, and endothelial cells. Persister PDAC_3 subpopulations showed enriched LR SEMA3 signaling with CAFs, myeloid cells, and endothelial cells consistent with axon guidance pathways. Conclusions: This study presents a molecular and cellular atlas of PDAC response to RT, revealing persister cells populations with adaptations in lipid metabolism and ferroptosis defense that could be targeted to enhance RT efficacy. ST highlighted complex immune dynamics within the TME demonstrating a balance between inflammatory and immunosuppressive populations. Our findings provide a foundation for combinatorial therapeutic strategies and suggest that axon guidance pathways may also play an important role in the response to RT in PDAC.
 Citation Format: Vincent Bernard, Li-Ting Ku, Tianyu Wang, Kimal Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, Manoop S. Bhutani, Paola A. Guerrero, Ethan B. Ludmir, Albert C. Koong, Cullen M. Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay.Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P018">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd18fd5a66ac51a4015b7b7d9504296b9a16919" target='_blank'>
              Abstract P018: Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, K. Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, M. Bhutani, Paola A. Guerrero, E. Ludmir, Albert C Koong, Cullen M Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The exploration of cellular heterogeneity within the tumor microenvironment (TME) via single-cell RNA sequencing (scRNA-seq) is essential for understanding cancer progression and response to therapy. Current scRNA-seq approaches, however, lack spatial context and rely on incomplete datasets of ligand-receptor interactions (LRIs), limiting accurate cell type annotation and cell-cell communication (CCC) inference. This study addresses these challenges using a novel graph neural network (GNN) model that enhances cell type prediction and cell interaction analysis. Our study utilized a dataset consisting of 49,020 cells from 19 patients across three cancer types: Leukemia, Breast Invasive Carcinoma, and Colorectal Cancer. The proposed scGSL model demonstrated robust performance, achieving an average accuracy of 84.83%, precision of 86.23%, recall of 81.51%, and an F1 score of 80.92% across all datasets. These metrics represent a significant enhancement over existing methods, which typically exhibit lower performance metrics. Additionally, by reviewing existing literature on gene interactions within the TME, the scGSL model proves to robustly identify biologically meaningful gene interactions in an unsupervised manner, validated by significant expression differences in key gene pairs across various cancers. The source code and data used in this paper can be found in https://github.com/LiYuechao1998/scGSL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd31c962d460de902bace72d06f995f9fc390f7" target='_blank'>
              Graph Structure Learning for Tumor Microenvironment with Cell Type Annotation from non-spatial scRNA-seq data
              </a>
            </td>
          <td>
            Yu-an Huang, Yue C. Li, Hai-Ru You, Jie Pan, Xiyue Cao, Xinyuan Li, Zhi-an Huang, Zhu-Hong You
          </td>
          <td>2025-02-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 218


 Background:
 Peritoneal metastases (PM) in colorectal cancer (CRC) portend a poor prognosis. We sought to elucidate spatially resolved molecular features facilitating transcoelomic dissemination and progression.
 Methods:
 50 PT, 116 PM and 37 primary normal samples from 96 patients were retrieved and profiled with digital spatial profiling (GeoMx DSP, Nanostring Technologies). Through unsupervised clustering, we characterized the microenvironment of tumor-stroma compartments and studied the roles of stromal phenotypes in promulgating tumorigenesis. These findings were orthogonally validated through spatial proteomics (multiplex IHC [mIHC] - COMET, Lunaphore Technologies) of 10 paired PT-PM samples retrieved from 6 patients.
 Results:
 A median of 37 region of interests (ROIs) (IQR: 32-38) were selected from a total of 9 tissue microarray (TMA) slides. A total of 269 ROIs (181 tumor, 88 stroma) from PM samples, 45 ROIs (34 tumor, 11 stroma) from PT samples and 28 (all stroma) ROIs from primary normal samples were retrieved and profiled. Through consensus clustering of stroma ROIs, we identify a fibro-collagenous and immune infiltrated stromal phenotype (stromal cluster [SC] 2) characterized by increased cancer associated fibroblasts (CAFs), memory B cells, M2 macrophages and T-cell exhaustion. Patients with SC2 stroma had poorer survival (p=0.036). SC2 stroma was also observed to support adjacent tumor compartments with enriched oncogenic pathways such as TGF-beta, TNF-alpha, hypoxia and JAK-STAT. Through discriminatory gene expression profiles, we developed a 20-gene composite bidirectional signature of SC2. The prognostic significance of the SC2-signature was externally validated through several cohorts including TCGA and MSI-H only cohorts. Next, by inspecting mIHC retrieved immune cell type densities across ROIs, we confirmed a similar phenomenon in which two distinct stromal clusters were found – one being fibroblast infiltrated but T-cell depleted (SC2-like, n=67 ROI) and another which is T-cell infiltrated but fibroblast depleted (SC1-like, n=104 ROI). Diverging spatial distributions of T-cells and fibroblasts between SC1 and SC2-like stroma was appreciated as well. Between both PT and PM, we find close clustering of T cells in SC1-like stromal compartments and fibroblasts in SC2-like stromal compartments.
 Conclusions:
 We describe SC2, a pro-tumorigenic stromal phenotype characterized by increased CAFs, T cell exhaustion and is associated with poor prognosis in CRC PM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76074158017484a4495c691d7f74783fe9c4e65" target='_blank'>
              Diverse stromal phenotypes at spatial resolution in colorectal cancer peritoneal metastasis.
              </a>
            </td>
          <td>
            Joseph J. Zhao, J. Ong, Ying Liu, Supriya Srivastava, Melissa Ching Ching Teo, Joey Jia-Ying Lee, Robert Walsh, Lit-Hsin Loo, Qiu Xuan Tan, Gillian Ng, J. W. Tan, Haoran Ma, Xuewen Ong, S. Tay, T. Sheng, Jeffrey H Y Lum, W. Yong, F. Pietrantonio, P. Tan, R. Sundar
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Background Glioblastoma (GBM) is the most common primary malignant brain tumor in adults, characterized by aggressive growth and a dismal prognosis. Interleukin-33 (IL-33) and its receptor ST2 have emerged as regulators of glioma growth, but their exact function in tumorigenesis has not been deciphered. Indeed, previous studies on IL-33 in cancer have yielded somewhat opposing results as to whether it is pro- or anti-tumorigenic. Methods IL-33 expression was assessed in a GBM tissue microarray and public databases. As in vivo models we used orthotopic xenografts of patient-derived GBM cells, and syngenic models with grafted mouse glioma cells. Results We analyzed the role of IL-33 and its receptor ST2 in nonmalignant cells of the glioma microenvironment and found that IL-33 levels are increased in cells surrounding the tumor. Protein complexes of IL-33 and ST2 are mainly found outside of the tumor core. The IL-33-producing cells consist primarily of oligodendrocytes. To determine the function of IL-33 in the tumor microenvironment, we used mice lacking the ST2 receptor. When glioma cells were grafted to ST2-deficient mouse brains, the resulting tumors exhibited a more invasive growth pattern, and are associated with poorer survival, compared to wild-type mice. Tumors in ST2-deficient hosts are more invasive, with increased expression of extracellular matrix remodeling enzymes and enhanced tumor angiogenesis. Furthermore, the absence of ST2 leads to a more immunosuppressive environment. Conclusions Our findings reveal that glia-derived IL-33 and its receptor ST2 participate in modulating tumor invasiveness, tumor vasculature, and immunosuppression in glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc52b5f579f7e0f0d9dd54d423d1abfbfb33ee41" target='_blank'>
              Lack of ST2 aggravates glioma invasiveness, vascular abnormality, and immune suppression
              </a>
            </td>
          <td>
            Grzegorz Wicher, Ananya Roy, Alessandra Vaccaro, K. Vemuri, Mohanraj Ramachandran, Tommie Olofsson, Rebeca-Noemi Imbria, M. Belting, Gunnar Nilsson, A. Dimberg, K. Forsberg-Nilsson
          </td>
          <td>2025-01-27</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Cancer is a complex disease driven by dynamic regulatory shifts that cannot be fully captured by individual molecular profiling. We employ a data-driven approach to construct a coarse-grained dynamic network model based on hallmark interactions, integrating stochastic differential equations with gene regulatory network data to explore key macroscopic dynamic changes in tumorigenesis. Our analysis reveals that network topology undergoes significant reconfiguration before hallmark expression shifts, serving as an early indicator of malignancy. A pan-cancer examination across $15$ cancer types uncovers universal patterns, where Tissue Invasion and Metastasis exhibits the most significant difference between normal and cancer states, while the differences in Reprogramming Energy Metabolism are the least pronounced, consistent with the characteristic features of tumor biology. These findings reinforce the systemic nature of cancer evolution, highlighting the potential of network-based systems biology methods for understanding critical transitions in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a279852f8f3e4dea6bf2c322ab3c789c81b3e6c" target='_blank'>
              How cancer emerges: Data-driven universal insights into tumorigenesis via hallmark networks
              </a>
            </td>
          <td>
            Jiahe Wang, Yan Wu, Yuke Hou, Yang Li, Dachuan Xu, Changjing Zhuge, Yue Han
          </td>
          <td>2025-02-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND AND AIM
Intrahepatic cholangiocarcinoma (ICC) is the second most common primary liver cancer whose incidence is increasing globally. However, the high tumor heterogeneity of ICC restricts the efficacy of available systematic therapies. We aim to dissect the tumor heterogeneity of ICC utilizing high-resolution single-cell RNA sequencing to identify novel therapeutic targets.


METHODS
We performed single-cell RNA sequencing (scRNA-seq) of 26 tumor samples from 23 ICC patients and spatial transcriptomic sequencing of six tumor sections from six ICC patients. Bulk RNA-seq data from two public datasets were used for validation. Additionally, immunohistochemical staining and multiplex immunofluorescence staining were conducted to validate the infiltration and distribution of cells in the tumor microenvironment.


RESULTS
We discovered that malignant cells in ICC samples exhibited a remarkably high degree of tumor heterogeneity. We identified a basal-like tumor cell subpopulation characterized by the expression of basal epithelial related genes including KRT5, KRT6A, and KRT17. The basal-like tumor subpopulation was characterized by activation of MET signaling and extracellular matrix organization associated with tumor invasion and correlated with poor prognosis. Cell-cell communication analysis further showed significant HGF-MET interaction between inflammatory cancer-associated fibroblasts (iCAFs) and basal-like tumor cells. We found that iCAFs were the major source of HGF in tumor environment and contributed to the basal-like phenotype formation of tumor cells by HGF-MET axis.


CONCLUSIONS
We identified an aggressive basal-like tumor cell subpopulation, which correlated with poor prognosis in ICC. The MET pathway contributes to the aggressiveness of basal-like tumor cells and serves as a novel therapeutic target for ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfff83a3b2e44394201ebb9147dbd5e27d9e3ca" target='_blank'>
              Single-Cell Transcriptomic Analysis Reveals an Aggressive Basal-Like Tumor Cell Subpopulation Associated With Poor Prognosis in Intrahepatic Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Chang-Pin Liao, Yuting Zhang, Jing Yang, Shuo Wang, Zhijuan Li, Shuling Chen, Yubin Xie, Lixia Xu, Sui Peng, Xuezhen Zeng, Ming Kuang, Bangde Xiang, Kaiyu Sun, Xiao Zhao
          </td>
          <td>2025-02-24</td>
          <td>Journal of gastroenterology and hepatology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Solid tumors often have an abundance of collagen-I that stiffens the tissue, and they are invariably driven by mutations that include chromosome losses and gains. These observations are linked here by showing that 3D matrix stiffness induces heritable changes to a cell’s DNA. We use live-cell chromosome reporters (ChReporters) and hydrogels of tunable stiffness to show mitotic compression, micronuclei counts, ChReporter losses and heterogeneity all increase as functions of stiffness. Increased mistakes occur despite suppressed cell division in stiff matrix and minimal size variation between spheroids. Colonies of ChReporter-negative cells within cancer spheroids align with Luria-Delbruck’s seminal theory for heritable mutations, which predicts inter-spheroid variances that exceed Poisson statistics. Suppression of the contractility motor Myosin-II also increases chromosome loss in 3D but not 2D and does not affect spheroid growth – thus clarifying Myosin-II’s putative role as a tumor suppressor. Consistent with experiments, pan-cancer analyses of clinical data associates chromosome losses and gains with collagen-I levels and genetic variation. Stiff extracellular matrix thus drives mechano-evolution of solid tumors as a Darwin-Lamarck process with heterogeneity that complicates therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21422305f6e2cd46c2231ab78870bc6be52cefd" target='_blank'>
              Matrix stiffness induces heritable changes in chromosome numbers, consistent with solid tumor heterogeneity
              </a>
            </td>
          <td>
            Alişya A. Anlaş, Markus T. Sprenger, Mai Wang, Nicholas M. Ontko, Steven H. Phan, D.E. Discher
          </td>
          <td>2025-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8d5845a6bd00754561294af9f1ae51a6774014" target='_blank'>
              Single-cell pseudotime and intercellular communication analysis reveals heterogeneity and immune microenvironment in oral cancer
              </a>
            </td>
          <td>
            Hanjun Liu, Lian Xie, Xuemin Xing, Lili Hou, Jinfeng Zhang, Ling Yang
          </td>
          <td>2025-02-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f2358565c36d3daf9c70fc71b645b6b73d557a4" target='_blank'>
              SELP+ TEC:CD8+ T cell crosstalk associates with improved radiotherapy efficacy in cervical cancer
              </a>
            </td>
          <td>
            Qingyu Huang, Wenhui Yang, Fuhao Wang, R. Huang, Qian Wang, Xiaohui Li, T. Lei, Shengqin Yue, Wen-Lu Zou, Qi An, Jinbo Yue, Qinyong Hu, Chao Liu
          </td>
          <td>2025-02-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd3cbd55a53252e9bb3dfd532de1bccedf85f35d" target='_blank'>
              Tumor-associated-fibrosis and active collagen-CD44 axis characterize a poor-prognosis subtype of gastric cancer and contribute to tumor immunosuppression
              </a>
            </td>
          <td>
            Yingqi Yang, Haohan Sun, Hongkai Yu, Luyao Wang, Chang Gao, Haokun Mei, Xiaomeng Jiang, Minghui Ji
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47537700619a16e63c5ebabefc327d2e5284da8b" target='_blank'>
              Transcriptome analysis of novel B16 melanoma metastatic variants generated by serial intracarotid artery injection
              </a>
            </td>
          <td>
            J. C. Kienzler, Erick Contreras, J. Treger, Linda M. Liau, Geoffrey Owens, Robert M. Prins
          </td>
          <td>2025-01-16</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background: Circulating tumor cells (CTCs) and CTC-clusters are pivotal in the metastatic process of breast cancer (BC). Owing to their low frequency, models replicating their biology should provide a robust platform for investigating the molecular mechanisms driving metastasis and identifying new biomarkers. We established and characterized a CTC-derived cell model from a mouse xenograft to explore its metastatic behavior and molecular profile, which allowed us to investigate the expression and prognostic significance of a set of genes associated with the metastatic potential of CTCs. Methods: The CTC line (mCTC) derived from a MDA-MB-231 mouse xenografts was used in comparative functional analyses including cell cycle evaluation, colony formation, invasion, adhesion, and metastatic competency in zebrafish models. Transcriptomic profiling and functional assays were conducted to identify candidate genes and understand their roles in metastasis. Moreover, publicly available gene expression datasets of CTCs, CTC-clusters, and tumor tissue, from GEO and TCGA, were analyzed for the identification of a gene signature that was correlated with survival data. The signature was validated in an independent cohort. Results: Compared with MDA-MB-231 cells, mCTC cells presented enhanced colony formation, invasion, and adhesion, and increased dissemination and survival in zebrafish. Transcriptomic analysis revealed that SPARC was significantly upregulated. Functional assays showed that SPARC overexpression was correlated with increased invasion and migration. Analysis of public datasets confirmed the high expression of SPARC in BC CTCs and CTC-clusters. Additionally, a 4-gene signature involving SPARC, THBS1, VCL, and HSP90AB1 was identified that demonstrated strong prognostic value, predicting shorter overall and distant metastasis-free survival in the primary tumor setting. Validation cohorts confirmed its ability to distinguish high-risk patients. Elevated expression of the 4-gene signature in CTCs was also indicative of increased mortality risk. Conclusion: mCTC exhibit distinct metastatic traits and molecular characteristics, highlighting a possible role of SPARC in CTC biology and its potential as a prognostic marker in BC metastasis. The identified 4-gene signature provides a robust prognostic tool for assessing patient risk and guiding therapeutic strategies. Further investigations into the mechanistic role of SPARC may reveal new therapeutic targets for managing BC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785682c178d882b31b862712e1108b5022d056bd" target='_blank'>
              A CTC Model Uncovers Metastatic Drivers and Prognostic Markers in Breast Cancer
              </a>
            </td>
          <td>
            C. Fernández-Santiago, I. Martínez-Peña, M. Paramés, M. Rodriguez, P. Hurtado, C. Abuín, C. Costa, A. B. Dávila Ibáñez, R. López-López, R. Piñeiro
          </td>
          <td>2025-01-17</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tyrosine kinase inhibitors (TKIs) have transformed the treatment of epidermal growth factor receptor (EGFR)-mutant non-small cell lung cancer. However, treatment resistance remains a major challenge in clinical practice. The tumor microenvironment (TME) is a complex system composed of tumor cells, immune and non-immune cells, and non-cellular components. Evidence indicates that dynamic changes in TME during TKI treatment are associated with the development of resistance. Research has focused on identifying how each component of the TME interacts with tumors and TKIs to understand therapeutic targets that could address TKI resistance. In this review, we describe how TME components, such as immune cells, fibroblasts, blood vessels, immune checkpoint proteins, and cytokines, interact with EGFR-mutant tumors and how they can promote resistance to TKIs. Furthermore, we discuss potential strategies targeting TME as a novel therapeutic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f6f1b01ac68ddc2dc3220a920e6b272e7041d2" target='_blank'>
              Targeting the Tumor Microenvironment in EGFR-Mutant Lung Cancer: Opportunities and Challenges
              </a>
            </td>
          <td>
            Jeong Uk Lim, Junyang Jung, Y. Kim, Chi Young Kim, Sang Hoon Lee, Dong Won Park, Sue In Choi, W. Ji, C. Yeo, Seung Hyeun Lee
          </td>
          <td>2025-02-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a small subset within the tumor mass significantly contributing to cancer progression through dysregulation of various oncogenic pathways, driving tumor growth, chemoresistance and metastasis formation. The aggressive behavior of CSCs is guided by several intracellular signaling pathways such as WNT, NF-kappa-B, NOTCH, Hedgehog, JAK-STAT, PI3K/AKT1/MTOR, TGF/SMAD, PPAR and MAPK kinases, as well as extracellular vesicles such as exosomes, and extracellular signaling molecules such as cytokines, chemokines, pro-angiogenetic and growth factors, which finely regulate CSC phenotype. In this scenario, tumor microenvironment (TME) is a key player in the establishment of a permissive tumor niche, where CSCs engage in intricate communications with diverse immune cells. The “oncogenic” immune cells are mainly represented by B and T lymphocytes, NK cells, and dendritic cells. Among immune cells, macrophages exhibit a more plastic and adaptable phenotype due to their different subpopulations, which are characterized by both immunosuppressive and inflammatory phenotypes. Specifically, tumor-associated macrophages (TAMs) create an immunosuppressive milieu through the production of a plethora of paracrine factors (IL-6, IL-12, TNF-alpha, TGF-beta, CCL1, CCL18) promoting the acquisition by CSCs of a stem-like, invasive and metastatic phenotype. TAMs have demonstrated the ability to communicate with CSCs via direct ligand/receptor (such as CD90/CD11b, LSECtin/BTN3A3, EPHA4/Ephrin) interaction. On the other hand, CSCs exhibited their capacity to influence immune cells, creating a favorable microenvironment for cancer progression. Interestingly, the bidirectional influence of CSCs and TME leads to an epigenetic reprogramming which sustains malignant transformation. Nowadays, the integration of biological and computational data obtained by cutting-edge technologies (single-cell RNA sequencing, spatial transcriptomics, trajectory analysis) has significantly improved the comprehension of the biunivocal multicellular dialogue, providing a comprehensive view of the heterogeneity and dynamics of CSCs, and uncovering alternative mechanisms of immune evasion and therapeutic resistance. Moreover, the combination of biology and computational data will lead to the development of innovative target therapies dampening CSC-TME interaction. Here, we aim to elucidate the most recent insights on CSCs biology and their complex interactions with TME immune cells, specifically TAMs, tracing an exhaustive scenario from the primary tumor to metastasis formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90ce167799371ac069e6df125e4a468fd20b0208" target='_blank'>
              Cancer stem cells and tumor-associated macrophages as mates in tumor progression: mechanisms of crosstalk and advanced bioinformatic tools to dissect their phenotypes and interaction
              </a>
            </td>
          <td>
            Francesco Verona, Sebastiano Di Bella, Roberto Schirano, Camilla Manfredi, Francesca Angeloro, Giulia Bozzari, M. Todaro, Giuseppe Giannini, Giorgio Stassi, V. Veschi
          </td>
          <td>2025-02-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="While pediatric cancer patients receive intensive chemotherapy, its impact on peripheral T cells and subsequently to disease outcomes are not fully characterized. Here, we assessed T-cell dynamics during treatment, identifying associations with outcomes through immune phenotyping and T-cell Receptor (TCR) sequencing in pediatric solid and hematologic malignancies. We show that while levels of immune checkpoint proteins (PD-1, LAG3, and TIM3) at baseline were highest in lymphomas compared to other cancer groups, they increased significantly in response to therapy in all cancers. Levels of Central Memory (CM) T cells increased in leukemias and solid tumors, while naïve T cells and cell-free TCR diversity decreased in lymphomas. By combining immune cell and TCR repertoire features across all timepoints, we proposed the Dynamic Immunogenomic Score (DIS) to measure patient-specific effects of therapy on the peripheral T-cell population. Higher DIS was associated with high-risk cancer types and logistic regression analysis revealed it may predict incidence of relapse in leukemia patients. TCR specificity analysis revealed patient-specific clonal dynamics and differential detection of virally-associated TCRs in cancer patients compared to healthy individuals. Our results highlight the potential of early upfront immunogenomic profiling in identifying high-risk patients that may be predictive in light of emerging cellular immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cebbb1d9bf5db9174d2b007c121d14b77fe78d1" target='_blank'>
              Immunogenomic profiling of circulating T cells in pediatric cancer patients during standard-of-care
              </a>
            </td>
          <td>
            A. Nabbi, Yiyue Jiang, Osvaldo Espin Garcia, Suluxan Mohanraj, S. Pedersen, Johanna Regala, Lauren M. Vernau, Jessica Perazzelli, David M. Barrett, Trevor J. Pugh
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background & Aims
Identifying patients with intrahepatic cholangiocarcinoma (ICC) likely to benefit from immunochemotherapy, the new front-line treatment, remains challenging. We aimed to unveil a novel radiotranscriptomic signature that can facilitate treatment response prediction by multi-omics integration and multi-scale modelling.


Methods
We analyzed bulk, single-cell and spatial transcriptomic data comprising 457 ICC patients to identify an immune-related score (IRS), followed by decoding its spatial immune context. We mapped radiomics profiles onto spatial-specific IRS using machine-learning to define a novel radiotranscriptomic signature, followed by multi-scale and multi-cohort validation covering 331 ICC patients. The signature was further explored for the potential therapeutic target from in vitro to in vivo.


Results
We revealed a novel 3-gene (PLAUR, CD40LG and FGFR4) IRS whose down-regulation correlated with better survival and improved sensitivity to immunochemotherapy. We highlighted functional IRS-immune interactions within tumor epithelium, rather than stromal compartment, irrespective of geospatial locations. Machine-learning pipeline identified the optimal 3-feature radiotranscriptomic signature that was well-validated by immunohistochemical assays in molecular cohort, exhibited favorable external prognostic validity with C-index over 0.64 in resection cohort, and predicted treatment response with an area under the curve of up to 0.84 in immunochemotherapy cohort. We also showed that anti-uPAR/PLAUR alone or in combination with anti-programmed cell death protein 1 therapy remarkably curbed tumor growth, using in vitro ICC cell lines and in vivo humanized ICC patient-derived xenograft mouse models.


Conclusions
This proof-of-concept study sheds light on the spatially-resolved radiotranscriptomic signature to improve patient selection for emerging immunochemotherapy and high-order immunotherapy combinations in ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b216340a3ee900bc1d9420b9d6cc7048a9f68c8" target='_blank'>
              Radiogenomics of intrahepatic cholangiocarcinoma predicts immunochemotherapy response and identifies therapeutic target.
              </a>
            </td>
          <td>
            Gu-Wei Ji, Zheng Xu, Shuo-Chen Liu, Shu-Ya Cao, C. Jiao, Ming Lu, Biao Zhang, Yue Yang, Qing Xu, Xiao-Feng Wu, Ke Wang, Yong-xiang Xia, Xiang-Cheng Li, Xue-Hao Wang
          </td>
          <td>2025-02-10</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Personalized precision medicine (PPPM) in cancer immunology and oncology is a rapidly advancing field with significant potential. Gliomas, known for their poor prognosis, rank among the most lethal brain tumors. Despite advancements, there remains a critical need for precise, individualized treatment strategies. Methods We conducted a comprehensive analysis of RNA-seq and microarray data from the TCGA and GEO databases, supplemented by single-cell RNA sequencing (scRNA-seq) data from glioma patients. By integrating single-cell sequencing analysis with foundational experiments, we investigated the molecular variations and cellular interactions within neural glioma cell subpopulations during tumor progression. Results Our single-cell sequencing analysis revealed distinct gene expression patterns across glioma cell subpopulations. Notably, differentiation trajectory analysis identified NUSAP1 as a key marker for the terminal subpopulation. We found that elevated NUSAP1 expression correlated with poor prognosis, prompting further investigation of its functional role through both cellular and animal studies. Conclusions NUSAP1-based risk models hold potential as predictive and therapeutic tools for personalized glioma treatment. In-depth exploration of NUSAP1’s mechanisms in glioblastoma could enhance our understanding of its response to immunotherapy, suggesting that targeting NUSAP1 may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5625f39e4872fec67e5df1a449a03f506dae3b79" target='_blank'>
              Single-cell sequencing elucidates the mechanism of NUSAP1 in glioma and its diagnostic and prognostic significance
              </a>
            </td>
          <td>
            Meng-Yu Zhao, Zhao-Lei Shen, Hongzhen Dai, Wan-Yan Xu, Li-Na Wang, Yu- Gu, Jie-Hui Zhao, Tian-Hang Yu, Cun-Zhi Wang, Jia-feng Xu, Guan-Jun Chen, Dong-Hui Chen, Wen-Ming Hong, Fan Zhang
          </td>
          <td>2025-02-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Spatially resolved transcriptomics (SRT) has emerged as a transformative technology for elucidating cellular organization and tissue architecture. However, a significant challenge remains in identifying pathology-relevant spatial functional landscapes within the tissue microenvironment, primarily due to the limited integration of cell-cell communication dynamics. To address this limitation, SpaDCN, a Spatially Dynamic graph Convolutional Network framework is proposed, which aligns cell-cell communications and gene expression within a spatial context to reveal the spatial functional regions with the coherent cellular organization. To effectively transfer the influence of cell-cell communications on expression variation, SpaDCN respectively generates the node layer and edge layer of spatial graph representation from expression data and the ligand-receptor complex contributions and then employs a dynamic graph convolution to switch the propagation of node graph and edge graph. It is demonstrated that SpaDCN outperforms existing methods in identifying spatial domains and denoising expression across various platforms and species. Notably, SpaDCN excels in identifying marker genes with significant prognostic potential in cancer tissues. In conclusion, SpaDCN offers a powerful and precise tool for spatial domain detection in spatial transcriptomics, with broad applicability across various tissue types and research disciplines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8e1ac36ae0a43d7b28635df6f8edaf8554a72a9" target='_blank'>
              SpaDCN: Deciphering Spatial Functional Landscape from Spatially Resolved Transcriptomics by Aligning Cell-Cell Communications.
              </a>
            </td>
          <td>
            Xiaosheng Bai, Xinyu Bao, Chuanchao Zhang, Qianqian Shi, Luonan Chen
          </td>
          <td>2025-02-17</td>
          <td>Small methods</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Rhabdomyosarcoma (RMS) is a rare, yet aggressive skeletal muscle malignancy predominantly affecting children under ten years old. RMS tumors are canonically enriched with several myeloid populations, many of which can sense and respond to soluble and biophysical cues in the tumor microenvironment (TME). Store-operated Ca2+ Entry (SOCE) was the presumed calcium signaling mechanism that myeloid cells, specifically dendritic cells (DCs) and macrophages, utilize to perform their critical functions. Shockingly, in 2015, Vaeth et. al determined that SOCE was not necessary for their functionality begging the question: what calcium channel(s) underly myeloid cells’ phenotypic and functional plasticity in the highly dynamic TME? We propose expression of the mechanosensitive cation channel, PIEZO1, supports a pro-tumor phenotype in tumor-associated myeloid cells (TAMCs) while its deletion skews this population to anti-tumor functioning thereby propagating the RMS anti—tumor response. The impact of myeloid cells’ expression of PIEZO1 on tumor progression was assessed by orthotopically inoculating a novel murine CD11b-conditional Piezo1 knockout (CD11b cKO) strain with syngeneic RMS cells and performing downstream analyses. Compared to wild-type (WT) mice, we found CD11b cKO mice developed tumors at slower rates and had smaller primary tumors and metastases at the pre-clinical endpoint. Analysis of the bulk tumors’ supernatants revealed enhanced anti-tumor changes in the cKO mice including increased CCL21 and CXCL10 and decreased EGF. Flow cytometry revealed anti-tumor immunophenotype changes in the cKO mice including increased numbers of intra-tumoral monocytes, CD11b+ dendritic cells, CD8+ T-cells, and FOXP3- CD4+ T-cells. To home in on the population(s) driving the CD11b cKO phenotype, we performed parallel tumor growth experiments in our newly generated CX3CR1-conditional homozygous Piezo1 knockout (CX3CR1 cKO) mouse strain. We found although the CX3CR1 cKO tumors were smaller than WT, their size was not significantly different from either WT or CD11b cKO. Combined, these in vivo data suggest CD11b+ DCs, specifically monocyte-derived dendritic cells, are a main driver of my CD11b cKO phenotype. To further characterize these DCs, we performed phenotypic and functional analyses on isolated splenic CD11b+ DCs and bone marrow-derived dendritic cells (BMDCs). Flow cytometric analysis demonstrated that WT and CD11b cKO DCs had similar activation marker expression. Surprisingly, co-culture of either CD11b cKO DC population with naïve OT-I CD8+ T-cells after soluble OVA pulse invoked enhanced CD69 and Interferon-g expression in the T-cells. Together, our findings suggest Piezo1 deletion in CD11b+ DCs drives a robust RMS anti-tumor response. Without PIEZO1, the ability to endocytose and cross-present soluble antigen to CD8+ T-cells was unleashed- functions never described as being PIEZO1-related. We will further characterize these mechanisms through scRNA-Seq of bulk tumors with the ultimate hope of developing better DC-related cancer therapeutics.
 Citation Format: Melissa Romain Bonner, David Askew, Saada Eid, Suzanne L Tomchuck, muta Abiff, Jay Myers, Justin Garyu, Alex Huang. Deletion of Piezo1 in CD11b+ Dendritic Cells Amplifies the Local and Systemic Rhabdomyosarcoma Anti-Tumor Response [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B099.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba54436455eb8d5189fb45709f89284b938e52f3" target='_blank'>
              Abstract B099: Deletion of Piezo1 in CD11b+ Dendritic Cells Amplifies the Local and Systemic Rhabdomyosarcoma Anti-Tumor Response
              </a>
            </td>
          <td>
            Melissa Bonner, David Askew, S. Eid, Suzanne L Tomchuck, muta Abiff, Jay T. Myers, Justin Garyu, Alex Huang
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer deaths in the United States. New targeted therapies against the once-deemed undruggable oncogenic KRAS are changing current therapeutic paradigms. However, resistance to targeted KRAS inhibitors almost inevitably occurs; resistance can be driven by tumor cell–intrinsic changes or by changes in the microenvironment. Here, we utilized a genetically engineered mouse model of KRASG12D-driven lung cancer that allows for inducible and reversible expression of the oncogene: activation of oncogenic KRASG12D induces tumor growth; conversely, inactivation of KRASG12D causes tumor regression. We showed that in addition to regulating cancer cell growth and survival, oncogenic KRAS regulated the transcriptional status of cancer-associated fibroblasts and macrophages in this model. Utilizing ex vivo approaches, we showed that secreted factors from cancer cells induced the expression of multiple cytokines in lung fibroblasts, and in turn drove expression of immunosuppressive factors, such as arginase 1, in macrophages. In summary, fibroblasts emerged as a key source of immune regulatory signals, and a potential therapeutic target for improving the efficacy of KRAS inhibitors in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdae42d5132ec7ce05cdb9d5dd6bd5ef386a7a36" target='_blank'>
              KRASG12D drives immunosuppression in lung adenocarcinoma through paracrine signaling
              </a>
            </td>
          <td>
            Emily L. Lasse-Opsahl, Ivana Barravecchia, Elyse McLintock, Jen-Mei Lee, Sarah F Ferris, Carlos E Espinoza, Rachael Hinshaw, Sophia Cavanaugh, Marzia Robotti, Lily Rober, Kristee L Brown, Kristena Y. Abdelmalak, C. Galbán, Timothy L. Frankel, Yaqing Zhang, M. Pasca di Magliano, Stefanie Galbán
          </td>
          <td>2025-01-09</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background: Melanoma is a highly heterogeneous disease, and a deeper molecular classification is essential for improving patient stratification and treatment approaches. Here, we describe the histopathology-driven proteogenomic landscape of 142 treatment-naïve metastatic melanoma samples to uncover molecular subtypes and clinically relevant biomarkers. Methods: We performed an integrative proteogenomic analysis to identify proteomic subtypes, assess the impact of BRAF V600 mutations, and study the molecular profiles and cellular composition of the tumor microenvironment. Clinical and histopathological data were used to support findings related to tissue morphology, disease progression, and patient outcomes. Results: Our analysis revealed five distinct proteomic subtypes that integrate immune and stromal microenvironment components and correlate with clinical and histopathological parameters. We demonstrated that BRAF V600-mutated melanomas exhibit biological heterogeneity, where an oncogene-induced senescence-like phenotype is associated with improved survival. This led to a proposed mortality risk-based stratification that may contribute to more personalized treatment strategies. Furthermore, tumor microenvironment composition strongly correlated with disease progression and patient outcomes, highlighting a histopathological connective tissue-to-tumor ratio assessment as a potential decision-making tool. We identified a melanoma-associated SAAV signature linked to extracellular matrix remodeling and SAAV-derived neoantigens as potential targets for anti-tumor immune responses. Conclusions: This study provides a comprehensive stratification of metastatic melanoma, integrating proteogenomic insights with histopathological features. The findings may aid in the development of tailored diagnostic and therapeutic strategies, improving patient management and outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557557036af2f6be01a0b554ada4c910b8468881" target='_blank'>
              Proteogenomic Profiling of Treatment-Naïve Metastatic Malignant Melanoma
              </a>
            </td>
          <td>
            M. Kuras, L. Betancourt, R. Hong, L. Szadai, Jimmy Rodriguez, Peter Horvatovich, I. Pla, Jonatan Eriksson, B. Szeitz, Bartłomiej Deszcz, Charlotte Welinder, Y. Sugihara, Henrik Ekedahl, Bo Baldetorp, C. Ingvar, L. Lundgren, H. Lindberg, Henriett Oskolás, Z. Horváth, M. Rezeli, Jeovanis Gil, R. Appelqvist, L. Kemény, Johan Malm, Aniel Sanchez, A. M. Szasz, Krzysztof Pawłowski, E. Wieslander, David Fenyö, I. Németh, Gyorgy Marko-Varga
          </td>
          <td>2025-02-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Approximately 50% of cancers exhibit decreased CDKN2A expression (CDKN2ALow), which is linked to immune checkpoint blockade (ICB) resistance. While CDKN2A is traditionally recognized as a tumor suppressor and cell cycle regulator, we have previously put forth a new paradigm demonstrating its role in intracellular metabolic reprogramming. Whether the metabolic derangement due to CDKN2A loss alters metabolites within the tumor microenvironment (TME) and how that affects the immune compartment and ICB response has never been investigated. Here we found that CDKN2ALow cancer cells reorganize zinc compartmentalization by upregulating the zinc importer SLC39A9 in the plasma membrane, leading to intracellular zinc accumulation in cancer cells and concurrent zinc depletion in the TME. This competition for zinc results in zinc-starved macrophages, leading to reduced phagocytic activity. Remarkably, restoring zinc levels in the TME through a dietary intervention re-educates macrophages to a pro-phagocytic phenotype, sensitizing CDKN2ALow tumors to ICB. Unexpectedly, T cells are not required for this response. Clinically, macrophages from CDKN2ALow cancer patients have decreased zinc signatures, corresponding to reduced phagocytosis signatures. Moreover, patients with low circulating zinc levels have reduced time-to-event outcomes compared to those with higher zinc levels. Our work reveals a previously unrecognized mechanism through which CDKN2ALow cancer cells outcompete macrophages for zinc, directly disrupting their function and ICB efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6884694ba6496f9db08cf20da9f0a850d79297b0" target='_blank'>
              CDKN2ALow cancer cells outcompete macrophages for microenvironmental zinc to drive immunotherapy resistance
              </a>
            </td>
          <td>
            R. Buj, Aidan R Cole, Jeff Danielson, Jimmy Xu, Drew Hurd, Akash Kishore, Katarzyna M. Kedziora, Jie Chen, Baixue Yang, David Barras, Apoorva Uboveja, A. Amalric, Juan J Apiz Saab, J. Wickramasinghe, N. Tangudu, Evan Levasseur, Hui Wang, Aspram Minasyan, Rebekah Dadey, Allison C. Sharrow, F. Vendetti, Dayana B Rivadeneira, Christopher J. Bakkenist, Greg M Delgoffe, Nadine Hempel, Nathaniel W. Snyder, Riyue Bao, A. Soloff, John M. Kirkwood, Denarda Dangaj Laniti, Andrew V. Kossenkov, Alexander Muir, Jishnu Das, D. Davar, C. Mesaros, K. Aird
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72ace1bf5ccc7bc1a835b1fd333db7f3f67e0b96" target='_blank'>
              Lessons learned from spatial transcriptomic analyses in clear-cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Jesper Jespersen, Cecilie Lindgaard, Laura Iisager, J. Ahrenfeldt, Iben Lyskjær
          </td>
          <td>2025-01-09</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729ab293566d42d02b2188c02492d9921394583a" target='_blank'>
              Organoid modeling reveals the tumorigenic potential of the alveolar progenitor cell state.
              </a>
            </td>
          <td>
            Jingyun Li, Susanna M Dang, Shreoshi Sengupta, Paul Schurmann, A. F. Dost, Aaron L. Moye, M. F. Trovero, Sidrah Ahmed, Margherita Paschini, Preetida J Bhetariya, Roderick Bronson, Shannan J. Ho Sui, Carla F. Kim
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/997e954cb156caa69cf5f6fb240e0f9297f38fc1" target='_blank'>
              Single-cell and spatial-resolved profiling reveals cancer-associated fibroblast heterogeneity in colorectal cancer metabolic subtypes
              </a>
            </td>
          <td>
            Youpeng Wang, Xingfeng Qiu, Qinghai Li, Jiale Qin, Lvlan Ye, Xiang Zhang, Xingxiang Huang, Xiangqiong Wen, Ziyang Wang, Weiling He, Yuqin Di, Qi Zhou
          </td>
          <td>2025-02-11</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Dendritic cells (DCs) are promising targets for cancer immunotherapies because of their central role in the initiation and control of immune responses. The rare cDC1 population is of particular interest because of its remarkable ability to cross-present antigens (Ag) to CD8+ T cells, to promote Th1 cell polarization and NK cell activation and recruitment. However, the spatial organization and specific functions of cDC1s in response to immunotherapy remain to be clearly characterized in human tumors. Here, we implemented a multiplexed immunofluorescence analysis pipeline coupled with computational image analysis to determine the spatial organization of the cDC1 subset in a cohort of skin lesions from advanced melanoma patients treated with immune checkpoint inhibitors (ICI). For this, we performed a whole-slide image analysis of cDC1 infiltration and distribution as well as their spatial interactions with key immune partners such as CD8+ T cells and pDC according to the response of patients to ICI. We also analyzed LAMP3+-DC, which correspond to a mature subset of tumor-infiltrating DCs. Distance and cell network analyses demonstrated that cDC1s exhibited a scattered distribution compared to tumor-infiltrating pDCs and LAMP3+-DCs, which were preferentially organized in dense areas with high homotypic connections. Interestingly, the proximity and interactions between CD8+ T cells and cDC1s were positively associated with the response to ICI. In conclusion, our study unravels the complex spatial organization of cDC1s and their interactions with CD8+ T cells in melanoma patient lesions, shedding light on their pivotal role in shaping the response to ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f10d121483bbab07523097b63187cdc94830056d" target='_blank'>
              The spatial organization of cDC1 with CD8+ T cells is critical for the response to immune checkpoint inhibitors in melanoma patients.
              </a>
            </td>
          <td>
            Elisa Gobbini, Margaux Hubert, A. Doffin, Anaïs Eberhardt, Léo Hermet, Danlin Li, Pierre Duplouye, Sarah Ghamry-Barrin, Justine Berthet, Valentin Benboubker, Maxime Grimont, C. Sakref, Jimmy Perrot, Garance Tondeur, Olivier Harou, Jonathan Lopez, Bertrand Dubois, Stéphane Dalle, Christophe Caux, J. Caramel, J. Valladeau-Guilemond
          </td>
          <td>2025-01-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e89dc2d087d9d2c969c5bb0a4b45e01fc9cc0f7a" target='_blank'>
              Single-cell and spatial genomic landscape of non-small cell lung cancer brain metastases.
              </a>
            </td>
          <td>
            Somnath Tagore, Lindsay A. Caprio, Amit Dipak Amin, K. Bestak, Karan Luthria, Edridge K. D’Souza, Irving Barrera, Johannes C. Melms, Sharon Wu, Sinan Abuzaid, Yiping Wang, Viktoria Jakubikova, Peter Koch, D. Z. Brodtman, Banpreet Bawa, S. Deshmukh, Leon Ebel, Miguel A. Ibarra-Arellano, A. Jaiswal, Carino Gurjao, Jana Biermann, Neha Shaikh, Priyanka Ramaradj, Yohanna Georgis, Galina G Lagos, Matthew I Ehrlich, P. Ho, Zachary H. Walsh, Meri Rogava, Michelle Garlin Politis, Devanik Biswas, A. Cottarelli, Nikhil Rizvi, CA Shu, Benjamin O Herzberg, N. Anandasabapathy, G. Sledge, Emmanuel Zorn, Peter D Canoll, Jeffrey N. Bruce, Naiyer A Rizvi, Alison M Taylor, Anjali Saqi, H. Hibshoosh, Gary K Schwartz, Brian S Henick, Fei Chen, Denis Schapiro, Parin Shah, Benjamin Izar
          </td>
          <td>2025-02-27</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Mouse models that faithfully represent the biology of human brain tumors are critical tools for unraveling the underlying tumor biology and screening for potential precision therapies. This is especially true of rare tumor types, many of which have correspondingly few xenograft or cell lines available. Although our understanding of the specific biological pathways driving cancer has improved significantly, identifying the appropriate progenitor populations to drive oncogenic processes represents a significant barrier to efficient mouse model production. In this issue of Cancer Research, Jessa and colleagues developed an innovative transcription factor fingerprinting method to map the cellular origin of central nervous system neuroblastoma, FOXR2-activated to medial ganglionic eminence-derived interneurons, which could then be efficiently targeted in the developing mouse brain using in utero electroporation. This approach serves as a blueprint for investigating other rare pediatric brain tumors, potentially accelerating progress toward the development of mouse models and identification of effective therapies. See related article by Jessa et al., p. 231.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b36b1d9ff773da8434bf45b0c13f61a915472b5f" target='_blank'>
              Transcription Factor Fingerprint Provides Clues for Brain Tumor Cell of Origin.
              </a>
            </td>
          <td>
            Brent A. Orr
          </td>
          <td>2024-01-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 256


 Background:
 Resistance to Androgen Receptor Pathway Inhibitors (ARPI) in metastatic prostate cancer (mPC) is universal and can be driven by complex genomic alterations. The evolution of lineage state transitions from adenocarcinomas to neuroendocrine prostate cancer (NEPC) has also been shown to drive treatment resistance and poor survival. Identifying the timing and association of genomic mutations with lineage state transitions has been limited by the need for serial tumor biopsies. We report an integrated analysis of clinical next-generation sequencing (NGS) data and mPC lineage states with a novel circulating tumor cell (CTC) RNA sequencing methodology.
 Methods:
 We collected 273 samples from 117 unique patients with mPC in a prospective biomarker trial. CTCs were purified via immunomagnetic capture on an automated microfluidic technology and analyzed via RNA-seq. A subset of patients had a clinical grade genetic test performed on a contemporary metastatic tissue biopsy. We compared CTC gene expression for lineage states (luminal A, luminal B, NEPC) with somatic gene mutation subtypes known to confer resistance to ARPIs (AR, p53, RB, PTEN, HRR genes) and overall survival (OS).
 Results:
 Single sample pathway analysis of high CTC purity samples identified four transcriptional phenotypes: luminal A-like (LumA), luminal B-like (LumB), low proliferation (LP), and neuroendocrine (NE). Compared to patients with low CTC burden/purity (median OS not reached), patients with LumA and LP phenotypes had similar survival, patients with LumB and NE CTC phenotypes had shorter survival (LumB: median OS 6m, HR 9.1 [3.8-21.8], p<0.0001, NE: median OS 3.7m, HR 11.8 [2.4-57.2], p=0.0019). AR alterations were found in samples with prior ARPI exposure and at similar frequencies across all CTC phenotypes but NE where they were absent. RB mutations were enriched in the unfavorable CTC phenotypes (LumB, NE; p=0.0288). Integrating CTC phenotype with presence or absence of at least one high risk genomic alteration (AR, RB, TP53, PTEN), patients with favorable CTC phenotype (Low burden, LP and LumA) and no high risk alterations had the longest median survival (median OS NR), followed by patients with favorable phenotype but a high risk alteration (median OS 12.7m), patients with an unfavorable phenotype (LumB, NE) but no high risk alteration (median OS 8m), and unfavorable phenotype with a high risk alteration (median OS 4m).
 Conclusions:
 Somatic mutations in mCRPC influence lineage state acquisitions and treatment resistance. Lineage state transitions themselves are associated with poor outcomes and decreased survival which is worsened in the presence of high-risk genetic mutations including RB. This data presents potentially targetable patient populations that would benefit from treatment intensification and early disease monitoring for more aggressive mCRPC subtypes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dba71a7a1acb2ee64ae5bd588ebd4c00f7170961" target='_blank'>
              Investigating clinical associations between genomic alterations and prostate cancer lineage states using circulating tumor cell RNA sequencing.
              </a>
            </td>
          <td>
            Katharine E Tippins, Joshua M Lang, Marina N. Sharifi, J. Sperger, Shannon R Reese, Amy K Taylor, Viridiana Carreno, Alex H Chang, Meghan Wells, John M Floberg, H. Emamekhoo, Shuang Zhao, D. Kosoff
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbee10e353cac2b48554f45773102121a37f67" target='_blank'>
              Endothelial-secreted Endocan activates PDGFRA and regulates vascularity and spatial phenotype in glioblastoma
              </a>
            </td>
          <td>
            Soniya Bastola, M. Pavlyukov, Neel Sharma, Yasmin Ghochani, Mayu A. Nakano, S. Muthukrishnan, Sang Yul Yu, Min Soo Kim, Alireza Sohrabi, Natalia P. Biscola, Daisuke Yamashita, Ksenia S. Anufrieva, Tatyana F. Kovalenko, Grace Jung, Tomas Ganz, Beatrice O’Brien, R. Kawaguchi, Yue Qin, S. Seidlits, Alma L. Burlingame, J. Oses-Prieto, Leif A. Havton, Steven A. Goldman, A. Hjelmeland, Ichiro Nakano, H. Kornblum
          </td>
          <td>2025-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="A comprehensive spatial distribution of the proteome in human body and cancers is fundamental for understanding human biology and diseases including cancers. Here, we present an anatomically resolved human proteome derived from 1781 benign and malignant samples from 58 major tissue types encompassing 251 specific tissues and 25 carcinomas. Based on a spectral library covering over 75% of the human protein-coding genes with 208 understudied and 82 missing proteins characterized, we quantified over 13,000 proteins in these samples using data-independent acquisition proteomics. This data resource presents the so far most comprehensive quantitative proteomic landscape of human tissues and common carcinomas. It allows systematic evaluation of tissue-specific drug responses, identification of drug candidates that may be repurposed as antineoplastics, and discovery of novel targets for anticancer therapy. This resource, available as an online knowledgebase, refines our knowledge of spatial distribution of the human proteome and tumor-specific protein modulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5372012557b72610a59f72be2bf39f2688654856" target='_blank'>
              Spatial distribution of the proteome in human body and cancers
              </a>
            </td>
          <td>
            Liang Yue, Wenhao Jiang, Sainan Li, Meng Luo, Ning Fan, Xiaolu Zhan, Kexin Chen, Tian Lu, Fang Guo, Dongwei Li, Weigang Ge, Zongxiang Nie, M. Lyu, A. Jun, Yingrui Wang, Yingdan Chen, Zhenhai Fu, Nan Xiang, Lu Li, Fengchao Yu, Guo Ci Teo, Alexey I. Nesvizhskii, Meng Wang, Michael P. Snyder, Ben C. Collins, Ruedi Aebersold, Fei Xu, Yi Zhu, Tong Liu, Yan Li, Tiannan Guo
          </td>
          <td>2025-02-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Although somatic mutations are fundamentally important to human biology, disease, and aging, many outstanding questions remain about their rates, spectrum, and determinants in apparently healthy tissues. Here, we performed high-coverage exome sequencing on 265 samples from 14 GTEx donors sampled for a median of 17.5 tissues per donor (spanning 46 total tissues). Using a novel probabilistic method tailored to the unique structure of our data, we identified 8,470 somatic variants. We leverage our compendium of somatic mutations to quantify the burden of deleterious somatic variants among tissues and individuals, identify molecular features such as chromatin accessibility that exhibit significantly elevated somatic mutation rates, provide novel biological insights into mutational mechanisms, and infer developmental trajectories based on patterns of multi-tissue somatic mosaicism. Our data provides a high-resolution portrait of somatic mutations across genes, tissues, and individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93698e2f53feff8e8bfbf3cca400ebd477df43e0" target='_blank'>
              Landscape of human protein-coding somatic mutations across tissues and individuals
              </a>
            </td>
          <td>
            Huixin Xu, Rob Bierman, Dayna Akey, Cooper Koers, Troy Comi, Claire McWhite, Joshua M. Akey
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5472095a4c3c0313c90e7e9724e09004bb9a2fb6" target='_blank'>
              From silent partners to potential therapeutic targets: macrophages in colorectal cancer
              </a>
            </td>
          <td>
            Hayat Khizar, K. Ali, Jianwei Wang
          </td>
          <td>2025-02-25</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The mechanisms contributing to metastasis tropism in breast cancer are far from being understood. The ability of cancer cells to adapt to new microenvironments plays a crucial role in that process. In this issue of Cancer Research, Jiang and colleagues explored the tumor-intrinsic differences that affect seeding and tissue colonization by focusing on two major sites of breast cancer metastasis: the lung and bone. They used genetic approaches and metastasis modeling to assess the role of forkhead box F2 (FOXF2) in remodeling the tumor microenvironment and promoting metastasis outgrowth. Their findings suggest that FOXF2 promotes a micrometastasis-to-macrometastasis transition in bones through the direct activation of NF-κB and BMP4 signaling pathways, independently of tumor subtype. In contrast, FOXF2 was found to hinder the ability of basal models to colonize the lung microenvironment through inhibition of TGFβ signaling. Collectively, these findings underscore the subtype-specific and tissue-dependent roles of FOXF2 in organotropism. See related article by Jiang et al., p. 644.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/022dab004bf16dba9f0a4cb5a10ba6900ae86601" target='_blank'>
              Deterministic Role of FOXF2 in Organ-Specific Macrometastasis Transition.
              </a>
            </td>
          <td>
            Igor L Bado
          </td>
          <td>2025-02-17</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1f7c17606ae67d5ce7fe611db0e7bce958444bb" target='_blank'>
              Decoding the heterogeneous subpopulations of glioblastoma for prognostic stratification and uncovering the promalignant role of PSMC2
              </a>
            </td>
          <td>
            Nini Zhou, Jingsong Yan, Manya Xiong, Shunqin Zhu
          </td>
          <td>2025-02-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa45e1efe02786e2e4534cf1ae408e65c7ffadc" target='_blank'>
              To describe the subsets of malignant epithelial cells in gastric cancer, their developmental trajectories and drug resistance characteristics
              </a>
            </td>
          <td>
            Tingting Xu, Tianying Zhang, Yan Sun, Sijia Wu
          </td>
          <td>2025-01-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab8b68e7b5a2f581513237acff3c4e88418f66b" target='_blank'>
              Haploinsufficient phenotypes promote selection of PTEN and ARID1A-deficient clones in human colon.
              </a>
            </td>
          <td>
            Nefeli Skoufou-Papoutsaki, Sam Adler, Shenay Mehmed, Claire Tume, Cora Olpe, Edward Morrissey, Richard Kemp, Anne-Claire Girard, Elisa B. Moutin, C. Chilamakuri, Jodi L Miller, Cecilia Lindskog, Fabian Werle, Kate Marks, Francesca Perrone, Matthias Zilbauer, D. Tourigny, Doug Winton
          </td>
          <td>2025-02-07</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Liver and lung metastases are major drivers of breast cancer-related morbidity, characterized by poor therapeutic outcomes and diminished overall survival. These dynamic tumor microenvironments (TMEs) display distinct, site-specific cellular architectures, including cytotoxic lymphocytes, immune suppressor cells, and stromal components, which together facilitate tumor growth and metastatic spread. Varied responses to immune checkpoint inhibition have also been observed in patients with differential tumor burden, improved responses in those with higher lung metastatic burden and poor responses in those with higher liver metastatic burden. Specific immune populations driving observed differential responses have not been fully characterized. Utilizing syngeneic orthotopic mouse models and FFPE tissue samples from patients with primary and metastatic breast cancer, we investigated the organ-specific cellular composition, cell-cell interactions, and molecular pathways underlying tumor progression, immune escape, and site-specific metastasis. Flow cytometry analysis of orthotopic murine primary breast, breast-to-liver, and breast-to-lung tumors demonstrated that CD3+ lymphocytes initially accumulate in primary tumors soon after tumor inoculation and decline significantly over time. Despite this overall lymphocyte depletion within TMEs, the lungs consistently exhibit the highest lymphocyte levels. In contrast, myeloid-derived suppressor cells (MDSCs) progressively infiltrate both primary and distant sites, exerting immunosuppressive functions that likely promote tumor progression and inhibit immune responses. Both granulocytic and monocytic MDSCs suppress CD8+ T cell proliferation through distinct organ-specific mechanisms, with the strongest suppressive effects observed in the liver TME. Validation of these findings in patient specimens using high-resolution spatial transcriptomics from breast and distant breast metastases using 10X Genomics (Visium and Visium-HD platforms) revealed that liver metastases harbor lower immune-to-stromal ratio with greater MDSCs: CD8+ colocalization, and higher expression of genes associated with proliferation, angiogenesis, anti-apoptosis and antiinflammation relative to primary breast tumors. In addition, spatiotemporal plasticity of breast TMEs is specific to the disease site, and investigation of how these observations could impact tumor progression and response to immune checkpoint inhibition is underway. In summary, tissue-specific programming of MDSCs in breast-to-liver metastasis compared with primary and breast-to-lung lesions leads to reduced T cell infiltration and function, and could contribute to a lack of response to checkpoint inhibition and could guide therapeutic options for patients.
 Citation Format: Batul Al-zubeidy, Edgar Gonzalez, Aaron Baugh, Dominic Zavala, Mathew Jacobo, Michael Press, Evanthia Roussos Torres. Myeloid suppression within metastatic tumor microenvironments in advanced breast cancer—friend or foe to checkpoint inhibition? [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A086.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01243dcc84f5d3d7fbdd4aa36ab8b1b63e350fae" target='_blank'>
              Abstract A086: Myeloid suppression within metastatic tumor microenvironments in advanced breast cancer—friend or foe to checkpoint inhibition?
              </a>
            </td>
          <td>
            Batul Al-zubeidy, Edgar Gonzalez, A. Baugh, Dominic Zavala, Mathew Jacobo, Michael F. Press, E. R. Torres
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Breast cancer is the most frequently diagnosed malignancy and a leading cause of cancer-related mortality among women worldwide. Although NADPH oxidase 4 (NOX4) has been implicated in various oncogenic processes, its exact function in breast cancer progression, metabolic reprogramming, and immune modulation remains unclear. Methods We used murine 4T1 and EO771 breast cancer models to generate NOX4 knockout (KO) cell lines via CRISPR/Cas9. In vitro assays (cell proliferation, colony formation, wound healing, and Seahorse metabolic analyses) and in vivo orthotopic tumor studies assessed the impact of NOX4 loss. Transcriptomic changes were identified through RNA sequencing and gene set enrichment analysis. We performed MYC knockdown in NOX4 KO cells to investigate its mechanistic role. Flow cytometry characterized tumor-infiltrating immune cells. Finally, NOX4-overexpressing cells were tested for survival benefit and response to dual-checkpoint immunotherapy (anti-PD-1/anti-CTLA-4). Results NOX4 deletion accelerated tumor growth in vivo and enhanced proliferation, colony formation, and migratory capacity in vitro. Metabolic profiling showed that NOX4 KO cells had elevated glycolysis and fatty acid oxidation, along with increased mitochondrial mass. Transcriptomic and enrichment analyses revealed MYC pathway activation in NOX4 KO cells; suppressing MYC reversed these hyperproliferative and metabolic changes. Immunologically, NOX4 KO reduced CD8+ T cell infiltration and function, partially due to lowered CCL11/CCL5 levels, while PD-L1 expression was upregulated. In contrast, NOX4 overexpression improved survival in mice and synergized with checkpoint blockade, demonstrating a positive effect on anti-tumor immunity. Discussion These findings show that NOX4 constrains breast cancer aggressiveness by limiting MYC-driven metabolic adaptations and supporting CD8+ T cell-mediated immunity. Loss of NOX4 promotes a more malignant phenotype and dampens T cell responses, whereas its overexpression prolongs survival and enhances checkpoint inhibitor efficacy. Therapeutically targeting the NOX4–MYC axis and leveraging NOX4’s immunomodulatory capacity could offer promising strategies for breast cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b9114ae3608de23e0dbc2cbea2c8b3a88bf3557" target='_blank'>
              NOX4 modulates breast cancer progression through cancer cell metabolic reprogramming and CD8+ T cell antitumor activity
              </a>
            </td>
          <td>
            Yingying Xiong, Y. Weng, Shan Zhu, Jian Qin, Jia Feng, Xiaopeng Jing, Chao Luo, Wei Gong, Rui Sun, Min Peng
          </td>
          <td>2025-02-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Solid tumours are intricate and highly heterogeneous ecosystems, which grow in and invade normal organs. Their progression is mediated by cancer cells’ interaction with different cell types, such as immune cells, stromal cells and endothelial cells, and with the extracellular matrix. Owing to its high incidence, aggressive growth and resistance to local and systemic treatments, liver cancer has particularly high mortality rates worldwide. In recent decades, spatial heterogeneity has garnered significant attention as an unfavourable biological characteristic of the tumour microenvironment, prompting extensive research into its role in liver tumour development. Advances in spatial omics have facilitated the detailed spatial analysis of cell types, states and cell‒cell interactions, allowing a thorough understanding of the spatial and temporal heterogeneities of tumour microenvironment and informing the development of novel therapeutic approaches. This review illustrates the latest discovery of the invasive zone, and systematically introduced specific macroscopic spatial heterogeneities, pathological spatial heterogeneities and tumour microenvironment heterogeneities of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27ba251a9613c3090a1b350987469c7d7cfc3f02" target='_blank'>
              Spatial‒temporal heterogeneities of liver cancer and the discovery of the invasive zone
              </a>
            </td>
          <td>
            Jiayan Yan, Zhifeng Jiang, Shiyu Zhang, Qichao Yu, Yijun Lu, Runze Miao, Zhaoyou Tang, Jia Fan, Liang Wu, Dan G. Duda, Jian Zhou, Xinrong Yang
          </td>
          <td>2025-02-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background The role of cancer-associated pericytes (CAPs) in tumor microenvironment (TME) suggests that they are potential targets for cancer treatment. The mechanism of CAP heterogeneity in esophageal squamous cell carcinoma (ESCC) remains unclear, which has limited the development of treatments for tumors through CAPs. Therefore, a comprehensive understanding of the classification, function, cellular communication and spatial distribution of CAP subpopulations in ESCC is urgently needed. Methods This study used large-sample single-cell transcriptome sequencing (scRNA-seq) data to investigate pericytes’ subpopulation characteristics, functions, upstream and downstream regulation and interactions with other components of the TME in the ESCC, and analyzed prognostically in conjunction with Bulk RNA-seq data. In addition, pericyte subpopulations were validated and their spatial distribution in the ESCC TME was observed by multiplex immunofluorescence. Drug prediction and molecular docking was further used to validate the medicinal value of drug targets. Results CAPs in the ESCC TME were found to be highly heterogeneous, and we identified six pericyte subtypes: c1_ARHGDIB, c2_BCAM, c3_LUM, c4_SOD2, c5_TYMS, and c6_KRT17, which have commonality in a part of their functions, and each of them has a major function to play, by having different strengths of interaction with different components in the TME. In addition, we found that c4_SOD2 was negatively correlated with prognosis, conversely, c5_TYMS was positively correlated with prognosis. The drug with a better effect on c5_TYMS was docetaxel (binding energy = -8.1, -8.7 kcal/mol); raloxifene may be more effective against c4_SOD2, although raloxifene has a slightly lower binding energy to SOD2 (-6.4 kcal/mol), it has a higher binding energy to PDGFRβ (-8.1 kcal/mol). Conclusion The present study identified and discovered pericyte subpopulations that were significantly associated with prognosis, which provides new biomarkers for predicting patient prognosis and adds usable targets for immunotherapy, and it is also important for gaining insights into the composition of the TME in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e295485d04626d24ad4217e848a133499f9df967" target='_blank'>
              Single-cell sequencing analysis reveals cancer-associated pericyte subgroup in esophageal squamous cell carcinoma to predict prognosis
              </a>
            </td>
          <td>
            Kai Xiong, Bing Pan, Hao Fang, Ziyou Tao
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Studying the functional consequences of structural variants (SVs) in mammalian genomes is challenging because (i) SVs arise much less commonly than single-nucleotide variants or small indels and (ii) methods to generate, map, and characterize SVs in model systems are underdeveloped. To address these challenges, we developed Genome-Shuffle-seq, a method that enables the multiplex generation and mapping of thousands of SVs (deletions, inversions, translocations, and extrachromosomal circles) throughout mammalian genomes. We also demonstrate the co-capture of SV identity with single-cell transcriptomes, facilitating the measurement of SV impact on gene expression. We anticipate that Genome-Shuffle-seq will be broadly useful for the systematic exploration of the functional consequences of SVs on gene expression, the chromatin landscape, and three-dimensional nuclear architecture, while also initiating a path toward a minimal mammalian genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd57d0a97983513ebb6d083a8255399d3304d8d" target='_blank'>
              Multiplex generation and single-cell analysis of structural variants in mammalian genomes.
              </a>
            </td>
          <td>
            Sudarshan Pinglay, Jean-Benoît Lalanne, R. Daza, Sanjay Kottapalli, Faaiz Quaisar, Jonas Koeppel, Riddhiman K. Garge, Xiaoyi Li, David S Lee, Jay Shendure
          </td>
          <td>2025-01-31</td>
          <td>Science</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="
 Melanoma is the 8th most commonly diagnosed cancer in the United States, predominantly driven by exposure to UV radiation leading to an abundant oncogenic mutational burden. However, very few models incorporate these mechanisms in their representation of the disease significantly influencing treatment responses. To address this, our lab uses the Yale University Melanoma Model (YUMM) irradiated cell line that contains clinically represented mutations that are imperative to melanoma development. As a result, these cells are sensitive to combination immune checkpoint inhibition making them a superior system to study the immune response in melanoma over less representative lines such as B16. These cells are injected into a syngeneic in vivo murine system C57/Bl6 INK-ATTAC mice with a senescent cell depletion cassette exclusively expressed in the host stroma, incorporating cellular senescence as a key contributor to the host response to cancer. This dual system allows for the exploration of the immune response that patients experience after receiving ICI treatment in concert with senescent stromal cell burden in the tumor microenvironment (TME). INK-ATTAC mice bearing single cell clone YUMM tumors are treated with a combination of immune checkpoint inhibitors (aPD-1+aCTLA-4) to generate both responder and non-responder phenotypes. Upon clearance of senescent cells from the TME via injection of AP20187, the proportions of these response phenotypes shift in both pro- and anti-tumor directions. Immunohistochemical analyses show a significant negative correlation of immune infiltration with tumor size, where T cells show a 2-fold increase as tumor size is reduced as a result of ICI and AP+ICI treatment. More in-depth analysis of tumor infiltrating lymphocytes and myeloid cells through Cytometry Time of Flight (CyTOF) of a specific single cell clone reveal that the proportions of immunosuppressive myeloid cells to cytotoxic T cells are key drivers of response phenotypes in AP+ICI treated tumors. Our findings suggest that stromal cellular senescence potentially have pro- and anti-tumor effects, thus influencing both treatment efficacy and resistance mechanisms that prevent response to immune checkpoint inhibition.
 Citation Format: Giselle Burton Sojo, Marcel O Schmidt. Stromal senescence and response to immune checkpoint inhibition in melanoma [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B047.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0608cf84711b7461bf4c33725871d3360fc3c9a5" target='_blank'>
              Abstract B047: Stromal senescence and response to immune checkpoint inhibition in melanoma
              </a>
            </td>
          <td>
            Giselle Burton Sojo, Marcel O Schmidt
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Afatinib-induced tumor and microenvironment modifications in head and neck squamous cell carcinoma were evaluated by spatial transcriptomics in surgical specimens and RNA-sequencing in tumor biopsies of patients included in the EORTC-90111-24111 window-of-opportunity study. The aim was to explore tumor evolution and composition under anti-HER therapy. Based on our previous investigations by RNA-seq on tumor biopsies, surgical slides of ID08 and ID15 from the epithelial-to-mesenchymal (EMT) cluster and ID30 from the non-EMT cluster were investigated with spatial transcriptomics. Dimension reduction in ID30 revealed 14 clusters, with clusters overlapping three tumor nodules and the stroma. Differential expression analysis between tumor nodules showed enrichment of the hallmark EMT genelist, with 123 genes in common between the analyses. These genes were involved in PDGF and MET signaling pathways. By comparing gene expression in paired tumor biopsies and the 123 genes from differential analyses obtained in ID30, a list of 13 genes involved in cancer pathways and EMT emerged, which were also highly expressed in ID08 and ID15. These results show a progressive apparition of genes implicated in EMT, MET, and PDGF pathways in tumors after afatinib. Notably, a list of 13 genes emerged which may contain targets to prevent tumor evolution after anti-HER therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ce3fcfb23899394262473ee209f0d7889b6808e" target='_blank'>
              Spatial Transcriptomic Analysis of Surgical Resection Specimens of Primary Head and Neck Squamous Cell Carcinoma Treated with Afatinib in a Window-of-Opportunity Study (EORTC90111-24111)
              </a>
            </td>
          <td>
            S. Beyaert, A. Loriot, J. Machiels, S. Schmitz
          </td>
          <td>2025-02-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Abstract A cell fate change such as tumorigenesis incurs critical transition. It remains a longstanding challenge whether the underlying mechanism can be unraveled and a molecular switch that can reverse such transition is found. Here a systems framework, REVERT, is presented with which can reconstruct the core molecular regulatory network model and a reversion switch based on single‐cell transcriptome data over the transition process is identified. The usefulness of REVERT is demonstrated by applying it to single‐cell transcriptome of patient‐derived matched organoids of colon cancer and normal colon. REVERT is a generic framework that can be applied to investigate various cell fate transition phenomena.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbd1a9318f2abd76a8f7539133f45f2810ab7c74" target='_blank'>
              Attractor Landscape Analysis Reveals a Reversion Switch in the Transition of Colorectal Tumorigenesis
              </a>
            </td>
          <td>
            Dongkwan Shin, Jeong-Ryeol Gong, Seoyoon D Jeong, Youngwon Cho, Hwang‐Phill Kim, Tae-You Kim, Kwang-Hyun Cho
          </td>
          <td>2025-01-22</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
Tumor hypoxia reshapes the microenvironment, driving progression, invasion, metastasis, and therapy resistance. Patient-derived tumor organoids, formed under three-dimensional conditions, preserve cellular heterogeneity and nutrient gradients, making them ideal for studying hypoxia-induced tumor responses. This study examines hypoxia-induced changes in lung tumor organoids from two patients, focusing on tumor-associated markers, stem cell markers, and migration capabilities.


RESULTS
Our findings demonstrate that hypoxia distinctively modulates the expression of lung cancer markers thyroid transcription factor-1, cytokeratin 7, and ΔNP63 variant. Hypoxia also induces the upregulation of stem cell-associated markers, resulting in an increased proportion of cancer stem cells. Furthermore, hypoxic lung tumor organoids exhibit unique migratory behavior upon reoxygenation, driven by epithelial-mesenchymal transition and the elevated expression of matrix metalloproteinases 7 and matrix metalloproteinases 9, indicating their enhanced invasive potential.


CONCLUSIONS
These findings highlight the value of lung tumor organoids as models for studying hypoxia's complex role in lung cancer. Hypoxia significantly modulates lung tumor organoids growth, stemness, and migratory behavior, providing critical insights into tumor progression and therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c10484a1b1c15548c942fa338e1afd0e25fd59" target='_blank'>
              Lung tumor organoids migrate as cell clusters containing cancer stem cells under hypoxic condition.
              </a>
            </td>
          <td>
            Yanjiao Li, Jiarong Zou, Yanhua Fang, Jianing Zuo, Ruoyu Wang, Shanshan Liang
          </td>
          <td>2025-02-01</td>
          <td>Biology of the cell</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Somatic activating mutations in KRAS can cause complex lymphatic anomalies (CLAs). However, the specific processes that drive KRAS-mediated CLAs have yet to be fully elucidated. Here, we used single-cell RNA sequencing to construct an atlas of normal and KrasG12D-malformed lymphatic vessels. We identified six subtypes of lymphatic endothelial cells (LECs) in the lungs of adult wild-type mice (Ptx3, capillary, collecting, valve, mixed, and proliferating). To determine when the LEC subtypes were specified during development, we integrated our data with data from four stages of development. We found that proliferating and Ptx3 LECs were prevalent during early lymphatic development and that collecting and valve LECs emerged later in development. Additionally, we discovered that the proportion of Ptx3 LECs decreased as the lymphatic network matured but remained high in KrasG12D mice. We also observed that the proportion of collecting and valve LECs was lower in KrasG12D mice than in wild-type mice. Last, we found that immature lymphatic vessels in young mice were more sensitive to the pathologic effects of KrasG12D than mature lymphatic vessels in older mice. Together, our results expand the current model for the development of the lymphatic system and suggest that KRAS mutations impair the maturation of lymphatic vessels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b83680aa397127e25da0c4fb9fc494573ba691b" target='_blank'>
              A single-cell atlas of normal and KRASG12D-malformed lymphatic vessels.
              </a>
            </td>
          <td>
            Lorenzo M. Fernandes, Danielle Griswold-Wheeler, Jeffrey D Tresemer, Angelica Vallejo, Neda Vishlaghi, Benjamin Levi, Abigail Shapiro, Joshua P. Scallan, Michael T. Dellinger
          </td>
          <td>2025-01-28</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8d7052c816a7546a33fc12f61bcaaa555f6b028" target='_blank'>
              Integrated analysis of single-cell and bulk transcriptomics reveals cellular subtypes and molecular features associated with osteosarcoma prognosis
              </a>
            </td>
          <td>
            Feng Liu, Tingting Zhang, Yongqiang Yang, Kailun Wang, Jinlan Wei, Ji-Hua Shi, Dong Zhang, Xia Sheng, Yi Zhang, Jing Zhou, Faming Zhao
          </td>
          <td>2025-02-17</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Prevention of metastases is critical for the future of breast cancer research, as metastatic cancer is extremely difficult to manage and treat. Solid tumors often contain patches of hypoxic regions that yield aggressive, pro-metastatic phenotypes. Recent work in our lab has shown that long-term hypoxia suppresses type I interferon (IFN) signaling in breast cancer cells, and this suppression of IFN signaling is maintained even after reoxygenation, indicative of “hypoxic memory.” Breast cancer cells and circulating tumor cells (CTCs) that have exited hypoxia but maintain such hypoxic memory showed enhanced metastatic potential. In addition to tumor intrinsic effects, IFN suppression can dampen surrounding immune cell populations within the tumor microenvironment, primarily through interferon-stimulated genes (ISGs) which carry out immune functions through activation of various immune cell populations. More research is needed to understand how ISGs are downregulated in hypoxia and how those changes in gene expression contribute to immunosuppressive ability of hypoxic and post-hypoxic cells. Interferon-inducible 44-like (IFI44L) is an ISG that has been associated with tumor-infiltrating lymphocytes and enhanced metastasis in other cancers, but its role in breast cancer has not been studied. We found that IFI44L is significantly suppressed in breast cancer cells in hypoxia and maintained after reoxygenation as hypoxic memory. Overexpressing interferon regulatory factors (IRFs), which are positive regulators of the type I IFN pathway, did not rescue hypoxic suppression and memory of IFI44L, suggesting IRFs are not the sole regulator of IFN and ISG expression in hypoxia. Current experiments in the lab are ongoing to address the mechanisms responsible for hypoxic memory of IFI44L suppression. Additionally, co-culture studies of IFI44L knockdown and overexpression breast cancer cell lines with different immune populations will shed light on the role of hypoxic suppression of IFI44L in immune evasion. To better understand implications of maintained IFI44L suppression, we will also examine immune cell activity and metastatic potential using in vivo mouse studies. We expect to see less immune cell effect and enhanced metastasis with IFI44L knockdown and reversed phenotypes with overexpression of IFI44L. Understanding mechanisms of immune suppression and evasion mediated via IFI44L can lead to strategies for targeting hypoxia-mediated tumorigenesis and metastasis.
 Citation Format: Rebecca Marker, Aidan Moriarty, Remi Klotz, Min Yu. Mechanisms mediating hypoxic memory of type I IFN signaling suppression and implications for immune evasion in luminal breast cancer cells [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A062.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c96757ce9569828ea0b78de759efc296d25cef" target='_blank'>
              Abstract A062: Mechanisms mediating hypoxic memory of type I IFN signaling suppression and implications for immune evasion in luminal breast cancer cells
              </a>
            </td>
          <td>
            Rebecca Marker, Aidan Moriarty, Remi Klotz, Min Yu
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Effective medical therapies for metastatic paraganglioma (mPPGL) are currently lacking, leading to dismal prognosis. Building on our knowledge of molecular mechanisms driving PPGL progression, we assessed the therapeutic potential of targeting two critical processes: PI3K signaling and cell cycle regulation. The efficacy of buparlisib (PI3Ki) and ribociclib (CDK4/6i), individually and combined, was assessed in vitro using PPGL cell lines, rat– and patient-derived primary cells, and in vivo using PPGL cells-derived mouse xenografts. The combination therapy demonstrated superior antitumor activity compared to single agents, particularly in vivo. Mechanistically, the efficacy of the combination therapy was associated to the downregulation of FOXM1-controlled genes implicated in mitotic spindle assembly and chromosomal segregation, leading to mitotic catastrophe. Data mining and qRT-PCR showed this genetic signature to be upregulated in human mPPGLs. This suggests that aggressive PPGLs exhibit heightened vulnerability to dual PI3K and CDK4/6 inhibition, offering a promising therapeutic avenue for these challenging cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/312876965c04c1de5ff4c1cc2f5814c15a87434b" target='_blank'>
              A genetic signature predicts aggressive paraganglioma sensitivity to dual PI3K-CDK4/6 inhibition therapy
              </a>
            </td>
          <td>
            Bhargavi Karna, Federica Gentile, Sebastian Gulde, Daniela De Martino, Julian Maurer, Katyayni Ganesan, Katharina Wang, Nicole Bechmann, E. Rapizzi, Mercedes Robledo, Ester Arroba, Stephan Herzig, S. Nölting, Hermine Mohr, Natalia S Pellegata
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains a formidable challenge in neuro-oncology, characterized by heterogeneity and aggressive tumor growth. The identification of novel biomarkers and therapeutic targets is crucial for advancing promising therapies. This study profiled patient-derived tumor samples utilizing single-cell RNA sequencing and proteomics platforms to uncover the upregulation of glycoprotein non-metastatic melanoma protein B (GPNMB) in tumor cells, particularly in post-treatment recurrent tumor cells, as well as in tumor-associated macrophages (TAMs), which constitute a major population of immune cells in treatment-refractory recurrent GBM. By exploring its expression patterns against normal tissue specimens and utilizing a series of patient-derived xenograft and humanized mouse models, we explored GPNMB as a target for Chimeric Antigen Receptor T cells (CAR-Ts) as a monotherapy and in rationally designed combination treatment regimens. We also demonstrated the utility of the GPNMB CAR-T cell in eliminating TAMs, a major contributor to immunosuppression in solid tumors. Our study reveals the development of GPNMB-targeting therapies as part of a promising and effective combinatorial treatment strategy for GBM.
 Citation Format: Sheila K Singh, Neil Savage, Franz J Zemp, Vaseem Shaikh, Shan Grewal, Nicholas Mikolajewicz, Hinda Najem, Chitra Venugopal, Thomas Kislinger, Amy Heimberger, Douglas Mahoney, Jason Moffat. GPNMB CAR T cells target both Glioblastoma and its immunosuppressive niche to relieve immune evasion [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A015.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a27b18e366d22bf8a8e6f5808c8d789666dea57" target='_blank'>
              Abstract A015: GPNMB CAR T cells target both Glioblastoma and its immunosuppressive niche to relieve immune evasion
              </a>
            </td>
          <td>
            Sheila K Singh, N. Savage, F. Zemp, Vaseem Shaikh, Shan Grewal, N. Mikolajewicz, Hinda Najem, C. Venugopal, T. Kislinger, Amy B. Heimberger, Douglas Mahoney, Jason Moffat
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Spatial transcriptomics (ST) offers enormous potential to decipher the biological and pathological heterogeneity in precious archival cancer tissues. Traditionally, these tissues have rarely been used and only examined at a low throughput, most commonly by histopathological staining. ST adds thousands of times as many molecular features to histopathological images, but critical technical issues and limitations require more assessment of how ST performs on fixed archival tissues. In this work, we addressed this in a cancer‐heterogeneity pipeline, starting with an exploration of the whole transcriptome by two sequencing‐based ST protocols capable of measuring coding and non‐coding RNAs. We optimised the two protocols to work with challenging formalin‐fixed paraffin‐embedded (FFPE) tissues, derived from skin. We then assessed alternative imaging methods, including multiplex RNAScope single‐molecule imaging and multiplex protein imaging (CODEX). We evaluated the methods’ performance for tissues stored from 4 to 14 years ago, covering a range of RNA qualities, allowing us to assess variation. In addition to technical performance metrics, we determined the ability of these methods to quantify tumour heterogeneity. We integrated gene expression profiles with pathological information, charting a new molecular landscape on the pathologically defined tissue regions. Together, this work provides important and comprehensive experimental technical perspectives to consider the applications of ST in deciphering the cancer heterogeneity in archived tissues. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0ac2d361f4c48551940640dc92bcb661d02f42d" target='_blank'>
              Assessing spatial sequencing and imaging approaches to capture the molecular and pathological heterogeneity of archived cancer tissues
              </a>
            </td>
          <td>
            Tuan Vo, P. Prakrithi, Kahli Jones, Sohye Yoon, P. Y. Lam, Y. Kao, Ning Ma, Samuel X. Tan, Xinnan Jin, Chenhao Zhou, Joanna Crawford, S. Walters, Ishaan Gupta, Peter H Soyer, K. Khosrotehrani, M. Stark, Quan H Nguyen
          </td>
          <td>2025-01-23</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) presents significant clinical challenges due to its limited treatment options and aggressive behavior, often associated with poor prognosis. This study focuses on Kindlin-2, an adaptor protein, and its role in TNBC progression, particularly in hematopoiesis-mediated immune evasion. TNBC tumors expressing high levels of Kindlin-2 induce a notable reshaping of hematopoiesis, promoting expansion of myeloid cells in bone marrow (BM) and spleen. This shift correlated with increased levels of neutrophils and monocytes in tumor-bearing mice over time. Conversely, genetic knockout of Kindlin-2 mitigated this myeloid bias and fostered T cell infiltration within the tumor microenvironment, indicating Kindlin-2's pivotal role in immune modulation. Further investigations revealed that Kindlin-2 deficiency led to reduced expression of PD-L1, a critical immune checkpoint inhibitor, in TNBC tumors. This molecular change sensitized Kindlin-2-deficient tumors to host anti-tumor immune responses, resulting in enhanced tumor suppression in immune-competent mouse models. Single-cell RNA sequencing, bulk RNA-seq, and immunohistochemistry data supported these findings by highlighting enriched immune-related pathways and increased infiltration of immune cells in Kindlin-2-deficient tumors. Therapeutically, targeting PD-L1 in Kindlin-2-expressing TNBC tumors effectively inhibited tumor growth, akin to the effects observed with genetic Kindlin-2 knockout or PD-L1-KO. Our data underscore Kindlin-2 as a promising therapeutic target in combination with immune checkpoint blockade to bolster anti-tumor immunity and counteract resistance mechanisms typical of TNBC and other immune evasive solid tumors. Implications: Kindlin-2 regulates tumor immune evasion through the systemic modulation of hematopoiesis and PD-L1 expression, which warrants therapeutic targeting of Kindlin-2 in TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37ece9caa74b6cb63185e70bd155db52308de96e" target='_blank'>
              Kindlin-2-mediated hematopoiesis remodeling regulates triple-negative breast cancer immune evasion.
              </a>
            </td>
          <td>
            Wei Wang, Rahul Chaudhary, Justin Szpendyk, Lamyae El Khalki, Neelum Aziz Yousafzai, Ricky Chan, Amar B. Desai, Khalid Sossey-Alaoui
          </td>
          <td>2025-02-07</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8ecf8ff6118aceaa2b844571144d55332e29e96" target='_blank'>
              Tumor microenvironment and drug resistance in lung adenocarcinoma: molecular mechanisms, prognostic implications, and therapeutic strategies
              </a>
            </td>
          <td>
            Qian Cao, Chenxuan Li, Ying Li, Xiangjing Kong, Shoushi Wang, Jun Ma
          </td>
          <td>2025-02-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Resistance to genotoxic therapies and tumor recurrence are hallmarks of glioblastoma (GBM), an aggressive brain tumor. In this study, we investigated functional drivers of post-treatment recurrent GBM through integrative genomic analyses, genome-wide genetic perturbation screens in patient-derived GBM models and independent lines of validation. Specific genetic dependencies were found consistent across recurrent tumor models, accompanied by increased mutational burden and differential transcript and protein expression compared to its primary GBM predecessor. Our observations suggest a multi-layered genetic response to drive tumor recurrence and implicate PTP4A2 (protein tyrosine phosphatase 4A2) as a modulator of self-renewal, proliferation and tumorigenicity in recurrent GBM. Genetic perturbation or small-molecule inhibition of PTP4A2 acts through a dephosphorylation axis with roundabout guidance receptor 1 (ROBO1) and its downstream molecular players, exploiting a functional dependency on ROBO signaling. Because a pan-PTP4A inhibitor was limited by poor penetrance across the blood–brain barrier in vivo, we engineered a second-generation chimeric antigen receptor (CAR) T cell therapy against ROBO1, a cell surface receptor enriched across recurrent GBM specimens. A single dose of ROBO1-targeted CAR T cells doubled median survival in cell-line-derived xenograft (CDX) models of recurrent GBM. Moreover, in CDX models of adult lung-to-brain metastases and pediatric relapsed medulloblastoma, ROBO1 CAR T cells eradicated tumors in 50–100% of mice. Our study identifies a promising multi-targetable PTP4A–ROBO1 signaling axis that drives tumorigenicity in recurrent GBM, with potential in other malignant brain tumors.
 Citation Format: Sheila K Singh, Chirayu R Chokshi, Vaseem Shaikh, Benjamin Brakel, Martin Rossotti, John Lazo, Chitra Venugopal, Thomas Kislinger, Kevin Henry, Jason Moffat. Targeting axonal guidance dependencies in glioblastoma with ROBO1 CAR T cells [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92c7df772c96f54556294e7c86632af2e8315a41" target='_blank'>
              Abstract A016: Targeting axonal guidance dependencies in glioblastoma with ROBO1 CAR T cells
              </a>
            </td>
          <td>
            Sheila K Singh, C. Chokshi, Vaseem Shaikh, B. Brakel, Martin A. Rossotti, John Lazo, C. Venugopal, T. Kislinger, Kevin A Henry, Jason Moffat
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Disseminated tumor cells (DTCs) contribute to metastasis formation by adapting to the hostile environmental conditions encountered at the seeded secondary organs. This phenotypic plasticity is supported by the epigenetic reprogramming of cis-regulatory elements (CREs), representing a hallmark of metastatic cells. DTCs can enter a reversible state of cancer dormancy, resulting from their ability to enter quiescence after mitosis and escape immune surveillance. In this respect, mitosis may represent a window of opportunity to increase DTCs’ fitness by priming CREs, ensuring the reactivation of a dormancy-related transcriptional program while preserving the epigenetic information. By profiling the chromatin changes occurring during the early steps of mitosis, we found that DTCs experience fluctuations in chromatin accessibility at CREs, which are bookmarked by pro-metastatic TFs. Among these, the pro-dormancy TF SOX9 remained associated with mitotic chromatin, facilitating transcriptional reactivation upon exiting mitosis. By dissecting the consequences of degrading SOX9 before mitosis, we found that its priming of dormancy-related CREs, combined with the recruitment of the Mediator kinase module, is instrumental for metastatic cells to enter quiescence and escape NK-mediated immune surveillance. These findings indicate that cancer dormancy is primed during mitosis by pro-metastatic TFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The development of treatment resistance remains universal for patients with metastatic prostate cancer, driven by AR alterations and lineage state transitions. Identifying the evolution of lineage transitions in treatment resistance has been limited by the challenges of collecting serial tissue biopsies on treatment, which can be overcome using blood-based liquid biopsies. Utilizing a novel circulating tumor cell (CTC) isolation approach, we collected 273 CTC samples from 117 patients with metastatic prostate cancer for RNA sequencing. 146 samples from 70 patients had tumor purity comparable to tissue biopsies. We identified four CTC transcriptional phenotypes, mirroring lineage states identified in tissue. Patients with a luminal-B-like CTC phenotype defined by persistent AR signaling and high proliferation, as well as those with a neuroendocrine CTC phenotype, had significantly shorter survival than patients with luminal-A-like and low proliferation phenotypes. In a prospective substudy, pre-treatment CTC luminal-B-like phenotype was associated with early progression on 177Lu-PSMA-617.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166ffe8ca69e378ce5cd3ea06cb57373f1b4e28e" target='_blank'>
              High-purity CTC RNA sequencing identifies prostate cancer lineage phenotypes prognostic for clinical outcomes.
              </a>
            </td>
          <td>
            Marina N. Sharifi, J. Sperger, Amy K Taylor, Katharine E Tippins, Shannon R Reese, Viridiana Carreno, K. Kaufmann, Alex H Chang, Luke A Nunamaker, Charlotte Linebarger, Leilani Mora-Rodriguez, J. Schehr, H. Krause, K. Helzer, M. Bootsma, G. Blitzer, John M Floberg, C. Kyriakopoulos, H. Emamekhoo, Elizabeth I. Heath, Meghan Wells, S. Tagawa, M. Sjöström, A. Choudhury, Menggang Yu, Andrew J. Armstrong, D. Rathkopf, H. Beltran, Peter S Nelson, Felix Y. Feng, S. Dehm, D. Kosoff, Xiao X Wei, Rana R Mckay, Shuang G. Zhao, Joshua M Lang
          </td>
          <td>2025-02-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Changes in cell type composition play an important role in human health and disease. Recent advances in single-cell technology have enabled the measurement of cell type composition at increasing cell lineage resolution across large cohorts of individuals. Yet this raises new challenges for statistical analysis of these compositional data to identify changes in cell type frequency. We introduce crumblr (DiseaseNeurogenomics.github.io/crumblr), a scalable statistical method for analyzing count ratio data using precision-weighted linear mixed models incorporating random effects for complex study designs. Uniquely, crumblr performs statistical testing at multiple levels of the cell lineage hierarchy using a multivariate approach to increase power over tests of one cell type. In simulations, crumblr increases power compared to existing methods while controlling the false positive rate. We demonstrate the application of crumblr to published single-cell RNA-seq datasets for aging, tuberculosis infection in T cells, bone metastases from prostate cancer, and SARS-CoV-2 infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4cb0d543e456b8e0b6125dd4a4f449c19fe2812" target='_blank'>
              Fast, flexible analysis of differences in cellular composition with crumblr
              </a>
            </td>
          <td>
            Gabriel E. Hoffman, P. Roussos
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffc5afe786b47a61ca4c6c86b698a4b6f0aa1df3" target='_blank'>
              Characterizing tertiary lymphoid structures associated single-cell atlas in breast cancer patients
              </a>
            </td>
          <td>
            Xiaokai Fan, Daqin Feng, Donggui Wei, Anqi Li, Fangyi Wei, Shufang Deng, Muling Shen, Congzhi Qin, Yongjia Yu, Lun Liang
          </td>
          <td>2025-01-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bc6fce345e0628d21c6c9477e460943646693b" target='_blank'>
              Spatial deconvolution from bulk DNA methylation profiles determines intratumoral epigenetic heterogeneity
              </a>
            </td>
          <td>
            Binbin Liu, Yumo Xie, Yu Zhang, Guannan Tang, Jinxin Lin, Ze Yuan, Xiaoxia Liu, Xiaolin Wang, Meijin Huang, Yanxin Luo, Huichuan Yu
          </td>
          <td>2025-01-23</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) thrives in an ecosystem that is shaped by intricate intercellular interactions that contribute to tumor progression and treatment resistance. Recent studies have shown that the oligodendrocyte lineage (OL) is abundantly enriched in the GBM tumor microenvironment. In this study, we utilized single-cell RNA sequencing (scRNA-seq) profiles from 120 primary and recurrent GBM tumors, spatial transcriptomics, immunohistochemistry, cytokine profiling, and in vitro migration assays to demonstrate that OLs are selectively recruited to the GBM tumor border, and subsequently incorporated into the tumor mass at recurrence. Using a pan-disease single-cell human OL meta-atlas, we identified a population of malignancy-associated reactive OLs, akin to those seen in demyelinating inflammatory and traumatic conditions. The interferon-mediated induction of this population was validated using in vivo GL261 and CT2A syngeneic mouse models. Cytokine profiling and ligand-receptor (LR) interaction analyses identified several pro-tumorigenic cytokines secreted by reactive OLs, including CCL5. Moreover, we found that CCR5, the CCL5 receptor, was preferentially expressed in glioma stem-like cells (GSCs) and upregulated at recurrence. Targeting CCR5 in GBM tumor cells with genetic knockdown or pharmacological inhibition impaired GSC stemness and prolonged the survival of mice bearing GBM tumors. Our work highlights the functional interplay between OLs and GBM cells and positions the CCL5/CCR5 axis as a druggable target for GBM.
 Citation Format: Jason Moffat. Reactive oligodendrocytes promote glioblastoma progression through CCL5/CCR5-mediated glioma stem cell maintenance [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A060.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a751c087d4afc3ab315830aaec4e9b6e54e60f3" target='_blank'>
              Abstract A060: Reactive oligodendrocytes promote glioblastoma progression through CCL5/CCR5-mediated glioma stem cell maintenance
              </a>
            </td>
          <td>
            Jason Moffat
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0871f3b2bbb54a6ef4ad184b9e9c10fad8f88b51" target='_blank'>
              Spatial single-cell maps reveal ST6GAL1 promoting ovarian cancer metastasis.
              </a>
            </td>
          <td>
            Lan-hui Qin, Zijian Jiang, Chongze Yang, Rui Song, Pei-Yin Chen, Weihui Xu, Guanzhen Zeng, Jin-yuan Liao, Liling Long
          </td>
          <td>2025-01-30</td>
          <td>Glycoconjugate journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7635dc0474dacb31cc3822f8f3055dfb7b7481" target='_blank'>
              Integrating transcriptomics and scPagwas analysis predicts naïve CD4 T cell-related gene DRAM2 as a potential biomarker and therapeutic target for colorectal cancer
              </a>
            </td>
          <td>
            Rui Feng, Xiaofang Li, Benhua Li, Tiankuo Luan, Jiaming He, Guojing Liu, Jian Yue
          </td>
          <td>2025-02-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8124e4116b706bf406200aaca77b811cd619c61b" target='_blank'>
              Metabolically activated and highly polyfunctional intratumoral VISTA+ regulatory B cells are associated with tumor recurrence in early-stage NSCLC
              </a>
            </td>
          <td>
            D. Lo Tartaro, Beatrice Aramini, V. Masciale, Nikolaos Paschalidis, F. Lofaro, Anita Neroni, R. Borella, Elena Santacroce, Alin Liviu Ciobanu, A. V. Samarelli, Federica Boraldi, Daniela Quaglino, Alessandra Dubini, Michele Gaudio, G. Manzotti, F. Reggiani, Federica Torricelli, A. Ciarrocchi, A. Neri, Federica Bertolini, Massimo Dominici, Pier Luigi Filosso, F. Stella, L. Gibellini, S. De Biasi, Andrea Cossarizza
          </td>
          <td>2025-01-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659f72bed8c9bc066ac845266c3179b755922c46" target='_blank'>
              Epithelial cell diversity and immune remodeling in bladder cancer progression: insights from single-cell transcriptomics
              </a>
            </td>
          <td>
            Jianpeng Li, Yunzhong Jiang, Minghai Ma, Lu Wang, Minxuan Jing, Zezhong Yang, Mengzhao Zhang, Ke Chen, Jinhai Fan
          </td>
          <td>2025-01-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Newly diagnosed prostate cancers differ dramatically in mutational composition and lethality. The most accurate clinical predictor of lethality is tumor tissue architecture, quantified as tumor grade. To interrogate the evolutionary origins of prostate cancer heterogeneity, we analyzed 666 prostate tumor whole genomes. We identified a compendium of 223 recurrently mutated driver regions, most influencing downstream mutational processes and gene expression. We identified and validated individual germline variants that predispose tumors to acquire specific somatic driver mutations: these explain heterogeneity in disease presentation and ancestry differences. High-grade tumors have a superset of the drivers in lower-grade tumors, including increased frequency of BRCA2 and MYC mutations. Grade-associated driver mutations occur early in tumor evolution, and their earlier occurrence strongly predicts cancer relapse and metastasis. Our data suggest high- and low-grade prostate tumors both emerge from a common pre-malignant field, influenced by germline genomic context and stochastic mutation-timing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9c23cc7532d469971ff1994cdcbf780a4650cf6" target='_blank'>
              The Germline and Somatic Origins of Prostate Cancer Heterogeneity.
              </a>
            </td>
          <td>
            Takafumi N. Yamaguchi, Kathleen E. Houlahan, Helen Zhu, Natalie J Kurganovs, J. Livingstone, Natalie S. Fox, Jiapei Yuan, Jocelyn Sietsma Penington, Chol-Hee Jung, T. Schwarz, Weerachai Jaratlerdsiri, J. van Riet, P. Georgeson, S. Mangiola, Kodi Taraszka, R. Lesurf, Jue Jiang, K. Chow, Lawrence E Heisler, Yu-Jia Shiah, Susmita G Ramanand, Michael J Clarkson, Anne Nguyen, S. M. G. Espiritu, R. Stuchbery, Richard Jovelin, Vincent Huang, Connor Bell, Edward O'Connor, Patrick J. McCoy, Christopher M. Lalansingh, Marek Cmero, A. Salcedo, Eva K F Chan, Lydia Y. Liu, Phillip D Stricker, Vinayak Bhandari, R. Bornman, Dorota H. Sendorek, A. Lonie, S. Prokopec, M. Fraser, J. Peters, Adrien Foucal, S. Mutambirwa, Lachlan Mcintosh, M. Orain, Matthew J. Wakefield, V. Picard, Daniel J Park, H. Hovington, Michael Kerger, A. Bergeron, Veronica Y. Sabelnykova, J-H. Seo, M. Pomerantz, Noah Zaitlen, Sebastian M. Waszak, Alexander Gusev, L. Lacombe, Yves Fradet, Andrew Ryan, Amar U Kishan, M. Lolkema, Joachim Weischenfeldt, Bernard Têtu, Anthony J Costello, Vanessa Hayes, R. J. Hung, H. He, John D McPherson, Bogdan Pasaniuc, T. H. van der Kwast, AT Papenfuss, M. Freedman, B. Pope, Robert G Bristow, Ram-Shankar Mani, N. Corcoran, J. Reimand, Christopher M Hovens, P. Boutros
          </td>
          <td>2025-02-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f8ae5d4748b0e6bd25320bd0093825226598117" target='_blank'>
              Mapping the topography of spatial gene expression with interpretable deep learning.
              </a>
            </td>
          <td>
            Uthsav Chitra, Brian J. Arnold, Hirak Sarkar, Kohei Sanno, Cong Ma, Sereno Lopez-Darwin, Benjamin J. Raphael
          </td>
          <td>2025-01-23</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND Gastric cancer (GC) poses a substantial risk to human health due to its high prevalence and mortality rates. Nevertheless, current therapeutic strategies remain insufficient. Single-cell RNA sequencing (scRNA-seq) offers the potential to provide comprehensive insights into GC pathogenesis. AIM To explore the distribution and dynamic changes of cell populations in the GC tumor microenvironment using scRNA-seq techniques. METHODS Cancerous tissues and paracancerous tissues were obtained from patients diagnosed with GC at various stages (I, II, III, and IV). Single-cell suspensions were prepared and analyzed using scRNA-seq to examine transcriptome profiles and cell-cell interactions. Additionally, quantitative real-time polymerase chain reaction (qRT-PCR) and flow cytometry were applied for measuring the expression of cluster of differentiation (CD) 2, CD3D, CD3E, cytokeratin 19, cytokeratin 8, and epithelial cell adhesion molecules. RESULTS Transcriptome data from 73645 single cells across eight tissues of four patients were categorized into 25 distinct cell clusters, representing 10 different cell types. Variations were observed in these cell type distribution. The adjacent epithelial cells in stages II and III exhibited a degenerative trend. Additionally, the quantity of CD4 T cells and CD8 T cells were evidently elevated in cancerous tissues. Interaction analysis displayed a remarkable increase in interaction between B cells and other mast cells in stages II, III, and IV of GC. These findings were further validated through qRT-PCR and flow cytometry, demonstrating elevated T cells and declined epithelial cells within the cancerous tissues. CONCLUSION This study provides a comprehensive analysis of cell dynamics across GC stages, highlighting key interactions within the tumor microenvironment. These findings offer valuable insights for developing novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3adb4df70cb2b109a0b602e7b75fd956596d307f" target='_blank'>
              Landscape of four different stages of human gastric cancer revealed by single-cell sequencing
              </a>
            </td>
          <td>
            Xushan Tang, Chun-Lei Xu, Na Li, Jian-Qing Zhang, Yong Tang
          </td>
          <td>2025-02-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures.
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Rationale: Glioblastoma (GBM) is the most fatal form of brain cancer and its treatment represents a persistent challenge. Mesenchymal stromal cells (MSCs) have been explored as therapeutic tools in cancer management owing to their tumor-homing abilities. However, their clinical application is limited due to the controversial role of MSCs in carcinogenesis. This study investigates how MSCs influence tumor behavior and explores the synergistic anticancer effects in combination with melatonin (Mel). Methods: Orthotopic and subcutaneous GBM xenograft mouse models were used to assess the antitumor effect of Mel pre-treated MSCs (MSC Mel ). Histological, immunohistochemical, and ultrastructural analyses were conducted to identify phenotypic changes in tumors. Through a set of in vitro assays, including direct and indirect co-cultures, dynamic single-cell tracking and tumorsphere assay, we explored the impact of MSC Mel on primary and non-primary GBM cells. Transcriptomic profiling was used to identify genes and pathways modulated by this synergistic therapy. Results: MSC Mel delayed tumor growth in mice and increased collagen deposition. Additionally, MSC Mel showed enhanced capacity to prevent GBM cell migration compared to untreated MSCs. Molecular analysis identified genes and proteins related to cell migration, cytoskeletal dynamics and extracellular matrix remodeling in GBM cells exposed to MSC Mel , including reduced vimentin expression. Finally, a genetic signature associated with the clinical outcomes of GBM patients was identified. Conclusions: Our study demonstrates that melatonin enhances the anticancer properties of MSCs, providing new insights into their interaction with GBM cells and tumor environment. These findings offer valuable guidance for advancing MSC-based therapies in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f57dedd75349ecd50ac992a7b9ebfe735e08696" target='_blank'>
              Glioblastoma progression is hindered by melatonin-primed mesenchymal stromal cells through dynamic intracellular and extracellular reorganizations
              </a>
            </td>
          <td>
            Laura Olmedo-Moreno, Concepción Panadero-Morón, J. M. Sierra-Párraga, Rubén Bueno-Fernández, E. Norton, Y. Aguilera, N. Mellado-Damas, Patricia García-Tárraga, Raquel Morales-Gallel, María Antequera-Martínez, Raúl V. Durán, Jaime Ferrer-Lozano, Germaine Escames, J. García-Verdugo, Alejandro Martín-Montalvo, Hugo Guerrero-Cázares, V. Capilla-González
          </td>
          <td>2025-02-10</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38fe8b7dd282acd4da49fd77a60ef2b609353e0" target='_blank'>
              S100A13-driven interaction between pancreatic adenocarcinoma cells and cancer-associated fibroblasts promotes tumor progression through calcium signaling
              </a>
            </td>
          <td>
            Liuyuan Xia, Xin Guo, Dong Lu, Yixin Jiang, Xiaohui Liang, Yiwen Shen, Jia-Yi Lin, Lijun Zhang, Hongzhuan Chen, Jinmei Jin, Xin Luan, Weidong Zhang
          </td>
          <td>2025-01-28</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Unraveling the complex cell-type composition and gene expression patterns at the cellular spatial resolution is crucial for understanding intricate cell functions in the brain. In this study, we developed Deep Neural Network-based Spatial Cell Typing (DSCT), an innovative framework for spatial cell typing within spatial transcriptomic datasets. This approach utilizes a synergistic integration of an enhanced gene selection strategy and a lightweight deep neural network for data training, offering a more rapid and accurate solution for the analysis of spatial transcriptomic data. Based on comprehensive analysis, DSCT achieved exceptional accuracy in cell-type identification across various brain regions, species, and spatial transcriptomic platforms. It also performed well in mapping finer cell types, thereby showcasing its versatility and adaptability across diverse datasets. Strikingly, DSCT exhibited high efficiency and remarkable processing speed, with fewer computational resource demands. As such, this novel approach opens new avenues for exploring the spatial organization of cell types and gene expression patterns, advancing our understanding of biological functions and pathologies within the nervous system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99cf0a76120059ff7d91f5c179d0e2d602129b88" target='_blank'>
              DSCT: A novel deep learning framework for rapid and accurate spatial transcriptomic cell typing
              </a>
            </td>
          <td>
            Yiheng Xu, Bin Yu, Xuan Chen, Aibing Peng, Quyuan Tao, Youzhe He, Yueming Wang, Xiao-Ming Li
          </td>
          <td>2025-01-28</td>
          <td>National Science Review</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Understanding tumor heterogeneity is a major challenge that was recognized as one of the first Cancer Grand Challenges, with a call to provide solutions to visualize tumor heterogeneity. The Rosetta team took on this challenge, exploiting advances in spatial-omics approaches centered around mass spectrometry imaging to map tumor heterogeneity at the cellular and molecular scales with different levels of resolution. See related article by Bressan et al., p. 16 See related article by Stratton et al., p. 22 See related article by Bhattacharjee et al., p. 28.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d308601bb28236e43c3bc271c19d29c8dd8720ed" target='_blank'>
              Visualizing Cancer Heterogeneity at the Molecular and Cellular Levels: Lessons from Rosetta.
              </a>
            </td>
          <td>
            R. Goodwin, John F Marshall, G. Poulogiannis, Mariia Yuneva, K. Brindle, Z. Takáts, Owen J. Sansom, Josephine Bunch, Simon T Barry
          </td>
          <td>2025-01-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Hypoxia, a common feature in many malignancies, is particularly prominent in cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC). Investigating the mechanisms underlying hypoxia is essential for understanding the heterogeneity of CESC and developing personalized therapeutic regimens. Firstly, the CESC-specific hypoxia gene sets shared between single-cell RNA sequencing (scRNA-seq) and bulk data were identified through Weighted Gene Correlation Network Analysis (WGCNA)and FindMarkers analyses. A CESC-specific hypoxia-related score (CSHRS) risk model was constructed using the least absolute shrinkage and selection operator (LASSO)and Cox regression analyses based on these genes. The prognostic differences were analyzed in terms of immune infiltration, mutations, and drug resistance. Finally, a nomogram model was constructed by integrating clinicopathological features to facilitate precision treatment for CESC. This study constructed a CSHRS risk model that divides patients into two groups, and this model can comprehensively evaluate the tumor microenvironment characteristics of CESC, provide accurate prognostic predictions, and offer rational treatment options for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec7d7f2c99de4e5f9ef9acea8ebc376cd53f13e8" target='_blank'>
              Integrating Hypoxia Signatures from scRNA-seq and Bulk Transcriptomes for Prognosis Prediction and Precision Therapy in Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma
              </a>
            </td>
          <td>
            Kexin Yu, Shibo Zhang, Jiali Shen, Meini Yu, Yangguang Su, Ying Wang, Kun Zhou, Lei Liu, Xiujie Chen
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Metastasis remains a leading cause of cancer‐related deaths, defined by a complex, multi‐step process in which tumor cells spread and form secondary growths in distant tissues. Despite substantial progress in understanding metastasis, the molecular mechanisms driving this process and the development of effective therapies remain incompletely understood. Elucidating the molecular pathways governing metastasis is essential for the discovery of innovative therapeutic targets. The rapid advancements in sequencing technologies and the expansion of biological databases have significantly deepened our understanding of the molecular drivers of metastasis and associated drug resistance. This review focuses on the molecular drivers of metastasis, particularly the roles of genetic mutations, epigenetic changes, and post‐translational modifications in metastasis progression. We also examine how the tumor microenvironment influences metastatic behavior and explore emerging therapeutic strategies, including targeted therapies and immunotherapies. Finally, we discuss future research directions, stressing the importance of novel treatment approaches and personalized strategies to overcome metastasis and improve patient outcomes. By integrating contemporary insights into the molecular basis of metastasis and therapeutic innovation, this review provides a comprehensive framework to guide future research and clinical advancements in metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/647f5025b06ef5ed04146a75e8b99dcf5629913e" target='_blank'>
              Tumor Metastasis: Mechanistic Insights and Therapeutic Intervention
              </a>
            </td>
          <td>
            Lin Tang, Shao‐Cong Peng, Xiao‐Wan Zhuang, Yan He, Yu‐Xiang Song, Hao Nie, Can-Can Zheng, Zhen‐Yu Pan, Alfred King‐Yin Lam, Ming‐Liang He, Xingyuan Shi, Bin Li, W. Xu
          </td>
          <td>2025-02-17</td>
          <td>MedComm – Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a common malignant tumor of the digestive system with a high incidence that seriously threatens patients’ lives and health. However, with the rise and application of new treatments, such as immunotherapy, there are still some restrictions in the treatment and diagnosis of HCC, and the therapeutic effects on patients are not ideal. Methods Two single-cell RNA sequencing (scRNA-seq) datasets from HCC patients, encompassing 25,189 cells, were analyzed in the study. We utilized non-negative matrix factorization (NMF) clustering to identify mitophagy patterns in HCC TME cells, including cancer-associated fibroblasts (CAFs), T cells, B cells, and tumor-associated macrophages (TAMs). Cell-to-cell communication was analyzed using the CellChat package, and pseudotime trajectory analysis was performed using the Monocle package. Gene regulatory networks were investigated with the SCENIC package, and survival analyses were conducted with mitophagy-related signatures. Results HCC samples analysis identified 22 clusters, including 7 principal cell types. Complex cell communications were observed among these cell types. Mitophagy-related CAFs, TAMs, CD8+ T cells, and B cells were identified. These subtypes had different biological states, cell-cell communications, and metabolic pathways. Mitophagy levels were elevated in tumor samples. Changes in mitophagy-related genes within specific cell subtypes were associated with different overall survival rates. However, mitophagy did not seem to affect the effectiveness of immunotherapy. Conclusion This study provides evidence that mitophagy within the HCC TME modulates intercellular communication, influencing tumor progression and patient prognosis. Targeting mitophagy may offer a promising approach to improve the long-term prognosis of HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/689de3b101ffbbea61436b63812b3f0a30320588" target='_blank'>
              Single-cell mitophagy patterns within the tumor microenvironment modulate intercellular communication, impacting the progression and prognosis of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zhengyan Li, Wei Chen, Shu Yao, Zuxiang Peng, Hongming Liu, Yongliang Tang, Yi Feng
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/982b46ad0f67ebc704b6d142139e5b777f0090b1" target='_blank'>
              Exosomal lncRNA Mir100hg from lung cancer stem cells activates H3K14 lactylation to enhance metastatic activity in non-stem lung cancer cells
              </a>
            </td>
          <td>
            Lei Shi, Bowen Li, Jiyu Tan, Ling Zhu, Sicheng Zhang, Yuhan Zhang, Meng Xiang, Jie Li, Yan Chen, Xue Han, Jiacheng Xie, Yao Tang, H. Xing, Jingyu Li, Jianyu Wang, cnJingyu Li
          </td>
          <td>2025-02-28</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde0e9abf6b50983ec6394f87b77c188d773d609" target='_blank'>
              YAP-driven malignant reprogramming of oral epithelial stem cells at single cell resolution
              </a>
            </td>
          <td>
            F. Faraji, Sydney I Ramirez, Lauren M. Clubb, Kuniaki Sato, Valeria Burghi, Thomas S. Hoang, Adam Officer, Paola Y. Anguiano Quiroz, William M. G. Galloway, Zbigniew Mikulski, Kate Medetgul-Ernar, P. Marangoni, Kyle B Jones, Yuwei Cao, A. Molinolo, Kenneth Kim, Kanako Sakaguchi, Joseph A. Califano, Quinton Smith, Alon Goren, Ophir D. Klein, Pablo Tamayo, J. S. Gutkind
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eee1a8b871b1885a1b34200cb093a48900beddf" target='_blank'>
              The phenogenomic landscapes of pleural mesothelioma tumor microenvironment predict clinical outcomes
              </a>
            </td>
          <td>
            Federica Torricelli, Filomena Spada, Cynthia Bishop, K. Todd, Daisuke Nonaka, N. Petrov, Manuela Terranova Barberio, Alan G Ramsay, Richard Ellis, A. Ciarrocchi, B. Apollonio, Andrea Billè
          </td>
          <td>2025-02-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75bdacf15f78eabb80ad03155a6f7ee66350fd9" target='_blank'>
              Genomic alterations and transcriptional phenotypes in circulating free DNA and matched metastatic tumor
              </a>
            </td>
          <td>
            N. Takahashi, L. Pongor, Shivam P. Agrawal, Mariya Shtumpf, Ankita Gurjar, V. Rajapakse, Ahmad Shafiei, Christopher W. Schultz, Sehyun Kim, Diana Roame, Paula Carter, R. Vilimas, Samantha Nichols, Parth Desai, W. Figg, Mohammad Bagheri, V. B. Teif, Anish Thomas
          </td>
          <td>2025-02-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="CAR T cells therapies have demonstrated success in treating hematologic malignancies, but have proven less effective in eradicating solid tumors. While suppressive immune cells may contribute to reduced CAR T cell efficacies in malignant masses, cancer-associated fibroblasts (CAFs) are also believed to facilitate tumor survival by secreting growth factors, immunosuppressive cytokines, and extracellular matrix components that inhibit drug and immune cell filtration and facilitate metastasis. In an effort to eliminate both CAFs and cancer cells simultaneously, we have employed a universal CAR T cell that can attack both cell types when supplemented with appropriate bispecific adapters. We show here that tumor regression is indeed enhanced when CAR T cells are directed to concurrently kill both cancer cells and CAFs. We further demonstrate that simultaneous targeting of both cell types enhances CAR T cell proliferation, activation, tumor infiltration, and tumor distribution relative to targeting only a single cell type. Because all of these benefits are achieved in both cold and hot tumors without significant toxicity, we conclude that use of a universal CAR T cell in combination with multiple bispecific adapters can provide a safe, potent, cost-effective, and scalable alternative to the treatment of solid tumors with conventional CAR T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d409b2d0039c4bfb4e9ea46673be49bfaabe71cf" target='_blank'>
              Use of a universal targeting CAR T cell to simultaneously kill cancer cells and cancer-associated fibroblasts
              </a>
            </td>
          <td>
            Bo Huang, Suilan Zheng, Kasireddy Sudarshan, Ramesh Mukkamala, Madduri Srinivasarao, Tushar Sardesai, Xiaofei Yang, Haiyan Chu, Philip S Low
          </td>
          <td>2025-02-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) accumulate somatic mutations over time, some conferring a fitness advantage that can lead to clonal hematopoiesis (CH). Mutations in DNMT3A, particularly at hotspot R882, are the most prevalent in CH and carry an increased risk of acute myeloid leukemia (AML). Although DNMT3A R882 mutations are linked to global DNA hypomethylation, the mechanisms underlying their selective advantage remain unclear. Here, we show that Dnmt3a-R882H mutant HSCs exhibit resilience under inflammatory and genotoxic stress. During IL-1β-induced emergency granulopoiesis, Dnmt3a R882H/+ HSCs uncouple increased proliferation from stem cell exhaustion. In contrast, wild-type HSCs rapidly progress to terminal differentiation. We link this phenotype to a delayed activation of the p53-p21-DREAM axis, that allows mutant HSCs to avoid attrition, despite increased replication stress. Similarly, mutant HSCs exhibit delayed Trp53 activation following irradiation, but eventually recover a physiological Trp53 response. Analysis of patient data reveals shared phenotypic features between DNMT3A and monoallelic TP53 mutations in CH and myeloid neoplasms, highlighting potential functional similarities. Collectively, these findings suggest that the expansion of DNMT3A-mutant clones is affected by impaired TP53 signaling, which confers resilience against stressors. Therapeutic strategies targeting inflammatory pathways or the p53-p21-DREAM axis may reduce DNMT3A-CH expansion and/or progression and its associated risks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e56fa3aad088d2f09d1437903fb4b00ec728238c" target='_blank'>
              Delayed Trp53 activation protects Dnmt3a-mutant hematopoietic stem cells from inflammatory attrition
              </a>
            </td>
          <td>
            Ludovica Marando, G. Giotopoulos, Pedro Madrigal, Dhoyazan M A Azazi, Ryan Asby, M. E. Calleja-Cervantes, M. Gu, David Lara-Astiaso, Nicola K Wilson, R. Hannah, Jaana Bagri, Emily F. Calderbank, E. Laurenti, E. Bernard, Sarah J Horton, Berthold Göttgens, E. Papaemmanuil, Shuchi Agrawal-Singh, M. Gozdecka, G. Vassiliou, B. Huntly
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="The ESCRT (endosomal sorting complex required for transport) machinery is essential for various cellular processes, yet its role in head and neck squamous cell carcinoma (HNSCC) is poorly understood. We utilized The Cancer Genome Atlas (TCGA) datasets to analyze the expression of ESCRT genes. Bulk RNA-sequencing data and HNSCC tissue microarrays (TMAs) were used to evaluate VPS25 expression and its clinical significance. Single-cell RNA sequencing of tumor tissues and VPS25 knockdown experiments in CAL27 cells were used to investigate its biological functions. Immunohistochemistry, spatial transcriptomics, and immunotherapy datasets highlighted the involvement of VPS25 in immune suppression and its potential as a predictive biomarker. The results demonstrated significant VPS25 overexpression in HNSCC tissues, which correlated with poor clinical outcomes. It promoted tumor cell proliferation and migration while reducing immune cell infiltration in the tumor microenvironment (TME). Additionally, by upregulating PVR expression in tumor cells, VPS25 activated the immunosuppressive PVR-TIGIT signaling axis, thereby facilitating immune evasion. Furthermore, VPS25 emerged as a potential biomarker for predicting immunotherapy response. These findings highlight VPS25 as a pivotal regulator of tumor progression and immune evasion in HNSCC and a promising target for therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/902195b920715f84e3501a67211a1883f6d279dd" target='_blank'>
              VPS25 Promotes an Immunosuppressive Microenvironment in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Liguo Chen, Yu-Han Fang, Kui-Ming Wang, Wei Zhang, Gang Chen
          </td>
          <td>2025-02-22</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc019fa42801c8582b11a460b1609019f96845de" target='_blank'>
              A minimal gene set characterizes TIL specific for diverse tumor antigens across different cancer types
              </a>
            </td>
          <td>
            Zhen Zeng, Tianbei Zhang, Jiajia Zhang, Shuai Li, Sydney C Connor, Boyang Zhang, Yimin Zhao, Jordan Wilson, Dipika Singh, R. Kulikauskas, C. Church, T. Pulliam, Saumya Jani, P. Nghiem, S. Topalian, Patrick M Forde, Drew M Pardoll, Hongkai Ji, Kelli N. Smith
          </td>
          <td>2025-02-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90df80a01384ee300b159e30e515e05880476d0f" target='_blank'>
              Artificial intelligence-based spatial analysis of tertiary lymphoid structures and clinical significance for endometrial cancer
              </a>
            </td>
          <td>
            Haruka Suzuki, Kohei Hamada, J. Hamanishi, Akihiko Ueda, R. Murakami, Mana Taki, Rin Mizuno, Koichi Watanabe, Hanako Sato, Y. Hosoe, Hiroaki Ito, Koji Yamanoi, Hiroyuki Yoshitomi, N. Kakiuchi, Ken Yamaguchi, N. Matsumura, Seishi Ogawa, Hideki Ueno, Masaki Mandai
          </td>
          <td>2025-02-01</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fd1f4bb9cc6b18c1df47d2212202e677a571ba9" target='_blank'>
              Single cell and spatial analysis of immune-hot and immune-cold tumours identifies fibroblast subtypes associated with distinct immunological niches and positive immunotherapy response
              </a>
            </td>
          <td>
            Benjamin Jenkins, Ian Tracy, Maria Fernanda S. D. Rodrigues, Melanie J. L. Smith, Begoña R. Martinez, Mark Edmond, Sangeetha Mahadevan, Anjali Rao, Hailing Zong, Kai Liu, Abhishek Aggarwal, Li Li, Lauri Diehl, Emma V. King, Jamie G. Bates, Christopher J Hanley, Gareth J. Thomas
          </td>
          <td>2025-01-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd1722df9025221021ddfd3a6fcbf675ba9705b" target='_blank'>
              OLIG2 mediates a rare targetable stem cell fate transition in sonic hedgehog medulloblastoma
              </a>
            </td>
          <td>
            Kinjal Desai, Siyi Wanggou, Erika Luis, H. Whetstone, Chunying Yu, R. Vanner, Hayden J. Selvadurai, Lilian Lee, Jinchu Vijay, J. E. Jaramillo, Jerry J. Fan, P. Guilhamon, M. Kushida, Xuejun Li, Gregory Stein, Santosh Kesari, B. D. Simons, Xi Huang, Peter B. Dirks
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0b1fd2e12bcf45ef4e0b8a0b897b112e846447" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Takafumi N. Yamaguchi, Yu-Jia Shiah, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, A. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Activation of invasion and metastasis is a central hallmark of cancer, contributing to the primary cause of death for cancer patients. In the multistep metastatic process, cancer cells must infiltrate the circulation, survive, arrest at capillary beds, extravasate, and form metastatic clones in distant organs. However, only a small proportion of circulating tumor cells (CTCs) successfully form metastases, with transit of CTCs in the circulation being the rate-limiting step. The fate of CTCs is influenced by the circulating tumor microenvironment (cTME), which encompasses factors affecting their biological behaviors in the circulation. This liquid and flowing microenvironment differs significantly from the primary tumor microenvironment or the pre-metastatic niche. This review summarizes the latest advancements in identifying the biophysical cues, key components, and biological roles of the cTME, highlighting the network among biophysical attributes, blood cells, and non-blood factors in cancer metastasis. In addition to the potential of the cTME as a therapeutic target for inhibiting metastasis, the cTME could also represent as a biomarker for predicting patient outcomes and developing strategies for treating cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ebef94ee6fd9d5b0a0f131941d243a6a93e28d" target='_blank'>
              Circulating Tumor Microenvironment in Metastasis.
              </a>
            </td>
          <td>
            Rui Tang, Shujuan Luo, Hui Liu, Yan Sun, Manran Liu, Lu Li, Haoyu Ren, Martin K Angele, N. Börner, Keda Yu, Zufeng Guo, Guobing Yin, Haojun Luo
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Epithelial cancers are typically heterogeneous with primary prostate cancer being a typical example of histological and genomic variation. Prior studies of primary prostate cancer tumour genetics revealed extensive inter and intra-patient genomic tumour heterogeneity. Recent advances in machine learning have enabled the inference of ground-truth genomic single-nucleotide and copy number variant status from transcript data. While these inferred SNV and CNV states can be used to resolve clonal phylogenies, however, it is still unknown how faithfully transcript-based tumour phylogenies reconstruct ground truth DNA-based tumour phylogenies. We sought to study the accuracy of inferred-transcript to recapitulate DNA-based tumour phylogenies. We first performed in-silico comparisons of inferred and directly resolved SNV and CNV status, from single cancer cells, from three different cell lines. We found that inferred SNV phylogenies accurately recapitulate DNA phylogenies (entanglement = 0.097). We observed similar results in iCNV and CNV based phylogenies (entanglement = 0.11). Analysis of published prostate cancer DNA phylogenies and inferred CNV, SNV and transcript based phylogenies demonstrated phylogenetic concordance. Finally, a comparison of pseudo-bulked spatial transcriptomic data to adjacent sections with WGS data also demonstrated recapitulation of ground truth (entanglement = 0.35). These results suggest that transcript-based inferred phylogenies recapitulate conventional genomic phylogenies. Further work will need to be done to increase accuracy, genomic, and spatial resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79d56f0a1a843a737c66bbddbc1e662cc71b163e" target='_blank'>
              Clonal phylogenies inferred from bulk, single cell, and spatial transcriptomic analysis of epithelial cancers
              </a>
            </td>
          <td>
            Andrew Erickson, Sandy Figiel, Timothy Rajakumar, Srinivasa Rao, Wencheng Yin, D. Doultsinos, Anette Magnussen, Reema Singh, Ninu Poulose, Richard J. Bryant, Olivier Cussenot, Freddie C Hamdy, Dan Woodcock, I. Mills, Alastair D. Lamb
          </td>
          <td>2025-01-03</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e7693290b0f56e7e4e347ce73c837cd2d47cd1" target='_blank'>
              Stromal architecture and fibroblast subpopulations with opposing effects on outcomes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yifei Cheng, Xiaofang Chen, Li Feng, Zhicheng Yang, Liyun Xiao, Bin Xiang, Xiaodong Wang, Dongbing Liu, Penghui Lin, Jieyi Shi, Guohe Song, Wulei Qian, Boan Zhang, Yanan Xu, Zheng Gao, Lv Chen, Y. Wu, Jiaqiang Ma, Youpei Lin, Haichao Zhao, Lihua Peng, Xuebin Mao, Yang Liu, Hao Hou, Mingyu Yang, Yuan Ji, Xiaoying Wang, Jian Zhou, Xun Xu, Xiyang Liu, Wu Wei, Xiaoming Zhang, Qiang Gao, Hu Zhou, Yidi Sun, Kui Wu, Jia Fan
          </td>
          <td>2025-01-28</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff35b3725766793dc6e344fca09fac4e90d00646" target='_blank'>
              LATS1/2 inactivation in the mammary epithelium drives the evolution of a tumor-associated niche.
              </a>
            </td>
          <td>
            Joseph G. Kern, Lina Kroehling, Anthony J Spinella, Stefano Monti, X. Varelas
          </td>
          <td>2025-02-14</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/197782f3d8b81c382c4e0d5197a6f09c7baeb2a9" target='_blank'>
              Immune microenvironment spatial landscapes of tertiary lymphoid structures in gastric cancer
              </a>
            </td>
          <td>
            Yi Xie, Haoxin Peng, Yajie Hu, Keren Jia, Jiajia Yuan, Dan Liu, Yanyan Li, Xujiao Feng, Jian Li, Xiaotian Zhang, Yu Sun, Lin Shen, Yang Chen
          </td>
          <td>2025-02-04</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The role of glioma-associated myeloid cells in tumor growth and immune evasion remains poorly understood. We performed single-cell RNA sequencing of immune and tumor cells from 33 gliomas, identifying two distinct myeloid-derived suppressor cell (MDSC) populations in isocitrate dehydrogenase-wild-type (IDT-WT) glioblastoma: an early progenitor MDSC (E-MDSC) population with up-regulation of metabolic and hypoxia pathways and a monocytic MDSC (M-MDSC) population. Spatial transcriptomics demonstrated that E-MDSCs geographically colocalize with metabolic stem-like tumor cells in the pseudopalisading region. Ligand-receptor analysis revealed cross-talk between these cells, where glioma stem-like cells produce chemokines attracting E-MDSCs, which in turn produce growth factors for the tumor cells. This interaction is absent in IDH-mutant gliomas, associated with hypermethylation and repressed gene expression of MDSC-attracting chemokines. Our study elucidates specific MDSCs that may facilitate glioblastoma progression and mediate tumor immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c3594c9527f59638cf5b92d7a7853cdf4cfa76" target='_blank'>
              Distinct myeloid-derived suppressor cell populations in human glioblastoma.
              </a>
            </td>
          <td>
            Christina Jackson, Chris C. Cherry, Sadhana Bom, Arbor G. Dykema, Rulin Wang, Elizabeth A Thompson, Ming Zhang, Runzhe Li, Zhicheng Ji, Wenpin Hou, Wentao Zhan, Hao Zhang, John Choi, A. Vaghasia, Landon Hansen, W. Wang, Brandon Bergsneider, Kate M Jones, Fausto Rodriguez, Jon D. Weingart, Calixto-Hope G Lucas, Jonathan Powell, Jennifer H. Elisseeff, Srinivasan Yegnasubramanian, Michael Lim, C. Bettegowda, Hongkai Ji, Drew M Pardoll
          </td>
          <td>2025-01-17</td>
          <td>Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The intricate relationship between tumor-associated macrophages (TAMs) and cancer cells is pivotal for carcinogenesis, with TAMs being integral to the tumor microenvironment (TME). This study explores the novel mechanisms by which TAMs regulate the progression of triple-negative breast cancer (TNBC) within the TME. Using a co-culture system and methodologies such as cytokine arrays, proteomics, and CRISPR-Cas9, we investigated the crosstalk between TAMs and TNBC cells. We found that high levels of CD163 + TAMs in TNBC tissues correlate with poor prognosis. TNBC cell-conditioned medium induces macrophage polarization towards the M2 phenotype, enhancing TNBC cell migration, invasion, and stemness through the secretion of extracellular nicotinamide phosphoribosyltransferase (eNAMPT). eNAMPT binding to CCR5 on TNBC cells activates STAT3, leading to the downregulation of the tumor suppressor DIRAS2 and an increase in CCL2, which promotes a macrophage recruitment loop. Intervention at the eNAMPT/CCR5 or CCL2 level disrupts this loop, mitigating TAM-induced effects. Our findings uncover a cytokine communication mechanism between immune and cancer cells, suggesting potential targets for TNBC detection and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c28b010de3d535257f1a7d38f1bc850e5db43ed" target='_blank'>
              eNAMPT/Ac-STAT3/DIRAS2 Axis Promotes Development and Cancer Stemness in Triple-Negative Breast Cancer by Enhancing Cytokine Crosstalk Between Tumor-Associated Macrophages and Cancer Cells
              </a>
            </td>
          <td>
            Lifen Zhang, Lu Wang, Ziyao Xu, Xingmei Zhang, Shaoyu Guan, Zhe Liu, Shanzhi Gu, Lin Zhao, Weichao Bai, Tian Li, Xinhan Zhao
          </td>
          <td>2025-02-18</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="We seek to establish a parsimonious mathematical framework for understanding the interaction and dynamics of the response of pancreatic cancer to the NGC triple chemotherapy regimen (mNab-paclitaxel, gemcitabine, and cisplatin), stromal-targeting drugs (calcipotriol and losartan), and an immune checkpoint inhibitor (anti-PD-L1). We developed a set of ordinary differential equations describing changes in tumor size (growth and regression) under the influence of five cocktails of treatments. Model calibration relies on three tumor volume measurements obtained over a 14-day period in a genetically engineered pancreatic cancer model (KrasLSLG12D-Trp53LSLR172H-Pdx1-Cre). Our model reproduces tumor growth in the control and treatment scenarios with an average concordance correlation coefficient (CCC) of 0.99±0.01. We conduct leave-one-out predictions (average CCC=0.74±0.06), mouse-specific predictions (average CCC=0.75±0.02), and hybrid, group-informed, mouse-specific predictions (average CCC=0.85±0.04). The developed mathematical model demonstrates high accuracy in fitting the experimental tumor data and a robust ability to predict tumor response to treatment. This approach has important implications for optimizing combination NGC treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99a317eb5daf4775d488a8cf45bdb23487a0a8c3" target='_blank'>
              Modeling tumor dynamics and predicting response to chemo-, targeted-, and immune-therapies in a murine model of pancreatic cancer
              </a>
            </td>
          <td>
            Krithik Vishwanath, Hoon Choi, Mamta Gupta, Rong Zhou, A. Sorace, T. Yankeelov, Ernesto A. B. F. Lima
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d017ae1208a1788cb6dfdbf710e4c68d672f7f57" target='_blank'>
              Network-based transfer of pan-cancer immunotherapy responses to guide breast cancer prognosis
              </a>
            </td>
          <td>
            Xiaobao Ding, Lin Zhang, M. Fan, Lihua Li
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3fe6ab149b32eadab919b62a58d2ecdad6d28ca" target='_blank'>
              Senescence-specific molecular subtypes stratify the hallmarks of the tumor microenvironment and guide precision medicine in bladder cancer
              </a>
            </td>
          <td>
            Luzhe Yan, Haisu Liang, Tiezheng Qi, D. Deng, Jinhui Liu, Yunbo He, Jinbo Chen, Benyi Fan, Yi-min Yao, Kun Wang, Xiongbing Zu, Minfeng Chen, Yuanqing Dai, Jiao Hu
          </td>
          <td>2025-02-19</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53b339228b07860b7f7b0ff107251f16ed54dc97" target='_blank'>
              Tumor microenvironment as a novel therapeutic target for lymphoid leukemias.
              </a>
            </td>
          <td>
            Shahrzad Mousavi, Soheil Nouri, Arezoo Sadeghipour, Amir Atashi
          </td>
          <td>2025-02-25</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e408aa172c15a4c549b8743b41474549ca768526" target='_blank'>
              Immunomodulatory role of exosome-derived content in pediatric medulloblastoma: a molecular subgroup perspective.
              </a>
            </td>
          <td>
            Jessica Oliveira de Santis, G. R. de Sousa, Rosane Gomes Queiroz, M. F. Cândido, Fausto Almeida, C. P. de Rezende, Patricia Cassia de Ruy, G. S. Arini, Beth Coyle, Philippa Wade, M. Brassesco, C. Scrideli, L. Tone, E. Valera
          </td>
          <td>2025-02-17</td>
          <td>Human cell</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The three most prevalent malignant skin tumors are basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and cutaneous melanoma (CM). While BCC and SCC generally exhibit low levels of malignancy, some tumors can become invasive and metastasize. In contrast, CM is the most malignant and lethal form of skin cancer among these three. In recent years, the increasing incidence of skin tumors has garnered significant attention within the medical community. The advent of single-cell RNA sequencing (scRNA-seq) technology has revolutionized the analysis of the tumor microenvironment (TME) in skin tumors, offering novel insights for clinical research. In this review, we provide a concise introduction to scRNA-seq technology and its application in delineating the heterogeneity of the TME in skin tumors, elucidating the mechanisms of tumor progression and pathogenicity, and uncovering new therapeutic targets. Our review offers a comprehensive overview for researchers, offering insights that may advance the understanding of the skin TME in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c806552e713787b83ea0933cd31a52d109ce5cf9" target='_blank'>
              Advances and applications of single-cell RNA sequencing in deciphering the tumor microenvironment of malignant skin tumors
              </a>
            </td>
          <td>
            Yiting Feng, Lanlan Liu, Chunlan Hu, Jie Sun, Mingzhu Yin, Xin Li
          </td>
          <td>2025-01-10</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Racial disparities in cancer outcomes are well documented across tumor types. For patients with breast cancer, Black women are more likely to present with more aggressive molecular features and more likely to die from disease, even after accounting for those features. Recent efforts have been aimed at developing translational model systems for precision medicine strategies, and a major focus has been on patient-derived organoids. Organoids allow for robust in vitro experimental platforms, including drug and CRISPR screens while maintaining more complex cancer and tumor microenvironment subpopulations than cell lines. For results that are broadly translationally relevant, it is important that cancer models are derived from the spectrum of human disease and humans with disease. In this issue of Cancer Research, Madorsky Rowdo and colleagues derive breast cancer organoids from patients with African ancestry and use CRISPR-Cas9 screens to identify novel therapeutic vulnerabilities. These findings demonstrate the promise of representative cancer model systems to facilitate discoveries that are most likely to translate to improved therapy for all patients. See related article by Madorsky Rowdo et al., p. 551.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af090730ccebba2da2c3e50d4cbf6a340cfee095" target='_blank'>
              In Search of Representative Translational Cancer Model Systems.
              </a>
            </td>
          <td>
            Hannah E. Trembath, Philip M. Spanheimer
          </td>
          <td>2025-02-01</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2ea126a4602d2754c646d3838a0c077f74d91b" target='_blank'>
              CXCL14 in prostate cancer: complex interactions in the tumor microenvironment and future prospects
              </a>
            </td>
          <td>
            Lei Tang, Xin Chen, Jianquan Hou, Xuedong Wei
          </td>
          <td>2025-01-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9629af3f9c8496403aa9d7445a8f7f79cca8a69" target='_blank'>
              Genes and proteins expression profile of 2D vs 3D cancer models: a comparative analysis for better tumor insights.
              </a>
            </td>
          <td>
            Sunaina Bhuker, Abhinav Kumar Sinha, Anuksha Arora, H. Tuli, Sonal Datta, A. Saini, R. Saini, S. Ramniwas
          </td>
          <td>2025-01-25</td>
          <td>Cytotechnology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Acute lymphoblastic leukemia (ALL) is the most common childhood cancer. The translocation t(1;19), encoding the TCF3‐PBX1 fusion, is associated with intermediate risk and central nervous system (CNS) infiltration at relapse. Using our previously generated TCF3‐PBX1 conditional knock‐in mice, we established a model to study relapsed clones after in vivo chemotherapy treatment, CNS infiltration, and clonal dynamic evolution of phenotypic diversity at the single cell‐level using next‐generation sequencing technologies and mass cytometry. Mice transplanted with TCF3‐PBX1 + leukemia cells and treated with vehicle succumbed to disease, whereas 40% of treated mice with prednisolone or daunorubicin survived. Bulk and single‐cell RNA sequencing of FACS‐sorted GFP+ cells from TCF3‐PBX1 + leukemias arising after chemotherapy treatment revealed that apoptosis, interleukin‐, and TGFβ‐signaling pathways were regulated in CNS‐infiltrating leukemic cells. Across tissues, upregulation of the MYC signaling pathway was detected in persisting leukemic cells and its downregulation by BRD3/4 inhibition increased sensitivity to chemotherapy. In TCF3‐PBX1+ leukemia cells collected after chemotherapy treatment, mass cytometry identified increased phosphorylation of STAT3/5 upon preBCR stimulation, which was susceptible to inhibition by the proteasome inhibitor bortezomib. In summary, we developed a TCF3‐PBX1+ ALL mouse model and characterized relapsed disease after in vivo chemotherapy and cell phenotype dependence on microenvironment. Transcriptomics and phospho‐proteomics revealed distinct pathways that may underlie chemotherapy resistance and might be suitable for pharmacological interventions in human ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278da44c640128811ffe9cd7b7d31669827014f" target='_blank'>
              Dynamic evolution of TCF3‐PBX1 leukemias at the single‐cell level under chemotherapy pressure
              </a>
            </td>
          <td>
            Mira Kusterer, M. Lahnalampi, M. Voutilainen, Alexandra Brand, S. Pennisi, J. Norona, Gaia Gentile, Heike Herzog, G. Greve, Michael Lübbert, M. Sipola, Emma Kaartinen, R. Sankowski, Marco Prinz, S. Killmer, M. S. Lago, Bertram Bengsch, Stepan Cysar, K. Aumann, Martin Werner, J. Duyster, O. Lohi, M. Heinäniemi, J. Duque-Afonso
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c81eee6b668168bd971313aa0fe51bd578f98e" target='_blank'>
              Meta-analyses of mouse and human prostate single-cell transcriptomes reveal widespread epithelial plasticity in tissue regression, regeneration, and cancer
              </a>
            </td>
          <td>
            Luis C. Aparicio, Laura Crowley, John R. Christin, Caroline J. Laplaca, H. Hibshoosh, Raul Rabadan, Michael M. Shen
          </td>
          <td>2025-01-17</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The development of therapies that target cancer stem cells (CSCs) and bulk tumors is both crucial and urgent. Several signaling pathways, like Notch and Hedgehog (Hh), have been strongly associated with CSC stemness maintenance and metastasis. However, the extensive crosstalk present between these two signaling networks complicates the development of long-term therapies that also minimize adverse effects on healthy tissues and are not overcome by therapy resistance from CSCs. The present work aims to overview the roles of Notch and Hh in cancer outburst and the intersection of the two pathways with one another, as well as with other networks, such as Wnt/β-catenin, TGF, and JAK/STAT3, and to explore the shaping of the tumor microenvironment (TME) with specific influence on CSC development and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f140e4496eee327489d912de5427944cb351ca1" target='_blank'>
              Notch and Hedgehog Signaling Unveiled: Crosstalk, Roles, and Breakthroughs in Cancer Stem Cell Research
              </a>
            </td>
          <td>
            Sabina Iluta, Mădălina Nistor, Sanda Buruiana, D. Dima
          </td>
          <td>2025-02-01</td>
          <td>Life</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f63f6a23a3983ad60b3216e3e5876ec300a9938" target='_blank'>
              Deciphering the generation of heterogeneity in esophageal squamous cell carcinoma metastasis via single-cell multiomics analysis
              </a>
            </td>
          <td>
            Kaiwen Sheng, Jun Chen, Ruitang Xu, Haoqiang Sun, Ran Liu, Yongjie Wang, Wenwen Xu, Jiao Guo, Miao Zhang, Shuai Liu, Juan Lei, Yawen Sun, Yang Jia, Dianhao Guo
          </td>
          <td>2025-02-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8641c47a99cb4e08fcabebc4fec5794fc9e23e2f" target='_blank'>
              Role of CD4+ T cells in cancer immunity: a single-cell sequencing exploration of tumor microenvironment
              </a>
            </td>
          <td>
            Qi An, Li Duan, Yuanyuan Wang, Fuxin Wang, Xiangyu Liu, Chao Liu, Qinyong Hu
          </td>
          <td>2025-02-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c52c76852d3ba58984faf618de957dc0aff431e" target='_blank'>
              Spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis.
              </a>
            </td>
          <td>
            A. Vannan, Ruqian Lyu, A. L. Williams, N. Negretti, E. D. Mee, J. Hirsh, S. Hirsh, Niran Hadad, David Nichols, C. Calvi, Chase J Taylor, V. Polosukhin, A. P. Serezani, A. S. McCall, J. Gokey, Heejung Shim, Lorraine B. Ware, M. Bacchetta, C. Shaver, Timothy S. Blackwell, Rajat Walia, J. Sucre, JA Kropski, Davis J. McCarthy, N. Banovich
          </td>
          <td>2025-02-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The tumor microenvironment (TME) milieu directs a plethora of tumor-modulating functions. Recent years have seen pivotal breakthroughs in our understanding of the TME's role in tumor initiation and progression, with tangible clinical applications. Individual components of the TME exert their function predominantly by cell-cell crosstalk, both in the form of physical interaction and secreted factors. Notably, different spatial niches represent exclusive signaling hubs in the TME, propagating pro- or antitumoral functions. The exploration of these interactions has been vastly facilitated by novel molecular technologies, each of which provides a different perspective on this intricate intercellular communication network. Together, these complementary methods paint a detailed, high-resolution map of the TME's interaction landscape. In this viewpoint, we explore how cellular interactions can unlock a new level of understanding of TME complexity, and discuss the promises and challenges of characterizing tumors based on their cellular interaction footprint.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/072aea9d500a6665bd7f3f737b00c34169b4d84a" target='_blank'>
              Decoding multicellular interaction networks-a new horizon in tumor microenvironment research.
              </a>
            </td>
          <td>
            Roi Balaban, Merav Cohen
          </td>
          <td>2025-02-05</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Antigen processing and presentation via major histocompatibility complex (MHC) molecules are central to immune surveillance. Yet, quantifying the dynamic activity of MHC class I and II antigen presentation remains a critical challenge, particularly in diseases like cancer, infection and autoimmunity where these pathways are often disrupted. Current methods fall short in providing precise, sample-specific insights into antigen presentation, limiting our understanding of immune evasion and therapeutic responses. Here, we present PSAA (PINN-empowered Systems Biology Analysis of Antigen Presentation Activity), which is designed to estimate sample-wise MHC class I and class II antigen presentation activity using bulk, single-cell, and spatially resolved transcriptomics or proteomics data. By reconstructing MHC pathways and employing pathway flux estimation, PSAA offers a detailed, stepwise quantification of MHC pathway activity, enabling predictions of gene-specific impacts and their downstream effects on immune interactions. Benchmarked across diverse omics datasets and experimental validations, PSAA demonstrates a robust prediction accuracy and utility across various disease contexts. In conclusion, PSAA and its downstream functions provide a comprehensive framework for analyzing the dynamics of MHC antigen presentation using omics data. By linking antigen presentation to immune cell activity and clinical outcomes, PSAA both elucidates key mechanisms driving disease progression and uncovers potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/582181263bb354bc38b72cdfdf043ee2535ac41b" target='_blank'>
              A physics informed neural network approach to quantify antigen presentation activities at single cell level using omics data
              </a>
            </td>
          <td>
            Chi Zhang, Jia Wang, Pengtao Dang, Yuhui Wei, Xiao Wang, Julie A. Brothwell, Yifan Sun, Haiqi Zhu, K. So, Jing Liu, Yijie Wang, Xiongbin Lu, Stanley M Spinola, Xinna Zhang, Sha Cao
          </td>
          <td>2025-01-17</td>
          <td>Research Square</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Purpose Papillary renal cell carcinoma (pRCC), the second most common subtype of renal cancer, exhibits heterogeneity in molecular features and response to targeted therapies, including antiangiogenic drugs. Discovering molecular biomarkers able to stratify pRCC patients into clinically relevant subgroups and understanding the underlying mechanisms are urgently needed to advance precision medicine. Here, we molecularly dissect a large pRCC series through the expression of the targets of hypoxia inducible factors (HIF), master regulators of angiogenesis, metabolic reprogramming and immune microenvironment. Experimental Design We merged and analyzed multi-omic and clinical data of 346 patients derived from two pRCC series (TCGA-KIRP and a Spanish metastatic series). Altered pathways and differences in tumor microenvironment were identified through tumor transcriptomic analyses. Tumor metabolome analysis was performed in selected cases. Results Molecular revision of driver mutations classified 302 patients as pRCC, while uncovering misclassified cases. Analysis of HIF targets gene expression identified a subset of pRCC tumors with increased HIF activity (31%; “HIF-active”). These tumors were characterized by high hypoxia scores, increased angiogenesis, low expression of Krebs cycle genes and mitochondrial activity, high immune infiltration and increased epithelial-mesenchymal transition (P<0.0001; each feature) and they were associated with increased metastasis risk and worse overall survival (P<0.005). Metabolome analyses revealed that HIF-active tumors accumulated L-2-hydroxyglutarate (L-2-HG), an oncometabolite that leads to pseudohypoxia by inhibiting HIF prolyl hydroxylases and preventing HIFα degradation. L-2-HG-accumulation agreed with diminished L2HGDH expression, associated with losing one copy of the gene and low expression of PPARGC1A, which regulates its transcription. Conclusions HIF-pathway overactivation defines a pseudohypoxic pRCC molecular subgroup characterized by high angiogenesis, immune infiltration and aggressive features. These findings advance our understanding of pRCC diversity, revealing potential clues for the variable response of patients to targeted therapies and paving the way to more personalized pRCC treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b17c0ff397a2a18624c6810af2df85c7a525f01" target='_blank'>
              Identification of a novel papillary renal cell carcinoma molecular subtype characterized by HIF-pathway over-activation
              </a>
            </td>
          <td>
            Javier de Nicolás-Hernández, Alicia Arenas, Carlos Valdivia, María Santos, Javier Lanillos, Nicole Bechmann, M. Peitzsch, Eduardo Gil-Vilariño, María Monteagudo, Sara Mellid, C. Montero‐Conde, A. Cascón, L. Leandro-García, G. de Velasco, Ignacio Durán, Javier Puente, J. García-Donás, Eduardo Caleiras, Mercedes Robledo, Cristina Rodríguez-Antona
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Grade 3 ependymoma is classified as a fast-growing solid tumor that often occurs in the base of the brain with a median survival of less than two years. Despite aggressive standard treatments (surgery, radiation therapy and chemotherapy), grade 3 ependymoma remains uniformly fatal, especially if the tumor cannot be completely removed or if it spreads to other areas of the central nervous system (CNS). Achieving a responsive therapy with chimeric antigen receptor (CAR) T cell therapy has been a significant challenge due to evasive tumor microenvironment, specifically utilized by slid tumors, to resist therapy. Recent studies have highlighted the substantial role of glioma-associated stromal cells in solid tumors by depositing various collagen subunits, including COL1A1, within the extracellular matrix (ECM). This collagen deposition leads to increased ECM stiffness, modulating the tumor microenvironment, inducing immune suppression (e.g. SPP1 overexpression), and hindering T cell trafficking, ultimately worsening clinical outcomes and patient survival. To gain a better understanding of the role of the tumor microenvironment in CAR T persistence and efficacy, we designed and performed ultra-high plex immunofluorescent staining on tissue biopsies obtained from a patient with grade 3 ependymoma before and after treatment with IL13Ra2 CAR T cells. The patient has been received three CAR T infusions at weekly intervals. The tissues were stained for CD3, CD19 (CAR T cells), CD68, CD163, HLA-DR (macrophages), CD3, CD8 (endogenous T cells), SOX2 (stem cells), SPP1 (suppression) and COL1A1 (extracellular matrix). The presence of CAR T cells in tissue biopsies indicated persistence of IL13Ra2 CAR T cells for more than a year. Our preliminary analysis revealed an increase in the number of HLA-DR+ CD163+ macrophages within cellular neighborhoods associated with CAR T cells (around 50% of CD163+ express HLA-DR after treatment) as compared to baseline tumors, which suggests a dynamic interaction between myeloid subsets and CAR T cells within the tumor microenvironment. Ongoing experiments are currently investigating the presence of antigen presenting natural killer cells (CD56, HLA-DR), proliferative T cells (CD3, Ki67) and inflammatory macrophages and/or T cells (IFNg) to gain a better understanding of IL13Ra2 persistence in brain tumor patients.
 Citation Format: Maryam Aftabizadeh, Brenda Aguilar, Massimo D'Apuzzo, Behnam Badie, Christine E Brown. A deeper understanding of tumor microenvironment modulation induced by IL13Ra2 CAR T cells in a patient with grade 3 ependymoma [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A030.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0786e178852bb9b6b2a0945adb3bddac988e519" target='_blank'>
              Abstract A030: A deeper understanding of tumor microenvironment modulation induced by IL13Ra2 CAR T cells in a patient with grade 3 ependymoma
              </a>
            </td>
          <td>
            Maryam Aftabizadeh, B. Aguilar, M. D’Apuzzo, B. Badie, Christine E Brown
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf42e4d5c5ce1d7e5ee6b6939079dd4741136db7" target='_blank'>
              Revolutionizing ESCC prognosis: the efficiency of tumor-infiltrating immune cells (TIIC) signature score
              </a>
            </td>
          <td>
            Haixia Wang, Shaowei Ma, Zixin Yang, Ren Niu, Haiyong Zhu, Shujun Li, Shao Gao, Zhirong Li, Yanhua Tian
          </td>
          <td>2025-01-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CAR-T) cell therapy has emerged as a promising treatment for hematological malignancies, offering a targeted approach to cancer treatment. Understanding the complexities of CAR-T cell therapy within solid tumors poses challenges due to the intricate interactions within the tumor microenvironment. Mathematical modeling may serve as a valuable tool to unravel the dynamics of CAR-T cell therapy and improve its effectiveness in solid tumors. This study aimed to investigate the impact of spatial aspects in CAR-T therapy of solid tumors, utilizing cellular automata for modeling purposes. Our main objective was to deepen our understanding of treatment effects by analyzing scenarios with different spatial distributions and varying the initial quantities of tumor and CAR-T cells. Tumor geometry significantly influenced treatment efficacy in-silico, with notable differences observed between tumors with block-like arrangements and those with sparse cell distributions, leading to the concept of immune suppression due to geometrical effects. This research delves into the intricate relationship between spatial dynamics and the effectiveness of CAR-T therapy in solid tumors, highlighting the relevance of tumor geometry in the outcome of cellular immunotherapy treatments. Our results provide a basis for improving the efficacy of CAR-T cell treatments by combining them with other ones reducing the density of compact tumor areas and thus opening access ways for tumor killing T-cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b56368d398e6734b66e55599843f0e9cefd36d23" target='_blank'>
              Geometric immunosuppression in CAR-T cell treatment: Insights from mathematical modeling
              </a>
            </td>
          <td>
            Silvia Bordel-Vozmediano, S. Sabir, Luc'ia Benito-Barca, B. Weigelin, V'ictor M. P'erez-Garc'ia Mathematical Oncology Laboratory, Instituto de Matem'atica Aplicada a la Ciencia y la Ingenier'ia, U. C. Mancha, Ciudad Real, Spain., D. D. Matem'aticas, Escuela T'ecnica Superior de Ingenieros Industriales, Facultad de Ciencias Experimentales, Universidad Francisco de Vitoria, Pozuelo de Alarc'on, Department of Functional Imaging, Radiopharmacy, Multiscale Immunoimaging, E. University, Tubingen, Germany, Cluster of Excellence iFIT ''Image-Guided, Functionally Instructed Tumor Therapies''
          </td>
          <td>2025-01-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d719de0ed34b62d63deb744432982b8548a3e52" target='_blank'>
              TGF-β signaling orchestrates cancer-associated fibroblasts in the tumor microenvironment of human hepatocellular carcinoma: unveiling insights and clinical significance
              </a>
            </td>
          <td>
            Junwei Ge, Hongwei Jiang, Junjun Chen, Xuemin Chen, Yue Zhang, Liangrong Shi, Xiao Zheng, Jingting Jiang, Lujun Chen
          </td>
          <td>2025-01-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Liver cancer exhibits diverse molecular characteristics and distinct immune cell infiltration patterns, which significantly influence patient outcomes. In this study, we thoroughly examined the liver cancer tumor environment by analyzing data from 419,866 individual cells across nine datasets involving 99 patients. By categorizing patients into different groups based on their immune cell profiles, including immune deficiency, B cells-enriched, T cells-enriched and macrophages-enriched, we better understood how these cells change in various patient subgroups. Our investigation of liver metastases from intestinal cancer uncovered a group of mast cells that might promote metastasis through pathways like inositol phosphate metabolism. Using genomic and clinical data from The Cancer Genome Atlas, we identified specific cell components linked to tumor characteristics and genetics. Our detailed study of cancer-associated fibroblasts (CAFs) revealed how they adapt and acquire new functions in the tissue environment, highlighting their flexibility. Additionally, we found a significant connection between CAF-related genes and the prognosis of hepatocellular carcinoma patients. This research provides valuable insights into the makeup of the liver cancer tumor environment and its profound impact on patient outcomes, offering fresh perspectives for managing this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dff3872da8e8b83984c596296a89bdd2fda835d8" target='_blank'>
              Deciphering the heterogeneity and plasticity of the tumor microenvironment in liver cancer provides insights for prognosis
              </a>
            </td>
          <td>
            Yihao Sun, Guojuan Shi, Jian Yang, Chun-Zhong Zhou, Chuhan Peng, Yu-Hong Luo, Ying Pan, Rui-Qi Wang
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Colorectal cancer (CRC) is a histologically heterogeneous disease with variable clinical outcome. The role the tumour microenvironment (TME) plays in determining tumour progression is complex and not fully understood. To improve our understanding, it is critical that the TME is studied systematically within clinically annotated patient cohorts with long‐term follow‐up. Here we studied the TME in three clinical cohorts of metastatic CRC with diverse molecular subtype and treatment history. The MISSONI cohort included cases with microsatellite instability that received immunotherapy (n = 59, 24 months median follow‐up). The BRAF cohort included BRAF V600E mutant microsatellite stable (MSS) cancers (n = 141, 24 months median follow‐up). The VALENTINO cohort included RAS/RAF WT MSS cases who received chemotherapy and anti‐EGFR therapy (n = 175, 32 months median follow‐up). Using a Deep learning cell classifier, trained upon >38,000 pathologist annotations, to detect eight cell types within H&E‐stained sections of CRC, we quantified the spatial tissue organisation and colocalisation of cell types across these cohorts. We found that the ratio of infiltrating endothelial cells to cancer cells, a possible marker of vascular invasion, was an independent predictor of progression‐free survival (PFS) in the BRAF+MISSONI cohort (p = 0.033, HR = 1.44, CI = 1.029–2.01). In the VALENTINO cohort, this pattern was also an independent PFS predictor in TP53 mutant patients (p = 0.009, HR = 0.59, CI = 0.40–0.88). Tumour‐infiltrating lymphocytes were an independent predictor of PFS in BRAF+MISSONI (p = 0.016, HR = 0.36, CI = 0.153–0.83). Elevated tumour‐infiltrating macrophages were predictive of improved PFS in the MISSONI cohort (p = 0.031). We validated our cell classification using highly multiplexed immunofluorescence for 17 markers applied to the same sections that were analysed by the classifier (n = 26 cases). These findings uncovered important microenvironmental factors that underpin treatment response across and within CRC molecular subtypes, while providing an atlas of the distribution of 180 million cells in 375 clinically annotated CRC patients. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f2afcc9e64651d9dfd348eba0187246fdd6ea3" target='_blank'>
              Computational pathology applied to clinical colorectal cancer cohorts identifies immune and endothelial cell spatial patterns predictive of outcome
              </a>
            </td>
          <td>
            Nicholas Trahearn, Chirine Sakr, Abhirup Banerjee, Seung Hyun Lee, Ann-Marie Baker, Hemant M. Kocher, V. Angerilli, F. Morano, F. Bergamo, G. Maddalena, R. Intini, C. Cremolini, Giulio Caravagna, Trevor Graham, F. Pietrantonio, S. Lonardi, Matteo Fassan, A. Sottoriva
          </td>
          <td>2025-01-09</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) techniques hold great value in evaluating the heterogeneity and spatial characteristics of hematopoietic cells within tissues. These two techniques are highly complementary, with scRNA-seq offering single-cell resolution and ST retaining spatial information. However, there is an urgent demand for well-organized and user-friendly toolkits capable of handling single-cell and spatial information. Here, we present HemaScope, a specialized bioinformatics toolkit featuring modular designs to analyze scRNA-seq and ST data generated from hematopoietic cells. It enables users to perform quality control, basic analysis, cell atlas construction, cellular heterogeneity exploration, and dynamical examination on scRNA-seq data. Also, it can perform spatial analysis and microenvironment analysis on ST data. Meanwhile, HemaScope takes into consideration hematopoietic cell-specific features, including lineage affiliation evaluation, cell cycle prediction, and marker gene collection. To enhance the user experience, we have deployed the toolkit in user-friendly forms: HemaScopeR (an R package), HemaScopeCloud (a web server), HemaScopeDocker (a Docker image), and HemaScopeShiny (a graphical interface). In case studies, we employed it to construct a cell atlas of human bone marrow, analyze age-related changes, and identify acute myeloid leukemia cells in mice. Moreover, we characterized the microenvironments in angioimmunoblastic T cell lymphoma and primary central nervous system lymphoma, elucidating tumor boundaries. HemaScope is freely available at https://zhenyiwangthu.github.io/HemaScope_Tutorial/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93bd105f231e9996b6a3ab55c7d2018c2946edbd" target='_blank'>
              HemaScope: A Tool for Analyzing Single-cell and Spatial Transcriptomics Data of Hematopoietic Cells.
              </a>
            </td>
          <td>
            Zhenyi Wang, Yuxin Miao, Hongjun Li, Wenyan Cheng, Minglei Shi, Lv Gang, Yating Zhu, Junyi Zhang, Tingting Tan, Jin Gu, Michael Q. Zhang, Jianfeng Li, Hai Fang, Zhu Chen, Saijuan Chen
          </td>
          <td>2025-01-25</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 232


 Background:
 Resistance to first-line Androgen Receptor Pathway Inhibitors (ARPI) in metastatic prostate cancer (mPC) is universal and can be driven by androgen receptor (AR) alterations driving constitutive AR signaling, or lineage state transitions that bypass AR and culminate in small cell/neuroendocrine PC (NEPC). Recent advances in cell surface targeted therapies, including radioligand therapies and antibody drug conjugates show promise for treatment of metastatic castrate-resistant prostate cancer (mCRPC). Circulating tumor cells (CTCs) are a minimally-invasive source of tumor material to track these known and novel targets for therapeutic development. We report differential expression of cell surface targets using the largest CTC RNA sequencing cohort of patients with mPC across lineage states and clinical outcomes.
 Methods:
 We isolated CTCs using automated microfluidic technology on a 273-sample cohort from patients with histologically confirmed mPC. CTCs purified via anti-EpCAM conjugated magnetic beads were analyzed via RNA-seq. We examined CTC gene expression of cell surface targets, including PSMA, TROP2, B7H3, DLL3, stratified by: disease category (metastatic castration sensitive PC [mCSPC], mCRPC or NEPC), metastatic site (bone +/- lymph nodes, liver, other visceral), and prior ARPI treatment.
 Results:
 CTCs from patient with mCRPC showed high expression of canonical adenocarcinoma cell surface targets
 STEAP1
 ,
 KLK2
 and
 FOLH1
 (PSMA) and epithelial
 TACSTD2
 (TROP2), intermediate tumor immune checkpoint cell surface protein
 CD276
 (B7H3), and low expression of NEPC targets
 DLL3
 and
 SSTR2
 . CTCs from NEPC patients, as expected, demonstrated lower expression of most adenocarcinoma targets and a trend towards higher
 DLL3
 and
 SSTR2
 and associates with worse OS regardless of disease site in multivariate analysis (HR 4.76 [1.09-8.44], p=0.011). CTCs from patients with mCSPC had highest
 TACSTD2
 (p=0.0076) and a trend towards lower
 DLL3
 expression. When comparing metastatic sites, CTCs from non-NEPC mCRPC patients with liver metastases had significantly higher expression of
 DLL3
 (p=0.012) compared to bone or non-liver visceral metastases and can be detected in high-risk adenocarcinomas though expression is heterogeneous. Conversely, CTCs from mCRPC with liver metastases had lower
 TACSTD2
 expression (p=0.039). Prior ARPI therapy associated with lower
 TACSTD2
 (p=0.025) and a trend toward higher
 DLL3
 in patients with early evidence of lineage transition.
 Conclusions:
 We report differential cell surface target expression across lineage states and disease sites using novel CTC RNA-seq and single CTC phenotyping. Altered target expression was observed in PC lineage transitions. These findings have implications for treatment response towards surface targeted therapies, and future studies will utilize these biomarkers to explore mechanisms of response and resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/806ce0b8ce24491e542e1befccfa41c6236040a0" target='_blank'>
              Circulating tumor cell (CTC) expression patterns of cell surface targets in metastatic prostate cancer.
              </a>
            </td>
          <td>
            Kendra D. Marr, J. Sperger, Amy K Taylor, Katharine E Tippins, Shannon R Reese, K. Helzer, M. Bootsma, G. Blitzer, John M Floberg, D. Kosoff, Rana R McKay, Xiao X Wei, Shuang Zhao, Joshua M Lang, Marina N. Sharifi
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Metastatic melanoma is a lethal form of cancer. Immunotherapy is recommended for metastatic melanoma and known to elicit beneficial response even after disease progression per RECIST. On-treatment samples hold irreplaceable predictive value. We collected needle biopsies from the metastases of different body sites from the same patient that progressed upon PD-1 treatment. Leveraging Direct Nuclear Tagmentation and RNA sequencing (DNTR- seq), we conducted single-cell genomic analyses on melanoma metastases and jointly sequenced the transcriptomes of each single melanoma cell. After 3 months of PD-1 inhibitor treatment, single-cell DNA-seq analyses on melanoma metastases from different anatomic locations displayed distinct cancer subclones evidenced by their copy number aberrations. Joint transcriptomic analyses of the dominant genomic subclones from each body site showed converging phenotypes, under the selective pressure of immunotherapy. Our preliminary analyses of on-treatment needle biopsies shed light on the challenges of treating heterogeneous tumors and underscore the pressing need to identify an ideal denominator to reach disease control.
 Citation Format: Muyi Yang, Solrun Kolbeinsdottir, Katarina Hammarlund, Vitali Grozman, Martin Enge, Suzanne Egyhazi-Brage, Hildur Helgadottir. Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A108.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059cc807139dc64898d45632991b55c44cdfb1d6" target='_blank'>
              Abstract A108: Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients
              </a>
            </td>
          <td>
            Muyi Yang, Sólrún Kolbeinsdóttir, Katarina Hammarlund, V. Grozman, Martin Enge, Suzanne Egyhazi-Brage, H. Helgadottir
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8713ce532252c75bfca85781a29fddd0540c27c7" target='_blank'>
              Brd7 loss reawakens dormant metastasis initiating cells in lung by forging an immunosuppressive niche
              </a>
            </td>
          <td>
            Jayanta Mondal, Junfeng Zhang, Feng Qing, Shunping Li, Dhiraj Kumar, Jason T Huse, Filippo G. Giancotti
          </td>
          <td>2025-02-05</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Upper aerodigestive squamous cell carcinoma (UASCC) presents significant challenges in clinical management due to its aggressive nature. Here, we elucidate the role of MLL3 mutations as early, clonal genomic events in UASCC tumorigenesis, highlighting their role as foundational drivers of cancer development. Utilizing CRISPR-edited, cross-species organoid modeling, we demonstrate that loss of MLL3 contributes to early squamous neoplastic evolution. Furthermore, we identify an MLL3/GRHL2 protein complex that regulates the UASCC epigenome, particularly impacting immune response pathways. Notably, a novel MLL3/GRHL2-IRF1 axis promotes the expression of Th1 chemokines, enhancing anti-tumor immunity by facilitating T cell infiltration into the tumor microenvironment. Consequently, MLL3 regulates the in vivo efficacy of immune checkpoint blockade (ICB) therapy, corroborated by the strong association between MLL3 expression and human patients' clinical response to ICB therapy. Our work underscores the significance of MLL3 in UASCC pathogenesis and highlights the interplay between MLL3/GRHL2 and immune response pathways as potential therapeutic targets for UASCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50aaf113e900513b3c01d9cc2408109f9601e35a" target='_blank'>
              The MLL3/GRHL2 complex regulates malignant transformation and anti-tumor immunity in squamous cancer.
              </a>
            </td>
          <td>
            Chehyun Nam, Guowei Huang, Yueyuan Zheng, Hua Zhao, Yuhao Pan, Boyan Hu, Talia Wenger, Hieu T. Van, Liyan Xu, E. Li, H. Koeffler, Kai Ge, Y. Dou, Uttam K Sinha, Young Min Park, De-Chen Lin
          </td>
          <td>2025-02-18</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="The mammary microenvironment has been shown to suppress tumor progression by redirecting cancer cells to adopt a normal mammary epithelial progenitor fate in vivo. However, the mechanism(s) by which this alteration occurs has yet to be defined. Here, we test the hypothesis that mitochondrial transfer from normal mammary epithelial cells to breast cancer cells plays a role in this redirection process. We evaluate mitochondrial transfer in 2D and 3D organoids using our unique 3D bioprinting system to produce chimeric organoids containing normal and cancer cells. We demonstrate that breast cancer tumoroid growth is hindered following interaction with mammary epithelial cells in both 2D and 3D environments. Furthermore, we show mitochondrial transfer occurs between donor mammary epithelial cells and recipient cancer cells primarily through tunneling nanotubes (TNTs) with minimal amounts seen from extracellular transfer of mitochondria, likely via extracellular vesicles (EVs). This organelle exchange results in various cellular and metabolic alterations within cancer cells, reducing their proliferative potential, and making them susceptible to microenvironmental control. Our results demonstrate that mitochondrial transfer contributes to microenvironmental redirection of cancer cells through alteration of metabolic and molecular functions of the recipient cancer cells. To the best of our knowledge, this is the first description of a 3D bioprinter-assisted organoid system for studying mitochondrial transfer. These studies are also the first mechanistic insights into the process of mammary microenvironmental redirection of cancer and provide a framework for new therapeutic strategies to control cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d74a99d567922b268e37075173798f1b540e4d47" target='_blank'>
              Intercellular mitochondrial transfer contributes to microenvironmental redirection of cancer cell fate.
              </a>
            </td>
          <td>
            J. S. Bjerring, Yara Khodour, Emilee Anne Peterson, Patrick C. Sachs, Robert David Bruno
          </td>
          <td>2025-02-11</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) drive tumor progression through restructuring of the tumor microenvironment. This investigation aim to elucidate the function of molecular subtypes (MS) derived from cancer cells communication with CAFs, depicting the hallmarks of the tumor microenvironment and precise bladder cancer (BLCA) treatment. The BLCA data from TCGA and several external sources are utilized to generate a novel ligand, receptor, and transcription factor (LRT) associated molecular subtype and their corresponding score (LRT score). The LRT-mediated molecular subtype is identified via unsupervised clustering. LRT score is measured by principal component analysis. Then, the association of LRT clusters to established MS, immunophenotypes, and medical endpoints, together with BLCA treatment strategies is investigated. Two LRT clusters (A and B) are identified. LRT cluster (LRT score) can precisely propose immunophenotypes, classical MS, clinical outcomes, and BLCA therapeutic strategies. Cluster B (Low LRT score) represent a basal subtype and inflamed phenotype specified by high immunity against tumors and unfavorable clinical outcomes. Furthermore, it is highly sensitive to cancer immunotherapy; however, it has low sensitivity to antiangiogenic and targeted therapies. The novel LRT clusters with a strong association with biological characteristics and precise BLCA treatment strategies are derived from the communication between cancer cells and cancer-associated fibroblasts. The LRT may be a useful clinician tool for developing individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f8d25128e68026c13f6a41f5701ec7d66c8485" target='_blank'>
              Cancer Cell and Cancer-Associated Fibroblast Communication-Mediated Molecular Subtypes Portray Non-Inflamed Tumor Microenvironment and Guide the Precision Treatment of Bladder Cancer.
              </a>
            </td>
          <td>
            Shenglin Gao, Chuan Liu, Lixin Mao, Yin Chen, Xiaokai Shi, C. Yue, Shouchun Li, Xihu Qin
          </td>
          <td>2025-02-17</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Lung cancer continues to be the leading cause of cancer-related deaths, projected to claim over 125,000 lives in the United States in 2024. While therapeutic options have improved with the introduction of immune checkpoint inhibitors (ICI) to the clinic, many patients do not respond to immunotherapy or develop resistance following initial response. A significant subset of non-small cell lung cancer (NSCLC) patients harbors somatic co-mutations in Kirsten rat sarcoma virus (KRAS) and Liver kinase 1 [LKB1, also known as serine/threonine kinase 11 (STK11)] genes, whose tumors are described by an immunosuppressive tumor microenvironment (TME) that are resistant to ICI. Our studies revealed that all-trans retinoic acid (ATRA), a metabolite derived from vitamin A, sensitized a murine model of NSCLC with increased tumor mutational burden (Kras G12D P53 -/- Lkb1 -/- ; KPL-3M) to PD-1 blockade. The ATRA and anti-PD-1 combination therapy improved local and systemic T cell activation and generated systemic tumor-specific immunity. To investigate ATRA-mediated anti-tumor effects, we conducted single cell RNA sequencing (scRNA-seq) of KPL-3M murine tumors with or without ATRA treatment. scRNA-seq analysis revealed a reduction of neutrophils as well as an enrichment in T cell and natural killer (NK) cell populations in the TME. Furthermore, scRNA-seq trajectory and RNA velocity analysis suggested an enrichment in neutrophil clusters associated with differentiation and effector function. ATRA increased expression of maturity markers on neutrophils isolated from KPL-3M tumors as well as neutrophils differentiated in vitro from murine bone marrow. Additionally, preliminary ex vivo studies indicate that ATRA increases NOX2 levels, intracellular reactive oxygen species (ROS), and interferon (IFN) response signaling in the neutrophils in KPL-3M tumors. Our findings suggest that ATRA may reshape the TME by altering neutrophil composition to sensitize resistant tumors to ICI immunotherapy.
 Citation Format: William P Crosson, Rui Li, Ramin Salehi-Rad, Bitta P Kahangi, Samantha Chin, Raymond J Lim, Jensen Abascal, Edgar Perez Reyes, Michael Oh, Camelia Dumitras, Nico Edgar, Ryan Chew, Rashel Jacobo, Adam Tfayli, Kostyantyn Krysan, Linh M Tran, Steven M Dubinett, Bin Liu. ATRA reprograms neutrophils in the tumor microenvironment and overcomes resistance to immune checkpoint inhibition in murine models of LKB1-deficient non-small cell lung cancer [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B090.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f07daeb74e80fef9e4142d5ed3323341effa067" target='_blank'>
              Abstract B090: ATRA reprograms neutrophils in the tumor microenvironment and overcomes resistance to immune checkpoint inhibition in murine models of LKB1-deficient non-small cell lung cancer
              </a>
            </td>
          <td>
            William P Crosson, Rui Li, R. Salehi-Rad, Bitta Kahangi, Samantha Chin, R. Lim, J. Abascal, E. P. Reyes, M. Oh, C. Dumitras, Nico Edgar, Ryan Chew, Rashel Jacobo, Adam Tfayli, K. Krysan, L. Tran, Steven M Dubinett, Bin Liu
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background We conducted an investigation into the characteristics of single-cell differentiation data in gliomas, with a focus on developing DAPK1-based prognostic markers to predict patient outcomes. Dysregulated expression of DAPK1 has been associated with the invasive behavior of various malignancies, including gliomas. However, the precise role and underlying mechanisms of DAPK1 in gliomas remain inadequately understood. Methods We performed analyses on RNA-seq and microarray datasets from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO), in addition to single-cell RNA sequencing (scRNA-seq) data from glioma patients available in GEO. Utilizing the Seurat R package, we identified gene clusters associated with survival from the scRNA-seq data. Prognostic models were developed using LASSO and stepwise regression algorithms. Furthermore, we assessed the predictive potential of these genes within the immune microenvironment and their relevance in immunotherapy contexts. Results Our scRNA-seq data analysis revealed 32 distinct cell clusters corresponding to 10 cell types. Through dimensionality reduction and clustering, we identified three glial cell subpopulations based on their differentiation trajectories. DAPK1, serving as a marker gene for the terminal subpopulation, exhibited an association with poor prognosis. Conclusions DAPK1-based prognostic models show promise for accurately predicting outcomes in glioblastoma and glioma. An in-depth examination of DAPK1’s specific mechanisms in glioblastoma could elucidate its role in immunotherapy response. Targeting the DAPK1 gene may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fd67f17577e9d1b4649b5fe0ecd703f94585d" target='_blank'>
              Single-cell sequencing uncovers the mechanistic role of DAPK1 in glioma and its diagnostic and prognostic implications
              </a>
            </td>
          <td>
            Tian-Hang Yu, Yan-Yu Ding, Si-Guo Zhao, Jie-Hui Zhao, Yu- Gu, Dong-Hui Chen, Fan Zhang, Wen-Ming Hong
          </td>
          <td>2025-01-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ceeafcf1da623087d79a8767b5543bf2a7043f8" target='_blank'>
              Genomic and micro-environmental insights into drug resistance in colorectal cancer liver metastases
              </a>
            </td>
          <td>
            Junjie Kuang, Jun Li, Siwei Zhou, Yi Li, Jinbo Lin, Wei-zeng Huang, Xia Yuan
          </td>
          <td>2025-02-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97569fe84c28d5e70fd4d5d44460cfdc4508b03d" target='_blank'>
              Quantifying cell divisions along evolutionary lineages in cancer.
              </a>
            </td>
          <td>
            Martin Blohmer, David M Cheek, Wei-Ting Hung, Maria Kessler, Foivos Chatzidimitriou, Jiahe Wang, William Hung, I-Hsiu Lee, Alexander N Gorelick, Emma Ce Wassenaar, Ching-Yeuh Yang, Yi-Chen Yeh, H. Ho, Dorothee Speiser, Maria M Karsten, Michael Lanuti, Sara I Pai, O. Kranenburg, J. Lennerz, Teh-Ying Chou, Matthias Kloor, Kamila Naxerova
          </td>
          <td>2025-02-04</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>55</td>
        </tr>

        <tr id="PURPOSE
Digital proteomic profiling was performed to identify spatial context in relationship to patient response and survival after anti-PD-1 therapy in metastatic colorectal cancer (CRC).


EXPERIMENTAL DESIGN
Primary CRCs with deficient mismatch repair (d-MMR) from patients treated with anti-PD-1 antibodies were analyzed (N=30) using Digital Spatial Profiling (GeoMx® nCounter). At the invasive margin, 71 proteins were profiled in 10 regions of interest /slide that were segmented into 3 compartments labeled with pan-cytokeratin (PanCK; epithelia), CD45 (stromal cells) and SYTO13 (nuclei). In an independent cohort (n=13), DSP data and single cell transcriptomic data were analyzed. Differential protein abundance, after Benjamini-Hochberg correction, was examined by response and progression-free survival (PFS) using multivariable Cox regression.


RESULTS
Protein abundance varied significantly between epithelial and stromal compartments. Nonresponders (NR) to anti-PD-1 showed higher fibronectin and smooth muscle actin (SMA) abundance in the epithelial compartment that were associated with significantly shorter PFS (HRadj: 6.49 and 4.52, respectively; p < 0.05). In CD45+ stroma, increased expression of proteins related to T-cells (CD3, CD4), NK cells (CD56), antigen presentation (CD40), immune activation (CD27, ICOS), and apoptosis (GZMA) were found in responders (vs NR) to anti-PD-1; each marker was significantly associated with longer patient PFS (0.02 <HRadj < 0.17; p < 0.05). In a separate cohort, consistent results by compartment were found for fibronectin and CD56. Gene expression data revealed that fibronectin and SMA were primarily derived from cancer-associated fibroblasts.


CONCLUSIONS
Spatially resolved protein profiles within microenvironments of d-MMR CRCs can influence patient response and survival after anti-PD-1 highlighting their potential clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7e12e22717ab1e00830f945fd4f97470fd6160b" target='_blank'>
              Spatially resolved, multi-region proteomics for prediction of immunotherapy outcome in deficient mismatch repair metastatic colorectal cancer.
              </a>
            </td>
          <td>
            B. Saberzadeh-Ardestani, Zhenglong Liu, Mariam I Stein, Will A Sherman, Christy E Trussoni, Charles W Abbott, Dongyao Yan, Skyler Smith, K. Shanmugam, Rondell P. Graham, Alos Diallo, Joshua J. Levy, Tamas Ordog, F.A. Sinicrope
          </td>
          <td>2025-02-19</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6682bfe465d2269cc4df5f7beb19176975b61b47" target='_blank'>
              ELK3-CYFIP2 axis-mediated actin remodeling modulates metastasis and natural killer cell responses in triple-negative breast cancer
              </a>
            </td>
          <td>
            Seung Hee Choi, H. Jang, J. Park, Ki Seo Ryu, Eunchong Maeng, Seohyun Cho, Hail Park, Hae-Yun Jung, Kyung-Soon Park
          </td>
          <td>2025-02-10</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Adenocarcinoma of the esophagogastric junction (AEGJ) is a highly aggressive tumor that frequently metastasizes to the liver. Understanding the cellular and molecular mechanisms that drive this process is essential for developing effective therapies. Methods We employed single-cell RNA sequencing to analyze the tumor heterogeneity and microenvironmental landscape in patients with AEGJ liver metastases. This approach enabled us to characterize the diverse cell populations involved in the liver metastatic process. Results Our analysis revealed a significant involvement of fibroblasts and mural cells in AEGJ liver metastasis. We identified a specific fibroblast type in AEGJ liver metastasis and observed distinct gene expression patterns between adenocarcinoma of the esophagogastric junction and other stomach adenocarcinomas. Our study demonstrated high expression of the SFRP2 gene in pericyte cells during the liver metastasis of AEGJ. The incorporation of GEO, TCGA, and immunofluorescence staining of SFRP2 expression enhanced our study. High expression of SFRP2 in pericytes may influence vascular stability and angiogenesis through the Wnt pathway. Conclusion Our study provides novel insights into the cellular interactions and molecular mechanisms that underlie AEGJ liver metastasis. Targeting the identified subtype of fibroblasts or influencing SFRP2 gene expression in pericytes may offer new therapeutic strategies for combating this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76cf1003bffe0097e28dee22754d6f4e46b48db" target='_blank'>
              Integrated single-cell RNA sequencing reveals the tumor heterogeneity and microenvironment landscape during liver metastasis in adenocarcinoma of esophagogastric junction
              </a>
            </td>
          <td>
            Junrui Xu, Ussama Sadiq, Wangruizhi Zhao, Hengbo Xia, Yiwei Liu, Renquan Zhang, Aman Xu
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5e4b3c71ff4758983a38793023cdc6005fc52e5" target='_blank'>
              Tumor-infiltrating myeloid cells; mechanisms, functional significance, and targeting in cancer therapy.
              </a>
            </td>
          <td>
            F. Toghraie, Maryam Bayat, Mahsa Sadat Hosseini, Amin Ramezani
          </td>
          <td>2025-02-25</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f1f8dfa9377169d8a09f717c0d18ed8c6b2867" target='_blank'>
              Prospective validation of ORACLE, a clonal expression biomarker associated with survival of patients with lung adenocarcinoma
              </a>
            </td>
          <td>
            D. Biswas, Yun-Hsin Liu, Javier Herrero, Yin Wu, David A Moore, Takahiro Karasaki, Kristiana Grigoriadis, Wei-Ting Lu, S. Veeriah, C. Naceur-Lombardelli, Neil Magno, S. Ward, A. Frankell, M. Hill, Emma C Colliver, Sophie de Carné Trécesson, Philip East, A. Malhi, Daniel M. Snell, Olga O’Neill, Daniel Leonce, J. Mattsson, A. Lindberg, P. Micke, Judit Moldvay, Z. Megyesfalvi, B. Dome, J. Fillinger, J. Nicod, Julian Downward, Zoltan Szallasi, A. Hackshaw, M. Jamal-Hanjani, N. Kanu, Nicolai J. Birkbak, C. Swanton
          </td>
          <td>2025-01-09</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Cervical cancer is the fourth most common cancer in women globally, and the main cause of the disease has been found to be ongoing HPV infection. Cervical cancer remains the primary cause of cancer-related death despite major improvements in screening and treatment approaches, especially in low- and middle-income nations. Therefore, it is crucial to investigate the tumor microenvironment in advanced cervical cancer in order to identify possible treatment targets. Materials and methods In order to better understand malignant cervical cancer epithelial cells (EPCs), this study used bulk RNA-seq data from UCSC in conjunction with single-cell RNA sequencing data from the ArrayExpress database. After putting quality control procedures into place, cell type identification and clustering analysis using the Seurat software were carried out. To clarify functional pathways, enrichment analysis and differential gene expression were carried out. The CIBERSORT and ESTIMATE R packages were used to evaluate the immune microenvironment characteristics, and univariate and multivariate Cox regression analyses were used to extract prognostic features. Furthermore, assessments of drug sensitivity and functional enrichment were carried out. Results Eight cell types were identified, with EPCs showing high proliferative and stemness features. Five EPC subpopulations were defined, with C1 NNMT+ CAEPCs driving tumor differentiation. A NNMT CAEPCs Risk Score (NCRS) model was developed, revealing a correlation between elevated NCRS scores and adverse patient outcomes characterized by immune evasion. In vitro experiments validated that the prognostic gene PLOD2 significantly enhances proliferation, migration, and invasion of cervical cancer cells. Conclusion This investigation delineated eight cell types and five subpopulations of malignant EPCs in cervical cancer, establishing the C1 NNMT+ CAEPCs as a crucial therapeutic target. The NCRS model demonstrated its prognostic capability, indicating that higher scores are associated with poorer clinical outcomes. The validation of PLOD2 as a prognostic gene highlights its therapeutic potential, underscoring the critical need for integrating immunotherapy and targeted treatment strategies to enhance diagnostic and therapeutic approaches in cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a0fda708b879e8e016206b2e92375962d615799" target='_blank'>
              Single-cell RNA sequencing and immune microenvironment analysis reveal PLOD2-driven malignant transformation in cervical cancer
              </a>
            </td>
          <td>
            Zhiheng Lin, Fengxin Wang, Renwu Yin, Shengnan Li, Yuquan Bai, Baofang Zhang, Chenlin Sui, Hengjie Cao, Dune Su, Lianwei Xu, Honghong Wang
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 10],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>