<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-16 10:03:50 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>103</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a0eaac11594dae27086c42ea3c9abe9629eb38" target='_blank'>
              Spatial multi-omics unveils the monoclonal origin, neuroendocrine plasticity, and microenvironment niches in combined small cell lung cancer
              </a>
            </td>
          <td>
            Zhuo Wang, Qing Luo, Jie Wu, Li Lu, Wenjie Ding, Yichun Zhao, Yongfeng Yu, Ruoran Qiu, Ling Zhu, Xinxing Ouyang, Wendi Xuzhang, Shun Lu, Wei Wei, Qihui Shi, Ziming Li
          </td>
          <td>2026-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/090135f960d5678025e9450059e754bf1debd8ae" target='_blank'>
              Spatial immune hubs defined by conserved activated dendritic cells are remodeled by immunotherapy
              </a>
            </td>
          <td>
            Tomoyuki Minowa, Sunita Keshari, Shan He, Akata Saha, Alexander S. Shavkunov, Josué E. Pineda, Y. Yeo, Bokai Zhu, Vakul Mohanty, Sonali Jindal, Ken Chen, , Linghua Wang, GP Nolan, S. Watowich, Matthew M. Gubin
          </td>
          <td>2026-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1041f2c4ce227782834ad6496b1ab2bc0d63ef0" target='_blank'>
              Detection of Somatic Point Mutations Directly from Spatial Transcriptomics Enables in vivo Spatiotemporal Lineage Tracing
              </a>
            </td>
          <td>
            Zhirui Yang, Mengdie Yao, Qing Yang, Yiheng Du, Jinhong Lu, Xing Wu, Jinran Lin, Zhuoyu Qian, Song Hu, Yonghe Xia, Huan Liu, Qiji Zhou, Xiaoyu Ma, Yan Luo, Wenxuan Fan, Weike Pei, Yuhao Xia, Xiuyan Yu, Jing Luan, Qiaoan Zhang, Yue Zhang, Qin Wang, Jianyang Zeng, Yue Zhang, Wenyu Wu, Yanmei Dou
          </td>
          <td>2026-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e69c4c3aa7ac04b1dca84c51fe6263d52efa060" target='_blank'>
              Breast cancer through the lens of whole transcriptome spatial imaging
              </a>
            </td>
          <td>
            Claire Williams, Yi Cui, Michael Patrick, Giang T Ong, Terence Theisen, Megan Vandenberg, Joseph M. Beechem, Patrick Danaher
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="SUMMARY Spatial omics transforms our understanding of cancer by revealing how tumor cells and the microenvironment are organized, interact, and evolve within tissues. Here, we synthesize advances in spatial technologies that map tumor ecosystems with unprecedented fidelity. We highlighted analytical breakthroughs—including multimodal integration and emerging spatial foundation models—that resolve functional niches and spatial communities, converting spatial patterns into mechanistic insights. We summarize how spatially organized features, from immune hubs to microbiota and neural interfaces, shape tumor evolution and clinical outcomes. We then outline how spatial approaches illuminate precancer biology, metastatic adaptation, and therapy response. Bridging discovery and translation, we provide a practical roadmap for incorporating spatial readouts into clinically oriented study design. We conclude by discussing persistent challenges in standardization and scalability and how high-plex spatial discoveries may be distilled into scalable, AIenabled, clinically deployable assays, positioning spatial omics as a cornerstone of next-generation predictive and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6fb3dcd04b45926dce1096c02f6d3fedd6c13cb" target='_blank'>
              Spatial omics at the forefront: emerging technologies, analytical innovations, and clinical applications
              </a>
            </td>
          <td>
            Yunhe Liu, Yibo Dai, Linghua Wang
          </td>
          <td>2025-12-01</td>
          <td>Cancer cell</td>
          <td>3</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad9383618813c7fb20bfa893bf5368770766ebe" target='_blank'>
              Analysis of intratumoral immune heterogeneity reveals spatial organisation of immunosuppression in breast cancer
              </a>
            </td>
          <td>
            Christina Metoikidou, P. Bonté, Vadim K. Karnaukhov, Laëtitia Lesage, L. Djerroudi, André Nicolas, P. Sirven, A. Vincent-Salomon, Olivier Lantz, Emanuela Romano, Sebastián Amigorena
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d63cfbf90a2e544c2d24c8ab605e5d44e8ba24" target='_blank'>
              A Pan-Cancer Single-Cell Compendium of Intratumoural Heterogeneity
              </a>
            </td>
          <td>
            Ido Nofech-Mozes, Ashton Cook, Tom W. Ouellette, Sara Hafezi-Bakhtiari, Philip Awadalla, Sagi Abelson
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are the main constituents of the tumor microenvironment. Several studies have delineated CAF heterogeneity in different types of tumors, however, it is still unknown how the distinct CAF transcriptional profiles are established during tumor progression. We reanalyzed a previously published single-cell RNA-sequencing dataset of MMTV-PyMT tumors at higher resolution using Seurat , and CytoTRACE to characterize CAF subtypes and their differentiation states. Wilcoxon rank sum test was applied for differential gene expression. Multiplex immunostaining (Akoya PhenoImager HT) was performed on 38 murine mammary tumors from MMTV-PyMT mice to identify the distinct CAF subtypes. Whole-slide imaging and spatial analysis were conducted using QuPath and Cellpose , followed by neighborhood clustering and interaction mapping with CytoMAP . Cellular distances from CAFs to immune, tumor, and endothelial cells were quantified using SPIAT and Wilcoxon tests for comparisons. In parallel, human spatial transcriptomics data from the 10X Genomics Xenium platform were integrated for cross-species validation. Here, by single cell RNA-sequencing and multiplex immunostaining, we identify six CAF substates. Spatial analysis on immunostained murine mammary tumors and human spatial transcriptomics data outlined temporal changes in stromal composition and the existence of distinct functional niches enriched with different CAF substates. Immunomodulatory CAFs co-localized with immune cells while myofibroblastic CAFs formed a shield around the tumor core, thus preventing immune infiltration. Our work supports the idea that distinct spatial locations dictate different CAF transcriptional programs. Targeting specific functional niches will ultimately hinder tumor progression by inhibiting signaling between distinct CAF substates and the surrounding tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d802e2b1323bfcdbf953412180c241803eb33a" target='_blank'>
              Spatial and temporal dynamics of cancer-associated fibroblast niches in breast cancer
              </a>
            </td>
          <td>
            Jessica Pantaleo, J. Sjölund, P. Bolivar, M. Bocci, B. Phung, Maria Malmberg, G. Jönsson, Kristian Pietras
          </td>
          <td>2026-01-11</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Deciphering the spatial organization of cell states is fundamental for understanding development, tissue homeostasis and disease. Emerging advances in spatial transcriptomic profiling techniques allow transcript localization but face limitations in unambiguous cell state assignments due to cellular boundary inference, low gene detection and prohibitive cost. Here, a method, Stamp-seq, is developed that leverages custom-fabricated high-density DNA sequencing chips to label single nuclei with restriction enzyme-cleavable spatial barcodes. Stamp-seq spatial barcodes are distributed at a density of 1.6 μm on the chip, allowing for single physical cell resolution with precise subtype classification and spatial mapping (with an average 4 μm localization error) and reduced cost. We utilize Stamp-seq to delineate chemoimmunotherapy-responsive cellular ecosystems in non-small cell lung carcinoma, including a distinct IGHG1+ plasma cell-enriched community. Through a novel application of Stamp-seq to spatially resolve BCR clonotypes, we elucidate the spatiotemporal trajectory of treatment-potentiating IGHG1+ plasma cells, which originate from tertiary lymphoid structures (TLSs) or the vasculature, migrate through antigen-presenting CAF (apCAF)-enriched survival niches, and ultimately contact tumor cells. We highlight the power of spatial cellular subtyping and molecular tracking using Stamp-seq and suggest that the IGHG1+ plasma cell niche is a better prognostic biomarker for the chemoimmunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2465810bf851b9f1dce318f6f488baa73c576a5c" target='_blank'>
              Integrating single-nucleus barcoding with spatial transcriptomics via Stamp-seq to reveal immunotherapy response-enhancing functional modules in NSCLC
              </a>
            </td>
          <td>
            Yitong Pan, Huan Yan, Jinhuan Han, Rui Wu, Caiming Xu, Guang Lei, Xingyong Ma, Ying Guan, Zhao Li, Junyuan Deng, Keyu Li, Qingquan Wei, Guangxin Zhang, Lei Liu, A. Goel, Zhou Yang, Shaozhuo Jiao, Yongchang Zhang, Chenxi Tian
          </td>
          <td>2026-02-05</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7a3ed4eb95927059b414292cf9cce743a10321" target='_blank'>
              Spatiotemporal Mapping of Tertiary Lymphoid Structure Heterogeneity Shapes Immune Niches and Clinical Outcomes in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Liaoliao Gao, Jie Mei, Libing Hong, Yuzhi Jin, Jinlin Cheng, Xuqi Sun, Chuan Liu, Bin Li, Xiaomeng Dai, Bo Lin, Yajie Sun, Peng Zhao, Minshan Chen, Rongping Guo, Shan Xin, Jingping Yun, Inmaculada Martínez-Reyes, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f84aada80c780deec8ebaf81d914dc52c85820f2" target='_blank'>
              HEDeST: An Integrative Approach to Enhance Spatial Transcriptomic Deconvolution with Histology
              </a>
            </td>
          <td>
            Luca Gortana, Loïc Chadoutaud, Raphael Bourgade, Emmanuel Barillot, Thomas Walter
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: In contrast to most solid tumors, high immune cell infiltration in clear cell renal cell carcinoma (ccRCC) is associated with poor patient prognosis. The biological mechanisms underlying this paradox remain unclear, particularly regarding tumor cell-microenvironment interactions promoting local invasion and recurrence. This study aimed to identify spatially resolved tumor, immune, and stromal features that define aggressive phenotypes in localized ccRCC. Methods: Multiplex immunofluorescence was performed using a 33-marker panel on 1,728 multi-region tissue cores from 435 surgically treated patients with localized ccRCC. Samples systematically included tumor centers, invasive borders, and adjacent benign tissue. Single-cell analyses quantified immune, stromal, endothelial, and epithelial cell populations within their spatial context. Results: Spatially resolved profiling uncovered a highly aggressive tumor subtype distinguished by fibroblast activation protein (FAP) expression on tumor epithelial cells, a marker typically associated with stromal cells. Tumor-cell-specific FAP expression characterized an epithelial-to-mesenchymal transition (EMT)-like state and was spatially associated with profound immunosuppression, marked by enrichment of regulatory T cells, exhausted CD8+ T cells, and M2-like macrophages, particularly at the invasive border. Tumor-cell FAP promoted invasion and independently predicted significantly poorer recurrence-free survival (RFS), even in early-stage disease (multivariable Cox p = 0.022 for pT1-2), surpassing established biomarkers such as PD-L1 in capturing aggressive biological features. Conclusions: Tumor epithelial FAP expression identifies an aggressive, immune-rich subtype of localized ccRCC, integrating EMT with spatially organized immunosuppression. These findings establish tumor-cell FAP as a promising biomarker with substantial translational potential for patient risk stratification, targeted imaging (FAPI-PET), and FAP-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42470b1512d22d2033f3b6dc6829a3e316db8519" target='_blank'>
              Tumor cell FAP orchestrates EMT and immune suppression in aggressive localized ccRCC
              </a>
            </td>
          <td>
            T. Pellinen, L. Luomala, K. Mattila, A. Hemmes, K. Välimäki, Mariliina Arjama, Oscar Brück, L. Paavolainen, Elisa Kankkunen, H. Nísen, P. Järvinen, Leticia Castillon, Sakari Vanharanta, P. Vainio, Olli Kallioniemi, Panu M. Jaakkola, T. Mirtti
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The tumor immune microenvironment (TIME) is closely involved in tumor initiation, malignant progression, immune escape, and response to immunotherapy. With the continued development of high-throughput sequencing technologies, transcriptomic approaches have become essential for examining the cellular and molecular features of the TIME. Bulk RNA sequencing offers tissue-level gene expression profiles and allows the estimation of immune cell composition through computational deconvolution. Single-cell RNA sequencing provides finer resolution, revealing cellular heterogeneity, lineage relationships, and functional states. Spatial transcriptomics (ST) retains the native anatomical context, making it possible to localize gene expression patterns and cell–cell interactions within intact tissues. These approaches, when considered together, have shifted TIME research from averaged measurements toward a more detailed and mechanistic understanding. This review summarizes the principles, applications and limitations of bulk, single-cell and spatial transcriptomic methods, highlighting emerging strategies for integrative analysis. Such multi-scale frameworks are increasingly important for studying immune dynamics and may contribute to the development of more precise biotechnological and immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a95188583167d2d5ed84b42177b48c6d850371a9" target='_blank'>
              Multi-Scale Transcriptomics Redefining the Tumor Immune Microenvironment
              </a>
            </td>
          <td>
            Jing Sun, Yingxue Xiao, Lingling Xie, Dan Qin, Yue Zou, Yingying Liu, Yitong Zhai, Minyi Zhang, Tong Li, Youjin Hao, Bo Li
          </td>
          <td>2026-01-15</td>
          <td>BioTech</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5486f6aaee8639d9484243cb88e9c2a7fbf4ba3" target='_blank'>
              Cancer-myeloid cell invasive program in pediatric-type diffuse high-grade glioma
              </a>
            </td>
          <td>
            C. Ruiz-Moreno, R. Collot, T. J. van den Broek, E. J. Wehrens, N. Bessler, Gitanjali Dharmadhikari, B. T. Te Pas, Ignacio L Ibarra, D. Metselaar, M. Kranendonk, E. Hoving, J. van der Lugt, F. Calkoen, A. Zomer, F. Theis, E. Hulleman, D. V. van Vuurden, A. Rios, H. Stunnenberg
          </td>
          <td>2026-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Cervical cancer (CC), a leading cause of cancer-related deaths among women worldwide, is primarily driven by high-risk human papillomavirus (HPV) infections and comprises two major histological subtypes: adenocarcinoma (AC) and squamous cell carcinoma (SCC). Despite advances in prevention and treatment, the molecular and cellular heterogeneity of these subtypes poses significant challenges to achieving optimal clinical outcomes. Here, we integrate single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) to dissect the cellular and spatial heterogeneity of AC and SCC, uncovering distinct tumor microenvironment (TME) dynamics that underlie their divergent clinical behaviors. Our scRNA-seq analysis reveals that AC is enriched in epithelial cells, while SCC exhibits a more immunogenic TME with elevated plasma cells and NK/T cells. Spatial transcriptomics further highlights robust interactions between CD8 + T cells and epithelial subtypes in SCC, contrasting with the stromal-rich, immune-cold phenotype of AC. We identify subtype-specific immune and stromal features, including ICOS+ Tregs, IDO1+ cancer-associated fibroblasts (CAFs), and PLVAP+ endothelial cells, which may drive immune evasion, angiogenesis, and metastasis. These findings provide a comprehensive framework for understanding CC heterogeneity and offer actionable insights for developing subtype-specific therapeutic strategies, such as combining immune checkpoint inhibitors with stromal-targeting agents. This study underscores the potential of spatial multi-omics technologies to advance precision oncology and improve outcomes for cervical cancer patients. Single-cell and spatial transcriptomics uncovered a stark contrast in the TMEs of cervical cancer subtypes: squamous cell carcinoma is more immunogenic, whereas adenocarcinoma is stromal-rich and immune-cold.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b0c553f54a8eda4d3dc1abe42f3db69a41cfe1" target='_blank'>
              Single-cell and spatial transcriptomic profiling reveals distinct tumor microenvironment dynamics in cervical adenocarcinoma and squamous cell carcinoma
              </a>
            </td>
          <td>
            Yaqin Wu, Bei Wei, Zhentong Wei, Hongyan Wang, Yunyun Liu, Dongdong Xu, Xiaokang Li, Liyuan Guo, Huaiwu Lu, Huiling Shang
          </td>
          <td>2025-12-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba1256bc5be1c15c5a52165e8dbea8c1e3c75014" target='_blank'>
              Integrative Single-cell and Spatial Transcriptomic Analysis of Osteosarcoma Reveals Conserved and Distinct Ecosystems Across Sites and Species
              </a>
            </td>
          <td>
            Yogesh Budhathoki, Matthew V. Cannon, T. A. McEachron, Anand G. Patel, Matthew J Gust, Jaime F Modiano, D.T. Ammons, Kathryn E Cronise, M. Macchietto, A. Sarver, Ruben Dries, B. Gryder, Daniel P. Regan, Heather L. Gardner, Ryan D. Roberts
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3333bb3dc7aa46ec7dea00023d761c89be2478c2" target='_blank'>
              Large-scale single-cell analysis and in silico perturbation reveal dynamic evolution of HCC: from initiation to therapeutic targeting.
              </a>
            </td>
          <td>
            Peng Xia, Shuang Si, Daosen Fu, Luyao Liu, Dongliang Yang, Yanan Guo, Yixiao Tian, Pengfei Ji, Xinyi Yuan, Yingxia Tian, Rong Shen, Degui Wang
          </td>
          <td>2026-01-30</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/678343a85aff9663bdb1c0682309c95b1ea357b4" target='_blank'>
              Impact of single-cell RNA reference selection for the deconvolution of breast cancer spatial transcriptomics datasets
              </a>
            </td>
          <td>
            Stefan Altendorfer, Scott J. Walker, C. Daub
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/342dd98c58335efcd7b4ec06f157649f394ee9d4" target='_blank'>
              Clinical feasibility of spatial transcriptomics using discarded tissue from diagnostic breast biopsies
              </a>
            </td>
          <td>
            Jeffrey P Sheridan, Quynh Sun, Jiaji G Chen, Yibing Wei, Veronica Jarzabek, Anirudh Jaishankar, Iqra Amin, L. Sultan, Thea Nalan, Sarah B. Mueller, Eric J. Burks, Lubna H. Suaiti, Christopher M. Heaphy, Naomi Y. Ko, Ruben Dries
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafddf4eb7b4f74e9d7699b13e08df2afba97c1e" target='_blank'>
              Pan-cancer tumor classification by a holistic tumor microenvironment atlas
              </a>
            </td>
          <td>
            Shishang Qin, Xiao Du, Jinhu Li, Nan Jiang, Tian Diao, Yufei Bo, Qinhang Gao, Liangtao Zheng, Xinnan Ling, Qianqian Gao, Xiangjie Li, Sen Gao, Fei Tang, Wenjie Zhang, Chenwei Li, Peihong Fang, Linna Zhu, Dongfang Wang, Zemin Zhang
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Understanding the spatial organization of individual cell types within tissue and how this organization is disrupted in disease, is a central question in biology and medicine. Hematoxylin and eosin-stained slides are widely available and provide detailed morphological context, while spatial gene expression profiling offers complementary molecular insights, though it remains costly and limited in accessibility. Predicting gene expression directly from histological images is therefore an attractive goal. However, existing approaches typically rely on small image patches, limiting resolution and the ability to capture fine-grained morphological variation. Here, we introduce a deep learning approach that predicts single-cell gene expression from morphology, matching patch-based methods on spot level prediction tasks. The model recovers biologically meaningful expression patterns across two cancer datasets and distinguishes fine cell populations. This approach enables molecular-level interpretation of standard histological slides at scale, offering new opportunities to study tissue organization and cellular diversity in health and disease. Predicting gene expression from H&E slides is a cost-effective alternative to Spatial Transcriptomics for clinical use. Here the authors introduce a Multiple Instance Learning approach to infer single-cell expression, unlike prior methods which operate at coarser patch level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d903aacfbf3a6dfc2c450d8cb95280e66a5321f" target='_blank'>
              sCellST predicts single-cell gene expression from H& E images
              </a>
            </td>
          <td>
            Loïc Chadoutaud, Marvin Lerousseau, Daniel Herrero-Saboya, Julian Ostermaier, Jacqueline Fontugne, Emmanuel Barillot, Thomas Walter
          </td>
          <td>2026-01-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e6a0c7866dff4b99709467cf25215e8947b79ac" target='_blank'>
              Coevolution of neoplastic and non-neoplastic reactive astrocyte states converges on mesenchymal-like and injury-response programs during murine glioblastoma progression and post-radiotherapy recurrence
              </a>
            </td>
          <td>
            D. Lindgren, R. Rosberg, K. Smolag, E. Johansson, D. Manou, J. Sjölund, S. Braun, B. Phung, K. Harbst, P. Jeannot, Maria Malmberg, C. Ceberg, G. Jönsson, Kristian Pietras, A. Pietras
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be57704ac8a67d3c2570360452cb391401b265a" target='_blank'>
              A Comprehensive Treatment-Induced Resistance Atlas of Glioblastoma Reveals a Fibrotic Niche Shielding the Tumor from Immunotherapy
              </a>
            </td>
          <td>
            Fei Wang, Run Huang, Hongyi Ling, Yuhan Bai, Chen Yang, Guozhen Zhao, Xin Wu, Wenqian Cao, Yue Lu, Yining Zhang, Deng-Feng Lu, Youjia Qiu, Juyi Zhang, Bixi Gao, Chao Ma, Hanhan Dang, Yulian Zhang, Yanbo Yang, Ting Sun, Zhouqing Chen, Zhong Wang
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Understanding how prostate cancers evolve toward treatment resistance requires experimental systems that both recreate the phenotypic diversity of the disease and resolve lineage dynamics at high resolution. We developed complementary platforms to meet these challenges. The Prostate Mouse Platform (ProMPt) is a genetically scalable organoid- and allograft-based system encompassing over 150 combinatorial genotypes, enabling reconstruction of the principal phenotypic states of prostate cancer, from androgen-responsive tumors to castration-resistant double-negative, hybrid/mixed, and neuroendocrine-like variants. Through multi-omic and functional analyses, ProMPt models capture the context-specific vulnerabilities that arise during adaptation to androgen deprivation or growth-factor loss. To map and track this evolving phenoscape, we designed XCODE, a universal, multiplatform barcoding technology that enables clonal tracing across sequencing-, cytometry-, and imaging-based modalities (CyTOF, Phenocycler, Xenium). XCODE integrates lineage information across bulk, single-cell, and spatial layers, linking histology, clonal dynamics, phenotype, and microenvironmental context. Together, ProMPt and XCODE provide a powerful preclinical framework to reconstruct, resolve, and perturb tumor evolution within a controlled yet clinically relevant landscape. This integrated approach reveals how defined genetic combinations reshape lineage plasticity, signaling networks, and therapeutic responses, exposing new intervention points to prevent or redirect adaptive evolution in advanced prostate cancer.



 Marco Bezzi. From reconstruction to resolution: platform approaches to engineer the prostate cancer phenoscape and map its evolution [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A002.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3245085c50a80831901ee082ba85bf39b096b337" target='_blank'>
              Abstract A002: From reconstruction to resolution: platform approaches to engineer the prostate cancer phenoscape and map its evolution
              </a>
            </td>
          <td>
            Marco Bezzi
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Although well-differentiated thyroid carcinoma (WDTC) is characterized by a robust treatment response, aggressive subtypes, such as anaplastic thyroid carcinoma (ATC), remain highly lethal. To understand thyroid cancer evolution in both children and adults, we analyzed single-cell transcriptomes of 423,733 cells from 81 samples and spatially resolved key tumor and microenvironment populations across 28 tumors with spatial transcriptomics, including rare and unique composite WDTC/ATC tumors and pediatric diffuse sclerosing thyroid carcinomas. Additionally, we identified gene signatures of stromal cell populations in 5 large thyroid cancer bulk RNA-sequencing cohorts. Through this multi-institutional effort, we defined a population of POSTN+ myofibroblast cancer-associated fibroblasts (myCAFs) that are intimately associated with invasive tumor cells and correlate with poor prognosis, lymph node metastasis, and disease progression in thyroid carcinoma. We also revealed a population of inflammatory CAFs that are distant to tumor cells and are found in the inflammatory stromal microenvironment of autoimmune thyroiditis. Together, our study provides spatial profiling of thyroid cancer evolution in samples with mixed WDTC/ATC histopathology and identifies a prognostic myCAF subtype with potential clinical utility in predicting aggressive disease in both children and adults.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9674219bbd66cbdd5e33b2f92b59cb54d20e94cd" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations
              </a>
            </td>
          <td>
            M. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey P. Caroland, Megan L. Tigue, H. Hartmann, J. Gallant, Courtney J. Phifer, Andres A Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H. Belcher, K. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2026-01-09</td>
          <td>JCI Insight</td>
          <td>3</td>
          <td>47</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma is heterogeneous, with low tumor purity, a prominent microenvironment, and complex architecture, which preclude the identification of shared tumor-intrinsic and stromal biology within and across patients. We overcame these challenges by achieving necessary resolution and context through the application of complementary genomics, pathology, and machine-learning approaches to characterize primary untreated tumors from 39 patients. We captured 340,000 spatial low-bulk and 530,000 spatial single-cell transcriptomes and observed a spectrum of classical-to-basal tumor subtypes present within all patients. We found that each subtype has distinct regulators, stromal neighborhoods, microenvironment, extracellular matrix, and histology corresponding to multiple immunosuppressive and therapy resistance mechanisms. We defined key tumor heterogeneity features, including the presence of mixed KRAS mutations and tertiary lymphoid structures, identifying biomarkers that distinguish the latter from lymph nodes. Lastly, by leveraging patient, cell, and mouse data, we determined which aspects of tumor biology are recapitulated in bulk datasets and reductionist models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dfa5ca299bf4b962d7809782c25bc48f5abb0f7" target='_blank'>
              In situ multi-modal characterization of pancreatic cancer reveals tumor cell identity as a defining factor of the surrounding microenvironment.
              </a>
            </td>
          <td>
            Anna Lyubetskaya, Brian Rabe, Andrew Kavran, Yulong Bai, Andrew Fisher, Alba Font-Tello, Anne Lewin, Hannah Pliner, Yunfan Fan, Lauren Giampapa, Yelena Cheng, Chao Dai, Ruifeng Hu, Tom Lila, Alexandre P. Alloy, Mike Mason, Constance Brett, Todd Brett, Fayaz Seifuddin, Steven Vasquez Grinnell, S. Stahlschmidt, Yilin Zhao, R. Golhar, Isaac Neuhaus, Daniel Carrera, Carlos Rios, P. Kar, Abhishek Shukla, R. Bashford-Rogers, Matthew J. Meyer, Enas Abu Shah, Lara Heij, Shivan Sivakumar, J. Trillo-Tinoco, Benjamin Chen, Konstantinos J. Mavrakis, Eugene Drokhlyansky
          </td>
          <td>2026-01-01</td>
          <td>Cell reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27e17b141d07dfd3357f3ee906c1f3a2792dd3c" target='_blank'>
              SigDyn: single-cell mutational signature dynamics
              </a>
            </td>
          <td>
            Sara B. Costa, Joana P. Gonçalves
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ee6608293723ba24e369164c3802a52bef8a361" target='_blank'>
              Redefining the topology of the human bone marrow using augmented spatial transcriptomic analysis
              </a>
            </td>
          <td>
            Rosalin Cooper, Emily Thomas, Muhammad Dawood, H. Ryou, Anna M. Sozanska, C. Pescia, Oliver McCallion, Muskaan Gupta, Renuka Teague, J. Hester, F. Issa, Dan J Woodcock, B. Psaila, Adam J. Mead, Jens Rittscher, D. Royston
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18e77f40b4649c94b742354f1776a0273d7ae6d5" target='_blank'>
              A Hallmark-Integrated, Agent-Based Framework for Intratumor Heterogeneity in Melanoma Evolution
              </a>
            </td>
          <td>
            Khola Jamshad, Trachette L. Jackson
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f98b24512687fbcc11862186cf26ac85b52f95e1" target='_blank'>
              Benchmarking cell type deconvolution in spatial transcriptomics and application to cancer immunotherapy
              </a>
            </td>
          <td>
            Amanda Sun, Tamjeed Azad, Chrysothemis C. Brown, Yuri Pritykin
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67c0bdeec747ad1c41e7a933728f867f5f704e68" target='_blank'>
              Spatial pharmaco-multiomics reveals drug distribution, metabolic niches, and spatially constrained resistance in medulloblastoma
              </a>
            </td>
          <td>
            Tuan Vo, Cedric Cui, Aitor Benedicto, Yibi Chen, Andrew Causer, Xiao Tan, Eun Ju Kim, Hani Vu, Albert Xiong, Mark Hodson, Yak Deng, Ana Maria Romero Jimenez, Ning Liu, Brett Hamilton, Thomas Robertson, Laura A. Genovesi, Brandon J. Wainwright, Jenny Fung, John Lee, Quan Nguyen
          </td>
          <td>2026-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb72dd83254c43d1127b19ea69b553e5fd51c4d" target='_blank'>
              TMPO promotes cellular dissemination and metastasis in circulating tumor cells
              </a>
            </td>
          <td>
            A. Giacobbe, Aleksandar Z Obradovic, Jinqiu Lu, Soonbum Park, Carlos Pedraz-Valdunciel, G. Fanelli, Aunika Zheng, Jaime Y Kim, Maya Stella Dixon, Jung Seung Nam, Florencia Picech, Caroline J. Laplaca, R. Virk, Matteo Di Bernardo, Alexander Chui, J. M. Arriaga, Stephanie N Afari, Francisca Nunes de Almeida, Min Zou, Helen Garcia, Brian D. Robinson, Hongshan Guo, S. Maheswaran, D. Haber, D. Miyamoto, D. Nanus, S. Tagawa, Tian Zheng, Massimo Loda, I. I. C. Chio, Michael M. Shen, P. Giannakakou, Andrea Califano, Peter A. Sims, Cory Abate-Shen
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71c46aa519f5755b995d1d19efbf9695d85981f7" target='_blank'>
              Molecular Profiling and High-Content Drug Screening of Metastatic Colorectal Cancer Organoids Reveal Evolutionary Mechanisms of Pan-KRAS Inhibitor Resistance
              </a>
            </td>
          <td>
            Gizem Damla Yalcin, Kübra Yılmaz, Raffaella Klima, O. Kurtulan, Duriye Ozer Turkay, O. Aktepe, A. B. Doğrul, Cenk Sokmensuer, V. Oter, E. B. Bostancı, Javier Fernandez-Mateos, Suayib Yalcin, Luca Braga, Ahmet Acar
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), a highly lethal disease often developing in the context of chronic liver disease, is marked by high relapse rates and low 5‐year survival. To investigate the multicellular ecosystem and molecular features of hepatocarcinogenesis and progression, a comprehensive analysis integrating single‐cell and spatial transcriptomics was conducted. Significant alterations were observed in various cell types, notably an increase in liver parenchymal and endothelial cells, which play critical roles in tissue repair and angiogenesis. Scissor+ cells, predominantly hepatocytes, were identified as potentially evolving into malignant cells with poor prognosis and enhanced immune evasion capabilities. Key genes associated with Scissor+ cells exhibited potential as cancer prognostic markers. Additionally, the disruption of cellular communication networks during malignant progression revealed the evolution of hepatocytes into cancerous phenotypes. The enhanced signaling pathways, such as MDK‐SDC4 and COL4A‐SDC, were identified as pivotal in driving the malignant transformation of hepatocytes. This transformative process, wherein Scissor+ cells acquire malignant features, highlights novel mechanisms of cancer progression. These findings provide deeper insights into hepatocarcinogenesis and offer potential avenues for targeted and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea05c9d911e5acfb02ab472396d64afacfb09aa" target='_blank'>
              Spatiotemporal Evolution of Malignant Hepatocyte Clones Unveils Immune Evasion Features and Therapeutic Vulnerabilities in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Shizheng Xiong, Dekang Ren, Jiaming Jin, Linjie Zhao, Chengjun Gong, Jiarui Liu, Wanjie Zheng, Zhiheng He, Haochuan Guo, Li Guo, Tingming Liang
          </td>
          <td>2026-01-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/153f97057ff8573dac73f49cf5c6ecd0096ebd36" target='_blank'>
              Single-cell profiling of synchronous multi-organ metastasis reveals a systemic CD74+ lipid-associated macrophage niche driving polymetastatic breast cancer
              </a>
            </td>
          <td>
            Jacob Insua-Rodríguez, Pascal Naef, Hannah Savage, Isam Adam, Sung Kook Chun, Axel A. Almet, Gautham Prabhakar, Sharon Kwon, Cassandra G. Sweet, Sevilla F. Hennessey, Angela Lincy Prem Antony Samy, Hamad Alshetaiwi, Maren Pein, Sharmila Mallya, Delia F. Tifrea, Declan M. Edwards, Angela Gomez-Arboledas, Zander Esh, Carina Sandoval, Katrina T. Evans, Alex Hsu, Victoria Mai, Ngoc Bao Kim Nguyen, Erika Zagni, Aaron Longworth, Paige V. Halas, Aaron Simon, Cholsoon Jang, A. Mohyeldin, Robert A. Edwards, Qing Nie, Selma Masri, Kai Kessenbrock, Devon A. Lawson
          </td>
          <td>2026-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Neoadjuvant PD-1/PD-L1 blockade yields robust efficacy in advanced cutaneous squamous cell carcinoma (cSCC), yet many patients fail to achieve a complete or major pathologic response. The reasons why some patients experience response but others do not are unclear. Methods We profiled cSCC specimens before, after 1 dose, and after 3–4 doses of PD-1/PD-L1 blockade to uncover resistance mechanisms and predict therapeutic response. In total, 27 patients across three cohorts, including two phase II trials, were studied. We created 1.7 mm tissue-core microarrays and performed single-cell spatial transcriptomics, including spatial clustering, gene-set enrichment, and spatial correlation analyses. Results After profiling all samples, six distinct spatial niches emerged, each differentially enriched in responders versus non-responders. A high antigen presentation niche, B/plasma cell enriched niche, and inflammatory keratinocyte niche were more frequent in responders, whereas proliferative keratinocyte, low antigen presentation myeloid, and fibroblast-rich epithelial–mesenchymal transition niches prevailed in non-responders. Notably, spatial niche profiling on pretreatment samples outperformed PD-L1 status in predicting pathologic response. Each niche displayed unique gene coexpression modules, suggesting niche-specific resistance mechanisms. Individual tumor analyses revealed varied immune evasion strategies, including defective interferon-induced antigen presentation, immunosuppressive myeloid environments, and epithelial–mesenchymal transition. Conclusions Our single-cell spatial transcriptomic approach identifies six spatial niches that predict immunotherapy response better than PD-L1 status using only 1.7 mm tissue cores and may inform the development of biomarkers. Our results further underscore the heterogeneity of resistance mechanisms among cSCC patients, highlighting the need for tailored therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf68ddba765459abfda326b7a403a032b5df3da" target='_blank'>
              Single-cell spatial transcriptomics uncovers niches that govern response to PD-1/PD-L1 blockade in cutaneous squamous cell carcinoma
              </a>
            </td>
          <td>
            Maxwell Y Lee, Alistaire R. Sherman, Luis Martinez Ramirez, Mobeen Rahman, Sabrina Zdravkovic, June Ho Shin, Ivan Stepanek, A. D. Colevas, John B. Sunwoo, V. Divi
          </td>
          <td>2026-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Lung cancer remains a significant global health challenge, with metastatic progression being the leading driver of mortality. Organoid technology provides a tractable, physiologically relevant platform to model key aspects of lung cancer metastasis in vitro. The present review summarized methodologies for constructing and interrogating these models, covering tissue sources, culture modalities, gene editing and in vivo transplantation; applications in studying metastatic mechanisms, drug screening and capturing intra- and intertumoral heterogeneity are also highlighted. Persistent challenges include standardizing derivation and culture conditions, improving preservation of tumor-microenvironmental interactions, expanding immune-competent and vascularized models, and addressing scalability, cost, and regulatory and ethical considerations for clinical translation. Future directions include integrating multi-omics approaches and spatial profiling, leveraging artificial intelligence for image and response analytics, advancing immune-organoid models and establishing shared standards, reference materials and reporting guidelines to enhance reproducibility and clinical impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9831c4c96b7e8c9d326d8d56192940951debd787" target='_blank'>
              Recapitulating lung cancer metastasis in vitro: Advances in organoid models and challenges in clinical translation (Review)
              </a>
            </td>
          <td>
            Jian Jiang, Guomin Dong, Zhiyun Zhang, Xiaoyuan Lei, Fanshu Huang, Suoni Li, Jiequn Ma, Jie Bai, Qian Ge, Hui Wei, Zheng Zhao
          </td>
          <td>2026-01-20</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A fundamental challenge in studying therapy resistance is understanding whether it results from pre-existing cellular states (“priming”) or drug-induced changes (“adaptation”). While lineage barcoding enables retrospective analysis of cells before and after treatment, current methods struggle to efficiently capture rare lineages in single-cell RNA sequencing (scRNA-seq) or isolate multiple specific lineages simultaneously for functional study. To overcome these limitations, we developed CloneSweeper, a multiplexed lineage tracking platform that pools enrichment libraries to isolate or enrich multiple rare lineages. CloneSweeper utilizes a dual-function barcode expressed as both a Cas9 gRNA for live-cell sorting and a 3’ UTR transcript for high-recovery detection in 10x Genomics scRNA-seq. We applied CloneSweeper to a model of BRAF V600E melanoma, where we identified that resistance to targeted therapy emerges from a polyclonal population of rare, pre-existing lineages. By simultaneously targeting and enriching 21 distinct rare lineages prior to treatment, we defined a heritable, primed state characterized by de-differentiation and elevated mesenchymal markers. We demonstrate that these primed cells are not quiescent but instead exhibit upregulated inflammatory and stress response signaling, specifically via the AP-1 and NF-κB1 pathways. CloneSweeper thus provides a robust framework for dissecting the molecular mechanisms of rare biological phenomena through simultaneous, multiplexed lineage isolation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfe1b1a36a95588a97f5cc3024b429ba771c00" target='_blank'>
              Multiplexed enrichment and tracking of lineages with CloneSweeper
              </a>
            </td>
          <td>
            Robert J. Vander Velde, Raymond W. S. Ng, Christopher Coté, Sydney M. Shaffer
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b786943f0e4090faa02f89896c2ed59c657e0d0" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations
              </a>
            </td>
          <td>
            M. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey P. Caroland, Megan L. Tigue, H. Hartmann, Jean-Nicolas Gallant, Courtney J. Phifer, Andres A Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H. Belcher, K. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains clinically heterogeneous. We integrated single-cell and spatial transcriptomics with explainable machine learning to define a lethal tumor axis and establish an interpretable prognostic model. From 141,986 high-quality single cells spanning localized, hormone-sensitive, and castration-resistant PCa, we identified a malignant C4 epithelial subpopulation characterized by high chromosomal instability, androgen receptor and cell-cycle activation, and stemness potential. Spatial mapping further revealed immune-enriched yet suppressive niches, where fibroblasts and myeloid cells coexisted with exhausted lymphocytes, reflecting functional immune imbalance. We benchmarked 101 machine learning pipelines, selecting a Lasso plus PLS-Cox model that achieved strong concordance across independent cohorts. The C4-based risk score independently predicted recurrence-free survival after adjustment for age, Gleason score and T stage, and a nomogram combining this score with clinical variables showed good discrimination. SHAP interpretation highlighted MT1M, PCSK1N, and ACSL3 as major risk-driving features. PCSK1N was progressively upregulated from normal prostate to castration-resistant disease and promoted proliferation, clonogenicity, migration and enzalutamide resistance, while its inhibition sensitized organoids and xenografts to AR-targeted therapy. These findings define a C4-centered lethal tumor axis and provide an explainable, experimentally supported framework for prognostic stratification in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d360e6a196e9de4b8313a49f5e3b66fe3b338c6" target='_blank'>
              Integrative single-cell and spatial transcriptomics with explainable AI reveal lethal prognostic axis in prostate cancer
              </a>
            </td>
          <td>
            Qintao Ge, Zhenda Wang, Yangyun Wang, Tonghui Chu, Zhongyuan Wang, Wenkai Zhu, Youzhao Zhang, Yonghao Chen, Dingwei Ye, Wenhao Xu, Zhongyuan Wang, Mierxiati Abudurexiti
          </td>
          <td>2026-01-06</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The limited availability of tissue samples from rare tumors poses a major barrier to advances in precise diagnosis, prognostic evaluation, and therapeutic research—challenges exemplified by primary central nervous system diffuse large B cell lymphoma (PCNS-DLBCL). Here, we developed a dual-mode microfluidic immunostaining (Dumi) device, which integrates diagnostic and research workflows into a single, automated platform, reducing tissue section consumption by more than 90%. Using just one to two slides, it enables both regional detection of biomarkers for diagnostic subtyping (≤16) and construction of multiplex tumor microenvironment (TME) maps. Joint analysis of multiregion diagnostic biomarkers in the TME map indicates that tumor cell subpopulations defined by specific diagnostic biomarkers can actively shape their in situ microenvironmental niche. Dumi offers an efficient, cost-effective, and multifunctional immunostaining method, overcoming the limitations of scarce tissue resources and providing a clinically accessible solution to the diagnostic and therapeutic challenges with rare tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/242fa7959d0a90b74624ebf05f15aaa2c75a7e96" target='_blank'>
              Dual-mode microfluidic immunostaining device for diagnostic biomarkers detection and tumor microenvironment evaluation
              </a>
            </td>
          <td>
            Yu Zhang, Yanhua Huang, Jing Jia, Xiaoguang Guo, Fuxiu Liu, Bing Shi, Zhenbao Huang
          </td>
          <td>2026-01-16</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a19bfda47de17fb8ef4014da55fa6d5d86101b2f" target='_blank'>
              Addressing complex autofluorescence signatures in solid tissue samples to enhance full spectrum flow cytometry of non-immune cells
              </a>
            </td>
          <td>
            Christina Gkantsinikoudi, Manuela Terranova-Barberio, Neil P Dufton
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Rationale: Esophageal squamous cell carcinoma (ESCC) is a highly aggressive malignancy. The metastasis and poor prognosis of ESCC are closely associated with tumor microenvironment (TME) heterogeneity, which is driven by epithelial-mesenchymal transition (EMT). Clinically, how to diagnose and target EMT progression remains a key challenge for ESCC. Methods: Integration of pathological images and bulk RNA sequencing profiles identified a high-risk subtype exhibiting EMT enrichment and immunosuppression. Single-cell and spatial transcriptomics revealed EMT macrostates and their spatial distribution. The role of CACNA1C in programming malignant phenotype was tested in vitro. A pathological image-based deep learning model successfully predicted the spatial expression distribution of CACNA1C, indicating possible clinical utility. Results: EMT progression comprised three macrostates: the early state (high epithelial and metastatic potential), the stable state (hybrid E/M phenotype and high stemness), and the late state (high mesenchymal and invasive propensity). ITGA3 and ITGB4 antagonistically regulate malignant phenotype in the early state. Notably, suppression of CACNA1C induced transdifferentiation from stable/late-state cells to normal epithelium-like cells. Conclusions: This study provides novel insights into the EMT mechanism in ESCC, proposes an intervention strategy, and emphasizes the promising clinical application of pathological images in EMT assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2bd67040364633d082b64fbfe1d8f04e033d05" target='_blank'>
              Pathogenomic analysis reveals clinically relevant epithelial-mesenchymal plasticity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Ruzhen Chen, Chenyi Xie, Ziyu Ning, Meng Yang, Zezhuo Su, Jiahui Chen, Kunheng Du, Yihuai Hu, Chu Han, Shaojun Zhang, Qingling Zhang, Meng Liu, Zaiyi Liu
          </td>
          <td>2026-01-14</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Integrating single-cell RNA sequencing and spatial transcriptomics is the current imperative to manually explore the landscape of cellular mixtures. Herein, we developed CellMap (https://github.com/liuhong-jia/CellMap), a computational tool that allows spatial transcriptomic spots to be resolved at single-cell resolution. CellMap combines strategies that incorporate the co-linearity of seed genes, the random forest model, and the linear assignment algorithm to achieve optimal assignment of single cells to spatial spots. Using comprehensive benchmarking across various platforms and tissue types, we demonstrated that CellMap outperforms existing methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30a13b481b7fd69e69b6cf039d418f1f27486f06" target='_blank'>
              CellMap: precision mapping of cellular landscape in spatial transcriptomics
              </a>
            </td>
          <td>
            Hongjia Liu, Huamei Li, Amit Sharma, Guoyan Tang, Zongyu Xie, Yunyao Shen, Qiong Li, Chen Gong, Xiao Sun, Kun Luo, Hongde Liu
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Invasive lobular carcinoma (ILC) is the second most common histological subtype of breast cancer and displays distinct clinical and biological behavior compared to breast cancer of no special type. However, current molecular classifications largely overlook its complex spatial organization and tumor microenvironment (TME). Here, we performed spatial transcriptomics on 43 hormone receptor-positive, HER2-negative (HR+/HER2-) ILC tumors with detailed morphological annotation and long-term clinical follow-up. By integrating spatial gene expression with histology and single-cell deconvolution, we characterized the composition and architecture of the TME and revealed high inter- and intratumor heterogeneity. Spatial clustering uncovered cell populations and pathways linked to clinical outcome. We then developed a multimodal classification of ILC by integrating gene expression, morphology, and spatial metrics, identifying four distinct subtypes: normal/stroma-enriched (NSE), proliferative (P), androgen receptor-enriched (ARE), and metabolic/immune-enriched (MIE). These subtypes, collectively termed ILC4TME, reflect the interplay between tumor and microenvironmental features. Gene signatures derived from the spatial data enabled subtype assignment in external bulk RNA-seq and microarray datasets (SCAN-B, METABRIC), revealing reproducible biology and significant associations with survival. In multivariable models, ILC4TME retained prognostic value beyond established gene signatures and clinicopathological variables. Notably, the P subtype was linked to poor prognosis, even in patients treated with endocrine therapy alone, while the NSE subtype was associated with favorable outcomes. Our findings uncover spatial and cellular heterogeneity in ILC that is not captured by existing classification approaches, offering a refined framework for risk stratification and therapeutic targeting based on tumor microenvironment architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/048f8386277c3de4abe3f625fd3318ada9b08489" target='_blank'>
              Spatial transcriptomics reveals tumor microenvironment-driven subtypes of invasive lobular carcinoma.
              </a>
            </td>
          <td>
            M. Serra, Mattia Rediti, L. Collet, F. Lifrange, D. Venet, N. Occelli, A. Papagiannis, D. Vincent, G. Rouas, D. Larsimont, M. Vikkula, F. Duhoux, F. Rothé, C. Sotiriou
          </td>
          <td>2026-02-04</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3083560b7513f7f17aefe85af32208b0613ecab1" target='_blank'>
              Cellular neighborhoods in cancer
              </a>
            </td>
          <td>
            Lichun Ma, Barbara Xiong, Meng Liu, Kai Tan
          </td>
          <td>2026-01-01</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d29acb3217c08dff3be7e7d47281f950232e5a4" target='_blank'>
              Identification of an epigenetically and phenotypically distinct peritumoral glioblastoma cell population linked to inferior patient outcome
              </a>
            </td>
          <td>
            I. Neves, Xi Lu, Veera Jokinen, Francesco Latini, N. Maturi, A. Sundström, Yonglong Dang, Irem Uppman, Tobias Bergström, Pengwei Xing, M. Ryttlefors, Xingqi Chen, F. Swartling, L. Uhrbom
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Gastroenteropancreatic neuroendocrine tumors (GEP-NETs) are clinically heterogeneous malignancies whose biology and microenvironmental organization remain poorly understood. Here, we integrated single-nucleus multiomic (snRNA-seq and snATAC-seq) and spatial transcriptomic profiling across 38 well-differentiated pancreatic (pNET) and small-intestinal (siNET) tumors to define conserved malignant programs, their regulatory circuits, and spatial niches. We observed two conserved malignant cell programs spanning a continuous transcriptional spectrum: a neuronal-like program (si-cNMF1/p-cNMF1), and a secretory neuroendocrine program (si-cNMF2/p-cNMF2). Matched chromatin accessibility profiles uncovered distinct, tissue-specific regulatory networks, including MAX::MYC and MITF transcription factor binding motifs in siNETs versus ISL1 and TFAP4 in pNETs, indicating organ-specific epigenetic control. Spatial transcriptomic analyses revealed that si/p-cNMF1-high regions localized to high cell density, immune-rich tumor areas, whereas si/p-cNMF2-high regions occupied stromal and vascularized niches and co-occured with fibroblast and endothelial compartments enriched for TGFB1-ITGB1, VEGFA-FLT1, and LAMA2-ITGA1 signaling. Across both tumor types, the cNMF2 program was enriched in metastatic lesions and was enrichedfor pro-fibrotic and pro-angiogenic gene signatures. Thus, GEP-NETs are organized along a conserved neuronal-to-secretory axis defined by distinct epigenetic programs and spatially coupled to specific microenvironmental niches. This framework unifies NET heterogeneity across organ sites and identifies pathway-specific, microenvironment-linked vulnerabilities for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f5c183623da01cf434288165ef5499fec974dff" target='_blank'>
              Conserved Neuronal-like and Secretory Programs Define the Spatial Architecture of Gastroenteropancreatic Neuroendocrine Tumors
              </a>
            </td>
          <td>
            Julie Karam, Samantha E. Hoffman, Amanda E. Garza, Dan Y Gui, Hannah I. Hoffman, Breanna M. Titchen, Yutaro Tanaka, Erica M Pimenta, Theodora Pappa, Laura Valderrábano, Kevin Bi, R. Gillani, Lauren Brais, Erin Shannon, Jason L. Hornick, Jihye Park, Jennifer Chan, E. V. Van Allen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the leading causes of cancer-related mortality worldwide. Compared with traditional two-dimensional (2D) models, patient-derived CRC organoids more faithfully preserve the genomic, transcriptomic, and architectural features of primary tumors, making them a powerful intermediate platform bridging basic discovery and clinical translation. Over the past several years, organoid systems have rapidly expanded beyond conventional epithelial-only cultures toward increasingly complex architectures, including immune-organoid co-culture models and mini-colon systems that enable long-term, spatially resolved tracking of tumor evolution. These advanced platforms, combined with high-throughput technologies and clustered regularly interspaced short palindromic repeats (CRISPR)-based functional genomics, have substantially enhanced our ability to dissect CRC mechanisms, identify therapeutic vulnerabilities, and evaluate drug responses in a physiologically relevant context. However, current models still face critical limitations, such as the lack of systemic physiology (e.g., gut–liver or gut–brain axes), limited standardization across platforms, and the need for large-scale, prospective clinical validation. These gaps highlight an urgent need for next-generation platforms and computational frameworks. The development of high-throughput multi-omics, CRISPR-based perturbation, drug screening technologies, and artificial intelligence-driven predictive approaches will offer a promising avenue to address these challenges, accelerating mechanistic studies of CRC, enabling personalized therapy, and facilitating clinical translation. In this perspective, we propose a roadmap for CRC organoid research centered on two major technical pillars: advanced organoid platforms, including immune co-culture and mini-colon systems, and mechanistic investigations leveraging multi-omics and CRISPR-based functional genomics. We then discuss translational applications, such as high-throughput drug screening, and highlight emerging computational and translational strategies that may support future clinical validation and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ed98ccf8eb7ee4178076cc0e4368c14e7e399d9" target='_blank'>
              An Integrative Roadmap for Advancing Colorectal Cancer Organoid
              </a>
            </td>
          <td>
            Youqing Zhu, Ke He, Zhi Shi
          </td>
          <td>2026-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immune dysfunction in cancer is enacted by multiple programs, including tumor cell-intrinsic responses to distinct immune subpopulations. A subset of these immune evasion programs can be systematically recapitulated through direct tumor-immune interactions in vitro. Here, we present an integrated, high-throughput single-cell CRISPR screening framework focused on the protein kinome for mapping the tumor-intrinsic regulation of T cell-driven immune pressure in glioblastoma (GBM). We combine pooled CRISPR interference and activation (CRISPRi/a) with immune-matched NY-ESO-1 antigen-specific allogeneic GBM-T cell co-culture and massively multiplexed single-cell transcriptomics to systematically quantify how genetic perturbation reshapes baseline tumor state and adaptive responses across graded effector-to-target ratios. We further leverage deep generative models for analyzing pooled CRISPR screens to decipher the effects of genetic perturbations on the mechanisms of tumor resistance. This framework resolves distinct modules of immune evasion and survival, including the regulation of the antigen-presentation machinery, interferon/NF-κB signaling, oxidative stress resilience, and checkpoint/cytokine programs, while identifying perturbations that reroute the continuous tumor transcriptional trajectory induced by T cell engagement. A secondary chemical screen in patient-derived GBM cultures identified putative kinase targets of immune evasion phenotypes (e.g., EPHA2 and PDGFRA), whose inhibition leads to the blockade of evasive programs and enhances T cell-mediated GBM killing. Together, this workflow provides a scalable blueprint for comprehensive charting of the genetic control of tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e76a058bed6b1c1e4d91cd692a05e66fab6beee" target='_blank'>
              Mapping kinase-dependent tumor immune adaptation with multiplexed single-cell CRISPR screens
              </a>
            </td>
          <td>
            Lingting Shi, Ross M. Giglio, Qingyuan Cai, Mathini Vaikunthan, Justin Hong, Abdullah Naqvi, Marina Milea, Hannah Khanshali, Anna C. M. Schoonen, Nicholas Hou, Jonathan Guo, Melanie Fraidenburg, Xumin Shen, Seth Malinowski, K. L. Ligon, Raúl Rabadán, Elham Azizi, José L. McFaline-Figueroa
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bf612b75850cf10e9bb17c10ec3f99a040c4e2d" target='_blank'>
              Lineage plasticity: a new dilemma in lung cancer treatment
              </a>
            </td>
          <td>
            Cainan Huo, Yongxuan Wang, Rujia Li, Xiaofei Xie, Weiqi Lyu, Dingke Chen, Yanqin Sun
          </td>
          <td>2026-02-04</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b86fac5bb6fe4f8e5891fadb0d9a3f2b55917a3" target='_blank'>
              Impact and correction of segmentation errors in spatial transcriptomics
              </a>
            </td>
          <td>
            Jonathan Mitchel, Teng Gao, Viktor Petukhov, Eli Cole, Petr N. Kharchenko
          </td>
          <td>2026-01-20</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Ovarian cancer remains a lethal disease marked by profound therapeutic resistance, largely orchestrated by a complex tumor microenvironment (TME) governed by metabolism-immune crosstalk. This review focuses on the spatiotemporal dynamics of the metabolism-immune axis in ovarian cancer progression and resistance, with particular emphasis on how cutting-edge spatial multi-omics technologies reveal previously unrecognized layers of intratumoral heterogeneity and geographic organization that cannot be captured by bulk analyses. Using tools such as MALDI-MSI, GeoMx DSP, and CODEX, these approaches enable high-resolution, spatially resolved mapping of metabolite gradients (e.g., lactate, lipids, kynurenine), immune cell niches, and immunometabolic checkpoints within distinct tumor regions. Such spatial profiling uncovers how metabolic reprogramming-dysregulated glycolysis, lipid metabolism, and glutaminolysis-drives localized immunosuppression and chemoresistance through compartment-specific interactions among tumor cells, cancer-associated fibroblasts (CAFs), adipocytes, and immune populations. These geographically defined insights reshape our understanding of therapeutic failure and highlight precise, location-aware vulnerabilities. Accordingly, we propose spatially informed therapeutic strategies, including regional glycolysis inhibition, glutaminase blockade, lipid pathway interference, and their rational combination with immune checkpoint inhibitors (ICIs), to disrupt pathogenic metabolic-immune circuits and improve immunotherapy outcomes. Looking ahead, advances in vivo spatial imaging, gut microbiota modulation, and AI-powered integrative multi-omics frameworks promise truly personalized treatment of ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc5b7b1f229dc616042558b3b060c33208889755" target='_blank'>
              Spatial profiling of the metabolism-immune axis in ovarian cancer
              </a>
            </td>
          <td>
            Zhi-Bin Wang, Ming-Hui Long, Ping Yu, Ya-Li Wang, Zheng Yang, Ma-Sha Huang
          </td>
          <td>2026-01-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lineage tracing at single-cell resolution is vital for mapping cell fate decisions, yet synthetic barcoding faces limitations in precision, diversity, and toxicity-especially in human pluripotent stem cells (hPSCs). Here, we repurpose naturally occurring somatic mutations in mitochondrial transcripts from single-cell RNA sequencing as endogenous genetic barcodes. By enriching mitochondrial reads and applying a robust computational pipeline, we identified clonally informative variants to trace hematopoietic lineage emergence from hPSCs during early embryogenesis. Integrating mitochondrial barcoding with synthetic lineage tracing, we modeled embryonic tissue development and reconstructed the transcriptional logic and regulatory networks driving fate specification using a dynamical systems model. Extending this approach to spatial transcriptomics, we mapped the clonal architecture of human embryonic organoids, revealing spatial zonation orchestrated by NOTCH-mediated crosstalk between stromal cells and hematopoietic progenitors. This multimodal strategy links clonal dynamics with niche-driven fate decisions, offering a scalable, non-invasive method to dissect tissue organization in development and disease. Together, our work establishes a scalable, non-invasive multimodal framework that leverages endogenous mitochondrial DNA variants to reconstruct high-resolution spatiotemporal clonal dynamics and decode niche-driven fate decisions in a human stem cell-derived model. This approach provides a powerful strategy for dissecting tissue self-organization in development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44d3ba922d5d14a9018eca6710873d3d64119d9" target='_blank'>
              Single-Cell Mitochondrial Lineage Tracing Decodes Fate Decision and Spatial Clonal Architecture in Human Hematopoietic Organoids.
              </a>
            </td>
          <td>
            Yan Xue, J. Su, Yiming Chao, Lu Liu, Xinyi Lin, Yang Xiang, Mun Kay Ho, Zezhuo Su, Junyi Chen, Zhuojuan Luo, Chengqi Lin, Ruibang Luo, Theo Aurich, Jianfeng Wu, Kelvin Sin Chi Cheung, Yuanhua Huang, Joshua W. K. Ho, Ryohichi Sugimura
          </td>
          <td>2026-01-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Metastatic triple-negative breast cancer (TNBC) is highly aggressive and lacks targeted therapies. Circulating tumor cells (CTC) are invaluable for monitoring metastatic tumor progression and treatment response but are difficult to capture because of their rarity and heterogeneity. Surface-based staining for live CTCs is essential to preserve RNA quality in single cells, but current markers tend to perform poorly on more mesenchymal tumor cells such as TNBCs. To enhance live TNBC CTC detection, we developed a workflow for live CTC capture and single-cell RNA sequencing (scRNA-seq). Using a mouse model of metastatic TNBC, we identified four new CTC surface markers, AHNAK2, CAVIN1, ODR4, and TRIML2, which specifically stain tumor cells. Combining antibodies against these markers improved CTC detection rates in multiple TNBC mouse models and patient samples. Also, combining these new markers with traditional CTC surface markers enhanced detection sensitivity, achieving the highest CTC coverage. This approach identifies diverse CTC populations, while preserving RNA quality for scRNA-seq, which is essential for understanding and therapeutically targeting metastatic breast cancer. The use of these newly identified CTC markers significantly enhances both detection and live capture of CTCs, paving the way for more effective use of liquid biopsy to monitor patient prognosis and treatment response in clinical settings. Significance: CTCs are a powerful indicator of cancer metastasis; however, their scarcity makes them difficult to isolate. Current markers favor epithelial CTCs over mesenchymal populations. Our workflow for live CTC capture and sequencing enables discovery of new markers for both epithelial and mesenchymal CTCs. When combined with existing markers, we improve live CTC capture for more holistic studies of the metastatic process and offer a scalable method for discovering CTC markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b002ef6362d1a4ee73fe011a8af50d517c31c57" target='_blank'>
              Surface Marker Identification to Capture Live Circulating Tumor Cells in Metastatic Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Bree M Lege, Khushali J. Patel, Brendan Panici, Ping Gong, Michael T. Lewis, M.J. Ellis, Chonghui Cheng
          </td>
          <td>2026-01-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite significant advances in understanding lung development, the intricate cellular interactions and spatial organization of the developing human lung remain incompletely defined. Spatial transcriptomics enables gene expression profiling within the native tissue context, providing unprecedented insights into complex developmental processes. In this study, we applied the 10X Genomics Visium platform to characterize spatially resolved transcriptional profiles of prenatal human lungs during the pseudoglandular and canalicular stages.Spatial transcriptomic analysis of 12 prenatal lung samples (13–20 weeks gestation) identified 10 distinct transcriptional niches corresponding to unique combinations of epithelial, mesenchymal, endothelial, and immune cell populations. Unsupervised clustering revealed developmental shifts in spot/niche composition from the pseudoglandular to canalicular stage, with a progressive increase in alveolar epithelial spots and a concomitant decline in mesenchymal regions, particularly in peripheral lung areas. Differential gene expression analysis demonstrated stage-specific transcriptional transitions in individual spot types, including downregulation of cell cycle and structural pathways and upregulation of secretory pathways as the lung matures. Spatial organization analysis revealed increasing compartmentalization of pulmonary cell types, highlighting the progressive structuring of the distal lung microenvironment. In summary, this study provides a spatial map of the developing human lung, offering novel insights into pulmonary lineage dynamics and cellular interactions during early organogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4cdf7e14676d1f19fec1630ce3c773e08720b2c" target='_blank'>
              Spatial transcriptomics of developing human lungs defines cellular phenotypes associated with age, lineage and location
              </a>
            </td>
          <td>
            Yue Ren, Soula Danopoulos, Gail H. Deutsch, Ian A. Glass, Thomas J. Mariani, S. Bhattacharya
          </td>
          <td>2026-01-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 The tumor microenvironment (TME) plays a pivotal role in shaping cancer progression, immune evasion, and therapeutic response. However, a systematic understanding of immune and stromal cell heterogeneity across cancer types—and their associations with clinical phenotypes—remains lacking due to limited single‐cell data and the associated clinical metadata. Here, we present a comprehensive pan‐cancer TME cell atlas to date, quantifying the composition of 102 immune and stromal subtypes across 30 cancer types and encompassing over 11,000 bulk RNA‐seq samples from TCGA. To enable this unprecedented resolution, we developed DECEPTICONx, a flexible computational framework that integrates single‐cell RNA‐seq data by constructing high‐resolution expression templates, which are then paired with context‐optimized deconvolution strategies, dynamically selecting the best‐matched template‐deconvolution combinations for each task. By leveraging 535,665 single cells spanning T cells, B cells, fibroblasts, macrophages, and NK cells, we constructed high‐resolution expression templates and systematically deconvolved bulk samples. The resulting atlas revealed robust and novel associations between specific cell subtypes and patient prognosis, immunotherapy response, and inter‐cellular coordination within the TME. In summary, the DECEPTICONx‐based atlas not only advances our understanding of tumor‐immune ecology but also offers a foundation for biomarker discovery and therapeutic development in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaaef7a98ed82e86ab1fb8032603b073ca5c5bb6" target='_blank'>
              A Pan‐Cancer Microenvironment Atlas Constructed by DECEPTICONx
              </a>
            </td>
          <td>
            Yida Gu, Fulan Deng, Lianchong Gao, Zhiyu Chen, Jianping Chen, Jiale Wu, Zengqiang Xia, Zhijian Li, Fangkaikai Cheng, Jie Chen, HoiYee Tong, Wantao Chen, Xiaoqing Pan, Xin Zou, Jie Hao
          </td>
          <td>2026-01-17</td>
          <td>iMetaMed</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Regional lymph nodes (LNs) in the thoracic cavity serve as essential immunological hubs that coordinate humoral and cell-mediated responses against the development and progression of non-small cell lung cancer (NSCLC). To investigate immune dysregulation in the non-metastatic regional LNs of patients with aggressive NSCLC, we performed multimodal profiling on 36 LNs from 11 patients undergoing curative-intent resection including CITE-seq, scRNA-seq, and Imaging Mass Cytometry (IMC). Regional N1 LNs from patients with more aggressive disease (stage IB-IIIA) exhibited a significant enrichment of dysfunctional CD8+ T cells and regulatory T cells (Tregs) compared to N2 LNs and LNs from patients with less aggressive disease (stage IA). These immune subsets were spatially co-localized with mature regulatory dendritic cells (mregDCs; CD1c+, TIM3+, LAMP3+), forming an immunosuppressive niche uniquely enriched in the N1 LNs of higher-stage patients. Concurrently, higher-stage N1 LNs contained larger number of decorticated B-cell follicles characterized by decreased encapsulation of the mantle zone layer surrounding the germinal centers. This mantle zone disorganization was associated with increased spatial niches involving Tregs, CD68+ CD163+ TIM3+ Macrophages, CD163+ TIM3dim Monocytic-Myeloid Derived Suppressor Cells (M-MDSC), plasma B cells, and a decrease in spatial niches involving CD4+ T helper cells and fibroblastic reticular cells (FRCs). Together, our findings reveal parallel alterations in humoral and cell-mediated immunity within the regional LNs of patients with aggressive NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6a9f04f8595229db875548fb40d4f6cfb792216" target='_blank'>
              Multimodal single-cell and spatial profiling reveals altered T cell-mediated immunity and B-cell follicular architecture in non-metastatic lymph nodes of patients with aggressive non-small cell lung cancer
              </a>
            </td>
          <td>
            Zhanhao Xi, Yusuke Koga, Shannon McDermott, Erin E Kane, Roxanna Pfefferkorn, Ehab Billatos, Paul R Hosking, J. Beane, Eric J. Burks, S. Mazzilli, Kei Suzuki, Joshua D Campbell
          </td>
          <td>2026-01-18</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is considered immunologically “cold,” however, we have previously identified an immunogenic subset of high-risk localized PCa with T-cell tumor infiltration within antigen-presenting-cell (APC) niches. Here, we used high-dimensional single-cell spatial proteomics to dissect tumor-infiltrating-lymphocyte in (TIL)-high (H), TIL-low (L), and TIL-excluded (X) PCas, to understand the tumor-specific and conserved mechanisms of immune dysregulation for targeted therapeutics.



 Treatment-naive prostatectomy specimens of all three tumor immune microenvironment (TIME) phenotypes, were assembled into tissue microarrays representing within-tumor and adjacent foci. TIL-X was exclusively observed in cribriform pattern 4 cases. A 40-plex antibody panel made of phenotyping and functional markers (e.g AR) was applied to the TMAs, enabling single-cell annotation of key immune cell types (e.g CD8+ T cells) and assessment of their functional states within distinct spatial microenvironments.



 In TIL-X PCa, immunosuppressive macrophages and DCs were enriched in the tumor nest whereas T cells accumulated at the periphery with minimal tumor infiltration. Tumor adenosine 2A receptor (A2AR) expression was uniquely enriched within tumor, suggesting a metabolically suppressive microenvironment that reinforces immune exclusion. TIL-H tumors had comparable T cell proportions inside and outside the tumor, accompanied by the enrichment of myeloid cells within the tumor. CD8+ T cells expressing high levels of AR and B7H3 frequently co-localized with tumor and APCs, forming APC niches and tertiary lymphoid-like structures. CD8+ARhi and CD8+B7H3hi subsets with elevated exhaustion scores preferentially infiltrated tumor, whereas CD8+ARlo and CD8+B7H3lo subsets with moderate exhaustion levels remained peripheral; defining TIL-H PCa as tumors harboring antigen-experienced but exhausted CD8 T cells in a TIME that supports immune cell engagement but restrains anti-tumor activity. TIL-L PCa had sparse T cell infiltration and enrichment of myeloid cells with moderate AR expression within tumor, suggesting a role of androgen signaling and myeloid cells in shaping the TIME, marked by high PD-1+CD8+ cells and PD-L1+ tumor cells. Thus, TIL-L PCa shows low immune recruitment, yet strong tumor-mediated inhibitory signaling that constrains anti-tumor immunity.



 TILs in TIL-H PCa are phenotypically, functionally, and spatially distinct from those in TIL-L or TIL-X tumors, reflecting tumor-intrinsic immune programs. High-dimensional spatial proteomics reveal differential molecular modules and signaling pathways that define unique and potentially targetable mechanisms of immunosuppression across the three TIME phenotypes; additional spatial transcriptomics validation and dissection experiments are underway. These findings demonstrate that the PCa TIME is diverse yet spatially organized, and that comprehensive understanding of baseline spatial and transcriptional immune states may guide personalization of immunotherapy strategies in PCa.



 Precious Cramer, Yang Wang, Wenrui Wu, Huaying Qiu, Lindsay Parmelee, Berkay Simsek, Gabriel Roberti. De Oliveira, Seifeldin Awad, Yao Yu Yeo, Stephanie Pei Tung. Yiu, Hendrik Michel, Sabina Signoretti, Yue Sun, Steven Balk, Sizun Jiang, David Einstein Einstein. High-dimensional spatial profiling identifies distinct and targetable tumor-infiltrating lymphocyte phenotypes in localized prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d786dbf212bd5a71f25f7e11f429f8dba252ddc" target='_blank'>
              Abstract PR010: High-dimensional spatial profiling identifies distinct and targetable tumor-infiltrating lymphocyte phenotypes in localized prostate cancer
              </a>
            </td>
          <td>
            Precious Cramer, Yang Wang, Wenrui Wu, Huaying Qiu, Lindsay Parmelee, Berkay Simsek, Gabriel Roberti. De Oliveira, Seifeldin Awad, Yao Yu Yeo, Stephanie Pei Tung. Yiu, Hendrik A. Michel, Sabina Signoretti, Yue Sun, Steven P. Balk, Sizun Jiang, D. Einstein
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Prostate cancer cell invasion into the surrounding muscle capsule, known as extracapsular extension (ECE), is associated with a significantly worse prognosis (5-year survival reduction: 98% to 31%). Using a live ECE mouse xenograft model with DU145 tumor cells, our group previously demonstrated that functional heterogeneity among adhesion subtypes within invasive clusters is essential for muscle invasion. Bulk RNA sequencing from three spatial compartments (pre-invasive, invasive, and post-invasive) revealed distinct gene expression profiles, with hypoxia-related and cell–cell/cell–ECM interaction genes prominently dysregulated in muscle-invasive disease. In this study, we sought to define the single-cell transcriptional heterogeneity across the three ECE compartments and compare these patterns to a human patient-derived xenograft (PDX) model. Spatial transcriptomic analysis (STA) was performed using the Bruker CosMx platform with a 6K gene panel. Data was preprocessed (quality control and dimensional reduction) and analyzed in R 4.4.0 using Seurat v5. Notable downstream analyses included unsupervised Louvain clustering, trajectory inference with Monocle 3, and cluster-specific differential gene expression. In the ECE model, STA resolved at least 11 distinct tumor cell identity clusters previously undetectable by bulk methods and revealed a clear spatial demarcation between pre-invasive and invasive populations crossing the muscle barrier. Trajectory analysis identified a pre-invasive population branching into two invasive fates with unique expression profiles. Dispersion scores, calculated using principal component embeddings, exhibited progressively increasing intracluster heterogeneity along one branch. Strong expression enrichment of the Prolaris 31-gene prognostic panel, a clinically established marker of aggressiveness, was observed at the bifurcation point of tumor evolution. STA of the human PDX model similarly revealed a high-dispersion invasive cluster branching into two fates, one exhibiting decreased heterogeneity consistent with specialization of invasive phenotypes. Together, these findings reveal, for the first time, that tumor invasion of the extracapsular muscle layer induces substantial transcriptional heterogeneity. This dynamic process highlights potential therapeutic vulnerabilities during the transition to extracapsular extension. Targeting tumor cells at this critical juncture may represent a key strategy for preventing ECE and improving prostate cancer outcomes. Supported by P30CA023074, University of Arizona Comprehensive Cancer Center Support Grant.



 Saptarshi Mallick, Rafael Sainz, Jayati Chakrabarti, Kendra D. Marr, Noel A. Warfel, Gregory C. Rogers, Beatrice S. Knudsen, Rashid Sayyid, Kelvin W. Pond, Anne E. Cress. Single cell spatial transcriptomics identifies unique cell populations driving muscle invasive prostate adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcc6a26c8a5d704d47fd40ba9c2b9c0030fcf73d" target='_blank'>
              Abstract B045: Single cell spatial transcriptomics identifies unique cell populations driving muscle invasive prostate adenocarcinoma
              </a>
            </td>
          <td>
            Saptarshi Mallick, Rafael Sainz, Jayati Chakrabarti, Kendra D. Marr, N. Warfel, Gregory C Rogers, Beatrice S. Knudsen, R. Sayyid, Kelvin W. Pond, A. Cress
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="CD8+ T cells are an important weapon in the therapeutic armamentarium against cancer. While CD8+CD103+ T cells with a tissue-resident memory T (TRM) cell phenotype are associated with favorable prognoses, the tumor microenvironment also contains dysfunctional exhausted T (TEX) cells that exhibit a variety of TRM-like features. Here we deconvolute TRM and TEX cells across human cancers, ascribing markers and gene signatures that distinguish these populations and enable their functional distinction. Although TRM cells have superior functionality and are associated with long-term survival post-tumor resection, they are not associated with responsiveness to immune checkpoint blockade. Tumor-associated TEX and TRM cells are clonally distinct, with the latter comprising tumor-independent bystanders and tumor-specific cells segregated from cognate antigen. Intratumoral TRM cells can be forced toward an exhausted fate when chronic antigen stimulation occurs, indicating that the presence or absence of continuous antigen exposure within the microenvironment is the key distinction between tumor-associated TEX and TRM populations. These results highlight unique functions for TRM and TEX cells in tumor control, underscoring the need for distinct strategies to harness these populations for cancer therapies. Here the authors show that tissue-resident memory and exhausted T cells in tumors are distinct populations that are shaped by relative presence or absence of TCR signals, suggesting that a tailored therapeutic strategy is needed to target each subset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a712967a1f06be04c88812bdec6b8089c56c0b" target='_blank'>
              Antigen reactivity defines tissue-resident memory and exhausted T cells in tumors
              </a>
            </td>
          <td>
            Thomas N. Burn, Jan Schröder, Luke C. Gandolfo, M. Osman, Elanor N. Wainwright, E. Lam, Keely M. McDonald, Rachel B. Evans, Shihan Li, Daniel Rawlinson, Lachlan Dryburgh, Ali Zaid, Z. Maliga, Dominick Schienstock, Philippa Meiser, H. J. Lee, Hongjin Lai, Marcela L. Moreira, Pirooz Zareie, Louis H-Y. Lee, Lutfi Huq, Susan N. Christo, Justine Seow, Keith A. Ching, Stéphane M. Guillaume, Kathy Knezevic, Simone L. Park, Maximilien Evrard, Jason Waithman, T. Gebhardt, Scott N. Mueller, G. Riddiough, M. V. Perini, S. C. Tsao, Terence P. Speed, P. Sorger, S. Loi, Francis R. Carbone, Stephanie Gras, Timothy S Fisher, Bas J Baaten, M. Dawson, Laura K. Mackay
          </td>
          <td>2025-12-29</td>
          <td>Nature Immunology</td>
          <td>3</td>
          <td>52</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a biologically and clinically heterogeneous cancer in which the spatial organization of the tumor microenvironment dictates progression, immune escape, and therapeutic response. Traditional genomic and transcriptomic profiling have revealed key molecular pathways, yet they do not capture the spatial relationships that govern tumor-immune-stromal interactions. Spatial omics technologies now bridge this gap, allowing molecular profiling within intact tissue architecture and revealing how cellular neighborhoods, immune barriers, and stromal niches shape tumor behavior. In this review, we summarize recent advances in technologies like spatial transcriptomics, proteomics, genomics, epigenomics, and metabolomics as applied to HCC. We discuss how these technologies have illuminated the spatial dynamics of the invasive margin, macrophage and fibroblast niches, and immune evasion, identifying both prognostic and predictive biomarkers. We further highlight how spatially resolved analyses have uncovered previously unrecognized axes of communication between cancer cells and key stromal and parenchymal populations that were not apparent from bulk or single-cell molecular analyses alone. These spatially mapped ligand-receptor networks reveal coordinated signaling circuits that define targetable mechanisms of resistance and progression within the microenvironment. Finally, we explore the translational implications where spatial signatures may guide risk stratification, treatment selection, and clinical trial design. By integrating spatial architecture with molecular function, spatial omics is transforming our understanding of liver cancer biology and paving the way for spatially informed precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d51c0211c082179b9a7f39ecfe5a5186ceabc189" target='_blank'>
              Unraveling the Tapestry: Lessons from multiomics and spatial biology in hepatocellular cancer.
              </a>
            </td>
          <td>
            D. Massoudi, Ankit Srivastava, R. Dhanasekaran
          </td>
          <td>2026-02-12</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) maturity and cellular composition shape resistance or sensitivity to immune-checkpoint blockade (ICB) across cancers. Single-cell spatial resolution data for the endothelial compartment of TLSs are lacking; therefore, we investigated the cellular composition, endothelial dynamics, and cell-cell interactions within TLS regions. We applied high-resolution Xenium In Situ spatial transcriptomics to eight samples collected from non-small-cell lung cancers (NSCLC) (four after chemoimmunotherapy, four untreated), extracted 90 TLS regions, generated approximately 320,000 single-cell profiles from those regions, and analyzed the endothelial compartment within each TLS. We independently assessed the exploratory findings using multiplex immunofluorescence and survival analysis in a separate cohort of 45 patients receiving immunotherapy. Spatial mapping revealed a reproducible, B-cell-dominant TLS architecture across all samples. Mature TLSs contained CXCL13-rich follicular cores and were associated with a favorable prognosis. Within the endothelial compartment, high endothelial cells (HECs) forming high-endothelial venules (HEVs) segregated from other endothelial cells; in mature TLSs these HECs exhibited high inhibitor of DNA binding 1 (ID1) expression and an adhesion molecular signature, promoting lymphocyte recruitment. Patients with tumors harboring ID1-high HEVs showed higher response rates and pronounced survival advantage; multivariable analysis confirmed ID1-high status as an independent prognostic factor. Conversely, ICB exposure eventually reduced ID1 expression and adhesion molecule levels in HECs, impaired lymphocyte trafficking, and promoted a shift toward a remodeled endothelial state. ID1expression, linked to HEC differentiation during TLS maturation or ICB exposure, marks a functionally mature, lymphocyte-recruiting HEV that predict durable response to immunotherapy. These findings provide a novel mechanistic framework and practical biomarkers for HEV-centered immuno-oncology strategies in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a47785e0dbc1da1e3fb62fce1edd50b239726393" target='_blank'>
              Single-cell spatial analysis identifies ID1-high endothelial cells in tertiary lymphoid structures as predictors of durable response to immunotherapy in non-small cell lung cancer
              </a>
            </td>
          <td>
            Kinnosuke Matsumoto, Y. Noda, Kensuke Hachiya, F. Muramatsu, Naoki Okamoto, Weizhen Jia, T. Shiroyama, Masahide Mori, M. Tamiya, Yuhei Kinehara, A. Tamiya, Shigeki Shimizu, Hidekazu Suzuki, Kiyonobu Ueno, T. Niki, Satoshi Tetsumoto, O. Morimura, Akio Osa, Toshiyuki Minami, Satoshi Nojima, Y. Takeda, Y. Shintani, Atsushi Kumanogoh, Nobuyuki Takakura
          </td>
          <td>2026-01-06</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Colitis-associated colorectal cancer (C-CRC) arises in the context of chronic inflammation, where tumor microenvironment interactions strongly influence progression. Spatial transcriptomics enables characterization of these interactions by retaining tissue architecture and cellular context. We aimed to characterize the cellular and microenvironmental features that underlie tumor development in C-CRC.



 Seven tissue samples from three C-CRC patients were analyzed using consecutive sections processed by 10x Xenium and 10x Visium. Spatial coordinates of Xenium cells were mapped onto Visium spots to enable integrated spatial transcriptomic profiling (Figure 1A).



 Xenium analysis identified 3,506,705 cells with diverse epithelial, fibroblast, and immune cell populations (Figure 1B-C). Tumor cell subclustering revealed two functionally segregated populations: (1) proliferating tumor cells characterized by cell-cycle genes (MKI67, TOP2A) and (2) invasive tumor cells with elevated epithelial mesenchymal transition (EMT)-associated genes (CD44, CXCL1). Cellular neighborhood analysis based on distances between cells identified a distinct microenvironmental niche unique to invasive tumor cells, characterized by spatial proximity of cancer-associated fibroblasts (CAFs) and SPP1+ macrophages (Figure 1D). Pathway enrichment analysis showed selective up-regulation of angiogenesis, hypoxia, and inflammatory pathways in invasive versus proliferating tumor regions (Figure 1E). Receptor–ligand analysis identified TNC–ITGB6 interactions between CAFs and invasive tumor cells (Figure 2A). A differentially expressed gene signature distinguishing invasive from proliferating tumor cells stratified TCGA sporadic CRC samples into invasive-tumor-like and proliferating-tumor-like groups. Invasive-tumor-like tumors predominantly displayed consensus molecular subtype (CMS) 4 characteristics, consistent with the known enrichment of CMS4 in UC-CRC and supporting a role for CAF-mediated interactions in mesenchymal phenotype development (Figure 2B).



 This integrated spatial transcriptomics study identified two distinct tumor populations in C-CRC with segregated microenvironmental contexts. The invasive tumor phenotype is characterized by CAF proximity, SPP1+ macrophages, and TNC–ITGB6-mediated signaling. These findings reveal that tumor-stromal composition and architecture are linked to mesenchymal phenotype development, suggesting the microenvironment as a potential target for biomarker and therapeutic development in C-CRC.
 Conflict of interest:
 Lee, Yoonho: No conflict of interest
 Kim, Yongjae: No conflict of interest
 Baek, Jiwon: No conflict of interest
 Hwang, Sung Wook: no conflicts
 Lee, Jong Lyul: None.
 Park, Sang Hyoung: No conflicts
 Yang, Dong-Hoon: Nothing to declare
 Kim, Jihun: No conflict of interest
 Yang, Suk-Kyun: No conflict of interest
 Han, Buhm: No conflict of interest
 Song, Kyuyoung: No conflict of interest
 Yoon, Yong Sik: No conflict of interest
 Ye, Byong Duk: Byong Duk Ye reports consulting fees from AbbVie Korea, BMS Pharmaceutical Korea Ltd., Celltrion, Chong Kun Dang Pharm, CJ Red BIO, Curacle, Daewoong Pharm, Dong-A ST, Ferring Korea, Hanmi Pharmaceutical, Imscout, IQVIA, Johnson & Johnson, Johnson & Johnson Korea, Jeil Pharmaceutical Co., Kangstem Biotech, Korea Otsuka Pharm, Korea United Pharm, Lilly Korea, Medtronic Korea, NanoEntek, ORGANOIDSCIENCES Ltd., Pfizer Korea, Samsung Bioepis, Takeda, Takeda Korea and Yuhan speaker fees from AbbVie Korea, BMS Pharmaceutical Korea Ltd., Celltrion, Cornerstones Health, Curacle, Daewoong Pharm, Eisai Korea, Ferring Korea, IQVIA, Johnson & Johnson Korea, Pfizer Korea, Samsung Bioepis, and Takeda Korea and research support from Celltrion and Pfizer Korea.
 Lee, Ho-Su: No conflict of interest


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c70793b20108beca09788feb076288ec350d6d1" target='_blank'>
              DOP110 Spatial mapping of tumor-stromal cell interactions defines mesenchymal phenotypes in colitis-associated colorectal cancer
              </a>
            </td>
          <td>
            Y. Lee, Y. Kim, J. Baek, S. Hwang, J. Lee, S. Park, D. H. Yang, J. Kim, S. K. Yang, B. Han, K. Song, Y. Yoon, B. Ye, H. S. Lee
          </td>
          <td>2026-01-01</td>
          <td>Journal of Crohn’s and Colitis</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96b319260cf980924d910407801fa072aa43365f" target='_blank'>
              Multi-omic profiling of pleural effusions: a reservoir-incubator for non-invasive insights into tumor heterogeneity
              </a>
            </td>
          <td>
            Paula Nieto, G. Caratù, M. Mulet, M. Sorolla, Virginia Pajares, Domenica Marchese, Ana Pardessus, Rubén Osuna-Gómez, A. Rafecas-Codern, Pere Serra-Mitjà, J. Porcel, Silvia Vidal, Holger Heyn, Juan C. Nieto
          </td>
          <td>2026-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Worldwide, lung cancer stands as the predominant contributor to cancer mortality, characterized by substantial heterogeneity alongside intricate tumor microenvironment interactions. Gaining insight into how immune cells behave across space and time, coupled with profiling key gene expression, offers pathways toward comprehending disease advancement and uncovering therapeutic innovations. Characterization of the lung cancer immune cell landscape was achieved through single-cell RNA sequencing technology. Advanced dimensionality reduction techniques, encompassing UMAP and t-SNE methodologies, enabled multimodal examination of how cells are composed and distributed spatially. Gene Ontology (GO) enrichment alongside pathway analyses were executed to pinpoint biological processes and signaling networks operating within the immune microenvironment. To validate the transcriptomic findings, quantitative real-time PCR (qRT-PCR) was conducted in lung cancer cell lines to assess the relative expression of CXCL12, CXCR4, and PD-L1 using GAPDH as an internal control. Our investigation uncovered an intricate immune cell subpopulation architecture displaying distinctive spatial arrangement patterns. Among various cell types, key genes exhibited particular expression signatures, with tumor-associated macrophages and cytotoxic T cells showing elevated expression levels. Functioning as a principal coordinator of immune cell interactions, the CXCL12-CXCR4 signaling axis stood out prominently. Through pseudotime trajectory examination, we documented evolving patterns in cellular proportions and representative gene expression, reflecting how immune responses transform throughout tumor development. qRT-PCR analysis further confirmed that CXCL12, CXCR4, and PD-L1 were significantly upregulated in lung cancer cell lines, consistent with the single-cell sequencing results. Through investigating immune cell spatiotemporal behaviors and key gene activities, our research illuminates the intricate immune architecture within lung cancer microenvironments. Personalized immunotherapy development will be guided by these insights, which clarify how specific immune cells function while exposing prospective therapeutic intervention points. Additionally, qRT-PCR validation in lung cancer cell lines confirmed the differential expression of CXCL12, CXCR4, and PD-L1, reinforcing the robustness and translational relevance of our single-cell findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73743e96a9fb170b328b9a3bdc455c5b165f804f" target='_blank'>
              Single cell transcriptomic profiling uncovers shared inflammatory pathways and immune dysregulation underlying the asthma to lung cancer transition
              </a>
            </td>
          <td>
            He Huang, Gang Teng, Jing Zhou, Ling Liu, Mei Dong, Wei Chen, Nianzhi Zhang
          </td>
          <td>2025-12-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Intratumoral heterogeneity (ITH) is a major driver of mortality in breast cancer (BC) patients and a critical factor in the variable therapeutic outcomes observed in BC treatment. Understanding the mechanisms underlying ITH is essential for advancing both clinical and basic BC research. Chromatin accessibility is critical for regulation of gene expression and cellular identity and plays a central role in shaping ITH and tumor evolution. However, studying chromatin accessibility in situ has been challenging due to the availability of technical platforms. Here, we leveraged the spatial ATAC-seq platform to profile the chromatin accessibility landscape of tumors from six BC patients. Our analyses revealed prominent heterogeneity within tumor regulatory modules and spatial variations in immune cell composition and stromal structures, offering a framework for investigation of the molecular architecture underlying ITH. Moreover, we identified two tumor subclones with potential common origin but distinct immune infiltration conferred by regulatory cascades, suggesting that epigenetic regulation may further contribute to the divergent tumor microenvironments and phenotypic diversity of these subclones. Our study provides novel insights into the molecular mechanisms driving ITH and opens up potential avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b8d2e84c5991207c662ff2b4c48eb4734a447e" target='_blank'>
              Spatial Chromatin Accessibility Analysis of Intratumor Heterogeneity in Breast Cancer.
              </a>
            </td>
          <td>
            Yingying Qian, Miao Zhu, C. Ren, Yeyong Zhou, Jian Xu, Liang Dong, Guangyu Zhang, Cheukfai Li, Jiaoyi Lv, Qiaorui Xing, Guochun Zhang, Guangdun Peng, Ning Liao
          </td>
          <td>2026-01-12</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Breast cancer is a malignant tumor originating from breast epithelial tissue. In essence, breast epithelial cells undergo gene mutation under the influence of carcinogenic factors, leading to abnormal cell proliferation and loss of organism regulation, ultimately leading to the formation of tumors with invasive and metastatic capabilities. Carcinogenic factors of breast cancer involve multiple cellular and molecular mechanisms. Among them, disseminated tumor cells (DTCs) are considered important for treating breast cancer. However, traditional bulk sequencing techniques have limitations, such as the inability to distinguish individual cell differences and dilution of information from key cell subpopulations (such as cancer stem cells and rare immune cells). Single-cell sequencing (scRNA-seq) overcomes the heterogeneity of tumors that traditional sequencing cannot capture by analysing the molecular characteristics of single cells, providing a high-resolution perspective for precise typing of breast cancer, exploration of the mechanism of the microenvironment, and personalized treatment. Through this technology, researchers can identify specific gene expression profiles of different cell subpopulations, thus providing a new basis for the molecular typing and personalized treatment of breast cancer. This article explains how single-cell sequencing is used to describe the origin of disseminated tumor cells (DTCs), analyse tumor heterogeneity, metastasis, etc., and review the current literature on the use of scRNA-seq in breast cancer treatment. In the future, cell separation and processing steps in single-cell sequencing will be further improved to ensure the accuracy of the results and broader application in clinical diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c8a6702266de5ec2199ce11aee24ada48d0a237" target='_blank'>
              Single-cell Sequencing: Application in the Study of Disseminated Tumour Cells and Breast Cancer Treatment
              </a>
            </td>
          <td>
            Mingyue Wang
          </td>
          <td>2026-02-06</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Antigen-specific T cell populations are of great value for studying immune recognition but tedious to generate by limiting dilution or cloning. Here, we develop a streamlined approach to generate antigen-specific T cell clones directly from peripheral blood using the cloneXplorer, a live-cell analysis and clone isolation platform based on conical microwell arrays. This platform continuously monitors cell proliferation, cytokine secretion, and surface markers in up to 100,000 single cell co-cultures, enabling the identification of rare, functionally defined T cells, which can be recovered for clonal expansion or sequence analysis. We benchmark the platform by performing several key demonstrations. First, we show that this platform can efficiently generate monoclonal cell populations from cell lines and human T cells. Next, we demonstrate that antigen-specificity can be identified at single cell resolution using a co-culture of Jurkat cells expressing NFAT-GFP, CD8, and a T cell receptor and K562 antigen presenting cells (APC) expressing a peptide library. Thereafter, we show that immune activation in mouse and human primary samples can be monitored by time lapse analysis of Interferon gamma (IFN-γ) secretion in individual microwell co-cultures using a fluorescent sandwich assay. Finally, we combine these capabilities in a proof-of-concept demonstration, which uses IFN-γ secretion and the presence of CD8 surface markers as hierarchical gates to isolate and expand antigen-specific T cells from human peripheral blood, and we verify their specificity by tetramer staining. Together, these results showcase potential applications of the cloneXplorer platform in cell line development, and in screening and validating immune receptor interactions with specific antigens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3978ecc77ada817bdece83a601e88a3c9767dbf7" target='_blank'>
              cloneXplorer: A high-throughput clone discovery platform based on conical microwell arrays
              </a>
            </td>
          <td>
            Guido K. Stadler, Eugene Tkachenko, Oscar Neri, Michael Zakharov, Orr Zohar, Debbie X. Deng, Kitt D. Paraiso, Hajar Rajaei, Shaun Steele, Xiling Shen, Alex Chenchik, Benjamin B. Yellen
          </td>
          <td>2026-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Spatial organization of the disease microenvironment informs patient prognosis. Key modalities for studying spatial biology include H&E images (WSIs) for tissue structure and spatial transcriptomics (ST) for transcriptome-level programs. Spatial analysis aims to (1) identify markers linked to clinical outcome, (2) understand functional programs driving these associations, and (3) guide targeted therapies. Current research addresses these topics but offers limited explainability across the full morphology - molecular mechanism - outcome axis. Further, given the abundance of WSIs and limited availability of ST, there is a need for analyses integrating these complementary datasets. We present an AI-driven framework combining foundation-model features, multiple-instance learning, unsupervised clustering, and molecular analyses to identify mechanisms underlying outcome associated patterns. Applied to HER2+ breast cancer, we identify CCND1 and PTK6 signaling in tumor regions linked to trastuzumab resistance, consistent with prior studies. Our approach offers interpretable insights for multi-level resistance mechanisms, tissue-specific drug targeting, and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7a60accd94c10ba89d66c522cd744cd2f973bb5" target='_blank'>
              A multimodal framework to identify molecular mechanisms driving patient group-associated morphology through the integration of spatial transcriptomics and whole slide imaging
              </a>
            </td>
          <td>
            Reva Kulkarni, A. Maddox, Sara Bailey, Arvind Rao
          </td>
          <td>2026-01-16</td>
          <td>Npj Artificial Intelligence</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression is strongly shaped by the tumor microenvironment (TME), where complex interactions between epithelial, immune, and stromal cells orchestrate immune suppression and tumor evolution. To dissect these relationships at single-cell resolution, we analyzed CRC scRNA-seq datasets using Seurat for data integration and CellChat for ligand–receptor inference. We identified extensive cellular heterogeneity within the TME, dominated by CMS2/CMS3 epithelial states, SPP1+ tumor-associated macrophages, diverse T-cell subsets, and CXCR4+ B cells. Communication analysis revealed MIF-centered signaling—including MIF–CD74–CXCR4 and MIF–CD74–CD44—as the predominant axis linking tumor epithelial cells with T cells, B cells, and macrophage subpopulations. CMS3 epithelial cells displayed particularly strong connectivity to SPP1+ macrophages and cytotoxic lymphocytes through both MIF- and APP–CD74-mediated pathways. Differential gene expression confirmed elevated levels of MIF, CD74, CD44, and SPP1 in tumor tissues, while pathway enrichment analyses highlighted cytokine signaling, antigen presentation, and chemokine-regulated immune modulation as key biological processes. Collectively, our study provides a high-resolution map of CRC intercellular communication and identifies MIF-CD74-associated signaling as a central immunoregulatory hub with potential relevance for therapeutic targeting and biomarker development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb818bacc4907287357388f42145a8afa7d517aa" target='_blank'>
              Single-Cell Mapping Reveals MIF-Centered Immunoregulatory Networks in Colorectal Cancer
              </a>
            </td>
          <td>
            Marios Gkoris, Ilias Georgakopoulos-Soares, A. Zaravinos
          </td>
          <td>2026-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e234e293312dfb9093dad9f87bd40ef8d26f2e28" target='_blank'>
              Spatial transcriptomics reveals molecular heterogeneity and subtype-specific therapeutic targets in small cell lung cancer.
              </a>
            </td>
          <td>
            Tongji Xie, Le Tang, G. Fan, Xiaohong Han, Yuankai Shi
          </td>
          <td>2026-01-16</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Phenotypic plasticity is a prominent cancer feature that contributes to metastatic potential and resistance to therapy across multiple cancer types. Cancer cell state transitions have been attributed to transcriptional programs, such as the AP1/TEAD-regulated gene network driving the mesenchymal-like (MES) phenotype. In addition, during dissemination, tumor cells are subjected to variable loads of physical mechanical pressure and constriction across transited tissue, which are thought to impact nuclear molecular crowding. How the interplay between mechanical pressure, global 3D nuclear architecture and transcriptional programs contributes to MES identity and metastatic adaptation remains unclear. Using cutaneous melanoma as a model for early dissemination, we integrate in vitro and in vivo epigenomic profiling with nanoscale imaging of cell lines and patient samples to investigate chromatin organization features underlying the MES phenotype. We find that in MES cells, CTCF is relocated from domain boundaries to regulatory regions of EMT-like genes, leading to reduced insulation, extended topological associated domains (TADs) and increased inter-domain contacts, and de novo formation of chromatin hubs. This conformational rewiring, along with loss of heterochromatin, supports nuclear deformability during invasion and dissemination. Conversely, physical constriction of melanocytic cells induces MES-like chromatin features —including CTCF repositioning and heterochromatin loss— and promotes metastasis in vivo. Similarly, pharmacological inhibition of the heterochromatin mark H3K9me3 triggers MES characteristics and increases invasiveness. These results demonstrate that metastatic competency involves both epigenetic and structural nuclear reprogramming, enabling shifts in gene networks and physical adaptability. Our findings reveal mechanistic links between nuclear architecture and aggressive tumor behavior, identifying potential biomarkers and therapeutic targets to intercept metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92c8bf70d1df18ff53fcb785d7213684e545662a" target='_blank'>
              Chromatin architecture and physical constriction cooperate in phenotype switching and cancer cell dissemination
              </a>
            </td>
          <td>
            Pietro Berico, Cody L Dunton, L. Almassalha, Amanda Flores-Yanke, Karla I Medina, Nicolas Acosta, Tara Muijlwijk, Catherine Do, Soobeom Lee, S. Edmiston, D. Corcoran, Allison Reiner, Caroline E. Kostrzewa, Kathy Dorsey, Milad Ibrahim, Ronglai Shen, Nancy E. Thomas, Amanda W. Lund, Ata S. Moshiri, Iman Osman, I. Aifantis, Jane A. Skok, Vadim Backman, Eva Hernando
          </td>
          <td>2026-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d27ce10a067c8461179490fc90529630fe99743" target='_blank'>
              A modular patient-derived organoid–xenograft platform reveals molecular and clinical trajectories of prostate cancer progression
              </a>
            </td>
          <td>
            R. Parmentier, R. Dolgos, R. Servant, Jing Wang, L. Roma, R. Mevel, D. Bossi, H. Pueschel, Z. Diamantopoulou, T. Vlajnic, F. Stenner, A. J. Templeton, H. Seifert, Nicola Aceto, J-P. Theurillat, Cyril Rentsch, L. Bubendorf, C. Le Magnen
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Bladder cancer (BLCA) is a malignant tumor originating from the urothelial lining, characterized by a complex tumor microenvironment (TME) and heterogeneous tumor mutation burden (TMB). Cancer-associated epithelial cells (EpiCs) exhibit substantial heterogeneity during BLCA initiation and progression. Therefore, elucidating the diversity and functional states of EpiCs is essential for improving future diagnostic and therapeutic strategies. Methods We integrated multi-omics datasets, including 13 single-cell RNA-seq samples, 514 bulk transcriptome profiles, and 30 whole-exome sequencing (WES) samples, to comprehensively characterize EpiC subtypes. Nonlinear dimensional reduction (UMAP) and clustering analyses were performed to identify major epithelial subsets, followed by secondary clustering. TMB values calculated from self-generated WES data were incorporated into scAB and Ro/e algorithms to determine the TMB-associated epithelial subset, ultimately identifying the key cluster Epi14. Differentially expressed genes (DEGs) of Epi14 were analyzed, and CellChat was used to infer intercellular communication networks. CytoTRACE and Monocle2 were applied to assess stemness potential and differentiation trajectories. Random survival forest (RSF) combined with DEG analysis was used to identify hub genes. Immune infiltration, drug sensitivity, and functional pathway analyses were subsequently conducted. Spatial transcriptomics were deconvoluted using spacexr, CellChat, and PROGENy to map cellular composition, signaling activity, and pathway nodes. Finally, qPCR and Western blot assays were performed to validate hub gene expression in tumor versus adjacent tissues. Results A total of 77,263 cells and 3,000 highly variable genes were included, yielding 32 annotated cell clusters. Secondary clustering combined with WES-derived TMB identified 14 epithelial subpopulations, among which Epi14 was confirmed as the key TMB-associated subset using the Ro/e algorithm. Integration of DEGs, RSF, and multi-cohort datasets revealed ABRACL and ARPC3 as the pivotal hub genes, from which a risk-score model was constructed. Notably, ABRACL expression showed a strong positive association with tumor TMB and exhibited pronounced enrichment in spatial transcriptomic tumor regions. Conclusion By integrating multi-omics and spatial datasets, this study reveals the epithelial heterogeneity of BLCA and identifies ABRACL and ARPC3 as key TMB-associated hub genes within EpiCs. The established risk-score model and validated functional markers provide valuable insights for future mechanistic studies and potential clinical translation in BLCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3d01e34654acc5d2fc42e0f3045f827529cbe0f" target='_blank'>
              Integrated single-cell and spatial transcriptomics combined with whole-exome sequencing reveal key hub genes and epithelial heterogeneity in bladder cancer
              </a>
            </td>
          <td>
            Lin Li, Qianyue Li, Jiaxin Liu, Yifan Wang, Yanjun Ma, Lei Tang, Yawei Zhao
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3490b5722169f26207fe3782f1d2c2b6c0970030" target='_blank'>
              Dual Lineage Tracing Identifies Cellular Mechanisms Underlying Radiation-Associated Changes in Atherosclerotic Lesion Composition
              </a>
            </td>
          <td>
            Rebecca A. Deaton, Tajbir Raihan, Victoria M. Milosek, Fatema Allaham, Alexandra L. Krinsky, Laura S. Shankman, G. F. Alencar, A. Salamon, Nazanin Moradinasab, Subhashis Banerjee, Gary K. Owens
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT Head and neck squamous cell carcinoma (HNSCC) is the sixth most common cancer worldwide. HPV-negative HNSCC, arising in diverse upper airway mucosal niches, is particularly aggressive, with poor 5-y survival and a limited response to immune checkpoint inhibitors. A deeper understanding of the tumor-localized immune landscape is essential to uncover actionable immunotherapeutic targets. Here, we integrated two single-cell RNA sequencing (scRNA-seq) datasets from 29 samples totaling nearly 300,000 immune cells to dissect immune rewiring during tumor progression and lymph node metastasis in HPV-negative HNSCC. We identified distinct shifts in adaptive immune cell populations across 14 peripheral blood mononuclear cell (PBMC) and 21 tumor-infiltrating immune cell (TIC) states. Notably, TICs exhibited enriched interferon response and immunomodulatory gene signatures, in contrast to PBMCs, indicating tumor-specific immune imprinting. Ligand–receptor analysis revealed that immunosuppressive crosstalk between macrophages and cytotoxic cells was associated with advanced disease. To spatially validate these transcriptional states, we conducted multiplexed immunofluorescence profiling on nine locally invasive HPV-negative HNSCCs, all from the ventrolateral tongue mucosa. Spatial proteomics confirmed peritumoral enrichment of activated (CD107a+, ICOS+) NK and CD8+ T cells and intratumoral accumulation of exhausted (PD-1+, PD-L1+) phenotypes, mirroring pseudotime trajectories inferred from scRNA-seq. These findings highlight spatially localized cytotoxic cell exhaustion as a key immune evasion mechanism in HPV-negative HNSCC and underscore the value of integrating spatial and single-cell data to reveal therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e184d6ff5ae8303f9ef6f76673a4a076479b44fb" target='_blank'>
              Single-cell landscape of peripheral and tumor-infiltrating immune cells in HPV-negative HNSCC
              </a>
            </td>
          <td>
            Rômulo Gonçalves Agostinho Galvani, Adolfo Alexis Rojas Hidalgo, Carlos Alberto Biagi-Junior, B. Matuck, Jelte Martinus Maria Krol, Brittany T. Rupp, Nikhil V. Kumar, Khoa L. A. Huynh, Jinze Liu, Siddharth Sheth, Vinicius Maracaja-Coutinho, K. Byrd, Patricia Severino
          </td>
          <td>2025-12-30</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9542f45171a12fa01fc5a868a4f227534778e8" target='_blank'>
              Iron-Associated Mesenchymal Plasticity and Tumor Invasion in Glioblastoma
              </a>
            </td>
          <td>
            C. Petry, A. Boecke, A. Mohammed, N. Ghariwala, V. Jariwala, Mohammad Al Shhab, O. Buchholz, I. Vasilikos, J. M. Nakagawa, Andreas Vlachos, J. Grauvogel, M. Shah, Roland Rölz, Marco Prinz, U. G. Hofmann, Jürgen Beck, Kevin Joseph, V. Ravi
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal cancer, with liver metastases significantly worsening outcomes. However, features of the tumor microenvironment (TME) that are distinct between primary and metastatic sites remain poorly defined. Cellular neighborhoods within the TME are recognized as functional units that influence tumor behavior. Conventional spatial methods, which assign equal weights to all cells in a region, fail to capture the nuances of cellular interactions. To address this, we report here the development of Functional Cellular Neighborhood (FunCN) quantification, which integrates both the proportion and proximity of surrounding cells. Applying FunCN to PDAC imaging mass cytometry data, we identified neutrophil-enriched interactions in liver metastases compared to primary tumors, correlating with elevated VISTA expression by tumor cells. Additionally, FunCN clusters around CD8⁺ T cells in pancreas and liver were associated with higher TIGIT and LAG3, respectively. These findings demonstrate the importance of spatial immune landscapes in PDAC and identify potential therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87979b18b917426107034a7951a93426ded0148e" target='_blank'>
              Computational modeling of cellular influence delineates functionally relevant and distinct cellular neighborhoods in primary and metastatic pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Yeonju Cho, Jae W. Lee, Sarah M. Shin, Alexei G Hernandez, Xuan Yuan, Jowaly Schneider, Jody E. Hooper, Laura D. Wood, E. Jaffee, Atul Deshpande, W.J. Ho
          </td>
          <td>2026-01-23</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The tissue architecture of classic Hodgkin Lymphoma (CHL) is unique among cancers and characterized by rare malignant Hodgkin and Reed-Sternberg cells that co-evolve with a complex ecosystem of immune cells in the tumor microenvironment (TME). The lack of a comprehensive systems-level interrogation has hindered the description of disease heterogeneity and clinically relevant molecular subtypes. Here, we employed an integrative, multimodal approach to characterize CHL tumors using malignant cell sequencing, spatial transcriptomics and imaging mass cytometry. We identified four molecular subtypes (CST, CN913, STB, and CN2P), each characterized by distinct clinical features, mutational patterns, malignant cell gene expression profiles, and spatial architecture involving immune cell populations. Functional modeling of CSF2RB mutations, a characteristic feature of the CST subtype, revealed dysregulated oncogenic signaling and unique TME crosstalk. These findings highlight the significance of multi-dimensional profiling in elucidating patterns of molecular alterations that drive immune ecosystems and underlie therapeutically exploitable vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6324873c99ed0026f9e046294cb9724bf214ad" target='_blank'>
              Multidimensional Characterization of Tumor-Immune Architecture Reveals Clinically Relevant Classic Hodgkin Lymphoma Subtypes.
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew Lytle, Yifan Yin, Makoto Kishida, Michael Yu Li, Cecilia Lee, Denise Smorra, L. Hilton, S. Healy, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O Brien, Jasper Ch Wong, Claudia Cassidy, Manabu Fujisawa, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Rangel Patiño, Michael Hong, Shaocheng Wu, Eric Lee, Ali Sakhdari, Katsuyoshi Takata, T. Miyata-Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander Xu, A. Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, R. Morin, A. Alizadeh, K. Savage, David W Scott, Christian Steidl
          </td>
          <td>2026-01-30</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Chronic liver disease (CLD) represents a major global public health challenge, necessitating a systematic understanding of its complex immunopathological mechanisms. This review comprehensively summarizes the groundbreaking applications of cutting-edge technologies—including single-cell sequencing, spatial transcriptomics, and organoid models—in chronic liver disease immunology research: Single-cell sequencing resolves immune cell heterogeneity at unprecedented resolution, identifies rare cell subsets, and reveals dynamic changes and regulatory networks through multi-omics integration; Spatial transcriptomics complements this by mapping immune-stromal interactions within structural contexts such as the portal tract, fibrotic septa, and tumor niches, uncovering spatially organized immune evasion mechanisms and microenvironmental remodeling; Organoid technology constructs humanized liver-immune models that recapitulate disease-specific features—such as fibrosis, steatohepatitis, and hepatocellular carcinoma—enabling mechanistic validation, drug screening, and individualized therapeutic exploration. The synergistic integration of multi-omics profiling, spatial mapping, and organoid modeling is driving a paradigm shift in chronic liver disease immunology—transitioning from static cellular descriptions to spatiotemporal mechanism decoding, and from population-level insights to individualized pathophysiology and treatment prediction. These advanced approaches establish a technological foundation for building precision immunotherapeutic strategies tailored to spatiotemporal regulation of the liver immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1390600cdb0085be9c69b75a3de5a6e93ec2fda5" target='_blank'>
              Integrated multi-omics, spatial profiling and organoid modeling drive transformative advances in chronic liver disease and hepatocellular carcinoma immunomicroenvironment research
              </a>
            </td>
          <td>
            Qi Liu, Chenyu Wang, Cheng Ye, Huawei Zhang, Teng Li, Wenjuan Wei, Senyan Wang, Huapeng Zhang
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Immunophenotyping is fundamental to characterizing tissue cellular composition, pathogenic processes, and immune infiltration, yet its accuracy and reproducibility remain constrained by heuristic antibody panel design and manual gating. Here, we present ImmunoPheno, an open-source computational platform that repurposes large-scale single-cell proteo-transcriptomic data to guide immunophenotyping experimental design and analysis. ImmunoPheno integrates existing datasets to automate the design of optimal antibody panels, gating strategies, and cell identity annotation. We used ImmunoPheno to construct a harmonized reference (HICAR) comprising 390 monoclonal antibodies and 93 human immune cell populations. Leveraging this resource, we algorithmically designed minimal panels to isolate rare populations, such as MAIT cells and pDCs, which we validated experimentally. We further demonstrate accurate cell identity annotation across publicly available and newly generated cytometry datasets spanning diverse technologies, including spatial platforms like CODEX. ImmunoPheno complements expert curation and supports continual expansion, providing a scalable framework to enhance the accuracy, reproducibility, and resolution of immunophenotyping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a903ef7ed6f167b5b3cfd9d693e0e6daac33243" target='_blank'>
              ImmunoPheno: A Computational Framework for Data-Driven Design and Analysis of Immunophenotyping Experiments
              </a>
            </td>
          <td>
            Lincoln Wu, Minh A. Nguyen, Zhangliang Yang, Sriya Potluri, Shamaline Sivagnanam, Nell Kirchberger, Apoorva Joshi, Kyung J. Ahn, Joseph S. Tumulty, Emylette Cruz Cabrera, N. Romberg, Kai Tan, L. Coussens, Pablo G. Cámara
          </td>
          <td>2026-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Background The spatial distribution and functional heterogeneity of tumor-infiltrating lymphocytes (TILs) significantly impact patient outcomes in non-small cell lung cancer (NSCLC). While T cell factor 1 (TCF1) expressing TILs have emerged as key players in sustaining anti-tumor immunity, their subset characterization, localization, and clinical significance within the tumor microenvironment remain poorly defined. Method We performed multiplex immunohistochemistry and immunofluorescence to characterize TCF1+ immune cell subsets, in 102 NSCLC tumors, separately analyzing the tumor center (TC) and invasive front (IF). We integrated this data with publicly available single-cell RNA-sequencing datasets and clinical outcome analyses. Results CD4+ T cells and CD79α+ B cells, dominate the TCF1+ landscape, while CD8+ T cells constitute a minority of TCF1+ immune cells, particularly in the TC. We demonstrated the presence of tumor-infiltrating IgG+/IgA+ plasma cells co-expressing TCF1. PD1+TCF1- cells were more frequent than PD1+TCF1+ cells both in the TC and IF, reflecting that terminally differentiated exhausted TILs predominate within the tumor microenvironment. Survival analyses revealed significantly different prognostic impact of TILs including TCF1-expressing cells based on topography. Multivariate analysis showed that increased CD8+TCF1+ cells (HR: 2.5; p=0.039) and increased TCF1 expression by cancer cells (HR: 2,7; p=0.041) in the TC and CD4+TCF1+ cells (HR: 0.4; p=0.043) in the IF emerged as negative and positive independent prognostic markers for Disease-free survival (DFS), respectively. Integrating PD-L1 expression with TILs, PD-L1 immunopositivity was correlated with increased CD8+ and PD1+TCF1- cell infiltration and was associated with favorable DFS especially in the TC. Conclusions Our findings support a more refined framework for TCF1+ TIL assessment and TCF1 expression across cellular populations in the tumor microenvironment, with implications for prognostication in operable NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb42a95fd19ed986ce09103139534fde5284c2fa" target='_blank'>
              Immunophenotyping TCF1-expressing TILs: spatial profiling and prognostic value in operable non-small cell lung cancer
              </a>
            </td>
          <td>
            Konstantinos Ntostoglou, Georgios Christodoulopoulos, Katie Stoker, Jean Descarpentrie, A. Xagara, D. Chatzidaki, Vasiliki Anastasopoulou, Ilias P. Nikas, A. Ieronimaki, Lucy Booth, Sophia Tsoka, Ioannis Vamvakaris, A. Klinakis, Eleni Patsea, Sophia N. Karagiannis, Ioannis G. Panayiotides, Teresa Frisan, Vassilis Georgoulias, A. Kotsakis, I. Pateras
          </td>
          <td>2026-01-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Neoadjuvant immunotherapy has reshaped the treatment paradigm for esophageal squamous cell carcinoma (ESCC), yet mechanisms of resistance in non-responders remain poorly understood. Specifically, the spatial orchestration of the tumor microenvironment that limits effective antitumor immunity is not fully elucidated. Methods Integrating spatial transcriptomic analysis with large-scale tissue pathology, we mapped the spatial architecture of the immune landscape in immunotherapy-resistant ESCC to identify cellular and molecular determinants of treatment failure. Results We revealed a distinct immune exclusion phenotype in non-responders, characterized by peritumoral CD8+ T cell enrichment coupled with intratumoral depletion. At the invasive front, we identified COL11A1+ cancer-associated fibroblasts (CAFs) and SPP1+ tumor-associated macrophages as spatially correlated markers of immune exclusion, demarcating regions characterized by limited T cell infiltration. Mechanistically, a distinct subpopulation of LAMC2+ tumor cells localized at the tumor boundary acts as the master orchestrator of this barrier. These LAMC2+ cells exhibit aberrant lactate metabolism and elevated stemness, driving CAF activation via Semaphorin 3C secretion. Strikingly, this ‘LAMC2+ boundary tumor cell-immune-privileged niche’ axis exhibits pan-cancer correlations with immunotherapy resistance, positioning LAMC2 as a robust predictive biomarker. Conclusion In this study, we identify the tumor invasive margin as a critical determinant of immunotherapy resistance in ESCC. By defining the specific spatial markers and molecular architecture of an immune-privileged niche associated with the immune-exclusion barrier, our findings demonstrate the value of spatially resolved microenvironmental analysis. Moreover, we propose that therapeutic targeting of this boundary niche represents a promising strategy to overcome resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/858298d1227537940c73aee34b8d245e5b4c9498" target='_blank'>
              An Immune-privileged niche mediates immunotherapy resistance in esophageal carcinoma
              </a>
            </td>
          <td>
            Li Wan, Boye Li, Tianyun Ma, Min Wang, Fei Xie, Haotian Ge, Zixuan Xiao, Xinyi Chen, Lingda Meng, Li Zhang, Kunxin Xie, Qilong Wang
          </td>
          <td>2026-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Distal cholangiocarcinoma (dCCA) arises from the distal bile duct and is anatomically embedded within the pancreatic head, adjacent to abundant autonomic nerve plexuses. This unique location renders dCCA particularly prone to perineural invasion (PNI), a pathological hallmark that contributes to its dismal prognosis. However, the spatial architecture and molecular drivers that orchestrate PNI remain poorly defined. Here, we applied Xenium subcellular resolution spatial transcriptomics platform to profile resected tumor tissues from dCCA patients stratified by PNI status pathologically. A spatially resolved atlas comprising a total of 20 cell types was generated, uncovering enrichment of Schwann cells, type 2 conventional dendritic cells (cDC2), M2-like macrophages, cancer associated fibroblasts (CAFs) and B/plasma cells in PNI-high tumors, along with depletion of exhausted CD8+ T cells. Heterogeneous malignant cells in PNI-high tumors demonstrated activation of extracellular matrix remodeling and axonogenesis pathways, in line with the initial pathological classification. Spatial mapping further revealed distinct PNI-associated niches, notably matrix-producing CAFs (mCAFs)-macrophage clusters exhibiting coordinated enrichment of inflammatory and fibrotic programs. We further identified the LAMB3-DAG1 axis as a potential mediator of dCCA cells-Schwann cell interaction, while the preferential proximity of arteries to Schwann cells suggested additional microenvironmental support for nerve invasion. Collectively, our study provides a comprehensive subcellular atlas of PNI in dCCA, uncovering coordinated epithelial, stromal, and immune remodeling that drives perineural invasion. The identified biomarkers not only hold promise for patient stratification but may also guide intraoperative navigation and surgical margin determination, offering new avenues for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7ed53173a0ee6bb5d8c5c5791ee85be378c7900" target='_blank'>
              A subcellular spatial atlas illuminates the microenvironmental remodeling of perineural invasion in distal cholangiocarcinoma
              </a>
            </td>
          <td>
            Fansen Ji, Hao Chen, Huan Li, Jiawei Zhang, Sijia Li, Pengfei Wang, Hao Liu, Cui Ge, Bingjun Tang, Hongfang Yin, Xuedong Wang, Jiahong Dong
          </td>
          <td>2026-01-08</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Molecular subtyping of bladder cancer (BCa) using bulk transcriptomic data is hindered by tumor microenvironment (TME) contamination and intra-tumoral heterogeneity. These factors also contribute to variable responses to immune checkpoint inhibitor (ICI) therapy. Single-cell RNA sequencing offers a high-resolution approach to overcome these limitations by enabling tumor-specific analysis and improving therapeutic target identification.


MATERIALS AND METHODS
We constructed a large multicenter single-cell atlas of BCa by integrating nine cohorts comprising 123 samples and 842 127 cells. Tumor epithelial cells (n = 265 962) were identified via copy number variation (CNV) analysis using inferCNV. Pseudobulk RNA profiles were generated to simulate conventional bulk samples (Pseudobulkall) and purified tumor profiles (Pseudobulktumor), enabling systematic evaluation of bulk-based classifiers. Inter-tumoral and intra-tumoral heterogeneity programs were defined based on transcriptional profiles and validated through in vitro and in vivo experiments. Additionally, we cross-analyzed five immunotherapy datasets encompassing 13 immune response phenotypes to identify molecular features associated with therapeutic resistance and sensitivity.


RESULTS
Unsupervised clustering identified 11 major cell types and 55 non-epithelial subtypes, with TME heterogeneity driven by tissue origin and malignancy status. Existing molecular classifiers UNC (University of North Carolina at Chapel Hill) showed compromised concordance between Pseudobulkall and Pseudobulktumor, particularly in stroma-rich samples. We defined three inter-tumoral (GEinter2-4) subtypes with distinct CNV and transcription factor signatures that remained stable even at low tumor purity. Seven intra-tumoral (GEintra) programs captured key biological states, including an EMT/Stem-like module associated with immunosuppression and ICI resistance. Mechanistically, TGF-β1 induced expression of EMT/Stem-like genes (e.g., PTHLH), while Pthlh knockdown synergized with anti-PD-L1 therapy in murine models by enhancing T cell cytotoxicity without altering T cell infiltration.


CONCLUSIONS
This study provides a robust single-cell framework for resolving inter- and intra-tumoral heterogeneity in BCa, overcoming limitations of bulk-based subtyping. The TGF-β1-PTHLH axis contributes to ICI resistance, positioning PTHLH as a promising therapeutic target. These findings offer new insights for improving immunotherapeutic strategies in BCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d5eb03c1d8a0a6927854d1055fb19018c50409" target='_blank'>
              Integrated multicenter single-cell atlas of bladder cancer reveals tumor heterogeneity associated with immunotherapy.
              </a>
            </td>
          <td>
            Ziwei Wang, Wei He, Maoyu Wang, Meimian Hua, Yidie Ying, Chen Zhang, Qing Chen, Yue Wang, Xufeng Yu, Yi Wang, Huiqing Wang, Zhensheng Zhang, Shuxiong Zeng, Chuanliang Xu
          </td>
          <td>2026-02-03</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Terminally exhausted CD8+ T cells (Ttex) are emerging as clinically relevant immune subsets across solid tumors, marked by sustained inhibitory receptor expression, loss of TCF1, and limited proliferative capacity. Once considered functionally inert, Ttex are now recognized for their residual cytotoxic potential and strong associations with tumor immunogenicity, including microsatellite instability (MSI), high tumor mutational burden (TMB), and neoantigen load. Importantly, the prognostic significance of Ttex is highly tumor-context-dependent, shaped by stromal architecture, mutational burden, and progenitor Tpex availability. This review examines the biology, spatial localization, and prognostic value of Ttex, highlighting the Ttex/CD8+ ratio as a promising biomarker in cancers such as colorectal, lung, and esophageal carcinoma. We summarize recent advances in multiplex imaging, digital pathology, and AI-driven quantification that support the clinical integration of Ttex assessment. In addition, we discuss emerging therapeutic strategies targeting Ttex through immune checkpoint combinations, thymocyte selection-associated high mobility group box protein (TOX) and circRNA-mediated reprogramming, and exhaustion-resistant T cell engineering. Finally, we outline translational priorities including assay harmonization, functional validation, and longitudinal profiling to advance Ttex-based precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af1a0b32b60b15c52819ffc31235e60745d2593b" target='_blank'>
              Terminally exhausted CD8+ T cells in solid tumors: biology, biomarker potential and translational tools for precision oncology
              </a>
            </td>
          <td>
            Xuejun Guo, Shuhan Ma, Jingwen Wang, Yilin Fu, Wenxue Ma
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61f15c4d487d3eaa6d3f657b22ab80de3efa105" target='_blank'>
              immgenT: A Comprehensive Reference of Convergent T-cell States in the Mouse
              </a>
            </td>
          <td>
            I. Magill, O. Casey, Dania Mallah, Siba Smarak Panigrahi, Leon Zhou, Olga Barreiro del Rio, Derek J. Bangs, G. Bee, Samantha M. Borys, Josh Choi, Can Ergen-Mehr, Enxhi Ferraj, Maurizio Fiusco, A. Freuchet, Giovanni Galletti, A. Globig, Taylor Heim, C. Imianowski, Rocky Lai, Zhitao Liang, Andrea Lebron Figueroa, Erin D. Lucas, Julia Merkenschlager, Kevin Osum, Shannelle Reilly, Tomoyo Shinkawa, Claire E. Thefaine, Eric S. Weiss, Liang Yang, Shanshan Zhang, A. L. Zorzetto-Fernandes, Josh Croteau, Maria-Luisa Alegre, S. Behar, Rémy Bosselut, L. Brossay, Ken Cadwell, Alexander Chervonsky, Laurent Gapin, Sara E. Hamilton, J. R. Huh, Iliyan Iliev, B. Jabri, Steve Jameson, Marc K. Jenkins, Susan Kaech, Daniel H. Kaplan, Vijay Kuchroo, Michel Nussenzweig, M. Pepper, David A. Sinclair, Stefani Spranger, Michael Starnbach, Matthew Stephens, Ulrich von Andrian, Nir Yosef, Joonsoo Kang, Mitch Kronenberg, Ananda W Goldrath, M. Brbić, C. Benoist, D. Zemmour
          </td>
          <td>2026-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d7c1910610648515431116d59c0a3a122ecb43" target='_blank'>
              Multimodal Data Fusion Reveals Morpho-Genetic Variations in Human Cortical Neurons Associated with Tumor Infiltration
              </a>
            </td>
          <td>
            Yufeng Liu, Zhixi Yun, Lingli Zhang, Wen Ye, Kaifeng Chen, Xiefeng Wang, Mengzhu Ou, Jing Rong, Xiaomin Yang, Lei Mao, Liang Chen, Nan Ji, Yongping You, Chiyuan Ma, Ying Mao, Junxia Zhang, Liwei Zhang, Hanchuan Peng
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Osteosarcoma (OS) exhibits substantial genomic complexity and inter-patient heterogeneity, necessitating longitudinal, patient-matched analyses to understand acquired features of tumor evolution. However, most published OS data is limited to primary tumor samples, limiting insight into patient-specific resistance mechanisms. To address this, we characterized the genomic landscape of paired primary and metastatic tumor samples from dogs with spontaneous OS. Whole-genome and single-cell RNA sequencing reveal mutation and gene expression profiles that are predominantly organized by patient identity. Mutational burden and pathway alterations such as those involving PI3K, NOTCH, TP53, MAPK, RAS and epigenetic regulation differ between primary and metastatic samples. Variants present in tumor tissue are readily detectable in paired cfDNA samples, demonstrating the utility of this assay for identifying tumor-specific alterations associated with treatment resistance. Analysis of bulk RNA-seq data to estimate cell-type composition shows greater immune cell representation in metastases, underscoring the importance of immune signaling pathways in OS. These findings exemplify the presence of patient-specific alterations in genomic architecture over the course of tumor progression, linking CNV amplification, pathway reprogramming, and immune evasion in metastatic OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian M. Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Multiplexed immunofluorescence (mIF) imaging has revolutionized the study of cellular interactions within tissue microenvironments, enabling complex pattern analysis critical to understanding disease biology. However, current analytical methods assume uniform cellular patterns across tissues, overlooking the spatial heterogeneity that characterizes tumor microenvironments. Here, we introduce ISPat (Informed Spatially aware Patterns), a fully Bayesian framework that identifies both shared and region-specific interaction patterns while integrating domain knowledge to enhance spatial pattern estimation. ISPat models spatial cellular densities through kernel density estimation, then constructs interaction networks from precision matrices that capture conditional dependencies between cell types while controlling for confounding effects. The resulting networks reveal direct cellular relationships, with non-zero precision matrix entries indicating significant interactions. We applied ISPat to analyze 119 pancreatic ductal adenocarcinoma (PDAC) and 53 intraductal papillary mucinous neoplasm (IPMN) patients, partitioning tissues into five regions based on tumor intensity gradients. Our analysis revealed fundamentally distinct immune architectures: PDAC maintains a rigid, stable immunosuppressive microenvironment across tumor heterogeneity gradients, whereas IPMN exhibits dynamic spatial remodeling with marked regional variability. Critically, we identified multiple ligand-receptor (LR) interactions that consistently differ between disease conditions specifically in intermediate tumor intensity regions, while extreme conditions showed no significant differences. These include interactions spanning multiple functional axes of anti-tumor immunity: antigen presentation and T cell activation (APC\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}CTL, THelper\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}APC, Epithelial\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}APC), effector function and tumor cell killing (Epithelial\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}CTL, CTL\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}Treg), and immune regulation (Epithelial\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}Treg, Treg\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}APC, THelper\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}Treg). Notably, the APC\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}CTL interaction, fundamental for adaptive immunity activation, differs significantly in high tumor density regions, alongside Epithelial\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}CTL interactions critical for direct tumor elimination. These spatially resolved signatures provide quantitative evidence for distinct immune evasion mechanisms and represent promising biomarker candidates for disease classification and risk stratification. Through simulation studies, we demonstrated ISPat’s accuracy in pattern recovery and its computational efficiency, achieving 8-10 fold speedup over comparable methods through variational Bayesian inference. The framework exhibits robust scalability and handles naturally occurring partition size imbalances, making it well-suited for analyzing heterogeneous tissues. Our findings demonstrate that spatial context fundamentally shapes cellular interactions in pancreatic cancer, with critical implications for understanding immune evasion mechanisms and developing spatially informed therapeutic strategies. The identification of differential interactions across antigen presentation, effector function, and immune regulation pathways suggests that therapeutic interventions must address multiple axes of immune dysfunction rather than single targets. ISPat provides a generalizable framework for spatial analysis applicable to emerging technologies, enabling precision oncology approaches guided by spatially resolved biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba722b3f6a63490b9a491569ba714e8cdd49aa3e" target='_blank'>
              Informed spatially aware patterns for multiplexed immunofluorescence data
              </a>
            </td>
          <td>
            Sagnik Bhadury, Michele Peruzzi, Satwik Acharyya, Joel Eliason, M. P. di Magliano, Timothy L. Frankel, V. Ravikumar, S. Krishnan, Arvind Rao
          </td>
          <td>2026-01-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Introduction Mismatch repair-deficient (dMMR) colorectal cancers (CRCs) exhibit variable clinical responses to immune checkpoint inhibitors (ICIs) despite their high immunogenicity. Methods To investigate the molecular and spatial determinants of this heterogeneity, we analyzed five metastatic sites (colon, liver, peritoneum, left ovary, and right ovary) from a dMMR CRC patient treated with pembrolizumab using bulk and single-cell RNA sequencing combined with multiplex immunohistochemistry. Results Responsive lesions were characterized by enriched cytotoxic and interferon-γ signatures, greater CD8+ T-cell infiltration, and close spatial proximity between programmed death-ligand 1(PD-L1)+ tumor cells and M1-like macrophages. In contrast, resistant lesions demonstrated reduced effector cell presence and were enriched in immunosuppressive programs, including SPP1+ and CD163+ macrophages, noncanonical WNT5A/B signaling, and TGF-β–mediated matrix remodeling. Conclusion Collectively, these findings highlight the importance of spatial immune architecture and macrophage polarization in shaping ICI responses in dMMR CRC and underscore the need for spatial profiling to guide immunotherapy strategies in metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7dee5edb3a5c46063fed969eabd6424ae6c3c71" target='_blank'>
              Comprehensive spatial and immune profiling of metastatic mismatch repair–deficient colorectal cancer reveals response to immunotherapy
              </a>
            </td>
          <td>
            Ilkyu Park, Hari Kang, Se-Hee Kim, YunJae Jung, Won-Suk Lee
          </td>
          <td>2026-01-09</td>
          <td>Immunotherapy Advances</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Spatial omics (SO) technologies enable spatially resolved molecular profiling, while hematoxylin and eosin (H&E) imaging remains the gold standard for morphological assessment in clinical pathology. Recent computational advances increasingly place H&E images at the center of SO analysis, bridging morphology with transcriptomic, proteomic, and other spatial molecular modalities, and pushing resolution toward the single-cell level. In this survey, we systematically review the computational evolution of SO from a histopathology-centered perspective and organize existing methods into three paradigms: integration, which jointly models paired multimodal data; mapping, which infers molecular profiles from H&E images; and foundation models, which learn generalizable representations from large-scale spatial datasets. We analyze how the role of H&E images evolves across these paradigms from spatial context to predictive anchor and ultimately to representation backbone in response to practical constraints such as limited paired data and increasing resolution demands. We further summarize actionable modeling directions enabled by current architectures and delineate persistent gaps driven by data, biology, and technology that are unlikely to be resolved by model design alone. Together, this survey provides a histopathology-centered roadmap for developing and applying computational frameworks in SO.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab1a5c1f63e9af7d8a5576f25debcb69f3b3021a" target='_blank'>
              Histopathology-centered Computational Evolution of Spatial Omics: Integration, Mapping, and Foundation Models.
              </a>
            </td>
          <td>
            Ninghui Hao, Xinxing Yang, Boshen Yan, Dong Li, Junzhou Huang, Xintao Wu, Emily S. Ruiz, Arlene Ruiz de Luzuriaga, Chen Zhao, Guihong Wan
          </td>
          <td>2026-01-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b936d6bfcdfb468036f4e78c02a0afc5a831e69" target='_blank'>
              Spatially resolved diversity in molecular states underlies congenital melanocytic nevi and associated tumors
              </a>
            </td>
          <td>
            Daniel Aldea, Sarah Guégan, Israa Fakih, Mathias Moreno, A. de la Fouchardière, D. Pissaloux, F. Tirode, J. Puig-Butillé, J. Villanueva-Cañas, Teresa Torres-Moral, Anton Forniés-Mariné, S. Mallet, Nathalie Degardin, A. Barlier, Susanna Puig, Sylvie Fraitag, Cristina Carrera, Nicolas Macagno, H.C. Etchevers
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Mouse CD8 T cell differentiation has been studied extensively in models of infections and cancer, yet no unified framework spans the full spectrum of immunological contexts. We present the CD8 immgenT framework, integrating >200,000 single-cell transcriptomes and 128-plex surface proteomes from 734 samples spanning multiple perturbations, tissues, and timepoints. Unbiased analysis identifies 21 states encompassing naive, effector, circulating memory, tissue-resident memory, progenitor-exhausted, and terminally-exhausted compartments, among others. These states re-emerge with striking molecular convergence across acute/chronic infections, cancer, autoimmunity, aging, and homeostasis, showing that near-identical transcriptional programs support protective or dysfunctional outcomes depending on developmental history and microenvironment. Classic archetypes map to discrete clusters but exhibit unappreciated heterogeneity and overlap, cautioning against rigid nomenclature. We provide validated combinatorial markers, flow cytometry gating strategies, and immgenT reference-based integration for reproducible annotation of new datasets. This universal coordinate system harmonizes fragmented CD8 T cell literature and clarifies relationships across diverse immune challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/284621f04331918b9c34b2c80d444efa37799099" target='_blank'>
              The CD8 immgenT framework as a universal reference of mouse CD8αβ T cell differentiation states
              </a>
            </td>
          <td>
            Giovanni Galletti, A. Globig, Olga Barreiro, Taylor Heim, Shuozhi Liu, Samantha M. Borys, O. Casey, Alexander Monell, Dhruv Patravali, Nicole E. Scharping, Sara Quon, Kennidy K. Takehara, Amir Ferry, Kitty P. Cheung, Ellen Duong, Tomoyo Shinkawa, Stefani Spranger, S. Behar, Susan Kaech, Ananda W. Goldrath, D. Zemmour
          </td>
          <td>2026-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02ad1efaa62d98c7f16dd57c71f8980e8844d4c9" target='_blank'>
              Tumor microenvironment signature associated with morphology in systemic ALK-positive anaplastic large cell lymphoma
              </a>
            </td>
          <td>
            Chloé Bessiere, Loélia Babin, Sandra Dailhau, P. Gaulard, Stéphane Pyronnet, Fabienne Meggetto, Laurence Lamant
          </td>
          <td>2026-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell studies have significantly advanced our understanding of the transcriptional and functional heterogeneity in cancers. Recent studies have identified distinct states of cancer, immune, and stromal cells in the tumor microenvironment (TME), with growing evidence highlighting their clinical significance and therapeutic potential. Here, we present CancerSEA-X, an expanded version of CancerSEA that offers a comprehensive atlas of TME cell states. CancerSEA-X integrates 25 cancer cell states, 105 immune cell states, and 26 stromal cell states from systematically curated publications. Combining 239 single-cell datasets across 32 cancer types, encompassing over 9 million cells from 2120 patients, CancerSEA-X provides functional activity spectra and cancer-specific gene associations for these 156 cell states. These cell state-gene relationships were mapped onto networks, providing a systematic view of the TME. To improve usability, we redesigned the user interface to feature cell state characterization, state-gene correlation analysis, and interactive visualization of cell state-gene networks, enabling researchers to comprehensively explore these states and their functional relevance. Overall, CancerSEA-X serves as a valuable platform for investigating TME cell states, deepening our understanding of cancer heterogeneity, and potentially advancing the design of more effective clinical therapies. CancerSEA-X is freely available at http://biocc.hrbmu.edu.cn/CancerState.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc11040a019ecb4b6c2065cd731c8ff500837ec3" target='_blank'>
              CancerSEA-X: A Single-cell Resource for Tumor Microenvironment Cell States Across over 30 Cancer Types.
              </a>
            </td>
          <td>
            Lantian Deng, Wei Liu, Jinyuan Xu, Bo Zhang, Shengyuan He, Kun Liu, Xinxin Zhang, Huating Yuan, Fei Quan, Yun Xiao
          </td>
          <td>2026-01-16</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The tumor microenvironment (TME) poses a major barrier to effective immunotherapy, yet high-throughput perturbation-mapping approaches to dissect TME spatial complexity and its contextual immune modulators remain lacking. Here, we introduce CRISPR-laser-captured microdissection (LCM) integration mapping of the tumor-immune microenvironment (CLIM-TIME), a scalable platform that integrates CRISPR screening with LCM of metastatic tumors for transcriptomic, deconvolution, and immunofluorescence analyses. CLIM-TIME enables spatially resolved mapping of how tumor suppressor gene (TSG) loss reshapes the TME and modulates immune responses. We identified seven distinct TME subtypes, revealing that DNA repair and Polycomb repressive complex (PRC) TSG loss is linked to immune-infiltrated TMEs sensitive to T cell therapy. In contrast, knockouts of TSGs in the Hippo pathway promoted immune evasion and therapy resistance by fostering myeloid-enriched but T cell-excluded TMEs with elevated extracellular matrix (ECM). Targeting the ECM-crosslinking enzyme LOXL2 effectively remodeled the metastatic TME, enhancing T cell infiltration and improving therapeutic efficacy in lung metastases across multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a648a685f5af4c0a496d9f1d7cc52384d6bf90ac" target='_blank'>
              CLIM-TIME identifies metastatic microenvironment modulators for T cell therapy response.
              </a>
            </td>
          <td>
            Yinghua Wang, Weiwei Hu, Rui Xia, Xianfa Yang, Yange Gu, Zihan Ning, Tiange Yang, Chune Yu, Lulu Zhang, Dun Li, Yitian Jin, Jianhua Li, Feifei Zhang, Yaochen Xu, Chenqi Xu, Zhengxin Wang, Naihe Jing, Luonan Chen, Guangchuan Wang
          </td>
          <td>2026-02-01</td>
          <td>Cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Introduction Hepatocellular carcinoma (HCC) exhibits pronounced intratumoral heterogeneity and a complex immune microenvironment, which together limit therapeutic efficacy. Identifying actionable biomarkers and mechanisms of immune modulation remains critical for improving patient outcomes. Method We integrated single-cell RNA sequencing data with bulk transcriptomic datasets to comprehensively characterize tumor cell heterogeneity and immune landscape features in HCC. Associations between RDH16 expression and clinicopathological characteristics were evaluated, and in vitro functional assays were conducted. Mendelian randomization analyses were performed to assess causal relationships. Results RDH16 expression was significantly reduced in HCC tissues compared with non-tumor liver tissues and was associated with vascular invasion, elevated alpha-fetoprotein levels, poor tumor differentiation, and advanced T stage. Higher RDH16 expression was consistently associated with improved overall prognosis. Functional assays indicated that RDH16 did not directly affect tumor cell proliferation, migration, or invasion. Notably, RDH16 expression was inversely correlated with infiltration of CD163⁺ M2 macrophages, suggesting a potential immunomodulatory role in the tumor microenvironment. Mendelian randomization analyses further supported a protective effect of higher RDH16 expression against HCC risk. Discussion These findings identify RDH16 as an immune-modulating biomarker in HCC, highlighting its potential role in shaping the tumor immune microenvironment and suggesting new avenues for personalized immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/444ab6526b7dd129322dafa940c094f836b618fb" target='_blank'>
              Single-cell transcriptomics reveals a novel mechanism of RDH16 regulating immune infiltration in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zhenzhen Zhang, Rui Fan, Jing Ma, Dehui Li, Yan Jiang, Fahui Liu, Qiming Gong
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Multiple myeloma (MM) is recognized as a malignancy shaped by its complex tumor microenvironment (TME), which fuels disease progression and therapeutic resistance. Recent advances in single-cell omics, spatial transcriptomics, mass cytometry, and advanced imaging have enabled high-resolution mapping of tumor and immune cell interactions within their native context, also revealing spatial heterogeneity that influences clinical outcomes. These tools, complemented by scalable computational frameworks and artificial intelligence, provide cost-effective alternatives to dissect immune landscapes and derive prognostic biomarkers from both bulk and single-cell data. However, technical complexity, resource demands, and the need for robust standardization limit their immediate clinical application. On the other hand, machine learning techniques enhance integration and predictive power of existing datasets, supporting the development of personalized, immune-informed therapeutic strategies. This review highlights recent advances, discusses the strengths and limitations of emerging technologies with a particular focus on their integration to decipher TME biology and pave the way toward precision medicine in MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/085f22e2309e321faeb8024153146e39d1ce17a9" target='_blank'>
              Cutting-Edge Technologies to Decode Tumor Microenvironment in Multiple Myeloma.
              </a>
            </td>
          <td>
            Doriana Gramegna, P. Mondelli, S. A. Pappagallo, G. Gargano, M. C. Vegliante, A. Quinto, Bernardo Rossini, Gianvito Pasciolla, A. Roccaro, Michele Cea, Enrico Iaccino, A. Guarini, S. Ciavarella
          </td>
          <td>2026-02-08</td>
          <td>Hematological oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b4d870dc4ba763551ef81671814fc477f8046f" target='_blank'>
              CancerSTFormer enables multi-scale analysis of spot-resolution spatial transcriptomes and dissects gene and immune regulatory responses to targeted therapies
              </a>
            </td>
          <td>
            Benjamin S Strope, Dana Varghese, William Z Bowie, Stacy Wang, Qian Zhu
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Breast cancer is genetically and histologically heterogenous, and is influenced by a variety of factors, including the tumor microenvironment (TME). The PAM50 subtypes; Luminal A, Luminal B, Normal-like, Basal-like and Her2-enriched, are associated with different tumor phenotypes and overall survival. The quantity and quality of immune cell infiltration in breast tumors play a key role in cancer development and progression and are associated with survival and treatment response. We used multiplex immunohistochemistry, single-cell RNA sequencing, and two deconvolution algorithms to explore the immune landscape across PAM50 subtypes. Immunostaining of CD3+ T cells, tryptase+ mast cells, CD20+ B cells, CD68+ CD163+ macrophages, and CD66b+ granulocytes revealed marked differences in tumor immune infiltrates according to breast cancer subtypes. Luminal tumors were relatively deprived of T cells and B cells, while exhibiting sparse to moderate amounts of macrophage infiltration. In contrast, Her2-enriched tumors exhibited a moderate immune presence, with sparse to moderate T cell infiltration and moderate infiltration of B cells and macrophages. At the other end of the spectrum, Basal-like tumors stood out for their strikingly rich immune environment and are heavily infiltrated by T cells, B cells, and macrophages. The results from the single-cell and deconvolution analyses confirmed subtype-specific immune microenvironments, which also allowed us to observe increased levels of natural killer (NK) and CD8+ T cells in Her2-enriched and Basal-like subtypes. In conclusion, our findings demonstrate significant differences in the immune tumor microenvironment between the established breast cancer molecular subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c99c23b4dfb7da23c9b0f3a691c05aec9a1b68" target='_blank'>
              Multi-platform analysis of the tumor immune microenvironment associated with breast cancer subtypes
              </a>
            </td>
          <td>
            Lilly Anne Torland, Jürgen Geisler, Y. Azimzade, Astri Frafjord, Hogne Røed Nilsen, Linn Buer, J. Lømo, Ø. Garred, Andliena Tahiri, M. Lyngra, Inger Øynebråten, Alexandre Corthay, Kristine Sahlberg, Vessela N. Kristensen, Xavier Tekpli
          </td>
          <td>2026-01-13</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains among the most lethal solid tumors, largely due to its intricate and immunosuppressive tumor microenvironment (TME). While single-cell sequencing technologies have begun to unravel the cellular heterogeneity of PDAC, a comprehensive understanding of how genetic determinants influence and are influenced by the TME is still lacking. To bridge this knowledge gap, our study employs an integrated multi-omics approach, incorporating single-cell transcriptomics, genomics, and proteomics, complemented by computational biology and machine learning. We aimed to delineate the core molecular drivers of PDAC pathogenesis, with subsequent
 in vitro
 functional validation focusing on the role of UNC93B1 in malignant phenotypes. The ultimate goal of this research is to inform the development of precise therapeutic strategies to enhance patient survival and quality of life.





 We assembled a comprehensive multi-omics dataset, including single-cell RNA-seq data from 22 PDAC samples (GSE154778, GSE212966), bulk transcriptomic cohorts (GSE28735, GSE62452), survival data from the TCGA-PAAD project (n=172), spatial transcriptomics, and genome-wide association study data (bbj-a-140, n=196,187). The single-cell data were processed using Seurat v5, which involved rigorous quality control, batch effect correction with Harmony, unsupervised clustering, and cell type annotation to characterize TME heterogeneity. Genetic susceptibility was mapped onto single-cell data using scPagwas to calculate trait-regulated scores (TRS) and identify trait-associated genes. Co-expression networks were constructed via high-diversity WGCNA (hdWGCNA), and key candidate genes were refined through survival analysis and a machine learning framework integrating LASSO regression, Random Forest, and Support Vector Machine algorithms. The functional role of the pivotal gene, UNC93B1, was systematically investigated through Gene Set Variation Analysis (GSVA), pseudotime trajectory inference (Monocle2), and cell-cell communication analysis (CellChat).
 In vitro
 validation was performed using four PDAC cell lines (PANC-1, BxPC-3, Capan-1, SW1990). Following qPCR confirmation of high UNC93B1 expression, a stable knockdown model (sh-UNC93B1) was generated in Capan-1 cells. Functional consequences were assessed using CCK-8, wound healing, transwell and colony formation assays. A subcutaneous xenograft model was established to evaluate tumor growth
 in vivo
 . Mechanistic insights were gained through flow cytometry for cell cycle analysis and molecular profiling of the cGAS-STING pathway, senescence markers (e.g., p16^INK4a), and epithelial-mesenchymal transition (EMT)-related genes.





 Single-cell transcriptomic profiling delineated nine distinct cell populations within the PDAC TME. hdWGCNA identified three gene modules (8, 11, 16) positively associated with tumorigenesis. The intersection of these modules with differentially expressed genes yielded 320 candidates, which were subsequently filtered to 61 genes significantly linked to patient prognosis (P < 0.05) via Cox regression. Cross-validation across machine learning models and scPagwas analysis converged on UNC93B1 as the sole overlapping gene with consistent diagnostic and prognostic relevance. UNC93B1 was robustly upregulated in tumor tissues across independent datasets (TCGAxGTEx, bulk RNA-seq), a finding corroborated at the protein level by HPA and CPTAC data (P < 0.01). Its expression positively correlated with higher pathological grade and was spatially enriched within tumor regions. Functional enrichment analysis (GSVA) suggested that UNC93B1 is involved in the suppression of the cGAS-STING signaling axis. Pseudotime analysis indicated that UNC93B1 expression escalates along tumor progression trajectories. CellChat suggested strengthened intercellular communication networks in UNC93B1-high cells, particularly modulated by the cGAS-STING pathway.
 In vitro
 , UNC93B1 knockdown in Capan-1 cells significantly attenuated proliferative capacity (22.3% reduction in OD450 at 72h, P < 0.05), migratory ability (29.6% reduction in wound closure, P < 0.05), and clonogenic survival (342 fewer colonies, P < 0.01). Mechanistically, sh-UNC93B1 cells exhibited G1/S phase arrest (8.9% increase, P < 0.05), activation of the STING/IFN-β/CXCL10 cascade, elevated p16^INK4a expression, and a reversal of EMT, evidenced by downregulation of VIM and upregulation of CDH1. Consistently,
 in vivo
 xenograft experiments demonstrated that UNC93B1 silencing markedly impeded tumor growth, concomitant with reduced UNC93B1 protein and enhanced STING pathway activation. Critically, the tumor-suppressive phenotypes induced by UNC93B1 knockdown, including the inhibition of proliferation, migration, and clonogenicity, were largely reversed upon treatment with the selective STING inhibitor H-151, confirming that the observed functional consequences are causally mediated through the activation of the cGAS-STING pathway.




 By integrating multi-omics data, including GWAS and spatial transcriptomics, this study systematically defines a pivotal role for UNC93B1 in PDAC progression. Our findings demonstrate that UNC93B1 is associated with an immunosuppressive TME and facilitates metastatic spread, potentially through inhibiting the cGAS-STING-mediated innate immunity pathway. The strong correlation between UNC93B1 overexpression and adverse clinical outcomes underscores its potential as a dual diagnostic biomarker and therapeutic target. This work not only provides a mechanistic foundation for novel precision immunotherapies in PDAC but also establishes a robust methodological paradigm for multi-omics-driven discovery in oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bb10353febe80e8ef377d1403743c79b95316f" target='_blank'>
              UNC93B1 promotes pancreatic cancer progression through modulation of cGAS–STING signaling
              </a>
            </td>
          <td>
            Hao Yang, Yukun Li, Jing Su, Haiyan Zhang, Wenxin Zhu, Kanger Shen, Wei Xu
          </td>
          <td>2026-02-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Remodeling of the tumor microenvironment (TME) under therapeutic pressure is a critical determinant of treatment response and resistance in hepatocellular carcinoma (HCC). Triple-combination therapy integrating targeted agents, immune checkpoint inhibitors, and radiotherapy (T+I+R) has shown potential synergistic effects in intermediate to advanced HCC, particularly in patients with portal vein tumor thrombus (PVTT), yet the spatial and cellular mechanisms underlying its efficacy remain largely unknown. In this retrospective clinical cohort study, we compared T+I+R with targeted therapy plus radiotherapy (T+R) in advanced HCC, and further employed single-cell spatial transcriptomics and spatial proteomics to generate an integrated multi-omics atlas mapping tumor and stromal compartments, cellular compositions, and intercellular interactions with spatial resolution. Clinically, T+I+R achieved superior tumor shrinkage and disease control compared with T+R. Spatial multi-omics revealed marked region-specific remodeling, with myofibroblastic cancer-associated fibroblasts, angiogenic tip endothelial cells, and conventional dendritic cells enriched at the invasive margin and associated with therapeutic resistance, while CD8+ effector T cells were redistributed away from immunosuppressive niches, a spatial configuration correlating with enhanced response. These findings identify spatial segregation between cytotoxic and suppressive immune elements as a potential hallmark of effective therapy, providing a high-resolution spatial framework for understanding T+I+R induced TME remodeling and offering mechanistic insights to guide biomarker discovery and the optimization of combination strategies in advanced HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d6c3edb9469068b18d4bde88276d91593b43ce6" target='_blank'>
              Spatial multi-omics mapping of tumor microanatomy dynamics following radiotherapy combined with targeted-immunotherapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Fansen Ji, Haoming Xia, Ying Xiao, Jiawei Zhang, Hao Chen, Bingjun Tang, Huan Li, Hao Liu, Boyang Wu, Xiaojuan Wang, Shizhong Yang, Jiahong Dong
          </td>
          <td>2026-01-08</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Bone sarcomas remain lethal despite multimodal therapy, primarily because the mineralized, immunosuppressive tumor microenvironment (TME) promotes chemo- and immune-resistance. Integrating single-cell and spatial omics across osteosarcoma, Ewing sarcoma and chondrosarcoma delineates subtype-specific TME archetypes dominated by M2 macrophages, exhausted T cells and a stiff extracellular matrix. Mechanistic dissection reveals tractable vulnerabilities, myeloid reprogramming, extracellular matrix modulation and metabolic and epigenetic checkpoints, that can be targeted with bone-selective delivery systems and biomarker-driven combination trials to convert therapeutic failure into durable remission. Therefore, the aim of the present review is to synthesize the latest single-cell, spatial and functional data to map bone-sarcoma TME heterogeneity, dissect resistance mechanisms and propose integrated, biomarker-guided therapeutic strategies that can be translated into treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a42d68e5855e4884a22e5af558a0428be6a50022" target='_blank'>
              Tumor microenvironment in bone sarcomas: Implications for immunotherapy and emerging therapeutic vulnerabilities (Review)
              </a>
            </td>
          <td>
            Wentao Li, Lijun Lv, Yibin Jin, Xin Yuan
          </td>
          <td>2026-01-16</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lactate accumulation is a central feature of tumor metabolic reprogramming, yet its spatial and cell-type-specific effects in cancer, such as lung adenocarcinoma (LUAD), remain poorly defined. We integrated single-cell transcriptomics, spatial transcriptomics, spatial metabolomics, and immunofluorescence with TCGA survival data and machine-learning models. High-lactate tumors exhibited increased epithelial and fibroblast abundances, whereas T/NK cells and monocytes/macrophages were enriched in low-lactate samples. Spatial metabolomics revealed cell-type–restricted lactate and pyruvate distributions, with endothelial cells showing minimal lactate accumulation. Endothelial subclusters in high-lactate tissues displayed angiogenic and stress-response signatures and were strongly associated with poor prognosis. Multiple machine-learning frameworks—including random forest, elastic-net regression, SVM, ANN, and decision-tree models—consistently identified endothelial and fibroblast programs as key determinants of high-lactate states and adverse clinical outcomes. Collectively, our multi-omics spatial profiling demonstrates that lactate reshapes the LUAD microenvironment by driving angiogenesis, immune suppression, and prognostic stratification, highlighting lactate-centered pathways as promising therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a59f51e7895cef79efeacf0e1e7ab108d79743ff" target='_blank'>
              Machine learning-enabled spatial multi-omics uncovers lactate-driven targets and tumor microenvironmental reprogramming in cancer
              </a>
            </td>
          <td>
            Yingzheng Tan, Wenliang Tan, Yanchao Liang, Yunzhu Long, Shuanghua Chen, Qihao Hu, Yangjing Ou, Jingli Fu, Huan Chen, Fangyuan Ren, Jun Ye, Qing Zhou, Sheng Li, Xiaojin He, Qianqian Wang, Yueming Shen, Haiyuan Lu, Daichao Wu, An-bo Gao, Xun Chen, Yukun Li
          </td>
          <td>2025-12-30</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Colorectal ovarian metastasis (CROM), a distinct metastatic subtype of colorectal cancer (CRC), is associated with early disease onset and aggressive progression. CROM lacks specific treatment options, highlighting the need to elucidate the underlying biological mechanisms and potential therapeutic vulnerabilities. In this study, we performed integrated analyses of single-cell RNA sequencing (scRNA-seq) datasets from 155,163 cells across 35 patients from the in-house cohort and public datasets, with matched bulk transcriptomic profiling. The analysis identified AKT3⁺ EMT-like cells at the invasive tumor-stroma interface as metastasis-initiating cells. Functional validation using in vivo xenograft models demonstrated that AKT3 deficiency reduced ovarian colonization, while AKT3 overexpression conferred a mesenchymal phenotype with invasive capacity. Furthermore, reciprocal crosstalk between AKT3⁺ mesenchymal-like cells and cancer-associated fibroblasts (CAFs) played a key role in remodeling the tumor microenvironment. Multiplex immunofluorescence staining of primary tumor specimens revealed spatially coordinated AKT3+/SNAIL+/ITGB1+ tumor buds adjacent to α-SMA+ CAFs at the invasive front. Critically, AKT3 inhibition or knockdown in patient-derived CROM organoids (CROM-PDOs) significantly suppressed malignant phenotypes, recapitulating the AKT3 dependency. Collectively, these findings elucidate an AKT3-driven feedforward loop coupling EMT plasticity with CAF activation as a critical driver of CROM and propose CROM-PDOs as a robust platform for developing precision therapies targeting this aggressive CRC subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bd536a96521b7d0987ed07a91fda25944479eb4" target='_blank'>
              AKT3-Driven Epithelial-Mesenchymal Plasticity Governs Ovarian Metastasis in Colorectal Cancer via Tumor Microenvironment Remodeling.
              </a>
            </td>
          <td>
            Jingyi Shi, Xiaowen Wang, Wutong Zhang, Zhaoya Gao, Chang Zhang, Zixin Tao, Yong Yang, Jingxuan Xu, Haopeng Hong, Yunan Ma, Baojun Chen, Yunfan Wang, D. Ji, Ming Li, Guifang Jia, Jin Gu
          </td>
          <td>2026-02-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a leading cause of cancer-related mortality worldwide, largely due to late-stage diagnosis and the limited sensitivity of current biomarkers such as α-fetoprotein (AFP). Early detection requires molecularly defined targets that capture the initial steps of malignant transformation. Single-cell RNA sequencing (scRNA-seq) offers high-resolution insight into tumor heterogeneity and lineage progression to enable the identification of early biomarkers. This study aimed apply scRNA-seq analysis to detect clinically important molecular patterns that define the early stages of malignant transformation in HCC and facilitate the diagnosis of small or ambiguous lesions. Methods Two independent scRNA-seq datasets (GSE149614 and GSE189903) comprising non-tumor and HCC tissues were analyzed. Following batch correction and clustering, hepatocyte subpopulations were characterized by differential expression, pseudotime, and CytoTRACE analyses to reconstruct the trajectory from normal to malignant states. High-dimensional weighted gene co-expression network analysis (hdWGCNA) was used to identify stage-associated modules, while CellChat and protein-protein interaction analyses delineated intercellular signaling networks. Target expression was validated in paired human liver specimens using quantitative immunofluorescence. Results scRNA-seq revealed a continuum of hepatocyte states characterized by progressive stemness and oncogenic pathway activation (MYC, E2F, G2M). Module Hep-M20 exhibited the strongest correlation with tumor stage and identified BSG/CD147 as a central hub gene with monotonic upregulation along pseudotime and strong correlation with stemness potential. CellChat analysis uncovered a cyclophilin (PPIA/PPIB)-dependent tumor-stroma signaling axis that positions BSG/CD147 as the key mediator for intercellular communication between tumor hepatocytes, fibroblasts and T cells. Ex vivo validation confirmed significantly higher BSG/CD147 protein expression in HCC versus background liver (P = 2.9×10−11) with excellent diagnostic accuracy (AUC = 0.93–0.96; sensitivity 86–87%; specificity 93–97%) including in lesions < 2 cm that are frequently indeterminate on conventional imaging. Conclusions This study establishes BSG/CD147 upregulation as an early molecular event in hepatocarcinogenesis that integrates hepatocyte dedifferentiation, microenvironmental signaling, and tumor progression. Strong and specific expression in small lesions < 2 cm underscores potential as a precision biomarker and imaging target for early HCC detection, risk stratification, and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91b69381a1a33432cf5fe739884155cd3bedeaf6" target='_blank'>
              Multimodal single-cell network analysis uncovers BSG/CD147 as an early biomarker and signaling hub in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Thomas DW Wang, , Shuo Feng, Hui Jiang, Thomas DS Wang, Sangeeta Jaiswal
          </td>
          <td>2026-01-27</td>
          <td>Research Square</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor evolution involves genetic, transcriptional, and phenotypic alterations that shape cancer cell behavior and interactions with the microenvironment. While single-cell technologies have advanced our understanding of this process, spatial dynamics remain incompletely characterized. Here, whole-exome sequencing (WES), imaging mass cytometry (IMC), and spatial transcriptomics (ST) were integrated to study molecular evolution and immune responses in two lung adenocarcinoma (LUAD) mouse models: a genetically engineered model (129S4/Sv-KrasLSL-G12D, termed 129S4 K) and a carcinogen-induced precancer model (129S4 U). Compared to 129S4 K, 129S4 U tumors exhibited higher mutational, neoantigen but lower copy number variation (CNV) burdens at matched developmental timepoints, consistent with findings of higher mutational burden in human smoking-related LUAD than nonsmoking LUAD. We profiled over 1.4 million spatial single cells from 284 IMC regions of interest and 51,531 spatial transcriptomic spots from 156 lesions across 141 mice. Macrophage abundance increased with tumor progression, while CD8 T-cell and B-cell densities declined in late-stage LUAD. 129S4 U showed greater immune infiltration in both tumor and adjacent normal tissue, higher T-cell cytotoxicity signature score in line with its higher mutational and neoantigen burdens. LUAD progression was marked by early morphological shifts and late-stage changes in cell states and interactions. These data define spatial and genetic landscapes of LUAD development and provide a framework for investigating immune evolution and therapeutic strategies in early carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e7ad7c0953d4cab7fb1f9dcb251447c108ead" target='_blank'>
              Spatial Profiling Reveals Distinct Molecular and Immune Evolution of Mouse Lung Adenocarcinoma Precancers with or Without Carcinogen Exposure.
              </a>
            </td>
          <td>
            Bo Zhu, M. Aminu, Pingjun Chen, Jian-Rong Li, Chuanpeng Dong, Chenyang Li, Yanhua Tian, Shao-Wei Lu, Hong Chen, Chenxi Ma, Xin Hu, Jie-Tong Ye, Andrew Y Liu, Beibei Huang, Frank R. Rojas, Parra Cuentas Edwin Roger, Ou Shi, M. Nilsson, A. Poteete, Khaja B Khan, Wei Lu, Luisa M. Solis Soto, Junya Fujimoto, C. Haymaker, Ignacio I. Wistuba, Zhubo Wei, Linghua Wang, Don L. Gibbons, Ken Chen, Alexandre Reuben, Jason M Schenke, J. Heymach, Chao Cheng, Jia Wu, Jianjun Zhang
          </td>
          <td>2026-01-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cef7edb3493fb7ab6d3516e83b3f06b2dd845210" target='_blank'>
              VAETracer: Mutation-Guided Lineage Reconstruction and Generational State Inference from scRNA-seq
              </a>
            </td>
          <td>
            Linnuo Pan, Kaiyu Wang, Luonan Chen
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The progression of prostate cancer (PCa) to metastatic disease remains a critical clinical challenge, making it essential to gain a detailed understanding of the factors that enable certain subclones to invade and spread. Building on our previous study, which examined genomic variations in benign and malignant prostate tissues by inferring copy number alterations (CNA) from spatial transcriptomics data1, this study aims to trace the spatial evolutionary trajectories of PCa subclones from the primary tumor to matched lymph node metastases and to identify alterations in associated microenvironments and early diagnostic markers. To achieve this, we performed spatial transcriptomics on entire prostate axial disks and patient-matched lymph node metastases from 10 individuals. Standard spatial transcriptomics (55 µm) was used for tissue profiling; from the spatial transcriptomics data, copy number alterations (CNAs) were inferred to identify distinct tumor subclones, and a clone-tree was constructed to describe their evolutionary relationships. Additionally, a high-resolution spatial transcriptomics (2 µm) technology was employed to gain a cellular-level view of the immediate tumor-microenvironment (TME) surrounding specific subclones. Analysis of over 1,000,000 barcoded regions identified many distinct tumor subclones, enabling us to trace their evolutionary trajectories spatially. We observed notable subclonal events within the lymph nodes, including polyclonal colonization, indicating multiple dissemination events during the evolution of the primary disease. Exploration of the immediate TME revealed significant cellular heterogeneity and upregulation of genes related to antigen presentation and inflammatory pathways concentrated near ancestral tumor clones, specifically at the tumor border. By focusing on the inferred CNA profiles of the metastasizing clone across patients, we identified several common features. Additionally, we were able to identify potential lethal disease as early as the diagnostic core needle biopsy stage by detecting features of metastasis and pinpointing the metastasizing clone. In summary, our study provides a detailed spatial map of PCa clonal evolution and dissemination, linking primary tumors to nodal metastases and revealing altered cell composition and gene expression around tumor clone borders. Importantly, it demonstrates subclonal events within lymph nodes, polyclonal colonization, and the potential to identify metastasizing clones as early as the diagnostic phase, which holds strong implications for precision oncology and early intervention.
 1. Erickson, A., He, M., Berglund, E. et al. Spatially resolved clonal copy number alterations in benign and malignant tissue. Nature 608, 360–367 (2022). https://doi.org/10.1038/s41586-022-05023-2



 Mengxiao He, Sandy Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, Ian G. Mills, Joakim Lundeberg, Alastair D. Lamb. Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604426087ba878274c47234a7ce00263acf256a4" target='_blank'>
              Abstract PR013: Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers
              </a>
            </td>
          <td>
            Mengxiao He, S. Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, I. Mills, J. Lundeberg, Alastair D. Lamb
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Double-Negative T (DNT) cells, lacking both CD4 and CD8 expression, play critical roles in cancer immunology, and have garnered increasing attention in cancer research. However, their heterogeneity and functional diversity within the tumor microenvironment (TME) remain underexplored. In-house and publicly available single-cell RNA sequencing (scRNA-seq) data for different cancer types were integrated after quality control and batch effect correction, followed by DNT cells separation from CD3+ T cells subtypes. Functional characteristics, intercellular communication, differentiation trajectories, regulatory networks, and clinical relevance were analyzed among different DNT subsets. Key findings were validated using multiplex immunofluorescence and spatial transcriptomics to investigate the spatial localization of DNT subsets and their interactions within the TME. Impact of γδ T cells on immunotherapy response was also assessed using MC38-based murine tumor model. By integrating scRNA-seq data from 2,369 samples across 23 cancer types, we established a comprehensive single-cell atlas of 157,025 high-quality DNT cells. Fourteen distinct DNT subsets (6 αβ DNT and 8 γδ T cell subsets) were identified, demonstrating tumor both type-specific and shared distribution patterns, as well as unique cell-cell interaction network within the TME. These subsets displayed specialized functional profiles, including cytotoxicity, antigen presentation, and immune modulation, indicating that the functional diversity of DNT cells is largely subset-specific rather than a manifestation of multifunctionality within a single population. We also delineated divergent trajectories for αβ DNT and γδ T cell subsets, including the functional plasticity of gut-resident γδ T cells transitioning between cytotoxic and immunosuppressive states. Notably, several DNT subsets were significantly associated with favorable clinical treatment outcomes, including improved responses to cancer immunotherapy. Consistently, depletion of γδ T cells in the murine tumor model significantly decreased the efficacy of PD-1 blockade, underscoring their critical role in therapeutic response. Our study uncovers the previously underappreciated heterogeneity and functional diversity of DNT cells in the TME and demonstrates their profound impact on tumor progression and immunotherapy outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4d4eea61e2f155251ae235dac3fc5c5c43d18" target='_blank'>
              A pan-cancer single cell landscape reveals heterogeneity and functional diversity of double-negative T cells
              </a>
            </td>
          <td>
            Qing Hao, Tao Zhou, Huayun Yan, Zhixiang Ren, Wei Mao, Li-Bin Huang, Yue-Yun Chen, Yuelan Wang, Minqin Xiao, Yajiao He, Xuyang Xia, Jing Wu, Wei-Han Zhang, Hai-Ning Chen, Maoxiang Qian, Feng Zhang, Li Yang, Lunzhi Dai, Huiyuan Zhang, Canhua Huang, Yang Shu, Yu Xu, Yang Luo, Yong Peng, Heng Xu
          </td>
          <td>2026-01-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8aa42749ccdec32f1c569952aff3860b5f6d3b49" target='_blank'>
              Integration of imaging-based and sequencing-based spatial omics mapping on the same tissue section via DBiTplus.
              </a>
            </td>
          <td>
            Archibald Enninful, Zhaojun Zhang, D. Klymyshyn, Matthew Ingalls, Mingyu Yang, Hailing Zong, Zhiliang Bai, Negin Farzad, Graham Su, Alev Baysoy, Jungmin Nam, Yao Lu, Shuozhen Bao, Siyan Deng, Nancy Zhang, Oliver Braubach, Mina L. Xu, Zongming Ma, Rong Fan
          </td>
          <td>2026-01-15</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Glioblastoma is characterized by heterogeneous and plastic cellular populations that adopt transcriptional programs shaped by genetic alterations and microenvironmental cues. Recent studies have identified at least four partially inconvertible cell states-astrocytic-like, neural progenitor-like, oligodendrocyte progenitor-like, and mesenchymal-like-that represent aberrant developmental programs. Expanded analysis further reveals hybrid and intermediate states that form continuous transcriptional and metabolic gradients. These states exhibit spatial organization, assembling into three distinct microanatomical niches: a perivascular niche enriched with mesenchymal-like and oligodendrocyte progenitor-like cells, a hypoxic niche harboring quiescent and stressed cells of all states, and an invasive niche containing astrocyte-like or proneural populations. Niches continuously remodel as cell states transition, migrate, and re-establish new programming in response to angiogenesis, hypoxia, immune infiltration, and neuronal activity. This interplay between states and the microenvironment generates a self-renewing spatial architecture, maintaining expansion at the edge and protection within the core. This review integrates single-cell, single-nucleus, and spatial studies to describe a microenvironmental-driven model of cell state organization. Understanding how these multiscale drives converge to generate a continuum of cell state identities may reveal strategies to disrupt the spatial architecture underlying glioblastoma plasticity and recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c762648021d36dbbe67c2cb45c25c28113bb3f2d" target='_blank'>
              Tumor Microenvironment Shapes the Spatial Organization of Glioblastoma Cell States.
              </a>
            </td>
          <td>
            P. Prakash, James Trippett, Cameron Ehsan, Joseph Namkung, Meeki K. Lad, M. Aghi
          </td>
          <td>2026-01-10</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Spatial omics technologies enable the simultaneous measurement of molecular features within their spatial context, providing unprecedented insights into cellular organization and tissue architecture. The advent of single-cell resolution platforms has further enhanced our ability to uncover microenvironment-dependent cell states. While numerous computational methods have been developed for spatial omics analysis, most focus on spatial domain detection rather than on resolving cell identities. Traditional single-cell clustering methods rely solely on intrinsic molecular features to determine cell identities, missing the impact of the local microenvironment on cell states. To address this gap, we present MEcell, a parameter-free method that explicitly incorporates spatial context and automatically adjusts its contribution to improve cell identity modeling. Applied to 90 simulated and 7 real datasets spanning multiple spatial transcriptomics platforms and tissue types, including MERFISH/Vizgen, Xenium, CosMx, Visium HD, Slide-seqV2, and open-ST, MEcell consistently outperformed existing methods in accurately inferring cell identities. These results highlight the critical role of microenvironment in defining cell identity and demonstrate the power of MEcell for capturing spatially informed cellular heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/505b13c396e81cc038e0a25abd6d0dddfe31497e" target='_blank'>
              Microenvironment-aware spatial modeling for accurate inference of cell identity
              </a>
            </td>
          <td>
            Qi Liu, Yu Wang, Chih-Yuan Hsu, Celestine N. Wanjalla, K. Lau, Yu Shyr
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity and plasticity. GBM tumors contain neoplastic cells within a fluid spectrum of diverse transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we develop scFOCAL (Single-Cell Framework for -Omics Connectivity and Analysis via L1000), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identify compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states, and leverage this capability to predict combination synergy. We validate the significance of these findings in vitro, ex vivo, and in vivo, and identify a combination of an OLIG2 inhibitor and Depatux-M for the treatment of GBM. Our studies suggest that scFOCAL identifies cell states that are sensitive and resistant to targeted therapies in GBM using a measure of cell and drug connectivity, which can be applied to identify new synergistic combinations. Glioblastoma is characterised by high levels of intratumoural heterogeneity and plasticity, hindering treatment. Here, the authors develop an analytical framework, scFOCAL, to predict the sensitivity of glioblastoma cell subpopulations to therapies based on reversal of disease transcriptional signatures to identify synergistic therapeutic combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef55e3b53c83b7969434910c5b4838c55ce9672" target='_blank'>
              Drug and single-cell gene expression integration identifies sensitive and resistant glioblastoma cell populations
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, R. Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, R. Komotar, Macarena I. de la Fuente, Gregory Stein, Alexandre Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, Nagi G. Ayad
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Highlights What are the main findings? We established and characterized a unique pair of isogenic cervical cancer cell lines (AdMer35/AdMer43) from a patient with squamous cell carcinoma of the cervix before and after chemoradiotherapy, revealing that resistance is not caused by a single factor but by complex reorganization of cancer cells. Single-cell analysis identified key resistance hallmarks: transcriptional reprogramming towards embryonic stemness (HOX, POU5F1), a shift from fibrillar to non-fibrillar collagen expression, and activation of interferon/inflammatory pathways. What are the implications of the main findings? The study maps a remodeled tumor ecosystem with a metabolically reprogrammed, pro-tumorigenic senescent-like cell population and a progenitor-like cell reservoir, identifying them as critical therapeutic targets for eliminating resistant cell pools. The identified key pathways, Wnt, TGFβ, BMP, and the specific ligands EREG and SEMA3C, along with the metabolic vulnerabilities of the senescent compartment, provide a robust framework for future therapeutic exploration. Abstract Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V Semenov, Irina S Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
ERBB2/HER2 is overexpressed or mutated in ~15% of gastric cancers due to amplification or mutation of the ERBB2 locus. While the tumor cell-intrinsic consequences of ERBB2 overexpression are well understood, much less is known about its effects on the tumor microenvironment.


METHODS
We have developed genetically engineered ectopic and orthotopic syngeneic models of organoid-based gastric cancer that have allowed us to study the tumor microenvironment of ErbB2-overexpressing tumors, using spectral flow cytometry, single cell RNA sequencing, and TCR repertoire sequencing. Interventions such as anti-Erbb2 and anti-PD1 antibody treatments were used as well.


RESULTS
We find that ErbB2 drives the infiltration of CD4+ and CD8+ T-cells, which express granzymes, FasL, and surface markers indicating chronic activation, and in the case of CD8+ T-cells, have undergone clonal expansion. The adoptive transfer of T-cells sorted from ErbB2-overexpressing tumors reduces the growth of ErbB2-expressing, but not control tumors in T-cell-deficient recipients. PD-1-specific checkpoint blockade synergizes with an ErbB2-targeting antibody to reduce the growth of ErbB2-expressing, but not control tumors. Mechanistically, ErbB2 overexpression results in micronuclei formation and the transcriptional activation of numerous interferon-responsive genes in vitro and in vivo; mice lacking the type I interferon receptor show higher engraftment rates and lower T-cell infiltration than wild-type controls.


CONCLUSIONS
The combined data indicate that ErbB2, perhaps by driving micronuclei formation, has immunogenic properties that manifest in the form of increased T-cell infiltration and expansion, which can be exploited therapeutically by combining PD1-directed checkpoint blockade with ErbB2-targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a534bc7afbf1125fe30219c1640f7b649c8bcf15" target='_blank'>
              Genetically engineered ErbB2 overexpression sensitizes organoid-derived tumors to checkpoint inhibition in a syngeneic model of gastric cancer.
              </a>
            </td>
          <td>
            Jiazhuo He, Lydia Kirsche, Zuzana Nascakova, Francesco Manfredi, Chiara F. Magnani, Giovanni Papa, Flora Azizi, Peter Leary, Anne Müller
          </td>
          <td>2026-02-01</td>
          <td>Journal for immunotherapy of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="A central challenge in cancer research is to identify the secreted factors that sustain tumor cell survival. This is best exemplified in Hodgkin lymphoma, where malignant cells constitute a minor fraction of the tumor and rely on signals from the microenvironment for survival. Using genome-wide transcriptional profiling with spatial and single-cell resolution, we show that the neighborhood around malignant cells forms a distinct niche of 31 non-malignant cell types, enriched in helper T cells and myeloid cells, but depleted of plasma cells. Moreover, our spatial analysis nominates IL13 as a candidate survival factor. Recombinant IL13 augments malignant cell growth in vitro, and genome-wide loss-of-function screens across >1000 human cancer cell lines identify IL4R and IL13RA1, heterodimeric components of the IL13 receptor, as uniquely essential in Hodgkin lymphoma. Importantly, blocking antibodies phenocopy genetic inactivation. Our findings provide a biological rationale for testing IL13-directed therapies, which are already FDA-approved, in Hodgkin lymphoma. Hodgkin Reed Sternberg (HRS) cells and their surrounding microenvironment in Hodgkin lymphoma remain poorly characterized. Here, the authors perform genome-wide transcriptional profiling with spatial and single-cell resolution to explore the cellular and molecular composition of the Hodgkin lymphoma microenvironment and used machine learning to identify IL13 as a potential HRS cell survival factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdc2f4fd014e0601edcc3f78406fd8afaf2448ea" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, D. Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson A. Weir, O. Ashenberg, Geraldine S Pinkus, Scott J. Rodig, Caroline Uhler, Evan Z. Macosko, Margaret Shipp, A. Louissaint, Fei Chen, Todd R Golub
          </td>
          <td>2025-12-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Somatic mutations accumulate during the first zygotic division and continue throughout an organism’s lifespan. The characteristics and frequency of these mutations are contingent on developmental timing and tissue type, giving rise to somatic mosaicism, defined as the presence of unique genomic alterations across different cells. They serve as endogenous cellular barcodes, enabling detailed reconstruction of cell lineages and clonal dynamics. Although lineage tracing techniques have advanced from early microscopic observation and dye staining to the introduction of artificial barcodes via gene editing, owing to ethical considerations, such genetic manipulations in human developmental research are unavailable. Therefore, spontaneously arising somatic mutations are the most suitable strategy for tracing human lineages. Current approaches can be broadly categorized into two strategies: (i) high-resolution methods, including single-cell clonal expansion or laser-capture microdissection, which construct precise phylogenetic trees based on shared mutation profiles; and (ii) bulk sequencing methods, which infer lineage proximity by comparing variant allele frequencies across samples. As more lineage-tracing studies are being conducted focusing on a wider variety of organs, the integration of such data will make it possible to discover the general principles governing human development. This review highlights how the concept of somatic mutations has been applied across diverse biological contexts and discusses the insights and common principles that can be drawn from these findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b635d70a77d777444fb98a226730074956b4517" target='_blank'>
              Reconstructing developmental lineages: a retrospective approach using somatic mutations and variant allele frequency
              </a>
            </td>
          <td>
            Mahnoor Sajjad, Seong Gyu Kwon
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Angioimmunoblastic T-cell lymphoma (AITL) represents the second most prevalent subtype of peripheral T-cell lymphoma, characterized by a dismal prognosis. However, a systematic exploration of tumor microenvironment (TME) features and their prognostic significance in AITL remains notably deficient. To address this knowledge gap, we conducted spatial transcriptome sequencing (ST-SEQ) and whole-exome sequencing in four AITLs and two noncancerous lymph nodes for discovery purposes, complemented by immunohistochemistry analyses on 37 AITL cases for validation. We identified 14 ST clusters, including five neoplastic clusters, wherein a global shift in B-cell phenotypes and enrichment of myeloid cells were observed. These findings underscore a hallmark of exacerbated inflammation and immune dysregulation within the neoplastic TME. Among the 4 ST-sequenced AITLs, 3 expressed high CD40-CD40LG activity, accompanied by the upregulation of immune-suppressive-associated genes, such as CCL17 and PDCD1. Conversely, the remaining patient displayed an uncommon absence of CD40-CD40LG activity but harbored a phagocytosis-associated tumor-associated macrophage (TAM)-enriched TME, which correlated with significantly reduced relapse rates and longer event-free survival (EFS), highlighting the critical value of precise TME stratification in tailoring AITL therapeutic strategies. Finally, trajectory analysis unveiled a distinct trajectory of molecular evolution within this TME landscape. Collectively, our findings illuminate the heterogeneity and prognostic implications of the TME in AITL, providing a robust foundation for the rational design of targeted immunotherapeutic approaches. These insights may substantially advance the development of personalized treatment strategies for AITL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47887a6371994feccf99a507b06079fcceafb4cb" target='_blank'>
              Revealing tumor microenvironmental heterogeneity and prognostic value in angioimmunoblastic T-cell lymphoma via spatial transcriptome sequencing
              </a>
            </td>
          <td>
            Xiang Zhang, Yong Sun, Duo Wu, F. Yu, Hanjin Yang, X. Ye, Juying Wei, Xuewu Zhang, Yanan Zhu, Yunfei Lv, Zijing Xu, Yuxiang Chen, Hongyan Tong, Jie Jin, Xiaofei Ye, Wenjuan Yu
          </td>
          <td>2026-01-09</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatial proteomics has enabled high-resolution characterization of protein organization within tumor microenvironments, yet most computational approaches implicitly assume spatial homogeneity and focus on clustering rather than diffusion constraints imposed by tissue morphology. Here, we model morphology–protein coupling in triplenegative breast cancer using geographically weighted regression (GWR) applied to 41 publicly available Multiplexed Ion Beam Imaging (MIBI) samples comprising 36 protein markers. Single-cell morphometric features were extracted from MIBI spots and combined with spatial adjacency graphs to model location-specific protein dispersion. Compared with ordinary least squares and ridge regression baselines, GWR consistently demonstrated superior performance across regression metrics, explaining substantially greater spatial variance in protein intensity (+.4 R2 improvements across markers) while reducing mean absolute and squared errors. Information-theoretic analysis showed lower (Aikake Information Criterion Corrected) AICc values for GWR across the majority of markers, indicating improved model fit. Spatial autocorrelation diagnostics further confirmed that GWR residuals exhibited near-random structure, with significant reductions in Moran’s I and Geary’s C relative to global models, demonstrating effective capture of local heterogeneity. Eight proteins with significant spatial autocorrelation, including B7-H3 and -catenin, showed pronounced morphology-dependent dispersion patterns that were not recoverable using global regression. These results demonstrate that explicitly modeling spatial heterogeneity yields more accurate and interpretable representations of protein organization and supports a diffusion-barrier view of pathoproteomics beyond agglomeration alone.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db951a4a2419b28fdecf539ba062b45ce28797e3" target='_blank'>
              Spatial Regression of Morphology-Protein Coupling in Tumour Proteomics
              </a>
            </td>
          <td>
            Alejandro Leyva, M. Khalid, Khan Niazi
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 The lung tumor microenvironment (TME) plays a crucial role in the progression and metastasis of lung cancer. It consists of various cell types that interact in complex ways to influence tumor behavior. CD45
 +
 cells, as a component of the TME, have complex and multifaceted roles in lung cancer. The balance between the anti-tumor and pro-tumor functions of CD45
 +
 cells can significantly affect lung cancer outcomes. Understanding these roles is essential for developing targeted therapies that harness the beneficial effects of CD45
 +
 cells while mitigating their harmful effects.





 We performed single-cell RNA sequencing of sorted CD45
 +
 immune cells from healthy lungs, orthotopic LLC1 tumors, and Kras
 LA2
 (Kras) genetically engineered tumors. Analyses included immune composition, transcriptional programs, differentiation trajectories, metabolic states, and ligand-receptor-based intercellular communication networks.





 Four major immune compartments, B cells, T cells, NK cells, and macrophages, underwent model-specific remodeling. LLC1 tumors showed B cell expansion and T and NK cell reduction, with inflammatory, stress-response, and NF−κB/TNF-dominant programs. Kras
 LA2
 tumors retained a balanced immune composition but exhibited metabolic rewiring, elevated antigen-presentation signatures, and selective intercellular signaling. Subclustering revealed specialized changes across B cell (resting, mature, pre-Bcr, late pro-B, plasma), T cell (Cd4
 +
 , Cd8
 +
 , memory, activated, Treg, Th17), NK cell (Fcgr3
 high
 , Fcgr3
 low
 , Xcl1
 +
 ), and macrophage (Ace
 +
 , Bcr
 +
 , Ccr2
 +
 , Cd3
 +
 , metabolic, MHCII
 +
 ) subsets. Ligand-receptor analyses highlighted dense inflammatory networks in LLC1 tumors versus metabolically tuned signaling in Kras
 LA2
 tumors.





 Distinct CD45
 +
 immune landscapes, characterized by inflammatory suppression in LLC1 and metabolic adaptation in Kras
 LA2
 tumors, shape lung tumor biology. This atlas identifies genotype-specific immune vulnerabilities with potential relevance for precision immunotherapy in non-small cell lung cancer.

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79901879733c947d518064333e696fb75c22e645" target='_blank'>
              Divergent CD45+ immune landscapes shape the lung tumor microenvironment
              </a>
            </td>
          <td>
            S. Dizdarevic, R. Wiegandt, A. Weigert, Thorsten Stiewe, B. Eul, Stefan Guenther, F. Grimminger, Werner Seeger, Soni S Pullamsetti, Mario Looso, Kati Turkowski, R. Savai
          </td>
          <td>2026-02-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="


 Prostate cancer is genomically heterogeneous with few shared drivers. We characterized tumor-specific DNA methylation programs that define evolutionary subtypes, remodel the tumor–microenvironment ecosystem, and yield liquid-biopsy biomarkers.



 Illumina EPIC-seq methylation was profiled in matched benign/tumor tissues from the UK Prostate ICGC cohort. A covariance-based DMR caller identified robust tumor-specific regions that were intersected with bulk RNA-seq, network analyses, and single-cell/spatial transcriptomics to localize programs in situ. A FACS-sorted, cell-type methylome atlas (luminal, basal, endothelial, fibroblast, immune, smooth muscle) anchored cell-of-origin assignments. Cell-free DNA was deconvolved against this atlas to quantify luminal tumor DNA and circulating microenvironmental DNA (cmDNA). Prognostic associations used penalized Cox models.



 Tumor-specific DMRs separated cancers into two “mevotypes” mirroring genomic evotypes (ERG-fusion vs SPOP-mutant), indicating genomic–epigenomic interplay in evolutionary commitment. Cell-resolved methylomes attributed most tumor DMRs to luminal epithelium and uncovered neuronal/plasticity pathways in luminal cells, while immune/endothelial-assigned DMRs converged on ECM remodeling, angiogenesis, TGF-β, and Rho-GTPase signaling. Single-cell pseudotime and spatial mapping localized neuronal-enriched luminal programs to late-trajectory regions and a CHGA/CHGB-positive duct-adjacent epithelial circuit interfacing with fibroblasts at tumor–stroma borders. Network analysis highlighted MYC-centric modules and EZH2-linked transcription factor contexts, consistent with epigenetic priming toward quasi-neuroendocrine states in treatment-naïve localized adenocarcinoma. In plasma, deconvolution detected both luminal ctDNA and cmDNA; luminal fractions tracked tumor burden in metastatic disease (ichorCNA-concordant), whereas cmDNA—predominantly attributable to immune compartments—suggested an early, tumor-specific microenvironmental response that can precede and is independent of luminal ctDNA shedding. Methylation signatures derived from neuronal and microenvironmental gene sets associated with inferior relapse-free survival.



 Epigenetic drivers in early prostate cancer encode luminal neuronal plasticity and reorganize the tumor niche. A cell-resolved atlas enables cfDNA deconvolution to reveal luminal ctDNA plus cmDNA, supporting liquid-biopsy biomarker strategies for earlier detection and improved risk stratification in prostate cancer.



 Harveer Dev, Toby Milne-Clark, Henson Lee Yu, CRUK-ICGC Prostate Cancer Group. Epigenetic Drivers of Localised Prostate Cancers Reveal Neuronal and Microenvironmental Reprogramming and inform Liquid Biopsy Biomarkers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2503538f0dcd516b3bf974c3bb6eeaf7a1544a19" target='_blank'>
              Abstract A015: Epigenetic Drivers of Localised Prostate Cancers Reveal Neuronal and Microenvironmental Reprogramming and inform Liquid Biopsy Biomarkers
              </a>
            </td>
          <td>
            H. Dev, Toby Milne-Clark, Henson L. Lee Yu
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3789c5447c0f1d5a0fe59b4f852d13e8acfbe641" target='_blank'>
              Unlocking single-cell level and continuous whole-slide insights in spatial transcriptomics with PanoSpace.
              </a>
            </td>
          <td>
            Hui-Feng He, Pai Peng, Shi-Tong Yang, Meng-Guo Wang, Xiao-Fei Zhang, Luonan Chen
          </td>
          <td>2026-01-06</td>
          <td>Nature computational science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Tumor growth is constrained by spatial, mechanical, and metabolic factors whose alignment progressively breaks down across cellular, mesoscopic, and tissue scales as tumors expand. We hypothesize that this misalignment drives tumors toward a distinct architectural regime, termed hyperdisorder. Hyperdisorder is not defined by increased heterogeneity alone, but by the coexistence of elevated disorder across scales and spatial nonstationarity within the same tumor. Unlike ordinary randomness, where independent fluctuations diminish under spatial averaging, disorder here persists, reorganizes, or even amplifies with increasing observation scale, preventing convergence toward a stable architectural description. Using hematoxylin and eosin stained whole-slide images of gastric cancer from The Cancer Genome Atlas, we quantify tumor architecture using tile-based metrics that capture complementary aspects of organization, including texture entropy, microstructural fragmentation, orientation isotropy, and multiscale entropy variation. These measures are combined into a standardized hyperdisorder index, enabling unsupervised comparison across spatial regions. We find that architectural disruption is unevenly distributed and partially decoupled across scales within individual slides, consistent with growth-driven multiscale incoherence rather than uniform stochastic variability. Testable consequences include anomalous scaling of heterogeneity with sampling size, failure of coarse graining to converge, and systematic differences between tumor cores and invasive fronts. In diagnostic and clinical contexts, this framework clarifies when measurements from limited tissue samples are representative of the whole tumor and when they are dominated by scale- and location-dependent effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33332947ca1f92d72dfa2870dfb77a377f39ce75" target='_blank'>
              Hyperdisorder in tumor growth
              </a>
            </td>
          <td>
            Arturo Tozzi
          </td>
          <td>2026-01-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="HCC is characterized by extensive ECM remodeling, primarily mediated by stromal cells. While cancer-associated fibroblasts are known contributors to tumor fibrosis, their transcriptional diversity and role in immune modulation within the HCC tumor microenvironment remain poorly resolved. Integrative analysis of six public HCC scRNA-seq datasets was employed to identify ECM-active stromal cells. High-ECM cells were re-clustered and fibroblast subtypes were defined through differential expression, pathway enrichment, lineage scoring and cell-cell interaction modeling. Spatial transcriptomic mapping and tissue-level profiling were performed to validate and prioritize ECM-related genes for experimental analysis. COL1A1 and COL3A1 were selected for siRNA-mediated knockdown in HepG2 and HepB3 cells followed by RT-qPCR, Western blot, proliferation, colony formation and wound healing assays. ECM activity analysis identified fibroblasts as the most ECM-enriched stromal population. Reclustering of ECM-high stromal cells identified eight distinct subtypes and targeted fibroblast clustering revealed five functionally diverse states. GO and spatial transcriptomic analysis confirmed subtype-specific functions and localization. In silico tissue profiling further prioritized COL1A1 and COL3A1 as pan-mesenchymal ECM genes enriched in fibrogenic fibroblast subsets. CellChat analysis revealed myofibroblasts and inflammatory CAFs as dominant signal senders. Knockdown reduced COL1A1 and COL3A1 expression at both mRNA and protein levels, and enhanced HCC cell proliferation, migration and colony formation. This study identifies transcriptionally and functionally distinct fibroblast subtypes in HCC and highlights COL1A1 and COL3A1 as key matrix-regulatory genes expressed in fibrogenic stromal subsets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2fd864648cbb63d6bef938be32a938f167aaf27" target='_blank'>
              Single-cell profiling reveals lineage-specific fibroblast stromal subtypes drive ECM remodeling and immune modulation in the hepatocellular carcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Zhichao Jiang, Haitao Wang, Han Li, Zhiqiang Chen, Baiyu Sun
          </td>
          <td>2026-01-03</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8108044a48cdaf798eb1d753171bee039c59ab0d" target='_blank'>
              Dissecting and steering cell dynamics using spatially-informed RNA velocity with veloAgent
              </a>
            </td>
          <td>
            Brent Yoon, Vishvak Raghavan, Jun Ding
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma is traditionally characterized as a glycolytic tumor. However, the extent and clinical relevance of its metabolic heterogeneity remain poorly understood. In this study, we investigated whether glycolytic activity follows a consistent expression pattern across pancreatic ductal adenocarcinoma patients and explored how metabolic diversity influences therapeutic responses. Using spatial transcriptomics of ex vivo primary human pancreatic ductal adenocarcinoma specimens, along with single-cell and bulk RNA sequencing, we mapped glycolytic heterogeneity within the tumor microenvironment. Patient-derived cell models representing distinct glycolytic phenotypes were employed to assess metabolic profiles and responses to glycolytic pathway inhibition. A multiomics approach—including metabolomics, proteomics, and lipidomics—was integrated through a robust bioinformatics pipeline to identify pathway-specific variations. Our findings revealed pronounced glycolytic heterogeneity across pancreatic ductal adenocarcinoma tumors, with distinct transcriptional profiles that maintained cellular identity and spatial architecture. These glycolytic patterns are associated with clinical outcomes, suggesting their potential as prognostic indicators. Functional studies confirmed differential sensitivity to metabolic inhibitors in organoids and demonstrated their safety across models, supporting the therapeutic relevance of glycolytic stratification. Overall, this study reveals clinically significant metabolic heterogeneity in pancreatic ductal adenocarcinoma and proposes a glycolysis-based framework for patient stratification, which could guide personalized metabolic therapies and advance precision oncology in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a7a7cc42452c0706050a87a40d055f8405c2ca" target='_blank'>
              Glycolytic heterogeneity drives metabolic-targeted therapy in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ugo Chianese, Chiara Papulino, Gerardo Saggese, Ahmad Ali, Marianna Ciotola, Enza Lonardo, Mirko Cortese, Gregorio Favale, Annabella Di Mauro, Danila La Gioia, Valentina Golino, E. Sommella, Pietro Campiglia, Renato Franco, F. Ciardiello, F. De Vita, Vincenzo Carafa, Lucia Altucci, R. Benedetti
          </td>
          <td>2026-01-20</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Interactions among tumor, immune, and vascular niches play major roles in glioblastoma (GBM) malignancy and treatment responses. The composition and heterogeneity of extracellular core matrix proteins (CMPs) that mediate such interactions are not well understood. Here, we present an analysis of the clinical relevance of CMP expression in GBM at bulk, single-cell, and spatial anatomical resolution. We show that CMP enrichment is associated with worse patient survival, specific driver oncogenic alterations, mesenchymal state, pro-tumor immune infiltration, and immune checkpoint expression. Matrisome expression is enriched in vascular and leading edge/infiltrative niches that are known to harbor glioma stem cells. Finally, we identify a 17-gene CMP signature, termed Matrisome 17 (M17), which is a stronger prognostic factor compared to MGMT promoter methylation status as well as canonical subtypes, and importantly, may predict responses to PD1 blockade. Patient stratification based on matrisome profiles can contribute to the selection and optimization of treatment strategies. A comprehensive genomic analysis of glioblastoma tumors reveals matrisome enrichment associated with mesenchymal phenotypes and pro-tumor immune signatures, a matrisome-based classification of GBM patients, and a prognostic 17-gene matrisome signature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984fecbf9126f6a0596ab278d28e0d91cbca56fc" target='_blank'>
              A prognostic matrix gene expression signature defines functional glioblastoma phenotypes and niches
              </a>
            </td>
          <td>
            Monika Vishnoi, Zeynep Dereli, Zheng Yin, Elisabeth Kong, Meric Kinali, Kisan Thapa, O. Babur, Kyuson Yun, Nourhan Abdelfattah, Xubin Li, B. Bozorgui, M. Farach‐Carson, Robert C. Rostomily, Anil Korkut
          </td>
          <td>2026-01-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6aac86da3cbb3bd85a8ebe57c0ff220d334a71" target='_blank'>
              Circulating immune profiling reveals impaired monocyte states and trajectories driving immunosuppression in glioblastoma
              </a>
            </td>
          <td>
            A. Scafidi, T. Kaoma, Claudia Cerella, Eleonora Campus, K. Grzyb, Bakhtiyor Nosirov, Frida Lind-Holm Mogensen, E. Klein, Raul da Costa, A. Skupin, Frank Hertel, G. Berchem, Michel Mittelbronn, A. Cosma, Beatrice Melin, Simone P. Niclou, Petr V Nazarov, A. Gołȩbiewska, Aurélie Poli, A. Michelucci
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The various cellular composition of the tumor microenvironment (TME) comprises the fundamental units of tumor tissue. The types of stromal cells in the TME are genetically stable, with reduced risk of tumor recurrence and drug resistance. More and more evidence shows their clinicopathological significance and therapeutic effect in predicting prognosis. Therefore, we performed an integrated analysis of the breast cancer TME, correlating it with genomic landscapes and clinical profiles. In this work, we first conducted unsupervised hierarchical clustering on 830 tumors in the breast cancer cohort. Then, we defined three TME phenotypes and applied principal component analysis to construct a TMEscore for quantifying TME. Analysis revealed that patients stratified into the high TMEscore cohort exhibited superior survival compared to the low‐scoring group. Additionally, a high TMEscore is associated with an improved response to immunotherapy. Through TME gene signature analysis, KIF15 was identified as a pivotal driver of the immunosuppressive microenvironment in breast cancer. KIF15 knockdown may promote dendritic cell infiltration and function, thereby inducing CD8+ T cell recruitment. In summary, the immune microenvironment‐derived TMEscore represents an independent prognostic biomarker in breast cancer, while KIF15 emerges as a crucial molecular determinant of its immunosuppressive niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44851ced88ac81dcb9eaccc6f7fcac4727fd09a" target='_blank'>
              Tumor Microenvironment Characterization Identifies KIF15 as an Immunosuppressive Driver in Breast Cancer
              </a>
            </td>
          <td>
            Bo Zhang, Feiran Wang, Huiwei Huang, Xiang Zhong, Qian Chen, Xiancheng Liu, Sujit Nair
          </td>
          <td>2026-01-01</td>
          <td>Human Mutation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Penile cancer is a rare malignancy with poor prognosis in advanced and recurrent stages. The currently limited treatment options and poor survival outcomes represent an unmet need for patients with advanced or recurrent penile cancer. Innovation in therapeutic options for patients with penile cancer relies on insight of underlying tumor biology and tumor–immune system interactions. In this paper, we use multiplex immunofluorescence to investigate the immune microenvironment of penile cancer, focusing specifically on geospatial clustering patterns of various immune effector cells within various stages of penile cancer and human papillomavirus status. We identified spatial and clustering patterns of pro-immunogenic macrophages, tumor-associated macrophages, and helper T cells as potential prognosticators of survival in penile squamous cell carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14fd3b38e31e27ab5192f8dbd3698cf76eb34d9f" target='_blank'>
              Geospatial and Cell Density Analysis Using Multiplex Immunofluorescence Reveals an Important Role of Clustering Patterns of Immunosuppressive Macrophages in Survival Outcomes of Penile Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            A. Fazili, Keerthi Gullapalli, Gabriel Roman Souza, Firas Hatoum, Justin W Miller, Youngchul Kim, Junmin Whiting, Jeffrey S Johnson, J. Dhillon, Jonathan Nguygen, C. M. Segura, Philippe E Spiess, J. Chahoud
          </td>
          <td>2026-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Medulloblastoma and glioblastoma are the most common malignant primary brain tumors in children and adults, respectively. Tumor-associated macrophages and microglia are key non-cancerous cell types in these tumors. These cells interact with CD24, a so called “don’t eat me signal” expressed on tumor cells, through Siglec-10, a receptor that contributes to immune evasion by promoting an immunosuppressive environment. The CD24/Siglec-10 interaction in context of malignant brain tumors has been scarcely studied.In silico analyses reveal that CD24 gene expression correlates with specific gene signatures associated with prognosis in both medulloblastoma and glioblastoma. In both human- and mouse brain tumors, Siglec-10+ cells co-express the microglia-associated molecule TREM2. Treatment with mitoxantrone as an immunogenic cell-death-inducing cytostatic agent led to a dose-dependent reduction in cell viability and cell surface CD24 levels in both murine and human brain tumor cell cultures. Intratumoral mitoxantrone administration in a murine CD24-high glioma model extended survival, decreased tumor size, reduced Siglec-10+/TREM2+ cell populations, and increased anti-tumor CD8+ cells. These findings suggest that targeting the CD24/Siglec-10 axis with mitoxantrone may modulate the tumor microenvironment and enhance anti-tumor immunity. Keywords: CD24, Siglec-10, Mitoxantrone, Malignant brain tumor, Immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b532e86a6b6dc44d55a8d26467db7ca733b62270" target='_blank'>
              Mitoxantrone alters CD24/Siglec-10 expression in malignant brain tumor models
              </a>
            </td>
          <td>
            Jan Kopecky, Julio Enríquez Pérez, Stevanus Jonathan, Tom Milos, Poi Kwanyuen, J. Biskupiak, E. Visse, K. G. de Oliveira, H. Talbot, M. Cerezo-Magaña, V. Governa, Jinquan Cai, Marie Arsenian Henriksson, Nils Ståhl, David Cederberg, M. Belting, P. Siesjö, A. Darabi
          </td>
          <td>2026-01-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5076c3231fa10b8e1b6c94efd5aa380d1b021cf8" target='_blank'>
              Single-Cell Metabolic Profiling in a Glioblastoma Co-culture Model Using AP-MALDI-based Mass Spectrometry Imaging
              </a>
            </td>
          <td>
            U. Kontrimaite, Kei F. Carver Wong, Sandra Martínez-Jarquín, Phoebe McCrorie, Ruman Rahman, Dong-Hyun Kim
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/283f47aefbdff4ebd6faa24c4a2c93b8e125039d" target='_blank'>
              Translating spatial transcriptomic signatures in adenosquamous carcinoma into bulk prognostic biomarkers in lung adenocarcinoma: a bottom-up approach.
              </a>
            </td>
          <td>
            K. Hatakeyama, T. Kawata, Koji Muramatsu, T. Oishi, Takeshi Nagashima, Ai Sakai, Terumi Nishimura, K. Ohshima, Sumiko Ohnami, S. Ohnami, Y. Shimoda, A. Naruoka, Koji Maruyama, K. Urakami, Y. Akiyama, Y. Ohde, Takashi Sugino, Ken Yamaguchi
          </td>
          <td>2026-01-23</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Introduction Cells in tissues interact by direct physical contact or over short and long distances via secreted mediators. Cell-cell communication inference has now become routine in downstream scRNAseq analysis but this mostly fails to capture physical cell-cell interactions due to tissue dissociation. Multiplets (mostly doublets) in scRNAseq may represent undissociated physically attached cells that become sequenced together. Hence, identifying multiplets may serve as a good starting point to harness scRNAseq data for physical cell-cell interaction inference. Method In this study, we develop a computational method, called ULMnet, which utilizes univariate linear models to identify multiplets in scRNAseq datasets, predict their cellular compositions, and infer physical cell-cell interaction networks. ULMnet was applied to multiple datasets ranging from cell pairs, partially dissociated tissues to conventional scRNAseq datasets of healthy and cancer tissues. Result and Discussion Indeed, our method showed good sensitivity (~56%) with excellent precision (~99%) in predicting FACS sorted doublet cell pairs with known constituents (ground truth), recording comparable or superior performance over two other existing methods. Also, applying it to scRNAseq data of partially dissociated tissues containing real multiplets unraveled physical networks which recapitulated the microanatomical structures of the tested tissues. This further underscores the accuracy in our predictions to capture biologically meaningful interactions. Finally, we tested our method on classical scRNAseq datasets and obtained biologically reasonable results. For example, when tested on classical cancer scRNAseq datasets, we recovered important interactions which followed biologically plausible cell interactions, validated by cell-cell colocalization in matched spatial transcriptomics datasets. This reassured the accuracy of our method in depicting physical interactions only between cells that were truly in close proximity in tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25687de2db7a8e46c3677b956694abd5fba9c87c" target='_blank'>
              ULMnet: inferring physical cell-cell communication networks from scRNAseq data using univariate linear models
              </a>
            </td>
          <td>
            S. Hameed, L. F. Iglesias-Martinez, Walter Kolch, Vadim Zhernovkov
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Penile squamous cell carcinoma (PSCC) is a rare genitourinary malignancy, and factors of its tumor microenvironment (TME) could serve as prognostic indicators for tumor recurrence and metastasis. Here, we generated a comprehensive single-cell map of PSCC (66 421 cells) and identified 9 distinct cell populations with samples from nine tumor samples and six adjacent normal samples. Among the malignant cells, SEMA3Chigh Mals was found to be associated with epithelial-mesenchymal transition. T cells in tumor tissues are in a highly exhausted state, while SPP1high TAMs were observed to promote tumor progression. Cancer-associated fibroblasts were found to interact with malignant cells to facilitate EMT through several pathways. Notably, there is a specific type of pericyte called POSTN+ pericytes, which can promote angiogenesis and extracellular matrix remodeling in PSCC. Finally, SEMA3C was identified as an effective biomarker reflecting cancer stage and microvessel density. Overall, we investigated the heterogeneity of TME from a single-cell perspective and demonstrated that SEMA3C serve as an effective biomarker for predicting lymph node metastasis and prognosis in PSCC. These findings may offer valuable insights for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b5b537b7833eaaeb7747d5123ca286f841e6ac7" target='_blank'>
              Decoding the Cellular Heterogeneity and Malignant Progression of Human Penile Squamous Cell Carcinoma by Single-Cell RNA Sequencing.
              </a>
            </td>
          <td>
            Xiheng Hu, Wensheng Shi, Liang Dong, Lingjuan Huang, Xiyuan Zhang, Yiting Feng, Jie Sun, Lanlan Liu, Teng Liu, Jun Fu, Bowen Zhong, Qihao Leng, Xiaohua Wu, Minfeng Chen, Lingfang Li, Yuan Li, Xin Jin, Long Wang, Jian-xia Cao, Xin Li, Mingzhu Yin, Xiang Chen
          </td>
          <td>2026-02-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Spatial proteomics enables high-resolution mapping of protein expression and can transform our understanding of biology and disease. However, major challenges remain for clinical translation, including cost, complexity and scalability. Here we present H&E to protein expression (HEX), an AI model designed to computationally generate spatial proteomics profiles from standard histopathology slides. Trained and validated on 819,000 histopathology image tiles with matched protein expression from 382 tumor samples, HEX accurately predicts the expression of 40 biomarkers encompassing immune, structural and functional programs. HEX demonstrates substantial performance gains over alternative methods for protein expression prediction from H&E images. We develop a multimodal data integration approach that combines the original H&E image and AI-derived virtual spatial proteomics to enhance outcome prediction. Applied to six independent non-small-cell lung cancer cohorts totaling 2,298 patients, HEX-enabled multimodal integration improved prognostic accuracy by 22% and immunotherapy response prediction by 24–39% compared with conventional clinicopathological and molecular biomarkers. Biological interpretation revealed spatially organized tumor–immune niches predictive of therapeutic response, including the co-localization of T helper cells and cytotoxic T cells in responders, and immunosuppressive tumor-associated macrophage and neutrophil aggregates in non-responders. HEX provides a low-cost and scalable approach to study spatial biology and enables the discovery and clinical translation of interpretable biomarkers for precision medicine. A model trained to generate spatial proteomics profiles and predict the expression of 40 biomarkers directly from histopathology slides is shown to improve survival and treatment response prediction in patients with lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/812a5ae318d320a30cb51379e4bd5ce695c6ba0e" target='_blank'>
              AI-enabled virtual spatial proteomics from histopathology for interpretable biomarker discovery in lung cancer
              </a>
            </td>
          <td>
            Zhe Li, Yuchen Li, Jinxi Xiang, Xiyue Wang, Sen Yang, Xiaoming Zhang, Feyisope R. Eweje, Yijiang Chen, Xiangde Luo, Yuanyuan Li, Jonathan Mulholland, Colin Bergstrom, Ted Kim, F. Olguin, S. Willens, Jeffrey J Nirschl, Robert West, J. Neal, M. Diehn, Ruijiang Li
          </td>
          <td>2026-01-01</td>
          <td>Nature Medicine</td>
          <td>2</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) elicit durable responses in only a subset of patients with solid tumors, underscoring the need to define the cellular architectures that govern effective antitumor immunity. Here we identify a spatially organized multicellular immune unit comprising macrophages, cDC1s, CD4⁺ T-cells, and CD8⁺ T-cells that emerges in response to anti-CTLA-4 or dual checkpoint blockade. We term these structures tetrads. Using multiplexed imaging and spatial transcriptomics in mouse and human tumors, we show that tetrads assemble early during immune priming, depend on the ICOS–ICOSL pathway, and are enriched for ICOS⁺ Th1-like CD4⁺ T cells and ICOSLhigh cDC1s. CD8⁺ T-cells within tetrads exhibit an activated, non-terminally differentiated state, while tetrad-associated macrophages display an interferon-γ–responsive program that sustains CD8⁺ T-cell function and prevents dysfunction. Functionally, ICOSL⁺ cDC1s are required for tumor eradication in vivo. In patients with bladder cancer treated with neoadjuvant dual checkpoint blockade, tetrad, but not triad or dyad formation correlates with clinical response. These findings establish tetrads as a fundamental cellular unit coordinating antitumor immunity and responsiveness to ICIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c6a060eabfcbbb0a36b483e2912054f711b0272" target='_blank'>
              Macrophage-Dendritic Cell-T-Cell Tetrads Orchestrate Antitumor Immunity and Response to Checkpoint Blockade
              </a>
            </td>
          <td>
            Mehdi Chaib, M. Aminu, Shelley M Herbrich, Mahshid Arabi, Yue Xuan, Akshay V. Basi, Anna Casasent, Marc D. Macaluso, Kenneth H. Hu, Matthew M. Gubin, Xi Chen, James J. Mancuso, Sreyashi Basu, Sonali Jindal, Jared K. Burks, S. Watowich, Jia Wu, James P. Allison, Padmanee Sharma
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="


 Aggressive variant prostate cancers (AVPC) are a poorly understood subset of prostate cancers characterized by high mortality and extensive molecular heterogeneity. To address a lack of AVPC representative models for investigating underlying molecular and physiological mechanisms, we established NCI-LYM-1, a novel AVPC organoid/patient derived xenograft (PDX) derived from a patient with metastatic prostate cancer with small cell morphology displaying a unique clinical history and defined genomic features. It was hypothesized that this model would be able to accurately characterize biological underpinnings of the patient’s tumor profile and allow for comparative analysis to similar prostate cancers.



 NCI-LYM-1 was developed from a patient-derived AR-negative, ASCL1/Synaptophysin (SYP)-positive lymph node metastasis with small cell morphology. Short-read whole-genome sequencing (WGS) was used to identify genomic alterations and long-read WGS was used to identify epigenetic modifications. RNA-seq was used to characterize gene expression profiles. The resulting genomic, epigenetic, and transcriptomic data from NCI-LYM-1 was compared to molecular profiles from the LuCaP series of PDX models. In-vitro drug sensitivity assays were also conducted using organoid cultures.



 NCI-LYM-1 harbored biallelic inactivation of known neuroendocrine prostate cancer (NEPC) drivers: PTEN, TP53, RB1 and BRCA2, in concordance with patient diagnostic history. In addition, epigenetic evidence supported a consistent phenotype between NCI-LYM-1 and the donor tumor, including greater gene expression of neuroendocrine marker ASCL1 with decreased AR expression. Comparisons between NCI-LYM-1 and LuCaP models further supported transcriptional similarities between the model and a subset of AVPC cancer subtypes. Drug sensitivity assays were also consistent with the patient’s prior treatment exposure, displaying PARP inhibitor sensitivity but a lack of sensitivity to platinum treatments.



 NCI-LYM-1 faithfully recapitulates the phenotypic and genomic characteristics of aggressive metastatic AVPCs, including the display of neuroendocrine features. NCI-LYM-1 will be a useful model system to study molecular frameworks of understudied cancers like those of AVPC, including aspects such as drug sensitivity, disease progression, and lineage plasticity.



 Ana S. Jaramillo, Chennan Li, JuanJuan Yin, Melissa L. Abel, Dana S. Vargas, Kinjal Bhadresha, Sumeyra Kartal, Joseph Twohig, Tri M. Truong, Anish Thomas, Cindy H. Chau, Kathleen Kelly, William D. Figg, Adam G. Sowalsky. NCI-LYM-1 molecularly and phenotypically recapitulates aggressive variant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483313e79c5e529d10092ffd1001e71580f4403a" target='_blank'>
              Abstract A025: NCI-LYM-1 molecularly and phenotypically recapitulates aggressive variant prostate cancer
              </a>
            </td>
          <td>
            Ana S. Jaramillo, Chennan Li, JuanJuan Yin, Melissa L Abel, Dana S. Vargas, Kinjal Bhadresha, Sumeyra Kartal, Joseph Twohig, Tri M Truong, Anish Thomas, C. Chau, Kathleen Kelly, W. Figg, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Routine histology contains rich prognostic information in stage II/III colorectal cancer, much of which is embedded in complex spatial tissue organisation. We present INSIGHT, a graph neural network that predicts survival directly from routine histology images. Trained and cross-validated on TCGA (n=342) and SURGEN (n=336), INSIGHT produces patient-level spatially resolved risk scores. Large independent validation showed superior prognostic performance compared with pTNM staging (C-index 0.68-0.69 vs 0.44-0.58). INSIGHT spatial risk maps recapitulated canonical prognostic histopathology and identified nuclear solidity and circularity as quantitative risk correlates. Integrating spatial risk with data-driven spatial transcriptomic signatures, spatial proteomics, bulk RNA-seq, and single-cell references revealed an epithelium-immune risk manifold capturing epithelial dedifferentiation and fetal programs, myeloid-driven stromal states including $\mathrm{SPP1}^{+}$ macrophages and $\mathrm{LAMP3}^{+}$ dendritic cells, and adaptive immune dysfunction. This analysis exposed patient-specific epithelial heterogeneity, stratification within MSI-High tumours, and high-risk routes of CDX2/HNF4A loss and CEACAM5/6-associated proliferative programs, highlighting coordinated therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0506faf43ebc888bee635ffd236772260977faab" target='_blank'>
              INSIGHT: Spatially resolved survival modelling from routine histology crosslinked with molecular profiling reveals prognostic epithelial-immune axes in stage II/III colorectal cancer
              </a>
            </td>
          <td>
            Piotr Keller, Mark Eastwood, Zedong Hu, A. Selten, Ruqayya Awan, Gertjan Rasschaert, Sara Verbandt, V. Popovici, Hubert Piessevaux, Hayley T Morris, Petros K Tsantoulis, Thomas Mckee, A. D'Hoore, Cédric Schraepen, X. Sagaert, G. Hertogh, S. Tejpar, Fayyaz Minhas
          </td>
          <td>2025-12-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131d5075adfc8a892d5a30b581d5ddea0c0afd87" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular heterogeneity and prognostic subtypes in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhen Han, Yuan Gao, Yuxing Zhao, Jiahong Wu, Jia Chen, Jinling He, Hong Bai, Liang Qiao, Yali Ren, Lei Sun
          </td>
          <td>2025-12-22</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/423b9ad510934f726aebaeeb465a2ff10426e96b" target='_blank'>
              A new cancer progression model: from synthetic tumors to real data and back
              </a>
            </td>
          <td>
            Daniela Volpatto, S. G. Contaldo, Simone Pernice, Marco Beccuti, Francesca Cordero, R. Sirovich
          </td>
          <td>2026-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains incurable despite advances in treatment options. Although tumor subtypes and specific DNA abnormalities are linked to worse prognosis, the impact of immune dysfunction on disease emergence and/or treatment sensitivity remains unclear. We developed an Immune Atlas of MM by generating profiles of 1,397,272 single cells from the bone marrow (BM) of 337 newly diagnosed participants and characterized immune and hematopoietic cell populations. Cytogenetic risk-based analysis revealed heterogeneous associations with T cells of BM, with 17p13 deletion showing distinct enrichment of a type 1 interferon signature. The disease progression-based analysis revealed the presence of a proinflammatory immune senescence-associated secretory phenotype in rapidly progressing participants. Furthermore, signaling analyses suggested active intercellular communication involving a proliferation-inducing ligand and B cell maturation antigen, potentially promoting tumor growth and survival. Lastly, using independent discovery and validation cohorts, we demonstrated that integrating immune cell signatures with known tumor cytogenetics and individual characteristics significantly improves stratification for the prediction of survival. Pilcher et al. present a single-cell transcriptomics-based immune atlas of participants with newly diagnosed multiple myeloma, reporting on the association of cell types, gene expression and intercellular interactions with disease progression phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f88dd8011aa8686433c4d02b3a0e4dfd59a9255" target='_blank'>
              A single-cell atlas characterizes dysregulation of the bone marrow immune microenvironment associated with outcomes in multiple myeloma
              </a>
            </td>
          <td>
            William C. Pilcher, Lijun Yao, Edgar Gonzalez-Kozlova, Yered H. Pita-Juárez, Dimitra Karagkouni, Chaitanya R. Acharya, Marina E. Michaud, Mark Hamilton, Shivani Nanda, Yizhe Song, Kazuhito Sato, Julia Wang, Sarthak Satpathy, Yuling Ma, Jessica Schulman, D. D’souza, R. Jayasinghe, Denis J Ohlstrom, Katherine E. Ferguson, Giulia Cheloni, Mojtaba Bakhtiari, Nick Pabustan, K. Nie, J. Foltz, Isabella Saldarriaga, Rania Alaaeldin, Eva Lepisto, Rachel Chen, M. Fiala, Beena E. Thomas, April Cook, Junia Vieira Dos Santos, Chiang I-ling, Igor Figueiredo, J. Fortier, Michael Slade, Stephen T. Oh, Michael P. Rettig, Emilie I. Anderson, Ying Li, Surendra Dasari, Michael A. Strausbauch, Vernadette A. Simon, , I-ling Chiang, N. Kalavros, Jennifer Rogers, T. Dawson, Brian H. Lee, G. Kelly, Laura A Walker, Nicolas F. Fernandez, John Leech, Jarod Morgenroth-Rebin, K. Angeliadis, Matthew A. Wyczalkowski, Song Cao, Omar Ibrahim, Roderick Lin, T. Fehniger, Andrew J. Houston, E. Radkevich, Adeeb Rahman, Zhihong Chen, A. Laganà, John F. DiPersio, J. Rosenblatt, S. Kim-Schulze, S. Lonial, Shaji K. Kumar, Swati S. Bhasin, T. Kourelis, M. Dhodapkar, Ravi Vij, D. Avigan, H. Cho, George Mulligan, Li Ding, Sacha Gnjatic, Ioannis S. Vlachos, Manoj K. Bhasin
          </td>
          <td>2026-01-01</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="The formation of intratumoral spatial niches has been reported for many human malignancies. However, the translational potential of such spatial niches is understudied. Herein, we utilize digital spatial profiling (DSP) to explore the prognostic relevance of spatially defined protein expression in high-risk prostate cancer. A total of 49 patient samples were analyzed for the expression of 46 proteins in 463 regions of interest (ROIs) from the tumor center (n = 198) and the tumor periphery (n = 265) resulting in 21,298 primary data points (mean per patient n = 9.4). Expression data from either the tumor center or the tumor periphery were not found to be prognostic. Protein expression of tumor center and periphery was then integrated into single datapoints by calculating the log2-transformed relative expression between the two niches for each protein and patient. Unsupervised hierarchical clustering of these data yielded two distinct patient subgroups. These clusters did not show a statistically significant correlation with known prognostic parameters yet significantly correlated with progression-free survival (p = 0.014, log-rank, HR 0.43; 95% CI, 0.22–0.86). Our results thus reveal that spatial protein expression contains prognostic information, however, only when expression data from both spatial niches are taken into account. In conclusion, our proof-of-concept study shows that DSP can be exploited for the development of novel prognostic biomarkers that rely on spatially resolved protein expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c6b3a4672d8a49cdbb7af4005dce769ab547c41" target='_blank'>
              Prognostic impact of spatial niches in prostate cancer
              </a>
            </td>
          <td>
            Felix Schneider, Sarah Böning, Beatriz Coelho Antunes, A. Kaczorowski, M. Görtz, V. Schütz, Johannes Huber, Albrecht Stenzinger, Markus Hohenfellner, S. Duensing, A. Duensing
          </td>
          <td>2026-01-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4904674c057fccfc7a5b2f04d9d72ab6c8a77ba" target='_blank'>
              Discrete Transcriptional States Define Biphasic Immune Response and Dynamic CMS Transitions in Colorectal Cancer
              </a>
            </td>
          <td>
            Khalid Ishani, Dechen Wangmo, Atef Ali, Travis Gates, Zejie Yan, Ava Gustafson, Ella Boytim, Katie Storey, Paolo Goffredo, Justin Hwang, Subbaya Subramanian
          </td>
          <td>2026-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832c2464c80a294ed1bb7a8527b6613a4dd9fa1e" target='_blank'>
              Spatial gene expression analysis reveals drivers of extremely early lymph node metastasis in breast cancer.
              </a>
            </td>
          <td>
            Satoi Nagasawa, Keiko Kajiya, Erina Ishikawa, A. Kanai, Ayako Suzuki, A. Motoyoshi, Tsuguo Iwatani, Manabu Kubota, Masaru Nakamura, T. Onishi, Akiyoshi Hoshino, Ichiro Maeda, Akihiko Morozumi, Kenji Takatsuka, Junki Koike, M. Seki, K. Tsugawa, Yutaka Suzuki
          </td>
          <td>2026-01-23</td>
          <td>NPJ breast cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53e9617142706d92615a2fad08d2ab214304d6df" target='_blank'>
              Real-time monitoring of pediatric high-grade glioma invasion using organotypic brain cultures reveals a developmentally dependent sensitivity
              </a>
            </td>
          <td>
            Yoav Biala, Ramakrishna Sai kumar Aravapalli, Hala Shalata, Zakhariya Manevitch, Oded Behar
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271f2b42923ccb0c522e0defc7aee243eaf5e39d" target='_blank'>
              Reconstructing clone-resolved transcriptional programs from bulk tumor sequencing
              </a>
            </td>
          <td>
            Jiaying Lai, Yi Yang, Kathleen Noller, YunZhou Liu, Archana Balan, Prathima B. Nagendra, Luciane T. Kagohara, E. Fertig, Laura D. Wood, Rachel Karchin
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Simple Summary Epithelial-to-mesenchymal transition (EMT) is a hallmark of cancer and plays a critical role in breast cancer progression. Here, we analyzed the expression of EMT-related markers, including Vimentin, E-cadherin, Cytokeratin-18, and α-SMA, in a cohort of 95 breast cancer tissue samples, revealing marked intra- and inter-tumoral heterogeneity. Positive correlations between epithelial and mesenchymal markers supported the presence of hybrid epithelial/mesenchymal phenotypes and high cellular plasticity. The investigation of the molecular basis of this plasticity was performed by integrative bioinformatics analyses, leading to the identification of a novel EMT gene signature significantly associated with prognosis. Functional enrichment analyses underscored the dynamic interplay between tumor cells and microenvironment. Moreover, a gene cluster associated with cancer stem cell-like features that may be clinically relevant for patient risk stratification was identified. Overall, our findings emphasize the complexity of EMT regulation in breast cancer and propose a new EMT signature with potential prognostic/therapeutic relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8f918695720cb00b27c92a80788402b1520e396" target='_blank'>
              Decoding the Molecular Drivers of Epithelial to Mesenchymal Transition in Breast Cancer: Insights into Epithelial Plasticity and Microenvironment Crosstalk
              </a>
            </td>
          <td>
            Emanuela Peri, M. Buttacavoli, E. Roz, I. Pucci‐Minafra, Salvatore Feo, Patrizia Cancemi
          </td>
          <td>2026-02-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 PSMA expression is heterogeneous across advanced prostate cancer, limiting the durability of PSMA-radioligand therapy (RLT). To characterize tumor heterogeneity and identify determinants of RLT resistance, we established the SHAPE cohort (Spatial Heterogeneity Atlas of Prostate Cancer Evolution). SHAPE is a multimodal resource integrating PET imaging, clinical treatment history, histopathology, immunohistochemistry, spatial transcriptomics, and genomic profiling across 46 patient tumor samples collected through rapid autopsy, biopsy, and surgical resection. The cohort spans adenocarcinoma, castration-resistant (CRPC), and neuroendocrine (NEPC) prostate cancer states and includes eight xenograft models.



 Twenty-five samples profiled using the 10x Genomics Visium spatial transcriptomics platform, comprising 195,905 spots, were integrated and analyzed using Seurat (v5) in R. Targeted genomic profiling was performed using the Dana-Farber Cancer Institute OncoPanel platform on 36 samples derived from 7 patients and 2 animal models. Among the cohort, seven cases had prior Lu-PSMA exposure, and ten cases were untreated. Eleven clinical and preclinical samples had PSMA PET/CT, including six with both pre- and post-treatment imaging. Data integration connected structural genomic events with spatial lineage programs and imaging features.



 RLT–treated tumors exhibited a copy-number–dominant profile compared with untreated tumors. Recurrent alterations enriched in treated tumors included PTEN deletion (p<0.01), KLLN deletion (p<0.01), AURKA gain (p<0.01), and ZNF217 gain (p<0.01). Treated tumors also showed higher per-sample copy number loss (47.6 vs 37.5, p<0.05) and amplification burden (7.91 vs 2, p<0.05). In contrast, untreated tumors displayed more single nucleotide variants (14.9 vs 6.3, ns) and indels (4.7 vs 1.9, ns). Spatial transcriptomics provided complementary insights. Treated samples with PSMA SUV below the liver reference demonstrated higher copy-number loss burden (p<0.05) and contained unique transcriptomic clusters enriched for glucose metabolism. These clusters displayed variable expression of classical NEPC markers but consistently upregulated genes involved in glucose uptake and utilization. In preclinical studies, head-to-head FDG and Ga-PSMA imaging of WCM12 xenografts demonstrated that FDG, as a surrogate marker of glucose uptake, was significantly increased following relapse after PSMA-RLT (FC, p<0.05).



 The SHAPE cohort integrates genomic, spatial, and imaging data to characterize tumor heterogeneity in advanced prostate cancer. Genomic profiling identified PTEN/KLLN loss and AURKA/ZNF217 gain as recurrent alterations enriched in Lu-PSMA–exposed tumors, while spatial transcriptomics revealed PSMA-low metastases with distinct. clusters enriched for glucose metabolic programs. Together, SHAPE can inform rational strategies for combination or alternative targeted therapies.



 Martin K. Bakht, Jacob Egelberg, Yasutaka Yamada, Louise Clark, Xiao Wei, Varadha Balaji Venkadakrishnan, Anthony P. Belanger, Alice Bernard-Tessier, Sarah J. Hill, Francesca Khani, David J. Einstein, Mary-Ellen Taplin, Eliezer Van Allen, Heather A. Jacene7, Himisha Beltran. Spatial heterogeneity atlas of prostate cancer evolution (SHAPE): Spatial and genomic landscapes of advanced prostate cancer with and without Lu-PSMA therap [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912633affb19287de4d61286123b6004f19890a4" target='_blank'>
              Abstract PR027: Spatial heterogeneity atlas of prostate cancer evolution (SHAPE): Spatial and genomic landscapes of advanced prostate cancer with and without Lu-PSMA therapy
              </a>
            </td>
          <td>
            Martin K. Bakht, Jacob Egelberg, Yasutaka Yamada, Louise Clark, Xiao X Wei, V. B. Venkadakrishnan, Anthony P. Belanger, Alice Bernard-Tessier, Sarah J. Hill, F. Khani, David J Einstein, M. Taplin, E. V. Van Allen, Heather A. Jacene, H. Beltran
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Immunotherapies using CAR T cells are revolutionizing B-cell acute lymphoblastic leukemia treatments. However, the majority of patients remain unresponsive, and chronic stimulation of T cells is a common contributor that reduces effector function and persistence. We apply Molecular Pixelation, a recently developed single-cell technology for characterizing cellular surface proteomes, to determine characteristic topological surface-based proteomic signatures of CAR T cell exhaustion. We analyze 76 surface proteins on 8504 CAR T cells at a single-cell level, collected from three donors and either stimulated once or repeatedly, six times over two weeks. The abundances, polarizations, and colocalizations of surface proteins can each distinguish CAR T cells that were stimulated acutely or chronically, and all but one marker with polarization changes increased in polarization. These data also reveal disrupted adhesion signatures of protein colocalization in the peripheral supramolecular activation complex (pSMAC) and increased CD37/CD82 colocalization after chronic stimulation. These Molecular Pixelation results convey new spatial signatures for proteomic polarization and colocalization on the cell surface that represent new cell-state axes for immunology and systems biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f42f2edf6fca1aaa5ae0d0d4bd07f6aa08cdb17" target='_blank'>
              Molecular pixelation of the CAR T cell surface proteome
              </a>
            </td>
          <td>
            Anthony J. Cesnik, Oliver Takacsi-Nagy, Trang Le, Theodore L. Roth, Ansu Satpathy, Emma Lundberg
          </td>
          <td>2026-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Objective: This study aimed to identify clinically relevant regulators of pancreatic ductal adenocarcinoma (PDAC), a disease characterized by stromal remodeling and immune suppression, and to define their links to malignant progression and microenvironmental reprogramming. Methods: We integrated multi-cohort bulk, single-cell, and spatial transcriptomic datasets and subsequently validated bulk differential expression and network analyses with machine learning-based prioritization in an independent combined cohort (TCGA-PAAD plus GSE62452). Single-cell mapping was used to assess cell-type specificity, positioning candidates along inferCNV- and pseudotime-defined malignant continua. In Visium sections, a DKK1-associated program score quantified intratumoral spatial heterogeneity and informed our analyses of ligand–receptor communication. Bulk immune deconvolution linked gene levels to immune infiltration patterns, and functional assays were used to test the impact of DKK1 knockdown on migration, proliferation, clonogenic growth, and apoptosis in PDAC cells. Results: Four reproducible tumor-associated genes—DKK1, COL10A1, SULF1, and SLC24A3—were prioritized and validated externally. DKK1 was predominantly expressed by epithelial tumor cells and tracked along a malignant progression continuum. Spatially, the DKK1 program localized to epithelial-dominant regions, revealed pronounced intratumoral heterogeneity, and highlighted epithelial–endothelial and endothelial–immune signaling in high-score areas. Immune deconvolution associated higher DKK1 expression with increased myeloid infiltration and reduced cytotoxic lymphocyte signatures. Functionally, DKK1 knockdown impaired migration, proliferation, and clonogenicity while increasing apoptosis. Conclusions: We demonstrate that DKK1 is an epithelial-derived regulator linked to malignant progression and tumor–stroma–immune remodeling, supporting its potential as a biomarker and therapeutic target in PDAC treatment, including rational combinations with stroma-modulating strategies and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96a91f34d70864c659b210aeaa2777d7f019925a" target='_blank'>
              Multi-Omic and Spatial Profiling Identifies an Epithelial DKK1 Associated with Microenvironmental Remodeling in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Jiajia Xu, Kaiqiang Qian, Yanyu Ding, Jianghao Cheng, Xu Zhang, Yong Huang, Bo Liu
          </td>
          <td>2026-02-05</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVE
Glioma recurrence severely impacts patient prognosis, with current treatments showing limited efficacy. Traditional methods struggle to analyze recurrence mechanisms due to challenges in assessing tumor heterogeneity, spatial dynamics, and gene networks. Single-cell combined spatial transcriptomics (ST) offers innovative solutions.


METHODS
We analyzed glioma mRNA data from TCGA and single-cell and ST data from GEO. Following quality control, dimensionality reduction, clustering, and cell annotation of single-cell sequencing data, we identified cell types exhibiting significantly aberrant distributions between primary and recurrent samples by analyzing the deviation degree of Ro/e values. Fibroblasts demonstrating the greatest intergroup differences were subsequently selected as the key cellular population for further investigation. Key differentially expressed genes (DEGs) were identified via random survival forest analysis. Drug sensitivity was assessed using GDSC. Deconvolution algorithms mapped cellular spatial distribution, while PROGENy quantified pathway activity. MISTy modeling revealed cell-cell interactions.


RESULTS
Fibroblasts were the primary recurrence-associated subpopulation, with marker genes enriched in extracellular matrix and adhesion pathways. AEBP1, ZNF708, and TSHZ2 were identified as key genes: AEBP1/TSHZ2 correlated with poor prognosis, while ZNF708 showed an inverse trend. These genes were linked to chemosensitivity (Irinotecan, Carmustine, Vincristine, and Cisplatin). Recurrent tumors exhibited increased plasma cell infiltration, with key genes regulating IL-17, Notch, and Toll-like receptor pathways. Spatial analysis highlighted oligodendrocyte-astrocyte interactions in the tumor microenvironment.


INTERPRETATION
Fibroblasts drive glioma recurrence, with AEBP1, ZNF708, and TSHZ2 predicting recurrence and chemoresistance. These genes promote immune suppression (via plasma cells) and activate recurrence pathways. Oligodendrocyte-astrocyte interactions shape the recurrent microenvironment, suggesting new therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec7c524a9377c361243faed8c72c15c26a0c97d" target='_blank'>
              Unraveling the Molecular Mechanisms of Glioma Recurrence: A Study Integrating Single-Cell and Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Lei Qiu, Yinjiao Fei, Jiaxuan Ding, Kexin Shi, Jinyan Luo, Yuchen Zhu, Xingjian Sun, Gefei Jiang, Yuandong Cao, Weilin Xu, Shu Zhou
          </td>
          <td>2026-01-06</td>
          <td>Annals of clinical and translational neurology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chronic obstructive pulmonary disease (COPD) is clinically and molecularly heterogeneous. To investigate COPD heterogeneity, we profiled lung tissue by single-nucleus RNA sequencing from 141 study participants (1,516,727 nuclei) and identified shifts in cell composition and emergent cell states that correlated with lung function, emphysema and composite symptom scores. Epithelial regenerative states peaked in early COPD and declined thereafter, whereas inflamed nonimmune cells and profibrotic/remodeling states, together with select immune populations, expanded with disease progression. Clustering study participants by the proportion of pathologic cells coupled with spatial transcriptomics identified distinct patterns of cellular co-occurrence within spatially localized niches. Proteomic analyses identified plasma biomarkers of cell states and their impact on the extracellular matrix. Mediation and cell communication analyses revealed cell-autonomous and intercellular communication networks associated with disease. These data define the cellular landscape of COPD heterogeneity, revealing molecular drivers and biomarkers that could inform therapeutic strategies. This study uses single-nucleus RNA sequencing to analyze lung tissue from 141 individuals with chronic obstructive pulmonary disease. Distinct patterns of spatially resolved changes in cell composition and cell states that correlate with clinical features are highlighted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2abe4b42866b70674a2326e827a11e496603c2" target='_blank'>
              Aberrant cellular communities underlying disease heterogeneity in chronic obstructive pulmonary disease
              </a>
            </td>
          <td>
            Yuening Zhang, Huanhuan Wei, J. Nouws, Wenhao Jiang, Reginald M Brewster, Jenny P Nguyen, SiRu Liang, Samuel M Pass, Weiwei Wang, Florine Collin, A. Oill, Sang-Hun Kim, Saul S. Siller, Jinjiang Liu, Amy Zhao, Phillip Hansbro, Charles Dela Cruz, Clemente Britto, Jose L. Gomez, Suzanne M. Cloonan, Erica L Herzog, Tukiet Lam, N. Banovich, M. Raredon, Xuchen Zhang, S. Mangiola, R. Homer, N. Kaminski, John E. McDonough, F. Polverino, Xiting Yan, M. Sauler
          </td>
          <td>2026-01-23</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Rising incidence of early-onset prostate and other solid tumors underscores the need for experimental systems that model how normal tissues traverse premalignant states, acquire mutations, and become therapy-responsive malignancies under authentic immune and stromal pressures. A major barrier has been the lack of tractable in vivo platforms that enable genome-wide discovery while preserving continuous tumor evolution without catastrophic chromosomal instability. To address this, the Bose Lab developed Stochastically Emergent Tumors (SETs), an organoid-derived in vivo evolution engine that redefines discovery for early-onset and understudied patient groups. In this system, mismatch-repair deficiency is induced in non-malignant human organoids, which are passaged to accumulate stochastic point mutations and transplanted into mice to permit malignancy to emerge under physiologic selection. SETs evolve primarily through high-resolution point mutations rather than broad copy-number changes, yielding bioinformatically tractable clonal dynamics ideally suited for whole-genome driver discovery and machine learning. Compared with conventional xenografts, SETs display greater intertumoral heterogeneity and reproducible recovery of sensitizing and resistance alleles under therapeutic pressure. As proof of principle in prostate cancer, endocrine therapy applied to SET pools recovered known determinants of androgen-pathway sensitivity and uncovered new drivers. Loss of ZFHX3, typically obscured within a multigene suppressor locus in bulk cohorts, promoted luminal histology and sensitized tumors to androgen-receptor inhibition in vivo, whereas KMT2D or CIC alterations mediated resistance. Consistent with model predictions, ZFHX3 loss in patients correlated with significantly improved survival, a finding comparable in magnitude to the most favorable molecular subtypes of advanced prostate cancer. SETs also quantify evolutionary thresholds: in a Pten-null background, approximately 900 coding mutations accumulated over 208 days were sufficient for malignant transformation in half of grafts. This defines a measurable axis linking mutation burden, genotype, and tumor incidence. Because SETs generate neoantigen-rich point-mutation landscapes, they can be extended to immunocompetent hosts to study tumor–immune coevolution, early T-cell surveillance, macrophage-mediated immune exclusion, and myeloid checkpoints that enable immune escape. In summary, SETs provide a scalable, evolution-aware platform that connects mutational dynamics to therapeutic vulnerability, enabling identification of lineage- and ancestry-associated drivers, immunopreventive targets, and biomarkers of early-onset prostate cancer.



 Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, Valbona Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C.K. Cornelia Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, Charles Sawyers, Felix Feng, Hani Goodarzi, Rohit Bose. Reconstructing prostate evolution with Stochastically Emergent Tumors (SETs) reveals in vivo therapeutic vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23678ab77681ac1f2c46c1e6d9b9824f918c99b" target='_blank'>
              Abstract A008: Reconstructing prostate evolution with Stochastically Emergent Tumors (SETs) reveals in vivo therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, V. Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C. Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R Karthaus, Charles Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND
Meningioma brain invasion encumbers surgical resection and increases the risk of tumor recurrence, but the molecular mechanisms underlying this process are poorly understood.


METHODS
To identify molecular and cellular features of brain-invasive meningiomas, we (1) analyzed bulk RNA sequencing data from 199 meningiomas, including 33 brain-invasive tumors, (2) analyzed patient-matched single-cell RNA sequencing data of spatially mapped meningioma samples from the tumor core or brain-tumor interface (BTI), and (3) performed spatial transcriptomic sequencing of brain-invasive meningioma samples. Multiplexed immunofluorescence (IF) was used to validate bioinformatic spatial expression patterns. Functional interactions between meningioma cells and neurons were studied in meningioma/neuron co-cultures using confocal microscopy, multi-electrode array recordings, and live cell calcium imaging.


RESULTS
Transcriptomic analyses showed conserved enrichment of TGM2, S100A11, ZYX, and PDGFRA at the BTI across bulk, single-cell, and spatial RNA sequencing datasets. The expression of these genes at the BTI was confirmed using multiplexed IF, and single-cell bioinformatic and microscopy analyses further demonstrated enrichment of macrophages at the BTI. Co-culture assays showed neuronal hyperexcitability and increased proliferation of meningioma cells, suggesting functional communication between meningioma cells and the tumor microenvironment may contribute to meningioma growth in cases with brain invasion.


CONCLUSIONS
Meningioma brain invasion is defined by molecular remodeling of tumor cells and functional interactions within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4352a0cf42108bac27d918f350130e856ef324b" target='_blank'>
              Meningioma cell reprogramming and microenvironment interactions underlie brain invasion.
              </a>
            </td>
          <td>
            Ayush Aggarwal, Mark W. Youngblood, T. Picart, H. Najem, S. Oten, M. A. Cady, Stephen T. Magill, Craig M Horbinkski, James P. Chandler, A. Heimberger, K. Mirchia, S. Hervey-Jumper, D. Raleigh
          </td>
          <td>2025-12-31</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4167a37773942f9ca4b809bad0d95f60d49ec113" target='_blank'>
              Impacts of mutation accumulation and order on tumor initiation revealed by engineered murine colorectal cancer organoids
              </a>
            </td>
          <td>
            Yanping Li, Xiaoxin Xie, Daqi Deng, Zhiyuan Sun, Zhaoan Huang, Yisen Tang, Liang Fang, Wei Chen, Qionghua Zhu
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a2ccfb681e29a65f21a96b62e66cbd637cbe33" target='_blank'>
              CELLWHISPER: INFERENCE OF DIRECT CELL-CELL COMMUNICATION FROM SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            Anurendra Kumar, Felix Rivera, Bhavay Aggarwal, Nicholas Zhang, Ahmet F. Coskun, Saurabh Sinha
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Sinonasal squamous cell carcinoma (SNSCC) is a rare malignancy with poorly understood molecular drivers. Consequently, its cellular composition and tumor microenvironment (TME) remain largely undefined. Here, we performed integrated bulk and single-nucleus multi-omic analyses to map the SNSCC ecosystem. Within the malignant compartment, we identified five distinct populations, with hypoxic (TC1) and proliferative (TC2) subtypes associated with adverse clinical outcomes. Functionally, TC1 cells orchestrate a hypoxia-driven angiogenic program via coordinated secretion of adrenomedullin (ADM), MIF, and VEGFA, promoting endothelial tip cell (EC1) differentiation. Integrative analysis revealed these transcriptional programs are underpinned by tumor-specific chromatin accessibility and DNA hypomethylation, particularly at AP-1-enriched regulatory elements. Mechanistically, in vitro studies confirmed that this response depends on cooperative AP-1 and HIF1A signaling. Furthermore, histological analysis of patient tissues demonstrated spatial co-localization of GLUT1-expressing TC1 cells with DLL4-positive EC1 cells. These findings elucidate the epigenetic landscape underlying tumor-stromal interactions and establish the ADM/VEGFA axis as a critical therapeutic target to disrupt epigenetically controlled angiogenesis in SNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bae1ad73527bcabf41b52f28eb9547020179867e" target='_blank'>
              Single-Nucleus Multi-Omics Reveals Hypoxia-Driven Angiogenic Programs and Their Epigenetic Control in Sinonasal Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, J. Noh, Jaehyun Lee, Geunho Kwon, Myeong S. Yu, Yoo-Sam Chung, Seung-Jun Lee, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2026-01-07</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5e712867c904dd12e6307954591e726635cae8e" target='_blank'>
              Differences in T-cell counts and neighborhood patterns in human colorectal adenomas and sessile serrated lesions
              </a>
            </td>
          <td>
            Souvik Seal, Lauren R. Fanning, Evan Bagley, Christine Bookhout, Elizabeth L. Barry, Elizabeth O’Quinn, Dale C. Snover, David N. Lewin, Silvia Guglietta, A. Kourtidis, M. Shrubsole, John A. Baron, Todd A. Mackenzie, Alexander V. Alekseyenko, Kristin Wallace
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="The immune microenvironment profoundly shapes the progression and therapeutic response of thyroid carcinoma. Through comprehensive analysis of single-cell RNA sequencing data, this review delineates the immune landscapes of papillary thyroid carcinoma (PTC), poorly differentiated thyroid carcinomas (PDTC), and anaplastic thyroid carcinomas (ATC), revealing a differentiation dependent trajectory of tumor immune microenvironment remodeling—from immune activation suppression coexistence in PTC, to immune exclusion in PDTC, and terminal exhaustion in ATC. This single-cell based approach enables high resolution dissection of cellular heterogeneity, immune crosstalk, and spatial organization that are often masked in bulk analyses. Such insights provide a scientific basis for precision immunotherapy, offering guidance for differentiation tailored strategies to overcome immune escape and improve clinical outcomes in thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/662dd84def56730e4ddb7a83c80225823dda796d" target='_blank'>
              Single-cell sequencing reveals the tumor immune microenvironment in thyroid cancer: a narrow review
              </a>
            </td>
          <td>
            Kangcong Liang, Ziyu Wang, Zhiqiang Zhang, Gengluan Liu, Xidi Wang, Heng Cao, Ming Zhong, Liping Ye, Xin Zhong, Jingyu Xun, Kefeng Lei, Ningning Li
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer is a highly malignant disease, posing a significant threat to global health. The presence of tumor heterogeneity results in substantial variations in prognosis and therapeutic responses among patients. Advances in bulk RNA sequencing and single-cell RNA sequencing have facilitated the identification of driver gene mutations and the exploration of cellular diversity within tumors. However, tumors are complex ecosystems comprising both tumor cells and their microenvironment, where interactions among different cell types give rise to specific functional structural units that collectively drive tumorigenesis and progression. The emergence of spatial omics technologies has allowed for the analysis of tumor ecosystems, providing unprecedented insights into tumor heterogeneity. This review aims to present updates on spatial omics technologies and data analysis algorithms, discuss current technical limitations, and explore potential future developments. Furthermore, we summarize the latest applications of spatial omics in elucidating lung cancer heterogeneity, investigating mechanisms of lung cancer progression and drug resistance, and identifying novel biomarkers. Based on these findings, we propose strategies for integrating spatial omics into lung cancer research, offering new perspectives for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eca909ba170b3407f8c473dadaa8f67180cd2619" target='_blank'>
              The Evolution of Spatial Omics Technologies Introduces A Novel Avenue for Lung Cancer Research.
              </a>
            </td>
          <td>
            Yue He, Zifan Li, Wenxiang Wang, Xu Liu, Shanshan Lu, Jing Bai, Lin Weng, Qingna Zhang, Jun Wang, Kezhong Chen
          </td>
          <td>2026-01-30</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer liver metastasis (CRLM) is a leading cause of cancer-related death in patients with colorectal cancer (CRC) and remains a major clinical challenge. Emerging evidence indicates that metabolic reprogramming within the tumor microenvironment profoundly influences immune cell behavior and disease progression, yet the specific role of lipid metabolism in macrophage heterogeneity during CRLM is still unclear. We integrated single-cell RNA sequencing and spatial transcriptomics to characterize lipid metabolic states, differentiation trajectories, intercellular communication, and spatial distribution of macrophages across normal colon, primary CRC (pCRC), and CRLM. Lipid metabolic activity was quantified using multiple scoring algorithms, pseudotime analysis was performed with the “Monocle” R package, cell–cell communication was evaluated using the “CellChat” R package, and spatial localization was inferred using the RCTD method and the “MISTy” R package. Macrophages exhibited the highest lipid metabolic activity among immune cells, and this activity was significantly elevated in CRLM compared with pCRC. Subclustering analysis identified three lipid metabolism–related macrophage subsets (CD1C⁺, CXCL10⁺, CX3CR1⁺) enriched in CRLM, among which CD1C⁺ and CXCL10⁺ subsets displayed distinct differentiation dynamics. Cell–cell communication analysis revealed metastasis-specific remodeling, including diversified VEGF signaling in CD1C⁺ macrophages and enhanced SPP1-integrin and C3–(ITGAX + ITGB2) interactions in CXCL10⁺ macrophages. Spatial transcriptomics analysis showed that lipid metabolism activity was highest in macrophage-rich regions of the normal colon, higher in epithelial and tumor regions of pCRC than in fibroblast regions, and highest in hepatocyte regions of CRLM. Moreover, CD1C⁺ macrophages predominantly localized to tumor regions in both pCRC and CRLM and maintained close associations with other immune cells. Lipid metabolism shapes macrophage heterogeneity and function states during CRLM progression, with CD1C⁺ populations emerging as key macrophage subsets linked to metastatic niche remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/227b6dd6f421783cfe0a266a6aeffd63f7f91d00" target='_blank'>
              Integrative single-cell and spatial transcriptomic analysis reveals lipid metabolism–mediated macrophage heterogeneity during colorectal cancer liver metastasis progression
              </a>
            </td>
          <td>
            Xin Qi, Siyu Hou, Shaozhuo Xie, Jiajia Chen
          </td>
          <td>2025-12-22</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most lethal brain tumors, characterized by extensive immune evasion and a macrophage-dominated tumor microenvironment (TME). However, the molecular determinants governing tumor-associated macrophage (TAM) states and their immunoregulatory functions remain poorly understood. We integrated bulk- and single-cell transcriptomic datasets (TCGA, CGGA, Ivy GAP, and Brain Immune Atlas) to systematically characterize the expression, prognostic relevance, and immune contexture of the myeloid biomarker membrane-spanning 4-domain A6A, MS4A6A, in GBM. Differential expression, survival, and pathway enrichment analyses were performed. Single-cell mapping and CellChat modeling delineated MS4A6A-associated TAM subpopulations, intercellular communication networks, and ligand–receptor signaling dynamics. Spatial transcriptomic validation and pharmacogenomic modeling were conducted to assess anatomic enrichment and therapeutic vulnerabilities. High MS4A6A expression predicted unfavorable survival and correlated with increased stromal and immune infiltration. Single-cell analyses localized MS4A6A predominantly to TAMs, especially Regulatory- and Ribo-TAM states enriched for antigen presentation, T-cell regulation, and ribosomal biogenesis pathways. CellChat analysis revealed that MS4A6A-high TAMs exhibited markedly enhanced communication with CD4+ T cells and Tregs through upregulated PGE2–PTGER2/PTGER4, PECAM1–CD38, and THBS1–CD36 signaling axes, implicating MS4A6A in prostaglandin-driven immune suppression. Spatial profiling confirmed preferential localization of MS4A6A within perivascular and angiogenic niches. Pharmacogenomic prediction indicated that MS4A6A-high tumors were more sensitive to ERK, mTOR, and CDK4/6 inhibition. MS4A6A defines a macrophage-centered, immunosuppressive ecosystem in GBM, mediated by the activation of the PGE2 signaling axis. These findings position MS4A6A both as a prognostic biomarker and as a potential therapeutic node linking myeloid reprogramming to actionable pathway vulnerabilities in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aace77685200c650b039b4e217d58ff78ecf5af" target='_blank'>
              The Myeloid Biomarker MS4A6A Drives an Immunosuppressive Microenvironment in Glioblastoma via Activation of the PGE2 Signaling Axis
              </a>
            </td>
          <td>
            Jianan Chen, Qiong Wu, Anders E. Berglund, Robert J. Macaulay, James J. Mulé, Arnold B. Etame
          </td>
          <td>2025-12-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7129daca11db4cd5b5b60ff63c51368361b7db2" target='_blank'>
              Multimodal single-cell profiling reveals crosstalk between macrophages and stromal cells in poor prognostic cholangiocarcinoma patients.
              </a>
            </td>
          <td>
            Lara Heij, Sikander Hayat, Konrad Reichel, Sidrah Maryam, C. O'Rourke, X. Tan, Marlous van den Braber, J. Verhoeff, Maurice Halder, Fabian Peisker, G. Wiltberger, Jan Bednarsch, Daniel Heise, Julia Campello Deierl, Sven A Lang, F. Ulmer, Tom Luedde, E. Dahl, Danny Jonigk, Jochen Nolting, Shivan Sivakumar, J. Siveke, Florian Vondran, Flavio G. Rocha, H. Baba, S. Hartmann, Jesper B. Andersen, Zaynab Hobloss, A. Ghallab, J. Hengstler, J. J. G. Vallejo, R. Kramann, Ulf Neumann
          </td>
          <td>2026-01-28</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly invasive and refractory subtype of breast cancer. Despite the promise of immune checkpoint blockade (ICB) therapy, response rates remain limited. The immune resistance driven by the tumor microenvironment has not yet been understood entirely, which hinders the personalized precision treatment of TNBC. We integrated single-cell RNA data from 12 cohorts with TNBC and performed a multi-omics analysis combining spatial transcriptomics (ST), bulk RNA sequencing, and multiplex immunofluorescence (mIF) staining to identify immune-resistant subpopulations. Cell-to-cell communication was explored based on NicheNet and CellChat, and the function of CAF was verified by gene knockdown and overexpression in human mammary fibroblasts, followed by co-culture experiments with TNBC cell lines. ST and mIF data were used to analyze and verify cellular co-localization, while deconvolution was used to examine the relationship between two-cell characteristics and immunotherapy or antibody–drug conjugates (ADC) agent benefit. We identified CA9+cancer-associated fibroblasts (CA9+CAF) as a key subset enriched in non-responders to ICB that promotes immune resistance by establishing a hypoxic and immunosuppressive microenvironment via abnormal angiogenesis and glycolysis. ST and mIF analyses revealed a strong co-localization and interaction between CA9+CAF and SPP1+tumor-associated macrophages (SPP1+TAM), forming a stroma-myeloid axis that promotes immune escape through VEGFA/NRP2 axis in co-localization core region compared to the boundary. In vitro experiments demonstrated that the over-expression of CA9 in fibroblasts enhanced the proliferation, invasion, and migration of TNBC cells, while CA9 knockdown inhibited the tumorigenic effects. The high CA9+CAF/SPP1+TAM profile indicated a poor prognosis, reduced effector T cell infiltration, and attenuated response to immunotherapy, may benefit from TROP2, MUC1, and NECTIN4-based ADC agents. The result was validated in TNBC samples treated with neoadjuvant immunotherapy from our center. This study unveils the critical immunosuppressive axis orchestrated by CA9+CAF and SPP1+TAM in TNBC, offering novel insights into the stromal regulatory mechanisms driving immune resistance. The cell-to-cell interaction signature holds promise as predictor of immunotherapy response and potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9990df240d9cf9afd4d976bdcb6753a231a07876" target='_blank'>
              CA9+ cancer-associated fibroblasts cooperate with SPP1+ tumor-associated macrophages driving immune resistance in triple-negative breast cancer
              </a>
            </td>
          <td>
            Qin Ma, Jing Wang, Qian Jiang
          </td>
          <td>2026-01-09</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a major issue in the sphere of head and neck cancer since it is very heterogeneous, which also leads to the poor treatment results and low survival rates in the advanced stages. Here, the review will discuss how spatial omics methods explore tumor heterogeneity in OSCC, which includes cellular, molecular, and immune microenvironment alterations that occur due to cancer stem cells, stromal interactions, genetic instability, epigenetic reorganization, and metabolic reprogramming. The focus of heterogeneity is given on the contribution of the tumor microenvironment such as immune cells, cancer-associated fibroblasts and extracellular matrix remodeling to the stimulation of progression, metastasis, as well as therapeutic resistance. The introduction of spatial omics technologies, including spatial transcriptomics, proteomics, and metabolomics, has revolutionized the field by preserving tissue architecture, enabling high-resolution mapping of gene expression, protein distribution, and metabolite profiles. Significant developments around the spatial omics technologies are discussed, while how they are used in the identification of ligand-receptor network, signaling pathways, and spatial patterns of heterogeneity in OSCC are described. Integration of multi-omics approaches bridges gaps between transcriptomic, proteomic and metabolic, facilitating the discovery of biomarkers for prognosis, immune evasion mechanisms, and precision therapies targeting epithelial-to-mesenchymal transition and immunosuppressive networks. Despite challenges in data integration, cost, and clinical translation, spatial omics holds promise for personalized oncology, with future directions involving artificial intelligence-driven modeling to enhance diagnostic accuracy and therapeutic efficacy in OSCC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00bf591ef53e5e31062caa9f75b7e97f0abcc8b" target='_blank'>
              Spatial Omics in Decoding Oral Squamous Cell Carcinoma Heterogeneity: Microenvironment Crosstalk and Multi-Omics Integration
              </a>
            </td>
          <td>
            Xiaotong You, Yu Liu
          </td>
          <td>2025-12-31</td>
          <td>GenoMed Connect</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Dogs play a major role in sustaining transmission of Leishmania infantum to people, thus prevention and treatment of canine leishmaniosis (CanL) to reduce transmission represents an unmet public health need. Although advances have been made in understanding how immunopathology correlates to infectiousness as CanL progresses, the immune mechanisms underlying transition between disease stages and terminal decline remain ill-defined. To address this knowledge gap, we generated a comprehensive atlas of peripheral immune cells from control dogs and naturally-infected dogs using single-cell RNA sequencing. The LeishDog Atlas captures the cellular and transcriptional complexity underlying CanL, tracing shifts in immune composition and gene expression across progressive, well-defined disease stages. Notably, we identified distinct myeloid, CD4+ and CD8+ T cell phenotypes and transcriptional states associated with disease progression. This resource provides a valuable framework for understanding systemic immune dysregulation in CanL, including determinants of T cell exhaustion and myeloid cell activation, and establishes a foundation for comparative, translational, and mechanistic studies of CanL immunopathology and transmissibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8412687f28e64e4d6b2558529c494e2ee4440eca" target='_blank'>
              A single-cell transcriptomic atlas of peripheral blood immune cells spanning progressive canine leishmaniosis
              </a>
            </td>
          <td>
            Danielle P. Uhl, Daniel J. Holbrook, Max C Waugh, S. Dey, Najmeeyah Brown, Karen I. Cyndari, Jacob J. Oleson, Paul M. Kaye, Christine A. Petersen
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genomic alterations are a hallmark of cancer, and extrachromosomal DNA (ecDNA) has emerged as a key source of oncogene selection, tumor growth, and drug resistance. The intratumor heterogeneity and clonal selection of ecDNA is, however, poorly understood. In this study, we pursue a computational approach that leverages allelic imbalance and outlier expression from standard single-cell RNA sequencing (scRNA-seq) to deconvolve the tumor heterogeneity of ecDNA at the single-cell level (ecSingle). Using this approach, we identify oncogene-carrying ecDNAs in tumor samples at the single-cell level, which we validate using genome sequencing. Moreover, we show the superiority of using single-molecule long-read sequencing in resolving ecDNA. ecDNAs displayed extensive intratumor heterogeneity, including subclonal oncogene-carrying ecDNA in primary tumor cells that segregate with distinct transcriptional cell states. Importantly, we show that a rare ecDNA+ clone in the primary tumor can expand to form dominant clones in relapse tumors. Our study introduces a novel approach to studying ecDNA at the single-cell level, enabling both clonal evolution and transcription cell state analysis. We apply this approach to cancer samples to gain deeper insights into the role of ecDNA in intratumor heterogeneity and cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df14db8ad5c3596e9b413772fed54a722b4bb2b" target='_blank'>
              Resolving clonal evolution and selection of extrachromosomal DNA at single-cell resolution
              </a>
            </td>
          <td>
            J. Hendriksen, A. Locallo, B. Schlotmann, Francisco Germán Rodríguez González, Jane Skjøth-Rasmussen, C. Yde, D. Nørøxe, H. S. Poulsen, U. Lassen, J. Weischenfeldt
          </td>
          <td>2026-01-29</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="While cancer immunotherapies have primarily focused on activation of cytotoxic CD8 cells, CD4 T cell activity is also associated with survival and immunotherapeutic response in numerous cancers. We applied integrated single-cell RNA sequencing and multiplexed protein epitope profiling to breast cancer samples to resolve the complexity of immune cell states within the tumor microenvironment. This approach enhanced phenotypic resolution, identifying three distinct states within the CD4 T follicular helper-like (Tfh) cell cluster. A CXCR4high progenitor state gave rise to two differentiated states: an IGFL2high subset resembling conventional Tfh cells and localised to B cell-rich lymphoid aggregates, and a CD103+ subset, exhibiting features of tissue residency, exhaustion, and cytotoxicity, which co-localised with tumor foci. CD103+ Tfh-like cells were found to interact with CXCL10+ macrophages through production of CCL chemokines and CSF1. A higher CD103+ Tfh to IGFL2high Tfh ratio, together with the selective clonal expansion of the CD103+ subset, was strongly associated with improved tumour immunity and superior responses to anti-PD-1 checkpoint blockade, surpassing the predictive value of exhausted CD8 T cells. These findings integrate Tfh and CD4 with cytotoxic potential in breast cancer, offering new insight into anti-tumor immunity and response to checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/570ecff53156bbe18de4ac45af83e6cec5c103da" target='_blank'>
              Proteotranscriptomic Dissection of Breast Cancer T Cell States Identifies CD103+ Tfh-derived Cytotoxic Cells Linked to Immunotherapy Response
              </a>
            </td>
          <td>
            Alexander Swarbrick, G. Al-Eryani, Sophie van der Leij, Etienne Masle-Farquhar, Chia-Ling Chan, K. Harvey, Sunny Z. Wu, D. Roden, Taopeng Wang, John Reeves, Bertrand Yeung, Christopher Goodnow, Cindy Ma, C. Perou, N. Hacohen, Aziz Al’Khafaji, Mats Nilsson, J. Lundeberg, Marcel Batten, S. Junankar
          </td>
          <td>2026-01-07</td>
          <td>Research Square</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0d75b8ce4639debe11371b460bb9a8a9f43f83" target='_blank'>
              Deciphering the tumor microenvironment and role of immunotherapy in Diffuse Midline Glioma: a scoping review.
              </a>
            </td>
          <td>
            Christian K. Ramsoomair, Felipe Sarmiento, Deryn Ramsoomair, Manav Daftari, Jiasen He, Benjamin Glazer, M. Monje, Danny Reinberg, Ashish H Shah
          </td>
          <td>2026-02-02</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ovarian cancer(OC) poses a significant clinical challenge due to its frequent peritoneal dissemination and development of chemotherapy resistance, contributing to poor patient outcomes. Chronic inflammation is a pivotal driver of disease initiation and progression. While traditional population-level studies fail to capture cellular heterogeneity, recent advances in single-cell technologies including scRNA-seq, spatial transcriptomics, and multi-omics, have enabled high-resolution analysis of inflammatory regulation within the ovarian cancer microenvironment(OCME). This review synthesizes how single-cell approaches have elucidated inflammation-driven remodeling of OCME components, immune cell signaling pathways, and dynamic inflammatory-immune interactions. Focusing on the “dual microenvironments” of ovarian cancer, we discuss mechanisms of immune suppression, stromal reprogramming, and therapy resistance, along with current progress and challenges in translating these insights into precise diagnostic, targeted therapeutic, and immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b692e5a25b9e7346ecfe0d5da66ab9df1f209d5e" target='_blank'>
              Decoding inflammatory regulation in ovarian cancer at single-cell resolution
              </a>
            </td>
          <td>
            Jianghao Yu, Tingting Zhou, Shuangyu Chen, Jialin Zhu, Lu Xu, Yibo He, Wei Chen
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47f826d643f2d1e8e34a85ebebb36d9ff4d366b1" target='_blank'>
              Temporal and spatial atlas of eosinophil specialization across tissues
              </a>
            </td>
          <td>
            Yanan Hu, Lianfu Wu, Sihao Qu, Wan Ting Kong, Xiaotong Yu, Jiaqian Xu, Ziyi Xu, Jialing He, Chong Wang, Zhaoyuan Liu, Liming Lu, Lai Guan Ng, Camille Blériot, Bing Su, F. Ginhoux, Ziyi Li, S. Chakarov
          </td>
          <td>2026-01-09</td>
          <td>Nature Immunology</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Metastasis is a major threat in breast cancer (BRCA), often involving lymph node (LN) dissemination. However, the cellular composition and signaling networks within the metastatic microenvironment remain incompletely characterized, resulting in limited understanding of the molecular mechanisms driving LN metastasis. In this study, we constructed a comprehensive single-cell atlas of metastatic niches using single-cell RNA-sequencing (scRNA-seq) data from 78 primary breast tumors and their paired LN metastatic samples. Among the epithelial cell subpopulations, we identified distinct clusters representing early-disseminated cancer cells (EDCs). Notably, we observed profound metabolic reprogramming and immune modulation during the malignant transformation of epithelial cells, which may contribute to the invasive and metastatic phenotype of EDCs. Furthermore, the communication network within the metastatic microenvironment has also been delineated systematically, where crosstalk among lymphocyte, macrophage and epithelial cells drove immunosuppressive features of LN metastasis. To explore potential therapeutic interventions, we conducted computational drug repositioning and identified four tyrosine kinase receptor inhibitors that may target key interactions within this crosstalk. These results were validated using spatial transcriptomics (ST) data of four metastatic LN tissue sections from integrated scRNA-seq cohort. Collectively, this study discussed the cellular architecture and regulatory interplay underlying LN metastasis in BRCA, offering novel insights that may inform targeted therapeutic strategies for patients with metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c96def9e5d2ad3a5b2337af0512537501ee5cb" target='_blank'>
              Deciphering the cellular and metabolic landscape of lymph node metastasis in breast cancer using single-cell and spatial multi-omics.
              </a>
            </td>
          <td>
            Rui Zhu, Guijie Jiang, Jie Shen, Chengxuan Gong, Hongyu Shan, Tingming Liang, Li Guo
          </td>
          <td>2026-01-01</td>
          <td>The American journal of pathology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The regulation of T cell exhaustion within the tumor microenvironment plays a pivotal role in shaping the immune response to cancer and determining the efficacy of immunotherapy. However, the molecular factors governing this process in colon cancer remain poorly understood. This study investigates the expression characteristics and functional significance of the transmembrane protein CD82 in the colon cancer immune microenvironment, with emphasis on its regulatory role in CD8+ T cell exhaustion and clinical outcomes. Publicly available transcriptomic datasets were integrated with multiplex immunohistochemistry on colon cancer tissue microarrays to characterize the cell–type–specific distribution of CD82 and its associations with key markers of T cell dysfunction. CD82 expression was markedly increased in tumor-infiltrating immune and epithelial cells compared with normal tissues, particularly within exhausted CD8+ T cells. Elevated CD82 levels showed strong positive correlations with canonical exhaustion markers such as programmed cell death protein 1 and T cell immunoglobulin and mucin domain-containing protein 3. Multiplex immunohistochemical analysis further revealed that enrichment of CD82-positive epithelial regions and expansion of the CD82+TIM-3+PD-1+CD8+ T cell subset were associated with poor prognosis and were confirmed by multivariate Cox regression as independent risk factors for unfavorable survival. In patients who failed to achieve a complete pathological response following immunotherapy, exhausted CD8+ T cells exhibited significantly higher CD82 expression. Single-cell regulatory network analysis identified BATF and BHLHE40 as potential transcriptional regulators of CD82. Collectively, these findings demonstrate that CD82 promotes CD8+ T cell exhaustion, contributing to tumor progression and immunotherapy resistance in colon cancer. This study provides novel insight into the molecular mechanisms underlying immune dysfunction and offers a potential therapeutic target for reversing immunosuppression and improving immunotherapy efficacy in colon malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254f481971ba1782bc6c638ef396c6a18b14c937" target='_blank'>
              CD82-associated exhausted CD8+ T cells define prognosis and immunotherapy resistance in colon cancer
              </a>
            </td>
          <td>
            Jingfeng Zhang, Hengwei Cui, Shaoxian Wu, Hongwei Shi, Haitao Wang, Jingting Jiang
          </td>
          <td>2026-01-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c768fd92ec0a5075aecde92ab99ab504cc56d9" target='_blank'>
              STEAP4+ neutrophils: a promising circulating biomarker for lung metastasis
              </a>
            </td>
          <td>
            Chen Ai, Huijuan Wu, Huihui Yang, Jing Wang, Chang Liu, Melissa S F Ng, Ziheng Zhou, Fenghui Zhuang, Qinhang Gao, Lingqian Wang, Changming Shi, Linna Zhu, Z. Bu, Chang Chen, Lai Guan Ng, Zemin Zhang
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Giant cell tumor of bone (GCTB) presents considerable complexity in tumor microenvironment (TME) because of its intricate intercellular heterogeneity and the presence of an immunosuppressive milieu. In order to understand the complex gene expression patterns and cell interactions in GCTB, we carried out a thorough investigation using single‐cell RNA sequencing (scRNA‐seq). Methods We examined scRNA‐seq data from 7091 cells that were collected after surgical removal of GCTB. Following the initial quality control process, 10 separate groups of cells were distinguished, which consisted of dendritic cells, endothelial cells, macrophage cells, mast cells, monocyte cells, neutrophil cells, tumor cells, osteoclast cells, pericyte cells, and T cells. Additional analysis uncovered distinct categories within tumor‐associated macrophages (TAMs), CD8+ T cells, and CD4+ T cells. The differentiation mechanisms of TAMs, CD8+ T cells, and CD4+ T cells were explored using pseudo‐time trajectory analysis. The CellPhoneDB study revealed the interactions between various cell types within the TME of GCTB. Results TAMs have been identified as the main infiltrating cells in GCTB. These TAMs exhibit several subtypes that are characterized by specific marker genes and functional states. The identification of several subgroups within CD8+ T cells that are involved in regulating immunological checkpoints underscores the difficulties encountered when attempting to employ immune checkpoint blockade therapy for GCTB. T cell exhaustion poses a major barrier to the efficacy of antitumor immune responses. Research suggests a strong correlation between TAMs and exhausted T cells (Texs) in the TME. The high number of regulatory T cells (Tregs) highlights the immunosuppressive nature of the immunological environment in GCTB. Significant interactions have been observed between TAMs and tumor cells, highlighting their crucial involvement in immune evasion strategies. Conclusion This scRNA‐seq study provides a general overview of the different cellular compositions and immune interactions within GCTB. The identified subtypes and communication networks provide valuable information about the immunosuppressive environment of GCTB, laying the foundation for prospective therapeutic approaches targeting specific cell types or interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e77dc33f2650929ccb5e19eeed4e2c7dc7b4c11a" target='_blank'>
              Single‐Cell Sequencing Reveals the Immunosuppressive Trajectory in the Tumor Microenvironment of Human Giant Cell Tumor of Bone
              </a>
            </td>
          <td>
            Yiming Liu, Wei Luo, Yude Xu, Xiguan Yao, Libing Dai, Qiao Feng, Peigeng Wang, Weichao Yang, Yi Feng, Haixiong Miao, Suixiang Huang, Dongping Ye
          </td>
          <td>2026-01-01</td>
          <td>BioMed Research International</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/126eb3bb80a65b589c6798760093749f010ab57b" target='_blank'>
              Robust characterization and interpretation of rare pathogenic cell populations from spatial omics using GARDEN.
              </a>
            </td>
          <td>
            Xinming Zhang, Zhuohang Yu, Gaoyang Hao, Qi Yao, Yanmei Hu, Fuzhou Wang, Xingjian Chen, Linjing Liu, Ka-chun Wong, Xiangtao Li
          </td>
          <td>2026-01-17</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Urothelial carcinoma (UC), encompassing both bladder cancer and upper tract UC, is a biologically heterogeneous malignancy that has long complicated patient stratification and therapeutic prediction. Recent genomic and transcriptomic analyses, however, have delineated reproducible molecular subtypes across non‐muscle‐invasive and muscle‐invasive disease. These classification frameworks have refined biological understanding and improved both risk stratification and treatment selection. Emerging evidence also indicates that genomic alterations, such as FGFR3 mutations, may confer distinct immunologic phenotypes within specific transcriptomic contexts, thereby challenging prior assumptions. The advent of immune checkpoint inhibitors has transformed UC management, yet variable response rates underscore the complexity of the tumor microenvironment (TME). Single‐cell and spatial analyses have identified immune‐inflamed, ‐desert, and ‐excluded TME subtypes, with cancer‐associated fibroblasts playing a key role in immune exclusion and therapeutic resistance. Liquid biopsy, particularly circulating tumor DNA, has demonstrated utility for real‐time disease monitoring, minimal residual disease detection, and treatment stratification, as highlighted in trials such as IMvigor010 and IMvigor011. Urinary extracellular vesicles also represent promising non‐invasive biomarkers, though further standardization is required. In addition, novel therapeutic strategies—including antibody‐drug conjugates, gene therapies, and sustained‐release delivery systems—are broadening treatment options across disease stages. Collectively, these translational advances mark a paradigm shift toward precision immuno‐oncology in UC. Nonetheless, successful clinical implementation will require prospective validation, optimized analytical platforms, and sustained physician‐scientist engagement to translate these discoveries into improved patient outcomes. This review provides a comprehensive overview of recent progress in UC translational research, with a focus on genomic and transcriptomic insights, the evolving role of immunotherapy, advances in liquid biopsy, and the development of novel therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e44cd39c3a6847a9cce1aefa2b7bb98a79ab335f" target='_blank'>
              Translational Advances in Urothelial Carcinoma: From Bench to Bedside
              </a>
            </td>
          <td>
            T. Iwasawa, T. Tsujino, S. Tsukahara, Hiroshi Fukushima, Daisuke Ito, E. Tomiyama, Fumihiko Urabe
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Urology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract The process of migration and colonization is important in evolution; for example, modern humans experienced multiple waves of migrations out of Africa. However, no data cover the spatio-temporal patterns sufficiently to be truly informative. Metastatic cancer provides a unique in vivo model to study these processes through rapid somatic evolution. Here, we apply the high-resolution sampling technique (Dense 3D Crypt-scale Sampling) to analyze hundreds of spatially mapped micro-samples from the primary colorectal cancer and liver metastases in two representative cases. This would be analogous to recording the “out-of-Africa” events in two repeats. Our results support that liver metastases arise from polyphyletic and polyclonal seeding events where multiple, genetically distinct clones colonize a new site together. Following colonization, these multi-clonal populations can evolve into distinct spatial architectures: segregated territories formed by cells with low motility, or highly intermixed patterns driven by high motility. The colonization (or seeding) process begins within the first third of the primary tumor's progression, creating a large number of widespread but clinically undetectable micrometastatic colonies. These findings support a model where metastatic competence is not an intrinsic trait of a single “winner” clone but an emergent property of multiple concurrent clones. Collectively, our work supports metastasis as a multi-stage process initiated early in tumor development, characterized by continuous polyclonal dissemination and the formation of spatially distinct clonal architectures. This general pattern may echo the ecology of migration and colonization in organismal evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9d8108fc71756559fb44982b7725fb8d93fb854" target='_blank'>
              Full spatio-temporal analyses of migration and colonization in evolution—dense 3D mapping of cancer metastases provides new insights
              </a>
            </td>
          <td>
            Qihang Chen, Senmao Li, Xianrui Wu, Qing Xu, Ranran Zhu, Yongsen Ruan, Ao Lan, Zihan Liu, Jiarui Weng, Yanjiang Zhao, Xiying Xu, Xinyue Qi, Jinhong Lai, Leyi Xiao, Ping Lan, Chung-I Wu, Bingjie Chen
          </td>
          <td>2026-01-12</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e961f589165d98aa8aff64f0539527c2012d419" target='_blank'>
              Pooled single-cell screen in colorectal cancer defines transcriptional modules linked to oncogenes.
              </a>
            </td>
          <td>
            Viola Hollek, Francisca Böhning, Catalina Florez Vargas, A. Sieber, M. Morkel, Nils Blüthgen
          </td>
          <td>2026-01-19</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Innate immune response modulating impurities (IIRMI) with adjuvant potential have emerged as important factors in the immunogenicity risk assessment of protein, peptide, and oligonucleotide therapeutics, particularly for follow-on products where minimal or no clinical studies are available. To assess the impact of differences in impurities on specific cell types, we developed a new IIRMI assay termed multiplexed immunophenotyping for innate activation assessment (MIIAA) that employs spectral flow cytometry to capture single-cell responses to drug products and potential impurities. This technique introduces a new live fluorescent cell barcoding platform that enables sample multiplexing for homogeneous staining with a single fluorescent antibody cocktail composed of identity and activation markers that are acquired simultaneously with a five laser Cytek Aurora. Samples are digitally reassigned to their original testing conditions by positive and negative gating of barcode dyes. Cellular subsets are identified by dimensionality reduction of surface markers with UMAP then gated using cell-specific markers. Here we use trace levels of TLR3, 7/8 and 9 agonists (Poly(I:C), R848, and CpG ODN) to characterize specific responses in B cells, monocytes, cDC and pDC. Importantly, MIIAA captures single-cell responses to nucleic acid impurities in the presence of therapeutic oligonucleotides or monoclonal antibodies with high sensitivity. Taken together, MIIAA offers a powerful immunophenotyping tool to characterize single-cell responses to drug products and potential immunomodulatory impurities that may find utility in drug pipelines to characterize the impact of therapeutics on specific immune cells and to interrogate immunogenic or immunomodulatory risk in comparisons between reference and follow-on products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66966d24c4df4e2f25e177ce784351c7794e0868" target='_blank'>
              Multiplexed Immunophenotyping for Innate Activation Assessment Detects Single-Cell Responses to Immunomodulatory Nucleic Acid Impurities in Therapeutics
              </a>
            </td>
          <td>
            Joseph A. Balsamo, Mirian Mendoza, Logan Kelly-Baker, Seth G. Thacker, Daniela Verthelyi
          </td>
          <td>2026-01-01</td>
          <td>The AAPS Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Lung cancer remains the leading cause of cancer−related mortality. Single−cell RNA sequencing (scRNA−seq) enables high−resolution mapping of tumor microenvironment, but how malignant cell states connect to patient outcomes and therapeutic vulnerabilities is not fully understood. Methods We analyzed scRNA−seq profiles from 13 primary lung tumors and matched normal lung tissues (48,149 cells). Malignant versus non−malignant epithelial cells were defined by inferCNV, co−expression programs resolved with Hotspot, transcription factor regulons inferred by pySCENIC, and differentiation potential estimated by CytoTRACE and diffusion pseudotime. Key malignant programs were cross−checked in an independent LUAD scRNA−seq cohort (GSE131907). Program signatures were projected into TCGA−LUAD to build an eight−gene LASSO−Cox risk model, which was further validated in two external NSCLC/LUAD cohorts (GSE30219 and GSE31210). Immune contexture was inferred by CIBERSORT and TIDE, drug sensitivities predicted with pRRophetic, and the function of FAM189A2 tested by knockdown and overexpression in A549 and NCI−H23 cells. Results We delineated 12 major cell populations and six malignant epithelial subclusters, including an early GPRC5A+ subpopulation with high developmental potential. Malignant programs corresponding to cell−cycle and inflammatory states (Modules 8, 14 and 16) were associated with worse overall survival, whereas Module 15, preferentially active in the GPRC5A+ state, was linked to better outcomes. An eight−gene signature (LAMA5, ACTB, B4GALT1, KLF5, KRT18, FAM189A2, SLC34A2, S100A11) robustly stratified patients into high− and low−risk groups across TCGA−LUAD and two validation cohorts. High−risk tumors displayed an immune−enriched yet matrix−restrained microenvironment with higher CD8+ T cells, whereas low−risk tumors showed greater predicted sensitivity to Aurora kinase, IGF−1R, mTOR and TGF−β inhibition. FAM189A2 was down−regulated in tumors, higher expression predicted better survival, and bidirectional perturbation in A549 and H23 cells demonstrated that loss of FAM189A2 enhanced, while overexpression attenuated, migration and invasion in vitro. Conclusions Integrating single−cell and bulk transcriptomes links malignant epithelial state programs to prognosis, yields a practical eight−gene risk model validated in multiple LUAD cohorts, and nominates FAM189A2 as a putative tumor suppressor and potential biomarker in lung cancer. These findings suggest testable strategies for risk stratification and therapy selection that warrant prospective evaluation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9149373826a7f4a30a6edb092024413c1a5945d" target='_blank'>
              Integration of single cell and bulk transcriptomic analyses identifies FAM189A2 as a key prognostic gene in lung cancer
              </a>
            </td>
          <td>
            Peng Guo, Mengqi Xiang, Jing Chen, Huachuan Zhang
          </td>
          <td>2026-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Objective Metabolic dysfunction-associated steatotic liver disease-related hepatocellular carcinoma (MASLD-HCC) is an emerging malignancy with limited therapeutic options. The identity and function of cancer stem cells (CSCs) in MASLD-HCC remain poorly understood. In this study, we characterised CSCs in MASLD-HCC and investigated their contribution to MASLD-HCC tumourigenesis and therapy response. Design We performed expression profiling in human MASLD-HCC samples (n=29 pairs of tumour and adjacent normal tissues). Advanced in vivo genetic lineage tracing coupled with single-cell RNA sequencing was used to characterise CD133+ CSCs in preclinical models. To establish causality, we developed a hepatocyte-specific CD133-overexpressing mouse model of MASLD-HCC. We identified CD133 protein interactors by mass spectrometry. A novel strategy combining CD133-targeted small interfering RNA (siRNA) nanoparticles with first-line therapy was assessed in clinically relevant MASLD-HCC models. Results CD133+ CSCs were significantly enriched in human MASLD-HCC tumours and positively correlated with established markers of malignancy. In vivo genetic lineage tracing in mice revealed that CD133+ cells exhibit hallmark CSC properties, including self-renewal, tumour-initiating capacity and multipotent differentiation, as compared with CD133− counterparts. Hepatocyte-specific CD133 overexpression in mice accelerated MASLD-HCC tumourigenesis. Mechanistically, CD133 interacts with myosin heavy chain 9 (MYH9) to stabilise active β-catenin, thereby propagating Wnt/β-catenin signalling that drives CSC phenotypes and tumourigenic potential. Therapeutically, genetic ablation of CD133+ cells or systemic delivery of CD133-siRNA nanoparticles potently sensitised MASLD-HCC to sorafenib and lenvatinib, significantly improving outcomes in MASLD-HCC. Conclusion This study established CD133+ CSCs as critical mediators through the CD133-MYH9/β-catenin axis in MASLD-HCC. Targeting CD133 enhances multikinase inhibitor efficacy, offering a promising therapeutic strategy for MASLD-HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36b58e21741e760fa3c8f05638f038862353300e" target='_blank'>
              Targeting cancer stem cells enhances multikinase inhibitor therapy in metabolic dysfunction-associated steatotic liver disease-related hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yasi Pan, Xiang Zhang, C. Wong, Huarong Chen, Stephianie Ma, T. K. Lee, Kai Yuan, Cong Liang, Xingyu Zhou, H. C. Lau, Pingmei Huang, Danyu Chen, Lina Wang, Yanqiang Ding, Qinyao Wei, Alvin Ho Kwan Cheung, K. To, Jun Yu
          </td>
          <td>2025-12-24</td>
          <td>Gut</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Osteosarcoma (OS) is the most common primary bone malignancy affecting children and adolescents, for which survival has not improved in more than four decades. The lack of accurate OS preclinical models hinders the understanding of tumor heterogeneity and its interaction with the surrounding extracellular matrix (ECM), limiting the discovery of predictive biomarkers and the development of effective therapies. Four 2D preclinical models from OS patients were established and characterized for their functional differences in comparison to OS cell lines for their growth, cellular phenotypic attributes, osteogenic differentiation capabilities and metabolic responses to growth factors and BMP-2. Molecular and cellular profiling revealed intra-tumoral heterogeneities that were very distinct from the endorsed OS cell lines. The OS patient-derived (PD) cells also displayed differential sensitivity to Doxorubicin and Cisplatin and resistance against Methotrexate. Subsequently, the 3D PDTs (Patient- derived tumoroids) models were developed by self-aggregating spheroids with and without Matrigel® ECM matrix. These PDTs models were screened for selective ECM and bone-specific gene markers, revealing dynamic differences between 2D and 3D models with and without ECM, with heightened dysregulation observed in 3D systems. The drug response variances observed among 2D OS cells and 3D tumoroids model within Matrigel® highlights the need for optimized platforms for in vitro personalized drug screening. Thus, our findings support the screening of preclinical PD OS models for phenotypic profiling and elucidating ECM contributions to drug responses and pathophysiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895f2c94cf5a05025fd5ceb276a27c6fd2f5f6fa" target='_blank'>
              Patient-derived osteosarcoma tumoroid model reveals functional phenotypic diversity with implications for drug responses
              </a>
            </td>
          <td>
            Sugandha Bhatia, Briony Claxton, Erik W Thompson, Martin Lowe, Michael Wagels, Wayne Nicholls, Philip D Rowell, D. Hutmacher, J. McGovern
          </td>
          <td>2026-01-22</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) presents a unique immunological paradox: abundant CD8+ tumor-infiltrating lymphocytes (TILs) correlate with poor prognosis. To clarify their functional status and therapeutic potential, we performed single-cell transcriptomic profiling of TILs from 15 patients with ccRCC and functionally validated dominant T cell receptor (TCR) clonotypes using autologous tumor-derived organoids. Single-cell RNA sequencing revealed dynamic shifts in T cell composition, with effector and progenitor-exhausted CD8+ T cells declining and terminally exhausted CD8+ and regulatory CD4+ T cells enriched in advanced tumors. Despite this exhausted phenotype, in an exploratory analysis with five patients, approximately half of the top 20 TCR clonotypes retained anti-tumor reactivity when re-expressed in non-exhausted T cells, as evidenced by TCR-T cell-mediated cytotoxicity and IFN-γ production against autologous organoids. Transcriptomic signatures enabled the development of a penalized logistic regression classifier that distinguished tumor-reactive from bystander T cells with high accuracy, with AUCs of 0.903 (training) and 0.913 (test). Cross-cancer comparison with pancreatic ductal adenocarcinoma (PDAC) datasets revealed limited generalizability, highlighting the need for cancer type-specific models. Notably, ccRCC-specific TILs exhibited mature, functionally differentiated profiles with limited proliferation, consistent with chronic antigen exposure, whereas PDAC-reactive TILs showed highly proliferative and activated phenotypes indicative of ongoing clonal expansion. Collectively, these findings suggest key features of the immune landscape in ccRCC and provide a preliminary, proof-of-concept transcriptomic framework for prioritizing candidate tumor-reactive TCRs. These insights suggest the feasibility of identifying candidate TCRs for future development of TCR-based adoptive T cell therapies in ccRCC and emphasize the importance of integrating single-cell profiling with functional analyses to refine immunotherapeutic strategies. Given the limited sample size, our results should be considered exploratory and hypothesis-generating, and future studies will be required to validate these findings in larger, independent ccRCC cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e62c016264580977157f1e59b0fc3cfcc38dd55" target='_blank'>
              Tumor-reactive TCRs within exhausted TILs reveal cancer type-specific immune landscapes in renal cell carcinoma
              </a>
            </td>
          <td>
            Mitsuru Komahashi, S. Horaguchi, Kayoko Tsuji, Daisuke Hoshino, Takeshi Kishida, Kimitsugu Usui, N. Nakaigawa, Shinya Sato, Hiroshi Hamana, Hiroyuki Kishi, Feifei Wei, Yasunobu Mano, T. Kouro, Shuichiro Uehara, T. Sasada
          </td>
          <td>2026-01-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2092223941ef46884e67c45fd51c2462a350c8db" target='_blank'>
              Unambiguous signatures of malignancies extracted from images of growing cells
              </a>
            </td>
          <td>
            G. Kalweit, M. Kalweit, W. Checinska, M. Saric, R. Berger, E. Bodurova-Spassova, J. Rawluk, N. Talvard-Balland, A. Klett, M. Follo, S. Kreutmair, J. Duque-Afonso, M. LuÌbbert, R. Zeiser, J. Frank, R. Mertelsmann
          </td>
          <td>2026-01-13</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The tumor microenvironment in high-grade serous ovarian carcinoma (HGSC) is a complex network of malignant-host cell interactions, yet its orchestration remains poorly understood. We present a single-cell spatial atlas of metastatic HGSC from 280 patients, integrating high-dimensional imaging and molecular profiling. Analyzing 929 single-cell maps, we identify spatial domains with diverse cell compositions and show that immune cell co-infiltration at the tumor-stroma interface impacts clinical outcomes. Using CEFIIRA, we find that tumor cell MHCII expression is a key predictor of prolonged survival. Validation with deconvoluted, single-cell, and two distinct spatial transcriptomic datasets, along with immunopeptidomic analysis, confirms that MHCII expression correlates with immune activation, antigen presentation, and TCR clonality. Using a patient-derived immuno-oncology platform, we demonstrate that tumor MHCII expression associates with increased CD8+ T cell cytotoxicity after PD-1 blockade, while blocking MHCII inhibits this activation. Our atlas offers new insights into immune activation, potentially improving patient stratification in HGSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fe767539847acd8124897cd09e4464f04ce8afe" target='_blank'>
              Single-cell spatial atlas of high-grade serous ovarian cancer uncovers MHC class II as a key predictor of spatial tumor ecosystems and clinical outcomes.
              </a>
            </td>
          <td>
            Fernando Pérez-Villatoro, Aleksandra Shabanova, Lilian van Wagensveld, Ada Junquera, Iga Niemiec, María Hincapié-Otero, Ziqi Kang, Matias M. Falco, Kürşat Birgin, Sarah Wolf, Ella Anttila, Gayani Anandagoda, J. Casado, Eric Marcus, Duco H K Gaillard, Essi Kahelin, Foteini Chamchougia, M. Salko, Saundarya Shah, Salvatore Russo, Jacopo Chiaro, M. Grönholm, Joseph Ndika, Otto K. Kari, iCAN iCAN, G. Sonke, K. K. van de Vijver, Rutgerus Fpm Kruitwagen, M. A. van der Aa, A. Virtanen, Vincenzo Cerullo, Anna Vähärautio, P. Sorger, H. Horlings, Anniina Farkkila
          </td>
          <td>2026-02-09</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Myxofibrosarcoma (MFS) is a rare and aggressive soft tissue sarcoma characterized by high genomic instability, resulting in high local recurrence rates and limited effective therapeutic options in advanced stages. Recent progress in cancer immunology research has encouraged investigation into the Tumor Microenvironment (TME) of sarcomas, including MFS, to identify immune-related biomarkers of prognostic and therapeutic relevance. Although data remain limited in MFS, existing evidence suggests a heterogeneous immune landscape, including: i) variable expression of immune checkpoint molecules such as Programmed Cell Death Protein 1 (PD-1) and Programmed Death-Ligand 1 (PD-L1), ii) presence of tumor-infiltrating lymphocytes, iii) alterations in antigen presentation pathways, and iv) a pronounced angiogenic signature. These findings underscore the potential role of immune biomarkers for patients’ clinical stratification and the consequent possibility of developing new immunotherapeutic strategies. This review will focus on the cellular and molecular architecture of immune infiltration, vascular remodeling, and lymphoid neogenesis, assessing their prognostic and predictive value as potential biomarkers. Finally, we will present ongoing clinical trials aimed at modulating the immune-vascular niche to inform innovative therapeutic strategies for this challenging sarcoma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f542fc2b8b6a3c1d5def7187e73b0d930cab87" target='_blank'>
              Unveiling the myxofibrosarcoma tumor microenvironment: implications for immunotherapy
              </a>
            </td>
          <td>
            Cecilia Profumo, Massimiliano Grassi, Valentina Rigo, Matteo Mascherini, Paola Monti, Giorgia Arcovito, Giorgia Anselmi, Laura Damele, G. Timon, Danila Comandini, Michela Croce
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) enables genome-wide gene expression profiling while preserving tissue architecture, bridging the gap between bulk, single-cell, and histological analyses. Originating in 2016 and rapidly evolving since, ST has transformed infectious disease research by mapping host–pathogen interactions directly within intact tissues. Current platforms fall into two categories: sequencing-based methods (Visium, GeoMx, Stereo-seq) offering whole-transcriptome coverage at modest resolution and imaging-based platforms (Xenium, CosMx, MERFISH) providing single-cell or subcellular detail with targeted gene panels. These technologies reveal spatially organized immune responses, local tissue remodeling, and pathogen niches across viruses, bacteria, and parasites. In viral infection, ST uncovered heterogeneity in COVID-19 lung microenvironments, spatial immune activation in lymphoid tissues, and variant-specific inflammatory patterns. In bacterial disease, ST delineated granuloma architecture in tuberculosis and mapped vaccine-induced lung responses in Shigella studies. Parasitic infection studies identified localized inflammatory hotspots and microenvironmental control of T-cell differentiation in malaria. Despite powerful insights, ST faces constraints including RNA quality limitations, tradeoffs between resolution and transcript breadth, high cost, and analytical complexity. Nonetheless, ST increasingly informs vaccine design by identifying tissue-specific immune programs and protective microenvironments and is poised to become a standard tool for infectious disease biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd879a8a18bd7edf093e16c2214bb3570575ec48" target='_blank'>
              Advances in Spatial Transcriptomics for Infectious Disease Research: Insight for Vaccine Development
              </a>
            </td>
          <td>
            Taehwan Oh
          </td>
          <td>2026-02-07</td>
          <td>Vaccines</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor heterogeneity drives chemotherapy resistance in cancer, necessitating the development of strategies to target diverse cellular subpopulations. Single-cell-derived tumor organoids (STOs), derived from individual tumor stem cells (TSCs), retain well-defined genomic and phenotypic attributes, making them instrumental in elucidating unique diversification. Here, we developed a microfluidic approach that utilizes STO arrays for personalized drug screening, aimed at overcoming heterogeneity induced drug resistance in colorectal cancer. We utilized a microwell array format microfluidic chip to the generation of a high scale STO array. This array encompasses two key features: (1) STOs in individual microwells are derived from single TSCs, which allows for a high-resolution display of cellular heterogeneity; (2) it exhibits thousands of STOs at predefined locations on the same focal plane, thereby facilitating automated, high-content analyses using imaging data. Utilizing the STO array, we assess the responses of various subpopulations of colorectal cancer to a conventional chemotherapeutic regimen, facilitating the phenotypic identification of resistant STOs. Through the analysis of RNA sequencing data from the resistant STOs, candidate drugs targeting resistance-associated pathways were identified for individual patients. Furthermore, the candidate drugs were validated through STO-array drug testing, demonstrating enhanced clearance effects when combined with the conventional regimen. Our study establishes a scalable, patient-specific drug screening approach that integrates phenotypic and molecular analyses, offering a transformative strategy to overcome heterogeneity-driven drug resistance in colorectal cancer. The microfluidic platform presented herein holds significant potential for advancing precision oncology and personalized therapeutic design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5a5ebfd822bf50840db029f6b0e013241dad68" target='_blank'>
              Single-Cell-Derived Tumor Organoid (STO) arrays on a microfluidic chip for personalized drug screening to address heterogeneity-induced drug resistance in colorectal cancer
              </a>
            </td>
          <td>
            Dongguo Lin, Yanzhang Luo, Jueming Chen, Ziqing Ma, H. Kang, Xiaogang Wang, Lihui Wang, Dayu Liu
          </td>
          <td>2025-12-01</td>
          <td>Microsystems & Nanoengineering</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal cancer (CRC) frequently develops aggressive metastatic disease, yet the cellular features that enable dissemination remain poorly defined. IKKα, a kinase traditionally linked to stress and inflammatory signaling, is increasingly recognized for broader functions in cancer. Here, we show that loss of IKKα unexpectedly promotes metastasis in CRC. Using patient-derived organoids, we find that genetic or pharmacological inhibition of IKKα stabilizes tight-junction components, leading to the emergence of compact epithelial clusters with a heightened ability to spread and colonize the liver. Single-cell transcriptomics reveals expansion of a CDH17⁺/CLDN2⁺ epithelial subpopulation that dominates metastatic lesions, a finding validated by tissue staining. Remarkably, disrupting CLDN2 completely eliminates the metastatic advantage caused by IKKα loss. These results identify a metastasis-competent epithelial state driven by tight-junction remodeling and uncover a vulnerable node that may be exploited therapeutically in aggressive colorectal cancer. The IKKα kinase was previously reported to promote metastasis. Here, the authors reveal that loss of IKKα function promotes colorectal cancer liver metastasis by expanding a CDH17⁺/CLDN2⁺ epithelial subpopulation characterized by tight junction stabilization and collective migration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214214b3f4bf431d9df6a0363a7ab05464474373" target='_blank'>
              Tight junction-high and CDH17-positive cell population is the source of colorectal cancer liver metastases
              </a>
            </td>
          <td>
            Daniel Álvarez-Villanueva, Mariano R Maqueda, Dina Harti, E. Cantón, Evelyn Andrades, Joan Bertran, M. Martínez-Iniesta, Laura Solé, Violeta García‐Hernández, Ángela Montoto, T. Lobo-Jarne, J. Alonso-Marañón, Patricia Herrero-Molinero, Monica Larrubia-Loring, Anna Tramuns, Kieran Wynne, Indrani Bera, D. Matallanas, Alberto Villanueva, A. Bigas, Mar Iglesias, Lluís Espinosa
          </td>
          <td>2026-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive subtype of breast cancer characterized by the absence of estrogen, progesterone, and HER2 receptor expression. This malignancy is often associated with a poor prognosis, early recurrence, and limited treatment options. The tumor microenvironment (TME) in TNBC plays a pivotal role in tumor progression, immune evasion, and therapeutic resistance. In recent years, an increasing body of evidence has highlighted the critical interactions between cancer cells and the components within the TME, including immune cells and soluble components. These interactions influence not only the biological behavior of the tumor but also its response to treatment. Exploring the complex interplay between tumor cells and immune components continues to inform the development of more effective therapeutic approaches. In this study, we provide a synopsis of advancements regarding the TME in TNBC. In light of different cellular compartments, we delineate multiscale interplays within the stroma–tumor symbiosis and highlight their antitumor functions and promising targeting strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11bfc269114236644c5ca2675c1d14983686fe40" target='_blank'>
              Prominent protumoral cellular compartments of the tumor microenvironment in triple-negative breast cancer
              </a>
            </td>
          <td>
            Shuai Sun, Pan Zhao, Changrong Wang, Junjun Du, Tingting Zhang, Xiangyun He, Zhibo Zuo, Nan Li, Rongjing Zhou
          </td>
          <td>2026-01-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c745807b38045a754b3f57a50a75c7b86c914ca" target='_blank'>
              A Scalable Framework for Pan-Cancer Tumor Evolution Analysis Enables Transfer of Progression Mechanisms Across Tumor Entities
              </a>
            </td>
          <td>
            Simon Pfahler, Andreas Lösch, Y. L. Hu, Rudolf Schill, Rainer Spang, T. Wettig
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebe9bf9c3ccb5a673d6338b3ea4c99abb09a8ff3" target='_blank'>
              A robust human airway organoid platform enables scalable expansion and trajectory mapping of pulmonary neuroendocrine cells
              </a>
            </td>
          <td>
            Noah Candeli, Lisanne den Hartigh, Nicholas Hou, Andrés Marco, José Antonio Sánchez-Villicaña, Andrea García-González, Shashank Gandhi, Francesca Sgualdino, Alyssa J. Miller, Jason R. Spence, Susana M. Chuva de Sousa Lopes, J. McFaline-Figueroa, Hans Clevers, Talya L. Dayton
          </td>
          <td>2026-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell and spatial transcriptomics technologies have transformed the landscape of cardiovascular research, leading to novel insights into cellular heterogeneity and tissue architecture in health and disease. These technologies enable researchers to deconvolute complex tissues and map gene expression patterns within their spatial contexts, providing critical information on the interplay between cell types and pathways affecting tissue regeneration or progression to fibrosis. This review presents an overview of the recently developed applications of single-cell and spatial transcriptomics methods and their impact on cardiovascular research. We discuss the principles underlying emerging solutions to process fixed and low-integrity RNA samples like formalin-fixed paraffin-embedded tissues. In addition, we highlight advances in high-resolution spatial transcriptomics assays, from imaging-based techniques like MERFISH (multiplexed error-robust fluorescence in situ hybridization) and Xenium to sequencing-based platforms like Visium HD, Stereo-seq, and Open-ST, each contributing unique strengths for tissue-level analysis. The integration of these technologies with machine learning and multiomics approaches further enhances the ability to uncover novel biology. These approaches have already led to the discovery of spatially resolved gene expression patterns in human atherosclerosis, hypertrophic cardiomyopathy, myocardial infarction, and myocarditis. These case studies showcase how these methods can be applied to decode the cellular and molecular dynamics of human disease processes, identify potential novel therapeutic targets, and enable predictive modeling of cellular perturbations and patient disease trajectories. We provide a comprehensive overview of the growing data repository landscape, the principles and applications of novel machine learning approaches, which are becoming more and more standard analytical tools. Integrating these areas underscores how recent advancements in single-cell and spatial transcriptomics are offering an increasingly detailed and comprehensive understanding of the cellular landscape of cardiovascular disease while also highlighting the challenges and future directions that will shape innovations in cardiovascular biology and medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31765e8355b6603e6f061451c773b6eba179cda2" target='_blank'>
              Advancing Cardiovascular Research With Single-Cell and Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Kai Li, Samaneh Samiei, Daryna Pikulska, Sebastian Foecking, C. Kuppe
          </td>
          <td>2026-01-02</td>
          <td>Circulation research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5108b05893ed4ed9a64a7f2a10e9459a930c1b89" target='_blank'>
              Analysis of the transcriptomic and metabolomic landscape of prostate cancer with different anatomical origins using snFLARE-seq and mxFRIZNGRND.
              </a>
            </td>
          <td>
            Dongyin He, Haoran Hu, Kai Xiao, Yuhang Zhang, Yongbing Cheng, Shaozhuo Jiao, Yimei Hao, Yuanyuan Cai, Ziqun Liu, Xinran Yan, Qinsheng Chen, Xiyan Mu, Qi Wang, Shan Peng, Guoqin Sang, Xiaoling Zhi, Yanxia Chang, Qing Ye, Yuyao Yang, Meixia Che, Shengsong Huang, Hongqian Guo, Luonan Chen, Huiru Tang, Xuefeng Qiu, Zhenfei Li
          </td>
          <td>2026-02-07</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background: Hepatocellular carcinoma (HCC) is characterized by marked intratumoral heterogeneity and poor clinical outcomes. Dysregulated ribosome biogenesis has emerged as a fundamental hallmark of tumor initiation and progression; however, the specific molecular drivers linking this machinery to HCC pathogenesis remain largely undefined. Methods: By integrating multi-omics data from the TCGA and ICGC cohorts, FBLL1 was identified as a key prognostic candidate gene. Its cellular and spatial distribution was analyzed using single-cell RNA sequencing and spatial transcriptomics. Its biological functions in vitro and in vivo were validated through functional experiments, including lentivirus-mediated ectopic expression and siRNA-mediated gene knockdown. Finally, its molecular mechanism was elucidated through transcriptomic analysis and Western blotting. Results: FBLL1 was significantly upregulated in HCC and correlated with poor patient survival. Spatial and single-cell analyses showed that FBLL1 expression was preferentially enriched in malignant hepatocytes within the tumor region. Functionally, knockdown FBLL1 could inhibit the proliferation and clonogenic capacity of HCC cells, while overexpression FBLL1 in non-tumorigenic hepatocytes could promote the tumorigenic phenotype in xenograft models. Transcriptomic analysis indicated that FBLL1 overexpression was associated with the synergistic upregulation of c-Myc and multiple EGFR ligands, as well as decreased expression of hepatocyte functional markers. Consistently, modulation of FBLL1 expression affected the activity of the EGFR–MAPK signaling pathway. Conclusions: Our study identifies FBLL1 as a previously unrecognized regulator associated with malignant state transition in HCC. Rather than acting as a direct regulator of core signaling components, FBLL1 is associated with ligand-dependent activation of the EGFR–MAPK pathway in conjunction with c-Myc upregulation. These findings indicate that FBLL1 represents a promising therapeutic target for disrupting oncogenic signaling programs in liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73452fadba8b76804fe5e945db231c49dcf6ea8b" target='_blank'>
              Integrated Multi-Omics and Spatial Transcriptomics Identify FBLL1 as a Malignant Transformation Driver in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Junye Xie, Shujun Guo, Yujie Xiao, Yibo Zhang, An Hong, Xiaojia Chen
          </td>
          <td>2026-01-27</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 The mechanisms by which a small subset of high-risk localized prostate cance becomes immunogenic is not well understood. The objective of this study is to identify the tumor intrinsic and extrinsic mechanisms associated with immunogenicity in this population with the goal of harnessing this knowledge to further immunotherapy in PCa. Methods We performed Visium and Xenium spatial transcriptomic profiling on radical prostatectomy specimens from untreated, high-risk, localized prostate cancer patients obtained from the UCLA biobank with varying level of intra-tumoral T-cell infiltration (TILs). Validation was performed using OPAL multiplex immune-fluorescence (mxIF) staining for CD4, CD8A, CD68, PIGR (marker of club cells), and PSA on 18 specimens from a separate cohort of untreated, intermediate to high-risk localized prostate cancer. Summary Spatial transcriptomic analysis identified cells with club-like features to be associated with global T cell infiltration. Both benign epithelium and PCa can adopt the club-like features by the expression of PIGR protein. Benign glands with club-like features have reduced androgen receptor activity while club-like PCa are enriched with PTEN loss by IHC. Regardless of cell type, expression of PIGR leads to close contact of T cells and club-like cells. Xenium profiling revealed that most T cells are activated CD4+ stem T cells expressing TCF1 protein that are spatially enriched with myeloid dendritic cells and plasmacytoid dendritic cells. Conclusions We defined a subset of immunogenic localized prostate cancer that expresses PIGR either in benign or tumor epithelium. Epithelial expression of PIGR is associated with TILs, suggesting a non-canonical route for immune recruitment and activation.



 Anson Ku, Rong Rong Huang, Sumeyra Kartel, Xingyu Chen, Kayla Heyward, Ross Lake, Rosina Lis, Nikil Pramod, Shana Trostel, Anna Baj, Etyan Ruppin, Scott Wilkinson, John Fenimore, Huihui Ye, Adam Sowalsky. Tumor PTEN-Loss is Associated with Club-Like Cells Phenotype and CD4 T-Cell Infiltration in High-Risk Localized Prostate Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae61f5a94b5b745d520039bbb08b848e6edeb40a" target='_blank'>
              Abstract A035: Tumor PTEN-Loss is Associated with Club-Like Cells Phenotype and CD4 T-Cell Infiltration in High-Risk Localized Prostate Cancer
              </a>
            </td>
          <td>
            A. Ku, R. Huang, Sumeyra Kartel, Xingyu Chen, Kayla Heyward, R. Lake, Rosina T. Lis, Nikil Pramod, S. Trostel, A. Baj, Etyan Ruppin, S. Wilkinson, J. Fenimore, H. Ye, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="PURPOSE
Undifferentiated pleomorphic sarcoma (UPS) is an aggressive subtype of soft tissue sarcoma with poor outcomes, particularly in metastatic cases. The mechanisms driving metastasis in UPS remain poorly understood, limiting therapeutic advances.


EXPERIMENTAL DESIGN
A multi-omics approach was employed to analyze paired primary and metastatic UPS tumor samples. Spatial transcriptomics, bulk RNA sequencing, and deconvolution analyses were performed to identify molecular pathways and immune microenvironment alterations associated with metastasis. Functional assays using CRISPR-Cas9 knockout UPS cell lines, alongside in vivo models, were used for functional validation experiments.


RESULTS
Transcriptomic analyses on 13 UPS patients revealed significant upregulation of hypoxia, epithelial-mesenchymal transition (EMT), and immune-suppressive pathways in metastatic UPS. ADORA2B was identified as a key driver of these processes, with elevated expression correlating with poor disease-free survival in UPS patients. Functional studies confirmed that ADORA2B promotes proliferation, migration, invasion, and matrix remodeling via metalloprotease regulation. In vivo, ADORA2B knockout reduced primary tumor growth and metastatic dissemination in UPS models.


CONCLUSIONS
This study identifies ADORA2B as a critical regulator of metastatic progression in UPS, implicating it as a promising therapeutic target. Ongoing clinical trials targeting adenosine pathways further support the translational potential of ADORA2B inhibition to disrupt metastasis and improve outcomes for UPS patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afcc9cdde9fce44c9f1c91b8acb720e8fa688da9" target='_blank'>
              Adenosine A2B Receptor Promotes Tumor Progression and Metastases in Undifferentiated Pleomorphic Sarcoma.
              </a>
            </td>
          <td>
            Mariella Spalato Ceruso, J. Guégan, A. Bourdon, V. Chaire, Yanina Valverde Timana, Alban Giese, C. Rey, A. Bessede, Raoul Perret, Lucile Vanhersecke, Valérie Velasco, P. Finetti, F. Bertucci, S. Verbeke, A. Italiano
          </td>
          <td>2026-02-05</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Tumor-associated neutrophils (TANs) are abundant across cancers, yet their phenotypic diversity and functional states remain poorly defined. Here, we introduce a cell-type probability classifier that recovers low-transcript neutrophils from scRNAseq datasets, enabling pan-cancer analyses of TAN heterogeneity. Across >190 human and murine tumors, we identify a conserved differentiation trajectory that culminates in a terminal CCL3hi state. This state exhibits pro-tumor transcriptional programs, including those involved in hypoxic adaptation and senescence. Consistently, CCL3hi TANs are enriched in hypoxic tumor niches in both humans and mice. Through mechanistic perturbations of neutrophil-derived CCL3 in mice, we show that it sustains TAN survival in hypoxic tumor regions via CCR1-dependent signaling. These findings establish CCL3 as a conserved marker and functional driver of pro-tumor neutrophils in growing tumors, and provide a scalable framework for dissecting neutrophil biology across cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a440321137f32ffdb6eab52c5f0b3275d2799103" target='_blank'>
              CCL3 is produced by aged neutrophils across cancers and promotes tumor growth.
              </a>
            </td>
          <td>
            Evangelia Bolli, P. Wirapati, Mehdi Hicham, Yuxuan Xie, Marie Siwicki, Florent Duval, A. Goubet, Máté Kiss, Béatrice Zitti, Thomas Zwahlen, Sheri Mcdowell, Ruben Bill, Simona Angerani, Camilla Engblom, Seth Anderson, Aiping Jiang, Oliver Hartley, David B Sykes, Maja Jankovic, Nadine Fournier, Matthias Gunzer, D. Tarussio, S. Tissot, Peter M. Sadow, William C. Faquin, Moshe Sade-Feldman, Ralph Weissleder, Sara Pai, François E. Mercier, R. Manguso, Mikael J. Pittet
          </td>
          <td>2026-02-01</td>
          <td>Cancer cell</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e2990159413c5229bc09533053b7a0234e2d63" target='_blank'>
              Distinct tumor immune microenvironmental (TIME) landscapes drive divergent immunotherapy responses in glioblastoma.
              </a>
            </td>
          <td>
            Linqian Weng, Krish Skandha Gopalan, Mélanie Guyot, M. Vanmechelen, Pouya Nazari, Marie Duhamel, Clelia Donisi, Carla Pallarés-Moratalla, A. Garg, K. White, A.T. Byrne, D. Lambrechts, F. De Smet, G. Bergers
          </td>
          <td>2026-01-31</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de1bfe3dcce6678a9d15f49378aba0a2c24733ef" target='_blank'>
              Ensemble-DeepSets: an interpretable deep learning framework for single-cell resolution profiling of immunological aging
              </a>
            </td>
          <td>
            Yin Xu, Zhengchao Luo, Kai He, Feifan Zhang, Jinzhuo Wang, Han Wen, Yongge Li, Dali Han
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor cell plasticity drives metastasis and therapy resistance, yet its regulation by oncoprotein dosage dynamics remains poorly understood. In Ewing sarcoma (EwS), variations in EWS::FLI1 (EF) fusion oncoprotein activity have been associated with epithelial-mesenchymal plasticity (EMP). Using degron technology, we precisely modulated endogenous EF in EwS cells and linked phenotypic states to distinct oncoprotein dosages. Strikingly, modest EF depletion promoted a pro-metastatic phenotype that diminished upon near-complete EF loss, revealing a paradoxical effect of submaximal EF inhibition. Nascent RNA-sequencing uncovered distinct gene clusters with heterogenous transcriptional responses to graded EF loss. Genes most sensitive to subtle EF depletion harbored GGAA microsatellites within EF-bound enhancers, while chromatin profiling uncovered candidate cofactors regulating EF-repressed EMP programs. Transient EF depletion followed by rapid restoration, modelling oncoprotein fluctuations, caused persistent dysregulation of genes functionally linked to enhanced extravasation and metastatic burden in preclinical models. This study highlights the therapeutic challenge of incomplete EF elimination, serving a paradigm in which oncoprotein dosage dynamics act as non-genetic drivers of disease progression and reveal novel vulnerabilities of advanced disease. The partial and transient suppression of the EF oncoprotein was found to paradoxically promote Ewing sarcoma metastatic potential through persistent transcriptional dysregulation, establishing the oncoprotein dosage dynamics as a key regulator of metastatic plasticity and therapeutic vulnerability. Graded EF depletion using dTAGV-1 elicited a non-linear phenotypic response, with modest EF loss being sufficient to induce a pro-metastatic state. Integration of acute transcriptional responses with chromatin profiling led to the identification of EF co-regulators. EF restoration following transient depletion resulted in lasting dysregulation of pro-metastasis-related genes and emergence of hybrid epithelial-mesenchymal states, thereby enhancing extravasation and lung metastasis in vivo. Partial or transient EF inhibition may promote disease progression and worsen clinical outcomes, highlighting that therapeutic strategies targeting EF must achieve complete and sustained suppression. Graded EF depletion using dTAGV-1 elicited a non-linear phenotypic response, with modest EF loss being sufficient to induce a pro-metastatic state. Integration of acute transcriptional responses with chromatin profiling led to the identification of EF co-regulators. EF restoration following transient depletion resulted in lasting dysregulation of pro-metastasis-related genes and emergence of hybrid epithelial-mesenchymal states, thereby enhancing extravasation and lung metastasis in vivo. Partial or transient EF inhibition may promote disease progression and worsen clinical outcomes, highlighting that therapeutic strategies targeting EF must achieve complete and sustained suppression. The partial and transient suppression of the EF oncoprotein was found to paradoxically promote Ewing sarcoma metastatic potential through persistent transcriptional dysregulation, establishing the oncoprotein dosage dynamics as a key regulator of metastatic plasticity and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5051c5027b542c8e38511f6f4e8d71524906c31f" target='_blank'>
              Modelling EWS::FLI1 protein fluctuations reveal determinants of tumor plasticity in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, Andri Konstantinou, S. Grissenberger, C. Sturtzel, M. Zylka, F. Cidre‐Aranaz, A. Wenninger-Weinzierl, Karla Queiroz, D. Kurek, Martin Distel, Anna C. Obenauf, T. Grünewald, F. Halbritter, H. Kovar, Valerie Fock
          </td>
          <td>2026-01-03</td>
          <td>EMBO Molecular Medicine</td>
          <td>2</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98cf179f4f4e7762fc39a8377027aab26eafcc9e" target='_blank'>
              Image-Based Profiling of Induced Trophoblast Stem Cells Identifies Signatures Associated with Sex, Schizophrenia Genomic Risk and Placental Stress
              </a>
            </td>
          <td>
            Frank J. Piscotta, Jiyoung Kim, Jisu Ha, Bonna Sheehan, Julia Johnston, John Peters, Thomas M. Hyde, Brady J. Maher, Juan Caicedo, D. Weinberger, Evgeny Shlevkov, Gianluca Ursini
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) provides valuable insights into the tumor microenvironment by integrating molecular features with spatial context; however, its clinical utility is limited by high costs. To address this, we develop a multi-scale convolutional deep learning framework, HiST, which utilizes ST to learn the relationship between spatially resolved gene expression profiles (GEPs) and histological morphology. HiST accurately predicts tumor regions across multiple cancer types (e.g., breast cancer; area under the curve: 0.96), demonstrating high concordance with pathologist annotations. Moreover, HiST reconstructs spatially resolved GEPs from histological images with an average Pearson correlation coefficient of 0.74 across five cancer types, outperforming existing models by about two-fold. These high-fidelity spatial GEPs enable tumor heterogeneity assessment from histological images, including identification of tumor subtypes with distinct DNA copy number variations. We demonstrate the clinical utility of the predicted GEPs, which robustly stratify patient prognosis across five cancer types from The Cancer Genome Atlas (e.g., breast cancer; concordance index: 0.78). The predicted profiles further facilitate immunotherapy response prediction and enrichment analyses of relevant biological pathways and markers. Collectively, HiST achieves state-of-the-art performance in spatial GEP reconstruction, providing a reliable molecular representation that enhances downstream tasks such as tumor profiling and clinical analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1a859f42b2a4317c7e56d1a0957cb174c649e8f" target='_blank'>
              HiST: Histological Images Reconstruct Tumor Spatial Transcriptomics via MultiScale Fusion Deep Learning.
              </a>
            </td>
          <td>
            Wei Li, Dong Zhang, Eryu Peng, Shijun Shen, Hamid Alinejad-Rokny, Yao Liu, Junke Zheng, Cizhong Jiang, Youqiong Ye
          </td>
          <td>2026-01-05</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c6d4659f58c58136e6e49d7f75c45ec0d6f013" target='_blank'>
              SenCat: Cataloging human cell senescence through multiomic profiling of multiple senescent primary cell types
              </a>
            </td>
          <td>
            C. Anerillas, Gisela Altés, Katarína Grešová, Dimitrios Tsitsipatis, K. Mazan-Mamczarz, R. Banarjee, Ana S.G. Cunningham, Martin Salamini-Montemurri, Jen-Hao Yang, R. Munk, Martina Rossi, Yulan Piao, B. Olinger, Quinn Strassheim, J. Martindale, Jinshui Fan, , Supriyo De, Delaney V Rutherford, Ying Hao, Ziyi Li, Jessica Roberts, Y. A. Qi, K. Abdelmohsen, Rafael de Cabo, Allison B. Herman, Manolis Maragkakis, N. Basisty, Myriam Gorospe
          </td>
          <td>2026-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Neoadjuvant therapy (NAT) is increasingly used for pancreatic ductal adenocarcinoma (PDAC); yet most patients only achieve partial response. Pathological treatment response grading focuses on assessing residual tumor burden, often overlooking changes in tumor microenvironment (TME). To address this gap, we compared tumor cells and TME of 13 NAT-naïve and 23 post-NAT PDACs using integrated spatial pathomics and transcriptomics, with validation in an independent single-cell spatial dataset. NAT significantly reduced tumor burden (14.7%-6.2%, p = 0.004), but systemic comparison of 13 cytomorphometric features of tumor cells alone did not reliably distinguish between naïve and NAT cases. In contrast, NAT profoundly remodeled TME by increasing cancer-associated fibroblast (CAF) and CD8+ T cell densities, promoting CD8+ T cell-tumor cell proximity and fibrosis, reducing tumor-associated neutrophils, and redistributing tertiary lymphoid structures (TLSs). Spatial transcriptomics shows NAT induced apoptosis, DNA-damage response, and AGC-kinase (S_TK_X) signaling in tumor cells, and upregulated complement pathway, p53 signaling, and cellular senescence program in TME. Cross-platform single-cell spatial analysis revealed decreased regulatory T cells (Treg) and a shift from myofibroblastic (mCAF) to inflammatory CAF (iCAF). Importantly, post-NAT patients with more fibrosis had longer overall survival (p = 0.02), and higher B-cell density showed a favorable trend (p = 0.06). Together, these results suggest that beyond tumor debulking, NAT induces a coordinated TME remodeling characterized by fibroblast reprogramming, matrix fibrosis, and immune spatial reorganization. Incorporating assessment of NAT-induced stromal and immune changes into TRG may improve prognostication and guide more precise therapy in post-NAT PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc4f312a97dc39d70b704d780122da11f4cc3af9" target='_blank'>
              Neoadjuvant Therapy-Induced Remodeling in Pancreatic Ductal Adenocarcinoma: Multimodal Spatial Analysis and Prognosis.
              </a>
            </td>
          <td>
            Xiaofei Zhang, Ruoxin Lan, Danting Li, Yongjun Liu, Sonu Kalyan, Momin Iqbal, Nancy Liu, Jerry Zhang, Iman Hanna, Mala Gupta, C. Zhao, Weiguo Liu, Jonathan Melamed, Michael Shusterman, Jessica Widmer, John Allendorf, Yao-zhong Liu
          </td>
          <td>2026-01-13</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Pancreatic cancer (PC) is among the most aggressive and lethal malignancies, characterized by development within a complex tumor microenvironment (TME) that includes a desmoplastic stroma composed of extracellular matrix (ECM) and various cellular components.


OBJECTIVES
This study aims to elucidate the cellular and molecular mechanisms regulating PC progression through an integrated analysis of single-cell pseudotime trajectories and intercellular communication.


MATERIAL AND METHODS
We constructed pseudotime trajectories using single-cell RNA sequencing (scRNA-seq) data from PC tissues to trace the developmental progression of cancer cells. Transitional cell states and critical genes involved in the shift from early-to-advanced disease stages were identified. Through a comprehensive analysis, we pinpointed key transcription factors and signaling pathways implicated in tumor progression. Expression of stemness-associated genes in pancreatic stellate cells (PSC) was validated using immunofluorescence and transmission electron microscopy (TEM). Additionally, cell-cell communication analysis was performed to examine interactions within the TME, with particular emphasis on ligand-receptor pairings.


RESULTS
Our analysis identified key transcription factors and signaling pathways that drive the cellular transitions associated with cancer progression. The findings revealed extensive intercellular crosstalk between cancer cells, stromal fibroblasts, and diverse immune cell subpopulations. Notably, the study underscored the distinct functional contributions of these cell populations to tumor development, immune evasion and metastatic dissemination.


CONCLUSIONS
The study uncovers the complex cellular diversity and intercellular crosstalk in PC, providing novel avenues for therapeutic interventions and early predictive markers in diagnosis. These findings support the potential for more targeted, personalized treatment strategies in combating PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1072a0d2e360d78c1789524d36e7d5c1df78fbc" target='_blank'>
              Single-cell pseudotime and cell communication analysis of pancreatic cancer.
              </a>
            </td>
          <td>
            Chengming Ni, Xiaohang Wang, Zhensheng Cai, Yang Chen, Huan Wang, Qianqian Wang, Hao Lin, Yunting Zhou, Yang Yuan, Bo Sun, Zilin Sun
          </td>
          <td>2026-02-09</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Rationale: Tertiary lymphoid structures (TLSs) play a key role in the adaptive immune response within the local tumor immune microenvironment (TIME) and can predict the clinical outcome of several solid tumors. However, the clinical relevance of TLSs and their formation mechanism in gastrointestinal stromal tumors (GISTs) remain unclear. Methods: Integration analysis was performed on a single-cell RNA sequencing (scRNA-seq) cohort (n = 8), a public transcriptome cohort (n = 65), a public scRNA-seq cohort (n = 7), a tissue cohort (n = 197) and a serum cohort (n = 169) to decode the characteristics of the immunological microenvironment of GIST. The multiplex immunohistochemistry (mIHC) staining, in vitro cell culture, chemotaxis assays and antibody-dependent cellular phagocytosis (ADCP) experiments were used to validate the results of the bioinformatics analysis. Results: Preoperative imatinib targeted therapy significantly enhanced TLS formation in GIST tissues, predicting improved therapeutic efficacy and favorable prognosis. Mechanistically, imatinib remodeled the local microenvironment via tumour-associated macrophages, recruiting B cells via the CXCR4-CXCL12 axis to drive TLS development. Functionally, TLS dominated germinal centre (GC) B-cell differentiation and the formation of IgG+ plasma cells (PCs), which preferentially enhanced the adaptive immune response through the ADCP effect. From a clinical perspective, we identified three distinct GIST immune classes (GICs A-C). GIC-A tumors featured abundant CD20+ B cells and TLS, as well as a favorable prognosis. They were accompanied by enhanced antigen presentation, accumulation of IgG+ PCs, increased immunosuppressive properties and a high frequency of KIT exon 11 mutations. These mutations potentially serve as a predictive biomarker for future targeted and immunotherapies. Furthermore, patients with high serum IgG levels experienced significant therapeutic benefits. Conclusions: Our data show that local adaptive immunity dominated by TLS is a key factor in the efficacy of targeted therapy, and suggest that inducing IgG could be a feasible strategy for improving the prognosis of patients with GIST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/485f7a996f0585ddffcf4dd3ac8a1112abd5a345" target='_blank'>
              Enhanced formation of tertiary lymphoid structures shapes the anti-tumor microenvironment in gastrointestinal stromal tumors after imatinib targeted therapy
              </a>
            </td>
          <td>
            Ping Tao, Jiongyuan Wang, Peidang Fan, Wen-Shuai Liu, Wei-Qi Lu, Yujie Hu, Q. Lu, Lijie Ma, Hanxing Tong, Yong Zhang
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The Tracking Cancer Evolution Through Therapy (TRACERx) program represents the most comprehensive effort to characterize tumor evolution in real time. Through longitudinal, multiregion, and multiomic profiling of tumors—and particularly of non-small-cell lung cancer and clear cell renal cell carcinoma—TRACERx has illuminated the dynamic interplay between genetic, nongenetic, and (micro)environmental factors that drive cancer progression, immune evasion, and therapeutic resistance. A central insight from TRACERx has been that not all tumor evolution is genomic: Transcriptomic diversity, epigenetic alterations, RNA editing, and changes in cell–cell interactions also drive adaptation. Methodological innovations—including tumor-informed and ultrasensitive circulating tumor DNA assays, representative sequencing, and integrative immune–genomic analyses—have yielded biomarkers resistant to sampling bias and/or predictive of recurrence, metastasis, and treatment response. By demonstrating that intratumor heterogeneity is a key determinant of clinical outcome and revealing its molecular, transcriptional, and ecosystem-level drivers, TRACERx has established a framework for linking evolutionary dynamics to patient care. As both a scientific framework and a clinical paradigm, TRACERx demonstrates how adaptive, iterative research can refine evolutionary models, improve patient risk stratification, and inspire next-generation cancer evolution studies across malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a59431b9259c4928178579f6c460cf8e369f4f1" target='_blank'>
              TRACERx as an Optimized Paradigm for Understanding Cancer Evolution
              </a>
            </td>
          <td>
            Naomi Iris van den Berg, C. Swanton, S. Turajlic
          </td>
          <td>2025-12-24</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/706f238b315ed6aac316adf86f94df56f8c2fd50" target='_blank'>
              An integrated single-cell lung cancer atlas reveals distinct fibroblast phenotypes between adenocarcinoma and squamous cell carcinomas.
              </a>
            </td>
          <td>
            Yuta Hirano, Hironori Suzuki, Jun Nakayama, Tomofumi Yamamoto, Eishu Hirata, Jun Araya, Yu Fujita, Yusuke Yamamoto
          </td>
          <td>2026-01-23</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Background Smooth muscle cells (SMCs) exhibit remarkable plasticity, undergoing extensive phenotypic switching to generate a highly heterogeneous population within atherosclerotic plaques. While recent studies have highlighted the contribution of SMC-derived macrophage-like cells to plaque inflammation, the specific molecular drivers governing the transition to these pathogenic states remain poorly understood. Methods Here, we re-analyzed single-cell RNA sequencing data from lineage-traced mice to dissect SMC heterogeneity during atherogenesis. Trajectory analysis revealed that SMCs transdifferentiate into a distinct pro-inflammatory macrophage-like subpopulation (macrophage 4) via an intermediate “stem–endothelial–monocyte" cell state. Integrated gene regulatory network inference and in silico perturbation modeling identified interferon regulatory factor 7 (IRF7) as a master transcriptional regulator orchestrating this specific pathogenic transition. Results Clinically, IRF7 expression was significantly upregulated in unstable and advanced human atherosclerotic plaques, correlating strongly with inflammatory macrophage burden. In vivo, ApoE−/− mice challenged with a high-fat diet exhibited robust upregulation of IRF7 in aortic plaques, which co-localized with macrophage markers. Crucially, SMC-specific knockdown of Irf7 using an AAV-SM22α-shIRF7 vector significantly attenuated atherosclerotic plaque progression, reduced necrotic core formation, and enhanced fibrous cap stability. Mechanistically, Irf7 silencing preserved the contractile SMC phenotype and inhibited the accumulation of pro-inflammatory SMC-derived macrophage-like cells within the lesion. Conclusions These findings identify IRF7 as a critical checkpoint in maladaptive SMC phenotype switching. We demonstrate that IRF7 drives the transdifferentiation of SMCs into a pro-inflammatory macrophage-like state, thereby fueling plaque instability. Consequently, therapeutic strategies capable of inhibiting IRF7-mediated SMC plasticity may prove effective in stabilizing vulnerable atherosclerotic plaques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc7d29acba6562a1494d454fb457ef77e3152441" target='_blank'>
              IRF7 orchestrates maladaptive smooth muscle cell phenotype switching in atherosclerosis
              </a>
            </td>
          <td>
            Rundong Cai, Xin Chen, Hongxia Zhang, Qi Wang, Wanrong Xie, Xinghua Pan, Chun Liang, Haiying Zhu
          </td>
          <td>2025-12-27</td>
          <td>Precision Clinical Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Spatial proteogenomics marks a paradigm shift in oncology by integrating molecular analysis with spatial information from both spatial proteomics and other data modalities (e.g., spatial transcriptomics), thereby unveiling tumor heterogeneity and dynamic changes in the microenvironment.


METHODS
We systematically reviewed the evolution of spatial proteogenomics, from single-modality profiling to integration with transcriptomics and metabolomics, from the detection of abundant proteins to exploration of "dark proteome" with low abundance or stability, and from analytic software based on traditional machine learning algorithms to advanced artificial intelligence-driven analytical frameworks.


RESULTS
Key advances of sub-fields of spatial proteogenomics include: RNA-protein co-localization: Spatial CITE-seq, enabling RNA-protein co-localization to reveal immune microenvironmental patterns and neoantigen distribution. Spatial Proteomics + Spatial Metabolomics: Matrix-assisted laser desorption/ionization imaging (MALDI), overcoming protein detection bottlenecks and capturing metabolic reprogramming. Deep visual proteomics (DVP): achieving unbiased spatial analysis via AI-guided microdissection. Spatial-aware multiplex dark proteome approaches: Examples are nanodroplet processing in one pot for trace samples (NanoPOTS) and proteoform imaging mass spectrometry (PiMS). Multimodal foundation AI models: Examples are KRONOS and HEIST, which integrate multiple data modalities and significantly improve diagnostic precision and therapeutic prediction.


CONCLUSIONS AND FUTURE DIRECTIONS
Despite challenges of resolution, standardization, and data complexity, spatial proteomics is advancing rapidly. Together with frontier technologies such as quantum computing, live imaging, and organoid integration, it is driving breakthroughs in cancer diagnosis, personalized immunotherapy, and drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae06afb0f045f1c14c02032ac2dc72df3bf1009d" target='_blank'>
              Integrating Spatial Proteogenomics in Cancer Research.
              </a>
            </td>
          <td>
            Yida Wang, Yang Wu, Feng Zhang, Parthiban Periasamy, Haiyue You, Denise Goh, R. E. Fincham, Xin Ning, Danping Wu, Lu Liu, Ying Jiang, Zhiwen Qian, Joe Yeong, Yan Zhang
          </td>
          <td>2026-02-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer‐related mortality worldwide, with tumor microenvironment (TME) heterogeneity playing a critical role in disease progression and therapeutic response. Immune escape (IE) mechanisms facilitate tumor evasion from host immune surveillance, yet their characterization at the single‐cell level in CRC is incomplete. This study integrated single‐cell RNA sequencing (scRNA‐seq) and bulk transcriptomic data from multiple public cohorts to systematically explore IE‐related signatures in CRC. We identified major and minor cell populations within the TME and performed differential gene expression analysis. Using high‐dimensional weighted gene coexpression network analysis (hdWGCNA), we identified gene modules correlated with IE activity. Subsequent survival analysis across six independent cohorts revealed Gamma‐glutamylcyclotransferase (GGCT) as a novel prognostic biomarker associated with poor survival. Functional enrichment analysis indicated GGCT′s involvement in critical oncogenic pathways. Furthermore, GGCT expression correlated with altered immune infiltration profiles and stromal components, suggesting its role in modulating the immunosuppressive TME. Additionally, GGCT demonstrated potential predictive value for response to immunotherapy across multiple datasets. Our findings highlight GGCT as a key player in CRC immune evasion and a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fcee0e763a97a07d783d6f8f51f97fc2a1182ca" target='_blank'>
              Integrative Multiomics Nominate GGCT as a Crucial Regulator of Immunosuppression in Colorectal Cancer
              </a>
            </td>
          <td>
            Qichao Niu, Yang Liu, Kejin Huang, Lisheng Nie, Shiming Zhao, Shifeng Yang, Changlei Su, Shuyu Li
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Cutaneous melanoma is one of the most aggressive forms of skin cancer and remains difficult to treat due to its ability to adapt and develop resistance to therapy. Recent research suggests that a small subset of tumor cells with stem-like properties play an important role in disease progression and treatment failure. One molecule of growing interest is ABCB5, a membrane transporter associated with drug resistance, immune evasion, and tumor survival. This review summarizes current knowledge on how ABCB5 contributes to melanoma progression by influencing tumor cell plasticity and the surrounding tumor microenvironment. We discuss the biological mechanisms linked to ABCB5 expression, its potential value as a prognostic biomarker, and its relevance for future therapeutic strategies. A better understanding of ABCB5 may help guide the development of more effective and personalized treatment approaches for patients with melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e464449af07273c0b0477651738ff182ea906fd3" target='_blank'>
              ABCB5: A Key Regulator Linking Stem Cell Plasticity, Tumor Microenvironment, and Therapy Resistance in Cutaneous Melanoma
              </a>
            </td>
          <td>
            A. Tinca, Adriana Sabău, A. Cozac-Szőke, D. Chiorean, B. Lazar, Raluca-Diana Hagău, I. Cocuz, Raluca Niculescu, Irina-Bianca Kosovski, Sofia Teodora Muntean, S. Turdean, O. Cotoi
          </td>
          <td>2026-01-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2438dcb543ce3b34986c62da919180266191c9c9" target='_blank'>
              Optimised fluorescence-activated nuclei sorting for epigenomic analysis of cortical cell types
              </a>
            </td>
          <td>
            B. Chioza, Stefania Policicchio, J. Burrage, Georgina E T Blake, Rosemary A. Bamford, Alice Franklin, Darren Soanes, Philippa M. Wells, Ann Babtie, Marina Flores Payan, Jonathan P. Davies, A. Klokkaris, Emma M. Walker, Joy N. Ismail, Paulina Urbanaviciute, S. Marzi, E. Hannon, J. Mill, E. Dempster
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="(Isocitrate dehydrogenase) IDH-mutant astrocytoma is classified as World Health Organization (WHO) grade 2–4 and is second only to IDH wild-type glioblastoma in the incidence of adult glioma. However, few studies use single-cell and spatial transcriptome sequencing to analyze its malignant progression. Intraoperative navigation and yellow fluorescence visualization were utilized to accurately isolate high-grade (WHO grade 3–4) and low-grade (WHO grade 2) samples of IDH-mutant astrocytoma for single-cell and spatial transcriptome sequencing. By combining single-cell, spatial transcriptome, The Cancer Genome Atlas (TCGA), and The Chinese Glioma Genome Atlas (CGGA) data, analyses of survival, enriched pathways, transcription factors, intercellular communication, differentiation trajectories, and immune response were performed to identify the characteristics of a unique subpopulation of high-grade IDH-mutant astrocytoma. Our single-cell RNA sequencing analysis identified a distinct subpopulation (Cluster 7) present in high-grade IDH-mutant astrocytoma, which was localized to the terminus of the pseudotime trajectory. Importantly, this cluster not only exhibited an immunosuppressive phenotype correlated with poor clinical prognosis, but also demonstrated significant enrichment in Developmental Biology and Calcium Signaling pathways. Furthermore, this subpopulation engaged in prominent ligand-receptor interactions, particularly through PTN_PTPRZ1 and MIF_CD74 pairs. Notably, comparative analysis revealed that high-grade astrocytoma displayed both quantitatively and qualitatively enhanced communication networks when compared to their low-grade counterparts. Our single-cell RNA sequencing analysis identifies a distinct tumor cell subpopulation present in high-grade (WHO grade 3–4) adult IDH-mutant astrocytoma. This cluster, which likely arises from malignant progression in adult astrocytoma, may provide new insights for developing therapeutic strategies against this clinically challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/586fc47b7312366f86c32ecb5b1181c5a720190b" target='_blank'>
              Single-cell and spatial transcriptome sequencing analysis reveals characteristics of a unique subpopulation in high-grade IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            Wei Chen, Min Zheng, Jia-Yin An, Guo-Hao Huang, Jing-Peng Liu, Lin Yang, Peng Ren, Tingting Wang, Jean-Philippe Hugnot, Sheng-Qing Lv
          </td>
          <td>2025-12-29</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b1158dbe48515eb2e89f44b8a0aabe075c2b9fc" target='_blank'>
              The αβTCR repertoire at scale in the immgenT dataset
              </a>
            </td>
          <td>
            Myriam Croze, Liang Yang, Serge Candéias, I. Magill, O. Casey, Val Piekarsa, Brinda Vijaykumar, V. Giudicelli, Sofia Kossida, D. Zemmour, C. Benoist
          </td>
          <td>2026-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Functional precision oncology complements genomic approaches by directly testing treatment options on patient-derived models. However, existing platformssuch as patient-derived xenografts (PDXs) and patient-derived organoids (PDOs), face major barriers in clinical use due to technical challenges, including limited standardization, high costs, long assay times, scalability constraints, and incomplete recapitulation of the patient tumor microenvironment (TME). Here, we present a scalable, low-cost Organ Chip (OC) platform fabricated entirely from thermoplastics via injection molding. Leveraging a patented channel geometry and surface treatment, the device achieves barrier-free hydrogel confinement through capillary pinning without porous membranes, micropillars, or other barrier structures. This automation-compatible platform supports tissue-specific extracellular matrices and co-culture through versatile perfusion modes, with robust imaging compatibility. We demonstrate its feasibility for drug sensitivity testing using multiple cell lines and patient-derived primary cells, with imaging-based phenotypic profiling for accurate quantification of drug responses, closely aligning with clinical outcomes. Additionally, we integrated a deep learning-based image translation model that predicts fluorescence staining from bright-field images. This approach enables longitudinal, label-free phenotypic analysis with higher sensitivity than conventional endpoint staining. Together, this integrated cancer OC system overcomes key technical challenges and offers a promising framework for functional precision oncology through high-throughput, patient-relevant drug testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4db65b80021560cec602e95093b6024fe2e70f55" target='_blank'>
              Deep Learning-Powered Scalable Cancer Organ Chip for Cancer Precision Medicine.
              </a>
            </td>
          <td>
            Yu-Chieh Yuan, Beibei Xu, Jenna McCormack, XuHai Huang, Jingzhe Ma, Thomas Marshall, Yacong Sun, Hardeep Singh, Alyssa Fanelli, Gauri Kulkarni, Ji Hye Seo, Paige Gilbride, Bing Wei, Bo Wang, Yanyan Liu, Fei Ma, Lin Zhou, Shuyang Wang, Xiaohua Qian, Zhiyong Xie, P. Golland, Longlong Si, Y. Zhang, Xin Xie, Haiqing Bai
          </td>
          <td>2026-02-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Simple Summary Clear cell renal cell carcinoma is the most common form of kidney cancer. Recent advances in spatial and long-read sequencing now allow detailed examination of gene activity within tumours. We used non-passaged, patient-derived tumour organoid models which preserve the structure and cellular composition of the original tumours. Long-read spatial transcriptomics was then applied to study gene expression and transcript variants across organoid regions. We also examined how these patterns change following treatment with the experimental medicine NUC-7738. There are distinct spatial patterns in genes related to protein synthesis and energy metabolism, as well as region-specific differences in transcript isoforms. This work provides new insight into the molecular diversity of kidney cancer and demonstrates how advanced sequencing technologies can be used to study treatment effects in physiologically relevant tumour models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11ea98f536949abee4bc7391c3123466548631ed" target='_blank'>
              Long-Read Spatial Transcriptomics of Patient-Derived Clear Cell Renal Cell Carcinoma Organoids Identifies Heterogeneity and Transcriptional Remodelling Following NUC-7738 Treatment
              </a>
            </td>
          <td>
            Hazem Abdullah, Ying Zhang, Kathryn Kirkwood, Alexander Laird, Peter Mullen, David J. Harrison, Mustafa Elshani
          </td>
          <td>2026-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Somatic mosaicism is pervasively observed in human aging, with clonal expansions of cells harboring mutations in recurrently mutated driver genes. Bulk sequencing of tissues captures mutation frequencies, but cannot reconstruct clonal architectures nor delineate how driver mutations impact cellular phenotypes. We developed single-cell Genotype-to-Phenotype sequencing (scG2P) for high-throughput, highly-multiplexed, joint capture of genotyping of mutation hotspots and mRNA markers. We applied scG2P to aged esophagus samples from six individuals and observed large numbers of clones with a single driver event, accompanied by rare clones with two driver mutations. NOTCH1 mutants dominate the clonal landscape and are linked to stunted epithelial differentiation, while TP53 mutants promote clonal expansion through both differentiation biases and increased cell cycling. Thus, joint single-cell highly multiplexed capture of somatic mutations and mRNA transcripts enables high resolution reconstruction of clonal architecture and associated phenotypes in solid tissue somatic mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e453d2ebb8e890d0f12017626a25469ee8fcb34" target='_blank'>
              Genotype-to-phenotype mapping of somatic clonal mosaicism via single-cell co-capture of DNA mutations and mRNA transcripts.
              </a>
            </td>
          <td>
            Dennis J. Yuan, John Zinno, Theo Botella, Dalia Dhingra, Shu Wang, Allegra G Hawkins, Ariel D Swett, Jesus Sotelo, Ramya Raviram, Clayton Hughes, Catherine Potenski, Katharine D Godfrey, Kara M Ainsworth, Shuzhen Xu, Jianwen Que, Julian A Abrams, Akira Yokoyama, N. Kakiuchi, Seishi Ogawa, D. Landau
          </td>
          <td>2025-12-31</td>
          <td>Cancer discovery</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Single-cell profiling of T cell state with immune repertoire is critical for understanding heterogenous T cell phenotypes and responses to antigen, however, existing technologies struggle to generate this information at sufficient throughput to match biological complexity. We present “REFLEX”, a novel single-cell method enabling highly scalable, cost-efficient, multiomic profiling with paired-chain TCR sequencing. REFLEX utilizes in-situ reverse transcription with integrated sample multiplexing barcodes in a way that merges seamlessly with the commonly used 10x FLEX platform to allow capture of TCR sequences at unprecedented scale and depth. We profile >2 million cells from CMV-peptide-pulsed T cell expansions, capturing TCR sequences and rich multiomic information from 1.4M T cells, identifying many putative novel CMV reactive clonotypes and illustrating the scale and transformative impact on our understanding of T cell mediated adaptive immunity achievable with REFLEX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b028fd9098c324cb997d47ef6d1f001635c38358" target='_blank'>
              Massively Scalable Single-cell Multiomic Profiling of T Cell Repertoire with REFLEX
              </a>
            </td>
          <td>
            Peter Skene, Matthew Hart, Zach Thomson, Saransh N. Kaul, Saskia Ilkisin, Matthieu Landreau, Tyanna J. Stuckey, Peter Wittig, Michael Keller, Chase D. McCann, Troy R. Torgerson
          </td>
          <td>2025-12-23</td>
          <td>Research Square</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Immune cells play essential roles in maintaining tissue homeostasis and responding to abnormal growth, but how innate immune cells adapt to chronic apoptotic signaling remains poorly understood. In Drosophila melanogaster, hemocytes, particularly plasmatocytes, are recruited to tumor-like overgrowths, yet their transcriptional diversity and lineage dynamics under these conditions remain undefined. Here, we apply single-cell RNA sequencing to nearly 50,000 circulating and sessile hemocytes from larvae bearing undead overgrown eye discs, a model of regenerative overgrowth driven by sustained caspase activity. We resolve 17 transcriptionally distinct hemocyte clusters, including known lineages and 13 previously unrecognized plasmatocyte subtypes. Interestingly, specific plasmatocyte populations are differentially expanded or depleted under overgrowth conditions. Notably, we identify a matrix-remodeling plasmatocyte population marked by high expression of Jonah-family serine proteases. Pseudotime analysis reveals unexpected plasmatocyte plasticity and two novel terminally differentiated effector states. These findings define the immune landscape of tumor-like overgrowth and establish Drosophila as a platform for dissecting innate immune responses to tissue stress and dysregulated growth in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afdb698f387c427fb5a2af38f4716fdae66bdb8a" target='_blank'>
              Single-cell RNA analysis reveals unexpected hemocyte plasticity and immune cell specialization in a Drosophila overgrowth model
              </a>
            </td>
          <td>
            Prathibha Yarikipati, Andreas Bergmann
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b75da29edbdbad60fea18ff4b3049824f5017d" target='_blank'>
              Critical role for a high-plasticity cell state in lung cancer.
              </a>
            </td>
          <td>
            J. Chan, Chun-Hao Pan, Jonathan Rub, Gary Guzman, Klavdija Krause, Emma Brown, Zeda Zhang, Hannah Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2026-01-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eec9f36f629bd2b6951e1744a883847ea5c28f8" target='_blank'>
              Programming T cells for Early Cancer Detection with Customized Protease-Activatable Receptors
              </a>
            </td>
          <td>
            H. Phuengkham, Ying Chen, Anirudh Sivakumar, Ali H. Zamat, Lena Gamboa, Quoc D. Mac, Hee Jun Lee, Leonard C. Rogers, Jicheng You, Sean A. Steele, Sichen Zhu, Matthew S. Gollins, J. Blazeck, Peng Qiu, G. Kwong
          </td>
          <td>2026-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Innate-like T cells (iLTCs) are rapid sentinels at epithelial surfaces, yet their spatial organisation and tissue-linked programmes in psoriatic inflammation remain incompletely defined. Spatial transcriptomics from independent cohorts maps γδT and mucosal-associated invariant T cells (MAIT) niches across psoriatic skin and reveals sharply divergent skin-layer arrangements. Psoriatic plaques show expansion of both niches, with γδT transcriptional signatures present in dermis and epidermis and MAIT signatures strongly enriched in the epidermis. Their compartment-specific positioning is mirrored by distinct transcriptional activities that support dermal-sentinel behaviour for γδT-enriched niches and epithelial-retention programmes for MAIT niches. Clinical severity associates with opposite niche dynamics, marked by decreasing dermal γδT frequencies and increasing epidermal MAIT abundance. Functional profiles reinforce this divergence, as dermal γδT niches display rising exhaustion-associated features with greater severity, whereas epidermal MAIT niches show stronger inflammatory and proliferation-related signals. Peripheral CITE-seq profiling identifies parallel systemic patterns, with reduced γδT frequencies and increased MAIT frequencies in blood, along with exhaustion-associated features in γδT cells and MAIT-specific trafficking cues that align with their behaviour in psoriatic tissue. Together the findings define a spatially imbalanced γδT–MAIT axis in psoriatic inflammation that is linked to layer-specific organisation to local inflammatory cues, systemic immune engagement and clinical severity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e673a877b802f73c6ecf1627dc524c9b6a077d7" target='_blank'>
              Spatial Imbalance of Innate-like T-Cell Niches Underlies Clinical Trajectories in Psoriasis
              </a>
            </td>
          <td>
            Caio Santos Bonilha
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Intratumoral heterogeneity drives immunotherapy resistance, hindering clinical efficacy. Functional assays that capture heterogeneous responses to immunotherapies remain limited. Here, we present a 3D live imaging assay using patient-derived ovarian cancer organoids to quantify natural killer (NK) cell-induced apoptosis across hundreds of individual organoids. Our assay reveals inter- and intratumor response heterogeneity and detects rare populations unresponsive to NK cytotoxicity, enabling the study of subclonal immune responses and underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c2a1ea73f6fab9f1966b90164d74f94fad3f917" target='_blank'>
              Large scale quantification of natural killer cell-induced apoptosis in patient-derived organoids reveals intratumoral response heterogeneity
              </a>
            </td>
          <td>
            Marisa Mercadante, Armin Scheben, Jacob Estrada, Jan Savas-Carstens, William Sullivan, Nicholas Housel, Maria Sapar, Tatiana Volpari, Jax Hebner, Tomasz Rusielewicz, Frederick J. Monsma, Stefan Semrau, Yinan Wang, Laura A. Martin
          </td>
          <td>2025-12-26</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Single-cell transcriptomics has redefined our understanding of cancer by exposing the complexity of tumor ecosystems and their therapeutic vulnerabilities. scRNA-seq studies have identified lineage hierarchies, immune evasion programs, and resistance-associated states across solid and liquid tumors, informing biomarker development and drug discovery. Advanced computational frameworks integrate these data with longitudinal profiling, RNA velocity, and network diffusion to prioritize targets and predict therapeutic response. Emerging multi-omics approaches further expand the scope of precision oncology by linking genetic alterations, protein-level markers, and spatial context to functional states. This narrative review aims to synthesize current applications of single-cell transcriptomics for target discovery, highlight computational frameworks that translate high-dimensional data into actionable insights, and explore how multi-omics integration is shaping future directions. By bridging molecular complexity with target prioritization, these approaches hold promise for translating single-cell insights into clinically actionable biomarkers and therapeutic strategies for personalized cancer treatment and rational drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc90ad6637cf26b6fef95756c86b88733af331f8" target='_blank'>
              Single-Cell Transcriptomics and Computational Frameworks for Target Discovery in Cancer
              </a>
            </td>
          <td>
            Martina Tarozzi, Nicolas Derus, Stefano Polizzi, C. Sala, Gastone Castellani
          </td>
          <td>2026-02-03</td>
          <td>Targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Gastric cancer (GC) exhibits molecular heterogeneity and diverse immune cell infiltration patterns closely associated with patient prognosis. However, a comprehensive understanding of the variations in immune cell phenotypes among different patient subgroups still needs to be improved. In this study, we performed a detailed analysis of the tumor microenvironment in GC by integrating 200,466 single cells from 72 patients across six datasets. We classified patients into immune-deserted, B, T, and myeloid cell subtypes. Using genomic and clinical data from TCGA samples, we identified cellular components associated with tumor histology and genotypes. GC patients were stratified into immune-deserted, B cell, T cell, and myeloid cell subtypes, and we described the pathway and transcription factor activity characteristics of different microenvironment subtypes. Integration of bulk RNA-seq data reveals that fibroblasts and endothelial cells were associated with adverse patient outcomes whereas NK and T cells were notably correlated with improved prognosis. Subsequently, we focused on characterizing cancer-associated fibroblasts (CAFs) and discovered that they acquire new functional properties within the tissue microenvironment, providing evidence of CAF plasticity. We constructed a novel four-gene CAF signature including SPARC, EFEMP1, RGS5 and SERPINE1 which may enhance patient stratification and prognostic prediction of GC patients. qPCR analysis revealed that the significant expressions of SPARC, EFEMP1, RGS5 and SERPINE1 were significantly upregulated in gastric cancer tissues compared to the normal tissues. Our study provides insights into the composition of the tumor microenvironment and construction of a four-gene CAF signature associated with clinical prognosis, offering new perspectives for the clinical management of gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6b22dc4e8c3bd98a01915b46ab3333a465c3c7e" target='_blank'>
              Integrative single-cell atlas unveils heterogeneity and prognostic value of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Ziyu Jiang, Xiaomei Zhuang, Junkui Guo, Minyi Zhu, Chunhong Hong, Chunhui Sun, Kaiming Wu, Haofan Yin, Cuncan Deng, P. Jiang
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ab2ef7cfea18c9d2b8d0291643654a27ff4767a" target='_blank'>
              ISG15-driven immune modulation and tumor progression in breast cancer metastasis: insights from single-cell and spatial transcriptomics.
              </a>
            </td>
          <td>
            Hua Shao, Hanlu Tang, Huiying Lin, Yongqing Xu
          </td>
          <td>2026-02-04</td>
          <td>BMC medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous malignancy characterized by complex interactions between tumor cells and the immune system. The tumor microenvironment (TME) plays a crucial role in CRC progression and response to therapy. However, the mechanisms regulating TME composition remain poorly understood due to the genetic and phenotypic diversity of tumor cells. In this study, we investigated the tumor-intrinsic factors contributing to TME formation and evaluated genotype-based combination strategies to enhance the efficacy of immunotherapy in CRC. Using RNA sequencing, single-cell analysis, and immunohistochemistry, we identified pro-oncogenic proteins associated with low immune activation. Functional studies using in vitro co-culture systems, subcutaneous CRC tumor models, flow cytometry, and immunohistochemistry revealed a role for CLK1 in tumor progression and immunosuppressive TME remodeling. Mechanistically, CLK1 activation led to hyperactivation of the Hippo signaling pathway, promoting nuclear translocation of YAP and subsequent transcriptional upregulation of the chemokine CXCL1. Elevated CLK1 expression correlated with increased infiltration of myeloid-derived suppressor cells (MDSCs) and impaired antitumor immune responses. Knockdown of CLK1 significantly reduced MDSC recruitment and restored CD8+ T-cell activity. Moreover, combined CLK1 knockdown and anti-PD1 therapy enhanced intratumoral CD8+ T-cell infiltration to a greater extent and elicited robust antitumor responses in murine CRC models. Collectively, our findings identify the CLK1-Hippo/YAP-CXCL1 signaling axis as a regulator of immune evasion and TME remodeling in CRC and highlight the potential of therapeutically targeting this axis to improve the efficacy of immune checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b873ab484ecd90dcc4e4881af5bdcfefa69529" target='_blank'>
              CLK1 Promotes Myeloid-Derived Suppressor Cell Trafficking and Reprograms the Tumor Microenvironment by activating Hippo/YAP signaling in Colorectal Cancer.
              </a>
            </td>
          <td>
            Peisi Li, Yumo Xie, Dawang Zhou, Xuan Li, Qishan Liu, Ziming Li, Xiaoxia Liu, Jinxin Lin, Meijin Huang, Qian Yan, Yanxin Luo, Huichuan Yu, Xiaolin Wang
          </td>
          <td>2026-02-13</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Brain invasion of meningioma is a controversial clinicopathological grading criterion that correlates with prognosis and is a stand-alone criterion for WHO grade 2 meningioma. Molecular correlates of meningioma cells and immune cell infiltration at the brain-meningioma border and its possible role in brain invasion are not known. We hypothesized that brain-invasive meningiomas have distinct gene expression profiles in meningioma cell populations and in tumour-associated microglia and macrophages (TAM) populations, most prominently at the brain-meningioma border. We performed spatial transcriptomics using NanoString GeoMx Digital Spatial Profiling to investigate the gene expression profiles of meningioma cell-enriched populations (Iba1-/CD68-) and TAM-enriched populations (Iba1+/CD68+) across 16 tumours. Regions of interest included core regions of meningiomas, the brain-meningioma border, and brain regions with confirmed invasion. Using an 1800 gene panel, we analysed differential gene expression across regions within each tumour and compared brain-invasive and non-invasive meningiomas. Meningioma cell-enriched populations from brain-invasive meningiomas (n = 8) showed significant upregulation of KRT18, implicated in filament reorganization, as well as genes involved in the cell cycle, glycolysis, and the growth factor receptor PDGFRB compared to non-invasive meningiomas (n = 8). Meningioma cell-enriched populations from the brain-meningioma border showed significant upregulation of KRT18, NDUFA4L2, and PKM relative to core areas. In TAM-enriched populations in brain tissue, we found upregulation of the chemokine receptor gene CSF1R relative to TAM-enriched populations in meningioma tissue and increased TAM infiltration in brain-invasive cases. In conclusion, our results demonstrate molecular changes in brain-invasive meningiomas compared to non-invasive meningiomas as well as spatial changes along the core-border axis. The expression pattern of TAMs also changed from meningioma to brain tissue. Additional studies are needed to confirm these findings and further reveal how we can target meningioma brain invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/269f040bbd53f8bc31b9e6b6faae7f3967b17a9f" target='_blank'>
              Novel insights into meningioma brain invasion with spatial transcriptomic profiling
              </a>
            </td>
          <td>
            A. Maier, Claes Ottzen Laurentiussen, S. Michaelsen, Lorenzo Perino, T. Stannius, D. Harwood, M. N. Klausen, J. Haslund-Vinding, N. Bager, David Scheie, A. Schmidt, L. Melchior, A. Areškevičiūtė, Frederik Vilhardt, K. Josefsen, D. Raleigh, F. Bagger, Tiit Mathiesen, B. W. Kristensen
          </td>
          <td>2026-01-04</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background The complexities of nucleotide metabolism in breast cancer (BC) cells are not yet fully understood. A deeper exploration of the various tumor subpopulations and their interactions with the tumor microenvironment (TME) could provide important insights into these clinically relevant signaling pathways. Methods We integrated five distinct single-cell enrichment scoring methodologies to conduct a comprehensive enrichment analysis of BC cell populations. The analytical findings underwent subsequent validation using an independent single-cell cohort. Tumor cell clusters were categorized based on their average enrichment scores. Functional analyses were carried out using several tools, including CellChat, Monocle, CopyKAT, SCENIC, and CytoTRACE. The RCTD method was then employed to map the single-cell clusters onto spatial transcriptomics data, facilitating the evaluation of cellular dependencies and pathway activities to differentiate tumor cell subtypes. A prognostic framework was subsequently established using large-scale transcriptomic datasets, enabling prediction of immunotherapy responsiveness. Experimental validation further confirmed expression patterns of pivotal genes implicated in therapeutic outcomes. Results Tumor cells exhibit significantly upregulated nucleotide metabolic activity, enabling their classification into two distinct subgroups: NUhighepi and NUlowepi. Cells within the NUhighepi subgroup demonstrate pronounced malignant phenotypes. Intercellular communication analysis performed with the stLearn platform revealed robust interactions between NUhighepi cells and fibroblasts. Supporting this finding, spatial transcriptomic analysis via MISTy revealed a distinct dependency of NUhighepi on fibroblasts. A robust prognostic model, developed using various machine learning algorithms, was able to predict survival outcomes and responses to immunotherapy. Furthermore, targeted drugs were identified for both the high and low scoring groups. Experimental investigations confirmed the expression of core genes in different breast cancer cells. Discussion In conclusion, we developed a nucleotide metabolism-derived prognostic signature for BC, with DCTPP1 highlighted as a promising biomarker and therapeutic target. These findings provide a valuable framework for early clinical intervention and show promising potential for predicting responses to immunotherapy in BC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502ca564395f021e293ba0e223c1f50472f85541" target='_blank'>
              Spatial transcriptome and single-cell sequencing reveal the role of nucleotide metabolism in breast cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yuejun Pan, Yong Xu, Ke Gong, Songlin Yuan
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="High-throughput"pathomic"analysis of Whole Slide Images (WSIs) offers new opportunities to study tissue characteristics and for biomarker discovery. However, the clinical relevance of the tissue characteristics at the micro- and macro-environment level is limited by the lack of tools that facilitate the measurement of the spatial interaction of individual structure characteristics and their association with clinical parameters. To address these challenges, we introduce HistoWAS (Histology-Wide Association Study), a computational framework designed to link tissue spatial organization to clinical outcomes. Specifically, HistoWAS implements (1) a feature space that augments conventional metrics with 30 topological and spatial features, adapted from Geographic Information Systems (GIS) point pattern analysis, to quantify tissue micro-architecture; and (2) an association study engine, inspired by Phenome-Wide Association Studies (PheWAS), that performs mass univariate regression for each feature with statistical correction. As a proof of concept, we applied HistoWAS to analyze a total of 102 features (72 conventional object-level features and our 30 spatial features) using 385 PAS-stained WSIs from 206 participants in the Kidney Precision Medicine Project (KPMP). The code and data have been released to https://github.com/hrlblab/histoWAS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407dd7f4b75161f7922e517e4737dcf219aed8fa" target='_blank'>
              HistoWAS: A Pathomics Framework for Large-Scale Feature-Wide Association Studies of Tissue Topology and Patient Outcomes
              </a>
            </td>
          <td>
            Yuechen Yang, Junlin Guo, Yanfan Zhu, Jialin Yue, Junchao Zhu, Yu Wang, Shilin Zhao, Haichun Yang, Xingyi Guo, Jovan Tanevski, Laura Barisoni, Avi Rosenberg, Yuankai Huo
          </td>
          <td>2025-12-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Prostate cancer (PCa) has long been viewed as an immunologically “cold” malignancy because immune checkpoint inhibitors (ICIs) show limited benefit in unselected patients, particularly after progression to metastatic castration-resistant PCa (mCRPC) or treatment-related neuroendocrine PCa (NEPC). Single-cell and spatial profiling now reveal immune heterogeneity across patients, between lesions, and along the path from localized disease to metastasis. Primary tumors form mosaics of immune-excluded glands, myeloid-suppressed stromal borders, and focal lymphocyte-rich niches with B-cell aggregates and tertiary lymphoid structures (TLS). TLS-high regions represent an actionable “hot minority” resembling inflamed, ICI-responsive cancers, supporting biomarker-guided neoadjuvant or focal immunotherapy. With dissemination, heterogeneity expands across sites; bone metastases become marrow immune organs dominated by suppressive macrophage/monocyte programs and dysfunctional T cells, often driven by the CCL2–CCR6 axis. Standard therapies remodel these ecosystems, creating inflammatory windows yet fostering adaptive resistance. Mechanistically, myeloid-driven, inflammation-coupled rewiring is central to escape: IL-8/CXCR2 signaling and therapy-induced senescence/SASP recruit and polarize suppressive myeloid cells, reinforcing T-cell exclusion and exhaustion. Variable HLA class I loss and hypoxic or metabolic “functional cold zones” add lesion-specific immune invisibility. Clinically, these insights motivate a heterogeneity-aware framework integrating genomic responder subsets with microenvironmental stratification. Barrier-matched strategies include T-cell redirection (PSMA/STEAP1 engagers, bispecifics, CAR-T) and combinations that heat or modulate myeloid cells. Treating immune heterogeneity as a clinical variable enables durable immunotherapy in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9ca8f1edda00999e41285037eb35603c7bbf58e" target='_blank'>
              Clinically oriented immune heterogeneity in prostate cancer: emerging targets and strategies
              </a>
            </td>
          <td>
            Mingwei Zhan, BinBin Zhao, Junjie Wu, Kai Li, Yibo Chen, Haote Chen, Lin Zhao, Jingyu Zhu
          </td>
          <td>2026-02-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intratumoral microbiota are now recognized as an integral component of the tumor microenvironment, affecting tumor initiation, metastatic potential, immune modulation, and treatment response. However, their extremely low biomass poses significant challenges for accurate detection, functional interpretation, and reproducibility, largely because the detection process is highly susceptible to environmental contamination. Standardization of analytical procedures has not yet been established; consequently, variability in sampling protocols, sequencing workflows, and bioinformatic pipelines further complicates cross-study comparisons and hampers the consolidation of robust evidence in this field. Recent advances in technology have begun to provide opportunities to overcome these barriers. Improved contamination-control strategies and more sophisticated decontamination algorithms have enhanced the reliability of microbial detection in low-biomass tissues. High-resolution approaches, such as single-cell RNA sequencing, spatial transcriptomics and optimized anaerobic cultivation, enable the sensitive identification, spatial localization, and mechanistic study of tumor associated microbes. Parallel developments in genome-resolved and enzyme-level analysis reveal microbial metabolic pathways that shape immune responses, drug resistance, and tumor progression. Organoid-based co-culture models further provide physiologically relevant platforms to dissect host-microbe-immune interactions and interpret microbiota-driven modulation of therapeutic responses. Integrating microbiome data with clinical and multi-omics profiles, assisted by artificial intelligence, is accelerating biomarker discovery and informing microbe-guided therapeutic strategies. Taken together, the standardization of research strategies, combined with the application of advanced detection technologies, is propelling the field beyond descriptive profiling toward mechanistic understanding and clinical translation, thereby unlocking the potential of intratumoral microbiota for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c30f5b43755a6fb82c98ee347498e9577b80237" target='_blank'>
              Emerging technologies and current challenges in intratumoral microbiota research
              </a>
            </td>
          <td>
            Zhiyue Wang, Tianqi Zhang, Yang Liu
          </td>
          <td>2026-01-02</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acne, the most prevalent inflammatory skin condition globally, lacks comprehensive understanding of its cellular and molecular mechanisms. This research integrates single-cell transcriptomics with spatial immunofluorescence to detail cellular composition and functional states in healthy skin, mild acne, and severe acne. Findings indicate that keratinocytes in mild acne secrete antimicrobial peptides and maintain barrier defense, but transition to abnormal differentiation and pathological keratinization in severe acne. Natural killer (NK) cells initially amplify inflammation and activate the immune response, but later become exhausted and apoptotic. This interaction creates a “dual immune-barrier imbalance,” driving acne progression. Functional enrichment and pseudo-time analyses demonstrated a strong association between disease severity and TNF, and chemokine signaling pathways, as well as NK cell-mediated cytotoxicity and Wnt/p53-apoptosis pathways. These findings were corroborated by immunofluorescence validation. This study introduces an innovative pathological model that establishes theoretical foundations for early diagnosis and precision interventions, while also outlining future therapeutic strategies centered on “immune-barrier co-regulation.” The findings provide essential insights into the pathogenesis of acne and open avenues for investigating novel intervention targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e83afe735e0121344e0a07c8a405ac15ec431c" target='_blank'>
              Single-cell transcriptomics and spatial validation reveal dysfunction of keratinocytes and NK cells driving acne development
              </a>
            </td>
          <td>
            Xinheng Wang, Changzhao Gao, Dandan Yang, Jiahui Pan, Rihua Jiang
          </td>
          <td>2025-12-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Diffuse large B-cell lymphoma (DLBCL) is an aggressive cancer with profound heterogeneity. While standard immunochemotherapy is curative for the majority, treatment failure in 30–40% of patients due to refractory or relapsed disease remains a significant clinical challenge, emphasizing the need for better prognostic markers. We investigated the interplay between tumor glycolytic metabolism and the tumor immune microenvironment. We identified four distinct glycolysis-related patterns that were associated with three immune subtypes (known as immune-inflamed, immune-excluded, and immune-desert). Importantly, these patterns stratified clinical outcomes and predicted chemotherapy sensitivity. For example, tumors with an immune-inflamed subtype were more sensitive to doxorubicin. Collectively, our findings propose a metabolism- and immune-based classification framework for DLBCL that may facilitate more precise risk stratification and treatment optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e894f2d6a04501c694424859815514794b12befc" target='_blank'>
              Glycolysis-Driven Immune Subtypes in the Tumor Microenvironment Determine Clinical Outcomes in Diffuse Large B-Cell Lymphoma
              </a>
            </td>
          <td>
            Ying Du, Zhenyu Liu, Chuanyan Liu, Yining Yuan, Mengkun Fang, Tong Zhou, Mengge Pan, Weilu Xu, Xu Liu, Peipei Xu
          </td>
          <td>2025-12-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43834c7632e582a60435566324798a1fd37dbe2b" target='_blank'>
              Fibroblast heterogeneity in 2025
              </a>
            </td>
          <td>
            A. Croft, A. Hackland
          </td>
          <td>2025-12-19</td>
          <td>Nature Reviews Rheumatology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc6a6f856a7083dd2ac5b388edeaab80c7f5a72" target='_blank'>
              A spatially resolved atlas of gastric cancer characterises a lymphocyte-aggregated region.
              </a>
            </td>
          <td>
            Sen Gao, Shishang Qin, Dongfang Wang, Anqiang Wang, Linna Zhu, Yang Li, Qiang Shi, Hongtao Fan, Yufei Bo, Yunshan Zhong, Yuxuan Sun, Kun Dong, Liqin Fu, Ranran Gao, Yan Wu, Ye Liang, Luyujie Huang, Xueda Hu, Xianwen Ren, Z. Bu, Jiafu Ji, Zemin Zhang
          </td>
          <td>2026-01-27</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec774215b12f2be4ebfe5634529efaca54fcef6" target='_blank'>
              Toward Personalized Treatment of Urogenital Cancers: The Role of Patient-Derived Organoids.
              </a>
            </td>
          <td>
            S. Sagliocchi, Michele Musone, Stefano Chianese, A. G. Cicatiello, Silvia Del Mastro, Francesco Del Giudice, M. Dentice, F. Crocetto
          </td>
          <td>2026-01-08</td>
          <td>Oncology and therapy</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 Computational pathology has emerged as an attractive option for improving risk stratification in prostate cancer (PCa), but current approaches either lack interpretability or focus solely on tumor morphology, potentially limiting their utility. We aimed to identify an interpretable, immune microenvironment-derived computational pathology biomarker for PCa.



 We retrospectively identified two independent cohorts (Discovery and Validation) with localized PCa who underwent radical prostatectomy (RP) and had digitized H&E-stained whole-slides images (WSIs) and longitudinal outcomes data. We identified a third cohort from The Cancer Genome Atlas (TCGA) with localized PCa which was treated with RP and had WSIs, bulk RNA sequencing, and whole-exome sequencing. Immune cells were identified from WSIs using a publicly available deep learning model (CellViT), and spatially dense immune clusters were quantified using density-based spatial clustering of applications with noise (DBSCAN). CIBERSORTx was utilized for immune cell deconvolution and TRUST4 for immune receptor repertoire reconstruction from bulk RNA sequencing. Cox proportional hazard regression was used to examine associations between clinicopathologic features and the primary outcome of time to distant metastasis (DM), as well as secondary outcomes of biochemical recurrence (BCR) and overall survival (OS).



 Median follow-up periods for the Discovery (n=272) and Validation (n=218) Cohorts were 12.6 and 8.1 years, respectively. In the Discovery Cohort, median age was 63, 14% had Gleason 8-10 disease, 94% had pT2-T3a disease, and median prostate-specific antigen level at diagnosis was 6.2 ng/mL. Median immune cell proportion was 4.3% (interquartile range 3.0-5.9%), and median immune cluster density was 2.4 per 400 mm2 of tissue (interquartile range 0-7.1). Similar values were observed in the Validation Cohort. In both cohorts, immune cell proportion was not associated with BCR, DM, or OS (P>=0.09). In the Discovery Cohort, while increasing immune cluster was not associated with BCR (P>=0.10), it was independently associated with a decreased risk of DM for Gleason 8-10 (adjusted hazard ratio [AHR] 0.42, 95% confidence interval [CI] 0.19-0.93) but not Gleason 6-7 patients (AHR 1.26, 95% CI 0.77-2.05; Pint=0.020). Similar results were observed in the Validation Cohort, where increasing immune cluster was not associated with BCR (P>=0.10) but was associated with a lower risk of DM in Gleason 8-10 disease (AHR 0.60, 95% CI 0.37-0.98), though not Gleason 6-7 (AHR 1.19, 95% CI 0.74-1.91; Pint=0.043). In TCGA Cohort (n=326), immune cluster was not associated with somatic alterations. For Gleason 8-10 but not Gleason 6-7 disease, high-cluster samples were enriched for CD8+ T cells, activated memory CD4+ T cells, and Tregs (P<=0.037), as well as clonal T cell populations (P<=0.039).



 Our findings nominate immune spatial clustering as a novel, interpretable computational pathology biomarker and provide insights into the unique immune features of high-grade PCa.



 David D. Yang, Aya Abdelnaser, Alexander J. Haas, Jeremiah Wala, Alfred A. Barney, Eddy Saad, Jett P. Crowdis, Cora A. Ricker, Seifeldin Awad, Bora Gurel, Jihye Park, Martin T. King, Paul L. Nguyen, Toni K. Choueiri, David J. Einstein, Steven P. Balk, Alok K. Tewari, Johann S. de Bono6, Keyan Salari, Mary-Ellen Taplin, Chin-Lee Wu, Eliezer M. Van Allen. . Immune spatial organization predicts metastasis risk in aggressive localized prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aace1806fe7d01338e5e9857c054699a561213e" target='_blank'>
              Abstract PR001: Immune spatial organization predicts metastasis risk in aggressive localized prostate cancer
              </a>
            </td>
          <td>
            D. Yang, Aya Abdelnaser, Alexander J. Haas, Jeremiah Wala, Alfred A. Barney, Eddy Saad, Jett Crowdis, Cora A. Ricker, Seifeldin Awad, B. Gurel, Jihye Park, Martin T. King, P. L. Nguyen, T. Choueiri, David J Einstein, Steven P. Balk, A. Tewari, J. D. De Bono, Keyan Salari, M. Taplin, Chin-Lee Wu, E. V. Van Allen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe264f31f9c4a5de4c7512002b7b04159fd2f32" target='_blank'>
              immgenT Treg: A Reference Landscape of Regulatory T Cell States in Mice
              </a>
            </td>
          <td>
            A. Freuchet, N. Mehrotra, I. Magill, O. Casey, X. Chi, S. Zhang, C. Imianowski, J. R. Huh, D.A.A. Vignali, C. Benoist, D. Zemmour
          </td>
          <td>2026-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="This review systematically elaborates on the spatiotemporal dynamics and dual role of Therapy-Induced Senescence (TIS) in remodeling the Tumor Microenvironment (TME). The hallmark of TIS is the Senescence-Associated Secretory Phenotype (SASP), which drives multidimensional TME reprogramming through the secretion of various factors. These effects include the activation of Cancer-Associated Fibroblasts (CAFs), promotion of Vasculogenic Mimicry (VM), induction of metabolic reprogramming, and bidirectional regulation of the immune landscape. The article provides a focused analysis of the heterogeneous manifestations of this dual effect across different treatment stage and spatial locations, highlighting the definition of the threshold between its tumor-suppressive and tumor-promoting functions as a central current challenge. Finally, it explores future strategies involving multi-omics dynamic monitoring, artificial intelligence analysis, and spatiotemporally specific targeted interventions. In summary, this review aims to provide a theoretical foundation and translational directions for developing novel combination therapies targeting the senescent microenvironment by offering an in-depth analysis of the spatiotemporal dynamics of TIS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0a417455b1f2d68f74a2db9622b023e3a544a5b" target='_blank'>
              Research progress on the spatiotemporal dynamics of therapy-induced senescence in remodeling the tumor microenvironment
              </a>
            </td>
          <td>
            Qingqing Zhao, Yunyan Yu, Chaorui Pu, Shujuan Zheng, Lin Chen, Feng Zeng, Li Liu, Dan Li
          </td>
          <td>2026-01-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6c25b39df997323e228b3c6d265efc22e9a7958" target='_blank'>
              Multi-omics characterization of IDH-mutant astrocytoma-derived cell lines reveals NOTCH-regulated plastic quiescent astrocyte-like state
              </a>
            </td>
          <td>
            L.Garcia, C.Granotier-Beckers, D.Pineau, D.Sika, S.Hideg, C.Labadie, K.Aguilar Cázarez, J.Kundnani, L.Stuani, L.Gauthier, F.Boussin, M-A.Mouthon, S.Urbach, K.El Koulali, M.Séveno, C.Ripoll, K.Daddi, M.Verreault, A.Idbaih, S.O’Connor, C.Plaisier, L.Zhang, M.Zheng, V.Rigau, H.Duffau, J. Hugnot
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cb53da812dbac416c29e7f7225ad2ab6decfdb9" target='_blank'>
              Exploring the Space of Tumor Phylogenies Consistent with Single-Cell Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Samin Rahman Khan, P. Sashittal
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Triple Negative Breast Cancer (TNBC) is an aggressive subtype of breast cancer with limited treatment options and poor prognosis. Coupled with this, TNBC shows a high rate of metastasis which is known to be aided by tumour vasculature. Endothelial cells that form the lining of the tumour vasculature exhibit distinct genotype and phenotype differences compared with normal tissue vasculature. However, little is known about endothelial signatures that drive metastasis from a primary tumour, particularly in the context of immune infiltration. In this study, we utilized GeoMX Digital Spatial Profiling to investigate spatial proteomics differences in endothelial cells between primary and secondary sites of TNBC. By segmenting tissues using epithelial (PanCK), immune (CD45), and endothelial (CD31) markers, we analysed the microvasculature for a panel of 79 target proteins. In paired primary and secondary TNBC tissues, we identified significant downregulation of fibronectin (− log2(Fold-Change) = − 1.7, p < 0.001) in secondary sites. Specifically in epithelial regions, S100B was found to be downregulated in secondary microvasculature when compared to primary tumours. Additionally, metastasis-free primary tissues exhibited upregulated expression of S100B when compared to primary tissues that metastasized. Our study highlights the potential contributions of microvasculature to metastatic progression in TNBC, presenting new opportunities to explore them as potential biomarkers of TNBC metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9355568219b2a2ea616006baa27f93ab8c621b2" target='_blank'>
              Digital spatial profiling identifies features of primary and locoregional metastatic vasculature in triple negative breast cancer
              </a>
            </td>
          <td>
            Akhilandeshwari Ravichandran, Kyle Upton, Shiva Taheri, Cheng Liu, Kaltin Ferguson, Mark N. Adams, Laura J. Bray
          </td>
          <td>2026-01-19</td>
          <td>Clinical & Experimental Metastasis</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Objective Irinotecan, a standard therapeutic agent for metastatic colorectal cancer (mCRC), often faces significant limitations due to drug resistance, with treatment failure observed in approximately 30%–50% of patients, leading to poor clinical outcomes. This study aims to systematically elucidate the molecular mechanisms underlying irinotecan resistance in colorectal cancer (CRC) by constructing patient‐derived organoid (PDO) models combined with single‐cell transcriptomics technology. Methods PDO models were successfully established from irinotecan‐resistant and sensitive CRC patients. Single‐cell RNA sequencing (scRNA‐seq) was performed on the organoids, analyzing the transcriptomic heterogeneity of 12,360 high‐quality cells. Gene Set Variation Analysis (GSVA), transcriptional regulatory networks, and cell communication networks were employed to dissect the resistance mechanisms. Results Drug sensitivity assays demonstrated that the IC50 value of irinotecan in CRC5 was significantly higher than that in CRC11, which was entirely consistent with their respective clinical phenotypes. Single‐cell sequencing identified CRC5‐specific drug‐resistant cell clusters, Cluster 1 and Cluster 6. Cluster 1 (MARCKSL1+) was characterized by the activation of the Wnt signaling pathway and extracellular matrix (ECM) remodeling, which collectively contributed to the maintenance of stem cell‐like properties, while Cluster 6 (AKR1C3+) exhibited significant enrichment in lipid metabolism and the Notch signaling pathway. Conclusion This study integrates PDO models with single‐cell transcriptomics technology to reveal key cell subpopulations and molecular mechanisms underlying irinotecan resistance. The core mechanisms driving resistance involve the activation of Wnt signaling and the synergistic effect of lipid metabolism‐Notch pathways. Cluster 1 and Cluster 6 are identified as potential therapeutic targets, providing a theoretical basis for developing combination therapies targeting cancer stem cells or the metabolic microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f0a63ee429f4f2e263c5a1e20c2a5052c1bd9d" target='_blank'>
              Colorectal Cancer Organoid Model Reveals the Mechanisms of Irinotecan Resistance at Single‐Cell Resolution
              </a>
            </td>
          <td>
            Yi Pan, Lin Chen, Yuqing Hu, Jie Chang, Xifeng Xu, ShuoChen Xu, Yiwen Li, Jinlin Du, Jianping Wang, Wenxia Xu
          </td>
          <td>2026-02-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Uveal melanoma (UM) is a highly aggressive intraocular malignancy with limited therapeutic options for metastatic disease. Existing transgenic UM mouse models inadequately recapitulate human disease progression, while transplant models lack immune competence for studying the tumor immune microenvironment and therapeutic interventions. To address these limitations, we developed a genetically engineered mouse model incorporating stepwise genetic alterations implicated in human UM progression. Spatiotemporally controlled expression of mutant GNAQQ209L from the endogenous locus induced choroidal nevi with limited penetrance. Concomitant BAP1 deletion enhanced nevus formation, while further MYC activation led to fully penetrant intraocular tumors with the potential to disseminate. Single-cell RNA sequencing revealed malignant cells segregated into melanocytic and neural crest-like subpopulations characterized by distinct transcriptional and biosynthetic programs. Trajectory analyses inferred dedifferentiation from the melanocytic toward the neural crest-like state during tumor progression. Comparison to human UM revealed commonalities with highly aggressive class 2 UM, including gene expression signatures and copy number gains affecting genes that map to human chromosome 8q beyond the activated MYC allele, suggesting cooperative effects of multiple drivers in this chromosomal region. The tumor microenvironment featured immunosuppressive macrophage populations and exhausted T cells, closely resembling human UM. This physiologically relevant, immune-competent model provides a platform for investigating UM biology, functionally characterizing candidate driver genes, and developing immune-based therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8fdbc1108ba636781e58c18b409a85d4d4db19d" target='_blank'>
              A Multi-Step Immune-Competent Genetically Engineered Mouse Model Reveals Phenotypic Plasticity in Uveal Melanoma.
              </a>
            </td>
          <td>
            Xiaonan Xu, Xiaoxian Liu, Vinesh Jarajapu, Sathya Neelature Sriramareddy, Filip Konecny, Benjamin Posorske, J. Dollar, Xiao Liu, Neel Jasani, Kaizhen Wang, Nicol Mecozzi, Zulaida Soto-Vargas, Shaaron L. Ochoa-Rios, Harini Murikipudi, Nhan Phan, Manon Chadourne, J. Kuznetsoff, John Sinard, Richard L Bennett, Jonathan D. Licht, K. Smalley, J. W. Harbour, Xiaoqing Yu, F. Karreth
          </td>
          <td>2026-02-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) represents one of the most prevalent malignancies worldwide and poses a critical public health challenge due to difficulties in early diagnosis, therapy resistance, and high mortality rates. The complex tumor microenvironment (TME) of HCC plays a pivotal role in tumor progression, immune evasion, metastasis, and treatment resistance. Single-cell sequencing (scRNA-seq) has emerged as a revolutionary tool for resolving the intricacies and cellular heterogeneity of the TME, with its applications in advancing therapeutic research attracting considerable attention. As the primary battleground for antitumor immune responses, the HCC tumor TME warrants comprehensive analysis of immune cell subsets at distinct developmental and functional states to elucidate the complexity of tumor immunology. This review synthesizes extensive research on TME immune cellular subpopulations, in order to summarize mainstream classifications of immune subsets at single-cell resolution and analyze their functional significance and therapeutic value through biomarker gene profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04e89b37b343268e54c54a4f61fb15cb643f6a03" target='_blank'>
              Single-cell dissection of hepatocellular carcinoma immunity: from heterogeneous subtypes to precision therapeutics
              </a>
            </td>
          <td>
            Yuni Liang, Hemeng Wu, Yunshan Qiu, Qiulian Mo, Peipei Chen, Mingfen Li, Hongsheng Lin
          </td>
          <td>2026-02-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Epigenetic reprogramming is a major contributor to therapy resistance in prostate cancer (PCa). Indeed, chromatin accessibility profiling classified PCa into distinct subtypes, predictive of clinical outcome. Histone modifications influence nucleosome positioning, ultimately dictating chromatin compactness. The histone methyltransferase G9a, frequently upregulated in advanced PCa, catalyzes a repressive histone modification (di-methylation of histone H3 lysine 9) that signals for gene transcriptional repression. CM-272 is a reversible and highly selective inhibitor against G9a activity, and we have previously shown in cell line models of PCa that it depicted strong anti-tumoral effects. However, fundamental aspects of altered transcriptomic programs remain unexplored. Here, we validated, in vivo, the CM-272 effects in advanced PCa, and explored the molecular mechanisms underscoring G9a inhibition-mediated effects. Specifically, the PCa patient-derived xenograft model BM18 was used to assess the mode of action of CM-272 in vivo. Moreover, RNA sequencing of tumor tissues was performed to further explore the underlying gene expression signature. Finally, chromatin immunoprecipitation followed by qPCR was used to investigate the underlying functional consequence of epigenetic targeting. Our results showed that CM-272 significantly delayed BM18 tumor growth, specifically impairing tumor cell G9a activity and proliferation. Transcriptomic profiling revealed 119 significantly upregulated and 99 downregulated genes in response to G9a inhibition. Notably, by integrating our gene expression dataset with published chromatin accessibility data, an increase in the androgen receptor (AR) signature and a concomitant decrease in the neuroendocrine (NE) features were observed in CM-272-treated samples. Additionally, pathway enrichment and master regulator analysis pinpointed both the androgen receptor and TP53 networks as significantly upregulated in treated tissues. In line with the reduction of the NE score, the MYC/MYCN were the most significantly downregulated transcription factors. Moreover, under the androgen pathway, upregulation of AZGP1, STK39, and DPP4 was observed. At the promoter level, a significant reduction in H3K9me2 levels was detected in treated samples, allowing, in turn, an increase in AR binding. Additionally, upon treatment, higher p53 binding was detected at the promoter of the p53-repressed genes AURKA, CDC25B, and CD44, whose expression was downregulated. Overall, targeting G9a with CM-272 might hold promising therapeutic potential for the management of advanced disease by modulating relevant tumoral transcriptional programs associated with core vulnerabilities in PCa.



 Filipa Moreira-Silva, Mark Zimmerli, Panagiotis Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio. G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5911786a7c980e0c86fa5660d05b013ec96ea309" target='_blank'>
              Abstract A045: G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay
              </a>
            </td>
          <td>
            Filipa Moreira-Silva, Mark Zimmerli, P. Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a prevalent and aggressive malignancy with a persistently high mortality rate, largely attributable to therapy resistance and tumor recurrence. This review comprehensively explores the critical interplay between epigenetic dysregulation and the tumor microenvironment (TME) in driving OSCC progression. We detail how key epigenetic mechanisms, including DNA methylation, histone modifications, and non-coding RNAs (ncRNAs), intrinsically transform cancer cells and actively orchestrate pro-tumorigenic TME. These alterations substantially contribute to resistance against conventional therapies. Furthermore, we discuss the therapeutic potential of targeting these pathways using epigenetic drugs (epi-drugs), such as DNA methyltransferase (DNMT) inhibitors and histone deacetylase (HDAC) inhibitors, as well as engineered extracellular vesicles (EVs). The primary objective of this review is to synthesize current knowledge on the epigenetic-TME axis, thereby providing a mechanistic foundation for developing novel therapeutic strategies. We emphasize that rational combinations of epigenetic-targeting agents with conventional treatments or immunotherapy hold significant promise for overcoming drug resistance and improving clinical outcomes in OSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96aaa3d5aaab2d3886a4c338f86161bbd9cbefef" target='_blank'>
              Dissecting the role of epigenetic regulation in oral squamous cell carcinoma microenvironment: mechanisms and therapeutics
              </a>
            </td>
          <td>
            Xuechao Li, Yifei Ren, Shenghua Pei, Kai Zhao, Guanyu Chen, Zhenglin He
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer progression involves dynamic crosstalk between tumor-intrinsic pathways and microenvironmental remodeling, and identifying pan-cancer biomarkers is critical for precision oncology. CDKN2AIPNL exhibits a paradoxical role in cancer, acting as a tumor suppressor in myeloid malignancies but promoting solid tumor progression, yet its systematic pan-cancer characteristics remain unelucidated. This study aimed to comprehensively analyze CDKN2AIPNL’s expression patterns, prognostic value, genetic alterations, and molecular mechanisms across multiple tumor types using public datasets including TCGA, GTEx, HPA, and tools such as GEPIA2, cBioPortal, TIMER2, STRING, and BioGRID. We performed expression difference analysis, survival analysis (overall survival, disease-free survival, progression-free survival), genetic alteration analysis, cancer-associated fibroblast (CAF) infiltration analysis, and gene/protein interaction enrichment analysis. Results showed that CDKN2AIPNL was significantly upregulated in multiple tumors (e.g., LIHC, UVM, BRCA, LUAD) and downregulated in others (e.g., KICH, KIRP, THCA), with high tumor specificity. Elevated CDKN2AIPNL expression correlated with poor overall survival in LIHC (HR = 1.7, p = 0.0026), UVM (HR = 26, p = 2.2e-6), BRCA (HR = 26, p = 2.2e-6), LUAD (HR = 1.36, p = 0.049), PCPG (HR = 1.71, p = 0.0012), and TGCT (HR = 0.37, p=0.023), and was associated with advanced tumor stages in metabolically active cancers. Genetic alterations (amplifications and mutations) were frequent in KIRC (>5%) and ACC (>4%), with all mutations localized to the XTBD region, and amplification predicted poor prognosis in PRAD (p = 0.008) while mutations conferred favorable outcomes in BLCA. CDKN2AIPNL expression positively correlated with CAF infiltration in ESCA, KICH, UVM, and other tumors, and interacted with MYC, XRN2, and CHAMP1 to regulate metabolic reprogramming, cell cycle, and immune suppression. Our findings systematically reveal CDKN2AIPNL’s dual role in tumorigenesis and validate it as a potential pan-cancer prognostic biomarker, providing novel insights for cancer diagnosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1804517cd011102843d1d1b41c7599a28bf25c60" target='_blank'>
              CDKN2AIPNL: a potential pan-cancer biomarker
              </a>
            </td>
          <td>
            Yulin Yuan, Shengjie Ma, Heshi Liu, Yang Gong, Xuan Sun, Quan Wang, Weifu Zhang
          </td>
          <td>2026-01-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Metastatic castration-resistant prostate cancer (mCRPC) is a heterogeneous disease, in which patients each carry distinct molecular features that drive metastatic progression. However, the lack of clinically representative models that recapitulate this disease have limited our understanding in the heterogeneity between patients. LuCaP 147, established from a liver metastasis biopsy, is a prostate cancer transplantable xenograft carrying AR and SPOP mutations with a further hypermutated phenotype, as well as loss of MSH2 and MSH6 expression.



 Using LuCaP 147, we developed a novel patient-derived metastasis (PDM) model in NSG mice via intracardiac injection. The development of metastasis was monitored through weekly bioluminescent imaging. We analyzed the tumor burden and survival rate of the PDM147 models. At the endpoint, long bones, spines and soft tissues were collected. To evaluate the histologic phenotype of PDM147, tissue sections were stained by immunohistochemistry (IHC) for adenocarcinoma markers including AR and neuroendocrine markers (which include SOX2, ASCL1, and SYP).



 After intracardiac injection, tumor burden (BLI) was detected after 5 weeks, and mice developed morbidity after 7-8 weeks. H&E stains showed that PDM147 developed bone metastases in spines, long bones, as well as visceral metastases in liver, brain, spinal cord, kidney and adrenal glands. IHC indicated that metastases were heterogeneous with both AR+/NE- and AR-/NE+ cells. AR+ and SYP+ cells were intermingled in the tumor, but the stains themselves were mutually exclusive between cels. Interestingly, no androgen-deprivation therapy was needed to promote NE differentiation, showcasing the biological novelty [SA([1] of this model.



 PDM147 is a hypermutated tumor with mutually exclusively expressing AR+ and NE+ markers in metastatic sites, including soft tissues and nervous tissues. Future work to define distinct pathways and molecular markers that lead to de novo vs. treatment-induced NE phenotype may offer a better understanding of the biology driving metastatic spread in these similar yet distinct NE subtypes and the role a hypermutator tumor can have in prostate cancer progression. These findings underscore the value of patient-derived models to study mCRPC progression and heterogeneity. Further use of these models for high-throughput drug screens may uncover targeted therapies for advanced prostate cancer.



 Dana S. Vargas Solivan, JuanJuan (Ivy) Yin, Tri Truong, Joseph Twohig, Ross Lake, Adam G. Sowalsky. A patient-derived metastasis model with spontaneous neuroendocrine transformation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B080.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8565cd695486e6d924d7f7c053c2a965518cb8d0" target='_blank'>
              Abstract B080: A patient-derived metastasis model with spontaneous neuroendocrine transformation
              </a>
            </td>
          <td>
            Dana S Vargas Solivan, JuanJuan Yin, Tri M Truong, Joseph Twohig, R. Lake, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The effective application of foundation models to translational research in immune-mediated diseases requires multimodal patient-level representations that can capture complex phenotypes emerging from multicellular interactions. Yet most current biological foundation models focus only on single-cell resolution and are evaluated on technical metrics often disconnected from actual drug development tasks and challenges. Here, we introduce EVA, the first cross-species, multimodal foundation model of immunology and inflammation, a therapeutic area where shared pathogenic mechanisms create unique opportunities for transfer learning. EVA harmonizes transcriptomics data across species, platforms, and resolutions, and integrates histology data to produce rich, unified patient representations. We establish clear scaling laws, demonstrating that increasing model size and compute translates to improvements in both pretraining and downstream tasks performance. We introduce a comprehensive evaluation suite of 39 tasks spanning the drug development pipeline: zero-shot target efficacy and gene function prediction for discovery, cross-species or cross-diseases molecular perturbations for preclinical development, and patient stratification with treatment response prediction or disease activity prediction for clinical trials applications. We benchmark EVA against several state-of-the-art biological foundation models and baselines on these tasks, and demonstrate state-of-the-art results on each task category. Using mechanistic interpretability, we further identify biological meaningful features, revealing intertwined representations across species and technologies. We release an open version of EVA for transcriptomics to accelerate research on immune-mediated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40457e1ef9a55b517338162ae28fc6a5f9cd198d" target='_blank'>
              EVA: Towards a universal model of the immune system
              </a>
            </td>
          <td>
            Ethan Bandasack, V. Bouget, Apolline Bruley, Yannis Cattan, Charlotte Claye, M. Corney, Julien Duquesne, K. E. Kanbi, Aziz Fouch'e, P. Marschall, F. Strozzi
          </td>
          <td>2026-02-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Medullary thyroid carcinoma (MTC) is a rare neuroendocrine malignancy with a 10-year mortality rate up to 50%. Current therapies for metastatic MTC comprise various tyrosine-kinase inhibitors, but resistance often appears due to the need for lifelong treatments. Like in other tumors, genetic, epigenetic, post-transcriptional, post-translational, and cell-cell interaction events influence drug response. However, progress in understanding MTC biology is limited by the lack of reliable in vivo and in vitro models. This study aims to develop a patient-derived model faithfully reproducing the microenvironmental alterations present in MTC.


METHODS
We applied a two-step protocol consisting of a first phase in which primary cells are cultivated as multicellular spheroids and a second phase in which they are switched to adherent cultures. After evaluation of the genetic background by targeted Next Generation Sequencing, we characterized our cells phenotype by examining a panel of stem/progenitor-related markers, the secretory abilities by ELISAs, the drug response by proliferation assays, the in vivo angiogenic and invasiveness by the use of zebrafish model, the in vitro invasivity by Matrigel Dome assays and the spatial variation of stem/progenitor marker in both 3D cell models and tissue samples by confocal microscopy.


RESULTS
Our model allowed the establishment of eight MTC patient-derived cell lines with different genetic backgrounds. The cultures faithfully reproduced the changes in stem and progenitor markers that we detected in our cohort of MTC tissue samples and could be successfully xenotransplanted in zebrafish model, showing both angiogenetic and invasive properties. Drug screening assays revealed the potential of our model for the study of patient-specific responses, as we were able to identify different candidate regulators of the sensitivity to currently available therapies for MTC.


CONCLUSIONS
Our two-step protocol successfully generated primary MTC lines that maintain high plasticity, can be cultivated for several passages, and recreate the heterogeneity observed in patients' tissues. Our model will offer a robust platform for preclinical drug testing and mechanistic studies, addressing a longstanding gap in MTC research. It enables exploration of tumor microenvironment interactions and personalized therapeutic responses, supporting progress beyond current genomic-driven frameworks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efc387a5fab7de78625be6d806bb5f60287b5f7b" target='_blank'>
              Patient-Derived in Vitro Models Reveal Insights into Medullary Thyroid Cancer Microenvironment and Resistance to Tyrosine Kinase Inhibitors.
              </a>
            </td>
          <td>
            E. Grassi, V. Ghiandai, G. Gaudenzi, E. Massardi, V. Cirello, Erika Carbone, Davide Gentilini, Sarah Uraghi, G. Gazzano, G. Dionigi, C. Colombo, Giovanni Vitale, L. Fugazzola, L. Persani
          </td>
          <td>2026-02-06</td>
          <td>Thyroid : official journal of the American Thyroid Association</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="PURPOSE
Immune checkpoint blockade (ICB) benefits only a subset of sarcoma patients. Biomarkers of response and resistance are needed to help guide patient selection.


PATIENTS AND METHODS
We analyzed peripheral blood and tumor samples from sarcoma patients treated on five ICB-based clinical trials. Baseline peripheral blood mononuclear cells (PBMCs) underwent 11-color flow cytometry to define T cell immunotypes. Baseline tumor tissue underwent RNA sequencing to classify tumors into four tumor microenvironment (TME) subtypes using consensus clustering of 29 functional gene expression signatures. Associations between immune features and clinical outcomes were assessed. A deep-learning model was applied to baseline hematoxylin and eosin (H&E) slides to detect and quantify lymphoid aggregates in patients with available RNA sequencing.


RESULTS
Among 178 patients with PBMCs available for analysis, a proliferative (PRO) circulating T cell immunotype was associated with inferior overall survival (OS) compared with LAG- or LAG+ immunotypes. RNA sequencing from 67 tumors identified an immune-enriched/non-fibrotic TME subtype associated with higher response rate, longer progression-free survival, and longer OS compared to immune-enriched/fibrotic, immune-depleted, and fibrotic subtypes. Automated analysis of 48 baseline H&E slides identified lymphoid aggregates in five tumors; four were classified as immune-enriched and two responded to ICB.


CONCLUSIONS
Sarcoma patients with a PRO circulating T cell immunotype had inferior outcomes to ICB, while those with an immune-enriched/non-fibrotic TME had superior outcomes. Automated analysis of H&E slides showed promise in identifying patients with an immune-enriched TME. These findings support utilization of a multimodal approach toward identifying predictors of response to immunotherapy in sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20da94b38d82fe9dba8a8ce6c91ac335c7b826c4" target='_blank'>
              Association of Circulating T Cell and Tumor Microenvironment Profiles with Immune Checkpoint Blockade Outcome in Sarcoma.
              </a>
            </td>
          <td>
            E. Rosenbaum, Fiona D. Ehrich, M. Yosofvand, M. Bradic, Jasme Lee, Mathew Adamow, S. Movva, C. Kelly, V. Avutu, Lauren B Banks, Jason E Chan, P. Chi, M. Dickson, M. Gounder, M. Keohan, Robert G. Maki, Damon R. Reed, Paige Fuentes, Paige Collins, R. Desir, Allison Reiner, Oleg Baranov, Konstantin Chernyshov, N. Kotlov, A. Subramanian, E. Moding, , Phillip Wong, W. Tap, C. Antonescu, K. Panageas, Ronglai Shen, S. D’Angelo
          </td>
          <td>2026-02-12</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>139</td>
        </tr>

        <tr id="The molecular basis of spermatogenesis, which is a tightly regulated spatiotemporal process in testicular seminiferous tubules where germ cell differentiation and somatic-germ cell interactions drive sperm production, remains incompletely understood. Histological staining techniques lack molecular resolution, while scRNA-seq loses spatial context. Conventional spatial transcriptomics (approx. 100 μm resolution) is too coarse-grained for testicular cells (10–20 μm in diameter), leading to mixed-cell signals. In this study, we used Salus-STS high-resolution spatial transcriptomics (∼1 μm resolution) and Salus Cellbins Algorithm to characterize the spatial transcriptomic profile of mouse testes at single-cell level. Integrating the spatial data with scRNA-seq via RCTD annotated major cell subtypes, whose distributions aligned with histology. Pseudotime and spatial gradient analyses revealed a basement membrane-to-lumen developmental axis, with luminal genes (e.g., Prm2) enriched in sperm maturation and basal genes (e.g., mt-Nd4) linked to mitochondrial metabolism—validated by PPI and GO analyses. The biological relevance of these marker genes is underscored by the fact that mutations in Prm2 and mt-Nd4 are known to be associated with human male infertility, highlighting their potential diagnostic value. This work enables high-resolution dissection of spermatogenesis’ spatiotemporal dynamics, providing insights into male reproductive biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86f42d010ab2a2f75bb593b6d47ba4a6304d93f3" target='_blank'>
              Spatiotemporal dynamics of spermatogenesis: insights from high-resolution spatial transcriptomics and pseudotime trajectories in mouse testes
              </a>
            </td>
          <td>
            Han Liang, Jianlong Sun, Zaoxu Xu, Defeng Fu, Hangyu Zhang, Xiaoran Zhou, Chen Li, Huihua Xia, Gailing Li, Renjie Liao, Qi Wang, Erkai Liu, Luyang Zhao, Yuanye Bao, Gufeng Wang
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Reproductive Health</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Identifying the cell of origin that harbors an initial driver mutation is key to understanding tumor evolution and for the development of new treatments. For isocitrate dehydrogenase (IDH)-mutant gliomas, the most common malignant primary brain tumor in young adults, the cell of origin is currently poorly understood. We conducted deep sequencing on 142 tissues from 70 individuals comprising tumors, peritumoral cortex or subventricular zones, and blood. Low-level IDH mutations were found in the peritumoral cortex in 37.9% (11 of 29) of patients. Integrating cell-type-specific mutation analysis, the direction of clonal evolution, spatial transcriptomics from patient brains, and a cancer mouse model arising from mutant oligodendrocyte progenitor cell, we determined that glial progenitor cells harboring an initial IDH mutation were responsible for the development of IDH-mutant gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87878ebfd4fd1887849bd103fe178e14f5e98a11" target='_blank'>
              IDH-mutant gliomas arise from glial progenitor cells harboring the initial driver mutation.
              </a>
            </td>
          <td>
            Jung Won Park, Jiehoon Kwak, K. Kim, Saehoon Jung, Chang Hyun Nam, Hyun Jung Kim, Sang Mee Lee, Chanho Choi, Yongjin Ahn, Ji-Hyung Park, Jihwan Yoo, J. Shim, H. Cho, E. Kim, Chungyeul Kim, Sangjeong Ahn, Stefan Pusch, Andreas von Deimling, Jong Hee Chang, Se Hoon Kim, Hoon Kim, Young Seok Ju, S. Kang, J. H. Lee
          </td>
          <td>2026-01-08</td>
          <td>Science</td>
          <td>2</td>
          <td>19</td>
        </tr>

        <tr id="Tumorigenesis is a complex and dynamic process in which the tumor microenvironment (TME) plays a central role. In solid tumors, the TME contributes to key mechanisms of tumor progression, including metastasis, immune evasion, and resistance to therapies. One major challenge in preclinical cancer research is the development of reliable three-dimensional (3D) in vitro models, which more accurately replicate the in vivo tumor architecture and microenvironmental conditions, such as hypoxia and extracellular matrix (ECM) organization. However, reproducing functional vascular networks and neo-angiogenesis within these models remains a key challenge. In this study, an advanced 3D tumor model, referred to as angiotumoroids, was developed by co-culturing primary murine breast tumor cells (PTCs) with species-specific adipose-derived microvascular fragments (MVFs). Angiotumoroids were characterized using scanning electron microscopy and immunostaining, and angiogenesis was evaluated through collagen gel sprouting assays. High-resolution proteomic profiling was conducted, focusing on signatures associated with angiogenesis, extracellular matrix (ECM) composition, and tissue remodeling. Additionally, the response and internalization to anticancer drug treatments were evaluated. MVFs are successfully integrated in angiotumoroids, resulting in the formation of vasculature-like structures and demonstrating robust structural organization with dynamic modulation of matrix metalloproteinase 9. Formation of neovasculature was visualized through sprouting and branching, driven by both direct PTC–MVF interactions and PTC-conditioned media, highlighting the roles of juxtacrine and paracrine signaling. Proteomic profiling revealed distinct expression patterns associated with angiogenesis, ECM components (including collagen types I and IV), and active ECM remodeling with elevated MMP expression. Additionally, angiotumoroids showed increased expression of ATP-binding cassette (ABC) transporters, particularly ABCB1 (P-glycoprotein), suggesting potential mechanisms of drug efflux. Functionally, angiotumoroids demonstrated reduced sensitivity to doxorubicin compared to PTC spheroids, maintaining structural integrity and higher cell viability post-treatment. Time-course analysis revealed preferential doxorubicin accumulation in MVF-enriched regions, as confirmed by colocalization with CD31, indicating a spatially regulated distribution of the drug mediated by the vascular compartment. Collectively, these findings establish angiotumoroids as a robust and physiologically relevant in vitro model for studying tumor vascularization, ECM dynamics, and therapeutic response. This platform holds significant promise for predictive cancer research and preclinical drug screening, bridging the gap between traditional in vitro systems and in vivo models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba884bb3dbf2cdd1df613387c3fda90c3178112" target='_blank'>
              A vascularized three-dimensional model integrating primary breast tumor cells and microvascular fragments: mimicking the tumor microenvironment involved in chemoresistance
              </a>
            </td>
          <td>
            Alessandra Lo Cicero, Francesco La Monica, Gabriele Lo Buglio, S. Campora, Federico Gangemi, Paolo Cinà, Monica Salamone, Margot Lo Pinto, S. D. Scilabra, G. Ghersi
          </td>
          <td>2026-01-08</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Fibro-adipogenic progenitors (FAPs) in skeletal muscle have been implicated in type 2 diabetes (T2D) risk, yet their heterogeneity and context-dependent regulation remain poorly understood. Here, we establish induced pluripotent stem cell (iPSC)-derived FAPs as a faithful model of primary FAPs by leveraging a unique resource: iPSC lines and skeletal muscle biopsies obtained from the same 30 individuals. Donor-matched comparisons reveal that iPSC-FAPs recapitulate the transcriptome, epigenome, and subtype composition of muscle tissue FAPs. Using single-nucleus multiomics, we show that high-insulin exposure drives iPSC-FAPs toward an adipogenic fate - and that this adipogenic subtype is enriched for T2D GWAS signals, an enrichment undetectable under baseline conditions. We map the T2D-associated rs3814707 non-coding signal to LTBP3, a gene that influences FAP adipogenic differentiation. These findings reveal how disease-relevant regulatory mechanisms can be masked in unstimulated cells and establish iPSC-FAPs as a powerful platform for dissecting the state-dependent biology of complex metabolic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c24422e44bd269d8bb1c46a8a551de00d7a317f" target='_blank'>
              Donor-matched iPSC model reveals context-dependent T2D genetic signals in fibro-adipogenic progenitors
              </a>
            </td>
          <td>
            Christa Ventresca, Arushi Varshney, Peter Orchard, Ha T.H. Vu, Yao-chang Tsan, André Monteiro da Rocha, Michael R. Erdos, Leena Kinnunen, T. Lakka, J. Saramies, M. Laakso, J. Tuomilehto, K. Mohlke, M. Boehnke, Laura J. Scott, H. Koistinen, Francis S. Collins, Todd J. Herron, Stephanie Bielas, Stephen C. J. Parker
          </td>
          <td>2026-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>152</td>
        </tr>

        <tr id="Abstract Liquid biopsies can detect actionable mutations and infer broad tumor states from genome-wide cfDNA measurements, but quantitative transcriptome-like phenotyping at single gene resolution still largely requires tissue. Here, we asked whether 6-base whole-genome sequencing that jointly quantifies 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC) could infer gene expression directly from plasma. We applied this framework to plasma from patients enrolled in a phase 2 clinical trial of the PARP inhibitor olaparib plus the PD-L1 inhibitor durvalumab for metastatic castration-resistant prostate cancer. Inferred plasma transcriptomes distinguished adenocarcinoma vs. neuroendocrine phenotypes and identified a noncanonical WNT5A-associated signature linked to poor clinical response. Integrating longitudinal cfDNA methylomic profiles with phylogenetic reconstruction further revealed two resistance trajectories: one featuring high tumor heterogeneity with persistent AR signaling, and another marked by an AR-independent, stem-like program with metabolic reprogramming. These findings demonstrate that ctDNA can inform phenotype-driven tumor biology at gene-level resolution, integrating epigenetic modifications, inferred transcriptional programs, and clonal dynamics as a function of treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/577ed4ef29e6ee2ab69965179663102c2fe760c9" target='_blank'>
              Integrative dual ctDNA 5mC/5hmC methylomics and clonal reconstruction infer tumor transcription and resistance phenotypes in metastatic prostate cancer
              </a>
            </td>
          <td>
            A. Sowalsky, Chennan Li, A. Baj, Clara C Y Seo, N. Terrigino, J. Bright, S. Hennigan, I. King, S. Wilkinson, Tzu-Ting Huang, S. Trostel, William Figg, William L Dahut, David Y. Takeda, Jung-Min Lee, F. Karzai
          </td>
          <td>2026-02-05</td>
          <td>Research Square</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Precise regional patterning is fundamental to tissue organization, yet the spatial logic that governs it remains poorly defined for many tissues. In the vertebrate retina, molecular domains along the dorsoventral and nasotemporal axes provide positional cues for regional specializations such as the high-acuity area (HAA). We combined multiplexed in situ hybridization data with single-cell transcriptomic data to create quantitative two-dimensional maps of developing retinal cells. In the developing chicken retina, this approach resolved sharp expression boundaries of genes involved in patterning, and revealed novel candidates enriched in the anlagen of the HAA. Comparative analysis of chicken, mouse, and human data demonstrated conserved axis-based programs, but distinct fine-scale organization consistent with presence/absence of an HAA. Here, we show that spatial reconstruction from scRNA-seq data, anchored by experimental benchmarks, enables comparative 2D topographic mapping of gene expression across species and provides a generalizable strategy to investigate the spatial logic of molecular organization in developing tissues. GRAPHICAL ABSTRACT HIGHLIGHTS Quantification of multiplexed RNA-FISH signals generated reproducible spatial expression maps in the developing chicken retina. Resolution of relative boundaries for early patterning genes revealed nested expression domains marking the boundaries of the developing HAA. RNA-FISH experimental data using genes with axis-specific expression allowed the generation of DV and NT scores which enabled the localization of each cell in a scRNA-seq dataset onto a 2D topographical map of the developing retina. 2D topographic maps derived from scRNA-seq data revealed novel HAA-enriched genes in the developing chicken retina. Cross-species comparisons showed conserved DV/NT programs but divergent fine-scale organization in retinal development. Cyp26c1 expression pattern was validated in human retina correlating with the location of the future HAA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a3cc17dc30cd06a2816413ac3056bbe7f0b4c6a" target='_blank'>
              Integration of in situ hybridization and scRNA-seq data provides a 2D topographical map of the developing retina across species
              </a>
            </td>
          <td>
            H. Joisher, ChangHee Lee, Chaitra Prabhakara, Isabella van der Weide, Yichen Si, Nicholas Lonfat, Constance L. Cepko
          </td>
          <td>2026-01-04</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Spatiotemporal coordination of cellular and molecular events is crucial for cell fate commitment during mouse gastrulation. However, the high-precision mechanisms governing the timing and spatial dynamics remain poorly understood. Here, we present a time-series single-cell multi-omic dataset of the gastrulating mouse embryos and construct a hierarchical gene regulatory landscape. Integrating this with real three-dimensional transcriptomic coordinate, we created ST-MAGIC and ST-MAGIC (+) atlas, dissecting the spatiotemporal logics of regulatory networks and signaling responsiveness underpinning the lineage commitment at gastrulation. Specifically, we delineated the multi-omic basis for left-right symmetry breaking events in the gastrula and also revealed the spatiotemporal molecular relay for axial mesendoderm lineage, where early and intermediate transcription factors first open the chromatin regions and setup the responsiveness to signaling, followed by terminal factors to consolidate the transcriptomic architecture. In summary, our study presents a spatiotemporal regulatory logic framework of mouse gastrulation for advancing our understanding of mammalian embryogenesis. Spatiotemporal coordination of cellular and molecular events is crucial for cell fate commitment. Here, Yang et al. describe how transcription factors, signaling pathways, and epigenomic states regulate left-right patterning and axial mesendoderm lineage commitment during mouse gastrulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97406edf1e8aa57b2a229cfd271f6141ceb517ac" target='_blank'>
              Integrated multi-omic atlas reveals the hierarchy of spatiotemporal regulatory networks of mouse gastrulation
              </a>
            </td>
          <td>
            Xianfa Yang, Bingbing Xie, Penglei Shen, Yingying Chen, Chunjie Li, Fengxiang Tan, Yumeng Yang, Yun Yang, Rui Song, Panpan Mi, Zhiwen Liu, Mingzhu Wen, Patrick P. L. Tam, Shengbao Suo, Naihe Jing
          </td>
          <td>2026-01-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78e4ae0869f5e784613b7036dab4e240738c3eaf" target='_blank'>
              Spatial proteomic mapping of the human and mouse retina using IBEX
              </a>
            </td>
          <td>
            Yuxuan Meng, Jakub Kubiak, Zuzanna Dzieniak, Lorna Fowler, Rose Avient, Jason Hopley, Linyulong Li, Chaoran Li, Yuan Tian, Bruno Charbit, Colin J. Chu
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The human prostate develops from the urogenital sinus through epithelial bud formation and branching, however the spatial dynamics of progenitor populations during these early stages remain poorly characterized. To addresses this gap, we used a human fetal prostate at 14 weeks of gestation for serial sectioning, yielding approximately 200 sections of 10 µm thickness. Six sections were profiled using spatial transcriptomics with the Visium HD platform, while the remaining sections were H&E-stained for 3D reconstruction of glandular architecture. We mapped transcriptional heterogeneity across developing prostate buds and identified distinct spatial organization of epithelial subdomains with basal-like and luminal-like characteristics. Notably, we also identified a unique cluster of tubule-initiating cells (TIC cells), that have been linked to cancer initiation and androgen-deprivation therapy resistance in adult prostate. This study provides the first high-resolution spatial map of developing human prostate, highlighting the organisation of progenitor populations.



 Emmanouela Perisynaki. 3D spatial mapping of developing human prostate glands [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B059.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9e67e64e1e6f9570a547e8411a3ed76fac26e1" target='_blank'>
              Abstract B059: 3D spatial mapping of developing human prostate glands
              </a>
            </td>
          <td>
            Emmanouela Perisynaki
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND Breast cancer is one of the most prevalent malignancies affecting women worldwide, with approximately 2.3 million new cases diagnosed annually. Breast cancer stem cells (BCSCs) play pivotal roles in tumor initiation, progression, metastasis, therapeutic resistance, and disease recurrence. Cancer stem cells possess self-renewal capacity, multipotent differentiation potential, and enhanced tumorigenic activity, but their molecular characteristics and regulatory mechanisms require further investigation. AIM To comprehensively characterize the molecular features of BCSCs through multi-omics approaches, construct a prognostic prediction model based on stem cell-related genes, reveal cell-cell communication networks within the tumor microenvironment, and provide theoretical foundation for personalized treatment strategies. METHODS Flow cytometry was employed to detect the expression of BCSC surface markers (CD34, CD45, CD29, CD90, CD105). Transcriptomic analysis was performed to identify differentially expressed genes. Least absolute shrinkage and selection operator regression analysis was utilized to screen key prognostic genes and construct a risk scoring model. Single-cell RNA sequencing and spatial transcriptomics were applied to analyze tumor heterogeneity and spatial gene expression patterns. Cell-cell communication network analysis was conducted to reveal interactions between stem cells and the microenvironment. RESULTS Flow cytometric analysis revealed the highest expression of CD105 (96.30%), followed by CD90 (68.43%) and CD34 (62.64%), while CD29 showed lower expression (7.16%) and CD45 exhibited the lowest expression (1.19%). Transcriptomic analysis identified 3837 significantly differentially expressed genes (1478 upregulated and 2359 downregulated). Least absolute shrinkage and selection operator regression analysis selected 10 key prognostic genes, and the constructed risk scoring model effectively distinguished between high-risk and low-risk patient groups (P < 0.001). Single-cell analysis revealed tumor cellular heterogeneity, and spatial transcriptomics demonstrated distinct spatial expression gradients of stem cell-related genes. MED18 gene showed significantly higher expression in malignant tissues (P < 0.001) and occupied a central position in cell-cell communication networks, exhibiting significant correlations with tumor cells, macrophages, fibroblasts, and endothelial cells. CONCLUSION This study comprehensively characterized the molecular features of BCSCs through multi-omics approaches, identified reliable surface markers and key regulatory genes, and constructed a prognostic prediction model with clinical application value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86e7f7dec5d3e190d876efe5f1afc46eeaf94a9" target='_blank'>
              Breast cancer stem cell activity driven by ME18D gene expression in the tumor microenvironment
              </a>
            </td>
          <td>
            De-Yang Guo, Zhang-Yi Liu, Qian-Chuan Yi
          </td>
          <td>2026-01-26</td>
          <td>World Journal of Stem Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a malignant disease that poses a significant threat to human health; however, early diagnostic and treatment strategies for it remain limited. Immune evasion is a critical factor contributing to treatment failure in CRC. Various cell subtypes within the tumor microenvironment (TME) play essential roles in this process. However, there is currently a lack of a systematic and novel classification of immune evasion-related cell subtypes and an analysis of their dynamic interaction networks within the CRC TME. This study aims to explore a novel classification of immune evasion-related subtypes in CRC, elucidate their underlying mechanisms, and assess their value for immunotherapy and prognosis. Methods This study investigates immune escape-related gene expression profiles utilizing single-cell RNA sequencing (scRNA-seq), which were subsequently validated through multiple immunohistochemistry (mIHC) techniques. Non-negative matrix factorization (NMF) clustering was employed to identify novel subtypes associated with immune escape. Additionally, CellChat and pseudotime analysis were utilized to explore intercellular interactions and differentiation pathways. Kyoto Encyclopedia of Genes and Genomes (KEGG), Single-Cell Regulatory Network Inference and Clustering (SCENIC), and immune checkpoint analyses were conducted to elucidate the functional characteristics of these novel subtypes. Furthermore, Cox proportional hazards regression analysis and Kaplan-Meier survival analysis were performed to assess the response to immunotherapy and prognosis. Results The expression profile of immune escape-related genes in the TME of CRC was initially plotted. This analysis identified 11 distinct types of immune escape-related cells in the TME, and confirmed that TGF-β+JAK1+Calretinin+ could serve as a candidate cell marker for the immunosuppressive state of the TME. Furthermore, novel subtypes of cancer-associated fibroblasts (CAFs), CD8+ T cells, macrophages, and B cells were identified. These subtypes exhibit unique gene expression profiles and functional characteristics associated with CRC immune escape. The cell interaction network, formed by these subtypes and other cells within the TME, facilitates CRC immune escape by reshaping the immunosuppressive microenvironment. Additionally, CFLAR+B_cells-C3, CALR+CD8+T_cells-C2, and TAP1+Mac-C2 may serve as potential biomarkers for predicting responses to immunotherapy in CRC patients. In contrast, HEXIM1+CAF-C1 may act as an independent risk factor for poor prognosis in CRC. Conclusions Our findings enhance understanding of immune escape mechanisms in CRC, show how novel subtypes affect prognosis, and offer insights for new diagnostic and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08981a5b19bb84f7880bb0a8f963b09e3fc11c47" target='_blank'>
              Single-cell analysis reveals the prognostic role of immune escape in the colorectal cancer microenvironment
              </a>
            </td>
          <td>
            Qixin Gan, Xuan Xu, Haifen Liu, Yuejun Li
          </td>
          <td>2026-01-01</td>
          <td>Translational Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5342473ebabd904e74c8bc0079d51547ba07dfc1" target='_blank'>
              Spatial transcriptomics reveals a key role of fibroblast-like vascular smooth muscle cells in human atherosclerotic cell crosstalk and stability.
              </a>
            </td>
          <td>
            Isabel Gonçalves, Mengyu Pan, Pratibha Singh, Wenqi Wang, Jing Zhao, L. Dib, L. Sundius, A. Persson, C. Gialeli, P. Fountas, M. Nitulescu, Jan Nilsson, Stephen Malin, Claudia Monaco, Helle F Jørgensen, Jiangming Sun, A. Edsfeldt
          </td>
          <td>2026-02-13</td>
          <td>European heart journal</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Prostate cancer depends on androgen receptor (AR) signaling for growth, which is why androgen deprivation (castration) therapy is effective at early stages. However, many tumors eventually progress to a lethal form known as castration-resistant prostate cancer (CRPC). A subset of CRPC tumors bypass dependency on AR signaling by acquiring lineage plasticity, where prostate cancer cells transdifferentiate into alternate cellular states through epigenetic reprogramming. Neuroendocrine (NE) prostate cancer represents one well-known lineage plasticity phenotype. Nevertheless, most AR-independent tumors do not exhibit NE features and are defined as AR-negative/NE-negative or “double-negative prostate cancer” (DNPC). In a collaboration with Dr. Ekta Khurana’s computational genomics lab at Weill Cornell Medicine, we recently classified CRPC into four epigenetic subtypes, including the well-established 1) AR and 2) NE, as well as the novel DNPC subgroups 3) WNT and 4) stem cell-like (SCL) (PMID: 35617398). We focused on the SCL subtype as it is the second most common group in CRPC patients and lacks therapeutic targets. Using functional genomic approaches, we found that YAP/TAZ/TEAD cooperates with FOSL1 to drive the SCL lineage and growth of SCL models. We therefore hypothesize that the heightened dependency on the YAP/TAZ/TEAD/FOSL1 transcriptional program represents a therapeutic vulnerability in CRPC-SCL. To test this, we exposed CRPC models to TEAD inhibitors and found robust growth suppression in SCL cells compared to non-SCL cells in vitro. To evaluate whether the TEAD inhibitors are on-target, we performed transcriptomic profiling in SCL models and observed downregulation of YAP/TAZ gene signature as well as FOSL1 expression, which phenocopies the effects of YAP/TAZ double knockdown. To define the cistromes of these factors upon TEAD inhibition, we performed ChIP-seq and observed reduced co-occupancy at consensus sites, suggesting the disruption of the YAP/TAZ/TEAD/FOSL1 transcriptional circuit by the small molecule compound. To determine whether these phenotypes are recapitulated in vivo, we will treat mice harboring CRPC-SCL xenografts with TEAD inhibitors to assess growth response and evaluate epigenetic and transcriptional response using single-nucleus Multiome (ATAC+RNA). These studies will establish whether small molecule inhibition of TEAD is a promising strategy for the treatment of CRPC-SCL and allow high-resolution analysis of cell state transitions, with a focus on loss of SCL-specific signatures and potential emergence of AR/NE programs as adaptive resistance mechanisms.



 Chen Khuan Wong, Dan Li, Hongsu Wang, Marjorie Roskes, Weiling Li, Shipra Shukla, Dana Schoeps, Ekta Khurana, Yu Chen. Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A074.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912fdd784be0fd097e91b40d9e4c6933dd2c1b14" target='_blank'>
              Abstract A074: Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer
              </a>
            </td>
          <td>
            C. K. Wong, Dan Li, Hongsu Wang, M. Roskes, Weiling Li, Shipra Shukla, Dana M. Schoeps, E. Khurana, Yu Chen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Cancer cells are heterogeneous, each harboring distinct molecular aberrations and being dependent on different genes for their survival and proliferation. While targeted therapies based on driver DNA mutations have shown success, many tumors lack druggable mutations, limiting treatment options. We hypothesize that new precision oncology targets may be identified through "expression-driven dependency," where cancer cells with high expression of specific genes are more vulnerable to the knockout of those same genes.


RESULTS
We developed BEACON, a Bayesian approach to identify expression-driven dependency targets by analyzing global transcriptomic and proteomic profiles alongside genetic dependency data from cancer cell lines across 17 tissue lineages. BEACON successfully identified known druggable genes, including BCL2, ERBB2, EGFR, ESR1, and MYC, while revealing novel targets confirmed by both mRNA and protein-expression driven dependency. The identified genes showed a 3.8-fold enrichment for approved drug targets and a 7 to 10-fold enrichment for druggable oncology targets. Experimental validation demonstrated that depletion of GRHL2, TP63, and PAX5 effectively reduced tumor cell growth and survival in their dependent cells.


CONCLUSIONS
Our approach provides a systematic method to identify precision oncology targets based on expression-driven dependency patterns. By integrating multi-omics data with genetic dependency screens, we've created a comprehensive catalog of potential therapeutic targets that may expand treatment options for cancer patients lacking druggable mutations. This resource offers new opportunities for precision oncology target discovery beyond mutation-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d9df8d8c54b9f6f78829d05c5029fee1f72554" target='_blank'>
              Expression-Driven Genetic Dependency Reveals Targets for Precision Oncology.
              </a>
            </td>
          <td>
            Abdulkadir Elmas, Hillary M. Layden, Jacob D Ellis, Luke N. Bartlett, Xian Zhao, Reika Kawabata-Iwakawa, Zishan Wang, Hideru Obinata, Scott W. Hiebert, Kuan-lin Huang
          </td>
          <td>2026-01-29</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Predicting spatial gene expression from H&E histology offers a scalable and clinically accessible alternative to sequencing, but realizing clinical impact requires models that generalize across cancer types and capture biologically coherent signals. Prior work is often limited to per-cancer settings and variance-based evaluation, leaving functional relevance underexplored. We introduce HistoPrism, an efficient transformer-based architecture for pan-cancer prediction of gene expression from histology. To evaluate biological meaning, we introduce a pathway-level benchmark, shifting assessment from isolated gene-level variance to coherent functional pathways. HistoPrism not only surpasses prior state-of-the-art models on highly variable genes , but also more importantly, achieves substantial gains on pathway-level prediction, demonstrating its ability to recover biologically coherent transcriptomic patterns. With strong pan-cancer generalization and improved efficiency, HistoPrism establishes a new standard for clinically relevant transcriptomic modeling from routinely available histology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c31313036e644587313c31ba04ae584ee9cd4119" target='_blank'>
              HistoPrism: Unlocking Functional Pathway Analysis from Pan-Cancer Histology via Gene Expression Prediction
              </a>
            </td>
          <td>
            Susu Hu, Qinghe Zeng, Nithya Bhasker, J. N. Kather, Stefanie Speidel
          </td>
          <td>2026-01-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9afdc571150d3d83aad404d1fcf565224b19e64" target='_blank'>
              Integrated meta-analysis of public human pancreatic single-cell transcriptomes reveals distinct beta-cell state trajectories across aging and diabetes
              </a>
            </td>
          <td>
            Elhadi Iich
          </td>
          <td>2026-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The nervous system plays a dynamic and active role in cancer progression, influencing tumor growth, invasion, metastasis, and response to therapy. Neural components within the tumor microenvironment communicate with cancer cells through neurotransmitters, neurotrophic factors, and electrical signaling, whereas tumors reciprocally remodel surrounding neural circuits. This Scientific Reports Collection on Cancer Neuroscience highlights recent advances uncovering these complex interactions across molecular, cellular, and systems levels. The featured studies investigate mitochondrial and metabolic modulation in glioblastoma, neural regulatory pathways driving tumor invasiveness, and functional neuroimaging correlates of cognitive outcomes in glioma survivors. Together, these publications illustrate how integrating neuroscience with oncology offers new mechanistic insight and therapeutic opportunities. By illuminating the neural dimension of tumor biology, this Collection aims to inspire interdisciplinary approaches to decipher and therapeutically target the bidirectional crosstalk between the nervous system and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4c1a8971cc563fa201dc2d52b5da68d0c8726e1" target='_blank'>
              Cancer neuroscience: illuminating the neural dimension of tumor biology
              </a>
            </td>
          <td>
            Saurabh Agarwal
          </td>
          <td>2025-12-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720c9004235c7c3aef4fbef8c74bcb96b3903c12" target='_blank'>
              Multiomics analysis reveals that senescent CXCL16+ macrophages promote lung adenocarcinoma progression through TGF-β signalling.
              </a>
            </td>
          <td>
            Zhen-Hu Zhang, , Wei Li, Han Xiao, Dong Wang, Meng-Hang Yang
          </td>
          <td>2026-02-02</td>
          <td>Journal of translational medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The survival mechanisms of circulating tumor cells (CTCs) remain poorly understood while these rare cell populations transit through the blood stream and colonize distant organs including the brain. Using single-cell RNA-seq of microfluidically-enriched CTCs and patient-matched brain metastatic tumor cells of lung adenocarcinoma (LUAD), we revealed CTC-selective upregulation of core gene signatures associated with ferroptosis. The malignancy of CTCs was confirmed through comparative genomic instability and copy number variation (CNV) analyses of paired CTCs and metastatic tumor cells using both transcriptomic and whole-exome sequencing. Among the transcription factors elevated in CTCs, Chromobox 3 (CBX3) was the top hit, which was tightly correlated with GPX4 expression. Functionally, CBX3 co-operated with EP300 to protect CTCs from ferroptosis by upregulating GPX4 expression. Genetic depletion of CBX3 triggered ferroptosis and substantially reduced tumor cell survival and invasiveness in two independent cancer cell models. Conversely, CBX3 overexpression was sufficient to promote tumor growth, migration, and invasion in vitro, and metastatic progression in vivo. Clinically, CBX3 expression was significantly correlated with TNM stage and was predictive of both progression-free interval and overall survival in LUAD. In a prospective cohort of LUAD and melanoma patients, CTCs co-expressing CBX3 and GPX4 were selectively elevated in the blood samples of metastatic cases compared to the non-metastatic group, highlighting the clinical association between ferroptosis-resistant CTCs and metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7fe0bbf1d19b9968391a14825a2e7d78f15cc38" target='_blank'>
              CBX3 confers ferroptosis resistance during blood-borne metastasis
              </a>
            </td>
          <td>
            Chun Wu, Xuefei Liu, Boxi Zhao, Mao Zhao, Binyu Zhang, Guanyin Huang, Yixin Cheng, Shuqian Zheng, Jianyang Hu, Ling Guo, Wei-Jie Guo, Jun Tan, Xin Hong
          </td>
          <td>2026-01-15</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad062077323547175d6b721e78e8771a9325ce15" target='_blank'>
              Phenotypic Plasticity and Competition Shape Therapy Sequencing in HER2+/HER2− Breast Cancer: A Mathematical Framework
              </a>
            </td>
          <td>
            Aleksandra Gavrilova, Trachette L. Jackson, Nizhum Rahman
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="While cancer characteristics can vary significantly across types, methods that distinguish malignant cells from normal ones hold promise by targeting shared cellular anomalies. Among these, morphological differences play a key role in driving the aggressive behavior and altered function typical of cancer cells. Detecting and analyzing such cells within complex, densely packed tissue environments requires advanced imaging techniques. Polarization-resolved fluorescence microscopy offers rich insights into cellular composition, molecular binding affinities, and structural organization, particularly in revealing biomolecular order and subcellular polarity loss. In this work, we study polarization-resolved two-photon excitation fluorescence tissue imaging microscopy in vitro to investigate ordered versus disordered chromatin organization within cell nuclei. We employ an innovative phasor map analysis to facilitate quick interpretation, using colorectal cancer identification and liquid crystal as a case study and baseline, respectively. Our method aims to identify cancer within tissue by adding polarimetric contrast to fluorescence due to the anisotropic feature of fluorescent molecular probes. Accordingly, the proposed phasor map provides a graphically transformed representation of polarization-based fluorescence imaging for histopathological tissue identification on a pixel-wise basis, facilitating comprehensive classification of diverse tissue samples. This study presents initial steps toward showing the potential for cancer identification and lays a foundation for future diagnostic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be1896fecc86522b8e023da943f4a2f4142d0e9d" target='_blank'>
              Cancer cell identification in tissue imaging via phasor map analysis in polarization-resolved two-photon excitation fluorescence microscopy
              </a>
            </td>
          <td>
            Ali Mohebi, Lasse Bo Mortensen, Gavrielle R. Untracht, Madhu Veettikazhy, A. Le Gratiet, P. Andersen
          </td>
          <td>2025-12-23</td>
          <td>Biomedical Optics Express</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Motivation Single-cell omics analysis has unveiled the heterogeneity of various cell types within tumors. However, no methodology currently reveals how this heterogeneity influences cancer patient survival at single-cell resolution. Here, we introduce scSurv, combining a Cox proportional hazards model with a deep generative model of single-cell transcriptome, to estimate individual cellular contributions to clinical outcomes. Results The accuracy of scSurv was validated using both simulated and real datasets. This method identifies cells associated with favorable or adverse prognoses and extracts genes correlated with their contribution levels. In melanoma, scSurv reproduces known prognostic macrophage classifications and facilitates hazard mapping through spatial transcriptomics in renal cell carcinoma. We also identified genes consistently associated with prognosis across multiple cancers and demonstrated the applicability of this method to infectious diseases. scSurv is a novel framework for quantifying the heterogeneity of individual cellular effects on clinical outcomes. Availability The implementation of scSurv is available on GitHub (https://github.com/3254c/scSurv) and Zenodo (https://doi.org/10.5281/zenodo.17793054).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98bd46ab28d9f09fcd249606230d4459ff93bbef" target='_blank'>
              scSurv: a deep generative model for single-cell survival analysis
              </a>
            </td>
          <td>
            Chikara Mizukoshi, Yasuhiro Kojima, Shuto Hayashi, Ko Abe, Daisuke Kasugai, Teppei Shimamura
          </td>
          <td>2025-12-22</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Colon adenocarcinoma (COAD) is the most common type of colon cancer, posing a significant threat to public health. In the tumor microenvironment (TME), T cells differentiate into terminally exhausted T cells (TEX), but the relationship between TEX and COAD has not been fully elucidated. To identify TEX-related signatures, we integrated transcriptomic data from TCGA and GEO databases (GSE103479, GSE17536). A prognostic model was constructed using GSVA, univariate Cox, LASSO, and random forest algorithms. The tumor immune microenvironment was characterized using CIBERSORTx and GSEA. The functional role of FAT4 was validated in vitro using FAT4-knockdown COAD cell lines assessed by flow cytometry and RT-qPCR. We developed a prognostic signature based on five TEX-related genes (IL21R, FCRL3, TIFAB, TNFSF14, SLAMF1). Patients in the high-risk group showed significantly poorer overall survival and distinct immune cell infiltration patterns, characterized by decreased CD8 + T cells and M1 macrophages. At single-cell resolution, CD8 + TEX cells exhibited high expression of immune checkpoints like LAG3. Furthermore, in vitro experiments demonstrated that knockdown of FAT4, a frequently mutated gene in COAD, promoted apoptosis and induced G0/G1 cell cycle arrest in COAD cells. We proposed a non-invasive prediction method based on TEX-related genes, which effectively predicts survival outcomes and therapeutic responses in COAD patients. Additionally, FAT4 was found to regulate proliferative and apoptotic phenotypes in COAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83927c69818d8de262c593f498367f67a4898a30" target='_blank'>
              T-cell exhaustion indicator characterizes the tumor microenvironment landscape and predicts colon adenocarcinoma prognosis via integrating single-cell RNA-seq and bulk RNA-sequencing
              </a>
            </td>
          <td>
            Haifeng Sun, Ning Hu, Shaowei Ma, Yu Zheng, Ren Niu, Wenya Zhu, Shaofan Qiu
          </td>
          <td>2026-01-14</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Single-cell genomics has emerged as a transformative approach to unravel the complexity of somatic variation in specific cells within the human body. This field has profound implications for understanding the role of somatic mutations in aging, cardiovascular disease, and tissue-specific pathologies. By focusing on circulating cells and cardiac resident cells, including cardiomyocytes, within the heart, this review explores how single-cell genomics provides insights into cellular heterogeneity and clonal evolution. We discuss the implications of somatic variation for cardiovascular health, highlight technological innovation, and future directions in this rapidly evolving field. This review sheds light on national initiatives, such as the National Institutes of Health Somatic Mosaicism Across Human Tissues Network (SMaHT) and the European Somatic Mutations in Vascular-Wall Function and Age-Associated Disease (EU SOMATICART) project, which aim to generate reference atlases of somatic mutations across human tissues. It also explores challenges and future directions in leveraging single-cell approaches to improve diagnostics and therapeutics in cardiovascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ec9fb81424f8866ab25b7b24005bdae8410be68" target='_blank'>
              Single-Cell Genomics and Somatic Variation in Circulating and Cardiac Resident Cells.
              </a>
            </td>
          <td>
            N. Hilal, Maniteja Arava, S. Choudhury
          </td>
          <td>2026-01-02</td>
          <td>Circulation research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cell-in-cell (CIC) structures, characterized by one cell engulfing another, play a paradoxical role in cancer metastasis. This review synthesizes their dual nature: while CIC can suppress metastasis by eliminating aberrant cells or activating immune responses, it also drives tumor progression through clonal evolution, metabolic adaptation, and genomic instability. The clinical relevance of CIC varies across cancer types, correlating with both favorable and poor prognoses depending on disease stage and microenvironmental context. Emerging therapeutic strategies aim to harness protective CIC mechanisms or inhibit pro-metastatic pathways, though challenges such as biomarker validation and dynamic tracking persist. Future research integrating advanced technologies will elucidate the spatiotemporal complexity of CIC, enabling tailored interventions to disrupt metastatic cascades. By bridging fundamental biology and clinical translation, CIC represents a promising yet intricate frontier in the battle against metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e6d8a02bb8fcc15958d3d8c8939ab24272dbaf6" target='_blank'>
              Cell-in-cell structures and cancer metastasis.
              </a>
            </td>
          <td>
            Yuchen Cao, Jianing Du, Jianguo Lai, Chunjun Liu
          </td>
          <td>2026-01-23</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acral melanoma (AM), the predominant melanoma subtype in Asia, responds poorly to immune checkpoint inhibitors, representing a critical unmet medical need. The efficacy of tumor-infiltrating lymphocyte (TIL) therapy in this population is unknown. An Investigator-Initiated Trial evaluates autologous TIL therapy (LM-103) in four Chinese patients with advanced AM, achieving a 75% disease control rate (DCR) and a 25% objective response rate (ORR), including one durable complete response. To define the determinants of response, we performed integrated single-cell RNA and T-cell receptor sequencing on infused TIL products, tumors, and longitudinal peripheral blood. Responders' infused products were significantly enriched for T follicular helper (Tfh) and intermediate exhausted (TEX_int) CD8⁺ T cells, which mediated robust cell-cell signaling networks (e.g., CD40, FASLG). In contrast, the non-responder's product was dominated by terminally exhausted (TEX_term) cells. Clonal tracking revealed that these Tfh and TEX_int subsets possessed higher clonality, and in the complete responder, a dominant clone originating from the TEX_int population persisted systemically by differentiating into a progenitor-like (TEX_prog) state. These findings demonstrate that TIL therapy is clinically active in AM and that durable response is mechanistically linked to the infusion and persistence of Tfh and TEX_int subsets, defining a key cellular and clonal architecture for therapeutic success.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4d4d63db111617fa12c56b70ba0807cf90683e2" target='_blank'>
              Single-Cell Dissection of Tumor-Infiltrating Lymphocytes Reveals Cellular Architecture Predictive of Therapeutic Efficacy in Acral Melanoma.
              </a>
            </td>
          <td>
            Chao Zhang, Wanyi Xiao, Hong-xiu Shen, Fenge Li, Ting Li, Weihong Zhang, Haotian Liu, Ziwei Gao, Hongyu Wang, Xiubao Ren, Kexin Chen, Xiangchun Li, Jilong Yang
          </td>
          <td>2026-01-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The cardiovascular system, composed of the heart and vasculature, is essential for blood circulation, nutrient exchange, and waste removal. In the past, our understanding of cardiovascular development and function has largely been shaped by bulk tissue analyses, which obscures cellular heterogeneity. The emergence of single-cell omics has transformed the field by enabling unbiased transcriptional profiling of individual cells, revealing the diversity of stem cells and progenitor cells driving embryogenesis, resulting in the various mature cardiovascular cell types in the adult heart and vasculature. This technology has provided unprecedented insights into the molecular mechanisms governing cardiovascular development and function by identifying novel cell subpopulations, characterizing their unique properties, and tracing their temporal evolution through advanced analytical approaches. In this review, we discuss how single-cell omics has reshaped our understanding of cardiovascular developmental biology, highlight key analytical tools and emerging approaches, examine preclinical models that have facilitated these discoveries, and explore how these technologies have defined the cellular landscape of the heart and vasculature. We conclude by looking ahead to emerging technologies such as spatial transcriptomics and clonal barcoding for lineage tracing, as well as new strategies in addressing the gender gap in cardiovascular research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f10251a7ea893f7554f7a7ea1632ee30b37891ee" target='_blank'>
              Unraveling Cardiovascular Development and Function: Insights From Single-Cell Omics.
              </a>
            </td>
          <td>
            Rebecca L Harper, P. Lelliott, S. B. Bender, Alexander R. Pinto
          </td>
          <td>2026-01-02</td>
          <td>Circulation research</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [8, 19],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>