<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-24 09:28:24 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d9a8ffa548a2f4ea069cb9bc5e6205adbb07d1" target='_blank'>
              Tumor cell villages define the co-dependency of tumor and microenvironment in liver cancer
              </a>
            </td>
          <td>
            Meng Liu, M. Hernandez, Darko Castven, Hsin-Pei Lee, Wenqi Wu, Limin Wang, M. Forgues, Jonathan M. Hernandez, Jens U. Marquardt, Lichun Ma
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Immune checkpoint inhibition (ICI) has fundamentally changed cancer treatment. However, only a minority of patients with metastatic triple negative breast cancer (TNBC) benefit from ICI, and the determinants of response remain largely unknown. To better understand the factors influencing patient outcome, we assembled a longitudinal cohort with tissue from multiple timepoints, including primary tumor, pre-treatment metastatic tumor, and on-treatment metastatic tumor from 117 patients treated with ICI (nivolumab) in the phase II TONIC trial. We used highly multiplexed imaging to quantify the subcellular localization of 37 proteins in each tumor. To extract meaningful information from the imaging data, we developed SpaceCat, a computational pipeline that quantifies features from imaging data such as cell density, cell diversity, spatial structure, and functional marker expression. We applied SpaceCat to 678 images from 294 tumors, generating more than 800 distinct features per tumor. Spatial features were more predictive of patient outcome, including features like the degree of mixing between cancer and immune cells, the diversity of the neighboring immune cells surrounding cancer cells, and the degree of T cell infiltration at the tumor border. Non-spatial features, including the ratio between T cell subsets and cancer cells and PD-L1 levels on myeloid cells, were also associated with patient outcome. Surprisingly, we did not identify robust predictors of response in the primary tumors. In contrast, the metastatic tumors had numerous features which predicted response. Some of these features, such as the cellular diversity at the tumor border, were shared across timepoints, but many of the features, such as T cell infiltration at the tumor border, were predictive of response at only a single timepoint. We trained multivariate models on all of the features in the dataset, finding that we could accurately predict patient outcome from the pre-treatment metastatic tumors, with improved performance using the on-treatment tumors. We validated our findings in matched bulk RNA-seq data, finding the most informative features from the on-treatment samples. Our study highlights the importance of profiling sequential tumor biopsies to understand the evolution of the tumor microenvironment, elucidating the temporal and spatial dynamics underlying patient responses and underscoring the need for further research on the prognostic role of metastatic tissue and its utility in stratifying patients for ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1bdaa11baefcf7560def6afb49fb7f0020d09d" target='_blank'>
              Temporal and spatial composition of the tumor microenvironment predicts response to immune checkpoint inhibition
              </a>
            </td>
          <td>
            N. Greenwald, I. Nederlof, Cameron Sowers, Daisy Yi Ding, Seongyeol Park, Alex Kong, Kathleen E. Houlahan, Sricharan Reddy Varra, Manon de Graaf, Veerle Geurts, Candace C. Liu, Jolene S Ranek, L. Voorwerk, M. de Maaker, Adam Kagel, E. Mccaffrey, Aziz Khan, C. Yeh, C. Fullaway, Zumana Khair, Yunhao Bai, Hadeesha Piyadasa, Tyler Risom, A. Delmastro, Felix J. Hartmann, Lise Mangiante, Cristina Sotomayor-Vivas, Ton N. Schumacher, Zhicheng Ma, M. Bosse, Marc J van de Vijver, R. Tibshirani, Hugo M. Horlings, Christina Curtis, Marleen Kok, Michael Angelo
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>157</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer deaths; nonetheless, how tumor cells adapt to vastly different organ contexts is largely unknown. To investigate this question, we generated a transcriptomic atlas of primary tumor and diverse metastatic samples from a patient with pancreatic ductal adenocarcinoma who underwent rapid autopsy. Unsupervised archetype analysis identified both shared and site-specific gene programs, including lipid metabolism and gastrointestinal programs prevalent in peritoneum and stomach wall lesions, respectively. We developed a probabilistic approach for inferring clonal phylogeny from single-cell and matched whole-exome data. Distantly related genetic clones in the peritoneum express the lipid metabolism program, likely due to signaling by the adipocyte-rich peritoneum environment, and cells in most clones express multiple programs, suggesting that transcriptomic plasticity is a prevalent feature of metastatic cells. These deeply annotated analyses using a patient-centric platform provide a model for investigating metastatic mechanisms and plasticity in advanced cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b77f53eedac93ceda8af4b898f6071abded0e4" target='_blank'>
              Transcriptomic plasticity is a hallmark of metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Alejandro Jiménez-Sánchez, S. Persad, Akimasa Hayashi, Shigeaki Umeda, Roshan Sharma, Yubin Xie, A. Mehta, Wungki Park, I. Masilionis, T. Chu, Feiyang Zhu, Jungeui Hong, R. Chaligné, Eileen M. O’Reilly, L. Mazutis, T. Nawy, I. Pe’er, Christine A. Iacobuzio-Donahue, D. Pe’er
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2886703d3c5dcc191767cc197fd5d1a7d169072" target='_blank'>
              A spatially resolved single cell proteomic atlas of Small Bowel Adenocarcinoma
              </a>
            </td>
          <td>
            Z. Dereli, B. Bozorgui, M. Sanchez, Nicholas Hornstein, Guillaume Thibault, Huamin Wang, Gordon B. Mills, John N. Weinstein, M. Overman, Anil Korkut
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3061625679fbc2f861a2180efc0a648894c17487" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment
              </a>
            </td>
          <td>
            Shoaib Ajaib, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F. Stead
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed05a90f6b1d7a182ae84bb7e554725c91c44235" target='_blank'>
              Quantifying and interpreting biologically meaningful spatial signatures within tumor microenvironments
              </a>
            </td>
          <td>
            Si Jing, He-Qi Wang, Ping Lin, Jiao Yuan, Zhixuan Tang, Hong Li
          </td>
          <td>2025-03-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98801c9770ab85a801e0ca53d0db3280fc312fc" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="To date, few studies have detailed the tumor microenvironment (TME) of Ewing sarcoma (EwS). The TME has a vital role in cancer survival and progression with implications in drug resistance and immune escape. By performing spatially resolved transcriptomic analysis of primary treatment-naïve EwS samples, we discovered greater stromal enrichment in localized EwS tumors compared to metastatic EwS tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumorigenic role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling, suggesting a potential immune-evasive mechanism and immunotherapy target. Our findings provide insight into tumor cell/stromal cell interactions in EwS and serve as a valuable resource for further investigations in the tumor immune microenvironment of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245c7e47704099e340bfbb4047050883a6a92e6d" target='_blank'>
              Tumor-associated stroma shapes the spatial tumor immune microenvironment of primary Ewing sarcomas
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/619b0b12a0a0a0f0529c78aba3cfb6008e7a474e" target='_blank'>
              Sparse deconvolution of cell type medleys in spatial transcriptomics
              </a>
            </td>
          <td>
            Nuray Sogunmez Erdogan, Deniz Eroglu
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc63f157070e0fd70ef6717ad423137ac275d75" target='_blank'>
              Stochastically Emergent Tumors offer in vivo whole genome interrogation of cancer evolution from non-malignant precursors
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel B. Hoelscher, V. Luga, Arun A. Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, C.K. Cornelia Ding, Bradley A. Stohr, Peng Jin, Tejasveeta V. Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R. Karthaus, C.L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b08309318ca287b808eae53da393baf17b1d09b8" target='_blank'>
              Multi-omic landscape of human gliomas from diagnosis to treatment and recurrence
              </a>
            </td>
          <td>
            Hadeesha Piyadasa, Benjamin Oberlton, Mikaela Ribi, Jolene S Ranek, Inna Averbukh, K. Leow, M. Amouzgar, Candace C. Liu, Noah F. Greenwald, E. Mccaffrey, Rashmi Kumar, Selena Ferrian, Albert G. Tsai, Ferda Filiz, C. Fullaway, M. Bosse, Sricharan Reddy Varra, Alex Kong, Cameron Sowers, M. Gephart, Pablo Nuñez-Perez, EnJun Yang, Michael Travers, Michael Schachter, Samantha Liang, Maria R. Santi, Samantha Bucktrout, P. Gherardini, Kristina Cole, Michael Barish, Christine E. Brown, Derek A. Oldridge, Richard R. Drake, Joanna J. Phillips, Hideho Okada, Robert Prins, Sean C. Bendall, Michael Angelo
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d655b9c3b36c5c88b85d107ee0f6e182ac411fa5" target='_blank'>
              A quantitative spatial cell-cell colocalizations framework enabling comparisons between in vitro assembloids and pathological specimens
              </a>
            </td>
          <td>
            Gina Bouchard, Weiruo Zhang, Ilayda Ilerten, Irene Li, Asmita Bhattacharya, Yuanyuan Li, W. Trope, Joseph B Shrager, Calvin J. Kuo, Michael G Ozawa, A. Giaccia, Lu Tian, Sylvia K Plevritis
          </td>
          <td>2025-02-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC), accounting for 90% of pancreatic neoplasms, is characterized by its poor prognosis, with a 5-year survival rate of only 12%. Most patients are diagnosed with metastatic or locally advanced disease, leaving only 15% eligible for curative resection. PDAC exhibits resistance to chemotherapy, targeted therapies, and immunotherapy, largely due to its highly heterogeneous tumor microenvironment (TME). In this study, we performed an integrative analysis of publicly available scRNA, spatial transcriptomics, and bulk RNA sequencing datasets to investigate the influence of TME composition and tumor architecture on PDAC progression, treatment response, and clinical outcomes. We identified TME subtypes with distinct cellular compositions, functional signatures, and immunomodulatory cell-cell interactions. Spatially distinct cellular niches and gene modules revealed heterogeneity across primary tumors and metastatic lesions. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications. These findings underscore the importance of integrating multi-omics approaches to unravel the complexity of the PDAC TME and highlight its potential to inform therapeutic strategies and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768379a4460d13ebf34f45a5af50c7d930ab8e2" target='_blank'>
              Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel F. Pozo de Mattos P., Marvin Paulo Lins, Julia Fontoura, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In this study, we developed an integrated single cell transcriptomic (scRNAseq) atlas of human breast cancer (BC), the largest resource of its kind, totaling > 600,000 cells across 138 patients. Rigorous integration and annotation of publicly available scRNAseq data enabled a highly resolved characterization of epithelial, immune, and stromal heterogeneity within the tumor microenvironment (TME). Within the immune compartment we were able to characterize heterogeneity of CD4, CD8 T cells and macrophage subpopulations. Within the stromal compartment, subpopulations of endothelial cells (ECs) and cancer associated fibroblasts (CAFs) were resolved. Within the cancer epithelial compartment, we characterized the functional heterogeneity of cells across the axes of stemness, epithelial-mesenchymal plasticity, and canonical cancer pathways. Across all subpopulations observed in the TME, we performed a multi-resolution survival analysis to identify epithelial cell states and immune cell types which conferred a survival advantage in both The Cancer Genome Atlas (TCGA) and METABRIC. We also identified robust associations between TME composition and clinical phenotypes such as tumor subtype and grade that were not discernible when the analysis was limited to individual datasets, highlighting the need for atlas-based analyses. This atlas represents a valuable resource for further high-resolution analyses of TME heterogeneity within BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7db0b505659425c4bfdef92aba55ca31e10f8c9" target='_blank'>
              A highly resolved integrated transcriptomic atlas of human breast cancers
              </a>
            </td>
          <td>
            Andrew Chen, Lina Kroehling, Christina S. Ennis, Gerald V. Denis, S. Monti
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88360a99c9dba219540b793cdb094157ffa2ec6a" target='_blank'>
              Lineage Tracing Reveals Clone-Specific Responses to Doxorubicin in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Daylin Morgan, Andrea L. Gardner, Amy Brock
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2ee7e019ae5fcae794fe1bb25f818cd218fc98" target='_blank'>
              Spatial profiling identifies regionally distinct microenvironments and targetable immunosuppressive mechanisms in pediatric osteosarcoma pulmonary metastases
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice Pereira, Douglass W Morris-II, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael Kelly, Amy K LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74fb81e29120e8309bf4f3e49e31706525768da7" target='_blank'>
              Integrating multi-modal transcriptomics identifies cellular subtypes with distinct roles in PDAC progression
              </a>
            </td>
          <td>
            Jun Wu, Tenghui Dai, Ziyue Li, Meng Pan, Wei Zhang, Hao Chen, Guansheng Zheng, Li Qiao, Qizhou Lian, Yang Liu, Jierong Chen
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="A key challenge in cancer research is to identify the secreted factors that contribute to tumor cell survival. Nowhere is this more evident than in Hodgkin lymphoma, where malignant Hodgkin Reed Sternberg (HRS) cells comprise only 1-5% of the tumor mass, the remainder being infiltrating immune cells that presumably are required for the survival of the HRS cells. Until now, there has been no way to characterize the complex Hodgkin lymphoma tumor microenvironment at genome scale. Here, we performed genome-wide transcriptional profiling with spatial and single-cell resolution. We show that the neighborhood surrounding HRS cells forms a distinct niche involving 31 immune and stromal cell types and is enriched in CD4+ T cells, myeloid and follicular dendritic cells, while being depleted of plasma cells. Moreover, we used machine learning to nominate ligand-receptor pairs enriched in the HRS cell niche. Specifically, we identified IL13 as a candidate survival factor. In support of this hypothesis, recombinant IL13 augmented the proliferation of HRS cells in vitro. In addition, genome-wide CRISPR/Cas9 loss-of-function studies across more than 1,000 human cancer cell lines showed that IL4R and IL13RA1, the heterodimeric partners that constitute the IL13 receptor, were uniquely required for the survival of HRS cells. Moreover, monoclonal antibodies targeting either IL4R or IL13R phenocopied the genetic loss of function studies. IL13-targeting antibodies are already FDA-approved for atopic dermatitis, suggesting that clinical trials testing such agents should be explored in patients with Hodgkin lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0453e5a36d23742e3f6af668adec8ef2f3f547e8" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, Daniel Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail A Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson Weir, Orr Ashenberg, Geraldine Pinkus, Scott J Rodig, Caroline Uhler, Evan Z. Macosko, Margaret M. Shipp, A. Louissaint, Fei Chen, Todd R. Golub
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Cellular organization is central to tissue function and homeostasis, influencing development, disease progression, and therapeutic outcomes. The emergence of spatial omics technologies, including spatial transcriptomics and proteomics, has enabled the integration of molecular and histological features within tissues. Analyzing these multimodal data presents unique challenges, including variable resolutions, imperfect tissue alignment, and limited or variable spatial coverage. To address these issues, we introduce CORAL, a probabilistic deep generative model that leverages graph attention mechanisms to learn expressive, integrated representations of multimodal spatial omics data. CORAL deconvolves low-resolution spatial data into high-resolution single-cell profiles and detects functional spatial domains. It also characterizes cell-cell interactions and elucidates disease-relevant spatial features. Validated on synthetic data and experimental datasets, including Stero-CITE-seq data from mouse thymus, and paired CODEX and Visium data from hepatocellular carcinoma, CORAL demonstrates robustness and versatility. In hepatocellular carcinoma, CORAL uncovered key immune cell subsets that drive the failure of response to immunotherapy, highlighting its potential to advance spatial single-cell analyses and accelerate translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/295042adfe589c5c9974e6c8e7ec5b6246fdd726" target='_blank'>
              Learning single-cell spatial context through integrated spatial multiomics with CORAL
              </a>
            </td>
          <td>
            Siyu He, Matthew Bieniosek, Dongyuan Song, Jingtian Zhou, Benjamin Chidester, Zhenqin Wu, Joseph Boen, Padmanee Sharma, Alexandro E. Trevino, James Zou
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/190acfc8ea743ab5360d22a148d803b01bb31afc" target='_blank'>
              A pathogenic subpopulation of human glioma associated macrophages linked to glioma progression
              </a>
            </td>
          <td>
            Kenny Kwok Hei Yu, Z. Abou-Mrad, Kristof Törkenczy, Isabell Schulze, Jennifer Gantchev, Gerard Baquer, Kelsey Hopland, Evan D. Bander, Umberto Tosi, Cameron Brennan, Nelson S. Moss, Pierre-Jacques Hamard, R. Koche, Caleb Lareau, N. Agar, T. Merghoub, Viviane Tabar
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Abstract Background The tumor microenvironment (TME) is a complex and dynamic ecosystem that is known to influence responses to immunotherapy. We leveraged single-cell spatial transcriptomics to systematically dissect the intricate complexity of the TME, in particular the cellular heterogeneity and spatial interactions. Their collective impact on immunotherapy efficacy was studied in the context of a homogeneous group of patients with vulvar high-grade squamous intraepithelial lesions (vHSIL) treated with an immunotherapeutic tumor-specific peptide vaccine. Methods We performed single-cell spatial transcriptomics on 20 pretreatment vHSIL lesions, stratified by clinical response to immunotherapeutic vaccination into complete responders (CR), partial responders (PR) and non-responders (NR). Using a 1,000-gene panel, we mapped over 274,000 single cells in situ, identifying 18 cell clusters and 99 distinct non-epithelial cell states. Findings were validated against public single-cell transcriptomic data sets to assess their broader relevance across tumor types. Results Profound heterogeneity within the TME was detected across the response groups. CR lesions exhibited a higher ratio of immune-supportive to immune-suppressive cells—a pattern mirrored in other solid tumors following neoadjuvant checkpoint blockade. Key immune populations enriched in CRs included CD4+CD161+ effector T cells and chemotactic CD4+ and CD8+ T cells. Conversely, PRs were characterized by increased proportions of T helper 2 cells and CCL18-expressing macrophages, which are associated with the recruitment of type 2 T cells and regulatory T cells. NRs displayed preferential infiltration with immunosuppressive fibroblasts. Distinct spatial immune ecosystems further defined response groups. Although a number of immune cells were detected in all patients, type 1 effector cells dominated interactions in CRs, type 2 cells were prominently interacting in PRs, while NRs lacked organized immune cell interactions. Conclusions This study underscores the dual importance of both cellular composition and spatial organization in steering clinical response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b353470be584af7d95e65b2aa9a5dffe4241b32d" target='_blank'>
              Single-cell spatial transcriptomics unravels cell states and ecosystems associated with clinical response to immunotherapy
              </a>
            </td>
          <td>
            Z. Abdulrahman, Roderick C Slieker, Daniel McGuire, M. J. Welters, M. V. van Poelgeest, S. H. van der Burg
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ef6d6e98a4c414a78cbaa5ec4c9ee19d233f609" target='_blank'>
              ResolVI - addressing noise and bias in spatial transcriptomics
              </a>
            </td>
          <td>
            Can Ergen, N. Yosef
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>62</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) tumor inter-patient heterogeneity has been well described with two major prognostic subtypes (classical and basal-like). An important intra-patient heterogeneity has been reported but has not yet been extensively studied due to the lack of standardized, reproducible and easily accessible high throughput methods. Material and Methods We built an immunohistochemical (IHC) tool capable of differentiating RNA-defined classical and basal-like tumors by selecting relevant antibodies using a multi-step process. The successive stages of i) an in-silico selection from a review literature and a bulk transcriptome analysis of 309 PDACs, ii) a tumor-specific selection from 30 patient-derived xenografts followed by iii) the validation on tissue microarrays in 50 PDAC were conducted. We used our final IHC panel on two independent cohorts of resected PDAC (n=95, whole-slide, n=148, tissue microarrays) for external validation. After digitization and registration of pathology slides, we performed a tile-based-analysis in tumor and pre-neoplastic epithelial areas and a k-means clustering to identify relevant marker combinations. Results Sequential marker selection led to the following panel: GATA6, CLDN18, TFF1, MUC16, S100A2, KRT17, PanBasal. Four different phenotypes were identified: 1 classical, 1 intermediate (KRT17+) and 2 basal-like (MUC16+ vs S100A2+) with specific biological properties. The presence of a minor basal contingent drastically reduced overall survival, even in classical predominant PDACs (HR=2.36, p=0.01). Analysis of preneoplastic lesions suggested that pancreatic carcinogenesis may follow a progressive evolution from classical toward a basal through an early intermediate phenotype. Conclusion Our IHC panel redefined and easily assessed the high degree of intra- and inter-tumoral heterogeneity of PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e9c9cb0346cebac5553bfe30b873ba04769f272" target='_blank'>
              Redefining phenotypic intratumor heterogeneity of pancreatic ductal adenocarcinoma: a bottom‐up approach
              </a>
            </td>
          <td>
            M. Hilmi, F. Delecourt, J. Raffenne, Taib Bourega, Nelson Dusetti, J. Iovanna, Yuna Blum, Magali Richard, C. Neuzillet, A. Couvelard, Matthieu Tihy, L. de Mestier, Vinciane Rebours, R. Nicolle, Jérôme Cros
          </td>
          <td>2023-11-18</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors (TKIs) provide therapeutic benefit in EGFR mutation–positive non–small cell lung cancer, but some individuals develop early resistance. We performed spatial transcriptomics analysis of pre- and posttreatment tumor samples from the same patients to explore the underlying mechanisms of such early resistance. The proportion and activation of fibroblasts increased in association with the development of early resistance, whereas a distinct tumor cell cluster showed activation of tumor necrosis factor–α signaling via the NF-κB pathway even before treatment. Also in the early resistance sample, specific tumor cell clusters interacted with immune and stromal cells. Immature tertiary lymphoid structures (TLSs) were enriched in the early resistance sample, whereas mature TLSs were observed in the long-term response sample. These findings implicate tumor heterogeneity and an inflammatory tumor microenvironment in early EGFR-TKI resistance, providing insight into potential therapeutic strategies to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59cd540165dee7cba115de4f78c74d67fa9e26e" target='_blank'>
              Spatial dynamics of the tumor microenvironment associated with emerging resistance to targeted therapy in EGFR-mutated non–small cell lung cancer
              </a>
            </td>
          <td>
            Satoshi Nakamura, Daisuke Shibahara, Kentaro Tanaka, Yasuyuki Kishikawa, Mikiko Hashisako, K. Nakatomi, Noriaki Nakagaki, M. Kohno, Koichi Azuma, R. Ibusuki, K. Otsubo, Y. Yoneshima, E. Iwama, Yoshinao Oda, I. Okamoto
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8b6d1c2313c8e86b7cbcc29c94f8060fb9e7af4" target='_blank'>
              Abstract B048: Spatiotemporal analysis during tumor rejection reveals coordinated cellular dynamics underlying successful immune-checkpoint therapy
              </a>
            </td>
          <td>
            Ruan FV Medrano, Vladimir Sukhov, Maxim N Artyomov, Robert Schreiber
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Spatially resolved transcriptomics (SRT) has transformed tissue biology by linking gene expression profiles with spatial information. However, sequencing-based SRT methods aggregate signals from multiple cell types within capture locations (“spots”), masking cell-type-specific gene expression patterns. Traditional cell-type deconvolution methods estimate cell compositions within spots but fail to resolve cell-type-specific gene expression, limiting their ability to uncover critical biological processes such as cellular interactions and microenvironmental dynamics. Here, we present STged (spatial transcriptomic gene expression deconvolution), a novel computational framework that goes beyond traditional deconvolution by reconstructing cell-type-specific gene expression profiles from mixed spots. STged integrates graph-based spatial correlations and reference-derived gene signatures using a non-negative least-squares regression framework, achieving precise and biologically meaningful deconvolution. Comprehensive simulations show that STged consistently outperforms existing methods in accuracy and robustness. Applications to human pancreatic ductal adenocarcinoma and human squamous cell carcinoma datasets reveal its capacity to identify microenvironment-specific highly variable genes, reconstruct spatial cell–cell communication networks, and resolve tissue architecture at near-single-cell resolution. In mouse kidney tissues, STged uncovers dynamic spatial gene expression patterns and distinct gene programs, advancing our understanding of tissue heterogeneity and cellular dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db33671170dacba5e74f384054d9b15e8f2088ef" target='_blank'>
              Precise gene expression deconvolution in spatial transcriptomics with STged
              </a>
            </td>
          <td>
            Jia-Juan Tu, Hong Yan, Xiao-Fei Zhang, Zhixiang Lin
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda3dd1ef6762a74a82f982637ae107d4fb74853" target='_blank'>
              An integrated perspective on single-cell and spatial transcriptomic signatures in high-grade gliomas
              </a>
            </td>
          <td>
            Célia Lemoine, Marc-Antoine Da Veiga, B. Rogister, Caroline Piette, Virginie Neirinckx
          </td>
          <td>2025-02-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98eeee496484cc85e8f46ccc055600cb6afe1b1e" target='_blank'>
              A harmonized ovarian cancer scRNA-seq atlas to dissect disease heterogeneity underlying metastatization and chemoresistance
              </a>
            </td>
          <td>
            M. Sallese, Vittorio Aiello, C. E. Villa, Pietro Lo Riso, Giuseppe Testa
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407e0ecf5209e24e1ad27b286f77ee0657cbbbda" target='_blank'>
              Abstract A120: Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis
              </a>
            </td>
          <td>
            Veronica Aedo-Lopez
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61c2f79171ce9194bffe681aecda7bf66be2273" target='_blank'>
              STModule: identifying tissue modules to uncover spatial components and characteristics of transcriptomic landscapes
              </a>
            </td>
          <td>
            Ran Wang, Yan Qian, Xiaojing Guo, Fangda Song, Zhiqiang Xiong, Shirong Cai, Xiuwu Bian, Man-Hon Wong, Qin Cao, Lixin Cheng, Gang Lu, K. Leung
          </td>
          <td>2025-03-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7461a1f413e55baf162ee7fd7309ee3cb97ce7a" target='_blank'>
              Deep visual multi-omics profiling reveals mechanisms that underly cancer cell differentiation and aggressiveness in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            H. A. Bolck, Ede Migh, A. Kriston, Natalia Zajac, Susanne Kreutzer, Tiberiu Totu, Peter Leary, Ferenc Kovács, D. Rutishauser, Sybille Pfammatter, Jonas Grossmann, Cassandra Litchfield, Marija Buljan, Niels J Rupp, Péter Horváth, Holger Moch
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly malignant tumor in women, characterized by high morbidity, mortality, and recurrence rates. Although surgical treatment, radiotherapy, and chemotherapy are the mainstays of current treatment methods, the high heterogeneity of TNBC results in unsatisfactory outcomes with low 5-year survival rates. Rapid advancements in omics technology have propelled the understanding of TNBC molecular biology. The emergence of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has significantly enhanced knowledge of tumor heterogeneity and the distribution, functionality, and intercellular interactions of various cell types within the tumor microenvironment, including tumor cells, T cells, B cells, macrophages, and fibroblasts. The present study provides an overview of the technical characteristics of scRNA-seq and ST, highlighting their applications in exploring TNBC heterogeneity, cell spatial distribution patterns, and intercellular interactions. This review aims to enhance the comprehension of TNBC at the cellular level for the development of effective therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db2a16a0771f525b51537059ff8768102022813" target='_blank'>
              Analysis of single-cell and spatial transcriptomics in TNBC cell-cell interactions
              </a>
            </td>
          <td>
            Yan Xin, Qiji Ma, Qiang Deng, Tielin Wang, Dongxu Wang, Gang Wang
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Microbes have been found in various tumors and the research of tumor microbiome has been garnering increased attention. However, it remains challenge to investigate the microbiome in cancer at spatial resolution. The rapid advent of spatially resolved transcriptomics techniques has given rise to map transcripts at single-cell resolution in various types of cancer. Here, we constructed a comprehensive spatial meta-transcriptome resource by manually curating 203 fresh frozen (FF) slices from 20 cancers encompassing 334,253 spots and 1,908,646 cells. SMTdb (http://bio-bigdata.hrbmu.edu.cn/SMTdb/) was constructed to provide comprehensive insights into the abundance, distribution and enriched TME regions of 1218 microbiota in spatial tissue slices. SMTdb enables vast interactive data exploration of spatial distribution and expression of microbiota, host gene modules associated with certain microbiota and co-occurrence between microbiota and immune cells within tumor microenvironment. The atlas resource serves as a one-stop and time-effective platform to investigate the interactions among microbial ecosystems and hosts in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803a0dfe5a2e579dc8f160c9acd2466a2dcd43" target='_blank'>
              SMTdb: A comprehensive spatial meta-transcriptome resource in cancer
              </a>
            </td>
          <td>
            Weiwei Zhou, Qingyi Yang, Jiyu Guo, Si Li, Minghai Su, Feng Leng, Tingyu Rong, Jingyi Shi, Yueying Gao, Tiantongfei Jiang, Juan Xu, Yongsheng Li
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Lung adenocarcinoma, the most common subtype of lung cancer, is genomically complex, with tumors containing tens to hundreds of non-synonymous mutations. However, little is understood about how genes interact with each other to enable the evolution of cancer in vivo, largely due to a lack of methods for investigating genetic interactions in a high-throughput and quantitative manner. Here, we employed a novel platform to generate tumors with inactivation of pairs of ten diverse tumor suppressor genes within an autochthonous mouse model of oncogenic KRAS-driven lung cancer. By quantifying the fitness of tumors with every single and double mutant genotype, we show that most tumor suppressor genetic interactions exhibited negative epistasis, with diminishing returns on tumor fitness. In contrast, Apc inactivation showed positive epistasis with the inactivation of several other genes, including synergistic effects on tumor fitness in combination with Lkb1 or Nf1 inactivation. Sign epistasis was extremely rare, suggesting a surprisingly accessible fitness landscape during lung tumorigenesis. These findings expand our understanding of the interactions that drive tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e03ac1e8a6ee59b9db5be527feeaddbf8b397192" target='_blank'>
              Combinatorial In Vivo Genome Editing Identifies Widespread Epistasis and an Accessible Fitness Landscape During Lung Tumorigenesis
              </a>
            </td>
          <td>
            Jess D. Hebert, Yuning J Tang, Márton Szamecz, Laura Andrejka, Steven S. Lopez, D. Petrov, Gábor Boross, M. Winslow
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ca2d699ba974ef22ac49a1f2990095215e2dd0" target='_blank'>
              Bacterial population dynamics during colonization of solid tumors
              </a>
            </td>
          <td>
            Serkan Sayin, Motasem ElGamel, Brittany Rosener, Michael Brehm, Andrew Mugler, Amir Mitchell
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam U. Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan A. Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f697113c6b546482f0f42def83352f20ea919c" target='_blank'>
              Adaptive Plasticity Tumor Cells Modulate MAPK-Targeting Therapy Response in Colorectal Cancer
              </a>
            </td>
          <td>
            O. Villarreal, Yixin Xu, Ha Tran, Annette A. Machado, Dionne Prescod, Amanda Anderson, R. Minelli, Mike Peoples, Alejandro Hernandez Martinez, H. Lee, Chi Wut Wong, Natalie W Fowlkes, Preeti Kanikarla, Alexey Sorokin, J. Alshenaifi, Oluwadara Coker, Kangyu Lin, Chris Bristow, A. Viale, J. P. Shen, C. Parseghian, Joseph R Marszalek, Ryan Corcoran, S. Kopetz
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4421024c849016285df42c8f390ac35abcedd51e" target='_blank'>
              Pan-cancer spatial characterization of key immune biomarkers in the tumor microenvironment
              </a>
            </td>
          <td>
            James R. Lindsay, Jennifer Altreuter, J. Alessi, Jason L. Weirather, Anita Giobbie-Hurder, Ian D Dryg, Katharina V. Hoebel, B. Sharma, K. Felt, F. S. Hodi, N. Lindeman, L. Sholl, E. Cerami, Jonathan A. Nowak, Mark M Awad, Scott J Rodig, William E. Lotter
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Spatial transcriptomics has emerged as a pivotal technology for profiling gene expression of cells within their spatial context. The rapid growth of publicly available spatial data presents an opportunity to further our understanding of microenvironments that drive cell fate decisions and disease progression. However, existing foundation models, largely pretrained on single-cell RNA sequencing (scRNA-seq) data, fail to resolve the spatial relationships among samples or capture the unique distributions from various sequencing protocols. We introduce scGPT-spatial, a specialized foundation model for spatial transcriptomics continually pretrained on our previously published scGPT scRNA-seq foundation model. We also curate SpatialHuman30M, a comprehensive spatial transcriptomics dataset comprising of 30 million spatial transcriptomic profiles, encompassing both imaging- and sequencing-based protocols. To facilitate integration, scGPT-spatial introduces a novel MoE (Mixture of Experts) decoder that adaptively routes samples for protocol-aware decoding of gene expression profiles. Moreover, scGPT-spatial employs a spatially-aware sampling strategy and a novel neighborhood-based training objective to better capture spatial co-localization patterns among cell states within tissue. Empirical evaluations demonstrate that scGPT-spatial robustly integrates spatial data in mulit-slide and multi-modal settings, and effectively supports cell-type deconvolution and contextualized missing gene expression imputation, outperforming many existing methods. The scGPT-spatial codebase is publicly available at https://github.com/bowang-lab/scGPT-spatial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8468eae54f0cb47f9e40c0c406a76d27cb1b4ef3" target='_blank'>
              scGPT-spatial: Continual Pretraining of Single-Cell Foundation Model for Spatial Transcriptomics
              </a>
            </td>
          <td>
            Chloe Wang, Haotian Cui, Andrew Zhang, Ronald Xie, Hani Goodarzi, Bo Wang
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Background Triple-negative breast cancer is a prevalent breast cancer subtype with the lowest 5-year survival. Several factors contribute to its treatment response, but the inherent molecular and cellular tumor heterogeneity are increasingly acknowledged as crucial determinants. Methods Spatial transcriptomic profiling was performed on FFPE tissues from a retrospective, treatment-naive group of women with differential prognoses (17 with >15 years survival-good prognosis (GPx) and 15 with <3 years survival-poor prognosis (PPx)) using GeoMX® Digital Spatial Profiler. Regions of interest were segmented on pan-cytokeratin and analyzed for tumor and stromal components, probed using GeoMx human whole transcriptome atlas (WTA) panel. Data quality control, normalization, and differential analysis was performed in R using GeomxTools and linear mixed models. Additional analyses including cell-type deconvolution, spatial entropy, functional enrichment, TF-target / ligand-receptor analysis and convolution neural networks were employed to identify significant gene signatures contributing to differential prognosis. Results Here we report on the spatial and molecular heterogeneity underlying differential prognosis. We observe that the state of the epithelia and its microenvironment (TME) are transcriptionally distinct between the two groups. Invasive epithelia in GPx show a significant increase in immune transcripts with the TME exhibiting increased immune cell presence (via IF), while in PPx they are more metabolically and translationally active, with the TME being more mesenchymal/fibrotic. Specifically, pre-cancerous epithelia in PPx display a prescience of aggressiveness as evidenced by increased EMT-signaling. We identify distinct epithelial gene signatures for PPx and GPx, that can, with high accuracy, classify samples at the time of diagnosis and likely inform therapy. Conclusions To the best of our knowledge, this is the first study to leverage spatial transcriptomics for an in-depth delineation of the cellular and molecular underpinnings of differential prognosis in TNBC. Our study highlights the potential of spatial transcriptomics to not only uncover the molecular drivers of differential prognosis in TNBC but also to pave the way for precision diagnostics and tailored therapeutic strategies, transforming the clinical landscape for this aggressive breast cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31351362773ec3359f5dd1e2c1fcb9f73d1b49df" target='_blank'>
              Spatially distinct cellular and molecular landscapes define prognosis in triple negative breast cancer
              </a>
            </td>
          <td>
            Kavitha Mukund, Darya Veraksa, David Frankhouser, Lixin Yang, Jerneja Tomsic, Raju Pillai, Srijan Atti, Zahra Mesrizadeh, D. Schmolze, Xiao-Cheng Wu, Mary-Anne LeBlanc, Lucio Miele, Augusto Ochoa, Victoria L. Seewaldt, Shankar Subramaniam
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell spatial omics analysis requires consideration of biological functions and mechanisms in a microenvironment. However, microenvironment analysis using bioinformatic methods is limited by the need to detect histological morphology and extend it to the surrounding area. In this study, we developed SpatialKNifeY (SKNY), an image-processing-based toolkit that detects spatial domains that potentially reflect histology and extends these domains to the microenvironment. Using spatial transcriptomic data from breast cancer, we applied the SKNY algorithm to identify tumor spatial domains, followed by clustering of the domains, trajectory estimation, and spatial extension to the tumor microenvironment (TME). The results of the trajectory estimation were consistent with the known mechanisms of cancer progression. We observed tumor vascularization and immunodeficiency at mid- and late-stage progression in TME. Furthermore, we applied the SKNY to integrate and cluster the spatial domains of 14 patients with metastatic colorectal cancer, and the clusters were divided based on the TME characteristics. In conclusion, the SKNY facilitates the determination of the functions and mechanisms in the microenvironment and cataloguing of the features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796429c918dab0f1b4d04958c50f7cf17d100c5e" target='_blank'>
              SpatialKNifeY (SKNY): Extending from spatial domain to surrounding area to identify microenvironment features with single-cell spatial omics data
              </a>
            </td>
          <td>
            Shunsuke A Sakai, Ryosuke Nomura, Satoi Nagasawa, SungGi Chi, Ayako Suzuki, Yutaka Suzuki, Mitsuho Imai, Y. Nakamura, Takayuki Yoshino, Shumpei Ishikawa, K. Tsuchihara, S. Kageyama, Riu Yamashita
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b78a9a8e3280a5c16b1080da5f7e2a241cd45990" target='_blank'>
              Diffusion Smart-seq3 of breast cancer spheroids to explore spatial tumor biology and test evolutionary principles of tumor heterogeneity
              </a>
            </td>
          <td>
            Antony Cougnoux, Loay Mahmoud, Per A Johnsson, Alper Eroglu, Louise Gsell, Jakob Rosenbauer, R. Sandberg, Jean Hausser
          </td>
          <td>2025-01-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d2f0eb0a60c64efe2bd230e8933a5f80d2d405" target='_blank'>
              Intratumoral heterogeneity and drug resistance in cancer
              </a>
            </td>
          <td>
            Yue-Chun Fu, Shao-Bo Liang, Min Luo, Xue-Ping Wang
          </td>
          <td>2025-03-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f644ee3c95f246053a935ce676721b85861fe7f5" target='_blank'>
              Time-dependent chromatin maturation during 3D spheroid culture improves preclinical modeling of non-small cell lung cancer
              </a>
            </td>
          <td>
            Anaïs Darracq, Nicolas Sgarioto, Marielle Huot, Gabrielle McInnes, Antoine Meant, Alexandra Langford, Oscar D Villarreal, François Marois, Maxime Caron, P. Saint-Onge, S. Leclerc, G. Cagnone, Gregor Andelfinger, Stéphane Richard, Daniel Sinnett, Serge McGraw, Noël J-M Raynal
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer cells adapt to treatment, leading to the emergence of clones that are more aggressive and resistant to anti-cancer therapies. We have a limited understanding of the development of treatment resistance as we lack technologies to map the evolution of cancer under the selective pressure of treatment. To address this, we developed a hierarchical, dynamic lineage tracing method called FLARE (Following Lineage Adaptation and Resistance Evolution). We use this technique to track the progression of acute myeloid leukemia (AML) cell lines through exposure to Cytarabine (AraC), a front-line treatment in AML, in vitro and in vivo. We map distinct cellular lineages in murine and human AML cell lines predisposed to AraC persistence and/or resistance via the upregulation of cell adhesion and motility pathways. Additionally, we highlight the heritable expression of immunoproteasome 11S regulatory cap subunits as a potential mechanism aiding AML cell survival, proliferation, and immune escape in vivo. Finally, we validate the clinical relevance of these signatures in the TARGET-AML cohort, with a bisected response in blood and bone marrow. Our findings reveal a broad spectrum of resistance signatures attributed to significant cell transcriptional changes. To our knowledge, this is the first application of dynamic lineage tracing to unravel treatment response and resistance in cancer, and we expect FLARE to be a valuable tool in dissecting the evolution of resistance in a wide range of tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f240aad73d27cb602765176a2877d013637b5963" target='_blank'>
              Hierarchical Lineage Tracing Reveals Diverse Pathways of AML Treatment Resistance
              </a>
            </td>
          <td>
            Rachel Saxe, Hannah Stuart, Abigail Marshall, Fahiima Abdullahi, Zoë Chen, F. Emiliani, Aaron McKenna
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca97e870a1822de5485cb1751fdbfdd64fadf250" target='_blank'>
              Ontogeny Dictates Oncogenic Potential, Lineage Hierarchy, and Therapy Response in Pediatric Leukemia
              </a>
            </td>
          <td>
            Ke Wang, Shayan Saniei, Nikita Poddar, Subrina Autar, Saul Carcamo, Meghana Sreenath, Jack H Peplinski, R. Ries, Isabella G Martinez, Clifford Chao, Anna Huo-Chang Mei, Noshin Rahman, Levan Mekerishvili, Miguel Quijada-álamo, Grace Freed, Mimi Zhang, Katherine Lachman, Zayna Diaz, Manuel M Gonzalez, Jing Zhang, Giang Pham, Dan Filipescu, Mirela Berisa, Tommaso Balestra, J. Reisz, A. D’Alessandro, Daniel J Puleston, Emily Bernstein, Jerry E. Chipuk, M. Wunderlich, S. Tasian, Bridget K. Marcellino, Ian A. Glass, Christopher M Sturgeon, D. Landau, Zhihong Chen, E. Papapetrou, Franco Izzo, S. Meshinchi, Dan Hasson, Elvin Wagenblast
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2410faffb025161d800cf5841a3c4049fde9d25" target='_blank'>
              Integration of single-cell and spatial transcriptomics reveals fibroblast subtypes in hepatocellular carcinoma: spatial distribution, differentiation trajectories, and therapeutic potential
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-02-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee29e34be0729357213c5d8d79a8e1845ecaf78" target='_blank'>
              Multidimensional characterization of cellular ecosystems in Hodgkin lymphoma
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew G Lytle, Yifan Yin, Makoto Kishida, Michael Li, Denise Smorra, Laura K. Hilton, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O'Brien, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Patino Rangel, Michael Hong, Shaocheng Wu, Eric Lee, Katsuyoshi Takata, T. Miyata‐Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander M Xu, Akil Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, Ryan D Morin, A. Alizadeh, K. Savage, David W. Scott, Christian Steidl
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The spatial arrangement of cells plays a critical role in determining their functions and interactions within tissues. However, single-cell RNA sequencing (scRNA-seq) dissociates cells from their native tissue context, resulting in a loss of spatial information. Here, we show that complex tissue structures can be reassembled from the gene expression profiles of dissociated cells. To achieve this, we developed LUNA, a generative AI model that reconstructs tissues conditioned solely on gene expressions of cells by learning spatial priors over existing spatially resolved datasets. We show that LUNA effectively reconstructs slices from the MERFISH whole mouse brain atlas with over 1.2 million cells, including cells from cell types never seen during model training. Applying LUNA to the mouse central nervous system scRNA-seq atlas, we show that LUNA is applicable for de novo generation of tissue structures. Additionally, LUNA can infer locations of nuclei lost during cell profiling with Slide-tags technology and correctly places cells belonging to spatially distinct compartments in a human metastatic melanoma sample. We envision that AI-driven tissue reassembly can help to overcome current technological limitations and advance our understanding of tissue organization and function, paving the way towards virtual tissue models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2575a743b02c9647eeddfbc3632592d6900d90aa" target='_blank'>
              Tissue reassembly with generative AI
              </a>
            </td>
          <td>
            Tingyang Yu, C. Ekbote, Nikita Morozov, Jiashuo Fan, Pascal Frossard, Stéphane d’Ascoli, M. Brbić
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b5519e4d897e90daae110bc4297e9d131ea31b" target='_blank'>
              Biobank of genetically defined murine prostate cancer tumoroids uncovers oncogenic pathways and drug vulnerabilities driven by PTEN-loss
              </a>
            </td>
          <td>
            Jessica Kalla, Thomas E. Dillinger, Zlata Pavlovicova, Reema Jacob, Emine Atas, Anil Baskan, Kristina Draganić, A. Tiefenbacher, T. Limberger, Theresia Mair, Gabriel Wasinger, Ludovica Villanti, S. Kubicek, Lukas Kenner, G. Egger
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Brain metastasis occurs in approximately 16% of metastatic breast cancer patients, and incidence is increasing. Patients with breast cancer brain metastases frequently develop neurological problems including cognitive impairments and seizures. We previously demonstrated accumulation of activated microglia around the tumour site in a breast cancer brain metastasis model, which co-localised with spontaneously occurring local field potential events that resembled interictal epileptic discharges. However, the mechanisms underlying these spatially restricted effects on neuronal excitability are poorly understood. Methods To better understand the underlying cellular changes and potential mechanisms, we used digital spatial profiling of brain cell subtype-specific markers to chart the spatial organisation of neurons, oligodendrocytes, astrocytes, and microglia in relation to metastatic lesion site. We developed a novel chordline-based approach to analyse these spatial changes in biomarker distribution. Results Several protein markers associated with proliferation (Ki67), astrocytes (GFAP) and microglia (CD11b, IBA1, CD45, MSR1) were upregulated in the tumour compared to healthy contralateral brain parenchyma. In contrast, some neuroglial markers (MAP2, Neurofilament light, NeuN, S100B and TMEM119) were lower in the lesion compared to normal tissue. Conclusions Overall, the protein marker changes in the lesion are indicative of neuronal cell loss and changes in the immune microenvironment of the metastatic lesion involving activated microglia, astrocytes and recruitment of peripheral immune cells. Such cellular changes may contribute to the changes in electrical activity resulting from brain metastases observed in mice and patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61dbeafc4e0030d90e60f5d024e3f1200854ddc2" target='_blank'>
              Spatial profiling of central nervous system cellular markers in murine breast cancer to brain metastases
              </a>
            </td>
          <td>
            Karen Hogg, Grant Calder, Alastair P. Droop, Ming Yang, Anna Simon, Mark J Hunt, Paul M Kaye, Miles A. Whittington, Sangeeta Chawla, W. Brackenbury
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT Hepatoblastoma (HB) is a common paediatric liver malignancy characterised by significant intratumoral heterogeneity and a complex tumour microenvironment (TME). Using single‐cell RNA sequencing (scRNA‐seq), we analysed 43,592 cells from three tumour regions and adjacent normal tissue of an HB patient. Our study revealed distinct cellular compositions and varying degrees of malignancy across different tumour regions, with the T1 region showing the highest malignancy and overexpression of HMGB2 and TOP2A. Survival analysis demonstrated that high HMGB2 expression is associated with poor prognosis and increased recurrence, suggesting its potential as a prognostic marker. Additionally, we identified a diverse immune microenvironment enriched with regulatory T cells (Tregs) and CD8+ effector memory T cells (Tem), indicating potential immune evasion mechanisms. Notably, CTLA‐4 and PD‐1 were highly expressed in Tregs and Tem cells, highlighting their potential as immunotherapy targets. Myeloid cells, including Kupffer cells and dendritic cells, also exhibited distinct functional roles in different tumour regions. This study provides the first comprehensive single‐cell atlas of HB, revealing critical insights into its intratumoral heterogeneity and immune microenvironment. Our findings not only advance the understanding of HB biology but also offer new directions for precision medicine, including the development of targeted therapies and immunotherapeutic strategies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/634b11e52e203c006a13f961efc458d554db20a9" target='_blank'>
              Intratumoral Heterogeneity and Immune Microenvironment in Hepatoblastoma Revealed by Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Mingdi Ma, Chen Jin, Qian Dong
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Spatial transcriptomics technology remains resource-intensive and unlikely to be routinely adopted for patient care soon. This hinders the development of novel precision medicine solutions and, more importantly, limits the translation of research findings to patient treatment. Here, we present DeepSpot, a deep-set neural network that leverages recent foundation models in pathology and spatial multi-level tissue context to effectively predict spatial transcriptomics from H&E images. DeepSpot substantially improved gene correlations across multiple datasets from patients with metastatic melanoma, kidney, lung, or colon cancers as compared to previous state-of-the-art. Using DeepSpot, we generated 1 792 TCGA spatial transcriptomics samples (37 million spots) of the melanoma and renal cell cancer cohorts. We anticipate this to be a valuable resource for biological discovery and a benchmark for evaluating spatial transcriptomics models. We hope that DeepSpot and this dataset will stimulate further advancements in computational spatial transcriptomics analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e5c9f327828cd33b4c0fe8505c7428b8e43eb60" target='_blank'>
              DeepSpot: Leveraging Spatial Context for Enhanced Spatial Transcriptomics Prediction from H&E Images
              </a>
            </td>
          <td>
            Kalin Nonchev, Sebastian Dawo, Karina Silina, Holger Moch, Sonali Andani, Tumor Profiler Consortium, Viktor H Koelzer, Gunnar R¨atsch
          </td>
          <td>2025-02-12</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Spatial transcriptomics provides insights into tissue architecture by linking gene expression with spatial localization. Current deconvolution methods rely heavily on single-cell RNA sequencing (scRNA-seq) references, which are costly and often unavailable, mainly if the tissue under evaluation is limited, such as in a core biopsy specimen. We present a novel tool, CITEgeist, that deconvolutes spatial transcriptomics data using antibody capture from the same slide as the reference, directly leveraging cell surface protein measurements from the same tissue section. This approach circumvents the limitations of scRNA-seq as a reference, offering a cost-effective and biologically grounded alternative. Our method employs mathematical optimization to estimate cell type proportions and gene expression profiles, incorporating sparsity constraints for robustness and interpretability. Benchmarks against state-of-the-art deconvolution methods show improved accuracy in cell type resolution, particularly in dense tumor microenvironments, while maintaining computational efficiency. This antibody-based tool advances spatial transcriptomics by providing a scalable, accurate, and reference-independent solution for deconvolution in complex tissues. We validate this tool by using a combined approach of simulated data and clinical samples by applying CITEgeist to translational pre-treatment and post-treatment ER+ breast tumors from an ongoing clinical trial, emphasizing the applicability and robustness of CITEgeist.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4308d97c43b1d495c969105587ab2b638fc0a951" target='_blank'>
              CITEgeist: Cellular Indexing of Transcriptomes and Epitopes for Guided Exploration of Intrinsic Spatial Trends
              </a>
            </td>
          <td>
            A. Chang, Brent T. Schlegel, N. Carleton, Priscilla F McAuliffe, S. Oesterreich, Russell Schwartz, Adrian V. Lee
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6fee264333ab195499909e9bd27046249a7701" target='_blank'>
              STmiR: A Novel XGBoost-Based Framework for Spatially Resolved miRNA Activity Prediction in Cancer Transcriptomics
              </a>
            </td>
          <td>
            Jiaqi Yuan, Peng Xu, Zheng Ye, Wenbin Liu
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51f0d69b3ba41ef9a508aa6acb9b650626ef267d" target='_blank'>
              MorphoITH: A Framework for Deconvolving Intra-Tumor Heterogeneity Using Tissue Morphology
              </a>
            </td>
          <td>
            Aleksandra Weronika Nielsen, H. E. Manoochehri, Hua Zhong, Vandana Panwar, V. Jarmale, Jay Jasti, Mehrdad Nourani, Dinesh Rakheja, J. Brugarolas, Payal Kapur, Satwik Rajaram
          </td>
          <td>2025-02-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Abstract The intrinsic characteristics of metastatic tumors are of great importance in terms of the development of antimetastatic treatment strategies. Elucidation from a spatial immune perspective has the potential to provide a more comprehensive understanding of the mechanisms underlying immune escape, effectively addressing the limitations of relying solely on the analysis of immune cell subpopulation transcriptional profiles. Advances in spatial omics technology enable researchers to precisely analyze precious liver metastasis samples in a high-throughput manner, revealing spatial alterations in immune cell distribution induced by metastasis and exploring the molecular basis of the remodeling process. The aggregation of specific cell subpopulations in distinct regions not only modifies local immune characteristics but also concurrently affects global biological behaviors of liver metastatic tumors. Identifying specific spatial immune characteristics in pretreatment or early-stage treatment tissue samples may achieve accurate clinical predictions. Moreover, developing strategies that target spatial immune remodeling is a promising avenue for future antimetastatic therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ea479cd554af75a0d1dcd42be6ffca84a03793" target='_blank'>
              Spatial immune remodeling of the liver metastases: discovering the path to antimetastatic therapy
              </a>
            </td>
          <td>
            Wenchao Xu, Yibo Wang, Nanzhou Wang, Jianzhou Liu, Li Zhou, Junchao Guo
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bec213fd287708367fee6bf0c6de9efb0e1c0cb" target='_blank'>
              Natural killer cells occupy unique spatial neighborhoods in HER2- and HER2+ human breast cancers
              </a>
            </td>
          <td>
            Femke A I Ehlers, Katie E. Blise, C. Betts, Shamilene Sivagnanam, L. Kooreman, E. S. Hwang, Gerard M. J. Bos, L. Wieten, L. Coussens
          </td>
          <td>2025-01-24</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79158e9c810aea3af6c5a5e81cece5524c3196f" target='_blank'>
              Single Cell Spatial Transcriptomics Reveals Immunotherapy-Driven Bone Marrow Niche Remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R. Herzog, Abigail Wong-Rolle, Laura W Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine Calvo, Christopher S Hourigan, Kasper Hansen, Chen Zhao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb1760a6c58270791a1de56692df43805b36e2d0" target='_blank'>
              Single Cell RNA Sequencing Analysis of Human Melanoma Reveals A Distinct Prognostic Myeloid Cell
              </a>
            </td>
          <td>
            M. U. Ahmad, Lyong Heo, Ramesh V. Nair, Saurabh Sharma, E. Maverakis, Amanda R. Kirane
          </td>
          <td>2025-03-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Growing evidence supports the importance of characterizing the organizational patterns of various cellular constituents in the tumor microenvironment in precision oncology. Most existing data on immune cell infiltrates in tumors, which are based on immune cell counts or nearest neighbor-type analyses, have failed to fully capture the cellular organization and heterogeneity. Methods We introduce a computational algorithm, termed Tumor-Immune Partitioning and Clustering (TIPC), that jointly measures immune cell partitioning between tumor epithelial and stromal areas and immune cell clustering versus dispersion. As proof-of-principle, we applied TIPC to a prospective cohort incident tumor biobank containing 931 colorectal carcinoma cases. TIPC identified tumor subtypes with unique spatial patterns between tumor cells and T lymphocytes linked to certain molecular pathologic and prognostic features. T lymphocyte identification and phenotyping were achieved using multiplexed (multispectral) immunofluorescence. In a separate hepatocellular carcinoma cohort, we replaced the stromal component with specific immune cell types—CXCR3+CD68+ or CD8+—to profile their spatial relationships with CXCL9+CD68+ cells. Results Six unsupervised TIPC subtypes based on T lymphocyte distribution patterns were identified, comprising two cold and four hot subtypes. Three of the four hot subtypes were associated with significantly longer colorectal cancer (CRC)-specific survival compared to a reference cold subtype. Our analysis showed that variations in T-cell densities among the TIPC subtypes did not strictly correlate with prognostic benefits, underscoring the prognostic significance of immune cell spatial patterns. Additionally, TIPC revealed two spatially distinct and cell density-specific subtypes among microsatellite instability-high colorectal cancers, indicating its potential to upgrade tumor subtyping. TIPC was also applied to additional immune cell types, eosinophils and neutrophils, identified using morphology and supervised machine learning; here two tumor subtypes with similarly low densities, namely ‘cold, tumor-rich’ and ‘cold, stroma-rich’, exhibited differential prognostic associations. Lastly, we validated our methods and results using The Cancer Genome Atlas colon and rectal adenocarcinoma data (n = 570). Moreover, applying TIPC to hepatocellular carcinoma cases (n = 27) highlighted critical cell interactions like CXCL9-CXCR3 and CXCL9-CD8. Conclusions Unsupervised discoveries of microgeometric tissue organizational patterns and novel tumor subtypes using the TIPC algorithm can deepen our understanding of the tumor immune microenvironment and likely inform precision cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95d3eb1387144868bd261357332833886cbc29f0" target='_blank'>
              Tumor-immune partitioning and clustering algorithm for identifying tumor-immune cell spatial interaction signatures within the tumor microenvironment
              </a>
            </td>
          <td>
            M. Lau, J. Borowsky, J. Väyrynen, K. Haruki, Melissa Zhao, Andressa Dias Costa, Simeng Gu, Annacarolina da Silva, Tomotaka Ugai, Kota Arima, Minh N Nguyen, Yasutoshi Takashima, Joe Yeong, D. Tai, Tsuyoshi Hamada, Jochen K Lennerz, Charles S Fuchs, Catherine J Wu, J. Meyerhardt, S. Ogino, J. Nowak
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729ab293566d42d02b2188c02492d9921394583a" target='_blank'>
              Organoid modeling reveals the tumorigenic potential of the alveolar progenitor cell state
              </a>
            </td>
          <td>
            Jingyun Li, Susanna M Dang, Shreoshi Sengupta, Paul Schurmann, A. F. Dost, Aaron L. Moye, M. F. Trovero, Sidrah Ahmed, Margherita Paschini, Preetida J Bhetariya, Roderick Bronson, Shannan J. Ho Sui, Carla F. Kim
          </td>
          <td>2025-02-10</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd5809c9e6b6aaac22c97a1616ff68cb7ac8d0d" target='_blank'>
              High-dimensional deconstruction of HNSC reveals clinically distinct cellular states and ecosystems that are associated with prognosis and therapy response
              </a>
            </td>
          <td>
            Lei Xiao, Zhe Shen, Zhaoyu Pan, Yuanzheng Qiu, Donghai Huang, Yong Liu, Chao Liu, Xin Zhang
          </td>
          <td>2025-03-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Wilms tumor (WT) is the most common childhood renal malignancy, with recurrence linked to poor prognosis. Identifying the molecular features of tumor phenotypes that drive recurrence and discovering novel targets are crucial for improving treatment strategies and enhancing patient outcomes. Methods Single-nuclei RNA sequencing (snRNA-seq), spatial transcriptomics (ST), bulk RNA-seq, and mutation/copy number data were curated from public databases. The Seurat package was used to process snRNA-seq and ST data. Scissor analysis was applied to identify tumor subpopulations associated with poor relapse-free survival (RFS). Univariate Cox and LASSO analyses were utilized to reduce features. A prognostic ensemble machine learning model was developed. Immunohistochemistry was used to validate the expression of key features in tumor tissues. The CellChat and Commot package was utilized to infer cellular interactions. The PERCEPTION computational pipeline was used to predict the response of tumor cells to chemotherapy and targeted therapies. Results By integrating snRNA-seq and bulk RNA-seq data, we identified a subtype of Scissor+ tumor cells associated with poor RFS, predominantly derived from cap mesenchyme-like blastemal and fibroblast-like tumor subgroups. These cells displayed nephron progenitor signatures and cancer stem cell markers. A prognostic ensemble machine learning model was constructed based on the Scissor+ tumor signature to accurately predict patient RFS. TGFA was identified as the most significant feature in this model and validated by immunohistochemistry. Cellular communication analysis revealed strong associations between Scissor+ tumor cells and cancer-associated fibroblasts (CAFs) through IGF, SLIT, FGF, and PDGF pathways. ST data revealed that Scissor+ tumor cells were primarily located in immune-desert niche surrounded by CAFs. Despite reduced responsiveness to conventional chemotherapy, Scissor+ tumor cells were sensitive to EGFR inhibitors, providing insights into clinical intervention strategies for WT patients at high risk of recurrence. Conclusion This study identified a relapse-associated tumor subtype resembling nephron progenitor cells, residing in immune-desert niches through interactions with CAFs. The proposed prognostic model could accurately identify patients at high risk of relapse, offering a promising method for clinical risk stratification. Targeting these cells with EGFR inhibitors, in combination with conventional chemotherapy, may provide a potential therapeutic strategy for WT patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05daf7364d02794afe753ef756b89d494ba7b356" target='_blank'>
              Integration of single-nuclei and spatial transcriptomics to decipher tumor phenotype predictive of relapse-free survival in Wilms tumor
              </a>
            </td>
          <td>
            Ran Yang, Lulu Xie, Rui Wang, Yi Li, Yifei Lu, Baihui Liu, Shu-yang Dai, Shan Zheng, Kuiran Dong, Rui Dong
          </td>
          <td>2025-03-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/022fdf3eed7c33ba165a376f1b3fb14ad94870ef" target='_blank'>
              Spotiphy enables single-cell spatial whole transcriptomics across an entire section.
              </a>
            </td>
          <td>
            Jiyuan Yang, Ziqian Zheng, Yun Jiao, Kaiwen Yu, Sheetal Bhatara, Xu Yang, Sivaraman Natarajan, Jiahui Zhang, Qingfei Pan, John Easton, Koon-Kiu Yan, Junmin Peng, Kaibo Liu, Jiyang Yu
          </td>
          <td>2025-03-12</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background: Liver progenitor cells (LPCs) with bipotential differentiation capacities are essential for restoring liver homeostasis and hepatocyte population after damage. However, the low proportion and shared markers with epithelial cells make studying LPC heterogeneity difficult, especially in humans. To address this gap, we explored over 259,400 human liver single cells across 4 conditions (fetal, healthy, cirrhotic, and HCC-affected livers). Methods: Human liver tissue samples were analyzed using spatial transcriptomics sequencing technologies to describe the heterogeneity of LPCs. Liver tissue was characterized by LPC heterogeneity at single-cell resolution by employing cellular modules, differentiation trajectories, and gene co-expression patterns. Results: We annotated and identified 1 LPC cluster, 3 LPC subpopulations, and 4 distinct cellular modules, indicating the heterogeneity within LPC and the diversity between LPCs and epithelial cells. LPCs showed spatial colocalization with cholangiocytes and comprised a small proportion (2.95±1.91%) within the merged epithelial cells and LPC populations, exhibiting marked differences in marker expression patterns compared to those in mice. LPCs exhibited distinct cellular states in functional restoration, activation, proliferation, and cell transition. Additionally, the gene co-expression network of LPCs exhibited 3 unique modules, reflecting the distinct connectivity of genes encoding apolipoproteins and heat shock proteins in the gene co-expression network modules. Conclusions: Our study provides valuable insights into the multifaceted heterogeneity of human LPCs. Future studies focusing on spatial gene expression dynamics will contribute to our understanding of the spatial arrangement of liver regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/363fd866c375a553c02fda34c7798688e167a041" target='_blank'>
              Integrative single-cell and spatial transcriptome analysis reveals heterogeneity of human liver progenitor cells
              </a>
            </td>
          <td>
            Chuanjun Liu, Kai Wang, Junpu Mei, Ruizhen Zhao, Juan Shen, Wei Zhang, Liangyu Li, Bhaskar Roy, Xiaodong Fang
          </td>
          <td>2025-02-26</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neurodegenerative diseases and infections can produce lasting effects on brain function, yet the spatial molecular mechanisms underlying these changes remain poorly understood. Here, we present high-resolution spatial transcriptomics of 40 postmortem brain samples from patients with Parkinson’s disease, frontotemporal dementia, dementia with Lewy bodies, and severe COVID-19. Analyzing over 1.5 million spatially resolved cells across dorsolateral prefrontal cortex and anterior cingulate cortex revealed disease-specific transcriptional signatures with pronounced layer-and region-specificity. In Parkinson’s disease, we identified stressed neurons creating distinctive microenvironmental gradients where metabolic and protein degradation pathways are elevated near stress epicenters, while regenerative processes increase with distance. COVID-19 brains displayed extensive peripheral immune cell infiltration, particularly in the subcortical white matter, accompanied by compromised blood-brain barrier and coordinated neuroinflammatory responses from microglia, astrocytes, and endothelial cells. Integration of miRNA sequencing with spatial transcriptomics uncovered layer-specific regulatory patterns, including neuroinflammation-associated miR-155. This atlas provides unprecedented insights into disease pathology and highlights the critical importance of spatial molecular context in understanding brain disorders. Key Messages [1] A high-resolution single-cell spatial transcriptomics atlas of the dorsolateral prefrontal cortex and anterior cingulate cortex across neurodegenerative conditions and severe COVID-19 [2] Region-and layer-specific transcriptional dysregulation across disease comparisons reveals disease-specific differential vulnerability [3] Metabolically stressed cells found selectively in the anterior cingulate cortex of Parkinson’s disease patients but not in dementia with Lewy bodies, along with detailed characterization of their spatial microenvironment [4] Peripheral immune cell clusters identified in the white matter of the cerebral cortex of COVID-19 patients, with detailed characterization of their spatial microenvironment [5] Integration of bulk miRNA sequencing reveals cortical layer-specific miRNA regulatory patterns">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97c071df2a109d09f7501e38a14ec4947fe8e317" target='_blank'>
              Single-cell Spatial Transcriptomics Reveals Disease-specific Microenvironmental Niches in Neurodegeneration and COVID-19
              </a>
            </td>
          <td>
            Simon Graf, N. Ludwig, Matthias Flotho, Annika Engel, I. F. Diks, Paula Zimmer, Wiebke M. Jürgens-Wemheuer, Arne Wrede, Daniela Mirzac, G. González-Escamilla, Victoria Wagner, Vilas Menon, Sergiu Groppa, P. D. De Jager, Lai Yiwei, Miguel A. Esteban, Walter J. Schulz-Schaeffer, Andreas Keller
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatially resolved transcriptomics has transformed our ability to study complex tissues at the cellular and subcellular resolution. However, targeted spatial technologies require pre-selected gene panels, which are typically curated based on existing biological knowledge and prior research hypotheses. While current methods often prioritize capturing cell type information, we argue that an effective gene panel should also capture cell type diversity, cell states, pathway-level information, and minimize redundancy. To address these broader requirements, we developed a gene panel characterization platform that characterizes panels across multiple perspectives, thus allowing us to compare panels comprehensively. Notably, computationally constructed gene panels performed competitively in capturing major cell types when compared to our in-house manually curated panel. However, refined manual curation offered distinct advantages, particularly in capturing minor and rare cell types and exhibited lower information redundancy comparatively. Building on this framework, we integrated these metrics into a deep learning platform, panelScope, leveraging them as a loss function to design holistic gene panels. Using an acute myeloid leukemia (AML) dataset with 42 well-defined cell types and the 5K Xenium panel from 10X Genomics, we demonstrate the utility of our framework in comprehensively characterizing gene panels, enabling the design of tailored panels that address diverse research needs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef9065cee3317a4c66d56d6484e04cef47751f06" target='_blank'>
              Multi-view gene panel characterization for spatially resolved omics
              </a>
            </td>
          <td>
            Daniel Kim, Wenze Ding, Akira Nguyen Shaw, Marni Torkel, Cameron J Turtle, Pengyi Yang, J. Yang
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Recent advances in spatial transcriptomics (ST) technologies have transformed our ability to profile gene expression while retaining the crucial spatial context within tissues. However, existing ST platforms suffer from high costs, long turnaround times, low resolution, limited gene coverage, and small tissue capture areas, which hinder their broad applications. Here we present iSCALE, a method that predicts super-resolution gene expression and automatically annotates cellular-level tissue architecture for large-sized tissues that exceed the capture areas of standard ST platforms. The accuracy of iSCALE were validated by comprehensive evaluations, involving benchmarking experiments, immunohistochemistry staining, and manual annotation by pathologists. When applied to multiple sclerosis human brain samples, iSCALE uncovered lesion associated cellular characteristics that were undetectable by conventional ST experiments. Our results demonstrate iSCALE’s utility in analyzing large-sized tissues with automatic and unbiased tissue annotation, inferring cell type composition, and pinpointing regions of interest for features not discernible through human visual assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca1037b8c19366713347f95c55ebd14edfcd843" target='_blank'>
              Scaling up spatial transcriptomics for large-sized tissues: uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE
              </a>
            </td>
          <td>
            Amelia Schroeder, Melanie Loth, Chunyu Luo, Sicong Yao, Hanying Yan, Daiwei Zhang, Sarbottam Piya, Edward Plowey, Wen-Tuo Hu, Jean R. Clemenceau, Inyeop Jang, Minji Kim, Isabel Barnfather, Su Jing Chan, Taylor L. Reynolds, Thomas Carlile, Patrick Cullen, Ji-Youn Sung, Hui-Hsin Tsai, Jeong Hwan Park, Tae Hyun Hwang, Baohong Zhang, Mingyao Li
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1221249520ea9d5b52e0d651f4686ac33466a610" target='_blank'>
              Staining-free Visualization of Cell Boundaries in Sliced Tissues for Accurate Binning of Gene Expression in Spatial Transcriptomics
              </a>
            </td>
          <td>
            Zhiming Zhang, Xiaoxia Chen, Youyuan Chen, Zixuan Liao, T. Liu
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64eb12bd7d9a78fd223bedc136a7a4d7a78d40" target='_blank'>
              Abstract B068: A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer
              </a>
            </td>
          <td>
            M. Oh, J. Abascal, L. Tran, R. Salehi-Rad, R. Lim, C. Dumitras, Cara E. Yean, Sunny Xiao, Bitta Kahangi, William P Crosson, E. P. Reyes, Nalani Coleman, Austin K. Rennels, K. Krysan, Bin Liu, Steven M Dubinett
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ead468254f12c6dc5934a33a000887d505597fb4" target='_blank'>
              Spatial transcriptomics and lethality-associated stromal remodeling of colorectal cancer.
              </a>
            </td>
          <td>
            C. Wood, João Da Silva Filho, Tengyu Zhang, Campbell Roxburgh, Paul G. Horgan, J. Edwards, Matthias Marti, Colin W. Steele, Nigel B Jamieson
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a529afcdab161e3ac0972bc45b1449999ddb519" target='_blank'>
              EMT-driven plasticity prospectively increases cell–cell variability to promote therapeutic adaptation in breast cancer
              </a>
            </td>
          <td>
            Lauriane Muller, Frédérique Fauvet, Christelle Chassot, Francesca Angileri, Angèle Coutant, Cyril Dégletagne, Laurie Tonon, P. Saintigny, Alain Puisieux, A. Morel, Maria Ouzounova, Pierre Martinez
          </td>
          <td>2025-02-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24163c4c446c34923a67d83b5c03139aa6a3cb4" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics reveal the heterogeneity and intercellular communication of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xijie Zhang, Bo Ren, Bo Liu, Rui Wang, Sen Li, Yuzhou Zhao, Wence Zhou
          </td>
          <td>2025-03-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69f29862a6e18da7c70e3c9d22cd4b7cabc18a3" target='_blank'>
              Abstract B028: Multimodal approach to characterization of metabolism and immune modulation with insights from spatial metabolomics and transcriptomics
              </a>
            </td>
          <td>
            Chelsea L. Rahiman, Roberto A Ribas, Manan Vij, Julia An, Patrick McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z Obradovic, Ramon C. Sun, Catherine S. Spina
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) is a cutting-edge technology that provides comprehensive insights into gene expression patterns from a spatial perspective. A key research focus within this field is spatial domain identification, which is essential for exploring tissue organization, biological development, and disease mechanisms. Although methods have been developed, they still face challenges in modeling the gene expression information together with the spatial locations, resulting in suboptimal accuracy. We introduce DisConST (Distribution-aware Contrastive Learning for Spatial Transcriptomics), a novel deep-learning method designed to improve spatial domain detection within spatial transcriptomics datasets. DisConST addresses key challenges, such as the high dropout rates and the complex integration of spatial and gene expression data, by incorporating contrastive learning strategies that are aware of the underlying data distributions. It employs the zero-inflated negative binomial (ZINB) distribution, along with graph contrastive learning, to generate more informative latent representations. These representations efficiently integrate spatial positions, transcriptomic profiles, and cell-type proportions within spots. We validated DisConST across diverse ST datasets of tissues, organs, and embryos from various sequencing platforms in both normal and disease states. Our results consistently demonstrated that DisConST achieves superior spatial domain recognition accuracy compared to existing state-of-the-art methods. Furthermore, our experiments highlighted the utility of DisConST in advancing research on tissue organization, embryonic development, and tumor immune microenvironment dissection. The source code for DisConST is freely available at https://github.com/Zhenpm/DisConST/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745c9ac0a0fd2c2ed7fb914bb3fa37f78a95e73c" target='_blank'>
              DisConST: Deciphering Spatial Domains Using Distribution-aware Contrastive Learning for Spatial Transcriptomics
              </a>
            </td>
          <td>
            Peimeng Zhen, Xiaofeng Wang, Han Shu, Jialu Hu, Yongtian Wang, Jiajie Peng, Xuequn Shang, Jing Chen, Tao Wang
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58e073c2b3967f3720af1b89697ce0e1aad33c4c" target='_blank'>
              Abstract B086: Disease-critical macrophage populations interact with stroma to modulate the tumor microenvironment
              </a>
            </td>
          <td>
            Weiwei Zhong, Mitra Ansariola, Micaela Tatman, Rachel Wester, Kim Van Naarden Braun, Shameek Biswas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/644814c562e273342aaa2172d8ae291ec9003332" target='_blank'>
              Characterization of tumour heterogeneity through segmentation-free representation learning on multiplexed imaging data.
              </a>
            </td>
          <td>
            Jimin Tan, H. Le, Jiehui Deng, Yingzhuo Liu, Yuan Hao, Michelle Hollenberg, Wenke Liu, Joshua M. Wang, Bo Xia, Sitharam Ramaswami, Valeria Mezzano, Cynthia Loomis, Nina Murrell, Andre L. Moreira, Kyunghyun Cho, Harvey I Pass, Kwok-Kin Wong, Yi Ban, Benjamin G. Neel, A. Tsirigos, David Fenyö
          </td>
          <td>2025-02-20</td>
          <td>Nature biomedical engineering</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/807d24ffeaa1b756156a596b5f9b0d95ce824b53" target='_blank'>
              Batch correcting single-cell spatial transcriptomics count data with Crescendo improves visualization and detection of spatial gene patterns
              </a>
            </td>
          <td>
            N. Millard, Jonathan H Chen, Mukta G. Palshikar, K. Pelka, Maxwell Spurrell, Colles Price, Jiang He, N. Hacohen, Soumya Raychaudhuri, Ilya Korsunsky
          </td>
          <td>2025-02-25</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b0ad7d9e2bbe3827fd09e473275b031d3cd8d0" target='_blank'>
              Abstract P019: Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy
              </a>
            </td>
          <td>
            Gina Bouchard
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac4b19bdc51a77041e08cca430b7dd7ca4fef49" target='_blank'>
              Abstract PR007: Heterotypic CD8 T cell clusters isolated from clinical samples are distinct and enriched for antitumor activity
              </a>
            </td>
          <td>
            D. Peeper, Sofía Ibáñez-Molero, Johanna Veldman, Juan Simon-Nieto, J. Traets, Austin George, K. Hoefakker, Anita Karomi, Jonas Nilsson, John Haanen, W. V. van Houdt
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) represent a central cell population of the tumor microenvironment (TME). Recently, single‐cell RNA‐sequencing (scRNA‐seq) analyses of primary tumors of different cancer entities yielded different classifications of CAF subsets underscoring the heterogeneity of CAFs within the TME. Here, we analyzed the transcriptional signatures of approximately 8400 CAFs and normal fibroblasts by scRNA‐seq and compared genetic profiles of CAFs from murine melanoma primary tumors to CAFs from corresponding melanoma lung metastases. This revealed distinct subsets for primary tumor and metastasis‐specific CAF populations, respectively. Combined with the spatial characterization of metastasis CAFs at the RNA and protein level, scRNA analyses indicate tumor‐dependent crosstalk between neutrophils and CAFs, mediated via SAA3 and IL1b‐related signaling pathways, which can be recapitulated in vitro. Analyzing tissue sections of human patient samples, this interaction was found to be present in human melanoma metastasis. Taken together, our data highlight unique characteristics of metastasis CAFs with potential therapeutic impact for melanoma metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae4d264150cbc5dcfaa11bf8a1b1f48e7dcf511" target='_blank'>
              A Unique Signature for Cancer‐Associated Fibroblasts in Melanoma Metastases
              </a>
            </td>
          <td>
            Saskia Tauch, Joschka Hey, Bettina Kast, N. Gengenbacher, Lena Weiß, Melanie Sator‐Schmitt, S. Lohr, Alexander Brobeil, P. Schirmacher, Jochen Utikal, H. Augustin, C. Plass, Peter Angel
          </td>
          <td>2025-02-09</td>
          <td>Pigment Cell & Melanoma Research</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce737a0ad997a56048c24df55247b712e440df3d" target='_blank'>
              Abstract PR002: T cell engager therapy affects the spatial distribution and phenotype of T cells in the tumor microenvironment
              </a>
            </td>
          <td>
            Billy Tomaszewski, Matthew Curtis, Conrad Foo, Gu Zhang, Patrick S Chang, Robyn Clark, Thao Nguyen, Joshua Webster, Sandra Rost, Raj Jesudason, Robert Piskol, Klara Totpal, Lisa Mcginnis, Kevin B Walsh, Weilan Ye
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary Cancer stem cells (CSCs) are pivotal in tumor growth, recurrence, and resistance to standard therapies. This review delves into CSCs’ ability to self-renew and differentiate, fueling tumor heterogeneity and metastasis. It highlights critical signaling pathways like Wnt, Notch, and Hedgehog, which are essential for targeting CSCs. Emerging technologies, such as CRISPR/Cas9 and single-cell sequencing, promise to enhance CSC research and management. The review also explores the complex interaction between CSCs and the tumor microenvironment (TME), which influences therapeutic effectiveness. Addressing CSC-driven resistance, the article proposes advanced strategies for overcoming these challenges. It emphasizes the role of artificial intelligence in designing personalized, CSC-targeted therapies, ushering in a new era for precision medicine in oncology. This comprehensive approach aims to improve cancer treatment outcomes through innovative technologies and inter-disciplinary collaboration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99151138c6fc7abf05753313b772a7f6497b1128" target='_blank'>
              Deciphering the Role of Cancer Stem Cells: Drivers of Tumor Evolution, Therapeutic Resistance, and Precision Medicine Strategies
              </a>
            </td>
          <td>
            Mohamed El-Tanani, S. Rabbani, S. Satyam, I. Rangraze, A. Wali, Yahia El-Tanani, Alaa A. A. Aljabali
          </td>
          <td>2025-01-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77e71e2d20d9e52879b8c2d8cc5014dd16b7a335" target='_blank'>
              SVCFit: Inferring structural variant cellular fraction in tumors
              </a>
            </td>
          <td>
            Yunzhou Liu, Jiaying Lai, Laura D. Wood, Rachel Karchin
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ea420e4a0256eeeba7153fed74e4eb3b92c386" target='_blank'>
              Spatial analysis of a complete DIPG-infiltrated brainstem reveals novel ligand-receptor mediators of tumour-to-TME crosstalk
              </a>
            </td>
          <td>
            Anja Kordowski, Onkar Mulay, Xiao Tan, Tuan Vo, Ulrich Baumgartner, Mellissa Maybury, Timothy Hassall, L. Harris, Quan Nguyen, B. Day
          </td>
          <td>2025-02-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e047830d6d15c84fdba5a27411c84ea5a08197" target='_blank'>
              Multi-Modal Disentanglement of Spatial Transcriptomics and Histopathology Imaging
              </a>
            </td>
          <td>
            Hassaan Maan, Zongliang Ji, Elliot Sicheri, Tiak Ju Tan, Alina Selega, Ricardo Gonzalez, Rahul G Krishnan, Bo Wang, Kieran R. Campbell
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9547a039c0fe93efd8fd0787388be5b2aa0012a6" target='_blank'>
              Integrating spatial and single-cell transcriptomes reveals the role of COL1A2(+) MMP1(+/-) cancer-associated fibroblasts in ER-positive breast cancer
              </a>
            </td>
          <td>
            Zhi-Hao Yu, Huan-Ling Xu, Shuo Wang, Yingxi Li, Guixin Wang, Yao Tian, Zhaohui Chen, Wenbin Song, Long He, Xin Wang, Xuchen Cao, Yue Yu
          </td>
          <td>2025-03-07</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14cda68a9739dfb6fdc7acbc975a0b67601ec1ae" target='_blank'>
              Quantitative characterization of cell niches in spatially resolved omics data.
              </a>
            </td>
          <td>
            Sebastian Birk, Irene Bonafonte-Pardàs, Adib Miraki Feriz, Adam Boxall, E. Agirre, Fani Memi, A. Maguza, Anamika Yadav, Erick Armingol, Rong Fan, Gonçalo Castelo-Branco, F. Theis, O. Bayraktar, Carlos Talavera-López, M. Lotfollahi
          </td>
          <td>2025-03-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Quiescence, a reversible state of cell-cycle arrest, is an adaptive feature of many adult tissue stem cells, including those in the adult brain. In gliomas, brain tumour stem cells that reside in a quiescent state preferentially survive chemotherapy and radiotherapy, highlighting their critical role in therapy resistance and disease progression. To date, it remains unclear whether the molecular programs governing these states are functionally conserved between neural stem cells and brain tumour stem cells. Here, we establish novel in vitro models to study quiescence and find that glioma stem cells are markedly more resistant to entering quiescence than neural stem cells, suggesting that glioma stem cell quiescence more closely resembles a slow-cycling phenotype or shallow quiescence. Nonetheless, direct comparison of quiescent neural stem cells and quiescent/slow-cycling glioma stem cells, as they transition towards proliferation, reveals conserved gene expression trajectories, indicating shared molecular mechanisms. Furthermore, we find that pathways influencing quiescence in neural stem cells exert similar effects in glioma stem cells, underscoring the functional parallels between these populations. Finally, we identify that inhibition of TGF-β signalling might provide an avenue to improve current standard-of-care treatments by targeting quiescent glioma stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfaba977c1ae379d70e521eff155f6c1e15b6182" target='_blank'>
              Shared molecular regulation of quiescence in neural and glioma stem cells reveals therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Chandra Choudhury, Matthew Singleton, Stephanie Brauer, Dana Friess, Jessica Hart, N. Skarne, Karthik Pullela, Likai Mao, Bryan W Day, Lachlan Harris
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec81235338a787d945989660c2ddefa3be4c0bd" target='_blank'>
              Novel CHI3L1-Associated Angiogenic Phenotypes Define Glioma Microenvironments: Insights From Multi-Omics Integration.
              </a>
            </td>
          <td>
            Yu-Hang Zhao, Yu-Xiang Cai, Zhi-yong Pan, F. Tang, Chao Ma, Ze-Fen Wang, Gang Li, Hang Chang, Su-Fang Tian, Zhi-Qiang Li
          </td>
          <td>2025-02-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Atherosclerosis involves complex interactions between lipids, immune cells, vascular smooth muscle cells (VSMCs), and fibroblasts within the arterial wall. While significant advances in single-cell technologies have shed light on the roles of immune cells and VSMCs in plaque development, fibroblasts remain underexplored, leaving critical gaps in understanding their contributions to disease progression and plaque stability. Comprehensive characterization of fibroblast phenotypes in atherosclerosis is essential to unravel their diverse functions and to distinguish between subsets that may play protective versus pathogenic roles in the disease process. Methods Here, we utilized CITE-seq (Cellular Indexing of Transcriptomes and Epitopes by Sequencing) to comprehensively profile fibroblast diversity in a mouse model of atherosclerosis. Mice were fed an atherogenic diet for 0, 8, 19, and 26 weeks, representing distinct stages of disease progression, enabling a detailed phenotypic characterization of fibroblasts throughout the course of atherosclerosis development. Results We identified four distinct fibroblast subpopulations, including a myofibroblast population closely resembling VSMC-derived chondromyocytes. The proportions of these fibroblast subsets exhibited a modest decline as atherosclerosis progressed. Through multimodal analysis, we identified CD26 as a highly expressed and specific marker for one of these fibroblast subpopulations, distinguishing it from other subsets. Using a combination of flow cytometry and immunohistochemistry, we demonstrated that CD26+ fibroblasts predominantly reside in the adventitia of healthy arteries. During atherosclerosis progression, these cells expand into the intima and primarily localize within the fibrous cap of the lesion. Conclusions Our multi-omic analysis highlights the phenotypic diversity and dynamic changes of fibroblasts during atherosclerosis progression. Among these, CD26+ fibroblasts emerge as a distinct subpopulation that expands within atherosclerotic lesions and may play a critical role in promoting plaque stability through their migration into the fibrous cap.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcefa069d1b01c8875c90b650f37b9a970f0c89d" target='_blank'>
              Single-Cell Multimodal Profiling Reveals a Novel CD26+ Fibroblast Subpopulation in Atherosclerosis
              </a>
            </td>
          <td>
            A. Bashore, Johana Coronel, Chenyi Xue, Lucie Y. Zhu, Muredach P. Reilly
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95f806851975b88a84521b58227335a097ae890e" target='_blank'>
              Spatial transcriptomics in autoimmune rheumatic disease: potential clinical applications and perspectives
              </a>
            </td>
          <td>
            Atsuko Tsujii Miyamoto, Hiroshi Shimagami, Atsushi Kumanogoh, Masayuki Nishide
          </td>
          <td>2025-02-20</td>
          <td>Inflammation and Regeneration</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNAseq) of human tissues has expanded our understanding of the complexity of cellular subsets and their changes in disease. The availability of scRNAseq data in different tissues and disease states provides an opportunity to compare cellular subsets and identify common and unique cellular activation. In this study, we aimed to characterize shared and tissue-specific myeloid and stromal phenotypes and uncover key cellular subtypes involved in pathogenic tissue activation. We analyzed scRNAseq data from 14 public datasets, comprising heart, lung, skin, and synovium in both healthy and diseased states. Our analysis identified distinct and overlapping myeloid and stromal cell populations in these tissues. Despite significant inter-individual variability, we were able to identify both shared and disease-specific changes in these cell populations. These findings provide insights into the conserved and tissue-specific roles of myeloid and stromal cells in health and disease and contribute to a better understanding of tissue pathology and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a4d83ee093d91b7fc9e6f8d4d131194bc8ae33" target='_blank'>
              Human Fibroblast-Myeloid cell tissue atlas across lung, synovium, skin and heart
              </a>
            </td>
          <td>
            L. MacDonald, Olympia M Hardy, Melpomeni Toitou, Gabriela Kania, P. Błyszczuk, R. Micheroli, Thomas D. Otto, Mariola Kurowska-Stolarska, C. Ospelt
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0761133026ec3ba173f2c421844c6435e13b010" target='_blank'>
              Real-Time Tracking of Tumor Invasion Dynamics in Organotypic Brain Slices at Subcellular Resolution
              </a>
            </td>
          <td>
            Jana B. Petr, Meng-Syuan Lin, Dina Hochuli, Andreas Hierlemann, Martin Baumgartner, Mario M. Modena
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3418175c8e687d05cd4cb3259c7f39ab58b56c25" target='_blank'>
              Dynamic responses to rejection in the transplanted human heart revealed through spatial transcriptomics
              </a>
            </td>
          <td>
            K. Amancherla, Angela M. Taravella Oill, Xavier Bledsoe, A. L. Williams, Nelson Chow, Shilin Zhao, Quanhu Sheng, David W. Bearl, Robert D. Hoffman, J. Menachem, Hasan K. Siddiqi, Douglas M. Brinkley, E. D. Mee, Niran Hadad, Vineet Agrawal, Jeffrey Schmeckpepper, A. Rali, Stacy Tsai, E. Farber-Eger, Quinn S Wells, Jane E. Freedman, Nathan R. Tucker, K. Schlendorf, Eric R. Gamazon, Ravi V. Shah, N. Banovich
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Therapeutic drug treatments of solid tumors are often undermined by various resistance mechanisms. Identification of drug-resistance phenotypes at the single cell level is challenging because conventional molecular methods are cell-destructive, labor-intensive, and cost-prohibitive. To overcome these challenges, we developed an orthogonal approach to drug-resistance phenotyping, through the use of deep-learning-driven morphology analysis of single, high resolution cell images. Specifically, we trained deep learning-based drug resistance classifiers using cell images from 5 different cell lines that were rendered resistant to 5 different therapeutic agents, using a foundation model framework. With high accuracy, the classifier correctly predicted naive or resistance phenotypes across all cancer types and across all the therapeutic agent types (chemotherapeutic, targeted) tested. These results showed that morphology can capture complex phenotype information in the context of drug treatment. To demonstrate the potential clinical utility of the drug resistance classifier, it was applied to a dissociated tumor biopsy and the resulting phenotype predictions were in close concordance with scRNASeq analysis of the biopsy. Our study highlights the potential of deep-learning-driven morphology analysis to provide complex phenotype information, and ultimately shape oncology drug treatment strategies at the patient-level in a clinical context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65e32bdcccdadfa02831a9867b05cc627a1045f4" target='_blank'>
              Deep learning-driven morphology analysis enables label-free classification of therapeutic agentnaive versus resistant cancer cells
              </a>
            </td>
          <td>
            Cyril Y Ramathal, K. Saini, Zhouyang Lian, Christian Corona, T. Pham, Ryan Carelli, Stephane C. Boutet, Manisha Ray, Sunantha Sethuraman, Vivian Prindle, Evelyn Lattmann, Maria-Grazia Molvetti, A. Dzung, Mitchell P. Levesque, Matt Barnes, Andreja Jovic
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59376f615ed819023304612aaccafe8e9882b1f" target='_blank'>
              Mapping T cell dynamics to molecular profiles through behavior-guided transcriptomics.
              </a>
            </td>
          <td>
            A. K. Wezenaar, U. Pandey, F. Keramati, M. Hernandez-Roca, P. Brazda, M. Barrera Román, A. Cleven, F. Karaiskaki, T. Aarts-Riemens, S. de Blank, P. Hernandez-Lopez, S. Heijhuurs, A. Alemany, J. Kuball, Z. Sebestyen, J. F. Dekkers, H. Stunnenberg, M. Alieva, A. Rios
          </td>
          <td>2025-02-10</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/520e7a98e4f661979d2c0ecb98d246afd9f267aa" target='_blank'>
              Translating genetics into tissue: inflammatory cytokine-producing TAMs and PD-L1 tumor expression as poor prognosis factors in cutaneous melanoma
              </a>
            </td>
          <td>
            Celia Barrio-Alonso, Alicia Nieto-Valle, Lucía Barandalla-Revilla, J. Avilés-Izquierdo, V. Parra-Blanco, Paloma Sánchez-Mateos, Rafael Samaniego
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96305f2e853a0fad4ae17b8dba8c38f0edd28b14" target='_blank'>
              Integrated analysis of single-cell and bulk transcriptomes uncovers clinically relevant molecular subtypes in human prostate cancer.
              </a>
            </td>
          <td>
            Tao Ding, Lina He, Guowen Lin, Lei Xu, Yanjun Zhu, Xinan Wang, Xuefei Liu, Jianming Guo, Fanghong Lei, Zhixiang Zuo, Jianghua Zheng
          </td>
          <td>2025-01-30</td>
          <td>Chinese journal of cancer research = Chung-kuo yen cheng yen chiu</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary This study investigates metastatic melanoma through an in-depth analysis of molecular and genetic characteristics of tumor samples. Five distinct melanoma subtypes were identified based on the protein expression profiles within the tumors and their surrounding microenvironments. Our findings highlight the role of specific mutations, particularly in the BRAF gene, which can affect patient survival, with some patients showing better outcomes due to a particular immune response triggered by the mutation. Additionally, our study uncovered genetic changes in melanoma tumors that could support the development of treatments that target the immune system. The study shows that proteins linked to survival are distributed in complex patterns in both primary tumors and metastases, offering deeper insights into the molecular heterogeneity of melanoma. By combining detailed molecular analysis with clinical and tissue examination, this research provides new insights into melanoma biology and offers guidance for improving treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557557036af2f6be01a0b554ada4c910b8468881" target='_blank'>
              Proteogenomic Profiling of Treatment-Naïve Metastatic Malignant Melanoma
              </a>
            </td>
          <td>
            M. Kuras, L. Betancourt, R. Hong, L. Szadai, Jimmy Rodriguez, Peter Horvatovich, I. Pla, Jonatan Eriksson, B. Szeitz, Bartłomiej Deszcz, Charlotte Welinder, Y. Sugihara, Henrik Ekedahl, Bo Baldetorp, C. Ingvar, L. Lundgren, H. Lindberg, Henriett Oskolás, Z. Horváth, M. Rezeli, Jeovanis Gil, R. Appelqvist, L. Kemény, Johan Malm, Aniel Sanchez, A. M. Szasz, Krzysztof Pawłowski, E. Wieslander, David Fenyö, I. Németh, Gyorgy Marko-Varga
          </td>
          <td>2025-02-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Human genome sequencing efforts in healthy and diseased individuals continue to identify a broad spectrum of genetic variants associated with predisposition, progression, and therapeutic outcomes for diseases like cancer1–6. Insights derived from these studies have significant potential to guide clinical diagnoses and treatment decisions; however, the relative importance and functional impact of most genetic variants remain poorly understood. Precision genome editing technologies like base and prime editing can be used to systematically engineer and interrogate diverse types of endogenous genetic variants in their native context7–9. We and others have recently developed and applied scalable sensor-based screening approaches to engineer and measure the phenotypes produced by thousands of endogenous mutations in vitro10–12. However, the impact of most genetic variants in the physiological in vivo setting, including contextual differences depending on the tissue or microenvironment, remains unexplored. Here, we integrate new cross-species base editing sensor libraries with syngeneic cancer mouse models to develop a multiplexed in vivo platform for systematic functional analysis of endogenous genetic variants in primary and disseminated malignancies. We used this platform to screen 13,840 guide RNAs designed to engineer 7,783 human cancer-associated mutations mapping to 489 endogenous protein-coding genes, allowing us to construct a rich compendium of putative functional interactions between genes, mutations, and physiological contexts. Our findings suggest that the physiological in vivo environment and cellular organotropism are important contextual determinants of specific gene-variant phenotypes. We also show that many mutations and their in vivo effects fail to be detected with standard CRISPR-Cas9 nuclease approaches and often produce discordant phenotypes, potentially due to site-specific amino acid selection- or separation-of-function mechanisms. This versatile platform could be deployed to investigate how genetic variation impacts diverse in vivo phenotypes associated with cancer and other genetic diseases, as well as identify new potential therapeutic avenues to treat human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f71f3527a2a35f380d7482a2555f7e6eeef4ab" target='_blank'>
              Multiplexed in vivo base editing identifies functional gene-variant-context interactions
              </a>
            </td>
          <td>
            Jonuelle Acosta, Grace A. Johnson, Samuel I. Gould, Kexin Dong, Yovel Lendner, Diego Detrés, Ondine Atwa, Jari Bulkens, Samuel Gruber, Manuel E. Contreras, Alexandra N. Wuest, Varun K. Narendra, Michael T. Hemann, Francisco J. Sánchez-Rivera
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8af0488d6a2ccbb080a6ebfdc837df22774e9cf9" target='_blank'>
              Integrated interactome, proteomic and functional analyses reveal molecular pathways driving L1TD1-induced aggressiveness in CNS embryonal tumor cells.
              </a>
            </td>
          <td>
            T. Mitsugi, Matt Sherwood, Elisa Helena Farias Jandrey, Amanda Faria Assoni, Joseph Bell, Brandon Coke, Paul Skipp, Rob M. Ewing, Oswaldo Keith Okamoto
          </td>
          <td>2025-02-06</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f69de6f588f5dc88e965e9e468e394e888395b6" target='_blank'>
              Single-cell and spatial transcriptomics reveal a tumor-associated macrophage subpopulation that mediates prostate cancer progression and metastasis.
              </a>
            </td>
          <td>
            Shenglin Mei, Hanyu Zhang, Taghreed Hirz, N. Jeffries, Yanxin Xu, Ninib Baryawno, Shulin Wu, , Akash Patnaik, P. Saylor, David B. Sykes, Douglas M Dahl
          </td>
          <td>2025-03-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Transitions between subsets of differentiating hematopoietic cells are widely regarded as unidirectional in vivo. Here, we introduce clonal phylogenetic tracer (CP-tracer) that sequentially introduces genetic barcodes, enabling high-resolution analysis of ∼100,000 subclones derived from ∼500 individual hematopoietic stem cells (HSC). This revealed previously uncharacterized HSC functional subsets and identified bidirectional fate transitions between myeloid-biased and lineage-balanced HSC. Contrary to the prevailing view that the more self-renewing My-HSCs unidirectionally transition to balanced-HSCs, phylogenetic tracing revealed durable lineage bidirectionality with the transition favoring My-HSC accumulation over time1,2. Further, balanced-HSCs mature through distinct intermediates—My-HSCs and lymphoid-biased-HSCs—with lymphoid competence here shown by CRISPR/Cas9 screening to be dependent on the homeobox gene, Hhex. Hhex enables Ly-HSC differentiation, but its expression declines with age. These findings establish HSC plasticity and Hhex as a determinant of myeloid-lymphoid balance with each changing over time to favor the age-related myeloid bias of the elderly. Highlights Sequenctial introduction of DNA barcodes in vivo was developed to assess time dependent changes in cell fate. Clonal phylogenetic tracer (CP-tracer) enabled high-resolution phylogenetic analysis of ∼100,000 subclones derived from ∼500 individual hematopoietic stem cells (HSC). Bidirectional fate transitions between myeloid-biased haematopoietic stem cells (My-HSCs) and lineage-balanced haematopoietic stem cells (balanced-HSCs) were observed. Hhex was identified as a molecular driver of HSC lymphoid competence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/610d89d7916a0a478467cd537a729eff40e0674a" target='_blank'>
              Hematopoietic stem cells undergo bidirectional fate transitions in vivo
              </a>
            </td>
          <td>
            Tsuyoshi Fukushima, Trine A. Kristiansen, L. Wong, Samuel Keyes, Yosuke Tanaka, M. Mazzola, Ting Zhao, Lingli He, Masaki Yagi, Konrad Hochedlinger, Satoshi Yamazaki, Ruslan Sadreyev, David T Scadden
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/562ecaa8f1395c90f98e5b2a88e96773ab95655e" target='_blank'>
              CD44+ cells enhance pro-tumor stroma in the spatial landscape of colorectal cancer leading edge.
              </a>
            </td>
          <td>
            F. Tang, Yong-Gui Zhu, Jia Shen, Bowen Yuan, Xiang He, Yuxi Tian, Liang Weng, Lunquan Sun
          </td>
          <td>2025-03-12</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd1fbf6a111cb790d6d05a256a03567072e5318c" target='_blank'>
              AI-Driven Variant Annotation for Precision Oncology in Breast Cancer
              </a>
            </td>
          <td>
            Kriti Shukla, Yue Wang, Philip M. Spanheimer, Elizabeth Brunk
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36c2c77f99d55aa9b36b1f371ad8b8cec2aac50" target='_blank'>
              Evolutionary fingerprints of epithelial-to-mesenchymal transition.
              </a>
            </td>
          <td>
            Luigi Perelli, Li Zhang, Sarah Mangiameli, Francesca Giannese, Krishnan K. Mahadevan, F. Peng, F. Citron, Hania Khan, Courtney Le, Enrico Gurreri, F. Carbone, Andrew J. C. Russell, M. Soeung, T. N. Lam, Sebastian Lundgren, Sujay Marisetty, Cihui Zhu, Desiree Catania, Alaa M T Mohamed, Ningping Feng, J. Augustine, A. Sgambato, G. Tortora, Giulio F. Draetta, Giovanni Tonon, Andrew Futreal, Virginia Giuliani, A. Carugo, A. Viale, Michael P. Kim, T. Heffernan, Linghua Wang, Raghu Kalluri, D. Cittaro, Fei Chen, G. Genovese
          </td>
          <td>2025-03-05</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33631c265f68921d79fbd004eb59ad3a5ea83b81" target='_blank'>
              Spatially mapping the tumour immune microenvironments of non-small cell lung cancer
              </a>
            </td>
          <td>
            Lysanne Desharnais, Mark Sorin, M. Rezanejad, Bridget Liu, E. Karimi, Aline Atallah, Anikka M Swaby, Miranda W. Yu, Samuel Doré, Saskia Hartner, B. Fiset, Yuhong Wei, Baharak Kadang, R. Rayes, Philippe Joubert, Sophie Camilleri-Broët, P. Fiset, D. Quail, Jonathan D Spicer, L. Walsh
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183356cbf79c71bfdd985789fe82d5c12c548853" target='_blank'>
              Gemistocytic tumor cells programmed for glial scarring characterize T cell confinement in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            L. van Hijfte, M. Geurts, I. D. de Heer, S. Ghisai, H. Balcioglu, Y. Hoogstrate, Wies R Vallentgoed, R. Head, Rosa Luning, Thierry van den Bosch, Bart A Westerman, Pieter Wesseling, J. Joyce, P. French, R. Debets
          </td>
          <td>2025-01-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Motivation Despite significant advances in spatial transcriptomics, the analysis of formalin-fixed paraffin-embedded (FFPE) tissues, which constitute most clinically available samples, remains challenging. Additionally, capturing both coding and noncoding RNAs in a spatial context poses significant challenges. We recently introduced Patho-DBiT, a technology designed to address these unmet needs. However, the marked differences between Patho-DBiT and existing spatial transcriptomics protocols necessitate specialized computational tools for comprehensive whole-transcriptome analysis in FFPE samples. Results Here, we present ASTRO, an automated pipeline developed to process spatial transcriptomics data. In addition to supporting standard datasets, ASTRO is optimized for whole-transcriptome analyses of FFPE samples, enabling the detection of various RNA species, including non-coding RNAs such as miRNAs. To compensate for the reduced RNA quality in FFPE tissues, ASTRO incorporates a specialized filtering step and optimizes spatial barcode calling, increasing the mapping rate. These optimizations allow ASTRO to spatially quantify coding and non-coding RNA species in the entire transcriptome and achieve robust performance in FFPE samples. Availability Codes are available at GitHub (https://github.com/gersteinlab/ASTRO).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d569a6b53e774b129f0d53a5ac8f2383815ea030" target='_blank'>
              ASTRO: Automated Spatial Whole-Transcriptome RNA-Expression Workflow
              </a>
            </td>
          <td>
            Dingyao Zhang, Zhiyuan Chu, Yiran Huo, Zhiliang Bai, Rong Fan, Jun Lu, Marc H. Gerstein
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell transcriptomics has revolutionized our understanding of neurodevelopmental cell identities, yet, predicting a cell type’s developmental state from its transcriptome remains a challenge. We perform a meta-analysis of developing human brain datasets comprising over 2.8 million cells, identifying both tissue-level and cell-autonomous predictors of developmental age. While tissue composition predicts age within individual studies, it fails to generalize, whereas specific cell type proportions reliably track developmental time across datasets. Training regularized regression models to infer cell-autonomous maturation, we find that a cell type-agnostic model achieves the highest accuracy (error = 2.6 weeks), robustly capturing developmental dynamics across diverse cell types and datasets. This model generalizes to human neural organoids, accurately predicting normal developmental trajectories (R = 0.91) and disease-induced shifts in vitro. Furthermore, it extends to the developing mouse brain, revealing an accelerated developmental tempo relative to humans. Our work provides a unified framework for comparing neurodevelopment across contexts, model systems, and species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd7e20b6112e2c0e549d107034486f23f4f7693" target='_blank'>
              Cell Type-Agnostic Transcriptomic Signatures Enable Uniform Comparisons of Neurodevelopment
              </a>
            </td>
          <td>
            Sridevi Venkatesan, Jonathan M. Werner, Yun Li, Jesse Gillis
          </td>
          <td>2025-02-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20c3f321acba286bf84a8f4af78044117acbf8c" target='_blank'>
              Abstract B105: Predicting spatial transcriptomics from histopathology with Path2Space: AI charts breast cancer patients tumor immune microenvironment on an unprecedented scale
              </a>
            </td>
          <td>
            Roshan Lodha, E. Shulman, Emma M. Campagnolo, Thomas Cantore, Tom Hu, M. Nasrallah, Danh-Tai Hoang, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0f6840421639f0eeeffb398c6fad844f8d444a6" target='_blank'>
              InSituPy – A framework for histology-guided, multi-sample analysis of single-cell spatial transcriptomics data
              </a>
            </td>
          <td>
            Johannes Wirth, Anna Chernysheva, Birthe Lemke, Isabel Giray, Aitana Egea Lavandera, K. Steiger
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cabdeb7c0a8725d1b43bd88dd9faad675928ee91" target='_blank'>
              Extraction of a stromal metastatic gene signature in breast cancer via spatial profiling
              </a>
            </td>
          <td>
            G. Bertolazzi, Valeria Cancila, Davide Vacca, Beatrice Belmonte, D. Lecis, Parsa Sirati Moghaddam, A. Di Napoli, M. P. Colombo, G. Pruneri, Giannino Del Sal, Giorgio Scita, Matteo Fassan, A. Vecchione, Silvio Bicciato, Claudio Tripodo
          </td>
          <td>2025-03-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Spatial transcriptomics has transformed our understanding of gene expression by preserving the spatial context within tissues. This review focuses on the application of spatial transcriptomics in human cardiac tissues, exploring current technologies with a focus on commercially available platforms. We also highlight key studies utilizing spatial transcriptomics to investigate cardiac development, electro-anatomy, immunology, and ischemic heart disease. These studies demonstrate how spatial transcriptomics can be used in conjunction with other omics technologies to provide a more comprehensive picture of human health and disease. Despite its transformative potential, spatial transcriptomics comes with several challenges that limit its widespread adoption and broader application. By addressing these limitations and fostering interdisciplinary collaboration, spatial transcriptomics has the potential to become an essential tool in cardiovascular research. We hope this review serves as a practical guide for researchers interested in adopting spatial transcriptomics, particularly those with limited prior experience, by providing insights into current technologies, applications, and considerations for successful implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8ad32890339541586531b7c09fb574e34d7781c" target='_blank'>
              Spatial Transcriptomics in Human Cardiac Tissue
              </a>
            </td>
          <td>
            Quynh Nguyen, Lin Wei Tung, Bruce Lin, Raam Sivakumar, F. Sar, G. Singhera, Ying Wang, Jeremy Parker, Stephane Le Bihan, Amrit Singh, Fabio M V Rossi, Colin Collins, Jamil Bashir, Zachary Laksman
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be893eec80147faf7d7cca1af35e6472e40e7cc" target='_blank'>
              Statistical identification of cell type-specific spatially variable genes in spatial transcriptomics
              </a>
            </td>
          <td>
            Lulu Shang, Peijun Wu, Xiang Zhou
          </td>
          <td>2025-01-26</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91820cda6d428b7441857912e71a4c3a105594b5" target='_blank'>
              Density and entropy of immune cells within the tumor microenvironment of primary tumors and matched brain metastases
              </a>
            </td>
          <td>
            M. Kleinberger, D. Cifci, C. Paiato, E. Tomasich, M. Mair, A. Steindl, Zoltán Spiró, Z. I. Carrero, L. Berchtold, J. Hainfellner, L. Müllauer, G. Heller, M. Preusser, Jakob N Kather, A. Berghoff
          </td>
          <td>2025-02-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Abstract T-Cell-derived extracellular vesicles (TcEVs) play key roles in immune regulation and tumor microenvironment modulation. However, the heterogeneity of TcEV remains poorly understood due to technical limitations of EV analysis and the lack of comprehensive data. To address this, we constructed TcEVdb, a comprehensive database that explores the expression and cluster of TcEV by the SEVtras method from T-cell single-cell RNA sequencing data. TcEVdb contains 277 265 EV droplets from 51 T-cell types across 221 samples from 21 projects, covering 9 tissue sources and 23 disease conditions. The database provides two main functional modules. The Browse module enables users to investigate EV secretion activity indices across samples, visualize TcEV clusters, analyze differentially expressed genes (DEGs) and pathway enrichment in TcEV subpopulations, and compare TcEV transcriptomes with their cellular origins. The Search module allows users to query specific genes across all datasets and visualize their expression distribution. Furthermore, our analysis of TcEV in diffuse large B-cell lymphoma revealed increased EV secretion in CD4+ T exhausted cells compared to healthy controls. Subsequent analyses identified distinct droplet clusters with differential expression genes, including clusters enriched for genes associated with cell motility and mitochondrial function. Overall, TcEVdb serves as a comprehensive resource for exploring the transcriptome of TcEV, which will contribute to advancements in EV-based diagnostics and therapeutics across a wide range of diseases. Database URL: https://guolab.wchscu.cn/TcEVdb">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39cae304e2b4aafe6ad38b975f3d2d2df66e8a5c" target='_blank'>
              TcEVdb: a database for T-cell-derived small extracellular vesicles from single-cell transcriptomes
              </a>
            </td>
          <td>
            Tao Luo, Wen-Kang Shen, Chu-Yu Zhang, Dan-Dan Song, Xiuqing Zhang, An-Yuan Guo, Qian Lei
          </td>
          <td>2025-02-28</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2ac9b04d746f5cf36d236d7ac9a449435141cc" target='_blank'>
              Spatial transcriptomics reveals prognostically LYZ+ fibroblasts and colocalization with FN1+ macrophages in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            L. Dai, N. Lou, Liling Huang, Lin Li, Le Tang, Yuankai Shi, Xiaohong Han
          </td>
          <td>2025-02-25</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efd1362638fd8682aac622f2d9c6d7d60b693242" target='_blank'>
              Single-cell and spatial transcriptome profiling reveal CTHRC1+ fibroblasts promote EMT through WNT5A signaling in colorectal cancer
              </a>
            </td>
          <td>
            Yunfei Lu, Yang Chen, Zhenling Wang, Hengyang Shen, Lei Xu, Changzhi Huang, Ying Tong, Yu Shao, Hongqiang Zhang, Zan Fu
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8b3640c44bd80cadd9a05e0404e0638d900b1d2" target='_blank'>
              Double–Multiplex Immunostainings for Immune Profiling of Invasive Breast Carcinoma: Emerging Novel Immune-Based Biomarkers
              </a>
            </td>
          <td>
            Sofia D P Theodorou, Konstantinos Ntostoglou, I. Nikas, Dimitrios Goutas, V. Georgoulias, Christos Kittas, Ioannis S. Pateras
          </td>
          <td>2025-03-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Peritoneal carcinomatosis is a common yet deadly manifestation of gastrointestinal cancers, with few effective treatments. To identify targetable determinants of peritoneal metastasis, we focused on appendiceal adenocarcinoma (AC), a gastrointestinal cancer that metastasizes almost exclusively to the peritoneum. Current treatments are extrapolated from colorectal cancer (CRC), yet AC has distinct genomic alterations, mucinous morphology and peritoneum restricted metastatic pattern. Further, no stable preclinical models of AC exist, limiting drug discovery and representing an unmet clinical need. We establish a first-in-class stable biobank of 16 long-term cultured AC patient-derived organoids (PDOs), including 3 matched, simultaneously resected primary AC-peritoneal carcinomatosis (AC-PC) pairs. By enriching for cancer cells, AC PDOs enable accurate genomic characterization relative to paucicellular AC tissue. We establish an organoid orthotopic intraperitoneal xenograft model that recapitulates diffuse peritoneal carcinomatosis and show that PC-organoids retain increased metastatic capacity, decreased growth factor dependency and sensitivity to standard of care chemotherapy relative to matched primary AC organoids. Single cell profiling of AC-PC pairs reveals dedifferentiation from mucinous differentiated states in primary AC into intestinal stem cell and fetal progenitor states in AC-PC, with upregulation of oncogenic signaling pathways. Through hypothesis-driven drug testing, we identify KRASMULTI-ON inhibitor RMC-7977 and Wnt-targeting tyrosine kinase inhibitor WNTinib as novel, clinically actionable strategies to target AC-PC more effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3baef141d668078d350e52fdb87357fa4d118d" target='_blank'>
              Paired primary-metastasis patient-derived organoids and mouse models identify phenotypic evolution and druggable dependencies of peritoneal metastasis from appendiceal cancer
              </a>
            </td>
          <td>
            Ahmed Mahmoud, Philip H. Choi, Christine Sukhwa, Jura Pintar, H. Walch, Nan Zhao, J. Bermeo, Sebastian K Chung, Manisha Raghavan, Samhita Bapat, Qingwen Jiang, G. Karagkounis, Julia Meredith, Michael Giarrizzo, C. Firat, A. Cercek, Michael B Foote, Nikolaus Schultz, W. Chatila, G. Nash, J. Shia, F. Sánchez-Vega, Steven Larson, Arvin C. Dar, Neal Rosen, K. Ganesh
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Simple Summary Pancreatic cancer has a low survival rate with limited treatment options. Perineural invasion (PNI), where cancer cells infiltrate the nerves in the pancreas, is associated with a poor prognosis and local recurrence. Pancreatic cancer is one of the most painful cancers, with PNI contributing to the pain experienced by patients. However, the mechanisms involved in the cancer invasion of nerves are not completely understood. Here, we used spatial transcriptomics to study the expression of both cancer and nerve cells in microscopic regions of PNI and compared it to regions without PNI evidence to better understand the mechanisms involved in PNI. We identified novel mechanisms that may impact cancer growth, PNI and pain experienced by patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1818d7eac1f6857a9dc4b88e4ee2f868f4f36db6" target='_blank'>
              Spatial Transcriptomics Reveals Novel Mechanisms Involved in Perineural Invasion in Pancreatic Ductal Adenocarcinomas
              </a>
            </td>
          <td>
            V. Lakis, Noni L Chan, Ruth J Lyons, Nicola Blackburn, Tam Hong Nguyen, Crystal Chang, Andrew Masel, Nicholas West, Glen M. Boyle, A. Patch, Anthony J. Gill, K. Nones
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Spatial transcriptomics has emerged as an invaluable tool, helping to reveal molecular status within complex tissues. Nonetheless, these techniques have a crucial challenge: the absence of single-cell resolution, resulting in the observation of multiple cells in each spatial spot. While reference-based deconvolution methods have aimed to solve the challenge, their effectiveness is contingent upon the quality and availability of single-cell RNA (scRNA) datasets, which may not always be accessible or comprehensive. In response to these constraints, our study introduces STsisal, a reference-free deconvolution method meticulously crafted for the intricacies of spatial transcriptomics (ST) data. STsisal leverages a novel approach that integrates marker gene selection, mixing ratio decomposition, and cell type characteristic matrix analysis to discern distinct cell types with precision and efficiency within complex tissues. The main idea of our method is its adaptation of the SISAL algorithm, which expertly disentangles the ratio matrix, facilitating the identification of simplices within the ST data. STsisal offers a robust means to unveil the intricate composition of cell types in spatially resolved transcriptomic data. To verify the efficacy of STsisal, we conducted extensive simulations and applied the method to real data, comparing its performance against existing techniques. Our findings highlight the superiority of STsisal, underscoring its utility in capturing the cell composition within complex tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/537297555122a8c531c29f52cfb730430bee08e6" target='_blank'>
              STsisal: a reference-free deconvolution pipeline for spatial transcriptomics data
              </a>
            </td>
          <td>
            Yinghao Fu, Leqi Tian, Weiwei Zhang
          </td>
          <td>2025-03-03</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f8ba9e3fcdae65a1005fa8694699f4a2467c8be" target='_blank'>
              Intratumour oxidative hotspots provide a niche for cancer cell dissemination.
              </a>
            </td>
          <td>
            Yoshifumi Ueda, Shigeki Kiyonaka, L. Selfors, Keisuke Inoue, Hiroshi Harada, Tomohiro Doura, Kunishige Onuma, Makoto Uchiyama, Ryuhei Kurogi, Yuji Yamada, Jiacheng H Sun, Reiko Sakaguchi, Yuki Tado, Haruki Omatsu, Harufumi Suzuki, M. Aoun, Takahiro Nakayama, Taketoshi Kajimoto, Tetsuya Yano, Rikard Holmdahl, I. Hamachi, Masahiro Inoue, Yasuo Mori, Nobuaki Takahashi
          </td>
          <td>2025-02-21</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76368b27f3c58a581bef9e47250d9059ca708920" target='_blank'>
              Multiplexed spatial mapping of chromatin features, transcriptome and proteins in tissues.
              </a>
            </td>
          <td>
            Pengfei Guo, Liran Mao, Yufan Chen, Chin Nien Lee, Angelysia Cardilla, Mingyao Li, Marek Bartošovič, Yanxiang Deng
          </td>
          <td>2025-01-27</td>
          <td>Nature methods</td>
          <td>2</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38354bc1c5ef98aa13d3c5253963e1d30d5518bf" target='_blank'>
              Mesothelioma cell heterogeneity identified by single cell RNA sequencing
              </a>
            </td>
          <td>
            Licun Wu, Zongjie Wang, Amin Zia, S. Kelley, Marc de Perrot
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e969ce26ddc70310682b3f3db6a8aee97b5128" target='_blank'>
              Integrative transcriptome profiling elucidates molecular and immunovascular characteristics of macrotrabecular hepatocellular carcinoma.
              </a>
            </td>
          <td>
            T. Taniai, S. Shimada, Y. Akiyama, Megumi Hatano, Koya Yasukawa, Yosuke Igarashi, Shu Tsukihara, Yoshiaki Tanji, K. Kodera, Kohei Okazaki, K. Haruki, Atsushi Nara, Kohei Yagi, K. Akahoshi, D. Ban, Yasuhiro Asahina, Toru Ikegami, Minoru Tanabe, Shinji Tanaka
          </td>
          <td>2025-02-26</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20f7ade1ee5b9d0bf71b29249fe64f13f03ddc98" target='_blank'>
              Characterization of single neurons reprogrammed by pancreatic cancer.
              </a>
            </td>
          <td>
            Vera Thiel, Simon Renders, Jasper Panten, Nicolas Dross, K. Bauer, D. Azorin, Vanessa Henriques, V. Vogel, C. Klein, Aino-Maija Leppä, Isabel Barriuso Ortega, J. Schwickert, I. Ourailidis, Julian Mochayedi, Jan-Philipp Mallm, Carsten Müller-Tidow, Hannah Monyer, J. Neoptolemos, T. Hackert, O. Stegle, D. T. Odom, R. Offringa, A. Stenzinger, Frank Winkler, M. Sprick, A. Trumpp
          </td>
          <td>2025-02-17</td>
          <td>Nature</td>
          <td>1</td>
          <td>85</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d1d629c974d4b9e986ad888f6cfe60cc9b53f8" target='_blank'>
              Splicing diversity enhances the molecular classification of pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Yue Huang, Jing Guo, Xueshuai Han, Yang Zhao, Xuejing Li, Peiqi Xing, Yulou Liu, Yingxuan Sun, Song Wu, Xuan Lv, Lei Zhou, Yazhuo Zhang, Chuzhong Li, Weiyan Xie, Zhaoqi Liu
          </td>
          <td>2025-02-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most prevalent and aggressive histological subtype of ovarian cancer and often presents with metastatic disease. The drivers of metastasis in HGSOC remain enigmatic. APOBEC3A (A3A), an enzyme that generates mutations across various cancers, has been proposed as a mediator of tumor heterogeneity and disease progression. However, the role of A3A in HGSOC has not been explored. We observed an association between high levels of APOBEC3-mediated mutagenesis and poor overall survival in primary HGSOC. We experimentally addressed this correlation by modeling A3A expression in HGSOC, and this resulted in increased metastatic behavior of HGSOC cells in culture and distant metastatic spread in vivo, which was dependent on catalytic activity of A3A. A3A activity in both primary and cultured HGSOC cells yielded consistent alterations in expression of epithelial-mesenchymal transition (EMT) genes resulting in hybrid EMT and mesenchymal signatures, providing a mechanism for their increased metastatic potential. Inhibition of key EMT factors TWIST1 and IL-6 resulted in mitigation of A3A-dependent metastatic phenotypes. Our findings define the prevalence of A3A mutagenesis in HGSOC and implicate A3A as a driver of HGSOC metastasis via EMT, underscoring its clinical relevance as a potential prognostic biomarker. Our study lays the groundwork for the development of targeted therapies aimed at mitigating the deleterious effect of A3A-driven EMT in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5397c063e336101c76892114dddd3f14218105fb" target='_blank'>
              APOBEC3A drives ovarian cancer metastasis by altering epithelial-mesenchymal transition.
              </a>
            </td>
          <td>
            J. Devenport, Thi Tran, Brooke R. Harris, Dylan F. Fingerman, Rachel A. DeWeerd, Lojain H Elkhidir, Danielle LaVigne, Katherine C Fuh, Lulu Sun, Jeffrey J Bednarski, Ronny Drapkin, Mary M Mullen, Abby M. Green
          </td>
          <td>2025-03-10</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f973214ce129b073da8e0d3c1c3cc399902a2e8" target='_blank'>
              Spatial single-cell proteomics landscape decodes the tumor microenvironmental ecosystem of intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Libing Hong, Jie Mei, Xuqi Sun, Yifan Wu, Zhen Dong, Yuzhi Jin, Liaoliao Gao, Jinlin Cheng, Weihong Tian, Chuan Liu, Bin Li, Pingping Hu, Lulu Liu, Shan Xin, Xiaomeng Dai, Peng Zhao, Rongping Guo, Minshan Chen, Jingping Yun, Bo Lin, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-02-25</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Summary Spatial and temporal intra-tumor heterogeneity drives tumor evolution and therapy resistance. Existing visualization tools often fail to capture both dimensions simultaneously. To address this, we developed Jellyfish, a tool that integrates phylogenetic and sample trees into a single plot, providing a holistic view of tumor evolution and capturing both spatial and temporal evolution. Available as a JavaScript library and R package, Jellyfish generates interactive visualizations from tumor phylogeny and clonal composition data. We demonstrate its ability to visualize complex subclonal dynamics using data from ovarian high-grade serous carcinoma. Availability and implementation Jellyfish is freely available with MIT license at https://github.com/HautaniemiLab/jellyfish (JavaScript library) and https://github.com/HautaniemiLab/jellyfisher (R package).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0210199a3e67c966717695406003ace2c046fe" target='_blank'>
              Jellyfish: integrative visualization of spatio-temporal tumor evolution and clonal dynamics
              </a>
            </td>
          <td>
            K. Lavikka, Altti Ilari Maarala, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-02-25</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7453b499aa0f4f56b20054ed92fb6b1a0f0d81c" target='_blank'>
              Analysis of thyroid carcinoma composition and spatial architecture in the progression of dedifferentiation, lymphatic metastasis, and gastric metastasis
              </a>
            </td>
          <td>
            Di Wang, Ruichun Lu, Fenglian Yan, Yansong Lin, Hao Wang, Huabao Xiong
          </td>
          <td>2025-02-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumor cells to adapt to and exploit their microenvironment for sustained growth. The liver is a common site of metastasis, but the interactions between tumor cells and hepatocytes remain poorly understood. In the context of liver metastasis, these interactions play a crucial role in promoting tumor survival and progression. This study leverages multiomics coverage of the microenvironment via liquid chromatography and high-resolution, high-mass-accuracy mass spectrometry-based untargeted metabolomics, 13C-stable isotope tracing, and RNA sequencing to uncover the metabolic impact of co-localized primary hepatocytes and a colon adenocarcinoma cell line, SW480, using a 2D co-culture model. Metabolic profiling revealed disrupted Warburg metabolism with an 80% decrease in glucose consumption and 94% decrease in lactate production by hepatocyte–SW480 co-cultures relative to SW480 control cultures. Decreased glucose consumption was coupled with alterations in glutamine and ketone body metabolism, suggesting a possible fuel switch upon co-culturing. Further, integrated multiomics analysis indicates that disruptions in metabolic pathways, including nucleoside biosynthesis, amino acids, and TCA cycle, correlate with altered SW480 transcriptional profiles and highlight the importance of redox homeostasis in tumor adaptation. Finally, these findings were replicated in three-dimensional microtissue organoids. Taken together, these studies support a bioinformatic approach to study metabolic crosstalk and discovery of potential therapeutic targets in preclinical models of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d939bb36944c04a3083bae73bc1b5307c8fa9271" target='_blank'>
              Deciphering Colorectal Cancer–Hepatocyte Interactions: A Multiomics Platform for Interrogation of Metabolic Crosstalk in the Liver–Tumor Microenvironment
              </a>
            </td>
          <td>
            Alisa B. Nelson, Lyndsay E Reese, Elizabeth Rono, Eric D. Queathem, Yinjie Qiu, Braedan M. McCluskey, Alexandra Crampton, Eric Conniff, Katherine A Cummins, Ella Boytim, Senali Dansou, Justin H. Hwang, S. Safo, Patrycja Puchalska, David K. Wood, K. Schwertfeger, Peter A. Crawford
          </td>
          <td>2025-02-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The tumor microenvironment plays a pivotal role in cancer progression and therapeutic resistance, with tumor-associated macrophages significantly influencing immune suppression and tumor growth. Colorectal cancers (CRC) classified as Consensus Molecular Subtype 4 (CMS4) and triple-negative breast cancers subsets are particularly characterized by a mesenchymal phenotype, immune exclusion, and extensive macrophage infiltration. This study aimed to investigate how targeting cancer cell stemness with specific inhibitors could modulate macrophage polarization in CRC in vitro and breast cancer in vivo, potentially shifting the immune balance from pro-tumor M2-like to anti-tumor M1-like macrophages. We used four stemness inhibitors—salinomycin, SB-431542, JIB-04, and napabucasin—each targeting different pathways (Wnt/β-catenin, TGF-β, histone demethylation, and STAT3, respectively), to evaluate their effects on CMS4 CRC cell lines (HCT116 and SW620) and human peripheral blood-derived macrophages in an indirect co-culture model. Our results showed that CMS4 CRC cell lines induced distinct macrophage polarization patterns, with HCT116 promoting M2-like macrophages and SW620 leaning toward M1-like profile. Notably, the combination of stemness inhibitors reduced stemness markers (CD133, CD44) in colorectal cancer cells and shifted macrophage polarization toward an M1-like phenotype, particularly in co-culture with HCT116. In vivo studies using the syngeneic immunocompetent EO771 breast cancer mouse model demonstrated that combination of stemness inhibitors increased the M1/M2 macrophage ratio.Our study highlights the dual potential of stemness inhibitors to target both cancer cells and the immune microenvironment. These findings offer promising strategies for enhancing favorable immunomodulation in mesenchymal-like colorectal tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/211f6363e023c7088c47db3138c8836f0d0dca08" target='_blank'>
              Targeting stemness pathways modulates macrophage polarization and reprograms the tumor microenvironment
              </a>
            </td>
          <td>
            Austeja Butkute, Marius Baltramonaitis, Simona Malmige, A. Darinskas, V. Pašukonienė, A. Mlynska
          </td>
          <td>2025-03-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61e79fb6328397b7897b70522e3b8824a0291600" target='_blank'>
              Spatial recognition and semi-quantification of epigenetic events in pancreatic cancer subtypes with multiplexed molecular imaging and machine learning
              </a>
            </td>
          <td>
            Krzysztof Szymoński, Natalia Janiszewska, K. Sofińska, Katarzyna Skirlińska-Nosek, Dawid Lupa, Michał Czaja, Marta Urbańska, Katarzyna Jurkowska, Kamila Konik, Marta Olszewska, Dariusz Adamek, Kamil Awsiuk, E. Lipiec
          </td>
          <td>2025-02-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) technologies have revolutionized our understanding of cellular ecosystems. However, these technologies face challenges such as sparse gene signals and limited gene detection capacities, which hinder their ability to fully capture comprehensive spatial gene expression profiles. To address these limitations, we propose leveraging single-cell RNA sequencing (scRNA-seq), which provides comprehensive gene expression data but lacks spatial context, to enrich ST profiles. Herein, we introduce SpaIM, an innovative style transfer learning model that utilizes scRNA-seq information to predict unmeasured gene expressions in ST data, thereby improving gene coverage and expressions. SpaIM segregates scRNA-seq and ST data into data-agnostic contents and data-specific styles, with the contents capture the commonalities between the two data types, while the styles highlight their unique differences. By integrating the strengths of scRNA-seq and ST, SpaIM overcomes data sparsity and limited gene coverage issues, making significant advancements over 12 existing methods. This improvement is demonstrated across 53 diverse ST datasets, spanning sequencing- and imaging-based spatial technologies in various tissue types. Additionally, SpaIM enhances downstream analyses, including the detection of ligand-receptor interactions, spatial domain characterization, and identification of differentially expressed genes. Released as open-source software, SpaIM increases accessibility for spatial transcriptomics analysis. In summary, SpaIM represents a pioneering approach to enrich spatial transcriptomics using scRNA-seq data, enabling precise gene expression imputation and advancing the field of spatial transcriptomics research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98a9f336c4dffd59ace975ec6b6a1760203849d4" target='_blank'>
              SpaIM: Single-cell Spatial Transcriptomics Imputation via Style Transfer
              </a>
            </td>
          <td>
            Bo Li, Ziyang Tang, Aishwarya Budhkar, Xiang Liu, Tonglin Zhang, Baijian Yang, Jing Su, Qianqian Song
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="A long-sought goal of cancer immunotherapy is to mass-produce T cells that specifically target tumor neoantigens. One decisive challenge is the identification of neoantigens derived from cancer driver genes. Here, we identify T cells that recognize the NSCLC-associated EGFR C797S mutation, which confers resistance to current inhibitors and is linked to poor prognosis. To overcome limitations in T cell availability, we reprogrammed EGFR C797S-specific T cells into induced pluripotent stem cells (iPSCs) and re-differentiated them into CD8 T cells. These iPSC-derived T cells specifically recognized the EGFR C797S mutation and effectively killed cancer cells expressing this mutation. Our findings underscore the potential of targeting driver mutation-derived neoantigens for immunotherapy and demonstrate that iPSC-derived T cells can mediate antitumor effects. Collectively, this approach combining neoantigen identification with T cell reprogramming may offer a promising strategy for targeting drug-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d765ea0e5a654cde03906984170408ff216e1ed1" target='_blank'>
              Development of iPSC-Derived T Cells Targeting EGFR Neoantigens in Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Kouta Niizuma, T. Nishimura, Jonathan Villanueva, Laura Amaya, J. Fowler, Taichi Isobe, Yusuke Nakauchi, Brandon Saavedra, Haojun Xu, Mahito Nakanishi, Adam C. Wilkinson, Kyle M. Loh, Joseph B Shrager, Hiromitsu Nakauchi
          </td>
          <td>2025-02-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cellular interactions form the fundamental/core circuits that drive development, physiology, and disease within tissues. Advances in spatial genomics (SG) and artificial intelligence (AI) offer unprecedented opportunities to computationally analyze and predict the behavior of cell intricate networks, and to identify interactions that drive disease states. However, challenges arise in both methodology and scalability: (i) how to computationally characterize complicated cellular interactions of multi-scale nature where chemical genes/circuits in individual cells process information and drive interactions among large numbers of diverse cell types, and (ii) how to scale up the pipeline to accommodate the increasing volumes of SG data that map transcriptome-scale gene expression and spatial proximity across millions of cells. In this paper, we introduce the Cellular Interaction Foundation Model (CI-FM), an AI foundation model functioning to analyze and simulate cellular interactions within living tissues. In the CI-FM pipeline, we explicitly capture and embed cellular interactions within microenvironments by leveraging the powerful and scalable geometric graph neural network model, and optimize the characterization of cellular interactions with a novel self-supervised learning objective – we train it to infer gene expressions of cells based upon their interacting microenvironment. As a result, we construct CI-FM with 100 million parameters by consuming SG data of 23 million cells. Our benchmarking experiments show CI-FM effectively infers gene expressions conditional on the microenvironmental contexts: we achieve a high correlation and a low mismatch error (MSE of 1.1% relative to the square median expression), with 79.4% of cells on average being annotated as the similar cell type based on their predicted and actual expressions. We demonstrate the downstream utility of CI-FM by: (i) applying CI-FM to embed tumor samples to capture cellular interactions within tumor microenvironments (ROC-AUC score of 0.76 on classifying sample conditions via linear probing on embeddings), and identifying shared signatures across samples; and (ii) using CI-FM to simulate changes in microenvironmental composition in response to T cell infiltration, which highlights how CI-FM can be leveraged to model cellular responses to tissue perturbations – an essential step toward constructing “AI virtual tissues”. Our model is open source and publicly accessible at https://huggingface.co/ynyou/CIFM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc301670fcf5e5e9540479c78aa400e91f732606" target='_blank'>
              Building Foundation Models to Characterize Cellular Interactions via Geometric Self-Supervised Learning on Spatial Genomics
              </a>
            </td>
          <td>
            Yuning You, Zitong Jerry Wang, Kevin Fleisher, Rex Liu, Matt Thomson
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell proteomics is an emerging field with significant potential to characterize heterogeneity within biological tissues. It offers complementary insights to single-cell transcriptomics by revealing unbiased proteomic changes downstream of the transcriptome. Recent advancements have focused on enhancing proteome coverage and depth, mostly in cultured cell lines, and a few recent studies have explored the potential of analyzing tissue micro-samples but were limited to homogenous peripheral tissues. In this current work, we utilize the power of spatial single cell-proteomics through immunostaining-guided laser capture microdissection (LCM) coupled with LC-MS to investigate the heterogenous central nervous system. We used this method to compare neuronal populations from cortex and substantia nigra, two brain regions associated with motor and cognitive function and various neurological disorders. Moreover, we used the technique to understand the neuroimmune changes associated with stab wound injury. Finally, we focus our application on the peripheral nervous system, where we compare the proteome of the myenteric plexus cell ganglion to the nerve bundle. This study demonstrates the utility of spatial single-cell proteomics in neuroscience research toward understanding fundamental biology and the molecular drivers of neurological conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e060f62a68156851b45076f66f6e5883e7b7f674" target='_blank'>
              Molecularly-guided spatial proteomics captures single-cell identity and heterogeneity of the nervous system
              </a>
            </td>
          <td>
            Sayan Dutta, Marion Pang, G. Coughlin, Sirisha Gudavalli, Michael L Roukes, Tsui-Fen Chou, V. Gradinaru
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Despite significant advancements in identifying novel therapeutic targets and compounds, cancer stem cells (CSCs) remain pivotal in driving therapeutic resistance and tumor progression in gastric cancer (GC). High-resolution knowledge of the transcriptional programs underlying the role of CSC niche in driving tumor stemness and progression is still lacking. Herein, spatial and single-cell RNA sequencing of 32 human gastric mucosa tissues at various stages of malignancy, illuminating the phenotypic plasticity of tumor epithelium and transcriptional trajectory from mature gastric chief cells to the CSC state, which is associated with activation of EGFR and WNT signaling pathways, is conducted. Moreover, the CSCs interact with not only the immunosuppressive CXCL13+ T cells and CCL18+ M2 macrophages to evade immune surveillance, but also the inflammatory cancer-associated fibroblasts (iCAFs) to promote tumorigenesis and maintain stemness, which construct the CSC niche leading to inferior prognosis. Notably, it is uncovered that amphiregulin (AREG) derived from iCAFs promotes tumor stemness by upregulating the expression of SOX9 in tumor cells, and contributes to drug resistance via the AREG-ERBB2 axis. This study provides valuable insight into the characteristics of CSC niche in driving tumor stemness and progression, offering novel perspective for designing effective strategies to overcome GC therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0d6cf84d15c0f17d3af8a79006d512aec0aa37" target='_blank'>
              Dissecting the Spatial and Single-Cell Transcriptomic Architecture of Cancer Stem Cell Niche Driving Tumor Progression in Gastric Cancer.
              </a>
            </td>
          <td>
            Guangyu Zhang, Xin Zhang, Wenting Pan, Xizhao Chen, Lingfei Wan, Chunjie Liu, Yuting Yong, Yue Zhao, Shuli Sang, Lihua Zhang, Sheng Yao, Yushu Guo, Mingmei Wang, Xinhui Wang, Guangdun Peng, Xinglong Yan, Yanchun Wang, Min Zhang
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb3b0d5b454cf059f699fd40f05952bb5941c97" target='_blank'>
              Unveiling spatial heterogeneity in medulloblastoma: a multi-omics analysis of cellular state and geographical organization.
              </a>
            </td>
          <td>
            Jiankang Li, Hailong Liu, Ziwei Wang, Jiao Zhang, Xuan Chen, Craig Daniels, Xiaochong Wu, Olivier Saulnier, Hiromichi Suzuki, Pasqualino de Antonellis, Alexandra Rasnitsyn, Winnie W. Ong, Evan Y Wang, Liam D Hendrikse, Yu Su, Yu Tian, Dongming Han, Ruohan Wang, Jialin Mo, Fei Liu, Kaiwen Deng, Dongyang Wang, Zhaoyang Feng, Yifei Jiang, Yanong Li, Yuting Ma, Zijia Liu, Meiyu Li, Peiyi Tian, Yanfeng Shi, Yong Jiang, Tao Yang, Shouwei Li, Jianfeng Liang, Jingchuan Wu, Ying Wang, Wanjing Zou, Yina Jiang, Lusheng Wang, Fang Chen, Xin Jin, Shuaicheng Li, Xiaoguang Qiu, Chunde Li, Ya Gao, Yujie Tang, Michael D. Taylor, Tao Jiang
          </td>
          <td>2025-02-17</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd18fd5a66ac51a4015b7b7d9504296b9a16919" target='_blank'>
              Abstract P018: Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, K. Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, M. Bhutani, Paola A. Guerrero, E. Ludmir, Albert C Koong, Cullen M Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd31c962d460de902bace72d06f995f9fc390f7" target='_blank'>
              Graph Structure Learning for Tumor Microenvironment with Cell Type Annotation from non-spatial scRNA-seq data
              </a>
            </td>
          <td>
            Yu-an Huang, Yue C. Li, Hai-Ru You, Jie Pan, Xiyue Cao, Xinyuan Li, Zhi-an Huang, Zhu-Hong You
          </td>
          <td>2025-02-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ecdf804c61fcb68669b4e697cde8e1d82f6075a" target='_blank'>
              Transcription factor 19 is an androgen responsive gene that modulates vessel homeostasis and sustains metastatic prostate cancer
              </a>
            </td>
          <td>
            Amaia Ercilla, Jana R. Crespo, Saioa Garcia-Longarte, Marta Fidalgo, Natalia Martín-Martín, Onintza Carlevaris, I. Astobiza, Sonia Fernández-Ruiz, M. Guiu, Laura Bárcena, Isabel Mendizabal, A. Aransay, M. Graupera, R. Gomis, Arkaitz Carracedo
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Metastasis drives mortality and morbidity in cancer. While some patients develop broad metastatic disease across multiple organs, others exhibit organ-specific spread. To identify mechanisms underlying metastatic organotropism, we analyzed clinico-genomic data from over 7,000 patients with metastatic cutaneous melanoma in three independent cohorts (one primary discovery and two validation cohorts including a nationwide electronic health record-derived deidentified database), leveraging machine learning approaches to clinical data. We found that female sex and increased tumor mutational burden associate with decreased metastatic potential, while older age associates with more lung and adrenal metastases. Using unsupervised analyses, patients clustered into five metastatic patterns: a “highly metastatic” cluster characterized by involvement of many organs, a “low metastatic” cluster characterized by few metastatic sites (mostly lymph node metastases), and three additional clusters each characterized by metastasis to specific sites (brain, lung, liver). Mutations in B2M and PTEN associated with increased overall metastatic potential. PTEN mutations were also associated with brain metastases but were enriched only in the “highly metastatic” cluster and not the brain-specific cluster. Mutations in GNAQ or GNA11 (GNA) associated with increased liver metastasis. To validate this association, we tested and demonstrated liver tropism in two GNA-mutant genetically engineered cutaneous melanoma mouse models of metastasis. Overall, our study elucidates distinct phenotypes of metastasis in patients with melanoma and identifies novel clinical and genomic associations that illuminate the clinical drivers of metastatic organotropism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7031286fbc9875d02c4079e2f16707bda19ea728" target='_blank'>
              Clinico-genomic features predict distinct metastatic phenotypes in cutaneous melanoma
              </a>
            </td>
          <td>
            Tyler J. Aprati, Chi-Ping Day, Daniel Lee, Alexander Pan, J. Jee, Giuseppe Tarantino, M. Manos, Hannah Faulkner, Marta M. Holovatska, K. Khaddour, Catherine H. Feng, Kelly P. Burke, Marc Glettig, Zoe Weaver Ohler, R. El Meskini, F. S. Hodi, Rizwan Haq, A. Shoushtari, N. Schultz, Jeffrey J. Ishizuka, Alexander Gusev, Maryclare Griffith, Kenneth L. Kehl, David Liu
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Crohn's disease (CD) involves a complex intestinal microenvironment driven by chronic inflammation. While single-cell RNA sequencing has provided valuable insights into this biology, the spatial context is lost during single-cell preparation of mucosal biopsies. To deepen our understanding of the distinct inflammatory signatures of CD and overcome the limitations of single-cell RNA sequencing, we combined spatial transcriptomics of frozen CD surgical tissue sections with single-cell transcriptomics of ileal CD mucosa. Coexpressed genes and cell-cell communication from single-cell analyses and factorized genes from spatial transcriptomics revealed overlapping pathways affected in inflamed CD, like antigen presentation, phagosome activity, cell adhesion, and extracellular matrix. Within the pathways, early epithelial cells showed evidence of significant changes in gene expression and subtype composition, while spatial mapping revealed the location of the events, particularly antigen presentation from epithelial cells in the base of the crypt. Furthermore, we identified early epithelial cells as a potential mediator of the MHC class II pathway during inflammation, which we validated by spatial transcriptomics cell subtype deconvolution. Therefore, the inflammation from CD appears to change the types of interactions detectable between epithelial cells with immune and mesenchymal cells, likely promoting the conditions for more macrophage infiltration into these inflammatory microdomains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00594471c2ef5f7cea997fb7b198a677a1e23c9" target='_blank'>
              Altered inflammatory mucosal signatures within their spatial and cellular context during active ileal Crohn's disease.
              </a>
            </td>
          <td>
            Vasantha L. Kolachala, S. Maddipatla, Shanta Murthy, Yeonjoo Hwang, A. Dodd, Garima Sharma, Sachith Munasinghe, R. Pelia, S. Venkateswaran, M. Anbazhagan, Tarun Koti, Navdeep Jhita, Gaurav N Joshi, Chrissy A. Lopez, Duke Geem, Hong Yin, David J. Cutler, Peng Qiu, J. Matthews, S. Kugathasan
          </td>
          <td>2025-03-10</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4573d60560710e744b74b82ab85a673cb84bfba" target='_blank'>
              Evaluating feature extraction in ovarian cancer cell line co-cultures using deep neural networks
              </a>
            </td>
          <td>
            Osheen Sharma, Greta Gudoityte, Rezan Minozada, Olli P. Kallioniemi, Riku Turkki, L. Paavolainen, B. Seashore-Ludlow
          </td>
          <td>2025-02-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abee2ab2885700afbd38b00c669afff895861f19" target='_blank'>
              2002 NY-ESO-1-Specific TCR-T Cells Effectively Targets Glioblastoma Cancer Stem Cells
              </a>
            </td>
          <td>
            Michelot Michel, Alan Nguyen, Alexis Parker, Hongqiang Wang, Chen Qian, Tesfahun Admasu, Andre E. Boyke, Seokyoung Yoon, Keith L. Black, Rongfu Wang, John S. Yu
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, Monia Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank Colon-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, C. A. Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b71083b48d8d0e95bc226f7370321af11b26e25b" target='_blank'>
              Spatial and single-cell transcriptomic analysis reveals fibroblasts dependent immune environment in colorectal cancer.
              </a>
            </td>
          <td>
            Hang Jia, Xiang Liu, Guimin Wang, Yue Yu, Ning Wang, Tianshuai Zhang, Liqiang Hao, Wei Zhang, Guanyu Yu
          </td>
          <td>2025-03-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ae7b5dd668485b7f400a3f84e79d9d667bb33c" target='_blank'>
              Single-cell transcriptomic analysis reveals dynamic changes in the liver microenvironment during colorectal cancer metastatic progression
              </a>
            </td>
          <td>
            Yue Jiang, Guojie Long, Xiaoming Huang, Wenyu Wang, B. Cheng, Weidong Pan
          </td>
          <td>2025-03-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76074158017484a4495c691d7f74783fe9c4e65" target='_blank'>
              Diverse stromal phenotypes at spatial resolution in colorectal cancer peritoneal metastasis.
              </a>
            </td>
          <td>
            Joseph J. Zhao, J. Ong, Ying Liu, Supriya Srivastava, Melissa Ching Ching Teo, Joey Jia-Ying Lee, Robert Walsh, Lit-Hsin Loo, Qiu Xuan Tan, Gillian Ng, J. W. Tan, Haoran Ma, Xuewen Ong, S. Tay, T. Sheng, Jeffrey H Y Lum, W. Yong, F. Pietrantonio, P. Tan, R. Sundar
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff246fd6cb059fea5f5e9204ca830e7b6e029678" target='_blank'>
              Characterizing macrophage diversity in colorectal malignancies through single-cell genomics
              </a>
            </td>
          <td>
            Tingshuo Zhao, Yinyi Luo, Yuanjie Sun, Zhigang Wei
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Germinal centres (GCs) are specialized structures where B cells undergo iterative steps of B-cell receptor (BCR) somatic hypermutation and selection of best antigen binders in a darwinian-like fashion. The accelerated evolutionary process leads to the production of high-affinity antibodies that are crucial for robust and long-term humoral immunity. Within this frame, single-cell BCR sequencing analysis is a method of choice to track GC B cell dynamics as somatic mutations can be utilised as an in vivo molecular tracer. Herein, we present SeQuoIA, a start-to-finish pipeline for the analysis of BCR repertoire sequencing data at the single-cell level, including improved clonotype assignment and phylogeny reconstruction. Most importantly, we introduce a new method for the inference of BCR-driven selection pressure based on somatic mutation patterns, that was validated with biological data. With this pipeline, we explored public datasets and proposed new selection mechanisms in GCs. Significance Our pipeline should contribute to a better understanding of the basic biology of GC dynamics, and potentially help in laboratory animal usage reduction. Clinical applications could include assessment of vaccine efficacy, monitoring of B cell anti-tumoral responses, and identification of BCR-mediated processes in B cell lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1235a7befe3184ce5a84373898145bc5761dacf1" target='_blank'>
              SeQuoIA: a single-cell BCR sequencing analysis pipeline for tracking selection mechanisms in germinal centres
              </a>
            </td>
          <td>
            Eglantine Hector, P. Milpied
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7948bcd29ed4c332a5d0dc90dea3c9f39d631a84" target='_blank'>
              Identifying distinct prognostic and predictive contributions of tumor epithelium versus tumor microenvironment in colorectal cancer
              </a>
            </td>
          <td>
            Mingli Yang, M. Nebozhyn, Michael J. Schell, N. Gandhi, Lance Pflieger, Andrey Loboda, W. J. Pledger, Ramani Soundararajan, Michelle M Maurin, Heiman Wang, Jetsen Rodriguez Silva, Ashley J Alden, Domenico Coppola, Andrew Elliott, George W Sledge, M. Khushman, Emil Lou, Sanjay Goel, Timothy J Yeatman
          </td>
          <td>2025-03-12</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a279852f8f3e4dea6bf2c322ab3c789c81b3e6c" target='_blank'>
              How cancer emerges: Data-driven universal insights into tumorigenesis via hallmark networks
              </a>
            </td>
          <td>
            Jiahe Wang, Yan Wu, Yuke Hou, Yang Li, Dachuan Xu, Changjing Zhuge, Yue Han
          </td>
          <td>2025-02-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfff83a3b2e44394201ebb9147dbd5e27d9e3ca" target='_blank'>
              Single-Cell Transcriptomic Analysis Reveals an Aggressive Basal-Like Tumor Cell Subpopulation Associated With Poor Prognosis in Intrahepatic Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Chang-Pin Liao, Yuting Zhang, Jing Yang, Shuo Wang, Zhijuan Li, Shuling Chen, Yubin Xie, Lixia Xu, Sui Peng, Xuezhen Zeng, Ming Kuang, Bangde Xiang, Kaiyu Sun, Xiao Zhao
          </td>
          <td>2025-02-24</td>
          <td>Journal of gastroenterology and hepatology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8d5845a6bd00754561294af9f1ae51a6774014" target='_blank'>
              Single-cell pseudotime and intercellular communication analysis reveals heterogeneity and immune microenvironment in oral cancer
              </a>
            </td>
          <td>
            Hanjun Liu, Lian Xie, Xuemin Xing, Lili Hou, Jinfeng Zhang, Ling Yang
          </td>
          <td>2025-02-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tyrosine kinase inhibitors (TKIs) have transformed the treatment of epidermal growth factor receptor (EGFR)-mutant non-small cell lung cancer. However, treatment resistance remains a major challenge in clinical practice. The tumor microenvironment (TME) is a complex system composed of tumor cells, immune and non-immune cells, and non-cellular components. Evidence indicates that dynamic changes in TME during TKI treatment are associated with the development of resistance. Research has focused on identifying how each component of the TME interacts with tumors and TKIs to understand therapeutic targets that could address TKI resistance. In this review, we describe how TME components, such as immune cells, fibroblasts, blood vessels, immune checkpoint proteins, and cytokines, interact with EGFR-mutant tumors and how they can promote resistance to TKIs. Furthermore, we discuss potential strategies targeting TME as a novel therapeutic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f6f1b01ac68ddc2dc3220a920e6b272e7041d2" target='_blank'>
              Targeting the Tumor Microenvironment in EGFR-Mutant Lung Cancer: Opportunities and Challenges
              </a>
            </td>
          <td>
            Jeong Uk Lim, Junyang Jung, Y. Kim, Chi Young Kim, Sang Hoon Lee, Dong Won Park, Sue In Choi, W. Ji, C. Yeo, Seung Hyeun Lee
          </td>
          <td>2025-02-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Ongoing mutagenesis in cancer drives genetic diversity throughout the natural history of cancers. As the activities of mutational processes are dynamic throughout evolution, distinguishing the mutational signatures of ‘active’ and ‘historical’ processes has important implications for studying how tumors evolve. This can aid in understanding mutagenic states at the time of presentation, and in associating active mutational process with therapeutic resistance. As bulk sequencing primarily captures historical mutational processes, we studied whether ultra-low-coverage single-cell whole-genome sequencing (scWGS), which measures the distribution of mutations across hundreds or thousands of individual cells, could enable the distinction between historical and active mutational processes. While technical challenges and data sparsity have limited mutation analysis in scWGS, we show that these data contain valuable information about dynamic mutational processes. To robustly interpret single nucleotide variants (SNVs) in scWGS, we introduce ArtiCull, a method to identify and remove SNV artifacts by leveraging evolutionary constraints, enabling reliable detection of mutations for signature analysis. Applying this approach to scWGS data from pancreatic ductal adenocarcinoma (PDAC), triple-negative breast cancer (TNBC), and high-grade serous ovarian cancer (HGSOC), we uncover temporal and spatial patterns in mutational processes. In PDAC, we observe a temporal increase in mismatch repair deficiency (MMRd). In cisplatin-treated TNBC patient-derived xenografts, we identify therapy-induced mutagenesis and inactivation of APOBEC3 activity. In HGSOC, we show distinct patterns of APOBEC3 mutagenesis, including late tumor-wide activation in one case and clade-specific enrichment in another. Additionally, we detect a clone-specific increase in SBS17 activity, in a clone previously linked to recurrence. Our findings establish ultra-low-coverage scWGS as a powerful approach for studying active mutational processes that may influence ongoing clonal evolution and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed6d1d9f289244977b2dfb8fbeaff86f943a227d" target='_blank'>
              Inferring active mutational processes in cancer using single cell sequencing and evolutionary constraints
              </a>
            </td>
          <td>
            Gryte Satas, Matthew A. Myers, Andrew McPherson, Sohrab P. Shah
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8148f78a26bf47463c0acd3c067c50a7bec19c0" target='_blank'>
              Spatio-temporal dynamics of M1 and M2 macrophages in a multiphase model of tumor growth
              </a>
            </td>
          <td>
            I. Lampropoulos, Panayotis Kevrekidis, Christos Zois, Helen Byrne, Mihail E Kavousanakis
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Motivation Single-cell RNA sequencing (scRNA-seq) provides comprehensive gene expression data at a single-cell level but lacks spatial context. In contrast, spatial transcriptomics captures both spatial and transcriptional information but is limited by resolution, sensitivity, or feasibility. No single technology combines both the high spatial resolution and deep transcriptomic profiling at the single-cell level without trade-offs. Spatial mapping tools that integrate scRNA-seq and spatial transcriptomics data are crucial to bridge this gap. However, we found that Tangram, one of the most prominent spatial mapping tools, provides inconsistent results over repeated runs. Results We refine Tangram to achieve more consistent cell mappings and investigate the challenges that arise from data characteristics. We find that the mapping quality depends on the gene expression sparsity. To address this, we (1) train the model on an informative gene subset, (2) apply cell filtering, (3) introduce several forms of regularization, and (4) incorporate neighborhood information. Evaluations on real and simulated mouse datasets demonstrate that this approach improves both gene expression prediction and cell mapping. Consistent cell mapping strengthens the reliability of the projection of cell annotations and features into space, gene imputation, and correction of low-quality measurements. Our pipeline, which includes gene set and hyperparameter selection, can serve as guidance for applying Tangram on other datasets, while our benchmarking framework with data simulation and inconsistency metrics is useful for evaluating other tools or Tangram modifications. Availability The refinements for Tangram and our benchmarking pipeline are available in https://github.com/daisybio/Tangram_Refinement_Strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e61472d084d289b15913b8aca9ccedd7f6d461" target='_blank'>
              Refinement Strategies for Tangram for Reliable Single-Cell to Spatial Mapping
              </a>
            </td>
          <td>
            Merle Stahl, Lena J. Straßer, Chit Tong Lio, J. Bernett, Richard Röttger, Markus List
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Motivation The complex dynamics of cancer evolution, driven by mutation and selection, underlies the molecular heterogeneity observed in tumors. The evolutionary histories of tumors of different patients can be encoded as mutation trees and reconstructed in high resolution from single-cell sequencing data, offering crucial insights for studying fitness effects of and epistasis among mutations. Existing models, however, either fail to separate mutation and selection or neglect the evolutionary histories encoded by the tumor phylogenetic trees. Results We introduce FiTree, a tree-structured multi-type branching process model with epistatic fitness parameterization and a Bayesian inference scheme to learn fitness landscapes from single-cell tumor mutation trees. Through simulations, we demonstrate that FiTree outperforms state-of-the-art methods in inferring the fitness landscape underlying tumor evolution. Applying FiTree to a single-cell acute myeloid leukemia dataset, we identify epistatic fitness effects consistent with known biological findings and quantify uncertainty in predicting future mutational events, offering a unified framework for understanding tumor progression and potentially for guiding therapeutic strategies. Availability and implementation The Python package FiTree and the analysis workflows are available at https://github.com/cbg-ethz/FiTree.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ec98bbff17d8ff95bd2ee61ee88e0f50ed30add" target='_blank'>
              Bayesian inference of fitness landscapes via tree-structured branching processes
              </a>
            </td>
          <td>
            X. Luo, Jack Kuipers, Kevin Rupp, Koichi Takahashi, N. Beerenwinkel
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8e1ac36ae0a43d7b28635df6f8edaf8554a72a9" target='_blank'>
              SpaDCN: Deciphering Spatial Functional Landscape from Spatially Resolved Transcriptomics by Aligning Cell-Cell Communications.
              </a>
            </td>
          <td>
            Xiaosheng Bai, Xinyu Bao, Chuanchao Zhang, Qianqian Shi, Luonan Chen
          </td>
          <td>2025-02-17</td>
          <td>Small methods</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/894325e4ae969ecac0174d65aab45ba97411ecc8" target='_blank'>
              SOX2 empowers a rapid tumorigenic programme from the tumour-resistant population in the skin
              </a>
            </td>
          <td>
            Patricia P. Centeno, Christopher Chester, Catriona A. Ford, P. Cammareri, Gareth J. Inman, Thomas Jamieson, Rachel A Ridgway, Richard Marais, A. Campbell, Owen J. Sansom
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d026aba47659bcc0e46d8766e11f46d409c8da83" target='_blank'>
              Multidimensional Classification Framework for Human Breast Cancer Cell Lines
              </a>
            </td>
          <td>
            Diogo Dias, Catarina Franco Jones, Ana Catarina Moreira, Gil Gonccalves, Mustafa Bilgin Djamgoz, Frederico Castelo Ferreira, Paola Sanjuan‐Alberte, Rosalia Moreddu
          </td>
          <td>2025-02-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1234b532b5af1ee94b089ab7d8fa57fb3a92e94f" target='_blank'>
              Precise metabolic dependencies of cancer through deep learning and validations
              </a>
            </td>
          <td>
            Tao Wu, Xiangyu Zhao, Yu Zhang, Die Qiu, Kaixuan Diao, Dongliang Xu, Weiliang Wang, Xiaopeng Xiong, Xinxiang Li, Xue-song Liu
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccad37f8db740ee741e31d76d75cb5a07f7e923f" target='_blank'>
              Biology as vulnerability in follicular lymphoma: genetics, epigenetics, and immunogenetics.
              </a>
            </td>
          <td>
            F. Araujo-Ayala, W. Béguelin
          </td>
          <td>2025-03-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41ea1b0559ba72f593b3a6674f424957c7f5230c" target='_blank'>
              Abstract B016: Genomic features of the tumor microenvironment and antigen presentation machinery vary by clinical and molecular subtypes of breast cancer
              </a>
            </td>
          <td>
            Constandina E. O'Connell, Yoon Jin Cha, Alexander V Lobanov, Cheng Fan, Brooke M. Felsheim, A. Fernandez-Martinez, C. Perou
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>163</td>
        </tr>

        <tr id="Abstract Breast cancer remains a significant global health challenge due to its complexity, which arises from multiple genetic and epigenetic mutations that originate in normal breast tissue. Traditional machine learning models often fall short in addressing the intricate gene interactions that complicate drug design and treatment strategies. In contrast, our study introduces GEMDiff, a novel computational workflow leveraging a diffusion model to bridge the gene expression states between normal and tumor conditions. GEMDiff augments RNAseq data and simulates perturbation transformations between normal and tumor gene states, enhancing biomarker identification. GEMDiff can handle large-scale gene expression data without succumbing to the scalability and stability issues that plague other generative models. By avoiding the need for task-specific hyper-parameter tuning and specific loss functions, GEMDiff can be generalized across various tasks, making it a robust tool for gene expression analysis. The model’s ability to augment RNA-seq data and simulate gene perturbations provides a valuable tool for researchers. This capability can be used to generate synthetic data for training other machine learning models, thereby addressing the issue of limited biological data and enhancing the performance of predictive models. The effectiveness of GEMDiff is demonstrated through a case study using breast mRNA gene expression data, identifying 307 core genes involved in the transition from a breast tumor to a normal gene expression state. GEMDiff is open source and available at https://github.com/xai990/GEMDiff.git under the MIT license.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3472d3e5ad8a58f2f1d0054cb37345e502bfbbc" target='_blank'>
              GEMDiff: a diffusion workflow bridges between normal and tumor gene expression states: a breast cancer case study
              </a>
            </td>
          <td>
            Xusheng Ai, Melissa C. Smith, F. A. Feltus
          </td>
          <td>2025-03-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4e7d8569cb5aa3f3831a8692c395636f7587796" target='_blank'>
              Cancer‑associated fibroblasts: a pivotal regulator of tumor microenvironment in the context of radiotherapy
              </a>
            </td>
          <td>
            Linhui Zheng, Wenqi Cai, Yuan Ke, Xiaoyan Hu, Chunqian Yang, Runze Zhang, Huachao Wu, Dong Liu, Haijun Yu, Chaoyan Wu
          </td>
          <td>2025-03-20</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5d48556d03df2b88d1c06ed57c2bca71cfaf9b2" target='_blank'>
              Spatial genomics reveals cholesterol metabolism as a key factor in colorectal cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Andrew J. Kavran, Yulong Bai, Brian Rabe, Anna Kreshock, Andrew Fisher, Yelena Cheng, Anne Lewin, Chao Dai, Matthew J. Meyer, Konstantinos J. Mavrakis, Anna Lyubetskaya, Eugene Drokhlyansky
          </td>
          <td>2025-03-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87b1608b4f56eb9c6e1216a160d63214aa03c88b" target='_blank'>
              Spatially resolved protein profiling in pancreatic adenosquamous carcinoma.
              </a>
            </td>
          <td>
            Adel Kardosh, Rosalie C. Sears, Zheng Xia, Eizabaru M Sasatomi, Brian Mau, Jonathan R Brody, Melissa Wong, Charles D. Lopez, E. Chen, Guillaume Pegna, Cristiane M Franca, John Muschler, Joanna Pucilowska, Shaun M Goodyear
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/997e954cb156caa69cf5f6fb240e0f9297f38fc1" target='_blank'>
              Single-cell and spatial-resolved profiling reveals cancer-associated fibroblast heterogeneity in colorectal cancer metabolic subtypes
              </a>
            </td>
          <td>
            Youpeng Wang, Xingfeng Qiu, Qinghai Li, Jiale Qin, Lvlan Ye, Xiang Zhang, Xingxiang Huang, Xiangqiong Wen, Ziyang Wang, Weiling He, Yuqin Di, Qi Zhou
          </td>
          <td>2025-02-11</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e89dc2d087d9d2c969c5bb0a4b45e01fc9cc0f7a" target='_blank'>
              Single-cell and spatial genomic landscape of non-small cell lung cancer brain metastases.
              </a>
            </td>
          <td>
            Somnath Tagore, Lindsay A. Caprio, Amit Dipak Amin, K. Bestak, Karan Luthria, Edridge K. D’Souza, Irving Barrera, Johannes C. Melms, Sharon Wu, Sinan Abuzaid, Yiping Wang, Viktoria Jakubikova, Peter Koch, D. Z. Brodtman, Banpreet Bawa, S. Deshmukh, Leon Ebel, Miguel A. Ibarra-Arellano, A. Jaiswal, Carino Gurjao, Jana Biermann, Neha Shaikh, Priyanka Ramaradj, Yohanna Georgis, Galina G Lagos, Matthew I Ehrlich, P. Ho, Zachary H. Walsh, Meri Rogava, Michelle Garlin Politis, Devanik Biswas, A. Cottarelli, Nikhil Rizvi, CA Shu, Benjamin O Herzberg, N. Anandasabapathy, G. Sledge, Emmanuel Zorn, Peter D Canoll, Jeffrey N. Bruce, Naiyer A Rizvi, Alison M Taylor, Anjali Saqi, H. Hibshoosh, Gary K Schwartz, Brian S Henick, Fei Chen, Denis Schapiro, Parin Shah, Benjamin Izar
          </td>
          <td>2025-02-27</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a122d9a6ddd44faaac6dadb018913abf2350d5" target='_blank'>
              Reconstructing Cell Lineage Trees from Phenotypic Features with Metric Learning
              </a>
            </td>
          <td>
            Da Kuang, Guanwen Qiu, Junhyong Kim
          </td>
          <td>2025-03-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5372012557b72610a59f72be2bf39f2688654856" target='_blank'>
              Spatial distribution of the proteome in human body and cancers
              </a>
            </td>
          <td>
            Liang Yue, Wenhao Jiang, Sainan Li, Meng Luo, Ning Fan, Xiaolu Zhan, Kexin Chen, Tian Lu, Fang Guo, Dongwei Li, Weigang Ge, Zongxiang Nie, M. Lyu, A. Jun, Yingrui Wang, Yingdan Chen, Zhenhai Fu, Nan Xiang, Lu Li, Fengchao Yu, Guo Ci Teo, Alexey I. Nesvizhskii, Meng Wang, Michael P. Snyder, Ben C. Collins, Ruedi Aebersold, Fei Xu, Yingchuan Zhu, Tong Liu, Yan Li, Tiannan Guo
          </td>
          <td>2025-02-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background CD4+ T cells are a highly differentiated cell type that maintain enough transcriptomic plasticity to cycle between activated and memory statuses. How the 1D chromatin state and 3D chromatin architecture support this plasticity is under intensive investigation. Methods Here, we wished to test a commercially available in situ Hi-C kit (Arima Genomics Inc.) to establish whether published performance on limiting cell numbers from clonal cell lines copies across to a primary immune cell type. We achieved comparable contact matrices from 50,000, 250,000, and 1,000,000 memory CD4+ T-cell inputs. We generated multiple Hi-C and RNA-seq libraries from the same biological blood donors under three separate conditions: unstimulated fresh ex vivo, IL-2-only stimulated, and T cell receptor (TCR)+CD28+IL-2-stimulated, conferring increasingly stronger activation signals. We wished to capture the magnitude and progression of 3D chromatin shifts and correlate these to expression changes under the two stimulations. Results Although some genome organization changes occurred concomitantly with changes in gene expression, at least as many changes occurred without corresponding changes in expression. Counter to the hypothesis that topologically associated domains (TADs) are largely invariant structures providing a scaffold for dynamic looping contacts between enhancers and promotors, we found that there were at least as many dynamic TAD changes. Stimulation with IL-2 alone triggered many changes in genome organization, and many of these changes were strengthened by additional TCR and CD28 co-receptor stimulation. Conclusions This suggests a stepwise process whereby mCD4+ T cells undergo sequential buildup of 3D architecture induced by distinct or combined stimuli likely to “prime” or “deprime” them for expression responses to subsequent TCR-antigen ligation or additional cytokine stimulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd1aa3f310b47dde1e0fdd44c2b4206007ebe632" target='_blank'>
              Memory CD4+ T cells sequentially restructure their 3D genome during stepwise activation
              </a>
            </td>
          <td>
            Alexander I. Ward, J. I. de las Heras, Eric C. Schirmer, A. Fassati
          </td>
          <td>2025-02-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37e24bbb19a6e12af52d533253b1371130f86db3" target='_blank'>
              Integrating genomic, transcriptomic and epigenetic data to identify phenotypically impactful driver pathways in glioblastoma
              </a>
            </td>
          <td>
            Junhua Zhang
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Motivation Spatial transcriptomic technologies allow researchers to explore the diversity and specificity of gene expression within their original tissue structure. Accurately identifying regions that are spatially coherent in both gene expression and physical tissue structures is an emerging topic, but challenging due to the lack of ground truth labels which renders complicating validation of clustering consistency and reproducibility. This highlights a need for a computational evaluation framework to rigorously and unbiasedly assess clustering performance. Results To address this gap, we propose STEAM (Spatial Transcriptomics Evaluation Algorithm and Metric), a user-friendly computational pipeline designed to evaluate the consistency and reliability of clustering results by leveraging machine learning classification and prediction methods, with the goal of maintaining the spatial proximity and gene expression patterns within clusters. We benchmarked STEAM on various public datasets, spanning multi-cell to single-cell resolution, as well as spatial transcriptomics and proteomics. The results highlighted its robustness and generalizability through comprehensive statistical evaluation metrics, such as Kappa score, F1 score, accuracy, and adjusted rand index. Notably, STEAM supports multi-sample training, enabling cross-replicate clustering consistency assessment. Moreover, STEAM provides practical guidance by comparing clustering results across multiple approaches; here, we evaluated four different methods, including spatial-aware and spatial-ignorant approaches. In summary, we believe that STEAM provides researchers a promising tool for evaluating clustering robustness and benchmarking clustering performance for spatial omics data, offering valuable insights to drive reproducible discoveries in spatial biology. Availability and implementation Source code and the R software tool STEAM are available from https://github.com/fanzhanglab/STEAM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a0bb40c1ac826ccb8d6dd553bb5b01c202994f" target='_blank'>
              STEAM: Spatial Transcriptomics Evaluation Algorithm and Metric for clustering performance
              </a>
            </td>
          <td>
            Samantha Reynoso, Courtney Schiebout, Revanth Krishna, Fan Zhang
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a148fdaf03fd46b2f2df2ac24d19cb5232bc753" target='_blank'>
              CytoPheno: Automated descriptive cell type naming in flow and mass cytometry
              </a>
            </td>
          <td>
            AmandaR. Tursi, Celine S. Lages, Kenneth Quayle, Zachary T. Koenig, Rashi Loni, Shruti Eswar, José Cobeña-Reyes, Sherry Thornton, Tamara Tilburgs, Sandra Andorf
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1f7c17606ae67d5ce7fe611db0e7bce958444bb" target='_blank'>
              Decoding the heterogeneous subpopulations of glioblastoma for prognostic stratification and uncovering the promalignant role of PSMC2
              </a>
            </td>
          <td>
            Nini Zhou, Jingsong Yan, Manya Xiong, Shunqin Zhu
          </td>
          <td>2025-02-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa45e1efe02786e2e4534cf1ae408e65c7ffadc" target='_blank'>
              To describe the subsets of malignant epithelial cells in gastric cancer, their developmental trajectories and drug resistance characteristics
              </a>
            </td>
          <td>
            Tingting Xu, Tianying Zhang, Yan Sun, Sijia Wu
          </td>
          <td>2025-01-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b581086a365c6022e8cfc4df2d851d53411c3c94" target='_blank'>
              494 Multi-Omics Integration and Mapping of Invasion Markers Reveal Tumor-Specific Druggable Targets in Primary Human Glioblastoma
              </a>
            </td>
          <td>
            Paola Suarez-Meade, Jean R. Clemenceau, Isabel Barnfather, Mark Edgar, Aleeshba N. Basil, Tae H. Hwang, A. Quiñones-Hinojosa
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab8b68e7b5a2f581513237acff3c4e88418f66b" target='_blank'>
              Haploinsufficient phenotypes promote selection of PTEN and ARID1A-deficient clones in human colon
              </a>
            </td>
          <td>
            Nefeli Skoufou-Papoutsaki, Sam Adler, Shenay Mehmed, Claire Tume, Cora Olpe, Edward Morrissey, Richard Kemp, Anne-Claire Girard, Elisa B. Moutin, C. Chilamakuri, Jodi L Miller, Cecilia Lindskog, Fabian Werle, Kate Marks, Francesca Perrone, Matthias Zilbauer, D. Tourigny, Doug Winton
          </td>
          <td>2025-02-07</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1db80b341443231830586b45a4d64283910100" target='_blank'>
              Identification of increased dedifferentiation along the Prom1+ cancer cells in Müllerian adenosarcoma with sarcomatous overgrowth.
              </a>
            </td>
          <td>
            Xingming Ye, Jianfeng Zheng, Dan Hu, Li Liu, Fukun Chen, Xintong Cai, Yangmei Xu, Lifeng Li, Jie Lin, Qinying Liu, Yang Sun
          </td>
          <td>2025-02-07</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Glioblastoma (GBM) is the most common and aggressive primary malignant brain tumor of adults. Current therapeutic options yield dismal prognoses that have remained essentially unchanged over nearly two decades. Diffuse growth patterns, high intratumoral heterogeneity, and variable blood–brain barrier integrity limit treatment efficacy, creating challenges that rational small molecule design has not overcome. Antibody–drug conjugates have shown some promise, leading us to hypothesize that smaller folded DNA aptamers, developed in vivo via principles of natural selection, might eventually have advantages for drug delivery. Here, we document the first in vivo DNA aptamer selection involving an orthotopic patient-derived xenograft GBM mouse model to identify tumor-homing DNA aptamers. We demonstrate the preferential accumulation of these aptamers in the tumor relative to other tissues 4 h after intraperitoneal injection. The aptamers can be detected by quantitative polymerase chain reaction, fluorescent tumor staining, and stain GBM tumor section from untreated mice and the GBM tumor cells in culture. Two of three candidates are selective for the target cell line in vitro and do not bind other human tumor cells. In vivo selection of tumor-specific DNA aptamers demonstrates a novel approach for diagnostics or toxin delivery that might allow for the development of individualized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a69e4c2d2724fca734736e524ece9241f19793a" target='_blank'>
              In vivo selection of anti-glioblastoma DNA aptamers in an orthotopic patient-derived xenograft model
              </a>
            </td>
          <td>
            Caroline D Doherty, Brandon A Wilbanks, Sonia Jain, Keenan S Pearson, Katie K Bakken, D. Burgenske, Nay Won Lett, J. Sarkaria, L. Maher
          </td>
          <td>2025-02-18</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="To better understand the homeostatic mechanisms governing melanocytes, we performed deep phenotyping of clonal expansions of single melanocytes from human skin. In total, we interrogated the mutational landscapes, gene expression profiles, and morphological features of 297 melanocytes from 31 donors. To our surprise, a population of melanocytes with low mutation burden was maintained in sun damaged skin. These melanocytes were more stem-like, smaller, less dendritic and displayed distinct gene expression profiles compared to their counterparts with high mutation burdens. We used single-cell spatial transcriptomics (10X Xenium) to reveal the spatial distribution of melanocytes inferred to have low and high mutation burdens (LowMut and HighMut cells), based on their gene expression profiles. LowMut melanocytes were found in hair follicles as well as in the interfollicular epidermis, whereas HighMut melanocytes resided almost exclusively in the interfollicular epidermis. We propose that melanocytes in the hair follicle occupy a privileged niche, protected from UV radiation, but periodically migrate out of the hair follicle to replenish the interfollicular epidermis after waves of photodamage. More broadly, our study illustrates the advantages of a cell atlas that includes mutational information, as cells can change their cellular states and positional coordinates over time, but mutations are like scars, providing a historical record of the homeostatic processes that were operative on each cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35fcdb72f7ddf292a52ab622c06a7d378c43f17" target='_blank'>
              Somatic mutations distinguish melanocyte subpopulations in human skin
              </a>
            </td>
          <td>
            Bishal Tandukar, Delahny Deivendran, Limin Chen, Neda Bahrani, Beatrice Weier, Harsh Sharma, Noel Cruz-Pacheco, Min Hu, Kayla Marks, Rebecca G. Zitnay, Aravind K. Bandari, Rojina Nekoonam, I. Yeh, R. Judson-Torres, A. H. Shain
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01243dcc84f5d3d7fbdd4aa36ab8b1b63e350fae" target='_blank'>
              Abstract A086: Myeloid suppression within metastatic tumor microenvironments in advanced breast cancer—friend or foe to checkpoint inhibition?
              </a>
            </td>
          <td>
            Batul Al-zubeidy, Edgar Gonzalez, A. Baugh, Dominic Zavala, Mathew Jacobo, Michael F. Press, E. R. Torres
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f6e80235cbbb65b993c05ceb81c0545572bfc88" target='_blank'>
              Robust Spatial Cell-Type Deconvolution with Qualitative Reference for Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Qishi Dong, Yi Yang, Ziye Luo, Haipeng Shen, Xingjie Shi, Jin Liu
          </td>
          <td>2025-03-09</td>
          <td>Small methods</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad8fec02f869aff60be41da2c4550105b35eaada" target='_blank'>
              Functional Tumor Boundaries Revealed by Dynamic Electroimpedance Imaging Under Weak Stimuli: Dissipative Structures in Melanoma Area
              </a>
            </td>
          <td>
            Yuri F. Babich
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Solid tumours are intricate and highly heterogeneous ecosystems, which grow in and invade normal organs. Their progression is mediated by cancer cells’ interaction with different cell types, such as immune cells, stromal cells and endothelial cells, and with the extracellular matrix. Owing to its high incidence, aggressive growth and resistance to local and systemic treatments, liver cancer has particularly high mortality rates worldwide. In recent decades, spatial heterogeneity has garnered significant attention as an unfavourable biological characteristic of the tumour microenvironment, prompting extensive research into its role in liver tumour development. Advances in spatial omics have facilitated the detailed spatial analysis of cell types, states and cell‒cell interactions, allowing a thorough understanding of the spatial and temporal heterogeneities of tumour microenvironment and informing the development of novel therapeutic approaches. This review illustrates the latest discovery of the invasive zone, and systematically introduced specific macroscopic spatial heterogeneities, pathological spatial heterogeneities and tumour microenvironment heterogeneities of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27ba251a9613c3090a1b350987469c7d7cfc3f02" target='_blank'>
              Spatial‒temporal heterogeneities of liver cancer and the discovery of the invasive zone
              </a>
            </td>
          <td>
            Jiayan Yan, Zhifeng Jiang, Shiyu Zhang, Qichao Yu, Yijun Lu, Runze Miao, Zhaoyou Tang, Jia Fan, Liang Wu, Dan G. Duda, Jian Zhou, Xinrong Yang
          </td>
          <td>2025-02-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2f30006d88490586d800953c99040977f4d92b" target='_blank'>
              Differential cellular origins of the extracellular matrix of tumor and normal tissues according to colorectal cancer subtypes.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Sangkeun Park, Jun Hyeong Lee, Shin Young Chang, Sang Mi Oh, S. Mun, Junho Kang, Jong-Eun Park, Jung Kyoon Choi, Tae Il Kim, Jin Young Kim, Pilnam Kim
          </td>
          <td>2025-03-03</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Oral squamous cell carcinoma (OSCC) is an increasingly prevalent malignancy worldwide. This study aims to understand molecular alterations associated with lymph node metastasis of OSCC in order to improve treatment strategies. We analysed a cohort of 46 patients with primary OSCC, including 10 with lymph node metastasis and 36 without. Using a comprehensive multi‐omics approach – encompassing genomic, transcriptomic, proteomic, epigenetic, single‐cell, and spatial analyses – we integrated data to delineate the molecular landscape of OSCC in the context of lymph node metastasis. Our genomic analysis identified significant mutations in key genes within the MAPK, TGF‐β and WNT signalling pathways, which are essential for tumour development. The proteogenomic analysis highlighted pathways critical for lymph node dissemination and factors contributing to an immunosuppressive tumour microenvironment. Elevated levels of POSTN were found to reorganise the extracellular matrix (ECM), interact with TGF‐β, disrupt cell cycle regulation and suppress the immune response by reducing VCAM1 activity. Integrated analyses of single‐cell and spatial transcriptome data revealed that cancer‐associated fibroblasts (CAFs) secrete TGF‐β1/2, promoting cancer cell metastasis through epithelial–mesenchymal transition (EMT). Our integrated multi‐omics analysis provides a detailed understanding of molecular mechanisms driving lymph node metastasis of OSCC. These insights could lead to more precise diagnostics and targeted treatments. This article is protected by copyright. All rights reserved">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a9d23bd90d2baf9dbe230266b6f429b3c96696a" target='_blank'>
              Proteogenomic characterisation of primary oral cancer unveils extracellular matrix remodelling and immunosuppressive microenvironment linked to lymph node metastasis
              </a>
            </td>
          <td>
            Yu Liu, Zhenyu Yang, J. Pu, Jie Zhong, U. Khoo, Yu-Xiong Su, Gao Zhang
          </td>
          <td>2025-03-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Fanconi Anemia (FA) is a heritable syndrome characterized by DNA damage repair deficits, frequent malformations and a significantly elevated risk of bone marrow failure, leukemia, and mucosal head and neck squamous cell carcinomas (HNSCC). Hematopoietic stem cell gene therapy can prevent marrow failure and lower leukemia risk, but mucosal gene therapy to lower HNSCC risk remains untested. Major knowledge gaps include an incomplete understanding of how rapidly gene-corrected cellular lineages could spread through the oral epithelium, and which delivery parameters are critical for ensuring efficient gene correction. To answer these questions, we extended an agent-based model of the oral epithelium to include the delivery of gene correction in situ to FA cells and the competitive dynamics between cellular lineages with and without gene correction. We found that only gene-corrected lineages with substantial proliferative advantages (probability of resisting displacement out of the basal layer ≥ 0. 1) could spread on clinically relevant timelines, and that these lineages were initially at high risk of loss in the generations following correction. Delivering gene correction to many cells minimizes the risk of loss, while delivery to many distinct locations within a tissue maximizes the rate of spread. To determine the impact of mucosal gene therapy in preventing the clonal expansion of pre-cancerous mutations, we compared the expected burden of TP53 mutations in simulated tissue sections with and without gene correction. We found that when FA cells have elevated genome instability or a TP53-dependent proliferative advantage, gene correction can substantially reduce the accumulation of pro-tumorigenic mutations. This model illustrates the power of computational frameworks to identify critical determinants of therapeutic success to enable experimental optimization and support novel and effective gene therapy applications. Author summary We investigated factors influencing the success of oral mucosal gene therapy for Fanconi Anemia (FA), a genetic syndrome marked by DNA repair defects in conjunction with a heightened risk of cancer. We used a computational model of the oral epithelium to determine how gene therapy corrected cells compete with FA background cells and the best gene delivery approaches to promote effective tissue replacement by gene-corrected cells. We find that gene-corrected cells require strong proliferative advantages to spread effectively, and that initially delivering gene correction to more cells reduces the chance that these cells are stochastically eliminated before they can spread. We also demonstrate that gene correction reduces the accumulation of pro-tumorigenic TP53 mutations in an FA context, where genomic instability can elevate the mutation rate and FA-specific selective pressures could favor accelerated TP53 clonal expansion. This research provides a useful framework for guiding mucosal gene therapy experiments and the development of effective oral gene therapy protocols for cancer prevention in FA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb3b56fe96b81aec45dd517d7b11cc8ca9c35" target='_blank'>
              Epithelial competition determines gene therapy potential to suppress Fanconi Anemia oral cancer risk
              </a>
            </td>
          <td>
            Hunter L. Colegrove, Raymond J Monnat, Alison F Feder
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c10484a1b1c15548c942fa338e1afd0e25fd59" target='_blank'>
              Lung tumor organoids migrate as cell clusters containing cancer stem cells under hypoxic condition.
              </a>
            </td>
          <td>
            Yanjiao Li, Jiarong Zou, Yanhua Fang, Jianing Zuo, Ruoyu Wang, Shanshan Liang
          </td>
          <td>2025-02-01</td>
          <td>Biology of the cell</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Somatic activating mutations in KRAS can cause complex lymphatic anomalies (CLAs). However, the specific processes that drive KRAS-mediated CLAs have yet to be fully elucidated. Here, we used single-cell RNA sequencing to construct an atlas of normal and KrasG12D-malformed lymphatic vessels. We identified six subtypes of lymphatic endothelial cells (LECs) in the lungs of adult wild-type mice (Ptx3, capillary, collecting, valve, mixed, and proliferating). To determine when the LEC subtypes were specified during development, we integrated our data with data from four stages of development. We found that proliferating and Ptx3 LECs were prevalent during early lymphatic development and that collecting and valve LECs emerged later in development. Additionally, we discovered that the proportion of Ptx3 LECs decreased as the lymphatic network matured but remained high in KrasG12D mice. We also observed that the proportion of collecting and valve LECs was lower in KrasG12D mice than in wild-type mice. Last, we found that immature lymphatic vessels in young mice were more sensitive to the pathologic effects of KrasG12D than mature lymphatic vessels in older mice. Together, our results expand the current model for the development of the lymphatic system and suggest that KRAS mutations impair the maturation of lymphatic vessels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b83680aa397127e25da0c4fb9fc494573ba691b" target='_blank'>
              A single-cell atlas of normal and KRASG12D-malformed lymphatic vessels.
              </a>
            </td>
          <td>
            Lorenzo M. Fernandes, Danielle Griswold-Wheeler, Jeffrey D Tresemer, Angelica Vallejo, Neda Vishlaghi, Benjamin Levi, Abigail Shapiro, Joshua P. Scallan, Michael T. Dellinger
          </td>
          <td>2025-01-28</td>
          <td>JCI insight</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c96757ce9569828ea0b78de759efc296d25cef" target='_blank'>
              Abstract A062: Mechanisms mediating hypoxic memory of type I IFN signaling suppression and implications for immune evasion in luminal breast cancer cells
              </a>
            </td>
          <td>
            Rebecca Marker, Aidan Moriarty, Remi Klotz, Min Yu
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Asymmetric cell division (ACD) is a fundamental process to balance cell proliferation and differentiation during development and in the adult. Cancer stem cells (CSCs), a very small but highly malignant population within many human tumors, are able to provide differentiated progeny by ACD that contribute to the intratumoral heterogeneity, as well as to proliferate without control by symmetric, self-renewing divisions. Thus, ACD dysregulation in CSCs could trigger cancer progression. Here we consistently find low expression levels of RAP2A, the human homolog of the Drosophila ACD regulator Rap2l, in glioblastoma (GBM) patient samples, and observe that scarce levels of RAP2A are associated with poor clinical prognosis in GBM. Additionally, we show that restitution of RAP2A in GBM neurosphere cultures increases the ACD of glioblastoma stem cells (GSCs), decreasing their proliferation and expression of stem cell markers. Our results support that ACD failures in GSCs increases their spread, and that ACD amendment could contribute to reduce the expansion of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/518970f2ca9840fa8703b183e169e114b9e64218" target='_blank'>
              Human RAP2A Homolog of the Drosophila Asymmetric Cell Division Regulator Rap2l Targets the Stemness of Glioblastoma Stem Cells
              </a>
            </td>
          <td>
            Maribel Franco, R. Gargini, Víctor M. Barberá, Miguel Saceda, A. Carmena
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8677e695bbefb8bee72d8f6205f8c94375119315" target='_blank'>
              Developmental Origins and Oncogenesis in Medulloblastoma.
              </a>
            </td>
          <td>
            K. Kiang, Yogesh K H Wong, Soma Sengupta, Martine F. Roussel, Q. R. Lu
          </td>
          <td>2025-02-14</td>
          <td>Annual review of neuroscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6fe7c1118b5b40de590086593aa209906bee3a7" target='_blank'>
              Abstract A049: Application of a combinatorial functional precision medicine platform to predict immunotherapy response in DLBCL
              </a>
            </td>
          <td>
            Sharon Pei Yi Chan, J. Lim, M. Rashid, Edward Kai-Hua Chow
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatial transcriptomics has revolutionized our ability to characterize tissues and diseases by contextualizing gene expression with spatial organization. Available methods require researchers to either train a model using histology-based annotations or use annotation-free clustering approaches to uncover spatial domains. However, few methods provide researchers with a way to jointly analyze spatial data from both annotation-free and annotation-guided perspectives using consistent inductive biases and levels of interpretability. A single framework with consistent inductive biases ensures coherence and transferability across tasks, reducing the risks of conflicting assumptions. To this end, we propose the Spatial Topic Model (SpaTM), a topic-modeling framework capable of annotation-guided and annotation-free analysis of spatial transcriptomics data. SpaTM can be used to learn gene programs that represent histology-based annotations while providing researchers with the ability to infer spatial domains with an annotation-free approach if manual annotations are limited or noisy. We demonstrate SpaTM’s interpretability with its use of topic mixtures to represent cell states and transcriptional programs and how its intuitive framework facilitates the integration of annotation-guided and annotation-free analyses of spatial data with downstream analyses such as cell type deconvolution. Finally, we demonstrate how both approaches can be used to extend the analysis of large-scale snRNA-seq atlases with the inference of cell proximity and spatial annotations in human brains with Major Depressive Disorder.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/072bf737cab98bafa5f0e5c13caff2f010a9c881" target='_blank'>
              SpaTM: Topic Models for Inferring Spatially Informed Transcriptional Programs
              </a>
            </td>
          <td>
            Adrien Osakwe, Wenqi Dong, Qihuang Zhang, Robert Sladek, Yue Li
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7183df86db67e8819a668aa6143ceb640672852d" target='_blank'>
              PRODE recovers essential and context-essential genes through neighborhood-informed scores
              </a>
            </td>
          <td>
            T. Cantore, P. Gasperini, Riccardo Bevilacqua, Yari Ciani, Sanju Sinha, E. Ruppin, F. Demichelis
          </td>
          <td>2025-02-28</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c40c61d7dcc3833421bbc3568f9b1a472c0c4795" target='_blank'>
              Cell death in tumor microenvironment: an insight for exploiting novel therapeutic approaches
              </a>
            </td>
          <td>
            Wenxin Wang, Tong Li, Kui Wu
          </td>
          <td>2025-03-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166ffe8ca69e378ce5cd3ea06cb57373f1b4e28e" target='_blank'>
              High-purity CTC RNA sequencing identifies prostate cancer lineage phenotypes prognostic for clinical outcomes.
              </a>
            </td>
          <td>
            Marina N. Sharifi, J. Sperger, Amy K Taylor, Katharine E Tippins, Shannon R Reese, Viridiana Carreno, K. Kaufmann, Alex H Chang, Luke A Nunamaker, Charlotte Linebarger, Leilani Mora-Rodriguez, J. Schehr, H. Krause, K. Helzer, M. Bootsma, G. Blitzer, John M Floberg, C. Kyriakopoulos, H. Emamekhoo, Elizabeth I. Heath, Meghan Wells, S. Tagawa, M. Sjöström, A. Choudhury, Menggang Yu, Andrew J. Armstrong, D. Rathkopf, H. Beltran, Peter S Nelson, Felix Y. Feng, S. Dehm, D. Kosoff, Xiao X Wei, Rana R Mckay, Shuang G. Zhao, Joshua M Lang
          </td>
          <td>2025-02-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e9a98022a15f5fa6336dcfa874e713b8fce8db6" target='_blank'>
              AGP-Net: A Universal Network for Gene Expression Prediction of Spatial Transcriptomics
              </a>
            </td>
          <td>
            Yan Yang, Xuesong Li, Liyuan Pan, Guoxun Zhang, Liu Liu, Eric A. Stone
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0871f3b2bbb54a6ef4ad184b9e9c10fad8f88b51" target='_blank'>
              Spatial single-cell maps reveal ST6GAL1 promoting ovarian cancer metastasis.
              </a>
            </td>
          <td>
            Lan-hui Qin, Zijian Jiang, Chongze Yang, Rui Song, Pei-Yin Chen, Weihui Xu, Guanzhen Zeng, Jin-yuan Liao, Liling Long
          </td>
          <td>2025-01-30</td>
          <td>Glycoconjugate journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The sentinel lymph node (SLN) is the first lymph node encountered by a metastatic cancer cell and serves as a predictor of poor prognosis, as patients with clinically occult SLN metastases are classified as stage III with elevated rates of recurrence and diminished overall survival. However, the dynamics of immune infiltrates in SLNs remain poorly characterized. Here, using an unbiased Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq) technique, we profiled 97,777 cells from SLN tissues obtained from patients with stages I/II and III cutaneous melanoma. We describe the transcriptional programs of a multitude of T, B, and myeloid cell subtypes in SLNs. Based on the proportions of cell types, we determined that SLN subtypes are stratified along a naive → activated axis; patients with a 'high activated' signature score appear to be undergoing a robust melanoma antigen-driven adaptive immune response and thus, could be responsive to immunotherapy. Additionally, we identified transcriptomic signatures of SLN-infiltrating dendritic cell subsets that compromise anti-tumor immune responses. Our analyses provide valuable insights into tumor-driven immune changes in the SLN tissue, offering a powerful tool for the informed design of immune therapies for high-risk melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ff6ed2d70346b46dbac007982b480af5731bdaf" target='_blank'>
              Single-cell transcriptomics of melanoma sentinel lymph nodes identifies unique immune cell signatures associated with metastasis.
              </a>
            </td>
          <td>
            Eric Engelbrecht, Bryce F Stamp, Lewis C. Chew, Omar S Sarkar, Phillip Harter, Sabine J Waigel, Eric C Rouchka, J. Chariker, Andrei Smolenkov, Jason A Chesney, K. McMasters, Corey T Watson, K. Yaddanapudi
          </td>
          <td>2025-03-06</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b83f8e329366e14ce122de3c8f5608b479cd4df9" target='_blank'>
              Mapping glioma progression: single-cell RNA sequencing illuminates cell–cell interactions and immune response variability
              </a>
            </td>
          <td>
            Xia Li, Shenbo Chen, Ming Ding, Hui Ding, Kun Yang
          </td>
          <td>2025-03-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/010275e5ed8a3481a55e4db6d64687206e7bea48" target='_blank'>
              The characteristic of epithelial-specific phenotypes and immunosuppressive microenvironment in the context of tumour budding in colorectal cancer
              </a>
            </td>
          <td>
            Phimmada Hatthakarnkul, Natalie Fisher, Leonor Schubert Santana, H. Leslie, A. Legrini, A. Ammar, Amna Ahmed Mohemmed Malty, K. Pennel, Ian R Powley, C. Bull, P. Alexander, Janos Sztanko, Jean Quinn, Jennifer Hay, Hannah Morgan, Claire Kennedy Dietrich, Yoana Doncheva, Gerard Lynch, N. Maka, Campbell S D Roxburgh, H. V. van Wyk, Xiao Fu, Donald Mcmillan, P. Dunne, N. Jamieson, C. Thuwajit, Joanne Edwards
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Newly diagnosed prostate cancers differ dramatically in mutational composition and lethality. The most accurate clinical predictor of lethality is tumor tissue architecture, quantified as tumor grade. To interrogate the evolutionary origins of prostate cancer heterogeneity, we analyzed 666 prostate tumor whole genomes. We identified a compendium of 223 recurrently mutated driver regions, most influencing downstream mutational processes and gene expression. We identified and validated individual germline variants that predispose tumors to acquire specific somatic driver mutations: these explain heterogeneity in disease presentation and ancestry differences. High-grade tumors have a superset of the drivers in lower-grade tumors, including increased frequency of BRCA2 and MYC mutations. Grade-associated driver mutations occur early in tumor evolution, and their earlier occurrence strongly predicts cancer relapse and metastasis. Our data suggest high- and low-grade prostate tumors both emerge from a common pre-malignant field, influenced by germline genomic context and stochastic mutation-timing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9c23cc7532d469971ff1994cdcbf780a4650cf6" target='_blank'>
              The Germline and Somatic Origins of Prostate Cancer Heterogeneity.
              </a>
            </td>
          <td>
            Takafumi N. Yamaguchi, Kathleen E. Houlahan, Helen Zhu, Natalie J Kurganovs, J. Livingstone, Natalie S. Fox, Jiapei Yuan, Jocelyn Sietsma Penington, Chol-Hee Jung, T. Schwarz, Weerachai Jaratlerdsiri, J. van Riet, P. Georgeson, S. Mangiola, Kodi Taraszka, R. Lesurf, Jue Jiang, K. Chow, Lawrence E Heisler, Yu-Jia Shiah, Susmita G Ramanand, Michael J Clarkson, Anne Nguyen, S. M. G. Espiritu, R. Stuchbery, Richard Jovelin, Vincent Huang, Connor Bell, Edward O'Connor, Patrick J. McCoy, Christopher M. Lalansingh, Marek Cmero, A. Salcedo, Eva K F Chan, Lydia Y. Liu, Phillip D Stricker, Vinayak Bhandari, R. Bornman, Dorota H. Sendorek, A. Lonie, S. Prokopec, M. Fraser, J. Peters, Adrien Foucal, S. Mutambirwa, Lachlan Mcintosh, M. Orain, Matthew J. Wakefield, V. Picard, Daniel J Park, H. Hovington, Michael Kerger, A. Bergeron, Veronica Y. Sabelnykova, J-H. Seo, M. Pomerantz, Noah Zaitlen, Sebastian M. Waszak, Alexander Gusev, L. Lacombe, Yves Fradet, Andrew Ryan, A. Kishan, M. Lolkema, Joachim Weischenfeldt, Bernard Têtu, Anthony J Costello, Vanessa M Hayes, R. J. Hung, H. He, John D McPherson, Bogdan Pasaniuc, T. H. van der Kwast, AT Papenfuss, M. Freedman, B. Pope, Robert G Bristow, Ram-Shankar Mani, N. Corcoran, J. Reimand, Christopher M Hovens, P. Boutros
          </td>
          <td>2025-02-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659f72bed8c9bc066ac845266c3179b755922c46" target='_blank'>
              Epithelial cell diversity and immune remodeling in bladder cancer progression: insights from single-cell transcriptomics
              </a>
            </td>
          <td>
            Jianpeng Li, Yunzhong Jiang, Minghai Ma, Lu Wang, Minxuan Jing, Zezhong Yang, Mengzhao Zhang, Ke Chen, Jinhai Fan
          </td>
          <td>2025-01-30</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Rationale: Tumor cell epigenetics, especially chromosome accessibility, has been reported to be closely related to the tumor immune landscape and immunotherapy. However, the exact mechanism remains unknown. Methods: Whole-exome sequencing was used to analyze 13 colorectal tumor samples treated with PD1 immunotherapy. The assays for transposase-accessible chromatin using sequencing (ATAC-seq) and RNA sequencing were used to detect tumor cells' chromosome accessibility status and screen regulatory pathways. Results: Polybromo-1 (PBRM1) was one of the 12 genes with the highest frequency of somatic mutations associated with immunotherapy sensitivity. PBRM1/Pbrm1 deficiency in colorectal cancer promoted PD-1 immunotherapy sensitivity and chemotaxis of CD8+ T and NK cells in the microenvironment in vivo and in vitro. ATAC sequencing revealed that deletion of Pbrm1, a critical component of the SWI/SNF complex, increased chromosomal accessibility in tumor cells and triggered the release of cytokines, such as CCL5 and CXCL10, by activating the NF-κB signaling pathway. Application of ACBL1, a PROC inhibitor of PBRM1, in BALB/C mice or colorectal patient-derived tumor organoids (PDTOs) significantly promoted the sensitivity to PD1 antibody immunotherapy. Conclusions: Our study established that PBRM1/Pbrm1 deficiency was positively correlated with PD1 immunotherapeutic sensitivity in colorectal cancer. The underlying molecular mechanisms involved regulation of chromosome accessibility, activation of the NF-κB signaling pathway, and immune cell infiltration in the microenvironment. These findings identify potential molecular targets for enhancing immunotherapy for colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7444e0c65d77850e20585c8a5ec38a58620ecd" target='_blank'>
              PBRM1 deficiency enhances PD1 immunotherapeutic sensitivity via chromosomal accessibility in colorectal cancer
              </a>
            </td>
          <td>
            Rui Li, Jie He, Chaoqun Liu, Zesheng Jiang, Jiasheng Qin, Kun Liang, Zhuocheng Ji, Liang Zhao
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec33c3cfadc05d6eacf145975ac6bfa380605adc" target='_blank'>
              Artificial intelligence-enabled automated analysis of transmission electron micrographs to evaluate chemotherapy impact on mitochondrial morphology in triple negative breast cancer
              </a>
            </td>
          <td>
            Argenis Arriojas, Lily M. Baek, Mariah J. Berner, Alexander Zhurkevich, Antentor Hinton, M. D. Meyer, L. Dobrolecki, Michael T. Lewis, Kourosh Zarringhalam, Gloria V. Echeverria
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures.
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1178b1ac2d65f9262b22b808cb25c2ac38fa27ab" target='_blank'>
              Mutation of SMARCA4 Induces Cancer Cell-Intrinsic Defects in the Enhancer Landscape and Resistance to Immunotherapy.
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Kadir C Akdemir, Jack A Roth, Yonathan Lissanu
          </td>
          <td>2025-03-13</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e110685415835cdd67555abc6bde1dc3e70baed" target='_blank'>
              Interactions Between Ploidy and Resource Availability Shape Clonal Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, J. Teer, Didem Ilter, Richard J Beck, Jackson P Cole, Ana M Forero Pinto, Joanne D Tejero, Elisa Scanu, Thomas Veith, William Dominguez-Viqueira, Konstantin Maksin, Francisco Carrillo-Perez, Olivier Gevaert, Xiaonan Xu, F. Karreth, Mahmoud A Abdalah, Giada Fiandaca, Stefano Pasetto, Sandhya Prabhakaran, Andrew Schultz, A. M. E. Ojwang', J. Barnholtz-Sloan, Joaquim M Farinhas, Ana P. Gomes, Parag Katira, Noemi Andor
          </td>
          <td>2025-02-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4405a7f71591c3b2e80c4392e32cd43a5138621" target='_blank'>
              3D genome landscape of primary and metastatic colorectal carcinoma reveals the regulatory mechanism of tumorigenic and metastatic gene expression
              </a>
            </td>
          <td>
            Xiang Xu, Jingbo Gan, Zhaoya Gao, Ruifeng Li, Dandan Huang, Lin Lin, Yawen Luo, Qian Yang, Jingxuan Xu, Yaru Li, Qing Fang, Ting Peng, Yaqi Wang, Zihan Xu, An Huang, Haopeng Hong, F. Lei, Wensheng Huang, Jianjun Leng, Tingting Li, Xiaochen Bo, Hebing Chen, Cheng Li, Jin Gu
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ac5e3595c3552c277cd820f66cb6eb10faa0266" target='_blank'>
              MDA-9/Syntenin as a therapeutic cancer metastasis target: current molecular and preclinical understanding.
              </a>
            </td>
          <td>
            Swadesh K. Das, Paul B Fisher
          </td>
          <td>2025-03-08</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad75c766d4e07168f2e27a4c036f888ffea4b32" target='_blank'>
              Integrating model systems and genomic insights to decipher mechanisms of cancer metastasis.
              </a>
            </td>
          <td>
            Michelle M Leung, C. Swanton, N. Mcgranahan
          </td>
          <td>2025-03-10</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eee1a8b871b1885a1b34200cb093a48900beddf" target='_blank'>
              The phenogenomic landscapes of pleural mesothelioma tumor microenvironment predict clinical outcomes
              </a>
            </td>
          <td>
            Federica Torricelli, Filomena Spada, Cynthia Bishop, K. Todd, Daisuke Nonaka, N. Petrov, Manuela Terranova Barberio, Alan G Ramsay, Richard Ellis, A. Ciarrocchi, B. Apollonio, Andrea Billè
          </td>
          <td>2025-02-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/756a3a4550f68c97b6f384cfa81b3289c4be032f" target='_blank'>
              Expression of ENL YEATS domain tumor mutations in nephrogenic or stromal lineage impairs kidney development
              </a>
            </td>
          <td>
            Zhaoyu Xue, Hongwen Xuan, Kin Lau, Yangzhou Su, Marc Wegener, Kuai Li, Lisa Turner, Marie Adams, Xiaobing Shi, Hong Wen
          </td>
          <td>2025-03-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75bdacf15f78eabb80ad03155a6f7ee66350fd9" target='_blank'>
              Genomic alterations and transcriptional phenotypes in circulating free DNA and matched metastatic tumor
              </a>
            </td>
          <td>
            N. Takahashi, L. Pongor, Shivam P. Agrawal, Mariya Shtumpf, Ankita Gurjar, V. Rajapakse, Ahmad Shafiei, Christopher W. Schultz, Sehyun Kim, Diana Roame, Paula Carter, R. Vilimas, Samantha Nichols, Parth Desai, W. Figg, Mohammad Bagheri, V. B. Teif, Anish Thomas
          </td>
          <td>2025-02-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90df80a01384ee300b159e30e515e05880476d0f" target='_blank'>
              Artificial intelligence-based spatial analysis of tertiary lymphoid structures and clinical significance for endometrial cancer
              </a>
            </td>
          <td>
            Haruka Suzuki, Kohei Hamada, J. Hamanishi, Akihiko Ueda, R. Murakami, Mana Taki, Rin Mizuno, Koichi Watanabe, Hanako Sato, Y. Hosoe, Hiroaki Ito, Koji Yamanoi, Hiroyuki Yoshitomi, N. Kakiuchi, Ken Yamaguchi, N. Matsumura, Seishi Ogawa, Hideki Ueno, Masaki Mandai
          </td>
          <td>2025-02-01</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd1722df9025221021ddfd3a6fcbf675ba9705b" target='_blank'>
              OLIG2 mediates a rare targetable stem cell fate transition in sonic hedgehog medulloblastoma
              </a>
            </td>
          <td>
            Kinjal Desai, Siyi Wanggou, Erika Luis, H. Whetstone, Chunying Yu, R. Vanner, Hayden J. Selvadurai, Lilian Lee, Jinchu Vijay, J. E. Jaramillo, Jerry J. Fan, P. Guilhamon, M. Kushida, Xuejun Li, Gregory Stein, Santosh Kesari, B. D. Simons, Xi Huang, Peter B. Dirks
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0b1fd2e12bcf45ef4e0b8a0b897b112e846447" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Takafumi N. Yamaguchi, Yu-Jia Shiah, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, Amar U. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e7693290b0f56e7e4e347ce73c837cd2d47cd1" target='_blank'>
              Stromal architecture and fibroblast subpopulations with opposing effects on outcomes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yifei Cheng, Xiaofang Chen, Li Feng, Zhicheng Yang, Liyun Xiao, Bin Xiang, Xiaodong Wang, Dongbing Liu, Penghui Lin, Jieyi Shi, Guohe Song, Wulei Qian, Boan Zhang, Yanan Xu, Zheng Gao, Lv Chen, Y. Wu, Jiaqiang Ma, Youpei Lin, Haichao Zhao, Lihua Peng, Xuebin Mao, Yang Liu, Hao Hou, Mingyu Yang, Yuan Ji, Xiaoying Wang, Jian Zhou, Xun Xu, Xiyang Liu, Wu Wei, Xiaoming Zhang, Qiang Gao, Hu Zhou, Yidi Sun, Kui Wu, Jia Fan
          </td>
          <td>2025-01-28</td>
          <td>Cell Discovery</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/197782f3d8b81c382c4e0d5197a6f09c7baeb2a9" target='_blank'>
              Immune microenvironment spatial landscapes of tertiary lymphoid structures in gastric cancer
              </a>
            </td>
          <td>
            Yi Xie, Haoxin Peng, Yajie Hu, Keren Jia, Jiajia Yuan, Dan Liu, Yanyan Li, Xujiao Feng, Jian Li, Xiaotian Zhang, Yu Sun, Lin Shen, Yang Chen
          </td>
          <td>2025-02-04</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf00ce3ce13ab5e7ee7ff1ae10bf120f0be9305e" target='_blank'>
              Molecular biology of the deadliest cancer – glioblastoma: what do we know?
              </a>
            </td>
          <td>
            Aly Ismailov, Aldo Spallone, Alexey Belogurov, Alan Herbert, Maria Poptsova
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47b5ee70b22223e164afab1c522ee489fff66db" target='_blank'>
              Deep learning prioritizes cancer mutations that alter protein nucleocytoplasmic shuttling to drive tumorigenesis
              </a>
            </td>
          <td>
            Yongqiang Zheng, Kai Yu, Jin-Fei Lin, Zhuoran Liang, Qingfeng Zhang, Junteng Li, Qi-Nian Wu, Cai-Yun He, Mei Lin, Qi Zhao, Zhi-Xiang Zuo, Huai-qiang Ju, Rui-Hua Xu, Ze-xian Liu
          </td>
          <td>2025-03-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098b5888d894097b207e005b18110c0c81abf3d8" target='_blank'>
              Modeling cell differentiation in neuroblastoma: insights into development, malignancy, and treatment relapse
              </a>
            </td>
          <td>
            Simon F. Martina-Perez, Luke A. Heirene, Jennifer C. Kasemeier, P. Kulesa, Ruth E. Baker
          </td>
          <td>2025-02-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9629af3f9c8496403aa9d7445a8f7f79cca8a69" target='_blank'>
              Genes and proteins expression profile of 2D vs 3D cancer models: a comparative analysis for better tumor insights.
              </a>
            </td>
          <td>
            Sunaina Bhuker, Abhinav Kumar Sinha, Anuksha Arora, H. Tuli, Sonal Datta, A. Saini, R. Saini, S. Ramniwas
          </td>
          <td>2025-01-25</td>
          <td>Cytotechnology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The development of therapies that target cancer stem cells (CSCs) and bulk tumors is both crucial and urgent. Several signaling pathways, like Notch and Hedgehog (Hh), have been strongly associated with CSC stemness maintenance and metastasis. However, the extensive crosstalk present between these two signaling networks complicates the development of long-term therapies that also minimize adverse effects on healthy tissues and are not overcome by therapy resistance from CSCs. The present work aims to overview the roles of Notch and Hh in cancer outburst and the intersection of the two pathways with one another, as well as with other networks, such as Wnt/β-catenin, TGF, and JAK/STAT3, and to explore the shaping of the tumor microenvironment (TME) with specific influence on CSC development and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f140e4496eee327489d912de5427944cb351ca1" target='_blank'>
              Notch and Hedgehog Signaling Unveiled: Crosstalk, Roles, and Breakthroughs in Cancer Stem Cell Research
              </a>
            </td>
          <td>
            Sabina Iluta, Mădălina Nistor, Sanda Buruiana, D. Dima
          </td>
          <td>2025-02-01</td>
          <td>Life</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8641c47a99cb4e08fcabebc4fec5794fc9e23e2f" target='_blank'>
              Role of CD4+ T cells in cancer immunity: a single-cell sequencing exploration of tumor microenvironment
              </a>
            </td>
          <td>
            Qi An, Li Duan, Yuanyuan Wang, Fuxin Wang, Xiangyu Liu, Chao Liu, Qinyong Hu
          </td>
          <td>2025-02-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumor microenvironment (TME) milieu directs a plethora of tumor-modulating functions. Recent years have seen pivotal breakthroughs in our understanding of the TME's role in tumor initiation and progression, with tangible clinical applications. Individual components of the TME exert their function predominantly by cell-cell crosstalk, both in the form of physical interaction and secreted factors. Notably, different spatial niches represent exclusive signaling hubs in the TME, propagating pro- or antitumoral functions. The exploration of these interactions has been vastly facilitated by novel molecular technologies, each of which provides a different perspective on this intricate intercellular communication network. Together, these complementary methods paint a detailed, high-resolution map of the TME's interaction landscape. In this viewpoint, we explore how cellular interactions can unlock a new level of understanding of TME complexity, and discuss the promises and challenges of characterizing tumors based on their cellular interaction footprint.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/072aea9d500a6665bd7f3f737b00c34169b4d84a" target='_blank'>
              Decoding multicellular interaction networks-a new horizon in tumor microenvironment research.
              </a>
            </td>
          <td>
            Roi Balaban, Merav Cohen
          </td>
          <td>2025-02-05</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b37779136ebd9daabbf26c5f811d4e44cc19986" target='_blank'>
              Spatially resolved transcriptomics and graph-based deep learning improve accuracy of routine CNS tumor diagnostics
              </a>
            </td>
          <td>
            M. Ritter, C. Blume, Yiheng Tang, A. Patel, B. Patel, N. Berghaus, Jasim Kada Benotmane, J. Kueckelhaus, Y. Yabo, Junyi Zhang, E. Grabis, Giulia Villa, David Niklas Zimmer, Amir Khriesh, P. Sievers, Z. Seferbekova, F. Hinz, V. Ravi, M. Seiz-Rosenhagen, M. Ratliff, C. Herold-Mende, Oliver Schnell, J. Beck, Wolfgang Wick, Andreas von Deimling, M. Gerstung, D. Heiland, F. Sahm
          </td>
          <td>2025-01-29</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c52c76852d3ba58984faf618de957dc0aff431e" target='_blank'>
              Spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis
              </a>
            </td>
          <td>
            A. Vannan, Ruqian Lyu, A. L. Williams, N. Negretti, E. D. Mee, J. Hirsh, S. Hirsh, Niran Hadad, David Nichols, C. Calvi, Chase J Taylor, V. Polosukhin, A. P. Serezani, A. S. McCall, J. Gokey, Heejung Shim, Lorraine B. Ware, M. Bacchetta, C. Shaver, Timothy S. Blackwell, Rajat Walia, J. Sucre, JA Kropski, Davis J. McCarthy, N. Banovich
          </td>
          <td>2025-02-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="ABSTRACT The lack of natural aging‐inducing Alzheimer's disease (AD) model presents a significant gap in the current preclinical research. Here, we identified a unique cohort of 10 naturally aging tree shrews (TSs) displaying distinct Alzheimer's‐like pathology (ALP) from a population of 324, thereby establishing a novel model that closely mirrors human AD progression. Using single‐nucleus RNA sequencing, we generated a comprehensive transcriptome atlas, revealing the cellular diversity and gene expression changes underlying AD pathology in aged TSs. Particularly, distinct differentiation trajectories of neural progenitor cells were highly associated with AD pathology. Intriguingly, cross‐species comparisons among humans, TSs, monkeys, and mice highlighted a greater cellular homogeneity of TSs to primates and humans than to mice. Our extended cross‐species analysis by including a direct comparison between human and TS hippocampal tissue under AD conditions uncovered conserved cell types, enriched synaptic biological processes, and elevated excitatory/inhibitory imbalance across species. Cell–cell communication analysis unveiled parallel patterns between AD human and ALP TSs, with both showing reduced interaction strength and quantity across most cell types. Overall, our study provides rich, high‐resolution resources on the cellular and molecular landscape of the ALP TS hippocampus, reinforcing the utility of TSs as a robust model for AD research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed955c468c754bc7548c36de5110f980925f515b" target='_blank'>
              A Single‐Nucleus Transcriptomic Atlas Reveals Cellular and Genetic Characteristics of Alzheimer's‐Like Pathology in Aging Tree Shrews
              </a>
            </td>
          <td>
            Liulin Xiong, Rong He, Ruo-Lan Du, Rui-Ze Niu, Lu-Lu Xue, Li Chen, Li-Ren Huangfu, Qiu-Xia Xiao, Jing Li, Yong-Ping Li, Si‐Min Zhang, Chang-yin Yu, Xiao-He Tian, Ting-Hua Wang
          </td>
          <td>2025-03-15</td>
          <td>MedComm</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0786e178852bb9b6b2a0945adb3bddac988e519" target='_blank'>
              Abstract A030: A deeper understanding of tumor microenvironment modulation induced by IL13Ra2 CAR T cells in a patient with grade 3 ependymoma
              </a>
            </td>
          <td>
            Maryam Aftabizadeh, B. Aguilar, M. D’Apuzzo, B. Badie, Christine E. Brown
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21d853448d817f079816c9b14dd1957f3e34109e" target='_blank'>
              The Pre-metastatic Niche: How Cancer Stem Cell-Derived Exosomal MicroRNA Fit into the Puzzle.
              </a>
            </td>
          <td>
            Behjat Kheiri Yeghaneh Azar, F. Vakhshiteh
          </td>
          <td>2025-03-17</td>
          <td>Stem cell reviews and reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1ae2ca598b5cdc041c4c163b222a0b0f6d55027" target='_blank'>
              Tumor suppressor collateral damage screens reveal mRNA homeostasis protein HBS1L as a novel vulnerability in ch9p21 driven FOCAD deleted cancer
              </a>
            </td>
          <td>
            Hongxiang Zhang, M. Tonini, Lauren Martires, Charlotte B Pratt, Helena N. Jenkins, Eden N. Gordon, Shanchuan Zhao, Ashley H Choi, Samuel R. Meier, Tenzing Khendu, Shangtao Liu, B. Shen, Hannah Stowe, Katerina Pashiardis, Xuewen Pan, Madhavi Bandi, Minjie Zhang, Yi Yu, Chengyin Min, A. Huang, Jannik N Andersen, Hilary E. Nicholson, T. Teng
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b56368d398e6734b66e55599843f0e9cefd36d23" target='_blank'>
              Geometric immunosuppression in CAR-T cell treatment: Insights from mathematical modeling
              </a>
            </td>
          <td>
            Silvia Bordel-Vozmediano, S. Sabir, Luc'ia Benito-Barca, B. Weigelin, V'ictor M. P'erez-Garc'ia Mathematical Oncology Laboratory, Instituto de Matem'atica Aplicada a la Ciencia y la Ingenier'ia, U. C. Mancha, Ciudad Real, Spain., D. D. Matem'aticas, Escuela T'ecnica Superior de Ingenieros Industriales, Facultad de Ciencias Experimentales, Universidad Francisco de Vitoria, Pozuelo de Alarc'on, Department of Functional Imaging, Radiopharmacy, Multiscale Immunoimaging, E. University, Tubingen, Germany, Cluster of Excellence iFIT ''Image-Guided, Functionally Instructed Tumor Therapies''
          </td>
          <td>2025-01-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Image‐based screening is an efficient drug discovery strategy. Combining high‐content screening (HCS) with artificial intelligence efficiently detects drug‐induced cellular phenotypes from a large pool of compounds. However, previous studies primarily focus on cell classification and cannot interpret models using morphological features. Additionally, the existing morphology‐based studies use only few features, which cannot accurately characterize cell phenotypic perturbations. Herein, π‐PhenoDrug, a deep learning‐based pipeline, is developed for cell phenotype‐driven drug screening. It integrates cell segmentation, morphological profile construction, and phenotype analysis modules of HCS processes. π‐PhenoDrug is applied to evaluate drug response in various human melanoma cell lines. The results demonstrate that π‐PhenoDrug can evaluate drug killing effects across different cell lines via both supervised and unsupervised modes. Furthermore, π‐PhenoDrug identifies drugs with potential killing effects on melanoma cells from a library of diverse compounds. Compared with traditional single‐readout assays, this method is more sensitive to compounds that induce weaker phenotypes, reducing the risk of overlooking effective drugs. These results confirm that π‐PhenoDrug can achieve high‐throughput and accuracy analysis of cell phenotypic data using an unbiased and automated workflow, improving drug discovery efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d59d452770f50a368d33283eccf620d3785158da" target='_blank'>
              π‐PhenoDrug: A Comprehensive Deep Learning‐Based Pipeline for Phenotypic Drug Screening in High‐Content Analysis
              </a>
            </td>
          <td>
            Xiao Li, Qinxue Ouyang, Mingfei Han, Xiaoqing Liu, Fuchu He, Yunping Zhu, Ling Leng, Jie Ma
          </td>
          <td>2025-03-02</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Liver cancer exhibits diverse molecular characteristics and distinct immune cell infiltration patterns, which significantly influence patient outcomes. In this study, we thoroughly examined the liver cancer tumor environment by analyzing data from 419,866 individual cells across nine datasets involving 99 patients. By categorizing patients into different groups based on their immune cell profiles, including immune deficiency, B cells-enriched, T cells-enriched and macrophages-enriched, we better understood how these cells change in various patient subgroups. Our investigation of liver metastases from intestinal cancer uncovered a group of mast cells that might promote metastasis through pathways like inositol phosphate metabolism. Using genomic and clinical data from The Cancer Genome Atlas, we identified specific cell components linked to tumor characteristics and genetics. Our detailed study of cancer-associated fibroblasts (CAFs) revealed how they adapt and acquire new functions in the tissue environment, highlighting their flexibility. Additionally, we found a significant connection between CAF-related genes and the prognosis of hepatocellular carcinoma patients. This research provides valuable insights into the makeup of the liver cancer tumor environment and its profound impact on patient outcomes, offering fresh perspectives for managing this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dff3872da8e8b83984c596296a89bdd2fda835d8" target='_blank'>
              Deciphering the heterogeneity and plasticity of the tumor microenvironment in liver cancer provides insights for prognosis
              </a>
            </td>
          <td>
            Yihao Sun, Guojuan Shi, Jian Yang, Chun-Zhong Zhou, Chuhan Peng, Yu-Hong Luo, Ying Pan, Rui-Qi Wang
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f184d245ff8ab9e1166b05912194311eabc4fc2" target='_blank'>
              Abstract A034: Cancer neutrophil encyclopedia: A deep dive into antigen-presenting warriors
              </a>
            </td>
          <td>
            Y. Wu
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93bd105f231e9996b6a3ab55c7d2018c2946edbd" target='_blank'>
              HemaScope: A Tool for Analyzing Single-cell and Spatial Transcriptomics Data of Hematopoietic Cells.
              </a>
            </td>
          <td>
            Zhenyi Wang, Yuxin Miao, Hongjun Li, Wenyan Cheng, Minglei Shi, Lv Gang, Yating Zhu, Junyi Zhang, Tingting Tan, Jin Gu, Michael Q. Zhang, Jianfeng Li, Hai Fang, Zhu Chen, Saijuan Chen
          </td>
          <td>2025-01-25</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0ef45b0d0460ea2916e782f771c000702057b" target='_blank'>
              1347 Using Living Human Glioblastoma Samples to Develop Tumor-Targeted AAV Gene Therapy Vectors
              </a>
            </td>
          <td>
            M. Guevara, JP McGinnis, Joshua Ortiz-Guzman, Sai Mallannagari, Daniel Curry, Benjamin R. Arenkiel
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/806ce0b8ce24491e542e1befccfa41c6236040a0" target='_blank'>
              Circulating tumor cell (CTC) expression patterns of cell surface targets in metastatic prostate cancer.
              </a>
            </td>
          <td>
            Kendra D. Marr, J. Sperger, Amy K Taylor, Katharine E Tippins, Shannon R Reese, K. Helzer, M. Bootsma, G. Blitzer, John M Floberg, D. Kosoff, Rana R McKay, Xiao X Wei, Shuang Zhao, Joshua M Lang, Marina N. Sharifi
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059cc807139dc64898d45632991b55c44cdfb1d6" target='_blank'>
              Abstract A108: Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients
              </a>
            </td>
          <td>
            Muyi Yang, Sólrún Kolbeinsdóttir, Katarina Hammarlund, V. Grozman, Martin Enge, Suzanne Egyhazi-Brage, H. Helgadottir
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8227f78d1879affbb26ec2351982ff6e814cbb7b" target='_blank'>
              Abstract P004: Mapping the tumor-sentinel node immune migratome demonstrates a key role for CCR7+ dendritic cells in the successful response to immunoradiotherapy
              </a>
            </td>
          <td>
            R. Saddawi-Konefka, Riyam Al Msari, Shiqi Tang, Lauren M. Clubb, Santiago Fassardi, Riley Jones, F. Faraji, Shiruyeh Schokrpur, Bryan S. Yung, M. Allevato, S. Jensen, B. A. Fox, R. Bell, J. S. Gutkind, Andrew Sharabi, Joseph A Califano
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature Cancer</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50aaf113e900513b3c01d9cc2408109f9601e35a" target='_blank'>
              The MLL3/GRHL2 complex regulates malignant transformation and anti-tumor immunity in squamous cancer.
              </a>
            </td>
          <td>
            Chehyun Nam, Guowei Huang, Yueyuan Zheng, Hua Zhao, Yuhao Pan, Boyan Hu, Talia Wenger, Hieu T. Van, Liyan Xu, E. Li, H. Koeffler, Kai Ge, Y. Dou, Uttam K Sinha, Young Min Park, De-Chen Lin
          </td>
          <td>2025-02-18</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d74a99d567922b268e37075173798f1b540e4d47" target='_blank'>
              Intercellular mitochondrial transfer contributes to microenvironmental redirection of cancer cell fate.
              </a>
            </td>
          <td>
            J. S. Bjerring, Yara Khodour, Emilee Anne Peterson, Patrick C. Sachs, Robert David Bruno
          </td>
          <td>2025-02-11</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f7b4d0623804b6e4ad33245c8e8741d967977b" target='_blank'>
              Relationship of spatial transcriptomic analyses with distance-associated transcriptomic spectrum and morphology-specific signatures of localized prostate cancer.
              </a>
            </td>
          <td>
            Brendan Raizenne, Hanbing Song, Nancy Y. Greenland, Maggie Tsui, Jared Khan, Taiqi Li, Chih-Hao Chang, M. Cooperberg, Franklin W. Huang
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ceeafcf1da623087d79a8767b5543bf2a7043f8" target='_blank'>
              Genomic and micro-environmental insights into drug resistance in colorectal cancer liver metastases
              </a>
            </td>
          <td>
            Junjie Kuang, Jun Li, Siwei Zhou, Yi Li, Jinbo Lin, Wei-zeng Huang, Xia Yuan
          </td>
          <td>2025-02-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Intrahepatic cholangiocarcinoma (ICC) is a highly invasive bile duct cancer with poor prognosis due to frequent recurrence and limited effective treatments. Cancer stem cells (CSCs) contribute to ICC’s therapeutic resistance and recurrence, driven by distinct cellular subpopulations with variable tumorigenic properties. Recent advances in single-cell RNA sequencing (scRNA-seq) have enabled a deeper exploration of cellular heterogeneity in tumors, offering insights into unique CSC subgroups that impact ICC progression and patient outcomes. This study aimed to investigate the effect of CSC heterogeneity on the prognosis of ICC. Methods The scRNA-seq dataset GSE142784 was retrieved from the Gene Expression Omnibus (GEO) database, and Bulk RNA-seq data were obtained from The Cancer Genome Atlas (TCGA) databases. Hallmarks and AUCell R package were adopted for analyzing the signaling pathway activity, CellChat for observing cell communication between subgroups, and SCENIC for analyzing transcription factors expression. The immune cell infiltration and drug sensitivity of the model were analyzed using the CIBERSORT algorithm and the “pRRophetic” R packages, respectively. And immunohistochemistry (IHC) tests were used to evaluate expression of transcription factors in ICC patients. Results Based on scRNA-seq data, five clusters (DLK+, CD13+, CD90+, CD133+, and other cholangiocarcinoma cells) were observed in ICC, which presented different signaling pathway activities, such as HSF1 and STAT1 were highly expressed in the CD133 cluster, and consistent with the results of IHC tests. Pathways like Notch and Wnt/β-catenin signaling transferred among above subgroups. Further, subgroups favored varied immune response and drug sensitivity, and CD133+ subgroup patients showed significantly shortened recurrence-free survival (RFS). Conclusions Configuring the subgroup of ICC is helpful for predicting the prognosis and drug resistance in ICC and can provide new strategies for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05f80675fdf55da009309dc76350f2eac30cf105" target='_blank'>
              Role of cancer stem cell heterogeneity in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Yiwang Zhang, Juping Xie, Xiangqi Huang, Jintian Gao, Zhiyong Xiong
          </td>
          <td>2025-02-01</td>
          <td>Translational Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80467786e28909f1b20adae7a9b9f2d12bea9a9a" target='_blank'>
              Pan-cancer bioinformatics analysis of TIPRL in human tumors
              </a>
            </td>
          <td>
            Xuezhong Zhang, Hao Xue, Yuanyuan Lv, Yuntao Zhou, Kaihui Sha, Tonggang Liu
          </td>
          <td>2025-03-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97569fe84c28d5e70fd4d5d44460cfdc4508b03d" target='_blank'>
              Quantifying cell divisions along evolutionary lineages in cancer.
              </a>
            </td>
          <td>
            Martin Blohmer, David M Cheek, Wei-Ting Hung, Maria Kessler, Foivos Chatzidimitriou, Jiahe Wang, William Hung, I-Hsiu Lee, Alexander N Gorelick, Emma Ce Wassenaar, Ching-Yeuh Yang, Yi-Chen Yeh, H. Ho, Dorothee Speiser, Maria M Karsten, Michael Lanuti, Sara I Pai, O. Kranenburg, J. Lennerz, Teh-Ying Chou, Matthias Kloor, Kamila Naxerova
          </td>
          <td>2025-02-04</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>56</td>
        </tr>

        <tr id="Background Head and neck cancers harbor dormant cancer stem cells (CSCs). This study explores how platinum therapy impacts these cells in a non-genetic manner and the role of hypoxia in this process. Previously, we identified a novel population of CSCs exhibiting an “altruistic” phenotype, sacrificing self-renewal to promote niche defense (tumor stemness defense, TSD), potentially protecting a dormant subpopulation of CSCs, the reawakening CSC (R-CSC) retaining stress memory. This TSD phenotype involves the activation of the MYC-HIF2α pathway and, importantly, is linked to a hypoxic tumor microenvironment. We termed these TSD+ CSCs “altruistic cancer stem cells” (A-CSCs). Here we investigated the potential role of tumor hypoxia in the mobilization of TSD+ CSCs to the circulation as a part of niche defense against platinum therapy. Methods We isolated CTCs and primary tumor cells from head and neck squamous cell carcinoma (HNSCC) patients undergoing platinum therapy (n = 14). We analyzed the TSD phenotype and markers of hypoxia in these cells. Additionally, we further characterized a previously reported pre-clinical model of platinum-induced tumor stemness to study the link between hypoxia, TSD+ CSC emergence, and mobilization to the circulation and bone marrow. Results We isolated TSD+ CTCs with a hypoxic signature from eight out of 14 HNSCC patients. These cells displayed increased proliferation and invasion upon cisplatin treatment, suggesting a role in niche defense. Our pre-clinical model confirmed that hypoxia directly correlates with the expansion of TSD+ CSCs and their mobilization into the circulation and bone marrow following cisplatin treatment. We demonstrated the protection of R-CSCs by TSD+ CSCs. Notably, inhibiting hypoxia alone with tirapazamine did not reduce TSD+ CSCs, CTCs, or R-CSCs. However, combining tirapazamine with FM19G11, a MYC-HIF2α pathway inhibitor, significantly reduced the platinum-induced expansion of both TSD+ CSCs, CTCs, and the presence of R-CSCs in the bone marrow. Conclusions This study reveals that HNSCC patients undergoing platinum therapy can harbor TSD+ CTCs exhibiting an altruistic phenotype and a hypoxic signature. Additionally, the pre-clinical study provides a novel non-genetic mechanism of therapy resistance-the altruistic tumor self-defense. The tumor microenvironment, through the emergence of TSD+ CSCs, appears to act collectively to defend the tumor self-identity by hijacking an altruistic stem cell niche defense mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc47428ef4dce00c70a03b6d84031451c648467" target='_blank'>
              Hypoxia-driven mobilization of altruistic cancer stem cells in platinum-treated head and neck cancer
              </a>
            </td>
          <td>
            Lekhika Pathak, Bidisha Pal, Joyeeta Talukdar, Parthajyoti Saikia, Sorra Sandhya, Wale Tasabehji, Hong Li, Jyotirmoy Phukan, Anjan Bhuyan, Sanjukta Patra, Bikul Das
          </td>
          <td>2025-02-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7e12e22717ab1e00830f945fd4f97470fd6160b" target='_blank'>
              Spatially resolved, multi-region proteomics for prediction of immunotherapy outcome in deficient mismatch repair metastatic colorectal cancer.
              </a>
            </td>
          <td>
            B. Saberzadeh-Ardestani, Zhenglong Liu, Mariam I Stein, Will A Sherman, Christy E Trussoni, Charles W Abbott, Dongyao Yan, Skyler Smith, K. Shanmugam, Rondell P. Graham, Alos Diallo, Joshua J. Levy, Tamas Ordog, F.A. Sinicrope
          </td>
          <td>2025-02-19</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d479e3aee16692284dd5875a578cce134082060b" target='_blank'>
              Cell signaling communication within papillary craniopharyngioma revealed by an integrated analysis of single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Xiaoyue Zhu, Yanfei Jia, Zicheng Zhao, Xiaoyu Zhang, Yunlong Zhao, Songbai Gui, Xiu-An Yang
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29965c5d14b51c6573686cfc61d8b621a40178b6" target='_blank'>
              Abstract IA019: Addressing genomic heterogeneity in glioblastoma personalized vaccine design
              </a>
            </td>
          <td>
            Elaine R Mardis, E. Garfinkle, T. Johanns, Katherine E Miller, Gavin P Dunn
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1610b8e09ca06ead7a9826cda40c37396bdae40e" target='_blank'>
              Atlas of multilineage stem cell differentiation reveals TMEM88 as a developmental regulator of blood pressure
              </a>
            </td>
          <td>
            Sophie Shen, Tessa Werner, S. Lukowski, Stacey Andersen, Yuliangzi Sun, W. J. Shim, Dalia Mizikovsky, Sakurako Kobayashi, Jennifer Outhwaite, Han Sheng Chiu, Xiaoli Chen, G. Chapman, E. M. Martin, Di Xia, Duy Pham, Zezhuo Su, Daniel Kim, Pengyi Yang, Men Chee Tan, Enakshi Sinniah, Qiongyi Zhao, Sumedha Negi, Meredith A. Redd, Joseph E Powell, Sally L. Dunwoodie, Patrick P L Tam, Mikael Bodén, J. W. Ho, Quan H Nguyen, N. Palpant
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa65e1fd6a4ddcb57e0eaf994498d400b2eb5d2" target='_blank'>
              Tissue-specific mutagenesis from endogenous guanine damage is suppressed by Polκ and DNA repair
              </a>
            </td>
          <td>
            Yang Jiang, M. Przybilla, Linda Bakker, Foster C. Jacobs, Dylan Mckeon, Roxanne van der Sluijs, Juliette Wezenbeek, Joeri van Strien, Jeroen Willems, Alex E E Verkennis, Jamie Barnett, Adrian Baez Ortega, Federico Abascal, Peter W. Villalta, P. Knipscheer, I. Martincorena, S. Balbo, Juan Garaycoechea
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c13871b20a0df0f49db2baf60ca96fdc2c42238" target='_blank'>
              Spatially-structured inflammatory response in the presence of a uniform stimulus
              </a>
            </td>
          <td>
            Elizabeth R. Jerison, Nicolas Romeo, Stephen R. Quake
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57fc0173898db1615005d0ab09ce022fdfc322eb" target='_blank'>
              The Transcriptomic Landscapes of Appendiceal Primary and Metastatic Tumors are Distinct.
              </a>
            </td>
          <td>
            Margaret A. Park, Richard Jacobson, Maria Genilo-Delgado, Amir Mohammadi, Carlos M. Morán-Segura, Solomon Alhassan, Yukihiro Nakanishi, J. Permuth, I. Imanirad, S. Dineen
          </td>
          <td>2025-02-22</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bdfee01c97ef1e7e57c28ea058914128289212e" target='_blank'>
              Establishment and transcriptomic characteristics of radio-resistant meningioma cell lines.
              </a>
            </td>
          <td>
            Jinxiu Yu, L. Ren, Tianqi Wu, Lingyang Hua, Daijun Wang, Yang Wang, Qing Xie, Jiaojiao Deng, Ye Gong
          </td>
          <td>2025-02-28</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/516e69fd130c4ec40a2f332c478c1dccdc4e566a" target='_blank'>
              Systematic inference of super-resolution cell spatial profiles from histology images
              </a>
            </td>
          <td>
            Peng Zhang, Chaofei Gao, Zhuoyu Zhang, Zhiyuan Yuan, Qian Zhang, Ping Zhang, Shiyu Du, Weixun Zhou, Yan Li, Shao Li
          </td>
          <td>2025-02-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81dd756da5910df487dda469964a4404cb5624ff" target='_blank'>
              Methodology of murine lung cancer mimics clinical lung adenocarcinoma progression and metastasis
              </a>
            </td>
          <td>
            Edison Q Kim, Emily Y. Kim, Eric P Knott, Yujie Wang, Cheng-Bang Chen, Jose R Conejo-Garcia, M. Wangpaichitr, Diane C Lim
          </td>
          <td>2025-02-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Germline mutations in the tumor suppressor phosphatase and tensin homolog (PTEN) cause PTEN hamartoma tumor syndrome (PHTS). PHTS is characterized by an elevated lifetime risk of differentiated thyroid cancer (DTC), 30 times higher than the general population. However, only 1 in 3 PHTS patients develop DTC, and it remains unknown whether specific PTEN variants are associated with an increased risk of DTC. PTEN antagonizes the phosphatidylinositol 3-kinase (PI3K)–AKT signaling pathway, a frequently affected pathway in sporadic DTC. PTEN also acts as a guardian of the genome by interacting with other tumor suppressors. Here, we report how ionizing radiation, an environmental tumorigenic contributor, modifies the DNA damage response based on the type of germline PTEN variants. We hypothesized that certain PTEN variants associated with DTC create a pro-oncogenic molecular signature upon radiation-induced DNA damage. DTC-associated (PTENM134R) or DTC-non-associated (PTENG132D) germline PTEN mutant alleles were introduced into a human induced pluripotent cell (hiPSC) line derived from a healthy donor utilizing CRISPR-Cas9 gene editing technology. We determined radiation-induced transcriptomic changes in functional thyroid organoids induced from wild-type and both heterozygous PTEN mutant hiPSCs. Both bulk and single-cell RNA sequencing data indicated that radiation upregulated the p53 network more potently in the thyroid organoids with PTENWT/G132D than those with PTENWT/M134R, which could be mediated by AKT-dependent MDM2 inactivation and PTEN–p53 physical interaction. Our data suggest that the lack of p53 pathway activation through PTEN–p53 network interactions explains why PTENM134R is a DTC-susceptible variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c92c95603d11b5160f145d679137c4a2e82b3ae" target='_blank'>
              Divergent PTEN–p53 interaction upon DNA damage in a human thyroid organoid model with germline PTEN mutations
              </a>
            </td>
          <td>
            Juan Venegas, Omer Enes Onur, Shin Chung Kang, Masahiro Hitomi, C. Eng
          </td>
          <td>2025-02-01</td>
          <td>Endocrine-Related Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Advances in spatially-resolved transcriptomics (SRT) technologies have propelled the development of new computational analysis methods to unlock biological insights. The lowering cost of SRT data generation presents an unprecedented opportunity to create large-scale spatial atlases and enable population-level investigation, integrating SRT data across multiple tissues, individuals, species, or phenotypes. Here, unique challenges are described in the SRT data integration, where the analytic impact of varying spatial and biological resolutions is characterized and explored. A succinct review of spatially-aware integration methods and computational strategies is provided. Exciting opportunities to advance computational algorithms amenable to atlas-scale datasets along with standardized preprocessing methods, leading to improved sensitivity and reproducibility in the future are further highlighted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76adb13a2a1a29db8847f590c746a2e59e284b7" target='_blank'>
              Integrating Spatially-Resolved Transcriptomics Data Across Tissues and Individuals: Challenges and Opportunities.
              </a>
            </td>
          <td>
            Boyi Guo, Wodan Ling, S. H. Kwon, Pratibha Panwar, S. Ghazanfar, Keri Martinowich, SC Hicks
          </td>
          <td>2025-02-11</td>
          <td>Small methods</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cc0740c1e17e6a1efafc307103e61f95f2a3a59" target='_blank'>
              Mathematical modeling of multicellular tumor spheroids quantifies inter-patient and intra-tumor heterogeneity
              </a>
            </td>
          <td>
            Adam A. Malik, Kyle C. Nguyen, John T. Nardini, C. Krona, Kevin B. Flores, Sven Nelander
          </td>
          <td>2025-02-15</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7891a3a8df530bd356f3a5da9253356957c1253d" target='_blank'>
              Hyper-mutational processes provide a head-start for non-optimal cancer driver mutations explaining atypical KRAS variants
              </a>
            </td>
          <td>
            Michael D. Nicholson, Ian Tomlinson
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cee75be70d4938d7f15deed0722cf5fa17de26a" target='_blank'>
              The Biology in the Pattern: Metastatic organotropism and clinical outcome depend on DNA damage response and immune interactions in pancreatic cancer.
              </a>
            </td>
          <td>
            A. Papargyriou, Maximilian Reichert
          </td>
          <td>2025-03-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Variants of uncertain significance (VUS) limit the actionability of genetic testing. A prominent example is MUTYH, a base excision repair factor associated with polyposis and colorectal cancer, which has a pathogenic variant carrier rate approaching 1 in 50 individuals in some populations. To systematically interrogate variant function in MUTYH, we coupled deep mutational scanning with a DNA repair reporter containing its lesion substrate, 8OG:A. Our variant-to-function map covers >97% of all possible MUTYH point variants (n=10,941) and achieves 100% accuracy classifying the pathogenicity of known clinical variants (n=247). Leveraging a large clinical registry, we observe significant associations with colorectal polyps and cancer, with more severely impaired missense variants conferring greater risk. We recapitulate known functional differences between pathogenic founder alleles, and highlight sites of complete missense intolerance, including residues that intercalate DNA and coordinate essential Zn2+ or Fe-S clusters. This map provides a resource to resolve the 1,032 existing missense VUS and 90 variants with conflicting interpretations in MUTYH, and demonstrates a scalable strategy to interrogate other clinically relevant DNA repair factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/123ddb79d358a7742980d1fc41247ef4a3c39e8c" target='_blank'>
              Saturation mapping of MUTYH variant effects using DNA repair reporters
              </a>
            </td>
          <td>
            Shelby L. Hemker, Ashley Marsh, Felicia Hernandez, Elena Glick, Grace Clark, Alyssa Bashir, Krystal Jiang, Jacob O. Kitzman
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18bbde28601716c5dd5df4d5397a630f94586b32" target='_blank'>
              Loss of Fanconi anemia proteins causes a reliance on lysosomal exocytosis
              </a>
            </td>
          <td>
            Becky Xu Hua Fu, Albert Xu, Hua Li, Daniel E Johnson, Jennifer R. Grandis, Luke A. Gilbert
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="The landscapes of somatic mutation in normal cells inform on the processes of mutation and selection operative throughout life, permitting insight into normal ageing and the earliest stages of cancer development. Here, by whole-genome sequencing of 238 microdissections from 30 individuals, including 18 with gastric cancer, we elucidate the developmental trajectories of normal and malignant gastric epithelium. We find that gastric glands are units of monoclonal cell populations which accrue ∼28 somatic single nucleotide variants per year, predominantly attributable to endogenous mutational processes. In individuals with gastric cancer, glands often show elevated mutation burdens due to acceleration of mutational processes linked to proliferation and oxidative damage. These hypermutant glands were primarily detected in the gastric antrum and were mostly associated with chronic inflammation and intestinal metaplasia, known cancer risk factors. Unusually for normal cells, gastric epithelial cells often carry recurrent trisomies of specific chromosomes, which are highly enriched in a subset of individuals. Surveying approximately 8,000 gastric glands by targeted sequencing, we found somatic driver mutations in a distinctive repertoire of known cancer genes, including ARID1A, CTNNB1, KDM6A and ARID1B. Their prevalence increases with age to occupy approximately 5% of the gastric epithelial lining by age 60 years. Our findings provide insights into the intrinsic and extrinsic influences on somatic evolution in the gastric epithelium, in healthy, precancerous and malignant states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a80de4630550f795baa857c6ee76f35822aa4bc" target='_blank'>
              The somatic mutation landscape of normal gastric epithelium
              </a>
            </td>
          <td>
            Tim H. H. Coorens, G. Collord, Hyungchul Jung, Yichen Wang, L. Moore, Y. Hooks, Krishnaa Mahbubani, K. Saeb-Parsy, P. J. Campbell, I. Martincorena, Suet Yi Leung, Michael R Stratton
          </td>
          <td>2024-03-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>88</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024', '2025'],
    y: [0, 1, 19],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>