<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-27 09:26:04 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="The tumor microenvironment (TME) is a complex network of interactions between malignant and host cells, yet its orchestration in advanced high-grade serous ovarian carcinoma (HGSC) remains poorly understood. We present a comprehensive single-cell spatial atlas of 280 metastatic HGSCs, integrating high-dimensional imaging, genomics, and transcriptomics. Using 929 single-cell maps, we identify distinct spatial domains associated with phenotypically heterogeneous cellular compositions, and demonstrate that immune cell co-infiltration at the tumor-stroma interface significantly influences clinical outcomes. To uncover the key drivers of the tumor ecosystem, we developed CEFIIRA (Cell Feature Importance Identification by RAndom forest), which identified tumor cell-intrinsic MHC-II expression as a critical predictor of prolonged survival, independent of clinicomolecular profiles. Validation with external datasets confirmed that MHC-II-expressing cancer cells drive immune infiltration and orchestrate spatial tumor-immune interactions. Our atlas offers novel insights into immune surveillance mechanisms across HGSC clinicomolecular groups, paving the way for improved therapeutic strategies and patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/684d76f91d1a8b1bc3e208feca47d927bec6a2ec" target='_blank'>
              Single-cell spatial atlas of high-grade serous ovarian cancer unveils MHC class II as a key driver of spatial tumor ecosystems and clinical outcomes
              </a>
            </td>
          <td>
            Fernando Pérez-Villatoro, Lilian van Wagensveld, Aleksandra Shabanova, Ada Junquera, Ziqi Kang, Iga Niemiec, Matias M. Falco, Ella Anttila, J. Casado, Eric Marcus, Essi Kahelin, Foteini Chamchougia, M. Salko, Saundarya Shah, Salvatore Russo, Jacopo Chiaro, M. Grönholm, G. Sonke, K. K. van de Vijver, Rutgerus Fpm Kruitwagen, Maaike Avan der Aa, A. Virtanen, Vincenzo Cerullo, Anna Vähärautio, P. Sorger, Hugo M. Horlings, A. Färkkilä
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="In bladder cancer (BLCA), the cellular expression of lineage markers with predictive biomarker function can have important implications for tumor progression, treatment response, and survival. However, tumor heterogeneity and the coexistence of distinct tumor subpopulations can complicate the utility of using a primary, clinically assigned tumor signature. In this report, we have applied the reference atlas, Tabula Sapiens, and deep learning model, UniCell, to conduct in-depth transcriptional analysis of unique lineage marker-defined cell clusters in a carcinogen (BBN) induced bladder cancer model. UniCell deconvolution has identified tumor populations, including urothelial, adenocarcinoma, squamous cell carcinoma, and mesenchymal tumor populations, each with cell-intrinsic gene expression signatures relevant to human BLCA progression. The identified tumor clusters contain uniquely basal, luminal, stromal, or hybrid cells in cell lineage marker expression. To understand the significance of these populations during progression, we used trajectory and pseudo-time analysis to show that cells uniquely basal are plastic in lineage identity and can evolve to form tumor populations composed of other lineage marker-defined signatures. Finally, pathway and drug enrichment analysis of tumor cell clusters were used to identify therapeutics that may preferentially target the identified tumor cell populations. These data collectively define a molecular template that may uniquely profile molecular plasticity occurring during progression and response to therapy with important implications for human disease. Significance Tumor heterogeneity is a mechanism for treatment resistance. Our study defines unique tumor subpopulations having differential therapeutic sensitivities and potential for lineage plasticity. Our modeling may impact the treatment of BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e51b8c18f48b163ace1388c13e342c3f5124f692" target='_blank'>
              Mapping of Multilineage Tumor Cell Populations in Mouse Bladder Cancer
              </a>
            </td>
          <td>
            Nadine Schrode, Yang Hu, Haocheng Yu, M. Galsky, J. Sfakianos, R. Sebra, Kristin G Beaumont, David J Mulholland
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) is evolving rapidly as a pivotal technology in studying the biology of tumors and their associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. In this study, we used serial 5-m sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma tumor samples in tissue microarrays to compare the performance of the single cell ST platforms CosMx, MERFISH, and Xenium (uni/multi-modal) platforms in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx Digital Spatial Profiler, and hematoxylin and eosin staining data for the same samples. In addition to objective assessment of automatic cell segmentation and phenotyping, we performed pixel-resolution manual evaluation of phenotyping to carry out pathologically meaningful comparison between ST platforms. Our study detailed the intricate differences between the ST platforms, revealed the importance of parameters such as tissue age and probe design in determining the data quality, and suggested reliable workflows for accurate spatial profiling and molecular discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17805d448b42a38f2a1b50b0eb2fd17aaa264ce4" target='_blank'>
              Comparison of imaging-based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed, paraffin-embedded tumor samples
              </a>
            </td>
          <td>
            Nejla Ozirmak Lermi, Max Molina Ayala, Sharia D Hernandez, Wei Lu, Khaja B. Khan, Alejandra Serrano, Idania Lubo, Leticia Hamana, K. Tomczak, Sean Barnes, Jinzhuang Dou, Qingnan Liang, Rti Team, M. Gabriela Raso, Ximing Tang, Mei Jiang, B. Sanchez-Espiridion, A. Weissferdt, J. Heymach, Jianjun Zhang, B. Sepesi, T. Cascone, Anne Tsao, Mehmet Altan, Reza J Mehran, Don L Gibbons, Ignacio I. Wistuba, C. Haymaker, Ken Chen, L. S. Solis Soto
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), a distinct subpopulation within tumors, are pivotal in driving treatment resistance and tumor recurrence, posing substantial challenges to conventional therapeutic strategies. Precise quantification and profiling of these cells are essential for improving cancer treatment outcomes. We present ACSCeND, an advanced deep neural network model accompanied by a robust workflow, specifically developed to quantify cellular compositions from bulk RNA-seq data, enabling accurate CSC profiling. By integrating bulk RNA-seq data with insights derived from single-cell RNA-seq datasets, ACSCeND effectively captures the diversity and hierarchical organization of tumor-resident cell states, alongside cell-specific gene expression profiles (GEPs). Compared to current tissue deconvolution models, ACSCeND exhibits superior performance, achieving significantly higher Concordance Correlation Coefficient (CCC) values and lower Root Mean Square Error (RMSE) across various pseudobulk and real-world bulk tissue samples. Application of ACSCeND to TCGA and PRECOG datasets reveals a strong association between CSC abundance and poorer disease-free survival outcomes, underscoring the clinical relevance of CSCs in cancer progression. Furthermore, cell-specific GEPs for distinct CSC states unveil novel molecular signatures and illuminate the origins of CSC-driven tumor heterogeneity. In summary, ACSCeND provides a powerful, scalable platform for high-throughput quantification of cellular compositions and distinct potency states within normal tissues as well as highly heterogeneous tissues, such as tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b3b0a8bb89725a847aae6e2af230a0666ba9d6" target='_blank'>
              Comprehensive Enumeration of Cancer Stem-like Cell Heterogeneity Using Deep Neural Network
              </a>
            </td>
          <td>
            Debojyoti Chowdhury, Shreyansh Priyadarshi, Sayan Biswas, Bhavesh Neekhra, Debayan Gupta, Shubhasis Haldar
          </td>
          <td>2024-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to random barcode sets. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fe68cd8b32acb61197d1f1f0a4cc878efa91c6" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) carries an extremely poor prognosis, in part resulting from cellular heterogeneity that supports overall tumorigenicity. Cancer associated fibroblasts (CAF) are key determinants of PDAC biology and response to systemic therapy. While CAF subtypes have been defined, the effects of patient-specific CAF heterogeneity and plasticity on tumor cell behavior remain unclear. Here, multi-omics was used to characterize the tumor microenvironment (TME) in tumors from patients undergoing curative-intent surgery for PDAC. In these same patients, matched tumor organoid and CAF lines were established to functionally validate the impact of CAFs on the tumor cells. CAFs were found to drive epithelial-mesenchymal transition (EMT) and a switch in tumor cell classificiaton from classical to basal subtype. Furthermore, we identified CAF-specific interleukin 8 (IL-8) as an important modulator of tumor cell subtype. Finally, we defined neighborhood relationships between tumor cell and T cell subsets. Statement of Significance This multidimensional analysis highlights the diverse role CAFs have in influencing other cell types in the TME, including epithelial-derived tumor and infiltrating immune cells. Our methods provide a platform for evaluating emerging therapeutic approaches and for studying mechanisms that dictate tumor behavior in a manner that reflects patient-specific heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d173d5090e9fb969bbd5c7fddf327d5b9643a1d" target='_blank'>
              Cancer associated fibroblasts drive epithelial to mesenchymal transition and classical to basal change in pancreatic ductal adenocarcinoma cells with loss of IL-8 expression
              </a>
            </td>
          <td>
            Samantha Guinn, Brayan Perez, Joseph A Tandurella, Mili Ramani, Jae W. Lee, Daniel J. Zabransky, Emma Kartalia, Jignasha Patel, H. Zlomke, Norman G Nicolson, Sarah M. Shin, Benjamin Barrett, Nicholas Sun, Alexei Hernandez, Erin M Coyne, Courtney D Cannon, Nicole E. Gross, S. Charmsaz, Yeonju Cho, James Leatherman, Melissa R Lyman, Jacob T. Mitchell, Luciane T. Kagohara, Michael G. Goggins, Kelly J. Lafaro, Jin He, Christopher R Shubert, William R Burns, Lei Zheng, E. Fertig, Elizabeth M. Jaffee, Richard A Burkhart, W. Ho, Jacquelyn W. Zimmerman
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Dedifferentiation programs are commonly enacted during breast cancer progression to enable novel cellular phenotypes. Increased cellular plasticity within the neoplastic compartment of tumors correlates with disease aggressiveness, often culminating in greater resistance to cytotoxic therapies or the augmented ability to metastasize to distant organs. Here we report that subpopulations of dedifferentiated neoplastic breast epithelial cells express canonical leukocyte cell surface receptor proteins and have thus named this unique cellular program “immune mimicry.” We document neoplastic cells engaging in immune mimicry within public human breast tumor single-cell RNA-seq datasets, histopathological breast tumor specimens, breast cancer cell lines, as well as in murine transgenic and cell line-derived mammary cancer models. Immune-mimicked neoplastic cells harbor hallmarks of dedifferentiation and appear enriched in treatment-resistant and high-grade breast tumors. We corroborated these observations in aggressive breast cancer cell lines where growth-arresting cytotoxic chemotherapies drove epithelial cells toward immune mimicry. Moreover, in subsequent proof-of-concept studies, we demonstrate that expression of the CD69 leukocyte activation marker by neoplastic cells confers a proliferative advantage that enhances early tumor growth. We conclude neoplastic breast epithelial cells that upregulate leukocyte surface receptors potentiate malignancy. Moving forward, neoplastic immune mimicry should be evaluated for prognostic utility in breast cancer to determine its stratification potential for patients with increased risks of tumor recurrence, metastasis, and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79211f06fe387981092f12a48bc76b415afdfd2" target='_blank'>
              Neoplastic immune mimicry is a generalizable phenomenon in breast cancer and epithelial CD69 enables early tumor progression
              </a>
            </td>
          <td>
            Eric B. Berens, Sokchea Khou, Elaine Huang, Amber Hoffman, Briana Johnson, Nell Kirchberger, Sam Sivagnanam, Nicholas L. Calistri, Daniel Derrick, T. Liby, Ian C. Mclean, Aryn A. Alanizi, F. Ozmen, T. Y. Ozmen, Gordon B. Mills, E. S. Hwang, P. Schedin, Hugo Gonzalez, Z. Werb, Laura M. Heiser, L. Coussens
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="The impact of cancer driving mutations in regulating immunosurveillance throughout tumor development remains poorly understood. To better understand the contribution of tumor genotype to immunosurveillance, we generated and validated lentiviral vectors that create an epi-allelic series of increasingly immunogenic neoantigens. This vector system is compatible with autochthonous Cre-regulated cancer models, CRISPR/Cas9-mediated somatic genome editing, and tumor barcoding. Here, we show that in the context of KRAS-driven lung cancer and strong neoantigen expression, tumor suppressor genotype dictates the degree of immune cell recruitment, positive selection of tumors with neoantigen silencing, and tumor outgrowth. By quantifying the impact of 11 commonly inactivated tumor suppressor genes on tumor growth across neoantigenic contexts, we show that the growth promoting effects of tumor suppressor gene inactivation correlate with increasing sensitivity to immunosurveillance. Importantly, specific genotypes dramatically increase or decrease sensitivity to immunosurveillance independently of their growth promoting effects. We propose a model of immunoediting in which tumor suppressor gene inactivation works in tandem with neoantigen expression to shape tumor immunosurveillance and immunoediting such that the same neoantigens uniquely modulate tumor immunoediting depending on the genetic context. One Sentence Summary Here we uncover an under-appreciated role for tumor suppressor gene inactivation in shaping immunoediting upon neoantigen expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a2622766d9e8241bb1972eff5c441baddd14069" target='_blank'>
              Tumor suppressor genotype influences the extent and mode of immunosurveillance in lung cancer
              </a>
            </td>
          <td>
            Keren M. Adler, Haiqing Xu, Amy C. Gladstein, Valerie M. Irizarry-Negron, Maggie R. Robertson, Katherine R. Doerig, D. Petrov, M. Winslow, D. Feldser
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The organizational principles of nephronal segments are based on longstanding anatomical and physiological attributes that are closely linked to the homeostatic functions of the kidney. Novel molecular approaches have recently uncovered layers of deeper signatures and states in tubular cells that arise at various timepoints on the spectrum between health and disease. For example, a dedifferentiated state of proximal tubular cells with mesenchymal stemness markers is frequently seen after injury. The persistence of such a state is associated with failed repair. Here, we introduce a novel analytical pipeline applied to highly multiplexed spatial protein imaging to characterize proximal tubular subpopulations and neighborhoods in reference and disease human kidney tissue. The results were validated and extended through integration with spatial and single cell transcriptomics. We demonstrate that, in reference tissue, a large proportion of S1 and S2 proximal tubular epithelial cells express THY1, a mesenchymal stromal and stem cell marker that regulates differentiation. Kidney disease is associated with loss of THY1 and transition towards expression of PROM1, another stem cell marker shown recently to be linked to failed repair. We demonstrate that the trajectory of proximal tubular cells to THY1 expression is clearly distinct from that of PROM1, and that a state with PROM1 expression is associated with niches of inflammation. Our data support a model in which the interplay between THY1 and PROM1 expression in proximal tubules associates with their regenerative potential and marks the timeline of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e12887f03204d9141acb86ce8374e99f9dcbad" target='_blank'>
              Integration of spatial multiplexed protein imaging and transcriptomics in the human kidney tracks the regenerative potential timeline of proximal tubules
              </a>
            </td>
          <td>
            Mahla Asghari, Angela R. Sabo, Daria Barwinska, R. M. Ferreira, M. Ferkowicz, William S. Bowen, Ying-Hua Cheng, D. Gisch, Connor J. Gulbronson, Carrie L. Phillips, Katherine J. Kelly, T. Sutton, James C Williams, Miguel Vazquez, John O’Toole, Paul M. Palevsky, Sylvia E. Rosas, S. Waikar, Krzysztof Kiryluk, C. Parikh, Jeff Hodgins, P. Sarder, Ian H. de Boer, Jonathan Himmelfarb, Matthias Kretzler, Sanjay Jain, Michael T. Eadon, S. Winfree, Tarek M. El-Achkar, P. Dagher
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="High-risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single-cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this gap, we profiled untreated and chemotherapy-treated high-risk neuroblastomas from 2 patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p-gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells and we experimentally demonstrated that CCL18 increases neuroblastoma cell migration. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical-like cells expressed the ALK ligand ALKAL2 and were predicted to communicate with neighboring ALK expressing cancer cells. We demonstrated a unique developmental pattern of adrenal medulla-specific expression of ALK and adrenocortical-specific expression of ALKAL2, suggesting a role of these signaling interactions in neuroblastoma carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cacb0dcc803b6c58db82e59f6d5ec533dad1808" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment unveils novel paracrine interactions
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlson, A. Claeys, Eva Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Thyroid cancer progression from curable well-differentiated thyroid carcinoma to highly lethal anaplastic thyroid carcinoma is distinguished by tumor cell de-differentiation and recruitment of a robust stromal infiltrate. Combining an integrated thyroid cancer single-cell sequencing atlas with spatial transcriptomics and bulk RNA-sequencing, we define stromal cell subpopulations and tumor-stromal cross-talk occurring across the histologic and mutational spectrum of thyroid cancer. We identify distinct inflammatory and myofibroblastic cancer-associated fibroblast (iCAF and myCAF) populations and perivascular-like populations. The myCAF population is only found in malignant samples and is associated with tumor cell invasion, BRAFV600E mutation, lymph node metastasis, and disease progression. Tumor-adjacent myCAFs abut invasive tumor cells with a partial epithelial-to-mesenchymal phenotype. Tumor-distant iCAFs infiltrate inflammatory autoimmune thyroid lesions and anaplastic tumors. In summary, our study provides an integrated atlas of thyroid cancer fibroblast subtypes and spatial characterization at sites of tumor invasion and de-differentiation, defining the stromal reorganization central to disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b786943f0e4090faa02f89896c2ed59c657e0d0" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations
              </a>
            </td>
          <td>
            Matthew A. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey Caroland, Megan L. Tigue, H. Hartmann, J. Gallant, Courtney J. Phifer, Andres Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H Belcher, Ken S. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Understanding cell-cell communication and its dependence on spatial organization is critical for unraveling tissue complexity and organ function. This study integrates single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics (ST) to systematically assess how spatial niches influence gene expression and intercellular communication. Using breast cancer, brain cortex, and heart datasets, our analyses reveal limited global transcriptional changes in cells depending on their spatial microenvironment, with differential gene expression observed in less than half the samples explored. Moreover, cell-cell communication predictions, derived from ligand-receptor pairs, exhibit minimal correlation with spatial colocalization of cell types. Overall, our study underscores the limitations of using scRNA-seq data to capture niche-specific molecular interactions, even when spatial information is leveraged, and it highlights the need for novel strategies to refine our understanding of intercellular communication dynamics at molecular level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790c5632749fda7ad54ef1434d0ac14122b25196" target='_blank'>
              Systematic assessment of microenvironment-dependent transcriptional patterns and intercellular communication
              </a>
            </td>
          <td>
            Elena Pareja-Lorente, Patrick Aloy
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The intra- and inter-tumoral heterogeneity of glioblastoma represents a significant therapeutic challenge, as well as difficulty in generating reliable models for in vitro studies. Historical 2D adherent cell lines do not recapitulate this complexity, whereas both patient-derived neurospheres (PDN) and organoids (PDO) demonstrate intra-tumoral heterogeneity. Here, we quantify the tumor cell composition from matched models established from the same primary tumor using a series of multi-omic interrogations. We find that both patient-derived models recapitulate the genomic, epigenomic and tumor cell heterogeneity of the primary tissue. Furthermore, single-nuclei RNA sequencing revealed a subset of organoids containing small numbers of non-malignant cells from neuron and immune cell lineages. Harnessing the intra-tumoral heterogeneity of PDN models, we reveal the impact of temozolomide chemotherapy on individual cell states, altering composition of tumors over time in response to treatment. Our data confirms that both patient-derived models recapitulate patient intra-tumoral heterogeneity providing a platform for tumor cell state refined therapeutic studies. Key Points Generation of matched patient-derived neurosphere and organoid models from resected GBM tissue Neurosphere models exhibit greater proliferative signatures Both patient-derived models recapitulate genomic and epigenomic features of the primary tissue Single-nuclei RNA sequencing reveals both models recapitulate intra-tumoral heterogeneity of the primary tissue Neurosphere models enable interrogation of therapeutic responses in the context of heterogeneity Importance of study Patient derived models can be powerful tools when they faithfully recapitulate the tumor tissue from which they are derived. In glioblastoma, patient derived neurospheres (PDN) and organoids (PDO) have both been used in studies, however the differences between these models and the recapitulation of patient heterogeneity remain to be fully characterized. To address this, we performed multi-omic profiling of PDN and PDO models generated from the same tumor tissue. We find that that across a range of data modalities, both model systems exhibit a high level of resemblance to tissue, and critically, maintain heterogeneity and tumor cell composition. The importance of modeling heterogeneity was demonstrated in PDN models, where temozolomide treatment specifically alters the abundance of MES-like and AC-like tumor cells. Our findings demonstrate that neurosphere and organoid models effectively preserve cellular heterogeneity, genomic alterations, methylation signatures and transcriptomic features, both highly suitable to model glioblastoma’s complex cellular landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c3caa2d109062b5ff4f7f9f68c99c0d6be111c" target='_blank'>
              Three-dimensional patient-derived models of glioblastoma retain intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            Zachery Moore, Claire Storey, Daniel V. Brown, A. Valkovic, Montana Spiteri, Jasmine F. Pignatelli, Shannon J Oliver, Alana Fakhri, Kate Drummond, Seth Malinowski, Keith L. Ligon, Oliver M. Sieber, J. Whittle, S. Freytag, Sarah A. Best
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Inadequate response to androgen deprivation therapy (ADT) frequently arises in prostate cancer, driven by cellular mechanisms that remain poorly understood. Here, we integrated single-cell RNA sequencing, single-cell multiomics, and spatial transcriptomics to define the transcriptional, epigenetic, and spatial basis of cell identity and castration response in the mouse prostate. Leveraging these data along with a meta-analysis of human prostates and prostate cancer, we identified cellular orthologs and key determinants of ADT response and resistance. Our findings reveal that mouse prostates harbor lobe-specific luminal epithelial cell types distinguished by unique gene regulatory modules and anatomically defined androgen-responsive transcriptional programs, indicative of divergent developmental origins. Androgen-insensitive, stem-like epithelial populations - resembling human club and hillock cells - are notably enriched in the urethra and ventral prostate but are rare in other lobes. Within the ventral prostate, we also uncovered two additional androgen-responsive luminal epithelial cell types, marked by Pbsn or Spink1 expression, which align with human luminal subsets and may define the origin of distinct prostate cancer subtypes. Castration profoundly reshaped luminal epithelial transcriptomes, with castration-resistant luminal epithelial cells activating stress-responsive and stemness programs. These transcriptional signatures are enriched in tumor cells from ADT-treated and castration-resistant prostate cancer patients, underscoring their likely role in driving treatment resistance. Collectively, our comprehensive cellular atlas of the mouse prostate illuminates the importance of lobe-specific contexts for prostate cancer modeling and reveals potential therapeutic targets to counter castration resistance. Significance Statement Androgen deprivation therapy is a mainstay in prostate cancer treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer new directions for prostate cancer modeling and pave the way toward novel therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47e36c2b5c9a5688d815f8fbf4d281fd5d7c15cd" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>167</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23b3d699dd705e1b450a74d7c903ab854deff777" target='_blank'>
              Increased spatial coupling of integrin and collagen IV in the immunoresistant clear-cell renal-cell carcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, Taylor Peak, Oscar E. Ospina, N. Chakiryan, Anders E. Berglund, Paul A Stewart, Jonathan V Nguyen, C. M. Segura, Natasha L. Francis, P. R. Echevarria, J. Chahoud, Roger Li, Kenneth Y. Tsai, Jodi A. Balasi, Yamila Caraballo Peres, J. Dhillon, Lindsey A. Martinez, Warren E. Gloria, Nathan Schurman, Seang-Jae Kim, M. Gregory, James J Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-12-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="Uncontrolled proliferation is a hallmark of cancer, yet tumour cells can enter G0 arrest by halting the cell cycle reversibly (quiescence) or irreversibly (senescence) to survive under stress and in hostile tumour microenvironments (TME). G0 arrested cells contribute to drug tolerance, metastasis and recurrence, but their identification remains challenging due to their rarity and elusive regulatory pathways. Here, we quantify G0 arrest and proliferation decisions in single-cell and spatially profiled breast primary tumours to unveil adaptive responses driving immune compartmentalisation. We identify a genomically-constrained and prolonged G0 arrest state resembling that of dormant precursors of cancer progression. This state featured adaptive transcriptional reprogramming, including unfolded protein response stress, reduced mitochondrial translation and epithelial-mesenchymal plasticity enabled by semaphorin signalling. Spatial transcriptomics revealed G0 arrest pockets encapsulated by APOE+ lipid-associated macrophages, myofibroblastic CAFs and immature perivascular cells, suggestive of an immunosuppressive niche contrasting with cytotoxic environments in proliferative hotspots and displaying distinct drug sensitivities. To facilitate future research, we provide a foundation model capturing G0 arrest and proliferation with 94% accuracy in single cell data, available at https://github.com/secrierlab/G0-LM. Our findings provide new insights into the spatial organisation of cell cycle arrest in breast cancer, highlighting the role of G0 states in tumour heterogeneity and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5d540e388314aece2599568858b5b0ad8004482" target='_blank'>
              Balancing tumour proliferation and sustained cell cycle arrest through proteostasis remodelling drives immune niche compartmentalisation in breast cancer
              </a>
            </td>
          <td>
            Cenk Celik, Eloise Withnell, Shi Pan, Tooki Chu, Johnathan Labbadia, M. Secrier
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioma is an aggressive brain tumor with a poor prognosis. Establishing an in vitro culture model that closely replicates the cellular composition and microenvironment of the original tumor has been challenging, limiting its clinical applications. Here, we present a novel approach to generate glioma organoids with a microenvironment (GlioME) from patient-derived glioma tissue. These organoids maintain the genetic and epigenetic characteristics of the primary tumor and preserve cell-to-cell interactions within the tumor microenvironment, including resident immune cells. Bulk RNA sequencing, whole exome sequencing, and DNA methylation analysis were used to confirm the molecular similarities between the organoids and primary glioma tissues. Immunofluorescence and flow cytometry were used to assess immune cell viability, comparing GlioME with floating glioma organoids. GlioME exhibited high responsiveness to chemotherapy and targeted therapy, demonstrating its potential for therapeutic screening applications. Notably, GlioME accurately predicted patient response to the recently approved MET inhibitor, vebreltinib. Thus, this organoid model provides a reliable in vitro platform for glioma microenvironment-related research and clinical drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076fe0fc7a2a2145464370151771e2e7246f87bc" target='_blank'>
              GlioME: A Novel Organoid Model Retaining the Glioma Microenvironment for Personalized Drug Screening and Therapeutic Evaluation
              </a>
            </td>
          <td>
            Chengjun Zheng, Qiaodong Chen, Peng Wang, Delong Zhang, Zheng Fang, Yutong Feng, Jie Chen, Jiahong Chen, Yiwen Fu, Bao Yang, Qian Zhao, Fei Sun, Ying Zhang, Tao Jiang, Lemin Zheng, Zhaoshi Bao
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell omics analysis has unveiled the heterogeneity of various cell types within tumors. However, no methodology currently reveals how this heterogeneity influences cancer patient survival at single-cell resolution. Here, we introduce scSurv, combining a Cox proportional hazards model with a deep generative model of single-cell transcriptome, to estimate individual cellular contributions to clinical outcomes. The accuracy of scSurv was validated using both simulated and real datasets. This method identifies cells associated with favorable or adverse prognoses and extracts genes correlated with their contribution levels. In melanoma, scSurv reproduces known prognostic macrophage classifications and facilitates hazard mapping through spatial transcriptomics in renal cell carcinoma. We also identified genes consistently associated with prognosis across multiple cancers and demonstrated the applicability of this method to infectious diseases. scSurv is a novel framework for quantifying the heterogeneity of individual cellular effects on clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5139bf1027dffb0022f2f770e24762808926adf5" target='_blank'>
              scSurv: a deep generative model for single-cell survival analysis
              </a>
            </td>
          <td>
            Chikara Mizukoshi, Yasuhiro Kojima, Shuto Hayashi, Ko Abe, Daisuke Kasugai, Teppei Shimamura
          </td>
          <td>2024-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Dysregulation of intracellular signaling networks underpins cancer. However, a systems-level elucidation of how signaling networks within distinct cell subpopulations drive cancer progression in vivo has been unattainable due to technical limitations. We developed INSIGHT (INvestigating SIGnaling network of specific cell subpopulation in Heterogeneous Tissue), a new platform technology combining fluorescence-activated cell sorting with ultra-sensitive mass spectrometry to enable phosphoproteomic characterization of rare and discrete cell subpopulations from fixed tissues. We demonstrated the broad utility of INSIGHT by analyzing the oligodendroglial cell-specific signaling network in the mouse brain. We then applied INSIGHT to investigate the rare, disseminated tumor cell subpopulation in glioblastoma patient-derived xenograft models. INSIGHT uncovered a global rewiring of signaling networks with tumor cell dissemination, marked by a transition from proliferation-associated signaling in the primary tumor cells to signaling associated with postsynapse, neuronal migration, and ion homeostasis in disseminated tumor cells. We reveal interconnections between signaling circuitries within the networks, with numerous proteins, including GluA2, exhibiting altered phosphorylation without protein expression changes, emphasizing the role of post-translational modifications in glioblastoma dissemination. We validated key phosphorylation changes and inferred differentially active kinases with tumor spread to offer new systems-level insights into glioblastoma dissemination mechanisms in vivo. INSIGHT is generally applicable to a wide range of biological systems without genetic engineering and provides quantitative phosphorylation and protein expression data for selected cell subpopulations from heterogeneous tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/decaf507fd537acca35393bd50f3bedca402f1f6" target='_blank'>
              Uncovering the signaling networks of disseminated glioblastoma cells in vivo with INSIGHT
              </a>
            </td>
          <td>
            Ryuhjin Ahn, Alicia D. D’Souza, Lawrence Long, Yufei Cui, D. Burgenske, K. Bakken, Lauren L Ott, Brett L. Carlson, Grace Zhou, Tomer M. Yaron-Barir, Ishwar N. Kohale, Charles A. Whittaker, Cameron T. Flower, Jeffrey Wyckoff, A. Tuma, Jared L. Johnson, Jann Sarkaria, Forest M. White
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="DICER1 syndrome predisposes children and young adults to tumor development across various organs. Many of these cancers are sarcomas, which uniquely express the RNase IIIb domain-deficient form of DICER1 and exhibit consistent histological and molecular similarities regardless of their anatomical origins. To uncover their cellular origin and developmental hierarchy, we established a lineage-traceable genetically engineered mouse model that allows for precise activation of Dicer1 mutations in Hic1+ mesenchymal stromal cells. This model resulted in the development of renal tumors closely mirroring human DICER1 sarcoma histologically and molecularly. Single-cell transcriptomics coupled with targeted spatial gene expression analysis revealed a Hic1+ progenitor population marked by Pdgfra, Dpt, and Mfap4, corresponding to universal fibroblasts of steady-state kidneys. These fibroblastic progenitors exhibit the capacity to undergo rhabdomyoblastic differentiation or transition to highly proliferative anaplastic sarcoma. Investigation of patient samples identified analogous cell states. This study uncovers a fibroblastic origin for DICER1 sarcoma and provides a faithful model for mechanistic investigation and therapeutic development for tumors within the rhabdomyosarcoma spectrum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7612b5f96fea7c628834a8ae0728263c8ee1372" target='_blank'>
              Spatial single cell transcriptomic analysis of a novel DICER1 Syndrome GEMM informs the cellular origin and developmental hierarchy of associated sarcomas
              </a>
            </td>
          <td>
            F. Kommoss, Joyce Zhang, Branden Lynch, S. Chen, J. Senz, Yana Moscovitz, Lesley A. Hill, Wilder Scott, Jonathan Bush, Kenneth S. Chen, Andreas von Deimling, William D. Foulkes, Gregg B. Morin, T. M. Underhill, Yemin Wang, D. Huntsman
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Despite significant advancements in single-cell sequencing analysis for characterizing tissue sample heterogeneity, identifying the associations between cell subpopulations and disease phenotypes remains a challenging task. Here, we introduce scPAS, a new bioinformatics tool designed to integrate bulk data to identify phenotype-associated cell subpopulations within single-cell data. scPAS employs a network-regularized sparse regression model to quantify the association between each cell in single-cell data and a phenotype. Additionally, it estimates the significance of these associations through a permutation test, thereby identifying phenotype-associated cell subpopulations. Utilizing simulated data and various single-cell datasets from breast carcinoma, ovarian cancer, and atherosclerosis, as well as spatial transcriptomics data from multiple cancers, we demonstrated the accuracy, flexibility, and broad applicability of scPAS. Evaluations on large datasets revealed that scPAS exhibits superior operational efficiency compared to other methods. The open-source scPAS R package is available at GitHub website: https://github.com/aiminXie/scPAS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d51db6c241bc052ded3595c6cf2f201311e079" target='_blank'>
              scPAS: single-cell phenotype-associated subpopulation identifier
              </a>
            </td>
          <td>
            Aimin Xie, Hao Wang, Jiaxu Zhao, Zhaoyang Wang, Jinyuan Xu, Yan Xu
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Upper aerodigestive squamous cell carcinoma (UASCC) is an aggressive and lethal neoplasm, with its early neoplastic transformation mechanisms remaining poorly understood. Here, we characterize over 25 genetically-defined organoid models derived from murine and human oral/esophageal tissues harboring key driver mutations. Double knockout of TP53 and CDKN2A induced morphological dysplasia, hyperproliferation, loss of squamous differentiation, and tumorigenicity, which were further exacerbated by additional driver mutations (e.g., PIK3CA, NOTCH1, KMT2C). Single-cell analysis revealed an expansion of quiescent basal cells and proliferative squamous cells, alongside a loss of differentiated squamous cells during malignant transformation. A distinct senescence program, regulated by ANXA1, was markedly diminished during early neoplastic evolution. Mechanistically, the ANXA1-SMAD3-p27KIP1 pathway was identified as a critical regulator of this senescence program, acting to suppress neoplastic features in organoid models. Lastly, our high-throughput, single-organoid-resolution drug screens unexpectedly revealed PIK3CA-driven organoids exhibited sensitivity to Mitomycin C and Onalespib. This study provides novel mechanistic insights into early neoplastic evolution and underscores the value of genetically-defined organoid models for investigating cancer biology and identifying targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7127b7b7361c88e16950c954ef2bf66d966c5299" target='_blank'>
              Genetically Defined Organoid Models Reveal Mechanisms Driving Squamous Cell Neoplastic Evolution and Identify Potential Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Hua Zhao, Young Min Park, Yueyuan Zheng, Qiong Mao, Casey Collet, Boyan Hu, Tianming Zhou, Luda Lin, Stephanie Wong, Yuhao Pan, Anette Vistoro Monreal, Uttam Sinha, Parish Sedghizadeh, A. Soragni, De-Chen Lin
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90bace8a9791e95b3eff7c16fd892d053802f89b" target='_blank'>
              Single-cell transcriptomes of dissecting the intra-tumoral heterogeneity of breast cancer microenvironment
              </a>
            </td>
          <td>
            Peixian Chen, Kaifeng Liang, Xiaofan Mao, Qiuyuan Wu, Zhiyan Chen, Yabin Jin, Kairong Lin, Tiancheng He, Shuqing Yang, Huiqi Huang, Guolin Ye, Juntao Gao, Dan Zhou, Zhihao Zeng
          </td>
          <td>2024-12-26</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Dysregulation of enhancer-promoter communication in the context of the three-dimensional (3D) nucleus is increasingly recognized as a potential driver of oncogenic programs. Here, we profiled the 3D enhancer-promoter networks of primary patient-derived glioblastoma stem cells (GSCs) in comparison with neuronal stem cells (NSCs) to identify potential central nodes and vulnerabilities in the regulatory logic of this devastating cancer. Specifically, we focused on hyperconnected 3D regulatory hubs and demonstrated that hub-interacting genes exhibit high and coordinated expression at the single-cell level and strong association with oncogenic programs that distinguish IDH-wt glioblastoma patients from low-grade glioma. Epigenetic silencing of a recurrent 3D enhancer hub—with an uncharacterized role in glioblastoma—was sufficient to cause concordant downregulation of multiple hub-connected genes along with significant shifts in transcriptional states and reduced clonogenicity. By integrating published datasets from other cancer types, we also identified both universal and cancer type-specific 3D regulatory hubs which enrich for varying oncogenic programs and nominate specific factors associated with worse outcomes. Genetic alterations, such as focal duplications, could explain only a small fraction of the detected hyperconnected hubs and their increased activity. Overall, our study provides computational and experimental support for the potential central role of 3D regulatory hubs in controlling oncogenic programs and properties. HIGHLIGHTS - 3D regulatory “hubs” in glioblastoma enrich for highly coregulated genes at a single-cell level and expand oncogenic regulatory networks. - Targeted perturbation of a highly recurrent 3D regulatory hub in GSCs results in altered transcriptional states and cellular properties. - 3D regulatory hubs across cancer types associate with tumor-specific and universal oncogenic programs and worse outcomes. - The majority of hyperconnected hubs do not overlap with structural variants, suggesting epigenetic mechanisms. eTOC Here we profile the 3D enhancer connectomes of primary patient-derived human glioblastoma stem cells (GSCs), identify hyperconnected 3D regulatory “hubs”, and examine the impact of 3D hub perturbation on the transcriptional program and oncogenic properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef46a4bf29419cbc8b345c6cf8f3bc9f4287626" target='_blank'>
              Three-dimensional regulatory hubs support oncogenic programs in glioblastoma
              </a>
            </td>
          <td>
            Sarah L. Breves, Dafne Campigli Di Giammartino, James Nicholson, S. Cirigliano, Syed Raza Mahmood, UkJin Lee, Alexander Martínez-Fundichely, Johannes Jungverdorben, R. Singhania, Sandy Rajkumar, Raphael Kirou, L. Studer, Ekta Khurana, A. Polyzos, Howard A. Fine, E. Apostolou
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Fate-restricted cells can acquire stem cell-like properties through dedifferentiation, enabling them to gain the plasticity required for differentiation into multiple lineages. Tumour plasticity is prominently observed in brain cancers, where transient cell state changes are linked to resistance to conventional therapies. In this study, we demonstrate that a sub-population of dedifferentiated tumour neural stem cells (NSCs) in Drosophila, induced by the knockdown of prospero (pros), can generate its own glial niche. Temporal patterning, known to influence oncogenic competence and tumour malignancy, plays a key role in this process. Specifically, we show that de novo gliogenesis occurs in the more differentiated Syncrip+ (Syp+) NSC population. Modulating Syp levels alters the size of the glial niche, subsequently affecting tumour size. Furthermore, the tumour-associated glial niche expands through cell division and fails to cease proliferation on time due to dysregulated ecdysone signalling, contributing to niche expansion. Our findings reveal that tumours arising via dedifferentiation establish their own supportive glial microenvironment, which sustains tumour growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d30097dbfea218ae747551091d256416f71d4db" target='_blank'>
              Tumour-derived gliogenesis sustains dedifferentiation-dependent tumour growth in the Drosophila CNS
              </a>
            </td>
          <td>
            Edel Alvarez-Ochoa, Qian Dong, Hannah Truong, Louise Y Cheng
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Human lung is a complex organ susceptible to various diseases. Single-cell transcriptomic studies provide rich data to targeting specific research questions. Here, we present uniLUNG, the largest lung transcriptomic cell atlas, comprising over 10 million cells across 20 disease states and healthy controls. We ensembled a universal hierarchical annotation framework and conducted a full benchmarking of data integration to define a standardized nomenclature and marker genes for lung cell types. Using uniLUNG, we identified Lym-monocyte and T-like B cells, new cell types in specific lung diseases, confirming their existence by comparing with external single-cell atlases. Additionally, we discovered the NSCLC-like SCLC subpopulation, a transitional malignant cell population associated with the transition from NSCLC to SCLC, which was validated and further characterized in spatial dimensions, revealing its complex role in tumour progression. Overall, uniLUNG represents a comprehensive range of human lung cell diversity, providing valuable data resources and a reliable foundation for lung single-cell research. HIGHLIGHTS The largest scRNA atlas for human lung covers 10 million cells from 20 lung states. A four-level universal cell annotation framework encompasses 120 lung cell types. Comprehensive benchmarking on 18 strategies guides data integration. Specific distribution of Lym-monocytes and T-like B cells in specific lung diseases. The NSCLC-like SCLC subpopulation in transitional events of malignant cells from NSCLC to SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707c7b05c0b869d7c0046d311738148202a1849b" target='_blank'>
              A universal single-cell transcriptomics atlas of human lung decodes multiple pulmonary diseases
              </a>
            </td>
          <td>
            Fanjie Wu, Wenhao Cai, Hai Tang, Shikang Zheng, Haiyue Zhang, Yixin Chen, Yutong Han, Dingli Zhou, Ruihan Wang, Mingli Ye, Renke You, Amin Chen, Jiaqi Li, Xuegong Zhang, Weizhong Li
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The ability of cancer cells to alter their identity, known as lineage plasticity, is crucial for tumor progression and therapy resistance. In lung adenocarcinoma (LUAD), tumor progression is characterized by a gradual loss of lineage fidelity and the emergence of non-pulmonary identity programs. This can lead to hybrid-identity (hybrid-ID) states in which developmentally incompatible identity programs are co-activated within individual cells. However, the molecular mechanisms underlying these identity shifts remain incompletely understood. Here, we identify the gastrointestinal (GI) transcriptional regulator HNF4α as a critical driver of tumor growth and proliferation in KRAS-driven LUAD. In LUAD cells that express the lung lineage specifier NKX2-1, HNF4α can induce a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also forms an aberrant protein complex with NKX2-1, which disrupts NKX2-1 localization and dampens pulmonary identity within hybrid-ID LUAD. Sustained signaling through the RAS/MEK pathway is critical for maintaining the hybrid-ID state. Moreover, RAS/MEK inhibition augments NKX2-1 chromatin binding at pulmonary-specific genes and induces resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion enhances sensitivity to pharmacologic KRASG12D inhibition. Collectively, our data show that co-expression of opposing lineage specifiers leads to a hybrid identity state that can drive tumor progression and dictate response to targeted therapy in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207f2a257f041c211d79bb19f39ca01a49097c42" target='_blank'>
              Opposing lineage specifiers induce a pro-tumor hybrid-identity state in lung adenocarcinoma
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Rushmeen Tariq, Anna Waters, Kayla T. O'Toole, Eric L. Snyder
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Brain metastases (BrMets), common for advanced-stage breast cancer patients, are associated with poor median survival and accompanied by severe neurologic decline. Halting the progression of breast cancer brain metastases (BCBMs) may require modulation of the tumor microenvironment (TME), yet little is known about the impact of the primary breast TME on brain tropism, or how, once there, metastatic breast cancer cells coexist with brain-resident cells (e.g., neurons and glia). Traditionally, studies in this space have focused on differential expression analysis, overlooking potential insights gained from investigating genes with equivalent expression between groups. This is particularly crucial in distant metastasis, where tumor cells may co-opt the transcriptional programs of the host organ (e.g., brain) to facilitate successful seeding and outgrowth. Prior to our work, no computational framework existed to determine biologically-relevant equivalent gene expression. To resolve molecular mechanisms of BCBM enabled by metastatic cancer cells and/or resident brain cells, we leveraged Nanostring GeoMx to perform spatially-resolved transcriptomic profiling on 235 patient-derived tissue cores from BCBM (including adjacent normal brain), primary invasive breast cancers, and normal (non-cancer) brain; analyzing 18,677 RNAs in 450 areas of interest (AOIs). We introduce the “Equivalent Expression Index” a highly specific and accurate algorithm that identifies statistically significant “Equivalently-Expressed Genes”. This method facilitated the identification of molecular remodeling and mimicry genes within tissue-specific TMEs. By integrating differential expression analysis with the Equivalent Expression Index, we discovered multiple novel gene signatures associated with BCBM and primary tumor brain-metastatic potential. We demonstrate that the Equivalent Expression Index is a powerful tool to uncover shared gene expression programs representing the adaptation of metastatic cells and brain-resident cells to the BCBM microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/015422dee0790a994280a3f1f594af35182af70e" target='_blank'>
              Spatial Profiling Reveals Equivalence-Derived Molecular Signatures of Brain Mimicry and Adaptation in Breast Cancer Brain Metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, C. Yeh, Bryanna Godfrey, Pablo Nunez Perez, Giuseppe Barisano, Sushama Varma, Saman Ahmadian, A. Toland, M. Granucci, Thy T. H. Trinh, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia K Plevritis, M. Gephart
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Spatial transcriptomics offers unprecedented insights into the complex cellular landscapes of tissues, particularly in cancer research where understanding the tumor microenvironment is crucial. However, interpreting the vast and intricate data generated by this technology remains a significant challenge. This study explores the potential of Large Language Models (LLMs) to assist in the analysis and interpretation of spatial transcriptomic data from a murine melanoma tumor model. We first evaluated the performance of multiple LLM models in describing and quantifying spatial gene expression patterns. Our benchmarking revealed that spatial transcriptomics data interpretation proved challenging for most models, with only a few demonstrating sufficient capability for this complex task. Using Claude 3.5 Sonnet, which showed the highest accuracy in spot quantification and pattern recognition, we developed a systematic workflow for analyzing the tumor immune landscape. The model first assisted in identifying and prioritizing panels of M1 and M2 macrophage-associated markers through knowledge-driven scoring. It then demonstrated remarkable ability to integrate spatial expression data with extensive immunological knowledge, providing sophisticated interpretation of local immune organization. When analyzing individual tumor regions, the model identified coordinated immunosuppressive mechanisms including metabolic barriers and disrupted pro-inflammatory signaling cascades, findings that both aligned with and extended current understanding of tumor immunology. This study highlights the potential of LLMs as powerful assistive tools in spatial transcriptomics analysis, capable of combining advanced pattern recognition with extensive knowledge integration to enhance data interpretation. While significant development work remains to make such workflows scalable, our proof of concept demonstrates that LLMs can help accelerate the translation of spatial transcriptomics data into biological insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1287bc0c305d5950e4e6f2901503d04b32465b5e" target='_blank'>
              Tackling the Complexity of Spatial Transcriptomics Data Interpretation with Large Language Models
              </a>
            </td>
          <td>
            Taushif Khan, Colleen M. Farley, John J. Wilson, Chih-Hao Chang, Damien Chaussabel
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cellular senescence features a durable exit from the cell cycle triggered by stress or carcinogens. The INK4 locus is inactivated in various cancers, yet in senescence, p16Ink4a is activated. Whether senescence is tumor-suppressing or -promoting remains a conundrum. We discovered an evolutionally-conserved Vertebrata INK4-homolog. This ink4ab triggers senescence upon oxidative- and/or carcinogenic-stress. Adult Ink4ab-deficient animals failed to activate senescence and developed spontaneous cancers. Combined Ink4ab and Tp53 deficiency revealed a reciprocal senescence and apoptosis regulation, controlling tumorigenesis, including retinoblastoma. INK4-hematopoietic-deficient mice exhibited p19Arf-dependent enhanced senescence-like phenotypes, uncontrolled cell proliferation, defective stem cell differentiation, and splenomegaly, with single-splenocytes spatially-enriched in senescence-associated secretory profiles. Our studies reveal the evolutionary origin of paradigms co-regulating senescence and tumor suppression and offer strategies to exploit these reciprocal pathways for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9243e085293669a418053332a11c9df19226137a" target='_blank'>
              Evolutionary conserved reciprocal senescence and tumor suppressor signals limit lifetime cancer
              </a>
            </td>
          <td>
            Liqiong Liu, Stephani Davis, Shamila Yusuff, Ann Strange, Yonghua Zhuang, Prasanna Vaddi, Kathleen Flaherty, Kelly Jara, Joseph Kramer, Christine Archer, Schuyler Lee, Bifeng Gao, A. van Bokhoven, Wei Wang, Sharon R. Pine, Tetsuya Nakamura, Hatim E. Sabaawy
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Central Nervous System (CNS) tumors have sophisticated tumor microenvironment (TME) with different cell types such as astrocytes, microglia, neurons, vascular endothelial cells and immune cells. These non-cancerous cells orchestrate the brain TME to regulate cancer progression and therapeutic response. This study aimed to develop a cell composition deconvolution method for CNS tumor and to determine the impact of these cell compositions on patients’ outcomes. Methods We identified the cell type-specific CpG loci using the pairwise differential methylation analysis for 13 major cell types in CNS. Using non-negative least squares (NNLS) methods, we established this cell-type deconvolution approach, MDBrainT, for brain tumors. The predictive accuracy of our MDBrainT model was tested in the DNA methylation profiling of the purified cell samples and compared against another algorithm. Cell composition was predicted by MDBrainT in several brain tumor (glioma, ependymoma, medulloblastoma and ATRT) cohorts and the correlation between cell composition and tumor molecular subtypes and patient outcomes was also assessed. Results Cell type-specific CpG loci for CNS TME was used to build MDBrainT model. Based on these DNA methylation markers, MDBrainT predicted the cell composition in the TME including tumor cells with high accuracy. Endothelial cells were predominately presented in glioblastomas while the percentage of CD8 T cells wassignificantly higher in ATRT. A substantial difference of cell composition was two molecular groups of posterior fossa ependymoma (PFA vs PFB). A higher percentage of cells in TME was usually associated with worse outcomes. Conclusions MDBrainT is a robust algorithm for cell composition prediction for brain TME. Cell composition in brain TME is distinct across different pathological types and molecular subtypes. Key Points MDBrainT is a robust DNA methylation-based deconvolution approach for brain tumor microenvironment (TME). Different molecular subtypes of brain tumors have distinct cell composition patterns. Cell composition in brain TME informs patients of outcomes. Importance of this study DNA methylation is a cell type specific marker that has been utilized for tumor molecular diagnosis, disease progression and therapeutic monitoring. A DNA methylation-based classifier for brain tumors precisely predicts the molecular subtypes but not the cell composition of tumor microenvironment. Brain tumors are a complex cell mixture where tumor microenvironment is critical for tumorigenesis and therapeutic resistance. Here, we developed a novel deconvolution approach (MDBrainT) to predict cell composition for brain tumors. Our model has revealed the heterogeneity of cell composition between tumor types. In addition, tumors with different molecular subtypes have distinct cell composition. Cell percentage in TME also informs patients of outcomes. The tumor microenvironment including cell composition of each patient may direct the different regimen of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1823256c94dd6a377f1b3165f4ae951356ef07" target='_blank'>
              DNA Methylation-Based Cell Type Deconvolution Reveals the Distinct Cell Composition in Brain Tumor Microenvironment
              </a>
            </td>
          <td>
            Feili Liu, Jin Qian, Chenkai Ma
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Low-grade endometrial stromal sarcoma (LG-ESS) is a rare uterine malignancy characterized by its complex tumor microenvironment (TME) and high recurrence rates, posing challenges to accurate prognosis and effective treatment. Identifying prognostic biomarkers is essential for improving patient stratification and guiding therapeutic strategies. Methods Using single-cell transcriptome analysis combined with H&E and multiplex immunofluorescence staining, we identified a subpopulation of tumor cells in LG-ESS and further validated the association of this subpopulation and its characteristic genes with LG-ESS prognosis by molecular characterization and bulk transcriptome data. Results Our analysis reveals multiple cellular subpopulations within the tumor tissue, particularly a tumor cell subpopulation among them which is associated with poor prognosis. Originating from normal stromal fibroblasts, this subpopulation appears to play a crucial role in TME remodeling, smooth muscle cell behavior, and potentially in tumorigenesis and metastasis. Of particular interest in this subpopulation is the highly expressed FGF12 gene, which is significantly associated with a shortened survival in ESS, highlighting its potential as a prognostic biomarker. Conclusion Our study reveals the complexity of TME within the LG-ESS and highlights the role that tumor cell subpopulations play in disease progression and patient prognosis. The identification of FGF12 as a prognostic biomarker suggests a new approach for the personalized treatment and prognosis monitoring of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e49616fef8441ab9c968f5c960e3e63e7204d31e" target='_blank'>
              Single-cell RNA-seq reveals FGF12 as a prognostic biomarker in low-grade endometrial stromal sarcoma
              </a>
            </td>
          <td>
            Yu Miao, Meng Dong, Qiyin Zhou, Julia Thiel, Na Li, Ying Cai, Dan Yuan, Haitao Wang, Su-Han Jin, Hua Yang, Jinjing Wang, Benjamin Frey, Udo S. Gaipl, Hu Ma, Jianghu Zhou
          </td>
          <td>2024-11-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Studying the spatiotemporal dynamics of cells in living organisms is a current frontier in bioimaging. Intravital Microscopy (IVM) provides direct, long-term observation of cell behavior in living animals, from tissue to sub-cellular resolution. Hence, IVM has become crucial for studying complex biological processes in motion and across scales, such as the immune response to pathogens and cancer. However, IVM data are typically kept in private repositories inaccessible to the scientific community, hampering large-scale analysis that aggregates data from multiple laboratories. To solve this issue, we introduce Immunemap, an atlas of immune cell motility based on an Open Data platform that provides access to over 58’000 single-cell tracks and 1’049’000 cell-centroid annotations from 360 videos in murine models. Leveraging Immunemap and unsupervised learning, we systematically analyzed cell trajectories, identifying four main patterns of cell migration in immune cells. Two patterns correspond to behaviors previously characterized: directed movement and arresting. However, we identified two other patterns, characterized by low directionality and twisted paths, often considered random migration. We show that the newly defined patterns can be subdivided into two distinct types: within small areas, suggesting a focused patrolling around one or a few cells, and over larger areas, indicative of a more extended tissue patrolling. Furthermore, we show that the percentage of cells displaying these motility patterns changes in response to immune stimuli. Altogether, Immunemap embraces the FAIR principles, promoting data reuse to extract novel insights from immune cell dynamics through an image-based systems biology approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fcec1b17c98e94c83b787c02db01f7ce0422437" target='_blank'>
              Systematic analysis of immune cell motility leveraging Immunemap, an open intravital microscopy atlas
              </a>
            </td>
          <td>
            D. U. Pizzagalli, Pau Carrillo-Barberà, Elisa Palladino, Kevin Ceni, Benedikt Thelen, Alain Pulfer, Enrico Moscatello, R. F. Cabini, Johannes Textor, Inge M. N. Wortel, Rolf Krause, S. F. Gonzalez
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Computerized chest tomography (CT)-guided screening in populations at risk for lung cancer has increased the detection of preinvasive subsolid nodules, which progress to solid invasive adenocarcinoma. Despite the clinical significance, there is a lack of effective therapies for intercepting the progression of preinvasive to invasive adenocarcinoma. To uncover determinants of early disease emergence and progression, we used integrated single-cell approaches, including scRNA-seq, multiplexed imaging mass cytometry and spatial transcriptomics, to construct the first high-resolution map of the composition, lineage/functional states, developmental trajectories and multicellular crosstalk networks from microdissected non-solid (preinvasive) and solid compartments (invasive) of individual part-solid nodules. We found that early disease initiation and subsequent progression are associated with the evolution of immune-suppressive cellular phenotypes characterized by decreased cytotoxic CD8 T and NK cells, increased T cell exhaustion and accumulation of immunosuppressive regulatory T cells (Tregs) and M2-like macrophages expressing TREM2. Within Tregs, we identified a unique population of 4-1BB+ Treg subset enriched for the IL2-STAT5 suppressive pathway with transcription profiles supporting discrete metabolic alterations. Spatial analysis showed increased density of suppressive immune cells around tumor cells, increased exhaustion phenotype of both CD4 and CD8 T cells expressing chemokine CXCL13, and spatial micro-complex of endothelial and lymphocyte interactions within tertiary lymphoid structures. The single-cell architecture identifies determinants of early disease emergence and progression, which may be developed not only as diagnostic/prognostic biomarkers but also as targets for disease interception. Additionally, our dataset constitutes a valuable resource for the preinvasive lung cancer research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f08ee8028223311574c20ea0561992916e40482b" target='_blank'>
              Acquisition of discrete immune suppressive barriers contributes to the initiation and progression of preinvasive to invasive human lung cancer
              </a>
            </td>
          <td>
            L. Yoffe, B. Bhinder, Sung Wook Kang, Haoran Zhang, Arshdeep Singh, H. Ravichandran, G. Markowitz, Mitchell T. Martin, Junbum Kim, Chen Zhang, O. Elemento, Wesley Tansey, Stewart Bates, T. McGraw, Alain Borczuk, Hyun-Sung Lee, Nasser K Altorki, Vivek Mittal
          </td>
          <td>2025-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Tissue regeneration relies on the ability of cells to undergo de novo patterning. While tissue patterning has been viewed as the transition from initially identical un-patterned cells to an arrangement of different cell types, recent evidence suggests that initial heterogeneities between cells modulate tissue-scale pattern formation. Yet, how such heterogeneities arise and, thereafter, regulate cell type emergence in a population of cells is poorly understood. Using in vivo and in vitro mouse regenerative systems, we identify a critical tissue density that is required to induce heterogeneous inactivation of the mechanosensor YAP1. Experimental and biophysical approaches demonstrate that YAP1 cell-to-cell heterogeneity pre-patterns the first cell fate decision, via both chromatin remodelling and a supracellular feedback between FOXA1 and Delta-Notch signalling. This feedback motif induces cell fate bistability endowing memory to the system and the maintenance of patterns during homeostasis. These findings reveal a generalisable framework in which transient cell-to-cell heterogeneity, regulated by tissue-scale properties, serves as a critical control parameter for the emergence of cell fate and stable patterning during regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4e71565210ea1e9031332a79fd8f2875c11cbbd" target='_blank'>
              Cell heterogeneity and fate bistability drive tissue patterning during intestinal regeneration
              </a>
            </td>
          <td>
            C. Schwayer, S. Barbiero, D. B. Brückner, C. Baader, N. A. Repina, O. E. Diaz, L. C. Meylan, V. Kalck, S. Suppinger, Q. Yang, J. Schnabl, U. Kilik, J. G. Camp, B. Stockinger, M. Bühler, M. B. Stadler, É. Hannezo, P. Liberali
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fbdae748bda0a4cdbd87979473a945cde8f436f" target='_blank'>
              Comprehensive single-cell atlas of colorectal neuroendocrine tumors with liver metastases: unraveling tumor microenvironment heterogeneity between primary lesions and metastases
              </a>
            </td>
          <td>
            Yiqiao Deng, Qichen Chen, Chengyao Guo, Jinghua Chen, Xin Li, Zhi-yu Li, Ye‐fan Zhang, Jian‐jun Zhao, Jianguo Zhou, Jianqiang Cai, T. Yan, Xiaobing Wang, X. Bi, Zhen Huang, Hong Zhao
          </td>
          <td>2025-01-21</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The tumor secretome comprises the totality of protein factors secreted by various cell components within the tumor microenvironment, serving as the primary medium for signal transduction between tumor cells and between tumor cells and stromal cells. The deletion or mutation of the p53 gene leads to alterations in cellular secretion characteristics, contributing to the construction of the tumor microenvironment in a cell non-autonomous manner. This review discusses the critical roles of mutant p53 in regulating the tumor secretome to remodel the tumor microenvironment, drive tumor progression, and influence the plasticity of cancer-associated fibroblasts (CAFs) as well as the dynamics of tumor immunity by focusing on both secreted protein expression and secretion pathways. The aim is to provide new insights for targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e8ef931c53193f10440efa87a9f6b650fe8f46e" target='_blank'>
              Mutant p53-Mediated Tumor Secretome: Bridging Tumor Cells and Stromal Cells
              </a>
            </td>
          <td>
            Lei Qiu, Zelong Ma, Xiaoming Wu
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding the phenotypic transitions of cancer cells is crucial for elucidating tumor progression mechanisms, particularly the transition from a non-invasive spheroid phenotype to an invasive network phenotype. We developed an agent-based model (ABM) using Compucell3D, an open-source biological simulation software, to investigate how varying biophysical and biochemical parameters influence emerging properties of cellular communities, including cell growth, division, and migration. Our focus was on cell-cell contact adhesion and matrix remodeling effects on cancer cell migration. We simplified enzymatic remodeling of the extracellular matrix and the subsequent enhancements to cellular chemotaxis or durotaxis as a combined effect of localized cellular secretion of a chemoattractant. By varying the chemoattractant secretion rate and contact adhesion energy, we simulated their effects on cellular behavior and driving the transition from a spheroid phenotype to a network phenotype. The model serves as a digital twin for 3D cancer cell culture, simulating cancer cell growth, division, and invasion over 1 week, validated against published data. The simulations track the emergent morphological and collective phenotype changes using key metrics such as cell circularity and invasion. Our findings indicate that increased chemoattractant secretion enhances the invasiveness of the collective cells, promoting the transition to a network phenotype. Additionally, changing cell-cell contact energy from a strong cell-cell adhesion to a weak cell-cell adhesion affects the compactness of the spheroids, resulting in lower circularity and increased collective cell invasion. Our work advances the understanding of tumor progression by providing insights into the biophysical mechanisms behind invasive cancer cell phenotypic transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6e2c13239072839fce8a0eaeadba76fadb7701" target='_blank'>
              Agent-based modeling reveals impacts of cell adhesion and matrix remodeling on cancer collective cell migration phenotypes
              </a>
            </td>
          <td>
            Temitope O. Benson, Mohammad Aminul Islam, Kailei Liu, Ashlee N. Ford Versypt
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Identifying spatial domains is critical for understanding breast cancer tissue heterogeneity and providing insights into tumor progression. However, dropout events introduces computational challenges and the lack of transparency in methods such as graph neural networks limits their interpretability. This study aimed to decipher disease progression-related spatial domains in breast cancer spatial transcriptomics by developing the three graph regularized non-negative matrix factorization (TGR-NMF). A unitization strategy was proposed to mitigate the impact of dropout events on the computational process, enabling utilization of the complete gene expression count data. By integrating one gene expression neighbor topology and two spatial position neighbor topologies, TGR-NMF was developed for constructing an interpretable low-dimensional representation of spatial transcriptomic data. The progressive lesion area that can reveal the progression of breast cancer was uncovered through heterogeneity analysis. Moreover, several related pathogenic genes and signal pathways on this area were identified by using gene enrichment and cell communication analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744a8472d128321ec52aed7c2a2480cf5c10946b" target='_blank'>
              Deciphering progressive lesion areas in breast cancer spatial transcriptomics via TGR-NMF
              </a>
            </td>
          <td>
            Juntao Li, Shan Xiang, Dongqing Wei
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. A better understanding of the temporal dynamics and specific pathways leading into and out of the persister state is needed to identify strategies to prevent treatment failure. Using spatial transcriptomics in patient-derived xenograft models, we captured clonal lineage evolution during treatment. The persister state showed increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients. Phylogenetic tracing identified intrinsic and acquired resistance mechanisms (e.g., dual specific phosphatases, reticulon-4, and CDK2) and suggested specific temporal windows of potential therapeutic susceptibility. Deep learning-enabled analysis of histopathological slides revealed morphological features correlating with specific cell states, demonstrating that juxtaposition of transcriptomics and histological data enabled identification of phenotypically distinct populations from using imaging data alone. In summary, this study defined state change and lineage selection during melanoma treatment with spatiotemporal resolution, elucidating how choice and timing of therapeutic agents will impact the ability to eradicate resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d694b41718972dca1b5b8aa26a4498984d65d307" target='_blank'>
              Spatiotemporal Profiling Defines Persistence and Resistance Dynamics During Targeted Treatment of Melanoma.
              </a>
            </td>
          <td>
            Jill C. Rubinstein, Sergii Domanskyi, Todd B Sheridan, Brian Sanderson, SungHee Park, Jessica Kaster, Haiyin Li, Olga Anczuków, M. Herlyn, Jeffrey H. Chuang
          </td>
          <td>2024-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Background Thymomas are malignancies originating from the thymus, the primary lymphoid organ crucial for T cell development and selection. Normal thymopoiesis relies on physical and functional interactions between T-cell precursors and thymic stromal cells. Abnormalities in the frequency of T-cell developmental stages within dissociated thymoma tissue suggest that the tumour microenvironment (TME) substantially impacts thymopoiesis. Yet a detailed understanding of the precise cellular and spatial irregularities of thymoma is incomplete. This study aims to map the thymoma TME to analyse the cellular organization and interactions associated with T-cell development in the tissue. Methods Multiplexed immunofluorescence imaging detecting 28 distinct epitopes was employed to identify lymphoid and stromal cells in type B2 thymoma tissue and healthy human thymus of young and adult individuals. A computational pipeline for spatial profiling of microenvironments was developed that includes the accurate segmentation of complex-shaped stromal cells and the quantitative and qualitative analysis of cellular neighbourhoods and interactions. This pipeline was designed and validated to allow comparison of extracted spatial proteomic data across multiple tissue groups. Results Simultaneous characterisation of 57 cell-types and states was achieved for T-cells at various developmental stages, epithelial cells, hematopoietic antigen-presenting cells, and non-epithelial stromal cells. Detailed analyses of healthy tissue have been conducted on the cell-type composition and density, parenchymal distribution, tissue organization, and cell-cell interactions relevant to normal thymopoiesis. The density of T-cells as well as composition and protein expression of stromal populations in the TME substantially deviates from that observed in healthy tissue, particularly in the cortical compartments. Novel tissue niches have been identified within thymic histological compartments for both healthy and thymoma tissue, where well-defined regions of healthy thymi are either absent or reduced, in the TME, and where abnormalities in cellular interactions have been observed. Conclusions Our study established analytical methods allowing the mapping of type B2 thymoma TME at single-cell resolution as well as for high-throughput and in-depth analysis of spatial proteomic datasets. This investigation reveals region-specific spatial irregularities and abnormal cellular interactions within novel functional niches in the TME, contributing to an understanding of abnormalities of T-cell development in thymomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da72056429c6ea9f103a488ee050b57b4f824034" target='_blank'>
              AB056. Spatial analysis of T-cell development in type B2 thymoma tumour microenvironment at single-cell resolution
              </a>
            </td>
          <td>
            Kirolous S. Hanna, Fatima Dhalla, Andreas Tarcevski, M. Deadman, Pietro Zara, Bernard Liem, Rhona Taberham, Antonios Kourliouros, Dionisios Stavroulias, Adam E. Handel, M. I. Leite, Georg A. Hollander
          </td>
          <td>2024-12-01</td>
          <td>Mediastinum</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af37e793e7291e5ec4163cfb7481755bd374769" target='_blank'>
              Spatial transcriptomics reveals unique metabolic profile and key oncogenic regulators of cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Limin Zhou, Jiejie Liu, Peipei Yao, Xing Liu, Fei Chen, Yu Chen, Li Zhou, Chao Shen, You Zhou, Xin Du, Junbo Hu
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Organ development is attributed to stem/progenitor cells which self-renew and differentiate into mature cells critical for tissue form and function. The prostate is an epithelial organ that participates in the production of seminal fluid. Bipotent and unipotent stem/progenitor cells resident in prostatic epithelia are shown to support its development. While the stroma is crucial for prostate organogenesis, precise stromal cells including immature stromal subsets involved in prostate biology are unknown. Utilizing genetic reporter and lineage tracing mouse models, we identify a PDGFRα+ mesenchymal population in the fibromuscular stroma that harbors progenitors which undergo a mesenchymal-to-epithelial transition, generating prostatic luminal and basal epithelial lineages during postnatal development. Further, these mesenchymal progenitors and their epithelial progeny persist in the adult, demonstrating their self-renewal potential. Our findings unveil a prostatic progenitor beyond the epithelium, laying down a framework for probing the contribution of stromal progenitors to the normal and diseased prostate. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf2d33a5ed2ebe363dab200069d9c9378002c800" target='_blank'>
              PDGFRα+ Mesenchymal Stromal Cells Contribute to Epithelial Lineages during Prostate Development
              </a>
            </td>
          <td>
            Dibyo Maiti, Hsin-Jung Tien, Purna A. Joshi
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3852f7c6358b758c264d481b7403f1d4972dcd97" target='_blank'>
              Immunotherapy in ovarian cancer: spatial functional genomics to unravel resistance mechanisms
              </a>
            </td>
          <td>
            Martina Rausch, Karlotta Bartels, Josef Leibold
          </td>
          <td>2025-01-22</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de8ac06c638611539d33f6ef5fdb316eabe9f02c" target='_blank'>
              Spatio-temporal transcriptomics of chromothriptic SHH-medulloblastoma identifies multiple genetic clones that resist treatment and drive relapse
              </a>
            </td>
          <td>
            Ilia Kats, Milena Simovic-Lorenz, Hannah Sophia Schreiber, Pooja Sant, Jan-Philipp Mallm, Verena Körber, Albert Li, Pravin Velmurugan, Sophie Heuer, Luisa Kües, Frauke Devens, M. Sill, M. Jugold, Mahmoud Moustafa, Amir Abdollahi, Frank Winkler, A. Korshunov, Stefan M Pfister, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Mismatch repair deficiency leads to high mutation rates and microsatellite instability (MSI-H), associated with immune infiltration and responsiveness to immunotherapies. In early stages, MSI-H tumors generally have a better prognosis and lower metastatic potential than microsatellite-stable (MSS) tumors, especially in colorectal cancer. However, in advanced stages, MSI-H tumors lose this survival advantage for reasons that remain unclear. We developed a syngeneic mouse model of MSI cancer by knocking out the MMR gene Msh2 in the metastatic 4T1 breast cancer cell line. This model mirrored genomic features of MSI-H cancers and showed reduction in metastatic incidence compared to their MSS counterparts. In MSI-H tumors, we observed an enrichment of immune gene-signatures that negatively correlated with metastasis incidence. Importantly, a hybrid epithelial-mesenchymal signature, related to aggressiveness was detected only in metastatic MSI-H tumors which may explain the worse outcomes after recurrence of MSI tumors compared to MSS. Interestingly, we identified immature myeloid cells at primary and metastatic sites in MSI-H tumor-bearing mice, suggesting that MMR deficiency elicits specific immune responses beyond T-cell activation. Significance A novel syngeneic mouse model of MSI cancer demonstrates that the immune system regulates MSI cancer cell dissemination, offering an important tool to model advanced stages of human MSI-driven disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4d06352a13d99fbaae03568dc6632e6da89c31" target='_blank'>
              Effect of MisMatch Repair Deficiency on metastasis occurrence modelized in a syngeneic mouse model
              </a>
            </td>
          <td>
            Pierre Laplante, Reginaldo Rosa, L. Nebot-Bral, Jordane Goulas, Sergey I. Nikolaev, Aymeric Silvin, P. Kannouche
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In most cancers, including endometrial cancer, tumor suppressor genes harboring inactivating mutations have been systematically cataloged. However, locus-specific epigenetic alterations contributing to cancer initiation and progression remain only partly described, creating knowledge gaps about functionally significant tumor suppressors and underlying mechanisms associated with their inactivation. Here, we show that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event not associated with promoter hypermethylation. PAX2 is expressed throughout normal endometrial glands; however, microscopic clones with PAX2 protein loss arise spontaneously in adulthood and progress to endometrial precancers and cancers. Here we show that PAX2 protein loss occurs via transcriptional silencing in 80% of endometrial cancers. Mechanistically, transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that silencing is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive features (H3K27me3). The spread of the H3K27me3 signal is restrained by cohesin loops, preventing transcriptional dysregulation of neighboring genes. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. Genetically engineered mouse models together with organoid and human cell line studies established Pax2 as an in vivo endometrial tumor suppressor that cooperates with Pten to produce lethal cancers. Our discovery of a specific and recurring epigenetic alteration that transcriptionally silences PAX2 to initiate most endometrial cancers opens new lines of investigation into the origins of endometrial cancer with diverse implications for its diagnosis and treatment. One Sentence Summary We demonstrate that PAX2 is silenced by a highly recurrent epigenetic mechanism to initiate most endometrial cancers, establishing a novel mechanism of tumor suppressor inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c481e61973d015ee3841a58286f99ac21dc81f" target='_blank'>
              PAX2 is Transcriptionally Silenced by a Distinct Mechanism of Epigenetic Reprogramming to Initiate Endometrial Carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly disease. Over recent decades, a better understanding of the biological mechanisms implicated in its pathogenesis has led to the development of targeted therapies and immunotherapy, resulting in improvements in patient outcomes. To enhance our understanding of lung cancer tumor biology and advance towards precision oncology, we need a comprehensive tumor profile. In recent years, novel in situ spatial multiomics approaches have emerged, offering a more detailed view of the spatial location of tumor and tumor microenvironment cells. These advancements in in situ profiling may unveil further molecular and immune mechanisms in tumor biology that will lead to the discovery of biomarkers for treatment prediction and prognosis. In this review, we provide an overview of current and emerging pathology-based approaches for spatial immune profiling in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55e58f728c302c9eea20cd5a44be684287b92352" target='_blank'>
              Novel Spatial Approaches to Dissect the Lung Cancer Immune Microenvironment
              </a>
            </td>
          <td>
            Idania Lubo, Sharia D Hernandez, I. Wistuba, L. S. Solis Soto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Despite substantial drug discovery investments, the lack of any significant therapeutic advancement in the treatment of glioblastoma (GBM) over the past two decades calls for more innovation in the identification of effective treatments. The inter-and intra-patient heterogeneity of GBM presents significant obstacles to effective clinical progression of novel treatments by contributing to tumour plasticity and rapid drug resistance that confounds contemporary target directed drug discovery strategies. Phenotypic drug screening is ideally suited to heterogeneous diseases, where targeting specific oncogenic drivers have been broadly ineffective. Our hypothesis is that a modern phenotypic led approach using disease relevant patient derived GBM stem cell systems will be the most productive approach to identifying new therapeutic targets, drug classes and future drug combinations that target the heterogeneity of GBM. In this study we incorporate a panel of patient derived GBM stem cell lines into an automated and unbiased ‘Cell Painting’ assay to quantify multiple GBM stem cell phenotypes. By screening several compound libraries at multiple concentrations across a panel of patient-derived GBM stem cells we provide the first comprehensive survey of distinct pharmacological classes and known druggable targets, including all clinically approved drug classes and oncology drug candidates upon multiple GBM stem cell phenotypes linked to cell proliferation, survival and differentiation. Our data set representing, 3866 compounds, 2.2million images and 64000 datapoints is the largest phenotypic screen carried out to date on a panel of patient-derived GBM stem cell models that we are aware off. We seek to identify agents and targets classes which engender potent activity across heterogenous GBM genotypes and phenotypes, in this study we further characterize two validated target classes, histone deacetylase inhibitors and cyclin dependent kinases that exert broad and potent effects on the phenotypic and transcriptomic profiles of GBM stem cells. Here we present all validated hit compounds and their target assignments for the GBM community to explore.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5b141f1935b93d7a4e33f68e3c71f2d2847678" target='_blank'>
              A comprehensive pharmacological survey across heterogeneous patient-derived GBM stem cell models
              </a>
            </td>
          <td>
            Richard J. R. Elliott, P. Nagle, M. Furqan, John C Dawson, Aoife McCarthy, A. Munro, C. Drake, Gillian M Morrison, Michael Marand, Daniel Ebner, Steven M. Pollard, V. Brunton, M. Frame, Neil O. Carragher
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Effective treatment of breast cancer remains a formidable challenge, partly due to our limited understanding of the complex microenvironmental factors that contribute to disease pathology. Among these factors are tissue-resident perivascular cells, which play crucial roles in shaping vascular basement membranes, maintaining vessel integrity, and communicating with adjacent endothelial cells. Despite their essential functions, perivascular cells have been relatively overlooked. Identifying them by immunostaining has been challenging due to their low abundance, inherent heterogeneity, and shared marker expression with other cell types. These challenges have hindered efforts to purify pericytes and generate primary cell models for studying their biology. Methods Using a recently developed FACS method, we successfully identified and purified each cell type from breast tissues, allowing us to deep-sequence their transcriptomes and generate primary cell models of each cell type—including pericytes. Here, we used these data to analyze cell-type-specific gene expression in tumors, which revealed a strong association between pericyte-specific genes and breast cancer patient mortality. To explore this association, we defined the heterogeneity of breast pericytes using single-cell RNA sequencing and identified a broad marker for visualizing perivascular cells in breast tumors. Results Remarkably, we discovered perivascular cells dissociated from vessels and emerged as a dominant mesenchymal cell type in a subset of breast tumors that contrasted with their normal perivascular location. Moreover, when we purified pericytes from the breast and cultured them alongside breast tumor cells, we discovered that they induced rapid tumor cell growth significantly greater than isogenic fibroblast controls. Discussion These findings identify perivascular cells as a key microenvironmental factor in breast cancer, highlighting the critical need for further research to explore their biology and identify specific stimulatory mechanisms that could be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920638149619a80dd215db038826c980f92e7f53" target='_blank'>
              Breast pericytes: a newly identified driver of tumor cell proliferation
              </a>
            </td>
          <td>
            Katelyn Del Toro, Y. Licon-Munoz, William Crabtree, Tristan Oper, Christine Robbins, William C. Hines
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f233ae172c7106f684ef62640304f7d51d73186" target='_blank'>
              Integrative mapping of human CD8+ T cells in inflammation and cancer.
              </a>
            </td>
          <td>
            Ziwei Xue, Lize Wu, Ruonan Tian, Bing Gao, Yu Zhao, Bing He, Di Sun, Bingkang Zhao, Yicheng Li, Kaixiang Zhu, Lie Wang, Jianhua Yao, Wanlu Liu, Linrong Lu
          </td>
          <td>2024-11-29</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Bone is a multifaceted tissue requiring orchestrated interplays of diverse cells within specialized microenvironments. Although significant progress has been made in understanding cellular and molecular mechanisms of component cells of bone, revealing their spatial organization and interactions in native bone tissue microenvironment is crucial for advancing precision medicine, as they govern fundamental signaling pathways and functional dependencies among various bone cells. In this study, we present the first integrative high-resolution map of human bone and bone marrow, using spatial and single-cell transcriptomics profiling from femoral tissue. This multi-modal approach discovered a novel bone formation-specialized niche enriched with osteoblastic lineage cells and fibroblasts and unveiled critical cell–cell communications and co-localization patterns between osteoblastic lineage cells and other cells. Furthermore, we discovered a novel spatial gradient of cellular composition, gene expression and signaling pathway activities radiating from the trabecular bone. This comprehensive atlas delineates the intricate bone cellular architecture and illuminates key molecular processes and dependencies among cells that coordinate bone metabolism. In sum, our study provides an essential reference for the field of bone biology and lays the foundation for advanced mechanistic studies and precision medicine approaches in bone-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de859f69ecf65ef803fd91ebf44b36784f4b9786" target='_blank'>
              Mapping the spatial atlas of the human bone tissue integrating spatial and single-cell transcriptomics
              </a>
            </td>
          <td>
            Weiqiang Lin, Yisu Li, Chuan Qiu, Binghao Zou, Yun Gong, Xiao Zhang, Di Tian, William Sherman, Fernando Sanchez, Di Wu, Kuan-Jui Su, Xinyi Xiao, Zhe Luo, Qing Tian, Yiping Chen, Hui Shen, Hongwen Deng
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Copy number variations (CNVs), the gain or loss of genomic regions, are associated with different diseases and cancer types, where they are related to tumor progression and treatment outcome. Single cell technologies offer new possibilities to measure CNVs in individual cells, allowing to assess population heterogeneity and to delineate subclonal structures. Single cell whole-genome sequencing is considered the gold-standard for the quantification of CNVs in single cells. However, the majority of existing single cell datasets interrogate gene expression, using scRNA-seq. Consequently, several computational approaches have been developed to identify CNVs from that data modality. Nevertheless, an independent benchmarking of these methods is lacking. We used 15 scRNA-seq datasets and evaluated six popular computational methods in their ability to recover the ground truth CNVs using a large set of performance metrics. Additionally, we explored whether they could correctly identify euploid cells, especially also in fully diploid samples, and subclonal structures in heterogeneous tumor samples. We discovered several dataset-specific factors that influence the performance of the methods, such as the dataset size and the number and type of CNVs in the analyzed sample. We found that the choice of the reference dataset can have a large impact on the performance. Methods which included additional allelic information from the scRNA-seq reads performed more robustly across scenarios, but at the cost of higher runtime. Furthermore, the methods differed substantially in their additional functionalities and resource requirements. We offer a benchmarking pipeline to help identify the optimal CNV calling method for newly generated scRNA-seq datasets, and to benchmark and improve new methods performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa3f23333379961a358cfa4029d6bc3607c70188" target='_blank'>
              Benchmarking scRNA-seq copy number variation callers
              </a>
            </td>
          <td>
            Katharina T. Schmid, Aikaterini Symeonidi, Dmytro Hlushchenko, M. Richter, Maria Colomé-Tatché
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, this study develops engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples reveal the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f073ff296b054e7e4c285a38a9d240d687627e67" target='_blank'>
              Genetically Engineered Brain Organoids Recapitulate Spatial and Developmental States of Glioblastoma Progression.
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A DeSouza, Xuan Qu, Sonika Dahiya, Albert H Kim, J. Millman
          </td>
          <td>2025-01-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b347c8724e3b90166bfc42ee1abaa89c4ab82766" target='_blank'>
              A spheroid whole mount drug testing pipeline with machine-learning based image analysis identifies cell-type specific differences in drug efficacy on a single-cell level
              </a>
            </td>
          <td>
            Mario Vitacolonna, Roman Bruch, Richard Schneider, Julia Jabs, Mathias Hafner, Markus Reischl, Rüdiger Rudolf
          </td>
          <td>2024-12-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Cancer initiation and progression are controlled not only by intrinsic factors, such as genetic mutations and mesenchymal trait acquisition, but also by dynamic spatiotemporal shifts within the tumor microenvironment. These elements induce plasticity, and enhance invasiveness, metastasis, and treatment resistance, while promoting immune evasion through the reprogramming of the immune microenvironment. However, the molecular mechanisms driving this reprogramming remain unclear. In this review, we summarize the recent pivotal findings on pancreatic cancer, which is a highly malignant disease, and overview the roles of plasticity, metabolic reprogramming, and the creation of an immunosuppressive microenvironment in disease progression. We also present our research on two crucial signaling pathways, i.e., the dynamic interplay between mutational load and the tumor microenvironment, and the acquisition of cancer plasticity via inflammatory responses and metabolic reprogramming. A comprehensive understanding of these processes is expected to transform therapeutic strategies and improve the outcomes of pancreatic cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d6b2b31d56874111eca289fe4cec7a0df598ab" target='_blank'>
              Plasticity and Tumor Microenvironment in Pancreatic Cancer: Genetic, Metabolic, and Immune Perspectives
              </a>
            </td>
          <td>
            A. Hashimoto, S. Hashimoto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Existing single-cell clustering methods are based on gene expressions that are susceptible to dropout events in single-cell RNA sequencing (scRNA-seq) data. To overcome this limitation, we proposed a pathway-based clustering method for single cells (scPathClus). scPathClus first transforms single-cell gene expression matrix into pathway enrichment matrix and generates its latent feature matrix. Based on the latent feature matrix, scPathClus clusters single cells using the method of community detection. Applying scPathClus to PDAC scRNA-seq datasets, we identified two types of cancer-associated fibroblasts (CAFs), termed csCAFs and gapCAFs, which highly expressed complement system and gap junction-related pathways, respectively. Spatial transcriptome analysis revealed that gapCAFs and csCAFs are located at cancer and non-cancer regions, respectively. Pseudotime analysis suggest a potential differentiation trajectory from csCAFs to gapCAFs. Bulk transcriptome analysis showed that gapCAFs-enriched tumors are more endowed with tumor-promoting characteristics and worse clinical outcomes, while csCAFs-enriched tumors confront stronger antitumor immune responses. Compared to established CAF subtyping methods, this method displays better prognostic relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e5196bb420146fb22bc069bed5ff6f7bc8606d" target='_blank'>
              Pathway-based clustering identifies two subtypes of cancer-associated fibroblasts associated with distinct molecular and clinical features in pancreatic ductal carcinoma
              </a>
            </td>
          <td>
            Hongjing Ai, R. Nie, Xiaosheng Wang
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancers arising from dysregulation of generally operative signaling pathways are often tissue specific, but the mechanisms underlying this paradox are poorly understood. Based on striking cell-type specificity, we postulated that these mechanisms must operate early in cancer development and set out to study them in a model of von Hippel Lindau (VHL) disease. Biallelic mutation of the VHL ubiquitin ligase leads to constitutive activation of hypoxia inducible factors HIF1A and HIF2A and is generally a truncal event in clear cell renal carcinoma. We used an oncogenic tagging strategy in which VHL-mutant cells are marked by tdTomato, enabling their observation, retrieval, and analysis early after VHL-inactivation. Our findings reveal markedly different consequences of HIF1A and HIF2A activation, but that both contribute to renal cell-type specific consequences of VHL-inactivation in the kidney. Early involvement of HIF2A in promoting proliferation within proximal tubular epithelium supports therapeutic targeting of HIF2A early in VHL disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be97f9c0f0b00dada5534b4bd4b5d705fdb68b10" target='_blank'>
              HIFα isoform specific activities drive cell-type specificity of VHL-associated oncogenesis
              </a>
            </td>
          <td>
            Joana Darc Carola Correia Lima, Madeleine Hooker, Ran Li, Ayslan B. Barros, N. Masson, Christopher W. Pugh, D. Mole, Julie Adam, Peter J. Ratcliffe, Samvid Kurlekar
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="The advent of spatial transcriptomics and spatial proteomics have enabled profound insights into tissue organization to provide systems-level understanding of diseases. Both technologies currently remain largely independent, and emerging same slide spatial multi-omics approaches are generally limited in plex, spatial resolution, and analytical approaches. We introduce IN-situ DEtailed Phenotyping To High-resolution transcriptomics (IN-DEPTH), a streamlined and resource-effective approach compatible with various spatial platforms. This iterative approach first entails single-cell spatial proteomics and rapid analysis to guide subsequent spatial transcriptomics capture on the same slide without loss in RNA signal. To enable multi-modal insights not possible with current approaches, we introduce k-bandlimited Spectral Graph Cross-Correlation (SGCC) for integrative spatial multi-omics analysis. Application of IN-DEPTH and SGCC on lymphoid tissues demonstrated precise single-cell phenotyping and cell-type specific transcriptome capture, and accurately resolved the local and global transcriptome changes associated with the cellular organization of germinal centers. We then implemented IN-DEPTH and SGCC to dissect the tumor microenvironment (TME) of Epstein-Barr Virus (EBV)-positive and EBV-negative diffuse large B-cell lymphoma (DLBCL). Our results identified a key tumor-macrophage-CD4 T-cell immunomodulatory axis differently regulated between EBV-positive and EBV-negative DLBCL, and its central role in coordinating immune dysfunction and suppression. IN-DEPTH enables scalable, resource-efficient, and comprehensive spatial multi-omics dissection of tissues to advance clinically relevant discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec2a15d874ba82ad119e2c09f851c2a8a6c7876e" target='_blank'>
              Same-Slide Spatial Multi-Omics Integration Reveals Tumor Virus-Linked Spatial Reorganization of the Tumor Microenvironment
              </a>
            </td>
          <td>
            Yao Yu Yeo, Yuzhou Chang, Huaying Qiu, Stephanie Pei, Tung Yiu, Hendrik A. Michel, Wenrui Wu, Xiaojie Jin, Shoko Kure, Lindsay Parmelee, Shuli Luo, Precious Cramer, Jia Le Lee, Yang Wang, Jason Yeung, N. Ahmar, Berkay Simsek, R. Mohanna, McKayla Van Orden, Wesley Lu, K. Livak, Shuqiang Li, J. Shahryari, Leandra Kingsley, Reem N Al-Humadi, Sahar Nasr, Dingani Nkosi, Sam Sadigh, Philip Rock, Leonie Frauenfeld, Louisa Kaufmann, Bokai Zhu, Ankit Basak, Nagendra Dhanikonda, Chi Ngai Chan, Jordan Krull, Ye Won Cho, Chia-Yu Chen, Jia Ying, Joey Lee, Hongbo Wang, Bo Zhao, L. Loo, David M Kim, V. Boussiotis, Baochun Zhang, Alex K. Shalek, Brooke Howitt, S. Signoretti, Christian M. Schürch, Stephan Hodi, R. Burack, Scott J Rodig, Qin Ma, Sizun Jiang
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69505c7dd30d328b855b8e26bb2a91fa4c9f9efa" target='_blank'>
              Computer-aided analysis reveals metallothionein-positive cancer-associated fibroblasts promote angiogenesis in gastric adenocarcinoma
              </a>
            </td>
          <td>
            Xiaolong Jin, Yu Tian, Haoran Zhu, Yuewen Sun, Zhenxing Zhang
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Peripheral nerve sheath tumors (PNSTs) encompass entities with different cellular differentiation and degrees of malignancy. Spatial heterogeneity complicates diagnosis and grading of PNSTs in some cases. In malignant PNST (MPNST) for example, single cell sequencing data has shown dissimilar differentiation states of tumor cells. Here, we aimed at determining the spatial and biological heterogeneity of PNSTs.


METHODS
We performed spatial transcriptomics on formalin-fixed paraffin-embedded diseased peripheral nerve tissue. We used spatial clustering and weighted correlation network analysis to construct niche-similarity networks and gene expression modules. We determined differential expression in primary pathologies, analysed pathways to investigate the biological significance of identified meta-signatures, integrated the transcriptional data with histological features and existing single cell data, and validated expression data by immunohistochemistry.


RESULTS
We identified distinct transcriptional signatures differentiating PNSTs. Immune cell infiltration, APOD and perineurial fibroblast marker expression highlighted the neurofibroma component of hybrid PNSTs (HPNSTs). While APOD was evenly expressed in neurofibromatous tumor tissue in both, HPNST and pure neurofibromas, perineurial fibroblast markers were evenly expressed in HPNST, but restricted to the periphery in plexiform neurofibromas. Furthermore, we provide a spatial cellular differentiation map for MPNST, locating Schwann cell precursors, neural crest-like cells and those with mesenchymal transition.


CONCLUSIONS
This pilot study shows that applying spatial transcriptomics to PNSTs provides important insight into their biology. It helps establishing new markers and provides spatial information about cellular composition and distribution of cellular differentiation states. By integrating morphological and high-dimensional molecular data it can improve PNSTs classification in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2233f7b03c50daacd9edb143f0325a91952a1c6" target='_blank'>
              Spatially resolved transcriptomics of benign and malignant peripheral nerve sheath tumors.
              </a>
            </td>
          <td>
            J. Bremer, Pamela Franco, J. A. Menstell, S. Tey, K. Zajt, Klimentina Popzhelyazkova, Kay Nolte, Jürgen Schlegel, Maria Teresa Pedro, Anja Osterloh, D. Delev, M. Hohenhaus, Christoph Scholz, O. Schnell, J. Beck, Joachim Weis, D. Heiland
          </td>
          <td>2025-01-23</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ultraviolet (UV)-induced DNA mutations produce genetic drivers of cutaneous melanoma initiation and numerous neoantigens that can trigger anti-tumor immune responses in the host. Consequently, melanoma cells must rapidly evolve to evade immune detection by simultaneously modulating cell-autonomous epigenetic mechanisms and tumor-microenvironment interactions. Angiogenesis has been implicated in this process; although an increase of vasculature initiates the immune response in normal tissue, solid tumors manage to somehow enhance blood flow while preventing immune cell infiltration. By comparing the expression of transcription factors (TFs) across early-stage melanoma, naevi, and other cancer types, we found the homeodomain-containing TF HOXD13 drives a melanoblast-like developmental program, which is upregulated in melanoma and strongly correlated with angiogenesis and immune cell exclusion. Using transcriptomics, 3D chromatin profiling, and in vivo models, we demonstrate that HOXD13 upregulation promotes tumor growth in vivo by concomitantly enhancing angiogenesis and suppressing T-cell infiltration. HOXD13 orchestrates 3D chromatin contacts between distal enhancers and promoters, simultaneously activating VEGFA, SEMA3A, and CD73. VEGFA and SEMA3A remodel the tumor vasculature and CD73 elevates extracellular levels of adenosine, a vasodilator and immune suppressor that binds adenosine receptors (AdR) on endothelial and T cells. In line with these findings, HOXD13-induced growth advantage in vivo was significantly reversed by the concomitant administration of VEGFR and AdR inhibitors. By revealing a dual pro-angiogenic and immunosuppressive HOXD13-CD73/VEGF gene regulatory axis, we identify a subset of patients who might benefit from combinations of AdR and VEGFR inhibitors which are both currently being tested in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65e09ae66ed07e79880f3fb8e2a1de2aa1f2113" target='_blank'>
              A targetable developmental program co-regulates angiogenesis and immune evasion
              </a>
            </td>
          <td>
            P. Berico, Amanda Flores Yanke, Fatemeh Van Rajabpour, Catherine Do, Irving Simonin Wilmer, Theodore Sakellaropoulos, Ines Delclaux, Robert Stagnitta, Estefania Vázquez-Cruz, Iman Osman, Jane A Skok, C. D. Robles-Espinoza, Amanda W. Lund, Markus Schober, Eva Hernando
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a leading cause of cancer-related deaths globally. The heterogeneity of the tumor microenvironment significantly influences patient prognosis, while the diversity of tumor cells shapes its unique characteristics. A comprehensive analysis of the molecular profile of tumor cells is crucial for identifying novel molecular targets for drug sensitivity analysis and for uncovering the pathophysiological mechanisms underlying CRC. Methods We utilized single-cell RNA sequencing technology to analyze 13 tissue samples from 4 CRC patients, identifying key cell types within the tumor microenvironment. Intercellular communication was assessed using CellChat, and a risk score model was developed based on eight prognostic genes to enhance patient stratification for immunotherapeutic approaches. Additionally, in vitro experiments were performed on DLX2, a gene strongly associated with poor prognosis, to validate its potential role as a therapeutic target in CRC progression. Results Eight major cell types were identified across the tissue samples. Within the tumor cell population, seven distinct subtypes were recognized, with the C0 FXYD5+ tumor cells subtype being significantly linked to cancer progression and poor prognosis. CellChat analysis indicated extensive communication among tumor cells, fibroblasts, and immune cells, underscoring the complexity of the tumor microenvironment. The risk score model demonstrated high accuracy in predicting 1-, 3-, and 5-year survival rates in CRC patients. Enrichment analysis revealed that the C0 FXYD5+ tumor cell subtype exhibited increased energy metabolism, protein synthesis, and oxidative phosphorylation, contributing to its aggressive behavior. In vitro experiments confirmed DLX2 as a critical gene associated with poor prognosis, suggesting its viability as a target for improving drug sensitivity. Conclusion In summary, this study advances our understanding of CRC progression by identifying critical tumor subtypes, molecular pathways, and prognostic markers that can inform innovative strategies for predicting and enhancing drug sensitivity. These findings hold promise for optimizing immunotherapeutic approaches and developing new targeted therapies, ultimately aiming to improve patient outcomes in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8647de1e0192ab64820f770599e15f3edf4689c3" target='_blank'>
              Innovative strategies to optimise colorectal cancer immunotherapy through molecular mechanism insights
              </a>
            </td>
          <td>
            Quanjun Lin, Zhiqiang Wang, Jue Wang, Ming Xu, Xinyi Zhang, Peng Sun, Yihang Yuan
          </td>
          <td>2024-12-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The cascade of events leading to tumor formation includes induction of a tumor supporting neovasculature, as a primary hallmark of cancer. Developing vasculature is difficult to evaluate in vivo but can be captured using microfluidic chip technology and patient derived cells. Herein, we established an on chip approach to investigate the mechanisms promoting tumor vascularization and vascular targeted therapies via co-culture of cancer spheroids and endothelial cells in a three dimensional environment. Methods: We investigated both, tumor neovascularization and therapy, via co-culture of human derived endothelial cells and adjacently localized metastatic renal cell carcinoma spheroids on a commercially available microfluidic chip system. Metastatic renal cell carcinoma spheroids adjacent to primary vessels model tumor, and induce vessels to sprout neovasculature towards the tumor. We monitored real time changes in vessel formation, probed the interactions of tumor and endothelial cells, and evaluated the role of important effectors in tumor vasculature. In addition to wild type endothelial cells, we evaluated endothelial cells that overexpress Prostate Specific Membrane Antigen (PSMA), that has emerged as a marker of tumor associated neovasculature. We characterized the process of neovascularization on the microfluidic chip stimulated by enhanced culture medium and the investigated metastatic renal cell carcinomas, and assessed endothelial cells responses to vascular targeted therapy with bevacizumab via confocal microscopy imaging. To emphasize the potential clinical relevance of metastatic renal cell carcinomas on chip, we compared therapy with bevacizumab on chip with an in vivo model of the same tumor. Results: Our model permitted real-time, high-resolution observation and assessment of tumor-induced angiogenesis, where endothelial cells sprouted towards the tumor and mimicked a vascular network. Bevacizumab, an antiangiogenic agent, disrupted interactions between vessels and tumors, destroying the vascular network. The on chip approach enabled assessment of endothelial cell biology, vessel's functionality, drug delivery, and molecular expression of PSMA. Conclusion: Observations in the vascularized tumor on chip permitted direct and conclusive quantification of vascular targeted therapies in weeks as opposed to months in a comparable animal model, and bridged the gap between in vitro and in vivo models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0cee76c44f4581ac79061d246894e755a88b77d" target='_blank'>
              Vascularized tumor on a microfluidic chip to study mechanisms promoting tumor neovascularization and vascular targeted therapies
              </a>
            </td>
          <td>
            Magdalena Skubal, B. M. Larney, N. Phung, Juan Carlos Desmaras, Abdul Vehab Dozic, A. Volpe, Anuja Ogirala, Camila Longo Machado, Jakob Djibankov, V. Ponomarev, J. Grimm
          </td>
          <td>2025-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Squamous cell carcinomas (SCC) are often preceded by potentially malignant precursor lesions, most of which remain benign. The terminal exhaustion phenotypes of effector T-cells and the accumulation of myeloid-derived suppressor cells (MDSC) have been thoroughly characterized in established SCC. However, it is unclear what precancerous lesions harbor a bona fide high risk for malignant transformation and how precancerous epithelial dysplasia drives the immune system to the point of no return. Here we show that expression of SRY-box transcription factor 2 (SOX2) in precancerous lesions imparts an irreversible risk that recruits suppressive myeloid cells by promoting the release of CCL2. We developed a unique genetically engineered mouse model (GEMM) to recapitulate the malignant transformation of epithelial dysplasia to SCC in the oral mucosa with high histologic and phenotypic fidelity. Using a combination of longitudinal human specimens and the Sox2-GEMM, we found that the myeloid cells in precancerous epithelial dysplasia exhibit a distinctive dichotomous profile featuring high levels of IL-1α-SLC2A1 and low levels of type-I interferon (IFN-I) signatures, which occurs before SCC emerges histologically. Brief priming of myeloid cells with IL-1α desensitizes them to IFN-I agonists and makes myeloid-derived suppressor cells (MDSC) even more suppressive of T-cell activation. Mechanistically, IL-1 activation represses the expression of DHHC3/7 enzymes, which are responsible for the palmitoylation of stimulator of interferon genes (STING). Early blockade of IL1 signaling using pharmacologic and genetic approaches similarly reduces MDSC and SLC2A1high myeloid cells, suppresses epithelial dysplasia transformation, and extends survival. This work establishes a previously unrecognized SOX2-CCL2-IL1 pathway that leads to irreversible immune escape when precancerous epithelial lesions transform.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68906ee8a14c680b07808e1d20602c4baa417802" target='_blank'>
              SOX2-induced IL1α-mediated immune suppression drives epithelial dysplasia malignant transformation
              </a>
            </td>
          <td>
            Hülya F. Taner, Wang Gong, Z. Fitzsimonds, Zaiye Li, Yuesong Wu, Yumin He, Kohei Okuyama, Wanqing Cheng, Jung Kuczura, Sashider Rajesh, Andriana Manousidaki, Shuo Feng, Miki Lee, Felipe Nör, E. Lanzel, S. Demehri, Peter J. Polverini, Jacques E. Nör, Thomas D. Wang, Jianwen Que, Haitao Wen, Yuying Xie, James J. Moon, Yu Leo Lei
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Single cell RNA-seq (scRNA-seq) technologies provide unprecedented resolution representing transcriptomics at the level of single cell. One of the biggest challenges in scRNA-seq data analysis is the cell type annotation, which is usually inferred by cell separation approaches. In-silico algorithms that accurately identify individual cell types in ongoing single-cell sequencing studies are crucial for unlocking cellular heterogeneity and understanding the biological basis of diseases. In this study, we focus on robustly identifying cell types in single-cell RNA sequencing data; we conduct a comparative analysis using methods established in biology, like Seurat, Leiden, and WGCNA, as well as Infomap, statistical inference via Stochastic Block Models (SBM), and single-cell Graph Neural Networks (scGNN). We also analyze preprocessing pipelines to identify and optimize key components in the process. Leveraging two independent datasets, PBMC and ROSMAP, we employ clustering algorithms on cell-cell networks derived from gene expression data. Our findings reveal that while clusters detected by WGCNA exhibit limited correspondence with cell types, those identified by multiresolution Infomap and Leiden, and SBM show a closer alignment, with Infomap standing out as a particularly effective approach. Infomap notably offers valuable insights for the precise characterization of cellular landscapes related to neurodegenration and immunology in scRNA-seq.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b8c009cdac4512de3acd925b5a468747284429" target='_blank'>
              Cell Type Differentiation Using Network Clustering Algorithms
              </a>
            </td>
          <td>
            Fatemeh Sadat Fatemi Nasrollahi, Filipi Nascimento Silva, Shiwei Liu, Soumilee Chaudhuri, Meichen Yu, Juexin Wang, K. Nho, A. Saykin, David A. Bennett, Olaf Sporns, Santo Fortunato
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Novel multiplexed spatial proteomics imaging platforms expose the spatial architecture of cells in the tumor microenvironment (TME). The diverse cell population in the TME, including its spatial context, has been shown to have important clinical implications, correlating with disease prognosis and treatment response. The accelerating implementation of spatial proteomic technologies motivates new statistical models to test if cell-level images associate with patient-level endpoints. Few existing methods can robustly characterize the geometry of the spatial arrangement of cells and also yield both a valid and powerful test for association with patient-level outcomes. We propose a topology-based approach that combines persistent homology with kernel testing to determine if topological structures created by cells predict continuous, binary, or survival clinical endpoints. We term our method TopKAT (Topological Kernel Association Test) and show that it can be more powerful than statistical tests grounded in the spatial point process model, particularly when cells arise along the boundary of a ring. We demonstrate the properties of TopKAT through simulation studies and apply it to two studies of triple negative breast cancer where we show that TopKAT recovers clinically relevant topological structures in the spatial distribution of immune and tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286f79d159b2d5895452d1adff9339095baad447" target='_blank'>
              Detecting Clinically Relevant Topological Structures in Multiplexed Spatial Proteomics Imaging Using TopKAT
              </a>
            </td>
          <td>
            Sarah Samorodnitsky, Katie Campbell, Amarise Little, Wodan Ling, Ni Zhao, Yen-Chi Chen, Michael C. Wu
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cellular plasticity, involving loss of lineage determination and emergence of hybrid cell states, plays a pivotal role in non-small cell lung cancer (NSCLC) disease progression and therapy resistance. However, the full spectrum of atypical states generated in human NSCLC and the pathways that regulate them are yet to be fully elucidated. Here we examine the role of developmental programmes, alveogenesis and branching morphogenesis (BM), in regulating phenotypic diversity in NSCLC. Methods Transcriptomic analysis of epithelial cells isolated from murine lungs at different stages of organogenesis were used to derive gene signatures for developmental programmes. Bulk tissue transcriptomic datasets from human NSCLC and non-neoplastic control samples were used to identify whether developmental programmes were associated with molecular, morphological, and clinical parameters. Single-cell RNA-sequencing was used to identify malignant cell states in human NSCLC (n = 16,621 epithelial cells from 72 samples) and protein level validation of these states was carried out using multiplexed immunohistochemistry (n = 40). Results Mutually antagonistic regulation of alveogenesis and BM was found to account for a significant proportion of transcriptomic variance in human NSCLC bulk tissue datasets. BM activation was associated with poor overall survival rates in five independent lung adenocarcinoma (LUAD) cohorts (p=2.04e-13); and was significantly prognostic for resistance to tyrosine kinase inhibitors (TKIs; p=0.003) and immune checkpoint blockade (ICBs; p=0.014), in pre-treatment biopsies. Single-cell RNA-sequencing analysis revealed that malignant LUAD cells with loss of alveolar lineage fidelity predominantly acquired inflamed or basal-like cellular states, which were variably persistent in samples from TKI and ICB recurrence. Conclusions Our results show LUAD tumours undergo reversion from an alveogenic to branching morphogenic phenotype during disease progression, generating inflamed or basal-like cell states that are variably persistent following TKI or ICB treatments. These findings identify prognostic biomarkers for therapy response and underscore the role of different cell states in resistance to multiple treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/390cd095a97603dc458c49ab8aaa4b76e0155529" target='_blank'>
              Developmental programmes drive cellular plasticity, disease progression and therapy resistance in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kamila J Bienkowska, Stephany Gallardo Y, Nur S Zainal, Matthew Ellis, Maria-Antoinette Lopez, Judith Austine, Sai Pittla, S. Chee, A. Alzetani, Emily C Shaw, Christian H Ottensmeier, Gareth J Thomas, Christopher Hanley
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="PURPOSE
Pancreatic ductal adenocarcinoma (PDAC) patients with tumors enriched for the basal-like molecular subtype exhibit enhanced resistance to standard of care treatments and have significantly worse overall survival (OS) compared to patients with classical subtype enriched tumors. It is important to develop genomic resources, enabling identification of novel putative targets in a statistically rigorous manner.


EXPERIMENTAL DESIGN
We compiled a single cell RNA sequencing (scRNAseq) atlas of the human pancreas with 229 patient samples, aggregated from publicly available raw data. We mapped cell-type specific scRNAseq gene signatures in bulk RNAseq (n=744) and spatial transcriptomics (ST) (n=22) and performed validation using multiplex immunostaining.


RESULTS
Analysis of tumor cells from our scRNAseq atlas revealed nine distinct populations, two of which aligned with the basal subtype, correlating with worse OS in bulk RNAseq. Deconvolution identified one of the basal populations to be the predominant tumor subtype in non-dissociated ST tissues and in vitro tumor cell and patient-derived organoid lines. We discovered a novel enrichment and spatial association of CXCL10+ cancer associated fibroblasts (CAFs) with basal tumor cells. We identified that besides immune cells, ductal cells also express CXCR3, the receptor for CXCL10, suggesting a relationship between these cell types in PDAC tumor microenvironment.


CONCLUSIONS
We show that our scRNAseq atlas (700,000 cells), integrated with ST data, has increased statistical power and is a powerful resource, allowing for expansion of current subtyping paradigms in PDAC. We uncovered a novel signaling niche marked by CXCL10+ CAFs and basal tumor cells that could be explored for future targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07269a9cb871b60ea57ca5b2a01a49e35b66cc9b" target='_blank'>
              Human pancreatic cancer single cell atlas reveals association of CXCL10+ fibroblasts and basal subtype tumor cells.
              </a>
            </td>
          <td>
            Ian M Loveless, Samantha B. Kemp, Kailee M Hartway, Jacob T. Mitchell, Yuesong Wu, Samuel D Zwernik, D. Salas-Escabillas, Sydney Brender, Madison George, Yetunde Makinwa, Thais Stockdale, K. Gartrelle, Rohit G Reddy, Daniel W Long, Allison Wombwell, Julie M Clark, Albert M Levin, David Kwon, Ling Huang, R. Francescone, D. Vendramini-Costa, Ben Z. Stanger, Adam Alessio, Andrew M. Waters, Yuehua Cui, E. Fertig, L. T. Kagohara, Brian Theisen, Howard C Crawford, Nina G. Steele
          </td>
          <td>2024-12-05</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Muscle-invasive bladder cancer (MIBC) is a prevalent cancer characterized by molecular and clinical heterogeneity. Assessing the spatial heterogeneity of the MIBC microenvironment is crucial to understand its clinical significance. Methods In this study, we used imaging mass cytometry (IMC) to assess the spatial heterogeneity of MIBC microenvironment across 185 regions of interest in 40 tissue samples. We focused on three primary parameters: tumor (T), leading-edge (L), and nontumor (N). Cell gating was performed using the Cytobank platform. We calculated the Euclidean distances between cells to determine cellular interactions and performed single-cell RNA sequencing (scRNA-seq) to explore the molecular characteristics and mechanisms underlying specific fibroblast (FB) clusters. scRNA-seq combined with spatial transcriptomics (ST) facilitated the identification of ligand–receptor (L–R) pairs that mediate interactions between specific FB clusters and endothelial cells. Machine learning algorithms were used to construct a prognostic gene signature. Results The microenvironments in the N, L, and T regions of MIBC exhibited spatial heterogeneity and regional diversity in their components. A distinct FB cluster located in the L region—identified as S3—is strongly associated with poor prognosis. IMC analyses demonstrated a close spatial association between S3 and endothelial cells, with S3-positive tumors exhibiting increased blood vessel density and altered vascular morphology. The expression of vascular endothelial growth factor receptor and active vascular sprouting were significant in S3-positive tumors. scRNA-seq and ST analyses indicated that the genes upregulated in S3 were associated with angiogenesis. NOTCH1–JAG2 signaling pathway was identified as a significant L–R pair specific to S3 and endothelial cell interactions. Further analysis indicated that YAP1 was a potential regulator of S3. Machine learning algorithms and Gene Set Variation Analysis were used to establish an S3-related gene signature that was associated with the poor prognosis of tumors including MIBC, mesothelioma, glioblastoma multiforme, lower-grade glioma, stomach adenocarcinoma, uveal melanoma, kidney renal clear cell carcinoma, kidney renal papillary cell carcinoma, and lung squamous cell carcinoma. Conclusions We assessed the spatial landscape of the MIBC microenvironment and revealed a specific FB cluster with prognostic potential. These findings offer novel insights into the spatial heterogeneity of the MIBC microenvironment and highlight its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fcfdce6314523f58922cfd8b6a6e220595a1f7" target='_blank'>
              Spatially-resolved analyses of muscle invasive bladder cancer microenvironment unveil a distinct fibroblast cluster associated with prognosis
              </a>
            </td>
          <td>
            Chao Feng, Yaobang Wang, Wuyue Song, Tao Liu, Han Mo, Hui Liu, Shulin Wu, Zezu Qin, Zhenxing Wang, Yuting Tao, Liangyu He, Shaomei Tang, Yuanliang Xie, Qiuyan Wang, Tianyu Li
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Thymic epithelial cells (TECs) play a crucial role in T cell development and can be classified as cortical (cTEC) or medullary (mTEC). While cTECs support the expansion of T-lineage committed progenitors, mTECs are responsible for development of immunological self-tolerance. Thymic epithelial tumors (TETs) disrupt T cell development, predisposing patients to autoimmune disease (AD), which lowers their quality of life and limits treatment options. Herein, we conducted multimodal analyses of tumor-T cell interactions to dissect the immune microenvironment in TETs and explore its role in tumorigenesis and TET-related AD. Methods We applied the AutoGeneS algorithm to predict the frequency of various cell types within bulk TET samples using gene signatures from the Human Cell Atlas. Subsequently, we performed scRNAseq on a type AB thymoma from a patient with AD. We integrated these data into the Human Thymus Atlas to identify distinct cell populations. We used the CellphoneDB algorithm to detect signaling pathways linked to tumor growth. Tissue samples were derived from patients enrolled in a National Institutes of Health Institutional Review Board (NIH IRB)-approved clinical trial (NCT02146170). Results Preliminary analysis predicted high proportions of developing double positive (DP) thymocytes within AB, B1, and B2 thymomas, consistent with the World Health Organization (WHO) classification. Analysis of the epithelial and stromal compartments revealed that thymomas and thymic carcinomas consisted predominantly of cTECs and mTECs, respectively. scRNAseq of an AB thymoma identified developing thymocytes, infiltrating macrophages, and malignant TECs. The epithelial compartment consisted exclusively of cTECs. Comparison of cells from TETs with normal datasets unveiled diminished signaling between DPs and heightened signaling between immature single-positive precursors (ISPs) and tumor cells. Conclusions An imbalance of TEC subtypes is highly suggestive of aberrant thymic function resulting from distinct ontologies between TET subtypes, which enables autoreactive T cells to develop, resulting in AD. Detection of active thymopoiesis in an AB thymoma provides a potential explanation for an ongoing risk of AD during the course of the disease. Malignant cTECs interact differently with their microenvironment compared with non-malignant cTECs. These findings highlight the importance of a deeper study of tumor-T cell interactions for potential clinical innovations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/420b7cf353efc71ffbdaea039e5e002eef8fcc6e" target='_blank'>
              AB055. Tumor-T cell interactions in thymic epithelial tumors
              </a>
            </td>
          <td>
            Laercio DaSilva, M. Lavaert, Alisa K. Sivapiromrat, S. Swift, Meredith J. McAdams, Chen Zhao, Arun Rajan, A. Bhandoola
          </td>
          <td>2024-12-01</td>
          <td>Mediastinum</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff90b5963386b78b07e8f87307689a97c4a0bc10" target='_blank'>
              iFlpMosaics enable the multispectral barcoding and high-throughput comparative analysis of mutant and wild-type cells.
              </a>
            </td>
          <td>
            Irene Garcia-Gonzalez, Stefano Gambera, Susana F Rocha, Alvaro Regano, Lourdes García-Ortega, Mariya Lytvyn, Luis Diago-Domingo, Maria S. Sanchez-Muñoz, Aroa Garcia-Cabero, Ivana Zagorac, Wen Luo, Macarena De Andrés-Laguillo, Macarena Fernández-Chacón, Verónica Casquero-García, F. F. Lunella, C. Torroja, F. Sánchez-Cabo, Rui Benedito
          </td>
          <td>2024-12-13</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Fibrolamellar carcinoma (FLC) is a rare malignancy disproportionately affecting adolescents and young adults with no standard of care. FLC is characterized by thick stroma, which has long suggested an important role of the tumor microenvironment. Over the past decade, several studies have revealed aberrant markers and pathways in FLC. However, a significant drawback of these efforts is that they were conducted on bulk tumor samples. Consequently, identities and roles of distinct cell types within the tumor milieu, and the patterns of intercellular communication, have yet to be explored. In this study we unveil cell-type specific gene signatures, transcription factor networks, and super-enhancers in FLC using a multi-omics strategy that leverages both single-nucleus ATAC-seq and single-nucleus RNA-seq. We also infer completely rewired cell-to-cell communication patterns in FLC including signaling mediated by SPP1-CD44, MIF-ACKR3, GDF15-TGFBR2, and FGF7-FGFR. Finally, we validate findings with loss-of-function studies in several models including patient tissue slices, identifying vulnerabilities that merit further investigation as candidate therapeutic targets in FLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5969e5a104cf835bc8976096535f834d7534207" target='_blank'>
              Single-cell multi-omic analysis of fibrolamellar carcinoma reveals rewired cell-to-cell communication patterns and unique vulnerabilities
              </a>
            </td>
          <td>
            Alaa R. Farghli, Marina Chan, Marc S. Sherman, L. K. Dickerson, Bo Shui, M. Nukaya, Andreas Stephanou, Rosanna K. Ma, Brian J Pepe-Mooney, Colton J. Smith, Donald Long, Paul R Munn, Adrian McNairn, Jennifer K. Grenier, Michael Karski, Sean M. Ronnekleiv-Kelly, V. Pillarisetty, Wolfram Goessling, Taranjit S. Gujral, K. Vakili, Praveen Sethupathy
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Drug resistance and distant metastases are major contributors to mortality in colorectal cancer (CRC). Here we investigate mechanisms underlying acquired resistance to oxaliplatin, a first-line, standard-of-care CRC treatment. We generated oxaliplatin-resistant CRC tumor cells with clinically relevant dosing regimen, which displayed enhanced metastatic potential. Transcriptomic and phenotypic analyses revealed a critical function for cholesterol biogenesis in modulating TGF-β signaling activity, which in turn regulates SERPINE1 expression, a gene we identified as a key player in promoting drug resistance and metastasis. Additionally, we uncovered a SERPINE1-associated nine-gene expression signature, RESIST-M, that can predict overall and relapse-free survival (RFS) in clinical cohorts and is able to stratify patients into CMS4/iCMS3-fibrotic CRC-subtypes, underscoring its clinical utility. Using mouse tumor models, we provide further evidence that targeting SERPINE1 and cholesterol biogenesis can be viable approaches to re-sensitize the resistant pro-metastatic CRC cells to oxaliplatin. This study not only elucidates the molecular underpinnings of drug resistance and metastasis in primary CRC, but also offers prognostic and therapeutic strategies to guide clinical management of the disease. Significance This study reveals critical resources and insights on oxaliplatin resistance and metastasis in CRC via a novel TGF-β cholesterol axis. We generated improved oxaliplatin-resistant models that enabled identification of a prognostic SERPINE1-based gene signature to predict oxaliplatin resistance-induced metastasis in CRC. This gene signature derived from our models showed that the models can mimic CMS-4/iCMS-fibrotic-like metastatic CRC patients. We validated therapeutic candidates targeting CMS-4/iCMS-fibrotic-like metastatic CRC cells which can reverse drug resistance and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec01a84e2257e6ca8a1bf9c64a17d3d5c5ca0b06" target='_blank'>
              Modelling oxaliplatin resistance in colorectal cancer reveals a SERPINE1-based gene signature (RESIST-M) and therapeutic strategies for pro-metastatic CMS4 subtype
              </a>
            </td>
          <td>
            Stephen Qi Rong Wong, Mohua Das, Niranjan Shirgaonkar, Kenzom Tenzin, Huiwen Chua, Lin Xuan Chee, Sae Yeoh Ahpa, Astley Aruna Murugiah, Wei Yong Chua, Madelaine Skolastika Theardy, M. Thangavelu, Jane Vin Chan, Choon Kong Yap, Iain Bee Huat Tan, Petros K Tsantoulis, Sabine Tejpar, J. Loo, Ramanuj DasGupta
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Reduced therapy response in breast cancer has been correlated with heterogeneity in biomarker composition, expression level, and spatial distribution of cancer cells within a patient tumor. Thus, there is a need for models to replicate cell-cell, cell-stromal, and cell-microenvironment interactions during cancer progression. Traditional two-dimensional (2D) cell culture models are convenient but cannot adequately represent tumor microenvironment histological organization, in vivo 3D spatial/cellular context, and physiological relevance. Recently, three-dimensional (3D) in vitro tumor models have been shown to provide an improved platform for incorporating compositional and spatial heterogeneity and to better mimic the biological characteristics of patient tumors to assess drug response. Advances in 3D bioprinting have allowed the creation of more complex models with improved physiologic representation while controlling for reproducibility and accuracy. This review aims to summarize the advantages and challenges of current 3D in vitro models for evaluating therapy response in breast cancer, with a particular emphasis on 3D bioprinting, and addresses several key issues for future model development as well as their application to other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ccd5bcde9ada94a158d2c3401e2be4034200cf" target='_blank'>
              Modeling intratumor heterogeneity in breast cancer.
              </a>
            </td>
          <td>
            E. McDonough, M. Barroso, Fiona Ginty, David T Corr
          </td>
          <td>2024-12-06</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background The hepatocellular carcinoma is one of the leading causes of cancer-related mortality and is characterized by high heterogeneity and subsequently adaptation by developing resistance to current treatments. In this scenario the application of individualized models is crucial to understand the potential of approved therapies. Recently, we established a series of individual cell lines derived from patients who developed HCC on different entities, serving as a platform for individual approaches. In this study, we classified the LC4 cells derived from the center region of a HCC with underlying HIV-HCV co-infection, by using deep analysis on the pathway regulation level. Methods We employed DEG analysis, followed by pathway analysis to characterize the preservation level of the LC4 cells and the level of adoption. Next, we classify the model, by employing healthy donor samples, commonly used HCC cell lines and global RNAseq data sets. Results We showed that the LC4 cells reflect significant characteristics of the parental region, including the replication of the immuno-suppressive and the proliferative milieu. The LC4 cells exhibit a metabolic reprogramming characterized by the downregulation of drug-metabolizing CYP enzymes compared to healthy individuals, indicating a transition to alternate metabolic pathways. Moreover, we identified common Biomarkers in the parental tissue, global datasets and the LC4 cells. Conclusion We showed that the LC4 cell line is applicable as an individual model for pre-clinical testing of treatment regimens in HCC driven research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7af70ab32bac191d0044ebaf22a9c582e4995355" target='_blank'>
              Patient-derived HCC cells recapitulating the transcriptomic landscape of primary HCV-related liver cancer
              </a>
            </td>
          <td>
            Janine Kah, Lisa Staffeldt, T. Volz, K. Schulze, A. Heumann, Götz Rövenstrunk, Meike Goebel, S. Peine, M. Dandri, Stefan Lüth
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) tumor cells and their interactions with the immune microenvironment, particularly at the leading-edge area, have been underexplored. This study employs single-cell RNA sequencing (scRNA-seq) and spatial transcriptome (ST) analysis on samples from the tumor core, adjacent non-tumorous tissue, and the leading-edge area of nine ICC patients. These findings indicate that tumor cells at the leading-edge area demonstrate enhanced proliferation and are tightly associated with the stroma, including endothelial cells and POSTN+ FAP+ fibroblasts. Notably, CD8+ T cells in this region exhibit a naive phenotype with low cytotoxicity and signs of exhaustion, likely due to compromised antigen presentation by antigen-presenting cells (APCs). The predominant CD8+ T cell subset, mucosal-associated invariant T (MAIT) cells, recruits SPP1+ macrophages within the stroma. This interaction, along with the presence of POSTN+ cancer-associated fibroblasts (CAFs) and endothelial cells, forms a unique "triad structure" that fosters tumor growth and ICC progression. The research highlights the intricate characteristics and interactions of ICC tumor cells in the leading-edge area, offering insights into potential therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4692f475931cf20c24dd17b389c955de7ede39" target='_blank'>
              Single-Cell and Spatial Transcriptomics Delineate the Microstructure and Immune Landscape of Intrahepatic Cholangiocarcinoma in the Leading-Edge Area.
              </a>
            </td>
          <td>
            Li Zuyin, Li Zhao, Cheng Qian, Changkun Zhang, Ma Delin, Jialing Hao, Chen Zhuomiaoyu, Li Yuzi, Zheng Jiaxi, Gao Jie, Zhu Jiye
          </td>
          <td>2024-12-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gamma-delta ( γδ ) T cells are a crucial component of the tumor immune microenvironment which are considered a promising potential therapeutic strategy and target. Increasing evidence suggests that these unique immune cells play significant roles across various cancers. However, γδ T cells are often regarded as having dual roles in tumors, and their influence on lung adenocarcinoma (LUAD) remains controversial. In this research, we employed a wild-ranging approach using multi-omics data to investigate the function of γδ T cells in LUAD. The abundance of γδ T cell infiltration is linked to a positive prognosis in lung adenocarcinoma. The tumor-inhibiting role of γδ T cells was played through intrinsic lineage evolution, acquiring cytotoxic functions and engaging in signal transduction with antigen-presenting cells. Furthermore, patients with higher γδ T cells infiltration abundance might be more favorable for immunotherapy. Lastly, we established a predictive model using CT images based on radiomics, providing a non-invasive strategy to assess γδ T cells infiltration in LUAD patients. These findings offer new insights and perspectives the personalized therapies of γδ T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ada3c2d9795ca814751eeb983c8b55cf90854b2" target='_blank'>
              Integrating of Radiomics and Single-cell Transcriptomics Uncovers the Crucial Role of γδ T Cells in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Ziyi Chen, Changqing Yang, Jiajing Wang, Baichuan Wang, Long He, Zhaoyi Liu, Yingxi Li, Tingting Qin, Peng Chen
          </td>
          <td>2025-01-06</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Epithelial-mesenchymal plasticity (EMP) is a cell-fate switching program that enables cells to adopt a spectrum of phenotypes ranging from epithelial (E) to mesenchymal (M) and hybrid E/M states. Hybrid E/M phenotypes are considered optimal for cancer metastasis due to their association with metastatic initiation and traits such as stemness, drug resistance, and collective migration. Boolean models of gene regulatory networks (GRNs) underlying EMP have provided valuable insights into the dynamics of E and M phenotypes. However, these models fail to capture hybrid phenotypes effectively, as Boolean frameworks allow only binary gene expression, while hybrid E/M cells exhibit partial expression of both E and M genes. To address this limitation, we modified a threshold-based Boolean formalism to incorporate intermediate gene expression levels. The resulting multilevel model reveals new hybrid steady states with partial expression of both E and M genes, expanding the phenotypic landscape beyond that captured by traditional Boolean models. These hybrid states exhibit lower frustration and higher abundance, indicating enhanced stability. By resolving degeneracy in the dynamics, we further demonstrate that hybrid states identified by the multilevel model are more stable. These findings suggest that adding minimal complexity to Boolean models can uncover previously hidden qualitative features of phenotypic landscapes governed by GRNs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29922a69bbdff28fdd118c9e80f2fc16659ff8c" target='_blank'>
              A multi-level formalism to capture the characteristics of hybrid phenotypes in Epithelial-Mesenchymal Plasticity
              </a>
            </td>
          <td>
            K. Hari, Shubham Tripathi, Vaibhav Anand, M. Jolly, Herbert Levine
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The adaptation of metastatic cells to their host tissue critically determines the pathogenicity of a cancer and therefore patient survival. Yet, it remains elusive to what extent the host environment drives gene expression programs in metastatic cells. Here we identify adaptive mechanisms that enable metastases to establish themselves in a novel tissue context. We performed single-cell RNA-sequencing on malignant and benign tissue samples from untreated donors with colorectal adenocarcinoma and liver metastasis to deduce tissue adaptive expression patterns. A novel computational approach identified genes and pathways that consistently adapted to the host tissue at the transition from the primary tumor to the paired metastasis across donors. This analysis revealed that the majority of expression changes in the metastasis reflect an expression signature reminiscent of benign liver epithelial cells. Cellular processes adapting to the liver environment include basic cellular functions such as energy metabolism, as well as tissue-specific pathways such as the regulation of lipid metabolism by PPAR-α. These adaptations potentially increase the pathogenicity of the metastatic cells and may provide new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189b4d35fab487ddefe0f36a861c9c03b2e18b5a" target='_blank'>
              Gene expression adaptation of metastases to their host tissue
              </a>
            </td>
          <td>
            L. Nagel, Marten C. Wenzel, Sascha Hoppe, Patrick S. Plum, Mohammad Karimpour, Marek Franitza, Roger Wahba, Marc Bludau, Christiane J. Bruns, Alexander Quaas, A. Beyer, Axel M. Hillmer
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) account for 0.01 to 2% of the total tumor mass; however, they play a key role in tumor progression, metastasis and resistance to current cancer therapies. The generation and maintenance of CSCs are usually linked to the epithelial–mesenchymal transition (EMT), a dynamic process involved in reprogramming cancer cells towards a more aggressive and motile phenotype with increased stemness potential. Cells that undergo an EMT process have shown to be more resistant to conventional chemo/radiotherapies. In this context, aldehyde dehydrogenase (ALDH) enzymes, known for their role in the cellular detoxification of aldehydes and enhancement of cell survival, are often upregulated in cancer cells, promoting their resistance to conventional cancer treatments. Indeed, high ALDH levels have become a hallmark biomarker of CSCs and are often used to isolate this sub-population from the more abundant cancer cell populations. Herein, we isolated human breast cancer epithelial cells with higher ALDH abundance (ALDHHigh) and compared them to those with low ALDH abundance (ALDHLow). ALDHHigh sub-populations exhibited more characteristic EMT biomarkers by adopting a more mesenchymal phenotype with increased stemness and enhanced migratory potential. Furthermore, ALDHHigh sub-populations displayed elevated senescent markers. Moreover, these cells also demonstrated higher levels of mitochondria DNA/mass, as well as greater mitochondrial and glycolytic metabolic function. Conversely, ALDHLow sub-populations showed a higher efficiency of mammosphere/colony formation and an increased proliferative capacity. Therefore, we demonstrated that these ALDH sub-populations have distinct characteristics, underscoring their role in EMT, the formation of tumors and the mechanisms of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c656cc9a0663a21dc7738f32f6803ef9c5938a5" target='_blank'>
              ALDHHigh Breast Cancer Stem Cells Exhibit a Mesenchymal–Senescent Hybrid Phenotype, with Elevated Metabolic and Migratory Activities
              </a>
            </td>
          <td>
            Luis Larrea Murillo, Conor J. Sugden, Bela Ozsvari, Zahra Moftakhar, Ghada S. Hassan, F. Sotgia, Michael P Lisanti
          </td>
          <td>2024-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="The complex interplay between metal abundance, transport mechanisms, cell distribution, and tumor progression-related biological pathways (e.g., metabolism, collagen remodeling) remains poorly understood. Traditionally, genes and metals have been studied in isolation, limiting insights into their interactions. Recent advances in spatial transcriptomics and elemental profiling now enable comprehensive exploration of tissue-wide metal-gene interactions, though integration remains challenging. In this proof-of-concept study, we investigated metal-dependent signaling within the tumor microenvironment of a unique colorectal cancer (CRC) tumor. We implemented a spatial multimodal workflow which integrated elemental imaging, gene expression, cellular composition, and histopathological features to uncover metals-related pathways through spatially resolved differential expression analysis. Preliminary findings revealed significant associations, for instance: elevated iron correlated with mesenchymal phenotypes located at the tumor's proliferative front, reflecting epithelial-to-mesenchymal transition pathways, and extracellular matrix remodeling. High concentrations of copper were predominantly localized in regions of active tumor growth and associated with upregulation of immune response genes. This proof-of-concept workflow demonstrates the feasibility of integrating elemental imaging with spatial transcriptomics to identify metals-based gene correlates. Future application of this workflow to larger patient cohorts will pave the way for expansive comparisons across the metallome and transcriptome, ultimately identifying novel targets for tumor progression biomarkers and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b7bf64840757cf5e603b33283a2afe6a77fb5a3" target='_blank'>
              Integration of Elemental Imaging and Spatial Transcriptomic Profiling for Proof-of-Concept Metals-Based Pathway Analysis of Colon Tumor Microenvironment
              </a>
            </td>
          <td>
            A. Srivastava, N. Shaik, Y. Lu, M. Chan, A. B. Diallo, S. Han, T. Punshon, B. P. Jackson, L. Vahdat, X. Liu, V. Mittal, K. S. Lau, J. Gui, L. Vaickus, J. Hoopes, F. Kolling, L. Perreard, J. Marotti, J. Levy
          </td>
          <td>2024-12-10</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Cryoablation induces antitumor immune responses. Spatial transcriptomic landscape technology has been used to determine the micron-level panoramic transcriptomics of tissue slices in situ.


METHODS
The effects of cryoablation on the immune microenvironment in non-small cell lung cancer (NSCLC) were explored by comparing the Whole Transcriptome Atlas (WTA) panel of immune cells before and after cryoablation using the spatial transcriptomic landscape.


RESULTS
The bioinformatics analysis showed that cryoablation significantly affected the WTA of immune cells, particularly genes related to cellular components, biological processes, molecular functions, proliferation and migration, and cytokine-cytokine receptor interaction signaling pathways.


CONCLUSIONS
The findings of this study suggest that cryoablation significantly impacts the biological functions of immune cells in the tumor microenvironment of NSCLC through multiple mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3a0230d879f0812d1beffd1bcd78bcdc486ebd" target='_blank'>
              Effect of cryoablation on the spatial transcriptomic landscape of the immune microenvironment in non-small cell lung cancer.
              </a>
            </td>
          <td>
            Jun Zhou, Shengxi Wang, Ming Liu, Zhaopei Li
          </td>
          <td>2024-12-01</td>
          <td>Journal of cancer research and therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastatic cancer, a major cause of mortality, has been understudied compared to primary tumors, leaving gaps in our understanding of how cancer genes adapt between these states. We analyzed the association between mutations and copy-number alterations in 25,000 tumor samples from both primary and metastatic cancers. Our findings show that cancer genes display distinct interaction strengths across these states, with 27.45% of genes, including ARID1A, FBXW7, and SMARCA4, shifting between one-hit and two-hit drivers. Interaction strengths varied by cancer state and treatment conditions, revealing seven state-specific interactions. We also identified 38 primary-specific and 21 metastatic-specific high-order interactions, enriched in cancer hallmarks, indicating unique tumor progression mechanisms. These findings highlight dynamic tumor progression mechanisms and underscore the importance of considering cancer state in research and treatment strategies for precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968f72ea3f3d388ad8546b594ad1735637aaf86d" target='_blank'>
              Mapping cancer gene dynamics through state-specific interactions
              </a>
            </td>
          <td>
            Adrián Maqueda-Real, Laia Ollé-Monràs, Solip Park
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d052ec02be986508482ae1f68379c54cb0ed96e8" target='_blank'>
              Cell-type specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Spatial metabolomics holds great promise for unraveling the complexities of metabolic reprogramming in cancer, yet its development lags behind the rapid progress of spatial transcriptomics (ST). To bridge this gap and maximize the value of spatial gene expression datasets, we present Spatial Flux Balance Analysis (spFBA), a novel framework that transcends the limitations of existing gene set enrichment methods. The spFBA approach builds upon previous work designed for bulk and single-cell data to return Flux Enrichment Scores, up to the level of single reactions, that can distinguish their preferred directional usage retaining spatial resolution. spFBA integrates differential constraints on flux boundaries in steady-state metabolic modeling, informed by spatial gene expression, and utilizes corner-based flux sampling. We first tuned and validated spFBA using a publicly available 10x Visium ST dataset from renal cancer samples at the tumor-normal tissue interface. spFBA demonstrated the ability to recapitulate the tissue architecture in renal tumor sections, clearly delineating sustained metabolic growth in the tumor core compared to adjacent normal renal parenchyma. Furthermore, spFBA captured the Warburg effect, revealing distinct metabolic subpopulations within the tumor core, while highlighting lactate excretion as a hallmark across all subpopulations. We then applied spFBA to our own high-resolution ST datasets generated with stereo-seq from a colon cancer patient, including both primary tumor and liver metastases. Unlike the renal case, spFBA uncovered extensive areas of tumor cells consuming lactate in both primary and metastatic sites, with stromal regions in liver metastases producing lactate, suggesting a reverse Warburg effect at play. These findings highlight that cancer metabolic rewiring and the tumor composition of metabolic subpopulations depend highly on the tissue of origin and the spatial context within the target organ. spFBA emerges as a powerful tool to leverage the rapidly growing collection of spatial gene expression data, offering unprecedented insights into the intricate metabolic landscape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a4d090d655d07f06eab7f172bddcddf9fddf74f" target='_blank'>
              Spatial Flux Balance Analysis reveals tissue-of-origin and spatially dependent metabolic rewiring in renal and colorectal cancer
              </a>
            </td>
          <td>
            Davide Maspero, Giovanni Marteletto, Francesco Lapi, B. Galuzzi, Irene Ruano, Ben Vandenbosch, Ke Yin, Sabine Tejpar, Alex Graudenzi, Holger Heyn, Anna Pascual-Reguant, Chiara Damiani
          </td>
          <td>2024-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Background Pericytes are crucial for the development, stabilization, and functional regulation of microvasculature, especially in the retina. In diabetic retinopathy (DR), early loss of pericytes is a key event that drives microvascular dysfunction. Despite their critical role, the mechanisms underlying the functional heterogeneity of pericytes in DR remain poorly understood, impeding the development of effective therapeutic strategies. Methods We employed single-cell RNA sequencing to construct a comprehensive single- cell atlas of non-diabetic and diabetic retinas. Using bioinformatic clustering and subcluster analysis, we identified a specific pericyte subcluster associated with diabetic microvascular complications. Differential gene expression analysis and immunofluorescence validation highlighted PTTG1 as a potential key regulator of pericyte dysfunction. To investigate its functional role, we emplyed CRISPR/Cas9 and adenoviral vectors to modulate PTTG1 expression in vitro and in vivo. Combined transcriptomic and metabolomic approaches were used to explore the mechanistic pathways through which PTTG1 influences pericyte biology and vascular function. Results We identified a novel pericyte subcluster characterized by elevated expression of PTTG1, which was strongly correlated with diabetic microvascular dysfunction. Silencing PTTG1 using CRISPR/Cas9 and siRNA in vitro mitigated pericyte dysfunction under high- glucose conditions. Targeted knockdown of PTTG1 using viral vectors improved retinal vascular integrity and reduced neovascularization in diabetic mice. Transcriptomic and untargeted metabolomic analyses revealed that PTTG1 knockdown reprogrammed pericyte energy metabolism by modulating glycolysis pathway genes, reducing oxidative stress, and restoring pericyte function, ultimately alleviating microvascular dysfunction in DR. Conclusions PTTG1 plays a critical role in regulating pericyte dysfunction and maintaining vascular homeostasis in diabetic retinopathy. By modulating key metabolic pathways and pericyte phenotypes, PTTG1 represents a promising therapeutic target for treating diabetic microvascular complications. These insights offer a novel molecular framework for developing targeted therapies aimed at restoring retinal vascular health in diabetic patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d2cfb264dd76b8c3dd1f5b9db0dce030fd7e6a3" target='_blank'>
              PTTG1-Mediated Pericyte Dysfunction Drives Diabetes-Induced Microvascular Dysfunction
              </a>
            </td>
          <td>
            Linyu Zhang, Ling Ren, Jingyue Zhang, Min Xia, Xiaosa Li, Mudi Yao, Fanfei Ma, Chang Jiang, Jin Yao, Biao Yan
          </td>
          <td>2024-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The metastasis of solid tumors hinges on cancer cells navigating through complex three-dimensional tissue environments, characterized by mechanical heterogeneity and biological diversity. This process is closely linked to the dynamic migration behavior exhibited by cancer cells, which dictates the invasiveness of tumors. In our study, we investigate tumor spheroids composed of breast cancer cells embedded in three-dimensional (3D) collagen matrices. Through a combination of quantitative experiments, artificial-intelligence-driven image processing, and mathematical modeling, we uncover rapid transitions in cell phenotypes and phenotype-dependent motility among disseminating cells originating from tumor spheroids. Persistent invasion leads to continuous remodeling of the extracellular matrix surrounding the spheroids, altering the landscape of migration phenotypes. Consequently, filopodial cells emerge as the predominant phenotype across diverse extracellular matrix conditions. Our findings unveil the complex mesoscale dynamics of invading tumor spheroids, shedding light on the complex interplay between migration phenotype plasticity, microenvironment remodeling, and cell motility within 3D extracellular matrices.




 Published by the American Physical Society
 2024


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afedef8827a246c2a67d180498da9a907634ab61" target='_blank'>
              Coupled Dynamics in Phenotype and Tissue Spaces Shape the Three-Dimensional Cancer Invasion
              </a>
            </td>
          <td>
            Austin Naylor, Maximilian Libmann, Izabel Raab, W.-J. Rappel, Bo Sun
          </td>
          <td>2024-12-24</td>
          <td>PRX Life</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb406f40baccfe449ee7910f107c5384df35a006" target='_blank'>
              Spatially resolved transcriptomics reveals gene expression characteristics in uveal melanoma
              </a>
            </td>
          <td>
            Jing-Ying Xiu, Yu-Ning Chen, Ya-Li Mao, Jingqiu Luo, Hao-Wen Li, Yang Li, 
          </td>
          <td>2025-01-15</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Neuroblastoma is a heterogeneous disease with adrenergic (ADRN)- and therapy resistant mesenchymal (MES)-like cells driven by distinct transcription factor networks. Here, we investigate the expression of immunotherapeutic targets in each neuroblastoma subtype and propose pan-neuroblastoma and cell state specific targetable cell-surface proteins.


METHODS
We characterized cell lines, patient-derived xenografts, and patient samples as ADRN-dominant or MES-dominant to define subtype-specific and pan-neuroblastoma gene sets. Targets were validated with ChIP-sequencing, immunoblotting, and flow cytometry in neuroblastoma cell lines and isogenic ADRN-to-MES transition cell line models. Finally, we evaluated the activity of MES-specific agents in vivo and in vitro.


RESULTS
Most immunotherapeutic targets being developed for neuroblastoma showed significantly higher expression in the ADRN subtype with limited expression in MES-like tumor cells. In contrast, CD276 (B7-H3) and L1CAM maintained expression across both ADRN and MES states. We identified several receptor tyrosine kinases (RTKs) enriched in MES-dominant samples and showed that AXL targeting with ADCT-601 was potently cytotoxic in MES-dominant cell lines and showed specific anti-tumor activity in a MES cell line-derived xenograft.


CONCLUSIONS
Immunotherapeutic strategies for neuroblastoma must address the potential of epigenetic downregulation of antigen density as a mechanism for immune evasion. We identified several RTKs as candidate MES-specific immunotherapeutic target proteins for the elimination of therapy-resistant cells. We hypothesize that the phenomena of immune escape will be less likely when targeting pan-neuroblastoma cell surface proteins such as B7-H3 and L1CAM, and/or dual targeting strategies that consider both the ADRN- and MES-cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ce67e86c3ed1eb1595b4d5bed2ad026b261bd59" target='_blank'>
              Lineage-dependence of the neuroblastoma surfaceome defines tumor cell state-dependent and independent immunotherapeutic targets.
              </a>
            </td>
          <td>
            N. Kendsersky, Michal Odrobina, Nathaniel W. Mabe, Alvin Farrel, Liron D. Grossmann, M. Tsang, David Groff, A. Wolpaw, Alaa Narch, F. Zammarchi, P. V. van Berkel, C. V. Dang, Y. Mossé, K. Stegmaier, John M. Maris
          </td>
          <td>2025-01-18</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="BACKGROUND
Growth hormone-secreting pituitary adenomas (GHPA) display diverse biological behaviors and clinical outcomes, necessitating the identification of tumor heterogeneity and prognostically relevant markers.


METHODS
In this study, we performed single-cell RNA sequencing (scRNA-seq) on 10 GHPA samples, four of which also underwent spatial transcriptome sequencing, and used scRNA-seq data from four normal pituitary samples as controls. Cell subtype characterization in GHPA was analyzed using multiple algorithms to identify malignant bias regulators, which were then validated using a clinical cohort.


RESULTS
We constructed the first single-cell and spatial transcriptome profiles of GHPA, which contained 87,862 cells and revealed 16 tumor cell subtypes. Among the tumor cells, we identified distinct developmental trajectories and three malignant-biased subtypes (PIT1_C05, PIT1_C06, and PIT1_C10). The spatial distribution characteristics of these malignant-biased cells may influence the growth characteristics and prognosis of GHPA. We screened specific regulatory transcription factors, including FOXO1, GTF2IRD1, and MAX. Clinical cohort validation indicated that FOXO1 might be associated with tumor invasion and progression, while high expression of MAX could result in poor endocrine outcomes.


CONCLUSION
GHPA exhibits rich heterogeneity and diverse cell subtypes, with specific transcription factors potentially regulating cell malignant bias, thereby influencing tumor characteristics and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a105784b2d156549c656072efae7d8dac904126" target='_blank'>
              Single-cell and spatial transcriptomic analyses reveal heterogeneity characteristics and specific cell subtype regulators in growth hormone-secreting pituitary adenomas.
              </a>
            </td>
          <td>
            Yu Zhang, Lifeng Wang, Xingcheng Yi, Xinyu Ma, Hongyu Wu, Mingzhao Zhang, Zhenqi Yang, Lizhen Ma, Zenghua Mi, Wei-jia Zhi, Cong Fu, Pinan Liu, Zhijun Yang
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="A comprehensive understanding of tumor heterogeneity, tumor microenvironment and the mechanisms of drug resistance is fundamental to advancing breast cancer research. While single-cell RNA sequencing has resolved the issue of “temporal dynamic expression” of genes at the single-cell level, the lack of spatial information still prevents us from gaining a comprehensive understanding of breast cancer. The introduction and application of spatial transcriptomics addresses this limitation. As the annual technical method of 2020, spatial transcriptomics preserves the spatial location of tissues and resolves RNA-seq data to help localize and differentiate the active expression of functional genes within a specific tissue region, enabling the study of spatial location attributes of gene locations and cellular tissue environments. In the context of breast cancer, spatial transcriptomics can assist in the identification of novel breast cancer subtypes and spatially discriminative features that show promise for individualized precise treatment. This article summarized the key technical approaches, recent advances in spatial transcriptomics and its applications in breast cancer, and discusses the limitations of current spatial transcriptomics methods and the prospects for future development, with a view to advancing the application of this technology in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9484c611ac13b7e4460cc6b1b5eb2c6b25e4a9" target='_blank'>
              Spatial transcriptomics in breast cancer: providing insight into tumor heterogeneity and promoting individualized therapy
              </a>
            </td>
          <td>
            Junsha An, Yajie Lu, Yuxi Chen, Yuling Chen, Zhaokai Zhou, Jianping Chen, Cheng Peng, Ruizhen Huang, Fu Peng
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer evolution follows complex trajectories involving diverse genomic alterations and clonal dynamics, making it challenging to validate computational methods for single-cell DNA sequencing analysis. Here we present cancerSimCraft, a comprehensive framework for simulating cancer genome data at both clonal and single-cell resolution. cancerSimCraft combines deterministic rules with stochastic processes to model various genomic events including CNVs, SNVs, and WGDs. The framework enables integration of real cancer genome patterns with user-defined parameters, supporting customizable simulation designs that reflect both empirical data and theoretical models. Through systematic benchmarking, we demonstrate cancerSimCraft’s utility in evaluating computational methods under various conditions, particularly focusing on the impact of dataset size and parameter sensitivity. We further demonstrated its application in exploring clonal evolution and mutation patterns through controlled in silico experiments. These comprehensive simulation capabilities make cancerSimCraft a valuable resource for both computational method development and theoretical studies in cancer genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617e592c6c24284ee3c33b23e1a58a1555ca1d22" target='_blank'>
              cancerSimCraft: A Multi-resolution Cancer Genome Simulator with Comprehensive Ground Truth Tracking
              </a>
            </td>
          <td>
            Haijing Jin, Nicholas Navin, Ken Chen
          </td>
          <td>2024-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Thymic epithelial cells (TECs) consist of two distinct components: cortical TECs (cTECs) and medullary TECs (mTECs), which are crucial for the positive and negative selection of thymocytes, respectively. Among thymic epithelial tumors (TETs) originating from TECs, thymic carcinomas (TCs) exhibit the worst prognosis. While the cell of origin of TCs has long been enigmatic, our previous study has demonstrated that TCs reflect the signature of mTECs utilizing bioinformatics approaches and immunohistochemistry of autoimmune regulator (AIRE), a molecule specific to mTECs. However, the cellular heterogeneity of TCs relevant to their pathogenesis remains elusive. It motivated us to dissect the tumor heterogeneity in more detail. Methods To investigate the cancer ecosystem underlying TCs, we have conducted a comprehensive spatial transcriptome analysis on two cases of TCs. We also utilized the bulk RNA-seq dataset of TETs from The Cancer Genome Atlas project archive and single-cell RNA-seq datasets of normal TECs and TC from the public database, ensuring the validity of our findings. Results Our spatial transcriptome analysis identified the spatial distribution of key molecules associated with TCs. While the well-known TC markers, KIT and CD5, were broadly expressed across the tumor tissues, the expression of AIRE was sporadic, mirroring its distribution in the normal thymus. Unsupervised clustering also identified a distinct cluster with a high level of AIRE expression, and this cluster expressed genes relevant to AIRE-expressing mTECs in the normal thymus, suggesting that mTEC heterogeneity was maintained in TC to some extent. Furthermore, re-analysis of single-cell RNA-seq data on TC revealed the existence of unique tumor subpopulations resembling thymic myoid cells and post-AIRE mTECs. Our spatially aware analysis also highlighted a unique gene module localized at the margin of tumor nests. Conclusions Spatial transcriptomics has given us new insights into the biology of TCs, further suggesting the cellular heterogeneity in TCs associated with mTECs. These findings have significant implications for our understanding of TCs and their pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49d6f81c5ca890050ea45a0fe342b2a626e984cb" target='_blank'>
              AB028. Spatial transcriptomics to dissect the cellular heterogeneity in thymic carcinoma
              </a>
            </td>
          <td>
            Minoru Matsumoto, Yasuyo Saijo, Koichi Tsuneyama, T. Oya
          </td>
          <td>2024-12-01</td>
          <td>Mediastinum</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Kirsten rat sarcoma viral oncogene homolog (KRAS) is the most frequently mutated oncogene in lung adenocarcinoma, with G12C and G12V being the most predominant forms. Recent breakthroughs in KRASG12C inhibitors have transformed the clinical management of patients with the G12C mutation and advanced our understanding of the function of this mutation. However, little is known about the targeted disruption of KRASG12V, partly due to a lack of specific inhibitors. Here, we leverage the degradation tag (dTAG) system to develop a KRASG12V-transgenic mouse model. We explored the therapeutic potential of KRASG12V degradation and characterized its effect on the tumor microenvironment (TME). Our study reveals that degradation of KRASG12V abolished lung and pancreatic tumors in mice and caused a robust inhibition of KRAS-regulated cancer-intrinsic signaling. Importantly, targeted degradation of KRASG12V reprogrammed the TME toward a stimulatory milieu and drove antitumor immunity, elicited mainly by effector and cytotoxic CD8+ T cells. Our work provides insights into the effect of KRASG12V degradation on both tumor progression and the immune response, highlighting degraders as a powerful strategy for targeting KRAS-mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7557f24725afcb0b1e2df48764b1f6cd0b9ab54" target='_blank'>
              Targeted degradation of oncogenic KRASG12V triggers antitumor immunity in lung cancer models
              </a>
            </td>
          <td>
            Dezhi Li, Ke Geng, Yuan Hao, Jiajia Gu, Saurav Kumar, Annabel T Olson, Christina C. Kuismi, Hye Mi Kim, Yuanwang Pan, Fiona Sherman, Asia M. Williams, Yiting Li, Fei Li, Ting Chen, C. Thakurdin, Michela Ranieri, Mary Meynardie, Daniel S. Levin, Janaye Stephens, Alison Chafitz, Joy Chen, Mia S Donald-Paladino, Jaylen M Powell, Ze-Yan Zhang, Wei Chen, Magdalena Ploszaj, Han Han, S. Gu, Tinghu Zhang, Baoli Hu, B. Nacev, Medard Ernest Kaiza, Alice H. Berger, Xuerui Wang, Jing Li, Xuejiao Sun, Yang Liu, Xiaoyang Zhang, T. Bruno, N. Gray, Behnam Nabet, Kwok-Kin Wong, Hua Zhang
          </td>
          <td>2024-12-24</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Simple Summary Hybrid epithelial/mesenchymal (E/M) cells are the most metastatic of carcinoma cell types. Hybrid E/M cells have both epithelial and mesenchymal properties reminiscent of chimeras, mythical creatures composed of different animals. The co-expression of epithelial and mesenchymal proteins is a defining feature of hybrid E/M cancer cells. In this review, we discuss the use of cytokeratin (epithelial) and vimentin (mesenchymal) intermediate filaments as markers in hybrid E/M cells. We describe the structures and functions of these filaments that are uniquely expressed in hybrid E/M cells and speculate on potential hybrid structures formed between vimentin and cytokeratin. We also describe cell-surface cytokeratin and vimentin and their functions and discuss their potential as predictive biomarkers and as targets for anti-cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0a63915420ccb6800640641805de420f117dd3" target='_blank'>
              The Cancer Chimera: Impact of Vimentin and Cytokeratin Co-Expression in Hybrid Epithelial/Mesenchymal Cancer Cells on Tumor Plasticity and Metastasis
              </a>
            </td>
          <td>
            Nick A. Kuburich, Julia M. Kiselka, P. den Hollander, Andrew A. Karam, S. Mani
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Gliomas are aggressive tumors with a poor prognosis. The protocols presented here outline the methods used to study tumor progression, the tumor microenvironment (TME), and the effects of experimental treatments. The Sleeping Beauty (SB) transposase system induces tumors de novo to generate mouse models that recapitulate human gliomas. Plasmids are constructed with oncogenic drivers and other genetic alterations of interest. which are recognized by their unique position in between inverted/direct repeat (IR/DR) sequences. Luciferase enzyme is used to monitor the uptake of the plasmid, tumor growth, and response to experimental therapies. The genes of interest are tracked using fluorescent markers. Tumors will arise in immunocompetent hosts, which provides a relevant preclinical platform for analysis of tumor initiation, progression, survival, immune microenvironment, and histopathological features. Once the tumor grows within the desired brain location, it can be harvested to generate cell cultures of neurospheres for future experimentation. The benefit of implantable models generated from SB tumors is that they provide specific anatomical and genetic context, in which specific genetic characteristics can be tracked, as they are co-expressed with fluorescent markers. Post glioma cell implantation, additional analysis of the TME and tumor growth can be performed through immunohistochemistry (IHC) and flow cytometry. © 2025 Wiley Periodicals LLC. Basic Protocol 1: Creation of mouse glioma models by Sleeping-Beauty-mediated transposition Basic Protocol 2: Generation of orthotopic implantable brain tumors and neurospheres Basic Protocol 3: Hematoxylin and eosin staining of glioma tissue samples Basic Protocol 4: Immunohistochemistry of glioma tissue samples Basic Protocol 5: Flow cytometry for immune cell analysis of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/886ac84c05876a63108c36f36b7b7af78f353102" target='_blank'>
              Genetically Engineered and Implantable Mouse Brain Tumor Models: Characterization by Immunohistochemistry and Flow Cytometry.
              </a>
            </td>
          <td>
            Apoorva Mirji, Gurveer Singh, Anzar A. Mujeeb, Brandon L. McClellan, YingXiang Li, Makayla Perez, Maria G Castro
          </td>
          <td>2025-01-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa47d77b71a7337ad0ecaef200be807283fa4101" target='_blank'>
              Heterogeneity-driven phenotypic plasticity and treatment response in branched-organoid models of pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            A. Papargyriou, Mulham Najajreh, David P Cook, Carlo Maurer, S. Bärthel, H. Messal, Sakthi K Ravichandran, Till Richter, Moritz Knolle, Thomas Metzler, Akul R Shastri, R. Öllinger, Jacob Jasper, Laura Schmidleitner, Surui Wang, C. Schneeweis, H. Ishikawa-Ankerhold, T. Engleitner, Laura Mataite, Mariia Semina, Hussein Trabulssi, S. Lange, Aashreya Ravichandra, Maximilian Schuster, S. Mueller, K. Peschke, A. Schäfer, S. Dobiasch, S.E. Combs, Roland M. Schmid, Andreas R. Bausch, R. Braren, I. Heid, C. Scheel, Günter Schneider, Anja Zeigerer, Malte Luecken, K. Steiger, G. Kaissis, J. van Rheenen, F. Theis, Dieter Saur, R. Rad, Maximilian Reichert
          </td>
          <td>2024-12-10</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Despite recent therapeutic breakthroughs, cancer patients continue to face high recurrence and mortality rates due to treatment resistance. Cancer stem cells (CSCs), a subpopulation with self-renewal capabilities, are key drivers of refractive disease. This review explores the application of molecular imaging techniques, such as PET and SPECT, for the noninvasive detection of CSCs. By providing real-time monitoring of CSCs, these imaging methods have the potential to predict therapy resistance and guide personalized treatment approaches. Here, we cover the biological characteristics of CSCs, mechanisms of therapy resistance, and the identification and targeting of CSC-specific biomarkers with molecular imaging. Additionally, we address the challenges and opportunities for the clinical translation of CSC imaging, highlighting strategies where CSC imaging can be used to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c598784fb4bf91a45e00e3bbc55f45f0d4b538" target='_blank'>
              Molecular Imaging of Cancer Stem Cells and Their Role in Therapy Resistance
              </a>
            </td>
          <td>
            Sofia N Dos Santos, T. Witney
          </td>
          <td>2025-01-01</td>
          <td>Journal of Nuclear Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, Kevin Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88ba70c1f7c9e0c6817db8a69bc543b13dfed9df" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics of esophageal squamous cell carcinoma with lymph node metastases.
              </a>
            </td>
          <td>
            Wei Guo, Bolun Zhou, Lizhou Dou, Lei Guo, Yong Li, J. Qin, Zhen Wang, Qilin Huai, Xuemin Xue, Yin Li, Jianming Ying, Qi Xue, Shugeng Gao, Jie He
          </td>
          <td>2025-01-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Glioma evolution is governed by a multitude of dynamic interactions between tumor cells and heterogenous neighboring, non-cancerous cells. This complex ecosystem, termed the tumor microenvironment (TME), includes diverse immune cell types that have gained increasing attention for their critical and paradoxical roles in tumor control and tumorigenesis. Recent work has revealed that the cellular composition and functional state of immune cells in the TME can evolve extensively depending on the tumor stage and intrinsic features of surrounding glioma cells. Concurrently, adaptations to the glioma cellular phenotype, including activation of various cellular states, occur in the context of these immune cell alterations. In this review, we summarize important features of the immune TME that play key roles during each stage of glioma progression, from initiation to immune escape, invasion and recurrence. Understanding the complex interplay between tumor and immune cells is critical for the development of effective immunotherapies for glioma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6197e8609e0a64b363d0e798255e45455eaf7c8" target='_blank'>
              Co-evolution of glioma and immune microenvironment
              </a>
            </td>
          <td>
            Mahmoud M Elguindy, Jacob S Young, Winson S Ho, R. Lu
          </td>
          <td>2024-12-03</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Breast cancer heterogeneity has been well understood based solely on tumor components. However, the tumor microenvironment, which plays a crucial role in cancer by regulating tumor cells’ differentiation, maturation, and malignant potential, remains under-explored. This study conducted high-throughput RNA-Seq analysis on fresh breast cancer tissues from each molecular subtype and a pure population of cancer-associated fibroblasts (CAFs) isolated from cancer tissue and adjacent normal parts. Based on their functions, we identified three populations of CAFs: a) immune response-related, b) ECM remodeling, and c) calcium/protein binding. Validation of differentially expressed genes among these CAF populations revealed subtype-specific correlations. For instance, CAFs expressing immune response-related genes were significantly enriched in Luminal A/B and TNBC (log +1-fold; p =0.00047), compared to Her-2 positive (log -2.3-fold; p =0.0026). ECM remodeling genes exhibited greater expression in Her-2 positive and TNBC (log +4 to +11-fold; p =7.87E-08) when compared to Luminal subtypes (log +1.5-fold to +6-fold; p =8.66E-05). Calcium binding genes displayed overall similar expression across breast cancer subtypes. Thus, this study identified and underscored the functional heterogeneity of CAFs within the tumor microenvironment across the molecular subtypes of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e870b934402b9a2b10afe0f5e6b9ec8fa60b89" target='_blank'>
              Deciphering Functional Heterogeneity of Cancer-Associated Fibroblasts Across Molecular Subtypes of Breast Cancer
              </a>
            </td>
          <td>
            D. Kashyap
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) constitute the primary cellular component of the stroma in chordomas, characterized by an abundance of mucinous stromal elements, potentially facilitating their initiation and progression; however, this inference has yet to be fully confirmed. In this study, single-cell RNA sequencing (scRNA-seq), spatial transcriptomics (ST), bulk RNA-seq, multiplexed quantitative immunofluorescence (QIF), and in vivo and in vitro experiments were performed to determine the heterogeneity, spatial distribution, and clinical significance of CAFs in chordoma. ScRNA-seq was performed on 87,693 single cells derived from seven tumor samples and four control nucleus pulposus samples. A distinct CAF cluster distinguished by the upregulated expression of inflammatory genes and enriched functionality in activating inflammation-associated cells was identified. Pseudotime trajectory and cell communication analyses suggested that this inflammatory CAF (iCAF) subset originated from normal fibroblasts and interacted extensively with tumors and various other cell types. By integrating the scRNA-seq results with ST, the presence of iCAF in chordoma tissue was further confirmed, indicating their positioning at a distance from the tumor cells. Bulk RNA-seq data analysis from 126 patients revealed a correlation between iCAF signature scores, chordoma invasiveness, and poor prognosis. QIF validation involving an additional 116 patients found that although iCAFs were not in close proximity to tumor cells compared with other CAF subsets, their density correlated with malignant tumor phenotypes and adverse outcomes. In vivo and in vitro experiments further confirmed that iCAFs accelerate the malignant progression of chordomas. These findings could provide insights into the development of novel therapeutic strategies. © 2024 The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e399d12e04054b6320b0ebd4486fa4b22de699e6" target='_blank'>
              Multi-omics analysis unveils the role of inflammatory cancer-associated fibroblasts in chordoma progression.
              </a>
            </td>
          <td>
            Bo-Wen Zheng, Wei Guo
          </td>
          <td>2024-11-29</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Transgenic mice and organoid models, such as three-dimensional tumoroid cultures, have emerged as powerful tools for investigating cancer development and targeted therapies. Yet, the extent to which these preclinical models recapitulate the cellular identity of heterogeneous malignancies, like neuroblastoma (NB), remains to be validated. Here, we characterized the transcriptional landscape of TH-MYCN tumors by single-cell RNA sequencing (scRNA-seq) and developed ex vivo tumoroids. Integrated analysis with murine fetal adrenal samples confirmed that both TH-MYCN tumors and tumoroids closely mirror the cellular profiles of normal embryonic sympathoblasts and chromaffin cells. Comprehensive comparison between tumors from NB patients and TH-MYCN mice demonstrated similarities in adrenergic tumor cell composition. Ex vivo tumoroid cultures displayed histological resemblance and shared transcriptional profiles with the originating TH-MYCN tumors and human NB. Importantly, subpopulations within tumoroids exhibited gene expression associated with poor NB patient survival. Notably, recurrent observations of a low-proliferative chromaffin phenotype connected to the highly proliferative sympathetic phenotype suggested that pushing sympathoblasts into a chromaffin-like state may offer an interesting therapeutic strategy for NB. Together, this study not only deepens our understanding of a widely used transgenic mouse NB model but also introduces an ex vivo model that maintains critical adrenergic cell state identity, thereby enhancing its translational potential for NB research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5619b05ffd717eb9d8928b0e83ee66ca9cda8b" target='_blank'>
              Comparative Single-Cell Transcriptomics of Human Neuroblastoma and Preclinical Models Reveals Conservation of an Adrenergic Cell State.
              </a>
            </td>
          <td>
            Bethel Tesfai Embaie, Hirak Sarkar, A. Alchahin, Jörg Otte, T. Olsen, Conny Tümmler, Polina Kameneva, Artem V Artemov, N. Akkuratova, I. Adameyko, J. Stukenborg, Malin Wickström, P. Kogner, J. Johnsen, Shenglin Mei, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract Background Primary cardiac angiosarcoma (PCAS) is a rare and aggressive heart tumour with limited treatment options and a poor prognosis. Understanding cellular heterogeneity and tumour microenvironment (TME) is crucial for the development of effective therapies. Here, we investigated the intratumoural heterogeneity and TME diversity of PCAS using single‐cell RNA sequencing (scRNA‐seq). Methods We performed scRNA‐seq analysis on tumour samples from four patients with PCAS, supplemented with multicolour immunohistochemistry for identification. We used scRNA‐seq data from five normal cardiac tissue samples downloaded from public databases for comparative analyses. Bioinformatic analyses, including Cell Ranger, Seurat, Monocle2, hdWGCNA, SCENIC and NicheNet, were utilized to identify distinct cell populations, transcriptional patterns, and co‐regulating gene modules. Results Our analysis revealed significant intratumoural heterogeneity in PCAS driven by diverse biological processes such as protein synthesis, degradation, and RIG‐I signalling inhibition. The SCENIC analysis identified three primary transcription factors' clusters (CEBPB, MYC and TAL1). T‐cell subset analysis showed exhausted antigen‐specific T‐cells, complicating the efficacy of immune checkpoint blockade. Furthermore, we observed suppressive macrophages (SPP1+ and OLR1+) and reduced mitochondrial gene MT‐RNR2 (MTRNR2L12) expression in TME‐infiltrating cells, indicating impaired mitochondrial function. Conclusion This study elucidates the complex cellular landscape and immune microenvironment of PCAS, highlighting potential molecular targets for the development of novel therapies. These findings underscore the importance of a multifaceted therapeutic approach for addressing the challenges posed by PCAS's heterogeneity and immune evasion. Key points Insights into the heterogeneity and transcriptional patterns of sarcoma cells may explain the challenges in treating primary cardiac angiosarcoma (PCAS) using the current therapeutic modalities. Characterization of the immune microenvironment revealed significant immunosuppression mediated by specific myeloid cell populations (SPP1+ and OLR1+ macrophages). Identification of mitochondrial dysfunction in immune cells within the PCAS microenvironment, particularly the notable downregulation of the MTRNR2L12 protein, suggests a new avenue for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8e48a3df1b9f97d9505e0baebe97b568b07e258" target='_blank'>
              Investigating intra‐tumoural heterogeneity and microenvironment diversity in primary cardiac angiosarcoma through single‐cell RNA sequencing
              </a>
            </td>
          <td>
            Jingyuan Huo, Zhen Wang, Wenting Zhao, Miao Chen, Haoyang Li, Fengpu He, Xiao Tian, Yaqi Ma, Firyuza Husanova, Liang Ma, Yiming Ni, Hongda Ding, Weidong Li, Hongfei Xu
          </td>
          <td>2024-12-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly metastatic subtype of breast cancer. The epithelial-to-mesenchymal transition is a nonbinary process in the metastatic cascade that generates tumor cells with both epithelial and mesenchymal traits known as hybrid EM cells. Recent studies have elucidated the enhanced metastatic potential of cancers featuring the hybrid EM phenotype, highlighting the need to uncover molecular drivers and targetable vulnerabilities of the hybrid EM state. Here, we discovered that hybrid EM breast tumors are enriched in CD38, an immunosuppressive molecule associated with worse clinical outcomes in liquid malignancies. Altering CD38 expression in tumor cell impacted migratory, invasive, and metastatic capabilities of hybrid EM cells. Abrogation of CD38 expression stimulated an antitumor immune response, thereby preventing the generation of an immunosuppressive microenvironment in hybrid EM tumors. CD38 levels positively correlated with PD-L1 expression in samples from patients with TNBC. Moreover, targeting CD38 potentiated the activity of anti-PD-L1, eliciting strong antitumor immunity, with reduced tumor growth in hybrid EM models. Overall, this research exposes upregulation of CD38 as a specific survival strategy utilized by hybrid EM breast tumors to suppress immune cell activity and sustain metastasis, with strong implications in other carcinomas that have hybrid EM properties. Significance: Hybrid cells co-featuring epithelial and mesenchymal traits in triple-negative breast cancer express elevated levels of CD38 to induce immunosuppression and metastasis, indicating CD38 inhibition as potential strategy for treating breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4010598df27d14103f8e6a8fed15fa186e34fb5" target='_blank'>
              Accumulation of CD38 in Hybrid Epithelial/Mesenchymal Cells Promotes Immune Remodeling and Metastasis in Breast Cancer.
              </a>
            </td>
          <td>
            Tanvi H Visal, R. Bayraktar, P. den Hollander, Michael A Attathikhun, Tieling Zhou, Jing Wang, Li Shen, Corina Minciuna, Meng Chen, Elizve Barrientos-Toro, H. Batra, M. Gabriela Raso, Fei Yang, E. R. Parra, Aysegul Sahin, George A Calin, Sandurai A Mani
          </td>
          <td>2025-01-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background: Microvascular invasion (MVI) is a critical prognostic factor in intrahepatic cholangiocarcinoma (ICC), strongly associated with postoperative recurrence. However, the phenotypic features and spatial organization of MVI remain inadequately understood. Methods: We performed a spatial transcriptomic analysis on 29,632 spots from six ICC samples, manually delineating MVI clusters using the cloupe software. Key biomarkers were identified and validated in an independent cohort of 135 ICC patients. Functional and survival analyses were conducted to assess clinical relevance, and cell-cell communication pathways were investigated. Results: MVI regions exhibited heightened proliferation, angiogenesis, and epithelial-mesenchymal transition, driven by increased expression of transcription factors SOX10, ZEB1, and SNAI2. CXCL5 and SLC6A14 were identified as potential MVI biomarkers and showed high expression in tumor-invasive areas. Serum CXCL5 demonstrated strong predictive power for vascular invasion (AUC = 0.92) and intrahepatic metastasis (AUC = 0.96). High expression of both CXCL5 and SLC6A14 was associated with the worst survival outcomes. MVI regions were enriched with immunosuppressive MRC1+ macrophages and exhibited elevated immune checkpoint expression, including HAVCR2 and TIGHT, indicative of immune resistance. Cell-cell communication analysis revealed CXCL5-CXCR2 and LGALS9-HAVCR2 as key ligand-receptor pairs contributing to the immunosuppressive microenvironment. Conclusions: This study identifies CXCL5 and SLC6A14 as key biomarkers of MVI, highlighting their roles in tumor proliferation, immune resistance, and poor clinical outcomes. These findings provide valuable insights into the spatial organization of MVI and its contribution to ICC progression, offering potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80f1aeb6ba61c9f3ee11c5dafb6bfea9349b09d" target='_blank'>
              Spatial analyses revealed CXCL5 and SLC6A14 as the markers of microvascular invasion in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            G. Fan, L. Dai, Tongji Xie, Lin Li, Le Tang, Xiaohong Han, Yuankai Shi
          </td>
          <td>2024-12-11</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In 2024, the United States was projected to experience 2 million new cancer diagnoses and approximately 611,720 cancer-related deaths, reflecting a broader global trend in which cancer cases are anticipated to exceed 35 million by 2050. This increasing burden highlights ongoing challenges in cancer treatment despite significant advances that have reduced cancer mortality by 31% since 1991. Key obstacles include the disease’s inherent heterogeneity and complexity, such as treatment resistance, cancer stem cells, and the multifaceted tumor microenvironment (TME). The TME—comprising various tumor and immune cells, blood vessels, and biochemical factors—plays a crucial role in tumor growth and resistance to therapies. Recent innovations in cancer treatment, particularly in the field of immuno-oncology, have leveraged insights into TME interactions. An emerging example is the FDA-approved therapy using tumor-infiltrating lymphocytes (TILs), demonstrating the potential of cell-based approaches in solid tumors. However, TIL therapy is just one of many strategies being explored. This review provides a comprehensive overview of the emerging field of immuno-oncology, focusing on how novel therapies targeting or harnessing components of the TME could enhance treatment efficacy and address persistent challenges in cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69efe2b80b4ba6460d216813b5cdf5408b15b18" target='_blank'>
              Targeting Cancer: Microenvironment and Immunotherapy Innovations
              </a>
            </td>
          <td>
            Irena Padzińska-Pruszyńska, B. Taciak, Łukasz Kiraga, A. Smolarska, M. Górczak, P. Kucharzewska, M. Kubiak, J. Szeliga, Agata Matejuk, M. Król
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Melanoma is an aggressive malignancy defined by significant intratumoral heterogeneity, driving its capacity for therapeutic resistance and recurrence. This study adopts a systems-level approach to dissect the melanoma microenvironment, focusing on intricate interactions between malignant cells and immune infiltrates. We identified critical regulatory networks and intercellular communication pathways that appear to influence disease progression. These findings highlight the dynamic interplay between tumor-intrinsic factors and the surrounding microenvironment, revealing potential mechanisms underlying immune evasion and therapy resistance. By mapping these complex interactions, the present study builds on our foundation for precision-based therapeutic strategies tailored to the unique biological landscape of melanoma, offering promise for improved clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f383353b5d7dd45f21551edcc7dd626f8651941" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA Data Reveals Complexity and Significance of the Melanoma Interactome
              </a>
            </td>
          <td>
            Michael J Diaz, J. T. Tran, Arthur M. Samia, M. Forouzandeh, Jane M. Grant-Kels, Marjorie E. Montanez-Wiscovich
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaacb86999aeed003c707b4000adb56d67bb6547" target='_blank'>
              Spatial transcriptome profiling identifies DTX3L and BST2 as key biomarkers in esophageal squamous cell carcinoma tumorigenesis
              </a>
            </td>
          <td>
            Rutao Li, Na Li, Qianqian Yang, Xing Tong, Wei Wang, Chang Li, Jun Zhao, Dong Jiang, Haitao Huang, Chen Fang, Kai Xie, Jiamin Yuan, Shaomu Chen, Guangbin Li, Haitao Luo, Zhibo Gao, Dongfang Wu, Xiaoli Cui, Wei Jiang, Lingchuan Guo, Haitao Ma, Yu Feng
          </td>
          <td>2024-12-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cerebral cavernous malformations (CCMs) are clusters of thin-walled enlarged blood vessels in the central nervous system that are prone to recurrent hemorrhage and can occur in both sporadic and familial forms. The familial form results from loss-of-function variants in the CCM1, CCM2, or CCM3 gene. Despite a better understanding of CCM pathogenesis in recent years, it is still unclear why CCM3 mutations often lead to a more aggressive phenotype than CCM1 or CCM2 variants. By combining high-throughput differentiation of blood vessel organoids from human induced pluripotent stem cells (hiPSCs) with a CCM1, CCM2, or CCM3 knockout, single-cell RNA sequencing, and high-content imaging, we uncovered both shared and distinct functions of the CCM proteins. While there was a significant overlap of differentially expressed genes in fibroblasts across all three knockout conditions, inactivation of CCM1, CCM2, or CCM3 also led to specific gene expression patterns in neuronal, mesenchymal, and endothelial cell populations, respectively. Taking advantage of the different fluorescent labels of the hiPSCs, we could also visualize the abnormal expansion of CCM1 and CCM3 knockout cells when differentiated together with wild-type cells into mosaic blood vessel organoids. In contrast, CCM2 knockout cells showed even reduced proliferation. These observations may help to explain the less severe clinical course in individuals with a pathogenic variant in CCM2 and to decode the molecular and cellular heterogeneity in CCM disease. Finally, the ability to differentiate blood vessel organoids in a 96-well format will further facilitate their use in drug discovery and other biomedical research studies. STATEMENTS AND DECLARATIONS Conflicts of interest statement The authors declare no competing interests. The here described protocol for high-throughput organoid synthesis has been filed as a patent application at the European Patent Office (Process number: EP24213596.0) Author contribution statement MR, DSk, and UF designed the study. DSk, VS, LM, and RAP performed most of the functional experiments. SH and TA performed the CAM assays. SR performed the immunohistochemical stainings. SB, DSi, DSk, and VS performed the confocal microscopy and high-content imaging analyses. AE, CB, and EMB performed the scRNA sequencing analysis. AW and CAH performed and analyzed the karyotyping of the hiPSC clones. DSk, RAP, VS, KC, MR, and SB analyzed the data. DSk, VS, LM, and MR prepared figures. All authors contributed to the interpretation of the results. DSk, RAP, VS, and MR drafted the manuscript, and all authors contributed to writing. Ethics statement This study does not involve human participants or animal subjects. Availability of data and materials All relevant data are published within the paper and the supplementary files. ScRNA sequencing data can be accessed through the Gene Expression Omnibus (GEO) database (record number: GSE276497).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd15c9e6b5586c61e834020b3489f3f5a4ba9d5" target='_blank'>
              High-throughput differentiation of human blood vessel organoids reveals overlapping and distinct functions of the cerebral cavernous malformation proteins
              </a>
            </td>
          <td>
            Dariush Skowronek, Robin A. Pilz, Valeriia V. Saenko, Lara Mellinger, D. Singer, Silvia Ribback, Anja Weise, Kevin Claaßen, Christian Büttner, Emily M. Brockmann, Christian A. Hübner, Thiha Aung, S. Haerteis, S. Bekeschus, Arif B. Ekici, Ute Felbor, M. Rath
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The resistance of tumors to immune checkpoint inhibitors (ICI) may be intricately linked to cellular senescence, although definitive clinical validation remains elusive. In this study, comprehensive pan-cancer scRNA-seq analyses identify fibroblasts as exhibiting the most pronounced levels of cellular senescence among tumor-associated cell populations. To elucidate this phenomenon, a fibroblast senescence-associated transcriptomic signature (FSS), which correlated strongly with protumorigenic signaling pathways and immune dysregulation that fosters tumor progression, is developed. Leveraging the FSS, the machine learning (ML) framework demonstrates exceptional accuracy in predicting ICI response and survival outcomes, achieving superior area under curve (AUC) values across validation, testing, and in-house cohorts. Strikingly, FSS consistently outperforms established signatures in predictive robustness across diverse cancer subtypes. From an integrative analysis of 17 CRISPR/Cas9 libraries, CDC6 emerges as a pivotal biomarker for pan-cancer ICI response and prognostic stratification. Mechanistically, experimental evidence reveals that CDC6 in tumor cells orchestrates fibroblast senescence via TGF-β1 secretion and oxidative stress, subsequently reprogramming the tumor microenvironment and modulating ICI response. These findings underscore the translational potential of targeting fibroblast senescence as a novel therapeutic strategy to mitigate immune resistance and enhance antitumor efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83eb29659bd661ae208e4642398f8b3b7069e404" target='_blank'>
              A Distinguished Roadmap of Fibroblast Senescence in Predicting Immunotherapy Response and Prognosis Across Human Cancers.
              </a>
            </td>
          <td>
            Dongjie Chen, Pengyi Liu, Jiayu Lin, Longjun Zang, Yihao Liu, Shuyu Zhai, Xiongxiong Lu, Yuanchi Weng, Hongzhe Li
          </td>
          <td>2024-12-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Anaplastic Thyroid Cancer (ATC) is an aggressive form of cancer with poor prognosis, heavily influenced by its tumor immune microenvironment (TIME). Understanding the cellular and gene expression dynamics within the TIME is crucial for developing targeted therapies. This study analyzes the immune microenvironment of ATC and Papillary Thyroid Cancer (PTC) using single-cell RNA sequencing (scRNA-seq). We performed a comprehensive scRNA-seq analysis on ATC and PTC samples, incorporating cell type annotation, marker gene identification, and clustering based on gene expression. A specific focus was on the prevalence and biomarkers of Pre-Exhausted CD8+ T cells in ATC, utilizing the single-cell tumor immune atlas for immune cell characterization. The scRNA-seq analysis identified distinct immune cell populations and differentially expressed genes in ATC and PTC samples. Notably, Pre-Exhausted CD8+ T cells were found to be prevalent in ATC datasets. Additional immunofluorescence staining and co-culture experiments with the ATC cell line identified GNLY, a member of the saposin-like protein family as a potential biomarker for Pre-Exhausted CD8+ T cells in ATC. This study provides valuable insights into the immune landscape of ATC, emphasizing the prevalence of Pre-Exhausted CD8+ T cells and identifying GNLY as a potential biomarker. Understanding the TIME composition and the role of specific immune cells in cancer progression can inform the development of targeted immunotherapies for ATC. Future research should explore the functional implications of GNLY and other identified biomarkers in modulating the immune response in thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bee3b0c483d956dde8850d164e4f812a4b30008" target='_blank'>
              Persistent pre-exhausted CD8+ T cells shape the tumor immune microenvironment in anaplastic thyroid cancer.
              </a>
            </td>
          <td>
            X. Ruan, Mei Tao, Yuanxing Dong, Linfei Hu, Guangwei Xu, Chuanxiang Hu, Yue Huang, Yuqi Wang, Jialong Yu, Wei Luo, Ming Gao, Min Zhao, Xiangqian Zheng
          </td>
          <td>2025-01-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Single-cell omics approaches profile molecular constituents of individual cells. Replicated multi-condition experiments in particular aim at studying how the molecular makeup and composition of cell subpopulations changes at the sample-level. Two main approaches have been proposed for these tasks: firstly, cluster-based methods that group cells into (non-overlapping) subpopulations based on their molecular profiles and, secondly, cluster-free but neighborhood-based methods that identify (overlapping) groups of cells in consideration of cross-condition changes. In either approach, discrete cell groups are subjected to differential testing across conditions; and, a low-dimensional cell embedding, which is in turn derived from a subset of selected features, is required to delineate subpopulations or neighborhoods. We hypothesized that decoupling differences in cell type (i.e., between subpopulations) and cell state (i.e., between conditions) for feature selection would yield an embedding space that captures different aspects of cellular heterogeneity. And, that type-not-state embeddings would arrive at differential testing results that are comparable between clusterand neighborhood-based differential testing approaches. Our study leverages a simulation framework with competing type and state effects, as well as an experimental dataset, to evaluate a set of feature scoring and selection strategies, and to compare results from downstream differential analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32208ce9dfad15375b406ef8e29e742700235e50" target='_blank'>
              Disentangling cell type and state transcriptional programs
              </a>
            </td>
          <td>
            Jiayi Wang, Helena L. Crowell, Mark D. Robinson
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is the most common and aggressive type of primary brain cancer, known for its rapid growth, invasion into surrounding brain tissue, and strong resistance to current therapies. Despite intensive treatment efforts, patients have a median survival of just 15 months. One major treatment challenge is the tumor’s complex vascular network which causes the limited success of antiangiogenic therapies in extending patients’ survival. Pericytes are cells essential for maintaining the integrity of this vascular network. Additional to their structural functions, they play diverse roles that support tumor progression and treatment resistance. This review aims to understand the multifaced functions of pericytes in the glioblastoma microenvironment; the comprehension of those roles may guide new therapeutic strategies to improve outcomes for patients affected with this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/835c71d63c5567604bb678c8f9634fc408771476" target='_blank'>
              Pericytes in Glioblastoma: Hidden Regulators of Tumor Vasculature and Therapy Resistance
              </a>
            </td>
          <td>
            Irene Salazar-Saura, María Pinilla-Sala, J. Megías, Lara Navarro, Esther Roselló-Sastre, T. San-Miguel
          </td>
          <td>2024-12-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Aneuploidy, the major cause of miscarriages in humans, is pervasive in early embryos but robustly dampened during development, allowing for healthy births. Later in life, aneuploidy correlates with pathological conditions including cancer. Identification of the mechanisms underlying the elimination of aneuploid cells is relevant in development and disease. Here we generated cells carrying molecularly defined segmental monosomies and trisomies and characterized their immediate impact on cellular behavior. Our data reveal signs of out-competition of cells carrying monosomies in genomic regions devoid of previously known haploinsufficient genes. Dose-dependent effects of single genes or a discrete number of genes contribute to the observed cellular behaviors. By simultaneously inducing cells carrying monosomies and trisomies of the same genomic location, we present evidence that segmental trisomies potentiate or alleviate the negative effects of the monosomy on growth, thus revealing a key role of cell interactions in defining the in vivo elimination of aneuploid cells. Highlights Segmental monosomies cause growth impairment and are out-competed. Growth impairment relies on dose-dependent effects of single genes or a discrete number of genes. Segmental trisomies of up to 1500 genes do not have a major impact on proliferation and survival. Cell competition is modulated by the presence of cells trisomic for the same genomic region. eTOC Blurb Aneuploidy has a negative impact on the growth and proliferation of all animal cells analyzed so far. Fusari et al. unravel a role of cell interactions in defining the in vivo elimination of aneuploid cells through cell competition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc128d47b336584a75037f0f1e1bbbd450a62027" target='_blank'>
              Depletion of aneuploid cells is shaped by cell-to-cell interactions
              </a>
            </td>
          <td>
            Elena Fusari, M. Muzzopappa, Juliette Gracia, Marco Milán
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Despite the vast diversity of life forms and living histories, it appears that all branches of Metazoa face the challenge of tumor growth. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we first identified the rise of a glycolytic cell population over time, resembling the Wartburg effect observed in human tumors. We further identified two key metabolic changes in these fly tumors. First, a systemic depletion of acetyl-CoA leads to a reduction in histone acetylation levels and stochastic silencing of actively-transcribed genes. Second, defects in the methionine cycle cause a systemic depletion of S-Adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process strongly inhibits tumor growth. Finally, to understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors, and identified human tumors that exhibit metabolic signatures similar to those of fly tumors. We found that human tumors with high metabolic similarity to fly tumors have a lower mutational load, younger patient age, and lower DNA methylation levels. This study suggests that tumorigenesis processes have a deep evolutionary origin and highlights that depletion of key metabolites is an evolutionarily-ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe6bf208acb7d71c056481ed083b94ace1ace601" target='_blank'>
              Epigenetic reprogramming induced by Acetyl-CoA and SAM depletion is an evolutionarily-ancient path to malignant growth
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58121f921b0bd9812f510ea0edf4c0cb4c87aa52" target='_blank'>
              Spatial transcriptomic analysis drives PET imaging of tight junction protein expression in pancreatic cancer theranostics
              </a>
            </td>
          <td>
            James Wang, J. Seo, A. Kare, Martin Schneider, Mallesh Pandrala, Spencer K. Tumbale, M. Raie, Gokce Engudar, Nisi Zhang, Yutong Guo, Xiaoxu Zhong, Sofia Ferreira, Bo Wu, Laura D Attardi, G. Pratx, Andrei Iagaru, Ryan L. Brunsing, Gregory W Charville, Walter G. Park, Katherine W Ferrara
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background: Circulating tumor cells (CTCs) and CTC-clusters are pivotal in the metastatic process of breast cancer (BC). Owing to their low frequency, models replicating their biology should provide a robust platform for investigating the molecular mechanisms driving metastasis and identifying new biomarkers. We established and characterized a CTC-derived cell model from a mouse xenograft to explore its metastatic behavior and molecular profile, which allowed us to investigate the expression and prognostic significance of a set of genes associated with the metastatic potential of CTCs. Methods: The CTC line (mCTC) derived from a MDA-MB-231 mouse xenografts was used in comparative functional analyses including cell cycle evaluation, colony formation, invasion, adhesion, and metastatic competency in zebrafish models. Transcriptomic profiling and functional assays were conducted to identify candidate genes and understand their roles in metastasis. Moreover, publicly available gene expression datasets of CTCs, CTC-clusters, and tumor tissue, from GEO and TCGA, were analyzed for the identification of a gene signature that was correlated with survival data. The signature was validated in an independent cohort. Results: Compared with MDA-MB-231 cells, mCTC cells presented enhanced colony formation, invasion, and adhesion, and increased dissemination and survival in zebrafish. Transcriptomic analysis revealed that SPARC was significantly upregulated. Functional assays showed that SPARC overexpression was correlated with increased invasion and migration. Analysis of public datasets confirmed the high expression of SPARC in BC CTCs and CTC-clusters. Additionally, a 4-gene signature involving SPARC, THBS1, VCL, and HSP90AB1 was identified that demonstrated strong prognostic value, predicting shorter overall and distant metastasis-free survival in the primary tumor setting. Validation cohorts confirmed its ability to distinguish high-risk patients. Elevated expression of the 4-gene signature in CTCs was also indicative of increased mortality risk. Conclusion: mCTC exhibit distinct metastatic traits and molecular characteristics, highlighting a possible role of SPARC in CTC biology and its potential as a prognostic marker in BC metastasis. The identified 4-gene signature provides a robust prognostic tool for assessing patient risk and guiding therapeutic strategies. Further investigations into the mechanistic role of SPARC may reveal new therapeutic targets for managing BC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785682c178d882b31b862712e1108b5022d056bd" target='_blank'>
              A CTC Model Uncovers Metastatic Drivers and Prognostic Markers in Breast Cancer
              </a>
            </td>
          <td>
            C. Fernández-Santiago, I. Martínez-Peña, M. Paramés, M. Rodriguez, P. Hurtado, C. Abuín, C. Costa, A. B. Dávila Ibáñez, R. López-López, R. Pineiro
          </td>
          <td>2025-01-17</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286075cb8f68cf0d587bedcc36d60116c411594e" target='_blank'>
              Spatially resolved gene expression profiling of tumor microenvironment reveals key steps of lung adenocarcinoma development
              </a>
            </td>
          <td>
            Yuma Takano, Jun Suzuki, Kotaro Nomura, Gento Fujii, Junko Zenkoh, Hitomi Kawai, Yuta Kuze, Y. Kashima, Satoi Nagasawa, Yuka Nakamura, M. Kojima, K. Tsuchihara, Masahide Seki, A. Kanai, D. Matsubara, Takashi Kohno, Masayuki Noguchi, Akihiro Nakaya, Masahiro Tsuboi, G. Ishii, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-12-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Cancer immunotherapy is increasingly used in clinical practice, but its success rate is reduced by tumor escape from the immune system. This may be due to the genetic instability of tumor cells, which allows them to adapt to the immune response and leads to intratumoral immune heterogeneity. The study investigated spatial immune heterogeneity in the tumor microenvironment and its possible drivers in a mouse model of tumors induced by human papillomaviruses (HPV) following immunotherapy. Gene expression was determined by RNA sequencing and mutations by whole exome sequencing. A comparison of different tumor areas revealed heterogeneity in immune cell infiltration, gene expression, and mutation composition. While the mean numbers of mutations with every impact on gene expression or protein function were comparable in treated and control tumors, mutations with high or moderate impact were increased after immunotherapy. The genes mutated in treated tumors were significantly enriched in genes associated with ECM metabolism, degradation, and interactions, HPV infection and carcinogenesis, and immune processes such as antigen processing and presentation, Toll-like receptor signaling, and cytokine production. Gene expression analysis of DNA damage and repair factors revealed that immunotherapy upregulated Apobec1 and Apobec3 genes and downregulated genes related to homologous recombination and translesion synthesis. In conclusion, this study describes the intratumoral immune heterogeneity, that could lead to tumor immune escape, and suggests the potential mechanisms involved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10b06a4b14e37ec85c1d027d3a7d9fbdaf7aa59e" target='_blank'>
              Spatial immune heterogeneity in a mouse tumor model after immunotherapy.
              </a>
            </td>
          <td>
            Michal Šmahel, S. Johari, J. Smahelova, L. Pfeiferová, Jaroslav Nunvar
          </td>
          <td>2024-12-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/602ca4326619720e872bb3125e86fd5844dc2d44" target='_blank'>
              A multiscale model of immune surveillance in micrometastases gives insights on cancer patient digital twins
              </a>
            </td>
          <td>
            Heber L Rocha, Boris Aguilar, Michael Getz, I. Shmulevich, P. Macklin
          </td>
          <td>2024-12-04</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>2</td>
          <td>83</td>
        </tr>

        <tr id="Immune checkpoint blockade (ICB) has revolutionized the current immuno-oncology and significantly improved clinical outcome for cancer treatment. Despite the advancement in clinics, only a small subset of patients derives immune response to the ICB therapy. Therefore, a robust predictive biomarker that identifies potential candidate becomes increasingly crucial in delivering this technology to the public. In this review, we first discuss the biomarkers that focus on tumor genome, tumor microenvironment and tumor-host interaction. Then, we compare existing databases for biomarker discovery for ICB response. We also present IOhub - an interactive web portal that incorporates 36 bulk and 10 single-cell transcriptome datasets for benchmark analysis of the current biomarkers. Finally, we highlight the trending interest in antibody drug conjugate and combination treatment and their use in precision immuno-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344de8d9818f0d313e9f50e4d12aa92a03af95b7" target='_blank'>
              Current transcriptome database and biomarker discovery for immunotherapy by immune checkpoint blockade
              </a>
            </td>
          <td>
            Han Zhang, Binfeng Lu, Xinghua Lu, A. Saeed, Lujia Chen
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Our research aims to understand the adaptive, ergo potentially metastatic, responses of prostate cancer to changing microenvironments. Emerging evidence implicates a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study revealed that PACC presence in patient prostate tumors at the time of radical prostatectomy was predictive of future metastasis. To test for a causative relationship between PACC state biology and metastasis in prostate cancer, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) from animal models. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Specifically, we applied this approach to analysis of subcutaneous, caudal artery, and intracardiac murine models. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. Furthermore, in vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites. Additional in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence of pro-metastatic mechanisms specific to the PACC state. Implications: Considering that many anti-cancer agents induce the PACC state, our data positions the increased metastatic competency of PACC state cells as an important unforeseen ramification of neoadjuvant regimens, which may help explain clinical correlations between chemotherapy and metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c130a571d99ddcd76ddd1df86bc68f827f48ad3" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic.
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T A Rolle, Michael J Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, S. Amend
          </td>
          <td>2024-12-10</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive subtype of breast cancer, with the worst prognosis among all subtypes. The impact of distinct cell subpopulations within the tumor microenvironment (TME) on TNBC patient prognosis has yet to be clarified. Methods Utilizing single-cell RNA sequencing (scRNA-seq) integrated with bulk RNA sequencing (bulk RNA-seq), we applied Cox regression models to compute hazard ratios, and cross-validated prognostic scoring using a GLMNET-based Cox model. Cell communication analysis was used to elucidate the potential mechanisms of CPVL and MSR1. Ultimately, RNA interference-mediated gene knockdown was utilized to validate the impact of specific genes on the polarization of tumor-associated macrophages (TAMs). Results Our findings revealed that the function of immune cells is more pivotal in prognosis, with TAMs showing the strongest correlation with TNBC patient outcomes, compared with other immune cells. Additionally, we identified CPVL and MSR1 as critical prognostic genes within TAMs, with CPVL expression positively correlated with favorable outcomes and MSR1 expression associated with poorer prognosis. Mechanistically, CPVL may contribute to favorable prognosis by inhibiting the SPP1-CD44 ligand-receptor and promoting CXCL9-CXCR3, C3-C3AR1 ligand-receptor, through which TAMs interact with other cells such as monocytes, neutrophils, and T cells. Moreover, cytokines including IL-18, IFNγR1, CCL20, and CCL2, along with complement-related gene like TREM2 and complement component CFD, may participate in the process of CPVL or MSR1 regulating macrophage polarization. Furthermore, RT-PCR experiments confirmed that CPVL is positively associated with M1-like TAM polarization, while MSR1 is linked to M2-like TAM polarization. Finally, the prognostic significance of these two genes is also validated in HER2-positive breast cancer subtypes. Conclusions CPVL and MSR1 are potential biomarkers for macrophage-mediated TNBC prognosis, suggesting the therapeutic potential of macrophage targeting in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af61ce193245e3b400806d4fcea4eed6dfe425f5" target='_blank'>
              Single-cell Atlas reveals core function of CPVL/MSR1 expressing macrophages in the prognosis of triple-negative breast cancer
              </a>
            </td>
          <td>
            Xinan Wang, Li Lin, Xue Zhang, Minghui Zhang, Zhuo Sun, Yichen Yang, Xiuna Zhang, Yonghui Yuan, Yong Zhang, Hao Chen, Ti Wen
          </td>
          <td>2024-12-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer deaths in the United States. New targeted therapies against the once-deemed undruggable oncogenic KRAS are changing current therapeutic paradigms. However, resistance to targeted KRAS inhibitors almost inevitably occurs; resistance can be driven by tumor cell–intrinsic changes or by changes in the microenvironment. Here, we utilized a genetically engineered mouse model of KRASG12D-driven lung cancer that allows for inducible and reversible expression of the oncogene: activation of oncogenic KRASG12D induces tumor growth; conversely, inactivation of KRASG12D causes tumor regression. We showed that in addition to regulating cancer cell growth and survival, oncogenic KRAS regulated the transcriptional status of cancer-associated fibroblasts and macrophages in this model. Utilizing ex vivo approaches, we showed that secreted factors from cancer cells induced the expression of multiple cytokines in lung fibroblasts, and in turn drove expression of immunosuppressive factors, such as arginase 1, in macrophages. In summary, fibroblasts emerged as a key source of immune regulatory signals, and a potential therapeutic target for improving the efficacy of KRAS inhibitors in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdae42d5132ec7ce05cdb9d5dd6bd5ef386a7a36" target='_blank'>
              KRASG12D drives immunosuppression in lung adenocarcinoma through paracrine signaling
              </a>
            </td>
          <td>
            Emily L. Lasse-Opsahl, Ivana Barravecchia, Elyse McLintock, Jen-Mei Lee, Sarah F Ferris, Carlos E Espinoza, Rachael Hinshaw, Sophia Cavanaugh, Marzia Robotti, Lily Rober, Kristee L Brown, Kristena Y. Abdelmalak, C. Galbán, Timothy L. Frankel, Yaqing Zhang, M. Pasca di Magliano, Stefanie Galbán
          </td>
          <td>2025-01-09</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72ace1bf5ccc7bc1a835b1fd333db7f3f67e0b96" target='_blank'>
              Lessons learned from spatial transcriptomic analyses in clear-cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Jesper Jespersen, Cecilie Lindgaard, Laura Iisager, J. Ahrenfeldt, I. Lyskjær
          </td>
          <td>2025-01-09</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) have an overall poor prognosis with many high-risk cases co-opting stem cell gene regulatory programs, yet the mechanisms through which this occurs remain poorly understood. Increased expression of the stem cell transcription factor, MECOM, underlies one key driver mechanism in largely incurable AMLs. How MECOM results in such aggressive AML phenotypes remains unknown. To address existing experimental limitations, we engineered and applied targeted protein degradation with functional genomic readouts to demonstrate that MECOM promotes malignant stem cell-like states by directly repressing pro-differentiation gene regulatory programs. Remarkably and unexpectedly, a single node in this network, a MECOM-bound cis-regulatory element located 42 kb downstream of the myeloid differentiation regulator CEBPA, is both necessary and sufficient for maintaining MECOM-driven leukemias. Importantly, targeted activation of this regulatory element promotes differentiation of these aggressive AMLs and reduces leukemia burden in vivo, suggesting a broadly applicable differentiation-based approach for improving therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c42a2a7df7281b8834a5ccf4f4a2d74201a60" target='_blank'>
              CEBPA repression by MECOM blocks differentiation to drive aggressive leukemias
              </a>
            </td>
          <td>
            Travis J. Fleming, Mateusz Antoszewski, Sander Lambo, Michael Gundry, Riccardo Piussi, Lara Wahlster, Sanjana Shah, Fiona E. Reed, Kevin Dong, Joao A. Paulo, Steve P Gygi, C. Mimoso, Seth R Goldman, Karen Adelman, Jennifer A. Perry, Y. Pikman, K. Stegmaier, Maria N. Barrachina, Kellie R. Machlus, Volker Hovestadt, A. Arruda, Mark D. Minden, Richard A. Voit, V. G. Sankaran
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Mouse models that faithfully represent the biology of human brain tumors are critical tools for unraveling the underlying tumor biology and screening for potential precision therapies. This is especially true of rare tumor types, many of which have correspondingly few xenograft or cell lines available. Although our understanding of the specific biological pathways driving cancer has improved significantly, identifying the appropriate progenitor populations to drive oncogenic processes represents a significant barrier to efficient mouse model production. In this issue of Cancer Research, Jessa and colleagues developed an innovative transcription factor fingerprinting method to map the cellular origin of central nervous system neuroblastoma, FOXR2-activated to medial ganglionic eminence-derived interneurons, which could then be efficiently targeted in the developing mouse brain using in utero electroporation. This approach serves as a blueprint for investigating other rare pediatric brain tumors, potentially accelerating progress toward the development of mouse models and identification of effective therapies. See related article by Jessa et al., p. 231.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b36b1d9ff773da8434bf45b0c13f61a915472b5f" target='_blank'>
              Transcription Factor Fingerprint Provides Clues for Brain Tumor Cell of Origin.
              </a>
            </td>
          <td>
            Brent A. Orr
          </td>
          <td>2024-01-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The growth and progression of a neoplasm is a multi–stage process mediated by interactions between tumor cells and their surrounding stroma including various cell types (fibroblasts, endothelial cells, immune cells), cell-free components (cytokines, extracellular vesicles and extracellular matrix), blood and lymphatic vessels that form the tumor microenvironment. During the progression of neoplasms, tumor cells can evade the immune response by suppressing or losing tumor antigens releasing immunosuppressive extracellular vesicles and exosomes, secreting various cytokines, chemokines and other factors, and begin to reprogram and remodel the surrounding stroma. The most relevant areas of TME study are cells and molecules of innate and acquired immunity exhibited both suppressive and stimulating activity against tumor cells due to their high heterogeneity and plasticity. Special attention is paid to immune control points, tumor microbiota, components of intercellular interaction as well as immunotherapy of tumor diseases.
In this paper, a systematic review of scientific publications in PubMed database devoted to the study of TME and published in the period from 2019 to 2024 was carried out. 2547 sources were studied and 68 articles were selected during the screening process. Search queries and keywords included «tumor microenvironment», «immune cells», «tumor», «oncogenesis», «metastasis». The formed group of scientific articles will allow to study in more detail the cellular and molecular
composition of the tumor microenvironment, as well as the main targets for the development of new approaches for malignant neoplasms diagnosis and therapy involving TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/568399709b478d3282ff1a127c4698c1e28c9300" target='_blank'>
              Tumor microenvironment: systematic review
              </a>
            </td>
          <td>
            A. V. Vialichka, D. B. Nizheharodava, Elizaveta M. Nazaranka, Bogdan A. Muzychenka, M. Zafranskaya
          </td>
          <td>2024-12-27</td>
          <td>Journal of the Belarusian State University. Ecology.</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="STUDY QUESTION Can network modelling of single cell transcriptomic data identify cellular developmental trajectories of fallopian tube (FT) epithelium and reveal functional and pathological divergence from the endometrium? SUMMARY ANSWER A bidirectional secretory and ciliated differentiation trajectory was apparent from a novel OVGP1+ progenitor population of FT epithelial cells. A causal network model of whole transcriptome action in the FT and endometrium revealed specific functional divergence between secretory cells of these tissues. The network model reflected the latest ectopic pregnancy genome wide association study (GWAS), invoking MUC1 and other candidate genes in mature secretory cells for ectopic and eutopic implantation. WHAT IS KNOWN ALREADY The fallopian tube forms the in vivo peri-conceptual environment, which has a significant impact on programming offspring health. The fallopian tube epithelium establishes this environment, however the epithelial cell types are poorly characterised in health and disease. STUDY DESIGN, SIZE, DURATION Publicly available benign FT single cell RNA sequencing (scRNA-seq) samples from thirteen women across three studies were combined. Endometrial scRNA-seq samples from thirteen women from one study were used to demonstrate transcriptomic differences between the epithelia of the two tissues. Network models of transcriptomic action were constructed with hypergraphs. PARTICIPANTS/MATERIALS, SETTING, METHODS A meta-analysis of FT scRNA-seq samples was performed to identify epithelial populations. Differential gene expression assessed differences between fallopian tube and endometrial epithelial scRNA-seq data. Functional differences between secretory cells in the tissues were characterised using hypergraph models. To identify associations with ectopic pregnancy, expression quantitative trait loci (eQTLs) from a recent GWAS were mapped onto the network models. MAIN RESULTS AND THE ROLE OF CHANCE Epithelial cells (n=14,360) were clustered into 8 secretory and ciliated epithelial populations in the meta-analysis of 3 scRNA-seq datasets. A novel OVGP1+ epithelial progenitor cell was also identified, and its bi-directional differentiation to mature secretory or mature ciliated populations was mapped by RNA velocity analysis. This progenitor exhibited a high velocity magnitude (12.47) and low confidence (0.69), a combination strongly indicative of multipotent progenitor status. Comparing FT epithelial cells with endometrial epithelial cells revealed 5.3-fold fewer shared genes between FT and endometrial glandular secretory cells than between FT and endometrial ciliated cells, suggesting functional divergence of secretory cells along the reproductive tract. Hypergraphs were used to identify highly coordinated regions of the transcriptome robustly associated with functional gene networks. In the FT secretory cells, these networks were enriched for lipid (FDR<0.002) and immune (FDR<0.00007) related pathways. We mapped eQTLs from a GWAS meta-analysis of 7070 women with ectopic pregnancy over a range of significance (P = 1.68 x 10-21– 5.8 x 10-4) to the hypergraphs of FT and endometrium. Of the 22 genes present in the hypergraphs, 13 of these clustered as highly coordinated genes. This demonstrated the functional importance of MUC1 in the FT and endometrium, (GWAS Study P = 5.32×10-9) and identified additional genes (SLC7A2, CLDN1, GLS, PEX6, PLXNA4, NR2F1, CLGN, PGGHG, ANKRD36) implicated in ectopic pregnancy and eutopic pregnancy. LIMITATIONS, REASONS FOR CAUTION The sample size of reproductive age women was limited in previous studies, and though causal network modelling was used and previous mechanistic data supports candidate gene involvement, no in vitro or in vivo validation of candidate was performed. WIDER IMPLICATIONS OF THE FINDINGS These findings consolidate the existing single cell transcriptomic datasets of the FT to provide a comprehensive understanding of epithelial populations and define functionally distinct secretory cells that contribute to the peri-conceptual environment of the FT. We further implicate the role of MUC1 and secretory cells in ectopic pregnancy and suggest future targets for investigating embryo implantation in the FT and endometrium.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db0759728199513a24cfe58260fd0439ec7a104e" target='_blank'>
              Single Cell Transcriptomic Modelling of the Fallopian Tube Epithelium Identifies Cellular Specialisation, Novel Differentiation Trajectories, and Gene Network Associations with Ectopic Pregnancy
              </a>
            </td>
          <td>
            Lily I Wright, Ivan Wangsaputra, Terence Garner, M. Sharps, Roger G. Sturmey, Peter T Ruane, A. Stevens
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35c843d34d2321464470845817a613dfc68c436a" target='_blank'>
              Integrin α8 is a useful cell surface marker of alveolar lipofibroblasts
              </a>
            </td>
          <td>
            A. Fukada, Yasunori Enomoto, R. Horiguchi, Y. Aoshima, S. Meguro, H. Kawasaki, I. Kosugi, T. Fujisawa, N. Enomoto, Naoki Inui, T. Suda, T. Iwashita
          </td>
          <td>2025-01-13</td>
          <td>Respiratory Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Adrenocortical cancer (ACC) is a rare malignant neoplasm originating from the adrenal cortex, presenting limited therapeutic options. An avenue for improving therapeutic efficacy may involve a deeper understanding of the role of adrenocortical stem/progenitor cells in the pathogenesis of this disease. Although existing data suggest stem/progenitor characteristics in certain cell populations within ACC, the challenge remains to identify adrenocortical stem cell markers directly involved in its carcinogenesis. In our study, we aimed to identify multipotency markers such as LGR5 and CD90 in various ACC types to confirm their presence and localization. The study included tumor tissue samples from 13 patients with ACC treated at the Endocrinology Research Centre (Moscow, Russia) between 2005 and 2023. We conducted immunohistochemical analyses to identify the aforementioned markers and examined the association between their expression and clinico-morphological parameters. Our pilot study results demonstrate the presence of LGR5- and CD90-positive tumor cells in all samples. Despite the small sample size, we observed statistically significant differences in disease-free survival based on the number of CD90-positive cells. These findings suggest a potential diagnostic, prognostic, and predictive value of cancer stem cell markers, underscoring the need for their further analysis in a larger cohort of patients with ACC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47a88e4cdd4cc0eadd5984efc94aa478394597ed" target='_blank'>
              Analysis of Cancer Stem Cell Markers in Various Histological Subtypes of Adrenocortical Cancer
              </a>
            </td>
          <td>
            N. Pachuashvili, Asya Bastrich, E. Porubayeva, Alina Elfimova, A. Tertychnyy, Dmitry Beltsevich, Evgeniya Kogan, Igor Reshetov, Ekaterina Troshina, Natalia Tarbaeva, Natalia Mokrysheva, L. Urusova
          </td>
          <td>2024-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Interrogating regulatory epigenetic alterations during tumor progression at the resolution of single cells has remained an understudied area of research. Here we developed a highly sensitive single-nucleus CUT&RUN (snCUT&RUN) assay to profile histone modifications in isogenic primary, metastatic, and cisplatin-resistant head and neck squamous cell carcinoma (HNSCC) patient-derived tumor cell lines. We find that the epigenome can be involved in diverse modes to contribute towards HNSCC progression. First, we demonstrate that gene expression changes during HNSCC progression can be comodulated by alterations in both copy number and chromatin activity, driving epigenetic rewiring of cell states. Furthermore, intratumour epigenetic heterogeneity (ITeH) may predispose subclonal populations within the primary tumour to adapt to selective pressures and foster the acquisition of malignant characteristics. In conclusion, snCUT&RUN serves as a valuable addition to the existing toolkit of single-cell epigenomic assays and can be used to dissect the functionality of the epigenome during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ccec03801dafc8df15febd551c3a9189ef4f702" target='_blank'>
              Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression.
              </a>
            </td>
          <td>
            H. Womersley, Daniel Muliaditan, Ramanuj DasGupta, L. F. Cheow
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSCs) tend to acquire genetic aberrations upon culture in vitro. Common aberrations are mutations in the tumor suppressor TP53, suspected to confer a growth-advantage to the mutant cells. However, their full impact in the development of malignant features and safety of hPSCs for downstream applications is yet to be elucidated. Here, TP53 is knocked out in hPSCs using CRISPR-Cas9 and compared them with isogenic wild-type hPSCs and human germ cell tumor lines as models of malignancy. While no major changes in proliferation, pluripotency, and transcriptomic profiles are found, mutant lines display aberrations in some of the main chromosomal hotspots for genetic abnormalities in hPSCs. Additionally, enhanced clonogenic and anchorage-free growth, alongside resistance to chemotherapeutic compounds is observed. The results indicate that common TP53-depleting mutations in hPSCs, although potentially overlooked by standard analyses, can impact their behavior and safety in a clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b157fa230ecf42b51895b57dbccc944884bf0d8" target='_blank'>
              Depletion of TP53 in Human Pluripotent Stem Cells Triggers Malignant-Like Behavior.
              </a>
            </td>
          <td>
            Joaquin Montilla-Rojo, T. F. Eleveld, Marnix van Soest, Sanne Hillenius, Dennis M. Timmerman, A. Gillis, B. Roelen, C L Mummery, Leendert H J Looijenga, Daniela C. F. Salvatori
          </td>
          <td>2025-01-06</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="To enhance understanding of cellular heterogeneity and disease from single-cell sequencing data, we introduce ResidPCA, a robust method for cell state identification that models cell type heterogeneity. Simulations demonstrate ResidPCA’s efficacy, particularly in complex scenarios, with its accuracy more than four times higher than conventional Principal Component Analysis (PCA) and over three times higher than Non-negative Matrix Factorization (NMF)-based methods in identifying states expressed across multiple cell types. In scRNA-seq data from light-stimulated mouse visual cortex cells, ResidPCA captures stimulus-driven variability with an accuracy more than five times higher than NMF methods. In single nucleus data from an Alzheimer’s disease cohort, ResidPCA identified 44 snATAC-based and 42 snRNA-based states. 30 snATAC states were significantly enriched for Alzheimer’s disease heritability and were often more significantly enriched than established cell types such as microglia. The ResidPCA-based snATAC state most significantly enriched for Alzheimer’s disease heritability further elucidates a recently identified mechanism involving the neuron-ODC-microglial axis. This state links early amyloid production in neurons and oligodendrocytes with later-stage microglial activation and immune response, driving Alzheimer’s disease progression. These results demonstrate ResidPCA’s ability to reveal additional biological variation in single-cell data and uncover disease-relevant cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca44f2bab9201756fbfd5e90a83f848209acc96f" target='_blank'>
              Discovery of disease-associated cellular states using ResidPCA in single-cell RNA and ATAC sequencing data
              </a>
            </td>
          <td>
            Shaye Carver, Kodi Taraszka, S. Groha, Alexander Gusev
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="RNA velocity has emerged as a popular approach for modeling cellular change along the phenotypic landscape but routinely omits regulatory interactions between genes. Conversely, methods that infer gene regulatory networks (GRNs) do not consider the dynamically changing nature of biological systems. To integrate these two currently disconnected fields, we present RegVelo, an end-to-end dynamic, interpretable, and actionable deep learning model that learns a joint model of splicing kinetics and gene regulatory relationships and allows us to perform in silico perturbation predictions. When applied to datasets of the cell cycle, human hematopoiesis, and murine pancreatic endocrinogenesis, RegVelo demonstrates superior predictive power for interactions and perturbation simulations, for example, compared to methods that focus solely on dynamics or GRN inference. To leverage RegVelo’s full potential, we studied the dynamics of zebrafish neural crest development and underlying regulatory mechanisms using our Smart-seq3 dataset and shared gene expression and chromatin accessibility measurements. Using RegVelo’s in silico perturbation predictions, validated by CRISPR/Cas9-mediated knockout and single-cell Perturb-seq, we establish transcription factor tfec as an early driver and elf1 as a novel regulator of pigment cell fate and propose a gene-regulatory circuit involving tfec and elf1 interactions via the toggle-switch model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb495aa52d71e37d301ee44c41f833b2bc276eae" target='_blank'>
              RegVelo: gene-regulatory-informed dynamics of single cells
              </a>
            </td>
          <td>
            Weixu Wang, Zhiyuan Hu, P. Weiler, Sarah Mayes, Marius Lange, Jingye Wang, Zhengyuan Xue, Tatjana Sauka-Spengler, F. Theis
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Introduction Metastatic cancer affects millions of people worldwide annually and is the leading cause of cancer-related deaths. Most patients with metastatic disease are not eligible for surgical resection, and current therapeutic regimens have varying success rates, some with 5-year survival rates below 5%. Here, we test the hypothesis that metastatic cancer can be genetically targeted by exploiting single base substitution mutations unique to individual cells that occur as part of normal aging prior to transformation. These mutations are targetable because ~10% of them form novel tumor-specific “NGG” protospacer adjacent motif (PAM) sites targetable by CRISPR-Cas9. Methods Whole genome sequencing was performed on five rapid autopsy cases of patient-matched primary tumor, normal and metastatic tissue from pancreatic ductal adenocarcinoma decedents. CRISPR-Cas9 PAM targets were determined by bioinformatic tumor-normal subtraction for each patient and verified in metastatic samples by high-depth capture-based sequencing. Results We found that 90% of PAM targets were maintained between primary carcinomas and metastases overall. We identified rules that predict PAM loss or retention, where PAMs located in heterozygous regions in the primary tumor can be lost in metastases (private LOH), but PAMs occurring in regions of loss of heterozygosity (LOH) in the primary tumor were universally conserved in metastases. Conclusions Regions of truncal LOH are strongly retained in the presence of genetic instability and, therefore, represent genetic vulnerabilities in pancreatic adenocarcinomas. A CRISPR-based gene therapy approach targeting these regions may be a novel way to genetically target metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada3070ea15ff6901b235b693417e61df449621f" target='_blank'>
              Islands of genomic stability in the face of genetically unstable metastatic cancer
              </a>
            </td>
          <td>
            Kirsten Bowland, Jiaying Lai, Alyza M. Skaist, Yan Zhang, S. Teh, Nicholas J. Roberts, Elizabeth Thompson, Sarah Wheelan, Ralph H. Hruban, Rachel Karchin, Matthew H. Bailey, Christine A. Iacobuzio-Donahue, J. Eshleman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aede480e99c41ff823fe07aee10fb1b86fa8b" target='_blank'>
              DNA methylation-based analysis reveals accelerated epigenetic aging in giant cell-enriched adult-type glioblastoma
              </a>
            </td>
          <td>
            P. Cakmak, Philipp Jurmeister, I. Divé, P. Zeiner, Joachim P. Steinbach, Tim R Fenton, K. H. Plate, M. Czabanka, Patrick Harter, Katharina J. Weber
          </td>
          <td>2024-12-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background The role of cancer-associated pericytes (CAPs) in tumor microenvironment (TME) suggests that they are potential targets for cancer treatment. The mechanism of CAP heterogeneity in esophageal squamous cell carcinoma (ESCC) remains unclear, which has limited the development of treatments for tumors through CAPs. Therefore, a comprehensive understanding of the classification, function, cellular communication and spatial distribution of CAP subpopulations in ESCC is urgently needed. Methods This study used large-sample single-cell transcriptome sequencing (scRNA-seq) data to investigate pericytes’ subpopulation characteristics, functions, upstream and downstream regulation and interactions with other components of the TME in the ESCC, and analyzed prognostically in conjunction with Bulk RNA-seq data. In addition, pericyte subpopulations were validated and their spatial distribution in the ESCC TME was observed by multiplex immunofluorescence. Drug prediction and molecular docking was further used to validate the medicinal value of drug targets. Results CAPs in the ESCC TME were found to be highly heterogeneous, and we identified six pericyte subtypes: c1_ARHGDIB, c2_BCAM, c3_LUM, c4_SOD2, c5_TYMS, and c6_KRT17, which have commonality in a part of their functions, and each of them has a major function to play, by having different strengths of interaction with different components in the TME. In addition, we found that c4_SOD2 was negatively correlated with prognosis, conversely, c5_TYMS was positively correlated with prognosis. The drug with a better effect on c5_TYMS was docetaxel (binding energy = -8.1, -8.7 kcal/mol); raloxifene may be more effective against c4_SOD2, although raloxifene has a slightly lower binding energy to SOD2 (-6.4 kcal/mol), it has a higher binding energy to PDGFRβ (-8.1 kcal/mol). Conclusion The present study identified and discovered pericyte subpopulations that were significantly associated with prognosis, which provides new biomarkers for predicting patient prognosis and adds usable targets for immunotherapy, and it is also important for gaining insights into the composition of the TME in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e295485d04626d24ad4217e848a133499f9df967" target='_blank'>
              Single-cell sequencing analysis reveals cancer-associated pericyte subgroup in esophageal squamous cell carcinoma to predict prognosis
              </a>
            </td>
          <td>
            Kai Xiong, Bing Pan, Hao Fang, Ziyou Tao
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a central role in tumor progression, recurrence, and resistance to conventional therapies, making them a critical focus in oncology research. This review provides a comprehensive analysis of CSC biology, emphasizing their self-renewal, differentiation, and dynamic interactions with the tumor microenvironment (TME). Key signaling pathways, including Wnt, Notch, and Hedgehog, are discussed in detail to highlight their potential as therapeutic targets. Current methodologies for isolating CSCs are critically examined, addressing their advantages and limitations in advancing precision medicine. Emerging technologies, such as CRISPR/Cas9 and single-cell sequencing, are explored for their transformative potential in unraveling CSC heterogeneity and informing therapeutic strategies. The review also underscores the pivotal role of the TME in supporting CSC survival, promoting metastasis, and contributing to therapeutic resistance. Challenges arising from CSC-driven tumor heterogeneity and dormancy are analyzed, along with strategies to mitigate these barriers, including novel therapeutics and targeted approaches. Ethical considerations and the integration of artificial intelligence in designing CSC-specific therapies are discussed as essential elements of future research. The manuscript advocates for a multi-disciplinary approach that combines innovative technologies, advanced therapeutics, and collaborative research to address the complexities of CSCs. By bridging existing gaps in knowledge and fostering advancements in personalized medicine, this review aims to guide the development of more effective cancer treatment strategies, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99151138c6fc7abf05753313b772a7f6497b1128" target='_blank'>
              Deciphering the Role of Cancer Stem Cells: Drivers of Tumor Evolution, Therapeutic Resistance, and Precision Medicine Strategies
              </a>
            </td>
          <td>
            Mohamed El-Tanani, S. Rabbani, S. Satyam, I. Rangraze, A. Wali, Yahia El-Tanani, Alaa A. A. Aljabali
          </td>
          <td>2025-01-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Over the past century, colorectal cancer (CRC) has become one of the most devastating cancers impacting the human population. To gain a deeper understanding of the molecular mechanisms driving this solid tumor, researchers have increasingly turned their attention to the tumor microenvironment (TME). Spatial transcriptomics and proteomics have emerged as a particularly powerful technology for deciphering the complexity of CRC tumors, given that the TME and its spatial organization are critical determinants of disease progression and treatment response. Spatial transcriptomics enables high-resolution mapping of the whole transcriptome. While spatial proteomics maps protein expression and function across tissue sections. Together, they provide a detailed view of the molecular landscape and cellular interactions within the TME. In this review, we delve into recent advances in spatial biology technologies applied to CRC research, highlighting both the methodologies and the challenges associated with their use, such as the substantial tissue heterogeneity characteristic of CRC. We also discuss the limitations of current approaches and the need for novel computational tools to manage and interpret these complex datasets. To conclude, we emphasize the importance of further developing and integrating spatial transcriptomics into CRC precision medicine strategies to enhance therapeutic targeting and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b317770be242dc222a3004766249a5b9de43dcc" target='_blank'>
              Current state and future prospects of spatial biology in colorectal cancer
              </a>
            </td>
          <td>
            Francisco G. Carranza, Fernando C. Diaz, Maria Ninova, Enrique Velazquez-Villarreal
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ace411cdcea09360bc78194a610964a1bfb00c6" target='_blank'>
              AXL: shapers of tumor progression and immunosuppressive microenvironments
              </a>
            </td>
          <td>
            Yihui Liu, Lei Xu, Yuanyao Dou, Yong He
          </td>
          <td>2025-01-11</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) is the most common histologic type of RCC. However, the spatial and functional heterogeneity of immunosuppressive cells and the mechanisms by which their interactions promote immunosuppression in the ccRCC have not been thoroughly investigated. Methods To further investigate the cellular and regional heterogeneity of ccRCC, we analyzed single-cell and spatial transcriptome RNA sequencing data from four patients, which were obtained from samples from multiple regions, including the tumor core, tumor-normal interface, and distal normal tissue. On the basis, the findings were investigated in vitro using tissue and blood samples from 15 patients with ccRCC and validated in the broader samples on tissue microarrays. Results In this study, we revealed previously unreported subsets of both stromal and immune cells, as well as mapped their spatial location at finer resolution. In addition, we validated the clusters of tumor cells after removing batch effects according to six characterized gene sets, including epithelial-mesenchymal transitionhigh clusters, metastatic clusters and proximal tubulehigh clusters. Importantly, we identified a special regulatory T (Treg) cell subpopulation that has the molecular characteristics of terminal effector Treg cells but expresses multiple cytokines, such as interleukin (IL)-1β and IL-18. This group of Treg cells has stronger immunosuppressive function and was associated with a worse prognosis in ccRCC cohorts. They were colocalized with MRC1+FOLR2+ tumor-associated macrophages (TAMs) at the tumor-normal interface to form a positive feedback loop, maintaining a synergistic procarcinogenic effect. In addition, we traced the origin of IL-1β+ Treg cells and revealed that IL-18 can induce the expression of IL-1β in Treg cells via the ERK/NF-κB pathway. Conclusions We demonstrated a novel cancer-promoting Treg cell subset and its interactions with MRC1+FOLR2+TAMs, which provides new insight into Treg cell heterogeneity and potential therapeutic targets for ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2214f0f0d02f003fb8cfcf936f4f2873d564786b" target='_blank'>
              Spatial and single-cell transcriptomics reveal cellular heterogeneity and a novel cancer-promoting Treg cell subset in human clear-cell renal cell carcinoma
              </a>
            </td>
          <td>
            Xiyu Song, Yumeng Zhu, Wenwen Geng, J. Jiao, Hongjiao Liu, Ruo Chen, Qian He, Lijuan Wang, Xiuxuan Sun, Weijun Qin, Jiejie Geng, Zhinan Chen
          </td>
          <td>2025-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cellular activity is spatially organized across different organelles. While several structures are well-characterized, many organelles have unknown roles. Profiling biomolecular composition is key to understanding function but is difficult to achieve in the context of small, dynamic structures. Photoproximity labeling has emerged as a powerful tool for mapping these interaction networks, yet maximizing catalyst localization and reducing toxicity remains challenging in live cell applications. Here, we disclose a new intracellular photocatalyst with minimal cytotoxicity and off-target binding, and we utilize this catalyst for HaloTag-based microenvironment-mapping (μMap) to spatially catalog subnuclear condensates in living cells. We also specifically develop a novel RNA-focused workflow (μMap-seq) to enable parallel transcriptomic and proteomic profiling of these structures. After validating the accuracy of our approach, we generate a spatial map across the nucleolus, nuclear lamina, Cajal bodies, paraspeckles, and PML bodies. These results provide potential new insights into RNA metabolism and gene regulation while significantly expanding the μMap platform for improved live-cell proximity labeling in biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cedc48a841755a5a6c7dc0077077c40278c4804" target='_blank'>
              Parallel Proteomic and Transcriptomic Microenvironment Mapping (μMap) of Nuclear Condensates in Living Cells.
              </a>
            </td>
          <td>
            Steve D. Knutson, C. Pan, Niels Bisballe, Brandon J Bloomer, Philip Raftopolous, Iakovos Saridakis, David W. C. MacMillan
          </td>
          <td>2024-12-21</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bc9bd11ddbadc141e7404167a9d1237fb1c3a74" target='_blank'>
              Encompassing view of spatial and single-cell RNA sequencing renews the role of the microvasculature in human atherosclerosis.
              </a>
            </td>
          <td>
            Tore Bleckwehl, Anne Babler, Merel Tebens, Sidrah Maryam, Michael Nyberg, Markus Bosteen, Maurice Halder, Isaac Shaw, Susanne Fleig, Charles Pyke, Henning Hvid, Louise M. Voetmann, J. V. van Buul, J. Sluimer, Vivek Das, Simon J. Baumgart, R. Kramann, Sikander Hayat
          </td>
          <td>2024-12-23</td>
          <td>Nature cardiovascular research</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5daf8fc78edec4dc67cc0ac87030a2ee471c5078" target='_blank'>
              Cancer-associated fibroblasts: heterogeneity, tumorigenicity and therapeutic targets
              </a>
            </td>
          <td>
            Keke Lv, Tianlin He
          </td>
          <td>2024-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT High-risk non-muscle-invasive bladder cancer (NMIBC) presents high recurrence and progression rates. Despite the use of Bacillus Calmette-Guérin gold-standard immunotherapy and the recent irruption of anti-PD-1/PD-L1 drugs, we are missing a comprehensive understanding of the tumor microenvironment (TME) that may help us find biomarkers associated to treatment outcome. Here, we prospectively analyzed TME composition and PD-L1 expression of tumor and non-tumoral tissue biopsies from 73 NMIBC patients and used scRNA-seq, transcriptomic cohorts and tissue micro-array to validate the prognostic value of cell types of interest. Compared to non-tumoral tissue, NMIBC presented microvascular alterations, increased cancer-associated fibroblast (CAF) and myofibroblast (myoCAF) presence, and varied immune cell distribution, such as increased macrophage infiltration. Heterogeneous PD-L1 expression was observed across subsets, with macrophages showing the highest expression levels, but cancer cells as the primary potential anti-PD-L1 binding targets. Unbiased analysis revealed that myoCAF and M2-like macrophages are specifically enriched in high-grade NMIBC tumors. The topological distribution of these two cell types changed as NMIBC progresses, as shown by immunofluorescence. Only myoCAFs were associated with higher rates of progression and recurrence in three independent cohorts (888 total patients), reaching prediction values comparable to transcriptomic classes, which we further validated using tissue micro-array. Our study provides a roadmap to establish the landscape of the NMIBC TME, highlighting myoCAFs as potential prognostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe97543b0d1de2e394926985ac37960e50a2bea" target='_blank'>
              Characterisation of the tumour microenvironment and PD-L1 granularity reveals the prognostic value of cancer-associated myofibroblasts in non-invasive bladder cancer
              </a>
            </td>
          <td>
            Carmen G. Cañizo, F. Guerrero-Ramos, M. Perez Escavy, I. Lodewijk, C. Suárez-Cabrera, L. Morales, Sandra P Nunes, E. Munera-Maravilla, Carolina Rubio, Rebeca Sánchez, Marta Rodríguez-Izquierdo, Jaime Martínez de Villarreal, Francisco X. Real, D. Castellano, Cristina Martín-Arriscado, David Lora Pablos, A. Rodríguez Antolín, M. Dueñas, J. Paramio, Victor G. Martínez
          </td>
          <td>2024-12-19</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Aging is a multifaceted process that manifests differently in various cell types and within the same cell type. Therefore, understanding age-related changes with single-cell resolution may illuminate the mechanistic details of tissue aging and offer targets for intervention. We utilized high-throughput single-cell RNA-sequencing (scRNA-seq) to analyze senescence-associated changes within the CD4 T cell compartment. First, we sequenced total human CD4 T cells from multiple donors, revealing that naïve CD4 T cells exhibited a less senescent state compared to memory cells. We focused on naïve CD4 T cells to determine if individual cells differ in expression of aging-associated markers. By sorting cells based on senescence-associated beta-galactosidase (B-gal) activity and applying scRNA-seq, we identified transcriptional signatures associated with more youthful and senescent states. Surprisingly, we discovered cells with lower B-gal expression comprised two distinct populations: one marked by high expression of HELIOS and low expression of SESN3, and another marked by high expression of BACH2 and low expression of TC2N. In comparison, 2 youthful subpopulation and senescence subpopulations differ in markers of immunological activation We then investigated if the composition of naive CD4 compartment changes with age, finding that the abundance of youthful subpopulations was reduced in the older donors’ CD4 populations. Thus, aging in the naïve CD4 compartment is associated with a shifted distribution between more youthful and more senescent cells. Moving forward, we will attempt to manipulate naïve CD4 T cells into a more youthful state by altering expression of the identified transcriptional signatures associated with youthful and senescent states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bab6ece44b0e3064c9cd597254aa9a3e1045d71" target='_blank'>
              DISTINCT CELL SUBTYPES WITHIN NAÏVE CD4 T CELLS COMPARTMENT SHAPE ITS AGING AND SENESCENCE
              </a>
            </td>
          <td>
            C. J. Kelly, Ashley Jeon, Jacob Rodriguez, Dina Prosser, Morgan Diegel, Nikolay Burnaevskiy
          </td>
          <td>2024-12-01</td>
          <td>Innovation in Aging</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55057507874b2d1db614aaeb55e2719a4aaa942d" target='_blank'>
              Metabolic rewiring in skin epidermis drives tolerance to oncogenic mutations.
              </a>
            </td>
          <td>
            Anupama Hemalatha, Zong-Hu Li, David G. Gonzalez, C. Matte-Martone, Karen Tai, Elizabeth Lathrop, Daniel Gil, S. Ganesan, Lauren E Gonzalez, Melissa C. Skala, Rachel J. Perry, Valentina Greco
          </td>
          <td>2025-01-06</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Ambiguous understanding of tumors and the tumor microenvironments (TMEs) hinders accurate diagnosis and available treatment for multifocal hepatocellular carcinoma (HCC) covering intrahepatic metastasis (IM) and multicentric occurrence (MO). Here, we characterized the diverse TMEs of IM and MO identified by whole-exome sequencing (WES) at single-cell resolution.


APPROACH AND RESULTS
We performed parallel WES and single-cell RNA sequencing (scRNA-seq) on twenty-three samples from seven patients to profile their TMEs when major results were validated by immunohistochemistry in the additional cohort. Integrative analysis of WES and scRNA-seq found that malignant cells in IM showed higher intra-tumor heterogeneity, stemness and more activated metabolism than those in MO. Tumors from IM shared similar TMEs while distinct TMEs were noticed in those from MO. Furthermore, CD20+ B cells, plasma cells and conventional type II dendritic cells (cDC2s) were decreased in IM relative to MO while T cells in IM exhibited a more terminally exhausted capacity with a higher proportion of proliferative/exhausted T cells than that in MO. Both CD20 and CD1C correlated with better prognosis in multifocal HCC. Additionally, MMP9+ tumor-associated macrophages were enriched across IM and MO which formed cellular niches with regulatory T cells (Tregs) and proliferative/exhausted T cells.


CONCLUSIONS
Our findings deeply decipher the heterogeneous TMEs between IM and MO, which provide a comprehensive landscape of multifocal HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2c7a45eee2228edb84e8693a219534a9ab548a" target='_blank'>
              Genomic and the tumor microenvironment heterogeneity in multifocal hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yongheng Yang, Qingqiang Ni, Hongguang Li, Jiuzheng Sun, Xia Zhou, Lingxin Qu, Liyuan Wang, Chuanzong Zhao, Xiaolu Zhang
          </td>
          <td>2024-12-12</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89f4dafce3e68eb14cab151613eca45fd2ce0bb6" target='_blank'>
              Opportunities and challenges of single-cell and spatially resolved genomics methods for neuroscience discovery.
              </a>
            </td>
          <td>
            B. Bonev, Castelo-Branco Gonçalo, Fei Chen, Simone Codeluppi, M. Corces, Jean Fan, Myriam Heiman, Kenneth D. Harris, Fumitaka Inoue, M. Kellis, Ariel Levine, M. Lotfollahi, Chongyuan Luo, Kristen R. Maynard, Mor Nitzan, Vijay Ramani, Rahul Satijia, Lucas Schirmer, Yin Shen, Na Sun, G. Green, F. Theis, Xiao Wang, Joshua D. Welch, Ozgun Gokce, Genevieve Konopka, SA Liddelow, Evan Z. Macosko, O. Bayraktar, Naomi Habib, T. Nowakowski
          </td>
          <td>2024-12-01</td>
          <td>Nature neuroscience</td>
          <td>3</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4bbd7ae49e10933acda8e20cbd7e3bb473c50d" target='_blank'>
              Endosomal pH is an evolutionarily conserved driver of phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            Hari Prasad, H. Bv, A. R. Subbalakshmi, S. Mandal, M. Jolly, Sandhya S. Visweswariah
          </td>
          <td>2024-12-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Hepatic arterial infusion chemotherapy (HAIC) has emerged as a promising treatment strategy for hepatocellular carcinoma (HCC), but a detailed understanding of the multicellular ecosystem after HAIC treatment is lacking. Here, we collected tumor samples from treatment-naïve primary and post-HAIC HCC, and integrated single-nucleus RNA sequencing with spatial transcriptomics to characterize the tumor ecosystem in the post-HAIC HCC. Increased fractions and enhanced cellular communication of CD4+ T, CD20+ B, and dendritic cell subtypes were identified in post-HAIC tumors. Moreover, it is substantiated that HAIC promoted tertiary lymphoid structures (TLS) formation, and addressed the roles of TLSs as spatial niches of cellular communication. Specifically, intermediate exhausted CD8+ T cells expressing Granzyme-K and PD-1 (PD-1+CD8+ Tex-int) expanded following HAIC and exhibited a functionally antitumor phenotype. PD-1+CD8+ Tex-int accumulated in the TLS vicinity and disseminated throughout the tumor microenvironment, demonstrating potential as an effective biomarker for HAIC-based treatment in HCC. This study provides valuable resources and biological insights in the cellular underpinnings of HAIC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62624d4b04f46d42e65710fe0c98f6d6afaa46d" target='_blank'>
              Single-Nucleus and Spatial Transcriptome Profiling Delineates the Multicellular Ecosystem in Hepatocellular Carcinoma After Hepatic Arterial Infusion Chemotherapy.
              </a>
            </td>
          <td>
            YeXing Huang, Zefeng Du, Zhicheng Lai, DongSheng Wen, Lichang Huang, Minke He, ZiChao Wu, Huifang Li, Hanyue Ouyang, Wenchao Wu, Anna Kan, Ming Shi
          </td>
          <td>2024-12-16</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffc5afe786b47a61ca4c6c86b698a4b6f0aa1df3" target='_blank'>
              Characterizing tertiary lymphoid structures associated single-cell atlas in breast cancer patients
              </a>
            </td>
          <td>
            Xiaokai Fan, Daqin Feng, Donggui Wei, Anqi Li, Fangyi Wei, Shufang Deng, Muling Shen, Congzhi Qin, Yongjia Yu, Lun Liang
          </td>
          <td>2025-01-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12d5f36d08b67eac2a4dda22420dc7c18c1857db" target='_blank'>
              Cancer immune evasion, immunoediting and intratumour heterogeneity.
              </a>
            </td>
          <td>
            Malte Roerden, Stefani Spranger
          </td>
          <td>2025-01-02</td>
          <td>Nature reviews. Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8124e4116b706bf406200aaca77b811cd619c61b" target='_blank'>
              Metabolically activated and highly polyfunctional intratumoral VISTA+ regulatory B cells are associated with tumor recurrence in early-stage NSCLC
              </a>
            </td>
          <td>
            D. Lo Tartaro, Beatrice Aramini, V. Masciale, Nikolaos Paschalidis, F. Lofaro, Anita Neroni, R. Borella, Elena Santacroce, Alin Liviu Ciobanu, A. V. Samarelli, Federica Boraldi, Daniela Quaglino, Alessandra Dubini, Michele Gaudio, G. Manzotti, F. Reggiani, Federica Torricelli, A. Ciarrocchi, A. Neri, Federica Bertolini, Massimo Dominici, Pier Luigi Filosso, F. Stella, L. Gibellini, S. De Biasi, Andrea Cossarizza
          </td>
          <td>2025-01-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Tumor microenvironment (TME) plays a crucial role in the oncogenesis, early distant metastasis, and limited immunotherapeutic responses of pancreatic ductal adenocarcinoma (PDAC). Myeloid cell populations, particularly tumor-associated macrophages (TAMs), located within this immunosuppressive niche, are pivotal in these processes. With aid of single cell RNA sequencing (scRNA-seq), multiple studies have successfully validated distinct macrophage subtypes pathological functions in different cancer types, yet the correlation of TAMs in hepatic metastatic and primary PDAC lesions remains insufficiently understood. In this study, we reanalyzed samples from both primary and hepatic metastatic PDACs to elucidate the functional conservation and variability of TAMs. Pseudotime trajectory inference, together with pseudobulk analysis based on subtype level, manifests TAMs undergo similar metabolism reprogramming and pro-inflammatory style transitions with specific signaling pathway activations. Compared to primary tumors, hypoxic conditions are alleviated, which largely depends on the physiological context and the preliminary ligand-receptor interactive network within the metastatic niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a56bbf6854596a35b27cf40482bd3f5c23fafab" target='_blank'>
              In-depth analysis of pathological function of tumor-associated macrophage populations in hepatic metastatic and primary PDAC
              </a>
            </td>
          <td>
            Fanyu Meng
          </td>
          <td>2024-12-12</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L. Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The TEM8 protein coded by the ANTXR1 gene represents an emerging biomarker in solid tumors. In addition to the various roles TEM8 plays in oncogenesis, including angiogenesis, epithelial-to-mesenchymal transition, and cell migration, it has also been shown that the overexpression of the ANTXR1 gene in solid tumors correlates with poor prognostic indicators in several solid tumor histologies. As such, TEM8 has been identified as the target of novel oncologic therapies. It is especially attractive given its selective expression on the surface of solid tumor cells and associated stromal cells, such as cancer stem cells, invasive cancer cells, and immune cells, such as macrophages, angiogenic endothelial cells, pericytes, and cancer-associated fibroblasts. Furthermore, TEM8 plays this unique role as a mostly non-mutated gene in solid cancers. Here, we demonstrate that elevated expression of ANTXR1 in bladder cancer showed a statistical difference not only in overall survival (OS) but in progression-free survival (PFS), confirming the prognostic biomarker power of ANTXR1 expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dff315ebf7de91444ff283e65fc1f528bbeae62" target='_blank'>
              Elevated expression of ANTXR1 gene in tumors is a poor prognostic biomarker for patients with bladder cancer
              </a>
            </td>
          <td>
            L. S. Franco, S. Arunachalam, A. Chauhan, S. A. Kareff, P. Hallenbeck
          </td>
          <td>2025-01-23</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In this study, we delve into the intrinsic mechanisms of cell communication in hepatocellular carcinoma (HCC). Initially, employing single-cell sequencing, we analyze multiple malignant cell subpopulations and cancer-associated fibroblast (CAF) subpopulations, revealing their interplay through receptor-ligand interactions, with a particular focus on SPP1. Subsequently, employing unsupervised clustering analysis, we delineate two clusters, C1 and C2, and compare their infiltration characteristics using various tools and metrics, uncovering heightened cytotoxicity and overall invasion abundance in C1. Furthermore, our gene risk scoring model indicates heightened activity of the immune therapeutic pathway in C1. Lastly, employing a formulated scoring system, we stratify patients into high and low-risk groups, revealing notably poorer outcomes in the high-risk cohort on Kaplan-Meier curves. Risk scores exhibit a negative correlation with model genes and immune cell infiltration scores, indicating poor prognosis in the high-risk group. Further characterization elucidates the regulatory landscape of the high and low-risk groups across various signaling pathways. In addition, we used wet lab experiments to prove that ABCA1 plays a pro-oncogenic role in hepatocellular carcinoma cells by promoting proliferation, invasion, migration, and reducing apoptosis. In summary, these findings provide crucial insights, offering valuable clues and references for understanding HCC pathogenesis and patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ebd5bca77500ddcf823c450cf7ed50074d3c6d" target='_blank'>
              ABCA1 promote tumor environment heterogeneity via epithelial mesenchymal transition in Huh7 and HepG2 liver cancer cell
              </a>
            </td>
          <td>
            Dinglai Yu, Fang Guo, Qiyu Zhang, Huajun Yu, Wenmin Wang, Yunzhi Chen
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d2a730533f465c1760e5f39354ae03ee2bf1e3c" target='_blank'>
              Spatial expression of fibroblast activation protein-α in clear cell renal cell carcinomas revealed by multiplex immunoprofiling analysis of the tumor microenvironment
              </a>
            </td>
          <td>
            G. Larrinaga, M. Redrado, Ana Loizaga-Iriarte, Amparo Pérez-Fernández, Aida Santos-Martín, Javier C. Angulo, José A Fernández, Alfonso Calvo, José I. López
          </td>
          <td>2025-01-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Urothelial carcinoma (UC) can arise from either the lower urinary tract or the upper tract; they represent different disease entities and require different clinical treatment strategies. A full understanding of the cellular characteristics in UC may guide the development of novel therapies. Here, we performed single-cell transcriptome analysis from four patients with UC of the bladder (UCB), five patients with UC of the ureter (UCU), and four patients with UC of the renal pelvis (UCRP) to develop a comprehensive cell atlas of UC. We found the rare epithelial cell subtype EP9 with epithelial-to-mesenchymal transition (EMT) and cancer stem cell (CSC) features, and specifically expressed SOX6, which was associated with poor prognosis. We also found that ACKR1+ endothelial cells and inflammatory cancer-associated fibroblasts (iCAFs) were more enriched in UCU, which may promote pathogenesis. While ESM1+ endothelial cells may more actively participate in UCB and UCRP tumorigenesis by promoting angiogenesis. Additionally, CD8 + effector T cells were more enriched in UCU and UCRP patients, while Tregs were mainly enriched in UCB tumors. C1QC+ macrophages and LAMP3+ dendritic cells were more enriched in UCB, which is closely related to the formation of the heterogeneous immunosuppressive microenvironment. Furthermore, we found strong interactions between iCAFs, EP9, and Endo_ESM1, and different degrees of activation of the FGF-FGFR3 axis and immune checkpoint pathway were observed in different UC subtypes. Our study elucidated the cellular heterogeneity and the components of the microenvironment in UC arising from the upper and lower urinary tracts and provided novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28a7771d79701d4fbee36981bff69a8e0ebdd585" target='_blank'>
              Single-Cell RNA Sequencing Reveals the Tumor Heterogeneity and Immunosuppressive Microenvironment in Urothelial Carcinoma.
              </a>
            </td>
          <td>
            Tianqi Lyu, Kerong Wu, Yincong Zhou, Tong Kong, Lin Li, Kaizhe Wang, Pan Fu, Pengyao Wei, Ming Chen, Jianping Zheng
          </td>
          <td>2024-12-26</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a common malignant tumor of the digestive system with a high incidence that seriously threatens patients’ lives and health. However, with the rise and application of new treatments, such as immunotherapy, there are still some restrictions in the treatment and diagnosis of HCC, and the therapeutic effects on patients are not ideal. Methods Two single-cell RNA sequencing (scRNA-seq) datasets from HCC patients, encompassing 25,189 cells, were analyzed in the study. We utilized non-negative matrix factorization (NMF) clustering to identify mitophagy patterns in HCC TME cells, including cancer-associated fibroblasts (CAFs), T cells, B cells, and tumor-associated macrophages (TAMs). Cell-to-cell communication was analyzed using the CellChat package, and pseudotime trajectory analysis was performed using the Monocle package. Gene regulatory networks were investigated with the SCENIC package, and survival analyses were conducted with mitophagy-related signatures. Results HCC samples analysis identified 22 clusters, including 7 principal cell types. Complex cell communications were observed among these cell types. Mitophagy-related CAFs, TAMs, CD8+ T cells, and B cells were identified. These subtypes had different biological states, cell-cell communications, and metabolic pathways. Mitophagy levels were elevated in tumor samples. Changes in mitophagy-related genes within specific cell subtypes were associated with different overall survival rates. However, mitophagy did not seem to affect the effectiveness of immunotherapy. Conclusion This study provides evidence that mitophagy within the HCC TME modulates intercellular communication, influencing tumor progression and patient prognosis. Targeting mitophagy may offer a promising approach to improve the long-term prognosis of HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/689de3b101ffbbea61436b63812b3f0a30320588" target='_blank'>
              Single-cell mitophagy patterns within the tumor microenvironment modulate intercellular communication, impacting the progression and prognosis of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zhengyan Li, Wei Chen, Shu Yao, Zuxiang Peng, Hongming Liu, Yongliang Tang, Yi Feng
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cadee24cc5226a29346aba0f25d7d1cb128d245d" target='_blank'>
              Targeting SRSF1 improves cancer immunotherapy by dually acting on CD8+T and tumor cells
              </a>
            </td>
          <td>
            G. Zhu, Zheng Tang, Tianhao Chu, Biao Wang, , C. Tao, , Rui Yang, Weifeng Qu, Yi Wang, Qianfu Zhao, Run Huang, , Yuan Fang, Jun Gao, Xiaoling Wu, Jian Zhou, Weiren Liu, Zhi Dai, Yinghong Shi, Jia Fan
          </td>
          <td>2025-01-22</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde0e9abf6b50983ec6394f87b77c188d773d609" target='_blank'>
              YAP-driven malignant reprogramming of oral epithelial stem cells at single cell resolution
              </a>
            </td>
          <td>
            F. Faraji, Sydney I. Ramirez, Lauren M. Clubb, Kuniaki Sato, Valeria Burghi, Thomas S. Hoang, Adam Officer, Paola Y. Anguiano Quiroz, William M. G. Galloway, Zbigniew Mikulski, Kate Medetgul-Ernar, P. Marangoni, Kyle B Jones, Yuwei Cao, A. Molinolo, Kenneth Kim, Kanako Sakaguchi, Joseph A. Califano, Quinton Smith, Alon Goren, Ophir D. Klein, Pablo Tamayo, J. S. Gutkind
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC), an aggressive and lethal subtype of prostate cancer (PCa), often arises as a resistance mechanism in patients undergoing hormone therapy for prostate adenocarcinoma. NEPC is associated with a significantly poor prognosis and shorter overall survival compared to conventional prostate adenocarcinoma due to its aggressive nature and limited response to standard of care therapies. This transdifferentiation, or lineage reprogramming, to NEPC is characterised by the loss of androgen receptor (AR) and prostate-specific antigen (PSA) expression, and the upregulation of neuroendocrine (NE) biomarkers such as neuron-specific enolase (NSE), chromogranin-A (CHGA), synaptophysin (SYP), and neural cell adhesion molecule 1 (NCAM1/CD56), which are critical for NEPC diagnosis. The loss of AR expression culminates in resistance to standard of care PCa therapies, such as androgen-deprivation therapy (ADT) which target the AR signalling axis. This review explores the drivers of NE transdifferentiation. Key genetic alterations, including those in the tumour suppressor genes RB1, TP53, and PTEN, and changes in epigenetic regulators, particularly involving EZH2 and cell-fate-determining transcription factors (TFs) such as SOX2, play significant roles in promoting NE transdifferentiation and facilitate the lineage switch from prostate adenocarcinoma to NEPC. The recent identification of several other key novel drivers of NE transdifferentiation, including MYCN, ASCL1, BRN2, ONECUT2, and FOXA2, further elucidates the complex regulatory networks and pathways involved in this process. We suggest that, given the multifactorial nature of NEPC, novel therapeutic strategies that combine multiple modalities are essential to overcome therapeutic resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f20c9ed86b45bc7aeb40ff01ef36c96d8711e59c" target='_blank'>
              Therapeutic Exploitation of Neuroendocrine Transdifferentiation Drivers in Prostate Cancer
              </a>
            </td>
          <td>
            Zoe R. Maylin, Christopher Smith, Adam Classen, Mohammad Asim, H. Pandha, Yuzhuo Wang
          </td>
          <td>2024-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract A robust understanding of the cellular mechanisms underlying diseases sets the foundation for the effective design of drugs and other interventions. The wealth of existing single-cell atlases offers the opportunity to uncover high-resolution information on expression patterns across various cell types and time points. To better understand the associations between cell types and diseases, we leveraged previously developed tools to construct a standardized analysis pipeline and systematically explored associations across four single-cell datasets, spanning a range of tissue types, cell types and developmental time periods. We utilized a set of existing tools to identify co-expression modules and temporal patterns per cell type and then investigated these modules for known disease and phenotype enrichments. Our pipeline reveals known and novel putative cell type–disease associations across all investigated datasets. In addition, we found that automatically discovered gene co-expression modules and temporal clusters are enriched for drug targets, suggesting that our analysis could be used to identify novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7d3c40be164f07a47b84f28fda619af27f3f711" target='_blank'>
              Mining single-cell data for cell type–disease associations
              </a>
            </td>
          <td>
            Kevin G Chen, Kathryn O. Farley, Timo Lassmann
          </td>
          <td>2024-09-28</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Background Metastasis is the major cause of cancer‐related mortality. The premetastatic niche is a promising target for its prevention. However, the generality and cellular dynamics in premetastatic niche formation have remained unclear. Aims This study aimed to elucidate the generality and cellular dynamics in premetastatic niche formation. Materials and Methods We performed comprehensive flow cytometric analysis of lung and peripheral immune cells at three time points (early premetastatic, late premetastatic, and micrometastatic phases) for mice with subcutaneous implants of three types of cancer cells (breast cancer, lung cancer, or melanoma cells). The immuno‐cell profiles were then used to predict the metastatic phase by machine learning. Results We found a common pattern of changes in both lung and peripheral immune cell profiles across the three cancer types, including a decrease in the proportion of eosinophils in the early premetastatic phase, an increase in that of regulatory T cells in the late premetastatic phase, and an increase in that of polymorphonuclear myeloid‐derived suppressor cells and a decrease in that of B cells in the micrometastatic phase. Machine learning using immune cell profiles could predict the metastatic phase with approximately 75% accuracy. Discussion Validation of our findings in humans will require data on the presence or absence of micrometastases in patients and the accumulation of comprehensive and temporal information on immune cells. In addition, blood proteins, extracellular vesicles, DNA, RNA, or metabolites may be useful for more accurate prediction. Conclusion The discovery of generalities in premetastatic niche formation allow prediction of metastatic phase and provide a basis for the development of methods for early detection and prevention of cancer metastasis in a cancer type‐independent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f02612a5b53055050e50dcce6e173f615e1f3275" target='_blank'>
              Immune Cell Profiling Reveals a Common Pattern in Premetastatic Niche Formation Across Various Cancer Types
              </a>
            </td>
          <td>
            Shigeaki Sugiyama, Kanae Yumimoto, Keiichi I Nakayama
          </td>
          <td>2024-12-30</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a multifactorial and heterogeneous disease, ranking among the most prevalent malignancies in men. In 2020, there were 1,414,259 new cases of PCa worldwide, accounting for 7.3% of all malignant tumors. The incidence rate of PCa ranks third, following breast cancer and lung cancer. Patients diagnosed with high-grade PCa frequently present with existing or developing metastases, complicating their treatment and resulting in poorer prognoses, particularly for those with bone metastases. Utilizing single-cell RNA sequencing (scRNA-seq), we identified specific malignant cell subtypes that are closely linked to high-grade PCa. By investigating the mechanisms that govern interactions within the tumor microenvironment (TME), we aim to offer new theoretical insights that can enhance the prevention, diagnosis, and treatment of PCa, ultimately striving to improve patient outcomes and quality of life. Methods Data on scRNA-seq was obtained from the GEO database. The gene ontology and gene set enrichment analysis were employed to analyze differential expression genes. Using inferCNV analysis to identify malignant epithelial cells. We subsequently employed Monocle, Cytotrace, and Slingshot packages to infer subtype differentiation trajectories. The cellular communication between malignant cell subtypes and other cells was predicted using the CellChat package. Furthermore, we employed pySCENIC to analyze and identify the regulatory networks of transcription factors (TFs) in malignant cell subtypes. The MDA PCa 2b and VCap cell lines were employed to validate the analysis results through cellular functional experiments. In addition, a risk scoring model was developed to assess the variation in clinical characteristics, prognosis, immune infiltration, immune checkpoint, and drug sensitivity. Results A malignant cell subtype in PCa with high expression of NEFH was identified through scRNA-seq analysis. This subtype was situated at the differentiation terminal, exhibited a higher level of malignancy, and exhibited characteristics that were more prone to advanced tumor lesions. In addition, our research underscored the intricate interactions that exist within the TME, particularly the interaction between PTN secreted by this subtype and fibroblasts via the NCL receptor. This interaction may be closely associated with cancer-associated fibroblasts and tumor progression. Subsequently, we determined that the NEFH+ malignant cell subtype was significantly correlated with the TF IRX4. This TF is linked to a worse prognosis in PCa and may affect disease progression by regulating gene transcription. Our conclusions were additionally verified through cellular experiments. Furthermore, the prognostic model we developed demonstrated satisfactory predictive performance, with gene sets from the high NmRS group facilitating tumor progression and deterioration. The analysis of immune infiltration was instrumental in the development of clinical intervention strategies and patient prognosis. Conclusion By examining the cellular heterogeneity of a unique NEFH+ malignant cell subtype within the PCa microenvironment, we were able to disclose their reciprocal interaction with disease progression. This offers a novel viewpoint on the diagnosis and treatment of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/850edd1c421faf9a184e4ab73c61663fd59239af" target='_blank'>
              Unveiling the NEFH+ malignant cell subtype: Insights from single-cell RNA sequencing in prostate cancer progression and tumor microenvironment interactions
              </a>
            </td>
          <td>
            Jieya Wang, Fu Zhao, Qiang Zhang, Zhoujie Sun, Zhikai Xiahou, Changzhong Wang, Yan Liu, Zongze Yu
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Immunotherapy represents a groundbreaking therapeutic approach, based on the immune system’s intrinsic capacity to interfere with tumor progression, that opens the horizons in the treatment of endometrial cancer. However, the clinical efficacy of immunotherapy is hampered by the development of resistance in patients. The resistance to immunotherapy is multifactorial mechanism, encompassed genetic and epigenetic alterations in tumor cells modulating immune checkpoint molecules, resulted in escaping immune surveillance. The tumor microenvironment can orchestrate an immunosuppressive milieu, attenuating the immune response and facilitating tumor progression. To overcome immunotherapeutic resistance in endometrial cancer we must bring to light the mechanisms of intricate interplay between neoplastic cells, the host immune system, and the tumor microenvironment. The identification of predictive biomarkers for immunotherapeutic response and the innovative agents capable of reversing resistance pathways must be developed. Our review summarizes accumulated data on the role of cells of the tumor microenvironment and their regulatory molecules in the mechanisms underlying therapeutic effects of immune checkpoint inhibitors, including resistance to therapy. Major question we raise here – which group of patients is the most favorable to achieve durable immunotherapy response in endometrial cancer?">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cd4ff458f46525f9be97feafd21ef2784338c06" target='_blank'>
              Predicting immunotherapy efficacy in endometrial cancer: focus on the tumor microenvironment
              </a>
            </td>
          <td>
            L. Tashireva, Irina V. Larionova, N. A. Ermak, A. Maltseva, Ekaterina I. Livanos, A. Kalinchuk, M. Stakheyeva, L. Kolomiets
          </td>
          <td>2025-01-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background Vein grafts are commonly employed in revascularisation surgery for multivessel coronary artery disease, yet neointimal hyperplasia (NIH) remains a critical impediment to the long‐term patency of these grafts. Despite this, effective methods to precisely identify and target interventions for the neointima are still inadequate. Methods In this study, Sprague–Dawley (SD) rats were used to establish an external jugular vein transplantation model, and the NIH pathophysiological process was tracked across 11 time points (0–35 days) using various histological stains. Spatial transcriptomics was performed on normal veins and 19‐day grafts to explore gene expression in neointimal regions. Immunohistochemical analysis identified neointima‐specific markers, while NIH progression was assessed in SD rats with four and a half LIM domains protein 1 (Fhl1) knockout and in human saphenous veins (HSV) with adenovirus‐mediated Fhl1 overexpression. Results Typical neointimal formation commenced by day 11 postgrafting and peaked at day 19. Neointimal cells originated from newly generated α‐SMA(+) repair cells located outside the grafted vein, displaying a hybrid fibroblast‐smooth muscle cell phenotype. Spatial transcriptomics identified stable and sustained Fhl1 expression within the neointima throughout the entire NIH phase. Systemic knockout of Fhl1 in SD rats via the phosphoinositide 3‐kinase pathway exacerbated graft inflammation, heightened cell proliferation, and accelerated NIH. Conversely, FHL1 overexpression in cultured HSV suppressed NIH. Conclusion These findings indicate that, following grafting into the arterial system, the newly formed repair cells external to the grafted vein play a pivotal role in NIH, with neointimal cells exhibiting stable and continuous Fhl1 expression. Fhl1 serves as a protective factor against NIH both in vivo and in HSV, likely due to its anti‐inflammatory and anti‐proliferative effects. Key points This study firstly used spatial transcriptomics technique to analyse the neointima and generated a specific neointimal transcriptomic atlas. Fhl1 exhibits specific and stable expression in the spatial region of the neointima. It has thus far the highest enrichment of expression in the neointima in NIH phases, suggesting that it is a prominent molecular biomarker of neointima. We generated rats with a Fhl1 deletion and found that insufficient Fhl1 expression caused an increase in the severity of vascular inflammation and proliferation during neointimal hyperplasia. Adenovirus‐mediated FHL1 overexpression in human saphenous vein have beneficial effects in preventing neointimal hyperplasia. These highlight its potential as a therapeutic target for mitigating vein graft failure associated with cardiovascular procedures. Spatial transcriptomics profiles and morphological observations demonstrated that a newly generated cell population outside the grafted vein with hybrid phenotype between SMCs and fibroblasts contributes to neointimal formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a9f540b21618eabf1f7a31ec78a413b6ce43cc9" target='_blank'>
              Fhl1, a new spatially specific protein, regulates vein graft neointimal hyperplasia
              </a>
            </td>
          <td>
            Chaoqun Wang, Jiantao Chen, Zicong Feng, B. Jian, Suiqing Huang, Kangni Feng, Haoliang Liu, Zhuoming Zhou, Ziyin Ye, Jing Lu, M. Liang, Zhongkai Wu
          </td>
          <td>2024-12-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background Immunosuppressive phenotype compromised immunotherapy efficacy of hepatocellular carcinoma. Tumor cells intrinsic mitochondria dynamics could pass effects on the extracellular microenvironment through mtDNA stress. PGAM5 anchors at mitochondria and regulates mitochondria functions. We aim to explore whether the regulation of tumor-intrinsic PGAM5 on mitochondria affects tumor-infiltrating immune cells in the microenvironment and whether tumor-intrinsic PGAM5 can be a therapeutic target to enhance the immunotherapy efficacy of hepatocellular carcinoma (HCC). Methods We analyzed the correlation of PGAM5 expression and immune cells infiltration using Gene Expression Omnibus (GEO) and The Cancer Genome Atlas Liver Hepatocellular Carcinoma (TCGA-LIHC) data sets based on cibersort algorithm and tumor-tissue arrays from two independent cohorts. To further validate our findings, we established subcutaneous and orthotopic mouse HCC models with tumor-intrinsic Pgam5 deficiency and analyzed tumor-infiltrating immune cells by flow cytometry and single-cell RNA sequencing. Mechanistically, we established an in vitro co-culture system and analyzed proteomics data to find out the bridge between tumor cell PGAM5 and tumor-associated macrophages (TAMs) in the microenvironment. Immunofluorescence, chromatin-immunoprecipitation, ELISA, mass spectrometry were conducted to explore the molecular pathway. Macrophages were depleted to investigate whether the effects of tumor-intrinsic PGAM5 on TAMs could affect immunotherapy efficacy in HCC orthotopic and subcutaneous mouse models. Results PGAM5 expression in tumor was positively correlated with M2-phenotype TAM infiltration in patients with both HCC and mouse HCC tumor models. High tumor-intrinsic PGAM5 expression promoting M2 TAMs infiltration correlated with poor clinical-pathological characteristics and prognosis in patients with HCC. Disruption of tumor-intrinsic Pgam5 reduced TAM M2 polarization and inhibited HCC tumor growth in tumor-bearing mice. Mechanistically, in HCC cells PGAM5 deficiency inhibited mitochondria fission by promoting TRIM28 binding with DRP1, which increased ubiquitination and degradation of DRP1. Tumor-intrinsic PGAM5 deficiency mediated mitochondria fusion and reduced cytosolic mtDNA stress which attenuated TLR9 activation and downstream NF-κB-regulated CCL2 secretion. Furthermore, disruption of tumor-intrinsic Pgam5 significantly facilitated CD8+ T cells activation and improved anti-programmed cell death protein-1 therapeutic efficacy with macrophages depletion compromising synergistic antitumor immune response. Conclusion Our results shed light on the effect of tumor mitochondria dynamics on TAMs in tumor microenvironment. Tumor-intrinsic PGAM5 can be a therapeutic target to improve immunotherapy efficacy in patients with HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57038f9d670a2014b6380167dbf39b47021effa4" target='_blank'>
              Disruption of tumor-intrinsic PGAM5 increases anti-PD-1 efficacy through the CCL2 signaling pathway
              </a>
            </td>
          <td>
            Xiaoying Wei, Hong Wang, Huiquan Liu, Jianguo Wang, Peijie Zhou, Xiaoyang Li, Yuanlong He, Yan Li, Dong Han, Ting Mei, Yuwen Wang, Ziye Li, Junhao Ning, Zilong Xu, Anlin Wang, Yixuan Li, Jingjing Cheng, Dong Qian
          </td>
          <td>2025-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Epithelial cancers are typically heterogeneous with primary prostate cancer being a typical example of histological and genomic variation. Prior studies of primary prostate cancer tumour genetics revealed extensive inter and intra-patient genomic tumour heterogeneity. Recent advances in machine learning have enabled the inference of ground-truth genomic single-nucleotide and copy number variant status from transcript data. While these inferred SNV and CNV states can be used to resolve clonal phylogenies, however, it is still unknown how faithfully transcript-based tumour phylogenies reconstruct ground truth DNA-based tumour phylogenies. We sought to study the accuracy of inferred-transcript to recapitulate DNA-based tumour phylogenies. We first performed in-silico comparisons of inferred and directly resolved SNV and CNV status, from single cancer cells, from three different cell lines. We found that inferred SNV phylogenies accurately recapitulate DNA phylogenies (entanglement = 0.097). We observed similar results in iCNV and CNV based phylogenies (entanglement = 0.11). Analysis of published prostate cancer DNA phylogenies and inferred CNV, SNV and transcript based phylogenies demonstrated phylogenetic concordance. Finally, a comparison of pseudo-bulked spatial transcriptomic data to adjacent sections with WGS data also demonstrated recapitulation of ground truth (entanglement = 0.35). These results suggest that transcript-based inferred phylogenies recapitulate conventional genomic phylogenies. Further work will need to be done to increase accuracy, genomic, and spatial resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79d56f0a1a843a737c66bbddbc1e662cc71b163e" target='_blank'>
              Clonal phylogenies inferred from bulk, single cell, and spatial transcriptomic analysis of epithelial cancers
              </a>
            </td>
          <td>
            Andrew Erickson, Sandy Figiel, Timothy Rajakumar, Srinivasa Rao, Wencheng Yin, D. Doultsinos, Anette Magnussen, Reema Singh, Ninu Poulose, Richard J. Bryant, Olivier Cussenot, Freddie C Hamdy, Dan Woodcock, I. Mills, Alastair D. Lamb
          </td>
          <td>2025-01-03</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/456d5e0f44d3a56861aef434239933c23d1853a0" target='_blank'>
              Single-cell RNA sequencing analysis reveals the dynamic changes in the tumor microenvironment during NMIBC recurrence.
              </a>
            </td>
          <td>
            Ziang Chen, Tianxiang Zhang, Weijian Li, Jiawei Hu, Y. Ou, Fangdie Ye, Jinhao Zhang, Haowen Jiang, Shenghua Liu
          </td>
          <td>2024-12-04</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42792eb6b1b9dc9fadf7e40b88589b27ef00da73" target='_blank'>
              Intra-tumoral sphingobacterium multivorum promotes triple-negative breast cancer progression by suppressing tumor immunosurveillance
              </a>
            </td>
          <td>
            Zhikai Mai, Liwu Fu, Jiyan Su, K. To, Chuansheng Yang, Chenglai Xia
          </td>
          <td>2025-01-08</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The three most prevalent malignant skin tumors are basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and cutaneous melanoma (CM). While BCC and SCC generally exhibit low levels of malignancy, some tumors can become invasive and metastasize. In contrast, CM is the most malignant and lethal form of skin cancer among these three. In recent years, the increasing incidence of skin tumors has garnered significant attention within the medical community. The advent of single-cell RNA sequencing (scRNA-seq) technology has revolutionized the analysis of the tumor microenvironment (TME) in skin tumors, offering novel insights for clinical research. In this review, we provide a concise introduction to scRNA-seq technology and its application in delineating the heterogeneity of the TME in skin tumors, elucidating the mechanisms of tumor progression and pathogenicity, and uncovering new therapeutic targets. Our review offers a comprehensive overview for researchers, offering insights that may advance the understanding of the skin TME in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c806552e713787b83ea0933cd31a52d109ce5cf9" target='_blank'>
              Advances and applications of single-cell RNA sequencing in deciphering the tumor microenvironment of malignant skin tumors
              </a>
            </td>
          <td>
            Yiting Feng, Lanlan Liu, Chunlan Hu, Jie Sun, Mingzhu Yin, Xin Li
          </td>
          <td>2025-01-10</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7713144e1809faa54b6475ca46362db89d138cc3" target='_blank'>
              In vitro model of retinoblastoma derived tumor and stromal cells for tumor microenvironment (TME) studies
              </a>
            </td>
          <td>
            Emily Alefeld, André Haase, Dario Van Meenen, Bettina Budeus, Oliver Dräger, Natalia Miroschnikov, Saskia Ting, Deniz Kanber, Eva Biewald, Nikolaos Bechrakis, Nicole Dünker, M. Busch
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13e8f0b27ff9601215e0751c29f5ce6a0e2b2d75" target='_blank'>
              Metabolism and spatial transcription resolved heterogeneity of glutamine metabolism in cervical carcinoma
              </a>
            </td>
          <td>
            Qian Liu, Jiayu Zhu, Guzalinuer Abulizi, A. Hasim
          </td>
          <td>2024-12-06</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 The reactivation of neurodevelopmental programs in cancer highlights parallel biological processes that occur in both normal development and brain tumors. Achieving a deeper understanding of how dysregulated developmental factors play a role in the progression of brain tumors is therefore crucial for identifying potential targets for therapeutic interventions. Single-cell RNA sequencing (scRNA-Seq) provides an opportunity to understand how developmental programs are dysregulated and reinitiated in brain tumors at single-cell resolution. The aim of this study is to identify developmental origins of brain tumors using scRNA-Seq data.



 Here, we introduce COORS (Cell Of ORigin like CellS), a computational tool trained on developmental human brain single-cell datasets that annotates “developmental-like” cell states in brain tumors. COORS leverages cell type-specific multilayer perceptron models and incorporates a developmental cell type tree that reflects hierarchical relationships and models cell type probabilities.



 Applying COORS to various brain cancer datasets, including medulloblastoma (MB), glioma, and diffuse midline glioma (DMG), we identified developmental-like cells that represent putative cells of origin in these tumors. Our method provides both cell of origin classification and cell age regression, offering insights into the developmental cell types of tumor subgroups. COORS identified outer radial glia (oRG) developmental cells within IDHWT glioma cells whereas oligodendrocyte precursor cells (OPCs) and neuronal-like cells in IDHMut. Interestingly, IDHMut subgroup cells that map to OPC show bimodal distributions, that are both early and late weeks in development. Furthermore, COORS offers a valuable resource by providing novel markers linked to developmental states within of MB, glioma and DMG tumor subgroups.



 Our work adds to our cumulative understanding of brain tumor heterogeneity and helps pave the way for tailored treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e403b1c16cfb5c2e03317b6d1828fc8f0bdaf21" target='_blank'>
              Inferred Developmental Origins of Brain Tumors from Single Cell RNA-Sequencing Data
              </a>
            </td>
          <td>
            Su Wang, R. Curry, Malcolm F McDonald, Hyun Yong Koh, Anders W Erickson, Claudia L Kleinman, Michael D. Taylor, Ganesh Rao, Benjamin Deneen, Arif O. Harmanci, A. Harmanci
          </td>
          <td>2025-01-23</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background/Objectives: Liquid biopsy methods have gained prominence as minimally invasive tools to improve cancer treatment outcomes. Circulating tumor cells (CTCs) offer valuable insights into both primary and metastatic lesions. However, validating the CTC test results requires confirmation that the detected cells originate from cancer tissue. While studies have identified CTCs in colorectal cancer (CRC) patients using molecular markers, simultaneous validation of their cancer tissue origin remains unexplored. Methods: This study introduces a simple approach to detect adenomatous polyposis coli (APC) gene abnormalities alongside established CTC markers using a molecular imaging flow cytometer (MI-FCM). Given that APC gene abnormalities occur in 60-70% of CRC patients, their detection serves as strong evidence of cancer origin. Results: Our method achieved 92% concordance with DNA sequence analysis of tumor-derived cells. In a proof-of-concept study using 5 mL of whole blood from CRC patients, we observed a high frequency of cells exhibiting APC abnormalities, cytokeratin (CK), and vimentin (Vim) expression. Extending the study to 80 CRC patients across pathological stages I-IV confirmed CK and Vim as valid CTC markers. Three distinct cell populations were identified in blood: CK+/Vim-, CK+/Vim+, and CK-/Vim+. CTC number and frequency increased progressively with cancer stage. Conclusions: This is the first report demonstrating CK and Vim as effective markers for direct CTC detection in CRC patients. Our findings provide evidence-based validation of CTC markers, offering new insights and advancing approaches for patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/390f8e6ef574e7d75ad707fb30c643ea89661bb9" target='_blank'>
              Detection and Characterization of Circulating Tumor Cells in Colorectal Cancer Patients via Epithelial-Mesenchymal Transition Markers.
              </a>
            </td>
          <td>
            Yusuke Takahashi, Yuichi Ijiri, S. Fujino, Nakhaei Elnaz, Ayuko Kishimoto, Kentaro Shirai, Shigeki Iwanaga, Masatoshi Yanagida, A. A. S. Bhagat, Norikatsu Miyoshi
          </td>
          <td>2025-01-18</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c81eee6b668168bd971313aa0fe51bd578f98e" target='_blank'>
              Meta-analyses of mouse and human prostate single-cell transcriptomes reveal widespread epithelial plasticity in tissue regression, regeneration, and cancer
              </a>
            </td>
          <td>
            Luis C. Aparicio, Laura Crowley, John R. Christin, Caroline J. Laplaca, H. Hibshoosh, Raul Rabadan, Michael M. Shen
          </td>
          <td>2025-01-17</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Cancer, a prominent global health concern, involves the uncontrolled growth and dissemination of abnormal cells, forming tumors that can infiltrate nearby tissues and metastasize to distant sites. Early and precise tumor detection and treatments are crucial to impede their spread and mitigate disease progression. This study presents the development of a scoring system that harnesses single-cell RNA sequencing (scRNA-seq) data to predict the suitability of solid tumors, specifically glioblastoma (GBM) and lymphoma, for advanced cell therapies such as CAR-T, NK cell, and TIL therapies. By integrating key cellular factorsincluding tumor-infiltrating cells, immunosuppressive cells, and immune escape markersthis system provides a quantitative assessment of a tumors potential responsiveness to these therapies. Validation with patient-derived samples has demonstrated the system's potential to improve precision in therapy selection, offering a promising tool for personalized oncology. This research not only aims to enhance the effectiveness of cell therapies for cancer patients but also lays the groundwork for applying similar methodologies to other cancer types, advancing the integration of precision medicine in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa40086c8127d386e515ff58d140655d2ef491d" target='_blank'>
              A Scoring System that Leverages Single Cell RNA-Sequencing Data to Predict Cell Therapy Outcome in Solid Tumor Treatment
              </a>
            </td>
          <td>
            Yuwen Li
          </td>
          <td>2024-12-26</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="INTRODUCTION Alzheimer’s disease (AD), Dementia with Lewy bodies (DLB), and Parkinson’s disease (PD) represent a spectrum of neurodegenerative disorders (NDDs). Here, we performed the first direct comparison of their transcriptomic landscapes. METHODS We profiled the whole transcriptomes of NDD cortical tissue by snRNA-seq. We used computational analyses to identify common and distinct differentially expressed genes (DEGs), biological pathways, vulnerable and disease-driver cell subtypes, and alteration in cell-to-cell interactions. RESULTS The same vulnerable inhibitory neuron subtype was depleted in both AD and DLB. Potentially disease-driving neuronal cell subtypes were present in both PD and DLB. Cell-cell communication was predicted to be increased in AD but decreased in DLB and PD. DEGs were most commonly shared across NDDs within inhibitory neuron subtypes. Overall, we observed the greatest transcriptomic divergence between AD and PD, while DLB exhibited an intermediate transcriptomic signature. DISCUSSION These results help explain the clinicopathological spectrum of this group of NDDs and provide unique insights into the shared and distinct molecular mechanisms underlying the pathogenesis of NDDs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377545b1eb6117f0f10ff3c8fe9499887d75cc9d" target='_blank'>
              Comparative mapping of single-cell transcriptomic landscapes in neurodegenerative diseases
              </a>
            </td>
          <td>
            E. K. Shwab, Zhaohui Man, Daniel C. Gingerich, Julia E. Gamache, M. Garrett, G. Serrano, T. Beach, Gregory E. Crawford, Allison E Ashley-Koch, O. Chiba-Falek
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8645f518787a0ca20fba12896526944d30b5806b" target='_blank'>
              Development of patient-derived lymphomoids with preserved tumor architecture for lymphoma therapy screening
              </a>
            </td>
          <td>
            A. Santamaria-Martínez, J. Epiney, Divyanshu Srivastava, Daniele Tavernari, Marco Varrone, Dina Milowich, Igor Letovanec, Thorsten Krueger, Rafael Duran, Giovanni Ciriello, A. Cairoli, E. Oricchio
          </td>
          <td>2024-12-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17b53b2d6763b6a8cc385c262233a02ff50e016f" target='_blank'>
              Tissue-resident immune cells: from defining characteristics to roles in diseases
              </a>
            </td>
          <td>
            Jia Li, Chu Xiao, Chunxiang Li, Jie He
          </td>
          <td>2025-01-17</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer spheroids are spherical, three-dimensional (3D), in vitro assemblies of cancer cells, which are gaining importance as a useful model in cancer behavior studies. Designed to simulate key features of the in vivo tumor microenvironment, spheroids offer reliable insights for drug screening and testing applications. We observed contrasting phenotypes in 3D cervical cancer (CC) cultures. Thus, in this study, we compared the proteomes of 3D and traditional two-dimensional (2D) cultures of CC cell lines, HeLa, SiHa, and C33A. When cultured in in-house poly-(2-hydroxyethyl methacrylate)-coated plates under conditions suitable for 3D spheroid formation, these CC cell lines yielded spheroids exhibiting different features. Proteomic analysis of cells cultured in 2D and 3D cultures revealed similar protein profiles but remarkable differences in the expression levels of some proteins. In SiHa and C33A cells, the upregulation of key proteins required for spheroid formation was insufficient for the formation of compact spheroids. In contrast, HeLa cells could form compact spheroids because they upregulated the proteins, including cadherin-binding, cytoskeleton, and adhesion proteins, necessary for spheroid formation during the remodeling process. Overall, this study unravels the mechanisms underlying the formation of spheroids in the commonly used CC cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/815cc2d468c5e900a469f3a75fe62c8fb5f3a378" target='_blank'>
              Proteomic Analysis Reveals Cadherin, Actin, and Focal Adhesion Molecule-Mediated Formation of Cervical Cancer Spheroids
              </a>
            </td>
          <td>
            P. Molika, K. Leetanaporn, W. Chiangjong, Pongsakorn Choochuen, R. Navakanitworakul
          </td>
          <td>2024-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4843e60a35dd8061e986640d8a344bfa3d97b50" target='_blank'>
              Cancer Clones Revised
              </a>
            </td>
          <td>
            L. Laplane
          </td>
          <td>2024-12-23</td>
          <td>Biological Theory</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6984511f4cd3709e0746596a9a57c5a7832bd7" target='_blank'>
              Ovarian tumor cells gain competitive advantage by actively reducing the cellular fitness of microenvironment cells.
              </a>
            </td>
          <td>
            E. Madan, António M Palma, Vignesh Vudatha, Amit Kumar, P. Bhoopathi, Jochen Wilhelm, T. Bernaś, P. Martin, Gaurav Bilolikar, Aenya Gogna, M. L. Peixoto, Isabelle Dreier, Thais Fenz Araujo, Elena Garre, Anna Gustafsson, K. Dorayappan, Narsimha Mamidi, Zhaoyu Sun, Michail Yekelchyk, Davide Accardi, Amalie Lykke Olsen, Lin Lin, Asaf Ashkenazy Titelman, Michael Bianchi, Phil Jessmon, Elnaz Abbasi Farid, A. Pradhan, L. Neufeld, E. Yeini, Santanu Maji, Christopher J Pelham, Hyobin Kim, Daniel Oh, Hans Olav Rolfsnes, Rita C Marques, Amy Lu, Masaki Nagane, Sahil Chaudhary, Kartik Gupta, Keshav C Gogna, Ana Bigio, Karthikeya Bhoopathi, Padmanabhan Mannangatti, K. G. Achary, Javed Akhtar, S. Belião, Swadesh K. Das, I. Correia, Cláudia L da Silva, A. Fialho, M. Poellmann, Kaila Javius-Jones, Adam M Hawkridge, Sanya Pal, Kumari S Shree, E. Rakha, Sambhav Khurana, Gaoping Xiao, Dongyu Zhang, Arjun Rijal, Charles Lyons, Steven R. Grossman, David P. Turner, R. Pillappa, Karanvir Prakash, Gaurav Gupta, Gary L W G Robinson, Jennifer E Koblinski, Hongjun Wang, Gita Singh, Sujay Singh, Sagar Rayamajhi, M. Bacolod, Hope Richards, Sadia Sayeed, Katherine P Klein, David Chelmow, R. Satchi‐Fainaro, K. Selvendiran, Denise Connolly, F. Thorsen, Rolf Bjerkvig, K. Nephew, Michael O Idowu, M. Kühnel, Christopher Moskaluk, Seungpyo Hong, William L Redmond, Göran Landberg, Antonio Lopez-Beltran, Andrew Poklepovic, Arun J Sanyal, Paul B Fisher, George M. Church, U. Menon, Ronny Drapkin, Andrew K Godwin, Yonglun Luo, Maximilian Ackermann, A. Tzankov, Kirsten D. Mertz, Danny Jonigk, Allan Tsung, David Sidransky, Jose Trevino, Arturo P Saavedra, Robert Winn, Kyoung Jae Won, Eduardo Moreno, Rajan Gogna
          </td>
          <td>2024-12-09</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da415d9954dd04158cfcfaa83dc1880ca38c0523" target='_blank'>
              Single-cell and spatial transcriptomics reveal SPP1-CD44 signaling drives primary resistance to immune checkpoint inhibitors in RCC
              </a>
            </td>
          <td>
            Junfeng Zhang, Qingyan Peng, Jin Fan, Fuzhong Liu, Hongbo Chen, X. Bi, Shuai Yuan, Wei Jiang, Ting Pan, Kailing Li, Sihai Tan, Peng Chen
          </td>
          <td>2024-12-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c214c85284c0f7c6d4f85e6f9096b58eba34d42" target='_blank'>
              Dissecting tumor transcriptional heterogeneity from single-cell RNA-seq data by generalized binary covariance decomposition.
              </a>
            </td>
          <td>
            Yusha Liu, P. Carbonetto, Jason Willwerscheid, Scott A. Oakes, Kay F Macleod, Matthew Stephens
          </td>
          <td>2025-01-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="We conducted an investigation into the characteristics of single-cell differentiation data in gliomas, with a focus on developing DAPK1-based prognostic markers to predict patient outcomes. Dysregulated expression of DAPK1 has been associated with the invasive behavior of various malignancies, including gliomas. However, the precise role and underlying mechanisms of DAPK1 in gliomas remain inadequately understood.We performed analyses on RNA-seq and microarray datasets from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO), in addition to single-cell RNA sequencing (scRNA-seq) data from glioma patients available in GEO. Utilizing the Seurat R package, we identified gene clusters associated with survival from the scRNA-seq data. Prognostic models were developed using LASSO and stepwise regression algorithms. Furthermore, we assessed the predictive potential of these genes within the immune microenvironment and their relevance in immunotherapy contexts.Our scRNA-seq data analysis revealed 32 distinct cell clusters corresponding to 10 cell types. Through dimensionality reduction and clustering, we identified three glial cell subpopulations based on their differentiation trajectories. DAPK1, serving as a marker gene for the terminal subpopulation, exhibited an association with poor prognosis.DAPK1-based prognostic models show promise for accurately predicting outcomes in glioblastoma and glioma. An in-depth examination of DAPK1’s specific mechanisms in glioblastoma could elucidate its role in immunotherapy response. Targeting the DAPK1 gene may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fd67f17577e9d1b4649b5fe0ecd703f94585d" target='_blank'>
              Single-cell sequencing uncovers the mechanistic role of DAPK1 in glioma and its diagnostic and prognostic implications
              </a>
            </td>
          <td>
            , Yan-Yu Ding, Si-Guo Zhao, Jie-Hui Zhao, Yu Gu, , Fang Zhang, 
          </td>
          <td>2025-01-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genetic modifications using CRISPR-Cas9 have revolutionized cancer research and other pre-clinical studies. Exceptionally, these efficient tools are inadequate in a few disease models and cell lines due to the aberrant differentiation states and the accumulation of excessive somatic mutations that compromise the robustness of viral gene delivery and stable transduction. A couple of B lymphoma cell lines fall into this category where lentiviral transfection becomes inefficient and exhibits variable efficiency. Additionally, lentiviral delivery requires high biosafety levels. To address this challenge, we have developed a two-step strategy that supports CRISPR-Cas9 through lentivirus and murine ecotropic γ-retrovirus. By engineering B lymphoma cell lines to express Cas9 and mCat-1, a specific receptor for ecotropic retroviruses, we enable efficient and safe gene editing through ecotropic γ-retrovirus. We demonstrate the efficacy of this method by generating IgM-deficient B lymphoma cell lines. This innovative approach simplifies protocols, enhances accessibility, and paves the way for standardized gene manipulation of B cell lymphoma models for molecular cell biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a2c45cfa5f916fccda93d3f9c47decafd7f279" target='_blank'>
              Facilitating Gene Editing in Human Lymphoma Cells Using Murine Ecotropic γ-Retroviruses
              </a>
            </td>
          <td>
            Manish Kumar, Eva Gentner-Göbel, Palash Chandra Maity
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Stromal cells are non‐hematopoietic cells that consist of endothelial cells and various mesenchymal cell populations. The composition of the stromal cell compartment is diverse in different organs. Numerous recent studies demonstrated that the lung environment contains heterogeneous mesenchymal stromal cell populations with distinctive genomic signatures and location preferences. Besides their role in supporting organ structure and remodeling tissue, mesenchymal stromal cells fulfill critical immune functions. These stromal cells show alterations during lung fibrosis and infectious disorders like COVID‐19 or flu infection. To date, their identification and isolation were challenging, and most information about their heterogeneity was derived from scRNAseq data. In this protocol, we describe an isolation, comprehensive flow cytometry assessment, and purification strategy for murine lung stromal cells. The described method is optimized for minimizing cell death while keeping a high level of cell purity. This protocol can be also used for ex‐vivo analysis of these cells in downstream functional assays. © 2024 The Author(s). Current Protocols published by Wiley Periodicals LLC. Basic Protocol 1: Isolation of stromal cells from murine lung tissue Basic Protocol 2: Flow cytometry assessment of lung stromal populations Basic Protocol 3: Purification of lung fibroblastic stromal cells Alternate Protocol: Positive selection of fibroblastic stromal cells">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768210ba3f920bfb05c8ef8764e7bd739b49227" target='_blank'>
              Isolation, Purification, and Comprehensive Flow Cytometry Assessment of Lung Stromal Cells
              </a>
            </td>
          <td>
            Sophia Rottmann, Veronika Lukacs-Kornek
          </td>
          <td>2024-12-01</td>
          <td>Current Protocols</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="There is an urgent need of precision medicine for ovarian cancer patients to identify patients who respond to chemotherapy and PARP inhibitors, a therapy targeting homologous recombination deficiency (HRD). Here we established a panel of 37 long-term patient-derived tumor organoids (PDTO) models of various histological subtypes from 224 patients and demonstrated that they mimic the histological and molecular characteristics of original tumors. Screening of chemotherapeutic drugs showed that PDTO exhibit heterogeneous responses, and that response of PDTO from high-grade serous ovarian carcinoma to carboplatin recapitulated patient response to first-line treatment. Additionally, the detection of HRD phenotype of PDTO using functional assay was associated with the results of the HRD test Genomic Instability Scar (GIScar). Although larger-scale investigations are needed to confirm the predictive potential of PDTO, these results provide further evidence of the potential interest of ovarian PDTO for functional precision medicine even if many challenges remain to be addressed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9373b0fdccc45bac208c5469698121e6a4da07c" target='_blank'>
              Long-term patient-derived ovarian cancer organoids closely recapitulate tumor of origin and clinical response
              </a>
            </td>
          <td>
            L. Thorel, E. Dolivet, Pierre-Marie Morice, R. Florent, J. Divoux, M. Perréard, L. Lecouflet, G. Desmartin, C. Marde Alagama, F. Giffard, A. Leconte, J. Lequesne, B. Clarisse, M. Briand, A. Traore, C. Villenet, JP. Meneboo, G. Babin, L. Gaichies, S. Martin-Françoise, JF. Le Brun, R. Rouzier, E. Brotin, C. Denoyelle, N. Vigneron, R. Leman, D. Vaur, L. Castera, C. Blanc-Fournier, N. Elie, B. Plancoulaine, F. Joly, M. Meryet-Figuiere, M. Figeac, L. weiswald, L. Poulain
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma multiforme (GBM) is one of the deadliest and most heterogeneous forms of brain cancer, characterized by its resistance to conventional therapies. Within GBM, a subpopulation of slow-cycling cells, often linked to quiescence and stemness, plays a crucial role in treatment resistance and tumor recurrence. This study aimed to identify novel biomarkers associated with these slow-cycling GBM cells.


METHODS
We utilized The Cancer Genome Atlas (TCGA)-GBM dataset and presented the reproducible bioinformatics analysis for our results.


RESULTS
Our analysis highlighted Membrane-Associated Protein 17 (MAP17) as strongly associated with the slow-cycling phenotype. We found that the protein cargo MAP17 expression is related to mesenchymal signatures and stem cell-related pathways. Also, MAP17 was linked to a distinct metabolic profile, characterized by significant enrichment in pathways related to folate, zinc, and fatty acids. Moreover, the immune cell distribution analysis revealed that MAP17 correlates with key molecular immune processes, including interferon-gamma (IFN-γ) signaling and antigen presentation, as well as immunosuppressive cells like myeloid-derived suppressor cells (MDSCs) and macrophages. MAP17-high tumors also showed elevated expression of several immune checkpoint inhibitors, indicating an immunosuppressive microenvironment.


CONCLUSION
These findings provide insight into the role of MAP17 in quiescence, stemness, and immune evasion, positioning it as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15fc0b51c9baf70fb9322e4160d56c406b168bfc" target='_blank'>
              Glioblastoma Stem Cells: MAP17 as a Novel Predictive Biomarker and Therapeutic Target Associated with Quiescence and Immune Evasion.
              </a>
            </td>
          <td>
            S. Aghamiri, Rada Amin
          </td>
          <td>2025-01-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59e5b41f611d8bd26c1558060c6ef330913ca3b" target='_blank'>
              Pan-cancer drivers of metastasis
              </a>
            </td>
          <td>
            Ryan Lusby, Engin Demirdizen, Mohammed M A Inayatullah, Paramita Kundu, O. Maiques, Ziyi Zhang, Mikkel G Terp, Victoria Sanz-Moreno, Vijay K. Tiwari
          </td>
          <td>2025-01-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/452c65f4f9faa535e4f661db53c177e8e8787cfc" target='_blank'>
              KANK1 promotes breast cancer development by compromising Scribble-mediated Hippo activation
              </a>
            </td>
          <td>
            S. S. Guo, Zhiying Liu, Guan M. Wang, Zhiqi Sun, Kaikai Yu, James P Fawcett, Reinhard Buettner, Bo Gao, Reinhard Fässler
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary Canine oral melanoma is an aggressive cancer in dogs that may respond to new treatments designed to boost the immune system’s ability to fight the disease. In this study, we investigated whether mesenchymal stem cells derived from fat tissue could stimulate the immune system against this cancer. Mesenchymal stem cells are known to produce proteins called cytokines that help support immune functions. We tested whether combining these stem cells with another type of immune cell could inhibit cancer cell viability and migration. We also examined three key interleukins—2, 8, and 12—associated with immune responses to understand how they might reduce cancer cell growth and migration. Our findings showed that mesenchymal stem cells indeed reduced cancer cell migration and survival, although the effects varied depending on the cancer cell type, suggesting that some cells respond better than others. These results could aid in developing new, less invasive therapies that harness the immune system to fight cancer in dogs, potentially reducing the need for more aggressive treatments and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6dfb3b999d343db4306ba7210d51ed8bf9a10aa" target='_blank'>
              Canine Adipose-Derived Mesenchymal Stromal Cells Reduce Cell Viability and Migration of Metastatic Canine Oral Melanoma Cell Lines In Vitro
              </a>
            </td>
          <td>
            Fwu Shing Teng, Patrícia de Faria Lainetti, Mayara Simão Franzoni, Antonio Fernando Leis Filho, Cristina de Oliveira Massoco Salles Gomes, Renée Laufer-Amorim, Rogério Martins Amorim, C. Fonseca‐Alves
          </td>
          <td>2024-12-01</td>
          <td>Veterinary Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="BACKGROUND
To meet their high energy needs, tumor cells undergo aberrant metabolic reprogramming. A tumor cell may expertly modify its metabolic pathways and the differential expression of the genes for metabolic enzymes. The physiological requirements of the host tissue and the tumor cell of origin mostly dictate metabolic adaptation. Ameloblastoma (AB) is a benign odontogenic tumor of epithelial origin. Due to its unrestricted growth potential, local aggressiveness, and high likelihood of recurrence, this condition poses a significant risk to the patient's health. This study aimed to characterize the metabolic heterogeneity at single-cell resolution of AB.


METHODS
Single-cell RNA sequencing (scRNA-seq) was performed on 17,284 cells from three AB donors. Bioinformatic analysis was used to examine differentially expressed genes, subtypes, and regulatory mechanisms when combined with odontogenic keratocyst scRNA-seq data. Based on metabolic pathway gene sets, the metabolic landscape of AB tumor cells was examined.


RESULTS
Using scRNA-seq, we discovered that AB tumor cells had substantial heterogeneity. The biggest contributor to tumor cell metabolic characteristics is determined to be variation in mitochondrial programming and glycolysis. Surprisingly, hypoxia corresponds with both oxidative phosphorylation and glycolysis activity in AB tumor cells at the single-cell level.


CONCLUSION
This study presents a computational framework for defining metabolism using single-cell expression data and identifies oxidative phosphorylation and glycolysis as critical components of metabolism for AB tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c116028597b1c5eaf72b4b7ed869064e15002ef" target='_blank'>
              Metabolic Analysis of Tumor Cells Within Ameloblastoma at the Single-Cell Level.
              </a>
            </td>
          <td>
            Rui-Fang Li, Yi Zhao, Qi-Wen Man
          </td>
          <td>2024-12-31</td>
          <td>Oral diseases</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The molecular features determining the risk of metachronous metastases in clear cell renal cell carcinoma (ccRCC) are poorly defined. Using a systematic tumor transcriptome deconvolution approach, we investigated the genomic and transcriptomic profiles of 192 ccRCC primary tumors with extended clinical follow-up to identify cancer and stromal cell molecular features associated with metastatic risk. At the genomic level, we identified a significantly higher frequency of copy number loss at 1p31-36 in primary tumors that later progressed with metastases. Tumor transcriptome deconvolution identified significant down-regulation of epithelial cell polarity, including PATJ (1p31), and fatty acid metabolism, including CYP4A11 (1p33), in cancer cells of tumors that developed metastatic progression. We developed and benchmarked a compact 5-feature predictive model (5G) that demonstrated improved accuracy over existing ccRCC gene signatures in the prediction of metachronous metastasis risk. Overall, our study highlights convergent genomic and transcriptomic alterations in chromosome 1p, driving dysregulation of epithelial cell polarity and fatty acid metabolism, as putative novel risk factors of metachronous metastasis in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b85898b20987df3b296ae9761d05ca0764e6d312" target='_blank'>
              Convergent genomic and molecular features predict risk of metachronous metastasis in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            M. Naeini, Mengyuan Pang, Neha Rohatgi, Sinem Kadioglu, Umesh Ghoshdastider, R. DiNatale, Roy Mano, A. Hakimi, A. Skanderup
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Tumor growth and progression involve coordinated regulation by internal, microenvironmental, and systemic signals and often display conspicuous sexual dimorphism. The mechanisms governing the integration and coordination of these signals, along with their sex-based differences, remain largely unknown. Using a Drosophila tumor model originating from nonreproductive tissue, we show that female-biased tumor growth involves multifaceted communications among tumor cells, hemocytes, and neuroendocrine insulin-producing cells (IPCs). Notch-active tumor cells recruit hemocytes carrying the tumor necrosis factor–α (TNF-α) homolog Eiger to the tumor microenvironment (TME), activating the c-Jun N-terminal kinase (JNK) pathway in tumor cells, instigating the sexually dimorphic up-regulation of cytokine Unpaired 2 (Upd2). Upd2, in turn, exerts a distal influence by modulating the release of a Drosophila insulin-like peptide (Dilp2) from IPCs. Dilp2 then activates the insulin signaling in the tumor, thereby fostering sexual-dimorphic tumor growth. Together, these findings reveal a relay mechanism involving the TME and systemic signals that collectively control the sexual dimorphism of tumor growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4504e9cb20bc72b2973f3610b4d0ff83ae5829a" target='_blank'>
              Sex-dimorphic tumor growth is regulated by tumor microenvironmental and systemic signals
              </a>
            </td>
          <td>
            Xianfeng Wang, Hongcun Bao, Yi-Chun Huang, Anindita Barua, Chun-Ming Lai, Jie Sun, Youfang Zhou, Fei Cong, Shangyu Gong, Chih-Hsuan Chang, Wu-Min Deng
          </td>
          <td>2024-12-06</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9558944d48747d281f714408c8c756ade188a3" target='_blank'>
              Genetic and epigenetic characterization of sarcoma stem cells across subtypes identifies EZH2 as a therapeutic target
              </a>
            </td>
          <td>
            Edmond O'Donnell, Maria Muñoz, Ryan Davis, Jessica Bergonio, R. L. Randall, Clifford Tepper, Janai R. Carr-Ascher
          </td>
          <td>2025-01-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="INTRODUCTION
Biliary tract cancer (BTC) comprises a clinically diverse and genetically heterogeneous group of tumors along the intra- and extrahepatic biliary system (intrahepatic and extrahepatic cholangiocarcinoma) and gallbladder cancer with the common feature of a poor prognosis, despite increasing molecular knowledge of associated genetic aberrations and possible targeted therapies. Therefore, the search for even more precise and individualized therapies is ongoing and preclinical tumor models are central to the development of such new approaches.


AREAS COVERED
The models described in the current review include simple and advanced in vitro and in vivo models, including cell lines, 2D monolayer, spheroid and organoid cultures, 3D bioprinting, patient-derived xenografts, and more recently, machine-perfusion platform-based models of resected liver specimens. All these models have individual advantages, disadvantages and limitations that need to be considered depending on the desired application.


EXPERT OPINION
In addition to potential cost limitations, availability of BTC cell types, time required for model establishment and growth success rate, the individual models differently reflect relevant characteristics such as tumor heterogeneity, spatial tumor-stroma microenvironment interactions, metabolic and nutritional gradients and immunological interactions. Therefore, a consequent combination of different models may be required to improve clinical study outcomes by strengthening the preclinical data basis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22dfdd087063cb506f430d33a11c5c6121ad3df5" target='_blank'>
              The importance of preclinical models for cholangiocarcinoma drug discovery.
              </a>
            </td>
          <td>
            F. J. Krendl, Florian Primavesi, Rupert Oberhuber, Daniel Neureiter, Matthias Ocker, Dino Bekric, T. Kiesslich, C. Mayr
          </td>
          <td>2025-01-22</td>
          <td>Expert opinion on drug discovery</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Adenocarcinoma of the esophagogastric junction (AEGJ) is a highly aggressive tumor that frequently metastasizes to the liver. Understanding the cellular and molecular mechanisms that drive this process is essential for developing effective therapies. Methods We employed single-cell RNA sequencing to analyze the tumor heterogeneity and microenvironmental landscape in patients with AEGJ liver metastases. This approach enabled us to characterize the diverse cell populations involved in the liver metastatic process. Results Our analysis revealed a significant involvement of fibroblasts and mural cells in AEGJ liver metastasis. We identified a specific fibroblast type in AEGJ liver metastasis and observed distinct gene expression patterns between adenocarcinoma of the esophagogastric junction and other stomach adenocarcinomas. Our study demonstrated high expression of the SFRP2 gene in pericyte cells during the liver metastasis of AEGJ. The incorporation of GEO, TCGA, and immunofluorescence staining of SFRP2 expression enhanced our study. High expression of SFRP2 in pericytes may influence vascular stability and angiogenesis through the Wnt pathway. Conclusion Our study provides novel insights into the cellular interactions and molecular mechanisms that underlie AEGJ liver metastasis. Targeting the identified subtype of fibroblasts or influencing SFRP2 gene expression in pericytes may offer new therapeutic strategies for combating this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76cf1003bffe0097e28dee22754d6f4e46b48db" target='_blank'>
              Integrated single-cell RNA sequencing reveals the tumor heterogeneity and microenvironment landscape during liver metastasis in adenocarcinoma of esophagogastric junction
              </a>
            </td>
          <td>
            Junrui Xu, Ussama Sadiq, Wangruizhi Zhao, Hengbo Xia, Yiwei Liu, Renquan Zhang, Aman Xu
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="While optimizing cancer cell growth conditions, we discovered that cancer stem cells can generate second membranes outside the plasma membranes forming compartments separated from the extracellular matrix. The encapsulating membranes can extend and generate long cytocapsular tubes, wherein multiple cells can migrate. SILAC proteomics of the second cytocapsular membranes identified 400 membrane proteins, and a small subset of them are highly upregulated in cytocapsular cancers compared to normal tissues. The ATP-dependent calcium pump PMCA2 is one of the highest upregulated factors of the cytocapsular membrane, and antibodies serve as biomarkers for malignant tumors, as checked for 293 subtypes of cancers. Cytocapsular tumors have not been described before, possibly because the CC membranes do not exhibit epitopes targeted by conventional methods, and no efforts have been made to search for new cancer specific organelles. Antibodies against PMCA2 can now be used to map cancer evolution pathways in human bodies by comparisons of more than 12 000 annotated specimens from tissue banks worldwide. The current research reveals that the native malignant cancer cell is enclosed in a cytocapsular membrane. With the PMCA2 cancer biomarker available, the development of human cancers can be studied from cancer tissue banks and clinical cancer biopsies with a previously unknown diversity. The emerging knowledge on cancer-driving biomarkers opens doors for new routes in cancer diagnosis, surgery, therapy, and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4e5b795160ba02ecb2832797cada359373d1df" target='_blank'>
              Discovery of the Cytocapsular Membrane as Hallmark of Malignant Tumors.
              </a>
            </td>
          <td>
            Tingfang Yi, G. Wagner
          </td>
          <td>2024-12-16</td>
          <td>Biochemistry</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Cervical cancer is the fourth most common cancer in women globally, and the main cause of the disease has been found to be ongoing HPV infection. Cervical cancer remains the primary cause of cancer-related death despite major improvements in screening and treatment approaches, especially in low- and middle-income nations. Therefore, it is crucial to investigate the tumor microenvironment in advanced cervical cancer in order to identify possible treatment targets. Materials and methods In order to better understand malignant cervical cancer epithelial cells (EPCs), this study used bulk RNA-seq data from UCSC in conjunction with single-cell RNA sequencing data from the ArrayExpress database. After putting quality control procedures into place, cell type identification and clustering analysis using the Seurat software were carried out. To clarify functional pathways, enrichment analysis and differential gene expression were carried out. The CIBERSORT and ESTIMATE R packages were used to evaluate the immune microenvironment characteristics, and univariate and multivariate Cox regression analyses were used to extract prognostic features. Furthermore, assessments of drug sensitivity and functional enrichment were carried out. Results Eight cell types were identified, with EPCs showing high proliferative and stemness features. Five EPC subpopulations were defined, with C1 NNMT+ CAEPCs driving tumor differentiation. A NNMT CAEPCs Risk Score (NCRS) model was developed, revealing a correlation between elevated NCRS scores and adverse patient outcomes characterized by immune evasion. In vitro experiments validated that the prognostic gene PLOD2 significantly enhances proliferation, migration, and invasion of cervical cancer cells. Conclusion This investigation delineated eight cell types and five subpopulations of malignant EPCs in cervical cancer, establishing the C1 NNMT+ CAEPCs as a crucial therapeutic target. The NCRS model demonstrated its prognostic capability, indicating that higher scores are associated with poorer clinical outcomes. The validation of PLOD2 as a prognostic gene highlights its therapeutic potential, underscoring the critical need for integrating immunotherapy and targeted treatment strategies to enhance diagnostic and therapeutic approaches in cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a0fda708b879e8e016206b2e92375962d615799" target='_blank'>
              Single-cell RNA sequencing and immune microenvironment analysis reveal PLOD2-driven malignant transformation in cervical cancer
              </a>
            </td>
          <td>
            Zhiheng Lin, Fengxin Wang, Renwu Yin, Shengnan Li, Yuquan Bai, Baofang Zhang, Chenlin Sui, Hengjie Cao, Dune Su, Lianwei Xu, Honghong Wang
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colon cancer is the second most common cause of cancer death worldwide. Despite advances in the development of new molecular strategies for stratifying patients with colon cancer, many of these patients do not respond adequately to the standard of care. While previous studies have focused on the development of prognostic gene expression signatures, the exploration of predictive signatures to inform treatment decisions remains incomplete. In this study, we leveraged public gene expression datasets to design and experimentally validate a 37-gene expression signature for prognosis in colon cancer patients. We obtained a C-index of 0.732 (0.610-0.853) in four independent studies. Specifically, we discovered that the signature is associated with the mitotic phase of the cell cycle. Furthermore, the signature identified a population of colon cancer patients sensitive to tubulin inhibitor drugs. In particular, we validated in vitro and in vivo the efficacy of paclitaxel, a commonly used tubulin inhibitor in breast cancer treatment, in patient-derived preclinical models. These results highlight the importance of incorporating gene expression signatures to identify new therapeutic options for colon cancer treatment. Furthermore, the identification of alternative treatment options with potentially improved efficacy holds promise for the development of new clinical trials, and reshapes the biomarker-based treatment strategy for second line and refractory colon cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec4c73cdc7ad79204cf105e9ddd7970b91e50e10" target='_blank'>
              Transcriptomic signature reveals sensitivity to tubulin inhibitors in colon cancer
              </a>
            </td>
          <td>
            J. Liñares-Blanco, I. Chicote, Javier Ros, Anna M Alcántara, Jordi Martínez-Quintanilla, Elena Elez, Héctor G Palmer, Carlos Fernandez-Lozano, Jose A Seoane
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Neuroblastoma (NB) remains associated with high mortality and low initial response rate, especially for high-risk patients, thus warranting exploration of molecular markers for precision risk classifiers. Through integrating multiomics profiling, we identified a range of hub genes involved in cell cycle and associated with dismal prognosis and malignant cells. Single-cell transcriptome sequencing revealed that a subset of malignant cells, subcluster 1, characterized by high proliferation and dedifferentiation, was strongly correlated with the hub gene signature and orchestrated an immunosuppressive tumor microenvironment (TME). Furthermore, we constructed a robust malignant subcluster 1 related signature (MSRS), which was an independent prognostic factor and superior to other clinical characteristics and published signatures. Besides, TME differences conferred remarkably distinct therapeutic responses between high and low MSRS groups. Notably, polo-like kinase-1 (PLK1) was one of the most crucial contributors to MSRS and remarkably correlated with malignant subcluster 1, and PLK1 inhibition was effective for NB treatment as demonstrated by in silico analysis and in vitro experiments. Overall, our study constructs a novel molecular model to further guide the clinical classification and individualized treatment of NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33949844ac45f5e41b1ac34112dc1705f6129d2" target='_blank'>
              Multiomics Analysis Reveals Neuroblastoma Molecular Signature Predicting Risk Stratification and Tumor Microenvironment Differences.
              </a>
            </td>
          <td>
            Xing Zhou, Zhaokai Zhou, Xiaohan Qin, Jian Cheng, Yongcheng Fu, Yuanyuan Wang, Jingyue Wang, Pan Qin, Da Zhang
          </td>
          <td>2025-01-06</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4f1962ef1598f22c4d709373f72cc9202dbf33" target='_blank'>
              Mesenchymal stromal cells promote the formation of lung cancer organoids via Kindlin-2
              </a>
            </td>
          <td>
            Zhi-lin Sui, Xianxian Wu, Jiaxin Wang, ShihJye Tan, Chao Zhao, Zhentao Yu, Chuanyue Wu, Xiaoxia Wang, Ling Guo
          </td>
          <td>2025-01-09</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Nongenetic plasticity has emerged as a key driver of cancer drug resistance, yet its precise origins, nature, and consequences are not fully clarified. This review examines technological, computational, and conceptual developments in the nongenetic determinants of drug resistance. We begin by proposing refined definitions of cellular state, fate, and plasticity. We subsequently contextualize the findings from multimodal approaches to investigate plasticity, highlighting how new single-cell lineage-tracing methods provide opportunities for quantitatively capturing state transitions, their timescales and heritability, and how they contribute to resistance mechanisms. We also draw parallels with concepts from developmental biology and microbial persistence research. Next, we cover the role that computational approaches have played in revealing the otherwise latent patterns of heterogeneity that underlie plasticity from complex datasets. We conclude by emphasizing the need for standardized terminology in this rapidly evolving field and the path from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38fadd15ca85570a489d3058bda72d96573ce1d" target='_blank'>
              Navigating Nongenetic Plasticity in Cancer Drug Resistance
              </a>
            </td>
          <td>
            Hanxiao Sun, Nitu Kumari, Madeline E. Melzer, Vadim Backman, Yogesh Goyal
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The study by Grossmann and colleagues uses single-nucleus RNA sequencing in a cohort of matched high-risk neuroblastoma primary tumor samples, obtained from the same patient at diagnosis and definitive surgery, to identify persister cells that survive induction chemotherapy. These persister cells utilize mechanisms of chemoresistance that are both tumor-intrinsic and tumor-extrinsic, are highly dependent on the original genetic profile of the tumor, and represent novel, patient-specific targets to precisely inhibit chemoresistance and disease recurrence. See related article by Grossmann et.al., p. 2387.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17960d164122a139c932c5adc28d8d07f55f0399" target='_blank'>
              Characterization of Persister Cells Provides Insights into Mechanisms of Therapy Resistance in Neuroblastoma.
              </a>
            </td>
          <td>
            Amber B. Wolf, C. P. Reynolds, Eveline Barbieri
          </td>
          <td>2024-12-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite recent advances, improvements to long-term survival in metastatic carcinomas, such as pancreatic or ovarian cancer, remain limited. Current therapies suppress growth-promoting biochemical signals, ablate cells expressing tumor-associated antigens, or promote adaptive immunity to tumor neoantigens. However, these approaches are limited by toxicity to normal cells using the same signaling pathways or expressing the same antigens, or by the low frequency of neoantigens in most carcinomas. Here, we report a fundamentally different strategy for designing safer and more effective anti-cancer therapies through the sensing of cancer-driving biochemical signals and their rewiring to virotherapeutic activation. Specifically, we rationally engineer a RNA vector to self-replicate and cause cytotoxicity in cancer cells exhibiting hyperactive HER2 (ErbB2), but not in normal cells with normal HER2 signaling. Compared to a widely tested virotherapeutic from the same vector family, our hyperactive ErbB2-restricted RNA vector (HERV) exhibits lower toxicity and greater activity against metastatic HER2-positive ovarian cancer in mice, extending survival independently of tumor antigenicity. Most importantly, HERV synergizes with standard-of-care chemotherapy against ovarian cancer metastases in vivo, with 43% of combination-treated subjects surviving for months beyond subjects treated with either therapy alone. Taken together, these results introduce rewiring of cancer-driving signaling pathways to virotherapeutic activation as a strategy for more specific and effective cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519eb18e674bcbdbaf122e51b6002c650afbd522" target='_blank'>
              Rewiring oncogenic signaling to RNA vector replication for the treatment of metastatic cancer
              </a>
            </td>
          <td>
            Xinzhi Zou, Cynthia Zhao, Kevin T. Beier, Chil-Yong Kang, Michael Z. Lin
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Human cancers are heterogeneous. Their genomes evolve from genetically diverse germlines in complex and dynamic environments, including exposure to potential carcinogens. This heterogeneity of humans, our environmental exposures, and subsequent tumours makes it challenging to understand the extent to which cancer evolution is predictable. Addressing this limitation, we re-ran early tumour evolution hundreds of times in diverse, inbred mouse strains, capturing genetic variation comparable to and beyond that found in human populations. The sex, environment, and carcinogenic exposures were all controlled and tumours comprehensively profiled with whole genome and transcriptome sequencing. Within a strain, there was a high degree of consistency in the mutational landscape, a limited range of driver mutations, and all strains converged on the acquisition of a MAPK activating mutation with similar transcriptional disruption of that pathway. Despite these similarities in the phenotypic state of tumours, different strains took markedly divergent paths to reach that state. This included pronounced biases in the precise driver mutations, the strain specific occurrence of whole genome duplication, and differences in subclonal selection that reflected both cancer susceptibility and tumour growth rate. These results show that interactions between the germline genome and the environment are highly deterministic for the trajectory of tumour genome evolution, and even modest genetic divergence can substantially alter selection pressures during cancer development, influencing both cancer risk and the biology of the tumour that develops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caebf07ac0a60cd920fb19088ba70b1c3f57d1ba" target='_blank'>
              Genetic background sets the trajectory of cancer evolution
              </a>
            </td>
          <td>
            Sarah J. Aitken, Frances Connor, Christine Feig, Tim F. Rayner, M. Lukk, Juliet Luft, Stuart Aitken, Claudia Arnedo-Pac, James F. Hayes, Michael D. Nicholson, Vasavi Sundaram, Jan C. Verburg, John Connelly, Craig J. Anderson, Mikaela Behm, Susan Campbell, Maëlle Daunesse, Ailith Ewing, Vera B. Kaiser, Elissavet Kentepozidou, O. Pich, A. Redmond, Javier Santoyo-Lopez, Inés Sentís, Lana Talmane, Paul Flicek, Núria López-Bigas, Colin A. Semple, Martin S. Taylor, D. T. Odom
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5675284f659a316a50b22f64df578d9b883b676a" target='_blank'>
              Single-cell RNA sequencing highlights the unique tumor microenvironment of small cell neuroendocrine cervical carcinoma
              </a>
            </td>
          <td>
            Tianyou Wang, Li Zhang, Song Mei, Bo Wang, Jiaqi Liu, Weiping Yang, Jiongbo Liao, Chao Wang
          </td>
          <td>2025-01-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f2150cfbdcf9bf561de7bbb85cf14321344131" target='_blank'>
              Extrajunctional CLDN10 cooperates with LAT1 and accelerates clear cell renal cell carcinoma progression
              </a>
            </td>
          <td>
            Akifumi Onagi, K. Sugimoto, M. Kobayashi, Yumi Sato, Yasuyuki Kobayashi, Kei Yaginuma, Satoru Meguro, Seiji Hoshi, Jyunya Hata, Yuko Hashimoto, Yoshiyuki Kojima, Hideki Chiba
          </td>
          <td>2024-12-05</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Transformative technologies to sequence tumor genomes at large scale and single-cell resolution have exposed the repertoire of genetic alterations that are present in leukemia genomes, the timing of their acquisition and patterns of their co-occurrence. In parallel, single-cell multi-omics technologies are allowing us to map the differentiation paths and hierarchical structures of malignant cells and giving us a glimpse into hematopoietic development in prenatal life. We propose that interrogating how the genetic evolution, differentiation hierarchy and ontogeny of malignant myeloid cells intersect with each other, using new experimental systems and multimodal technologies, will fuel the next generation of research breakthroughs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f504f6d5de23edcc7a44b61924d8a2be55a1c9e" target='_blank'>
              The crossroads of clonal evolution, differentiation hierarchy and ontogeny in leukemia development.
              </a>
            </td>
          <td>
            Christopher M Sturgeon, Elvin Wagenblast, Franco Izzo, E. Papapetrou
          </td>
          <td>2024-12-09</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 The heterogeneity of ovarian mesenchymal/stromal cells has just been revealed in both mice and humans. However, it remains unclear about the cellular development trace and the intercellular communication network in the whole life of the ovary. In the study, we integrated ours and published single-cell RNA sequencing data from E11.5 (embryonic day 11.5) until M12 (12-month-old) ovaries to show the dynamics of somatic cells along the developmental timeline. The intercellular crosstalk among somatic cell types was depicted with collagen signaling pathway as the most outgoing signals from stromal cells. We identified mesenchymal progenitor cells (CD24+) as the origin of stromal cells. Although their numbers decreased significantly in adults, the cells served as the major signal sender until ovarian senescence. Moreover, the ovarian injury could activate these stem cells and induce stroma remodeling in the aged ovary. Thus, mesenchymal progenitor cells may represent a new strategy to delay ovarian aging in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/738761dd66266e63f55d4f08fe8f7233527bdb3d" target='_blank'>
              Single-cell RNA sequencing reveals the intercellular crosstalk and the regulatory landscape of stromal cells during the whole life of the mouse ovary
              </a>
            </td>
          <td>
            Wan Jiang, Wenya Sun, Yue Peng, Hao Xu, Haonan Fan, Xin Jin, Yue Xiao, Yuxiang Wang, Pin Yang, Wenjie Shu, Jing Li
          </td>
          <td>2024-12-01</td>
          <td>Life Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a00464522656fb99b3714169ed11d185f90a82" target='_blank'>
              A hybrid epithelial-mesenchymal transition program enables basal epithelial cells to bypass stress-induced stasis and contributes to a metaplastic breast cancer progenitor state
              </a>
            </td>
          <td>
            Joseph A. Caruso, Chira Chen-Tanyolac, T. Tlsty
          </td>
          <td>2024-12-18</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="In invasive breast cancer, cell clusters of varying sizes and shapes are embedded in the fibrous extracellular matrix (ECM). Although the prevailing view attributes this structure to increasing disorder resulting from loss of function and dedifferentiation, our findings reveal that it arises through a process of active self-organization driven by cancer cell motility. Simulations and histological analyses of tumours from over 2,000 breast cancer patients reveal that motile, aligned cancer cells within clusters move as active nematic aggregates through the surrounding highly aligned ECM fibres, which form a confining, passive nematic phase. Cellular motion leads to cluster splitting and coalescence. The degree of cluster activity, combined with heterogeneity in cell motility, is reflected in specific scaling behaviours for cluster shape, size distribution, and the distance between cluster boundaries and nematic defects in ECM alignment. Increased activity estimates correlate with tumour progression and are associated with a poorer prognosis for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8e12ce985b910cf49768ffcf8df2c34a1c4178" target='_blank'>
              Self organisation of invasive breast cancer driven by the interplay of active and passive nematic dynamics
              </a>
            </td>
          <td>
            Pablo Gottheil, S. Bhattacharyya, Kolya Lettl, Philip Friedrich, Kilian Roth, Salvador Rivera-Moreno, Mario Merkel, B. Aktas, Igor Sauer, Assal Daneshgar, Jonas Wieland, H. Kubitschke, Anne-Sophie Wegscheider, J. M. Yeomans, Josef A. Käs
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Identifying the genes capable of driving tumorigenesis in different tissues is one of the central goals of cancer genomics. Computational methods that exploit different signals of positive selection in the pattern of mutations observed in genes across tumors have been developed to this end. One such signal of positive selection is clustering of mutations in areas of the three-dimensional (3D) structure of the protein above the expectation under neutrality. Methods that exploit this signal have been hindered by the paucity in proteins with experimentally solved 3D structures covering their entire sequence. Here, we present Oncodrive3D, a computational method that by exploiting AlphaFold 2 structural models extends the identification of proteins with significant mutational 3D clusters to the entire human proteome. Oncodrive3D shows sensitivity and specificity on par with state-of-the-art cancer driver gene identification methods exploiting mutational clustering, and clearly outperforms them in computational efficiency. We demonstrate, through several examples, how significant mutational 3D clusters identified by Oncodrive3D in different known or potential cancer driver genes can reveal details about the mechanism of tumorigenesis in different cancer types and in clonal hematopoiesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f19ed942c2bb363999085861c8459ffabedab4e8" target='_blank'>
              Oncodrive3D: Fast and accurate detection of structural clusters of somatic mutations under positive selection
              </a>
            </td>
          <td>
            Stefano Pellegrini, Ferran Muiños, Núria López-Bigas, A. González-Pérez
          </td>
          <td>2025-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Introduction Ovarian cancer (OC) is the sixth most common malignancy in women and the poor 5-year survival emphasises the need for novel therapies. NK cells play an important role in the control of malignant disease but the nature of tumour-infiltrating and peripheral NK cells in OC remains unclear. Methods Using flow cytometric analysis, we studied the phenotype and function of NK cells in blood, primary tumour and metastatic tissue in 80 women with OC. The cell type contexture of metastatic OC tissue was explored utilising scRNAseq analysis, with a focus on portraying an immunogenic tumour microenvironment and determining the characteristics of a dysfunctional NK cell population. Results The proportion of peripheral NK cells was markedly elevated with a highly activated profile and increased cytotoxicity. In contrast, NK cell numbers in primary tumour and metastasis were substantially reduced, with downregulation of activatory receptors together with elevated PD-1 expression. scRNA-Seq identified 5 NK cell subpopulations along with increased exhausted and immature NK cells within tumour tissue compared to normal tissue. These features were attenuated following chemotherapy where higher levels of activated and cytotoxic NK cells associated with improved disease-free survival. Correlation of NK cell phenotype with clinical outcomes revealed high levels of DNAM-1 expression on tissue-localised and peripheral NK cells to be associated with reduced survival. Expression of PVR, the DNAM-1 ligand, was significantly increased on tumours and DNAM-1 mediated NK cell lysis of primary tumour tissue was observed in vitro. Discussion These findings reveal profound modulation of the tumour tissue and systemic profile of NK cells which likely contributes to the high rates of local progression and metastasis seen with OC. Immunotherapeutic approaches that overcome local immune suppression and enhance DNAM-1-targeted lysis of OC offer the potential to improve disease control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec3263f862b7dea4d3ddaf31f9b8ccd7a6ea974" target='_blank'>
              The emergence of DNAM-1 as the facilitator of NK cell-mediated killing in ovarian cancer
              </a>
            </td>
          <td>
            R. Pounds, W. Croft, Hayden Pearce, Tasnia Hossain, Kavita Singh, J. Bálega, D. Jeevan, S. Sundar, S. Kehoe, J. Yap, Paul Moss, J. Zuo
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Cancer stem cells (CSCs) constitute a small and elusive subpopulation of cancer cells within a tumor mass and are characterized by stem cell properties. Reprogrammed CSCs exhibit similar capability to initiate tumor growth, metastasis, and chemo- and radio-resistance and have similar gene profiles to primary CSCs. However, the efficiency of cancer cell reprogramming remained relatively low. There is limited literature available on the reprogramming of lung cancer cells. Hence, in this study we have conducted reprogramming of human lung cancer cells towards more benign type of cells. Materials and methods The reprogramming process was carried out with the use of STEMCCA vector. We have investigated the gene expression profile of induced CSCs (iCSCs) using the microarray technique. Results The lung iCSCs demonstrate morphology characteristics of induced pluripotent stem cells (iPSCs) and gene expression profile that significantly differ from cells before reprogramming. We have also presented the elevated level of expression of genes associated with the cancer stemness and thus revealed new interesting CSC-like molecular markers. Conclusions These preliminary results demonstrated that the reprogramming process in vitro leads to the remarkable changes in cells at the gene level and has potential to be an alternative method of generating CSC-like cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5e391fa7a69486ab2113454ce0ec04bcbfac417" target='_blank'>
              Can the reprogrammed cancer cells serve as an alternative source of (induced) cancer stem cells?
              </a>
            </td>
          <td>
            E. Stelcer, Małgorzata Blatkiewicz, K. Jopek, W. Suchorska, Marcin Ruciński
          </td>
          <td>2024-12-04</td>
          <td>Reports of Practical Oncology and Radiotherapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Introduction Cancer-associated fibroblasts (CAFs) are a diverse group of cells that significantly contribute to reshaping the tumor microenvironment (TME), and no research has systematically explored the molecular landscapes of senescence related CAFs (senes CAF) in NB. Methods We utilized pan-cancer single cell and spatial transcriptomics analysis to identify the subpopulation of senes CAFs via senescence related genes, exploring its spatial distribution characteristics. Harnessing the maker genes with prognostic significance, we delineated the molecular landscapes of senes CAFs in bulk-seq data. We established the senes CAFs related signature (SCRS) by amalgamating 12 and 10 distinct machine learning (ML) algorithms to precisely diagnose stage 4 NB and to predict prognosis in NB. Based on risk scores calculated by prognostic SCRS, patients were categorized into high and low risk groups according to median risk score. We conducted comprehensive analysis between two risk groups, in terms of clinical applications, immune microenvironment, somatic mutations, immunotherapy, chemotherapy and single cell level. Ultimately, we explore the biological function of the hub gene JAK1 in pan-cancer multi-omics landscape. Results Through integrated analysis of pan-cancer spatial and single-cell transcriptomics data, we identified distinct functional subgroups of CAFs and characterized their spatial distribution patterns. With marker genes of senes CAF and leave-one-out cross-validation, we selected RF algorithm to establish diagnostic SCRS, and SuperPC algorithm to develop prognostic SCRS. SCRS demonstrated a stable predictive capability, outperforming the previously published NB signatures and clinic variables. We stratified NB patients into high and low risk group, which showed the low-risk group with a superior survival outcome, an abundant immune infiltration, a different mutation landscape, and an enhanced sensitivity to immunotherapy. Single cell analysis reveals biologically cellular variations underlying model genes of SCRS. Spatial transcriptomics delineated the molecular variant expressions of hub gene JAK1 in malignant cells across cancers, while immunohistochemistry validated the differential protein levels of JAK1 in NB. Conclusion Based on multi-omics analysis and ML algorithms, we successfully developed the SCRS to enable accurate diagnosis and prognostic stratification in NB, which shed light on molecular landscapes of senes CAF and clinical utilization of SCRS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/900ec2c5a181f8ad61d932465735fe6366505e3c" target='_blank'>
              Pan-cancer single cell and spatial transcriptomics analysis deciphers the molecular landscapes of senescence related cancer-associated fibroblasts and reveals its predictive value in neuroblastoma via integrated multi-omics analysis and machine learning
              </a>
            </td>
          <td>
            Shan Li, Junyi Luo, Junhong Liu, Dawei He
          </td>
          <td>2024-12-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Lung adenocarcinoma (LUAD) is a highly aggressive tumor with one of the highest morbidity and mortality rates in the world. Nucleotide metabolic processes are critical for cancer development, progression, and alteration of the tumor microenvironment. However, the effect of nucleotide metabolism on LUAD remains to be thoroughly investigated. Methods Transcriptomic and clinical data of LUAD were downloaded and organized from TCGA and GEO databases. Genes related to nucleotide metabolism were downloaded from the Msigdb database. Genes associated with LUAD prognosis were identified using univariate COX analysis, and a prognostic risk model was constructed using the machine learning combination of Lasso + Stepcox. The model’s predictive validity was evaluated using KM survival and timeROC curves. Based on the prognostic model, LUAD patients were classified into different nucleotide metabolism subtypes, and the differences between patients of different subtypes were explored in terms of genomic mutations, functional enrichment, tumor immune characteristics, and immunotherapy responses. Finally, the key gene SNRPA was screened, and a series of in vitro experiments were performed on LUAD cell lines to explore the role of SNRPA in LUAD. Result LUAD patients could be accurately categorized into subtypes based on the nucleotide metabolism-related prognostic risk score (NMBRS). There were significant differences in prognosis between patients of different subtypes, and the NMBRS showed high accuracy in predicting the prognosis of LUAD patients. In addition, patients of different subtypes showed significant differences in genomic mutation and functional enrichment and exhibited different anti-tumor immune profiles. Importantly, NMBRS can be used to predict the responsiveness of LUAD patients to immunotherapy. The results of in vitro cellular experiments indicate that SNRPA plays an important role in the development and progression of lung adenocarcinoma. Conclusion This study comprehensively reveals the prognostic value and clinical application of nucleotide metabolism in LUAD. A prognostic signature constructed based on genes related to nucleotide metabolism accurately predicted the prognosis of LUAD patients, and this signature can be used as a guide for LUAD immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f247cf9ee6d123c8afb18b9aa087b0c11d86c73" target='_blank'>
              Integration of single-cell transcriptomics and bulk transcriptomics to explore prognostic and immunotherapeutic characteristics of nucleotide metabolism in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kai Zhang, Luyao Wang, Huili Chen, Lili Deng, Mengling Hu, Ziqiang Wang, Yiluo Xie, Chaoqun Lian, Xiaojing Wang, Jing Zhang
          </td>
          <td>2025-01-08</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Atypical teratoid rhabdoid tumor (ATRT) is the most common malignant brain tumor in infants, and more than 60% of children with ATRT die from their tumor. ATRT is associated with mutational inactivation/deletion of SMARCB1, a member of the SWI/SNF chromatin remodeling complex, suggesting that epigenetic events play a critical role in tumor development and progression. Moreover, disruption of SWI/SNF allows unopposed activity of epigenetic repressors, which contribute to tumorigenicity. We therefore explored the role of the CoREST repressor complex in ATRT. Methods We evaluated the effects of the bifunctional LSD1/HDAC1/2 small molecule CoREST inhibitor, corin, on ATRT tumor cell growth, apoptosis, differentiation, gene expression and chromatin accessibility. Results We found that corin inhibited the growth of ATRT cells regardless of their epigenetic subgroup, and was associated with increased tumor cell apoptosis and differentiation. ATAC-seq showed increases in chromatin accessibility in corin-treated ATRT cells, with changes seen at genes associated with neuronal differentiation and synaptic function. RNA-seq confirmed increased expression of neuronal differentiation genes and decreased DNA replication/cell cycle-associated genes in ATRT cells treated with corin. Corin suppressed orthotopic ATRT tumor growth, leading to significant extension of lifespan. In addition, increased histone acetylation (H3K9ac, H3K27ac) and methylation (H3K4Me1) was seen in corin-treated ATRT orthotopic xenografts, consistent with on-target pharmacodynamics. Conclusion The CoREST inhibitor, corin, suppresses tumor growth, induces differentiation, and promotes apoptosis in ATRT, leading to significantly increased survival of mice bearing ATRT orthotopic xenografts. Our results suggest a potential application of CoREST complex inhibitors in patients with ATRT. Key Points CoREST complex inhibition by corin leads to decreased cell growth and increased apoptosis in ATRT Corin promotes chromatin accessibility and neuronal differentiation in ATRT Corin inhibits tumor growth and extends lifespan in ATRT animal models Importance of the Study Loss of function of SMARCB1 is a hallmark of ATRT which leads to dysfunction of the mammalian SWI/SNF complex and an inability to counteract epigenetic repressor complexes. The CoREST complex functions as a chromatin remodeling complex that represses neuronal differentiation genes during development. Inhibition of the CoREST complex by corin in ATRT leads to decreased tumor cell growth, induction of apoptosis and increased survival of mice bearing ATRT orthotopic xenografts. These changes are associated with increased chromatin accessibility and expression of genes associated with neuronal differentiation. Corin therefore reverses the primary block of differentiation that maintains a stem cell state in ATRT, which contributes to tumorigenesis. These studies significantly improve our understanding of how to therapeutically address the underlying epigenetic drivers of ATRT and support further development of corin for ATRT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024c492e649d7b4c64d548dac81278e98253c902" target='_blank'>
              The CoREST complex inhibitor, corin, leads to decreased tumor growth, increased cellular differentiation and extended lifespan in atypical teratoid rhabdoid tumor xenograft models
              </a>
            </td>
          <td>
            Anupa Geethadevi, Nikhil Vaidya, Robert J. Fisher, Tyler Findlay, Yiming Deng, Khoa Pham, Vikas Kumar, Samuel Beck, Calixto-Hope G Lucas, C. Eberhart, Jinchong Xu, Philip A. Cole, Jeffrey A. Rubens, Marianne Collard, Eric H. Raabe, R. Alani
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024cdb898f44026b61ab427711870c26e06ca884" target='_blank'>
              Identification of Cell Fate Determining Transcription Factors for Generating Brain Endothelial Cells.
              </a>
            </td>
          <td>
            Roya Ramezankhani, J. De Smedt, B. Toprakhisar, Bernard K. van der Veer, Tine Tricot, Gert Vanmarcke, Bradley Balaton, Leo A. van Grunsven, Massoud Vosough, Y. C. Chai, C. Verfaillie
          </td>
          <td>2025-01-24</td>
          <td>Stem cell reviews and reports</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract We performed comprehensive histopathologic, genomic, and epigenomic evaluation of paired initial and recurrent glioblastoma specimens from 106 patients to investigate the molecular evolution and cellular phenotypes underlying differential treatment responses. While TERT promoter mutation and CDKN2A homozygous deletion were early events during gliomagenesis shared by initial and recurrent tumors, most other recurrent genetic alterations (e.g., EGFR, PTEN, NF1) were commonly private to initial or recurrent tumors indicating acquisition later during clonal evolution. Further, glioblastomas exhibited heterogeneous epigenomic evolution with subsets becoming more globally hypermethylated, hypomethylated, or remaining stable. Glioblastoma that underwent sarcomatous transformation at recurrence had inferior survival and were significantly enriched in NF1, TP53, and RB1 alterations and the Mesenchymal epigenetic class. Patients who developed somatic hypermutation following temozolomide treatment had significantly longer interval to disease recurrence and prolonged overall survival, and hypermethylation at four specific CpG sites in the promoter region of MGMT was significantly associated with this development of hypermutation. Finally, an epigenomic evolution signature incorporating change in DNA methylation levels across 347 critical CpG sites was developed that significantly correlated with clinical outcomes. This talk will describe how GBM undergoes heterogeneous genetic, epigenetic, and cellular evolution that underlies prognostically different treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/704f3941c3c62c19e62495b9dd1824a7c49e3146" target='_blank'>
              IL-1 IDHWT GLIOBLASTOMA LONGITUDINAL MOLECULAR EVOLUTION
              </a>
            </td>
          <td>
            C. G. Lucas, N. Al-Adli, J. S. Young, R. Gupta, R. Morshed, J. Wu, A. Ravindranthan, A. Shai, N. Oberheim Bush, J. W. Taylor, J. de Groot, J. Villanueva-Meyer, M. Pekmezci, A. Perry, A. W. Bolen, P. Theodosopoulos, M. Aghi, E. Chang, S. Hervey-Jumper, D. R. Raleigh, A. M. Molinaro, J. Costello, A. A. Diaz, J. Clarke, N. Butowski, J. J. Phillips, S. M. Chang, M. S. Berger, D. A. Solomon
          </td>
          <td>2024-11-29</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="The zebrafish (Danio rerio) is a valuable model organism for studying human biology due to its easy genetic manipulation and small size. It is optically transparent and shares genetic similarities with humans, making it ideal for studying developmental processes, diseases, and drug screening via imaging-based approaches. Solid malignant tumors often contain hypoxic areas that stimulate the release of extracellular vesicles (EVs), lipid-bound structures released by cells into the extracellular space, that facilitate short- and long-range intercellular communication and metastatization. Here we investigate the effects of EVs derived from neuroblastoma (NB), a pediatric solid tumor, on metastatic niche formation using the zebrafish as an in vivo model. Intravascular injection in zebrafish embryos allows a non-invasive visualization of EVs dispersion, uptake, and interactions with host cells. To improve repeatability of our results and ease the injection steps, we used an agarose device replica molded from a custom designed micromilled aluminum mold. We first demonstrated that EVs released under hypoxic conditions promote angiogenesis and are more easily internalized by endothelial cells than those purified from normoxic cells. We also showed that injection of with hypoxic EVs increased macrophages mobilization. We then focused on the caudal hematopoietic tissue (CHT) region of the embryo as a potential metastatic site. After hypoxic EVs injection, we highlighted changes in the expression of mmp-9 and cxcl8b genes. Furthermore, we investigated the ability of NB-derived EVs to prime a metastatic niche by a two-step injection of EVs first, followed by NB cells. Interestingly, we found that embryos injected with hypoxic EVs had more proliferating NB cells than those injected with normoxic EVs. Our findings suggest that EVs released by hypoxic NB cells alter the behavior of recipient cells in the zebrafish embryo and promote metastatic outgrowth. In addition, we demonstrated the ability of the zebrafish embryo to be a suitable model for studying the interactions between EVs and recipient cells in the metastatic process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2525012f5692c0555303a6b2e55ae094fb8a3938" target='_blank'>
              Neuroblastoma-derived hypoxic extracellular vesicles promote metastatic dissemination in a zebrafish model
              </a>
            </td>
          <td>
            Anna Fietta, P. Fusco, Giuseppe Germano, S. Micheli, M. Sorgato, G. Lucchetta, Elisa Cimetta
          </td>
          <td>2024-12-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Genome duplication, critical for cell survival and identity, requires precise origin activation to ensure accurate DNA replication and chromatin structure maintenance. In colorectal cancer cells, prolonged replication stress does not hinder cell proliferation, although the mechanisms driving cancer cell adaptability remain largely unclear. Here, we demonstrate that upon recovery, cancer cells are able to activate new replication origins in distinct domains, causing persistent changes in chromatin accessibility, nuclear morphology, and replication timing, ultimately promoting chromatin instability. Chromatin accessibility, particularly in promoter regions, increased following replication stress in a subset of cells, correlating with altered gene expression and nuclear expansion. Additionally, some genes with enhanced promoter accessibility displayed sustained expression changes, further suggesting a transcriptional shift linked to stress adaptation. Our findings reveal that colorectal cancer cells recover from severe replication stress through new origin activation, a mechanism that not only maintains cell proliferation under stress but may also accelerate tumour heterogeneity. This research underscores replication origin activation as a potential therapeutic target in combating cancer cell resilience. GRAPHICAL ABSTRACT MAIN HIGHLIGHTS Colorectal cancer cells under severe replication stress activate new replication origins upon recovery. The firing of new origins in distinct replication domains results in long-lasting changes in chromatin accessibility, especially in promoter regions, and is associated with alterations in nuclear morphology and replication timing. Increased chromatin accessibility in certain promoter regions correlates with prolonged gene expression changes. The capacity of CRC cells to recover from severe replication stress fosters chromatin instability and tumour heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04cc1950d705dd8a02a8a5d1f21cfa69a97d0c9c" target='_blank'>
              Adaptive replication origin activation alters chromatin dynamics and stability in cancer cells
              </a>
            </td>
          <td>
            Alejandro Pérez-Venteo, Fernando Unzueta, Sonia Feu, P. Llinàs-Arias, Caroline Mauvezin, Neus Agell
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Since its first conceptualization over a century ago, the mesenchymal phenotype has traditionally been viewed as either a transient phase between successive epithelial stages or as a feature of cell types primarily devoted to structural support. However, recent findings in cancer research challenge this limited view, demonstrating that mesenchymal traits and hybrid mesenchymal/epithelial states can mark cancer cells with stem cell properties. By analyzing publicly available single-cell transcriptome datasets from early embryonic stages and adult tissues, this study aims to extend this concept beyond pathological contexts, suggesting that a partial or fully mesenchymal phenotype may represent the morphological expression of undifferentiated and multipotent states in both the developing embryo and adult organs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8b86efcb43fa26b36795e33eead52234f743d4e" target='_blank'>
              Mesenchymal Traits as an Intrinsic Feature of Undifferentiated Cells
              </a>
            </td>
          <td>
            M. Galiè
          </td>
          <td>2024-12-24</td>
          <td>Journal of Developmental Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cell phenotype underlies prostate cancer presentation and treatment resistance and can be regulated by epigenomic features. However, the osteotropic tendency of prostate cancer limits access to metastatic tissue, meaning most prior insights into prostate cancer chromatin biology are from preclinical models that do not fully represent disease complexity. Noninvasive chromatin immunoprecipitation of histones in plasma cell-free in humans may enable capture of disparate prostate cancer phenotypes. Here, we analyzed activating promoter- and enhancer-associated H3K4me2 from cfDNA in metastatic prostate cancer enriched for divergent patterns of metastasis and diverse clinical presentation. H3K4me2 density across prostate cancer genes, accessible chromatin, and lineage-defining transcription factor binding sites correlated strongly with circulating tumor DNA (ctDNA) fraction-demonstrating capture of prostate cancer-specific biology and informing the development of a statistical framework to adjust for ctDNA fraction. Chromatin hallmarks mirrored synchronously measured clinico-genomic features: bone versus liver-predominant disease, serum PSA, biopsy-confirmed histopathological subtype, and RB1 deletions convergently indicated phenotype segregation along an axis of differential androgen receptor activity and neuroendocrine identity. Detection of lineage switching after sequential progression on systemic therapy in select patients indicates potential utility for individualized resistance monitoring. Epigenomic footprints of metastasis-induced normal tissue destruction were evident in bulk cfDNA from two patients. Finally, a public epigenomic resource was generated using a distinct chromatin marker that has not been widely investigated in prostate cancer. These results provide insight into the adaptive molecular landscape of aggressive prostate cancer and endorse plasma cfDNA chromatin profiling as a biomarker source and biological discovery tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15869a6834cce04dbe66dca3a25682ba07c367af" target='_blank'>
              Plasma cell-free DNA chromatin immunoprecipitation profiling depicts phenotypic and clinical heterogeneity in advanced prostate cancer.
              </a>
            </td>
          <td>
            J. Sipola, Aslı D Munzur, Edmond M Kwan, Clara C Y Seo, Benjamin J Hauk, Karan Parekh, Yi Jou Ruby Liao, C. Bernales, G. Donnellan, Ingrid Bloise, Emily Fung, Sarah W. S. Ng, Gang Wang, G. Vandekerkhove, M. Nykter, M. Annala, C. Maurice-Dror, Kim Chi, C. Herberts, Alexander W Wyatt, David Y. Takeda
          </td>
          <td>2024-12-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most prevalent primary malignant brain tumor, remains challenging to treat due to extensive inter- and intra-tumor heterogeneity. This variability demands combination treatments to improve therapeutic outcomes. A significant obstacle in treating GBM is the expression of O6-methylguanine-DNA methyltransferase, a DNA repair enzyme that reduces the efficacy of the standard alkylating agent, temozolomide, in about 50% of patients. This underscores the need for novel, more targeted therapies. Our study investigates the metabolic-epigenetic impact of combining SN-38, a novel topoisomerase inhibitor inducing DNA double-strand breaks, with rabusertib, a checkpoint kinase 1 inhibitor. We identified this synergistic combination through high-throughput drug screening across a panel of GBM cell lines using a cancer drug library combined with SN-38. A secondary metabolic screening with the PEDS algorithm demonstrated a synergistic modulation of purine, one-carbon, and redox metabolism. Furthermore, the combined treatment led to the significant depletion of epigenetically relevant metabolites such as 5-methyl-cytosine, acetyl-lysine, and trimethyl-lysine. Reduced intermediates of the glutathione cycle indicated increased cellular stress following combinatorial treatment. Overall, the combination of SN-38 and rabusertib synergistically disrupts metabolites associated with epigenetic adaptations, leading to cytotoxicity independent of O6-methylguanine-DNA methyltransferase status, thereby underpinning this combination as a promising candidate for combinatorial therapy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed32abd0bdaa8ba2724fd42a8df45084943d6e1" target='_blank'>
              Targeting Metabolic and Epigenetic Vulnerabilities in Glioblastoma with SN-38 and Rabusertib Combination Therapy.
              </a>
            </td>
          <td>
            J. Chiou, Valeria Impedovo, Yen Bao Huynh, Ruggiero Gorgoglione, Luiz O. F. Penalva, A. Lodi, Andrew J. Brenner, Stefano Tiziani
          </td>
          <td>2025-01-08</td>
          <td>International journal of molecular sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer cell lines are the most common in-vitro models for the evaluation of anti-cancer drug sensitivities. Past studies have been conducted to decipher and characterize the pharmacogenomic feature of cell lines based on other omics data, such as genomic mutation data and whole-genome RNA sequencing (RNA-seq) profiles. In particular, proteomic data is also an essential component for the characterization of tumours. However, different from RNA-seq datasets rich in numerous transcriptome profiles of cancer cell lines and cell viability assay of drug responses, the pharmacogenomic protein quantifications are relatively scarce. With the availability of the recently enriched proteomic dataset ProCan-DepMapSanger, we systematically evaluated the interplays among genomic mutations, transcription, and protein expressions across cancer cell lines. In general, blood cancers have higher RNA-protein correlations than those in solid cancers. The differential expression analysis on protein data helped identify more expressional and functional impact of genomic mutations of cancer genes. We also integrated the proteomic map with drug molecular chemical features to construct a bi-modal machine learning model to infer the drug sensitivities of cancer cell lines. Our results demonstrated that protein quantifications can lead to better drug response prediction performance than the model trained on transcriptome profiles. In addition, integrating protein data with drug chemical features, represented as molecular graphs and learned by Graph Neural Network, outperformed the state-of-the-art model DeepOmicNet for drug response prediction in proteomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c89e8ade30b207677f590882fddb023a8cd320d3" target='_blank'>
              Drug Response Modeling across Cancers: Proteomics vs. Transcriptomics
              </a>
            </td>
          <td>
            Zetian Zheng, Lei Huang, Fuzhou Wang, Linjing Liu, Jixiang Yu, Weidun Xie, Xingjian Chen, Xiangtao Li, Ka-Chun Wong
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The advent of single-cell transcriptomics has revolutionized our ability to analyze cellular heterogeneity and dynamics at a fine resolution, yet covering the vast array of potential perturbations remains challenging due to biological variability. To address this, we propose scCADE, a novel computational approach utilizing contrastive learning and an attention mechanism to decouple gene expression signatures and predict cellular responses to perturbations. scCADE excels in predicting responses in cells to perturbations observed in other cells but not yet seen in the target cells. Through rigorous ablation studies and validation across three datasets involving drug and gene editing perturbations, scCADE consistently outperformed existing methods, underscoring its efficacy and potential to advance genomics and personalized medicine by accurately forecasting responses to novel perturbations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d36ceda28edb31805eb4f9032509797809b3ca7" target='_blank'>
              scCADE: A Superior Tool for Predicting Perturbation Responses in Single-Cell Gene Expression Using Contrastive Learning and Attention Mechanisms
              </a>
            </td>
          <td>
            Jingfeng Ou, Jiawei Li, Zhiliang Xia, Shurui Dai, Yulian Ding, Yan Guo, Limin Jiang, Jijun Tang
          </td>
          <td>2024-12-03</td>
          <td>2024 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Esophageal squamous cell carcinoma (ESCC) remains a significant challenge in oncology due to its aggressive nature and heterogeneity. As one of the deadliest malignancies, ESCC research lags behind other cancer types. The balance between ubiquitination and deubiquitination processes plays a crucial role in cellular functions, with its disruption linked to various diseases, including cancer. Methods Our study utilized diverse analytical approaches, encompassing Cox regression models, single-cell RNA sequencing, intercellular communication analysis, and Gene Ontology enrichment. We also conducted mutation profiling and explored potential immunotherapeutic agents. Furthermore, in vitro cellular experiments and in vivo mouse models were performed to validate findings. These methodologies aimed to establish deubiquitination-related gene signatures (DRGS) for predicting ESCC patient outcomes and response to immunotherapy. Results By integrating datasets from TCGA-ESCC and GSE53624, we developed a DRGS model based on 14 deubiquitination-related genes (DUBGs). This signature effectively forecasts ESCC prognosis, drug responsiveness, and immune cell infiltration patterns. It also influences the mutational landscape of patients. Those classified as high-risk exhibited reduced survival rates, increased genetic alterations, and more complex cellular interactions, potentially explaining their poor outcomes. Notably, in vitro and in vivo experiments identified MTOR, a key component of the signature, as a promising therapeutic target for ESCC treatment. Conclusion Our research highlights the significance of 14 DUBGs in ESCC progression. The risk score derived from this gene set enables clinical stratification of patients into distinct prognostic groups. Moreover, MTOR emerges as a potential target for personalized ESCC therapy, offering new avenues for treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f8ef21367e3b48a6b59d4e0e2f8cebf967d4f23" target='_blank'>
              Leveraging single-cell and multi-omics approaches to identify MTOR-centered deubiquitination signatures in esophageal cancer therapy
              </a>
            </td>
          <td>
            Kang Tian, Ziang Yao, Da Pan
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Structural maintenance of chromosome-1A (SMC1A) is overexpressed in various malignancies including triple-negative breast cancer (TNBC). As a core component of the cohesin complex, SMC1A was initially recognized for its involvement in chromosomal cohesion and DNA-repair pathways. However, recent studies have unveiled its pivotal role in epithelial-mesenchymal transition (EMT), metastasis, and chemo- and radio-resistance in cancer cells. In hepatocellular carcinoma, aberrant phosphorylation of SMC1A has been associated with enhanced cell proliferation and migration. Despite these insights, the precise role of SMC1A phosphorylation in breast cancer remains largely unexplored. This study represents the first investigation to test the phosphorylation status and subcellular localization of SMC1A (p-SMC1A) in breast cancer and normal breast tissues. Immunohistochemical (IHC) staining was conducted using previously validated phospho-SMC1A antibodies on a histological section and tissue microarray (TMA) comprising samples from primary, invasive, and metastatic breast cancer and normal breast tissues. Our results revealed that p-SMC1A staining intensity was lower in normal breast tissues compared to invasive or metastatic breast cancer tissues (p < 0.001). Approximately 40% of breast cancer tissue exhibited cytoplasmic/membranous localization of p-SMC1A, whereas nuclear expression was observed in normal breast tissues. Moreover, elevated phosphorylation levels were significantly associated with higher tumor grade and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ac66fe9226183dafb37845efa13c23cb0dc9fe" target='_blank'>
              Association of Structural Maintenance of Chromosome-1A Phosphorylation with Progression of Breast Cancer.
              </a>
            </td>
          <td>
            Sushma Yadav, C. Kowolik, Daniel Schmolze, Yuan Yuan, Min Lin, Arthur D. Riggs, David A. Horne
          </td>
          <td>2025-01-17</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, Martin Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Over the course of the last decade, genomic studies in the context of normal human hematopoiesis have provided new insights into the early pathogenesis of myeloproliferative neoplasms (MPN). A preclinical phase of MPN, termed clonal hematopoiesis (CH) was identified and subsequent lineage tracing studies revealed a multi-decade long time interval from acquisition of an MPN phenotypic driver mutation in a hematopoietic stem cell (HSC) to the development of overt MPN. Multiple germline variants associated with MPN risk have been identified through genome-wide association studies (GWAS) and in some cases functional interrogation of the impact of the variant has uncovered new insights into HSC biology and MPN development. Increasingly sophisticated methods to study clonal contributions to human hematopoiesis and measure HSC fitness have helped discern the biology underlying the tremendous clinical heterogeneity observed in MPN. Despite these advances, significant knowledge gaps remain particularly with respect to germline genetic contributors to both MPN pathogenesis and phenotypic diversity, as well as limitations in the ability to prospectively quantify rates of clonal expansion in individual MPN patients. Ultimately, we envisage a personalized approach to MPN care in the future, where an individualized genetic assessment can predict MPN trajectory and this information will be used to inform and guide therapy. MPN is particularly amenable to precision medicine strategies and our increased understanding of the evolution of MPN from normal blood stem cells provides a unique opportunity for early therapeutic intervention approaches and potentially MPN prevention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd917b44573f25d09202b5ca694e87a98f431b63" target='_blank'>
              Evolution of myeloproliferative neoplasms from normal blood stem cells.
              </a>
            </td>
          <td>
            S. Hormoz, Vijay G Sankaran, Ann Mullally
          </td>
          <td>2024-12-05</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract The patient‐derived xenograft (PDX) model is a crucial in vivo model extensively employed in cancer research that has been shown to maintain the genomic characteristics and pathological structure of patients across various subtypes, metastatic, and diverse treatment histories. Various treatment strategies utilized in PDX models can offer valuable insights into the mechanisms of tumor progression, drug resistance, and the development of novel therapies. This review provides a comprehensive overview of the establishment and applications of PDX models. We present an overview of the history and current status of PDX models, elucidate the diverse construction methodologies employed for different tumors, and conduct a comparative analysis to highlight the distinct advantages and limitations of this model in relation to other in vivo models. The applications are elucidated in the domain of comprehending the mechanisms underlying tumor development and cancer therapy, which highlights broad applications in the fields of chemotherapy, targeted therapy, delivery systems, combination therapy, antibody–drug conjugates and radiotherapy. Furthermore, the combination of the PDX model with multiomics and single‐cell analyses for cancer research has also been emphasized. The application of the PDX model in clinical treatment and personalized medicine is additionally emphasized.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0edb643ec9b4460c4eb4c6d7bacb588877a32d4f" target='_blank'>
              Patient‐derived xenograft model in cancer: establishment and applications
              </a>
            </td>
          <td>
            Ao Gu, Jiatong Li, Meng-Yao Li, Yingbin Liu
          </td>
          <td>2025-01-19</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Odorant receptors (ORs), which constitute approximately 50% of all human G protein-coupled receptors, are increasingly recognized for their diverse roles beyond odor perception, including functions in various pathological conditions like brain diseases and cancers. However, the roles of ORs in glioblastoma (GBM), the most aggressive primary brain tumor with a median survival of only 15 months, remain largely unexplored. Here, we performed an integrated transcriptomic analysis combining The Cancer Genome Atlas RNA-seq and single-cell RNA sequencing data from GBM patients to uncover cell-type-specific roles of ORs within the tumor and its microenvironment. Our findings reveal that ORs display distinct expression patterns, with OR51E1 enriched in pericytes linked to vascular remodeling and angiogenesis, OR2B11 associated with tumor-associated macrophages supporting immunosuppressive phenotypes, and OR2L13 correlated with synaptic activity in recurrent tumors, potentially mediating treatment-induced neuronal adaptations. These results highlight ORs as potential therapeutic targets, offering new insights into their regulatory roles in GBM progression, immune modulation, and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0d78b05c85d74f15c35ba8438bef22107b35ac7" target='_blank'>
              Comprehensive Transcriptomic Analysis Reveals Cell-Type-Specific Roles of Human Odorant Receptors in Glioblastoma and the Tumor Microenvironment
              </a>
            </td>
          <td>
            Hee Jin Cho, Dong Jun Yeo, HeeWoong Yang, JaeHyung Koo
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Endogenous retroviral (ERV) RNA is highly expressed in cancer, although the molecular causes and consequences remain unknown. We found that ZC3H18 (Z18), a component of multiple nuclear RNA surveillance complexes, has recurrent truncating mutations in cancer. We show that Z18trunc mutations are oncogenic and that Z18 plays an evolutionarily conserved role in nuclear RNA surveillance of ERV RNA. In zebrafish, Z18trunc expedited melanoma onset and promoted a specific accumulation of ERV RNA. Z18 mutant human cell lines from the Cancer Cell Line Encyclopedia also expressed higher levels of ERV RNA. In engineered human melanoma cells, Z18trunc enhanced ERV RNA accumulation more than loss of one Z18 copy, indicating dominant negative activity. Z18trunc directly bound and stabilized ERV RNA. Notably, expression of ERV RNA was sufficient to expedite oncogenesis in a zebrafish model, which is the first evidence of which we are aware that ERV transcripts can play a functional role in cancer. Our work illuminates a mechanism for elevated ERV transcripts in cancer and supports that aberrant RNA accumulation is broadly oncogenic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85414cac93dcb00a3b2cbff32569ab0497071b3c" target='_blank'>
              Recurrent oncogenic ZC3H18 mutations stabilize endogenous retroviral RNA
              </a>
            </td>
          <td>
            Tanzina Tanu, Anna M. Cox, Jennifer A. Karlow, Priyanka Sharma, Xueyang He, Constance Wu, Swathy Babu, Jared Brown, Kevin M. Brown, Stephen J. Chanock, David Liu, Tongwu Zhang, Kathleen H. Burns, P. Boutz, Megan L. Insco
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Immunologically hot tumors, characterized by an inflamed tumor microenvironment (TME), contrast significantly with immunologically cold tumors. The identification of these tumor immune subtypes holds clinical significance, as hot tumors may exhibit improved prognoses and heightened responsiveness to checkpoint blockade therapy. Nevertheless, as yet there is no consensus regarding the clinically relevant definition of hot/cold tumors, and the influence of immune genes on the formation of hot/cold tumors remains poorly understood.Data for 33 different types of cancer were obtained from The Cancer Genome Atlas database, and their immune composition was assessed using the CIBERSORT algorithm. Tumors were categorized as either hot or cold based on their distinct immune composition, ongoing immune response, and overall survival. A customized immunogram was created to identify important immunological characteristics. Kyoto Encyclopedia of Genes and Genomes and Hallmark pathway enrichment were evaluated through gene set variation analysis. Additionally, hub genes that regulate the tumor microenvironment were identified, and their expression patterns were analyzed using single-cell RNA sequencing. Furthermore, drug sensitivity and molecular docking analyses were performed to identify potential drug candidates capable of transforming cold tumors into hot tumors. For validation, a clinical cohort of patients diagnosed with pancreatic adenocarcinoma was examined using multiplex immunohistochemistry.We were able to differentiate between hot and cold tumors in various types of cancer (bladder urothelial carcinoma, pancreatic adenocarcinoma, and cervical squamous cell carcinoma) by analyzing the presence of CD8+ T cells, activated natural killer cells, and M2-type macrophages, as well as the cytolytic activity and T cell proliferation. Hub genes that regulate the TME, including PDCD1, CD276, and NT5E, were discovered. The increased expression of NT5E and its prognostic significance were confirmed through multiplex immunohistochemistry in pancreatic adenocarcinoma. Finally, dasatinib and tozasertib were identified as drug candidates capable of converting cold pancreatic adenocarcinoma tumors into hot tumors.In this study, we developed a framework for discerning clinically significant immune subtypes across various cancer types, further identifying several potential targets for converting cold tumors into hot tumors to enhance anticancer treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cb5bf4fd6fa63b0c99ea155f3c748ee67916281" target='_blank'>
              Immune regulatory genes impact the hot/cold tumor microenvironment, affecting cancer treatment and patient outcomes
              </a>
            </td>
          <td>
            Mengmeng Sang, Jia Ge, Juan Ge, Gu Tang, Qiwen Wang, Jiarun Wu, Liming Mao, Xiaoling Ding, Xiaorong Zhou
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mutations in the exonuclease domains of the replicative nuclear DNA polymerases POLD1 and POLE are associated with increased cancer incidence, elevated tumor mutation burden (TMB), and enhanced response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond, highlighting the need for a better understanding of how TMB affects tumor biology and subsequently immunotherapy response. To address this, we generated mice with germline and conditional mutations in the exonuclease domains of Pold1 and Pole. Engineered mice with Pold1 and Pole mutator alleles presented with spontaneous cancers, primarily lymphomas, lung cancer, and intestinal tumors, while Pold1 mutant mice also developed tail skin carcinomas. These cancers had highly variable tissue-type dependent increased TMB with mutational signatures associated with POLD1 and POLE mutations found in human cancers. The Pold1 mutant tail tumors displayed increased TMB, however, only a subset of established tumors responded to ICB. Similarly, introducing the mutator alleles into mice with lung cancer driven by mutant Kras and Trp53 deletion did not improve survival, whereas passaging these tumor cells in vitro without immune editing and subsequently implanting them into immune-competent mice caused tumor rejection in vivo. These results demonstrated the efficiency by which cells with antigenic mutations are eliminated in vivo. Finally, ICB treatment of mutator mice earlier, before observable tumors had developed delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the potential of ICB in high-risk individuals for cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274ef0221dce61440b76214e33f127197aac5688" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer Development and Extends Survival in DNA Polymerase Mutator Syndromes.
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. Cararo Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse R. Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f1f8dfa9377169d8a09f717c0d18ed8c6b2867" target='_blank'>
              Prospective validation of ORACLE, a clonal expression biomarker associated with survival of patients with lung adenocarcinoma.
              </a>
            </td>
          <td>
            D. Biswas, Yun-Hsin Liu, Javier Herrero, Yin Wu, David A Moore, Takahiro Karasaki, Kristiana Grigoriadis, Wei-Ting Lu, S. Veeriah, C. Naceur-Lombardelli, Neil Magno, S. Ward, A. Frankell, M. Hill, Emma C Colliver, Sophie de Carné Trécesson, Philip East, A. Malhi, Daniel M. Snell, Olga O’Neill, Daniel Leonce, J. Mattsson, A. Lindberg, P. Micke, Judit Moldvay, Z. Megyesfalvi, B. Dome, J. Fillinger, J. Nicod, Julian Downward, Zoltan Szallasi, A. Hackshaw, M. Jamal-Hanjani, N. Kanu, Nicolai J. Birkbak, C. Swanton
          </td>
          <td>2025-01-09</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Abstract The blood–brain barrier is often altered in glioblastoma (GBM) creating a blood–brain–tumor barrier (BBTB) composed of pericytes. The BBTB affects chemotherapy efficacy. However, the expression signatures of BBTB‐associated pericytes remain unclear. We aimed to identify BBTB‐associated pericytes in single‐cell RNA sequencing data of GBM using pericyte markers, a normal brain pericyte expression signature, and functional enrichment. We identified parathyroid hormone receptor‐1 (PTH1R) as a potential marker of pericytes associated with BBTB function. These pericytes interact with other cells in GBM mainly through extracellular matrix–integrin signaling pathways. Compared with normal pericytes, pericytes in GBM exhibited upregulation of several ECM genes (including collagen IV and FN1), and high expression levels of these genes were associated with a poor prognosis. Cell line experiments showed that PTH1R knockdown in pericytes increased collagen IV and FN1 expression levels. In mice models, the expression levels of PTH1R, collagen IV, and FN1 were consistent with these trends. Evans Blue leakage and IgG detection in the brain tissue suggested a negative correlation between PTH1R expression levels and blood–brain barrier function. Further, a risk model based on differentially expressed genes in PTH1R+ pericytes had predictive value for GBM, as validated using independent and in‐house cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee0728b81bf82d09f8e05a6d94f03f2e545717c1" target='_blank'>
              Single‐cell transcriptomic analysis of glioblastoma reveals pericytes contributing to the blood–brain–tumor barrier and tumor progression
              </a>
            </td>
          <td>
            Yuzhe Li, Changwu Wu, Xinmiao Long, Xiangyu Wang, Wei Gao, Kun-Kang Deng, Bo Xie, Sen Zhang, Minghua Wu, Qing Liu
          </td>
          <td>2024-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a central role in cancer progression, therapy resistance, and disease recurrence. With the use of a quadruple-mutant mouse intestinal cancer organoid model and single-cell RNA-sequencing analysis, we have now identified glycosylphosphatidylinositol-specific phospholipase D1 (GPLD1), an enzyme that catalyzes the cleavage of glycosylphosphatidylinositol (GPI) anchors of membrane proteins, as a marker of slowly cycling CSCs. Ablation of Gpld1+ cells in combination with 5-fluorouracil treatment greatly attenuated cell viability in and regrowth of the intestinal cancer organoids. In addition, we identified serine protease 8 (PRSS8) as a key substrate of GPLD1 in human colorectal cancer cells. GPLD1 cleaves the GPI anchor of PRSS8 and thereby mediates release of the protease from the plasma membrane, resulting in the activation of Wnt signaling and promotion of the epithelial-mesenchymal transition (EMT) in the cancer cells. Pharmacological inhibition of GPLD1 suppressed Wnt signaling activity and EMT in association with upregulation of the amount of functional PRSS8 at the plasma membrane. Our findings suggest that targeting of GPLD1 in colorectal cancer might contribute to a new therapeutic strategy that is based on suppression of Wnt signaling and EMT-related cancer progression driven by CSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53a2de13be0cbce183bb9da3ac05db7fd47362af" target='_blank'>
              GPLD1+ cancer stem cells contribute to chemotherapy resistance and tumor relapse in intestinal cancer.
              </a>
            </td>
          <td>
            Taisuke Mizoo, Takeru Oka, Osamu Sugahara, Takafumi Minato, T. Higa, Keiichi I Nakayama
          </td>
          <td>2025-01-02</td>
          <td>Journal of biochemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Recurrent therapy resistance is a major limitation in clinical efficacy and for the outcome of glioblastoma (GBM) patients, positioning GBM among the tumor types with the poorest survival outcomes. In this work, we dissected resistance mechanisms in GBM, which resulted in the identification of FGFR1 pathway as a major regulator of the signaling and metabolic rewiring associated with temozolomide (TMZ) resistance in GBM. Hence, we described a mechanism of resistance that operates at two major levels. First, a p53-mediated regulation of cell cycle inducing cell cycle arrest to allow DNA repair in response to TMZ. And second, a complete metabolic rewiring promoting lipid catabolism and preventing lipid peroxidation. Both the p53-mediated response and the metabolic adaptation are controlled by FGFR1, as inhibition of the FGFR1 pathway completely abolishes this signaling and metabolic reprograming, restoring sensitivity to TMZ. Our results also indicated a correlation of FGFR1 levels with poor prognosis in GBM patients, and validated the treatment of TMZ in combination with FGFR1 inhibitors as an efficient strategy to induce tumor cell death in pre-clinical animal models. This data position the receptor FGFR1 as a very promising candidate for evaluation in future clinical approaches to limit the development of therapy resistance to TMZ in GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba711301ae1507c841a26c809bbac2e4e721e807" target='_blank'>
              FGFR1 inhibition improves therapy efficacy and prevents metabolic adaptation associated with temozolomide resistance in glioblastoma
              </a>
            </td>
          <td>
            Laura Zarzuela, Ignacio G. López-Cepero, K. Rattigan, Elena Sánchez-Escabias, Macarena Morillo-Huesca, Ana Reina-Bando, Sara S. Oltra, J. M. Sierra-Párraga, María Ceballos-Chávez, Vivian Capilla-González, Gema Moreno-Bueno, P. S. Murdoch, JC Reyes, G. V. Helgason, Mercedes Tomé, Raúl V. Durán
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The existence of cancer stem cells (CSCs) in various tumors has become increasingly clear in addition to their prominent role in therapy resistance, metastasis, and recurrence. For early diagnosis, disease progression monitoring, and targeting, there is a high demand for clinical-grade methods for quantitative measurement of CSCs from patient samples. Despite years of active research, standard measurement of CSCs has not yet reached clinical settings, especially in the case of solid tumors. This is because detecting this plastic heterogeneous population of cells is not straightforward. This review summarizes various techniques, highlighting their benefits and limitations in detecting CSCs from patient samples. In addition, methods designed to detect CSCs based on secreted and niche-associated signaling factors are reviewed. Spatial and single-cell methods for analyzing patient tumor tissues and noninvasive techniques such as liquid biopsy and in vivo imaging are discussed. Additionally, methods recently established in laboratories, preclinical studies, and clinical assays are covered. Finally, we discuss the characteristics of an ideal method as we look toward the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a142badfcf7c429680b2b941eddecbf41e3d46a6" target='_blank'>
              Detection of Cancer Stem Cells from Patient Samples.
              </a>
            </td>
          <td>
            Sofia Hakala, Anna Hämäläinen, Sanne Sandelin, Nikolaos Giannareas, Elisa Närvä
          </td>
          <td>2025-01-20</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to their target tissues remains a fundamental challenge. To address this, we assessed whether primary and metastatic tumors more closely resemble their tissues of origin or target tissues in terms of gene expression. We analyzed expression profiles from multiple cancer types and normal tissues, including single-cell and bulk RNA sequencing data from both paired and unpaired patient cohorts. Primary tumors were overall more transcriptomically similar to their tissues of origin, while metastases shifted toward their target tissues. However, pathway-level analysis highlighted critical metabolic and immune transcriptomic changes toward target tissues during metastasis in both primary and metastatic tumors. In addition, primary tumors exhibited higher activity in cancer hallmarks such as “Activating Invasion and Metastasis” when compared to metastases. This comprehensive analysis provides a transcriptome-wide view of the processes through which cancer tumors adapt to their metastatic environments before and after metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3572d2ae3cdf5735eb942be44afbfd38a211b4d7" target='_blank'>
              Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues
              </a>
            </td>
          <td>
            N. Sanghvi, Camilo Calvo-Alcañiz, P. Rajagopal, S. Scalera, V. Canu, Sanju Sinha, Fiorella Schischlik, Kun Wang, Sanna Madan, E. Shulman, Antonios Papanicolau-Sengos, Giovanni Blandino, E. Ruppin, N. Nair
          </td>
          <td>2024-12-06</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Cellular differentiation is controlled by intricate layers of gene regulation, involving the modulation of gene expression by various transcriptional regulators. Due to the complexity of gene regulation, identifying master regulators across the differentiation trajectory has been a longstanding challenge. To tackle this problem, a computational framework, single‐cell Boolean network inference and control (BENEIN), is presented. Applying BENEIN to human large intestinal single‐cell transcriptome data, MYB, HDAC2, and FOXA2 are identified as the master regulators whose inhibition induces enterocyte differentiation. It is found that simultaneous knockdown of these master regulators can revert colorectal cancer cells into normal‐like enterocytes by synergistically inducing differentiation and suppressing malignancy, which is validated by in vitro and in vivo experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c7fb2686fa44d6df149bd4d9108d57834cc43d" target='_blank'>
              Control of Cellular Differentiation Trajectories for Cancer Reversion
              </a>
            </td>
          <td>
            Jeong-Ryeol Gong, Chun-Kyung Lee, Hoon-Min Kim, Juhee Kim, Jaeog Jeon, Sunmin Park, Kwang-Hyun Cho
          </td>
          <td>2024-12-11</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The ovary is one of the first organs to exhibit signs of aging, characterized by reduced tissue function, chronic inflammation, and fibrosis. Multinucleated giant cells (MNGCs), formed by macrophage fusion, typically occur in chronic immune pathologies, including infectious and non-infectious granulomas and the foreign body response1, but are also observed in the aging ovary2–4. The function and consequence of ovarian MNGCs remain unknown as their biological activity is highly context-dependent, and their large size has limited their isolation and analysis through technologies such as single-cell RNA sequencing. In this study, we define ovarian MNGCs through a deep analysis of their presence across age and species using advanced imaging technologies as well as their unique transcriptome using laser capture microdissection. MNGCs form complex interconnected networks that increase with age in both mouse and nonhuman primate ovaries. MNGCs are characterized by high Gpnmb expression, a putative marker of ovarian and non-ovarian MNGCs5,6. Pathway analysis highlighted functions in apoptotic cell clearance, lipid metabolism, proteolysis, immune processes, and increased oxidative phosphorylation and antioxidant activity. Thus, MNGCs have signatures related to degradative processes, immune function, and high metabolic activity. These processes were enriched in MNGCs compared to primary ovarian macrophages, suggesting discrete functionality. MNGCs express CD4 and colocalize with T-cells, which were enriched in regions of MNGCs, indicative of a close interaction between these immune cell types. These findings implicate MNGCs in modulation of the ovarian immune landscape during aging given their high penetrance and unique molecular signature that supports degradative and immune functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ded887bdf4a0069dbb82e21f70ee7c8e91b67f" target='_blank'>
              Multinucleated giant cells are hallmarks of ovarian aging with unique immune and degradation-associated molecular signatures
              </a>
            </td>
          <td>
            Aubrey Converse, Madeline J. Perry, Shweta S. Dipali, J. Isola, Emmett B. Kelly, Joseph M. Varberg, M. Zelinski, J. Gerton, Michael B. Stout, Michele T. Pritchard, Francesca E. Duncan
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, Mikkel G Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Cancer research has historically focused on the somatic mutation theory, viewing cancer as a consequence of genetic mutations. However, this perspective has limitations in explaining phenomena like tumor reversion and cancer heterogeneity. This paper introduces an alternative approach: viewing cancer as a complex information-processing system shaped by its microenvironment. By integrating historical data on tumor reversion and insights into evolutionary dynamics, I propose a reframing of cancer biology. This process-oriented perspective highlights the role of cellular plasticity and adaptive behaviors, offering new pathways for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c3cc92fad268ef72893a64fd8d179664753cb" target='_blank'>
              Rethinking cancer evolution: from genetic mutations to complex information systems in tumor reversion
              </a>
            </td>
          <td>
            M. Tez
          </td>
          <td>2024-12-13</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="SET domain bifurcated histone lysine methyltransferase 1 (SETDB1/ESET), a pivotal H3K9 methyltransferase, has been extensively studied since its discovery over two decades ago. SETDB1 plays critical roles in immune regulation, including B cell maturation, T-cell activity modulation, and endogenous retrovirus (ERV) silencing. While essential for normal immune cell function, SETDB1 overexpression in cancer cells disrupts immune responses by suppressing tumor immunogenicity and facilitating immune evasion. This is achieved through the repression of anti-tumor immune cell production, ERV silencing, and interference with the type I interferon pathway leading to inhibiting immune checkpoint blockade (ICB) efficacy. Beyond its immunological implications, SETDB1 overexpression fosters tumor growth and metastasis via transcriptional silencing of tumor suppressor genes through histone regulation and activating oncogenic signaling by non-histone regulation. These multifaceted roles make SETDB1 an attractive epigenetic target for novel cancer therapies. This review explores SETDB1's dual function in immune regulation and tumor progression, emphasizing its potential in the development of innovative cancer treatments targeting epigenetic dysregulation and oncogenic signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a673433d5768545e8da0592cd5b401ba83b5720" target='_blank'>
              Targeting SETDB1 in cancer and immune regulation: Potential therapeutic strategies in cancer.
              </a>
            </td>
          <td>
            Bo-Syong Pan, Cheng‐Yu Lin, Gilbert Aaron Lee, Hui-Kuan Lin
          </td>
          <td>2025-01-07</td>
          <td>The Kaohsiung journal of medical sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04e91a402cb4fbb4283cc9e644cf527d569eebab" target='_blank'>
              Inhibition of programmed cell death by melanoma cell subpopulations reveals mechanisms of melanoma metastasis and potential therapeutic targets
              </a>
            </td>
          <td>
            Yuepeng An, Fu Zhao, Hongling Jia, Siyu Meng, Ziwei Zhang, Shuxiao Li, Jiusi Zhao
          </td>
          <td>2025-01-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7be5c20b4cd0ad9d6c8146c5936042d6f39bb61" target='_blank'>
              CRYβB2 alters cell adhesion to promote invasion in a triple-negative breast cancer cell line
              </a>
            </td>
          <td>
            Amr A. Waly, London Harper, Jodie M. Fleming, Lindsey M Costantini
          </td>
          <td>2025-01-21</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Extraprostatic extension (EPE) of prostate cancer (PCa) cells in the adipose space (pT3a stage) is an unfavourable prognostic factor associated with recurrence and poor free metastatic disease-free survival outcomes of patients. However, molecular adaptation of PCa cells associated with EPE remains poorly understood. The aim of our research was to identify genomic determinants associated with EPE. We identified an overexpression of the heat shock protein genes: HSP90s, HSP70s, and HSP40s in PCa cells which infiltrate the adipose space. Moreover, we showed that EPE is also related to specific pathways associated with metastatic spread and enhancement of the androgen receptor pathway. These results provided valuable additional and new information to understand adaptative tumour changes during invasion of the adipose space This could be useful for the development of new markers or targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0439b796fb004e399c7026630fcceb991df01654" target='_blank'>
              Spatial Genomics Identifies Heat Shock Proteins as Key Molecular Changes Associated to Adipose Periprostatic Space Invasion in Prostate Cancer
              </a>
            </td>
          <td>
            Olivier Cussenot, Lucie Poupel, Coralie Mousset, Julien Lavergne, Franck Bruyere, Alix Fontaine, G. Cancel-Tassin, Gaelle Fromont-Hankard
          </td>
          <td>2024-12-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background: Circulating rare cells participate in breast cancer evolution as systemic components of the disease and thus, are a source of theranostic information. Exploration of cancer-associated rare cells is in its infancy. Objectives: We aimed to investigate and classify abnormalities in the circulating rare cell population among early-stage breast cancer patients using fluorescence marker identification and cytomorphology. In addition, we sought to determine the dependency of these markers on the presence of tumors. Design: We evaluated the validity of a multi-rare-cell detection platform and demonstrated the utility of a specific rare cell subset as a novel approach to characterize the breast cancer system. Sampling was conducted both before and after tumor resection. Methods: Linearity of the Rarmax platform was established using a spike-in approach. The platform includes red blood cell lysis, leukocyte depletion and high-resolution fluorescence image recording. Rare cell analysis was conducted on 28 samples (before and after surgery) from 14 patients with breast cancer, 20 healthy volunteers and 9 noncancer control volunteers. In-depth identification of rare cells, including circulating tumor cells, endothelial-like cells, erythroblasts, and inflammation-associated cells, was performed using a phenotype and morphology-based classification system. Results: The platform performed linearly over a range of 5 to 950 spiked cells, with an average recovery of 84.6%. Circulating epithelial and endothelial-like cell subsets have been demonstrated to be associated with or independent of cancer with tumor presence. Furthermore, certain cell profile patterns may be associated with treatment-related adverse effects. The sensitivity in detecting tumor-presence and cancer-associated abnormality before surgery was 43% and 85.7%, respectively, and the specificity was 100% and 96.6%, respectively. Conclusion: This study supports the idea of a cancer-associated rare cell abnormality to represent tumor entities as well as systemic cancer. The latter is independent of the apparent clinical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dddc8e100310019b58c6626c61326f8e0d4cf2c" target='_blank'>
              Rare Cell Population Analysis in Early-Stage Breast Cancer Patients
              </a>
            </td>
          <td>
            Stefan Schreier, Prapaphan Budchart, S. Borwornpinyo, Lakkana Adireklarpwong, P. Chirappapha, W. Triampo, P. Lertsithichai
          </td>
          <td>2025-01-01</td>
          <td>Breast Cancer : Basic and Clinical Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb23096de05b742040245374a7585b16f9e2179" target='_blank'>
              Therapeutic implications and comprehensive insights into cellular senescence and aging in the tumor microenvironment of sarcoma
              </a>
            </td>
          <td>
            P. Zan, Yi Zhang, Yidong Zhu, Qingjing Chen, Zhengwei Duan, Yonghao Guan, Kaiyuan Liu, Anquan Shang, Zihua Li
          </td>
          <td>2025-01-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background Increasing evidence indicates that cancer stem cells (CSCs) and cancer stem-like cells form a special subpopulation of cells that are ubiquitous in tumors. These cells exhibit similar characteristics to those of normal stem cells in tissues; moreover, they are capable of self-renewal and differentiation, as well as high tumorigenicity and drug resistance. In prostate cancer (PCa), it is difficult to kill these cells using androgen signaling inhibitors and chemotherapy drugs. Consequently, the residual prostate cancer stem cells (PCSCs) mediate tumor recurrence and progression. Objective This review aims to provide a comprehensive and up-to-date overview of PCSCs, with a particular emphasis on potential therapeutic strategies targeting these cells. Methods After searching in PubMed and Embase databases using ‘prostate cancer’ and ‘cancer stem cells’ as keywords, studies related were compiled and examined. Results In this review, we detail the origin and characteristics of PCSCs, introduce the regulatory pathways closely related to CSC survival and stemness maintenance, and discuss the link between epithelial–mesenchymal transition, tumor microenvironment and tumor stemness. Furthermore, we introduce the currently available therapeutic strategies targeting CSCs, including signaling pathway inhibitors, anti-apoptotic protein inhibitors, microRNAs, nanomedicine, and immunotherapy. Lastly, we summarize the limitations of current CSC research and mention future research directions. Conclusion A deeper understanding of the regulatory network and molecular markers of PCSCs could facilitate the development of novel therapeutic strategies targeting these cells. Previous preclinical studies have demonstrated the potential of this treatment approach. In the future, this may offer alternative treatment options for PCa patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/537b50df573d4150f73440d71dd0d947241b969e" target='_blank'>
              Stemness regulation in prostate cancer: prostate cancer stem cells and targeted therapy
              </a>
            </td>
          <td>
            Hao Liang, Bin Zhou, Peixin Li, Xiaoyi Zhang, Shijie Zhang, Yaozhong Zhang, Shengwen Yao, Sifeng Qu, Jun Chen
          </td>
          <td>2024-12-23</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [9, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>