<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-09 09:30:33 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Tumor microenvironments (TMEs) are spatially complex and dynamic systems shaped by evolutionary pressures, tissue architecture, and cellular interactions. To capture this complexity, we developed the Tumor Landscape Analysis (TLA) pipeline, a computational framework applying principles from landscape ecology and spatial statistics to quantitatively characterize tumor spatial heterogeneity. TLA integrates spatially resolved pathology data, including whole-cell, point-based, and region-level formats, and computes multiscale metrics to assess cell distributions, neighborhood relationships, and tissue-level organization. The framework leverages ecological indices such as the Morisita-Horn index, Ripley’s H function, and Shannon diversity to quantify intercellular proximity, spatial clustering, and cellular diversity. It also uses the concept of local microenvironments (LMEs), data-driven ecological niches defined by local cell-type abundance and spatial uniformity, enabling unsupervised and reproducible classification of tumor regions. The use of fragmentation metrics, including patch density, shape complexity, and interspersion, provide further insight into spatial disorganization and emergent tissue architecture. TLA is agnostic to imaging modality and biological context, supporting broad applicability across tumor types and sample formats. By translating complex tissue architectures into interpretable spatial metrics, the pipeline enables integrative analyses that link spatial ecology to clinical and molecular phenotypes. This approach facilitates a deeper understanding of how spatial features contribute to tumor progression, therapeutic resistance, and clinical outcomes, offering new opportunities for precision oncology rooted in spatial systems biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/718b2bbad37dfa5c06ac139aad6c2d7117e031ab" target='_blank'>
              Tumor Landscape Analysis: An Ecologically Informed Framework to Understand Tumor Microenvironments
              </a>
            </td>
          <td>
            Luis Cisneros, M. D. Toruner, Martin E Fernandez-Zapico, Carlo C. Maley, Ryan M. Carr
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Over the past years, cancer research has transitioned from a ‘cancer cell-centered’ focus to a more integrative view of tumors as dynamic ecosystems. This paradigm shift emphasizes the tumor microenvironment (TME) as a complex network of interacting cellular and acellular components, where tumor cells orchestrate a supportive environment that facilitates progression, metastasis, and immune evasion. Understanding the spatial organization of these components within the TME is crucial, as the positioning and interactions between cancerous and non-cancerous cells significantly influence tumor behavior and therapy response. Spatial proteomics has emerged as a powerful tool for TME analysis, enabling the detection and quantification of proteins within intact tissue architecture at subcellular resolution. This approach provides insights into cellular interactions, signaling pathways, and functional states, facilitating the discovery of novel biomarkers and therapeutic targets linked to specific tissue regions and cellular contexts. Translating spatial proteomics into clinical practice requires overcoming challenges related to technology refinement, standardization of workflows, and adaptation to routine pathology settings. Melanoma is an aggressive, highly immunogenic malignancy with variable response rates to existing immunotherapies. Given that over half of patients treated with immune checkpoint inhibitors (ICIs) fail to respond or experience disease progression, the identification of novel biomarkers and therapeutic targets to enhance current therapies is urgently required. Spatial imaging technologies are increasingly being utilized to dissect the complex interplay between stroma, melanoma, and immune cell types within the TME to address this need. This review examines key spatial proteomics methods, their applications in melanoma biology, and associated image analysis pipelines. We highlight the current limitations, and future directions, emphasizing the potential for clinical translation to guide personalized treatment strategies, inform prognosis, and predict therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1eb1624eaa05c3beff7c464020ea5f0d816374c" target='_blank'>
              Spatial proteomics of the tumor microenvironment in melanoma: current insights and future directions
              </a>
            </td>
          <td>
            Chiara Bungaro, Michele Guida, Benedetta Apollonio
          </td>
          <td>2025-05-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae3a20489c547fc5b01d13afcfc1a7edcf2969d" target='_blank'>
              An image analysis pipeline to quantify the spatial distribution of cell markers in stroma-rich tumors
              </a>
            </td>
          <td>
            Antoine A. Ruzette, Nina Kozlova, Kayla A. Cruz, Taru Muranen, S. Nørrelykke
          </td>
          <td>2025-05-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b426c5f702b08645ac569d89b1da888e84c491" target='_blank'>
              Deciphering the cellular tumor microenvironment landscape in salivary gland carcinomas using multiplexed imaging mass cytometry
              </a>
            </td>
          <td>
            M. Mayer, L. Nachtsheim, L. Jansen, P. Wolber, M. Schmiel, A. Quaas, J. P. Klussmann, C. Arolt
          </td>
          <td>2025-05-13</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Identification of the consensus molecular subtypes (CMS) opened significant potential for understanding the tumor biology and intertumoral heterogeneity of colorectal cancer (CRC). However, molecular subtyping in CRC traditionally relies on bulk transcriptomics, therefore, lacks spatial and single-cell level aspect. Methods We constructed tissue microarrays using tumor cores from 222 CRC patients. Arrays were stained and imaged using 54 cell identity and cancer hallmark markers, delivering spatially resolved protein profiles of >2 million cells. RNA sequencing data and CMS classification were also available for these patients. After segmentation of cancer, stromal and immune cells, we investigated intratumoral heterogeneity within CMS subtypes using spatially resolved single-cell protein profiling (>2 million cells). We compared cell types, their spatial organization and their expression of cancer hallmark-related proteins in CMS 1-4 subtypes. Results We revealed tissue atlases illustrating the cell types/states, spatial heterogeneity, cellular neighborhoods, cellular network, and single-cell protein profiles of CMS tumors. CMS1 tumors had more CD3+, CD8+, and PD1+ immune cells that were found in the epithelial layer frequently. CMS1 was also associated with higher levels of metabolic reprogramming markers such as upregulated glycolysis. CMS2 showed immune segregation, reactive stroma patterns and higher levels of apoptotic and proliferative signaling proteins. CMS3 exhibited clustered cancer cells with high RIP3 levels, suggesting a pro-inflammatory microenvironment. CMS4 displayed stromal-centric and immune-evasive tumors characterized by decreased HLA-1 levels and regulatory T-cell exclusion from epithelium. Conclusion We present a spatial protein atlas of CRC at single-cell resolution and demonstrate novel aspects of CMS tumour structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd0aeac5c1de86e47bfdfaa33c8809067d3de4e" target='_blank'>
              High-Resolution Spatial Proteomics Characterises Colorectal Cancer Consensus Molecular Subtypes
              </a>
            </td>
          <td>
            Batuhan Kısakol, Anna Matveeva, Sanghee Cho, Mohammadreza Azimi, A. Lindner, M. Salvucci, E. McDonough, Joanna Fay, Tony O’Grady, N. McCawley, John P Burke, Deborah A. McNamara, John Graf, Simon McDade, Daniel B. Longley, Fiona Ginty, J. Prehn
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL) therapy, recently approved by the FDA for melanoma, is an emerging modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors such as glioblastoma remains limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas using spectral flow cytometry, TCR sequencing, single-cell RNA-seq, Xenium in situ transcriptomics, and CODEX spatial proteomics. Comparative analysis of TIL-generating (TIL+) versus non-generating (TIL−) tumors revealed that IL7Rexpression, structured perivascular immune clustering, and tumor-intrinsic metabolic programs such as ACSS3 were associated with successful TIL expansion. In contrast, TIL−tumors were enriched for neuronal lineage signatures, immunosuppressive transcripts including TOX and FERMT1, and tumor-connected macrophages. This study defines spatial and molecular correlates of TIL manufacturing success and establishes a genomics-enabled selection platform for adoptive T cell therapy. The profiling approach is now being prospectively implemented in the GIANT clinical trial (NCT06816927), supporting its translational relevance and scalability across glioblastoma and other immune-excluded cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7799f3cb909d60657cf83c09eaecb6b9bad79c3" target='_blank'>
              A Spatial Multi-Omic Framework Identifies Gliomas Permissive to TIL Expansion
              </a>
            </td>
          <td>
            Mustafa Khasraw, K. Hotchkiss, Kenan Zhang, Anna M Corcoran, Elizabeth Owens, P. Noldner, C. Railton, Kyra Van Batavia, Ying Zhou, Jodie Jepson, Kirit Singh, R. McLendon, Kristen Batich, Anoop P Patel, Katayoun Ayasoufi, Michael Brown, Evan Calabrese, Jichun Xie, Jose R Conejo-Garcia, B. Shaz, John Hickey
          </td>
          <td>2025-04-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Motivation Spatial transcriptomic (ST) technologies, such as GeoMx Digital Spatial Profiler, are increasingly utilized to investigate the role of diverse tumor microenvironment components, particularly in relation to cancer progression, treatment response, and therapeutic resistance. However, in many ST studies, the spatial information obtained from immunofluorescence imaging is primarily used for identifying regions of interest (ROIs) rather than as an integral part of downstream transcriptomic data analysis and interpretation. Results We developed ROICellTrack, a deep learning-based framework that better integrates cellular imaging with spatial transcriptomic profiling. By analyzing 56 ROIs from urothelial carcinoma of the bladder and upper tract urothelial carcinoma, ROICellTrack identified distinct cancer–immune cell mixtures, characterized by specific transcriptomic and morphological signatures and receptor–ligand interactions linked to tumor content and immune infiltrations. Our findings demonstrate the value of integrating imaging with transcriptomics to analyze spatial omics data, improving our understanding of tumor heterogeneity and its relevance to personalized and targeted therapies. Availability and implementation ROICellTrack is publicly available at https://github.com/wanglab1/ROICellTrack.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af060e4cc93259c4d19b62b9cc2136ad5699039" target='_blank'>
              ROICellTrack: a deep learning framework for integrating cellular imaging modalities in subcellular spatial transcriptomic profiling of tumor tissues
              </a>
            </td>
          <td>
            Xiaofei Song, Xiaoqing Yu, Carlos M. Morán-Segura, Hongzhi Xu, Tingyi Li, Joshua T Davis, A. Vosoughi, G. D. Grass, Roger Li, Xuefeng Wang
          </td>
          <td>2025-04-08</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d556d25908485193c4e548584bab10bfa70daa1" target='_blank'>
              Oncogenic drivers shape the tumor microenvironment in human gliomas
              </a>
            </td>
          <td>
            Bo Zhao, Chae Yun Cho, Lingqun Ye, Tula Keal, Thomas Mitchell, Inés Martín-Barrio, C. A. Faraj, Mesut Unal, Jessica Alsing, Nora M Lawson, Ellie Rahm Kim, Francisco Chavez, Kristy Mendoza Rangel, Sanjay K Singh, Joseph H. McCarthy, D.M. Ashley, W. Yung, Vinay K. Puduvalli, Guy Nir, Jeffrey S. Weinberg, Linghua Wang, Keith L. Ligon, G. Genovese, Peter Van Loo, P. Futreal, Jason T. Huse, Jesse R. Dixon, Frederick F. Lang, Kadir C. Akdemir
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Tumor subclones refer to distinct cell populations within the same tumor that possess different genetic characteristics. They play a crucial role in understanding tumor heterogeneity, evolution, and therapeutic resistance. The formation of tumor subclones is driven by several key mechanisms, including the inherent genetic instability of tumor cells, which facilitates the accumulation of novel mutations; selective pressures from the tumor microenvironment and therapeutic interventions, which promote the expansion of certain subclones; and epigenetic modifications, such as DNA methylation and histone modifications, which alter gene expression patterns. Major methodologies for studying tumor subclones include single-cell sequencing, liquid biopsy, and spatial transcriptomics, which provide insights into clonal architecture and dynamic evolution. Beyond their direct involvement in tumor growth and invasion, subclones significantly contribute to tumor heterogeneity, immune evasion, and treatment resistance. Thus, an in-depth investigation of tumor subclones not only aids in guiding personalized precision therapy, overcoming drug resistance, and identifying novel therapeutic targets, but also enhances our ability to predict recurrence and metastasis risks while elucidating the mechanisms underlying tumor heterogeneity. The integration of artificial intelligence, big data analytics, and multi-omics technologies is expected to further advance research in tumor subclones, paving the way for novel strategies in cancer diagnosis and treatment. This review aims to provide a comprehensive overview of tumor subclone formation mechanisms, evolutionary models, analytical methods, and clinical implications, offering insights into precision oncology and future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1d4c21265c412fc5d4c0775f9aa0ecd36b6c8eb" target='_blank'>
              Advances in tumor subclone formation and mechanisms of growth and invasion
              </a>
            </td>
          <td>
            Yuhong Zhang, Weidong Wang
          </td>
          <td>2025-04-21</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="IDH-wildtype glioblastoma (GBM) is an aggressive brain tumor with poor survival and few therapeutic options. Transcriptionally-defined cell states coexist in GBM and occupy defined regions of the tumor. Evidence indicates that GBM cell states are plastic, but it remains unclear if they are determined by the underlying epigenetic state and/or by microenvironmental factors. Here, we present spatially-resolved epigenomic profiling of human GBM tissues that implicate chromatin structure as a key enabler of cell plasticity. We report two epigenetically-defined and spatially-nested tumor niches. Each niche activates short-range molecular signals to maintain its own state and, surprisingly, long-range signals to reinforce the state of the neighboring niche. The position of a cell along this gradient-like system of opposing signals determines its likelihood to be in one state or the other. Our results reveal an intrinsic system for cell plasticity that is encoded in the chromatin profiles of two adjacent niches that dot the topological architecture of GBM in cartesian space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc7a8f38a5d53281f1873999c12438505ab4d573" target='_blank'>
              Spatial epigenomic niches underlie glioblastoma cell state plasticity
              </a>
            </td>
          <td>
            Sam Kint, S. T. Younes, Shuozhen Bao, Gretchen Long, David Wouters, Erin Stephenson, Xing Lou, Mei Zhong, Di Zhang, Graham Su, Archibald Enninful, Mingyu Yang, Huey-Miin Chen, K. Ellestad, Colleen Anderson, Jennifer Moliterno, John J. Kelly, Jennifer A. Chan, Alejandro Sifrim, Jiangbing Zhou, Ana Nikolic, Rong Fan, Marco Gallo
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd408ac5f53831aa4c4d85811f6f5969702d07e" target='_blank'>
              High-Dimensional Protein Analysis Uncovers Distinct Immunological and Stromal Signatures Between Primary and Metastatic Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Emily Greene, Natalie K. Horvat, D. Doxie, Vaunita C. Parihar, Jayden Kim, Cameron J. Herting, Erin E. Grundy, Ayana T. Ruffin, Alyssa Krasinskas, Shishir Maithel, Juan M. Sarmiento, Mihir M. Shah, Mohammad Y Zaidi, Maria Diab, O. Alese, Kavita Dhodapkar, H. Kissick, Bassel F. El-Rayes, C. Paulos, Gregory B. Lesinski
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="In breast cancer related to the BRCA1 mutation, luminal progenitor cells are believed to be the cells of origin, yet how these cells transform into invasive cancer cells remain poorly understood. Here, we combine single-cell epigenomic and transcriptomic data to reconstitute sequences of events in luminal cells that lead to tumorigenesis. Upon deletion of Trp53 and Brca1, we find that luminal progenitors display an extensive epigenomic disorder associated with a loss of cell identity. These cells then progress to tumor formation through a partial epithelial-to-mesenchymal transition, orchestrated by Snail and the timely activation of immunosuppressive and FGF signaling with their microenvironment. In human samples, pre-tumoral changes can be detected in early stage, basal-like tumors, which rarely recur, as well as in normal-like mammary glands of BRCA1 mutation carriers who have had cancer. Our study fills critical gaps in our understanding of BRCA1-driven tumorigenesis, opening perspectives for the early monitoring of individuals with high cancer risk. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02331-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f69d9b899da12be47c2084b992747c9f0654fa" target='_blank'>
              Epigenomic disorder and partial EMT impair luminal progenitor integrity in Brca1-associated breast tumorigenesis
              </a>
            </td>
          <td>
            Camille Landragin, Mélissa Saichi, Marthe Laisné, Adeline Durand, Pacôme Prompsy, Renaud Leclère, Jérémy Mesple, Kyra Borgman, Amandine Trouchet, Marisa M Faraldo, A. Chiche, A. Vincent-Salomon, Hélène Salmon, Céline Vallot
          </td>
          <td>2025-04-28</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="
 Identifying the contribution and the role of each individual cell in the spatial context of the tumor microenvironment (TME) is crucial in determining tumor progression, examining the extent of immune infiltration, and identifying potential therapeutic targets. While single-cell sequencing-based techniques help reveal cellular heterogeneity in a cell population, they do not provide spatial context. In turn, recently emerged spatial transcriptomics techniques are challenged by the difficulty in pinpointing the spatial measurement to single cells, requiring either complex cell segmentation or deconvolution algorithms in the data processing.
 To address these limitations, we introduce Curio Trekker single-cell spatial mapping kit, a unique platform that turns a single-cell experiment into a spatial experiment, essentially providing a spatial measurement that has true single-cell resolution. It is based on the Slide-tags methodology (Russel et al, 2023) that spatially tags each nucleus within its native tissue environment. This approach utilizes existing single-nuclei sequencing infrastructure to generate high-resolution spatial transcriptomics data. Using archived FFPE samples, we demonstrate the effectiveness of Curio Trekker by aligning 25-30 µm tissue sections to spatially barcoded tiles. Spatial barcodes are cleaved and hybridized to cells and nuclei in the intact tissue, followed by tissue dissociation and nuclei isolation. The tagged nuclei are processed with established snRNA-seq workflows (e.g., 10x Chromium FLEX), producing gene expression and spatial libraries for the individual nuclei in the tissue section. This spatial tagging allows each nucleus to be bioinformatically mapped back to its original tissue location.
 We applied this solution to breast cancer biospecimens from different patients using whole transcriptome probes (10x Genomics). The resulting data allowed for detailed characterization of the TME, confidently identifying epithelial-malignant, stromal, and immune cells from each biospecimen. This enabled marker-guided regional selection and differential gene expression analysis based on both cell type and spatial location. Overall, the Curio Trekker platform provides a streamlined, high-resolution solution for profiling cellular heterogeneity and interactions within tumors, seamlessly integrating with existing workflows to generate rapid insights into disease mechanisms and therapeutic development.


 Cedric R. Uytingco, Julie Wilhelmy, Bryant Ngo, Jonathan Sakkos, Wanxin Wang, Christina Chang, Christina Fan. Characterizing the tumor microenvironment using spatially barcoded archival FFPE tissue: converting single-nucleus RNA-seq into spatial transcriptomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6604.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b28a229fd99f6a9c5521983f68e3ec9d09a032a" target='_blank'>
              Abstract 6604: Characterizing the tumor microenvironment using spatially barcoded archival FFPE tissue: converting single-nucleus RNA-seq into spatial transcriptomics
              </a>
            </td>
          <td>
            Cedric R. Uytingco, Julie Wilhelmy, Bryant Ngo, Jonathan Sakkos, Wanxin Wang, Christina Chang, Christina Fan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Drug delivery to solid tumors is challenged by multiple physiological barriers arising from the tumor microenvironment, including dense extracellular matrix, cellular heterogeneity, hypoxic gradients, and elevated interstitial fluid pressure. These features hinder the uniform distribution and accumulation of therapeutics, reducing treatment efficacy. Despite their widespread use, conventional two-dimensional monolayer cultures fail to reproduce these complexities, contributing to the poor translational predictability of many preclinical candidates. Three-dimensional multicellular tumor spheroids have emerged as more representative in vitro models that capture essential features of tumor architecture, stromal interactions, and microenvironmental resistance mechanisms. Spheroids exhibit spatially organized regions of proliferation, quiescence, and hypoxia, and can incorporate non-tumor cells to mimic tumor–stroma crosstalk. Advances in spheroid analysis now enable detailed evaluation of drug penetration, cellular migration, cytotoxic response, and molecular gradients using techniques such as optical and confocal imaging, large-particle flow cytometry, biochemical viability assays, and microfluidic integration. By combining physiological relevance with analytical accessibility, spheroid models support mechanistic studies of drug transport and efficacy under tumor-like conditions. Their adoption into routine preclinical workflows has the potential to improve translational accuracy while reducing reliance on animal models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e55ff70454b1bfef937caf2e04dc247b0607a96" target='_blank'>
              Modeling Tumor Microenvironment Complexity In Vitro: Spheroids as Physiologically Relevant Tumor Models and Strategies for Their Analysis
              </a>
            </td>
          <td>
            Shrey Shah, Gerard G. M. D’Souza
          </td>
          <td>2025-05-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ba430604b98f8f47f7bf5f84f30b7530579a6d" target='_blank'>
              Spatial Biology and Organoid Technologies Reveal a Potential Therapy-Resistant Cancer Stem Cell Population in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Pritha Adhikary, Jayati Chakrabarti, Jiang Wang, Ugonna Ezuma-Igwe, Xi Sun, Willian C. Bastian, Nicolas Braissand, Megan P. Corbett, Eugene F. Douglass, Sangmin Kim, Rohan Kanakamedala, Abigail G. Branch, Saptarshi Mallick, Meenu Priya Resmi, Payton Stevens, Marco Padilla-Rodriguez, Maga Sanchez, Gregory Lawrence Beatty, James Griffin, Petros G. Nikolinankos, Kellen Chen, Taylor Riall, Syed A. Ahmad, Rachna Shroff, Frédéric Hollande, Davendra P. S. Sohal, Yana Zavros
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abfdae0d9888d7167ae3464de1089da14e882abe" target='_blank'>
              Multi-cellular phenotypic dynamics during the progression of breast tumors
              </a>
            </td>
          <td>
            Louise Gsell, Spencer S. Watson, Iva Sutevski, Matteo Massara, Klara Soukup, Alper Eroglu, Jeff E Mold, Antony Cougnoux, Johanna A. Joyce, Jean Hausser
          </td>
          <td>2025-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is notoriously resistant or refractory to both modern and conventional therapies. These tumors are characterized by heterogeneous malignant epithelial lesions and immune cells embedded within a highly dense and complex stroma. The unique, multi-compartmental tumor microenvironment (TME) —encompassing diverse lineages such as fibroblasts, mural cells, endothelial cells, lymphoid and myeloid populations, cancer cells, and a rich extracellular matrix—indicates the presence of intricate cellular networks that drive tumor progression. Efforts to profile and therapeutically target specific compartments or cellular phenotypes have often led to unintended adverse effects on neighboring cells and their microenvironments, compromising drug efficacy. To identify vulnerabilities in these interactions and precisely quantify intra- and inter-tumoral spatial heterogeneity, we utilized single-cell RNA sequencing data from eight distinct datasets to design separate custom multiplexed histopathology panels for imaging mass cytometry against tumor, immune, and stromal targets. Three serial sections of a tissue microarray containing 4 cores from 221 patients on each were stained with compartment-specific panels. Cell segmentation and clustering of 7.9 million cells in our dataset revealed 104 unique cell types, their functional states along with biophysical responses. We identified the presence of multiple epithelial subtypes within a single patient, along with single-compartment cellular organization and phenotypes associated with patient outcomes. Advanced image alignment using pixel classification and image transformation across serial sections enabled integration of TME-wide information, defining reproducible microenvironments neighboring malignant ducts. These microenvironments demonstrated immune and stromal patterns closely associated with specific epithelial phenotypes, carrying significant prognostic implications. Laser capture microdissection combined with mass spectrometry provided deeper insights into the biological processes within our microenvironments. We further identified associations between our microenvironments with bulk-RNAseq and specific genomic aberrations. These findings contribute to a nuanced understanding of the TME in PDAC and highlight the therapeutic potential in addressing this complexity to overcome the challenges in treating patients with PDAC.


 Noor Shakfa, Ferris Nowlan, Elizabeth Sunnucks, Sibyl Drissler, Tiak Ju Tan, Chengxin Yu, Jennifer L. Gorman, Michael J. Geuenich, Sheng-Ben Liang, Edward L. Chen, Golnaz Abazari, Barbara Gruenwald, Ayelet Borgida, Cassandra J. Wong, Brendon Seale, Zhen Yuan Lin, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Anne-Claude Gringas, Grianne M. O'Kane, Faiyaz Notta, Steven Gallinger, Hartland W. Jackson. Spatial analysis of tumor-immune-stromal cellular phenotypes and microenvironments in pancreatic cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 155.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffdda1a300bd205aab897ffa9f139e98ec525b59" target='_blank'>
              Abstract 155: Spatial analysis of tumor-immune-stromal cellular phenotypes and microenvironments in pancreatic cancer
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Elizabeth Sunnucks, Sibyl Drissler, Tiak Ju Tan, Chengxin Yu, Jennifer L. Gorman, Michael J Geuenich, Sheng-Ben Liang, Edward L. Chen, Golnaz Abazari, Barbara Gruenwald, A. Borgida, Cassandra J. Wong, Brendon Seale, Zhen Yuan Lin, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Anne-Claude Gringas, Grianne M. O'Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf4d0e7843bbae99e8106885d03d265c924f51c" target='_blank'>
              A unified eIF1A+ luminal cells-centered hypoxic and “cold” tumor microenvironment promotes PCa progression among different subtypes
              </a>
            </td>
          <td>
            Lilin Wan, Enyao Huang, Zicheng Xu, Bing Zheng, Xuefei Ding, Chunyan Chu, Tiange Wu, Shengrong Chen, Junyong Zhuang, Ming Chen, Rong Na, Shenghong Ju, Dingxiao Zhang, Wenchao Li, Yifei Cheng, Bin Xu
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer‐associated fibroblasts (CAFs) are a heterogeneous population of stromal cells, which modulate the immune system and can have both pro‐ and anti‐tumorigenic effects. In classic Hodgkin lymphoma (cHL), the role of CAFs has remained largely undefined. We applied multiplexed immunofluorescence imaging and spatial analysis on tumor samples from two independent cHL patient cohorts (n = 131 and n = 148) to study CAFs and their interactions with Hodgkin Reed–Sternberg (HRS) and tumor microenvironment (TME) cells at the single‐cell resolution. We show that higher proportions of CAFs are associated with favorable outcomes, independent of the clinical covariables. In contrast, a subset of CD45+ immune cells with strong fibroblast‐activation protein positivity, classified as macrophages, was less abundant in nodular sclerosis subtype and associated with worse outcomes. Neighborhood analysis allowed for the identification of colocalization or regional exclusion of phenotypically defined cell types and recurrent cellular neighborhoods. Despite the positive impact of CAF proportions on survival, patients with enrichment of platelet‐derived growth factor receptor beta (PDGFRB)‐positive CAFs in the vicinity of HRS cells had worse survival in both cohorts, independent of the clinical determinants. Our findings distinguish various subsets of CAFs and macrophages impacting survival in cHL and underscore the importance of the spatial arrangements in the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbaf0281c74f211aa49d27cd219fa5ad14a4a2cf" target='_blank'>
              Characterization of cancer‐associated fibroblasts and their spatial architecture reveals heterogeneity and survival associations in classic Hodgkin lymphoma
              </a>
            </td>
          <td>
            S. Leivonen, Kristiina Karihtala, T. Pellinen, M. Karjalainen-Lindsberg, Tomohiro Aoki, Christian Steidl, S. Leppä
          </td>
          <td>2025-05-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 SCLC is the deadliest type of lung cancer, which is mostly diagnosed at late stages and accounts for 15% of pulmonary tumors. Chemotherapy is initially effective, but SCLC patients relapse quickly and develop resistant tumors. Large-scale studies uncovered key molecular patterns in SCLC, which include universal loss of TP53 and RB1, as well as transcriptional patterns based on the expression of 4 lineage transcription factors (TF). We aimed to dissect how genomic cues impact transcriptional phenotypes, and the dynamics of tumor and immune cell interactions, to thus decipher molecular mechanisms of phenotypic divergence in SCLC.
 We performed genome and single cell transcriptome profiling on 56 SCLC patient tumors, including primary tumors, metastases and paired relapse tumors, including a patient sample after treatment with DLL3-targeted immunotherapy. Single cell RNA-seq was performed with the 10x Genomics platform to study cancer cells and the tumor micro-environment (TME), and tumor cell intrinsic heterogeneity. Data were processed with an in-house pipeline.
 Single cell expression profiles were determined for > 200000 cells from 56 tumors. Transcriptome profiling at single cell levels revealed co-expression of at least two of the lineage TF ASCL1, NEUROD1 or POU2F3 for each SCLC tumor. Transcriptional profiles pointed to a marked intra- and interpatient, as well as to inter-metastatic heterogeneity. Cell-cell-interaction analysis across all samples uncovered recurrent patterns of intra-tumoral receptor-ligand interactions. The integrative analysis of genome and single cell transcriptome data pointed to somatic alterations and copy number alterations shaping transcriptional heterogeneity. Specifically, tumors with gene amplifications of MYC family members share similar expression profiles. We performed multi-regional and longitudinal studies of matched patient cases to project genomic patterns of clonal evolution to transcriptional profiles at single cell level, thus analyzing the effect of genome diversity on the transcriptional landscape. Additionally, tumor intrinsic profiles of heterogeneity were mapped to single cell data of patient tumors to elucidate the effects on tumor-immune cell interactions.
 Altogether we provide a first comprehensive framework for the study of underlying genetic alterations that shape the transcriptional landscape and phenotypic plasticity in SCLC. Our data points to a concerted regulation of a multitude of factors including lineage TF, transcriptional programs and interaction networks in SCLC.


 Marcel Schmiel, Laura Kaiser, Julie George, Roman K. Thomas. Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC) [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3891.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Objectives: Bone metastasis is a frequent and life-threatening event in advanced cancers, affecting up to 70–85% of prostate cancer patients. Understanding the cellular and molecular mechanisms underlying bone metastasis is essential for developing targeted therapies. This study aimed to systematically characterize the heterogeneity and microenvironmental adaptation of prostate cancer bone metastases using single-cell transcriptomics. Methods: We integrated the largest single-cell transcriptome dataset to date, encompassing 124 samples from primary prostate tumors, various bone metastatic sites, and non-malignant tissues (e.g., benign prostatic hyperplasia, normal bone marrow). After quality control, 602,497 high-quality single-cell transcriptomes were analyzed. We employed unsupervised clustering, gene expression profiling, mutation analysis, and metabolic pathway reconstruction to characterize cancer cell subtypes and tumor microenvironmental remodeling. Results: Cancer epithelial cells dominated the tumor microenvironment but exhibited pronounced heterogeneity, posing challenges for conventional clustering methods. By integrating genetic and metabolic features, we revealed key evolutionary trajectories of epithelial cancer cells during metastasis. Notably, we identified a novel epithelial subpopulation, NEndoCs, characterized by unique differentiation patterns and distinct spatial distribution across metastatic niches. We also observed significant metabolic reprogramming and recurrent mutations linked to prostate-to-bone microenvironmental transitions. Conclusions: This study comprehensively elucidates the mutation patterns, metabolic reprogramming, and microenvironment adaptation mechanisms of bone metastasis in prostate cancer, providing key molecular targets and clinical strategies for the precise treatment of bone metastatic prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64291da16ef528fb561e1af0f4b9ad8a8ddde042" target='_blank'>
              Comprehensive Integrated Analysis Reveals the Spatiotemporal Microevolution of Cancer Cells in Patients with Bone-Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Yinghua Feng, Xiuli Zhang, Guangpeng Wang, Fei-ya Yang, Ruifang Li, Lu Yin, Dong Chen, Wenkuan Wang, Mingshuai Wang, Zhiyuan Hu, Yuan Sh, N. Xing
          </td>
          <td>2025-04-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd09b27d0d8fe6fa80c372bc8924f459f7a4d33" target='_blank'>
              MOSAIC: Intra-tumoral heterogeneity characterization through large-scale spatial and cell-resolved multi-omics profiling
              </a>
            </td>
          <td>
            Caroline Hoffmann
          </td>
          <td>2025-05-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178f28f36aaf5c0de57a93091858bdbe9ce93e72" target='_blank'>
              Tumor Microenvironment in Ovarian Cancer through Spatial Transcriptomics and Identification of Key Gene Expression Profiles
              </a>
            </td>
          <td>
            K. K. Kırboğa, E. Küçüksille, M. Rudrapal, E. Aktaş, R. Achar, Gouri Deshpande, V. Stupin, E. Silina
          </td>
          <td>2025-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Advances in single-cell multiomic technologies such as RNA-seq, ATAC-seq, or V(D)J sequencing have enriched our understanding of tumor gene expression, chromatin accessibility, and immune receptor diversity, but they fall short in capturing the critical spatial context of cells within the tumor microenvironment (TME). This limitation hinders our understanding of how tumor cells organize and interact with surrounding cells during tumor progression, metastasis, and treatment response. Existing spatial methods, whether microscopy- or sequencing-based, primarily focus on gene or protein expression only. Additionally, they often lack true single-cell resolution, requiring computationally involved cell segmentation or deconvolution, and offer limited molecular complexity. To address this gap, Curio Trekker single-cell spatial mapping kit, based on Slide-tags technology, provides a paradigm-shifting solution by transforming single-cell sequencing assays into spatially resolved single-cell data. Curio Trekker spatially labels individual cells within a tissue section by releasing the DNA barcodes from a spatially indexed bead array surface into nearby nuclei. The spatially tagged nuclei are then dissociated from the array and processed on conventional single-cell sequencing assays and platforms. The technique expands spatial measurement beyond gene expression alone, enabling spatial multiomic analyses such as chromatin and V(D)J profiling. Moreover, given the spatial measurements are at true single-cell resolution, the resulting datasets are easier to analyze and interpret. In this study, we applied Curio Trekker to existing single-nuclei (sn) ATACseq and sn-V(D)J assays combined with gene expression to spatially analyze human melanoma fresh frozen tissues. These previously unattainable combinations enable the simultaneous spatial exploration of gene expression and epigenetic regulation within the spatial context of tumors and the mapping of immune receptor clones within TME. Joint unsupervised clustering on sn-RNAseq and sn-ATACseq data identified spatially defined tumor, stromal, immune, and vasculature cell types driven by both transcriptomic and epigenetic signatures. These cell types were further classified spatially into tumor zones and tumor-free areas, determined by their spatial locations and CNV profiles. We linked tumor-zone-specific transcriptional factors with accessible chromatin domains and binding motifs. Tumor-stromal interactions were inferred within their native spatial context using spatially aware ligand-receptor analysis. V(D)J and immune composition analysis revealed spatial distribution of different T cell clones. Our findings demonstrate the potential of Curio Trekker to advance oncology and immunotherapy by providing a comprehensive spatial view of tumor ecosystems.


 Wanxin Wang, Julie Wilhelmy, Christina Chang, Christina Fan. Novel spatial tagging of single nuclei spatializes single-cell ATAC-seq and V(D)J sequencing with gene expression for tumor microenvironment analysis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 167.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efe6de67c775e0cfab3b3b3734f163780c5cc959" target='_blank'>
              Abstract 167: Novel spatial tagging of single nuclei spatializes single-cell ATAC-seq and V(D)J sequencing with gene expression for tumor microenvironment analysis
              </a>
            </td>
          <td>
            Wanxin Wang, Julie Wilhelmy, Christina Chang, Christina Fan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) exhibits significant intratumoral heterogeneity, driving a spectrum of phenotypes ranging from indolent disease to aggressive metastasis, challenging risk stratification and treatment. A key feature of this heterogeneity is clonal diversity, encompassing both cancer cells and their interactions with the surrounding stromal microenvironment. Understanding clonal dynamics and stromal influences on tumor progression is crucial for identifying factors that contribute to metastasis and disease lethality, ultimately improving risk stratification and therapeutic strategies.



 We performed organ-wide spatial transcriptomic analyses (VisiumTM v2, 10x Genomics) on formalin-fixed paraffin-embedded tissues from ten patients with primary, nodal metastatic PCa and biopsy samples, recruited to a local trial (ISRCTN10046036). Two pathologists meticulously annotated each spot, including benign, Gleason grade, stroma, and immune cells. Fastq data were processed using SpaceRangerTM, and inferred clonal genomic identities were analyzed with SpatialInferCNV (https://github.com/aerickso/SpatialInferCNV). We reconstructed clonal evolution using phylogenetic trees and analyzed stromal gene expression around distinct tumor clones using differential expression analyses. In addition, we performed multiplex imaging on consecutive sections to assess protein expression and correlate it with gene expression data, enabling the identification of key tumor-stroma interactions associated with metastatic progression.



 We analyzed 122 spatial transcriptomics capture areas, collectively representing over 1, 000, 000 spots (55 µm diameter), or approximately 16, 000, 000 cells. We identified clonal subtypes across biopsy, primary and metastatic tissues, revealing distinct clonal relationships and polyclonal lymph node colonization at different evolutionary stages. Stromal cells around metastatic progenitor clones showed upregulation of antigen presentation and inflammatory pathway genes, contrasting with stromal profiles surrounding lethal clones (as defined by their ability to metastasize to lymph nodes). These findings suggest a dynamic interplay between tumor cells and their stroma, potentially influencing metastatic potential and immune evasion.



 This study provides a detailed cellular-level characterization of clonal heterogeneity in PCa and its relationship with stromal dynamics, offering insights into mechanisms driving metastatic progression. These insights could inform strategies for risk stratification and therapeutic intervention, including leveraging dual tumor-stromal profiles as biomarkers to distinguish lethal from indolent disease. Understanding the interplay between tumor clones and their stroma opens new avenues for targeted treatment approaches aimed at preventing metastasis and improving patient outcomes.



 Sandy Figiel, Mengxiao He, Wencheng Yin, Emmanouela Perisynaki, Renuka Teague, Thineskrishna Anbarasan, Eleanor O’Roberts, Nithesh Ranasinha, Sophia M. Abusamra, Trishna Desai, Dimitrios Doultsinos, Andrew Erickson, Srinivasa Rao, Clare Verrill, Richard Colling, Pelvender Gill, Tuomas Mirtti, Ibrahim Kulac, Karl Smith-Byrne, Ruth Travis, Charlotte Stadler, Richard J. Bryant, Freddie C. Hamdy, Dan J. Woodcock, Ian G. Mills, Joakim Lundeberg, Alastair D. Lamb. Mapping clonal heterogeneity and stromal dynamics in prostate cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2663.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a1265fddcea3943ae47df40af80fa7b2b32f84" target='_blank'>
              Abstract 2663: Mapping clonal heterogeneity and stromal dynamics in prostate cancer
              </a>
            </td>
          <td>
            S. Figiel, Mengxiao He, Wencheng Yin, Emmanouela Perisynaki, Renuka Teague, T. Anbarasan, Eleanor M O'Roberts, N. Ranasinha, Sophia M. Abusamra, Trishna A. Desai, D. Doultsinos, Andrew Erickson, Srinivasa Rao, Clare Verrill, Richard Colling, P. Gill, T. Mirtti, Ibrahim Kulac, K. Smith-Byrne, Ruth C. Travis, Charlotte Stadler, Richard J. Bryant, Freddie C Hamdy, Dan Woodcock, Ian G. Mills, J. Lundeberg, Alastair D. Lamb
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Pituitary neuroendocrine tumors (PitNETs) can be invasive or aggressive, yet the mechanisms behind these behaviors remain poorly understood, impeding treatment advancements. Here, we integrat single-cell RNA sequencing and spatial transcriptomics, analyzing over 177,000 cells and 35,000 spots across 57 tissue samples. This comprehensive approach facilitates the identification of PitNETs tumor populations and characterizes the reconfiguration of the tumor microenvironment (TME) as PitNETs progress and invade. We trace the trajectory of TPIT-lineage PitNETs and identify an aggressive tumor cluster marked by elevated p53-mediated proliferation and a higher Trouillas classification, both associated with tumor progression. Additionally, we document the heterogeneity of immune stromal cells within PitNETs, particularly noting the enrichment of SPP1+ tumor associated macrophages (TAMs) in invasive tumors. These TAMs facilitate tumor invasion through the SPP1-ITGAV/ITGB1 signaling pathway. Our in-depth single-cell and spatial analysis of PitNETs uncovers the molecular dynamics within the TME, suggesting potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b1b01507bfc7f0aa4eb306817808e84f0ce1133" target='_blank'>
              Single-cell and spatial transcriptome analyses reveal tumor heterogeneity and immune remodeling involved in pituitary neuroendocrine tumor progression
              </a>
            </td>
          <td>
            Wan Su, Zhang Ye, Jifang Liu, K. Deng, Jinghua Liu, Huijuan Zhu, Lian Duan, Chen Shi, Linjie Wang, Yuxing Zhao, Fengying Gong, Yi Zhang, Bo Hou, Hui You, Feng Feng, Qing Ling, Yu Xiao, Yongdong Guo, Wenyi Fan, Sumei Zhang, Zixin Zhang, Xiaomin Hu, Yong Yao, Chunhong Zheng, Lin Lu
          </td>
          <td>2025-05-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Delayed relapse is attributable to residual tumor cells that remain minimally proliferative/quiescent until favorable conditions for growth are met, but what determines the overall duration of remission is unresolved. Recent evidence suggests that epithelial-mesenchymal plasticity (EMP) facilitates phenotypic switching between dormant and proliferative cell states, in addition to enabling chemoresistance, metastasis, and immune evasion. However, which features of EMP are responsible for each or multiple of these phenotypes, and to what extent EMP is restricted to certain tumor populations remains unclear. To address these questions, we initially enriched for features of EMP (CD44hi, CD104low) in the mouse mammary tumor cell line TSAE1. Interestingly, no difference in tumor growth was observed between parental or EMP-enriched mammary fat pad (MFP) tumors in either syngeneic Balb/c or immune-deficient SCID-beige mice. Individual tumors were then cultured ex vivo, revealing a significant EMP-enrichment in parental tumors after one round of selection in vivo. Similar results were observed in the mouse mammary cell line D2.OR, suggesting that implantation alone gives rise to EMP which may mask important tumor cell-intrinsic differences in a heterogeneous tumor. Thus, EMP-enriched TSAE1 cells were further divided by isolating nine clonal lines. Although each expressed less epithelial markers than parental cells (Epcam, E-cadherin), individual TSAE1 clones displayed remarkably divergent growth rates and patterns in 3D culture. In vivo all clones were capable of forming tumors, but some were significantly delayed regardless of being implanted into Balb/c or SCID-beige animals. Furthermore, individual clones recruited distinct immune and stromal populations. To assess if intrinsic phenotypes change over time, 3D culture assays and bulk RNA sequencing was performed on three representative clones before implantation (“Pre”) and on three Balb/c MFP-derived tumors expanded ex-vivo from each clone (“Post”). Surprisingly, for each clone the Post tumors closely resembled the Pre clone and much of the difference in up/downregulated genes between individual clones and parental TSAE1 cells was already found in Pre samples. Finally, MFP tumors from EMP-enriched TSAE1 clones were differently sensitive to Doxorubicin therapy, suggesting that specific aspects of EMP may be important during different stages of BC survival and progression. Together, these data reveal that proliferation itself is intrinsically determined at the clonal level, independent of adaptive immunity, and highlight the potential for individual tumor clones in various EMP states to shape the tumor microenvironment for long-term survival.


 Timothy N. Trotter, Jason McBane, Xing Gao, Carina Dagotto, Tao Wang, H. Kim Lyerly, Zachary C. Hartman. Tumor clones display intrinsic and stable rates of progression and coordination of the tumor microenvironment in association with epithelial mesenchymal plasticity. [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2675.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11e4221fc799886f045c92cc1f8df645b2f899a5" target='_blank'>
              Abstract 2675: Tumor clones display intrinsic and stable rates of progression and coordination of the tumor microenvironment in association with epithelial mesenchymal plasticity
              </a>
            </td>
          <td>
            T. Trotter, Jason McBane, Xing Gao, Carina E Dagotto, Tao Wang, H. Lyerly, Z. Hartman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 The spatial organization and cellular composition of the tumor microenvironment (TME) has the potential to inform clinical and translational researchers about mechanisms of disease progression and therapeutic success. Of particular interest are immune cells, especially T cells, which can become exhausted due to chronic stimulation. This poses a significant challenge in cancer therapy as exhausted T cells have reduced effector functions and sustained expression of inhibitory receptors such as PD-1 and CTLA-4, resulting in failure to effectively eliminate tumor cells.
 Imaging Mass Cytometry™ (IMC™) technology is a high-plex imaging technique that enables deep characterization of the heterogeneity and complexity of the TME. The Hyperion™ XTi Imaging System utilizes IMC technology to provide signal intensities over a wide dynamic range and entails one-step detection of 40-plus markers without issues of tissue autofluorescence, making it ideally suited for spatial biology applications. Whole slide imaging modes and an automated slide loader function enable a streamlined, versatile scalable workflow for high-throughput analysis.
 We used IMC technology to characterize immune cell populations and the spatial distribution of T cell exhaustion markers across various cancer types. We applied a 40-plus-marker IMC panel by integrating the Human Immuno-Oncology IMC Panel with the Human T Cell Exhaustion IMC Panel to study the TME of multiple human tissues.
 IMC analysis revealed striking heterogeneity with distinct tumor and immune-rich niches in the TME. In colon adenocarcinoma, we detected functional effector T cells and multiple tertiary lymphoid structures (TLSs). In contrast, we observed fewer TLSs and more T cells expressing PD-1 and TIM-3 at the tumor periphery in the urothelial carcinoma tissue, suggesting that they are in advanced exhaustion stages. We also observed expression of PD-L1, IDO and VISTA in the tumor cells. t-SNE and PhenoGraph clustering analysis of Cell Mode data segregated and spatially resolved the T cells into effector and exhausted T cell subpopulations, which were mapped back to the segmented cell masks. Unsupervised pixel clustering and analyses of Tissue Mode data further delineated distinct tumor areas based on the presence of infiltrating immune cells, tumor replicative activity and spatial proximity to stromal components.
 This work characterizing T cell exhaustion markers in multiple cancers showcases the capabilities of IMC technology and establishes it as a reliable high-plex, high-throughput spatial biology imaging platform. IMC technology is ideally suited for developing future translational and clinical applications and has the potential to help guide personalized therapeutic strategies for cancer treatment.
 For Research Use Only. Not for use in diagnostic procedures.


 Thomas D. Pfister, Jyh Yun Chwee, Qanber Raza, Nikesh Parsotam, David Howell, Liang Lim, Christina Loh. Spatial profiling of exhausted T cells using high-plex imaging mass cytometry [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2089.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f029b0348e3538dc977fb4d11569d99d619e9e1" target='_blank'>
              Abstract 2089: Spatial profiling of exhausted T cells using high-plex imaging mass cytometry
              </a>
            </td>
          <td>
            T. Pfister, J. Y. Chwee, Qanber Raza, Nikesh Parsotam, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb74e2d2b03a62ef097600aaf5d84f08dfc469c4" target='_blank'>
              Prognostically favorable immune responses to ovarian cancer are distinguished by self-reactive intra-epithelial plasma cells
              </a>
            </td>
          <td>
            Allyson C. Banville, Céline M. Laumont, Karanvir Singh, Breeze Gladwin, Jaden Dedora, Liam Mitchell, Jen Wang, A. Miranda, Bridget Mateyko, E. Chavez, G. Negri, Se Wing Grace Cheng, Julian Smazynski, K. Milne, Kylee Wright, Malia Lampard, Nicole S. Gierc, P. Hamilton, Sandra E. Spencer, Shreena Kalaria, Talen Oostenbroek, Victor Negrea Puskas, Christian Steidl, G. Morin, F. Nathoo, Brad H Nelson
          </td>
          <td>2025-05-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Desmoplastic small round cell tumor (DSRCT) is a rare, aggressive sarcoma subtype that harbors a pathognomonic EWSR1::WT1 fusion protein (FP). DSRCT is usually incurable and FP-targeted agents are nonexistent. The five-year survival rate is below 20%. Tumor nests are classically surrounded by desmoplastic stroma in DSRCT, yet the interactions between the tumor cells and the stromal cells are poorly understood. Given this context, our study aims to understand the spatial heterogeneity within DSRCT and the tumor-stroma interactions. We utilized a 20-panel marker to study the cellular constituents of the tumor microenvironment (TME) in DSRCT and to characterize its multilineage expression, including epithelial and neural (NE) signatures. The markers included: DSRCT (ST6GALNAC, pan-Cytokeratin), Fibroblasts (Collagen, α-smooth muscle actin), Endothelial Cells (CD31, α-smooth muscle actin), T-cells (CD45, CD4, CD8), Macrophages (CD45, CD68, CD163, CD11c), and AR/NE markers (androgen receptor, neural-specific enolase, synaptophysin). Marker expression was explored in a nine-patient cohort using 9 x 9 mm2 tissue sections imaged by the Lunaphore COMET system. A deep learning model, Visiopharm, extracted high-quality regions and segmented cells based on nuclear staining. Spatial data were transformed into matrix format for analysis in Python, where UMAP and Leiden clustering identified distinct clusters consisting of tumoral cells, fibroblasts, and endothelial cells. Our analysis revealed two distinct fibroblast clusters—one with high α-SMA expression and the other with high collagen expression. These two clusters exhibited distinct spatial distributions and proportions across patient samples, indicating heterogeneity in stromal composition within the DSRCT microenvironment. Our previous study identified three DSRCT subtypes: AR+/NE-, AR-/NE+, and Hybrid AR+/NE+. The current spatial analysis suggests that AR+/NE- tumor nests are consistently surrounded by α-SMA-high fibroblasts, hinting at subtype-specific stromal interactions. Preliminary findings from co-culture experiments of DSRCT cells with human mesenteric fibroblasts suggest that fibroblasts enhance the proliferative capacity of DSRCT cells. These findings provide novel insights into the stromal heterogeneity of the DSRCT microenvironment and its association with tumor subtypes. Our future work will focus on quantifying spatial relationships and patterns, which includes how tumor nest characteristics are associated with fibroblast cell type or DSRCT subtype. Proximity and neighborhood analysis in python and Visiopharm analysis will be applied. Ultimately, our results might reveal novel drug targets that stop oncogenic tumor-stroma signaling.


 Jiaqian Fan, Adewale A. Adebayo, Diana Shamsutdinova, Davis Ingram, Clement Agyemang, Roberto Cardenas-Zuniga, Alexander Lazar, Danh Truong, Joseph Ludwig. Spatial heterogeneity of tumor and stromal cells in desmoplastic small round cell tumors [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5323.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/613002ca1a9de7d4f7ae92b0493c480e7bbe4c62" target='_blank'>
              Abstract 5323: Spatial heterogeneity of tumor and stromal cells in desmoplastic small round cell tumors
              </a>
            </td>
          <td>
            Jiaqian Fan, Adewale A. Adebayo, Diana Shamsutdinova, D. Ingram, Clement Agyemang, R. Cárdenas-Zúñiga, Alexander J. Lazar, Danh D. Truong, Joseph A Ludwig
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) has a poor prognosis, since most PDAC tumors develop resistance to the standard of care (SoC) treatments like chemotherapy, radiation, and targeted therapies. All tumors consist of multiple, genetically related subpopulations of cancer cells that evolve in parallel and display heterogeneity at genomic, epigenetic, or phenotypic levels. As cancer develops, some subpopulations of cancer cells may show faster growth, increased metastatic potential, and resistance toward SoC treatment. A key challenge in PDAC management is understanding chemoresistance driven by cancer subpopulation dynamics. This would broaden our understanding of tumor adaptation to treatments and guide targeting resistant subpopulations to improve therapeutic outcomes. In this study, we identified PDAC cell subpopulations resistant to chemotherapy using our DNA barcoding system B-GLI (barcode-guide lineage isolation). The B-GLI system leverages a highly complex DNA barcode library and CRISPR activation (CRISPRa) to trace and isolate sub-lineages within heterogeneous cell populations by their DNA barcodes. PDAC cell lines PANC-1 and Mia-PaCa-2 were barcoded with the B-GLI barcode library, which consist of two optimized guide RNA binding sites in each barcode. After DNA barcoding, we performed treatments with two SoC chemotherapeutics, gemcitabine + paclitaxel and FOLFIRINOX, to induce a selection pressure in the PDAC cell lines, and then identified the resistant subpopulations through differentially represented barcodes and sequencing. Furthermore, we designed barcode-specific single guide RNAs (sgRNAs) targeting the resistant subpopulations and activated the expression of the puromycin resistance gene by CRISPRa. This allowed us to enrich and isolate chemoresistant subpopulations by puromycin treatment. After having isolated the resistant PDAC subpopulations, we performed ATAC-seq and RNA-seq for molecular characterization of the resistant and parental cell populations, along with phenotypic drug screening with 384 compounds to identify novel treatment vulnerabilities. The compound testing confirmed that the resistant cells were less sensitive to the two SoC treatments, as well as identified selective sensitivity to specific compounds. ATAC-seq and RNA-seq data will allow us to decode whether and how chromatin structure and transcriptomic changes are associated with the treatment resistance, and the confirmation screen of the drug sensitivities will identify potential compounds for future pre-clinical and clinical testing in PDAC models and patients with resistant disease. In summary, this study will identify targeted treatment alternatives, accompanied by molecular biomarkers for chemotherapy resistant PDAC.


 Subhendu Roy Choudhury, Shixiong Wang, Yevhen Akimov, Katarina Willoch, Biswajyoti Sahu, Alfonso Urbanucci, Thomas Fleischer, Tero Aittokallio. Identification, isolation and molecular characterization of drug-resistant sub-populations of pancreatic cancer cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5544.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc3b5fa458a00b7ac6aa3d0d25d390376657ca" target='_blank'>
              Abstract 5544: Identification, isolation and molecular characterization of drug-resistant sub-populations of pancreatic cancer cells
              </a>
            </td>
          <td>
            S. Choudhury, Shixiong Wang, Y. Akimov, Katarina Willoch, Biswajyoti Sahu, Alfonso Urbanucci, Thomas Fleischer, T. Aittokallio
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Objective This study examines the cellular heterogeneity of epithelial cells within pancreatic ductal adenocarcinoma (PDAC) and their contributions to tumor progression, metastasis, and immunosuppressive interactions using single-cell RNA sequencing. Methods Single-cell RNA-sequencing data from two datasets (GSE154778 and GSE158356) were integrated using the Harmony algorithm, followed by quality control, clustering, and differential gene expression analysis. Distinct subpopulations of epithelial cells were identified, and their gene expression profiles were analyzed. To assess the malignancy of these subpopulations, single-cell copy number variation (CNV) analysis and trajectory analysis were conducted. Additionally, intercellular communication was examined using the CellChat platform. Results The analysis revealed pronounced heterogeneity among PDAC epithelial cells, with specific subpopulations exhibiting distinct roles in tumor proliferation, extracellular matrix remodeling, and metastatic dissemination. Subpopulations 4 and 6 were characterized by increased CNV levels and a more malignant phenotype, suggesting an enhanced capacity for metastasis. Single-cell trajectory analysis, along with CellChat, mapped the temporal evolution of epithelial cells, identifying key regulatory genes such as DCBLD2 and JUN. A prognostic model incorporating five key genes, including KLF6, was developed and demonstrated strong predictive accuracy for patient outcomes. Notably, KLF6 emerged as a critical prognostic marker associated with immune modulation, particularly through interactions with M2 macrophages. Conclusion The study highlights the pronounced heterogeneity of epithelial cells in PDAC and their distinct contributions to tumor progression, metastasis, and immune modulation. Through single-cell transcriptomic and CNV analyses, we identified epithelial subpopulations with varying malignant potentials and distinct interactions with the tumor microenvironment. Among these, KLF6 emerged as a key regulator associated with immune modulation and metastasis. Our findings emphasize the significance of epithelial cell heterogeneity in shaping pancreatic cancer progression. These insights provide a foundation for future investigations into novel prognostic markers and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12957-025-03793-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c02e096347df4014f9175b24f19fbd3cae4aa9d0" target='_blank'>
              Deciphering the heterogeneity of epithelial cells in pancreatic ductal adenocarcinoma: implications for metastasis and immune evasion
              </a>
            </td>
          <td>
            Jie Liu, Hui Li, Xiuyun Lin, Jiani Xiong, Guangfeng Wu, Lingyan Ding, Bin Lin
          </td>
          <td>2025-04-16</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Invasive ductal carcinoma (IDC) is the most common and aggressive form of breast cancer, characterized by tissue invasion and metastasis, contributing to significant morbidity and mortality. The tumor microenvironment (TME), composed of stromal and immune cells, plays a critical role in IDC progression and therapy resistance. However, the spatial organization and molecular heterogeneity within the IDC TME remain underexplored. Spatial transcriptomics enables the preservation of tissue architecture while revealing molecular signatures at single-cell resolution, offering new insights into cancer biology. We utilized the CosMx® Spatial Molecular Imager (SMI) platform with the CosMx Whole Transcriptome panel (approximately 19, 000 genes) to analyze a formalin-fixed, paraffin-embedded (FFPE) IDC tumor sample. The platform provides high-resolution, multiplexed RNA detection within its native tissue context. Hematoxylin and eosin (H&E) staining of the same slide provided a histological reference for spatial transcriptomics. Transcriptomic data were analyzed using Uniform Manifold Approximation and Projection (UMAP) for dimensionality reduction and Leiden clustering to identify cell clusters and spatial patterns. Marker genes for each cluster were used to annotate cell types and assess their roles within the TME. Analysis identified two distinct tumor regions, each subdivided into subregions with unique and shared gene expression profiles. These tumor regions aligned with histological features observed in H&E staining, validating the spatial transcriptomic findings. We observed region-specific stromal compositions, with fibroblast populations exhibiting distinct gene expression patterns. For example, one region was enriched for PIGR, CLU, and LTF, markers linked to immune response, tumor progression, and tumor-suppressive properties. These results highlight the molecular and spatial heterogeneity of IDC and the complex interactions within the TME. This study demonstrates the potential of spatial transcriptomics to uncover the molecular and spatial complexity of IDC. Integrating transcriptomic data with histological validation revealed novel insights into the TME, highlighting key gene signatures and stromal dynamics. The CosMx platform is a powerful tool for advancing precision oncology and guiding future therapeutic strategies in IDC.
 -Bruker Confidential-


 Liang Zhang, Shanshan He, Michael Patrick, Stefan Rogers, Haiyan Zhai, John Lyssand, Michael Rhodes, Joseph Beechem. Single-cell spatial whole transcriptome reveals tumor heterogeneity and stromal dynamics in invasive ductal carcinoma of the breast [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2067.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13f1f42b49de112548e7d274daf610e71f0196dd" target='_blank'>
              Abstract 2067: Single-cell spatial whole transcriptome reveals tumor heterogeneity and stromal dynamics in invasive ductal carcinoma of the breast
              </a>
            </td>
          <td>
            Liang Zhang, Shanshan He, Michael Patrick, Stefan Rogers, Haiyan Zhai, John S Lyssand, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b25dc1e79337f8eda4203cdcc57a33da41f963c" target='_blank'>
              Longitudinal single-cell RNA sequencing of a neuroendocrine transdifferentiation model reveals transcriptional reprogramming in treatment-induced neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            F. Sar, Hee Chul Chung, Yen-Yi Lin, Dong Lin, Tunc Morova, A. Haegert, Stanislav Volik, Robert Bell, Stéphane LeBihan, Dogancan Ozturan, Hui Xie, Xin Dong, Rebecca Wu, Jing Li, Hsing-Jien Kung, M. Gleave, Nathan A. Lack, Yuzhuo Wang, Colin C. Collins
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Colorectal cancer (CRC) is a leading cause of cancer-related deaths. An important contributor to CRC development, maintenance and metastasis is the tumor microenvironment (TME). This cellular milieu contains neighboring tumor epithelial cells, normal epithelium, fibroblasts, endothelial cells, immune cells and many others. The characterization of the TME is critical for understanding tumor biology as well as determining specific features with clinical implications. Representing a subset of the TME, the tumor immune microenvironment (TIME) has specific cellular characteristics that are important predictors for the response to treatments such as immunotherapy. However, the accurate interpretation of complex spatial cell patterns poses significant challenges. To address these issues, we developed a novel computational framework that systematically analyzes spatial data. We conducted a spatial analysis using imaging mass cytometry applied to stage III colorectal adenocarcinomas. This study used multiplexed markers to distinguish individual cells and their spatial organization from 52 colorectal cancers. We determined the landscape features of cellular spatial features in the CRC tumor microenvironment. This spatial single-cell analysis identified 10 unique cell phenotypes in the tumor microenvironment that included stromal and immune cells with a subset which had a proliferative phenotype. These special features included spatial neighborhood interactions between single cells as well as different tissue niches, especially the tumor infiltrating lymphocyte regions. We applied a robust statistical analysis to identify significant correlations of cell features with phenotypes such as microsatellite instability or recurrence. We determined that microsatellite stable (MSS) colorectal cancers had an increased risk of recurrence if they had the following features: 1) a low level of stromal tumor-infiltrating lymphocytes, and 2) low interactions between CD4+ T cells and stromal cells. Our computational framework precisely defined two types of tumor-infiltrating lymphocytes (TILs) based on their spatial properties in the local tumor microenvironment: 1) intra-tumoral TILs within cancer CNs and 2) sTILs within stromal CNs. These spatial characterizations of TILs provide deeper insights into their role, beyond simple lymphocyte abundance. Moreover, our results point to the utility of spatial single-cell interaction analysis in defining novel features of the tumor immune microenvironments and providing useful clinical cell-related spatial biomarkers.


 HoJoon Lee, Andrew Su, Minh Tran, Richard Delacruz, Anuja Sathe, Xiangqi Bai, Ignacio Wichmann, Lance Pflieger, Bryce Moulton, Tyler Barker, Derrick Haslem, David Jones, Lincoln Nadauld, Quan Nguyen, Hanlee P. Ji, Terence Rhodes. The spatial landscape of tumor-infiltrating lymphocytes in stage III colorectal cancers by single-cell spatial analysis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5263.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb7304b0f5fc3d1e6eaf91e29f30f258a3c3f94f" target='_blank'>
              Abstract 5263: The spatial landscape of tumor-infiltrating lymphocytes in stage III colorectal cancers by single-cell spatial analysis
              </a>
            </td>
          <td>
            Hojoon Lee, Andrew Su, Minh Tran, Richard Delacruz, A. Sathe, Xiangqi Bai, Ignacio A Wichmann, Lance Pflieger, Bryce Moulton, Tyler Barker, D. Haslem, David Jones, Lincoln Nadauld, Quan H Nguyen, Hanlee P. Ji, Terence Rhodes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Identification of the consensus molecular subtypes (CMS) opened significant potential for understanding the tumor biology and intertumoral heterogeneity of colorectal cancer (CRC). Molecular subtyping in CRC traditionally relies on bulk transcriptomics. However, single-cell transcriptomics studies have shown that CRC tumors may be composed of tumor cells displaying different CMS traits, suggesting additional intratumoral heterogeneity.



 To investigate intratumoral heterogeneity at protein level, we constructed tissue microarrays using tumor cores from 222 CRC patients. Arrays were stained and imaged with Cell DIVE multiplexed immunofluorescence (MxIF) imaging using 54 selected protein markers, ranging from cell identity markers to cancer hallmark proteins of metabolism, proliferation, cell death and differentiation. RNA sequencing data and CMS classification were also available for all patients. After segmentation of tumor, stromal and immune cells, we investigated intratumoral heterogeneity within CMS subtypes using spatially resolved single-cell protein profiling (>2 million cells). We compared cell types, their spatial organization and their expression of cancer hallmark-related proteins in CMS 1-4 subtypes.



 We revealed tissue atlases illustrating the cell types/states, spatial heterogeneity, cellular neighborhoods, cellular network, and single-cell protein profiles of CMS tumors. CMS1 tumors had more CD3+, CD8+, and PD1+ immune cells that were found in the epithelial layer more frequently than in the other subtypes. CMS1 was also associated with higher levels of metabolic reprogramming markers such as upregulated glycolysis. CMS2 showed immune segregation, reactive stroma patterns, and higher levels of apoptotic and proliferative signaling proteins. CMS3 exhibited tightly clustered cancer cells with high RIP3 and low FLIP levels, suggesting a pro-inflammatory microenvironment driven by intense cell-cell interactions and stromal hubs. CMS4 displayed stromal-centric and immune-evasive tumors characterized by decreased HLA-1 levels and regulatory T cell exclusion from epithelium.



 We present the first spatial atlas of CRC tumors with regard to their molecular subtypes at single-cell protein resolution and demonstrate new spatial aspects of tumor structures in CRC.



 Batuhan Kisakol, Anna Matveeva, Sanghee Cho, Mohammedreza Azimi, Andreas U. Lindner, Manuela Salvucci, Elizabeth McDonough, Joanna Fay, Tony O'Grady, Niamh McCawley, John P. Burke, Deborah McNamara, John Graf, Simon McDade, Daniel B. Longley, Fiona Ginty, Jochen H. Prehn. High-resolution spatial proteomics characterizes colorectal cancer consensus molecular subtypes [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 161.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de704d9a88b35d65b9fbe36b0e36b4cae7473ce" target='_blank'>
              Abstract 161: High-resolution spatial proteomics characterizes colorectal cancer consensus molecular subtypes
              </a>
            </td>
          <td>
            Batuhan Kısakol, Anna Matveeva, Sanghee Cho, Mohammedreza Azimi, A. Lindner, M. Salvucci, E. McDonough, Joanna Fay, Tony O’Grady, N. McCawley, John P Burke, Deborah A. McNamara, John Graf, Simon McDade, Daniel B. Longley, Fiona Ginty, Jochen H M Prehn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Stromal cells in the prostate tumor microenvironment (TME) affect tumor progression and exhibit both a universal response to tumorigenesis and specific ones determined by the genotype that drives the tumor (Pakula et al., Nat Commun 2024). As a result, stromal clusters associated with murine and human prostate cancer identified by single cell RNA sequencing (scRNA-seq) fall into a) common subtypes: myofibroblasts/pericytes and innate immunity-associated; as well as b) two additional ones specific to either AR positive adenocarcinomas or neuroendocrine tumors (NEPC). Here, we defined their spatial localization leveraging spatial transcriptomics and multiplex in situ RNA hybridization.



 We computationally identify the most discriminative genes able to distinguish the four stromal groups. Specifically, we first identified highly discriminatory genes representative of each cluster. We then used integer programming-based optimization to obtain small sets of genes (n<4) that are differentially expressed in each group compared to the remaining three. For instance, group 0, encompassing myofibroblasts and pericytes, was characterized by the expression of Rgs5 and Myh11, while group 2, whose stromal cells are characterized by the expression of Ar and Wnt signaling genes, was characterized by the expression of Wif1 and Coch. A decision-tree analysis was able to discriminate all the different stromal groups. These were validated using 1) in situ RNA hybridization + multiplex immunofluorescence panels and 2) spatial transcriptomics. Myofibroblasts/pericytes were found to be distant to tumor cells, common clusters were associated with all tumors of diverse phenotype (Tmprss2-ERG, cMYC, Pten-/-, Pten-/-/ Rb1-/- NMYC+/+), while periostin high/AR low clusters were immediately adjacent to NEPC foci in mouse models.



 Our integrated computational and experimental approach establishes a robust pipeline for validating stromal markers identified through scRNA-seq, enabling spatial mapping of stromal heterogeneity in prostate cancer.



 Filippo Pederzoli, Hubert Pakula, Wikum Dinalankara, Jeanne Jiang, Mohamed Omar, Silvia Rodrigues, Pier Vitale Nuzzo, Fabio Socciarelli, Giuseppe Nicolò Fanelli, Luigi Marchionni, Massimo Loda. In situ hybridization-based validation of transcriptionally-defined stromal clusters in the prostate cancer microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2584.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7516851ad82e74ba0e5d8528ea22ebc6695c2d0" target='_blank'>
              Abstract 2584: In situ hybridization-based validation of transcriptionally-defined stromal clusters in the prostate cancer microenvironment
              </a>
            </td>
          <td>
            Filippo Pederzoli, H. Pakula, Wikum Dinalankara, Jeanne Jiang, Mohamed Omar, Silvia Rodrigues, P. Nuzzo, Fabio Socciarelli, G. Fanelli, L. Marchionni, Massimo Loda
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Cancer-associated fibroblasts (CAFs) are a highly diverse cell population that plays a crucial role in shaping the tumor microenvironment (TME). Their spatial interactions within the TME remain to be systematically characterized, and the factors driving their phenotypic heterogeneity are not yet fully understood. Recent advancements in single-cell spatial transcriptomics technologies and cell segmentation approaches enable the precise measurement of thousands of transcripts at subcellular resolution. Combined with innovative bioinformatics, these advancements have unlocked unprecedented opportunities for in-depth characterization of CAFs and their interactions within the TME.



 We conducted a comprehensive spatial transcriptomic analysis of CAFs across 8 cancer types, using the CosMx and MERSCOPE platforms to profile nearly 6 million cells as part of our discovery cohorts. We developed a novel computational method to spatially cluster CAF subpopulations and defined four distinct spatial CAF subtypes. We further examined their phenotypic differences, spatial niche organization, and interactions with neighboring cells. Additionally, we investigated the influence of CAF subtypes on immune cell phenotypes and infiltration levels. To validate our findings, we employed multiple spatial platforms, including Visium, Xenium, COMET, CODEX, and IMC, to ensure reproducibility across diverse technologies. Moreover, we performed correlative studies in large-cohorts with rich clinical data to evaluate the clinical significance of these CAF subtypes.



 Our approach provided an unprecedented high-resolution examination of the spatial phenotypes of CAFs in large, complex tissues, uncovering novel insights. We comprehensively characterized CAFs across various solid tumor types and identified four conserved spatial CAF subtypes. These subtypes are characterized by distinct cell organizational patterns, neighboring cell compositions, interaction networks, and transcriptomic features. Moreover, they demonstrate significant correlations with the abundance, distribution, and phenotypes of tumor-infiltrating immune cells, as well as associations with tumor histopathological features and clinical outcomes.



 Our study provides a pan-cancer, high-resolution characterization of CAF spatial organization and interactions in solid tumors and identified conserved spatial CAF subtypes linked to distinct organizational patterns. Our findings reveal that the transcriptomic states and phenotypes of CAFs are significantly shaped by their interactions with neighboring cancer and immune cells. Beyond offering novel insights into the spatial heterogeneity of CAFs, this work paves the way for developing strategies to effectively target and modulate CAFs for therapeutic benefit.



 Yunhe Liu, Ansam Sinjab, Jimin Min, Guangchun Han, Francesca Paradiso, Yuanyuan Zhang, Yang Liu, Ruiping Wang, Guangsheng Pei, Kyung Serk Cho, Enyu Dai, Akshay Basi, Jared K. Burks, Kimal I. Rajapakshe, Yanshuo Chu, Jiahui Jiang, Daiwei Zhang, Xinmiao Yan, Paola A. Guerrero, Alejandra Serrano, Mingyao Li, Tae Hyun Hwang, Andrew Futreal, Jaffer A. Ajani, Luisa M Solis Soto, Amir A. Jazaeri, Humam Kadara, Anirban Maitra, Linghua Wang. Spatial subtypes and cellular interactions of cancer-associated fibroblasts revealed by single-cell spatial omics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 166.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b2e8b55d68338e1058f7b78eb076c9fe5aa970" target='_blank'>
              Abstract 166: Spatial subtypes and cellular interactions of cancer-associated fibroblasts revealed by single-cell spatial omics
              </a>
            </td>
          <td>
            Yunhe Liu, Ansam Sinjab, Jimin Min, Guangchun Han, Francesca Paradiso, Yuanyuan Zhang, Yang Liu, Ruiping Wang, Guangsheng Pei, K. S. Cho, Enyu Dai, Akshay V. Basi, Jared K. Burks, K. Rajapakshe, Yanshuo Chu, Jiahui Jiang, Daiwei Zhang, Xinmiao Yan, Paola A. Guerrero, Alejandra G Serrano, Mingyao Li, Tae Hyun Hwang, Andrew Futreal, Jaffer A Ajani, Luisa M. Solis Soto, A. Jazaeri, H. Kadara, Anirban Maitra, Linghua Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Lung cancer, the leading cause of cancer-related deaths worldwide, primarily consists of non-small cell lung cancer (NSCLC), including subtypes like lung adenocarcinoma (LUAD) and lung squamous cell carcinoma (LUSC). These subtypes exhibit distinct histopathological features but share overlapping molecular characteristics. Understanding the spatial and molecular heterogeneity of these tumors is critical for identifying novel biomarkers and therapeutic targets. This study utilizes the CosMx® Spatial Molecular Imager (SMI) platform to profile LUAD and LUSC, aiming to uncover shared and unique gene signatures, tumor heterogeneity, and rare tumor cell subtypes, while integrating histopathological insights from H&E staining. Two formalin-fixed, paraffin-embedded (FFPE) lung tumor samples—one LUAD and one LUSC—were profiled using the CosMx Whole Transcriptome (WTX) panel, which profiles approximately 19,000 genes with single-cell subcellular resolution. Spatial transcriptomic data were analyzed using Uniform Manifold Approximation and Projection (UMAP) for dimensionality reduction and Leiden clustering to identify cell clusters. Marker genes were used to annotate cell types and generate a high-resolution cellular atlas of the tumor microenvironment (TME). Hematoxylin and eosin (H&E) staining was performed on the same slides, providing histopathological context and validating CosMx-derived cell types, including immune and tumor hotspots. The alignment of H&E and CosMx SMI data highlights their potential for AI model training that integrates spatial transcriptomics and histopathology. Our analysis revealed shared and unique gene signatures between LUAD and LUSC. Shared signatures included common tumorigenic processes such as cell proliferation, immune modulation, and angiogenesis. Unique signatures distinguished each subtype, with LUAD enriched in epithelial-to-mesenchymal transition pathways, while LUSC showed upregulation of squamous differentiation markers. UMAP and Leiden clustering identified distinct tumor and immune cell subpopulations, demonstrating spatial and molecular heterogeneity with clear immune and tumor hotspots. CosMx SMI data and H&E staining corresponded well, suggesting their potential for AI model integration. Integrating spatial transcriptomics and histopathology using CosMx SMI provided valuable insights into the molecular and spatial complexity of LUAD and LUSC. This study identified rare tumor cell subtypes and mechanisms of tumor progression, underscoring the -Bruker Confidential-potential for AI-driven biomarker discovery and therapeutic strategies in NSCLC and beyond.
 -Bruker Confidential-


 Michael Patrick, Liang Zhang, Patrick Danaher, Shanshan He, Sierra Mckinzie, Haiyan Zhai, Michael Rhodes, Joseph Beechem. Integrating single-cell spatial whole transcriptome and histopathology to uncover drivers of tumor heterogeneity in lung adenocarcinoma and squamous cell carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2091.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae3fe0115c5e85ed3d7113d8af2a6917e00f98fd" target='_blank'>
              Abstract 2091: Integrating single-cell spatial whole transcriptome and histopathology to uncover drivers of tumor heterogeneity in lung adenocarcinoma and squamous cell carcinoma
              </a>
            </td>
          <td>
            Michael Patrick, Liang Zhang, Patrick Danaher, Shanshan He, Sierra Mckinzie, Haiyan Zhai, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Colon adenocarcinoma (CRC) features complex molecular changes and a dynamic remodeling of the tumor microenvironment (TME). Understanding spatial organization of gene expression in tumor and adjacent normal tissues is crucial for insights into tumorigenesis, immune modulation, and therapeutic resistance. Traditional methods like single-cell RNA sequencing (scRNA-seq) provide cellular insights but lack spatial context. Spatial transcriptomics addresses this gap, offering high-resolution, whole transcriptome analysis within tissue architecture. This study applies spatial transcriptomics to CRC, correlating molecular data with histopathological features to explore tumor heterogeneity, immune dynamics, and therapeutic targets. FFPE tissue sections (5 µm) from colon adenocarcinoma and adjacent normal tissues were analyzed using the CosMx® Spatial Molecular Imager (SMI) to create a spatially resolved whole transcriptome, single-cell map. The sections were stained with H&E for histopathological context, and spatial gene expression data was aligned with H&E-stained sections using custom software. Additionally, scRNA-seq was performed on a thicker curl (25 µm) from the same tissue block for comparative analysis. Spatial transcriptomic analysis revealed distinct gene expression patterns between tumor and normal regions, highlighting tumor heterogeneity with subpopulations of cancer cells showing varying differentiation states. Immune cell infiltration varied spatially, indicating diverse immune responses across the tumor. While both platforms identified similar transcriptional profiles, the CosMx Whole Transcriptome (WTX) panel preserved tissue architecture, allowing exploration of gene expression in its native context. CosMx WTX showed a more accurate cell composition ratio than scRNA-seq, which may introduce bias due to tissue dissociation. CosMx WTX also captured rare cell populations, such as tumor-infiltrating lymphocytes, often underrepresented in scRNA-seq due to sample size limitations. The spatial context enabled pathway activity analysis and ligand-receptor interaction mapping, providing deeper insights into tumor biology. This study highlights the power of spatial transcriptomics in colon adenocarcinoma, offering insights into tumor heterogeneity, immune infiltration, and the TME. Integrating spatial gene expression with histopathological features enhances understanding of tumor progression and immune modulation. The comparison with scRNA-seq underscores the unique benefits of spatially resolved profiling, presenting a more complete picture of cellular functions within their native tissue architecture. These findings emphasize the potential of spatial transcriptomics to advance cancer research, guide therapeutic strategies, and reveal novel insights into tumor biology.


 Margaret Hoang, Yi Cui, Shanshen He, Michael Patrick, Megan Vandenberg, Stefan Rogers, Evelyn Metzger, Haiyan Zhai, Martin Shelton, Michael Rhodes, Joseph Beechem. Single-cell spatial transcriptomics in colon adenocarcinoma reveals tumor heterogeneity and immune microenvironmental shifts [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e2b0637dd6be9b8d44c6621977a5ed5666d298" target='_blank'>
              Abstract 2078: Single-cell spatial transcriptomics in colon adenocarcinoma reveals tumor heterogeneity and immune microenvironmental shifts
              </a>
            </td>
          <td>
            Margaret L Hoang, Yi Cui, Shanshen He, Michael Patrick, Megan Vandenberg, Stefan Rogers, Evelyn Metzger, Haiyan Zhai, Martin Shelton, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1326a43bcf280179de51c8de12b664e06eaa82" target='_blank'>
              Single cell resolution of an epigenetic signature of persister tumor cells
              </a>
            </td>
          <td>
            Mihai Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohamed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c897be046eee0151ae3759dbbfd55d0d5c4d5799" target='_blank'>
              TIP-seq: A Single-cell Multiomics Approach for Simultaneous Transcriptome and Intracellular Protein Profiling
              </a>
            </td>
          <td>
            Yingying Xue, Jingyu Qi, Guangting Xie, Mingsheng Wang, Wenhua Xu, Chaoqun Ma, Xueqi Xu, Yuan Yang, Ying He, Yu Guo, Shaoyi Chen, Zhiyin Chen, Mantang Qiu, Changjiang Feng, Jianfeng Liu, Hao Wu, Ya Liu, Tao Xu, Xiaobao Cao
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In the progression from Inflammatory Bowel Disease to associated cancer, the clonal mutational landscape shifts from selection of mutations in inflammatory genes to selection of cancer-driver mutations1–4. How prevalence and expansion of either type of mutated clones could be impacted by the cellular environment in which they arise, and how this affects the neoplastic outcome of colitis is unknown. Here, we combine in vivo lineage tracing, in-silico modelling, mutational profiling and spatial transcriptomics in a mouse model of colitis-associated tumorigenesis to capture clone fates associated with chronic inflammation. We identify epithelial- and immune-enriched neighbourhoods and propose a model in which establishment of a reparative tissue environment facilitates tumours initiation by promoting the selection and expansion of pro-oncogenic clones, reducing the span of inflammation-resistant neighbourhoods containing non-oncogenic clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b08041bfbc2e608a3daf26d2af9f4a68fced7b6" target='_blank'>
              Paired mutation calling and spatial transcriptomics identify cellular neighbourhoods dictating the neoplastic outcome of colitis
              </a>
            </td>
          <td>
            Doug Winton, Elisa B. Moutin, Linus Chang, Giada Giavara, Shenay Mehmed, Mathilde Colombé, Filipe C Lourenço, Maria-Nefeli Skoufou-Papoutsaki, Richard Kemp, Philippe Gascard, T. Tlsty, David Tourigny
          </td>
          <td>2025-05-08</td>
          <td>Research Square</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Breast cancers present intricate microenvironments comprising heterotypic cellular interactions, yet a comprehensive spatial map remained to be established. Here, we employed the DNA nanoball-based genome-wide in situ sequencing (Stereo-seq) to visualize the geospatial architecture of 30 primary breast tumors and metastatic lymph nodes across different molecular subtypes. This unprecedented high-resolution atlas unveils the fine structure of the tumor vasculature, highlighting heterogeneity in phenotype, spatial distribution, and intercellular communication within both endothelial and perivascular cells. In particular, venular smooth muscle cells are identified as the primary source of CCL21/CCL19 within the microenvironment. In collaboration with ACKR1-positive endothelial cells, they create a chemokine-rich venular niche to synergistically promote lymphocyte extravasation into tumors. High venule density predicts increased immune infiltration and improved clinical outcomes. This study provides a detailed spatial landscape of human breast cancer, offering key insights into the venular regulation of tumor immune infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3853b930721fb6df8f03ce40a70213e79309b5e" target='_blank'>
              Spatially resolved atlas of breast cancer uncovers intercellular machinery of venular niche governing lymphocyte extravasation
              </a>
            </td>
          <td>
            Xin Wang, Zhanyu Wang, Qijun Liao, Pei Yuan, Junpu Mei, Yin Zhang, Chao Wu, Xiyu Kang, Sufei Zheng, Chen-En Yang, Jiaxiang Liu, Qingyao Shang, Jiangtao Li, Bingning Wang, Liangyu Li, Hui Liu, Weining Hu, Zhensheng Dong, Jie Zhao, Linying Wang, Tao Liu, Yusheng Den, Chengrui Wang, Lijuan Han, Qianjun Chen, Huanming Yang, Xun Xu, Jie He, Zhen Yue, Nan Sun, Xiaodong Fang, Jianming Ying
          </td>
          <td>2025-04-09</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>17</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) often exhibits poor response rates to immune checkpoint inhibitor (ICI) therapies, largely owing to the intricate composition and spatial organization of immune cells within the tumor-immune microenvironment (TIME). The diversity of immune cell populations, their spatial relationships, and dynamic interactions significantly influence the immunosuppressive nature of the TIME, thereby limiting the efficacy of immunotherapy. To address these challenges and enhance the therapeutic potential of ICIs in HNSCC, a comprehensive analysis of the TIME is essential. Spatial transcriptomics (ST), a cutting-edge technology, enables high-resolution mapping of gene expression within the spatial context of the tumor, providing critical insights into the functional roles and interactions of immune cells in the TIME. This review highlights the importance of ST in uncovering the complexities of the TIME in HNSCC and proposes strategies for leveraging these insights to develop more effective immunotherapeutic approaches. By integrating spatial and molecular information, this review aims to pave the way for personalized and precision-based treatments in HNSCC, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef3b95effc04f0811082363795fb3dc9666b0f8" target='_blank'>
              Mapping immunotherapy potential: spatial transcriptomics in the unraveling of tumor-immune microenvironments in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Seo-Won Choi, Jeong Heon Kim, Jisu Hong, Minsu Kwon
          </td>
          <td>2025-04-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 The tumor microenvironment (TME) is a highly dynamic and complex ecosystem, where the diversity of cell types, their functional states, and spatial interactions determine responses to immunotherapy. Despite its critical role, the precise mechanisms through which these factors influence clinical outcomes remain poorly understood. In this study, we leveraged single-cell spatial transcriptomics to systematically dissect the TME and identify determinants of immunotherapeutic response in vulvar high-grade squamous intraepithelial lesions (vHSIL).
 We analyzed 20 treatment-naïve vHSIL lesions, stratified based on clinical response to tumor-specific peptide vaccination into: complete responders (CR, 100% lesion clearance, n=6), partial responders (PR, >50% lesion clearance, n=7), and non-responders (NR, <50% lesion clearance, n=7). Using a 1000-gene panel, we mapped over 274, 000 single cells in situ, revealing 18 cell clusters that were further delineated into 99 distinct non-epithelial cell states. Validation against public single-cell transcriptomic datasets confirmed the broader relevance of these findings across tumor types.
 Our findings demonstrated profound heterogeneity within the TME across response groups. Notably, CR lesions exhibited a significantly higher ratio of immune-supportive to immune-suppressive cells compared to PR and NR lesions—a pattern mirrored in other solid tumors following neoadjuvant checkpoint blockade therapy (anti-PD1, anti-PDL1, anti-CTLA4). Key immune populations enriched in CR lesions included CD4+CD161+ effector T cells and chemotactic CD4+ and CD8+ T cells, implicating these populations as critical drivers of favorable therapeutic outcomes. Conversely, PR lesions were characterized by increased proportions of T helper 2 cells and CCL18-expressing macrophages, which are associated with the recruitment of type 2 T cells and regulatory T cells. NR lesions displayed preferential infiltration with immunosuppressive fibroblasts. Distinct spatial immune ecosystems further defined response groups: type 1 effector cells dominated interactions in CR lesions, type 2 cells were prominently interacting in PR lesions, while NR lesions lacked organized immune cell interactions.
 Thus, these findings emphasize the dual importance of cellular composition and spatial organization in steering response to immunotherapy. By elucidating the mechanisms underlying clinical responses to immunotherapeutic vaccination, this study provides actionable insights to advance precision immunotherapy.


 Ziena Abdulrahman, Roderick C. Slieker, Daniel McGuire, Marij J.P. Welters, Mariette I.E. van Poelgeest, Sjoerd H. van der Burg. Single-cell spatial transcriptomics unravels cell states and ecosystems associated with clinical response to immunotherapy [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 154
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd78a6499d1079983203268939cd1b90e4adb3f" target='_blank'>
              Abstract 154: Single-cell spatial transcriptomics unravels cell states and ecosystems associated with clinical response to immunotherapy
              </a>
            </td>
          <td>
            Z. Abdulrahman, Roderick C. Slieker, Daniel McGuire, M. J. Welters, M. V. van Poelgeest, S. H. van der Burg
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="
 Heterogeneity in colorectal cancer (CRC) manifests as inter- and intra-patient diversity in tumor and immune cell populations as well as varied patterns of tumor interaction with surrounding stroma. Spatial transcriptomics (ST) captures gene expression within the native spatial context of tissues, offering a high-resolution approach to dissect cancer molecular landscapes. Prior ST studies in CRC have been constrained by small sample sizes and tissue selection biases, reducing clinical relevance. Here we performed ST (10x Genomics Visium Spatial Gene Expression) on 40 retrospectively-collected CRC samples with available clinicopathological data (stages I-IV: 13, 8, 14, and 5 samples, respectively) and long-term clinical outcomes to define the molecular topography and spatial heterogeneity of tumors with their stromal context. Using ST data, we identified recurring spot-level (55-micron resolution) tumor and stromal sub-populations across the cohort, designating them as T1 (marker genes EPCAM, LGALS4, and FXYD3) and T2 (marker genes KRT8, KRT18, and KRT19) for tumor subtypes, and F1 (marker genes SPARC and LUM) and F2 (marker genes COL3A1 and PTGDS) for stromal subtypes. Comprehensive analysis of tumor/stromal pairs—including copy number variations (CNVs), cell type compositions, pathway activations, metabolic profiles, spatial distributions, and histological characteristics—revealed pathways enriched in T2 compared to T1, including epithelial-mesenchymal transition (EMT), angiogenesis, hypoxia response, TGF-beta signaling, and metabolic pathways supporting the energy and biosynthetic demands of aggressive tumor cells. We also identified recurrent spatially co-localized tumor and stromal sub-clonal pairs—T1&F2 and T2&F1—that consistently preserved their spatial architecture across samples. Notably, the prevalence of T1&F2 was significantly correlated with disease-specific survival (DSS) (log-rank test, p=0.05), highlighting the prognostic importance of tumor-microenvironment interactions. Using a transformer-based image feature encoder, we demonstrated that these clusters are morphologically distinguishable in hematoxylin and eosin (H&E)-stained images co-registered with the ST data, achieving an AUC (the Area Under the ROC Curve) of 0.84 for distinguishing T1 from T2 and an AUC of 0.75 for distinguishing F1 from F2. These clusters were also detected in H&E images from external datasets, including The Cancer Genome Atlas (TCGA, N=232) and an independent validation set (N=107). Deep learning models trained on TCGA data further revealed a trend where higher T2 prevalence or lower T1 prevalence correlated with shorter overall survival. This ST study establishes novel molecular subtypes and their spatial co-occurrence in CRC, linking spatial behaviors to clinically prognostic and H&E-based morphological characteristics.


 Jie Zhou, Todd B. Sheridan, Sergii Domanskyi, Stephanie L. Cowles, Sunghee Park, Ilham Putra, Olga Anczukow, Jeffrey H. Chuang, Jill C. Rubinstein. Unveiling the spatial landscape of tumor and stroma heterogeneity in colorectal cancer with spatial transcriptomics and deep learning [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 756.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/623dee1630864493156f7c52b0b73a1aacf3c1e1" target='_blank'>
              Abstract 756: Unveiling the spatial landscape of tumor and stroma heterogeneity in colorectal cancer with spatial transcriptomics and deep learning
              </a>
            </td>
          <td>
            Jie Zhou, Todd B Sheridan, Sergii Domanskyi, Stephanie L. Cowles, SungHee Park, Ilham Putra, Olga Anczuków, Jeffrey H. Chuang, Jill C. Rubinstein
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 e14574


 Background:
 Understanding cellular interactions within the tumor microenvironment (TME) is essential for elucidating disease progression and advancing immunotherapy. The TME is a complex ecosystem composed of cells with dysregulated metabolism immune response and signaling pathways, which in turn influence tumor development and treatment response. Multiplexed assessment serves as an important tool for clinical oncology. The simultaneous readout of multiple processes can provide biological insights and elucidate disease mechanism. Imaging Mass Cytometry (IMC) is a spatial biology imaging technique that utilizes CyTOF technology and enables deep characterization of the diversity and complexity of the TME. IMC technology offers scalable, high-throughput acquisition while generating high-quality data with true dynamic range of signal without amplification or fluorescence-based limitations such as spectral overlap and autofluorescence.
 Methods:
 To study cellular processes and their roles in tumor progression, we utilized the Human Cell Metabolism and Human Cell Signaling Panels to investigate energy production, cellular homeostasis and mitogenic signaling pathways. We then mapped these processes to the types of cells in the TME by using the Human Immuno-Oncology IMC Panel and the Human T Cell Exhaustion IMC Panel or the Maxpar Neuro Phenotyping IMC Panel Kit to characterize immune cell and neurological phenotypes in detail. We first acquired data using Preview Mode to assess the whole tissue, followed by higher-resolution imaging of selected regions of interest using Cell Mode or of the whole tissue section using Tissue Mode.
 Results:
 Our data analysis revealed significant insights into the spatial organization and metabolic profile of cells across cancer tissues. Elevated glycolysis and mTOR pathway activation suggested adaptations to hypoxia and anabolic growth in tumor areas, while interactions between fibroblasts and immune cells highlighted crosstalk within the TME. Our Neuro Phenotyping Panel was used to reveal potential for immune response in glioblastoma. Unsupervised pixel clustering and hierarchical clustering using MCD SmartViewer highlighted metabolic activity and activation of signaling pathways within tumor regions.
 Conclusions:
 Comprehensive spatial biology profiling using the IMC approach highlights the interconnected roles these pathways play in promoting tumor survival and resistance to therapies. These findings, which illuminate the metabolic and signaling heterogeneity of the TME, are crucial for developing future prognostic assessments and have the potential to guide more effective, personalized cancer therapies. For Research Use Only. Not for use in diagnostic procedures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b16079bb313271e366d31125c0f7c2617d3a11a" target='_blank'>
              Spatial profiling of solid tumor microenvironment using imaging mass cytometry with high-plex panels.
              </a>
            </td>
          <td>
            T. Pfister, J. Y. Chwee, Nick Zabinyakov, Qanber Raza, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39cd802b620e803fe61dfbfe7d21f2c3b02ece22" target='_blank'>
              Spatial mapping of transcriptomic plasticity in metastatic pancreatic cancer.
              </a>
            </td>
          <td>
            Guangsheng Pei, Jimin Min, K. Rajapakshe, Vittorio Branchi, Yunhe Liu, B. Selvanesan, F. Thege, Dorsay Sadeghian, Daiwei Zhang, K. S. Cho, Yanshuo Chu, Enyu Dai, Guangchun Han, Mingyao Li, Cassian Yee, Kazuki Takahashi, Bharti Garg, Hervé Tiriac, Vincent Bernard, A. Semaan, Jean L. Grem, T. Caffrey, Jared K. Burks, A. Lowy, Andrew J. Aguirre, P. Grandgenett, Michael A. Hollingsworth, Paola A. Guerrero, Linghua Wang, Anirban Maitra
          </td>
          <td>2025-04-23</td>
          <td>Nature</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="Solid cancers are complex 'ecosystems' comprised of diverse cell types and extracellular molecules, where heterotypic interactions significantly influence disease etiology and therapeutic response. However, our current understanding of tumor microenvironments (TMEs) remains incomplete, hindering the development and implementation of novel TME-targeted drugs. To maximize cancer therapeutic development we require a systems-level understanding of the malignant, stromal, and immune states that define the tumor and determine treatment response. In their recent study, Liu and colleagues took a new approach to resolving the complexity of stromal heterogeneity. They leveraged extensive single-cell spatial multi-omics datasets across various cancer types and platforms to identify four conserved spatial Cancer-Associated Fibroblast (CAF) subtypes, classified by their spatial organization and cellular neighborhoods. Their work expands upon prior efforts to develop a CAF taxonomy, which primarily relied on single-cell RNA-Sequencing (scRNA-Seq) and yielded a multitude of classification systems. This study advances our understanding of CAF biology by establishing a link between spatial context and CAF identity across diverse tumor types. Departing from recent single-cell transcriptomics studies that employed a marker-based approach for sub-state identification, Liu and colleagues conducted de novo discovery of CAF subtypes using spatial neighborhood information alone. By positioning spatial organization as the defining axis of CAF heterogeneity, this research redefines CAF classification and provides a new framework for exploring the rules governing tumor ecosystems and developing novel ecosystem-based therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940389ee957141f2833e0d9de53711c0ead3a30c" target='_blank'>
              Pan-Cancer Spatial Profiling Reveals Conserved Subtypes and Niches of Cancer-Associated Fibroblasts.
              </a>
            </td>
          <td>
            Hani Jieun Kim, Travis Ruan, Alex Swarbrick
          </td>
          <td>2025-05-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The cellular heterogeneity of prostate cancer cells has been well-established, yet the spatial diversity of stromal cells in the tumor microenvironment remains largely unexplored. In this study, we utilized cyclic immunofluorescence imaging of whole-tissue sections to map spatial relationships between cancer cells and various immune and non-immune stromal populations in primary Gleason pattern 3 and 4 tumors. Our analysis identified over 20 androgen receptor (AR)-positive stromal cell types, many of which have minimal data on their response to AR inhibitors. Notably, we identified distinct AR+ mast cell populations exhibiting unique spatial associations with M2 macrophages and regulatory T cells. Using spatial statistical tools and recurrent cellular neighborhood analysis, we identified disease-specific microenvironments predominantly driven by AR+ stromal cells and uncovered inflammatory gene networks active within AR+ stromal regions. Future work will expand this analysis to spatiotemporal profiling of intermediate-risk patients, aiming to improve tumor stratification for more precise clinical decision-making.


 Çiğdem Ak, Zeynep Sayar, Guillaume Thibault, M.J. Kuykendall, George Thomas, Vasilis Stavrinides, Young Hwan Chang, Ece S. Eksi. The spatial cellular landscape of Gleason pattern 3 and 4 prostate tumors [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6461.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9981c24a8ff91461ed8c8211a765a8781b3dfd60" target='_blank'>
              Abstract 6461: The spatial cellular landscape of Gleason pattern 3 and 4 prostate tumors
              </a>
            </td>
          <td>
            Çiğdem Ak, Z. Sayar, Guillaume Thibault, M. J. Kuykendall, George V. Thomas, V. Stavrinides, Young Hwan Chang, E. S. Eksi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Tumors are dynamic ecosystems where interactions between cancer cells and the tumor microenvironment (TME) shape malignancy. Using spatial transcriptomics, we analyzed 63 untreated tumors from 10 cancer types to investigate the spatial distribution of 13 cancer hallmarks. Our findings reveal that hallmark activity is spatially organized, with distinct compartmentalization of hallmark phenotypes across neoplastic and TME regions. Hallmarks such as "Resisting Cell Death" and "Enabling Replicative Immortality" were enriched in cancer-dominant areas, while "Inducing Angiogenesis" and "Activating Invasion and Metastasis" were primarily localized to the TME. Genomic analysis showed that subclonal diversity influences hallmark specialization, with genetically distinct clones often occupying specific ecological niches. These insights suggest that hallmark activities are not uniformly distributed but instead reflect the interplay between genetic heterogeneity and tumor spatial organization.
 We further modeled the spatial interdependence of hallmarks between the neoplastic and TME compartments, demonstrating that hallmark activities are shaped by ecological dynamics across the tumor. For example, hallmarks such as "Sustaining Proliferative Signaling" in the TME interface strongly influenced cancer cell hallmark expression. Application of this framework to 33 bladder cancer patients from the DUTRENEO trial revealed correlations between hallmark spatial patterns and treatment response. Notably, differences in the spatial localization of "Inducing Angiogenesis" and "Resisting Cell Death" were linked to neoadjuvant chemotherapy and immunotherapy sensitivity. These findings underscore the importance of spatial hallmark distribution in determining drug efficacy and suggest that spatial hallmark signatures could serve as predictive biomarkers for personalized cancer therapy.


 Eduard Porta-Pardo, Mustafa Sibai, Sergi Cervilla, Daniela Grases, Manel Esteller. The spatial landscape of cancer hallmarks reveals patterns of tumor ecological dynamics and drug sensitivity [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB265.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89052d3ed2d82ce2288e348ab845b3ae781f2935" target='_blank'>
              Abstract LB265: The spatial landscape of cancer hallmarks reveals patterns of tumor ecological dynamics and drug sensitivity
              </a>
            </td>
          <td>
            E. Porta-Pardo, Mustafa Sibai, Sergi Cervilla, D. Grases, M. Esteller
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity. GBM tumors contain neoplastic cells within a spectrum of different transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we developed a novel framework, ISOSCELES (Inferred cell Sensitivity Operating on the integration of Single-Cell Expression and L1000 Expression Signatures), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identified compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states. We validated the significance of these findings in vitro, ex vivo, and in vivo, and identified a novel combination of an OLIG2 inhibitor and Depatux-M for GBM. Our studies suggest that ISOSCELES identifies cell states sensitive and resistant to targeted therapies in GBM and that it can be applied to identify new synergistic combinations. Highlights Integration of GBM single-cell RNA sequencing data with L1000-derived drug response signatures facilitates clustering of tumor cells and small molecules on cell-drug connectivity. Cell-drug connectivity predicts the identities of drug-sensitive and resistant cell states. In silico perturbation analysis using cell-drug connectivity predicts drug-induced changes in the cell-drug connectivity landscape in vivo. In silico perturbation analysis to predict drug-induced changes in the tumor cell-drug connectivity landscape predicts drug combinations that synergize in vivo to extend survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25613f3a85e6691e66bbf8705e07ffcebf8c6cbf" target='_blank'>
              DRUG AND SINGLE-CELL GENE EXPRESSION INTEGRATION IDENTIFIES SENSITIVE AND RESISTANT GLIOBLASTOMA CELL POPULATIONS
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, Ricardo J Komotar, M. I. De La Fuente, Gregory Stein, Alexander Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, N. Ayad
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c45c976013cae1b807eb9d7fbbf98792d114f7a" target='_blank'>
              Cell-type-specific patterns and consequences of somatic mutation in development and aging brain
              </a>
            </td>
          <td>
            Andrea J. Kriz, Shulin Mao, Diane D. Shao, Daniel A. Snellings, Rebecca E. Andersen, Guanlan Dong, Chanthia C. Ma, Hayley E. Cline, A. Huang, Eunjung Lee, Christopher A. Walsh
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lymphomas are highly heterogeneous malignancies characterized by diverse genetic, epigenetic, and phenotypic profiles, which complicate diagnosis, prognosis, and treatment. Traditional bulk sequencing methods mask the complexity of intratumoral variation, often overlooking rare subclones that may drive disease progression and therapeutic resistance. Single-cell genomics has emerged as a transformative approach to decipher tumor heterogeneity at unprecedented resolution. This technology enables the dissection of individual cellular populations within lymphomas, offering insights into clonal evolution, transcriptional diversity, and microenvironmental interactions. By applying single-cell RNA sequencing (scRNA-seq), chromatin accessibility assays (scATAC-seq), and single-cell DNA sequencing (scDNA-seq), researchers can unravel lineage relationships, identify resistant subpopulations, and track dynamic changes in response to therapy. In lymphomas such as diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL), and mantle cell lymphoma (MCL), single-cell approaches have revealed distinct malignant and non-malignant cell states that correlate with treatment outcomes. Moreover, integrating single-cell data with spatial transcriptomics and immune profiling enhances the understanding of the tumor microenvironment, including immune evasion mechanisms. These insights can inform personalized treatment strategies, identify novel therapeutic targets, and enable early detection of relapse. Despite technical challenges such as data complexity, sample viability, and cost, the application of single-cell genomics in lymphoma research is rapidly advancing. Future directions include multi-omics integration, real-time patient monitoring, and clinical translation of predictive biomarkers. This review underscores the pivotal role of single-cell genomics in resolving tumor heterogeneity and predicting therapeutic resistance, positioning it as a cornerstone for next-generation precision oncology in lymphomas,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/438856abbab164e715372bfe3f2a6c07df9594ec" target='_blank'>
              Role of mutational signatures and clonality assessments in tailoring targeted therapies for lymphoma
              </a>
            </td>
          <td>
            Andoh Sheikh Atta-ullah, Uche Philip, Sampson Janice Candy
          </td>
          <td>2025-05-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is a highly lethal malignant intracranial tumor, distinguished from low-grade glioma by histopathological hallmarks such as pseudopalisading cells around necrosis (PAN) and microvascular proliferation (MVP). To date the spatial organization of the molecular and cellular components of these specific histopathological features has not been fully elucidated. Methods Here, using bulk RNA sequencing, spatial transcriptomic and single cell RNA sequencing (scRNA-seq) data of GBM patients, we identified niche-specific transcriptional programs and characterized the differences in molecular expression and cellular organization between PAN and MVP. Results Notably, we discovered spatially distinct domains within the tumor core and identified niche-specific signatures: NDRG1 and EPAS1, specifically expressed in the PAN and MVP regions. The clustering results showed two distinct phenotypes of endothelial cells (ECs) were enriched in the MVP and PAN regions, respectively. PAN-associated endothelial cells exhibit copy number variations similar to those in GBM cells. Single cell trajectory analysis reveals a pseudotime trajectory, indicating the differentiation of glioblastoma stem cells (GSCs) toward ECs. Conclusions Necrosis cores which are surrounded by hypoxic and perivascular niches and microvascular proliferation area within the glioblastoma tumor microenvironment, have been considered as standardized morphological indicators of aggressive GBM. Our findings provide a cellular and molecular insights into GBM progression. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06185-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75320d960d6bc41d40bd96ad524b1fc28b650a0b" target='_blank'>
              Multi-transcriptomics reveals niche-specific expression programs and endothelial cells in glioblastoma
              </a>
            </td>
          <td>
            Jiukun Hu, Xiaohan Sa, Yue Yang, Yuwen Han, Jie Wu, Minxuan Sun, S. Shafi, Nafees Ahmad, Sami Siraj, Jiao Yang, Yuanshuai Zhou
          </td>
          <td>2025-04-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bca1af35819fcfb6f9d7674ff4443096f1ec742" target='_blank'>
              Single-Cell Pharmacogenomic Landscapes of Epigenetic Therapy in Gastric Cancer
              </a>
            </td>
          <td>
            Chang Xu, Haoran Ma, T. Sheng, M. Lee, S. A. A. Abdul Ghani, Laura Perlaza-Jiménez, K. Huang, Shen Kiat Lim, Supriya Srivastava, Xuewen Ong, S. Tay, S. Ho, Angie Tan, Feng Zhu, Vincenzo Nasca, G. Randon, Hassan Ashktorab, A. S. Cheng, A. Jeyasekharan, Shang Li, Ming Teh, Raghav Sundar, David R. Powell, Joseph Rosenbluh, F. Pietrantonio, W. Yong, J. So, Patrick Tan
          </td>
          <td>2025-04-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) represents the most common pathological type of lung cancer, and the combination of neoadjuvant immunotherapy with chemotherapy has emerged as the first-line treatment for NSCLC. Nevertheless, the efficacy of this therapeutic approach remains variable. The present study aims to examine the impact of chemoimmunotherapy in NSCLC patients, with a view to identifying key molecules, critical cell subpopulations, communication patterns and spatial distributions that potentially correlate with therapeutic sensitivity. A total of 16 lung cancer tissue samples were collected from a cohort of 12 NSCLC patients and subjected to single-cell RNA and spatial transcriptome sequencing. Our data demonstrated that the distribution of CD4 + Treg T cells and mCAFs indicated an immunosuppressive tumor microenvironment, while the accumulation of CD4 + Th17 T cells and iCAFs could act as a positive marker for the sensitivity to chemoimmunotherapy. Furthermore, a significant high level of SELENOP-macrophages was observed in tissues from positive responders, and a strong co-localization between SELENOP-macrophages and antigen-presenting cancer associated fibroblasts (CAFs) in the tumor boundaries was identified, indicating the cooperative roles of these two cell types in response to combined therapy. Moreover, SELENOP-macrophages were observed to be accumulated in tertiary lymphoid structures, which further suggested its critical role in recruiting lymphocytes. Furthermore, analysis of cell–cell communication, based on spatial transcriptomics, suggests that the interactions between SELENOP-macrophages, apCAFs, CD4 + and CD8 + T cells were significantly enhanced in responders. In addition, SELENOP-macrophages recruited CD4 + Naïve, Helper and CD8 + Naïve T cells through pathways such as the cholesterol, interleukin, chemokine and HLA when responding to combined therapy. The present study further unveils the dynamic spatial and transcriptional changes in the tumor microenvironment of non-small cell lung cancer in response to combination therapy. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02287-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86da4dd75d392abc980cae7b3d81379fef557c28" target='_blank'>
              Single-cell and spatial transcriptomic analyses revealing tumor microenvironment remodeling after neoadjuvant chemoimmunotherapy in non-small cell lung cancer
              </a>
            </td>
          <td>
            Xiaolu Cui, Siyuan Liu, He Song, Jingjing Xu, Yanbin Sun
          </td>
          <td>2025-04-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Nasopharyngeal carcinoma (NPC) is a common malignancy with a complex pathogenesis and diverse cellular composition. This study aimed to systematically analyze the transcriptional characteristics of different cell types in NPC and their roles in tumor development using single-cell transcriptomics. Methods The research team collected NPC and normal tissue samples, and performed in-depth single-cell sequencing analysis of the transcriptomes. Single-cell RNA sequencing was performed on the collected samples to obtain high-resolution transcriptional profiles of individual cells. This allowed the researchers to identify and characterize the diverse cell populations present within the NPC tumor microenvironment. Results The results showed that NPC samples contained multiple distinct cell subpopulations, including epithelial cells, immune cells (such as macrophages and T cells), endothelial cells, and stromal cells. These cell types exhibited marked differences in spatial distribution and transcriptional profiles, reflecting the high degree of heterogeneity within the tumor microenvironment. Further functional analysis revealed significant dysregulation of mitochondrial-related pathways, extracellular matrix-receptor interactions, as well as Wnt, Notch, and other signaling cascades in NPC. Conclusion This study employed single-cell transcriptomics to comprehensively elucidate the complexity of the NPC tumor microenvironment, providing new insights into the underlying mechanisms of disease pathogenesis and suggesting potential therapeutic targets. These findings lay the groundwork for the development of precision medicine approaches for NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9793611f5914170a344f624509f87b1fa27aa37" target='_blank'>
              Single-cell transcriptomic landscape reveals complex cellular heterogeneity and dysregulated signaling pathways in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Xiaoyan Zhang, Zuojia Guan, Zhan Xu
          </td>
          <td>2025-05-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Renal cell carcinoma (RCC) is characterized by a high degree of genomic but also functional intratumoral heterogeneity (ITH). Mutations in VHL, chromatin remodeling genes such as SETD2 and genes that regulate the PI3K/AKT/mTOR pathway have been identified as recurrent drivers despite genomic ITH. Whether and to what extent these mutations shape functional ITH including the formation of spatial niches is incompletely understood. Herein, we analyze the correlation between mutational drivers and their functional proxies in a spatially defined manner. Methods A total of 23 RCCs were analyzed by panel next-generation sequencing followed by immunohistochemistry for five functional proxies for key genetic alterations including the expression of CD31, GLUT1, phospho-mTOR S2448, H3K36me3 and Ki-67. Antibody stainings were scored semiquantitatively in the tumor periphery and the tumor center. Results Unexpectedly, the presence of a VHL mutation was not found to correlate with its functional proxies including the expression of CD31/microvessel density or the expression of the glucose transporter GLUT1. Likewise, there was no correlation between the presence of activating mutations in genes of the PI3K/AKT/mTOR pathway and the expression of activated phospho-mTOR S2448. Furthermore, mutations in the methyltransferase gene SETD2 were not found to correlate with the expression level of its downstream target H3K36me3. Lastly, there was no correlation between the expression of the proliferation marker Ki-67 and the number of driver mutations. Conclusion This proof-of-concept study adds genotype-phenotype heterogeneity as additional layer of complexity to the known genomic and functional ITH in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/455b536369691fca3d4bb3ee3d0cab12ab897412" target='_blank'>
              Extensive genotype-phenotype heterogeneity in renal cell carcinoma - a proof-of-concept study
              </a>
            </td>
          <td>
            Jakob Wieke, Christina Jurcic, A. Kaczorowski, Sarah Böning, Martina Kirchner, C. Schwab, V. Schütz, Markus Hohenfellner, A. Duensing, A. Stenzinger, Stefan Duensing
          </td>
          <td>2025-04-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5025875828f543583bc1d12169d95499fa95592" target='_blank'>
              Robust statistical assessment of Oncogenotype to Organotropism translation in xenografted zebrafish
              </a>
            </td>
          <td>
            D. Saucier, Xuexia Jiang, Divya Rajendran, Roshan Ravishankar, Erin Butler, A. Marchetto, R. Kurmasheva, T. G. Grünewald, James F. Amatruda, G. Danuser
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy characterized by early liver metastasis and high mortality. The tumor microenvironment plays a pivotal role in tumor progression; however, the immune microenvironment's involvement in PDAC liver metastasis remains poorly understood. Methods: This study investigates cellular heterogeneity in primary tumor (PT) and liver metastasis (LM) tissues of PDAC using single-nucleus RNA sequencing and spatial transcriptomics. Intra-tumor heterogeneity and cell interactions were elucidated through deconvolution, intercellular signalling, pseudotime analysis, and immune infiltration profiling. The spatial distribution of immune cells was assessed by multiplexed immunofluorescence staining, and prognostic models were developed and validated through immunohistochemistry (IHC). Analyzing the regulatory role of CITED4 in the invasion and metastasis of pancreatic cancer cells through transwell assay and scratch wound healing assay. Results: A total of 62,326 cells were sequenced, with metastatic dissemination cells showing significant upregulation of epithelial-mesenchymal transition (EMT)-related genes during liver metastasis. Spatial transcriptomics revealed the enrichment of metastatic dissemination cells and FOXP3-related Treg cells at the tumor front in PT tissues. In comparison to LM tissues, the tumor front in PT tissues fosters an immunosuppressive microenvironment through the accumulation of Treg cells. Interaction analysis identified the SPP1 pathway as a key promoter of this immunosuppressive environment. Furthermore, prognostic models highlighted CITED4 as critical biomarkers in PDAC. Elevated CITED4 expression is correlated with liver metastasis and poor prognosis in patients with PDAC. siRNA-mediated knockdown of CITED4 suppresses the invasion and metastasis of pancreatic cancer cells. Conclusions: In summary, this study revealed that Treg cell alterations, mediated by metastatic dissemination cells within the immune microenvironment, significantly contribute to PDAC liver metastasis, and that CITED4 enhances the metastatic potential of metastatic dissemination cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7cfcf741ce365a1a3bfe1ca1a5f5a71b8d0a0d3" target='_blank'>
              Single-nucleus RNA sequencing and spatial transcriptomics reveal an immunosuppressive tumor microenvironment related to metastatic dissemination during pancreatic cancer liver metastasis
              </a>
            </td>
          <td>
            Hongsen Liu, Mengting Chen, Bo Hong, Yufei Xiao, Qianming Chen, Yun Qian
          </td>
          <td>2025-04-13</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Despite advances in single-cell technologies, integrating multimodal data to capture both the genetic and phenotypic dynamics of tumor cells in their microenvironment remains a challenge.



 We leverage variations in cell cycle state as a natural barcode to link data across different modalities. We have demonstrated the feasibility of this method in a gastric cancer cell line (NCI-N87) using two widely different technologies: single-cell RNA sequencing and live-cell imaging. Our approach begins with the transfection ofFUCCI probes into cells, followed by acquisition of 3D images using a Leica microscope. These images are then utilized to train a label-freeU-Net-based prediction model (FNet) to infer subcellular organization, specifically for the nucleus, mitochondria, and cytoplasm. FNet achieved correlation coefficients of 0.76, 0.70, and 0.78 for these compartments, respectively. Segmentation masks generated by CellPose enabled cell statistics calculations. We further applied deep learning methods, including a customized convolutional neural network (CNN)and a truncated ResNet50, to distinguish between cells in G1, S and G2M phases of the cell cycle. The CNN achieved an accuracy of 90.67% when using the nucleus alone and 91.63% when using all three compartments (nucleus, mitochondria, and cytoplasm).
 Deep features were extracted by removing the classification layer and flattening the convolutional layer feature maps. These features were then employed for cell cycle pseudotime inference. When projecting the features onto pseudotime, we observed a strong correlation between the inferred pseudotime and FUCCI-derived cell cycle states (Pearson r =-0.899, p-value < 1E-6). This finding demonstrates the ability to assign continuous cell cycle states directly from imaging data. Comparing imaging- and sequencing-derived cell cycle pseudotime obtained for the same cell line, we assigned 449 imaged cells to their corresponding 693 sequenced cells, achieving an average resolution of 3.4 cells per sequencing profile and 2.4 cells per imaging profile.



 This integration has opened new possibilities for linking genotype (from sequencing) and phenotype (from imaging), providing a forward-looking approach to genotype-phenotype mapping at single-cell resolution. Assigning the transcriptome of a sequenced cell to its closest living relative (still actively growing and expanding), will enable tracing the cellular trajectory in real-time, enabling a deeper understanding of how tumor cells adapt to their microenvironment and evolve over time.



 Saeed S. Alahmari, Andrew Schultz, Jordan Albrecht, Vural Tagal, Zaid Siddiqui, Sandhya Prabhakaran, Issam El Naqa, Alexander Anderson, Laura Heiser, Noemi Andor. Cell identity revealed by precise cell cycle state mapping links data modalities [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3703.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4356e915cd9638ea3a28d0766da31c4fd8473bcf" target='_blank'>
              Abstract 3703: Cell identity revealed by precise cell cycle state mapping links data modalities
              </a>
            </td>
          <td>
            Saeed S. Alahmari, Andrew Schultz, Jordan Albrecht, Vural Tagal, Zaid Siddiqui, Sandhya Prabhakaran, I. El Naqa, Alexander Anderson, Laura Heiser, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Small Cell Lung Cancer (SCLC) is a highly aggressive and deadly neuroendocrine lung cancer with no major effective treatments. SCLCs are usually detected after wide metastasis, which leads to extremely low five-year survival rate (around 7%). To address this challenge, we leveraged network-based computational approach to first dissect the heterogeneity of SCLC tumors and to discover Master Regulator (MR) proteins representing pharmacologically accessible, mechanistic determinants of molecularly distinct SCLC cell populations. Specifically, we first profiled transcriptomes of 182, 189 cells from 41 fresh SCLC human tissue samples, acquired from 14 patients. Samples are collected from both primary site (lung) and metastases (Mediastinum, liver and adrenal gland). Built upon this dataset, we reverse-engineered single-cell based SCLC-specific regulatory networks through ARACNe algorithm. Using these networks, we then measured protein activity of over 6, 000 regulatory and signaling proteins for all the cells via VIPER. To dissect intra-tumor heterogeneity of SCLC, protein activity-based analysis of tumor cells identified cell subpopulations representing distinct differentiation and proliferation stages. Our analysis helped identify cell populations presenting highly aberrant activity of proliferative MRs, which are termed as “Tumor Checkpoint” regulatory module. We then identified SCLC cell lines showing high protein activity similarity with the target tumor cell populations using rank-based statistics. Following perturbation of the selected cancer cell lines with hundreds of oncology drugs, we identified potent drugs that could invert the activities of the MRs of target tumor subpopulations using OncoTreat algorithm. The identified drugs are subsequently validated using in vivo mouse model. The outcome of this project is four-fold: First, the patient derived SCLC single-cell RNA-Seq dataset generated in this project represents the largest cohort of this kind up to date; Second, our analysis helps study SCLC intra-tumor heterogeneity from the protein activity angle, which is complementary to gene expression measurement alone; Third, our drug perturbation dataset is the first whole-genome based drug perturbation gene expression profile in SCLC cell lines. Fourth, our comprehensive analysis identifies drugs effectively induce tumor population demise in SCLC. Currently, we are working on the drug perturbation experiment on SCLC cell lines to finalize potent drug predictions.


 Lucas ZhongMing Hu, Anish Thomas, Andrea Califano. Single cell-based elucidation and pharmacologic targeting of master regulator dependencies in small cell lung cancer (SCLC) subpopulations [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5088.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81fc3716db0c11759429e639a63e62e0f3409b05" target='_blank'>
              Abstract 5088: Single cell-based elucidation and pharmacologic targeting of master regulator dependencies in small cell lung cancer (SCLC) subpopulations
              </a>
            </td>
          <td>
            Lucas ZhongMing Hu, Anish Thomas, Andrea Califano
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c327cba87285dc890d1f7b106ff92f090a08a250" target='_blank'>
              Single-Cell Analysis of Meningiomas Reveals Mutation-Associated Tumor and Immune Cell Gene Expression Programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus I Anzaldua-Campos, Gregory J Zipfel, Albert H. Kim, J. Dowling, Eric C. Leuthardt, Joshua W. Osbun, A. Vellimana, Michael R. Chicoine, Gavin P. Dunn, Allegra A. Petti
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Colorectal cancer (CRC) metastasis is highly aggressive and associated with poor outcomes. Recent studies have highlighted greater heterogeneity in gene expression within metastatic tumor cells compared to primary tumor cells. However, the underlying factors driving these differences remain poorly understood. We hypothesize that both genetic alterations and the tumor microenvironment play key roles in shaping this heterogeneity. To investigate these aspects, we established a cohort through the Human Tumor Atlas Network composed of 26 primary CRC samples and 41 metastatic CRC samples from multiple organs. We conducted whole exome sequencing (WES), single-nucleus ATAC/RNA-seq (snATAC/RNA-seq), and spatial transcriptomics (Xenium) assays to comprehensively analyze genetic alterations, gene expression patterns, and their spatial relations. Our findings revealed a higher proportion of TP53 and KRAS mutations in metastatic samples compared to primary tumor samples. Analysis of snRNA-seq data identified distinct tumor cell populations characterized by gene signatures from 12 published gene expression modules. Primary tumor cells were predominantly enriched for tumor intestinal stem cell-like and intestinal modules, while metastatic tumor cells showed higher representation of injury repair, epithelial-mesenchymal transition (EMT), osteoblast-like, and neuroendocrine modules. Notably, a microsatellite instability-high (MSI-high) metastatic sample exhibited relatively higher signatures of absorptive intestinal and squamous cancer modules than other microsatellite-stable samples, suggesting a unique gene expression profile in MSI-high CRC metastasis. To spatially study these cancer modules, we defined spatial tumor microregions as regions of tumor cells separated by stromal tissues. Using the Xenium assay, we mapped the gene modules within these microregions and observed significant enrichment of EMT and injury repair modules at the peripheral tumor microregions in metastatic samples. Conversely, the hypoxia module was more prominently enriched at the core of tumor microregions in metastatic samples compared to primary tumors. These findings suggest a strong association between tumor cell location within the microenvironment and their gene expression profiles, shedding light on the spatial dynamics of tumor heterogeneity in CRC metastasis. In summary, our results suggest that the heterogeneity of gene expression in metastatic CRC is driven by genetic alterations and strongly influenced by the tumor microenvironment.


 Chien-Wei Peng, Michael Iglesia, Adrienne Visani, Xiang Li, Xiangwei Fang, Reyka G. Jayasinghe, Jingxian Liu, John Herndon, Jacqueline Mudd, Andrew Houston, Preet Lal, Siqi Chen, Ambrose Plante, André Targino da Costa, Kapur B. Dhami, Jennifer Ponce, Robert Fulton, Michael Heinz, Milan Chheda, Feng Chen, Ryan C. Fields, Li Ding. Spatial and multi-omic profiling of metastatic colorectal cancer reveals key cellular ecosystems and drivers of heterogeneity [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6257.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/947de9f123e944f865748b3a75364432746d6f15" target='_blank'>
              Abstract 6257: Spatial and multi-omic profiling of metastatic colorectal cancer reveals key cellular ecosystems and drivers of heterogeneity
              </a>
            </td>
          <td>
            Chien-Wei Peng, Michael D. Iglesia, Adrienne Visani, Xiang Li, Xiangwei Fang, R. Jayasinghe, Jingxian Liu, John M Herndon, Jacqueline L Mudd, Andrew Houston, Preet Lal, Siqi Chen, Ambrose Plante, A. T. D. Targino da Costa, Kapur B. Dhami, J. Ponce, Robert S Fulton, Michael Heinz, M. Chheda, Feng Chen, Ryan C. Fields, Li Ding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 The tumor microenvironment (TME) of diffuse large B-cell lymphoma (DLBCL) has been recently dissected and different TME-based subtypes of DLBCL proposed (Kotlov et al. Cancer Discov. 2021, Steen et al. Cancer Cell. 2021). Previous studies have, however, not systematically explored the spatial organization of the DLBCL TME and its association with biological and clinical factors.



 We performed multiplex immunofluorescence using a 12-plex antibody panel on diagnostic tumor samples from 99 patients with DLBCL. We characterized B cells, T cells, tumor-associated macrophages (TAMs), and immune checkpoint molecules. We processed images using Ilastik and CellProfiler softwares, and segmented nuclei with a pretrained deep-learning model. We phenotyped the cells using the Phenograph algorithm combined with manual thresholding. We identified recurrent cellular neighborhoods (RCNs) and performed cell-cell interaction analyses using the scimap package (Python 3.9). We correlated the findings with biological and clinical characteristics and survival.



 We analyzed over 1.2 x 106 single cells with a median of 12 354 cells per patient. We identified 17 cell phenotypes, including various subgroups of B cells, T cells, and TAMs. Of the cell types, B cells were the most abundant (mean 55%), followed by CD4+ helper and CD8+ cytotoxic T cells (means 14% and 10%), and M2-like TAMs (mean 8%) with great heterogeneity between the samples. We identified 10 different RCNs with distinct cellular compositions across DLBCLs. The most common RCN was an immune-poor RCN (30% of the cells), followed by an RCN with high B cell content and a moderate immune cell infiltrate (22% of the cells). We also identified different T cell and TAM-rich RCNs. The RCNs subdivided DLBCLs into immune-poor areas and areas with diverse immune cell infiltrates. Notably, HLA-ABC positive lymphomas consisted of larger proportions of CD4+ T cell-rich RCNs (p < 0.05). Defined by the shortest average distance, immune-poor RCNs were located close to the RCNs that had high B cell content combined with moderate immune cell infiltrates but far from the T cell-rich RCNs. However, when CD8+ T cell-rich and immune-poor RCNs were close to each other, the outcome was favorable (progression-free survival (PFS); p < 0.05), whereas the proximity of PD-L1+ B cell-rich and CD8+ T cell-rich RCNs was associated with poor survival (PFS; p < 0.05). Finally, we correlated RCN proportions with lymphoma microenvironment (LME) and genetic subtypes. Immune-poor RCNs were common in the depleted LME subtype, whereas PD-L1+ M2-like TAM-rich RCNs were more common in the inflamed LME. In contrast, PD-L1+ M2-like TAM-rich RCNs were rare in EZB-type DLBCLs.



 Our data show that DLBCL TME is organized into RCNs that are associated with distinct molecular and subtypic features and patient outcomes.



 Matias Autio, Suvi-Katri Leivonen, Leo Meriranta, Marja-Liisa Karjalainen-Lindsberg, Teijo Pellinen, Sirpa Leppä. Spatial analysis of the tumor microenvironment in diffuse large B-cell lymphoma reveals clinically relevant recurrent cellular neighborhoods [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6470.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4db159a15c79e0c166f89e06a6be6cfce7707f63" target='_blank'>
              Abstract 6470: Spatial analysis of the tumor microenvironment in diffuse large B-cell lymphoma reveals clinically relevant recurrent cellular neighborhoods
              </a>
            </td>
          <td>
            M. Autio, S. Leivonen, L. Meriranta, M. Karjalainen-Lindsberg, T. Pellinen, S. Leppä
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="PURPOSE
Peritoneal metastases (PM) in colorectal cancer (CRC) portend a poor prognosis. We sought to elucidate molecular features differentiating primary tumors (PTs) from PMs and actionable targets facilitating transcoelomic dissemination and progression.


EXPERIMENTAL DESIGN
We performed multi-omic profiling of 227 samples from 136 patients, including 56 primary tumor (PT) and 120 synchronous PMs comprising 34 matched PT-PM pairs. Whole exome, and bulk RNA-seq analysis was conducted to identify underlying genomic aberrations and transcriptomic differences between primary and peritoneal lesions. We spatially characterized the microenvironment of tumor-stroma compartments and studied the roles of stromal phenotypes in promulgating tumorigenesis.


RESULTS
Whole exome sequencing found genomic alterations and clonality patterns between PTs and PMs remain broadly similar. Transcriptomic profiles however, suggest a transition as tumors reach the peritoneum towards a more mesenchymal tumor profile and fibrotic tumor microenvironment. Applying spatial profiling, we identify a fibro-collagenous and immune-infiltrated stromal phenotype (stromal cluster [SC] 2) characterized by increased cancer-associated fibroblasts, memory B cells, M2 macrophages and T-cell exhaustion. These findings were orthogonally validated by multiplex immunohistochemistry. Patients with SC2 stroma had poorer survival and were characterized by high SERPINE-1 (PAI-1) expression. PM in patients with SC2 stroma were associated with enriched oncogenic pathways such as TGF-β. PAI-1 inhibition of CRC PM cell-lines with a novel biologic demonstrated reduced IL2-STAT5 and TGF-β pathways and cell death.


CONCLUSIONS
Our findings unveil distinctive and actionable molecular signatures, offering deeper insights into the intricate crosstalk between tumor cells and stromal microenvironments enabling PM in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e55caa0c96eda1ece0ec721a1f932b608bbc992" target='_blank'>
              Spatial heterogeneity, stromal phenotypes, and therapeutic vulnerabilities in colorectal cancer peritoneal metastasis.
              </a>
            </td>
          <td>
            J. C. Ong, Joseph J Zhao, Ying Liu, Supriya Srivastava, D. Chia, Ying En Quek, Xiaonan Fan, Haoran Ma, Kiekyong Huang, T. Sheng, Qiu Xuan Tan, Gillian Ng, J. W. Tan, J. J. Lee, L. Loo, Li Yen Chong, Xue Wen Ong, S. Tay, T. Hagihara, Angie Tan, Craig Ryan Cecil Joseph, Melissa C C Teo, Josephine Hendrikson, C. Chong, Wanyu Guo, C. Chia, Jolene S M Wong, Chin Jin Seo, Mingzhe Cai, Yvonne Tay, Kevin M S Sim, R. Tay, Robert Walsh, M. Guaglio, F. Morano, Ming Teh, Huey Yew Jeffrey Lum, Tony K H Lim, Louis Vermeulen, M. Bijlsma, K. Lenos, S. Klempner, J. Yeong, W. Yong, F. Pietrantonio, P. Tan, R. Sundar
          </td>
          <td>2025-04-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Triple negative breast cancer (TNBC) is a breast cancer subtype that manifests fast disease progression and very poor overall survival. Despite initial response to standard of care, development of therapy resistance remains the major challenge in TNBC clinical management. Delineating the mechanisms underlying resistance development has been impeded by the still unknown cell population of origin and lack of longitudinal molecular profiling. To investigate this question, we applied a CRISPR based lineage tracing system into TNBC cell lines. Longitudinal samples from an Antibody Drug Conjugate (ADC), Sacituzumab Govitican sensitive to resistance were collected in vitro via stepwise drug dose escalation and in vivo via xnenograft serial transplantation. We leveraged cellular barcoding with single cell multiome from 10X genomics to simultaneously capture clonal identity, accessible chromatin and gene expression and hence, directly linked lineage with functional cell states. Preliminary results suggest that TNBC cells adopted distinct cell states shortly after drug exposure with characteristics including upregulation of epithelial-to-mesenchymal transition (EMT) genes, stemness markers and downregulation of immune related pathways. Clonal level analysis revealed only a subset of clones survived initial treatment and are enriched in specific “primed” cell states. Next, we will construct single cell lineage trees and quantify clonal dynamics and cell state transition rates. We will identify and validate regulators of cellular plasticity and explore if potential drug combinations will prevent emergence of resistance cell state. Overall, this study will build a comprehensive map of the phenotypic trajectory towards chemotherapy resistance and pinpoints biomarkers of therapy response in triple negative breast cancer.


 Chufan Zhang, Chu Pan, Dave Cescon, Mathieu Lupien. Dissecting chemotherapy resistance development in triple negative breast cancer through single cell multiome and lineage tracing [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5541.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96dda01278fdb738773a58b49aca47aa0308c83d" target='_blank'>
              Abstract 5541: Dissecting chemotherapy resistance development in triple negative breast cancer through single cell multiome and lineage tracing
              </a>
            </td>
          <td>
            Chufan Zhang, Chu Pan, D. Cescon, M. Lupien
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a1ac1da292bdfa7cb083f4691cc613492b11ea" target='_blank'>
              Pan-cancer Analysis Identified Ectopic RUNX1T1 Associated with Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, I. Kim, Su Deng, Ping Mu
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="
 Prostate cancer (PCa) remains a leading cause of cancer-related deaths, with castration-resistant metastatic disease posing significant therapeutic challenges. The tumor microenvironment, particularly cancer-associated fibroblasts (CAFs), plays a crucial role in cancer development and in its progression to an aggressive and lethal stage. Previously, we identified a specific CAF subtype expressing fibroblast activation protein and platelet-derived growth factor receptor beta (FAP+/PDGFRβ+) as a strong predictor of poor patient prognosis, associated with PTEN loss and PI3K/pAKT activation in adjacent tumor cells. This study aimed to functionally characterize and pharmacologically inhibit this lethal cancer-stroma interaction phenotype. We first developed a 3D co-culture system using patient-derived primary CAFs and human PCa cells that recapitulates the lethal tumor microenvironment phenotype ex vivo and in vitro, providing robust models for functional studies. Utilizing 96-plex single-cell RNA sequencing with antibody barcoding of live cells, we analyzed the transcriptomic landscape of primary CAFs and assessed the impact of mono- and co-culture conditions on both cell types. Each patient-derived CAF model possessed a unique transcriptional profile, indicating patient-specific CAF characteristics. In 3D co-culture with PCa cells, CAFs exhibited upregulation of FAP and degradome genes, suggesting cancer-induced reprogramming of CAFs to remodel the extracellular matrix and support malignant progression. Concurrently, co-cultured CAFs induced pAKT upregulation in PCa cells, implicating stromal cells in promoting cancer cell proliferation. Hence, we functionally confirmed reciprocal cancer-stroma interactions in co-cultured models, mirroring changes seen in human tumor tissues of aggressive prostate cancers. To investigate potential therapeutic interventions, we applied a 96-plex single-cell pharmacotranscriptomic platform to evaluate drug responses using a library of 30 cancer drugs and key pathway inhibitors. Our findings indicate that dual inhibition of PI3K-AKT signaling in cancer cells and stromal signaling in CAFs could pave the way for combinatorial therapies targeting the reciprocal interactions between PCa cells and CAFs. This approach highlights the opportunity to develop precision cancer medicine strategies aimed at disrupting critical cell interactions within the tumor microenvironment in a tailored fashion.


 Astrid Murumägi, Frida Rantanen, Alice Dini, Mariliina Arjama, Katja Välimäki, Elina Multamäki, Tuomas Mirtti, Antti Rannikko, Teijo Pellinen, Daniela Ungureanu, Olli Kallioniemi. Multiplexed single-cell pharmacotranscriptomics and spatial tumor biology reveals targetable reciprocal cancer-stromal interactions in prostate cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6481.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad68c593de3480a301b38845ef8182dbdcc71b6" target='_blank'>
              Abstract 6481: Multiplexed single-cell pharmacotranscriptomics and spatial tumor biology reveals targetable reciprocal cancer-stromal interactions in prostate cancer
              </a>
            </td>
          <td>
            A. Murumägi, Frida Rantanen, Alice Dini, Mariliina Arjama, K. Välimäki, Elina Multamäki, T. Mirtti, Antti S. Rannikko, T. Pellinen, Daniela Ungureanu, Olli Kallioniemi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Nasopharyngeal carcinoma (NPC) is an Epstein-Barr virus (EBV) associated malignancy characterized by a dense lymphocyte infiltrate and a complex tumor microenvironment (TME). This heterogenous ecosystem influences tumor progression, immune evasion and the variable treatment outcomes observed in NPC patients. Conventional approaches such as bulk RNA sequencing lack the resolution to capture the complex heterogeneity of the TME. Spatial transcriptomics allows us to uncover the heterogeneity and cellular interactions at a single-cell resolution, providing insights into the tumor biology of NPC.



 Recent formalin-fixed, paraffin-embedded (FFPE) NPC tissue samples were selected for analysis with the NanoString CosMx Spatial Molecular Imaging platform. Quality control, dimension reduction and unsupervised clustering at a resolution of 0.6 using the Louvain/Leiden algorithm were performed using Seurat package (v5.1.0). Cluster level annotation was performed using the singleR package using the human primary cell atlas reference. Cell type identities were validated by comparing cluster specific marker genes with canonical markers of known cell types.



 Six NPC tissue biopsies were fully interrogated comprising 254 ROIs. Post-filtering, 162, 290 cells were examined. A total of 7 clusters were identified, broadly comprising immune, stromal and epithelial cells. Tumor composition was heterogenous (B cell 4.68-50.4%, T cell 0.74-8.78%, endothelial cells 0.91-3.66%, epithelial cells 28.5-66.8%, fibroblasts 1.16%-23.9%, macrophages 6.43%-29.1%, pro B cell CD34+ 0.01-7.1%) despite similar clinical stages III-IV.



 We describe the single-cell spatial landscape of nasopharyngeal carcinoma, providing insights into the heterogeneity of the TME. We will evaluate epithelial - immune cell interactions, differential biological processes at the cellular level and compare them with deep gene expression profiles from our larger laser-capture microdissection mRNA-Seq cohort of patients to glean further insights.



 Chee Yit Lim, Yi Ren, Wei Keat Teo, Chor Hiang Siow, Kwok Seng Loh, Raymond KY Tsang, Bingcheng Wu, Joshua K. Tay. Interrogating the tumor microenvironment of nasopharyngeal carcinoma by single-cell spatial transcriptomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6463.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6af959f7143f19a656c32e73ddcf95ebee2a8fbb" target='_blank'>
              Abstract 6463: Interrogating the tumor microenvironment of nasopharyngeal carcinoma by single-cell spatial transcriptomics
              </a>
            </td>
          <td>
            Chee Yit Lim, Yi Ren, W. K. Teo, Chor Hiang Siow, K. Loh, Raymond KY Tsang, Bingcheng Wu, Joshua K. Tay
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d1c1b28cd57c3d7a9126fb0efbcbda9de4833e" target='_blank'>
              The genomic and transcriptional landscape of the spinal cord H3K27-altered diffuse midline glioma suggests the potential therapeutic strategy
              </a>
            </td>
          <td>
            Yiwei Xiao, Mengyao Li, Qiang Gao, Zhihong Qian, Yukui Shang, Linkai Jing, Zili Zhen, Yong Ai, Guihuai Wang, Kehkooi Kee, Wei Zhang
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Cancer research has long sought to understand the complex tumor microenvironment and the molecular underpinnings of tumor progression. Traditional methods often fail to capture the spatial organization and cell-cell interactions within tumors. The CosMx® Spatial Molecular Imager (SMI) with its CosMx Whole Transcriptome (WTX) panel offers a transformative solution, providing spatially resolved, single-cell transcriptomics with nanometer-scale precision. This platform enables the mapping of RNA molecules within their native tissue context, which is essential for uncovering critical biological insights related to cancer cell behavior, tumor microenvironment interactions, and therapeutic responses.
 We developed and optimized a tailored protocol for in vitro cell-based assays using the CosMx WTX platform. Key steps, including permeabilization and enzymatic digestion, were fine-tuned to preserve RNA integrity and cellular morphology, ensuring high-quality transcript detection. Cancer cell lines, U2OS (osteosarcoma) and SH-SY5Y (neuroblastoma), were cultured and analyzed using CosMx WTX to map the spatial distribution of RNA transcripts. The data were compared to bulk RNA-seq results to validate reproducibility and accuracy. The assay’s spatial resolution allows for precise mapping of individual RNA molecules within their spatial context, facilitating phenotype-associated spatial transcriptomics.
 In our analysis, the CosMx WTX platform successfully quantified approximately 7,300 transcripts per cell in U2OS cells and 1,900 transcripts per cell in SH-SY5Y cells, showing high reproducibility and strong agreement with bulk RNA-seq data. This high-throughput capability allows for robust transcriptomic profiling of diverse cancer cell types. By correlating gene expression data with morphological and functional phenotypes, we were able to visualize the spatial organization of transcripts within cells. The platform’s nanometer-scale resolution also supports mapping of cancer organoids, offering insights into intratumoral heterogeneity, treatment resistance, and metastatic processes.
 The CosMx WTX cell-based assay represents a significant advancement in spatially resolved transcriptomics. By enabling high-resolution analyses, this platform provides valuable insights into cancer biology and offers a powerful tool for drug screening and companion diagnostic development. The integration of spatial transcriptomics with protein-level data holds the potential to transform our approach to cancer diagnostics and therapy, advancing precision medicine and improving patient outcomes.


 Yi Cui, Margaret Hoang, Shanshan He, Rachel Liu, Tushar Rane, Rustem Khafizov, Haiyan Zhai, Christine Kang, Michael Rhodes, Joseph Beechem. Whole transcriptome spatial exploration of cultured tumor cells at subcellular resolution [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2779.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddc6a274f48d59a3fed08cd82e2b131cfa0c1209" target='_blank'>
              Abstract 2779: Whole transcriptome spatial exploration of cultured tumor cells at subcellular resolution
              </a>
            </td>
          <td>
            Yi Cui, Margaret L Hoang, Shanshan He, Rachel Liu, Tushar Rane, R. Khafizov, Haiyan Zhai, Christine Kang, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Neuroblastoma (NB) is a tumor of the sympathetic nervous system and is the most common extracranial solid malignancy in children. Patients with high-risk NB have much poorer outcomes compared with patients with low- and intermediate-risk neuroblastoma. These striking differences in clinical outcomes led us to investigate how the tumor microenvironment (TME) differs between high-risk and low-risk NB patients. We examined the cancerous and non-cancerous cells, along with other components, in the TME to determine their potential to either promote or suppress tumor growth, and to determine their association with a specific tissue spatial organization.



 In this study, we profiled 27 pediatric NB patients (n = 14 high-risk and 13 low-risk samples) using GeoMx DSP Whole Transcriptome Atlas (WTA) (Nanostring Technologies) and Phenocycler Fusion (Akoya Biosciences). We analyzed a total of 67 regions of interest (ROIs) using spatial transcriptomics, guided by fluorescent markers (CD45, Synaptophysin, CD56, and DAPI). ROIs were segmented into tumor regions (CD56+Synaptophysin+CD45-) and TME regions (CD45+CD56-). We conducted spatial proteomics to phenotype cells, map their locations, and assess functional states by profiling 53 markers. This 53-plex protein panel enabled us to identify various cell types within the NB TME, covering relevant lineages (tumor, lymphoid, myeloid, and stromal), subtypes (e.g., CD4 T cells, CD8 T cells, B cells, macrophages and neutrophils), and functional states (e.g., Ki67 and PD-1). Spatially resolved signatures in the NB identified characteristics of cellular neighborhoods that are linked to NB patients’ survival outcomes.



 Within the tumor regions, we identified an upregulation of the glutathione metabolism pathways that are involved in the suppression of ferroptosis in high-risk NB tumors, which could be targeted with GPX4 inhibitors. We also discovered that NB with macrophage-rich and stroma-secluded immune cell neighborhoods had worse outcomes. Stroma-secluded immune cells were more frequent in high-risk NB, which may contribute to the resistance mechanisms to immunotherapy and overall survival of patients with high-risk NB.



 Our study highlights the need to combine the genomic profile of NB TME with spatially defined cellular neighborhoods to better understand tumor aggressiveness. Considering the upregulation of ferroptosis suppressive pathways and the macrophage-rich neighborhood in high-risk NB, our findings suggest that inducing ferroptosis with GPX4 inhibitors might be a promising therapy for high-risk NB. Overall, our study revealed key genes and pathways in high-risk NB and provides a new therapeutic avenue for patients with high-risk NB. This hypothesis is currently being investigated in our lab using animal models.



 Cui Tu, Chin Wee Tan, James Monkman, Ana Clara Almeida, Gabrielle Antonio-Carreon, Natacha Omer, Giaan Hull, Kimberly Chung, Ahmed Mehdi, Antonietta Salerno, Orazio Vittorio, Aaron Mayer, Cleber Machado de Souza, Fernanda de Almeida Brehm Pinhati, Selene Elifio-Esposito, Wayne Nicholls, Lucia de Noronha, Arutha Kulasinghe, Fernando Souza-Fonseca-Guimaraes. Spatial profiling of tumor microenvironment uncovers potential novel therapeutic approach in pediatric neuroblastoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2705.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6ce39bb3cec413d591d56991415df20b385ea43" target='_blank'>
              Abstract 2705: Spatial profiling of tumor microenvironment uncovers potential novel therapeutic approach in pediatric neuroblastoma
              </a>
            </td>
          <td>
            Cui Tu, Chin Wee Tan, J. Monkman, Ana Clara Almeida, Gabrielle Antonio-Carreon, Natacha Omer, Giaan Hull, Kimberly Chung, Ahmed M Mehdi, Antonietta Salerno, Orazio Vittorio, Aaron Mayer, Cleber Machado de Souza, Fernanda de Almeida Brehm Pinhati, Selene Elifio-Esposito, Wayne Nicholls, Lucia de Noronha, A. Kulasinghe, Fernando Souza-Fonseca-Guimaraes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Tumor immune microenvironment (TME) heterogeneity, shaped by diverse immune and stromal cells, significantly influences tumor metastasis, yet the specific immune ecotypes involved and their relationship to intra-tumor heterogeneity remain unclear. In this study, we present a comprehensive spatial and cellular multi-omics analysis of 44 primary and 54 lung metastatic lesions from 34 HCC patients. By integrating public single-cell RNA sequencing data from 187, 290 tumor cells across 146 HCC cases, we define HCC cellular states and uncover their organization in high-dimensional spatial expression profiles from paired HCC primary and metastatic lung lesions. Our integrated analysis, incorporating data with three independent cohorts, uncovers three pro-metastatic modules—Cell cycle(G2/M), Cellular Stress and Hypoxia —that contribute to early HCC lung metastasis. Notably, the first two modules are specific to lung metastasis, while the latter one is common across all extrahepatic organs. We show how tumor cells with distinct modules interact with various immune cell types, facilitating metastasis through different mechanisms. Additionally, these tumor cells form multi-layered structures of varying sizes, with specific state pairs preferentially clustering across multiple scales, all within a 55 µm limit. In vivo studies further reveal that interactions between G2/M-high tumor cells and Angio-macrophages and T3-neutrophils accelerate lung dissemination. Treatment with the G2/M inhibitor paclitaxel effectively disrupts this process, reducing metastasis. These findings provide critical insights into how tumor cell heterogeneity reshapes the TME to drive metastasis, underscoring the importance of spatially informed features for personalized cancer therapies. Our work emphasizes the potential of tailored strategies to prevent and manage metastasis based on tumor-immune interactions.


 Jinghan Zhu, Guangzhen Cai, Bin Guo, Xiaoping Chen, Huilan Zhang, Hong Zhu, Wanguang Zhang. Comprehensive spatial mapping uncovers immune ecotypes linked to tumor heterogeneity in HCC lung metastasis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5276.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/527fb3619bec4cdd8432c9c09f5972cd49b69aac" target='_blank'>
              Abstract 5276: Comprehensive spatial mapping uncovers immune ecotypes linked to tumor heterogeneity in HCC lung metastasis
              </a>
            </td>
          <td>
            Jinghan Zhu, Guangzhen Cai, Bin Guo, Xiaoping Chen, Huilan Zhang, Hong Zhu, Wanguang Zhang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="

 Multicellular programs in the tumor microenvironment (TME) drive cancer pathogenesis and response to therapy but remain challenging to identify and profile clinically. Here, we present Spatial EcoTyper, a multimodal machine learning framework for large-scale profiling of spatially dependent cell states and multicellular ecosystems, termed spatial ecotypes (SEs).



 and Findings: By integrating five million single-cell and spot-level spatial transcriptomes from 10 diverse human neoplasms, including carcinomas and melanomas, we identified a rich tapestry of conserved regional plasticity across nine TME cell types. We then applied Spatial EcoTyper to deeply dissect these data, revealing nine novel SEs with broad conservation - each with unique biology, TME cell states, geospatial features, and clinical outcome associations. SEs were linked to specific biogeographic domains, including the tumor core, margin, and adjacent stroma, and were remarkably well-validated in 235 held-out tumor samples profiled by either single-cell RNA sequencing, sequencing-based spatial transcriptomics (10x Visium), or high-resolution imaging-based spatial transcriptomics (Vizgen MERSCOPE and NanoString CosMx). By deconvolving SEs from over 1,200 bulk tumor RNA-seq profiles, we discovered striking associations between SE levels and response to immune checkpoint inhibitors (ICIs), outperforming nearly 40 potential correlates of ICI response, including previously described signatures of tertiary lymphoid structures and tumor tissue PD-L1 expression. Notably, baseline levels of a proinflammatory SE - localized to the tumor core and defined by a hub of unique T cell, macrophage, fibroblast, and endothelial cell states - strongly forecasted ICI benefit, while another SE - localized to the tumor margin and defined by myofibroblast and hypoxia-associated endothelial cell states - portended ICI resistance.



 Our data reveal fundamental units of TME organization and demonstrate a platform for large-scale profiling of spatial cellular ecosystems in any tissue, with implications for improved forecasting of immunotherapy response and the development of novel therapeutic strategies.



 Wubing Zhang,Erin L. Brown,Abul Usmani,Noah Earland,Hyun Soo Jeon,Irfan Alahi,Nicholas P. Semenkovich,Janella C. Schwab,Chloé B. Steen,Chloe M. Sachs,Bilge Gungoren,Faridi Qaium,Peter K. Harris,Jacqueline Mudd,Qingyuan Cai,Antonella Bacchiocchi,Ryan C. Fields,Mario Sznol,Ruth Halaban,David Y. Chen,Aadel A. Chaudhuri,Aaron M. Newman. High-resolution profiling of the tumor microenvironment with spatial ecotypes [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 153.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483a65f81564c748a7e6be963230e929d2717f09" target='_blank'>
              Abstract 153: High-resolution profiling of the tumor microenvironment with spatial ecotypes
              </a>
            </td>
          <td>
            Wubing Zhang, Erin L. Brown, A. Usmani, N. Earland, Hyun Soo Jeon, I. Alahi, N. Semenkovich, Janella C. Schwab, Chloé B. Steen, Chloe M. Sachs, Bilge Gungoren, F. Qaium, P. Harris, Jacqueline L Mudd, Qingyuan Cai, A. Bacchiocchi, Ryan C. Fields, Mario Sznol, R. Halaban, David Y. Chen, A. Chaudhuri, Aaron M. Newman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the third most common malignancy, with over 1.85 million new cases and 850, 000 deaths annually. Significant intratumoral heterogeneity in CRC contributes to recurrence, chemotherapy resistance, and poor prognosis.
 This study aimed to explore the spatial distribution of subtype clones and the diversity of the tumor microenvironment (TME), which are not captured by the traditional gene expression-based CMS classification. Additionally, it sought to investigate cancer trajectories and their contributions to metastatic phenotypes using spatial transcriptomics (ST).



 Spatial analysis was performed using public 10X Genomics Visium data from 4 CRC and 4 CRC-liver metastasis (CRC-LM) patients (GSE226997, GSE217414). Cell-type deconvolution utilized single-cell RNA-seq datasets (GSE178318, GSE132465). Copy number alterations (CNA) were inferred, and subclone development trajectories were modeled using inferCNV, DPT, and PAGA analyses. Pathway and functional enrichment analyses, expression score calculations, and correlation analyses were conducted using ClusterProfiler, AddModuleScore, and statistical methods. A Wilcoxon test with an adjusted p-value < 0.05 was used to determine statistical significance.



 Distinct CRC and CRC-LM subclones were identified through CNA inference, revealing spatially segregated CNA clusters. These included malignant subclones and boundary-associated clusters, characterized by varying proportions of differentiation and mesenchymal traits. Immune-related signaling pathways demonstrated differential activation, with increased MAPK, JAK-STAT, and TNFα-NFκB in specific regions and elevated WNT, VEGF signaling, MYC, and E2F4 in others. Certain subclones also showed enrichment of SPP1+ macrophages and myofibroblasts, indicating diverse invasive characteristics. Evolutionary trajectories of subclones, inferred through hierarchical trees, revealed CSC features and a common ancestral clone. Progression toward malignancy was associated with enhanced pathway activity linked to critical biological processes. In CRC-LM, subclones were categorized by TAM subtypes, with one subtype displaying significantly elevated SPP1 expression compared to primary CRC. Immune-related features were reduced in metastatic tissues, reflecting a less immunogenic TME.



 Intratumoral heterogeneity in CRC evolves from a single common ancestral clone into distinct progression states, resulting in spatially unique TMEs. Understanding the interactions between clones across different progression states offers valuable insights into disease dynamics and therapeutic opportunities.



 YeonJu Lee, Woong-Yang Park. Mapping intratumoral heterogeneity in colorectal cancer: Spatially resolved subclones and tumor microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2666.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcbbcac8a736304b9cab15ef0da632985a22219d" target='_blank'>
              Abstract 2666: Mapping intratumoral heterogeneity in colorectal cancer: Spatially resolved subclones and tumor microenvironment
              </a>
            </td>
          <td>
            YeonJu Lee, Woong-Yang Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Objective The tumor microenvironment (TME) plays a crucial role in tumor progression. Recent advancements in single-cell sequencing technology have identified Tc17 cells, a newfound subset of CD8+ T cells with a phenotype similar to Th17, within the gastric cancer (GC) microenvironment. However, the role of Tc17 cells are unclear. Materials and method We collected 296,879 cells from the single-cell sequencing data of 65 GC samples, along with spatial transcriptomic data from 10 GC samples, to further explore the role of Tc17 cells. Multicolor immunohistochemical staining demonstrated the presence of Tc17 cells in the GC TME. SCENIC analysis was performed to identify the specific transcription factors of Tc17. Pseudotime analysis of T cells was conducted to decipher the differentiation trajectory of Tc17. Additionally, cell-cell interaction analysis revealed the interactions between Tc17 cells and tumor cells. Finally, functional validation through CCK-8 proliferation assays, wound Healing assays, and transwell assays conclusively demonstrated the tumor-promoting effects of IL-17A and IL-26 on gastric tumor cells. Results Our results indicate that Tc17 cells are absent from blood and exclusively found in tissues, with greater enrichment in tumor tissues compared to normal tissues. Among CD8+ T cells, Tc17 exhibits the weakest cytotoxic capacity and the highest anti-inflammatory profile, suggesting reduced tumor-killing ability. Pseudotime analysis revealed that Tc17 cells ultimately progress towards a state of exhaustion, losing their capacity to eliminate tumor cells. Survival analysis further correlates the presence of Tc17 cells with poor prognosis. Notably, a robust interaction between Tc17 cells and tumor cells was observed in GC. We found that tumor cells can recruit Tc17 cells via the CXCL16-CXCR6 axis, this finding was validated on spatial transcriptome data. And in turn, Tc17 cells may promote tumor progression by secreting IL-17A and IL-26, as functionally corroborated by CCK-8 Assay, wound healing, and transwell assay in vitro. Conclusions These findings reveal the immunosuppressive role of Tc17 cells in gastric cancer, provide a new perspective for understanding the immunosuppressive mechanism of the gastric cancer tumor microenvironment, and provide a potential target for the development of targeted therapeutic strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4a49cf0a193cdeef05a09af604560eb6cfc60d0" target='_blank'>
              A novel Tc17 population recruited by tumor cells promotes tumor progression in gastric cancer
              </a>
            </td>
          <td>
            Yichao Wang, Hao Yuan, Yayun Tan, Li Wu, Xinchen Zhao, Zeyuan Yang, Shaoxing Dai, Naixue Yang
          </td>
          <td>2025-05-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Spatial transcriptomic technologies demonstrate the ability to comprehensively investigate the complexity of the tumor microenvironment, providing crucial insights into the interplay of different cellular and acellular components that orchestrate the tumor progression, angiogenesis, immune evasion, and metastasis. Here, we introduce Visium HD 3’, a novel assay for unbiased spatial gene expression profiling of fresh frozen tissue sections mounted on a standard glass slide. The Visium HD slide’s array has a gapless design that enables integration of unsupervised gene expression clustering data with high-resolution H&E images from the same tissue section, allowing precise profiling of finer anatomical features with high tissue coverage at single cell-scale resolution. Furthermore, this novel reverse transcription-based assay generates cDNA products compatible with both short reads and long reads sequencing, extending its utility beyond gene expression analysis to enable immune profiling and the discovery of isoforms and novel transcripts on a spatial level unlocking deeper insights into cancer diversity.
 Our research focused on deciphering the spatial heterogeneity of gene expression patterns within the tumor microenvironment, shedding light on the interactions between cancer cells, stromal cells, and vasculature. Visium HD 3’ was performed on fresh frozen human breast, pancreatic, liver, and lung cancer tissues. Spatial profiles of clinically-relevant cancer biomarkers such as AR, MUC1, CDH1, and ERBB2 were able to highlight tumor-prone regions in the tissues. Expression of known prognostic angiogenic markers such as VEGF, FGFR, PDGF was inline with the tumor region, indicating tumor growth and metastasis. Additionally, targeting stromal markers such as POSTN, LUM, MMP1, ACTA2, and SFRP4 revealed a distinct boundary between the tumor and stromal layer. The high resolution of this technology further enabled profiling of macrophage and T cell infiltration in cells surrounding the tumor by identification of immune markers like CD4, CD68, CD3E, PTPRC and CCL5.
 Our study highlights the importance of exploring the spatial organization of the entire transcriptome in cancer tissues and demonstrates that Visium HD 3’ platform is a powerful tool for unraveling the complexities of the tumor microenvironment. These findings lay the groundwork for the development of innovative therapeutic strategies and precision medicine approaches, ultimately contributing to improve outcomes for cancer patients.


 Debashish Chitnis, Marco Serra, Josh Gu, Anushka Gupta, Nancy Conejo, Aarushi Kalaimani, Govinda Kamath, Zixue Ma, Monica Nagendran, Joey Arthur, Julia Cowen, Anuj Patel, David Sukovich, Augusto M. Tentori. Visium HD 3’ enables unbiased whole transcriptome spatial profiling of tumor microenvironment in fresh frozen cancer tissues at single cell-scale resolution [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5301.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/310c5ef14d052c47aef8e3e074bf1ba11aa00b97" target='_blank'>
              Abstract 5301: Visium HD 3’ enables unbiased whole transcriptome spatial profiling of tumor microenvironment in fresh frozen cancer tissues at single cell-scale resolution
              </a>
            </td>
          <td>
            Debashish Chitnis, Marco Serra, Josh Gu, Anushka Gupta, Nancy Conejo, Aarushi Kalaimani, Govinda Kamath, Zixue Ma, Monica Nagendran, Joey Arthur, Julia Cowen, Anuj Patel, David Sukovich, Augusto Tentori
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Tertiary lymphoid structures (TLS) are frequently observed in various solid tumors, where they play a pivotal role in regulating anti-tumor immune responses. Despite their importance, our understanding of TLS and their interactions with cancer cells within complex tissue environments remains limited. Recent advances in spatial transcriptomics (ST) technology offer new possibilities for investigating the spatial and phenotypic heterogeneity of TLS.
 We developed TLSphenotyper, a computational framework for automatic TLS segmentation, labeling, extraction, and comprehensive profiling of TLS using spot-level ST data. It enables reproducible and efficient large-scale analysis of TLS. To accurately map TLS and characterize their cellular composition, we created a robust TLS-specific gene signature and enhanced the resolution of spot-level ST data to near-single-cell resolution. TLSphenotyper computes cell type-specific gene signature scores to generate a detailed cell type probability map across the tissue section. This map facilitates segmentation of key cellular components within TLS, enabling precise spatial mapping and colocalization analysis of major cell types and their transcriptomic states. Additionally, TLSphenotyper automatically analyzes and classifies TLSs based on their maturation stages, offering insights into their phenotypic and functional heterogeneity. Its accuracy and precision were validated using single-cell ST data generated from adjacent tissue sections. Further benchmarking against established deconvolution methods demonstrated superior accuracy and granularity in spatial cell type inference, cell organization and co-location analysis within TLS regions.
 To further investigate TLS and their interactions with cancer cells in a pan-cancer setting, we curated a comprehensive dataset of publicly available and in-house spot-level ST data, encompassing 340 tissue sections from 12 distinct cancer types. This represents the largest collection of ST data on human cancer assembled to date. Using TLSphenotyper, we identified and characterized 695 TLSs across 340 tissues. Our analysis revealed substantial variation in TLS presence, size, location, density, cellular composition, and maturation stage across cancer types. TLSs at different maturation stages exhibited distinct transcriptomic profiles. Notably, cancer cells in proximity to TLS demonstrated significantly different activity across key signaling pathways compared to those farther away, suggesting that TLS interaction can shape cancer cell expression profiles and cellular states. Additionally, patients classified based on the maturation stages of TLS demonstrated distinct survival outcomes. Together, this study provides novel insights into the spatial dynamics and phenotypic heterogeneity of TLS, identifies potential targets for modulating TLS, and establishes a valuable resource for the research community.


 Kyung Serk Cho, Yunhe Liu, Guangsheng Pei, Jianfeng Chen, Tieling Zhou, Khalida M. Wani, Alejandra G. Serrano, Sharia D. Hernandez, Akshaya S. Jadhav, Daiwei Zhang, Hannah C. Beird, Jiahui Jiang, Diana Shamsutdinova, Xinmiao Yan, Yibo Dai, Enyu Dai, Ashley L. Wilson, Anne Waldrep, Fuduan Peng, Wei Lu, Yang Liu, Jimin Min, Paola A. Guerrero, Idania C. Julio, Tong Yin, Shin Hsu Shi, Sabitha Prabhakaran, Scott E. Kopetz, Dipen M. Maru, Alexander Lazar, Amir A. Jazaeri, Michael Green, Mingyao Li, Catherine Sautes-Fridman, Wolf H. Fridman, Andrew Futreal, Luisa M. Soto, Humam Kadara, Anirban Maitra, Suet Yi Leung, Jianjun Gao, Linghua Wang. Pan-cancer spatial characterization of tertiary lymphoid structures [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3962.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f29ad2e19f93fc33c2a089984de394bfc868d5e" target='_blank'>
              Abstract 3962: Pan-cancer spatial characterization of tertiary lymphoid structures
              </a>
            </td>
          <td>
            K. S. Cho, Yunhe Liu, Guangsheng Pei, Jianfeng Chen, Tieling Zhou, Khalida M Wani, Alejandra G Serrano, Sharia D Hernandez, Akshaya S. Jadhav, Daiwei Zhang, Hannah C. Beird, Jiahui Jiang, Diana Shamsutdinova, Xinmiao Yan, Yibo Dai, Enyu Dai, Ashley L. Wilson, Anne Waldrep, F. Peng, Wei Lu, Yang Liu, Jimin Min, Paola A. Guerrero, Idania C. Julio, Tong Yin, Shin Hsu Shi, Sabitha Prabhakaran, S. Kopetz, D. Maru, Alexander J. Lazar, A. Jazaeri, Michael Green, Mingyao Li, C. Sautès-Fridman, Wolf H. Fridman, Andrew Futreal, L. Soto, H. Kadara, Anirban Maitra, Suet Yi Leung, Jianjun Gao, Linghua Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/222e5440bee40a1cfc03b8b975d31d717ee09614" target='_blank'>
              Integrating Single-Cell Biophysical and Transcriptomic Features to Resolve Functional Heterogeneity in Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            Ye Zhang, Lydie Debaize, Adam Langenbucher, Jenalyn Weekes, Ioulia Vogiatzi, Teemu P. Miettinen, Mingzeng Zhang, Emily Sumpena, Huiyun Liu, Sarah M. Duquette, L. Hackett, Jeremy Zhang, Sona Baghiyan, R. Redd, M. Aryee, M. Davids, Austin I. Kim, C. Ryan, David M. Weinstock, S. Manalis, M. Murakami
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35acfa591bf2d48662dc328bde4741026068f8bb" target='_blank'>
              Neoantigens and Stochastic Fluctuations Regulate T Cell Proliferation in Primary and Metastatic Malignant Brain Tumors
              </a>
            </td>
          <td>
            Maheshwor Poudel, William Stewart, Ciriyam Jayaprakash, Jayajit Das
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GB), particularly IDH-wildtype, is the most aggressive brain malignancy with a dismal prognosis. Despite advances in molecular profiling, the complexity of its tumor microenvironment and spatial organization remains poorly understood. This study aimed to create a comprehensive single-cell and spatial atlas of GB to unravel its cellular heterogeneity, spatial architecture, and clinical relevance.


METHODS
We integrated single-cell RNA sequencing data from 26 datasets, encompassing over 1.1 million cells from 240 patients, to construct GBmap, a harmonized single-cell atlas. High-resolution spatial transcriptomics was employed to map the spatial organization of GB tissues. We developed the Tumor Structure Score (TSS) to quantify tumor organization and correlated it with patient outcomes.


RESULTS
We showcase the applications of GBmap for reference mapping, transfer learning, and biological discoveries. GBmap revealed extensive cellular heterogeneity, identifying rare populations such as tumor-associated neutrophils and homeostatic microglia. Spatial analysis uncovered seven distinct tumor niches, with hypoxia-dependent niches strongly associated with poor prognosis. The TSS demonstrated that highly organized tumors, characterized by well-defined vasculature and hypoxic niches, correlated with worse survival outcomes.


CONCLUSIONS
This study provides a comprehensive resource for understanding glioblastoma heterogeneity and spatial organization. GBmap and the TSS provide an integrative view of tumor architecture in GB, highlighting hypoxia-driven niches that may represent avenues for further investigation. Our resource can facilitate exploratory analyses and hypothesis generation to better understand disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f160a16e8e12a7fc63680546f0557a806342875" target='_blank'>
              Charting the Single-Cell and Spatial Landscape of IDH-Wildtype Glioblastoma with GBmap.
              </a>
            </td>
          <td>
            Cristian Ruiz-Moreno, Sergio Marco Salas, Erik R Samuelsson, Mariia Minaeva, Ignacio L Ibarra, Marco Grillo, S. Brandner, Ananya Roy, K. Forsberg-Nilsson, M. Kranendonk, Fabian J. Theis, Mats Nilsson, H. Stunnenberg
          </td>
          <td>2025-05-02</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Current objective response rates of recurrent or metastatic head and neck squamous cell carcinoma (RM HNSCC) to immune checkpoint inhibition (ICI) remain less than 20%, highlighting the innate variability of patient tumors that preclude their immunogenic clearance. Such outcomes in HNSCC have driven the exploration of the immune and tumor centric properties that drive such refractory responses to ICIs. This study expands on this knowledge base by leveraging highly multiplexed imaging combined with a deep learning cell type classification pipeline to comprehensively evaluate the spatial properties of tumor and immune cells associated with ICI response.



 65 RM HNSCC pre-ICI biopsies on tissue microarrays were imaged using highly multiplexed immunofluorescence performed on the Akoya PhenoCycler platform. Cell type classification was performed using a deep learning platform developed by Nucleai. Marker positivity was used to further assign cell type functional states. Supervised spatial clustering and tumor cell density contours were used to define tissue regions in TMA cores. Cell frequencies, tumor-immune cell ratios and cell-cell interactions were computed within these regions as input for tests against clinical correlates.



 95, 000 tumor cells and 76, 000 immune cells were phenotyped for both functional and metabolic states. Macrophages and CD4 T cells were the most abundant immune cell types in the cohort, with both CD4 and CD8 T cells demonstrating highly OXPHOS+ metabolically active states. Univariate analysis for progression free survival indicated CD8 T cells in tumor regions were associated with better PFS, while CD4 Tregs expressing ICOS in stromal regions correlated with poorer PFS. Interestingly, both B cells and Granulocytes at the tumor-stroma interface correlated with poorer PFS. Furthermore, the interaction between tumor cells and highly metabolically active fibroblasts within tumor regions was also associated with poorer PFS.



 and future directions: Our study interrogated spatial compartments in HNSCC tissues for cellular features that may predict response to ICI therapy. Such approaches are limited by the provision of sufficient cells to derive meaningful biology. Future aims for this study include deep learning methods to contrast tissue subgraphs to identify regions that differ significantly between patients, and thereby provide novel insights into the relevant spatial communities that drive cancer immunotherapy responses in patient biopsies.



 Ettai Markovits, James Monkman, Aaron Kilgallon, Thazin Nwe Aung, David Rimm, Arutha Kulasinghe. Tumor microenvironment profiling of multiplexed images in head and neck cancer reveals spatial features associated with immunotherapy response [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 156.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8315f0147f2ad2f550ec4867f837baf7efd724be" target='_blank'>
              Abstract 156: Tumor microenvironment profiling of multiplexed images in head and neck cancer reveals spatial features associated with immunotherapy response
              </a>
            </td>
          <td>
            E. Markovits, J. Monkman, Aaron Kilgallon, T. Aung, David L Rimm, A. Kulasinghe
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 e14575


 Background:
 Over the last decade, immunotherapy has revolutionized cancer treatment, offering transformative outcomes for many patients. However, its effectiveness in treating solid tumors remains limited due to challenges such as T cell exhaustion, immunosuppressive tumor microenvironment (TME), lack of tumor-specific antigens and a complex immune landscape. Solid tumors also induce systemic immune changes that evolve dynamically with tumor progression. To improve patient outcome, it is essential to understand the interplay between localized and systemic immune responses.
 Methods:
 To address these challenges, we employed an integrative approach leveraging CyTOF and Imaging Mass Cytometry (IMC) technologies. Matched samples, including peripheral blood mononuclear cells (PBMCs), tumor-derived cells (TDCs) and formalin-fixed, paraffin-embedded (FFPE) tumor tissues from cancer patients, were analyzed. A custom CyTOF panel with over 40 markers was designed to profile immune subsets, checkpoint molecules, cytokines and transcription factors. Samples were barcoded, stained and analyzed using a CyTOF XT system. For spatial immune profiling, FFPE tumor tissues were stained with a 40-plus-marker IMC panel to characterize immune, stromal and tumor cells and assess immune activation states. Data was acquired with the Hyperion XTi Imaging System, enabling high-resolution imaging of tumor microenvironments.
 Results:
 CyTOF analysis revealed heterogeneity in T cell exhaustion and activation profiles in PBMCs and TDCs, highlighting the functional diversity of immune responses. IMC technology provided spatial insights into immune cell interactions within the TME, identifying distinct patterns of immune infiltration and congregation around tumor cells and stromal elements. Comparative analysis of CyTOF and IMC data uncovered shared and divergent phenotypes between peripheral and tumor-resident immune compartments, identifying biomarkers that link systemic immune profiles with TME dynamics.
 Conclusions:
 These findings offer invaluable insights into the immunological mechanisms driving antitumor responses. By integrating functional and spatial immune profiling, our approach identifies predictive biomarkers and potential therapeutic targets, guiding the development of personalized immunotherapies. This work highlights the importance of understanding systemic and localized immune responses to improve cancer treatment outcomes and optimize treatment strategies. For Research Use Only. Not for use in diagnostic procedures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8358df8b3da76d7b7dab6d3ef70735cb54fac5bc" target='_blank'>
              Integrated immune profiling approach for immuno-oncology: Unveiling biomarkers and therapeutic targets with mass cytometry.
              </a>
            </td>
          <td>
            Ling Wang, T. Pfister, J. Y. Chwee, Stephen Li, Qanber Raza, Shakir Hasan, Michael J. Cohen, Nikesh Parsotam, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Recent advances have shed light on the molecular heterogeneity of small cell lung cancer (SCLC), yet the spatial organizations and cellular interactions in tumor immune microenvironment remain to be elucidated. Here, we employed co-detection by indexing (CODEX) and multi-omics profiling to delineate the spatial landscape for 165 SCLC patients, generating 267 high-dimensional images encompassing over 9.3 million cells. Integrating CODEX and genomic data revealed a multi-positive tumor cell neighborhood within ASCL1+ (SCLC-A) subtype, characterized by high SLFN11 expression and associated with poor prognosis. We further developed a cell colony detection algorithm (ColonyMap) and revealed a spatially assembled immune niche consisting of antitumoral macrophages, CD8+ T cells and natural killer T cells (MT2) which highly correlated with superior survival, and predicted improving immunotherapy response in an independent cohort. This study serves as a valuable resource to study SCLC spatial heterogeneity and offers insights into potential patient stratification and personalized treatments.


 Chaoqiang Deng, Changbin Zhu, Tao Hu, Jing Wang, Haiquan Chen. Integrative spatial analysis reveals tumor heterogeneity and immune colony niche related to clinical-outcomes in small cell lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2100.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1bafbb3ba9f42ba8adab0b213f43d4265484aab" target='_blank'>
              Abstract 2100: Integrative spatial analysis reveals tumor heterogeneity and immune colony niche related to clinical-outcomes in small cell lung cancer
              </a>
            </td>
          <td>
            C. Deng, Changbin Zhu, Tao Hu, Jing Wang, Haiquan Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT Background Breast cancer is a highly heterogeneous disease, characterized by tumor and nontumor cells at various cell states. Ecotyper is an innovative machine learning framework that quantifies the tumor microenvironment and delineates the tumor ecosystem, demonstrating clinical significance. However, further validation is needed in breast cancer. Methods Ecotyper was applied to identify multiple cellular states and tumor ecotypes using large‐scale breast cancer bulk sequencing data, followed by a detailed analysis of their associations with clinical classification, molecular subtypes, survival prognosis, and immunotherapy response. Identified subtypes were further characterized using single‐cell and spatial data sets to reveal molecular profiles. Results In a comprehensive analysis of 6578 breast cancer samples from four data sets, Ecotyper identified 69 cellular states and 10 tumor ecotypes. Of these, 37 cellular states significantly correlated with overall survival. Notably, specific states within epithelial cells, macrophages/monocytes, and fibroblasts were linked to a worse prognosis. CE2 abundance was identified as the most significant marker indicating unfavorable prognosis and was further validated in an additional data set of 116 HER2‐negative patients. These biomarkers also indicated the efficacy of neoadjuvant immunotherapy in breast cancer. CE2‐high cancers were characterized by an abundance of basal‐like epithelial cells, scant lymphocytic infiltration, and activation of hypoxia signaling. Single‐cell analysis showed that CE2‐high areas were rich in SPP1‐positive tumor‐associated macrophages(TAM), basal‐like epithelial cells, and hypoxic cancer‐associated fibroblasts(CAF). Spatially, these regions were often peripheral in triple‐negative breast cancer, adjacent to fibrotic/necrotic zones. Multiplex immunofluorescence confirmed the enrichment of SPP1+CD68+TAM and HIF1A+SMA+CAF in hypoxic triple‐negative breast cancer (TNBC) regions. Conclusions Ecotyper identified novel biomarkers for breast cancer prognosis and treatment prediction. The CE2‐high region may represent a hypoxic immune‐suppressive niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed777dbf5c26fef9472ca8bd002f079780d98595" target='_blank'>
              The Identification of Novel Prognostic and Predictive Biomarkers in Breast Cancer via the Elucidation of Tumor Ecotypes Using Ecotyper
              </a>
            </td>
          <td>
            Feng Du, J. Ju, F. Zheng, Song-Lin Gao, Peng Yuan
          </td>
          <td>2025-05-27</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Tumor progression and metastasis is driven by interplays between cancer cells and their surrounding tumor microenvironment (TME). Investigating the spatiotemporal transcriptomic profile of primary and metastatic tumors sheds light on how the differences in spatial architecture shape the TME and promote tumor metastasis. Here, we profiled three primary and three lung metastatic clear cell renal cell carcinoma (ccRCC) samples using the CosMx platform, which provides a 999-gene transcriptome with spatial coordinates. After cell segmentation and annotation, we further incorporate the cellular context and spatial information to identify niches by grouping the cells with similar neighboring cells. Cell-cell communication and architecture were analyzed to understand the spatial organization in primary and metastatic ccRCC. From niche analysis, 12 distinctive niches were identified, including cancer, stroma, alveolar, tertiary lymphoid structure (TLS), and mast niches, formed by 25 different cell types. Analyzing cancer niches, we found that the metastatic TME is much more immunosuppressive compared to the TME in primary ccRCC. There is a higher infiltration of exhausted T cells, Treg and M2-like macrophages in metastases. Moreover, the fibroblasts in metastases have higher expression of VEGF and metalloproteinases, which led to angiogenesis and worse prognosis. Architecture analysis showed that lung metastatic ccRCC cells, surrounded by a stromal barrier, are not directly interacting with tumor infiltrating leukocytes (TILs), whereas primary ccRCC cells interact more with TILs. Moreover, compared to the TLSs in primary ccRCC, the ones in metastases showed features of dysfunction, which was correlated with a worse prognosis. Our results show the impact of spatial architecture on primary and metastatic ccRCC, as well as the relation between spatial organization of the cellular context and cell-cell communication. In addition, it suggests that immune cells and cancer-associated fibroblasts (CAFs) are essential components of the ccRCC TME. Their interaction constitutes a major factor not only for tumor progression but also limiting therapy response in metastatic ccRCC.


 Yufei Wang, Jae-Won Cho, Yasmin Nabil Laimon, Wenxin Xu, Aseman Bagheri Sheshdeh, Nithyassree Murugan, Hsien-Chi Yuan, Jon Wee, David Braun, Toni Choueiri, Catherine Wu, Sabina Signoretti, Gordon Freeman, Martin Hemberg, Wayne Marasco. ccRCC metastasis rewires tumor microenvironment to increase immunosuppression and fibrosis. [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 770.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2a1c8fd12302b7cec59cd6c236cab65714760b" target='_blank'>
              Abstract 770: ccRCC metastasis rewires tumor microenvironment to increase immunosuppression and fibrosis
              </a>
            </td>
          <td>
            Yufei Wang, Jae-Won Cho, Y. Laimon, Wenxin Xu, A. B. Sheshdeh, Nithyassree Murugan, Hsien-Chi Yuan, Jon Wee, David Braun, Toni K. Choueiri, Catherine J. Wu, S. Signoretti, Gordon Freeman, M. Hemberg, Wayne A. Marasco
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer (BRCA) is a complex disease influenced by the tumor microenvironment, where interactions between immune cells and cancer cells play a crucial role in tumor progression and response to therapy. Understanding the intricacies of these interactions requires detailed analysis at the single-cell level, enabling the identification of specific immune cell subpopulations and their functional roles within the tumor milieu. This study comprehensively analyzed immune cell subpopulations and macrophage subtypes in BRCA using single-cell RNA sequencing technology and various computational tools. Initially, Sc-Type software accurately identified and annotated immune cell subpopulations, followed by CNV analysis using infercnv software, revealing significant CNV variations in epithelial cells. Subsequently, macrophages were re-clustered into 5 clusters, and their biological significance and functional features were assessed. CellChat analysis elucidated potential interactions between macrophage subtypes and BRCA cells, primarily through SPP1-CD44 and LGALS9-CD44 signaling networks. Additionally, CytoTRACE and Monocle were employed to analyze cellular plasticity and differentiation trajectories of macrophage subtypes. Furthermore, efferocytosis-related gene set scoring, transcription factor analysis, and risk score development were conducted, followed by immune infiltration and tumor mutation burden analysis, revealing increased immune infiltration and higher TMB levels in the high-risk group. These findings offer crucial insights into the interaction mechanisms of immune cells and macrophage subtypes within the BRCA tumor microenvironment, aiding in the understanding of tumor progression and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61362329e16982a536f86bb6af1a06a1e1ff5d77" target='_blank'>
              Mapping immune cell dynamics and macrophage plasticity in breast cancer tumor microenvironment through single-cell analysis
              </a>
            </td>
          <td>
            Wang Chen, Siyu Zeng, Junyong Zhong, Jian Zou, Yanli Lei, Xiaohan Chen, Qinghua Mei, Qianhua Luo
          </td>
          <td>2025-04-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1477b05d2464a1bcee44f14957311d44000b130" target='_blank'>
              scTransMIL Bridges Patient-Level Phenotypes and Single-Cell Transcriptomics for Cancer Screening and Heterogeneity Inference
              </a>
            </td>
          <td>
            Zhenchao Tang, Fang Wang, Fan Yang, Jiangning Song, C. Chen, Jianhua Yao
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Prostate cancer is marked by extensive cellular diversity within its tumor microenvironment (TME), significantly impacting disease progression and responses to treatment. The study aims to dissect this heterogeneity at the single-cell level, identify distinct subpopulations within key cell types such as circulating tumor cells (CTCs) and T lymphocytes, and uncover potential therapeutic targets.



 We analyzed single-cell RNA-seq data from 16 prostate cancer samples, which included over 170, 000 cells. Initially, we performed data integration and normalization using Reciprocal Principal Component Analysis (R-PCA) to mitigate batch effects and enhance data quality. For dimensionality reduction, both PCA and UMAP (Uniform Manifold Approximation and Projection) were applied, allowing us to visualize and identify patterns in the high-dimensional dataset. Clustering of cells was conducted using the k-means algorithm to categorize cells into distinct clusters based on their gene expression profiles. Finally, cell type classification within these clusters was achieved using Support Vector Machines (SVMs), enabling the precise identification of cellular identities and states based on their molecular signatures.



 Our comprehensive analysis unveiled a remarkable diversity among the circulating tumor cells (CTCs) within the Taiwanese prostate cancer tumor microenvironment. Specifically, we identified four distinct CTC clusters, each infused with different cellular components that suggest unique roles in tumor dynamics. CTC1 is comprised purely of epithelial cells, presenting a classical phenotype of circulating tumor cells, which underscores the conventional understanding of CTCs in cancer dissemination. Meanwhile, CTC2 is characterized by an infusion with N1 neutrophils, highlighting its potential role in promoting inflammatory responses that could affect tumor growth and patient prognosis. In contrast, CTC3 includes an infusion with N2 neutrophils, indicating a possible contribution to enhanced tumor progression and metastatic potential, thereby suggesting a target for therapeutic intervention to curb tumor spread. Lastly, CTC4 is infused with plasmacytoid dendritic cells, which are known for their role in immune suppression, thereby aiding the tumor in evading immune surveillance.



 This study underscores the utility of single-cell technologies in uncovering the complex cellular interactions within prostate cancer. By detailing the unique profiles of CTC subclusters and their associations with specific immune cells, our research provides novel insights into the tumor microenvironment and identifies new targets for immunotherapy. Future research will focus on validating these markers and integrating broader genomic data to refine treatment approaches and enhance patient outcomes.



 Han Hong Huynh, Tzu-Hao Chang, Shauh-Der Yeh, Chia-Ling Hsieh, Shian-Ying Sung. Advanced single-cell RNA-seq reveals complex interactions between circulating tumor cells and immune subtypes in prostate cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6269.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8794d489519dce88daef3e8a367a86c1c1372d6" target='_blank'>
              Abstract 6269: Advanced single-cell RNA-seq reveals complex interactions between circulating tumor cells and immune subtypes in prostate cancer
              </a>
            </td>
          <td>
            H. H. Huynh, Tzu-Hao Chang, S. Yeh, Chia-Ling Hsieh, S. Sung
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Background Gastric cancer (GC) exhibits high heterogeneity that relies on the oncogenic properties of cancer cells and multicellular interactions in the tumour microenvironment. However, the heterogeneity of GC and their molecular characteristics are still largely unexplored. Methods We employed single‐cell and spatial transcriptomics to comprehensively map the intra‐tumoural heterogeneity within GC. Additionally, in vitro experiments, clinical sample analyses, and patient‐derived organoid models (PDOs) were conducted to validate the key interaction patterns between tumor cells and stromal cells. Results Seven robust meta‐programs (MP1–MP7) in GC were defined with distinct biological significance and spatial distributions. MP3 and MP4 were intimately associated with distinct CD8 T cells skewed toward a cytotoxic or exhaustion state, while MP7, characterised by the highest degree of malignancy, harboured an immune lockdown microenvironment around it and spatially associated with myofibroblasts (myCAFs). Notably, we clarified the interplay between the MP7 and myCAFs, where MP7 induces the chemotactic migration of fibroblasts and promoting their transformation into myCAFs via GDF15/TGFBR2, and in turn, myCAFs‐derived RSPO3 up‐regulates EGR1 to promote the transformation to MP7 in GC cells and human PDOs. Ultimately, the accumulation of myCAFs around MP7 led to fewer infiltration of CD8 T cells, resulting an immune‐deprived microenvironment and the diminished efficacy of immunotherapy. Additionally, based on the gene expression signatures of MP7 GC cells, we predicted specific drugs and verified more potent inhibitory effects of Taselisib and Lapatinib for MP7 GC cells than conventional drugs at the same concentration. Conclusion Taken together, these results deepened the understanding of GC heterogeneity and paved the way for novel therapeutic strategies by targeting MP7 GC cells and their interaction loop with myCAFs in GC treatment. Key points Seven robust meta‐programs (MP1‐MP7) were identified in gastric cancer. MP7 was strongly correlated with cancer metastasis and poor survival of gastric cancer patients. MP7 promoted fibroblast transformation into myCAFs via GDF15/TGFBR2, creating an immune lockdown microenvironment. MyCAFs induced MP7 transformation via the RSPO3/EGR1 pathway, promoting gastric cancer cell migration. Taselisib and Lapatinib were potent inhibitors of MP7 GC cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6915321c1b17b2ab2500b930b7287f803a2304b" target='_blank'>
              Deciphering of intra‐tumoural heterogeneity and the interplay between metastasis‐associated meta‐program and myofibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xiong-yan Wu, Zhijian Jin, Baolong Li, Yifan Lu, Junyi Hou, Lizhong Yao, Zhenjia Yu, Qingqing Sang, Beiqin Yu, Jian-fang Li, Chen Li, Chao Yan, Zhenggang Zhu, Kaiwen Tang, Bingya Liu, Liping Su
          </td>
          <td>2025-04-28</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Tumor heterogeneity, characterized by the presence of diverse cell populations within a tumor, is a key feature of the complex nature of cancer. This diversity arises from the emergence of cells with varying genomic, epigenetic, transcriptomic, and phenotypic profiles over the course of the disease. Host factors and the tumor microenvironment play crucial roles in driving both inter-patient and intra-patient heterogeneity. These diverse cell populations can exhibit different behaviors, such as varying rates of proliferation, responses to treatment, and potential for metastasis. Both inter-patient heterogeneity and intra-patient heterogeneity pose significant challenges to cancer therapeutics and management. In retinoblastoma, while heterogeneity at the clinical presentation level has been recognized for some time, recent attention has shifted towards understanding the underlying cellular heterogeneity. This review primarily focuses on retinoblastoma heterogeneity and its implications for therapeutic strategies and disease management, emphasizing the need for further research and exploration in this complex and challenging area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c59ee10750e92caf754d8bc3c66525cdef7d43" target='_blank'>
              Tumor heterogeneity in retinoblastoma: a literature review
              </a>
            </td>
          <td>
            Rani Pallavi, Bihari lal Soni, Gaurab Kumar Jha, Shalini Sanyal, Azima Fatima, S. Kaliki
          </td>
          <td>2025-04-22</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the second leading cause of cancer-related deaths globally, claiming over 900, 000 lives annually. In the US, CRC disproportionately affects certain populations, with higher prevalence among racial and ethnic minorities. The tumor microenvironment (TME) plays a pivotal role in CRC progression, therapeutic response, and overall patient outcomes. However, the spatially intricate interactions within the TME that distinguish effective from ineffective tumor control remain poorly understood. Addressing these gaps is essential for enhancing our understanding of spatial heterogeneity and advancing immunotherapy development.



 To investigate these dynamics, we employed a multi-modal approach using re-engineered co-detection by indexing (CODEX) on publicly available data (Schürch et al., 2020) from 35 late-stage CRC patients. Our systematic workflow classified cells, analyzed protein expression, and spatially represented the data. Cells were grouped into three main classes—stroma, tumor, and immune—based on structural characteristics and functional roles. Protein expression analysis was conducted by integrating data for individual surface protein expression, enabling direct comparisons between the stroma, tumor, and immune classes. Through this framework, 56 protein markers were profiled to identify conserved cellular neighborhoods (CNs) that characterize the spatial and functional organization within the CRC TME.



 Protein expression analysis revealed distinct patterns across the stroma, tumor, and immune classes for the biomarkers profiled. These markers exhibited differential expression profiles across the three classes, highlighting their unique roles within the TME. For instance, immune markers such as CD3, CD8, CD68, and PD-1 showed significant variation between the tumor and immune classes, underscoring their spatial organization and functional importance within the TME. Similarly, functional markers such as MMP9 and Ki-67 demonstrated distinct expression dynamics, reflecting their roles in tumor progression and immune responses. Auxiliary markers, including cytokeratins, highlighted structural differences unique to the stroma class. These results provided a comprehensive understanding of the spatial and functional interplay between immune, tumor, and stromal components in the CRC TME.



 This study provides a deeper understanding of protein expression within the CRC TME, enhancing the classification and spatial characterization of TME. These results provide key insights into spatial heterogeneity within the TME through distinct protein expression profiles. These findings provide valuable guidance for future research in drug discovery, with the potential to significantly advance the field of precision oncology.



 Enrique Velazquez Villarreal, Ei-Wen Yang, Fernando C. Diaz, Maria Ninova. Spatial profiling of the tumor microenvironment in colorectal cancer: Insights into immune, tumor, and stromal interactions using high-dimensional protein analysis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5322.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c0b646451e8f6a513c46aa0a01996017a71aac6" target='_blank'>
              Abstract 5322: Spatial profiling of the tumor microenvironment in colorectal cancer: Insights into immune, tumor, and stromal interactions using high-dimensional protein analysis
              </a>
            </td>
          <td>
            Enrique Velazquez Villarreal, E.-W. Yang, Fernando C. Diaz, Maria Ninova
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Gastric cancer exhibits diverse malignant cell states that contribute to intratumor heterogeneity (ITH), a fundamental factor that drive tumor progression, metastasis, and treatment resistance. While previous single-cell RNA sequencing (scRNA-seq) studies have provided novel insights into ITH in gastric cancer, they are often constrained by small sample sizes and limited transcriptional coverage.



 In this study, we curated, annotated, and integrated scRNA-seq datasets from 21 studies, comprising 626 samples from 316 individuals, including healthy donors and patients with precancerous lesions and gastric cancer. Within the epithelial compartment, malignant and non-malignant cells were distinguished using three complementary computational methods. To refine malignant cell subclustering, we developed a novel approach that incorporates patient-specific copy number variations, minimizing the impact of inter-patient heterogeneity on clustering analysis.



 Following rigorous quality control and data integration, unsupervised clustering analysis classified 2, 088, 266 high-quality cells into six major cell types or compartments: B cells, T cells, innate lymphoid cells, myeloid cells, stromal cells, and epithelial cells. Within the epithelial cell compartment, we identified 12 distinct malignant cell states characterized by unique expression profiles and phenotypic characteristics, such as cell proliferation capacity and lineage-specific plasticity. These cancer cell states exhibited substantial variation in the activity of hallmark cancer signaling pathways, with their presence and composition closely linked to tumor aggressiveness and clinical outcomes. Proliferative cancer cells and translationally active cancer cells predominated in intermediate tumor stages, dormant cells were prevalent in advanced stages, and epithelial-mesenchymal transition (EMT) states were enriched in metastatic tumors. Additionally, we identified a malignant cell population with high expression of intestinal lineage markers in patients with signet ring cell carcinoma, and these findings were validated using spatial transcriptomic.



 This study presents a comprehensive single-cell transcriptomic atlas of gastric cancer epithelial cells, providing valuable insights into transcriptional ITH. Our findings define distinct malignant cell states, elucidate their dynamics, and highlight their clinical significance, establishing a framework for understanding how these states contribute to disease progression and metastasis.



 Enyu Dai, Yanshuo Chu, Yang Liu, Ruiping Wang, Yibo Dai, Jaffer A. Ajani, Linghua Wang. A comprehensive single-cell atlas of gastric cancer reveals malignant cell state dynamics and plasticity in tumor progression and metastasis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7475.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d545d97bc8751905973e21664bea8353ccb95ea" target='_blank'>
              Abstract 7475: A comprehensive single-cell atlas of gastric cancer reveals malignant cell state dynamics and plasticity in tumor progression and metastasis
              </a>
            </td>
          <td>
            Enyu Dai, Yanshuo Chu, Yang Liu, Ruiping Wang, Yibo Dai, Jaffer A Ajani, Linghua Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Neoadjuvant chemo-immunotherapy has significantly improved the treatment landscape for patients with ALK/EGFR wt resectable non-small cell lung cancer (NSCLC), offering novel opportunities for translational and clinical investigations. By leveraging deep molecular profiling through spatial transcriptomics integrated with single-cell RNA-sequencing from public atlases, and serum proteomic profiling, we report a case of a patient with resectable NSCLC and a solitary synchronous brain metastasis showing a complete pathologic response after chemo-immunotherapy, and a durable event-free survival. The deep profiling of the tumor immune microenvironment of both pre-treatment brain metastasis and post-treatment primary lung tumor tissues unveiled a key role of B lymphocytes at different maturation state, spanning from naïve to plasma cells, in mediating tumor eradication. Notably, the formation of several tertiary lymphoid structures in the regression bed of post-treatment primary tumor was observed, suggesting the in situ generation of high-affinity antibody and specific immune memory response. This multimodal approach paves the way for the discovery of novel biomarkers at both tissue and systemic levels, fostering improved patient stratification and guiding clinical decisions on post-surgical treatment escalation or de-escalation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ca3e849d66be330813305765da5566fa9efc94" target='_blank'>
              Spatial transcriptomics highlights B cells as key contributors to a complete and durable response to chemo-immunotherapy in a patient with resectable NSCLC
              </a>
            </td>
          <td>
            Nicla Porciello, F. Gallina, G. Frisullo, Francesca Fusco, L. D’Ambrosio, V. Balzano, F. de Nicola, Paolo Visca, L. Landi, F. Cappuzzo, Paola Nisticò
          </td>
          <td>2025-05-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed6e6004750330d65706e8b7059577e8d1bc2385" target='_blank'>
              Using 3D Invasion properties of RCC Cell Lines In Vitro to predict their Metastatic Potential In Vivo
              </a>
            </td>
          <td>
            B. Cesana, L. Nemoz-Billet, V. Azemard, C. Pillet, E. Bigot, N. Chaumontel, J.-L. Descotes, N. Osmani, J. Goetz, C. Cochet, O. Filhol
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Novel cellular therapy has the potential to change cancer treatment. However, to design new therapies requires a better understanding of the tumor microenvironment (TME) and the cellular dynamics that occur within the tumor. This requires a high resolution and subcellular view of cancer cells, immune cells while identifying the neighborhood architecture that exists between them. Spatial transcriptomics and spatial proteomics show subcellular information and can be used to identify cellular cell types and find spatial deviation between cells. However, there are a lack of tools to investigate cellular behaviors on an individual cell basis while integrating pathological information, limiting the future of spatial biology. We introduce CellClique, a tool that assists in dissecting TMEs using spatial transcriptomics with the Xenium platform and Hematoxylin and Eosin (H&E) stains. CellClique integrates both generative and analytical capabilities to provide a comprehensive view of TMEs. On the generative end, CellClique uses generative AI to model both gene expressions and cell-neighborhood interactions for cell annotation, gene expression prediction, and gene imputation. On the analytical end, CellClique offers a comprehensive four-way spatial analysis scheme aimed at 1) understanding the effects of neighboring cells on a target cell, 2) dissecting cancer-normal cell boundaries, 3) observing immune invasion mechanisms, and 4) exploring cellular subtypes with the TME. We applied CellClique to study the cellular landscape in lung and colon cancers using multiple different neighborhood analyses. We generated proximal (10 microns), local (60 microns), and distal neighborhoods (120 microns) for each cell within each tissue and found distinct expression profiles of tumor-infiltrated lymphocytes, which are functionally different from lymphocytes excluded from the tumor. Additionally, we discovered expression variation patterns in cancer-normal boundary cell communities with various cell compositions. These compositions changed significantly as distance from the boundary changed. We also compared lymphocytes at different stages of life by analyzing expression differences among lymphocytes classified with over 80% confidence, those with a 50% probability of being lymphocytes, and those with below 20% probability. We further compared how these neighborhoods changed between lung cancer and colon cancer and showed cancer specific spatial deviations and expression. Together, these methods paint a complex picture of the TME but also identifies numerous avenues for future discovery and molecular investigation. These new approaches will guide future cellular therapy and the development of new treatments. The future integration of AI and machine learning will continue to make a tremendous impact on spatial transcriptomics and our approach to treating disease.


 Sachini Weerasekara, Natasha Darras, Nicolas Fernandez, Melinda Chen, Alina Ainbinder, Colles Price. CellClique: Dissecting tumor microenvironments at the single cell level using generative AI and spatial transcriptomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2418.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8623aa3b6277db74a5df00d2e59a3be79b68bfd2" target='_blank'>
              Abstract 2418: CellClique: Dissecting tumor microenvironments at the single cell level using generative AI and spatial transcriptomics
              </a>
            </td>
          <td>
            Sachini Weerasekara, Natasha Darras, Nicolas Fernandez, Melinda Chen, Alina Ainbinder, Colles Price
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Hepatocellular carcinoma (HCC), the third leading cause of cancer deaths worldwide, exhibits significant cellular and molecular heterogeneity. The advent of single-cell transcriptomic methodologies has significantly disclosed its intra-tumoral cellular diversity to an unprecedented level. Interactions between cells within the tumor microenvironment (TME) are crucial for cancer progression and affect how tumors respond to current therapies.



 This study aimed to utilize single-cell data to address two main objectives: (1) to dissect the heterogeneous nature of HCC tumors by identifying tumor subclones and their molecular features, key transcription factors, and implicated pathways; and (2) to explore the intricate communication between tumor cells and immune cells, with a focus on the pathways that significantly influence tumor growth and metastasis.



 The tissue samples of HCC were obtained from nine patients undergoing surgery immediately from the operation theater after resection, with informed consent. Dissociated tumor single cells were subjected to scRNA-seq by 10X Genomics platform.



 Our analysis identified seven distinct tumor subclones, validating the presence of high tumor heterogeneity. Among these, one subclone, designated as SC6, exhibited particularly distinctive molecular characteristics. Further investigation revealed that NFE2L3 exhibited the highest activity score during the TFs analysis, indicating its significant regulatory role in SC6 cluster, and was intricately associated with pathways involved in extracellular matrix organization and immune regulation. Through cellular communication analysis, we discovered that tumor-associated macrophages (TAMs) and dendritic cells (DCs) exhibited the highest communication intensity with SC6 cluster when acting as the source of ligands. Further investigation revealed critical ligand-receptor pairs, including some integrin family genes functioning as receptors. This interaction suggests that these integrins may engage with ligands secreted by TAMs or DCs within the TME, potentially promoting tumor cell proliferation and migration.



 Through comprehensive analysis of scRNA-seq data, we elucidated the intricate heterogeneity within HCC, identifying distinct characteristics of various tumor subclones. Our investigation revealed key transcription factors that predominate in specific subclones. Additionally, we uncovered robust intercellular communication between tumor cells and myeloid cells.



 Qingyang Zhang, Yu-Man Tsui, Vanilla Xin Zhang, Daniel Wai-Hung Ho, Irene Oi-Lin Ng. Single-cell transcriptomic analysis of hepatocellular carcinoma: elucidating intratumoral heterogeneity and immune microenvironment crosstalk [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6484.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca7ecdbac1d09bcfe33d7f9c228aa944ed8e3ef5" target='_blank'>
              Abstract 6484: Single-cell transcriptomic analysis of hepatocellular carcinoma: elucidating intratumoral heterogeneity and immune microenvironment crosstalk
              </a>
            </td>
          <td>
            Qingyang Zhang, Y. Tsui, Vanilla Xin Zhang, Daniel Wai-Hung Ho, Irene Oi-Lin Ng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afbfafb6e752991df103b290fb3be622c923aa5e" target='_blank'>
              Immune Spatial Organization Predicts Metastasis Risk in Aggressive Localized Prostate Cancer
              </a>
            </td>
          <td>
            David D Yang, Aya Abdelnaser, Jeremiah Wala, Alexander J Haas, Alfred A Barney, Eddy Saad, Jett Crowdis, Cora A. Ricker, Jihye Park, Martin T King, P. L. Nguyen, T. Choueiri, A. Tewari, Keyan Salari, M. Taplin, Chin-Lee Wu, E. V. Van Allen
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="The tumor microenvironment (TME) significantly influences cancer prognosis and therapeutic outcomes, yet its composition remains highly heterogeneous, and currently, no highly accessible, high-throughput method exists to define it. To address this complexity, the TMEclassifier, a machine-learning tool that classifies cancers into three distinct subtypes: immune Exclusive (IE), immune Suppressive (IS), and immune Activated (IA), is developed. Bulk RNA sequencing categorizes patient samples by TME subtype, and in vivo mouse model validates TME subtype differences and differential responses to immunotherapy. The IE subtype is marked by high stromal cell abundance, associated with aggressive cancer phenotypes. The IS subtype features myeloid-derived suppressor cell infiltration, intensifying immunosuppression. In contrast, the IA subtype, often linked to EBV/MSI, exhibits robust T-cell presence and improved immunotherapy response. Single-cell RNA sequencing is applied to explore TME cellular heterogeneity, and in vivo experiments demonstrate that targeting IL-1 counteracts immunosuppression of IS subtype and markedly improves its responsiveness to immunotherapy. TMEclassifier predictions are validated in this prospective gastric cancer cohort (TIMES-001) and other diverse cohorts. This classifier could effectively stratify patients, guiding personalized immunotherapeutic strategies to enhance precision and overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e59035a4145d3d05ce3897eefa444405fb68dd" target='_blank'>
              Immunotyping the Tumor Microenvironment Reveals Molecular Heterogeneity for Personalized Immunotherapy in Cancer.
              </a>
            </td>
          <td>
            D. Zeng, Yunfang Yu, Wenjun Qiu, Q. Ou, Qianqian Mao, Luyang Jiang, Jianhua Wu, Jiani Wu, Huiyan Luo, Peng Luo, Wenchao Gu, N. Huang, S. Zheng, Shaowei Li, Yonghong Lai, Xiatong Huang, Yiran Fang, Qiongzhi Zhao, R. Zhou, Huiying Sun, Wei Zhang, J. Bin, Yulin Liao, Masami Yamamoto, Tetsuya Tsukamoto, Sachiyo Nomura, Min Shi, Wangjun Liao
          </td>
          <td>2025-05-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a major health concern, with over 150,000 new diagnoses and more than 50,000 deaths annually in the United States, underscoring an urgent need for improved screening, prognostication, disease management, and therapeutic approaches. The tumor microenvironment (TME)- comprising cancerous and immune cells interacting within the tumor's spatial architecture- plays a critical role in disease progression and treatment outcomes, reinforcing its importance as a prognostic marker for metastasis and recurrence risk. However, traditional methods for TME characterization, such as bulk transcriptomics and multiplex protein assays, lack sufficient spatial resolution. Although spatial transcriptomics (ST) allows for the high-resolution mapping of whole transcriptomes at near-cellular resolution, current ST technologies (e.g., Visium, Xenium) are limited by high costs, low throughput, and issues with reproducibility, preventing their widespread application in large-scale molecular epidemiology studies. In this study, we refined and implemented Virtual RNA Inference (VRI) to derive ST-level molecular information directly from hematoxylin and eosin (H&E)-stained tissue images. Our VRI models were trained on the largest matched CRC ST dataset to date, comprising 45 patients and more than 300,000 Visium spots from primary tumors. Using state-of-the-art architectures (UNI, ResNet-50, ViT, and VMamba), we achieved a median Spearman's correlation coefficient of 0.546 between predicted and measured spot-level expression. As validation, VRI-derived gene signatures linked to specific tissue regions (tumor, interface, submucosa, stroma, serosa, muscularis, inflammation) showed strong concordance with signatures generated via direct ST, and VRI performed accurately in estimating cell-type proportions spatially from H&E slides. In an expanded CRC cohort controlling for tumor invasiveness and clinical factors, we further identified VRI-derived gene signatures significantly associated with key prognostic outcomes, including metastasis status. Although certain tumor-related pathways are not fully captured by histology alone, our findings highlight the ability of VRI to infer a wide range of "histology-associated" biological pathways at near-cellular resolution without requiring ST profiling. Future efforts will extend this framework to expand TME phenotyping from standard H&E tissue images, with the potential to accelerate translational CRC research at scale.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9c463795082212654b2fe4f8e1d20c9792788e" target='_blank'>
              Histology-Based Virtual RNA Inference Identifies Pathways Associated with Metastasis Risk in Colorectal Cancer
              </a>
            </td>
          <td>
            Gokul Srinivasan, Minh-Khang Le, Zarif L. Azher, Xiaoying Liu, Louis J Vaickus, Harsimran Kaur, F. Kolling, Scott M. Palisoul, L. Perreard, Ken S. Lau, Keluo Yao, Joshua Levy
          </td>
          <td>2025-04-23</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Tumor cell heterogeneity and the tumor microenvironment are fundamental questions that heavily influence the process of cancer development, disease progression, and the response to treatments. Knowledge of cancer biology facilitates understanding of pharmacological effects and mechanism of action to improve clinical outcomes. Spatial transcriptomics technology, particularly Stereo-seq, makes the groundbreaking advancement in cancer research with its outstanding performance with high capture efficiency and single cell resolution. It allows scientists to visualize and analyze the complexity of tumors at an unprecedented level of detail. In this study, we mapped tumor heterogeneity and microenvironment of fresh frozen section from colorectal cancer tissue (CRC) at a single-cell resolution with Stereo-seq v1.3 kit. We identified10 cell populations using cell2location, including B cells, fibroblasts, CD4+ T cells, CD8+ T cells, dendritic cells, endothelial cells, epithelial cells, macrophages, mast cells and NK cells. In situ mapping and portion analysis of cell identification revealed a B-cell dominated area which was confirmed by H&E staining of adjacent section. To comprehensively understand the immune activities, we quantified the interactions among different cell types (by StereoSite framework), revealing biologically meaningful intercellular communications mediated by GZMA-PARD3 and CCR10- CCL28 between B cells and epithelial cells. Further pathway enrichment analysis indicated cooperated molecular activities and underlying functions in CRC, which showed interaction activities significantly associated with EGF or BMP signaling pathways. Patteren recognition method identified how functional units assocatied with pathways. Take EGF pathway as an example, results revealed CD4+ T cells and macrophages may coordinated to impact EGF pathway by communication activities. Analysis in this study showed that spatial transcriptomic data at single-cell resolution is vital for exploring intercellular communications, molecular activities, and functional units by providing spatial context. Stereo-seq possess the ability to efficiently gathering evidence of mode of action and resistance mechanims by deeply mapping the spatial profiling in TME. It paves the ways for drug development by removing uncertainties and reducing time and cost. Implementation of Stereo-seq workflow could not only offer impetus to solve unmet needs in area of oncological research, but also bring new findings from lab to patients faster.


 Jiajun Zhang, Xing Liu, Zexian Zeng, Rui Zhang, Tibby Tianbi Duan, Jing Guo, Liang Wu, Sha Liao, Ao Chen. Mapping the tumor microenvironment at single-cell resolution with Stereo-seq v1.3 [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5302.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99969b328faea200d4bc4333f10cd0e2095990a" target='_blank'>
              Abstract 5302: Mapping the tumor microenvironment at single-cell resolution with Stereo-seq v1.3
              </a>
            </td>
          <td>
            Jiajun Zhang, Xing Liu, Zexian Zeng, Rui Zhang, Tibby Tianbi Duan, Jing Guo, Liang Wu, Sha Liao, Ao Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease at the clinical, population, and molecular levels. Using key molecular markers, breast cancer can be classified into at least four major intrinsic subtypes: luminal A, luminal B, HER2-enriched, and basal-like or triple-negative. These subtypes are associated with distinct risk factors, clinical behaviors, treatment responses, and outcomes. Recent research highlights the importance of the breast tumor microenvironment (TME) in shaping tumor evolution and cancer progression. Both tumor and TME cells within an individual patient often exhibit substantial intratumoral heterogeneity (ITH), the extent of which may impact accurate tumor classification and treatment response, but has not been well characterized especially in women of African Ancestry. The goal of this study was to comprehensively investigate ITH of intrinsic subtypes and immune TME markers in breast cancer patients from an East African population. Specifically, we employed the Nanostring GeoMxTM platform to quantify 44 key breast cancer intrinsic and immune subtyping markers in spatially defined regions of breast tumors from 31 Kenyan breast cancer patients. We found that all 31 tumors exhibited some degree of ITH, with a third of them showing substantial ITH. Cell composition (epithelial vs. stromal), spatial location of TME immune cells (tumor-leading edge, proximity to tumor or stroma), and intrinsic subtype (specifically, basal-like vs. luminal A) all contributed significantly to ITH of these markers. When comparing results to those obtained from an in-situ multiplex staining platform InSituPlex (ultivue) and bulk Nanostring nCounter RNA expression, all generated from the same set of cases, we found that intrinsic or immune subtyping by bulk approaches, which is heavily influenced by the predominant subtypes within a tumor, may lead to inaccurate classification when substantial ITH exists. Similarly, sampling-based approaches like tissue microarrays (TMAs) or region-of-interest (ROI)-based assays are also likely to be subjective to inaccurate and inconsistent measurements of immune markers due to sampling bias. In low-resource settings, in-situ assays combined with multi-region scoring may offer a more accurate yet practical alternative to bulk or single-cell assays, which are often costly and technically challenging.


 Mustapha Abubakar, Shahin Sayed, Hela Koka, Scott Lawrence, Karun Mutreja, Xing Hua, Viviane Oluoch, Veronica Ngundo, Zaitun Ajuoga, Difei Wang, Chad Highfill, Petra Lenz, Maria Brown, Aaron M. Rozeboom, Kristine Jones, Amy Hutchinson, Belynda Hicks, Paul S. Albert, Francis Makokha, Stefan Ambs, Jonine Figueroa, Jianxin Shi, Xiaohong Rose Yang. Multi-platform spatial profiling reveals intra-tumor heterogeneity in intrinsic subtypes and immune markers in breast tumors from Kenyan patients [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5316.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfaf1ffd1f6785c103e919dc021f81aaaeb67090" target='_blank'>
              Abstract 5316: Multi-platform spatial profiling reveals intra-tumor heterogeneity in intrinsic subtypes and immune markers in breast tumors from Kenyan patients
              </a>
            </td>
          <td>
            Mustapha Abubakar, Shahin Sayed, H. Koka, Scott Lawrence, Karun Mutreja, X. Hua, V. Oluoch, Veronica M Ngundo, Zaitun Ajuoga, Difei Wang, Chad A. Highfill, Petra Lenz, Maria Brown, Aaron M. Rozeboom, Kristine Jones, A. Hutchinson, B. Hicks, Paul S. Albert, Francis W. Makokha, S. Ambs, Jonine D Figueroa, Jianxin Shi, Xiaohong R. Yang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Identifying cell types and states remains a time-consuming, error-prone challenge for spatial biology. While deep learning increasingly plays a role, it is difficult to generalize due to variability at the level of cells, neighborhoods, and niches in health and disease. To address this, we develop TACIT, an unsupervised algorithm for cell annotation using predefined signatures that operates without training data. TACIT uses unbiased thresholding to distinguish positive cells from background, focusing on relevant markers to identify ambiguous cells in multiomic assays. Using five datasets (5,000,000 cells; 51 cell types) from three niches (brain, intestine, gland), TACIT outperforms existing unsupervised methods in accuracy and scalability. Integrating TACIT-identified cell types reveals new phenotypes in two inflammatory gland diseases. Finally, using combined spatial transcriptomics and proteomics, we discover under- and overrepresented immune cell types and states in regions of interest, suggesting multimodality is essential for translating spatial biology to clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b412946f8bf4bd4d20615dca698261f02621a53" target='_blank'>
              Deconvolution of cell types and states in spatial multiomics utilizing TACIT
              </a>
            </td>
          <td>
            Khoa L. A. Huynh, Katarzyna M Tyc, B. Matuck, Quinn T. Easter, Aditya Pratapa, Nikhil Kumar, Paola Pérez, R. Kulchar, T. Pranzatelli, Deiziane de Souza, Theresa Weaver, Xufeng Qu, Luiz Alberto Valente Soares Júnior, Marisa Dolhnokoff, D. Kleiner, Stephen M. Hewitt, Luiz Fernando Ferraz da Silva, V. Rocha, B.M. Warner, K. Byrd, Jinze Liu
          </td>
          <td>2025-04-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 Head and neck squamous cell carcinomas (HNSCC) exhibit significant inter- and intra-tumor heterogeneity, complicating diagnosis and treatment. Our previous work demonstrated that spheroids developed from patient-derived cells and HNSCC cell lines display distinct proliferative and migratory behaviors. We classified them as hyperproliferative and hypoproliferative spheriods. To uncover the molecular basis of this functional heterogeneity, we utilized single-cell RNA sequencing (scRNA-seq) and advanced computational tools like CellChat. Our analysis revealed key signaling pathways, cellular interactions, and surface markers defining spheroid phenotypes, offering insights into their genetic and functional diversity.



 scRNA-seq was performed on hyperproliferative and hypoproliferative spheroids derived from the UMSCC-22B cell line. Spheroids were cultured, dissociated into single cells, and subjected to high-throughput scRNA-seq. Bioinformatic analyses identified differentially expressed genes, signaling pathways, and cellular states. CellChat was employed to analyze intercellular communication networks within spheroids. Ligand-receptor interactions were modelled to identify pathways mediating cross-talk between distinct cell clusters.



 scRNA-seq revealed distinct transcriptional profiles between the two spheroids. Hyperproliferative spheroids expressed genes related to cell division and growth, while hypoproliferative spheroids expressed genes linked to migration and invasion. Pathway analysis identified differences in angiogenesis, metastasis, and tumor microenvironment interactions. Midkine signalling(MDK-SDC) through basal cells was found to be the principal contributor to the proliferative advantage in hyperproliferative spheriods, supporting their role in tumor expansion and progression. While, Desmosomal signaling (DSC2-DSG2) through keratinocytes and fibroblasts was found to play a critical role in maintaining cellular adhesion and promoting migratory capacity in hypoproliferative spheroids. TCGA data linked MDK to advanced tumor stages, poor survival, and therapy resistance, while DSG2 and DSC2 were overexpressed in metastatic tumors. These findings highlight the distinct functional states of spheroids contributing to tumor heterogeneity.



 The balance between proliferation and migration in spheroids reflects tumor heterogeneity's dual role in cancer progression. Midkine-driven hyperproliferative spheroids anchor tumor growth, while desmosome-mediated hypoproliferative spheroids drive metastasis. Targeting Midkine or modulating desmosomal remodeling could inhibit growth and prevent dissemination, enabling patient-specific therapies.



 Anshu Yadav, Rupesh Chaturvedi, Rana P. Singh. Understanding the genetic heterogeneity within clonal spheroids and its interaction with the tumor microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5054.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195e5c9d321296053fe8059a5c1ae5b1f465a2aa" target='_blank'>
              Abstract 5054: Understanding the genetic heterogeneity within clonal spheroids and its interaction with the tumor microenvironment
              </a>
            </td>
          <td>
            Anshu Yadav, Rupesh Chaturvedi, Rana P. Singh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e13119


 Background:
 The tumor microenvironment (TME) of primary breast tumors (PTs) and breast-to-liver metastases shows heterogeneous infiltration and interactions between tumor, immune, and stromal cells. Hepatic metastases exhibit a more immunosuppressive TME with fewer cytotoxic T cells (CTLs) relative to PTs, which may contribute to increased tumor growth and resistance to systemic therapies. To investigate the immune cell infiltration and immune-tumor interactions in PT and liver metastases, we utilized whole transcriptome analysis and immunofluorescence-based protein expression as part of spatial TME profiling.
 Methods:
 We identified 13 chemo-naïve PTs and 3 breast-to-liver metastases. Formalin-fixed paraffin-embedded (FFPE) samples were used for hematoxylin and eosin (H&E) staining and spatial transcriptomics (ST) profiling with the 10X Genomics Visium CytAssist and Visium HD platforms. The H&E slides were manually annotated in collaboration with a board-certified pathologist, and ST data were deconvoluted using breast-specific scRNA-sequenced references and canonical marker labeling.
 Results:
 Our analysis of the 13 PTs and 3 breast-to-liver metastases revealed significant heterogeneity in tumor clusters, immune cell infiltration, gene set enrichments, and gene signatures between primary and visceral TMEs. Unsupervised clustering of hormone receptor-positive (HR+) HER2-negative (HER2) samples demonstrated considerable intratumoral and intertumoral variability, with differences in the expression of tumor markers (EPCAM, ER, PR, ERBB2, KRT14, KRT18), hypoxia-related signatures (HIF1a), and pro-angiogenic drivers (VEGF). Breast-to-liver metastases showed higher expression of pathways involved in proliferation, angiogenesis, and anti-apoptosis compared to primary tumors. Hepatic metastases were enriched for HALLMARK gene sets related to proliferation and metabolic pathways, including PI3K/AKT/mTOR, mTORC1 signaling, E2F targets, MYC targets, G2M checkpoint, mitotic spindle, glycolysis, oxidative phosphorylation, and fatty acid metabolism when compared to primary tumors. In contrast, primary tumors were enriched for immune-related pathways (TNF-α, interferon-γ, inflammatory response, IL-6 JAK/STAT3), epithelial-mesenchymal transition (EMT), myogenesis, and angiogenesis gene signatures, which are consistent with their metastatic potential. Cellular profiling of liver lesions revealed higher myeloid suppressor cells (TAMs and MDSCs): CD8
 +
 ratio with shorter cell-cell distances relative to PTs suggesting a key role in immune evasion, tumor progression, and therapeutic resistance.
 Conclusions:
 ST data from primary and metastatic breast cancer samples revealed notable tumor heterogeneity, characterized by highly active metabolic and proliferative pathways, along with a higher abundance of myeloid cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/389d6dee563fb2269c25d3de12654b70850dddf7" target='_blank'>
              Spatial transcriptomics and functional mapping of the suppressive TME in primary breast tumor and breast-to-liver metastases.
              </a>
            </td>
          <td>
            Batul Al-zubeidy, Edgar Gonzalez, A. Baugh, Dominic Zavala, Cheol Park, Matthew Jacobo, Marqus Dela Cruz, Michelle Li, Sabrina Carrel, Elexa Rallos, Michael Press, Evanthia T. Roussos Torres
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Single-cell RNA sequencing (scRNA-seq) provides enhanced insights into the cellular composition of human tissues. In cancer, normal proportions of single cells are aberrantly dysregulated, and thus targeting subpopulations of cells within a tumor might be more effective than targeting all cells of a tumor. Here we present Single Cells to Drugs And Targets (sc2DAT), a unique workflow for the analysis of bulk and single cell RNA-seq datasets collected from tumors of the Clinical Proteomic Tumor Analysis Consortium (CPTAC). sc2DAT can take in as input single cell or bulk RNA profiles from one or two conditions. When processing scRNA-seq samples, sc2DAT first clusters the single cell vectors of expression to identify cell type subpopulations. It then bulkifies the vectors of expression in each cluster of single cells. For bulk RNA-seq, sc2DAT applies deconvolution with BayesPrism to infer cell-type-specific expression profiles for each identified subpopulation within a tumor. The sc2DAT platform currently has over 20 precomputed single-cell reference datasets derived from human and mouse studies. The expression vectors in each cluster of single cells are analyzed to discover genes that give rise to cell-surface proteins that are highly expressed in the tumor’s subpopulation, but lowly expressed across all human healthy cell types and tissues. These identified cell-surface proteins can become immunotherapy targets for selectively elimination of subpopulation of single cells within a tumor. In addition, sc2DAT integrates the LINCS L1000 perturbational gene expression dataset to predict drugs and preclinical compounds that could induce a healthy phenotype in subpopulations of single cells within a tumor. Once datasets are uploaded to sc2DAT, the results are presented in a comprehensive report with an abstract, introduction, methods, results, conclusions, and references sections. The reports have many figures and tables, and it can be exported and shared in several formats including a PDF file. We applied sc2DAT to several CPTAC3 and CPTAC4 datasets to identify cell-type-specific subpopulation targets and reverser compounds, uncovering many potential personalized therapeutic strategies for immunotherapy and small-molecule interventions. sc2DAT is available from: https://sc2dat.maayanlab.cloud.


 Giacomo B. Marino, Nasheath Ahmed, Daniel J. Clarke, Avi Ma'ayan. sc2DAT: workflow for targeting tumor subpopulations of single cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6290.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c5186055b7098bddfe8ece68d797911822c432e" target='_blank'>
              Abstract 6290: sc2DAT: workflow for targeting tumor subpopulations of single cells
              </a>
            </td>
          <td>
            Giacomo B. Marino, Nasheath Ahmed, Daniel J. B. Clarke, Avi Ma’ayan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Metastatic bladder cancer (mBLCA) exhibits significant heterogeneity at the genomic, transcriptomic, molecular, and epigenetic levels, contributing to challenges in diagnosis, prognosis, and treatment. While 75% of mBLCA cases are urothelial carcinomas, 25% show divergent differentiation with variant histological features. This work characterizes intra-patient and inter-patient heterogeneity through comprehensive genomic and transcriptomic analyses of a rapid autopsy cohort of 20 patients with multiple tumors, matched normal tissue, and cell-free DNA (cfDNA). Our findings provide critical insights into tumor heterogeneity, clonal evolution, metastatic seeding, and the tumor microenvironment.



 Using bioinformatics tools, we reconstructed clonal evolution trees from whole-genome sequencing (WGS) analysis of tumors and inferred migration patterns to determine metastatic seeding. COSMIC mutational signatures were examined to correlate histological subtypes with clinical history. Single-nucleus RNA sequencing (snRNA-seq) revealed cell types and transcriptional heterogeneity using the human protein atlas and known marker annotations. To investigate alterations and clonality in tumors and their matched cfDNA, we used custom methodologies to evaluate mutations, CNAs, SVs, and rearrangements.



 The clonal evolution trees and bulk RNA-seq highlight cisplatin resistance in the plasmacytoid urothelial carcinoma subtype, driven by upregulation of the DNA damage response pathway. Significant mutational heterogeneity (∼20-30% subclonal) emerges in later stages of tumor evolution, potentially contributing to therapy resistance. Additionally, a substantial proportion (60-70%) of structural variants arise later in tumor progression, particularly in driver genes. snRNA-seq revealed distinct clusters of epithelial cells, macrophages, fibroblasts, and other cell types. cfDNA analysis captured over 80% of subclonal deleterious tumor mutations in BLCA driver genes. Nucleosome profiling of cfDNA helps distinguish mBLCA from healthy samples and identify key transcription factors to differentiate between variants.



 This comprehensive study of mBLCA, leveraging a rapid autopsy dataset, uncovers critical insights into variant histologies across multiple omics layers. Integrating cfDNA analysis captures intra-patient and inter-patient tumor heterogeneity, providing a holistic view of clonal dynamics. By elucidating the patterns of clonal evolution, this work advances our understanding of tumor progression and establishes a foundation for precision therapies targeting adaptive and evolving clones.



 Pushpa Itagi, Samantha Schuster, Sonali Arora, Patricia Galipeau, Thomas Persse, Funda Vakar-Lopez, John K. Lee, Petros Grivas, Robert Montgomery, Jonathan Wright, Hung-Ming Lam, Andrew Hsieh, Gavin Ha. Deciphering heterogeneity and tumor evolution in bladder cancer using multi-omics and liquid biopsies from rapid autopsies [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1241.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f3af6173f4bd387223a85f81701f7e3b9e33817" target='_blank'>
              Abstract 1241: Deciphering heterogeneity and tumor evolution in bladder cancer using multi-omics and liquid biopsies from rapid autopsies
              </a>
            </td>
          <td>
            P. Itagi, Samantha L. Schuster, Sonali Arora, Patricia Galipeau, Thomas W Persse, F. Vakar‐López, John K. Lee, P. Grivas, R. Montgomery, Jonathan L. Wright, Hung-Ming Lam, Andrew C. Hsieh, Gavin Ha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Purpose The aim of this study is to explore the pathological niche of cancer metastasis and the site-specific interactions between tumor cells and the microenvironment, understand the mechanisms driving metastasis progression and identify potential therapeutic targets. Methods Data from four lung cancer metastasis datasets (GSE123902, GSE131907, GSE148071, and GSE186344) were downloaded and subjected to stringent quality control and filtering. Cell types were identified using canonical markers, and pseudotime trajectory analysis was performed to evaluate cell differentiation. Functional and pathway enrichment analyses, including ssGSEA and GO/KEGG, were conducted. CellphoneDB was used to analyze intercellular communication, ranking receptor-ligand interactions based on communication strength. Results Eleven cell types were identified after quality control, revealing significant heterogeneity and site-specific functionality in lung cancer metastases. CTLs showed notable activity in antigen presentation and T-cell differentiation pathways, with DNAJB1⁺ CTLs playing a dominant role in cytotoxicity and immune regulation. B cells, myeloid cells, and CAFs were involved in immune modulation, defense, and matrix remodeling through specific signaling pathways. Tumor cell subclusters drove proliferation, migration, and immune evasion via immune-regulatory, Hippo, and TGF-beta pathways. No overlapping pathways were observed across metastatic sites. Cell communication analysis identified PPIA-BSG and APP-CD74 as key axes in brain and lymph node metastases, while FN1-Integrin and CTLA4-CD86 dominated in bone and adrenal metastases, respectively. Conclusions In summary, this study highlights the functional heterogeneity and site-specific interactions of cells in lung cancer metastases, providing insights into the mechanisms shaping metastatic niches and potential therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02269-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c4f954c571841166277c9ca7ec19c26848339cb" target='_blank'>
              Single-cell profiling uncovers the intricate pathological niche diversity in brain, lymph node, bone, and adrenal metastases of lung cancer
              </a>
            </td>
          <td>
            Le Liu, Yuan Zhou, Zhenjun Ye, Zhiyong Chen, Benchao Yuan, Liyi Guo, Haiyan Zhang, Yuanyuan Xu
          </td>
          <td>2025-04-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac2fa84e64df92a9b47c54f45baf0d93294fea97" target='_blank'>
              Substrate heterogeneity promotes cancer cell dissemination
              </a>
            </td>
          <td>
            Zuzana Dunajova, Saren Tasciyan, Juraj Májek, J. Merrin, E. Sahai, M. Sixt, É. Hannezo
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="


 The limited efficacy of surgical resection and high rate of tumor recurrence in glioblastoma underscores the urgency of better understanding tumor invasion in the central nervous system. High-throughput sequencing technologies, such as single-cell RNA-sequencing (scRNA-seq) and spatial genomics, efficiently identify gene expression states of rare invading cells and microenvironmental interactions. We previously described cell state heterogeneity in glioblastoma by scRNA-seq and uncovered the contribution of immune cells to the heterogeneity. Yet it is unclear whether the cell states observed in clinical specimens are implicated in invading glioblastoma cells in the brain parenchyma. We hypothesized that the distance from the primary tumor site and the local microenvironment of invading cells might affect the invasion phenotype.



 To test this hypothesis, we developed an in vivo approach with mixtures of human glioblastoma spheroid lines for mouse intracranial xenografts. We validated this multiplex method with 20 patient-derived models and profiled, by scRNA-seq, in vivo models capable of infiltrating the brain parenchyma. We then used spatial transcriptomics to explore invasion routes and distribution of the invasive programs across the brain.



 By comparing cells in the contralateral invasive site to the primary tumor, we identified a set of genes upregulated in invading glioblastoma cells, significantly enriched with NPC-like and OPC-like programs we identified previously, as well as genes not associated with the predefined states. Next, we analyzed the program expressed by cells undergoing invasion along blood vessels; the up-regulated program was enriched with AC-like and OPC-like programs. As observed with contralateral invasion, the program seen in blood vessel invasion was distinct from the previously defined states. The ST analysis highlighted this distinction and detailed the characteristics of each program. The contralateral invasion signature was associated with neuronal and myeloid-specific regions in the contralateral hemisphere. In contrast, the blood vessel invasion signature was localized to vascular regions at the periphery of the primary tumor.



 Our study found that an abundance of OPC-like cells was linked to more efficient invasion into the opposite side of the brain, suggesting that these cells might play a crucial role in distal invasion. In addition, we observed that models with a MES-like expression profile exhibit local perivascular invasion. Our comprehensive approach, coupled with high-throughput transcriptome profiling, provides a detailed blueprint of invasion potential and the functional transcriptional programs that drive GBM invasion along different anatomical routes, offering new insights for potential therapeutic strategies against glioblastoma.



 Toshiro Hara. Dissecting in situ states of invading glioblastoma cells using single-cell and spatial transcriptomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6622.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1870d444cf3f0fec43c774decd3ef3d91c4b6138" target='_blank'>
              Abstract 6622: Dissecting in situ states of invading glioblastoma cells using single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Toshiro Hara
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e20030


 Background:
 Tumoral heterogeneity has been widely acknowledged to influence clinical outcome of lung adenocarcinoma (LUAD). This study investigated spatial gene profiling of different pathological subtypes of early-stage invasive LUAD with Stereo-seq transcriptomics technique. The tumor microenvironment (TME) was decoded with cellular and molecular events at perspectives of tumoral activities and immune activities.
 Methods:
 We collected spatial RNA profiles of nine Stereo-seq chips (captured area of 1 x1 cm) from 4 invasive LUAD patients. Pathologist identified histological subtypes including lepidic adenocarcinoma (LP), acinar adenocarcinoma (AC) and other invasive adenocarcinoma like papillary adenocarcinoma (IA) on adjacent H&E images. Spatial data matrix was obtained based on pathological annotation accordingly. Spatially resolved gene expression of each subtypes was obtain based on pathological annotation to conduct high resolution analyses. TME functional units were calculated by analysis of cellular neighborhoods (CNs) based on cell compositions within resolution of bin 100 (window of 50um x 50um). Sub-populations of tumoral cells were stratified by molecular functions (GSEA and GSVA score) and stemness (CytoTRACE).
 Results:
 All pathological subtypes shared the identical 10 heterogeneous CNs and 5 tumoral clusters (TM) with various propotions. Tumor cluster 0 (TM0) and TM3 were identified as the malignant population with significant high stemness and significant high activities of MTORC1 signaling, TGF-BETA signaling, PI3K-ATK signaling and hypoxia. AC and IA were mainly composed of TM0 and TM3. Compared to TM0, TM3 had higher EMT activity. We further identified two immune-related CNs (CN4 and CN8) with significant high GSVA score of Interferon-Alpha response, Interferon-Gamma response and Inflammatory response. Spatial association showed that the lymphoid-cells-dominated CN4 was enriched in LP and IA and nearly absent in AC. Meanwhile, the myeloid-cells-dominated CN8 was enriched in AC. GSEA score showed that immune activity of CN4 was more intense in IA when compared to LP. Notably, TM0 mainly present in CN8, and TM3 mainly present in CN4. Therefore, the LP was speculated to reach an inert and statical status. TME of AC was featured as colocalization of myeloid cells and tumor cells with high stemness. TME of IA was featured as colocalization of lymphoid cells and tumor cells with high EMT activity.
 Conclusions:
 This is the first study to delineate the homogeneity of tumor cells and heterogeneity of immune cells among different pathological types of NSCLC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344f597f0ff9f98416e718719f81b290a2438fe0" target='_blank'>
              Decoding spatial transcriptomics-based heterogeneity of TME in early-stage invasive pulmonary adenocarcinoma.
              </a>
            </td>
          <td>
            Xin Liu, Jingyi Tian, Z. Shang, C. Xiang, Fei Teng, Jiajun Zhang, Yuchen Han
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Next generation cancer models, including organoids, neurospheres and cell lines, are significantly advancing our ability to study individual human tumors in vitro and to develop novel cancer therapeutics in settings that resemble in vivo conditions. However, recent questions have emerged as to the degree to which in vitro models may faithfully represent intra- and inter-patient cell state heterogeneity at the transcriptional level of single cells. The Human Cancer Models Initiative (HCMI), a global collaboration involving the NCI (NIH), Cancer Research UK, the Wellcome Sanger Institute and the Hubrecht Organoid Technology foundation, has generated a large repertoire of 665 patient-derived models of human cancers, including organoids, neurospheres and conditionally reprogrammed cells, with matching parental tumor and clinical annotations. We conducted a tumor-model fidelity analysis using bulk profiles from Whole Genome Sequencing (WGS) and RNA-seq and confirmed strong cell-state concordance between each model and its parental tumor across most HCMI tumor-model pairs, thus showcasing models’ ability to retain key genetic, transcriptional and epigenetic attributes of their parental tumors. However, our analysis also identified divergent features in a subset (<8%) of models, potentially resulting in a loss of fidelity and challenging their translational use. To identify whether this discordance might be driven by ex vivo adaptations or selective evolutionary pressures from in vitro culture, we profiled a representative subset (n=13 pairs) of glioblastoma (GBM, n=7) and pancreatic cancer (PAAD, n=6) tumor-model pairs using single-nucleus RNA-seq (snRNA-seq). Analysis of 132,593 nuclei (tumors: 66,054; models: 66,539) identified outlier GBM models undergoing proneural-to-mesenchymal (PTM) transition and highlighted serum culture media as a major associated factor of this process. Additionally, we found that divergences in selected PAAD (72,688 nuclei: tumors: 30,254; models: 42,434) pairs were linked to the complete loss of the rich tumor microenvironment cells in the original tumors and the in vitro expansion of select malignant subpopulations, as identified by network-based analysis of VIPER-inferred protein activity profiles. The observed divergences suggest that in vitro culture conditions can occasionally drive cellular reprogramming and alter phenotypic fidelity, resulting in varying degrees of state-specific transitions. However, the majority of the models (even those classified as outliers) are preserved for state-frequency in cellular subpopulations between tumors and models, and most if not all of the cellular states found in the parental tumor, thus faithfully representing intra-tumor heterogeneity. Taken together, these findings highlight the translational potential of the HCMI model repository and its translational relevance as an invaluable tool to support precision oncology.


 Dina ElHarouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, Toshinori Hinoue, Sean A. Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W. Kyi, Julyann Perez-Mayoral, Megan J. Stine, Eva Tonsing-Carter, James M. Clinton, The HCMI Network, Peter W. Laird, Calvin J. Kuo, Olivier Elemento, David L. Spector, Andrew D. Cherniack, Kyle Elrott, Martin L. Ferguson, Rameen Beroukhim, Katherine A. Hoadley, Nicolas Robine, Andrew McPherson, Matthew J. Garnett, David A. Tuveson, Andrea Califano, Paul T. Spellman, Keith L. Ligon, Daniela S. Gerhard, Louis M. Staudt, Jesse Boehm. Single cell transcriptional dynamics in the HCMI cancer model collection [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6358.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c16f756ddb575cdf26b92c703a6e6c80ae8425f" target='_blank'>
              Abstract 6358: Single cell transcriptional dynamics in the HCMI cancer model collection
              </a>
            </td>
          <td>
            Dina Elharouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, T. Hinoue, Sean A. Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W. Kyi, Julyann Pérez-Mayoral, Megan J. Stine, Eva Y. Tonsing-Carter, James M. Clinton, The Hcmi Network, Peter W. Laird, Calvin J. Kuo, O. Elemento, David L. Spector, Andrew D Cherniack, Kyle Elrott, M. Ferguson, R. Beroukhim, K. Hoadley, N. Robine, Andrew McPherson, M. Garnett, D. Tuveson, Andrea Califano, Paul T. Spellman, Keith L. Ligon, Daniela S. Gerhard, Louis M. Staudt, Jesse S. Boehm
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>149</td>
        </tr>

        <tr id="We are pleased to present this special section of Small Methods , which highlights the rapidly advancing ﬁelds of single-cell and spatial transcriptomics. Single-cell transcriptomics and spatial transcriptomics have emerged as transformative tools for high-resolution proﬁling of gene expression. Single-cell approaches reveal cellular diversity at unprecedented resolution, while spatial transcriptomics preserves the spatial context of gene activity, enabling precise mapping of tissue architecture. Together, these technologies provide complementary insights into biological systems, uncovering cellular heterogeneity, dynamic interactions, and spatially driven molecular processes across diverse ﬁelds including developmental biology, cancer biology, immunology, and neuroscience. This collection includes 3 reviews and 10 research articles from prominent scientists with their valuable insights. In this special section, we explore the transformative potential of spatial transcriptomics data analysis. Gao et al. (smtd. 2401451) introduce PASSAGE, a deep learning framework for identifying phenotype-associated signatures across heterogeneous spatial slices. Yang et al. (smtd.2400975) propose a novel cell segmentation method UCS, optimized for large-scale subcellular spatial transcriptomics data. Ishaque et al. (smtd. 2401123) present Sainsc, a cell-segmentation-free approach for transcriptome-wide, nanoscale-resolution spatial data. Yuan et al. (smtd.2401199) present a kernel-based strategy to model spatially continuous variations of the tissue microenvironment us-ing a new kernel-based strategyFei et al. (smtd.2401056) intro-S">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9aa8e9995463c05e1c6e24838fae6f8d6bcbd7" target='_blank'>
              Unraveling Tissue Complexity Through Single‐Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shiquan Sun, Chaoyong Yang, Lulu Shang, Rong Fan
          </td>
          <td>2025-05-01</td>
          <td>Small Methods</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Syngeneic mouse models have a functionally intact immune system, and therefore, serve as a crucial tool for evaluation of the immune response to tumors and pre-clinical evaluation of immunotherapeutic strategies. However, the immune response varies between different syngeneic models. Identification and selection of appropriate models for pre-clinical evaluation is challenging. To understand the dynamic interplay between the tumor and immune system, we implemented multiplexed spatial proteomics across various syngeneic tumor models. Spatial mapping of multiple tumor and immune markers in untreated tumor provides an overview of the tumor immune architecture. This information can be applied to identify optimal models for preclinical immunotherapy approaches, to understand the response to therapies at the preclinical stage, and to identify potential biomarkers for clinical evaluation. The Cell DIVE Multiplexed Imaging Solution, in combination with IHC-validated antibody conjugates from Cell Signaling Technology (CST), enables profiling of immune cell populations in the context of human and mouse tumor tissue. Here, we demonstrate multiplexed Cell DIVE imaging of syngeneic mouse tumor models using a novel CST® panel of mouse-reactive antibodies. We apply segmentation and clustering analysis to identify spatially co-localized populations of cells and discuss the complexities of immune cells and their interactions. Cell type specific markers combined with multiplexed tissue imaging and spatial mapping is an approach that enables a better understanding of the anti-tumor immune response, which can be applied to address challenges associated with developing immune-based therapies. The study highlights the importance of simultaneously observing multiple immune cell types and tumor characteristics in spatial context.


 Arindam Bose, Emily Quann Alonzo, Sophie Struble, Jeremy Fisher, Richard A. Heil-Chapdelaine, Natasha F. Diaz Granados, Vasundhara Agrawal. Spatial proteomics and tumor-immune phenotyping identifies a diverse immune landscape in syngeneic murine tumor models [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1896.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d379694e8835fc398019d9f1f787e108dd62b53d" target='_blank'>
              Abstract 1896: Spatial proteomics and tumor-immune phenotyping identifies a diverse immune landscape in syngeneic murine tumor models
              </a>
            </td>
          <td>
            Arindam Bose, E. Alonzo, Sophie Struble, Jeremy Fisher, R. Heil-Chapdelaine, Natasha F. Diaz Granados, Vasundhara Agrawal
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) presents substantial heterogeneity and unpredictability in its progression. Despite therapeutic advancements, mortality from advanced PCa remains a significant challenge. Understanding the intercellular communication within the tumor microenvironment (TME) is critical for uncovering mechanisms driving tumorigenesis and identifying novel therapeutic targets. Methods We employed an integrative approach combining bulk RNA sequencing, single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics to investigate interactions between FAP+ fibroblasts and tumor-associated macrophages in PCa. Key findings were validated using immunohistochemical and immunofluorescence staining techniques. Results Analysis of 23,519 scRNA-seq data from 23 prostate samples revealed a pronounced accumulation of FAP+ fibroblasts in tumor tissues. Spatial transcriptomics and bulk RNA sequencing demonstrated strong associations between FAP+ fibroblasts and SPP1+ macrophages. Notably, tumor-specific intercellular signaling pathways, such as CSF1/CSF1R and CXCL/ACKR1, were identified, highlighting their potential role in fostering an immunosuppressive TME. Conclusion Our findings unveil a distinct pattern of crosstalk between FAP+ fibroblasts and SPP1+ macrophages in PCa, shedding light on potential therapeutic targets for advanced PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d000b5dbafb928d866718b2a6f1d1452be0aec5" target='_blank'>
              Intercellular communication between FAP+ fibroblasts and SPP1+ macrophages in prostate cancer via multi-omics
              </a>
            </td>
          <td>
            Tingting Wu, Xinyu Li, Fei Zheng, Hanchao Liu, Yang Yu
          </td>
          <td>2025-05-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9faa1273d62a3ded4dd4ed42a59e3c9f4c050ab5" target='_blank'>
              Spatial omics technology potentially promotes the progress of tumor immunotherapy.
              </a>
            </td>
          <td>
            Zhen Lan, Yuanyuan Yang, Lingling Li, Chaoguan Wang, Zhenqiang Sun, Qiming Wang, Yang Liu
          </td>
          <td>2025-06-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common malignancies worldwide, and its complex pathogenesis and significant tumor cell heterogeneity remain major challenges. With the rapid development of single-cell sequencing technology, we can now delve deeper into the cellular composition and dynamic changes within the tumor microenvironment, revealing cellular interactions and their potential roles in tumorigenesis. Method In this study, we systematically analyzed comprehensive single-cell RNA sequencing data from 25 colorectal cancer and 10 adjacent normal tissue samples. We explored the characteristics and biological significance of tumor cell subpopulations, performed quality control, dimensionality reduction, and cell type identification, and further investigated epithelial cell copy number variations, cell communication, and pseudotime analysis. Subsequently, Boruta feature selection algorithm was combined to identify prognosis related genes. The expression patterns, clinical significance and biological effects of PRSS22 were validated in vitro. Results Our analysis found an epithelial cell subcluster with high expression of PRSS22 exhibited high proliferation and migration abilities, and it was also associated with the dysregulated immune microenvironment. After further experimental verification, we proved the high expression patterns and clinical significance of PRSS22. Downregulation of PRSS22 in CRC cells resulted in a reduction of proliferation, migration and invasion. Conclusion Our study has identified a cell subcluster that is closely linked to progression, immune dysregulation and prognosis in CRC, and we have also identified PRSS22 as its hub gene that has great potential to become a new immunotherapeutic targets target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683616b1a667ef7717c017121e0ce36b95c50974" target='_blank'>
              Spatial multi-omics profiling of breast cancer oligo-recurrent lung metastasis.
              </a>
            </td>
          <td>
            Yang Gao, Bin Li, Yuzhi Jin, Jinlin Cheng, Weihong Tian, Lixiong Ying, Libing Hong, Shan Xin, Bo Lin, Chuan Liu, Xuqi Sun, Jun Zhang, Haibo Zhang, Jindong Xie, X. Deng, Xiaomeng Dai, Lulu Liu, Yi Zheng, Peng Zhao, Guangchuang Yu, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-04-15</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is an aggressive disease with poor overall survival rates that have not substantially changed in decades. A key feature underlying the lack of therapeutic success in GBM is its inherent morphological and phenotypic diversity. To better understand intra-tumor heterogeneity, single cell sequencing studies have delineated core transcriptional states to characterize GBM cell phenotypes. This study investigates the role of hypoxia in driving the establishment and transformation of GBM cell state. To characterize the spatial organization of cell states relative to hypoxia, spatial transcriptomics (ST) was performed on samples from GBM patients administered pimonidazole prior to surgery as part of a clinical trial. By overlaying serial pimonidazole-stained sections with ST, we constructed a hypoxia gene signature to identify hypoxic regions in ST. Our ST data was integrated with public datasets to create the largest global GBM dataset. In this dataset, we identified novel, ST-specific states that represent recurring cellular phenotypes. We used geographically weighted models to assess hypoxia’s spatial colocalization with previously defined cell states and our independently derived states. Finally, we investigated underlying mechanisms by which hypoxia may influence cell state using patient derived glioma stem cells (GSCs) cultured under hypoxic conditions. Using ATAC-seq, ChIP-seq, and bulk RNA-seq, we identified specific genes and chromatin regions differentially regulated under hypoxia. We find that hypoxia demonstrates remarkable spatial correlation with mesenchymal-like cell states, while states reminiscent of neural and glial lineages are notably absent in hypoxic regions. We also identify a leukocyte migratory phenotype strongly spatially correlated with hypoxia, which may indicate immune infiltration in these regions. Epigenetic profiling in GSCs identified specific genomic regions epigenetically regulated by hypoxia. Chromatin regions open under hypoxia with corresponding increased expression are enriched for known hypoxia-associated transcription factors. Conversely, chromatin regions closed and with corresponding decreased expression are enriched for neurodevelopment and differentiation transcription factors. These epigenetically regulated genes are strongly spatially correlated with hypoxic regions in patient ST data and are prognostic as a signature in data from TCGA. Together, our work demonstrates that hypoxia drives remodeling of the transcriptional and epigenetic landscape in GBM. In patients, hypoxia and genes epigenetically regulated under hypoxic conditions associate with specific GBM cell states, suggesting that hypoxia-driven epigenetic remodeling may contribute to these state phenotypes and, consequently, to malignant phenotypes in tumors.


 Phoebe Lombard, Mark Zaidi, Ronald Wu, Sheila Mansouri, Gelareh Zadeh, Bradly G. Wouters. Hypoxia-driven spatial dynamics of epigenetic and transcriptional cell states in glioblastoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 752
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef350217417b22617b2248d742f15859d0838a2a" target='_blank'>
              Abstract 752: Hypoxia-driven spatial dynamics of epigenetic and transcriptional cell states in glioblastoma
              </a>
            </td>
          <td>
            Phoebe Lombard, M. Zaidi, Ronald Wu, S. Mansouri, Gelareh Zadeh, B. Wouters
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Tumor microenvironment (TME) heterogeneity plays a crucial role in cancer progression and therapeutic response. Advanced multi-omics spatial and single-cell technologies offer unprecedented resolution for dissecting TME complexity. However, the application of these technologies to formalin-fixed paraffin-embedded (FFPE) samples remains underexplored. This study aims to demonstrate the capabilities of these technologies in analyzing FFPE samples, as well as to benchmark and cross-validate different platforms and assays, and deepen our understanding of TME heterogeneity at the single-cell level and its spatial context.



 We performed following assays including 1) High-plex multiple IHC platform: PhenoCycler Fusion (PCF) protein-level spatial analysis (n=10), 2) FFPE Single-Cell RNA Sequencing (FFPE scRNA-seq) (n=4) and 3) Bulk RNAseq RNA-level analysis with FFPE NSCLC samples.



 Firstly, to evaluate the PCF marker expression pattern, cell density of PanCK+PD-L1+ cell ratio (%) was compared to PD-L1 IHC TC score, which showed a high correlation. Further, By image co-registration between H&E with ML-based tumor/stromal classification and PCF PanCK-PD-L1 staining, PD-L1+ cell density in tumor cells specifically can be precisely calculated. To further evaluate the marker gene expression consistency, high gene expression concordance was observed across PhenoCycler Fusion (PCF), formalin-fixed paraffin-embedded (FFPE) single-cell RNA sequencing (scRNA-seq), and bulk RNA sequencing (bulk RNA-seq) data. This high concordance (average pearson r=0.84) validates the consistency and reliability of our multi-omics approach, ensuring that the data obtained from different sample types are comparable and robust. A diverse TME was identified, highlighting the complexity and heterogeneity of TME components. This diversity was evident in the range of immune cell subsets and cancer-associated stromal cells detected, underscoring the importance of comprehensive TME analysis even in small cohorts. In addition, FFPE scRNAseq enabled a deeper profiling at transcriptome level with a median gene recovery rate at 1130 genes per cell, which contribute to a more detailed cell lineage identification including small populations like Mast cells and neutrophils.



 Overall, the study demonstrates the effectiveness and reliability of PCF in evaluating PD-L1 expression and the TME. The high concordance across multi-omics approaches and the detailed transcriptome profiling provided by FFPE scRNAseq highlight the robustness of these latest methodology on FFPE samples. Despite the limited sample size, the diverse TME identified emphasizes the need for comprehensive and personalized TME analysis. These findings provide a foundation for further investigations into the TME.



 Qiming Zhou, Jiadong Zhou, Beixi Wang, Jingwen Shi, Han Yan, Yun Zhang, Pei Zhang, Zhirong Shen. Integrative spatial and single-cell analysis elucidates tumor microenvironment heterogeneity in NSCLC [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2090.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9cb5b1581890bc09659329e338f756aca67f340" target='_blank'>
              Abstract 2090: Integrative spatial and single-cell analysis elucidates tumor microenvironment heterogeneity in NSCLC
              </a>
            </td>
          <td>
            Qiming Zhou, Jiadong Zhou, Beixi Wang, Jingwen Shi, Han Yan, Yun Zhang, Peilin Zhang, Zhirong Shen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding the spatial distribution of cancer cells and how they interact with other cell types in the tumor microenvironment (TME) is crucial to elucidate the mechanisms of cancer development and progression. The identification of cell types in the TME relies on the distinct expression profiles of different cell types. Compared to sequencing-based spatial transcriptomics analysis, multiplex nucleic acid in-situ hybridization (ISH) at a single-molecule level is more focused and cost-effective. Based on ISH technology, RNAscope developed novel approaches to detect multiple gene expression profiles on tissue sections at a single-molecule level. It is highly sensitive and specific and can simultaneously quantify gene expression at sub-cellular levels. Although the technology has been applied to study spatial gene expression in various cancer types, few have applied it to elucidate the association between different markers in different cell compartments within the TME.This protocol describes an image analysis pipeline for multiplex fluorescent signals detected by RNAscope technology. It was developed based on a previous CellProfiler pipeline, facilitating spatial expression analyses across different cell compartments through a flexible, user-friendly interface for non-computational biologists. We describe specific steps to analyze the expression and spatial arrangement of four endogenous housekeeping genes as quality controls (QCs): POLR2A, PPIB, HPRT1, and UBC. We further apply these pipelines to analyze target genes in breast cancer samples, comparing those with and without skin metastasis. The analyses were demonstrated in one sample each using formalin-fixed paraffin-embedded tissue (FFPET) and fresh-frozen tissue (FFT) sections with the ISH steps followed the manufacturer’s protocol.


 Meng-Shin Shiao, Natini Jinawath, Katherine Copeland, Ariestya Indah Permata Sari. A CellProfiler pipeline for RNAscope analyzing multiple markers across adjacent tissue sections spatially [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6302.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16fe30c659d6f8c68794471b6bb6c1c23bb6c9f4" target='_blank'>
              Abstract 6302: A CellProfiler pipeline for RNAscope analyzing multiple markers across adjacent tissue sections spatially
              </a>
            </td>
          <td>
            M. Shiao, N. Jinawath, Katherine Copeland, Ariestya Indah Permata Sari
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Colorectal cancer, a significant cause of cancer-related mortality, often exhibits drug resistance, highlighting the need for improved tumor models to advance personalized drug testing and precision therapy. We generated organoids from primary colorectal cancer cells cultured through the conditional reprogramming technique, establishing a framework to perform short-term drug testing studies on patient-derived cells. To model interactions with stromal cells in the tumor microenvironment, we combined cancer cell organoids with carcinoma-associated fibroblasts (CAFs), a cell type implicated in disease progression and drug resistance. Our organotypic models revealed that CAFs promote cancer cell proliferation and stemness primarily through HGF-MET paracrine signaling and activation of Cyclin-Dependent Kinases (CDKs). Disrupting these tumor-stromal interactions reduced organoid size while limiting oncogenic signals and cancer stemness. Leveraging this tumor model, we identified effective drug combinations targeting colorectal cancer cells and their tumorigenic activities. Our study highlights a path to incorporate patient-derived cells and tumor-stromal interactions into a drug testing workflow that could identify effective therapies for individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dfbdf3e1bc24e3e89e9ac8874b0428cc6cc6092" target='_blank'>
              Targeting CAFs-Mediated Stromal Signaling in a Patient-Derived Organotypic Colorectal Tumor Model.
              </a>
            </td>
          <td>
            Astha Lamichhane, Prasiddha Guragain, Jacob Heiss, Pouria Rafsanjani Nejad, Anju Rana Magar, Nicholas Ciavattone, Seema Agarwal, G. Luker, Hossein Tavana
          </td>
          <td>2025-04-14</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Human diseases affect various cell types within the local niche, but the multicellular responses of human diseases have not been collectively assessed across organs. Here, we categorize multicellular responses of human diseases based on a remapped human cell atlas, incorporating 12 million single-cell transcriptome profiles acquired from an automated data collection pipeline. Additional curation expands the collection up to 46 million cells, covering 160 disease conditions across 49 distinct organ identities. From the re-aligned atlas of single-cell transcriptome data, we constructed a hierarchical cell classification framework comprising 32 major cell types and 165 cell subtype annotations. Systematic profiling of disease and organ association patterns of cellular subtype reveals archetypes of human diseases repeatedly found across organs, including a multicellular phenotype highly specific to the tumor microenvironment. The universal cell atlas could serve as a reference for unbiased deconvolution of spatial transcriptome datasets, revealing spatial distribution patterns associated with immunotherapy responses. Collectively, our study provides foundational resources for cross-tissue cell type annotation, presents a comprehensive multicellular overview of disease-associated responses across human tissues, and highlights multicellular responses in tumor microenvironments.


 Seongryong Kim, Sungmin Cheong, Suho Lee, Yeju Kim, Jong-Eun Park. Cross-tissue atlas of human disease identifies tumor-specific components in tumor microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6265.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf8353686e8bd9d104e11c8e9f95fa9e551e49d0" target='_blank'>
              Abstract 6265: Cross-tissue atlas of human disease identifies tumor-specific components in tumor microenvironment
              </a>
            </td>
          <td>
            Seong-Kun Kim, Sungmin Cheong, Suho Lee, Yeju Kim, Jong-Eun Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 11508


 Background:
 Leiomyosarcoma (LMS) is a smooth muscle-derived tumor with significant heterogeneity and limited treatment options for recurrent/metastatic disease. A lack of targetable driver mutations and prognostic and predictive biomarkers have hampered the care of patients with LMS. There is a great need to better understand the biology of LMS and develop novel therapeutics. Advances in single cell RNA sequencing (scRNA-seq) have allowed for better understanding of intratumoral heterogeneity in diverse cancer subtypes. However, tissue dissociation during this process leads to loss of spatial context. Spatial gene expression analysis builds upon scRNA-seq and has the potential to yield information about tissue organization, cell-cell interactions, niches, and cell states. To date there have been limited application of spatial transcriptomics to sarcoma.
 Methods:
 We performed single nucleus multiome (snRNA-seq and snATAC-seq) on a cohort of 16 primary, untreated LMS samples including 12 soft tissue (STLMS) and 4 uterine (ULMS) tumors. We then designed a custom 480-gene panel using the differentially expressed genes from clusters identified in snRNA-seq data to be able to identify spatial relationships between these clusters and to assess these clusters on a larger scale. We utilized the 10x Genomics Xenium platform. This was applied to LMS tissue microarrays (TMAs) comprising a total of 326 tissue cores from 127 unique patients. Matched primary and metastatic samples from the same patient were available for 33 patients.
 Results:
 Analysis of scRNAseq data identified 2 distinct subtypes: a dedifferentiated subtype with mesenchymal features (MES) and a differentiated subtype with enrichment of smooth muscle cell markers (SMC). Integration of chromatin accessibility data from snATACseq showed enrichment of nuclear factor I (NFI) transcription factor (TF) motifs in the MES and AP-1 motifs in the SMC group. Whole genome sequencing did not reveal an obvious genomic etiology for these subtypes. Spatial transcriptomics was able to identify these 2 subtypes in a larger cohort of tumors. Consistent with snRNAseq data, we find that most tumors had almost exclusively either MES or SMC cells. We assessed spatial relationships between these subtypes and infiltrating immune cells. This revealed an enrichment in immunosuppressive macrophages and exhausted T cells in MES tumors compared to SMC tumors. Analysis of matched primary and metastatic tumors demonstrated that the subtype (MES or SMC) generally remains consistent between different sites of disease.
 Conclusions:
 We identify 2 novel LMS subtypes (MES and SMC) driven by distinct TFs. Spatial transcriptomic analysis confirmed the presence of these 2 subtypes in a larger cohort and demonstrated that MES tumors are associated with a more immunosuppressive tumor microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/170994843a3a0b5e571cb27c29ebdaf0b414317f" target='_blank'>
              Spatial transcriptomic profiling from over 300 leiomyosarcoma samples.
              </a>
            </td>
          <td>
            Ryan A Denu, Zhao Zheng, Veena Kochat, Yingda Jiang, William I. Padron, D. Ingram, Khalida M Wani, Larissa A Meyer, P. Soliman, R. Ratan, Alexander J. Lazar, E. Keung, Kunal Rai, Elise F Nassif Haddad
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are integral components of the tumor microenvironment playing key roles in tumor progression, metastasis, and therapeutic resistance. However, challenges persist in understanding their heterogeneity, origin, and functional diversity. One major obstacle is the lack of standardized naming conventions for CAF subpopulations, with current systems failing to capture their full complexity. Additionally, the identification of CAFs is hindered by the absence of specific biomarkers, limiting the precision of diagnostic and therapeutic strategies. In vitro culture conditions often fail to maintain the in vivo characteristics of CAFs, which complicates their study and the translation of findings to clinical practice. Although current detection methods, such as antibodies, mRNA probes, and single-cell transcriptomics, offer insights into CAF biology, they lack standardization and fail to provide reliable quantitative measures. Furthermore, the dynamic interactions between CAFs, tumor cells, and immune cells within the TME remain insufficiently understood, and the role of CAFs in immune evasion and therapy resistance is an area of ongoing research. Understanding how CAFs influence drug resistance and the immune response is essential for developing more effective cancer therapies. This review aims to provide an in-depth analysis of the challenges in CAF research, propose future research directions, and emphasize the need for improved CAF-targeted therapeutic strategies. By addressing these gaps, it seeks to highlight the potential of CAFs as targets for overcoming therapeutic resistance and enhancing the efficacy of cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a9e1a9ae9c6282d886b43a42912fd5730bf65" target='_blank'>
              Revisiting the role of cancer-associated fibroblasts in tumor microenvironment
              </a>
            </td>
          <td>
            Xiaolei Lan, Wenyang Li, Kai Zhao, Jianpeng Wang, Shifang Li, Hai Zhao
          </td>
          <td>2025-04-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2702678fe4bf9d99269192e19243ce5bd9b25ff1" target='_blank'>
              Patient-Derived Three-Dimensional Lung Tumor Models to Evaluate Response to Immunotherapy
              </a>
            </td>
          <td>
            Kayla F. Goliwas, Kenneth P. Hough, Sruti Sivan, Sierra L. Single, Sameer S. Deshmukh, Joel L. Berry, Maya Khalil, Benjamin Wei, Yanis Boumber, Mohammad Athar, Aakash Desai, S. Ponnazhagan, James M. Donahue, J. Deshane
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Renal cell carcinomas (RCC) are highly heterogeneous and aggressive cancers with diverse histological subtypes, molecular profiles, tumor microenvironments (TMEs), and clinical behaviors. Intratumoral heterogeneity (ITH), a hallmark of RCC, arises from distinct subclones within the same tumor that vary histopathologically and molecularly. To elucidate the biological mechanisms underlying RCC tumorigenesis at single-cell resolution, we examined 155 tumor specimens from 78 cases, encompassing clear cell RCC (ccRCC), papillary RCC (pRCC), chromophobe RCC (chRCC), and rare subtypes. We utilized 80 single-nucleus RNA sequencing (snRNA-seq) and 55 single-nucleus ATAC sequencing (snATAC-seq) datasets alongside advanced spatial biology techniques, including 27 Xenium In Situ, 12 Visium spatial transcriptomics (ST), and 7 co-detection by indexing (CODEX) samples. The connection of matched bulk proteogenomic data, clinical annotations, and histopathological evaluations from H&E staining further enhanced our spatial multimodal investigation. From single-nucleus sequencing of over half a million nuclei, we identified specific transcriptional programs and chromatin accessibility patterns among RCC subtypes, including HIF pathway dysregulation in ccRCC and MET pathway activation in pRCC. We also identified markers and constructed signature scores for rare but high-grade phenotypes, such as the aggressive rhabdoid and sarcomatoid features in ccRCC. Spatial analyses confirmed the utility of these feature-associated signatures for detection, quantification, and potential therapeutic targeting. Furthermore, we characterized the spatial organization and tumor-TME interactions, providing critical insights into spatially diverse ITH in RCC. Notably, a Grade-4 ccRCC sample exhibited three distinct tumor subclones and regions of necrosis characterized by Xenium, CODEX, and Visium profiling. Among these, a rhabdoid-associated subclone showed tumor-intrinsic signatures of elevated cell cycle activity and epithelial-mesenchymal transition (EMT), distinguishing it from conventional ccRCC subclones and indicating higher metastatic potential. We further revealed a unique macrophage subpopulation closely interacting with the rhabdoid subclone, forming an immunosuppressive niche that likely contributes to immune evasion, resistance to immunotherapies, and disease recurrence. Collectively, our spatial multimodal findings delineate the molecular signatures and mechanisms underlying RCC heterogeneity and aggressive features. By linking spatially distinct subclones to tumor progression and therapeutic resistance, this study provides valuable insights for developing personalized and effective therapeutic strategies to improve outcomes for RCC patients.


 Yize Li, Wagma Caravan, Yuwei Zhang, Andrew Houston, Siqi Chen, Chia-Kuei Mo, Preet Lal, Xiangwei Fang, Xiyi Wei, Ruiyang Liu, Ilya Strunilin, Yizhe Song, Song Cao, Cody Weimholt, Hui Zhang, Feng Chen, Li Ding Spatial multimodal understanding of disease heterogeneity and aggressiveness in renal cell carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 159.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f34cbb1aaff4c24b87ad94662408c71987a782" target='_blank'>
              Abstract 159: Spatial multimodal understanding of disease heterogeneity and aggressiveness in renal cell carcinoma
              </a>
            </td>
          <td>
            Yize Li, Wagma Caravan, Yuwei Zhang, Andrew Houston, Siqi Chen, Chia-Kuei Mo, Preet Lal, Xiangwei Fang, Xiyi Wei, Ruiyang Liu, Ilya Strunilin, Yizhe Song, Song Cao, Cody Weimholt, Hui Zhang, Feng Chen, Li Ding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains one of the deadliest cancers, with a five-year survival rate of 13%. The tumor microenvironment (TME) in PDAC includes a heterogeneous population of cells. In genetically engineered mouse models, the TME evolves alongside the formation and progression of epithelial precursor lesions to cancer. However, the evolution of the TME in human samples has not been thoroughly characterized, primarily due to limited access to pre-malignant pancreatic tissue. Leveraging a collaboration with Gift of Life Michigan, an organ donor organization, we previously reported that precursor lesions are abundant in the normal human pancreas (Carpenter et al., Cancer Discovery, 2023). To date, we have collected over 100 human pancreata, a dataset that enables us to map the characteristics of these lesions and their surrounding microenvironment across variables such as age, sex, and race. Here, we aim to model the co-evolution of the stroma and neoplastic epithelium during tumor progression. We utilized spatial transcriptomics technology with the 10x Visium platform, analyzing over 20 human pancreata during the discovery phase. We applied a spatially aware clustering approach to delineate distinct tissue domains across normal, tumor-adjacent, and tumor samples. We leveraged a previously established single-cell RNA-seq atlas (n=41) to deconvolute cell-type fractions and cell-type-specific gene expression profiles within each spatial transcriptomics spot. As a result, we identified distinct epithelial structures (e.g., Acinar, Ducts, PanINs, tumor) as well as stromal domains (e.g., tertiary lymphoid structures (TLS), fibrotic regions). We hypothesized that stromal gene expression is influenced by the proximity to and the density of different epithelial structures. Therefore, we employed a kernel density estimation approach to model gene expression in stromal cells, specifically fibroblasts and macrophages, as a function of epithelial-specific density. This analysis uncovered distinct characteristics of the stroma surrounding precursor lesions compared to tumor stroma, emphasizing spatially regulated gene expression programs that likely drive TME remodeling in PDAC. Future directions include performing cell-cell interaction analysis to further elucidate epithelial-stromal crosstalk, complemented by in vitro functional experiments to validate the pro- or anti-tumorigenic roles of the identified markers. This work lays the foundation for spatially resolved insights into PDAC biology, informing therapeutic strategies targeting the TME.


 Ahmed M. Elhossiny, Padma Kadiyala, Alexander Bray, Jamie Mills, Hannah Watkoske, Arvind Rao, Jude Okoye, Atul Deshpande, Jiaqi Shi, Elana J. Fertig, Filip Bednar, Yaqing Zhang, Eileen S. Carpenter, Timothy Frankel, Marina Pasca di Magliano. Spatially resolved insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7493.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3bcf08a317f0fb50f59a7b481b2ba31a08d66c" target='_blank'>
              Abstract 7493: Spatially resolved insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, Padma Kadiyala, Alexander Bray, Jamie N. Mills, Hannah R Watkoske, Arvind Rao, J. Okoye, Atul Deshpande, Jiaqi Shi, E. Fertig, Filip Bednar, Yaqing Zhang, Eileen S. Carpenter, Timothy L. Frankel, M. Pasca di Magliano
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 Cancer continues to be one of the leading causes of death and financial burden on public health worldwide. Despite advances in detection and treatments, there is still a significant level of uncertainty in predicting how an individual's cancer will respond to treatment due to tumor heterogeneity among different patients and multiple subclones from the same individual. A more comprehensive understanding of cancer cell populations, their molecular interactions, and their microenvironment can offer valuable insights into optimal treatment strategies and patient stratification. This can be achieved through the integrated detection of both RNA and protein targets, enabling a deeper exploration of the molecular mechanisms driving cancer.
 Here, we analyzed formalin-fixed, paraffin-embedded (FFPE) human tissue sections from diverse tissue types using spatial multi-omics technologies from 10x Genomics. A new add-on capability enables visualization and analysis of protein expression for key immuno-oncology markers including markers for immune cell typing, cell structure, proliferation, and differentiation, as well as immune checkpoint markers (e.g., PD-1, PD-L1, LAG-3, and VISTA). Simultaneous spatial profiling of 480 genes and 28 proteins, integrated with H&E imaging, provided detailed insights into tissue architecture, tumor origins, immune cell infiltration, and the immune status of the tumor microenvironment.
 The combined RNA and protein profiling method is a powerful approach for validating RNA-based findings and providing definitive evidence at the functional biological level. Protein data also offers critical insights into the expression and function of different isoforms that are challenging to discern at the RNA level. For instance, isoforms such as CD45RA and CD45RO, which arise from alternative splicing of the PTPRC gene, are often indistinguishable using RNA profiling alone due to the shared exonic sequences in their transcripts. However, protein-level analysis can differentiate these isoforms and enables a more precise understanding of their distinct roles in the immune response, including the delineation of naïve and memory T-cell subsets.
 Furthermore, RNA expression does not always directly correlate with protein levels, underscoring the importance of integrating both data types. For instance, discrepancies between RNA and protein expression were observed for β-catenin, encoded by the CTNNB1 gene, across tissues. β-catenin plays a pivotal role in tumor progression through its dual functions in cell adhesion and signaling. By integrating RNA and protein data, researchers can uncover the complexities of post-transcriptional regulation, subcellular localization, and their combined influence on cancer progression, providing deeper insights into disease mechanisms and potential therapeutic targets.


 Thanutra Zhang, Smritee Dadhwal, Joshua Thao, Anatalia Robles, Cheyenne Christopherson, Suyog Pathare, Kristen Pham, Minkyung Park, Hsin Wen Chang, Janine Hensel, Cedric Espenel, Hayato Ikoma, Gabriele Girelli, Paul Ryvkin, Veronica Gonzalez Muñoz. Advancing tumor characterization and immune phenotyping through multiomic in situ profiling of RNA and proteins [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7466.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183f97a84615b6dbc6366f5d2cb1c5539c88c0f6" target='_blank'>
              Abstract 7466: Advancing tumor characterization and immune phenotyping through multiomic in situ profiling of RNA and proteins
              </a>
            </td>
          <td>
            Thanutra Zhang, Smritee Dadhwal, Joshua Thao, Anatalia Robles, Cheyenne Christopherson, Suyog Pathare, Kristen Pham, Minkyung Park, Hsin Wen Chang, Janine Hensel, Cedric Espenel, Hayato Ikoma, Gabriele Girelli, Paul Ryvkin, Veronica Gonzalez Muñoz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neural stem cells (NSCs) in the subventricular zone (SVZ) are identified as cells-of-origin harboring driver mutations in glioblastoma (GBM), which is the most devastating brain tumor with highly heterogeneous nature. However, the sequential transformation of a limited number of mutation-harboring NSCs into a distant tumor with high intratumoral heterogeneity remains poorly understood. In this study, we have identified transcriptionally distinct types of mutation-harboring precancerous cells in our spontaneous, somatic mouse model recapitulating human GBM evolution as well as in tumor-free SVZ tissues from patients. These precancerous cells emerge via oligodendrocyte lineage specification, exhibiting unique transcriptional programs involving dysregulated translations and extracellular matrix remodeling. Subsequently, they give rise to heterogeneous tumor cell populations by activating multiple programs crucial for gliomagenesis. Our findings highlight the pivotal role of precancerous cells in tumor evolution and intratumoral heterogeneity, suggesting their potential as a novel therapeutic target for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a32f7726bad7de5bd6ab6b676111c1867233e6" target='_blank'>
              Precancerous Cells Initiate Glioblastoma Evolution and Contribute to Intratumoral Heterogeneity.
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, Jong Hee Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, J. H. Lee
          </td>
          <td>2025-04-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Epithelial-to-mesenchymal transition (EMT), a complex biological pathway that facilitates cellular plasticity, is used by tumor cells to enable metastasis and drug resistance. Our functional understanding of the impact of EMT on cancer has been limited by the lack of effective tools to ablate tumor cells as they become mesenchymal. In a recent study published in Nature, Perelli and colleagues used elegant genetically engineered lineage tracing and ablation strategies to track and eliminate tumor cells as they undergo EMT in pancreatic cancer. In a two-pronged approach, they queried the functional consequences of ablating EMT tumor cells before pancreatic ductal adenocarcinoma (PDAC) formation or in advanced PDAC tumors. These experiments collectively revealed that epithelial tumor cells only progress to low-grade lesions with minimal proliferative potential, while mesenchymal tumor cells undergo EMT early on to become malignant and metastasize. Profiling of mesenchymal tumor cell lineages revealed an altered chromatin landscape that leads to chromosomal instability (CIN) and disease progression. CIN is facilitated through complex structural rearrangements and chromothripsis, ultimately driving increased tumor heterogeneity and enhanced proliferation in EMT cells. This work reveals that EMT is an important driver of tumor heterogeneity and progression as a downstream consequence of CIN and provides mechanistic insight into how cellular plasticity can lead to genomic changes that drive disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92ab05366d53bbf0f6bb5cf31fdb12098dce09df" target='_blank'>
              To EMT or not to EMT: Ablation of mesenchymal tumor cell lineages reveals the essential role of EMT in pancreatic cancer initiation and evolution.
              </a>
            </td>
          <td>
            Jessica Peura, Calvin Johnson, Jason R. Pitarresi
          </td>
          <td>2025-04-08</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 The tumor-immune microenvironment (TIME) in solid tumors is a complex ecosystem consisting of diverse interactions between cellular populations. Deeper investigation into the TIME reveals a prominent role of immune cells in regulating tumor progression- a novel phenomenon that has encouraged the evaluation of the crosstalk between cell types. The development of spatial omic technologies has thus enabled the exploration of these nuances at previously inaccessible resolutions, but the currently available platforms come at the cost of introducing bias, excluding molecular information, and incompatibility with single cell technologies. Here, we leverage a photoactivatable (PA) dye Janelia-Fluor-646 in combination with a digital light processing (DLP) microscope to address these issues by developing a platform that enables the spatially guided isolation of cell populations from a region of interest (ROI) for downstream multiomic and single cell analyses. As proof of concept, we show that ROIs identified via immunofluorescence could successfully be barcoded in mouse spleen and tumor sections using the dye and distinguished from the background by fluorescence activated cell sorting (FACS). We also demonstrate the ability to extract high quality DNA, RNA, and protein- as well as maintain high cell viability post-sorting. We have also developed a pipeline for the analysis of the sequenced RNA to identify differential patterns of gene expression present in immune cells from spatially distinct regions. Moving forward, we hope to establish this as a platform for immune landscape analysis within the TIME to uncover unique targets that could be explored within the context of tumor-immune interactions as well as immunotherapeutic development. Our findings will also help us generate the first single-cell atlas of niche-specific immune cells.


 Elie Abi Khalil, Yi-Chien Wu, Jingtian Zheng, Joshua Plank, Alexander Lippert, Steve Lee. Immune landscape analysis of solid tumors via spatially guided cell isolation for enhanced immunotherapeutic development [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5253.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cac8b257d411c793fd0016b4f5f926fd3d98de50" target='_blank'>
              Abstract 5253: Immune landscape analysis of solid tumors via spatially guided cell isolation for enhanced immunotherapeutic development
              </a>
            </td>
          <td>
            Elie Abi Khalil, Yichao Wu, Jingtian Zheng, Joshua Plank, Alexander Lippert, Steve Seung‐Young Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in tumor progression, immune evasion, and therapeutic responses. Among its key components, endothelial cells (ECs) are crucial regulators of angiogenesis, immune cell trafficking, and metabolic adaptations. This study integrates single-cell and transcriptomic analyses to identify tumor-specific endothelial cell signatures in hepatocellular carcinoma (HCC) and stratify tumors into three distinct molecular subtypes. These subtypes exhibit unique immune landscapes and biological characteristics, including pathway activation and differential responses to immunotherapy and targeted treatments. Using machine learning, we developed a robust prognostic scoring model to predict patient outcomes and therapy responsiveness, which was validated across independent cohorts. Our findings highlight the critical role of endothelial cells in modulating the TME and underscore the potential of targeting EC-specific molecular features to enhance the efficacy of immunotherapy and optimize personalized cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02543-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/727e8918f7771cb02692376a24c49897f7676656" target='_blank'>
              Unraveling the TEC-associated landscape in hepatocellular carcinoma: a comprehensive study based on multi-omics analyses
              </a>
            </td>
          <td>
            Jianwei Lan, Longhui Xie, Dekun Song, Pengpeng Liu, Quanyan Liu
          </td>
          <td>2025-05-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Skin squamous cell carcinoma (SCC) is characterized by heterogeneity in differentiation states and immune exclusion within the tumor microenvironment (TME). Using the Bruker Spatial Biology CosMx® Whole Transcriptome (WTX) panel, which profiles approximately 19, 000 genes at single-cell resolution, we examined spatial gene expression in FFPE SCC sections. Individual single cell boundaries were defined utilizing a trained AI cell segmentation model. H&E staining on the same tissue provided histopathological context, enabling the integration of molecular and morphological findings. Combining CosMx WTX data with H&E staining, downstream AI models could be used to further predict cell types and transcriptomic signatures directly from H&E images. FFPE SCC samples were subjected to CosMx WTX imaging to generate spatially resolved transcriptomic data. H&E staining, performed after CosMx WTX, was aligned with transcriptomic data using custom software. Histopathological analysis identified keratin pearls, invasive fronts, and stromal compartments. Bioinformatics mapped gene expression signatures of keratinization, immune exclusion, and extracellular matrix remodeling, providing a detailed view of tumor architecture. Thousands of genes were mapped with high resolution, allowing in-depth exploration of the SCC TME. Cell typing identified spatially distinct cancer subpopulations and their relationship with stromal and immune components. Ligand-receptor analysis revealed spatial patterns of cell-cell communication, while proximity analyses highlighted unique immune interactions. Macrophages near cancer cells showed increased transcription of specific genes compared to distant macrophages, suggesting spatially restricted tumor-immune signaling. Integration with H&E staining revealed molecular correlates of keratinization, immune exclusion, and invasive front dynamics, bridging histopathology with spatial transcriptomics. This integration offers the potential to train AI models that predict tumor cell types and molecular profiles directly from H&E images, enabling faster and more accessible analysis. This study integrates single-cell spatial transcriptomics with histopathological features to reveal the molecular mechanisms of tumor differentiation, immune evasion, and stromal remodeling in SCC. The combination of spatial transcriptomics and H&E images holds potential for future AI models that enable non-invasive, real-time tumor profiling, providing a foundation for therapeutic strategies targeting SCC progression and immune suppression.


 Patrick Danaher, Michael Patrick, Shanshan He, Liang Zhang, Stefan Rogers, Michael Rhodes, Haiyan Zhai, Joseph Beechem. High-resolution and AI-enabled single-cell spatial transcriptomics and histopathology integrated to reveal tumor differentiation and immune exclusion in skin squamous cell carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 751.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3f7948650b331b965b944ec4df04f2a43214f4" target='_blank'>
              Abstract 751: High-resolution and AI-enabled single-cell spatial transcriptomics and histopathology integrated to reveal tumor differentiation and immune exclusion in skin squamous cell carcinoma
              </a>
            </td>
          <td>
            Patrick Danaher, Michael Patrick, Shanshan He, Liang Zhang, Stefan Rogers, Michael Rhodes, Haiyan Zhai, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Diffuse large B-cell lymphoma (DLBCL) is a highly heterogeneous malignancy with challenges in treatment resistance and relapse. Single-cell RNA sequencing (scRNA-seq) has provided important insights into tumor heterogeneity, microenvironment interactions, resistance mechanisms, and prognostic biomarkers. This review summarizes key findings from scRNA-seq studies, which have deepened our understanding of DLBCL and contributed to the development of precision therapeutic strategies. Integrating scRNA-seq with spatial transcriptomics and single-cell multi-omics may further elucidate disease mechanisms and identify novel therapeutic targets, supporting the advancement of precision medicine in DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/451fee19bba3ef10ba2c6c99b87597bd237ca7e2" target='_blank'>
              Single-cell RNA sequencing in diffuse large B-cell lymphoma: tumor heterogeneity, microenvironment, resistance, and prognostic markers
              </a>
            </td>
          <td>
            Linwei Li, Qiwei Li, Rui Niu, Wei Sun, Hua Liang
          </td>
          <td>2025-04-09</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 Advanced melanoma is an aggressive cancer with a high metastatic potential often resulting in melanoma brain metastases (MBM). The complexity of the MBM tumor microenvironment (TME) impacts tumor progression and therapy response, leading to poor outcomes. Our study integrates spatial transcriptomics profiles with tissue morphology annotations of MBM and clinical data to investigate the TME landscape in relation to treatment and patient outcomes.



 We performed spatial transcriptomics on 21 MBM samples collected after various treatment strategies. All samples were preserved in formalin-fixed paraffin-embedded (FFPE) blocks and profiled using the Visium Spatial Gene Expression kit (10X Genomics) with a 6.5x6.5mm Visium Capture slide. All sequencing was performed on a NovaSeq6000 platform (Illumina) with an average of 71,391 reads per spots. H&E slides were annotated by our study pathologist (DL) to enable pathology-guided cluster analysis. Spatial transcriptomics data analysis was conducted using Seurat R pipeline which involved data quality check, normalization, integration, cell type annotation and deconvolution, and clustering. Further differential gene expression analyses were performed between different identified tumor clusters or between clinical groups, leading to the identification of enriched pathways and cell-cell interaction patterns.



 Our phenotyping of 25,208 spots revealed the presence of diverse cell populations, including neurons, melanoma, plasma, endothelial, stromal, and myeloid cells. We identified 8 distinct functional clusters characterized by specific gene expression and activated pathways. These include 5 tumor clusters with different immune infiltration or expression profiles, identifying differences in cell metabolism and heterogeneity in the TME. Importantly, we found MBM from patients exposed to radiation were significantly enriched for epithelial-to-mesenchymal transition pathway and genes related to cell adhesion. We further showed a reduced immune response in patients received radiation compared those who did not, and we noted that an active immune response was generally positively correlated with survival.



 Our study is the largest spatial analysis of MBM reported to date. Advances in spatial technologies provide novel insights into the structure and composition of MBM TME. Our findings highlight the importance of spatial context in understanding MBM biology and the potential of spatial transcriptomics in advancing precision oncology.



 Clemence J. Belle, Sandra Brosda, Vanessa F. Bonazzi, Victor Bulteau, Thomas Stuart, Zherui Xiong, Duncan Lambie, Peter A. Johansson, Lauren G. Aoude, Kalpana Patel, Samantha J. Stehbens, Mitchell S. Stark, Nikolas K. Haass, Wen Xu, Mark B. Pinkham, Matthew C. Foote, Sarah Olson, Victoria Atkinson, Arutha Kulasinghe, Quan H. Nguyen, Andrew Barbour. Unravelling melanoma brain metastasis tumor microenvironment characteristics using spatial transcriptomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5076.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed241f7ab2b6bafb6e0e97061f34d2c422a2e850" target='_blank'>
              Abstract 5076: Unravelling melanoma brain metastasis tumor microenvironment characteristics using spatial transcriptomics
              </a>
            </td>
          <td>
            Clemence J. Belle, S. Brosda, V. Bonazzi, Victor Bulteau, Thomas Stuart, Zherui Xiong, Duncan Lambie, Peter A. Johansson, L. Aoude, Kalpana Patel, Samantha J Stehbens, Mitchell S. Stark, N. Haass, Wen Xu, M. Pinkham, Matthew C. Foote, Sarah Olson, Victoria Atkinson, A. Kulasinghe, Quan H. Nguyen, Andrew Barbour
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive and lethal adult brain tumor. The cellular heterogeneity within the tumor microenvironment (TME) plays a critical role in the complexity of treatment and poor survival. GBM is typically classified into 3 molecular subtypes-Classical, Mesenchymal, and Proneural-associated with EGFR, NF1, and PDGFRA genetic drivers, respectively. Yet, the role of these driver mutations on the GBM TME is not fully understood. Here, we utilized single-cell RNA-sequencing of genetically engineered mouse GBM models incorporating human-relevant EGFRvIII, PDGFB, and NF1 driver mutations to systematically characterize the genotype-immunophenotype relationship of the three GBM subtypes. Murine genetic GBM models at the single-cell level effectively mimic the inter- and intra-tumor heterogeneity found in human counterparts. Our analysis revealed that PDGFB-driven tumors were more proliferative and enriched for Wnt signaling interactions, while EGFRvIII-driven tumors showed an elevated interferon signaling response. Moreover, Nf1-silenced tumors displayed higher myeloid abundance, myeloid immunosuppressive interactions involving Osteopontin, Treg infiltration, and expression of immune checkpoint molecule Ctla4. Overall, we established a human-mouse analytical platform for genotype-aware target discovery and validation, which offers promising new avenues for more effective, personalized treatments in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f30684782f70f49100b5a4c8e6aa8a01fed878d3" target='_blank'>
              Single-cell dissection of the genotype-immunophenotype relationship in glioblastoma.
              </a>
            </td>
          <td>
            N. Soni, Kavita Rawat, Zhihong Chen, Angela DiMauro, Bruno Giotti, Dolores Hambardzumyan, A. Tsankov
          </td>
          <td>2025-04-11</td>
          <td>Brain : a journal of neurology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Neuroblastoma, the most common pediatric extracranial solid tumor, has heterogeneous clinical outcomes ranging from malignant progression to spontaneous regression. With the highest frequency of the elusive spontaneous regression, low-risk INSS Stage 4S neuroblastoma represents an ideal model for mechanistic investigation. Spontaneous regression is often accompanied by tumor differentiation, but the mechanisms underlying this process remain largely unclear. Methods Single-nucleus transcriptomics (snRNA-seq) data of neuroblastoma samples were obtained from the Synapse repository to investigate the composition of heterogeneous tumor cell clusters. The feature of the Stage 4S-specific tumor cell subpopulation was revealed through differential expression analysis, pathway enrichment analysis and pseudotime analysis, followed by clinical significance validation on public cohort datasets. The biological function of secreted SLIT3 was validated using multiple in vitro models, including recombinant protein treatment, conditioned medium treatment, and cell lines coculture, to confirm the intratumoral crosstalk effect. Orthotopic and subcutaneous xenograft models were established to verify SLIT3's in vivo function. Cellular bulk RNA-seq analysis was performed with or without SLIT3 recombinant protein treatment to discover the downstream pathways activated by SLIT3, followed by validation with specific pathway inhibitors. Results Analysis of snRNA-seq revealed a distinct subpopulation of tumor cells within INSS Stage 4S neuroblastoma, characterized by a spontaneous regression-like program progressing toward differentiation. Activated SLIT-ROBO signaling was found in the Stage 4S-specific tumor cell subpopulation, which strongly correlated with favorable prognosis. Further investigation into the secreted ligands in SLIT-ROBO related pathways revealed that SLIT3 displayed the most potent enrichment in Stage 4S tumors and the strongest differentiation-inducing effect. In vitro experiments using recombinant SLIT3 protein, conditioned medium, and cell lines coculture consistently demonstrated the capacity of SLIT3 to induce neuroblastoma cell differentiation via intratumoral crosstalk, as evidenced by increased neurite outgrowth and elevated expression of neuronal differentiation markers. Both orthotopic xenograft and subcutaneous xenograft models demonstrated that SLIT3 expression suppressed tumor growth, leading to in vivo tumor differentiation. Mechanistically, PLCβ/PKC signaling mediates the SLIT3-induced neuroblastoma cell differentiation. Conclusions Stage 4S-specific tumor cell subpopulation exhibits a spontaneous regression-like program, from which SLIT3 mediates intratumoral crosstalk and promotes neuroblastoma differentiation via PLCβ/PKC signaling. These findings provide new insights into the mechanism of spontaneous regression in neuroblastoma and offer novel therapeutic targets for differentiation-based treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06621-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc7b67c542baf51616516c8722d1f59d34fe27f7" target='_blank'>
              SLIT3-mediated intratumoral crosstalk induces neuroblastoma differentiation via a spontaneous regression-like program
              </a>
            </td>
          <td>
            Meiling Liu, Dekang Lv, Wenjing Yan, Yi Wu, Shulan Wang, Luoxuan Wang, Jie Lei, Deshun Zeng, Zifeng Wang, Fang Liu, Bing Deng, Quentin Liu, Bin He, M. Yan
          </td>
          <td>2025-05-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Schwannomas are nerve sheath tumors arising at cranial and peripheral nerves, either sporadically or in patients with a schwannomatosis-predisposition syndrome. There is limited understanding of the transcriptional heterogeneity of schwannomas across genetic backgrounds and anatomic locations. Methods Here, we prospectively profile by single-cell full-length transcriptomics tumors from 22 patients with NF2-related schwannomatosis, non-NF2-related schwannomatosis, and sporadic schwannomas, resected from cranial and peripheral nerves. We profiled 11,373 cells (after QC), including neoplastic cells, fibroblasts, T cells, endothelial cells, myeloid cells, and pericytes. Results We characterize the intra-tumoral genetic and transcriptional heterogeneity of schwannoma, identifying six distinct transcriptional metaprograms, with gene signatures related to stress, myelin production, antigen presentation, interferon signaling, glycolysis, and extracellular matrix. We demonstrate the robustness of our findings with analysis of an independent cohort. Conclusions Overall, our atlas describes the spectrum of gene expression across schwannoma entities at the single-cell level and will serve as an important resource for the community. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01462-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a8868eb66d6c72488d55b6daa0e1d4183e3adcb" target='_blank'>
              A single-cell atlas of Schwannoma across genetic backgrounds and anatomic locations
              </a>
            </td>
          <td>
            L. G. Gonzalez Castro, Avishai Gavish, L. Bussema, Christopher W Mount, Cyril Neftel, M. Nomura, E. Chiocca, W. Bi, Omar Arnaout, Fred G. Barker, Justin M. Brown, Justin T Jordan, Tracy T. Batchelor, A. Stemmer-Rachamimov, Scott Plotkin, I. Tirosh, M. Suvà
          </td>
          <td>2025-04-11</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a biologically heterogeneous malignancy of clonal plasma cells, often progressing from MGUS or smoldering MM. It causes anemia, bone lesions, and immune dysfunction due to abnormal plasma cell expansion in the bone marrow. Neuroinflammatory and neurotrophic factors may influence MM progression by affecting immune cells and the bone marrow niche. Growing evidence points to a role for neuroimmune regulation in tumor immunity. Despite therapeutic progress, disease heterogeneity and resistance highlight the need for new strategies targeting the tumor microenvironment and neuroimmune axis. Methods This investigation exploited single-cell RNA sequencing (scRNA-seq) to analyze MM and high-risk smoldering multiple myeloma (SMMh) samples, identifying 11 distinct cell types. We examined their transcriptional signatures, stemness, proliferative properties, and metabolic pathways, with particular attention to neuroimmune interactions in the tumor microenvironment. Using trajectory inference tools such as CytoTRACE, Monocle2, and Slingshot, we traced the differentiation paths of MM cell subpopulations and identified key signaling pathways that may influence immune responses and tumor progression. Results The analysis identified four distinct subpopulations of myeloma cells, with the C0 IGLC3+ myeloma cells representing the least differentiated and most proliferative subset. These cells played a critical role in MM progression and may contribute to immune evasion mechanisms. Additionally, receptor-ligand interactions within the tumor microenvironment were identified, which may be influenced by neuroinflammatory and neurotrophic factors. These findings suggest that the nervous system and immune modulation significantly affect tumor biology, highlighting potential therapeutic targets that could be exploited to overcome resistance to conventional therapies. Conclusion This single-cell analysis provided new insights into the cellular diversity and differentiation trajectories in MM, offering a deeper understanding of the complex neuroimmune interactions that drive tumor progression and resistance. By incorporating the role of neuroinflammation and immune modulation, our study suggested novel therapeutic strategies targeting the neuroimmune axis in oncology, ultimately contributing to the development of more effective, personalized treatment approaches for MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb42682593bbacba35775b457e4d73a57648a9" target='_blank'>
              Decoding multiple myeloma: single-cell insights into tumor heterogeneity, immune dynamics, and disease progression
              </a>
            </td>
          <td>
            Zhenzhen Zhao, Zhijie Zhao, Zhiheng Lin, Lu Fan, Zhikai Xiahou, Yujiang Dong, Weiying Bao
          </td>
          <td>2025-05-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Oligonucleotide therapeutics, such as antisense oligonucleotides (ASOs) and small interfering RNAs (siRNAs), are promising tools for targeting oncogenic pathways in cancer. However, their impact on tumor microenvironment (TME) architecture and cellular phenotypes within cancer tissues remains insufficiently understood. This study integrates oligonucleotide-based treatments with advanced spatial phenotyping to explore their effects on cancer tissues at a high-definition, single-cell scale.



 Cancer tissues treated with oligonucleotide therapeutics were analyzed using multiplexed immunohistochemistry (mIHC) and spatial transcriptomics. Cellular phenotypes, including immune and stromal cell populations, were characterized in situ to assess changes in the TME. High-resolution imaging and transcriptomic profiling were used to localize therapeutic effects, while computational analysis integrated phenotypic and molecular data to uncover spatially resolved mechanisms.



 Oligonucleotide treatment led to distinct spatial alterations in the TME, including reprogramming of tumor associated macrophages and increased infiltration of cytotoxic T cells in peritumoral regions. Spatial transcriptomics revealed suppression of target oncogenes, alongside upregulation of immune activation pathways in responsive regions of the tissue. Notably, these effects were localized to specific niches, suggesting cell-type-specific and spatially restricted therapeutic responses.



 This study highlights the power of integrating oligonucleotide therapeutics with spatial phenotyping to uncover their impact on the TME in cancer tissues. The findings provide valuable insights into the spatial dynamics of oligonucleotide therapy, paving the way for more effective design and deployment of these treatments in oncology.



 Corinne Ramos, Sandra Boute, Amandine Gerstenberg, Richard Ruez. Spatially resolved effects of oligonucleotide therapeutics on cellular phenotypes in cancer tissues [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3100.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09be66602da590319fde64cb1853d326259197c5" target='_blank'>
              Abstract 3100: Spatially resolved effects of oligonucleotide therapeutics on cellular phenotypes in cancer tissues
              </a>
            </td>
          <td>
            Corinne Ramos, Sandra Boute, Amandine Gerstenberg, Richard Ruez
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="One of the leading causes of tumor cell heterogeneity is the complex nature of the tumor immune microenvironment involving crosstalk between different signaling pathways of cancer cells and immune cells. For cancer cells to grow into a primary tumor and then metastasize into secondary tumors, they must avoid or overcome the attack of different types of immune cells, including macrophages, natural killer (NK) cells, and T lymphocytes. For example, cancer cells can polarize macrophages from “tumor killing” to “tumor promoting” type using specific signaling molecules. Such tumor immune crosstalk is dependent on the clonal and sub-clonal heterogeneity in genomic, transcriptomic and protein signaling networks across different cancer cell sub-populations, immune cells and its surrounding microenvironment. Since structural aberrations affect gene dysregulation, the study of the 3D- genome organization in single cells can help in better understanding of disease progression and prognosis. The genome is organized and packaged in a highly structured manner inside the nucleus of mammalian cells and exhibits dynamic reorganization throughout disease progression. Understanding this 3D organization at the single-cell level with high spatial resolution is crucial for immune-oncology disease research such as investigating tumor-immune cell interactions.
 In this abstract, we present a novel jebFISHTM protocol on the PaintScapeTM platform that can be used to investigate immune cell modulated differences in global 3D genome organization among different localized and metastatic breast cancer cell lines at single cell, sub-population and population level. Using a genome wide panel of chromosomal targets on an onco-immune co-culture system involving different breast cancer cell lines and CD4+ T-cells or macrophages, we will show disruption in chromosome territory, radius of gyration, chromosomal instability between localized and metastatic breast cancer cells at single cell and sub-population level. We observed signatures of specific 3D genome structural alterations and interchromosomal interactions in cancer cells and/or immune cells, related to the effect of different types of immune response. We will show how the 3D architecture of key genomic loci and its associated regulatory elements belonging to major cancer signaling pathways exhibit specific signatures including differential folding, chromosomal instability, differential interchromosomal interaction and structural variation patterns at single cell, sub-chromosomal and sub-population level. We envision this study will improve our understanding on breast cancer tumor immune microenvironment and provide deeper insight into the underlying genomic heterogeneity of single cancer cells and sub-populations which might help to design better treatment options in the future.


 Jude Dunne, Huy Nguyen, Shyamtanu Chattoraj. Mapping spatial 3D genome landscape of breast cancer cells in different tumor immune microenvironments using onco-immune co-culture as a model [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5241.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e688452c1e795984dc287acda606201c118b68e6" target='_blank'>
              Abstract 5241: Mapping spatial 3D genome landscape of breast cancer cells in different tumor immune microenvironments using onco-immune co-culture as a model
              </a>
            </td>
          <td>
            Jude Dunne, Huy Nguyen, Shyamtanu Chattoraj
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="PURPOSE
Immunotherapy has demonstrated promise in small cell lung cancer (SCLC), but certain patients encounter limited benefits, highlighting the need for immunosuppressive biomarkers. Extrachromosomal circular DNA (ecDNA) promotes amplification of MYC-paralogs (MYC, MYCN and MYCL), driving cross-resistance in SCLC. Here, we aim to investigate whether ecDNA-mediated MYC-paralogs amplification (ecMYC+) represents immunosuppressive features in SCLC.


EXPERIMENTAL DESIGN
Bulk RNA sequencing data were retrieved from public database and paraffin-embedded samples. The overexpression and amplification of MYC-paralogs were identified using immunohistochemistry and fluorescence in situ hybridization. Imaging mass cytometry and multiplex immunohistochemistry were used to characterize spatial distribution of tumor immune microenvironment (TIME). The copy number of MYC-paralogs was investigated using real-time polymerase chain reaction. RNA-sequencing and flow cytometry were performed in SCLC cell lines.


RESULTS
The mean copy number of ecDNAs and the frequency of ecMYC+ cell lines were higher in SCLC than that in the other lineages (SCLC 22/47 vs others 15/282). In ecMYC+ SCLC, multiple immune-related pathways were downregulated while nucleotide metabolism processes were upregulated. Inhibition of nucleotide metabolism induced ecDNA elimination, along with activated antigen presenting pathways. Highly dispersed MYC-paralogs amplifications were detected in resected treatment-naïve SCLC samples. Through the resolution of 103,341 cells from 24 pathological regions, we observed higher expression of MKI67, VEGFA, FAP and FOXP3 and reduced T cell infiltration in ecMYC+ samples. Moreover, ecMYC+ samples exhibited elevated cellular neighborhoods dominated by Ki67+ tumors, with reduced spatial interaction with immune cells.


CONCLUSIONS
Extrachromosomal amplification of MYC-paralogs shapes suppressive TIME, identifying potential subgroup of immunotherapy resistant patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202c6e4f5a1bfae884ce6819555bba6dc3de7227" target='_blank'>
              Amplification of extrachromosomal MYC paralogs shapes immunosuppressive tumor microenvironment in small cell lung cancer.
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Haodong Lin, Jiaming Deng, Yan Ju, Xueyan Hu, Jianqi She, Zhijian Liang, Kongxu Dai, Mantang Qiu, Kunkun Sun, Jun Wang, Fan Yang, Jian Chen, Ence Yang, Xiao Li
          </td>
          <td>2025-04-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cervical cancer represents a significant global health challenge, with complex cellular and molecular mechanisms driving its progression from HPV infection to invasive malignancy. This study employed an integrated approach combining single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (stRNA-seq) to comprehensively characterize the tumor microenvironment (TME) across different stages of cervical cancer development. Through analysis of samples from normal cervix, HPV-infected normal cervix, high-grade squamous intraepithelial lesions (HSIL), and invasive cervical cancer, we identified distinct cellular populations and their dynamic changes during disease progression. Our findings revealed significant heterogeneity in immune cell populations, particularly highlighting the role of SPP1+ macrophages that were substantially enriched in cervical cancer compared to precancerous and normal tissues. Cell-cell communication networks and spatial mapping demonstrated that SPP1+ macrophages interact extensively with immune cells through the SPP1-CD44 signaling axis. This interaction contributes to an immunosuppressive microenvironment through modulation of T cell function and promotion of tumor cell survival. Furthermore, high expression of SPP1 correlated with advanced tumor stages and poor overall survival in cervical cancer patients, highlighting its potential as a prognostic biomarker. Our comprehensive characterization of the cellular landscape and intercellular communication networks in cervical cancer progression provides valuable insights for the development of targeted therapeutic strategies aimed at modulating the TME, particularly through disruption of the SPP1-CD44 axis. These findings establish a foundation for more effective personalized approaches to improve clinical outcomes in cervical cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87cbb4dd12be6f1664aacd1df3e0b1efa6fe30d7" target='_blank'>
              Deciphering the cellular and molecular landscape of cervical cancer progression through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Peng Xia, Juanhong Zhou, Rong Shen, Degui Wang
          </td>
          <td>2025-05-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma is the most aggressive and lethal cancer of the central nervous system, presenting substantial treatment challenges. The current standard treatment, which includes surgical resection followed by temozolomide and radiation, offers limited success. While immunotherapies, such as immune checkpoint inhibitors, have proven effective in other cancers, they have not demonstrated significant efficacy in GBM. Emerging research highlights the pivotal role of tumor-associated macrophages (TAMs) in supporting tumor growth, fostering treatment resistance, and shaping an immunosuppressive microenvironment. Preclinical studies show promising results for therapies targeting TAMs, suggesting potential in overcoming these barriers. TAMs consist of brain-resident microglia and bone marrow-derived macrophages, both exhibiting diverse phenotypes and functions within the tumor microenvironment. This review delves into the origin, heterogeneity, and functional roles of TAMs in GBM, underscoring their dual roles in tumor promotion and suppression. It also summarizes recent progress in TAM-targeted therapies, which may, in combination with other treatments like immunotherapy, pave the way for more effective and personalized strategies against this aggressive malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f77ce2f24a7dfd7badc0d732e7d82ccdd3d1f004" target='_blank'>
              Exploring tumor-associated macrophages in glioblastoma: from diversity to therapy
              </a>
            </td>
          <td>
            Wenwen Zhao, Zhi Zhang, Mingyuan Xie, Feng Ding, Xiangrong Zheng, Shicheng Sun, Jianyang Du
          </td>
          <td>2025-05-02</td>
          <td>NPJ Precision Oncology</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="One of the key challenges in defeating advanced tumors is the ability of cancer cells to evade the selective pressure imposed by chemotherapy, targeted therapies, immunotherapy and cellular therapies. Both genetic and epigenetic alterations contribute to the development of resistance, allowing cancer cells to survive initially effective treatments. In this narration, we explore how genetic and epigenetic regulatory mechanisms influence the state of tumor cells and their responsiveness to different therapeutic strategies. We further propose that an altered balance between cell growth and cell death is a fundamental driver of drug resistance. Cell death programs exist in various forms, shaped by cell type, triggering factors, and microenvironmental conditions. These processes are governed by temporal and spatial constraints and appear to be more heterogeneous than previously understood. To capture the intricate interplay between death-inducing signals and survival mechanisms, we introduce the concept of Death-ision. This framework highlights the dynamic nature of cell death regulation, determining whether specific cancer cell clones evade or succumb to therapy. Building on this understanding offers promising strategies to counteract resistant clones and enhance therapeutic efficacy. For instance, combining DNMT inhibitors with immune checkpoint blockade may counteract YAP1-driven resistance or the use of transcriptional CDK inhibitors could prevent or overcome chemotherapy resistance. Death-ision aims to provide a deeper understanding of the diversity and evolution of cell death programs, not only at diagnosis but also throughout disease progression and treatment adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24390bbe04bc2b2ed371e1dae568af6c9f084554" target='_blank'>
              Death-ision: the link between cellular resilience and cancer resistance to treatments
              </a>
            </td>
          <td>
            Gustavo Baldassarre, Ivana L. de la Serna, François M Vallette
          </td>
          <td>2025-05-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Advances in our understanding of the tumor microenvironment (TME) will help inform the development of next generation immunotherapies and engineered immune-cell therapies. Spatially characterizing immune cells with cell marker protein and activation marker RNAs is crucial to determine therapeutic efficacy and potential therapeutic toxicity. To address this, we have developed a high throughput RNAscope™ assay on the BOND RX to spatially visualize RNA and protein markers on the same slide. We utilized the new RNAscope™ Multiomic LS assay that can detect up to 6 RNA and protein targets on the same slide. The TSA-based amplification strategy offers signal boost for both RNA and protein targets. To optimize protein detection, the workflow is completely protease-free there by preserving antigen integrity and tissue morphology. The assay can be customized to include any target RNA and proteins of interest. Here, we demonstrate the use of our pre-conjugated antibody panel which includes CD8, CD4, FoxP3 and PanCK to visualize tumor infiltrating lymphocytes (TILs). We also utilized unconjugated primary antibodies for CD68 and CD163 to detect tumor macrophages in multiple tumor tissues. Using the HALO® image analysis software from Indica Labs, we performed RNA and protein quantification as well as cell phenotyping and spatial analysis. Infiltrating Cytotoxic T cell lymphocytes were detected using CD8, GZMB and IFNG co-expression. Regulatory T cells were detected using CD4, FOXP3 co-expression. PanCK was used as a tumor marker to delineate tumor from stroma. Similarly, tumor associated macrophages were detected using CD163, CD68 , IL10 and IL-1B expression. We were also able to identify distinct M1 and M2 macrophage populations in the tumor samples. The protease-free RNAscope Multiomic LS assay was successfully able to interrogate different subpopulation of immune cells with high fidelity RNA and protein signal while maintaining tissue morphology. The higher plex capability provides detection of more markers simultaneously, in turn enabling immune phenotype profiling while maintaining the sensitivity and specificity. Finally, the assay affords a very high throughput on the BOND Rx allowing large-scale pre-clinical and patient sample analysis.


 Debia Wakhloo, Julia Yu, Sonali Deshpande, Donald Allen, Anne Hellebust, Anushka Dikshit. Identifying immune cell phenotypes and their function by using high throughput spatial multiomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5318.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7803e29b219e38d64e2cdad32375f3e68aa64a3" target='_blank'>
              Abstract 5318: Identifying immune cell phenotypes and their function by using high throughput spatial multiomics
              </a>
            </td>
          <td>
            Debia Wakhloo, Julia S. Yu, Sonali A Deshpande, Donald Allen, Anne Hellebust, A. Dikshit
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Microwave thermochemotherapy (MTC) has been applied to treat lip squamous cell carcinoma (LSCC), but a deeper understanding of its therapeutic mechanisms and molecular biology is needed. To address this, we used single-cell transcriptomics (scRNA-seq) and spatial transcriptomics (ST) to highlight the pivotal role of tumor-associated neutrophils (TANs) among tumor-infiltrating immune cells and their therapeutic response to MTC. MNDA+ TANs with anti-tumor activity (N1-phenotype) are found to be abundantly infiltrated by MTC with benefit of increased blood perfusion, and these TANs are characterized by enhanced cytotoxicity, ameliorated hypoxia, and upregulated IL1B, activating T&NK cells and fibroblasts via IL1B-IL1R. In this highly anti-tumor immunogenic and hypoxia-reversed microenvironment under MTC, fibroblasts accumulated in the tumor front (TF) can recruit N1-TANs via CXCL2-CXCR2 and clear N2-TANs (pro-tumor phenotype) via CXCL12-CXCR4, which results in the aggregation of N1-TANs and extracellular matrix (ECM) deposition. In addition, we construct an N1-TANs marker, MX2, which positively correlates with better prognosis in LSCC patients, and employ deep learning techniques to predict expression of MX2 from hematoxylin-eosin (H&E)-stained images so as to conveniently guide decision making in clinical practice. Collectively, our findings demonstrate that the N1-TANs/fibroblasts defense wall formed in response to MTC effectively combat LSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2560362b63b6e3eb203f753fb2eef7cc7ed8926f" target='_blank'>
              Single-cell and spatial transcriptomics reveals an anti-tumor neutrophil subgroup in microwave thermochemotherapy-treated lip cancer
              </a>
            </td>
          <td>
            Bingjun Chen, Huayang Fan, Xin Pang, Zeliang Shen, Rui Gao, Haofan Wang, Zhenwei Yu, Tianjiao Li, Mao Li, Yaling Tang, Xinhua Liang
          </td>
          <td>2025-05-13</td>
          <td>International Journal of Oral Science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Intratumoral heterogeneity of breast cancer cells causes undesired drug resistance and predispose to disease recurrence. We investigate the molecular heterogeneity of breast cancer cells derived from malignant pleural effusions (MPE) to understand variations in drug resistance and cellular evolution. MPE provides a unique environment, with cells experiencing significant microenvironmental changes that promote intratumoral heterogeneity and therapeutic resistance. By establishing 24 cell lines and 3 organoids from MPE samples of breast cancer patients, we performed extensive genomic, transcriptomic, and drug sensitivity analyses. Our findings reveal substantial genetic and transcriptomic diversity among MPE-derived cell lines, highlighting dynamic alterations in driver mutations and signaling pathways that correlate with variable drug responses. Notably, temporal and spatial heterogeneity within patient-derived samples emphasized the adaptability of breast cancer cells in MPE, as different subclones displayed distinct drug sensitivities. This work underscores the critical role of molecular profiling in understanding treatment resistance in breast cancer, presenting MPE-derived cell lines as valuable preclinical models for exploring personalized therapies in aggressive cancer phenotypes. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02032-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/650f36fd9412d209953690565be430355d5470f8" target='_blank'>
              Establishment and characterization of 24 breast cancer cell lines and 3 breast cancer organoids reveals molecular heterogeneity and drug response variability in malignant pleural effusion-derived models
              </a>
            </td>
          <td>
            Soon-Chan Kim, Ga-Hye Kim, Jae-Hyeon Park, Kyung-Hun Lee, Jiwon Koh, Tae-Yong Kim, Dae-won Lee, Yu-Jin Kim, Seongyeong Kim, Song-Yi Park, A. Min, Y. Shin, Seock-Ah Im, Ja-Lok Ku
          </td>
          <td>2025-05-01</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Immune checkpoint inhibitor (ICI) therapy has demonstrated limited efficacy in MASLD/MASH-associated hepatocellular carcinoma (HCC), suggesting distinct anti-tumor immune responses within the tumor microenvironment (TME) of this cancer type compared to other cancers. Understanding the mechanisms driving MASLD/MASH-HCC progression and identifying novel therapeutic targets are therefore critical. While ICI therapy alone may not be sufficient, combining it with drugs targeting key cancer progression pathways may hold promising enhancement of anti-tumor efficacy in this context. In this study, we sought to elucidate the tumor-promoting pathways in MASLD/MASH-associated HCC by characterizing the comprehensive transcriptomic landscape of cancer cells and stromal cells within the tumor microenvironment (TME). We conducted single-cell RNA sequencing on the samples from 31 HCC patients and 8 healthy donors. Using this data, we classified the samples based on the cellular composition of the TME and performed interactome analysis with CellChat version 2. This analysis stratified the samples into five distinct groups: Group 1, TME with stem cell-like cancer cells and immune cell infiltration; Group 2, TME with Wnt signaling-activated cancer cells regardless of immune cell infiltration; Group 3, TME containing cancer cells not classified in the categories mentioned above with immune cell infiltration; Group 4, TME containing cancer cells not classified in the categories mentioned above without immune cell infiltration; and Group 5, TME with normal hepatocytes, as observed in healthy donor livers. Detailed analysis highlights several notable features of the tumor stroma. For example, “senescent” cancer-associated fibroblasts (CAFs) exhibiting a senescence-associated secretory phenotype (SASP) are frequently observed in the TMEs of Groups 1-3. Interactome analysis revealed a VEGFA-VEGFR2 interaction between endothelial cells and various other cell types, consistent with the hypervascularity characteristic of HCCs and supporting the clinical efficacy of VEGF inhibitors. Furthermore, group-specific interactome analysis uncovered distinct novel interactions, indicating molecular network variations among TME groups. We will present spatial distributions of each cell type in TME and TME-dependent inter-cellular interactions. We discuss their potential as biomarkers and therapeutic targets for MASLD/MASH-HCC treatment.


 Naoko Ohtani, Yoshiki Nonaka, Kanae Echizen, Yi Cheng, Takahiro Kodama. Single-cell-based analysis of tumor microenvironment heterogeneity in human MASLD/MASH-associated HCC: Classifications by inter-cellular network variations [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5309.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/938952a283824cbae8acde9bd23dfb3cd1a44259" target='_blank'>
              Abstract 5309: Single-cell-based analysis of tumor microenvironment heterogeneity in human MASLD/MASH-associated HCC: Classifications by inter-cellular network variations
              </a>
            </td>
          <td>
            Naoko Ohtani, Yoshiki Nonaka, Kanae Echizen, Yi Cheng, Takahiro Kodama
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Target heterogeneity and intensity of expression in patient tumors are two of many critical factors in understanding the clinical potential of antibody drug conjugates (ADCs) and T cell engagers (TCEs) in oncology.1, 2 Despite extensive clinical usage, conventional immunohistochemistry (IHC) is often restricted to a handful of markers per section, limiting its capacity to identify complex patterns of antigen expression in patient samples. In contrast, high-parameter, spatially resolved, proteomic analysis has the potential to not only provide widespread characterization of tumor-associated antigens (TAAs) but also to reveal impactful cell-cell and protein-protein neighborhoods for bispecific pairings and/or novel biomarkers.
 Here, we utilized imaging mass cytometry (IMC) complemented by multiplex immunofluorescence (mIF) to spatially evaluate the expression of greater than 30 protein markers simultaneously in 35 tumor specimens from 7 indications. Using whole slide imaging and a bespoke analysis pipeline, coupled with statistical modeling, we profiled multiple TAAs, including HER2, TROP2, B7-H3, and B7-H4 for expression in tumor and adjacent non-tumor tissues.3 Notably, whole-slide mIF was used in tandem with IMC on the same tissue section to enable both region of interest (ROI) selection and image co-registration. With this approach, we observed a high level of concordance between the two modalities for markers of interest such as B7-H3 and B7-H4, helping to further validate our findings. Stromal and immune markers such as SMA (cancer-associated fibroblasts [CAFs, ]) CD3/CD8/FoxP3 (T cells, ) and CD206 (macrophages) were also included to help characterize the tumor microenvironment and cellular distribution of each TAA. The panel was also incorporated with markers of sensitivity to common ADC payloads, such as SLFN11 and TOP1. In addition to differences in target prevalence and level of expression, we describe the spatial relationship between various TAAs across tumors including such parameters as the average distance between cytotoxic T cells and a given TAA+ cell.
 Altogether, these results provide a framework to apply dual-modality imaging of mIF and IMC on the same tumor samples for comprehensive characterization of TCE and ADC targets. In the future, the total number of markers which can be characterized by both mIF and IMC are expected to increase, allowing for an even deeper characterization of tumor biology which may inform bispecific pairings and more precise treatment strategies for ADCs and TCEs.
 References:
 1 - Cancers(Basel). 2023 Mar 19; 15(6):1845. doi: 10.3390/cancers15061845 2 - Cancers(Basel). 2020 Apr 26; 12(5):1075. doi: 10.3390/cancers12051075 3 - Cell Rep Methods. 2023 Oct 23;3(10):100595. doi: 10.1016/j.crmeth.2023.100595


 Michael Alan Conner, Fang Xie, Lijun Zhou, Eliot McKinley, Jimmy Tarrant, Takahiro Sato, Ya-Fang Chang, Derek Poore, Shannon McKearnan, Ken Hance, Jeremy Waight. Characterization of ADC and TCE targets in patient tumors using high-parameter spatial proteomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5311.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960ea9a4f2e5765b1208ba1d0b8da13c01720033" target='_blank'>
              Abstract 5311: Characterization of ADC and TCE targets in patient tumors using high-parameter spatial proteomics
              </a>
            </td>
          <td>
            Michael Alan Conner, Fang Xie, Lijun Zhou, Eliot McKinley, Jimmy Tarrant, Takahiro Sato, Ya-Fang Chang, Derek Poore, Shannon McKearnan, Ken Hance, Jeremy Waight
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d748941bf68f55dce06ac739e5c0e7d9fe6de07" target='_blank'>
              A gel-top model for characterizing mesenchymal features of glioblastoma cells
              </a>
            </td>
          <td>
            Yuanzhong Pan, Priscilla Chan, Jeremy N. Rich, Steve A. Kay, Jinseok Park
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Spatial Multiomics allows for context-based study of the tumor microenvironment (TME). TME consists of diverse cells that collectively influence tumor progression, therapeutic response, and immune activation. CD8 T-cells, essential for antitumor immunity, often become exhausted in the TME. Recent studies show that PD1+ TCF1+ stem-like CD8 T-cells retain differentiation potential and can be rejuvenated through cytokine signaling and immune checkpoint modulation. This detailed spatial profiling of RNA and protein expression in the TME is crucial to understanding immune cell function, activation, and differentiation. Here we introduce a protease-free workflow for RNAscope™ Multiplex Fluorescent v2 assay combining RNA in-situ hybridization (ISH) and protein immunofluorescence to study cell marker proteins (CD8, PD1) with RNA targets (TNFA, TCF7, IFNG) in a tumor microarray (TMA). The Manual Multiplex Protease-Free workflow enables simultaneous RNA and protein visualization without enzymatic disruption, preserving antigen integrity and tissue morphology. Using this Multiomics assay we can study CD8 T-cell differentiation and activation. Co-detection of TNFA, TCF7, and IFNG transcripts with CD8 and PD1 proteins provides insights into specific T cell phenotypes and their activation/exhaustion status. Data reveal robust co-expression patterns of these targets, particularly in the tumor-cell neighboring stromal and immune cells in breast, cervical and stomach cancers, reflecting immune activation and exhaustion across distinct TME niches. This methodology lays a foundation for refining immunotherapeutic strategies targeting PD1+ TCF7+ CD8 T-cells, enhancing effector functions, and advancing precision oncology interventions.


 Steve Zhou, Yifan Wang, Debia Wakhloo, Anushka Dikshit, Li-Chong Wang, Maithreyan Srinivasan. Spatial multiomics reveals T-cell activation and exhaustion states in the tumor microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6471.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b35fc24142fa9e46cb8827397a0b39028b4c9b9" target='_blank'>
              Abstract 6471: Spatial multiomics reveals T-cell activation and exhaustion states in the tumor microenvironment
              </a>
            </td>
          <td>
            Steve Zhou, Yifan Wang, Debia Wakhloo, A. Dikshit, Li-chong Wang, Maithreyan Srinivasan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: Disease modeling is vital for our understanding of disease mechanisms and for developing new therapeutic strategies. Accurately modeling the intact tumor microenvironment (TME) is increasingly recognized as essential for gaining insights into cancer biology and therapeutic response. Preclinical mouse models have provided utility for studying the evolving TME, but these models are costly and can lead to animal suffering and the discontinuation of drug investigations. To address these limitations, particularly in hepatocellular carcinoma (﻿HCC), we have developed an ex vivo model using tumor precision-cut slices (TPCS) derived from orthotopic liver tumors. Methods: Murine HCC tumors were generated via intrahepatic injection of Hep-53.4 cells, providing a source of tumor tissue for TPCS generation. Subsequent scaling to a 96-well format and modification to include a secreted luciferase enabled longitudinal ex vivo screening of 26 drugs applied at 2 doses over an 8-day period, using just 5 tumors. One drug identified in the screen, salinomycin, was then validated in vivo via intraperitoneal injection of mice with orthotopic liver tumors. Results: Histological characterization determined that TPCS maintain the architecture, cellular complexity, and drug responsiveness of the original HCC-TME under simplified culture conditions that preserve viability and metabolic activity. In addition to typical HCC therapies, sorafenib and anti-PD1 immunotherapy, the screen identified 2 drugs as potent anticancer agents capable of impacting the viability of TPCS: salinomycin and rottlerin. Salinomycin was further validated in vivo, significantly reducing tumor burden without evidence of toxicity. Conclusions: We present a 3Rs (Reduction, Refinement, Replacement) approach for studying HCC biology and performing 96-well-scale drug screening within an intact, metabolically active TME, offering a more ethical and effective platform for drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0fc9ab2a0c87013cf6a5b53d9f8f813f5001d3b" target='_blank'>
              Precision-cut tumor slices for modeling hepatocellular carcinoma enable at-scale drug screening
              </a>
            </td>
          <td>
            Amy L. Collins, Keara Kirkness, Erik Ramon‐Gil, Eleni Tzortzopoulou, D. Geh, Jack Dishington, Eleanor Graham, Rhys Muir, Rainie Cameron, Saimir Luli, Eman Khurram, Daniel Storey, Hannah L. Paish, Glyn Nelson, David McDonald, Andrew Filby, Lee A Borthwick, Fiona Oakley, Derek A Mann, J. Leslie
          </td>
          <td>2025-05-16</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Emerging spatial profiling technologies have revolutionized our understanding of how tissue architecture shapes disease progression, yet the contribution of cellular diversity remains underexplored. Here, Ding and colleagues introduce multiomics and ecological spatial analysis (MESA), an ecology-inspired framework that integrates spatial and single-cell expression data to quantify tissue diversity across multiple scales. MESA both identifies distinct cellular neighborhoods and computes a variety of diversity metrics alongside the identification of diversity "hotspots". Applied to human tonsil tissue, MESA revealed previously undetected germinal center organization, while in spleen tissue of a murine lupus model, MESA highlights increasing cellular diversity with disease progression. Importantly, diversity hotspots do not correspond to conventional compartments identified by existing methods, presenting an orthogonal metric of spatial organization. In colorectal cancer, MESA's diversity metrics outperformed established subtypes at predicting patient survival, while in hepatocellular carcinoma, multi-omic integration identified significantly more ligand-receptor interactions between immune cells compared to single-modality analysis. This work establishes cellular diversity within tissues as a critical correlate of disease progression and underscores the value of multi-omic integration in spatial biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eb774cd8ef924c70b07f117ccf0af3bd80ea8f6" target='_blank'>
              Cells keep diverse company in diseased tissues.
              </a>
            </td>
          <td>
            Kieran R. Campbell, A. Goeva
          </td>
          <td>2025-05-16</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) has the highest mortality rate among major cancers, with over 50% of patients having liver metastases at diagnosis. Given that conventional T-cell based immunotherapies have been ineffective against PDAC liver metastases, it is vital to elucidate the mechanisms regulating tumor immunity within the hepatic microenvironment to enhance anti-tumor T cell. Notably, PDAC cells exhibit exceptional plasticity and intrinsic heterogeneity, contributing to the evolution of immunosuppressive primary and early liver PDAC metastases. Our study leverages an array of congenic PDAC clones with varying degrees of immunogenicity to delineate the signaling pathways driving primary and liver immunosuppression.



 We investigate the growth kinetics of PDAC clones both in vitro and in vivo, alongside their transcriptomic variations. Using model antigens (chicken ovalbumin), we assess how different clones modulate the immune microenvironment to elicit antigen-specific adaptive anti-tumor responses. Transcriptomic profiling informs CRISPR-based knockout and overexpression strategies to uncover intrinsic gene contributions to primary anti-tumor immunity in vivo. Additionally, we have developed stable PDAC liver metastasis models through portal vein injection to evaluate each clone’s metastatic efficiency and associated immune landscape within the hepatic microenvironment.



 In vivo engraftment of PDAC clones revealed differential infiltration of CD4+/CD8+ T cells and neutrophils. While all OVA+ PDAC clones triggered antigen-specific immune responses, only immune-sensitive tumors were eradicated, whereas immune-insensitive tumors persisted. Despite OVA expression, all clones successfully established liver metastases following intraportal injection. RNA-seq analysis identified upregulation of key pancreatic development regulators, including PDX1 and RBPJ, in immune-insensitive clones. CRISPR knockout of these transcription factors in vivo demonstrated altered immune cell composition and reduced tumor growth in immunocompetent mice expressing OVA.



 Using diverse PDAC models, we created tumors with immune heterogeneity and altered transcriptional profiles. We identified validated immune-altering cytokines (CXCL1 and CSF3) and immunomodulatory roles of PDX1 and RBPJ. Immune-insensitive clones evaded antigen-specific immunity in both subcutaneous and liver metastasis models, while immune-sensitive clones were eradicated subcutaneously but gained protection in the liver. Ongoing studies confirm the phenotypes of CRISPR-KO clones and delineate the immunomodulatory roles of these transcription factors in liver metastasis.



 Bushangqing Liu, Ethan Agritelley, Junping Wei, Gangjun Lei, Tao Wang, Jason McBane, Daniel Nussbaum, Zachary Hartman. Decoding the role of intrinsic pancreatic cellular signaling in shaping the immunosuppressive tumor microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 908.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ed535cc83be9bb714c00d9bb8580fac8efc104e" target='_blank'>
              Abstract 908: Decoding the role of intrinsic pancreatic cellular signaling in shaping the immunosuppressive tumor microenvironment
              </a>
            </td>
          <td>
            Bushangqing Liu, Ethan S Agritelley, Junping Wei, Gangjun Lei, Tao Wang, Jason McBane, Daniel Nussbaum, Z. Hartman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822c59156ea3b1fc22260f9e8a6f1db29bc4fc3a" target='_blank'>
              Pancreatic cancer patient-derived organoids capture therapy response and tumor evolution
              </a>
            </td>
          <td>
            J. Gout, L. Perkhofer, Jessica Lindenmayer, Y. J. Resheq, J. Schwab, J. Kraus, E. Roger, Dharini Srinivasan, E. Zimmer, B. Mayer, Ralf Marienfeld, N. Gaisa, Julia K. Kühlwein, Mark Hänle, T. Ettrich, K. Danzer, Alica K Beutel, Bo Kong, Christoph Michalski, André L Mihaljević, N. Rahbari, Marko Kornmann, Hans A. Kestler, Thomas Seufferlein, Alexander Kleger
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Characterization of the tumor immune microenvironment (TIME) of pituitary neuroendocrine tumors (PitNETs) is crucial for understanding the behavior of different types of PitNETs and identification of possible causes of their aggressiveness, rapid growth, and resistance to therapy. High‐dimensional flow cytometry (FC) is a promising technology for studying TIME but poses unique technical challenges, especially when applied to solid tissues and PitNETs, in particular. This paper evaluates the potential of FC for analyzing TIME in PitNETs by addressing methodological difficulties across all stages of the workflow and proposing solutions. We developed a protocol for preparing single‐cell suspensions from PitNET tissues for FC. This involved optimization of enzymatic digestion and comparison of it with mechanical tissue dissociation assessing cell yield, viability, and target antigen expression. We designed four multicolor FC panels to analyze major lymphocyte and myeloid cell subsets including determination of subpopulations of T, B, NK cells and their activation and cytotoxic potential, neutrophils, monocytes, CD68 + CD64 + CD11blow macrophages of M2 and M1 subtypes, and two types of myeloid suppressor cells ‐ PMN‐MDSC and M‐MDSC. Principles of multicolor panel design, spreading error, and importance of voltage balance for proper flow cytometer setting are discussed. The panels were validated and demonstrated the feasibility of their simultaneous use on pituitary tumor surgical tissue for comprehensive TIME characterization. We compared lymphocyte frequencies in blood, PitNETs, and three sequential PitNET eluates to find out the contamination level of PitNET samples with blood leukocytes. To address technical challenges, we propose a strategy of logical data gating that removes spurious signals from aggregates, dead cells, and subcellular debris that can interfere with analysis. Our results indicate that despite all technical difficulties, multiparametric FC can effectively characterize different types of PitNETs. This enhanced understanding of the immune infiltrate provides valuable insights into PitNET biology and advances clinical diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2da518e1bcc8583485db353efda083c10dbac39e" target='_blank'>
              Methodology of high‐dimensional flow cytometry in monitoring immune microenvironment of pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Marina Loguinova, Valeria V Mazeeva, Daria V Lisina, Elena N. Zakharova, Alyona V Sorokina, L. Dzhemileva, Andrei Yu Grigoriev, Alexandra Shutova, Ekaterina A Pigarova, Larisa K Dzeranova, Galina A. Melnichenko, Natalia G Mokrysheva, Sergei A Rumiantsev, Vladimir P Chekhonin
          </td>
          <td>2025-04-13</td>
          <td>Cytometry Part B: Clinical Cytometry</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="
 Cancer immunotherapy is a promising treatment for many forms of cancer, however only a small subset of patients benefit, and relapse/recurrence frequently occurs due to various resistance mechanisms. To overcome this challenge combinatorial therapeutic strategies are needed targeting multiple steps of the cancer-immunity cycle to achieve long-term efficacy in a large population of cancer patients. It is now known that DNA damage activates the innate immune response and improves immune checkpoint blockade (ICB) efficacy. Combination immunotherapy can target several immunosuppressive elements in the tumor microenvironment and activate multiple steps of the cancer-immunity cycle. For developing an effective combination treatment strategy, including ICB with various other therapeutic modalities, we need to first understand the diverse expression pattern of various factors within tumor immune microenvironment. Multiplex immunofluorescence (mIF) is an important tool for profiling protein expression patterns in tumor tissues. This can provide a framework to understand the sensitivity to a particular combination of therapies and identify potential predictive biomarkers. Here we show spatial proteomics profiling of a diverse range of biomarkers in human colon adenocarcinoma tissue. Our findings demonstrate that spatial proteomics can be tailored to precision oncology using artificial intelligence to predict a patient’s response to combination treatment strategies. The Cell DIVE Multiplex Imaging Solution in combination with AI-based image processing software, AIVIA delivers a comprehensive solution for exploring immune landscape of tumor tissue. The broad portfolio of robust IHC- validated antibodies from Cell Signaling Technology (CST) enables the detection of key proteins in the tumor microenvironment (TME). Here, we demonstrate multiplexed Cell DIVE imaging using a novel CST panel to probe human colon adenocarcinoma. The availability of cell type specific biomarkers, combined with the ability to interrogate using multiplexed tissue imaging, provides unprecedented and novel insights and spatial resolution of diverse cell populations in the TME.


 Arindam Bose, Susan Keezer, Emily Quann Alonzo, Sophie Struble, Jeremy Fisher, Matthew Norton, Richard A. Heil-Chapdelaine, Natasha F. Diaz Granados, Vasundhara Agrawal. Spatial mapping of the tumor microenvironment for combination therapy design [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6464.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258df61f2a4c963aeb2c62a2f4585ccce0f62b1b" target='_blank'>
              Abstract 6464: Spatial mapping of the tumor microenvironment for combination therapy design
              </a>
            </td>
          <td>
            Arindam Bose, Susan Keezer, E. Alonzo, Sophie Struble, Jeremy Fisher, Matthew Norton, R. Heil-Chapdelaine, Natasha F. Diaz Granados, Vasundhara Agrawal
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Glioblastoma (GBM), or grade 4 glioma, is the most common and aggressive primary brain tumor in adults with a median survival of 15 months. Increasing evidence suggests that GBM’s aggressiveness, invasiveness, and therapy resistance are driven by glioma stem cells (GSCs), a subpopulation of tumor cells that share molecular and functional characteristics with neural stem cells (NSCs). GSCs are heterogeneous and highly plastic. They evade conventional treatments by shifting their state and entering in quiescence, where they become metabolically inactive and resistant to radiotherapy and chemotherapy. GSCs can exit quiescence and be reactivated to divide into highly proliferative tumor cells which contributes to recurrence. Understanding the molecular mechanisms regulating the biology of GSCs, their plasticity, and the switch between quiescence and mitotic activity is essential to shape new therapeutic strategies. This review examines the latest evidence on GSC biology, their role in glioblastoma progression and recurrence, emerging therapeutic approaches aimed at disrupting their proliferation and survival, and the mechanisms underlying their resistance to therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1418264d0938e38785a30378512e83f1e451aa8" target='_blank'>
              Targeting Glioma Stem Cells: Therapeutic Opportunities and Challenges
              </a>
            </td>
          <td>
            Asma Mahdi, Mohamed Aittaleb, Fadel Tissir
          </td>
          <td>2025-05-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We find that the KPC model harbors a basal state, which harbors transcriptional signature that aligns with the “basal” signature in human PDAC and is linked to poor prognosis and chemoresistance. The basal state co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. To functionally interrogate the basal state in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the basal state has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the basal state is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the basal state over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the basal state is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the basal state. Taken together, our data indicates that PDAC harbors a striking dependency on the basal state and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.


 Anupriya Singhal, Hannah Styers, Jonathan Rub, Kate Ryan, Jung Yun Kim, Zeynep C. Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional dissection of the highly plastic basal cell state in pancreatic cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3880.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9d10ba962b31f50d67da52456440cdb7f8eb0c" target='_blank'>
              Abstract 3880: Functional dissection of the highly plastic basal cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Kate Ryan, Jung Yun Kim, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G. Christensen, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="


 The efficacy of treating colorectal cancer (CRC) liver metastases is hindered by significant tumor heterogeneity, both within and between tumors. This diversity stems from varied cell populations and complex interactions in the tumor microenvironment, necessitating a deeper understanding of spatial organization and cellular communication patterns.



 We examined 23 formalin-fixed paraffin-embedded tissue samples from 11 patients, comprising primary lesions and matched liver metastasis (6 with synchronous and 5 with metachronous metastases) using NanoString 6K CosMx Spatial Molecular Imaging (SMI). Pathologists selected optimal fields of view at tumor invasive fronts. Image processing, cell segmentation, and feature extraction utilized an in-house CosMx SMI pipeline with the Cellpose algorithm. Following quality control, the remaining cells underwent unsupervised clustering with differential gene expression analysis and cell-to-cell interaction analyses performed using the Seurat R package.



 Analysis of 105, 624 high-quality cells revealed 23 distinct clusters, categorized as ten epithelial tumors, three cancer-associated fibroblast (CAF), two endothelial cells, five immune cells (T cell, Monocyte, Macrophage, SPP1+ Macrophage, and Plasma cell), alveolar cells, hepatocytes, and low complexity cell populations. We identified significant differences in gene expression between colon and liver CAFs, with liver CAFs displaying upregulation of 92 genes associated with epithelial-mesenchymal transition (EMT). Enhanced communication between CAFs, T cells, and macrophages was observed in liver metastases, with specific ligand-receptor pairs (SPP1-ITGA4/ITGB1, LGALS9-P4HB, LGALS9-CD44, CXCL12-CXCR4) enriched in the metastatic microenvironment.



 Our findings demonstrate distinct spatial heterogeneity and intercellular communication networks in CRC liver metastases. The identification of liver-specific CAF gene expression profiles and unique ligand-receptor interactions provides new insights into metastatic mechanisms and potential therapeutic targets.



 Jongwon Lee, Yeseul Kim, Hyo Seon Ryu, Jongmin Sim, Chungyeul Kim, Hyun Woo Kwon, Hyun Je Kim, Jong Min Park, Ah-Reum Lim, Jung Sun Kim, Hwa Jung Sung, Xingyi Guo, Jungmin Choi, Jungyoon Choi. Mechanisms of metastasis revealed through single-cell spatial transcriptome analysis in patients with liver metastatic colorectal cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5289.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78e3d7344b2788e380616083356db8cc35b78722" target='_blank'>
              Abstract 5289: Mechanisms of metastasis revealed through single-cell spatial transcriptome analysis in patients with liver metastatic colorectal cancer
              </a>
            </td>
          <td>
            Jongwon Lee, Yeseul Kim, Hyo Seon Ryu, Jongmin Sim, Chungyeul Kim, Hyun Woo Kwon, Hyun Je Kim, Jong Min Park, Ah-reum Lim, Jung Sun Kim, Hwa Jung Sung, Xingyi Guo, Jungmin Choi, Jungyoon Choi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Recurrence and metastasis following curative treatment remain significant challenges in colorectal cancer (CRC). Circulating tumor cells (CTCs) are pivotal in the dissemination of cancer, leading to distant metastases and eventual recurrence. Despite their importance, the cellular lineages driving these processes in CRC remain poorly defined. POU5F1, a transcription factor associated with stemness, is highly expressed in recurrent CRC tumors and is linked to poor prognosis and a high incidence of liver metastasis.



 We developed a patient-derived cancer cell model, termed isolated tumor-derived cancer cells, cultured as two-dimensional organoids (2DOs). These 2DOs retain tumor heterogeneity and tumor microenvironment interactions, providing a robust platform for investigating treatment-resistant lineages. Using EGFP-reporter 2DOs, POU5F1-expressing cells were isolated and characterized through drug sensitivity assays, single-cell RNA sequencing, immunocytochemistry, and epigenetic analyses. Therapeutic studies were conducted to evaluate the efficacy of targeting key pathways.



 POU5F1-expressing cells were highly enriched in blood samples from relapsed CRC patients as CTCs. These cells exhibited cancer stem cell properties, including self-renewal and differentiation, and demonstrated a high capacity for liver metastasis in vivo. Single-cell RNA sequencing revealed that POU5F1-positive cells generate heterogeneous populations enriched in the Wnt signaling pathway. Immunocytochemistry confirmed the co-expression of POU5F1 and CTLA4, with epigenetic analysis showing demethylation at transcriptional start sites of both genes, implicating epigenetic regulation in their aggressiveness. The Wnt/β-catenin pathway inhibitor XAV939 effectively suppressed adhesion and survival of POU5F1-positive cells in vitro. Remarkably, early administration of XAV939 completely inhibited liver metastasis induced by these cells in vivo.



 POU5F1-expressing cells are critical drivers of CRC metastasis and recurrence, exhibiting stem-like properties and treatment resistance. Their enrichment in Wnt signaling and CTLA4 expression identifies actionable therapeutic targets. Our findings highlight the potential of Wnt/β-catenin inhibitors like XAV939 to prevent CRC metastasis and recurrence, paving the way for improved therapeutic strategies in high-risk patients.



 Shiki Fujino, Aya Ito, Rie Hayashi, Masayoshi Yasui, Masayuki Ohue, Norikatsu Miyoshi. Targeting POU5F1-expressing circulating tumor cells to prevent metastasis and recurrence in colorectal cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 87.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3bfe2b136df9484a23fdf6a59f1b40397fdc9bc" target='_blank'>
              Abstract 87: Targeting POU5F1-expressing circulating tumor cells to prevent metastasis and recurrence in colorectal cancer
              </a>
            </td>
          <td>
            S. Fujino, Aya Ito, Rie Hayashi, M. Yasui, M. Ohue, N. Miyoshi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The study of circulating tumor cells (CTCs) provides critical insights into the biological mechanisms underlying metastasis. This study aims to demonstrate the applicability of an integrated DEPArray-based phenotypic analysis combined with transcriptomic characterization to investigate the biology of CTCs isolated from 10 patients with metastatic breast cancer (MBC). The transcriptional profiles of CTCs were consistent with both the cancer type and epithelial characteristics. Gene set enrichment analysis identified pathways associated with synapse organization and calcium channel activity. Furthermore, distinct gene expression profiles were linked to specific metastatic sites, particularly bone metastases. We also report a rare and understudied population of CTCs, characterized by the co-expression of epithelial and leukocyte markers, observed exclusively in patient-derived samples and not in blood samples from healthy volunteers spiked with SKBR-3 and MCF-7 cell lines. This suggests that these double-positive CTCs (dpCTCs) may have a specific role in the metastatic process. The transcriptomic characterization of these rare CTCs enhances our understanding of their biology and potential involvement in metastasis. As a pilot study, our findings underscore the potential of CTC-based transcriptomics as a valuable tool for advancing clinical and biological understanding of MBC, particularly regarding metastatic mechanisms and organotropism. Moreover, it provides preliminary insights into dpCTCs, a poorly characterized population that may play a pivotal role in metastasis but remains largely unexplored. These findings could pave the way for developing targeted therapies aimed at the pathways driving metastasis and for improving patient monitoring through CTC profiling. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00659-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5c8d5b6cac842e5c06154018a6fe136a8682dd8" target='_blank'>
              Transcriptomic profiling of circulating tumor cells from metastatic breast cancer patients reveals new hints in their biological features and phenotypic heterogeneity
              </a>
            </td>
          <td>
            T. Rossi, Sara Bandini, Michele Zanoni, M. Cortesi, M. Palleschi, E. Bandini, Andrea Rocca, Giulia Gallerani, Ivan Vannini, Meropi Plousiou, L. Gerratana, A. Musolino, Giovanni Tallini, Giovanni Martinelli, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-05-06</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 Colorectal cancer is the second most fatal cancer worldwide, accounting for 9% of all reported cancer deaths, and is the number one cause of death among people under the age of 50. Rapid technical advances in the field of spatial biology are allowing us to gain new insight into colorectal cancer biology and patient heterogeneity, which can help to guide the development of immunotherapies. Using a combination of modalities such as sequential multiplex immunofluorescence (mIF) and spatial transcriptomics (STx) on a colon cancer sample we determined the spatial distribution, phenotype, function, and a cytokine and chemokine profile within the colon cancer tissue.



 5 um FFPE sections were prepared from a colon cancer sample exhibiting moderate to poorly differentiated adenocarcinoma of T3, N0 staging. mIF was performed using Lunaphore COMET with a 20-antibody panel, followed by H&E staining and imaging. An adjacent section was used for STx with the 10x Genomics Visium CytAssist platform. Data was analyzed with Lunaphore Horizon®, 10x Genomics Loupe Browser software and PredxBio SpaceIQ™ unbiased cell typing, microdomain discovery and network biology platform that co-analyzes mIF and STx datasets.



 The tumor showed an aberrant growth pattern and a highly proliferated crypt region. Three distinct microdomains in the tumor tissue were discovered: one with high levels of PD-L1+ cells, a second with high levels of NK cells and a third with low levels of infiltrating cells in the lumen and crypt. There was significant presence of VISTA+ and IDO1+ cells throughout the lumen. These cells interacted with CD4+ and CD8+ cells which were found to be present in both lumen and crypt regions. PD-L1 was prominent in the cells found in both lumen and crypt. Finally, NK cells were abundant throughout the lumen and in regions adjacent to the crypt.



 Using multimodal spatial imaging and spatial co-analysis of mIF and STx data, we were able to elucidate relationships between different cell types in a complex tumor tissue and assign context to their activities. Despite high levels of NK cells, indicative of a possible active anti-tumor immune response, the tumor has developed an immune evasion mechanism due to the presence of cells expressing high levels of the anti-tumor immune response markers such as PD-L1, IDO1, and VISTA. Based on these results, therapies targeting more than one immune checkpoint protein may yield improved response due to an abundance of the existing NK cells. The transcriptomic profiling reveals a complex relationship between various cytokines secreted by different cell populations which alter the landscape of the tumor microenvironment, facilitating the emergence of an immunosuppressive phenotype.



 Gaurav Joshi, Nicholas Sciascia, Michael Yang, Brian Falkenstein, Raymond Yan, Shannon Quinn, Brenna Fearey, Junya Yoshioka, Arif Burak Tosun, S. Chakra Chennubhotla, Filippo Pullara, Fumiki Yanagawa. Multimodal spatial analysis of colon cancer tissue reveals emergence of an immunosuppressive tumor maintenance mechanism [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5317.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db705f60f289f122950733aa7588b5a6d8c6c985" target='_blank'>
              Abstract 5317: Multimodal spatial analysis of colon cancer tissue reveals emergence of an immunosuppressive tumor maintenance mechanism
              </a>
            </td>
          <td>
            Gaurav N Joshi, Nicholas Sciascia, Michael T Yang, B. Falkenstein, R. Yan, S. Quinn, Brenna Fearey, Junya Yoshioka, Arif Burak Tosun, S. Chennubhotla, F. Pullara, Fumiki Yanagawa
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Intermediate-risk prostate cancer (PCa) poses a challenge for treatment decisions, made more complex by its heterogeneous nature. Growing evidence highlights the critical role of the tumor microenvironment (TME), in cancer initiation, progression, and response to treatment, underscoring the importance of profiling cancer cells but also the TME within their spatial context. Here, we present single-cell in situ transcriptomic and spatial proteomics results from 300 intermediate-risk PCa prostatectomy samples. Multiple regions of the prostate, including regions of the index lesion, secondary lesions, regions of benign and normal were sampled then analyzed using Bruker’s CosMx SMI platform for a 6000-plex RNA assay in a subset of patients; in addition to proteomic profiling using the GeoMx DSP platform. Clustering assigned cells into 15 cell types based on their transcriptional profiles. Each cell type was tested for significantly different proportions in tumor versus non-tumor samples. The cell type with the greatest bias had a 9-fold greater proportion in tumor samples (adjusted P < 0.01). ERG, largely associated with the TMPRSS2-ERG fusion in PCa, was most specifically expressed in cells of this type. Conversely, the cell type with the greatest bias towards non-tumor samples (3-fold greater proportion, adjusted P < 0.01) is characterized most specifically by the expression of MSMB, which encodes an immunoglobulin binding factor previously found to have decreased expression in PCa. Other evidence of intratumoral heterogeneity was found in samples that are not dominated by these stereotypical cell types. For example, a relatively higher-grade sample from an index tumor possessed high proportion of the tumor-dominant cell type whereas a relatively lower-grade sample from the same tumor did not. Similarly, the tumor and non-tumor samples arising from the same patient showed similar cell type composition, highlighting inter-patient heterogeneity. Moreover, we identified proteins with significantly different expression in tumor versus non-tumor samples. In the TME, EpCAM and CD56 were significantly overexpressed and underexpressed in tumor versus non-tumor regions, respectively (adjusted P < 0.01). In the epithelium, cleaved caspase-9, which initiates the caspase cascade leading to apoptosis, was the most significantly underexpressed protein in tumor samples (adjusted P < 0.01). However, the expression of the mRNA encoding caspase-9 is not highly correlated with the expression of cleaved caspase-9, highlighting the added benefit of protein expression profiling. Taken together, this multi-modal spatial analysis of multiple samples from the same PCa patients facilitates a detailed understanding of early PCa for biomarker discovery and pathways of therapeutic intervention.


 Anna Y. Lee, Megan Hopkins, Linda Liao, Vida Talebian, Tamara Jamaspishvili, David M. Berman, Melanie Spears, Jane Bayani. Revealing the transcriptional and proteomic spatial neighborhoods of early prostate cancer and the tumor microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 777.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cbc83941350f959c8dff2b5403f5812073037ec" target='_blank'>
              Abstract 777: Revealing the transcriptional and proteomic spatial neighborhoods of early prostate cancer and the tumor microenvironment
              </a>
            </td>
          <td>
            A. Lee, M. Hopkins, L. Liao, Vida Talebian, Tamara Jamaspishvili, David M. Berman, Melanie Spears, Jane Bayani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Cervical cancer (CC) is a major global health issue, ranking sixth in cancer-related mortality. The tumor microenvironment (TME) plays a crucial role in tumor growth. This study explored the cellular composition and immunological landscape of CC using various genomic data sources. Methods Data from the Cancer Genome Atlas and Gene Expression Omnibus were analyzed, including single-cell RNA sequencing, spatial transcriptome analysis, and survival data. Gene set variation analysis (GSVA) identified pathways in CD8+ cells, macrophages, and epithelial cells. Immunohistochemistry assessed marker expression in CC and normal tissues. Tumor immune dysfunction and exclusion (TIDE) scores differentiated high- and low-macrophage groups. Cell–cell communication analyses highlighted interactions between macrophages and epithelial cells. Results Macrophage markers correlated with overall survival (OS) and disease-free survival (DFS). Epithelial cell subgroups 1 and 4, along with CD8+ T cells, were associated with OS. TIDE scores varied between groups. Specific ligand-receptor interactions were found between macrophages and epithelial cell subgroup 1. Triptolide was effective in epithelial cell subgroup 1, while memantine was more effective in macrophages. Conclusion Epithelial-macrophage interactions in the TME are crucial for CC progression and treatment, offering a potential immunotherapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6446bb065e28f5d7b90ac2e8452e69b230ac374" target='_blank'>
              Epithelial and macrophage cell interaction in cervical cancer through single-cell RNA-sequencing and spatial analysis
              </a>
            </td>
          <td>
            Zhichao Wang, Long Cheng, Guanghui Li, Huiyan Cheng
          </td>
          <td>2025-04-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Small cell lung cancer (SCLC) represents a major medical challenge due to its aggressive progression and a dismal 5-year overall survival rate of only 7%. While SCLC initially responds well to chemotherapy, the rapid relapse and emergence of therapy-resistant tumors remain critical barriers to effective treatment. We hypothesized that therapy response is associated with genome evolution, which, in turn, influences transcriptional plasticity. This study aims to decipher the clonal evolution and transcriptomic adaptations in response to chemotherapy in SCLC.



 We established over 100 patient-derived xenotransplant models from clinical SCLC specimens, capturing diverse genetic features and clinical characteristics. A representative subset of 33 models was selected to replicate clinical scenarios in mice, modeling the effects of platinum-based chemotherapy and evaluating tumor responses under controlled conditions. Whole-exome sequencing and single-cell transcriptome profiling were performed to investigate genomic evolutionary trajectories and transcriptional plasticity in therapy-naïve and relapsed tumors.



 Xenotransplant models accurately reflect patient responses to chemotherapy. To our surprise, several models from relapsed patients retained sensitivity to treatment. Chemotherapy-resistant models maintain their tumor clones, whereas sensitive models show dynamic shifts from a common ancestral clone with emerging subclones shaped by mutations associated with platinum damage, APOBEC activity, and defective DNA damage repair. The genomic landscape, defined by subclonal mutations and copy number alterations, strongly impacts transcriptomic patterns. Specifically sensitive models post-chemotherapy reveal a strong shift of transcriptional programs reflecting adaptive plasticity as a consequence of genome evolution.



 Here we provide an in-depth functional in vivo characterization of clonal adaptions in SCLC under therapy. Chemotherapy drives the selection of subclones with favorable genomic features, which in turn impact the broader transcriptomic landscape. Our data provides critical insights into the molecular mechanisms of therapy resistance.



 Laura Constanze Kaiser, Marcel Schmiel, Christian Müller, Maria Cartolano, Martin Peifer, Roman K. Thomas, Julie George. Deciphering evolutionary dynamics and transcriptomic adaptions under therapy in small cell lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1258.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f680cbd3c45701162ce05be4c984e82cf3cb95" target='_blank'>
              Abstract 1258: Deciphering evolutionary dynamics and transcriptomic adaptions under therapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Kaiser, Marcel Schmiel, Christian Müller, Maria Cartolano, M. Peifer, Roman K. Thomas, Julie George
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="
 e14555


 Background:
 Natural killer (NK) cells are innate lymphocytes that engage in direct cytotoxic responses against malignant cells and promote pro-inflammatory tumor microenvironments. While studies have predominantly focused on conventional NK cell subsets, recent findings uncovered adaptive and tissue-resident (TR) subsets with unique functional properties. The influence of distinct NK cell subsets on tumor immunobiology and clinical outcomes remains unknown.
 Methods:
 Single-cell RNA-sequencing (scRNA-seq) of primary, therapy-naïve tumors from 276 patients across 14 cancer types, as well as 39 patients treated with immune checkpoint blockade (ICB), was accessed through publicly available datasets. We employed systematic computational analyses to phenotype and examine intra-tumoral NK cells. Gene signatures for NK cell subsets were applied to bulk RNA-sequencing from 212,105 tumor samples to delineate their associations with clinical outcomes using the Caris CODEai database.
 Results:
 Six main subsets of NK cells were identified in solid tumors, including conventional CD56
 dim
 and CD56
 bright
 NK cells, alongside adaptive, TR adaptive, TR CD56
 bright
 , and TR CD56
 bright
 /adaptive subsets. Across cancer, TR adaptive NK cells emerged as the most prognostic subset, with high infiltration conferring significantly longer overall survival (HR[95% CI] = 0.68[0.66-0.69], p < 0.001) than conventional CD56
 dim
 and CD56
 bright
 subsets. This was also observed in cohorts of head and neck, non-small cell lung, and esophageal cancers, particularly those treated with ICB. Notably, elevated TR adaptive NK cell infiltration was more prognostic than high tumor mutational burden in these settings. This was most apparent in ICB-treated head and neck cancers (HR = 0.48[0.41-0.57], p < 0.001 vs HR = 0.90[0.76-1.04], p = 0.16). Additionally, TR adaptive NK cells were strongly correlated to the infiltration of CD8
 +
 T cells and B cells in tumors. Underlying these observations, our scRNA-seq analyses demonstrated that TR adaptive NK cells exhibit the highest immunomodulatory activity of all subsets, expressing high levels of pro-inflammatory cytokines (
 CCL3
 ,
 CXCL13
 , and
 IFNG
 ) and low levels of immunosuppressive cytokines (
 TGFB1
 ). Mechanistically, this activity was linked to the upregulation of transcription factors associated with NK cell activation, function, and persistence and downregulation of genes related to stress response and exhaustion. Interestingly, scRNA-seq from paired pre- and post-ICB-treated tumors revealed a unique expansion of cytotoxic
 GZMB
 + TR adaptive NK cells after treatment.
 Conclusions:
 TR adaptive NK cells represent a novel subset of NK cells that may drive anti-tumor immune responses, particularly in the context of ICB treatment. Features of this unique population may be harnessed to design more effective NK cell-based therapies against solid tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdbcf1a77672e37211fd7cd09d5cd12367404c3a" target='_blank'>
              Identification of tissue-resident adaptive NK cells and their association with reduced mortality across diverse solid tumors.
              </a>
            </td>
          <td>
            John R Lozada, A. Ali, Christine Luo, Rosemary N. Plagens, A. Elliott, Ella Boytim, Nicholas A Zorko, W. El-Deiry, Elisabeth I. Heath, Akash Patnaik, A. Vanderwalde, Jeffrey S Miller, E. Antonarakis, Frank Cichocki, Justin H Hwang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Dynamic adaptations in cancer cell-state due to therapy is a growing concern. Despite multimodal clinical therapy, tumor cells surpass treatment resulting in frequent relapses in rapidly decreasing timeline. We have shown that, in accordance with therapy pressure associated tumor microenvironment, progressive disease (PD) cells exhibit reversible lineage transformation and unparalleled plasticity. However, our understanding of the genetic determinant(s) and/ or mechanism(s) that coordinate cancer cell plasticity is a far reach. Herein, we investigated the crucial cancer cell phenotypic switching in response to therapy pressure. Using a panel of 14 well characterized human neuroblastoma (NB) cells derived during diagnosis (Dx) and PD, we investigated the association of retinal degeneration protein 3 (RD3) status (ELISA) to their migratory and invasive capabilities (transwell assay), clonal expansion (BrDu), EMT (E-CAD, N-CAD, Vimentin, SLUG), metastatic state (Ron-β, MMP-9, MMP-2), stemness maintenance (CD133, CD117, CD114, Oct3/4, Sox2, NANOG, CD54, NOTCH1) and cellular dedifferentiation (β-3 tubulin, NF-L, GFAP, GAP43) (immunoblotting). Further, to investigate the function of RD3 in coordinating cancer cell plasticity, we utilized six select NB cells, three derived during Dx and three derived during PD, representing both primary and metastatic tumors. These patient derived cells were reverse engineered for stable expression of RD3 in RD3- PD cell lines; stable silencing of RD3 in RD3+ Dx cell lines and were subjected to end-point readouts as above. RD3 loss in PD correlated with the increased cellular migration, invasion, metastatic potential, EMT and stemness maintenance. Directed simulation experiments with stable restoration and silencing systems clearly identified the function of RD3-loss in cancer cell plasticity by regulating stemness (ALDH, CD133), EMT (Vimentin, E-CAD, N-CAD), metastasis (Ron-β, MMP-9), pluripotency maintenance (Oct3/4, Sox2, NANOG), dedifferentiation (β-3 tubulin, NF-H, GFAP, GAP43) and clonal expansion (BrDu). Our findings for the first time identified the defined function and relevance of RD3 in regulating cancer cell EMT, stemness and pluripotency maintenance, metastatic state and prompting differentiation. Crucially, our findings recognized therapy pressure directed RD3 loss dictates cancer cell lineage transformation to cancer stem cells. These outcomes together pave way to develop novel and effective molecular-targeted maintenance therapeutic strategy for patients presented with therapy defying progressive disease.


 This work was partially or in full, funded by Department of Defense CA-210339; OCAST-HR19-04; NIH-P20GM103639 to Dr. Aravindan; and by the NIH-NCI-Cancer center support Grant P30CA225520, NIGMS P30GM154635, Oklahoma Tobacco Settlement Endowment Trust [R23-03] to the OU Health Stephenson Cancer Center.
 Citation Format: Poorvi Subramanian, Sreenidhi Mohanvelu, Sivasubramani Narayanan, Afsana Parveen Jahir Hussain, Sabir Salim, Madhi Oli Ramamurthy, Loganayaki Periyasamy, Natarajan Aravindan. Driver and signaling cascade in therapy pressure driven cancer cell plasticity [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1246.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37bbf9a9279b9ea882191a6ab180a2f1195c66ee" target='_blank'>
              Abstract 1246: Driver and signaling cascade in therapy pressure driven cancer cell plasticity
              </a>
            </td>
          <td>
            Poorvi Subramanian, Sreenidhi Mohanvelu, Sivasubramani Narayanan, Afsana Parveen Jahir Hussain, Sabir Salim, Madhi Oli Ramamurthy, Loganayaki Periyasamy, N. Aravindan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Esophageal adenocarcinoma is a prevalent and serious disease, ranking eighth globally, with a low survival rate of 15-25% and a rising incidence in the United States. However, the occurrence of brain metastasis in esophageal adenocarcinoma (EAC) is rare, affecting only approximately 2-6%. Despite its rarity, brain metastasis in EAC is associated with a poor prognosis, compounded by the limited availability of effective treatment options. Due to the infrequent occurrence of brain metastasis in EAC patients, our understanding of the tumor composition in these cases and how they differ from those in primary sites remains limited. To address these gaps, we analyzed the tumor microenvironment of brain metastasis tumors and their matched primary tumors using spatial transcriptomic data generated by the 10x Xenium Analyzer. By employing a brain tumor panel, an immune panel, and a custom panel, we characterized the tumor microenvironment, including neuronal and immune cell populations present in these samples. Overall, primary samples contained a higher proportion of immune cells such as macrophages, T cells, B cells, dendritic cells, plasma cells, and mast cells, compared to brain metastases, which had a greater composition of neuronal cells. We additionally investigated the oncogenic expression patterns at the single-cell level within these matched samples, revealing areas with high ERBB2 and EGFR expression density within malignant cell populations. Notably, brain metastasis samples contained a higher proportion of malignant cells co-expressing ERBB2 and EGFR than primary samples, suggesting clonal evolution during disease progression from primary to metastatic sites. Our findings indicate that ERBB2 and EGFR amplifications may represent potential targets for therapeutic interventions in these patients. These findings enhance our understanding of brain metastasis in EAC, offering insights into the tumor microenvironment that may guide future therapeutic strategies and improve clinical management for this previously understudied metastatic type.


 Nora Lawson, Lingqun Ye, Chae Yun Cho, Bo Zhao, Thomas Mitchell, Ines Martín, Andy Futreal, Kadir Akdemir. Single cell spatial profiling reveals ERBB2 and EGFR co-expression in primary esophageal adenocarcinoma and brain metastasis tumors [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2676.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/446717528f9ab0573ae7634a1587a6e025056fd4" target='_blank'>
              Abstract 2676: Single cell spatial profiling reveals ERBB2 and EGFR co-expression in primary esophageal adenocarcinoma and brain metastasis tumors
              </a>
            </td>
          <td>
            Nora M Lawson, Lingqun Ye, Chae Yun Cho, Bo Zhao, Thomas Mitchell, Ines Martín, Andy Futreal, Kadir C Akdemir
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Molecular subtyping of pancreatic ductal adenocarcinoma (PDAC) has converged on basal-like and classical subtypes with the former being associated with worse prognosis. Sequencing-based tools have emerged to bring PDAC subtyping closer to incorporation into clinical practice. We sought to test the utility of PDAC subtyping using a cohort of patient tumor samples in a clinical setting. For 29 patients with resectable PDAC, two formalin-fixed paraffin-embedded (FFPE) blocks were sequenced from each patient tumor sample (total n=58 blocks) using Nanostring nCounter. 18/58 (31%) of tumors had inconsistent PurIST-based basal-like or classical labels between the two blocks. Two tumors were selected for digital spatial transcriptomics based on having either consistent or inconsistent subtype labelling. Regions of interest (ROI) were selected to capture a diverse range of cellular compartments within the tumor microenvironment including tumor, stroma, immune cells, tumor buds and neurons. ROIs were sequenced across two and three FFPE blocks from the subtype consistent and inconsistent tumors, respectively. Gene expression was segmented by PanCK positive (epithelial) and negative (non-epithelial) signal. Across all ROIs (n=35), PanCK+ segments showed higher expression of basal-like (p=1.9e-7) and classical (p=4.2e-9) genes compared to PanCK- segments. For the subtype-consistent (classical) tumor, all ROIs (n=13) across two blocks showed predominate classical gene expression in the PanCK+ segment. In the subtype-inconsistent tumor that originally had blocks labelled PurIST classical and intermediate, ROIs (n=22) taken across three blocks showed 50, 60 and 83% of ROIs having higher basal-like expression, while remaining ROIs had predominant classical gene expression, and log2 fold change basal-like vs. classical (median) gene expression ranged from -2.4 (T-cells) to 1.3 (tumor and stroma) across ROIs. Overall, the classical gene signature was positively correlated with GATA6 expression (rho=0.40, p=3.2e-4) across all ROIs and PanCK segmentations. While these data support the notion of subtyping expression being specific to the tumor epithelial component, nearly one-third of patient samples had different subtype labels depending on which area of the tumor epithelium was captured. We demonstrate the ability for spatial transcriptomics to resolve subtype heterogeneity and the contributing cellular compartments, providing potential to better understand how aggressive disease develops and could be treated.


 James Topham, Jenny Chu, Maya Kevorkova, Steve Kalloger, Joanna Karasinska, Andrew Metcalfe, Hassan Ali, Dongxia Gao, Christine Chow, Jonathan Loree, Daniel Renouf, David Schaeffer. Spatial transcriptomics reveals subtype heterogeneity within the tumor epithelial compartment of pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4596.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a22b42357df2b30b92286c1a429534a02e045698" target='_blank'>
              Abstract 4596: Spatial transcriptomics reveals subtype heterogeneity within the tumor epithelial compartment of pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            J. Topham, Jenny E. Chu, Maya Kevorkova, Steve E. Kalloger, J. Karasinska, Andrew Metcalfe, Hassan Ali, Dongxia Gao, Christine Chow, Jonathan M Loree, D. Renouf, David Schaeffer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Lung adenocarcinoma (LUAD) is one of the most prevalent and lethal cancer types worldwide. However, our understanding of the transition from normal-appearing tissue (NAT) to adenomatous lung premalignant lesions (aPMLs) and LUADs, particularly within a spatial context, remains limited. This study aims to address this gap by systemically analyzing the pathologic continuum of NAT > aPML > LUAD using spatial transcriptomics (ST).



 High-resolution spatial profiling was conducted on 56 samples from 25 patients with paired aPMLs and LUADs using the Visium ST platform. Non-negative matrix factorization was conducted to identify transcriptional programs for each sample, following clustering analysis to define consensus metaprograms across the cohort. Additionally, spatial molecular imaging was performed on an expanded cohort of 188 cores arranged into eight tissue microarrays using the Xenium in situ platform with a customized lung cancer gene panel to establish a single-cell spatial atlas of the disease continuum and systemically investigate spatial organization, cellular neighborhoods and interactions.



 Eight distinct metaprograms (MP1∼8) were identified, distinguishing stromal (MP2), myeloid (MP4), lymphoid (MP6), and epithelial (MP3, MP5) compartments. Additionally, metaprograms were identified for the lung capillary bed (MP7), stressed cellular state (MP8), and mosaic cellular patterns (MP1). These metaprograms correlated strongly with pathological annotations and captured fine tissue structures like lymphoid aggregates. LUADs and aPMLs showed distinct MP profiles, with MP3 and MP6 being highly abundant in LUADs, while MP1, MP4, MP5 and MP7 were more abundant in aPMLs. Specifically, we observed positive correlations between MP3 and MP6, as well as between MP4 and MP5 in sample distributions, indicating co-evolution of tumor and immune microenvironments during disease progression. Xenium data complemented these findings by revealing distinct compositions of microenvironmental cellular states and tumor neighborhood structures across aPML and LUAD lesions. A total of 44 cell states were defined. Specifically, NK cells were depleted in aPMLs and LUADs compared with NAT, accompanied by an enrichment of regulatory T cells and myofibroblastic cancer-associated fibroblasts (CAFs) in the latter. Additionally, pro-inflammatory cellular subsets, including IL1B+ macrophages and IL6+ inflammatory CAFs were more abundant in tumor neighborhoods of aPMLs compared to LUADs, potentially contributing to the pre-malignant to malignant transition.



 This study offers a comprehensive molecular and cellular landscape of aPML and LUAD, laying a crucial foundation for future mechanistic studies aimed at early disease interception.



 Yibo Dai, Fuduan Peng, Ansam Sinjab, Sujuan Yang, Minyue Chen, Warapen Treekitkarnmongkol, Lorena I. Gomez Bolanos, Tieling Zhou, Alejandra G. Serrano, Jianlong Liao, Guangsheng Pei, Yunhe Liu, Yang Liu, Jiahui Jiang, Kyung Serk Cho, Yanshuo Chu, Kai Yu, Ruiping Wang, Jiping Feng, Zahraa Rahal, Guangchun Han, Naoe Jimbo, Takuo Hayashi, Satsuki Kishikawa, Kazuya Takamochi, Akshay Basi, Avrum Spira, Steven Dubinett, Tomokazu Itoh, Takashi Yao, Kenji Suzuki, Luisa M. Solis, Stephen Swisher, Mingyao Li, Junya Fujimoto, Ignacio I. Wistuba, Jared Burks, Kadir Akdemir, Hind Refai, Katy Rezvani, Jeffrey Myers, Humam Kadara, Linghua Wang. Spatial transcriptomics identifies unique tumor and microenvironment pathomic programs that are associated with the lung premalignancy and adenocarcinoma continuum [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 759.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d66d16b13f8aa6c453618719a245bd84fb380319" target='_blank'>
              Abstract 759: Spatial transcriptomics identifies unique tumor and microenvironment pathomic programs that are associated with the lung premalignancy and adenocarcinoma continuum
              </a>
            </td>
          <td>
            Yibo Dai, F. Peng, Ansam Sinjab, Sujuan Yang, Minyue Chen, Warapen Treekitkarnmongkol, Lorena I. Gomez Bolanos, Tieling Zhou, Alejandra G Serrano, Jianlong Liao, Guangsheng Pei, Yunhe Liu, Yang Liu, Jiahui Jiang, K. S. Cho, Yanshuo Chu, Kai Yu, Ruiping Wang, Jiping Feng, Zahraa Rahal, Guangchun Han, N. Jimbo, Takuo Hayashi, S. Kishikawa, K. Takamochi, Akshay V. Basi, Avrum E. Spira, Stephen Dubinett, Tomokazu Itoh, Takashi Yao, Kenji Suzuki, Luisa M. Solis, S. Swisher, Mingyao Li, Junya Fujimoto, I. Wistuba, Jared K. Burks, Kadir C Akdemir, Hind Refai, K. Rezvani, Jeffrey Myers, H. Kadara, Linghua Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 The tumor microenvironment consists of different cell populations, whose interactions contribute to tumor heterogeneity and plasticity and pose challenges to therapy efficacy and prognosis prediction. Breast cancer is one of the highly heterogeneous diseases, and estrogen receptor (ER) positive subtype, the most common one, shows worse prognosis once metastasized. Spatial transcriptomics (ST) is emerging as a powerful tool that offers valuable insights into spatial complexity of tumor microenvironment as well as the cell interactions underpinning tumor heterogeneity. In this study, we aimed to examine tumor heterogeneity of ER positive metastatic breast cancer by applying our geographic information science (GIS)-augmented ST analysis pipeline. Visium datasets were prepared from four metastatic liver tumors from xenograft mouse model and twelve primary and metastatic bone tumors from human patients, including three matching pairs. We integrated geospatial tools, such as multivariate spatial association analysis, with spatial omics data structures to check degree of auto-correlations and visualize localizations of single gene or gene sets. We further compensated for multi-cellular resolution of our spatial data via cell type mapping and deconvolution of cell type proportion (STdeconvolve). We identified hotspot regions displaying extensive co-localization between estrogen response gene set and lipid metabolism gene sets in the fulvestrant-treated metastatic liver tumors from xenograft mouse model. These hotspots were not concentrated in the same area within the tumor and differed by gene sets, suggesting intra-tumor metabolic heterogeneity of the endocrine responsive cells. Our primary samples from human patients showed similar pattern of co-occurrence between estrogen response and metabolic programs as well as cell-cycle related gene sets. On the other hand, mutual exclusivity was observed for estrogen response with immune-related gene sets and metastasis-related ones (epithelial mesenchymal transition, angiogenesis). Compared to primary tumors, metastatic bone tumors showed inter-tumor heterogeneity as such consistency in spatial pattern was obscured. Deconvoluting cell type proportions mapped a higher proportion of mature luminal cells in endocrine responsive hotspots, whereas cancer associated fibroblasts and pericytes were prevalent in the hotspots of metastasis-related gene sets. Our study fully utilized the potential power of ST data by delineating the tumor microenvironment in relation to intra-tumoral heterogeneity as well as cell type proportions. Findings from our study support the development of strategies to target different endocrine resistant cell populations with a combination of metabolic and metastasis-related signaling inhibitors.


 Jin Young Yoo, Sabrina Akter, Qianying Zuo, Mahima Goel, Audrey Lam, Debapriya Dutta, Betsy Barnick, Maria Grosse-Perdekamp, Aiman Soliman, Zeynep Madak-Erdogan. In silico reconstruction of primary and metastatic estrogen receptor positive breast tumor architecture using geographic information science-augmented spatial transcriptomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4735.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f55f59e35cc571551e6bd44fc89c978612969a99" target='_blank'>
              Abstract 4735: In silico reconstruction of primary and metastatic estrogen receptor positive breast tumor architecture using geographic information science-augmented spatial transcriptomics
              </a>
            </td>
          <td>
            Jin Young Yoo, Sabrina Akter, Qianying Zuo, Mahima Goel, Audrey Lam, Debapriya Dutta, Betsy Barnick, Maria Grosse-Perdekamp, Aiman Soliman, Zeynep Madak-Erdogan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, J. Cook, Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 e20542


 Background:
 Current diagnostic criteria for multifocal lung adenocarcinoma rely primarily on histopathology and next-generation sequencing. However, distinguishing multiple primary lung adenocarcinomas (MPLC) from intrapulmonary metastasis (IM) is challenging due to similar histology. This study explores key differences in the tumor microenvironment (TME) and tumor heterogeneity between MPLC and IM.
 Methods:
 Eight patients with 16 lung nodules were selected for single-cell sequencing. Samples were categorized into three pairs for the MPLC group and five pairs (five primary and five metastatic) for the IM group. Differences in the TME and tumor cell heterogeneity were analyzed using multiplex immunofluorescence, spatial proteomics, and bioinformatics.
 Results:
 In the TME, IM group T cells showed stronger immunosuppressive features, while MPLC group had enriched CD8 effector and follicular helper T cells. IM group macrophages exhibited pro-inflammatory and pro-angiogenic traits, contrasting with MPLC group's lipid metabolism-linked macrophages. MPLC group endothelial cells expressed tumor-suppressor genes, whereas IM group fibroblasts showed genes related to stromal remodeling and tumor progression. Tumor cell heterogeneity revealed upregulated CRABP2, CEACAM5, and PLAT in IM group, associated with aggressive phenotype, while MPLC group showed higher extracellular matrix-related protein expression. Four genes (CXCL8, RNASE1, CTSH, AQP4) showed significant transcriptional differences.
 Conclusions:
 Our study identifies key TME differences between IM and MPLC, particularly in immune suppression and cellular composition. The distinct roles of macrophages, endothelial cells, and fibroblasts in IM suggest increased tumor aggressiveness. The identified interaction pairs and differential genes may serve as biomarkers for distinguishing MPLC from IM, aiding in diagnostic and therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24593b501d640a253836bb81a758a90b930a596e" target='_blank'>
              Unraveling the complex code of tumor microenvironment and heterogeneity by comparing multiple primary lung adenocarcinomas and intrapulmonary metastasis: A multidimensional exploration.
              </a>
            </td>
          <td>
            Shaobin Yu, Shuchen Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell therapies have been groundbreaking for B-cell lymphomas but optimizing efficacy for solid tumors remains challenging. Lack of consensus on potency measures, unknowns about novel engineering strategies, and product heterogeneity drive extensive characterization to discover product-specific mechanisms of action (MoAs). This process is costly and limited by variable results from simple molecular assays and need to infer relationships between data from disparate technologies. Thus, we created a multi-modal technology with which to establish a multi-functional, longitudinal cellular characterization workflow. With a single sample input, this workflow evaluates multiple live phenotypes from the same individual cells across tens of thousands of cells at once. We leveraged TALL-104 cells before validating compatibility with clinical-stage cell therapies with AIC100 - a Phase 1 CAR-T product targeting thyroid cancer (NCT04420754). Cellanome’s technology enables multi-modal evaluation of live, cellular behavior at the resolution of single cells or single-cell interactions. Within an 8-lane flow cell, tens of thousands of single TALL-104 cells were enclosed with K562 target cells in bio-compatible hydrogel compartments called CellCage™ enclosures (CCEs). These CCEs enable longitudinal monitoring of single TALL-104 cells interacting with K562 cells for hours to days with time-lapse imaging. During this period, imaging-based evaluation of surface markers, cytokine secretion, and cytotoxicity are integrated from the same single T-cells. Our analysis pipeline links target cell death to single T cells in CCEs, enabling cell killing to be studied with unprecedented resolution. Similar to flow cytometry via CD107 as a surrogate for killing, with Cellanome’s technology, we could identify cytotoxic T cells (∼85%). Uniquely, we can directly evaluate killing by single TALL-104 cells and further segment them into groups. Group 1 consists of strong killers (killing 100% of targets), Group 2 of medium killers (50-99%), and Group 3 of weaker killers (0-49%). We next characterized single AIC100 cells enclosed with HeLa as targets and observed that, unlike TALL-104 cells where Group 2 was the largest, for AIC100, Group 3 was the largest, suggesting distinct killing mechanisms. Thus, we asked whether AIC100’s heterogeneous composition of CD4 / 8 cells impacts product efficacy. We configured this workflow to test killing by ∼100 combinations of CD4:CD8 AIC100 cells in a single experiment. Preliminarily, more killing occurred in CCEs containing a mixture of CD4 and CD8 T cells relative to CCEs containing only CD4 or CD8 cells. We are continuing to identify the optimal CD4:CD8 compositions for greater potency. We envision multi-modal characterization of clinical products with our platform, pre and post-patient infusion, will inform efforts to advance cell therapies for solid tumors.


 Richard Yau, Shreya Deshmukh, JangKeun Kim, Jiwoon Park, Yanping Yang, Makenzie Sacca, Shan Sabri, Teresa Ai, Pier Federico Gherardini, Gary Schroth, Christopher E. Mason, Moonsoo M. Jin. Establishing a multi-modal, cell therapy characterization workflow to identify cell subsets with heightened cytotoxicity [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2181.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96a94c903e0937bcd16fe7b5fa29e52d24143783" target='_blank'>
              Abstract 2181: Establishing a multi-modal, cell therapy characterization workflow to identify cell subsets with heightened cytotoxicity
              </a>
            </td>
          <td>
            Richard Yau, Shreya Deshmukh, Jangkeun Kim, Jiwoon Park, Yanping Yang, Makenzie Sacca, Shan Sabri, Teresa Ai, P. F. Gherardini, Gary Schroth, Christopher E. Mason, Moonsoo M. Jin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Glioblastoma multiforme (GBM) is the most common and lethal brain cancer. Standard of care treatments (SOC) are limited to surgery, radiation and chemotherapies, providing an average overall survival of only ∼16 months. Tumor heterogeneity and cellular plasticity are major contributions to treatment resistance in this disease. We constructed an integrated patient-level and cohort-level framework and analyzed single-nucleotide RNA sequencing data in matched primary and recurrence specimens to understand tumor heterogeneity, evolution, and drug resistance in SOC treated GBM patients (N=23). Patient-level tumor analysis revealed distinct tumor cell transcriptional subclones representing widespread intra-patient heterogeneity. Tumor evolution analysis from primary to recurrence revealed diverse treatment responses representing sensitive, intrinsic resistant, and acquired resistant subclones. Further characterization of tumor subclones and clustering analyses indicated that tumor heterogeneity under SOC treatments was linked to diverse biological mechanisms including neural development and oncogenic pathways, contributing to varying levels of subclonal resistance. Our analysis also highlighted intrinsic resistance was prominent as primary tumors often harbored substantial proportion of resistant tumor cells. This insight on resistant status at cell level then allowed us to compare resistant to sensitive cells and identify both established and potentially novel mechanisms of SOC resistance that were not always obvious in the comparison between primary and recurrence samples. These included activation of PI3K/AKT, MAPK, TGF-β, EMT, and WNT signaling pathways. Taken together, our novel analytical strategy revealed complex tumor heterogeneity and evolution landscape in SOC treated GBM, offering insights on potential mechanisms driving these phenomena and providing a resource for further exploration and development of therapeutic strategies.


 Yueshan Zhao, Xiaoyu Lu, Verah Nyarige, Junfei Zhao, Gonzalo Lopez, Ann Forslund, Maria Ortiz-Estevez, Jorge Benitez-Hernandez, Wei Zhang, Celia Fontanillo, Kai Wang, Ha Dang. Single-cell analysis reveals intricate subclonal heterogeneity and evolution in response to standard of care treatments in glioblastoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1253.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/339780e1e052f73bd7a155d38cbe11ed7059d53c" target='_blank'>
              Abstract 1253: Single-cell analysis reveals intricate subclonal heterogeneity and evolution in response to standard of care treatments in glioblastoma
              </a>
            </td>
          <td>
            Yueshan Zhao, Xiaoyu Lu, Verah Nyarige, Junfei Zhao, Gonzalo Lopez, Ann Forslund, Maria Ortiz-Estevez, Jorge Benitez-Hernandez, Wei Zhang, C. Fontanillo, Kai Wang, Ha Dang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Although esophageal squamous cell carcinoma (ESCC) is a highly aggressive malignancy, there is little knowledge about its tumor heterogeneity affecting treatment response. To decipher transcriptomic heterogeneity of tumor microenvironment (TME) from a therapeutic perspective, we conducted CITE-seq and immunostaining by Hyperion in ESCC with or without chemotherapeutic stimuli.
 Tumor biopsies of ESCC were sequentially collected by endoscopy from 3 patients during chemotherapy treatments. CITE-seq samples were processed with Chromium Next GEM Single Cell 5′ Reagent Kit v2 (10 x Genomics) and TotalSeq Universal Cocktails (BioLegend) as per the manufacturer′s recommendations, respectively. Cell-to-cell interaction analyses in CITE-seq and neighborhood analyses in Hyperion were performed.
 Clinical responses of 3 patients for ESCC tumor biopsies were PD, SD, and PR, respectively. Cell annotation analyses in TME components revealed that populations of myeloid cells exhibited an overall decrease during the treatment across three patients, while the decrease in myeloid cell populations was more pronounced in the PR patient. The cluster analyses in epithelial cells exhibited 6 unique subclusters, which were transcriptionally characterized in proliferation, EMT, inflammation, and metabolic pathways. In spatial analyses, the epithelial cells in the PD patient adhered tightly to each other, while the epithelial cells in the SD or PR patient clustered relatively mixed with immune and stromal cells. In SD or PR patients, furthermore, varieties of T cell populations were more accumulated in response to the chemotherapy treatments compared to the PD patient. Neighborhood analysis and cell-cell interaction analysis together identified cell type pairs that showed consistent changes in physical and molecular perspectives. In PD patient, dendritic cell (DC)-macrophage pair was enriched in the proximity of each other, and also increased molecular interactions after chemotherapy. In contrast, activated T-Treg pair was depleted and decreased molecular interactions. Interestingly not just activated T-Treg interaction decreased, but proliferating T cells were also depleted from the tumor.
 We conducted integrative analyses with cutting-edge technologies to determine temporal dynamics of TME landscape in ESCC during chemotherapy treatment. Our observations may indicate that chemotherapy increased DC-macrophage interaction to suppress antigen presentation by DC, which suppressed T cell activation and proliferation. The mechanism might contribute to the chemotherapy resistance. These findings will shed light on the molecular insights to deeply understand complexity of the TME networks for chemotherapy resistance in ECSS.


 Hiroki Yamashita, Chiaki Mashima, Maiko Narahara, Ryo Kamata, Kenji Watanabe, Ryuichi Nakamura, Sumie Muramatsu, Ayumi Tezuka, Eri Otsuka, Chisa Wada, Yoko Ishimoto, Daisuke Kotani, Takashi Kojima, Naoya Wada, Hiroyuki Sumi, Tomonori Yano, Gensuke Takayama, Akihiro Ohashi. Temporal dynamics of transcriptional and spatial landscape in tumor microenvironment of esophageal squamous cell carcinoma during chemotherapy treatment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5303.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99aa7af8e7e4a84d84918f1a59bfdf70d970aa43" target='_blank'>
              Abstract 5303: Temporal dynamics of transcriptional and spatial landscape in tumor microenvironment of esophageal squamous cell carcinoma during chemotherapy treatment
              </a>
            </td>
          <td>
            Hiroki Yamashita, Chiaki Mashima, Maiko Narahara, R. Kamata, Kenji Watanabe, R. Nakamura, Sumie Muramatsu, Ayumi Tezuka, Eri Otsuka, Chisa Wada, Yoko Ishimoto, D. Kotani, Takashi Kojima, Naoya Wada, Hiroyuki Sumi, Tomonori Yano, Gensuke Takayama, Akihiro Ohashi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Pulmonary pleomorphic carcinoma (PPC) is a rare and aggressive subtype of non-small cell lung cancer characterized by the presence of both epithelial and sarcomatoid components. We conducted multi-omics analyses on surgically resected specimens from nine primary PPC cases to explore characteristics of this tumor at molecular level.



 Whole-exome sequencing and RNA sequencing on bulk tumor specimens to identify mutational profiles. Spatial transcriptomics using the Digital Spatial Profiler was employed to analyze gene expression separately for epithelial and sarcomatoid tumor components. Single-cell RNA sequencing (scRNA-seq) was conducted on three cases to explore intratumoral heterogeneity and clonal evolution. Gene signature based on extracellular matrix (ECM) remodeling was developed and validated in public lung cancer datasets to assess its association with tumor aggressiveness and prognosis.



 Mutational analyses revealed MET exon 14 skipping in two cases and an ALK-EML4 fusion in one case, while no EGFR mutations were observed. We found various differential expressed genes (DEGs) between components. Further analysis (GSEA and Transcriptional factor scoring) revealed there was a common enrichment trend observed among pathways related to MYC, such as those involved in cell proliferation, division, and the cell cycle. On the other hand, significant differences were observed in pathways related to cancer invasion and cell-cell interactions, such as extracellular matrix (ECM) remodeling and hypoxia. identified an atypical cell population bridging epithelial cells and fibroblasts, which we defined as sarcomatoid cells. The clusters defined as epithelial tumor cells and sarcomatoid cells showed upregulation of DEGs corresponding to the respective component identified through spatial transcriptomics. Trajectory analysis visualized tumor evolution, suggesting a transition from epithelial tumor cells to sarcomatoid cells. A consistent upregulation of seven ECM remodeling-related genes was observed in both tumor components compared to normal lung tissue. When applied to a GEO lung adenocarcinoma cohort, this gene signature was associated with more aggressive histological subtypes (P < 0.01). Survival analysis using TCGA demonstrated that high ECM remodeling signature scores correlated with significantly poorer prognosis (P < 0.01).



 This study provides a comprehensive molecular analysis of PPC, revealing both shared and distinct features between its components. The trajectory analysis supports the hypothesis of a common cellular origin for these components and highlights the transition to a more aggressive sarcomatoid phenotype. These findings provide new insights into the biology of PPC and identify potential therapeutic targets to address its intratumoral heterogeneity and clonal evolution.



 Atsushi Matsuoka, Kazuhiko Shien, Shuta Tomida, Hirofumi Inoue, Kazuya Hisamatsu, Ryota Fujiwara, Kousei Ishimura, Ryunosuke Fujii, Ryo Yoshichika, Tomoaki Higashihara, Naohiro Hayashi, Kazuhiro Okada, Fumiaki Mukohara, Mao Yoshikawa, Ken Suzawa, Hidetaka Yamamoto, Shinichi Toyooka. Spatial and single-cell transcriptomics reveal molecular heterogeneity in pulmonary pleomorphic carcinoma. [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2664.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0640112dbbdf90099adbfa03bf4452da14a83a22" target='_blank'>
              Abstract 2664: Spatial and single-cell transcriptomics reveal molecular heterogeneity in pulmonary pleomorphic carcinoma
              </a>
            </td>
          <td>
            Atsushi Matsuoka, K. Shien, Shuta Tomida, Hirofumi Inoue, Kazuya Hisamatsu, Ryota Fujiwara, Kousei Ishimura, Ryunosuke Fujii, Ryo Yoshichika, Tomoaki Higashihara, Naohiro Hayashi, Kazuhiro Okada, Fumiaki Mukohara, Mao Yoshikawa, K. Suzawa, Hidetaka Yamamoto, Shinichi Toyooka
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Purpose
Triple-negative breast cancer (TNBC) is a particularly aggressive subtype of breast cancer, with approximately 30% of patients eventually developing brain metastases (BM), which result in poor outcomes. An understanding of the tumor microenvironment (TME) at both primary and metastatic sites offers insights into the mechanisms underlying BM and potential therapeutic targets.


Materials and Method
Spatial RNA sequencing (spRNA-seq) was performed on primary TNBC and paired BM tissues from three patients, one of whom had previously received immune checkpoint inhibitors before BM diagnosis. Specimen regions were categorized into tumor, proximal, and distal TME based on their spatial locations. Gene expression differences across these zones were analyzed, and immune cell infiltration was estimated using TIMER. A gene module analysis was conducted to identify key gene clusters associated with BM.


Results
Distinct gene expression profiles were noted in the proximal and distal TMEs. In BM, the proximal TME exhibited neuronal gene expression, suggesting neuron-tumor interactions compared to tumor, and upregulation of epithelial genes compared to the distal TME. Immune cell analysis revealed dynamic changes in CD8+ T cells and macrophages across the tumor and TME zones. Gene module analysis identified five key modules, including one related to glycolysis, which correlated with patient survival. Drug repurposing analysis identified potential therapeutic targets, including VEGFA, RAC1, EGLN3, and CAMK1D.


Conclusion
This study provides novel insights into the transcriptional landscapes in TNBC BM using spRNA-seq, emphasizing the role of neuron-tumor interactions and immune dynamics. These findings suggest new therapeutic strategies and underscore the importance of further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f10bf47822156d7b368733d148999f0f14fb4c81" target='_blank'>
              Spatial Transcriptomic Landscape of Brain Metastases from Triple-Negative Breast Cancer: Comparison of Primary Tumor and Brain Metastases Using Spatial Analysis.
              </a>
            </td>
          <td>
            Jihwan Yoo, Inho Park, Hyun Jung Kim, Hun Ho Park, Sora Lee, Jee Hung Kim, Yoon Jin Cha
          </td>
          <td>2025-04-15</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Cholangiocarcinoma is an aggressive biliary adenocarcinoma with limited treatment options. Stromal infiltration and intratumoral heterogeneity in this tumor type prevents identification of actionable dependencies from bulk-tissue profiles. While cholangiocarcinoma illustrates these challenges, they are common across many other aggressive tumors.To address these challenges, we introduce a single-cell RNA-Sequencing (scRNA-Seq) framework for prioritization of drugs to treat heterogeneous tumor cell subpopulations. Single-cell analyses can bypass limitations of sample size in a precision medicine approach by providing large tumor cell “cohorts” even from individual patients, with the advantage that cell subpopulations can be effectively distinguished. Unfortunately, >80% of genes in each cell profiled by scRNA-Seq are typically undetectable, limiting discoverability of therapeutic targets and drug sensitivities. The VIPER algorithm addresses this by inferring the transcriptional activity of a protein from expression of its downstream target genes. By then identifying aberrantly activated proteins, each cell can be matched to an empiric database of cell line drug perturbation RNA-Seq, ranking drugs which empirically inhibit the proteins aberrantly activated in each cell. We refer to this approach as single-cell OncoTreat.Analysis of tumor cells across six patients by this approach has revealed three distinct sub-phenotypes of cholangiocarcinoma cells, with conserved proteomic signatures, and distinct predicted drug sensitivities. A seventh patient recapitulated these phenotypes, with drug predictions validated in a low-passage, patient-derived xenograft (PDX) model. The top-ranked drug by integrated analysis of all tumor cells was plicamycin, which at sub-cluster level only two of three clusters were predicted to respond to. Conversely, dacinostat was inferred to be active against all three tumor-cell sub-phenotypes. Tumor growth studies in PDX demonstrated growth rate control with both plicamycin and dacinostat, but much improved survival time with dacinostat, outperforming gemcitabine + cisplatin standard of care. Post-treatment scRNA-Seq confirmed predicted subpopulation-specific effects, such that dacinostat depleted all three tumor sub-phenotypes, while plicamycin depleted only the two predicted, allowing the third to reconstitute a treatment-resistant tumor. To our knowledge, this is the first report of dacinostat as an HDAC inhibitor effective in cholangiocarcinoma treatment, in vivo. Ultimately, this approach suggests promising candidates for follow-up trials in cholangiocarcinoma, and is highly generalizable to elucidate complementary dependencies of other aggressive, heterogeneous tumors.


 Aleksandar Obradovic, Lorenzo Tomassoni, Daoqi You, Charles Drake, Filemon Dela Cruz, Andrew Kung, Andrea Califano. Single-cell OncoTreat: Novel therapeutic candidates by high resolution inference and validation of drug sensitivities in heterogeneous cholangiocarcinoma tumors [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7479.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6d37955c551700a448a3cd5642f0bb308460197" target='_blank'>
              Abstract 7479: Single-cell OncoTreat: Novel therapeutic candidates by high resolution inference and validation of drug sensitivities in heterogeneous cholangiocarcinoma tumors
              </a>
            </td>
          <td>
            Aleksandar Z Obradovic, Lorenzo Tomassoni, Daoqi You, Charles G. Drake, F. D. Dela Cruz, Andrew L Kung, Andrea Califano
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Understanding cellular behavior within the context of the tumor-immune microenvironment is essential to developing next-generation cancer therapies and advancing precision medicine. The inherent challenges of this problem reflect the complexity of human biology - patient and tissue heterogeneity, a multitude of interacting signaling pathways, dynamic short- and long-range interactions between the tumor and the immune system, and the intrinsic limitations of measurement techniques. Machine Learning foundation models trained on multimodal patient data present an opportunity to grapple with this complexity and push the field forward leveraging recent advances in spatial biology.



 A custom multimodal transformer was trained on 1399 primary resections from lung cancer patients profiled via H&E, CosMx spatial transcriptomics, whole-exome sequencing, and a custom multiplex immunofluorescence panel. To our knowledge this is the largest extant spatial transcriptomics dataset, comprising more than 40 million cells. The transformer was trained via self-supervised learning to predict expression of each CosMx panel gene for a single cell, conditioned on both spatially proximal and patient-level data from all modalities. This training task induces the model to learn fundamental rules that govern cell state and cell-cell interactions within the context of disease. The resulting model, Celleporter, can generate gene expression in a “virtual” cell at a particular location within real or simulated patient tissue. Counterfactual simulations with modified patient data can be used to predict the effects of genetic alterations, gene expression changes, or external interventions on the tumor-immune microenvironment.



 Celleporter accurately predicted spatial gene expression patterns from sparsely sampled data, resolving the limitations of traditional experimental approaches. Virtual cell simulations reproduced distinct biological states, such as cytotoxic and naïve transitions of CD8+ T cells within and outside tumor regions and reproduced foundational immunology, including the relationship between MHC-I and T cell activation. Comparative analyses across patient cohorts identified immune-suppressive mechanisms in STK11-mutant tumors resistant to immunotherapy. Perturbation simulations highlighted therapeutic targets predicted to restore cytotoxic activity in STK11-mutant tumor microenvironments.



 This study demonstrates that a self-supervised foundation model trained on large-scale multimodal patient data can learn fundamental aspects of cancer immunology, and accurately reproduce the impact of the tumor-immune microenvironment on cell state in a patient-specific manner. This flexible system for interrogating cell and tissue biology has direct application to patient stratification and target discovery.



 Yubin Xie, Eshed Margalit, Tyler Van Hensbergen, Dexter Antonio, Jake Schmidt, Yu Phoebe Guo, Jérémie Decalf, Maxime Dhainaut, Lucas Cavalcante, Hargita Kaplan, Rodney Collins, Francis Fernandez, Rob Schiemann, Eric Siefkas, Michela Meister, Joy Tea, Carl Ebeling, Anastasia Mavropoulos, Nicole Snell, Shafique Virani, Ron Alfa, Lacey Padron, Jacob Rinaldi, Daniel Bear. Celleporter: a foundation model of cell and tissue biology with application to patient stratification and target discovery [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3652.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/700ea58a4c1d36566c271fc146a1aaff828a43d8" target='_blank'>
              Abstract 3652: Celleporter: a foundation model of cell and tissue biology with application to patient stratification and target discovery
              </a>
            </td>
          <td>
            Yubin Xie, Eshed Margalit, Tyler Van Hensbergen, Dexter Antonio, Jake Schmidt, Yu (Phoebe) Guo, Jérémie Decalf, Maxime Dhainaut, Lucas Cavalcante, Hargita Kaplan, Rodney Collins, Francis Fernandez, Rob Schiemann, Eric Siefkas, Michela Meister, Joy Tea, Carl Ebeling, Anastasia Mavropoulos, Nicole Snell, Shafique Virani, Ronald Alfa, Lacey J Padrón, Jacob Rinaldi, Daniel Bear
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) is a highly heterogeneous malignancy. Traditional clinical staging systems have limited value in prognostic evaluation and treatment guidance, often failing to capture the profound impact of intratumoral diversity on patient outcomes. Single-cell RNA sequencing (scRNA-seq) provides new perspectives on the cellular makeup and conditions in tumor tissues, with promising implications for functional classification and personalized therapeutic strategies in esophageal cancer.In this study, we integrated scRNA-seq data with bulk RNA-seq profiles from esophageal cancer tissues to construct a comprehensive cellular atlas, focusing on the transcriptional characteristics of epithelial cells. Malignant epithelial cells were identified based on copy number variation (CNV) features using inferCNV analysis. Their developmental states and regulatory mechanisms were further characterized via transcription factor activity inference (SCENIC) and pseudotime trajectory analysis (Monocle). Based on marker genes of malignant epithelial subpopulations, we developed a multi-gene risk scoring model using data from the TCGA and GEO (GSE53624) cohorts. The model’s predictive value for immune landscape, mutational features, and drug sensitivity was also evaluated. Additionally, qRT-PCR was conducted to quantify the expression levels of model genes in ESCC tissue samples, further evaluating their biological relevance. Functional roles of the key gene HMGB3 were validated in vitro through CCK-8 proliferation assays, Transwell invasion assays, and colony formation assays following gene knockdown in ESCC cell lines.At the single-cell level, we identified ten major cell types and six distinct malignant epithelial subclusters, which exhibited pronounced heterogeneity in cell cycle states, transcriptional regulatory networks, and differentiation trajectories. High CNV scores and the enrichment of specific transcription factors (e.g., FOXC1, E2F1, RUNX1) suggested a proliferative and immune-evasive phenotype. A six-gene prognostic model (HMGB3, CHORDC1, CTSD, BTG2, MT1E, PHYHD1) showed strong predictive accuracy for overall survival in the TCGA and GSE53624 cohorts. Furthermore, the risk score showed a significant correlation with an immunosuppressive tumor microenvironment, increased tumor purity, and the activation of certain immune-related pathways. Analysis of drug sensitivity suggests that patients classified as low-risk could respond better to various targeted therapies and chemotherapeutic agents, underscoring their potential clinical relevance. Functional assays revealed that HMGB3 knockdown markedly suppressed ESCC cell proliferation, invasion, and colony formation, supporting its oncogenic role in esophageal cancer progression.This study systematically characterized epithelial cell heterogeneity in esophageal cancer at single-cell resolution and established a risk model based on malignant epithelial markers that effectively predicts prognosis, immune status, and potential drug response. Combined with experimental validation, our findings highlight the pivotal role of HMGB3 in ESCC progression and provide a novel theoretical and practical framework for functional tumor classification and individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70d92eb766fe9a932637e37c8c15e4650793e3d8" target='_blank'>
              Malignant epithelial cell marker–driven risk signature enables precise stratification in esophageal cancer
              </a>
            </td>
          <td>
            Hao Zhang, Shizhao Cheng, Yijun Xu
          </td>
          <td>2025-05-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="End stage breast cancer often presents with oligometastatic tumors and lacks effective treatment options. Previous studies leveraged post-mortem tissue procurement (also called rapid autopsy) programs to collect tumors shortly after death for profiling by bulk whole exome sequencing (WES). This has revealed substantial heterogeneity in the genomes of metastatic estrogen receptor-positive (ER+) and triple-negative breast cancers. In order to understand the heterogeneity of both the genotypes, transcriptional phenotypes, and evolution of these tumors, we collected multi-site metastatic tumors from two end-stage ER+ breast cancer patients enrolled in the LEGACY rapid post-mortem tissue procurement trial. Metastatic tumors were profiled using WES and single-nuclei RNA sequencing (snRNA-seq). We reconstructed high resolution phylogenies for each patient using the mutation called from the snRNA-seq reads to determine the subclonal architecture of cancer cells. Based on the mutation patterns, the gene expression profiles, and the subclonal architecture, we found that the tumors in each patient were divided into two distinct subtypes. A majority of the tumors were grouped into a subtype characterized by a high intra-tumor heterogeneity, with many cell lineages shared widely across metastasis, indicating extensive and undirected dissemination. The cancer cells of this subtype exhibited an EMT gene expression signature. In contrast, the second tumor subtype observed was rare and demonstrated little intra-tumor heterogeneity, a proliferative gene expression signature, and an interferon response signature present widely across cells in the microenvironment. Across all tumors, macrophages comprised the majority of the immune cells. Low diversity tumor macrophages exhibited strong activation of multiple interferon signaling pathways compared to high diversity tumor macrophages. Interestingly, these macrophages expressed a mixture of M1 and M2 polarization markers as STAT1, SOCS1, ISG15, ISG20, MX1, MX2, OAS1, OAS3 that did not correspond to classical M1 or M2 polarization states. This contrast demonstrates that multi-site metastatic tumors and their microenvironment adopt dichotomous states, which may contribute to the difficulties in treating progressive disease. It also underscores the need for a comprehensive and personalized late-stage therapeutic strategy to target the diversity of existing tumor phenotypes.


 Isaac Bishara, Xuan Liu, Jason l. Griffiths, Patrick A. Cosgrove, Jasmine R. McQuerry, Jiayi Liu, Kena K. Ihle, Eliza R. Bacon, Feng Chi, Pierre Wallet, Vince Grolmusz, Diana Simons, Benjamin Copeland, Lance Pflieger, JinFeng Chen, Sumana Majumdar, Terron T. Crowder, Rena Emond, Rachel Factor, David D. Bowtell, Adam L. Cohen, Daniel D. Schmolze, Peter P. Lee, Lusine Tumyan, James R. Waisman, Andrea Bild, Aritro Nath, Jeffrey T. Chang. End-stage breast cancer metastases manifest as two subtypes with distinct dissemination patterns, proliferation/EMT signatures, and immune microenvironments [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5133.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/610c59165b019703a1e62acb49ebba39754e14fb" target='_blank'>
              Abstract 5133: End-stage breast cancer metastases manifest as two subtypes with distinct dissemination patterns, proliferation/EMT signatures, and immune microenvironments
              </a>
            </td>
          <td>
            Isaac Bishara, Xuan Liu, Jason l. Griffiths, Patrick A. Cosgrove, Jasmine R. McQuerry, Jiayi Liu, Kena Ihle, Eliza R. Bacon, Feng Chi, Pierre Wallet, V. Grolmusz, D. Simons, Benjamin Copeland, Lance Pflieger, Jinfeng Chen, Sumana Majumdar, Terron T. Crowder, R. Emond, Rachel Factor, D. Bowtell, Adam L. Cohen, Daniel Schmolze, Peter P Lee, Lusine Tumyan, James R Waisman, Andrea H Bild, Aritro Nath, Jeffrey T. Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="


 Vasculogenic mimicry (VM) represents an alternative vascularization mechanism contributing to cancer progression and treatment resistance. Despite its clinical significance, the spatial and molecular relationships between VM networks and immune components remain poorly characterized in non-small cell lung cancer (NSCLC).
 Methods: We conducted a comprehensive spatial analysis of treatment-naïve, surgically resectable advanced non-small cell lung cancer (NSCLC) specimens (n=4), encompassing both EGFR-mutant and wild-type variants. High-dimensional spatial phenotyping was performed using PhenoCycler-Fusion multiplexed immunofluorescence technology, enabling precise visualization and quantification of vasculogenic mimicry (VM) architectures and immune cell infiltrates at subcellular resolution. Complementary molecular characterization of discrete tumor microenvironmental regions was achieved through GeoMx Digital Spatial Profiling platform. Spatial transcriptomics analysis was subsequently employed to validate primary observations and elucidate region-specific transcriptional signatures.
 Results: Through high-dimensional spatial analysis, we identified a previously undescribed vessel-like population characterized by CD31-VEGFR1+PanCK+ expression, representing vasculogenic mimicry (VM) structures in non-small cell lung cancer (NSCLC). Comprehensive spatial mapping revealed distinct topographical relationships between VM networks and immune cell populations, including CD4+ and CD8+ T lymphocytes, as well as functionally diverse macrophage subsets (CD68+, CD206+, iNOS+, and HLA-DR+). Molecular characterization through digital spatial profiling demonstrated enhanced transcriptional heterogeneity at the tumor margin, with significant enrichment of pathways involved in extracellular matrix (ECM) organization and hypoxia response (FDR < 0.01). Integration of spatial transcriptomics with an established 70-gene VM signature revealed a core set of 10 genes operating at the intersection of VM and ECM remodeling programs. This gene module, predominantly expressed at the tumor-stromal interface, represents a previously unrecognized regulatory hub coordinating VM formation through ECM modification.
 Conclusions: Our integrated analysis reveals a novel VM phenotype in NSCLC and identifies a core gene signature linking VM formation to ECM remodeling at the invasive front. These findings provide new insights into tumor-stromal-immune interactions and suggest potential therapeutic strategies targeting the VM-ECM-immune axis.



 Chun-Hui Lee, Chun-Hua Hung, Shu-Hsien Wang, Wei-Pang Chung, Yu-Min Yeh, Peng-Chan Lin, Wu-Chou Su. Mapping the complex interplay between vasculogenic mimicry and immune response in lung cancer using digital spatial profiling [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 162.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b881f06a481256ab1b5e06c0de554804f026d184" target='_blank'>
              Abstract 162: Mapping the complex interplay between vasculogenic mimicry and immune response in lung cancer using digital spatial profiling
              </a>
            </td>
          <td>
            Chun-Hui Lee, Chun-Hua Hung, Shu-Hsien Wang, Wei-Pang Chung, Yu-Min Yeh, Peng-Chan Lin, Wu-Chou Su
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSC) is an aggressive malignancy with poor treatment outcomes. Although transformative for some cancer types, immune checkpoint inhibitors (ICIs) have been only marginally successful at best in HGSC. Comprehensive interrogation of the cellular and molecular features of the tumor and immune microenvironment (TIME) of HGSC cases with varying responses to ICI could provide novel insights into the potential drivers of therapeutic response.



 We applied 10x Genomics Visium Spatial Transcriptomics (ST) to archival tissue sections from 10 patients treated with ICI (nivolumab or pembrolizumab) with varying degrees of response. Integrated cluster analysis of 39,147 ST features using Louvain optimization resulted in 20 integrated clusters. Molecular annotation using GSEA, ESTIMATE, and CIBERSORTx was performed on individual clusters to assess spatially resolved cellular transcriptional states within the cohort and by ICI response.



 Bioinformatics analysis of marker gene data from each group (responder or non-responder) provided insights into cell types for each cluster defined by transcriptional states. Major cell type populations by cluster included tumor, immune, fibroblast, epithelial, and smooth muscle. Cluster 1 cell type of both responders and non-responders is molecularly defined by known HGSC genes including CD24, WFDC2, SLPI, and MUC1. Non-responders also expressed CRIP1 and ELF3, which are associated with metastasis and cancer cell stemness, respectively. Pathway enrichment analysis across the entire cohort identified distinct immune-related pathways, including allograft rejection, IL6-JAK-STAT3 signaling and interferon responses, corroborating the immune cell deconvolution findings from CIBERSORTx. Interestingly, clusters 6 and 8 for both groups seem to represent a specialized population of lipid-associated macrophages (LAMs), marked by expression of LYZ, APOE, APOC1, GPNMB. However, in responders, we observed upregulation of antigen presenting genes, including LYZ, C1QA, C1QC, CD74, and CXCR4. In contrast, non-responders had high expression of immunosuppressive macrophages including MMP9 and IFI30 within the LAM compartment. Non-responders demonstrated higher immune composition scores in several clusters, predominantly for M0 and M2 macrophages, which are associated with immunosuppressive phenotypes. In contrast, responders displayed a more immune-activated profile, with increased infiltration of CD8+ T cells, activated CD4+ memory T cells, and activated NK cells. Notably, M2 macrophages were more abundant in the tumor cluster of non-responders, indicating a more immunosuppressive TME.



 These results underscore the heterogeneity within the tumor and immune microenvironments in HGSC tumors that may uncover novel markers or targets of response to ICI.



 Jing Qian, Lynda D. Roman, Rania Bassiouni, Hanim I. Ozkizilkaya, Kathy Kwock, Yuxin Jin, Javier Arias-Stella, Joseph W. Carlson, John D. Carpten. Spatial transcriptomics reveals differences in the tumor and immune microenvironment of high-grade serous ovarian cancers with differing responses to immune checkpoint inhibitors [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB076.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29faac1f752106215ff7dadafa3504675aecf0f8" target='_blank'>
              Abstract LB076: Spatial transcriptomics reveals differences in the tumor and immune microenvironment of high-grade serous ovarian cancers with differing responses to immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Jing Qian, Lynda D. Roman, Rania Bassiouni, Hanim I Ozkizilkaya, Kathy Kwock, Yuxin Jin, Javier Arias-Stella, Joseph W. Carlson, John D. Carpten
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/193dde17ef6934508de69e6a33e445fcab88581e" target='_blank'>
              Impaired T cell and neoantigen retention in time-serial analysis of metastatic non-small cell lung cancer in patients unresponsive to TIL cell therapy.
              </a>
            </td>
          <td>
            Chao Wang, Xiaoqing Yu, Jamie K Teer, Jiqiang Yao, Dongliang Du, Xiaoxian Liu, Zachary J Thompson, Min Hsuan Wang, Eric A. Welsh, Danish Memon, Timothy A Chan, Vladimir Makarov, C. Anadon, Lamees Saeed, Theresa A. Boyle, Bin Fang, John M. Koomen, Cheryl A Cox, Ana M Landin, Sean J Yoder, Sungjune Kim, Dung-Tsa Chen, Shari A Pilon-Thomas, Jose R Conejo-Garcia, S. Antonia, E. Haura, Ben C Creelan
          </td>
          <td>2025-05-08</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains an untreatable disease. Understanding of GBM’s infiltrative biology at the resection margin is limited, despite causing disease recurrence and progression. To address this, we generated a high-throughput single-nucleus (sn)RNA-seq and snATAC-seq multi-omic dataset from six tumors with distinct genomic drivers and combined it with spatial transcriptomics to characterize the unique molecular phenotype of GBM near the margin. By contrasting GBM-specific biology in matching “Core” vs. “Margin” dissections, we define unique, shared “GBM infiltration” and chromatin accessibility signatures near the margin. We prioritize EGFR as a top differentially expressed and accessible “Margin” marker across GBM subtypes, show its dynamic expression along a core-to-margin infiltration trajectory, and validate its role in migration through CRISPR/Cas9 deletion in two patient-derived models. ChIP-seq studies further corroborate preferential TEAD1 binding at EGFR’s accessible regulatory elements. This validated multi-omic dataset enables further studies into tumor and microenvironment biology in the context of residual GBM disease.


 Balagopal Pai, Susana Ramos, Wan Sze Cheng, Tanvi Joshi, Gabrielle Price, Jessica Tome-Garcia, Kristin Beaumont, Robert Sebra, Raymund Yong, Elena Zaslavsky, Nadejda M. Tsankova. Spatial multi-omics defines a shared glioblastoma infiltrative signature at the resection margin [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5290.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db10e8dfa73f80517f73e1a05e405d99f33973b4" target='_blank'>
              Abstract 5290: Spatial multi-omics defines a shared glioblastoma infiltrative signature at the resection margin
              </a>
            </td>
          <td>
            Balagopal Pai, S. Ramos, W. Cheng, T. Joshi, Gabrielle Price, Jessica Tomé-García, Kristin Beaumont, Robert P Sebra, Raymund Yong, E. Zaslavsky, N. Tsankova
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 e15561


 Background:
 Colorectal cancer (CRC), the third most prevalent cancer globally, remains a major cause of cancer-related mortality. Genomic and epigenomic plasticity during CRC progression, particularly metastasis, poses significant challenges to effective treatment. Patient-derived organoids (PDOs) reflect the heterogeneous genetic and morphological composition of cancer cells, enabling ex vivo disease modeling. This study investigates the epigenomic makeup of human primary and metastatic CRC to advance precision epigenetic therapies.
 Methods:
 We analyzed 45 organoid models, including 40 CRC organoids (from primary tumors and liver/lung metastases) and 5 generated from healthy colonic tissue. Using integrative epigenomic profiling, including chromatin accessibility, chromatin state mapping (via six histone modifications), and 3D genome conformation, we constructed a comprehensive CRC epigenomic landscape.
 Results:
 Our analysis of CRC patient-derived organoids revealed a core set of accessible genomic regions shared across all organoids, irrespective of tumor location or metastatic status. This conserved chromatin signature highlights common regulatory pathways in CRC during disease progression. Unsupervised clustering based on epigenomic features revealed four distinct CRC subtypes, encompassing both primary and metastatic models. Notably, organoids derived from liver and lung metastases did not cluster by metastatic site, underscoring epigenomic convergence. Subtype D comprised 57.1% (8/14) of primary tumors, while subtypes B and C contained 47.3% (9/19) of liver metastases and 42.8% (3/7) of lung metastases, respectively. KRAS mutation status further stratified subtypes, with wild-type KRAS observed exclusively in subtypes A and B, while KRAS-mutant variants were enriched in subtypes C (6/7) and D (4/10). Histone modification patterns, defined by six marks (H3K4me1, H3K4me3, H3K27me3, H3K27ac, H3K36me3, and H3K9me3) revealed subtype-specific regulatory chromatin landscapes. Three-dimensional chromatin interactions were mapped, and genome-wide interaction contacts are used to identify various chromatin structures that may influence metastatic capacity.
 Conclusions:
 This study highlights distinct and shared epigenomic signatures in CRC PDOs, offering novel insights into tumor progression and metastasis. Integrative chromatin analysis provides a framework for identifying epigenetic biomarkers and epigenome targeting strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66760aad28585c2f41842514d0645e6e13f51968" target='_blank'>
              Deciphering colorectal cancer progression through integrative epigenomic analysis for precision therapeutics.
              </a>
            </td>
          <td>
            Ghazaleh Tavallaee, Amin Nooranikhojasteh, Elias Orouji
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Biological systems exhibit remarkable heterogeneity, characterized by intricate interplay among diverse cell types. Resolving the regulatory processes of specific cell types is crucial for delineating developmental mechanisms and disease etiologies. While single-cell sequencing methods such as scRNA-seq and scATAC-seq have revolutionized our understanding of individual cellular functions, adapting bulk genome-wide assays to achieve single-cell resolution of other genomic features remains a significant technical challenge. Here, we introduce Deep-learning-based DEconvolution of Tissue profiles with Accurate Interpretation of Locus-specific Signals (DeepDETAILS), a novel quasi-supervised framework to reconstruct cell-type-specific genomic signals with base-pair precision. DeepDETAILS’ core innovation lies in its ability to perform cross-modality deconvolution using scATAC-seq reference libraries for other bulk datasets, benefiting from the affordability and availability of scATAC-seq data. DeepDETAILS enables high-resolution mapping of genomic signals across diverse cell types, with great versatility for various omics datasets, including nascent transcript sequencing (such as PRO-cap and PRO-seq) and ChIP-seq for chromatin modifications. Our results demonstrate that DeepDETAILS significantly outperformed traditional statistical deconvolution methods. Using DeepDETAILS, we developed a comprehensive compendium of high-resolution nascent transcription and histone modification signals across 39 diverse human tissues and 86 distinct cell types. Furthermore, we applied our compendium to fine-map risk variants associated with Primary Sclerosing Cholangitis (PSC), a progressive cholestatic liver disorder, and revealed a potential etiology of the disease. Our tool and compendium provide invaluable insights into cellular complexity, opening new avenues for studying biological processes in various contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/275a4074dd5de5430ad371087a262308cca6fd4d" target='_blank'>
              High-resolution reconstruction of cell-type specific transcriptional regulatory processes from bulk sequencing samples
              </a>
            </td>
          <td>
            Li Yao, Sagar R. Shah, A. Ozer, Junke Zhang, Xiuqi Pan, Tianyu Xia, Vrushali D. Fangal, A. K. Leung, Meihan Wei, John T. Lis, Haiyuan Yu
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="

 Patient-derived organoids (PDOs) offer new opportunities to model various cancers. However, their application in prostate cancer (PCa) has been hampered by poor success rates and overgrowth of cell types which are not representative of the patient samples. Here, we aimed at modulating culture conditions and investigating in vitro-associated cellular heterogeneity to identify refined PCa PDO culture conditions.



 A cohort of 164 diverse samples was obtained from 162 PCa patients undergoing radical prostatectomy, prostatic biopsies, transurethral resection of the prostate or the bladder or metastasis biopsy/resection. Out of this cohort, PDOs were established in various extracellular matrices (ECM: Matrigel, Collagen I, Collagen IV, Laminin-I, or no ECM) and medium compositions. PDOs were characterized using IF, IHC, and FISH. Single-cell RNA sequencing (scRNA-seq) was performed on 11 tumor and organoid samples, followed by unsupervised clustering, trajectory inferring, cell type annotation, and CNV analysis. Published spatial and bulk transcriptomic data were re-analyzed to validate specific markers. A prostate PDO scRNA-seq atlas was generated by integrating three published PDO datasets (Huang 2023, Song 2022, McCray 2019) together with our newly-generated data.



 Out of 5 ECM conditions, an ECM-free culture system increased the take-rate of PDOs with luminal-like (CK5-, CK8+, AR+) and PCa features (AMACR+, PTEN-), while standard Matrigel-based culture conditions yielded basal-like benign organoids (CK5+, AR-, AMACR-). Cellular composition of organoids generated in ECM-free conditions differed significantly from those generated in Matrigel, with minimal overlap observed in their transcriptomic profiles. ECM-free PDOs comprised cell populations associated with known PCa signatures, exhibited transcriptomic resemblance with their respective parental tumors, and maintained patient-specific epithelial populations. Furthermore, we defined organoid cell type signatures and identified markers discriminating between several types of benign cells (basal, hillock, club, transitioning) versus tumor cell populations ex vivo and in situ. Finally, our integrative single-cell atlas highlighted that Matrigel-based organoids derived from primary PCa are essentially composed of benign epithelial cells, irrespective of the dataset or the malignant nature of the tissue of origin. In contrast, ECM-free conditions preserved heterogenous tumor-like cell populations with higher inferred copy-number variations and were enriched in intermediate club-like cell populations.



 ECM-free PCa PDOs exhibit improved fidelity and in vitro-enriched cell populations, while aberrant benign lineage programs are associated with Matrigel culture. Our work contributes to significantly enhancing the potential of PDOs in PCa research.



 Robin Dolgos, Romuald Parmentier, Jing Wang, Raphaëlle Servant, Arnoud J. Templeton, Tobias Zellweger, Alastair Lamb, Kirsten Mertz, Svetozar Subotic, Tatjana Vlajnic, Helge Seifert, Ashkan Mortezavi, Cyrill A. Rentsch, Lukas Bubendorf, Clementine Le Magnen. ECM-free patient-derived organoids preserve diverse prostate cancer lineages and uncover in vitro-enriched cell types [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5205.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417776fcb75ac4a7513d08a7fa65829e73ccec03" target='_blank'>
              Abstract 5205: ECM-free patient-derived organoids preserve diverse prostate cancer lineages and uncover in vitro-enriched cell types
              </a>
            </td>
          <td>
            Robin Dolgos, R. Parmentier, Jing Wang, Raphaëlle Servant, Arnoud J. Templeton, T. Zellweger, Alastair D Lamb, Kirsten Mertz, S. Subotic, T. Vlajnic, Helge Seifert, A. Mortezavi, Cyril Rentsch, Lukas Bubendorf, C. Le Magnen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 Immune checkpoint inhibitor (ICI) therapies have markedly improved the prognosis for patients with stage III & IV metastatic melanoma by prolonging progression-free and overall survival rates. However, the variability in immune evasion and resistance mechanisms presents significant challenges to the clinical efficacy of ICIs. This project aims to define drivers of immunotherapy response and resistance by employing advanced genomic, single-cell mRNA analyses, and spatial profiling techniques on tissue biopsies from metastatic melanoma patients.



 In this study, we developed a framework to analyze response and resistance, both intrinsic and acquired, via immune features in the tumor microenvironment in a standardized, uniformly processed, and deeply clinically annotated cohort of metastatic melanoma patients (n=61) treated with ICB as part of the human tumor atlas network (HTAN) initiative. From the tumor samples, we conducted single-nucleus RNA sequencing, and for a subset of the samples high-resolution spatial imaging (including protein mIHC, CODEX, and transcriptomics MERFISH). Standardized processing and data pipelines allowed for integrating genomic, transcriptomic, and spatial features to elucidate characteristics and mechanisms in tumor microenvironment and their relationships with resistance.



 Studies of the pretreatment samples demonstrated that CD4 and CD8 T cells, particularly TCF7+ CD8 T cells, are significantly more prevalent in responders to immunotherapy. Conversely, macrophages, especially Angio-TAMs, show higher levels of enrichment in non-responders. Moreover, the presence of B cells and follicular dendritic cells in non-lymph node biopsies supports the presence of tertiary lymphoid structures within the TME. Three levels of immune enrichment were identified through spatial analysis, and samples with more immune enrichment tend to have better responses. We also identified 10 recurrent cellular neighborhoods (RCNs) and found that RCN2,4, and 7 with high lymphocyte infiltration are significantly more enriched in responders than non-responders. In addition, RCN4 is associated with immune infiltration while RCN7 is TLS-like.



 Our findings indicate that patients with low immune infiltration exhibited enrichment of macrophages associated with the hypoxia and angiogenesis phenotypes, while patients with high immune infiltrates displayed enrichment in lymphocytes, particularly TCF7+ CD8+ T cells, confirming previous findings and indicating a robust T cell-mediated immune response. This project integrates genomic, transcriptomic, and spatial features to elucidate shared tumor and microenvironmental states and their relationships with resistance, and guide more personalized and effective treatment strategies for metastatic melanoma.



 Xinyu Cui, Giuseppe Tarantino, Yiwen He, Priyanka Solanky, Kathleen Pfaff, Aaron R. Thorner, Tyler J. Aprati, Boyang Zhang, Timothy Blosser, Emily Robitschek, Jiajia Chen, Junko Tsuji, Elliot Boblitt, Allison Frangieh, Hannah M. Faulkner, Marta Holovatska, Aleigha Lawless, Michael Manos, Karla Helvie, Tatyana Sharova, Dennie Frederick, James L. Fahey, Diego Villamarin, Sachi Krishna, Chanell Mangum, Ajit J. Nirmal, Domenic Abbondanza, Cai McCann, Bruce Johnson, Alex K. Shalek, Eliezer Van Allen, Xiaowei Zhuang, Ryan Sullivan, Barbara E. Engelhardt, Samouil L. Farhi, Scott Rodig, F. Stephen Hodi, Genevieve M. Boland, David Liu. Dissecting tumor-immune microenvironment in response and resistance to immune checkpoint blockade in metastatic melanoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4536.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149ae335dfeb331ab7d81fc0ac4db62e86f66444" target='_blank'>
              Abstract 4536: Dissecting tumor-immune microenvironment in response and resistance to immune checkpoint blockade in metastatic melanoma
              </a>
            </td>
          <td>
            Xinyu Cui, Giuseppe Tarantino, Yiwen He, Priyanka Solanky, K. Pfaff, A. Thorner, Tyler J. Aprati, Boyang Zhang, Timothy R Blosser, Emily J. Robitschek, Jiajia Chen, Junko Tsuji, Elliot Boblitt, Allison Frangieh, H. Faulkner, Marta M. Holovatska, Aleigha Lawless, M. Manos, K. Helvie, T. Sharova, Dennie T. Frederick, J. L. Fahey, Diego Villamarin, Sachi Krishna, Chanell Mangum, A. J. Nirmal, Domenic Abbondanza, Cai McCann, B. Johnson, Alex K. Shalek, Eliezer Van Allen, Xiaowei Zhuang, Ryan J. Sullivan, Barbara Engelhardt, Samouil L. Farhi, Scott J Rodig, F. S. Hodi, Genevieve M. Boland, David Liu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 The tumor microenvironment (TME) is the cellular environment in which cancer cell exist and is made up of a combination of stroma cells and extra cellular matrix (ECM), secreted biomolecules, including cytokines, from both stroma cells and tumor cells. The TME composition drives tumor growth and drug resistance. We previously established a 96-wells based bioprinted glioblastoma (GBM) model that includes a neurovascular unit (NVU) made up of endothelial cells, astrocytes and pericytes. We demonstrated its applicability for drug screening by using high content imaging system to measure tumor size and vascular integrity. We then performed single cell RNA sequencing on this bioprinted NVU-GBM model. We used the single cell transcriptomics data to investigate ligand-receptor interactions between stroma and GBM cells, to find molecular pathways that potentially drive tumor growth and be targets for pharmacological intervention as anticancer drugs. We identified 50 ligand-receptors pairs and selected those with genes exclusively expressed in GBM cells but not in the stroma cells, to minimize potential toxic effects of drugs to the non-cancer cells. We performed a drug screen of 34 compounds targeting many of these receptors on the GBM cells, and identified 15 compounds that reduced GBM cells growth, with minimal effects on vascular integrity. This result demonstrates a promising pharmacogenomics approach to using bioengineered tumor growth tissue models that include the TME, integrated with single cell transcriptomics, to find actionable pharmacological targets that inhibit tumor growth without affecting the stroma cells.


 Yen-Ting Tung, Min Jae Song, Marc Ferrer. Single cell transcriptomics on a bioprinted 3D neurovascular unit glioblastoma model identifies pharmacological interventions that selectively target tumor cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3759.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c4c0df9de0d71dd67f18dd27bf1bf7b9c3857af" target='_blank'>
              Abstract 3759: Single cell transcriptomics on a bioprinted 3D neurovascular unit glioblastoma model identifies pharmacological interventions that selectively target tumor cells
              </a>
            </td>
          <td>
            Yen-Ting Tung, M. J. Song, Marc Ferrer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 The development of metastatic disease is a conserved feature across most solid organ malignancies and accounts for the majority of cancer-related deaths. While adaptations within the local immune microenvironment of primary tumors are well-appreciated hallmarks of tumor progression, there remains much unknown about the specific impact of metastasis on both local and systemic immune responses. Here, we present an analysis of changes in the adaptive immune response to tumors during metastasis and clonal selection. Our group previously developed a syngeneic melanoma mouse model of spontaneous lymph node metastasis, performing nine generations of in vivo serial passaging to select for metastatic phenotypes. As previous work suggests, we find that tumors undergo clonal selection during metastasis using lentivirus integration mapping to measure clonality. Even amidst reduced genomic diversity, we find that metastases exhibit comparable or increased transcriptomic diversity when compared to primary tumors. In light of this discordance between genomic and transcriptomic diversity, we ask whether the immune response, especially T cells, exhibit similar dynamics. Performing both bulk and single cell T cell receptor (TCR) sequencing, we find that T cell clonality mirrors the reduced clonal diversity of metastatic tumors both in the total T cell compartment and specifically in regulatory T cells. Through paired investigation of evolution of tumors and the responding immune cells, we develop a model of tumor-immune coevolution in which reduced tumor clonality selects for narrower T cell repertoires; yet, diverse tumor transcriptional profiles support the differentiation of a variety of T cell subsets that are advantageous for tumors including regulatory T cells and exhausted CD8+ T cells. We further perturb this dynamic by evolving new metastatic cell lines in the absence of T cell pressure, finding unique evolution of metastatic pathways when the immune system is altered, impairing metastasis when reimplanted into immunocompetent mice. This model of tumor-immune coevolution establishes the importance of ongoing, dynamic communication between the host and tumors throughout metastatic progression.


 Cort B. Breuer, Marcos Labrado, Nathan E. Reticker-Flynn. Antagonism between tumors and lymphocytes drives coevolutionary processes throughout metastasis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3458.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d9f71c97b02d4a1d072a400c199093fb87572c0" target='_blank'>
              Abstract 3458: Antagonism between tumors and lymphocytes drives coevolutionary processes throughout metastasis
              </a>
            </td>
          <td>
            Cort B Breuer, Marcos Labrado, Nathan E. Reticker-Flynn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer remains the second leading cause of death globally, posing an ongoing threat to public health. A hallmark of cancer cells is their capacity to invade adjacent tissues and evolve into malignant forms, often resulting in aggressive tumors resistant to conventional treatments. At the heart of this therapeutic challenge are cancer stem cells (CSCs), which possess distinctive capabilities for self-renewal, differentiation, and generation of diverse tumor cell populations. These CSCs have been identified across multiple tissue types, including lung, colon, breast, pancreas, and ovary. Research has demonstrated that CSC subpopulations contribute significantly to therapeutic resistance, tumor recurrence, and metastasis by regulating multiple signaling pathways, making them compelling targets for cancer therapy. Notably, emerging evidence suggests that natural products may offer protective benefits against cancer development while potentially targeting CSCs. This review synthesizes current knowledge of CSCs, examining their identifying markers, isolation techniques, study methods, and associated signaling pathways. Additionally, we explore various natural products that specifically target CSCs across different cancer types, presenting potential strategies to address the persistent challenges of drug resistance and cancer relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cedb6597be1303e7e75516814053b5e6b9747717" target='_blank'>
              Natural products targeting cancer stem cells: a promising therapeutic approach
              </a>
            </td>
          <td>
            Julia K. Opara, Sierra Sanchez, S. M. Thomas, S. Anant
          </td>
          <td>2025-04-18</td>
          <td>Exploration of Drug Science</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="
 Aneuploidy, prevalent in most solid tumors, significantly alters cellular phenotype and fitness. Despite its critical role in tumor evolution, a detailed quantitative understanding of its evolutionary dynamics remains elusive. To address this gap, we developed ALFA-K, a novel method to infer the fitness landscape of chromosome-level karyotypes. Utilizing longitudinal single-cell karyotype data from evolving cell populations, our method estimates the fitness of thousands of karyotypes proximal to the input data in karyotype space, enabling predictions of unseen karyotypes. We validated our approach using both synthetic data from an agent-based model (ABM) and empirical data from in vitro and in vivo passaged cell lines. The empirical data comprised over 40, 000 sequenced cells across five cell lines, each sequenced longitudinally at 5-10 timepoints per lineage. Analysis of the fitted landscapes yielded several key insights: 1) WGD alters the distribution of fitness effects of copy number alterations (CNAs) (K.S. test: P<10-). Specifically, karyotypes with WGD exhibit more mutations with positive selection coefficients compared to diploid karyotypes; 2) Mutations in cisplatin-treated lineages showed changes in fitness effects on average 2.3 times larger than those in untreated lineages (t-test: P<2.2×10-1); 3) The fitness impact of a CNA varies depending on the parental karyotype; 4) The rate of chromosome missegregation can dictate the predominant karyotypes in an evolving population.
 To further investigate how the tumor microenvironment shapes karyotype evolution, we are integrating these inferred fitness landscapes into an existing spatial ABM. This modeling simulates the spatially heterogeneous tumor microenvironment and its influence on karyotypic selection and adaptation. By leveraging this ABM, we aim to capture the effects of spatial constraints—such as resource availability, competition, and cellular interactions—on the fitness and distribution of karyotypes. Complementing these simulations, we are developing a bioinformatics pipeline for spatial transcriptomics data from glioblastoma clinical samples. This dataset includes samples from various regions (normal, tumor, and tumor-infiltrating) across 31 samples from 19 patients, using 10X Visium with an average of 3, 100 spots per sample. The pipeline is designed to provide detailed information about the spatial organization of karyotypic diversity, elucidating how aneuploid karyotypes are distributed and interact within the tumor microenvironment. By integrating this spatially resolved data with the ABM, we aim to generate a comprehensive framework to explore how microenvironmental pressures in glioblastoma tumors shape the evolution of karyotypic diversity and drive the emergence of therapeutic resistance in patient-specific contexts.


 Richard J. Beck, Noemi Andor. Decoding tumor evolution through fitness landscapes of aneuploid cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2486.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Accumulating research suggests that Epstein-Barr Virus-positive Diffuse Large B-cell Lymphoma (EBV+DLBCL) is associated with immune dysfunction and tumor microenvironment (TME) heterogeneity. While the prognostic role of the TME in EBV-DLBCL is established, its impact on EBV+DLBCL survival remains unclear. Here, we integrated 10X Visium spatial transcriptomics (ST) with single-cell RNA sequencing (scRNA-seq) to map TME heterogeneity in EBV+DLBCL. Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses identified PD-1/PD-L1 signaling as a hallmark of EBV+DLBCL’s immunosuppressive TME. Functional validation using the PD-1/PD-L1 inhibitor BMS202 revealed dose-dependent suppression of proliferation and enhanced apoptosis in EBV+Farage cells, with TLR4 emerging as a downstream effector showing EBV-status-dependent regulation. These findings not only link TME-driven PD-1/PD-L1 activation to EBV+DLBCL’s poor prognosis but also provide preclinical evidence for PD-1/PD-L1 blockade as a therapeutic strategy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00410-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1796a1ec9b1ce6b656d31aa53096b6765637e098" target='_blank'>
              Spatial transcriptomics reveals tumor microenvironment heterogeneity in EBV positive diffuse large B cell lymphoma
              </a>
            </td>
          <td>
            Mei-Yao He, Meng Liu, Jiayin Yuan, Jin Lv, Wei Li, Qianwen Yan, Yujiao Tang, Luyi Wang, Li Guo, Fang Liu
          </td>
          <td>2025-05-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose Bladder cancer (BC) is characterized by high heterogeneity, with non-muscle-invasive (NMIBC) and muscle-invasive (MIBC) stages differing significantly in clinical behavior and outcomes. The transition from NMIBC to MIBC involves extensive tumor microenvironment (TME) remodeling, particularly in endothelial cells (ECs), which drive angiogenesis and modulate immune and extracellular matrix (ECM) interactions. However, the precise roles of ECs in this progression remain poorly defined. Methods Public single-cell RNA sequencing (scRNA-seq) datasets from 47 BC patients were analyzed to characterize endothelial cell heterogeneity and functional states across NMIBC and MIBC. Computational tools such as CellChat were applied to reconstruct cell–cell communication networks, focusing on pathways related to angiogenesis, immune crosstalk, and ECM remodeling. Results Twelve major cell types were identified, with endothelial cells exhibiting distinct transcriptional profiles between NMIBC and MIBC. NMIBC-associated ECs promoted adhesion and migration through HMGB1 and CXCL12 signaling. In contrast, MIBC was enriched in an ADAM10+ endothelial subset associated with vascular remodeling and activation of Wnt signaling via CTNNB1. Key ligand-receptor interactions highlighted the dynamic roles of ECs in TME modulation during BC progression. Conclusions This study reveals stage-specific endothelial cell phenotypes and signaling networks in BC. The identification of an MIBC-specific ADAM10+ endothelial subset underscores its potential role in driving tumor progression and highlights opportunities for stage-adapted vascular-targeted therapies. These findings advance our understanding of BC pathogenesis and provide the foundation for novel therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02297-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7505578630274eaf54930e9821b19f4ea81e3ca3" target='_blank'>
              A single-cell atlas of bladder cancer unveils dynamic cellular composition and endothelial functional shifts during progression
              </a>
            </td>
          <td>
            Hongjian Song, Guixiang Xie, Yaowei Li, Xiaowei Hu, Zongzheng Yang, Yubo Zhao, Qing Shi, Haonan Li, Ziyi Liu, Zhihao Yin, Ziqi Wang, Zhichao Tong, Wanhai Xu
          </td>
          <td>2025-04-09</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Metastatic bladder cancer (BLCA) is an aggressive disease complicated by the emergence of variant histological subtypes and variable responses to treatment. Understanding the heterogeneity that exists both between these histological subtypes and within individual BLCA tumors is imperative in advancing precision medicine for BLCA.



 The University of Washington/Fred Hutchinson Cancer Center metastatic BLCA rapid autopsy program is a first-of-its-kind program collecting both primary and metastatic tumor samples from advanced BLCA patients, with a focus on aggressive variant subtype tumors. To explore intra-tumoral heterogeneity of both tumor and stromal compartments, single-nucleus RNA sequencing, in combination with matching bulk DNA and RNA sequencing, was performed on 15 tumors from 9 patients. The resulting dataset totaled 39,888 cells across a variety of histological subtypes: conventional urothelial carcinoma (UC), UC with squamous differentiation (UCSD), and plasmacytoid UC (PUC).



 Our unique snRNAseq dataset, the first in either metastatic or variant subtype BLCA, was used to dissect genomic and transcriptomic heterogeneity within BLCA tumors. On a genomic level, we observe subclonal copy number variations (CNVs) in BLCA driver genes in almost all (8/9) patients. Furthermore, we observe varying levels of intra-tumoral transcriptomic heterogeneity, where metastatic survival time is positively associated with tumor heterogeneity. Despite this heterogeneity, particular pathways define the epithelial transcriptomics of variant histological subtypes, including the upregulation of cell cycle and DNA repair upregulated in PUC tumors, which is validated in both bulk RNA sequencing and immunohistochemistry of a larger UW/FH BLCA rapid autopsy cohort. Investigation of the tumor microenvironment revealed that PUC tumors may be more immune-hot than UC, with increased B- and T-cell infiltration, while UCSD tumors have immunosuppressive microenvironments enriched in tissue-associated macrophages. Interestingly, these differences across subtypes are supported by changes in inter-cellular signaling between epithelial and fibroblast cells and the immune cells.



 These findings lay the groundwork for development of precision medicines for histological subtypes in BLCA by revealing heterogeneity and immune microenvironments that underlie disease aggressiveness and therapeutic resistance.



 Samantha L. Schuster, Pushpa Itagi, Sonali Arora, Thomas Persse, Patricia C. Galipeau, John K. Lee, Robert B. Montgomery, Jonathan L. Wright, Petros Grivas, Funda Vakar-Lopez, Andrew C. Hsieh, Hung-Ming Lam, Gavin Ha. Histological subtypes of metastatic bladder cancer have distinct tumor microenvironments and high intratumoral heterogeneity [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB468.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827f74e9a451d846709a409162aebd5ef66250d5" target='_blank'>
              Abstract LB468: Histological subtypes of metastatic bladder cancer have distinct tumor microenvironments and high intratumoral heterogeneity
              </a>
            </td>
          <td>
            Samantha L. Schuster, P. Itagi, Sonali Arora, Thomas W Persse, Patricia Galipeau, John K. Lee, R. Montgomery, Jonathan L. Wright, P. Grivas, F. Vakar‐López, Andrew C. Hsieh, Hung-Ming Lam, Gavin Ha
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Metastasis is responsible for approximately 90% of cancer-related deaths, yet current therapies fail to effectively address this critical challenge. The tumor microenvironment (TME), particularly carcinoma-associated fibroblasts (CAFs), plays a pivotal role in metastatic progression by driving extracellular matrix (ECM) remodeling, epithelial-to-mesenchymal transition (EMT), and immune evasion. Within metastatic CAFs (mCAFs), we identified a key protein that regulates these processes and developed Akos, a small molecule that selectively inhibits this protein, reprogramming the TME to disrupt the metastatic cascade. This study evaluates the preclinical efficacy, specificity, and safety of Akos across multiple solid tumor models.



 Primary stromal cell cultures were obtained from needle biopsies of primary tumors from patients with advanced/metastatic cancer (mCAFs), non-metastatic cancer (CAFs), and non-cancerous patients (BAFs). Protein expression was validated through gene profiling, immunohistochemistry, and western blot. In vitro assays were conducted to assess Akos’ ability to inhibit mCAF-mediated tumor cell invasion, proliferation, and EMT marker expression. In vivo xenograft models of prostate, breast, colorectal, and pancreatic cancers were utilized to evaluate Akos’ effects on tumor growth, weight, and metastatic spread. A spontaneous metastasis assay, replicating the full metastatic cascade, was employed to evaluate its anti-metastatic potential. Toxicology and pharmacokinetic studies were conducted to assess systemic safety and drug stability.



 Akos demonstrated significant in vitro efficacy, inhibiting mCAF-driven tumor cell invasion and proliferation while reducing EMT marker expression. In in vivo xenograft models, Akos reduced tumor volume by 50-fold and tumor weight by 30-fold compared to untreated controls. In the spontaneous metastasis assay, Akos completely blocked 100% of metastatic foci across all tested models. Toxicology studies confirmed Akos’ safety profile, with no significant off-target effects observed, while pharmacokinetics showed stable systemic exposure.



 These findings position Akos as a promising therapeutic candidate for targeting the TME to address the unmet need in metastatic cancer treatment. By inhibiting a key protein in mCAFs, Akos effectively disrupts the processes driving metastatic progression, offering a novel approach to halting the spread of cancer. The observed efficacy across multiple tumor types, combined with its favorable safety profile, supports further clinical investigation of Akos as a first-in-class therapeutic for metastatic solid tumors. These results underscore the potential of targeting the TME to improve patient outcomes and advance the standard of care in oncology.



 Javier Cerda-Infante, Benjamin Prieto, Valentina Cerda, Muriel Nuñez. Akos: A novel CAF-targeted therapy to reprogramming the tumor microenvironment to reverse metastatic traits in all solid tumors [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5666.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29fa9279efe44a23f2837544c024f634f88198ac" target='_blank'>
              Abstract 5666: Akos: A novel CAF-targeted therapy to reprogramming the tumor microenvironment to reverse metastatic traits in all solid tumors
              </a>
            </td>
          <td>
            J. Cerda-Infante, Benjamín Prieto, Valentina Cerda, M. Nuñez
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 The mechanisms underlying the interaction between disseminated tumor cells (DTCs) and their tissue microenvironment during metastasis colonization are poorly understood. We integrated multimodal single-cell and spatial profiling from liver cancer mouse models and human metastases to track the spatiotemporal dynamics of DTCs and their microenvironments across nine sequential time points from single-cell seeding until overt lung metastasis. We identified a residual population of quiescent Phgdhhigh DTCs that survive initial innate immune clearance and become transiently enriched in micrometastases. These cells shape an immune-scarce microenvironment through PHGDH-dependent, H3K27me3-mediated epigenetic silencing of chemokine transcription, promoting metastatic expansion. Cx3cr1high interstitial macrophages were also found to be transiently enriched before DTCs expansion, creating an immune-privileged niche for metastatic outgrowth by recruiting immunosuppressive cells. Inactivating the Phgdh-H3K27me3 axis in DTCs or depleting interstitial macrophages restored immune surveillance and inhibited metastatic colonization. These findings provide new therapeutic concepts for developing micrometastasis-targeting regimens.


 Liang Wu, Yu Zhong, Shang liu, Zefan Zhang, Chunqing Wang, Yunfan Sun, Jia Fan, Jian Zhou, Xun Xu. Spatiotemporal multi-omics uncover tumor ecosystem dynamics driving metastatic colonization [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 163.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d9f43797fc3f6bdf9945ba43d2d8daf8093af3c" target='_blank'>
              Abstract 163: Spatiotemporal multi-omics uncover tumor ecosystem dynamics driving metastatic colonization
              </a>
            </td>
          <td>
            Liang Wu, Yu Zhong, Shang liu, Zefan Zhang, Chunqing Wang, Yunfan Sun, Jia Fan, Jian Zhou, Xun Xu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 9562


 Background:
 The groundbreaking neoadjuvant ICB clinical trials have established beyond doubt that ICB before surgery will become the new standard of care for metastatic melanoma. Importantly, the shift from radiographic to pathologic response scoring offers an unprecedented window of opportunity to interrogate a goldmine of ‘on-treatment’ biospecimens. To date, single-cell profiling and region-based spatial transcriptomics (ST) have highlighted the importance of tertiary lymphoid structures (TLS) and stem-like T-cells as positive predictors of ICB response. However, the molecular mechanisms by which tumor infiltrating lymphocytes communicate and organize multicellular immune hubs within the spatial context of the tumor ecosystem remains poorly understood.
 Methods:
 To identify robust biomarkers of response to neoadjuvant ICB, we assembled a cohort of 58 stage III melanoma patients treated with neoadjuvant ICB [24 ipilimumab-nivolumab (IPI-NIVO), 21 NIVO-relatlimab (RELA), 13 PD1 mono] and deployed transformative technologies and computational methods, including single-cell FFPE sequencing, multiplexed FISH single-cell ST (MERFISH 305 genes), digital pathology, 3D open-top light-sheet imaging and AI-based computational pathology, to decipher the complex neoadjuvant ICB tumor ecosystem in response to therapy. We also developed 2 critical computational tools, SCIRA (Spatial Cellular Interaction and Receptor Activation), to compute receptor-ligand (R-L) interactions in whole slide images, and GC-SCAN (graph-based spatial clustering against noise), a graph-based algorithm that quantifies locally clustered structures from spatial -omics data.
 Results:
 Our results showed that the quantity and size of hyper-expanded germinal center/TLS, with increased GC: non-GC B-cell ratio, plasma cells and spatially resolved stem-like T-cells are strongly associated with response. IPI-NIVO elicited significantly stronger GC proliferation compared to NIVO-RELA and PD1 monotherapy, suggesting anti-CTLA4 can robustly induce germinal center/TLS proliferation. SCIRA spatial R-L analyses revealed the critical chemokine R-L interactions that organize the GC- and T-cell zones in response to therapy. Lastly, 3D light-sheet imaging revealed remarkable morphologic heterogeneity in 3D, with interconnected GC-TLS networks that are indicative of long-range molecular gradients.
 Conclusions:
 Our investigations herein have provided a comprehensive characterization of the immune architectures, cellular communications and 3D large-scale morphologic organizations of the TME that drive response to neoadjuvant ICB therapy. We believe the results of this study will enable the development of robust predictive biomarkers to guide the design of next generation combination ICB therapies in the clinical trial setting for melanoma and other cancer types.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb50bf439bc9a3d8fb630c6a0899f89e6769df7" target='_blank'>
              Multidimensional, spatially resolved immunologic hallmarks of response to neoadjuvant immune checkpoint blockade (ICB) therapies.
              </a>
            </td>
          <td>
            Zichao Liu, Xiaofei Song, Shikhar Dhingra, Wei-Shen Chen, Justin He, Jodi A. Balasi, Jonathan Nguyen, Carlos M. Morán-Segura, Joseph Johnson, Chaomei Zhang, Zena Sayegh, Douglas C. Marchion, Sean Yoder, Nicholas Reder, Jeffrey H. Chuang, Pei-Ling Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction Unique from the other tumor cells, tumorigenic cancer stem cells (CSCs) manifest as a subpopulation of cells within the tumor that exhibit genetic and phenotypic features and signaling processes, which escape traditional anti-oncogenic treatments, thereby triggering metastases and relapses of cancers. Critical to cancer biology is the crosstalk between CSCs and tumor microenvironment (TME), implicating a CSC-based cancer immunotherapy. Cognizant of CSCs’ significant role in cancer pathology and treatment, finding a biological model that recapitulates CSCs and TME may allow a better understanding of tumor onset and progression for testing CSC-based therapies. In this review paper, we examined the CSC and TME characteristics of the human embryonal carcinoma NTERA-2 clonal cell line called NTERA-2 cl.D1 or NT2/D1 cells and discussed their potential utility for research and development of treatments for cancer and central nervous system (CNS) disorders. Methods To probe our hypotheses that NT2/D1 cells display CSC and TME properties key to tumor development, which can serve as a screening platform to test cancer and CNS therapeutics, we conducted a literature review over a 10-year period (2014–2024), focusing on PUBMED and Science Direct published articles on cellular models of cancer, with emphasis on milestone research discoveries on NT2/D1 cells relevant to CSCs and TME. We categorized the studies under pre-clinical and clinical investigations in supporting the existence of CSC and TME features in NT2/D1 cells and providing a laboratory-to-clinic translational basis for cancer and CNS therapeutics. Conclusions NT2/D1 cells stand as a feasible biological model that recapitulates the crosstalk of CSCs and TME, which may critically contribute to our understanding of cancer and CNS biology and therapeutics. Designing therapeutics against CSCs' distinct self-renewal and differentiation capacities within the TME opens new avenues for treating cancers and CNS disorders. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3a1e53568bc1fb8336c7da8fdb33286a101a01" target='_blank'>
              Modeling of cancer stem cells and the tumor microenvironment Via NT2/D1 cells to probe pathology and treatment for cancer and beyond
              </a>
            </td>
          <td>
            Mia C Borlongan, Thomas Rodriguez, Napasiri Putthanbut, Hongbin Wang, Jea-Young Lee
          </td>
          <td>2025-04-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Despite remarkable results for many patients, cancer therapies have still a poor response rate. Biomarkers used to select patients for treatment are few and only single-biomarker assays are clinically approved. Such single-biomarker assays are unable to capture the complexity and dynamic nature of the TiME and fail to improve patients’ clinical response rate. Here we aim to enable a comprehensive understanding of the molecular, morphological and functional landscape of tumors using a multi-omics approach combining spatial transcriptomics and proteomics assays. Vizgen’s MERSCOPE® platform allows for high-resolution, spatially resolved analysis of up to 1000 genes, enabling the study of molecular and cellular signatures associated with the TiME while Vizgen’s spatial proteomics ISP assays allow for ultrasensitive and high-throughput detection of tumor and immune cell interactions, functions and spatial distribution. We used these approaches to generate datasets from multiple tumors and evaluated the complementary nature of analytical spatial detection methods to elucidate how local microenvironment influence gene expression and protein activity. Formalin-fixed paraffin-embedded (FFPE) specimens from multiple tumors were cut into 5µm sections for both technologies. For mIF experiments, slides were stained with a cocktail of primary antibodies using Leica Biosystems BOND RX autostainer, amplified simultaneously, and imaged on the Zeiss AxioScan.Z1 using multiple rounds of detection. Images were co-registered and exported for analysis using Ultivue’s STARVUETM image data science platform. Serial sections were profiled with Vizgen’s MERSCOPE Platform and the 500 Gene Immune-Oncology (IO) Panel with MERFISH 2.0 chemistry. The resulting spatial transcriptomic data was segmented using Cellpose2 and analyzed using both UMAP and spatial projections for visualization. Concordance of RNA and protein results were established by evaluating the expression of markers present in both the 500-gene IO panel and the ISP probes. We then evaluated immune responses measured by gene signatures for multiple functions and the corresponding protein markers. We similarly examined changes between the tumor boundary and within the tumor mass. We demonstrate the ability to orthogonally validate findings and expand on the biomarker-driven protein data with pathway-level data from the transcriptomic data. The integration of gene expression profiling enabling screening of a broad range of targets, and mIF assays for target validation, allows for a holistic understanding of spatial and molecular insights of the TiME. These complementary technologies can be used in parallel for biomarker discovery and IO pathways interrogation providing synergistic insights into the complex and dynamic TiME.


 Angela Vasaturo, Cassandra Kysilovsky, Lauren Duro, Sudhir Tattikota, Renchao Chen, Justin He, Manisha Ray, Jiang He He, George Emmanuel. Combining Vizgen’s MERSCOPE spatial transcriptomics profiling and InSituPlex (ISP) multiplex immunofluorescence (mIF) to unravel the complexity of the tumor-immune microenvironment (TiME) [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2075.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f1c5b61915bb42d8ac0674ffd7a657285c0c6e2" target='_blank'>
              Abstract 2075: Combining Vizgen’s MERSCOPE spatial transcriptomics profiling and InSituPlex (ISP) multiplex immunofluorescence (mIF) to unravel the complexity of the tumor-immune microenvironment (TiME)
              </a>
            </td>
          <td>
            Angela Vasaturo, Cassandra Kysilovsky, Lauren Duro, Sudhir Tattikota, Renchao Chen, Justin He, Manisha Ray, Jiang He, George Emmanuel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Tumoroids are patient-derived cancer cells grown as 3D, self-organized multicellular structures. Recently, tumoroids have emerged as powerful and reliable platforms for preclinical cancer drug discovery due to their ability to retain key characteristics from the original tumor. However, traditional tumoroid culture relies on complex media formulations and labor-intensive culture workflows. To tackle this issue, we have developed the serum-free, conditioned medium-free GibcoTM OncoProTM Tumoroid Culture Medium for derivation and expansion of tumoroids. Our previous data demonstrated that OncoPro medium enabled the transition of 18 out of 19 previously established tumoroid models of various cancer indications from the National Cancer Institute Patient-Derived Models Repository (NCI PDMR) and another commercial source and preserved their genetic and transcriptional characteristics. Here, we extend this work demonstrating the compatibility of OncoPro medium with many previously established tumoroid lines and present a stable reporter pool generation custom project for a leading pharmaceutical company using a client-provided colorectal tumoroid line. We first tested the ability for the tumoroids to adapt to OncoPro medium and a suspension culture protocol. Tumoroids displayed higher growth rate when cultured in OncoPro medium compared to client-provided media (CM) in both embedded and suspension culture. To enable scalability, suspension culture was selected for downstream reporter pool generation. Firefly luciferase (fLuc)- and green fluorescent protein (GFP)-expressing tumoroids were generated by lentivirus transduction followed by antibiotic selection in OncoPro medium and CM. 5 days after transduction, >90% of cells were GFP-positive, minimizing the selection bias from editing and preserving population heterogeneity. 6 weeks after selection began, GFP-expressing pools evaluated by flow cytometry showed >97% GFP positive cells, while fLuc-expressing pools showed a cell number-dependent increase of luciferase activity in both media systems. Notably, OncoPro medium displayed a considerable advantage for reporter pool expansion, saving >50% of media and hands-on time compared to CM. To test for genetic and transcriptional stability, stable pools were compared with parental cells grown in CM using targeted NGS and bulk RNA sequencing. High correlation coefficients (r ≥ 0.99 for targeted NGS and ≥ 0.90 for RNA sequencing) were observed in pre- and post-edited pools, suggesting minimal genetic and gene expression drift over the engineering process. Taken together, these findings represent the ability of OncoPro medium to support cell engineering workflows on established tumoroids while preserving characteristics of the parental tumoroids. These tumoroid reporter pools serve as effective and clinically relevant models for evaluating cancer therapies in downstream applications.


 Xiaoyu Yang, Jacob Delgadillo, Garrett Wong, Matthew Woods, Jakhan Nguyen, Colin Paul, Chris Yankaskas, Pradip Shahi Thakuri, Shyanne Salen, Sylvia Beam, Matt Dallas, David Kuninger, Jason Sharp. Generation of stable reporter pools in established patient-derived tumoroid models [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6504.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3647eb8af6a90e48e8ea3e05fe8efdc14aa0cad" target='_blank'>
              Abstract 6504: Generation of stable reporter pools in established patient-derived tumoroid models
              </a>
            </td>
          <td>
            X. Yang, Jacob Delgadillo, Garrett Wong, Matthew Woods, Jakhan Nguyen, Colin D. Paul, Chris Yankaskas, Pradip S. Thakuri, Shyanne Salen, Sylvia Beam, M. Dallas, David Kuninger, J. Sharp
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma remains a highly malignant, inevitably recurring brain tumor with dismal prognosis. Tumor progression is influenced by cellular interactions between the tumor and its microenvironment, yet quantitative data on these interactions, particularly under standard of care therapy, remains sparse. We utilized spatial, single-cell, multiome profiling to map therapy-induced changes in the architecture of >700 paired glioblastoma samples derived from 101 patients at diagnosis and recurrence. This approach categorized patients into groups reflecting specific evolutionary responses to therapy, with implications for prognosis and treatment. Favourable outcomes were linked to shifts towards enriched perivascular, oligodendrocyte-progenitor-like niches with deep tissue infiltration of activated immune cells, whereas poor outcomes correlated with shifts towards an immunosuppressive mesenchymal environment with poor immune infiltration. Finally, while mutations in EGFR/PTEN, or MGMT-promoter methylation influenced tumor evolution, we also identified a defined subset of patients that were more responsive to lomustine after early recurrence. Our findings offer a framework to stratify patient treatment based on tumor and microenvironmental evolution.


 Maxime Vanmechelen, Pouya Nazari, Jan Beckervordersandforth, Daphne Leunissen, Basiel Cole, Brecht Decraene, Yanti De Visser, Gautam Shankar, Maikel Verduin, Daniele Pantano, Carmen Bravo Gonzalez-Blas, Chiara Caprioli, Sien Bevers, Tom Moors, Jayesh Telang, Ivey Sebasstian, Julie Messiaen, Yannick Van Herck, Emma Geens, Danielle Eekers, Annelies Claeys, Marleen Derweduwe, Axel Zur Hausen, Francesca Bosisio, Frank Weyns, Thomas Daenekindt, Peter Van Eyken, Mieke Govers, Ferdinand Mennens, Koos Hovinga, Steven De Vleeschouwer, Paul Clement, Martijn Broen, Marc Vooijs, Raf Sciot, Asier Antoranz, Jon Pey, Ernst-Jan Speel, Ann Hoeben, Frederik De Smet. Characterization of therapeutic effects in glioblastoma through integrated spatial single-cell multiome profiling of longitudinal samples: Unveiling clinically relevant subtypes of pathological changes [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6362.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d4da2d95c6bade92c76a9deae19dad9fbd73c0" target='_blank'>
              Abstract 6362: Characterization of therapeutic effects in glioblastoma through integrated spatial single-cell multiome profiling of longitudinal samples: Unveiling clinically relevant subtypes of pathological changes
              </a>
            </td>
          <td>
            M. Vanmechelen, Pouya Nazari, J. Beckervordersandforth, D. Leunissen, B. Cole, B. Decraene, Y. De Visser, Gautam Shankar, M. Verduin, Daniele Pantano, C. B. González-Blas, Chiara Caprioli, Sien Bevers, Tom Moors, Jayesh Telang, Ivey Sebasstian, J. Messiaen, Y. Van Herck, Emma Geens, D. Eekers, A. Claeys, M. Derweduwe, A. Z. Hausen, Francesca M Bosisio, Frank Weyns, T. Daenekindt, Peter Van Eyken, Mieke Govers, Ferdinand Mennens, Koos E. Hovinga, S. De Vleeschouwer, Paul Clement, M. Broen, Marc Vooijs, R. Sciot, A. Antoranz, Jon Pey, Ernst-Jan Speel, A. Hoeben, F. De Smet
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a prevalent and lethal disease with limited treatment options. Tumor-associated macrophages (TAMs), which comprise up to 30% of the GBM mass, play a pivotal role in shaping the TME by supporting tumor growth and establishing an immunosuppressive and inflammatory environment. To investigate how tumor cells affect TAMs during TME evolution, we analyzed single-cell transcriptomics (SCT) datasets from GBM patient-derived tumor mass (Core) and peri-tumoral adjacent tissues (Margin), alongside murine datasets from healthy and GL261-bearing brains. This comparison aimed to clarify the immunological landscapes of GBM and identify specific interactions between tumor cells and TAMs. Margin SCT data revealed increased cellular heterogeneity compared to the Core, with immune cells, tumor cells, and brain stromal cells contributing 57%, 3%, and 40% of the total cell population, respectively. UCell analysis of GBM subtype signatures demonstrated a strong association between TAMs in the Margin and mesenchymal signatures, which are linked to poor prognosis. Further reclustering of TAMs in Margin and the comparison with murine SCT datasets showed that these cells were predominantly derived from peripheral monocytes and macrophages, with M1-like TAMs representing the majority. Pseudotime trajectory analysis revealed that M1-like TAMs serve as precursors to protumoral M2 TAMs. Cell-cell communication mediated by ligand-receptor interactions among tumor cells, TAMs, T cells, and NK cells identified a critical interaction between tumor-derived ribosomal protein S19 (RPS19) and C5AR1 on TAMs. C5AR1 was highly expressed in M1-like TAMs across patient and murine datasets, while RPS19 was ubiquitously expressed by all cell types. In vitro models confirmed that M1 macrophages and microglia exhibited higher C5AR1 expression than their M2 counterparts. Analysis of public datasets (The Cancer Genome Atlas-GBM and Chinese Glioma Genome Atlas) showed elevated C5AR1 expression in GBM tissues, particularly in recurrent tumors, and a strong correlation between high C5AR1 expression and poor patient survival. Additionally, tumor-derived RPS19 was released extracellularly under growth stress or therapeutic interventions, promoting an inflammatory TME through M1 TAM polarization and driving GBM progression. These findings suggest that targeting the RPS19-C5AR1 axis, in combination with current therapies, may represent a promising strategy to improve GBM outcomes.


 Chan-Chuan Liu, Gangga Anuraga, Cheng-Lin Wu, Jian-Ying Chuang, Chih-Yang Wang, Pin-Yuan Chen, Kwang-Yu Chang. RPS19-C5AR1 axis in tumor-associated macrophages drives glioblastoma progression: Insights from single-cell transcriptomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5216.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfe55329ddb31e27a6a2e90cca8f16e695b75bc6" target='_blank'>
              Abstract 5216: RPS19-C5AR1 axis in tumor-associated macrophages drives glioblastoma progression: Insights from single-cell transcriptomics
              </a>
            </td>
          <td>
            Chan-Chuan Liu, G. Anuraga, Cheng-Lin Wu, J. Chuang, Chih-Yang Wang, Pin-Yuan Chen, K. Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Cancer progression remains a clinical challenge due to tumor heterogeneity. Effective T cell infiltration is critical for mounting an anti-tumor response. Tumors are classified as “hot” or “cold” based on the extent of T cell infiltration. In many cancers, particularly breast cancer, extracellular matrix (ECM) geometry evolves with tumor progression, characterized by Tumor-Associated Collagen Signatures (TACS). Specifically, empirically observed ECM topologies are random fibers (TACS1), circumferentially aligned fibers (TACS2), and radially arranged fibers (TACS3). Clinically, patients with TACS3+ breast cancer exhibit poorer survival compared to TACS3- cases, but the underlying mechanisms and the role of evolving ECM geometry in tumor evolution and tumor-T cell interaction remain unclear. The lack of biomarkers or methods to track TACS evolution underscores the need for physical models linking ECM geometry to tumor-immune co-evolution.



 To address this, we developed the EVO-ACT (EVOlutionary Agent-based Cancer T cell interaction) model, built upon the Gillespie algorithm.



 Our analysis of the three TACS geometries revealed distinct T cell migration patterns and efficiencies: TACS3 demonstrated the highest efficiency, TACS1 intermediate, and TACS2 the lowest. While TACS geometry significantly influences T cell infiltration, it does not act as an absolute barrier; stronger chemokine gradients enhance T cell infiltration across all TACS types. Notably, TACS3 facilitates immune recognition and correlates with higher survival in our simulations, which contrasts with the clinically observed lower survival of TACS3+ breast cancer patients. As TACS3 is typically linked to late-stage breast tumors with phenotypic adaptations, we incorporated mechanisms such as EMT into our model to address this discrepancy. The progression from TACS2 to TACS3, coupled with EMT, more accurately reproduced clinical outcomes. Considering the mesenchymal phenotype’s ability to upregulate immune checkpoint molecules such as PD-1/PD-L1, we further explored TACS-specific tumor evolution and its impact on responses to immune checkpoint inhibitors and patient survival. Our findings reveal lower survival in TACS3/EMT+/PD-L1+ cases compared to TACS2/EMT-/PD-L1+ cases, highlighting that TACS3-associated poor survival arises not solely from ECM geometry but from late-stage cancer adaptations such as EMT and upregulated immune checkpoint expression.



 This study underscores the complex interplay between evolving ECM geometry and tumor-T cell interaction dynamics, offering a new explanation for TACS-associated survival differences. It also provides novel insights into TACS-specific responses to PD-1/PD-L1 inhibitors and their implications for patient survival.



 Yijia Fan, Jason Tomas George. The evolving ECM architecture shapes unique evolutionary trajectories of breast tumor immune escape and elimination [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7495.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d95da358ae944228ec7e4ac72604761c1f9aaad9" target='_blank'>
              Abstract 7495: The evolving ECM architecture shapes unique evolutionary trajectories of breast tumor immune escape and elimination
              </a>
            </td>
          <td>
            Yijia Fan, J. T. George
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3403a48f5780117716d0b711f94672490029280d" target='_blank'>
              Universal Dual-Target CAR-γδT Cells Targeting B7-H3 and IL13Rα2 for Glioblastoma Therapy
              </a>
            </td>
          <td>
            Yang Xu, Yunhao Ye, Yi Wang, Guangna Liu
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Carcinosarcoma (CS) is a rare aggressive biphasic tumor characterized by the presence of both carcinomatous (epithelial) and sarcomatous (mesenchymal) components. It can occur in various organs, including uterus and ovaries, and often exhibits rapid progression. The biphasic nature of CS makes it challenging to treat. In this study, we characterized a patient-derived xenograft (PDX) model developed from a 27 year-old female with CS of unknown primary origin in the NCI Patient-Derived Models Repository (https://pdmr.cancer.gov) to understand the transcriptomic features of these different components.
 PDX samples of the tumor were iteratively passaged in separate NSG mice, and samples from different lineages across passages P0-P2 were collected for analysis. Twenty-five separate carcinoma/sarcoma regions from 7 samples through laser capture microdissection were obtained for bulk RNA-seq. Fourteen samples were sequenced through single-cell RNA-seq using 10x Genomics Single Cell Chromium, including 2 samples in P0, 8 in P1 and 4 in P2. Mouse reads were excluded using XenoCell. Human-only reads were processed through standard QC filtering, integration and annotation with Cell Ranger, Seurat and other R packages.
 After quality-based filtering, we obtained 24,648 single cells from the integrated tumor samples. Through unsupervised clustering, cells separated into 16 Seurat-identified clusters, with consistent percentages of carcinoma and sarcoma components in P0, P1 and P2. Based on gene expression signature scores, each cluster was given a label descriptive of the cluster status. Five clusters had more epithelial/carcinoma characteristics (including one with a distinct ciliated signature), 2 clusters displayed a clear mesenchymal/sarcoma phenotype, 4 had low reads or gene counts, 2 had high cell cycle signature score and 3 clusters did not have a distinct carcinoma or sarcoma status.
 We identified differentially expressed genes in each annotated cluster relative to the other clusters and performed gene set enrichment analysis. Individual carcinoma clusters demonstrated upregulation of pathways related to apoptosis, hypoxia and inflammation (p-value < 0.05), while sarcoma clusters demonstrated upregulation of WNT signaling, EMT and angiogenesis (p-value < 0.05). Clusters with unclear carcinoma/sarcoma phenotype showed patterns of expression similar to carcinoma clusters, with the additional activation of oxidative phosphorylation and DNA repair pathways.
 Single cell data for a CS PDX model show that the biphasic nature of this histology is stable through passaging. Using single cell data, we identified the presence of clusters that are not fully committed to either the carcinoma or sarcoma phenotype. These clusters could represent transitional cell populations which may provide insight into biological drivers of cell plasticity in this tumor histology.


 Alida Palmisano, Yuri Kotliarov, Parimal Kumar, Ting-Chia Chang, Li Chen, Brandie Fullmer, Alyssa Chapman, Amanda Peach, Biswajit Das, Tomas Vilimas, Thomas Forbes, Shahanawaz Jiwani, Melinda G. Hollingshead, Yvonne A. Evrard, Sarah Shin, Hari Sankaran, Julia Krushkal, Alice Chen, Yingdong Zhao, Chris A. Karlovich, Mickey Williams, Lisa M. McShane, James H. Doroshow. Single-cell characterization of a carcinosarcoma patient-derived xenograft model [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2fdb51de20c5dbd0fcd0301755454ca3bb28a10" target='_blank'>
              Abstract 5078: Single-cell characterization of a carcinosarcoma patient-derived xenograft model
              </a>
            </td>
          <td>
            Alida Palmisano, Yuri Kotliarov, Parimal Kumar, Ting-Chia Chang, Li Chen, B. Fullmer, Alyssa Chapman, A. Peach, Biswajit Das, T. Vilimas, Thomas D. Forbes, S. Jiwani, M. Hollingshead, Yvonne A. Evrard, Sarah J Shin, Hari Sankaran, Julia Krushkal, Alice P. Chen, Yingdong Zhao, C. Karlovich, Mickey Williams, Lisa M. McShane, James H. Doroshow
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Spatial biology has advanced our understanding of cancer biology and shed light on the immense transcriptomic and proteomic heterogeneity that exists within the tumor-immune microenvironment (TiME). To date, most analytical and diagnostic approaches only provide one type of biological readout per tissue section, and the generation of true multi-omic data has remained challenging. We developed a workflow that allows the integration of multiple omic modalities on a single section. The workflow is centered around a new signal removal strategy, EpicIF™ technology, which effectively and rapidly removes fluorophores while leaving tissue and probe chemistries unaffected. Using this workflow on the CellScape™ platform, we iteratively performed spatial proteomic, transcriptomic, and interactomic assays all on the same tissue sections. Formalin-fixed, paraffin-embedded tissue sections were subjected to standard histological processing and then incubated with primary antibodies against PD1 and PD-L1, which were then treated with oligonucleotide-modified secondary probes designed for an in situ Proximity Ligation Assay (isPLA) to detect PD1 and PD-L1 interactions. Next, the detection of RNA targets was accomplished by adding 12 HCR™ HiFi RNA-ISH probes, which were visualized by addition of target-specific fluorescently labeled HCR™ Gold amplifiers (RNA-FISH). Lastly, spatial proteomic labeling was conducted with VistaPlex™ multiplex immunofluorescence (mIF) panels targeting 30+ biomarkers. We deployed this tri-omic assay on samples of primary CNS lymphoma (PCNSL), a rare and aggressive form of B-cell lymphoma that remains an area of unmet clinical need, with over half of diagnosed patients succumbing to the disease. Unlike diffuse large b-cell lymphoma (DLBCL), its morphologically identical systemic counterpart, PCNSL shows sensitivity to PD1 checkpoint inhibition. We investigated if our assay could describe unique interactions between the PD1-PDL1 axis and the lymphoma TiME in PCNSL (n=24), in comparison to DLBCL (n=152) cases. High-resolution imaging with the CellScape platform allowed for precise detection and quantification of isPLA, RNA-FISH and mIF signals on the same slide. The combined assay showed differential biomarker expression and interaction profiles associated with immune-regulatory processes in the interrogated tumor samples, revealing remarkable inter- and intra-tumoral heterogeneity. This spatial multi-omics approach allows for a more comprehensive insight into the complex interplay of immune and non-immune cell populations in the TiME.


 Arne Christians, Jannik Boog, Daniel Jimenez Sanchez, Sara Bodbin, Agata Wicher, Subham Basu, Aneesh Acharya, Randy Chen, Hong Liang, Shruti Shridar, Tan Char Loo, Ng Siok Bian, Michelle Poon, Anand Jeyasekharan, Oliver Braubach. Development of a novel tri-omic spatial biology assay that provides in-depth immune profiling of the tumor immune microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2084.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f07f0f25d717cf3d7b38eb63170b02349e56a68b" target='_blank'>
              Abstract 2084: Development of a novel tri-omic spatial biology assay that provides in-depth immune profiling of the tumor immune microenvironment
              </a>
            </td>
          <td>
            A. Christians, J. Boog, Daniel Jimenez Sanchez, Sara Bodbin, A. Wicher, Subham Basu, Aneesh Acharya, Randy Chen, Hong Liang, Shruti Shridar, Tan Char Loo, Ng Siok Bian, Michelle Poon, Anand Jeyasekharan, O. Braubach
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Tumor heterogeneity remains an obstacle in the treatment of melanoma, with unique tumor microenvironments contributing to therapy resistance and differential tumor growth. Dedifferentiated melanoma states have been associated with resistance to targeted and immunotherapy. AXL and NGFR are dedifferentiation markers linked to these states; however, the mechanisms by which they interact or result in distinct tumor programs remain unclear. Both AXL and NGFR have been linked to therapy resistance. AXL has been associated with cell survival, proliferation and immune suppression and NGFR with tumor progression, invasion, metastasis and therapy resistance. Understanding these interactions is critical for identifying therapeutic strategies and improving patient outcomes. In a melanoma patient with longitudinal tumor samples, we used highly multiplexed cyclic immunofluorescence (CyCIF) and matched H&E to study the tumor and immune states associated with progression. Spatial transcriptomic profiling was performed using GeoMX on selected regions of interest to further study tumor states characterized by different morphology and proteomic differentiation markers. In the primary and in-transit metastasis, we observed intratumoral heterogeneity, with histologically distinct tumor nodules. CyCIF of the in-transit sample showed upregulation of AXL in one nodule and NGFR in the other. The NGFR nodule was characterized by densely packed tumor cells, while the AXL nodule featured an inner cleft region encased by dense peripheral tumor cells. In the NGFR nodule, differential expression analysis (DEA) revealed upregulation of EPAS1, HIF1A, ERBB3 and VEGFA suggesting hypoxic stress, cell signaling, active angiogenesis and cell migration. Transcription factor analysis (TFA) showed upregulation of HDGF, TGFB1I1, HOXA7, IRF6 and SRSF2, TFs associated with proliferation, differentiation and increased translational activity. Downregulated HIP2K and PAWR suggest apoptosis inhibition, further facilitating tumor survival and expansion. In the AXL nodule, DEA revealed increased CDKN1A, indicating stress response and altered cell cycle regulation. Gene set enrichment analysis (GSEA) highlighted WNT and Hedgehog signaling, hormonal responses (e.g., estrogen signaling), and metabolic alterations (bile acid metabolism, myogenesis). EMT upregulation suggested a mesenchymal-like phenotype associated with resistance. TFA showed elevated KLF8 and TGFB1I1 driving EMT and metastasis, while SRSF2, HSF4, CXXC1, and NR1H2 suggested stress resilience, apoptosis regulation, and ER stress mitigation. Distinct signaling in AXL and NGFR regions suggests NGFR drives metabolic activity and proliferation, while AXL promotes resilience via stress and hormonal signaling. These findings offer insights into therapies targeting melanoma heterogeneity and resistance.


 Mark Woodnorth, Giuseppe Tarantino, Anne Zaremba, Tuulia Vallius, Florian Rambow, Lisa Zimmer, Antje Sucker, Elisabeth Livingstone, Eva Hadaschik, Roxanne Pelletier, Yingxiao Shi, Mariana Lopez, Samira Makhzami, Christine Lian, George Murphy, Genevieve Boland, Peter Sorger, Dirk Schadendorf, David Liu. Tumor heterogeneity and differential pathway activation in melanoma: insights from spatial profiling of AXL and NGFR upregulated regions [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 169.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86fb33430b6b85815a3c78b337e0ccc5a2f35f2c" target='_blank'>
              Abstract 169: Tumor heterogeneity and differential pathway activation in melanoma: insights from spatial profiling of AXL and NGFR upregulated regions
              </a>
            </td>
          <td>
            M. Woodnorth, Giuseppe Tarantino, Anne Zaremba, Tuulia Vallius, F. Rambow, Lisa Zimmer, A. Sucker, E. Livingstone, E. Hadaschik, R. Pelletier, Yingxiao Shi, Mariana Lopez, S. Makhzami, Christine Lian, George Murphy, Genevieve M Boland, P. Sorger, Dirk Schadendorf, David Liu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are the most abundant non-cancerous cell type in glioblastoma (GBM) and heavily influence GBM biology, contributing to tumor progression, therapeutic resistance, immune evasion, and neovascularization. Current in vitro models that utilize IL-4/IL-13 stimulation fail to capture the transcriptional and functional heterogeneity of TAMs observed in vivo. In this study, we utilize a serum-free indirect co-culture model with patient-derived xenolines to polarize primary human macrophages and characterize their molecular and functional phenotypes. We demonstrate that xenoline-polarized macrophages diverge from classical M1/M2 states and instead adopt transcriptional signatures reflective of TAM subsets identified from patients. Notably, macrophages polarized with the radiation-therapy selected xenoline, JX14P-RT, exhibited gene expression patterns enriched for interferon response and hypoxia, mirroring recurrent GBM samples. In contrast, JX14P TAMs showed enrichment in phagocytic gene sets. Functional validation of these phenotypes revealed discrepancies between the transcriptionally predicted and observed phenotypes, emphasizing the importance of integrating phenotypic validation in sequencing studies. Altogether, our findings establish xenoline-polarized macrophages as a more physiologically relevant alternative to traditional models, offering a useful model for studying tumor-immune interaction in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e59e91c3e7e604dd7d80dd0333a4e13f653a5b4e" target='_blank'>
              Xenoline-polarized macrophages as a physiologically relevant in vitro model of tumor-associated macrophages in glioblastoma
              </a>
            </td>
          <td>
            Hasan Alrefai, Benjamin Lin, Amr Elkohly, Manoj Kumar, Taylor L. Schanel, Kevin J. Lee, Patricia H Hicks, Joshua C Anderson, Gao Guo, Eun-Young Erin Ahn, C. R. Miller, Christopher Willey
          </td>
          <td>2025-05-09</td>
          <td>Research Square</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Single cell transcriptomics (scRNAseq) remains the method of choice to quantify and annotate cellular heterogeneity in the tumor and its microenvironment (TME), yet most studies are under-powered to associate changes in tumor heterogeneity with phenotypes such as tumor subtype, grade, and patient age, among others. In this study, we created an integrated atlas of human breast cancer (BC), the largest resource of its kind, totaling more than 700, 000 cells across 145 patients, and optimized computational methods to benchmark integration performance, and robustly perform hierarchical cell type annotation. By combining single profiles, a higher-resolution annotation of immune, stromal, and epithelial cell types was achieved. Further, using this integrated atlas, differential abundance analyses identified significant changes in immune and stromal cell type composition associated with tumor grade, subtype, and inferred metabolic comorbidities. For insulin resistance and obesity, we identified significant expansion in exhausted CD4 T Cell populations and interferon signalling in CD8 and macrophage populations. These changes in TME heterogeneity were not discernible or had effects in discordant directions when the analysis was limited to individual BC studies, highlighting the need for atlas-based mega-analysis approaches. This highly-resolved integrated scRNAseq BC atlas represents a valuable resource for hypothesis-driven analyses of tumor heterogeneity.


 Andrew D. Chen, Christina Ennis, Lina Kroehling, Gerald Denis, Stefano Monti. Integrated single cell atlas of human breast cancer enables highly-resolved analysis of tumor heterogeneity and metabolic comorbidities [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1073.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b642d8d5ad00a97352332e89bf3390c538a71e8e" target='_blank'>
              Abstract 1073: Integrated single cell atlas of human breast cancer enables highly-resolved analysis of tumor heterogeneity and metabolic comorbidities
              </a>
            </td>
          <td>
            Andrew Chen, Christina S. Ennis, Lina Kroehling, Gerald V. Denis, Stefano Monti
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Hepatocellular carcinoma is a highly aggressive and heterogeneous malignancy with limited understanding of its heterogeneity.In this study, we applied ten multi-omics classification algorithms to identify three distinct molecular subtypes of HCC (C1–C3). To further explore the immune microenvironment of these molecular subtypes, we leveraged single-cell transcriptomic data and employed CIBERSORTx to deconvolute their immune landscape.Among them, C3 exhibited the worst prognosis, whereas C1 and C2 were associated with relatively better clinical outcomes. Patients in the C3 group exhibited a high burden of copy number variations, mutation load, and methylation silencing. Our results revealed that compared to C1 and C2, C3 had a lower proportion of hepatocytes but a higher proportion of cholangiocytes and macrophages. Through analyses of hepatocyte, cholangiocyte, and macrophage subpopulations, we characterized their functional states, spatial distribution preferences, evolutionary relationships, and transcriptional regulatory networks, ultimately identifying cell subpopulations significantly associated with patient survival. Furthermore, we identified key ligand-receptor interactions, such as APOA1-TREM2 and APOA2-TREM2 in hepatocyte-macrophage crosstalk, and VTN-PLAUR in cholangiocyte-macrophage communication.Finally, we employed machine learning methods to construct a prognostic model for HCC patients and identified novel potential compounds for high risk patients. In summary, our novel multi-omics classification of HCC provides valuable insights into tumor heterogeneity and prognosis, offering potential clinical applications for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64917cbcdeb277a734016cb34d2bfe7542a8101f" target='_blank'>
              Multi-omics and single-cell approaches reveal molecular subtypes and key cell interactions in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Xueqing Zou, Yongmei Wang, Mingyuan Luan, Yizheng Zhang
          </td>
          <td>2025-05-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Drug resistance in cancer therapy remains a significant challenge, driven by the ability of cancer cells to evade treatment through complex epigenetic adaptations. Identifying distinct molecular and structural signatures in drug-resistant cells is crucial for uncovering predictive biomarkers and understanding the mechanisms underlying resistance. In this study, we combine highly multiplexed fluorescence imaging and label-free quantitative phase imaging technologies with genomic analyses to investigate the morphological and molecular alterations associated with drug resistance. Using the colon cancer cell line SW480, we developed a drug-resistant phenotype by gradually increasing concentration of 5-fluorouracil (5-FU). Our approach includes morphological analysis, profiling 26 proteins and chromatin modifications, and transcriptomic evaluation. Genomic analyses—including CUT&RUN, RNA-seq, and ATAC-seq—further characterize histone modifications, gene expression, and chromatin accessibility in drug-resistant and parental cells. Our analysis reveals distinct morphological features in drug-resistant (DR) cells compared to their parental counterparts, such as increased morphological plasticity, an elongated shape and pronounced protrusions characteristic of cells undergoing epithelial-to-mesenchymal transition (EMT). A deep neural network analysis of these morphological features effectively distinguishes drug-resistant cells from parental cells, achieving a high area under the receiver operating characteristic curve (AUC) of 0.95. Biomarker expression analysis identifies significant changes in protein expression in drug-resistant cells, notably H3K27me3, H3K27ac, CD44, and EZH2, indicating substantial epigenetic reprogramming and increased stemness. Gene ontology analysis highlights the upregulation of Wnt and EMT pathways, alongside the downregulation of cell adhesion and tight junction pathways, reflecting enhanced invasiveness in drug-resistant cells. Additionally, ATAC-seq analysis reveals a more compact chromatin landscape in drug-resistant cells, consistent with epigenetic modifications that drive transcriptional repression of specific pathways. In conclusion, our multiscale high-content imaging platform proves to be a powerful tool for uncovering the morphological and molecular characteristics of drug-resistant cancer cells. We demonstrate that drug-resistant cells adopt mesenchymal-like morphology, coupled with significant epigenetic reprogramming, chromatin compaction, and enhanced stemness. These results underscore the interplay between morphological adaptations and epigenetic alterations in the development of drug resistance. Our study provides critical insights into the mechanisms driving resistance and lays the groundwork for identifying novel biomarkers and therapeutic targets.


 Yaxin Yang, Jianquan Xu, Chaojie Zhang, Hongqiang Ma, Sarah J. Hainer, Yang Liu. Multiscale high-content imaging to identify molecular and morphological signatures in drug-resistant cancer cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6356.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eed8fdb3c30cedec701229d2292e4e2a7ab7ba3" target='_blank'>
              Abstract 6356: Multiscale high-content imaging to identify molecular and morphological signatures in drug-resistant cancer cells
              </a>
            </td>
          <td>
            Yaxin Yang, Jianquan Xu, Chaojie Zhang, Hongqiang Ma, Sarah J. Hainer, Yang Liu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Breast cancer has become the most commonly diagnosed malignancy, with metastasis remaining the leading cause of mortality, particularly in triple-negative breast cancer (TNBC). In this study, we applied spatial transcriptomics with an 8 μm resolution to analyze the primary tumor and lung metastasis, occurring six months post-recurrence, of a TNBC patient, encompassing a total of 1, 033, 074 bins. Through this analysis, six tumor clusters (T0-T5) with distinct gene expression profiles were identified. Among these, T0 and T3 were the predominant clusters observed in the metastatic tumor. T0 was primarily distributed in the tumor core, while T3 localized at the tumor boundary. Both clusters exhibited high expression of VIM, a marker associated with epithelial-mesenchymal transition (EMT). Differentially expressed gene (DEG) analysis revealed that T3 displayed elevated levels of genes associated with extracellular matrix (ECM) remodeling pathways. Trajectory analysis indicated that T3 represented the most advanced differentiation stage as a subclone. Our findings provide insights into the dynamic spatial heterogeneity and metastatic progression of primary and metastatic TNBC tumor subtypes.


 Yu-Ting Kang, Tzu-Hung Hsiao. Spatial transcriptomics reveals clone-specific features of primary and metastatic tumors in triple-negative breast cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5313.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8492f82ea4a9bedb9da34190b6e586980dd59c" target='_blank'>
              Abstract 5313: Spatial transcriptomics reveals clone-specific features of primary and metastatic tumors in triple-negative breast cancer
              </a>
            </td>
          <td>
            Yu-Ting Kang, Tzu-Hung Hsiao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 The cancer cell cycle possesses an incredible capacity to adapt or change in response to oncogenic mutations or drug treatment (cell cycle “plasticity”) to generate a diversity of solutions to the cell cycle (distinct cell cycle “programs”) that differ in key regulatory effectors and mechanisms. To study the extent and role of cell cycle plasticity in tumor progression and drug resistance, we developed cell cycle mapping - a single-cell proteomic imaging platform that combines highly multiplexed immunofluorescence with manifold learning to visualize cell cycle plasticity among and within tumors. Applying cell cycle mapping to a large library of hormone receptor (HR)+ breast tumors, we observed 4 distinct cell cycle programs driving tumor growth. These programs differed in the expression/activity of key cyclins and CDKs. In many cases, we observed multiple cell cycle programs within a single tumor correlating with differences in the cellular composition of the local tumor environment. These results also highlight the challenge of targeting the cancer cell cycle (e.g. with CDK4/6 inhibitors), given the heterogeneity of cell cycle programs actin within a single tumor. To further explore the role of cell cycle plasticity in targeted therapy, we used a combination of cell cycle mapping and time-lapse imaging to reveal how the breast cancer cell cycle responds to a panel of FDA-approved drugs targeting a range of cell cycle effectors, including multiple CDK inhibitors. We observed clear heterogeneity in the mechanisms of cell cycle adaptation executed across a panel of cell lines modeling distinct molecular (HR+, HER2+, basal) and histological subtypes (invasive ductal carcinoma, invasive lobular carcinoma) of breast cancer. Altered expression/activity of key cell cycle regulators provides a new “path” through the cell cycle, a molecular detour that allows cells to continue proliferating in the presence of drug to drive therapeutic resistance.


 Yian Yang, Thu Pham, Janet McLaughlin, Steffi Oesterreich, Adrian Lee, Katarzyna Kedziora, Wayne Stallaert. Cell cycle plasticity drives tumor progression and drug resistance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY22-02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05724e131a84cae0882ab8214c7527b7f572cfbc" target='_blank'>
              Abstract SY22-02: Cell cycle plasticity drives tumor progression and drug resistance
              </a>
            </td>
          <td>
            Yian Yang, Thu Pham, Janet McLaughlin, S. Oesterreich, Adrian Lee, Katarzyna Kedziora, Wayne Stallaert
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Immune checkpoint blockade (ICB) has transformed cancer therapy and is now approved in every type of cancer, yet it remains difficult to identify the ∼ 20% of patients that will benefit from such therapy. Explainable models that predict ICB response can potentially spare ‘responders’ from toxic chemotherapies while identifying alternative options for ‘non-responders’. The only widely approved ICB biomarker of response, PD-L1, treats TME cells as individual components and is only weakly predictive of outcomes. Recent studies have suggested that biomarkers built on cell-to-cell spatial proximity, such as tumor cell to CD8+ T cell distance, can improve upon PD-L1 in predicting ICB efficacy. Yet, there is a lack of generalized approaches for identifying tumor spatial biology relationships relevant to ICB response, especially considering the wealth of information generated by high-resolution spatial biology techniques. Our approach was (1) to integrate spatial transcriptomics datasets from 96 tumors across twelve tumor types and (2) to apply statistical covariation approaches to identify organizing principles of spatial biology that would facilitate the comparison of these datasets. The basis of this approach is our identification of a conserved unit of pan-tumor spatial organization that we termed ‘Spatial Groups’. Spatial Groups (SGs) are multi-cellular units demonstrating coordinated transcriptional programs contained within spatial domains ranging in size from tens of microns (dozens of cells) to mm-scale (large regions of a tumor). SGs are hierarchically organized such that large-scale SGs exert contextual field effects on their nested small-scale SGs. For example, we found that T cell presence in large-scale SGs impacted metabolic pathway shifts occurring within nested medium-scale SGs, which in turn impacted cell adhesion programs within nested small-scale SGs. We found that SGs revealed immune-related spatial biology patterns that we hypothesized would serve as a genome-wide spatial biomarker of ICB response. To test this idea, we performed a retrospective study of 16 patients with metastatic non-small cell lung cancer treated with ICB or chemotherapy-ICB in the frontline setting. Unlike PD-L1 status, our SG-derived biomarker was highly significant in predicting progression-free survival after therapy initiation. Additionally, analysis of SGs in non-responder patients provided insights into patient-specific spatial biology architectures limiting ICB efficacy. In summary, our discovery of Spatial Groups provides a general framework for using spatial biology to identify pan-tumor ICB response.


 Vivek Behera, Hannah Giba, Ue-Yu Pen, Anna Di Lello, Benjamin A. Doran, Alessandra Esposito, Apameh Pezeshk, Christine Bestvina, Justin P. Kline, Marina C. Garassino, Arjun S. Raman. Spatial Groups provide a pan-tumor spatial transcriptomic biomarker of immunotherapy response [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7480.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fd0e67d45bf4c5f69a873c6cb3c79e9be60dadd" target='_blank'>
              Abstract 7480: Spatial Groups provide a pan-tumor spatial transcriptomic biomarker of immunotherapy response
              </a>
            </td>
          <td>
            Vivek Behera, Hannah Giba, Ue-yu Pen, A. Di Lello, Benjamin A. Doran, Alessandra Esposito, A. Pezeshk, Christine Bestvina, Justin Kline, M. Garassino, Arjun S. Raman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Spatially resolved transcriptomics (SRT) offers insights into tumor microenvironments (TME), mapping gene and cell activities across tissue regions. While systems biology methods utilize SRT data to characterize tissue and cell neighborhood variations, high costs and limited scalability hinder large-scale applications and clinical correlations. In contrast, histological imaging (HI) provides accessible, high-resolution morphological insights with clinical context. Existing HI-based gene expression imputation methods rely on small datasets and cover a limited gene fraction, leaving much of the transcriptome unexplored. This study develops a robust framework for imputing spatially resolved gene expression profiles from HI data by integrating HI-based imputation with scGPT - a foundation models trained on single-cell RNA-seq (scRNA-seq) data. We hypothesize HI data can impute genes linked to TME, tissue structure, and cell types, while scGPT infers additional genes using gene and cell type-specific relationships.



 The proposed framework merges an advanced HI-based gene imputation algorithm with scGPT to predict spatially resolved full-transcriptome profiles. The HI-based model first imputes 500-1,000 tissue structure- and cell type-dependent genes. We enhanced existing methods by (1) incorporating cross-pixel multi-attentions for spatial adjustments, (2) enabling residual connections to preserve early features, and (3) introducing a composite loss function for spatial coherence, gene-gene interaction preservation, and distribution alignment. Tissue annotations provide contextual information that enhances model predictivity. scGPT refines and expands predictions into whole gene profiles, leveraging its robust architecture to capture complex molecular relationships.



 The framework demonstrates strong performance across unseen specimens, achieving low mean squared error (0.38) and high correlations (0.79) while mitigating batch effects. Spatial heatmaps capture tissue heterogeneity, with tumor, immune, and stromal boundaries aligning well with annotations. Gene Enrichment analysis shows the model's capability to capture immune, stromal, and tumor compartments, though challenges remain for underrepresented genes. Integrating scGPT generates accurate and meaningful whole-genome profiles. This enables implementation of systems biology models for transcriptomics - including pathway, metabolism, stemness and CNV analysis - into HI data analysis.



 This framework bridges histology and transcriptomics by combining advanced imputation algorithms with powerful capabilities of scGPT, enabling scalable, spatially resolved gene expression mapping. Its robust extrapolation transforms understanding of tissue biology and the TME complexity.



 Paveethran Swaminathan, Pengtao Dang, Haiqi Zhu, Zheng An, Xin Lu, Zhi Li, Yue Fang, Min Yang, Yuhui Wei, Tingbo Guo, Xinyu Zhou, Xiao Wang, Jia Wang, Chi Zhang, Sha Cao. Characterizing biological functional changes in cancer tumor microenvironment by integrating histology-based gene imputation with scRNA-seq foundation models [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1353.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b467e1bacc4555eacc88c9ff469f39b84deb0abe" target='_blank'>
              Abstract 1353: Characterizing biological functional changes in cancer tumor microenvironment by integrating histology-based gene imputation with scRNA-seq foundation models
              </a>
            </td>
          <td>
            Paveethran Swaminathan, Pengtao Dang, Haiqi Zhu, Zheng An, Xin Lu, Zhi Li, Yue Fang, Min Yang, Yuhui Wei, Tingbo Guo, Xinyu Zhou, Xiao Wang, Jia Wang, Chi Zhang, Sha Cao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) remains one of the most lethal cancers globally, driven by chronic liver disease and a complex tumor microenvironment (TME). Recent advances in spatial omics, single-cell analyses, and AI-driven digital pathology have shed light on the intricate heterogeneity of HCC, highlighting key roles for immune suppression, angiogenesis, and fibrosis in tumor progression. This review synthesizes current epidemiological trends, noting a shift from viral hepatitis to metabolic syndrome as a primary etiology in Western populations, and elucidates how TME components—such as tumor-associated macrophages, cancer-associated fibroblasts, vascular endothelial cells, and immunosuppressive cytokines—contribute to resistance against conventional therapies. We detail the evolution of immunotherapeutic strategies from monotherapy to combination regimens, including dual immune checkpoint blockade and the integration of antiangiogenic agents. Emerging circulating and tissue-based biomarkers offer promise for enhanced patient stratification and real-time monitoring of treatment responses. Collectively, these innovations herald a paradigm shift toward TME-directed precision oncology in HCC, emphasizing the need for multi-targeted approaches to synergistically modulate interacting cellular constituents and ultimately improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3328a9c38f07dccf03f3c0dd4ddca9fc18bfd0" target='_blank'>
              Bridging Immune Evasion and Vascular Dynamics for Novel Therapeutic Frontiers in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Sulin Wu, Namrata Anand, Zhoubo Guo, Mingyang Li, Marcos Santiago Figueroa, Lauren Jung, Sarah Kelly, J. Franses
          </td>
          <td>2025-05-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Tissue microarrays (TMAs) are widely used for high-throughput research but present challenges due to variations in tissue fixation, morphology, and cellular composition, which can impact staining, RNA/protein preservation, and segmentation accuracy. Analyzing diverse tissue types, especially when comparing normal and tumor samples, requires advanced imaging and computational tools. The CosMx® Spatial Molecular Imager (SMI) addresses these challenges by enabling accurate definition of single cells with multi-tissue AI segmentation models and high-resolution data extraction from complex TMA samples. We employed the CosMx SMI with the CosMx Human Whole Transcriptome (WTX) panel, which targets approximately 19,000 RNA targets, to analyze human FFPE tissue microarrays. Two FFPE TMAs—one with normal and the other with cancerous tissue—were constructed with cores from multiple organs including breast, skin, lung, colon, liver, prostate, and brain. RNA was detected in its native tissue context, and hematoxylin and eosin (H&E) staining provided histological reference for spatial transcriptomics. Uniform Manifold Approximation and Projection (UMAP) and Leiden clustering were used for dimensionality reduction and cell type identification based on marker gene expression. Differential expression (DE) and pathway analyses identified molecular signatures and biological processes across tissue types. CosMx SMI achieved high cell counts per sample and precise segmentation, enabling reliable analysis of diverse tissue types. H&E staining on the same slide enhanced AI algorithm training, improving tissue structure identification and segmentation accuracy. We identified distinct cell types and tissue-specific biomarkers. DE and pathway analysis revealed potential therapeutic targets and biomarkers, such as PSA, PSMA, AR, PTEN, and ERG fusion in prostate cancer, along with key pathways like AR signaling and DNA damage repair. Transcript profiles localized to specific cell types, demonstrating the platform's robustness in spatial transcriptomics. CosMx SMI significantly enhances single-cell transcriptome studies, providing detailed molecular insights across diverse tissue types. This platform is a powerful tool for tissue analysis, biomarker discovery, and spatial genomics, supporting a wide range of molecular and cellular research applications.
 -Bruker Confidential-


 Wei Yang, Isabel Lee, Michael Patrick, Yi Cui, Shanshan He, Martin Shelton, Haiyan Zhai, Michael Rhodes, Joseph Beechem. Whole transcriptome single cell spatial imaging of tumor vs normal tissue microarray [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2080.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/153415bd6baa96a665cbee8cebea02a357bb0395" target='_blank'>
              Abstract 2080: Whole transcriptome single cell spatial imaging of tumor vs normal tissue microarray
              </a>
            </td>
          <td>
            Wei Yang, Isabel Lee, Michael Patrick, Yi Cui, Shanshan He, Martin Shelton, Haiyan Zhai, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary This review highlights the role of tumor-associated macrophages (TAMs) in shaping the immunosuppressive tumor microenvironment (TME) of glioblastoma (GBM), an aggressive brain tumor with limited treatment options. TAMs, which constitute a substantial portion of the TME, exhibit high phenotypic plasticity and dynamically shift between pro-inflammatory and immunosuppressive states. GBM exploits this plasticity to drive tumor growth by inducing TAM polarization toward the immunosuppressive phenotype through mechanisms involving cytokine secretion, immune checkpoint pathways, and metabolic interactions. This polarization contributes to immune evasion, tumor proliferation, angiogenesis, and treatment resistance. Therapeutic strategies targeting TAMs include depletion, reprogramming toward the pro-inflammatory phenotype, and inhibiting pro-tumor signals. Several approaches are under investigation, such as blocking CSF-1R, disrupting CCL2/CCR2 and PI3K pathways, targeting PD-L1 expression, and utilizing nanoparticle-based delivery systems for selective TAM modulation. Further research targeting TAM plasticity and polarization is essential for developing treatments that overcome GBM’s robust immunosuppressive TME and improve patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4f2380ce9e95379acbe519ba8ae82cc97186303" target='_blank'>
              The Basis for Targeting the Tumor Macrophage Compartment in Glioblastoma Immunotherapy
              </a>
            </td>
          <td>
            Thomas Eckert, Chase Walton, Marcus Bell, Coulter Small, Nathan C. Rowland, Charlotte Rivers, A. Zukas, S. Lindhorst, Peter E. Fecci, Ben A. Strickland
          </td>
          <td>2025-05-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, has emerged as a pivotal tumor suppressor altered in a broad range of human malignancies. Its frequent inactivation across diverse cancer types has revealed pleiotropic roles that intersect multiple Hallmarks of Cancer. In this review, we integrate current knowledge on how ARID1A loss influences cellular processes including proliferative signaling, resistance to cell death, genomic instability, metabolic reprogramming, immune evasion, and more. We discuss the context-specific consequences of ARID1A deficiency, its cooperation with other oncogenic events, and its implications for therapeutic vulnerability—particularly in the realm of synthetic lethality and immune modulation. By mapping ARID1A’s functional impact onto the established hallmarks framework, we highlight its centrality in cancer biology and underscore opportunities for biomarker-driven strategies and targeted interventions. Understanding ARID1A’s multifaceted roles offers a compelling lens through which to explore chromatin dysregulation in cancer and guide translational advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017483c65324200e0adc9a76376f32832abb8dbc" target='_blank'>
              ARID1A and Its Impact Across the Hallmarks of Cancer
              </a>
            </td>
          <td>
            Bridger Kearns, Andralyn McKell, Isaac Steveson, Peyton Worley, Braeden Barton, Jordan Bennett, DeLaney Anderson, Jacob Harris, James Christensen, Jared Barrott
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer remains a significant global health burden due to its high morbidity and mortality. Oncogene-targeted therapy and immunotherapy have markedly improved the 5-year survival rate in the patients with advanced or metastatic tumors compared to outcomes in the era of chemotherapy/radiation. Nevertheless, the majority of patients remain incurable. Initial therapies eliminate the bulk of tumor cells, yet residual populations termed drug-tolerant persister cells (DTPs) survive, regenerate tumor and even drive distant metastases. Notably, DTPs frequently render tumor cross-resistance, a detrimental phenomenon observed in the patients with suboptimal responses to subsequent therapies. Analogous to species evolution, DTPs emerge as adaptative products at the cellular level, instigated by integrated intracellular stress responses to therapeutic pressures. These cells exhibit profound heterogeneity and adaptability shaped by the intricate feedforward loops among tumor cells, surrounding microenvironments and host ecology, which vary across tumor types and therapeutic regimens. In this review, we revisit the concept of DTPs, with a focus on their generation process upon targeted therapy or immunotherapy. We dissect the critical phenotypes and molecule mechanisms underlying DTPs to therapy from multiple aspects, including intracellular events, intercellular crosstalk and the distant ecologic pre-metastatic niches. We further spotlight therapeutic strategies to target DTP vulnerabilities, including synthetic lethality approaches, adaptive dosing regimens informed by mathematical modeling, and immune-mediated eradication. Additionally, we highlight synergistic interventions such as lifestyle modifications (e.g., exercise, stress reduction) to suppress pro-tumorigenic inflammation. By integrating mechanistic insights with translational perspectives, this work bridges the gap between DTP biology and clinical strategies, aiming for optimal efficacy and preventing relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a66d9e3af1fc508d0ed83941eccffc587ef2ea" target='_blank'>
              Drug-induced tolerant persisters in tumor: mechanism, vulnerability and perspective implication for clinical treatment
              </a>
            </td>
          <td>
            Shujie Liu, Anfeng Jiang, Faqing Tang, Minghao Duan, Bin Li
          </td>
          <td>2025-05-24</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937e7d64004dc2ecd2a28936483a775c209f1f11" target='_blank'>
              Proteomic Remodeling During Tumor Cell-Induced Platelet Aggregation Unveils Metastatic Drivers in Colorectal Cancer
              </a>
            </td>
          <td>
            Thorben Sauer, Caroline Gruner, Katharina Kern, Antje Rackisch, Lea Tischner, Katharina Schulz, Jasmin Ostermann, Lena Cohrs, Michael Kohl, A. Verschoor, Timo Gemoll
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Chemotherapy remains the most used systemic treatment option for triple-negative breast cancer. Lineage tracing studies demonstrated that the treatment does not significantly alter the clonal composition of primary breast tumors. Nevertheless, the specific transcriptional phenotype(s) conferring chemoresistance at primary tumor (PT) level, as well as at metastatic level, are still largely undetermined, specifically at single-clone resolution.To tackle such unmet clinical issues, we leveraged our recently published in vivo single-cell lineage tracing approach under therapeutic pressure, using two chemotherapeutic agents: adriamycin and cyclophosphamide (A+C). Upon administration of A+C we obtained stable disease, as confirmed by histopathological and IHC analyses. Chemoresistance was formally demonstrated by ex vivo IC50.By using single-cell lineage tracing on treated and untreated matched PT-lung metastases, we found that chemotherapy shifted clonal hierarchy, with few high-fit clones outcompeting the others, at primary level. Despite this, we noticed the striking increase of pro-metastatic clone frequencies in the chemoresistant model, thus suggesting that chemotherapy selects for few highly-aggressive clones. At metastatic level we found general drop of clonality.Using scRNA-seq, we observed that chemotherapy shapes distinct chemoresistant patterns in the PT and metastases. We then explored the transcriptomics features of the most fit clones within treated tumors, so-called dominant: they were characterized by enrichment of distinctive cancer hallmarks with respect of the whole chemoresistant tumor, including tumor-promoting inflammation and sustained angiogenesis.Chemoresistant/pro-metastatic clones were characterized by the expression of genes that were prognostic for poor survival across patients. Importantly, we observed that the chemoresistant/pro-metastatic phenotype was exacerbated by re-challenge with the same treatment but also stable across multiple re-transplantations. Among the of chemoresistant/pro-metastatic genes, PAEP appeared as the top marker. This was corroborated by GRN analyses, where we observed that the network around PAEP was extremely perturbed by therapeutic pressure. We tested the pro-metastatic effect of PAEP expression in a chemoresistant scenario. We orthotopically injected MDA-MB-231 cells silenced for PAEP expression via shRNA into recipient NSG mice and we performed neoadjuvant and adjuvant chemotherapy. Upon PAEP silencing, we observed slight increase of chemosensitivity at primary tumor level. Strikingly, chemoresistant metastases formation was significantly impaired by PAEP downregulation, as compared to controls.Overall, our data demonstrate that the pro-metastatic phenotype of chemoresistant breast cancer is stable, transcriptionally defined and driven by the expression of PAEP.


 Alberto Dalmasso, Andrea Cossa, Ilaria Servidio, Guido Campani, Elisa Bugani, Francesco Antonio Tucci, Lucrezia Stabellini, Salvatore Pece, Pier Giuseppe Pelicci. The pro-metastatic phenotype of chemoresistant breast cancer is stable and transcriptionally defined [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1243.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7119dba646488771d805798083884f45a999e" target='_blank'>
              Abstract 1243: The pro-metastatic phenotype of chemoresistant breast cancer is stable and transcriptionally defined
              </a>
            </td>
          <td>
            Alberto Dalmasso, Andrea Cossa, Ilaria Servidio, Guido Campani, Elisa Bugani, F. A. Tucci, Lucrezia Stabellini, S. Pece, P. Pelicci
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 The rise of spatial transcriptomics (ST) is transforming our understanding of tumor heterogeneity by enabling high-resolution, location-specific mapping of gene expression across the tumor microenvironment. However, the translational potential of spatial transcriptomics is still limited by its high cost, hindering the assembly of large patient cohorts needed for robust biomarker discovery.



 Here we present Path2Space, a deep learning model trained on spatial transcriptomics data to predict spatial gene expression directly from histopathology slides. Studying breast cancer, the model was first trained on one ST breast cancer patient cohort and further independently tested and validated on two other ST cohorts.



 Path2Space identifies 4, 500 genes whose spatial expression is robustly predicted in all three cohorts, markedly more accurate than existing ST predictors. Second, we show that based on the inferred ST, Path2Space accurately infers cell-type abundances in the tumor microenvironment (TME). Thirdly, Applying Path2Space to the TCGA breast cancer cohort, we identify three TME spatially grounded subsets of patients that have markedly different survival rates. Fourth, analyzing two large breast cancer patient cohorts treated with trastuzumab, we show that the heterogeneity of the inferred HER2 levels across the tumor microenvironment is predictive of response, concordant with the bystander hypothesis. Finally, we develop a spatially interpretable model for predicting trastuzumab response directly from the H&E-slides. Interestingly, this model’s performance surpasses that of existing response prediction models that use measured bulk multi-omics data.



 Path2Space is the first approach capable of robustly delineating the tumor microenvironment on an unprecedented scale directly from the tumor slides. It heralds the upcoming development of robust spatially grounded biomarkers of patients survival and treatment that promises to transform precision oncology in both the developed and developing worlds.



 Eldad D. Shulman, Emma M. Campagnolo, Roshan Lodha, Amos Stemmer, Thomas Cantore, Beibei Ru, Andrew Wang, Tom Hu, Maclean Nasrallah, Danh-Tai Hoang, Kenneth Aldape, Eytan Ruppin. Path2Space: An AI approach for cancer biomarker discovery via histopathology inferred spatial transcriptomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6353.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae0c72c8a9ee37e81165f38f3251f77dd4884159" target='_blank'>
              Abstract 6353: Path2Space: An AI approach for cancer biomarker discovery via histopathology inferred spatial transcriptomics
              </a>
            </td>
          <td>
            E. Shulman, Emma M. Campagnolo, Roshan Lodha, Amos Stemmer, Thomas Cantore, Beibei Ru, Andrew Wang, Tom Hu, MacLean P Nasrallah, Danh-Tai Hoang, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 The tumor microenvironment (TME) is complex and plays a key role in cancer development, progression and metastasis. Two layers of information are critical for understanding tumor compositions (1) the proportion of each cell type (2) the level of gene expression in each cell type.
 Many single-cell profile-assisted algorithms have been developed to infer cell type proportions from bulk RNA-seq data, including CIBERSORTx, MuSic and Scaden, and there are several benchmark studies published to compare the methods. However, most of the deconvolution methods do not support prediction of gene expression in a heterogenous population of tumor cells, and there’s no published study systemically comparing the gene expression prediction performance of the methods.
 To overcome the limitations, we simulate large-scale of pseudobulk data using scRNA-seq profiling from lung tumors, representing various levels of tumor purities and cell lineages, to compare the cell fraction and gene expression prediction performance of recently published TME deconvolution methods. Our results indicate BayesPrism is the best at predicting gene expression in each of the cell types, and differential gene expression between two groups across cell types, especially in tumor epithelial cells, with higher balanced accuracy and correlation with ground truth. The inclusion of normal epithelial cells in the reference could increase additional risks for tumor deconvolution.
 Altogether we evaluate both cell type proportion and gene expression prediction performance of recently published methods and offer a guideline for future benchmarking work. Currently we're working on the downstream application of inferring cellular composition and gene expression in large cohorts of bulk RNA-seq data, allowing for more insights into tumor-intrinsic expression and tumor-microenvironment interactions.


 Fanying Tang, Dean Lee, Lexiang Ji, Jincheng Wu. Performance evaluation of tumor microenvironment deconvolution and gene prediction methods in lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6283.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/653b80a32b949cb3e328b878333ade7e654abf52" target='_blank'>
              Abstract 6283: Performance evaluation of tumor microenvironment deconvolution and gene prediction methods in lung cancer
              </a>
            </td>
          <td>
            Fanying Tang, Dean Lee, Lexiang Ji, Jincheng Wu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Renal clear cell carcinoma (ccRCC) develops through significant molecular and spatial reprogramming, transforming normal kidney tissue into a malignant state and reshaping the tumor microenvironment. To investigate the transition from normal to cancerous tissue, we utilized the CosMx® Whole Transcriptome (WTX) panel to perform high-resolution spatial transcriptomic imaging of FFPE sections containing ccRCC and adjacent normal kidney. H&E staining on the same tissue sections enabled direct correlation of molecular profiles with histopathological features, uncovering novel insights into tumorigenesis. FFPE tissue sections comprising equal parts ccRCC and adjacent normal kidney tissue underwent CosMx WTX imaging to analyze approximately 19, 000 genes at single-cell resolution. The same sections were stained with H&E, providing histopathological context for tumor boundaries, tumor-normal interfaces, and regions of vascularization or necrosis. Custom software aligned transcriptomic and histopathological images, enabling the integration of molecular and morphological data for downstream analyses and AI model development. CosMx WTX detected thousands of transcripts per cell with exceptional sensitivity and accuracy. Spatial transcriptomics revealed distinct transcriptional programs and cell-type-specific changes across normal and tumor regions. Ligand-receptor analysis identified critical interactions driving tumorigenesis, while comparative analysis between transcriptome profile and H&E stain highlighted molecular shifts in tumor-adjacent normal areas, preserving renal tubule morphology but exhibiting subtle transcriptomic alterations. The tumor-normal interface displayed unique gene expression signatures, identifying spatially restricted therapeutic targets. This study presents a high-resolution spatial atlas of ccRCC, detailing molecular and cellular transitions from normal to tumor tissue. Integrating CosMx WTX with H&E-guided annotation not only identifies the tumor-normal interface as critical for understanding progression but also highlights the potential for AI-powered diagnostic tools. By linking transcriptomic signatures to routine histopathological images, this work sets the stage for future advancements in automated cell typing and transcriptome prediction, transforming research in clinical workflows and precision oncology.
 -Bruker Confidential-


 Shanshan He, Liang Zhang, Michael Patrick, Sierra McKenzie, Haiyan Zhai, Christine Kang, Michael Deryck Rhodes, Joseph Beechem. Mapping the spatial whole transcriptome from normal to tumor tissue in renal clear cell carcinoma: tumorigenesis and microenvironmental shifts at single-cell resolution [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2068.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2aaaf6ecddf1942a365097ba44e7686790d4a91" target='_blank'>
              Abstract 2068: Mapping the spatial whole transcriptome from normal to tumor tissue in renal clear cell carcinoma: tumorigenesis and microenvironmental shifts at single-cell resolution
              </a>
            </td>
          <td>
            Shanshan He, Liang Zhang, Michael Patrick, Sierra McKenzie, Haiyan Zhai, Christine Kang, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 e17615


 Background:
 Uterine carcinosarcoma (UCS) is a rare gynecologic malignancy that has poor clinical outcomes and few targeted therapies. This aggressive variant of endometrial carcinoma has a uniquely biphasic appearance and morphology of metastases from these tumors can be carcinoma, sarcoma, or carcinosarcoma. The tumor immune microenvironment (TIME) plays a crucial role in the behavior of cancers. Recent advances in single cell spatial transcriptomics allow for greater insight into the TIME of these aggressive tumors.
 Methods:
 Ten samples from four patients with multiple matched longitudinal samples underwent single-cell spatial transcriptomic (scST) profiling (Nanostring CosMx 6k). Standard segmentation and QC filters were applied. Cell typing was performed with confirmation by expert pathologist annotation of tissue regions. Normalized gene expression was compared within the clusters based on clinicopathologic characteristics.
 Results:
 scST profiling of 80,843 high-quality cells revealed 38 distinct clusters (11 carcinoma, 12 sarcoma, 9 immune, 7 benign mesenchymal). Tumor clusters were largely patient-specific with tumors demonstrating diverse transcriptional heterogeneity while metastatic tumors were composed of several less diverse clones. While immune cell clusters represented cells from all patients, there was significant patient and sample heterogeneity in the proportions of immune cell subsets observed. Macrophages were the most abundant immune cell type in all samples, plasma cells were common in one patient's samples, and rare T cells were found only in specific cellular niches. Transcriptomic features of intra-tumoral macrophages were assessed based on site (primary versus distant metastasis) and temporal (naïve versus treated with chemotherapy) groupings. Compared to tumor associated macrophages (TAM) in primary tumors, TAMs in distant metastases were more likely to have M2-like features with higher expression of
 CXCL2, CD163, CD209,
 and
 CD44
 (P<0.005). TAMs residing in chemotherapy-treated tumors had higher expression of
 IGF2, CXCL2, CXCL1, CD163, CD209, CXCL16, CD24
 ,
 CD44, CD93
 (P<0.005). Upregulation of pro-angiogenic pathways (i.e.
 VEGFA
 ) was common in both distant metastases and post-chemotherapy UCS TAMs.
 Conclusions:
 Uterine carcinosarcoma is a highly heterogeneous tumor both with respect to tumor cell and immune microenvironment features. This work highlights transcriptomic characteristics in macrophages in the UCS microenvironment. Notably, macrophages in distant metastases and in patients with prior chemotherapy were found to have more M2 TAM-like macrophages than primary untreated tumors. These results provide rationale for both TAM-based immunotherapy and anti-angiogenic approaches in UCS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ce5dd582063cc9e8b6f92d0391909010dfa4c1" target='_blank'>
              Single cell characterization of uterine carcinosarcoma and analysis of transcriptomic shifts in tumor-associated macrophages in metastatic and chemotherapy-treated tumors.
              </a>
            </td>
          <td>
            C. Nief, Sabrina Zdravkovic, Sahar Nasr, Phoebe Hammer, Brooke Howitt
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Upper aerodigestive squamous cell carcinoma (UASCC) represents an aggressive and often fatal group of malignancies. Mechanistically, the early steps of neoplastic transformation remain poorly defined. Translationally, despite extensive genomic characterization of UASCC, the application of these findings to clinical practice remains highly limited. We have identified a crucial gap in the field - the lack of viable and valid human models that can: i) faithfully recapitulate the stepwise progression from precursor lesions to tumors, and ii) rigorously model genotype-phenotype relationships. Here, we developed two complementary, cross-species, genetically-defined organoid systems: (1) a genome-sequenced, patient-derived organoid platform comprising 46 models spanning normal tissue, precursor lesions, and tumors along the oral-esophageal neoplastic continuum; and (2) a CRISPR/Cas9-engineered organoid platform of 16 models, targeting key UASCC genomic drivers to recreate the full spectrum of malignant transformation (hyperplasia → dysplasia → carcinoma). Using these models, we uncovered critical insights into early squamous neoplastic evolution. For example, single-cell RNA sequencing of TP53/CDKN2A double-knockout (DKO) organoids revealed an expanded population of quiescent basal and proliferative squamous cells, alongside a reduction in differentiated squamous cells. This differentiation imbalance was exacerbated by the introduction of additional drivers, such as PIK3CAE545K mutation (DKOP). Furthermore, a senescence-like program, identified as a hallmark of normal squamous epithelium, was markedly attenuated in DKO organoids and further diminished in DKOP organoids, with ANXA1 emerging as a master regulator. We identified a novel ANXA1-SMAD3-p27KIP1 pathway as a critical driver of this senescence-like program, suppressing neoplastic phenotypes in squamous organoid models. Leveraging these genetically-defined organoid systems, we further performed high-throughput drug screening at single-organoid resolution, utilizing bioprinting technology coupled with machine learning. As anticipated, PIK3CA mutations conferred a survival advantage against most therapies tested. However, intriguingly, genetically-engineered organoids with PIK3CA mutations demonstrated heightened sensitivity to mitomycin C and onalespib. These findings were further validated using independent patient-derived HNSCC organoids harboring PIK3CA mutations. In summary, this study provides critical insights into the mechanisms of early squamous neoplastic progression and underscores the utility of genetically-defined organoid platforms for forward genetic studies in cancer. These rigorously validated models offer a powerful platform for identifying genome-guided therapies and advancing precision oncology.


 Hua Zhao, Young Min Park, Yueyuan Zheng, Qiong Mao, Luda Lin, Casey Collet, Stephanie Wong, Boyan Hu, Uttam K. Sinha, Alice Soragni, Dechen Lin. Genetically-defined organoid models reveal early mechanisms of squamous neoplastic progression and therapeutic vulnerabilities in squamous cell carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1250.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417447ea57e32086f8cc2713f35e946d6685f5c1" target='_blank'>
              Abstract 1250: Genetically-defined organoid models reveal early mechanisms of squamous neoplastic progression and therapeutic vulnerabilities in squamous cell carcinoma
              </a>
            </td>
          <td>
            Hua Zhao, Y. Park, Yueyuan Zheng, Qiong Mao, Luda Lin, Casey Collet, Stephanie Wong, Boyan Hu, Uttam K Sinha, A. Soragni, De-Chen Lin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Tumor heterogeneity is predicted to confer inferior clinical outcomes with precision-based strategies, however, modeling heterogeneity in a manner that still represents the tumor of origin remains a formidable challenge. Sequencing technologies are limited in their ability to identify rare subclonal populations and predict response to treatments for patients. Patient-derived organotypic cultures have significantly improved the modeling of cancer biology by faithfully representing the molecular features of primary malignant tissues. Patient-derived cancer organoid (PCO) cultures contain subclonal populations with the potential to recapitulate heterogeneity, although treatment response assessments commonly ignore diversity in the molecular profile or treatment response. Here, we demonstrate the advantage of evaluating individual PCO heterogeneity to enhance the sensitivity of these assays for predicting clinical response. Additionally, organoid subcultures identify subclonal populations with altered treatment response. Finally, dose escalation studies of PCOs to targeted anti-EGFR therapy are utilized which reveal divergent pathway expression when compared to pretreatment cultures. Overall, these studies demonstrate the importance of population-based organoid response assessments, the use of PCOs to identify molecular heterogeneity not observed with bulk tumor sequencing, and PCO heterogeneity for understanding therapeutic resistance mechanisms. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-96204-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d73b21d8a323278ef745932d923e8c41351e22f0" target='_blank'>
              Subclonal response heterogeneity to define cancer organoid therapeutic sensitivity
              </a>
            </td>
          <td>
            J.D. Kratz, Shujah Rehman, Katherine A. Johnson, A. Gillette, Aishwarya Sunil, Peter F Favreau, Cheri A. Pasch, Devon D Miller, Lucas Zarling, Austin H. Yeung, L. Clipson, Samantha J. Anderson, Alyssa K Steimle, Carley M. Sprackling, K. Lemmon, Daniel E Abbott, M. Burkard, Michael F. Bassetti, J. Eickhoff, Eugene F Foley, Charles P Heise, Randall J Kimple, Elise H Lawson, N. LoConte, S. Lubner, Daniel L Mulkerin, Kristina A Matkowskyj, Cristina B. Sanger, Nataliya V. Uboha, Sean J Mcilwain, Irene M Ong, Evie H Carchman, Melissa C Skala, Dustin A. Deming
          </td>
          <td>2025-04-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Checkpoint therapies and engineered immune cell therapies are increasingly becoming the mainstay when it comes to cancer treatments. Continued and improved success of these therapies relies on our deep understanding of the checkpoint marker and immune cell biology, cell-cell interaction in the tumor microenvironment (TME), and immune cell activation. Spatially visualizing the expression of soluble factors such as cytokines and chemokines along with immune cell markers can provide information about the immune cell composition and enable a comprehensive understanding of mechanisms underlying immune cell recruitment, infiltration and exclusion. To address this, we have developed a fully automated spatial multiomics protocol on the COMET™ platform that enables RNA detection using the RNAscope™ HiPlex Pro assay combined with protein detection using sequential immunofluorescence (seqIF™, (Rivest et al, 2023) ) to integrate same-section sequential detection of up to 12 RNAs followed by up to 24 proteins. This workflow allows the detection of any RNA and protein target of interest by utilizing existing RNAscope probes or generating a custom design for any RNA target and use standard, non-conjugated primary antibodies for protein detection. We used Lunaphore’s HORIZON™ software to analyze this multiomics dataset. Here, we have demonstrated the precise spatial profiling of FFPE solid tumors through the detection of key cytokines indicative of activated T cells and macrophages by using cytokine RNA probes targeting IFNG, IL-1B, TNFA, TGFB1, IL-4, IL-6, IL-8, IL-10 among others in combination with cell marker antibodies detecting CD3, CD8, CD4, FOXP3 and CD68.In addition, we were also able to visualize T cell recruiting chemokines and receptors such as CXCL10, CXCL9, CXCL13, CXCL12 and CXCR3, and spatially map tumor-infiltrating T cells within tissue context and with subcellular resolution. To that end, we successfully demonstrated same-slide RNA-protein detection for profiling the TME. Using single-cell spatial multiomics we were able to identify unique phenotypes within the tumors that might directly correlate to immunosuppressive or immunostimulatory microenvironments. Detailed mapping of the cellular components of the TME has the potential to better predict therapeutic response for checkpoint and immune-based therapies.


 Anushka Dikshit, Cansaran Saygili Demir, Rose Delvillar, Emerald Doolittle, Alice Comberlato, Arec Manoukian, Maria-Giuseppina Procopio, Pino Bordignon, Saska Brajkovic, Maithreyan Srinivasan. Profiling the tumor- immune landscape with a same-section multiomics assay [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5304.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0577e3703eceb9dd16f950f2b2b372372b328f4f" target='_blank'>
              Abstract 5304: Profiling the tumor- immune landscape with a same-section multiomics assay
              </a>
            </td>
          <td>
            A. Dikshit, Cansaran Saygili Demir, Rose Delvillar, Emerald Doolittle, Alice Comberlato, Arec Manoukian, M. Procopio, Pino Bordignon, S. Brajkovic, Maithreyan Srinivasan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Metastatic breast cancer is the most aggressive form of breast cancer leading to poor disease-free patient survival and accounting for 90% of all breast cancer-related deaths. The specific mechanisms that promote metastasis are not fully understood owing to the complex interplay between cancer cells and their surrounding microenvironment. This microenvironment involves clonal and sub-clonal heterogeneity in genomic, transcriptomic and protein signaling network across different cancer cell sub-populations. Understanding the molecular mechanism behind cancer metastasis demands state of the art single cell and spatial technologies that can provide deeper insight on disease progression.
 Since, cancer is heavily influenced by genetic changes, the study of the 3D- genome organization in single cells helps to improve understanding of disease progression and prognosis. The genome is organized and packaged in a highly structured manner inside the nucleus of mammalian cells and exhibits dynamic reorganization throughout disease progression. Understanding this 3D organization at the single-cell level with high spatial resolution is crucial for disease research.
 In this abstract, we present a novel jebFISHTM protocol on the PaintScapeTM platform that can be used to understand differences in global 3D genome organization among several localized and metastatic breast cancer cell lines at single cell, sub-population and population level. Using a highly multiplexed genome wide panel of chromosomal targets, we will show disruption in chromosome territory, radius of gyration, chromosomal instability between localized and metastatic breast cancer cells at single cell and sub-population level. We will also show the signature of specific 3D genome structural alterations and interchromosomal interactions related to breast cancer metastasis compared to normal breast cells. In addition, we investigated differences in global 3D genome architecture of different breast cancer sub-types such as HER2 enriched vs triple negative breast cancer cells. We will show how different breast cancer sub-types carry specific chromosomal instability and differential interchromosomal interaction patterns at single cell, sub-chromosomal and sub-population level. We envision this study will improve our understanding on breast cancer disease progression and provide deeper insight into the underlying genomic heterogeneity of single cancer cells and sub-populations which might help to design better treatment options in the future. We propose that this study serves as proof-of-principle and is generalizable to other types of cancers where changes in 3D genome organization may occur at different cancer stages.


 Shyamtanu Chattoraj, Huy Nguyen, Jude Dunne. Mapping single cell spatial 3D genome landscape of localized and metastatic breast cancer sub-types [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2852.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b565f8797f4fa651d27cbd6107a1a5335c33485f" target='_blank'>
              Abstract 2852: Mapping single cell spatial 3D genome landscape of localized and metastatic breast cancer sub-types
              </a>
            </td>
          <td>
            Shyamtanu Chattoraj, Huy Nguyen, Jude Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC), one of the most aggressive subtypes of breast cancer, is associated with poor prognosis and limited treatment options, particularly when diagnosed at later stages. The extensive inter- and intra-tumoral heterogeneity observed in TNBC further complicates its clinical management. Understanding the molecular difference within this TNBC is crucial for elucidating its biological characteristics and clinical behaviors, as well as for developing personalized therapeutic approaches. In this study, we employ digital spatial profiling (DSP) technology to quantitatively analyze transcript and protein abundance across spatially distinct regions of TNBC, including primary tumor cores, surrounding stroma regions, and metastatic sites. The analysis reveal marked spatial heterogeneity in gene expression profiles, cellular composition, and the immune microenvironment across these regions. Additionally, we identify distinct TNBC subtypes based on unique gene expression patterns. In primary tumor, Basal-like 1 (BL1) emerged as one of the predominant subtypes and was associated with an increased present of memory B cells. We also identified a subgroup resembling unclassified (UNS) subtypes, characterized by low macrophage levels. Moreover, tumor tissues exhibited multiple TNBC subtypes within a single tumor site. This intricate subtyping within the tumor mass underscores the challenges associated with developing effective treatment for TNBC. Our findings demonstrate the power of DSP technology in elucidating the spatial dynamics of TNBC at the molecular level. The identified molecular signatures and subtypes hold potential for improving diagnostic accuracy, enabling targeted drug development, and guiding personalized treatment approaches. Ultimately, this work underscores the utility of spatial transcriptomics for addressing the challenges posed by TNBC heterogeneity and advancing precision medicine for this aggressive cancer subtype.


 Fengyuan Huang, Deepa Bedi, Clayton Yates. Inter- and intra-tumoral heterogeneity of TNBC determined by spatial gene transcriptomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6258.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb874fc1b76423af08fc061b2412437123e6829d" target='_blank'>
              Abstract 6258: Inter- and intra-tumoral heterogeneity of TNBC determined by spatial gene transcriptomics
              </a>
            </td>
          <td>
            Fengyuan Huang, Deepa Bedi, Clayton C. Yates
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Radiation remains a primary treatment for glioblastoma, yet tumors frequently recur within two years. As a model for investigating the biology of recurrent GBMs, we used orthotopic xenografts initiated of GBM-stem like cells (GSCs) implanted into the right striatum of nude mice. Thirty-five days post-implant, mice bearing brain tumor xenografts were locally irradiated (10Gy) and brains collected 14 and 21 days later. In untreated mice, tumors were highly diffuse with tumor cells invading from the injection site in the right hemisphere (RH) to the anterior part of the brain, invading the corpus callosum, cortex, and olfactory bulb (OB). A different growth pattern was detected in tumors that regrew after irradiation. At 14 days post-10Gy, tumors were located primarily in the OB with an increased cell density and a clearly demarcated border. 21 days post-10Gy tumor cell density increased with the demarcated border extending further into the hemisphere, i.e., farther away from the OB. This growth pattern suggests that tumor regrowth after irradiation is initiated in the OB, consistent with previous results showing that GBM cells in the OB are radioresistant compared to those in the RH. To compare transcriptomes of untreated and recurrent tumors we used digital spatial profiling (DSP). Tumor cells were identified by Sox2 staining, and regions of interest (ROIs) were selected in various locations within the OB and the RH of untreated tumors, and throughout recurrent tumors (D21 post-10Gy). Cluster analysis showed that ROIs across the OB and RH of untreated tumors were more similar to each other as compared to ROIs from the tumors that regrew after irradiation. Variance in the expression of individual genes across the ROIs was reduced in the recurrent tumors compared to the untreated tumors. These results suggest a reduction in phenotypic heterogeneity among the cells of the regrown tumor. Comparative gene expression analysis identified 463 differentially expressed genes with 319 downregulated and 144 upregulated in the recurrent versus untreated tumors. GSEA revealed a significant enrichment of pathways related to cell cycle regulation in the regrown tumors as compared to non-irradiated tumors. Further examination of the cell cycle control related pathways indicated an enrichment in proteasome genes. Specifically, 22 genes encoding proteasome proteins were significantly upregulated in the recurrent tumors compared to the untreated group, suggesting that the proteasome may provide a therapeutic target unique to the recurrent GBM. Altogether, these results suggest that this xenograft model of GBM regrowth after irradiation, characterized by an altered growth pattern and a distinct molecular profile may be useful for investigations into the biology and treatment of recurrent GBM.


 Charlotte Marie Degorre. Spatial transcriptome profiling shows a decrease in heterogeneity of GBM xenografts that regrow after irradiation and suggests novel therapeutic targets [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1823.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782f6f0e92847d06cb3d67cdcc10afaa17e2f75c" target='_blank'>
              Abstract 1823: Spatial transcriptome profiling shows a decrease in heterogeneity of GBM xenografts that regrow after irradiation and suggests novel therapeutic targets
              </a>
            </td>
          <td>
            Charlotte M. Degorre
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 The emergence of tools enabling researchers to perform high-plex spatial transcriptomics with single-cell resolution has revolutionized our understanding of tumor development. However, tissue samples with degraded RNA or extensive crosslinking present challenges for gene expression measurements. For decades, cancer tissue samples have been collected and preserved using formalin-fixation and embedding in paraffin (FFPE), which is not optimal for preserving RNA integrity. This has limited the depth of insights researchers can obtain from archival samples.
 The Multiplexed Error-Robust Fluorescence In Situ Hybridization (MERFISH) technology facilitates direct RNA profiling in situ with high sensitivity and resolution. The high accuracy and high plexity are enabled by tiling probes along the length of a transcript, which is particularly challenging in samples with fragmented RNA, such as in FFPE. The MERFISH 2.0 chemistry and sample preparation workflow was developed specifically to improve transcript detection efficiency in measurement of up to 1000 genes in low quality tissues. These improvements can be applied to tissues of up to 3cm2 using the MERSCOPE® Ultra™ Platform. Here we applied the improved capabilities of MERFISH 2.0 to profile human breast cancer and human lung cancer tumors over a range of sample qualities.
 Breast and lung tumor samples, both whole sections and tissue microarrays (TMA), were run on the MERSCOPE Ultra Platform using a panel of 815 immuno-oncology genes with both MERFISH 1.0 and MERFISH 2.0. The resulting data was used for spatial and single-cell analyses. We show that in all tissues, particularly in lower quality tissues, MERFISH 2.0 increases the counts and subsequent quality of data analysis, such as increased detection of cell types and increased neighborhood associations. Inter- and intra-tumor immune profiling showed improved detection of low-expressing markers, enabling deeper study of the tumor microenvironment.
 Spatially resolved transcriptomic profiling of low-quality samples at single-cell level offers significant opportunities for understanding how cancers develop in situ. These improvements will enable new genomic inquiries into how tumors develop and modulate their environment, which in turn will open up new areas of therapeutic research.


 Manisha Ray, Justin He, Bin Wang, Bing Yang, Sudhir Tattikota, Timothy Wiggin, Hao Wang, Jichuan Zhang, Lizi Maziashvili, Alexander Genshaft, Peter Reinhold, Brittany Auclair, Robert Mathis, Shawn Wang, Jiang He, George Emanuel. New developments in spatial transcriptomics unlock insights into the tumor microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5307.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e947118fcfc484004bcc041ad01d4d6f777a678" target='_blank'>
              Abstract 5307: New developments in spatial transcriptomics unlock insights into the tumor microenvironment
              </a>
            </td>
          <td>
            Manisha Ray, Justin He, Bin Wang, Bing Yang, Sudhir Tattikota, Timothy Wiggin, Hao Wang, Jichuan Zhang, Lizi Maziashvili, Alexander Genshaft, Peter Reinhold, Brittany Auclair, Robert Mathis, Shawn Wang, Jiang He, George Emanuel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="It is now well appreciated that tumor growth depends on the tumor microenvironment (TME) comprising various non-malignant stromal cell types, including cancer-associated fibroblasts (CAFs). Cancer-associated fibroblasts (CAFs) are responsible for secreting the extracellular matrix (ECM) prominent in certain solid tumors such as pancreatic adenocarcinoma (PDAC). Despite our increasing knowledge of CAFs, therapeutic targeting of CAFs is currently not a feasible strategy, as CAFs are highly heterogeneous and extremely plastic, with each CAF population able to adopt both tumor-promoting and -restraining functions. We hypothesize that the plasticity and functional heterogeneity of CAFs can be explained by their flexible metabolism in response to environmental cues and the existence of discrete metabolic states within CAF subpopulations. While metabolic states can be predicted from single-cell transcriptomic data, this does not provide information on metabolic activity and potential, nor does it allow perturbations required for validation. Scalable and tractable methods to assess metabolic activity and CAF heterogeneity in parallel at the single cell level are needed to distinguish numerous existing CAF states do not exist. To address this need, in this work, we developed a strategy to study CAF metabolic activity and phenotypic heterogeneity at the single-cell level. We designed and optimized a 25-plex immunophenotyping panel of CAF and fibroblast markers that can be combined with numerous metabolic probes via Full Spectrum Profiling (FSP) flow cytometry. In addition, we compiled an analysis pipeline to facilitate the analysis of high-dimensional flow cytometry data. Using established 2D and 3D mono- and co-culture assays, we demonstrate the potential of our panel to be multiplexed with chemical probes for cell proliferation, lipid content, lipid uptake, fatty acid oxidation, glucose uptake, cholesterol content, hypoxia, or mitochondrial potential. Deploying our strategy in a KPC organoid-derived murine model of PDAC, we observed subpopulations of CAFs with differing metabolic potentials that are consistent with our predictions from transcriptomic metabolic modeling. With the high throughput and single-cell nature of FSP, our strategy represents a novel and modular tool for studying CAF phenotypic and metabolic heterogeneity.


 Kevin Muñoz Forti, Marta Storl-Desmond, Daniel Martínez, Sayana Issac, Shan Cao, Simon Schwörer. Full spectrum profiling flow cytometry to study cancer-associated fibroblast heterogeneity [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2667.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50ba163056d130b5c3aed32e5c9041e544f454cf" target='_blank'>
              Abstract 2667: Full spectrum profiling flow cytometry to study cancer-associated fibroblast heterogeneity
              </a>
            </td>
          <td>
            Kevin Muñoz Forti, Marta Storl-Desmond, Daniel Martínez, Sayana Issac, Shan Cao, Simon Schwörer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Fibroblast activating protein (FAP) is an attractive target for therapeutic intervention being mainly expressed in malignant tissues. Previous studies have targeted FAP using a variety of methods yet failed in clinics. Here, we utilize FAP-targeted T cell engager (TcE) in KPCY tumor models to recapitulate a pre-clinical environment. The use of FAP- TcE efficiently depleted FAP+ fibroblasts in vitro and in vivo from tumors. This resulted in bystander killing of tumor cells and tumor growth control in vivo. This therapeutic effect was not dependent on priming via lymph node activation as shown in other publications. To understand the complex interactions with FAP expression, we perform comprehensive single cell RNA-sequencing profiling before and after FAP-TcE depletion to understand the pivotal genetic roles of FAP depletion in all cell types. This analysis initially revealed a T cell exhaustion phenotype with upregulated of PD-1 on T cells after FAP-TcE treatment. However, while addition of anti-PD-1 therapy significantly enhanced tumor efficacy, complete control was absent. RNA sequencing of the fibroblasts identified depletion of a myCAF-like fibroblast and an outgrowth of a mixed CAF population with phenotypes of both myCAF and iCAF. This population correlated with tumor growth, and cell cycle gene signatures. Future work will investigate the implications of this fibroblast population’s role in subverting immune suppression.


 Joshua R. Tagore, Robert Joseph Norgard, Pratha Budhani, Sai Charan Penikalapati, Jessica N. Egan, Di Feng, Leticia Corrales-Pecino, Anna Bachmayr-Heyda, Claudia Reichel-Voda, Abhishek Kashyap, Youli Xia, Sarah A. O’Brien. A T cell engager targeting cancer associated fibroblasts inhibits tumor growth and modulates the tumor microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2583.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528f741dd4b2dd6cad4ba0b800a0b727e027e98c" target='_blank'>
              Abstract 2583: A T cell engager targeting cancer associated fibroblasts inhibits tumor growth and modulates the tumor microenvironment
              </a>
            </td>
          <td>
            Joshua R Tagore, Robert J Norgard, Pratha Budhani, Sai Charan Penikalapati, Jessica N Egan, Di Feng, Leticia Corrales-Pecino, Anna Bachmayr-Heyda, Claudia Reichel-Voda, Abhishek Kashyap, Youli Xia, Sarah A O'Brien
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c490b5674b718496318b2118c9f55620a929ba49" target='_blank'>
              Dictionary of human intestinal organoid responses to secreted niche factors at single cell resolution
              </a>
            </td>
          <td>
            Meghan M. Capeling, Bob Chen, Kazeera Aliar, Elisa Penna, Veronica Ibarra Lopez, Conrad Foo, Sandra Rost, Loryn Holokai, Xinming Tong, Devan Phillips, Caden W. Sweet, Jing Li, Sharmila Chatterjee, Elizabeth Skippington, Z. Modrusan, Lisa M. McGinnis, Runmin Wei, M. Keir, O. Rozenblatt-Rosen, Michelle B. Chen
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="
 2042


 Background:
 Glioblastoma (GBM) is the most lethal primary brain tumor in adults, with a median survival of ~15 months despite current therapies (surgery, radiation, temozolomide). Advances like immune checkpoint inhibitors, anti-angiogenic agents, and tumor vaccines have shown suboptimal results. The 2021 WHO classification highlights molecular markers (e.g., IDH, MGMT) for better stratification, but these fail to fully capture tumor microenvironmental dynamics. Using an AI-driven transcriptomic approach, we identified novel prognostic subtypes in IDH-wildtype GBM, aiming to refine stratification, enhance understanding of tumor biology, and guide personalized therapeutic strategies.
 Methods:
 We accessed microarray data from The Cancer Genome Atlas (TCGA) (n=353 newly diagnosed, IDH-WT GBM) for a training set and RNA-seq data from the Chinese Glioma Genome Atlas (CGGA) (n=170 primary and n=106 recurrent tumors) for validation. A proprietary SphereBio machine learning–based algorithm was used to derive transcriptomic signatures with prognostic relevance. Subtypes were assessed via Kaplan–Meier analyses in the training cohort and tested in both primary and recurrent validation cohorts. Immune/stromal infiltration was quantified using a tumor deconvolution tool (DA_505), and pathway enrichment (GAGE) was performed on the validation sets.
 Results:
 AI-driven clustering revealed three transcriptomic subtypes with significant survival differences in both the training (p<0.0001) and primary validation (p=0.0004) cohorts. In the recurrent cohort, a similar survival trend by subtype was observed, though significance was diminished (p=0.12), likely due to limited sample size and therapy-related changes. Immune/stromal deconvolution showed distinct infiltration patterns: subtypes enriched for CD4+ and CD8+ T cells correlated with prolonged survival. Pathway enrichment analysis in both primary and recurrent tumors highlighted potential targets involving embryogenesis, immune modulation, cell cycle, and stress response. The persistence of a consistent survival trend and comparable microenvironment and pathway patterns suggest that these transcriptomic subtypes remain biologically relevant even after standard treatment.
 Conclusions:
 Our integrated transcriptomic and microenvironment-focused approach identified three prognostically distinct GBM subtypes, validated across independent cohorts. These findings underscore the utility of AI-driven transcriptomic signatures for personalized stratification, with the potential to guide targeted therapeutic strategies and inform clinical trial design in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff54807210ca2a362e060a895b6f6e635241ee" target='_blank'>
              AI-driven transcriptomic classification of glioblastoma: Associations with survival and tumor microenvironment.
              </a>
            </td>
          <td>
            J. Fernandez-Muñoz, Román Oberti, D. A. Reardon, Francisco Quintana, Guido Nicolas Molina, M. E. Guerrero-Gimenez
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Non-small cell lung cancer (NSCLC) accounts for the majority of lung cancer cases, with EGFR-mutant NSCLC among never smokers being particularly prevalent in Asian populations. A major clinical challenge in treating advanced NSCLC is the development of resistance to targeted tyrosine kinase inhibitors (TKIs), highlighting the need to comprehensively profile the cellular mechanisms underlying drug resistance. We propose to tackle this by investigating the adaptive features of residual cancer cells and the role of the tumour microenvironment in sustaining tumour growth, resistance, and disease progression. In this study, we utilise single nucleus and spatial transcriptome profiling to decipher the response of EGFR-mutant tumours in more than 30 NSCLC patients before and during on-TKI treatment, including those that have developed resistance. We present an analytical framework that maps individual residual cancer cell populations that persist after treatment. We then examine the functional phenotypes of neighbouring immune cells including T lymphocytes and mononuclear phagocytes on these populations, and then comparing the immune dynamics to tumours that have fully acquired resistance. Our study reveals a potential mechanism by which metabolic plasticity and paracrine signaling from myeloid cells in the lung tumour microenvironment contribute to the maintenance of cancer stem cell properties, likely driving disease recurrence. We propose to incorporate a wider range of ongoing clinical trial and preclinical data to support our current findings, and with the aim of defining novel therapeutic strategy that can selectively eliminate drug tolerant persister population in EGFR-mutant NSCLC.


 Regina Hoo, Aaron Tan, Dawn Lau, Komal Gupta, Kenneth Chow, Jia Chi Yeo, Lan Ying Wang, Poh Sheng Joe Yeong, Anders J. Skanderup, Daniel S. Tan. Tracking adaptation and response to targeted therapies in EGFR-mutant non-small cell lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1600.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3627b48e9188c47ac8adb02879b131e72038b2ed" target='_blank'>
              Abstract 1600: Tracking adaptation and response to targeted therapies in EGFR-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Regina Hoo, Aaron C Tan, Dawn Lau, Komal Gupta, Kenneth Chow, Jia-Chi Yeo, Lan Ying Wang, Poh Sheng Joe Yeong, A. Skanderup, D. S. Tan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Prostate cancer (PCa) metastasis significantly worsens patient prognosis, particularly when it spreads to the lungs. Understanding the mechanisms underlying PCa metastasis is crucial for developing effective therapies. Here, we developed a genetically engineered mouse model by activating MAPK pathway in the PB-Cre Pten p53 Smad4 mice, creating a model that mimics the aberrant cell signaling associated with human metastatic PCa and exhibits strong metastatic phenotype. We observed dynamic changes in epithelial-to-mesenchymal transition (EMT) and mesenchymal-to-epithelial transition (MET) during metastasis: circulating tumor cells (CTCs) displayed mesenchymal characteristics, while lung metastases regained epithelial features.
 Transcriptomic profiling of primary tumors, CTCs, and lung metastases revealed that lung metastatic cells (LM) upregulated pro-inflammatory cytokines, notably IL-1α. This cytokine profile correlated with increased recruitment and activation of neutrophils in the premetastatic lung niche, as confirmed by single-cell RNA sequencing and flow cytometry. Functional experiments demonstrated that depleting neutrophils using anti-Ly6G antibodies or inhibiting their recruitment and activation via a CXCR1/2 antagonist (currently in clinical trials) significantly reduced lung metastasis and extended survival in mice. Similarly, IL-1 receptor knockout mice showed decreased neutrophil infiltration and fewer metastatic nodules, indicating the pivotal role of IL-1 signaling in neutrophil-mediated metastasis.
 Mechanistically, we identified the transcription factor KLF4 as the master regulator of both the EMT-to-MET plasticity and IL-1α expression in metastatic cells. Chromatin immunoprecipitation confirmed KLF4 binding to the IL-1α promoter, leading to the upregulation of neutrophil-recruiting cytokines. Knockdown of KLF4 in LM cells reduced IL-1α levels and metastasis, while overexpression enhanced them. Co-culture of tumor cells with neutrophils increased KLF4 expression and cytokine production, suggesting a feedback loop promoting metastasis.
 Clinical data analysis showed that high levels of KLF4 and IL-1α correlate with increased neutrophil infiltration and poorer disease-free survival in PCa patients. Our findings identify the crucial role of the KLF4/IL-1α axis and neutrophil recruitment in PCa lung metastasis. Targeting neutrophil activation or the KLF4/IL-1α signaling pathway may offer new therapeutic strategies for preventing or treating metastatic prostate cancer.


 Tianhe Huang, Yaoyin Li, Jun Wan, Liang Cheng, Xin Lu. Decoding and disrupting the coupled cellular plasticity and myeloid cell instigation in metastatic prostate cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5283.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94a034a47df46d31c3bc02070b210d92025cd351" target='_blank'>
              Abstract 5283: Decoding and disrupting the coupled cellular plasticity and myeloid cell instigation in metastatic prostate cancer
              </a>
            </td>
          <td>
            Tianhe Huang, Yaoyin Li, Jun Wan, Liang Cheng, Xin Lu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite inducing strong and durable clinical responses, targeted therapies do not eliminate advanced cancers, as a subset of tumor cells survives within residual tumors, eventually developing resistance. The ability of tumor cells to avoid therapeutic elimination can be mediated both by cell-intrinsic and microenvironmental mechanisms. Whilst the specific molecular mediators of cell-intrinsic and microenvironmental resistance are well understood, their relative contribution to in vivo therapeutic responses remains poorly defined. Using spatial histological inferences from experimental models of ALK+ NSCLC, we found that peristromal niches protected tumor cells from therapeutic elimination in vivo, enabling in vivo persistence. Whereas the development of bona fide resistance is associated with the development of the development of cell-intrinsic resistance, relapse of tumor growth reflects a combined effect of both cell-intrinsic and microenvironmental mechanisms. Mechanistically, the protective effect of the peristromal niche is not reducible to a single mechanism, instead reflecting a combined effect of multiple juxtacrine and paracrine mediators. The lack of reducibility to a single molecular mediator presents an obvious challenge to the therapeutic paradigms of targeting individual resistance mechanisms. We found that this challenge could be mitigated by shifting the therapeutic focus to orthogonal collateral sensitivities of residual tumors. Exploiting adaptive upregulation of HER2, associated with both cell-intrinsic and microenvironmental persistence, using the antibody-drug conjugate T-DXd strongly enhanced the effect of targeted therapies and suppressed the development of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc8346640dd25bd2d6003531c884b9d124f4ad3" target='_blank'>
              Multifactorial stroma-mediated resistance is a major contributor to residual disease under targeted therapies in lung cancers
              </a>
            </td>
          <td>
            A. Marusyk, Bina Desai, Tatiana Miti, Sandhya Prabhakaran, Daria Miroshnychenko, Menkara-Chinua N Henry, Viktoriya Marusyk, Pragya Kumar, Hilal Ozakinci, Chandler D. Gatenbee, Marilyn M Bui, Theresa A. Boyle, Jacob G. Scott, Philipp M. Altrock, E. Haura, Alexander Anderson, D. Basanta
          </td>
          <td>2025-04-24</td>
          <td>Research Square</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Purpose Small cell carcinoma of the ovary, hypercalcemic type (SCCOHT) is a rare and lethal cancer lacking effective treatment. Its genomic mutations and tumor microenvironment need further exploration. Methods We performed whole-exome sequencing or gene panel test to explore the SMARCA4 mutation spectrum in SCCOHT (15 samples). Single-cell RNA sequencing was conducted on one primary lesion with matched normal ovarian tissue and one recurrent lesion to investigate the intra-tumoral heterogeneity and immune microenvironment. Multiplex immunofluorescence staining validated T cell infiltration and PD-1 expression. Results 13/15 (86.7%) patients harbored SMARCA4 mutations. The loss of heterozygosity (LOH) occurred in 10/15 (66.7%) patients. Cancer cells and immune cells were observed in SCCOHT tumors. Cancer cells were further divided into seven subtypes and one from recurrent lesion exhibited the highest stemness accompanied by high expression of genes related to cell mitosis (AURKB, CHEK2, CCNB1, WEE1), DNA repair (BRCA1, RAD51) and epigenetic (EZH2, DNMT1). Immune cells mainly included macrophages and T cells. Lipid-associated tumor-associated macrophages (TAMs) was mainly in primary lesion while inflammatory cytokine-enriched TAMs in recurrent lesion. CD4+/ CD8+ T cell infiltration was observed in SCCOHT tumor and a certain proportion of T cells expressed PD-1. Conclusions SCCOHT exhibits universal SMARCA4 LOH and significant intra-tumoral heterogeneity, suggesting potential therapeutic targets, including CHEK2, CCNB1, and WEE1. Exhausted T cells and distinct TAM subsets infiltrate tumors. Targeting macrophage polarization or cytokine signaling may also be promising. These findings provide insights for developing novel therapies to improve outcomes in SCCOHT. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01649-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdba27a2b17d1b242308fa85ab562553932c973d" target='_blank'>
              Single-cell RNA sequencing reveals the intra-tumoral heterogeneity and immune microenvironment of small cell carcinoma of the ovary, hypercalcemic type
              </a>
            </td>
          <td>
            Yijun Gao, Kewei Zheng, Haowen Tan, Mingyi Kang, Bingjian Lu, Ling Chen, Jing Xu, Chong Lu, Ranran Chai, Congjian Xu, Yu Kang
          </td>
          <td>2025-04-09</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Glioblastoma (GBM), the most frequent and aggressive primary brain tumor, almost invariably recurs as an incurable disease after standard therapy. Molecular heterogeneity underlies the failure of current treatment and drives GBM progression. Although significant efforts have been directed at resolving the subclonal architecture, identifying and characterizing the genetic subpopulation in GBM has proven challenging. Recent advances in single-cell genomics (scWGS) have provided new opportunities for exploring the genomic complexity of tumors at unprecedented resolution. However, earlier methods of scWGS faced limitations due to poor amplification uniformity resulting in high number of artefacts. The introduction of primary template-directed amplification (PTA) has overcome this challenge allowing for more accurate detection of genomic alterations at single-cell resolution. In this study, we combined PTA scWGS of 5,689 single cells with single-nucleus RNA sequencing (snRNAseq) of 41,071 single nuclei from 25 GBM samples to explore the clonal architecture and the dynamics of genetic and non-genetic subclonal programs activated in GBM. PTA-based scWGS data showed optimal performance in terms of overdispersion and coverage breadth when compared to other single cell methods for measuring somatic copy number alterations (SCNA). The SCNA substructure of GBM was reconstructed by applying a graph-based clustering to scWGS data, which identified a median of 4 subclones per tumor. GBM samples showed a heterogenous distribution of genetic clones constituted by variable numbers of cells. For each sample we computed the consensus SCNA profiles to reconstruct the genetic phylogeny of GBM. The predicted evolutionary model showed a first trajectory from the diploid cell to the most recent common ancestor (MRCA) with the highest number of SCNA (median 14), including frequent GBM driver alterations (chr 7 gain, chr 10 loss, EGFR amp, CDKN2A del). Conversely, number and frequency of SCNA decreased in the clades diversifying the MRCA into the later ancestors. By aligning the scWGS clonal lineages with snRNAseq gene expression profiles from the same tumors, we found significant coordination of genetic and non-genetic mechanisms that diversified tumor subpopulations and led to divergent phenotypes recurring across the patients. The functional phylogeny of GBM identified two main branches of tumor evolution that emerged from the MRCA with the activation of neuronal functions and differentiation in one clade, and the acquisition of cell cycle programs, stem cell activities, and epithelial to mesenchymal transition in the opposite clade, respectively. This multi-omics approach enabled the reconstruction of the evolutionary dynamics of GBM at both genetic and transcriptional levels, revealing critical insights into clonal evolution and therapeutic resistance.


 Fulvio D'Angelo, Aram Ko, Luciano Garofano, Pedro Davila, Luigi Ferraro, Benjamin Currall, Sion Williams, Sakir Gultekin, Ashish Shah, Ricardo Komotar, Michael Ivan, Corneliu Sologon, Alice Laurenge, Alberto Picca, Amel Dridi-Aloulou, Bertrand Mathon, Franck Bielle, Stephan Zuchner, Macarena de la Fuente, Marica Eoli, Chul-Kee Park, Michele Ceccarelli, Marc Sanson, Anna Lasorella, Antonio Iavarone. Exploring genetic and non-genetic evolution of glioblastoma by integrating single cell DNA and RNA sequencing [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6350.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a60e85c34b84d3ee7bcc4641260225bbd7766477" target='_blank'>
              Abstract 6350: Exploring genetic and non-genetic evolution of glioblastoma by integrating single cell DNA and RNA sequencing
              </a>
            </td>
          <td>
            F. D’Angelo, A. Ko, L. Garofano, Pedro Davila, Luigi Ferraro, Benjamin Currall, S. Williams, S. Gultekin, Ashish Shah, Ricardo J Komotar, Michael Ivan, Corneliu Sologon, A. Laurenge, A. Picca, Amel Dridi-Aloulou, B. Mathon, Franck Bielle, Stephan Zuchner, M. D. L. Fuente, M. Eoli, Chul-Kee Park, Michele Ceccarelli, Marc Sanson, A. Lasorella, A. Iavarone
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="
 Head and neck squamous cell carcinoma (HNSCC) is a globally prevalent malignancy. Standard treatments typically include surgical resection, radiation therapy, and chemotherapy. However, for patients with recurrent or distantly metastatic tumors where surgery or radiation is no longer feasible, the primary treatment strategy shifts to combining targeted therapies with chemotherapy or immune checkpoint blockade (ICI) therapy. Despite these approaches, the overall prognosis remains poor. Therefore, identifying key changes in immune cell composition within the tumor microenvironment of malignant or metastatic tumors is crucial for improving the management and outcomes in HNSCC therapy.
 The aim of this study is to analyze changes in the immune microenvironment composition across different HNSCC specimens. We utilized tools including the GeoMx Digital Spatial Profiler (DSP) for imaging-based digitized gene expression analysis. We identified region of interesting (ROI) from distinct tumor regions, such as the tumor invasive front, tumor core, and paired lymph node metastases. Through the Cancer Transcriptome Atlas (CTA) analysis, we identified two distinct epithelial-mesenchymal transition (EMT) gene signatures, which were highly expressed in particularly aggressive tumor tissues. Additionally, within the myeloid activation signature, we observed that immune cell composition in primary tumor site was predominantly neutrophil-activated, whereas in lymph node metastases, macrophages were the dominant immune cells. To further validate these observations, we applied other spatial transcriptomics tools, including 10x Visium and 10x Xenium, to analyze specimens from six HNSCC patients. Transcriptomic trajectory analysis and ligand-receptor pair analysis revealed significant changes in the immune microenvironment of malignant HNSCC tissues.
 These results provide crucial insights into head and neck cancer treatment strategies tailored to different tumor microenvironment (TME) characteristics.


 Chih-Hung Chung, Muh-Hwa Yang. Mapping tumor metastasis and EMT-associated immune changes in HNSCC through spatial transcriptomic analysis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5863.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f43447092efa26d9874b71c88b8fd148bb746d9b" target='_blank'>
              Abstract 5863: Mapping tumor metastasis and EMT-associated immune changes in HNSCC through spatial transcriptomic analysis
              </a>
            </td>
          <td>
            Chih-Hung Chung, Muh-Hwa Yang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Cancer-associated fibroblasts (CAFs) are key components of the tumor microenvironment and play a critical role in tumor progression and therapy resistance. Recent advances in spatial transcriptomics (ST) enable precise analysis of gene expression within the spatial context of tissues, offering new avenues for target discovery. Using spatial and single-cell transcriptomic data, we identified a novel target protein, POR-CAF-001, predominantly expressed in CAFs within solid tumor tissues and minimally in normal tissues, making it a promising candidate for therapeutic intervention. Method Spatial and single-cell transcriptomics data from human tumor samples were analyzed using our proprietary target discovery program to identify CAF-specific targets. Two human IgG antibodies (POR-CAF-001-01 and POR-CAF-001-02) against the POR-CAF-001 were generated through phage display, ELISA screening and recombinant production in mammalian cells. Surface plasmon resonance (SPR) was employed to validate a binding affinity of antibodies against POR-CAF-001 protein. Also, flow cytometry and immunocytochemistry (ICC) were performed to evaluate the binding specificity of the antibodies. Antibody internalization was assessed via an imaging-based internalization assay using pH-sensitive fluorophore-labeled antibodies, and their internalization was monitored over time. POR-CAF-001-positive, -negative and knockout cell lines were used to confirm binding specificity and internalization. Results Differential expression and spatial specificity analyses identified POR-CAF-001 with high expression in various types of tumor tissues with minimal expression in normal tissues. SPR analysis demonstrated binding affinities (KD) of 1.15 nM and 4.1 nM for POR-CAF-001-01 and POR-CAF-001-02, respectively. Flow cytometry and ICC analyses confirmed robust binding of both antibodies to POR-CAF-001-positive cells, with negligible binding in POR-CAF-001-negative and the target knockout cells. Both antibodies showed substantial internalization in POR-CAF-001-positive cells during the internalization assay, while minimal internalization was observed in the negative cell line. Conclusion Through our target discovery program with ST data, we identified POR-CAF-001 as a novel CAF-specific target protein and developed two human IgG antibodies against it. These antibodies demonstrated high specificity and successful internalization in target-positive cells. These findings provide a solid foundation for further preclinical development, including antibody-drug conjugates (ADCs), radiopharmaceutical therapy (RPT) or other therapeutic modalities targeting POR-CAF-001. Our study underscores the potential of ST in identifying novel targets and advancing cancer therapy.



 Jin Yeong Choi, Shengjun Li, MinKyu Kim, Yeonjae Jung, Jae Eun Lee, Daeseung Lee, Hongyoon Choi, Hyung-Jun Im. Development of human antibodies targeting a novel cancer-associated fibroblast protein identified from spatial transcriptomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5456.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d6445efc526771002f7bfac5c1cb947546e45a" target='_blank'>
              Abstract 5456: Development of human antibodies targeting a novel cancer-associated fibroblast protein identified from spatial transcriptomics
              </a>
            </td>
          <td>
            Jinyeoung Choi, Shengjun Li, Minkyu Kim, Yeonjae Jung, Jae Eun Lee, D. Lee, Hongyoon Choi, Hyung-Jun Im
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 This study tests the hypothesis that quantitative profiling of cell type composition and tissue architecture in isocitrate dehydrogenase (IDH)-mutant gliomas can provide valuable insights into tumor aggressiveness, treatment response, and survival. Towards this goal, we have established a novel method to analyze cell composition and tissue architectural features in IDH mutant gliomas based on multiplex immunohistochemistry staining and automated image analysis.Two independent clinically well-annotated TMAs, derived from approximately 200 patients with IDH1-mutant tumors, are being analyzed. A multiplex fluorescence antibody panel was designed to include markers for astrocytes (GFAP+/IDH1-R132H-), glioma cells (IDH1-R132H+), oligodendrocytes (IDH1-R132H-/SOX10+), leukocytes (CD45+), tumor-associated microglia and macrophages (TAMMs) (IBA1+), endothelial cells (CD34+) and perivascular cells (PDGFRß+). QuPath software was employed for image analysis. Statistical analyses were performed using Python.For cell classification we firstly created single-core “ground truth data” from 20 cores by manual/visual training of core-specific cell classifiers. Second, we created a “multiple-core-trained” classifier by training on 10 of the ground truth cores and testing on the remaining 10 cores. Comparison between ground truth data and “multiple-core-trained” data indicated overall cell classification accuracy of 0.8.A pilot-study of 20 tumors demonstrated that the most abundant cell types were glioma cells, while leukocytes and perivascular cells were the least abundant. The abundance of glioma and perivascular cells showed good consistency between cores from the same patient (corr = 0.8 for both). In contrast, the abundance of oligodendrocytes and leukocyte cells was less consistent (corr = 0.4 and 0.3, respectively).Analyses of spatial pairing of cell types demonstrated strongest positive spatial enrichment between endothelial cells and pericytes and between endothelial cells and TAMMs. In contrast, glioma cells and endothelial cells showed negative spatial enrichment. Notably, inside-case concordance for these “pair-metrics” was high (0.8-0.9).Analyses aiming to identify co-regulated “pair-metrics” revealed strong positive correlations between the three pair-metrics TAMMs/perivascular cells, TAMMs/endothelial cells and endothelial cells/perivascular cells indicating inter-case variability of multi-cellular vascular niches."Cell abundance metrics" and "cell pair metrics" are currently being explored for their associations with clinical characteristics and their potential to enhance prognostic value relative to 1p19q status and WHO grades.


 Elisabet Rodríguez Tomàs, Lina Leiss, Rasul Tabiev, Fredrik Nysjö, Anna Klemm, Carolina Wählby, Thomas Olsson Bontell, Anna Dénes, Asgeir S. Jakola, Anja Smits, Arne Östman. Defining architectural tissue features and cell composition of IDH-mutant gliomas [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6469.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e56cfb98ead022484cef458f223d5c6fb21104d" target='_blank'>
              Abstract 6469: Defining architectural tissue features and cell composition of IDH-mutant gliomas
              </a>
            </td>
          <td>
            Elisabet Rodríguez Tomàs, L. Leiss, Rasul M. Tabiev, Fredrik Nysjö, Anna Klemm, C. Wählby, T. O. Bontell, A. Dénes, A. Jakola, Anja Smits, Arne Östman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Intratumor heterogeneity and evolution underlie treatment failures but are poorly understood due to a limited number of tumor biopsies. In this study, we examine IDH mutant gliomas, utilizing a 3D whole tumor sampling approach and exome sequencing (32 patients, 345 samples), along with RNAseq (12 of the 32 patients, 88 samples). We investigate and redefine intratumoral heterogeneity and evolution of insertion/deletion (indel) and single nucleotide variant (SNV) in driver genes on a whole tumor scale.
 In the tumor suppressor capicua transcriptional repressor (CIC), a canonical driver in IDH mutant 1p/19q co-deleted (IDHmut-Codel) tumors, our exome and RNAseq data analysis highlights a wide spectrum of heterogeneity and inactivation by SNVs and indels in independent subclonal expansions. For instance, in nine samples from one patient’s recurrent tumor, five showed distinct CIC SNVs and eight exhibited indels. Notably, all samples showed one or more CIC SNVs or indels. Among five samples harboring SNVs, three had 2 to 3 co-occurring SNVs, and among the 8 samples with indels, three had 2 and one had 5 indels. None had the same combinations of indels and SNVs, indicating extreme heterogeneity and subclonality. Additionally, TA-cloning of a long-range PCR from one sample suggested that each individual SNV and indel were in different tumor cells, raising the possibility of early diversification of subclonal expansions driven by independent CIC alterations. Similar patterns were observed in IDH-mutant and 1p/19q non-codeleted tumors involving tumor suppressor ATRX.
 These findings demonstrate that lower grade IDH-mutant gliomas (LGGs) exhibit significant intratumoral heterogeneity, a trait typically associated with glioblastoma. To determine if CIC alterations were inactivating or simply passengers, we analyzed gene expression in those spatially mapped samples. Within the ETS-domain transcription factor family, ETS1, ETV1, ETV4, and ETV5 were upregulated in CIC mutant samples versus wild type samples. We also identified novel candidate genes potentially regulated by CIC, suggesting possible mechanisms contributing to tumorigenesis and new treatment targets for IDHmut-Codel tumors.
 This study elucidates the substantial intratumor heterogeneity and evolutionary dynamics in the drivers of LGGs on a whole tumor scale, highlighting their important role in tumorigenesis. By utilizing spatially mapped samples within the same tumors, our research reveals convergence on CIC loss of function through many mutually exclusive SNVs and indels in IDHmut-Codel tumors. Our findings demonstrate the critical nature of CIC inactivation across entire tumors, emphasizing the importance of indels and SNVs in tumor progression. This research also identifies candidate genes upregulated by CIC inactivation as possible treatment targets for IDHmut-Codel tumors.


 Lee H. Chen, Nicholas O. Stevers, Chibo Hong, Ivan V. Smirnov, Nadia R. Grishanina, Joshua S. Wu, Anny Shai, Susan M. Chang, Mitchel S. Berger, Shawn L. Hervey-Jumper, Joanna J. Phillips, Joseph F. Costello. Three-dimensional whole tumor genomic analysis redefine intratumor heterogeneity of gliomas, with implications for early tumorigenesis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3894.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a70d8ffb57c8661082ae6d08c258b2d7875bf6f" target='_blank'>
              Abstract 3894: Three-dimensional whole tumor genomic analysis redefine intratumor heterogeneity of gliomas, with implications for early tumorigenesis
              </a>
            </td>
          <td>
            Lee H. Chen, Nicholas O Stevers, Chibo Hong, Ivan V. Smirnov, Nadia R. Grishanina, Joshua S. Wu, Anny Shai, Susan Marina Chang, Mitchell S. Berger, Shawn L. Hervey-Jumper, Joanna J Phillips, Joseph F Costello
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Racial disparities in triple-negative breast cancer (TNBC) outcomes have been reported. However, the biological mechanisms underlying these disparities remain unclear. We integrated imaging mass cytometry and spatial transcriptomics, to characterize the tumor microenvironment (TME) of African American (AA) and European American (EA) patients with TNBC. The TME in AA patients was characterized by interactions between endothelial cells, macrophages, and mesenchymal-like cells, which were associated with poor patient survival. In contrast, the EA TNBC-associated niche is enriched in T-cells and neutrophils suggestive of an exhaustion and suppression of otherwise active T cell responses. Ligand-receptor and pathway analyses of race-associated niches found AA TNBC to be “immune cold” and hence immunotherapy resistant tumors, and EA TNBC as ‘inflamed’ tumors that evolved a distinctive immunosuppressive mechanism. Our study revealed the presence of racially distinct tumor-promoting and immunosuppressive microenvironments in AA and EA patients with TNBC, which may explain the poor clinical outcomes.
 Statement of Significance We have uncovered distinct tumor-promoting and immunosuppressive microenvironments that may contribute to the observed disparities in patient outcomes. Our findings lay the groundwork for developing race-specific therapeutic interventions, potentially improving outcomes for patients with TNBC. Given that chemotherapy and anti-PD1 therapy are currently available treatments for TNBC, our finding of race-specific spatial multicellular niche-associated cell-cell interactions suggest that these treatments may have racially distinct response profiles among TNBC patients. Hence, our novel insights on molecular and cellular interactions in TNBC stratified by race has the potential to inform personalized treatment strategies.


 Qian Zhu, Akhila Balasubramanian, Jaya Ruth Asirvatham, Badrajee Piyarathna, Jaspreet Kaur, Nada Mohamed, Ling Wu, Megha Chatterjee, Stacy Wang, Niloufar Pourfarrokh, Uttam Rasaily, Yitian Xu, Junjun Zheng, Deborah Jebakumar, Arundathi Rao, Shu-hsia Chen, Yi Li, Eric Chang, Xiaoxian Li, Ritu Aneja, Xiang H.-F. Zhang, Arun Sreekumar. Integrative spatial omics reveals distinct tumor-promoting multicellular niches and immunosuppressive mechanisms in African American and European American patients with TNBC [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5320.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e120967c6039c9b8a8507c569dbb9140d2239a00" target='_blank'>
              Abstract 5320: Integrative spatial omics reveals distinct tumor-promoting multicellular niches and immunosuppressive mechanisms in African American and European American patients with TNBC
              </a>
            </td>
          <td>
            Qian Zhu, Akhila Balasubramanian, Jaya Ruth Asirvatham, Badrajee Piyarathna, Jaspreet Kaur, Nada Mohamed, Ling Wu, Megha Chatterjee, Stacy Wang, Niloufar Pourfarrokh, Uttam Rasaily, Yitian Xu, Junjun Zheng, D. Jebakumar, Arundathi Rao, Shu-Hsia Chen, Yi Li, Eric Chang, Xiaoxian Li, R. Aneja, X. Zhang, Arun Sreekumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02046d3dbdf78b8dbf1ae4df369d0858d2ab96f2" target='_blank'>
              Cellular hierarchies of embryonal tumors with multilayered rosettes are shaped by oncogenic microRNAs and receptor-ligand interactions.
              </a>
            </td>
          <td>
            Alexander Beck, Lisa Gabler-Pamer, G. Alencastro Veiga Cruzeiro, S. Lambo, Bernhard Englinger, McKenzie L. Shaw, Olivia A Hack, Ilon Liu, Rebecca D. Haase, Carlos A O de Biagi, A. Baumgartner, Andrezza Do Nascimento Silva, M. Klenner, Pia Freidel, Jochen Herms, Louisa von Baumgarten, Joerg-Christian Tonn, Niklas Thon, K. Bruckner, S. Madlener, L. Mayr, D. Senfter, A. Peyrl, I. Slavc, D. Lötsch, C. Dorfer, Rene Geyregger, N. Amberg, C. Haberler, Norman L Mack, Benjamin Schwalm, Stefan M. Pfister, Andrey Korshunov, Lissa C Baird, Edward Yang, Susan N Chi, S. Alexandrescu, J. Gojo, Marcel Kool, Volker Hovestadt, Mariella G. Filbin
          </td>
          <td>2025-05-26</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Immunotherapy and targeted therapy have altered the standard of care for cancer patients, improving survival rates and cancer control, even in intransigent metastasis. Yet, only 20-40% of patients respond to immunotherapy, and up to 50% experience severe adverse events, limiting therapy application. Notably, cellular heterogeneity and diversity in the makeup of the tumor immune microenvironment (TiME) present extraordinary challenges in finding predictive biomarkers correlating to therapeutic response. Existing profiling methods, even with highly resolved, cutting-edge transcriptomic and proteomic mapping methods remain costly, time-intensive, and do not faithfully capture structural or post-translational modifications actively impacting tumor behavior. Here, we present a high-fidelity, single-cell omics approach based on Raman spectroscopy and machine learning to provide rapid, cellular profiling that enables non-destructive therapeutic response characterization.
 Using Raman, we first demonstrate single-cell differentiation and functional state phenotyping of various melanoma and immune cells found in the TiME. Our ML models yield a classification accuracy of >94% across all cell types and parent and derivative cancer lines. We then extend our characterization to track temporal changes across multicellular systems; by investigating cell growth in normal and apoptotic-inducing environments, we show how melanoma-immune cell interactions preserve certain spectral features while introducing new features via cell fusioning under selective pressures. By stimulating cells with small molecule therapeutics serving as growth and inhibiting factors, we demonstrate the potential to monitor melanoma cell plasticity and macrophage polarization changes. Finally, we illustrate the potency of Raman phenotyping by examining the collective drug response in clinically relevant patient melanoma samples and extract pertinent biomarkers associated with small molecule and immunotherapy treatment outcomes. In combination with neural networks, our approach correctly predicts therapy response with >75% accuracy for unseen melanoma patient samples, demonstrating Raman as a powerful rapid, label-free omics approach with groundbreaking potential to advance clinical biomarker efforts and impact the precision treatment decisions of cancer patients.


 Kai Chang, Mamatha Serasanambati, Baba Ogunlade, Ariel Stiber, Fareeha Safir, Hamish Carr Delgado. Single-cell Raman-based omics of the melanoma tumor-immune microenvironment for precision therapy [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5770.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08487b7db3c7e22e3795c18f3a3ea30e842f09b5" target='_blank'>
              Abstract 5770: Single-cell Raman-based omics of the melanoma tumor-immune microenvironment for precision therapy
              </a>
            </td>
          <td>
            Kai Chang, Mamatha Serasanambati, Baba Ogunlade, Ariel Stiber, Fareeha Safir, H. Delgado, Amanda R. Kirane, Jennifer A. Dionne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Little is known about how interactions between the uveal melanoma (UM) and host cells in the tumor microenvironment (TME) drive the progression of liver metastases. Single cell RNA-seq analysis of UM samples identified marked differences in cell-cell communication between primary and metastatic UM characterized by increased signaling strength and numbers of interactions emanating from the hepatic stellate cells (HSCs) and endothelial cells in the metastatic samples. In co-culture, HSCs modulated the transcriptional heterogeneity of the UM cells, enriching for cell states that expressed genes implicated in cell survival, metabolic reprogramming and the angiogenic response. Co-culture of HSCs and UM cells also led to increased expression of growth factors and receptors implicated in angiogenesis including IL-8, VCAM1, VEGF and ICAM1. One candidate driver of HSC reprogramming was the TGF-beta family member GDF15. Increased expression of GDF15 was associated with a Class 2 metastatic UM phenotype in clinical UM samples and UM cell lines, with silencing of BAP1 in UM cells increasing both GDF15 expression and H3K27ac marks at the GDF15 promoter. Treatment of HSCs with GDF15 led to increased expression of multiple ECM proteins, inflammatory cytokines and angiogenic factors, including IL- 8. Both exogenous GDF15, IL-8 and conditioned media from UM-HSC co-cultures increased endothelial cell network formation in vitro, an effect that could be blocked by anti-GDF15 antibodies. In an eye->liver metastasis model of UM, silencing of GDF15 did not prevent the seeding of UM cells to the liver, but did inhibit the outgrowth of metastatic lesions. Although the GDF15-silenced UM cells were proliferative they did not recruit endothelial cells, leading to a significant reduction in metastatic burden. In sum, we provide the first evidence that UM liver metastasis development is dependent upon GDF15-mediated remodeling of the liver microenvironment leading to an angiogenic response that supports tumor growth.


 Sathya Neelature Sriramareddy, Navya Siddarajappa, Thanh Nguyen, Jeffim Kunetsov, Stefan Kurtenbach, James Jarad Dollar, Jonathan D. Licht, Ann Y. Chen, William J. Harbour, Keiran S. Smalley. GDF15 reprograms the microenvironment to drive the development of uveal melanoma liver metastases [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2578.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc725ddd29c73799b619a4c59fece243a3bbe0ad" target='_blank'>
              Abstract 2578: GDF15 reprograms the microenvironment to drive the development of uveal melanoma liver metastases
              </a>
            </td>
          <td>
            S. Sriramareddy, Navya Siddarajappa, Thanh Nguyen, Jeffim Kunetsov, Stefan Kurtenbach, J. Dollar, Jonathan D. Licht, Ann Y. Chen, William J. Harbour, K. Smalley
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 Melanoma is the most serious form of skin cancer, developed by the malignant evolution of melanocytes. Malignant melanoma incidence is increasing faster than most other cancers. While stage zero melanoma is highly treatable, survivability dramatically decreases in its advanced stages. Melanoma has shown to be one of the most heterogeneous cancers from RNA and exome analyses by The Cancer Genome Atlas and other groups. A better understanding of the key genomic and epigenomic events that characterize the diverse subclonal populations in melanoma may reveal key insights into what drives its progression and therapeutic resistance.
 In this study, we leveraged Nanopore long-read sequencing to study the evolution of the mouse B2905 melanoma cell line. Twenty-four distinct clonal sublines were derived in vitro from single cells of the cell line, and the genetically homogeneous population from each subline was sequenced using PromethION R10 flow cells. Enabled by long reads to perform haplotype phasing and accurate structural variation detection, our goal is to integrate small and structural variants to better our understanding of melanoma evolution, and build upon prior analyses of short-read sequenced sublines.
 We employed multiple SNV calling approaches, including DeepVariant and Clair, in order to provide highly accurate variants for phylogeny reconstruction using Trisicell. We performed structural variant calling with our cancer somatic structural variant (SV) caller Severus as well as copy-number alteration (CNA) analysis with our method Wakhan. Lastly, we placed SNVs, SVs, and CNAs on our reconstructed phylogeny to examine the progression of different types of variants during subline evolution.
 We identified approximately 560k unique SNVs and around 2, 400 unique SVs. The majority of SNVs (19%) are either clonal or private (73%); however, a meaningful fraction of subclonal variants were available for phylogenetic tree reconstruction. SVs are distributed across the phylogenetic tree branches similarly to SNVs. We identified loss of heterozygosity (LOH) events throughout the subline evolution as well as subclonal CNAs resulting from chromosomal translocations. We find clonal and subclonal evidence of densely clustered SNVs and SV, resembling kataegis; however, our analysis of mutational signatures did not reveal APOBEC-mediated mutations. By analyzing mutational signatures within individual branches of the phylogenetic tree, we observed relative timing of different mutational processes, such as early clonal signatures of UV damage.
 By incorporating structural variations, copy number changes, and small variant data in the phylogenetic reconstruction, our analysis offers a better characterization of the genetic landscape of subclonal evolution in melanoma.


 Anton Goretsky, Yuelin Liu, Ayse Keskus, Tanveer Ahmad, Salem Malikic, Glenn Merlino, Chi-Ping Day, Erin Molloy, S. Cenk Sahinalp, Mikhail Kolmogorov. Nanopore sequencing of single-cell derived sublines provides insights into melanoma heterogeneity and evolution [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7497.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8f2b1ddae01e13edf72bbcb5e1a3343151981e" target='_blank'>
              Abstract 7497: Nanopore sequencing of single-cell derived sublines provides insights into melanoma heterogeneity and evolution
              </a>
            </td>
          <td>
            Anton Goretsky, Yuelin Liu, A. Keskus, Tanveer Ahmad, S. Malikić, Glenn Merlino, Chi-Ping Day, Erin K. Molloy, S. C. Sahinalp, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Coupling single-cell transcriptomics with tumor-initiating cell enrichment identified IFN response gene expression not previously reported in bulk RNA-sequencing–derived signatures and proposed IFN/STAT1 signaling as a candidate therapeutic target in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49032c731c7f43ad3396d95549c8d805e477fd3" target='_blank'>
              Lineage Tracing and Single-Cell RNA Sequencing Reveal a Common Transcriptional State in Breast Cancer Tumor–Initiating Cells Characterized by IFN/STAT1 Activity
              </a>
            </td>
          <td>
            Eric P Souto, Ping Gong, John D. Landua, R. Srinivasan, Abhinaya Ganesan, L. Dobrolecki, S. C. Purdy, Xingxin Pan, Michael Zeosky, Anna Chung, S. S. Yi, H. Ford, Michael T. Lewis, Connie Cepko, Dr. Kenneth L. Scott, Dr. Michael Davidson
          </td>
          <td>2025-02-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Head and neck cancers are the 7th most common cancer type globally accounting for over 900, 000 cases annually. With increasing numbers of systemic and targeted therapies, including immune checkpoint inhibitors (ICI), it has become increasingly important to understand the complex tumour microenvironment (TME) milieu dictating clinical responses to therapy. In this study, we profiled 40 whole slide, head and neck cancer tissues from patients prior to ICI therapy and profiled the TME for spatial interactions using the NaveniPlex in situ proximity ligation assay (isPLA, Navinci ). Using the Leica BOND RX, formalin-fixed, paraffin-embedded (FFPE) tissue sections were prepared by baking, deparaffinization, and epitope unmasking. Staining was then carried out using the NaveniPlex isPLA technology on the Leica BOND RX instrument. To assess interactions between PD1/PD-L1, CD8/MHCI, and CD3+ T cells, three pairs of Navenibodies were employed simultaneously. Briefly, the NaveniPlex isPLA protocol involves tissue blocking, followed by incubation with primary Navenibodies, ligation, and rolling circle amplification (RCA). Each Navenibody pair generates a distinct RCA product. The RCA products for PD1/PD-L1, CD8/MHCI, and CD3+ T cells were visualized using oligonucleotides conjugated to Atto647N, Atto550, and AlexaFluor488, respectively.Our spatial interaction analysis for PD1/PDL1, CD8/MHCI and CD3+ T-cells showed distinct TME interactions associated with responders and non-responders to ICI therapy. Interactions mapped with cellular phenotypes revealed the cellular neighbourhoods and communities at which these interactions may be useful prognostically. Taken together, this study provides a methodological framework to profile pre-ICI treatment tissues to identify spatial interactions associated with clinical endpoints.


 Vahid Yaghoubi Naei, Sara Bodbin, Habib Sadeghirad, Rafael Tubelleza, James Monkman, Agata Wicher, Subham Basu, Arutha Kulasinghe. Multi-plexed spatial interaction analysis of the tumor microenvironment in head and neck cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5254.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e187b60797f80ff80763a75f6d8cd55229743e9" target='_blank'>
              Abstract 5254: Multi-plexed spatial interaction analysis of the tumor microenvironment in head and neck cancers
              </a>
            </td>
          <td>
            V. Y. Naei, Sara Bodbin, H. Sadeghirad, Rafael Tubelleza, J. Monkman, A. Wicher, Subham Basu, A. Kulasinghe
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) is among the most frequently occurring types of cancer, and its metastasis is a major contributor to its elevated mortality. Before the primary tumor metastasizes to secondary or distant organs, it remodels the microenvironment of these sites, creating a pre-metastatic niche (PMN) conducive to the colonization and growth of metastatic tumors. RCC releases a variety of biomolecules that induce angiogenesis, alter vascular permeability, modulate immune cells to create an immunosuppressive microenvironment, affect extracellular matrix remodeling and metabolic reprogramming, and determine the organotropism of metastasis through different signaling pathways. This review summarizes the principal processes and mechanisms underlying the formation of the premetastatic niche in RCC. Additionally, we emphasize the significance and potential of targeting PMNs for the prevention and treatment of tumor metastasis in future therapeutic approaches. Finally, we summarized the currently potential targeted strategies for detecting and treating PMN in RCC and provide a roadmap for further in-depth studies on PMN in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42609e8acc6a516bc77c99c7fb3a7dbaa2238142" target='_blank'>
              Exploring molecular and cellular mechanisms of Pre-Metastatic niche in renal cell carcinoma
              </a>
            </td>
          <td>
            Xiao Zhou, Ruirui Li, Maode Lai, Chong Lai
          </td>
          <td>2025-04-22</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Over 50% of melanomas are driven by BRAFV600 mutations. Clinically, targeted inhibitors of mutant BRAF and MEK are the primary treatment for more than 50% of melanoma patients that are refractory to standard-of-care immunotherapy. While targeted therapies are initially clinically effective, a subset of residual cancer cells persist and inevitably drive tumor recurrence and drug resistance. While some cells persist due to pre-existing mechanisms (e.g. genetic mutations), others can non-genetically adapt to tolerate drug by rewiring their internal signaling cascades. Importantly, a key feature of these non-genetically adapted persister cells is that their drug-tolerant phenotype is acquired in response to drug pressure, making them extremely challenging to identify and characterize.
 Using cutting-edge long-term time-lapse microscopy and an automated single-cell tracking pipeline, we studied the behaviors of persister cells in drug-treated BRAFV600E melanoma populations. Notably, we found significant heterogeneity within persister populations. While all persister cells initially enter a state of drug-induced quiescence, a subset of these cells—termed “escapees”—can escape drug action and resume proliferation. By computationally reconstructing single-cell lineages from long-term time-lapse imaging experiments, we monitored key signaling pathways before, during, and after escape from drug. Using this approach, we identified a significant increase in mTorC1 signaling activity in persisters that would go on to escape drug. Additionally, mTorC1 signaling caused increases in protein translation and cell growth rates during the quiescent periods of these cells.
 The abundance of Cyclin D1, a key driver of proliferation, is extremely sensitive to cellular translation rates due to its short half-life. In order to determine if the observed increase in mTorC1-mediated growth and translation is driving the upregulation of Cyclin D1 levels past a threshold necessary for escape, we used CRISPR to tag Cyclin D1 at the endogenous locus. Importantly, we found that escapees significantly upregulate Cyclin D1 levels in the hours prior to their escape, compared to non-escapees. These findings have important clinical implications because they suggest that Cyclin D1 protein levels are a key bottleneck for cell-cycle re-entry under MAPK inhibition. Additionally, these findings implicate Cyclin D1 as one of the first markers that can distinguish future escapees from non-escapees in the quiescence period before these distinct cell-cycle fates emerge.


 Varuna Nangia, Humza Ashraf, Nasreen Marikar, Victor Passanisi, Christopher Ryland Ill, Timothy Hoffman, Sabrina Leigh Spencer. Non-genetic drug adaptation to MAPK pathway inhibitors in melanoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6418.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13bcea3579836571808ac96de5e2e45ad09f6441" target='_blank'>
              Abstract 6418: Non-genetic drug adaptation to MAPK pathway inhibitors in melanoma
              </a>
            </td>
          <td>
            Varuna Nangia, Humza M. Ashraf, Nasreen Marikar, Victor J. Passanisi, Christopher Ryland Ill, Timothy E. Hoffman, Sabrina L. Spencer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Within the stroma of pancreatic ductal adenocarcinoma (PDAC), mesenchymal cells differentiate into cancer-associated fibroblast (CAF) subtypes that differentially mediate disease progression. Defining the regulatory mechanism and diversity of CAF subtypes could identify potential therapeutic strategies to harness the tumor suppressive activities of CAFs. To address this, we utilized single-cell RNA sequencing to profile fibroblast activation protein-alpha (FAP) expressing mesenchymal cells in human PDAC. The mesenchymal subpopulations in PDAC reflected mesenchymal cell heterogeneity found in the normal developing pancreas. In addition to characterizing inflammatory CAF (iCAF) and myofibroblastic CAF (myCAF) subpopulations in detail, the analysis uncovered a previously undescribed interferon-response CAF (ifCAF) subtype. Tumor-derived signals induced specific CAF subtypes from pancreatic stellate cells (PSCs) in an organoid-based co-culture model, and time-course experiments revealed regulatory mechanisms that govern subtype formation. STING agonists promoted an ifCAF phenotype in vivo and in vitro. Importantly, induction of an ifCAF phenotype suppressed tumor cell invasiveness and induced an anti-tumor phenotype in tumor-associated neutrophils. Together, this study resolves FAP+ stromal cell heterogeneity in PDAC and identifies an ifCAF subtype that can be induced to suppress pro-tumorigenic features of PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5743bfda49f4ffbda353732b6aae266b1159c9ec" target='_blank'>
              Dissecting FAP+ Cell Diversity in Pancreatic Cancer Uncovers an Interferon-Response Subtype of Cancer-Associated Fibroblasts with Tumor-Restraining Properties.
              </a>
            </td>
          <td>
            Joshua Cumming, Parniyan Maneshi, M. Dongre, Tala Alsaed, Mohammad Javad Dehghan-Nayeri, Agnes Ling, Kristian Pietras, Cedric Patthey, D. Öhlund
          </td>
          <td>2025-04-11</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="


 Organoids are powerful tumor models for cancer research surpassing 2D-cultures in maintaining tumor heterogeneity and genetic fidelity. However, working with organoids encounters challenges in imaging, drug-screening, and in-vivo efficacy studies, due to their structural complexity. Additionally, the absence of other relevant cell types including cancer-associated-fibroblasts (CAFs) and immune cells, limits their ability to fully recapitulate the tumor-microenvironment (TME). To overcome these limitations, we developed GFP- or RFP-expressing KRAS-mutant-PDAC-organoids for efficient in-vitro and in-vivo drug-screening in mono- and co-culture systems.



 We developed KRAS-mutant-PDAC-organoids that overexpress GFP or RFP using lentiviral-transduction. The growth and gene expression profile of these engineered organoid lines were compared to the parental line using Incucyte, DRUG-seq, and immunoblotting. These modifications enabled in-vitro mechanistic studies and drug sensitivity assays via confocal-microscopy and cell viability assays, as well as in-vivo quantification of tumor growth, drug-response, and gene expression.



 Antibiotic-sensitivity assays showed that 10 μM of antibiotic induces complete cell death within 5 days. Based on this, we stably overexpressed GFP or RFP in organoid lines, with strong GFP and RFP signals confirmed through microscopy. Optimizing culture conditions revealed that the addition of growth-factors (EGF and FGF) is crucial in initial phases of growth, though they can be removed later for drug-screening without impacting the organoids growth curve. This is important for accurate drug-screening, particularly for growth-factor pathway inhibitors. Further, we compared the growth rates and protein expression of the engineered organoids to their parental lines and observed no substantial differences. In high-throughput drug screening, we tested the effects of chemotherapy drugs (Gemcitabine and SN38) on organoids at varying seeding densities. While lentiviral transduction did not significantly affect the IC50 values, increasing the number of cells significantly raised the IC50 values (e.g., IC50 for Gemcitabine increased from 6-70 nM with 1k cells to 32-517 nM with 5k cells).



 We successfully engineered KRAS-mutant-PDAC-organoids, which will serve as invaluable tools for in-vitro and in vivo efficacy experiments. Engineered organoids will simplify detection in co-culture systems and eliminate the need for additional staining. Future studies will focus on evaluating the growth and gene expression of engineered lines in-vivo compared to their parental lines. Additionally, these novel co-culture systems are being used to investigate the effects of novel KRAS inhibitors on organoid growth in-vitro and in-vivo.



 Alireza Savadipour, Krishna Tummala, Eunsil Park, Vania Vidimar, Mike Veling, Mark Buchanan, Eric Muise, Gustav Hebert, Brian Long, Nicolas Solban. Engineering KRAS-mutant-organoids from pancreatic ductal adenocarcinoma for advanced drug screening applications in-vitro and in-vivo [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 22.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29fa8e4c695e79e6a9bb2f4892060fece00752d1" target='_blank'>
              Abstract 22: Engineering KRAS-mutant-organoids from pancreatic ductal adenocarcinoma for advanced drug screening applications in-vitro and in-vivo
              </a>
            </td>
          <td>
            Alireza Savadipour, Krishna S. Tummala, Eunsil Park, Vania Vidimar, Mike Veling, Mark Buchanan, Eric Muise, Gustave Hebert, Brian Long, Nicolas Solban
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/743a748682182612c504b30793e3105a56193c50" target='_blank'>
              TIGIT expression dictates the immunosuppressive reprogramming of myeloid cells in glioblastoma
              </a>
            </td>
          <td>
            Mohammad Asad, Julio Inocencio, Stefan Mitrasinovic, Minori Aoki, C. Crisman, Patrick Lasala, Emad Eskandar, Chandan Guha, Xingxing Zang, I. Parney, B. Himes
          </td>
          <td>2025-05-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a significant health burden due to tumor heterogeneity and treatment resistance, emphasizing the need for improved biological understanding and tailored therapies. In this study, we enrolled 31 HNSCC patients for the establishment of patient-derived tumor organoids (PDOs), which faithfully maintained the genomic features and histopathological traits of the primary tumors. Long-term culture preserved key characteristics, affirming PDOs as robust representative models. PDOs demonstrated predictive capability for cisplatin treatment responses, with ex vivo drug sensitivity correlating with patient outcomes. Bulk and single-cell RNA sequencing unveiled molecular subtypes and intratumor heterogeneity (ITH) in PDOs, paralleling patient tumors. Notably, a hybrid epithelial-mesenchymal transition (hEMT)-like ITH program was associated with cisplatin resistance and poor patient survival. Functional analyses identified amphiregulin (AREG) as a potential regulator of the hybrid epithelial/mesenchymal state. Moreover, AREG contributed to cisplatin resistance via EGFR pathway activation, corroborated by clinical samples. In summary, HNSCC PDOs serve as reliable and versatile models, offer predictive insights into ITH programs and treatment responses, and uncover potential therapeutic targets for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87978b2579ac3a33bf3369fd95388496b8a42e2" target='_blank'>
              Genomic and Single-Cell Analyses Characterize Patient-Derived Tumor Organoids to Enable Personalized Therapy for Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Jung Hyun Um, Yueyuan Zheng, Qiong Mao, Chehyun Nam, Hua Zhao, Yoon Woo Koh, Su-Jin Shin, Y. Park, De-Chen Lin
          </td>
          <td>2025-04-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Classically, cancer initiation is attributed to a sequential acquisition of discrete genetic events such as driver mutations. However, homeostatically-stable tissues, including skin, harbor numerous somatic mutations spread across the tissue’s clones. Yet, most of these clones never give rise to cancer. Using the model of UV-driven cuSCC, we tested our hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of goliaths as evidenced by the presence of cell density hotspots and higher mutational burden (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level drivers via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25, 085 clades from 14 mice over 3 months, and 14, 525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little across UV exposure, our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3) observed essentially only within UV-exposed skin and occupying the extreme tail of the distribution of clade sizes. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes, some with areas of unusually high local densities (“micro-lumps”). Targeted DNA sequencing of goliath revealed Kmt2a, Kmt2d, Prex2, and SCN11A as the most mutated genes. Giant clades do not typically possess common driver mutations of SCC, e.g. TP53 and NOTCH, and display two extremes of mutational profiles, with some goliaths having dominant clones (VAF >0.75), and others having no discernable clonal structure (VAF <0.025), demonstrating that strong mutational drivers are not a prerequisite for developing goliaths. scRNAseq analysis of both bulk skin and sorted clades revealed upregulation of the fraction of cycling basal-like cells, and downregulation of Alox12e-expressing granular cells, along with increased proportions of plasmacytoid dendritic cells (pDC) and CD4+ regulatory T-cells (Tregs) in UV-exposed samples, reflecting early epidermal de-differentiation and immunosuppression. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions emerged between months 6 and 7, only in UV-exposed skin. Based on our adaptation of the Drake equation, we conclude that the probability of observing macroscopic lesions arising in 2 of the 21 randomly sampled goliaths we followed for 3 months to be less than 10-7. Taken together, our results support the interplay between cell autonomous and non-cell autonomous compartments during cancer initiation, the earliest phase of which presage the acquisition of driver mutations. Our data constitute an unprecedented look at clonal dynamics in UV exposed skin defining the transition from normal tissue to tumor.


 Stanislav S. Avdieiev, Leticia Tordesillas, Karol Prieto, Omar Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, Luiza Silva Simoes, Y. Ann Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai. Eco-evolutionary profiling of the UV-driven clonal dynamics and tumor initiation [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2672.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18fc5411928817db2e4ae82f8b06dabe1ec683eb" target='_blank'>
              Abstract 2672: Eco-evolutionary profiling of the UV-driven clonal dynamics and tumor initiation
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Although immunotherapy-based therapeutics such as checkpoint inhibitors (CPI) have revolutionized cancer therapy, a lot needs to be done in the field. In preclinical settings, ex vivo models still face significant challenges in translating the results into clinical responses. Traditional approaches, including cell-lines, are limited by their lack of genetic heterogeneity. Similarly, 2D systems fail to capture spatial complexity, significantly reducing their translational relevance. Clinical data on CPI are not available for commercially available cell lines since they were developed before CPI treatments were approved. These limitations hinder the accurate modeling of CPI efficacy and the prediction of therapeutic responses. To address these issues, Champions Oncology leverages a tumor bank of 1500 patient-derived xenograft (PDX) models across solid tumors and hematological malignancies offering 3D co-culture assays specifically tailored for different mechanisms of action. To provide a clinical translational platform that retains human tumor heterogeneity, we developed Champions TumorGraft3D (CTG3D) low-passage 3D PDX-organoid models. The use of proprietary, indication-specific media enabled CTG3D to preserve the genetic, proteomic, and phenotypic diversity of the original tumors. Additionally, their 3D architecture accurately models the cell-cell interactions that are crucial for mimicking immune cell engagement. The in-depth clinical annotation of our bank, which includes detailed patients’ clinical history, enables this system to provide predictive insights into the CPI efficacy. In this project, we assessed the translational value of our CTG3D co-culture system, based on its ability to recapitulate the clinical response in patients. We selected models derived from patients displaying resistance or sensitivity to CPI, by leveraging the data collected in Champions Oncology Tumor Bank. Then, we optimized the CTG3D-PBMCs co-culture settings including media and cell ratios. High throughput flow cytometry was used to interrogate the model. We profiled the co-culture with a custom-designed panel allowing for cancer cell killing and immune phenotyping readout in parallel. Beyond immune activation, we were able to determine the correlation of the data obtained from the co-culture with the clinical records of the original patients. In summary, we developed a cutting-edge ex vivo CTG3D PBMC co-culture platform allowing for dynamic evaluation of immune engagement and checkpoint inhibitor responses, therefore providing valuable translational datasets. By overcoming the limitations of traditional assays, CTG3D co-culture systems represent a transformative step forward in preclinical immunotherapy research and the development of personalized cancer treatments.


 Mara Gilardi, BanuPriya Sridharan, Veena Jagannathan, Brandon Walling, Stefano Cairo, Marianna Zipeto, Michael Ritchie. Predicting clinical checkpoint inhibitor responses with TumorGraft3D PBMC co-culture [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7281.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a46a367c224b7418d28861de504de205b9d8027a" target='_blank'>
              Abstract 7281: Predicting clinical checkpoint inhibitor responses with TumorGraft3D PBMC co-culture
              </a>
            </td>
          <td>
            Mara Gilardi, BanuPriya Sridharan, Veena Jagannathan, Brandon Walling, Stefano Cairo, Marianna Zipeto, M. Ritchie
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 High tumor heterogeneity and low mutational burden in cancers pose significant challenges for immunotherapy. To address this, we developed a novel in silico pipeline to characterize cancer-specific splicing events (neojunctions; NJs) expressed ubiquitously across ten cancer types. This approach successfully identified tumor-wide, public, alternatively splicing neoantigens (ASNs) that elicit CD8+ T-cell-mediated cytotoxicity across multiple cancers.



 Our SSNIP pipeline identified recurring public NJs expressed in TCGA RNA-seq data (positive sample rate (PSR) > 10%) but not in GTEx normal tissue RNA-seq data (PSR < 1%) across 10 cancer types. We utilized multi-site RNA sequencing data to characterize intratumorally conserved NJs. Our in-house glioma dataset included 56 patients with approximately 10 spatially distinct intratumoral biopsy sites per patient (n=535). In vitro sensitization (IVS) of CD8+ T-cells from healthy donors against high-confidence ASN candidates, followed by 10x VDJ scRNA-seq, identified ASN-specific TCR sequences. TCR-transduced donor-derived CD8+ T-cells were cultured against NJ-expressing glioma and melanoma cell lines to evaluate tumor-specific killing. CRISPRi KD of disease-specific splicing factors was performed to assess NJ modulation.



 Our pipeline identified 789 public NJs, with 32 NJs concurrently identified in transcriptomic and proteomic data and predicted to be presented by HLA-A*02:01 with high confidence. We captured TCR clonotypes reactive against NJs in RPL22 (n=7) and GNAS (n=1), the latter being highly intratumorally-conserved (detected in > 90% of spatially-mapped biopsies across 17/56 patients (26.78%)). Mutant GNAS-derived ASN-specific CD8+ T-cell clones were detectable from glioma patient PBMC by IVS. TCR-transduced T-cells demonstrated recognition and tumor-specific killing against endogenously processed and presented ASNs in multiple glioblastoma and melanoma cell lines. Furthermore, IDH1-mutant oligodendroglioma samples demonstrated significantly elevated expression of NJs over IDH1-mutant astrocytoma and IDH1wt subtypes. Differential gene expression identified decreased expression of splicing factors due to oligodendroglioma-specific co-deletion of Chromosomes 1p/19q. CRISPRi KD of these splicing factors (e.g. SF3A3, SNRPD2) in IDH1wt glioma cells resulted in significantly increased expression of corresponding NJs. Performing DESeq2 on the remaining cancer types revealed similar splicing gene set-associated differences.



 SSNIP identified novel public tumor-wide splice-derived neoantigen candidates and ASN-specific TCRs, offering a promising off-the-shelf immunotherapy approach for diverse cancer types. Characterization of intratumorally conserved neoantigens addresses the critical challenge of intratumoral heterogeneity in immunotherapy resistance.



 Darwin Kwok, Nicholas Stevers, Inaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee Chen, Jangham Jung, Kaori Okada, Senthilnath Lakshmanachetty, Marco Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel Wu, Emilio Ramos, Akane Yamamichi, Payal Watchmaker, Hirokazu Ogino, Atsuro Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Susan Chang, Joanna Phillips, Arun Wiita, Christopher Klebanoff, Joseph Costello, Hideho Okada. Tumor-wide RNA splicing aberrations generate immunogenic public neoantigens across cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr XX.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f549d282ce700f7e5436de47eb2b3446fdb14fc8" target='_blank'>
              Abstract 4250: Tumor-wide RNA splicing aberrations generate immunogenic public neoantigens across cancers
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, T. Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia R. Grishanina, James Woo, Aaron A. Diaz, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, H. Okada
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0c39aa857bbfd9543aa84644d12df58e1a71f0a" target='_blank'>
              Single-cell and spatial RNA sequencing identify divergent microenvironments and progression signatures in early- versus late-onset prostate cancer.
              </a>
            </td>
          <td>
            Yifei Cheng, Bingxin Liu, Junyi Xin, Xiaobin Wu, Wenchao Li, Jinwei Shang, Jiajin Wu, Zhengdong Zhang, Bin Xu, Mulong Du, Gong Cheng, Meilin Wang
          </td>
          <td>2025-04-10</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Patient derived xenografts (PDX) are important preclinical models of cancer as they faithfully recapitulate many aspects of human tumors. Currently, models derived from patients of non-European genetic ancestry are underrepresented, limiting the potential of precision medicine advancements to equitably improve patient outcomes. The United for Health Equity Living PDX Program (U4HELPP) seeks to address this need by generating new PDX models representative of the diverse population served by the VCU Health System. This study aims to 1) characterize the histologic, genomic, and transcriptomic features of a new cohort of ancestrally diverse PDX models of breast, pancreatic, colon, and lung cancer, and 2) identify drug sensitivities and response markers that will be targeted in ongoing studies utilizing these models.



 Tumor fragments were implanted in the bilateral mammary glands or subcutaneous flank of NSG mice, and growth rates were defined. Tissue was fixed and embedded for histological staining to identify the cancer type and subtype. Whole genome and bulk RNA sequencing were performed to assess the mutational profile, genetic ancestry, and global transcriptomic landscape. Single cell RNA sequencing was utilized to capture the transcriptomic heterogeneity within each model. Finally, high throughput drug screening with the NCI NExT Oncology Interrogation Tools Set of 555 agents was performed in vitro to assess the drug sensitivities of each PDX.



 Whole genome sequencing and bulk RNA sequencing positively identified distinct genetic ancestries within the cohort. Histopathologic analysis revealed a rare metaplastic breast carcinoma, and immunohistochemical staining stratified breast cancer models into estrogen receptor positive and triple negative subtypes. Bulk RNA sequencing was used to subtype pancreatic ductal adenocarcinoma models as basal-like, classical, or intermediate, and single-cell RNA sequencing confirmed the presence of mixed subtype populations within these samples. High throughput drug screening identified sensitivities that correlated with mutations detected through whole genome sequencing; overall, several classes of drugs including proteasome and cyclin-dependent kinase inhibitors were found to be highly cytotoxic in all cancer types tested.



 This study integrates multiple modalities to characterize the histological features, molecular signatures, and cellular heterogeneity of this new cohort of PDXs, which will provide researchers with extensive data on which to base future studies utilizing these models. Furthermore, it identifies promising therapeutic agents and explores genomic markers of drug sensitivity which could be used to predict patient response to novel treatments.



 Nina M. Dashti-Gibson, Amy L. Olex, Katarzyna M. Tyc, Carson J. Walker, Oniya M. Smith, Rachel K. Myrick, Bin Hu, Praveen Bhoopathi, Madhavi Puchalapalli, Li Wang, Yue Zhang, Brydon P. Wall, Valentina Robila, Mikhail G. Dozmorov, Jinze Liu, Jennifer E. Koblinski, Jose G. Trevino, J Chuck Harrell. Genetic, genomic, and pharmacologic characterization of a new cohort of ancestrally diverse patient-derived xenograft models of cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 49.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5507e82127e26407851957911f722cad0e285824" target='_blank'>
              Abstract 49: Genetic, genomic, and pharmacologic characterization of a new cohort of ancestrally diverse patient-derived xenograft models of cancer
              </a>
            </td>
          <td>
            Nina M. Dashti-Gibson, Amy Olex, Katarzyna M Tyc, Carson J Walker, Oniya M. Smith, Rachel K. Myrick, Bin Hu, Praveen Bhoopathi, Madhavi Puchalapalli, Li Wang, Yue Zhang, Brydon P. G. Wall, Valentina Robila, Mikhail G. Dozmorov, Jinze Liu, Jennifer E Koblinski, Jose G. Trevino, J. Harrell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive brain tumor originating from glial cells and presents difficulty in treatment. GBM are highly heterogeneous, presenting a challenge in treatment. In addition, cells migrate away often as single cells from primary tumor sites making GBM impossible to be fully resected. GBM cells adapt their behavior based on the mechanical properties of their surrounding microenvironment, making it significant to investigate gene markers associated with these mechanical changes. Our recent efforts to investigate particular gene markers included using an interface hydrogel platform that replicates the local microenvironment from the tumor edge to peritumoral tissue. Using this model we successfully separated migratory and non-migratory cells. Results from single-cell RNA (scRNA) sequencing identified gene targets that were overexpressed within the migratory group. We found the gene PTPRN2, which plays a role in vesicle trafficking and exocytosis, was overexpressed in migratory cells. This could be a promising target for future therapeutic methods to limit glioblastoma cell migration. In addition to PTPRN2, we are exploring the role of MT1X, a gene in the metallothionein family, to validate these findings. We are currently investigating the expression of the PTPRN2 and MT1X genes using GBM xenograft mouse models. We have observed PTPRN2 expression in the GBM mouse models, with the highest expression detected in migratory cells compared to non-migratory cells in soft gel conditions. In the future, we plan to perform western blot analysis to further confirm PTPRN2 and MT1X protein expression and validate their roles as potential target proteins for GBM treatment.


 Ananya Bharadwaj, Talia Sanazzaro, Stephanie K. Seidlits. Investigating the expression of PTPRN2 and MT1X genes in glioblastoma cell migration using hydrogel-based models [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1454.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5253c5eabd95acbc435d8663b104faf3baf67ce" target='_blank'>
              Abstract 1454: Investigating the expression of PTPRN2 and MT1X genes in glioblastoma cell migration using hydrogel-based models
              </a>
            </td>
          <td>
            Ananya Bharadwaj, Talia M. Sanazzaro, S. Seidlits
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Introduction Immune checkpoint inhibitors (ICIs) have significantly improved survival for patients with metastatic melanoma, yet many experienceresistance due to immunosuppressive mechanisms within the tumor immune microenvironment (TIME). Understanding how the spatial architecture of immune and inflammatory components changes across disease stages may reveal novel prognostic biomarkers and therapeutic targets. Methods We performed high-dimensional spatial profiling of two melanoma tissue microarrays (TMAs), representing Stage III (n = 157) and Stage IV (n = 248) metastatic tumors. Using imaging mass cytometry (IMC) and multiplex immunofluorescence (mIF), we characterized the phenotypic, functional, and spatial properties of the TIME. Cellular neighborhoods were defined by inflammatory marker expression, and spatial interactions between immune and tumor cells were quantified using nearest-neighbor functions (G-cross). Associations with survival were assessed using Cox proportional hazards models with robust variance estimation. Results Stage IV tumors exhibited a distinct immune landscape, with increased CD74- and MIF-enriched inflammatory neighborhoods and reduced iNOS-associated regions compared to Stage III. Cytotoxic T lymphocytes (CTLs) and tumor cells were more prevalent in Stage IV TIME, while B cells and NK cells were depleted. Spatial analysis revealed that CTL–Th cell, NK–T cell, and B–NK cell interactions were linked to improved survival, whereas macrophage aggregation and excessive B–Th cell clustering in inflammatory regions correlated with worse outcomes. Organ-specific analyses showed that CTL infiltration near tumor cells predicted survival in gastrointestinal metastases, while NK–T cell interactions were prognostic in lymph node and skin metastases. Discussion Our results reveal stage-specific shifts in immune composition and spatial organization within the melanoma TIME. In advanced disease, immunosuppressive neighborhoods emerge alongside changes in immune cell localization, with spatial patterns of immune coordination—particularly involving CTLs, NK cells, and B cells—strongly predicting survival. These findings highlight spatial biomarkers that may refine patient stratification and guide combination immunotherapy strategies targeting the inflammatory architecture of the TIME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26ea85ba1c4980ace36f2d1785fed9a6143d63d8" target='_blank'>
              Characterizing spatial immune architecture in metastatic melanoma using high-dimensional multiplex imaging
              </a>
            </td>
          <td>
            Joel Eliason, S. Krishnan, Yasunari Fukuda, Matias A. Bustos, Daniel G. Winkowski, Sungnam Cho, Akshay V. Basi, Regan Baird, Elizabeth A. Grimm, Michael A. Davies, Dave S. B. Hoon, Arvind Rao, Jared K. Burks, S. Ekmekcioglu
          </td>
          <td>2025-04-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the third leading cause of cancer-related deaths in the US, with rectal cancer (RC) accounting for one-third of cases. Only 20% of RC patients have a complete response to chemoradiation therapy (CRT) and may be able to avoid surgery, therefore, therapeutic resistance remains a major barrier to improving outcomes. The genetic and cellular heterogeneity of RC, particularly in treatment-resistant sub-clones, suggests that this diversity could be a harbinger of adverse outcomes. A detailed characterization of treatment responses at the subclonal level is critical to advancing targeted therapeutic strategies.



 Patient-derived xenograft (PDX) models and single-cell RNA sequencing were employed to investigate the cellular and molecular landscape of RC tumors. Two PDX models were established from the primary tumors of pre-treatment RC patients. Mice bearing tumors were subjected to a standard CRT regimen, consisting of 2 Gy daily radiation and 100mg/kg capecitabine daily for two weeks, followed by a week-long rest period. Tumor tissues were collected at baseline and at various post-treatment intervals for sequencing.



 Our scRNA-seq analysis of one PDX model identified five distinct tumor clusters with varying responses to treatment. Two clusters demonstrated initial sensitivity to CRT, followed by a resurgence, indicating potential therapy resistance. Pathway analysis revealed cluster-specific activation of critical oncogenic pathways, including HIF-1, p53, and FoxO signaling pathways. These pathways are implicated in tumor progression and therapeutic resistance, and the associated marker genes present potential targets for intervention. Lineage tracing of tumor cells uncovered two differentiation trajectories, with one showing increased resistance. Cell-cell communication analysis highlighted extensive interactions among tumor sub-populations, particularly mediated by CLDN3-CLDN3 adhesion, which may contribute to therapeutic resistance. Additionally, our investigation into copy number variations identified alterations in the chr6p21.31-p21.1 region, which harbors several oncogenes with copy number changes directly implicated in tumor progression. In the second PDX model, scRNA-seq identified two therapy-resistant tumor clusters among five detected clusters. The resistant tumor clusters exhibited enrichment of HIF-1, Hippo, and p53 signaling pathways, similar to the first PDX model but also unique activation of pathways like TGF-β and Hippo signaling.



 Our study highlights the dynamic, heterogeneity of RC tumors to CRT, identifying both shared and patient-specific resistance mechanisms in two PDX models. These findings underscore the complexity of RC tumor biology and offer a foundation for developing precision therapies targeting treatment-resistant subclones.



 Yuwei song, Regina K. Irwin, Karin M. Hardiman, Zechen Chong. Single-cell RNAseq revealed multiple resistance mechanisms in patient-derived xenograft model of rectal cancer during treatment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1248.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9d728476d907142dd31d1c87ffc2d2c3f782be" target='_blank'>
              Abstract 1248: Single-cell RNAseq revealed multiple resistance mechanisms in patient-derived xenograft model of rectal cancer during treatment
              </a>
            </td>
          <td>
            Yuwei Song, Regina K. Irwin, Karin M. Hardiman, Zechen Chong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The development of chemoresistance in many cancer patients poses a significant clinical challenge, while the underlying causes for heterogeneous drug responses remain largely elusive. Increasing evidence suggests that tumors exhibit extensive heterogeneity and harbor cancer subpopulations in less differentiated states that are linked to treatment resistance. Therefore, dissecting immature cancer cell states pre-existing in tumors is essential for identifying biomarkers associated with chemoresistance and establishing targeted therapeutic strategies, particularly in breast cancer, where specific biomarkers are largely lacking.



 We utilized CytoTRACE, a robust computational framework for predicting the cellular differentiation status from single-cell RNA sequencing data, to unveil the immature cancer cell state and its associated markers in single-cell atlases of human breast cancer. The association between the identified immature cell state and patient survival was determined by inferring the proportions of epithelial cancer cells expressing the marker gene in bulk gene expression datasets. The MMTV-PyMT mouse breast cancer model was used to experimentally investigate the role of the immature cancer cell state in conferring chemoresistance. Single-cell profiling was conducted on tumor samples before and after treatment to examine the inherent characteristics of immature cancer cells and their response to drug pressure. The results obtained from mouse tumors were subsequently confirmed using human breast cancer models.



 We revealed that BCL11B, a zinc finger transcription factor, defines a distinct immature cancer cell state in human breast cancer that displays basal cell features and demonstrates a heterogeneous expression pattern across breast tumors. Breast cancer patients with higher levels of BCL11B+ cancer cells face a significantly increased risk of relapse following chemotherapy. This pre-existing tumor subpopulation exhibits intrinsic fitness advantages and demonstrates a remarkable ability to evolve into a drug-resistant state through various BCL11B-dependent pre-existing and transcriptional adaptation mechanisms. Notably, TNF-α was pinpointed as a natural inhibitor of BCL11B, and treatment with TNF-α significantly increases chemotherapy efficacy by inducing the exit of the drug-resistant cell state. In conclusion, our findings identify BCL11B as a valuable biomarker for predicting chemotherapy resistance and suggest TNF-α combination treatment for breast cancer patients at high risk of developing chemoresistance.



 Zhen Qi,1 Gunsagar S. Gulati,2 Angera H. Kuo,1 Shaheen S. Sikandar,3 William Hai Dang Ho,1 Dalong Qian,1 Frederick M. Dirbas,1 Aaron M. Newman,1 Shang Cai,4 Michael F. Clarke1. BCL11B predetermines an immature drug-resistant cell state in breast cancer that is vulnerable to TNF-α treatment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1256.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b6836817685e87c48dea5b327743f821ab98f7f" target='_blank'>
              Abstract 1256: BCL11B predetermines an immature drug-resistant cell state in breast cancer that is vulnerable to TNF-α treatment
              </a>
            </td>
          <td>
            Zhen Qi, Gunsagar S. Gulati, Angera H. Kuo, Shaheen S. Sikandar, William Hai Dang Ho, D. Qian, F. Dirbas, Aaron M. Newman, Shang Cai, Michael F. Clarke
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Breast cancer, characterized by considerable heterogeneity, showcases genetic and epigenetic variability among individual tumors and their constituent cells. Notably, this intratumoral heterogeneity (ITH) can drive clonal evolution and therapy resistance, particularly in Human Epidermal Growth Factor Receptor 2 (HER2)-positive cancers. Despite the efficacy of HER2-targeted therapies, many patients develop resistance, resulting in poor prognosis. This study aims to elucidate the dynamic interplay between genetic alterations and epigenetic modifications during anti-HER2 therapy using advanced single-cell resolution techniques. Biopsy samples were collected at three time points from patients undergoing neoadjuvant HER2-targeted therapy as part of the TRIO-US B07 clinical trial. Utilizing methods such as low-pass whole-genome sequencing (WGS, n=47), single-nuclei WGS (snWGS, n=21), and matched single-nuclei RNA/ATAC-seq (n=22), we explored copy-number variations (CNVs), subclonal tumor dynamics, and alterations in the tumor microenvironment (TME). We also analyzed additional deep WGS (n= 205) to reconstruct the structure of HER2 amplicons. Our analysis revealed distinct CNV dynamics in HER2-positive breast cancers during treatment. We found that while broad-level CNVs remained stable, focal amplifications exhibited temporal variability, suggesting the presence of extrachromosomal DNA (ecDNA). Specifically, the structure of HER2 amplificons emerged as a pivotal determinant of aggressiveness, with higher numbers of co-amplified oncogenes correlating with non-pathological complete response (pCR) and metastasis. Single-cell resolution analyses unraveled significant subclonal diversity in a subset of patients exhibiting complex tumor architectures. Integration of transcriptomic and epigenomic profiling highlighted that pCR patients exhibited higher lymphocyte infiltration and dynamic immune shifts post-treatment. Conversely, non-pCR tumors displayed increased regulatory T cells. Furthermore, fibroblast state transitions characterized by altered extracellular matrix (ECM) gene expression suggested a potential role in modulating immune responses. Our findings provide novel insights into the genetic and epigenetic factors driving therapy resistance in HER2-positive breast cancer. We emphasize the importance of longitudinal, multimodal profiling to refine patient stratification and inform personalized treatment strategies. Overall, this study offers new avenues for understanding tumor evolution in response to therapy and highlights the need for comprehensive assessments of ITH and microenvironmental dynamics for improved clinical outcomes.


 Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis. Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6355.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce63b80e10f58d9373f4a7b7bf33e6cdea133649" target='_blank'>
              Abstract 6355: Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy
              </a>
            </td>
          <td>
            Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen E. Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Intrahepatic cholangiocarcinoma (ICC) is a highly lethal cancer of the bile ducts that exhibits a diversity of driver oncogenic mutations, several of which have been shown to be frequently subclonal. However, existing tumor evolution studies in ICC have been limited by either the number of sub-regions assessed or a paucity of driver mutation heterogeneity. Here, we conducted a multi-omic analysis of 90 tumor regions from 11 discrete tumors in 10 patients, identifying subclonal driver mutations in 5 of the patients, in the genes PBRM1, PIK3CA, TP53, ARID1A, BRAF, MLH1, and CDKN2A. This high-number multi-region approach also allowed us to unambiguously identify distinct subclonal evolutionary branches and map them to the spatial location of the samples. Importantly, each subclonal driver mutation was tightly associated with these defined branches, implicating them as a frequent mechanism of driving subclonal evolution. Additionally, subclonal branching showed a spatial correlation on surgical resection depicting the strength of multi-region sequencing in treatment. In total, we conclude that while subclonal branching can be driven by a number of factors, the acquisition of bona fide driver mutations strongly propels subclonal evolution and spatial patterns of tumor growth.


 Evit John, Pooja Shukla, Anish Jain, Alexandria Lau, Tom Lesluyes, Huw Ogilvie, Maxime Tarabichi, Jason Roszik, Yinyin Yuan, Peter Van Loo, Lawrence Kwong, Yun Shin Chun. Driver mutations propel subclonal evolution and spatial patterns of tumor growth in intrahepatic cholangiocarcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7499.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea1a8d6967192526fb7f6ecd11f9a23af479e446" target='_blank'>
              Abstract 7499: Driver mutations propel subclonal evolution and spatial patterns of tumor growth in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Evit R. John, Pooja Shukla, Anish J Jain, Alexandria Lau, T. Lesluyes, Huw Ogilvie, M. Tarabichi, J. Roszik, Yinyin Yuan, P. Van Loo, Lawrence Kwong, Yun Shin Chun
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="
 Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. Here, we apply computational methods to detect and reconstruct ecDNA from whole-genome sequencing data from 595 patients prospectively recruited into the Chinese Urothelial Carcinoma Genome Atlas (CUGA) study. We demonstrate that ecDNA contributes to clonal evolution through structural rearrangements and copy number variation during malignant transformation and recurrence of urothelial carcinoma (UC). This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate ubiquitous ecDNA presence across clones in multi-focal cancers, directly linking ecDNA to the evolution of multifocality. Notably, we observe highly variable levels of ecDNA across multiple foci within the same tumor, and ecDNA contributes to increased intratumoral heterogeneity and serves as a substrate for natural selection and evolution. EcDNA is present in 36% of UC tumors and correlates with an unstable cancer genome, an immunosuppressive phenotype, and significantly reduced 5-year survival. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell mediated immune responses. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA, underscoring its diagnostic potential. This study delineates the evolutionary trajectory of ecDNA during tumor progression and provides further insights into immune evasion mechanisms associated with ecDNA.


 Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Jens Luebeck, Huiying Tao, Yanfen Zhu, Xiaolong Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xiang Xi, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, Roel G. Verhaak, Bishoy M. Faltas, Jacob B. Hansen, Sihan Wu, Paul S. Mischel, Anton G. Henssen, Vineet Bafna, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han. Spatial-temporal diversity of extrachromosomal DNA shapes urothelial carcinoma evolution and tumor-immune microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2668.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77c67aedf7052778400f6b8258e4d803910f72b0" target='_blank'>
              Abstract 2668: Spatial-temporal diversity of extrachromosomal DNA shapes urothelial carcinoma evolution and tumor-immune microenvironment
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, J. Luebeck, Huiying Tao, Yanfen Zhu, X. Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xiang Xi, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B. Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40b543a5c9888e9eb175f7e7cf94681bdf0bd1dd" target='_blank'>
              An oncolytic herpesvirus expressing a CXCR4 antagonist interferes with Glioblastoma cells stemness features and migration
              </a>
            </td>
          <td>
            D’arrigo Paolo, Dubois Maxime, Sanchez Gil Judit, Lassence Cédric, Brouwers Benoit, L. Arnaud, Rogister Bernard, Neirinckx Virginie, Lebrun Marielle, Sadzot-Delvaux Catherine
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Melanoma brain metastases (MBM) have a unique molecular profile compared to extracranial metastases (ECM). Description of the biological features and clinical outcomes of MBM will facilitate the design of rational therapies. Methods We examined the mutational landscape and gene expression profiles of MBM (74 patients) and ECM (34 patients) in paired patient samples from a previously published dataset with whole-exome sequencing (WES) and RNA sequencing (RNAseq) data from MD Anderson Cancer Center (MDACC). We also present findings from MBM from a new cohort of 14 patients from Duke University to strengthen investigation of somatic mutations and gene expression profiles. Gene Set Enrichment Analysis (GSEA) was used to compare paired MBM versus lymph node (LN) metastases and skin metastases. Relative immune cell abundance was inferred using deconvolution methods. Survival outcomes from craniotomy and associations with biological features, BRAF mutation status, and PTEN expression were assessed. Results GSEA found that autophagy signaling pathways are enriched in MBM versus LN and skin metastases. BRAF was the most frequently mutated clinically relevant gene in MBM and ECM, with NRAS and PTEN also frequently altered in MBM. The most strongly upregulated genes in autophagy pathways were glial fibrillary acidic protein (GFAP) and hemoglobin beta (HBB). An increased proportion of immune-suppressive M2 compared to tumor-suppressive M1 macrophages in MBM and ECM was identified. There was not sufficient evidence for an association between BRAF V600 mutation status or expression and overall survival (OS) from craniotomy. Conclusions The mutational landscape and gene expression of MBM from the Duke cohort resembled those previously reported in the MDACC cohort. Upregulation of autophagy pathways was observed in patient-matched MBM versus LN and skin metastases due to upregulation of two genes, GFAP and HBB. In MBM, higher M2:M1 ratio may contribute to a therapeutically relevant immune-suppressive tumor microenvironment (TME). Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02035-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd2acbbd5230dc401bd3ec3c002ad4d3ee572431" target='_blank'>
              A comprehensive, multi-center, immunogenomic analysis of melanoma brain metastases
              </a>
            </td>
          <td>
            L. B. Kennedy, Amanda E. D. Van Swearingen, Marissa R Lee, Layne W. Rogers, A. Sibley, Jeff Sheng, Dadong Zhang, Xiaodi Qin, E.S. Lipp, Swaminathan Kumar, Aron Y Joon, Pixu Shi, Michael A. Davies, K. Owzar, Carey K. Anders, April K S Salama
          </td>
          <td>2025-06-02</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) continues to present a therapeutic challenge due to the fact that by definition, these cancer cells lack the expression of targetable receptors. Current treatment options include cytotoxic chemotherapy, antibody–drug conjugates (ADC), and the PD-1 checkpoint inhibitor, pembrolizumab. Due to high rates of recurrence, current guidelines for early-stage TNBC recommend either multi-agent chemotherapy or chemo–immunotherapy in all patients other than those with node-negative tumors < 0.5 cm. This approach can lead to significant long-term effects for TNBC survivors, driving a growing interest in de-escalating therapy where appropriate. Tumor infiltrating lymphocytes (TILs) represent a promising prognostic and predictive biomarker for TNBC. These diverse immune cells are present in the tumor microenvironment and within the tumor itself, and multiple retrospective studies have demonstrated that a higher number of TILs in early-stage TNBC portends a favorable prognosis. Research has also explored the potential of TIL scores to predict the response to immunotherapy. However, several barriers to the widespread use of TILs in clinical practice remain, including logistical and technical challenges with the scoring of TILs and lack of prospective trials to validate the trends seen in retrospective studies. This review will present the current understanding of the role of TILs in TNBC and discuss the future directions of TIL research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179f3f7188fc55a777be68aae10786476ade4158" target='_blank'>
              Harnessing Tumor-Infiltrating Lymphocytes in Triple-Negative Breast Cancer: Opportunities and Barriers to Clinical Integration
              </a>
            </td>
          <td>
            Cara Coleman, Tharakeswari Selvakumar, A. Thurlapati, Kevin Graf, Sushma Pavuluri, Shikhar Mehrotra, Ozgur Sahin, A. Sivapiragasam
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The evolution of isocitrate dehydrogenase (IDH)-wildtype glioblastoma (GBM) after standard-of-care therapy remains poorly understood. Here we analyzed matched primary and recurrent GBMs from 59 patients using single-nucleus RNA sequencing and bulk DNA sequencing, assessing the longitudinal evolution of the GBM ecosystem across layers of cellular and molecular heterogeneity. The most consistent change was a lower malignant cell fraction at recurrence and a reciprocal increase in glial and neuronal cell types in the tumor microenvironment (TME). The predominant malignant cell state differed between most matched pairs, but no states were exclusive or highly enriched in either time point, nor was there a consistent longitudinal trajectory across the cohort. Nevertheless, specific trajectories were enriched in subsets of patients. Changes in malignant state abundances mirrored changes in TME composition and baseline profiles, reflecting the co-evolution of the GBM ecosystem. Our study provides a blueprint of GBM’s diverse longitudinal trajectories and highlights the treatment and TME modifiers that shape them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0990e85585d1a285c19cfc504ce05c954e344b" target='_blank'>
              Deciphering the longitudinal trajectories of glioblastoma ecosystems by integrative single-cell genomics
              </a>
            </td>
          <td>
            Avishay Spitzer, K. Johnson, M. Nomura, L. Garofano, Djamel Nehar-Belaid, Noam Galili Darnell, Alissa C. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, Simona Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, Jong Bae Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, Simone P Niclou, Labeeba Nusrat, Sorcha Kellet, Sunit Das, H. Moon, S. Paek, Franck Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, Marc Sanson, S. Tanaka, Nobuhito Saito, D.M. Ashley, Stephen T Keir, Keith L. Ligon, Jason T Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-05-09</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>62</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) transforms a diploid cell into a tetraploid state, making it prone to genomic instability, aneuploidy, and malignancy. Tumors with evidence of past WGD events range from less than 10% in Non-Hodgkin’s Lymphoma to almost 60% in germ cell cancer. WGD confers advantages, including resistance to stressors, making it a powerful macroevolutionary force. Why, then, isn’t WGD a precursor for most cancers? In Glioblastoma (GBM), ∼86% of tumors lack evidence of WGD in their evolutionary history. We integrated a multi-modal dataset of 11 histopathology whole slide images (WSIs), 68 medical images, and whole-genome sequencing data (WGS) from six patient-matched primary and recurrent GBM. Three atlases were derived from FDG PET/MRI (glucose uptake), quantitative BOLD imaging (oxygenation), and magnetic resonance elastography (stiffness), capturing variability across brain regions and patient age. Using HATCHet, which infers clone-specific karyotypes from WGS, we quantified GBM clonal composition and its changes between surgeries. Chromosome 7 amplification and chromosome 10 loss were detected in at least one clone in all tumors. Genetic intra-tumor heterogeneity (coexistence of ≥2 clones) was found in 4/6 patients, with moderate to strong clonal changes between surgeries in all four. This data informed parameters governing how cells of variable (poly)aneuploidy respond to oxygen and glucose by dying (α_d, s_r), switching to glycolysis (O_th), or migrating (v_max). Two parameters significantly differed among patients: oxygen-dependent death constant α_d (ANOVA p = 1.7E-5) and oxygen level prompting a metabolic shift to glycolysis O_th (ANOVA p = 1.9E-4). Higher-ploidy GBMs showed elevated O_th values, and higher-ploidy clones within tumors had consistently higher O_th than coexisting lower-ploidy clones. Between surgeries, simulated tumors with distinct karyotype compositions progressed at different rates. Whether higher ploidy predicted fast recurrence was however a function of the tumor anatomic location, in line with the hypothesis that environments with limited resources are likely to exert selective pressure against WGD. Compared to intra-brain differences in resource supply, the differences observed between organs can be larger, providing an opportunity to further test this hypothesis. Indeed, the average ploidy of 20 different cancer types was positively correlated with the oxygen levels recorded in their respective organ of origin (Spearman r = 0.65, P = 0.002) and negatively correlated to the glucose uptake rates in the organ of origin (Spearman r = -0.70, P = 0.026). Overall, our findings suggest that metabolic substrate availability in the tissue micro-environment influences cell fate decisions based on karyotype, shaping GBM initiation and recurrence.


 Zuzanna Nowicka, Frederika Rentzeperis, Vural Tagal, Konstantin Maksin, Parag Katira, Noemi Andor. Interactions between ploidy and resource availability shape clonal interference in glioblastoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6238.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63a435551ab128bd453aa26cc898c619184a9dc" target='_blank'>
              Abstract 6238: Interactions between ploidy and resource availability shape clonal interference in glioblastoma
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, Konstantin Maksin, Parag Katira, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Cutaneous squamous cell carcinoma (SCC) is known for its stepwise progression from healthy skin to premalignant actinic keratosis (AK), followed by a malignant transformation to SCC. Unfortunately, less attention has been paid to changes in gene expression in the tumour microenvironment during this process. We retrospectively selected early‐stage cutaneous SCC tissue samples containing both invasive and premalignant portions and conducted a spatial transcriptomic experiment using a NanoString GeoMx Digital Spatial Profiler (DSP). First, we selected invasive and premalignant regions of interest (ROIs) for each tissue. We then compared the gene expression patterns between the two portions (invasive versus premalignant) of the three segments: tumour cells, immune cells and fibroblasts, in each ROI. As a result, early‐stage cutaneous SCC tissue samples from 17 patients were selected for this study. We identified 29, 14 and 15 differentially expressed genes (DEGs) between the invasive and premalignant portions of the tumour cells, immune cells and fibroblasts, respectively. The top three genes with the highest absolute log2 fold‐change were CCDC88C, GJD3 and COMP in tumour cells; SVEP1, TSLP and PPP2R5C in immune cells; and SPAG6, PPP1CA and CCDC68 in fibroblasts. Notably, several genes, such as COMP, SVEP1 and SPAG6, have been linked to the development and function of cancer‐associated fibroblasts. Functional enrichment analysis revealed that several pathways were altered in tumour and immune cells. In conclusion, distinctive changes in gene expression patterns were observed as AK progressed to SCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1212cb843dc2d0cad33b53da437b7c501e408d51" target='_blank'>
              Spatial Transcriptomics Shows a Distinctive Tumour Microenvironment in the Invasive Versus Premalignant Portion of Early Cutaneous Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jeong-An Gim, Chungyeul Kim, Hyun Jyung Oh, Ko Eun Kim, J. Jeon, Aeree Kim, Y. Baek
          </td>
          <td>2025-06-01</td>
          <td>Experimental Dermatology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Cell-free DNA (cfDNA) tumor fraction (TF) quantification is an emerging biomarker with many promising applications, including the potential to complement tumor size measurements from imaging. Genomic approaches for TF estimation are limited by the presence of cancer driver alterations, detection sensitivity, and may be confounded by non-tumor derived variants. Therefore, we have previously developed a method using differentially methylated regions to robustly estimate TF (Chen et al AACR 2023). Here we leverage a rich dataset of >50, 000 cfDNA samples from >20 cancer types to better understand the variability of epigenomic signals in cancer and non-cancer individuals and inform an update of this TF estimation algorithm.



 We employed a data-driven approach to update the TF algorithm leveraging a large cohort of cfDNA samples with simultaneous genomic and epigenomic profiling. This extensive dataset allowed for robust modeling of non-tumor variability in epigenetic profiles, and better calibration of tumor signals across diverse primary tissue sources shedding signals into plasma. The method update includes a data-informed region selection targeting high pan-cancer signal to noise, as well as more robust accounting for rare sources of non-tumor methylation signals including technical variation and treatment associated tissue damage.



 We evaluated the performance of epigenomic TF estimates using 1203 samples with somatic maximum MAF, establishing 88% concordance within 0.2-5X range. Next, samples were computationally titrated into non-tumor background samples, demonstrating consistent concordance with genomic data at up to 10-fold dilutions and maintenance of precision well below the resolution of genomic profiling, largely driven by algorithmic improvements in this updated release. Case studies of discordant samples show the benefit of TF over genomic quantification, especially in scenarios with an ambiguous genomic driver, or low genomic precision. In analytical validation studies of experimental titrations over replicates, linearity and quantitative precision of TF was demonstrated down to an order of ∼0.01%, compared to >0.4% for genomic TF.



 Our liquid only TF has been analytically validated for use in clinical, research, and trial settings in patients with solid tumors. In addition to its use for absolute quantification, it is incorporated into downstream applications including therapy response monitoring and algorithms for labeling germline variants, clonal hematopoiesis, and wild-type variant calling. Finally, its application in a clinical cohort of over 50, 000 samples has led to insights into the dynamics of cfDNA tumor shedding across pre- and on- treatment individuals improving accuracy of quantification and providing results for more patients.



 Andrew Gross, Jun Zhao, Katie Quinn, Sai Chen, Kyle Chang, Catalin Barbacioru, Tingting Jiang, Carin Espenschied, Kristin Price, Jack Tung, Darya Chudova. Landscape of epigenomic tumor fraction in large pan-cancer cfDNA cohort [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5936.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12c4fbbacd791008bfb4979677b21d5ce6d494b1" target='_blank'>
              Abstract 5936: Landscape of epigenomic tumor fraction in large pan-cancer cfDNA cohort
              </a>
            </td>
          <td>
            Andrew Gross, Jun Zhao, Katie Quinn, Sai Chen, Kyle Chang, Catalin C. Barbacioru, Tingting Jiang, Carin R. Espenschied, Kristin Price, Jack Tung, Darya I. Chudova
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Quiescent, or G0 cell cycle phase, cells are common in most prostate cancer (PCa) cases, yet are relatively resistant to many treatments, especially cytotoxic chemotherapy. They also play key roles in tumorigenesis, and dormancy and recurrence after curative intent therapy, but are difficult to identify when viable. To identify characteristics of quiescent prostate cancer cells, we used previously published FACS methods to isolate cancer cells from bone or bone marrow of eight metastatic castration resistant prostate cancer (mCRPC) patient samples and generated single cell RNA sequencing (scRNA-seq) gene expression data of individually sorted cells. We also generated scRNA-seq data from six PCa cell lines and two normal (cancer free) bone marrow samples as positive and negative controls for cell identification. We eliminated a priori any potentially contaminating cells based on tSNE cluster location, expression of a 12 gene panel of PCa markers, and lack of expression of bone marrow markers (PTPRC / CD45, and GYPA / CD235a), leaving 297 high confidence PCa cells. To identify characteristics of G0 cells, we separated the patient PCa cells into quiescent and cycling groups based on expression of MKI67 (Ki67). GSEA analysis of KEGG pathways revealed enrichment of primarily immune related pathways in quiescent cells with the top 3 pathways; “allograft rejection, ” “autoimmune thyroid disease, ” and “intestinal immune network for IgA production.” Similarly, for GSEA analysis using gene ontology (GO) groups, “beta-2-microglobulin binding” was the second most enriched group in quiescent cells. Type 1 HLA genes, were common among multiple pathway groups suggesting potential differences in antigen presentation. In subsequent validation experiments, induction of quiescence in LNCaP, C4-2B, PC3, and RM1 PCa cell lines increased cell surface HLA class 1 or Mhc-1. Overall, these results suggest that antigen presentation is increased in quiescent PCa cells, which may be important for their elimination by the immune system, or useful for future immunotherapy approaches.


 Kristina G. Ibrahim, Julie Boerner, Steven Zielske, Frank C. Cackowski. Single cell analysis reveals upregulation of immune stimulatory pathways in quiescent metastatic castration resistant prostate cancer cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5097.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4707d957a214c6885ebfa8cd85de26bb43f4b999" target='_blank'>
              Abstract 5097: Single cell analysis reveals upregulation of immune stimulatory pathways in quiescent metastatic castration resistant prostate cancer cells
              </a>
            </td>
          <td>
            K. G. Ibrahim, Julie Boerner, Steven Zielske, F. Cackowski
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is the complex ecosystem surrounding a tumor. It is a complex and continuously evolving entity. The TME comprises a variety of complex components, such as cancer cells, immune cells, stromal cells, and extracellular matrix, that precisely regulate the interaction of tumor cells with other components, allowing tumor cells to continue to metastasize, escape immune surveillance, resist apoptosis, and proliferate. Researchers believe that the TME is not just a silent bystander but instead an active promoter of cancer progression. At the onset of tumor growth, a dynamic and reciprocal relationship is developed between cancer cells and components of the tumor microenvironment; this relationship supports cancer cell survival, local invasion, and metastatic dissemination. In this article, we discuss the role of the various components of the TME in the progression and metastasis of cancer, the pathways influenced by the TME, and the drugs that can treat tumors by targeting the components of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9869eb44a45ce4947a2fc2cac8a08a25b3dc82" target='_blank'>
              The Role of the Tumor Microenvironment in Cancer Progression and Metastasis
              </a>
            </td>
          <td>
            Oseghale, Ikalo David, Ikokwu, Godwin Mmaduabuchi, Ugoagwu, Kingsley Ugonna, Elizabeth Anuoluwa Akintayo, Matthew Gyang, Okiki Nosiru Quadri, Catherine Rono, O. Ralph-Okhiria, Odunvbun Praise
          </td>
          <td>2025-05-29</td>
          <td>Journal of Cancer and Tumor International</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Isoform switching in cancer is a prevalent phenomenon with significant implications for immunotherapy, as actionable neoantigens derived from these cancer-specific events would be applicable to broad categories of patients, reducing the necessity for personalized treatments. By integrating five large-scale transcriptomic datasets comprising over 19,500 samples across 29 cancer and 54 normal tissue types, we identified cancer-associated isoform switching events common to multiple cancer types, several of which involve genes with established mechanistic roles in oncogenesis. The presence of neoantigen-containing peptides derived from these transcripts was confirmed in broad cancer and normal tissue proteome datasets and the binding affinity of predicted neoantigens to the human leukocyte antigen (HLA) complex via molecular dynamics simulations. The study presents strong evidence that isoform switching in cancer is a significant source of actionable neoantigens that have the capability to trigger an immune response. These findings suggest that isoform switching events could potentially be leveraged for broad immunotherapeutic strategies across various cancer types. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00817-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f67b4f15a19d7345c9bb37824d41a5eb5aa238" target='_blank'>
              Potential shared neoantigens from pan-cancer transcript isoforms
              </a>
            </td>
          <td>
            Jirapat Techachakrit, Aijaz Ahmad Malik, T. Pisitkun, S. Sriswasdi
          </td>
          <td>2025-05-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="


 Glioblastoma (GBM), the most aggressive brain tumor, is treated with surgery, chemotherapy, and radiation, yet 90% recur within two years. PD-1 blockade improves outcomes in other cancers, but shows limited effect in GBM, likely due to tumor heterogeneity and an immunosuppressive microenvironment. This study aims to investigate post-treatment transcriptional changes in tumor cells and tumor-associated macrophages (TAMs) in matched primary and recurrent GBM, focusing on neoadjuvant PD-1 blockade.



 We used the GeoMx Digital Spatial Profiler to spatially select and transcriptomically profile tumor cells (SOX2+) and TAMs (IBA1+). Gene expression changes between primary and recurrent GBMs were analyzed in 30 patients (26 IDHwt, 4 IDHm) with 20 paired primary samples. All patients received standard treatment, and 20 also received second-line nivolumab one week before recurrence surgery. Tumor purity was confirmed using InferCNV and cell signatures. Subsequently, differential expression and pathway enrichment analyses were performed to compare recurrent tumors treated with nivolumab to those untreated, as well as primary vs. recurrent tumors. We also evaluated Neftel signatures in matched tumors to assess treatment- and recurrence-associated malignant state changes.



 Principal component analysis of all samples revealed distinct transcriptional profiles between tumor cells and TAMs. Separate analyses showed clear differences in tumor cell and TAM segments based on tumor status (primary/recurrent), though no separation related to nivolumab treatment. Similarly, comparing nivolumab-treated recurrent tumors with untreated controls showed no significant changes in malignant states or gene expression in tumor cells or TAMs, consistent with the lack of clinical survival benefit.To investigate overall transcriptional changes induced by standard treatment, differential expression analysis comparing tumor cell segments in paired primary and recurrent tumors revealed few significant genes, likely reflecting high inter-patient variability driven by tumor heterogeneity. Unsupervised clustering of tumor cell segments identified a distinct cluster specific to recurrent tumors, with DEGs highlighting pathways linked to this particular tumor cell phenotype. In TAM segments, increased expression of checkpoint-related genes in recurrent tumors suggests TAM phenotype changes that may influence immune dynamics in the microenvironment.



 Spatial profiling of matched primary and recurrent GBM samples reveals distinct transcriptional changes in both tumor cells and TAMs, reflecting the complex microenvironment of recurrent GBM. However, our findings point to a limited impact of neoadjuvant PD-1 blockade on transcriptional activity, highlighting ongoing challenges in targeting GBM with immunotherapy.



 Sara Artzi, Marc N. Klausen, Dylan S. Harwood, Signe R. Michaelsen, Simone B. Maarup, Alessio Locallo, Nicolai S. Bager, Nadine M. Hammouda, Benedikte Hasselbalch, Ulrik Lassen, Joachim Weischenfeldt, Bjarne W. Kristensen. Spatial profiling of matched primary and recurrent glioblastoma samples reveals changes in cellular states and the immune microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 758.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce16964e484673e282fc2269d3a4d1174c03a3ea" target='_blank'>
              Abstract 758: Spatial profiling of matched primary and recurrent glioblastoma samples reveals changes in cellular states and the immune microenvironment
              </a>
            </td>
          <td>
            S. Artzi, Marc N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, N. Bager, N. M. Hammouda, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 The spatial heterogeneity of tumors poses significant challenges for effective treatment, resulting in varied therapeutic responses across different regions of the same tumor. This challenge is especially pronounced in brain cancers, where uneven drug distribution—largely due to biological barriers like the blood-brain barrier—complicates treatment. Techniques that overlay drug distribution with transcriptional profiles would enable a comprehensive, unbiased investigation of drug effects.
 In this study, we present a spatial multiomics approach that integrates spatial metabolomics and spatial transcriptomics within the same tissue section of mouse brain samples treated with Palbociclib, a selective CDK4/6 inhibitor with potential for targeting various cancers, including breast cancer and medulloblastoma (MB). Our high-resolution (2-55 µm) concurrent multiomics maps reveal that the differential distribution of Palbociclib results in heterogeneous effects, promoting differentiation and reducing proliferation in specific tumor regions. Through this approach, we identified tissue regions exhibiting distinct transcriptional and metabolic signatures, including areas unaffected by drug treatment, which could contribute to recurrence.
 To support this analysis, we developed SpaMTP, a state-of-the-art software package providing a comprehensive pipeline for in-depth analysis of spatial metabolomics for integrating spatial multiomics data. This package bridges processing functionalities for metabolomic data from Cardinal with user-friendly cell-centric analyses implemented in Seurat. SpaMTP’s comprehensive analysis pipeline covers (1) automated mass-to-charge ratio (m/z) metabolite annotation, (2) a wide range of downstream statistical analyses including differential metabolite expression and metabolic pathway analysis, and (3) integrative spatial-omics analysis. For broad implementation and interoperability, SpaMTP includes various functions for data visualisation, data object conversion and data import/export.
 Overall, our work describes novel experimental and analytical methods to map drug distribution and its transcriptional impacts directly, offering critical insights into the heterogeneous effects of Palbociclib across brain tumor tissues.


 Tuan Vo, Cedric Cui, Andrew Causer, Xiao Tan, Eunju Kim, Zherui Xiong, Hani Vu, Mark Hodson, Tianyao Lu, Saskia Freytag, Jenny Fung, John Lee, Quan Nguyen. Concurrent spatial transcriptomics and metabolomics map differential effects of drug across brain cancer tissues [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5067.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d749b2402cb9dc98e8178e8ebdbbf205242ef15b" target='_blank'>
              Abstract 5067: Concurrent spatial transcriptomics and metabolomics map differential effects of drug across brain cancer tissues
              </a>
            </td>
          <td>
            Tuan Vo, Cedric Cui, Andrew Causer, Xiao Tan, Eunju Kim, Zherui Xiong, Hani Vu, Mark Hodson, Tianyao Lu, S. Freytag, Jenny Fung, John Lee, Quan H. Nguyen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="broblasts to produce CXCL14, thereby enhancing the invasive and migratory capabilities of prostate cancer epithelial cells. CONCLUSIONS: This study reveals fundamental differences in the TME between mHSPC and nmHSPC, uncovering a novel signaling pathway between APOE-expressing macrophages and CXCL14-secreting ﬁ broblasts. These ﬁ ndings provide a foundation for developing targeted therapeutic strategies aimed at inhibiting the metastatic spread of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b1f765b0672c6b96a6a57ddf33b8e2c4a07cf9f" target='_blank'>
              PD10-11 DIGITAL SPATIAL PROFILING REVEALS TRANSCRIPTOMIC HETEROGENEITY OF PROSTATE CANCER IN PATIENTS WITH GERMLINE MUTATIONS
              </a>
            </td>
          <td>
            S. Nallandhighal, Zixiong Huang, Piroz Bahar, Steven M Monda, A. Zaslavsky, Udit Singhal, Udager Aaron, Sethu Pitchiaya, G. Palapattu, Todd M. Morgan, V. Dadhania, S. Salami
          </td>
          <td>2025-05-01</td>
          <td>The Journal of Urology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 e15100


 Background:
 Intratumoral heterogeneity, driven by phenotypic plasticity, contributes to disease progression and therapy failures. However, a thorough understanding of its underlying mechanisms is still lacking. Breast cancer exemplifies this with its histological and molecular diversity; estrogen receptor-positive (ER+), progesterone receptor-positive (PR+), and HER2- luminal A subtypes, such as the MCF7 cell line, are not the exception. While single-cell RNA sequencing (scRNA-seq) and three-dimensional multicellular tumor spheroids (MCTS) represent initial approaches to understanding heterogeneity, integrating these tools with computational strategies offers new paths to uncover the regulatory groundwork of tumor behavior. When modeled via Boolean frameworks, transcriptional regulatory networks (TRNs) provide a dynamic platform for studying gene interactions, identifying attractors (stable cellular states corresponding to distinct phenotypes), and simulating cellular transitions. This study represents the first step in constructing such a framework for ER+ breast cancer to advance precision medicine and improve therapeutic strategies.
 Methods:
 Regulatory elements relevant to the MCF7 cell line were identified through an extensive review of NCBI literature. Interactions were strictly curated based on evidence quality (ranging from robust experimental validation to predictive hypotheses), and evaluated for type and significance, such as feedback loops, activation, coactivation, and inhibition. Logical operators (AND, OR, NOT) were used to translate regulatory connections into Boolean functions to simulate binary gene expression dynamics. The final TRN was visualized using Cytoscape (v3.10.3) with a hierarchical layout.
 Results:
 The TRN consists of 54 nodes and 81 interactions, representing the regulatory landscape of ER+, PR+, and HER2- breast cancer. Boolean equations offer a framework for simulating the evolution of binary states to attractors, enabling the observation of how perturbations, such as gene overexpression or silencing, influence heterogeneity. While simulation and attractor analyses are ongoing, the network provides a foundation for elucidating the regulatory mechanisms underlying breast cancer plasticity.
 Conclusions:
 This study highlights the potential of integrating experimental data, TRN reconstruction, and Boolean modeling to understand intratumoral heterogeneity. Our rigorously curated model establishes a foundational framework for precision oncology and the integration of experimental and computational methods.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10947cef519fbed46d1a42e2676b4ea1fd778e16" target='_blank'>
              Boolean network modeling: The first step toward a new understanding of breast cancer heterogeneity.
              </a>
            </td>
          <td>
            Lia Medina Montalvo, Thelma Escobedo-Tapia, Osbaldo Resendis-Antonio
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Cancer heterogeneity poses a significant medical challenge, necessitating therapies tailored not only to individual patients but also to the unique cancer clones that exist within diverse microenvironmental contexts. Despite this diversity, nearly all cancer cells share a reliance on telomeres to maintain genomic stability. The telomerase (hTERT) gene is hyperactive in many cancers and has been identified as a potential target for treatment. While this has led to the development of multiple telomerase-targeting approaches, disappointingly, none have been approved in clinical applications yet. To circumvent this concern, our team has applied a genetic approach called synthetic dosage lethality (SDL), to exploit hTERT overexpression to identify potential targets to treat cancer. SDL is a genetic concept, where a normally non-lethal gene inactivation kills cells only in the context of overexpression of another gene like hTERT. Our laboratories have used lentiviral-based pooled CRISPR/Cas9 and pooled shRNA-screening platforms to systematically query the genome and recently identified several SDL partners of hTERT. Through extensive validation using pooled in vivo CRISPR/Cas9 screens across multiple cell lines, tumor xenografts, and patient-derived organoids, we identified RMT1, an RNA 2'-O-methyltransferase, as a prominent SDL partner of hTERT. In collaboration with the AtomWise and Thoth Biosimulations companies, we have developed novel inhibitors for RMT1 that preferentially suppress hTERT-overexpressing cell lines growth and viability. We have also performed a large-scale screen of FDA-approved small molecule inhibitors with the goal of repurposing these drugs to target hTERT-overexpressing cancers. Subsequently, we plan to apply these therapies in combination to amplify the efficiency of treatment against cancers, yielding preclinical evidence to support the development of novel cancer therapies. Since hTERT activity is essential in all cancer cells, our findings introduce a transformative approach to overcoming cancer cell heterogeneity by targeting the SDL interactions of hTERT. This strategy also capitalizes on the ubiquitous overexpression of hTERT in all cancer types, enabling the selective eradication of hTERT-dependent malignancies.


 Vincent Maranda, Frederick S. Vizeacoumar, Yue Zhang, Liliia Kyrylenko, Andrew Freywald, Franco J. Vizeacoumar. Developing combination therapies for telomerase-overexpressing cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5412.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41aaa58f8e3f0c715544a400d743c926b588c9a" target='_blank'>
              Abstract 5412: Developing combination therapies for telomerase-overexpressing cancers
              </a>
            </td>
          <td>
            Vincent Maranda, F. Vizeacoumar, Yue Zhang, L. Kyrylenko, A. Freywald, Franco J Vizeacoumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [12],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>