<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-26 09:44:11 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>103</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intertumoral heterogeneity in glioblastoma—driven by both genomic and transcriptomic variation—complicates our understanding of how different tumor cell populations contribute to disease progression. Infiltrating tumor cells, which invade surrounding brain tissue and evade surgical resection, are thought to play a central role in recurrence. To address this, we aimed to characterize the gene expression profiles and cellular states of infiltrative tumor cells in glioblastoma. We performed high-plex spatial transcriptomics using the CosMx Spatial Molecular Imager (NanoString) on tumor tissue from eight glioblastoma patients. Formalin-fixed paraffin-embedded samples were selected to capture both the tumor core and invasive margin. A targeted panel of 1,000 genes enabled spatially resolved gene expression profiling at single-cell resolution, allowing precise identification and localization of malignant and non-malignant cell states. We show that malignant cells can be distinguished from non-malignant populations by using patient-specific clustering. Based on this annotation, we identified several known malignant states—including AC-, OPC-, NPC-, and MES-like cells—as well as a recently characterized glial-progenitor (GPC)-like state. This population co-expressed genes associated with both astrocytic and oligodendrocyte progenitor lineages and was found to be more proliferative than the traditional AC-like state. The GPC-like state was most enriched in the classical glioblastoma subtype and was strongly associated with EGFR amplification or mutation. Spatial analyses investigating malignant differences between tumor and infiltrated tissue showed heterogeneous infiltration patterns across patients. In the most extreme case, the dominant GPC-like population in the tumor core gave way to increased proportions of AC-like cells in infiltrated regions. Our study highlights diverging infiltration patterns across glioblastoma tumors, with indications of a GPC-like to AC-like transition occurring in classical-subtyped tumors. This shift is associated with a decrease in cell proliferation and may have implications for clinical treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300bf1ce4707bea01a2973482ed7fc72841204e2" target='_blank'>
              Genomic heterogeneity drives distinct infiltration patterns in glioblastoma
              </a>
            </td>
          <td>
            D. Harwood, S. Artzi, V. Pedersen, A. Locallo, M. Lü, David Scheie, D. Nørøxe, N. Hammouda, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-11-29</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f38657311790b6bfe59be5cdd1d26d9f9bbb5cc7" target='_blank'>
              Advancing small cell lung cancer metastasis research: innovations in preclinical mouse models
              </a>
            </td>
          <td>
            Qiqi Zhao, Liang Hu, Hongbin Ji
          </td>
          <td>2025-11-29</td>
          <td>Cancer and Metastasis Reviews</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e69c4c3aa7ac04b1dca84c51fe6263d52efa060" target='_blank'>
              Breast cancer through the lens of whole transcriptome spatial imaging
              </a>
            </td>
          <td>
            Claire Williams, Yi Cui, Michael Patrick, Giang T Ong, Terence Theisen, Megan Vandenberg, Joseph M. Beechem, Patrick Danaher
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Melanoma outcomes have dramatically improved over the past decade, but some patients still experience disease recurrence, particularly those who present at later stage of disease. Prior studies identified an altered immune microenvironment in the sentinel lymph node (SLN) including the presence of dysfunctional CD8 T cells and CD4 T regulatory cells (Tregs), but the spatial organization of the SLN and the interactions of individual cell types have not been extensively studied. To understand how spatial organization of immune cells in the SLN is related to outcomes, we performed spatial proteomic profiling of Stage I and II melanoma SLN to generate an atlas of cell types. Following multiplexed immunofluorescence imaging, deep learning-based segmentation and clustering, we identified 33 subsets of T cells, B cells and Tumor cells. Lymphoid region analyses established a foundational spatial map of the melanoma SLN, revealing higher regional frequencies of activated and memory CD4 T cells in Stage I SLN and consistent positioning of T cells at functional spatial locations. To better understand cellular interactions, we evaluated the immediate neighbors around each index cell using Effect Size Interaction mapping (ESI-map), a novel computational toolkit for understanding spatial interactions. Nearest-neighbor analyses across biological replicates revealed rewiring in cell-cell interacting pairs across disease progression and patient outcomes, particularly with respect to exhausted TOX+ CD8 T cells. Stage II patients who experienced disease recurrence had notable enrichment of cellular interactions between Tregs and TOX+ CD8 T cells that was associated with reduced expression of granzyme B and interferon-γ that was most pronounced in exhausted CD8 T cells. Together, these data demonstrate that the specific spatial interactions between Tregs and exhausted CD8 T cells in the SLN provide a novel immune signature for understanding melanoma outcomes. One Line Summary Treg-CD8 T cell interactions in the sentinel lymph node predict recurrence outcomes in Stage II SLN+ melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927407a38e9474aa6152c45daf9e0743472dc6ee" target='_blank'>
              Cellular interactions in the sentinel lymph node predict melanoma recurrence
              </a>
            </td>
          <td>
            Sabrina M. Solis, Yang Yang, Yee Hoon Foong, Anushka Dheer, K. Ma, M. S. Farooq, Shira G. Rosenberg, Sophie L. Gray-Gaillard, G. Karakousis, Xiaowei Xu, Rami S. Vanguri, Alexander C. Huang, R. Herati
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d2c494a1cd657524f51f9577b21d3baa396ea7" target='_blank'>
              Spatiotemporal regulation of cell cycle states within the complex tumor microenvironment
              </a>
            </td>
          <td>
            Gianni Zanardelli, Olivier Tassy, Maulik K. Nariya, Nacho Molina
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, I. Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatial transcriptomics has emerged as a transformative technology in biomedical research, offering unprecedented insights into gene and protein expression within their native tissue context. Unlike conventional bulk or single-cell sequencing approaches, spatial omics has the advantage of preserving the spatial structure of tissues, allowing researchers to directly map molecular information onto histological structures. This review provides an overview of the current state of spatial omics technologies, highlighting their application in cancer research. Spatial omics has enabled detailed characterization of the tumor microenvironment (TME), revealing spatial heterogeneity, immune cell infiltration patterns, and complex mechanisms of tumor progression and therapy resistance across various cancer types. The review covers future directions, including artificial intelligence–driven analytics, improved standardization, and cost reduction to accelerate clinical translation. Ultimately, spatial omics is poised to play a central role in precision oncology, enabling a deeper understanding of tumor biology and informing more effective individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a77ce136fd5fd93d240b66bc2895fcb841db1de" target='_blank'>
              Spatial omics: applications and utility in profiling the tumor microenvironment
              </a>
            </td>
          <td>
            Ji-Eun See, Sarah Barlow, Wani Arjumand, Hannah DuBose, Felipe Segato Dezem, Jasmine T. Plummer
          </td>
          <td>2025-12-01</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad9383618813c7fb20bfa893bf5368770766ebe" target='_blank'>
              Analysis of intratumoral immune heterogeneity reveals spatial organisation of immunosuppression in breast cancer
              </a>
            </td>
          <td>
            Christina Metoikidou, P. Bonté, Vadim K. Karnaukhov, Laëtitia Lesage, L. Djerroudi, André Nicolas, P. Sirven, A. Vincent-Salomon, Olivier Lantz, Emanuela Romano, Sebastián Amigorena
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d63cfbf90a2e544c2d24c8ab605e5d44e8ba24" target='_blank'>
              A Pan-Cancer Single-Cell Compendium of Intratumoural Heterogeneity
              </a>
            </td>
          <td>
            Ido Nofech-Mozes, Ashton Cook, Tom W. Ouellette, Sara Hafezi-Bakhtiari, Philip Awadalla, Sagi Abelson
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Protein spatial imaging, rooted in spatial proteomics, enables the localization of proteins within their native environment, thus preserving structural and functional context. Its integration into cancer research has provided insights into tumor progression, especially in breast cancer, where ductal carcinoma in situ (DCIS) may transition into invasive breast cancer (IBC). Multiplexed ion beam imaging by time-of-flight (MIBI-TOF) has emerged as a powerful method to visualize protein distribution, tumor–stroma interactions, and immune cell dynamics. Evidence indicates that the tumor microenvironment (TME)—including stromal architecture, extracellular matrix (ECM), and immune cell infiltration—plays a central role in malignant progression. Here, we highlight recent advances in protein spatial imaging, discuss optimization strategies, and examine its implications for breast cancer progression and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a7eef8cc6668ecc3d6de49f721002ee2756192" target='_blank'>
              Spatial Proteomics Imaging to Decode Disease Progression: A Perspective on Tumor Microenvironments
              </a>
            </td>
          <td>
            Haonan Zhang
          </td>
          <td>2025-11-30</td>
          <td>Clinical Medicine and Pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="While advances in single-cell genomics have helped to chart the cellular components of tumor ecosystems, it has been more challenging to characterize their specific spatial organization and functional interactions. Here, we combine single-cell RNA-seq, spatial transcriptomics by Slide-seq, and in situ multiplex RNA analysis to create a detailed spatial map of healthy and dysplastic colon cellular ecosystems and their association with disease progression. We profiled inducible genetic CRC mouse models that recapitulate key features of human CRC, assigned cell types and epithelial expression programs to spatial tissue locations in tumors, and computationally used them to identify the regional features spanning different cells in the same spatial niche. We find that tumors were organized in cellular neighborhoods, each with a distinct composition of cell subtypes, expression programs, and local cellular interactions. Comparing to scRNA-seq and bulk RNA-seq data from human CRC, we find that both cell composition and layout features were conserved between the species, with mouse neighborhoods correlating with malignancy and clinical outcome in human patient tumors, highlighting the relevance of our findings to human disease. Our work offers a comprehensive framework that is applicable across various tissues, tumors, and disease conditions, with tools for the extrapolation of findings from experimental mouse models to human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af74f8f0f1a2428bab07b6671a3e436ab63d622f" target='_blank'>
              Spatially defined multicellular functional units in colorectal cancer revealed from single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            I. Avraham-Davidi, Simon Mages, J. Klughammer, Noa Moriel, Shinya Imada, Matan Hofree, Evan Murray, Jonathan Chen, Karin Pelka, Arnav Mehta, Genevieve M Boland, T. Delorey, Leah Caplan, Danielle Dionne, Robert Strasser, Jana Lalakova, Anežka Niesnerová, Hao Xu, Morgane Rouault, I. Tirosh, Hacohen Nir, Fei Chen, Omer Yilmaz, Jatin Roper, O. Rozenblatt-Rosen, Mor Nitzan, A. Regev
          </td>
          <td>2025-12-12</td>
          <td>eLife</td>
          <td>0</td>
          <td>195</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7a3ed4eb95927059b414292cf9cce743a10321" target='_blank'>
              Spatiotemporal Mapping of Tertiary Lymphoid Structure Heterogeneity Shapes Immune Niches and Clinical Outcomes in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Liaoliao Gao, Jie Mei, Libing Hong, Yuzhi Jin, Jinlin Cheng, Xuqi Sun, Chuan Liu, Bin Li, Xiaomeng Dai, Bo Lin, Yajie Sun, Peng Zhao, Minshan Chen, Rongping Guo, Shan Xin, Jingping Yun, Inmaculada Martínez-Reyes, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Greater understanding of differential therapeutic sensitivity, specifically to immunotherapy, in small-cell lung cancer (SCLC) is required. We explored SCLC heterogeneity through integrated molecular characterization of tumor tissue samples from 159 treatment-naive patients, utilizing genetic, epigenetic, transcriptional, and proteomic profiling, immunohistochemistry staining for multiple biologically relevant markers including transcriptional subtype-defining proteins, and spatial immune profiling using multiplex immunofluorescence. Multi-omics analysis confirmed high heterogeneity across/within neuroendocrine and non-neuroendocrine subtypes. Methylomics analysis identified four methylome clusters that may enhance subtype prediction, prognosis, and longitudinal monitoring of subtype evolution. Immunohistochemistry analysis showed high MHC-I expression in non-neuroendocrine subtypes, which have greatest potential benefit from adding immunotherapy to chemotherapy; high DLL3 expression associated with neuroendocrine subtypes and an immune-cold tumor microenvironment. Multiplex immunofluorescence demonstrated associations of MHC-I with spatial arrangement and phenotypic features of immune cells in the tumor microenvironment of high-MHC-I-expressing SCLC, providing mechanistic rationale for MHC-I as a potential biomarker of immunotherapy response. This multimodal profiling analysis provides further insights into the biologic complexity of SCLC and highlights potential therapeutic vulnerabilities of distinct disease subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7c241dbb9b8047ecee9d971f81cea0a1e193e3" target='_blank'>
              Multi-omic profiling provides insights into the heterogeneity, microenvironmental features, and biomarker landscape of small-cell lung cancer
              </a>
            </td>
          <td>
            Mingchao Xie, Miljenka Vuko, Shashank Saran, Siyu Liu, Andrew G Chambers, Hana Baakza, Helen K Angell, Felicia Ng, C. Gay, R. Cardnell, F. Segerer, Alma Andoni, Jaime Rodriguez-Canales, Paul M Waring, Markus Schick, J. C. Barrett, L. Byers, Giulia Fabbri
          </td>
          <td>2025-12-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Background : In contrast to most solid tumors, high immune cell infiltration in clear cell renal cell carcinoma (ccRCC) is associated with poor patient prognosis. The biological mechanisms underlying this paradox remain unclear, particularly regarding tumor cell– microenvironment interactions promoting local invasion and recurrence. This study aimed to identify spatially resolved tumor, immune, and stromal features that define aggressive phenotypes in localized ccRCC. Methods : Multiplex immunofluorescence was performed using a 33-marker panel on 1,728 multi-region tissue cores from 435 surgically treated patients with localized ccRCC. Samples systematically included tumor centers, invasive borders, and adjacent benign tissue. Single-cell analyses quantified immune, stromal, endothelial, and epithelial cell populations within their spatial context. Results : Spatially resolved profiling uncovered a highly aggressive tumor subtype distinguished by fibroblast activation protein (FAP) expression on tumor epithelial cells, a marker typically associated with stromal cells. Tumor-cell-specific FAP expression characterized an epithelial-to-mesenchymal transition (EMT)-like state and was spatially associated with profound immunosuppression, marked by enrichment of regulatory T cells, exhausted CD8+ T cells, and M2-like macrophages, particularly at the invasive border. Tumor-cell FAP promoted invasion and independently predicted significantly poorer recurrence-free survival (RFS), even in early-stage disease (multivariable Cox p = 0.022 for pT1–2), surpassing established biomarkers such as PD-L1 in capturing aggressive biological features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42470b1512d22d2033f3b6dc6829a3e316db8519" target='_blank'>
              Tumor cell FAP orchestrates EMT and immune suppression in aggressive localized ccRCC
              </a>
            </td>
          <td>
            T. Pellinen, L. Luomala, K. Mattila, A. Hemmes, K. Välimäki, Mariliina Arjama, Oscar Brück, L. Paavolainen, Elisa Kankkunen, H. Nísen, P. Järvinen, Leticia Castillon, Sakari Vanharanta, P. Vainio, Olli Kallioniemi, Panu M. Jaakkola, T. Mirtti
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The tumor immune microenvironment (TIME) is closely involved in tumor initiation, malignant progression, immune escape, and response to immunotherapy. With the continued development of high-throughput sequencing technologies, transcriptomic approaches have become essential for examining the cellular and molecular features of the TIME. Bulk RNA sequencing offers tissue-level gene expression profiles and allows the estimation of immune cell composition through computational deconvolution. Single-cell RNA sequencing provides finer resolution, revealing cellular heterogeneity, lineage relationships, and functional states. Spatial transcriptomics (ST) retains the native anatomical context, making it possible to localize gene expression patterns and cell–cell interactions within intact tissues. These approaches, when considered together, have shifted TIME research from averaged measurements toward a more detailed and mechanistic understanding. This review summarizes the principles, applications and limitations of bulk, single-cell and spatial transcriptomic methods, highlighting emerging strategies for integrative analysis. Such multi-scale frameworks are increasingly important for studying immune dynamics and may contribute to the development of more precise biotechnological and immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a95188583167d2d5ed84b42177b48c6d850371a9" target='_blank'>
              Multi-Scale Transcriptomics Redefining the Tumor Immune Microenvironment
              </a>
            </td>
          <td>
            Jing Sun, Yingxue Xiao, Lingling Xie, Dan Qin, Yue Zou, Yingying Liu, Yitong Zhai, Minyi Zhang, Tong Li, Youjin Hao, Bo Li
          </td>
          <td>2026-01-15</td>
          <td>BioTech</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="While single-cell technologies have advanced bladder cancer research, the microenvironment of lymph node metastases and its role in progression remain unclear. We aimed to elucidate dynamic tumor-microenvironment interactions across metastatic and primary sites, identify poor-prognosis features, and uncover novel therapeutic targets. We profiled matched single-cell and spatial transcriptomic samples from lymph node-metastasized bladder cancer to construct a high-resolution atlas. Bioinformatic analyses were used to inferred ligand-receptor interactions, regulatory networks, and spatial co-localization. Integration with multiple prognostic cohorts revealed key cell subpopulations linked to tumor biology and poor outcomes. Our atlas included 32 samples, 61 cell subtypes, key ligand signals (e.g., TGFB1, OSM, IFNG), 343 transcription factors, 188 co-localization events, over 1,000 spatially validated interactions, and 21 cell niches. Tumor-driven convergence was observed across distinct microenvironments. Notably, a pericyte-like myofibroblast subtype, myoCAF_PLXDC1, was enriched in malignant tissues, associated with poor prognosis, and exhibited enhanced angiogenic but reduced chemotactic activity compared to myoCAF_CCL21. Its phenotype was linked to intracellular regulators (PRRX1 and MXD4) and extracellular signals (IFNG and TGFB1). Spatially, myoCAF_PLXDC1 co-localized with ITGA8+ endothelial cells and enriched in tumor regions. This study offers a comprehensive resource for lymph node-metastasized bladder cancer and identifies PLXDC1+ fibroblasts co-localized with ITGA8+ ECs as potential components of a tumor-specific vascular niche and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d7c9cdf4d9cefce93639f3970e5da305a8fc5c" target='_blank'>
              Single-cell and spatial atlas unveil tumor-specific microenvironment convergence and a prognosis-associated PLXDC1+ myofibroblast population in metastatic bladder cancer
              </a>
            </td>
          <td>
            Ziwei Wang, Jiaying Miao, Maoyu Wang, Yidie Ying, Zhensheng Zhang, Shuxiong Zeng, Chuanliang Xu
          </td>
          <td>2025-12-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Understanding how tissues remodel in response to perturbations requires computational tools that can untangle condition-specific changes from the conserved tissue architecture. We present Haruka, a spatially aware contrastive learning framework that identifies salient (condition-specific) and background (shared) spatial domains across tissue slices and experimental conditions. Haruka integrates contrastive variational inference with an auxiliary microenvironment reconstruction task, enabling the model to learn spatial-context-informed embeddings that capture both perturbation effects and local neighborhood context. Through benchmarking on simulated and real datasets, Haruka outperforms state-of-the-art methods in detecting spatially heterogeneous responses. Applied to diverse spatial omics platforms, Haruka distinguished immunotherapy responders in melanoma, traced fibrosis progression in human lung tissue, and mapped treatment-resistant microenvironments in KRASG12D-mutated lung cancer. Thus, Haruka provides a generalizable framework for spatial contrastive analysis, enabling systematic dissection of tissue organization, cellular plasticity, and microenvironmental remodeling across disease, development, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa467a4fed6deacca0c759e8099e8c763a30e66" target='_blank'>
              Haruka Resolves Perturbation Response Heterogeneity in Spatial Cell Niches
              </a>
            </td>
          <td>
            Yan Cui, J. Blandin, K. Weiskopf, Na Sun
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Intratumour heterogeneity (ITH) is one of the key characteristics of cancer and is closely associated with patient prognosis, treatment resistance, and tumor metastasis. Nevertheless, the study of ITH in hepatocellular carcinoma (HCC) remains limited. Methods: The present study elucidated the influence of ITH on the tumor microenvironment (TME) in HCC. We applied Non-negative Matrix Factorization (NMF) analysis to a cohort of 78 single-cell RNA sequencing (scRNA-seq) HCC samples to systematically characterize ITH. Furthermore, by integrating spatial transcriptomics (ST) data from five HCC patients, we comprehensively analyzed the spatial organization and functional properties of distinct niches within HCC. We conducted a detailed analysis of the cell-type co-localization relationships within the TME and constructed a comprehensive atlas of HCC spatial organization. Results: We observed a co-localization relationship between hypoxia tumor cells, plasmalemma vesicle-associated protein (PLVAP+) endothelial cells (EC), and vascular endothelial growth factor A (VEGFA+) cancer-associated fibroblasts (CAF), suggesting a key role for hypoxia tumor cells in VEGFA+ CAF transformation and tumor angiogenesis. We identified a unique boundary region enriched with dendritic cells1 (DC1), interferon-expressing tumor cells, lymphatic EC, C–X–C Motif Chemokine Ligand 10 (CXCL10+) macrophages (Mac), and secreted phosphoprotein 1 (SPP1+) Mac located between the tumor-infiltrating immune cells and tumor regions. Furthermore, we found that CXCL10+ Mac and SPP1+ Mac, despite co-localizing in the boundary region, exhibit distinct functions, which may be attributed to their unique spatial locations, with the former being closer to the immune-infiltrated region and the latter more proximal to the tumor area. Conclusions: Our study highlights the critical role of spatial interactions between tumor cells and the microenvironment in HCC. The findings offer new insights into ITH and underscore the importance of spatial organization in understanding cancer biology and designing future precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e826ac6739dc6ba19b8b336aa7292a375efd8d3e" target='_blank'>
              Integrative Analysis of Single-Cell and Spatial Transcriptomics Reveals Intratumor Heterogeneity Shaping the Tumor Microenvironment in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-12-01</td>
          <td>Livers</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cervical cancer (CC), a leading cause of cancer-related deaths among women worldwide, is primarily driven by high-risk human papillomavirus (HPV) infections and comprises two major histological subtypes: adenocarcinoma (AC) and squamous cell carcinoma (SCC). Despite advances in prevention and treatment, the molecular and cellular heterogeneity of these subtypes poses significant challenges to achieving optimal clinical outcomes. Here, we integrate single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) to dissect the cellular and spatial heterogeneity of AC and SCC, uncovering distinct tumor microenvironment (TME) dynamics that underlie their divergent clinical behaviors. Our scRNA-seq analysis reveals that AC is enriched in epithelial cells, while SCC exhibits a more immunogenic TME with elevated plasma cells and NK/T cells. Spatial transcriptomics further highlights robust interactions between CD8 + T cells and epithelial subtypes in SCC, contrasting with the stromal-rich, immune-cold phenotype of AC. We identify subtype-specific immune and stromal features, including ICOS+ Tregs, IDO1+ cancer-associated fibroblasts (CAFs), and PLVAP+ endothelial cells, which may drive immune evasion, angiogenesis, and metastasis. These findings provide a comprehensive framework for understanding CC heterogeneity and offer actionable insights for developing subtype-specific therapeutic strategies, such as combining immune checkpoint inhibitors with stromal-targeting agents. This study underscores the potential of spatial multi-omics technologies to advance precision oncology and improve outcomes for cervical cancer patients. Single-cell and spatial transcriptomics uncovered a stark contrast in the TMEs of cervical cancer subtypes: squamous cell carcinoma is more immunogenic, whereas adenocarcinoma is stromal-rich and immune-cold.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b0c553f54a8eda4d3dc1abe42f3db69a41cfe1" target='_blank'>
              Single-cell and spatial transcriptomic profiling reveals distinct tumor microenvironment dynamics in cervical adenocarcinoma and squamous cell carcinoma
              </a>
            </td>
          <td>
            Yaqin Wu, Bei Wei, Zhentong Wei, Hongyan Wang, Yunyun Liu, Dongdong Xu, Xiaokang Li, Liyuan Guo, Huaiwu Lu, Huiling Shang
          </td>
          <td>2025-12-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba1256bc5be1c15c5a52165e8dbea8c1e3c75014" target='_blank'>
              Integrative Single-cell and Spatial Transcriptomic Analysis of Osteosarcoma Reveals Conserved and Distinct Ecosystems Across Sites and Species
              </a>
            </td>
          <td>
            Yogesh Budhathoki, Matthew V. Cannon, T. A. McEachron, Anand G. Patel, Matthew J Gust, Jaime F Modiano, D.T. Ammons, Kathryn E Cronise, M. Macchietto, A. Sarver, Ruben Dries, B. Gryder, Daniel P. Regan, Heather L. Gardner, Ryan D. Roberts
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern, and a high relapse rate. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, where malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, we identify pericytes as the most active paracrine signaling hub within the tumor parenchyma. Their depletion through genetic engineering results in accelerated tumor progression and shortened survival. Mechanistic studies reveal that pericyte deficiency remodels the endothelium and impacts the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, the pericyte-deprived endothelium recruits perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages express Hepatocyte Growth Factor, which reinforces activation of its receptor tyrosine kinase MET on GBM cells harboring a pronounced mesenchymal subtype driven by the key phenotypic regulator Fosl1. Indeed, orthotopic implantation of MET-expressing GBM cells corroborates their superior tumor-initiating and invasive capabilities. Thus, pericytes represent critical modulators of GBM development by orchestrating a tumor-suppressive microenvironment, highlighting the importance of their preservation in therapy. Improved understanding of the tumor ecosystem in glioblastoma is critical for developing new treatment strategies for the disease. Here, the authors identify pericytes as an active paracrine signaling hub within the tumor parenchyma that orchestrates a tumor-suppressive microenvironment, suggesting pericyte preservation as a key feature of future therapeutic regimens for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c09a05790107e087a632df06714ceec3176e20b4" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, Mehrnaz Safaee Talkhoncheh, B. Phung, E. Cordero, R. Rosberg, E. Johansson, G. Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafddf4eb7b4f74e9d7699b13e08df2afba97c1e" target='_blank'>
              Pan-cancer tumor classification by a holistic tumor microenvironment atlas
              </a>
            </td>
          <td>
            Shishang Qin, Xiao Du, Jinhu Li, Nan Jiang, Tian Diao, Yufei Bo, Qinhang Gao, Liangtao Zheng, Xinnan Ling, Qianqian Gao, Xiangjie Li, Sen Gao, Fei Tang, Wenjie Zhang, Chenwei Li, Peihong Fang, Linna Zhu, Dongfang Wang, Zemin Zhang
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be57704ac8a67d3c2570360452cb391401b265a" target='_blank'>
              A Comprehensive Treatment-Induced Resistance Atlas of Glioblastoma Reveals a Fibrotic Niche Shielding the Tumor from Immunotherapy
              </a>
            </td>
          <td>
            Fei Wang, Run Huang, Hongyi Ling, Yuhan Bai, Chen Yang, Guozhen Zhao, Xin Wu, Wenqian Cao, Yue Lu, Yining Zhang, Deng-Feng Lu, Youjia Qiu, Juyi Zhang, Bixi Gao, Chao Ma, Hanhan Dang, Yulian Zhang, Yanbo Yang, Ting Sun, Zhouqing Chen, Zhong Wang
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Understanding how prostate cancers evolve toward treatment resistance requires experimental systems that both recreate the phenotypic diversity of the disease and resolve lineage dynamics at high resolution. We developed complementary platforms to meet these challenges. The Prostate Mouse Platform (ProMPt) is a genetically scalable organoid- and allograft-based system encompassing over 150 combinatorial genotypes, enabling reconstruction of the principal phenotypic states of prostate cancer, from androgen-responsive tumors to castration-resistant double-negative, hybrid/mixed, and neuroendocrine-like variants. Through multi-omic and functional analyses, ProMPt models capture the context-specific vulnerabilities that arise during adaptation to androgen deprivation or growth-factor loss. To map and track this evolving phenoscape, we designed XCODE, a universal, multiplatform barcoding technology that enables clonal tracing across sequencing-, cytometry-, and imaging-based modalities (CyTOF, Phenocycler, Xenium). XCODE integrates lineage information across bulk, single-cell, and spatial layers, linking histology, clonal dynamics, phenotype, and microenvironmental context. Together, ProMPt and XCODE provide a powerful preclinical framework to reconstruct, resolve, and perturb tumor evolution within a controlled yet clinically relevant landscape. This integrated approach reveals how defined genetic combinations reshape lineage plasticity, signaling networks, and therapeutic responses, exposing new intervention points to prevent or redirect adaptive evolution in advanced prostate cancer.



 Marco Bezzi. From reconstruction to resolution: platform approaches to engineer the prostate cancer phenoscape and map its evolution [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A002.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3245085c50a80831901ee082ba85bf39b096b337" target='_blank'>
              Abstract A002: From reconstruction to resolution: platform approaches to engineer the prostate cancer phenoscape and map its evolution
              </a>
            </td>
          <td>
            Marco Bezzi
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Although well-differentiated thyroid carcinoma (WDTC) is characterized by a robust treatment response, aggressive subtypes, such as anaplastic thyroid carcinoma (ATC), remain highly lethal. To understand thyroid cancer evolution in both children and adults, we analyzed single-cell transcriptomes of 423,733 cells from 81 samples and spatially resolved key tumor and microenvironment populations across 28 tumors with spatial transcriptomics, including rare and unique composite WDTC/ATC tumors and pediatric diffuse sclerosing thyroid carcinomas. Additionally, we identified gene signatures of stromal cell populations in 5 large thyroid cancer bulk RNA-sequencing cohorts. Through this multi-institutional effort, we defined a population of POSTN+ myofibroblast cancer-associated fibroblasts (myCAFs) that are intimately associated with invasive tumor cells and correlate with poor prognosis, lymph node metastasis, and disease progression in thyroid carcinoma. We also revealed a population of inflammatory CAFs that are distant to tumor cells and are found in the inflammatory stromal microenvironment of autoimmune thyroiditis. Together, our study provides spatial profiling of thyroid cancer evolution in samples with mixed WDTC/ATC histopathology and identifies a prognostic myCAF subtype with potential clinical utility in predicting aggressive disease in both children and adults.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9674219bbd66cbdd5e33b2f92b59cb54d20e94cd" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations.
              </a>
            </td>
          <td>
            M. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey P. Caroland, Megan L. Tigue, H. Hartmann, Jean-Nicolas Gallant, Courtney J. Phifer, Andres A Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H. Belcher, K. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2026-01-09</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27e17b141d07dfd3357f3ee906c1f3a2792dd3c" target='_blank'>
              SigDyn: single-cell mutational signature dynamics
              </a>
            </td>
          <td>
            , 
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ee6608293723ba24e369164c3802a52bef8a361" target='_blank'>
              Redefining the topology of the human bone marrow using augmented spatial transcriptomic analysis
              </a>
            </td>
          <td>
            Rosalin Cooper, Emily Thomas, Muhammad Dawood, H. Ryou, Anna M. Sozanska, C. Pescia, Oliver McCallion, Muskaan Gupta, Renuka Teague, J. Hester, F. Issa, Dan J Woodcock, B. Psaila, Adam J. Mead, Jens Rittscher, D. Royston
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18e77f40b4649c94b742354f1776a0273d7ae6d5" target='_blank'>
              A Hallmark-Integrated, Agent-Based Framework for Intratumor Heterogeneity in Melanoma Evolution
              </a>
            </td>
          <td>
            Khola Jamshad, Trachette L. Jackson
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bdb3fc96051ec080fed6bcf88bde02391e145c9" target='_blank'>
              Integrative spatial multi-omics reveals prognostic tumor niches in female genital tumors
              </a>
            </td>
          <td>
            Tian Xu, Siyu Lin, Yuling Wu, Yufeng He, Zongxu Zhang, Yanping Zhao, Peng Zhang, Luyang Zhao, Wen Yang, M. Ye, Rui Zhang, Liang Wen, Pengfei Ren, Ce Luo, Zhe Zhang, Zexian Zeng, Yuanguang Meng
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb72dd83254c43d1127b19ea69b553e5fd51c4d" target='_blank'>
              TMPO promotes cellular dissemination and metastasis in circulating tumor cells
              </a>
            </td>
          <td>
            A. Giacobbe, Aleksandar Z Obradovic, Jinqiu Lu, Soonbum Park, Carlos Pedraz-Valdunciel, G. Fanelli, Aunika Zheng, Jaime Y Kim, Maya Stella Dixon, Jung Seung Nam, Florencia Picech, Caroline J. Laplaca, R. Virk, Matteo Di Bernardo, Alexander Chui, J. M. Arriaga, Stephanie N Afari, Francisca Nunes de Almeida, Min Zou, Helen Garcia, Brian D. Robinson, Hongshan Guo, S. Maheswaran, D. Haber, D. Miyamoto, D. Nanus, S. Tagawa, Tian Zheng, Massimo Loda, I. I. C. Chio, Michael M. Shen, P. Giannakakou, Andrea Califano, Peter A. Sims, Cory Abate-Shen
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Lineage plasticity, the ability of cancer cells to alter their differentiated state through transcriptional and epigenetic reprogramming, has emerged as a key mechanism of therapeutic resistance across cancers. This adaptive process can manifest in multiple ways, including epithelial–mesenchymal transition, acquisition of stem-like features, and histological transformation, the most striking and clinically apparent example. In EGFR-mutant lung adenocarcinoma (LUAD), lineage plasticity is increasingly recognized as a prevalent mechanism of acquired resistance to tyrosine kinase inhibitors (TKIs). Among its visible manifestations, histologic transformation into small-cell lung cancer (SCLC) is the most frequent, while squamous transformation and other phenotypic shifts also occur. Transformed tumors typically retain the initiating EGFR mutation but lose EGFR dependence, acquire neuroendocrine features, and display aggressive clinical behavior with poor clinical outcomes compared with both de novo SCLC and non-transformed LUAD. Recent studies show that plasticity arises through combined genomic, transcriptomic, and epigenetic reprogramming, often foreshadowed by molecular alterations before overt histological change. Spatial and single-cell profiling reveal heterogeneous trajectories and intermediate states, while functional models and multi-omics approaches have begun to identify therapeutic vulnerabilities distinct from both de novo EGFR-mutated SCLC and classical EGFR-mutated LUAD. Thus, lineage plasticity, whether manifested as histologic transformation or through more subtle epigenetic reprogramming, represents a formidable resistance mechanism in NSCLC. Defining its molecular basis and temporal dynamics will be essential for early detection, prognostication, and the development of tailored therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99eda567a0f2b2f5e4b0555a0ae32377856dcb42" target='_blank'>
              Lineage Plasticity and Histologic Transformation in EGFR-TKI Resistant Lung Cancer
              </a>
            </td>
          <td>
            Li Yieng Eunice Lau, A. Skanderup, Aaron C. Tan
          </td>
          <td>2025-12-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Identifying drivers of cancer progression to guide treatment selection is hindered by our limited understanding of tumor heterogeneity and its impact on tumor evolution. Here, we delineate the phenotypic variability across ~300,000 cells collected from multiple tumor loci in primary prostate and matched locoregional metastases using single-cell chromatin accessibility and gene expression sequencing. We find inter-patient heterogeneity to be confined to malignant populations. Within individual tumor loci, we see phenotypic heterogeneity among malignant cell populations despite a shared clonal genotypic architecture. We also observe that malignant cell populations disseminating to locoregional lymph nodes mirror the clonal architecture and phenotypic heterogeneity across primary tumor loci, while shifting from canonical prostate-cancer states to non-canonical inflammatory-like states. Our findings suggest a bottleneck imposed during the dissemination process, funneling prostate cancer cells toward an inflammatory-like cell state. These insights into the interplay between phenotypic identity and clonal architecture refine our understanding of prostate cancer progression and suggest that convergence of cancer cells towards an inflammatory-like state underlies dissemination to lymph nodes, offering a critical framework for future studies into prostate cancer metastatic potential. Understanding tumor heterogeneity and its impact on prostate cancer progression remains elusive. Here, single nucleus snATAC and snRNA sequencing of a multi-loci sampled cohort of advanced prostate cancer patients identifies an inflammatory-like state underlying metastatic dissemination to the lymph nodes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b9d84066de3071bee8ea7d75cdd27aba04e0e02" target='_blank'>
              Prostate cancer cells converge to an inflammatory-like state upon metastatic dissemination
              </a>
            </td>
          <td>
            Tina Keshavarzian, K. Furlano, Giacomo Grillo, Lisanne Mout, C. Arlidge, Faizan Hasan, Ankita Nand, Migle Mikutenaite, Evdoxia Karadoulama, Ashish Goyal, Elisabeth L Pezzuto, Jessica Heilmann, Jakub Sykora, S. Minner, Thorsten Schlomm, G. Sauter, R. Simon, H. He, Joachim Weischenfeldt, C. Plass, Clarissa Gerhäuser, M. Lupien
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb4486dfc086ad91b487a1c5269e719293310a3" target='_blank'>
              Machine perfusion and single-cell spatial transcriptome mapping identifies novel immune escape mechanisms in colorectal cancer liver metastasis
              </a>
            </td>
          <td>
            Peter Kok-Ting Wan, Ranchu Cheng, David Johnson, Carl Lee, Shihong Wu, Areeb Mian, Ahmet Hazini, Sorayya Moradi, Kate Friesen, Flurin Caviezel, Syed Hussain Abbas, Hatem Sadik, A. Lakha, Keaton Jones, Girishkumar Kumaran, M. Bottomley, Rob Jones, Mark C. Coles, R. Bashford-Rogers, Constantin C. Coussios, Len Seymour, Robert Carlisle, Kerry Fisher, Alex Gordon-Weeks
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, M.-R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICI) have demonstrated prolonged efficacy in certain melanoma patients, yet a significant portion of patients do not experience clinical improvement, with the mechanisms underlying this resistance still not fully understood. Using established cell markers, we partitioned the single‐cell transcriptome into clusters, finding a notable link between NK cells and patient response to immunotherapy. We further identified four distinct subpopulations of NK cells, profiling marker gene sets and unique biological functions associated with each subpopulation. This analysis provides insights into the trajectory of NK cell development and differentiation, along with identifying the transcription factors driving these processes. The study pinpointed NK cluster 01 as pivotal in influencing patient sensitivity and prognosis during immunotherapy. Single‐cell transcriptome and spatial transcriptomics (ST) analysis revealed the proximity of NK cluster 01 cells to melanoma cells, hinting at a potential regulation of cell‐cell interaction via the IFN‐II signaling pathway network. ST analysis revealed the spatial arrangement and interaction of NK cluster 01 cells with melanoma cells. This study explores the feasibility of targeting NK cluster 01 cells with small molecule drugs via molecular docking, offering a promising approach to bolster the clinical utility of NK cell therapy. We comprehensively analyze the heterogeneity of NK cells within melanoma, elucidate the potential regulatory interactions between NK cells and other microenvironmental components, and establish a basis for the future clinical utilization of distinct NK cell subsets as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fbdc1aaba32b7600dd3221ae61c5e31ee96bd3" target='_blank'>
              Integrating Single‐Cell and Spatial Transcriptomics Reveals NK Cell Subpopulations Associated With Immunotherapy for Melanoma
              </a>
            </td>
          <td>
            Zhicheng Hu, Yongfei Chen, Hao Yang, Qiuming Pan, Hongrui Li, Shuting Li, Junxi Wang, Yudi Huang, Guanglong Huang, Shanqiang Qu
          </td>
          <td>2025-12-01</td>
          <td>Smart Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Bladder cancer (BCa) is a prevalent malignancy and major cause of cancer-related mortality in men, with clinical outcomes still varying despite advances in personalized treatments. High inter- and intra-tumor heterogeneity significantly contributes to this variability. While traditional high-throughput sequencing has provided insights into BCa mechanisms, driver genes, and clinical strategies, it falls short in completely elucidating cellular heterogeneity. Recently, single-cell sequencing (SCS) technologies have substantially enhanced the detection of tumor heterogeneity by improving sensitivity, accuracy, and efficiency. Single-cell transcriptome sequencing offers unbiased, high-resolution analysis of gene expression patterns at the single-cell level, offering essential insights into BCa pathogenesis. This article reviews advancements in SCS technology and its applications in evaluating tumor heterogeneity, the tumor microenvironment, metastasis, and treatment resistance, offering new perspectives for future BCa research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/128cca3d1edd8a7877ad22a89e79521ff40bea75" target='_blank'>
              Single-cell sequencing in bladder cancer: new insights from tumor cell diversity to individualized treatment strategies
              </a>
            </td>
          <td>
            Cen Liufu, Shuai Ye, Cong Yin, Tao Zhu, Jiahao Jiang, Yan Wang, Bentao Shi
          </td>
          <td>2025-12-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, Christin-Therese Müller, Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faf55c0c7d18000c8ba092357d0dd95271e7c35c" target='_blank'>
              Spatial dissection of ADC/RPT targets defines therapeutic opportunities in rhabdoid tumors
              </a>
            </td>
          <td>
            N. Reitsam, V. Fincke, Maria Daniela Hernandez Ramirez, M. Mucha, Eva Sipos, Lisa Siebenhüter, J. Enke, Maurice Loßner, C. Vokuhl, Constantin Lapa, M. Hasselblatt, Michael C. Frühwald, B. Märkl, Pascal D. Johann
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the leading causes of cancer-related morbidity and mortality worldwide. Tumor sidedness, distinguishing left- (LCRC) and right-sided (RCRC) cancers, has emerged as a critical clinical determinant, influencing patient prognosis and therapeutic response. However, the cellular and molecular mechanisms underlying these differences remain poorly understood. Recent advances in single-cell RNA sequencing (sc-seq) provide high-resolution insights into CRC heterogeneity, revealing distinct tumor, immune, and stromal cell populations and their context-specific interactions. In this review, we synthesize sc-seq studies that dissect the molecular programs driving progression, therapy resistance, and metastasis in CRC. We highlight malignant subclusters characterized by metabolic reprogramming and spatially organized oncogenic signaling; specialized immune cell states, including macrophage subsets, exhausted T cells, and mast cells, that shape tumor immunity; and stromal elements such as cancer-associated fibroblasts and endothelial tip cells that remodel the extracellular matrix, promote angiogenesis, and foster immune evasion. Importantly, sc-seq demonstrates that LCRC and RCRC represent distinct multicellular ecosystems with differential immune recruitment and stromal signaling, underscoring the need for sidedness-informed therapeutic strategies. We propose that future interventions should target cell-cell communication networks and spatially defined tumor–microenvironment interactions to overcome heterogeneity and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03b9e3bca233f1317bcd2e2e43fb69a5fedfb984" target='_blank'>
              Sidedness matters: single-cell perspectives on left- and right-sided colorectal cancer
              </a>
            </td>
          <td>
            M. Yanova, Diana Maltseva, Alexander Tonevitsky
          </td>
          <td>2025-12-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor cell plasticity plays a key role in the progression of malignant neoplasms and therapy resistance, yet its spatial dynamics remain poorly understood. This study investigates the spatial distribution of epithelial–mesenchymal transition (EMT) and stemness, and their prognostic relevance in lung adenocarcinoma (LUAD). Single‐cell RNA sequencing was performed on 10 tumor fragments from three regions (core, core‐adjacent, and edge) of a single LUAD using a SURFSeq 5000 platform. Stemness potential was identified by CytoTRACE2, EMT by UCell and the Hallmark_epithelial_mesenchymal_transition signature, and tumor cells associated with disease prognosis by Scissor. LUAD displayed pronounced spatial heterogeneity of tumor cell plasticity. Stemness potential was heightened at the edge, while EMT was most prominent in the core. The correlation between EMT and stemness was slightly higher at the edge, though it remained negligible (R = 0.17, p = 5.5e−7). Higher EMT scores were observed in tumor cells associated with poor overall survival (r = 0.433, p < 2e−16), whereas stemness was prevalent in tumor cells associated with poor progression‐free survival (r = 0.251, p = 7.15e−6). Tumor cells associated with poor prognosis were enriched with the MYC‐targets v1 signature with PCBP1 and PA2G4 as the top contributing genes. Immunohistochemical data showed that PCBP1 and PA2G4 proteins predominantly localized within tumor cells of LUAD. Taken together, these findings highlight the spatial heterogeneity of tumor cell plasticity features in LUAD and uncover biomarkers associated with poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badf7545e441ad3330c93dcdb325456db2158597" target='_blank'>
              Spatial Dynamics of Tumor Cell Plasticity in Lung Adenocarcinoma Revealed by Region‐Specific Single Cell Transcriptomics
              </a>
            </td>
          <td>
            A. Khozyainova, Maxim E. Menyailo, Vera G. Subrakova, Dmitry M. Loos, P. Iamshchikov, Anton A Fedorov, M. S. Tretyakova, Ustinia A Bokova, Evgeny O Rodionov, Sergey V Miller, Evgeny B Topolnitsky, D. V. Piankov, Sergei A Korostelev, Evgeny V. Denisov
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a heterogeneous malignancy shaped by genetic alterations and immune microenvironmental interactions. While single-cell RNA sequencing (scRNA-seq) has uncovered diverse epithelial and T cell subsets, the spatial architecture of their crosstalk remains poorly understood. We integrated scRNA-seq and spatial transcriptomics data from 36 CRC patients to comprehensively characterize epithelial and T cell heterogeneity, differentiation dynamics, and intercellular communication networks. Data preprocessing and integration were conducted using Seurat and Harmony. Cell trajectory inference was performed via CytoTRACE and Slingshot. Ligand-receptor signaling was assessed using CellChat, while spatial mapping was achieved using CellTrek. Nine epithelial and eight T cell subpopulations were identified, each exhibiting distinct transcriptional states, CNV burdens, and pseudotime trajectories. CD8⁺ T_GZMK cells displayed tumor-specific activation signatures and engaged in enriched communication with epithelial subsets through CEACAM, APP, and MIF signaling pathways. Spatial transcriptomics confirmed the in situ colocalization of CD8⁺ T_GZMK cells with PTP4A3⁺ epithelial cells in tumor regions, revealing spatially organized epithelial–immune niches. These interactions suggest potential mechanisms of immune modulation and tumor progression. This study provides an integrative single-cell and spatial atlas of CRC, revealing structured epithelial–T cell communication and spatial architecture within the tumor microenvironment. Our findings offer novel insights into immune-epithelial crosstalk and identify signaling pathways that may serve as therapeutic targets or biomarkers for CRC precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6c4f2ec4971acec7d7b9addcd8348e71b0ec1ff" target='_blank'>
              Decoding epithelial-T cell interactions in colorectal cancer through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Rentao Yu, Xinyu Lu, Zhuangzhuang Yue
          </td>
          <td>2025-11-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d29acb3217c08dff3be7e7d47281f950232e5a4" target='_blank'>
              Identification of an epigenetically and phenotypically distinct peritumoral glioblastoma cell population linked to inferior patient outcome
              </a>
            </td>
          <td>
            I. Neves, Xi Lu, Veera Jokinen, Francesco Latini, N. Maturi, A. Sundström, Yonglong Dang, Irem Uppman, Tobias Bergström, Pengwei Xing, M. Ryttlefors, Xingqi Chen, F. Swartling, L. Uhrbom
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Single-cell sequencing (scRNA-seq) has emerged as a pivotal technology for deciphering the complex cellular heterogeneity and tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC), positioning it as a critical tool for informing novel therapeutic strategies. This review explores how scRNA-seq reveals diverse cellular subpopulations and their functional roles within the PDAC TME, including malignant epithelial cells with transitional phenotypes, heterogeneous cancer-associated fibroblasts (CAFs), functionally distinct immune cells such as tumor-associated neutrophils (TANs) and macrophages (TAMs), and actively participating neural components like Schwann cells. These cellular constituents form specialized functional units that drive tumor progression, immune evasion, neural invasion, and therapy resistance through metabolic reprogramming, immunosuppressive signaling, and cellular plasticity. The review further examines technological advances in single-cell sequencing from 2023 to 2025, focusing on sample preprocessing innovations, multi-omics integration (combining transcriptomics with epigenomics and proteomics), spatial resolution enhancements, and customized computational tools that address PDAC-specific challenges. Clinically, single-cell sequencing enables precise cellular subtyping, identification of novel biomarkers, and development of personalized therapeutic approaches, including combination therapies targeting specific cellular subpopulations and their interactions. Despite these advances, significant challenges remain in standardizing clinical applications such as liquid biopsy for early detection and tumor microenvironment assessment for diagnostic staging, validating biomarkers like CLIC4, GAS2L1, Cytokeratins, Vimentin and N-cadherin in circulating tumor cells, and comprehensively integrating multi-omics data. Future research focusing on both technology refinement and biological validation will be essential for translating single-cell insights into improved diagnostic and therapeutic outcomes for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9576dbb6fe77bb5f04f2ad67b474589557936" target='_blank'>
              Single-Cell Sequencing Unravels Pancreatic Cancer: Novel Technologies Reveal Novel Aspects of Cellular Heterogeneity and Inform Therapeutic Strategies
              </a>
            </td>
          <td>
            Keran Chen, Zeyu Chen, Jinai Wang, Mo Zhou, Yun Liu, Bin Xu, Zhi-Jing Yu, Yiming Li, Guanhu Yang, Tiancheng Xu
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Immune dysfunction in cancer is enacted by multiple programs, including tumor cell-intrinsic responses to distinct immune subpopulations. A subset of these immune evasion programs can be systematically recapitulated through direct tumor-immune interactions in vitro. Here, we present an integrated, high-throughput single-cell CRISPR screening framework focused on the protein kinome for mapping the tumor-intrinsic regulation of T cell-driven immune pressure in glioblastoma (GBM). We combine pooled CRISPR interference and activation (CRISPRi/a) with immune-matched NY-ESO-1 antigen-specific allogeneic GBM-T cell co-culture and massively multiplexed single-cell transcriptomics to systematically quantify how genetic perturbation reshapes baseline tumor state and adaptive responses across graded effector-to-target ratios. We further leverage deep generative models for analyzing pooled CRISPR screens to decipher the effects of genetic perturbations on the mechanisms of tumor resistance. This framework resolves distinct modules of immune evasion and survival, including the regulation of the antigen-presentation machinery, interferon/NF-κB signaling, oxidative stress resilience, and checkpoint/cytokine programs, while identifying perturbations that reroute the continuous tumor transcriptional trajectory induced by T cell engagement. A secondary chemical screen in patient-derived GBM cultures identified putative kinase targets of immune evasion phenotypes (e.g., EPHA2 and PDGFRA), whose inhibition leads to the blockade of evasive programs and enhances T cell-mediated GBM killing. Together, this workflow provides a scalable blueprint for comprehensive charting of the genetic control of tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e76a058bed6b1c1e4d91cd692a05e66fab6beee" target='_blank'>
              Mapping kinase-dependent tumor immune adaptation with multiplexed single-cell CRISPR screens
              </a>
            </td>
          <td>
            Lingting Shi, Ross M. Giglio, Qingyuan Cai, Mathini Vaikunthan, Justin Hong, Abdullah Naqvi, Marina Milea, Hannah Khanshali, Anna Schoonen, Nicholas Hou, Jonathan Guo, Melanie Fraidenburg, Xumin Shen, Seth Malinowski, K. L. Ligon, Raúl Rabadán, Elham Azizi, José L. McFaline-Figueroa
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly aggressive cancer of the biliary tract, distinguished by significant intratumoral heterogeneity (ITH), which contributes to therapy resistance and unfavorable clinical outcomes. Traditional genome profiling has revealed recurring driver changes in CCA; yet, genomic data alone fails to elucidate functional pathway activation, adaptive signaling, and the diverse treatment responses reported among tumor locations and disease subtypes. This review analyses the use of integrated sequencing technologies, proteogenomics, and phosphoproteomics to systematically characterize intratumoral heterogeneity in cholangiocarcinoma and convert molecular diversity into therapeutically applicable discoveries. We present evidence that the combination of genomic sequencing and mass spectrometry–based proteomics facilitates the direct correlation of genetic mutations with protein expression, post-translational modifications, and signaling system activity. Phosphoproteomic profiling specifically offers functional insights into kinase-driven networks that dictate tumor aggressiveness, therapeutic susceptibility, and adaptive resistance mechanisms, which cannot be anticipated only from DNA-level analysis. We propose that integrating proteogenomic and phosphoproteomic analyses into diagnostic and therapeutic assessments can enhance molecular classification, reveal subtype- and region-specific therapeutic dependencies, and guide rational combination treatment strategies, based on recent extensive proteogenomic studies and functional proteomic investigations in CCA. Pathway-level analysis of intratumoral heterogeneity provides a framework for selecting targeted medicines, predicting resistance, and informing personalized treatment strategies in CCA. The combination of sequencing, proteogenomics, and phosphoproteomics is essential for advancing precision oncology in cholangiocarcinoma. The implementation of this multi-layered analytical approach may better patient classification, refine therapy choices, and eventually improve clinical outcomes for individuals with this particular heterogeneous cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a96c2c6b9428d020026e895b0e1d0644f1ab872b" target='_blank'>
              Integrative Sequencing and Proteogenomic Approaches to Intratumoral Heterogeneity in Cholangiocarcinoma: Implications for Precision Diagnosis and Therapy
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, Apinya Jusakul, N. Namwat, Poramate Klanrit, Sitthirug Roytrakul, Hasaya Dokduang, Thitinat Duangchan, Yanisa Rattanapan, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Teh Bin Tean, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2026-01-07</td>
          <td>Medical Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a05de91b68f189c5a7a8fad28c43d535bcdc834" target='_blank'>
              A spatial atlas of colorectal cancer reveals the influence of stromal niches on tumour differentiation
              </a>
            </td>
          <td>
            Andrew D. Pattison, Rebekah M. Engel, Wing Hei Chan, Spencer Greatorex, David Nickless, Liam Skinner, Camilla Cohen, Julie Hickey, Christine Georges, Anne L. Fletcher, P. McMurrick, Thierry Jardé, Lochlan Fennell, Helen E Abud
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44d3ba922d5d14a9018eca6710873d3d64119d9" target='_blank'>
              Single-Cell Mitochondrial Lineage Tracing Decodes Fate Decision and Spatial Clonal Architecture in Human Hematopoietic Organoids.
              </a>
            </td>
          <td>
            Yan Xue, J. Su, Yiming Chao, Lu Liu, Xinyi Lin, Yang Xiang, Mun Kay Ho, Zezhuo Su, Junyi Chen, Zhuojuan Luo, Chengqi Lin, Ruibang Luo, Theo Aurich, Jianfeng Wu, Kelvin Sin Chi Cheung, Yuanhua Huang, Joshua W. K. Ho, Ryohichi Sugimura
          </td>
          <td>2026-01-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Metastatic triple-negative breast cancer (TNBC) is highly aggressive and lacks targeted therapies. Circulating tumor cells (CTC) are invaluable for monitoring metastatic tumor progression and treatment response but are difficult to capture because of their rarity and heterogeneity. Surface-based staining for live CTCs is essential to preserve RNA quality in single cells, but current markers tend to perform poorly on more mesenchymal tumor cells such as TNBCs. To enhance live TNBC CTC detection, we developed a workflow for live CTC capture and single-cell RNA sequencing (scRNA-seq). Using a mouse model of metastatic TNBC, we identified four new CTC surface markers, AHNAK2, CAVIN1, ODR4, and TRIML2, which specifically stain tumor cells. Combining antibodies against these markers improved CTC detection rates in multiple TNBC mouse models and patient samples. Also, combining these new markers with traditional CTC surface markers enhanced detection sensitivity, achieving the highest CTC coverage. This approach identifies diverse CTC populations, while preserving RNA quality for scRNA-seq, which is essential for understanding and therapeutically targeting metastatic breast cancer. The use of these newly identified CTC markers significantly enhances both detection and live capture of CTCs, paving the way for more effective use of liquid biopsy to monitor patient prognosis and treatment response in clinical settings. Significance: CTCs are a powerful indicator of cancer metastasis; however, their scarcity makes them difficult to isolate. Current markers favor epithelial CTCs over mesenchymal populations. Our workflow for live CTC capture and sequencing enables discovery of new markers for both epithelial and mesenchymal CTCs. When combined with existing markers, we improve live CTC capture for more holistic studies of the metastatic process and offer a scalable method for discovering CTC markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b002ef6362d1a4ee73fe011a8af50d517c31c57" target='_blank'>
              Surface Marker Identification to Capture Live Circulating Tumor Cells in Metastatic Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Bree M Lege, Khushali J Patel, Brendan Panici, Ping Gong, Michael T. Lewis, M.J. Ellis, Chonghui Cheng
          </td>
          <td>2026-01-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 The tumor microenvironment (TME) plays a pivotal role in shaping cancer progression, immune evasion, and therapeutic response. However, a systematic understanding of immune and stromal cell heterogeneity across cancer types—and their associations with clinical phenotypes—remains lacking due to limited single‐cell data and the associated clinical metadata. Here, we present a comprehensive pan‐cancer TME cell atlas to date, quantifying the composition of 102 immune and stromal subtypes across 30 cancer types and encompassing over 11,000 bulk RNA‐seq samples from TCGA. To enable this unprecedented resolution, we developed DECEPTICONx, a flexible computational framework that integrates single‐cell RNA‐seq data by constructing high‐resolution expression templates, which are then paired with context‐optimized deconvolution strategies, dynamically selecting the best‐matched template‐deconvolution combinations for each task. By leveraging 535,665 single cells spanning T cells, B cells, fibroblasts, macrophages, and NK cells, we constructed high‐resolution expression templates and systematically deconvolved bulk samples. The resulting atlas revealed robust and novel associations between specific cell subtypes and patient prognosis, immunotherapy response, and inter‐cellular coordination within the TME. In summary, the DECEPTICONx‐based atlas not only advances our understanding of tumor‐immune ecology but also offers a foundation for biomarker discovery and therapeutic development in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaaef7a98ed82e86ab1fb8032603b073ca5c5bb6" target='_blank'>
              A Pan‐Cancer Microenvironment Atlas Constructed by DECEPTICONx
              </a>
            </td>
          <td>
            Yida Gu, Fulan Deng, Lianchong Gao, Zhiyu Chen, Jianping Chen, Jiale Wu, Zengqiang Xia, Zhijian Li, Fangkaikai Cheng, Jie Chen, HoiYee Tong, Wantao Chen, Xiaoqing Pan, Xin Zou, Jie Hao
          </td>
          <td>2026-01-17</td>
          <td>iMetaMed</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cellular plasticity, the ability of cells to dynamically alter their phenotypes, is a key driver of tumor evolution. This process is a hallmark of cancer which enables the acquisition of malignant traits, leading to metastasis, progression, and therapy resistance. It is governed by cell‐intrinsic factors, such as genomic instability and epigenetic reprogramming, and extrinsic stimuli from the tumor microenvironment. However, a unified framework is still needed to position plasticity as the central process that links these drivers to diverse cancer hallmarks. In this review, we first explore how plasticity enables key steps of tumor evolution, including tumorigenesis, metastasis driven by epithelial–mesenchymal plasticity (EMP), therapy resistance, and cancer stem cell (CSC) dynamics. We then summarize the intrinsic and extrinsic mechanisms that govern this adaptability. Finally, we discuss clinical advances in monitoring and targeting plasticity and highlight how new spatiotemporal technologies can address current research challenges. This review provides a framework positioning cellular plasticity as a central mechanism in cancer evolution, connecting its fundamental drivers to clinical translation. By synthesizing the latest advances, we offer perspectives for developing therapies that integrate prediction, monitoring, and targeting of plasticity to proactively guide cancer evolution toward manageable outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683e6aa0ee59731275431238ec93cc43ffca8292" target='_blank'>
              Tumor Cell Plasticity in Cancer: Signaling Pathways and Pharmaceutical Interventions
              </a>
            </td>
          <td>
            Shangwei Sun, Yunwei Sun, Ling Lan, S. Luan, Jin Zhou, Jiehui Deng, Yong Yuan, Zhong Wu
          </td>
          <td>2025-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is considered immunologically “cold,” however, we have previously identified an immunogenic subset of high-risk localized PCa with T-cell tumor infiltration within antigen-presenting-cell (APC) niches. Here, we used high-dimensional single-cell spatial proteomics to dissect tumor-infiltrating-lymphocyte in (TIL)-high (H), TIL-low (L), and TIL-excluded (X) PCas, to understand the tumor-specific and conserved mechanisms of immune dysregulation for targeted therapeutics.



 Treatment-naive prostatectomy specimens of all three tumor immune microenvironment (TIME) phenotypes, were assembled into tissue microarrays representing within-tumor and adjacent foci. TIL-X was exclusively observed in cribriform pattern 4 cases. A 40-plex antibody panel made of phenotyping and functional markers (e.g AR) was applied to the TMAs, enabling single-cell annotation of key immune cell types (e.g CD8+ T cells) and assessment of their functional states within distinct spatial microenvironments.



 In TIL-X PCa, immunosuppressive macrophages and DCs were enriched in the tumor nest whereas T cells accumulated at the periphery with minimal tumor infiltration. Tumor adenosine 2A receptor (A2AR) expression was uniquely enriched within tumor, suggesting a metabolically suppressive microenvironment that reinforces immune exclusion. TIL-H tumors had comparable T cell proportions inside and outside the tumor, accompanied by the enrichment of myeloid cells within the tumor. CD8+ T cells expressing high levels of AR and B7H3 frequently co-localized with tumor and APCs, forming APC niches and tertiary lymphoid-like structures. CD8+ARhi and CD8+B7H3hi subsets with elevated exhaustion scores preferentially infiltrated tumor, whereas CD8+ARlo and CD8+B7H3lo subsets with moderate exhaustion levels remained peripheral; defining TIL-H PCa as tumors harboring antigen-experienced but exhausted CD8 T cells in a TIME that supports immune cell engagement but restrains anti-tumor activity. TIL-L PCa had sparse T cell infiltration and enrichment of myeloid cells with moderate AR expression within tumor, suggesting a role of androgen signaling and myeloid cells in shaping the TIME, marked by high PD-1+CD8+ cells and PD-L1+ tumor cells. Thus, TIL-L PCa shows low immune recruitment, yet strong tumor-mediated inhibitory signaling that constrains anti-tumor immunity.



 TILs in TIL-H PCa are phenotypically, functionally, and spatially distinct from those in TIL-L or TIL-X tumors, reflecting tumor-intrinsic immune programs. High-dimensional spatial proteomics reveal differential molecular modules and signaling pathways that define unique and potentially targetable mechanisms of immunosuppression across the three TIME phenotypes; additional spatial transcriptomics validation and dissection experiments are underway. These findings demonstrate that the PCa TIME is diverse yet spatially organized, and that comprehensive understanding of baseline spatial and transcriptional immune states may guide personalization of immunotherapy strategies in PCa.



 Precious Cramer, Yang Wang, Wenrui Wu, Huaying Qiu, Lindsay Parmelee, Berkay Simsek, Gabriel Roberti. De Oliveira, Seifeldin Awad, Yao Yu Yeo, Stephanie Pei Tung. Yiu, Hendrik Michel, Sabina Signoretti, Yue Sun, Steven Balk, Sizun Jiang, David Einstein Einstein. High-dimensional spatial profiling identifies distinct and targetable tumor-infiltrating lymphocyte phenotypes in localized prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d786dbf212bd5a71f25f7e11f429f8dba252ddc" target='_blank'>
              Abstract PR010: High-dimensional spatial profiling identifies distinct and targetable tumor-infiltrating lymphocyte phenotypes in localized prostate cancer
              </a>
            </td>
          <td>
            Precious Cramer, Yang Wang, Wenrui Wu, Huaying Qiu, Lindsay Parmelee, Berkay Simsek, Gabriel Roberti. De Oliveira, Seifeldin Awad, Yao Yu Yeo, Stephanie Pei Tung. Yiu, Hendrik A. Michel, Sabina Signoretti, Yue Sun, Steven P. Balk, Sizun Jiang, D. Einstein
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Prostate cancer cell invasion into the surrounding muscle capsule, known as extracapsular extension (ECE), is associated with a significantly worse prognosis (5-year survival reduction: 98% to 31%). Using a live ECE mouse xenograft model with DU145 tumor cells, our group previously demonstrated that functional heterogeneity among adhesion subtypes within invasive clusters is essential for muscle invasion. Bulk RNA sequencing from three spatial compartments (pre-invasive, invasive, and post-invasive) revealed distinct gene expression profiles, with hypoxia-related and cell–cell/cell–ECM interaction genes prominently dysregulated in muscle-invasive disease. In this study, we sought to define the single-cell transcriptional heterogeneity across the three ECE compartments and compare these patterns to a human patient-derived xenograft (PDX) model. Spatial transcriptomic analysis (STA) was performed using the Bruker CosMx platform with a 6K gene panel. Data was preprocessed (quality control and dimensional reduction) and analyzed in R 4.4.0 using Seurat v5. Notable downstream analyses included unsupervised Louvain clustering, trajectory inference with Monocle 3, and cluster-specific differential gene expression. In the ECE model, STA resolved at least 11 distinct tumor cell identity clusters previously undetectable by bulk methods and revealed a clear spatial demarcation between pre-invasive and invasive populations crossing the muscle barrier. Trajectory analysis identified a pre-invasive population branching into two invasive fates with unique expression profiles. Dispersion scores, calculated using principal component embeddings, exhibited progressively increasing intracluster heterogeneity along one branch. Strong expression enrichment of the Prolaris 31-gene prognostic panel, a clinically established marker of aggressiveness, was observed at the bifurcation point of tumor evolution. STA of the human PDX model similarly revealed a high-dispersion invasive cluster branching into two fates, one exhibiting decreased heterogeneity consistent with specialization of invasive phenotypes. Together, these findings reveal, for the first time, that tumor invasion of the extracapsular muscle layer induces substantial transcriptional heterogeneity. This dynamic process highlights potential therapeutic vulnerabilities during the transition to extracapsular extension. Targeting tumor cells at this critical juncture may represent a key strategy for preventing ECE and improving prostate cancer outcomes. Supported by P30CA023074, University of Arizona Comprehensive Cancer Center Support Grant.



 Saptarshi Mallick, Rafael Sainz, Jayati Chakrabarti, Kendra D. Marr, Noel A. Warfel, Gregory C. Rogers, Beatrice S. Knudsen, Rashid Sayyid, Kelvin W. Pond, Anne E. Cress. Single cell spatial transcriptomics identifies unique cell populations driving muscle invasive prostate adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcc6a26c8a5d704d47fd40ba9c2b9c0030fcf73d" target='_blank'>
              Abstract B045: Single cell spatial transcriptomics identifies unique cell populations driving muscle invasive prostate adenocarcinoma
              </a>
            </td>
          <td>
            Saptarshi Mallick, Rafael Sainz, Jayati Chakrabarti, Kendra D. Marr, N. Warfel, Gregory C Rogers, Beatrice S. Knudsen, Rashid Sayyid, Kelvin W. Pond, A. Cress
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7276a25631d754628d5b48707ca83be4de135812" target='_blank'>
              Biomarker Quantification in Breast Cancer using Xenium In Situ
              </a>
            </td>
          <td>
            Amanda Janesick, Stephanie N. Kravitz, Weston Stauffer, Miriam Valencia, Sarah E. B. Taylor
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1654ebba084e6a42978b44a5cecff88e9f2e8ee" target='_blank'>
              DISCO-seq: 3D single-cell transcriptomics of intact biological systems
              </a>
            </td>
          <td>
            H. Bhatia, Laurent H. A. Simons, L. B. Kuemmerle, Cristin McCabe, S. Jansen, Zhisong He, Mayar Ali, Denise Jeridi, M. Todorov, Luciano Hoeher, Sudharsan Padmarasu, Ha Eun Park, Justus Thevis, L. Bartos, Inderjeet Singh, S. Ussar, Kıvanç Görgülü, Hana Algül, Kenny Roberts, O. Bayraktar, Matthias Brendel, F. Theis, Barbara Treutlein, Aviv Regev, Ali Ertürk
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b48b1c55225e03672e68d4c772d4807ab8a0a9" target='_blank'>
              Critical role of cell competition in gliomagenesis
              </a>
            </td>
          <td>
            Ying Jiang, Ryuhjin Ahn, Arthur Huang, Phillippe P. Gonzalez, Jungeun Kim, Guoxin Zhang, Zihao Liu, Zhenqiang He, Lindsey A Dudley, Kunal S. Patel, Godfrey A Dzhivhuho, Sam Crowl, Piotr Przanowski, Luisa Quesada Camacho, Sijie Hao, Jianhao Zeng, S. Hippenmeyer, Mohammad Fallahi-Sichani, Kevin A. Janes, Kristen M. Naegle, M. Hammarskjold, Steven A. Goldman, H. Kornblum, Maojin Yao, Forest White, Hui Zong
          </td>
          <td>2026-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/238054edb782d465f94eaf0c685da8f4229110b1" target='_blank'>
              An integrated platform for high-throughput phenospace learning of 3D multilineage organoid systems
              </a>
            </td>
          <td>
            Ryo Okuda, Christoph Harmel, Quan Xu, Héloise Mary, Philip Schulz, Linda Steinacher, Elisa D’Arcangelo, Bruno Gjeta, Marina Signer, Irineja Cubela, Marc Bickle, Matthias P. Lutolf, Lauriane Cabon, Ilya Lukonin, J. G. Camp
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Intratumoral heterogeneity (ITH) is a major driver of mortality in breast cancer (BC) patients and a critical factor in the variable therapeutic outcomes observed in BC treatment. Understanding the mechanisms underlying ITH is essential for advancing both clinical and basic BC research. Chromatin accessibility is critical for regulation of gene expression and cellular identity and plays a central role in shaping ITH and tumor evolution. However, studying chromatin accessibility in situ has been challenging due to the availability of technical platforms. Here, we leveraged the spatial ATAC-seq platform to profile the chromatin accessibility landscape of tumors from six BC patients. Our analyses revealed prominent heterogeneity within tumor regulatory modules and spatial variations in immune cell composition and stromal structures, offering a framework for investigation of the molecular architecture underlying ITH. Moreover, we identified two tumor subclones with potential common origin but distinct immune infiltration conferred by regulatory cascades, suggesting that epigenetic regulation may further contribute to the divergent tumor microenvironments and phenotypic diversity of these subclones. Our study provides novel insights into the molecular mechanisms driving ITH and opens up potential avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b8d2e84c5991207c662ff2b4c48eb4734a447e" target='_blank'>
              Spatial Chromatin Accessibility Analysis of Intratumor Heterogeneity in Breast Cancer.
              </a>
            </td>
          <td>
            Yingying Qian, Miao Zhu, C. Ren, Yeyong Zhou, Jian Xu, Liang Dong, Guangyu Zhang, Cheukfai Li, Jiaoyi Lv, Qiaorui Xing, Guochun Zhang, Guangdun Peng, Ning Liao
          </td>
          <td>2026-01-12</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The human brain contains a milieu of extracellular matrix (ECM) components that promote normal development and physiology. ECM signaling pathways are often dysregulated in brain pathologies including the malignant cancer glioblastoma (GBM). Here, we used single-cell spatial transcriptomic platforms to map the expression of nearly 400 ECM genes in matching non-cancerous brain and GBM samples. At least four different GBM cell populations have been identified that show unique ECM expression profiles and spatial enrichment in distinct intratumor regions. Spatial mapping demonstrates largely non-overlapping expression signatures of ECM components in GBM stromal cell types, particularly in vascular endothelial cells and microglia/macrophages. Comparisons of GBM versus lower grade astrocytoma samples identifies differential expression of key ECM components. Computational analysis reveals novel ECM ligand-receptor networks between GBM and stromal cells. This spatial atlas provides new insights into ECM control of brain tumor initiation and progression and identifies potential targets for therapy in GBM. Using spatial transcriptomics, this study profiled the expression of hundreds of matrisome genes at single-cell resolution in human brain tumors, revealing targetable adhesion and signalling components in glioblastoma and lower-grade gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7358fd5e226c097b2efb1a3758a0fef198cc402b" target='_blank'>
              Single cell spatial profiling of the matrisome identifies region-specific adhesion and signaling networks in glioblastoma
              </a>
            </td>
          <td>
            Arpan De, Santiago A Forero, Ali Pirani, John E. Morales, Marisol De La Fuente-Granada, Sumod Sebastian, Jason T. Huse, Leomar Ballester, J.S. Weinberg, Frederick F. Lang, Kadir C. Akdemir, Joseph H. McCarty
          </td>
          <td>2025-12-09</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Spatial organization of the disease microenvironment informs patient prognosis. Key modalities for studying spatial biology include H&E images (WSIs) for tissue structure and spatial transcriptomics (ST) for transcriptome-level programs. Spatial analysis aims to (1) identify markers linked to clinical outcome, (2) understand functional programs driving these associations, and (3) guide targeted therapies. Current research addresses these topics but offers limited explainability across the full morphology - molecular mechanism - outcome axis. Further, given the abundance of WSIs and limited availability of ST, there is a need for analyses integrating these complementary datasets. We present an AI-driven framework combining foundation-model features, multiple-instance learning, unsupervised clustering, and molecular analyses to identify mechanisms underlying outcome associated patterns. Applied to HER2+ breast cancer, we identify CCND1 and PTK6 signaling in tumor regions linked to trastuzumab resistance, consistent with prior studies. Our approach offers interpretable insights for multi-level resistance mechanisms, tissue-specific drug targeting, and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7a60accd94c10ba89d66c522cd744cd2f973bb5" target='_blank'>
              A multimodal framework to identify molecular mechanisms driving patient group-associated morphology through the integration of spatial transcriptomics and whole slide imaging
              </a>
            </td>
          <td>
            Reva Kulkarni, A. Maddox, Sara Bailey, Arvind Rao
          </td>
          <td>2026-01-16</td>
          <td>Npj Artificial Intelligence</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background KRAS is one of the most frequently mutated genes in colorectal cancer (CRC) and plays a crucial role in tumorigenesis, progression, immune evasion, and treatment resistance. The pronounced heterogeneity within KRAS-mutant CRC highlights the urgent need for more precise and personalized therapeutic approaches. Methods To investigate this heterogeneity, we employed single-cell RNA sequencing and spatial transcriptomics to comprehensively characterize the tumor microenvironment of KRAS-mutant CRC. Data preprocessing and clustering were performed using Scanpy. Spatial cell-type deconvolution was conducted via Cell2location, whereas intercellular communication and spatial dependencies were analyzed using CellChat, MISTy, and stLearn. Results Our analyses revealed that KRAS-mutant tumor epithelial cells recruit Mono_S100A8 monocytes via the MDK_SDC4 signaling axis. Concurrently, surrounding Fib_CTHRC1 fibroblasts secrete collagen, which interacts with integrin receptors on KRAS-mutant epithelial cells and contributes to the exclusion of lymphocyte infiltration. Conclusion These cellular components collaboratively established an immunosuppressive spatial niche. These findings offer novel theoretical insights and potential targets for the development of immunoregulatory strategies tailored to KRAS-mutant CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059c9dfc013c2c370152c3fa0d39b4efc600e01e" target='_blank'>
              Single-cell and spatial transcriptome profiling identifies the immunosuppressive spatial niche in KRAS-mutant colorectal cancer
              </a>
            </td>
          <td>
            Sheng‐Li Yang, Chao Gu, Xinsheng Miao, Hao Zuo, Wei Xu, Yan Zhang, Wei Tang, Jianhua Zhu, Zheng Yuan, Xinhua Gu, Chenyi Zhong, Yueming Sun, Jiahui Zhou
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ffa550baca76e714d9b657ce67086b4b77a9db" target='_blank'>
              Deciphering Tumor Microenvironment Dynamics in Tumorigenesis and Lymph Node Metastasis of Esophageal Squamous Cell Carcinoma using Single-cell RNA Sequencing
              </a>
            </td>
          <td>
            Hansoll Na, T. Hong, Chung Lee, Y. Yang, Ha Eun Kim, Byung Jo Park, Hyun Ki Kim, Dae Joon Kim
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e234e293312dfb9093dad9f87bd40ef8d26f2e28" target='_blank'>
              Spatial transcriptomics reveals molecular heterogeneity and subtype-specific therapeutic targets in small cell lung cancer.
              </a>
            </td>
          <td>
            Tongji Xie, Le Tang, G. Fan, Xiaohong Han, Yuankai Shi
          </td>
          <td>2026-01-16</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d27ce10a067c8461179490fc90529630fe99743" target='_blank'>
              A modular patient-derived organoid–xenograft platform reveals molecular and clinical trajectories of prostate cancer progression
              </a>
            </td>
          <td>
            R. Parmentier, R. Dolgos, R. Servant, Jing Wang, L. Roma, R. Mevel, D. Bossi, H. Pueschel, Z. Diamantopoulou, T. Vlajnic, F. Stenner, A. J. Templeton, H. Seifert, Nicola Aceto, J-P. Theurillat, Cyril Rentsch, L. Bubendorf, C. Le Magnen
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is one of the most aggressive brain tumors because its cells spread widely through the brain, making complete removal impossible. We implanted fluorescently labeled glioma stem-like cells (GSCs) into the mouse brain and observed their infiltrative growth using advanced 3D imaging. Within 6–10 weeks, tumors caused about 15% brain swelling and showed a strong preference for invading the corpus callosum, a major white matter tract. By measuring Sox2-positive cells, we found very high densities in the tumor bulk (over 4000 cells/mm2) but much lower levels at the invasive margin (about 30 cells/mm2) and a clear gradient in the corpus callosum (3010 to 1.5 cells/mm2 across 1 mm). Using CLARITY tissue clearing combined with light-sheet microscopy, we visualized how tumor cells infiltrate along blood vessels and brain structures, adopting distinct shapes and strategies. Single-cell gene analysis of the invading corpus callosum cells identified a 7-gene “invasion signature” (NES, CCND1, GUSB, NOTCH1, E2F1, EGFR, TGFB1), which was linked to worse survival and may represent new therapeutic targets. Abstract Background/Objectives: High-grade gliomas (HGGs), including glioblastomas, are among the most aggressive brain tumors due to their high intratumoral heterogeneity and extensive infiltration. Glioma stem-like cells (GSCs) frequently invade along white matter tracts such as the corpus callosum, but the molecular programs driving this region-specific invasion remain poorly defined. The aim of this study was to identify transcriptional signatures associated with GSC infiltration into the corpus callosum. Methods: We established an orthotopic xenograft model by implanting fluorescently labeled human GSCs into nude mouse brains. Tumor growth and invasion patterns were assessed using tissue clearing, light-sheet fluorescence microscopy, and histological analyses. To characterize region-specific molecular profiles, we performed microfluidic-based single-cell RNA expression analysis of 48 invasion- and stemness-related genes in cells isolated from the tumor bulk (TB) and corpus callosum (CC). Results: By six weeks post-implantation, GSCs displayed marked tropism for the corpus callosum, with distinct infiltration patterns captured by three-dimensional imaging. Single-cell gene expression profiling revealed significant differences in 7 of the 48 genes (14.6%) between TB- and CC-derived GSCs. These genes—NES, CCND1, GUSB, NOTCH1, E2F1, EGFR, and TGFB1—collectively defined a “corpus callosum invasion signature” (CC-Iv). CC-derived cells showed a unimodal, high-expression profile of CC-Iv genes, whereas TB cells exhibited bimodal distributions, suggesting heterogeneous transcriptional states. Importantly, higher CC-Iv expression correlated with worse survival in patients with low-grade gliomas. Conclusions: This multimodal approach identified a corpus callosum-specific invasion signature in glioma stem-like cells, revealing how local microenvironmental cues shape transcriptional reprogramming during infiltration. These findings provide new insights into the spatial heterogeneity of gliomas and highlight potential molecular targets for therapies designed to limit tumor spread through white matter tracts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561590a9b7ca738409c5933b00e64b421edbb629" target='_blank'>
              Innovative In Vivo Imaging and Single Cell Expression from Tumor Bulk and Corpus Callosum Reveal Glioma Stem Cells with Unique Regulatory Programs
              </a>
            </td>
          <td>
            Natália dos Santos, Aline Aquino, Friedrich Preusser, Fabio Rojas Rusak, E. H. Jandrey, Miyuki Uno, T. K. Furuya, C. Lancellotti, M. Maldaun, Roger Chammas, Stephan Preibisch, A. Camargo, C. Masotti, Erico Tosoni Costa
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT Head and neck squamous cell carcinoma (HNSCC) is the sixth most common cancer worldwide. HPV-negative HNSCC, arising in diverse upper airway mucosal niches, is particularly aggressive, with poor 5-y survival and a limited response to immune checkpoint inhibitors. A deeper understanding of the tumor-localized immune landscape is essential to uncover actionable immunotherapeutic targets. Here, we integrated two single-cell RNA sequencing (scRNA-seq) datasets from 29 samples totaling nearly 300,000 immune cells to dissect immune rewiring during tumor progression and lymph node metastasis in HPV-negative HNSCC. We identified distinct shifts in adaptive immune cell populations across 14 peripheral blood mononuclear cell (PBMC) and 21 tumor-infiltrating immune cell (TIC) states. Notably, TICs exhibited enriched interferon response and immunomodulatory gene signatures, in contrast to PBMCs, indicating tumor-specific immune imprinting. Ligand–receptor analysis revealed that immunosuppressive crosstalk between macrophages and cytotoxic cells was associated with advanced disease. To spatially validate these transcriptional states, we conducted multiplexed immunofluorescence profiling on nine locally invasive HPV-negative HNSCCs, all from the ventrolateral tongue mucosa. Spatial proteomics confirmed peritumoral enrichment of activated (CD107a+, ICOS+) NK and CD8+ T cells and intratumoral accumulation of exhausted (PD-1+, PD-L1+) phenotypes, mirroring pseudotime trajectories inferred from scRNA-seq. These findings highlight spatially localized cytotoxic cell exhaustion as a key immune evasion mechanism in HPV-negative HNSCC and underscore the value of integrating spatial and single-cell data to reveal therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e184d6ff5ae8303f9ef6f76673a4a076479b44fb" target='_blank'>
              Single-cell landscape of peripheral and tumor-infiltrating immune cells in HPV-negative HNSCC
              </a>
            </td>
          <td>
            Rômulo Gonçalves Agostinho Galvani, Adolfo Alexis Rojas Hidalgo, Carlos Alberto Biagi-Junior, B. Matuck, Jelte Martinus Maria Krol, Brittany T. Rupp, Nikhil V. Kumar, Khoa L. A. Huynh, Jinze Liu, Siddharth Sheth, Vinicius Maracaja-Coutinho, K. Byrd, Patricia Severino
          </td>
          <td>2025-12-30</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9542f45171a12fa01fc5a868a4f227534778e8" target='_blank'>
              Iron-Associated Mesenchymal Plasticity and Tumor Invasion in Glioblastoma
              </a>
            </td>
          <td>
            C. Petry, A. Boecke, A. Mohammed, N. Ghariwala, V. Jariwala, Mohammad Al Shhab, O. Buchholz, I. Vasilikos, J. M. Nakagawa, Andreas Vlachos, J. Grauvogel, M. Shah, Roland Rölz, Marco Prinz, U. G. Hofmann, Jürgen Beck, Kevin Joseph, V. Ravi
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is an aggressive brain tumor with limited treatment options, and the tumor microenvironment plays a major role in its progression. Among stromal cells, cancer-associated fibroblasts (CAFs) are known to influence tumor growth and immune responses, but their diversity in glioblastoma has not been well understood. In this study, we analyzed both bulk and single-cell RNA sequencing data to identify CAF subtype-associated patterns within glioblastoma. We discovered four CAF populations—immune, inflammatory, antigen-presenting, and myofibroblastic—each associated with poor patient survival and unique immune and metabolic features. We also identified Spleen Tyrosine Kinase (SYK) as a key regulatory gene shared across these CAF subtypes. Our findings highlight the importance of stromal heterogeneity in glioblastoma and suggest that targeting CAF-related signaling, including SYK, may offer new therapeutic opportunities for this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77eebcce441ee2b3c1609920849b71d1de5f2a30" target='_blank'>
              Dissecting CAF Heterogeneity in Glioblastoma Reveals Prognostic Subtypes and a Central Regulatory Role for Spleen Tyrosine Kinase (SYK)
              </a>
            </td>
          <td>
            Ji-Yong Sung, K. Hwang
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PAAD) metastasis is driven by complex tumor–microenvironment interactions. Here, we integrated single-cell and bulk transcriptomic analyses of 104,855 cells from 10 patients to delineate the cellular and molecular landscape of primary versus metastatic PAAD. We identified metastasis-associated epithelial (LMO7⁺, TOP2A⁺, PIGR⁺), fibroblast (IGKC⁺, RGS5⁺), and M2-like macrophage (APOE⁺, CD14⁺, FOLR2⁺, SPP1⁺) subpopulations, validated via bulk deconvolution. Functional analyses revealed upregulated Wnt signaling, epithelial–mesenchymal transition, and angiogenesis in metastatic epithelial and fibroblast compartments. Intercellular communication analysis highlighted SPP1-mediated macrophage–epithelial/fibroblast crosstalk involving key receptor–ligand pairs, contributing to immune suppression and metastatic niche formation. Integrating gene expression and cell proportions, we developed a prognostic model with high predictive accuracy (C-index > 0.85), stratifying patients into risk groups with distinct immune landscapes. Furthermore, PTK6 was identified as a driver of PAAD proliferation, migration, and invasion. Collectively, our study elucidates TME-driven mechanisms of PAAD metastasis, identifies prognostic and therapeutic targets, and provides a framework for precision intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/759c03413ef4b1106fa2a25c75713885026e6884" target='_blank'>
              Single-cell transcriptomic analysis reveals key cell types and pathogenic genes associated with metastasis in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Shijun Shen, Zhiqiang Li, Hong Yang, Xinwei Zhang, Gang Chen, Chengzhou Pa
          </td>
          <td>2025-11-29</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0135229fffb59fe03ba95bebb58be45a4b3470c" target='_blank'>
              De novo H3.3K27M-altered Diffuse Midline Glioma in human brainstem organoids to dissect GD2 CAR T cell function
              </a>
            </td>
          <td>
            N. Bessler, A. Wezenaar, H. Ariese, C. Honhoff, E. J. Wehrens, N. Dommann, C. Ruiz Moreno, Thijs J M van den Broek, R. Collot, D.J. Kloosterman, F. Keramati, M. Roosen, S. de Blank, Esmée J. van Vliet, M. Barrera Román, L. C. Gatti, Ali Ertürk, J. Kuball, Z. Sebestyén, M. Kool, S. Patrizi, E. Miele, A. Künkele, M. Kranendonk, Annelisa M. Cornel, Stefan Nierkens, Christian Mayer, H. Stunnenberg, A. Alemany, Maria Alieva, A. Rios
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Neoadjuvant immunotherapy has reshaped the treatment paradigm for esophageal squamous cell carcinoma (ESCC), yet mechanisms of resistance in non-responders remain poorly understood. Specifically, the spatial orchestration of the tumor microenvironment that limits effective antitumor immunity is not fully elucidated. Methods Integrating spatial transcriptomic analysis with large-scale tissue pathology, we mapped the spatial architecture of the immune landscape in immunotherapy-resistant ESCC to identify cellular and molecular determinants of treatment failure. Results We revealed a distinct immune exclusion phenotype in non-responders, characterized by peritumoral CD8+ T cell enrichment coupled with intratumoral depletion. At the invasive front, we identified COL11A1+ cancer-associated fibroblasts (CAFs) and SPP1+ tumor-associated macrophages as spatially correlated markers of immune exclusion, demarcating regions characterized by limited T cell infiltration. Mechanistically, a distinct subpopulation of LAMC2+ tumor cells localized at the tumor boundary acts as the master orchestrator of this barrier. These LAMC2+ cells exhibit aberrant lactate metabolism and elevated stemness, driving CAF activation via Semaphorin 3C secretion. Strikingly, this ‘LAMC2+ boundary tumor cell-immune-privileged niche’ axis exhibits pan-cancer correlations with immunotherapy resistance, positioning LAMC2 as a robust predictive biomarker. Conclusion In this study, we identify the tumor invasive margin as a critical determinant of immunotherapy resistance in ESCC. By defining the specific spatial markers and molecular architecture of an immune-privileged niche associated with the immune-exclusion barrier, our findings demonstrate the value of spatially resolved microenvironmental analysis. Moreover, we propose that therapeutic targeting of this boundary niche represents a promising strategy to overcome resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/858298d1227537940c73aee34b8d245e5b4c9498" target='_blank'>
              An Immune-privileged niche mediates immunotherapy resistance in esophageal carcinoma
              </a>
            </td>
          <td>
            Li Wan, Boye Li, Tianyun Ma, Min Wang, Fei Xie, Haotian Ge, Zixuan Xiao, Xinyi Chen, Lingda Meng, Li Zhang, Kunxin Xie, Qilong Wang
          </td>
          <td>2026-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Distal cholangiocarcinoma (dCCA) arises from the distal bile duct and is anatomically embedded within the pancreatic head, adjacent to abundant autonomic nerve plexuses. This unique location renders dCCA particularly prone to perineural invasion (PNI), a pathological hallmark that contributes to its dismal prognosis. However, the spatial architecture and molecular drivers that orchestrate PNI remain poorly defined. Here, we applied Xenium subcellular resolution spatial transcriptomics platform to profile resected tumor tissues from dCCA patients stratified by PNI status pathologically. A spatially resolved atlas comprising a total of 20 cell types was generated, uncovering enrichment of Schwann cells, type 2 conventional dendritic cells (cDC2), M2-like macrophages, cancer associated fibroblasts (CAFs) and B/plasma cells in PNI-high tumors, along with depletion of exhausted CD8+ T cells. Heterogeneous malignant cells in PNI-high tumors demonstrated activation of extracellular matrix remodeling and axonogenesis pathways, in line with the initial pathological classification. Spatial mapping further revealed distinct PNI-associated niches, notably matrix-producing CAFs (mCAFs)-macrophage clusters exhibiting coordinated enrichment of inflammatory and fibrotic programs. We further identified the LAMB3-DAG1 axis as a potential mediator of dCCA cells-Schwann cell interaction, while the preferential proximity of arteries to Schwann cells suggested additional microenvironmental support for nerve invasion. Collectively, our study provides a comprehensive subcellular atlas of PNI in dCCA, uncovering coordinated epithelial, stromal, and immune remodeling that drives perineural invasion. The identified biomarkers not only hold promise for patient stratification but may also guide intraoperative navigation and surgical margin determination, offering new avenues for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7ed53173a0ee6bb5d8c5c5791ee85be378c7900" target='_blank'>
              A subcellular spatial atlas illuminates the microenvironmental remodeling of perineural invasion in distal cholangiocarcinoma
              </a>
            </td>
          <td>
            Fansen Ji, Hao Chen, Huan Li, Jiawei Zhang, Sijia Li, Pengfei Wang, Hao Liu, Cui Ge, Bingjun Tang, Hongfang Yin, Xuedong Wang, Jiahong Dong
          </td>
          <td>2026-01-08</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) exhibits pronounced spatial heterogeneity that limits therapeutic efficacy. The contribution of epithelial–mesenchymal transition (EMT) to regional tumor progression and immune evasion remains incompletely understood. Methods We integrated bulk transcriptomic datasets (TCGA, GSE14520), 34 single-cell RNA-seq samples, and 15 spatial transcriptomic datasets to delineate EMT activity across distinct HCC regions. Immune infiltration profiling, pathway enrichment, and multi-model machine learning were used to identify candidate EMT regulators. Functional validation was performed in Hep3B cells through wound healing, Transwell migration/invasion, and immunofluorescence assays. Results EMT activity was significantly elevated at tumor margins (fold change = 2.7, p < 0.001) and was associated with poorer overall survival (HR = 2.15, 95% CI: 1.41–3.27, p < 0.001). Regions with high EMT signatures showed reduced CD8⁺ T-cell infiltration and increased immunosuppressive cells, including MDSCs, M2 macrophages, and Tregs, along with elevated expression of immune checkpoints (PDCD1, CTLA4, LAG3). Among candidate regulators, UBC was consistently ranked as a top EMT-associated gene across all models. Functional assays confirmed that UBC overexpression enhanced migration, invasion, and vimentin expression, whereas UBC knockdown reversed these effects. Conclusion Through integrative spatial multi-omics and experimental validation, we identify UBC as a key mediator of EMT and immune suppression at HCC margins. These findings provide mechanistic insight into spatial heterogeneity and suggest that targeting UBC could have translational potential for overcoming immune evasion in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15e71d7d57f5aafdc6998bcde18c6af7f7890890" target='_blank'>
              Integrative Spatial Transcriptomics and Experimental Validation Reveal UBC-Mediated EMT Associated with Immune Evasion in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Xiaosong Li, Xian Qin, Kezhi Shi, Guangrui Lu, Guodong Tian, Yue Chen, Rucheng Yao
          </td>
          <td>2025-12-01</td>
          <td>ImmunoTargets and Therapy</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5bdb6dd873fe4b6bf8ca46144bcdfa66e477ac0" target='_blank'>
              Beyond one-size-fits-all: single-cell transcriptomic signatures predict drug efficacy and reveal responder subgroups in endometriosis
              </a>
            </td>
          <td>
            R. Pérez-Moraga, C. Bafligil, S. Harden, Santiago García‐Martín, M. J. Jiménez-Santos, Ioanna Tiniakou, Sophie Ribeiro-Volturo, Line Gies, María Teresa Pérez Zaballos, Cristina Fernández-Molina
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) enables high-resolution transcriptomic analysis but loses spatial context due to tissue dissociation, thereby limiting insights into spatially regulated cell–cell interactions critical for tissue function and disease, including cancer. Consequently, developing approaches capable of extracting spatially relevant intercellular interaction information from scRNA-seq data is of substantial importance for elucidating mechanisms underlying disease initiation and progression. Here, we present DeepPNCC, a novel deep learning framework that reconstructs pseudo-spatial cell-cell interaction networks (PSCCIs) from scRNA-seq data by leveraging latent spatial cues preserved in undissociated cell aggregates. Built on a variational graph autoencoder (VGAE) enhanced with adversarial regularization, DeepPNCC uniquely integrates local adjacency matrices derived from multiplet data, without relying on prior knowledge such as ligand-receptor pairs, to infer global, spatially informed interaction landscapes. DeepPNCC is available as an open-source Python package. DeepPNCC outperforms existing methods in recovering interactions aligned with spatial transcriptomics across mouse brain and breast cancer datasets. In breast cancer, the inferred cell-cell interaction network reveals a closed-loop signaling axis in triple-negative breast cancer, in which cancer-associated fibroblasts (CAFs) and perivascular-like cells (PVL) promote angiogenesis through the ADM-CALCRL-STAT1-CD40/CCL2/ICAM1 pathway. These findings provide important clues for elucidating the regulatory mechanisms of the tumor microenvironment. DeepPNCC provides an efficient tool for spatially informed analysis of scRNA-seq data, enabling the reconstruction of pseudo-spatial cell–cell interaction networks from datasets lacking explicit spatial information. This approach expands the spatial analytical scope of scRNA-seq and facilitates mechanistic dissection of cellular ecosystems in both health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee882b8d93f6b445af77befa6248cb5501417766" target='_blank'>
              DeepPNCC: reconstructing pseudo-spatial cell-cell interaction landscapes from single-cell data to decipher breast cancer pathogenesis
              </a>
            </td>
          <td>
            Xu-hua Li, Xiao-Ling Gao, Deng-Hui Guo, Jie Shen, Si-Rui Li, Na Wang, Xiao-ling Wen, Jiang-Ying Liang, Hong-jiu Wang, Zhen-Zhen Wang
          </td>
          <td>2025-12-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma is a lethal primary brain tumor with poor prognosis. Tumor cells exhibit substantial phenotypic variation, complicating treatment. As functional diversity is driven by underlying transcriptional states, characterizing tumor cell gene expression is essential for understanding tumor biology and therapeutic response. The GL261 tumor cell line is a common pre-clinical model system for investigating glioblastoma pathobiology. However, global gene expression patterns in this model are unknown. Here we describe the use of single-cell RNA sequencing (scRNA-Seq) to investigate transcriptional profiles of 5764 adherent and 4951 neurosphere GL261 cells, generating 133,442,221 sequenced reads. Following Principal Component Analysis (PCA) for dimensionality reduction, we applied Uniform Manifold Approximation and Projection (UMAP) to visualize transcriptionally distinct subpopulations (clusters) of GL261 cells grown adherently or as neurospheres. Highly expressed and differentially expressed genes were identified. Because the neurosphere phenotype is known to be more tumorigenic, we further examined differentially expressed genes with gene ontology expression analysis. We found that upregulated genes in neurosphere cells are associated with angiogenesis, cell adhesion, and cell signaling pathways. In addition, we specifically examined gene expression patterns of matrix metalloproteinases and purinergic receptors, glioblastoma drug targets known to be important for promoting tumor infiltration into adjacent healthy tissue. We found that P2RX7, MMP15 and MMP16 are upregulated in neurosphere cells, indicating a potential role for these genes in tumor formation. Together these results reveal global transcriptional profiles of GL261 cells, establish a resource for further scRNA-Seq-based analyses, and give insight into gene expression changes relevant to glioblastoma tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b536ff0acd69dbca3af6bd3122c752115f69d4a" target='_blank'>
              Single-Cell Transcriptomic Profiling of GL261 Glioblastoma Cells Reveals Gene Expression Signatures Underlying Tumorigenicity
              </a>
            </td>
          <td>
            Colton E Troxel, Ruby A Olvera, Emily A Freko, Suely S C Soeiro, Thandiswa Mdluli, Richard L Daniels
          </td>
          <td>2025-12-16</td>
          <td>Cellular and Molecular Neurobiology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Osteosarcoma (OS) exhibits substantial genomic complexity and inter-patient heterogeneity, necessitating longitudinal, patient-matched analyses to understand acquired features of tumor evolution. However, most published OS data is limited to primary tumor samples, limiting insight into patient-specific resistance mechanisms. To address this, we characterized the genomic landscape of paired primary and metastatic tumor samples from dogs with spontaneous OS. Whole-genome and single-cell RNA sequencing reveal mutation and gene expression profiles that are predominantly organized by patient identity. Mutational burden and pathway alterations such as those involving PI3K, NOTCH, TP53, MAPK, RAS and epigenetic regulation differ between primary and metastatic samples. Variants present in tumor tissue are readily detectable in paired cfDNA samples, demonstrating the utility of this assay for identifying tumor-specific alterations associated with treatment resistance. Analysis of bulk RNA-seq data to estimate cell-type composition shows greater immune cell representation in metastases, underscoring the importance of immune signaling pathways in OS. These findings exemplify the presence of patient-specific alterations in genomic architecture over the course of tumor progression, linking CNV amplification, pathway reprogramming, and immune evasion in metastatic OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02ad1efaa62d98c7f16dd57c71f8980e8844d4c9" target='_blank'>
              Tumor microenvironment signature associated with morphology in systemic ALK-positive anaplastic large cell lymphoma
              </a>
            </td>
          <td>
            C. Bessiere, L. Babin, Sandra Dailhau, P. Gaulard, S. Pyronnet, F. Meggetto, L. Lamant
          </td>
          <td>2026-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell studies have significantly advanced our understanding of the transcriptional and functional heterogeneity in cancers. Recent studies have identified distinct states of cancer, immune, and stromal cells in the tumor microenvironment (TME), with growing evidence highlighting their clinical significance and therapeutic potential. Here, we present CancerSEA-X, an expanded version of CancerSEA that offers a comprehensive atlas of TME cell states. CancerSEA-X integrates 25 cancer cell states, 105 immune cell states, and 26 stromal cell states from systematically curated publications. Combining 239 single-cell datasets across 32 cancer types, encompassing over 9 million cells from 2120 patients, CancerSEA-X provides functional activity spectra and cancer-specific gene associations for these 156 cell states. These cell state-gene relationships were mapped onto networks, providing a systematic view of the TME. To improve usability, we redesigned the user interface to feature cell state characterization, state-gene correlation analysis, and interactive visualization of cell state-gene networks, enabling researchers to comprehensively explore these states and their functional relevance. Overall, CancerSEA-X serves as a valuable platform for investigating TME cell states, deepening our understanding of cancer heterogeneity, and potentially advancing the design of more effective clinical therapies. CancerSEA-X is freely available at http://biocc.hrbmu.edu.cn/CancerState.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc11040a019ecb4b6c2065cd731c8ff500837ec3" target='_blank'>
              CancerSEA-X: A Single-cell Resource for Tumor Microenvironment Cell States Across over 30 Cancer Types.
              </a>
            </td>
          <td>
            Lantian Deng, Wei Liu, Jinyuan Xu, Bo Zhang, Shengyuan He, Kun Liu, Xinxin Zhang, Huating Yuan, Fei Quan, Yun Xiao
          </td>
          <td>2026-01-16</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cf48b273167de19bd485ee182b5df8f720d9f6b" target='_blank'>
              Spatially distinct chromatin compaction states predict neoadjuvant chemotherapy resistance in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Reut Mealem, T. A. Phillips, Leor Ariel Rose, S. Marcotti, M. Parsons, A. Zaritsky
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Immunological intratumor heterogeneity (ImTH) describes the variability in the types, spatial distribution, and functional states of immune cells within tumors. While evidence suggests that ImTH influences tumor progression and therapeutic response, few studies have provided a quantitative characterization of ImTH. Here, we present Scoring Immunological Intratumor Heterogeneity (ScImTH), a novel algorithm that quantifies ImTH by calculating the Shannon entropy of immune cell type proportions within the tumor microenvironment. Using bulk, single-cell, and spatial transcriptomic datasets, we show that reduced ScImTH scores are associated with unfavorable survival outcomes, tumor progression-related molecular and phenotypic features, immunosuppressive states, and resistance to immunotherapy across multiple cancer types. Compared with existing measures of tumor immunity, such as immune score and B-cell receptor diversity, the ScImTH score demonstrated stronger and more consistent associations with clinicopathological features. Notably, the ScImTH score outperformed established biomarkers, including tumor mutational burden and PD-L1 expression, in predicting immunotherapy response. These findings highlight the clinical potential of the ScImTH score as a biomarker for cancer prognosis and immunotherapy stratification. More broadly, our results support the hypothesis that loss of immune diversity is a hallmark of tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e493ac7b6aab5ae1ab712cb247fef1395ea0b1a" target='_blank'>
              Quantitative profiling of intratumor immune heterogeneity identifies loss of immune diversity as a hallmark of cancer progression
              </a>
            </td>
          <td>
            Qiqi Lu, Jian-qin Luo, Chia-Hao Tung, Xiaosheng Wang, Zhongming Zhao
          </td>
          <td>2025-12-16</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d903aacfbf3a6dfc2c450d8cb95280e66a5321f" target='_blank'>
              sCellST predicts single-cell gene expression from H& E images.
              </a>
            </td>
          <td>
            Loïc Chadoutaud, Marvin Lerousseau, Daniel Herrero-Saboya, Julian Ostermaier, Jacqueline Fontugne, Emmanuel Barillot, Thomas Walter
          </td>
          <td>2026-01-09</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b4d870dc4ba763551ef81671814fc477f8046f" target='_blank'>
              CancerSTFormer enables multi-scale analysis of spot-resolution spatial transcriptomes and dissects gene and immune regulatory responses to targeted therapies
              </a>
            </td>
          <td>
            Benjamin S Strope, Dana Varghese, William Z Bowie, Stacy Wang, Qian Zhu
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cervical cancer (CC) is one of the most prevalent malignancies that affects women worldwide. It is strongly influenced by lifestyle and environmental factors, most notably persistent HPV infection and smoking. A defining feature of CC is its high degree of heterogeneity, which poses significant challenges for conventional sequencing and diagnostic approaches in fully capturing its oncogenetic landscape. The advent of single-cell RNA sequencing (scRNA-seq) has transformed this field, offering an unprecedented ability to dissect the tumor microenvironment at a single-cell resolution. By mapping cell–cell communication networks and characterizing gene expression across diverse cellular populations, scRNA-seq enables the identification of biomarkers with strong diagnostic and prognostic potential. Beyond biomarker discovery, this technology also facilitates the recognition of novel therapeutic targets, paving the way for more precise and effective treatment strategies. This review summarizes the applications of scRNA-seq in cervical cancer research, with a focus on methodological advances, key findings from recent studies, and their implications for improving diagnosis, prognosis, and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e029b6816cfdfef9fa928cd92a7a3e4aaf306a" target='_blank'>
              Implications of Single-Cell RNA Sequencing in Cervical Cancer: Unravelling the Molecular Landscape
              </a>
            </td>
          <td>
            Blessy Kiruba, Raja Rishi Raghavendar Raja Karthikeyan, S. Anilkumar, Vino Sundararajan, S. Lulu S
          </td>
          <td>2025-12-02</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pulmonary pleomorphic carcinoma (PPC) is a rare subtype of lung cancer that comprises both epithelial and sarcomatoid components. The molecular basis of PPC, including the cellular dynamics of its components, remains largely unknown. To elucidate potential therapeutic targets for PPC, we perform a multi-omics analysis incorporating digital spatial profiling and single-cell RNA sequencing (scRNA-seq). PPC exhibits diverse driver gene alterations, including MET exon 14 skipping mutation (METex14) and ALK fusion. In spatial transcriptomics, MET gene and protein are overexpressed exclusively within the epithelial component and not in the sarcomatoid component, even in patients harboring METex14. Epithelial-mesenchymal transition (EMT)-related transcriptional changes, along with extracellular matrix (ECM) remodeling between the epithelial and sarcomatoid components, are observed. scRNA-seq identifies cell populations within the epithelial component that contribute to the malignant transformation and differentiation of the sarcomatoid component. They are characterized by an intermediate EMT state with ECM remodeling signature, suggesting their potential as novel therapeutic targets for PPC. Single-cell and spatial transcriptomic analysis of pulmonary pleomorphic carcinoma reveal epithelial–mesenchymal transitions and extracellular matrix remodeling that shape the distinct architecture of this rare lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f421d642675df6e6bdc9b51c7d9b23970c7bec20" target='_blank'>
              Single-cell and spatial transcriptomic characterization of pulmonary pleomorphic carcinoma
              </a>
            </td>
          <td>
            Atsushi Matsuoka, K. Shien, Shuta Tomida, Masayoshi Ohki, Kazuya Hisamatsu, Ryota Fujiwara, Kosei Ishimura, Ryunosuke Fujii, Tomoaki Higashihara, Naohiro Hayashi, Kazuhiro Okada, Ryo Yoshichika, Fumiaki Mukohara, Mao Yoshikawa, Yuma Fukumoto, K. Suzawa, Y. Tomioka, Shin Tanaka, Kentaroh Miyoshi, Mikio Okazaki, S. Sugimoto, Yusuke Otani, Atsushi Tanaka, Hirofumi Inoue, Yosuke Togashi, Hidetaka Yamamoto, D. Ennishi, Shinichi Toyooka
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Remodeling of the tumor microenvironment (TME) under therapeutic pressure is a critical determinant of treatment response and resistance in hepatocellular carcinoma (HCC). Triple-combination therapy integrating targeted agents, immune checkpoint inhibitors, and radiotherapy (T+I+R) has shown potential synergistic effects in intermediate to advanced HCC, particularly in patients with portal vein tumor thrombus (PVTT), yet the spatial and cellular mechanisms underlying its efficacy remain largely unknown. In this retrospective clinical cohort study, we compared T+I+R with targeted therapy plus radiotherapy (T+R) in advanced HCC, and further employed single-cell spatial transcriptomics and spatial proteomics to generate an integrated multi-omics atlas mapping tumor and stromal compartments, cellular compositions, and intercellular interactions with spatial resolution. Clinically, T+I+R achieved superior tumor shrinkage and disease control compared with T+R. Spatial multi-omics revealed marked region-specific remodeling, with myofibroblastic cancer-associated fibroblasts, angiogenic tip endothelial cells, and conventional dendritic cells enriched at the invasive margin and associated with therapeutic resistance, while CD8+ effector T cells were redistributed away from immunosuppressive niches, a spatial configuration correlating with enhanced response. These findings identify spatial segregation between cytotoxic and suppressive immune elements as a potential hallmark of effective therapy, providing a high-resolution spatial framework for understanding T+I+R induced TME remodeling and offering mechanistic insights to guide biomarker discovery and the optimization of combination strategies in advanced HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d6c3edb9469068b18d4bde88276d91593b43ce6" target='_blank'>
              Spatial multi-omics mapping of tumor microanatomy dynamics following radiotherapy combined with targeted-immunotherapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Fansen Ji, Haoming Xia, Ying Xiao, Jiawei Zhang, Hao Chen, Bingjun Tang, Huan Li, Hao Liu, Boyang Wu, Xiaojuan Wang, Shizhong Yang, Jiahong Dong
          </td>
          <td>2026-01-08</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="To identify the cell types that are infected with KSHV and the immune interactions in Kaposi sarcoma (KS) lesions, we performed spatial transcriptomics with seven KS skin tumors. We used a single‐cell RNA‐sequencing reference data set from healthy skin donors with a method to conduct spatially informed cell‐type deconvolution for spatial transcriptomics. This allowed us to predict the relative amounts of each cell type within the patient sample sections. We included custom probes for five KSHV genes that allowed us to measure human and KSHV expression patterns at the same time. We then compared the spatial gene expression data of KS skin samples with six normal skin samples and found higher expression of marker genes corresponding to macrophages/dendritic cells, lymphatic endothelial cells, and vascular endothelial cells in the KS skin lesions when compared to normal skin samples. Our spatial transcriptomic results from thousands of spots across multiple KS tumors indicated a correlation between high levels of STC1 and decreased expression of macrophage markers. Together, these analyses offer potential mechanisms by which KSHV infection may remodel skin tissue, inhibit immune responses against KSHV infection, and confer resistance to anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0e2360131ded34f6dbf590cb05f764fc0e85cc" target='_blank'>
              Spatial Transcriptomics of Patients With Kaposi Sarcoma Identifies Mechanisms of Immune Evasion
              </a>
            </td>
          <td>
            Bahman Afsari, R. Ramaswami, K. Lurain, Takanobu Tagawa, Daphne Knudsen-Palmer, Guruswamy Mahesh, Ameera Mungale, R. Yarchoan, Joseph Ziegelbauer
          </td>
          <td>2025-12-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is a lethal subtype of prostate cancer characterized by its androgen independence, resistance to chemotherapy, and display of neuroendocrine features which can emerge either de novo or via transformation after a prior diagnosis of adenocarcinoma. The poor clinical outcomes in patients with AVPC are associated with its profound molecular heterogeneity. In this study, we analyzed 23 consecutive AVPC cases treated at a dedicated small-cell clinic (2017-2025) using clinicogenomic and transcriptomic profiling. Transformed AVPC exhibited significantly shorter overall survival times than de novo AVPC (11.8 vs 26.0 months, P < 0.001). Integrative genomic analyses identified residual androgen signaling in subsets of cases harboring neuroendocrine lineage programs, highlighting a decoupling of lineage identity and morphology. To facilitate mechanistic and pharmacologic studies, we established NCI-LYM-1, a patient-derived organoid/PDX from an AR-negative, ASCL1+/SYP+ lymph node metastasis, which faithfully recapitulates the donor tumor's molecular and phenotypic features. Short- and long-read whole-genome sequencing combined with optical genome mapping identified biallelic inactivation of PTEN, TP53, RB1 and BRCA2 as potential drivers, demonstrating clonal concordance with circulating tumor DNA from the original patient donor. Pathway and perturbation analyses suggested that NCI-LYM-1 harbored a strong dependency on apoptotic pathways, which was confirmed by in vitro organoid testing with the BCL-2/BCL-xL inhibitor navitoclax (IC50: 0.27 M) and the MCL-1 inhibitor AZD-5991 (IC50: 0.060 M). Overall, NCI&-LYM-1 recapitulates the clinical aggressiveness and heterogeneity of AVPC, providing a tractable platform to identify novel precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e71e79208c8d37fdc48221a6ff9b64fb28fe818c" target='_blank'>
              Molecular decoupling of lineage identity and morphology in aggressive variant prostate cancer
              </a>
            </td>
          <td>
            Chennan Li, JuanJuan Yin, Melissa L Abel, Dana S Vargas Solivan, Kinjal Bhadresha, Sumeyra Kartal, Samantha Nichols, Kanak Parmar, Joseph Twohig, Tri M Truong, C. Chau, Kathleen Kelly, W. Figg, Anish Thomas, A. Sowalsky
          </td>
          <td>2026-01-09</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cef7edb3493fb7ab6d3516e83b3f06b2dd845210" target='_blank'>
              VAETracer: Mutation-Guided Lineage Reconstruction and Generational State Inference from scRNA-seq
              </a>
            </td>
          <td>
            Linnuo Pan, Kaiyu Wang, Luonan Chen
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The progression of prostate cancer (PCa) to metastatic disease remains a critical clinical challenge, making it essential to gain a detailed understanding of the factors that enable certain subclones to invade and spread. Building on our previous study, which examined genomic variations in benign and malignant prostate tissues by inferring copy number alterations (CNA) from spatial transcriptomics data1, this study aims to trace the spatial evolutionary trajectories of PCa subclones from the primary tumor to matched lymph node metastases and to identify alterations in associated microenvironments and early diagnostic markers. To achieve this, we performed spatial transcriptomics on entire prostate axial disks and patient-matched lymph node metastases from 10 individuals. Standard spatial transcriptomics (55 µm) was used for tissue profiling; from the spatial transcriptomics data, copy number alterations (CNAs) were inferred to identify distinct tumor subclones, and a clone-tree was constructed to describe their evolutionary relationships. Additionally, a high-resolution spatial transcriptomics (2 µm) technology was employed to gain a cellular-level view of the immediate tumor-microenvironment (TME) surrounding specific subclones. Analysis of over 1,000,000 barcoded regions identified many distinct tumor subclones, enabling us to trace their evolutionary trajectories spatially. We observed notable subclonal events within the lymph nodes, including polyclonal colonization, indicating multiple dissemination events during the evolution of the primary disease. Exploration of the immediate TME revealed significant cellular heterogeneity and upregulation of genes related to antigen presentation and inflammatory pathways concentrated near ancestral tumor clones, specifically at the tumor border. By focusing on the inferred CNA profiles of the metastasizing clone across patients, we identified several common features. Additionally, we were able to identify potential lethal disease as early as the diagnostic core needle biopsy stage by detecting features of metastasis and pinpointing the metastasizing clone. In summary, our study provides a detailed spatial map of PCa clonal evolution and dissemination, linking primary tumors to nodal metastases and revealing altered cell composition and gene expression around tumor clone borders. Importantly, it demonstrates subclonal events within lymph nodes, polyclonal colonization, and the potential to identify metastasizing clones as early as the diagnostic phase, which holds strong implications for precision oncology and early intervention.
 1. Erickson, A., He, M., Berglund, E. et al. Spatially resolved clonal copy number alterations in benign and malignant tissue. Nature 608, 360–367 (2022). https://doi.org/10.1038/s41586-022-05023-2



 Mengxiao He, Sandy Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, Ian G. Mills, Joakim Lundeberg, Alastair D. Lamb. Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604426087ba878274c47234a7ce00263acf256a4" target='_blank'>
              Abstract PR013: Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers
              </a>
            </td>
          <td>
            Mengxiao He, S. Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, I. Mills, J. Lundeberg, Alastair D. Lamb
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In cancer, extrachromosomal DNA (ecDNA) contributes to tumor heterogeneity and is associated with poor prognosis, but studies on patient-derived ecDNA are relatively limited at single-cell resolution. Here, we introduce scCirclehunter, a framework designed to identify ecDNA from scATAC-seq data and assign ecDNA to specific cell populations. Leveraging scCirclehunter and available glioblastoma (GBM) datasets, we uncover the inter-cellular heterogeneity of ecDNA-carrying cells across GBM patients and trace the trajectories of malignant cells within a single patient that harbors multiple ecDNAs. By integrating scRNA-seq data, we use ecNR2E1 as an example to demonstrate that ecDNA drives tumor progression in GBM through several mechanisms. Additionally, our findings suggest a potential link between ecDNA and increased mitochondrial transfer frequency. Overall, scCirclehunter provides a novel framework for analyzing patient-specific ecDNAs with single-cell precision, offering insights into the role of ecDNA-carrying cells in driving GBM heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c481a654d2e51f6263c828b5c2a73535459ef5" target='_blank'>
              scCirclehunter delineates ecDNA-containing cells using single-cell ATAC-seq, with a focus on glioblastoma
              </a>
            </td>
          <td>
            Rong Jiang, Zhengmao Lu, Fang Li, Yibei Zhu, Manqiu Yang, Shufan Zhang, Ping Wu, Chengliang Gong, Yiyuan Fei, Yonghua Sang, Yulun Huang, Jiong Jiong Guo, Moli Huang
          </td>
          <td>2025-12-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy characterised by remarkable cellular heterogeneity, which emerges early from the interplay of oncogenic KRAS signalling and inflammatory injury. However, the transcriptional, metabolic and functional properties of these pre-malignant cell states that initiate and drive PDAC progression remain elusive. Objective This study aimed to identify and functionally characterise the critical premalignant cell states that arise from this heterogeneity, to define novel biomarkers and targets for early intervention. Design Public and in-house scRNA-seq data of pancreatic tumour models were analysed to identify key subpopulations in early cellular heterogeneity. Genetic perturbation in KrasG12D-driven models was performed to assess functional impact. Mechanistic studies used TurboID proximity proteomics, epigenetic profiling and metabolic assays. Clinical relevance was validated in human PDAC cohorts. Results We identified LY6D as a marker of a distinct, gastric-like cell state that emerges early and persists throughout tumourigenesis. The LY6D+ population exhibits conserved stemness and a unique, pan-stage dependency on oxidative phosphorylation (OXPHOS). Genetic ablation of Ly6d specifically impaired the gastric lineage and delayed tumourigenesis, while its overexpression enhanced tumourigenic and metastatic potential. Mechanistically, the glycosylphosphatidylinositol (GPI)-anchored LY6D protein scaffolds a lipid raft-associated kinase network that drives FOSL1-dependent epigenetic-transcriptional reprogramming. In human PDAC, LY6D+ cells harbour stemness and Epithelial-Mesenchymal Transition (EMT) signatures, and high LY6D expression is an independent prognostic marker of poor survival. Conclusion Our work defines the LY6D+ gastric-like cell state as a key driver linking early pre-malignant heterogeneity to PDAC initiation and progression. LY6D represents a pan-stage therapeutic target and a candidate biomarker for early detection and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f42670138ad6a4407b9d3c018c7d78a840fd287d" target='_blank'>
              LY6D identifies persistent stem-like cells driving pancreatic tumourigenesis
              </a>
            </td>
          <td>
            Juanjuan Shi, Xian Wang, Yingying Tang, Shixin Meng, Zhengyan Zhang, Ping Lu, Junyi Xu, Feier Yu, Xueni Wang, Zheng Wang, Yongwei Sun, Jing Xue
          </td>
          <td>2026-01-16</td>
          <td>Gut</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 The spatial organization of the tumor microenvironment (TME) encodes the selective pressures that drive cancer evolution, but many tools focus on local cell contacts or single-modality readouts. We developed Tumor Landscape Analysis (TLA), a modular framework that borrows from landscape ecology and spatial statistics to quantify heterogeneity, fragmentation, and diversity across scales, from single cells to whole-slide ecosystems.



 TLA ingests whole-cell or nuclear centroids and region-level segmentations to compute (i) point statistics such as nearest-neighbor distance, Ripley’s H, and Morisita-Horn colocalization; (ii) kernel-smoothed local spatial profiles including Getis-Ord Gi* hot and cold spots; and (iii) fragmentation, shape, and diversity metrics such as patch density, contagion, shape index, and Shannon entropy. We also introduce Local Microenvironments (LMEs), unsupervised ecological niches defined by joint cell-type abundance and spatial mixing. Pilot analyses were performed in ductal carcinoma in situ (DCIS; n=353 samples, 175 patients) with cell-level maps of epithelial, lymphocyte, and fibroblast populations, and in pancreatic ductal adenocarcinoma (PDAC; n=203 resected cases after neoadjuvant therapy) using cancer and stroma segmentations.



 In DCIS, spatial relationships rather than raw abundance associated with recurrence risk. Shorter lymphocyte-to-fibroblast minimum nearest-neighbor distance and lower Ripley’s H, indicating greater intermixing, correlated with earlier recurrence independent of cell counts. This highlights immune–stromal proximity as a candidate eco-evolutionary driver of persistence. In PDAC, tissue-level fragmentation captured prognostic architecture after therapy. Two shape-based signatures emerged: (1) stromal uniformity, where lower standard deviation of stromal patch shape index associated with improved disease-free survival, consistent with a more constrained, less reactive stroma; and (2) tumor geometric complexity, where higher mean cancer patch shape index, reflecting more irregular or dendritic nests, associated with lower recurrence risk, whereas compact nests predicted worse outcomes. Across datasets, LME maps localized niche phenotypes, such as mixed-immune or segmented tumor habitats, that align with hypothesized selection landscapes and provide inputs for fragmentation analysis.



 TLA operationalizes an eco-evolutionary view of cancer by unifying interpretable, multiscale spatial metrics in a single workflow that works with any rasterized data input. By quantifying how habitats, edges, and mixing patterns are organized, TLA reveals niche structures and architectural motifs linked to clinical outcomes and complements foundation models and modality-specific pipelines with theory-grounded, clinically translatable descriptors. Ongoing work integrates TLA features with molecular data and outcomes to derive robust biomarkers and to inform adaptive, niche-targeted strategies in precision oncology.



 Ryan M. Carr, Luis Cisneros, Merih Deniz. Toruner, Martin E. Fernandez-Zapico, Carlo Maley. Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f5d8ed56157756f83dcf8a730cae9e8b632c46" target='_blank'>
              Abstract A015: Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments
              </a>
            </td>
          <td>
            Ryan M. Carr, L. Cisneros, M. D. Toruner, Martín E Fernandez-Zapico, Carlo C. Maley
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor heterogeneity is one of the key factors leading to diagnostic errors, difficulty in identifying subtle cellular differences, and the occurrence of immune escape. Although single-cell RNA sequencing (scRNA-seq) has elucidated the molecular characteristics and heterogeneity at the single-cell level, it fails to retain spatial location information. In contrast, spatial transcriptomics (ST) technology can simultaneously acquire gene expression information and cellular localization, providing a unique perspective for studying the tumor microenvironment, cell-cell interactions, and other aspects. The integrated application of scRNA-seq and ST technologies enables the analysis of tumor heterogeneity from both molecular and spatial dimensions, offering comprehensive information support for precision medicine. The complementary advantages of these two technologies not only overcome the limitations of traditional methods but also promote the research on complex mechanisms such as tumor heterogeneity and the tumor microenvironment, accelerating the translational process of clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9514d53fae21da5a677d774e552caceeac1e73" target='_blank'>
              Research Progress in Single-Cell RNA Sequencing and Spatial Transcriptomics for Precision Oncology
              </a>
            </td>
          <td>
            Minxin Mo
          </td>
          <td>2025-12-18</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity and plasticity. GBM tumors contain neoplastic cells within a fluid spectrum of diverse transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we develop scFOCAL (Single-Cell Framework for -Omics Connectivity and Analysis via L1000), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identify compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states, and leverage this capability to predict combination synergy. We validate the significance of these findings in vitro, ex vivo, and in vivo, and identify a combination of an OLIG2 inhibitor and Depatux-M for the treatment of GBM. Our studies suggest that scFOCAL identifies cell states that are sensitive and resistant to targeted therapies in GBM using a measure of cell and drug connectivity, which can be applied to identify new synergistic combinations. Glioblastoma is characterised by high levels of intratumoural heterogeneity and plasticity, hindering treatment. Here, the authors develop an analytical framework, scFOCAL, to predict the sensitivity of glioblastoma cell subpopulations to therapies based on reversal of disease transcriptional signatures to identify synergistic therapeutic combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef55e3b53c83b7969434910c5b4838c55ce9672" target='_blank'>
              Drug and single-cell gene expression integration identifies sensitive and resistant glioblastoma cell populations
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, R. Komotar, Macarena I. de la Fuente, Gregory Stein, Alexandre Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, Nagi G. Ayad
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Simple Summary Radiotherapy is one of the most effective treatments for cancer, but its success is often limited by tumor radioresistance and the complex interactions within the tumor microenvironment. Patient-derived organoids are miniature, three-dimensional models grown from a patient’s own tumor that closely mimic the biology and treatment response of real tumors. These organoids provide a promising tool to study why some tumors resist radiation and to test strategies that make radiotherapy more effective. This review summarizes how organoids are used to explore the mechanisms of radioresistance—such as DNA repair, tumor metabolism, immune remodeling, and radiation-induced senescence—and discusses how integrating organoid models with immune and vascular components could accelerate the discovery of personalized radiosensitizers and improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95eba7f430f22360f6d996dcc784b52bde1ba4ff" target='_blank'>
              Patient-Derived Organoids as a Platform to Decipher and Overcome Radioresistance: From the Tumor Microenvironment to Radiosensitizer Discovery
              </a>
            </td>
          <td>
            Dashan Yin, Xiujuan Hong, Xiaoqi Wang, Wenjia Ding, Chenli Wang, Jin Qian, Yi Zhou, Chuan Sun, Zhibing Wu
          </td>
          <td>2025-12-01</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A central challenge in cancer research is to identify the secreted factors that sustain tumor cell survival. This is best exemplified in Hodgkin lymphoma, where malignant cells constitute a minor fraction of the tumor and rely on signals from the microenvironment for survival. Using genome-wide transcriptional profiling with spatial and single-cell resolution, we show that the neighborhood around malignant cells forms a distinct niche of 31 non-malignant cell types, enriched in helper T cells and myeloid cells, but depleted of plasma cells. Moreover, our spatial analysis nominates IL13 as a candidate survival factor. Recombinant IL13 augments malignant cell growth in vitro, and genome-wide loss-of-function screens across >1000 human cancer cell lines identify IL4R and IL13RA1, heterodimeric components of the IL13 receptor, as uniquely essential in Hodgkin lymphoma. Importantly, blocking antibodies phenocopy genetic inactivation. Our findings provide a biological rationale for testing IL13-directed therapies, which are already FDA-approved, in Hodgkin lymphoma. Hodgkin Reed Sternberg (HRS) cells and their surrounding microenvironment in Hodgkin lymphoma remain poorly characterized. Here, the authors perform genome-wide transcriptional profiling with spatial and single-cell resolution to explore the cellular and molecular composition of the Hodgkin lymphoma microenvironment and used machine learning to identify IL13 as a potential HRS cell survival factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdc2f4fd014e0601edcc3f78406fd8afaf2448ea" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, D. Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson A. Weir, O. Ashenberg, Geraldine S Pinkus, Scott J. Rodig, Caroline Uhler, Evan Z. Macosko, Margaret Shipp, A. Louissaint, Fei Chen, Todd R Golub
          </td>
          <td>2025-12-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Molecular heterogeneity in head and neck squamous cell carcinoma (HNSCC) is well recognized, yet existing subtype frameworks remain largely descriptive and have not translated into therapeutic decision-making. Here, we establish a mechanistic platform that converts transcriptomic diversity into drug-actionable tumor states. Integrating multi-cohort RNA-seq from 727 tumors across five independent datasets, genome-scale CRISPR dependency maps, and pharmacologic screening, we define distinct tumor survival circuits across HPV-negative HNSCC and nominate subtype-matched therapeutic strategies. These circuits encompass a proliferative axis (MYC, MET/FAK, inflammatory and translational programs), an epithelial-differentiated/adhesion program, an EMT-like state with stromal activation, and mitochondrial/oxidative metabolic states, each mapping to selective liabilities (e.g., mitotic/autophagy control, ERBB/PI3K and cadherin signaling, OXPHOS/mitochondrial translation, and G2/M-integrin-Notch pathways, respectively). We then develop a transcriptomic predictor of EGFR-inhibitor response using machine learning and validate it in prospectively collected, fresh patient-derived 3D microtumors. The resulting 13-gene signature identifies erlotinib-responsive tumors (R = 0.93) and maps biologically to an epithelial-differentiated state, outperforming EGFR expression alone. Our study establishes a subtype-to-dependency-to-therapy framework, enabling precision stratification and providing a clinically feasible path for prospective biomarker deployment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29377070a0d6fb125a75d4b7cf89275cb55faab" target='_blank'>
              Subtype-Specific Dependencies and Drug Vulnerabilities Enable Precision Therapeutics in Head and Neck Cancer
              </a>
            </td>
          <td>
            Joel M Vaz, Songli Zhu, Mateo Useche, Lauren Shih, Emily Marchiano, Slobodan Beronja, Bruce E Clurman, Cristina Rodriguez, Brittany Barber, Marina Chan, Taranjit S. Gujral
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gastric cancer (GC), of which cases with peritoneal metastasis are particularly challenging, retains its position of being highly complex and remarkably resistant to therapy. Understanding the spatial heterogeneity and leveraging recent technologies such as machine learning to uncover explanatory patterns remains critical to truly understanding this disease. Here, we conducted spatial transcriptomics analysis to identify distinct niches within GC tissues. Among these, a niche enriched with fibroblasts and macrophages exhibited a striking spatial co-infiltration pattern with tumor cells dominant niches. Further validation by multiplex immunofluorescence highlighted the coordinated cellular interactions that characterize the TME. Through integration of sc-RNA with bulk RNA sequencing, we identified DAB2⁺ TAMs and ACTA2⁺ myCAFs as the main contributors to this co-infiltration pattern. NicheNet analysis further revealed that the PLAU-PLAUR signaling axis holds a central regulatory role in the communication between macrophages, fibroblasts and tumor cells. Given the prognostic value of this spatial pattern, we additionally applied transfer learning based on an ImageNet pre-trained ResNet-50 model to develop a machine learning framework that can accurately recognize the macrophage-fibroblast-malignant cell co-infiltration pattern, called Gastric-Discovery. Potentially, Gastric-Discovery could be a tool for precise patient stratification and provides novel insights into the dynamic architecture of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ca98a4e613e1447cb2daf8bdbf25c8434e308a3" target='_blank'>
              Multi-dimensional omics integrated machine learning framework identifies macrophage-fibroblast-tumor co-infiltration patterns to predict prognosis in gastric cancer
              </a>
            </td>
          <td>
            Qi Wang, Yuan Ni, Sheng Lu, Benyan Zhang, J. Ji, Qu Cai, Chao Yan, Feng Qi, M. Shi, Jun Zhang
          </td>
          <td>2025-12-09</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Angioimmunoblastic T-cell lymphoma (AITL) represents the second most prevalent subtype of peripheral T-cell lymphoma, characterized by a dismal prognosis. However, a systematic exploration of tumor microenvironment (TME) features and their prognostic significance in AITL remains notably deficient. To address this knowledge gap, we conducted spatial transcriptome sequencing (ST-SEQ) and whole-exome sequencing in four AITLs and two noncancerous lymph nodes for discovery purposes, complemented by immunohistochemistry analyses on 37 AITL cases for validation. We identified 14 ST clusters, including five neoplastic clusters, wherein a global shift in B-cell phenotypes and enrichment of myeloid cells were observed. These findings underscore a hallmark of exacerbated inflammation and immune dysregulation within the neoplastic TME. Among the 4 ST-sequenced AITLs, 3 expressed high CD40-CD40LG activity, accompanied by the upregulation of immune-suppressive-associated genes, such as CCL17 and PDCD1. Conversely, the remaining patient displayed an uncommon absence of CD40-CD40LG activity but harbored a phagocytosis-associated tumor-associated macrophage (TAM)-enriched TME, which correlated with significantly reduced relapse rates and longer event-free survival (EFS), highlighting the critical value of precise TME stratification in tailoring AITL therapeutic strategies. Finally, trajectory analysis unveiled a distinct trajectory of molecular evolution within this TME landscape. Collectively, our findings illuminate the heterogeneity and prognostic implications of the TME in AITL, providing a robust foundation for the rational design of targeted immunotherapeutic approaches. These insights may substantially advance the development of personalized treatment strategies for AITL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47887a6371994feccf99a507b06079fcceafb4cb" target='_blank'>
              Revealing tumor microenvironmental heterogeneity and prognostic value in angioimmunoblastic T-cell lymphoma via spatial transcriptome sequencing
              </a>
            </td>
          <td>
            Xiang Zhang, Yong Sun, Duo Wu, Fang Yu, Hanjin Yang, X. Ye, Juying Wei, Xuewu Zhang, Yanan Zhu, Yunfei Lv, Zijing Xu, Yuxiang Chen, Hongyan Tong, Jie Jin, Xiaofei Ye, Wenjuan Yu
          </td>
          <td>2026-01-09</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) enables simultaneous mapping of tissue morphology and spatially resolved gene expression, offering unique opportunities to study tumor microenvironment heterogeneity. Here, we introduce a computational framework that predicts spatial pathway activity directly from hematoxylin-and-eosin-stained histology images at microscale resolution 55 and 100 um. Using image features derived from a computational pathology foundation model, we found that TGFb signaling was the most accurately predicted pathway across three independent breast and lung cancer ST datasets. In 87-88% of reliably predicted cases, the resulting spatial TGFb activity maps reflected the expected contrast between tumor and adjacent non-tumor regions, consistent with the known role of TGFb in regulating interactions within the tumor microenvironment. Notably, linear and nonlinear predictive models performed similarly, suggesting that image features may relate to pathway activity in a predominantly linear fashion or that nonlinear structure is small relative to measurement noise. These findings demonstrate that features extracted from routine histopathology may recover spatially coherent and biologically interpretable pathway patterns, offering a scalable strategy for integrating image-based inference with ST information in tumor microenvironment studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29b8964f22d869e7cd99fbb783574fbfe821242b" target='_blank'>
              Digital Modeling of Spatial Pathway Activity from Histology Reveals Tumor Microenvironment Heterogeneity
              </a>
            </td>
          <td>
            Ling Liao, Changhuei Yang, Maxim N. Artyomov, Mark Watson, Adam Kepecs, Haowen Zhou, Alexey Sergushichev, R. Cote
          </td>
          <td>2025-12-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db951a4a2419b28fdecf539ba062b45ce28797e3" target='_blank'>
              Spatial Regression of Morphology-Protein Coupling in Tumour Proteomics
              </a>
            </td>
          <td>
            Alejandro Leyva, M. Khalid, Khan Niazi
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Prostate cancer is a highly heterogeneous disease, driven by genomic and transcriptional changes that impact disease progression and metastatic potential. The interplay between clonal evolution, transcriptional plasticity, and tumour microenvironment is, however, poorly understood. Here, we leverage and integrate single-nuclei RNA sequencing and whole-genome sequencing from 43 spatially distinct tumour samples from five patients with locally advanced prostate cancer to reconstruct clonal evolution trajectories and transcriptional changes driving metastasis at single-cell resolution. We find extensive clonal heterogeneity, including both monophyletic and polyphyletic metastatic dissemination, and ongoing clonal evolution in the primary tumour after metastatic spread. Metastatic seeding converges on disease trajectories involving both genomic and transcriptional changes, including androgen receptor independence and activation of estrogen-, WNT- and JAK-STAT- pathway activity, in spatially distinct areas. Our findings suggest an intricate interplay between clonal evolution and cellular plasticity driving metastatic seeding and point toward more integrative prognostic markers for improved patient management. The impact of tumour heterogeneity on metastatic potential in prostate cancer remains poorly understood. Here, the analysis of single nuclei RNA sequencing and whole-genome sequencing from samples from five patients suggests an interplay between clonal evolution and cellular plasticity driving metastatic seeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea384ac37d8c4c2acd2bd5fc9ab62cffd7ea81c" target='_blank'>
              Clonal evolution and transcriptional plasticity shape metastatic dissemination routes in prostate cancer
              </a>
            </td>
          <td>
            Migle Mikutenaite, Evdoxia Karadoulama, F. Favero, A. Locallo, Francisco German Rodriguez Gonzalez, Daria Kiriy, Tina Keshavarzian, K. Furlano, M. Graefen, Robert G. Bristow, Clarissa Gerhäuser, C. Plass, G. Sauter, R. Simon, M. Lupien, S. Minner, Thorsten Schlomm, Joachim Weischenfeldt
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="


 Prostate cancer is genomically heterogeneous with few shared drivers. We characterized tumor-specific DNA methylation programs that define evolutionary subtypes, remodel the tumor–microenvironment ecosystem, and yield liquid-biopsy biomarkers.



 Illumina EPIC-seq methylation was profiled in matched benign/tumor tissues from the UK Prostate ICGC cohort. A covariance-based DMR caller identified robust tumor-specific regions that were intersected with bulk RNA-seq, network analyses, and single-cell/spatial transcriptomics to localize programs in situ. A FACS-sorted, cell-type methylome atlas (luminal, basal, endothelial, fibroblast, immune, smooth muscle) anchored cell-of-origin assignments. Cell-free DNA was deconvolved against this atlas to quantify luminal tumor DNA and circulating microenvironmental DNA (cmDNA). Prognostic associations used penalized Cox models.



 Tumor-specific DMRs separated cancers into two “mevotypes” mirroring genomic evotypes (ERG-fusion vs SPOP-mutant), indicating genomic–epigenomic interplay in evolutionary commitment. Cell-resolved methylomes attributed most tumor DMRs to luminal epithelium and uncovered neuronal/plasticity pathways in luminal cells, while immune/endothelial-assigned DMRs converged on ECM remodeling, angiogenesis, TGF-β, and Rho-GTPase signaling. Single-cell pseudotime and spatial mapping localized neuronal-enriched luminal programs to late-trajectory regions and a CHGA/CHGB-positive duct-adjacent epithelial circuit interfacing with fibroblasts at tumor–stroma borders. Network analysis highlighted MYC-centric modules and EZH2-linked transcription factor contexts, consistent with epigenetic priming toward quasi-neuroendocrine states in treatment-naïve localized adenocarcinoma. In plasma, deconvolution detected both luminal ctDNA and cmDNA; luminal fractions tracked tumor burden in metastatic disease (ichorCNA-concordant), whereas cmDNA—predominantly attributable to immune compartments—suggested an early, tumor-specific microenvironmental response that can precede and is independent of luminal ctDNA shedding. Methylation signatures derived from neuronal and microenvironmental gene sets associated with inferior relapse-free survival.



 Epigenetic drivers in early prostate cancer encode luminal neuronal plasticity and reorganize the tumor niche. A cell-resolved atlas enables cfDNA deconvolution to reveal luminal ctDNA plus cmDNA, supporting liquid-biopsy biomarker strategies for earlier detection and improved risk stratification in prostate cancer.



 Harveer Dev, Toby Milne-Clark, Henson Lee Yu, CRUK-ICGC Prostate Cancer Group. Epigenetic Drivers of Localised Prostate Cancers Reveal Neuronal and Microenvironmental Reprogramming and inform Liquid Biopsy Biomarkers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2503538f0dcd516b3bf974c3bb6eeaf7a1544a19" target='_blank'>
              Abstract A015: Epigenetic Drivers of Localised Prostate Cancers Reveal Neuronal and Microenvironmental Reprogramming and inform Liquid Biopsy Biomarkers
              </a>
            </td>
          <td>
            H. Dev, Toby Milne-Clark, Henson L. Lee Yu
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="HCC is characterized by extensive ECM remodeling, primarily mediated by stromal cells. While cancer-associated fibroblasts are known contributors to tumor fibrosis, their transcriptional diversity and role in immune modulation within the HCC tumor microenvironment remain poorly resolved. Integrative analysis of six public HCC scRNA-seq datasets was employed to identify ECM-active stromal cells. High-ECM cells were re-clustered and fibroblast subtypes were defined through differential expression, pathway enrichment, lineage scoring and cell-cell interaction modeling. Spatial transcriptomic mapping and tissue-level profiling were performed to validate and prioritize ECM-related genes for experimental analysis. COL1A1 and COL3A1 were selected for siRNA-mediated knockdown in HepG2 and HepB3 cells followed by RT-qPCR, Western blot, proliferation, colony formation and wound healing assays. ECM activity analysis identified fibroblasts as the most ECM-enriched stromal population. Reclustering of ECM-high stromal cells identified eight distinct subtypes and targeted fibroblast clustering revealed five functionally diverse states. GO and spatial transcriptomic analysis confirmed subtype-specific functions and localization. In silico tissue profiling further prioritized COL1A1 and COL3A1 as pan-mesenchymal ECM genes enriched in fibrogenic fibroblast subsets. CellChat analysis revealed myofibroblasts and inflammatory CAFs as dominant signal senders. Knockdown reduced COL1A1 and COL3A1 expression at both mRNA and protein levels, and enhanced HCC cell proliferation, migration and colony formation. This study identifies transcriptionally and functionally distinct fibroblast subtypes in HCC and highlights COL1A1 and COL3A1 as key matrix-regulatory genes expressed in fibrogenic stromal subsets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2fd864648cbb63d6bef938be32a938f167aaf27" target='_blank'>
              Single-cell profiling reveals lineage-specific fibroblast stromal subtypes drive ECM remodeling and immune modulation in the hepatocellular carcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Zhichao Jiang, Haitao Wang, Han Li, Zhiqiang Chen, Baiyu Sun
          </td>
          <td>2026-01-03</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is a highly prevalent malignancy with poor prognosis, largely driven by lymph node metastasis (LNM). Despite its clinical significance, the underlying mechanisms of LNM remain elusive. In this study, we used single-cell transcriptomic data to dissect the cellular and molecular interactions within metastatic lymph nodes (MET). Specifically, we analyzed Single-cell RNA sequencing (scRNA-seq) data from GSE195655, GSE140042, GSE227156, and GSE159929 (n = 41) to delineate cellular heterogeneity, intercellular communication networks, and functional enrichment in primary tumors (PT), MET, and non-metastatic lymph nodes (LN). Our analysis identified a subpopulation of tumor-associated macrophages (TAMs) distinctly enriched in MET, characterized by high expression of SPP1. Functional analysis revealed that this TAM subpopulation promotes angiogenesis through specific ligand-receptor interactions with endothelial cells (ECs), involving the SPP1-ITGa9b1 and FN1-ITGa2b1 signaling axes. Furthermore, we leveraged bulk RNA-seq for prognostic research. Immunohistochemistry (IHC) confirmed the increased density and number of blood and lymphatic vessels in MET. Spatial analysis via multiplex immunohistochemistry (mIHC) confirmed the preferential localization of SPP1+ TAMs near ECs in MET. These results suggest a potential communication between TAMs and ECs that contributes to LNM in HNSCC, providing critical insights for HNSCC prognosis and precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa35cd24399d837bd70f92cab5dbd70391a0453c" target='_blank'>
              Single-cell identification of an endothelial cell proximal SPP1+ macrophage population defines the metastatic vascular niche in lymph nodes
              </a>
            </td>
          <td>
            Zhentao Lao, Rukeng Tan, Mailikanmuhan Maitikuerban, Xiaofang Xu, Sien Zhang, Jingxin Ma, Dikan Wang, Guiqing Liao, Le Yang
          </td>
          <td>2025-12-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Interactions among tumor, immune, and vascular niches play major roles in glioblastoma (GBM) malignancy and treatment responses. The composition and heterogeneity of extracellular core matrix proteins (CMPs) that mediate such interactions are not well understood. Here, we present an analysis of the clinical relevance of CMP expression in GBM at bulk, single-cell, and spatial anatomical resolution. We show that CMP enrichment is associated with worse patient survival, specific driver oncogenic alterations, mesenchymal state, pro-tumor immune infiltration, and immune checkpoint expression. Matrisome expression is enriched in vascular and leading edge/infiltrative niches that are known to harbor glioma stem cells. Finally, we identify a 17-gene CMP signature, termed Matrisome 17 (M17), which is a stronger prognostic factor compared to MGMT promoter methylation status as well as canonical subtypes, and importantly, may predict responses to PD1 blockade. Patient stratification based on matrisome profiles can contribute to the selection and optimization of treatment strategies. A comprehensive genomic analysis of glioblastoma tumors reveals matrisome enrichment associated with mesenchymal phenotypes and pro-tumor immune signatures, a matrisome-based classification of GBM patients, and a prognostic 17-gene matrisome signature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984fecbf9126f6a0596ab278d28e0d91cbca56fc" target='_blank'>
              A prognostic matrix gene expression signature defines functional glioblastoma phenotypes and niches
              </a>
            </td>
          <td>
            Monika Vishnoi, Zeynep Dereli, Zheng Yin, Elisabeth Kong, Meric Kinali, Kisan Thapa, O. Babur, Kyuson Yun, Nourhan Abdelfattah, Xubin Li, B. Bozorgui, M. Farach‐Carson, Robert C. Rostomily, Anil Korkut
          </td>
          <td>2026-01-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6aac86da3cbb3bd85a8ebe57c0ff220d334a71" target='_blank'>
              Circulating immune profiling reveals impaired monocyte states and trajectories driving immunosuppression in glioblastoma
              </a>
            </td>
          <td>
            A. Scafidi, T. Kaoma, Claudia Cerella, Eleonora Campus, K. Grzyb, Bakhtiyor Nosirov, Frida Lind-Holm Mogensen, E. Klein, Raul da Costa, A. Skupin, Frank Hertel, G. Berchem, Michel Mittelbronn, A. Cosma, Beatrice Melin, Simone P. Niclou, Petr V Nazarov, A. Gołȩbiewska, Aurélie Poli, A. Michelucci
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Tumor metastasis is the leading cause of death in cancer patients, with epithelial–mesenchymal transition (EMT) playing a key role. To systematically elucidate the cellular mechanisms and molecular networks through which EMT drives metastasis across cancers, this study integrated transcriptomic data from over 1.2 million single cells across 265 samples representing 12 primary epithelial cancers, constructing a comprehensive pan-cancer single-cell atlas covering diverse stages and metastatic states. By analyzing the metastatic features and interaction networks of malignant epithelial cells and cancer-associated fibroblasts (CAFs), we identified cancer-specific metastasis-related gene sets. Based on these genes, multiple machine learning algorithms were applied to build cancer-specific and cross-cancer metastasis prediction models, leading to the development of the metastasis prediction score (MPS) and global metastasis prediction score (GMPS). Both scores showed excellent predictive performance in independent test and external validation cohorts. MPS exhibited higher cancer specificity, whereas GMPS showed stronger cross-cancer generalization. Moreover, elevated MPS and GMPS reflected immunosuppressive tumor microenvironment features and were significantly associated with poor prognosis across multiple cancer types. Finally, through a drug repositioning framework, we identified several potential anti-metastatic compounds targeting the metastasis network, among which Fostamatinib demonstrated broad-spectrum therapeutic potential against metastasis in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e04f260cc09578e40147449863bf3ac22891b56d" target='_blank'>
              Construction of Metastasis Prediction Models and Screening of Anti-Metastatic Drugs Based on Pan-Cancer Single-Cell EMT Features
              </a>
            </td>
          <td>
            Yingqi Xu, Yawen Luo, Maohao Li, Na Lv, Yuanyuan Deng, Ning Li, Shichao Wan, Xing Gao, Xia Li, Congxue Hu
          </td>
          <td>2025-11-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) exhibits early systemic dissemination that precedes clinical detection, challenging the classical model of metastasis as a late evolutionary event. Mounting evidence indicates that molecular and cellular programs enabling invasion and distant colonization emerge at preinvasive or early carcinoma stages. This mini-review synthesizes recent advances defining how tumor-intrinsic mechanisms, such as KRAS-driven basal extrusion, epithelial–mesenchymal plasticity and chromosomal instability, cooperate with microenvironmental cues to promote early metastatic competence. The desmoplastic stroma, rich in fibroblasts, inflammatory mediators and aligned extracellular matrix, provides both structural and biochemical support for tumor-cell escape. Additionally, neural and immune interactions, including chemokine and cytokine signaling, facilitate perineural invasion and systemic pre-metastatic niche (PMN) conditioning via extracellular vesicles and cytokine networks. Recognizing PDAC as a systemic disease from inception reframes therapeutic priorities: early systemic therapy and biomarker-guided patient stratification may intercept occult dissemination. We propose that integrating mechanistic insights on tumor–microenvironment crosstalk with perioperative and liquid-biopsy-driven clinical strategies could redefine early intervention in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92dd387af76159f510d779a0044bf468159245fa" target='_blank'>
              The molecular logic of early metastasis in pancreatic cancer: crosstalk between tumor and microenvironment
              </a>
            </td>
          <td>
            Andrea Costamagna, Noemi Ghiglione, M. Martini
          </td>
          <td>2025-11-28</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a31e93002156cd8296588a7c36e93349dfff416" target='_blank'>
              Defining the cellular and molecular identities of histologic subtypes in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Jusung Lee, J. Y. Jeong, Mi Jeong Hong, Yoon Ha Choi, Ju Young Kim, Jang Hyuck Lee, Jin Eun Choi, Moonsik Kim, Young Woo Do, E. Lee, Sun Ha Choi, Seung Soo Yoo, Jae-Yong Park, Jong Kyoung Kim, S. Lee
          </td>
          <td>2026-01-23</td>
          <td>Experimental hematology & oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background/Objectives: Achieving complete resection (R0) during secondary cytoreduction (SCR) is a pivotal prognostic factor for patients with platinum-sensitive relapsed ovarian cancer (PSROC). Methods: By integrating single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and functional assays on primary cells, we identified tumor microenvironment (TME) heterogeneity, particularly the presence of distinct subpopulations of cancer-associated fibroblasts (CAFs), as a critical determinant of surgical outcomes for SCR. Results: We characterized a previously unrecognized CAF-C7 subpopulation that was selectively enriched in tumors from non-R0 patients. Mechanistically, CAF-C7 promoted tumor cell migration, suppressed anoikis, and facilitated angiogenesis via the IGF1-IGF1R signaling axis, thereby contributing to multifocal recurrence and reducing the likelihood of complete resection. Importantly, the inhibition of the IGF1-IGF1R pathway effectively attenuated the pro-tumorigenic functions of CAF-C7. Conclusions: These findings uncover a novel cellular driver of unfavorable surgical outcomes in PSROC and suggest promising biomarkers and therapeutic targets for improving patient stratification and treatment in the context of SCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c69a7316901a4f26ec9560008414f1bd2cba29" target='_blank'>
              Single-Cell Analysis Links C7+ Cancer-Associated Fibroblasts with Incomplete Resection in Platinum-Sensitive Relapsed Ovarian Cancer
              </a>
            </td>
          <td>
            Longxia Li, Shangbing Gao, Yilizhati Maimaiti, Lifeng Lin, Xiaoxia Xing, Wei Wu, Yulian Chen, Mei-Chun Cai, Guanglei Zhuang, Rongyu Zang
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Low-grade serous ovarian carcinoma (LGSOC) is a rare, treatment-resistant subtype of ovarian cancer characterized by peritoneal spread. Pressurized Intraperitoneal Aerosol Chemotherapy (PIPAC) delivers chemotherapy directly into the abdomen during serial laparoscopic surgeries, enabling repeated sampling of tumors for longitudinal analysis. Using single-nuclei RNA sequencing on 15 tumor samples across PIPAC cycles in two patients (a responder and a non-responder), we tracked molecular and cellular changes over time. Non-responder tumors showed upregulation of proliferation pathways (E2F, MYC, G2/M), KRAS signaling, epithelial-mesenchymal transition, and unfolded protein response, correlating with resistance. Responder tumors exhibited downregulation of proliferation and stress pathways but activated alternative survival mechanisms (PI3K, Wnt/β-catenin, Notch). Archetype analysis revealed dynamic shifts in metabolic and immune-related subpopulations in the responder tumors, while immunoprofiling showed greater immunosuppression in non-responder tumors. Despite the small cohort, these exploratory observations highlight tumor adaptation and underscore the need for multi-targeted therapies addressing proliferation, stress, survival, and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03dd1cf18442a46d00407d6ee891f98b166026d8" target='_blank'>
              Divergent oncogenic signaling and immune microenvironment changes in low-grade serous ovarian cancer patients undergoing intraperitoneal chemotherapy
              </a>
            </td>
          <td>
            Isaac Bishara, Patrick A. Cosgrove, Sumana Majumdar, Colt A. Egelston, Oscar Colunga Flores, Brad Nakamura, Nora H. Ruel, Paul Frankel, S. Yost, Sue Chang, Alexander Jung, Mihaela Cristea, Mustafa Raoof, Andrea H. Bild, Aritro Nath, T. Dellinger
          </td>
          <td>2025-12-14</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Aggressive variant prostate cancers (AVPC) are a poorly understood subset of prostate cancers characterized by high mortality and extensive molecular heterogeneity. To address a lack of AVPC representative models for investigating underlying molecular and physiological mechanisms, we established NCI-LYM-1, a novel AVPC organoid/patient derived xenograft (PDX) derived from a patient with metastatic prostate cancer with small cell morphology displaying a unique clinical history and defined genomic features. It was hypothesized that this model would be able to accurately characterize biological underpinnings of the patient’s tumor profile and allow for comparative analysis to similar prostate cancers.



 NCI-LYM-1 was developed from a patient-derived AR-negative, ASCL1/Synaptophysin (SYP)-positive lymph node metastasis with small cell morphology. Short-read whole-genome sequencing (WGS) was used to identify genomic alterations and long-read WGS was used to identify epigenetic modifications. RNA-seq was used to characterize gene expression profiles. The resulting genomic, epigenetic, and transcriptomic data from NCI-LYM-1 was compared to molecular profiles from the LuCaP series of PDX models. In-vitro drug sensitivity assays were also conducted using organoid cultures.



 NCI-LYM-1 harbored biallelic inactivation of known neuroendocrine prostate cancer (NEPC) drivers: PTEN, TP53, RB1 and BRCA2, in concordance with patient diagnostic history. In addition, epigenetic evidence supported a consistent phenotype between NCI-LYM-1 and the donor tumor, including greater gene expression of neuroendocrine marker ASCL1 with decreased AR expression. Comparisons between NCI-LYM-1 and LuCaP models further supported transcriptional similarities between the model and a subset of AVPC cancer subtypes. Drug sensitivity assays were also consistent with the patient’s prior treatment exposure, displaying PARP inhibitor sensitivity but a lack of sensitivity to platinum treatments.



 NCI-LYM-1 faithfully recapitulates the phenotypic and genomic characteristics of aggressive metastatic AVPCs, including the display of neuroendocrine features. NCI-LYM-1 will be a useful model system to study molecular frameworks of understudied cancers like those of AVPC, including aspects such as drug sensitivity, disease progression, and lineage plasticity.



 Ana S. Jaramillo, Chennan Li, JuanJuan Yin, Melissa L. Abel, Dana S. Vargas, Kinjal Bhadresha, Sumeyra Kartal, Joseph Twohig, Tri M. Truong, Anish Thomas, Cindy H. Chau, Kathleen Kelly, William D. Figg, Adam G. Sowalsky. NCI-LYM-1 molecularly and phenotypically recapitulates aggressive variant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483313e79c5e529d10092ffd1001e71580f4403a" target='_blank'>
              Abstract A025: NCI-LYM-1 molecularly and phenotypically recapitulates aggressive variant prostate cancer
              </a>
            </td>
          <td>
            Ana S. Jaramillo, Chennan Li, JuanJuan Yin, Melissa L Abel, Dana S. Vargas, Kinjal Bhadresha, Sumeyra Kartal, Joseph Twohig, Tri M Truong, Anish Thomas, C. Chau, Kathleen Kelly, W. Figg, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35ef55ec239800141399bd7138122aefbaef8e7" target='_blank'>
              Reconstructing developmental and disease progression with sample-level embeddings
              </a>
            </td>
          <td>
            Longda Jiang, Zhixin Cyrillus Tan, Isabella N. Grabski, Yuhan Hao, Nathan Nakatsuka, Sourav Sarkar, Anagha Shenoy, R. Satija
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Background: Tertiary Lymphoid Structures (TLS) are organized aggregates of immune cells that form in non-lymphoid tissues in response to chronic inflammation, including cancer. In melanoma, TLS presence within the tumor microenvironment is associated with a pre-existing anti-tumor immunity and linked to improved immunotherapy responses. However, their role in predicting recurrence, particularly in early-stage melanoma, remains unclear.Methods: We conducted a retrospective analysis of archival primary tumor samples from 65 patients with stage I-II melanoma and negative sentinel lymph node (Table 1). Tumor immune infiltrates and TLS were initially assessed using four-marker immunohistochemistry (IHC) panels targeting CD20, CD4, CD8, and Podoplanin (PDPN), a marker of stromal cells. TLS were objectively identified using the HDBSCAN AI-based clustering algorithm. Tumor samples were further analyzed for high-dimensional phenotyping and spatial analysis using a 17 markers panel with hyperplexed immunofluorescence imaging (HIFI). Associations between TLS features and clinical outcomes were evaluated.Results: IHC revealed that 88% of patients had peritumoral T cell infiltration, and TLS were detected in 54% of cases. TLS-positive patients had improved disease-free survival (DFS, p=0.2) and overall survival (OS, p=0.2) (Figure 1A, B). To investigate further, HIFI was performed on 8 patients (4 recurrent, 4 non-recurrent), revealing 13 distinct cell types: stromal (n=6), tumor (n=2), and immune (n=5) (Figure 1C). Spatial analysis identified 9 recurrent spatial modules (SM), each representing a distinct pattern of interaction between cell types. One SM, enriched for PD-L1⁺ stromal cells, was significantly associated with recurrence (p=0.019) (Figure 1D).Conclusions: TLS presence associates with improved DFS and OS in early-stage melanoma. However, beyond their mere presence, TLS spatial organization and local stromal-immune context could influence clinical outcomes. Our data demonstrated that a specific spatial module—defined by the proximity of PD-L1⁺ stromal cells—is associated with disease recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/978d2823a740581a2a5f2a0fdbf9de6949d49e50" target='_blank'>
              13 | Spatial contexture of tertiary lymphoid structures as a predictor of recurrence in early-stage melanoma
              </a>
            </td>
          <td>
            Italian Melanoma Intergroup
          </td>
          <td>2025-12-05</td>
          <td>Dermatology Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131d5075adfc8a892d5a30b581d5ddea0c0afd87" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular heterogeneity and prognostic subtypes in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhen Han, Yuan Gao, Yuxing Zhao, Jiahong Wu, Jia Chen, Jinling He, Hong Bai, Liang Qiao, Yali Ren, Lei Sun
          </td>
          <td>2025-12-22</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Colorectal carcinoma exhibits high heterogeneity, comprising subtypes that show poor efficacy of T cell-based immunotherapies, such as programmed cell death protein 1 (PD-1) checkpoint inhibitors. Although natural killer (NK) cells are considered a promising approach for cancer immunotherapy, it remains unclear what molecular mechanisms drive NK cell activation or suppression within the tumor microenvironment. Moreover, limitations in human tumor models that reflect the diversity of individual patient tumors hinder the ability to effectively select patients who would benefit most from NK cell-based therapies. Methods Here, we established a co-culture platform of genetically diverse colorectal cancer (CRC) patient-derived organoids (PDOs) with primary allogeneic NK cells. We performed bulk RNA sequencing analysis of sorted NK cells after exposure to PDOs and aligned gene expression signatures derived from our findings with publicly available single-cell RNA sequencing data of NK cells from peripheral blood and CRC tissues of patients. Moreover, we evaluated identified pathways using flow cytometry and IncuCyte live-cell imaging analysis to quantify phenotypic alterations and NK cell-mediated killing of PDOs over time, respectively. Ultimately, we tested CRISPR-Cas9-edited NK cells and PDOs, small molecule compounds, and clinically relevant monoclonal antibodies (mAbs) to increase NK cell potency. Results On co-culture, NK cells acquired common transcriptional signatures related to hypoxia and transforming growth factor-beta (TGF-β), similar to NK cells infiltrating CRC tissues of patients. In addition, we observed patient-specific differential PDO susceptibilities to NK cell-mediated lysis. Major histocompatibility complex class I deficiency and natural killer group 2, member D (NKG2D)-ligand expression on PDOs facilitated NK cell-mediated cytotoxicity, and induced phenotypic NK cell diversification related to activation and the acquisition of inflammation and tissue-residency-related transcriptional signatures. Genetic or pharmaceutical targeting of hypoxia-inducible factors HIF1A/EPAS1 or TGF-βR1, or the addition of anti-CEACAM1 mAbs, enhanced NK cell-mediated PDO killing or activation, respectively. Conclusions The NK cell/PDO co-culture platform allows the identification of both common and patient-specific impacts of the tumor microenvironment on NK cell function and can aid the development of patient-tailored immunotherapies. The majority of CRC (CMS2/CMS3) PDOs from our cohort were susceptible to NK cell-mediated killing and induced NK cell activation, highlighting the potential of NK cells for CRC immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfdbb13391ad7497b66effc8a4643668f2be84a" target='_blank'>
              Colorectal cancer organoids drive hypoxia, TGF-β, and patient-specific diversification of NK cell activation programs
              </a>
            </td>
          <td>
            Andreas von Kries, Irene Garcés-Lázaro, Bianca M Balzasch, Carsten Sticht, Indra A Shaltiel, Kim E. Boonekamp, Annika Sams, Alessia Triassi, Tomáš Hofman, Elke Burgermeister, J. Betge, Matthias P. Ebert, M. Boutros, Laura Helming, A. Stojanovic, A. Cerwenka
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="The formation of intratumoral spatial niches has been reported for many human malignancies. However, the translational potential of such spatial niches is understudied. Herein, we utilize digital spatial profiling (DSP) to explore the prognostic relevance of spatially defined protein expression in high-risk prostate cancer. A total of 49 patient samples were analyzed for the expression of 46 proteins in 463 regions of interest (ROIs) from the tumor center (n = 198) and the tumor periphery (n = 265) resulting in 21,298 primary data points (mean per patient n = 9.4). Expression data from either the tumor center or the tumor periphery were not found to be prognostic. Protein expression of tumor center and periphery was then integrated into single datapoints by calculating the log2-transformed relative expression between the two niches for each protein and patient. Unsupervised hierarchical clustering of these data yielded two distinct patient subgroups. These clusters did not show a statistically significant correlation with known prognostic parameters yet significantly correlated with progression-free survival (p = 0.014, log-rank, HR 0.43; 95% CI, 0.22–0.86). Our results thus reveal that spatial protein expression contains prognostic information, however, only when expression data from both spatial niches are taken into account. In conclusion, our proof-of-concept study shows that DSP can be exploited for the development of novel prognostic biomarkers that rely on spatially resolved protein expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c6b3a4672d8a49cdbb7af4005dce769ab547c41" target='_blank'>
              Prognostic impact of spatial niches in prostate cancer
              </a>
            </td>
          <td>
            Felix Schneider, Sarah Böning, Beatriz Coelho Antunes, A. Kaczorowski, M. Görtz, V. Schütz, Johannes Huber, Albrecht Stenzinger, Markus Hohenfellner, S. Duensing, A. Duensing
          </td>
          <td>2026-01-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background While adipose tissue constitutes a substantial proportion of breast composition, the functional characteristics and pathological relevance of the adipocyte microenvironment in breast carcinogenesis remain undercharacterized. This study employs single-nucleus RNA sequencing (snRNA-seq) to establish a comprehensive cellular atlas of adipocyte heterogeneity across molecular subtypes of breast cancer, aiming to elucidate subtype-specific adipocyte contributions to tumor microenvironment modulation. Methods snRNA-seq was performed on breast adipose tissues isolated from individuals without cancer and treatment-naïve breast cancer. Various adipocyte and pre-adipocyte subclusters were identified. Comparative analyses of cellular distribution and transcriptional profiles were performed across disease states and molecular subtypes. Pseudotime, cell communication, and immunofluorescence analyses were further implemented to investigate cellular dynamics and microenvironment interactions. Results snRNA-seq data of 86,529 nuclei were obtained. Three adipocyte and seven pre-adipocyte subclusters were identified, of which Adi_LDLR, Pre_Adi_LDLR, and Pre_Adi_LGR4_TGFBR1 were restricted to cancer-associated adipose (CAAs). Adi_LDLR and Pre_Adi_LDLR were enriched in estrogen receptor (ER)-positive CAAs and related to cell senescence and immunosuppression. Pre_Adi_LGR4_TGFBR1 was predominantly present in triple-negative breast cancer, functionally pro-proliferative, immunosuppressive, and lacked normal adipose function. The immunofluorescence intensity of LDLR (p=0.031) and TGFBR1 (p=0.038) was positively associated with disease recurrence, suggesting the formation of immunosuppressive niches by these cancer-specific adipose subsets in both subtypes. Cell communication analyses revealed a specific (pre-) adipocyte-macrophage interaction via ligand-receptor pairs involved in stromal remodeling and tumor migration for ER-positive tumors, whereas tumor proliferation and metastasis for triple-negative ones likely contribute to tumor progression. Conclusions This study delineated a distinct adipocyte landscape in breast cancer and subtype-specific immunosuppressive niches fostered by CAAs and (pre-) adipocyte-macrophage interactions. These findings provide novel therapeutic targets for microenvironment-directed interventions in breast oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c4c519ddf0de4fc6c0a04e6ac2d29ba332f1031" target='_blank'>
              Decoding adipocyte heterogeneity through single-nucleus transcriptomics unveils subtype-specific adipocytes orchestrate immunosuppressive niches in breast cancer
              </a>
            </td>
          <td>
            Yiwei Tong, Zheng Wang, Niu Qiao, Renhong Huang, Chenghui Wu, Haoyu Wang, X. Fei, Kunwei Shen, Xiao-song Chen
          </td>
          <td>2026-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Thymic epithelial tumors (TETs) exhibit marked heterogeneity that challenges clinical management and prognostic evaluation. By consensus clustering of immune-related gene expression from public datasets and an in-house validation cohort, we identified two immune subtypes: a lymphocyte-rich subtype (LRS, n = 86) characterized by strong T-cell activity and favorable prognosis, and a myeloid/stromal-rich subtype (MSRS, n = 33) defined by immunosuppressive features, stemness, oncogenic alterations, and poorer outcomes. An APC gene-based classification panel demonstrated robust performance in distinguishing these subtypes, as confirmed by ROC curve analysis and validated in the Ruijin cohort using FFPE-RNA sequencing and multiplex immunofluorescence. Single-cell transcriptomics further delineated subtype-specific tumor microenvironment features and revealed that MSRS tumors suppressed CD8+ T-cell function through MIF-mediated mechanisms. Collectively, these findings establish a clinically relevant immune classification of TETs that integrates molecular, immunological, and clinical features, providing insights into tumor heterogeneity and supporting the development of personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f6f13a57b79f0fdbdce54ed88ac9cc1b39fa61" target='_blank'>
              Immune landscape-driven subtyping reveals distinct microenvironment and prognostic profiles in thymic epithelial tumors
              </a>
            </td>
          <td>
            Yajie Zhang, Tong Lu, Yeke Huang, Xipeng Wang, Xiaoyan Chen, Wei Guo, Youqiong Ye, Hecheng Li
          </td>
          <td>2025-12-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832c2464c80a294ed1bb7a8527b6613a4dd9fa1e" target='_blank'>
              Spatial gene expression analysis reveals drivers of extremely early lymph node metastasis in breast cancer.
              </a>
            </td>
          <td>
            Satoi Nagasawa, Keiko Kajiya, Erina Ishikawa, A. Kanai, Ayako Suzuki, A. Motoyoshi, Tsuguo Iwatani, Manabu Kubota, Masaru Nakamura, T. Onishi, Akiyoshi Hoshino, Ichiro Maeda, Akihiko Morozumi, Kenji Takatsuka, Junki Koike, M. Seki, K. Tsugawa, Yutaka Suzuki
          </td>
          <td>2026-01-23</td>
          <td>NPJ breast cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="

 The complexities of nucleotide metabolism in breast cancer (BC) cells are not yet fully understood. A deeper exploration of the various tumor subpopulations and their interactions with the tumor microenvironment (TME) could provide important insights into these clinically relevant signaling pathways.



 We integrated five distinct single-cell enrichment scoring methodologies to conduct a comprehensive enrichment analysis of BC cell populations. The analytical findings underwent subsequent validation using an independent single-cell cohort. Tumor cell clusters were categorized based on their average enrichment scores. Functional analyses were carried out using several tools, including CellChat, Monocle, CopyKAT, SCENIC, and CytoTRACE. The RCTD method was then employed to map the single-cell clusters onto spatial transcriptomics data, facilitating the evaluation of cellular dependencies and pathway activities to differentiate tumor cell subtypes. A prognostic framework was subsequently established using large-scale transcriptomic datasets, enabling prediction of immunotherapy responsiveness. Experimental validation further confirmed expression patterns of pivotal genes implicated in therapeutic outcomes.



 Tumor cells exhibit significantly upregulated nucleotide metabolic activity, enabling their classification into two distinct subgroups: NUhighepi and NUlowepi. Cells within the NUhighepi subgroup demonstrate pronounced malignant phenotypes. Intercellular communication analysis performed with the stLearn platform revealed robust interactions between NUhighepi cells and fibroblasts. Supporting this finding, spatial transcriptomic analysis via MISTy revealed a distinct dependency of NUhighepi on fibroblasts. A robust prognostic model, developed using various machine learning algorithms, was able to predict survival outcomes and responses to immunotherapy. Furthermore, targeted drugs were identified for both the high and low scoring groups. Experimental investigations confirmed the expression of core genes in different breast cancer cells.



 In conclusion, we developed a nucleotide metabolism-derived prognostic signature for BC, with DCTPP1 highlighted as a promising biomarker and therapeutic target. These findings provide a valuable framework for early clinical intervention and show promising potential for predicting responses to immunotherapy in BC patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502ca564395f021e293ba0e223c1f50472f85541" target='_blank'>
              Spatial transcriptome and single-cell sequencing reveal the role of nucleotide metabolism in breast cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yuejun Pan, Yong Xu, Ke Gong, Songlin Yuan
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed06b707f53d2e1de85e352e5c2852fcf93afdb7" target='_blank'>
              Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer
              </a>
            </td>
          <td>
            Maria Vidal-Sabanés, R. Navaridas, Núria Bonifaci, Ada Gay-Rua, Damià Ortega-Peinado, Joaquim Egea, Mario Encinas, Xavier Matias-Guiu, David Llobet-Navás, Xavier Dolcet
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 PSMA expression is heterogeneous across advanced prostate cancer, limiting the durability of PSMA-radioligand therapy (RLT). To characterize tumor heterogeneity and identify determinants of RLT resistance, we established the SHAPE cohort (Spatial Heterogeneity Atlas of Prostate Cancer Evolution). SHAPE is a multimodal resource integrating PET imaging, clinical treatment history, histopathology, immunohistochemistry, spatial transcriptomics, and genomic profiling across 46 patient tumor samples collected through rapid autopsy, biopsy, and surgical resection. The cohort spans adenocarcinoma, castration-resistant (CRPC), and neuroendocrine (NEPC) prostate cancer states and includes eight xenograft models.



 Twenty-five samples profiled using the 10x Genomics Visium spatial transcriptomics platform, comprising 195,905 spots, were integrated and analyzed using Seurat (v5) in R. Targeted genomic profiling was performed using the Dana-Farber Cancer Institute OncoPanel platform on 36 samples derived from 7 patients and 2 animal models. Among the cohort, seven cases had prior Lu-PSMA exposure, and ten cases were untreated. Eleven clinical and preclinical samples had PSMA PET/CT, including six with both pre- and post-treatment imaging. Data integration connected structural genomic events with spatial lineage programs and imaging features.



 RLT–treated tumors exhibited a copy-number–dominant profile compared with untreated tumors. Recurrent alterations enriched in treated tumors included PTEN deletion (p<0.01), KLLN deletion (p<0.01), AURKA gain (p<0.01), and ZNF217 gain (p<0.01). Treated tumors also showed higher per-sample copy number loss (47.6 vs 37.5, p<0.05) and amplification burden (7.91 vs 2, p<0.05). In contrast, untreated tumors displayed more single nucleotide variants (14.9 vs 6.3, ns) and indels (4.7 vs 1.9, ns). Spatial transcriptomics provided complementary insights. Treated samples with PSMA SUV below the liver reference demonstrated higher copy-number loss burden (p<0.05) and contained unique transcriptomic clusters enriched for glucose metabolism. These clusters displayed variable expression of classical NEPC markers but consistently upregulated genes involved in glucose uptake and utilization. In preclinical studies, head-to-head FDG and Ga-PSMA imaging of WCM12 xenografts demonstrated that FDG, as a surrogate marker of glucose uptake, was significantly increased following relapse after PSMA-RLT (FC, p<0.05).



 The SHAPE cohort integrates genomic, spatial, and imaging data to characterize tumor heterogeneity in advanced prostate cancer. Genomic profiling identified PTEN/KLLN loss and AURKA/ZNF217 gain as recurrent alterations enriched in Lu-PSMA–exposed tumors, while spatial transcriptomics revealed PSMA-low metastases with distinct. clusters enriched for glucose metabolic programs. Together, SHAPE can inform rational strategies for combination or alternative targeted therapies.



 Martin K. Bakht, Jacob Egelberg, Yasutaka Yamada, Louise Clark, Xiao Wei, Varadha Balaji Venkadakrishnan, Anthony P. Belanger, Alice Bernard-Tessier, Sarah J. Hill, Francesca Khani, David J. Einstein, Mary-Ellen Taplin, Eliezer Van Allen, Heather A. Jacene7, Himisha Beltran. Spatial heterogeneity atlas of prostate cancer evolution (SHAPE): Spatial and genomic landscapes of advanced prostate cancer with and without Lu-PSMA therap [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912633affb19287de4d61286123b6004f19890a4" target='_blank'>
              Abstract PR027: Spatial heterogeneity atlas of prostate cancer evolution (SHAPE): Spatial and genomic landscapes of advanced prostate cancer with and without Lu-PSMA therapy
              </a>
            </td>
          <td>
            Martin K. Bakht, Jacob Egelberg, Yasutaka Yamada, Louise Clark, Xiao X Wei, V. B. Venkadakrishnan, Anthony P. Belanger, Alice Bernard-Tessier, Sarah J. Hill, F. Khani, David J Einstein, M. Taplin, E. V. Van Allen, Heather A. Jacene, H. Beltran
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Mechanosensitive ion channel PIEZO2 translates microenvironmental forces into Ca²⁺-dependent signals that shape tumor behavior. Across cancers, PIEZO2 exhibits context-dependent roles, with evidence for both pro- and anti-tumor functions, depending on tissue type, molecular subtype, disease stage, and stromal context. This review synthesizes emerging insights into the multifaceted roles of PIEZO2 in cancer biology, summarize clinical correlations, highlight areas with concordant directionality (e.g., colorectal cancer, medulloblastoma) versus paradoxical findings (e.g., breast cancer across subtypes, gastric cancer by differentiation status), and provide a pragmatic framework for precision, context aware targeting. While PIEZO2 remains attractive for combination strategies (anti-angiogenesis, barrier modulation, dormancy escape, radio/chemo sensitization) and cancer pain interfaces, the field lacks truly PIEZO2 selective small molecule modulators. We propose feasible future priorities encompassing both pharmacologic and non-pharmacologic strategies to accelerate translational applications. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2298c7009cdc540e541e43d7dcb3fb98a9de240" target='_blank'>
              PIEZO2 in tumors: from mechanobiological switches to activity-targeted therapies
              </a>
            </td>
          <td>
            Dan-Xia Huang, Qiu-Zhi Zhou, Hongmei Luo, M. Nasb, Yi-Zhou Liu, Yu-Jie Yang, Hong Chen
          </td>
          <td>2025-12-01</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Rising incidence of early-onset cancers demands experimental systems that model how normal tissues traverse premalignant states to form clinically relevant, therapy-responsive tumors, and how host immunity and ancestry-linked genetics shape that trajectory. A major obstacle has been the lack of tractable in vivo platforms that enable genome-wide discovery while preserving continuous tumor evolution without chromosomal instability. We developed Stochastically Emergent Tumors (SETs), an organoid-derived in vivo evolution engine that addresses this gap and reframes discovery for early-onset and understudied patient groups. In this system, mismatch repair deficiency (MMRD) is induced in nonmalignant human organoids, which are passaged to accumulate stochastic point mutations and transplanted into mice to permit malignancy to emerge under authentic selective pressures. SETs evolve primarily through high-resolution point mutations rather than broad copy number changes, producing bioinformatically tractable clonal dynamics ideally suited for whole-genome driver discovery and machine learning. Compared with conventional models, SETs exhibit increased intertumoral heterogeneity and reproducible recovery of sensitizing and resistance alleles under selective pressure. As proof of principle in prostate cancer, endocrine therapy applied to prostate SET pools recovered known determinants of sensitivity and revealed novel drivers. Loss of ZFHX3 function, typically obscured within a multi-gene tumor suppressor locus in bulk cohorts, promoted luminal histology and sensitized tumors to androgen pathway inhibition in vivo, while alterations in KMT2D and CIC mediated resistance. Consistent with model predictions, ZFHX3 loss in patients was associated with markedly increased survival, a novel finding comparable in magnitude to the longest-surviving molecular subtypes of advanced prostate cancer. These results identify selective vulnerabilities that arise from stochastic evolution. We also defined an oncogenic threshold by relating pregraft mutation burden to tumor incidence: in a Pten-null background, roughly 900 coding mutations accumulated over 208 days were sufficient for malignant transformation in half of grafts. This quantifies the median number of stochastic events required for premalignant-to-malignant transition and establishes a measurable axis to assess how genetics and microenvironmental pressures shift that threshold. Because SETs generate neoantigen-rich point mutation landscapes, they can be adapted to immunocompetent and premalignant paradigms to interrogate tumor–immune coevolution. Coupled with spatial and functional readouts, SETs enable mapping of early T-cell surveillance, macrophage-mediated immune exclusion, and myeloid checkpoints that permit immune escape. SETs therefore provide a scalable, evolution-aware platform to identify lineage- and ancestry-associated drivers, immunopreventive targets, and biomarkers of early-onset cancer.



 Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, Valbona Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, Cornelia Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, Charles Sawyers, Felix Feng, Hani Goodarzi, Rohit Bose. Reconstructing evolution with Stochastically Emergent Tumors (SETs) reveals in vivo vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr IA006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5e22cb0b560b5c34832027ffccc947dda1ce65" target='_blank'>
              Abstract IA006: Reconstructing evolution with Stochastically Emergent Tumors (SETs) reveals in vivo vulnerabilities
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, V. Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C. Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R Karthaus, Charles L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Metastasis is the primary cause of cancer-related mortality worldwide. This narrative review integrates recent advances in the molecular circuits orchestrating metastatic progression, encompassing epithelial–mesenchymal transition (EMT), organotropism, extracellular matrix remodeling, angiogenesis, hypoxia-inducible signaling, tumor-cell migration modes, and tumor–immune interactions through expert-guided literature selection. We examined therapeutic innovations that disrupt these pathways, including EMT modulators, matrix metalloproteinase inhibitors, VEGF/VEGFR-targeted regimens, hypoxia-activated prodrugs, and next-generation immunotherapies such as immune checkpoint blockade and chimeric antigen receptor T cells. Additionally, we discuss established nanotechnology-based delivery systems, advancing multi-omics integration, evolving single-cell analyses, and emerging CRISPR-Cas9 gene-editing applications as tools for improving metastasis detection, monitoring, and treatment. Despite this progress, translational obstacles persist, particularly regarding intratumoral heterogeneity, adaptive resistance, and limited preclinical model fidelity. Addressing these challenges requires biomarker-guided, multi-target therapeutic combinations, interdisciplinary collaboration, and globally inclusive clinical trials. This evidence underscores the importance of integrated strategies that simultaneously target intrinsic tumor plasticity and microenvironmental support to transform metastatic cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d7d3de1a04e69b49fb547b3a441611b4041aca5" target='_blank'>
              Deciphering molecular pathways driving cancer invasion and metastasis: advances and therapeutic prospects
              </a>
            </td>
          <td>
            Aziza Alshahrani, Arwa Alsubait, Z. Asiri, Sahar Alghamdi, Sarah Bin Saqyah, Tariq Alqahtani, Rawan Fitaihi, Njoud Altuwaijri, Yahya F. Jamous
          </td>
          <td>2025-12-02</td>
          <td>Frontiers in Oncology</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="OBJECTIVE
Glioma recurrence severely impacts patient prognosis, with current treatments showing limited efficacy. Traditional methods struggle to analyze recurrence mechanisms due to challenges in assessing tumor heterogeneity, spatial dynamics, and gene networks. Single-cell combined spatial transcriptomics (ST) offers innovative solutions.


METHODS
We analyzed glioma mRNA data from TCGA and single-cell and ST data from GEO. Following quality control, dimensionality reduction, clustering, and cell annotation of single-cell sequencing data, we identified cell types exhibiting significantly aberrant distributions between primary and recurrent samples by analyzing the deviation degree of Ro/e values. Fibroblasts demonstrating the greatest intergroup differences were subsequently selected as the key cellular population for further investigation. Key differentially expressed genes (DEGs) were identified via random survival forest analysis. Drug sensitivity was assessed using GDSC. Deconvolution algorithms mapped cellular spatial distribution, while PROGENy quantified pathway activity. MISTy modeling revealed cell-cell interactions.


RESULTS
Fibroblasts were the primary recurrence-associated subpopulation, with marker genes enriched in extracellular matrix and adhesion pathways. AEBP1, ZNF708, and TSHZ2 were identified as key genes: AEBP1/TSHZ2 correlated with poor prognosis, while ZNF708 showed an inverse trend. These genes were linked to chemosensitivity (Irinotecan, Carmustine, Vincristine, and Cisplatin). Recurrent tumors exhibited increased plasma cell infiltration, with key genes regulating IL-17, Notch, and Toll-like receptor pathways. Spatial analysis highlighted oligodendrocyte-astrocyte interactions in the tumor microenvironment.


INTERPRETATION
Fibroblasts drive glioma recurrence, with AEBP1, ZNF708, and TSHZ2 predicting recurrence and chemoresistance. These genes promote immune suppression (via plasma cells) and activate recurrence pathways. Oligodendrocyte-astrocyte interactions shape the recurrent microenvironment, suggesting new therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec7c524a9377c361243faed8c72c15c26a0c97d" target='_blank'>
              Unraveling the Molecular Mechanisms of Glioma Recurrence: A Study Integrating Single-Cell and Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Lei Qiu, Yinjiao Fei, Jiaxuan Ding, Kexin Shi, Jinyan Luo, Yuchen Zhu, Xingjian Sun, Gefei Jiang, Yuandong Cao, Weilin Xu, Shu Zhou
          </td>
          <td>2026-01-06</td>
          <td>Annals of clinical and translational neurology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Rising incidence of early-onset prostate and other solid tumors underscores the need for experimental systems that model how normal tissues traverse premalignant states, acquire mutations, and become therapy-responsive malignancies under authentic immune and stromal pressures. A major barrier has been the lack of tractable in vivo platforms that enable genome-wide discovery while preserving continuous tumor evolution without catastrophic chromosomal instability. To address this, the Bose Lab developed Stochastically Emergent Tumors (SETs), an organoid-derived in vivo evolution engine that redefines discovery for early-onset and understudied patient groups. In this system, mismatch-repair deficiency is induced in non-malignant human organoids, which are passaged to accumulate stochastic point mutations and transplanted into mice to permit malignancy to emerge under physiologic selection. SETs evolve primarily through high-resolution point mutations rather than broad copy-number changes, yielding bioinformatically tractable clonal dynamics ideally suited for whole-genome driver discovery and machine learning. Compared with conventional xenografts, SETs display greater intertumoral heterogeneity and reproducible recovery of sensitizing and resistance alleles under therapeutic pressure. As proof of principle in prostate cancer, endocrine therapy applied to SET pools recovered known determinants of androgen-pathway sensitivity and uncovered new drivers. Loss of ZFHX3, typically obscured within a multigene suppressor locus in bulk cohorts, promoted luminal histology and sensitized tumors to androgen-receptor inhibition in vivo, whereas KMT2D or CIC alterations mediated resistance. Consistent with model predictions, ZFHX3 loss in patients correlated with significantly improved survival, a finding comparable in magnitude to the most favorable molecular subtypes of advanced prostate cancer. SETs also quantify evolutionary thresholds: in a Pten-null background, approximately 900 coding mutations accumulated over 208 days were sufficient for malignant transformation in half of grafts. This defines a measurable axis linking mutation burden, genotype, and tumor incidence. Because SETs generate neoantigen-rich point-mutation landscapes, they can be extended to immunocompetent hosts to study tumor–immune coevolution, early T-cell surveillance, macrophage-mediated immune exclusion, and myeloid checkpoints that enable immune escape. In summary, SETs provide a scalable, evolution-aware platform that connects mutational dynamics to therapeutic vulnerability, enabling identification of lineage- and ancestry-associated drivers, immunopreventive targets, and biomarkers of early-onset prostate cancer.



 Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, Valbona Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C.K. Cornelia Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, Charles Sawyers, Felix Feng, Hani Goodarzi, Rohit Bose. Reconstructing prostate evolution with Stochastically Emergent Tumors (SETs) reveals in vivo therapeutic vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23678ab77681ac1f2c46c1e6d9b9824f918c99b" target='_blank'>
              Abstract A008: Reconstructing prostate evolution with Stochastically Emergent Tumors (SETs) reveals in vivo therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, V. Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C. Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R Karthaus, Charles Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma is a lethal brain cancer marked by functional plasticity driven by tumor cell-intrinsic mutations and their interplay with developmental programs. To investigate how canonical glioblastoma mutations promote functional plasticity, we have developed an isogenic human neural stem cell (NSC) model of glioblastoma by sequential addition of TERT promoter, TP53, and PDGFRA point mutations. TP53 loss-of-function increases TERT expression during serial mutagenesis, but only triple mutant NSCs reliably form lethal brain tumors in vivo that recapitulate glioblastoma. Tumor cell evolution triggers stress-related metabolic changes and transitions toward a neuronal progenitor network driven by transcription factor INSM1. INSM1 is highly expressed in human glioblastoma tumors and, during cortical development, in intermediate progenitor cells, which give rise to neurons. Remarkably, INSM1 knockdown in triple mutant NSCs and primary glioblastoma cells disrupts oncogenic gene expression and function and inhibits the in vivo tumorigenicity of triple mutant NSCs, highlighting the functional importance of an intermediate progenitor cell-like cell state in glioblastoma pathogenesis. Glioblastoma (GBM) is characterized by a high degree of heterogeneity and plasticity due to interplay with neural developmental programs. Here, the authors develop a model of GBM by introducing sequential oncogenic mutations in human neural stem cells and using this, identify INSM1 as a driver of a neural progenitor gene network promoting tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ee12cd01ff0dc670471af7bea5ea93e2f7413e0" target='_blank'>
              INSM1 governs a neuronal progenitor state that drives glioblastoma in a human stem cell model
              </a>
            </td>
          <td>
            Patrick A. DeSouza, Matthew Ishahak, Xuan Qu, Colin R. McCornack, Devi Annamalai, Diane D. Mao, Rajanikanth Vangipurapu, Yiwei Fu, Alexandre T. Vessoni, Ryan Cleary, Rowland H Han, Punn Augsornworawat, Timothy Woodiwiss, Darby Agovino, Braxton Sizemore, Jessica Kline, Maryam Borhani-Haghighi, Hao Chen, Sangami Pugazenthi, Hiroko Yano, Ting Wang, Luis F Z Batista, J. Millman, Albert H. Kim
          </td>
          <td>2025-12-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="BACKGROUND
Meningioma brain invasion encumbers surgical resection and increases the risk of tumor recurrence, but the molecular mechanisms underlying this process are poorly understood.


METHODS
To identify molecular and cellular features of brain-invasive meningiomas, we (1) analyzed bulk RNA sequencing data from 199 meningiomas, including 33 brain-invasive tumors, (2) analyzed patient-matched single-cell RNA sequencing data of spatially mapped meningioma samples from the tumor core or brain-tumor interface (BTI), and (3) performed spatial transcriptomic sequencing of brain-invasive meningioma samples. Multiplexed immunofluorescence (IF) was used to validate bioinformatic spatial expression patterns. Functional interactions between meningioma cells and neurons were studied in meningioma/neuron co-cultures using confocal microscopy, multi-electrode array recordings, and live cell calcium imaging.


RESULTS
Transcriptomic analyses showed conserved enrichment of TGM2, S100A11, ZYX, and PDGFRA at the BTI across bulk, single-cell, and spatial RNA sequencing datasets. The expression of these genes at the BTI was confirmed using multiplexed IF, and single-cell bioinformatic and microscopy analyses further demonstrated enrichment of macrophages at the BTI. Co-culture assays showed neuronal hyperexcitability and increased proliferation of meningioma cells, suggesting functional communication between meningioma cells and the tumor microenvironment may contribute to meningioma growth in cases with brain invasion.


CONCLUSIONS
Meningioma brain invasion is defined by molecular remodeling of tumor cells and functional interactions within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4352a0cf42108bac27d918f350130e856ef324b" target='_blank'>
              Meningioma cell reprogramming and microenvironment interactions underlie brain invasion.
              </a>
            </td>
          <td>
            Ayush Aggarwal, Mark W. Youngblood, T. Picart, H. Najem, S. Oten, M. A. Cady, Stephen T. Magill, Craig M Horbinkski, James P. Chandler, A. Heimberger, K. Mirchia, S. Hervey-Jumper, David R. Raleigh
          </td>
          <td>2025-12-31</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4167a37773942f9ca4b809bad0d95f60d49ec113" target='_blank'>
              Impacts of mutation accumulation and order on tumor initiation revealed by engineered murine colorectal cancer organoids
              </a>
            </td>
          <td>
            Yanping Li, Xiaoxin Xie, Daqi Deng, Zhiyuan Sun, Zhaoan Huang, Yisen Tang, Liang Fang, Wei Chen, Qionghua Zhu
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer progression arises from interactions across multiple biological layers, especially beyond morphological and across molecular layers that remain invisible to image-only models. To capture this broader biological landscape, we present EXAONE Path 2.5, a pathology foundation model that jointly models histologic, genomic, epigenetic and transcriptomic modalities, producing an integrated patient representation that reflects tumor biology more comprehensively. Our approach incorporates three key components: (1) multimodal SigLIP loss enabling all-pairwise contrastive learning across heterogeneous modalities, (2) a fragment-aware rotary positional encoding (F-RoPE) module that preserves spatial structure and tissue-fragment topology in WSI, and (3) domain-specialized internal foundation models for both WSI and RNA-seq to provide biologically grounded embeddings for robust multimodal alignment. We evaluate EXAONE Path 2.5 against six leading pathology foundation models across two complementary benchmarks: an internal real-world clinical dataset and the Patho-Bench benchmark covering 80 tasks. Our framework demonstrates high data and parameter efficiency, achieving on-par performance with state-of-the-art foundation models on Patho-Bench while exhibiting the highest adaptability in the internal clinical setting. These results highlight the value of biologically informed multimodal design and underscore the potential of integrated genotype-to-phenotype modeling for next-generation precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7747448149656db4d84a5fb568dfd6adf6c8b244" target='_blank'>
              EXAONE Path 2.5: Pathology Foundation Model with Multi-Omics Alignment
              </a>
            </td>
          <td>
            Juseung Yun, Sunwoo Yu, Sumin Ha, Jonghyun Kim, Janghyeon Lee, Jongseong Jang, Soonyoung Lee
          </td>
          <td>2025-12-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background

Immune checkpoint inhibitors have transformed the treatment landscape of non-small cell lung cancer (NSCLC); however, primary and acquired resistance remain major clinical challenges. Immune evasion in NSCLC is increasingly recognized as a complex, dynamic process driven by tumor heterogeneity and adaptive remodeling of the tumor microenvironment. Conventional bulk profiling approaches are limited in resolving this complexity, underscoring the need for higher-resolution methodologies.

Objective
This review aims to synthesize current evidence on how single-cell transcriptomic approaches have advanced the understanding of immune evasion mechanisms in treatment-resistant NSCLC, with a focus on tumor-intrinsic programs, immune cell dysfunction, and intercellular communication networks.

Methods

A structured narrative review with systematic elements was conducted, analyzing peer-reviewed studies employing single-cell RNA sequencing and related single-cell technologies in human NSCLC. Studies were evaluated for their insights into immune escape pathways, resistance to immunotherapy, and translational relevance.

Results

Single-cell transcriptomic analyses reveal profound intratumoral and immune heterogeneity in treatment-resistant NSCLC. Resistant tumors harbor distinct malignant cell states characterized by impaired antigen presentation, dysregulated interferon signaling, and lineage plasticity. Within the immune compartment, hierarchical T-cell exhaustion, depletion of progenitor-like exhausted T cells, and expansion of immunosuppressive myeloid populations—particularly tumor-associated macrophages—emerge as dominant features of resistance. Ligand–receptor interaction analyses further demonstrate coordinated immunosuppressive communication networks that sustain immune escape at a systems level. Several single-cell–defined cellular states correlate with poor response to immune checkpoint blockade and adverse clinical outcomes.

Conclusion

Single-cell transcriptomics has fundamentally reshaped the conceptual framework of immune resistance in NSCLC, redefining it as a multicellular, dynamic ecosystem rather than a single-pathway phenomenon. These insights provide a strong biological rationale for developing biomarker strategies and rational combination therapies that target both tumor-intrinsic adaptations and the immunosuppressive microenvironment. Continued integration of single-cell technologies into translational and clinical research is essential to overcome immunotherapy resistance and improve patient outcomes in NSCLC.

Keywords: Non-small cell lung cancer; Single-cell RNA sequencing; Immune evasion; Immunotherapy resistance; Tumor microenvironment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7495ff2bd82ff3331af5929ec1f5ca0f1ed50c9" target='_blank'>
              Single-Cell Transcriptomic Mapping of Immune Evasion Pathways in Treatment-Resistant Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Marco Rossi, Giulia Bianchi, Alessandro Conti
          </td>
          <td>2026-01-19</td>
          <td>Advanced Journal of Biomedicine &amp; Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven by heterogeneous tumor cell populations and microenvironmental interactions. However, a comprehensive single-cell atlas across patients that captures this heterogeneity, and its clinical implications has been lacking. Such an atlas could reveal rare tumor subpopulations that underpin disease aggressiveness and offer new prognostic biomarkers or therapeutic targets. We integrated two high-quality single-cell RNA sequencing datasets from 29 CRC patients to construct a cellular atlas encompassing immune, stromal and epithelial compartments. Malignant epithelial cells were distinguished via inferCNV-based copy number alteration analysis and reclustered, yielding seven transcriptionally distinct malignant subpopulations. One malignant epithelial cluster, marked by high expression of the long non-coding RNA ELFN1-AS1, exhibited the highest stemness signature. From this stem-like cluster, we derived a four-gene prognostic model (RPL21, GAL, ELFN1-AS1, AIF1L). In the TCGA-COAD cohort, this model stratified patients into high-risk and low-risk groups with significantly different survival outcomes, with the high-risk group experiencing significantly worse survival. High-risk tumors were enriched for metabolic and translational pathways and displayed distinct immune and genomic features. AIF1L was identified as a hub gene within the signature, and its knockdown in CRC cells enhanced migration and invasion, functionally validating its role in tumor progression. Our study provides a high-resolution single-cell atlas of CRC and identifies a previously unrecognized stem-like tumor cell subpopulation with prognostic significance. These findings highlight novel prognostic biomarkers and suggest potential therapeutic targets (such as AIF1L) that could inform patient stratification and the development of targeted therapies for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3d35184693bc48dd0fe85be6bfc55a03d6e4e7" target='_blank'>
              Single-cell RNA sequencing defines prognostic subtypes and identifies AIF1L as a therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Gao Li, Dingxue Wang, Guo Chen
          </td>
          <td>2025-12-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common type of primary liver cancer. Tumor heterogeneity is a major obstacle to effective treatment and is poorly understood using traditional bulk sequencing methods. This review highlights the transformative role of single-cell and multi-omics technologies in determining the cellular and molecular complexities of HCC. We summarize recent advances in single-cell transcriptomics, epigenomics, multi-omics, and spatial transcriptomics platforms, emphasizing their applications in characterizing tumor subclones, cancer-associated fibroblast-immune interactions, circulating tumor cells, and immune-resistant phenotypes. Spatial approaches have revealed the architecture of cancer stem cell niches and tertiary lymphoid structures, providing unprecedented insights into tumor organization and microenvironmental crosstalk. Although still in their early stages, clinical trials have begun to incorporate these technologies, underscoring their translational potential. Single-cell and spatial omics have reshaped HCC research by enabling high-resolution profiling of tumor ecosystems and driving the discovery of biomarkers, therapeutic targets, and strategies for patient stratification. However, high cost, technical expertise, and limited accessibility, particularly in resource-constrained settings, are major barriers to its widespread adoption. Addressing these challenges is critical for translating these powerful approaches into clinical practice and for advancing precision medicine for the treatment of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0abad77c5fcae33c11d7d851ad3eb337d3881a5" target='_blank'>
              Exploring Single-Cell and Multi-Omics Technologies and Their Role in Unravelling Tumor Heterogeneity of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Charmi Jyotishi, Suresh Prajapati, Mansi Patel, Reeshu Gupta
          </td>
          <td>2025-12-17</td>
          <td>Journal of liver cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Sinonasal squamous cell carcinoma (SNSCC) is a rare malignancy with poorly understood molecular drivers. Consequently, its cellular composition and tumor microenvironment (TME) remain largely undefined. Here, we performed integrated bulk and single-nucleus multi-omic analyses to map the SNSCC ecosystem. Within the malignant compartment, we identified five distinct populations, with hypoxic (TC1) and proliferative (TC2) subtypes associated with adverse clinical outcomes. Functionally, TC1 cells orchestrate a hypoxia-driven angiogenic program via coordinated secretion of adrenomedullin (ADM), MIF, and VEGFA, promoting endothelial tip cell (EC1) differentiation. Integrative analysis revealed these transcriptional programs are underpinned by tumor-specific chromatin accessibility and DNA hypomethylation, particularly at AP-1-enriched regulatory elements. Mechanistically, in vitro studies confirmed that this response depends on cooperative AP-1 and HIF1A signaling. Furthermore, histological analysis of patient tissues demonstrated spatial co-localization of GLUT1-expressing TC1 cells with DLL4-positive EC1 cells. These findings elucidate the epigenetic landscape underlying tumor-stromal interactions and establish the ADM/VEGFA axis as a critical therapeutic target to disrupt epigenetically controlled angiogenesis in SNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bae1ad73527bcabf41b52f28eb9547020179867e" target='_blank'>
              Single-Nucleus Multi-Omics Reveals Hypoxia-Driven Angiogenic Programs and Their Epigenetic Control in Sinonasal Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, J. Noh, Jaehyun Lee, Geunho Kwon, Myeong S. Yu, Yoo-Sam Chung, Seung-Jun Lee, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2026-01-07</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most prevalent malignant neoplasms worldwide, characterized by a high incidence of recurrence and metastasis, which substantially diminishes patient survival rates. This underscores the urgent need to identify novel biomarkers and therapeutic targets. Transcribed ultraconserved regions (T-UCRs), a category of non-coding RNAs with significant evolutionary conservation, are crucial to various biological processes. Recent studies have shown that T-UCRs play a pivotal role in tumorigenesis and tumor progression. A growing body of evidence indicates that T-UCRs significantly influence CRC development by modulating critical mechanisms, including cell proliferation, apoptosis, invasion, and metastasis. This review systematically explores the functions of T-UCRs in tumorigenesis, focusing on their regulatory roles, underlying molecular mechanisms, and potential clinical applications in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ca72dace2c26f564f76a90eafc66dbf14b419b5" target='_blank'>
              The transcribed ultraconserved regions: emerging players in colorectal cancer biology and therapy
              </a>
            </td>
          <td>
            Yi Zhang, Zichen Wei, Xin Wang, Pan Wang, Lei Pang, Hongliang Dong, Han Xu
          </td>
          <td>2026-01-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Highlights What are the main findings? Patient-derived organoids (PDOs) faithfully recapitulate the genetic heterogeneity and histological architecture of original tumors, serving as a superior platform for modeling intrinsic and acquired resistance mechanisms, particularly the reversible state of drug-tolerant persisters (DTPs). Despite their utility, current PDO models face significant challenges, including the absence of tumor microenvironment (TME) components, the lack of standardized drug sensitivity cutoff values, and the scarcity of acquired resistance models in lung cancer due to re-biopsy limitations. What is the implication of the main finding? Integrating PDOs with advanced technologies—such as single-cell RNA sequencing, spatial transcriptomics, and immune co-culture systems—enables the precise mapping of resistance evolutionary trajectories and the evaluation of novel therapeutics like antibody-drug conjugates (ADCs). Establishing standardized culture protocols and large-scale biobanks will transition PDOs from experimental tools to robust “pharmacotyping” platforms, facilitating functional precision medicine and improved treatment strategies for relapsed lung cancer patients. Abstract Background: Chemotherapy, targeted therapy and radiotherapy are the cornerstones of cancer treatment. However, therapeutic resistance—not only to these classic modalities but also to novel therapeutics like immune checkpoint inhibitors (ICIs) and antibody-drug conjugates—remains a major hurdle. Resistance significantly limits efficacy and increases recurrence rates. A deep understanding of the molecular mechanisms driving this resistance is critical for developing personalized therapeutic strategies and improving patient outcomes. Recent Advances: Patient-derived cancer organoids have emerged as a powerful preclinical platform that faithfully recapitulates the genetic, phenotypic, and histological characteristics of original tumors. Consequently, PDOs are being widely utilized to evaluate drug responses, investigate resistance mechanisms, and discover novel therapeutic targets for a range of therapies. Limitations: While organoid models have been instrumental in studying resistance, significant limitations persist. First, standard organoid-only models lack key tumor microenvironment components, such as immune cells, limiting immunotherapy research. Second, there is a significant lack of research on acquired resistance, particularly in lung cancer. This gap is largely driven by the clinical infeasibility of rebiopsy in patients with progressive diseases. Third, the absence of standardized protocols for generating and validating resistance models hinders reproducibility and complicates clinical translation. Conclusions: This review summarizes recent advances in using organoid models to study resistance to chemotherapy, radiotherapy, and novel therapeutics (ICIs and ADCs). We emphasize the critical need for standardization in resistance organoid research. We also propose future directions to overcome existing challenges, including the integration of co-culture systems (to include the TME) and advanced technologies (e.g., scRNA-seq, Spatial Transcriptomics). Our specific focus is on advancing lung cancer resistance modeling to enable functional precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02de70fc862ddc88f8c5994f14a3861da00f3847" target='_blank'>
              Modeling Drug and Radiation Resistance with Patient-Derived Organoids: Recent Progress, Unmet Needs, and Future Directions for Lung Cancer
              </a>
            </td>
          <td>
            Dahye Lee, Yoonjoo Kim, D. Kang, Chaeuk Chung
          </td>
          <td>2025-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eef0494e51b8128b410f9d5b4dc00c303aff4796" target='_blank'>
              Mapping lineage and functional diversity in the high-risk human mammary epithelium
              </a>
            </td>
          <td>
            Matthew Waas, Bowen Zhang, Meinusha Govindarajan, Pirashaanthy Tharmapalan, Abhijith Kuttanamkuzhi, Olivia Drummond Guy, Michael Woolman, H. Berman, Paul D. Waterhouse, Rama Khokha, T. Kislinger
          </td>
          <td>2026-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="TROP2 has emerged as a significant biomarker and therapeutic target in various epithelial cancers, owing to its tissue-specific expression and its association with tumor proliferation, invasion, and metastasis. Tumor cell migration is a critical determinant of tumor invasiveness and malignancy, serving as a pivotal factor in prognosis assessment and therapeutic response evaluation. Despite these insights, the molecular mechanisms underpinning TROP2's role in tumor progression─particularly in mediating tumor cell migration─remain incompletely understood. Desmosomes, as essential cell-cell adhesion structures, are integral to maintaining tissue architecture; however, how subtle alterations in their expression and structural organization influence tumor cell adhesion warrants further investigation. In this study, we employed direct stochastic optical reconstruction microscopy (dSTORM) coupled with biochemical approaches to elucidate the mechanism by which TROP2 regulates tumor cell migration via desmosomal cadherin DSG2. Co-localization imaging and coimmunoprecipitation assays confirmed an interaction between TROP2 and DSG2. By establishing TROP2 overexpression and knockdown cell lines, we observed that high TROP2 expression not only downregulated DSG2 levels but also impaired desmosome assembly. These findings were further confirmed at the tissue level. Moreover, we found that TROP2 facilitated tumor cell proliferation and migration by suppressing DSG2 expression and activating EGFR/AKT and FAK downstream signaling pathways. Our findings reveal the molecular mechanism by which TROP2 promotes migration, highlighting the critical role of intercellular junction integrity in cancer progression. These insights provide a foundation for developing targeted therapies against TROP2 and its associated signaling mechanisms in epithelial malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09e42296d19fac0502697d2ca49c38690dc7893f" target='_blank'>
              TROP2 Promotes Tumor Cell Migration through Downregulation of DSG2 Revealed by Super-Resolution Fluorescence Imaging.
              </a>
            </td>
          <td>
            Longxiang Li, Meng Li, Baofeng Li, Jincheng Wang, Haijiao Xu, Jinrui Zhang, Yan Shi, Yong Liu, Jilin Tang, Jing Gao, Hongda Wang
          </td>
          <td>2026-01-13</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The development of novel therapeutics for deadly diseases such as glioblastoma (GBM) is bottlenecked by poor preclinical models. GBM is the most common and deadliest primary brain tumor in adults, with an average prognosis of 12–15 months, primarily due to its high cellular heterogeneity and treatment resistance from GBM stem cells. The advancement of in vitro models into organoids, three-dimensional tissue-like modeling systems, has been a promising approach to improving translational medicine for GBM. However, the critical tradeoff between technical convenience and physiological relevance threatens the integrity and reproducibility of GBM organoid (GBO) biomanufacturing. This comprehensive review breaks down and discusses the key features of GBM tumor microenvironment (TME), traces the advancement of in vitro models from two-dimensional cultures to three-dimensional stem cell-derived GBOs, evaluates the process through an engineering perspective (genetic, biochemical, biophysical, and process engineering), and addresses critical translational gaps. Reviewing trends over the last fifteen years in biomanufacturing approaches to GBOs revealed fundamental oversights that address previous review focuses on the limitations of organoids (i.e., maturity, vasculature, and immune defense). To summarize, GBO’s translational gap and reproducibility challenges are rooted in the prioritization of technical convenience over physiological relevance. To achieve clinical relevance, future GBO development must focus on transitioning to fully defined components (excluding animal-derived ECM), developing sufficiently large-sized constructs to recapitulate the full TME, and integrating non-destructive and enhanced functional readouts of the GBOs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e994f4d6a33fd45263cff1ae3c39866677c30f73" target='_blank'>
              Bioengineering Stem Cell-Derived Glioblastoma Organoids: A Comprehensive Review
              </a>
            </td>
          <td>
            Alexandra D. Avera, Yonghyun Kim
          </td>
          <td>2025-12-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ce067367ca2b5f10cdfc82c800bdb989ef86d53" target='_blank'>
              Deep learning inference of universal dormancy pseudotime reveals the cellular targets of anti-cancer therapies
              </a>
            </td>
          <td>
            Haley M. Beckmann, Michelle Tong, Glenn Chang, Adi Steif
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a major issue in the sphere of head and neck cancer since it is very heterogeneous, which also leads to the poor treatment results and low survival rates in the advanced stages. Here, the review will discuss how spatial omics methods explore tumor heterogeneity in OSCC, which includes cellular, molecular, and immune microenvironment alterations that occur due to cancer stem cells, stromal interactions, genetic instability, epigenetic reorganization, and metabolic reprogramming. The focus of heterogeneity is given on the contribution of the tumor microenvironment such as immune cells, cancer-associated fibroblasts and extracellular matrix remodeling to the stimulation of progression, metastasis, as well as therapeutic resistance. The introduction of spatial omics technologies, including spatial transcriptomics, proteomics, and metabolomics, has revolutionized the field by preserving tissue architecture, enabling high-resolution mapping of gene expression, protein distribution, and metabolite profiles. Significant developments around the spatial omics technologies are discussed, while how they are used in the identification of ligand-receptor network, signaling pathways, and spatial patterns of heterogeneity in OSCC are described. Integration of multi-omics approaches bridges gaps between transcriptomic, proteomic and metabolic, facilitating the discovery of biomarkers for prognosis, immune evasion mechanisms, and precision therapies targeting epithelial-to-mesenchymal transition and immunosuppressive networks. Despite challenges in data integration, cost, and clinical translation, spatial omics holds promise for personalized oncology, with future directions involving artificial intelligence-driven modeling to enhance diagnostic accuracy and therapeutic efficacy in OSCC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00bf591ef53e5e31062caa9f75b7e97f0abcc8b" target='_blank'>
              Spatial Omics in Decoding Oral Squamous Cell Carcinoma Heterogeneity: Microenvironment Crosstalk and Multi-Omics Integration
              </a>
            </td>
          <td>
            Xiaotong You, Yu Liu
          </td>
          <td>2025-12-31</td>
          <td>GenoMed Connect</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a resilient subpopulation within tumors, capable of driving progression, metastasis, and recurrence. One mechanism that enables this plasticity is asymmetric cell division (ACD), a process by which CSCs generate both a self-renewing stem cell and a differentiated daughter cell. While traditionally associated with tissue development, ACD is now recognized as a dynamic and regulated feature of cancer biology, particularly in response to stress conditions such as hypoxia and radiation. In this review, we provide a comprehensive and mechanistic synthesis of how intrinsic factors such as polarity complexes, cell fate determinants (CFDs), spindle orientation cooperate with extrinsic cues from the tumor microenvironment to orchestrate ACD in CSCs. We explore how this process contributes to tumor heterogeneity, therapy resistance, and the emergence of quiescent, drug-tolerant CSCs across multiple malignancies, including brain, breast, colorectal, and hematologic cancers. Importantly, we highlight recent efforts to pharmacologically disrupt or redirect ACD using inhibitors of NOTCH, WNT, AURORA kinases, and MSI1, presenting ACD as a therapeutic vulnerability rather than a static trait. By shifting the focus from CSC markers to division mode, this review introduces a novel conceptual framework for targeting tumor hierarchy and plasticity. Understanding and manipulating ACD offers a promising frontier in precision oncology - one where altering the balance of cell fate decisions could limit relapse, reduce intratumoral complexity, and enhance long-term treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b911960284166687ce9462f6da53a77bba5e7d79" target='_blank'>
              Targeting asymmetric division in cancer stem cells: implications for relapse and resistance
              </a>
            </td>
          <td>
            Mona Latifi, Mahsa Sani, F. Sani
          </td>
          <td>2025-12-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Dogs play a major role in sustaining transmission of Leishmania infantum to people, thus prevention and treatment of canine leishmaniosis (CanL) to reduce transmission represents an unmet public health need. Although advances have been made in understanding how immunopathology correlates to infectiousness as CanL progresses, the immune mechanisms underlying transition between disease stages and terminal decline remain ill-defined. To address this knowledge gap, we generated a comprehensive atlas of peripheral immune cells from control dogs and naturally-infected dogs using single-cell RNA sequencing. The LeishDog Atlas captures the cellular and transcriptional complexity underlying CanL, tracing shifts in immune composition and gene expression across progressive, well-defined disease stages. Notably, we identified distinct myeloid, CD4+ and CD8+ T cell phenotypes and transcriptional states associated with disease progression. This resource provides a valuable framework for understanding systemic immune dysregulation in CanL, including determinants of T cell exhaustion and myeloid cell activation, and establishes a foundation for comparative, translational, and mechanistic studies of CanL immunopathology and transmissibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8412687f28e64e4d6b2558529c494e2ee4440eca" target='_blank'>
              A single-cell transcriptomic atlas of peripheral blood immune cells spanning progressive canine leishmaniosis
              </a>
            </td>
          <td>
            Danielle P. Uhl, Daniel J. Holbrook, Max C Waugh, S. Dey, Najmeeyah Brown, Karen I. Cyndari, Jacob J. Oleson, Paul M. Kaye, Christine A. Petersen
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Bone sarcomas remain lethal despite multimodal therapy, primarily because the mineralized, immunosuppressive tumor microenvironment (TME) promotes chemo‑ and immune‑resistance. Integrating single‑cell and spatial omics across osteosarcoma, Ewing sarcoma and chondrosarcoma delineates subtype‑specific TME archetypes dominated by M2 macrophages, exhausted T cells and a stiff extracellular matrix. Mechanistic dissection reveals tractable vulnerabilities, myeloid reprogramming, extracellular matrix modulation and metabolic and epigenetic checkpoints, that can be targeted with bone‑selective delivery systems and biomarker‑driven combination trials to convert therapeutic failure into durable remission. Therefore, the aim of the present review is to synthesize the latest single‑cell, spatial and functional data to map bone‑sarcoma TME heterogeneity, dissect resistance mechanisms and propose integrated, biomarker‑guided therapeutic strategies that can be translated into treatments..">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a42d68e5855e4884a22e5af558a0428be6a50022" target='_blank'>
              Tumor microenvironment in bone sarcomas: Implications for immunotherapy and emerging therapeutic vulnerabilities (Review).
              </a>
            </td>
          <td>
            Wentao Li, Lijun Lv, Yibin Jin, Xin Yuan
          </td>
          <td>2026-01-16</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Radiotherapy resistance in breast cancer remains a major clinical challenge. The key molecular determinants and cellular populations driving this resistance are not fully understood. Methods: A radiotherapy resistance (RR) gene panel was identified from TCGA-BRCA and GSE120798 cohorts. Single-cell and spatial transcriptomics characterized RRhigh epithelial cells (RRhighepi). A prognostic model, named SuperPC and StepCox-based Radiotherapy Resistance model (SSRR), was built via machine learning and Mendelian randomization. Functional roles of Prolyl 4-Hydroxylase Subunit Alpha 2 (P4HA2) were validated in vitro. Results: The RR gene panel was upregulated in tumors and enriched for cell cycle pathways. RRhighepi cells exhibited elevated stemness, activated cell cycle and metabolic programs, and enhanced DNA damage repair. RRhighepi represented a developmental origin and communicated with endothelial cells. The SSRR model stratified patients into high-risk groups with poorer survival and distinct therapeutic responses. P4HA2, a key model gene, was upregulated in multiple cancers. P4HA2 knockdown suppressed proliferation, invasion, and colony formation, and synergized with radiotherapy to reduce stemness and enhance DNA damage. WGCNA confirmed co-module membership of P4HA2 and the RR panel. Conclusions: This study, through multi-omics analysis, proposes a potential mechanistic model associated with radiotherapy resistance in breast cancer. P4HA2 is a potential therapeutic target that sensitizes breast cancer to radiotherapy. The RR gene panel and SSRR model provide insights into resistance mechanisms and prognostic stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166d8bb1b89b913023588472f1dc07c1ab7e8bdf" target='_blank'>
              Single-cell and spatial transcriptomics reveal P4HA2-mediated radiotherapy resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Huimin Li, Junzhi Liu, Yuheng Jiao, Fengyu Xu, Shurui Wang, Qiang Tang
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="While cancer immunotherapies have primarily focused on activation of cytotoxic CD8 cells, CD4 T cell activity is also associated with survival and immunotherapeutic response in numerous cancers. We applied integrated single-cell RNA sequencing and multiplexed protein epitope profiling to breast cancer samples to resolve the complexity of immune cell states within the tumor microenvironment. This approach enhanced phenotypic resolution, identifying three distinct states within the CD4 T follicular helper-like (Tfh) cell cluster. A CXCR4high progenitor state gave rise to two differentiated states: an IGFL2high subset resembling conventional Tfh cells and localised to B cell-rich lymphoid aggregates, and a CD103+ subset, exhibiting features of tissue residency, exhaustion, and cytotoxicity, which co-localised with tumor foci. CD103+ Tfh-like cells were found to interact with CXCL10+ macrophages through production of CCL chemokines and CSF1. A higher CD103+ Tfh to IGFL2high Tfh ratio, together with the selective clonal expansion of the CD103+ subset, was strongly associated with improved tumour immunity and superior responses to anti-PD-1 checkpoint blockade, surpassing the predictive value of exhausted CD8 T cells. These findings integrate Tfh and CD4 with cytotoxic potential in breast cancer, offering new insight into anti-tumor immunity and response to checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/570ecff53156bbe18de4ac45af83e6cec5c103da" target='_blank'>
              Proteotranscriptomic Dissection of Breast Cancer T Cell States Identifies CD103+ Tfh-derived Cytotoxic Cells Linked to Immunotherapy Response
              </a>
            </td>
          <td>
            Alexander Swarbrick, G. Al-Eryani, Sophie van der Leij, Etienne Masle-Farquhar, Chia-Ling Chan, K. Harvey, Sunny Z. Wu, D. Roden, Taopeng Wang, John Reeves, Bertrand Yeung, Christopher Goodnow, Cindy Ma, Charles Perou, N. Hacohen, Aziz Al’Khafaji, Mats Nilsson, J. Lundeberg, Marcel Batten, S. Junankar
          </td>
          <td>2026-01-07</td>
          <td>Research Square</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Cancer is more than just a collection of tumor cells. The complex tumor system, including the tumor immune microenvironment (TIME), is continually changing. Tumor cells are in constant communication with all stromal elements (e.g., fibroblasts, endothelial cells, and extracellular matrix) and immune effector cells (e.g., T cells, B cells, natural killer cells, dendritic cells, macrophages, and myeloid‐derived suppressor cells). Together, these intricate interactions among cell and molecular signaling pathways collectively drive tumor growth, tumor invasion, and metastasis and significantly affect the efficacy of cancer treatments. Recent investigations, from a tumor‐centric research paradigm to a complete evaluation of the local tumor microenvironment, have revealed the importance of the TIME. Although reviews in these fields typically focus on cellular/molecular breakdowns of the TIME and evasion of the immune system, a systematic study of its dynamic evolution is lacking. This review comprehensively discusses the major regulators and networks involved in the dynamic evolution of the TIME, the spatiotemporal dynamics of TIME components, metabolic reprogramming as an engine of TIME evolution, the targeting of metabolic regulators, and niches for TIME modulation, clinical and translational challenges, and future prospects. This information could help researchers explore the TIME and generate new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c31a148a8aa12a68db53f4f9448b71eb8d371267" target='_blank'>
              Dynamic Evolution of the Tumor Immune Microenvironment in Malignant Tumors and Emerging Therapeutic Paradigms
              </a>
            </td>
          <td>
            Ying Sun, C. Shao, Hongtao Duan, Zhaoyang Wang, Shaopeng Xu, Chao Wang, Jiawei Xiu, Jin Liu, Xuejiao Wang, Xin Yao, Yuan Gao, Xiaolong Yan
          </td>
          <td>2025-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/126eb3bb80a65b589c6798760093749f010ab57b" target='_blank'>
              Robust characterization and interpretation of rare pathogenic cell populations from spatial omics using GARDEN.
              </a>
            </td>
          <td>
            Xinming Zhang, Zhuohang Yu, Gaoyang Hao, Qi Yao, Yanmei Hu, Fuzhou Wang, Xingjian Chen, Linjing Liu, Ka-chun Wong, Xiangtao Li
          </td>
          <td>2026-01-17</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 Pancreatic neuroendocrine tumors (panNETs) undergo high-grade (HG) transformation often, with the liver being a common site for metastasis​. Surgical resection being considered as long-term outcomes for selected patients. However, recurrence and grade transformation remain potential significant challenges, with recurrence rates approaching 50–95%.1,2 Despite extensive whole exome analysis and construction of phylogenetic trees, we were unable to identify either a consistent genetic driver or any specific patterns of tumor transformation.2 We are also concerned that some of our treatments may inadvertently contribute to grade transformation. To assess the treatment effects, we performed pre- and post- trans arterial embolization bulk-transcriptome analysis, which revealed cell population shifts in panNETs that may be linked to grade transformation.4To explore cellular shifts within the liver tumor microenvironment during panNET progression, we performed single-cell RNA sequencing. The resulting data was processed using Seurat.v5. The clusters of panNET cells, fibroblasts, hepatocytes, and immune cells were annotated by known markers. Additionally, we distinguished malignant cells from a reference of immune cells, fibroblasts, and hepatocytes by InferCNV. To understand the tumor progression, we performed trajectory analysis using monocle3. Differential metabolic activity between the panNETs grades were assessed using the scMetabolism package. Cell-cell communication and related ligand-receptor pairs were inferred using CellChatDB. Our results revealed distinct transcriptional trajectories, with pseudotime analysis demonstrating a trajectory of tumor evolution from low-grade (LG) to HG states. HG tumors were enriched for proliferative and aggressive subclusters compared to LG cells. Metabolic profiling further highlighted grade-specific adaptations, with HG tumors showing enhanced oxidative phosphorylation and TCA cycle across tumor cells, whereas LG tumors displayed relatively lower metabolic activity. Notably, immune cells were found metabolically active in LG via glycolysis. Conversely, HG panNETs exhibit glucose depletion in immune cells and less active microenvironment. The CellChat analysis revealed distinct cellular communication patterns in panNET grade. For instance, in HG tumors, signaling was primarily mediated between fibroblasts and panNET cells, with Glu-(SLC1A2+GLS)-GRM7 as the dominant ligand–receptor pair. In LG tumors, major interactions occurred among immune cells, hepatocytes, and panNET cells, with Glu-(SLC1A3+GLS)-GRIK2 representing the highest contributor. The results suggest that metabolic rewiring, altered intercellular signaling, and transcriptional plasticity underlie progression from LG to HG disease.
 Reference:
 1. Spolverato etal J Gastrointest Surg. 2017, 21(12):2039-2047
 2. Mayo etal Ann Surg Onc. 2011, 18(13):3657-65.
 3. Singh et al. Early ... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium. 2023.
 4. Singh et al. Post... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium.2023



 Himanshu N. Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jungeui Hong, Nitya Raj, Olca Basturk, Zeynep Tarcan, Christine A. Iacobuzio, Etay Ziv. Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f19a2d46016dbf4a94424ac84c3f39682a28b2d" target='_blank'>
              Abstract B025: Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors
              </a>
            </td>
          <td>
            Himanshu Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jung-Pyo Hong, Nitya Raj, O. Basturk, Z. Tarcan, Christine A. Iacobuzio, E. Ziv
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89051c5dd8812abec9fbc1c1e0e2195eabcd8e47" target='_blank'>
              Uncovering the Adaptive Tumor Immunity Interactions from a Single-Cell Level.
              </a>
            </td>
          <td>
            Tiankun Liu, Yuan Pang, Chang Zhou, Zhenzhen Zhou, Guoliang Li, Yuxuan Zheng, Weihao Hu, Zhihua Liu, Li Yu, Kaitai Zhang, Shujun Cheng, Wen Zhang, Wei Sun
          </td>
          <td>2026-01-20</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The detection of rare cell types in single-cell transcriptomics data is crucial for elucidating disease pathogenesis and tissue development dynamics. However, a critical gap that persists in current methods is their inability to provide an explanation based on genes for each cell they have detected as rare. We identify three primary sources of this deficiency. First, the anomaly detectors often function as"black boxes", designed to detect anomalies but unable to explain why a cell is anomalous. Second, the standard analytical framework hinders interpretability by relying on dimensionality reduction techniques, such as Principal Component Analysis (PCA), which transform meaningful gene expression data into abstract, uninterpretable features. Finally, existing explanation algorithms cannot be readily applied to this domain, as single-cell data is characterized by high dimensionality, noise, and substantial sparsity. To overcome these limitations, we introduce a framework for explainable anomaly detection in single-cell transcriptomics data which not only identifies individual anomalies, but also provides a visual explanation based on genes that makes an instance anomalous. This framework has two key ingredients that are not existed in current methods applied in this domain. First, it eliminates the PCA step which is deemed to be an essential component in previous studies. Second, it employs the state-of-art anomaly detector and explainer as the efficient and effective means to find each rare cell and the relevant gene subspace in order to provide explanations for each rare cell as well as the typical normal cell associated with the rare cell's closest normal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddba44f1e7e5ecb1a732ab848f2143eaae645b95" target='_blank'>
              A New Framework for Explainable Rare Cell Identification in Single-Cell Transcriptomics Data
              </a>
            </td>
          <td>
            Di Su, Kai Ming Ting, Jie Zhang, Xiaorui Zhang, Xinpeng Li
          </td>
          <td>2026-01-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e961f589165d98aa8aff64f0539527c2012d419" target='_blank'>
              Pooled single-cell screen in colorectal cancer defines transcriptional modules linked to oncogenes.
              </a>
            </td>
          <td>
            Viola Hollek, Francisca Böhning, Catalina Florez Vargas, A. Sieber, M. Morkel, Nils Blüthgen
          </td>
          <td>2026-01-19</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 225


 Background:
 Colorectal cancer (CRC) is the second leading cause of cancer-related deaths in the United States. Although immune checkpoint blockade benefits ~15% of patients with high mutational burden and microsatellite instability, the majority of CRC remains refractory, with cytotoxic chemotherapy as the standard of care. Identifying mechanisms that sustain cancer cell survival and proliferation is therefore critical to uncovering vulnerabilities that may enhance therapeutic efficacy. The complement peptide C5a, a potent effector generated by proteolytic activation of the complement cascade, signals through its receptor C5AR1 to drive cell survival, cytokine production, and chemotaxis. While historically regarded as a systemic pathway, recent evidence suggests that complement activation can also occur locally. We hypothesized that CRC tumors locally generate C5a, with cancer cell-intrinsic C5AR1 driving survival and chemoresistance.
 Methods:
 CRC tissue microarrays (TMAs) were generated from de-identified specimens (Hackensack Meridian
 Health
 ,
 N
 = 28; commercial TMA,
 N
 = 17). Preliminary spatial transcriptomics was performed on the commercial TMA using the Nanostring CosMx Spatial Molecular Imager. Publicly available single-cell transcriptomes from two independent CRC cohorts (
 N
 = 63 and
 N
 = 62) were reanalyzed. Multiplexed immunofluorescence (24-plex Cyclic IF) was applied to detect complement fragments (C3, C3b/iC3b, C5a) and C5AR1, and to map their cell type-specific expression in situ. Cancer cell-intrinsic C5AR1 expression was validated in two human CRC cell lines and four patient-derived organoids. Functional relevance of C5AR1 was assessed using in vitro cytotoxicity assays combining chemotherapy with pharmacologic C5AR1 inhibition.
 Results:
 Spatial transcriptomics revealed induction of complement pathway transcripts in CRC tumors, further confirmed by reanalysis of two independent CRC single-cell datasets. Multiplexed CycIF revealed an enriched localization of complement ligands (C3, C3b/iC3b, C5a) to tumor-adjacent regions, whereas C5AR1 was enriched in the tumor core. Furthermore, CycIF demonstrated robust C5AR1 protein expression in cytokeratin-positive cancer cells and infiltrating myeloid phagocytes; cancer cell-specific C5AR1 expression was further validated in two human CRC cell-lines and patient-derived organoids. Functionally, cancer cell-intrinsic C5AR1 proved essential for survival, and its pharmacologic inhibition synergistically enhanced 5-fluorouracil cytotoxicity.
 Conclusions:
 Complement activation is spatially organized in CRC, with cancer cell-intrinsic C5AR1 supporting survival. Pharmacologic C5AR1 inhibition synergizes with chemotherapy, positioning the C5a-C5AR1 axis as a tractable therapeutic vulnerability. Ongoing studies aim to define how C5AR1 supports cancer cell survival and mediates chemoresistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a950393c39eca04ec13c0ab1c5dcbbbb94f802" target='_blank'>
              Role of cell-intrinsic C5AR1 in colorectal cancer survival and chemoresistance.
              </a>
            </td>
          <td>
            Ashley Carlo, Raju Shivarathri, Adela Karuli, Richa Mandrekar, Anastasia Zenkevich, Chen-Hua Ma, Kar Chow, Alvin Makohon-Moore, Kevin Tong, Jigar V. Desai
          </td>
          <td>2026-01-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Transcription factors (TFs) orchestrate cellular responses to environmental signals and intercellular communication. The activity of TFs is influenced by neighboring cells, impacting cellular fate and function. Spatial transcriptomics (ST) allows for the mapping of mRNA expression across tissue samples, providing insights into the local microenvironment. However, the potential of ST data to systematically infer TF activity and its role in cell identity has not been fully exploited. We introduce STAN (Spatially informed Transcription factor Activity Network), a linear mixed-effects computational approach that predicts spatially informed, spot-specific TF activities by integrating curated TF–target gene priors, mRNA expression, spatial coordinates, and histological features. We demonstrate the utility of STAN on lymph node, dorsolateral prefrontal cortex, breast cancer, and glioblastoma ST datasets, identifying TFs associated with specific cell types, spatial regions, pathological zones, and ligand–receptor pairs. STAN enhances the utility of ST data, revealing the intricate interplay between TFs and spatial organization in diverse biological contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6512bfd807e57e3f57d134aa5a0e2a1701f3351b" target='_blank'>
              STAN, a computational framework for inferring spatially informed transcription factor activity
              </a>
            </td>
          <td>
            Linan Zhang, April Sagan, Bin Qin, Haoyu Wang, Elena Kim, Baoli Hu, H. Osmanbeyoglu
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba722b3f6a63490b9a491569ba714e8cdd49aa3e" target='_blank'>
              Informed spatially aware patterns for multiplexed immunofluorescence data.
              </a>
            </td>
          <td>
            Sagnik Bhadury, Michele Peruzzi, Satwik Acharyya, Joel Eliason, M. P. di Magliano, Timothy L. Frankel, V. Ravikumar, S. Krishnan, Arvind Rao
          </td>
          <td>2026-01-11</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Background Colorectal cancer (CRC) continues to represent a major cause of cancer-related mortality worldwide, with its progression and therapeutic outcomes strongly shaped by the complexity and heterogeneity of the tumor immune microenvironment (TME). This review critically examines the cellular and molecular mechanisms driving immune evasion in CRC, emphasizing the dual roles of immune cell populations—including tumor-associated macrophages, neutrophils, dendritic cells, T cells, B cells, and natural killer cells—as well as non-cellular elements such as the extracellular matrix and extracellular vesicles. Objective A key objective is to evaluate recent developments in immunotherapeutic approaches, including immune checkpoint inhibitors, tumor vaccines, adoptive cell transfer, and novel combinatorial regimens, while addressing their therapeutic promise and inherent limitations, especially in microsatellite-stable (MSS) tumors that exhibit primary resistance to standard immunotherapies. Further analysis integrates perspectives on metabolic reprogramming within the TME, epigenetic alterations, and advances in engineered cellular therapies, thereby providing a comprehensive framework for overcoming immunosuppressive mechanisms. Discussion and conclusion Special consideration is directed toward the translational value of targeting immune-metabolic interactions and spatial dynamics within the TME. Ultimately, this work synthesizes current knowledge and outlines forward-looking strategies to advance personalized, multi-target immunotherapy, with the potential to reshape clinical paradigms in CRC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03fb3a1b140db8c8ae9afa2ff97c1f40f6b8cd5d" target='_blank'>
              Cellular characteristics of the immune microenvironment of colorectal cancer and progress in immunotherapy research
              </a>
            </td>
          <td>
            Chunbaixue Yang, Jianchun Fan, Yixuan Zhang, Yixiao Zhang, Lei Xia, Xinran Cao, Xueliang Wu
          </td>
          <td>2025-11-30</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa68ef0a898a9de3aed5f44e700f118ea58e14a" target='_blank'>
              A cellular epigenetic classification system for glioblastoma.
              </a>
            </td>
          <td>
            Dana Silverbush, Liv Jürgensen, Nelson F. Freeburg, Channing Pooley, Fabio Boniolo, Federico Gaiti, M. Suvà, V. Hovestadt
          </td>
          <td>2026-01-07</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc440df78b8e3b097a9180b03a9b83b2a7188ec3" target='_blank'>
              Senescence-Linked Fibrosis in the Aging Human Ovary Revealed by p16-Based Histological Profiling and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Mark Watson, P. Devrukhkar, N. Murad, Fan Wu, Moo Joong Kim, Hannah Anvari, Uyen Tran, Nicholas Martin, Tommy Tran, Giuliana Zaza, Kevin Schneider, Bikem Soygur, Elisheva D. Shanes, Denis Wirtz, M. Pavone, Simon Melov, Pei-Hsun Wu, D. Furman, Francesca E. Duncan, B. Schilling
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/747c7b8e23a309f86e30344cfec39c8a2da47823" target='_blank'>
              DeepDynamics resolves cell-subtype and clinicopathological dynamics from bulk RNA-seq to identify mediators of Alzheimer’s disease risk
              </a>
            </td>
          <td>
            Yifat Haddad, Yuval Rom, G. Green, Anael Cain, Barak Raveh, Naomi Habib
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background Early tumour vascular invasion contributes to cancer progression. Tip cells, a subset of tumour endothelial cells, significantly decline after anti‐angiogenic therapy. However, their behaviour and the roles of their signature genes during early invasion are incompletely understood. Methods This study employed single‐cell transcriptomic analysis and 10x Genomics Visium spatial transcriptomics on fresh lung tissues from patients with pulmonary nodules and from KrasG12D (K) and KrasG12DTgfbr2−/− (KT) mice. The role of plasma vesicle‐associated protein (PLVAP), a tip cell marker, was further examined using survival databases, immunofluorescence, in vitro co‐culture, cell migration, invasion assays and endothelial tube formation. Results Tip cell proportions were elevated in early‐stage lung adenocarcinoma (LUAD) tissues and KT mice, with evidence suggesting they arise from capillaries type I. PLVAP expression was enriched in tumour endothelial cells, induced by TGFβ1, and negatively correlated with patient prognosis. Functionally, PLVAP promoted endothelial cell invasion, migration and angiogenesis, and regulated tumour cell invasiveness. Intercellular analysis revealed that some tip cells also expressed TGFβ1, which may act on adjacent tumour cells to enhance invasion during early tumour development. Conclusion Tip cells increased during early LUAD progression and likely evolved from capillaries type I. Their marker PLVAP was associated with poor prognosis and pro‐invasive endothelial behaviour. Tumour‐secreted TGFβ1 upregulated PLVAP in endothelial cells, promoting angiogenesis and tumour invasion. Additionally, tip‐cell‐derived TGFβ1 may further stimulate tumour aggressiveness, highlighting a reciprocal interaction that contributes to early tumour progression. Key points Tip cells expand during early LUAD progression and likely originate from capillary type I endothelial cells. Tumour‐derived TGFβ1 induces PLVAP expression in endothelial cells, linking tumour signals to vascular activation. PLVAP enhances endothelial cell migration, invasion and angiogenic capacity. Endothelial PLVAP promotes tumour cell invasiveness, revealing a reciprocal endothelial‐tumour interaction that drives early tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79021428fe4068225ddb196bfe79a293a09eade5" target='_blank'>
              PLVAP mediates the regulation of the tumour microenvironment in early‐stage lung adenocarcinoma
              </a>
            </td>
          <td>
            Linshan Xie, Mengting Sun, Yujie Zheng, Zezhong Yu, Hui Kong, Jinjie Yu, Shaohua Lu, Yong Zhang, Jie Hu, Hongyi Xin, Jian Zhou, Xiangdong Wang, Charles A Powell, Fred R Hirsch, Chunxue Bai, Yuanlin Song, Jun Yin, Dawei Yang
          </td>
          <td>2025-12-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) and pancreatic ductal adenocarcinoma (PDAC) are highly lethal cancers marked by profound epigenetic and metabolic reprogramming. Among the candidate biomarkers, the DNA methyltransferase DNMT3A and the guanine monophosphate synthetase (GMPS) have emerged as potential prognostic drivers, yet their roles across tumor contexts remain unclear. Here, we demonstrate the application of KMplotter to interrogated pan-cancer transcriptomic and survival datasets encompassing over 7000 patients, complemented by expression profiling of normal, tumor, and metastatic tissues, and integrated tumor microenvironment (TME) analyses. Elevated DNMT3A and GMPS expression correlated with worse overall survival in HCC, particularly in Asian patients, while in PDAC, high DNMT3A but low GMPS expression predicted favorable outcomes. Both genes were consistently upregulated in tumors relative to normal tissues, with further increases in metastatic HCC. Immune deconvolution revealed that DNMT3A was linked to Th2/Treg-enriched niches, whereas GMPS overexpression coincided with high mutational burden or stromal enrichment, fostering immunosuppressive microenvironments. Comparative analysis of toll-like receptor signatures highlighted divergent antigen-sensing pathways, with HCC reflecting viral-driven immune exhaustion and PDAC showing self-antigen–associated signaling. Collectively, these findings position DNMT3A and GMPS as context-dependent biomarkers that integrate metabolic and immune cues to shape prognosis in liver and pancreatic cancer, offering mechanistic insight and translational relevance for patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c71a88a2f1a7eea4d4abf0ff75d340e4a7fe137" target='_blank'>
              Comparative Tumor Microenvironment Analysis for HCC and PDAC Using KMplotter
              </a>
            </td>
          <td>
            Wen-Han Chang, Drashya Shah, Scott Myers, Michael Potts, Sanjive Qazi, Vuong Trieu
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4d4eea61e2f155251ae235dac3fc5c5c43d18" target='_blank'>
              A pan-cancer single cell landscape reveals heterogeneity and functional diversity of double-negative T cells.
              </a>
            </td>
          <td>
            Qing Hao, Tao Zhou, Huayun Yan, Zhixiang Ren, Wei Mao, Li-Bin Huang, Yue-Yun Chen, Yuelan Wang, Minqin Xiao, Yajiao He, Xuyang Xia, Jing Wu, Wei-Han Zhang, Hai-Ning Chen, Maoxiang Qian, Feng Zhang, Li Yang, Lunzhi Dai, Huiyuan Zhang, Canhua Huang, Yang Shu, Yu Xu, Yang Luo, Yong Peng, Heng Xu
          </td>
          <td>2026-01-03</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 The introduction of bowel cancer screening has led to an increase in the detection and removal of pre-malignant adenomas, however many of these would never progress to colorectal cancer (CRC) within a patient’s lifetime. This implies a key evolutionary bottleneck at the transition from adenoma to carcinoma, with immune surveillance likely playing a central role in suppressing the outgrowth of invasive cells. Here we analysed a cohort of “ca-in-ads” (adenomas with a small focus of cancer) to provide a unique snapshot of the invasive transition. We combined genomics, spatial transcriptomics, digital pathology and multiplex imaging, concurrently deriving the distribution of tumour clones, their gene expression and their interplay with the immune microenvironment. We performed multi-region low-coverage whole genome sequencing (lcWGS) and T-cell receptor sequencing (using FUME-TCRseq, a novel method developed in our lab) on archival ca-in-ads from 40 patients. For a subset of cases we performed spatial transcriptomics using the 10x Genomics Visium platform, including custom probes to detect T-cell clonotypes of interest. We applied a deep learning cell classifier to haematoxylin and eosin (H&E) stained sections to determine the abundance and spatial distribution of eight cell types, with detailed immunophenotyping on selected cases (using a 50-marker panel on Akoya’s PhenoCycler-Fusion platform). We found a significant increase in copy number alterations in carcinoma regions relative to adenomas, and this was accompanied by a large shift in the composition of the local T-cell repertoire. TCR sequencing revealed expanded T-cell clonotypes that were adenoma-specific or carcinoma-specific, and we validated the spatial exclusivity of these using a custom Visium assay. Although overall lymphocyte density remained unchanged, digital pathology and cyclic immunofluorescence revealed significantly increased infiltration of macrophage and neutrophil populations in regions of invasive cancer. Taken together our analysis suggests a striking reshaping of tumour-immune dynamics at the transition to an invasive phenotype in CRC. Understanding what governs this invasive shift could highlight new avenues for cancer prevention or identify individuals at risk of CRC.



 Ann-Marie Baker, Eleanor Smith, Chirine Sakr, Konstantin Bräutigam, Nick Trahearn, Holly Eggington, Miangela M. Lacle, Manuel Rodriguez-Justo, Simon J. Leedham, Trevor A. Graham. Multi-omic spatial analysis reveals reshaping of tumour-immune dynamics at the transition to invasive colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr PR004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b4fa90cafae68daf434680b6837e68209ae59d6" target='_blank'>
              Abstract PR004: Multi-omic spatial analysis reveals reshaping of tumour-immune dynamics at the transition to invasive colorectal cancer
              </a>
            </td>
          <td>
            Ann-Marie Baker, Eleanor Smith, Chirine Sakr, Konstantin Bräutigam, N. Trahearn, Holly Eggington, M. Laclé, M. Rodriguez-Justo, Simon J. Leedham, Trevor A. Graham
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive subtype of breast cancer characterized by the absence of estrogen, progesterone, and HER2 receptor expression. This malignancy is often associated with a poor prognosis, early recurrence, and limited treatment options. The tumor microenvironment (TME) in TNBC plays a pivotal role in tumor progression, immune evasion, and therapeutic resistance. In recent years, an increasing body of evidence has highlighted the critical interactions between cancer cells and the components within the TME, including immune cells and soluble components. These interactions influence not only the biological behavior of the tumor but also its response to treatment. Exploring the complex interplay between tumor cells and immune components continues to inform the development of more effective therapeutic approaches. In this study, we provide a synopsis of advancements regarding the TME in TNBC. In light of different cellular compartments, we delineate multiscale interplays within the stroma–tumor symbiosis and highlight their antitumor functions and promising targeting strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11bfc269114236644c5ca2675c1d14983686fe40" target='_blank'>
              Prominent protumoral cellular compartments of the tumor microenvironment in triple-negative breast cancer
              </a>
            </td>
          <td>
            Shuai Sun, Pan Zhao, Changrong Wang, Junjun Du, Tingting Zhang, Xiangyun He, Zhibo Zuo, Nan Li, Rongjing Zhou
          </td>
          <td>2026-01-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c745807b38045a754b3f57a50a75c7b86c914ca" target='_blank'>
              A Scalable Framework for Pan-Cancer Tumor Evolution Analysis Enables Transfer of Progression Mechanisms Across Tumor Entities
              </a>
            </td>
          <td>
            Simon Pfahler, Andreas Lösch, Y. L. Hu, Rudolf Schill, Rainer Spang, T. Wettig
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Purpose
Colorectal cancer (CRC) lung metastases exhibit high recurrence rates after resection, underscoring the need for improved therapeutic strategies. This study aimed to characterize the tumor microenvironment (TME) of CRC lung metastases and identify the factors associated with recurrence.


Materials and Methods
Fifteen CRC patients who underwent lung metastasectomy were enrolled. Multiplex immunohistochemistry (IHC), whole exome sequencing, transcriptome profiling, and single-cell RNA sequencing (scRNA-seq) were conducted on matched tumor, adjacent and distant normal lung tissues. Immune cell populations and gene expression profiles were analyzed and correlated with clinical recurrence outcomes.


Results
Exome and transcriptome analyses revealed frequent TP53, KRAS, and APC mutations. Most tumors corresponded to consensus molecular subtypes 2 and 4, characterized by immune-depleted and fibrotic features. Tumors showed downregulation of effector T and NK cell signatures. IHC revealed reduced density and increased distance of CD8+ T cells and macrophages from the epithelial cells. scRNA-seq demonstrated increased regulatory T cells and decreased NK and effector T cells in tumor. Tumor-associated macrophages (TAMs), particularly SPP1 (osteopontin)-expressing subsets, were markedly enriched in tumor and correlated with suppressed effector T cella activity. High SPP1 expression was associated with early recurrence and poor overall survival. Patients with recurrence had higher proportion of PD-1+ CD8+ T cells in adjacent normal tissues.


Conclusion
Immunosuppressive features including enrichment of SPP1+ TAMs and depletion of effector T and NK cells contribute to recurrence after CRC lung metastasectomy. Therapeutic strategies targeting both TAMs and T cells may enhance clinical outcomes in this patient population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae972bf762bae15538dd4da3de79b673b4d58b2d" target='_blank'>
              Immunosuppressive Tumor Microenvironment in Colorectal Cancer Lung Metastases: Implications for Recurrence After Metastasectomy.
              </a>
            </td>
          <td>
            Minsuk Kwon, Minkyue Shin, M. An, Y.J. Jeon, T. Hong, Jung Kyong Shin, S. Lim, Yoonah Park, Y. Cho, Seung-Tae Kim, Y. Choi, Jeeyun Lee
          </td>
          <td>2025-12-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="


 The mechanisms by which a small subset of high-risk localized prostate cance becomes immunogenic is not well understood. The objective of this study is to identify the tumor intrinsic and extrinsic mechanisms associated with immunogenicity in this population with the goal of harnessing this knowledge to further immunotherapy in PCa. Methods We performed Visium and Xenium spatial transcriptomic profiling on radical prostatectomy specimens from untreated, high-risk, localized prostate cancer patients obtained from the UCLA biobank with varying level of intra-tumoral T-cell infiltration (TILs). Validation was performed using OPAL multiplex immune-fluorescence (mxIF) staining for CD4, CD8A, CD68, PIGR (marker of club cells), and PSA on 18 specimens from a separate cohort of untreated, intermediate to high-risk localized prostate cancer. Summary Spatial transcriptomic analysis identified cells with club-like features to be associated with global T cell infiltration. Both benign epithelium and PCa can adopt the club-like features by the expression of PIGR protein. Benign glands with club-like features have reduced androgen receptor activity while club-like PCa are enriched with PTEN loss by IHC. Regardless of cell type, expression of PIGR leads to close contact of T cells and club-like cells. Xenium profiling revealed that most T cells are activated CD4+ stem T cells expressing TCF1 protein that are spatially enriched with myeloid dendritic cells and plasmacytoid dendritic cells. Conclusions We defined a subset of immunogenic localized prostate cancer that expresses PIGR either in benign or tumor epithelium. Epithelial expression of PIGR is associated with TILs, suggesting a non-canonical route for immune recruitment and activation.



 Anson Ku, Rong Rong Huang, Sumeyra Kartel, Xingyu Chen, Kayla Heyward, Ross Lake, Rosina Lis, Nikil Pramod, Shana Trostel, Anna Baj, Etyan Ruppin, Scott Wilkinson, John Fenimore, Huihui Ye, Adam Sowalsky. Tumor PTEN-Loss is Associated with Club-Like Cells Phenotype and CD4 T-Cell Infiltration in High-Risk Localized Prostate Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae61f5a94b5b745d520039bbb08b848e6edeb40a" target='_blank'>
              Abstract A035: Tumor PTEN-Loss is Associated with Club-Like Cells Phenotype and CD4 T-Cell Infiltration in High-Risk Localized Prostate Cancer
              </a>
            </td>
          <td>
            A. Ku, R. Huang, Sumeyra Kartel, Xingyu Chen, Kayla Heyward, R. Lake, Rosina T. Lis, Nikil Pramod, S. Trostel, A. Baj, Etyan Ruppin, S. Wilkinson, J. Fenimore, Huihui Ye, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077d8d4a45150f5dcadd79e304aadcb9aba01396" target='_blank'>
              YAP1 Is a Crucial Nexus in the Tumor Microenvironment.
              </a>
            </td>
          <td>
            Raghavan Narasimhan, Anshula Narayanasamy, Jaya Padmanabhan, Srikumar Chellappan, Durairaj Mohan Kumar
          </td>
          <td>2026-01-13</td>
          <td>Current medical science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-18</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b86fac5bb6fe4f8e5891fadb0d9a3f2b55917a3" target='_blank'>
              Impact and correction of segmentation errors in spatial transcriptomics.
              </a>
            </td>
          <td>
            Jonathan Mitchel, Teng Gao, Viktor Petukhov, Eli Cole, Petr N. Kharchenko
          </td>
          <td>2026-01-20</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1f51bba74a8851f75d52e66b786b9875c816a69" target='_blank'>
              Ex Vivo Immuno-Oncology Platform Reveals Spatial T Cell Infiltration Patterns Linked to ATR Inhibition Responses in High-Grade Serous Ovarian Cancer.
              </a>
            </td>
          <td>
            Ashwini Sakrepatna Nagaraj, M. Salko, Aditi Sirsikar, Ziqi Kang, E. Erkan, E. Pietilä, Iga Niemiec, Jie Bao, G. Marchi, Angéla Szabó, María Hincapié-Otero, Anastasia Lundgren, Kirsten Nowlan, S. Pikkusaari, A. Kanerva, J. Tapper, Riitta Koivisto-Korander, H. Lassus, L. Kauppi, S. Hautaniemi, Anna Vähärautio, Jing Tang, E. Kekäläinen, Ulla-Maija Haltia, A. Virtanen, J. Jukonen, T. Salo, Anniina Farkkila
          </td>
          <td>2026-01-21</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61684838fc3c53a5cd2329af114682e03471ba75" target='_blank'>
              Single-Cell Mendelian Randomization Identifies Cell-Type Specific Genetic Drivers of Lung Cancer Subtypes
              </a>
            </td>
          <td>
            Q. Yang, C. Yan, X.-f. Wang
          </td>
          <td>2025-12-02</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Programmed cell death protein-1 (PD-1) blockade has shown promising clinical efficacy in hepatocellular carcinoma (HCC), yet the underlying immunological mechanisms governing response and resistance remain unclear. This study aimed to delineate the temporal and spatial dynamics of T-cell clonotypes and their relationship with pathological response in neoadjuvant PD-1 blockade therapy in HCC. By integrating T-cell receptor (TCR) repertoire analysis with transcriptomic profiling, we sought to elucidate the immune landscape alterations associated with treatment outcomes. Methods We analyzed longitudinal, matched tumor and blood samples from our previous clinical trial (NCT05471674), encompassing 114 specimens, including pre-nivolumab tumor biopsies, post-treatment resected tumors, adjacent non-tumorous livers, and peripheral blood collected before, after, and during follow-up from 19 treatment-naïve HCC. TCR clonality, diversity, and transcriptomic signatures were assessed to characterize immune responses. Results Elevated clonality of intratumoral TCR clonotypes (ITCs) post-treatment correlated with increased pathological tumor necrosis, predominantly driven by the most abundant top 1% ITCs. These dominant clonotypes occupied a significantly larger clonal space and demonstrated increased spatial sharing across tumor, non-tumorous liver, and peripheral blood compartments in nivolumab responders (nivo-Rs). Shared clonotypes were positively associated with tumor necrosis extent and cytolytic activity, suggesting active immune engagement. Post-treatment, shared clonotypes exhibited peripheral expansion and greater intratumoral dominance in nivo-Rs, whereas tumor unique intratumoral TCR clonotypes (ITCs) remained prevalent in non-responders (nivo-NRs). These divergent patterns were linked to higher chemokine expression within the tumor microenvironment of nivo-Rs and impaired human leukocyte antigen (HLA) class II antigen presentation in nivo-NRs, indicating distinct immune landscape configurations influencing therapeutic response. Conclusions Our findings demonstrate that the peripheral infiltration and expansion of high-frequency intratumoral T-cell clonotypes are critical drivers of pathological response to neoadjuvant PD-1 blockade in HCC, highlighting potential immune biomarkers and therapeutic targets to enhance immunotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/901935af9edd3945285033fa6917d3b60bbe154f" target='_blank'>
              Spatiotemporal dynamics of T cells in peripheral blood and tumor underlying differential responses to neoadjuvant PD-1 blockade in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yajing Zhang, Y. Tsui, J. Lee, T. Cheung, DanielWai-Hung Ho, I. Ng
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive tumor type with poor patient outcomes. Most patients present with metastatic disease, which generally has reduced immune infiltration compared to primary tumors. Further work to elucidate the specific cellular features of metastatic PDAC is needed to guide the development of future immunotherapy strategies. Here, we investigated the hypothesis that PDAC tumors harbor distinct immunologic and stromal features depending on their anatomical site. Multiplex immunohistochemistry (mIHC), spatial analysis, and single-cell mass cytometry (CyTOF) uncovered dominant immune and stromal cell populations in tumors derived from 27 primary and 26 liver metastases. Metastatic liver tumors from PDAC patients contained reduced T cell infiltration, fibroblast populations, and collagen accumulation than primary lesions, while CD68+ cells, often co-expressing CCR2, were more abundant. Spatial analyses revealed distinct immune cell communities in primary and metastatic PDAC, whereby CK19+ cells clustered differentially with α-SMA+, CD3+, and CD68+ cells, depending on the tumor site. When comparing tumor-associated regions, the proportion of peritumoral CK19- cells remained consistent, but their composition varied by disease site. CD8+ T cells were significantly less frequent in metastatic tumors, while both CD4+ and CD8+ T cells present in primary tumors expressed more transcription factors (TFs) associated with suppressive properties, including FoxP3 and RORγt. CyTOF revealed that T cells co-expressed multiple inhibitory checkpoint receptors, most notably LAG-3 and PD-1. This report reveals that primary and metastatic tumors from PDAC patients harbor vastly distinct immunologic and stromal features at the protein level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bea286cd97f7f8064af812daefa94a7da6b7d73" target='_blank'>
              High-Dimensional Protein Analysis Uncovers Distinct Immunological and Stromal Features in Primary and Metastatic Pancreatic Ductal Adenocarcinoma.
              </a>
            </td>
          <td>
            E. Greene, N. Horvat, D. Doxie, V. Parihar, Jayden Kim, Cameron J Herting, Erin E. Grundy, Ayana T Ruffin, Alyssa M Krasinskas, Shishir K Maithel, Juan M. Sarmiento, Mihir M. Shah, Mohammad Y Zaidi, Maria Diab, O. Alese, Kavita M. Dhodapkar, H. Kissick, B. El-Rayes, Chrystal M. Paulos, G. Lesinski
          </td>
          <td>2025-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d360e6a196e9de4b8313a49f5e3b66fe3b338c6" target='_blank'>
              Integrative single-cell and spatial transcriptomics with explainable AI reveal lethal prognostic axis in prostate cancer.
              </a>
            </td>
          <td>
            Qintao Ge, Zhenda Wang, Yangyun Wang, Tonghui Chu, Zhongyuan Wang, Wenkai Zhu, Youzhao Zhang, Yonghao Chen, Dingwei Ye, Wenhao Xu, Zhongyuan Wang, Mierxiati Abudurexiti
          </td>
          <td>2026-01-06</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) displays marked cellular heterogeneity and immune microenvironment complexity that fundamentally influence transcatheter arterial chemoembolization (TACE) treatment responses and patient outcomes. Deciphering the molecular architecture underlying therapy resistance remains essential for advancing precision oncology in HCC management. Methods We integrated four single-cell RNA sequencing cohorts with bulk transcriptomic datasets and longitudinal clinical annotations from The Cancer Genome Atlas Liver Hepatocellular Carcinoma database and Gene Expression Omnibus repositories to perform multidimensional analyses. Computational frameworks including single-cell Phenotype Associated Score (scPAS), high-dimensional weighted gene co-expression network analysis (hdWGCNA), and Single-Cell Regulatory Network Inference and Clustering (SCENIC) were deployed to identify resistance-linked cellular subpopulations and pivotal regulatory modules. Functional validation employed adrenomedullin (ADM)-depleted Huh7 cellular models and xenograft tumor-bearing mouse systems, with mechanistic interrogation via Western immunoblotting, quantitative reverse transcription polymerase chain reaction, and Kaplan-Meier survival estimation to confirm ADM biological functions and clinical relevance. Results We identified a TACE-resistant malignant cell subset (scPAS+) characterized by pronounced activation of glycolytic, hypoxic, and epithelial-mesenchymal transition pathways alongside overexpression of resistance-conferring genes including LINC00221, hexokinase 2, and alpha-fetoprotein. This cellular phenotype demonstrated robust associations with TACE non-responsiveness, sorafenib cross-resistance, and abbreviated patient survival. Patient stratification based on scPAS + signature genes delineated two distinct molecular subgroups: the scPAS + -enriched cohort exhibited marked TACE refractoriness, elevated sorafenib failure rates, immunosuppressive microenvironmental architecture, and diminished 5-year survival probability. Mechanistic investigations established ADM as a critical driver orchestrating this resistance phenotype. ADM depletion attenuated Huh7 cell proliferative capacity, migratory potential, and invasive behavior, reduced xenograft tumor burden in murine models, and substantially potentiated sorafenib antitumor efficacy. Conclusion This study delineates an ADM-driven TACE-resistant HCC cellular subtype (scPAS+) that functions simultaneously as a prognostic biomarker and actionable therapeutic target for circumventing treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de9c2505a2f010ea34035f6766d4f925d037d481" target='_blank'>
              Adrenomedullin orchestrates treatment resistance in hepatocellular carcinoma via immune microenvironment remodeling
              </a>
            </td>
          <td>
            Xixi Gao, Yongliang Sun, Jia Huang, Li Xu, Hanchun Huang, Zhiying Yang
          </td>
          <td>2025-12-11</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917b475731c2b6642606d58ea37295a894578bc5" target='_blank'>
              Decoding Neoantigen-encoding Tumor-Specific Transcripts Unveils a Shared Target Reservoir for Immunotherapy in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Peng Lin, Yifan Wen, Jingjing Zhao, Feifei Zhang, Yaoming Su, Hongwu Yu, Qiaojuan Li, Chengye Liu, Zhixiang Hu, Yan Li, Zhuting Fang, Linhui Liang, Shenglin Huang
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The human endometrium is a regenerative tissue relying on stem/progenitor cells. Endometrial mesenchymal stem cells (eMSCs) are typically enriched using perivascular markers like CD140b and CD146. However, the identity of more primitive and quiescent eMSC subpopulations remains unclear. We performed single-cell RNA sequencing (scRNA-seq) on cultured CD140b⁺CD146⁺ eMSCs and integrated this with published scRNA-seq data of primary human endometrial cells. We identified a LEPR⁺ subpopulation and analyzed its characteristics through in vitro assays, flow cytometry, immunostaining, and bioinformatic tools including cell–cell interaction analysis and pseudotime trajectory inference. A LEPR⁺ eMSC subpopulation was found to reside at the root of the differentiation trajectory and showed high expression of Notch receptors. These cells exhibited quiescent features, resided predominantly in the G0 phase, and demonstrated superior clonogenic and self-renewal capacity compared to LEPR⁻ eMSCs and bulk eMSCs. Notch signaling, particularly via JAG1 and DLL1, was implicated in maintaining the LEPR⁺ phenotype and quiescence. LEPR⁺ eMSCs represent a primitive, quiescent subset of human endometrial stem cells. Notch signaling maintains their stemness and quiescence, suggesting therapeutic relevance for endometrial regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38ff042de127bdacda290e747aeba235e073b979" target='_blank'>
              Single-cell transcriptomics reveals notch regulation in quiescent LEPR⁺ endometrial mesenchymal stem cells
              </a>
            </td>
          <td>
            Yuan Fang, Dandan Cao, Cheuk-Lun Lee, PhilipC.N. Chiu, Ernest H Y Ng, William S B Yeung, Rachel W S Chan
          </td>
          <td>2025-12-01</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The Tracking Cancer Evolution Through Therapy (TRACERx) program represents the most comprehensive effort to characterize tumor evolution in real time. Through longitudinal, multiregion, and multiomic profiling of tumors—and particularly of non-small-cell lung cancer and clear cell renal cell carcinoma—TRACERx has illuminated the dynamic interplay between genetic, nongenetic, and (micro)environmental factors that drive cancer progression, immune evasion, and therapeutic resistance. A central insight from TRACERx has been that not all tumor evolution is genomic: Transcriptomic diversity, epigenetic alterations, RNA editing, and changes in cell–cell interactions also drive adaptation. Methodological innovations—including tumor-informed and ultrasensitive circulating tumor DNA assays, representative sequencing, and integrative immune–genomic analyses—have yielded biomarkers resistant to sampling bias and/or predictive of recurrence, metastasis, and treatment response. By demonstrating that intratumor heterogeneity is a key determinant of clinical outcome and revealing its molecular, transcriptional, and ecosystem-level drivers, TRACERx has established a framework for linking evolutionary dynamics to patient care. As both a scientific framework and a clinical paradigm, TRACERx demonstrates how adaptive, iterative research can refine evolutionary models, improve patient risk stratification, and inspire next-generation cancer evolution studies across malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a59431b9259c4928178579f6c460cf8e369f4f1" target='_blank'>
              TRACERx as an Optimized Paradigm for Understanding Cancer Evolution
              </a>
            </td>
          <td>
            Naomi Iris van den Berg, C. Swanton, S. Turajlic
          </td>
          <td>2025-12-24</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea05c9d911e5acfb02ab472396d64afacfb09aa" target='_blank'>
              Spatiotemporal Evolution of Malignant Hepatocyte Clones Unveils Immune Evasion Features and Therapeutic Vulnerabilities in Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Shizheng Xiong, Dekang Ren, Jiaming Jin, Linjie Zhao, Chengjun Gong, Jiarui Liu, Wanjie Zheng, Zhiheng He, Haochuan Guo, Li Guo, Tingming Liang
          </td>
          <td>2026-01-01</td>
          <td>Biotechnology journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is critically implicated in glioblastoma (GBM) recurrence, therapeutic resistance, and immune evasion. By analyzing both primary and recurrent GBM from Chinese Glioma Genome Atlas (CGGA) and clinical samples, we identified significant differences in TME. To overcome the limitations of traditional murine recurrent GBM model, we successfully established a new murine model and characterized the TME differences between primary and recurrent GBM in vivo. Based on immune chemokine profiling within the CGGA dataset, we selected PLX3397, a Colony stimulating factor 1 receptor (CSF1R) inhibitor, to target recurrent GBM in our murine model. PLX3397 treatment significantly attenuated tumor growth and remodeled TME. Collectively, our study firstly analyzed TME combine a large number of database samples and clinical samples, and made the first application of PLX3397 in a murine recurrent GBM model, thereby providing a novel therapeutic strategy and experimental foundation for recurrent GBM research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a92fc4a3398e8830aed11264d560ebd40da05b6" target='_blank'>
              PLX3397 attenuated tumor growth and remodeled tumor microenvironment of recurrent glioblastoma
              </a>
            </td>
          <td>
            Chen Yang, Linzhi Cai, Xi Xiang, Xiwei Zhu, Chengli Liu, Jianming Liao, Zhan Zhang, Mingchang Li
          </td>
          <td>2025-12-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tumor heterogeneity is one of the central challenges in oncology, contributing to treatment resistance and disease recurrence. Bulk RNA sequencing has advanced understanding of tumor biology, yet its averaging effect conceals cell type-specific alterations. Single-cell RNA sequencing overcomes this limitation by capturing gene expression and cellular phenotypes with high-resolution, thereby illuminating tumor composition and the surrounding microenvironment. Within this framework, differential abundance (DA) detection has emerged as a powerful strategy to quantify shifts in cell population proportions across conditions. Unlike differential gene expression, DA highlights compositional changes in cellular ecosystems, offering a structural perspective on tumor dynamics. This review introduces the main categories of DA methods in single-cell RNA sequencing analysis, outlining their modeling strategies, assumptions, and representative applications in oncology. We also discuss key challenges, including reliance on clustering quality and batch correction. By linking methodological principles with biological insight, this review clarifies the role of DA detection in single-cell oncology and provides a conceptual framework for integrating compositional analysis into efforts to understand tumor evolution, treatment response, and disease stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d95711e5982454360e330d82a51ace20f5d676f0" target='_blank'>
              Single-cell differential abundance detection: A new angle on dissecting tumor heterogeneity
              </a>
            </td>
          <td>
            Yingxue Xiao, Jing Sun, Lingling Xie, Yue Zou, Tong Li, , Bo Li
          </td>
          <td>2026-01-24</td>
          <td>World Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (iCCA) is a highly metastatic tumor type, and iCCA with intrahepatic metastasis is associated with poor prognosis. Previous studies have confirmed that the tumor microenvironment (TME) is closely related to tumor metastasis. However, the state of the TME during iCCA liver metastasis remains unknown.


APPROACH AND RESULTS
We profiled 28 specimens from 16 patients with iCCA (with and without intrahepatic metastasis) using integrated single-cell and spatial transcriptomics, bulk RNA sequencing, whole-exome sequencing, and in vivo and in vitro functional experiments to characterize the specific TME changes during iCCA liver metastasis. we identified the metastatic tumor cells, COL3A1+ epithelial cells, and revealed that COL3A1+ epithelial cells-endothelial-to-mesenchymal transition cells crosstalk promoted tumor invasion by inducing mesenchymal transformation of endothelial cells, while COL3A1+ epithelial cells-VEGFA+ CCL4+ neutrophils crosstalk promoted tumor colonization by forming neutrophil extracellular traps.


CONCLUSIONS
We revealed the key biological mechanisms involved in the invasion and colonization phases of iCCA liver metastasis. Moreover, we provide valuable data for understanding iCCA liver metastasis and a possible avenue for the treatment of advanced iCCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd684325d487cd1d69724768d37142d24e7c6b41" target='_blank'>
              Spatiotemporal landscape of intrahepatic cholangiocarcinoma liver metastasis at the single-cell level.
              </a>
            </td>
          <td>
            Ziyu Xun, Hao Wang, Zhengfeng Xuan, Kexu Xiong, Na Zhang, J. Long, Huishan Sun, Yiran Li, Chengpei Zhu, M. Piao, Ting Zhang, Longhao Zhang, Shuofeng Li, Chengjie Li, Jiongyuan Li, Boyu Sun, Zi-Qi Zhou, Shanshan Wang, Ziyue Huang, Kai Liu, Yang Tan, Xiaohua Shi, Xiaobo Yang, Hanping Wang, Ling Lu, Haitao Zhao
          </td>
          <td>2025-12-10</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b75da29edbdbad60fea18ff4b3049824f5017d" target='_blank'>
              Critical role for a high-plasticity cell state in lung cancer.
              </a>
            </td>
          <td>
            Jason E Chan, Chun-Hao Pan, Jonathan Rub, Gary Guzman, Klavdija Krause, Emma Brown, Zeda Zhang, Hannah Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2026-01-21</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="As per the global mortality-related data, metastasis and tumor-related relapse are the major determinants of cancer-related deaths. This phenomenon is largely driven by tumor dormancy - a state in which disseminated tumor cells (DTCs) persist in a non-proliferative phase. These dormant cells evade immune surveillance and resist conventional therapies, contributing to late relapse and metastatic outgrowth. Dormancy is maintained through intricate crosstalk between cancer cells and the microenvironment, involving extracellular matrix components, and various cellular signaling pathways. However, changes in these microenvironmental cues can disrupt this balance and reactivate dormant cells, leading to their proliferation and metastatic colonization. The undetectability of dormant DTCs complicate therapeutic targeting, underscoring the need to elucidate the molecular and epigenetic mechanisms that regulate dormancy maintenance and escape. This review explores the key signaling mechanisms and microenvironmental influences that regulates the tumor dormancy. Furthermore, we discuss emerging therapeutic strategies aimed at eradicating dormant cancer cells - either by maintaining dormant state, reactivating and sensitizing dormant cells to chemotherapy, or directly eliminating dormant populations. A deeper understanding of dormancy biology holds promise for developing innovative interventions to prevent recurrence and improve long-term patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8ce82456facc97c29458a64f714585ff1d48b53" target='_blank'>
              Sleepyhead, deadly awakening: the dynamics of metastatic organotropism, tumor dormancy and therapeutic implications
              </a>
            </td>
          <td>
            Soumyajit Sarkar, Suresh P. K.
          </td>
          <td>2025-12-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Head-and-neck squamous cell carcinoma (HNSCC) remains a lethal malignancy with stagnant survival despite advances in surgery, radiotherapy, and systemic therapy. Beyond cetuximab and PD-1 inhibitors, there are only a few targeted options, which benefit only a minority of patients, underscoring the need for new biomarkers and druggable dependencies. Genome-wide clustered regularly interspaced short palindromic repeats (CRISPR) Cas9 screening now enables systematic, high-specificity investigation of gene function to reveal determinants of tumor proliferation, survival, and therapy response. Compared with RNA interference, CRISPR provides cleaner on-target knockout and more interpretable phenotypes, allowing efficient discovery of essential genes and synthetic-lethal interactions. Although the Cancer Dependency Map profiled 89 OSCC/HNSCC lines to nominate baseline dependencies, drug-perturbed states critical for understanding platinum resistance remain underexplored. Only a handful of HNSCC studies have applied genome-wide CRISPR cas9 screening: two mapped core essential genes; two mapped cisplatin resistance and radiation resistance; and others uncovered synthetic-lethal targets, including vulnerabilities to mTOR inhibition, EGFR inhibition, glutamine metabolism inhibition, and host determinants of oncolytic HSV-1 efficacy. This review synthesizes these findings, highlights methodological considerations (library design, coverage, and treatment duration), and integrates complementary functional data to prioritize targets for rational combinations. This review also provides information on the TCGA database and in vivo CRISPR screening that can accelerate precision therapeutics for patients with HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31c18994958d899b77bdbb4cbe1e355e7819ca3" target='_blank'>
              Target Discovery in Head-and-Neck Squamous Cell Carcinoma: Genome-Wide CRISPR Screens Illuminate Therapeutic Resistance and Actionable Dependencies
              </a>
            </td>
          <td>
            V. K. Vincent-Chong
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) occurs as a complex, multifaceted event driven by the interplay of tumor-permissive genetic mutations, the nature of the cellular origin, and microenvironmental stress. In this study, using primary human pancreatic acinar 3D organoids, we performed a CRISPR-KO screen targeting 199 potential tumor suppressors curated from clinical PDAC samples. Our data revealed significant enrichment of a list of candidate genes, with neurofibromatosis type 2 associated gene (NF2) emerging as the top target. Functional validation confirmed that loss of NF2 promoted the transition of PDAC to an invasive state, potentially through extracellular matrix modulation. NF2 inactivation was found to enhance PDAC cell fitness under nutrient starvation. This adaptation not only reinforced the oncogenic state but also conferred therapeutic resistance. Additionally, we found that NF2 loss was associated with fibroblast heterogeneity and cancer-stroma communication in tumor evolution. These findings establish NF2 as a critical tumor suppressor in PDAC and uncover its role in mediating nutrient adaptation and drug resistance. Importantly, this study provides additional insights into drug resistance mechanisms and potential therapeutic targets in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c5d227985bf39716bd4f69d5a354d37cc46442" target='_blank'>
              NF2 loss malignantly transforms human pancreatic acinar cells and enhances cell fitness under environmental stress
              </a>
            </td>
          <td>
            Yi Xu, Michael H. Nipper, Angel A. Dominguez, Chenhui He, Francis E Sharkey, Sajid Khan, Han Xu, Daohong Zhou, Lei Zheng, Yu Luan, Jun Liu, Pei Wang
          </td>
          <td>2026-01-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Gastric cancer (GC) exhibits molecular heterogeneity and diverse immune cell infiltration patterns closely associated with patient prognosis. However, a comprehensive understanding of the variations in immune cell phenotypes among different patient subgroups still needs to be improved. In this study, we performed a detailed analysis of the tumor microenvironment in GC by integrating 200,466 single cells from 72 patients across six datasets. We classified patients into immune-deserted, B, T, and myeloid cell subtypes. Using genomic and clinical data from TCGA samples, we identified cellular components associated with tumor histology and genotypes. GC patients were stratified into immune-deserted, B cell, T cell, and myeloid cell subtypes, and we described the pathway and transcription factor activity characteristics of different microenvironment subtypes. Integration of bulk RNA-seq data reveals that fibroblasts and endothelial cells were associated with adverse patient outcomes whereas NK and T cells were notably correlated with improved prognosis. Subsequently, we focused on characterizing cancer-associated fibroblasts (CAFs) and discovered that they acquire new functional properties within the tissue microenvironment, providing evidence of CAF plasticity. We constructed a novel four-gene CAF signature including SPARC, EFEMP1, RGS5 and SERPINE1 which may enhance patient stratification and prognostic prediction of GC patients. qPCR analysis revealed that the significant expressions of SPARC, EFEMP1, RGS5 and SERPINE1 were significantly upregulated in gastric cancer tissues compared to the normal tissues. Our study provides insights into the composition of the tumor microenvironment and construction of a four-gene CAF signature associated with clinical prognosis, offering new perspectives for the clinical management of gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6b22dc4e8c3bd98a01915b46ab3333a465c3c7e" target='_blank'>
              Integrative single-cell atlas unveils heterogeneity and prognostic value of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Ziyu Jiang, Xiaomei Zhuang, Junkui Guo, Minyi Zhu, Chunhong Hong, Chunhui Sun, Kaiming Wu, Haofan Yin, Cuncan Deng, P. Jiang
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/517f57f9e6c335d60cb5a3e0fde4a76539819f12" target='_blank'>
              A Mechanophenotyping chip for high-throughput detection of metastatic bacteria-infected circulating tumor cells.
              </a>
            </td>
          <td>
            Wenxin Luo, Yanfeng Gao, Shujun Feng, Bangshun He, Jingjing Li, Jingjing Yang, Yi Yin, Meng Wang, Bin Xue, Yi Cao, Tony Y Hu, Yujun Song
          </td>
          <td>2026-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="In the tumor microenvironment, malignant cells coexist and interact with each other and with stromal, immune, and endothelial cells, as well as with extracellular matrix proteins. The interaction occurs through membrane contact or the production of multiple soluble factors. The composition of tumor and matrix cells changes continuously during tumor development, along with the infiltration of immune cells, forming heterogeneous niches that vary in space and time. We integrate current knowledge about the complex interaction between heterogeneous cell populations in the TME and the impact of these networks in supporting immune defense, which paradoxically promotes tumor progression. We summarize the involvement of immune cells and highlight the impact of certain homeostatic processes mediated by stromal cell populations and matrix components on tumor development. We propose the role of metabolic reprogramming and oxidative stress, as well as extracellular vesicle-mediated signaling, in conferring tumor resistance and therapeutic strategies to disrupt pro-tumor communication networks while enhancing anti-tumor immunity. Our goal is to provide a comprehensive framework for understanding and addressing the cellular interactions underlying cancer progression, fostering opportunities to formulate strategies that control tumor growth and eliminate resistance to treatment options. This integrative perspective provides a basis for designing multi-targeted immunotherapies aimed at rewiring pro-tumor communication networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686d41fdaf34500bc4a6bfe87c30a5182c7dc83f" target='_blank'>
              The tumor microenvironment: adding pieces to the puzzle
              </a>
            </td>
          <td>
            D. Aguilar-Cázares, Mario Perez-Medina, Jesus J Benito-Lopez, M. Galicia-Velasco, M. Meneses-Flores, Angel Camarena, J. Lopez-Gonzalez
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d749d1664733485d97fc50139d1df5bec7e20cae" target='_blank'>
              Reconstructing the metastatic journey: functional circulating tumor cells and disseminated tumor cells based models for translational oncology.
              </a>
            </td>
          <td>
            Caroline Sanglier, L. Cayrefourcq, C. Alix‐Panabières
          </td>
          <td>2026-01-14</td>
          <td>Journal of experimental & clinical cancer research : CR</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Spatial transcriptomics, which deals with tissue architecture in genetic investigation, is an innovative technique for examining cell heterogeneity and tissue organization. This review emphasizes major approaches, include spatially resolved transcriptome methods, immunohistochemistry as well as in situ hybridization, all of which permit the mapping of RNA molecules in their native tissue environment. These methods have proven essential in achieving our understanding of biological events such as tumor evolution, progression of cancer, and cancer tumor stem cell detection. Spatial transcriptomics, the study of patterns of gene expression in space, reveals the intricate nature of the tumor microenvironment (TME) and its effect on cancer biology. Although it delivers insight on the cellular connections that underlie disease, the significance of spatial transcriptomics in multiple organs has expanded.

Although its immense potential, there are still difficulties to be conquered, particularly within the areas of analysis of data, spatial resolution, and integration with other omics data. To be able to fully comprehend the complexities of tissues biology and ailments, this review additionally tackles future potential avenues, including the necessity for greater multiplexing, enhanced resolution, and the combination of functional genomics. With this synthesis, we intend provide an extensive summary of the state of spatial transcriptomics currently and demonstrate that it possesses the potential to improve precision medicine, cancer research, and our understanding of broader biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579b9e172c2769b4ad99d0da42fa24b04f1ee980" target='_blank'>
              Spatial Transcriptomics in Cancer Research: Methods and Applications
              </a>
            </td>
          <td>
            P. K. Kumar, K. M. Rao, Syed Mohiuddin Quadri, Bagadam Aparna, V. U. M. rao
          </td>
          <td>2025-12-18</td>
          <td>International Journal of Current Science Research and Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High-throughput"pathomic"analysis of Whole Slide Images (WSIs) offers new opportunities to study tissue characteristics and for biomarker discovery. However, the clinical relevance of the tissue characteristics at the micro- and macro-environment level is limited by the lack of tools that facilitate the measurement of the spatial interaction of individual structure characteristics and their association with clinical parameters. To address these challenges, we introduce HistoWAS (Histology-Wide Association Study), a computational framework designed to link tissue spatial organization to clinical outcomes. Specifically, HistoWAS implements (1) a feature space that augments conventional metrics with 30 topological and spatial features, adapted from Geographic Information Systems (GIS) point pattern analysis, to quantify tissue micro-architecture; and (2) an association study engine, inspired by Phenome-Wide Association Studies (PheWAS), that performs mass univariate regression for each feature with statistical correction. As a proof of concept, we applied HistoWAS to analyze a total of 102 features (72 conventional object-level features and our 30 spatial features) using 385 PAS-stained WSIs from 206 participants in the Kidney Precision Medicine Project (KPMP). The code and data have been released to https://github.com/hrlblab/histoWAS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407dd7f4b75161f7922e517e4737dcf219aed8fa" target='_blank'>
              HistoWAS: A Pathomics Framework for Large-Scale Feature-Wide Association Studies of Tissue Topology and Patient Outcomes
              </a>
            </td>
          <td>
            Yuechen Yang, Junlin Guo, Yanfan Zhu, Jialin Yue, Junchao Zhu, Yu Wang, Shilin Zhao, Haichun Yang, Xingyi Guo, Jovan Tanevski, Laura Barisoni, Avi Rosenberg, Yuankai Huo
          </td>
          <td>2025-12-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell spatial technologies have emerged in recent years, enabling characterization of tissue architecture and organization at unprecedented resolution. However, computational analysis of spatial transcriptomics data remains a bottleneck for scientific discoveries in the absence of dedicated bioinformatics expertise. Here, we describe a novel cell area normalization method and workflow to annotate 15 kidney cell types from a dataset generated using NanoString's CosMx single-cell-resolution spatial transcriptomic platform. This approach enabled a comparison between two healthy kidney biopsies and two diseased samples. We validated our pipeline's accuracy using gene expression analysis, demonstrating increased sensitivity compared with other normalization methods and consistency with pathological changes observed in biopsy-proven diabetic kidney disease (DKD). Using precise cell type annotation, we observed significant changes in the proportions of podocytes and immune cells in DKD, with regional enrichment of immune cells and differential gene expression. Injured proximal tubules showed the expected upregulation of HAVCR1 and VCAM1, as well as other genes associated with diabetes, including IL18, ITGA3, and ITGB1. The workflow, now fully integrated into the BioTuring SpatialX (Lens V2.0), is available as a platform designed for users with no formal bioinformatics training, providing accessible web-based tools for spatial data analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80e5709aa7f84bf6d5dd8e0a9ac394d2bcf504a4" target='_blank'>
              Streamlining single-cell spatial transcriptomics for human kidney tissue.
              </a>
            </td>
          <td>
            Son Vo, Kieran Meadows, Han Do, Kevin R. Chapman, Graciela Andonegui, Daniel A Muruve, Thang Pham, Justin Chun
          </td>
          <td>2025-12-01</td>
          <td>BioTechniques</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Diffuse large B-cell lymphoma (DLBCL) is an aggressive cancer with profound heterogeneity. While standard immunochemotherapy is curative for the majority, treatment failure in 30–40% of patients due to refractory or relapsed disease remains a significant clinical challenge, emphasizing the need for better prognostic markers. We investigated the interplay between tumor glycolytic metabolism and the tumor immune microenvironment. We identified four distinct glycolysis-related patterns that were associated with three immune subtypes (known as immune-inflamed, immune-excluded, and immune-desert). Importantly, these patterns stratified clinical outcomes and predicted chemotherapy sensitivity. For example, tumors with an immune-inflamed subtype were more sensitive to doxorubicin. Collectively, our findings propose a metabolism- and immune-based classification framework for DLBCL that may facilitate more precise risk stratification and treatment optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e894f2d6a04501c694424859815514794b12befc" target='_blank'>
              Glycolysis-Driven Immune Subtypes in the Tumor Microenvironment Determine Clinical Outcomes in Diffuse Large B-Cell Lymphoma
              </a>
            </td>
          <td>
            Ying Du, Zhenyu Liu, Chuanyan Liu, Yining Yuan, Mengkun Fang, Tong Zhou, Mengge Pan, Weilu Xu, Xu Liu, Peipei Xu
          </td>
          <td>2025-12-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Computational pathology has emerged as an attractive option for improving risk stratification in prostate cancer (PCa), but current approaches either lack interpretability or focus solely on tumor morphology, potentially limiting their utility. We aimed to identify an interpretable, immune microenvironment-derived computational pathology biomarker for PCa.



 We retrospectively identified two independent cohorts (Discovery and Validation) with localized PCa who underwent radical prostatectomy (RP) and had digitized H&E-stained whole-slides images (WSIs) and longitudinal outcomes data. We identified a third cohort from The Cancer Genome Atlas (TCGA) with localized PCa which was treated with RP and had WSIs, bulk RNA sequencing, and whole-exome sequencing. Immune cells were identified from WSIs using a publicly available deep learning model (CellViT), and spatially dense immune clusters were quantified using density-based spatial clustering of applications with noise (DBSCAN). CIBERSORTx was utilized for immune cell deconvolution and TRUST4 for immune receptor repertoire reconstruction from bulk RNA sequencing. Cox proportional hazard regression was used to examine associations between clinicopathologic features and the primary outcome of time to distant metastasis (DM), as well as secondary outcomes of biochemical recurrence (BCR) and overall survival (OS).



 Median follow-up periods for the Discovery (n=272) and Validation (n=218) Cohorts were 12.6 and 8.1 years, respectively. In the Discovery Cohort, median age was 63, 14% had Gleason 8-10 disease, 94% had pT2-T3a disease, and median prostate-specific antigen level at diagnosis was 6.2 ng/mL. Median immune cell proportion was 4.3% (interquartile range 3.0-5.9%), and median immune cluster density was 2.4 per 400 mm2 of tissue (interquartile range 0-7.1). Similar values were observed in the Validation Cohort. In both cohorts, immune cell proportion was not associated with BCR, DM, or OS (P>=0.09). In the Discovery Cohort, while increasing immune cluster was not associated with BCR (P>=0.10), it was independently associated with a decreased risk of DM for Gleason 8-10 (adjusted hazard ratio [AHR] 0.42, 95% confidence interval [CI] 0.19-0.93) but not Gleason 6-7 patients (AHR 1.26, 95% CI 0.77-2.05; Pint=0.020). Similar results were observed in the Validation Cohort, where increasing immune cluster was not associated with BCR (P>=0.10) but was associated with a lower risk of DM in Gleason 8-10 disease (AHR 0.60, 95% CI 0.37-0.98), though not Gleason 6-7 (AHR 1.19, 95% CI 0.74-1.91; Pint=0.043). In TCGA Cohort (n=326), immune cluster was not associated with somatic alterations. For Gleason 8-10 but not Gleason 6-7 disease, high-cluster samples were enriched for CD8+ T cells, activated memory CD4+ T cells, and Tregs (P<=0.037), as well as clonal T cell populations (P<=0.039).



 Our findings nominate immune spatial clustering as a novel, interpretable computational pathology biomarker and provide insights into the unique immune features of high-grade PCa.



 David D. Yang, Aya Abdelnaser, Alexander J. Haas, Jeremiah Wala, Alfred A. Barney, Eddy Saad, Jett P. Crowdis, Cora A. Ricker, Seifeldin Awad, Bora Gurel, Jihye Park, Martin T. King, Paul L. Nguyen, Toni K. Choueiri, David J. Einstein, Steven P. Balk, Alok K. Tewari, Johann S. de Bono6, Keyan Salari, Mary-Ellen Taplin, Chin-Lee Wu, Eliezer M. Van Allen. . Immune spatial organization predicts metastasis risk in aggressive localized prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aace1806fe7d01338e5e9857c054699a561213e" target='_blank'>
              Abstract PR001: Immune spatial organization predicts metastasis risk in aggressive localized prostate cancer
              </a>
            </td>
          <td>
            , Aya Abdelnaser, Alexander J. Haas, Jeremiah Wala, Alfred A. Barney, Eddy Saad, Jett Crowdis, Cora A. Ricker, Seifeldin Awad, B. Gurel, Jihye Park, Martin T. King, , T. Choueiri, David J Einstein, Steven P. Balk, A. Tewari, J. D. De Bono, Keyan Salari, M. Taplin, Chin-Lee Wu, E. V. Van Allen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive brain tumor that inevitably recurs after chemoradiotherapy, resulting in poor patient outcomes. Given GBM’s cellular heterogeneity, we hypothesized that radiation induces sub-population specific alterations to survive and adapt to radiation stress. We performed integrated single-cell RNA-seq and ATAC-seq analyses in Glioma Stem-Like Cultures three hours after they were exposed to clinically relevant radiation doses. Radiation reshaped the cellular landscape, altering both cell type composition and GBM subtype distribution. Cluster-specific and shared transcriptional programs were induced in a dose-dependent manner, with differentially expressed genes enriching distinct biological pathways. Chromatin accessibility analyses revealed parallel cluster-specific remodeling, with both opening and closing of regulatory elements linked to functional pathway shifts. Notably, 2 Gy and 6 Gy exposures elicited conserved transcriptional profiles in RNA clusters across different radiation doses. Together, these results highlight immediate radiation-induced transcriptional and chromatin remodeling programs in GBM at single-cell resolution and identify conserved cluster-specific adaptations that may underlie therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4393b9168f5ebc4a80f332a0eba92fab9bfeef3" target='_blank'>
              Single-Cell Multiomic Profiling Uncovers Radiation Dosage-Sensitive, Cluster-Specific Regulatory Dynamics in Glioblastoma
              </a>
            </td>
          <td>
            Khoi Huynh, Sara Nalina Barcik Weissman, Cheol Park, Maria L.G. Quiloan, Blake Sun Chang, Eric L Chang, Kelly Street, David Tran, A. Modrek
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary This review was conducted to address the complexities of breast cancer research, which is often limited by two-dimensional cell cultures that fail to accurately represent the tumor microenvironment. Although three-dimensional (3D) models are generally superior, the absence of standardized methodologies has resulted in inconsistent findings. The objective of this review was to systematically analyze how different 3D culture methods influence the epigenetic reprogramming of tumor cells, focusing on changes in DNA methylation, histone modifications, and chromatin organization, as well as the differential expression of various types of non-coding RNA. Our findings demonstrate that different 3D culture methods are not equivalent and yield distinct epigenetic signatures. This underscores the urgent need to standardize protocols and provide detailed reports on the properties of culture media to enhance reproducibility, which is crucial for identifying new therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/933fb504bd5970ab97b7de6c042e05e852c76571" target='_blank'>
              Epigenetic and Transcriptional Reprogramming in 3D Culture Models in Breast Cancer
              </a>
            </td>
          <td>
            Laura Cecilia Flores-García, Karla Rubio, Eloísa Ibarra-Sierra, M. B. Silva-Cázares, C. Palma-Flores, C. López-Camarillo
          </td>
          <td>2025-11-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9d84cab81cd323269c1a623f592cc3353c4b7c" target='_blank'>
              Organoid Pharmacotyping of Pancreatic Cancer Enables Functional Precision Oncology and Drug Repurposing
              </a>
            </td>
          <td>
            He Dong, F. Vizeacoumar, Yue Zhang, Nicholas Jette, J. Price, Vincent Maranda, Lihui Gong, T. Freywald, J. P. Vizeacoumar, M. Lazell-Wright, Saruul Uuganbayar, Jaylynn Quail, Alain Morejon Morales, Ayaan Rizvi, Rani Kanthan, Yuliang Wu, Anand Krishnan, Kathleen Felton, Bilal Marwa, Laura Hopkins, Gary Groot, J. Shaw, Gavin Beck, Yigang Luo, Maurice Ogaick, Mike Moser, A. Freywald, Shahid Ahmed, A. Zaidi, Franco J Vizeacoumar
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The field effect describes the phenomena where environmental exposures, infection, and genetic predisposition result in molecular changes in cells that predispose them to developing cancer. Though this is a well-established concept in pathology, it remains underexplored in the context of high-resolution omics. We utilized the Diagnostic Evidence Gauge of Single Cells (DEGAS) deep transfer learning framework to analyze prostate cancer spatial transcriptomics to identify cells and tissues that are highly associated with cancer progression. DEGAS highlighted morphologically benign glands that had reduced expression of MSMB, a differentiation marker that is decreased in aggressive tumors. These glands have upregulated genes associated with antigen presentation and aggressive neoplasms. Integration of single-cell transcriptomics and deep learning image analysis separately revealed altered immune-cell infiltration, suggesting a complex interplay in the tumor environment facilitating aggressiveness. We used immunohistochemistry to quantify the MSMB protein (PSP-94) expression on morphologically normal and tumor tissues from patients with and without 5-year distant metastasis. Samples from patients who developed metastasis consistently showed lower fractions of positively stained cells, indicating a subtle yet significant "field effect" in seemingly benign regions. These proteomic results validate the transcriptomic findings and further underscore that inflammatory or immune-related changes in ostensibly normal tissue may contribute to aggressive disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68d7d78a2fef9048f9bcafbf03164f9002e0042" target='_blank'>
              Deep Transfer Learning Links Benign Glands to Prostate Cancer Progression via Transcriptomics.
              </a>
            </td>
          <td>
            Justin L Couetil, Ziyuan Liu, Chao Chen, A. Alomari, Kun Huang, Jie Zhang, Travis S. Johnson
          </td>
          <td>2025-11-29</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progression involves dynamic remodeling of the tumor microenvironment (TME). However, the stage-specific dynamics of immune and stromal cell remodeling throughout LUAD progression remain incompletely understood. Here, the study systematically profiles the cellular composition and transcriptional states across multiple LUAD stages, integrating early-stage patient specimens with publicly available datasets encompassing advanced-stage disease. The analysis reveals a marked stage-dependent shift from a proliferative and immune-activated microenvironment in early LUAD to a hypoxia-enriched and immunosuppressive landscape in advanced disease. A distinct hypoxia-adapted epithelial tumor cell subpopulation (C5), exhibiting transcriptional features of metastasis, invasion, and hypoxia, and poor prognosis, is identified. Advanced LUAD featured immunosuppressive LGMN⁺ macrophages and STAT1-driven exhausted CD8⁺ T cells. FKBP11⁺ plasma B cells exhibited exhaustion-linked metabolic changes. POSTN⁺ CAFs emerged as central mediators of extracellular matrix (ECM) remodeling and immune exclusion. Collectively, the findings reveal a model of hypoxia-driven functional convergence, in which distinct TME components co-evolve toward phenotypes that collectively promote immune evasion, matrix remodeling, and tumor progression. These findings may provide insights into stage-specific cellular dynamics and highlight promising therapeutic targets for precision immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8876e01109cf8f024442bf59e35e9ed246f97f" target='_blank'>
              Characterizing Stage-Specific Cellular Dynamics and Microenvironmental Remodeling in Lung Adenocarcinoma by Single-Cell RNA Sequencing.
              </a>
            </td>
          <td>
            Bomiao Qing, Xiaolan Li, Xiang He, Junyi Wang, Yi Yang, Manling Jiang, Bingbing Yan, Lei Zhang, Anying Xiong, Q. Ran, Guoping Li
          </td>
          <td>2025-11-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Precision medicine has evolved through distinct phases, from the origins of the Human Genome Project to mutation‐based targeted therapies. This editorial posits that “stereological cell biomedicine” could be a new approach promoting the development of the next generation of precision medicine. This emerging discipline transitions the focus from genomic data to the multi‐dimensional and spatiotemporal complexity of single cells. Driven by advances in Stereo single‐cell multi‐omics (Stereo Cell‐seq), spatial transcriptomics (Stereo‐seq), and single‐cell surfaceomics (sc‐surfaceome), this approach aims to capture the stereologically dynamic interactions between organelles within a cell and between cells in the tissue. We argue that understanding the spatiotemporal location of molecules, particularly protein interactions at organelle interfaces and on the cell surface, is as critical as their abundance for defining cellular function in health and disease. Integrating these high‐resolution measurements with artificial intelligence and computational modelling will bridge the gap between advanced omics and pathology. Initiatives such as the newly established European Stereo Cell Center (ESCC) signal a global shift towards this new paradigm, which promises to unlock novel diagnostic biomarkers and therapeutic targets for truly multi‐factorial and dynamic precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acdcd69ccf472862bb486c6c2a16a93191b0f96" target='_blank'>
              Stereo cell: A new approach to the next generation of clinical precision medicine
              </a>
            </td>
          <td>
            Wanxin Duan, Mingjie Wang, Yifei Liu, Celine Desoyer, Christian Baumgartner, Xiangdong Wang
          </td>
          <td>2025-11-28</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Triple Negative Breast Cancer (TNBC) is an aggressive subtype of breast cancer with limited treatment options and poor prognosis. Coupled with this, TNBC shows a high rate of metastasis which is known to be aided by tumour vasculature. Endothelial cells that form the lining of the tumour vasculature exhibit distinct genotype and phenotype differences compared with normal tissue vasculature. However, little is known about endothelial signatures that drive metastasis from a primary tumour, particularly in the context of immune infiltration. In this study, we utilized GeoMX Digital Spatial Profiling to investigate spatial proteomics differences in endothelial cells between primary and secondary sites of TNBC. By segmenting tissues using epithelial (PanCK), immune (CD45), and endothelial (CD31) markers, we analysed the microvasculature for a panel of 79 target proteins. In paired primary and secondary TNBC tissues, we identified significant downregulation of fibronectin (− log2(Fold-Change) = − 1.7, p < 0.001) in secondary sites. Specifically in epithelial regions, S100B was found to be downregulated in secondary microvasculature when compared to primary tumours. Additionally, metastasis-free primary tissues exhibited upregulated expression of S100B when compared to primary tissues that metastasized. Our study highlights the potential contributions of microvasculature to metastatic progression in TNBC, presenting new opportunities to explore them as potential biomarkers of TNBC metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9355568219b2a2ea616006baa27f93ab8c621b2" target='_blank'>
              Digital spatial profiling identifies features of primary and locoregional metastatic vasculature in triple negative breast cancer
              </a>
            </td>
          <td>
            Akhilandeshwari Ravichandran, Kyle Upton, Shiva Taheri, Cheng Liu, Kaltin Ferguson, Mark N. Adams, Laura J. Bray
          </td>
          <td>2026-01-19</td>
          <td>Clinical & Experimental Metastasis</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ade8a8b9df4408ea89e61a0e1bd36268d586bf5" target='_blank'>
              Decoding osteosarcoma from heterogeneity to precision therapy.
              </a>
            </td>
          <td>
            Yu Xu, Yanfu Bai, Fan Yang, Binbin Yang, Zhisheng Huang, Lingling Su, Kuntao Peng
          </td>
          <td>2025-12-11</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Prostate cancer commonly presents as multifocal disease, with distinct tumours that are often genetically independent. Metastases, however, typically derive from a single dominant clone, highlighting the clinical challenge of identifying which lesion drives progression. Focal therapy, an emerging treatment strategy, aims to target only the most aggressive tumour focus or foci, sparing normal prostate structures and reducing morbidity. Yet, the lack of consistent genomic drivers in primary tumours complicates efforts to identify the focus most likely to progress. While prostate cancers are known to harbour widespread methylation alterations, the extent to which these epigenetic changes track with clonal origin remains poorly understood. To address this, we performed multiregional epigenomic sequencing of 189 prostate tissue samples, including 109 tumour foci and 80 non-tumour regions from 44 patients (2-8 tumour samples per case). Differential methylation analysis revealed widespread alterations distinguishing benign from tumour tissue, including a core ‘epigenetic trunk’ of >20,000 recurrently altered CpGs shared across cases. DNA variants called from non-methylation sites of the epigenomic sequencing closely matched variants from WGS in cases with WGS available. Phylogenetic trees constructed from DNA variants and phyloepigenetic trees from methylation sequencing revealed striking patterns. Some cases lacked shared DNA variants across all tumour foci, indicating a polyclonal tumour origin, yet these genetically unrelated foci shared extensive methylation alterations. This suggests widespread methylation convergence occurs in clonally distinct tumour foci. To dissect clonal relationships further, we applied two orthogonal lineage tracing approaches using the methylation sequencing data: fluctuating CpGs (sites that change stochastically over time) and highly entropic 8-mer ‘methylation barcodes’ within the protocadherin gene cluster. Fluctuating CpGs showed concordant states across tumours that shared DNA sequence variants, while the methylation barcodes likewise revealed high similarity among such tumours. Together, these methods support the independent evolutionary origins of polyclonal prostate tumour foci. Pathway analysis of recurrently altered methylation loci highlighted consistent enrichment in epithelial–mesenchymal transition, MYC signalling, DNA damage response, and hormonal signalling across cases, suggesting functional convergence at the pathway level despite divergent genetic origins, consistent with selective pressures acting on shared programmes. Together, these findings demonstrate that DNA methylation alterations represent fundamental and recurrent events in prostate cancer, more consistent than DNA sequence variants across multifocal disease. Lineage-tracing analyses further highlight that methylation processes record tumour evolutionary history, encoding both clonal relationships and convergent biology. These observations may inform methylation-based biomarkers and the rational selection of lesions for focal therapy.



 Tamsin J. Robb, Melissa Cheung, Rajbir Batra, Henson Lee Yu, John C. Thomas, Anne Warren, Andrew Lynch, Daniel Brewer, David Wedge, CRUK-ICGC Prostate Cancer Group, Charles Massie, Harveer Dev. Widespread methylation convergence in clonally distinct foci of multifocal prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90717932598b7c081fea25f61e12e100d9757059" target='_blank'>
              Abstract B003: Widespread methylation convergence in clonally distinct foci of multifocal prostate cancer
              </a>
            </td>
          <td>
            Tamsin J. Robb, Melissa Cheung, Rajbir Batra, Henson L. Lee Yu, John C. Thomas, Anne Y Warren, Andrew Lynch, Daniel Brewer, D. Wedge, Charles Massie, H. Dev
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Despite substantial progress in the field of bladder cancer management, the disease continues to represent an important health issue characterized by increased recurrence and progression rates. This is largely attributed to cancer stem cells (CSCs), a unique cell subpopulation capable of self-renewal, differentiation and resistance to conventional anti-cancer therapies. At the same time, our understanding of cancer biology has been revolutionized by the identification of non-coding RNAs (ncRNAs), a heterogeneous group of RNA molecules that do not translate into proteins yet function as pivotal regulators of gene expression. Emerging evidence demonstrates that ncRNAs modulate key hallmarks of CSCs, including self-renewal, epithelial–mesenchymal transition and drug resistance. This review investigates the intricate interplay between ncRNAs and the core signaling pathways that underlie bladder CSC biology. Unravelling the nexus between CSCs and ncRNAs is crucial for developing novel diagnostic biomarkers, better prognostic tools and innovative therapeutic strategies for patients with bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13091e7103acf49fcdc024b48e3b7a3f0f4f481d" target='_blank'>
              The Crossroads of Cancer Regulation: Discussing the Role of Non-Coding RNAs in Bladder Cancer Stem Cells
              </a>
            </td>
          <td>
            Alexandros Georgiou, Dimitrios Triantis, M. Goulielmaki, V. Zoumpourlis
          </td>
          <td>2025-12-11</td>
          <td>Uro</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays an important role in the development, progression, and treatment response of pediatric cancers, yet remains less elucidated compared to adult malignancies. Pediatric tumors are unique with a low mutational burden, an immature immune landscape, and unique stromal interactions. The resultant “cold” immune microenvironments limits the effectiveness of conventional immunotherapies. This review summarizes the key cellular and non-cellular components of the pediatric TME—including T cells, NK cells, tumor-associated macrophages, cancer-associated fibroblasts, extracellular matrix remodeling, angiogenesis, and hypoxia—and describes how these elements shape tumor behavior and therapy resistance. The role of TME in common pediatric cancers like leukemia, lymphoma, neuroblastoma, brain tumors, renal tumors, and sarcomas is discussed. Emerging therapeutic strategies targeting immune checkpoints, macrophage polarization, angiogenic pathways, and stromal barriers are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34cc7bfcda14da44bca02f277e5096c55bbdde0" target='_blank'>
              Tumor Microenvironment: Current Understanding and Therapeutic Implications in Adult and Pediatric Cancers
              </a>
            </td>
          <td>
            Satyendra Batra, Prashant Prabhakar, Debabrata Mohapatra, Noreen Grace George, Neha Goel, Bhavika Rishi, Aroonima Misra, Amitabh Singh
          </td>
          <td>2025-12-25</td>
          <td>Onco</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3083560b7513f7f17aefe85af32208b0613ecab1" target='_blank'>
              Cellular neighborhoods in cancer.
              </a>
            </td>
          <td>
            Lichun Ma, Barbara Xiong, Meng Liu, Kai Tan
          </td>
          <td>2026-01-16</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a self-renewing population capable of fueling long-term tumor growth. In gastric cancer, the identity of CSC populations remains unclear. In this study, we established a gastric CSC population marked by the water channel protein aquaporin-5 (AQP5), which resides in human and mouse pyloric tumors. Using multiple organoid and mouse models, we found a requirement for AQP5+ CSCs in both initiating and sustaining cancer progression and demonstrated that AQP5 expression also directly promotes tumor growth and invasion in a WNT, PI3K (phosphatidylinositol 3-kinase), and MAPK (mitogen-activated protein kinase)-dependent manner. Beyond primary cancers, AQP5 further enriches for a functional CSC population in metastatic tumors. Together, our findings support a CSC model in gastric tumors that may have application for therapeutic strategies targeting CSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b857f221b4120247f60ae689da7ba8ea312dc26b" target='_blank'>
              AQP5: A functional gastric cancer stem cell marker in mouse and human tumors.
              </a>
            </td>
          <td>
            H. Lim, Swathi Yada, Kazuhiro Murakami, B. Lee, Sowmya Sagiraju, Phyllis Phuah, Tanysha Chi-Ying Chen, Fidelia B Alvina, S. Tan, Kaushal Krishna Kaslikar, Nur Syahirah Binte Ruhazat, Menaka Priyadharsani Rajapakse, Katzrin Bte Ahmed Murad, Snezhina Kancheva, Liang Thing Tan, Seri Mustafah, Jimmy Bok Yan So, Nick Barker
          </td>
          <td>2025-12-18</td>
          <td>Science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0571bcac3b832c1f070aaad0c8312d4e4819ffc8" target='_blank'>
              TCF1lo CD8 T cells proliferate and persist autonomously in tumors
              </a>
            </td>
          <td>
            Megan M. Erwin, Natalie R. Favret, Claudia N. McDavid, Zachary D. Ewell, Melissa M. Wolf, , Jessica J. Roetman, Michael W. Rudloff, Kristen A Murray, Carlos R Detres Roman, Mary Philip
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Bladder cancer (BCA) shows significant prognostic differences between non-muscle-invasive (NMIBC) and muscle-invasive (MIBC) forms. While NMIBC frequently recurs and can progress to invasive disease, reliable biomarkers to monitor this transition are lacking. Extracellular matrix (ECM) remodeling is a critical factor influencing tumor aggressiveness, yet the key regulators of ECM changes across BCA stages remain unclear. In this study, we investigate the role of COL1A2 in ECM-related tumor biology and its potential as a prognostic biomarker for BCA progression. Methods We utilized a multi-step bioinformatics pipeline, analyzing RNA-seq data from TCGA and GEO datasets to identify molecular differences between NMIBC and MIBC. Prognostic markers were prioritized via differential expression analysis, Cox regression, and Kaplan–Meier survival analysis. The regulatory network was explored using protein-protein interaction analysis, and ECM-related activity was quantified through ssGSEA. Cell-type-specific insights were gained through single-cell RNA-seq analysis, and intercellular communication was deciphered using CellChat. Functional validation was performed through in vitro knockdown experiments in BCA cell lines. Results COL1A2 emerged as a key prognostic ECM-related gene associated with MIBC. Single-cell RNA-seq analysis revealed that COL1A2 and ECM components were predominantly enriched in matrix cancer-associated fibroblasts (CAFs), with PTK2 (FAK, focal adhesion kinase) upregulated in epithelial cells undergoing epithelial-mesenchymal transition (EMT). CellChat analysis uncovered a dominant COL1A2-mediated signaling axis from matrix CAFs to EMT epithelial cells via COL1A1/2–SDC1/4 ligand-receptor interactions. Functional assays confirmed that COL1A2 knockdown significantly impaired MIBC cell invasion and migration by suppressing ECM remodeling and EMT. Conclusion Our results suggest that the COL1A2–ECM–FAK signaling axis plays a critical role in MIBC progression, and COL1A2 could serve as a potential biomarker and therapeutic target for muscle-invasive bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda2a0be0d617f715d6d80984a257431f3977b47" target='_blank'>
              Integrative bulk and single-cell transcriptomic analysis reveals COL1A2-driven ECM remodeling and focal adhesion signaling associated with the transition from non-muscle-invasive to muscle-invasive bladder cancer
              </a>
            </td>
          <td>
            Menglu Li, Xinwei Liu, Yuhui Xue, Yichen Lu, Zhiqiang Chen, Yuwei Zhang, Weiguo Chen, Shan-Chao Zhao, Ke Wang, Ninghan Feng
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Bone metastases are a common and devastating complication of advanced prostate cancer (PCa), with up to 80–90% of men with castration-resistant PCa (CRPC) developing metastases to the bone. The development of bone metastasis serves as a critical indicator of worse long-term clinical outcomes. Understanding the factors driving PCa metastasis to the bone, including through myeloid cell reprogramming within the tumor microenvironment (TME), is critical. Our study examined published single-cell RNA sequencing data from 7 PCa patients with bone metastasis and 7 benign donors, focusing on myeloid cells across tumor, involved marrow, distal marrow, and benign bone factions. We used manual annotation methods and scANVI for whole integrated data annotation. Myeloid cells were sub-classed and manually annotated for classical monocytes (Mono1), intermediate monocytes (Mono2), non-classical monocytes (Mono3), myeloid dendritic cells (mDCs), tumor-infiltrating monocytes (TIMs), tumor-associated macrophages (TAMs), and osteoclasts. We applied pseudotime and used CellRank's Random-Walk Simulation for trajectory inference to determine final terminal states for the myeloid cells, i.e., tumor-associated macrophages (TAMs), myeloid dendritic cells (mDCs), and osteoclasts. Ranked driver gene lists for every sample type (benign, distal, involved, tumor) and cell type (Mono1/2/3, TIM) that were associated with the final terminal states were generated. QIAGEN Ingenuity Pathway Analysis (IPA) and Gene Set Enrichment Analysis (GSEA) were performed on driver gene lists across sample types and cell types, thus identifying upstream regulators influencing terminal state formation. IPA and GSEA identified ATF3, a stress-response regulator, and NF-κB, a central inflammatory mediator, to be significantly associated with TAM differentiation driver genes and were shared across extremes of sample types (benign & tumor) and cell types (Mono1 & TIM). CellOracle's in-silico perturbations were performed to assess the significance of ATF3 in context of the PCa bone TME while visualizing impacts on cell development. Through siRNA suppression of ATF3 in-vitro, followed by flow cytometry and qPCR for knockdown validation, our results show reduced M2/TAM differentiation with decreased associated markers, suggesting a role for ATF3 in myeloid reprogramming that may contribute to TAM-associated immunosuppression and the metastatic disease state



 Azimullah Rifai, Safiullah Rifai, Zumar Meher, Tausif Khan, Wei Guang, Linbo Wang, Mohamad Khan, Arif Hussain. Rewiring the tumor microenvironment: Uncovering lineage drivers of immunosuppressive tumor-associated macrophages in prostate cancer bone metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b3000b262dc2818edcc8efcb35668c9389013e" target='_blank'>
              Abstract A019: Rewiring the tumor microenvironment: Uncovering lineage drivers of immunosuppressive tumor-associated macrophages in prostate cancer bone metastasis
              </a>
            </td>
          <td>
            Azimullah Rifai, Safiullah Rifai, Zumar Meher, Tausif Khan, Wei Guang, Linbo Wang, Mohamad Khan, Arif Hussain
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Epigenetic reprogramming is a major contributor to therapy resistance in prostate cancer (PCa). Indeed, chromatin accessibility profiling classified PCa into distinct subtypes, predictive of clinical outcome. Histone modifications influence nucleosome positioning, ultimately dictating chromatin compactness. The histone methyltransferase G9a, frequently upregulated in advanced PCa, catalyzes a repressive histone modification (di-methylation of histone H3 lysine 9) that signals for gene transcriptional repression. CM-272 is a reversible and highly selective inhibitor against G9a activity, and we have previously shown in cell line models of PCa that it depicted strong anti-tumoral effects. However, fundamental aspects of altered transcriptomic programs remain unexplored. Here, we validated, in vivo, the CM-272 effects in advanced PCa, and explored the molecular mechanisms underscoring G9a inhibition-mediated effects. Specifically, the PCa patient-derived xenograft model BM18 was used to assess the mode of action of CM-272 in vivo. Moreover, RNA sequencing of tumor tissues was performed to further explore the underlying gene expression signature. Finally, chromatin immunoprecipitation followed by qPCR was used to investigate the underlying functional consequence of epigenetic targeting. Our results showed that CM-272 significantly delayed BM18 tumor growth, specifically impairing tumor cell G9a activity and proliferation. Transcriptomic profiling revealed 119 significantly upregulated and 99 downregulated genes in response to G9a inhibition. Notably, by integrating our gene expression dataset with published chromatin accessibility data, an increase in the androgen receptor (AR) signature and a concomitant decrease in the neuroendocrine (NE) features were observed in CM-272-treated samples. Additionally, pathway enrichment and master regulator analysis pinpointed both the androgen receptor and TP53 networks as significantly upregulated in treated tissues. In line with the reduction of the NE score, the MYC/MYCN were the most significantly downregulated transcription factors. Moreover, under the androgen pathway, upregulation of AZGP1, STK39, and DPP4 was observed. At the promoter level, a significant reduction in H3K9me2 levels was detected in treated samples, allowing, in turn, an increase in AR binding. Additionally, upon treatment, higher p53 binding was detected at the promoter of the p53-repressed genes AURKA, CDC25B, and CD44, whose expression was downregulated. Overall, targeting G9a with CM-272 might hold promising therapeutic potential for the management of advanced disease by modulating relevant tumoral transcriptional programs associated with core vulnerabilities in PCa.



 Filipa Moreira-Silva, Mark Zimmerli, Panagiotis Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio. G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5911786a7c980e0c86fa5660d05b013ec96ea309" target='_blank'>
              Abstract A045: G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay
              </a>
            </td>
          <td>
            Filipa Moreira-Silva, Mark Zimmerli, P. Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer is known to be a disease of altered cellular signaling; however, the relationship between mutation-specific changes to signal transduction and the phenotypic consequences produced remains poorly understood. Here, we investigate two common breast cancer driver mutations, the PIK3CAH1047R mutation and the ErbB2 amplification, both of which activate the PI3K-Akt pathway but paradoxically drive distinct cellular outcomes. Indeed, in nontransformed mammary epithelial cells, PI3KH1047R expression induced features of epithelial-mesenchymal transition (EMT), while ErbB2amp cells exhibited a hyperproliferative phenotype. Characterization of PI3K axis signaling revealed that ErbB2amp cells display prolonged, stimulus-dependent PI3K activation, whereas PI3KH1047R cells show constitutive, ligand-independent signaling. To test whether these distinct dynamics contribute to the phenotypic responses, we employed an iLID-based optogenetic system that enables precise, tunable control of endogenous PI3K activity. Using this tool to mimic the mutation-specific dynamics in MCF10A mammary epithelial cells, we found that PI3K signaling patterns alone were sufficient to reproduce key features of the PIK3CA H1047R-associated EMT phenotype but not the ErbB2-associated proliferative phenotype. These findings suggest that the temporal encoding of pathway activity, not merely its magnitude, can drive some phenotypic changes in oncogenic progression, explain how distinct mutations within a common signaling pathway can produce divergent cellular phenotypes, and provide a workflow for interrogating the functional consequences of changes in signaling dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51c6978b325cbac3a14efcc2bf1ac41d165bf7ce" target='_blank'>
              Oncogenic Alterations in PI3K Signaling Emulated Optogenetically Recapitulate Some Phenotypic Changes in Mammary Epithelia.
              </a>
            </td>
          <td>
            Keith A. Gagnon, Veronica W Hui, Terry Ching, Amy E. Stoddard, Esther Koh, J. Eyckmans, Ahmad S. Khalil, Christopher S. Chen
          </td>
          <td>2026-01-19</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Despite advances in cancer therapies, treatment failure from resistance and recurrence remains a major clinical challenge. Therapy-induced senescence (TIS), a state of stable cell cycle arrest with sustained metabolic activity, has emerged as a driver of inflammation, tumor persistence, and relapse. However, the heterogeneity of TIS complicates its detection and targeting. Here, we applied a multi-modal strategy to characterize metabolic alterations in senescent cancer cells induced by doxorubicin or γ-irradiation across three tumor cell lines: MCF7, HeLa, and TPC-1. Mitochondrial dysfunction was assessed using MitoTracker and JC-1 staining, while two-photon excitation fluorescence (TPEF) microscopy enabled label-free visualization of intracellular NAD(P)H and FAD distribution. Lipid remodeling was evaluated by MALDI mass spectrometry imaging, and RNA sequencing was performed on control, senescent, and engulfing-senescent MCF7 cells to identify differentially expressed genes and enriched pathways. Senescent cells displayed mitochondrial dysfunction, with altered NAD(P)H/FAD distribution and decreased membrane potential. TPEF confirmed redistribution of coenzymes, reflecting redox changes. Lipidomics revealed consistent remodeling, notably involving cardiolipin precursors. Transcriptomic profiling showed engulfing-senescent MCF7 cells possess a distinct signature marked by increased lipid metabolism, endocrine signaling, and cell–cell communication. Overall, our findings reveal conserved and cell type–specific metabolic traits of TIS, highlighting metabolic vulnerabilities for senolytic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4f815c5ea5ff8465ffcedac66d5a28bbf1f8878" target='_blank'>
              Metabolic profiling of therapy-induced senescent cancer cells via TPEF, MALDI-MS, and RNA-sequencing
              </a>
            </td>
          <td>
            S. Ghislanzoni, F. Padelli, Matteo Niero, A. Bertolotti, A. Belfiore, Simone Torelli, A. Bresci, Andrea Masella, Silvia Betti, Dario Polli, Luca Agnelli, I. Bongarzone
          </td>
          <td>2025-12-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Primary tracheobronchial tumors (PTBTs) can often be successfully treated by surgery; however, more effective strategies are still warranted for advance tumor cases. The present study aimed to provide a comprehensive understanding of the molecular and cellular features of PTBTs, along with the tumor microenvironment (TME). By performing single-cell RNA, we revealed a distinct TME landscape of PTBTs containing 52,739 single cells. For instance, these three pathological types were found have heterogeneous fibroblast populations, immune cells were more activated in inflammatory myofibroblastic tumor (IMT). Notably, a subpopulation of BIRC5+ cells was identified in primary tracheal squamous cell carcinoma (SCC) and adenoid cystic carcinoma (ACC) samples, which may have similar properties to cancer stem cells (CSCs). Moreover, IMT was characterized by the infiltration of pro-inflammatory immune cells, and might be a benign tumor. Importantly, based on ligand-receptor analysis, the PPIA/CD147 pair was found to be strongly upregulated during tumor and CD8+ T cell interactions. By applying multiplex immunofluorescence for selected markers, we validated the presence of the separate cell entities. Our findings provide deeper insights into the biology of PTBTs, which may be beneficial for advancing the diagnosis and therapy of PTBTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc41a4f5248143a0c3cf35d4c57326e220116bee" target='_blank'>
              A single-cell atlas reveals the tumorigenesis and microenvironment status of primary tracheobronchial tumors
              </a>
            </td>
          <td>
            Chen Shu, Xiyang Tang, Kaifu Zheng, Runze Zhang, Yulong Zhou, Mengchao Li, Weiguang Du, Yuquan Bai, Yang Shen, Yujian Liu, Haitang Yang, Feng Yao, Xiaolong Yan, Nan Ma, Jinbo Zhao
          </td>
          <td>2025-12-13</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/035f8f0a37c10191f821e77ad80531501f71ec67" target='_blank'>
              Cancer-associated fibroblasts in the tumor microenvironment: heterogeneity, crosstalk mechanisms, and therapeutic implications.
              </a>
            </td>
          <td>
            Ziyue Huang, Jie Chen, Tianyu Zhu, Jinxin Li, Ho Yin Ng, Yixiong Zhou, Xiang Gu, Shiqiong Xu, Renbing Jia
          </td>
          <td>2025-12-18</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Metastatic castration-resistant prostate cancer (mCRPC) is a heterogeneous disease, in which patients each carry distinct molecular features that drive metastatic progression. However, the lack of clinically representative models that recapitulate this disease have limited our understanding in the heterogeneity between patients. LuCaP 147, established from a liver metastasis biopsy, is a prostate cancer transplantable xenograft carrying AR and SPOP mutations with a further hypermutated phenotype, as well as loss of MSH2 and MSH6 expression.



 Using LuCaP 147, we developed a novel patient-derived metastasis (PDM) model in NSG mice via intracardiac injection. The development of metastasis was monitored through weekly bioluminescent imaging. We analyzed the tumor burden and survival rate of the PDM147 models. At the endpoint, long bones, spines and soft tissues were collected. To evaluate the histologic phenotype of PDM147, tissue sections were stained by immunohistochemistry (IHC) for adenocarcinoma markers including AR and neuroendocrine markers (which include SOX2, ASCL1, and SYP).



 After intracardiac injection, tumor burden (BLI) was detected after 5 weeks, and mice developed morbidity after 7-8 weeks. H&E stains showed that PDM147 developed bone metastases in spines, long bones, as well as visceral metastases in liver, brain, spinal cord, kidney and adrenal glands. IHC indicated that metastases were heterogeneous with both AR+/NE- and AR-/NE+ cells. AR+ and SYP+ cells were intermingled in the tumor, but the stains themselves were mutually exclusive between cels. Interestingly, no androgen-deprivation therapy was needed to promote NE differentiation, showcasing the biological novelty [SA([1] of this model.



 PDM147 is a hypermutated tumor with mutually exclusively expressing AR+ and NE+ markers in metastatic sites, including soft tissues and nervous tissues. Future work to define distinct pathways and molecular markers that lead to de novo vs. treatment-induced NE phenotype may offer a better understanding of the biology driving metastatic spread in these similar yet distinct NE subtypes and the role a hypermutator tumor can have in prostate cancer progression. These findings underscore the value of patient-derived models to study mCRPC progression and heterogeneity. Further use of these models for high-throughput drug screens may uncover targeted therapies for advanced prostate cancer.



 Dana S. Vargas Solivan, JuanJuan (Ivy) Yin, Tri Truong, Joseph Twohig, Ross Lake, Adam G. Sowalsky. A patient-derived metastasis model with spontaneous neuroendocrine transformation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B080.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8565cd695486e6d924d7f7c053c2a965518cb8d0" target='_blank'>
              Abstract B080: A patient-derived metastasis model with spontaneous neuroendocrine transformation
              </a>
            </td>
          <td>
            Dana S Vargas Solivan, , Tri M Truong, Joseph Twohig, R. Lake, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The tumour microenvironment (TME) is now recognized as pivotal to cancer progression and treatment outcomes, alongside cancer cell-intrinsic factors. Among TME constitute, cancer-associated fibroblasts (CAFs) are abundant and actively shape tumour biology. They are heterogenous population arising from multiple origins, and engage in dynamic crosstalk with cancer cells and immune infiltrates to promote virtually all the hallmarks of cancer. By influencing and reinforcement of cancer hallmark traits, CAFs create a pro-tumorigenic niche that also protects cancer cells from therapy, in this way, conferring resistance to conventional therapy. Consequently, CAFs have emerged as central players linking cancer hallmarks to therapeutic resistance. This review discusses the origins and phenotypic diversity of CAFs, the mechanisms of their communication with tumour and immune cells, and how these interactions enforce hallmark capabilities of cancer that underlie treatment resistance. We also highlight preclinical and clinical evidence for CAF-driven drug resistance and examine current strategies to target CAFs or their effects. A deeper understanding of CAF heterogeneity and function could pave the way for novel combination therapies that dismantle the tumour-CAF alliance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d10e61bcec209fefda855e7181e3c9e5807e7aba" target='_blank'>
              Cancer-associated fibroblasts: central players in cancer hallmarks and therapeutic resistance
              </a>
            </td>
          <td>
            Yuanjun Lu, Ya Wu, Xiaoyu Xu, Yibin Feng, Ning Wang
          </td>
          <td>2025-12-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0b876781bf210db756c35db260709c192ea56b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution.
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A McIntyre, Akimasa Hayashi, Nicolas Lecomte, Marc Hilmi, Wungki Park, Nan Pang, Eileen M. O’Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2026-01-22</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Colorectal cancer (CRC) often occurs in combination with sarcopenia (SP), which affects the prognosis of CRC, but the histopathological mechanisms underlying this process are unclear. Single-cell transcriptome technology provides a tool for resolving multi-organ pathological interactions, and previous studies have lacked direct molecular evidence of a trans-tissue microenvironment. To address this, we applied an integrated analytical approach. The integration of single-cell transcriptome data from 16 CRC tumor tissues (scRNA-seq) and 17 SP muscle tissues (snRNA-seq) was used to systematically resolve heterogeneity across tissue microenvironments using Harmony to correct for batch effects and combining cellular annotations, differential expression analyses, and CellChat communication modeling. Our analysis revealed that CRC and SP share 3 core cell types, namely, endothelial cells, fibroblasts and monocytes, but they show distinct transcriptional and functional differentiation in different tissues. CRC cells are enriched in the inflammatory, angiogenic and matrix remodeling pathways, accompanied by the upregulation of hypoxia-associated genes, whereas SP is biased toward muscle structural maintenance, metabolic regulation and repair functions. The MIF-CD74, COLLAGEN and THBS-CD47 pathways were active in CRC, whereas SDC4 and CD36-related signals were dominant in SP. VWF, PDGFRA and FCN1 were stably expressed in both groups, suggesting the existence of a conserved homeostatic regulatory network. In conclusion, this study reveals the cross-tissue functional heterogeneity of cell types shared by CRC and SP, suggests a set of potential pervasive regulators, and provides new perspectives for understanding the systemic effects of tumors and the microenvironmental mechanisms of sarcopenia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28aac551ab88d08b24ed6a70bb80afdcde46ff4f" target='_blank'>
              Single-cell transcriptome analysis elucidates the microenvironmental interactions between colorectal cancer and sarcopenia
              </a>
            </td>
          <td>
            Tihong Qiu, Xiaoying Wang, Jie Li, Yiping Guan, Hongyou Wen
          </td>
          <td>2025-11-28</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5407cdb757b46038485ad236efb4e8ad5a8a93" target='_blank'>
              Tumor-associated macrophages promote chemoresistance to Paclitaxel via activating NOTCH2-JAG1 juxtacrine signaling.
              </a>
            </td>
          <td>
            Fazhi Yu, Qin Zhou, Weiqiang Yu, Tong Zhou, Cheng Cao, Yijia Xie, Peng Zhang, Hanyuan Liu, Wei He, Aoxing Cheng, Xiaopeng Ma, Qingfa Wu, Qi Zhao, Jing Guo, Kaiguang Zhang, Ying Zhou, Jue Shi, Zhenye Yang
          </td>
          <td>2026-01-10</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Identifying the cell of origin that harbors an initial driver mutation is key to understanding tumor evolution and for the development of new treatments. For isocitrate dehydrogenase (IDH)-mutant gliomas, the most common malignant primary brain tumor in young adults, the cell of origin is currently poorly understood. We conducted deep sequencing on 142 tissues from 70 individuals comprising tumors, peritumoral cortex or subventricular zones, and blood. Low-level IDH mutations were found in the peritumoral cortex in 37.9% (11 of 29) of patients. Integrating cell-type-specific mutation analysis, the direction of clonal evolution, spatial transcriptomics from patient brains, and a cancer mouse model arising from mutant oligodendrocyte progenitor cell, we determined that glial progenitor cells harboring an initial IDH mutation were responsible for the development of IDH-mutant gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87878ebfd4fd1887849bd103fe178e14f5e98a11" target='_blank'>
              IDH-mutant gliomas arise from glial progenitor cells harboring the initial driver mutation.
              </a>
            </td>
          <td>
            Jung Won Park, Jiehoon Kwak, K. Kim, Saehoon Jung, Chang Hyun Nam, Hyun Jung Kim, Sang Mee Lee, Chanho Choi, Yongjin Ahn, , Jihwan Yoo, J. Shim, H. Cho, E. Kim, Chungyeul Kim, Sangjeong Ahn, Stefan Pusch, Andreas von Deimling, Jong Hee Chang, Se Hoon Kim, Hoon Kim, Young Seok Ju, S. Kang, J. H. Lee
          </td>
          <td>2026-01-08</td>
          <td>Science</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="


 The human prostate develops from the urogenital sinus through epithelial bud formation and branching, however the spatial dynamics of progenitor populations during these early stages remain poorly characterized. To addresses this gap, we used a human fetal prostate at 14 weeks of gestation for serial sectioning, yielding approximately 200 sections of 10 µm thickness. Six sections were profiled using spatial transcriptomics with the Visium HD platform, while the remaining sections were H&E-stained for 3D reconstruction of glandular architecture. We mapped transcriptional heterogeneity across developing prostate buds and identified distinct spatial organization of epithelial subdomains with basal-like and luminal-like characteristics. Notably, we also identified a unique cluster of tubule-initiating cells (TIC cells), that have been linked to cancer initiation and androgen-deprivation therapy resistance in adult prostate. This study provides the first high-resolution spatial map of developing human prostate, highlighting the organisation of progenitor populations.



 Emmanouela Perisynaki. 3D spatial mapping of developing human prostate glands [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B059.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9e67e64e1e6f9570a547e8411a3ed76fac26e1" target='_blank'>
              Abstract B059: 3D spatial mapping of developing human prostate glands
              </a>
            </td>
          <td>
            Emmanouela Perisynaki
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Clear cell renal cell carcinoma (ccRCC) is the most common subtype of kidney cancer and is marked by pronounced intra-tumoural heterogeneity that complicates therapeutic response. Patient-derived organoids offer a physiologically relevant model to capture this diversity and evaluate treatment effects. When integrated with spatial transcriptomics, they might enable the mapping of spatially resolved transcriptional and isoform-level changes within the tumour microenvironment. Methods: We established a robust workflow for generating patient-derived ccRCC organoids, that are not passaged and retain original cellular components. These retain key features of the original tumours, including cancer cell, stromal, and immune components. Results: Spatial transcriptomic profiling revealed multiple transcriptionally distinct regions within and across organoids, reflecting the intrinsic heterogeneity of ccRCC. Isoform-level analysis identified spatially variable expression of glutaminase (GLS) isoforms, with heterogeneous distributions of both the GAC and KGA variants. Treatment with NUC-7738, a phosphoramidate derivative of 3′-deoxyadenosine, induced marked transcriptional remodelling of organoids, including alterations in ribosomal and mitochondrial gene expression. Conclusions: This study demonstrates that combining long-read spatial transcriptomics with patient-derived organoid models provides a powerful and scalable approach for dissecting gene and isoform-level heterogeneity in ccRCC and for elucidating spatially resolved transcriptional responses to novel therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11ea98f536949abee4bc7391c3123466548631ed" target='_blank'>
              Long-Read Spatial Transcriptomics of Patient-Derived Clear Cell Renal Cell Carcinoma Organoids Identifies Heterogeneity and Transcriptional Remodelling Following NUC-7738 Treatment
              </a>
            </td>
          <td>
            Hazem Abdullah, Ying Zhang, Kathryn Kirkwood, Alexander Laird, Peter Mullen, David J. Harrison, Mustafa Elshani
          </td>
          <td>2026-01-14</td>
          <td>Cancers</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Polymorphonuclear myeloid-derived suppressor cells (PMN-MDSCs) are crucial mediators of tumor-induced immunosuppression, while their heterogeneity and spatial dynamics across malignancies remain poorly understood. By integrating single-cell RNA sequencing data from 576 samples across 19 cancer types and spatial transcriptomics data from three distinct malignancies, we identified a PMN-MDSC population. This cell population demonstrated characteristic upregulation of immunosuppressive genes and was associated with poor prognosis across multiple cancer cohorts. Notably, TREM1 was highly expressed in PMN-MDSCs and may mediate immunosuppressive processes. Multiplex immunofluorescence demonstrated that TREM1+ PMN-MDSCs exhibited significantly higher distribution in tumor regions compared to non-tumor tissues. Spatial transcriptomics analysis revealed their co-localization with fibroblasts and exhausted T cells. Moreover, CellChat analysis showed that TREM1+ PMN-MDSCs remodeled the tumor microenvironment through interactions with diverse cellular components. Collectively, our study revealed the conserved immunosuppressive features and spatial interaction networks of TREM1+ PMN-MDSCs from a pan-cancer perspective, highlighting TREM1 as a pivotal therapeutic target to disrupt PMN-MDSC-mediated tumor immune evasion. A pan-cancer analysis reveals TREM1+ PMN-MDSCs as key mediators of immunosuppression by remodeling the tumor microenvironment, highlighting TREM1 as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5174fd91c2ae4f071c06cfa6c978a0e63d85760" target='_blank'>
              Pan-cancer analysis reveals TREM1+ PMN-MDSCs as critical regulators of immune suppression and tumor microenvironment remodeling
              </a>
            </td>
          <td>
            Yangjie Cai, Shanhang Li, Hening Li, Zhuan Zou, Xinda Zheng, Haijun Tang, Mingxiu Yang, Pintian Wang, Weizhen Wu, Hongcai Teng, Kai Luo, Xinyu Huang, Wenyu Feng, Shijie Liao, Juliang He, Yun Liu
          </td>
          <td>2025-12-18</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Traditionally, scientists tend to approach cancer research in a reductionistic way: aiming at uncovering underlying, separate components in malignant processes. And indeed, great progress has been made by reducing the development of a tumor to single, specific genes and mutations. For instance, familial adenomatous polyposis (FAP) could be reduced to a germline mutation in the Adenomatous Polyposis Coli (APC) gene. The escape of tumor cells from immune surveillance could be reduced to the tumor expression of immune checkpoints, resulting in new approaches in tumor therapy by applying immune checkpoint inhibitors. However, a germline mutation in APC is not 1:1 related to colorectal cancer (CRC), and only some patients respond to immune checkpoint inhibitors. The point here is that biological systems, also comprising cancer, have properties that cannot be reduced to single components. The cooperation of the single components results in new, emergent properties. The outcome of an interaction in a complex network, like the immune system, depends on the many cell types involved and the numerous molecules that interact and activate or inhibit pathways. The way the composing elements are organized is a causal factor in itself for any emergent property. The rise of genomic analysis at the end of the previous century, enabling us to sequence a full genome at the DNA and RNA levels, has initiated an awareness of the need for ‘systems biology’: to consider a full system and how it is organized, in all of its aspects, to understand biological pathways and their outcomes. In this review, we outline the prospects and limitations of systems biology in cancer research and propose a causal framework that integrates upward and downward causation and multiple realizability to understand the emergent properties of tumors that determine the dynamics of tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bad26730d321f49013a748603a1b42d130f54226" target='_blank'>
              The Need for a Systems Biology Approach in Cancer Explained
              </a>
            </td>
          <td>
            Hehuan Zhu, Xi Zhang, E. Nazemalhosseini-Mojarad, J. Roelands, L. D. A. N. de Muynck, C. Ravensbergen, Rachel Hoorntje, I. Stouten, Marianne Hokland, A. L. Vahrmeijer, Rob A. E. M. Tollenaar, Edwin Koster, Peter J K Kuppen
          </td>
          <td>2025-12-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V. Semenov, Irina S. Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-08</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Ectonucleotide pyrophosphatase/phosphodiesterase 1 (ENPP1), 1st identified in breast cancer and subsequently in multiple other cancer types, is an innate immune checkpoint regulator that recently emerged as a promising biomarker and therapeutic target. Homologous recombination deficiency (HRD) has gained clinical relevance with therapeutic vulnerability, particularly in breast and ovarian cancers. Despite the increasing significance of ENPP1 and HRD in cancer biology and treatment, their potential relationships have not yet been comprehensively investigated. We analyzed the relationship between ENPP1 expression and HRD score across the Cancer Genome Atlas pan-cancer and individual tumor types using the Pearson and Spearman correlations. To account for heterogeneity, pan-cancer samples were clustered using linear regression into 3 groups based on Bayesian Information Criterion. Differential expression, functional enrichment, and survival analyses were performed for these clusters at both the pan-cancer and representative tumor type levels. Although the pan-cancer relationship between ENPP1 expression and HRD score was heterogeneous, significant correlations were observed in 11 tumor types. Linear regression-based clustering resolved this heterogeneity into 3 functionally and clinically distinct groups: Cluster 1 was characterized by proliferation programs; Cluster 2 by extracellular matrix remodeling, differentiation, and immune response; and Cluster 3, by metabolic reprogramming. Clinically, Cluster 3 was associated with better survival than Clusters 1 and 2 in a pan-cancer analysis (P < .0001). At the individual tumor type level, these global cluster features were further modified in tissue-specific contexts, reflecting local microenvironment adaptation. Significant survival differences were observed in patients with adrenocortical carcinoma, chromophobe renal cell carcinoma, low grade glioma, and mesothelioma, further underscoring the tissue-specific modification of global cluster features. Our comprehensive pan-cancer analysis revealed the intrinsic heterogeneity of ENPP1 expression and HRD score, which may arise from complex and dynamic interactions with diverse cancer hallmarks, including proliferation, extracellular matrix remodeling, immune response, and metabolic reprogramming, and can be generalized into 3 clusters with distinct molecular and clinical characteristics. At the individual tumor type level, these global cluster features were further modified to adapt to a tissue-specific microenvironment, manifesting distinct tissue-specific patterns. Collectively, these findings provide a foundation for refining biomarker-driven precision medicine strategies for diverse tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad738ec3c685dfb5e7e6594798ea773356192816" target='_blank'>
              Interaction between ENPP1 and homologous recombination deficiency defines distinct pan-cancer signatures: A retrospective observational study
              </a>
            </td>
          <td>
            Yong Min Kim, Sung‐Hak Lee, Woong Na, Il Ju Lee, Mihye Kwon, Oyeon Jo, Young Soo Song
          </td>
          <td>2026-01-02</td>
          <td>Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="This study addresses the limited mechanistic understanding behind medical imaging for tumor microenvironment (TME) assessment. We developed a novel framework that analyzes tumor imaging heterogeneity index (TIHI)-correlated genes to uncover underlying TME biology and therapeutic vulnerabilities. DCE-MRI and mRNA data from 987 high-risk breast cancer patients in the I-SPY2 trial, together with mRNA data from 508 patients in GSE25066, were analyzed. TIHI-associated genes were identified via Pearson correlation, clustered via weighted gene co-expression network analysis (WGCNA), and subgroups were defined via non-negative matrix factorization (NMF). The clinical relevance of the image-to-gene comprehensive (I2G-C) subtype defined by subgroups was assessed using logistic regression and Cox analysis. I2G-C comprised four clusters with distinct immune and replication/repair functions. It further stratified receptor, PAM50, and RPS5 subtypes. The “immune+/replication+” was more likely to achieve pathological complete response (pCR) (OR = 2.587, p < 0.001), while the “immune−/replication−” was the least likely to achieve pCR (OR = 0.402, p < 0.001). The “immune+/replication+” showed sensitivity to pembrolizumab (OR = 10.192, p < 0.001) and veliparib/carboplatin (OR = 5.184, p = 0.006), while “immune-/replication-” responded poorly to pembrolizumab (OR = 0.086, p < 0.001). Additionally, “immune+/replication-” had the best distant recurrence-free survival (DRFS), whereas “immune-/replication+” had the worst (log-rank p = 6 × 10−4, HR = 5.45). By linking imaging heterogeneity directly to molecular subtypes and therapeutic response, this framework provides a robust, non-invasive surrogate for genomic profiling and a strategic tool for personalized neoadjuvant therapy selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaac41c7947552aff84e1e324fe7769979665178" target='_blank'>
              Tumor Imaging Heterogeneity Index-Inspired Insights into the Unveiling Tumor Microenvironment of Breast Cancer
              </a>
            </td>
          <td>
            Qingpei Lai, Xinzhi Teng, Jiang Zhang, Xinyu Zhang, Yufeng Jiang, Yao Pu, Peixin Yu, Wen Li, Tian Li, J. Cai, Ge Ren
          </td>
          <td>2025-11-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed4de06e0917ea972558b5b84105d5368df0be3" target='_blank'>
              Key immune cells in the tumor immune microenvironment of colorectal cancer: Roles and research advances (Review).
              </a>
            </td>
          <td>
            Ming Qiu, Chongyuan Lan, Minglin Lin, Hui Ma
          </td>
          <td>2026-03-01</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Motivation Single-cell omics analysis has unveiled the heterogeneity of various cell types within tumors. However, no methodology currently reveals how this heterogeneity influences cancer patient survival at single-cell resolution. Here, we introduce scSurv, combining a Cox proportional hazards model with a deep generative model of single-cell transcriptome, to estimate individual cellular contributions to clinical outcomes. Results The accuracy of scSurv was validated using both simulated and real datasets. This method identifies cells associated with favorable or adverse prognoses and extracts genes correlated with their contribution levels. In melanoma, scSurv reproduces known prognostic macrophage classifications and facilitates hazard mapping through spatial transcriptomics in renal cell carcinoma. We also identified genes consistently associated with prognosis across multiple cancers and demonstrated the applicability of this method to infectious diseases. scSurv is a novel framework for quantifying the heterogeneity of individual cellular effects on clinical outcomes. Availability The implementation of scSurv is available on GitHub (https://github.com/3254c/scSurv) and Zenodo (https://doi.org/10.5281/zenodo.17793054).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98bd46ab28d9f09fcd249606230d4459ff93bbef" target='_blank'>
              scSurv: a deep generative model for single-cell survival analysis
              </a>
            </td>
          <td>
            Chikara Mizukoshi, Yasuhiro Kojima, Shuto Hayashi, Ko Abe, Daisuke Kasugai, Teppei Shimamura
          </td>
          <td>2025-12-22</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cervical cancer is a prevalent malignancy among women, yet the involvement of neutrophils in its tumor microenvironment remains insufficiently explored. This study utilized single-cell RNA sequencing (scRNA-seq) to delineate neutrophil subsets and elucidate their roles in disease progression and prognosis. Analysis of 20 cervical biopsy samples across different disease stages identified five neutrophil subsets (N0-N4), among which the N4 subset exhibited a marked increase during disease advancement. Spatial transcriptomics and tissue microarray analyses revealed that N4 neutrophils are enriched in tumor regions and are associated with genes implicated in proliferation, metastasis, and immune evasion. Functional characterization demonstrated that N4 promotes tumor progression via activation of the Wnt signaling pathway and extracellular matrix remodeling. A neutrophil infiltration-based risk model was established and validated through multi-omics approaches, highlighting its potential in prognostic prediction. These findings underscore the pivotal role of N4 neutrophils in cervical cancer and provide valuable insights for the development of targeted immunotherapies and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9eca7bc06f3fa7d3102669c9c6d4eefb56b347" target='_blank'>
              Single-Cell and Multiomic Analysis Reveals Neutrophil Heterogeneity and Prognostic Value in Cervical Lesions.
              </a>
            </td>
          <td>
            Ze Wang, Zhuang Liang, Shiyi Liu, C. Cao, Shimin Chen, Nan Yu, Xiaoyuan Huang, Tao Zhang
          </td>
          <td>2025-12-12</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Urothelial carcinoma (UC), encompassing both bladder cancer and upper tract UC, is a biologically heterogeneous malignancy that has long complicated patient stratification and therapeutic prediction. Recent genomic and transcriptomic analyses, however, have delineated reproducible molecular subtypes across non-muscle-invasive and muscle-invasive disease. These classification frameworks have refined biological understanding and improved both risk stratification and treatment selection. Emerging evidence also indicates that genomic alterations, such as FGFR3 mutations, may confer distinct immunologic phenotypes within specific transcriptomic contexts, thereby challenging prior assumptions. The advent of immune checkpoint inhibitors has transformed UC management, yet variable response rates underscore the complexity of the tumor microenvironment (TME). Single-cell and spatial analyses have identified immune-inflamed, -desert, and -excluded TME subtypes, with cancer-associated fibroblasts playing a key role in immune exclusion and therapeutic resistance. Liquid biopsy, particularly circulating tumor DNA, has demonstrated utility for real-time disease monitoring, minimal residual disease detection, and treatment stratification, as highlighted in trials such as IMvigor010 and IMvigor011. Urinary extracellular vesicles also represent promising non-invasive biomarkers, though further standardization is required. In addition, novel therapeutic strategies-including antibody-drug conjugates, gene therapies, and sustained-release delivery systems-are broadening treatment options across disease stages. Collectively, these translational advances mark a paradigm shift toward precision immuno-oncology in UC. Nonetheless, successful clinical implementation will require prospective validation, optimized analytical platforms, and sustained physician-scientist engagement to translate these discoveries into improved patient outcomes. This review provides a comprehensive overview of recent progress in UC translational research, with a focus on genomic and transcriptomic insights, the evolving role of immunotherapy, advances in liquid biopsy, and the development of novel therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e44cd39c3a6847a9cce1aefa2b7bb98a79ab335f" target='_blank'>
              Translational Advances in Urothelial Carcinoma: From Bench to Bedside.
              </a>
            </td>
          <td>
            T. Iwasawa, T. Tsujino, S. Tsukahara, Hiroshi Fukushima, Daisuke Ito, E. Tomiyama, Fumihiko Urabe
          </td>
          <td>2026-01-01</td>
          <td>International journal of urology : official journal of the Japanese Urological Association</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b370747cb8a1720741d860f50d6f1043c88dbdfc" target='_blank'>
              Cancer stem cell-driven drug resistance in colorectal carcinoma: molecular aspects and therapeutic potentials.
              </a>
            </td>
          <td>
            Minfeng Zhou, Huifang Niu, Dandan Cui, Menghao Xu, Jinxiao Li, Guichen Huang, Minquan Zhou, Chutong Xiong, Yunya Liu, Xiaojuan Xu, Hongxing Zhang, Fengxia Liang, Rui Chen
          </td>
          <td>2026-01-22</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Gastrointestinal stromal tumors (GISTs) exhibit significant heterogeneity, posing substantial challenges for personalized treatment strategies. Patients often display varied responses to different therapeutic agents and dosages. Currently, the absence of robust and physiologically relevant in vitro models for GISTs impedes accurate prediction of therapeutic efficacy, thereby constraining the advancement of effective treatment strategies. Traditional 2D cell cultures fail to replicate the tumor microenvironment (TME) and lack patient-specific characteristics, limiting their predictive value. In contrast, three-dimensional bioprinting (3DP) technology faithfully recapitulates key histological architecture and molecular features of their parental tumors, enhancing the physiological relevance of in vitro models. We employed patient-derived 3DP-GIST models via 3D bioprinting technology, followed by comprehensive histopathological, genomic, and transcriptomic analyses. Subsequently, we applied clinically approved targeted therapeutic agents to perform drug screening and response prediction on the 3DP-GIST models. The drug sensitivity profiles obtained from these models were then correlated with retrospective clinical data and patient follow-up records to assess the models’ potential in guiding the selection and prediction of effective GIST therapies. In our study, we successfully constructed 12 patient-derived 3DP-GIST models. Histopathological assessments, whole-exome sequencing (WES), and transcriptomic analyses confirmed that these models accurately recapitulate the histological architecture, biomarker expression, and molecular features of their corresponding parental tumors. Transcriptomic profiling further revealed gene expression signatures associated with GIST recurrence risk and imatinib resistance. Importantly, the 3DP-GIST models demonstrated the capacity to provide precise, individualized treatment recommendations within 10 days post-surgery, potentially reducing treatment delays and improving patient outcomes. Overall, the 3DP-GIST model represents a robust and efficient platform for evaluating patient-specific drug sensitivities in vitro, thereby guiding personalized therapeutic strategies for GIST patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492f9dec710186d0bb398ff8467a9fedaf4e7eda" target='_blank'>
              Three-dimensional bioprinting of patient-derived Gastrointestinal stromal tumor: a novel platform for precision oncology and drug response profiling
              </a>
            </td>
          <td>
            Liwei Du, Zicheng Zheng, Yanan Wang, Kai Zhang, Yuce Lu, Minghao Sun, M. Pang, Shangze Jiang, Yixuan He, Shunda Du, Haitao Zhao, Yilei Mao, Huayu Yang, Weiming Kang
          </td>
          <td>2025-12-18</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Accurate identification of cancer tissue/cell of origin (TOO/COO) is critical for diagnosis and treatment; yet existing whole-genome approaches demand extensive computation and often struggle with sparse mutation signals. Here, we introduce an informative regional subset framework that selects a small number of biologically and statistically significant 1Mbp genomic intervals to train a CatBoost prediction model. On a benchmark of 137 whole-genome samples across six cancer types, our method achieved a 4% gain in melanoma accuracy (from 88.0% using all 2,128 regions to 92.0% with 300 regions), a 4.4% gain in multiple myeloma (87.0% with 600 regions), and perfect (100%) accuracy in high-mutation cancers such as esophageal adenocarcinoma and glioblastoma with as few as 50 informative regions. When extended to 934 PCAWG samples spanning 14 cancer lineages, the same limited regional subsets matched or improved whole-genome performance, reaching up to 100% accuracy in gastrointestinal, skin, and brain cancers, demonstrating exceptional scalability. Our approach not only reduces computational burden and enhances interpretability but also provides a robust, generalizable tool for precision oncology and the diagnosis of cancers of unknown primary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3d967b1f97e63587e24e5aa5c7c6abe1cd4c7a5" target='_blank'>
              Biologically-informed regional subset analysis with CatBoost for robust tissue-of-origin prediction
              </a>
            </td>
          <td>
            Sungmin Yang, Hong-Ki Kim
          </td>
          <td>2025-12-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e6d8a02bb8fcc15958d3d8c8939ab24272dbaf6" target='_blank'>
              Cell-in-cell structures and cancer metastasis.
              </a>
            </td>
          <td>
            Yuchen Cao, Jianing Du, Jianguo Lai, Chunjun Liu
          </td>
          <td>2026-01-23</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Lung cancer symptoms typically do not appear until advanced stages, leading to late diagnoses. This delay is a major contributor to its poor prognosis, resulting in lung cancer being the leading cause of all cancer-related deaths worldwide. Essential biological understanding of how lung cancers arise and progress is still lacking. Studying premalignant lesions that eventually develop into invasive carcinomas helps to bridge this knowledge gap. We are focusing on adenocarcinomas (ACs), the most common subtype of lung cancer, and especially, activating mutations in the Kirsten rat sarcoma virus oncogene (KRAS), which is among the most common oncogenic mutations and has previously been reported in premalignant lung lesions. Given that KRAS signaling is growth-promoting, it is reasonable to assume that KRAS mut promotes tumor formation and progression. However, the extent of its influence, the timing of when it arises, and particularly its connection to histologic and phenotypic changes driving tumor evolution, remain unclear.



 We have collected a cohort from biopsy material, initially collected for diagnostic purposes from AC patients at varying stages of tumor evolution, with some matched ‘normal-histology’ samples when available. In total, tissue from 38 patients with AC premalignancies is available for investigation. To map KRAS mutations in situ, we employed a BaseScope assay with commercial detection probes against the different KRAS mut transcripts. To map KRAS statuses to cellular phenotypes in the microenvironment, we are running the 6K discovery panel on the CosMx platform (Bruker) on a subset of patients, which is a highly sensitive spatial transcriptomics technique at single-cell resolution. Spatial Mass Spectrometry (MS) tracking metabolic features on the timsTOF flex MALDI-2 (Bruker) is also being performed on a subset of patients.



 Initial tests of the BaseScope assay have revealed specific, yet not sensitive, signals of KRAS mut transcripts in premalignant lesions. While advanced cancers display distinct signals, premalignant lesions have fewer, if any, signals. This might be due to the lower expression levels of KRAS at the earlier stages. To address this, we are now applying a BaseScope assay with signal amplification to be able to track KRAS expressed at lower levels. Preliminary analyses of the Spatial MS matrix have revealed highly individual metabolomics features, with great inter-patient heterogeneity. This pattern is also reflected in bulk RNA-seq data, revealing expression profiles more similar across, e.g., Adenocarcinoma in Situ (AiS) and invasive AC from the same patient, than between the AiSs from different patients.



 By applying this state-of-the-art multi-omics approach to premalignant adenocarcinomas, we hope to identify key regulators of early lung cancer progression and to translate these findings to clinically relevant basic molecular diagnostics.



 Amanda Lindberg, Bartosz Sobocki, Patrick Micke, Carina Strell. In situ KRAS-mapping and spatial-omics characterization in lung premalignancy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A002.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23b558a4354aae9f9807afda6b85aec2e424e2cc" target='_blank'>
              Abstract A002:
 In situ
 KRAS-mapping and spatial-omics characterization in lung premalignancy
              </a>
            </td>
          <td>
            A. Lindberg, B. Sobocki, P. Micke, C. Strell
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAM) exert both pro- and anti-tumoral functions that influence cancer progression and patient prognosis. However, single-cell RNA sequencing (scRNA-seq) studies have revealed that TAM heterogeneity remains incompletely characterized. By performing an unbiased, integrated in silico analyses of publicly available scRNA-seq datasets, comprising samples from blood, tumor, and non-tumoral mammary tissue (NTMT) from both breast cancer (BRCA) patients and healthy individuals, we identified seven transcriptional signatures corresponding to distinct TAM subsets, exhibiting unique functional profiles, including heightened interferon responses, scavenging, and matrix remodeling, the latter two being characteristic of tissue repair. Notably, none of these subsets aligned with the M1/M2 classification of macrophage (Mφ) polarization. Interferon-associated genes were predominantly enriched in blood monocytes, whereas tissue-repair-associated-signatures were more abundant in tissue resident Mφ, suggesting that TAMs bearing these signatures resemble monocyte-derived or tissue-resident Mφs, respectively. Importantly, TAM subsets expressing interferon-associated genes were associated with improved survival compared to tissue-repairing TAMs in a BRCA cohort from The Cancer Genome Atlas (TCGA). Additionally, one signature was heightened in peripheral monocytes from BRCA patients compared with healthy individuals, which was experimentally validated in a pilot study of Mexican BRCA patients. We concluded that these signatures are a closer description of TAM heterogeneity in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769e9d0c6f0718581121627c404c58103ae137e7" target='_blank'>
              Single cell analysis identifies inflammatory and tissue remodeling tumor associated macrophages distinct from M1/M2 paradigm.
              </a>
            </td>
          <td>
            Eric López-Huerta, Diana P Portales-Pérez, M. E. Martínez-Leija, Vianney Ortiz-Navarrete, E. Fuentes-Pananá
          </td>
          <td>2025-12-05</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Cancer evolution, driven by genetic instability and clonal selection, challenges effective treatment. This review synthesizes computational approaches to model tumor progression, heterogeneity, and resistance, bridging biology with precision medicine. We integrate multi-omics data with mathematical frameworks to uncover adaptive behaviors, guiding therapies that exploit evolutionary vulnerabilities. Our methods span cancer’s multiscale nature. Ordinary differential equations (ODEs) model biochemical networks, like cyclin-dependent kinase dynamics, with cell cycle progression as \(\frac{d[CycD]}{dt} = k_s - k_d [CycD] - k_p [CycD][CDK4/6]\), revealing treatment effects. Probabilistic models, like branching processes, predict lineage evolution with exponential waiting times \(P(t) = \lambda e^{-\lambda t}\). Partial differential equations (PDEs) address tumor microenvironment influences, e.g., hypoxia-driven selection: \(\frac{\partial u}{\partial t} = D \nabla^2 u + f(u) - g(u,v)\), where \(u\) and \(v\) denote cell and nutrient density. Agent-based models (ABMs) capture phenotypic plasticity, while Boolean networks identify regulatory attractors. Dynamical systems theory quantifies chaos via Lyapunov exponents (\(\lambda = \lim_{t \to \infty} \frac{1}{t} \ln \left| \frac{\delta x(t)}{\delta x(0)} \right|\)) and fractal dimensions, signaling relapse risks. Algorithmic information dynamics, using Kolmogorov complexity \(K(s) = \min \{ |p| : U(p) = s \}\), infers causal gene networks via block decomposition. Machine learning, including recurrent neural networks, forecasts omics time-series, while phylogenetic tools like PiCnIc reconstruct clonal trajectories. Analysis of breast and pancreatic cancer datasets shows chaotic attractors linked to chemoresistance, with multifractal spectra (Hurst index \(H\)) quantifying growth irregularity. Adaptive dosing, modeled via optimal control, reduces resistance by 40-60%, with PDE-ABM hybrids predicting clonal expansion under therapies. In conclusion, these computational tools reveal cancer as an evolving ecosystem, enabling personalized, evolutionary-guided therapies to outmaneuver tumor adaptation for improved outcomes.



 Peter Oloche David. Decoding cancer evolution: Computational models for tumor dynamics and precision oncology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B030.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/606f30f8647268effc6912a14fbea981af7d1e8b" target='_blank'>
              Abstract B030: Decoding cancer evolution: Computational models for tumor dynamics and precision oncology
              </a>
            </td>
          <td>
            Peter Oloche O. David
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lung cancer, recognized as one of the most prevalent malignancies with the highest rates of incidence and mortality globally, presents a substantial challenge on a worldwide scale. This challenge is exacerbated by the disease’s difficulty in early detection, a pronounced rate of metastasis, and resistance to treatment, all of which contribute to elevated mortality rates. The tumor microenvironment (TME) plays a critical role in the sustenance and advancement of various solid tumors, including lung cancer. The intricate composition of the TME facilitates tumor proliferation, metastatic spread, and therapeutic resistance by supplying metabolic resources, fostering angiogenesis, and enabling immune evasion. Nonetheless, the regulatory frameworks operating within the TME remain poorly understood. An increasing body of evidence suggests that epigenetic regulation—encompassing mechanisms such as DNA methylation, histone modification, and the action of non-coding RNAs—is pivotal in the initiation and progression of lung cancer. Furthermore, epigenetic modifications significantly influence the functional dynamics of the tumor microenvironment, thereby impacting intercellular interactions and cellular behaviors within the TME, which in turn affects the trajectory of disease progression. This article aims to present the most recent advancements in research concerning the epigenetic regulation of tumor cell interactions with the TME in the context of lung cancer biology. Additionally, it examines the current implications of epigenetic regulation within the tumor microenvironment and its influence on lung cancer behavior. We also investigate the potential relevance and emerging therapeutic avenues presented by epigenetic regulation in the clinical diagnosis and treatment of lung cancer, aspiring to propose novel strategies to address existing treatment challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f9bbf3d9c9172ba516c36021afce36965e86d5" target='_blank'>
              Epigenetic regulation of the tumor microenvironment in lung cancer: mechanism insights and therapeutic prospects
              </a>
            </td>
          <td>
            Maoqin Yang, Xudong Lei, Dexiang Ren, Dakai Qin, Xiaojun Xia
          </td>
          <td>2025-11-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Immunoediting is known to constrain tumor growth, yet fundamental aspects of immune activity remain unknown. A better understanding of how tumor neoantigens are recognized and targeted by the immune system will aid immunotherapy and tumor vaccine development. Often, the available data consist of bulk sequencing from a single timepoint in tumor growth. Here we ask: what can those single time-point data tell us about when and how the immune system recognizes and then kills cancer cells? We show that tumor sequencing plus neoantigen prediction is sufficient to distinguish between major models of immune recognition and to quantify immunoediting in biologically interpretable terms. We summarize sequencing data from each tumor with a “neoantigen depletion curve”: at each cancer-cell fraction (how common a mutation is in the tumor), we compare the number of predicted neoantigenic mutations to matched non-neoantigenic mutations. In our model, once a neoantigen is recognized by the immune system, cells carrying that neoantigen are killed at a constant rate per cell. Their effective growth rate is thus negative, which reduces the prevalence of that neoantigen. Integrating over all neoantigens produced yields characteristic depletion curve shapes. We evaluate four biologically motivated models: (1) no immune recognition; (2) constant recognition rate once a given neoantigen appears in even a single cell; (3) constant recognition rate triggered only once the number of cells carrying a given neoantigen exceeds a size threshold; and (4) gradually increasing recognition rate based on the number of cells carrying a given neoantigen (as a Hill function). We fit generalized linear models to mutation counts to estimate parameters and provide diagnostics and model selection. From a single tumor, we can recover three dimensionless quantities that summarize biology without requiring growth rates from outside data: (i) the strength of immune killing relative to tumor growth; (ii) the baseline recognition rate relative to growth; and (iii) the relative neoantigen clone size of recognition (the “knee” of the curve). We highlight qualitative predictions that can be sought in data: no immune recognition yields a flat neoantigen depletion curve; constant recognition tilts the curve; recognition after a threshold creates a sharp knee with a uniform post-knee slope across neoantigen qualities; and gradually increasing recognition yields a smooth knee with stronger pre-knee depletion for higher-quality neoantigens. Together, these results show that standard tumor sequencing—without longitudinal sampling—can reveal whether immune pressure is continuous or size-triggered and can quantify its relative strength and timing. This framework offers practical estimators, uncertainty quantification, and diagnostics for immunoediting in human tumors.



 Heng Wu, Elizabeth S. Borden, Karen T. Hastings, Ryan N. Gutenkunst. What single-tumor sequencing reveals about immune recognition and killing via neoantigen depletion curves [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B042.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0a87f06df668433a73f507d09c0d47cd6124c63" target='_blank'>
              Abstract B042: What single-tumor sequencing reveals about immune recognition and killing via neoantigen depletion curves
              </a>
            </td>
          <td>
            Heng Wu, E. S. Borden, K. T. Hastings, R. Gutenkunst
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a59f51e7895cef79efeacf0e1e7ab108d79743ff" target='_blank'>
              Machine learning-enabled spatial multi-omics uncovers lactate-driven targets and tumor microenvironmental reprogramming in cancer.
              </a>
            </td>
          <td>
            Yingzheng Tan, Wenliang Tan, Yanchao Liang, Yunzhu Long, Shuanghua Chen, Qihao Hu, Yangjing Ou, Jingli Fu, Huan Chen, Fangyuan Ren, Jun Ye, Qing Zhou, Sheng Li, Xiaojin He, Qianqian Wang, Yueming Shen, Haiyuan Lu, Daichao Wu, An-bo Gao, Xun Chen, Yukun Li
          </td>
          <td>2025-12-30</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with a poor prognosis and limited response to current therapies. Cancer stem cells (CSCs) contribute to this heterogeneity, driving tumor progression, immune evasion, and therapeutic resistance. Epigenetic regulators have emerged as pivotal modulators of CSC phenotypes. The histone methyltransferase SETDB1, a key stemness factor in both embryonic and adult stem cells, catalyzes H3K9 trimethylation (H3K9me3) and acts as a major oncogenic driver in a wide range of solid and hematological malignancies. Yet, its specific role in maintaining stemness and contributing to therapeutic resistance in HCC remains poorly defined. Integrative analyses of bulk and single-cell transcriptomic datasets from public HCC cohorts were performed. Tumors were stratified by SETDB1 expression, followed by pathway enrichment, immune deconvolution, and stemness scoring. Associations with clinical outcomes, immune phenotypes, drug resistance signatures, and chromatin-binding partners (DNMT3A, TRIM28, HDAC1) were also explored. The role of SETDB1 was then examined in vitro, where a stem-like phenotype was induced in HCC cell lines under hypoxic conditions, and patterns of SETDB1 expression and stemness marker levels were assessed following paclitaxel treatment. SETDB1-low tumors were enriched in well-differentiated, immunologically active subclasses and were associated with increased predicted responsiveness to immune checkpoint blockade (ICB), sorafenib, and transarterial chemoembolization. Conversely, SETDB1-high tumors exhibited transcriptional features of dedifferentiation, elevated stemness scores, and enrichment in aggressive molecular subtypes. These tumors also displayed higher TP53 mutation rates, reduced immune infiltration, immune exclusion signatures, and activation of pathways linked to ferroptosis and ABC transporter dysfunction, consistent with resistance to ICB and conventional therapies. SETDB1 expression was also associated with hypoxia-related pathways, suggesting a role in maintaining CSC niches. In vitro, hypoxia-induced stemness coincided with increased SETDB1 levels, whereas paclitaxel treatment decreased both SETDB1 expression and stemness markers. A chromatin risk score incorporating SETDB1 and its partners predicted poor disease-free survival independently of clinical parameters. SETDB1 defines a stemness-enriched, immune-resistant HCC subtype associated with poor outcomes and therapeutic failure. Correlations between SETDB1 expression and stem-like features in vitro suggest a potential role in maintaining CSC phenotypes, supporting its relevance as a biomarker and candidate for epigenetic- and CSC-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c393716a837e5b1ef0091b6385eb92c64be4ab4" target='_blank'>
              The epigenetic regulator SETDB1 as a key component of cancer stem cells and drug resistance in primary liver cancer
              </a>
            </td>
          <td>
            Maël Padelli, Christophe Desterke, Aurore Devocelle, Georges Uzan, Antoinette Lemoine, Julien Giron-Michel
          </td>
          <td>2026-01-06</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background CTBP1, an NADH-dependent transcriptional co-repressor subject to succinylation, may orchestrate both systemic and cell-type–specific effects on tumor development. Methods This integrative study represents a novel approach, combining genetic regulation, single-cell expression quantitative trait loci (eQTLs), and imaging radiomics to investigate CTBP1 in breast cancer. We integrated tissue-relevant eQTLs with breast cancer GWAS to evaluate genetically proxied associations for CTBP1. We further explored cell-type–dependent effects using single-cell eQTL resources and examined expression, survival, and imaging correlations as hypothesis-generating evidence. Results Tissue-focused analyses suggested an inverse association between CTBP1 expression and breast cancer risk, with consistent directions observed in expression and survival datasets. Single-cell explorations indicated potential cell-type–specific regulation.Imaging correlations with MRI-derived features were modest and exploratory. Conclusion Our results prioritise CTBP1 as a context-dependent, tissue- and cell-type–specific candidate in breast cancer. The observational and exploratory components warrant validation in larger cohorts and functional assays. To our knowledge, this is the first integrative study combining GWAS, single-cell eQTL and MRI radiomics to prioritize CTBP1 in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e46c9272d341965ba7ccef388ea64fa15367aa5" target='_blank'>
              Tissue- and Cell-Type-Specific Genetic Regulation of CTBP1 in Breast Cancer: Integrative Analyses with Exploratory Single-Cell and Imaging Data
              </a>
            </td>
          <td>
            Minping Hong, Xiaowen Huang, Qin Xu, Zhenyi Ma, Keng Ling
          </td>
          <td>2025-12-01</td>
          <td>Breast Cancer : Targets and Therapy</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Why and how does cancer start? Building from a Symposium at the 2025 Society of Toxicology meeting, we convened a group of international experts to answer this seemingly simple question. As experimental evidence has evolved, perspectives on cancers' origins have shifted from the accumulation of DNA mutations in single cells to complex processes involving signals from an altered tissue microenvironment which promote tumorigenesis. Carcinogen exposures impact the biology of the microenvironment in complex and tissue-specific ways. These changes can include the infiltration of inflammatory cells that produce growth factors, neo-angiogenesis, morphological changes, and immune tolerance that avoids immune-mediated elimination. In this in-depth review, we discuss the evidence linking chemical-driven microenvironmental changes in the development of a range of solid and liquid tumors. We discuss specific phenotypic alterations, such as selection pressure driving clonal expansion and cellular plasticity and reacquisition of stem cell states, linked to carcinogen-induced changes in the microenvironment. We describe assays and biomarkers which can allow us to experimentally assess links between chemical exposures, the microenvironment, and cancer phenotypes. We end by discussing how understanding the role of the microenvironment and malignancy in toxicology is essential for accurate cancer hazard evaluation, development of next-generation risk assessment frameworks, identifying new strategies for cancer prevention, and improving patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c64691a327a7b8227d277fcb6bbfe9d59f21fba7" target='_blank'>
              Rethinking the Microenvironment's Role in Chemical-Induced Malignancy.
              </a>
            </td>
          <td>
            William H. Bisson, Richard A Currie, Emilia L Lim, Coraline Mlynarczyk, Erik J Tokar, Annamaria Colacci, Justin A. Colacino
          </td>
          <td>2025-12-17</td>
          <td>Toxicological sciences : an official journal of the Society of Toxicology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Angioimmunoblastic T-cell lymphoma (AITL) is a subtype of peripheral T-cell lymphoma originating from follicular helper T cells and characterized by profound clinical, pathological, and molecular heterogeneity. Despite advances in classification and diagnosis, therapeutic outcomes remain suboptimal, particularly in relapsed or refractory settings. In recent years, the evolution of risk stratification models and a deeper understanding of AITL’s molecular pathogenesis, including its unique pattern of clonal evolution, have paved the way for precision medicine. Notably, targeted treatment approaches, such as immunomodulatory agents, epigenetic therapies, kinase inhibitors, and immune checkpoint blockade, are demonstrating promising clinical efficacy, especially in patients harboring specific molecular aberrations. This review comprehensively summarizes the multilayered heterogeneity of AITL, emphasizes the role of immune and molecular profiling in informing therapeutic decisions, and outlines future directions for personalized, multi-agent treatment strategies aimed at overcoming resistance and improving survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1f48636814c0a428af30d1bb9e6df8d06e9d0f" target='_blank'>
              Unraveling the many faces of angioimmunoblastic T-cell lymphoma: clinical, pathological, and molecular heterogeneity
              </a>
            </td>
          <td>
            Jia-Ping Chen, Peng Ke, Xiao Qiu, Qiu-Yuan Xian, Jing-Song Wu, Chun Feng, Hui-Jun Li, Yang Wang, Ji-Hao Zhou
          </td>
          <td>2026-01-01</td>
          <td>Annals of Hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Adamantinomatous craniopharyngioma (ACP), a benign yet highly recurrent and therapy-resistant intracranial tumor, remains a considerable clinical challenge because of its complex pathological structure, infiltrative growth, and limited treatment options. Here, integrated spatially resolved multiomics is employed-including single-cell spatial transcriptomics via CosMx SMI and spatially resolved metabolomics via AFADESI-MSI, accompanied by bulk metabolomics and functional validation-to unravel the driving factors of ACP progression and recurrence. Analysis results reveal three interdependent biological hallmarks: first, the spatial segregation and molecular heterogeneity of 10 distinct tumor epithelial cell subpopulations within the ACP, each of which presents unique transcriptional signatures; second, in tumor regions and recurrent tumor epithelium tissues, stronger transporter-mediated choline/ethanolamine uptake from cystic fluid and significant upregulation of phosphatidylcholine (PC) and phosphatidylethanolamine (PE) synthesis is observed, creating the enhanced "cystic fluid-tumor cell" and "choline/ethanolamine-PC/PE" metabolic axis, and demonstrating the spatial metabolic remodeling of ACP; and third, this metabolic axis directly couples to autophagy activation of corresponding regions in ACP tissue, which is validated by multi-immunohistochemistry for Beclin1 and GABARAP. Together, these findings reveal metabolic remodeling and autophagic activation as critical drivers of ACP progression and recurrence and provide an opportunity for precise biomarker-driven treatment of this intractable tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4bf376edd94fbdf6f86d786c026d6877b062546" target='_blank'>
              Spatially Resolved Multiomics Reveals Metabolic Remodeling and Autophagy Activation in Adamantinomatous Craniopharyngiomas.
              </a>
            </td>
          <td>
            Dongting Chen, Yahui Gao, Yulin Wang, Ting Lei, Zheng Qu, Yuhan An, Jiaxu Fu, Xin Li, Fangjun Liu, Yan Li
          </td>
          <td>2026-01-05</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gene-specific expansion patterns were evident among the most frequent CH lesions, with DNMT3A-mutant clones exhibiting impaired hematopoietic differentiation and TET2-mutant clones showing multi-lineage propagation. Notably, identical CH clones were detected in 41% of corresponding lymphomas, displaying distinct clonal dynamics: tumor-promoting CH (expansion in B-NHL; 10/16 clones; mainly TP53) and tumor-infiltrating CH (no expansion; mainly DNMT3A). Moreover, we identified lymphoma-associated mutations in flow-sorted hematopoietic progenitors from patients with indolent but not aggressive B-NHL and observed a stepwise accumulation of mutations along the lymphoid differentiation path. Single-cell genotyping confirmed the presence of mutated progenitors in 3 follicular, 2 mantle cell and 2 marginal zone lymphoma patients, providing direct evidence of a pre-neoplastic state in disease pathogenesis. Our findings offer novel insight into the cellular origin of nodal B-NHLs and highlight a previously underappreciated role for early clonal events involving the stem/progenitor cell compartment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ec5b65b719f8d9e5db3ee2069637056c360c3b" target='_blank'>
              Clonal Hematopoiesis and Lymphoma-Associated Mutations in Hematopoietic Progenitors in B-Cell Non-Hodgkin Lymphoma.
              </a>
            </td>
          <td>
            Laura Wiegand, Patricia Silva, D. Noerenberg, F. Christen, Klara Kopp, B. Locher, P. Löwe, Marlon Tilgner, Robert Altwasser, Vanessa Storzer, Catarina M Stein, F. Briest, C. M. Arends, Mareike Frick, J. Ihlow, A. Dolnik, N. Ishaque, U. Keller, Il-Kang Na, L. Penter, Lars Bullinger, Raphael Hablesreiter, Frederik Damm
          </td>
          <td>2026-01-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Androgen-targeted therapy, chemotherapy and 177Lu-PSMA radioligand therapy have improved metastatic prostate cancer (mPC) patient survival. However, most patients develop resistance to these therapies. Studying genomic and transcriptomic changes that cause resistance requires repeated sample collection and is not suited to traditional invasive tissue biopsy techniques. We have developed a liquid-biopsy based technique to isolate cell-free DNA (cfDNA) and circulating tumor cell (CTC) RNA from mPC patients enabling genomic and transcriptomic profiling. Transcriptomic profiling of 146 high tumor purity mPC CTC samples from 70 patients identified 4 transcriptional phenotypes driven by changes in luminal, AR and proliferative signaling pathways: Low Proliferation (LP, n= 12, low luminal/AR/proliferation) Luminal A (LumA, n=24, high luminal/AR, low proliferation). Luminal B (LumB, n=31, high luminal/AR/proliferation) and Neuroendocrine Prostate Cancer (NEPC, n=3, low luminal/AR, high proliferation). Compared to LP and LumA, the LumB phenotype is associated with shorter overall survival and early progression on 177Lu-PSMA highlighting an unmet need for treatment options in patients with LumB CTCs. Here, we use mPC CTC transcriptional profiling to identify drivers of proliferation and treatment resistance in the LumB phenotype.



 Blood was collected from patients with mPC receiving standard of care treatment. CTCs were purified using immunomagnetic capture on a microfluidic platform, subjected to RNA-seq analysis and assessed for transcriptional phenotypes and gene expression signatures.



 We compared expression of gene sets related to established tumor suppressors or oncogenic drivers, DNA damage repair by homologous recombination repair (HRR), and cell cycle drivers in LP, LumA and LumB CTC phenotypes. LumB CTC samples display elevated expression of cell cycle genes related to S/G2/M phase progression (CCNE1, CCNA2 and CCNB1), but not G1 phase progression (CCND1 or CCND3), suggesting that increased S/G2/M progression is contributing to increased proliferation. In preclinical models, CCNE1 overexpression causes genomic instability and activation of DNA damage repair, cell cycle checkpoints and a S/G2/M phase transcriptional program mediated by the MYBL2-FOXM1 transcription factor complex. Indeed, we also observe increased expression of HRR, cell cycle checkpoint and MYBL2-FOXM1 transcriptional signatures in LumB subtypes.



 CTC transcriptional profiling has identified an aggressive and treatment resistant LumB phenotype characterized by high expression of genes involved in S/G2/M cell-cycle progression, HRR and cell cycle checkpoints. These findings suggest that elevated cell cycle progression and upregulated DNA repair capability may underlie the aggressive behavior and therapeutic resistance observed in patients with LumB CTCs. Ongoing efforts are focused on developing preclinical models of the mPC CTC transcriptional phenotypes to identify therapeutic vulnerabilities specific to the LumB subtype.



 David Gallo, Jamie M. Sperger, Amy K. Taylor, Kristen Rosche, Viridiana Carreno, Alex H. Chang, Emily Abella, Kaitlin Durnen, Muhammad Dar, Charlotte Linebarger, William M. Stump, Kendra Marr, Kyle T. Helzer, Matthew L. Bootsma, Grace C. Blitzer, John M. Floberg, David Kosoff, Rana R. McKay, Wei X. Xiao, Shuang G. Zhao, Joshua M. Lang, Marina N. Sharifi. Identifying therapeutic vulnerabilities in metastatic prostate cancer transcriptional phenotypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd3122808dac421339edd09df9a3f36d4488905" target='_blank'>
              Abstract B023: Identifying therapeutic vulnerabilities in metastatic prostate cancer transcriptional phenotypes
              </a>
            </td>
          <td>
            David Gallo, J. Sperger, , Kristin Rosche, Viridiana Carreno, Alex H Chang, Emily Abella, Kaitlin Durnen, Muhammad Dar, C. Linebarger, William M. Stump, Kendra Marr, K. Helzer, M. Bootsma, G. Blitzer, J. Floberg, D. Kosoff, Rana R. McKay, Wei X. Xiao, Shuang G. Zhao, Joshua M Lang, M. Sharifi
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="AI-driven virtual cell models show the potential to transform the paradigm of life sciences research by integrating multimodal omics data (e.g., single-cell transcriptomics and proteomics) with advanced algorithms such as deep generative models and graph neural networks to enable high-precision predictions of drug responses, gene perturbations, and disease progression. These models enable high-precision predictions of drug responses, gene perturbations, and disease progression. This review outlines the technical pathways and validation mechanisms of virtual cells, emphasizing a closed-loop workflow from computational evaluation to experimental verification using CRISPR assays and organoid platforms. The applications of virtual cells in personalized drug screening and disease modeling are highlighted, showcasing their potential to reduce animal testing and optimize therapy. However, challenges in regulatory acceptance, data privacy, and model interpretability remain. Global policy and standardization trends are driving clinical translation, and future advancements will involve cross-disciplinary integration and greater standardization to enhance the impact of virtual cells in precision medicine and drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5cc4085a09776c1b5f2f07f02af18c81780fd33" target='_blank'>
              AI-driven virtual cell models in preclinical research: technical pathways, validation mechanisms, and clinical translation potential
              </a>
            </td>
          <td>
            Chunyu Ma, Han Zhang, Yiwei Rao, Xinyu Jiang, Boheng Liu, Zhikang Sun, Zhenyu Song, Yuan Gao, Yuhao Cui, Xinyu Liu, Zedong Li
          </td>
          <td>2025-12-11</td>
          <td>NPJ Digital Medicine</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND: The spatial organization of immune cells within tumors is an important area of investigation in studies of interactions between tumor cells and the tumor microenvironment. This is particularly relevant because some immune cells function through direct contact with their target cells, whereas others communicate over a distance via paracrine signaling involving cytokines. Thus, the topography of tumor cells and microenvironmental cells may determine the possibility and nature of intercellular interactions and thereby influence the functional effects of immune cells. AIM: This study aimed to compare the spatial transcriptomic profiles of tumor and stromal regions enriched in macrophages in triple-negative breast cancer. METHODS: Eight patients with triple-negative breast cancer were included. Spatial transcriptomic analysis was performed on formalin-fixed, paraffin-embedded tissue sections using high-throughput RNA sequencing with the 10x Visium platform. The annotated spots enriched in intraepithelial and stromal macrophages were used for downstream bioinformatic analysis. RESULTS: A total of 437 differentially expressed genes were identified between the two groups of spots containing macrophages with distinct spatial localization. Spots with intraepithelial macrophages were characterized by activation of processes related to cytokine and chemokine signaling, regulation of regulatory T-cell differentiation, organization of cell – cell contacts, wound healing, and inhibition of viral activity. Spots enriched in stromal macrophages demonstrated activation of biological processes associated with the regulation of angiogenesis, cell migration and recruitment, cell adhesion, and stromal remodeling. CONCLUSION: Macrophage topography within primary tumors of triple-negative breast cancer is associated with their functional characteristics. These fundamental findings may be useful for developing prognostic criteria and therapeutic approaches aimed at modulating the tumor microenvironment to improve long-term outcomes in patients with triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1996d432009ab729b97a00ec19d18afda8c70949" target='_blank'>
              Spatial Characterization of Macrophage-Enriched Tumor Regions in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            I. Patskan, A. Kalinchuk, Elisaveta Tsarenkova, Evgeniia Grigoryeva, I. Larionova, Natalia Popova, L. Tashireva
          </td>
          <td>2025-12-02</td>
          <td>Genes &amp; Cells</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Somatic mutations in cancer cells can lead to diverse phenotypic consequences due to variations in mutation types and locations within the same gene. Interactions between mutations across different genes further contribute to tumor heterogeneity, affecting pathology, phenotype, and clinical outcomes. Current computational methods often treat all mutations within a gene as functionally equivalent and overlook combinatorial effects between genes. Additionally, the diversity of sequencing platforms and gene panels poses significant challenges for integrative genomic analysis across datasets. Here, we introduce SomAtt (Somatic Attention), a foundation model for tumor genomes that leverages self-attention mechanisms to model interactions among mutations within a tumor. Trained on 166,854 tumors from 149,831 unique cancer patients, SomAtt generates comprehensive tumor embeddings suitable for various downstream tasks. The model addresses three critical challenges in tumor genome analysis: (a) distinguishing the effects of different mutations within the same gene, (b) capturing complex interactions among multiple mutations within a tumor, and (c) integrating data from diverse targeted mutation panels without requiring uniformity. We demonstrate that SomAtt accurately classifies cancer types with an accuracy of 76.5% across 34 cancer types, outperforming the previous state of the art model by 2% on MSK-IMPACT panels. SomAtt also predicts patient survival outcomes, achieving a concordance index (C-index) of 0.76 across 89 gene panels and 108 cancer types. Notably, SomAtt also enables patient stratification beyond traditional subtyping methods by identifying previously unrecognized, molecularly defined patient subgroups with significantly different outcomes, suggesting a novel approach to cancer subtype classification based on tumor molecular profiles. Our model could enable better assessment of pre-cancerous lesions molecularly and aid in early detection. SomAtt produces a robust and scalable framework for comprehensive modeling of tumor genomes, addressing key challenges in cancer genomics and advancing the potential for precision oncology.



 David Arredondo, Luis Tafoya, Oladimeji Macaulay, Ala Jararweh, Avinash Sahu. SomAtt: A foundation model for the tumor genome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A023 .
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53845b6455ffb522448c775ae954ade68b7f7b42" target='_blank'>
              Abstract A023: SomAtt: A foundation model for the tumor genome
              </a>
            </td>
          <td>
            David Arredondo, Luis E. Tafoya, Oladimeji S Macaulay, Ala Jararweh, Avinash Sahu
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite improvements in the molecular profiling of pancreatic neuroendocrine tumors (PanNETs), predicting their clinical behavior and response to specific therapies remains challenging. We sought to elucidate the molecular basis underlying the broad phenotypic variations in these neoplasms through a genetic characterization of primary and metastatic PanNETs. Our findings revealed an enrichment of CDKN2A homozygous deletions and TSC2 somatic mutations in metastatic PanNETs when compared to non-metastatic lesions. Tumor evolution analysis further revealed the acquisition of such genetic alterations as late events in the progression of these neoplasms, conferring poor survival outcomes to the affected patients. Biallelic loss of DNA damage repair genes, ATRX and/or DAXX, was associated with a high fraction of the genome altered in PanNETs, with pathogenic alterations affecting those genes also being associated with a homologous recombination deficiency signature. These findings highlight molecular mechanisms driving PanNET progression and underscore the need for further molecular characterization and tumor evolution studies to evaluate targeted therapies for such a challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db2469c6632fab23ebab1e0e71bc53bea256e7dd" target='_blank'>
              CDKN2A homozygous deletions and TSC2 somatic mutations in metastatic pancreatic neuroendocrine tumors
              </a>
            </td>
          <td>
            Tito Teles Jesus, Lorenzo Ferrando, Lia Rodrigues, R. Martins, Luís Cardoso, José Manuel Lopes, Paula Soares, Arnaud Da Cruz Paula, J. Vinagre
          </td>
          <td>2025-12-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5632b562f5d8cd55ed3ab2b552edc3104ca3f554" target='_blank'>
              ONCOchannelome: A computational framework for investigating altered ion channels across tumor types
              </a>
            </td>
          <td>
            K. Parthasarathi, Soundharya Ramu, M. Jolly, Jyoti Sharma
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent a paradigm of cancer evolution, where spatial and temporal heterogeneity creates dynamic ecosystems that rapidly adapt to therapeutic pressures. Through multi-region multi-omics profiling of 45 patients across primary, irradiated, and recurrent tumors, we reconstructed evolutionary trajectories using an integrated analytical framework. DNA methylation (RRBS) was processed via Bismark, with CAMDAC deconvolution distinguishing cell type-specific signals from copy number alterations. Transcriptomic data were analyzed using Salmon and edgeR, while copy number landscapes were inferred through TitanCNA and InferCNV. Evolutionary dynamics were reconstructed using PRISM, enabling methylation-based phyloepigenome reconstruction to trace subclonal epigenetic evolution. Our analysis reveals that STS evolution follows a branching phylogenetic pattern, with early “trunk” alterations establishing the foundational epigenome and subsequent “branch” events driving radiation adaptation. PRISM-based methylation phylogenies demonstrated conserved evolutionary trajectories across patients, with epigenetic divergence increasing through treatment. At diagnosis, widespread methylome remodeling systematically rewires cellular identity, with convergent hypermethylation-mediated silencing of tumor suppressive networks (SMARCA4, DICER1, FOXO3) and hypomethylation-driven activation of oncogenic pathways (CCND1/2, CDK6). Across patients, these events were recurrent and enriched for functional gene expression changes, representing positive selection for survival advantages. Under radiotherapy selection pressure, the evolutionary dynamics shift from broad remodeling to targeted adaptation. The epigenome stabilizes, focusing on DNA repair (DDB2), stemness (ZBTB16), and proliferation (CCND1) pathways—evidence of directional selection for resistance mechanisms. Single-cell phylogenetics of >150,000 cells reveals that radiation selects for pre-existing subclones rather than generating novel populations, with overall ecosystem architecture maintained despite cellular turnover. Spatial-temporal mapping demonstrates that evolutionary trajectories vary dramatically both between patients and within individual tumors, generating multi-focal ecosystems where distinct subclones coexist, compete, and occasionally converge on shared adaptive solutions. This evolutionary framework explains radiotherapy resistance as the inevitable outcome of selection acting upon pre-existing heterogeneity. The convergent emergence of DNA repair enhancement, stem-like properties, and immunosuppressive niches across patients represents parallel evolution under common selective pressures. Together, these findings argue for the development of evolution-informed therapeutic strategies that anticipate and intercept adaptive trajectories, with the potential to overcome the formidable challenge of heterogeneity in treatment-resistant STS.



 Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, Raffi Avedian, Robert Steffner, David Mohler, Anusha Kalbasi, Matt van de Rijn, Gregory Charville, Everett Moding. Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbc7c928e0902865ecc73b6cf6d1b8b370047f" target='_blank'>
              Abstract B018: Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation
              </a>
            </td>
          <td>
            Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey C L Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, R. Avedian, R. Steffner, David Mohler, A. Kalbasi, Matt van de Rijn, Gregory Charville, E. Moding
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The lung is a structurally and immunologically complex organ, constantly exposed to airborne microbes, allergens, and pollutants. Understanding how diverse pulmonary immune cells respond to these challenges is critical for advancing respiratory disease research and identifying appropriate therapeutic interventions. Flow cytometry remains a cornerstone of immune profiling, and advances in high‐parameter spectral cytometry have significantly expanded its analytical capabilities. However, challenges, such as poor tissue dissociation, spectral overlap, and loss of spatial information, can hinder the comprehensive interpretation of the lung environment. To address these limitations, we developed an optimized spectral flow cytometry platform, utilizing 5‐laser Cytek Aurora spectral cytometers, for deep immunophenotyping of murine lung tissue. Our approach integrates in vivo CD45 antibody labeling—administered intravenously and oropharyngeally—to distinguish circulating, airway, and interstitial immune populations, preserving spatial context in single‐cell suspensions. We utilize complementary 25+ parameter panels targeting myeloid and lymphoid compartments, built on a shared backbone to enable consistent classification across datasets. Refined tissue processing protocols support optimal recovery of representative lung cell populations, and overnight intracellular staining enhances marker resolution. Using this platform, we reliably resolve stromal, endothelial, and epithelial cells alongside immune subsets—including macrophages, monocytes, dendritic cells, eosinophils, neutrophils, T and B cells, innate lymphoid cells (ILCs), and natural killer (NK) cells—subclassified by activation, function, and tissue residency. Validation with an influenza A virus model confirmed expected dynamic immune responses and revealed previously unrecognized populations. This spatially informed approach enables high‐resolution interrogation of pulmonary immunity in health and disease. © 2025 Wiley Periodicals LLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91eea359003311f7d1a7abfa2e8821468360d814" target='_blank'>
              Spatially Resolved Immune Profiling of the Murine Lung Using In Vivo Labeling and Spectral Cytometry
              </a>
            </td>
          <td>
            Rebecca Palmer, Katherine R Pilkington, Kit Moloney-Geany, Kathryn E Hally, K. Hilligan
          </td>
          <td>2025-12-01</td>
          <td>Current Protocols</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77606e4919065b0b8c161216da3a8b4fee1b4a2c" target='_blank'>
              Deficiency of SMARCB1 drives an immunosuppressive microenvironment in meningioma.
              </a>
            </td>
          <td>
            Ben Jin, Hao Hu, Yanhua Lu, Xia Tian, Guanghui Hu, Jingjing Xu, Xingqi Wu, Long Zhang, Juxiang Chen, Miaoxia He
          </td>
          <td>2026-01-24</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Ovarian cancers have been the most lethal gynecologic malignancies worldwide, with survival outcomes remaining poor despite therapeutic advances. Since TP53 is the most frequently mutated gene in ovarian cancers, it highlights the central role of p53 dysfunction in promoting tumor initiation, genomic instability, and progression. Beyond classical loss of transcriptional activity, mutant p53 may exert dominant‐negative effects on residual wild‐type protein and acquire oncogenic gain‐of‐function properties that promote invasion, metastasis, and resistance to chemotherapy. Nevertheless, despite extensive efforts, effective therapeutic agents targeting TP53‐mutant tumors remain elusive, representing a major unmet medical need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ae95b37ac19ede6887e4eb26c6f0ef7fba06778" target='_blank'>
              Novel Approaches for Detecting TP53 Mutations and Targeted Therapeutic Strategies: Emerging Insights Into Cytoplasmic p53 Aggregates in Ovarian Cancer
              </a>
            </td>
          <td>
            Naoyuki Iwahashi, Kazuhiko Ino
          </td>
          <td>2025-11-30</td>
          <td>Journal of Obstetrics and Gynaecology Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most malignant cancers, and studies have indicated that microbes within tumors play a crucial role in CRC. Advanced methodologies, including single-cell and spatial technologies, high-resolution sequencing, and multi-omic integration, are now unraveling the complex composition and function of the intratumoral microbiome. Mechanistically, these microbial communities contribute to CRC initiation by serving as direct mutagens that induce genomic instability, perpetuating a state of chronic inflammation, and activating specific carcinogenic pathways. Furthermore, they actively promote tumor progression and metastatic dissemination through multiple means, including the modulation of key oncogenic signaling pathways, extensive remodeling of the tumor immune microenvironment, and facilitation of a pro-metastatic niche. Given these profound and multifaceted influences, the intratumoral microbiome shows significant promise as a source of diagnostic and prognostic biomarkers, offering considerable potential for non-invasive monitoring and improved risk stratification in clinical practice. Therapeutically, intervention strategies are rapidly evolving, encompassing approaches such as microbiome modulation to enhance conventional therapies, precise clearance of pathogenic bacteria, utilization of intrinsically antitumor microbes, and the engineering of synthetic bacteria as targeted living therapeutics. This review comprehensively outlines the current research methods, elaborates on the mechanistic insights, and discusses the therapeutic targeting of the intratumoral microbiome, aiming to provide a foundational framework for developing new and effective strategies in CRC precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce1e9b5182a3afa421b56313133899ce3cefa90" target='_blank'>
              Intratumoral microbiome: a crucial regulating factor in development and progression of colorectal cancer
              </a>
            </td>
          <td>
            Yating Fan, Xiangshuai Gu, Hua Yang, Ye Chen, Chao Fang, Hongxin Deng, Lei Dai
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a highly plastic subpopulation within tumors that drives tumor initiation, progression, metastasis, therapeutic resistance, and disease relapse. Although CSC traits were initially attributed to stable genetic hierarchies, growing evidence suggests that cancer stemness is a dynamic and reversible state regulated by precise molecular events at the protein level. This literature review aims to synthesize experimental evidence regarding the impact of single amino acid mutations and alterations in regulatory proteins on the stemness properties of cancer cells. A systematic literature search was conducted using PubMed and Google Scholar to identify original research articles published between 2020 and 2025 that examined protein-level mutations, post-translational modifications, epigenetic regulators, and chromatin remodeling factors in diverse cancer models. The reviewed studies consistently demonstrate that single amino acid substitutions in oncogenic, tumor suppressor, or regulatory proteins can profoundly affect protein conformation, stability, enzymatic activity, and protein–protein interactions. These alterations frequently result in sustained activation of key stemness-associated signaling pathways, including Wnt/β-catenin, Notch, Hedgehog, and STAT3, thereby enhancing self-renewal capacity, stem cell marker expression, tumor-initiating potential, and resistance to conventional therapies. In parallel, dysregulation of post-translational modification systems, such as phosphorylation, ubiquitination, acetylation, and methylation, promotes epigenetic reprogramming and reactivation of embryonic transcriptional programs that favor cellular plasticity. Moreover, protein-level alterations strengthen cancer cell interactions with the tumor microenvironment, increasing responsiveness to hypoxia, inflammatory signals, and metabolic stress. Collectively, these findings highlight protein-level regulation as a central determinant of cancer stem cell maintenance and plasticity, and suggest that effective therapeutic strategies should target aberrant protein stability, post-translational modification machinery, and stemness-related signaling networks to achieve durable clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f961c9b291a85047c8455458a035edcd22afcac" target='_blank'>
              IMPACT OF SINGLE AMINO ACID MUTATIONS OR ALTERATIONS IN REGULATORY PROTEINS ON THE STEMNESS PROPERTIES OF CANCER CELLS
              </a>
            </td>
          <td>
            Dewi Sartika, Irmatul Laili, Sri Utami
          </td>
          <td>2025-12-30</td>
          <td>Journal of Stem Cell Research and Tissue Engineering</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3e338d4040b91a67afdb219d415b0b509c6561" target='_blank'>
              The role of PCMT1 in prognosis tumor immune microenvironment and therapeutic responses across cancers.
              </a>
            </td>
          <td>
            Bo Wang, Sijia Huang, Ruizhen Ren, Ruiqian Yao, Erwen Kou, Haixia Zhao, Hao Zhu, Mengyu Zhang, Liang-zhe Wang, Yuan-jie Zhu
          </td>
          <td>2026-01-05</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer is increasingly recognized not as a fixed genetic condition but as a dynamic and plastic disease state driven by reversible epigenetic, transcriptional and microenvironmental cues. This evolving understanding supports a therapeutic paradigm shift: from eradicating malignant cells to reprogramming them toward quiescence, differentiation or functional normalization. This review explores diverse strategies for redirecting cancer cell fate, including differentiation therapy, epigenetic remodeling, lineage reprogramming, senescence induction and tumor microenvironment resetting. These approaches exploit intrinsic cellular plasticity and contextual adaptability, offering novel avenues to contain malignancy and overcome resistance. By reframing cancer as a potentially reversible phenotype, this therapeutic strategy demands redefinition of clinical endpoints, incorporation of dynamic biomarkers, and development of integrative treatment frameworks. Ultimately, reprogramming‑based oncology holds the promise of transforming aggressive malignancies into manageable conditions while minimizing the collateral damage associated with conventional cytotoxic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99bfdfe1c6957af978b8fb4415d04fd96bdf7b85" target='_blank'>
              Beyond eradication: Therapeutic reprogramming strategies for cancer normalization (Review).
              </a>
            </td>
          <td>
            Aliasgar F. Shahiwala
          </td>
          <td>2025-12-04</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b954d4c894a7ba2af75533050256820b25f647" target='_blank'>
              Heterocellular crosstalk and architecture of the pancreatic tumour microenvironment.
              </a>
            </td>
          <td>
            F. Arnold, Annachiara Del Vecchio, Zainab Hussain, Mara H. Sherman
          </td>
          <td>2026-01-16</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Aging is accompanied by a marked increase in cancer incidence and mortality, yet most studies still consider cellular senescence, the tumor microenvironment, and the microbiome as largely separate axes. Here, we propose an integrative triad framework in aging-related cancers in which cellular senescence, tumor microenvironment (conceptualized here as part of a broader tumor microecology), and the microbiome dynamically interact to shape tumor initiation, evolution, and treatment response. We summarize how senescent cells, via context-dependent senescence-associated secretory phenotypes (SASPs), remodel stromal, immune, and metabolic niches in aging hosts and how gut and intratumoral microbiota both induce and are reshaped by senescence. Focusing on colorectal cancer (CRC), hepatocellular carcinoma (HCC) and pancreatic ductal adenocarcinoma (PDAC), together with pan-cancer transcriptomic and microbiome analyses. We highlight disease and subtype-specific patterns in which senescence signatures, immune contexture, and microbial features co-stratify prognosis and therapeutic outcomes, and integrate pan-cancer transcriptomic and microbiome analyses to illustrate shared and divergent triad configurations across tumor types. Finally, we discuss the therapeutic implications of this triad, including timing-dependent use of senolytics and senomorphics, diet and microbiome-targeted interventions, fecal microbiota transplantation (FMT), and the ecological risks of antibiotics, particularly in multimorbid older patients. We argue that triad-informed biomarkers and trial designs integrating senescence, microenvironment, and microbiome readouts will be important for mechanism-based, age-adapted cancer prevention and therapy in older adults, especially those with CRC, HCC, and PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c79c4b8b9fa9df17856107d73318990a71932296" target='_blank'>
              The Emerging Triad in Cancer and Aging: Cellular Senescence, Microbiome, and Tumor Microenvironment.
              </a>
            </td>
          <td>
            Xi Gong, Shaogang Wang, Qidong Xia
          </td>
          <td>2025-12-21</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, A. Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, C. Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, Stanley W. K. Ng, S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Li-Fraumeni Syndrome (LFS), caused by pathogenic germline Tp53 mutations, is a pediatric cancer predisposition disorder associated with markedly elevated risk of early-onset tumors, particularly rhabdomyosarcoma (RMS). Our study examined how developmental disruption may prime the tissues of LFS individuals for tumorigenesis. Single-nuclei RNA sequencing (snRNA-seq) was performed on RMS tumors, distal muscle from mice matched to RMS tumors, and cancer-free WT and LFS mouse muscle to define lineage-specific transcriptomic signatures of LFS-RMS. Embryonic limb buds were collected at embryonic days 10, 12, 14, and 16, subjected to snRNA-seq, and used to reconstruct developmental trajectories. These lineage-specific signatures were mapped back onto both WT and LFS embryonic lineages to determine whether oncogenic programs were detectable in utero and to identify the timing of their emergence. A stress-primed transcriptional program was identified as a defining feature of LFS tissues. This program distinguished precancer from postcancer states, becoming amplified in both RMS tumors and distal muscle matched to tumors. In LFS embryos, stage-specific alterations were observed in cell type proportions and maturation, including the emergence of unique myogenic and mesenchymal subpopulations absent in both WT embryos and postnatal muscle, reflecting developmental plasticity restricted to the LFS embryonic context. A lag in the transition to mature myogenic cells was detected at E14, preceded by a spike in the LFS stress-primed signature at E12 during windows of active lineage specification and proliferation. Pathway analyses of lineage-specific signatures indicated widespread dysregulation, with particularly strong changes in RNA-associated and developmental programs, further defining this stress-sensitive transcriptional state. These findings provide the first single-nuclei resolution map of embryogenesis in LFS and provide evidence that LFS-associated oncogenic programs are prenatally encoded. By linking altered lineage dynamics, developmental plasticity, and stress-primed transcriptional states to the emergence of RMS, this work establishes a developmental framework for understanding pediatric cancer predisposition. The results highlight potential windows for early detection and suggest opportunities to target embryonically primed programs for intervention in LFS.



 Ashby Kissoondoyal, Paula R. Quaglietta, Brianne Laverty, David Malkin. Single-nuclei profiling of LFS development reveals tumour susceptibility [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4bd18cd0e2b5ff6a30ceebfb643ff450b1c37a" target='_blank'>
              Abstract B033: Single-nuclei profiling of LFS development reveals tumour susceptibility
              </a>
            </td>
          <td>
            Ashby Kissoondoyal, Paula R. Quaglietta, B. Laverty, D. Malkin
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Tumor development and progression reshape the physical properties of the surrounding tumor microenvironment (TME), including its biomechanical traits. This is driven by a prominent cell type in the TME, cancer-associated fibroblasts (CAFs), which increases tissue stiffness via extracellular matrix deposition and remodeling. Currently, it is unclear whether there are also physical changes to CAFs at the cellular level and, if so, how they relate to patient outcome. Here, it is shown that CAFs have distinct morphological and biomechanical features from normal fibroblasts. Matched, patient-derived CAFs and non-malignant prostate fibroblasts (NPFs) from 35 patients with primary prostate cancer are examined. Morphologically, CAFs have more aligned stress fibers and larger and more elongated nuclei, based on quantitative image analysis of confocal microscopy images. In addition, single-cell mechanical measurements using real-time deformability cytometry showed that CAFs are larger and stiffer than NPFs. These changes are consistent across patients and validated with atomic force microscopy. A combined morphomechanical score encompassing these features is significantly associated with patient outcome. In transcriptomic analyses, the score is correlated with microtubule dynamics and a myofibroblast phenotype. Importantly, it is also demonstrated that morphomechanical features of prostate fibroblasts are modified by approved treatments for prostate cancer, such as docetaxel, and other small molecular inhibitors, particularly those targeting FGFR. In summary, changes in cellular morphomechanical properties are a consistent feature of CAFs and are associated with patient outcome. Moreover, cellular morphomechanical properties can be therapeutically targeted, potentially providing a new strategy for manipulating the TME to control cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe160be726a9c222a84b4d1334642ece40944d7" target='_blank'>
              Single-Cell Morphomechanics of Prostate Cancer-Associated Fibroblasts Identifies Distinct Features Associated with Patient Outcome.
              </a>
            </td>
          <td>
            Antje Garside, Angela Jacobi, S. Keerthikumar, Vaibhav Mahajan, M. Richards, B. Niranjan, Linda Teng, Nicholas K Choo, Johannes Low Jun Wei, G. Risbridger, Mitchell G. Lawrence, A. Taubenberger
          </td>
          <td>2026-01-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background Intratumor heterogeneity (ITH), a critical driver of tumor evolution and immune evasion, remains inadequately characterized at the transcriptomic level in colorectal cancer (CRC), and its clinical implications are not yet fully understood. Methods We integrated transcriptomic datasets from TCGA-COAD/READ and two independent GEO cohorts (GSE40967 and GSE87211) to develop an RNA-seq–based ITH score using the DEPTH2 algorithm and to con0struct an ITH-related gene (ITRG) prognostic model. A unified cutoff value of 0.64 was established to stratify patients into high- and low-ITH groups. Using 52 survival-associated ITRGs, we generated a nine-gene prognostic signature selected from 101 distinct combinations of feature selection techniques and modeling algorithms and validated its performance in two external datasets. The tumor microenvironment and potential responsiveness to immune checkpoint inhibitors were evaluated using ssGSEA, ESTIMATE, and TIDE algorithms. SHAP analysis, together with in vitro and in vivo experiments, was employed to identify and functionally validate key regulatory genes. Results Patients in the high-ITH group had markedly poorer overall survival (OS) than those in the low-ITH group. The ITH score correlated strongly with aggressive clinical features, including T3/4 invasion depth, N1/2 nodal status, and AJCC stage III. The nine-gene prognostic signature demonstrated consistent predictive capability across the TCGA training cohort and both GEO validation cohorts. In TCGA, the model yielded time-dependent AUCs of 0.669, 0.664, and 0.645 for 1-, 3-, and 5-year OS, respectively, and retained its status as an independent prognostic indicator in multivariate Cox regression analysis; in GSE40967 and GSE87211, the corresponding AUCs ranged from 0.544–0.573 and 0.648–0.744. The high-risk subgroup was characterized by a stromal immune phenotype enriched with cancer-associated fibroblasts and macrophages, elevated expression of multiple immune checkpoints, increased TIDE scores, and higher tumor mutational burden. SHAP analysis identified IL20RB as the top risk-associated gene, whose knockdown significantly suppressed CRC cell proliferation, migration, invasion, and tumorigenicity in vitro and in vivo. Conclusion This study introduces and validates a transcriptomic ITH score and a nine-gene ITRG-based prognostic model that delineate the immune landscape and enables effective survival stratification in CRC, complementing the limitations of current staging systems. Additionally, IL20RB is highlighted as a promising therapeutic target, supporting the development of personalized immuno-targeted combination therapies in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/385eed67dbbc79210270a5630b0149ac9e500c8e" target='_blank'>
              Intratumor heterogeneity score reveals immune landscape and survival stratification in colorectal cancer
              </a>
            </td>
          <td>
            Zijing Wang, Liyuan Ma, Jinzhong Cao, Xi Chen, Xin Chang, Ruxue Ma, Hengyi Lv, Zixin Zhang, Hai Li, Tao Jiang
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Homozygous deletion of the 9p21.3 genomic locus spanning the CDKN2A/B and MTAP genes is an event affecting 15% of cancers. While CDKN2A is a well-established tumor suppressor gene, the role of MTAP in tumorigenesis varies across cancer types. MTAP codes for methylthioadenosine phosphorylase, a key enzyme in the methionine salvage pathway, and its loss has been associated with several downstream synthetic vulnerabilities. Despite multiple efforts to exploit MTAP loss for targeted therapies, none of these efforts have yielded substantial results in clinical trials. In this review, we consolidate the existing literature along with our systematic analysis to provide an updated perspective on the incidence of MTAP loss in different cancers and elucidate its impact on metabolism, immune microenvironment, and tumor progression. In addition, we summarize the therapeutic strategies that have been investigated preclinically on MTAP-null tumors before and after the advent of functional genomic screening tools. We further assess the current landscape of clinical trials investigating MTAP-targeted inhibitors, evaluating their limitations and potential avenues for improvement. The insights gained from this review will inform future research directions beyond the promising PRMT5/MAT2A axis for rational combination therapies that would work synergistically to eradicate this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/846311f0527655d852715f68535274117ddc12d9" target='_blank'>
              MTAP-Null Tumors: A Comprehensive Review on Synthetic Vulnerabilities and Therapeutic Strategies
              </a>
            </td>
          <td>
            Bavani Subramaniam, Wai Chin Chong, Aylar Babaei, M. Bornhorst, Chunchao Zhang, R. Packer, Javad Nazarian
          </td>
          <td>2025-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The tumor microenvironment (TME) represents a highly dynamic and multifaceted ecosystem composed of both cellular and noncellular components that collectively modulate tumor initiation, progression, and metastatic potential. Within this intricate milieu, neuron-tumor interactions have recently emerged as critical determinants of cancer behavior, especially in malignancies of the nervous system, where neuronal activity appears to sustain and even accelerate tumor growth. Increasing evidence supports the existence of a bidirectional communication between neurons and cancer cells, involving the exchange of soluble factors, neurotransmitters, and vesicular signals that influence cell survival, proliferation, and morphology. To dissect the molecular mechanisms underlying this neuron-tumor crosstalk, we established and optimized a series of in vitro co-culture models. Primary cortical neurons derived from C57BL/6 mice and neurons differentiated from human induced pluripotent stem cells (hiPSCs) were co-cultured with U87 glioblastoma cells to reproduce the neural-tumor interface. Both transwell-based co-culture systems and conditioned media (CM) approaches were implemented to evaluate the impact of glioma-secreted components on neuronal viability and morphology. Neuronal dynamics were monitored in real time using the Incucyte live-cell imaging system equipped with the Neurotrack module. Exposure to glioma-conditioned medium led to pronounced neurotoxic effects in murine cells, including a 36% reduction in neurite length (p=0.0055), a 44% decrease in cell number (p=0.0024), and a significant increase in cytotoxicity after 48 h (p=0.0116). These findings highlight the detrimental influence of glioblastoma-derived soluble factors on neuronal structure and function. Complementary Western blot analyses confirmed this phenotype, revealing a marked decrease in βIII-tubulin (-60%) and synapsin (-80%) expression in CM-treated murine neurons, indicating impaired cytoskeletal stability and synaptic integrity. Current efforts focus on extending these observations to hiPSCderived neuronal cultures and enhancing the physiological relevance of the model by integrating advanced microfluidic co-culture systems. Such platforms will allow spatiotemporal control of cellular interactions, providing a more faithful in vitro representation of the brain tumor microenvironment and enabling the identification of molecular targets capable of disrupting the pro-tumor neuronal support network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47801036babe68c650156cad23ef9951afb3627d" target='_blank'>
              EXPLORING NEURON–GLIOMA SIGNALING THROUGH IN VITRO CO-CULTURES
              </a>
            </td>
          <td>

          </td>
          <td>2025-12-12</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b89d2858599b4236f22769f3a4af65ec9fd4eb7" target='_blank'>
              Deciphering a common code: Unitary gene profile factor enabling Spontaneous Tumor Regression
              </a>
            </td>
          <td>
            Arushi Misra, B. Kumari, Prasun K. Roy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Human papillomavirus type 16 (HPV16) is known to deregulate and cause hyperproliferation of the infected epithelium, but its full effect on the host’s tissue has remained elusive as earlier comprehensive studies were restricted to in vitro models. Aim The aim of the current study was to study HPV16-induced tissue changes in vulvar tissue in its full natural context by using single-cell spatial transcriptomics. Methods We applied spatial single-cell spatial transcriptomics to formalin-fixed paraffin-embedded healthy vulvar tissue (n=5) and HPV16+ high grade vulvar lesions (vHSIL, n=31). Results More than 415,000 individual cells were identified and annotated in situ, to create complete digital atlases of the healthy vulvar and HPV16+ vHSIL tissues for comparison. Two subpopulations of basal cells were identified, one of which lined the basal membrane of healthy tissue, the other is characterized by a transcriptomic signature like HPV16-infected keratinocytes and upregulated inflammatory genes and observed predominantly in vHSIL. Epithelial cells in the parabasal layers of vHSIL upregulated the expression of genes associated with inflammation and proliferation. Importantly, HPV16 profoundly remodeled stromal fibroblasts, endothelial, and immune cells, reorganizing them into distinct cellular neighborhoods of which three dominated vHSIL. A regenerative niche adjacent to the basal layer, a perilesional niche characterized by immune suppression, and a more distal niche enriched for adaptive immune activity. Conclusion In conclusion, spatial analysis reveals that HPV16 infection orchestrates a coordinated tissue response, driving basal layer restoration while locally suppressing immune activation to prevent pathological damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37831a90bf0a7b93c72da2814aa66c62626ce60" target='_blank'>
              The local cellular response to Human Papillomavirus focuses on basal layer restoration as visualized in situ by specific cellular neighborhoods near infected cells
              </a>
            </td>
          <td>
            Roderick C. Slieker, Z. Abdulrahman, Saskia Santegoets, M. V. van Poelgeest, Marij J. P. Welters, S. H. van der Burg
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c936d3b24a47f917e24b5388752187347f78add" target='_blank'>
              Advancing Precision Medicine in Malignant Pleural Mesothelioma: The Emerging Role of Patient-Derived Organoids
              </a>
            </td>
          <td>
            F. Gonnelli, F. Mei
          </td>
          <td>2025-12-01</td>
          <td>Current Pulmonology Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Emerging evidence suggests that oral squamous cell carcinoma (OSCC) harbors distinct microbial communities, yet their influence on tumor immunobiology remains unclear. This study investigated the prognostic value of intratumoral microbiota and their role in modulating CD8⁺ T-cell function. Using TCGA datasets, a 19-microorganism prognostic signature was constructed via Cox and LASSO regression and validated with Kaplan–Meier survival analysis. Comprehensive bioinformatics analyses, including GSEA/GSVA, CIBERSORT, ESTIMATE, pharmacogenomics, and mutational profiling, were performed to explore underlying mechanisms. The microbial signature demonstrated strong predictive performance, with higher risk scores significantly associated with reduced overall survival (p < 0.001). High-risk tumors exhibited enrichment of PI3K/AKT/mTOR signaling, metabolic reprogramming, and epithelial–mesenchymal transition, alongside an immunosuppressive microenvironment characterized by CD8⁺ T-cell depletion, M0 macrophage infiltration, and upregulation of immunosuppressive markers including CD276 and TGF-β1. Conversely, immune-activating checkpoints such as PD-1 and CTLA-4 were elevated in low-risk tumors. Notably, periodontal pathogens negatively correlated with immune effector activity, and TP53 mutations were more frequent in high-risk cases (82% vs. 67%). This study identified intratumoral microbial signatures as independent prognostic biomarkers and validated their reproducibility in an external cohort. Our findings support a microbiota–immune axis contributing to immune evasion in OSCC, offering novel avenues for prognostic stratification and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea87f1d3cad21e45f622b4777a7389fe481a0a6c" target='_blank'>
              Intratumoral microbiota drive immune evasion and disease progression in oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaolong Zang, Xiaoxia Li, Rongxin Sun, Xiaojie Zhang, Zhiyong Li, Zijian Cheng
          </td>
          <td>2025-12-01</td>
          <td>BDJ Open</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6172dabf52906e17a9170089cda881b60914418f" target='_blank'>
              A human multilineage gut organoid model for Parkinson disease
              </a>
            </td>
          <td>
            Mahya Hosseini Bondarabadi, Minqian Xu, Jim de Leeuw, S. Slingerland, S. van der Zee, Iris E. C. Sommer, H. Harmsen, T. van Laar, Sven C D van IJzendoorn
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [6, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>