<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-14 09:36:50 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c056a7bc97d38004639e645962dc9ff9c3cfad" target='_blank'>
              Modeling cellular influence delineates functionally relevant cellular neighborhoods in primary and metastatic pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yeonju Cho, Jae W. Lee, Sarah M. Shin, Alexei Hernandez, Xuan Yuan, Jowaly Schneider, Jody E. Hooper, Laura D. Wood, Elizabeth M. Jaffee, Atul Deshpande, W. Ho
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is characterized by high malignancy and early propensity for metastasis, and modest response to immunotherapy due to the immunosuppressive microenvironment. Surgical intervention has shown benefits in treating early-stage SCLC. However, most patients experience recurrence after surgery. The factors associated with relapse free survival in these patients remain unclear. We collected operation specimens from ten early-stage SCLC patients (N0M0), conducted long-term follow-up, and grouped them based on disease status. Subsequently, we performed a retrospective analysis using single-cell spatial imaging mass cytometry to explore the characteristics of tumor cells and differences in the tumor microenvironment, especially the single-cell constitute of immune cells, between the two groups. We found that, in early-stage SCLC, tumor cells display pronounced heterogeneity, both intra-group and inter-group. Patients with early recurrence are characterized by a distinct subpopulation of tumor cells with high Ki-67 expression. Non-relapse patients demonstrate better infiltration of M1 macrophages and stromal cells. Neighborhood analysis suggested that positive interactions between macrophages, stromal cells, and T cells with tumor cells may benefit patient prognosis. Additionally, recurrent tumor cells might enhance their metastatic capacity and remodel the microenvironment through upregulation of GranzymeB or reduction of c-Myc expression. In conclusion, SCLC tumor cells demonstrate tumor heterogeneity and microenvironmental changes in the early clinical stages. A higher proportion of M1 macrophages is associated with prolonged postoperative survival in early-stage SCLC patients. This research provides novel insights and evidence for treating and preventing postoperative recurrence in SCLC. Supplementary Information The online version contains supplementary material available at 10.1007/s10565-025-10056-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1755c9b716477f70da2c740f51d015cee2a36b" target='_blank'>
              Single-cell spatial proteomics of non-relapse small cell lung cancer identifies tumor microenvironment determinants of survival
              </a>
            </td>
          <td>
            Yin Li, L. Xia, Hui Wang, Xinghao Ai, Ying Wang, Qingquan Luo, Yuchen Han, Shun Lu, Xinghua Cheng
          </td>
          <td>2025-06-17</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bc89e4de14d8f34f2814e7cf735899a3c934813" target='_blank'>
              Spatial determinants of tumor cell dedifferentiation and plasticity in primary cutaneous melanoma
              </a>
            </td>
          <td>
            Tuulia Vallius, Yingxiao Shi, Edward Novikov, Shishir M Pant, R. Pelletier, Yu-An Chen, Juliann B. Tefft, Ajit Nirmal Johnson, Z. Maliga, Guihong Wan, George F. Murphy, S. Santagata, Yevgeniy R. Semenov, David Liu, C. G. Lian, P. Sorger
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a7baab96b7e0361b2e10254d2ab098fe969a9" target='_blank'>
              Tumor Cells Enriched for Interferon and Inflammatory Programs Pre-Exist in High Grade Serous Ovarian Cancer and are Proportionately Significantly Increased Post Chemotherapy
              </a>
            </td>
          <td>
            B. Winterhoff, Nomeda Girnius, Muyi Liu, Jaeyoung Kang, Mihir Shetty, M. A. Martinez-Gakidis, Weiyi Xu, Mayura Thomas, Linna Lahmadi, Aylin Z. Henstridge, Pilar Baldominos, Shobhana Talukdaer, Z. Chang, Jordan Mattson, Morgan Gruner, Sally A. Mullany, Peter Argenta, Gottfried E. Konecny, Dennis J. Slamon, Ronny Drapkin, Andrew C. Nelson, Timothy K. Starr, Joan S Brugge
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="A comprehensive understanding of cellular behavior and response to the tumor microenvironment (TME) in colorectal cancer (CRC) remains elusive. Here, we introduce the high-definition Visium spatial transcriptomic technology (Visium HD) and investigate formalin-fixed paraffin-embedded human CRC samples (n = 5). We demonstrate the high sensitivity, single-cell-scale resolution and spatial accuracy of Visium HD, generating a highly refined whole-transcriptome spatial profile of CRC samples. We identify transcriptomically distinct macrophage subpopulations in different spatial niches with potential pro-tumor and anti-tumor functions via interactions with tumor and T cells. In situ gene expression analysis validates our findings and localizes a clonally expanded T cell population close to macrophages with anti-tumor features. Our study demonstrates the power of high-resolution spatial technologies to understand cellular interactions in the TME and paves the way for larger studies that will unravel mechanisms and biomarkers of CRC biology, improving diagnosis and disease management strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f42a49dad6477b432ac875de1bc87a28316b76b" target='_blank'>
              High-definition spatial transcriptomic profiling of immune cell populations in colorectal cancer
              </a>
            </td>
          <td>
            Michelli F Oliveira, J. P. Romero, Meii Chung, Stephen R. Williams, Andrew D. Gottscho, Anushka Gupta, Susan E. Pilipauskas, Seayar Mohabbat, Nandhini Raman, David Sukovich, David Patterson, Sarah E B Taylor
          </td>
          <td>2025-06-05</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4028f5dcc0e86189184980c13bb56279010a29ab" target='_blank'>
              Clonal lineage tracing and parallel multiomics profiling reveal transcriptional diversification induced by ARID1A deficiency
              </a>
            </td>
          <td>
            Kenichi Miyata, Liying Yang, Yuxiao Yang, Kohei Kumegawa, R. Maruyama
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6cfe1f46dc56e476a3d0be0d700257cc9ae2152" target='_blank'>
              Single-Cell and Spatial Profiling of Tumor Microenvironment Heterogeneity in Human Osteosarcomas
              </a>
            </td>
          <td>
            Xuejing zheng, Jin Yuan, Peng Jin, Xuejun Wang, Runlei Xu, Helin Feng, Xinxin Zhang, Yongjie Xie, Wence Wu
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d556d25908485193c4e548584bab10bfa70daa1" target='_blank'>
              Oncogenic drivers shape the tumor microenvironment in human gliomas
              </a>
            </td>
          <td>
            Bo Zhao, Chae Yun Cho, Lingqun Ye, Tula Keal, Thomas Mitchell, Inés Martín-Barrio, C. A. Faraj, Mesut Unal, Jessica Alsing, Nora M Lawson, Ellie Rahm Kim, Francisco Chavez, Kristy Mendoza Rangel, Sanjay K Singh, Joseph H. McCarthy, D.M. Ashley, W. Yung, Vinay K. Puduvalli, Guy Nir, Jeffrey S. Weinberg, Linghua Wang, Keith L. Ligon, G. Genovese, Peter Van Loo, P. Futreal, Jason T. Huse, Jesse R. Dixon, Frederick F. Lang, Kadir C. Akdemir
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Background Breast cancer is a highly heterogeneous disease where variations of biomarker expression may exist between individual foci of a cancer (intra-tumoral heterogeneity). The extent of variation of biomarker expression in the cancer cells, distribution of cell types in the local tumor microenvironment and their spatial arrangement could impact on diagnosis, treatment planning and subsequent response to treatment. Methods Using quantitative multiplex immunofluorescence (MxIF) imaging, we assessed the level of variations in biomarker expression levels among individual cells, density of cell cluster groups and spatial arrangement of immune subsets from regions sampled from 38 multi-focal breast cancers that were processed using whole-mount histopathology techniques. Molecular profiling was conducted to determine the intrinsic molecular subtype of each analysed region. Results A subset of cancers (34.2%) showed intra-tumoral regions with more than one molecular subtype classification. High levels of intra-tumoral variations in biomarker expression levels were observed in the majority of cancers studied, particularly in Luminal A cancers. HER2 expression quantified with MxIF did not correlate well with HER2 gene expression, nor with clinical HER2 scores. Unsupervised clustering revealed the presence of various cell clusters with unique IHC4 protein co-expression patterns and the composition of these clusters were mostly similar among intra-tumoral regions. MxIF with immune markers and image patch analysis classified immune niche phenotypes and the prevalence of each phenotype in breast cancer subtypes was illustrated. Conclusions Our work illustrates the extent of spatial heterogeneity in biomarker expression and immune phenotypes, and highlights the importance of a comprehensive spatial assessment of the disease for prognosis and treatment planning. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02038-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185f08263c4f5dcd967322d5656a07ab8cc0083f" target='_blank'>
              Intra-tumoral spatial heterogeneity in breast cancer quantified using high-dimensional protein multiplexing and single cell phenotyping
              </a>
            </td>
          <td>
            Alison M. Cheung, Dan Wang, M. Quintayo, Yulia Yerofeyeva, Melanie Spears, John M. S. Bartlett, Lincoln Stein, Jane Bayani, Martin J Yaffe
          </td>
          <td>2025-05-21</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="The tumor microenvironment is heterogeneous, structurally complex, and continually evolving, making it difficult to fully capture. Common dissociative techniques thoroughly characterize the heterogeneity of cellular populations but lack structural context. The recent boom in spatial analyses has exponentially accelerated our understanding of the structural complexity of these cellular populations. However, to understand the dynamics of cancer pathogenesis, we must assess this heterogeneity across space and time. In this review, we provide an overview of current dissociative, spatial, and temporal analysis strategies in addition to existing and prospective spatiotemporal techniques to illustrate how understanding the tumor microenvironment, focusing on dynamic immune-cancer cell interactions, across four dimensions will advance cancer research and its diagnostic and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1f3339cce25c83c98dd45cea90c67e2a48bc606" target='_blank'>
              The tumor microenvironment across four dimensions: assessing space and time in cancer biology
              </a>
            </td>
          <td>
            Christina R. Larson, Ayushi Mandloi, Satwik Acharyya, Julienne L. Carstens
          </td>
          <td>2025-06-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b6a64a6d2ad28ecff7ab0f31663d8f3fd11b6a" target='_blank'>
              A spatial atlas of chemoradiation therapy in pancreatic cancer identifies cellular and microenvironmental determinants of persister populations
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, Ariana Acevedo-Diaz, K. Rajapakshe, Galia Jacobson, Daniela Tovar, Jimin Min, Guangsheng Pei, Candise Tat, Ayush Suresh, Ching-Wei Tzeng, Matthew HG Katz, M. Bhutani, Huaming Wang, Robert A. Wolff, C. Haymaker, E. Ludmir, Huocong Huang, Xiongfeng Chen, Liang Li, Albert C Koong, Linghua Wang, N. Navin, Dadi Jiang, Ziyi Li, Anirban Maitra, Eugene J. Koay
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83adb5995a58165fece89d2aea822be3362a5535" target='_blank'>
              Epigenetic Subtypes of High-Grade T1 Bladder Cancer Reveal Intra-Tumor Heterogeneity and Distinct Interactions with Tumor Microenvironment
              </a>
            </td>
          <td>
            J. Bellmunt, Yingtian Xie, N. Juanpere, M. Munoz, Shweta Kukreja, Sonsoles Liria Veiga, Rong Li, Xintao Qiu, Yijia Jiang, Alba Font-Tello, Marie Nunez Duarte, Ilana B. Epstein, Silvia Hernández-Llodrà, Marta Lorenzo, Silvia Menendez, T. Choueiri, Myles Brown, H. Long, Paloma Cejas
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1dc109a3a473a99af4a77b675644d925cf4104" target='_blank'>
              Oncogenic and tumor-suppressive forces converge on a progenitor-orchestrated niche to shape early tumorigenesis
              </a>
            </td>
          <td>
            José Reyes, Isabella S. Del Priore, Andrea C. Chaikovsky, Nikhita Pasnuri, Ahmed M. Elhossiny, Tobias Krause, Andrew Moorman, Catherine Snopkowski, Meril Takizawa, Cassandra Burdziak, Nalin Ratnayeke, Ignas Masillioni, Yu-jui Ho, R. Chaligné, P. Romesser, A. Filliol, T. Nawy, John P. Morris, Zhen Zhao, M. P. di Magliano, Direna Alonso-Curbelo, D. Pe’er, Scott W. Lowe
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ba430604b98f8f47f7bf5f84f30b7530579a6d" target='_blank'>
              Spatial Biology and Organoid Technologies Reveal a Potential Therapy-Resistant Cancer Stem Cell Population in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Pritha Adhikary, Jayati Chakrabarti, Jiang Wang, Ugonna Ezuma-Igwe, Xi Sun, Willian C. Bastian, Nicolas Braissand, Megan P. Corbett, Eugene F. Douglass, Sangmin Kim, Rohan Kanakamedala, Abigail G. Branch, Saptarshi Mallick, Meenu Priya Resmi, Payton Stevens, Marco Padilla-Rodriguez, Maga Sanchez, Gregory Lawrence Beatty, James Griffin, Petros G. Nikolinankos, Kellen Chen, Taylor Riall, Syed A. Ahmad, Rachna Shroff, Frédéric Hollande, Davendra P. S. Sohal, Yana Zavros
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47dca4509610b2f00715550942002b3b04fd28df" target='_blank'>
              Tracing colorectal malignancy transformation from cell to tissue scale
              </a>
            </td>
          <td>
            Helena L. Crowell, Irene Ruano, Zedong Hu, Yourae Hong, G. Caratù, Hubert Piessevaux, Ashley Heck, Rachel Liu, Max Walter, Megan Vandenberg, Kimberly Young, Dan McGuire, Evelyn Metzger, Margaret L Hoang, Joseph M. Beechem, Sabine Tejpar, A. Pascual-Reguant, Holger Heyn
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9da57971dc8950b12d4ca5ccbc0969a63be6c27" target='_blank'>
              Comparison of QuPath and HALO platforms for analysis of the tumor microenvironment in prostate cancer
              </a>
            </td>
          <td>
            Wei Zhang, Qin Zhou, Jonathan V. Nguyen, Erika Egal, Qian Yang, Michael R. Freeman, S. Hu-Lieskovan, Gita Suneja, Anna Coghill, Beatrice S Knudsen
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Circulating tumor cell (CTC) clusters are highly efficient metastatic seeds in various cancers. Yet, their genetic heterogeneity and clonal architecture is poorly characterized. Using whole-exome sequencing coupled with phylogenetic inference from CTC clusters of patients with breast and prostate cancer, as well as mouse cancer models alongside barcode-mediated clonal tracking in vivo, we demonstrate oligoclonal composition of individual CTC clusters. These results improve our understanding of metastasis-relevant clonal dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03c2f424fa808947c7f2df2ddf86dcfb503a1bd5" target='_blank'>
              Phylogenetic inference reveals clonal heterogeneity in circulating tumor cell clusters
              </a>
            </td>
          <td>
            David Gremmelspacher, Johannes Gawron, B. Szczerba, Katharina Jahn, Francesc Castro-Giner, Jack Kuipers, Jochen Singer, Francesco Marass, Ana Gvozdenovic, Selina Budinjas, Heike Pueschel, Cyril Rentsch, Alfred Zippelius, V. Heinzelmann-Schwarz, Christian Kurzeder, W. Weber, Christoph Rochlitz, Marcus Vetter, N. Beerenwinkel, Nicola Aceto
          </td>
          <td>2025-06-02</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Colorectal cancer (CRC), including both microsatellite instability (MSI) and microsatellite stability (MSS) subtypes, frequently exhibits intrinsic resistance to immunotherapy. However, the spatial tumor microenvironment (TME) and its role in distinguishing immunotherapy responders from non-responders remain poorly understood. In this study, spatial multiomics, including imaging mass cytometry (n = 50 in-house), spatial proteomics (n = 50 in-house), and spatial transcriptomics (n = 9 in-house), were employed to elucidate the spatial TME of metastatic CRC (mCRC) patients receiving immunotherapy. These methodologies were integrated with single-cell RNA sequencing (scRNA-seq), bulk RNA-seq, and bulk proteomics for comprehensive analysis and validation. A spatial immune atlas containing 314,774 cells was constructed. We found that C1QC+ resident tissue macrophages (RTMs) were more abundant in responders regardless of microsatellite status. Co-localization of C1QC+ RTMs with CD4+ T cells was observed in responders, and MHC-II expression facilitated their interaction. In contrast, cancer-associated fibroblasts inhibited this interaction in non-responders. Moreover, whole genome screening identified key genes involved in antigen presentation in C1QC+ RTMs. Hence, our study highlights the importance of spatial immune mapping in revealing the complex spatial topology of CRC and corresponding immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcb9f523addd61c5cd3fcf55f0e488cedb552f07" target='_blank'>
              Spatially resolved C1QC+ macrophage-CD4+ T cell niche in colorectal cancer microenvironment: implications for immunotherapy response
              </a>
            </td>
          <td>
            Hangyu Zhang, Libing Hong, Zhen Dong, Shan Xin, Bo Lin, Jinlin Cheng, Weihong Tian, Bin Li, Jing Wang, Xiaoyan Liu, Chuan Liu, Yuzhi Jin, Yanzhi Feng, Ge Su, Xuqi Sun, Qiqi Liu, Xiaomeng Dai, Yang Gao, Zhou Tong, Lulu Liu, Xudong Zhu, Yi Zheng, Peng Zhao, Tiannan Guo, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-07-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The spatial proteomic profiling of complex tissues is essential for investigating cellular function in physiological and pathological states. However, the imbalance among resolution, protein coverage, and expense precludes their systematic application to analyze whole tissue sections in an unbiased manner and with high resolution. Here, we introduce panoramic spatial enhanced resolution proteomics (PSERP), a method that combines tissue expansion, automated sample segmentation, and tryptic digestion with high-throughput proteomic profiling. The PSERP approach facilitates rapid quantitative profiling of proteomic spatial variability in whole tissue sections at sub-millimeter resolution. We demonstrated the utility of this method for determining the streamlined large-scale spatial proteomic features of gliomas. Specifically, we profiled spatial proteomic features for nine glioma samples across three different mutation types (IDH1-WT/EGFR-mutant, IDH1-mutant, and IDH1/EGFR-double-WT gliomas) at sub-millimeter resolution (corresponding to a total of 2,230 voxels). The results revealed over 10,000 proteins identified in a single slide, which helps us to portray the diverse proteins and pathways with spatial abundance patterns in the context of tumor heterogeneity and cellular features. Our spatial proteomic data revealed distinctive proteomic features of malignant and non-malignant tumor regions and depicted the distribution of proteins from tumor centers to tumor borders and non-malignant tumor regions. Through integrative analysis with single-cell transcriptomic data, we elucidated the cellular composition and cell–cell communications in a spatial context. Our PSERP also includes a spatially resolved tumor-specific peptidome identification workflow that not only enables us to elucidate the spatial expression patterns of tumor-specific peptides in glioma samples with different genomic types but also provides us with opportunities to select combinations of tumor-specific mutational peptides whose expression could cover the maximum tumor regions for future immune therapies. We further demonstrated that combining tumor-specific peptides might enhance the efficacy of immunotherapy in both patient-derived cell (PDC) and patient-derived xenograft (PDX) models. PSERP efficiently retains precise spatial proteomic information within the tissue context and provides a deeper understanding of tissue biology and pathology at the molecular level. Supplementary Information The online version contains supplementary material available at 10.1186/s13045-025-01710-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21c4be1dba037f0ddb640334fdf336656ba41adb" target='_blank'>
              Panoramic spatial enhanced resolution proteomics (PSERP) reveals tumor architecture and heterogeneity in gliomas
              </a>
            </td>
          <td>
            Ziyan Xu, Yunzhi Wang, Tao Xie, Rongkui Luo, Heng-Li Ni, Hang Xiang, Shaoshuai Tang, S. Tan, Rundong Fang, Peng Ran, Qiao Zhang, Xiaomeng Xu, Shan Tian, Fuchu He, Wenjun Yang, Chen Ding
          </td>
          <td>2025-05-26</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd09b27d0d8fe6fa80c372bc8924f459f7a4d33" target='_blank'>
              MOSAIC: Intra-tumoral heterogeneity characterization through large-scale spatial and cell-resolved multi-omics profiling
              </a>
            </td>
          <td>
            Caroline Hoffmann
          </td>
          <td>2025-05-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f703b1371a985c2aa75a6f16b3b87971f363e77b" target='_blank'>
              Spatial analysis reveals a novel inflammatory tumor transition state which promotes a macrophage-driven induction of sarcomatoid renal cell carcinoma
              </a>
            </td>
          <td>
            Allison M. May, Suguru Kadomoto, Claire Williams, Alex C. Soupir, Stephanie The, J. McGue, Tyler Robinson, G. Shelley, Mitchell T Hayes, Brooke L. Fridley, Jodi A. Balasi, Paolo M. Ramos Echevarria, J. Dhillon, S. Nallandhighal, Satwik Acharyya, Liying Chen, Jessica Aldous, Nathan Schurman, Veera Baladandayuthapani, Timothy L. Frankel, S. Salami, Brandon J. Manley, R. Mehra, Aaron M. Udager, Evan T. Keller
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6bec286991f2affb3f9522a83843f9469ba7b89" target='_blank'>
              Developmental Plasticity and Stromal Co-option Shape a Pituitary Neuroendocrine Tumor Transcriptional Continuum
              </a>
            </td>
          <td>
            Robert C. Osorio, Jun Y. Oh, Jangham Jung, Alexander J. Ehrenberg, A. Saha, Meeki K. Lad, Harmon Khela, Nicole Brennick, P. Giannikopoulos, William W. Seeley, Lea T. Grinberg, Aaron Diaz, M. Aghi
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 e20040


 Background:
 Immunotherapy has revolutionized cancer treatment, particularly in non-small cell lung cancer (NSCLC), yet a large fraction of patients remain refractory to treatment. Understanding treatment response requires characterization of immune cell populations and their spatial organization, as cellular neighborhoods critically influence immune cell recruitment, activation, and function. Current imaging-based and NGS-based methods struggle to achieve clear separation of individual cells in densely packed regions. Hence, we developed STAMP-seq, a novel method tagging single nuclei with cleavable spatial barcodes using high-density DNA sequencing chips, to uncover immunotherapy response-related modules and dynamic changes in NSCLC.
 Methods:
 STAMP-seq employs spatially-encoded DNA arrays to label individual nuclei with cleavable barcodes. Following validation in mouse brain tissue, we analyzed surgically resected NSCLC samples from immunotherapy-treated patients, integrating spatial transcriptomics with BCR profiling to characterize treatment response features.
 Results:
 STAMP-seq achieves precise spatial barcoding, detecting over 30,000 cells per adult mouse brain section with reliable cell subtype assignment. In immunotherapy-treated NSCLC samples, spatial analysis revealed distinct cellular communities associated with treatment response. Responding patients showed enrichment of tertiary lymphoid structures and specialized plasma cell niches, with tumor cells exhibiting elevated TNF-α signaling and reduced proliferation. These tumor cells showed proximity to

 ROBO2

 + CAFs while remaining distant from

 ACTA2

 + CAFs. BCR clonotype and pseudotime analysis revealed that

 IGHG1

 + plasma cells are recruited from TLS or vasculature by stromal-rich "transit stops" containing antigen-presenting capabilities CAFs through

 HMGB1

 -

 CXCR4

 interaction.
 Conclusions:
 STAMP-seq enables high-resolution spatial transcriptomics at single-nucleus level. Our characterization of NSCLC spatial architecture revealed previously unrecognized immune response organization, particularly how specialized stromal-rich "transit stops" coordinate cancer-suppressing plasma cell recruitment. These findings demonstrate how spatial information can advance diagnostic and prognostic module development for human diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/023df8c02dd2a154737c443a9b87fee23263bac2" target='_blank'>
              Use of STAMP-seq to enable spatial transcriptomics through single-nucleus in situ tagging and exploration of relationship of dynamic plasma cell modules and immunotherapy response in non-small cell lung cancer.
              </a>
            </td>
          <td>
            Yitong Pan, Huan Yan, Jinhuan Han, Rui Wu, Xingyong Ma, Ying Guan, Keyu Li, Qingquan Wei, Guangxin Zhang, Shaozhuo Jiao, Yongchang Zhang, Chenxi Tian
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Current approaches used to track stem cell clones through differentiation require genetic engineering1,2 or rely on sparse somatic DNA variants3,4, which limits their wide application. Here we discover that DNA methylation of a subset of CpG sites reflects cellular differentiation, whereas another subset undergoes stochastic epimutations and can serve as digital barcodes of clonal identity. We demonstrate that targeted single-cell profiling of DNA methylation5 at single-CpG resolution can accurately extract both layers of information. To that end, we develop EPI-Clone, a method for transgene-free lineage tracing at scale. Applied to mouse and human haematopoiesis, we capture hundreds of clonal differentiation trajectories across tens of individuals and 230,358 single cells. In mouse ageing, we demonstrate that myeloid bias and low output of old haematopoietic stem cells6 are restricted to a small number of expanded clones, whereas many functionally young-like clones persist in old age. In human ageing, clones with and without known driver mutations of clonal haematopoieis7 are part of a spectrum of age-related clonal expansions that display similar lineage biases. EPI-Clone enables accurate and transgene-free single-cell lineage tracing on hematopoietic cell state landscapes at scale.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/432419f2032b46caaaf3eb1aa445ee0f4fa38314" target='_blank'>
              Clonal tracing with somatic epimutations reveals dynamics of blood ageing
              </a>
            </td>
          <td>
            Michael Scherer, Indranil Singh, M. Braun, Chelsea Szu-Tu, Pedro Sanchez Sanchez, Dominik Lindenhofer, N. A. Jakobsen, Verena Körber, Michael Kardorff, Lena Nitsch, Pauline Kautz, Julia Rühle, Agostina Bianchi, Luca Cozzuto, R. Frömel, Sergi Beneyto-Calabuig, Caleb A Lareau, Ansuman T. Satpathy, Renée Beekman, Lars M Steinmetz, S. Raffel, Leif S. Ludwig, Paresh Vyas, Alejo E. Rodriguez-Fraticelli, Lars Velten
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3c75d8fc63308ed8697666b49a18333c902334" target='_blank'>
              A Single-cell and Spatially Resolved Cell Atlas of Human Esophageal Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Yong Shi, Ke An, Yu Qi, Xinhan Zhang, Yueqin Wang, Xuran Zhang, Shaoxuan Zhou, Ouwen Li, Yanan Song, Jiayi Zhou, Yue Du, Mingyang Hou, Yun-Gui Yang, QuanC. Kan, Xin Tian
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pituitary neuroendocrine tumors (PitNETs) can be invasive or aggressive, yet the mechanisms behind these behaviors remain poorly understood, impeding treatment advancements. Here, we integrat single-cell RNA sequencing and spatial transcriptomics, analyzing over 177,000 cells and 35,000 spots across 57 tissue samples. This comprehensive approach facilitates the identification of PitNETs tumor populations and characterizes the reconfiguration of the tumor microenvironment (TME) as PitNETs progress and invade. We trace the trajectory of TPIT-lineage PitNETs and identify an aggressive tumor cluster marked by elevated p53-mediated proliferation and a higher Trouillas classification, both associated with tumor progression. Additionally, we document the heterogeneity of immune stromal cells within PitNETs, particularly noting the enrichment of SPP1+ tumor associated macrophages (TAMs) in invasive tumors. These TAMs facilitate tumor invasion through the SPP1-ITGAV/ITGB1 signaling pathway. Our in-depth single-cell and spatial analysis of PitNETs uncovers the molecular dynamics within the TME, suggesting potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b1b01507bfc7f0aa4eb306817808e84f0ce1133" target='_blank'>
              Single-cell and spatial transcriptome analyses reveal tumor heterogeneity and immune remodeling involved in pituitary neuroendocrine tumor progression
              </a>
            </td>
          <td>
            Wan Su, Zhang Ye, Jifang Liu, K. Deng, Jinghua Liu, Huijuan Zhu, Lian Duan, Chen Shi, Linjie Wang, Yuxing Zhao, Fengying Gong, Yi Zhang, Bo Hou, Hui You, Feng Feng, Qing Ling, Yu Xiao, Yongdong Guo, Wenyi Fan, Sumei Zhang, Zixin Zhang, Xiaomin Hu, Yong Yao, Chunhong Zheng, Lin Lu
          </td>
          <td>2025-05-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90f49812ba9bf54efb86922eb7c7255926faf91" target='_blank'>
              Mapping and reprogramming microenvironment-induced cell states in human disease using generative AI
              </a>
            </td>
          <td>
            Amir Akbarnejad, Lloyd Steele, D. Jafree, Sebastian Birk, M. Sallese, K. Rademaker, Adam Boxall, Ben Rumney, Catherine Tudor, Minal Patel, M. Prete, Stanislaw Makarchuk, Tong Li, Heather Stanley, A. Foster, Kenny Roberts, Andrew L. Trinh, C. E. Villa, G. Testa, S. Mahil, Arash Mehrjou, Catherine Smith, Sattar Vakili, M. Clatworthy, Thomas J. Mitchell, O. Bayraktar, M. Haniffa, M. Lotfollahi
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1554b6f0212ba05651f81096c5d01da1508bdc40" target='_blank'>
              Cell simulation as cell segmentation.
              </a>
            </td>
          <td>
            Daniel C Jones, Anna E. Elz, Azadeh Hadadianpour, H. Ryu, David R. Glass, Evan W. Newell
          </td>
          <td>2025-05-22</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90517c7621a56cfd44f51891279305813cf3e121" target='_blank'>
              Notch3 regulates pericyte phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            N. Chalkidi, Athanasia Stavropoulou, V. Arvaniti, Christina Paraskeva, Artemis Monogyiou, Maria Sakkou, Christoforos Nikolaou, V. Koliaraki
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e652ee8a36b3b3637d56b8e5667a1abc73cc2445" target='_blank'>
              Differential Assembly of Mouse and Human Tumor Microenvironments
              </a>
            </td>
          <td>
            T. Courau, Rebecca G. Jaszczak, B. Samad, Emily Flynn, Nayvin W. Chew, Gabriella C. Reeder, Jessica Tsui, A. Ray, Harrison Wismer, Daniel Bunis, Leonard C. Lupin-Jimenez, Noah V. Gavil, David Masopust, John P Graham, Daniel A. Skelly, Xavier Vesco, Edison T. Liu, G. Fragiadakis, A. Combes, Matthew F. Krummel
          </td>
          <td>2025-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Given the graft-versus-leukemia effect observed with allogeneic hematopoietic stem cell transplantation in refractory or relapsed acute myeloid leukemia (AML), immunotherapies have been explored in nontransplant settings. We applied a multiomic approach to examine bone marrow interactions in patients with AML treated with pembrolizumab and decitabine. Using extensively trained nuclear and membrane segmentation models, we achieved precise transcript assignment and deep learning–based image analysis. To address read-depth limitations, we integrated single-cell RNA sequencing with single-cell spatial transcriptomics from the same sample. Quantifying cell-cell distances at the edge level enabled more accurate tumor microenvironment analysis, revealing global and local immune cell enrichment near leukemia cells postpembrolizumab treatment, potentially linked to clinical response. Furthermore, ligand-receptor analysis indicated potential alterations in specific signaling pathways between leukemia and immune cells following immunotherapy treatment. These findings provide insights into immune interactions in AML and may inform therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0066efa7bc3f52352d23a57bb535219a1adb0cf1" target='_blank'>
              Single-cell spatial transcriptomics reveals immunotherapy-driven bone marrow niche remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R. Herzog, Abigail Wong-Rolle, Laura W. Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann F. Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine R. Calvo, Christopher S. Hourigan, Kasper D. Hansen, Chen Zhao
          </td>
          <td>2025-07-09</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) is emerging as a powerful technology that transforms our understanding of thyroid cancer by offering a spatial context of gene expression within the tumor tissue. In this review, we synthesize the recent applications of ST in thyroid cancer research, with a particular focus on the heterogeneity of the tumor microenvironment, tumor evolution, and cellular interactions. Studies have leveraged the spatial information provided by ST to map distinct cell types and expression patterns of genes and pathways across the different regions of thyroid cancer samples. The spatial context also allows a closer examination of invasion and metastasis, especially through the dysregulation at the tumor leading edge. Additionally, signaling pathways are inferred at a more accurate level through the spatial proximity of ligands and receptors. We also discuss the limitations that need to be overcome, including technical limitations like low resolution and sequencing depth, the need for high-quality samples, and complex data handling processes, and suggest future directions for a wider and more efficient application of ST in advancing personalized treatment of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3b3022cb800cbef42ede8e65012209b885afa4" target='_blank'>
              Spatial Transcriptomics in Thyroid Cancer: Applications, Limitations, and Future Perspectives
              </a>
            </td>
          <td>
            Chaerim Song, Hye-Ji Park, Man S. Kim
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Developing high-resolution reference maps of disease-susceptible spatial niches is a critical step to mitigating the profound effects of lung disease. Here, we present an integrated multimodal single-nucleus human lung atlas (snHLA) profiling 746,047 nuclei from 49 mapped lung blocks spanning clinically relevant distal airways, alveoli, and interstitium across 11 healthy adults. Integrating snRNA-seq and SNARE-seq2, which co-assays chromatin accessibility and gene expression from the same nucleus, we resolved 70 molecularly distinct populations and captured 332,846 accessible chromatin regions, nominating new transcriptional regulators of human lung cell diversity. Spatial transcriptomics using MERFISH mapped 25 cell populations across 7 structural neighborhoods and multiplexed immunofluorescence localized cell subtypes and distal airway-defining protein markers, expanding and validating distinct lung structure-specific cell populations. This open access snHLA and companion Cell Type and Marker Gene Dictionary with anatomically aligned nomenclature delivers a foundational resource at an unprecedented resolution to interrogate the origins of lung pathophysiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/085ae835dd14cf9157e86d76e8875d20b81277fe" target='_blank'>
              A Multimodal Spatial and Epigenomic Atlas of Human Adult Lung Topography
              </a>
            </td>
          <td>
            T. Duong, D. Diep, Kimberly Y. Conklin, Indy Bui, J. Purkerson, Eric Boone, Jacqueline Olness, Sahil Patel, Beverly Peng, Colin Kern, Zoey Zhao, Ravi S. Misra, H. Huyck, J. Verheyden, Zea Borok, Yun Zhang, R. Scheuermann, Quan Zhu, Gail Deutsch, Jim Hagood, Xin Sun, Kun Zhang, Gloria S. Pryhuber
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d59997cf6c8e6c97ba758d271ce928cfa49100ba" target='_blank'>
              Transcriptomic Profiles from Normal and Tumor Tissue Samples Reveal Distinct Venule Populations and Novel Tumor Endothelial Cell Markers in Breast Cancer
              </a>
            </td>
          <td>
            K. Phoenix, Vijender Singh, Patrick A. Murphy, Kevin P. Claffey
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e14574


 Background:
 Understanding cellular interactions within the tumor microenvironment (TME) is essential for elucidating disease progression and advancing immunotherapy. The TME is a complex ecosystem composed of cells with dysregulated metabolism immune response and signaling pathways, which in turn influence tumor development and treatment response. Multiplexed assessment serves as an important tool for clinical oncology. The simultaneous readout of multiple processes can provide biological insights and elucidate disease mechanism. Imaging Mass Cytometry (IMC) is a spatial biology imaging technique that utilizes CyTOF technology and enables deep characterization of the diversity and complexity of the TME. IMC technology offers scalable, high-throughput acquisition while generating high-quality data with true dynamic range of signal without amplification or fluorescence-based limitations such as spectral overlap and autofluorescence.
 Methods:
 To study cellular processes and their roles in tumor progression, we utilized the Human Cell Metabolism and Human Cell Signaling Panels to investigate energy production, cellular homeostasis and mitogenic signaling pathways. We then mapped these processes to the types of cells in the TME by using the Human Immuno-Oncology IMC Panel and the Human T Cell Exhaustion IMC Panel or the Maxpar Neuro Phenotyping IMC Panel Kit to characterize immune cell and neurological phenotypes in detail. We first acquired data using Preview Mode to assess the whole tissue, followed by higher-resolution imaging of selected regions of interest using Cell Mode or of the whole tissue section using Tissue Mode.
 Results:
 Our data analysis revealed significant insights into the spatial organization and metabolic profile of cells across cancer tissues. Elevated glycolysis and mTOR pathway activation suggested adaptations to hypoxia and anabolic growth in tumor areas, while interactions between fibroblasts and immune cells highlighted crosstalk within the TME. Our Neuro Phenotyping Panel was used to reveal potential for immune response in glioblastoma. Unsupervised pixel clustering and hierarchical clustering using MCD SmartViewer highlighted metabolic activity and activation of signaling pathways within tumor regions.
 Conclusions:
 Comprehensive spatial biology profiling using the IMC approach highlights the interconnected roles these pathways play in promoting tumor survival and resistance to therapies. These findings, which illuminate the metabolic and signaling heterogeneity of the TME, are crucial for developing future prognostic assessments and have the potential to guide more effective, personalized cancer therapies. For Research Use Only. Not for use in diagnostic procedures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b16079bb313271e366d31125c0f7c2617d3a11a" target='_blank'>
              Spatial profiling of solid tumor microenvironment using imaging mass cytometry with high-plex panels.
              </a>
            </td>
          <td>
            T. Pfister, J. Y. Chwee, Nick Zabinyakov, Qanber Raza, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Solid cancers are complex 'ecosystems' comprised of diverse cell types and extracellular molecules, where heterotypic interactions significantly influence disease etiology and therapeutic response. However, our current understanding of tumor microenvironments (TMEs) remains incomplete, hindering the development and implementation of novel TME-targeted drugs. To maximize cancer therapeutic development we require a systems-level understanding of the malignant, stromal, and immune states that define the tumor and determine treatment response. In their recent study, Liu and colleagues took a new approach to resolving the complexity of stromal heterogeneity. They leveraged extensive single-cell spatial multi-omics datasets across various cancer types and platforms to identify four conserved spatial Cancer-Associated Fibroblast (CAF) subtypes, classified by their spatial organization and cellular neighborhoods. Their work expands upon prior efforts to develop a CAF taxonomy, which primarily relied on single-cell RNA-Sequencing (scRNA-Seq) and yielded a multitude of classification systems. This study advances our understanding of CAF biology by establishing a link between spatial context and CAF identity across diverse tumor types. Departing from recent single-cell transcriptomics studies that employed a marker-based approach for sub-state identification, Liu and colleagues conducted de novo discovery of CAF subtypes using spatial neighborhood information alone. By positioning spatial organization as the defining axis of CAF heterogeneity, this research redefines CAF classification and provides a new framework for exploring the rules governing tumor ecosystems and developing novel ecosystem-based therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940389ee957141f2833e0d9de53711c0ead3a30c" target='_blank'>
              Pan-Cancer Spatial Profiling Reveals Conserved Subtypes and Niches of Cancer-Associated Fibroblasts.
              </a>
            </td>
          <td>
            Hani Jieun Kim, Travis Ruan, Alex Swarbrick
          </td>
          <td>2025-05-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Disease heterogeneity across a diverse patient cohort poses challenges to cancer drug development due to inter-patient variability in treatment responses. However, current preclinical models fail to depict inter-patient tumor heterogeneity, leading to a high failure rate when translating preclinical leads into clinical successes. Methods: We integrated the expression profiles of prostate cancer (PC) lines and castration-resistant PC (CRPC) patient tumors to identify cell-lines that transcriptomically match distinct tumor subtypes in a clinical cohort. Representative cell-lines were co-cultured to create “mixed-cell” models depicting inter-patient heterogeneity in CRPC, which were employed to assess drug combinations. Results: When drug combinations previously tested in CRPC clinical cohorts, were assessed to establish proof-of-concept, in-vitro responses measured in our models concurred with their known clinical efficacy. Additionally, novel drug combinations computationally predicted to be efficacious in heterogeneous tumors, were evaluated. They demonstrated preclinical efficacy in the mixed-cell models, suggesting they will likely benefit heterogeneous patient cohorts. Furthermore, we showed that the current practice of screening cell-lines/xenografts separately and aggregating their responses, failed to detect their efficacy. Conclusions: We believe that the application of our models will enhance the accuracy of preclinical drug assessment, thereby improving the success rate of subsequent clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4d639da8c86d125f39c0d18727d1ba6b4439891" target='_blank'>
              Novel mixed cancer-cell models designed to capture inter-patient tumor heterogeneity for accurate evaluation of drug combinations
              </a>
            </td>
          <td>
            Sampreeti Jena, Daniel Kim, Adam Lee, Weijie Zhang, Kevin Zhan, Yingming Li, S. Dehm, R. S. Huang
          </td>
          <td>2025-05-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ac7f496e193e8ea2a2dad30237740bdee230d9" target='_blank'>
              Multiscale modeling predicts dependence of mesenchymally transitioned tumor niche fitness on cell-cell and cell-matrix adhesions
              </a>
            </td>
          <td>
            C. V. Sai Prasanna, M. Jolly, R. Bhat
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Lymphomas are highly heterogeneous malignancies characterized by diverse genetic, epigenetic, and phenotypic profiles, which complicate diagnosis, prognosis, and treatment. Traditional bulk sequencing methods mask the complexity of intratumoral variation, often overlooking rare subclones that may drive disease progression and therapeutic resistance. Single-cell genomics has emerged as a transformative approach to decipher tumor heterogeneity at unprecedented resolution. This technology enables the dissection of individual cellular populations within lymphomas, offering insights into clonal evolution, transcriptional diversity, and microenvironmental interactions. By applying single-cell RNA sequencing (scRNA-seq), chromatin accessibility assays (scATAC-seq), and single-cell DNA sequencing (scDNA-seq), researchers can unravel lineage relationships, identify resistant subpopulations, and track dynamic changes in response to therapy. In lymphomas such as diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL), and mantle cell lymphoma (MCL), single-cell approaches have revealed distinct malignant and non-malignant cell states that correlate with treatment outcomes. Moreover, integrating single-cell data with spatial transcriptomics and immune profiling enhances the understanding of the tumor microenvironment, including immune evasion mechanisms. These insights can inform personalized treatment strategies, identify novel therapeutic targets, and enable early detection of relapse. Despite technical challenges such as data complexity, sample viability, and cost, the application of single-cell genomics in lymphoma research is rapidly advancing. Future directions include multi-omics integration, real-time patient monitoring, and clinical translation of predictive biomarkers. This review underscores the pivotal role of single-cell genomics in resolving tumor heterogeneity and predicting therapeutic resistance, positioning it as a cornerstone for next-generation precision oncology in lymphomas,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/438856abbab164e715372bfe3f2a6c07df9594ec" target='_blank'>
              Role of mutational signatures and clonality assessments in tailoring targeted therapies for lymphoma
              </a>
            </td>
          <td>
            Andoh Sheikh Atta-ullah, Uche Philip, Sampson Janice Candy
          </td>
          <td>2025-05-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4036914e070de84fceb3196161df4ff74c7602d" target='_blank'>
              Predicting targeted- and immunotherapeutic response outcomes in melanoma with single-cell Raman Spectroscopy and AI
              </a>
            </td>
          <td>
            Kai Chang, Mamatha Serasanambati, Baba Ogunlade, Hsiu-Ju Hsu, James Agolia, Ariel Stiber, Jeffrey Gu, Saurabh Sharma, Jay Chadokiya, Amanda Gonçalves, Fareeha Safir, Nhat Vu, Daniel Delitto, Amanda R. Kirane, Jennifer A. Dionne
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f083d96ace3eccacc20f8f51f00bb48faf2731" target='_blank'>
              Pan-cancer prediction of tumor immune activation and response to immune checkpoint blockade from tumor transcriptomics and histopathology
              </a>
            </td>
          <td>
            S. Mukherjee, Sumeet Patiyal, L.R. Pal, Tiangen Chang, Sumona Biswas, S. R. Dhruba, Amos Stemmer, Arashdeep Singh, Abbas Yousefi-Rad, Tien-Hua Chen, Binbin Wang, Denis Marino, Wonwoo Shon, Yuan Yuan, Mark Faries, Omid Hamid, Karen L. Reckamp, B. Waissengrin, Beatriz Ornelas, P. Chu, Salah Boudjadi, L. Ley, Dilara Akbulut, N. El Ahmar, S. Signoretti, D. Braun, Hyunjeong Joo, Hyungsoo Kim, A. Osipov, R. Figlin, J. Bar, I. Barshack, Chi-Ping Day, Karine Sargsyan, A. Apolo, Kenneth D Aldape, Muh-Hwa Yang, Michael B. Atkins, Ze’ev A. Ronai, Danh-Tai Hoang, E. Ruppin
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Summary Advances in single‐cell and spatial omics technologies have revolutionised biology by revealing the diverse molecular states of individual cells and their spatial organization within tissues. The field of plant biology has widely adopted single‐cell transcriptome and chromatin accessibility profiling and spatial transcriptomics, which extend traditional cell biology and genomics analyses and provide unique opportunities to reveal molecular and cellular dynamics of tissues. Using these technologies, comprehensive cell atlases have been generated in several model plant species, providing valuable platforms for discovery and tool development. Other emerging technologies related to single‐cell and spatial omics, such as multiomics, lineage tracing, molecular recording, and high‐content genetic and chemical perturbation phenotyping, offer immense potential for deepening our understanding of plant biology yet remain underutilised due to unique technical challenges and resource availability. Overcoming plant‐specific barriers, such as cell wall complexity and limited antibody resources, alongside community‐driven efforts in developing more complete reference atlases and computational tools, will accelerate progress. The synergy between technological innovation and targeted biological questions is poised to drive significant discoveries, advancing plant science. This review highlights the current applications of single‐cell and spatial omics technologies in plant research and introduces emerging approaches with the potential to transform the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd263d7989d8d7fac3ccdc19b4b2541ba1d90c6a" target='_blank'>
              Exploring the untapped potential of single‐cell and spatial omics in plant biology
              </a>
            </td>
          <td>
            T. Nobori
          </td>
          <td>2025-05-21</td>
          <td>The New Phytologist</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Nasopharyngeal carcinoma (NPC) is a common malignancy with a complex pathogenesis and diverse cellular composition. This study aimed to systematically analyze the transcriptional characteristics of different cell types in NPC and their roles in tumor development using single-cell transcriptomics. Methods The research team collected NPC and normal tissue samples, and performed in-depth single-cell sequencing analysis of the transcriptomes. Single-cell RNA sequencing was performed on the collected samples to obtain high-resolution transcriptional profiles of individual cells. This allowed the researchers to identify and characterize the diverse cell populations present within the NPC tumor microenvironment. Results The results showed that NPC samples contained multiple distinct cell subpopulations, including epithelial cells, immune cells (such as macrophages and T cells), endothelial cells, and stromal cells. These cell types exhibited marked differences in spatial distribution and transcriptional profiles, reflecting the high degree of heterogeneity within the tumor microenvironment. Further functional analysis revealed significant dysregulation of mitochondrial-related pathways, extracellular matrix-receptor interactions, as well as Wnt, Notch, and other signaling cascades in NPC. Conclusion This study employed single-cell transcriptomics to comprehensively elucidate the complexity of the NPC tumor microenvironment, providing new insights into the underlying mechanisms of disease pathogenesis and suggesting potential therapeutic targets. These findings lay the groundwork for the development of precision medicine approaches for NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9793611f5914170a344f624509f87b1fa27aa37" target='_blank'>
              Single-cell transcriptomic landscape reveals complex cellular heterogeneity and dysregulated signaling pathways in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Xiaoyan Zhang, Zuojia Guan, Zhan Xu
          </td>
          <td>2025-05-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy with extremely poor prognosis. SCLC cells exhibit high plasticity and can progress from neuroendocrine (NE) to non-NE phenotypes. This dynamic evolution promotes treatment resistance and relapses, representing a challenge for targeted therapies in this elusive disease. Here we identify the transcription factor ONECUT2 (OC2) as a driver of plasticity in SCLC, leading to non-NE transcriptional states. OC2 is highly expressed in SCLC tumors compared to normal lung tissue and its expression is associated with heightened clinical stage and lymph node metastasis. We show that OC2 is a repressor of ASCL1, the NE master regulator transcription factor. In addition, OC2 upregulates non-NE programs through activation of c-MYC and Notch signaling. We also demonstrate that OC2 is required for growth and survival of SCLC cells and that it can be targeted with a small molecule inhibitor that acts synergistically with the standard combination of cisplatin and etoposide, providing a novel therapeutic strategy for OC2 active SCLC tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01267-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736f50c299b3fe9c13bfbd3d67e9601f1567eb2" target='_blank'>
              ONECUT2 reprograms neuroendocrine fate and is an actionable therapeutic target in small cell lung cancer
              </a>
            </td>
          <td>
            Mirian Gutiérrez, Irene Zamora, Raquel Iriarte, M. Pajares, Qian Yang, Chen Qian, N. Otegui, J. Fernández-Irigoyen, Enrique Santamaria, N. Alcala, A. Sexton-Oates, L. Fernandez-Cuesta, Miguel Barajas, Alfonso Calvo, Luis M. Montuenga, Beatrice S. Knudsen, Sungyong You, Michael R. Freeman, Ignacio J. Encío, Mirja Rotinen
          </td>
          <td>2025-06-11</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Breast cancer brain metastases (BCBM) affect nearly 90,000 patients annually in the United States and carry a significant risk of mortality. As metastatic lesions develop, the unique milieu of the brain microenvironment shapes disease progression and therapeutic response. Among resident brain cells, astrocytes are both the most common, and are increasingly recognized as key regulators of this process, yet their precise role remains poorly defined. Here, we present a hybrid agent-based model (ABM) to simulate tumor–astrocyte interactions on a two-dimensional lattice. In our model, metastatic tumor cells induce phenotypic reprogramming of astrocytes from an antito a pro-metastatic state, thereby enhancing tumor proliferation. We systematically evaluate how variations in astrocyte density, spatial distribution, and chemotherapy impact tumor expansion and spatial morphology, quantified by fractal dimension, lacunarity, and eccentricity. Our simulations reveal that astrocyte reprogramming accelerates tumor progression and contributes to increased morphological complexity and chemotherapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331a7ee541e9213b60f3df2d93320f1bf057f75f" target='_blank'>
              Astrocyte reprogramming drives tumor progression and chemotherapy resistance in agent-based models of breast cancer brain metastases
              </a>
            </td>
          <td>
            Rupleen Kaur, R. Barker-Clarke, Andrew Dhawan
          </td>
          <td>2025-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Organotropism results from the functional versatility of metastatic cancer cells to survive and proliferate in diverse microenvironments. This adaptivity can originate in clonal variation of the spreading tumor and is often empowered by epigenetic and molecular reprogramming of cell regulatory circuits. Related to organotropic colonization of metastatic sites are environmentally-sensitive, differential responses of cancer cells to therapeutic attack. Accordingly, understanding the organotropic profile of a cancer and probing the underlying driver mechanisms are of high clinical importance. However, determining systematically the organotropism of one cancer versus the organotropism of another cancer, potentially with the granularity of comparing the same cancer type between patients or tracking the evolution of a cancer in a single patient for the purpose of personalized treatment, has remained very challenging. It requires a host organism that allows observation of the spreading pattern over relatively short experimental times. Moreover, organotropic patterns often tend to be statistically weak and superimposed by experimental variation. Thus, an assay for organotropism must give access to statistical powers that can separate ‘meaningful heterogeneity’, i.e., heterogeneity that determines organotropism, from ‘meaningless heterogeneity’, i.e., heterogeneity that causes experimental noise. Here we describe an experimental workflow that leverages the physiological properties of zebrafish larvae for an imaging-based assessment of organotropic patterns over a time-frame of 3 days. The workflow incorporates computer vision pipelines to automatically integrate the stochastic spreading behavior of a particular cancer xenograft in tens to hundreds of larvae allowing subtle trends in the colonization of particular organs to emerge above random cell depositions throughout the host organism. We validate our approach with positive control experiments comparing the spreading patterns of a metastatic sarcoma against non-transformed fibroblasts and the spreading patterns of two melanoma cell lines with previously established differences in metastatic propensity. We then show that integration of the spreading pattern of xenografts in 40 – 50 larvae is necessary and sufficient to generate a Fish Metastatic Atlas page that is representative of the organotropism of a particular oncogenotype and experimental condition. Finally, we apply the power of this assay to determine the function of the EWSR1::FLI1 fusion oncogene and its transcriptional target SOX6 as plasticity factors that enhance the adaptive capacity of metastatic Ewing sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5025875828f543583bc1d12169d95499fa95592" target='_blank'>
              Robust statistical assessment of Oncogenotype to Organotropism translation in xenografted zebrafish
              </a>
            </td>
          <td>
            D. Saucier, Xuexia Jiang, Divya Rajendran, Roshan Ravishankar, Erin Butler, A. Marchetto, R. Kurmasheva, T. G. Grünewald, James F. Amatruda, G. Danuser
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acab5ca78a81e16038185b4373a584725e52ba21" target='_blank'>
              Low-branching vessel architecture shapes immune cell niches and predicts immune responses in renal cancer
              </a>
            </td>
          <td>
            M. Toma, Yangping Li, Melina Kehl, Tim Kempchen, L. Esser, Katharina Baschun, S. Leonardelli, Thomas Pinetz, R. Turiello, M. Yong, Natalie Pelusi, Guillermo Altamirano-Escobedo, Eduardo Bayro-Corrochano, Sebastian Kadzik, M. Eckstein, Lukas Flatz, Fabian Hörst, J. Kleesiek, J. Saal, Glen Kristiansen, Manuel Ritter, J. Ellinger, Viktor Grünwald, Niklas Klümper, Alexander Effland, M. Hölzel
          </td>
          <td>2025-06-06</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c327cba87285dc890d1f7b106ff92f090a08a250" target='_blank'>
              Single-Cell Analysis of Meningiomas Reveals Mutation-Associated Tumor and Immune Cell Gene Expression Programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus I Anzaldua-Campos, Gregory J Zipfel, Albert H. Kim, J. Dowling, Eric C. Leuthardt, Joshua W. Osbun, A. Vellimana, Michael R. Chicoine, Gavin P. Dunn, Allegra A. Petti
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d1c1b28cd57c3d7a9126fb0efbcbda9de4833e" target='_blank'>
              The genomic and transcriptional landscape of the spinal cord H3K27-altered diffuse midline glioma suggests the potential therapeutic strategy
              </a>
            </td>
          <td>
            Yiwei Xiao, Mengyao Li, Qiang Gao, Zhihong Qian, Yukui Shang, Linkai Jing, Zili Zhen, Yong Ai, Guihuai Wang, Kehkooi Kee, Wei Zhang
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The most common cause of death in neurofibromatosis type 1 (NF1) is the development of malignant peripheral nerve sheath tumor (MPNST), a deadly sarcoma that can transform from benign plexiform neurofibromas (PN) or premalignant atypical neurofibromas (AN). We built a single-cell dataset of 55 NF1-associated PN, AN, and MPNST to define cellular changes in neurofibroma at-risk of malignant transformation. Integrative analysis of changes in the tumor microenvironment revealed the emergence of malignant tumor cells, regulatory T cells, and loss of activated macrophages in MPNST. Using this reference dataset, we validated findings using anchor-based label transfer in an additional 19 NF1 nerve sheath tumors profiled with single cell sequencing, and public datasets. We then defined protein biomarkers of malignant transformation from high-throughput proteomic analysis of plasma samples collected from 45 NF1 patients that correlated to mRNAs specific to MPNST cell populations. Fifty plasma proteins accurately and non-invasively distinguished patients with MPNST from those with premalignant tumors. These markers should improve the ability to identify high-risk neurofibromas for improved cancer surveillance and enable early detection of malignant transformation in NF1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea5d7e2095827b87208bdc18119d43a501ce00d" target='_blank'>
              Single-cell tumor microenvironment profiling informs a circulating proteome test for the interception of malignant transformation in NF1 nerve sheath tumors
              </a>
            </td>
          <td>
            Jack Shern
          </td>
          <td>2025-06-18</td>
          <td>Research Square</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e12551


 Background:
 Racial and ethnic disparities in breast cancer mortality persist across breast cancer subtypes, and variations in immune response may contribute. Single-cell spatial profiling enables detailed insights into the tumor microenvironment.
 Methods:
 We leveraged a cohort of young, racially and ethnically diverse women enrolled from 1995-2005 in the population-based Northern California Breast Cancer Family Registry (NC-BCFR). We determined feasibility of imaging-based spatial profiling on NC-BCFR TMAs with CosMx 6K SMI single-cell spatial transcriptomics assay, evaluating 140 samples from 88 women. Cells were stained with nuclear and membrane markers, imaged, and segmented with internal algorithms. Cell-level transcriptomic data of 6519 genes were normalized across the TMA and integrated with SCTransform. Manual cell typing was performed using publicly available annotations. Cell type composition for each sample for immune, fibroblast, cancer epithelium, and normal epithelium, normalized to 100 tumor cells/sample, was validated using predictions from matched H&E stained slides annotated with the HoverNet deep learning algorithm. Spatial transcriptomics data were used to predict HER2 and hormone status (HR). The immune microenvironment was examined across race, ethnicity and breast cancer subtype.
 Results:
 Race and ethnicity of 88 NC-BCFR participants were as follows: 16 Black, 22 Asian, 34 Hispanic, and 16 non-Hispanic (NH) White women. Profiles revealed 917,804 cells with mean transcript per cell of 838 (range: 517-1141). Following QC, normalization, and integration, 867,237 cells were typed across diverse phenotypes. With cell type normalized to 100 tumor cells/sample, there was a trend toward NH White samples having increased immune infiltrate vs. other racial groups, within both SMI (NH White 158, Black 116, Hispanic 89, Asian 55.3; ANOVA p = 0.263) and HoverNet (NH White 40.2, Black 24.9, Asian 29.1, Hispanic 18.9; ANOVA p = 0.117). Samples from NH White women had higher B-cell infiltrates (ANOVA p = 0.165) and macrophage content (ANOVA p = 0.000753) compared to cores from Asian, Black, and Hispanic women. Similar trends were observed in NK-cells (ANOVA p = 0.643) and T-cell infiltrates (ANOVA p = 0.318), and tertiary lymphoid structures were identified in one sample.
 Conclusions:
 Single-cell spatial transcriptomics can characterize the tumor immune microenvironment in TMAs from the 1990s, thus leveraging a highly diverse, population-based cohort study with long-term follow-up for novel, clinically relevant discoveries.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e856f98b36026a70f6d6b068a77be4a5031069b" target='_blank'>
              Characterization of breast tumors and the tumor immune landscape in young, diverse participants of the Northern California Breast Cancer Family Registry.
              </a>
            </td>
          <td>
            Marni Blair McClure, Lise Mangiante, Brennan G. Simon, Clemens L. Weiss, Zhicheng Ma, Jocelyn Koo, J. Ransohoff, E. John, Christina Curtis, Jennifer L Caswell-Jin, A. Kurian
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 8539


 Background:
 Immune checkpoint inhibitors have greatly improved outcomes in advanced non-small cell lung cancer (NSCLC). However, patients with EGFR mutant NSCLC have a poor response to immune checkpoint inhibitor therapy. Emerging evidence suggests that an immunosuppressive tumor microenvironment plays an important role in this setting, however, we still lack fundamental knowledge about tumor endothelial cell biology. We hypothesized that oncogene specific changes in the expression of immune-related genes in tumor endothelial cells account for differences in the tumor microenvironment and efficacy of immune checkpoint inhibitor therapy.
 Methods:
 We utilized spatial transcriptomics (GeoMx Digital Spatial Profiling) on resected tumor tissue of EGFR mutant (n = 5) and KRAS mutant (n = 5) NSCLC patients to investigate the transcriptional signature of tumor, endothelial and stromal cells. Additionally, we used NicheNet to explore intercellular communication between tumor, endothelial, and stromal cells. Immune gene set enrichment analysis scores were calculated using ESTIMATE. Immune cell type proportions were estimated using CIBERSORT.
 Results:
 Using spatial transcriptomics, we dissected the tumor microenvironment into tumor, stromal and endothelial compartments. By analyzing predicted cellular communication, we found that tumor and stromal cells primarily affect an interferon-related gene signature in tumor endothelial cells. Notable differentially expressed interferon-related genes, that were significantly up-regulated in KRAS mutant and down-regulated in EGFR mutant NSCLC patients, included CXCL9, STAT1, WARS1, IRF1 and ICAM1. In the stromal compartment, immune gene set enrichment analysis scores were significantly lower in EGFR mutant than KRAS mutant NSCLC (median, 355 vs. 713, P = 0.026) indicating an immunosuppressive tumor microenvironment. We observed substantial heterogeneity while exploring the cellular landscape of the stromal compartment in EGFR mutant and KRAS mutant NSCLC. Notably, we found a significantly decreased proportion of pro-inflammatory macrophages in the stromal compartment of EGFR compared to KRAS mutant NSCLC (P = 0.006).
 Conclusions:
 We identified distinct interferon-related gene signatures in tumor endothelial cells of patients with EGFR and KRAS mutantNSCLC associated with variations in the cellular composition of the tumor microenvironment. This may provide a better understanding for the development of spatial biomarkers to identify which oncogene-driven NSCLC patients are most likely to benefit from immune checkpoint inhibitor therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3c21a018e08000029baaeb3ba8f18554c44ba0f" target='_blank'>
              Spatial transcriptomic profiling of the tumor microenvironment in EGFR and KRAS mutant non-small cell lung cancer.
              </a>
            </td>
          <td>
            F. J. Bolte, Nick Natale, Samantha DiBenedetto, Margaret Moore, E. Stelow, Richard Hall, D. Gioeli
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background: Gallbladder cancer (GBC) is a rare but aggressive malignancy, often detected late due to early asymptomatic stages. Understanding cellular and molecular changes from normal tissue to high-grade intraepithelial neoplasia (HGIN) and invasive GBC is vital for identifying early biomarkers and therapeutic targets. Methods: We performed single-cell RNA sequencing on 98,113 cells derived from 2 normal adjacent tissues (NAT), 2 HGIN, and 6 GBC samples. The cellular diversity and heterogeneity, particularly within epithelial and immune cell populations in NAT–HGIN–GBC, were investigated utilizing single-cell RNA sequencing, bulk RNA sequencing (bulk RNA-seq), and 10 machine learning methodologies. Furthermore, the intercellular crosstalk between epithelial cells and tumor immune microenvironment cells was examined and validated through multiplex immunofluorescence staining. Results: The constructed cell atlas elucidated alterations in the immune landscape across various states of NAT–HGIN–GBC, highlighting a more pronounced inhibitory immune microenvironment in GBC. The epithelial subtype TOP2A+ Epi is markedly elevated in GBC and is correlated with a poor prognosis. Key genes associated with this subtype may include GMNN, CYTOR, KLK6, and BIRC5. Similarly, immunosuppressive macrophages, identified as TOP2A+ Macro, also increase along the NAT–HGIN–GBC sequence and are linked to reduced patient survival. Furthermore, TOP2A+ Macro and CD8+ exhausted T cells (CD8+ Tex) engage in intercellular communication with epithelial TOP2A+Epi cells via the TWEAK/FN14 signaling pathway, thereby promoting tumor progression and immune evasion in GBC. The findings were further corroborated through multiplex immunofluorescence staining conducted on specimens from patients. Conclusions: This study elucidates significant alteration in the cellular ecosystems and intercellular signaling within the tumor immune microenvironment across the NAT–HGIN–GBC sequence. It identifies TOP2A, TWEAK, and FN14 as potential biomarkers and therapeutic targets for GBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f8ecb5a2d68898a868d59ee319a91eaeca48bb" target='_blank'>
              Single-cell analyses unravel ecosystem dynamics and intercellular crosstalk during gallbladder cancer malignant transformation
              </a>
            </td>
          <td>
            Zhaobin He, Jianqiang Cao, Xiqiang Wang, Shengbiao Yang, Huijie Gao, Yongzhe Yu, Zequn Di, Cheng Peng
          </td>
          <td>2025-05-16</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14d1b8aeaba5a454b04d6690efe6420aea37c43e" target='_blank'>
              Stochastic modelling of prostate progenitor architecture
              </a>
            </td>
          <td>
            Christo Morison, Esther Baena, Weini Huang
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The complex interplay between tumor cells and clusters with endothelial tissues during metastasis, in particular with regard to the exosomes in mediating intercellular communication, is still not well understood. Here, we develop a tumoral-transendothelial migration model to replicate the microenvironment of circulating tumor cells infiltrating blood vessels during metastasis. We propose an exosome-integrated approach combining a tumoral-transendothelial migration chip (TEMOC) with machine learning (ML) to enable the simulation and prediction of exosome-mediated invasion into the endothelial layer at both the single-cell and cluster levels. Leveraging a microfluidic trap array and the inherent self-organizing properties of cells, we conducted high-throughput studies on 121 specific tumor microenvironments on a chip. We uncovered the impact of exosomes derived from highly metastatic breast cancer on individual breast cancer cells and clusters: exosomes disrupt the adhesive matrix between endothelial cells and enhance tumor cell invasion. Additionally, highly metastatic cell-derived exosomes were found to stimulate the epithelial-mesenchymal transition (EMT) process in low-metastatic breast cancer cells (MCF-7), thereby promoting metastasis. An ML algorithm, K-nearest neighbor (KNN), was subsequently utilized to evaluate the correlation between multiple biomarkers on tumor cells and tumor invasion capability. The optimized biomarker combination strategy achieved a prediction accuracy of 93.5%. These findings contribute to a deeper understanding of the mechanisms by which exosomes derived from highly metastatic breast cancer cells induce metastasis. Furthermore, the combined use of the TEMOC and ML approach offers a platform for exploring the mechanisms of exosome-mediated tumor-vascular invasion and accelerating anti-metastatic therapeutic discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24f8d6a5cc78e18d87c8aa513a73cd505377083e" target='_blank'>
              Dissecting Exosomal-Tumoral-Vascular Interactions of Single Tumor Cells and Clusters Using a Tumoral-Transendothelial Migration Chip.
              </a>
            </td>
          <td>
            Xuan Zhang, Junjie Bai, Shi-Cheng Fan, Lan-Feng Liang, Xiao Song, M. Nai, Ruiping Zhang, Mingli Chen, Jianhua Wang, C. T. Lim
          </td>
          <td>2025-06-24</td>
          <td>ACS nano</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Intra-tumor heterogeneity impacts disease progression and therapeutic resistance but remains poorly characterized by conventional histologic, immunophenotypic, and molecular approaches. Single-cell biophysical properties distinguish functional phenotypes complementary to these approaches, providing additional insight into cellular diversity. Here we link both buoyant mass and stiffness to gene expression to identify clinically relevant phenotypes within primary mantle cell lymphoma (MCL) cells, employing MCL as a model of biological and clinical diversity in human cancer. Linked measurements reveal that buoyant mass and stiffness characterize B-cell development states from naïve to plasma cell and correlate with expression of oncogenic B-cell receptor signaling genes such as BLK and CD79A. Additionally, changes in cell buoyant mass within primary patient specimens ex vivo correlate with sensitivity to Bruton’s Tyrosine Kinase inhibitors in vivo in MCL and chronic lymphocytic leukemia, another B-cell malignancy. These findings highlight the value of biophysical properties as biomarkers of response in pursuit of future precision therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/222e5440bee40a1cfc03b8b975d31d717ee09614" target='_blank'>
              Integrating Single-Cell Biophysical and Transcriptomic Features to Resolve Functional Heterogeneity in Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            Ye Zhang, Lydie Debaize, Adam Langenbucher, Jenalyn Weekes, Ioulia Vogiatzi, Teemu P. Miettinen, Mingzeng Zhang, Emily Sumpena, Huiyun Liu, Sarah M. Duquette, L. Hackett, Jeremy Zhang, Sona Baghiyan, R. Redd, M. Aryee, M. Davids, Austin I. Kim, C. Ryan, David M. Weinstock, S. Manalis, M. Murakami
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="ABSTRACT Background Breast cancer is a highly heterogeneous disease, characterized by tumor and nontumor cells at various cell states. Ecotyper is an innovative machine learning framework that quantifies the tumor microenvironment and delineates the tumor ecosystem, demonstrating clinical significance. However, further validation is needed in breast cancer. Methods Ecotyper was applied to identify multiple cellular states and tumor ecotypes using large‐scale breast cancer bulk sequencing data, followed by a detailed analysis of their associations with clinical classification, molecular subtypes, survival prognosis, and immunotherapy response. Identified subtypes were further characterized using single‐cell and spatial data sets to reveal molecular profiles. Results In a comprehensive analysis of 6578 breast cancer samples from four data sets, Ecotyper identified 69 cellular states and 10 tumor ecotypes. Of these, 37 cellular states significantly correlated with overall survival. Notably, specific states within epithelial cells, macrophages/monocytes, and fibroblasts were linked to a worse prognosis. CE2 abundance was identified as the most significant marker indicating unfavorable prognosis and was further validated in an additional data set of 116 HER2‐negative patients. These biomarkers also indicated the efficacy of neoadjuvant immunotherapy in breast cancer. CE2‐high cancers were characterized by an abundance of basal‐like epithelial cells, scant lymphocytic infiltration, and activation of hypoxia signaling. Single‐cell analysis showed that CE2‐high areas were rich in SPP1‐positive tumor‐associated macrophages(TAM), basal‐like epithelial cells, and hypoxic cancer‐associated fibroblasts(CAF). Spatially, these regions were often peripheral in triple‐negative breast cancer, adjacent to fibrotic/necrotic zones. Multiplex immunofluorescence confirmed the enrichment of SPP1+CD68+TAM and HIF1A+SMA+CAF in hypoxic triple‐negative breast cancer (TNBC) regions. Conclusions Ecotyper identified novel biomarkers for breast cancer prognosis and treatment prediction. The CE2‐high region may represent a hypoxic immune‐suppressive niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed777dbf5c26fef9472ca8bd002f079780d98595" target='_blank'>
              The Identification of Novel Prognostic and Predictive Biomarkers in Breast Cancer via the Elucidation of Tumor Ecotypes Using Ecotyper
              </a>
            </td>
          <td>
            Feng Du, J. Ju, F. Zheng, Song-Lin Gao, Peng Yuan
          </td>
          <td>2025-05-27</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd1f3cec19d8b2ce70a881702528c91746b1cae" target='_blank'>
              CellLENS enables cross-domain information fusion for enhanced cell population delineation in single-cell spatial omics data.
              </a>
            </td>
          <td>
            Bokai Zhu, Sheng Gao, Shuxiao Chen, Yuchen Wang, Jason Yeung, Yunhao Bai, Amy Y. Huang, Y. Yeo, Guanrui Liao, Shulin Mao, Z. G. Jiang, Scott J Rodig, Ka-Chun Wong, Alex K. Shalek, GP Nolan, Sizun Jiang, Zongming Ma
          </td>
          <td>2025-05-22</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="While cervical cancer (CC) prognosis depends on tumor staging, the spatiotemporal evolution of tumor microenvironment (TME) heterogeneity during metastatic progression remains poorly characterized at single-cell resolution. We employed an integrative multi-omics approach, combining single-cell RNA sequencing (scRNA-seq; n = 11), spatial transcriptomics (ST), and bulk RNA-seq data from the TCGA-CESC cohort (n = 304), to systematically map TME remodeling across CC progression stages. scRNA-seq was performed on primary lesions from patients with localized (n = 3), regional (n = 4), and metastatic (n = 4) diseases, with in-depth analyses focusing on cellular characteristics, cell type composition alterations, functional changes, differentiation trajectories, and cell-cell interaction networks. These findings were further validated using spatial transcriptomics, bulk RNA-seq data, and multiple immunohistochemistry (mIHC) experiments. ScRNA-seq data revealed that the TME of the metastatic group displayed a distinct immunosuppressive phenotype. Three key subclusters closely linked to TME remodeling in this group were identified. Notably, a novel metastasis-associated epithelial subpopulation (Epi0_AGR2), characterized by both epithelial-mesenchymal transition (EMT) and chemokine secretory phenotypes, was discovered. Gene Set Variation Analysis (GSVA) revealed that transforming growth factor β (TGF-β) signaling activation served as its primary transcriptional driver. Additionally, a neutrophil subset with pro-tumor and immunosuppressive properties, as well as a cancer-associated fibroblasts (CAFs) subset that promoted angiogenesis, were enriched in the metastatic group. Cell-cell interaction analysis and spatial mapping further revealed the formation of coordinated Epi0-neutrophil-CAFs niches, which established TGF-β-CXCL1/2/8-OSM/OSMR feedforward loops. Importantly, a computational model derived from the TME metastatic niche signature demonstrated significant prognostic stratification in the TCGA cohort (HR = 2.5179, p = 0.0144). In all, this study provides the first comprehensive delineation of stage-specific TME dynamics in CC, revealing TGF-β-driven cellular cooperativity as a metastatic switch. The joint framework establishes a potential clinically translatable tool for precision prognosis and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac624640fa9f989701717857751510daeec92896" target='_blank'>
              Multi-Omics Analysis Reveals the transforming growth factor-β Signaling-Driven Multicellular Interactions with Prognostic Relevance in Cervical Cancer Progression
              </a>
            </td>
          <td>
            Yuhan Wang, Guang-xu Cao, Huimin Zeng, Yong Zhi, Meng-Xi Xu, Ying Wang, Min Liu, Yetian Ruan, Ka Yu Tse, Qingfeng Zhang, Jinli Gao, Zhiqiang Han, Fang Li
          </td>
          <td>2025-06-20</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35658443146621f3cb5a98587b659629d87dcfdc" target='_blank'>
              Longitudinal multi-omic profiling uncovers immune escape and predictors of response in multiple myeloma
              </a>
            </td>
          <td>
            Denis Ohlstrom, William C. Pilcher, Marina E. Michaud, Chaitanya R. Acharya, Sarthak Satpathy, Edgar Gonzalez-Kozlova, R. Jayasinghe, Katherine E. Ferguson, Hope L. Mumme, Shivani Nanda, Yizhe Song, Sowmitri Karthikeya Siddhartha Mantrala, D. Karagkouni, Jessica Schulman, Nick Pabustan, Junia Vieira Dos Santos, Daniel W. Sherbenou, J. Keats, Alex Gout, S. Foltz, A. Laganà, T. Kourelis, Ravi Vij, M. Dhodapkar, D. Avigan, H. Cho, L. Baughn, Ajay Nooka, S. Lonial, Shaji K. Kumar, Mehmet K Samur, Ioannis S. Vlachos, Li Ding, Sacha Gnjatic, George Mulligan, Manoj K. Bhasin
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Background Metastatic hormone-naïve prostate cancer (mHNPC) is an infrequent form of this tumor type that is characterized by metastasis at the time of diagnosis and accounts for up to 50% of prostate cancer-related deaths. Despite the extensive characterization of localized and metastatic castration-resistant prostate cancer, the molecular characteristics of mHNPC remain largely unexplored. Results Here, we provide the first extensive transcriptomics characterization of primary tumor specimens from patients with mHNPC. We generate discovery and validation bulk and single-cell RNA-seq datasets and perform integrative computational analysis in combination with experimental studies. Our results provide unprecedented evidence of the distinctive transcriptional profile of mHNPC and identify stroma remodeling as a predominant feature of these tumors. Importantly, we discover a central role for the SRY-box transcription factor 11 (SOX11) in triggering a heterotypic communication that is associated with the acquisition of metastatic properties. Conclusions Our study will constitute an invaluable resource for a profound understanding of mHNPC that can influence patient management. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03623-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bec724fe7758a24264be0a981015a60432a2227" target='_blank'>
              Transcriptional analysis of metastatic hormone-naïve prostate cancer primary tumor biopsies reveals a relevant role for SOX11 in prostate cancer cell dissemination
              </a>
            </td>
          <td>
            Natalia Martín-Martín, Saioa Garcia-Longarte, Jon Corres-Mendizabal, Uxue Lazcano, I. Astobiza, Laura Bozal-Basterra, Nicolas Herranz, Hielke van Splunder, Onintza Carlevaris, Mikel Pujana-Vaquerizo, María Teresa Blasco, Ana M Aransay, A. Rosino, J. Tudela, Daniel Jimenez, Alberto Martinez, Andrei Salca, Aida Santos-Martín, Sofía Rey, Aitziber Ugalde-Olano, David Gonzalo, M. Graupera, R. Gomis, J. Mateo, Miguel Unda, E. González-Billalabeitia, Ana Loizaga-Iriarte, Isabel Mendizabal, Arkaitz Carracedo
          </td>
          <td>2025-06-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78beb6714803e0c6118158e2cf27d989400224a0" target='_blank'>
              Single Cell RNA Sequencing Reveals Gene Expression Continuums Along the Spatial Hierarchy of the Pulmonary Circulation
              </a>
            </td>
          <td>
            S. Sveiven, C. Knutsen, Fabio Zanini, David N Cornfield, C. Alvira
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c2fd63e082e26fba9172752b056b5282ca5810" target='_blank'>
              Barcoded viral tracing identifies immunosuppressive astrocyte-glioma interactions.
              </a>
            </td>
          <td>
            Brian M Andersen, Camilo Faust Akl, Michael A. Wheeler, Zhaorong Li, M. Diebold, Michael Kilian, Joseph M Rone, Aditya Misra, Jessica E. Kenison, Joon-Hyuk Lee, Hong-Gyun Lee, C. M. Polonio, David Merrell, J. Weiss, Lillie Godinez, Gavin Piester, Tomer Illouz, Jessica J Ye, Arianna Ghia, Jazmin Martinez, Elizabeth N. Chung, Léna Srun, Daniel Farrenkopf, Lucas E. Flausino, Anton M Schüle, L. Sanmarco, Federico Giovannoni, Luca Fehrenbacher, M. Charabati, Cristina Gutiérrez-Vázquez, Margaret M Cusick, Prem Prabhakar, Connor C Bossi, Emily Lapinskas, Roni Nowarski, G. Getz, Keith L. Ligon, M. Prinz, E. Chiocca, David A. Reardon, Francisco J. Quintana
          </td>
          <td>2025-06-25</td>
          <td>Nature</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Abstract The tumor microenvironment (TME) significantly influences cancer prognosis and therapeutic outcomes, yet its composition remains highly heterogeneous, and currently, no highly accessible, high‐throughput method exists to define it. To address this complexity, the TMEclassifier, a machine‐learning tool that classifies cancers into three distinct subtypes: immune Exclusive (IE), immune Suppressive (IS), and immune Activated (IA), is developed. Bulk RNA sequencing categorizes patient samples by TME subtype, and in vivo mouse model validates TME subtype differences and differential responses to immunotherapy. The IE subtype is marked by high stromal cell abundance, associated with aggressive cancer phenotypes. The IS subtype features myeloid‐derived suppressor cell infiltration, intensifying immunosuppression. In contrast, the IA subtype, often linked to EBV/MSI, exhibits robust T‐cell presence and improved immunotherapy response. Single‐cell RNA sequencing is applied to explore TME cellular heterogeneity, and in vivo experiments demonstrate that targeting IL‐1 counteracts immunosuppression of IS subtype and markedly improves its responsiveness to immunotherapy. TMEclassifier predictions are validated in this prospective gastric cancer cohort (TIMES‐001) and other diverse cohorts. This classifier could effectively stratify patients, guiding personalized immunotherapeutic strategies to enhance precision and overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e59035a4145d3d05ce3897eefa444405fb68dd" target='_blank'>
              Immunotyping the Tumor Microenvironment Reveals Molecular Heterogeneity for Personalized Immunotherapy in Cancer
              </a>
            </td>
          <td>
            D. Zeng, Yunfang Yu, Wenjun Qiu, Q. Ou, Qianqian Mao, Luyang Jiang, Jianhua Wu, Jiani Wu, Huiyan Luo, Peng Luo, Wenchao Gu, N. Huang, S. Zheng, Shaowei Li, Yonghong Lai, Xiatong Huang, Yiran Fang, Qiongzhi Zhao, R. Zhou, Huiying Sun, Wei Zhang, J. Bin, Yulin Liao, Masami Yamamoto, Tetsuya Tsukamoto, Sachiyo Nomura, Min Shi, Wangjun Liao
          </td>
          <td>2025-05-28</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afbfafb6e752991df103b290fb3be622c923aa5e" target='_blank'>
              Immune Spatial Organization Predicts Metastasis Risk in Aggressive Localized Prostate Cancer
              </a>
            </td>
          <td>
            David D. Yang, Aya Abdelnaser, Jeremiah Wala, Alexander J Haas, Alfred A Barney, Eddy Saad, Jett Crowdis, Cora A. Ricker, Jihye Park, Martin T King, P. L. Nguyen, T. Choueiri, A. Tewari, Keyan Salari, M. Taplin, Chin-Lee Wu, E. V. Van Allen
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) lacks targeted therapies, leading to a poor prognosis. Younger patients with TNBC often present with aggressive disease and exhibit distinct tumor microenvironments (TME). We performed spatial profiling to understand the influence of menopausal status on the immune environment, tumor progression, and therapy response. Methods Eleven treatment-naive TNBC tumors were examined in epithelial and non-epithelial areas using digital spatial profiling to identify differentially expressed markers and pathways. Deconvolution methods were employed to identify immune cell subtypes, and the protein expression of T and B cells was confirmed. The prognostic utility of the identified genes and pathways was validated using the METABRIC and SCAN-B datasets. The expression of target genes was analyzed in the I-SPY 2 trial data to understand their influence on the response to specific therapies. Results Premenopausal tumors formed a distinct cluster characterized by the upregulation of cell activation and antigen presentation pathways. In contrast, T cell checkpoint, cancer antigen, and PI3K-AKT pathways were downregulated. External datasets validated these findings, showing a lower hazard and better prognosis for genes upregulated in premenopausal tumors. An enrichment of CD8 + T cells, endothelial cells, and monocytes was observed, along with increased intratumoral protein expression of CD8, CD4, and CD20. Premenopausal tumors demonstrated better responses to PARP and HSP90 inhibitors but showed lower sensitivity to Pembrolizumab and PI3K-AKT inhibitors compared to postmenopausal tumors in the I-SPY 2 trial data. Conclusion Our study underscores the importance of menopausal status in shaping the TME within TNBC, revealing distinct immune landscapes and therapy responses. These findings highlight the need for larger prospective studies to validate differential treatment strategies in younger patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06786-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c6a6cdaccc22fd9e9450a555636943aa5ed9adb" target='_blank'>
              Spatial profiling reveals unique immune microenvironment in premenopausal triple-negative breast cancer associated with therapy response
              </a>
            </td>
          <td>
            V. P. Nimbalkar, V. Snijesh, S. Rajarajan, C. Anupama, S. Mahalakshmi, Annie Alexander, Deshica Dechamma, Manju Moorthy, Gopalakrishna Ramaswamy, Rakesh Ramesh, B. Srinath, J. Prabhu
          </td>
          <td>2025-07-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 e13119


 Background:
 The tumor microenvironment (TME) of primary breast tumors (PTs) and breast-to-liver metastases shows heterogeneous infiltration and interactions between tumor, immune, and stromal cells. Hepatic metastases exhibit a more immunosuppressive TME with fewer cytotoxic T cells (CTLs) relative to PTs, which may contribute to increased tumor growth and resistance to systemic therapies. To investigate the immune cell infiltration and immune-tumor interactions in PT and liver metastases, we utilized whole transcriptome analysis and immunofluorescence-based protein expression as part of spatial TME profiling.
 Methods:
 We identified 13 chemo-naïve PTs and 3 breast-to-liver metastases. Formalin-fixed paraffin-embedded (FFPE) samples were used for hematoxylin and eosin (H&E) staining and spatial transcriptomics (ST) profiling with the 10X Genomics Visium CytAssist and Visium HD platforms. The H&E slides were manually annotated in collaboration with a board-certified pathologist, and ST data were deconvoluted using breast-specific scRNA-sequenced references and canonical marker labeling.
 Results:
 Our analysis of the 13 PTs and 3 breast-to-liver metastases revealed significant heterogeneity in tumor clusters, immune cell infiltration, gene set enrichments, and gene signatures between primary and visceral TMEs. Unsupervised clustering of hormone receptor-positive (HR+) HER2-negative (HER2) samples demonstrated considerable intratumoral and intertumoral variability, with differences in the expression of tumor markers (EPCAM, ER, PR, ERBB2, KRT14, KRT18), hypoxia-related signatures (HIF1a), and pro-angiogenic drivers (VEGF). Breast-to-liver metastases showed higher expression of pathways involved in proliferation, angiogenesis, and anti-apoptosis compared to primary tumors. Hepatic metastases were enriched for HALLMARK gene sets related to proliferation and metabolic pathways, including PI3K/AKT/mTOR, mTORC1 signaling, E2F targets, MYC targets, G2M checkpoint, mitotic spindle, glycolysis, oxidative phosphorylation, and fatty acid metabolism when compared to primary tumors. In contrast, primary tumors were enriched for immune-related pathways (TNF-α, interferon-γ, inflammatory response, IL-6 JAK/STAT3), epithelial-mesenchymal transition (EMT), myogenesis, and angiogenesis gene signatures, which are consistent with their metastatic potential. Cellular profiling of liver lesions revealed higher myeloid suppressor cells (TAMs and MDSCs): CD8
 +
 ratio with shorter cell-cell distances relative to PTs suggesting a key role in immune evasion, tumor progression, and therapeutic resistance.
 Conclusions:
 ST data from primary and metastatic breast cancer samples revealed notable tumor heterogeneity, characterized by highly active metabolic and proliferative pathways, along with a higher abundance of myeloid cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/389d6dee563fb2269c25d3de12654b70850dddf7" target='_blank'>
              Spatial transcriptomics and functional mapping of the suppressive TME in primary breast tumor and breast-to-liver metastases.
              </a>
            </td>
          <td>
            Batul Al-zubeidy, Edgar Gonzalez, A. Baugh, Dominic Zavala, Cheol Park, Matthew Jacobo, Marqus Dela Cruz, Michelle Li, Sabrina Carrel, Elexa Rallos, Michael Press, Evanthia T. Roussos Torres
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Objective To investigate the metabolic remodeling and functional specialization of immune cells within the tumor microenvironment (TME) of bone tumors, including Ewing’s sarcoma, osteosarcoma, and giant cell tumor of bone, through high-resolution single-cell RNA sequencing (scRNA-seq) analysis. Methods Immune cells were isolated from 13 bone tumor samples and profiled via scRNA-seq to delineate cellular compositions, metabolic adaptations, and intercellular communication networks. Differential gene expression analysis, metabolic pathway enrichment, and pseudotime trajectory inference were employed to characterize functional states and differentiation processes of immune cell subsets. Results We identified 12 major immune cell clusters with distinct functional and metabolic characteristics. Naïve T cells exhibited amino acid metabolism-dependent activation potential, whereas NK cells relied on lipid metabolism and the TCA cycle for cytotoxic activity. Macrophage subsets demonstrated functional divergence: C06 macrophages adopted lipid metabolism to facilitate immunosuppression and tissue repair, while C04 macrophages displayed pro-inflammatory characteristics associated with complement activation. Intercellular signaling analysis revealed FN1 as a central regulator of immune coordination, governing cell adhesion, migration, and homeostasis within the TME. Conclusion This study provides novel insights into the metabolic and functional plasticity of immune cells in bone tumor TMEs, underscoring the critical role of metabolic remodeling in immune regulation. Our findings highlight potential therapeutic targets for modulating immune cell function and offering new avenues to improve treatment outcomes for patients with bone tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06346-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e0d19e0f285642428880ab85bf21db7c2bbae62" target='_blank'>
              Single-cell transcriptomics reveals metabolic remodeling and functional specialization in the immune microenvironment of bone tumors
              </a>
            </td>
          <td>
            Jun Chen, Na Cui, Shaokai He, Chun-Yan Xia, Wei-Qing Li
          </td>
          <td>2025-05-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Resistance to chemotherapy of pancreatic ductal adenocarcinoma (PDAC) is largely driven by intratumoral heterogeneity (ITH) due to tumor cell plasticity and clonal diversity. In order to develop novel strategies to overcome this defined mechanism of resistance, tools to monitor and quantify ITH in a rapid and scalable fashion are needed urgently. Here, we employed label-free digital holographic microscopy (DHM) to characterize ITH in PDAC. We established a robust experimental and machine learning analysis pipeline to perform single cell phenotyping based on DHM-derived phase images of PDAC cells in suspension. Importantly, we are able to detect dynamic changes in tumor cell differentiation and heterogeneity of distinct PDAC subtypes upon induction of epithelial-to-mesenchymal transition and under treatment-imposed pressure in murine and patient-derived model systems. This platform allows us to assess phenotypic ITH in PDAC on a single cell level in real-time. Implementing this technology into the clinical workflow has the potential to fundamentally increase our understanding of tumor heterogeneity during evolution and treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b97958c7f42c79b39377ba5cf9b12106537163c" target='_blank'>
              Label-free single cell phenotyping to determine tumor cell heterogeneity in pancreatic cancer in real-time.
              </a>
            </td>
          <td>
            Katja Wittenzellner, M. Lengl, S. Röhrl, Carlo Maurer, C. Klenk, A. Papargyriou, Laura Schmidleitner, Nicole Kabella, Akul R Shastri, D. Fresacher, Farid Harb, Nawal Hafez, S. Bärthel, Daniele Lucarelli, Carmen Escorial-Iriarte, F. Orben, R. Öllinger, Ellen Emken, L. Fricke, Joanna Madej, Patrick Wustrow, I. Demir, Helmut Friess, Tobias Lahmer, Roland M. Schmid, Roland Rad, Günter Schneider, Bernhard Kuster, Dieter Saur, O. Hayden, Klaus Diepold, Maximilian Reichert
          </td>
          <td>2025-05-27</td>
          <td>JCI insight</td>
          <td>1</td>
          <td>51</td>
        </tr>

        <tr id="Abstract Background Small cell lung cancer (SCLC) is a lethal lung malignancy and patients are often diagnosed with distant metastasis. Nearly all patients suffer from disease relapsing with inherent chemoresistance. Lack of targeted SCLC therapies further worsens disease outcomes, making it highly desirable to identify novel and effective therapeutic targets. Methods To search for potential therapeutic targets in SCLC, we analyzed publicly available single-cell and bulk RNA-sequencing (RNA-seq) data from normal, lung adenocarcinoma, and SCLC tumor tissues. To assess the targeting potential of FOXM1, we developed various in vitro models, including DOX-On-shFOXM1 (Tet-ON) inducible stable knockdown systems. Cisplatin resistant human and murine SCLC cell lines were generated to assess the role of FOXM1 in chemotherapy resistance. Immunoblotting, immunohistochemistry (IHC), and immuno-fluorescence were used to analyze the expression of FOXM1 and target proteins. ChIP-assay was used to study protein-gene interactions. Further, multicolor flow cytometry was employed to study the effect of FOXM1 inhibition on human T cells activation and differentiation. Subcutaneous xenograft and SCLC spontaneous (RPM: RB fl/fl ; TP53 fl/fl ; LSL-MYC T58A ) mouse models were used to evaluate the efficacy of FOXM1 inhibitors. Results Single-cell as well as bulk RNA-seq data revealed that FOXM1, an oncogenic transcription factor, is overexpressed in SCLC, and it was recapitulated in human and murine SCLC tissues and cell lines. Interestingly, chemo-resistant (CR) SCLC showed a substantially higher FOXM1 expression than naïve SCLC. Silencing FOXM1 genetically or pharmacologically by FOXM1 inhibitors revealed a marked reduction in cell viability, colony formation, migration and sphere formation in naïve and CR SCLC cells. Moreover, FOXM1 inhibition induced apoptosis and cell cycle arrest in SCLC cells. Furthermore, FOXM1 inhibition in combination with first-line platinum-based chemotherapy showed synergistic anticancer effects in both xenograft and RPM mouse models of SCLC. Our RNA-seq analysis revealed that FOXM1 inhibition altered the Aurora Kinase B (AURKB) signaling pathway, which is dysregulated in SCLC. Moreover, we found FOXM1 inhibition enhanced T cell activation and supported the differentiation of CD8 + cytotoxic T cells, and T cell-mediated killing of cancer cells. Conclusions Our study demonstrates that FOXM1 targeting using small molecule inhibitors has the potential to be a novel therapeutic strategy to combat SCLC progression including chemotherapeutic resistance and reshaping the anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cf5514bc8f1dd3db6ac08b065580f4ed8890ea4" target='_blank'>
              FOXM1 targeting alters AURKB activity and reshapes antitumor immunity to curb the progression of small cell lung cancer
              </a>
            </td>
          <td>
            Md Arafat Khan, Parvez Khan, Mahek Fatima, Asad Ur Rehman, Laiba Anwar, Z. Alsafwani, Aatiya Ahmad, Mohammad Ali Abbas Zaidi, Jesse L. Cox, Areem Zahid, Sameer Mohiuddin, Sung Hoon Kim, Juan A. Santamaria-Barria, Imayavaramban Lakshmanan, B. Katzenellenbogen, John A Katzenellenbogen, A. Ganti, Surinder K Batra, M. Nasser
          </td>
          <td>2025-07-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac2fa84e64df92a9b47c54f45baf0d93294fea97" target='_blank'>
              Substrate heterogeneity promotes cancer cell dissemination
              </a>
            </td>
          <td>
            Zuzana Dunajova, Saren Tasciyan, Juraj Májek, J. Merrin, E. Sahai, M. Sixt, É. Hannezo
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="
 e20030


 Background:
 Tumoral heterogeneity has been widely acknowledged to influence clinical outcome of lung adenocarcinoma (LUAD). This study investigated spatial gene profiling of different pathological subtypes of early-stage invasive LUAD with Stereo-seq transcriptomics technique. The tumor microenvironment (TME) was decoded with cellular and molecular events at perspectives of tumoral activities and immune activities.
 Methods:
 We collected spatial RNA profiles of nine Stereo-seq chips (captured area of 1 x1 cm) from 4 invasive LUAD patients. Pathologist identified histological subtypes including lepidic adenocarcinoma (LP), acinar adenocarcinoma (AC) and other invasive adenocarcinoma like papillary adenocarcinoma (IA) on adjacent H&E images. Spatial data matrix was obtained based on pathological annotation accordingly. Spatially resolved gene expression of each subtypes was obtain based on pathological annotation to conduct high resolution analyses. TME functional units were calculated by analysis of cellular neighborhoods (CNs) based on cell compositions within resolution of bin 100 (window of 50um x 50um). Sub-populations of tumoral cells were stratified by molecular functions (GSEA and GSVA score) and stemness (CytoTRACE).
 Results:
 All pathological subtypes shared the identical 10 heterogeneous CNs and 5 tumoral clusters (TM) with various propotions. Tumor cluster 0 (TM0) and TM3 were identified as the malignant population with significant high stemness and significant high activities of MTORC1 signaling, TGF-BETA signaling, PI3K-ATK signaling and hypoxia. AC and IA were mainly composed of TM0 and TM3. Compared to TM0, TM3 had higher EMT activity. We further identified two immune-related CNs (CN4 and CN8) with significant high GSVA score of Interferon-Alpha response, Interferon-Gamma response and Inflammatory response. Spatial association showed that the lymphoid-cells-dominated CN4 was enriched in LP and IA and nearly absent in AC. Meanwhile, the myeloid-cells-dominated CN8 was enriched in AC. GSEA score showed that immune activity of CN4 was more intense in IA when compared to LP. Notably, TM0 mainly present in CN8, and TM3 mainly present in CN4. Therefore, the LP was speculated to reach an inert and statical status. TME of AC was featured as colocalization of myeloid cells and tumor cells with high stemness. TME of IA was featured as colocalization of lymphoid cells and tumor cells with high EMT activity.
 Conclusions:
 This is the first study to delineate the homogeneity of tumor cells and heterogeneity of immune cells among different pathological types of NSCLC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344f597f0ff9f98416e718719f81b290a2438fe0" target='_blank'>
              Decoding spatial transcriptomics-based heterogeneity of TME in early-stage invasive pulmonary adenocarcinoma.
              </a>
            </td>
          <td>
            Xin Liu, Jingyi Tian, Z. Shang, C. Xiang, Fei Teng, Jiajun Zhang, Yuchen Han
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Cervical cancer (CC) remains a significant global health challenge despite advancements in screening, HPV vaccination, and therapeutic strategies. Tumor heterogeneity, driven by epigenetic modifications, affects immune evasion, metastasis, and treatment response. Cancer-associated fibroblasts (CAFs) play a crucial role in CC progression and therapy resistance. Single-cell sequencing offers new insights but remains underutilized in CC research. This study integrates single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and deconvolution analysis to identify key genes and immunotherapy targets. By constructing a prognostic model and exploring the immune microenvironment, we aim to provide novel insights into CC pathogenesis and potential therapeutic strategies. Methods We utilized scRNA-seq, spatial transcriptomics, deconvolution analysis, and pseudotime trajectory mapping to delineate fibroblast subtypes within the tumor immune microenvironment (TIME) of CC. Functional annotations, differential gene expression profiling, cell–cell communication pathways, and transcription factor networks were systematically analyzed. A prognostic model based on bulk RNA-seq data was constructed and validated through survival analysis, with correlations to immune microenvironment characteristics. Functional experiments investigated the role of SDC1, a critical mediator of fibroblast-tumor crosstalk. Additionally, Fibroblast–tumor cell co-culture systems and functional assays were employed to investigate the paracrine role of SDC1. The CAF MYH11⁺ subpopulation was isolated via fluorescence-activated cell sorting (FACS). Multiplex immunofluorescence and immunohistochemical analyses were performed on both cultured cells and human cervical cancer tissue samples to characterize the spatial distribution and dynamic remodeling of MYH11 during stromal reorganization. Results Six distinct fibroblast subtypes were identified, including the C0 MYH11 + fibroblasts, which exhibited unique roles in stemness maintenance, metabolic activity, and immune regulation. Spatial and functional analyses revealed that the C0 subtype is central to tumor-fibroblast interactions, particularly through the MDK-SDC1 signaling axis. The prognostic model incorporating fibroblast-specific markers demonstrated robust predictive power for patient survival outcomes. Additionally, in vitro SDC1 knockdown significantly inhibited CC cell proliferation, migration, and invasion. Fibroblasts show spatially regulated heterogeneity, with activation markers enriched in the tumor zone and MYH11 highest in normal zones, indicating dynamic stromal remodeling. C0 MYH11 + CAF Promotes Tumor Cell Proliferation, Migration, and Inhibits Apoptosis via Soluble SDC1. Conclusion Our results illustrate, in some ways, the possible immunomodulatory and tumor supporting roles of CAFs in CC TIME and highlight the possibility that the MDK-SDC1 pathway is a promising therapeutic target. This study not only promotes a partially new understanding of temporal heterogeneity in CC, but also provides a possible reference base for the development of new biomarkers and immunotherapy approaches to improve clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03432-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d1387a936b1ef6d7e045f6c7bcff60d568a6f7" target='_blank'>
              Deciphering the tumor immune microenvironment: single-cell and spatial transcriptomic insights into cervical cancer fibroblasts
              </a>
            </td>
          <td>
            Zhiheng Lin, Youwei Zhou, Zhenran Liu, Wenyang Nie, Hengjie Cao, Shengnan Li, Xuanling Li, Lijun Zhu, Guangyao Lin, Yanyu Ding, Yi Jiang, Zuxi Gu, Lianwei Xu, Zhijie Zhao, Hua-Bao Cai
          </td>
          <td>2025-07-05</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Standing as the most aggressive form of primary malignant tumor, Glioblastoma (GBM) tumors with marked heterogeneity represents one of the enormous challenges in glioma treatment. Myeloid cells, which includes neutrophils, myeloid-derived suppressor cells, microglia, and macrophages, play a pivotal role in the tumor microenvironment of GBM. In the tumor microenvironment (TME), T cells and natural killer (NK) cells exert anti-tumor functions, whereas myeloid-derived suppressor cells (MDSCs) can promote tumor progression by suppressing these immune responses. Therefore, MDSCs play a critical role in shaping the effectiveness of immunotherapy. TME has constrained the ability of traditional GBM treatment approaches to significantly enhance prognostic outcomes for patients. This category encompasses conventional therapies like surgical resection and radiation therapy, along with cutting-edge methodologies such as immunotherapy. Through extensive investigations into the dynamic interactions between the GBM microenvironment and neoplastic cells, both targeted treatment strategies and innovative immunotherapeutic modalities have emerged, offering promising new directions for clinical intervention. This review focuses on the interactions between GBM and myeloid cells (MCs), providing novel insights into the oncogenesis and progression of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e527685c44af6c300c634e53af18ed422e9ae9" target='_blank'>
              Interactions between glioblastoma and myeloid cells
              </a>
            </td>
          <td>
            Yuting Li, Yuhong Chen, Kai Cai, Yujuan Qin, Xi Wang, Bo Zhang, Lin Shi, Zonglin He, Jiasheng Wang, Jiecun Long, Yishun Zeng, Qiong Gong
          </td>
          <td>2025-06-24</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9faa1273d62a3ded4dd4ed42a59e3c9f4c050ab5" target='_blank'>
              Spatial omics technology potentially promotes the progress of tumor immunotherapy.
              </a>
            </td>
          <td>
            Zhen Lan, Yuanyuan Yang, Lingling Li, Chaoguan Wang, Zhenqiang Sun, Qiming Wang, Yang Liu
          </td>
          <td>2025-06-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common malignancies worldwide, and its complex pathogenesis and significant tumor cell heterogeneity remain major challenges. With the rapid development of single-cell sequencing technology, we can now delve deeper into the cellular composition and dynamic changes within the tumor microenvironment, revealing cellular interactions and their potential roles in tumorigenesis. Method In this study, we systematically analyzed comprehensive single-cell RNA sequencing data from 25 colorectal cancer and 10 adjacent normal tissue samples. We explored the characteristics and biological significance of tumor cell subpopulations, performed quality control, dimensionality reduction, and cell type identification, and further investigated epithelial cell copy number variations, cell communication, and pseudotime analysis. Subsequently, Boruta feature selection algorithm was combined to identify prognosis related genes. The expression patterns, clinical significance and biological effects of PRSS22 were validated in vitro. Results Our analysis found an epithelial cell subcluster with high expression of PRSS22 exhibited high proliferation and migration abilities, and it was also associated with the dysregulated immune microenvironment. After further experimental verification, we proved the high expression patterns and clinical significance of PRSS22. Downregulation of PRSS22 in CRC cells resulted in a reduction of proliferation, migration and invasion. Conclusion Our study has identified a cell subcluster that is closely linked to progression, immune dysregulation and prognosis in CRC, and we have also identified PRSS22 as its hub gene that has great potential to become a new immunotherapeutic targets target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastatic prostate cancer (mPC) is a clinically and molecularly heterogeneous disease. While there is increasing recognition of diverse tumor phenotypes across patients, less is known about the molecular and phenotypic heterogeneity present within an individual. In this study, we aimed to define the patterns, extent, and consequences of inter- and intra-tumoral heterogeneity in lethal prostate cancer. By combining and integrating in situ tissue-based and sequencing approaches, we analyzed over 630 tumor samples from 52 mPC patients. Our efforts revealed phenotypic heterogeneity at the patient, metastasis, and cellular levels. We observed that intra-patient, inter-tumoral molecular subtype heterogeneity was common in mPC and showed associations with genomic and clinical features. Additionally, cellular proliferation rates varied within a given patient across molecular subtypes and anatomic sites. Single-cell sequencing studies revealed features of morphologically and molecularly divergent tumor cell populations within a single metastatic site. These data provide a deeper insight into the complex patterns of tumoral heterogeneity in mPC with implications for clinical management and the future development of diagnostic and therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb925628a7789a4c45633ccff7c788aadf4bd5ff" target='_blank'>
              Patterns of intra- and inter-tumor phenotypic heterogeneity in lethal prostate cancer.
              </a>
            </td>
          <td>
            Martine P. Roudier, Roman Gulati, Erolcan Sayar, Radhika A Patel, Micah Tratt, Helen Richards, Paloma Cejas, Miguel Muñoz Gomez, Xintao Qiu, Yingtian Xie, B. Hanratty, Samir Zaidi, Jimmy L. Zhao, M. Adil, Chitvan Mittal, Yibai Zhao, R. Dumpit, Ilsa Coleman, J. Low, Thomas W Persse, Patricia Galipeau, John K. Lee, Maria S Tretiakova, Meagan Chambers, F. Vakar‐López, Lawrence D. True, Marie E Perrone, Hung-Ming Lam, Lori Kollath, Chien-Kuang C. Ding, Stephanie Harmon, Heather H. Cheng, Evan Y. Yu, R. Montgomery, Jessica E. Hawley, Daniel W. Lin, Eva Corey, M. Schweizer, Manu Setty, G. Ha, Charles Sawyers, C. Morrissey, Henry Long, Peter S Nelson, Michael C. Haffner
          </td>
          <td>2025-06-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Globally, gastric cancer (GC) stands as the fifth most prevalent form of malignant neoplasm and represents a significant contributor to mortality associated with oncological conditions. Despite advancements in therapeutic strategies for GC, the outcomes for patients with advanced stages of the disease continue to be unfavorable, largely due to tumor heterogeneity and the challenges posed by resistance to therapeutic agents. Metabolic reprogramming is pivotal in driving the advancement of GC, contributing to the development of resistance to pharmacological treatments and facilitating the cancer’s ability to evade immune surveillance. Developing multi-target comprehensive treatment strategies by integrating tumor microenvironment (TME) modulation holds promise for significantly improving therapeutic efficacy.The study analyzed GC and identified key cell subtypes by integrating data derived from single-cell RNA-sequencing (scRNA-seq) alongside spatial transcriptomics information. Cell type identification was accomplished using the tool of Seurat, and the spatial distribution of cell types was revealed through the Robust Cell Type Decomposition technique. CellChat was used to analyze the interactions between key cell subtypes and other cells, and the “StLearn” package was employed to investigate spatial cell communication in depth. Additionally, the functional role of the key molecule ELK4 was validated through in vitro experiments.This research utilized scRNA-seq combined with spatial transcriptomics to comprehensively analyze GC, identifying the C1 NDUFAB1+ subtype, which exhibited high proliferative activity, metabolic reprogramming capabilities, and immune evasion properties. It was found that the C1 NDUFAB1+ subtype closely interacted with fibroblasts and pericytes via the PARs signaling pathway. Additionally, in vitro experiments confirmed that knockdown of ELK4 substantially curbed tumor cell proliferation, migration, and invasion.This study revealed the main significance of the C1 NDUFAB1+ subtype in GC, elucidating its core mechanisms in tumor progression, metabolic reprogramming, and immune evasion. ELK4 was identified as a key regulatory factor that markedly enhanced the proliferation, migratory capacity, and invasive potential of tumor cells, while changes in the TME were a driving force behind immune suppression and drug resistance. The findings underscored the importance of developing specific therapeutic targets, targeting metabolic reprogramming, and overcoming immune evasion, providing new theoretical foundations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94360dc80f5310630bd93066b812e27d59cfa9fb" target='_blank'>
              Integrative single-cell and spatial transcriptomics uncover ELK4-mediated mechanisms in NDUFAB1+ tumor cells driving gastric cancer progression, metabolic reprogramming, and immune evasion
              </a>
            </td>
          <td>
            Yuwei Sun, Wenyang Nie, Zhikai Xiahou, Xiaojing Wang, Wenjia Liu, Zongkai Liu, Zhiheng Lin, Zhaidong Liu
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer cell spheroids autonomously form in the ascites fluid and are considered a conduit for epithelial ovarian cancer metastasis within the peritoneal cavity. Spheroids are homotypic, avascular 3D structures that acquire resistance to anoikis to remain viable after cellular detachment. We used in vitro spheroid model systems to interrogate pathways critical for spheroid cell proliferation, distinct from those driving monolayer cancer cell proliferation. Using the 105C and KOC-7c human ovarian clear cell carcinoma (OCCC) cell lines, which have distinct proliferative phenotypes as spheroids but the same prototypical OCCC gene mutation profile of constitutively activated AKT signaling with the loss of ARID1A, we revealed therapeutic targets that efficiently kill cells in spheroids. RNA-seq analyses compared the transcriptome of 3-day monolayer and spheroid cells from these lines and identified the characteristics of dormant spheroid cell survival, which included the G2/M checkpoint, autophagy, and other stress pathways induced in 105C spheroids, in sharp contrast to the proliferating spheroid cells of the KOC-7c cell line. Next, we assessed levels of various G2/M checkpoint regulators and found a consistent reduction in steady-state levels of checkpoint regulators in dormant spheroid cells, but not proliferative spheroids. Our studies showed that proliferative spheroid cells were sensitive to Wee1 inhibition by AZD1775, but the dormant spheroid cells showed a degree of resistance to AZD1775, both in terms of EC50 values and spheroid reattachment abilities. Thus, we identified biomarkers of dormant spheroids, including the G2/M checkpoint regulators Wee1, Cdc25c, and PLK1, and showed that, when compared to proliferating spheroid cells, the transcriptome of dormant OCCC spheroids is a source of therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877fd898a1fb77cad11de61e359dcf933c175ace" target='_blank'>
              Transcriptomic Analyses of Ovarian Clear Cell Carcinoma Spheroids Reveal Distinct Proliferative Phenotypes and Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Bartlomiej Kolendowski, Sylvia Cheng, Yudith Ramos Valdés, Trevor G. Shepherd, G. DiMattia
          </td>
          <td>2025-05-27</td>
          <td>Cells</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 11508


 Background:
 Leiomyosarcoma (LMS) is a smooth muscle-derived tumor with significant heterogeneity and limited treatment options for recurrent/metastatic disease. A lack of targetable driver mutations and prognostic and predictive biomarkers have hampered the care of patients with LMS. There is a great need to better understand the biology of LMS and develop novel therapeutics. Advances in single cell RNA sequencing (scRNA-seq) have allowed for better understanding of intratumoral heterogeneity in diverse cancer subtypes. However, tissue dissociation during this process leads to loss of spatial context. Spatial gene expression analysis builds upon scRNA-seq and has the potential to yield information about tissue organization, cell-cell interactions, niches, and cell states. To date there have been limited application of spatial transcriptomics to sarcoma.
 Methods:
 We performed single nucleus multiome (snRNA-seq and snATAC-seq) on a cohort of 16 primary, untreated LMS samples including 12 soft tissue (STLMS) and 4 uterine (ULMS) tumors. We then designed a custom 480-gene panel using the differentially expressed genes from clusters identified in snRNA-seq data to be able to identify spatial relationships between these clusters and to assess these clusters on a larger scale. We utilized the 10x Genomics Xenium platform. This was applied to LMS tissue microarrays (TMAs) comprising a total of 326 tissue cores from 127 unique patients. Matched primary and metastatic samples from the same patient were available for 33 patients.
 Results:
 Analysis of scRNAseq data identified 2 distinct subtypes: a dedifferentiated subtype with mesenchymal features (MES) and a differentiated subtype with enrichment of smooth muscle cell markers (SMC). Integration of chromatin accessibility data from snATACseq showed enrichment of nuclear factor I (NFI) transcription factor (TF) motifs in the MES and AP-1 motifs in the SMC group. Whole genome sequencing did not reveal an obvious genomic etiology for these subtypes. Spatial transcriptomics was able to identify these 2 subtypes in a larger cohort of tumors. Consistent with snRNAseq data, we find that most tumors had almost exclusively either MES or SMC cells. We assessed spatial relationships between these subtypes and infiltrating immune cells. This revealed an enrichment in immunosuppressive macrophages and exhausted T cells in MES tumors compared to SMC tumors. Analysis of matched primary and metastatic tumors demonstrated that the subtype (MES or SMC) generally remains consistent between different sites of disease.
 Conclusions:
 We identify 2 novel LMS subtypes (MES and SMC) driven by distinct TFs. Spatial transcriptomic analysis confirmed the presence of these 2 subtypes in a larger cohort and demonstrated that MES tumors are associated with a more immunosuppressive tumor microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/170994843a3a0b5e571cb27c29ebdaf0b414317f" target='_blank'>
              Spatial transcriptomic profiling from over 300 leiomyosarcoma samples.
              </a>
            </td>
          <td>
            Ryan A Denu, Zhao Zheng, Veena Kochat, Yingda Jiang, William I. Padron, D. Ingram, Khalida M Wani, Larissa A Meyer, P. Soliman, R. Ratan, Alexander J. Lazar, E. Keung, Kunal Rai, E. N. Nassif Haddad
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2702678fe4bf9d99269192e19243ce5bd9b25ff1" target='_blank'>
              Patient-Derived Three-Dimensional Lung Tumor Models to Evaluate Response to Immunotherapy
              </a>
            </td>
          <td>
            Kayla F. Goliwas, Kenneth P. Hough, Sruti Sivan, Sierra L. Single, Sameer S. Deshmukh, Joel L. Berry, Maya Khalil, Benjamin Wei, Yanis Boumber, Mohammad Athar, Aakash Desai, S. Ponnazhagan, James M. Donahue, J. Deshane
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="This study explores the inflammatory response observed in pancreata and pancreatic lymph node (pLN) samples obtained throughout the natural history of type 1 diabetes (T1D) including non-diabetic individuals and non-diabetic autoantibody positive individuals with high susceptibility using spatial transcriptomics (ST). Integration of ST with public single-cell RNA sequencing data enabled interrogation of transcriptional alterations in T1D pathogenesis across both tissues and cellular scales. In the T1D pancreas, we observed global upregulation of multiple inflammation-associated transcripts, including regenerating islet-derived (REG) family genes, complement factor 3 (C3), SOD2, and OLFM4, and highlighted cellular candidates potentially contributing to these signatures. Within the T1D pLN, we observed spatially restricted upregulation of lymphotoxin-β (LTB) alongside follicular dendritic cell (FDC)-associated transcripts including FDCSP, CLU, and FCER2. Collectively, these findings highlight distinct inflammation signatures in the pancreas and regional pLN which can help inform the development of future therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d86fa210fd0a0e42f81c1f4633973c64cff969b8" target='_blank'>
              Spatial transcriptomics from pancreas and local draining lymph node tissue reveals a lymphotoxin-β signature in human type 1 diabetes
              </a>
            </td>
          <td>
            Miguel A. Medina-Serpas, Maigan A. Brusko, Gregory J. Golden, Martha Campbell-Thompson, Trevor Rogers, A. Posgai, Eline T. Luning Prak, Chengyang Liu, Klaus H. Kaestner, Ali Naji, Michael R. Betts, Lauren M McIntyre, M. Atkinson, Todd M. Brusko
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d66d1c34039c0624a67bfae6ffcc0194c9308ee" target='_blank'>
              Bioelectric and Epigenetic Landscapes in Lateralized Breast Tumors Reveal Distinct Tumor Microenvironment Signatures
              </a>
            </td>
          <td>
            Sebastián Real, Sergio R. Laurito, Pablo Gonzalez, Oscar Bello, Joao Carvalho, María Roqué
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/104badcf54cd0ff8c28f3f2bf4d66c298f91b54a" target='_blank'>
              CSFeatures improves identification of cell type-specific differential features in single-cell and spatial omics data
              </a>
            </td>
          <td>
            Rufeng Li, Yongkang Li, Heyang Hua, Zhen Li, Yu Zhang, Yufei Yao, Shengquan Chen, Yungang Xu
          </td>
          <td>2025-05-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Angiosarcoma, a rare and highly aggressive malignancy originating from vascular endothelial cells, is characterized by its rapid progression, high invasiveness, and poor prognosis. Due to the limited understanding of its tumor microenvironment (TME) and the absence of effective treatments, further research is essential to elucidate its pathogenic mechanisms and improve therapeutic strategies. Objective This study aims to characterize the cellular heterogeneity and unique TME of primary breast angiosarcoma using single-cell RNA sequencing (scRNA-seq), to identify potential therapeutic targets and improve clinical outcomes. Methods Tumor samples were obtained from a patient with bilateral primary breast angiosarcoma and two patients with invasive breast cancer. Single-cell RNA sequencing (scRNA-seq) was conducted to capture the transcriptomic profiles of individual cells within the tumor samples. Following stringent quality control, a total of 31,771 cells were analyzed using comprehensive bioinformatics approaches. Cell populations were identified and classified into distinct cell types, and differential gene expression analysis was performed to explore key signaling pathways. Functional enrichment analysis was used to identify pathways related to tumor progression and immune evasion. Additionally, cell-cell communication networks were mapped to understand interactions within the TME, with a focus on pathways that may serve as therapeutic targets. Results The scRNA-seq analysis revealed significant differences in the distribution of perivascular cells, fibroblasts, T cells, endothelial cells, and myeloid cells in breast angiosarcoma compared to invasive breast cancer. Key pathways enriched in angiosarcoma samples included growth factor binding, platelet-derived growth factor binding, and ribosome biogenesis, with abnormal expression of several ribosomal proteins. Notably, genes such as FAT4, KDR, FN1, and KIT were highly expressed in angiosarcoma endothelial cells, correlating with poor prognosis. Cell communication analysis highlighted the CXCL12-CXCR4 axis as a crucial mediator of the TME in angiosarcoma. Conclusion This study provides critical insights into the TME of primary breast angiosarcoma, highlighting potential molecular targets and pathways for therapeutic intervention. These findings may inform the development of more effective treatment strategies for this rare and challenging tumor type. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02022-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61afd778df17e939f7a978e09a799d8e740668a2" target='_blank'>
              Dissecting the tumor microenvironment in primary breast angiosarcoma: insights from single-cell RNA sequencing
              </a>
            </td>
          <td>
            Peikai Ding, Shengbin Pei, Yi Zhai, Zheng Qu, Yazhe Yang, Xiaolong Feng, Qiang Liu, Xiangyu Wang, Wenxiang Zhang, Zhongzhao Wang, Xiangyi Kong, Jing Wang, Yi Fang
          </td>
          <td>2025-06-05</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Breast cancer progression and metastasis involve the action of multiple transcription factors in tumors and in the cells of the tumor microenvironment (TME) and understanding how these transcription factors are coordinated can guide novel therapeutic strategies. Myocardin-related transcription factors A and B (MRTFA/B also known as MKL1/2) are two related transcription factors that redundantly control cancer cell invasion and metastasis in mouse models of breast cancer, but their roles in human cancer are incompletely understood. Here, we investigated the expression and activation of these transcription factors to better assess their tumorigenic and metastatic impact on breast cancer and cells of the tumor microenvironment. Methods We used a multiplexed immunofluorescence approach to label MRTFA, MRTFB, tumor cells by using pan Cytokeratin, endothelial cells by using CD31, and antigen presenting cells (APCs) by using HLA-DRA on two different breast cancer tissue microarrays (TMA): The breast cancer progression TMA provided by the Cooperative Human Tissue Network (CHTN_BrCaProg3) and the University of Illinois Breast Cancer Working Group (TMA BCWG UIC-001-TMA) that included primary tumor and lymph node metastases from patients residing in the West Side and South Side of Chicago. We also used bioinformatics analyses of the TCGA and METABRIC databases and the Broad Institute’s single-cell RNA sequencing portal to investigate MRTFA/B expression patterns in the cells of the tumor microenvironment (TME). Results We found that in human tumors, MRTFA/B are concurrently activated in cancer cells, but they show distinct patterns of expression across different histological subtypes and in the cells of the TME. Importantly, MRTFA expression was elevated in metastatic tumors of African American patients, who disproportionately die from breast cancer. Interestingly, in contrast to publicly available mRNA expression data, MRTFA was similarly expressed across estrogen receptor (ER) positive and negative breast tumors, while MRTFB expression was highest in ER+ breast tumors. Furthermore, MRTFA was specifically expressed in the perivascular antigen-presenting cells (APCs), which has been previously associated with immune suppression and breast cancer progression. We also found that MRTFA expression correlated with the expression of the immune checkpoint protein V-set immunoregulatory receptor (VSIR) in the TCGA data and found that MRTFA activity promotes VSIR expression in THP-1 monocytes and cultured HEK293 cells. Conclusions Our results provide unique insights into how MRTFA and MRTFB promote metastasis in human cancer, the differences of their expression patterns, and their immune suppressive function within the breast cancer TME. Our results will guide future studies on targeting MRTFA/B transcriptional activity and the resulting immune suppression in breast cancer. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06559-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9b888786c13a4f36a60caad28e0f780e4db8c77" target='_blank'>
              Multiplex imaging reveals novel patterns of MRTFA/B activation in the breast cancer microenvironment
              </a>
            </td>
          <td>
            Stephanie M. Wilk, Kihak Lee, Caitlyn C. Castillo, Mohamed Haloul, Alexa M. Gajda, Virgilia Macias, Elizabeth L. Wiley, Zhengjia Chen, Xinyi Liu, Xiaowei Wang, Maria Sverdlov, Kent F Hoskins, E. E. Er
          </td>
          <td>2025-05-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Introduction Medullary thyroid carcinoma (MTC) is a rare and aggressive thyroid cancer with a challenging prognosis. While the immune microenvironment plays a crucial role in cancer progression, its role in MTC remains underexplored compared to more common thyroid cancers. Methods this study investigates the immune landscape of MTC by systematically evaluating immune cell infiltration and expression of immune markers across various tissue topographies. We utilized advanced immunohistochemical techniques to analyze tissue samples from 24 MTC patients, focusing on the tumor core, interface with healthy tissue, adjacent normal thyroid tissue, and lymph node metastases. Results our findings reveal a distinct immune profile with increased CD3+, CD4+, CD8+ and CD20+ lymphocytes in normal tissues adjacent to tumors and a notable presence of granzyme B+ cells in the tumor interface, particularly in patients with structural disease. Additionally, we observed a significant enrichment of mast cells in metastatic tissues. Discussion these results highlight the complex and spatially dependent immune landscape of MTC, suggesting implications for targeted immunotherapy. This study provides novel insights into the immune microenvironment of MTC and emphasizes the need for further research to elucidate its impact on disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8ab2d28852a497483bb6cba642f6c18c52966ea" target='_blank'>
              Spatial immune profiling reveals distinct microenvironments in medullary thyroid carcinoma
              </a>
            </td>
          <td>
            Maria Eduarda de Castro, Gustavo Forlin de Siqueira, Jean Ferrante Mariano, M. Kizys, L. Ceolin, Fernando Augusto Soares, Rodrigo Natal, Humberto Carneiro, Rodrigo Nalio Ramos, Laura Sterian Ward, Niels Olsen Saraiva Camara, Cleber Pinto Camacho, Flavia de Oliveira Facuri Valente, Susan C. Lindsey, Diego Dias dos Santos, Cristiane Damas Gil, João Roberto Maciel Martins, Rui Monteiro de Barros Maciel, L. L. Cunha
          </td>
          <td>2025-05-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cervical cancer represents a significant global health challenge, with complex cellular and molecular mechanisms driving its progression from HPV infection to invasive malignancy. This study employed an integrated approach combining single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (stRNA-seq) to comprehensively characterize the tumor microenvironment (TME) across different stages of cervical cancer development. Through analysis of samples from normal cervix, HPV-infected normal cervix, high-grade squamous intraepithelial lesions (HSIL), and invasive cervical cancer, we identified distinct cellular populations and their dynamic changes during disease progression. Our findings revealed significant heterogeneity in immune cell populations, particularly highlighting the role of SPP1+ macrophages that were substantially enriched in cervical cancer compared to precancerous and normal tissues. Cell-cell communication networks and spatial mapping demonstrated that SPP1+ macrophages interact extensively with immune cells through the SPP1-CD44 signaling axis. This interaction contributes to an immunosuppressive microenvironment through modulation of T cell function and promotion of tumor cell survival. Furthermore, high expression of SPP1 correlated with advanced tumor stages and poor overall survival in cervical cancer patients, highlighting its potential as a prognostic biomarker. Our comprehensive characterization of the cellular landscape and intercellular communication networks in cervical cancer progression provides valuable insights for the development of targeted therapeutic strategies aimed at modulating the TME, particularly through disruption of the SPP1-CD44 axis. These findings establish a foundation for more effective personalized approaches to improve clinical outcomes in cervical cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87cbb4dd12be6f1664aacd1df3e0b1efa6fe30d7" target='_blank'>
              Deciphering the cellular and molecular landscape of cervical cancer progression through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Peng Xia, Juanhong Zhou, Rong Shen, Degui Wang
          </td>
          <td>2025-05-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) are functionally diverse stromal regulators that orchestrate tumor progression, metastasis, and therapy resistance through dynamic crosstalk within the tumor microenvironment (TME). Recent advances in single‐cell multiomics and spatial transcriptomics have identified conserved CAF subtypes with distinct molecular signatures, spatial distributions, and context‐dependent roles, highlighting their dual capacity to promote immunosuppression or restrain tumor growth. However, therapeutic strategies struggle to reconcile this functional duality, hindering clinical translation. This review systematically categorizes CAF subtypes by origin, biomarkers, and TME‐specific functions, focusing on their roles in chemoresistance, maintenance of stemness, and formation of immunosuppressive niches. We evaluate emerging targeting approaches, including selective depletion of tumor‐promoting subsets (e.g., fibroblast activation protein+ CAFs), epigenetic reprogramming toward antitumor phenotypes, and inhibition of CXCL12/CXCR4 or transforming growth factor‐beta signaling pathways. Spatial multiomics‐driven combinatorial therapies, such as the synergistic use of CAFs and immune checkpoint inhibitors, are highlighted as strategies to overcome microenvironment‐driven resistance. By integrating CAF biology with translational advances, this work provides a roadmap for developing subtype‐specific biomarkers and precision stromal therapies, directly informing efforts to disrupt tumor‐stroma coevolution. Key concepts include spatial transcriptomics, stromal reprogramming, and tumor‐stroma coevolution, offering actionable insights for both mechanistic research and clinical innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d8f3a8f9589a88ea165cf34934eda0905c9005" target='_blank'>
              Cancer‐Associated Fibroblasts: Heterogeneity, Cancer Pathogenesis, and Therapeutic Targets
              </a>
            </td>
          <td>
            Yutao Li, Qingyun Liu, Xilin Jing, Yuqi Wang, Xiaohua Jia, Xing Yang, Kezhong Chen
          </td>
          <td>2025-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pediatric low-grade gliomas (pLGG) comprise 35% of all brain tumors. Despite favorable survival, patients experience significant morbidity from disease and treatments. A deeper understanding of pLGG biology is essential to identify novel, more effective, and less toxic therapies. We utilized single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and cytokine analyses to characterize and understand tumor and immune cell heterogeneity of pilocytic astrocytoma (PA) and ganglioglioma (GG). scRNA-seq revealed tumor and immune cells within the tumor microenvironment (TME). Tumor cell subsets include both progenitor and mature cell populations. Immune cells included myeloid and lymphocytic cells. There was a significant difference between the prevalence of two major myeloid subclusters between PA and GG. Bulk and single-cell cytokine analyses evaluated the immune cell signaling cascade with distinct immune phenotypes among tumor samples. KIAA1549-BRAF tumors appeared more immunogenic, secreting higher levels of immune cell activators and chemokines, compared to BRAF V600E tumors. Spatial transcriptomics revealed the differential gene expression of these chemokines and their location within the TME. A multi-pronged analysis demonstrated the complexity of the PA and GG TME and differences between genetic drivers that may influence their response to immunotherapy. Further investigation of immune cell infiltration and tumor-immune interactions is warranted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dc01e4fe40878b7885262fd79ea7216b56b949" target='_blank'>
              Multi-pronged analysis of pediatric low-grade glioma and ganglioglioma reveals a unique tumor microenvironment associated with BRAF alterations.
              </a>
            </td>
          <td>
            S. Zahedi, Kent Riemondy, Tian Liu, A. Griesinger, A. Donson, April A Apfelbaum, Rui Fu, Julian Grandvallet Contreras, M. Crespo, J. DeSisto, Madeline Groat, Emil Bratbak, Adam L Green, Todd C Hankinson, Michael Handler, Rajeev Vibhakar, Nicholas Willard, Nicholas K Foreman, Tzu Phang, Jean M. Mulcahy Levy
          </td>
          <td>2025-06-30</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background Precursor lesions like polyps and adenomas in the colon commonly precede colorectal cancer (CRC), advancing through the “normal-polyp-adenoma-carcinoma” sequence towards malignancy. Yet, the cellular heterogeneity and molecular mechanisms involved in CRC development remain inadequately characterized. Methods To understand the molecular mechanisms driving the onset and progression of CRC, we conducted a comprehensive analysis of ten clinical colorectal samples representing sequential pathological stages using single-cell RNA sequencing (scRNA-seq). Validation was performed through immunofluorescence and immunohistochemistry analyses in a separate human colorectal tissue cohort. Additional verification was carried out using bioinformatics analyses of public TCGA and GEO datasets. Results Our comprehensive analyses not only reveal the cellular diversity and transcriptomic differences throughout disease progression but also highlight the importance of leveraging ligand–receptor gene expression to distinguish various cell subtypes. Subsequent examination and validation with a larger sample cohort uncover the specific involvement of ligand–receptor genes, transcription factors, immunoglobulin genes, and heat shock genes in regulating immune responses and microenvironment changes during colorectal tumorigenesis. Conclusions Our extensive transcriptome dataset provides valuable insights and acts as a fundamental resource to deepen our understanding of the complex molecular landscape in CRC. This dataset facilitates improved diagnostic accuracy and the creation of more personalized therapeutic approaches. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06785-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e15cc889bc4d3cf181a1a8c15dbc72bc14a7b3" target='_blank'>
              A dynamic molecular landscape in colorectal cancer progression at single-cell resolution
              </a>
            </td>
          <td>
            Jianwen Sheng, Nianshuang Li, Shuai Li, Yuman Ye, Yuanhang Liao, Yupeng Zhang, Huan Chen, Mengxian Tu, Guping Zhong, Chaoliang Xiong, Pan Zheng, Yuting Lei, Zekun He, Xingxing He, Yao Zhang, Lei Wang, Xiaolin Gao, Yin Zhu, Jianping Liu, Huizhen Fan
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Traditional gene expression deconvolution methods assess a limited number of cell types, therefore do not capture the full complexity of the tumor microenvironment (TME). Here, we integrate nine deconvolution tools to assess 79 TME cell types in 10,592 tumors across 33 different cancer types, creating the most comprehensive analysis of the TME. In total, we found 41 patterns of immune infiltration and stroma profiles, identifying heterogeneous yet unique TME portraits for each cancer and several new findings. Our findings indicate that leukocytes play a major role in distinguishing various tumor types, and that a shared immune-rich TME cluster predicts better survival in bladder cancer for luminal and basal squamous subtypes, as well as in melanoma for RAS-hotspot subtypes. Our detailed deconvolution and mutational correlation analyses uncover 35 therapeutic target and candidate response biomarkers hypotheses (including CASP8 and RAS pathway genes).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6b1e17d9e760b6c7495c8a6aea0db799ec80899" target='_blank'>
              Pan-cancer immune and stromal deconvolution predicts clinical outcomes and mutation profiles
              </a>
            </td>
          <td>
            B. Bhinder, Verena Friedl, Sunantha Sethuraman, Davide Risso, Kami E. Chiotti, R. J. Mashl, Kyle Ellrott, Jordan A. Lee, Christopher K. Wong, K. Gyan, Aditya Deshpande, Marcin Imielinski, R. Bareja, Josh Stuart, Myron Peto, K. Hoadley, Alexander J. Lazar, A. Cherniack, Jingchun Zhu, Shaolong Cao, Mark Rubin, Wenyi Wang, Oliver F. Bathe, N. Robine, Li-hua Ding, Peter W. Laird, Wanding Zhou, Hui Shen, V. Thorsson, Jen Jen Yeh, Matthew H. Bailey, Daniel Cui Zhou, X. Peng, M. Goldman, Yongsheng Li, Anil Korkut, Nidhi Sahni, D. N. Hayes, Michael K. A. Mensah, Ina Felau, Anab Kemal, Samantha J. Caesar-Johnson, John A. Demchok, Liming Yang, M. Ferguson, R. Tarnuzzer, Zhining Wang, J. Zenklusen, Adam Alana Andrea Andrew Angela Benjamin Benjamin Brian Margolin Weinstein Sboner Blair Brooks Berman Raph, Adam Margolin, Alana Weinstein, A. Sboner, Andrew Blair, Angela Brooks, Benjamin Berman, Benjamin Raphael, Brian Craft, Dante S. Bortone, David Heimain, David L Gibbs, David Haan, D. Betel, Duncan McColl, Emek Demir, Faeze Brahman, F. Farshidfar, Gaddy Getz, G. Gao, Gordon Robertson, Huy Dinh, Hyo Young Choi, I. Shmulevich, Ioannis Anastopoulous, Jasmine Yang, Jeff Damrauer, Ken Chen, Lauren Sanders, Lee Cooper, Liang-Bo Wang, Matt Reyna, Mike Noble, Mohammed El-Kebir, Molly Zhang, Nicole Yeager, Paul Little, Richard Moffitt, S. Meier, Teresa Swatloski, T. Knijnenburg, Thomas Matthew, Vicky Chen, Vlado Uzunangelov, Xinghua Lu, Yige Wu, Yulia Newton, Zeya Wang, Paul T. Spellman, O. Elemento
          </td>
          <td>2025-07-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5484870331a7d0dee6989fa1ddf5ea8e0b01c6cb" target='_blank'>
              Functional genomics and tumor microenvironment analysis reveal prognostic biological subtypes in Mantle cell lymphoma
              </a>
            </td>
          <td>
            Sunandini Sharma, Roshia Ali, Alyssa Bouska, Dylan Jochum, Meghana Kesireddy, Simeon Mahov, Joseph Lownik, Weiwei Zhang, Waseem G Lone, Mahfuza Afroz Soma, Alicia Gamboa, Vaishnavi Devarkonda, Dalia Elgamal, Atqiya Fariha, Adnan Mansoor, Douglas Stewart, Peter Martin, Brian K. Link, R. Advani, Paul M. Barr, A. Goy, A. Mehta, M. Kamdar, D. Stephens, Veronika Bachanova, L. Smith, Ryan D Morin, P. Pararajalingam, M. Lunning, Kai Fu, Dennis Wiesenberger, Wing C. Chan, J. Khoury, T. Greiner, J. Vose, Akil Merchant, Chengfeng Bi, J. Iqbal
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="
 Cancer treatment options including small molecules, antibody-drug conjugates (ADCs), CAR T-cells, and combination therapies are rapidly expanding to improve cancer patient survival. However, efficacy of targeted therapies is limited by interpatient and intratumoral heterogeneity in breast cancer, and may be further exacerbated by dysregulated tumor microenvironments. For example, studies suggested that changes in tumor and breast tissue microenvironment in obesity (e.g. inflammation, increased growth factors) could lead to worsened prognosis that could be related to therapy resistance. Systematic characterization of the cellular effects of emerging drug candidates, particularly across different tumor microenvironments, is challenging, as it needs to be surveyed in diverse cancer contexts beyond just a handful of cancer cell lines. To overcome this challenge, we created PRISM (Profiling Relative Inhibition Simultaneously in Mixtures), an effective drug discovery screening platform with over 900 genetically distinct cancer cell lines of diverse lineages, to rapidly test therapeutics in a high-throughput manner. These cell lines, including 33 different breast cancer cell lines covering diverse cell types and subgroups, carry a specific 24 nucleotide “barcode” tag and have rich baseline genomic and functional characterization. They are first pooled into 20-25 cell pools of different lineages with similar growth rates, then subsequently cultured together with or without drugs. The relative barcode quantity read out by NextGen sequencing determines cell line responses to drugs. This pooled approach significantly reduces the time and the resources required for drug screening. Importantly, utilizing the rich cell line characterization to interpret viability profiles enables the identification of drivers of differential sensitivity and potential biomarkers of compound response. Furthermore, we created a breast cancer cell specific pool to investigate how media conditions mirroring different tumor microenvironments [e.g. media filtration, addition of insulin (0.01 mg/mL) or EGF (0.01 ug/mL)] can impact drug effects on breast cancer cells. In the PRISM 900+ cell panel treated with ADCs conjugated to different payloads that target HER2, there was a clear association between cell line sensitivity and ERBB2 (encoding HER2) gene expression, with the strongest effects on viability observed in a subset of cell lines overexpressing ERBB2. Upon further comparison of biomarker profiles across different ADCs, we established relationships between payload resistance and ABCB1 efflux pumps emerged for a subset of ADC payloads. Using the breast cancer cell specific pool, our data showed that estrogen receptor (ER) degraders fulvestrant and vepdegestrant (ARV-471) had significantly greater effects in reducing viability of ER+ versus ER- breast cancer cell lines. Media filtering reduced the drug effects of fulvestrant but not ARV-471, suggesting that ARV-471 efficacy may be less sensitive to the tumor microenvironment than fulvestrant. Lapatinib (HER2/neu and EGF receptor inhibitor) was more effective in killing HER2+ than HER2- negative breast cancer cells, with cells cultured in EGF-spiked media being more sensitive to lapatinib, whereas media filtration diminished its effects. We did not observe breast cancer subgroup specific killing by alpelisib (PI3K inhibitor) or olaparib (PARP inhibitor), but breast cancer cells became less sensitive to alpelisib when cultured in either insulin- or EGF-spiked media, and less sensitive to olaparib in EGF-spiked media. Overall, our study highlights how the PRISM 900+ cell panel can provide insights into drug specificity, cancer subtype selectivity, and to uncover clinically relevant targets. The breast cancer specific pool can further increase the throughput of understanding how drug responses may be altered in different tumor microenvironments in breast cancer.
 Citation Format: Blanche C. Ip, Jillian N. Eskra, Tenzin Sangpo, Ashish Bino George, Melissa Ronan, Matthew G. Rees, Jennifer A. Roth. PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-10-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5ec03b1ee18de91282ff62b13d5cb24a1236003" target='_blank'>
              Abstract P4-10-26: PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells
              </a>
            </td>
          <td>
            Blanche C. Ip, Jillian N. Eskra, Tenzing Sangpo, Ashish Bino George, M. Ronan, M. Rees, Jennifer A. Roth
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Emerging spatial profiling technologies have revolutionized our understanding of how tissue architecture shapes disease progression, yet the contribution of cellular diversity remains underexplored. Here, Ding and colleagues introduce multiomics and ecological spatial analysis (MESA), an ecology-inspired framework that integrates spatial and single-cell expression data to quantify tissue diversity across multiple scales. MESA both identifies distinct cellular neighborhoods and computes a variety of diversity metrics alongside the identification of diversity "hotspots". Applied to human tonsil tissue, MESA revealed previously undetected germinal center organization, while in spleen tissue of a murine lupus model, MESA highlights increasing cellular diversity with disease progression. Importantly, diversity hotspots do not correspond to conventional compartments identified by existing methods, presenting an orthogonal metric of spatial organization. In colorectal cancer, MESA's diversity metrics outperformed established subtypes at predicting patient survival, while in hepatocellular carcinoma, multi-omic integration identified significantly more ligand-receptor interactions between immune cells compared to single-modality analysis. This work establishes cellular diversity within tissues as a critical correlate of disease progression and underscores the value of multi-omic integration in spatial biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eb774cd8ef924c70b07f117ccf0af3bd80ea8f6" target='_blank'>
              Cells keep diverse company in diseased tissues.
              </a>
            </td>
          <td>
            Kieran R. Campbell, A. Goeva
          </td>
          <td>2025-05-16</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/378e143896f70bf1c18f50eea249337f3e7bfc52" target='_blank'>
              Single-Cell RNA Sequencing and Inferred Protein Activity Analysis Reveal a Distinct Tumor Phenotype in Early-Onset Colorectal Cancer Patients
              </a>
            </td>
          <td>
            David A. Cohen, Aleksandar Z Obradovic, Luka Jovanovic, Yosuke Mitani, Tianxia Li, Soobeom Lee, Dechokyab De, Casidhe-Nicole R Bethancourt, Alice Shin, Karen Dunbar, Francesco Cambuli, Christopher Lengner, Ning Li, Caitlin Rogers, Jennifer Ho, Viktoria Jakubikova, Emma Blystone, Yoanna S. Pumpalova, Beatrice Dionigi, Sanja Vickovic, Chia-Wei Cheng, Joel Gabre
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Understanding the heterogeneity of Rheumatoid Arthritis (RA) and identifying therapeutic targets remain challenging using traditional bulk transcriptomics alone, as it lacks the spatial and protein-level resolution needed to fully capture disease and tissue complexities. In this study, we applied Laser Capture Microdissection (LCM) coupled with mass spectrometry-based proteomics to analyze histopathological niches of the RA synovium, enabling the identification of protein expression profiles of the diseased synovial lining and sublining microenvironments compared to their healthy counterparts. In this respect, key pathogenetic RA proteins like membrane proteins (TYROBP, AOC3, SLC16A3, TCIRG1, and NCEH1), and extracellular matrix (ECM) proteins (PLOD2, OGN, and LUM) showed different expression patterns in diseased synovium compartments. To enhance our understanding of cellular dynamics within the dissected regions, we further integrated the proteomic dataset with single-cell RNA sequencing (scRNA-seq), and deduced cell type enrichment, including T cells, fibroblasts, NK cells, myeloid cells, B cells, and synovial endothelial cells. By combining high-resolution spatial proteomics and transcriptomic analyses, we provide novel insights into the molecular mechanisms driving RA, and highlight potential protein targets for therapeutic intervention. This integrative approach offers a more comprehensive view of RA synovial pathology, and mitigates the limitations of traditional bulk transcriptomics in target discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92de30759136153ce270d6dfee3613164882bb0" target='_blank'>
              Integrative Spatial Proteomics and Single-Cell RNA Sequencing Unveil Molecular Complexity in Rheumatoid Arthritis for Novel Therapeutic Targeting
              </a>
            </td>
          <td>
            Xue Wang, Fei Wang, Archana S. Iyer, Heather Knight, Lori J. Duggan, Yingli Yang, Liang Jin, Baoliang Cui, Yupeng He, Jan Schejbal, Lucy A. Phillips, Bohdan P. Harvey, Sílvia Sisó, Yu Tian
          </td>
          <td>2025-05-22</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca80c0d5c8118b6afa306ae5816ee0be1c3263a" target='_blank'>
              Single-cell transcriptomics reveals hypoxia-driven iCAF_PLAU is associated with stemness and immunosuppression in anorectal malignant melanoma.
              </a>
            </td>
          <td>
            Hao Zhang, Lin Gan, Xiaofei Duan, Baojing Tuo, Hao Zhang, Senbo Liu, Yugui Lian, Enjie Liu, Zhenqiang Sun
          </td>
          <td>2025-07-04</td>
          <td>Journal of gastroenterology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer drug resistance is multi-factorial, driven by heritable (epi)genetic changes but also phenotypic plasticity. Here, we dissected drivers of resistance by perturbing colorectal cancer patient-derived organoids longitudinally with drugs in sequence. Combined longitudinal lineage tracking, single cell multi-omics analysis, evolutionary modelling, and machine learning revealed that different targeted drugs select for distinct subclones, supporting rationally designed drug sequences. The cellular memory of drug resistance was encoded as a heritable epigenetic configuration from which multiple transcriptional programs could run, supporting a one-to-many (epi)genotype-to-phenotype map that explains how clonal expansions and plasticity manifest together. This epigenetic landscape may ensure drug resistant subclones can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding for their selective advantage. Chemotherapy resistance was instead entirely driven by transient phenotypic plasticity rather than stable clonal selection. Inducing further chromosomal instability before drug application changed clonal evolution but not convergent transcriptional programs. Collectively, these data show how genetic and epigenetic alterations are selected to engender a "permissive epigenome" that enables phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83269ee41c4604edc0996e3b22f4bf8776d7c6" target='_blank'>
              Epigenetic Heritability of Cell Plasticity Drives Cancer Drug Resistance through a One-to-Many Genotype-to-Phenotype Paradigm.
              </a>
            </td>
          <td>
            É. Oliveira, Salvatore Milite, J. Fernández-Mateos, George D. Cresswell, Erika Yara-Romero, G. Vlachogiannis, Bingjie Chen, Chela James, Lucrezia Patruno, Gianluca Ascolani, Ahmet Acar, Timon Heide, Inma Spiteri, Alex Graudenzi, G. Caravagna, Andrea Bertotti, Trevor A. Graham, Luca Magnani, Nicola Valeri, A. Sottoriva
          </td>
          <td>2025-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are the most abundant non-cancerous cell type in glioblastoma (GBM) and heavily influence GBM biology, contributing to tumor progression, therapeutic resistance, immune evasion, and neovascularization. Current in vitro models that utilize IL-4/IL-13 stimulation fail to capture the transcriptional and functional heterogeneity of TAMs observed in vivo. In this study, we utilize a serum-free indirect co-culture model with patient-derived xenolines to polarize primary human macrophages and characterize their molecular and functional phenotypes. We demonstrate that xenoline-polarized macrophages diverge from classical M1/M2 states and instead adopt transcriptional signatures reflective of TAM subsets identified from patients. Notably, macrophages polarized with the radiation-therapy selected xenoline, JX14P-RT, exhibited gene expression patterns enriched for interferon response and hypoxia, mirroring recurrent GBM samples. In contrast, JX14P TAMs showed enrichment in phagocytic gene sets. Functional validation of these phenotypes revealed discrepancies between the transcriptionally predicted and observed phenotypes, emphasizing the importance of integrating phenotypic validation in sequencing studies. Altogether, our findings establish xenoline-polarized macrophages as a useful alternative to traditional models that can be used to study immune-interactions in vitro. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02057-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f15f733124e62cedabe127db49de61b0e9f489" target='_blank'>
              Xenoline-polarized macrophages as an alternative in vitro model of tumor-associated macrophages in glioblastoma
              </a>
            </td>
          <td>
            Acta Neuropathologica Communications, Christopher D. Willey, Hasan Alrefai, Benjamin Lin, Amr Elkholy, Manoj Kumar, Taylor L. Schanel, Kevin J. Lee, Patricia H. Hicks, Joshua C. Anderson, Gao Guo, Eun-Young Erin Ahn, Ryan Miller
          </td>
          <td>2025-06-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 e17536


 Background:
 High-grade cervical intraepithelial neoplasia (HG-CIN) affects 200,000 U.S. women annually, with rising prevalence among those aged 30 & older, many of whom missed the opportunity for HPV vaccination. Treatment remains predominantly surgical, with a notable absence of pharmacologic interventions. Clinical trial endpoints rely on histological regression, but standard microscopy cannot accurately distinguish between regressing and progressing lesions. Compelling evidence underscores the tumor microenvironment’s (TME) critical role in driving immune suppression and malignant transformation. Leveraging a novel spatial imaging platform, this study provides high-resolution molecular insights into HG-CIN directly from routine H&E-stained slides, enabling precise characterization of immune marker distributions and their implications for disease progression. This technology has the potential to transform tissue analysis and significantly impact drug development.
 Methods:
 Sixteen formalin-fixed paraffin-embedded (FFPE) tissues were obtained from the health system’s Biospecimen Repository, with twelve used to optimize the sequencing protocol directly from H&E slides. Sample was categorized as CIN3, benign, or micro-invasive cervical mucosa based on the history, Pap smear/HPV test results, and an expert review by a Gynecologic Pathologist. Library generation and sequencing were conducted using Visium Gene Expression (both SD & HD platforms) kits for FFPE (10x Genomics). Specific genetic markers for immunosuppressive cell activities were highlighted to demonstrate its differential presence in CIN3 vs normal cervical tissue.
 Results:
 Sixteen FFPE samples were analyzed, with 11 used to optimize sequencing protocols. Three CIN3, three normal, and one micro-invasive cervical cancer sample revealed distinct tumor, stromal, and immune microenvironments. CIN3 tissues had elevated Tregs markers (FOXP3, CD4), MDSCs (CD11b, CD33, HLA-DR), and the Treg trafficking molecules CCR7 and TGF-β within stromal regions. Additionally, CIN3 had high gene expression for CDKN2A, which encodes the immunosuppressive ligand PDL1.
 Conclusions:
 The CIN and cancer microenvironment demonstrated strong immunosuppressive markers across immune, stromal, and epithelial compartments, aligning with key known features of tumorigenesis and progression. This spatial imaging platform & optimized workflow captures biologically relevant data directly from standard routinely stained H&E slide, achieving near single-cell resolution, enabling histopathologic comparison to gene expression data. Our study revealed unique immune suppression profiles in CIN3 linked to HPV progression, highlighting the platform’s potential to transform tissue analysis and advance drug development applications.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65b12713b597d9f017570b06219fcd74b91e4800" target='_blank'>
              Spatial transcriptomics for profiling the microenvironment of cervical precancer progression from routine H&E slides: Applications for pharmaceutical development.
              </a>
            </td>
          <td>
            Mario R Castellanos, C. Regan, Seema Khutti, Kai Wang, M. Dhar, Jonathan Preall
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cell plasticity is a hallmark of cancer, enabling tumor cells to acquire multiple phenotypes responsible for tumor progression, metastasis, and therapy resistance. In this issue of the JCI, Kawai and colleagues leveraged genetically engineered mouse models (GEMM) of pancreatic ductal adenocarcinoma (PDAC) to demonstrate that loss of Pbrm1, a member of the SWI/SNF complex, drives dedifferentiation and aggressive tumor features. Pbrm1 loss activated a program of epithelial-to-mesenchymal transition (EMT) and allowed the emergence of poorly differentiated histologies that are commonly associated with high recurrence rate and dismal prognosis. These findings reveal the role of the SWI/SNF complex during PDAC evolution in maintaining cell identity and restraining the progression of this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51cf4e0b352b84ce0ee22909d350d6fe7531e126" target='_blank'>
              Switching on the evolutionary potential of pancreatic cancer: the tumor suppressor functions of PBRM1
              </a>
            </td>
          <td>
            Luigi Perelli, G. Genovese
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Radiomics is undergoing a paradigm shift from single-omics to multi-omics, from single-temporal to multi-temporal analysis, and from global to subregional analysis. These transformations have shown great potential in addressing key challenges related to imaging changes before and after neoadjuvant chemotherapy (NAC) in breast cancer. Furthermore, radiomics has achieved remarkable progress in tasks such as exploring tumor heterogeneity and uncovering underlying biological mechanisms. Integrating imaging data with gene data offers novel perspectives for understanding imaging changes driven by specific genetic alterations. However, current radiomics studies on neoadjuvant chemotherapy for breast cancer have not yet achieved a close integration of imaging changes with underlying biological mechanisms. They are largely limited to simple associations between models and genomic data, without in-depth interpretation of the biological significance inherent in imaging features, which is essential to directly link these features with the dynamic progression of the disease. This review seeks to explore the spatial-temporal heterogeneity of imaging alterations observed during NAC for breast cancer, while assessing their biological implications using established analytical approaches. It highlights the distinct advantages of spatial-temporal radiomics in predictive model development and examines potential correlations between imaging dynamics and gene expression profiles before and after NAC. Additionally, we critically examines previous radiogenomics studies, providing theoretical insights into their limitations. Finally, the review proposes future directions and innovative approaches for applying spatial-temporal radiogenomics in NAC for breast cancer, serving as a valuable reference and roadmap for researchers and clinical practitioners in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1413564d0b7a7765ec86c504011b8f276395913" target='_blank'>
              Spatial-temporal radiogenomics in predicting neoadjuvant chemotherapy efficacy for breast cancer: a comprehensive review
              </a>
            </td>
          <td>
            Tingfeng Zhang, Liang Zhao, Tingting Cui, Yi Zhou, Peifen Li, Chuqiao Luo, Junkang Wei, Hong Hu
          </td>
          <td>2025-06-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Non–small cell lung cancer (NSCLC) constitutes over 80% of lung cancer cases and remains a leading cause of cancer-related mortality worldwide. Despite the advent of immune checkpoint inhibitors, their efficacy is limited to 27 to 45% of patients. Identifying likely treatment responders is essential for optimizing healthcare and improving quality of life. We generated multiplex immunofluorescence (mIF) images, histopathology, and RNA sequencing data from human NSCLC tissues. Through the analysis of mIF images, we characterized the spatial organization of 1.5 million cells based on the expression levels for 33 biomarkers. To enable large-scale characterization of tumor microenvironments, we developed NucSegAI, a deep learning model for automated nuclear segmentation and cellular classification in histology images. With this model, we analyzed the morphological, textural, and topological phenotypes of 45.6 million cells across 119 whole-slide images. Through unsupervised phenotype discovery, we identified specific lymphocyte phenotypes predictive of immunotherapy response. Our findings can improve patient stratification and guide selection of effective therapeutic regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebcac8871256527456ef7fc38ff937674e4c6cc4" target='_blank'>
              Single-cell multimodal analysis reveals tumor microenvironment predictive of treatment response in non–small cell lung cancer
              </a>
            </td>
          <td>
            Yuanning Zheng, Christoph Sadée, Michael Ozawa, B. Howitt, Olivier Gevaert
          </td>
          <td>2025-05-23</td>
          <td>Science Advances</td>
          <td>2</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/938950b1958bbfaef501d56fa236f86c6e2c6141" target='_blank'>
              Multi-omic analysis of gallbladder cancer identifies distinct tumor microenvironments associated with disease progression.
              </a>
            </td>
          <td>
            Tao Zhou, Yanhong Wu, Shuai Li, Xinyao Qiu, Erdong Liu, Zhihua Xie, Xuebing Shi, Yani Zhang, Guosheng Ma, Wenbo Guo, Xiang Wang, Kaiting Wang, Xiaomeng Yao, Ji Hu, Siyun Shen, Shuai Yang, Xiaoqing Jiang, Jing Fu, Hongyang Wang, Jin Gu, Lei Chen
          </td>
          <td>2025-06-26</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54dd2ec057c7a25df4bed262e9ae15c8f28acaf6" target='_blank'>
              Mapping the Transcriptional Landscape of Drug Responses in Primary Human Cells Using High-Throughput DRUG-seq
              </a>
            </td>
          <td>
            Lauren Baugh, Sébastien Vigneau, Srijani Sridhar, Sarah Boswell, George Pilitsis, John Bradley, Olga Allen, Laura M. Rand, Amanda Riccardi, Kelsey Roach, Kevin Lema, Ayla Ergun
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer remains one of the leading causes of mortality worldwide, accounting for ≈10 million deaths annually. Critically, it is metastasis and not the primary tumour that causes most of these deaths. Understanding the mechanisms behind cancer dissemination and therapy resistance is thus a pressing challenge. Traditional bulk tissue analyses have failed to capture the full spectrum of intra-tumour heterogeneity and the dynamic interactions within the tumour microenvironment. Studying cancer at the single-cell level allows unravelling the roles of rare subpopulations, cell-cell interactions, and spatial dynamics that govern tumour evolution, metastasis, and immune evasion. This review explores how recent advances in microfluidic technologies are transforming ability to model and study cancer at the single-cell level. Cutting-edge platforms are highlighted, including droplet microfluidics, single cell-derived spheroids, and tumour-chips, that enable physiologically relevant 3D cancer models. By integrating immune components, biosensing, and patient-derived materials, these platforms hold promise for advancing drug screening, immunotherapy assessment, and personalised medicine. It is concluded by identifying key challenges and priorities for future work, which should focus on increasing model complexity, reproducibility, and integration of spatiotemporal multiomics to better dissect tumour heterogeneity and accelerate clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5e66db9196f5b254bfafbf9fe518880855d029d" target='_blank'>
              Advanced Microfluidics for Single Cell-Based Cancer Research.
              </a>
            </td>
          <td>
            Adriana Carneiro, Marta Aranda Palomer, Margarida Esteves, Carolina Rodrigues, José Maria Fernandes, Francisca Oliveira, Alexandra Teixeira, C. Honrado, Lorena Diéguez, S. Abalde‐Cela, Miguel Xavier
          </td>
          <td>2025-07-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="
 e14555


 Background:
 Natural killer (NK) cells are innate lymphocytes that engage in direct cytotoxic responses against malignant cells and promote pro-inflammatory tumor microenvironments. While studies have predominantly focused on conventional NK cell subsets, recent findings uncovered adaptive and tissue-resident (TR) subsets with unique functional properties. The influence of distinct NK cell subsets on tumor immunobiology and clinical outcomes remains unknown.
 Methods:
 Single-cell RNA-sequencing (scRNA-seq) of primary, therapy-naïve tumors from 276 patients across 14 cancer types, as well as 39 patients treated with immune checkpoint blockade (ICB), was accessed through publicly available datasets. We employed systematic computational analyses to phenotype and examine intra-tumoral NK cells. Gene signatures for NK cell subsets were applied to bulk RNA-sequencing from 212,105 tumor samples to delineate their associations with clinical outcomes using the Caris CODEai database.
 Results:
 Six main subsets of NK cells were identified in solid tumors, including conventional CD56
 dim
 and CD56
 bright
 NK cells, alongside adaptive, TR adaptive, TR CD56
 bright
 , and TR CD56
 bright
 /adaptive subsets. Across cancer, TR adaptive NK cells emerged as the most prognostic subset, with high infiltration conferring significantly longer overall survival (HR[95% CI] = 0.68[0.66-0.69], p < 0.001) than conventional CD56
 dim
 and CD56
 bright
 subsets. This was also observed in cohorts of head and neck, non-small cell lung, and esophageal cancers, particularly those treated with ICB. Notably, elevated TR adaptive NK cell infiltration was more prognostic than high tumor mutational burden in these settings. This was most apparent in ICB-treated head and neck cancers (HR = 0.48[0.41-0.57], p < 0.001 vs HR = 0.90[0.76-1.04], p = 0.16). Additionally, TR adaptive NK cells were strongly correlated to the infiltration of CD8
 +
 T cells and B cells in tumors. Underlying these observations, our scRNA-seq analyses demonstrated that TR adaptive NK cells exhibit the highest immunomodulatory activity of all subsets, expressing high levels of pro-inflammatory cytokines (
 CCL3
 ,
 CXCL13
 , and
 IFNG
 ) and low levels of immunosuppressive cytokines (
 TGFB1
 ). Mechanistically, this activity was linked to the upregulation of transcription factors associated with NK cell activation, function, and persistence and downregulation of genes related to stress response and exhaustion. Interestingly, scRNA-seq from paired pre- and post-ICB-treated tumors revealed a unique expansion of cytotoxic
 GZMB
 + TR adaptive NK cells after treatment.
 Conclusions:
 TR adaptive NK cells represent a novel subset of NK cells that may drive anti-tumor immune responses, particularly in the context of ICB treatment. Features of this unique population may be harnessed to design more effective NK cell-based therapies against solid tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdbcf1a77672e37211fd7cd09d5cd12367404c3a" target='_blank'>
              Identification of tissue-resident adaptive NK cells and their association with reduced mortality across diverse solid tumors.
              </a>
            </td>
          <td>
            John R Lozada, A. Ali, Christine Luo, Rosemary N. Plagens, A. Elliott, Ella Boytim, Nicholas A Zorko, W. El-Deiry, Elisabeth I. Heath, Akash Patnaik, A. Vanderwalde, Jeffrey S Miller, E. Antonarakis, Frank Cichocki, Justin H Hwang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background The heterogeneity of colorectal cancer (CRC) and its complex immune microenvironment pose significant challenges for treatment. Understanding the cellular composition and dynamic changes is essential for uncovering mechanisms of tumour progression. Methods To investigate the cellular heterogeneity and immune microenvironment of CRC, identifying critical subpopulations, functional pathways, and prognostic biomarkers, single-cell transcriptomic data from 41 CRC samples across four datasets were integrated. Bioinformatic analyses identified cellular subpopulations, cell communication networks, and prognostic biomarkers. The expression patterns, clinical significance and biological function of TUBB were validated in vitro. Results A distinct epithelial subpopulation with proliferative and invasive features was identified, promoting tumour progression by resisting apoptosis and remodelling the extracellular matrix. ActMono, a terminal state of myeloid cells, was enriched in tumours and linked to disease progression. Cell communication analysis highlighted galectin signalling in immune regulation. A prognostic model (CRS) based on secretory immune cell-related genes identified TUBB as a key molecule influencing the cell cycle and extracellular matrix remodelling, with its expression patterns, clinical significance and biological effects validated in vitro. Conclusion This study reveals critical subpopulations, signalling pathways, and biomarkers in CRC, providing insights into tumour progression and potential therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2e4b3d5d75559211564431db384708f82801552" target='_blank'>
              Single-cell sequencing reveals dysregulated cell type perturbations and critical mediator communication remodelling in colorectal cancer
              </a>
            </td>
          <td>
            Chengyuan Xu, Siqi Zhang, Zhouyu Zhang, Luoqiu Zhang, Bin Sun, Zicheng Yu, Hailong Liu
          </td>
          <td>2025-06-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction The pronounced heterogeneity of colorectal cancer (CRC) significantly impacts patient prognosis and therapeutic response, making elucidation of its molecular mechanisms critical for developing precision treatment strategies. This study aimed to systematically characterize tumor cell heterogeneity and explore its clinical implications. Methods Five single-cell RNA sequencing cohorts were integrated (comprising 70 CRC samples and 164,173 cells) to systematically analyze tumor cell heterogeneity. Unsupervised clustering analysis based on VEGFR+ tumor cell signature genes was used to stratify CRC patients. Key molecular mechanisms were validated through in vitro cellular experiments, in vivo animal models, molecular docking, and dynamics simulations. Results The analysis successfully identified five distinct tumor cell subtypes, with the VEGFR+ subtype exhibiting marked epithelial-mesenchymal transition (EMT) activation signatures and strong association with metastasis and poor clinical outcomes. Based on VEGFR+ signature genes, CRC patients were stratified into three subgroups: C1 (metabolically active), C2 (proliferative), and C3 (invasive), with the C3 subtype demonstrating high metastatic potential, stem-like properties, and an immunosuppressive microenvironment, along with a five-year survival rate below 50%. Mechanistic investigations identified HOXC6 as a key driver of the C3 subtype, with HOXC6 knockout significantly suppressing CRC cell proliferation, migration, and invasion. Furthermore, molecular docking revealed that the targeted agent abemaciclib effectively binds HOXC6, with both cellular and animal experiments confirming its ability to inhibit CRC cell functions and significantly reduce tumor burden in nude mice. Discussion This study establishes the first single-cell-resolution molecular classification system for CRC, delineates the mechanistic link between EMT subtypes and metastatic progression, and identifies HOXC6 as a novel therapeutic vulnerability. These findings provide a translational foundation for precision oncology and offer new rationale for precision diagnosis and treatment of colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4607a6764846d173d90b47364ed771e5e80f538" target='_blank'>
              Cross-cohort multi-omics analysis identifies novel clusters driven by EMT signatures in colorectal cancer
              </a>
            </td>
          <td>
            Wu Ning, Wenqing Jia, Jingyuan Ning, Lei Zhou, Zongze Li, Lin Zhang, Xin Song
          </td>
          <td>2025-06-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="During cancer development, mutations promote gene expression changes that cause transformation. Leukemia is frequently associated with aberrant HOXA expression driven by translocations in nucleoporin genes or KMT2A, and mutations in NPM1. How disparate mutations converge on this regulatory pathway is not understood. Here we demonstrate that mutant NPM1 (NPM1c) forms nuclear condensates in multiple human cell lines, mouse models, and primary patient samples. We show NPM1c phase separation is necessary and sufficient to coordinate the recruitment of NUP98 and KMT2A to condensates. Through extensive mutagenesis and pharmacological destabilization of phase separation, we find that NPM1c condensates are necessary for regulating gene expression, promoting in vivo expansion, and maintaining the undifferentiated leukemic state. Finally, we show that nucleoporin and KMT2A fusion proteins form condensates that are biophysically indistinguishable from NPM1c condensates. Together, these data define a new condensate underlying leukemias that we term coordinating bodies (C-bodies), and propose C-bodies as a therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0742ebc47874d0a681b5ec4cb75157b2b57840b5" target='_blank'>
              Nuclear Phase Separation Drives NPM1-mutant Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Gandhar K. Datar, Elmira Khabusheva, Archish Anand, Joshua Beale, Marwa Sadek, Chun-Wei Chen, Evdokiia Potolitsyna, Nayara Alcantara-Contessoto, Guangyuan Liu, Josephine De La Fuente, Christina Dollinger, A. Guzman, Alejandra G Martell, Katharina Wohlan, A. Maiti, N. Short, S. S. Yi, V. Andresen, Bjørn T. Gjertsen, B. Falini, R. E. Rau, L. Brunetti, Nidhi Sahni, M. Goodell, Joshua A. Riback
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="
 8091


 Background:
 Small cell lung cancer (SCLC) is a highly aggressive neuroendocrine malignancy with a median survival of 9–11 months. Despite the addition of immune checkpoint blockade (ICB) to frontline chemotherapy, improvements remain limited. SCLC evades anti-tumor immunity by suppressing major histocompatibility complex class I (MHC-I) and antigen presentation (AP). The variability of MHC repression across molecular subtypes of SCLC and high-grade neuroendocrine tumors is not fully understood. This study analyzed clinical SCLC and neuroendocrine tumor samples to identify molecular subtypes and regulators of MHC repression, enhancing the understanding of immune evasion.
 Methods:
 Molecular profiling was conducted on 944 SCLC and 5056 non-small cell lung cancer (NSCLC) tumors using next-generation DNA (592-gene panel/whole exome), RNA (whole transcriptome) sequencing, and immunohistochemistry. Transcriptomic profiling of 40 SCLC cell lines and genetically engineered mouse models (GEMMs) was also performed.
 Results:
 Key regulators and gene networks driving MHC-I suppression in SCLC and other high-grade neuroendocrine tumors were identified. Canonical (MHC-I and II) and non-canonical AP expression scores characterized the spectrum of MHC-I repression across subtypes. Low MHC-I scores correlated with reduced immune signatures (e.g., T cell-inflamed, NK cell, and STING pathway signatures; Spearman = 0.87, 0.58, 0.62, respectively) and predicted poor OS and PFS to immunotherapy. Enhancer network and gene expression analyses identified actionable regulators of MHC-I repression, including interferon signaling. Knockout studies of top MHC-I regulators and single cell analyses revealed non-homologous end-joining (NHEJ) genes (like DNAPKCs) inversely correlated with HLA-A/B/C expression. DNAPKCs inhibition upregulated MHC-I, TAP1, and TAP2, enhanced antigen-specific T-cell-mediated cytolysis in SCLC. This inhibition activated the STING pathway, increased CD8+ T-cell infiltration, and enhanced MHC Class-I. Combining DNAPKCs inhibitor NU1774 with anti-PD-L1 reactivated MHC-I and led to significant tumor regression in aggressive murine SCLC models, supporting new combination therapies.
 Conclusions:
 We identified molecular subtypes and actionable pathways to restore MHC-I expression in SCLC, advancing understanding of immune evasion mechanisms. DNAPKCs is established as a key therapeutic target, enhancing PD-L1 blockade by reactivating antigen presentation. The developed MHC-I expression scores and associated biomarkers offer predictive tools for identifying patients likely to benefit from immunotherapy. These findings support biomarker-driven, personalized approaches to improve treatment for SCLC, addressing a critical unmet need in this devastating cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cef19343d2cbcd91f40c5103e3abe3dd630690e8" target='_blank'>
              Unveiling drivers of MHC repression and therapeutic strategies to counter immune evasion in small cell lung cancer.
              </a>
            </td>
          <td>
            Triparna Sen, Subhamoy Chakraborty, A. Elliott, V. Jethalia, A. Vanderwalde, B. Halmos, D. Demircioglu, D. Hasson
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Esophageal squamous cell carcinoma (ESCC) is a highly heterogeneous malignancy. Traditional clinical staging systems have limited value in prognostic evaluation and treatment guidance, often failing to capture the profound impact of intratumoral diversity on patient outcomes. Single-cell RNA sequencing (scRNA-seq) provides new perspectives on the cellular makeup and conditions in tumor tissues, with promising implications for functional classification and personalized therapeutic strategies in esophageal cancer. Methods In this study, we integrated scRNA-seq data with bulk RNA-seq profiles from esophageal cancer tissues to construct a comprehensive cellular atlas, focusing on the transcriptional characteristics of epithelial cells. Malignant epithelial cells were identified based on copy number variation (CNV) features using inferCNV analysis. Their developmental states and regulatory mechanisms were further characterized via transcription factor activity inference (SCENIC) and pseudotime trajectory analysis (Monocle). Based on marker genes of malignant epithelial subpopulations, we developed a multi-gene risk scoring model using data from the TCGA and GEO (GSE53624) cohorts. The model’s predictive value for immune landscape, mutational features, and drug sensitivity was also evaluated. Additionally, qRT-PCR was conducted to quantify the expression levels of model genes in ESCC tissue samples, further evaluating their biological relevance. Functional roles of the key gene HMGB3 were validated in vitro through CCK-8 proliferation assays, Transwell invasion assays, and colony formation assays following gene knockdown in ESCC cell lines. Results At the single-cell level, we identified ten major cell types and six distinct malignant epithelial subclusters, which exhibited pronounced heterogeneity in cell cycle states, transcriptional regulatory networks, and differentiation trajectories. High CNV scores and the enrichment of specific transcription factors (e.g., FOXC1, E2F1, RUNX1) suggested a proliferative and immune-evasive phenotype. A six-gene prognostic model (HMGB3, CHORDC1, CTSD, BTG2, MT1E, PHYHD1) showed strong predictive accuracy for overall survival in the TCGA and GSE53624 cohorts. Furthermore, the risk score showed a significant correlation with an immunosuppressive tumor microenvironment, increased tumor purity, and the activation of certain immune-related pathways. Analysis of drug sensitivity suggests that patients classified as low-risk could respond better to various targeted therapies and chemotherapeutic agents, underscoring their potential clinical relevance. Functional assays revealed that HMGB3 knockdown markedly suppressed ESCC cell proliferation, invasion, and colony formation, supporting its oncogenic role in esophageal cancer progression. Conclusion This study systematically characterized epithelial cell heterogeneity in esophageal cancer at single-cell resolution and established a risk model based on malignant epithelial markers that effectively predicts prognosis, immune status, and potential drug response. Combined with experimental validation, our findings highlight the pivotal role of HMGB3 in ESCC progression and provide a novel theoretical and practical framework for functional tumor classification and individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70d92eb766fe9a932637e37c8c15e4650793e3d8" target='_blank'>
              Malignant epithelial cell marker–driven risk signature enables precise stratification in esophageal cancer
              </a>
            </td>
          <td>
            Hao Zhang, Shizhao Cheng, Yijun Xu
          </td>
          <td>2025-05-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug resistance poses a significant challenge in cancer therapy, contributing to rapid recurrence, disease progression, and high patient mortality. Despite its critical impact, few reliable predictors for cancer drug response or failure have been established for clinical application. Tumor heterogeneity and the tumor microenvironment (TME) are pivotal factors influencing cancer drug efficacy and resistance. Tumor heterogeneity leads to variable therapeutic responses among patients, while dynamic interactions between cancer cells and the TME enhance tumor survival and proliferation, underscoring the urgent need to identify cellular hallmarks for predicting drug response and resistance. Single-cell and spatial omics technologies provide high-resolution insights into gene expression at the individual cell level, capturing intercellular heterogeneity and revealing the underlying pathologies, mechanisms, and cellular interactions. This review delves into the principles, methodologies, and workflows of single-cell and spatial omics in cancer drug research, highlighting key hallmarks involving tumor heterogeneity, TME reprogramming, cell–cell interactions, metabolic modulation, and signaling pathway regulation in drug treatment at single-cell and spatial levels. Furthermore, we synthesize predictive cellular biomarkers for cancer drug response and resistance across 25 cancer types, paving the way for advancements in cancer precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af5a8672965af056452be6aaf8b56f1ff303b62" target='_blank'>
              Application of single-cell and spatial omics in deciphering cellular hallmarks of cancer drug response and resistance
              </a>
            </td>
          <td>
            Xiaoxia Cheng, Ting Peng, Tian Chu, Yiqun Yang, Jia Liu, Qinglei Gao, C. Cao, Juncheng Wei
          </td>
          <td>2025-07-02</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background
 The tumor microenvironment and biomarkers play a pivotal role in breast cancer research, yet there remains a pressing need for effective biomarkers. This study focuses on identifying a novel IGKC+ T Cell subpopulation and its related biomarkers to pave the way for innovative targeted therapies and improved clinical outcomes.


Methods
We first performed single-cell RNA sequencing (scRNA-seq) analysis to characterize immune cell heterogeneity within the tumor microenvironment, leading to the identification of series cell subpopulation. Then, by performing univariate analysis to correlate cell proportions with patient prognosis, we identified a novel IGKC+ T cell subpopulation. Next, we applied bulk RNA-seq deconvolution algorithms to estimate the abundance of this subpopulation across breast cancer cohorts. Differential expression analysis and weighted gene co-expression network analysis (WGCNA) were employed to identify genes associated with the IGKC+ T cell population. To pinpoint key regulatory genes, we applied machine learning algorithms. Based on the hub genes identified, we constructed a prognostic risk model and developed a nomogram to aid clinical decision-making. Immune infiltration patterns were further assessed in high- vs low-risk groups defined by the model. Finally, functional validation was performed through overexpression of BCL2L14 in vitro, and downstream signaling pathways were examined.


Results
We identified the novel IGKC+ T cell subpopulation and core genes. Machine learning pinpointed BCL2L14, IGHD, MAPT-AS1, NT5DC4, and TNIP3 as key regulators of breast cancer progression in this subpopulation. The model stratified patients into high- and low-risk groups, with high-risk patients showing worse prognosis and weaker immune infiltration. Overexpression of BCL2L14 was experimentally demonstrated to accelerate breast cancer progression, linked to enhanced phosphorylation of the NF-κB pathway.


Conclusion
Our results underscore BCL2L14 as a potential driver within the novel T-cell subpopulation and a critical biomarker for breast cancer diagnosis. These findings provide a basis for developing advanced diagnostic tools and targeted therapies, which may ultimately enhance patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/899ab08619ae94e923463082f4db3808d873da0a" target='_blank'>
              Integration of Bulk and Single-Cell Transcriptomics Reveals BCL2L14 as a Novel IGKC+ T Cell-Associated Therapeutic Target in Breast Cancer.
              </a>
            </td>
          <td>
            Jiaming He, Aiman Akhtar, Jing Li, Qiang Wei, Yang Yuan, Jianhua Ran, Yongping Ma, Dilong Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of inflammation research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Renal cell carcinoma is characterized by a poor prognosis. Recently, renal cell carcinoma has been recognized as a metabolic disease associated with fatty acid metabolic reprogramming, although in-depth studies on this topic are still lacking. We found that fatty acid metabolism reprogramming in renal cell carcinoma is primarily characterized by high expression of FABP1. FABP1 + tumors significantly impact survival and display distinct differentiation trajectories compared to other tumor subclusters. They show elevated expression of angiogenesis and cell migration signals, with PLG-PLAT-mediated interactions with endothelial cells notably enhanced. Spatial transcriptomics show a prominent co-localization of FABP1 + tumors with endothelial cells, and their spatial distribution closely aligns with that of PLAT + endothelial cells. FABP1 + tumors exhibit a unique pattern in spatial transcriptomics, enriched in Extracellular Matrix and angiogenesis-related pathways. Through receptor-ligand interaction analysis, a novel PLG-PLAT functional axis was found between tumor epithelial cells and endothelial cells. Based on results of experiments, we infer that FABP1 + tumors can promote plasmin-related tumor angiogenesis by triggering the PLG-PLAT signaling axis. Finally, utilizing preclinical models, we suggest that targeting the FABP1-PLG-PLAT axis may serve as promising strategy enhancing the sensitivity of Tyrosine Kinase Inhibitor therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02377-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27626d5d72e69d9402b111fc64dabcbee4819f07" target='_blank'>
              Single-cell multi-omics reveals that FABP1 + renal cell carcinoma drive tumor angiogenesis through the PLG-PLAT axis under fatty acid reprogramming
              </a>
            </td>
          <td>
            Yiqiu Wang, Yingchun Liang, Min Li, Jiayi Lu, Sian Zhou, Yaoyu Yu, Changwei Yang, Xinhuang Hou
          </td>
          <td>2025-06-16</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most aggressive and lethal brain tumor in adults and poses significant challenges to patient survival. This review provides a comprehensive exploration of the molecular and genetic landscape of GBM, focusing on key oncogenic drivers, such as epidermal growth factor receptor (EGFR), platelet-derived growth factor receptor (PDGFR), and the PI3K/AKT/mTOR pathway, which are critical for tumorigenesis and progression. We delve into the role of epigenetic alterations, including DNA methylation and histone modifications, in driving therapy resistance and tumor evolution. The tumor microenvironment is known for its pivotal role in immune evasion, with tumor-associated macrophages, myeloid-derived suppressor cells, and regulatory T cells creating an immunosuppressive niche that sustains GBM growth. Emerging therapies, such as immunotherapies, oncolytic viral therapies, extracellular vesicle-based approaches, and non-coding RNA interventions, are highlighted as promising avenues to disrupt GBM pathogenesis. Advances in precision medicine and innovative technologies, including electric field therapy and locoregional treatments, are discussed for their potential to overcome the blood‒brain barrier and treatment resistance. Additionally, this review underscores the importance of metabolic reprogramming, particularly hypoxia-driven adaptations and altered lipid metabolism, in fueling GBM progression and influencing the therapeutic response. The role of glioma stem cells in tumor recurrence and resistance is also emphasized, highlighting the need for targeted therapeutic approaches. By integrating molecular targeting, immune energetics, and technological advancements, this review outlines a multidisciplinary framework for improving GBM treatment outcomes. Ultimately, the convergence of genetic, metabolic, and immune-based strategies offers transformative potential in GBM management, paving the way for increased patient survival and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1445c096bc679931586564b48bbb15167d4847f8" target='_blank'>
              Glioblastoma at the crossroads: current understanding and future therapeutic horizons
              </a>
            </td>
          <td>
            Shilpi Singh, Devanjan Dey, Debashis Barik, Iteeshree Mohapatra, Stefan Kim, Mayur Sharma, Sujata Prasad, Peize Wang, Amar Singh, Gatikrushna Singh
          </td>
          <td>2025-07-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Cholangiocarcinoma (CCA) is a highly heterogeneous malignancy, primarily comprising intrahepatic (iCCA) and extrahepatic (eCCA) subtypes. Reconciling the variability between iCCAs and eCCAs in clinical trials remains a challenge, largely due to the inadequate understanding of their shared and subtype-specific cellular heterogeneity. We aim to address this issue using single-cell and spatially resolved transcriptomic approaches. Methods We performed comprehensive single-cell RNA sequencing (scRNA-seq) by profiling 109,071 single cells from 28 samples, including chronic biliary inflammatory conditions (n = 7) and CCAs from different anatomical sites (n = 21). Findings were validated using external multi-omics datasets, tissue microarray cohort, spatial RNA in situ sequencing, CCA patient-derived organoids (PDOs), and mouse models. Results iCCAs and eCCAs exhibited distinct tumor ecosystems, with notable differences in cellular composition, diversity, and abundance across various cell types. Non-malignant epithelial cells displayed divergent precancer hallmarks from different biliary sites, with inflammatory extrahepatic bile ducts exhibiting early hijacking of the gastrointestinal metaplastic process. We identified seven meta-programs within cancer cells, mapped into four major subtypes. This subtyping was validated using external CCA cohorts and PDO models, distinguishing patients based on clinical outcomes and drug vulnerabilities. Specifically, iCCAs were associated with a senescent program, while eCCAs were enriched in an IFN-responsive program linked to adverse clinical outcomes and increased drug resistance. We identified a basal-like LY6D+ cancer cell subpopulation specific to eCCAs, which displayed significant stemness, drug resistance, and IFN-responsive features. This subpopulation was closely associated with an interferon-stimulated gene 15 (ISG15)-enriched mesenchymal and immune microenvironment. Functional assays demonstrated that ISG15 stimulation significantly boosted stemness, basal-like features, and drug resistance in eCCA cells, highlighting its pivotal role in sustaining the LY6D+ progenitor niches. Conclusion We present a comprehensive single-cell landscape of CCAs, uncovering the molecular heterogeneity between iCCA and eCCA subtypes. Transcriptomic subtyping of CCA cancer cells offers implications for clinical stratification and functional precision oncology. We identify basal-like epithelial progenitors and characterize their associated ISG15-enriched microenvironment in eCCAs. These findings hold significant promise for the development of novel prognostic biomarkers, therapeutic targets, and treatment strategies for CCAs. Supplementary Information The online version contains supplementary material available at 10.1186/s13045-025-01716-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fd09acd4eb4bccd6f0a7c5305db1cb0247e6ead" target='_blank'>
              Deciphering cholangiocarcinoma heterogeneity and specific progenitor cell niche of extrahepatic cholangiocarcinoma at single-cell resolution
              </a>
            </td>
          <td>
            Chunliang Liu, Xiang Wang, Erdong Liu, Yali Zong, Wenlong Yu, Youhai Jiang, Jianan Chen, Mingye Gu, Zhengyuan Meng, Jingfeng Li, Yang Liu, Yongjie Zhang, Jing Tang, Hongyang Wang, Jing Fu
          </td>
          <td>2025-06-23</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Gliomas, particularly glioblastomas, are highly malignant brain tumors with high recurrence rates and poor prognosis. Despite advances in treatment, recurrence remains a major challenge. Epithelial-mesenchymal transition (EMT) plays a key role in tumor invasion and recurrence. This study explores the transcriptional and regulatory mechanisms driving glioma recurrence, focusing on mesenchymal-like (MES-like) subpopulations. Single-nucleus RNA sequencing was performed on 52 IDH wild-type GBM specimens, including 26 primary and 26 recurrent tumors. Spatial transcriptomics data were also incorporated. Tumor subpopulations were identified through gene regulatory network analysis, copy number variation detection, and nonnegative matrix factorization. Functional validation was conducted using gene knockdown experiments, followed by xenograft studies. We discovered novel MES-like subpopulations in recurrent GBM enriched with EMT-related genes like EGR1 and SERPINE1. These subpopulations exhibited increased transcriptional activity and were associated with poor prognosis and invasiveness. Knockdown of SERPINE1 significantly reduced cell proliferation and migration. Spatial transcriptomics showed MES-like cells concentrated at the tumor margins, highlighting their role in invasion and recurrence. MES-like subpopulations, driven by EGR1 and SERPINE1, are critical in GBM. Targeting these regulators could offer new therapeutic strategies to reduce glioma recurrence and improve outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02036-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f7432f6a50e6a5d830f429844d915dd10fcd2" target='_blank'>
              Single-cell transcriptome sequencing reveals new epithelial-stromal associated mesenchymal-like subsets in recurrent gliomas
              </a>
            </td>
          <td>
            Jinwei Li, Shengrong Long, Yang Zhang, Shuangqi Yu, Hongyu Xu, Rui Liang, Quan Liu, Jinnan Zhang, Xiang Li, Yixin Fu, Tao Xin, Yinyan Wang
          </td>
          <td>2025-06-07</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="This study aimed to identify the key cell types and their interactions in gynecological oncology of breast cancer, cervical cancer, and ovarian cancer. Single-cell RNA sequencing was performed on tumor samples of gynecological oncology from the GEO database. Cell types were identified using SingleR and cell composition was analyzed to understand the tumor microenvironment (TME). CellChat was used to analyze cell interactions, and pseudotemporal analysis was conducted on cancer-associated fibroblasts (CAFs) and tumor-associated macrophages (TAMs) to understand their differentiation status. Four CAF subtypes were identified: iCAF, myCAF, proCAF, and matCAF. The iCAF subpopulation secreted COL1A1 and promoted tumor cell migration, while myCAF was involved in angiogenesis. The matCAF subpopulation was present throughout tumor development. TAMs were found to promote angiogenesis through the VEGFA_VEGFR2 signaling pathway. CAFs and TAMs play pivotal roles in tumor progression through their interactions and signaling pathways. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-03017-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3feb7a90b872014c5f14c6c2321cfb01bca8e962" target='_blank'>
              Elucidating the tumor microenvironment interactions in breast, cervical, and ovarian cancer through single-cell RNA sequencing
              </a>
            </td>
          <td>
            Xiaoyue Zhu, Liang Zhang, Xiaomin Yu, Pengxian Yan, Xiaoyu Zhang, Yunlong Zhao, Dongze Wang, Xiu-An Yang
          </td>
          <td>2025-05-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Image mass cytometry (IMC) enables high-dimensional spatial profiling by combining mass cytometry's analytical power with spatial distributions of cell phenotypes. Recent studies leverage large language models (LLMs) to extract cell states by translating gene or protein expression into biological context. However, existing single-cell LLMs face two major challenges: (1) Integration of spatial information: they struggle to generalize spatial coordinates and effectively encode spatial context as text, and (2) Treating each cell independently: they overlook cell-cell interactions, limiting their ability to capture biological relationships. To address these limitations, we propose Spatial2Sentence, a novel framework that integrates single-cell expression and spatial information into natural language using a multi-sentence approach. Spatial2Sentence constructs expression similarity and distance matrices, pairing spatially adjacent and expressionally similar cells as positive pairs while using distant and dissimilar cells as negatives. These multi-sentence representations enable LLMs to learn cellular interactions in both expression and spatial contexts. Equipped with multi-task learning, Spatial2Sentence outperforms existing single-cell LLMs on preprocessed IMC datasets, improving cell-type classification by 5.98% and clinical status prediction by 4.18% on the diabetes dataset while enhancing interpretability. The source code can be found here: https://github.com/UNITES-Lab/Spatial2Sentence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb458acfc483ddf82cfc936e42c471e927676cb7" target='_blank'>
              Spatial Coordinates as a Cell Language: A Multi-Sentence Framework for Imaging Mass Cytometry Analysis
              </a>
            </td>
          <td>
            Chi-Jane Chen, Yuhang Chen, Sukwon Yun, Natalie Stanley, Tianlong Chen
          </td>
          <td>2025-06-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The role of the microbiome in cancer metastasis has emerged as a critical area of research, with growing evidence suggesting that microbial composition and interactions within the tumour microenvironment may significantly influence metastatic progression. This review explores the role of the microbiome in cancer metastasis, as well as potential key bacteria and their mechanisms through which they could impact tumour dissemination, seeding and growth. Biological models used to study metastasis are discussed to provide context for the further investigation of these interactions. In order to answer unresolved questions regarding the microbiome’s involvement in metastatic dissemination, recent advancements in spatial biology techniques are examined, including spatial genomics, transcriptomics, proteomics and metabolomics, which enable the spatial mapping of microbial interactions within the tumour microenvironment. Additionally, multimodal-omics imaging approaches are highlighted for their potential to integrate multiple molecular layers, offering comprehensive insights into the microbiome’s role in cancer metastasis. The review also addresses the challenges and limitations of these techniques, underscoring the complexity of studying microbiome-tumour interactions and offering directions for future research to better explore and target the microbiological landscape in metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bd737e751891e52e56b56db2c8e1ef16eeffa30" target='_blank'>
              Microbiome in cancer metastasis: biological insights and emerging spatial omics methods
              </a>
            </td>
          <td>
            Marianne Meyers, Charlotte B. A. Stoffels, G. Frache, Elisabeth Letellier, Maureen Feucherolles
          </td>
          <td>2025-06-04</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb25978847c5553a988089ffa28339e8dff51f1" target='_blank'>
              Single-Cell profiling reveals targetable malignant T-cell subtypes and immune evasion pathways in Sezary Syndrome
              </a>
            </td>
          <td>
            M.S.F Beth A. Childs, Ph.D Eslam A. Elghonaimy, Praveen, Ramakrishnan Geethakumari, Kiran A. Kumar, J. F. Merola, M. D. Heather W. Goff, M. Todd A Aguilera, Ph.D Todd
          </td>
          <td>2025-05-21</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Treatment-induced neuroendocrine prostate cancer (t-NEPC) is a lethal, castration-resistant subtype of prostate cancer. While t-NEPC typically arises from adenocarcinoma through neuroendocrine transdifferentiation after androgen pathway inhibition, the temporal dynamics and molecular drivers of this process remain poorly understood. Here, utilizing the first-in-field patient-derived xenograft (PDX) model of adenocarcinoma-to-NEPC transdifferentiation (LTL331/331R), we performed longitudinal single-cell transcriptomic sequencing (scRNA-seq) across seven timepoints spanning pre-castration to relapsed NEPC. Our analysis demonstrated 15 distinct cell clusters, including twelve adenocarcinoma clusters and two NEPC clusters (ASCL1high/FOXA2low and ASCL1low/FOXA2high clusters). Notably, we revealed a newly-discovered, early intermediate transitional cell state during t-NEPC development distinguished by epithelial-mesenchymal transition (EMT), stem cell-related, metabolically active, and HDAC-associated regulatory signatures. Analysis of this intermediate transitional cluster led to the identification of RUNX1T1 as a pivotal transcriptional regulator of NEPC transdifferentiation. Functionally, RUNX1T1 overexpression promoted AR pathway inhibition (ARPI) -induced NE transdifferentiation and increased resistance to ARPI treatment in prostate adenocarcinoma. RUNX1T1 knockdown reverses the NE transdifferentiation, inhibits NEPC cell proliferation and induces apoptosis, and cell cycle arrest. In summary, this study identifies a critical intermediate transitional cell state during t-NEPC development and reveals the heterogeneity of terminal NEPC, offering new insights into NEPC biology and emphasizing the importance of early intervention. Moreover, the discovery of RUNX1T1 as a key early driver active in both the initial and terminal phases of NEPC progression presents promising opportunities for therapeutic intervention. Highlights Longitudinal single-cell RNA sequencing of treatment-induced adeno-to-NE transdifferentiation model revealed an intermediate transitional cell state in NEPC development and progression. The sequential emergence of ASCL1high/FOXA2low and ASCL1low/FOXA2high NEPC subclusters indicates a temporal evolution of NEPC and highlights their role in contributing to NEPC heterogeneity. RUNX1T1, a transcriptional regulator showing increased expression in both intermediate cell state and NEPCs, is identified as an early driver of NEPC development. RUNX1T1 plays important functional roles in promoting adeno-to-NE transdifferentiation in early phase of NEPC development and maintaining NE phenotype and aggressiveness in terminal NEPC. Targeting RUNX1T1 or its complex could offer new therapeutic strategies for NEPC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3dfda20cdaed4a7ffd5a497959ce67fe25256c9" target='_blank'>
              Longitudinal single-cell analysis reveals RUNX1T1 as an early driver in treatment-induced neuroendocrine transdifferentiation
              </a>
            </td>
          <td>
            Yuchao Ni, Dong Lin, Mingchen Shi, Yen-Yi Lin, Hui Xue, Xin Dong, Liang Liu, F. Sar, Rebecca Wu, Tunc Morova, A. Haegert, Robert Bell, Xinyao Pang, Adam Classen, Yu Wang, Junru Chen, Stanislav Volik, S. L. Bihan, Nathan A. Lack, Christopher Ong, Gang Wang, H. Zeng, Colin C. Collins, Yuzhuo Wang
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Patients with high-grade serous ovarian cancer (HGSOC) typically present with widespread metastasis, obscuring a temporal understanding of tumor-immune dynamics. To address this, we perform multi-site global proteomics alongside matched immunohistochemistry (IHC) for CD4⁺ and CD8⁺ tumor-infiltrating lymphocytes (TILs) in patient samples. We order the protein expression profiles using an unbiased pseudotime analysis, recapitulating clinical observations of metastatic progression, and providing a framework to explore tumor-immune dynamics from localized to metastatic disease. Metastatic progression correlates with immune cell infiltration, the recruitment of regulatory T cells (Tregs) to counterbalance γδ T cell abundance, and an increased abundance of exhausted CD8⁺ T cells. The accumulation of Tregs at metastatic sites correlates with SNX8 expression, a critical regulator of the STING pathway. In early-stage tumors, keratin-expressing cancer cells recruit Tregs via MHC class II, fostering an inflammatory phenotype with limited IFNγ production and non-clonally expanded T cells. Together, our findings reveal novel mechanisms of immune escape associated with both localized disease and metastatic progression in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5284b73444d2932c5ad492644667337328f9e74" target='_blank'>
              A temporal model of tumor-immune dynamics during the metastatic progression of high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Donagh Egan, Kate Glennon, Ann Treacy, Aurelie Fabre, Janet Mccormack, Salisha Hill, Ryan Morrison, Vadim Zhernovkov, Daniel C Liebler, Walter Kolch, Donal J Brennan
          </td>
          <td>2025-06-16</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 e15561


 Background:
 Colorectal cancer (CRC), the third most prevalent cancer globally, remains a major cause of cancer-related mortality. Genomic and epigenomic plasticity during CRC progression, particularly metastasis, poses significant challenges to effective treatment. Patient-derived organoids (PDOs) reflect the heterogeneous genetic and morphological composition of cancer cells, enabling ex vivo disease modeling. This study investigates the epigenomic makeup of human primary and metastatic CRC to advance precision epigenetic therapies.
 Methods:
 We analyzed 45 organoid models, including 40 CRC organoids (from primary tumors and liver/lung metastases) and 5 generated from healthy colonic tissue. Using integrative epigenomic profiling, including chromatin accessibility, chromatin state mapping (via six histone modifications), and 3D genome conformation, we constructed a comprehensive CRC epigenomic landscape.
 Results:
 Our analysis of CRC patient-derived organoids revealed a core set of accessible genomic regions shared across all organoids, irrespective of tumor location or metastatic status. This conserved chromatin signature highlights common regulatory pathways in CRC during disease progression. Unsupervised clustering based on epigenomic features revealed four distinct CRC subtypes, encompassing both primary and metastatic models. Notably, organoids derived from liver and lung metastases did not cluster by metastatic site, underscoring epigenomic convergence. Subtype D comprised 57.1% (8/14) of primary tumors, while subtypes B and C contained 47.3% (9/19) of liver metastases and 42.8% (3/7) of lung metastases, respectively. KRAS mutation status further stratified subtypes, with wild-type KRAS observed exclusively in subtypes A and B, while KRAS-mutant variants were enriched in subtypes C (6/7) and D (4/10). Histone modification patterns, defined by six marks (H3K4me1, H3K4me3, H3K27me3, H3K27ac, H3K36me3, and H3K9me3) revealed subtype-specific regulatory chromatin landscapes. Three-dimensional chromatin interactions were mapped, and genome-wide interaction contacts are used to identify various chromatin structures that may influence metastatic capacity.
 Conclusions:
 This study highlights distinct and shared epigenomic signatures in CRC PDOs, offering novel insights into tumor progression and metastasis. Integrative chromatin analysis provides a framework for identifying epigenetic biomarkers and epigenome targeting strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66760aad28585c2f41842514d0645e6e13f51968" target='_blank'>
              Deciphering colorectal cancer progression through integrative epigenomic analysis for precision therapeutics.
              </a>
            </td>
          <td>
            Ghazaleh Tavallaee, Amin Nooranikhojasteh, Elias Orouji
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 7070


 Background:
 Follicular lymphoma (FL) exhibits considerable variability in biological features and clinical trajectories, compounded by a complex tumor microenvironment (TME) populated with nonmalignant immune cells. In this study, we comprehensively characterized FL by integrating multi-omics data that considered both tumor cells and microenvironmental components. This analysis revealed four subtypes of FL, each exhibiting distinct biological characteristics and differing clinical behaviors.
 Methods:
 Our study included 1,203 samples, 53 bulk transcriptomic samples from our center, 1,150 samples from public databases, and five single-cell RNA sequencing (scRNA-seq) cohorts. The Ecotyper algorithm was employed to identify immune cell states. Unsupervised clustering analysis using non-negative matrix factorization (NMF) was performed to identify distinct FL subtypes based on cellular infiltration.
 Results:
 A total of 30 unique cell states were identified from nine annotated cell populations (B cells, plasma cells, CD4 T cells, CD8 T cells, Tregs, NK cells, Tfh, monocytes/macrophages, and dendritic cells). Validation was performed across three independent scRNA-seq and bulk transcriptomic cohorts to ensure robustness. Four distinct B cell states were identified, each exhibiting unique gene expression profiles and varying prognostic implications. B cell S2 and S3 were associated with adverse outcomes. Notably, PRDM15 in B cell S2 had the most significant prognostic impact, remaining an independent adverse prognostic factor even after adjustment for m7-FLIPI. Mutational profile analysis demonstrated distinct mutation patterns across B cell states: mutations in CCND3, SPEN, and ARID1A were enriched in B cell S1; CREBBP and STAT6 mutations were prominent in B cell S2; and IRF8 and BCL2 mutations were observed in B cell S4, while no significant mutation enrichment was detected in B cell S3. Similarly, immune cells of the same type yet in different states exhibited functional and clinical heterogeneity. Four subtypes of follicular lymphoma characterized by distinct tumor cell states and varying immune infiltration were identified through unsupervised clustering analysis. FLE1 exhibits characteristics of a "cold tumor" with a high abundance of B cell S2. In contrast, FLE2 and FLE3 show moderate immune infiltration, while FLE4 is marked by abundant immune infiltration and elevated expression of immunosuppressive checkpoint molecules, resembling an "inflammatory" tumor phenotype. Prognostic analysis indicates that FLE1 has the most unfavorable prognosis, followed by FLE4.
 Conclusions:
 Our study stratifies FL patients based on the heterogeneity of tumor cells and the immune microenvironment, proposing an immune-based classification for FL. Targeting CREBBP and PRDM15 may offer promising new strategies for the clinical management of patients exhibiting a "cold tumor" phenotype.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0fb822abee7b71f9392d723669dd1a087479f12" target='_blank'>
              Clinical and biological subtypes of follicular lymphoma revealed by tumor and immune cell states.
              </a>
            </td>
          <td>
            Hengqi Liu, Yingfang Feng, Ning Zhang, Cong Sun, Ning Zhang, Xianhuo Wang, Huilai Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (ICC) is characterized by high malignancy, and its global incidence is predicted to continue to increase over the past decades. However, the mechanisms underlying ICC pathogenesis and progression remain unclear.


APPROACH AND RESULTS
The training cohort consisted of single-cell sequencing of 12 treatment-naïve ICC samples and spatial transcriptomics of four ICC samples. The validation cohort comprised of RNA-seq data from 87 ICC tumor samples. Finally, we validated our findings via multiplex immunofluorescence, organoids, and mice models both in vivo and in vitro. We found significant heterogeneity within the tumor microenvironment (TME) of ICC patients. ICC cells were classified into five molecular subtypes, and we revealed that aspartate beta-hydroxylase (ASPH) was a marker gene for invasion subtypes. We then selected cepharanthine (CEP) as an ASPH inhibitor that effectively suppressed tumor progression. Regarding the ICC lymphatic metastasis mechanism, we found that tumor cells in N1 lymph nodes highly expressed tumor-specific MHC-II molecules but lacked co-stimulatory factors CD80/CD86, inducing a state of anergy in CD4+ T cells, which might facilitate ICC immune evasion.


CONCLUSIONS
The TME of ICC was heterogeneous. ASPH markedly enhanced ICC invasion The ASPH inhibitor CEP significantly inhibits ICC progression and may serve as a targeted therapeutic drug for ICC. Tumor cells in N1 lymph nodes demonstrate high expression of tumor-specific MHC-II molecules, but silencing of co-stimulatory factors such as CD80/CD86 induces CD4+ T cells into an anergic state. Our study indicated that ASPH and MHC-II may serve as novel therapeutic targets for ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2012ee56dc72ff219a9edcce829b5a1b74d80284" target='_blank'>
              Tumor microenvironment heterogeneity and progression mechanisms in intrahepatic cholangiocarcinoma: A study based on single-cell and spatial transcriptomic sequencing.
              </a>
            </td>
          <td>
            Fengwei Li, Yao Li, Lishan Wang, Lei Xu, Hui Xue, Wenxin Wei, Yong Xia, Lei Wang, Feng Shen, Kui Wang
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/229292023d98ecab0e16b250609e553f46dd6a3a" target='_blank'>
              Organism-wide spatiotemporal profiling of gene expression utilizing a X-CreERT2/Ai9 tracing system
              </a>
            </td>
          <td>
            Feiyun Li, Qing Yao, Mingjue Chen, Shuangshuang He, Yi Ran, Jianglong Li, Lijun Lin, Guozhi Xiao
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer cell morphology reflects the dynamic interplay between genetic, epigenetic, and environmental factors that
drive tumor progression. This review explores the morphological evolution of cancer cells as a phenotypic manifestation of
underlying molecular dynamics, drawing on recent advances in evolutionary biology, cytoskeletal remodeling, computational
pathology, and tumor metabolism. Clonal evolution, both gradual and punctuated, generates morphological diversity that is
shaped by selective pressures from the tumor microenvironment and therapeutic interventions. Studies highlights the structural
and mechanical properties, such as reduced cellular stiffness and altered actin organization, are linked to metastatic potential
and cytoskeletal deregulation. Oncogenic signaling pathways (e.g., PI3K/AKT, Ras/MAPK) and metabolic reprogramming
further modulate cell shape and behavior. High-throughput imaging and machine learning have enabled quantification of both
static and dynamic morphological traits, correlating them with chromosomal instability and clinical outcomes. Morphological
heterogeneity has emerged as a powerful diagnostic and prognostic biomarker, with deep learning-derived metrics providing
scalable tools for cancer classification and risk stratification. Collectively, these insights underscore the clinical utility of
integrating morphological analysis with molecular profiling, offering new directions for personalized cancer diagnosis and
therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055067ac5bdde412ce5a95b6486dff3c99db46a4" target='_blank'>
              Morphology Matters: A Multidimensional Perspective on Cancer Cell Biology
              </a>
            </td>
          <td>
            Rose Rani
          </td>
          <td>2025-06-30</td>
          <td>International Journal for Research in Applied Science and Engineering Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/525bc26ccdba95190ad1d88938dd65bd9ec49925" target='_blank'>
              Cell painting in activated cells illuminates phenotypic dark space and uncovers novel drug mechanisms of action
              </a>
            </td>
          <td>
            Matylda Zietek, Akshar Lohith, Derfel Terciano, Beverley Marie Rabbitts, Aswad S. Khadilkar, John B MacMillan, Scott Lokey
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f86920663d31391d1e530628fe92866e6e71e1d" target='_blank'>
              stTrace: Detecting Spatial-Temporal Domains from spatial transcriptome to Trace Developmental Path
              </a>
            </td>
          <td>
            Zhangdi Song, Changyu Zheng, Jiaxing Chen
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell heterogeneity is a universal feature of life. Although biological processes affected by cell-to-cell variation are manifold, from developmental plasticity to tumour heterogeneity and differential drug responses, the sources of cell heterogeneity remain largely unclear1,2. Mutational and epigenetic signatures from cancer (epi)genomics are powerful for deducing processes that shaped cancer genome evolution3–5. However, retrospective analyses face difficulties in resolving how cellular heterogeneity emerges and is propagated to subsequent cell generations. Here, we used multigenerational single-cell tracking based on endogenously labelled proteins and custom-designed computational tools to elucidate how oncogenic perturbations induce sister cell asymmetry and phenotypic heterogeneity. Dual CRISPR-based genome editing enabled simultaneous tracking of DNA replication patterns and heritable endogenous DNA lesions. Cell lineage trees of up to four generations were tracked in asynchronously growing cells, and time-resolved lineage analyses were combined with end-point measurements of cell cycle and DNA damage markers through iterative staining. Besides revealing replication and repair dynamics, damage inheritance and emergence of sister cell heterogeneity across multiple cell generations, through combination with single-cell transcriptomics, we delineate how common oncogenic events trigger multiple routes towards polyploidization with distinct outcomes for genome integrity. Our study provides a framework to dissect phenotypic plasticity at the single-cell level and sheds light onto cellular processes that may resemble early events during cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc8a5871308fcb63b1e32d25ce00cce571108d0" target='_blank'>
              Multigenerational cell tracking of DNA replication and heritable DNA damage
              </a>
            </td>
          <td>
            Andreas Panagopoulos, Merula Stout, Sinan Kılıç, Peter Leary, Julia Vornberger, Virginia Pasti, A. Galarreta, Aleksandra Lezaja, Kyra Kirschenbühler, Ralph Imhof, H. Rehrauer, Urs Ziegler, Matthias Altmeyer
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>42</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy; however, no validated treatments are currently available. Clinically, the tumor position, head and uncinate process (HU), or body and tail (BT) of the pancreas are vital for surgical strategies; however, fundamental research has seldom revealed the heterogeneous tumor microenvironment (TME) among different types of PDAC. Here, we applied multicohort single-cell and spatial RNA-seq methods together with patient-derived organoid models to reveal the TME heterogeneity between HU and BT PDAC. Osteopontin, encoded by SPP1, is secreted by vessel endothelial cells in BT PDAC and is associated with increased tumor burden. The number of tumor cells marked by CRABP2 was lower in BT PDAC and was identified as a prognostic marker of overall survival, as well as CD8+ T-cell infiltration. The expression of CRABP2 was also validated to be downregulated in BT PDAC in patient-derived organoid models. Overall, we profiled the heterogeneous PDAC TME between HU and BT PDAC, which could provide novel insight into the relationships between clinical characteristics and TME molecular research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077490657a2feb3d7ea606c6fed7a402a475e6c0" target='_blank'>
              SPP1+ Venous Endothelial Cells and CRABP2+ Tumor Cells Contribute to Favorable Body and Tail Pancreatic Ductal Adenocarcinoma Tumor Microenvironment.
              </a>
            </td>
          <td>
            Yueze Liu, Yifan Fu, Tao Liu, Jun Wang, Zeyu Zhang, Yanan Shi, Zhe Cao, Gang Yang, Hao Chen, W. Luo, Jinxin Tao, Yuanyang Wang, Guihu Weng, Menggang Zhang, Liyuan Ye, Jianchun Xiao, J. Qiu, Taiping Zhang, Hua Huang
          </td>
          <td>2025-06-11</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Chromoanagenesis is a catastrophic genomic phenomenon involving sudden, extensive rearrangements within one or a few cell cycles. In osteosarcoma, the most prevalent malignant bone tumor in children and adolescents, these events dramatically alter the genomic landscape, frequently disrupting key tumor suppressor genes like TP53 and RB1, amplifying oncogene expression, and propelling tumor progression and evolution. This review elucidates how key chromoanagenic mechanisms, such as chromothripsis and chromoanasynthesis, arise from replication stress and impaired DNA repair pathways, ultimately contributing to genomic instability in osteosarcoma. Chromothripsis features prominently in osteosarcoma, occurring in up to 62% of tumor regions and driving intratumoral heterogeneity through persistent genomic crises. Next-generation sequencing, optical genome mapping, and emerging technologies like single-cell sequencing empower researchers to detect and characterize these complex structural variants, demonstrating how a single catastrophic event can profoundly influence osteosarcoma progression over time. While targeted therapies for osteosarcoma have proven elusive, innovative strategies harnessing comprehensive genomic profiling and patient-derived preclinical models hold promise for uncovering tumor-specific vulnerabilities tied to chromoanagenesis. Ultimately, unraveling how these rapid, large-scale rearrangements fuel osteosarcoma’s aggressive nature will not only refine disease classification and prognosis but also pave the way for novel therapeutic approaches to enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d502c33306d89faa688fa75a0e5f2710c0c03efc" target='_blank'>
              Chromoanagenesis in Osteosarcoma
              </a>
            </td>
          <td>
            Guozhuang Li, Nan Wu, Jennifer Ghabrial, Victoria Stinnett, Melanie Klausner, Laura A Morsberger, Patty Long, Ezra G. Baraban, J. Gross, Ying S. Zou
          </td>
          <td>2025-06-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Gliomas, particularly IDH-wildtype ones, are associated with poor prognosis, yet their immunological landscape remains uncertain. We analyzed RNA sequencing data from 55 glioma patients, estimating immune infiltration with CIBERSORTx and immune cell states via Ecotyper. IDH-wildtype gliomas showed significantly higher immune cell infiltration (p = 0.002), notably of regulatory T cells (Tregs) and macrophages, and a greater proportion of exhausted T cells compared to IDH-mutant gliomas. Clustering based on immune profiles revealed two groups. Cluster A, enriched for IDH-wildtype cases, exhibited heightened immune infiltration but also marked immunosuppression. Cluster B, which included both IDH-wildtype and mutant cases, showed lower levels of immune infiltration. Tumor-infiltrating lymphocyte (TIL) cultured from IDH-wildtype tumors demonstrated limited expansion following anti-PD-1, a CSF1R inhibitor, or a STAT3 inhibitor treatment, without clear cluster-specific differences. Tumor-reactive TILs were mainly observed in cluster A. These findings highlight that IDH-wildtype gliomas have an immunosuppressive and heterogeneous microenvironment, potentially limiting responses to single-agent immunotherapies. A personalized, multi-targeted approach addressing multiple immunosuppressive mechanisms may be essential to improve immunotherapy outcomes in this aggressive glioma subgroup.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb950007f7a53c8e4f48f1b9d238f1ea25d6914" target='_blank'>
              Dissecting the Immunological Microenvironment of Glioma Based on IDH Status: Implications for Immunotherapy
              </a>
            </td>
          <td>
            Miyu Kikuchi, Hirokazu Takami, Yukari Kobayashi, K. Nagaoka, Yosuke Kitagawa, Masashi Nomura, Shunsaku Takayanagi, Shota Tanaka, Nobuhito Saito, Kazuhiro Kakimi
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common and deadly malignancies worldwide, with a particularly low 5-year survival rate in advanced patients. Immune cells in the tumor microenvironment, especially mast cells, play crucial roles in tumor initiation and progression. However, the dual role of mast cells in CRC remains poorly understood. Methods In this study, we used single-cell RNA sequencing (scRNA-seq), bulk RNA sequencing, and bioinformatics analyses to explore the heterogeneity of mast cell subpopulations in the CRC tumor microenvironment and their relationship with prognosis. We analyzed gene expression signatures associated with mast cell subpopulations derived from single-cell data of 40 CRC tumor samples and combined bulk RNA-seq data from HMU, GEO, and TCGA cohorts for prognostic prediction. Non-negative matrix factorization was used for clustering of mast cell subpopulations, followed by analysis of their specific gene markers, transcription factor activity, and biological pathways. Survival analysis and ROC curves were performed to assess their prognostic significance. Results Mast cells in the CRC tumor microenvironment were classified into three distinct subpopulations, each with unique gene markers and functional pathways. Mast cell subpopulations 1 and 3 were highly associated with pro-tumor pathways, while mast cell subpopulation 2 primarily exhibited anti-tumor immune regulatory characteristics. High expression of mast cell subpopulations 1 and 3 was associated with poor survival prognosis, while high expression of subpopulation 2 was linked to a better survival outcome. Key marker genes such as DNAJB1, SEMA7A, and XCR1 were identified as potential prognostic factors, with high expression of DNAJB1 and SEMA7A being significantly associated with poor prognosis, while high expression of XCR1 was linked to a favorable prognosis. Conclusion This study reveals the functional heterogeneity of mast cell subpopulations in the CRC tumor microenvironment and their differential roles in tumor progression. Identification of mast cell subpopulation-specific marker genes provides new molecular targets for clinical diagnosis, prognostic prediction, and personalized immunotherapy in CRC. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04119-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f36d01ec68db5890c180d0142a0a3d3d4bc0a598" target='_blank'>
              Diversity of mast cell subpopulations in the tumor microenvironment of colorectal cancer and their prognostic implications
              </a>
            </td>
          <td>
            Tian-yu Qiao, Chao Ding, Songtao Yu, Wenyang Li, Yonghou Zhao, Guiyu Wang
          </td>
          <td>2025-06-30</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, M. T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Globally, endometrial cancer continues to impact a significant number of women. Immunotherapy provides those suffering from advanced or relapsed disease hope, but an important barrier is still the absence of trustworthy predictive biomarkers. To tackle this challenge, single-cell sequencing and spatial transcriptomics (ST) are increasingly applied. In cervical cancers of the no specific molecular profile (NSMP) subtype accompanied by p53 mutations. In many cases, the tumor microenvironment (TME) in endometrial cancer exhibits strong immunosuppression or poor immune cell infiltration, often leading to worse clinical outcomes. Single-cell sequencing reveals cellular heterogeneity and helps identify potential therapeutic targets and predict treatment responses. Conversely, ST assists in determining biomarkers that influence the effectiveness of immunotherapy by capturing the spatial organization of tumors. When combined, these technologies allow for integrated multi-omics analysis that aids in the development of immunotherapies, prognostication, and diagnosis. But there are still moral and legal issues. Clinicians may be able to improve outcomes for patients who don’t respond well to current immunotherapies by utilizing these combined approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f087dd53dee17003d5211dac4cb4360cce4847" target='_blank'>
              Single-cell technologies and spatial transcriptomics: decoding immune low - response states in endometrial cancer
              </a>
            </td>
          <td>
            Yumeng Li, Hua Qiu, Zhenzhen Zhao, Fanghua Qi, Pingping Cai
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease, with tumors capable of adapting to shifting conditions, making the development of effective personalized therapies particularly challenging. Patient-derived models, such as patient-derived organoids (PDOs) and circulating tumor cell (CTC) cultures, have emerged as powerful tools for investigating intra- and inter-tumor heterogeneity. These models largely retain the genetic, phenotypic, and microenvironmental features of the original tumors, providing valuable insights into disease progression, drug response, and resistance mechanisms. Furthermore, by enabling tumors’ spatiotemporal molecular profiling, PDOs and CTCs offer a dynamic approach to assess treatment efficacy over time. However, to fully capture the complexity of breast cancer heterogeneity, it is required to develop models from multiple tumor and blood samples collected throughout the course of treatment. This review explores the potential of integrating PDOs and CTC models to better understand intra-tumor heterogeneity while addressing key challenges in developing patient-derived models that accurately recapitulate patients’ tumors to advance personalized care. The integration of PDOs and CTCs could represent a paradigm shift in the personalized management of metastatic breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a82abf6024e151676efff540be9bde6be013c6" target='_blank'>
              Deciphering Breast Tumor Heterogeneity Through Patient-Derived Organoids and Circulating Tumor Cells
              </a>
            </td>
          <td>
            Benedetta Policastro, Nikoline Nissen, Carla L. Alves
          </td>
          <td>2025-06-25</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Brain tumors, in particular gliomas and glioblastoma multiforme (GBM), are thought to originate from different cells facing specific founding insults, a feature that partly justifies the complexity and heterogeneity of these severe forms of cancer. However, gliomas and GBM are usually reproduced in animal models by inducing molecular alterations in mature glial cells, which, though being part of the puzzle, do not represent the whole picture. To fill this conceptual gap, we previously developed a neurogenic model of brain cancer in Drosophila, demonstrating that the loss of cell polarity in neural stem cells (called neuroblasts in the fruit fly) is sufficient to promote the formation of malignant masses that continue to grow in the adult, displaying several phenotypic traits typical of human GBM. Here, we expand on previous work by restricting polarity disruption to Drosophila type II neuroblasts, whose self-renewal is comparable to that of mammalian neural progenitors, with the aim to capture the molecular signature of the resulting cancers in a specific and reproducible context. A comparison of the most deregulated transcripts with those found in human primary GBMs confirmed that our model can be proficiently used to delve into the roots of human brain tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/655c1526e0548f12b64a21fe0105b8cfbd172571" target='_blank'>
              Transcriptome-Wide Analysis of Brain Cancer Initiated by Polarity Disruption in Drosophila Type II Neuroblasts
              </a>
            </td>
          <td>
            Simona Paglia, P. Morciano, Dario de Biase, F. Giorgi, A. Pession, Daniela Grifoni
          </td>
          <td>2025-05-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Introduction Colorectal cancer (CRC), the third most common cancer worldwide, often shows limited responsiveness to immunotherapy due to its predominantly immune-excluded phenotype. Despite increasing insights into the complex tumor microenvironment (TME), the metabolic heterogeneity of CRC cells and their interactions with tumor-infiltrating immune cells remain poorly understood. Methods We analyzed 46,374 epithelial cells from 17 CRC patients treated with PD-1 blockade to develop an amino acid (AA) metabolism score using the AUCell algorithm. This score was applied to a separate single-cell RNA sequencing (scRNA-seq) dataset from 23 CRC patients to investigate cell-cell interactions and functions of tumor-infiltrating immune cells, revealing distinct immune TME landscapes shaped by tumor metabolism. An in vitro co-culture assay of CRC cells and CD8+ T cells was performed to validate the findings. Additionally, LASSO and Cox regression analyses were conducted to construct an AA metabolism-related risk score for predicting prognosis and drug sensitivity across multiple bulk transcriptome cohorts. Results This study identified a link between elevated amino acid metabolism in CRC epithelial cells and resistance to PD-1 blockade therapy. A 31-gene AA score was developed by intersecting differentially expressed genes between responders and non-responders to PD-1 blockade with amino acid metabolism-related genes from the Molecular Signature Database (MSigDB). Using this score, 23 additional CRC samples were classified into high and low AA score groups. Comparative analysis revealed that the low AA group exhibited a more robust immune response, characterized by a greater number and stronger cell-cell interactions. Tumor-infiltrating immune cells in this group demonstrated enhanced activation and anti-tumor functions. Furthermore, CD8+ T cells showed increased Granzyme B levels when co-cultured with CRC cells in which Psat1 or Shmt2 was knocked down. Finally, a machine learning-derived risk score based on six genes was established to translate single-cell findings to bulk transcriptomes. This risk score was found to correlate with immune checkpoint expression and immune cell infiltration, with potential implications for predicting prognosis and drug sensitivity. Conclusion Our findings highlight the role of elevated epithelial amino acid metabolism in shaping an immune-suppressive microenvironment, offering insights for patient stratification and therapeutic decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b9bda18d8cb73c7c5646b13e67e456908ec19a2" target='_blank'>
              Single-cell and bulk transcriptome analyses reveal elevated amino acid metabolism promoting tumor-directed immune evasion in colorectal cancer
              </a>
            </td>
          <td>
            Tianyue Sun, Yan Chen, Yingsheng Chen
          </td>
          <td>2025-05-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE Colorectal cancer (CRC) ranks third in global incidence and second in mortality, with rates increasing among younger populations. The enteric nervous system (ENS) is crucial for gastrointestinal function, and its dysfunction is associated with the progression of CRC. Neurodegeneration may disrupt ENS function and facilitate tumor growth although the mechanisms remain unclear. METHODS AND MATERIALS This study integrated single-nucleus, single-cell RNA sequencing, spatial transcriptomics (ST), and bulk transcriptomics data. Summary data-based Mendelian randomization (SMR) analysis identified neurodegeneration-related genes associated with CRC progression. Cell-cell communication was analyzed using CellCall and Cytotalk, and trajectory analysis performed using VECTOR. ST analysis was conducted using robust cell type decomposition, and spatial dependencies were assessed through MISTy and cell neighborhood network analyses. A deep learning–based prognostic model was developed using DeepSurv. Multiplex immunofluorescence was used to validate the functions of cells. RESULTS SMR analysis identified seven neurodegeneration-related genes, including MAPK9, associated with the development of CRC. High-neurodegeneration neurons were associated with poorer prognoses and infiltrated deeper into tumor niches. MISTy analysis revealed stronger spatial dependencies between high-neurodegeneration neurons and epithelial cells. Homotypic and heterotypic cell neighborhood network analyses revealed tighter neuron networks in high-degeneration groups, with enhanced neuron-epithelial communication. MAPK9-positive neurons were highly expressed in sensory neurons and facilitated tumor growth via EGFR signaling. The DeepSurv model demonstrated strong prognostic performance, with high-risk patients exhibiting worse outcomes and elevated expression of immune checkpoint genes. CONCLUSION Neurodegeneration, particularly involving MAPK9-positive neurons, promotes CRC progression through enhanced neuron-tumor interactions and EGFR signaling. This study underscores the significance of targeting neurodegeneration in CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ebbbaa1c310c2d03f5809c332bdce3f1b5993a" target='_blank'>
              Neurodegeneration Promotes Tumorigenesis in Colorectal Cancer: Insights From Single-Cell and Spatial Multiomics
              </a>
            </td>
          <td>
            Shaocong Mo, Bochen Pan, Liming Qin, Sheng Wang, Yulin Wang, Midie Xu, Fan Xia, Wenchao Gu
          </td>
          <td>2025-06-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c9003a002bf9a6c00cceca68fcf223699d2693" target='_blank'>
              Single-cell multi-omic integration analysis prioritizes druggable genes and reveals cell-type-specific causal effects in glioblastomagenesis
              </a>
            </td>
          <td>
            Y.-F. Huang, K.-L. Huang
          </td>
          <td>2025-07-01</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 2578


 Background:
 Beyond their vascular function, endothelial cells (ECs) regulate tumor growth through recently described angiocrine signaling, influencing cancer progression and treatment outcomes. Here, we applied a deep learning model, which we validated using spatial transcriptomics data, to a pan-cancer dataset to analyze the EC distribution associated with the response to immuno-oncology (IO) treatments.
 Methods:
 An AI-powered H&E analyzer, Lunit SCOPE IO, quantifies tumor microenvironment EC density and tumor-infiltrating lymphocytes (TILs) density in cancer epithelium and stroma. We validated AI-powered cell type prediction by evaluating cell-specific gene expression through spatial transcriptomics (10x Xenium). 7,467 pan-carcinoma samples from The Cancer Genome Atlas (TCGA) were analyzed for EC distribution and overall survival (OS). From a previously described multi-center, multi-national pan-cancer cohort (Pan-IO, Shen et al JITC 2024), 1,654 patients were analyzed for IO treatment response.
 Results:
 We validated our AI prediction of cell types by demonstrating consistency with known cardinal gene expressions from spatial transcriptomic results, where 77.5% of AI-predicted endothelial cells (ECs) expressed VEGFR2 (compared to 6.9% in tumor cells (TCs)), while VEGFA expression was 2.8 times higher in TCs. Consistent with previous studies, EC density was highest in RCC and HCC, while lowest in pancreatic adenocarcinoma, melanoma, and cholangiocarcinoma. While EC and TIL density were not correlated pan-cancer (r = 0.08), exceptions were head and neck cancer (r = 0.45, p < 0.001) and pancreatic cancer (r = 0.44, p < 0.001). In the TCGA cohort, the high EC density was associated with prolonged OS (HR 0.84, p < 0.001). In contrast, within the Pan-IO cohort (HR 1.26, p < 0.001) and its lung cancer subgroup (HR 1.26, p < 0.001) high EC density was associated with shorter progression-free survival (PFS) on treatment with IO monotherapy. Moreover, the predictive impact of EC density varied by TIL status. In the Pan-IO cohort, EC density predicted PFS in both TIL-high (HR 1.40, p < 0.001) and TIL-low cancers (HR 1.33, p < 0.001). Among four groups divided by median values, the EC-low and TIL-high group showed the longest PFS (median PFS of EC/TIL high/low: 2.5m, high/high: 3.2m, low/low: 3.6m, low/high: 5.6m, p < 0.001).
 Conclusions:
 EC distribution varied among cancer types, and importantly high EC content strongly correlated with poor IO monotherapy response, including NSCLC. These findings support exploring immunotherapy combination strategies that include anti-endothelial approaches, which could encompass the emerging class of PD-1/VEGFR bispecifics, for tumors with high EC content. Early evidence was observed for this for HCC (Chon et al ASCO GI 2024), with a hazard ratio of 0.62 for high EC HCC for atezolizumab/bevacizumab.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df61f044fca87805f1313365ca7a8942c487e0af" target='_blank'>
              Deep learning–powered H&E whole-slide image analysis of endothelial cells to characterize tumor vascular environment and correlate treatment outcome to immunotherapy.
              </a>
            </td>
          <td>
            Seungeun Lee, Jin Woo Oh, Soohyun Hwang, Jeanne Shen, Sehhoon Park, Hyojin Kim, Y. Chae, Se-Hoon Lee, Yoon-La Choi, Jin-Haeng Chung, Jaewoong Shin, Heon Song, Aaron Valero Puche, D. Yoo, Taebum Lee, Chiyoon Oum, Jeongmi Kim, S. Ali, C. Ock
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Recent advances in single-cell technologies have enabled the creation of comprehensive cell atlases, reference maps of various cell types within organisms. Here we specifically focus on T cell atlases, which offer a detailed catalog of the adaptive immune system at single-cell resolution. As such, they capture cellular diversity, functional states, and spatial dynamics across tissues, developmental stages, and disease conditions. Given the central role of T cells in orchestrating immune responses, their dysregulation underpins autoimmune disorders, cancer progression and failed immunotherapies. Therefore, a unified T cell atlas is critical for decoding such disease mechanisms, identifying therapeutic targets, and advancing personalized treatments. In this article, we explore the latest advances in T cell atlases, describing breakthroughs in multi-omics technologies, spatial profiling and computational frameworks that resolve transcriptional, epigenetic and proteomic heterogeneity. We also address persistent challenges and highlight strategies to address these gaps. Finally, we discuss emerging frontiers set to reshape our understanding of T cell dynamics in both health and diseases. Together, these insights underscore the transformative potential of T cell atlases in reconstructing precision immunology and accelerating therapeutic innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d0b0e5dda9a3845db88c70250149559b1315699" target='_blank'>
              Sketching T cell atlases in the single-cell era: challenges and recommendations.
              </a>
            </td>
          <td>
            Itana Bojović, António G. G. Sousa, S. Junttila, L. Elo
          </td>
          <td>2025-06-29</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Glioblastoma (GB) is the most aggressive tumor of the central nervous system (CNS), accounting for almost 80% of all primary brain tumors. Despite standard-of-care consisting of surgical resection, when possible, adjuvant radiotherapy (RT) and chemotherapy with Temozolomide (TMZ), GB remains highly fatal, with an estimated recurrence rate of over 90% and a median overall survival (OS) of around 15 months from diagnosis. Several factors contribute to such poor patient outcome, including a unique myeloid-rich tumor microenvironment (TME) that confers immunosuppression and therapeutic resistance. Multi-omics, single-cell transcriptomics and multi-modal spatial analyses of GB are unraveling the diversity of brain myeloid cells, including activated microglia, border-associated macrophages (BAM), and monocyte-derived glioma-associated macrophages (GAM), instructed by ontogeny, spatial distribution, cell-cell interactions and response to metabolic cues in the TME. In this review, we elaborate on the heterogeneity and plasticity of myeloid cells in GB and discuss the promise and challenges for rational therapeutic targeting of GAMs in GB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d921ef730c3240f4439955cc403f795493cbb47c" target='_blank'>
              Rational therapeutic targeting of myeloid cells in glioblastoma: challenges and perspectives
              </a>
            </td>
          <td>
            Faruk Akay, M. Saleh
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor diseases are characterized by high interindividual and intratumoral heterogeneity (ITH). The development and progression of neoplasms outline complex networks of extracellular and cellular signals that have yet to be fully elucidated. This narrative review provides a comprehensive overview of the literature related to the cellular and molecular mechanisms underlying the heterogeneity of the tumor mass. Furthermore, it examines the possible role of the tumor microenvironment in the development and support of the neoplasm, in order to highlight its potential in the construction of a diagnostic–therapeutic approach to precision medicine. Many authors underline the importance of the tumor microenvironment (TME) as it actively takes part in the growth of the neoplastic mass and in the formation of metastases and in the acquisition of resistance to anticancer drugs. In specific body districts, the ideal conditions occur for the TME establishment, particularly the inflammatory state, the recruitment of cell types, the release of specific cytokines and growth factors, hypoxic conditions. These components actively intervene by enabling tumor progression and construction of physical barriers shaped by the extracellular matrix that contribute to forming peripheral tolerance by intervention of myeloid precursors and the polarization of M2 macrophages. In recent years, ITH and the TME have assumed an important position in cancer research and pharmacology as they enable understanding the dense network of communication existing between the neoplasm and the surrounding environment, and to monitor and deepen the effects of drugs with a view to develop increasingly precise and effective therapies. In the last decade, knowledge of TME has been exploited to produce targeted molecular agents (inhibitory small molecules, monoclonal antibodies, gene therapy). Nonetheless, the bibliography shows the need to study ITH through new prognostic and predictive biomarkers (e.g., ctDNA and CTCs) and to increase its basic biology knowledge. Precision medicine is a new opportunity in the treatment of oncological diseases that is transforming the development of new drug approaches and their clinical use. Biology and biotechnologies are providing the bases for this revolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9eef0c89f58575a3286340faed6565d85cc9b7" target='_blank'>
              Microenvironment and Tumor Heterogeneity as Pharmacological Targets in Precision Oncology
              </a>
            </td>
          <td>
            S. Tonello, Roberta Rolla, Paolo Amedeo Tillio, PP. Sainaghi, Donato Colangelo
          </td>
          <td>2025-06-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Breast cancer exhibits heterogeneity at various levels, from subtype differences between patients (inter-patient heterogeneity) to diverse malignant cell compositions, variability in hormone receptor (HR) expression, and cellular makeup within single breast cancer samples (intra-tumour heterogeneity). Despite the availability of numerous effective therapies, many patients still experience incomplete treatment responses and subsequent relapse. These adverse outcomes are often due to the critical but frequently overlooked factor of cellular heterogeneity.
 To gain deeper insights into intra-tumour heterogeneity, we applied single-cell technologies to a cohort of 250 primary, untreated breast cancers. We optimized tissue cryopreservation methods, eliminating the need for fresh sample processing, and developed multiplex tissue profiling techniques. These cost-efficient processes improve handling of small tissue sizes, such as biopsies, and reduce batch effects. To ensure accurate and reliable data processing, we created a scalable computational workflow that includes benchmarked methods for sample SNP-demultiplexing, doublet detection, high-resolution cell annotation, and cellular integration. Additionally, we extended our existing methods to study the cellular heterogeneity of breast cancers.
 Our method, scSubtyper, examines the phenotypic differences between malignant cells within tumours by comparing each single cell to features associated with different molecular subtypes and assigning each cell to one of these subtypes. Our previous study and preliminary results from this project revealed that over 90% of samples exhibit a mix of malignant cells of different subtypes, and 50% of samples contain cells with characteristics of all subtypes, demonstrating that cellular heterogeneity exists not only between malignant cells but also within malignant cells of a tumour. Our second approach, known as ecotyping, assesses patterns of cell type frequencies across samples and groups them based on the similarity of these co-occurrences. Preliminary results have revealed the existence of five ecotypes that lack significant associations with clinical subtypes. Application of the same approach exclusively within the HR-positive samples identified four ecotypes characterized by distinct abundances of immune and stromal cells. This analysis revealed that ecotypes are not simple surrogates for clinical and molecular subtypes, but their presence could influence different responses to treatment.
 Together, our high-throughput tissue processing and computational approaches to study intra-tumour heterogeneity are now being applied to our large, well-annotated clinical cohort. Supported by preliminary results, we hypothesize that this study will play a vital role in optimizing breast cancer patient stratification to improve treatment management and outcomes.
 Citation Format: Beata Kiedik, Daniel L. Roden, Kate Harvey, Ghamdan Al-Eryani, Sunny Z. Wu, Mun N. Hui, Sandra O’Toole, Elgene Lim, Charles M. Perou, Alexander Swarbrick. Exploring cellular heterogeneity of localised breast cancers [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-09-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/355a116d2a0ae027b0d9081aa9023a8465a9971c" target='_blank'>
              Abstract P4-09-19: Exploring cellular heterogeneity of localised breast cancers
              </a>
            </td>
          <td>
            Beata Kiedik, D. Roden, K. Harvey, G. Al-Eryani, Sunny Z. Wu, M. Hui, Sandra O’Toole, Elgene Lim, C. Perou, Alex Swarbrick
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Simple Summary Single-label immunohistochemistry (sIHC) is used in infectious disease studies to detect etiological agents, and to quantify and spatially localize antigens in lesions. While sIHC is limited to staining one antigen per tissue section, multiplex immunohistochemistry (mIHC) offers significantly more detailed information and is more cost- and time-efficient than repeated sIHC. Given that swine lymph node pathology involves variations in T and B lymphocytes and macrophages, this study aims to apply mIHC to visualize simultaneously and on the same slide their spatial distribution and variation across the three main regions of the swine lymph node: follicles, interfollicular areas and medulla-like tissue. We demonstrated the use of bright-field mIHC for in situ analysis of spatial cellular distribution in swine lymph nodes during PCV2-SD and PRRSV lymphadenopathy, revealing distinct lymphocyte distributions compared to reactive lymphoid hyperplasia. The study highlights the potential of mIHC as a valuable tool for understanding and studying disease dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7c47de3fe690520b75f0437e0eae36becf35b80" target='_blank'>
              Bright-Field Multiplex Immunohistochemistry in Swine PCV2 and PRRSV Lymphadenopathies
              </a>
            </td>
          <td>
            G. D’Annunzio, L. V. Muscatello, Chiara Tugnoli, Stefano Pesaro, Andrea Luppi, Michelangelo Fiorentino, T. Franceschini, Alessia Grillini, G. Rugna, Giuseppe Sarli, L. Mandrioli
          </td>
          <td>2025-06-01</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cellular heterogeneity within cancer tissues determines cancer progression and treatment response. Single‐cell RNA sequencing (scRNA‐seq) has provided a powerful approach for investigating the cellular heterogeneity of both cancer cells and stroma cells in the tumor microenvironment. However, the common practice to characterize cell identity based on the similarity of their gene expression profiles may not really indicate distinct cellular populations with unique roles. Generally, the cell identity and function are orchestrated by the expression of given specific genes tightly regulated by transcription factors (TFs). Therefore, deciphering TF activity is essential for gaining a better understanding of the uniqueness and functionality of each cell type. Herein, metaTF, a computational framework designed to infer TF activity in scRNA‐seq data, is introduced and existing methods are outperformed for estimating TF activity. It presents the improved effectiveness in characterizing cell identity during mouse hematopoietic stem cell development. Furthermore, metaTF provides a superior characterization of the functional identity of breast cancer epithelial cells, and identifies a novel subset of neural‐regulated T cells within the tumor immune microenvironment, which potentially activates BCL6 in response to neural‐related signals. Overall, metaTF enables robust TF activity analysis from scRNA‐seq data, significantly enhancing the characterization of cell identity and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/310aebb046f221f9691b3d34683db0bbb309e243" target='_blank'>
              Accurate Transcription Factor Activity Inference to Decipher Cell Identity from Single‐Cell Transcriptomic Data with MetaTF
              </a>
            </td>
          <td>
            Yongfei Hu, Yuanyuan Zhu, Guangjue Tang, Ming Shan, Puwen Tan, Ying Yi, Xiyuan Zhang, Man Liu, Xinyu Li, Le Wu, Jia Chen, Hailong Zheng, Yan Huang, Zhuan Li, Xiaobo Li, Dong Wang
          </td>
          <td>2025-05-21</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Drug sensitivity is a major determinant influencing the efficiency of cancer treatment, and its evaluation faces critical challenges due to the cancer cell heterogeneity. Herein, a droplet-driven compartmentalization array (SCellEV chip) is presented to uncover single-cell trastuzumab sensitivity. Leveraging the differential flow resistance principle, the chip enables the flexible generation of single droplets orderly situated at designated spots. The single droplet includes parallel hydrodynamic microtraps empowers efficient compartments for single cell capture, and culture, and employs aptCD63-functionalized microbeads to immune-affinity capture single cell extracellular vesicles (EVs). The pairing capture of single cells and their EVs using single beads allows simultaneous identification of EVs and their original cells. The protein profiles (HER2 and CD63) and anti-cancer drug challenge revealed the cellular/EVs secretion heterogeneity at the single cell level and cell susceptibility to trastuzumab, further confirming that this chip achieves discriminating and predicting single breast cancer cells with different trastuzumab sensitivity using single cell/EVs profiling strategy. Furthermore, this analysis process retains the cell viability of measured cells to promote downstream detection integration. Overall, the SCellEV chip offers new avenues for single-cell analysis and exhibits great potential for cancer diagnostic and targeted treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdcd22f45999fb009740a6f20d00e3273f8e6d22" target='_blank'>
              A Microfluidic Droplet Array Promotes Trastuzumab Sensitivity Exploration of Single Breast Cancer Cells.
              </a>
            </td>
          <td>
            Junjie Bai, Xuan Zhang, Xing Wei, Hai-Yan Li, Ya-Nan Zhao, Ze-Jun Wang, Ting Yang, Jian-Hua Wang, Mingli Chen
          </td>
          <td>2025-06-16</td>
          <td>Small</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Nasopharyngeal carcinoma (NPC) is characterized by complex interactions within its tumor microenvironment. Understanding the immune landscape and gene expression patterns is crucial for developing effective therapeutic strategies. Methods We employed multiple analytical approaches including Mendelian randomization analysis, single-cell sequencing, gene expression profiling, and spatiotemporal analysis. The study investigated associations between NPC and comorbidities, characterized immune cell populations, and analyzed gene expression patterns. Cytokine profiles and their effects on disease risk were also examined. Results The analysis revealed potential associations between NPC and both allergic rhinitis and high myopia. Single-cell sequencing identified distinct cellular populations, including three unique B cell subpopulations (M1, M2, M3) with specific molecular signatures and spatial distributions. Temporal analysis showed dynamic changes in immune cell composition: endothelial and epithelial cells dominated the early phase, B cells peaked in the middle phase, and dendritic and T cells increased in the late phase. Gene expression clustered into four main patterns, with significant roles for immune-related genes, particularly the TNFRSF family and HLA-related genes. Cytokine analysis identified IL-6 as a significant risk factor (31.4% increased risk) while IL-10 showed protective effects (8% risk reduction). Conclusions This comprehensive analysis provides detailed insights into the complex immune microenvironment and molecular mechanisms of NPC. The identification of distinct cellular populations, temporal patterns, and key molecular players offers valuable information for understanding disease progression and developing targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a058ce64358d499d3ad8c6eb1186da1050c65662" target='_blank'>
              Mendelian randomization analysis reveals potential association between allergic rhinitis and nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Qingfu Bao, Jianjun Zou, Changyang Wang, Haiying Wang
          </td>
          <td>2025-05-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This study presents a pioneering exploration into the role of aggrephagy-related genes (ARGs) in glioblastoma (GB), a kind of malignant tumor which is highly invasive and resistant to a series of therapy. Utilizing single-cell sequencing to dissect their influence on the tumor microenvironment (TME) and tumorigenesis. By applying non-negative matrix factorization for dimensionality reduction and clustering of single-cell data, distinct cellular subtypes within the TME influenced by ARGs were identified, uncovering their functions and interactions. The investigation extends to validating the prognostic significance of ARGs and their potential in predicting immunotherapy outcomes. Molecular docking analysis of key ARGs further highlights TUBA1C and UBB as promising therapeutic targets, offering novel insights into GB’s complex biology and suggesting a targeted approach for therapy, which is characterized by some crucial pathways in our analysis, including PI3k-akt and TGF-beta pathways. This comprehensive single-cell level examination not only advances our understanding of aggrephagy’s role in GB but also proposes new avenues for prognosis and treatment strategies, emphasizing the critical impact of ARGs on the TME and GB progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d160ce3578d46173e199ce3a22ee88e96cde97" target='_blank'>
              Integrating multi-omics data to identify the role of Aggrephagy-related genes in tumor microenvironment and key tumorigenesis factors of GB from the perspective of single-cell sequencing
              </a>
            </td>
          <td>
            Zipei Chen, Shengke Zhang, Chenglu Jiang, Lai Jiang, Haiqing Chen, Jinbang Huang, Jie Liu, Guanhu Yang, Xiufang Luo, Hao Chi, Jiangping Fu
          </td>
          <td>2025-05-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 e14628


 Background:
 Detecting clinically relevant biomarkers in cancer tissues provides key insights into the unique tumor characteristics of patients, allowing for more personalized and effective therapies. Clinical biomarkers such as PD-1 and PD-L1 are associated with an immune-suppressive tumor microenvironment (TME), whereas HER2 is associated with metastasis and recurrence in multiple types of cancers. Immunohistochemistry (IHC) is the gold-standard technique for biomarker detection and is widely used by pathologists to grade tissue expression. Limitations related to plexity, quantitation and false signal detection are frequently observed using IHC, and day-to-day variability due to signal amplification and false positive signal can misinform pathologists about biomarker expression. Imaging Mass Cytometry (IMC) technology is a multiplexed spatial imaging technique that incorporates stoichiometric and quantitative assessment of 40-plus biomarkers simultaneously on a single slide and offers a large dynamic range of signal detection. We strove to determine whether IMC based spatial proteomics can be used for pathological evaluation of PD-1, PD-L1 and HER2 and provide key biological insights for clinical and translational studies.
 Methods:
 We stained serial sections of tissues using the same antibody clone and generated IHC and IMC data, which was assessed by a board-certified pathologist. For IMC technology, we detected single cells using the Human Immuno-Oncology IMC Panel, which highlights individual tumor, immune and stromal components of the TME. We conducted quantitative image analysis to detect expression of relevant biomarkers on cells and found enriched cellular neighborhoods associated with various pro- and antitumor processes.
 Results:
 Our analysis demonstrated that IMC technology and IHC similarly detected PD-1 and PD-L1. However, IMC accomplished it without signal amplification. For HER2, IMC technology and IHC provided comparable data. However, IMC detected the true dynamic range of signal intensities. Quantitative comparison of IMC technology combined with single-cell spatial proteomic analysis resolved the location of PD-1-, PD-L1- and HER2-expressing cells relative to other immune, stromal and tumor cell populations and offered additional biological insights into disease mechanisms.
 Conclusions:
 Clinical assessment of tissues using IMC technology offers an advantage over IHC by providing true biological context through multiplexing capabilities. High-dimensional spatially resolved data offered by IMC technology has the potential to expand our understanding of disease mechanisms of cancers and expedite development of personalized therapies for cancer patients in the clinic. For Research Use Only. Not for use in diagnostic procedures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8ff0c346242682cc50f675a70c19adc31dc281" target='_blank'>
              Comparative analysis of immunohistochemistry and imaging mass cytometry technologies for detection of clinical biomarkers in cancer tissues.
              </a>
            </td>
          <td>
            Qanber Raza, Smriti Kala, T. Pfister, Liang Lim, David King, C. Loh
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Mechanisms of HIV persistence in tissues are distinct from that in the blood. Spatial transcriptomic profiling examines HIV-infected cells, surrounding neighborhoods, and tissue microenvironment in unprecedented resolution. Spatial profiling captures cytokine gradients, distances between HIV-infected cells and immune effectors (and their function versus exhaustion), and cell-cell interactions. We present an overview of spatial transcriptomic platforms and a workflow of quality controls, sanity check, and bioinformatic analysis.


RECENT FINDINGS
The selection of spatial profiling methods should base on the research question, resolution, breadth of coverage, the expression level of RNA of interest, tissue quality, and tissue size. Advanced spatial transcriptomic profiling can capture RNA molecules at high resolution (<1 μm) and thus enable near-single cell profiling at genome-wide (~20 000 genes) breadth. Specifically, poly-A-based mRNA capture can identify previously unknown targets, while targeted RNA capture increases sensitivity in low-quality tissues. In targeted capture, however, the increase in target numbers frequently decreases sensitivity. Coupling ATAC-seq, protein capture, and T cell receptor sequencing to spatial platforms is ongoing.


SUMMARY
Spatial transcriptomic profiling uncovers mechanisms of HIV persistence in tissues and informs therapeutic strategies. Investigators should ensure the rigor of analysis, validate findings, and avoid reporting signatures with unknown biological significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f09209df407344118779f936014e334d9c123d1" target='_blank'>
              Roadmap for spatial transcriptomics of HIV in tissues.
              </a>
            </td>
          <td>
            Qijie Guan, Amare Eshetu, Ya-Chi Ho
          </td>
          <td>2025-07-07</td>
          <td>Current opinion in HIV and AIDS</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Melanoma is a highly heterogeneous malignancy with diverse molecular and clinical behaviors. A precise molecular classification is critical for improving prognostic assessment and guiding personalized therapy. Methods We performed an integrative multi-omics analysis of skin cutaneous melanoma using data from The Cancer Genome Atlas (TCGA) and validated our findings in independent cohorts. Multi-layered data, including transcriptomic, genomic, epigenetic, and immune landscape profiles, were analyzed using unsupervised clustering and machine learning approaches to define molecular subtypes. Functional assays and in silico drug screening were employed to explore subtype-specific vulnerabilities. Results Three robust molecular subtypes (CS1, CS2, CS3) were identified, each with distinct genomic alterations, tumor microenvironment characteristics, and clinical outcomes. The CS2 subtype was immunologically “hot,” characterized by high tumor mutational burden (TMB), elevated neoantigen load, strong immune infiltration, and activated IFN-γ signaling. CS2 tumors showed significant enrichment of immune checkpoint gene expression and were associated with favorable response to anti-PD-1 therapy in external validation cohorts. In contrast, CS1 and CS3 were immunologically “cold” with immune exclusion, high chromosomal instability, and activation of oncogenic pathways linked to immune evasion. Transcriptomic drug sensitivity modeling suggested that CS1 and CS3 may benefit from HSP90 or MEK inhibitors. Moreover, COL11A2 was identified as a subtype-enriched oncogenic driver predominantly expressed in CS1/CS3, and its silencing impaired tumor cell proliferation, invasion, and epithelial–mesenchymal transition (EMT) features. Conclusions This study presents a refined multi-omics classification of melanoma that reveals biologically and clinically distinct subtypes with divergent immune and therapeutic profiles. It offers a framework for subtype-specific treatment strategies, and identifies COL11A2 as a potential target in immune-cold melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47ce450bfa8af83563781e72095e944188c7b621" target='_blank'>
              Multi-omics-based subtyping of melanoma suggests distinct immune and targeted therapy strategies
              </a>
            </td>
          <td>
            Changchang Li, Xiaoqiong Lin, Jinhui Wang, Qiaochu Zhou, Fangfang Feng, Jie Xu
          </td>
          <td>2025-06-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Spatial transcriptomic analysis has proposed valuable insights into the behavior of tongue cancer. However, the specific cell types involved in chemically-induced carcinogenesis and the process of tumor development remain elusive.We leveraged artificial intelligence (AI) algorithms and spatial transcriptomic sequencing to meticulously characterize the spatial and temporal evolution of 4-nitroquinoline-1-oxide (4NQO)-induced tongue carcinogenesis and intratumor heterogeneity.An AI classifier effectively categorized dysplastic tongue tissue into 13 distinct groups. Spatial transcriptomics identified 13 corresponding cellular subgroups with unique features within the lesion. Both methods successfully distinguished subtle muscle phenotype and genetic lineage variations induced by 4NQO, despite limited morphological differences. Evolutionary tree analysis revealed the dynamic appearance and disappearance of functionally and genetically diverse cell subgroups during the progression from epithelial dysplasia to in situ carcinoma and invasive cancer. Key findings include the identification of specific switch genes associated with tumor invasion and the revelation of significant intratumor heterogeneity.This spatial transcriptomic analysis of 4NQO-induced tongue cancer provides a detailed characterization of tumor evolution and heterogeneity. It elucidates critical aspects of tongue cancer cell behavior and identifies potential therapeutic targets (switch genes). These findings offer novel insights for improving the diagnosis and treatment of tongue cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679c71fb0ae1356ab18cbbcb3f0859a89370b05e" target='_blank'>
              Spatial transcriptomic analysis of 4NQO-induced tongue cancer revealed cellular lineage diversity and evolutionary trajectory
              </a>
            </td>
          <td>
            Feng Liu, Xiaojun Wu, Shaoqing Yu, Deqiang Cheng, Jun Pan, Xiaodong Wang, Guanzhen Yu, Chaofu Li
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e17615


 Background:
 Uterine carcinosarcoma (UCS) is a rare gynecologic malignancy that has poor clinical outcomes and few targeted therapies. This aggressive variant of endometrial carcinoma has a uniquely biphasic appearance and morphology of metastases from these tumors can be carcinoma, sarcoma, or carcinosarcoma. The tumor immune microenvironment (TIME) plays a crucial role in the behavior of cancers. Recent advances in single cell spatial transcriptomics allow for greater insight into the TIME of these aggressive tumors.
 Methods:
 Ten samples from four patients with multiple matched longitudinal samples underwent single-cell spatial transcriptomic (scST) profiling (Nanostring CosMx 6k). Standard segmentation and QC filters were applied. Cell typing was performed with confirmation by expert pathologist annotation of tissue regions. Normalized gene expression was compared within the clusters based on clinicopathologic characteristics.
 Results:
 scST profiling of 80,843 high-quality cells revealed 38 distinct clusters (11 carcinoma, 12 sarcoma, 9 immune, 7 benign mesenchymal). Tumor clusters were largely patient-specific with tumors demonstrating diverse transcriptional heterogeneity while metastatic tumors were composed of several less diverse clones. While immune cell clusters represented cells from all patients, there was significant patient and sample heterogeneity in the proportions of immune cell subsets observed. Macrophages were the most abundant immune cell type in all samples, plasma cells were common in one patient's samples, and rare T cells were found only in specific cellular niches. Transcriptomic features of intra-tumoral macrophages were assessed based on site (primary versus distant metastasis) and temporal (naïve versus treated with chemotherapy) groupings. Compared to tumor associated macrophages (TAM) in primary tumors, TAMs in distant metastases were more likely to have M2-like features with higher expression of
 CXCL2, CD163, CD209,
 and
 CD44
 (P<0.005). TAMs residing in chemotherapy-treated tumors had higher expression of
 IGF2, CXCL2, CXCL1, CD163, CD209, CXCL16, CD24
 ,
 CD44, CD93
 (P<0.005). Upregulation of pro-angiogenic pathways (i.e.
 VEGFA
 ) was common in both distant metastases and post-chemotherapy UCS TAMs.
 Conclusions:
 Uterine carcinosarcoma is a highly heterogeneous tumor both with respect to tumor cell and immune microenvironment features. This work highlights transcriptomic characteristics in macrophages in the UCS microenvironment. Notably, macrophages in distant metastases and in patients with prior chemotherapy were found to have more M2 TAM-like macrophages than primary untreated tumors. These results provide rationale for both TAM-based immunotherapy and anti-angiogenic approaches in UCS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ce5dd582063cc9e8b6f92d0391909010dfa4c1" target='_blank'>
              Single cell characterization of uterine carcinosarcoma and analysis of transcriptomic shifts in tumor-associated macrophages in metastatic and chemotherapy-treated tumors.
              </a>
            </td>
          <td>
            C. Nief, Sabrina Zdravkovic, Sahar Nasr, Phoebe Hammer, Brooke Howitt
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 DNA methylation, an essential epigenetic modification, plays a critical role in establishing and maintaining cellular identities by regulating gene expression. In cancer, aberrant DNA methylation patterns contribute to tumor development and progression, yet the cell-type-specific methylation alterations within the tumor microenvironment (TME) remain poorly understood. In breast cancer, a highly heterogeneous malignancy, endothelial cells within the TME (TECs) are key players in tumor angiogenesis and progression. Methylation changes detected in bulk tumor analyses, however, reflect signals from multiple cell types present in the TME, making it challenging to pinpoint cell-type-specific epigenetic changes. Recently, methods have been developed to using DNA methylation to deconvolute cell type proportions which can in turn be used to attribute methylation alterations to specific cell types in the TME. A reference-based cell-type deconvolution algorithm, the Hierarchical Tumor Immune Microenvironment Epigenetic Deconvolution (HiTIMED) method, allows accurate quantification of cell type proportions within the TME. We accessed DNA methylation array data from 609 breast tumors and 331 nontumor normal breast tissues from the Gene Expression Omnibus (GEO) and the Genotype-Tissue Expression (GTEx) portal. To these DNA methylation profiles, we applied HiTIMED and measured tumor and non-tumor cell proportions. Then, applying a statistical interaction model called CellDMC we leveraged the cell type proportion data to identify differential methylation specific to cell types in the breast cancer TME, including TECs and tumor cells. We identified 1897 tumor-cell-specific and 180 TEC-specific differentially methylated cytosines (DMCs) in cancer compared to normal breast tissue (FDR ≤ 1e-10). Importantly, in the cell-type unadjusted differential analysis, tumors exhibited hypermethylation compared to normal tissues; however, in the tumor-cell-specific analysis, tumor cells were hypomethylated relative to normal samples. By applying these advanced techniques to breast cancer samples, our study unveils cell-type-specific DNA methylation changes that are pivotal in driving the pro-tumorigenic environment. These findings not only enhance our understanding of the molecular mechanisms underlying breast cancer progression but also highlight potential therapeutic targets within the TME. The integration of HiTIMED and CellDMC represents a significant methodological advancement in cancer epigenetics, offering a more precise and insightful approach to studying the complex interplay between different cell types within the TME.
 Citation Format: Barbara Barbara, Lucas A. Salas, Brock C. Christensen. Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-29.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77ae964d40b279e042709ea22a740c1da93f0fc" target='_blank'>
              Abstract P5-06-29: Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution
              </a>
            </td>
          <td>
            Barbara Barbara, Lucas A Salas, Brock C. Christensen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Gliomas are highly aggressive, life-threatening tumors with poor prognosis, and remain a leading cause of mortality among brain cancers. Although the role of mitochondrial proteins in cancer has garnered increasing attention, their specific functions in the nervous system, particularly in gliomas, remain poorly understood. Methods We integrated single-cell RNA sequencing with cellular assays and flow cytometry to investigate the molecular characteristics and cellular interactions within glioblastoma subpopulations during tumor progression. Results Single-cell RNA sequencing revealed several differentially expressed genes (DEGs) within glioblastoma subpopulations. Trajectory analysis identified CHCHD2P9 as a pivotal marker for the terminal subpopulation. Moreover, elevated expression of CHCHD2P9 was found to correlate with poorer clinical outcomes. Subsequent cellular experiments further explored the underlying mechanisms driving these observations. Conclusions CHCHD2P9 is significantly overexpressed in glioma patients, and its differential expression plays a crucial role in regulating glioma cell proliferation and migration. A CHCHD2P9-based risk model holds promise as both a prognostic biomarker and a potential therapeutic target, providing novel insights into the pathogenesis of gliomas and opening avenues for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d625edcb2052943e2c1f19b8599a0238510719f1" target='_blank'>
              Molecular insights into glioblastoma progression: role of CHCHD2P9 in tumor heterogeneity and prognosis
              </a>
            </td>
          <td>
            Yanyu Ding, Lizhi Xiao, Xiaoling Zhou, Jiaxin Zhao, Jianli Ke, Hua-Bao Cai, Mengyu Zhao, Cun-Zhi Wang, Tian-Hang Yu, Zhijie Zhao, Yucai Wang, Jiyuan Ke
          </td>
          <td>2025-06-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dd545ad2008fe46e54493be64fde82d63829c2a" target='_blank'>
              Improving Cellular Senescence Detection with Weak Gene Markers in Single-Cell RNA-seq vi Iterative Imputation
              </a>
            </td>
          <td>
            Peng Xu, Hantao Zhang, Siyao Zhu, Yimeng Kong
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e14635


 Background:
 Recent advances in computational pathology have revolutionized the analysis of histopathology images, enabling precise and rapid clinical outcome predictions. However, a significant challenge persists in translating computational insights into biologically meaningful data that informs clinical decisions. Human-interpretable image features (HIFs) have emerged as a solution, offering a detailed view of the tumor microenvironment (TME). In this study, we explored how HIFs derived from high-resolution histopathology images can predict the expression of genes associated with poor survival outcomes in stomach adenocarcinoma (STAD), aiming to bridge the gap between computational pathology and actionable clinical insights.
 Methods:
 Whole-slide histopathology images of STAD from The Cancer Genome Atlas were analyzed. Expert pathologists annotated labels were used to identify tissue types like cancer, stroma, necrosis, and normal tissue and cell types, including cancer cells, lymphocytes, macrophages, plasma cells, and fibroblasts. Using these annotations, convolutional neural networks were trained to classify these tissue and cell types, from which HIFs were extracted reflecting the biological composition of the TME, such as cell density ratios relative to surrounding tissues. We then identified genes with strong correlations to HIFs (ρ > 0.5) for further analysis.
 Results:
 Our results revealed high expression of four genes
 ABCA6, ABCA8, ADAM33,
 and
 ADAMTS10
 that were significantly associated with poor survival outcomes in STAD (p < 0.01). Each gene demonstrated a strong correlation (ρ > 0.5 and p = < 0.01) with specific HIFs.
 ABCA6
 and
 ABCA8
 , genes involved in lipid transport, are linked to a high fibroblast-to-stroma density ratio. This stromal-dominant environment is a hallmark of therapy resistance and poor outcomes.
 ADAM33
 and
 ADAMTS10
 , proteases involved in extracellular matrix remodeling, were strongly correlated (ρ > 0.5a and p = < 0.01) with a high macrophage-to-stroma density ratio. This feature represents an immunosuppressive TME, often seen in aggressive and invasive cancers.
 Conclusions:
 These findings underscore the power of HIFs to reveal key TME characteristics. A stromal-dominant TME, defined by elevated
 ABCA6
 and
 ABCA8
 , and an immunosuppressive, macrophage-rich environment associated with increased
 ADAM33
 and
 ADAMTS10
 , serve as markers of disease aggressiveness and poor prognosis. By integrating these insights into AI-driven models, clinicians may identify high-risk patients earlier in their treatment journey, enabling personalized treatment strategies to improve outcomes for STAD patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebad59e7c9b25a675a77d956eb6f9b7c2d45acd0" target='_blank'>
              Linking gene expression to tumor microenvironment using H&E features in stomach adenocarcinoma.
              </a>
            </td>
          <td>
            S. H. Bukhari, F. Vempalli, J. D. Warren, Shyam Aggarwal, Aditya Sarin, Mandeep Singh Malhotra, Rakesh K. Yadav, M. Macha, A. Bhat, Harry Lander, Tariq Masoodi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="BACKGROUND
Despite extensive research efforts, glioblastoma (GBM) remains a deadly disease with poor prognosis. Although previous studies have identified various cell states within GBM tumors, the molecular mechanism underlying adaptive GBM cell plasticity induced by conventional therapy remains unclear.


METHODS
We used fluorescent reporters for proneural (PN) and mesenchymal (MES) subtypes to monitor GBM cell plasticity in real-time across multiple patient-derived cell lines. This approach revealed cells that concurrently expressed both proneural and mesenchymal markers. To investigate this unique hybrid population, we implemented a comprehensive methodological approach encompassing bulk and single-cell RNA sequencing, single-cell ChIP sequencing, nuclear proteomics, high-resolution imaging, orthotopic mouse models, clinical dataset analysis, and pharmacological and genetic techniques. This multifaceted strategy allowed us to gain functional and molecular insights into this distinct cellular population.


RESULTS
We showed that these hybrid cells are increased by conventional therapies, and are resistant to these therapies. At the molecular level, hybrid cells display significant alterations in chromatin structure and nuclear protein composition, elevated transcriptional activity, Myc activation, and improved transport between the nucleus and cytoplasm. Genetic and pharmaceutical inhibition of the nuclear import/export shuttling machinery, increased in hybrid cells, effectively suppressed adaptive GBM cell plasticity and hybrid identity, thereby enhancing the sensitivity of GBM cells to therapies.


CONCLUSION
Our results indicate that GBM hybrid cells play a crucial role in chemoradiation resistance. The nuclear transport machinery presents a potential therapeutic target for hybrid cells, offering a way to counteract the typical resistance to treatment observed in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0e616d882e7bd8159604ae82c04f4049ee875a" target='_blank'>
              Proneural-Mesenchymal hybrid glioblastoma cells are resistant to therapy and dependent on nuclear import.
              </a>
            </td>
          <td>
            G. Bourmeau, O. Anezo, Jeremy Raymond, A. Ballestín, C. Pichol-Thievend, J. Reveilles, Adrien Thomas, Lin Wang, Melanie Miranda, Eve Moutaux, S. Liva, V. Ribecco, L. Besse, F. Dingli, D. Loew, Céline Vallot, G. Gargiulo, V. Ravi, Kevin Joseph, G. Seano
          </td>
          <td>2025-07-08</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background: Lung adenocarcinoma (LUAD) remains a leading cause of cancer-related mortality despite advances in treatments, necessitating more effective therapeutic strategies. Single-cell RNA sequencing (scRNA-seq) technology has revolutionized our ability to dissect the cellular complexity of cancers, which is often obscured in conventional bulk transcriptomic experiments. Methods: In this study, we performed an integrative analysis of scRNA-seq data from multiple LUAD patient cohorts to investigate cell-type-specific transcriptomic changes across disease stages. Clustering, lineage trajectory analysis, and transcriptional regulatory network reconstruction were employed to identify stage-specific gene markers and their upstream regulators. Additionally, we constructed intercellular communication networks to evaluate signaling changes within the tumor microenvironment (TME) during LUAD progression. Results: Our analysis revealed that epithelial cells from stage IV tumors exhibited a distinct transcriptional profile compared to earlier stages, a separation not observed in immune or stromal cell populations. We identified a panel of gene markers that differentiated epithelial cells across disease stages and effectively stratified patients into subgroups with distinct survival outcomes and TME compositions. Regulatory network analysis uncovered key transcription factors, including ATF3, ATF4, HSF1, KLF4, and NFIC, as potential upstream regulators of these stage-specific genes. Moreover, cell–cell communication analysis revealed a significant increase in signaling originating from epithelial cells and a concomitant decrease in immune-derived signals in late-stage LUAD. We identified several signaling pathways enriched in stage-specific crosstalk, including Wnt, PTN, and PDGF pathways, which may play critical roles in LUAD progression. Conclusions: This study provides a comprehensive single-cell resolution map of LUAD progression, highlighting epithelial-driven regulatory programs and dynamic intercellular communication within the TME. Our findings uncover novel molecular markers and regulatory mechanisms with potential prognostic and therapeutic value for more precise treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ecbc245a8fbf12aaba1f14070430ba2f845cb23" target='_blank'>
              Single-Cell Transcriptomic Analysis Unveils Key Regulators and Signaling Pathways in Lung Adenocarcinoma Progression
              </a>
            </td>
          <td>
            Jialu Ma, Caleb McQuay, John Talburt, Amit K. Tiwari, Mary Qu Yang
          </td>
          <td>2025-06-30</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 3593


 Background:
 Variation in response to radiotherapy for the treatment of rectal cancer is likely due to heterogeneity in the tumour microenvironment. However, to date, no reliable predictive biomarkers of response are in clinical use and the mechanisms underlying response are unknown. Tertiary lymphoid structures (TLS), which are ectopic lymphoid aggregates found in the tumour microenvironment, have been linked to response to immunotherapy, but little is known about their role in radiotherapy. Here, we aimed to explore the potential of lymphocytes and tertiary lymphoid structures as predictive biomarkers of response to radiotherapy and profile the tumour immune microenvironment in the context of response to radiotherapy.
 Methods:
 For this study, we accessed pre-treatment biopsies from 20 rectal cancer patients with known pathological response to long-course chemoradiotherapy (LCCRT). We selected regions of interest based on immunohistological identification of tumour and lymphocytic infiltrate in formalin-fixed paraffin-embedded tissue. We performed targeted proteomic profiling of 87 immuno-oncology proteins using the Nanostring GeoMx Digital Spatial Profiler to quantify protein expression with spatial resolution within regions of interest, including TLSs, in the tumour microenvironment.
 Results:
 Unsupervised clustering based on normalised protein expression showed a clear separation between the complete responders to LCCRT and all other tumours, and this separation is driven by differences in T cells within TLSs (CD3+). Differentially expressed proteins within CD3+ aggregates include depletion of the natural killer cell marker, CD56 and increased expression of the apoptosis marker, cleaved caspase 9. The distribution of TLS-tumour distance was also significantly different between response groups.
 Conclusions:
 The study highlights the role of TLSs in modulating the immunogenic landscape of the tumour microenvironment in rectal cancer, likely influencing the response to radiotherapy. Spatially resolved proteomic analyses identifies potential biomarkers for radiotherapy response and underscores the importance of profiling tumour-immune microenvironment complexity when stratifying patients for therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38e831504bf073567c720a00c7fb61ae0355f499" target='_blank'>
              Digital spatial profiling: Mapping tumor responses to radiotherapy in rectal cancer.
              </a>
            </td>
          <td>
            R. Purcell, Adèle Hegoburu, Mohammad Amer, Arielle Sulit, Frank Frizelle
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 2042


 Background:
 Glioblastoma (GBM) is the most lethal primary brain tumor in adults, with a median survival of ~15 months despite current therapies (surgery, radiation, temozolomide). Advances like immune checkpoint inhibitors, anti-angiogenic agents, and tumor vaccines have shown suboptimal results. The 2021 WHO classification highlights molecular markers (e.g., IDH, MGMT) for better stratification, but these fail to fully capture tumor microenvironmental dynamics. Using an AI-driven transcriptomic approach, we identified novel prognostic subtypes in IDH-wildtype GBM, aiming to refine stratification, enhance understanding of tumor biology, and guide personalized therapeutic strategies.
 Methods:
 We accessed microarray data from The Cancer Genome Atlas (TCGA) (n=353 newly diagnosed, IDH-WT GBM) for a training set and RNA-seq data from the Chinese Glioma Genome Atlas (CGGA) (n=170 primary and n=106 recurrent tumors) for validation. A proprietary SphereBio machine learning–based algorithm was used to derive transcriptomic signatures with prognostic relevance. Subtypes were assessed via Kaplan–Meier analyses in the training cohort and tested in both primary and recurrent validation cohorts. Immune/stromal infiltration was quantified using a tumor deconvolution tool (DA_505), and pathway enrichment (GAGE) was performed on the validation sets.
 Results:
 AI-driven clustering revealed three transcriptomic subtypes with significant survival differences in both the training (p<0.0001) and primary validation (p=0.0004) cohorts. In the recurrent cohort, a similar survival trend by subtype was observed, though significance was diminished (p=0.12), likely due to limited sample size and therapy-related changes. Immune/stromal deconvolution showed distinct infiltration patterns: subtypes enriched for CD4+ and CD8+ T cells correlated with prolonged survival. Pathway enrichment analysis in both primary and recurrent tumors highlighted potential targets involving embryogenesis, immune modulation, cell cycle, and stress response. The persistence of a consistent survival trend and comparable microenvironment and pathway patterns suggest that these transcriptomic subtypes remain biologically relevant even after standard treatment.
 Conclusions:
 Our integrated transcriptomic and microenvironment-focused approach identified three prognostically distinct GBM subtypes, validated across independent cohorts. These findings underscore the utility of AI-driven transcriptomic signatures for personalized stratification, with the potential to guide targeted therapeutic strategies and inform clinical trial design in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff54807210ca2a362e060a895b6f6e635241ee" target='_blank'>
              AI-driven transcriptomic classification of glioblastoma: Associations with survival and tumor microenvironment.
              </a>
            </td>
          <td>
            J. Fernandez-Muñoz, Román Oberti, D. Reardon, Francisco Quintana, Guido Nicolas Molina, M. E. Guerrero-Gimenez
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/129265cf9a8d9036012e909d0315f3224ece47d6" target='_blank'>
              The epigenetic regulator TRIM24 controls melanoma cell dedifferentiation and resistance to treatment in melanoma
              </a>
            </td>
          <td>
            Simon Durand, Félix Boivin, Roxane M. Pommier, L. Barbollat-Boutrand, Maxime Grimont, Félix Pham, Marion Dufeu, Eric Cumunel, Raphaël Schneider, Anthony Ferrari, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fb1152ca394367d1b31e6130a4964a702584963" target='_blank'>
              Multiscale Modeling Uncovers Macrophage Infiltration and TNF-α Signaling Networks for Targeting in Inflammatory Breast Cancer Tumor Emboli
              </a>
            </td>
          <td>
            Pritha Pai, Christophe Van Berckelaer, S. V. van Laere, Alexandra Bennion, Theresa Charity, Jinming Yang, F. Bertucci, Peter van Dam, Gregory M Palmer, Shannon McCall, Luc Y. Dirix, Naoto T. Ueno, G. Devi
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Abstract Objective We aim to investigate the spatiotemporal dynamics of intervertebral disc (IVD) cell subpopulations in IVD degeneration (IVDD). Methods To gain combined spatial and transcriptomic insights into IVDD, we employed both spatial transcriptomic sequencing (stRNA‐seq) and single nucleus RNA sequencing (snRNA‐seq) in a rat puncture‐induced IVDD model. The findings were verified in rat and human IVD by immunostaining and qRT‐PCR. Tamoxifen‐administered PdgfraCreERT2;R26tdTomato mice were adopted to track platelet‐derived growth factor receptor alpha (Pdgfra) positive cells. Results Puncture response regions were revealed on day 1 post‐puncture, for which oxidative stress emerged as a prominent pathway in a Stress Zone consisting of lipocalin‐2 (Lcn2)+ annulus fibrosus (AF) cells (AFC), which propagated and migrated into nucleus pulposus (NP), playing a key role in delivering injury signals and triggering pathological processes, including ferroptosis, fibrosis, and immune reactions. In the NP, Collagen 3‐high (Col3hi) NP cells (NPC) were another induced population demonstrating a fibrochondrocyte‐like phenotype and high epithelial–mesenchymal transition activation, an important pathway involved in tissue fibrosis. Crucially, lineage tracing experiments in PdgfraCreERT2;R26tdTomat mice revealed the significant migration and proliferation of Pdgfra+ AFCs from the AF into the NP following puncture. These findings provide direct evidence that both Pdgfra+ AFCs and Col3hi NP cells may contribute to NP fibrosis. Conclusion Puncture‐induced oxidative stress in a stress zone is the primary reaction playing an important role in initiating IVDD. Several puncture‐induced cell subpopulations were identified, including Lcn2+ AFC, Col3hi NPC, and Pdgfra+ AFC. Lcn2+ AFC plays a pivotal role in connecting oxidative stress with other pathological processes. Our results clarified the dual origin of Pdgfra+ cells, highlighting the contribution of AF‐derived cells to the NP during degeneration and emphasizing the complexity of cellular changes underlying NP fibrosis. Further investigation into the specific contributions of Pdgfra+ cells from different origins to fibrosis is warranted. Key points Puncture induced oxidative stress in a Stress Zone is the primary reaction in initiating IVDD. Puncture induced several IVD cell subpopulations, including Lcn2+ AFC, Col3hi NPC and Pdgfra+ AFC. Lcn2+ AFC plays a pivotal role in connecting oxidative stress with other pathological processes. Pdgfra+ cells in the NP derived from both Pdgfra+ AFC and Col3hi NPC, highlighting dual origin of NP fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811048891df0168757ff714b7c9e073ef14b40a7" target='_blank'>
              Single nuclear‐spatial transcriptomic sequencing reveals distinct puncture‐induced cell subpopulations in the intervertebral disc of a rat model
              </a>
            </td>
          <td>
            Guoyan Liang, Jing Tan, Chong Chen, Yuying Liu, Yongyu Ye, Xiaolin Pan, Qiujian Zheng, Yunbing Chang, Feng-Juan Lyu
          </td>
          <td>2025-06-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2411017a831bd32082b22e6f4221bf8c31f61827" target='_blank'>
              ETS-guided iPSC-endothelial models recapitulate malaria pathogenesis
              </a>
            </td>
          <td>
            François Korbmacher, Hannah Fleckenstein, Rory K. M. Long, Patryk Poliński, Livia Piatti, Borja López-Gutiérrez, Alina Batzilla, Dennis Crusius, Vikas Trivedi, Miki Ebisuya, Maria Bernabeu
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Background: Breast cancer (BC) stands as a prominent contributor to cancer-related fatalities on a global scale. Traditionally, the primary focus in assessing recurrence risk and guiding treatment decisions has revolved around scrutinizing the primary tumor’s histopathological characteristics and molecular compositions. However, recent findings suggest that exploring normal tissue adjacent to the tumor (NAT) could unveil valuable insights into the intricate characteristics of BC advancement and patient prognoses. Specifically, in our recent work, we discovered three phylogenetic tree groups of NAT and tumor tissues that were associated with distinct tumor and tumor microenvironment (TME) features using the whole genome sequencing data collected from 43 Hong Kong BC patients (Zhu et al. under review). Here, we aimed to further characterize the molecular heterogeneity of NAT by integrating methylation and gene expression profiling data in an expanded sample set, which may deepen our understanding of tumor evolution. Methods: Genome-wide DNA was extracted from fresh frozen paired tumor/NAT tissues in 188 patients diagnosed with breast cancer and treated in Hong Kong, China. DNA was then profiled using the Infinium Methylation 850K BeadChip array. Horvath clock was utilized to estimate epigenetic age, and MethylCIBERSORT was used to estimate cellular composition using the methylation data. For a subset of the patients (N= 76), we also analyzed RNA sequencing (RNA-Seq) data and used CIBERSORTx to estimate the proportions of immune cell subpopulations in NAT samples. Risk factor and clinical information was available for most patients. Results: The uniform manifold approximation and projection (UMAP) identified two distinct clusters of NAT samples (cluster 1, N= 139; cluster 2, N= 49) based on the top 10K most variable methylation probes. The two clusters were recapitulated by unsupervised clustering. The two sets of patients did not differ by chronological age at diagnosis, but when investigating epigenetic aging - as captured by Horvath clock – we found that cluster 2 displayed a lower age acceleration in NAT compared to cluster 1. Among the subset of patients with phylogenetic trees constructed in our previous work, we observed cluster 1 had a higher frequency of patients (64.3%) that showed distinct evolutionary trajectories in matched NAT and tumor samples (multiple-tree group) compared to cluster 2 (33.3%). Consistent with previous findings, we found that cluster 1, enriched with multiple-tree NAT samples, was associated with lower proportions of CD14 cell population (P=1.6x10-6), monocytes (P=7.1x10-6), and M2 macrophages (P=2.3x10-7) compared to cluster 2. Interestingly, genomic features in the matched tumors such as PAM50 subtype, TP53 mutation status, or tumor mutational burden, did not vary significantly between clusters 1 and 2. Lastly, established BC risk factors examined in this study did not show significant differences by this methylation-based cluster assignment. Conclusion: Using methylation data in an expanded dataset, we extended our previous analysis of paired tumor and NAT samples to characterize breast cancer field effect. Our main findings illuminate the diverse nature of the microenvironment surrounding the tumor and its potential influence on the genomic evolutionary path of the tumor.
 Citation Format: Hela Koka, Bin Zhu, Priscilla Lee, Kevin Wang, Avraam Tapinos, Difei Wang, Gary M. Tse, Koon-ho Tsang, Cherry Wu, Chad A. Highfill, Kristine Jones, Belynda Hicks, Amy Hutchinson, Montserrat Garcia-Closas, Stephen Chanock, David C. Wedge, Lap Ah Tse, Xiaohong R. Yang. Molecular heterogeneity in adjacent normal tissue among Chinese breast cancer patients [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f57f67b5c54214463999793ad38b86bf56cb8b35" target='_blank'>
              Abstract P2-02-06: Molecular heterogeneity in adjacent normal tissue among Chinese breast cancer patients
              </a>
            </td>
          <td>
            H. Koka, Bin Zhu, Priscilla Lee, Kevin Wang, Avraam Tapinos, Difei Wang, G. M. Tse, Koon-Ho Tsang, Cherry Wu, Chad A. Highfill, Kristine Jones, B. Hicks, A. Hutchinson, Montserrat García-Closas, Stephen Chanock, David C Wedge, L. Tse, Xiaohong R. Yang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Cancer progression and therapeutic resistance are propelled by the remarkable plasticity of signaling networks, which dynamically rewire under selective pressures to maintain proliferation, enable immune evasion and promote metastasis. Despite advances in precision oncology, the dynamic crosstalk between tumor cells, non-coding genomes and the microenvironment continues to undermine treatment efficacy. This call for submissions, Revolutionizing Cancer Treatment: Navigating the Intricate Landscape of Cellular Signaling Networks, seeks cutting-edge research that dissects these adaptive mechanisms through innovative technologies - from single-cell multi-omics and spatial transcriptomics to AI-powered network modeling. We welcome studies leveraging physiomimetic models (e.g., organoids, 3D-bioprinted ecosystems) to decode tumor heterogeneity, as well as translational work targeting emergent vulnerabilities at the intersection of epigenetics, metabolic reprogramming and stromal interactions. By integrating systems biology with computational and experimental approaches, this collection aims to catalyze the design of adaptive therapies that outmaneuver cancer's evolutionary resilience.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba13faeaf95516c4bb5cbd68aeea6df92e61f127" target='_blank'>
              Revolutionizing cancer treatment: Navigating the intricate landscape of cellular signaling networks.
              </a>
            </td>
          <td>
            Hao Zhang
          </td>
          <td>2025-05-26</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cisplatin is the primary chemotherapeutic agent for osteosarcoma. However, a significant proportion of patients develop resistance post-treatment, leading to disease recurrence and presenting profound clinical challenges. To understand the mechanisms underlying osteosarcoma recurrence and cisplatin resistance, particularly from the tumor microenvironment perspective, we consolidated numerous single-cell RNA sequencing datasets, offering an encompassing insight into the osteosarcoma microenvironment. When juxtaposing scRNA-seq with bulk RNA-seq data, we observed a strong correlation between high DCUN1D5 expression in osteosarcoma and patient survival. This gene amplifies osteosarcoma’s anti-apoptotic, invasive, stem-cell-like traits and PI3K/AKT/GSK3β pathway phosphorylation and fosters cisplatin resistance. Subsequent research revealed that cisplatin-resistant osteosarcoma cells excrete DCUN1D5-rich exosomes, facilitating the maturation of osteoclast precursors. Excessive osteoclast activity is a pivotal contributor to osteosarcoma recurrence and resistance. Given these insights, DCUN1D5 is a promising therapeutic target for osteosarcoma recurrence and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21215b3f5ca618360a5a84033d00f16e0330e332" target='_blank'>
              Single-cell analysis links DCUN1D5 to immune remodeling and cisplatin resistance in recurrent osteosarcoma
              </a>
            </td>
          <td>
            Xin Wu, Woding Deng, Qiangqiang Zhao, Jianbin Xiong
          </td>
          <td>2025-07-08</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic cancer is a highly malignant tumor of the digestive system with a dismal prognosis. Despite advances in diagnosis and treatment, overall survival remains extremely low. Early diagnostic markers and an improved understanding of tumor-microenvironment interactions are essential for developing more effective therapies.We analyzed 74 single-cell RNA sequencing (scRNA-seq) samples, performing unsupervised clustering and marker-gene expression profiling to define major cell types. Large-scale chromosomal copy-number variation (CNV) analysis distinguished malignant from non-malignant ductal cells. Non-negative matrix factorization (NMF) identified stage-associated gene modules, which were integrated with TCGA bulk-RNA data and machine-learning feature selection to pinpoint candidate prognostic genes. Two independent cohorts were used for validation. Regulatory network inference (pySCENIC) and ligand–receptor interaction analysis (CellPhoneDB) explored cross-talk between malignant cells and macrophages. Finally, in vitro knockdown of CTSV assessed its functional role in pancreatic cancer (PAC) cell proliferation and migration.Three prognosis-related genes—ANLN, NT5E, and CTSV—were selected based on their strong association with clinical stage and validated in external datasets. High expression of these genes correlated with poorer overall survival and an increased infiltration of M0 macrophages. CellPhoneDB predicted significant interactions between high-expression malignant ductal cells and M0 macrophages via CXCL14–CXCR4 and IL1RAP–PTPRF axes, with SPI1 identified as an upstream regulator of IL1RAP. In vitro CTSV knockdown significantly inhibited PAC cell proliferation and migration.Our integrative single-cell and bulk-RNA workflow identifies ANLN, NT5E, and CTSV as novel prognostic biomarkers in pancreatic cancer and highlights a pro-tumorigenic interaction between malignant ductal cells and macrophages. Targeting CTSV or disrupting CXCL14–CXCR4 and IL1RAP–PTPRF signaling may offer new therapeutic avenues for PAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50ce5676a35dfc57b1b078e8894ee25f419175d" target='_blank'>
              Integration of single-cell RNA and bulk RNA sequencing revealed malignant ductal cell heterogeneity and prognosis signatures in pancreatic cancer
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Recently, contrary to attacking cancer cells, the tumor microenvironment (TME) with genomic stability and vulnerable nature has emerged as promising therapeutic targets in hepatocellular carcinoma (HCC). Within TME ecosystem, tumor-associated macrophages (TAMs) play a pivotal role in tumor evasion and progression of HCC. However, their clinical and therapeutic implications remain unexplored.Utilizing a large-scale sc-RNA seq dataset, a landscape of HCC cellular ecosystem was depicted. Based on previous literature, an effectively differential TAMs subset classification was identified. Gene variations was extracted through trajectory analysis and then unsupervised clustering was conducted within RNA-seq data. Subsequently, survival analysis, specific pathway enrichment as well as hub regulatory network analysis were performed. Additionally, the immune cell infiltration and genomic variations were evaluated between clusters. Drug sensitivity and underlying therapeutic molecular were also explored. Through multiple immunofluorescence, our findings were verified.Herein, integrating single-cell RNA sequencing (scRNA-seq) and bulk RNA-seq data, we established a novel TAM classification system based on mutually exclusive SPP1 and FOLR2 signatures. According to the TAM trajectory genes, unsupervised clustering stratified HCC into three distinct clusters. Cluster 3 (C3), which is characterized by metabolic dysregulation and immunosuppressive TME, exhibited the poorest prognosis among the three groups. Hub network analysis of C3 further indicated its characteristic dysregulation of liver-specific metabolism. SPP1 was identified as a key signature of C3, which contributed to suppressing the infiltration of CD8+ T cells. Therapeutic evaluation revealed that C3 were sensitive to chemotherapy and tyrosine kinase inhibitors, while those C1 and C2 were more suitable for immunotherapy. Drug screening identified potential therapeutic compounds for each cluster.This study redefines the heterogeneity of TAMs beyond the M1/M2 paradigm, linking the TAMs trajectory genes to HCC patient stratification. SPP1 blockade emerged as a strategy for counteracting immunosuppression, and cluster-specific therapies may optimize the management of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d85b5a9a5d2466c00caeea207b03b8145b6f159" target='_blank'>
              SPP1+ tumor-associated macrophages define a high-risk subgroup and inform personalized therapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Wei-Xuan Xu, Yanjun Ye, Jia-Lin Chen, Xin-Ying Guo, Chen Li, Juan Luo, Lin-Bin Lu, Xiong Chen
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) presents significant clinical challenges owing to its dense stroma and complex tumor microenvironment (TME). In this study, large-scale single-cell transcriptomics and spatial transcriptomics (ST) were integrated to dissect the heterogeneity of fibroblasts and their crosstalk with epithelial cells, with a focus on key ligand-receptor interactions. Eight distinct fibroblast subpopulations were identified, among which extracellular matrix (ECM)-remodeling fibroblasts were particularly enriched in tumor tissues and associated with poor prognosis. ECM-remodeling fibroblasts were located at the terminal stage of the fibroblast pseudotime trajectory, and SOX11 was identified as a key transcription factor in this subpopulation. Further analyses revealed that ECM-remodeling fibroblasts can interact with epithelial cells through the POSTN-ITGAV/ITGB5 ligand-receptor axis, a critical pathway that promotes tumor progression. Clinical analyses demonstrated a strong correlation between POSTN expression and poor prognosis in patients with PDAC. Mechanistically, POSTN interacts with integrin ITGAV/ITGB5 on tumor cells, activating the PI3K/AKT/β-catenin pathway and promoting epithelial-mesenchymal transition (EMT) phenotype. Pharmacological inhibition of the POSTN-integrin axis partially reversed these malignant traits, highlighting its potential as a therapeutic target. This study provides new insights into fibroblast heterogeneity and its role in PDAC progression, emphasizing the POSTN-ITGAV/ITGB5 axis as a promising target for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e1d45940647abe0d0d6b914abc94702cee2e0ff" target='_blank'>
              Integrative Single-Cell and Spatial Transcriptomics Analysis Reveals ECM-remodeling Cancer-associated Fibroblast-Derived POSTN as a Key Mediator in Pancreatic Ductal Adenocarcinoma Progression
              </a>
            </td>
          <td>
            Yifan Wu, Shuquan Li, Hao Yu, Shan Zhang, Lianghua Yan, Xiao-ya Guan, Wei Xu, Zhen Wang, Ang Lv, Xiuyun Tian, Chunyi Hao, Jianhui Wu
          </td>
          <td>2025-05-27</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c42ccef135de9d53a5d5b1a6abe8bae988944a3" target='_blank'>
              Histo-LCM-Hi-C reveals the 3D chromatin conformation of spatially localized rare cells in tissues at high resolution
              </a>
            </td>
          <td>
            Yixin Liu, Min Chen, Xin Liu, Zeqian Xu, Xinhui Li, Yan Guo, D. Czajkowsky, Zhifeng Shao
          </td>
          <td>2025-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Embryonal tumor with multilayered rosettes (ETMR) is a pediatric brain tumor with dismal prognosis. Characteristic alterations of the chromosome 19 microRNA cluster (C19MC) are observed in most ETMR; however, the ramifications of C19MC activation and the complex cellular architecture of ETMR remain understudied. Here we analyze 11 ETMR samples from patients using single-cell transcriptomics and multiplexed spatial imaging. We reveal a spatially distinct cellular hierarchy that spans highly proliferative neural stem-like cells and more differentiated neuron-like cells. C19MC is predominantly expressed in stem-like cells and controls a transcriptional network governing stemness and lineage commitment, as resolved by genome-wide analysis of microRNA-mRNA binding. Systematic analysis of receptor–ligand interactions between malignant cell types reveals fibroblast growth factor receptor and Notch signaling as oncogenic pathways that can be successfully targeted in preclinical models and in one patient with ETMR. Our study provides fundamental insights into ETMR pathobiology and a powerful rationale for more effective targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02046d3dbdf78b8dbf1ae4df369d0858d2ab96f2" target='_blank'>
              Cellular hierarchies of embryonal tumors with multilayered rosettes are shaped by oncogenic microRNAs and receptor–ligand interactions
              </a>
            </td>
          <td>
            Alexander Beck, Lisa Gabler-Pamer, G. Alencastro Veiga Cruzeiro, S. Lambo, Bernhard Englinger, McKenzie L. Shaw, Olivia A Hack, Ilon Liu, Rebecca D. Haase, Carlos A O de Biagi, A. Baumgartner, Andrezza Do Nascimento Silva, M. Klenner, Pia Freidel, Jochen Herms, Louisa von Baumgarten, Joerg-Christian Tonn, N. Thon, K. Bruckner, S. Madlener, L. Mayr, D. Senfter, A. Peyrl, I. Slavc, D. Lötsch, C. Dorfer, Rene Geyregger, N. Amberg, C. Haberler, Norman L Mack, Benjamin Schwalm, Stefan M. Pfister, Andrey Korshunov, Lissa C Baird, Edward Yang, Susan N Chi, S. Alexandrescu, J. Gojo, Marcel Kool, Volker Hovestadt, Mariella G. Filbin
          </td>
          <td>2025-05-26</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a high-mortality cancer characterized by its aggressive, treatment-resistant phenotype and a complex tumour microenvironment (TME) featuring significant hypoxia. Bulk transcriptomic analysis has identified the “classical” and “basal-like” transcriptional subtypes which have prognostic value; however, it is not well-established how microenvironmental heterogeneity contributes to these transcriptional signatures. Here, we exploited the TRACER platform to perform single cell transcriptome analysis of organoids at specific spatial locations to explore the effect of oxygen and other cell-generated microenvironmental gradients on organoid heterogeneity. We found that the microenvironmental gradients present in TRACER significantly impact the distribution of organoid transcriptional phenotypes and the enrichment of gene sets linked to cancer progression and treatment resistance. More significantly, we found that microenvironmental gradients, predominantly oxygen, drive changes in the expression of classical and basal-like transcriptional subtype gene signatures. This work suggests that hypoxia contributes to determining transcriptional subtypes in PDAC tumour cells independent of additional cells in the TME and broadly highlights the importance of considering microenvironmental gradients such as oxygen in organoid-based studies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-98344-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d7b936c3248bb437b55ed176049e4ec9a8d43e" target='_blank'>
              Analysis of an engineered organoid model of pancreatic cancer identifies hypoxia as a contributing factor in determining transcriptional subtypes
              </a>
            </td>
          <td>
            Natalie Landon-Brace, Simon Latour, Brendan T. Innes, Michelle Nurse, Jose L. Cadavid, I. Co, Cassidy M Tan, Ferris Nowlan, Sibil Drissler, F. Notta, H. Jackson, Gary D Bader, A. McGuigan
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Type I collagen (Collagen I, Col I), a principal component of the tumor extracellular matrix (ECM) that accounts for 80% of total ECM collagen, has emerged as a crucial factor in tumor biology. Recent research has found that Col I has influenced tumor growth, invasion, migration, and prognosis by forming a physical barrier and interacting with components in the tumor microenvironment (TME). Pancreatic ductal adenocarcinoma (PDAC) is characterized by a prominent stromal microenvironment with remarkable cellular and spatial heterogeneity, which significantly impacts the biology of the disease and its resistance to treatment. As research on the influence of ECM on tumor behavior progresses, the role of Col I in tumor occurrence and development has also been gradually expounded. Unlike the pro-tumorigenic role caused by excessive deposition in other cancers, PDAC features a dense stroma with spatial heterogeneity, where Col I deposition exhibits paradoxical pro- and anti-tumor effects: Cancer-associated fibroblasts secrete heterotrimeric Col I (α1/α1/α2), which initially suppresses tumor growth by recruiting M1 macrophages, while tumor cells produce homotrimeric Col I (α1/α1/α1, 2–3% of total Col I) to promote immune evasion via CXCL5 secretion and CD8+ T cell reduction. Mechanistically, Col I activates TGF-β/SMAD3 signaling to drive epithelial–mesenchymal transition (EMT) and enhances matrix stiffness, increasing tumor cell migration by 60–80% in 3D cell culture models. Thus, understanding the complex interactions between Col I and other microenvironmental components may enhance its anti-tumor action while suppressing its pro-tumor activity, potentially improving PDAC patients’ survival. In this review, we discuss the current research progress regarding the function of Col I in PDAC tumors and summarize its contributions to PDAC progression, thus providing a basis for follow-up research and potential targets for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ceb222174824b91b91f72e2fc4e082f366f425a" target='_blank'>
              The type I collagen paradox in PDAC progression: microenvironmental protector turned tumor accomplice
              </a>
            </td>
          <td>
            Qian Li, Cheng Yang, Junqiang Li, Ronglin Wang, Jie Min, Yang Song, Haichuan Su
          </td>
          <td>2025-07-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 4155


 Background:
 The tumor microenvironment (TME) that surrounds pancreatic ductal adenocarcinoma (PDAC) is a multi-faceted and dynamic ecosystem in which stromal fibroblasts communicate with cancer cells to mediate tumor growth, metastasis, and chemotherapy resistance. It is well recognized that both cancer-associated fibroblasts (CAFs) and its non-cellular, fibrotic components within the TME can foster a protumorigenic environment for PDAC. However, we still lack a comprehensive understanding of the precise mechanisms in which this dense, fibrotic matrix can help drive malignant behaviors. Here, we reveal a novel mechanism in which the zinc-finger transcription factor GLI2 regulates type 1 collagen expression within CAFs and how the soluble variant of this collagen promotes irinotecan chemoresistance.
 Methods:
 We leveraged transcriptomic data from The Cancer Genome Atlas, International Cancer Genome Consortium, and Clinical Proteomic Tumor Analysis Consortium to evaluate GLI2 expression and stromal content of human PDAC tumors through bulk RNA-sequencing deconvolution. Using single-nucleus RNA sequencing of human PDAC tumors, we validated the association of GLI2 expression and stromal matrix constituents in CAFs. Chromatin immunoprecipitation assays in human CAFs confirmed GLI2 binding at the COL1A1 promoter. Through RNAi-based inactivation of GLI2, we determined how loss of GLI2 impacts regulation of type 1 collagen. Additionally, we conducted MTT assays to assess tumor viability in response to irinotecan treatment.
 Results:
 Transcriptomic analysis revealed that GLI2 is highly enriched in CAFs and strongly correlated with stromal fibrosis compared to other non-tumor cell constituents within the TME. We have shown that GLI2 directly binds to the promoter of COL1A1, a key component of type 1 collagen, in CAFs and regulates its transcription in a manner dependent on TGFβ1 signaling. Interestingly, PDAC tumors exposed to type 1 collagen show increased expression of pro-tumorigenic pathways involved in inflammation, EGR signaling, cytokine-receptor interactions, and irinotecan resistance. We further validate that human PDAC cells pre-treated with collagen can confer chemoresistance to irinotecan with viability assays.
 Conclusions:
 Taken together, our study demonstrates a novel mechanism in which GLI2 regulates the secretion of collagen within CAFs, which in turn can enable PDAC to acquire resistance to standard-of-care treatments. These findings highlight not only the oncogenic functions for CAFs and their fibrotic secretome, but also open new avenues in which therapeutic targeting of the TME may provide clinical benefit for PDAC patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3a2065caeb9cc2540ae0418d70044e4a839709" target='_blank'>
              Functional role of GLI2 in cancer-associated fibroblasts for modulation of the fibrotic tumor microenvironment within pancreatic cancer.
              </a>
            </td>
          <td>
            John Y. Kwon, Renzo E Vera, Maite G. Fernadez-Barrena, Jose M. Falero, Roberto G. Chaparro, Matthew D. Ross, M. D. Toruner, Murat Toruner, Ezequiel J Tolosa, Luciana L Almada, Huocong Huang, Rolf A. Brekken, Martin E Fernandez-Zapico
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7be43f99c39d72183ee247e245dd3934afb3150" target='_blank'>
              Decoding meningioma prognosis with multi-omics: macrophage diversity, immune-CNV interplay, and novel SPP1-targeted strategies.
              </a>
            </td>
          <td>
            Hailang Fan, Xiaojie Li, Yaqian Zhao, Lairong Song, Lanlan Sun, Zhen Wu, Junting Zhang, Haikun Zhang, Feifan Zhang, Yingdi Huang, Dake Zhang, Liang Wang
          </td>
          <td>2025-06-16</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) remains a leading cause of cancer-related mortality worldwide. Recent advancements have illuminated the intricate metabolic reprogramming that underpins NSCLC progression and resistance to therapy. Beyond the classical Warburg effect, emerging evidence highlights the pivotal roles of altered lipid metabolism, amino acid utilization, and the metabolic crosstalk within the tumor microenvironment (TME). This review delves into the latest discoveries in NSCLC metabolism, emphasizing novel pathways and mechanisms that contribute to tumor growth and survival. We critically assess the interplay between cancer cell metabolism and the TME, explore the impact of metabolic heterogeneity, and discuss how metabolic adaptations confer therapeutic resistance. By integrating insights from cutting-edge technologies such as single-cell metabolomics and spatial metabolomics, we identify potential metabolic vulnerabilities in NSCLC. Finally, we propose innovative therapeutic strategies that target these metabolic dependencies, including combination approaches that enhance the efficacy of existing treatments and pave the way for personalized metabolic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dca10fc3d1d70787bfbf9b5297005a8e7d8e3d0" target='_blank'>
              Metabolic reprogramming and therapeutic targeting in non-small cell lung cancer: emerging insights beyond the Warburg effect
              </a>
            </td>
          <td>
            Hong Cai, Feng Zhang, Fang Xu, Chunhui Yang
          </td>
          <td>2025-05-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly invasive and heterogeneous malignancy, posing challenges for reproducible modeling and functional phenotypic analysis. To address these limitations, we developed a standardized 3D patternoid platform using collagen-based microcavity arrays to enhance organoid formation consistency and quantify subtype-specific invasion mechanisms. We utilized murine primary PDAC cells stratified by epithelial–mesenchymal transition (EMT) into three subtypes: epithelial (E-9591), hybrid EMT (Mlow-8028), and mesenchymal (M-16992). The platform's sensitivity was verified by a strong correlation between EMT scores and invasive phenotypes, as well as responses to physiological concentrations of the protease inhibitor batimastat. Key invasion parameters—including invasive area, maximum invasion distance, and branching complexity—were measured under both genomic and drug-induced conditions. The platform demonstrated high inter-organoid reproducibility, with precise control over initial cell numbers ensuring batch-to-batch comparability. Invasion dynamics analysis revealed that epithelial cells (E-9591) primarily relied on spatial constraints within the microcavity to invade. Batimastat drug sensitivity assays further distinguished invasion dependencies of the mesenchymal subtypes, confirming that M-16992 patternoids exhibit a stronger sensitivity towards MMP inhibition compared to Mlow-8028 patternoids. Concurrentlty, both subtypes experienced a shift towards epithelial-like spatial constraint triggered invasion morphology, reflecting the plasticity of PDAC invasiveness. This scalable and adaptable 3D patternoid platform enables high-throughput analysis of invasive behaviors and therapeutic responses, offering significant potential for preclinical cancer research and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97101c783ed99f665df3978439f230398b92fc54" target='_blank'>
              A 3D patternoid model for the reproducible characterization of invasive phenotypes and drug sensitivity in PDAC
              </a>
            </td>
          <td>
            Sophie Christin Kurzbach, Violetta Carvajal-Heckele, T. Teshima, Maximilian Reichert, Andreas R. Bausch
          </td>
          <td>2025-05-26</td>
          <td>Lab on a Chip</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Tumor metabolism reprogramming is a hallmark of cancer, but metabolite-mediated intercellular communication remains poorly understood. To address this gap, we estimated and explored communication events exploring based on single‐cell RNA data, to explore the metabolic landscape of tumor microenvironment (TME) in lung adenocarcinoma (LUAD) and identify novel metabolite signaling axis. Methods The scRNA-seq dataset was subjected to dimensionality reduction using the Seurat package. Cell annotation was manually performed using typical markers from Cell Marker 2.0 and previous studies. Single‐cell metabolite abundance and communication events were inferred using MEBOCOST. The TCGA‐LUAD datasets was used to estimate and analyze immune cell infiltration levels and tumor hot score using the ESTIMATE and ssGSEA algorithms. Additionally, survival analysis was conducted on genes within relative signaling axis. All analysis above in TCGA‐LUAD dataset was validated by two Gene Expression Omnibus (GEO) datasets. The expression patterns of PTGDR and PTGDS were validated by RT‐qPCR and fluorescence in situ hybridisation. Results Five landmark metabolites across cell types were identified as prostaglandin D2 (PGD2), D-Mannose, Choline, L-Cysteine, and Cholesterol of TME in LUAD. Prostaglandin D2 (PGD2) emerged as a key player, primarily produced by fibroblasts and plasmacytoid dendritic cells (pDCs) by via the PTGDS gene and by mast cells via the HPGDS gene. PGD2 signaling was shown to primarily be received by the PGD2 receptor (PTGDR) on NK/T cells and transported by the SLCO2A1 transporter on endothelial cells. CX3CR1+ NK/T cells, which are prominent cytotoxic populations, as a PGD2 autocrine signaling axis, are involved in PGD2 autocrine signaling, while KLRC2+ NK, DNAJB1+ NK cells and CD8+ MAIT cells participate in PGD2 paracrine signaling. PGD2 may also assist lactate efflux via SLCO2A1 on endothelial cells. The clinical relevance of the PGD2 signaling axis was validated across multiple bulk RNA datasets, showing that it is associated with the infiltration of above immune cells such as DNAJB1+ NK cells, and linked to better prognosis in LUAD. Furthermore, we found that a risk model developed based on this signaling axis could predict responses to immune therapy in hot and cold tumors, suggesting potential drugs that may benefit low-risk patients. These findings were further supported by RT-qPCR and immunofluorescence data, which confirmed the downregulation of PTGDS and PTGDR in LUAD tumor tissues compared to normal tissues. Conclusion Collectively, these results suggest that PGD2 and its signaling axis play a significant role in tumor-suppressive and anti‐inflammatory effects in LUAD, with potential applications in prognosis management and therapy decision‐making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e73688d477cc5c7d43fdfe6c032ab583d5082b6" target='_blank'>
              Biological and prognostic insights into the prostaglandin D2 signaling axis in lung adenocarcinoma
              </a>
            </td>
          <td>
            Qiang Liu, Huiguo Chen, Dongfang Tang, Huibiao Zhang, Shaogeng Chen, Yiran Meng, Boying Zheng, Feifei Liu, Jing Zhou, Wen Zhang
          </td>
          <td>2025-05-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Ewing sarcoma (ES) is a rare and aggressive pediatric bone malignancy with poor prognosis, driven by therapy-resistant tumor microenvironments (TME). The TME plays a critical role in tumor progression through a complex and dynamic network of reciprocal interactions among immune cells (dysfunctional T cells, immunosuppressive macrophages), stromal components (cancer-associated fibroblasts), and tumor cells. These interactions collectively shape the immune landscape, promote immune evasion, and contribute to therapeutic resistance. Identifying reliable prognostic markers remains a critical challenge. Methods Here we performed an integrated single-cell RNA sequencing, WGCNA, and bulk RNA-seq analyses to investigate tumor-immune interactions. Differentially expressed genes (DEGs) intersected with T cell markers identified a total of 174 T cell-associated genes. Functional enrichment analysis and molecular subtyping were performed to explore immune-related pathways. A prognostic model based on CLEC11A, BDP1, and ID3 was constructed using Cox regression and validated in external datasets. Immune infiltration was assessed using the CIBERSORT algorithm. Results T cell marker analyses revealed key roles in pathways such as PI3K-Akt signaling and immune modulation. Molecular subtyping identified two clusters with distinct immune microenvironments: Cluster C1 (immunosuppressive phenotype and poorer prognosis) and Cluster C2 (functionally active immune profile associated with better prognosis). The prognostic model demonstrated high predictive accuracy for 1-, 3-, and 5-year survival (AUC: 0.85, 0.82, 0.78). Additionally, a higher tumor mutation burden (TMB) with low survival rate has been observed in High-risk group. Immune infiltration analysis showed higher CD8+ T cell and dendritic cell activity and immune checkpoint expression in low-risk groups. Experimental validation demonstrated that ID3 silencing inhibited tumor cell proliferation and induced cell cycle arrest in ES cell lines. Conclusion Together, our study identified CLEC11A, BDP1, and ID3 as key T cell associated prognostic markers and developed a validated model to predict survival outcomes in ES. Insights into T cell markers and tumor-immune dynamics offer promising advances in prognostic assessment and immunotherapy for ES. Furthermore, the role of ID3 in immune evasion and tumor proliferation underscores its potential as a therapeutic target, providing new avenues for immune checkpoint regulation and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61b182bc7d5b374e78a6d8b617b48c4b427c14cb" target='_blank'>
              Integrative analysis of T cell-associated markers in Ewing sarcoma reveals prognostic signatures and immune dynamics
              </a>
            </td>
          <td>
            U. Saddozai, Chenxu Liu, Fei Yan, Zhendong Lu, M. B. Khawar, Muhammad Usman Akbar, Saadullah Khattak, Haibo Sun, Ping Yang
          </td>
          <td>2025-06-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Gastric cancer (GC) is among the most lethal human malignancies with limited treatment options. Cell-cell interactions within the tumor microenvironment (TME) can promote tumor growth, yet their therapeutic value has not been fully explored. Here, bulk RNA-seq, single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics (ST) were integrated to analyze the heterogeneity of GC microenvironment. Tumor-specific GREM1+ fibroblasts and SPP1+ macrophages were significantly enriched in GC tissues and were involved in immunosuppression, inflammation regulation, and tumor progression. We then indicated that GREM1+ fibroblasts and SPP1+ macrophages were positively correlated in 12 independent GC datasets and validated their close localization by multiplex immunohistochemistry staining and spatial transcriptomics. Patients with both high GREM1+ fibroblasts and SPP1+ macrophages exhibited significantly shorter OS and showed enrichment of tumor-associated pathways. Our results demonstrated the complex interactions between GREM1+ fibroblasts and SPP1+ macrophages, which may serve as a potential therapeutic target for future treatment of GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad0abacc337130966d37d3605e0104e32183302" target='_blank'>
              Multi-omics analyses reveal interactions between GREM1+ fibroblasts and SPP1+ macrophages in gastric cancer
              </a>
            </td>
          <td>
            L. Qiu, Xiao Zhao, Sheng Yao, Yang Fei, Y. Gong, Zishan Zhou, Shunchang Jiao, Jianming Xu
          </td>
          <td>2025-06-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 e20051


 Background:
 Histone lactylation, a newly identified epigenetic modification, has been extensively implicated in the initiation and progression of tumors. However, the precise role of histone lactylation in modulating tumor cell responses to T cell-mediated cytotoxicity remains insufficiently explored.
 Methods:
 To elucidate the adaptive changes tumor cells undergo in response to T cell-induced cytotoxicity within the tumor microenvironment (TME), we performed a comprehensive, genome-wide CRISPR-Cas9 screen. This approach facilitated the identification of genetic alterations in lung cancer cells co-cultured with cytotoxic T lymphocytes (CTLs) that promote their evasion of T cell-mediated destruction. Complementary RNA sequencing (RNA-seq) and chromatin immunoprecipitation sequencing (ChIP-seq) analyses were employed to investigate the molecular mechanisms by which tumor cells modulate their response to T cell-mediated killing, particularly following the deletion of the monocarboxylate transporter 1 (MCT1).
 Results:
 MCT1 deletion significantly heightened the cancer-cell susceptibility to CTL-mediated killing. Furthermore, the loss of MCT1 led to mitochondrial dysfunction within the tumor cells. Mechanistically, MCT1 knockout resulted in a reduction of histone H3K18 lactylation and downregulation of ferridoxin reductase (FDXR), which disrupted mitochondrial homeostasis and precipitated mitochondrial dysfunction. Notably, the combination of MCT1 inhibition with immune checkpoint blockade (ICB) therapy substantially enhanced ICB efficacy, prolonged survival in both lung cancer and malignant pleural effusion (MPE) murine models, and alleviated the immunosuppressive milieu within the TME.
 Conclusions:
 Our findings reveal a novel role for lactate metabolism in modulating mitochondrial dysfunction and anti-tumor immunity, with lactylation-mediated regulation of mitochondrial homeostasis serving as a critical mechanism. These results provide new insights into the epigenetic regulation of tumorigenesis and highlight potential immune-metabolic strategies for targeted lactate metabolism therapies in the treatment of lung cancer and MPE.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7369949ad290ca8642cb013b1fc65d9e0567a5" target='_blank'>
              MCT1-mediated histone lactylation modulates lung cancer cell sensitivity to T-cell–mediated killing by influencing mitochondrial dysfunction.
              </a>
            </td>
          <td>
            Q. Xue, W. Peng, Qiong Zhou
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Many studies have shown that pancreatic cancer is one of the cancers with extremely high mortality. The poor prognosis and lack of early diagnostic methods remain major challenges in the treatment of this cancer. In this article, in order to better detect the occurrence of pancreatic cancer, tools such as monocle3, singleR, harmony in R, and scanpy in Python were used to analyze the cells and genes of pancreatic cancer tissues in mice. By analyzing the data, the following results were obtained: T-cells in healthy PBMCs exhibited broader spatial dispersion than in PDAC tissues, suggesting tumor-driven immune surveillance impairment, while PDAC-associated macrophages displayed expanded distribution linked to pro-tumorigenic functions such as COL1A1-mediated ECM remodeling; Pseudotemporal trajectory analysis revealed myeloid progenitor bifurcation into monocytes/macrophages, with PDAC macrophages showing epigenetically silenced cytotoxic pathways such as suppressed GZMA/NKG7 and enhanced ribosomal biogenesis; Tissue-specific markers such as LCN2 in healthy and CTRB1/AMY2A in PDAC) and spatial co-localization of macrophages/tumor cells highlighted NOP53 as a dual-function hubinhibiting PI3K-AKT while activating p53and SPP1 as a paradoxical regulator of metastasis and antitumor immunity; Differential expression and GO enrichment analyses identified ribosomal biogenesis and cytoplasmic translation as PDAC-enriched pathways, contrasting with suppressed stress responses. Our spatial transcriptomic profiling further resolved elevated NOP53, CFB, and SPP1 expression gradients in PDAC tissues, proposing these as diagnostic biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab94aa8cc2272f4a29c06caf0dcfe1338e28a0a0" target='_blank'>
              Spatiotemporal Transcriptomic Dissection of Tumor-Associated Macrophage Heterogeneity and Dual-Function Molecular Nodes in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Mengfei Li
          </td>
          <td>2025-06-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79b9ccad6349bc919fa38a076a3195bd6f23bab" target='_blank'>
              Loss of colonic fidelity enables multilineage plasticity and metastasis.
              </a>
            </td>
          <td>
            Patrizia Cammareri, Michela Raponi, Yourae Hong, C. Billard, Nat Peckett, Yujia Zhu, Fausto D Velez-Bravo, Nicholas T Younger, Donnchadh S Dunican, S. Ö. Pohl, Aslıhan Bastem Akan, Nora J Doleschall, John Falconer, M. White, Jean Quinn, K. Pennel, Roberta Garau, Sudhir B. Malla, P. Dunne, Richard R Meehan, Owen J. Sansom, J. Edwards, Malcolm G Dunlop, Farhat V N Din, Sabine Tejpar, Colin W. Steele, K. B. Myant
          </td>
          <td>2025-06-04</td>
          <td>Nature</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) is a leading cause of cancer-related deaths worldwide. Despite advancements in treatment, prognosis for patients with advanced stages remains poor. Metabolic reprogramming in the tumor microenvironment, particularly abnormal glycolysis, plays a crucial role in immune evasion and treatment response. We collected nine single-cell datasets to create a single-cell atlas of CD8 + T cells from 89 NSCLC patients, revealing ten distinct states of these cells. We employed a multimodal data analysis approach, integrating bulk transcriptomics, single-cell transcriptomics, spatial transcriptomics, and proteomics. Using 117 machine learning models, we identified key genes associated with NSCLC metastasis. Notably, the StepCox[forward] + Lasso model was instrumental in pinpointing key genes that significantly impact disease prognosis. Our analysis revealed that LTB + LDHA + CD8 + T cells have a distinct metabolic and immune phenotype, characterized by enhanced glycolysis and elevated lactate production. This not only facilitates tumor cell migration and invasion but also impairs the cytotoxic function of CD8 + T cells. Furthermore, our machine learning models identified four key genes significantly associated with NSCLC metastasis: TBCD, PTPRC, LDHA, and ACTR2. Of these, high LDHA expression was strongly linked to poorer responses to immunotherapy and a higher risk of therapy resistance. LTB + LDHA + CD8 + T cells also reduced antitumor immune responses by inhibiting the secretion of effector molecules like GNLY. Additionally, elevated LDHA expression was associated with reduced CD8 + T cell infiltration, which further promotes tumor immune evasion. This study highlights the heterogeneity of CD8 + T cells in NSCLC, emphasizing the unique role of the LTB + CD8 + Tn subpopulation in metastasis. LDHA is identified as a critical key gene with a significant impact on immunotherapy outcomes, presenting a potential therapeutic target. These insights offer new biomarkers and targeted strategies for personalized immune therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e078cc67ce9052ada65f40295ba6296864dfb9f6" target='_blank'>
              Metabolic alterations driven by LDHA in CD8 + T cells promote immune evasion and therapy resistance in NSCLC
              </a>
            </td>
          <td>
            Hao Wen, Panpan Zhang, Juan Zhao, Yakui Liu, Lei Wan, Haoran Li, Jun Yi, Xinqiang Li
          </td>
          <td>2025-07-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="HOX (homeobox) genes are virtually absent in healthy adult brains but are detected in malignant brain tumors, particularly gliomas. In 2021, the World Health Organization (WHO) classified adult-type diffuse gliomas into three distinct categories: astrocytomas (isocitrate dehydrogenase [IDH]-mutated), oligodendrogliomas (IDH-mutated and 1p/19q-deleted), and glioblastomas, IDH-wildtype (GBM). GBM is the most common and aggressive primary malignant tumor of the Central Nervous System (CNS), characterized by its high recurrence rate and rapid growth. Dysregulation of HOX genes is a well-established phenomenon in both solid and liquid malignancies, playing crucial roles in various fundamental characteristics of cancer, including GBM. In recent years, HOX genes have gained recognition not only as key regulators of tumor progression but also as potential biomarkers for predicting disease outcomes and as promising therapeutic targets for GBM. This review compiles the latest research on HOX genes in GBM, encompassing studies published before and after the 2021 WHO classification of CNS tumors. Our goal is to provide a comprehensive overview of key findings on the role of HOX gene clusters, which are groups of genes involved in regulating the development of the body plan along the anterior–posterior axis, in GBM initiation, progression, prognosis, and treatment response. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02684-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e4a53ead8582c3ade2d72aeb49d5ea960a31a96" target='_blank'>
              HOX gene dysregulation in glioblastoma: a narrative review of current advances
              </a>
            </td>
          <td>
            Suleyman Hekim, S. Bunda, Shaliman Dilibaerguli, Maierdan Palihati, Yasin Mamatjan
          </td>
          <td>2025-05-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Artificial intelligence has revolutionized computational biology. Recent developments in omics technologies, including single-cell RNA sequencing and spatial transcriptomics, provide detailed genomic data alongside tissue histology. However, current computational models focus on either omics or image analysis, lacking their integration. To address this, we developed OmiCLIP, a visual–omics foundation model linking hematoxylin and eosin images and transcriptomics using tissue patches from Visium data. We transformed transcriptomic data into ‘sentences’ by concatenating top-expressed gene symbols from each patch. We curated a dataset of 2.2 million paired tissue images and transcriptomic data across 32 organs to train OmiCLIP integrating histology and transcriptomics. Building on OmiCLIP, our Loki platform offers five key functions: tissue alignment, annotation via bulk RNA sequencing or marker genes, cell-type decomposition, image–transcriptomics retrieval and spatial transcriptomics gene expression prediction from hematoxylin and eosin-stained images. Compared with 22 state-of-the-art models on 5 simulations, and 19 public and 4 in-house experimental datasets, Loki demonstrated consistent accuracy and robustness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31363f03f452a7f272a10fe8c178ee6dfbae576" target='_blank'>
              A visual–omics foundation model to bridge histopathology with spatial transcriptomics
              </a>
            </td>
          <td>
            Weiqing Chen, Pengzhi Zhang, Tu Tran, Yiwei Xiao, Shengyu Li, Vrutant V. Shah, Hao Cheng, Kristopher W Brannan, Keith A Youker, Li Lai, Longhou Fang, Yu Yang, N. Le, Jun-ichi Abe, Shu-hsia Chen, Qin Ma, Ken Chen, Qianqian Song, John Cooke, Guangyu Wang
          </td>
          <td>2025-05-29</td>
          <td>Nature Methods</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbcccd092da4df387dc9187eeb5acec5c01b0b6" target='_blank'>
              Loss of chromosome Y shapes the immune cell fate with aging in men
              </a>
            </td>
          <td>
            A. A. Z. Dawoud, L. Green, O. Rackham
          </td>
          <td>2025-06-02</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a lethal brain cancer with no effective treatment; understanding how GBM cells respond to tumor microenvironment remains challenging as conventional cell cultures lack proper cytoarchitecture while in vivo animal models present complexity all at once. Developing a culture system to bridge the gap is thus crucial. Here, a multicellular approach is employed using human glia and vascular cells to optimize a 3D brain vascular niche model that enabled not only long-term culture of patient derived GBM cells but also recapitulation of key features of GBM heterogeneity, in particular invasion behavior and vascular association. Comparative transcriptomics of identical patient derived GBM cells in 3D and in vivo xenotransplants models revealed that glia-vascular contact induced genes concerning neural/glia development, synaptic regulation, as well as immune suppression. This gene signature displayed region specific enrichment in the leading edge and microvascular proliferation zones in human GBM and predicted poor prognosis. Gene variance analysis also uncovered histone demethylation and xylosyltransferase activity as main themes for gene adaption of GBM cells in vivo. Furthermore, the 3D model also demonstrated the capacity to provide a quiescence and a protective niche against chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58339daec87fc88b9de1cfb8daf1fd79e18b6ecc" target='_blank'>
              3D Brain Vascular Niche Model Captures Glioblastoma Infiltration, Dormancy, and Gene Signatures.
              </a>
            </td>
          <td>
            Vivian K. Lee, R. Tejero, Nathaniel Silvia, Anirudh Sattiraju, Aarthi Ramakrishnan, Li Shen, Alexandre Wojcinski, Santosh Kesari, R. Friedel, Hongyan Zou, Guohao Dai
          </td>
          <td>2025-06-19</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 e14562


 Background:
 T cell-based immunotherapies have demonstrated great success for the treatment of many solid tumors, but not all tumor types and patients respond well or durably. Reasons for these discrepancies are unclear. High grade serous ovarian cancer (HGSOC) is an example of a deadly disease that has exhibited poor responses to T cell targeted therapies, like checkpoint inhibitors. Thus, it would be an ideal model for studying other parts of the immune ecosystem that could contribute to anti-tumor immunity. Tumor infiltrating B cells (TIL-Bs) are being increasingly studied in HGSOC because of their association with improved prognosis, especially when differentiated into germinal center (GC) B cells. However, the true effector functions of B cells, including antigen presentation, cytokine production, and antibody production, in ovarian and other solid tumors remains understudied. Here, we study tumor antigen-specific B cell responses in a mouse model of HGSOC.
 Methods:
 We have engineered a murine HGSOC cell line ID8 p53 -/- brca2 -/- (ID8) to express a membrane bound mode antigen, hen egg lysozyme (mHEL), at two different affinities to primary HEL-specific B cells (MD4 B cells) from MD4 transgenic mice. We use flow cytometry to characterize HEL-specific B cell responses to ID8 mHEL-expressing cells
 in vitro
 . We use a syngeneic mouse model of HGSOC in which we intraperitoneally inject ID8 cells (with or without mHEL) and use flow cytometry to identify immune phenotypes of B and T cells in tumor tissue, spleen, lymph nodes, and ascites. We also use ELISA to measure HEL-specific antibody production in the ascites and serum of these mice.
 Results:
 We demonstrate that MD4 B cells can upregulate canonical B cell receptor signaling elements in response to stimulation with ID8 mHEL
 in vitro
 , and that these responses can be titrated based upon the affinity of the mHEL presented. We also show that the syngeneic ID8 p53 -/- brca2 -/- mouse model produces tumors in the ovaries and peritoneum that are enriched for B cells, and that the ascites is specifically enriched for antigen specific B2 B cells (with a much smaller innate like B1 compartment as compared to the peritoneal lavage of non-tumor controls). We plan to inject host mice with ID8 mHEL tumors and compare tumor progression in mice that subsequently receive intraperitoneal adoptive transfer of MD4 HEL-specific B cells versus not. We also plan to compare the phenotypes of B and T cells in the tumor, spleen, lymph nodes, and ascites of these mice, as well as the serum and ascites levels of HEL antibodies.
 Conclusions:
 We have generated a model antigen expressing ovarian cancer cell line that produces antigen-specific B cell responses
 in vitro
 , and have shown that there is an enrichment in antigen-specific B cells in these ovarian tumors and ascites; future
 in vivo
 studies will elucidate the important effector functions of TIL-Bs in ovarian cancer and other solid tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/107a1451e84aeccb51bc914d82c8c35407885897" target='_blank'>
              Dissecting B-cell responses in high grade serous ovarian cancer.
              </a>
            </td>
          <td>
            Ramya Parameswaran, James Mueller, Katherine C Fuh, J. Zikherman
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 e14585


 Background:
 Immune checkpoint blockade (ICB) targeting PD-1/PD-L1 is crucial in treating breast cancer. Despite clinical success, the mechanisms by which ICB therapies reshape the tumor microenvironment (TME) and enhance anti-tumor activity remain unclear. Recent studies reveal that anti-PD-1 therapies broadly alter TME cells beyond PD-1+ T cells, highlighting extensive cell-cell communication (CCC) as a key factor. Understanding CCC changes in response to anti-PD-1 therapy can illuminate its mechanisms of action (MOA) and TME dynamics. This study applies causal inference methodology to investigate CCC between T cells and non-T cells in the TME of breast cancer patients receiving anti-PD-1 therapy.
 Methods:
 We analyzed single-cell RNA-seq data from 31 breast cancer patients pre- and on-treatment with anti-PD-1 (pembrolizumab) reported by Bassez et al., 2021. We identified differentially expressed genes (DEGs) across cell types. We applied the instrumental variable method to identify causal relationships between signals in T-cell and non-T-cell DEGs to uncover treatment-induced CCC. We further searched for ligand-receptor pairs mediating CCC and identified gene expression modules (GEMs) regulated by these ligand-receptors. Finally, we constructed a CCC network from T to non-T cells.
 Results:
 Anti-PD1 treatment induced broad gene expression changes in diverse cell populations. Major T cell pathways influenced by anti-PD-1 therapy include NF-κB, interferon-γ and interleukins. CD4
 +
 and CD8
 +
 exhausted cells, expressing high levels of
 PDCD1
 (encoding PD-1), exhibited distinct activated pathways. We identified the CCC network from T cells to non-T cells via ligand-receptor interactions. For example, anti-PD1 treatment activated cellular stress, apoptosis, and pro-inflammatory cytokine signaling pathways in T cells, altering the expression of a GEM that included
 TSC22D3
 and
 TXNIP
 . This initiated signaling to myeloid cells via
 RPS19
 -
 C5AR1
 , leading to NF-κB activation. CD4+ exhausted T cells primarily signaled to monocytes, enhancing helper functions to recruit and activate monocytes, thereby promoting immune regulation and amplification. In contrast, CD8+ exhausted T cells primarily interacted with macrophages, intensifying cytotoxic responses that facilitated effective tumor-cell killing.
 Conclusions:
 Our analyses provide insights into the dynamic interplay of cells within the TMEs during anti-PD-1 therapy. These findings could facilitate the identification of new biomarkers for predicting heterogeneous treatment responses to anti-PD-1 regimens, potentially enhancing the design and customization of immunotherapeutic strategies for breast cancer patients. Finally, this study demonstrated the utility of causal inference methodologies for mechanistic studies of CCC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74715a58ffb2aed0b55ed5b7dec461d0fe94b624" target='_blank'>
              Use of causal modeling to uncover cell-cell communication dynamics in the tumor microenvironment during anti-PD-1 therapy in breast cancer patients.
              </a>
            </td>
          <td>
            Aodong Qiu, Han Zhang, Joseph Ramsey, Boyang Sun, Bryan Andrews, Kun Zhang, Gregory F. Cooper, Lujia Chen, Xinghua Lu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Cervical cancer (CC) remains a major malignancy threatening women’s health, with high-grade squamous intraepithelial lesions playing a critical role in the progression toward CC. Exploring the molecular characteristics of epithelial cells (EPCs) as high-stage intraepithelial neoplasia evolves into CC is essential for the development of effective targeted drugs for cervical cancer. Single-cell RNA sequencing technology can fully understand the immune response at each molecular level, providing new ideas and directions for the precise treatment of CC. Methods Single-cell RNA sequencing was employed to comprehensively map EPCs characteristics. The differentiation trajectory of EPCs was inferred using Slingshot, while enrichment analysis highlighted the biological functions of EPCs. Cellchat visualized cell-cell interactions, and SCENIC was used to infer transcription factor regulatory networks in EPCs. CCK-8, colony formation, and EDU experiments were used to verify cell proliferation changes. Scratch assays and transwell assays were used to verify cell migration and invasion. Results A distinct EPCs subpopulation with high TOP2A expression was identified, predominantly originating from tumor tissues. This subpopulation exhibited disrupted mitosis and cell cycle regulation, along with features of high proliferation, high energy metabolism, and matrix plasticity. It played a key role in shaping the tumor microenvironment via the LAMC1-(ITGA3-ITGB1) signaling pathway. FOXM1, a key transcription factor in this cell subpopulation, significantly inhibited the proliferation and invasion of cervical cancer cells. Conclusion Through in-depth analysis of EPCs, this study provides promising insights and potential therapeutic targets for precision targeted treatment strategies for CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dee7774b4a74cbb3d72e5ad05823a4742cd929c3" target='_blank'>
              Epithelial cells with high TOP2A expression promote cervical cancer progression by regulating the transcription factor FOXM1
              </a>
            </td>
          <td>
            Weisha Sun, Lu Chen, Xiaolin Feng
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Atherosclerosis involves complex interactions between lipids, immune cells, vascular smooth muscle cells, and fibroblasts within the arterial wall. While significant advances in single-cell technologies have shed light on the roles of immune cells and vascular smooth muscle cells in plaque development, fibroblasts remain underexplored, leaving critical gaps in understanding their contributions to disease progression and plaque stability. Comprehensive characterization of fibroblast phenotypes in atherosclerosis is essential to unravel their diverse functions and to distinguish between subsets that may play protective versus pathogenic roles in the disease process.


METHODS
Here, we utilized cellular indexing of transcriptomes and epitopes by sequencing to comprehensively profile fibroblast diversity in a mouse model of atherosclerosis. Mice were fed an atherogenic diet for 0, 8, 19, and 26 weeks, representing distinct stages of disease progression, enabling a detailed phenotypic characterization of fibroblasts throughout the course of atherosclerosis development.


RESULTS
We identified 4 distinct fibroblast subpopulations, including a myofibroblast population closely resembling vascular smooth muscle cell-derived chondromyocytes. The proportions of these fibroblast subsets exhibited a modest decline as atherosclerosis progressed. Through multimodal analysis, we identified CD26 as a highly expressed and specific marker for one of these fibroblast subpopulations, distinguishing it from other subsets. Using a combination of flow cytometry and immunohistochemistry, we demonstrated that CD26+ fibroblasts predominantly reside in the adventitia of healthy arteries. During atherosclerosis progression, these cells expand into the intima and primarily localize within the fibrous cap of the lesion.


CONCLUSIONS
Our multiomic analysis highlights the phenotypic diversity and dynamic changes of fibroblasts during atherosclerosis progression. Among these, CD26+ fibroblasts emerge as a distinct subpopulation that expands within atherosclerotic lesions and may play a critical role in promoting plaque stability through their migration into the fibrous cap.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af796b09d1dc7b9ff103a4aea099fa62c9a6120" target='_blank'>
              Single-Cell Multimodal Profiling Reveals a Novel CD26+ Fibroblast Subpopulation in Atherosclerosis.
              </a>
            </td>
          <td>
            A. Bashore, Johana Coronel, Chenyi Xue, Lucie Y. Zhu, Muredach P. Reilly
          </td>
          <td>2025-06-12</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abrogating the immunoevasive role of the tumor immune microenvironment (TIME) represents a critical yet still elusive challenge in cancer treatment. Progress in this area has been hampered by both technological limitations and incomplete understanding of TIME-dependent immunoevasion mechanisms. We hypothesize that the immune-evasive role of TIME subpopulations—including regulatory T cells, cancer-associated fibroblasts, and tumor-associated macrophages—is critically mediated by hyperconnected Master Regulator Checkpoint (MRC) modules whose aberrant activity, as induced by paracrine signals, can be abrogated or modulated either genetically or pharmacologically. MRCs are primarily composed of transcription and co-transcription factors, acting downstream of surface receptors and signal transduction cascades to control the transcriptional identity and, ultimately, the phenotype of individual TIME subpopulations. Pharmacological inhibition of subpopulation-specific MRC proteins can thus help reprogram the TIME and potentially abrogate or modulate its immunosuppressive state. This paradigm shift, away from single ligand/receptor targeting, is supported by recent algorithmic, experimental, and clinical advances allowing systematic identification of MRCs and their pharmacological modulators using systems immunology-based approaches. Refocusing the deployment of existing tools and experimental methods that have proven successful in tumor cell contexts to identify and validate MRC-targeting agents capable of remodeling the immunosuppressive cell states of the tumor microenvironment can potentially pave the road to novel combination therapy synergizing with immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/992372e5de760c711f403d3ef7aa2e616073ce3d" target='_blank'>
              Systematic identification and targeting of master regulator checkpoints (MRC) governing tumor microenvironment-mediated immune evasion
              </a>
            </td>
          <td>
            Pasquale Laise, G. Bosker, Mariana Babor, Xiaoyun Sun, Stuart Andrews, Lorenzo Tomassoni, Andrea Califano, Mariano J. Alvarez
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinoma (SCC) is known for its stepwise progression from healthy skin to premalignant actinic keratosis (AK), followed by a malignant transformation to SCC. Unfortunately, less attention has been paid to changes in gene expression in the tumour microenvironment during this process. We retrospectively selected early‐stage cutaneous SCC tissue samples containing both invasive and premalignant portions and conducted a spatial transcriptomic experiment using a NanoString GeoMx Digital Spatial Profiler (DSP). First, we selected invasive and premalignant regions of interest (ROIs) for each tissue. We then compared the gene expression patterns between the two portions (invasive versus premalignant) of the three segments: tumour cells, immune cells and fibroblasts, in each ROI. As a result, early‐stage cutaneous SCC tissue samples from 17 patients were selected for this study. We identified 29, 14 and 15 differentially expressed genes (DEGs) between the invasive and premalignant portions of the tumour cells, immune cells and fibroblasts, respectively. The top three genes with the highest absolute log2 fold‐change were CCDC88C, GJD3 and COMP in tumour cells; SVEP1, TSLP and PPP2R5C in immune cells; and SPAG6, PPP1CA and CCDC68 in fibroblasts. Notably, several genes, such as COMP, SVEP1 and SPAG6, have been linked to the development and function of cancer‐associated fibroblasts. Functional enrichment analysis revealed that several pathways were altered in tumour and immune cells. In conclusion, distinctive changes in gene expression patterns were observed as AK progressed to SCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1212cb843dc2d0cad33b53da437b7c501e408d51" target='_blank'>
              Spatial Transcriptomics Shows a Distinctive Tumour Microenvironment in the Invasive Versus Premalignant Portion of Early Cutaneous Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jeong-An Gim, Chungyeul Kim, Hyun Jyung Oh, Ko Eun Kim, J. Jeon, Aeree Kim, Y. Baek
          </td>
          <td>2025-06-01</td>
          <td>Experimental Dermatology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Ovarian cancer is one of the most common malignancies of the female reproductive system and is associated with poor prognosis. This study aimed to utilize single-cell RNA sequencing to investigate the heterogeneity of malignant epithelial cells in ovarian cancer, focusing on their potential functions and the implications for treatment and prognosis. Methods Single-cell RNA sequencing data were clustered using a single-cell transcriptome clustering method, and malignant epithelial cells were identified through copy number variation analysis. The interaction patterns between different malignant subpopulations and immune/stromal cells were analyzed using cell-to-cell communication analysis. A risk score (URS) model based on the UBE2C + epithelial subpopulation was then constructed through LASSO and multivariable Cox regression. High and low URS groups were compared in terms of tumor mutational burden (TMB), survival outcomes, and drug sensitivity. Finally, the role of GTF2F2 in ovarian cancer progression was validated through gene knockdown experiments in an ovarian cancer cell line (ES-2). Results Three major malignant epithelial cell subpopulations were identified (TMSB4X + Epi, TSC22D1 + Epi, and UBE2C + Epi). The UBE2C + Epi subpopulation exhibited higher stemness and greater invasive potential. The constructed URS model effectively stratified patients into high- and low-risk groups, with the high-risk group displaying a higher TMB level (p = 0.00011). Drug sensitivity predictions indicated that osimertinib, rapamycin, and dihydrorotenone might have stronger inhibitory effects in the high-risk group, whereas ERK inhibitors were more effective in the low-risk group. Functional assays demonstrated that GTF2F2 knockdown significantly suppressed ovarian cancer cell migration and invasion. Western blot analyses further showed elevated E-cadherin and reduced N-cadherin expression, suggesting that GTF2F2 may promote epithelial-mesenchymal transition (EMT). Conclusion The risk score model established in this study offers a novel framework for patient stratification and personalized therapy. Notably, the identification of the UBE2C + Epi subpopulation and key genes such as GTF2F2 highlights potential diagnostic and therapeutic targets, shedding light on the pathogenesis of ovarian cancer and paving the way for precision medicine approaches. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01686-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519b55d3039bd908fc634a743e91a97b194c1fa0" target='_blank'>
              Single-cell RNA sequencing reveals the role of GTF2F2 in ovarian cancer oncogenesis and progression
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-05-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains a leading cause of cancer-related mortality in men worldwide, with tumor heterogeneity, therapy resistance, and lineage plasticity posing significant clinical challenges. Androgen deprivation therapy (ADT), while initially effective, often culminates in castration-resistant prostate cancer (CRPC), may fueled by castration-resistant prostate cancer stem cells (CSCs) with adaptive self-renewal and regenerative capacities. Emerging evidence implicates prostate CSCs as pivotal contributors to tumor heterogeneity, drug resistance, and disease recurrence. These stem subpopulations exhibit intrinsic adaptability through genetic, epigenetic, and microenvironmental reprogramming, enabling survival under androgen-deprived conditions and fostering clonal diversification. This review synthesizes current knowledge on the identity, regulation, and functional dynamics of prostate stem cells, emphasizing their role in shaping the tumor ecosystem. By dissecting the genetic drivers, epigenetic alteration and crosstalk with microenvironment, this review underscores critical triggers on prostate CSCs determination and differentiation. Additionally, we discuss emerging strategies to target prostate CSCs-specific vulnerabilities, including molecular drivers of stemness and plasticity, to improve therapeutic outcomes for advanced prostate malignancies. This synthesis underscores the critical need to unravel prostate stem cells biology for developing precision therapies against prostate CSCs-driven adaptation in prostate diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/749e1d03aa4d620992d251430c919ea9331b3e5d" target='_blank'>
              Prostate cancer stem cell dynamics in the evolution of drug resistance.
              </a>
            </td>
          <td>
            Xiaoyu Zhang, Xiaoqing Zhang, Xinpei Deng, Jingna Cao, Qing Bao, Huan Wang, Dong Chen, Hailin Tang
          </td>
          <td>2025-06-04</td>
          <td>International journal of surgery</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Approximately 70% of breast cancer (BC) diagnoses are estrogen receptor positive (ER+) with ∼40% of ER+ BC patients presenting de novo resistance to endocrine therapy (ET). Recent studies identify the tumor microenvironment (TME) as having a key role in endocrine resistance in which adipose-derived stem cells (ASCs) play an essential role in cancer progression. Prior studies have indicated that ASC characteristics such as age and BMI may play a role in cancer progression. Unfortunately, most studies on ASC-BC cross talk have relied on established two-dimensional (2D) culture systems or the use of conditioned media that cannot replicate the complexity of the three-dimensional (3D) environment. This study used a microfluidic droplet trapping array and thiol-acrylate (TA) hydrogel scaffold to co-culture ER+ BC cells and ASCs as individual 3D spheroids (single culture) or organoids (co-culture) in a single device. Endocrine response was interrogated in both spheroids and organoids through the evaluation of proliferation following treatment with the selective estrogen receptor degrader (SERD) fulvestrant (ICI 182 780) followed by 17β-estradiol (E2). Terminal immunostaining for the proliferation marker (Ki67) was performed to evaluate how the presence of ASCs from different donor backgrounds (age and BMI) can modulate endocrine response. Results demonstrated that organoids containing two model ER+ cell lines (MCF7 and ZR-75) exhibited enhanced Ki67 expression even in the presence of ICI, suggesting a role for ASCs in cancer progression and endocrine resistance. Data clustering and classification algorithms were employed to categorize cellular behavior based on Ki67 expression and spheroid area to identify distinct clusters with high (H), intermediate (I), and low (L) Ki67 expression. Machine learning further stratified the data and revealed the direct effects of ASCs on Ki67 expression as well as how donor-specific features influenced ASC-driven changes in the TME. Notably, ASCs from an aged donor (>50) with lower BMI (<30) were able to enhance Ki67 expression even in the presence of endocrine therapy, while younger (<40) donors substantially enhanced Ki67 expression in the absence of both ICI and E2. Together, this study demonstrates the utility/development of a biomimetic culture system that recreates heterogenic 3D ER+ tumors through the co-culture of cancer cells with ASCs. This system provided insight into cell-extrinsic factors that govern ER+ breast cancer heterogeneity and response to endocrine therapy can be gained.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36b26f9d2f1fc166db1b577850e1e6422e08d444" target='_blank'>
              Droplet microfluidics integrated with machine learning reveals how adipose-derived stem cells modulate endocrine response and tumor heterogeneity in ER+ breast cancer.
              </a>
            </td>
          <td>
            B. O. Ortega Quesada, Calley Chauvin, Elizabeth Martin, Adam Melvin
          </td>
          <td>2025-06-19</td>
          <td>Lab on a chip</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is the complex ecosystem surrounding a tumor. It is a complex and continuously evolving entity. The TME comprises a variety of complex components, such as cancer cells, immune cells, stromal cells, and extracellular matrix, that precisely regulate the interaction of tumor cells with other components, allowing tumor cells to continue to metastasize, escape immune surveillance, resist apoptosis, and proliferate. Researchers believe that the TME is not just a silent bystander but instead an active promoter of cancer progression. At the onset of tumor growth, a dynamic and reciprocal relationship is developed between cancer cells and components of the tumor microenvironment; this relationship supports cancer cell survival, local invasion, and metastatic dissemination. In this article, we discuss the role of the various components of the TME in the progression and metastasis of cancer, the pathways influenced by the TME, and the drugs that can treat tumors by targeting the components of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9869eb44a45ce4947a2fc2cac8a08a25b3dc82" target='_blank'>
              The Role of the Tumor Microenvironment in Cancer Progression and Metastasis
              </a>
            </td>
          <td>
            Oseghale, Ikalo David, Ikokwu, Godwin Mmaduabuchi, Ugoagwu, Kingsley Ugonna, Elizabeth Anuoluwa Akintayo, Matthew Gyang, Okiki Nosiru Quadri, Catherine Rono, O. Ralph-Okhiria, Odunvbun Praise
          </td>
          <td>2025-05-29</td>
          <td>Journal of Cancer and Tumor International</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="
 e20113


 Background:
 Small cell lung cancer (SCLC) is a highly lethal malignancy, and there is an urgent need for more effective therapies. Antibody-drug conjugates (ADCs), which combine the precision of monoclonal antibodies with the cytotoxic power of small molecules, represent a promising approach. Despite extensive research, the relationship between ADC target overexpression and the transcriptional profile of SCLC remains unclear. Our study aimed to elucidate how ADC target expression correlates with transcriptional subtypes of SCLC, potentially guiding the selection of ADCs based on molecular profiling.
 Methods:
 RNA-seq and tandem mass spectrometry data for 107 patients with matched SCLC tumors and normal adjacent lung tissue were obtained from the CPTAC SCLC cohort. SCLC subtypes were defined according to expression of four transcription factors: achaete-scute homologue 1 (ASCL1), neurogenic differentiation factor 1 (NeuroD1), yes-associated protein 1 (YAP1) and POU class 2 homeobox 3 (POU2F3). ADC target expression for DLL3, SEZ6, B7-H3 (
 CD276
 ), and Trop2 (
 TACSTD2
 ) were assessed at the transcriptomic and proteomic level. The correlation between transcriptional subtype and ADC target expression was determined using Pearson’s correlation coefficient.
 Results:
 In 107 SCLC tumors, subtypes corresponded to 66 (61.6%) ASCL1, 21 (19.6%) NeuroD1, 3 (2.8%) YAP1, and 17 (15.9%) POU2F3. The ASCL1 subtype was positively correlated with DLL3 (R
 2
 =0.93, p=9E-87) and SEZ6 (R
 2
 =0.85, p=2.9E-57), and negatively correlated with TROP2 expression (R
 2
 =-0.72, p=1.5E-34). The NEUROD1 subtype showed moderate positive correlation with DLL3 (R
 2
 =0.52, p=4.8E-14) and SEZ6 (R
 2
 =0.58, p=3.6E-18), and a moderate negative correlation with TROP2 (R
 2
 =-0.52, p=1.6E-14). The POU2F3 subtype did not demonstrate strong positive or negative correlations with ADC targets. The YAP1 subtype was negatively correlated DLL3 (R
 2
 =-0.72, p=1.1E-32) and SEZ6 (R
 2
 =-0.71, p=9.5E-33), and positively correlated with Trop2 expression (R
 2
 =0.85, P=7.5E-61).
 Conclusions:
 Our findings reveal that molecular subtypes of SCLC exhibit unique patterns of ADC target expression. These profiles are crucial for further research and could lead to the development of subtype-specific treatments that improve clinical outcomes for patients with SCLC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34167c0dda56aaaeb23c8d98646f057387899f8a" target='_blank'>
              Proteogenomic characterization of antibody-drug conjugate target expression profiles in transcriptional subtypes of small cell lung cancer.
              </a>
            </td>
          <td>
            Edwin Lin, Mitch Singstock, Sanad Alhushki, Sameer S. Deshmukh, Changde Cheng, Kayla F. Goliwas, J. Deshane, A. R. Naqash, Fawzi Abu Rous, Rajat Thawani, Aakash Desai
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Reactivation of transposable elements (TEs) in somatic tissues, particularly of LINE-1, is associated with disease by causing gene mutations and DNA damage. Previous work has shown that the PIWI pathway is crucial for TE suppression in the germline. However, the status and function of this pathway has not been well characterized in differentiated somatic cells and there is lack of consensus on the role of the pathway in somatic tumorigenesis. To shed light on this conundrum, we examined the PIWI pathway in colon cancer through a combination of bioinformatic analyses and cell-based assays. Shifted Weighted Annotation Network (SWAN) analysis revealed that the pathway experiences significant allelic losses in colon cancer and that PIWIL2, the main catalytic component of the pathway responsible for TE silencing, experiences the highest percent deletions. PIWIL2 is downregulated in colon tumors of advanced stage, nodal metastasis, and in certain subtypes, correlating with poor survival, while it is even downregulated in ulcerative colitis, an inflammatory bowel disease that predisposes to colon cancer. Knockout studies in colon epithelial Caco2 cells show that PIWIL2 depletion leads to increased anchorage-independent growth, increased LINE-1 levels and activity, and in DNA damage, altogether highlighting a tumor-suppressing role of PIWIL2 in the colon. SUMMARY STATEMENT This study investigates the PIWI-piRNA pathway in colon cancer using a nuanced bioinformatic and cell-based approach, linking the downregulation of PIWIL2 to disease progression, transposable element activation and DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6553455e0dd4bcf44406169f133238204316d7c1" target='_blank'>
              PIWIL2 downregulation in colon cancer promotes transposon activity and pro-tumorigenic phenotypes
              </a>
            </td>
          <td>
            Alyssa Risner, J. Nair-Menon, Abhinav Cheedipudi, Joe R Delaney, Vamsi K Gangaraju, A. Kourtidis
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Although changes in mitochondrial morphology consistently associated with the aging of hematopoietic stem cells (HSCs), the specific molecular and cellular mechanisms involved are partially unclear. Live-cell super-resolution (SR) microscopy has been used to identify distinct HSC subsets that characterized by mitochondria unique morphologies and spatial distributions. The integration of SR microscopy with single-cell RNA sequencing enabled the classification of approximately 200 HSCs from young and aged mice into 5 discrete clusters. These clusters displayed molecular profiles that corresponded to the observed mitochondria states. An integrated approach combining RNA biomarker analysis and potential regulon assessment revealed previously unrecognized roles of GDF15 in mediating mitochondrial signals and morphologies that influence HSC fate. Thus, combining SR imaging with a bioinformatics pipeline provides an effective method for identifying key molecular players in specific phases of cellular transition, even with a relatively small dataset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c68fc79446f79d0f16056e1a01407f7af1f0e09" target='_blank'>
              Super-resolution imaging informed scRNA sequencing analysis reveals the critical role of GDF15 in rejuvenating aged hematopoietic stem cells
              </a>
            </td>
          <td>
            Zhongyu Shi, Xuefei Zhang, Huihui Yang, Xiaolu Zheng, Mengxiao Niu, Yongjian Zhang, Pengfei Yuan, Wensheng Wei, Gang Huang, Riguo Fang, Liangyi Chen
          </td>
          <td>2025-05-22</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related mortality globally, with non-small cell lung cancer (NSCLC) representing 85% of cases. Advances in treatment modalities, including stereotactic radiation therapy, have improved outcomes. However, possible synergistic effects of these therapies remain underexplored at the molecular level. This study investigated high-dose radiation-induced proteomic changes in lung adenocarcinoma cell line HCC-44 grown adherently and cell line A549, grown as adherent cells or spheroids. Our hypothesis was that proteins upregulated by 10 Gy irradiation serve as resistance drivers in cancerous cells and can thus represent potential therapeutic targets. The label-free mass spectrometry revealed distinct proteomic responses to 10 Gy irradiation, varying by cell line and culturing conditions. Differentially expressed proteins elevated in the irradiated samples included ephrin type-A receptor 2 (EPHA2) in adherent cells and insulin-like growth factor 2 receptor, tetraspanin 3 as well as cathepsin D (CTSD) in spheroids. The validation of these targets was carried out via Western blot, immunofluorescence, viability assay and spheroid formation assay which demonstrated that irradiation sensitized adherent NSCLC cells to EPHA2 and CTSD inhibitors. These findings underscore the potential of integrating radiation and targeted therapies in NSCLC treatment, and highlight EPHA2 as a promising candidate for future therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-09285-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b394ae32bb3fba2e6d32961ffafe24b8e9f20a6" target='_blank'>
              Proteome changes associated with effect of high dose single-fractionation radiation on lung adenocarcinoma cell lines
              </a>
            </td>
          <td>
            Elina Leis, Vijayachitra Modhukur, H. Lust, M. Vardja, Merilin Saarma, Ivar Ilves, Margaret Pütsepp, Sergei Kopanchuk, Jana Jaal, D. Lavogina
          </td>
          <td>2025-07-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Compelling evidences have manifested that breast cancer cells prefer to metastasize to certain distant organs, including brain, lung, bone and liver. According to the canonical “seed and soil” theory, this prominent biological behavior, termed as metastatic organotropism, involves intricate interactions between breast cancer cells (the “seeds”) and specific residents in the tumor microenvironment (the “soil”), initiating from pre-metastatic niche formation to metastatic outgrowth. Recently, multifaceted heterogeneity of tissue-specific macrophages (TSMs) and their roles played in organotropic metastases of breast cancer are incrementally unveiled. Herein, we decipher multiple diversities of TSMs, including evolvement, profiles, functions and metabolic characteristics under different polarization states. Further, we elaborate on bidirectional effects of TSMs on metastatic organotropism of breast cancer (both to the “seeds” and “soil”), and unearth underlying signaling pathways based on updated mechanistic researches. Lastly, we compile a series of clinical trials, hoping to illuminate promising TSM-targeting therapies against breast cancer organotropic metastases. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ed79b9316a9fed3f14b3bb67e4e61ce4c40709" target='_blank'>
              Heterogeneous tissue-specific macrophages orchestrate metastatic organotropism of breast cancer: implications for promising therapeutics
              </a>
            </td>
          <td>
            Cenzhu Wang, Pinchao Fan, Yu Zhou, Meican Ma, Haowei Zhong, Lei Liu, Qin Chen, Kun Xu
          </td>
          <td>2025-06-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 e15100


 Background:
 Intratumoral heterogeneity, driven by phenotypic plasticity, contributes to disease progression and therapy failures. However, a thorough understanding of its underlying mechanisms is still lacking. Breast cancer exemplifies this with its histological and molecular diversity; estrogen receptor-positive (ER+), progesterone receptor-positive (PR+), and HER2- luminal A subtypes, such as the MCF7 cell line, are not the exception. While single-cell RNA sequencing (scRNA-seq) and three-dimensional multicellular tumor spheroids (MCTS) represent initial approaches to understanding heterogeneity, integrating these tools with computational strategies offers new paths to uncover the regulatory groundwork of tumor behavior. When modeled via Boolean frameworks, transcriptional regulatory networks (TRNs) provide a dynamic platform for studying gene interactions, identifying attractors (stable cellular states corresponding to distinct phenotypes), and simulating cellular transitions. This study represents the first step in constructing such a framework for ER+ breast cancer to advance precision medicine and improve therapeutic strategies.
 Methods:
 Regulatory elements relevant to the MCF7 cell line were identified through an extensive review of NCBI literature. Interactions were strictly curated based on evidence quality (ranging from robust experimental validation to predictive hypotheses), and evaluated for type and significance, such as feedback loops, activation, coactivation, and inhibition. Logical operators (AND, OR, NOT) were used to translate regulatory connections into Boolean functions to simulate binary gene expression dynamics. The final TRN was visualized using Cytoscape (v3.10.3) with a hierarchical layout.
 Results:
 The TRN consists of 54 nodes and 81 interactions, representing the regulatory landscape of ER+, PR+, and HER2- breast cancer. Boolean equations offer a framework for simulating the evolution of binary states to attractors, enabling the observation of how perturbations, such as gene overexpression or silencing, influence heterogeneity. While simulation and attractor analyses are ongoing, the network provides a foundation for elucidating the regulatory mechanisms underlying breast cancer plasticity.
 Conclusions:
 This study highlights the potential of integrating experimental data, TRN reconstruction, and Boolean modeling to understand intratumoral heterogeneity. Our rigorously curated model establishes a foundational framework for precision oncology and the integration of experimental and computational methods.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10947cef519fbed46d1a42e2676b4ea1fd778e16" target='_blank'>
              Boolean network modeling: The first step toward a new understanding of breast cancer heterogeneity.
              </a>
            </td>
          <td>
            Lia Medina Montalvo, Thelma Escobedo-Tapia, Osbaldo Resendis-Antonio
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) has a poor survival rate due to late detection. PDAC arises from precursor microscopic lesions, termed pancreatic intraepithelial neoplasia (PanIN), that develop at least a decade before overt disease--this provides an opportunity to intercept PanIN-to-PDAC progression. However, immune interception strategies require full understanding of PanIN and PDAC cellular architecture. Surgical specimens containing PanIN and PDAC lesions from a unique cohort of five treatment-naïve patients with PDAC were surveyed using spatial-omics (proteomic and transcriptomic). Findings were corroborated by spatial proteomics of PanIN and PDAC from tamoxifen-inducible KPC (tiKPC) mice. We uncovered the organization of lymphoid cells into tertiary lymphoid structures (TLSs) adjacent to PanIN lesions. These TLSs lacked CD21+CD23+ B cells compared to more mature TLSs near the PDAC border. PanINs harbored mostly CD4+ T cells with fewer Tregs and exhausted T cells than PDAC. Peri-tumoral space was enriched with naïve CD4+ and central memory T cells. These observations highlight the opportunity to modulate the immune microenvironment in PanINs before immune exclusion and immunosuppression emerge during progression into PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6b6b93e0faedffa7b598bebfd58275c143f057" target='_blank'>
              Spatial Proteomics and Transcriptomics Reveal Early Immune Cell Organization in Pancreatic Intraepithelial Neoplasia.
              </a>
            </td>
          <td>
            Melissa R Lyman, Jacob T. Mitchell, Sidharth Raghavan, Luciane T. Kagohara, Amanda L. Huff, Saurav D Haldar, Sarah M. Shin, Samantha Guinn, Benjamin Barrett, Gabriella Longway, Alexei Hernandez, Erin M Coyne, Xuan Yuan, Lalitya Andaloori, Jiaying Lai, Yun Zhou Liu, Rachel Karchin, Anuj Gupta, A. Kiemen, André Forjaz, Denis Wirtz, Pei-Hsun Wu, Atul Deshpande, Jae W. Lee, T. Armstrong, Nilofer S. Azad, Jacquelyn W. Zimmerman, Laura D. Wood, Robert A Anders, Elizabeth D. Thompson, Elizabeth M. Jaffee, E. Fertig, W. Ho, N. Zaidi
          </td>
          <td>2025-06-26</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Cancer progression remains a significant clinical challenge. Phenotypic adaptation by tumour cells results in disease heterogeneity, which drives treatment resistance and immune escape. T-cell immunotherapy, while effective at treating some cancer subtypes, can also fail due to limits on tumour immunogenicity or T-cell recognition. For example, one potential contributor to immune escape involves the density and alignment of the extracellular matrix (ECM) surrounding tumours, also known as tumour-associated collagen signature (TACS). However, the specific mechanisms by which aligned fibres contribute to decreased patient survival rates have not yet been decoupled. Here, we developed EVO-ACT (EVOlutionary agent-based cancer T-cell interaction), a two-dimensional agent-based modelling framework designed to investigate how different TACS architectures impact tumour evolution and dynamic interactions with CD8[Formula: see text] T cells. Our results highlight that TACS-driven modulation of T-cell dynamics, combined with phenotypic adaptation, such as epithelial-to-mesenchymal transition, underlies differences in tumour immunogenicity and the application of our model can successfully recapitulate clinically observed breast cancer survival trends.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195f55d30b50b8b9b55b6c384b5267c2b8395881" target='_blank'>
              Distinct evolutionary patterns of tumour-immune escape and elimination determined by extracellular matrix architectures.
              </a>
            </td>
          <td>
            Yijia Fan, Jason T. George
          </td>
          <td>2025-07-01</td>
          <td>Journal of the Royal Society, Interface</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Objectives: TIGD1 (Trigger Transposable Element Derived 1) is a recently identified oncogene with largely unexplored biological functions. Emerging evidence suggests its involvement in multiple cellular processes across cancer types. This study aimed to perform a comprehensive pan-cancer analysis of TIGD1 to evaluate its expression patterns, diagnostic utility, prognostic value, and association with immunotherapy response and drug resistance. Methods: Transcriptomic and clinical data from TCGA and GTEx were analyzed using various bioinformatic tools. Expression profiling, survival analysis, immune correlation studies, gene set enrichment, single-cell sequencing, and drug sensitivity assessments were performed. Results: TIGD1 was found to be significantly upregulated in various tumor types, with notably high expression in colon adenocarcinoma. Elevated TIGD1 expression was associated with poor prognosis in several cancers. TIGD1 levels correlated with key features of the tumor immune microenvironment, including immune checkpoint gene expression, TMB, and MSI, suggesting a role in modulating anti-tumor immunity. GSEA and single-cell analyses implicated TIGD1 in oncogenic signaling pathways. Furthermore, high TIGD1 expression was linked to resistance to several therapeutic agents, including Zoledronate, Dasatinib, and BLU-667. Conclusions: TIGD1 may serve as a promising diagnostic and prognostic biomarker, particularly in colon, gastric, liver, and lung cancers. Its strong associations with immune modulation and therapy resistance highlight its potential as a novel target for precision oncology and immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1163400219b2ecb1878714587256ba612b794cd1" target='_blank'>
              Beyond Transposons: TIGD1 as a Pan-Cancer Biomarker and Immune Modulator
              </a>
            </td>
          <td>
            Merve Gülşen Bal Albayrak, T. Korak, Gurler Akpinar, Murat Kasap
          </td>
          <td>2025-05-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e20009


 Background:
 Osimertinib resistance in patients with EGFR-driven non-small cell lung cancer (NSCLC) contributes to mortality. Targeting bypass alterations have had limited efficacy. Here, we explored the role of CD105 (endoglin) expression, a co-receptor for bone morphogenetic protein (BMP) signaling as a factor of osimertinib treatment.
 Methods:
 Sixty-six NSCLC tumors, 44 of which had EGFR mutations were subjected to imaging mass cytometry to identify CD105 expression status as a predictor of clinical outcomes. To study the effects of EGFR and CD105 disruption two osimertinib-resistant EGFR-mutant cell lines were developed for subsequent characterization by single cell RNA sequencing. The promoter chromatin status and glycolytic state of the lines were evaluated by ATAC sequencing and Single Cell ENergetIc metabolism by profiling Translation inHibition analysis (SCENITH), respectively.
 Results:
 There was an inverse correlation between the expression CD105 and overall survival in treatment-naïve patients (
 P
 = 0.0071). Modeling osimertinib resistance highlighted the emergence of a persister cell population expressing tumor dormancy features. Further, the persister cells were found to have elevated CD105 levels with responsivity to the neutralizing antibody, carotuximab (ENV105), in restoring epigenetic and metabolic reprogramming to support osimertinib sensitivity. The carotuximab mechanism of action was through the re-engagement of EGFR signaling by potentiating physical CD105/EGFR interaction. Combination of osimertinib and carotuximab significantly reduced tumor growth in two independent NSCLC mouse models (
 P
 = 0.0056 and
 P =
 0.0008), where either single agent had little benefit.
 Conclusions:
 CD105 targeting provides a novel mutation-independent strategy to overcome resistance in EGFR-mutant NSCLC. Based on the efficacy of carotuximab capacity to limit tumor dormancy and restored osimertinib sensitivity, a Phase 1 trial (NCT05401110) investigating the combination therapy is currently accruing subjects.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/403b9eb716a17d3e6aa13d9c70998a95aef3047d" target='_blank'>
              Effect of carotuximab (ENV105)-mediated CD105 blockade on osimertinib sensitivity in EGFR-mutant non-small cell lung cancer persister cells.
              </a>
            </td>
          <td>
            Neil A. Bhowmick, Sandrine Billet, B. Waissengrin, K. Sankar, E. Posadas, Sungyong You, Jean Lopategui, Karen L. Reckamp, Manish Thiruvalluvan
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83fa54668a1bde6cb09830187a5dd62f51b3deb5" target='_blank'>
              Glioblastoma shift from bulk to infiltrative growth is guided by plexin-B2-mediated microglia alignment in invasive niches.
              </a>
            </td>
          <td>
            Sangjo Kang, Mary E. Ughetta, Jack Y Zhang, Valerie Marallano, Anirudh Sattiraju, Theodore Hannah, Shalaka Wahane, Aarthi Ramakrishnan, Molly E. Estill, N. Tsankova, Li Shen, A. Tsankov, R. Friedel, Hongyan Zou
          </td>
          <td>2025-05-29</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 e16171


 Background:
 Hepatocellular carcinoma (HCC), accounting for 70%-90% of primary liver cancers, often involves portal vein tumor thrombus (PVTT), a significant adverse prognostic factor linked to worsened liver function and treatment challenges. This study uses multi-omics analysis to investigate the tumor immune microenvironment in HCC with PVTT, focusing on the role of cancer-associated fibroblasts (CAFs) in PVTT formation and identifying potential therapeutic targets to improve clinical outcomes.
 Methods:
 We utilized nCounter technology, single-cell RNA sequencing (scRNA-seq), digital spatial profiling (DSP), and Olink proteomics to investigate intra-tumoral spatial heterogeneity from transcriptomics, proteomics, and immunology perspectives. Key molecules promoting PVTT formation were identified via differentially expressed genes (DEGs) analysis, spatial expression and single-cell analysis. Identified Acarbose as a potential drug for treating PVTT was conducted using 3D QSAR, pharmacophore modeling, and molecular dynamics simulations. The clinical potential of Acarbose targeting NID1 in PVTT treatment was further validated through cell proliferation assays, immune-functional mouse models, immunofluorescence, and real-world clinical data analysis.
 Results:
 A comprehensive multi-omics analysis of 85 samples from 47 HCC patients revealed that myofibroblast-associated cells (myCAFs) dominate the tumor microenvironment (TME) of PVTT, shaping a tumor immune-suppressive environment. The macrophage-to-myofibroblast transition (MMT) induced myCAF formation, establishing an immune barrier in the primary tumor. A cross-analysis of DEGs from DSP data and scRNA-seq identified Nidogen1 (NID1) as a key gene driving PVTT formation. NID1 was upregulated in MMT cells and myCAFs within the PVTT TME, contributing to immunosuppression and PVTT formation. Computational virtual screening identified acarbose as a potential therapeutic agent. When combined with tislelizumab, acarbose significantly enhanced T cell-mediated tumor cytotoxicity, inhibited tumor growth, and reduced NID1 expression, demonstrating superior efficacy compared to anti-PD-1 monotherapy, both in vitro and in vivo. Real-world data showed that long-term use of acarbose in diabetic HCC patients reduced NID1 levels and prevented PVTT formation, supporting acarbose as a promising strategy to target NID1 and inhibit PVTT.
 Conclusions:
 Our findings demonstrate the critical role of myCAFs in shaping the tumor immune-suppressive environment that promotes PVTT formation. Targeting NID1, a key driver of this process, with acarbose effectively disrupts the myCAF-mediated immune barrier and synergistically enhances the efficacy of immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e255ac4af9555248e208900e14562fa04e3faced" target='_blank'>
              Multi-omics investigation for the tumor immune microenvironment and spatial heterogeneity in hepatocellular carcinoma with PVTT: MyCAFs and NID1 in immune evasion.
              </a>
            </td>
          <td>
            Tao Han, Zhi Zhu, Heming Li, Kai Li
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 8084


 Background:
 Pleural mesothelioma (PM) is an aggressive malignancy that harbors significant inter- and intra-tumoral heterogeneity. Spatial transcriptomics enables the dissection of the tumor's molecular architecture by facilitating compartment-specific gene expression profiling. We performed high-resolution RNA-seq analysis of tumor (Tm) and stroma (St) compartments in PM samples to identify gene expression patterns and their association with clinical outcomes.
 Methods:
 Formalin-fixed paraffin-embedded (FFPE) tumor samples from untreated PM patients (pts) across three institutions were analyzed using the NanoString GeoMx Digital Spatial Profiling (DSP) platform. Regions of interest (ROIs) were selected based on histopathological features and fluorescently labeled antibodies for tumor and stromal areas. RNA expression of >1800 genes from selected ROIs was analyzed using GeoMx Cancer Transcriptome Atlas (CTA). Differential gene expression was assessed utilizing R “limma” package. A cutoff of absolute fold change ≥1 and p-value <0.05, with the Benjamini-Hochberg false discovery rate method, was applied to identify significant differentially expressed genes (DEGs). Elastic Net regression optimized through cross-validation methods was employed to identify genes associated with overall survival (OS) outcomes. Data from the TCGA PanCancer Atlas was utilized for external validation.
 Results:
 A total of 72 pts, 80.3% male, median age of 71y (range: 44-94) were identified for the analysis. Among them, 87.5% (63/72) were epithelioid (Ep) and 12.5% (9/72) non-epithelioid (NEp). After quality control, RNA data was available from 71 and 67 pts in Tm and St compartments, respectively. Across 132 ROIs in the Tm compartment, we identified 4 significantly DEGs between NEp (upregulated
 COL5A2, THBS1
 ; downregulated
 CLU, KRT19
 ) and Ep subgroups. No DEG between Ep and NEp subgroups were identified in 115 ROIs from the St compartment. Unsupervised clustering identified four molecular subgroups with distinct gene expression in the Tm compartment, of which Cluster 1 showed significantly decreased OS (6.3m vs. 16.4m; HR 3.3,
 p
 =0.001). Elastic Net regression identified 31 genes predictive of OS (R² = 0.43, Harrell’s c-index = 0.87), including nine genes (
 IFNGR2, FCER1G, MFGE8, CKLF, CBL, HLA-DRB3, HK1, PLAT, CD163
 ) associated with worse prognosis. Tumors in Cluster 1 demonstrated higher expression of these genes. External validation using the TCGA cohort confirmed four genes
 IFNGR2
 (
 p
 = 0.02),
 CBL
 (
 p
 = 0.01),
 HK1
 (
 p
 <0.001),
 PLAT
 (
 p
 <0.001), as significantly associated with decreased OS in PM.
 Conclusions:
 Spatially resolved transcriptomic profiling suggests Tm-enriched regions as the primary drivers of PM subtype and aggressiveness, identifying nine genes and a molecular subgroup associated with poorer survival outcomes. Further validation and functional studies are warranted.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e449bee363e2fcac7be12428599b558c6df5c9ff" target='_blank'>
              Digital spatial profiling for identification of prognostic genes and molecular subgroups in pleural mesothelioma.
              </a>
            </td>
          <td>
            Mercedes Herrera, David Lora, M. Peressini, Ramon Yarza, J. Gracia, Álvaro C. Ucero, Magdalena Molero Abraham, A. Enguita, David Gómez-Sánchez, Vera Adradas Rodriguez, Esther Conde, N. Carrizo, Igor Gómez-Randulfe, Roxana Reyes, Noemi Reguart, J. Baena, Helena Bote de Cabo, S. Ponce Aix, L. Paz-Ares, Jon Zugazagoitia
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cervical cancer (CC) is the fourth commonest malignant tumor among women worldwide and is characterized by high heterogeneity and a complex ecosystem. A comprehensive understanding of the heterogeneity of tumors and the tumor microenvironment (TME) is crucial for effective CC management. Single-cell RNA sequencing (scRNA-seq) is a powerful tool that can be employed to unveil the heterogeneity of tumors and the TME, as well as to elucidate the evolutionary trajectories of tumors. Spatial transcriptomics (ST) technology, on the other hand, can address the complexity and diversity of the spatial microenvironment of tumors, thereby compensating for the limitations of scRNA-seq. As emerging technologies, both scRNA-seq and ST are increasingly being utilized in CC research. In this review, we summarized the latest advancements in scRNA-seq and ST for CC, with a focus on investigating tumor heterogeneity, the TME, tumor evolutionary trajectories, treatment resistance mechanisms, and potential therapeutic targets. These insights collectively contribute to the development of more effective treatment and prevention strategies for CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f5b3526e206bb4dcf755c55607c83c389e4ccb" target='_blank'>
              Applications and Prospects of Single-Cell RNA Sequencing and Spatial Transcriptomics in Cervical Cancer
              </a>
            </td>
          <td>
            Yifu Wang, Li Yang, Yunzhi Liu, Huangrong Ma, Miaoying Cai, Chunyu Liang, Li Zhang, Zezhuo Su, Zhiyuan Xu
          </td>
          <td>2025-06-26</td>
          <td>BioMed Research International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Single-cell metabolomics reveals cell heterogeneity and elucidates intracellular molecular mechanisms. However, general concentration measurement of metabolites can only provide a static delineation of metabolomics, lacking the metabolic activity information of biological pathways. Herein, we develop a universal system for dynamic metabolomics by stable isotope tracing at the single-cell level. This system comprises a high-throughput single-cell data acquisition platform and an untargeted isotope tracing data processing platform, providing an integrated workflow for dynamic metabolomics of single cells. This system enables the global activity profiling and flow analysis of interlaced metabolic networks at the single-cell level and reveals heterogeneous metabolic activities among single cells. The significance of activity profiling is underscored by a 2-deoxyglucose inhibition model, demonstrating delicate metabolic alteration within single cells which cannot reflected by concentration analysis. Significantly, the system combined with a neural network model enables the metabolomic profiling of direct co-cultured tumor cells and macrophages. This reveals intricate cell-cell interaction mechanisms within the tumor microenvironment and firstly identifies versatile polarization subtypes of tumor-associated macrophages based on their metabolic signatures, which is in line with the renewed diversity atlas of macrophages from single-cell RNA-sequencing. The developed system facilitates a comprehensive understanding single-cell metabolomics from both static and dynamic perspectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166a51a7cbf1ecec5a77b31065e8e755e5e4f1d1" target='_blank'>
              Dynamic single-cell metabolomics reveals cell-cell interaction between tumor cells and macrophages
              </a>
            </td>
          <td>
            Yi Zhang, Mingying Shi, Mingxuan Li, Shaojie Qin, Daiyu Miao, Yu Bai
          </td>
          <td>2025-05-16</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Background Low-grade gliomas (LGGs) exhibit diverse bacterial infiltrations. This study delves into the intricate relationship between microbial infiltration in glioma samples and tumor multi-omics characteristics, aiming to elucidate its impact on tumor behavior and patient prognosis. Methods We included low-grade glioma (LGG) samples from The Cancer Genome Atlas (TCGA) as analysis cohort and used LGG tumor samples from patients who underwent surgical treatment as validation cohort. For the TCGA samples, utilizing advanced machine learning algorithms, this study identified distinct patterns of bacterial infiltration within the LGG population and constructed a prognostically relevant intra-tumoral bacteria risk model (PRIBR Index). For the clinically derived samples, we performed 16S rRNA sequencing, bulk RNA sequencing, and proteomics analysis. Subsequently, the samples were stratified into high-risk and low-risk groups. We then explored clinical information, tumor microenvironment, methylation status, and sensitivity to targeted therapies between these groups to elucidate the impact of varying bacterial infiltration levels on glioma behavior. Results A total of 32 common differentially expressed genes were identified between the TCGA-LGG samples and the clinical samples when comparing the high-risk and low-risk groups. The high-risk group demonstrated elevated bacterial infiltration levels, which were associated with increased infiltration of inflammatory factors. Patients in this group exhibited shorter survival periods, potentially attributable to the heightened expression of negative immune checkpoints. Predictive analysis for targeted drugs indicated that certain agents might achieve a lower half maximal inhibitory concentration (IC50) in the low-risk group compared to the high-risk group. Furthermore, while no significant differences were observed in tumor mutation burden or copy number variation between the two groups, the high-risk group showed increased methylation levels across multiple pathways. Conclusion These findings offer new insights into the biological characteristics of gliomas and provide novel avenues for exploring new therapeutic approaches, bringing fresh perspectives to the field of intra-tumoral bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce47d16f6844c4845072cd8d33be7948563dc386" target='_blank'>
              Deciphering the impact of intra-tumoral bacterial infiltration on multi-omics profiles in low-grade gliomas
              </a>
            </td>
          <td>
            Wenshu Li, Zixiang Zhu, Longyuan Li, Xin Wu, Jiaxuan Li, Yi Zhou, Lingwen Gu, Pranathi Vittal, Zhouqing Chen, Zhong Wang, Lingchuan Guo
          </td>
          <td>2025-06-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Melanoma is the most aggressive form of skin cancer, and despite significant therapeutic advances over the past decade, a substantial number of patients still progress to a fatal outcome. The initiation and progression of melanoma are strongly influenced by interactions between melanoma cells and other components of the tumor microenvironment (TME). In this review, we focus on the interplay between fibroblasts resident in the tumor microenvironment and tumor cells. In particular, we examine the molecular mechanisms through which melanoma cells induce the transformation of resident fibroblasts into their active form, known as cancer-associated fibroblasts (CAFs). We also explore the role of CAFs in shaping the melanoma microenvironment (MME) and in organizing the pre-metastatic niche, a specialized microenvironment that forms in distant organs or tissues to support the survival and expansion of metastatic melanoma cells. Finally, we discuss emerging therapeutic strategies aimed at disrupting the interactions between CAFs, melanoma cells, and other components of the tumor microenvironment to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d9f3ff9ff2ce42f0f728f6171249249cc7ed33f" target='_blank'>
              The Role of Fibroblasts in Melanoma Development: From Tumor Microenvironment Remodeling to Pre-Metastatic Niche Formation
              </a>
            </td>
          <td>
            Immacolata Belviso, Raffaele Pastore, Aldo Mileo, Emiliano Del Genio, Stefania Boccia, Stefano Palermi, C. Sellitto, M. L. Motti
          </td>
          <td>2025-06-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Zwilch Kinetochore Protein(ZWILCH) has been reported to prevent cells from prematurely exiting mitosis. However, the underlying mechanisms or involvement of ZWILCH in the tumor immune microenvironment in various cancers remain largely unknown. Methods Generalized dysregulation of ZWILCH was observed through the whole transcriptome analysis in this study. The spatial transcriptome analysis was utilized to identify expressed regions of ZWILCH. Next, cells that mainly expressed ZWILCH in the tumor microenvironment were determined using the single-cell transcriptome analysis. Also, the “cellchat” R package was applied to estimate the effect of ZWILCH on malignant cell communication. Combining multiple analytic approaches including GSEA, GSVA, KEGG enrichment analysis, and Aucell, with TCPA functional protein data, Genome-wide CRISPR screening, potential functions of ZWILCH and the pathways in which ZWILCH participated were thoroughly exploited. Univariate Cox regression analysis calculated the association between ZWILCH and cancer patients’ adverse outcomes. Results ZWILCH is universally highly expressed in tumors. The spatial transcriptome analysis showed that ZWILCH overexpression comes from the tumoral region or mixed tumoral region. At the single-cell level, ZWILCH is chiefly expressed by malignant cells and proliferative T cells. The expression of ZWILCH mRNA is positively correlated with cell proliferation, repair of DNA damage, and cell cycle score. Plenty of metabolic pathways are inhibited in patients with high expression of ZWILCH. Moreover, after ZWILCH knockout, a large number of cancer cell lines are stagnated, inhibited, or died. Additionally, the malignant cells with positive expression of ZWILCH have a stronger ability for cell communication. In short, ZWILCH is meant to be a risk factor for clinical outcomes of multiple tumors. Conclusions ZWILCH is a promising therapeutic target that influences patient prognosis by participating in cell proliferation, cell communication, and reshaping the tumor microenvironment across different cancers. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02981-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73f14c1c066485285bab172ec02a8c4271d565b9" target='_blank'>
              Zwilch kinetochore protein affects the prognosis of cancer patients by participating in cell proliferation, enhancing cell communication, and reshaping the tumor microenvironment
              </a>
            </td>
          <td>
            Long Yao, Lianpo Liu, Jinsong Wu, Yunlong Huang, Renquan Zhang, Haoxue Zhang
          </td>
          <td>2025-07-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic cancer (PC) is marked by extensive heterogeneity, posing significant challenges to effective treatment. The tumor microenvironment (TME), particularly cancer-associated fibroblasts (CAFs), plays a critical role in driving PC progression. However, the prognostic and functional contributions of distinct CAF subtypes remain inadequately understood. Here, we introduce a novel 7-gene risk model that not only robustly stratifies PC patients but also unveils the unique role of PHLDA1 as a key mediator in tumor-stroma crosstalk.By integrating single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and bulk RNA sequencing data, we comprehensively characterized the heterogeneity of CAFs in PC. We identified five CAF subtypes and focused on matrix CAFs (mCAFs), which were strongly associated with poor prognosis. A 7-gene mCAF-associated risk model was constructed using advanced machine learning algorithms, and the biological significance of PHLDA1 was validated through co-culture experiments and pan-cancer analyses.Our multiomics analysis revealed that the novel 7-gene model (comprising USP36, KLF5, MT2A, KDM6B, PHLDA1, REL, and DDIT4) accurately predicts patient survival, immunotherapy response, and TME status. Notably, PHLDA1 was uniquely overexpressed in CAFs and correlated with the activation of key protumorigenic pathways, including EMT, KRAS, and TGF-β, underscoring its central role in modulating the crosstalk between CAFs and malignant ductal cells. Pan-cancer analysis further supported PHLDA1’s prognostic and immunomodulatory significance across multiple tumor types.Our study presents a novel 7-gene prognostic model that significantly enhances risk stratification in PC and identifies PHLDA1+ CAFs as promising prognostic biomarkers and therapeutic targets. These findings provide new insights into the TME of PC and open avenues for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69308ca9c802c2244ef4487c21e12f41adc3c76e" target='_blank'>
              Integrated multiomics analysis identifies PHLDA1+ fibroblasts as prognostic biomarkers and mediators of biological functions in pancreatic cancer
              </a>
            </td>
          <td>
            Rui Wang, Guan-Hua Qin, Yifei Jiang, Fu-Xiang Chen, Zi-Han Wang, Linglin Ju, Lin Chen, Da Fu, En-Yu Liu, Su-Qing Zhang, 
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1093733a8ade229255066e064b631c0ae1860e" target='_blank'>
              ImmunoFusion: A Unified Platform for Investigating RNA-seq-Derived Gene Fusions in Cancer and Immunotherapy
              </a>
            </td>
          <td>
            YanKun Zhao, Shixiang Wang, Shensuo Li, Minjun Chen, Su-Han Jin, U. Gaipl, Hu Ma, Jian-Guo Zhou
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The intratumoral microbiota is closely associated with tumor initiation and progression in multiple solid tumors, including gastric cancer (GC). Single-cell analysis of host–microbiome interactions (SAHMI) is a pipeline used to systematically recover and denoise microbial signals in human clinical tissues and examine tumor–microbiome interactions at the single-cell transcriptome level. In a large GC cohort, we used SAHMI to detect 12 bacteria, among which Helicobacter pylori (H. pylori) was widely present in multiple tumor and normal samples. Meanwhile, we verified the presence of H. pylori in GC tissues via fluorescence in situ hybridization and immunohistochemistry. We performed single-cell RNA sequencing to analyze 11 cell populations, including B cells, T cells, and epithelial cells, and these cell types contained large numbers of H. pylori. We detected obvious enrichment of H. pylori in cancer cells and identified 13 upregulated differentially expressed genes exhibiting significantly negative correlations with patient survival in the H. pylori-positive tumor group compared with the findings in the other groups, indicating that these genes could represent prognostic biomarkers or therapeutic targets for H. pylori-infected patients with GC. Moreover, H. pylori-enriched immune cells, including T cells, B cells, and macrophages, were associated with cell-type-specific gene expression and pathway activities, including cell fate and immune signaling. In summary, tumor–microbiome interactions might reflect or influence tumorigenesis in GC, which has implications for clinical practice. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04048-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15eeb536ba65c3f26bacf7103413865d7c8287ff" target='_blank'>
              Analyses of single-cell RNA sequencing uncover the role of intratumoral Helicobacter pylori in shaping tumor progression and immunity in gastric cancer
              </a>
            </td>
          <td>
            Jiao Xu, Jin Yang, Qi Sun, Jingbo Chang, Fan Wang
          </td>
          <td>2025-05-24</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by significant heterogeneity and a poor prognosis. M2 macrophages (M2) play a crucial role in the microenvironment of PDAC, yet their exact functional significance remains poorly understood. This study sought to explore the potential communication and clinical implications of M2 in PDAC. The differentiation pathway of M2 and their interaction with malignant PDAC cells were analyzed using single-cell RNA sequencing (scRNA-seq). Additionally, a prognostic signature related to M2 marker genes was constructed and analyzed using bulk RNA sequencing. Three distinct differentiation states of M2 were characterized, each exhibiting unique transcription factor expression profiles and signaling pathway activations. Potential intercellular communication between M2, PDAC malignant cells, and fibroblasts was initially observed at the level of computer analysis. The prognostic model was identified through the LASSO algorithm. Patients stratified by risk level displayed significant variations in prognostic indicators such as signaling pathway activity, gene mutation frequency, immune cell infiltration, and drug sensitivity. These findings are of course preliminary, and future in vitro/in vivo experiments will be needed to knock out these signature genes or ligand-receptor pairs of genes in macrophages to validate their role in macrophage development. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02838-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/273ead8bc63462bd01d5d5a63b43ec9833ef2540" target='_blank'>
              Establishment of M2 macrophage-related prognostic signature in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Hui Chen, Jingyang Yin, Xianxing Wang, Renpei Xia, Fanbo Qin, Zhuo Li, Jiali Yang, Changjiang Liu, Shixiang Guo, Huaizhi Wang
          </td>
          <td>2025-06-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Colorectal cancers (CRCs) present across a range of differentiation grades, which impact patient outcome and management; however, the molecular features and drivers of differentiation status are not fully understood. To address this, 84 commonly used human CRC cell lines were grown as xenografts in mice, revealing models of low-grade (LG) and high-grade (HG) CRC. Transcriptional profiling revealed coordinate downregulation of multiple transcription factors involved in intestinal development and differentiation, markers of colonic lineage-specific differentiation, and effectors of normal functions of the colonic epithelium in HG tumours. Mechanistically, multiple genes suppressed in HG tumours harboured promoter methylation, indicative of stable epigenetic silencing. Furthermore, markers of LGR5+ colon stem cells were suppressed in HG tumours, while markers of cell proliferation, fetal-like intestinal stem cells, and non-canonical cell types including mesenchymal cells were increased. These changes manifested in HG cell line displaying increased proliferation, migration and metastatic capacity. Importantly, CRC cell line-derived transcriptional profiles of differentiation grade were reflected in LG and HG patient-derived tumour organoids and primary CRCs, revealing cell lines accurately model differentiation grade. The models and tumour differentiation-related properties identified herein may inform new approaches for tailored CRC treatments based on tumour grade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32dad53227914d2bb2c43008266ddcb90f56b84" target='_blank'>
              Model systems and unique biological features of high and low-grade colorectal cancer (CRC) revealed by xenografting 84 human CRC cell lines
              </a>
            </td>
          <td>
            Ian Y Luk, J. Mooi, D. Mouradov, Tao Tan, Cameron M. Scott, F. Chionh, Laura J Jenkins, Camilla M. Reehorst, Rebecca Nightingale, P. Savas, J. Tse, Rebekah Li Crake, Eduard Batlle, Diego Arango, Higino Dopeso, Peter Gibbs, Niall C. Tebbutt, Rod B Luwor, Andrew M Scott, Faiza Basheer, Amardeep Dhillon, N. Clemons, David S Williams, Ron Firestein, Oliver M. Sieber, J. Mariadason
          </td>
          <td>2025-06-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) are essential components of the tumor microenvironment (TME), contributing to tumorigenesis, progression, and resistance to therapy. However, the functional diversity of CAF subpopulations and their role in tumor progression and patient prognosis remain poorly understood. This study aims to explore CAF heterogeneity and their functional roles in the TME using single-cell RNA sequencing (scRNA-seq) and multi-omics data analysis. Methods scRNA-seq data were analyzed to cluster CAF subpopulations in the TME, with key genes identified through functional annotation. Differentially expressed genes were analyzed, and prognostic genes were selected via Cox and LASSO regression. A risk score model (RiskScore) was developed for survival prediction and immune therapy sensitivity evaluation. Core CAF genes were examined using siRNA interference, qPCR, and Western blotting. Drug sensitivity was assessed to explore the clinical relevance of these genes. Results Four CAF subpopulations (CAF-0, CAF-1, CAF-2, CAF-3) were identified, revealing differences in key tumor-associated signaling pathways (e.g., MYC, WNT, TGF-β). Thirteen core genes related to prognosis were identified, and a RiskScore model was developed, showing significantly worse survival rates for high-risk patients (p < 0.001) and features of immune suppression, including increased M0 macrophage infiltration. Drug sensitivity analysis indicated that core genes (e.g., KLRB1, MAP1B) were linked to drug sensitivity, suggesting potential biomarkers for targeted therapy. Experimental validation showed that knockdown of the HIP1R gene significantly reduced tumor cell expression, confirming its critical role in tumor development. Conclusion This study offers a comprehensive analysis of CAF heterogeneity and its impact on TME, patient prognosis, and drug sensitivity. The developed RiskScore model provides theoretical support for personalized treatment based on CAF-related genes, offering new insights into CAF-driven tumor progression and potential targets for precision oncology and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5c1ce9d5dedbb8252961d9100d080e0187631a3" target='_blank'>
              Therapeutic implications of cancer-associated fibroblast heterogeneity: insights from single-cell and multi-omics analysis
              </a>
            </td>
          <td>
            Yumin Wang, Yi Ding, Haolin Liu, Zhongyou Xia, Guoqiang Liao, Shicheng Fan, JunXiong Li, JingBo Qin, PinYao Liang, Peng Gu, Xiaodong Liu, Runan Dong
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Penile squamous cell carcinoma (PSCC) is a highly aggressive malignancy without effective treatment due to limited knowledge of its development and tumor microenvironment (TME). In this study, single-nucleus RNA sequencing (snRNA-seq) and high-resolution spatial transcriptomics are employed to comprehensively investigate the development trajectories and the TME. The results revealed that PSCC cells mimicked the differentiation and tissue organization of normal penile epithelium, independent of the human papillomavirus (HPV) infection status. Notably, a spatial subtype, Tum_1, appeared at early stage of tumor differentiation and in tumor-normal boundary regions. This subtype exhibited enhanced basal-like and stemness features and showed high LAMC2 expression, which activated laminin-integrin signaling via ITGA6/ITGB4, promoting tumor invasiveness. Furthermore, the results indicated that HPV-positive basal stem-like neoplasms dampened the immune function of T cells and macrophages, promoting an immunosuppressive environment that facilitates tumor progression. Supporting this, the patients with head and neck squamous cell carcinoma and lung squamous cell carcinoma who have high expression of HPV-positive Tum_1 signatures derived greater benefit from PD-1 blockade therapy. In summary, the findings provide a comprehensive spatial landscape of the PSCC TME and suggest potential treatment approaches targeting laminin-integrin interaction and immunotherapy, especially in HPV-positive patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c44a03a89b4201962b2248fcc19db445771e54" target='_blank'>
              Single-Cell and Spatial Transcriptomic Profiling of Penile Squamous Cell Carcinoma Reveals Dynamics of Tumor Differentiation and Immune Microenvironment.
              </a>
            </td>
          <td>
            Hongjian Song, Zhichao Tong, Guixiang Xie, Yaowei Li, Yubo Zhao, Fengpu Fan, Zongzheng Yang, Qing Shi, Qian Zhang, Feng Wen, Hongxue Meng, Haonan Li, Pengyu Guo, Dayong Hou, Zhenwei Zhang, Zhihao Yin, Ziyi Liu, Jianwei Wang, Peng Zhang, Peng Dai, Changfu Li, Lichen Teng, Yangyang Xu, Dexin Ding, Tao Xu, Jianzhang Li, Yongsheng Chen, Yu Qiu, Xiaowei Hu, Wentao Liu, Liang Wu, Roman Nawroth, Guibo Li, Hang Su, Ziqing Deng, Ziqi Wang, Di Wang, Wanhai Xu
          </td>
          <td>2025-06-05</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer metastasis constitutes a multifactorial phenomenon that continues to confound therapeutic strategies. The biochemical signals governing motile phenotypes have been extensively characterized, but mechanobiological interactions have only recently been integrated into cancer cell motility models and remain less well elucidated. The identification of the biochemically and mechanically controlled epithelial–mesenchymal transition (EMT) of cancer cells, which occurs either completely or partially, has led to a major breakthrough and a universal phenomenon in cancers. In addition, a relatively new theory based on mechanobiological aspects called “jamming-to-unjamming transition” is being proposed to explain the transition of cancer cells to an invasive phenotype. The latter transition may help to better understand the different types of 3D migration and invasion of cancer cells. Similarly to EMT, the transition from jamming to unjamming seems to be controlled by molecular and physical factors, including cell mechanics and mechanical cues from the extracellular matrix (ECM) of the tumor microenvironment (TME). It is challenging to grasp the distinctions between the transition from jamming to unjamming and EMT, as they appear to be the same at first glance. However, upon closer examination, the two transitions are quite separate. Moreover, it is still unclear whether both transitions may act synergistically. This review highlights the most important breakthroughs in the transition from jamming to unjamming, with a focus on mechanobiology and extracellular environmental aspects, and it compares them with those of EMT. In addition, the impact of the TME, such as ECM scaffold and cancer-associated fibroblasts (CAFs) on the jamming-to-unjamming transition is discussed. Finally, the research frontiers and future directions in the field of mechanobiological research in cancer metastasis are outlined.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/356465968ad0ebd4b69565d6368d70dc587c326c" target='_blank'>
              ECM Mechanics Control Jamming-to-Unjamming Transition of Cancer Cells
              </a>
            </td>
          <td>
            C. Mierke
          </td>
          <td>2025-06-20</td>
          <td>Cells</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Gastric cancer remains a leading cause of cancer-related mortality worldwide, driven significantly by gastric cancer stem cells (GCSCs), a unique subset of cells characterized by self-renewal, multipotency, and therapy resistance. This review synthesizes recent advancements in understanding GCSCs, focusing on their definition, characteristics, identification, isolation and role in tumor growth, metastasis, and resistance to conventional treatments. Key surface markers such as CD44, CD133, leucine rich repeat containing G protein-coupled receptor 5 (LGR5), and aquaporin 5 (AQP5) are critical for GCSC identification, with their expression linked to poor prognosis and tumor progression. The review summarizes mechanisms like epithelial-mesenchymal transition (EMT) and Wnt/β-catenin signaling that enhance GCSC stemness and malignancy. It also highlights emerging targeted therapies, including antibody-based approaches including CD44v6-specific antibodies, inhibitors like sulfasalazine, and novel immunological strategies such as chimeric antigen receptor T (CAR-T) cell therapy, aimed at eradicating GCSCs. By elucidating the molecular and functional roles of GCSCs, this review underscores GCSCs as therapeutic targets and provides a foundation for developing precision medicine strategies to improve gastric cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cacd867c51c08c4c5178ffbc92262edb25abafe2" target='_blank'>
              Current insights into gastric cancer stem cells: Mechanisms and therapeutic prospects
              </a>
            </td>
          <td>
            Chang-Yong Yin
          </td>
          <td>2025-05-30</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fdc708d82395eddf5ef9f62e5ba5f1c540963b1" target='_blank'>
              Single-cell and bulk transcriptome analysis identifies B-cell subpopulations and associated cancer subtypes with distinct clinical and molecular characteristics.
              </a>
            </td>
          <td>
            Yin A He, Li Zhao, Yufen Zheng, Xiaosheng Wang
          </td>
          <td>2025-06-17</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Targeted therapy interventions using tyrosine kinase inhibitors (TKIs) provide encouraging treatment responses in patients with ALK-rearranged lung adenocarcinomas, yet resistance occurs almost inevitably. In addition to tumor cell-intrinsic resistance mechanisms, accumulating evidence suggests that cancer-associated fibroblasts (CAFs) within the tumor microenvironment contribute to therapy resistance. This study aimed to investigate CAF-driven molecular networks that shape the therapeutic susceptibility of ALK-driven lung adenocarcinoma cells. Methods Three-dimensional (3D) spheroid co-cultures comprising ALK-rearranged lung adenocarcinoma cells and CAFs were utilized to model the tumor microenvironment. Single-cell RNA sequencing was performed to uncover transcriptional differences between TKI-treated homotypic and heterotypic spheroids. Functional assays assessed the effects of CAF-conditioned medium and CAF-secreted factors on tumor cell survival, proliferation, lipid metabolism, and downstream AKT signaling. The therapeutic potential of targeting metabolic vulnerabilities was evaluated using pharmacological inhibition of lipid metabolism and by ferroptosis induction. Results CAFs significantly diminished the apoptotic response of lung tumor cells to ALK inhibitors while simultaneously enhancing their proliferative capacity. Single-cell RNA sequencing identified lipogenesis-associated genes as a key transcriptional difference between TKI-treated homotypic and heterotypic lung tumor spheroids. CAF-conditioned medium and the CAF-secreted factors HGF and NRG1 activated AKT signaling in 3D-cultured ALK-rearranged lung tumor cells, leading to increased de novo lipogenesis and suppression of lipid peroxidation. These metabolic adaptations were critical for promoting tumor cell survival and fostering therapy resistance. Notably, both dual inhibition of ALK and the lipid-regulatory factor SREBP-1, as well as co-treatment with ferroptosis inducers such as erastin or RSL3, effectively disrupted the CAF-driven metabolic-supportive niche and restored sensitivity of resistant lung tumor spheroids to ALK inhibition. Conclusions This study highlights a critical role for CAFs in mediating resistance to ALK-TKIs by reprogramming lipid metabolism in ALK-rearranged lung cancer cells. It suggests that targeting these metabolic vulnerabilities, particularly through inhibition of lipid metabolism or induction of ferroptosis, could provide a novel therapeutic approach to overcome resistance and improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40170-025-00400-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a02a6efa6497a2bcee01f9f0743c25e6ad43b8" target='_blank'>
              Cancer-associated fibroblasts promote drug resistance in ALK-driven lung adenocarcinoma cells by upregulating lipid biosynthesis
              </a>
            </td>
          <td>
            Ann-Kathrin Daum, Lisa Schlicker, M. Schneider, T. Muley, Ursula Klingmüller, Almut Schulze, Michael Thomas, P. Christopoulos, Holger Sültmann
          </td>
          <td>2025-06-16</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Clinically aggressive lactotroph pituitary neuroendocrine tumors (PitNET) are invasive tumors with an unusually rapid growth rate despite maximally tolerated doses of dopamine agonist (DA). We aimed to unravel the molecular heterogeneity of lactotroph PitNET and to identify biomarkers of aggressiveness and resistance to pharmacological treatment. A total of 13 patients harboring DA-resistant lactotroph PitNET were included in this study. Visium Spatial Transcriptomics (ST), whole transcriptome sequencing (WTS), and whole exome sequencing (WES) were performed in tumors from 4 of these patients; WTS and WES was carried out in 5; tumors from two patients underwent ST and WES and tumors from two other patients underwent only ST. Tumors were classified as null or partial responders according to their response to DA treatment. The eight PitNET analyzed by ST exhibited significant intratumoral heterogeneity, with clones showing alterations in PI3K/AKT and lipid metabolism pathways, particularly inositol phosphate, glycerophospholipid, and sphingolipid metabolism. The cell-cell communication analysis showed FGF-FGFR ligand receptor interaction whilst the transcription factors RXRA and CREM showed participation in both groups. A trajectory exploration was performed by including all PitNET together in a single analysis to determine whether there was a tendency or molecular pathway showing a differentiation pattern that would guide the transition from a partially responsive PitNET to a completely unresponsive one. We did not observe any such pattern. All of these findings were corroborated in the cohort of DA-resistant PitNETs in which only bulk WTS and WES were performed. The bulk WTS corroborated lipid metabolism and PI3K-AKT pathway alteration in PitNET, whereas the WES showed only SF3β1 and TP53 variants in one tumor each. Our work suggests that the PI3K/AKT pathway may constitute a molecular target at which to aim therapeutic strategies designed to treat aggressive and DA-resistant lactotroph PitNET. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02025-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9584b357aff2d73647fc85b3bbd6a84a02ea98" target='_blank'>
              Spatial transcriptomics reveal PI3K-AKT and metabolic alterations in aggressive, treatment-resistant lactotroph pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            F. Martinez-Mendoza, Sergio Andonegui-Elguera, E. Sosa-Eroza, Erick Gomez-Apo, Aurea Escobar-España, Carolina González-Torres, Javier Gaytán-Cervantes, Alam Palma-Guzman, Hugo Torres-Flores, Alberto Moscona-Nissan, S. Hinojosa-Alvarez, J. Hernández-Perez, R. Chavez-Santoscoy, G. Guinto, Gerardo Y. Guinto-Nishimura, B. López-Félix, Erick U. Zepeda-Fernandez, Erick M. Estrada-Estrada, V. Correa-Correa, P. A. González-Zavala, Marco A. Asenscio-Montiel, M. A. García-Vargas, Emmanuel Cantu-Chavez, Rocío Arreola-Rosales, Keiko Taniguchi-Ponciano, D. Marrero-Rodríguez, Moises Mercado
          </td>
          <td>2025-05-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Matrix metalloproteinase 1 plays a pivotal role in tumor biology and immune modulation through its enzymatic remodeling of the extracellular matrix, facilitating tumor progression. In this study, we utilized large-scale single-cell RNA sequencing and spatial transcriptomics to investigate MMP1 expression, its cellular localization, and its impact on tumor progression and immune modulation. Our findings reveal that MMP1 expression is elevated in various tumor types and is strongly correlated with metastatic potential. High MMP1 expression was associated with increased activity in epithelial–mesenchymal transition signaling and TNFα/NF-κB pathways, which are known to promote tumor progression. Furthermore, MMP1+ malignant cells exhibited significant interactions with immune cells, particularly macrophages and CD8+ T cells. MMP1 expression correlated with enhanced macrophage infiltration and impaired CD8+ T-cell function, contributing to an immunosuppressive tumor microenvironment. Notably, the CXCL16-CXCR6 and ANXA1-FPR3 signaling axes were identified as key mediators of these interactions. Inhibition of MMP1 in vitro demonstrated reduced cell invasion, stemness, and proliferation, while increasing reactive oxygen species levels and promoting apoptosis. Our findings position MMP1 as a key player in the “tumor-immune” vicious cycle and a promising therapeutic target to enhance anti-tumor responses and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d0c741845c10c07b47cb268ec3e101fb8935759" target='_blank'>
              Decoding the impact of MMP1+ malignant subsets on tumor-immune interactions: insights from single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Da-Ming Xu, Ling-Xiao Chen, Ting Xue, Xiao-Yu Zhuang, Li-Chao Wei, Hui Han, Miao Mo
          </td>
          <td>2025-05-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Addressing the growing need for organized data on tumor homing peptides (THPs), we present TumorHoPe2, a manually curated database offering extensive details on experimentally validated THPs. This represents a significant update to TumorHoPe, originally developed by our group in 2012. TumorHoPe2 now contains 1847 entries, representing 1297 unique tumor homing peptides, a substantial expansion from the 744 entries in its predecessor. For each peptide, the database provides critical information, including sequence, terminal or chemical modifications, corresponding cancer cell lines, and specific tumor types targeted. The database compiles data from two primary sources: phage display libraries, which are commonly used to identify peptide ligands targeting tumor-specific markers, and synthetic peptides, which are chemically modified to enhance properties such as stability, binding affinity, and specificity. Our dataset includes 594 chemically modified peptides, with 255 having N-terminal and 195 C-terminal modifications. These THPs have been validated against 172 cancer cell lines and demonstrate specificity for 37 distinct tumor types. To maximize utility for the research community, TumorHoPe2 is equipped with intuitive tools for data searching, filtering, and analysis, alongside a RESTful API for efficient programmatic access and integration into bioinformatics pipelines. It is freely available at https://webs.iiitd.edu.in/raghava/tumorhope2/">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4673f139ff89b98a8a9cf242e48538be8a2f0a9d" target='_blank'>
              TumorHoPe2: An updated database for Tumor Homing Peptides
              </a>
            </td>
          <td>
            Diksha Kashyap, Devanshi Gupta, Naman Kumar Mehta, G. Raghava
          </td>
          <td>2025-05-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 e18514


 Background:
 Acute lymphoblastic leukemia (ALL) is a hematological malignancy characterized by impaired differentiation and abnormal proliferation of lymphoblasts. Despite advances in the understanding of its molecular mechanisms, accurate characterization of cellular heterogeneity and genetic alterations remains a challenge. The aim of this study is to take advantage of the scGPT, generative pre-trained transformer model adapted to the genome of one cell, to improve the distinction of cellular clusters and profile gene expression in all. We have used data files including B-cell subtypes and T-cells of ALL. Comparative analyzes with conventional tools, such as scVI and Harmony, have shown an excellent ability to maintain biological variability while providing accurate cell type annotations.
 Methods:
 This study utilized single-cell RNA sequencing (scRNA-seq) to obtain transcriptomic profiles from individual cells. Biological samples were processed through optimized cell capture and library preparation protocols to reduce technical noise and batch effects. NVIDIA GPU powered the computational environment with CUDA acceleration, and key software tools included scVI, scrublet, harmony, and scGPT. Highly variable genes were identified using scVI, leveraging Variational Autoencoders (VAEs) to detect nonlinear expression variability. Dimensionality reduction was performed with scVI, producing embeddings corrected for batch effects. Clustering analysis using the Leiden algorithm identified cell subpopulations and visualization tools such as UMAP and t-SNE displayed relationships among cell groups.
 Results:
 Visualization techniques, including UMAP and T-SNE, have confirmed the excellent ability of scGPT to identify rare cell populations and subsets otherwise missed by traditional methods. For example, in B cell subtypes, scGPT successfully distinguished naive B cells, memory B cells, and plasmablasts with high accuracy. Similarly, T cell subtypes such as regulatory T cells and cytotoxic T cells have been clearly characterized. Integration of scGPT with zero-shot learning enabled accurate profiling of gene expression across cell clusters and identified key differentially expressed genes relevant to acute lymphoblastic leukemia (ALL) pathology. These were genes associated with cell proliferation, immune evasion and metabolic pathways.
 Conclusions:
 Advanced scGPT modeling has enabled the prediction of potential therapeutic targets and paved the way for tailored interventions. Our research applies scGPT as a key tool to advance single-cell transcriptomic studies in ALL, offering a better understanding of its cellular and molecular complexities. This approach holds promise for improving therapeutic strategies and personalized medicine in hematologic malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6d87f169f0235ef41c3797b0a5ed2283cc4c8a3" target='_blank'>
              Empowering single-cell genomics with scGPT: Precision clustering and genetic insights into acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Pei-Hsuan Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="For over a century, scientists reported the disruption of normal nuclear shape and size in cancer. These changes have long been used as tools for diagnosis and staging of malignancies. However, to date, the mechanisms underlying these aberrant nuclear phenotypes and their biological significance remain poorly understood. Using a model of pancreatic ductal adenocarcinoma (PDAC), the major histological subtypes of pancreatic cancer, we found oncogenic mutant KRAS reduces nuclear size. Transcriptomic and protein expression analysis of mutant KRAS-expressing PDAC cells revealed differential levels of several nuclear envelope-associated genes. Further analysis demonstrated the nuclear lamina protein, Emerin (EMD), acted downstream of KRAS to mediate nuclear size reduction in PDAC. Analysis of human PDAC samples showed that increased EMD expression associates with reduced nuclear size. Finally, in vivo genetic depletion of EMD in a mutant KRAS-driven PDAC model resulted in an increased nuclear size and a reduced incidence of poorly differentiated PDAC. Thus, our data provides evidence of a novel mechanism underlying nuclear size regulation and its impact in PDAC carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fad3a87beb812ed2828574a6cdacbebbce33710" target='_blank'>
              Emerin is an effector of oncogenic KRAS-driven nuclear dynamics in pancreatic cancer.
              </a>
            </td>
          <td>
            Luis F Flores, David L Marks, Renzo E Vera, Ashley N Sigafoos, Ezequiel J Tolosa, Luciana L Almada, David R Pease, M. D. Toruner, Brian Chang, Brooke R Tader, Kayla C LaRue-Nolan, Ryan M. Carr, Rondell P Graham, Catherine E Hagen, Matthew R Brown, Aleksey V Matveyenko, Katherine L Wilson, David W Dawson, Christopher L. Pin, Kyle J Roux, Martin E Fernandez-Zapico
          </td>
          <td>2025-06-10</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f88d93c45659d05d4c112ca60db918cd868e7fbf" target='_blank'>
              Signalomics for molecular tumor boards and precision oncology of breast and gynecological cancers.
              </a>
            </td>
          <td>
            Tatiana V Denisenko, Anna E. Ivanova, A. Koval, Denis N. Silachev, Lee Jia, G. Sukhikh, Vladimir L. Katanaev
          </td>
          <td>2025-06-09</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Small-cell lung cancer (SCLC) is the most lethal type of lung cancer, characterized by rapid evolution from chemosensitivity to chemoresistance and limited treatment options. However, the mechanisms underlying this evolution remain poorly understood. Here, we show that Retinoid X receptor γ (RXRγ) is uniquely overexpressed in chemo-resistant SCLC tumors, and that RXRγ serves as an essential factor driving chemoresistance in SCLC. RXRγ forms phase-separated droplets with LSD1 in the nucleus, which enhances RXRγ-mediated gene transcription activity and reprograms gene expression, promoting tumor stemness and metastasis, and eventually driving SCLC chemoresistance. In turn, RXRγ antagonist disrupts RXRγ-LSD1 interaction, reducing their binding to the target gene locus, markedly suppressing the expression of the RXRγ target gene network. Finally, RXRγ antagonists strongly suppress tumor growth and metastasis and restore SCLC vulnerability to chemotherapy in multiple preclinical SCLC models, resulting in a substantial extension of survival in mouse models. Thus, these results establish RXRγ as a key player in SCLC by phase separation and as a potential therapeutic target for this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc6976505863353b005498741bc1fbef13303de9" target='_blank'>
              Phase separation of RXRγ drives tumor chemoresistance and represents a therapeutic target for small-cell lung cancer.
              </a>
            </td>
          <td>
            Hong Wang, Jie Huang, Zhenhua Zhang, Yana An, Huizi Sun, Jiang-liang Chen, Weineng Feng, Hao Duan, Yonggao Mou, Yuanxiang Wang, Peiqing Liu, Huihao Zhou, Hong-Wu Chen, Jian Zhang, Xiaoyun Lu, Junjian Wang
          </td>
          <td>2025-05-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Pyroptosis is a form of programmed cell death characterized by inflammasome activation and the release of inflammatory cytokines, which induce a strong immune response. Unlike apoptosis, pyroptosis can elicit potent immune stimulation, potentially playing a crucial role in anti-tumor immunity. However, it may also promote tumor progression by altering the tumor microenvironment and facilitating immune evasion. This study investigates pyroptosis-related gene expression in hepatocellular carcinoma (HCC), with a focus on identifying key genes that influence prognosis and tumor microenvironment dynamics. Methods Single-cell RNA sequencing (scRNA-seq) data from 10 HCC patients were obtained from the GEO database (GSE149614), along with spatial transcriptomic data and bulk RNA-seq data from TCGA. We performed data processing and quality control using the Seurat package and applied machine learning techniques, including LASSO regression, to identify key pyroptosis-related genes. Functional analyses, including Gene Ontology (GO), KEGG, and GSVA, were conducted to explore biological pathways. Pyroptosis levels were quantified across cell types, and survival analysis was performed to evaluate prognostic impacts. Cell communication and immune infiltration were also assessed to understand the tumor microenvironment. Results We identified CHMP4B as a key pyroptosis-related gene in HCC, significantly associated with poor prognosis. High CHMP4B expression was correlated with shorter overall survival (OS) and disease-free survival (DFS). Functional enrichment analysis showed that CHMP4B is involved in cell cycle regulation, DNA repair, and cytoskeletal organization. Spatial transcriptomics revealed heterogeneous CHMP4B expression in the tumor microenvironment, with higher levels found in advanced tumor stages. Moreover, high CHMP4B expression was associated with increased infiltration of immunosuppressive cells, such as monocytes and macrophages, and upregulation of immune checkpoint molecules (PD-L1, CTLA4), suggesting its role in promoting immune evasion. Conclusions Our findings highlight CHMP4B as a critical regulator of pyroptosis in HCC, influencing tumor progression and immune modulation. High CHMP4B expression may facilitate the development of an immunosuppressive microenvironment, enabling immune escape and tumor growth. The study underscores CHMP4B’s potential as a prognostic biomarker and therapeutic target in HCC. However, the limited sample size calls for further validation using larger datasets and multi-omics approaches, such as proteomics and metabolomics, to fully elucidate its functional role in HCC pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db1e712eb15666ba81f87054115a23e3c5a7897" target='_blank'>
              Pyroptosis drives tumor progression and immune evasion in Hepatocellular Carcinoma: a single-cell and spatial transcriptomic study
              </a>
            </td>
          <td>
            Siyu Tao, Yunhao Yang
          </td>
          <td>2025-05-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) play immunosuppressive roles in the tumor microenvironment (TME). Specifically, they reportedly act as physical barriers preventing immune cell infiltration. However, the spatial relationships between CAFs and cancer cells in antitumor immunity remain unknown. In this study, we established three-dimensional (3D) constructs, in which the spatial relationships were controlled using a 3D bioprinter. Using these models, we found that the mixed distribution of fibroblasts (FBs) and cancer cells suppressed the antitumor immunity more than the surrounding distribution of FBs as physical barriers. The 3D construct with mixed distribution promoted transforming growth factor (TGF)-β and periostin (encoded by Postn gene) crosstalk, resulting in immunosuppression. Postn-knockdown in FBs decreased the TGF-β production in the mixed 3D construct and activated antitumor immunity both in vitro and in vivo. Clinically, patients with head and neck cancer or lung cancer showing mixed distribution of α-smooth muscle actin+ myofibroblast-like CAFs exhibited worse prognosis after programmed death protein-1 blockade therapies, and lower CD8+ T cell infiltration than those that had CAFs surrounding cancer cells. Overall, our findings suggest that the close interactions of CAFs and cancer cells facilitate immunosuppression, rather than the physical barriers created by CAFs, highlighting their potential as biomarkers and therapeutic targets for cancer immunotherapies based on spatial relationships. Furthermore, this study highlights the beneficial applications of 3D bioprinters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37ca705d8cf5eb81819b79af3fbcdff5bc29b929" target='_blank'>
              Close Spatial Interactions between Cancer Cells and Cancer-Associated Fibroblasts Suppress Antitumor Immunity.
              </a>
            </td>
          <td>
            Y. Naoi, Yumi Inukai, Tomoka Izumikawa, Joji Nagasaki, Takamasa Ishino, Yuki Ueda, Yin Min Thu, Miho Fujiwara, Takahiro Baba, Go Makimoto, K. Suzawa, Kazuhiro Okada, Ken-Ichi Yamamoto, Masakiyo Sakaguchi, Shuta Tomida, Y. Maeda, Shinichi Toyooka, Mizuo Ando, Yosuke Togashi
          </td>
          <td>2025-07-01</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Trastuzumab deruxtecan (T-DXd) is the standard of care for HER2+ or HER2 low metastatic breast cancer (mBC). However, approximately 20% of patients exhibit resistance to T-DXd therapy. As the development of more antibody-drug conjugate (ADC) therapies progresses, understanding ADC resistance mechanisms is critical. Our study aims to elucidate the etiology of ADC resistance, such as characterizing molecular changes and identifying novel biomarkers in tumor-immune interactions between T-DXd responsive (R) and non-responsive (NR) patients with mBC. We employed Nanostring’s GeoMX spatial proteomics and transcriptomics profiling technology to analyze 29 metastatic tissue sections (15R, 14NR) from heavily pre-treated patients with HER2+/low mBC, prior to T-DXd treatment. For each section, 6-8 circular regions of interest (ROIs) with a diameter of 300µm were chosen by researchers blinded to T-DXd R vs. NR outcomes and HER2 scores. ROIs were selected to capture spatially diverse tumor-dense and tumor-sparse regions. Immune markers, stromal markers, and HER2 expression levels were quantified for each ROI. Consistent immune and stromal features related to T-DXd resistance were identified across various metastatic sites, including brain, bone, liver, lymph node, chest wall, and lung. NR tumors were characterized by relative immunosuppression and a disorganized immune response, with consistent upregulation of fibronectin and, surprisingly, granzyme B. Local immune response significantly varied based on HER2 expression. Organ-specific characteristics were also observed: Bone metastases showed an increased correlation between various immune markers and fibronectin in NR, suggesting that fibronectin may modulate immune infiltration and promote a supportive microenvironment for tumor growth. In contrast, chest wall metastases showed loss of correlation between various immune markers and smooth muscle actin in NR. This suggests that a remodeled, less dense smooth muscle environment might reduce physical barriers, potentially allowing better immune cell infiltration and activity in responsive tumors. Furthermore, a subset of these patient samples was selected for spatial whole transcriptomic analysis to gain deeper insights.
 Citation Format: Glori Das, Matthew Vasquez, Jeffrey Zhang, Ji-Hoon Lee, Yuan Gao, Jilun Zhang, Jenny Chang, Xiaoxian Li, Hong Zhao, Stephen T.C. Wong. Spatial Omics Analysis Uncovers Microenvironmental Remodeling and Immune Dynamics in T-DXd Resistant Metastatic Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS4-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c6cc9418499478f1051264c562f281d5a33003b" target='_blank'>
              Abstract PS4-09: Spatial Omics Analysis Uncovers Microenvironmental Remodeling and Immune Dynamics in T-DXd Resistant Metastatic Breast Cancer
              </a>
            </td>
          <td>
            Glori Das, Matthew Vasquez, Jeffrey Zhang, Ji-hoon Lee, Yuan Gao, Jilun Zhang, Jenny Chang, Xiaoxian Li, Hong Zhao, Stephen Wong
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains one of the most lethal malignancies, largely due to its pronounced cellular heterogeneity, therapy resistance, and metastatic propensity, traits driven by cancer stem cells (CSCs). Once thought to be a rare, static subpopulation, CSCs are now recognized as a dynamic, plastic cell state influenced by both intrinsic and extrinsic factors. In this review, we explore the “inner” (genetic, cellular, and metabolic) and “outer” (microenvironmental and macroenvironmental) determinants that govern CSC identity, emergence, and evolution in PDAC. We focus on how acinar and ductal cell plasticity, in combination with oncogenic mutations (e.g., KRAS, TP53), contribute to tumor initiation and CSC generation, how genomic instability further fuels CSC heterogeneity and adaptability, and we highlight how micro and macro TME properties, metabolism and external factors regulate CSC behavior afterwards. Altogether, CSCs exist at the nexus of genetic chaos and environmental cues. Understanding this complex interplay is key to developing therapies that not only eradicate CSCs but also prevent their reemergence. While still far from clinical application, disrupting CSC-supportive networks should be a research and clinical focus for improving outcomes for PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b08e2a3f4aedf4a7111928c29cee984777f2e319" target='_blank'>
              Inner and Outer Determinants of Pancreatic Cancer Stem Cells
              </a>
            </td>
          <td>
            J. C. López-Gil, Diego Navarro, B. Sainz
          </td>
          <td>2025-06-16</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/503cca6deb2a91fe468e2c781a6fa64738c62b22" target='_blank'>
              Drug resistance and tumor heterogeneity: cells and ensembles
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-05-31</td>
          <td>Biophysical Reviews</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="Liver diseases encompass a wide range of etiologies and involve highly heterogeneous cellular environments, yet the specific cellular states through which genetic risk contributes to disease remain incompletely understood. In this study, we integrated genome-wide association study (GWAS) data from six liver diseases and two metabolic traits with transcriptomic profiles of approximately 168,000 human liver cells at single-cell resolution, using the single-cell disease relevance score (scDRS) approach. Our results revealed that disease-associated genetic signals are predominantly localized to non-parenchymal cells—particularly liver sinusoidal endothelial cells (LSECs), cholangiocytes, and specific subsets of lymphocytes. Notably, we identified marked intra-cell-type heterogeneity, with disease associations confined to specific subpopulations exhibiting immune activation or stress-responsive transcriptional programs. For example, autoimmune and viral liver diseases were linked to immunologically active LSECs and cholangiocytes, whereas their metabolically active counterparts showed no enrichment. These findings highlight the necessity of resolving liver cell complexity to uncover the functional basis of genetic risk and suggest that susceptibility to liver disease is driven by specialized cell states within broader cellular categories. Our study provides a refined cellular map of liver disease susceptibility, offering new perspectives for understanding pathogenic mechanisms and informing targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/192d524a339350f75aa0e4bcaad1a36c461732e3" target='_blank'>
              Dissecting the Cellular Heterogeneity Underlying Liver Diseases Through the Integration of GWASs and Single-Cell RNA Sequencing
              </a>
            </td>
          <td>
            Miao Zhou, Meng Liu, Chao Xue
          </td>
          <td>2025-06-27</td>
          <td>Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Compared with other malignancies, brain tumors rank among the most fatal types of cancer affecting humans, with the lowest survival rates. The overall 5-year survival rate for all primary brain tumors is approximately 5%, although this figure varies significantly on the basis of factors such as tumor type, location, size, patient age, and overall health. Glioblastoma, in particular, is among the most aggressive lethal cancers, making it one of the most devastating brain tumors. Its molecular pathogenesis is highly complex and involves genetic mutations in key regulatory genes, such as IDH1/2, EGFR, PDGFRA, the hTERT promoter, and NF1, along with epigenetic alterations and contributions from the tumor microenvironment, all of which drive tumor progression and invasion. Addressing the extremely poor survival rate of glioblastoma requires a deeper understanding of its origins and the cells responsible for its initiation. Various cancer initiation theories—such as the two-hit hypothesis, random mutation model, and clonal selection hypothesis—have proven experimentally effective in explaining its development. However, the cancer stem cell hypothesis stands out, as it successfully accounts for glioblastoma’s unmanageable aggressiveness and recurrence. Glioblastoma cancer stem cells (GSCs) can self-renew, differentiate into multiple tumor cell types, resist conventional therapies, and contribute to tumor heterogeneity, metastasis, and infiltration into surrounding brain tissue. These characteristics significantly impact patient prognosis and mortality. The exact cellular origin of glioblastoma remains a subject of ongoing research. However, studies suggest that glioblastoma may arise from neural stem cells (NSCs) or glial precursor cells, which undergo oncogenic mutations that drive uncontrolled proliferation. The molecular subtype of a tumor is often influenced by the lineage of its originating cells, with mesenchymal stromal cells emerging as potential glioblastoma-initiating cells, particularly in tumors with mesenchymal molecular features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/116bed5615564a1347c07b174110d33b09b16bd8" target='_blank'>
              Origins of Glioblastoma: Initiation and Molecular Signatures of Extreme Aggressiveness
              </a>
            </td>
          <td>
            A. Tawfeeq, Esraa Jaafar Saheb, Bdoor Staar Abbas
          </td>
          <td>2025-06-04</td>
          <td>Iraqi Journal of Cancer and Medical Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="This study aims to investigate metabolic reprogramming heterogeneity in hepatocellular carcinoma (HCC) cells and identify novel therapeutic targets for HCC treatment. Single-cell RNA sequencing data from public databases were used to analyze the TME of HCC and reveal the characteristics of different cell subsets, including mononuclear phagocytes, epithelial cells, endothelial cells, NK/T cells, B cells, and unknown cells. The analysis revealed that these cell subsets play their own unique roles in tumor progression and immune escape. Analysis of copy number variations (CNVs) was performed on tumor-derived epithelial cells, with the epithelial cells in Cluster 3 subgroup showing the highest CNV levels. Gene Ontology (GO) enrichment analysis revealed that these cell subsets were involved in a variety of biological processes such as immune response, cell communication, and metabolic pathways, which were consistent with their functional roles. Pseudotemporal analysis further delineated the malignant trajectory of HCC cells, with Cluster 3 exhibiting enhanced phosphatidylinositol metabolism, suggesting a critical role for metabolic reprogramming in tumor invasion and proliferation. Furthermore, a diagnostic model incorporating metabolic reprogramming-associated gene signatures was established, which effectively distinguished HCC from normal tissues. Among these signatures, splicing factor 3a subunit 3 (SF3A3) was identified as both diagnostic and independent prognostic biomarker. Mechanistically, SF3A3 knockdown in HCC cell lines significantly suppressed proliferation, migration, PI3K/AKT signaling, and EMT marker expression, thereby demonstrating its role in driving HCC aggressiveness. In conclusion, these findings elucidate novel molecular characteristics of HCC based on metabolic reprogramming, while establishing SF3A3 as a promising multi-faceted target for HCC diagnosis, prognostic assessment, and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c37f071868689d97b4d0d4b6a9e7cefb9b68e3b5" target='_blank'>
              Exploring the level of metabolic reprogramming and the role of prognostic factor SF3A3 in hepatocellular carcinoma through integrated single-cell landscape analysis
              </a>
            </td>
          <td>
            Wanshuo Wei, Yuan Gan, Xindan Zhang, Yumo Chen, Zengfeng Huang, Shuhan Wang, Xiaomei Xie, Yongle Li, Pengtao Qin, Lihe Jiang
          </td>
          <td>2025-05-27</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Macrophages play a critical role in shaping the immune landscape of various diseases, with secreted phosphoprotein 1 (SPP1)-expressing macrophages emerging as a distinct subset implicated in both cancerous and non-cancerous conditions. Leveraging recent advances in single-cell RNA sequencing, numerous studies have identified SPP1+ macrophages across diverse pathological contexts, shedding light on their functional heterogeneity. In cancer, SPP1+ tumour-associated macrophages contribute to tumour growth, angiogenesis, and immune evasion, often interacting with T cells and stromal components to sustain an immunosuppressive microenvironment. Conversely, in non-cancerous diseases, these macrophages exhibit both profibrotic and disease-promoting properties, depending on the tissue context. This review provides a comprehensive synthesis of the latest findings on SPP1+ macrophages, highlighting their roles in tumour progression, immune suppression, tissue remodelling, and fibrosis. By comparing their shared traits and tissue-specific differences, we explore how SPP1+ macrophages adapt to distinct microenvironments and influence disease progression. Understanding their conserved and context-dependent functions may open new avenues for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2f870e69c0e0a9df1074bd00e100f8379d0713" target='_blank'>
              The Landscape of SPP1+ Macrophages Across Tissues and Diseases: A Comprehensive Review.
              </a>
            </td>
          <td>
            Alessandro Palma
          </td>
          <td>2025-05-21</td>
          <td>Immunology</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00377ff7110d150be1ad2ae02040cd414655a3e" target='_blank'>
              Protocol for spatial characterization of ECM collagen-GAG in CRC tumor microenvironment
              </a>
            </td>
          <td>
            Libby A. Boykin, Sneha Jayashankar, Khidr Kishan K. Budhwani, Brahma Mubarak K. Budhwani, Diya Samal, Chelsea L. Crawford, Allan Tsung, Karim I. Budhwani
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Spatial transcriptomics, an increasingly prominent technique, has been extensively utilized to examine tumors within the digestive tract (such as liver and colorectal cancers) and the nervous system. However, its application in prostate cancer research remains comparatively limited. This article provides a detailed overview of the principles and features of spatial transcriptomics, particularly highlighting its applications in studying the tumor microenvironment, heterogeneity, and clinical implications in prostate cancer. Through a systematic review and analysis of current literature, we identify the main focus areas and limitations of existing research on spatial transcriptomics and suggest potential future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0488613515f6c6175fd5e7d9f3f6457ebab77dff" target='_blank'>
              Applications and prospects of spatial transcriptomics in prostate cancer research: A narrative review
              </a>
            </td>
          <td>
            Yiling Jin, Zhiming Bai, Gang Wang, Yu Zhang, Jing Chen
          </td>
          <td>2025-05-23</td>
          <td>Current Urology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) typically develops without symptoms, and its aggressive progression combined with late-stage diagnosis underscores the critical need for improved early detection strategies. Circulating tumor cells (CTCs) in blood are potential biomarkers for PDAC. In this study, the detection of pancreatic cancer-associated CTCs was evaluated using two magnetic-based diagnostic systems in a comparative approach. Methods: Two distinct nanotheranostic platforms were developed: monoclonal antibody-conjugated magnetic nanoparticles (mAbs-MNPs) and magnetized exosomes (termed Magxosomes). Anti-mesothelin and anti-vimentin were used as monoclonal antibodies, while mesenchymal stem cells (MSCs) treated with MNPs served as the source of Magxosomes. Characterization of nano-systems was performed using dynamic light scattering (DLS), inductively coupled plasma mass spectrometry (ICP-MS), nanoparticle tracking analysis (NTA), and transmission electron microscopy (TEM). Blood samples were collected from pancreatic cancer mouse models, treated with nanotheranostic platforms, and analyzed using a homemade magnetic particle spectroscopy (MPS) device. Results: The detection of pancreatic cancer-associated CTCs was investigated using nanotheranostic platforms alongside an MPS instrument. In this context, the mAbs-MNPs systems demonstrated varying efficiencies in the diagnosis of CTCs, with Ant-V-MNPs (anti-vimentin conjugated MNPs) achieving 27.47%, Ant-M-MNPs (anti-mesothelin conjugated MNPs) at 13.59%, and a 50:50 mixture of Ant-M-MNPs: Ant-V-MNPs showing an efficiency of 19.73%. Conversely, the efficiencies of Magxosomes were notably higher. Bone marrow stem cell (BMSC)-derived Magxosomes achieved an average efficiency of 63.39%, while adipose-derived stem cell (ADSC)-derived Magxosomes exhibited an average efficiency of 56.23%. Conclusions: This study introduces a promising method for early pancreatic cancer diagnosis by detecting CTCs in blood. It employs a non-invasive, rapid test using an advanced MPS instrument (1 ng detection limit) and nanotheranostic platforms. Results confirm the system's robustness in identifying pancreatic cancer CTCs. This approach may support future developments in cancer diagnostics and monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9225b1261c51594e1c95548c864c314288fdcee" target='_blank'>
              Advanced Detection of Pancreatic Cancer Circulating Tumor Cells Using Biomarkers and Magnetic Particle Spectroscopy
              </a>
            </td>
          <td>
            Ali Dinari, Hafiz Ashfaq Ahmad, Seungjun Oh, Yun-Hee Kim, Jungwon Yoon
          </td>
          <td>2025-06-12</td>
          <td>Nanotheranostics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction The Tandem Duplicator Phenotype (TDP) is a prominent genomic metric prevalent in non-small cell lung cancer (NSCLC). Methods Multi-omics samples including DNA copy number, transcriptomics, somatic single-nucleotide variations, clinical data, and cell-line drug sensitivity from TCGA and CCLE are delved into TDP characteristics. Results Our thorough analysis reveals that patients with smaller span sizes TDP have a more intricate genome, moderate pathology, enhanced therapeutic response, favorable prognosis, a more active immune system, and higher drug sensitivity. In contrast, those with larger span sizes TDP exhibited opposite trends. Discussion The integrated analysis underscores that an in-depth molecular assessment of TDP can offer invaluable insights into its role in cancers. Classification strategy of TDP could recognize the chemotherapy sensitivity of different NSCLC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cb4599c86d747f344f255c2fef020d8615b2cf" target='_blank'>
              Comprehensive multi-omics analysis of tandem duplicator phenotypes in non-small cell lung cancer
              </a>
            </td>
          <td>
            Jie Li, Xinyan Pan, Yanting Bi, Jian Huang, Jun Peng, ChunLing Bai, Jianhong Hou, Yang Chen
          </td>
          <td>2025-06-04</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive brain tumor with limited treatment options and prognosis due to its highly invasive nature and therapy resistance. Investigating the nanomechanical and pathophysiological properties of GBM cells will shed light on tumor behavior. In our study, we investigated the mechanical properties, migration dynamics, and cytoskeletal organization of T98G and U87 MG glioblastoma cell lines using in vitro techniques: atomic force microscopy (AFM) and electric cell-substrate impedance sensing (ECIS). While U87 MG cells are Temozolomide (TMZ)-sensitive and exhibit increased susceptibility to cell death and growth inhibition, T98 cells exhibit improved survival and repair in response to TMZ therapy. This study found that T98G cells are rougher, stiffer, and more viscous, while U87 MG cells are smoother, more elastic, and less viscous, leading to distinct cellular migration patterns. Such differences indicate GBM cell heterogeneity and have consequences for tumor development and resistance to treatment. The key differences of the nanomechanical, viscoelastic, and migratory properties between T98G and U87 MG cells, demonstrated in this work will help us gauge the diverse effects of different dosages of radiation along with immunotherapeutic agents, identifying the ideal radioimmunotherapy option.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ba2cc0d72ec2de612a1d7d825b152f96e3640f" target='_blank'>
              Exploring the heterogeneity in glioblastoma cellular mechanics using in-vitro assays and atomic force microscopy
              </a>
            </td>
          <td>
            Nabila Masud, Md. Hasibul Hasan Hasib, Bayode Ibironke, Charlotte Block, Jayce Hughes, Andrew E. Ekpenyong, Anwesha Sarkar
          </td>
          <td>2025-06-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lineage plasticity is critical for tumor progression and therapy resistance, but the molecular mechanisms underlying cell identity shifts in cancer remain poorly understood. In lung adenocarcinoma (LUAD), the loss of pulmonary lineage fidelity and acquisition of alternate identity programs converge on hybrid identity (hybrid ID) states, which are postulated to be key intermediates in LUAD evolution and are characterized by the coactivation of developmentally incompatible identity programs within individual cells. Here, we uncover a previously unrecognized role for the gastrointestinal transcriptional regulator HNF4α in driving tumor growth and hybrid ID states in LUAD. In LUAD cells expressing the lung lineage specifier NKX2-1, HNF4α induces a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also disrupts NKX2-1 genomic localization and dampens pulmonary identity within hybrid ID LUAD. We show that this hybrid ID state is maintained by sustained RAS/MEK signaling. Inhibition of the RAS/MEK signaling cascade augments NKX2-1 chromatin binding at pulmonary-specific genes and induces drug resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion sensitizes LUAD cells to KRASG12D inhibition. Collectively, our data show that coexpression of opposing lineage specifiers is a novel mechanism of identity dysregulation in LUAD that influences both tumor progression and response to targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e47763b8688fbd0471d5f59c0d98d63dd2cf51e" target='_blank'>
              Opposing lineage specifiers induce a protumor hybrid identity state in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Kayla T. O'Toole, Rushmeen Tariq, Anna Waters, Katherine Gillis, Eric L. Snyder
          </td>
          <td>2025-06-12</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The breast peritumor microenvironment (pTME) is increasingly recognized as a mediator of breast cancer progression and treatment resistance. However, if and how growth-induced tumor compressive forces (i.e., solid stresses) influence the breast pTME remains unclear. Here we show using instant fluorescence lifetime imaging microscopy (FLIM)—a frequency-domain FLIM system capable of simultaneous image acquisition and instantaneous data processing—that breast tumor-mimicking in vitro compression promotes metabolic changes in stromal cells found in the breast pTME. Namely, compression shifts NIH3T3 fibroblasts and differentiated 3T3-L1 (d3T3-L1) adipocytes toward a more glycolytic state, while it promotes increased oxidative phosphorylation in 3T3-L1 undifferentiated adipocytes. The gold-standard Seahorse extracellular flux assay fails to capture these changes, underscoring the superior sensitivity of instant FLIM in detecting metabolic shifts. We validate these phenotypic findings at the transcriptomic level via RNA sequencing, confirming that compressed fibroblasts downregulate oxidative phosphorylation and upregulate glycolysis compared to uncompressed controls. We further demonstrate that compression induces mitochondrial dysregulation in undifferentiated adipocytes, driven in part by upregulated mitophagy and disrupted fusion dynamics. Finally, we confirm that these stromal cell types recapitulate these distinct metabolic states in human breast cancer patient samples, consistent with our in vitro findings. By elucidating mechano-metabolic interactions occurring at the tumor-host interface, these results will inform the development of innovative mechano-metabolic reprogramming treatment strategies to improve breast cancer patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d792df55ee5bac5edf40597207282a8a899c162" target='_blank'>
              Instant fluorescence lifetime imaging microscopy reveals mechano-metabolic reprogramming of stromal cells in breast cancer peritumoral microenvironments
              </a>
            </td>
          <td>
            Julian Najera, Hao Chen, Bianca Batista, Frank Ketchum, Aktar Ali, P. Zorlutuna, Scott Howard, Meenal Datta
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Isocitrate dehydrogenase wild-type glioblastoma, a malignant brain tumour of glial origin, confers a poor prognosis with a median survival of 12 to 16 months from diagnosis. Glioblastomas are aggressive tumours that rapidly proliferate and diffusely infiltrate surrounding brain tissue. Current multimodal standard treatment is typically ineffective and despite gross total surgical resection, tumours recur with more aggressive sub-clonal populations of malignant cells. A defining characteristic of glioblastoma is its highly heterogeneous nature and acquirement of somatic mutations advantageous to tumour growth and suppression of apoptotic pathways. Pathogenesis of malignant brain tumours as well as its mode of transformation to a more aggressive subtype is still largely unknown. Although genomic studies have elucidated a plethora of genetic markers associated with glioblastoma subtypes, only a few have been utilised in a clinical setting. One of the emerging approaches to studying glioblastomas is by investigating how an active proteome contributes to its aggressive nature. Furthermore, through activation of specific pathways via post-translational modifications of proteins such as phosphorylation, glioblastomas create an intricate network of signalling pathways which favour tumour growth and proliferation. Here, we investigated the feasibility of diverse methodological approaches to describe abnormal protein signalling across distinct intra-tumour regions of primary glioblastoma tissue, including proliferative core, peripheral rim, and invasive margin. Whilst we observe a broadly comparable proteome relative to the human non-diseased brain, we identify cytoplasmic proteins α-trypsin, actin, apolipoprotein A1 and transthyretin which may putatively be associated with the GBM infiltrative tumour margin. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-09228-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26cc570a29b72848055078fed6cd3945f46575d9" target='_blank'>
              Regional profiling reveals a distinct glioblastoma infiltrative margin proteome
              </a>
            </td>
          <td>
            A. Kocoń, Stuart J. Smith, B. Morentin, L. F. Callado, Wayne Carter, Ruman Rahman
          </td>
          <td>2025-07-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background The PCDH17 family has emerged as a prominent research focus in the field of oncology; however, the precise mechanism underlying the regulatory role of PCDH17 in shaping the inflammatory tumor microenvironment in pancreatic cancer remains elusive. Method A thorough examination was carried out to explore the presence of PCDH17 in cancerous tumor tissues and its association with genes related to inflammation. Additionally, we comprehensively examined the association between PCDH17 and immune cell infiltration as well as immunotherapy in pancreatic cancer. Moreover, the single-cell data of pancreatic cancer was utilized for analyzing PCDH17 expression, cell differentiation, and intercellular communication. Result PCDH17 exhibited differential expression across various tumor types. The low-expression group of PCDH17 showed reduced levels of inflammatory genes. Immunoinfiltration analysis indicated a significant association between T-cell infiltration and the expression of PCDH17. Analysis of single-cell sequencing data revealed that PCDH17 was primarily expressed in endothelial cells, with a decrease in expression observed during cellular differentiation trajectory. Notably, inflammation-related genes also displayed significant changes in their expression patterns along the endothelial cell differentiation trajectory. Cellular communication investigations unveiled multiple signaling pathways through which endothelial cells interacted with T cells. The presence of PCDH17 in endothelial cells was verified through various immunofluorescence techniques, and a simultaneous decline in its levels was observed alongside the decrease in inflammatory factors within the tumor microenvironment. Conclusion The predominant expression of PCDH17 was observed in endothelial cells, exhibiting a strong association with inflammatory genes and infiltration of immune cells. PCDH17 exhibits potential as a target for regulating the immune-suppressive tumor microenvironment in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06b73ddbf21e245d86c9b823def7478b1068908c" target='_blank'>
              Single-cell and transcriptomic analyses reveal the role of PCDH17 in the non-inflammatory tumor microenvironment of pancreatic cancer
              </a>
            </td>
          <td>
            Yong Sun, Hong Wan, Jie Xiong, Kun Cao, Dashuai Yang, Jun Huang
          </td>
          <td>2025-05-23</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Chemoresistance remains a major therapeutic challenge in high-grade serous ovarian cancer (HGSOC). Our study demonstrates that the functional restoration of death-associated protein kinase 1 (DAPK1), a novel tumor suppressor in HGSOC, induces potent tumor-specific cytotoxicity and inverts chemoresistance. Using molecular and preclinical analyses, we showed that full-length and truncated DAPK1 variants efficiently restore apoptotic signaling pathways and significantly enhance platinum-based and taxane treatment sensitivity. Therapeutic efficacy was validated across different model systems, including ovarian cancer cell lines and patient-derived organoids. Remarkably, the truncated DAPK1 constructs maintain strong apoptotic activity despite structural modifications, highlighting clinical potential. These findings suggest DAPK1 reactivation as a viable strategy to overcome treatment resistance in HGSOC, with the potential for translation into combination therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c04d0909c04866431b8c4c91c2104812e72b5aa2" target='_blank'>
              Truncated DAPK Variants Restore Tumor Suppressor Activity and Synergize with Standard Therapies in High-Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Monika Raab, Khayal Gasimli, Balázs Győrffy, Samuel Peña-Llopis, Sven Becker, M. Sanhaji, K. Strebhardt
          </td>
          <td>2025-06-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771ef0b5061a9c5213ccca9fb3801ba5c0ea8357" target='_blank'>
              Whole Exome Sequencing and Single-Cell DNA Sequencing for Assessment of Clonal Heterogeneity and Evolution in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Sadiksha Adhikari, S. Eldfors, Esa Pitkänen, Markus Vähä-Koskela, C. Heckman
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Testicular germ cell tumors (TGCTs) are the most common malignancy in young men, exhibit a unique developmental origin and exceptional chemosensitivity. However, the molecular distinctions between TGCT subtypes remain poorly understood. Here we present a comprehensive genomic analysis of 252 treatment-naive primary TGCTs, integrating deep whole-genome sequencing with matched transcriptomic and epigenomic data. We identify new driver genes and uncover defining features of TGCTs, including pervasive chromosome X amplification with subtype-specific X chromosome inactivation, and a germ cell-like transcriptional program. Although previously reported, whole genome doubling (WGD) in TGCTs is further characterized here as ubiquitous, developmentally early, and associated with age at onset. Seminomas are enriched for early driver mutations, secondary WGD events, sustained XIST expression and replication stress-associated indel mutational signatures, while non-seminomas show greater structural complexity, subclonal diversity, relatively earlier-onset WGD, extended tumor latency, and telomere elongation. Moreover, we identify a mutational signature, SBS87, that is exceptionally rare across cancers with exception of thiopurine-treated leukemia, but strikingly prevalent in TGCT, especially non-seminomas. SBS87 is linked to extended tumor latency and telomere elongation, implicating possible environmental or endogenous processes that mimic thiopurine-induced DNA damage in TGCT pathogenesis. Collectively, our findings define TGCTs as molecularly distinct tumors shaped by early genomic instability and highlight SBS87 as a novel mutational footprint with potential etiologic and clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797d0b2769352db6f37defdb43fa1da2c4040fb1" target='_blank'>
              A Thiopurine-like Mutagenic Process Defines TGCT Subtypes
              </a>
            </td>
          <td>
            Kevin Brown, Jun Zhong, Adriana Morales Miranda, Mengyan Zhang, Joycelyn Williams, Jacob Williams, Haoyu Zhang, Cheng Liang, Wenbo Li, Bin Zhu, Stephen J Chanock, K. Nathanson, Tongwu Zhang
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cfe224e356c6e68ceccf6ec7ce94377f472868" target='_blank'>
              KLF5 controls subtype-independent highly interactive enhancers in pancreatic cancer to regulate cell survival
              </a>
            </td>
          <td>
            Thomas L. Ekstrom, Zhangshuai Dai, Julia Thiel, Meghana Manjunath, Nadine Schacherer, Bishakha Joyeeta Saha, Yara Souto, A. Abdelrahman, Frank Essmann, Sven Beyes, M. Truty, Meng Dong, Steven A. Johnsen
          </td>
          <td>2025-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Glioblastoma is a highly aggressive brain tumor with an overall poor prognosis due to its immunosuppressive tumor microenvironment (TME). Microglia and tumor-associated macrophages (TAMs) with pro-tumorigenic properties are dominant populations of immune cells in the glioblastoma TME. To date, several studies targeting TAMs to fight tumor progression in different tumor entities have been initiated. However, the impact of standard therapy schemes of glioblastoma cells on macrophage polarization, activation, and phagocytosis remains controversial. The same applies to the relevance of PD-1/PD-L1 blockade in the interaction between macrophages and tumor cells. Our study, therefore, investigated patient-oriented treatment of GLIOBLASTOMA by examining the phagocytic capacity of polarized M1- and M2-like macrophages using GL261-luc2 tumor cells as a preclinical model system. Additionally, we analyzed the expression of activation and immune checkpoint markers on these macrophage subtypes following contact with tumor cells and their microenvironment. These factors were also determined after PD-1 blockade was initiated. The analyses revealed that the immunoregulatory M2-like macrophages generally exhibited a higher phagocytosis rate than the pro-inflammatory M1-like macrophages; however, this was not influenced by the pretreatment of glioblastoma cells with chemo- or radiotherapy. This could not be improved by blocking the PD-1 receptor. Furthermore, there were no modulations in the expression of differentiation, activation, or immune checkpoint molecules of M1- and M2-like macrophages after cell-to-cell contact with glioblastoma cells. But the medium conditioned by tumor cells strongly altered M1-like macrophages toward a more activated state, whereas M2-like cells were only mildly influenced. This was further enhanced by tumor cell treatment, with the most prominent effect after irradiation. These results suggest that conventional GLIOBLASTOMA tumor cell treatment affects the immunogenic status of macrophage subtypes, which is relevant for enhancing the anti-tumor immune response in brain tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47dbb5ff93613986d34e432ca007b413cfe59cb3" target='_blank'>
              Chemoradiation-Altered Micromilieu of Glioblastoma Cells Particularly Impacts M1-like Macrophage Activation
              </a>
            </td>
          <td>
            Mona Shojaei, Benjamin Frey, F. Putz, R. Fietkau, Udo S. Gaipl, A. Derer
          </td>
          <td>2025-07-08</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Heterogeneity in cancer gene expression is typically linked to genetic and epigenetic alterations, yet post-transcriptional regulation likely influences these patterns as well. However, the quantitative contribution of post-transcriptional mechanisms to cancer transcriptome dynamics remains unclear. Here, we systematically measured mRNA dynamics across diverse breast cancer models, revealing that mRNA stability significantly shapes gene expression variability. To decipher the regulatory grammar underlying these dynamics, we developed GreyHound, an interpretable multimodal deep-learning framework integrating RNA sequence features and RNA-binding protein (RBP) expression. GreyHound identified an extensive network of RBPs and their regulons underlying variations in mRNA stability. Among these, we uncovered a metastasis-suppressive regulatory axis centered on the RNA-binding protein RBMS3 and its post-transcriptional control of the redox regulator TXNIP. Functional and molecular analyses revealed that RBMS3 depletion resulted in targeted transcript destabilization, which was associated with poor clinical outcomes and enhanced metastatic potential in xenograft models. Using in vivo epistasis studies, we confirmed that RBMS3 regulation of TXNIP mRNA stability drives this metastasis-suppressive program. These findings position the RBMS3-TXNIP regulatory axis as a key post-transcriptional mechanism in breast cancer and illustrate how interpretable models of RNA dynamics can uncover hidden regulatory programs in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a157de4d34fa592e866b9d4ddb110508f65adde6" target='_blank'>
              Integrative analysis of mRNA stability regulation uncovers a metastasis-suppressive program in breast cancer
              </a>
            </td>
          <td>
            Heather Karner, Tabea Mittmann, Vicky W. Chen, Ashir A. Borah, Andreas Langen, Hassan Yousefi, L. Fish, Balyn W. Zaro, A. Navickas, Hani Goodarzi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95bb823c01319c257e9b78eae2e9bd2984b27b68" target='_blank'>
              CRISPR activation screens map the genomic landscape of cancer glycome remodeling
              </a>
            </td>
          <td>
            John Daly, Lidia Piatnitca, Mohammed Al-Seragi, Vignesh K Krishnamoorthy, S. Wisnovsky
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Tumor Treating Fields (TTFields) therapy is an approved cancer treatment modality, based on non-invasive application of electric fields to the tumor region. Proteomic and cell biology methods revealed a versatile mechanism of action to be involved in the response to TTFields. In the current research we performed whole transcriptome analysis across tumor types to identify pan-cancer responses to TTFields. For this we collected samples from control and TTFields-treated human cancer cell lines of gastric cancer, pancreatic cancer, ovarian cancer, non-small cell lung carcinoma, pleural mesothelioma, and glioblastoma. The transcriptomic analysis supported previous reported effects: downregulation of pathways associated with cell cycle, cell growth, and proliferation; downregulation of DNA replication and the FA-BRCA DNA repair pathway; and upregulation of cellular responses to stress—senescence, autophagy, and apoptosis. Notably, previously unrecognized downstream effects of TTFields were revealed on cellular metabolism, with downregulation of protein and RNA metabolism, and upregulation of steroid biosynthesis. Additional DNA repair pathways were also found to be downregulated, including nucleotide excision repair, base excision repair, and mismatch repair. In conclusion, this study revealed similar response patterns to TTFields across different tumor types, re-enforcing some already pinpointed mechanisms, while revealing new mechanisms. Unlocking these new mechanisms may allow identification of potential new cancer treatments for application together with TTFields based on mechanistical compatibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1627b07ec04103438850967d9453d926d5ec1e3d" target='_blank'>
              The transcriptomic fingerprint of cancer response to Tumor Treating Fields (TTFields)
              </a>
            </td>
          <td>
            K. Wainer-Katsir, A. Haber, H. Fishman, L. Ding, Michael D. Story, Renfei Du, U. D. Kahlert, L. Mannarino, F. Mirimao, M. Lupi, Maurizio D’Incalci, G. Lavy-Shahaf, Hila Ene, R. Frechtel-Gerzi, Zeina Drawshy, A. Martinez-Conde, E. Dor-On, Y. Porat, M. Giladi, U. Weinberg, Y. Palti
          </td>
          <td>2025-07-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b31f4399101f9b6167477a72fd46a9e859a5855" target='_blank'>
              Novel Computational Analysis Identifies Cytotoxic Lymphocyte-to-Monocyte Balance in Tumors as a Predictor of Recurrence-Free Survival in Colorectal Carcinoma.
              </a>
            </td>
          <td>
            Manuel Fernandez, L. Todeschini, Bridget P Keenan, David Rosenberg, Sophia Hernandez, M. Zampese, Guilin Qiao, Tommaso Pollini, A. Maker
          </td>
          <td>2025-06-21</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumor microenvironment is a highly dynamic and complex system where cellular interactions evolve over time, influencing tumor growth, immune response, and treatment resistance. In this study, we develop a graph-theoretic framework to model the tumor microenvironment, where nodes represent different cell types, and edges denote their interactions. The temporal evolution of the tumor microenvironment is governed by fundamental biological processes, including proliferation, apoptosis, migration, and angiogenesis, which we model using differential equations with stochastic effects. Specifically, we describe tumor cell population dynamics using a logistic growth model incorporating both apoptosis and random fluctuations. Additionally, we construct a dynamic network to represent cellular interactions, allowing for an analysis of structural changes over time. Through numerical simulations, we investigate how key parameters such as proliferation rates, apoptosis thresholds, and stochastic fluctuations influence tumor progression and network topology. Our findings demonstrate that graph theory provides a powerful mathematical tool to analyze the spatiotemporal evolution of tumors, offering insights into potential therapeutic strategies. This approach has implications for optimizing cancer treatments by targeting critical network structures within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80685bafd98ff8bdecc99e2498fa961cf364fcad" target='_blank'>
              Stochastic Graph-Based Models of Tumor Growth and Cellular Interactions
              </a>
            </td>
          <td>
            J. A. Rodrigues
          </td>
          <td>2025-05-29</td>
          <td>AppliedMath</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Improved biomarkers of treatment response are needed for patients with high-grade serous ovarian cancer (HGSC). A challenge is substantial anatomical site-to-site variation in expression. We completed data-independent acquisition–mass spectrometry (DIA-MS) analysis of 404 fresh frozen and 78 formalin-fixed, paraffin-embedded HGSC tissue samples from the ovary (adnexal) and a common secondary site (omentum) in 11 patients. This was compared with mutation testing, gene expression, and whole-genome copy number profiling. Proteins with relatively stable intra- and variable inter-individual expression (n = 1651), included a 52-protein module reflecting interferon-mediated tissue inflammation, indicative of a cGAS-STING pathway cytosolic double-stranded (ds) DNA response. The dsDNA sensing/inflammation score was higher in the omentum compared with the ovary. Ovarian HGSC samples showed marked inter-individual differences in inflammatory and immune responses to DNA damage. Stable discriminative features of the HGSC proteome, a prerequisite for clinical predictive biomarkers, are detectable in ovary (adnexal) tissue samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98b411dbc827e64e39c2f022da354cc6d36a3a9c" target='_blank'>
              Overcoming intra-tumoral heterogeneity for biomarker discovery in the high-grade serous ovarian cancer proteome
              </a>
            </td>
          <td>
            Srikanth S. Manda, Maiken M Espersen, C. Mapagu, N. Bouantoun, J. Boros, Y. Chiew, S. Srirangan, Swetansu Pattnaik, Catherine J Kennedy, A. Brand, D. W. Garsed, A. Pandey, D. Bowtell, Natasha Lucas, Dylan Xavier, S. Mahboob, Daniel Bucio-Noble, B. Tully, P. Hains, Phillip J. Robinson, Qing Zhong, R. Reddel, A. Defazio, R. L. Balleine
          </td>
          <td>2025-06-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Liquid biopsy through the analysis of circulating tumor DNA (ctDNA) is emerging as a powerful and non-invasive tool complementing tissue biopsy in lymphoma management. Whilst tissue biopsy remains the diagnostic gold standard, it fails to detect the molecular heterogeneity of the tumor’s multiple compartments and poses challenges for sequential disease monitoring. In diffuse large-B-cell lymphoma (DLBCL), ctDNA facilitates non-invasive genotyping by identifying hallmark mutations (e.g., MYD88, CD79B, EZH2), enabling molecular cluster classification. Dynamic changes in ctDNA levels during DLBCL treatment correlate strongly with progression-free survival and overall survival, underscoring its value as a predictive and prognostic biomarker. In Hodgkin’s lymphoma, characterized by a scarcity of malignant cells in tissue biopsies, ctDNA provides reliable molecular insights into tumor biology, response to therapy, and relapse risk. In primary central nervous system lymphoma, the detection of MYD88 L265P in ctDNA offers a highly sensitive, specific, and minimally invasive diagnostic option. Likewise, in aggressive T-cell lymphomas, ctDNA supports molecular profiling, aligns with tumor burden, and shows high concordance with tissue-based results. Ongoing and future clinical trials will be critical for validating and standardizing ctDNA applications, ultimately integrating liquid biopsy into routine clinical practice and enabling more personalized and dynamic lymphoma care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd65b74500ff54371d5e5843eec530ff6762698" target='_blank'>
              Liquid Biopsy in B and T Cell Lymphomas: From Bench to Bedside
              </a>
            </td>
          <td>
            Mohammad Almasri, Nawar Maher, B. Al Deeban, N. M. Diop, R. Moia, G. Gaidano
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="We aimed to investigate the genomic and immune microenvironmental characteristics and their prognostic value in limited-stage small cell lung cancer (SCLC). Whole exome sequencing and multiplex immunofluorescence analysis were conducted on 38 patients diagnosed with limited-stage SCLC. The two most frequently mutated cancer-related genes observed were RB1 (73.68%) and TP53 (63.16%). However, none of the cancer-related genes, including RB1 and TP53, were associated with prognosis. Furthermore, genomic factors such as tumor mutation burden, copy number instability, and mutant-allele tumor heterogeneity were unrelated to prognosis. Approximately 52.63% (20/38) of cases exhibited PD-L1 expression (combined positive score > 1). The average percentage of CD8-positive tumor-infiltrating lymphocytes (TILs) was 3.27%, with a range spanning from 0.04 to 18.96%. Survival analyses showed that PD-L1 positivity, a high proportion of CD8-positive TILs, and wild-type PI3K pathway were significantly associated with better survival. A predictive prognostic model was further developed based on these three biomarkers, resulting in more accurate stratification of patients according to disease-free survival (DFS, hazard ratio (HR) = 2.020, P < 0.001) and overall survival (OS, HR = 2.344, P < 0.001). Moreover, PD-L1 negative patients who did not undergo adjuvant chemotherapy exhibited significantly improved OS (P = 0.029) and a favorable trend in DFS (P = 0.053) compared to those who underwent adjuvant chemotherapy. In conclusion, this study analyzed the genomic and immune microenvironment characteristics of limited-stage SCLC and constructed a prognostic model based on PD-L1 expression, CD8-positive TILs, and PI3K pathway mutation, which may potentially contribute to the clinical management of limited-stage SCLC. Clinical trial number: Not applicable. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02925-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1642c4c568485d495eeb0f5b92e0e687c477a0a" target='_blank'>
              Genomic and immunological profiling reveals novel prognostic biomarkers for limited-stage small cell lung cancer
              </a>
            </td>
          <td>
            Ye Li, Xiaolong Ma, Zhiyi Wan, Pengfei Bao, Enli Zhang, Lihua Liu, Shunchang Jiao, Sheng-jie Sun
          </td>
          <td>2025-06-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Immunogenic cell death (ICD) is crucial for cancer development and enhances the effectiveness of immunotherapy by triggering an adaptive immune response. However, the impact of ICD-related genes (ICDRGs) on iCCA progression and patient prognosis remains unclear. iCCA samples from The Cancer Genome Atlas (TCGA) were categorized into two ICD-associated subtypes by consensus clustering, and patient prognosis in these subgroups was assessed. Single-cell RNA sequencing was conducted on viable cells from 8 iCCA patients to understand their transcriptomic profiles, heterogeneity, and immune microenvironment. Principal component analysis (PCA) and uniform manifold approximation and projection (UMAP) analysis were applied to identified 13 cell subpopulations. The gene set enrichment analysis (GSEA) was used to investigate pathway heterogeneity among these subpopulations. CellChat was employed for intercellular communication analysis. Pseudotime trajectory analyses with CytoTRACE and Monocle2 were used to explore differentiation trajectories and functional differences across various states. 30,485 single cells from human iCCA and normal tissues were sequenced, revealing two ICD-associated subtypes. The ICD-low subtype correlates with better clinical outcomes, whereas the ICD-high subtype is linked to increased immune cell infiltration and immune response signaling activity. Distinct biological functions and metabolic characteristics were observed in iCCA between high- and low-ICD groups. High infiltration of ICD-low subtype in iCCA is associated with poor prognosis, potentially through the ANXA-FPR pathway affecting macrophage proliferation and differentiation. Our study constructed an immunogenic cell death-related signature that can promise prognosis prediction and personalized medicine in iCCA. This finding might help guide clinicians for iCCA patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08602-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e45d84f628dac0dca7f3bfd929def33d5b1582" target='_blank'>
              Single cell RNA sequencing to identify an Immunogenic cell death related prognostic signature in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Bowen Sha, Fei Tong, Qinghua Shu, Miao Wu, Liang Zhao, Yufeng Zhang
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8969c8aec90d4da293cc00fb23b1d52373b13cc" target='_blank'>
              Macrophage activation determines muscle wasting in pancreatic cancer.
              </a>
            </td>
          <td>
            Chia-Jung Chang, Po-Hsien Huang, Szu-Ying Chen, Pei-Chia Su, Li-Yun Ding, Ya-Chin Hou, Miao-Neng Hung, Kuan-Lin Chen, Ya-Na Wu, Shang-Rung Wu, F. Lin, Peiqi Lu, Hsin-Yi Wu, Yu‐Ju Chen, Chih-Han Chang, Chih-Peng Chang, Daw-Yang Hwang, Yi-Ching Wang, Yanmei Shan
          </td>
          <td>2025-05-29</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal cancers, with an average survival time of only six months following diagnosis, even with currently available therapies. Thus, PDAC represents a significant therapeutic challenge, necessitating a deeper understanding of its biology and tumor microenvironment (TME) to develop more effective treatments and improve patient outcomes. Here, we report that the expression of Eukaryotic Elongation Factor-2 Kinase (eEF2K) is associated with shorter patient survival and demonstrate that eEF2K signaling is critical for the PDAC tumor growth and regulated by the TME. Furthermore, in vivo targeted genetic inhibition of eEF2K suppressed tumor growth in two different PDAC mouse models, reduced tumor-associated macrophages (TAMs), and induced marked apoptosis in tumor tissues without any signs of toxicity. Our data suggest that eEF2K knockdown diminishes the activity of the AXL receptor tyrosine kinase and reduces the expression of macrophage-derived factors, such as Monocyte Chemoattractant Protein-1 (MCP1), along with the Gas6/AXL signaling pathway in PDAC cells. Additionally, analysis of the NCI-TCGA PDAC patient database further showed that eEF2K expression, in the presence of TAM markers, correlates with even shorter patient survival. TAM-released factors, such as MCP1, Gas6, and exosomes, induce eEF2K expression in PDAC cells, as well as the activity of AXL, SRC, VEGF, Snail, and MMP2, contributing to epithelial-to-mesenchymal transition (EMT), invasion, metastasis, and angiogenesis. In conclusion, our findings reveal for the first time that eEF2K is a critical oncogenic driver of PDAC tumor growth and thus targeting eEF2K represents a promising and novel therapeutic strategy for PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca2a458a943c8611c08480c527e7133171db725" target='_blank'>
              eEF2K is a poor prognostic factor and novel molecular target in pancreatic cancer: regulating tumor growth and progression via the tumor microenvironment
              </a>
            </td>
          <td>
            Didem Karakaş, Ahmed Ashour, H. Mokhlis, N. Kahraman, R. Bayraktar, S. Dilmac, N. Kabil, M. Erdoğan, E. Dere, E. Ulukaya, B. Ozpolat
          </td>
          <td>2025-07-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cancer cells undergo widespread changes in epigenetic patterns that mediate cancer compromised gene expression programs during cancer progression. However, the alterations in higher-order genome organization in which these changes occur and their functional implications are less well understood. To explore how chromatin structure and epigenetic parameters of genome architecture changes during cancer progression at a fine scale and genome-wide, we generated high-resolution Micro-C contact maps in non-malignant, pre-cancerous, and metastatic MCF10 breast cancer epithelial cells. We profiled progression-associated reorganization of chromatin compartments, topologically associated domains (TADs), and chromatin loops, and also identified invariable chromatin features. We find large-scale compartmental shifts occur predominantly in early stages of cancer development, with more fine-scale structural changes in TADs and looping accumulating during the later transition to metastasis. We related these structural features to changes in gene expression, histone marks, and potential enhancers and found a large portion of differentially expressed genes physically connected to distal regulatory elements. While changes in chromatin loops were relatively rare during progression, differential loops were enriched for progression-associated genes, including those involved in proliferation, angiogenesis, and differentiation. Changes in either enhancer-promoter contacts or distal enhancer activity were accompanied by differential gene regulation, suggesting that changes in chromatin contacts are not necessary but can be sufficient for gene regulation. Together, our results demonstrate a functionally relevant connection between gene regulation and genome remodeling at many key genes during cancer progression. KEY FINDINGS - The cancer genome is reorganized throughout cancer progression at the level of compartments, chromatin domains, and loops - Compartmental shifts occur in early stages of cancer development, with more fine-scale structural changes accumulating during metastasis - Chromatin domain boundaries are weakened during cancer progression - Many progression-regulated genes exhibit changes in distal enhancer histone modifications that are bridged by stable chromatin loops - Changes in enhancer activity or subtle changes in chromatin contacts can rewire enhancer-promoter connections to facilitate changes in gene expression - Prominent changes in chromatin loops occur at a small subset of differentially regulated genes during progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f24eeb9036338ab76b344cd3bcd26b76e9bfec03" target='_blank'>
              Genome reorganization and its functional impact during breast cancer progression
              </a>
            </td>
          <td>
            K. S. Reed, Andrew J. Fritz, Haley Greenyer, Kerstin Heselmeyer-Haddad, Seth Frietze, Janet Stein, Gary S. Stein, Tom Misteli
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Macrophages comprise a significant portion of the glioblastoma tumor microenvironment and are essential in promoting immunosuppression and tumor progression. Solid tumors such as glioblastoma generate solid stress as they expand, creating a compressive microenvironment for mechanosensitive immune cells including macrophages. Macrophages are known to respond to various mechanical stimuli but have not yet been studied in the context of chronic compression observed in growing tumors. Here, we used a custom in vitro compression system to elucidate the effects of compressive solid stress on murine macrophages. We found that macrophages have significant morphological, transcriptional, metabolic, and functional responses to compression. These changes corresponded to both canonical pro- and anti-inflammatory macrophage states. The gene expression signatures of compressed macrophages more closely resembled those of glioma-associated macrophages known to be associated with worse patient outcomes. These results indicate that compression alone, independent from tumor cell-derived biochemical factors, may contribute to the pathological tumor-associated macrophage phenotype. This could represent a vicious cycle of tumor immunomechanics and mechano-immunology. Targeting solid stress in tumors or the response to solid stress by macrophages may interrupt this feedback loop to help normalize the tumor immune microenvironment and improve glioblastoma response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe1ba849d1b5e1c7454753ad531daa7a5fc2c35" target='_blank'>
              Chronic compression drives macrophages toward a pathological pro-tumor state
              </a>
            </td>
          <td>
            Alice Burchett, Hao Chen, Julian Najera, Scott Howard, Meenal Datta
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a3a93f5939a6e79dedc599d6cfe047baa006cdc" target='_blank'>
              Small extracellular vesicle-miR-183-5p mediated crosstalk between tumor cells and macrophages in high-risk pancreatic neuroendocrine tumors.
              </a>
            </td>
          <td>
            Wuhu Zhang, Junfeng Xu, Dan Huang, Yi Qin, X. Lou, Heli Gao, Zeng Ye, Fei Wang, Yan Wang, Desheng Jing, Guixiong Fan, Q. Zhuo, Chang Xin, Jie Chen, Xiaowu Xu, Xianjun Yu, S. Ji
          </td>
          <td>2025-06-05</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Ploidy determination across the genome has been challenging for low-pass-WGS tumor-only samples. We present BACDAC, a method that calculates tumor ploidy down to 1.2X effective tumor coverage. Allele fraction patterns displayed in the Constellation Plot verify tumor ploidy and reveal subclonal populations. BACDAC outputs a metric, 2N+LOH, that when combined with ploidy better distinguishes near-diploid from high-ploidy tumors. Validated using TCGA, BACDAC had good agreement with other methods and 88% agreement with experimental methods. Discrepancies occur mainly when BACDAC predicts diploidy with subclones rather than high-ploidy. Applied to 653 low-pass-WGS samples spanning 12 cancer subtypes, BACDAC calls 40% as high-ploidy. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03599-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4a5801328a2687dfdef502362bf6b982a0712bd" target='_blank'>
              Tumor ploidy determination in low-pass whole genome sequencing and allelic copy number visualization using the Constellation Plot
              </a>
            </td>
          <td>
            Sarah H. Johnson, J. Smadbeck, Roman M. Ženka, Michael T. Barrett, Athanasios Gaitatzes, Arnav Solanki, Angela B. Florio, M. Borad, John C. Cheville, G. Vasmatzis
          </td>
          <td>2025-05-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="
 Sex differences in breast cancer (BC) lead to distinct clinical and molecular characteristics. Sex-associated heterogeneity relies intricately on the oncogenic properties of cancer cells and multicellular interactions in tumor microenvironments. Here we conducted a multiomic analysis encompassing single-cell RNA sequencing, spatial transcriptomics and large-scale histological analysis, combined with genetic and pharmacological perturbations to systematically develop a high-resolution and spatially resolved map of intratumoral expression heterogeneity in male breast cancer (MBC). Among the malignant cells, we identify 9 consensus meta-programs (MPs, for example, estrogen response and interferon response), each consisting of dozens of genes that are coordinately upregulated in subpopulations of cells within MBC. Of these, MP5 exhibits significant male bias, characterized by aberrant activation of the estrogen receptor α (ERα) signaling pathway, inducing endocrine therapy resistance in MBC. Bidirectional interactions between MP5 malignant cells and immunosuppressive cancer-associated fibroblasts form an immune exclusionary microenvironment mediated by LRBA. Collectively, our study highlights the endocrine therapy resistant and immunosuppressive role of LRBA in MBC, and these findings may provide insights into the molecular features of MBC and facilitate the development of sex-specifical treatment strategies.
 Citation Format: Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, Jingyan Xue, Jiong Wu. Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df96734a9b8fda8be55365f44c01373b9990e46" target='_blank'>
              Abstract P3-02-26: Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer
              </a>
            </td>
          <td>
            Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, J. Xue, Jiong Wu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [14],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>