<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-23 10:04:11 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>103</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a0eaac11594dae27086c42ea3c9abe9629eb38" target='_blank'>
              Spatial multi-omics unveils the monoclonal origin, neuroendocrine plasticity, and microenvironment niches in combined small cell lung cancer
              </a>
            </td>
          <td>
            Zhuo Wang, Qing Luo, Jie Wu, Li Lu, Wenjie Ding, Yichun Zhao, Yongfeng Yu, Ruoran Qiu, Ling Zhu, Xinxing Ouyang, Wendi Xuzhang, Shun Lu, Wei Wei, Qihui Shi, Ziming Li
          </td>
          <td>2026-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1041f2c4ce227782834ad6496b1ab2bc0d63ef0" target='_blank'>
              Detection of Somatic Point Mutations Directly from Spatial Transcriptomics Enables in vivo Spatiotemporal Lineage Tracing
              </a>
            </td>
          <td>
            Zhirui Yang, Mengdie Yao, Qing Yang, Yiheng Du, Jinhong Lu, Xing Wu, Jinran Lin, Zhuoyu Qian, Song Hu, Yonghe Xia, Huan Liu, Qiji Zhou, Xiaoyu Ma, Yan Luo, Wenxuan Fan, Weike Pei, Yuhao Xia, Xiuyan Yu, Jing Luan, Qiaoan Zhang, Yue Zhang, Qin Wang, Jianyang Zeng, Yue Zhang, Wenyu Wu, Yanmei Dou
          </td>
          <td>2026-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICI) have improved clinical outcomes for some patients with advanced NSCLC, however a substantial proportion of patients remain treatment resistant. Here we analyze the NSCLC tumor microenvironment (TME) using multiplexed immunofluorescence (mIF) of biopsies taken from patients prior to ICI treatment. We apply a deep-learning model to classify the cellular phenotypes and probe functional and metabolic states of both tumor and immune cells, aiming to reveal predictive features of response to ICI. Tissue neighborhoods are generated to allow geometric profiling of spatial densities and interactions at a range of scales. Multivariate modelling of ICI response yields a model that predicts progression-free survival (PFS) over 24 months (AUC = 0.8). The selected features in the model imply a role for cell-cell proximities within discrete metabolic contexts. These tissue insights may supplement our understanding of the current paradigms around classical immunology in the NSCLC TME and its influence on immunotherapy outcomes. Immune checkpoint inhibition (ICI) has proven therapeutically valuable in non-small cell lung cancer (NSCLC) and other cancers but treatment failure is still seen. Here the authors use multiplex immunofluorescence to visualise the tumour microenvironment in NSCLC to probe functional and metabolic states and correlate this with survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73a544c51b5b0263f213bbaa933c91d4ecb10eff" target='_blank'>
              Metabolic characterization of tumor-immune interactions by multiplexed immunofluorescence reveals spatial mechanisms of immunotherapy response in non-small cell lung carcinoma (NSCLC)
              </a>
            </td>
          <td>
            J. Monkman, A. Kilgallon, Clara Lawler, Rafael Tubelleza, T. Aung, J. Warrell, Ioannis A. Vathiotis, Ioannis P Trontzas, N. Gavrielatou, Nay Nwe Nyein Chan, Rotem Czertok, S. Bookstein, K. O’Byrne, E. Markovits, D. Rimm, A. Kulasinghe
          </td>
          <td>2026-02-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="SUMMARY Spatial omics transforms our understanding of cancer by revealing how tumor cells and the microenvironment are organized, interact, and evolve within tissues. Here, we synthesize advances in spatial technologies that map tumor ecosystems with unprecedented fidelity. We highlighted analytical breakthroughs—including multimodal integration and emerging spatial foundation models—that resolve functional niches and spatial communities, converting spatial patterns into mechanistic insights. We summarize how spatially organized features, from immune hubs to microbiota and neural interfaces, shape tumor evolution and clinical outcomes. We then outline how spatial approaches illuminate precancer biology, metastatic adaptation, and therapy response. Bridging discovery and translation, we provide a practical roadmap for incorporating spatial readouts into clinically oriented study design. We conclude by discussing persistent challenges in standardization and scalability and how high-plex spatial discoveries may be distilled into scalable, AIenabled, clinically deployable assays, positioning spatial omics as a cornerstone of next-generation predictive and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6fb3dcd04b45926dce1096c02f6d3fedd6c13cb" target='_blank'>
              Spatial omics at the forefront: emerging technologies, analytical innovations, and clinical applications
              </a>
            </td>
          <td>
            Yunhe Liu, Yibo Dai, Linghua Wang
          </td>
          <td>2025-12-01</td>
          <td>Cancer cell</td>
          <td>5</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d63cfbf90a2e544c2d24c8ab605e5d44e8ba24" target='_blank'>
              A Pan-Cancer Single-Cell Compendium of Intratumoural Heterogeneity
              </a>
            </td>
          <td>
            Ido Nofech-Mozes, A. Cook, Tom W. Ouellette, Sara Hafezi-Bakhtiari, Philip Awadalla, Sagi Abelson
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are the main constituents of the tumor microenvironment. Several studies have delineated CAF heterogeneity in different types of tumors, however, it is still unknown how the distinct CAF transcriptional profiles are established during tumor progression. We reanalyzed a previously published single-cell RNA-sequencing dataset of MMTV-PyMT tumors at higher resolution using Seurat , and CytoTRACE to characterize CAF subtypes and their differentiation states. Wilcoxon rank sum test was applied for differential gene expression. Multiplex immunostaining (Akoya PhenoImager HT) was performed on 38 murine mammary tumors from MMTV-PyMT mice to identify the distinct CAF subtypes. Whole-slide imaging and spatial analysis were conducted using QuPath and Cellpose , followed by neighborhood clustering and interaction mapping with CytoMAP . Cellular distances from CAFs to immune, tumor, and endothelial cells were quantified using SPIAT and Wilcoxon tests for comparisons. In parallel, human spatial transcriptomics data from the 10X Genomics Xenium platform were integrated for cross-species validation. Here, by single cell RNA-sequencing and multiplex immunostaining, we identify six CAF substates. Spatial analysis on immunostained murine mammary tumors and human spatial transcriptomics data outlined temporal changes in stromal composition and the existence of distinct functional niches enriched with different CAF substates. Immunomodulatory CAFs co-localized with immune cells while myofibroblastic CAFs formed a shield around the tumor core, thus preventing immune infiltration. Our work supports the idea that distinct spatial locations dictate different CAF transcriptional programs. Targeting specific functional niches will ultimately hinder tumor progression by inhibiting signaling between distinct CAF substates and the surrounding tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d802e2b1323bfcdbf953412180c241803eb33a" target='_blank'>
              Spatial and temporal dynamics of cancer-associated fibroblast niches in breast cancer
              </a>
            </td>
          <td>
            Jessica Pantaleo, J. Sjölund, P. Bolivar, M. Bocci, B. Phung, Maria Malmberg, G. Jönsson, Kristian Pietras
          </td>
          <td>2026-01-11</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7a3ed4eb95927059b414292cf9cce743a10321" target='_blank'>
              Spatiotemporal Mapping of Tertiary Lymphoid Structure Heterogeneity Shapes Immune Niches and Clinical Outcomes in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Liaoliao Gao, Jie Mei, Libing Hong, Yuzhi Jin, Jinlin Cheng, Xuqi Sun, Chuan Liu, Bin Li, Xiaomeng Dai, Bo Lin, Yajie Sun, Peng Zhao, Minshan Chen, Rongping Guo, Shan Xin, Jingping Yun, I. Martínez-Reyes, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2551717e0bfdbffaec39c6cc9470da9416da5fa6" target='_blank'>
              Deciphering spatial heterogeneity of patient-derived organoids of colorectal tumors for drug discovery
              </a>
            </td>
          <td>
            Maxim Norkin, Jonathan Bac, Louis McConnell, Christophe Cisarovsky, Marina Alexandre-Gaveta, Sylvie André, Simon Tanaka, Antonia Digklia, Sylke Hoehnel, Stephanie Tissot, Joerg Huelsken, Marianna Rapsomaniki, Raphael Gottardo, K. Homicsko
          </td>
          <td>2026-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f84aada80c780deec8ebaf81d914dc52c85820f2" target='_blank'>
              HEDeST: An Integrative Approach to Enhance Spatial Transcriptomic Deconvolution with Histology
              </a>
            </td>
          <td>
            Luca Gortana, Loïc Chadoutaud, Raphael Bourgade, Emmanuel Barillot, Thomas Walter
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The tumor immune microenvironment (TIME) is closely involved in tumor initiation, malignant progression, immune escape, and response to immunotherapy. With the continued development of high-throughput sequencing technologies, transcriptomic approaches have become essential for examining the cellular and molecular features of the TIME. Bulk RNA sequencing offers tissue-level gene expression profiles and allows the estimation of immune cell composition through computational deconvolution. Single-cell RNA sequencing provides finer resolution, revealing cellular heterogeneity, lineage relationships, and functional states. Spatial transcriptomics (ST) retains the native anatomical context, making it possible to localize gene expression patterns and cell–cell interactions within intact tissues. These approaches, when considered together, have shifted TIME research from averaged measurements toward a more detailed and mechanistic understanding. This review summarizes the principles, applications and limitations of bulk, single-cell and spatial transcriptomic methods, highlighting emerging strategies for integrative analysis. Such multi-scale frameworks are increasingly important for studying immune dynamics and may contribute to the development of more precise biotechnological and immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a95188583167d2d5ed84b42177b48c6d850371a9" target='_blank'>
              Multi-Scale Transcriptomics Redefining the Tumor Immune Microenvironment
              </a>
            </td>
          <td>
            Jing Sun, Yingxue Xiao, Lingling Xie, Dan Qin, Yue Zou, Yingying Liu, Yitong Zhai, Minyi Zhang, Tong Li, Youjin Hao, Bo Li
          </td>
          <td>2026-01-15</td>
          <td>BioTech</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e1d8bc49fbea6d71f89b6e3278200b6554df4b0" target='_blank'>
              Spatial Decoding of Tertiary Lymphoid Structure Maturation in Non-Small Cell Lung Cancer Using Deep Neural Networks
              </a>
            </td>
          <td>
            Bob Chen, Conrad Foo, Alma Andersson, Brandon D. Kayser, Yiming Yang, Alsu Missarova, P. Kulkarni, Jan-Christian Huetter, Sharmila Chatterjee, Youngmi Kim, Yan Liang, Emily Killinbeck, Sarah Murphy, Eloisa Fuentes, J. Giltnane, Raj Jesudason, Bo Li, Aicha BenTaieb, David Richmond, Tommaso Biancalani, Robert J. Johnston, Eric Lubeck, Tyler Risom, O. Rozenblatt-Rosen, Runmin Wei, Lisa M. McGinnis
          </td>
          <td>2026-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5486f6aaee8639d9484243cb88e9c2a7fbf4ba3" target='_blank'>
              Cancer-myeloid cell invasive program in pediatric-type diffuse high-grade glioma
              </a>
            </td>
          <td>
            C. Ruiz-Moreno, R. Collot, T. J. van den Broek, E. J. Wehrens, N. Bessler, Gitanjali Dharmadhikari, B. M. Te Pas, Ignacio L Ibarra, D. Metselaar, M. Kranendonk, E. Hoving, J. van der Lugt, F. Calkoen, A. Zomer, F. Theis, E. Hulleman, D. V. van Vuurden, A. Rios, H. Stunnenberg
          </td>
          <td>2026-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba1256bc5be1c15c5a52165e8dbea8c1e3c75014" target='_blank'>
              Integrative Single-cell and Spatial Transcriptomic Analysis of Osteosarcoma Reveals Conserved and Distinct Ecosystems Across Sites and Species
              </a>
            </td>
          <td>
            Yogesh Budhathoki, Matthew V. Cannon, T. A. McEachron, Anand G. Patel, Matthew J Gust, Jaime F Modiano, D.T. Ammons, Kathryn E Cronise, M. Macchietto, A. Sarver, R. Dries, B. Gryder, Daniel P. Regan, Heather L. Gardner, Ryan D. Roberts
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3333bb3dc7aa46ec7dea00023d761c89be2478c2" target='_blank'>
              Large-scale single-cell analysis and in silico perturbation reveal dynamic evolution of HCC: from initiation to therapeutic targeting.
              </a>
            </td>
          <td>
            Peng Xia, Shuang Si, Daosen Fu, Luyao Liu, Dongliang Yang, Yanan Guo, Yixiao Tian, Pengfei Ji, Xinyi Yuan, Yingxia Tian, Rong Shen, Degui Wang
          </td>
          <td>2026-01-30</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cellular identity and function are linked to both their intrinsic genomic makeup and extrinsic spatial context within the tissue microenvironment. Spatial transcriptomics (ST) offers an unprecedented opportunity to study this, providing in situ gene expression profiles at single-cell resolution and illuminating the spatial and functional organization of cells within tissues. However, a significant hurdle remains: ST data is inherently noisy, large, and structurally complex. This complexity makes it intractable for existing computational methods to effectively capture the interplay between spatial interactions and intrinsic genomic relationships, thus limiting our ability to discern critical biological patterns. Here, we present CellScape, a deep learning framework designed to overcome these limitations for high-performance ST data analysis and pattern discovery. CellScape jointly models cellular interactions in tissue space and genomic relationships among cells, producing comprehensive representations that seamlessly integrate spatial signals with underlying gene regulatory mechanisms. This technique uncovers biologically informative patterns that improve spatial domain segmentation and supports comprehensive spatial cellular analyses across diverse transcriptomics datasets, offering an accurate and versatile framework for deep analysis and interpretation of ST data.w">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ba9e86aac88dd3979a7874807492f91849fe58" target='_blank'>
              Uncovering spatial tissue domains and cell types in spatial omics through cross-scale profiling of cellular and genomic interactions
              </a>
            </td>
          <td>
            R. Yan, Xiaohan Xing, Xun Wang, Zixia Zhou, Md Tauhidul Islam, Lei Xing
          </td>
          <td>2026-02-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/342dd98c58335efcd7b4ec06f157649f394ee9d4" target='_blank'>
              Clinical feasibility of spatial transcriptomics using discarded tissue from diagnostic breast biopsies
              </a>
            </td>
          <td>
            Jeffrey P Sheridan, Quynh Sun, Jiaji G Chen, Yibing Wei, Veronica Jarzabek, Anirudh Jaishankar, Iqra Amin, L. Sultan, Thea Nalan, Sarah B. Mueller, Eric J. Burks, Lubna H. Suaiti, Christopher M. Heaphy, N. Ko, R. Dries
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafddf4eb7b4f74e9d7699b13e08df2afba97c1e" target='_blank'>
              Pan-cancer tumor classification by a holistic tumor microenvironment atlas
              </a>
            </td>
          <td>
            Shishang Qin, Xiao-hui Du, Jinhu Li, Nan Jiang, Tian Diao, Yufei Bo, Qinhang Gao, Liangtao Zheng, Xinnan Ling, Qianqian Gao, Xiangjie Li, Sen Gao, Fei Tang, Wenjie Zhang, Chenwei Li, Peihong Fang, Linna Zhu, Dongfang Wang, Zemin Zhang
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Mucosal melanoma (MuM) is a rare and aggressive cancer with limited response to immunotherapy. Despite its clinical severity, the mechanisms that drive therapeutic resistance in MuM are still largely unclear. To elucidate the spatial and molecular determinants of therapeutic outcomes, we designed this study to identify the spatial and molecular factors that shape both responsiveness and resistance of MuM to immune checkpoint blockade therapies.



 We employed a comprehensive spatial multi-omics strategy that integrated COMET-based single-cell spatial proteomics with high-resolution metabolite imaging via MALDI-IMS. Using this platform, we analyzed the cellular composition and mapped spatially organized cellular neighborhoods (CNs) across 97 FFPE tissue cores obtained from 26 MuM patients who received PD-1/PD-L1 or CTLA-4 blockade. We further examined how spatial architecture, cell–cell interactions, proteomic signatures, and metabolomic profiles of CNs varied between treatment responders and non-responders.



 This integrative strategy enabled spatially resolved characterization of about 700K single cells, which were classified into 25 phenotypic clusters across nine major cellular lineages. Spatial neighborhood analysis identified 15 biologically distinct CNs, each exhibiting unique compositions and spatial arrangements of tumor, immune, and stromal components. Among patients who responded to immunotherapy, three tumor-associated CNs—the central tumor, invasive tumor, and tumor boundary CNs—were markedly enriched, displaying distinctive spatial architecture. Notably, the invasive and boundary CNs showed close spatial proximity among proliferating tumor cells, macrophages, and dendritic cells. These CD163+ macrophages demonstrated decreased IRF4 and Arg1 expression, indicative of reduced immunosuppressive function. In contrast, non-responders showed a predominance of a stromal-rich CN characterized mainly by SMA- stromal cells with limited immune cell infiltration, observed in both baseline and post-treatment samples. Spatial metabolomic analysis further revealed a substantial decrease in tryptophan-derived indole metabolites in responders, which strongly correlated with CD163 and CD11c expression, suggesting a coordinated immunometabolic reprogramming within the tumor microenvironment.



 Collectively, these insights underscore that immunotherapy response in MuM is orchestrated by an interplay of spatial tumor–immune architecture, stromal insulation, and metabolomic reprogramming collectively underpin differential immunotherapy responses in MuM. The spatially resolved signatures of tryptophan-derived metabolites not only emerge as promising biomarkers but also present tractable therapeutic targets to potentially overcome resistance in this aggressive melanoma subtype.



 Suhendan Ekmekcioglu, Jun Wang, Priyadharsini Nagarajan, SungNam Cho, Yunhe Liu, Erin H. Seeley, Jared K. Burks, Adi Diab, Linghua Wang. Spatial multi-omics identifies a tumor microenvironment signature predictive of immunotherapy response in mucosal melanomas [abstract]. In: Proceedings of the AACR Immuno-Oncology Conference (AACR IO): Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2026 Feb 18-21; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2026;14(2 Suppl):Abstract nr A036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9c9bb89b05e89256a692207be2cead562679a20" target='_blank'>
              Abstract A036: Spatial multi-omics identifies a tumor microenvironment signature predictive of immunotherapy response in mucosal melanomas
              </a>
            </td>
          <td>
            S. Ekmekcioglu, Jun Wang, P. Nagarajan, SungNam Cho, Yunhe Liu, Erin H. Seeley, Jared K. Burks, A. Diab, Linghua Wang
          </td>
          <td>2026-02-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Understanding the spatial organization of individual cell types within tissue and how this organization is disrupted in disease, is a central question in biology and medicine. Hematoxylin and eosin-stained slides are widely available and provide detailed morphological context, while spatial gene expression profiling offers complementary molecular insights, though it remains costly and limited in accessibility. Predicting gene expression directly from histological images is therefore an attractive goal. However, existing approaches typically rely on small image patches, limiting resolution and the ability to capture fine-grained morphological variation. Here, we introduce a deep learning approach that predicts single-cell gene expression from morphology, matching patch-based methods on spot level prediction tasks. The model recovers biologically meaningful expression patterns across two cancer datasets and distinguishes fine cell populations. This approach enables molecular-level interpretation of standard histological slides at scale, offering new opportunities to study tissue organization and cellular diversity in health and disease. Predicting gene expression from H&E slides is a cost-effective alternative to Spatial Transcriptomics for clinical use. Here the authors introduce a Multiple Instance Learning approach to infer single-cell expression, unlike prior methods which operate at coarser patch level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d903aacfbf3a6dfc2c450d8cb95280e66a5321f" target='_blank'>
              sCellST predicts single-cell gene expression from H& E images
              </a>
            </td>
          <td>
            Loïc Chadoutaud, Marvin Lerousseau, Daniel Herrero-Saboya, Julian Ostermaier, Jacqueline Fontugne, Emmanuel Barillot, Thomas Walter
          </td>
          <td>2026-01-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/126eb3bb80a65b589c6798760093749f010ab57b" target='_blank'>
              Robust characterization and interpretation of rare pathogenic cell populations from spatial omics using GARDEN
              </a>
            </td>
          <td>
            Xinming Zhang, Zhuohang Yu, Gaoyang Hao, Qi Yao, Yanmei Hu, Fuzhou Wang, Xingjian Chen, Linjing Liu, Ka-chun Wong, Xiangtao Li
          </td>
          <td>2026-01-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be57704ac8a67d3c2570360452cb391401b265a" target='_blank'>
              A Comprehensive Treatment-Induced Resistance Atlas of Glioblastoma Reveals a Fibrotic Niche Shielding the Tumor from Immunotherapy
              </a>
            </td>
          <td>
            Fei Wang, Run Huang, Hongyi Ling, Yuhan Bai, Chen Yang, Guozhen Zhao, Xin Wu, Wenqiang Cao, Yue Lu, Yining Zhang, Deng-Feng Lu, Youjia Qiu, Juyi Zhang, Bixi Gao, Chao Ma, Hanhan Dang, Yulian Zhang, Yanbo Yang, Ting Sun, Zhouqing Chen, Zhong Wang
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Understanding how prostate cancers evolve toward treatment resistance requires experimental systems that both recreate the phenotypic diversity of the disease and resolve lineage dynamics at high resolution. We developed complementary platforms to meet these challenges. The Prostate Mouse Platform (ProMPt) is a genetically scalable organoid- and allograft-based system encompassing over 150 combinatorial genotypes, enabling reconstruction of the principal phenotypic states of prostate cancer, from androgen-responsive tumors to castration-resistant double-negative, hybrid/mixed, and neuroendocrine-like variants. Through multi-omic and functional analyses, ProMPt models capture the context-specific vulnerabilities that arise during adaptation to androgen deprivation or growth-factor loss. To map and track this evolving phenoscape, we designed XCODE, a universal, multiplatform barcoding technology that enables clonal tracing across sequencing-, cytometry-, and imaging-based modalities (CyTOF, Phenocycler, Xenium). XCODE integrates lineage information across bulk, single-cell, and spatial layers, linking histology, clonal dynamics, phenotype, and microenvironmental context. Together, ProMPt and XCODE provide a powerful preclinical framework to reconstruct, resolve, and perturb tumor evolution within a controlled yet clinically relevant landscape. This integrated approach reveals how defined genetic combinations reshape lineage plasticity, signaling networks, and therapeutic responses, exposing new intervention points to prevent or redirect adaptive evolution in advanced prostate cancer.



 Marco Bezzi. From reconstruction to resolution: platform approaches to engineer the prostate cancer phenoscape and map its evolution [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A002.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3245085c50a80831901ee082ba85bf39b096b337" target='_blank'>
              Abstract A002: From reconstruction to resolution: platform approaches to engineer the prostate cancer phenoscape and map its evolution
              </a>
            </td>
          <td>
            Marco Bezzi
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Although well-differentiated thyroid carcinoma (WDTC) is characterized by a robust treatment response, aggressive subtypes, such as anaplastic thyroid carcinoma (ATC), remain highly lethal. To understand thyroid cancer evolution in both children and adults, we analyzed single-cell transcriptomes of 423,733 cells from 81 samples and spatially resolved key tumor and microenvironment populations across 28 tumors with spatial transcriptomics, including rare and unique composite WDTC/ATC tumors and pediatric diffuse sclerosing thyroid carcinomas. Additionally, we identified gene signatures of stromal cell populations in 5 large thyroid cancer bulk RNA-sequencing cohorts. Through this multi-institutional effort, we defined a population of POSTN+ myofibroblast cancer-associated fibroblasts (myCAFs) that are intimately associated with invasive tumor cells and correlate with poor prognosis, lymph node metastasis, and disease progression in thyroid carcinoma. We also revealed a population of inflammatory CAFs that are distant to tumor cells and are found in the inflammatory stromal microenvironment of autoimmune thyroiditis. Together, our study provides spatial profiling of thyroid cancer evolution in samples with mixed WDTC/ATC histopathology and identifies a prognostic myCAF subtype with potential clinical utility in predicting aggressive disease in both children and adults.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9674219bbd66cbdd5e33b2f92b59cb54d20e94cd" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations
              </a>
            </td>
          <td>
            M. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey P. Caroland, Megan L. Tigue, H. Hartmann, J. Gallant, Courtney J. Phifer, Andres A Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H. Belcher, K. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2026-01-09</td>
          <td>JCI Insight</td>
          <td>2</td>
          <td>47</td>
        </tr>

        <tr id="Motivation Single cell RNAseq (scRNAseq) is an ideal tool to characterize the heterogeneity within the tumor microenvironment, however, accurate identification of tumor cells can be a challenge. Reference-based methods can be inaccurate, if reference datasets are even available. Current purpose-built methods can be inaccurate, particularly with highly heterogeneous tumor types. Improved methods are needed. We explored the use of genetic variants to distinguish tumor from normal cells within scRNAseq data. Results We characterized the limitations inherent to calling variants from scRNAseq data, quantifying how data sparsity precludes genetic distance calculation between single cells. As a novel workaround, we pooled data from transcriptionally similar cell clusters to call high quality variants and then calculated pairwise differences between cell populations and performed hierarchical clustering. We quantified confidence in genetic divergence between tumor and normal cell populations using bootstrapping. We performed extensive validation to assess accurate identification of tumor cells using ground-truth datasets. Application of our method to human scRNAseq samples highlighted the utility of our approach and revealed how mutational burden influences successful tumor cell identification. Improved cell type assignment in scRNAseq data will facilitate analysis of tumor samples and, in turn, accelerate our understanding of the mechanisms underlying tumor progression and reveal potential biological vulnerabilities that can be exploited to develop improved treatment options. Availability and implementation Our method is publicly available as an R package: SCANBIT (Single Cell Altered Nucleotide Based Inference of Tumor) https://github.com/kidcancerlab/scanBit.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9bcdfb11e48000089460046e024824475367747" target='_blank'>
              SCANBIT facilitates identification of tumor cell populations in scRNAseq data using pseudobulked SNV calls
              </a>
            </td>
          <td>
            MV Cannon, MJ Gust, AC Gross, M. Cam, JB Reinecke, L. Jimenez Garcia, CH Strawser, L. Ryan, M. Sammons, CZ Zhang, R. Roberts
          </td>
          <td>2026-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma is heterogeneous, with low tumor purity, a prominent microenvironment, and complex architecture, which preclude the identification of shared tumor-intrinsic and stromal biology within and across patients. We overcame these challenges by achieving necessary resolution and context through the application of complementary genomics, pathology, and machine-learning approaches to characterize primary untreated tumors from 39 patients. We captured 340,000 spatial low-bulk and 530,000 spatial single-cell transcriptomes and observed a spectrum of classical-to-basal tumor subtypes present within all patients. We found that each subtype has distinct regulators, stromal neighborhoods, microenvironment, extracellular matrix, and histology corresponding to multiple immunosuppressive and therapy resistance mechanisms. We defined key tumor heterogeneity features, including the presence of mixed KRAS mutations and tertiary lymphoid structures, identifying biomarkers that distinguish the latter from lymph nodes. Lastly, by leveraging patient, cell, and mouse data, we determined which aspects of tumor biology are recapitulated in bulk datasets and reductionist models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dfa5ca299bf4b962d7809782c25bc48f5abb0f7" target='_blank'>
              In situ multi-modal characterization of pancreatic cancer reveals tumor cell identity as a defining factor of the surrounding microenvironment.
              </a>
            </td>
          <td>
            Anna Lyubetskaya, Brian Rabe, Andrew Kavran, Yulong Bai, Andrew Fisher, Alba Font-Tello, Anne Lewin, Hannah Pliner, Yunfan Fan, Lauren Giampapa, Yelena Cheng, Chao Dai, Ruifeng Hu, Tom Lila, Alexandre P. Alloy, Mike Mason, Constance Brett, Todd Brett, Fayaz Seifuddin, Steven Vasquez Grinnell, S. Stahlschmidt, Yilin Zhao, R. Golhar, Isaac Neuhaus, Daniel Carrera, Carlos Rios, P. Kar, Abhishek Shukla, R. Bashford-Rogers, Matthew J. Meyer, Enas Abu Shah, Lara Heij, Shivan Sivakumar, J. Trillo-Tinoco, Benjamin J Chen, Konstantinos J. Mavrakis, Eugene Drokhlyansky
          </td>
          <td>2026-01-01</td>
          <td>Cell reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27e17b141d07dfd3357f3ee906c1f3a2792dd3c" target='_blank'>
              SigDyn: single-cell mutational signature dynamics
              </a>
            </td>
          <td>
            S. Costa, Joana P. Gonçalves
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18e77f40b4649c94b742354f1776a0273d7ae6d5" target='_blank'>
              A Hallmark-Integrated, Agent-Based Framework for Intratumor Heterogeneity in Melanoma Evolution
              </a>
            </td>
          <td>
            Khola Jamshad, Trachette L. Jackson
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ffb9d54339eb6235677bb2a5769c1cdac9c920a" target='_blank'>
              A Pan-Cancer Single-Cell Atlas to Evaluate Tumor Identity, Cell Line Concordance, and Dependency Mapping
              </a>
            </td>
          <td>
            Rosyli Reveron-Thornton, J. Agolia, Chuner Guo, Maria Korah, , Peter Y. Xie, Renceh AB Flojo, Andrea E. Delitto, Amanda Gonçalves, Angela D. Tabora, Michael Janusyzk, , Kevin Nee, Biren Reddy, Wesley Bobst, Byrne Lee, George A. Poultsides, Amanda R. Kirane, Derrick C. Wan, Jeffrey A. Norton, , Aaron M. Newman, Michael T. Longaker, Deshka S. Foster, Daniel Delitto
          </td>
          <td>2026-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67c0bdeec747ad1c41e7a933728f867f5f704e68" target='_blank'>
              Spatial pharmaco-multiomics reveals drug distribution, metabolic niches, and spatially constrained resistance in medulloblastoma
              </a>
            </td>
          <td>
            Tuan Vo, Cedric Cui, Aitor Benedicto, Yibi Chen, Andrew Causer, Xiao Tan, Eun Ju Kim, Hani Vu, Albert Xiong, Mark Hodson, Ya-Feng Deng, Ana Maria Romero Jimenez, Ning Liu, Brett R. Hamilton, Thomas Robertson, Laura A. Genovesi, Brandon J. Wainwright, Jenny Fung, John Lee, Quan Nguyen
          </td>
          <td>2026-01-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71c46aa519f5755b995d1d19efbf9695d85981f7" target='_blank'>
              Molecular Profiling and High-Content Drug Screening of Metastatic Colorectal Cancer Organoids Reveal Evolutionary Mechanisms of Pan-KRAS Inhibitor Resistance
              </a>
            </td>
          <td>
            Gizem Damla Yalcin, Kübra Yılmaz, Raffaella Klima, O. Kurtulan, Duriye Ozer Turkay, O. Aktepe, A. B. Doğrul, Cenk Sokmensuer, V. Oter, E. B. Bostancı, Javier Fernandez-Mateos, Şuayib Yalçın, Luca Braga, Ahmet Acar
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), a highly lethal disease often developing in the context of chronic liver disease, is marked by high relapse rates and low 5‐year survival. To investigate the multicellular ecosystem and molecular features of hepatocarcinogenesis and progression, a comprehensive analysis integrating single‐cell and spatial transcriptomics was conducted. Significant alterations were observed in various cell types, notably an increase in liver parenchymal and endothelial cells, which play critical roles in tissue repair and angiogenesis. Scissor+ cells, predominantly hepatocytes, were identified as potentially evolving into malignant cells with poor prognosis and enhanced immune evasion capabilities. Key genes associated with Scissor+ cells exhibited potential as cancer prognostic markers. Additionally, the disruption of cellular communication networks during malignant progression revealed the evolution of hepatocytes into cancerous phenotypes. The enhanced signaling pathways, such as MDK‐SDC4 and COL4A‐SDC, were identified as pivotal in driving the malignant transformation of hepatocytes. This transformative process, wherein Scissor+ cells acquire malignant features, highlights novel mechanisms of cancer progression. These findings provide deeper insights into hepatocarcinogenesis and offer potential avenues for targeted and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea05c9d911e5acfb02ab472396d64afacfb09aa" target='_blank'>
              Spatiotemporal Evolution of Malignant Hepatocyte Clones Unveils Immune Evasion Features and Therapeutic Vulnerabilities in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Shizheng Xiong, Dekang Ren, Jiaming Jin, Linjie Zhao, Chengjun Gong, Jiarui Liu, Wanjie Zheng, Zhiheng He, Haochuan Guo, Li Guo, Tingming Liang
          </td>
          <td>2026-01-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Neoadjuvant PD-1/PD-L1 blockade yields robust efficacy in advanced cutaneous squamous cell carcinoma (cSCC), yet many patients fail to achieve a complete or major pathologic response. The reasons why some patients experience response but others do not are unclear. Methods We profiled cSCC specimens before, after 1 dose, and after 3–4 doses of PD-1/PD-L1 blockade to uncover resistance mechanisms and predict therapeutic response. In total, 27 patients across three cohorts, including two phase II trials, were studied. We created 1.7 mm tissue-core microarrays and performed single-cell spatial transcriptomics, including spatial clustering, gene-set enrichment, and spatial correlation analyses. Results After profiling all samples, six distinct spatial niches emerged, each differentially enriched in responders versus non-responders. A high antigen presentation niche, B/plasma cell enriched niche, and inflammatory keratinocyte niche were more frequent in responders, whereas proliferative keratinocyte, low antigen presentation myeloid, and fibroblast-rich epithelial–mesenchymal transition niches prevailed in non-responders. Notably, spatial niche profiling on pretreatment samples outperformed PD-L1 status in predicting pathologic response. Each niche displayed unique gene coexpression modules, suggesting niche-specific resistance mechanisms. Individual tumor analyses revealed varied immune evasion strategies, including defective interferon-induced antigen presentation, immunosuppressive myeloid environments, and epithelial–mesenchymal transition. Conclusions Our single-cell spatial transcriptomic approach identifies six spatial niches that predict immunotherapy response better than PD-L1 status using only 1.7 mm tissue cores and may inform the development of biomarkers. Our results further underscore the heterogeneity of resistance mechanisms among cSCC patients, highlighting the need for tailored therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf68ddba765459abfda326b7a403a032b5df3da" target='_blank'>
              Single-cell spatial transcriptomics uncovers niches that govern response to PD-1/PD-L1 blockade in cutaneous squamous cell carcinoma
              </a>
            </td>
          <td>
            Maxwell Y Lee, Alistaire Sherman, Luis Martinez Ramirez, Mobeen Rahman, Sabrina Zdravkovic, June Ho Shin, Ivan Stepanek, A. D. Colevas, John B. Sunwoo, V. Divi
          </td>
          <td>2026-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Lung cancer remains a significant global health challenge, with metastatic progression being the leading driver of mortality. Organoid technology provides a tractable, physiologically relevant platform to model key aspects of lung cancer metastasis in vitro. The present review summarized methodologies for constructing and interrogating these models, covering tissue sources, culture modalities, gene editing and in vivo transplantation; applications in studying metastatic mechanisms, drug screening and capturing intra- and intertumoral heterogeneity are also highlighted. Persistent challenges include standardizing derivation and culture conditions, improving preservation of tumor-microenvironmental interactions, expanding immune-competent and vascularized models, and addressing scalability, cost, and regulatory and ethical considerations for clinical translation. Future directions include integrating multi-omics approaches and spatial profiling, leveraging artificial intelligence for image and response analytics, advancing immune-organoid models and establishing shared standards, reference materials and reporting guidelines to enhance reproducibility and clinical impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9831c4c96b7e8c9d326d8d56192940951debd787" target='_blank'>
              Recapitulating lung cancer metastasis in vitro: Advances in organoid models and challenges in clinical translation (Review)
              </a>
            </td>
          <td>
            Jian Jiang, Guomin Dong, Zhiyun Zhang, Xiaoyuan Lei, Fanshu Huang, Suoni Li, Jiequn Ma, Jie Bai, Qian Ge, Hui Wei, Zheng Zhao
          </td>
          <td>2026-01-20</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A fundamental challenge in studying therapy resistance is understanding whether it results from pre-existing cellular states (“priming”) or drug-induced changes (“adaptation”). While lineage barcoding enables retrospective analysis of cells before and after treatment, current methods struggle to efficiently capture rare lineages in single-cell RNA sequencing (scRNA-seq) or isolate multiple specific lineages simultaneously for functional study. To overcome these limitations, we developed CloneSweeper, a multiplexed lineage tracking platform that pools enrichment libraries to isolate or enrich multiple rare lineages. CloneSweeper utilizes a dual-function barcode expressed as both a Cas9 gRNA for live-cell sorting and a 3’ UTR transcript for high-recovery detection in 10x Genomics scRNA-seq. We applied CloneSweeper to a model of BRAF V600E melanoma, where we identified that resistance to targeted therapy emerges from a polyclonal population of rare, pre-existing lineages. By simultaneously targeting and enriching 21 distinct rare lineages prior to treatment, we defined a heritable, primed state characterized by de-differentiation and elevated mesenchymal markers. We demonstrate that these primed cells are not quiescent but instead exhibit upregulated inflammatory and stress response signaling, specifically via the AP-1 and NF-κB1 pathways. CloneSweeper thus provides a robust framework for dissecting the molecular mechanisms of rare biological phenomena through simultaneous, multiplexed lineage isolation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfe1b1a36a95588a97f5cc3024b429ba771c00" target='_blank'>
              Multiplexed enrichment and tracking of lineages with CloneSweeper
              </a>
            </td>
          <td>
            Robert J. Vander Velde, Raymond W. S. Ng, Christopher Coté, Sydney M. Shaffer
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b786943f0e4090faa02f89896c2ed59c657e0d0" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations
              </a>
            </td>
          <td>
            M. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey P. Caroland, Megan L. Tigue, H. Hartmann, Jean-Nicolas Gallant, Courtney J. Phifer, Andres A Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H. Belcher, K. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>47</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains clinically heterogeneous. We integrated single-cell and spatial transcriptomics with explainable machine learning to define a lethal tumor axis and establish an interpretable prognostic model. From 141,986 high-quality single cells spanning localized, hormone-sensitive, and castration-resistant PCa, we identified a malignant C4 epithelial subpopulation characterized by high chromosomal instability, androgen receptor and cell-cycle activation, and stemness potential. Spatial mapping further revealed immune-enriched yet suppressive niches, where fibroblasts and myeloid cells coexisted with exhausted lymphocytes, reflecting functional immune imbalance. We benchmarked 101 machine learning pipelines, selecting a Lasso plus PLS-Cox model that achieved strong concordance across independent cohorts. The C4-based risk score independently predicted recurrence-free survival after adjustment for age, Gleason score and T stage, and a nomogram combining this score with clinical variables showed good discrimination. SHAP interpretation highlighted MT1M, PCSK1N, and ACSL3 as major risk-driving features. PCSK1N was progressively upregulated from normal prostate to castration-resistant disease and promoted proliferation, clonogenicity, migration and enzalutamide resistance, while its inhibition sensitized organoids and xenografts to AR-targeted therapy. These findings define a C4-centered lethal tumor axis and provide an explainable, experimentally supported framework for prognostic stratification in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d360e6a196e9de4b8313a49f5e3b66fe3b338c6" target='_blank'>
              Integrative single-cell and spatial transcriptomics with explainable AI reveal lethal prognostic axis in prostate cancer
              </a>
            </td>
          <td>
            Qintao Ge, Zhenda Wang, Yangyun Wang, Tong Chu, Zhongyuan Wang, Wenkai Zhu, Youzhao Zhang, Yonghao Chen, Dingwei Ye, Wenhao Xu, Zhongyuan Wang, Mierxiati Abudurexiti
          </td>
          <td>2026-01-06</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) technologies are reshaping our understanding of tissue organization and cellular context in health and disease. However, technical benchmarking across platforms remains limited, particularly in formalin-fixed, paraffin-embedded (FFPE) clinical samples, which represent the most common tissue format in oncology. Here, we systematically benchmark five commercial ST platforms (Visium v1, Visium v2/CytAssist, Visium HD, Xenium, and CosMx) using matched FFPE human tumor sections from six cancer types. Uniquely, our study includes both sequencing-based and imaging-based platforms profiled on the same samples, enabling direct technical comparisons across spatial capture modalities. We evaluate platform performance across multiple dimensions, including transcript and UMI detection, gene–histology concordance, cell type recovery, and integration with a targeted protein panel (Visium v2, 30 proteins), enabling spatial multi-omics. We also quantify the impact of sampling strategies and area coverage on cell type estimation, revealing trade-offs in spatial resolution versus tissue context. Notably, we present the first same-sample comparison of Xenium Multi-Tissue (377 genes) and Xenium Prime (5,000 genes), highlighting key differences in transcript recovery and spatial signal despite shared chemistry and imaging infrastructure. Finally, we integrate Visium targeted protein data with matched RNA profiles, uncovering widespread RNA–protein decoupling and spatial heterogeneity in concordance. Collectively, this work provides a harmonized dataset and technical reference for the spatial transcriptomics community, offering insight into the relative strengths, limitations, and design considerations associated with high-throughput spatial profiling of FFPE tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0862d70c416facd7be3c79aadcd298f65bea772a" target='_blank'>
              A technical comparison of spatial transcriptomics platforms across six cancer types
              </a>
            </td>
          <td>
            Sergi Cervilla, D. Grases, Elena Pérez, Francisco X. Real, E. Musulén, Julieta Aprea, M. Esteller, Eduard Porta-Pardo
          </td>
          <td>2026-01-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Rationale: Esophageal squamous cell carcinoma (ESCC) is a highly aggressive malignancy. The metastasis and poor prognosis of ESCC are closely associated with tumor microenvironment (TME) heterogeneity, which is driven by epithelial-mesenchymal transition (EMT). Clinically, how to diagnose and target EMT progression remains a key challenge for ESCC. Methods: Integration of pathological images and bulk RNA sequencing profiles identified a high-risk subtype exhibiting EMT enrichment and immunosuppression. Single-cell and spatial transcriptomics revealed EMT macrostates and their spatial distribution. The role of CACNA1C in programming malignant phenotype was tested in vitro. A pathological image-based deep learning model successfully predicted the spatial expression distribution of CACNA1C, indicating possible clinical utility. Results: EMT progression comprised three macrostates: the early state (high epithelial and metastatic potential), the stable state (hybrid E/M phenotype and high stemness), and the late state (high mesenchymal and invasive propensity). ITGA3 and ITGB4 antagonistically regulate malignant phenotype in the early state. Notably, suppression of CACNA1C induced transdifferentiation from stable/late-state cells to normal epithelium-like cells. Conclusions: This study provides novel insights into the EMT mechanism in ESCC, proposes an intervention strategy, and emphasizes the promising clinical application of pathological images in EMT assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2bd67040364633d082b64fbfe1d8f04e033d05" target='_blank'>
              Pathogenomic analysis reveals clinically relevant epithelial-mesenchymal plasticity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Ruzhen Chen, Chenyi Xie, Ziyu Ning, Meng Yang, Zezhuo Su, Jiahui Chen, Kunheng Du, Yihuai Hu, Chu Han, Shaojun Zhang, Qingling Zhang, Meng Liu, Zaiyi Liu
          </td>
          <td>2026-01-14</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Invasive lobular carcinoma (ILC) is the second most common histological subtype of breast cancer and displays distinct clinical and biological behavior compared to breast cancer of no special type. However, current molecular classifications largely overlook its complex spatial organization and tumor microenvironment (TME). Here, we performed spatial transcriptomics on 43 hormone receptor-positive, HER2-negative (HR+/HER2-) ILC tumors with detailed morphological annotation and long-term clinical follow-up. By integrating spatial gene expression with histology and single-cell deconvolution, we characterized the composition and architecture of the TME and revealed high inter- and intratumor heterogeneity. Spatial clustering uncovered cell populations and pathways linked to clinical outcome. We then developed a multimodal classification of ILC by integrating gene expression, morphology, and spatial metrics, identifying four distinct subtypes: normal/stroma-enriched (NSE), proliferative (P), androgen receptor-enriched (ARE), and metabolic/immune-enriched (MIE). These subtypes, collectively termed ILC4TME, reflect the interplay between tumor and microenvironmental features. Gene signatures derived from the spatial data enabled subtype assignment in external bulk RNA-seq and microarray datasets (SCAN-B, METABRIC), revealing reproducible biology and significant associations with survival. In multivariable models, ILC4TME retained prognostic value beyond established gene signatures and clinicopathological variables. Notably, the P subtype was linked to poor prognosis, even in patients treated with endocrine therapy alone, while the NSE subtype was associated with favorable outcomes. Our findings uncover spatial and cellular heterogeneity in ILC that is not captured by existing classification approaches, offering a refined framework for risk stratification and therapeutic targeting based on tumor microenvironment architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/048f8386277c3de4abe3f625fd3318ada9b08489" target='_blank'>
              Spatial transcriptomics reveals tumor microenvironment-driven subtypes of invasive lobular carcinoma.
              </a>
            </td>
          <td>
            M. Serra, Mattia Rediti, L. Collet, F. Lifrange, D. Venet, N. Occelli, A. Papagiannis, D. Vincent, G. Rouas, D. Larsimont, M. Vikkula, F. Duhoux, F. Rothé, C. Sotiriou
          </td>
          <td>2026-02-04</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3083560b7513f7f17aefe85af32208b0613ecab1" target='_blank'>
              Cellular neighborhoods in cancer
              </a>
            </td>
          <td>
            Lichun Ma, Barbara Xiong, Meng Liu, Kai Tan
          </td>
          <td>2026-01-01</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d29acb3217c08dff3be7e7d47281f950232e5a4" target='_blank'>
              Identification of an epigenetically and phenotypically distinct peritumoral glioblastoma cell population linked to inferior patient outcome
              </a>
            </td>
          <td>
            I. Neves, Xi Lu, Veera Jokinen, Francesco Latini, N. Maturi, A. Sundström, Yonglong Dang, Irem Uppman, Tobias Bergström, Pengwei Xing, M. Ryttlefors, Xingqi Chen, F. Swartling, L. Uhrbom
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Gastroenteropancreatic neuroendocrine tumors (GEP-NETs) are clinically heterogeneous malignancies whose biology and microenvironmental organization remain poorly understood. Here, we integrated single-nucleus multiomic (snRNA-seq and snATAC-seq) and spatial transcriptomic profiling across 38 well-differentiated pancreatic (pNET) and small-intestinal (siNET) tumors to define conserved malignant programs, their regulatory circuits, and spatial niches. We observed two conserved malignant cell programs spanning a continuous transcriptional spectrum: a neuronal-like program (si-cNMF1/p-cNMF1), and a secretory neuroendocrine program (si-cNMF2/p-cNMF2). Matched chromatin accessibility profiles uncovered distinct, tissue-specific regulatory networks, including MAX::MYC and MITF transcription factor binding motifs in siNETs versus ISL1 and TFAP4 in pNETs, indicating organ-specific epigenetic control. Spatial transcriptomic analyses revealed that si/p-cNMF1-high regions localized to high cell density, immune-rich tumor areas, whereas si/p-cNMF2-high regions occupied stromal and vascularized niches and co-occured with fibroblast and endothelial compartments enriched for TGFB1-ITGB1, VEGFA-FLT1, and LAMA2-ITGA1 signaling. Across both tumor types, the cNMF2 program was enriched in metastatic lesions and was enrichedfor pro-fibrotic and pro-angiogenic gene signatures. Thus, GEP-NETs are organized along a conserved neuronal-to-secretory axis defined by distinct epigenetic programs and spatially coupled to specific microenvironmental niches. This framework unifies NET heterogeneity across organ sites and identifies pathway-specific, microenvironment-linked vulnerabilities for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f5c183623da01cf434288165ef5499fec974dff" target='_blank'>
              Conserved Neuronal-like and Secretory Programs Define the Spatial Architecture of Gastroenteropancreatic Neuroendocrine Tumors
              </a>
            </td>
          <td>
            J. Karam, Samantha E. Hoffman, Amanda E. Garza, Dan Y Gui, Hannah I. Hoffman, Breanna M. Titchen, Yutaro Tanaka, Erica M Pimenta, T. Pappa, Laura Valderrábano, Kevin Bi, R. Gillani, Lauren Brais, E. Shannon, Jason L. Hornick, Jihye Park, Jennifer Chan, E. V. Van Allen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the leading causes of cancer-related mortality worldwide. Compared with traditional two-dimensional (2D) models, patient-derived CRC organoids more faithfully preserve the genomic, transcriptomic, and architectural features of primary tumors, making them a powerful intermediate platform bridging basic discovery and clinical translation. Over the past several years, organoid systems have rapidly expanded beyond conventional epithelial-only cultures toward increasingly complex architectures, including immune-organoid co-culture models and mini-colon systems that enable long-term, spatially resolved tracking of tumor evolution. These advanced platforms, combined with high-throughput technologies and clustered regularly interspaced short palindromic repeats (CRISPR)-based functional genomics, have substantially enhanced our ability to dissect CRC mechanisms, identify therapeutic vulnerabilities, and evaluate drug responses in a physiologically relevant context. However, current models still face critical limitations, such as the lack of systemic physiology (e.g., gut–liver or gut–brain axes), limited standardization across platforms, and the need for large-scale, prospective clinical validation. These gaps highlight an urgent need for next-generation platforms and computational frameworks. The development of high-throughput multi-omics, CRISPR-based perturbation, drug screening technologies, and artificial intelligence-driven predictive approaches will offer a promising avenue to address these challenges, accelerating mechanistic studies of CRC, enabling personalized therapy, and facilitating clinical translation. In this perspective, we propose a roadmap for CRC organoid research centered on two major technical pillars: advanced organoid platforms, including immune co-culture and mini-colon systems, and mechanistic investigations leveraging multi-omics and CRISPR-based functional genomics. We then discuss translational applications, such as high-throughput drug screening, and highlight emerging computational and translational strategies that may support future clinical validation and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ed98ccf8eb7ee4178076cc0e4368c14e7e399d9" target='_blank'>
              An Integrative Roadmap for Advancing Colorectal Cancer Organoid
              </a>
            </td>
          <td>
            Youqing Zhu, Ke He, Zhi Shi
          </td>
          <td>2026-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bf612b75850cf10e9bb17c10ec3f99a040c4e2d" target='_blank'>
              Lineage plasticity: a new dilemma in lung cancer treatment
              </a>
            </td>
          <td>
            Cainan Huo, Yongxuan Wang, Rujia Li, Xiaofei Xie, W. Lyu, Dingke Chen, Yanqin Sun
          </td>
          <td>2026-02-04</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Ovarian cancer remains a lethal disease marked by profound therapeutic resistance, largely orchestrated by a complex tumor microenvironment (TME) governed by metabolism-immune crosstalk. This review focuses on the spatiotemporal dynamics of the metabolism-immune axis in ovarian cancer progression and resistance, with particular emphasis on how cutting-edge spatial multi-omics technologies reveal previously unrecognized layers of intratumoral heterogeneity and geographic organization that cannot be captured by bulk analyses. Using tools such as MALDI-MSI, GeoMx DSP, and CODEX, these approaches enable high-resolution, spatially resolved mapping of metabolite gradients (e.g., lactate, lipids, kynurenine), immune cell niches, and immunometabolic checkpoints within distinct tumor regions. Such spatial profiling uncovers how metabolic reprogramming-dysregulated glycolysis, lipid metabolism, and glutaminolysis-drives localized immunosuppression and chemoresistance through compartment-specific interactions among tumor cells, cancer-associated fibroblasts (CAFs), adipocytes, and immune populations. These geographically defined insights reshape our understanding of therapeutic failure and highlight precise, location-aware vulnerabilities. Accordingly, we propose spatially informed therapeutic strategies, including regional glycolysis inhibition, glutaminase blockade, lipid pathway interference, and their rational combination with immune checkpoint inhibitors (ICIs), to disrupt pathogenic metabolic-immune circuits and improve immunotherapy outcomes. Looking ahead, advances in vivo spatial imaging, gut microbiota modulation, and AI-powered integrative multi-omics frameworks promise truly personalized treatment of ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc5b7b1f229dc616042558b3b060c33208889755" target='_blank'>
              Spatial profiling of the metabolism-immune axis in ovarian cancer
              </a>
            </td>
          <td>
            Zhi-Bin Wang, Ming-Hui Long, Ping Yu, Ya-li Wang, Zheng Yang, Marilyn S. Huang
          </td>
          <td>2026-01-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lineage tracing at single-cell resolution is vital for mapping cell fate decisions, yet synthetic barcoding faces limitations in precision, diversity, and toxicity-especially in human pluripotent stem cells (hPSCs). Here, we repurpose naturally occurring somatic mutations in mitochondrial transcripts from single-cell RNA sequencing as endogenous genetic barcodes. By enriching mitochondrial reads and applying a robust computational pipeline, we identified clonally informative variants to trace hematopoietic lineage emergence from hPSCs during early embryogenesis. Integrating mitochondrial barcoding with synthetic lineage tracing, we modeled embryonic tissue development and reconstructed the transcriptional logic and regulatory networks driving fate specification using a dynamical systems model. Extending this approach to spatial transcriptomics, we mapped the clonal architecture of human embryonic organoids, revealing spatial zonation orchestrated by NOTCH-mediated crosstalk between stromal cells and hematopoietic progenitors. This multimodal strategy links clonal dynamics with niche-driven fate decisions, offering a scalable, non-invasive method to dissect tissue organization in development and disease. Together, our work establishes a scalable, non-invasive multimodal framework that leverages endogenous mitochondrial DNA variants to reconstruct high-resolution spatiotemporal clonal dynamics and decode niche-driven fate decisions in a human stem cell-derived model. This approach provides a powerful strategy for dissecting tissue self-organization in development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44d3ba922d5d14a9018eca6710873d3d64119d9" target='_blank'>
              Single-Cell Mitochondrial Lineage Tracing Decodes Fate Decision and Spatial Clonal Architecture in Human Hematopoietic Organoids.
              </a>
            </td>
          <td>
            Yan Xue, J. Su, Yiming Chao, Lu Liu, Xinyi Lin, Yang Xiang, Mun Kay Ho, Zezhuo Su, Junyi Chen, Zhuojuan Luo, Chengqi Lin, Ruibang Luo, Theo Aurich, Jianfeng Wu, K. S. C. Cheung, Yuanhua Huang, Joshua W. K. Ho, Ryohichi Sugimura
          </td>
          <td>2026-01-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Tumor histology reflects disease aggressiveness and clinical outcomes in cancer patients. Lung adenocarcinomas (LUADs) are classified based on predominant histologic patterns, including high-grade micropapillary and solid subtypes which portend unfavorable clinical features and prognosis. However, the cellular and molecular characteristics underlying these histologic subtypes remain largely unknown. We used scRNA-seq to profile 117,266 cells from 18 treatment-naïve LUADs with heterogeneous histologic patterns and also performed spatial transcriptomic analysis (10x Visium) for representative cases. By integrating single-cell transcriptomics with spatial information, we aimed to characterize the cellular identity and spatial organization driving LUAD heterogeneity. We demonstrated that histologic subtypes can be distinguished by subtype-specific cancer cell subpopulations and immunosuppressive phenotypes in the tumor microenvironment (TME). Our data reveal how intercellular interactions among cancer cells, macrophages, and CD8+ T cells in the prognostically unfavorable solid subtype are associated with cancer cell plasticity and promote an immunosuppressive TME. Additionally, we identify HMGA1 as a potential clinically relevant biomarker and therapeutic target for the solid subtype LUAD. These findings deepen our understanding of the histologic heterogeneity of LUAD and may facilitate the development of subtype-specific biomarkers and targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a31e93002156cd8296588a7c36e93349dfff416" target='_blank'>
              Defining the cellular and molecular identities of histologic subtypes in lung adenocarcinoma
              </a>
            </td>
          <td>
            Jusung Lee, J. Y. Jeong, M. Hong, Yoon Ha Choi, Ju Young Kim, J. Lee, J. Choi, Moonsik Kim, Young Woo Do, E. Lee, S. Choi, S. Yoo, Jae-Yong Park, Jong Kyoung Kim, S. Lee
          </td>
          <td>2026-01-23</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Metastatic triple-negative breast cancer (TNBC) is highly aggressive and lacks targeted therapies. Circulating tumor cells (CTC) are invaluable for monitoring metastatic tumor progression and treatment response but are difficult to capture because of their rarity and heterogeneity. Surface-based staining for live CTCs is essential to preserve RNA quality in single cells, but current markers tend to perform poorly on more mesenchymal tumor cells such as TNBCs. To enhance live TNBC CTC detection, we developed a workflow for live CTC capture and single-cell RNA sequencing (scRNA-seq). Using a mouse model of metastatic TNBC, we identified four new CTC surface markers, AHNAK2, CAVIN1, ODR4, and TRIML2, which specifically stain tumor cells. Combining antibodies against these markers improved CTC detection rates in multiple TNBC mouse models and patient samples. Also, combining these new markers with traditional CTC surface markers enhanced detection sensitivity, achieving the highest CTC coverage. This approach identifies diverse CTC populations, while preserving RNA quality for scRNA-seq, which is essential for understanding and therapeutically targeting metastatic breast cancer. The use of these newly identified CTC markers significantly enhances both detection and live capture of CTCs, paving the way for more effective use of liquid biopsy to monitor patient prognosis and treatment response in clinical settings. Significance: CTCs are a powerful indicator of cancer metastasis; however, their scarcity makes them difficult to isolate. Current markers favor epithelial CTCs over mesenchymal populations. Our workflow for live CTC capture and sequencing enables discovery of new markers for both epithelial and mesenchymal CTCs. When combined with existing markers, we improve live CTC capture for more holistic studies of the metastatic process and offer a scalable method for discovering CTC markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b002ef6362d1a4ee73fe011a8af50d517c31c57" target='_blank'>
              Surface Marker Identification to Capture Live Circulating Tumor Cells in Metastatic Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Bree M Lege, Khushali J. Patel, Brendan Panici, Ping Gong, Michael T. Lewis, M.J. Ellis, Chonghui Cheng
          </td>
          <td>2026-01-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite significant advances in understanding lung development, the intricate cellular interactions and spatial organization of the developing human lung remain incompletely defined. Spatial transcriptomics enables gene expression profiling within the native tissue context, providing unprecedented insights into complex developmental processes. In this study, we applied the 10X Genomics Visium platform to characterize spatially resolved transcriptional profiles of prenatal human lungs during the pseudoglandular and canalicular stages.Spatial transcriptomic analysis of 12 prenatal lung samples (13–20 weeks gestation) identified 10 distinct transcriptional niches corresponding to unique combinations of epithelial, mesenchymal, endothelial, and immune cell populations. Unsupervised clustering revealed developmental shifts in spot/niche composition from the pseudoglandular to canalicular stage, with a progressive increase in alveolar epithelial spots and a concomitant decline in mesenchymal regions, particularly in peripheral lung areas. Differential gene expression analysis demonstrated stage-specific transcriptional transitions in individual spot types, including downregulation of cell cycle and structural pathways and upregulation of secretory pathways as the lung matures. Spatial organization analysis revealed increasing compartmentalization of pulmonary cell types, highlighting the progressive structuring of the distal lung microenvironment. In summary, this study provides a spatial map of the developing human lung, offering novel insights into pulmonary lineage dynamics and cellular interactions during early organogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4cdf7e14676d1f19fec1630ce3c773e08720b2c" target='_blank'>
              Spatial transcriptomics of developing human lungs defines cellular phenotypes associated with age, lineage and location
              </a>
            </td>
          <td>
            Yue Ren, Soula Danopoulos, Gail H. Deutsch, Ian A. Glass, Thomas J. Mariani, S. Bhattacharya
          </td>
          <td>2026-01-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 The tumor microenvironment (TME) plays a pivotal role in shaping cancer progression, immune evasion, and therapeutic response. However, a systematic understanding of immune and stromal cell heterogeneity across cancer types—and their associations with clinical phenotypes—remains lacking due to limited single‐cell data and the associated clinical metadata. Here, we present a comprehensive pan‐cancer TME cell atlas to date, quantifying the composition of 102 immune and stromal subtypes across 30 cancer types and encompassing over 11,000 bulk RNA‐seq samples from TCGA. To enable this unprecedented resolution, we developed DECEPTICONx, a flexible computational framework that integrates single‐cell RNA‐seq data by constructing high‐resolution expression templates, which are then paired with context‐optimized deconvolution strategies, dynamically selecting the best‐matched template‐deconvolution combinations for each task. By leveraging 535,665 single cells spanning T cells, B cells, fibroblasts, macrophages, and NK cells, we constructed high‐resolution expression templates and systematically deconvolved bulk samples. The resulting atlas revealed robust and novel associations between specific cell subtypes and patient prognosis, immunotherapy response, and inter‐cellular coordination within the TME. In summary, the DECEPTICONx‐based atlas not only advances our understanding of tumor‐immune ecology but also offers a foundation for biomarker discovery and therapeutic development in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaaef7a98ed82e86ab1fb8032603b073ca5c5bb6" target='_blank'>
              A Pan‐Cancer Microenvironment Atlas Constructed by DECEPTICONx
              </a>
            </td>
          <td>
            Yida Gu, Fulan Deng, Lianchong Gao, Zhiyu Chen, Jianping Chen, Jiale Wu, Zengqiang Xia, Zhijian Li, Fangkaikai Cheng, Jie Chen, HoiYee Tong, Wantao Chen, Xiaoqing Pan, Xin Zou, Jie Hao
          </td>
          <td>2026-01-17</td>
          <td>iMetaMed</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Protein function is inherently spatial: the same molecule can elicit distinct biological outcomes depending on its localization, interacting partners, and surrounding microenvironment. Spatial proteomics enables systematic in situ characterization of protein localization, abundance, and interactions across subcellular to tissue scales, surpassing the resolution and contextual information accessible to conventional bulk proteomics. Recent technological advances including DNA-barcoded multiplexing methods, cyclic fluorescence platforms, and mass spectrometry imaging have substantially increased multiplexing capacity, sensitivity, and spatial accuracy. These capabilities directly support clinically relevant applications, such as tumor immune microenvironment analysis, mapping of protein aggregation in neurodegeneration, growth factor dynamics during tissue repair, patient stratification, pharmacodynamic mapping, and target-engagement assessment. Computational innovations, including graph neural networks, self-supervised embeddings, and workflow management tools (e.g. Snakemake, Nextflow), further enhance cell segmentation, noise reduction, and multi-modal data integration, enabling extraction of robust, spatially resolved proteomic information from complex tissues. Future research will aim to standardize protocols, enable real-time clinical analysis, and develop 3D spatial proteome maps to advance spatial proteomics toward precision diagnostics and targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/425add3b64f4935a7f1fdffca5175483abf295c9" target='_blank'>
              Spatial proteomics in precision medicine: technologies, bioinformatics, and translational applications
              </a>
            </td>
          <td>
            Yiwen Li, Yusheng Zhang, Ying Zhang, Qing Wang, Boyang Ji, Hongjun Yang, Xianyu Li
          </td>
          <td>2026-01-08</td>
          <td>Precision Clinical Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is considered immunologically “cold,” however, we have previously identified an immunogenic subset of high-risk localized PCa with T-cell tumor infiltration within antigen-presenting-cell (APC) niches. Here, we used high-dimensional single-cell spatial proteomics to dissect tumor-infiltrating-lymphocyte in (TIL)-high (H), TIL-low (L), and TIL-excluded (X) PCas, to understand the tumor-specific and conserved mechanisms of immune dysregulation for targeted therapeutics.



 Treatment-naive prostatectomy specimens of all three tumor immune microenvironment (TIME) phenotypes, were assembled into tissue microarrays representing within-tumor and adjacent foci. TIL-X was exclusively observed in cribriform pattern 4 cases. A 40-plex antibody panel made of phenotyping and functional markers (e.g AR) was applied to the TMAs, enabling single-cell annotation of key immune cell types (e.g CD8+ T cells) and assessment of their functional states within distinct spatial microenvironments.



 In TIL-X PCa, immunosuppressive macrophages and DCs were enriched in the tumor nest whereas T cells accumulated at the periphery with minimal tumor infiltration. Tumor adenosine 2A receptor (A2AR) expression was uniquely enriched within tumor, suggesting a metabolically suppressive microenvironment that reinforces immune exclusion. TIL-H tumors had comparable T cell proportions inside and outside the tumor, accompanied by the enrichment of myeloid cells within the tumor. CD8+ T cells expressing high levels of AR and B7H3 frequently co-localized with tumor and APCs, forming APC niches and tertiary lymphoid-like structures. CD8+ARhi and CD8+B7H3hi subsets with elevated exhaustion scores preferentially infiltrated tumor, whereas CD8+ARlo and CD8+B7H3lo subsets with moderate exhaustion levels remained peripheral; defining TIL-H PCa as tumors harboring antigen-experienced but exhausted CD8 T cells in a TIME that supports immune cell engagement but restrains anti-tumor activity. TIL-L PCa had sparse T cell infiltration and enrichment of myeloid cells with moderate AR expression within tumor, suggesting a role of androgen signaling and myeloid cells in shaping the TIME, marked by high PD-1+CD8+ cells and PD-L1+ tumor cells. Thus, TIL-L PCa shows low immune recruitment, yet strong tumor-mediated inhibitory signaling that constrains anti-tumor immunity.



 TILs in TIL-H PCa are phenotypically, functionally, and spatially distinct from those in TIL-L or TIL-X tumors, reflecting tumor-intrinsic immune programs. High-dimensional spatial proteomics reveal differential molecular modules and signaling pathways that define unique and potentially targetable mechanisms of immunosuppression across the three TIME phenotypes; additional spatial transcriptomics validation and dissection experiments are underway. These findings demonstrate that the PCa TIME is diverse yet spatially organized, and that comprehensive understanding of baseline spatial and transcriptional immune states may guide personalization of immunotherapy strategies in PCa.



 Precious Cramer, Yang Wang, Wenrui Wu, Huaying Qiu, Lindsay Parmelee, Berkay Simsek, Gabriel Roberti. De Oliveira, Seifeldin Awad, Yao Yu Yeo, Stephanie Pei Tung. Yiu, Hendrik Michel, Sabina Signoretti, Yue Sun, Steven Balk, Sizun Jiang, David Einstein Einstein. High-dimensional spatial profiling identifies distinct and targetable tumor-infiltrating lymphocyte phenotypes in localized prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d786dbf212bd5a71f25f7e11f429f8dba252ddc" target='_blank'>
              Abstract PR010: High-dimensional spatial profiling identifies distinct and targetable tumor-infiltrating lymphocyte phenotypes in localized prostate cancer
              </a>
            </td>
          <td>
            Precious Cramer, Yang Wang, Wenrui Wu, Huaying Qiu, Lindsay Parmelee, Berkay Simsek, G. R. de Oliveira, Seifeldin Awad, Yao Yu Yeo, Stephanie Pei Tung. Yiu, Hendrik A. Michel, Sabina Signoretti, Yue Sun, Steven P. Balk, Sizun Jiang, D. Einstein
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Prostate cancer cell invasion into the surrounding muscle capsule, known as extracapsular extension (ECE), is associated with a significantly worse prognosis (5-year survival reduction: 98% to 31%). Using a live ECE mouse xenograft model with DU145 tumor cells, our group previously demonstrated that functional heterogeneity among adhesion subtypes within invasive clusters is essential for muscle invasion. Bulk RNA sequencing from three spatial compartments (pre-invasive, invasive, and post-invasive) revealed distinct gene expression profiles, with hypoxia-related and cell–cell/cell–ECM interaction genes prominently dysregulated in muscle-invasive disease. In this study, we sought to define the single-cell transcriptional heterogeneity across the three ECE compartments and compare these patterns to a human patient-derived xenograft (PDX) model. Spatial transcriptomic analysis (STA) was performed using the Bruker CosMx platform with a 6K gene panel. Data was preprocessed (quality control and dimensional reduction) and analyzed in R 4.4.0 using Seurat v5. Notable downstream analyses included unsupervised Louvain clustering, trajectory inference with Monocle 3, and cluster-specific differential gene expression. In the ECE model, STA resolved at least 11 distinct tumor cell identity clusters previously undetectable by bulk methods and revealed a clear spatial demarcation between pre-invasive and invasive populations crossing the muscle barrier. Trajectory analysis identified a pre-invasive population branching into two invasive fates with unique expression profiles. Dispersion scores, calculated using principal component embeddings, exhibited progressively increasing intracluster heterogeneity along one branch. Strong expression enrichment of the Prolaris 31-gene prognostic panel, a clinically established marker of aggressiveness, was observed at the bifurcation point of tumor evolution. STA of the human PDX model similarly revealed a high-dispersion invasive cluster branching into two fates, one exhibiting decreased heterogeneity consistent with specialization of invasive phenotypes. Together, these findings reveal, for the first time, that tumor invasion of the extracapsular muscle layer induces substantial transcriptional heterogeneity. This dynamic process highlights potential therapeutic vulnerabilities during the transition to extracapsular extension. Targeting tumor cells at this critical juncture may represent a key strategy for preventing ECE and improving prostate cancer outcomes. Supported by P30CA023074, University of Arizona Comprehensive Cancer Center Support Grant.



 Saptarshi Mallick, Rafael Sainz, Jayati Chakrabarti, Kendra D. Marr, Noel A. Warfel, Gregory C. Rogers, Beatrice S. Knudsen, Rashid Sayyid, Kelvin W. Pond, Anne E. Cress. Single cell spatial transcriptomics identifies unique cell populations driving muscle invasive prostate adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcc6a26c8a5d704d47fd40ba9c2b9c0030fcf73d" target='_blank'>
              Abstract B045: Single cell spatial transcriptomics identifies unique cell populations driving muscle invasive prostate adenocarcinoma
              </a>
            </td>
          <td>
            Saptarshi Mallick, Rafael Sainz, Jayati Chakrabarti, Kendra D. Marr, N. Warfel, Gregory C Rogers, Beatrice S. Knudsen, R. Sayyid, Kelvin W. Pond, A. Cress
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a biologically and clinically heterogeneous cancer in which the spatial organization of the tumor microenvironment dictates progression, immune escape, and therapeutic response. Traditional genomic and transcriptomic profiling have revealed key molecular pathways, yet they do not capture the spatial relationships that govern tumor-immune-stromal interactions. Spatial omics technologies now bridge this gap, allowing molecular profiling within intact tissue architecture and revealing how cellular neighborhoods, immune barriers, and stromal niches shape tumor behavior. In this review, we summarize recent advances in technologies like spatial transcriptomics, proteomics, genomics, epigenomics, and metabolomics as applied to HCC. We discuss how these technologies have illuminated the spatial dynamics of the invasive margin, macrophage and fibroblast niches, and immune evasion, identifying both prognostic and predictive biomarkers. We further highlight how spatially resolved analyses have uncovered previously unrecognized axes of communication between cancer cells and key stromal and parenchymal populations that were not apparent from bulk or single-cell molecular analyses alone. These spatially mapped ligand-receptor networks reveal coordinated signaling circuits that define targetable mechanisms of resistance and progression within the microenvironment. Finally, we explore the translational implications where spatial signatures may guide risk stratification, treatment selection, and clinical trial design. By integrating spatial architecture with molecular function, spatial omics is transforming our understanding of liver cancer biology and paving the way for spatially informed precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d51c0211c082179b9a7f39ecfe5a5186ceabc189" target='_blank'>
              Unraveling the Tapestry: Lessons from multiomics and spatial biology in hepatocellular cancer.
              </a>
            </td>
          <td>
            D. Massoudi, Ankit Srivastava, R. Dhanasekaran
          </td>
          <td>2026-02-12</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) maturity and cellular composition shape resistance or sensitivity to immune-checkpoint blockade (ICB) across cancers. Single-cell spatial resolution data for the endothelial compartment of TLSs are lacking; therefore, we investigated the cellular composition, endothelial dynamics, and cell-cell interactions within TLS regions. We applied high-resolution Xenium In Situ spatial transcriptomics to eight samples collected from non-small-cell lung cancers (NSCLC) (four after chemoimmunotherapy, four untreated), extracted 90 TLS regions, generated approximately 320,000 single-cell profiles from those regions, and analyzed the endothelial compartment within each TLS. We independently assessed the exploratory findings using multiplex immunofluorescence and survival analysis in a separate cohort of 45 patients receiving immunotherapy. Spatial mapping revealed a reproducible, B-cell-dominant TLS architecture across all samples. Mature TLSs contained CXCL13-rich follicular cores and were associated with a favorable prognosis. Within the endothelial compartment, high endothelial cells (HECs) forming high-endothelial venules (HEVs) segregated from other endothelial cells; in mature TLSs these HECs exhibited high inhibitor of DNA binding 1 (ID1) expression and an adhesion molecular signature, promoting lymphocyte recruitment. Patients with tumors harboring ID1-high HEVs showed higher response rates and pronounced survival advantage; multivariable analysis confirmed ID1-high status as an independent prognostic factor. Conversely, ICB exposure eventually reduced ID1 expression and adhesion molecule levels in HECs, impaired lymphocyte trafficking, and promoted a shift toward a remodeled endothelial state. ID1expression, linked to HEC differentiation during TLS maturation or ICB exposure, marks a functionally mature, lymphocyte-recruiting HEV that predict durable response to immunotherapy. These findings provide a novel mechanistic framework and practical biomarkers for HEV-centered immuno-oncology strategies in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a47785e0dbc1da1e3fb62fce1edd50b239726393" target='_blank'>
              Single-cell spatial analysis identifies ID1-high endothelial cells in tertiary lymphoid structures as predictors of durable response to immunotherapy in non-small cell lung cancer
              </a>
            </td>
          <td>
            Kinnosuke Matsumoto, Y. Noda, Kensuke Hachiya, F. Muramatsu, Naoki Okamoto, Weizhen Jia, T. Shiroyama, Masahide Mori, M. Tamiya, Yuhei Kinehara, A. Tamiya, Shigeki Shimizu, Hidekazu Suzuki, Kiyonobu Ueno, T. Niki, Satoshi Tetsumoto, O. Morimura, Akio Osa, Toshiyuki Minami, Satoshi Nojima, Y. Takeda, Y. Shintani, Atsushi Kumanogoh, Nobuyuki Takakura
          </td>
          <td>2026-01-06</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Single-cell transcriptomics has redefined our understanding of cancer by exposing the complexity of tumor ecosystems and their therapeutic vulnerabilities. scRNA-seq studies have identified lineage hierarchies, immune evasion programs, and resistance-associated states across solid and liquid tumors, informing biomarker development and drug discovery. Advanced computational frameworks integrate these data with longitudinal profiling, RNA velocity, and network diffusion to prioritize targets and predict therapeutic response. Emerging multi-omics approaches further expand the scope of precision oncology by linking genetic alterations, protein-level markers, and spatial context to functional states. This narrative review aims to synthesize current applications of single-cell transcriptomics for target discovery, highlight computational frameworks that translate high-dimensional data into actionable insights, and explore how multi-omics integration is shaping future directions. By bridging molecular complexity with target prioritization, these approaches hold promise for translating single-cell insights into clinically actionable biomarkers and therapeutic strategies for personalized cancer treatment and rational drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc90ad6637cf26b6fef95756c86b88733af331f8" target='_blank'>
              Single-Cell Transcriptomics and Computational Frameworks for Target Discovery in Cancer
              </a>
            </td>
          <td>
            Martina Tarozzi, Nicolas Derus, Stefano Polizzi, C. Sala, Gastone Castellani
          </td>
          <td>2026-02-03</td>
          <td>Targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Colitis-associated colorectal cancer (C-CRC) arises in the context of chronic inflammation, where tumor microenvironment interactions strongly influence progression. Spatial transcriptomics enables characterization of these interactions by retaining tissue architecture and cellular context. We aimed to characterize the cellular and microenvironmental features that underlie tumor development in C-CRC.



 Seven tissue samples from three C-CRC patients were analyzed using consecutive sections processed by 10x Xenium and 10x Visium. Spatial coordinates of Xenium cells were mapped onto Visium spots to enable integrated spatial transcriptomic profiling (Figure 1A).



 Xenium analysis identified 3,506,705 cells with diverse epithelial, fibroblast, and immune cell populations (Figure 1B-C). Tumor cell subclustering revealed two functionally segregated populations: (1) proliferating tumor cells characterized by cell-cycle genes (MKI67, TOP2A) and (2) invasive tumor cells with elevated epithelial mesenchymal transition (EMT)-associated genes (CD44, CXCL1). Cellular neighborhood analysis based on distances between cells identified a distinct microenvironmental niche unique to invasive tumor cells, characterized by spatial proximity of cancer-associated fibroblasts (CAFs) and SPP1+ macrophages (Figure 1D). Pathway enrichment analysis showed selective up-regulation of angiogenesis, hypoxia, and inflammatory pathways in invasive versus proliferating tumor regions (Figure 1E). Receptor–ligand analysis identified TNC–ITGB6 interactions between CAFs and invasive tumor cells (Figure 2A). A differentially expressed gene signature distinguishing invasive from proliferating tumor cells stratified TCGA sporadic CRC samples into invasive-tumor-like and proliferating-tumor-like groups. Invasive-tumor-like tumors predominantly displayed consensus molecular subtype (CMS) 4 characteristics, consistent with the known enrichment of CMS4 in UC-CRC and supporting a role for CAF-mediated interactions in mesenchymal phenotype development (Figure 2B).



 This integrated spatial transcriptomics study identified two distinct tumor populations in C-CRC with segregated microenvironmental contexts. The invasive tumor phenotype is characterized by CAF proximity, SPP1+ macrophages, and TNC–ITGB6-mediated signaling. These findings reveal that tumor-stromal composition and architecture are linked to mesenchymal phenotype development, suggesting the microenvironment as a potential target for biomarker and therapeutic development in C-CRC.
 Conflict of interest:
 Lee, Yoonho: No conflict of interest
 Kim, Yongjae: No conflict of interest
 Baek, Jiwon: No conflict of interest
 Hwang, Sung Wook: no conflicts
 Lee, Jong Lyul: None.
 Park, Sang Hyoung: No conflicts
 Yang, Dong-Hoon: Nothing to declare
 Kim, Jihun: No conflict of interest
 Yang, Suk-Kyun: No conflict of interest
 Han, Buhm: No conflict of interest
 Song, Kyuyoung: No conflict of interest
 Yoon, Yong Sik: No conflict of interest
 Ye, Byong Duk: Byong Duk Ye reports consulting fees from AbbVie Korea, BMS Pharmaceutical Korea Ltd., Celltrion, Chong Kun Dang Pharm, CJ Red BIO, Curacle, Daewoong Pharm, Dong-A ST, Ferring Korea, Hanmi Pharmaceutical, Imscout, IQVIA, Johnson & Johnson, Johnson & Johnson Korea, Jeil Pharmaceutical Co., Kangstem Biotech, Korea Otsuka Pharm, Korea United Pharm, Lilly Korea, Medtronic Korea, NanoEntek, ORGANOIDSCIENCES Ltd., Pfizer Korea, Samsung Bioepis, Takeda, Takeda Korea and Yuhan speaker fees from AbbVie Korea, BMS Pharmaceutical Korea Ltd., Celltrion, Cornerstones Health, Curacle, Daewoong Pharm, Eisai Korea, Ferring Korea, IQVIA, Johnson & Johnson Korea, Pfizer Korea, Samsung Bioepis, and Takeda Korea and research support from Celltrion and Pfizer Korea.
 Lee, Ho-Su: No conflict of interest


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c70793b20108beca09788feb076288ec350d6d1" target='_blank'>
              DOP110 Spatial mapping of tumor-stromal cell interactions defines mesenchymal phenotypes in colitis-associated colorectal cancer
              </a>
            </td>
          <td>
            Y. Lee, Y. Kim, J. Baek, S. Hwang, J. Lee, S. Park, D. H. Yang, J. Kim, S. K. Yang, B. Han, K. Song, Y. Yoon, B. Ye, H. S. Lee
          </td>
          <td>2026-01-01</td>
          <td>Journal of Crohn’s and Colitis</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="There is an acute need to accurately identify patients with advanced melanoma who are most likely to respond to anti-PD1 immune checkpoint blockade (ICB) therapy. While anti-PD1 therapy can be highly effective in advanced melanoma patients, only 30-40% of patients respond well. In this study, we apply single-cell spatial proteomics together with statistical and machine learning (ML) methods to successfully predict advanced melanoma patient response to anti-PD1 ICB in a cohort of 12 patients with >8 million cells. While no single molecular feature is sufficient to predict ICB response in our cohort, ML models integrating multiple molecular features accurately predict response in 11 of 12 patients. A recurrent cellular neighborhood analysis revealed a tumor-infiltrating lymphocytes niche that was present in the tumors of most responders. This neighborhood, tumor microenvironment immune cell composition, and levels of nitric oxide synthases were all important features used by our ML models to make accurate predictions. Optimal predictive performance by our ML models—a ROC AUC of 0.76—was achieved when using all molecular features, including cellular spatial relationships, but limiting our analysis to only immune-rich tissue regions. This study demonstrates the feasibility of using machine learning models to accurately predict patient response to anti-PD1 ICB therapy using spatial proteomics datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9443679161e4acaf278b330e53407ed7f24f3d06" target='_blank'>
              Predicting anti-PD-1 immune checkpoint blockade response in melanoma patients with spatially aware machine learning models
              </a>
            </td>
          <td>
            Alyssa Pybus, Raphael Kirchgaessner, Jonathan Nguyen, Carlos M Moran Segura, Paulo Cilas Morais Lyra, Trevor Rose, Jhanelle Gray, Jeremy Goecks, Joseph Markowitz
          </td>
          <td>2026-01-12</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Intratumoral heterogeneity (ITH) is a major driver of mortality in breast cancer (BC) patients and a critical factor in the variable therapeutic outcomes observed in BC treatment. Understanding the mechanisms underlying ITH is essential for advancing both clinical and basic BC research. Chromatin accessibility is critical for regulation of gene expression and cellular identity and plays a central role in shaping ITH and tumor evolution. However, studying chromatin accessibility in situ has been challenging due to the availability of technical platforms. Here, we leveraged the spatial ATAC-seq platform to profile the chromatin accessibility landscape of tumors from six BC patients. Our analyses revealed prominent heterogeneity within tumor regulatory modules and spatial variations in immune cell composition and stromal structures, offering a framework for investigation of the molecular architecture underlying ITH. Moreover, we identified two tumor subclones with potential common origin but distinct immune infiltration conferred by regulatory cascades, suggesting that epigenetic regulation may further contribute to the divergent tumor microenvironments and phenotypic diversity of these subclones. Our study provides novel insights into the molecular mechanisms driving ITH and opens up potential avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b8d2e84c5991207c662ff2b4c48eb4734a447e" target='_blank'>
              Spatial Chromatin Accessibility Analysis of Intratumor Heterogeneity in Breast Cancer.
              </a>
            </td>
          <td>
            Yingying Qian, Miao Zhu, C. Ren, Yeyong Zhou, Jian Xu, Liang Dong, Guangyu Zhang, Cheukfai Li, Jiaoyi Lv, Qiaorui Xing, Guochun Zhang, Guangdun Peng, Ning Liao
          </td>
          <td>2026-01-12</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Breast cancer is a malignant tumor originating from breast epithelial tissue. In essence, breast epithelial cells undergo gene mutation under the influence of carcinogenic factors, leading to abnormal cell proliferation and loss of organism regulation, ultimately leading to the formation of tumors with invasive and metastatic capabilities. Carcinogenic factors of breast cancer involve multiple cellular and molecular mechanisms. Among them, disseminated tumor cells (DTCs) are considered important for treating breast cancer. However, traditional bulk sequencing techniques have limitations, such as the inability to distinguish individual cell differences and dilution of information from key cell subpopulations (such as cancer stem cells and rare immune cells). Single-cell sequencing (scRNA-seq) overcomes the heterogeneity of tumors that traditional sequencing cannot capture by analysing the molecular characteristics of single cells, providing a high-resolution perspective for precise typing of breast cancer, exploration of the mechanism of the microenvironment, and personalized treatment. Through this technology, researchers can identify specific gene expression profiles of different cell subpopulations, thus providing a new basis for the molecular typing and personalized treatment of breast cancer. This article explains how single-cell sequencing is used to describe the origin of disseminated tumor cells (DTCs), analyse tumor heterogeneity, metastasis, etc., and review the current literature on the use of scRNA-seq in breast cancer treatment. In the future, cell separation and processing steps in single-cell sequencing will be further improved to ensure the accuracy of the results and broader application in clinical diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c8a6702266de5ec2199ce11aee24ada48d0a237" target='_blank'>
              Single-cell Sequencing: Application in the Study of Disseminated Tumour Cells and Breast Cancer Treatment
              </a>
            </td>
          <td>
            Mingyue Wang
          </td>
          <td>2026-02-06</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spatial organization of the disease microenvironment informs patient prognosis. Key modalities for studying spatial biology include H&E images (WSIs) for tissue structure and spatial transcriptomics (ST) for transcriptome-level programs. Spatial analysis aims to (1) identify markers linked to clinical outcome, (2) understand functional programs driving these associations, and (3) guide targeted therapies. Current research addresses these topics but offers limited explainability across the full morphology - molecular mechanism - outcome axis. Further, given the abundance of WSIs and limited availability of ST, there is a need for analyses integrating these complementary datasets. We present an AI-driven framework combining foundation-model features, multiple-instance learning, unsupervised clustering, and molecular analyses to identify mechanisms underlying outcome associated patterns. Applied to HER2+ breast cancer, we identify CCND1 and PTK6 signaling in tumor regions linked to trastuzumab resistance, consistent with prior studies. Our approach offers interpretable insights for multi-level resistance mechanisms, tissue-specific drug targeting, and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7a60accd94c10ba89d66c522cd744cd2f973bb5" target='_blank'>
              A multimodal framework to identify molecular mechanisms driving patient group-associated morphology through the integration of spatial transcriptomics and whole slide imaging
              </a>
            </td>
          <td>
            Reva Kulkarni, A. Maddox, Sara Bailey, Arvind Rao
          </td>
          <td>2026-01-16</td>
          <td>Npj Artificial Intelligence</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e234e293312dfb9093dad9f87bd40ef8d26f2e28" target='_blank'>
              Spatial transcriptomics reveals molecular heterogeneity and subtype-specific therapeutic targets in small cell lung cancer.
              </a>
            </td>
          <td>
            Tongji Xie, Le Tang, G. Fan, Xiaohong Han, Yuankai Shi
          </td>
          <td>2026-01-16</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Phenotypic plasticity is a prominent cancer feature that contributes to metastatic potential and resistance to therapy across multiple cancer types. Cancer cell state transitions have been attributed to transcriptional programs, such as the AP1/TEAD-regulated gene network driving the mesenchymal-like (MES) phenotype. In addition, during dissemination, tumor cells are subjected to variable loads of physical mechanical pressure and constriction across transited tissue, which are thought to impact nuclear molecular crowding. How the interplay between mechanical pressure, global 3D nuclear architecture and transcriptional programs contributes to MES identity and metastatic adaptation remains unclear. Using cutaneous melanoma as a model for early dissemination, we integrate in vitro and in vivo epigenomic profiling with nanoscale imaging of cell lines and patient samples to investigate chromatin organization features underlying the MES phenotype. We find that in MES cells, CTCF is relocated from domain boundaries to regulatory regions of EMT-like genes, leading to reduced insulation, extended topological associated domains (TADs) and increased inter-domain contacts, and de novo formation of chromatin hubs. This conformational rewiring, along with loss of heterochromatin, supports nuclear deformability during invasion and dissemination. Conversely, physical constriction of melanocytic cells induces MES-like chromatin features —including CTCF repositioning and heterochromatin loss— and promotes metastasis in vivo. Similarly, pharmacological inhibition of the heterochromatin mark H3K9me3 triggers MES characteristics and increases invasiveness. These results demonstrate that metastatic competency involves both epigenetic and structural nuclear reprogramming, enabling shifts in gene networks and physical adaptability. Our findings reveal mechanistic links between nuclear architecture and aggressive tumor behavior, identifying potential biomarkers and therapeutic targets to intercept metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92c8bf70d1df18ff53fcb785d7213684e545662a" target='_blank'>
              Chromatin architecture and physical constriction cooperate in phenotype switching and cancer cell dissemination
              </a>
            </td>
          <td>
            Pietro Berico, Cody L Dunton, L. Almassalha, Amanda Flores-Yanke, Karla I Medina, Nicolas Acosta, Tara Muijlwijk, Catherine Do, Soobeom Lee, S. Edmiston, D. Corcoran, Allison Reiner, Caroline E. Kostrzewa, Kathy Dorsey, Milad Ibrahim, Ronglai Shen, Nancy E. Thomas, Amanda W. Lund, Ata S. Moshiri, Iman Osman, I. Aifantis, Jane A. Skok, Vadim Backman, Eva Hernando
          </td>
          <td>2026-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d27ce10a067c8461179490fc90529630fe99743" target='_blank'>
              A modular patient-derived organoid–xenograft platform reveals molecular and clinical trajectories of prostate cancer progression
              </a>
            </td>
          <td>
            R. Parmentier, R. Dolgos, R. Servant, Jing Wang, L. Roma, R. Mevel, D. Bossi, H. Pueschel, Z. Diamantopoulou, T. Vlajnic, F. Stenner, A. J. Templeton, H. Seifert, Nicola Aceto, J-P. Theurillat, Cyril Rentsch, L. Bubendorf, C. Le Magnen
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed71c8afd376ce011a1a142f4c287f9d7c3670dc" target='_blank'>
              A spatially resolved human glioblastoma atlas reveals distinct cellular and molecular patterns of anatomical niches.
              </a>
            </td>
          <td>
            P. Sonpatki, Hyun Jung Park, Yao Lulu Xing, Kyung Yeon Han, B. A. Schroeder, Hyeon Jong Yu, Hye Jin Kim, Tamrin Chowdhury, Jong Ha Hwang, Sun Mo Nam, Yoon Hwan Byun, Ho Kang, Joo Ho Lee, , Jae-kyung Won, Tae Min Min Kim, Seung Hong Choi, Ja-Lok Ku, Sungyoung Lee, Ho-Joo Yun, , C. Petritsch, Chul-Kee Park, , Nameeta Shah
          </td>
          <td>2026-02-20</td>
          <td>Nature communications</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Background Bladder cancer (BLCA) is a malignant tumor originating from the urothelial lining, characterized by a complex tumor microenvironment (TME) and heterogeneous tumor mutation burden (TMB). Cancer-associated epithelial cells (EpiCs) exhibit substantial heterogeneity during BLCA initiation and progression. Therefore, elucidating the diversity and functional states of EpiCs is essential for improving future diagnostic and therapeutic strategies. Methods We integrated multi-omics datasets, including 13 single-cell RNA-seq samples, 514 bulk transcriptome profiles, and 30 whole-exome sequencing (WES) samples, to comprehensively characterize EpiC subtypes. Nonlinear dimensional reduction (UMAP) and clustering analyses were performed to identify major epithelial subsets, followed by secondary clustering. TMB values calculated from self-generated WES data were incorporated into scAB and Ro/e algorithms to determine the TMB-associated epithelial subset, ultimately identifying the key cluster Epi14. Differentially expressed genes (DEGs) of Epi14 were analyzed, and CellChat was used to infer intercellular communication networks. CytoTRACE and Monocle2 were applied to assess stemness potential and differentiation trajectories. Random survival forest (RSF) combined with DEG analysis was used to identify hub genes. Immune infiltration, drug sensitivity, and functional pathway analyses were subsequently conducted. Spatial transcriptomics were deconvoluted using spacexr, CellChat, and PROGENy to map cellular composition, signaling activity, and pathway nodes. Finally, qPCR and Western blot assays were performed to validate hub gene expression in tumor versus adjacent tissues. Results A total of 77,263 cells and 3,000 highly variable genes were included, yielding 32 annotated cell clusters. Secondary clustering combined with WES-derived TMB identified 14 epithelial subpopulations, among which Epi14 was confirmed as the key TMB-associated subset using the Ro/e algorithm. Integration of DEGs, RSF, and multi-cohort datasets revealed ABRACL and ARPC3 as the pivotal hub genes, from which a risk-score model was constructed. Notably, ABRACL expression showed a strong positive association with tumor TMB and exhibited pronounced enrichment in spatial transcriptomic tumor regions. Conclusion By integrating multi-omics and spatial datasets, this study reveals the epithelial heterogeneity of BLCA and identifies ABRACL and ARPC3 as key TMB-associated hub genes within EpiCs. The established risk-score model and validated functional markers provide valuable insights for future mechanistic studies and potential clinical translation in BLCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3d01e34654acc5d2fc42e0f3045f827529cbe0f" target='_blank'>
              Integrated single-cell and spatial transcriptomics combined with whole-exome sequencing reveal key hub genes and epithelial heterogeneity in bladder cancer
              </a>
            </td>
          <td>
            Lin Li, Qianyue Li, Jiaxin Liu, Yifan Wang, Yanjun Ma, Lei Tang, Yawei Zhao
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3490b5722169f26207fe3782f1d2c2b6c0970030" target='_blank'>
              Dual Lineage Tracing Identifies Cellular Mechanisms Underlying Radiation-Associated Changes in Atherosclerotic Lesion Composition
              </a>
            </td>
          <td>
            Rebecca A. Deaton, Tajbir Raihan, Victoria M. Milosek, Fatema Allaham, Alexandra L. Krinsky, Laura S. Shankman, G. F. Alencar, A. Salamon, Nazanin Moradinasab, Subhashis Banerjee, Gary K. Owens
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Introduction Non-small cell lung cancer is the leading cause of cancer-related mortality worldwide, with lung adenocarcinoma (LUAD) as the most common subtype. Although early-stage disease is often treated surgically, advanced LUAD typically requires chemotherapy, radiation, and/or immunotherapy, largely focused on T cell–mediated responses. Therapeutic efficacy, however, is also shaped by tumor-infiltrating (TI) B cells, whose roles in LUAD remain incompletely understood. Methods We performed cytometry by time of flight (CyTOF) using 44 markers on matched tumor, adjacent lung, and peripheral blood samples from 48 LUAD patients to define TI immune landscapes. 66 immune cell subsets were identified, and patients were stratified into four groups based on TI cell composition. Adaptive immune receptor repertoire sequencing (AIRR-seq) of IgM and IgG was conducted on matched samples from 29 patients to assess clonal expansion and affinity maturation. Subisotype-resolved AIRR-seq was additionally performed on tumor samples from 18 patients. Results CyTOF analysis revealed four patient groups with distinct TI immune profiles. AIRR-seq demonstrated increased clonal expansion and affinity maturation in tumors compared to adjacent lung and blood. Tumor-specific IGHV enrichment patterns were observed but were not associated with patient group assignment. Instead, clonal expansion was greatest in tumors with higher lymphocyte proportions. Subisotype-resolved analysis showed enrichment of IGHG2 and IGHG3 in tumors from patients with low TI B cell abundance, whereas IGHG4 was enriched in patients with high TI B cell infiltration and correlated with four CyTOF-defined immune subsets. Discussion These findings reveal substantial inter-individual variation in TI immune landscapes and highlight distinct B cell repertoire and subisotype features within LUAD tumors. Together, these data suggest that B cell composition and antibody subisotype usage may contribute to immune contexture and could inform the development of more tailored immunotherapeutic strategies in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6aa20b3c0324cfd852e979d6ecb7e9b1134c03" target='_blank'>
              B cell phenotypes and antibody signatures associate with interpatient variation in the lung adenocarcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Kamille M. Rasche, David Tieri, Samantha M. Morrissey, Hong Li, Fan Zhang, W. Gibson, EA Ford, Uddalok Jana, Melissa L. Smith, Jun Yan, Corey T. Watson
          </td>
          <td>2026-01-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Highlights • Spatial profiling reveals region-specific immune and tumor signaling in α-PD-1 treated breast cancer bone metastases.• Immune-suppressive markers are enriched in CD45+ cells in the endosteum of breast cancer bone metastases.• Proliferation and cell death pathways are enriched in tumor cells residing in the marrow vs the endosteum.• Both the marrow and endosteum enrich pro-tumorigenic pathways, but through distinct mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b157bc16f8d8f6a87ce65c94fbf059f6969006d9" target='_blank'>
              Digital spatial profiling of α-PD-1 treated breast cancer bone metastases reveals region-specific signaling and enrichment of immune-suppressive markers
              </a>
            </td>
          <td>
            Déja M. Grant, Gwenyth J. Joseph, M. Searcy, Rachelle W. Johnson
          </td>
          <td>2026-01-01</td>
          <td>Journal of Bone Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal cancer, with liver metastases significantly worsening outcomes. However, features of the tumor microenvironment (TME) that are distinct between primary and metastatic sites remain poorly defined. Cellular neighborhoods within the TME are recognized as functional units that influence tumor behavior. Conventional spatial methods, which assign equal weights to all cells in a region, fail to capture the nuances of cellular interactions. To address this, we report here the development of Functional Cellular Neighborhood (FunCN) quantification, which integrates both the proportion and proximity of surrounding cells. Applying FunCN to PDAC imaging mass cytometry data, we identified neutrophil-enriched interactions in liver metastases compared to primary tumors, correlating with elevated VISTA expression by tumor cells. Additionally, FunCN clusters around CD8⁺ T cells in pancreas and liver were associated with higher TIGIT and LAG3, respectively. These findings demonstrate the importance of spatial immune landscapes in PDAC and identify potential therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87979b18b917426107034a7951a93426ded0148e" target='_blank'>
              Computational modeling of cellular influence delineates functionally relevant and distinct cellular neighborhoods in primary and metastatic pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Yeonju Cho, Jae W. Lee, Sarah M. Shin, Alexei G Hernandez, Xuan Yuan, Jowaly Schneider, Jody E. Hooper, Laura D. Wood, E. Jaffee, Atul Deshpande, W.J. Ho
          </td>
          <td>2026-01-23</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The tissue architecture of classic Hodgkin Lymphoma (CHL) is unique among cancers and characterized by rare malignant Hodgkin and Reed-Sternberg cells that co-evolve with a complex ecosystem of immune cells in the tumor microenvironment (TME). The lack of a comprehensive systems-level interrogation has hindered the description of disease heterogeneity and clinically relevant molecular subtypes. Here, we employed an integrative, multimodal approach to characterize CHL tumors using malignant cell sequencing, spatial transcriptomics and imaging mass cytometry. We identified four molecular subtypes (CST, CN913, STB, and CN2P), each characterized by distinct clinical features, mutational patterns, malignant cell gene expression profiles, and spatial architecture involving immune cell populations. Functional modeling of CSF2RB mutations, a characteristic feature of the CST subtype, revealed dysregulated oncogenic signaling and unique TME crosstalk. These findings highlight the significance of multi-dimensional profiling in elucidating patterns of molecular alterations that drive immune ecosystems and underlie therapeutically exploitable vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6324873c99ed0026f9e046294cb9724bf214ad" target='_blank'>
              Multidimensional Characterization of Tumor-Immune Architecture Reveals Clinically Relevant Classic Hodgkin Lymphoma Subtypes.
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew Lytle, Yifan Yin, Makoto Kishida, Michael Yu Li, Cecilia Lee, Denise Smorra, L. Hilton, S. Healy, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O Brien, J. Wong, Claudia Cassidy, Manabu Fujisawa, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Rangel Patiño, Michael Hong, Shaocheng Wu, Eric Lee, Ali Sakhdari, Katsuyoshi Takata, T. Miyata-Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander Xu, A. Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, R. Morin, A. Alizadeh, K. Savage, David W Scott, Christian Steidl
          </td>
          <td>2026-01-30</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Chronic liver disease (CLD) represents a major global public health challenge, necessitating a systematic understanding of its complex immunopathological mechanisms. This review comprehensively summarizes the groundbreaking applications of cutting-edge technologies—including single-cell sequencing, spatial transcriptomics, and organoid models—in chronic liver disease immunology research: Single-cell sequencing resolves immune cell heterogeneity at unprecedented resolution, identifies rare cell subsets, and reveals dynamic changes and regulatory networks through multi-omics integration; Spatial transcriptomics complements this by mapping immune-stromal interactions within structural contexts such as the portal tract, fibrotic septa, and tumor niches, uncovering spatially organized immune evasion mechanisms and microenvironmental remodeling; Organoid technology constructs humanized liver-immune models that recapitulate disease-specific features—such as fibrosis, steatohepatitis, and hepatocellular carcinoma—enabling mechanistic validation, drug screening, and individualized therapeutic exploration. The synergistic integration of multi-omics profiling, spatial mapping, and organoid modeling is driving a paradigm shift in chronic liver disease immunology—transitioning from static cellular descriptions to spatiotemporal mechanism decoding, and from population-level insights to individualized pathophysiology and treatment prediction. These advanced approaches establish a technological foundation for building precision immunotherapeutic strategies tailored to spatiotemporal regulation of the liver immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1390600cdb0085be9c69b75a3de5a6e93ec2fda5" target='_blank'>
              Integrated multi-omics, spatial profiling and organoid modeling drive transformative advances in chronic liver disease and hepatocellular carcinoma immunomicroenvironment research
              </a>
            </td>
          <td>
            Qi Liu, Chenyu Wang, Cheng Ye, Huawei Zhang, Teng Li, Wenjuan Wei, Senyan Wang, Huapeng Zhang
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background The spatial distribution and functional heterogeneity of tumor-infiltrating lymphocytes (TILs) significantly impact patient outcomes in non-small cell lung cancer (NSCLC). While T cell factor 1 (TCF1) expressing TILs have emerged as key players in sustaining anti-tumor immunity, their subset characterization, localization, and clinical significance within the tumor microenvironment remain poorly defined. Method We performed multiplex immunohistochemistry and immunofluorescence to characterize TCF1+ immune cell subsets, in 102 NSCLC tumors, separately analyzing the tumor center (TC) and invasive front (IF). We integrated this data with publicly available single-cell RNA-sequencing datasets and clinical outcome analyses. Results CD4+ T cells and CD79α+ B cells, dominate the TCF1+ landscape, while CD8+ T cells constitute a minority of TCF1+ immune cells, particularly in the TC. We demonstrated the presence of tumor-infiltrating IgG+/IgA+ plasma cells co-expressing TCF1. PD1+TCF1- cells were more frequent than PD1+TCF1+ cells both in the TC and IF, reflecting that terminally differentiated exhausted TILs predominate within the tumor microenvironment. Survival analyses revealed significantly different prognostic impact of TILs including TCF1-expressing cells based on topography. Multivariate analysis showed that increased CD8+TCF1+ cells (HR: 2.5; p=0.039) and increased TCF1 expression by cancer cells (HR: 2,7; p=0.041) in the TC and CD4+TCF1+ cells (HR: 0.4; p=0.043) in the IF emerged as negative and positive independent prognostic markers for Disease-free survival (DFS), respectively. Integrating PD-L1 expression with TILs, PD-L1 immunopositivity was correlated with increased CD8+ and PD1+TCF1- cell infiltration and was associated with favorable DFS especially in the TC. Conclusions Our findings support a more refined framework for TCF1+ TIL assessment and TCF1 expression across cellular populations in the tumor microenvironment, with implications for prognostication in operable NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb42a95fd19ed986ce09103139534fde5284c2fa" target='_blank'>
              Immunophenotyping TCF1-expressing TILs: spatial profiling and prognostic value in operable non-small cell lung cancer
              </a>
            </td>
          <td>
            Konstantinos Ntostoglou, Georgios Christodoulopoulos, Katie Stoker, Jean Descarpentrie, A. Xagara, D. Chatzidaki, Vasiliki Anastasopoulou, Ilias P. Nikas, A. Ieronimaki, Lucy Booth, Sophia Tsoka, Ioannis Vamvakaris, A. Klinakis, Eleni Patsea, Sophia N. Karagiannis, Ioannis G. Panayiotides, Teresa Frisan, Vassilis Georgoulias, A. Kotsakis, I. Pateras
          </td>
          <td>2026-01-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Distal cholangiocarcinoma (dCCA) arises from the distal bile duct and is anatomically embedded within the pancreatic head, adjacent to abundant autonomic nerve plexuses. This unique location renders dCCA particularly prone to perineural invasion (PNI), a pathological hallmark that contributes to its dismal prognosis. However, the spatial architecture and molecular drivers that orchestrate PNI remain poorly defined. Here, we applied Xenium subcellular resolution spatial transcriptomics platform to profile resected tumor tissues from dCCA patients stratified by PNI status pathologically. A spatially resolved atlas comprising a total of 20 cell types was generated, uncovering enrichment of Schwann cells, type 2 conventional dendritic cells (cDC2), M2-like macrophages, cancer associated fibroblasts (CAFs) and B/plasma cells in PNI-high tumors, along with depletion of exhausted CD8+ T cells. Heterogeneous malignant cells in PNI-high tumors demonstrated activation of extracellular matrix remodeling and axonogenesis pathways, in line with the initial pathological classification. Spatial mapping further revealed distinct PNI-associated niches, notably matrix-producing CAFs (mCAFs)-macrophage clusters exhibiting coordinated enrichment of inflammatory and fibrotic programs. We further identified the LAMB3-DAG1 axis as a potential mediator of dCCA cells-Schwann cell interaction, while the preferential proximity of arteries to Schwann cells suggested additional microenvironmental support for nerve invasion. Collectively, our study provides a comprehensive subcellular atlas of PNI in dCCA, uncovering coordinated epithelial, stromal, and immune remodeling that drives perineural invasion. The identified biomarkers not only hold promise for patient stratification but may also guide intraoperative navigation and surgical margin determination, offering new avenues for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7ed53173a0ee6bb5d8c5c5791ee85be378c7900" target='_blank'>
              A subcellular spatial atlas illuminates the microenvironmental remodeling of perineural invasion in distal cholangiocarcinoma
              </a>
            </td>
          <td>
            Fansen Ji, Hao Chen, Huan Li, Jiawei Zhang, Sijia Li, Pengfei Wang, Hao Liu, C. Ge, Bingjun Tang, Hongfang Yin, Xuedong Wang, Jiahong Dong
          </td>
          <td>2026-01-08</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Molecular subtyping of bladder cancer (BCa) using bulk transcriptomic data is hindered by tumor microenvironment (TME) contamination and intra-tumoral heterogeneity. These factors also contribute to variable responses to immune checkpoint inhibitor (ICI) therapy. Single-cell RNA sequencing offers a high-resolution approach to overcome these limitations by enabling tumor-specific analysis and improving therapeutic target identification.


MATERIALS AND METHODS
We constructed a large multicenter single-cell atlas of BCa by integrating nine cohorts comprising 123 samples and 842 127 cells. Tumor epithelial cells (n = 265 962) were identified via copy number variation (CNV) analysis using inferCNV. Pseudobulk RNA profiles were generated to simulate conventional bulk samples (Pseudobulkall) and purified tumor profiles (Pseudobulktumor), enabling systematic evaluation of bulk-based classifiers. Inter-tumoral and intra-tumoral heterogeneity programs were defined based on transcriptional profiles and validated through in vitro and in vivo experiments. Additionally, we cross-analyzed five immunotherapy datasets encompassing 13 immune response phenotypes to identify molecular features associated with therapeutic resistance and sensitivity.


RESULTS
Unsupervised clustering identified 11 major cell types and 55 non-epithelial subtypes, with TME heterogeneity driven by tissue origin and malignancy status. Existing molecular classifiers UNC (University of North Carolina at Chapel Hill) showed compromised concordance between Pseudobulkall and Pseudobulktumor, particularly in stroma-rich samples. We defined three inter-tumoral (GEinter2-4) subtypes with distinct CNV and transcription factor signatures that remained stable even at low tumor purity. Seven intra-tumoral (GEintra) programs captured key biological states, including an EMT/Stem-like module associated with immunosuppression and ICI resistance. Mechanistically, TGF-β1 induced expression of EMT/Stem-like genes (e.g., PTHLH), while Pthlh knockdown synergized with anti-PD-L1 therapy in murine models by enhancing T cell cytotoxicity without altering T cell infiltration.


CONCLUSIONS
This study provides a robust single-cell framework for resolving inter- and intra-tumoral heterogeneity in BCa, overcoming limitations of bulk-based subtyping. The TGF-β1-PTHLH axis contributes to ICI resistance, positioning PTHLH as a promising therapeutic target. These findings offer new insights for improving immunotherapeutic strategies in BCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d5eb03c1d8a0a6927854d1055fb19018c50409" target='_blank'>
              Integrated multicenter single-cell atlas of bladder cancer reveals tumor heterogeneity associated with immunotherapy.
              </a>
            </td>
          <td>
            Ziwei Wang, Wei He, Maoyu Wang, Meimian Hua, Yidie Ying, Chen Zhang, Qing Chen, Yue Wang, Xufeng Yu, Yi Wang, Huiqing Wang, Zhensheng Zhang, Shuxiong Zeng, Chuanliang Xu
          </td>
          <td>2026-02-03</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Stage III colorectal cancer poses a significant threat of metastasis development, as tumour resection and adjuvant chemotherapy do not guarantee prolonged disease-free survival. Objective The spatial, quantitative, and qualitative characteristics of various cell types within tumour tissues could be key to developing accurate prognostic AI models. Design Tissue microarrays created from primary tumour tissues collected during surgical resection from a cohort of 493 stage III colorectal cancer (CRC) patients were analysed for 61 protein markers at the single-cell level using multiplexed immunofluorescence imaging via the Cell DIVE™ platform. Subsequent cell-type classification enabled quantitative cell-type analyses, co-localisation neighbourhood assessments, and cell-type-specific protein signature discoveries that distinguish between early and late/non-recurring patient samples. Results This study identifies a stem cell protein profile that drives tumour relapse. A deep neural network (DNN) model, based on a stem cell protein signature composed of BAX, MLKL, FLIP, GLUT1, and CDX2, provided accurate prognosis for stage III CRC patients in both discovery and validation cohorts and in an independent validation cohort. Nodal count-based metric further increased prognosis accuracy. Our study also revealed distinct spatial arrangements of immune, endothelial, and stem cells that were linked to early tumour recurrence. Conclusion Our findings propose a clinically promising prognostic tool based on a five-protein stem cell signature. These markers not only predict chemotherapy resistance in cancer stem cells but also suggest potential therapeutic strategies such as combinatorial treatments incorporating small molecule inhibitors targeting FLIP and GLUT1. Key messages What is already known on this topic More than 20% of stage III colorectal cancer patients will experience early tumour recurrence within the first 3 years post treatment that includes surgery and adjuvant 5-FU based chemotherapy treatment. Several studies pointed towards involvements of number of cell type specific spatial neighbourhoods in tumour progression where some immune tumour microenvironment promoting angiogenesis and intravasation events, some may provide immunosuppression. Cancer stem cells could be responsible for metastatic tumour spread, early recurrence and chemoresistance. What this study adds Spatial single cell quantitative multiplex profiling of 45 cancer hallmark proteins and 15 cell identity markers in 493 stage III CRC patients tissue samples demonstrated significant differences in cellular proximity neighbourhoods, cell type specific abundance and expression between the early and late recurrence samples. We discover that macrophages show spatial association with the blood vessels in early recurrence samples. Moreover, we observed conglomeration of B cells and macrophages with Tregulatory, Thelper and Tcytotoxic cells in association with early recurrences. We showed that stromal abundance of Tregulatory, Thelper, Tcytotoxic cells and monocytes are significantly in late, and no recurrence samples compared to early recurrence samples. The most differential expression profile that differentiates late and no recurrence samples from the early recurrence samples is related to the stem cell population. Particularly, we found overexpression of GLUT1, FLIP and downregulation of BAX, BAK, MLKL and CDX2 proteins in the cancer stem cell of early recurrence samples. We built a neural network based on the cancer stem cell protein signature (BAX, MLKL, FLIP, GLUT1 and CDX2 proteins) that delivers a high-performance prognostic classifier. How this study might affect research, practice or policy Our results propose a clinically promising prognostic tool based on a five-protein stem cell signature that outperforms existing clinical and proposed transcriptomic based signatures for separation between risk groups. Moreover, our five-protein signature markers not only predict stem cell chemotherapy resistance and therefore tumour recurrence but also suggest potential therapeutic strategies. For instance, this approach could guide combinatorial treatments at high risk of chemoresistance, such as incorporating small molecule inhibitors targeting FLIP (currently in discovery phase) and GLUT1 (already under preclinical trial evaluation).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f56eac89fc15024b42c21684b63bce91fb777dcd" target='_blank'>
              A neural network model delivers a highly prognostic protein signature in cancer stem cells that identifies relapse in stage III colorectal cancer patients
              </a>
            </td>
          <td>
            Anna Sturrock, Sanghee Cho, M. Salvucci, M. Sturrock, Joana Fay, Tony O’Grady, E. McDonough, Christine Surrette, J. Shia, C. Firat, Nil Urganci, Batuhan Kısakol, E. O’Connell, John P. Burke, N. McCawley, Deborah A. McNamara, John Graf, Simon S. McDade, Mohammadreza Azimi, Daniel B. Longley, F. Ginty, J. Prehn
          </td>
          <td>2026-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b55c41a895a4562f500627e7c41a6885170b27" target='_blank'>
              Clonal morphology-guided combination therapies overcome heterogeneity-driven drug tolerance
              </a>
            </td>
          <td>
            Xiaoxi Li, Ling Liu, Yuanyuan Yan, Mengfang Yang, Lingli Luo, Yong Jiang, Weijun Ren
          </td>
          <td>2026-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d7c1910610648515431116d59c0a3a122ecb43" target='_blank'>
              Multimodal Data Fusion Reveals Morpho-Genetic Variations in Human Cortical Neurons Associated with Tumor Infiltration
              </a>
            </td>
          <td>
            Yufeng Liu, Zhixi Yun, Lingli Zhang, Wen Ye, Kaifeng Chen, Xiefeng Wang, Mengzhu Ou, Jing Rong, Xiaoming Yang, Lei Mao, Liang Chen, Nan Ji, Yongping You, Chiyuan Ma, Ying Mao, Junxia Zhang, Liwei Zhang, Hanchuan Peng
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Osteosarcoma (OS) exhibits substantial genomic complexity and inter-patient heterogeneity, necessitating longitudinal, patient-matched analyses to understand acquired features of tumor evolution. However, most published OS data is limited to primary tumor samples, limiting insight into patient-specific resistance mechanisms. To address this, we characterized the genomic landscape of paired primary and metastatic tumor samples from dogs with spontaneous OS. Whole-genome and single-cell RNA sequencing reveal mutation and gene expression profiles that are predominantly organized by patient identity. Mutational burden and pathway alterations such as those involving PI3K, NOTCH, TP53, MAPK, RAS and epigenetic regulation differ between primary and metastatic samples. Variants present in tumor tissue are readily detectable in paired cfDNA samples, demonstrating the utility of this assay for identifying tumor-specific alterations associated with treatment resistance. Analysis of bulk RNA-seq data to estimate cell-type composition shows greater immune cell representation in metastases, underscoring the importance of immune signaling pathways in OS. These findings exemplify the presence of patient-specific alterations in genomic architecture over the course of tumor progression, linking CNV amplification, pathway reprogramming, and immune evasion in metastatic OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian M. Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ae5051d7b08a91cd55f10476439cbdbebb052d3" target='_blank'>
              Temporal and spatial composition of the tumor microenvironment predicts response to immune checkpoint inhibition in metastatic TNBC.
              </a>
            </td>
          <td>
            Noah F. Greenwald, I. Nederlof, Cameron Sowers, D. Ding, Seongyeol Park, Alex Kong, Kathleen E. Houlahan, Sricharan Reddy Varra, M. De Graaf, V. Geurts, Candace C. Liu, Jolene S Ranek, L. Voorwerk, Michiel de Maaker, Adam Kagel, E. Mccaffrey, Aziz Khan, C. Yeh, C. Fullaway, Zumana Khair, Brennan G. Simon, Yunhao Bai, Hadeesha Piyadasa, Tyler Risom, A. Delmastro, Felix J Hartmann, Lise Mangiante, Cristina Sotomayor-Vivas, Sean C. Bendall, T. Schumacher, Zhicheng Ma, Marc Bosse, M. van de Vijver, Robert Tibshirani, Hugo M Horlings, Christina Curtis, Marleen Kok, Michael Angelo
          </td>
          <td>2026-02-18</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Introduction Mismatch repair-deficient (dMMR) colorectal cancers (CRCs) exhibit variable clinical responses to immune checkpoint inhibitors (ICIs) despite their high immunogenicity. Methods To investigate the molecular and spatial determinants of this heterogeneity, we analyzed five metastatic sites (colon, liver, peritoneum, left ovary, and right ovary) from a dMMR CRC patient treated with pembrolizumab using bulk and single-cell RNA sequencing combined with multiplex immunohistochemistry. Results Responsive lesions were characterized by enriched cytotoxic and interferon-γ signatures, greater CD8+ T-cell infiltration, and close spatial proximity between programmed death-ligand 1(PD-L1)+ tumor cells and M1-like macrophages. In contrast, resistant lesions demonstrated reduced effector cell presence and were enriched in immunosuppressive programs, including SPP1+ and CD163+ macrophages, noncanonical WNT5A/B signaling, and TGF-β–mediated matrix remodeling. Conclusion Collectively, these findings highlight the importance of spatial immune architecture and macrophage polarization in shaping ICI responses in dMMR CRC and underscore the need for spatial profiling to guide immunotherapy strategies in metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7dee5edb3a5c46063fed969eabd6424ae6c3c71" target='_blank'>
              Comprehensive spatial and immune profiling of metastatic mismatch repair–deficient colorectal cancer reveals response to immunotherapy
              </a>
            </td>
          <td>
            Ilkyu Park, Hari Kang, Se-Hee Kim, YunJae Jung, Won-Suk Lee
          </td>
          <td>2026-01-09</td>
          <td>Immunotherapy Advances</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The release of circulating tumor DNA (ctDNA) is modulated not only by intrinsic tumor features but also by the tumor microenvironment (TME). In luminal breast cancer, distinct microenvironment niches may impact ctDNA release, yet the cellular and molecular mechanism that link TME composition to ctDNA dynamics remain poorly defined.



 We integrated single-cell full-length transcriptomic, mutational and spatial data from tumor tissue with mutation profiles from paired plasma of ten treatment-naïve early-stage HR+/HER2- breast cancers. A deep-learning multi-omics framework mapped ctDNA release at single-cell resolution by tracing mutations shared between tumor subclones and plasma ctDNA. First, we found microenvironmental niches linked with ctDNA release. Then for every malignant cell, we calculated the immune neighborhood and performed unsupervised clustering of epithelial cells, yielding five epithelial clusters. Afterwards, five TME-neighborhood signatures derived from these clusters were applied to METABRIC and TCGA cohorts to stratify patients into five microenvironment-defined groups. Normalized ctDNA release scores, mutation burden, and functional signatures were compared across groups; prognostic value was evaluated with 21-gene and 70-gene assays and 5-year survival.



 Microenvironmental niche analysis identified three TME architectures associated with ctDNA release: an intra-ductal luminal-progenitor niche attracting B cells, a hot immune niche enriched for T cells and macophages, and an immunomodulated, stemness niche featured by M2 macrophages. Furthermore, integration of single-cell neighborhoods with bulk transcriptomes aligned five microenvironment-defined patient groups (G1–G5) to the underlying cell clusters (C1–C5). G1/C1—enriched for intra-ductal luminal progenitors, B cells, and DCs —showed the highest per-epithelial-cell ctDNA release, yet the lowest total ctDNA burden because of its small malignant compartment. G2 and G3, linked to C2/C3, displayed antigen-presentation and estrogen-response phenotypes enriched for T cells and macrophages. G4/C4 formed an immune-quiescent compartment. G5/C5 combined the largest malignant compartment with prominent M2-TAM, DC3 and pDC infiltration plus high CD274 expression, producing the highest ctDNA release and the most immunomodulatory microenvironment. Clinically, the gradient from G1 to G5 paralleled increasing 21-gene and 70-gene recurrence scores and decreasing survival (p < 0.001). Elevated ctDNA-release signatures independently associated with worse 5-year survival (HR = 1.63, 95 % CI 1.2–2.2), confirming microenvironment-shaped ctDNA dynamics as a non-invasive prognostic biomarker.



 TME architecture impacts ctDNA release in luminal breast cancer. Immune-rich, B-cell/DC niches limit total shedding despite high single-cell release efficiency, whereas immunomodulatory, PD-L1-high niches maximize ctDNA output and portend poor prognosis. TME-informed ctDNA metrics provide a non-invasive tool for immune monitoring and precision risk stratification.



 Hengyi Xu, Pengming Pu, Jiaqi Liu. Single-cell spatial immune architectures shape ctDNA release and prognosis in luminal breast cancer [abstract]. In: Proceedings of the AACR Immuno-Oncology Conference (AACR IO): Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2026 Feb 18-21; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2026;14(2 Suppl):Abstract nr A037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4f79cbe11ecf7e46eb8edb04b49765c49cf20c8" target='_blank'>
              Abstract A037: Single-cell spatial immune architectures shape ctDNA release and prognosis in luminal breast cancer
              </a>
            </td>
          <td>
            Hengyi Xu, P. Pu, Jiaqi Liu
          </td>
          <td>2026-02-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b936d6bfcdfb468036f4e78c02a0afc5a831e69" target='_blank'>
              Spatially resolved diversity in molecular states underlies congenital melanocytic nevi and associated tumors
              </a>
            </td>
          <td>
            Daniel Aldea, Sarah Guégan, Israa Fakih, Mathias Moreno, A. de la Fouchardière, D. Pissaloux, F. Tirode, J. Puig-Butillé, J. Villanueva-Cañas, Teresa Torres-Moral, Anton Forniés-Mariné, S. Mallet, Nathalie Degardin, A. Barlier, Susanna Puig, Sylvie Fraitag, Cristina Carrera, Nicolas Macagno, H.C. Etchevers
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02ad1efaa62d98c7f16dd57c71f8980e8844d4c9" target='_blank'>
              Tumor microenvironment signature associated with morphology in systemic ALK-positive anaplastic large cell lymphoma
              </a>
            </td>
          <td>
            Chloé Bessiere, Loélia Babin, Sandra Dailhau, P. Gaulard, Stéphane Pyronnet, Fabienne Meggetto, Laurence Lamant
          </td>
          <td>2026-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a highly heterogeneous B‐cell malignancy, spanning a spectrum from indolent conventional forms (C‐CLL) to Richter transformation. Accelerated CLL (A‐CLL) represents an intermediate subtype, histologically and clinically more aggressive than C‐CLL, yet its diagnosis relies on subjective histological criteria. We hypothesized that high‐dimensional single‐cell morphometrics from routine hematoxylin and eosin‐stained lymph node biopsies could provide complementary metrics for positioning cases along the disease continuum. We implemented a novel, open‐source methodology using CellProfiler and Seurat to segment cells and measure hundreds of parameters from two indolent and two aggressive cases. Datasets were integrated to mitigate technical batch effects. Our analysis combined an unsupervised approach with trajectory analysis and a pathologist‐guided supervised machine learning model. Both unsupervised and supervised approaches successfully and independently distinguished aggressive from indolent cases. Aggressive cases were enriched in morphometric clusters corresponding to paraimmunoblasts, while indolent cases were enriched in those corresponding to small lymphocytes. Pseudo‐bulk analysis using a subset of key parameters also successfully classified patients. In this highly exploratory, proof‐of‐concept study utilizing a small, well‐characterized cohort (n = 4 CLL patients plus 2 RT reference cases), we present a robust and traceable single‐cell morphometric pipeline. By providing objective metrics of cellular and subcellular morphology, our method reveals strong segregation patterns reflecting the CLL continuum. Our findings warrant validation in larger cohorts, but offer novel, quantifiable insights that could potentially complement diagnostic criteria and aid in patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/950ccdcd5de9e7cf83d8ab0daab4aa818198a571" target='_blank'>
              Single‐Cell Morphometrics From Hematoxylin and Eosin‐Stained Images Reveals Subtype‐Specific Features in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            M. Lorenzo, Juan Quarroz Braghini, Gregorio Cordini, Tomás Lombardo, Laura Kornblihtt, Guillermo Blanco
          </td>
          <td>2026-01-01</td>
          <td>Cytometry Part A</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The tumor microenvironment (TME) poses a major barrier to effective immunotherapy, yet high-throughput perturbation-mapping approaches to dissect TME spatial complexity and its contextual immune modulators remain lacking. Here, we introduce CRISPR-laser-captured microdissection (LCM) integration mapping of the tumor-immune microenvironment (CLIM-TIME), a scalable platform that integrates CRISPR screening with LCM of metastatic tumors for transcriptomic, deconvolution, and immunofluorescence analyses. CLIM-TIME enables spatially resolved mapping of how tumor suppressor gene (TSG) loss reshapes the TME and modulates immune responses. We identified seven distinct TME subtypes, revealing that DNA repair and Polycomb repressive complex (PRC) TSG loss is linked to immune-infiltrated TMEs sensitive to T cell therapy. In contrast, knockouts of TSGs in the Hippo pathway promoted immune evasion and therapy resistance by fostering myeloid-enriched but T cell-excluded TMEs with elevated extracellular matrix (ECM). Targeting the ECM-crosslinking enzyme LOXL2 effectively remodeled the metastatic TME, enhancing T cell infiltration and improving therapeutic efficacy in lung metastases across multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a648a685f5af4c0a496d9f1d7cc52384d6bf90ac" target='_blank'>
              CLIM-TIME identifies metastatic microenvironment modulators for T cell therapy response.
              </a>
            </td>
          <td>
            Yinghua Wang, Weiwei Hu, Rui Xia, Xianfa Yang, Yange Gu, Zihan Ning, Tiange Yang, Chune Yu, Lulu Zhang, Dun Li, Yitian Jin, Jianhua Li, Feifei Zhang, Yaochen Xu, Chenqi Xu, Zhengxin Wang, Naihe Jing, Luonan Chen, Guangchuan Wang
          </td>
          <td>2026-02-01</td>
          <td>Cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Lung cancer remains a leading cause of cancer-related mortality owing to its aggressiveness and pronounced heterogeneity. Cancer stem cells (CSCs) have been implicated in tumor progression and therapeutic resistance; however, their transcriptional programs and spatial organization in non-small cell lung cancer (NSCLC) are not fully characterized. Here, we integrated single-cell RNA sequencing with spatial transcriptomics to systematically define CSC-associated phenotype in NSCLC. Weighted gene co-expression network analysis identified a 127-gene CSC signature enriched for stemness-related pathways, with DSG2 emerging as a dominant marker. Elevated DSG2 expression was associated with chemotherapy resistance and predicted response to EGFR tyrosine kinase inhibitors based on independent clinical proteomic datasets. Spatial mapping revealed preferential enrichment of DSG2+CSCs at tumor margins, where they co-localized with FAP+myofibroblasts (myCAFs). Multiplex immunofluorescence demonstrated that FAP+myCAFs expressed MMP9 and MMP12, consistent with a microenvironment supportive of epithelial-mesenchymal transition and CSC maintenance. Functional co-culture assays showed that myCAFs enhanced CSC-associated phenotypes in DSG2^high tumor cells in an MMP-dependent manner. Collectively, these findings delineate a spatially organized DSG2+CSC-myCAF niche that contributes to therapeutic resistance in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a5313526bb6f0160b56c7fabe77fa7494a3d29e" target='_blank'>
              DSG2+ Cancer Stem Cells Co-Located With FAP+ Myofibroblasts in the Tumor Boundary That Determines the Efficacy of Immunotherapy in Non-Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            G. Fan, Le Tang, Tongji Xie, Mengwei Yang, Lin Li, Sheng Yang, P. Xing, Xiaohong Han, Yuankai Shi
          </td>
          <td>2026-02-15</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background
Pancreatic ductal adenocarcinoma (PDAC) remains among the most lethal solid tumors, largely due to its intricate and immunosuppressive tumor microenvironment (TME). While single-cell sequencing technologies have begun to unravel the cellular heterogeneity of PDAC, a comprehensive understanding of how genetic determinants influence and are influenced by the TME is still lacking. To bridge this knowledge gap, our study employs an integrated multi-omics approach, incorporating single-cell transcriptomics, genomics, and proteomics, complemented by computational biology and machine learning. We aimed to delineate the core molecular drivers of PDAC pathogenesis, with subsequent in vitro functional validation focusing on the role of UNC93B1 in malignant phenotypes. The ultimate goal of this research is to inform the development of precise therapeutic strategies to enhance patient survival and quality of life.


Methods
We assembled a comprehensive multi-omics dataset, including single-cell RNA-seq data from 22 PDAC samples (GSE154778, GSE212966), bulk transcriptomic cohorts (GSE28735, GSE62452), survival data from the TCGA-PAAD project (n=172), spatial transcriptomics, and genome-wide association study data (bbj-a-140, n=196,187). The single-cell data were processed using Seurat v5, which involved rigorous quality control, batch effect correction with Harmony, unsupervised clustering, and cell type annotation to characterize TME heterogeneity. Genetic susceptibility was mapped onto single-cell data using scPagwas to calculate trait-regulated scores (TRS) and identify trait-associated genes. Co-expression networks were constructed via high-diversity WGCNA (hdWGCNA), and key candidate genes were refined through survival analysis and a machine learning framework integrating LASSO regression, Random Forest, and Support Vector Machine algorithms. The functional role of the pivotal gene, UNC93B1, was systematically investigated through Gene Set Variation Analysis (GSVA), pseudotime trajectory inference (Monocle2), and cell-cell communication analysis (CellChat). In vitro validation was performed using four PDAC cell lines (PANC-1, BxPC-3, Capan-1, SW1990). Following qPCR confirmation of high UNC93B1 expression, a stable knockdown model (sh-UNC93B1) was generated in Capan-1 cells. Functional consequences were assessed using CCK-8, wound healing, transwell and colony formation assays. A subcutaneous xenograft model was established to evaluate tumor growth in vivo. Mechanistic insights were gained through flow cytometry for cell cycle analysis and molecular profiling of the cGAS-STING pathway, senescence markers (e.g., p16^INK4a), and epithelial-mesenchymal transition (EMT)-related genes.


Results
Single-cell transcriptomic profiling delineated nine distinct cell populations within the PDAC TME. hdWGCNA identified three gene modules (8, 11, 16) positively associated with tumorigenesis. The intersection of these modules with differentially expressed genes yielded 320 candidates, which were subsequently filtered to 61 genes significantly linked to patient prognosis (P < 0.05) via Cox regression. Cross-validation across machine learning models and scPagwas analysis converged on UNC93B1 as the sole overlapping gene with consistent diagnostic and prognostic relevance. UNC93B1 was robustly upregulated in tumor tissues across independent datasets (TCGAxGTEx, bulk RNA-seq), a finding corroborated at the protein level by HPA and CPTAC data (P < 0.01). Its expression positively correlated with higher pathological grade and was spatially enriched within tumor regions. Functional enrichment analysis (GSVA) suggested that UNC93B1 is involved in the suppression of the cGAS-STING signaling axis. Pseudotime analysis indicated that UNC93B1 expression escalates along tumor progression trajectories. CellChat suggested strengthened intercellular communication networks in UNC93B1-high cells, particularly modulated by the cGAS-STING pathway. In vitro, UNC93B1 knockdown in Capan-1 cells significantly attenuated proliferative capacity (22.3% reduction in OD450 at 72h, P < 0.05), migratory ability (29.6% reduction in wound closure, P < 0.05), and clonogenic survival (342 fewer colonies, P < 0.01). Mechanistically, sh-UNC93B1 cells exhibited G1/S phase arrest (8.9% increase, P < 0.05), activation of the STING/IFN-β/CXCL10 cascade, elevated p16^INK4a expression, and a reversal of EMT, evidenced by downregulation of VIM and upregulation of CDH1. Consistently, in vivo xenograft experiments demonstrated that UNC93B1 silencing markedly impeded tumor growth, concomitant with reduced UNC93B1 protein and enhanced STING pathway activation. Critically, the tumor-suppressive phenotypes induced by UNC93B1 knockdown, including the inhibition of proliferation, migration, and clonogenicity, were largely reversed upon treatment with the selective STING inhibitor H-151, confirming that the observed functional consequences are causally mediated through the activation of the cGAS-STING pathway.


Conclusion
By integrating multi-omics data, including GWAS and spatial transcriptomics, this study systematically defines a pivotal role for UNC93B1 in PDAC progression. Our findings demonstrate that UNC93B1 is associated with an immunosuppressive TME and facilitates metastatic spread, potentially through inhibiting the cGAS-STING-mediated innate immunity pathway. The strong correlation between UNC93B1 overexpression and adverse clinical outcomes underscores its potential as a dual diagnostic biomarker and therapeutic target. This work not only provides a mechanistic foundation for novel precision immunotherapies in PDAC but also establishes a robust methodological paradigm for multi-omics-driven discovery in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bb10353febe80e8ef377d1403743c79b95316f" target='_blank'>
              UNC93B1 promotes pancreatic cancer progression through modulation of cGAS-STING signaling.
              </a>
            </td>
          <td>
            Hao Yang, Yukun Li, Jing Su, Haiyan Zhang, Wenxin Zhu, Kanger Shen, Wei Xu
          </td>
          <td>2026-02-04</td>
          <td>Frontiers in immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Hepatocellular carcinoma (HCC) exhibits pronounced intratumoral heterogeneity and a complex immune microenvironment, which together limit therapeutic efficacy. Identifying actionable biomarkers and mechanisms of immune modulation remains critical for improving patient outcomes. Method We integrated single-cell RNA sequencing data with bulk transcriptomic datasets to comprehensively characterize tumor cell heterogeneity and immune landscape features in HCC. Associations between RDH16 expression and clinicopathological characteristics were evaluated, and in vitro functional assays were conducted. Mendelian randomization analyses were performed to assess causal relationships. Results RDH16 expression was significantly reduced in HCC tissues compared with non-tumor liver tissues and was associated with vascular invasion, elevated alpha-fetoprotein levels, poor tumor differentiation, and advanced T stage. Higher RDH16 expression was consistently associated with improved overall prognosis. Functional assays indicated that RDH16 did not directly affect tumor cell proliferation, migration, or invasion. Notably, RDH16 expression was inversely correlated with infiltration of CD163⁺ M2 macrophages, suggesting a potential immunomodulatory role in the tumor microenvironment. Mendelian randomization analyses further supported a protective effect of higher RDH16 expression against HCC risk. Discussion These findings identify RDH16 as an immune-modulating biomarker in HCC, highlighting its potential role in shaping the tumor immune microenvironment and suggesting new avenues for personalized immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/444ab6526b7dd129322dafa940c094f836b618fb" target='_blank'>
              Single-cell transcriptomics reveals a novel mechanism of RDH16 regulating immune infiltration in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zhenzhen Zhang, Rui Fan, Jing Ma, Dehui Li, Yan Jiang, Fahui Liu, Qiming Gong
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/706f238b315ed6aac316adf86f94df56f8c2fd50" target='_blank'>
              An integrated single-cell lung cancer atlas reveals distinct fibroblast phenotypes between adenocarcinoma and squamous cell carcinomas
              </a>
            </td>
          <td>
            Yuta Hirano, Hironori Suzuki, Jun Nakayama, Tomofumi Yamamoto, Eishu Hirata, Jun Araya, Yu Fujita, Yusuke Yamamoto
          </td>
          <td>2026-01-23</td>
          <td>npj Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Multiple myeloma (MM) is recognized as a malignancy shaped by its complex tumor microenvironment (TME), which fuels disease progression and therapeutic resistance. Recent advances in single-cell omics, spatial transcriptomics, mass cytometry, and advanced imaging have enabled high-resolution mapping of tumor and immune cell interactions within their native context, also revealing spatial heterogeneity that influences clinical outcomes. These tools, complemented by scalable computational frameworks and artificial intelligence, provide cost-effective alternatives to dissect immune landscapes and derive prognostic biomarkers from both bulk and single-cell data. However, technical complexity, resource demands, and the need for robust standardization limit their immediate clinical application. On the other hand, machine learning techniques enhance integration and predictive power of existing datasets, supporting the development of personalized, immune-informed therapeutic strategies. This review highlights recent advances, discusses the strengths and limitations of emerging technologies with a particular focus on their integration to decipher TME biology and pave the way toward precision medicine in MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/085f22e2309e321faeb8024153146e39d1ce17a9" target='_blank'>
              Cutting-Edge Technologies to Decode Tumor Microenvironment in Multiple Myeloma.
              </a>
            </td>
          <td>
            Doriana Gramegna, P. Mondelli, S. A. Pappagallo, G. Gargano, M. C. Vegliante, A. Quinto, Bernardo Rossini, Gianvito Pasciolla, A. Roccaro, Michele Cea, Enrico Iaccino, A. Guarini, S. Ciavarella
          </td>
          <td>2026-02-08</td>
          <td>Hematological oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Breast cancer is genetically and histologically heterogenous, and is influenced by a variety of factors, including the tumor microenvironment (TME). The PAM50 subtypes; Luminal A, Luminal B, Normal-like, Basal-like and Her2-enriched, are associated with different tumor phenotypes and overall survival. The quantity and quality of immune cell infiltration in breast tumors play a key role in cancer development and progression and are associated with survival and treatment response. We used multiplex immunohistochemistry, single-cell RNA sequencing, and two deconvolution algorithms to explore the immune landscape across PAM50 subtypes. Immunostaining of CD3+ T cells, tryptase+ mast cells, CD20+ B cells, CD68+ CD163+ macrophages, and CD66b+ granulocytes revealed marked differences in tumor immune infiltrates according to breast cancer subtypes. Luminal tumors were relatively deprived of T cells and B cells, while exhibiting sparse to moderate amounts of macrophage infiltration. In contrast, Her2-enriched tumors exhibited a moderate immune presence, with sparse to moderate T cell infiltration and moderate infiltration of B cells and macrophages. At the other end of the spectrum, Basal-like tumors stood out for their strikingly rich immune environment and are heavily infiltrated by T cells, B cells, and macrophages. The results from the single-cell and deconvolution analyses confirmed subtype-specific immune microenvironments, which also allowed us to observe increased levels of natural killer (NK) and CD8+ T cells in Her2-enriched and Basal-like subtypes. In conclusion, our findings demonstrate significant differences in the immune tumor microenvironment between the established breast cancer molecular subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c99c23b4dfb7da23c9b0f3a691c05aec9a1b68" target='_blank'>
              Multi-platform analysis of the tumor immune microenvironment associated with breast cancer subtypes
              </a>
            </td>
          <td>
            Lilly Anne Torland, Jürgen Geisler, Y. Azimzade, Astri Frafjord, Hogne Røed Nilsen, Linn Buer, J. Lømo, Ø. Garred, Andliena Tahiri, M. Lyngra, Inger Øynebråten, Alexandre Corthay, Kristine Sahlberg, Vessela N. Kristensen, Xavier Tekpli
          </td>
          <td>2026-01-13</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Remodeling of the tumor microenvironment (TME) under therapeutic pressure is a critical determinant of treatment response and resistance in hepatocellular carcinoma (HCC). Triple-combination therapy integrating targeted agents, immune checkpoint inhibitors, and radiotherapy (T+I+R) has shown potential synergistic effects in intermediate to advanced HCC, particularly in patients with portal vein tumor thrombus (PVTT), yet the spatial and cellular mechanisms underlying its efficacy remain largely unknown. In this retrospective clinical cohort study, we compared T+I+R with targeted therapy plus radiotherapy (T+R) in advanced HCC, and further employed single-cell spatial transcriptomics and spatial proteomics to generate an integrated multi-omics atlas mapping tumor and stromal compartments, cellular compositions, and intercellular interactions with spatial resolution. Clinically, T+I+R achieved superior tumor shrinkage and disease control compared with T+R. Spatial multi-omics revealed marked region-specific remodeling, with myofibroblastic cancer-associated fibroblasts, angiogenic tip endothelial cells, and conventional dendritic cells enriched at the invasive margin and associated with therapeutic resistance, while CD8+ effector T cells were redistributed away from immunosuppressive niches, a spatial configuration correlating with enhanced response. These findings identify spatial segregation between cytotoxic and suppressive immune elements as a potential hallmark of effective therapy, providing a high-resolution spatial framework for understanding T+I+R induced TME remodeling and offering mechanistic insights to guide biomarker discovery and the optimization of combination strategies in advanced HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d6c3edb9469068b18d4bde88276d91593b43ce6" target='_blank'>
              Spatial multi-omics mapping of tumor microanatomy dynamics following radiotherapy combined with targeted-immunotherapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Fansen Ji, Haoming Xia, Ying Xiao, Jiawei Zhang, Hao Chen, Bingjun Tang, Huan Li, Hao Liu, Boyang Wu, Xiaojuan Wang, Shizhong Yang, Jiahong Dong
          </td>
          <td>2026-01-08</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Bone sarcomas remain lethal despite multimodal therapy, primarily because the mineralized, immunosuppressive tumor microenvironment (TME) promotes chemo- and immune-resistance. Integrating single-cell and spatial omics across osteosarcoma, Ewing sarcoma and chondrosarcoma delineates subtype-specific TME archetypes dominated by M2 macrophages, exhausted T cells and a stiff extracellular matrix. Mechanistic dissection reveals tractable vulnerabilities, myeloid reprogramming, extracellular matrix modulation and metabolic and epigenetic checkpoints, that can be targeted with bone-selective delivery systems and biomarker-driven combination trials to convert therapeutic failure into durable remission. Therefore, the aim of the present review is to synthesize the latest single-cell, spatial and functional data to map bone-sarcoma TME heterogeneity, dissect resistance mechanisms and propose integrated, biomarker-guided therapeutic strategies that can be translated into treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a42d68e5855e4884a22e5af558a0428be6a50022" target='_blank'>
              Tumor microenvironment in bone sarcomas: Implications for immunotherapy and emerging therapeutic vulnerabilities (Review)
              </a>
            </td>
          <td>
            Wentao Li, Lijun Lv, Yibin Jin, Xin Yuan
          </td>
          <td>2026-01-16</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is a lethal subtype of prostate cancer characterized by its androgen independence, resistance to chemotherapy, and display of neuroendocrine features which can emerge either de novo or via transformation after a prior diagnosis of adenocarcinoma. The poor clinical outcomes in patients with AVPC are associated with its profound molecular heterogeneity. In this study, we analyzed 23 consecutive AVPC cases treated at a dedicated small-cell clinic (2017-2025) using clinicogenomic and transcriptomic profiling. Transformed AVPC exhibited significantly shorter overall survival times than de novo AVPC (11.8 vs 26.0 months, P < 0.001). Integrative genomic analyses identified residual androgen signaling in subsets of cases harboring neuroendocrine lineage programs, highlighting a decoupling of lineage identity and morphology. To facilitate mechanistic and pharmacologic studies, we established NCI-LYM-1, a patient-derived organoid/PDX from an AR-negative, ASCL1+/SYP+ lymph node metastasis, which faithfully recapitulates the donor tumor's molecular and phenotypic features. Short- and long-read whole-genome sequencing combined with optical genome mapping identified biallelic inactivation of PTEN, TP53, RB1 and BRCA2 as potential drivers, demonstrating clonal concordance with circulating tumor DNA from the original patient donor. Pathway and perturbation analyses suggested that NCI-LYM-1 harbored a strong dependency on apoptotic pathways, which was confirmed by in vitro organoid testing with the BCL-2/BCL-xL inhibitor navitoclax (IC50: 0.27 M) and the MCL-1 inhibitor AZD-5991 (IC50: 0.060 M). Overall, NCI&-LYM-1 recapitulates the clinical aggressiveness and heterogeneity of AVPC, providing a tractable platform to identify novel precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e71e79208c8d37fdc48221a6ff9b64fb28fe818c" target='_blank'>
              Molecular decoupling of lineage identity and morphology in aggressive variant prostate cancer
              </a>
            </td>
          <td>
            Chennan Li, JuanJuan Yin, Melissa L Abel, Dana S Vargas Solivan, Kinjal Bhadresha, Sumeyra Kartal, Samantha Nichols, Kanak Parmar, Joseph Twohig, Tri M Truong, C. Chau, Kathleen Kelly, W. Figg, Anish Thomas, A. Sowalsky
          </td>
          <td>2026-01-09</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Lactate accumulation is a central feature of tumor metabolic reprogramming, yet its spatial and cell-type-specific effects in cancer, such as lung adenocarcinoma (LUAD), remain poorly defined. We integrated single-cell transcriptomics, spatial transcriptomics, spatial metabolomics, and immunofluorescence with TCGA survival data and machine-learning models. High-lactate tumors exhibited increased epithelial and fibroblast abundances, whereas T/NK cells and monocytes/macrophages were enriched in low-lactate samples. Spatial metabolomics revealed cell-type–restricted lactate and pyruvate distributions, with endothelial cells showing minimal lactate accumulation. Endothelial subclusters in high-lactate tissues displayed angiogenic and stress-response signatures and were strongly associated with poor prognosis. Multiple machine-learning frameworks—including random forest, elastic-net regression, SVM, ANN, and decision-tree models—consistently identified endothelial and fibroblast programs as key determinants of high-lactate states and adverse clinical outcomes. Collectively, our multi-omics spatial profiling demonstrates that lactate reshapes the LUAD microenvironment by driving angiogenesis, immune suppression, and prognostic stratification, highlighting lactate-centered pathways as promising therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a59f51e7895cef79efeacf0e1e7ab108d79743ff" target='_blank'>
              Machine learning-enabled spatial multi-omics uncovers lactate-driven targets and tumor microenvironmental reprogramming in cancer
              </a>
            </td>
          <td>
            Yingzheng Tan, Wenliang Tan, Yanchao Liang, Yunzhu Long, Shuanghua Chen, Qihao Hu, Yangjing Ou, Jing Fu, Huan Chen, F. Ren, Jun Ye, Qing Zhou, Sheng Li, Xiaojin He, Qianqian Wang, Yu Shen, Haiyuan Lu, Daichao Wu, An-bo Gao, Xun Chen, Yukun Li
          </td>
          <td>2025-12-30</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Nodal involvement constitutes a pivotal prognostic indicator in colorectal carcinoma, yet the transcriptional machinery governing lymphatic dissemination and tumor‐microenvironment crosstalk remains poorly elucidated. Conventional bulk sequencing methodologies lack sufficient resolution to deconvolve functionally distinct malignant subclones that drive the metastatic cascade. Methods We employed an integrative analytical framework combining tissue‐level gene expression profiling from TCGA and GEO repositories with eight single‐cell transcriptomic datasets comprising 266,995 individual cells. A phenotype‐guided computational algorithm was implemented to delineate metastasis‐driving malignant populations through correlating clinical parameters with cellular transcriptional profiles. Gene regulatory networks and transcription factor activity inference systematically decoded the molecular programs underlying metastatic phenotypes. Ligand‐receptor pairing analysis mapped intercellular communication architectures between neoplastic cells and microenvironmental constituents. Experimental validation encompassed genetic perturbation studies, functional characterization assays, and pharmacological response evaluation in preclinical systems. Results We discovered a phenotypically distinct malignant population exhibiting robust associations with lymph node involvement and adverse clinical outcomes across nine independent validation cohorts. Regulatory network dissection identified IRF9 as the master transcriptional orchestrator of this metastatic program through coordination of a discrete gene module. Relative to their nonmetastatic counterparts, these aggressive cells establish markedly expanded intercellular communication networks, characterized by prominent VEGF‐driven angiogenic signaling to endothelial compartments and integrin‐laminin–mediated adhesion with stromal elements. EFNA1 emerged as a key signaling mediator demonstrating preferential enrichment in this metastatic subset. Elevated EFNA1 levels correlated with progressive disease stages and microsatellite‐stable subtypes while showing inverse relationships with PD‐L1 expression and T cell infiltration density—suggesting a unique immunoevasion mechanism. Genetic ablation of Efna1 substantially impaired cellular proliferation, motility, and invasion capabilities, while simultaneously augmenting Linifanib sensitivity, indicating synergistic therapeutic potential. Conclusions Our investigation reveals a transcriptionally defined malignant population under IRF9 control that orchestrates immunosuppressive microenvironmental reprogramming via EFNA1‐mediated signaling networks. The EFNA1–Linifanib combination may represent a potential therapeutic approach to mitigate anti‐angiogenic resistance and restrain metastatic progression in colorectal carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/213ee7511b6e61fa3300d628f006d8273257969a" target='_blank'>
              Integrative Multiscale Analysis Reveals EFNA1‐Driven Immune Remodeling Promotes Colorectal Cancer Lymph Node Metastasis
              </a>
            </td>
          <td>
            Wu Ning, Nan Qiao, Lei Zhou, Zongze Li, Lin Zhang, Xin Song, Shaoqiu Chen
          </td>
          <td>2026-01-01</td>
          <td>Human Mutation</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Colorectal ovarian metastasis (CROM), a distinct metastatic subtype of colorectal cancer (CRC), is associated with early disease onset and aggressive progression. CROM lacks specific treatment options, highlighting the need to elucidate the underlying biological mechanisms and potential therapeutic vulnerabilities. In this study, we performed integrated analyses of single-cell RNA sequencing (scRNA-seq) datasets from 155,163 cells across 35 patients from the in-house cohort and public datasets, with matched bulk transcriptomic profiling. The analysis identified AKT3⁺ EMT-like cells at the invasive tumor-stroma interface as metastasis-initiating cells. Functional validation using in vivo xenograft models demonstrated that AKT3 deficiency reduced ovarian colonization, while AKT3 overexpression conferred a mesenchymal phenotype with invasive capacity. Furthermore, reciprocal crosstalk between AKT3⁺ mesenchymal-like cells and cancer-associated fibroblasts (CAFs) played a key role in remodeling the tumor microenvironment. Multiplex immunofluorescence staining of primary tumor specimens revealed spatially coordinated AKT3+/SNAIL+/ITGB1+ tumor buds adjacent to α-SMA+ CAFs at the invasive front. Critically, AKT3 inhibition or knockdown in patient-derived CROM organoids (CROM-PDOs) significantly suppressed malignant phenotypes, recapitulating the AKT3 dependency. Collectively, these findings elucidate an AKT3-driven feedforward loop coupling EMT plasticity with CAF activation as a critical driver of CROM and propose CROM-PDOs as a robust platform for developing precision therapies targeting this aggressive CRC subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bd536a96521b7d0987ed07a91fda25944479eb4" target='_blank'>
              AKT3-Driven Epithelial-Mesenchymal Plasticity Governs Ovarian Metastasis in Colorectal Cancer via Tumor Microenvironment Remodeling.
              </a>
            </td>
          <td>
            Jingyi Shi, Xiaowen Wang, Wutong Zhang, Zhaoya Gao, Chang Zhang, Zixin Tao, Yong Yang, Jingxuan Xu, Haopeng Hong, Yunan Ma, Baojun Chen, Yunfan Wang, D. Ji, Ming Li, Guifang Jia, Jin Gu
          </td>
          <td>2026-02-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a leading cause of cancer-related mortality worldwide, largely due to late-stage diagnosis and the limited sensitivity of current biomarkers such as α-fetoprotein (AFP). Early detection requires molecularly defined targets that capture the initial steps of malignant transformation. Single-cell RNA sequencing (scRNA-seq) offers high-resolution insight into tumor heterogeneity and lineage progression to enable the identification of early biomarkers. This study aimed apply scRNA-seq analysis to detect clinically important molecular patterns that define the early stages of malignant transformation in HCC and facilitate the diagnosis of small or ambiguous lesions. Methods Two independent scRNA-seq datasets (GSE149614 and GSE189903) comprising non-tumor and HCC tissues were analyzed. Following batch correction and clustering, hepatocyte subpopulations were characterized by differential expression, pseudotime, and CytoTRACE analyses to reconstruct the trajectory from normal to malignant states. High-dimensional weighted gene co-expression network analysis (hdWGCNA) was used to identify stage-associated modules, while CellChat and protein-protein interaction analyses delineated intercellular signaling networks. Target expression was validated in paired human liver specimens using quantitative immunofluorescence. Results scRNA-seq revealed a continuum of hepatocyte states characterized by progressive stemness and oncogenic pathway activation (MYC, E2F, G2M). Module Hep-M20 exhibited the strongest correlation with tumor stage and identified BSG/CD147 as a central hub gene with monotonic upregulation along pseudotime and strong correlation with stemness potential. CellChat analysis uncovered a cyclophilin (PPIA/PPIB)-dependent tumor-stroma signaling axis that positions BSG/CD147 as the key mediator for intercellular communication between tumor hepatocytes, fibroblasts and T cells. Ex vivo validation confirmed significantly higher BSG/CD147 protein expression in HCC versus background liver (P = 2.9×10−11) with excellent diagnostic accuracy (AUC = 0.93–0.96; sensitivity 86–87%; specificity 93–97%) including in lesions < 2 cm that are frequently indeterminate on conventional imaging. Conclusions This study establishes BSG/CD147 upregulation as an early molecular event in hepatocarcinogenesis that integrates hepatocyte dedifferentiation, microenvironmental signaling, and tumor progression. Strong and specific expression in small lesions < 2 cm underscores potential as a precision biomarker and imaging target for early HCC detection, risk stratification, and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91b69381a1a33432cf5fe739884155cd3bedeaf6" target='_blank'>
              Multimodal single-cell network analysis uncovers BSG/CD147 as an early biomarker and signaling hub in hepatocellular carcinoma
              </a>
            </td>
          <td>
            T. D. Wang, Eun-Young K. Choi, Shuo Feng, Hui Jiang, T. D. Wang, Sangeeta Jaiswal
          </td>
          <td>2026-01-27</td>
          <td>Research Square</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor evolution involves genetic, transcriptional, and phenotypic alterations that shape cancer cell behavior and interactions with the microenvironment. While single-cell technologies have advanced our understanding of this process, spatial dynamics remain incompletely characterized. Here, whole-exome sequencing (WES), imaging mass cytometry (IMC), and spatial transcriptomics (ST) were integrated to study molecular evolution and immune responses in two lung adenocarcinoma (LUAD) mouse models: a genetically engineered model (129S4/Sv-KrasLSL-G12D, termed 129S4 K) and a carcinogen-induced precancer model (129S4 U). Compared to 129S4 K, 129S4 U tumors exhibited higher mutational, neoantigen but lower copy number variation (CNV) burdens at matched developmental timepoints, consistent with findings of higher mutational burden in human smoking-related LUAD than nonsmoking LUAD. We profiled over 1.4 million spatial single cells from 284 IMC regions of interest and 51,531 spatial transcriptomic spots from 156 lesions across 141 mice. Macrophage abundance increased with tumor progression, while CD8 T-cell and B-cell densities declined in late-stage LUAD. 129S4 U showed greater immune infiltration in both tumor and adjacent normal tissue, higher T-cell cytotoxicity signature score in line with its higher mutational and neoantigen burdens. LUAD progression was marked by early morphological shifts and late-stage changes in cell states and interactions. These data define spatial and genetic landscapes of LUAD development and provide a framework for investigating immune evolution and therapeutic strategies in early carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e7ad7c0953d4cab7fb1f9dcb251447c108ead" target='_blank'>
              Spatial Profiling Reveals Distinct Molecular and Immune Evolution of Mouse Lung Adenocarcinoma Precancers with or Without Carcinogen Exposure.
              </a>
            </td>
          <td>
            Bo Zhu, M. Aminu, Pingjun Chen, Jian-Rong Li, Chuanpeng Dong, Chenyang Li, Yanhua Tian, Shaowen Lu, Hong Chen, Chenxi Ma, Xin Hu, Jie-Tong Ye, Andrew Liu, Beibei Huang, Frank R. Rojas, Parra Cuentas Edwin Roger, Ou Shi, M. Nilsson, A. Poteete, Khaja B Khan, Wei Lu, Luisa M. Solis Soto, Junya Fujimoto, C. Haymaker, Ignacio I. Wistuba, Zhubo Wei, Linghua Wang, Don L. Gibbons, Ken Chen, Alexandre Reuben, Jason M Schenke, J. Heymach, Chao Cheng, Jia Wu, Jianjun Zhang
          </td>
          <td>2026-01-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cef7edb3493fb7ab6d3516e83b3f06b2dd845210" target='_blank'>
              VAETracer: Mutation-Guided Lineage Reconstruction and Generational State Inference from scRNA-seq
              </a>
            </td>
          <td>
            Linnuo Pan, Kaiyu Wang, Luonan Chen
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Understanding the spatio-temporal processes of tissue growth, including how new cell types emerge and how cells form the tissue architecture, is a fundamental problem in biology. The emerging spatially resolved lineage tracing data, where three modalities, lineage barcodes, gene expression profiles, and spatial locations, are measured for each single cell, provides an unprecedented opportunity to understand these processes. Computational methods that take advantage of all three modalities to reconstruct cell lineage tree and ancestral cell states and locations are needed. We introduce LineageMap, a hybrid lineage inference algorithm that integrates the scalability of distance-based tree reconstruction methods with the flexibility of likelihood-based methods under a unified probabilistic framework. The input to LineageMap is spatially resolved lineage tracing data, where for each single cell, the gene expression, lineage barcode and spatial locations are available. LineageMap enables accurate, interpretable, and scalable inference of high-resolution lineage trees as well as locations of ancestral cells from the tri-modality single-cell data. Across simulated and experimental datasets, LineageMap consistently outperforms existing methods in the accuracy of reconstructed cell lineage trees, while revealing biologically coherent spatiotemporal trajectories. Our framework bridges molecular lineage tracing with spatial and transcriptomic information, advancing computational reconstruction of dynamic cellular ancestries in both time and space. LineageMap is available at: https://github.com/ZhangLabGT/LineageMap.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a6f6d0a1453abdee44ce8239ca0e225b2481a85" target='_blank'>
              Integrative Inference of Spatially Resolved Cell Lineage Trees using LineageMap
              </a>
            </td>
          <td>
            X. Pan, Yiru Chen, Xiuwei Zhang
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The progression of prostate cancer (PCa) to metastatic disease remains a critical clinical challenge, making it essential to gain a detailed understanding of the factors that enable certain subclones to invade and spread. Building on our previous study, which examined genomic variations in benign and malignant prostate tissues by inferring copy number alterations (CNA) from spatial transcriptomics data1, this study aims to trace the spatial evolutionary trajectories of PCa subclones from the primary tumor to matched lymph node metastases and to identify alterations in associated microenvironments and early diagnostic markers. To achieve this, we performed spatial transcriptomics on entire prostate axial disks and patient-matched lymph node metastases from 10 individuals. Standard spatial transcriptomics (55 µm) was used for tissue profiling; from the spatial transcriptomics data, copy number alterations (CNAs) were inferred to identify distinct tumor subclones, and a clone-tree was constructed to describe their evolutionary relationships. Additionally, a high-resolution spatial transcriptomics (2 µm) technology was employed to gain a cellular-level view of the immediate tumor-microenvironment (TME) surrounding specific subclones. Analysis of over 1,000,000 barcoded regions identified many distinct tumor subclones, enabling us to trace their evolutionary trajectories spatially. We observed notable subclonal events within the lymph nodes, including polyclonal colonization, indicating multiple dissemination events during the evolution of the primary disease. Exploration of the immediate TME revealed significant cellular heterogeneity and upregulation of genes related to antigen presentation and inflammatory pathways concentrated near ancestral tumor clones, specifically at the tumor border. By focusing on the inferred CNA profiles of the metastasizing clone across patients, we identified several common features. Additionally, we were able to identify potential lethal disease as early as the diagnostic core needle biopsy stage by detecting features of metastasis and pinpointing the metastasizing clone. In summary, our study provides a detailed spatial map of PCa clonal evolution and dissemination, linking primary tumors to nodal metastases and revealing altered cell composition and gene expression around tumor clone borders. Importantly, it demonstrates subclonal events within lymph nodes, polyclonal colonization, and the potential to identify metastasizing clones as early as the diagnostic phase, which holds strong implications for precision oncology and early intervention.
 1. Erickson, A., He, M., Berglund, E. et al. Spatially resolved clonal copy number alterations in benign and malignant tissue. Nature 608, 360–367 (2022). https://doi.org/10.1038/s41586-022-05023-2



 Mengxiao He, Sandy Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, Ian G. Mills, Joakim Lundeberg, Alastair D. Lamb. Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604426087ba878274c47234a7ce00263acf256a4" target='_blank'>
              Abstract PR013: Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers
              </a>
            </td>
          <td>
            Mengxiao He, S. Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, I. Mills, J. Lundeberg, Alastair D. Lamb
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e78908da394712537095f77c8b8136eb7ecf216" target='_blank'>
              Evolution of oncogene amplification across 86,000 cancer cell genomes
              </a>
            </td>
          <td>
            J. Lee, Sohrab Salehi, Matthew A. Myers, , Melissa A. Yao, Duaa H. Al-Rawi, , Eric G. Sun, Kerstin Thol, Seongmin Choi, Eliyahu Havasov, Asher Preska Steinberg, Michelle Wu, Nicole Rusk, Caitlin Timmons, Cheryl Zi Jin Phua, Stephen Martis, Neeman Mohibullah, P. Manoj, E. Redin, Á. Quintanal-Villalonga, P. Razavi, Sam Aparicio, Natasha Rekhtman, Viviane Tabar, Mark M. Awad, H. Yu, K. Yu, Andrea Ventura, Andrew Mcpherson, C. Rudin, Sohrab P. Shah
          </td>
          <td>2026-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Double-Negative T (DNT) cells, lacking both CD4 and CD8 expression, play critical roles in cancer immunology, and have garnered increasing attention in cancer research. However, their heterogeneity and functional diversity within the tumor microenvironment (TME) remain underexplored. In-house and publicly available single-cell RNA sequencing (scRNA-seq) data for different cancer types were integrated after quality control and batch effect correction, followed by DNT cells separation from CD3+ T cells subtypes. Functional characteristics, intercellular communication, differentiation trajectories, regulatory networks, and clinical relevance were analyzed among different DNT subsets. Key findings were validated using multiplex immunofluorescence and spatial transcriptomics to investigate the spatial localization of DNT subsets and their interactions within the TME. Impact of γδ T cells on immunotherapy response was also assessed using MC38-based murine tumor model. By integrating scRNA-seq data from 2,369 samples across 23 cancer types, we established a comprehensive single-cell atlas of 157,025 high-quality DNT cells. Fourteen distinct DNT subsets (6 αβ DNT and 8 γδ T cell subsets) were identified, demonstrating tumor both type-specific and shared distribution patterns, as well as unique cell-cell interaction network within the TME. These subsets displayed specialized functional profiles, including cytotoxicity, antigen presentation, and immune modulation, indicating that the functional diversity of DNT cells is largely subset-specific rather than a manifestation of multifunctionality within a single population. We also delineated divergent trajectories for αβ DNT and γδ T cell subsets, including the functional plasticity of gut-resident γδ T cells transitioning between cytotoxic and immunosuppressive states. Notably, several DNT subsets were significantly associated with favorable clinical treatment outcomes, including improved responses to cancer immunotherapy. Consistently, depletion of γδ T cells in the murine tumor model significantly decreased the efficacy of PD-1 blockade, underscoring their critical role in therapeutic response. Our study uncovers the previously underappreciated heterogeneity and functional diversity of DNT cells in the TME and demonstrates their profound impact on tumor progression and immunotherapy outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4d4eea61e2f155251ae235dac3fc5c5c43d18" target='_blank'>
              A pan-cancer single cell landscape reveals heterogeneity and functional diversity of double-negative T cells
              </a>
            </td>
          <td>
            Qing Hao, Tao Zhou, Huayun Yan, Zhixiang Ren, Wei Mao, Li-Bin Huang, Yue-Yun Chen, Yuelan Wang, Minqin Xiao, Yajiao He, Xuyang Xia, Jing Wu, Wei-Han Zhang, Hai-Ning Chen, Maoxiang Qian, Feng Zhang, Li Yang, Lunzhi Dai, Huiyuan Zhang, Canhua Huang, Yang Shu, Yu Xu, Yang Luo, Yong Peng, Heng Xu
          </td>
          <td>2026-01-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity and plasticity. GBM tumors contain neoplastic cells within a fluid spectrum of diverse transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we develop scFOCAL (Single-Cell Framework for -Omics Connectivity and Analysis via L1000), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identify compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states, and leverage this capability to predict combination synergy. We validate the significance of these findings in vitro, ex vivo, and in vivo, and identify a combination of an OLIG2 inhibitor and Depatux-M for the treatment of GBM. Our studies suggest that scFOCAL identifies cell states that are sensitive and resistant to targeted therapies in GBM using a measure of cell and drug connectivity, which can be applied to identify new synergistic combinations. Glioblastoma is characterised by high levels of intratumoural heterogeneity and plasticity, hindering treatment. Here, the authors develop an analytical framework, scFOCAL, to predict the sensitivity of glioblastoma cell subpopulations to therapies based on reversal of disease transcriptional signatures to identify synergistic therapeutic combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef55e3b53c83b7969434910c5b4838c55ce9672" target='_blank'>
              Drug and single-cell gene expression integration identifies sensitive and resistant glioblastoma cell populations
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, R. Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, R. Komotar, Macarena I. de la Fuente, Gregory Stein, Alexandre Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, Nagi G. Ayad
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Highlights What are the main findings? We established and characterized a unique pair of isogenic cervical cancer cell lines (AdMer35/AdMer43) from a patient with squamous cell carcinoma of the cervix before and after chemoradiotherapy, revealing that resistance is not caused by a single factor but by complex reorganization of cancer cells. Single-cell analysis identified key resistance hallmarks: transcriptional reprogramming towards embryonic stemness (HOX, POU5F1), a shift from fibrillar to non-fibrillar collagen expression, and activation of interferon/inflammatory pathways. What are the implications of the main findings? The study maps a remodeled tumor ecosystem with a metabolically reprogrammed, pro-tumorigenic senescent-like cell population and a progenitor-like cell reservoir, identifying them as critical therapeutic targets for eliminating resistant cell pools. The identified key pathways, Wnt, TGFβ, BMP, and the specific ligands EREG and SEMA3C, along with the metabolic vulnerabilities of the senescent compartment, provide a robust framework for future therapeutic exploration. Abstract Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V Semenov, Irina S Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Somatic mutations accumulate during the first zygotic division and continue throughout an organism’s lifespan. The characteristics and frequency of these mutations are contingent on developmental timing and tissue type, giving rise to somatic mosaicism, defined as the presence of unique genomic alterations across different cells. They serve as endogenous cellular barcodes, enabling detailed reconstruction of cell lineages and clonal dynamics. Although lineage tracing techniques have advanced from early microscopic observation and dye staining to the introduction of artificial barcodes via gene editing, owing to ethical considerations, such genetic manipulations in human developmental research are unavailable. Therefore, spontaneously arising somatic mutations are the most suitable strategy for tracing human lineages. Current approaches can be broadly categorized into two strategies: (i) high-resolution methods, including single-cell clonal expansion or laser-capture microdissection, which construct precise phylogenetic trees based on shared mutation profiles; and (ii) bulk sequencing methods, which infer lineage proximity by comparing variant allele frequencies across samples. As more lineage-tracing studies are being conducted focusing on a wider variety of organs, the integration of such data will make it possible to discover the general principles governing human development. This review highlights how the concept of somatic mutations has been applied across diverse biological contexts and discusses the insights and common principles that can be drawn from these findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b635d70a77d777444fb98a226730074956b4517" target='_blank'>
              Reconstructing developmental lineages: a retrospective approach using somatic mutations and variant allele frequency
              </a>
            </td>
          <td>
            Mahnoor Sajjad, Seong Gyu Kwon
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Angioimmunoblastic T-cell lymphoma (AITL) represents the second most prevalent subtype of peripheral T-cell lymphoma, characterized by a dismal prognosis. However, a systematic exploration of tumor microenvironment (TME) features and their prognostic significance in AITL remains notably deficient. To address this knowledge gap, we conducted spatial transcriptome sequencing (ST-SEQ) and whole-exome sequencing in four AITLs and two noncancerous lymph nodes for discovery purposes, complemented by immunohistochemistry analyses on 37 AITL cases for validation. We identified 14 ST clusters, including five neoplastic clusters, wherein a global shift in B-cell phenotypes and enrichment of myeloid cells were observed. These findings underscore a hallmark of exacerbated inflammation and immune dysregulation within the neoplastic TME. Among the 4 ST-sequenced AITLs, 3 expressed high CD40-CD40LG activity, accompanied by the upregulation of immune-suppressive-associated genes, such as CCL17 and PDCD1. Conversely, the remaining patient displayed an uncommon absence of CD40-CD40LG activity but harbored a phagocytosis-associated tumor-associated macrophage (TAM)-enriched TME, which correlated with significantly reduced relapse rates and longer event-free survival (EFS), highlighting the critical value of precise TME stratification in tailoring AITL therapeutic strategies. Finally, trajectory analysis unveiled a distinct trajectory of molecular evolution within this TME landscape. Collectively, our findings illuminate the heterogeneity and prognostic implications of the TME in AITL, providing a robust foundation for the rational design of targeted immunotherapeutic approaches. These insights may substantially advance the development of personalized treatment strategies for AITL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47887a6371994feccf99a507b06079fcceafb4cb" target='_blank'>
              Revealing tumor microenvironmental heterogeneity and prognostic value in angioimmunoblastic T-cell lymphoma via spatial transcriptome sequencing
              </a>
            </td>
          <td>
            Xiang Zhang, Yong Sun, Duo Wu, F. Yu, Hanjin Yang, X. Ye, Juying Wei, Xuewu Zhang, Yanan Zhu, Yunfei Lv, Zijing Xu, Yuxian Chen, Hongyan Tong, Jie Jin, Xiaofei Ye, Wenjuan Yu
          </td>
          <td>2026-01-09</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Prostate cancer is genomically heterogeneous with few shared drivers. We characterized tumor-specific DNA methylation programs that define evolutionary subtypes, remodel the tumor–microenvironment ecosystem, and yield liquid-biopsy biomarkers.



 Illumina EPIC-seq methylation was profiled in matched benign/tumor tissues from the UK Prostate ICGC cohort. A covariance-based DMR caller identified robust tumor-specific regions that were intersected with bulk RNA-seq, network analyses, and single-cell/spatial transcriptomics to localize programs in situ. A FACS-sorted, cell-type methylome atlas (luminal, basal, endothelial, fibroblast, immune, smooth muscle) anchored cell-of-origin assignments. Cell-free DNA was deconvolved against this atlas to quantify luminal tumor DNA and circulating microenvironmental DNA (cmDNA). Prognostic associations used penalized Cox models.



 Tumor-specific DMRs separated cancers into two “mevotypes” mirroring genomic evotypes (ERG-fusion vs SPOP-mutant), indicating genomic–epigenomic interplay in evolutionary commitment. Cell-resolved methylomes attributed most tumor DMRs to luminal epithelium and uncovered neuronal/plasticity pathways in luminal cells, while immune/endothelial-assigned DMRs converged on ECM remodeling, angiogenesis, TGF-β, and Rho-GTPase signaling. Single-cell pseudotime and spatial mapping localized neuronal-enriched luminal programs to late-trajectory regions and a CHGA/CHGB-positive duct-adjacent epithelial circuit interfacing with fibroblasts at tumor–stroma borders. Network analysis highlighted MYC-centric modules and EZH2-linked transcription factor contexts, consistent with epigenetic priming toward quasi-neuroendocrine states in treatment-naïve localized adenocarcinoma. In plasma, deconvolution detected both luminal ctDNA and cmDNA; luminal fractions tracked tumor burden in metastatic disease (ichorCNA-concordant), whereas cmDNA—predominantly attributable to immune compartments—suggested an early, tumor-specific microenvironmental response that can precede and is independent of luminal ctDNA shedding. Methylation signatures derived from neuronal and microenvironmental gene sets associated with inferior relapse-free survival.



 Epigenetic drivers in early prostate cancer encode luminal neuronal plasticity and reorganize the tumor niche. A cell-resolved atlas enables cfDNA deconvolution to reveal luminal ctDNA plus cmDNA, supporting liquid-biopsy biomarker strategies for earlier detection and improved risk stratification in prostate cancer.



 Harveer Dev, Toby Milne-Clark, Henson Lee Yu, CRUK-ICGC Prostate Cancer Group. Epigenetic Drivers of Localised Prostate Cancers Reveal Neuronal and Microenvironmental Reprogramming and inform Liquid Biopsy Biomarkers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2503538f0dcd516b3bf974c3bb6eeaf7a1544a19" target='_blank'>
              Abstract A015: Epigenetic Drivers of Localised Prostate Cancers Reveal Neuronal and Microenvironmental Reprogramming and inform Liquid Biopsy Biomarkers
              </a>
            </td>
          <td>
            H. Dev, Toby Milne-Clark, Henson L. Lee Yu
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3789c5447c0f1d5a0fe59b4f852d13e8acfbe641" target='_blank'>
              Unlocking single-cell level and continuous whole-slide insights in spatial transcriptomics with PanoSpace.
              </a>
            </td>
          <td>
            Hui-Feng He, Pai Peng, Shi-Tong Yang, Meng-Guo Wang, Xiao-Fei Zhang, Luonan Chen
          </td>
          <td>2026-01-06</td>
          <td>Nature computational science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Metabolic heterogeneity in head and neck squamous cell carcinoma (HNSCC) remains poorly characterized, limiting our understanding of how metabolic reprogramming shapes tumor progression and the tumor microenvironment (TME). Here, we aimed to elucidate the interplay between metabolic programs and immune evasion mechanisms in HNSCC.



 We performed an integrative analysis of transcriptomic data from nine independent HNSCC cohorts using pathway-level metabolic profiling. Findings were validated and expanded using single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to map cellular trajectories and cell-to-cell interactions within the TME.



 Consensus clustering identified three distinct metabolic subtypes (MS1–3), with MS3 characterized by the selective activation of glycan-related pathways and significantly poorer clinical outcomes. scRNA-seq analysis revealed that these glycan-related pathways were predominantly activated in tumor-infiltrating macrophages, which exhibited enhanced lysosomal and phagocytic activities. Trajectory analysis uncovered a spatiotemporal transition of monocyte-driven macrophages during HNSCC progression: initiating from FCN1+ monocyte-like cells, passing through intermediate states, and culminating in APOE+ lipid-associated macrophages (LAMs). Notably, this macrophage evolution closely mirrored stepwise tumor progression, where transitional precursors exhibiting glycan degradation activity emerged from early stages. Spatially, distinct architectures were observed: while monocyte-like cells resided near epithelial-like tumor cells, the evolved LAMs preferentially infiltrated the niche surrounding tumor cells exhibiting mesenchymal-like features. Within this niche, ligand-receptor interaction analysis demonstrated that these metabolically reprogrammed macrophages establish an immune-suppressive and pro-metastatic microenvironment through intercellular crosstalk with adjacent tumor fibroblasts and T cells.



 Collectively, our findings elucidate the metabolic reprogramming and spatiotemporal transition of tumor-associated macrophages, where the differentiation toward a lipid-associated phenotype promotes an immune-suppressive microenvironment enriched with mesenchymal-like tumor cells. These results suggest that targeting the metabolic vulnerabilities and specific interactions of these macrophages offers a novel therapeutic strategy to halt HNSCC progression.



 Ji-Hye Choi, Hyo Jung Kim, Bok-Soon Lee, Jin Roh, Hyun Goo Woo, Chul-Ho Kim. Integrative Single-Cell and Spatial Transcriptomics Reveal Metabolic Reprogramming and Spatiotemporal Transition of Macrophages Driving Immune Suppression in Head and Neck Cancer [abstract]. In: Proceedings of the AACR Immuno-Oncology Conference (AACR IO): Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2026 Feb 18-21; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2026;14(2 Suppl):Abstract nr A051.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd3f98cc26be21c5752cbefc21deabc33b1cd473" target='_blank'>
              Abstract A051: Integrative Single-Cell and Spatial Transcriptomics Reveal Metabolic Reprogramming and Spatiotemporal Transition of Macrophages Driving Immune Suppression in Head and Neck Cancer
              </a>
            </td>
          <td>
            , Hyo Jung Kim, , Jin Roh, Hyun Goo Woo, Chul-Ho Kim
          </td>
          <td>2026-02-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Invasive Lobular carcinomas (ILC) comprise a histologically and biologically diverse group of breast cancers. While classic invasive lobular carcinomas (ICLCs) are typically characterized by cohesive, infiltrative growth and immune-excluded microenvironments, pleomorphic invasive lobular carcinomas (IPLCs) display more aggressive behavior. Based on our previous findings of differential CAF cell densities in five phenotypes between CLCs and PLCs, we aimed to investigate the spatial relationships and distribution patterns within these phenotypes, as the spatial organization of fibroblasts and immune cells in these histologic subtypes remains underexplored.



 We performed high-dimensional multiplex immunofluorescence (mIF) spatial analysis on 12 cases of ILC (IPLCs: cases 7–12; ICLCs: cases 13–18). A comprehensive fibroblast-focused panel (including FAP, S100A4, Thy1, A-SMA) and immune markers (CD45) were used to characterize tumor (CK+), fibroblast, and immune compartments. Spatial clustering and neighborhood analysis were conducted using the Spatial Image Analysis of Tissues (SPIAT) R package v 4.5.1, to define the microenvironmental architecture.



 IPLCs demonstrated marked spatial heterogeneity, with fragmented CK+ tumor regions interspersed among stromal and immune elements. Fibroblast landscapes in IPLCs were notably diverse, featuring abundant hybrid subtypes (e.g., S100+Thy1+, A-SMA+FAP+) and frequent tumor-stroma integration within shared neighborhoods. These dynamic fibroblast populations supported active extracellular matrix (ECM) remodeling, potentially facilitating invasion and contributing to partial immune engagement. CD45+ immune cells were consistently more prominent in IPLCs, forming discrete immune-enriched clusters and mixed neighborhoods, suggesting an immune-intermediate or partially inflamed phenotype. In contrast, ICLCs displayed large, cohesive CK+ tumor regions with minimal fragmentation and low spatial heterogeneity. Fibroblast subtypes were simpler and predominantly peripherally distributed, and they rarely formed hybrid phenotypes. The stroma primarily acted as structural support, with limited ECM remodeling and minimal immune modulation. CD45+ cells were sparse and diffusely scattered, consistent with an immune-excluded (“cold”) microenvironment.



 While descriptive, our data reveal profound differences in the spatial microenvironments of IPLC and ICLCs. IPLCs exhibit a dynamic, heterogeneous stroma characterized by integrated fibroblast-tumor interactions and increased immune infiltration, aligning with their aggressive clinical behavior. In contrast, ICLCs maintain a cohesive tumor architecture with simplified, peripheral stroma and sparse immune presence. These insights highlight the significance of spatial context as an innovative and informative strategy in comprehending ILC biology and suggest that stromal and immune features may inform therapeutic strategies, including potential vulnerability to stroma- or immune-targeted interventions in IPLCs



 E. N. Barrientos-Toro, R. Pandurengan, H. Batra, Q. Ding, A. Contreras, A. A. Sahin, C. Haymaker, M. G. Raso. A Novel Spatial Profiling Approach Reveals Distinct Fibroblast and Immune Architectures in Invasive Pleomorphic versus Classic Lobular Carcinomas [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS3-12-21.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f03eb6c91ee2f8f7a8dbf22b71f475df4f1e4a1" target='_blank'>
              Abstract PS3-12-21: A Novel Spatial Profiling Approach Reveals Distinct Fibroblast and Immune Architectures in Invasive Pleomorphic versus Classic Lobular Carcinomas
              </a>
            </td>
          <td>
            E. Barrientos-Toro, R. Pandurengan, H. Batra, Q. Ding, A. Contreras, A. Sahin, C. Haymaker, M. Raso
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumor growth is constrained by spatial, mechanical, and metabolic factors whose alignment progressively breaks down across cellular, mesoscopic, and tissue scales as tumors expand. We hypothesize that this misalignment drives tumors toward a distinct architectural regime, termed hyperdisorder. Hyperdisorder is not defined by increased heterogeneity alone, but by the coexistence of elevated disorder across scales and spatial nonstationarity within the same tumor. Unlike ordinary randomness, where independent fluctuations diminish under spatial averaging, disorder here persists, reorganizes, or even amplifies with increasing observation scale, preventing convergence toward a stable architectural description. Using hematoxylin and eosin stained whole-slide images of gastric cancer from The Cancer Genome Atlas, we quantify tumor architecture using tile-based metrics that capture complementary aspects of organization, including texture entropy, microstructural fragmentation, orientation isotropy, and multiscale entropy variation. These measures are combined into a standardized hyperdisorder index, enabling unsupervised comparison across spatial regions. We find that architectural disruption is unevenly distributed and partially decoupled across scales within individual slides, consistent with growth-driven multiscale incoherence rather than uniform stochastic variability. Testable consequences include anomalous scaling of heterogeneity with sampling size, failure of coarse graining to converge, and systematic differences between tumor cores and invasive fronts. In diagnostic and clinical contexts, this framework clarifies when measurements from limited tissue samples are representative of the whole tumor and when they are dominated by scale- and location-dependent effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33332947ca1f92d72dfa2870dfb77a377f39ce75" target='_blank'>
              Hyperdisorder in tumor growth
              </a>
            </td>
          <td>
            A. Tozzi
          </td>
          <td>2026-01-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Primary breast cancer (BC) is highly heterogeneous and this drives divergent clinical outcomes. Many molecular features have significant prognostic value in hormone receptor positive (HR+) BC, and several commercial multigene prognostic assays currently guide chemotherapy decision making. The translational study from the monotherapy arms of the Arimidex, Tamoxifen, Alone or in Combination trial (transATAC), has been pivotal for the validation and regulatory approval of multiple prognostic assays. Spatial single cell analysis may lead to new insights into molecular features and tumor microenvironment (TME) that associate with prognosis and allow the future development of the next generation of prognostic assays. We present preliminary analysis of the transATAC spatial single-cell atlas of BC heterogeneity. We assess whether single cell analysis of a tissue micro-assay (TMA) core, sampling of ∼1000 cells per tumor, is sufficient to recapitulate the major features of the tumor.



 TMA blocks of 0.6mm diameter cores were generated from tumors of 1710 patients. Cores were taken from two or more distinct tumor regions in 27 patients. Every region/tumor had 3 spatially distant cores taken and spread across separate TMA blocks. We present initial analysis of single cell spatial transcriptomic analysis of 6108 genes using a CosMx spatial molecular imager from two TMAs. The preliminary analysis was generated data for 160 cores from 155 transATAC patients with data combined per patient to generate 155 tumor samples that would better represent molecular data used in prognostic assays. OncotypeDX recurrence scores on a surgery excision were available for 133 HR+ tumors. Cell type was determined by semi-supervised clustering with InSituType using BC specific reference profiles. Pseudo-bulk gene expression profiles were created by summing the total gene expression from all cells and separately all tumor cells. Large-scale chromosomal copy number alterations (CNA) in individual tumor cells were detected by InferCNV and used to understand tumor clonality.



 We analysed 155 transATAC HR+ and HR- tumours. A mean of 952 cells were analysed, with 967 transcripts from 603 genes. Across the full dataset, cell typing assigned 59% of cells to tumor cells, 21% to immune cells (T-cells, B-cells and Myeloid cells) and 20% other cells (Endothelial, Fibroblast and Pericytes). Single cell CNA analysis of individual tumors cells suggested only one patient had multiple clones (with >50 tumors cells per clone) and this patient had cores taken from 2 regions of the tumor. Recurrence scores generated from single cell pseudo-bulk data were significantly correlated with Oncotype Dx assay recurrence scores generated from surgery excisions (Pearson r=0.76; p<2.2e-16). Across all 155 tumors, 97% of ESR1 transcripts were expressed in tumor cells. Pseudo-bulk ESR1 expression from tumors cells was significantly correlated with ESR1 expression in the Oncotype assay (Spearman ρ=0.66, p<2.2e-16) and with the H score of ER protein expression (ρ=0.5, p=1.3e-9) taken from excision specimens. There was a significant inverse correlation between ESR1 expression in tumor cells and the percentage of immune cells in a tumor across all 155 samples (ρ =-0.27, p=0.0006) and for the HR+ subset with OncotypeDX scores (ρ = -0.28, p=0.001).



 Single cell spatial analysis of transATAC TMA cores reproduced key features of tumor biology assessed in resection specimens. Single cell analysis revealed an inverse association between ESR1 expression in tumor cell cells and immune cell infiltration in the tumour. Analysis of transATAC TMAs is ongoing, and updated analysis will be presented. Single cell spatial analysis is a powerful and robust technique to analyse single cell gene expression and may build a better understanding of BC biology and prognosis.



 E. F. Schuster, I. T. Kleijn, K. Dunne, R. Sardinha, E. O'Neill, G. Latifi, C. Kennedy-Dietrich, N. B. Jamieson, J. Cuzick, A. R. Brentnall, M. Dowsett, S. R. Johnston, N. C. Turner. Tumor Heterogeneity in the transATAC Breast Cancer Spatial Single Cell Atlas [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PD2-04.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8209ff126b9aa0af027a2a2bc93f9c56c7352fa9" target='_blank'>
              Abstract PD2-04: Tumor Heterogeneity in the transATAC Breast Cancer Spatial Single Cell Atlas
              </a>
            </td>
          <td>
            E. Schuster, I. Kleijn, K. Dunne, R. Sardinha, E. O'Neill, G. Latifi, C. Kennedy-Dietrich, N. Jamieson, J. Cuzick, A. Brentnall, M. Dowsett, S. Johnston, N. Turner
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="HCC is characterized by extensive ECM remodeling, primarily mediated by stromal cells. While cancer-associated fibroblasts are known contributors to tumor fibrosis, their transcriptional diversity and role in immune modulation within the HCC tumor microenvironment remain poorly resolved. Integrative analysis of six public HCC scRNA-seq datasets was employed to identify ECM-active stromal cells. High-ECM cells were re-clustered and fibroblast subtypes were defined through differential expression, pathway enrichment, lineage scoring and cell-cell interaction modeling. Spatial transcriptomic mapping and tissue-level profiling were performed to validate and prioritize ECM-related genes for experimental analysis. COL1A1 and COL3A1 were selected for siRNA-mediated knockdown in HepG2 and HepB3 cells followed by RT-qPCR, Western blot, proliferation, colony formation and wound healing assays. ECM activity analysis identified fibroblasts as the most ECM-enriched stromal population. Reclustering of ECM-high stromal cells identified eight distinct subtypes and targeted fibroblast clustering revealed five functionally diverse states. GO and spatial transcriptomic analysis confirmed subtype-specific functions and localization. In silico tissue profiling further prioritized COL1A1 and COL3A1 as pan-mesenchymal ECM genes enriched in fibrogenic fibroblast subsets. CellChat analysis revealed myofibroblasts and inflammatory CAFs as dominant signal senders. Knockdown reduced COL1A1 and COL3A1 expression at both mRNA and protein levels, and enhanced HCC cell proliferation, migration and colony formation. This study identifies transcriptionally and functionally distinct fibroblast subtypes in HCC and highlights COL1A1 and COL3A1 as key matrix-regulatory genes expressed in fibrogenic stromal subsets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2fd864648cbb63d6bef938be32a938f167aaf27" target='_blank'>
              Single-cell profiling reveals lineage-specific fibroblast stromal subtypes drive ECM remodeling and immune modulation in the hepatocellular carcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Zhichao Jiang, Haitao Wang, Han Li, Zhiqiang Chen, Baiyu Sun
          </td>
          <td>2026-01-03</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Interactions among tumor, immune, and vascular niches play major roles in glioblastoma (GBM) malignancy and treatment responses. The composition and heterogeneity of extracellular core matrix proteins (CMPs) that mediate such interactions are not well understood. Here, we present an analysis of the clinical relevance of CMP expression in GBM at bulk, single-cell, and spatial anatomical resolution. We show that CMP enrichment is associated with worse patient survival, specific driver oncogenic alterations, mesenchymal state, pro-tumor immune infiltration, and immune checkpoint expression. Matrisome expression is enriched in vascular and leading edge/infiltrative niches that are known to harbor glioma stem cells. Finally, we identify a 17-gene CMP signature, termed Matrisome 17 (M17), which is a stronger prognostic factor compared to MGMT promoter methylation status as well as canonical subtypes, and importantly, may predict responses to PD1 blockade. Patient stratification based on matrisome profiles can contribute to the selection and optimization of treatment strategies. A comprehensive genomic analysis of glioblastoma tumors reveals matrisome enrichment associated with mesenchymal phenotypes and pro-tumor immune signatures, a matrisome-based classification of GBM patients, and a prognostic 17-gene matrisome signature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984fecbf9126f6a0596ab278d28e0d91cbca56fc" target='_blank'>
              A prognostic matrix gene expression signature defines functional glioblastoma phenotypes and niches
              </a>
            </td>
          <td>
            Monika Vishnoi, Zeynep Dereli, Zheng Yin, Elisabeth Kong, Meric Kinali, Kisan Thapa, O. Babur, Kyuson Yun, Nourhan Abdelfattah, Xubin Li, B. Bozorgui, M. Farach‐Carson, Robert C. Rostomily, Anil Korkut
          </td>
          <td>2026-01-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6aac86da3cbb3bd85a8ebe57c0ff220d334a71" target='_blank'>
              Circulating immune profiling reveals impaired monocyte states and trajectories driving immunosuppression in glioblastoma
              </a>
            </td>
          <td>
            A. Scafidi, T. Kaoma, Claudia Cerella, Eleonora Campus, K. Grzyb, Bakhtiyor Nosirov, Frida Lind-Holm Mogensen, E. Klein, Raul da Costa, A. Skupin, Frank Hertel, G. Berchem, Michel Mittelbronn, A. Cosma, Beatrice Melin, Simone P. Niclou, Petr V Nazarov, A. Gołȩbiewska, Aurélie Poli, A. Michelucci
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d95637f1e45837803ecd5c25e150f1a702f3c52" target='_blank'>
              Integration of bulk and single-cell transcriptomic sequencing reveals the neutrophil heterogeneity in bladder cancer and establishes a prognostic model
              </a>
            </td>
          <td>
            Ying-Xue Song, X. Xia, Zhi-Ming Wu, Ye Yao, Jun Liang, Shengjie Guo, Kai Yao, Hui Chang
          </td>
          <td>2026-02-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The various cellular composition of the tumor microenvironment (TME) comprises the fundamental units of tumor tissue. The types of stromal cells in the TME are genetically stable, with reduced risk of tumor recurrence and drug resistance. More and more evidence shows their clinicopathological significance and therapeutic effect in predicting prognosis. Therefore, we performed an integrated analysis of the breast cancer TME, correlating it with genomic landscapes and clinical profiles. In this work, we first conducted unsupervised hierarchical clustering on 830 tumors in the breast cancer cohort. Then, we defined three TME phenotypes and applied principal component analysis to construct a TMEscore for quantifying TME. Analysis revealed that patients stratified into the high TMEscore cohort exhibited superior survival compared to the low‐scoring group. Additionally, a high TMEscore is associated with an improved response to immunotherapy. Through TME gene signature analysis, KIF15 was identified as a pivotal driver of the immunosuppressive microenvironment in breast cancer. KIF15 knockdown may promote dendritic cell infiltration and function, thereby inducing CD8+ T cell recruitment. In summary, the immune microenvironment‐derived TMEscore represents an independent prognostic biomarker in breast cancer, while KIF15 emerges as a crucial molecular determinant of its immunosuppressive niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44851ced88ac81dcb9eaccc6f7fcac4727fd09a" target='_blank'>
              Tumor Microenvironment Characterization Identifies KIF15 as an Immunosuppressive Driver in Breast Cancer
              </a>
            </td>
          <td>
            Bo Zhang, Feiran Wang, Huiwei Huang, X. Zhong, Qian Chen, Xiancheng Liu, Sujit Nair
          </td>
          <td>2026-01-01</td>
          <td>Human Mutation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Molecular classification of diseases that accurately reflects clinical behavior is fundamental to the realization of precision medicine. Single-cell protein analysis combined with coding technology offers a promising approach for constructing robust molecular classifiers. However, the low abundance of disease-related cells and the technical challenges in parallel profiling of multiple proteins remain major obstacles. Herein, we present a DNA nanostructure-based multicomponent coding strategy that enables multiprotein analysis at the single-cell level by precisely controlling the stoichiometry, orientation, and modularity of the magnetized tags and multicolor fluorescent tags. Compared with conventional linear DNA barcoding methods, our approach allows for the simultaneous magnetic separation of heterogeneous cell populations and multicolor fluorescence-based phenotypic encoding. By integrating single-cell trapping techniques, we demonstrate the accurate molecular subtyping of breast cancer based on fluorescence-encoded phenotypic features. This strategy expands the scope of applications in cell sorting, proteomic profiling, and genomic analysis, thus advancing the frontiers of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64e1af39bc3c0c6f365a94647d84b94b688262d8" target='_blank'>
              Multimodal DNA Nanostructure Barcodes for Single-Cell Protein Profiling and Tumor Subtyping.
              </a>
            </td>
          <td>
            Meirong Cui, Chenxu Hu, Jiahan Dong, Yiyu Cheng, Bicheng Sun, Chunhai Fan, Lianhui Wang, Jie Chao
          </td>
          <td>2026-02-11</td>
          <td>ACS nano</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Simple Summary Penile cancer is a rare malignancy with poor prognosis in advanced and recurrent stages. The currently limited treatment options and poor survival outcomes represent an unmet need for patients with advanced or recurrent penile cancer. Innovation in therapeutic options for patients with penile cancer relies on insight of underlying tumor biology and tumor–immune system interactions. In this paper, we use multiplex immunofluorescence to investigate the immune microenvironment of penile cancer, focusing specifically on geospatial clustering patterns of various immune effector cells within various stages of penile cancer and human papillomavirus status. We identified spatial and clustering patterns of pro-immunogenic macrophages, tumor-associated macrophages, and helper T cells as potential prognosticators of survival in penile squamous cell carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14fd3b38e31e27ab5192f8dbd3698cf76eb34d9f" target='_blank'>
              Geospatial and Cell Density Analysis Using Multiplex Immunofluorescence Reveals an Important Role of Clustering Patterns of Immunosuppressive Macrophages in Survival Outcomes of Penile Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            A. Fazili, Keerthi Gullapalli, Gabriel Roman Souza, Firas Hatoum, Justin W Miller, Youngchul Kim, Junmin Whiting, Jeffrey S Johnson, J. Dhillon, Jonathan Nguygen, C. M. Segura, Philippe E Spiess, J. Chahoud
          </td>
          <td>2026-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/283f47aefbdff4ebd6faa24c4a2c93b8e125039d" target='_blank'>
              Translating spatial transcriptomic signatures in adenosquamous carcinoma into bulk prognostic biomarkers in lung adenocarcinoma: a bottom-up approach.
              </a>
            </td>
          <td>
            K. Hatakeyama, T. Kawata, Koji Muramatsu, T. Oishi, Takeshi Nagashima, Ai Sakai, T. Nishimura, K. Ohshima, Sumiko Ohnami, S. Ohnami, Y. Shimoda, A. Naruoka, Koji Maruyama, K. Urakami, Y. Akiyama, Y. Ohde, Takashi Sugino, Ken Yamaguchi
          </td>
          <td>2026-01-23</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Penile squamous cell carcinoma (PSCC) is a rare genitourinary malignancy, and factors of its tumor microenvironment (TME) could serve as prognostic indicators for tumor recurrence and metastasis. Here, we generated a comprehensive single-cell map of PSCC (66 421 cells) and identified 9 distinct cell populations with samples from nine tumor samples and six adjacent normal samples. Among the malignant cells, SEMA3Chigh Mals was found to be associated with epithelial-mesenchymal transition. T cells in tumor tissues are in a highly exhausted state, while SPP1high TAMs were observed to promote tumor progression. Cancer-associated fibroblasts were found to interact with malignant cells to facilitate EMT through several pathways. Notably, there is a specific type of pericyte called POSTN+ pericytes, which can promote angiogenesis and extracellular matrix remodeling in PSCC. Finally, SEMA3C was identified as an effective biomarker reflecting cancer stage and microvessel density. Overall, we investigated the heterogeneity of TME from a single-cell perspective and demonstrated that SEMA3C serve as an effective biomarker for predicting lymph node metastasis and prognosis in PSCC. These findings may offer valuable insights for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b5b537b7833eaaeb7747d5123ca286f841e6ac7" target='_blank'>
              Decoding the Cellular Heterogeneity and Malignant Progression of Human Penile Squamous Cell Carcinoma by Single-Cell RNA Sequencing.
              </a>
            </td>
          <td>
            Xiheng Hu, Wensheng Shi, Liang Dong, Lingjuan Huang, Xiyuan Zhang, Yiting Feng, Jie Sun, Lanlan Liu, Teng Liu, J. Fu, Bowen Zhong, Qihao Leng, Xiaohua Wu, Minfeng Chen, Lingfang Li, Yuan Li, Xin Jin, Long Wang, Jian-xia Cao, Xin Li, Mingzhu Yin, Xiang Chen
          </td>
          <td>2026-02-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Spatial proteomics enables high-resolution mapping of protein expression and can transform our understanding of biology and disease. However, major challenges remain for clinical translation, including cost, complexity and scalability. Here we present H&E to protein expression (HEX), an AI model designed to computationally generate spatial proteomics profiles from standard histopathology slides. Trained and validated on 819,000 histopathology image tiles with matched protein expression from 382 tumor samples, HEX accurately predicts the expression of 40 biomarkers encompassing immune, structural and functional programs. HEX demonstrates substantial performance gains over alternative methods for protein expression prediction from H&E images. We develop a multimodal data integration approach that combines the original H&E image and AI-derived virtual spatial proteomics to enhance outcome prediction. Applied to six independent non-small-cell lung cancer cohorts totaling 2,298 patients, HEX-enabled multimodal integration improved prognostic accuracy by 22% and immunotherapy response prediction by 24–39% compared with conventional clinicopathological and molecular biomarkers. Biological interpretation revealed spatially organized tumor–immune niches predictive of therapeutic response, including the co-localization of T helper cells and cytotoxic T cells in responders, and immunosuppressive tumor-associated macrophage and neutrophil aggregates in non-responders. HEX provides a low-cost and scalable approach to study spatial biology and enables the discovery and clinical translation of interpretable biomarkers for precision medicine. A model trained to generate spatial proteomics profiles and predict the expression of 40 biomarkers directly from histopathology slides is shown to improve survival and treatment response prediction in patients with lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/812a5ae318d320a30cb51379e4bd5ce695c6ba0e" target='_blank'>
              AI-enabled virtual spatial proteomics from histopathology for interpretable biomarker discovery in lung cancer
              </a>
            </td>
          <td>
            Zhe Li, Yuchen Li, Jinxi Xiang, Xiyue Wang, Sen Yang, Xiaoming Zhang, Feyisope R. Eweje, Yijiang Chen, Xiangde Luo, Yuanyuan Li, Jonathan Mulholland, Colin Bergstrom, Ted Kim, F. Olguin, S. Willens, Jeffrey J Nirschl, Robert B West, J. Neal, M. Diehn, Ruijiang Li
          </td>
          <td>2026-01-01</td>
          <td>Nature Medicine</td>
          <td>2</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Aggressive variant prostate cancers (AVPC) are a poorly understood subset of prostate cancers characterized by high mortality and extensive molecular heterogeneity. To address a lack of AVPC representative models for investigating underlying molecular and physiological mechanisms, we established NCI-LYM-1, a novel AVPC organoid/patient derived xenograft (PDX) derived from a patient with metastatic prostate cancer with small cell morphology displaying a unique clinical history and defined genomic features. It was hypothesized that this model would be able to accurately characterize biological underpinnings of the patient’s tumor profile and allow for comparative analysis to similar prostate cancers.



 NCI-LYM-1 was developed from a patient-derived AR-negative, ASCL1/Synaptophysin (SYP)-positive lymph node metastasis with small cell morphology. Short-read whole-genome sequencing (WGS) was used to identify genomic alterations and long-read WGS was used to identify epigenetic modifications. RNA-seq was used to characterize gene expression profiles. The resulting genomic, epigenetic, and transcriptomic data from NCI-LYM-1 was compared to molecular profiles from the LuCaP series of PDX models. In-vitro drug sensitivity assays were also conducted using organoid cultures.



 NCI-LYM-1 harbored biallelic inactivation of known neuroendocrine prostate cancer (NEPC) drivers: PTEN, TP53, RB1 and BRCA2, in concordance with patient diagnostic history. In addition, epigenetic evidence supported a consistent phenotype between NCI-LYM-1 and the donor tumor, including greater gene expression of neuroendocrine marker ASCL1 with decreased AR expression. Comparisons between NCI-LYM-1 and LuCaP models further supported transcriptional similarities between the model and a subset of AVPC cancer subtypes. Drug sensitivity assays were also consistent with the patient’s prior treatment exposure, displaying PARP inhibitor sensitivity but a lack of sensitivity to platinum treatments.



 NCI-LYM-1 faithfully recapitulates the phenotypic and genomic characteristics of aggressive metastatic AVPCs, including the display of neuroendocrine features. NCI-LYM-1 will be a useful model system to study molecular frameworks of understudied cancers like those of AVPC, including aspects such as drug sensitivity, disease progression, and lineage plasticity.



 Ana S. Jaramillo, Chennan Li, JuanJuan Yin, Melissa L. Abel, Dana S. Vargas, Kinjal Bhadresha, Sumeyra Kartal, Joseph Twohig, Tri M. Truong, Anish Thomas, Cindy H. Chau, Kathleen Kelly, William D. Figg, Adam G. Sowalsky. NCI-LYM-1 molecularly and phenotypically recapitulates aggressive variant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483313e79c5e529d10092ffd1001e71580f4403a" target='_blank'>
              Abstract A025: NCI-LYM-1 molecularly and phenotypically recapitulates aggressive variant prostate cancer
              </a>
            </td>
          <td>
            Ana S. Jaramillo, Chennan Li, JuanJuan Yin, Melissa L Abel, Dana S. Vargas, Kinjal Bhadresha, Sumeyra Kartal, Joseph Twohig, Tri M Truong, Anish Thomas, C. Chau, Kathleen Kelly, W. Figg, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/423b9ad510934f726aebaeeb465a2ff10426e96b" target='_blank'>
              A new cancer progression model: from synthetic tumors to real data and back
              </a>
            </td>
          <td>
            Daniela Volpatto, S. G. Contaldo, Simone Pernice, Marco Beccuti, Francesca Cordero, R. Sirovich
          </td>
          <td>2026-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains incurable despite advances in treatment options. Although tumor subtypes and specific DNA abnormalities are linked to worse prognosis, the impact of immune dysfunction on disease emergence and/or treatment sensitivity remains unclear. We developed an Immune Atlas of MM by generating profiles of 1,397,272 single cells from the bone marrow (BM) of 337 newly diagnosed participants and characterized immune and hematopoietic cell populations. Cytogenetic risk-based analysis revealed heterogeneous associations with T cells of BM, with 17p13 deletion showing distinct enrichment of a type 1 interferon signature. The disease progression-based analysis revealed the presence of a proinflammatory immune senescence-associated secretory phenotype in rapidly progressing participants. Furthermore, signaling analyses suggested active intercellular communication involving a proliferation-inducing ligand and B cell maturation antigen, potentially promoting tumor growth and survival. Lastly, using independent discovery and validation cohorts, we demonstrated that integrating immune cell signatures with known tumor cytogenetics and individual characteristics significantly improves stratification for the prediction of survival. Pilcher et al. present a single-cell transcriptomics-based immune atlas of participants with newly diagnosed multiple myeloma, reporting on the association of cell types, gene expression and intercellular interactions with disease progression phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f88dd8011aa8686433c4d02b3a0e4dfd59a9255" target='_blank'>
              A single-cell atlas characterizes dysregulation of the bone marrow immune microenvironment associated with outcomes in multiple myeloma
              </a>
            </td>
          <td>
            William Pilcher, Lijun Yao, Edgar Gonzalez-Kozlova, Yered H. Pita-Juárez, Dimitra Karagkouni, Chaitanya R. Acharya, Marina E. Michaud, Mark Hamilton, Shivani Nanda, Yizhe Song, Kazuhito Sato, Julia Wang, Sarthak Satpathy, Yuling Ma, Jessica Schulman, D. D’souza, R. Jayasinghe, Denis J Ohlstrom, Katherine E. Ferguson, Giulia Cheloni, Mojtaba Bakhtiari, Nick Pabustan, K. Nie, J. Foltz, Isabella Saldarriaga, Rania Alaaeldin, Eva Lepisto, Rachel Chen, M. Fiala, Beena E. Thomas, April Cook, Junia Vieira Dos Santos, Chiang I-ling, Igor Figueiredo, J. Fortier, Michael Slade, Stephen T. Oh, Michael P. Rettig, Emilie I. Anderson, Ying Li, Surendra Dasari, Michael A. Strausbauch, Vernadette A. Simon, , I-ling Chiang, N. Kalavros, Jennifer Rogers, T. Dawson, Brian H. Lee, G. Kelly, Laura A Walker, Nicolas F. Fernandez, John Leech, Jarod Morgenroth-Rebin, K. Angeliadis, Matthew A. Wyczalkowski, Song Cao, Omar Ibrahim, Roderick Lin, T. Fehniger, Andrew J. Houston, E. Radkevich, Adeeb Rahman, Zhihong Chen, A. Laganà, John F. DiPersio, J. Rosenblatt, S. Kim-Schulze, S. Lonial, Shaji K. Kumar, Swati S. Bhasin, T. Kourelis, M. Dhodapkar, Ravi Vij, D. Avigan, H. Cho, George Mulligan, Li Ding, Sacha Gnjatic, Ioannis S. Vlachos, Manoj K. Bhasin
          </td>
          <td>2026-01-01</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="The formation of intratumoral spatial niches has been reported for many human malignancies. However, the translational potential of such spatial niches is understudied. Herein, we utilize digital spatial profiling (DSP) to explore the prognostic relevance of spatially defined protein expression in high-risk prostate cancer. A total of 49 patient samples were analyzed for the expression of 46 proteins in 463 regions of interest (ROIs) from the tumor center (n = 198) and the tumor periphery (n = 265) resulting in 21,298 primary data points (mean per patient n = 9.4). Expression data from either the tumor center or the tumor periphery were not found to be prognostic. Protein expression of tumor center and periphery was then integrated into single datapoints by calculating the log2-transformed relative expression between the two niches for each protein and patient. Unsupervised hierarchical clustering of these data yielded two distinct patient subgroups. These clusters did not show a statistically significant correlation with known prognostic parameters yet significantly correlated with progression-free survival (p = 0.014, log-rank, HR 0.43; 95% CI, 0.22–0.86). Our results thus reveal that spatial protein expression contains prognostic information, however, only when expression data from both spatial niches are taken into account. In conclusion, our proof-of-concept study shows that DSP can be exploited for the development of novel prognostic biomarkers that rely on spatially resolved protein expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c6b3a4672d8a49cdbb7af4005dce769ab547c41" target='_blank'>
              Prognostic impact of spatial niches in prostate cancer
              </a>
            </td>
          <td>
            Felix Schneider, Sarah Böning, Beatriz Coelho Antunes, A. Kaczorowski, M. Görtz, V. Schütz, Johannes Huber, Albrecht Stenzinger, Markus Hohenfellner, S. Duensing, A. Duensing
          </td>
          <td>2026-01-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background While adipose tissue constitutes a substantial proportion of breast composition, the functional characteristics and pathological relevance of the adipocyte microenvironment in breast carcinogenesis remain undercharacterized. This study employs single-nucleus RNA sequencing (snRNA-seq) to establish a comprehensive cellular atlas of adipocyte heterogeneity across molecular subtypes of breast cancer, aiming to elucidate subtype-specific adipocyte contributions to tumor microenvironment modulation. Methods snRNA-seq was performed on breast adipose tissues isolated from individuals without cancer and treatment-naïve breast cancer. Various adipocyte and pre-adipocyte subclusters were identified. Comparative analyses of cellular distribution and transcriptional profiles were performed across disease states and molecular subtypes. Pseudotime, cell communication, and immunofluorescence analyses were further implemented to investigate cellular dynamics and microenvironment interactions. Results snRNA-seq data of 86,529 nuclei were obtained. Three adipocyte and seven pre-adipocyte subclusters were identified, of which Adi_LDLR, Pre_Adi_LDLR, and Pre_Adi_LGR4_TGFBR1 were restricted to cancer-associated adipose (CAAs). Adi_LDLR and Pre_Adi_LDLR were enriched in estrogen receptor (ER)-positive CAAs and related to cell senescence and immunosuppression. Pre_Adi_LGR4_TGFBR1 was predominantly present in triple-negative breast cancer, functionally pro-proliferative, immunosuppressive, and lacked normal adipose function. The immunofluorescence intensity of LDLR (p=0.031) and TGFBR1 (p=0.038) was positively associated with disease recurrence, suggesting the formation of immunosuppressive niches by these cancer-specific adipose subsets in both subtypes. Cell communication analyses revealed a specific (pre-) adipocyte-macrophage interaction via ligand-receptor pairs involved in stromal remodeling and tumor migration for ER-positive tumors, whereas tumor proliferation and metastasis for triple-negative ones likely contribute to tumor progression. Conclusions This study delineated a distinct adipocyte landscape in breast cancer and subtype-specific immunosuppressive niches fostered by CAAs and (pre-) adipocyte-macrophage interactions. These findings provide novel therapeutic targets for microenvironment-directed interventions in breast oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c4c519ddf0de4fc6c0a04e6ac2d29ba332f1031" target='_blank'>
              Decoding adipocyte heterogeneity through single-nucleus transcriptomics unveils subtype-specific adipocytes orchestrate immunosuppressive niches in breast cancer
              </a>
            </td>
          <td>
            Yiwei Tong, Zheng Wang, Niu Qiao, Renhong Huang, Chenghui Wu, Haoyu Wang, X. Fei, Kunwei Shen, Xiao-song Chen
          </td>
          <td>2026-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4904674c057fccfc7a5b2f04d9d72ab6c8a77ba" target='_blank'>
              Discrete Transcriptional States Define Biphasic Immune Response and Dynamic CMS Transitions in Colorectal Cancer
              </a>
            </td>
          <td>
            Khalid Ishani, Dechen Wangmo, Atef Ali, Travis Gates, Zejie Yan, Ava Gustafson, Ella Boytim, Katie Storey, Paolo Goffredo, Justin Hwang, Subbaya Subramanian
          </td>
          <td>2026-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6b22a73c1f46495b3ff8ba940a6c1a1abbc7b2c" target='_blank'>
              scAmp analyzes focal gene amplifications at single-cell resolution
              </a>
            </td>
          <td>
            Matthew G. Jones, Natasha E. Weiser, King L. Hung, Xiaowei Yan, Sangya Agarwal, J. Luebeck, Aditi Gnanasekar, B. Howitt, Ellis J Curtis, Kevin Yu, John C. Rose, Katerina Kraft, Valeh Valiollah Pour Amiri, Leena Satpathy, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2026-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e0ef6b89aa5eebc12de51822e3919b402fc07a7" target='_blank'>
              Single-cell and spatial transcriptome analyses reveal tumor immunometabolism in lymph node metastasis of lung cancer.
              </a>
            </td>
          <td>
            Yang Zhang, Wangyang Zhu, Chaoqiang Deng, F. Fu, Yusheng Li, Ting Ye, Jiaqing Xiang, Yawei Zhang, Yihua Sun, Hong Hu, Yuan Li, Xiaoming Zhang, Haiquan Chen
          </td>
          <td>2026-02-19</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a3815eb1f32d503a09f41071a0e393a17ec3962" target='_blank'>
              Single-cell epigenetic profiling reveals a tumor-intrinsic interferon response program in ccRCC tied to poor prognosis and BAP1 loss.
              </a>
            </td>
          <td>
            S. Camp, Meng Xiao He, Michael S. Cuoco, Amanda E Garza, Sherin Xirenayi, Z. Bakouny, Eddy Saad, J. El Masri, Erica M Pimenta, K. Meli, C. Labaki, Breanna M. Titchen, Yun Jee Kang, J. Horst, Rachel Trowbridge, E. Shannon, K. Helvie, Aaron R Thorner, S. Vigneau, Angie Mayorga, Jahnavi Kodali, Hannah Lachmayr, Meredith Bemus, Pengsheng Chen, Haiteng Deng, Jihye Park, T. Choueiri, Kevin Bi, E. V. Van Allen
          </td>
          <td>2026-02-20</td>
          <td>Science advances</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53e9617142706d92615a2fad08d2ab214304d6df" target='_blank'>
              Real-time monitoring of pediatric high-grade glioma invasion using organotypic brain cultures reveals a developmentally dependent sensitivity
              </a>
            </td>
          <td>
            Yoav Biala, Ramakrishna Sai kumar Aravapalli, Hala Shalata, Zakhariya Manevitch, Oded Behar
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271f2b42923ccb0c522e0defc7aee243eaf5e39d" target='_blank'>
              Reconstructing clone-resolved transcriptional programs from bulk tumor sequencing
              </a>
            </td>
          <td>
            Jiaying Lai, Yi Yang, Kathleen Noller, YunZhou Liu, A. Balan, Prathima B. Nagendra, Luciane T. Kagohara, E. Fertig, Laura D. Wood, Rachel Karchin
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="BACKGROUND & AIMS
Unresectable advanced pancreatic ductal adenocarcinoma (PDAC) typically requires systematic chemotherapy, but it remains unclear how this treatment remodels tumor cell plasticity and the tumor microenvironment (TME) to influence clinical outcomes.


METHODS
We conducted single-cell RNA sequencing on paired pre- and posttreatment tumor biopsies and peripheral blood mononuclear cells from 28 patients with PDAC receiving abraxane plus gemcitabine chemotherapy. To validate the chemoresistant niche, we employed multiplex immunofluorescence and spatial transcriptomics, ranging from in situ sequencing to 10X Visium HD at 2-μm resolution. In addition, we performed functional validation experiments, including CRISPR-Cas9 knockout and tumor-killing assays in vitro, and in vivo studies using the KPC mouse model and xenograft tumors in nude mice, with a focus on CD276/B7-H3 as the key regulator identified in our study.


RESULTS
We characterized chemotherapy-induced dynamic remodeling of both malignant states and the immune microenvironment at single-cell resolution. Integrative analysis uncovered a chemoresistant niche composed of SNCG+ basal-like tumor cells, SPP1+ tumor-associated macrophages, and exhausted T cells, which progressively dominated the TME during treatment in nonresponders. Importantly, we identified CD276/B7-H3 as a dual-function immune checkpoint: it promotes tumor transition to a chemoresistant basal-like state, induces T cell exhaustion, and enhances the angiogenesis signature of tumor-associated macrophages.


CONCLUSIONS
Our work uncovers the plasticity of PDAC tumor cell states and interactions with the TME that are modified during chemotherapy of unresectable advanced PDAC and pinpoints CD276/B7-H3 as a critical regulator and a promising therapeutic target for overcoming chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc86f787dec2e2d771faf24c112ae1ed5b0b9252" target='_blank'>
              Single-Cell Analysis of Chemotherapy-induced Remodeling Reveals CD276-driven Basal-like Chemoresistance in Pancreatic Cancer.
              </a>
            </td>
          <td>
            Yao Zhang, Yanhua Du, Jiaxin Wang, Danshu Wang, Judong Li, Jing Zhang, Yizhou Zhao, Sishen Sun, Hongxiang Sun, Jingjing Qi, R. Bao, Chenghao Shao, Minmin Zhang, Xiangyi He, Ling Zhang, Chunhua Zhou, Dong Wang, Bing Su, Duowu Zou, Youqiong Ye
          </td>
          <td>2026-02-01</td>
          <td>Gastroenterology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 PSMA expression is heterogeneous across advanced prostate cancer, limiting the durability of PSMA-radioligand therapy (RLT). To characterize tumor heterogeneity and identify determinants of RLT resistance, we established the SHAPE cohort (Spatial Heterogeneity Atlas of Prostate Cancer Evolution). SHAPE is a multimodal resource integrating PET imaging, clinical treatment history, histopathology, immunohistochemistry, spatial transcriptomics, and genomic profiling across 46 patient tumor samples collected through rapid autopsy, biopsy, and surgical resection. The cohort spans adenocarcinoma, castration-resistant (CRPC), and neuroendocrine (NEPC) prostate cancer states and includes eight xenograft models.



 Twenty-five samples profiled using the 10x Genomics Visium spatial transcriptomics platform, comprising 195,905 spots, were integrated and analyzed using Seurat (v5) in R. Targeted genomic profiling was performed using the Dana-Farber Cancer Institute OncoPanel platform on 36 samples derived from 7 patients and 2 animal models. Among the cohort, seven cases had prior Lu-PSMA exposure, and ten cases were untreated. Eleven clinical and preclinical samples had PSMA PET/CT, including six with both pre- and post-treatment imaging. Data integration connected structural genomic events with spatial lineage programs and imaging features.



 RLT–treated tumors exhibited a copy-number–dominant profile compared with untreated tumors. Recurrent alterations enriched in treated tumors included PTEN deletion (p<0.01), KLLN deletion (p<0.01), AURKA gain (p<0.01), and ZNF217 gain (p<0.01). Treated tumors also showed higher per-sample copy number loss (47.6 vs 37.5, p<0.05) and amplification burden (7.91 vs 2, p<0.05). In contrast, untreated tumors displayed more single nucleotide variants (14.9 vs 6.3, ns) and indels (4.7 vs 1.9, ns). Spatial transcriptomics provided complementary insights. Treated samples with PSMA SUV below the liver reference demonstrated higher copy-number loss burden (p<0.05) and contained unique transcriptomic clusters enriched for glucose metabolism. These clusters displayed variable expression of classical NEPC markers but consistently upregulated genes involved in glucose uptake and utilization. In preclinical studies, head-to-head FDG and Ga-PSMA imaging of WCM12 xenografts demonstrated that FDG, as a surrogate marker of glucose uptake, was significantly increased following relapse after PSMA-RLT (FC, p<0.05).



 The SHAPE cohort integrates genomic, spatial, and imaging data to characterize tumor heterogeneity in advanced prostate cancer. Genomic profiling identified PTEN/KLLN loss and AURKA/ZNF217 gain as recurrent alterations enriched in Lu-PSMA–exposed tumors, while spatial transcriptomics revealed PSMA-low metastases with distinct. clusters enriched for glucose metabolic programs. Together, SHAPE can inform rational strategies for combination or alternative targeted therapies.



 Martin K. Bakht, Jacob Egelberg, Yasutaka Yamada, Louise Clark, Xiao Wei, Varadha Balaji Venkadakrishnan, Anthony P. Belanger, Alice Bernard-Tessier, Sarah J. Hill, Francesca Khani, David J. Einstein, Mary-Ellen Taplin, Eliezer Van Allen, Heather A. Jacene7, Himisha Beltran. Spatial heterogeneity atlas of prostate cancer evolution (SHAPE): Spatial and genomic landscapes of advanced prostate cancer with and without Lu-PSMA therap [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912633affb19287de4d61286123b6004f19890a4" target='_blank'>
              Abstract PR027: Spatial heterogeneity atlas of prostate cancer evolution (SHAPE): Spatial and genomic landscapes of advanced prostate cancer with and without Lu-PSMA therapy
              </a>
            </td>
          <td>
            Martin K. Bakht, Jacob Egelberg, Yasutaka Yamada, L. Clark, Xiao X Wei, V. B. Venkadakrishnan, Anthony P. Belanger, Alice Bernard-Tessier, Sarah J. Hill, F. Khani, David J Einstein, M. Taplin, E. V. Van Allen, H. Jacene, H. Beltran
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="
 Elucidating the spatial organization and local interactions of immune cells in the pancreatic ductal adenocarcinoma (PDAC) microenvironment is essential to identify biomarkers of disease initiation and progression, and to uncover targets for PDAC immunotherapy. By combining high-resolution spatial transcriptomics and functional experiments, we discovered key principles of architecture and function of pathogenic immune niches in human and mouse pancreatic cancer. We identified IL-1β+ tumor-associated macrophages (TAMs) as central organizers of inflammatory niches, as these cells engage in reciprocal crosstalk with multiple tumor, stromal and immune cell subsets via the production and sensing of prostaglandins and innate immune cytokines. Spatial analyses showed that pathogenic IL-1β+ TAM niches are composed of multiple cell types sharing a conserved inflammatory signature, highlighting spatial coordination of functional programs as well as spatial biomarkers of disease. Our analyses also identified specific factors and actionable targets driving defective differentiation of B cells and their organization in aberrant tertiary lymphoid structures (TLSs) lacking germinal centers and enriched in extrafollicular response programs. Together, our findings provide an integrated spatial and functional map of immune dysregulation in PDAC, highlighting local multicellular programs that shape tumor progression, inflammation and immune suppression. Targeting these niches may offer new strategies to restore anti-tumor immunity and overcome immunotherapy resistance in PDAC.


 Renato Ostuni. Spatial and molecular control of tumor immune microenvironments [abstract]. In: Proceedings of the AACR Immuno-Oncology Conference (AACR IO): Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2026 Feb 18-21; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2026;14(2 Suppl):Abstract nr IA01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dbd80e3809e62c96c07a1c9eeb824740a91de2f" target='_blank'>
              Abstract IA01: Spatial and molecular control of tumor immune microenvironments
              </a>
            </td>
          <td>
            R. Ostuni
          </td>
          <td>2026-02-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor development and progression are affected not only by cancer cell-intrinsic factors comprising complex genetic variations, but also by -extrinsic factors such as cell-cell communication (CCC)-mediated immunosuppression. However, whether and how these two types of factors influence each other remains an open question. We present Driver2Comm, a general computational framework designed to systematically identify intrinsic-extrinsic (IE) pathways that functionally connect cancer cell driver genes with their associated CCC signatures in the tumor microenvironment (TME). By applying Driver2Comm to single-cell and spatial transcriptomic datasets of multiple cancer types, we find that driver gene-associated CCC signatures play critical roles in immune regulation, metastasis, and therapy response. These signatures not only illuminate mechanisms of TME remodeling but also demonstrate clinical value in predicting patient survival and response to immune checkpoint blockade. Furthermore, Driver2Comm captures higher-order, cell-type-pair-specific CCC functional modules and spatially coherent CCC patterns in tissue contexts. As a generalizable tool, Driver2Comm bridges cancer genomics and cellular ecosystems, offering insights into biomarker discovery and combination therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cc67af9327e0c16dc02cf8d6cacc83326c18198" target='_blank'>
              Bridging cancer cell-intrinsic driver genes and -extrinsic cell-cell communication with Driver2Comm.
              </a>
            </td>
          <td>
            Runzhi Xie, Junping Li, Yuxuan Hu, Lin Gao
          </td>
          <td>2026-02-17</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Objective: This study aimed to identify clinically relevant regulators of pancreatic ductal adenocarcinoma (PDAC), a disease characterized by stromal remodeling and immune suppression, and to define their links to malignant progression and microenvironmental reprogramming. Methods: We integrated multi-cohort bulk, single-cell, and spatial transcriptomic datasets and subsequently validated bulk differential expression and network analyses with machine learning-based prioritization in an independent combined cohort (TCGA-PAAD plus GSE62452). Single-cell mapping was used to assess cell-type specificity, positioning candidates along inferCNV- and pseudotime-defined malignant continua. In Visium sections, a DKK1-associated program score quantified intratumoral spatial heterogeneity and informed our analyses of ligand–receptor communication. Bulk immune deconvolution linked gene levels to immune infiltration patterns, and functional assays were used to test the impact of DKK1 knockdown on migration, proliferation, clonogenic growth, and apoptosis in PDAC cells. Results: Four reproducible tumor-associated genes—DKK1, COL10A1, SULF1, and SLC24A3—were prioritized and validated externally. DKK1 was predominantly expressed by epithelial tumor cells and tracked along a malignant progression continuum. Spatially, the DKK1 program localized to epithelial-dominant regions, revealed pronounced intratumoral heterogeneity, and highlighted epithelial–endothelial and endothelial–immune signaling in high-score areas. Immune deconvolution associated higher DKK1 expression with increased myeloid infiltration and reduced cytotoxic lymphocyte signatures. Functionally, DKK1 knockdown impaired migration, proliferation, and clonogenicity while increasing apoptosis. Conclusions: We demonstrate that DKK1 is an epithelial-derived regulator linked to malignant progression and tumor–stroma–immune remodeling, supporting its potential as a biomarker and therapeutic target in PDAC treatment, including rational combinations with stroma-modulating strategies and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96a91f34d70864c659b210aeaa2777d7f019925a" target='_blank'>
              Multi-Omic and Spatial Profiling Identifies an Epithelial DKK1 Associated with Microenvironmental Remodeling in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Jiajia Xu, Kaiqiang Qian, Yanyu Ding, Jianghao Cheng, Xu Zhang, Yong Huang, Bo Liu
          </td>
          <td>2026-02-05</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="OBJECTIVE
Glioma recurrence severely impacts patient prognosis, with current treatments showing limited efficacy. Traditional methods struggle to analyze recurrence mechanisms due to challenges in assessing tumor heterogeneity, spatial dynamics, and gene networks. Single-cell combined spatial transcriptomics (ST) offers innovative solutions.


METHODS
We analyzed glioma mRNA data from TCGA and single-cell and ST data from GEO. Following quality control, dimensionality reduction, clustering, and cell annotation of single-cell sequencing data, we identified cell types exhibiting significantly aberrant distributions between primary and recurrent samples by analyzing the deviation degree of Ro/e values. Fibroblasts demonstrating the greatest intergroup differences were subsequently selected as the key cellular population for further investigation. Key differentially expressed genes (DEGs) were identified via random survival forest analysis. Drug sensitivity was assessed using GDSC. Deconvolution algorithms mapped cellular spatial distribution, while PROGENy quantified pathway activity. MISTy modeling revealed cell-cell interactions.


RESULTS
Fibroblasts were the primary recurrence-associated subpopulation, with marker genes enriched in extracellular matrix and adhesion pathways. AEBP1, ZNF708, and TSHZ2 were identified as key genes: AEBP1/TSHZ2 correlated with poor prognosis, while ZNF708 showed an inverse trend. These genes were linked to chemosensitivity (Irinotecan, Carmustine, Vincristine, and Cisplatin). Recurrent tumors exhibited increased plasma cell infiltration, with key genes regulating IL-17, Notch, and Toll-like receptor pathways. Spatial analysis highlighted oligodendrocyte-astrocyte interactions in the tumor microenvironment.


INTERPRETATION
Fibroblasts drive glioma recurrence, with AEBP1, ZNF708, and TSHZ2 predicting recurrence and chemoresistance. These genes promote immune suppression (via plasma cells) and activate recurrence pathways. Oligodendrocyte-astrocyte interactions shape the recurrent microenvironment, suggesting new therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec7c524a9377c361243faed8c72c15c26a0c97d" target='_blank'>
              Unraveling the Molecular Mechanisms of Glioma Recurrence: A Study Integrating Single-Cell and Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Lei Qiu, Yinjiao Fei, Jiaxuan Ding, Kexin Shi, Jinyan Luo, Yuchen Zhu, Xingjian Sun, Gefei Jiang, Yuandong Cao, Weilin Xu, Shu Zhou
          </td>
          <td>2026-01-06</td>
          <td>Annals of clinical and translational neurology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Rising incidence of early-onset prostate and other solid tumors underscores the need for experimental systems that model how normal tissues traverse premalignant states, acquire mutations, and become therapy-responsive malignancies under authentic immune and stromal pressures. A major barrier has been the lack of tractable in vivo platforms that enable genome-wide discovery while preserving continuous tumor evolution without catastrophic chromosomal instability. To address this, the Bose Lab developed Stochastically Emergent Tumors (SETs), an organoid-derived in vivo evolution engine that redefines discovery for early-onset and understudied patient groups. In this system, mismatch-repair deficiency is induced in non-malignant human organoids, which are passaged to accumulate stochastic point mutations and transplanted into mice to permit malignancy to emerge under physiologic selection. SETs evolve primarily through high-resolution point mutations rather than broad copy-number changes, yielding bioinformatically tractable clonal dynamics ideally suited for whole-genome driver discovery and machine learning. Compared with conventional xenografts, SETs display greater intertumoral heterogeneity and reproducible recovery of sensitizing and resistance alleles under therapeutic pressure. As proof of principle in prostate cancer, endocrine therapy applied to SET pools recovered known determinants of androgen-pathway sensitivity and uncovered new drivers. Loss of ZFHX3, typically obscured within a multigene suppressor locus in bulk cohorts, promoted luminal histology and sensitized tumors to androgen-receptor inhibition in vivo, whereas KMT2D or CIC alterations mediated resistance. Consistent with model predictions, ZFHX3 loss in patients correlated with significantly improved survival, a finding comparable in magnitude to the most favorable molecular subtypes of advanced prostate cancer. SETs also quantify evolutionary thresholds: in a Pten-null background, approximately 900 coding mutations accumulated over 208 days were sufficient for malignant transformation in half of grafts. This defines a measurable axis linking mutation burden, genotype, and tumor incidence. Because SETs generate neoantigen-rich point-mutation landscapes, they can be extended to immunocompetent hosts to study tumor–immune coevolution, early T-cell surveillance, macrophage-mediated immune exclusion, and myeloid checkpoints that enable immune escape. In summary, SETs provide a scalable, evolution-aware platform that connects mutational dynamics to therapeutic vulnerability, enabling identification of lineage- and ancestry-associated drivers, immunopreventive targets, and biomarkers of early-onset prostate cancer.



 Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, Valbona Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C.K. Cornelia Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, Charles Sawyers, Felix Feng, Hani Goodarzi, Rohit Bose. Reconstructing prostate evolution with Stochastically Emergent Tumors (SETs) reveals in vivo therapeutic vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23678ab77681ac1f2c46c1e6d9b9824f918c99b" target='_blank'>
              Abstract A008: Reconstructing prostate evolution with Stochastically Emergent Tumors (SETs) reveals in vivo therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, V. Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C. Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R Karthaus, Charles L Sawyers, Felix Y. Feng, H. Goodarzi, Rohit Bose
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND
Meningioma brain invasion encumbers surgical resection and increases the risk of tumor recurrence, but the molecular mechanisms underlying this process are poorly understood.


METHODS
To identify molecular and cellular features of brain-invasive meningiomas, we (1) analyzed bulk RNA sequencing data from 199 meningiomas, including 33 brain-invasive tumors, (2) analyzed patient-matched single-cell RNA sequencing data of spatially mapped meningioma samples from the tumor core or brain-tumor interface (BTI), and (3) performed spatial transcriptomic sequencing of brain-invasive meningioma samples. Multiplexed immunofluorescence (IF) was used to validate bioinformatic spatial expression patterns. Functional interactions between meningioma cells and neurons were studied in meningioma/neuron co-cultures using confocal microscopy, multi-electrode array recordings, and live cell calcium imaging.


RESULTS
Transcriptomic analyses showed conserved enrichment of TGM2, S100A11, ZYX, and PDGFRA at the BTI across bulk, single-cell, and spatial RNA sequencing datasets. The expression of these genes at the BTI was confirmed using multiplexed IF, and single-cell bioinformatic and microscopy analyses further demonstrated enrichment of macrophages at the BTI. Co-culture assays showed neuronal hyperexcitability and increased proliferation of meningioma cells, suggesting functional communication between meningioma cells and the tumor microenvironment may contribute to meningioma growth in cases with brain invasion.


CONCLUSIONS
Meningioma brain invasion is defined by molecular remodeling of tumor cells and functional interactions within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4352a0cf42108bac27d918f350130e856ef324b" target='_blank'>
              Meningioma cell reprogramming and microenvironment interactions underlie brain invasion.
              </a>
            </td>
          <td>
            Ayush Aggarwal, Mark W. Youngblood, T. Picart, H. Najem, S. Oten, M. A. Cady, Stephen T. Magill, Craig M Horbinkski, James P. Chandler, A. Heimberger, K. Mirchia, S. Hervey-Jumper, D. Raleigh
          </td>
          <td>2025-12-31</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4167a37773942f9ca4b809bad0d95f60d49ec113" target='_blank'>
              Impacts of mutation accumulation and order on tumor initiation revealed by engineered murine colorectal cancer organoids
              </a>
            </td>
          <td>
            Yanping Li, Xiaoxin Xie, Daqi Deng, Zhiyuan Sun, Zhaoan Huang, Yisen Tang, Liang Fang, Wei Chen, Qionghua Zhu
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Colorectal cancer (CRC) stands as a significant contributor to cancer-related mortality. Owing to its prognostic and therapeutic implications, intratumoural heterogeneity (ITH) presents a considerable challenge. We have developed an experimental framework integrating single-cell derived spheroids with proteomic profiling to facilitate a molecular, proteomic, and therapeutic characterization of intratumoural heterogeneity during CRC progression. Single cells from the commercially available colorectal cancer cell lines SW480 (primary colorectal adenocarcinoma) and SW620 (locoregional lymph node metastasis of the same donor) were isolated using fluorescence-activated cell sorting (FACS) and subsequently cultured forming spheroids. This platform allowed controlled interrogation of clonal diversity through proliferation and viability assays, alongside deep proteomic characterization using label-free liquid chromatography–mass spectrometry (LC-MS) with data-independent acquisition. To evaluate its utility for therapeutic testing, chemotherapy response was measured after 72 h of incubation with 5-fluorouracil (5-FU). The single-cell derived spheroid system demonstrated significant heterogeneity, as evidenced by variations in morphology, growth dynamics, viability, and proteomic signatures. Protein profiling identified ITH-associated proteins (WDR5, CKB, IPO11, ATP6V1F, DCXR and PCCB) and underscored pathway variations including tumour suppressor and proto-oncogenic signalling, vascularization and metabolic regulation. Furthermore, individual spheroids exhibited differential sensitivities to 5-fluorouracil, demonstrating the platform’s capacity to resolve heterogeneous therapeutic responses. Our study establishes a robust and scalable method that integrates single-cell spheroids with proteomics to model and quantify ITH in CRC. By capturing clinically relevant diversity across morphology, viability, proteomic profiles and drug response, this approach provides a foundation for translating spheroid- and proteomics-based assays into personalized therapeutic testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cc15b9af4c8cdc79c6ae94af0f6f4ec13bc73f7" target='_blank'>
              A single-cell derived spheroid approach to dissect intratumoural heterogeneity in colorectal cancer: cell lines show changes in proteomes and therapeutic response to 5-FU
              </a>
            </td>
          <td>
            Helene Sophia Radloff, Michael Kohl, Thorben Sauer, Sonja Hartwig, Sven Geisler, Stefan Lehr, Timo Gemoll
          </td>
          <td>2026-01-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Sinonasal squamous cell carcinoma (SNSCC) is a rare malignancy with poorly understood molecular drivers. Consequently, its cellular composition and tumor microenvironment (TME) remain largely undefined. Here, we performed integrated bulk and single-nucleus multi-omic analyses to map the SNSCC ecosystem. Within the malignant compartment, we identified five distinct populations, with hypoxic (TC1) and proliferative (TC2) subtypes associated with adverse clinical outcomes. Functionally, TC1 cells orchestrate a hypoxia-driven angiogenic program via coordinated secretion of adrenomedullin (ADM), MIF, and VEGFA, promoting endothelial tip cell (EC1) differentiation. Integrative analysis revealed these transcriptional programs are underpinned by tumor-specific chromatin accessibility and DNA hypomethylation, particularly at AP-1-enriched regulatory elements. Mechanistically, in vitro studies confirmed that this response depends on cooperative AP-1 and HIF1A signaling. Furthermore, histological analysis of patient tissues demonstrated spatial co-localization of GLUT1-expressing TC1 cells with DLL4-positive EC1 cells. These findings elucidate the epigenetic landscape underlying tumor-stromal interactions and establish the ADM/VEGFA axis as a critical therapeutic target to disrupt epigenetically controlled angiogenesis in SNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bae1ad73527bcabf41b52f28eb9547020179867e" target='_blank'>
              Single-Nucleus Multi-Omics Reveals Hypoxia-Driven Angiogenic Programs and Their Epigenetic Control in Sinonasal Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, J. Noh, Jaehyun Lee, Geunho Kwon, Myeong S. Yu, Yoo-Sam Chung, Seung-Jun Lee, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2026-01-07</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5e712867c904dd12e6307954591e726635cae8e" target='_blank'>
              Differences in T-cell counts and neighborhood patterns in human colorectal adenomas and sessile serrated lesions
              </a>
            </td>
          <td>
            Souvik Seal, Lauren R. Fanning, Evan Bagley, Christine Bookhout, Elizabeth L. Barry, Elizabeth O’Quinn, Dale C. Snover, David N. Lewin, Silvia Guglietta, A. Kourtidis, M. Shrubsole, John A. Baron, Todd A. Mackenzie, Alexander V. Alekseyenko, Kristin Wallace
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eef0494e51b8128b410f9d5b4dc00c303aff4796" target='_blank'>
              Mapping lineage and functional diversity in the high-risk human mammary epithelium
              </a>
            </td>
          <td>
            Matthew Waas, Bowen Zhang, Meinusha Govindarajan, Pirashaanthy Tharmapalan, Abhijith Kuttanamkuzhi, Olivia Drummond Guy, Michael Woolman, H. Berman, Paul D. Waterhouse, R. Khokha, T. Kislinger
          </td>
          <td>2026-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The immune microenvironment profoundly shapes the progression and therapeutic response of thyroid carcinoma. Through comprehensive analysis of single-cell RNA sequencing data, this review delineates the immune landscapes of papillary thyroid carcinoma (PTC), poorly differentiated thyroid carcinomas (PDTC), and anaplastic thyroid carcinomas (ATC), revealing a differentiation dependent trajectory of tumor immune microenvironment remodeling—from immune activation suppression coexistence in PTC, to immune exclusion in PDTC, and terminal exhaustion in ATC. This single-cell based approach enables high resolution dissection of cellular heterogeneity, immune crosstalk, and spatial organization that are often masked in bulk analyses. Such insights provide a scientific basis for precision immunotherapy, offering guidance for differentiation tailored strategies to overcome immune escape and improve clinical outcomes in thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/662dd84def56730e4ddb7a83c80225823dda796d" target='_blank'>
              Single-cell sequencing reveals the tumor immune microenvironment in thyroid cancer: a narrow review
              </a>
            </td>
          <td>
            Kangcong Liang, Ziyu Wang, Zhiqiang Zhang, Gengluan Liu, Xidi Wang, Heng Cao, Ming Zhong, Liping Ye, Xin Zhong, Jingyu Xun, Kefeng Lei, Ningning Li
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e02719271293d52c9c8a63dc2dfab40de0d7932" target='_blank'>
              Xenium-based spatial transcriptomic analyses uncover prognosis-associated heterogeneity in the tumor microenvironment (TME) of angioimmunoblastic T-cell lymphoma (AITL).
              </a>
            </td>
          <td>
            Jiyan Dong, Xiaoyue Xiao, Lin Nong, Xuemin Xue, Long Wang, Xujie Sun, Kang Jiang, Xiaoli Feng
          </td>
          <td>2026-02-19</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung cancer is a highly malignant disease, posing a significant threat to global health. The presence of tumor heterogeneity results in substantial variations in prognosis and therapeutic responses among patients. Advances in bulk RNA sequencing and single-cell RNA sequencing have facilitated the identification of driver gene mutations and the exploration of cellular diversity within tumors. However, tumors are complex ecosystems comprising both tumor cells and their microenvironment, where interactions among different cell types give rise to specific functional structural units that collectively drive tumorigenesis and progression. The emergence of spatial omics technologies has allowed for the analysis of tumor ecosystems, providing unprecedented insights into tumor heterogeneity. This review aims to present updates on spatial omics technologies and data analysis algorithms, discuss current technical limitations, and explore potential future developments. Furthermore, we summarize the latest applications of spatial omics in elucidating lung cancer heterogeneity, investigating mechanisms of lung cancer progression and drug resistance, and identifying novel biomarkers. Based on these findings, we propose strategies for integrating spatial omics into lung cancer research, offering new perspectives for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eca909ba170b3407f8c473dadaa8f67180cd2619" target='_blank'>
              The Evolution of Spatial Omics Technologies Introduces A Novel Avenue for Lung Cancer Research.
              </a>
            </td>
          <td>
            Yue He, Zifan Li, Wenxiang Wang, Xu Liu, Shanshan Lu, Jing Bai, Lin Weng, Qingna Zhang, Jun Wang, Kezhong Chen
          </td>
          <td>2026-01-30</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7129daca11db4cd5b5b60ff63c51368361b7db2" target='_blank'>
              Multimodal single-cell profiling reveals crosstalk between macrophages and stromal cells in poor prognostic cholangiocarcinoma patients.
              </a>
            </td>
          <td>
            Lara Heij, Sikander Hayat, Konrad Reichel, Sidrah Maryam, C. O'Rourke, X. Tan, Marlous van den Braber, J. Verhoeff, Maurice Halder, Fabian Peisker, G. Wiltberger, Jan Bednarsch, Daniel Heise, Julia Campello Deierl, S. Lang, F. Ulmer, Tom Luedde, E. Dahl, Danny Jonigk, Jochen Nolting, Shivan Sivakumar, J. Siveke, Florian Vondran, Flavio G. Rocha, H. Baba, S. Hartmann, Jesper B. Andersen, Zaynab Hobloss, A. Ghallab, J. Hengstler, J. J. G. Vallejo, R. Kramann, Ulf Neumann
          </td>
          <td>2026-01-28</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a major issue in the sphere of head and neck cancer since it is very heterogeneous, which also leads to the poor treatment results and low survival rates in the advanced stages. Here, the review will discuss how spatial omics methods explore tumor heterogeneity in OSCC, which includes cellular, molecular, and immune microenvironment alterations that occur due to cancer stem cells, stromal interactions, genetic instability, epigenetic reorganization, and metabolic reprogramming. The focus of heterogeneity is given on the contribution of the tumor microenvironment such as immune cells, cancer-associated fibroblasts and extracellular matrix remodeling to the stimulation of progression, metastasis, as well as therapeutic resistance. The introduction of spatial omics technologies, including spatial transcriptomics, proteomics, and metabolomics, has revolutionized the field by preserving tissue architecture, enabling high-resolution mapping of gene expression, protein distribution, and metabolite profiles. Significant developments around the spatial omics technologies are discussed, while how they are used in the identification of ligand-receptor network, signaling pathways, and spatial patterns of heterogeneity in OSCC are described. Integration of multi-omics approaches bridges gaps between transcriptomic, proteomic and metabolic, facilitating the discovery of biomarkers for prognosis, immune evasion mechanisms, and precision therapies targeting epithelial-to-mesenchymal transition and immunosuppressive networks. Despite challenges in data integration, cost, and clinical translation, spatial omics holds promise for personalized oncology, with future directions involving artificial intelligence-driven modeling to enhance diagnostic accuracy and therapeutic efficacy in OSCC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00bf591ef53e5e31062caa9f75b7e97f0abcc8b" target='_blank'>
              Spatial Omics in Decoding Oral Squamous Cell Carcinoma Heterogeneity: Microenvironment Crosstalk and Multi-Omics Integration
              </a>
            </td>
          <td>
            Xiaotong You, Yu Liu
          </td>
          <td>2025-12-31</td>
          <td>GenoMed Connect</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Genomic alterations are a hallmark of cancer, and extrachromosomal DNA (ecDNA) has emerged as a key source of oncogene selection, tumor growth, and drug resistance. The intratumor heterogeneity and clonal selection of ecDNA is, however, poorly understood. In this study, we pursue a computational approach that leverages allelic imbalance and outlier expression from standard single-cell RNA sequencing (scRNA-seq) to deconvolve the tumor heterogeneity of ecDNA at the single-cell level (ecSingle). Using this approach, we identify oncogene-carrying ecDNAs in tumor samples at the single-cell level, which we validate using genome sequencing. Moreover, we show the superiority of using single-molecule long-read sequencing in resolving ecDNA. ecDNAs displayed extensive intratumor heterogeneity, including subclonal oncogene-carrying ecDNA in primary tumor cells that segregate with distinct transcriptional cell states. Importantly, we show that a rare ecDNA+ clone in the primary tumor can expand to form dominant clones in relapse tumors. Our study introduces a novel approach to studying ecDNA at the single-cell level, enabling both clonal evolution and transcription cell state analysis. We apply this approach to cancer samples to gain deeper insights into the role of ecDNA in intratumor heterogeneity and cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df14db8ad5c3596e9b413772fed54a722b4bb2b" target='_blank'>
              Resolving clonal evolution and selection of extrachromosomal DNA at single-cell resolution
              </a>
            </td>
          <td>
            J. Hendriksen, A. Locallo, B. Schlotmann, Francisco Germán Rodríguez González, Jane Skjøth-Rasmussen, C. Yde, D. Nørøxe, H. S. Poulsen, U. Lassen, J. Weischenfeldt
          </td>
          <td>2026-01-29</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47f826d643f2d1e8e34a85ebebb36d9ff4d366b1" target='_blank'>
              Temporal and spatial atlas of eosinophil specialization across tissues
              </a>
            </td>
          <td>
            Yanan Hu, Lianfu Wu, Sihao Qu, W. Kong, Xiaotong Yu, Jiaqian Xu, Ziyi Xu, Jialing He, Chong Wang, Zhaoyuan Liu, Liming Lu, Lai Guan Ng, Camille Blériot, Bing Su, F. Ginhoux, Ziyi Li, S. Chakarov
          </td>
          <td>2026-01-09</td>
          <td>Nature Immunology</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Metastasis is a major threat in breast cancer (BRCA), often involving lymph node (LN) dissemination. However, the cellular composition and signaling networks within the metastatic microenvironment remain incompletely characterized, resulting in limited understanding of the molecular mechanisms driving LN metastasis. In this study, we constructed a comprehensive single-cell atlas of metastatic niches using single-cell RNA-sequencing (scRNA-seq) data from 78 primary breast tumors and their paired LN metastatic samples. Among the epithelial cell subpopulations, we identified distinct clusters representing early-disseminated cancer cells (EDCs). Notably, we observed profound metabolic reprogramming and immune modulation during the malignant transformation of epithelial cells, which may contribute to the invasive and metastatic phenotype of EDCs. Furthermore, the communication network within the metastatic microenvironment has also been delineated systematically, where crosstalk among lymphocyte, macrophage and epithelial cells drove immunosuppressive features of LN metastasis. To explore potential therapeutic interventions, we conducted computational drug repositioning and identified four tyrosine kinase receptor inhibitors that may target key interactions within this crosstalk. These results were validated using spatial transcriptomics (ST) data of four metastatic LN tissue sections from integrated scRNA-seq cohort. Collectively, this study discussed the cellular architecture and regulatory interplay underlying LN metastasis in BRCA, offering novel insights that may inform targeted therapeutic strategies for patients with metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c96def9e5d2ad3a5b2337af0512537501ee5cb" target='_blank'>
              Deciphering the cellular and metabolic landscape of lymph node metastasis in breast cancer using single-cell and spatial multi-omics.
              </a>
            </td>
          <td>
            Rui Zhu, Guijie Jiang, Jie Shen, Chengxuan Gong, Hongyu Shan, Tingming Liang, Li Guo
          </td>
          <td>2026-01-01</td>
          <td>The American journal of pathology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c768fd92ec0a5075aecde92ab99ab504cc56d9" target='_blank'>
              STEAP4+ neutrophils: a promising circulating biomarker for lung metastasis
              </a>
            </td>
          <td>
            Chen Ai, Huijuan Wu, Huihui Yang, Jing Wang, Chang Liu, Melissa S F Ng, Ziheng Zhou, Fenghui Zhuang, Qinhang Gao, Lingqian Wang, Changming Shi, Linna Zhu, Z. Bu, Chang Chen, Lai Guan Ng, Zemin Zhang
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Urothelial carcinoma (UC), encompassing both bladder cancer and upper tract UC, is a biologically heterogeneous malignancy that has long complicated patient stratification and therapeutic prediction. Recent genomic and transcriptomic analyses, however, have delineated reproducible molecular subtypes across non‐muscle‐invasive and muscle‐invasive disease. These classification frameworks have refined biological understanding and improved both risk stratification and treatment selection. Emerging evidence also indicates that genomic alterations, such as FGFR3 mutations, may confer distinct immunologic phenotypes within specific transcriptomic contexts, thereby challenging prior assumptions. The advent of immune checkpoint inhibitors has transformed UC management, yet variable response rates underscore the complexity of the tumor microenvironment (TME). Single‐cell and spatial analyses have identified immune‐inflamed, ‐desert, and ‐excluded TME subtypes, with cancer‐associated fibroblasts playing a key role in immune exclusion and therapeutic resistance. Liquid biopsy, particularly circulating tumor DNA, has demonstrated utility for real‐time disease monitoring, minimal residual disease detection, and treatment stratification, as highlighted in trials such as IMvigor010 and IMvigor011. Urinary extracellular vesicles also represent promising non‐invasive biomarkers, though further standardization is required. In addition, novel therapeutic strategies—including antibody‐drug conjugates, gene therapies, and sustained‐release delivery systems—are broadening treatment options across disease stages. Collectively, these translational advances mark a paradigm shift toward precision immuno‐oncology in UC. Nonetheless, successful clinical implementation will require prospective validation, optimized analytical platforms, and sustained physician‐scientist engagement to translate these discoveries into improved patient outcomes. This review provides a comprehensive overview of recent progress in UC translational research, with a focus on genomic and transcriptomic insights, the evolving role of immunotherapy, advances in liquid biopsy, and the development of novel therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e44cd39c3a6847a9cce1aefa2b7bb98a79ab335f" target='_blank'>
              Translational Advances in Urothelial Carcinoma: From Bench to Bedside
              </a>
            </td>
          <td>
            T. Iwasawa, T. Tsujino, S. Tsukahara, Hiroshi Fukushima, Daisuke Ito, E. Tomiyama, Fumihiko Urabe
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Urology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract The process of migration and colonization is important in evolution; for example, modern humans experienced multiple waves of migrations out of Africa. However, no data cover the spatio-temporal patterns sufficiently to be truly informative. Metastatic cancer provides a unique in vivo model to study these processes through rapid somatic evolution. Here, we apply the high-resolution sampling technique (Dense 3D Crypt-scale Sampling) to analyze hundreds of spatially mapped micro-samples from the primary colorectal cancer and liver metastases in two representative cases. This would be analogous to recording the “out-of-Africa” events in two repeats. Our results support that liver metastases arise from polyphyletic and polyclonal seeding events where multiple, genetically distinct clones colonize a new site together. Following colonization, these multi-clonal populations can evolve into distinct spatial architectures: segregated territories formed by cells with low motility, or highly intermixed patterns driven by high motility. The colonization (or seeding) process begins within the first third of the primary tumor's progression, creating a large number of widespread but clinically undetectable micrometastatic colonies. These findings support a model where metastatic competence is not an intrinsic trait of a single “winner” clone but an emergent property of multiple concurrent clones. Collectively, our work supports metastasis as a multi-stage process initiated early in tumor development, characterized by continuous polyclonal dissemination and the formation of spatially distinct clonal architectures. This general pattern may echo the ecology of migration and colonization in organismal evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9d8108fc71756559fb44982b7725fb8d93fb854" target='_blank'>
              Full spatio-temporal analyses of migration and colonization in evolution—dense 3D mapping of cancer metastases provides new insights
              </a>
            </td>
          <td>
            Qihang Chen, Senmao Li, Xianrui Wu, Qing Xu, Ranran Zhu, Yongsen Ruan, Ao Lan, Zihan Liu, Jiarui Weng, Yanjiang Zhao, Xiying Xu, Xinyue Qi, Jinhong Lai, L. Xiao, P. Lan, Chung-I Wu, Bingjie Chen
          </td>
          <td>2026-01-12</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Assessing therapeutic response in glioblastoma (GBM) is a major factor limiting the clinical development of effective therapies. The intracranial location limits serial biopsies and only provides an intermittent view of the tumor molecular profile from the initial resection. Liquid biopsy techniques, specifically small extracellular vesicle (sEV) analysis, have the potential to overcome these limitations by providing a window into the brain using peripheral blood. To address the need for monitoring tumor evolution and therapeutic resistance, we developed a GBM biomarker panel (ATPase subunit beta-2, excitatory amino acid transporter 2, CD24, CD44, CD133, and epidermal growth factor receptor) for multiplexed profiling of sEVs using an advanced GBM Extracellular Vesicle Monitoring Phenotypic Analyzer Chip. We successfully tracked patient response to treatment by monitoring changes in glioma stem cell markers on circulating sEVs. We propose that these results provide a strong rationale for using GBM sEVs as a serial monitoring tool in the future clinical management of patients with GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde6ebeebbfe29ec03351fcee4f74ee6e4a24a58" target='_blank'>
              Monitoring glioblastoma extracellular vesicle evolution using a nanodiagnostic platform to detect glioma stem cells driving recurrent disease
              </a>
            </td>
          <td>
            Zhen Zhang, R. Lobb, P. Tooney, Jing Wang, R. Lane, Quan Zhou, Xueming Niu, S. Faulkner, O. Al-iedani, B. Day, S. Puttick, S. Rose, M. Fay, M. Trau
          </td>
          <td>2026-01-30</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e961f589165d98aa8aff64f0539527c2012d419" target='_blank'>
              Pooled single-cell screen in colorectal cancer defines transcriptional modules linked to oncogenes.
              </a>
            </td>
          <td>
            Viola Hollek, Francisca Böhning, Catalina Florez Vargas, A. Sieber, M. Morkel, Nils Blüthgen
          </td>
          <td>2026-01-19</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cervical adenocarcinoma, a significant subtype of cervical cancer, exhibits variable responses to immune checkpoint inhibitors, with some patients experiencing hyperprogressive disease (HPD) characterised by accelerated tumour progression. The molecular and cellular mechanisms underlying HPD remain poorly understood. In this study, we employed single-cell RNA sequencing, T cell receptor profiling, and spatial transcriptomics to longitudinally analyse the tumour and its microenvironment of a single case of cervical adenocarcinoma who developed HPD following anti-PD-1 immunotherapy. Our integrated analyses revealed two key processes associated with HPD: Immunosuppression involving an IFNγ–STAT1–LAG3 axis in T cells, and fibroblast-supported epithelial proliferation and survival via IGF1–IGF1R signalling. Additionally, spatial transcriptomics identified the formation of a distinct pro-metastatic niche post-immunotherapy. These findings nominate LAG3 and IGF1R as candidate therapeutic targets in this patient and demonstrate the power of real-time, multi-modal profiling to identify adaptive hyperprogression mechanisms. Our work illustrates the potential value of longitudinal monitoring in precision oncology and provides a conceptual framework that could be tested for developing personalised interventions for minimal residual disease, non-responders, and HPD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e2cd6176acb40b189c0e94309b044cc9f1d5b0c" target='_blank'>
              Unveiling personalised targets of PD-1 blockade hyperprogression in a cervical adenocarcinoma via longitudinal single-cell and spatial monitoring
              </a>
            </td>
          <td>
            Zezhuo Su, Yue Xiao, Li Yang, Yifu Wang, Yanling Qu, Junru Liu, Shunrong Tang, K. S. C. Cheung, J. P. Cheung, Zhiyuan Xu
          </td>
          <td>2026-01-13</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Osteosarcoma (OS) is the most common primary bone malignancy affecting children and adolescents, for which survival has not improved in more than four decades. The lack of accurate OS preclinical models hinders the understanding of tumor heterogeneity and its interaction with the surrounding extracellular matrix (ECM), limiting the discovery of predictive biomarkers and the development of effective therapies. Four 2D preclinical models from OS patients were established and characterized for their functional differences in comparison to OS cell lines for their growth, cellular phenotypic attributes, osteogenic differentiation capabilities and metabolic responses to growth factors and BMP-2. Molecular and cellular profiling revealed intra-tumoral heterogeneities that were very distinct from the endorsed OS cell lines. The OS patient-derived (PD) cells also displayed differential sensitivity to Doxorubicin and Cisplatin and resistance against Methotrexate. Subsequently, the 3D PDTs (Patient- derived tumoroids) models were developed by self-aggregating spheroids with and without Matrigel® ECM matrix. These PDTs models were screened for selective ECM and bone-specific gene markers, revealing dynamic differences between 2D and 3D models with and without ECM, with heightened dysregulation observed in 3D systems. The drug response variances observed among 2D OS cells and 3D tumoroids model within Matrigel® highlights the need for optimized platforms for in vitro personalized drug screening. Thus, our findings support the screening of preclinical PD OS models for phenotypic profiling and elucidating ECM contributions to drug responses and pathophysiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895f2c94cf5a05025fd5ceb276a27c6fd2f5f6fa" target='_blank'>
              Patient-derived osteosarcoma tumoroid model reveals functional phenotypic diversity with implications for drug responses
              </a>
            </td>
          <td>
            Sugandha Bhatia, Briony Claxton, Erik W Thompson, Martin Lowe, Michael Wagels, Wayne Nicholls, Philip D Rowell, D. Hutmacher, J. McGovern
          </td>
          <td>2026-01-22</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a4cbeca96278012f41ed1d7ec23c2d477966b91" target='_blank'>
              Charting cell-type-specific positive genetic interaction at single-cell resolution for lung adenocarcinoma.
              </a>
            </td>
          <td>
            Bo Chen, Mingyue Liu, Q. Dong, Chen Lv, Kaidong Liu, Huiming Han, Linzhu Wang, Nan Zhang, Wenyuan Zhao, Junjie Lv, Yunyan Gu
          </td>
          <td>2026-02-18</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Clonal hematopoiesis of indeterminate potential (CHIP) is a precursor condition characterized by the expansion of blood cell clones harboring somatic mutations originating in hematopoietic stem cells (HSCs). Since individuals with CHIP face a high risk of developing myeloid malignancies, targeting CHIP clones could provide a viable strategy for leukemia prevention. Despite its clinical significance, the mechanisms underlying CHIP predisposition and progression remain poorly understood. Recent genome wide association studies (GWAS) have identified several non-coding genetic loci that are strongly associated with CHIP; however, their underlying mechanisms still remain unknown. We hypothesize that risk variants in these non-coding loci modulate enhancer elements active in HSCs. To test this, we selected 1,374 non-coding variants from 51 loci associated for CHIP risk in the UK Biobank and screened them for regulatory activity using a Massively Parallel Reporter Assay (MPRA). We performed our lentiviral MPRA screen in MUTZ-3 cells, a human hematopoietic cell line relevant to HSCs, which express CD34 surface marker and are dependent on HSC-specific transcription factors. Using a MPRA library of ∼73,000 constructs in CD34+ fraction of MUTZ-3 cells, we identified 87 variants representing 32 GWAS loci. We used targeted genome editing to demonstrate endogenous enhancer activity across 3 MPRA variants that affect the transcription of NKD2, FLT3, and MSI2. Our functional studies on MSI2 indicate that presence of higher levels of MSI2 mediated by CHIP risk allele enhances the clonal expansion of TET2 knockout hematopoietic stem and progenitor cells, providing a mechanistic link whereby non-coding genetic variants can influence the expansion of mutant CHIP clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103a0a6127003db01ce70e503d0d9d03c9b83f1d" target='_blank'>
              Systematic functional dissection of germline noncoding risk variants impacting clonal hematopoiesis
              </a>
            </td>
          <td>
            Trieu Nguyen, Jessica Jeejan, Takeshi Iwasaki, S. Kales, Joyeeta Chakraborty, Chie Yanase, A. Shekhar, Dominika Kwasniak, Aditi Hegde, Richard Voit, Kristy Stengel, Keisuke Ito, Ryan Tewhey, Satish K. Nandakumar
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e23018893423ab572d0573b2072c315fcd456326" target='_blank'>
              OncoBERT: Context-Aware Modeling of Somatic Mutations for Precision Oncology
              </a>
            </td>
          <td>
            Sushant Patkar, Noam Auslander, Stephanie A. Harmon, P. Choyke, B. Turkbey
          </td>
          <td>2026-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2092223941ef46884e67c45fd51c2462a350c8db" target='_blank'>
              Unambiguous signatures of malignancies extracted from images of growing cells
              </a>
            </td>
          <td>
            G. Kalweit, M. Kalweit, W. Checinska, M. Šarić, Ron Berger, E. Bodurova-Spassova, J. Rawluk, N. Talvard-Balland, A. Klett, M. Follo, S. Kreutmair, J. Duque-Afonso, M. LuÌbbert, R. Zeiser, J. Frank, R. Mertelsmann
          </td>
          <td>2026-01-13</td>
          <td>None</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The tumor microenvironment in high-grade serous ovarian carcinoma (HGSC) is a complex network of malignant-host cell interactions, yet its orchestration remains poorly understood. We present a single-cell spatial atlas of metastatic HGSC from 280 patients, integrating high-dimensional imaging and molecular profiling. Analyzing 929 single-cell maps, we identify spatial domains with diverse cell compositions and show that immune cell co-infiltration at the tumor-stroma interface impacts clinical outcomes. Using CEFIIRA, we find that tumor cell MHCII expression is a key predictor of prolonged survival. Validation with deconvoluted, single-cell, and two distinct spatial transcriptomic datasets, along with immunopeptidomic analysis, confirms that MHCII expression correlates with immune activation, antigen presentation, and TCR clonality. Using a patient-derived immuno-oncology platform, we demonstrate that tumor MHCII expression associates with increased CD8+ T cell cytotoxicity after PD-1 blockade, while blocking MHCII inhibits this activation. Our atlas offers new insights into immune activation, potentially improving patient stratification in HGSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fe767539847acd8124897cd09e4464f04ce8afe" target='_blank'>
              Single-cell spatial atlas of high-grade serous ovarian cancer uncovers MHC class II as a key predictor of spatial tumor ecosystems and clinical outcomes.
              </a>
            </td>
          <td>
            Fernando Pérez-Villatoro, Aleksandra Shabanova, Lilian van Wagensveld, Ada Junquera, Iga Niemiec, María Hincapié-Otero, Ziqi Kang, Matias M. Falco, Kürşat Birgin, Sarah Wolf, Ella Anttila, Gayani Anandagoda, J. Casado, Eric Marcus, Duco H K Gaillard, E. Kahelin, Foteini Chamchougia, M. Salko, Saundarya Shah, Salvatore Russo, Jacopo Chiaro, M. Grönholm, Joseph Ndika, Otto K. Kari, iCAN iCAN, G. Sonke, K. K. van de Vijver, Rutgerus Fpm Kruitwagen, M. A. van der Aa, A. Virtanen, Vincenzo Cerullo, Anna Vähärautio, P. Sorger, H. Horlings, Anniina Farkkila
          </td>
          <td>2026-02-09</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Myxofibrosarcoma (MFS) is a rare and aggressive soft tissue sarcoma characterized by high genomic instability, resulting in high local recurrence rates and limited effective therapeutic options in advanced stages. Recent progress in cancer immunology research has encouraged investigation into the Tumor Microenvironment (TME) of sarcomas, including MFS, to identify immune-related biomarkers of prognostic and therapeutic relevance. Although data remain limited in MFS, existing evidence suggests a heterogeneous immune landscape, including: i) variable expression of immune checkpoint molecules such as Programmed Cell Death Protein 1 (PD-1) and Programmed Death-Ligand 1 (PD-L1), ii) presence of tumor-infiltrating lymphocytes, iii) alterations in antigen presentation pathways, and iv) a pronounced angiogenic signature. These findings underscore the potential role of immune biomarkers for patients’ clinical stratification and the consequent possibility of developing new immunotherapeutic strategies. This review will focus on the cellular and molecular architecture of immune infiltration, vascular remodeling, and lymphoid neogenesis, assessing their prognostic and predictive value as potential biomarkers. Finally, we will present ongoing clinical trials aimed at modulating the immune-vascular niche to inform innovative therapeutic strategies for this challenging sarcoma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f542fc2b8b6a3c1d5def7187e73b0d930cab87" target='_blank'>
              Unveiling the myxofibrosarcoma tumor microenvironment: implications for immunotherapy
              </a>
            </td>
          <td>
            Cecilia Profumo, Massimiliano Grassi, Valentina Rigo, Matteo Mascherini, Paola Monti, Giorgia Arcovito, Giorgia Anselmi, Laura Damele, G. Timon, Danila Comandini, Michela Croce
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9001265d82b29583d84a9cb301d0975798f3de8" target='_blank'>
              ESFS: A Noise-Resilient Framework for Feature Selection and Marker Gene Discovery in Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Arthur Radley, Giulia L. M. Boezio, C. Shand, R. Pérez-Carrasco, J. Briscoe
          </td>
          <td>2026-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2227a7d7f1704d0d84703010e3af2b4bec3d96b9" target='_blank'>
              PRDM1 Drives Chemoradiotherapy-associated Enrichment of Adaptive NK Cells in Cervical Cancer
              </a>
            </td>
          <td>
            Meng Wan, Tang Zhong, Wenyang Shi, Jianyu Shen, Wei Zhang, Yizhe Sun
          </td>
          <td>2026-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma remains one of the most lethal brain tumors with limited therapeutic options and a dismal survival rate, largely due to its highly complex Tumor Microenvironment (TME) and frequent drug resistance. In response, three-dimensional (3D) organoid technology has emerged as a powerful tool for modeling glioblastoma, offering a physiologically relevant in vitro system that closely mimics the in vivo architecture, cellular heterogeneity, and drug response of human tumors. Unlike traditional 2D cultures or animal models, glioblastoma organoids enable highthroughput drug screening, personalized therapy testing, and early diagnostic research by preserving key features of the TME. However, despite their immense promise, current organoid models face substantial limitations, including the absence of immune components, functional vasculature, and region-specific neuronal subtypes, thereby restricting their full translational potential, especially for immunotherapy studies. Existing models, such as genetically engineered cerebral organoids (Neo- Cor) and glioblastoma spheroid co-cultures (GLICO), either fail to reflect patient heterogeneity or are constrained by time-intensive preparation. Additionally, patient-derived organoids may lose genetic fidelity over prolonged culture. The novelty of this work lies in its advocacy for engineered cell-based strategies and Adult Stem Cell (AdSC)-derived organoids to overcome these challenges. By enhancing the intrinsic properties of organoid cells and integrating endothelial and immune components, this approach offers a next-generation platform for more accurate modeling of glioblastoma, with greater relevance for drug responsiveness, chemosensitization studies, gene editing, and regenerative applications. This distinguishes the present work from previous studies by not only identifying the gaps in current organoid technologies but also proposing specific, actionable improvements that bring organoid culture closer to clinical translation in glioblastoma research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ee68ff8a6bcd38d17b8f62737e1680b8ae8252d" target='_blank'>
              Glioblastoma Organoid Technology: Approach to Target the Complex Tumor Microenvironment with Promising Drug Responsiveness.
              </a>
            </td>
          <td>
            Priya Bisht, Ruchi Pandey, T. Tabish, N. Thorat, Murali Kumarasamy, Krishna Murti, Nitesh Kumar
          </td>
          <td>2026-01-22</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) enables genome-wide gene expression profiling while preserving tissue architecture, bridging the gap between bulk, single-cell, and histological analyses. Originating in 2016 and rapidly evolving since, ST has transformed infectious disease research by mapping host–pathogen interactions directly within intact tissues. Current platforms fall into two categories: sequencing-based methods (Visium, GeoMx, Stereo-seq) offering whole-transcriptome coverage at modest resolution and imaging-based platforms (Xenium, CosMx, MERFISH) providing single-cell or subcellular detail with targeted gene panels. These technologies reveal spatially organized immune responses, local tissue remodeling, and pathogen niches across viruses, bacteria, and parasites. In viral infection, ST uncovered heterogeneity in COVID-19 lung microenvironments, spatial immune activation in lymphoid tissues, and variant-specific inflammatory patterns. In bacterial disease, ST delineated granuloma architecture in tuberculosis and mapped vaccine-induced lung responses in Shigella studies. Parasitic infection studies identified localized inflammatory hotspots and microenvironmental control of T-cell differentiation in malaria. Despite powerful insights, ST faces constraints including RNA quality limitations, tradeoffs between resolution and transcript breadth, high cost, and analytical complexity. Nonetheless, ST increasingly informs vaccine design by identifying tissue-specific immune programs and protective microenvironments and is poised to become a standard tool for infectious disease biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd879a8a18bd7edf093e16c2214bb3570575ec48" target='_blank'>
              Advances in Spatial Transcriptomics for Infectious Disease Research: Insight for Vaccine Development
              </a>
            </td>
          <td>
            Taehwan Oh
          </td>
          <td>2026-02-07</td>
          <td>Vaccines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c745807b38045a754b3f57a50a75c7b86c914ca" target='_blank'>
              A Scalable Framework for Pan-Cancer Tumor Evolution Analysis Enables Transfer of Progression Mechanisms Across Tumor Entities
              </a>
            </td>
          <td>
            Simon Pfahler, Andreas Lösch, Y. L. Hu, Rudolf Schill, Rainer Spang, T. Wettig
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5108b05893ed4ed9a64a7f2a10e9459a930c1b89" target='_blank'>
              Analysis of the transcriptomic and metabolomic landscape of prostate cancer with different anatomical origins using snFLARE-seq and mxFRIZNGRND.
              </a>
            </td>
          <td>
            Dongyin He, Haoran Hu, Kai Xiao, Yuhang Zhang, Yongbing Cheng, Shaozhuo Jiao, Yimei Hao, Yuanyuan Cai, Ziqun Liu, Xinran Yan, Qinsheng Chen, Xiyan Mu, Qi Wang, Shan Peng, Guoqin Sang, Xia Zhi, Yanxia Chang, Qing Ye, Yuyao Yang, Meixia Che, Shengsong Huang, Hongqian Guo, Luonan Chen, Huiru Tang, Xuefeng Qiu, Zhenfei Li
          </td>
          <td>2026-02-07</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="PURPOSE
Undifferentiated pleomorphic sarcoma (UPS) is an aggressive subtype of soft tissue sarcoma with poor outcomes, particularly in metastatic cases. The mechanisms driving metastasis in UPS remain poorly understood, limiting therapeutic advances.


EXPERIMENTAL DESIGN
A multi-omics approach was employed to analyze paired primary and metastatic UPS tumor samples. Spatial transcriptomics, bulk RNA sequencing, and deconvolution analyses were performed to identify molecular pathways and immune microenvironment alterations associated with metastasis. Functional assays using CRISPR-Cas9 knockout UPS cell lines, alongside in vivo models, were used for functional validation experiments.


RESULTS
Transcriptomic analyses on 13 UPS patients revealed significant upregulation of hypoxia, epithelial-mesenchymal transition (EMT), and immune-suppressive pathways in metastatic UPS. ADORA2B was identified as a key driver of these processes, with elevated expression correlating with poor disease-free survival in UPS patients. Functional studies confirmed that ADORA2B promotes proliferation, migration, invasion, and matrix remodeling via metalloprotease regulation. In vivo, ADORA2B knockout reduced primary tumor growth and metastatic dissemination in UPS models.


CONCLUSIONS
This study identifies ADORA2B as a critical regulator of metastatic progression in UPS, implicating it as a promising therapeutic target. Ongoing clinical trials targeting adenosine pathways further support the translational potential of ADORA2B inhibition to disrupt metastasis and improve outcomes for UPS patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afcc9cdde9fce44c9f1c91b8acb720e8fa688da9" target='_blank'>
              Adenosine A2B Receptor Promotes Tumor Progression and Metastases in Undifferentiated Pleomorphic Sarcoma.
              </a>
            </td>
          <td>
            Mariella Spalato Ceruso, J. Guégan, A. Bourdon, V. Chaire, Yanina Valverde Timana, Alban Giese, C. Rey, A. Bessede, Raoul Perret, Lucile Vanhersecke, Valérie Velasco, P. Finetti, F. Bertucci, S. Verbeke, A. Italiano
          </td>
          <td>2026-02-05</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Cutaneous melanoma is one of the most aggressive forms of skin cancer, marked by rapid progression, early metastasis, and high resistance to conventional therapies. Although targeted therapies and immune checkpoint blockade have improved patient outcomes, many still relapse or fail to respond. This underscores the need for novel biomarkers to refine risk stratification and predict therapeutic responses. Immunogenic cell death (ICD) is a critical process linking tumor cell death to immune activation, which could enhance melanoma treatment responses. This study investigates the role of ICD in melanoma by integrating bulk transcriptomic data with single-cell RNA sequencing (scRNA-seq) to develop a prognostic model based on ICD, and to explore its clinical relevance. Bulk transcriptomic data were sourced from TCGA and GEO, while single-cell RNA-seq data were obtained from GSE215120. ICD-related genes were curated from the literature, and ICD scores were calculated using ssGSEA and GSVA. Machine learning algorithms were used to construct and validate a prognostic model. Immune infiltration and tumor microenvironment (TME) were analyzed to assess the relationship between ICD activity and immune responses. High ICD scores correlated with poor prognosis, and ICD activity was predominantly observed in stromal and myeloid cells. MYO10, identified as a key gene in the ICD signature, was associated with immune evasion and poor clinical outcomes. The ICD-based risk score (ICDRS) demonstrated superior prognostic power in multiple melanoma cohorts and reflected immune activation and TME characteristics. The ICDRS, incorporating MYO10, provides a robust prognostic tool for melanoma and highlights MYO10 as a potential therapeutic target for improving immune responses in melanoma therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9d687935c22333fba1c02521a2fc46ab87e52a" target='_blank'>
              Single-cell profiling of immunogenic cell death in melanoma reveals prognostic signatures and therapeutic targets
              </a>
            </td>
          <td>
            Jingwen Huang, Taoxuan Jiang, Yiyang Han, Chunsheng Yang
          </td>
          <td>2026-01-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In breast cancer patients, metastasis is the stage of disease where prognosis significantly worsens. However, the timing at which metastasis initiates and the location of metastatic lesions in an organ are stochastic, limiting the timely identification of disease and the administration of treatments. Herein, we employ a synthetic metastatic niche comprised of a microporous scaffold to investigate the dynamic immune processes associated with metastatic progression. Upon implantation, the porous scaffold is infiltrated with immune cells and recruits tumor cells. We have previously reported stable tumor cell numbers in the scaffold, suggesting a state of metastatic dormancy. Towards understanding dormancy, we investigated the immune cell dynamics at the scaffold, including neutrophils, monocytes, and dendritic cells, and compared these changes to the lungs, the native metastatic niche in this model. The cell phenotypes within the scaffold microenvironment are initially polarized toward an anti-tumor phenotype and become progressively more pro-tumor with disease progression, similar to the lung microenvironment. However, the phenotypes at the scaffold are consistently less pro-tumor than the phenotypes in the lung, consistent with the lung supporting tumor cell expansion and the scaffold exhibiting dormancy. Signaling pathways identified from the analysis are consistent with the changing innate cell phenotypes, with macrophages having a significant role during the early responses and neutrophils dominating the latter stages of disease. Collectively, the scaffold captures the immune dynamics during disease progression and the signaling that underlies stable tumor cell numbers, providing a tool for investigating the mechanisms of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75a583c884e0287dc2a64d6b8d5b49487f66ecab" target='_blank'>
              Dynamic Immune Cell Composition, Phenotypes, and Signaling in an Engineered Metastatic Niche.
              </a>
            </td>
          <td>
            Rebecca S. Pereles, Jyotirmoy Roy, Michael D. Brooks, Max S. Wicha, J.S. Jeruss, Lonnie D. Shea, Sophia M. Orbach
          </td>
          <td>2026-01-10</td>
          <td>Biotechnology and bioengineering</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumor cell plasticity drives metastasis and therapy resistance, yet its regulation by oncoprotein dosage dynamics remains poorly understood. In Ewing sarcoma (EwS), variations in EWS::FLI1 (EF) fusion oncoprotein activity have been associated with epithelial-mesenchymal plasticity (EMP). Using degron technology, we precisely modulated endogenous EF in EwS cells and linked phenotypic states to distinct oncoprotein dosages. Strikingly, modest EF depletion promoted a pro-metastatic phenotype that diminished upon near-complete EF loss, revealing a paradoxical effect of submaximal EF inhibition. Nascent RNA-sequencing uncovered distinct gene clusters with heterogenous transcriptional responses to graded EF loss. Genes most sensitive to subtle EF depletion harbored GGAA microsatellites within EF-bound enhancers, while chromatin profiling uncovered candidate cofactors regulating EF-repressed EMP programs. Transient EF depletion followed by rapid restoration, modelling oncoprotein fluctuations, caused persistent dysregulation of genes functionally linked to enhanced extravasation and metastatic burden in preclinical models. This study highlights the therapeutic challenge of incomplete EF elimination, serving a paradigm in which oncoprotein dosage dynamics act as non-genetic drivers of disease progression and reveal novel vulnerabilities of advanced disease. The partial and transient suppression of the EF oncoprotein was found to paradoxically promote Ewing sarcoma metastatic potential through persistent transcriptional dysregulation, establishing the oncoprotein dosage dynamics as a key regulator of metastatic plasticity and therapeutic vulnerability. Graded EF depletion using dTAGV-1 elicited a non-linear phenotypic response, with modest EF loss being sufficient to induce a pro-metastatic state. Integration of acute transcriptional responses with chromatin profiling led to the identification of EF co-regulators. EF restoration following transient depletion resulted in lasting dysregulation of pro-metastasis-related genes and emergence of hybrid epithelial-mesenchymal states, thereby enhancing extravasation and lung metastasis in vivo. Partial or transient EF inhibition may promote disease progression and worsen clinical outcomes, highlighting that therapeutic strategies targeting EF must achieve complete and sustained suppression. Graded EF depletion using dTAGV-1 elicited a non-linear phenotypic response, with modest EF loss being sufficient to induce a pro-metastatic state. Integration of acute transcriptional responses with chromatin profiling led to the identification of EF co-regulators. EF restoration following transient depletion resulted in lasting dysregulation of pro-metastasis-related genes and emergence of hybrid epithelial-mesenchymal states, thereby enhancing extravasation and lung metastasis in vivo. Partial or transient EF inhibition may promote disease progression and worsen clinical outcomes, highlighting that therapeutic strategies targeting EF must achieve complete and sustained suppression. The partial and transient suppression of the EF oncoprotein was found to paradoxically promote Ewing sarcoma metastatic potential through persistent transcriptional dysregulation, establishing the oncoprotein dosage dynamics as a key regulator of metastatic plasticity and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5051c5027b542c8e38511f6f4e8d71524906c31f" target='_blank'>
              Modelling EWS::FLI1 protein fluctuations reveal determinants of tumor plasticity in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, Andri Konstantinou, S. Grissenberger, C. Sturtzel, M. Zylka, F. Cidre‐Aranaz, A. Wenninger-Weinzierl, Karla Queiroz, D. Kurek, Martin Distel, Anna C. Obenauf, T. Grünewald, F. Halbritter, H. Kovar, Valerie Fock
          </td>
          <td>2026-01-03</td>
          <td>EMBO Molecular Medicine</td>
          <td>2</td>
          <td>32</td>
        </tr>

        <tr id="


 Triple-negative breast cancer (TNBC) is a highly heterogeneous and aggressive subtype of breast cancer, with the features of tumor microenvironment and the underlying molecular mechanisms of immune-regulation remaining largely elusive. This study aimed to explore the immune landscape of TNBC by assessing tumor-infiltrating lymphocytes (TILs) level, programmed cell death ligand 1 (PD-L1) expression, and the degree of TIL infiltration within the tumor.



 Utilizing the GeoMX Digital Spatial Profiler platform, we also performed spatially resolved transcriptomic profiling of tumor, immune, and stromal cells, focusing on the distinctions between immune-activated and immune-suppressed phenotypes.



 Immune-activated phenotypes, characterized by high TILs, PD-L1 positivity, and close proximity between tumor cells and TILs, were found to correlate with the basal-like immune-activated subtype based on transcriptomic alterations derived solely from tumor cells. These phenotypes were also associated with the upregulation of known immune-related pathways. In contrast, immune-suppressed phenotypes were linked to pathways involved in tumor progression, such as epithelial-mesenchymal transition, TGF-beta signaling, and angiogenesis. Notably, our transcriptomic analysis suggests that tumor cells play a more dominant role than immune cells in shaping the immune phenotype, as indicated by the greater number of differentially expressed genes in tumor cells rather than immune cells. Stratifying patients based on TILs and PD-L1 status revealed that the TIL+PD-L1+ subtype exhibited the most favorable prognosis, a finding further supported by TIL+PD-L1+ signature with excellent survival outcomes in public gene datasets. Conversely, the TIL-PD-L1- subtype displayed characteristics of an "cold" tumor, potentially driven by desmoplastic changes within the tumor microenvironment. Interestingly, the TIL+PD-L1- subtype, despite a high TILs level, was associated with poorer prognosis, likely due to a higher proportion of myeloid cells, decreased activity of immune cells, and increased activation of the adipogenesis pathway across cell types. This finding underscores that in the TIL+PD-L1- subtype, immune cell activity remains low even when TILs infiltrate the intra-tumoral region, highlighting PD-L1 as a critical marker reflecting the immune system status in patients with high TILs.



 In summary, this study emphasizes the clinical significance of integrating TILs and PD-L1 expression as prognostic biomarkers in TNBC and underscores the central role of tumor cells in dictating immune phenotypes. Furthermore, our findings suggest potential underlying mechanisms and target genes regarding the immune system regulation in TNBC. Based on these insights, future research is warranted to develop targeted therapies, especially for immune-suppressed subtypes, aiming to improve clinical outcomes for patients with TNBC.



 Y. Cha, S. Bae, J. Kim, K. Kim, Y. Kook, A. Kim, S. Ahn, J. Jeon. Deciphering the tumor microenvironment related to immune regulation in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS2-11-29.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bff0dac32aaec5003976f59d24cf98dbe5e9986b" target='_blank'>
              Abstract PS2-11-29: Deciphering the tumor microenvironment related to immune regulation in triple-negative breast cancer
              </a>
            </td>
          <td>
            Y. Cha, , , K. Kim, Y. Kook, , S. Ahn, J. Jeon
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) provides valuable insights into the tumor microenvironment by integrating molecular features with spatial context; however, its clinical utility is limited by high costs. To address this, we develop a multi-scale convolutional deep learning framework, HiST, which utilizes ST to learn the relationship between spatially resolved gene expression profiles (GEPs) and histological morphology. HiST accurately predicts tumor regions across multiple cancer types (e.g., breast cancer; area under the curve: 0.96), demonstrating high concordance with pathologist annotations. Moreover, HiST reconstructs spatially resolved GEPs from histological images with an average Pearson correlation coefficient of 0.74 across five cancer types, outperforming existing models by about two-fold. These high-fidelity spatial GEPs enable tumor heterogeneity assessment from histological images, including identification of tumor subtypes with distinct DNA copy number variations. We demonstrate the clinical utility of the predicted GEPs, which robustly stratify patient prognosis across five cancer types from The Cancer Genome Atlas (e.g., breast cancer; concordance index: 0.78). The predicted profiles further facilitate immunotherapy response prediction and enrichment analyses of relevant biological pathways and markers. Collectively, HiST achieves state-of-the-art performance in spatial GEP reconstruction, providing a reliable molecular representation that enhances downstream tasks such as tumor profiling and clinical analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1a859f42b2a4317c7e56d1a0957cb174c649e8f" target='_blank'>
              HiST: Histological Images Reconstruct Tumor Spatial Transcriptomics via MultiScale Fusion Deep Learning.
              </a>
            </td>
          <td>
            Wei Li, Dong Zhang, Eryu Peng, Shijun Shen, Hamid Alinejad-Rokny, Yao Liu, Junke Zheng, Cizhong Jiang, Youqiong Ye
          </td>
          <td>2026-01-05</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Brain metastases have a distinctive vascular ecosystem—shaped by sprouting angiogenesis, vessel co-option, vasculogenic mimicry, and tumor cell transdifferentiation—that governs tumor perfusion, drug exposure, and therapeutic responsiveness. These heterogeneous vascularization patterns exhibit characteristic differences in enhancement morphology, perfusion levels, and metabolic uptake on contrast-enhanced MRI, perfusion imaging, and amino acid PET, providing crucial imaging cues for identifying routes of blood supply, inferring the state of the blood–tumor barrier, and guiding individualized therapeutic strategies. Anti-VEGF therapy is primarily used to alleviate cerebral edema and radiation necrosis, yet it confers limited survival benefit, underscoring the spatiotemporal heterogeneity of the blood–tumor barrier and the persistence of non-classical vascularization pathways. Building on the concept of “vascular normalization,” combinations of anti-angiogenic therapy with immunotherapy, radiotherapy, or targeted agents have shown encouraging intracranial activity in selected settings—most robustly in melanoma brain metastases—but remain insufficiently validated in randomized, brain-metastasis-focused trials. By integrating mechanistic, imaging, and therapeutic perspectives, this review outlines how vascular-ecosystem-based stratification and physics-informed drug-delivery strategies may help transition anti-vascular therapy from symptomatic control toward mechanism-driven precision intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84bf2d925859ab70c28c9ab904400685d28870d9" target='_blank'>
              Mechanisms, Imaging Phenotypes, and Therapeutic Advances of Neovascularization in Brain Metastases
              </a>
            </td>
          <td>
            Siheng Liu, Bingyang Shan, Yiming Zhang, Lixin Xu, Xiaolei Zhang, Liguo Ye, Huantong Diao, Ye Cheng, Jie Tang
          </td>
          <td>2026-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912fa9eb5623540917fa3209ad074268de213852" target='_blank'>
              Molecular and spatial profiling identifies immune endotypes for the stratification of OA patients
              </a>
            </td>
          <td>
            Nicolas Gaigeard, Anaïs Cardon, Romain Guiho, C. Çubuk, Astrid Ouattara, Célia Lamothe, Jordan Brouard, Julien De Lima, Manzoor Ahmed, Mathilde Le Mercier, A. Defois, Lucie Danet, Jimmy Perrot, N. Benzerdjeb, C. Vinatier, R. Danger, L. Delbos, Sophie Brouard, N. Degauque, S. Pagliuca, Myles J. Lewis, L. Fossati-Jimack, A. Nerviani, D. Waast, Benoit Le Goff, Frédéric Blanchard, David Moulin, Costantino Pitzalis, Jérôme Guicheux, M. Boutet
          </td>
          <td>2026-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Neoadjuvant therapy (NAT) is increasingly used for pancreatic ductal adenocarcinoma (PDAC); yet most patients only achieve partial response. Pathological treatment response grading focuses on assessing residual tumor burden, often overlooking changes in tumor microenvironment (TME). To address this gap, we compared tumor cells and TME of 13 NAT-naïve and 23 post-NAT PDACs using integrated spatial pathomics and transcriptomics, with validation in an independent single-cell spatial dataset. NAT significantly reduced tumor burden (14.7%-6.2%, p = 0.004), but systemic comparison of 13 cytomorphometric features of tumor cells alone did not reliably distinguish between naïve and NAT cases. In contrast, NAT profoundly remodeled TME by increasing cancer-associated fibroblast (CAF) and CD8+ T cell densities, promoting CD8+ T cell-tumor cell proximity and fibrosis, reducing tumor-associated neutrophils, and redistributing tertiary lymphoid structures (TLSs). Spatial transcriptomics shows NAT induced apoptosis, DNA-damage response, and AGC-kinase (S_TK_X) signaling in tumor cells, and upregulated complement pathway, p53 signaling, and cellular senescence program in TME. Cross-platform single-cell spatial analysis revealed decreased regulatory T cells (Treg) and a shift from myofibroblastic (mCAF) to inflammatory CAF (iCAF). Importantly, post-NAT patients with more fibrosis had longer overall survival (p = 0.02), and higher B-cell density showed a favorable trend (p = 0.06). Together, these results suggest that beyond tumor debulking, NAT induces a coordinated TME remodeling characterized by fibroblast reprogramming, matrix fibrosis, and immune spatial reorganization. Incorporating assessment of NAT-induced stromal and immune changes into TRG may improve prognostication and guide more precise therapy in post-NAT PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc4f312a97dc39d70b704d780122da11f4cc3af9" target='_blank'>
              Neoadjuvant Therapy-Induced Remodeling in Pancreatic Ductal Adenocarcinoma: Multimodal Spatial Analysis and Prognosis.
              </a>
            </td>
          <td>
            Xiaofei Zhang, Ruoxin Lan, Danting Li, Yongjun Liu, S. Kalyan, Momin Iqbal, Nancy Liu, Jerry Zhang, Iman Hanna, Mala Gupta, C. Zhao, Weiguo Liu, Jonathan Melamed, Michael Shusterman, Jessica Widmer, John Allendorf, Yao-zhong Liu
          </td>
          <td>2026-01-13</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Pancreatic cancer (PC) is among the most aggressive and lethal malignancies, characterized by development within a complex tumor microenvironment (TME) that includes a desmoplastic stroma composed of extracellular matrix (ECM) and various cellular components.


OBJECTIVES
This study aims to elucidate the cellular and molecular mechanisms regulating PC progression through an integrated analysis of single-cell pseudotime trajectories and intercellular communication.


MATERIAL AND METHODS
We constructed pseudotime trajectories using single-cell RNA sequencing (scRNA-seq) data from PC tissues to trace the developmental progression of cancer cells. Transitional cell states and critical genes involved in the shift from early-to-advanced disease stages were identified. Through a comprehensive analysis, we pinpointed key transcription factors and signaling pathways implicated in tumor progression. Expression of stemness-associated genes in pancreatic stellate cells (PSC) was validated using immunofluorescence and transmission electron microscopy (TEM). Additionally, cell-cell communication analysis was performed to examine interactions within the TME, with particular emphasis on ligand-receptor pairings.


RESULTS
Our analysis identified key transcription factors and signaling pathways that drive the cellular transitions associated with cancer progression. The findings revealed extensive intercellular crosstalk between cancer cells, stromal fibroblasts, and diverse immune cell subpopulations. Notably, the study underscored the distinct functional contributions of these cell populations to tumor development, immune evasion and metastatic dissemination.


CONCLUSIONS
The study uncovers the complex cellular diversity and intercellular crosstalk in PC, providing novel avenues for therapeutic interventions and early predictive markers in diagnosis. These findings support the potential for more targeted, personalized treatment strategies in combating PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1072a0d2e360d78c1789524d36e7d5c1df78fbc" target='_blank'>
              Single-cell pseudotime and cell communication analysis of pancreatic cancer.
              </a>
            </td>
          <td>
            Chengming Ni, Xiaohang Wang, Zhensheng Cai, Yang Chen, Huan Wang, Qianqian Wang, Hao Lin, Yunting Zhou, Yang Yuan, Bo Sun, Zilin Sun
          </td>
          <td>2026-02-09</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 The tumor microenvironment (TME) plays a prominent role in critical elements of breast cancer progression including metastatic dissemination and therapeutic response and resistance. Differences between efficacy of checkpoint inhibitors as monotherapies as compared to combinations with conventional chemotherapies or antibody drug conjugates promotes the hypothesis that cancer-directed therapies not only impact tumor cells directly but also shape TME dynamics with potentially pivotal consequences on therapeutic response. Unfortunately, the impact of even common therapies on TME interactions as they occur in patients remains incompletely defined, particularly in HR+ metastatic breast cancer (MBC).



 Here, we leveraged recent advances in single-cell and spatial profiling techniques to address this shortcoming. Biopsy cores of liver metastases from 40 patients with HR+ MBC were profiled with both single-nucleus RNA sequencing (snRNA-seq) and Multiplexed Error-Robust Fluorescence In Situ Hybridization (MERFISH).



 Integrating the snRNA-seq and MERFISH profiles yielded detailed description of the composition, gene expression profiles/programs and spatial organization of HR+ liver metastases. Of the 40 biopsies, 32 were from patients previously treated with taxanes: 16 in the adjuvant setting (treatment-biopsy interval 18-222 months; mean 84); 10 in both the adjuvant and metastatic settings (interval 0-61 months; mean 18); and 6 only in the metastatic setting (interval 1-48 months; mean 20). Multiple gene expression profiles were increased in biopsies from patients exposed in the metastatic setting only. The malignant cells from these were characterized by enriched oxidative phosphorylation (Wilcoxon rank sum p=0.022), glycolysis (p=0.029), and hypoxia (p=0.033) gene sets as well as increased MYC regulon activity. Correspondingly, the TME was enriched for HIF1A regulon activity as well as SPP1+ lipid-scavenging (p = 0.047) and angiogenic (p=0.035) macrophages, pericytes (p=0.031), lymphatic endothelial cells (p=0.009), and proliferating endothelial cells (p=0.003). Integrated spatial analyses revealed that this phenotype was elevated in regions of metastatic involvement compared to the adjacent normal liver (e.g. p=0.002 for lipid-scavenging macrophages), while interferon-induced inflammatory macrophages were relatively depleted (p=0.040).



 Taken together, these data not only furnish a detailed atlas of liver metastases in HR+ breast cancer, a common and clinically important site of metastatic spread with adverse prognostic implications in the most common, but seemingly least immunogenic, breast cancer subtype, but also map the imprint of taxane-based chemotherapy. We identify the setting of taxane exposure as a critical determinant of the impact of this key therapeutic class, with specific relevance in combination with immunotherapy, on the TME. Notably, taxane exposure limited to the metastatic setting was associated with a hypoxic/angiogenic and immunosuppressive/immune-excluded TME, even compared to cases with taxane treatment in both the adjuvant and metastatic setting, highlighting opportunities for future clinical translation.



 R. Strasser, Y. Tam, R. M. Oscanoa, Å. Segerstolpe, J. Nelson, C. McCann, L. DelloStritto, A. Frangieh, K. Helvie, S. Vigneau, A. R. Thorner, K. Nguyen, S. Strauss, N. U. Lin, S. M. Tolaney, S. Farhi, B. E. Johnson, N. Wagle, J. Klughammer, D. L. Abravanel. Mapping the imprint of taxane chemotherapy on the metastatic microenvironment in HR+ breast cancer with single cell and spatial profiling [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS3-12-15.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63a63859a22125c666b05abca948458114bf9b33" target='_blank'>
              Abstract PS3-12-15: Mapping the imprint of taxane chemotherapy on the metastatic microenvironment in HR+ breast cancer with single cell and spatial profiling
              </a>
            </td>
          <td>
            R. Strasser, Y. Tam, R. M. Oscanoa, Å. Segerstolpe, , C. McCann, L. Dellostritto, A. Frangieh, K. Helvie, S. Vigneau, A. Thorner, , S. Strauß, N. Lin, S. Tolaney, S. Farhi, , N. Wagle, J. Klughammer, D. Abravanel
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Automatic integration of whole slide images (WSIs) and gene expression profiles has demonstrated substantial potential in precision clinical diagnosis and cancer progression studies. However, most existing studies focus on individual gene sequences and slide level classification tasks, with limited attention to spatial transcriptomics and patch level applications. To address this limitation, we propose a multimodal network, BioMorphNet, which automatically integrates tissue morphological features and spatial gene expression to support tissue classification and differential gene analysis. For considering morphological features, BioMorphNet constructs a graph to model the relationships between target patches and their neighbors, and adjusts the response strength based on morphological and molecular level similarity, to better characterize the tumor microenvironment. In terms of multimodal interactions, BioMorphNet derives clinical pathway features from spatial transcriptomic data based on a predefined pathway database, serving as a bridge between tissue morphology and gene expression. In addition, a novel learnable pathway module is designed to automatically simulate the biological pathway formation process, providing a complementary representation to existing clinical pathways. Compared with the latest morphology gene multimodal methods, BioMorphNet's average classification metrics improve by 2.67%, 5.48%, and 6.29% for prostate cancer, colorectal cancer, and breast cancer datasets, respectively. BioMorphNet not only classifies tissue categories within WSIs accurately to support tumor localization, but also analyzes differential gene expression between tissue categories based on prediction confidence, contributing to the discovery of potential tumor biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62d9ab6f28be06b2fcafb69ae7d77db142d962de" target='_blank'>
              Tissue Classification and Whole-Slide Images Analysis via Modeling of the Tumor Microenvironment and Biological Pathways
              </a>
            </td>
          <td>
            Junzhu Liu, Xuemei Du, Daniel Reisenbüchler, Ye Chen, Markus Eckstein, C. Matek, Friedrich Feuerhake, D. Merhof
          </td>
          <td>2026-01-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="BACKGROUND
Spatial proteogenomics marks a paradigm shift in oncology by integrating molecular analysis with spatial information from both spatial proteomics and other data modalities (e.g., spatial transcriptomics), thereby unveiling tumor heterogeneity and dynamic changes in the microenvironment.


METHODS
We systematically reviewed the evolution of spatial proteogenomics, from single-modality profiling to integration with transcriptomics and metabolomics, from the detection of abundant proteins to exploration of "dark proteome" with low abundance or stability, and from analytic software based on traditional machine learning algorithms to advanced artificial intelligence-driven analytical frameworks.


RESULTS
Key advances of sub-fields of spatial proteogenomics include: RNA-protein co-localization: Spatial CITE-seq, enabling RNA-protein co-localization to reveal immune microenvironmental patterns and neoantigen distribution. Spatial Proteomics + Spatial Metabolomics: Matrix-assisted laser desorption/ionization imaging (MALDI), overcoming protein detection bottlenecks and capturing metabolic reprogramming. Deep visual proteomics (DVP): achieving unbiased spatial analysis via AI-guided microdissection. Spatial-aware multiplex dark proteome approaches: Examples are nanodroplet processing in one pot for trace samples (NanoPOTS) and proteoform imaging mass spectrometry (PiMS). Multimodal foundation AI models: Examples are KRONOS and HEIST, which integrate multiple data modalities and significantly improve diagnostic precision and therapeutic prediction.


CONCLUSIONS AND FUTURE DIRECTIONS
Despite challenges of resolution, standardization, and data complexity, spatial proteomics is advancing rapidly. Together with frontier technologies such as quantum computing, live imaging, and organoid integration, it is driving breakthroughs in cancer diagnosis, personalized immunotherapy, and drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae06afb0f045f1c14c02032ac2dc72df3bf1009d" target='_blank'>
              Integrating Spatial Proteogenomics in Cancer Research.
              </a>
            </td>
          <td>
            Yida Wang, Yang Wu, Feng Zhang, Parthiban Periasamy, Haiyue You, Denise Goh, R. E. Fincham, Xin Ning, Danping Wu, Lu Liu, Ying Jiang, Zhiwen Qian, Joe Yeong, Yan Zhang
          </td>
          <td>2026-02-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer‐related mortality worldwide, with tumor microenvironment (TME) heterogeneity playing a critical role in disease progression and therapeutic response. Immune escape (IE) mechanisms facilitate tumor evasion from host immune surveillance, yet their characterization at the single‐cell level in CRC is incomplete. This study integrated single‐cell RNA sequencing (scRNA‐seq) and bulk transcriptomic data from multiple public cohorts to systematically explore IE‐related signatures in CRC. We identified major and minor cell populations within the TME and performed differential gene expression analysis. Using high‐dimensional weighted gene coexpression network analysis (hdWGCNA), we identified gene modules correlated with IE activity. Subsequent survival analysis across six independent cohorts revealed Gamma‐glutamylcyclotransferase (GGCT) as a novel prognostic biomarker associated with poor survival. Functional enrichment analysis indicated GGCT′s involvement in critical oncogenic pathways. Furthermore, GGCT expression correlated with altered immune infiltration profiles and stromal components, suggesting its role in modulating the immunosuppressive TME. Additionally, GGCT demonstrated potential predictive value for response to immunotherapy across multiple datasets. Our findings highlight GGCT as a key player in CRC immune evasion and a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fcee0e763a97a07d783d6f8f51f97fc2a1182ca" target='_blank'>
              Integrative Multiomics Nominate GGCT as a Crucial Regulator of Immunosuppression in Colorectal Cancer
              </a>
            </td>
          <td>
            Qichao Niu, Yang Liu, Kejin Huang, Lisheng Nie, Shiming Zhao, Shifeng Yang, Changlei Su, Shuyu Li
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Cutaneous melanoma is one of the most aggressive forms of skin cancer and remains difficult to treat due to its ability to adapt and develop resistance to therapy. Recent research suggests that a small subset of tumor cells with stem-like properties play an important role in disease progression and treatment failure. One molecule of growing interest is ABCB5, a membrane transporter associated with drug resistance, immune evasion, and tumor survival. This review summarizes current knowledge on how ABCB5 contributes to melanoma progression by influencing tumor cell plasticity and the surrounding tumor microenvironment. We discuss the biological mechanisms linked to ABCB5 expression, its potential value as a prognostic biomarker, and its relevance for future therapeutic strategies. A better understanding of ABCB5 may help guide the development of more effective and personalized treatment approaches for patients with melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e464449af07273c0b0477651738ff182ea906fd3" target='_blank'>
              ABCB5: A Key Regulator Linking Stem Cell Plasticity, Tumor Microenvironment, and Therapy Resistance in Cutaneous Melanoma
              </a>
            </td>
          <td>
            A. Tinca, Adriana Sabău, A. Cozac-Szőke, D. Chiorean, B. Lazar, Raluca-Diana Hagău, I. Cocuz, Raluca Niculescu, Irina-Bianca Kosovski, Sofia Teodora Muntean, S. Turdean, O. Cotoi
          </td>
          <td>2026-01-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="(Isocitrate dehydrogenase) IDH-mutant astrocytoma is classified as World Health Organization (WHO) grade 2–4 and is second only to IDH wild-type glioblastoma in the incidence of adult glioma. However, few studies use single-cell and spatial transcriptome sequencing to analyze its malignant progression. Intraoperative navigation and yellow fluorescence visualization were utilized to accurately isolate high-grade (WHO grade 3–4) and low-grade (WHO grade 2) samples of IDH-mutant astrocytoma for single-cell and spatial transcriptome sequencing. By combining single-cell, spatial transcriptome, The Cancer Genome Atlas (TCGA), and The Chinese Glioma Genome Atlas (CGGA) data, analyses of survival, enriched pathways, transcription factors, intercellular communication, differentiation trajectories, and immune response were performed to identify the characteristics of a unique subpopulation of high-grade IDH-mutant astrocytoma. Our single-cell RNA sequencing analysis identified a distinct subpopulation (Cluster 7) present in high-grade IDH-mutant astrocytoma, which was localized to the terminus of the pseudotime trajectory. Importantly, this cluster not only exhibited an immunosuppressive phenotype correlated with poor clinical prognosis, but also demonstrated significant enrichment in Developmental Biology and Calcium Signaling pathways. Furthermore, this subpopulation engaged in prominent ligand-receptor interactions, particularly through PTN_PTPRZ1 and MIF_CD74 pairs. Notably, comparative analysis revealed that high-grade astrocytoma displayed both quantitatively and qualitatively enhanced communication networks when compared to their low-grade counterparts. Our single-cell RNA sequencing analysis identifies a distinct tumor cell subpopulation present in high-grade (WHO grade 3–4) adult IDH-mutant astrocytoma. This cluster, which likely arises from malignant progression in adult astrocytoma, may provide new insights for developing therapeutic strategies against this clinically challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/586fc47b7312366f86c32ecb5b1181c5a720190b" target='_blank'>
              Single-cell and spatial transcriptome sequencing analysis reveals characteristics of a unique subpopulation in high-grade IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            Wei Chen, Min Zheng, Jia-Yin An, Guo-Hao Huang, Jing-Peng Liu, Lin Yang, Peng Ren, Tingting Wang, Jean-Philippe Hugnot, Sheng-Qing Lv
          </td>
          <td>2025-12-29</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Gliomas are the most prevalent primary malignant neoplasms of the central nervous system, distinguished by their high recurrence rates and poor prognosis. Aerobic glycolysis in tumors generates excess lactate, which promotes lactylation, a post-translational modification (PTM). Although accumulating evidence implicates lactylation in glioma initiation and progression, previous lactylation-focused prognostic studies lacked single-cell resolution and broad validation, limiting their generalizability and clinical relevance. Single-cell and bulk RNA sequencing (RNA-seq) data were integrated to identify lactylation-enriched tumor cell populations and derive candidate genes. A risk model was developed using univariate Cox regression and the Least Absolute Shrinkage and Selection Operator (LASSO), and its predictive performance was validated in independent cohorts from the China Glioma Genome Atlas (CGGA). To improve clinical applicability, a nomogram integrating the risk score incorporating key clinical variables was constructed and externally validated. The risk groups showed distinct immune microenvironment profiles and differential drug sensitivity patterns. In this study, we established and validated a lactylation-related gene signature, with the derived risk score serving as a reliable prognostic biomarker for glioma. Furthermore, the model not only predicts overall survival (OS) but also exhibits the potential to inform drug selection and stratify patients for more precise and personalized therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f50a48a658977c34551d0825222a3b9aeb7a5b9" target='_blank'>
              Integrating Single-Cell and RNA Sequencing to Predict Glioma Prognosis Through Lactylation
              </a>
            </td>
          <td>
            Ruyi Shen, Yinan Chen, Yan Li, Zhijie Lin
          </td>
          <td>2026-02-08</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43773501df06a12f0f087012a397c0411bf828e5" target='_blank'>
              Differences in regional developmental origins of glioblastomas revealed by integrated molecular profiling
              </a>
            </td>
          <td>
            Ryan Gensler, A. K. Ahmed, Melanie Alfonzo Horowitz, J. Khalifeh, Marvin Li, Negar Sadeghipour, Mostafa Abdulrahim, Mike Glantz, Sonikpreet Aulakh, Theodore Nicolaides, Calixto-Hope G. Lucas Jr., Srinivasan Yegnasubramanian, Jordina Rincon-Torroella, Chetan Bettegowda
          </td>
          <td>2026-01-19</td>
          <td>Journal of Neuro-Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary Clear cell renal cell carcinoma is the most common form of kidney cancer. Recent advances in spatial and long-read sequencing now allow detailed examination of gene activity within tumours. We used non-passaged, patient-derived tumour organoid models which preserve the structure and cellular composition of the original tumours. Long-read spatial transcriptomics was then applied to study gene expression and transcript variants across organoid regions. We also examined how these patterns change following treatment with the experimental medicine NUC-7738. There are distinct spatial patterns in genes related to protein synthesis and energy metabolism, as well as region-specific differences in transcript isoforms. This work provides new insight into the molecular diversity of kidney cancer and demonstrates how advanced sequencing technologies can be used to study treatment effects in physiologically relevant tumour models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11ea98f536949abee4bc7391c3123466548631ed" target='_blank'>
              Long-Read Spatial Transcriptomics of Patient-Derived Clear Cell Renal Cell Carcinoma Organoids Identifies Heterogeneity and Transcriptional Remodelling Following NUC-7738 Treatment
              </a>
            </td>
          <td>
            Hazem Abdullah, Ying Zhang, Kathryn Kirkwood, Alexander Laird, Peter Mullen, David J. Harrison, Mustafa Elshani
          </td>
          <td>2026-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Somatic mosaicism is pervasively observed in human aging, with clonal expansions of cells harboring mutations in recurrently mutated driver genes. Bulk sequencing of tissues captures mutation frequencies, but cannot reconstruct clonal architectures nor delineate how driver mutations impact cellular phenotypes. We developed single-cell Genotype-to-Phenotype sequencing (scG2P) for high-throughput, highly-multiplexed, joint capture of genotyping of mutation hotspots and mRNA markers. We applied scG2P to aged esophagus samples from six individuals and observed large numbers of clones with a single driver event, accompanied by rare clones with two driver mutations. NOTCH1 mutants dominate the clonal landscape and are linked to stunted epithelial differentiation, while TP53 mutants promote clonal expansion through both differentiation biases and increased cell cycling. Thus, joint single-cell highly multiplexed capture of somatic mutations and mRNA transcripts enables high resolution reconstruction of clonal architecture and associated phenotypes in solid tissue somatic mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e453d2ebb8e890d0f12017626a25469ee8fcb34" target='_blank'>
              Genotype-to-phenotype mapping of somatic clonal mosaicism via single-cell co-capture of DNA mutations and mRNA transcripts.
              </a>
            </td>
          <td>
            Dennis J. Yuan, John Zinno, Theo Botella, Dalia Dhingra, Shu Wang, Allegra G Hawkins, Ariel D Swett, Jesus Sotelo, Ramya Raviram, Clayton Hughes, Catherine Potenski, Katharine D Godfrey, Kara M Ainsworth, Shuzhen Xu, Jianwen Que, Julian A Abrams, Akira Yokoyama, N. Kakiuchi, Seishi Ogawa, D. Landau
          </td>
          <td>2025-12-31</td>
          <td>Cancer discovery</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d88c7691d9d88eec5a2eb2535a93b8221e24efe" target='_blank'>
              SigFormer: an Attention-Based Framework for Robust Single-Sample Mutational Signature Decomposition
              </a>
            </td>
          <td>
            Yang Zhang, Muchun Niu, Chenghang Zong
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) remains a lethal malignancy lacking reliable biomarkers for precancerous lesion monitoring and intervention. In this study, formalin-fixed, paraffin-embedded archived tissue biopsies from 134 individuals across two cohorts are collected, spanning five stages of ESCC progression. Laser capture microdissection is employed to isolate the epithelial lesion (L) and the adjacent non-lesion (N) tissues. Proteomic profiling reveals a comprehensive spatio-temporal landscape of ESCC progression, encompassing 4461 proteins. Dynamic network biomarker analysis indicates moderate dysplasia (MOD) as the critical turning stage, warranting clinical attention. A seven-protein diagnostic panel (CCDC86, GBP6, PDCD6IP, C19orf53, SF3A3, GMPPB, ARPC5) with an area under the curve (AUC) of 0.956 achieves superior early detection, and the key signatures are validated by immunohistochemistry in an independent cohort. Functional validation using malignantly transformed HET-1A cells, ESCC cells, and mouse xenograft models identifies GBP6 as a novel target: DNA damage-induced progressive loss of GBP6 promotes ESCC progression by accelerating cell cycle and inducing epithelial-mesenchymal transition. Critically, PARP1 inhibition rescues GBP6 loss by suppressing TP63 and prevents ESCC progression. Overall, this study provides a systematic proteomic atlas of ESCC progression, identifies MOD as a pivotal clinical decision point, and proposes PARP1-TP63-GBP6 axis targeting as a novel intervention strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddf5c8a9af6fc57009a93bae9671de888e2c59fc" target='_blank'>
              Spatio-Temporal Proteomic Landscape Reveals Early Warning Signals of Esophageal Squamous Cell Carcinoma Progression.
              </a>
            </td>
          <td>
            Xumiao Li, Jie Yuan, Min Gao, Jibin Liu, Qinqin Wang, Yaqi Zhang, Mingtao Cao, Xiaolin Hu, Hui Yang, Jun Li, Chen Li, Xiaoguang Li, Hui Wang
          </td>
          <td>2026-01-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99817029794cbefbdaf247b4f46b15275fd31679" target='_blank'>
              Metabolic reprogramming in non-small cell lung cancer: unraveling therapeutic vulnerabilities and resistance mechanisms
              </a>
            </td>
          <td>
            T. Addissouky
          </td>
          <td>2026-01-17</td>
          <td>Journal of Rare Diseases</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eec9f36f629bd2b6951e1744a883847ea5c28f8" target='_blank'>
              Programming T cells for Early Cancer Detection with Customized Protease-Activatable Receptors
              </a>
            </td>
          <td>
            H. Phuengkham, Ying Chen, Anirudh Sivakumar, Ali H. Zamat, Lena Gamboa, Quoc D. Mac, Hee Jun Lee, Leonard C. Rogers, Jicheng You, Sean A. Steele, Sichen Zhu, Matthew S. Gollins, J. Blazeck, Peng Qiu, G. Kwong
          </td>
          <td>2026-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Intratumoral heterogeneity drives immunotherapy resistance, hindering clinical efficacy. Functional assays that capture heterogeneous responses to immunotherapies remain limited. Here, we present a 3D live imaging assay using patient-derived ovarian cancer organoids to quantify natural killer (NK) cell-induced apoptosis across hundreds of individual organoids. Our assay reveals inter- and intratumor response heterogeneity and detects rare populations unresponsive to NK cytotoxicity, enabling the study of subclonal immune responses and underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c2a1ea73f6fab9f1966b90164d74f94fad3f917" target='_blank'>
              Large scale quantification of natural killer cell-induced apoptosis in patient-derived organoids reveals intratumoral response heterogeneity
              </a>
            </td>
          <td>
            Marisa Mercadante, Armin Scheben, Jacob Estrada, Jan Savas-Carstens, William Sullivan, Nicholas Housel, Maria Sapar, Tatiana Volpari, Jax Hebner, Tomasz Rusielewicz, Frederick J. Monsma, Stefan Semrau, Yinan Wang, Laura A. Martin
          </td>
          <td>2025-12-26</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Gastric cancer (GC) exhibits molecular heterogeneity and diverse immune cell infiltration patterns closely associated with patient prognosis. However, a comprehensive understanding of the variations in immune cell phenotypes among different patient subgroups still needs to be improved. In this study, we performed a detailed analysis of the tumor microenvironment in GC by integrating 200,466 single cells from 72 patients across six datasets. We classified patients into immune-deserted, B, T, and myeloid cell subtypes. Using genomic and clinical data from TCGA samples, we identified cellular components associated with tumor histology and genotypes. GC patients were stratified into immune-deserted, B cell, T cell, and myeloid cell subtypes, and we described the pathway and transcription factor activity characteristics of different microenvironment subtypes. Integration of bulk RNA-seq data reveals that fibroblasts and endothelial cells were associated with adverse patient outcomes whereas NK and T cells were notably correlated with improved prognosis. Subsequently, we focused on characterizing cancer-associated fibroblasts (CAFs) and discovered that they acquire new functional properties within the tissue microenvironment, providing evidence of CAF plasticity. We constructed a novel four-gene CAF signature including SPARC, EFEMP1, RGS5 and SERPINE1 which may enhance patient stratification and prognostic prediction of GC patients. qPCR analysis revealed that the significant expressions of SPARC, EFEMP1, RGS5 and SERPINE1 were significantly upregulated in gastric cancer tissues compared to the normal tissues. Our study provides insights into the composition of the tumor microenvironment and construction of a four-gene CAF signature associated with clinical prognosis, offering new perspectives for the clinical management of gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6b22dc4e8c3bd98a01915b46ab3333a465c3c7e" target='_blank'>
              Integrative single-cell atlas unveils heterogeneity and prognostic value of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Ziyu Jiang, Xiaomei Zhuang, Jun Guo, Minyi Zhu, Chunhong Hong, Chunhui Sun, Kaiming Wu, Haofan Yin, Cuncan Deng, P. Jiang
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ab2ef7cfea18c9d2b8d0291643654a27ff4767a" target='_blank'>
              ISG15-driven immune modulation and tumor progression in breast cancer metastasis: insights from single-cell and spatial transcriptomics.
              </a>
            </td>
          <td>
            Hua Shao, Hanlu Tang, Huiying Lin, Yongqing Xu
          </td>
          <td>2026-02-04</td>
          <td>BMC medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Brain invasion of meningioma is a controversial clinicopathological grading criterion that correlates with prognosis and is a stand-alone criterion for WHO grade 2 meningioma. Molecular correlates of meningioma cells and immune cell infiltration at the brain-meningioma border and its possible role in brain invasion are not known. We hypothesized that brain-invasive meningiomas have distinct gene expression profiles in meningioma cell populations and in tumour-associated microglia and macrophages (TAM) populations, most prominently at the brain-meningioma border. We performed spatial transcriptomics using NanoString GeoMx Digital Spatial Profiling to investigate the gene expression profiles of meningioma cell-enriched populations (Iba1-/CD68-) and TAM-enriched populations (Iba1+/CD68+) across 16 tumours. Regions of interest included core regions of meningiomas, the brain-meningioma border, and brain regions with confirmed invasion. Using an 1800 gene panel, we analysed differential gene expression across regions within each tumour and compared brain-invasive and non-invasive meningiomas. Meningioma cell-enriched populations from brain-invasive meningiomas (n = 8) showed significant upregulation of KRT18, implicated in filament reorganization, as well as genes involved in the cell cycle, glycolysis, and the growth factor receptor PDGFRB compared to non-invasive meningiomas (n = 8). Meningioma cell-enriched populations from the brain-meningioma border showed significant upregulation of KRT18, NDUFA4L2, and PKM relative to core areas. In TAM-enriched populations in brain tissue, we found upregulation of the chemokine receptor gene CSF1R relative to TAM-enriched populations in meningioma tissue and increased TAM infiltration in brain-invasive cases. In conclusion, our results demonstrate molecular changes in brain-invasive meningiomas compared to non-invasive meningiomas as well as spatial changes along the core-border axis. The expression pattern of TAMs also changed from meningioma to brain tissue. Additional studies are needed to confirm these findings and further reveal how we can target meningioma brain invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/269f040bbd53f8bc31b9e6b6faae7f3967b17a9f" target='_blank'>
              Novel insights into meningioma brain invasion with spatial transcriptomic profiling
              </a>
            </td>
          <td>
            A. Maier, Claes Ottzen Laurentiussen, S. Michaelsen, Lorenzo Perino, T. Stannius, D. Harwood, M. N. Klausen, J. Haslund-Vinding, N. Bager, David Scheie, A. Schmidt, L. Melchior, A. Areškevičiūtė, Frederik Vilhardt, K. Josefsen, D. Raleigh, F. Bagger, Tiit Mathiesen, B. W. Kristensen
          </td>
          <td>2026-01-04</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background The complexities of nucleotide metabolism in breast cancer (BC) cells are not yet fully understood. A deeper exploration of the various tumor subpopulations and their interactions with the tumor microenvironment (TME) could provide important insights into these clinically relevant signaling pathways. Methods We integrated five distinct single-cell enrichment scoring methodologies to conduct a comprehensive enrichment analysis of BC cell populations. The analytical findings underwent subsequent validation using an independent single-cell cohort. Tumor cell clusters were categorized based on their average enrichment scores. Functional analyses were carried out using several tools, including CellChat, Monocle, CopyKAT, SCENIC, and CytoTRACE. The RCTD method was then employed to map the single-cell clusters onto spatial transcriptomics data, facilitating the evaluation of cellular dependencies and pathway activities to differentiate tumor cell subtypes. A prognostic framework was subsequently established using large-scale transcriptomic datasets, enabling prediction of immunotherapy responsiveness. Experimental validation further confirmed expression patterns of pivotal genes implicated in therapeutic outcomes. Results Tumor cells exhibit significantly upregulated nucleotide metabolic activity, enabling their classification into two distinct subgroups: NUhighepi and NUlowepi. Cells within the NUhighepi subgroup demonstrate pronounced malignant phenotypes. Intercellular communication analysis performed with the stLearn platform revealed robust interactions between NUhighepi cells and fibroblasts. Supporting this finding, spatial transcriptomic analysis via MISTy revealed a distinct dependency of NUhighepi on fibroblasts. A robust prognostic model, developed using various machine learning algorithms, was able to predict survival outcomes and responses to immunotherapy. Furthermore, targeted drugs were identified for both the high and low scoring groups. Experimental investigations confirmed the expression of core genes in different breast cancer cells. Discussion In conclusion, we developed a nucleotide metabolism-derived prognostic signature for BC, with DCTPP1 highlighted as a promising biomarker and therapeutic target. These findings provide a valuable framework for early clinical intervention and show promising potential for predicting responses to immunotherapy in BC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502ca564395f021e293ba0e223c1f50472f85541" target='_blank'>
              Spatial transcriptome and single-cell sequencing reveal the role of nucleotide metabolism in breast cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yuejun Pan, Yong Xu, Ke Gong, Songlin Yuan
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604fc75ae6b3d011181af39eae23a0a3c99bbefb" target='_blank'>
              MicroRNA spatial profiling for assessing drug efficacy in BRCA1-related triple-negative breast tumors
              </a>
            </td>
          <td>
            Omar N. Mohd, Lin Wang, Brian R. Sardella, David Jou, Gerburg M. Wulf, Frank J. Slack, Yujing J. Heng, Patrick S. Doyle
          </td>
          <td>2026-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Cancer progression is influenced not only by cancer cells themselves but also by surrounding supportive cells in the tumor environment. Among these, cancer-associated fibroblasts play important roles in shaping tumor behavior and immune responses, but their diversity and clinical significance in head and neck cancer are not fully understood. In this study, we analyzed single-cell gene expression data to identify distinct fibroblast subtypes in head and neck squamous cell carcinoma. We further examined whether gene signatures derived from these fibroblast subtypes could predict patient outcomes using large clinical datasets. We found that specific inflammatory fibroblast programs were strongly associated with patient survival and distinct immune environments. Importantly, these fibroblast-related patterns were reproducible across independent patient cohorts and were supported by additional experiments using cultured fibroblasts. Our findings provide new insights into the biological and clinical relevance of fibroblast diversity and suggest that stromal features may complement existing biomarkers to improve risk stratification in head and neck cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39c29db87141bfec7ddb84a086ca849e34133944" target='_blank'>
              Cancer-Associated Fibroblast Heterogeneity Shapes Prognosis and Immune Landscapes in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Hideyuki Takahashi, Hiroyuki Hagiwara, Hiroe Tada, Miho Uchida, T. Matsuyama, K. Chikamatsu
          </td>
          <td>2026-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Almost 200 years of histological and molecular analysis has established that functional shifts in vascular cell populations are associated with healthy vascular function and the progression of vascular disease. Now, new methods in single-cell analysis are serving to dramatically accelerate the study of vascular cell heterogeneity. Here, we will outline the experimental and computational technologies that have made high-throughput analysis of single cells possible, and review recent studies applying these approaches to vascular cells and tissues. In particular, the application of single-cell or single-nucleus RNA sequencing has identified rare and disease-specific cell populations, drivers of cellular heterogeneity, and specific vascular disease-relevant cell populations. High-throughput approaches linking CRISPR (clustered regularly interspaced short palindromic repeats) perturbations to single-cell RNA sequencing data are providing new insights into cell type-specific mechanisms of disease, and connecting human genetic data to these mechanisms. Other single-cell approaches are providing insights into regulatory mechanisms by linking chromatin accessibility to transcription in single cells and revealing the spatial positioning of rare cell types in vascular tissues. With a variety of well-established methods and the continued development of new technologies, single-cell approaches are becoming indispensable and powerful avenues for discovering and detailing new mechanisms of vascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2acac58657bce47289e81c369cc7e081a7dfdf40" target='_blank'>
              Decoding Vascular Cell Diversity: Single-Cell Approaches to Mechanisms of Vascular Disease.
              </a>
            </td>
          <td>
            William Schwartzman, Jiahao Jiang, J. S. Rao, Gavin R Schnitzler, Mohit Maurya, Lily Widdup, Mark E. Pepin, Helen Kang, C. Zheng, Rajat M. Gupta
          </td>
          <td>2026-01-02</td>
          <td>Circulation research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Prostate cancer (PCa) has long been viewed as an immunologically “cold” malignancy because immune checkpoint inhibitors (ICIs) show limited benefit in unselected patients, particularly after progression to metastatic castration-resistant PCa (mCRPC) or treatment-related neuroendocrine PCa (NEPC). Single-cell and spatial profiling now reveal immune heterogeneity across patients, between lesions, and along the path from localized disease to metastasis. Primary tumors form mosaics of immune-excluded glands, myeloid-suppressed stromal borders, and focal lymphocyte-rich niches with B-cell aggregates and tertiary lymphoid structures (TLS). TLS-high regions represent an actionable “hot minority” resembling inflamed, ICI-responsive cancers, supporting biomarker-guided neoadjuvant or focal immunotherapy. With dissemination, heterogeneity expands across sites; bone metastases become marrow immune organs dominated by suppressive macrophage/monocyte programs and dysfunctional T cells, often driven by the CCL2–CCR6 axis. Standard therapies remodel these ecosystems, creating inflammatory windows yet fostering adaptive resistance. Mechanistically, myeloid-driven, inflammation-coupled rewiring is central to escape: IL-8/CXCR2 signaling and therapy-induced senescence/SASP recruit and polarize suppressive myeloid cells, reinforcing T-cell exclusion and exhaustion. Variable HLA class I loss and hypoxic or metabolic “functional cold zones” add lesion-specific immune invisibility. Clinically, these insights motivate a heterogeneity-aware framework integrating genomic responder subsets with microenvironmental stratification. Barrier-matched strategies include T-cell redirection (PSMA/STEAP1 engagers, bispecifics, CAR-T) and combinations that heat or modulate myeloid cells. Treating immune heterogeneity as a clinical variable enables durable immunotherapy in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9ca8f1edda00999e41285037eb35603c7bbf58e" target='_blank'>
              Clinically oriented immune heterogeneity in prostate cancer: emerging targets and strategies
              </a>
            </td>
          <td>
            Mingwei Zhan, BinBin Zhao, Junjie Wu, Kai Li, Yibo Chen, Haote Chen, Lin Zhao, Jingyu Zhu
          </td>
          <td>2026-02-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intratumoral microbiota are now recognized as an integral component of the tumor microenvironment, affecting tumor initiation, metastatic potential, immune modulation, and treatment response. However, their extremely low biomass poses significant challenges for accurate detection, functional interpretation, and reproducibility, largely because the detection process is highly susceptible to environmental contamination. Standardization of analytical procedures has not yet been established; consequently, variability in sampling protocols, sequencing workflows, and bioinformatic pipelines further complicates cross-study comparisons and hampers the consolidation of robust evidence in this field. Recent advances in technology have begun to provide opportunities to overcome these barriers. Improved contamination-control strategies and more sophisticated decontamination algorithms have enhanced the reliability of microbial detection in low-biomass tissues. High-resolution approaches, such as single-cell RNA sequencing, spatial transcriptomics and optimized anaerobic cultivation, enable the sensitive identification, spatial localization, and mechanistic study of tumor associated microbes. Parallel developments in genome-resolved and enzyme-level analysis reveal microbial metabolic pathways that shape immune responses, drug resistance, and tumor progression. Organoid-based co-culture models further provide physiologically relevant platforms to dissect host-microbe-immune interactions and interpret microbiota-driven modulation of therapeutic responses. Integrating microbiome data with clinical and multi-omics profiles, assisted by artificial intelligence, is accelerating biomarker discovery and informing microbe-guided therapeutic strategies. Taken together, the standardization of research strategies, combined with the application of advanced detection technologies, is propelling the field beyond descriptive profiling toward mechanistic understanding and clinical translation, thereby unlocking the potential of intratumoral microbiota for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c30f5b43755a6fb82c98ee347498e9577b80237" target='_blank'>
              Emerging technologies and current challenges in intratumoral microbiota research
              </a>
            </td>
          <td>
            Zhiyue Wang, Tianqi Zhang, Yang Liu
          </td>
          <td>2026-01-02</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acne, the most prevalent inflammatory skin condition globally, lacks comprehensive understanding of its cellular and molecular mechanisms. This research integrates single-cell transcriptomics with spatial immunofluorescence to detail cellular composition and functional states in healthy skin, mild acne, and severe acne. Findings indicate that keratinocytes in mild acne secrete antimicrobial peptides and maintain barrier defense, but transition to abnormal differentiation and pathological keratinization in severe acne. Natural killer (NK) cells initially amplify inflammation and activate the immune response, but later become exhausted and apoptotic. This interaction creates a “dual immune-barrier imbalance,” driving acne progression. Functional enrichment and pseudo-time analyses demonstrated a strong association between disease severity and TNF, and chemokine signaling pathways, as well as NK cell-mediated cytotoxicity and Wnt/p53-apoptosis pathways. These findings were corroborated by immunofluorescence validation. This study introduces an innovative pathological model that establishes theoretical foundations for early diagnosis and precision interventions, while also outlining future therapeutic strategies centered on “immune-barrier co-regulation.” The findings provide essential insights into the pathogenesis of acne and open avenues for investigating novel intervention targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e83afe735e0121344e0a07c8a405ac15ec431c" target='_blank'>
              Single-cell transcriptomics and spatial validation reveal dysfunction of keratinocytes and NK cells driving acne development
              </a>
            </td>
          <td>
            Xinheng Wang, Changzhao Gao, Dandan Yang, Jiahui Pan, Rihua Jiang
          </td>
          <td>2025-12-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc6a6f856a7083dd2ac5b388edeaab80c7f5a72" target='_blank'>
              A spatially resolved atlas of gastric cancer characterises a lymphocyte-aggregated region.
              </a>
            </td>
          <td>
            Sen Gao, Shishang Qin, Dongfang Wang, Anqiang Wang, Linna Zhu, Yang Li, Qiang Shi, Hongtao Fan, Yufei Bo, Yunshan Zhong, Yuxuan Sun, Kun Dong, Liqin Fu, Ranran Gao, Yan Wu, Ye Liang, Lu-Yao Huang, Xueda Hu, Xianwen Ren, Z. Bu, Jiafu Ji, Zemin Zhang
          </td>
          <td>2026-01-27</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec774215b12f2be4ebfe5634529efaca54fcef6" target='_blank'>
              Toward Personalized Treatment of Urogenital Cancers: The Role of Patient-Derived Organoids.
              </a>
            </td>
          <td>
            S. Sagliocchi, Michele Musone, Stefano Chianese, A. G. Cicatiello, Silvia Del Mastro, Francesco Del Giudice, M. Dentice, F. Crocetto
          </td>
          <td>2026-01-08</td>
          <td>Oncology and therapy</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 Computational pathology has emerged as an attractive option for improving risk stratification in prostate cancer (PCa), but current approaches either lack interpretability or focus solely on tumor morphology, potentially limiting their utility. We aimed to identify an interpretable, immune microenvironment-derived computational pathology biomarker for PCa.



 We retrospectively identified two independent cohorts (Discovery and Validation) with localized PCa who underwent radical prostatectomy (RP) and had digitized H&E-stained whole-slides images (WSIs) and longitudinal outcomes data. We identified a third cohort from The Cancer Genome Atlas (TCGA) with localized PCa which was treated with RP and had WSIs, bulk RNA sequencing, and whole-exome sequencing. Immune cells were identified from WSIs using a publicly available deep learning model (CellViT), and spatially dense immune clusters were quantified using density-based spatial clustering of applications with noise (DBSCAN). CIBERSORTx was utilized for immune cell deconvolution and TRUST4 for immune receptor repertoire reconstruction from bulk RNA sequencing. Cox proportional hazard regression was used to examine associations between clinicopathologic features and the primary outcome of time to distant metastasis (DM), as well as secondary outcomes of biochemical recurrence (BCR) and overall survival (OS).



 Median follow-up periods for the Discovery (n=272) and Validation (n=218) Cohorts were 12.6 and 8.1 years, respectively. In the Discovery Cohort, median age was 63, 14% had Gleason 8-10 disease, 94% had pT2-T3a disease, and median prostate-specific antigen level at diagnosis was 6.2 ng/mL. Median immune cell proportion was 4.3% (interquartile range 3.0-5.9%), and median immune cluster density was 2.4 per 400 mm2 of tissue (interquartile range 0-7.1). Similar values were observed in the Validation Cohort. In both cohorts, immune cell proportion was not associated with BCR, DM, or OS (P>=0.09). In the Discovery Cohort, while increasing immune cluster was not associated with BCR (P>=0.10), it was independently associated with a decreased risk of DM for Gleason 8-10 (adjusted hazard ratio [AHR] 0.42, 95% confidence interval [CI] 0.19-0.93) but not Gleason 6-7 patients (AHR 1.26, 95% CI 0.77-2.05; Pint=0.020). Similar results were observed in the Validation Cohort, where increasing immune cluster was not associated with BCR (P>=0.10) but was associated with a lower risk of DM in Gleason 8-10 disease (AHR 0.60, 95% CI 0.37-0.98), though not Gleason 6-7 (AHR 1.19, 95% CI 0.74-1.91; Pint=0.043). In TCGA Cohort (n=326), immune cluster was not associated with somatic alterations. For Gleason 8-10 but not Gleason 6-7 disease, high-cluster samples were enriched for CD8+ T cells, activated memory CD4+ T cells, and Tregs (P<=0.037), as well as clonal T cell populations (P<=0.039).



 Our findings nominate immune spatial clustering as a novel, interpretable computational pathology biomarker and provide insights into the unique immune features of high-grade PCa.



 David D. Yang, Aya Abdelnaser, Alexander J. Haas, Jeremiah Wala, Alfred A. Barney, Eddy Saad, Jett P. Crowdis, Cora A. Ricker, Seifeldin Awad, Bora Gurel, Jihye Park, Martin T. King, Paul L. Nguyen, Toni K. Choueiri, David J. Einstein, Steven P. Balk, Alok K. Tewari, Johann S. de Bono6, Keyan Salari, Mary-Ellen Taplin, Chin-Lee Wu, Eliezer M. Van Allen. . Immune spatial organization predicts metastasis risk in aggressive localized prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aace1806fe7d01338e5e9857c054699a561213e" target='_blank'>
              Abstract PR001: Immune spatial organization predicts metastasis risk in aggressive localized prostate cancer
              </a>
            </td>
          <td>
            D. Yang, Aya Abdelnaser, Alexander J. Haas, Jeremiah Wala, Alfred A. Barney, Eddy Saad, Jett Crowdis, Cora A. Ricker, Seifeldin Awad, B. Gurel, Jihye Park, Martin T. King, P. L. Nguyen, T. Choueiri, David J Einstein, Steven P. Balk, A. Tewari, J. D. De Bono, Keyan Salari, M. Taplin, Chin-Lee Wu, E. V. Van Allen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e73b0cb2afeb220be22172b0dbec5fea7a5b5f" target='_blank'>
              Uncovering the Silent Invaders: Dormant Tumour Cells in the Brain Microenvironment.
              </a>
            </td>
          <td>
            Yufei Ze, Rongchen Dai, Yulong Zhang, Zhichao Xi, Hongxi Xu
          </td>
          <td>2026-02-18</td>
          <td>Critical reviews in oncology/hematology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6c25b39df997323e228b3c6d265efc22e9a7958" target='_blank'>
              Multi-omics characterization of IDH-mutant astrocytoma-derived cell lines reveals NOTCH-regulated plastic quiescent astrocyte-like state
              </a>
            </td>
          <td>
            L.Garcia, C.Granotier-Beckers, D.Pineau, D.Sika, S.Hideg, C.Labadie, K.Aguilar Cázarez, J.Kundnani, L.Stuani, L.Gauthier, F.Boussin, M-A.Mouthon, S.Urbach, K.El Koulali, M.Séveno, C.Ripoll, K.Daddi, M.Verreault, A.Idbaih, S.O’Connor, C.Plaisier, L.Zhang, M.Zheng, V.Rigau, H.Duffau, J. Hugnot
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cb53da812dbac416c29e7f7225ad2ab6decfdb9" target='_blank'>
              Exploring the Space of Tumor Phylogenies Consistent with Single-Cell Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Samin Rahman Khan, P. Sashittal
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Triple Negative Breast Cancer (TNBC) is an aggressive subtype of breast cancer with limited treatment options and poor prognosis. Coupled with this, TNBC shows a high rate of metastasis which is known to be aided by tumour vasculature. Endothelial cells that form the lining of the tumour vasculature exhibit distinct genotype and phenotype differences compared with normal tissue vasculature. However, little is known about endothelial signatures that drive metastasis from a primary tumour, particularly in the context of immune infiltration. In this study, we utilized GeoMX Digital Spatial Profiling to investigate spatial proteomics differences in endothelial cells between primary and secondary sites of TNBC. By segmenting tissues using epithelial (PanCK), immune (CD45), and endothelial (CD31) markers, we analysed the microvasculature for a panel of 79 target proteins. In paired primary and secondary TNBC tissues, we identified significant downregulation of fibronectin (− log2(Fold-Change) = − 1.7, p < 0.001) in secondary sites. Specifically in epithelial regions, S100B was found to be downregulated in secondary microvasculature when compared to primary tumours. Additionally, metastasis-free primary tissues exhibited upregulated expression of S100B when compared to primary tissues that metastasized. Our study highlights the potential contributions of microvasculature to metastatic progression in TNBC, presenting new opportunities to explore them as potential biomarkers of TNBC metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9355568219b2a2ea616006baa27f93ab8c621b2" target='_blank'>
              Digital spatial profiling identifies features of primary and locoregional metastatic vasculature in triple negative breast cancer
              </a>
            </td>
          <td>
            Akhilandeshwari Ravichandran, Kyle Upton, Shiva Taheri, Cheng Liu, K. Ferguson, Mark N. Adams, Laura J. Bray
          </td>
          <td>2026-01-19</td>
          <td>Clinical & Experimental Metastasis</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC), characterized by its distinctive hypoxic and immunosuppressive tumor microenvironment (TME), demonstrates suboptimal responses to current immunotherapeutic interventions. Tertiary lymphoid structures (TLS), defined as ectopically organized immune cell aggregates that develop in non-lymphoid tissues, function as “plastic immune organs” and exhibit considerable promise as both prognostic indicators and therapeutic targets. Notably, TLS in RCC manifest significant heterogeneity, with specific subsets associated with favorable clinical outcomes while others correlate with adverse prognosis. This review systematically examines the cellular composition, formation, classification criteria, and evaluation methods of TLS in RCC, with particular emphasis on the relationship between TLS heterogeneity and differential prognostic implications. We further explore potential regulatory mechanisms underlying these divergent clinical outcomes and provide a comprehensive synthesis of current TLS-targeted therapeutic strategies, including recent clinical advancements. Finally, we delineate the prevailing challenges in TLS research and propose future directions. This work provides a theoretical foundation and research framework for future patient outcome-oriented prospective studies and clinical trials, while also offering insights for the development of TLS-related precision immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc9e1754e10a1e3d3eef39cbe13c39fde7f3e5b" target='_blank'>
              Tertiary lymphoid structures in renal cell carcinoma: from heterogeneity dissection to translational precision immunotherapy
              </a>
            </td>
          <td>
            Zhexian Li, Sensen Ruan, Yang Yu, Hao Xiang, Wenjie Mi
          </td>
          <td>2026-01-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) exhibits heterogeneous treatment responses, yet molecular subtypes based on predefined biological pathways show limited prognostic value. We introduce tumor-specific total mRNA expression (TmS), a pathway-agnostic deconvolution metric derived from matched RNA/DNA sequencing, as a robust stratification tool. Analyzing 575 TNBC patients across Western and East Asian populations, TmS outperforms established subtypes in predicting chemotherapy outcomes, stratifying patients into high TmS with favorable prognosis and low TmS with poor prognosis. Stromal enrichment with immune exclusion emerges as a universal feature of chemotherapy-resistant low-TmS tumors across all cohorts. Population-specific features distinguish Asian cohorts: high-TmS tumors exhibit cell cycle-driven proliferation programs, and low-TmS tumors display immune dysfunction with memory B cell enrichment and divergent RAS/mitogen-activated protein kinase (MAPK) activation, compared to Western populations. Despite these differences, extracellular matrix organization represents a conserved therapeutic vulnerability in treatment-resistant low-TmS patients. TmS provides a unifying framework for dissecting TNBC heterogeneity and enabling precision therapy across diverse populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe40fd8043824e69aea7a200f58253ac0146fb9" target='_blank'>
              Tumor microenvironment transcriptional activity enables robust stratification of chemotherapy response in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Yaoyi Dai, Xiaoxi Pan, Shuai Guo, Shuangxi Ji, S. Cao, Matthew D. Montierth, Yujie Jiang, Jeffrey T. Chang, Leming Shi, Shabnam Shalapour, Gloria V Echeverria, Lucy Yates, Johan Staaf, B. Lim, Yinyin Yuan, Wenyi Wang
          </td>
          <td>2026-02-01</td>
          <td>Cell reports. Medicine</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Breast cancer, a highly heterogeneous malignancy and a leading cause of cancer-related mortality among women worldwide, is profoundly shaped in its initiation, progression, and therapeutic response by the tumor immune microenvironment (TIME). This review consolidates recent advances in deciphering the cellular, molecular, and metabolic complexity of breast cancer TIME and highlights mechanisms of immune suppression that impede durable treatment efficacy. We critically appraise current and emerging immunotherapeutic approaches, with a focus on strategies that aim to transform immunologically “cold” tumors into “hot,” immune-responsive phenotypes. Novel directions, including metabolic modulation, stromal reprogramming, and precision combination therapies, are discussed in the context of overcoming primary and acquired resistance. We highlight emerging biomarker strategies that integrate spatial transcriptomics to map immune exclusion zones, along with liquid biopsy monitoring of exosomal PD-L1 and circulating tumor DNA, to enable real-time adaptation of combination immunotherapy regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cef7103ba9742723c71c9b5ea5ad07b695982d0" target='_blank'>
              Reprogramming the immunosuppressive breast cancer microenvironment: integrating cellular, metabolic, and stromal targets for rational immunotherapy
              </a>
            </td>
          <td>
            Chang Ma, Hoon Koon Teoh, Yang Zhao, Yifan Wang, , Yan Liu, , Hooi Tin Ong
          </td>
          <td>2026-02-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/139541eb9ed0a662a3fb81fe085f09a1d9bc460d" target='_blank'>
              Recent advances in nanomaterial-based precision medicine for orthotopic tumor therapy
              </a>
            </td>
          <td>
            Min Luo, Fu-kun Zhao, C. Ouyang, Yuan-min Wang, Yong Luo, Jiang Bian
          </td>
          <td>2026-01-24</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Bladder cancer (BCA) shows significant prognostic differences between non-muscle-invasive (NMIBC) and muscle-invasive (MIBC) forms. While NMIBC frequently recurs and can progress to invasive disease, reliable biomarkers to monitor this transition are lacking. Extracellular matrix (ECM) remodeling is a critical factor influencing tumor aggressiveness, yet the key regulators of ECM changes across BCA stages remain unclear. In this study, we investigate the role of COL1A2 in ECM-related tumor biology and its potential as a prognostic biomarker for BCA progression. Methods We utilized a multi-step bioinformatics pipeline, analyzing RNA-seq data from TCGA and GEO datasets to identify molecular differences between NMIBC and MIBC. Prognostic markers were prioritized via differential expression analysis, Cox regression, and Kaplan–Meier survival analysis. The regulatory network was explored using protein-protein interaction analysis, and ECM-related activity was quantified through ssGSEA. Cell-type-specific insights were gained through single-cell RNA-seq analysis, and intercellular communication was deciphered using CellChat. Functional validation was performed through in vitro knockdown experiments in BCA cell lines. Results COL1A2 emerged as a key prognostic ECM-related gene associated with MIBC. Single-cell RNA-seq analysis revealed that COL1A2 and ECM components were predominantly enriched in matrix cancer-associated fibroblasts (CAFs), with PTK2 (FAK, focal adhesion kinase) upregulated in epithelial cells undergoing epithelial-mesenchymal transition (EMT). CellChat analysis uncovered a dominant COL1A2-mediated signaling axis from matrix CAFs to EMT epithelial cells via COL1A1/2–SDC1/4 ligand-receptor interactions. Functional assays confirmed that COL1A2 knockdown significantly impaired MIBC cell invasion and migration by suppressing ECM remodeling and EMT. Conclusion Our results suggest that the COL1A2–ECM–FAK signaling axis plays a critical role in MIBC progression, and COL1A2 could serve as a potential biomarker and therapeutic target for muscle-invasive bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda2a0be0d617f715d6d80984a257431f3977b47" target='_blank'>
              Integrative bulk and single-cell transcriptomic analysis reveals COL1A2-driven ECM remodeling and focal adhesion signaling associated with the transition from non-muscle-invasive to muscle-invasive bladder cancer
              </a>
            </td>
          <td>
            Menglu Li, Xinwei Liu, Yuhui Xue, Yichen Lu, Zhiqiang Chen, Yuwei Zhang, Weiguo Chen, Shan Zhao, Ke Wang, Ninghan Feng
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Oncology</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcde693a87853a15bccfb4a948cfd12597ab0e39" target='_blank'>
              Cancer Driver Gene Discovery: A Patient-Level Statistical Framework
              </a>
            </td>
          <td>
            Farideh Bahari, Hesam Montazeri
          </td>
          <td>2026-02-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) represents one of the most prevalent malignancies worldwide and poses a critical public health challenge due to difficulties in early diagnosis, therapy resistance, and high mortality rates. The complex tumor microenvironment (TME) of HCC plays a pivotal role in tumor progression, immune evasion, metastasis, and treatment resistance. Single-cell sequencing (scRNA-seq) has emerged as a revolutionary tool for resolving the intricacies and cellular heterogeneity of the TME, with its applications in advancing therapeutic research attracting considerable attention. As the primary battleground for antitumor immune responses, the HCC tumor TME warrants comprehensive analysis of immune cell subsets at distinct developmental and functional states to elucidate the complexity of tumor immunology. This review synthesizes extensive research on TME immune cellular subpopulations, in order to summarize mainstream classifications of immune subsets at single-cell resolution and analyze their functional significance and therapeutic value through biomarker gene profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04e89b37b343268e54c54a4f61fb15cb643f6a03" target='_blank'>
              Single-cell dissection of hepatocellular carcinoma immunity: from heterogeneous subtypes to precision therapeutics
              </a>
            </td>
          <td>
            Yuni Liang, Hemeng Wu, Yunshan Qiu, Qiulian Mo, Peipei Chen, Mingfen Li, Hongsheng Lin
          </td>
          <td>2026-02-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Epigenetic reprogramming is a major contributor to therapy resistance in prostate cancer (PCa). Indeed, chromatin accessibility profiling classified PCa into distinct subtypes, predictive of clinical outcome. Histone modifications influence nucleosome positioning, ultimately dictating chromatin compactness. The histone methyltransferase G9a, frequently upregulated in advanced PCa, catalyzes a repressive histone modification (di-methylation of histone H3 lysine 9) that signals for gene transcriptional repression. CM-272 is a reversible and highly selective inhibitor against G9a activity, and we have previously shown in cell line models of PCa that it depicted strong anti-tumoral effects. However, fundamental aspects of altered transcriptomic programs remain unexplored. Here, we validated, in vivo, the CM-272 effects in advanced PCa, and explored the molecular mechanisms underscoring G9a inhibition-mediated effects. Specifically, the PCa patient-derived xenograft model BM18 was used to assess the mode of action of CM-272 in vivo. Moreover, RNA sequencing of tumor tissues was performed to further explore the underlying gene expression signature. Finally, chromatin immunoprecipitation followed by qPCR was used to investigate the underlying functional consequence of epigenetic targeting. Our results showed that CM-272 significantly delayed BM18 tumor growth, specifically impairing tumor cell G9a activity and proliferation. Transcriptomic profiling revealed 119 significantly upregulated and 99 downregulated genes in response to G9a inhibition. Notably, by integrating our gene expression dataset with published chromatin accessibility data, an increase in the androgen receptor (AR) signature and a concomitant decrease in the neuroendocrine (NE) features were observed in CM-272-treated samples. Additionally, pathway enrichment and master regulator analysis pinpointed both the androgen receptor and TP53 networks as significantly upregulated in treated tissues. In line with the reduction of the NE score, the MYC/MYCN were the most significantly downregulated transcription factors. Moreover, under the androgen pathway, upregulation of AZGP1, STK39, and DPP4 was observed. At the promoter level, a significant reduction in H3K9me2 levels was detected in treated samples, allowing, in turn, an increase in AR binding. Additionally, upon treatment, higher p53 binding was detected at the promoter of the p53-repressed genes AURKA, CDC25B, and CD44, whose expression was downregulated. Overall, targeting G9a with CM-272 might hold promising therapeutic potential for the management of advanced disease by modulating relevant tumoral transcriptional programs associated with core vulnerabilities in PCa.



 Filipa Moreira-Silva, Mark Zimmerli, Panagiotis Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio. G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5911786a7c980e0c86fa5660d05b013ec96ea309" target='_blank'>
              Abstract A045: G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay
              </a>
            </td>
          <td>
            Filipa Moreira-Silva, Mark Zimmerli, P. Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a prevalent and aggressive malignancy with a persistently high mortality rate, largely attributable to therapy resistance and tumor recurrence. This review comprehensively explores the critical interplay between epigenetic dysregulation and the tumor microenvironment (TME) in driving OSCC progression. We detail how key epigenetic mechanisms, including DNA methylation, histone modifications, and non-coding RNAs (ncRNAs), intrinsically transform cancer cells and actively orchestrate pro-tumorigenic TME. These alterations substantially contribute to resistance against conventional therapies. Furthermore, we discuss the therapeutic potential of targeting these pathways using epigenetic drugs (epi-drugs), such as DNA methyltransferase (DNMT) inhibitors and histone deacetylase (HDAC) inhibitors, as well as engineered extracellular vesicles (EVs). The primary objective of this review is to synthesize current knowledge on the epigenetic-TME axis, thereby providing a mechanistic foundation for developing novel therapeutic strategies. We emphasize that rational combinations of epigenetic-targeting agents with conventional treatments or immunotherapy hold significant promise for overcoming drug resistance and improving clinical outcomes in OSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96aaa3d5aaab2d3886a4c338f86161bbd9cbefef" target='_blank'>
              Dissecting the role of epigenetic regulation in oral squamous cell carcinoma microenvironment: mechanisms and therapeutics
              </a>
            </td>
          <td>
            Xuechao Li, Yifei Ren, S. Pei, Kai Zhao, Guanyu Chen, Zhenglin He
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer is known to be a disease of altered cellular signaling; however, the relationship between mutation-specific changes to signal transduction and the phenotypic consequences produced remains poorly understood. Here, we investigate two common breast cancer driver mutations, the PIK3CAH1047R mutation and the ErbB2 amplification, both of which activate the PI3K-Akt pathway but paradoxically drive distinct cellular outcomes. Indeed, in nontransformed mammary epithelial cells, PI3KH1047R expression induced features of epithelial-mesenchymal transition (EMT), while ErbB2amp cells exhibited a hyperproliferative phenotype. Characterization of PI3K axis signaling revealed that ErbB2amp cells display prolonged, stimulus-dependent PI3K activation, whereas PI3KH1047R cells show constitutive, ligand-independent signaling. To test whether these distinct dynamics contribute to the phenotypic responses, we employed an iLID-based optogenetic system that enables precise, tunable control of endogenous PI3K activity. Using this tool to mimic the mutation-specific dynamics in MCF10A mammary epithelial cells, we found that PI3K signaling patterns alone were sufficient to reproduce key features of the PIK3CA H1047R-associated EMT phenotype but not the ErbB2-associated proliferative phenotype. These findings suggest that the temporal encoding of pathway activity, not merely its magnitude, can drive some phenotypic changes in oncogenic progression, explain how distinct mutations within a common signaling pathway can produce divergent cellular phenotypes, and provide a workflow for interrogating the functional consequences of changes in signaling dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51c6978b325cbac3a14efcc2bf1ac41d165bf7ce" target='_blank'>
              Oncogenic Alterations in PI3K Signaling Emulated Optogenetically Recapitulate Some Phenotypic Changes in Mammary Epithelia.
              </a>
            </td>
          <td>
            Keith A. Gagnon, Veronica W Hui, Terry Ching, Amy E. Stoddard, E. Koh, J. Eyckmans, Ahmad S. Khalil, Christopher S. Chen
          </td>
          <td>2026-01-19</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Small cell lung cancer (SCLC) is a subtype of lung cancer that is aggressive, progresses rapidly, and is prone to recurrence. The biological composition of SCLC is still under investigation. This study aims to characterize the intratumoral heterogeneity and immunosuppressive tumor microenvironment of SCLC using single-cell RNA sequencing (scRNA-seq), and to identify and validate the key genes BEX1 and MAP1b as potential therapeutic targets. Methods To comprehend the heterogeneity of SCLC and the tumor microenvironment, we used scRNA-seq to analyze the primary tumor and adjacent noncancerous tissue from a patient. The findings were tested with cell experiments. Results We observed that SCLC was mainly composed of neuroendocrine epithelial cells and displayed the immune-related cell failure state. The corresponding antitumor immune pathway activities were also downregulated, and the tumor microenvironment eventually showed immunosuppression. BEX1 and MAP1b were upregulated in most cell subtypes, which are verified by immunohistochemistry. In addition, downregulation of BEX1 in NCI-H209 and MAP1b in NCI-H82 significantly inhibited cell proliferation and migration, while increasing apoptosis. Based on preliminary data, BEX1 and MAP1b have been identified as promising candidates for the early diagnosis and therapy of SCLC; however, their clinical utility requires confirmation in subsequent studies. To investigate the heterogeneity and interaction among different cell types, we also constructed an intercellular communication network. Conclusions Our knowledge of the basic traits of SCLC is improved by this highly accurate single-cell study, which also offers fresh suggestions for potential future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b82d107b058cff3073eee2fb1f4d3d5227463da5" target='_blank'>
              Single-patient single-cell RNA sequencing reveals neuroendocrine predominance and immunosuppression in small-cell lung cancer
              </a>
            </td>
          <td>
            Yiru Wang, Siyi He, Rensheng Wang, Weimei Huang
          </td>
          <td>2026-01-01</td>
          <td>Translational Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 T cell-mediated immunotherapies are extensively investigated across a range of malignancies, demonstrating dramatic therapeutic response in certain entities. In contrast, clinical activity in pediatric cancers thus far is limited. Particularly cancer entities, characterized by an immune excluded or immune deserted tumor microenvironment (TME), also referred to as immunologically “cold”, a prevalent phenotype in pediatric solid cancers, are less accessible for T cell-mediated immunotherapies. Reprogramming the TME towards an inflammatory immune phenotype represents a novel and potentially highly efficient therapeutic strategy to overcome this obstacle and significantly improve the efficacy of T cell-mediated immunotherapies. In this study, we first screened a comprehensive multi-omics data set of relapsed pediatric cancers on signatures and targets associated with an immunologically “cold” TME, characterized among others by low T cell infiltrates, low INFg-signature and abundance of M2-like polarized macrophages. We identified high B7-H3 expression as a functionally linked target structure. To reprogram the TME of pediatric solid cancers by interleukin (IL) 2 and IL12 signals, we developed a novel fusion protein consisting of B7-H3 binding, as well as IL2-receptor and IL12-receptor agonistic domains. This molecule allows targeted delivery and accumulation of the immunostimulatory payload in the TME, limiting toxicities observed after IL2 and IL12 therapies. In vitro, the fusion protein induced NK activation, IFNγ secretion, and M2-to-M1 macrophage polarization. For in vivo evaluation, we utilized orthotopic xenograft and a fully immune competent syngeneic mouse model of rhabdomyosarcoma, resistant to chimeric antigen receptor (CAR) T cell therapy. In the xenograft model, systemic application of the fusion protein prior to CAR-T application results in complete eradication of established tumors. Functional analyses by high-content imaging, reveal in situ TME-reprogramming characterized by M2-like to M1-like macrophage conversion and significant infiltration of functionally active T and CAR-T cells. Strikingly, systemic application of the fusion protein as a single agent therapy is sufficient to induce sustained immunological clearance of established tumors in immunocompetent mice with limited toxicities. Detailed cellular and molecular mechanisms are studied by longitudinal single nuclear RNA sequencing and high content imaging. Here, we demonstrate a novel class of TME reprogramming fusion proteins, efficiently delivering an immunostimulatory payload in the TME of (pediatric) solid cancer after systemic application. By inducing an inflammatory cascade, these compounds “heat up” immunologically “cold” tumors, enabling T cell infiltration and antitumoral activity. Our findings have important clinical implications possibly broadening the group of cancer patients benefitting from T cell-mediated immunotherapies. Clinical translation of the compound is ongoing with a phase I/II trial in preparation.


 Simon Krost, Moustafa Moustafa-Oglou, Marta Emperador Melero, Roland Imle, Tom Fischer, Anke King, Lennart Schlager, Sophia Scheuermann, Daniel Blösel, Robert Autry, Stefan M. Pfister, Olaf Witt, Kristian Pajtler, Ana Banito, Karin Schilbach, Stephen D. Gillies, Christian M. Seitz. Reprogramming the Tumor Microenvironment Enables T Cell-Mediated Immunotherapies in Pediatric Solid Cancers [abstract]. In: Proceedings of the AACR Immuno-Oncology Conference (AACR IO): Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2026 Feb 18-21; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2026;14(2 Suppl):Abstract nr B065.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84c2040d1891820980ca9a29d1d88219c6bfc4ed" target='_blank'>
              Abstract B065: Reprogramming the Tumor Microenvironment Enables T Cell-Mediated Immunotherapies in Pediatric Solid Cancers
              </a>
            </td>
          <td>
            Simon Krost, Moustafa Moustafa-Oglou, Marta Emperador Melero, R. Imle, T. Fischer, Anke King, L. Schlager, Sophia Scheuermann, Daniel Blösel, R. Autry, Stefan M. Pfister, O. Witt, K. Pajtler, A. Banito, Karin Schilbach, S. D. Gillies, Christian Seitz
          </td>
          <td>2026-02-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Predicting recurrences in patients with stage II colon cancer remains a clinical challenge. This study aimed to uncover connections between gene expression, clinicopathological characteristics, and recurrence in patients with stage II colon cancer. Gene expression profiling was conducted on primary tumors from 94 well-characterized patients with stage II colon cancer using the PanCancer IO 360™ panel from NanoString Technologies, which includes 770 mRNAs related to tumor progression and the tumor microenvironment. Unsupervised hierarchical clustering, differential gene expression (DEG) and survival analyses were used to describe the relationship between gene expression, clinicopathological characteristics, and recurrence. Unsupervised hierarchical clustering revealed distinct gene expression patterns in pT4 tumors, particularly in pathways involving immune cell localization and myeloid cell activity. DEG analysis identified 156 differentially expressed genes in pT4 versus pT3 tumors, including the important chemokines, CCL2 and CXCL8, and the cytokine LIF. In addition, 35 upregulated genes associated with migration and extracellular processes were identified in patients with recurrence. Finally, SLC2A1 and VEGFA emerged as independent prognostic markers for time to recurrence and overall survival. In conclusion, this study unveiled distinct gene expression patterns in advanced pT4 tumors, and identified independent prognostic biomarkers that may prove useful in stage II colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d0c607b428f5aa182f74d71dfa8866e309cea23" target='_blank'>
              Transcriptional profiling of the tumor microenvironment of recurrent and non-recurrent stage II colon cancer
              </a>
            </td>
          <td>
            Ulrik Korsgaard, M. P. Kristensen, S. K. Frifeldt, J. Lindebjerg, T. Hansen, Henrik Hager, L. S. Kristensen
          </td>
          <td>2026-02-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer with poor prognosis and a high likelihood of recurrence. Residual disease after therapy is a key predictor of recurrence, often driven by intra-tumoral metabolic heterogeneity. Accumulating evidence indicates that tumors are able to shift between glycolysis and oxidative metabolism and alter nutrient preferences to sustain growth and resist therapy. We have developed a novel in vivo microscope that enables near-simultaneous measurements of fluorescent metabolic surrogates of glucose, fatty acids and oxidative phosphorylation through a combination of spectral separation and sequential delivery schemes. Widefield imaging with uniform illumination across the entire tumor landscape (5 mm x 5 mm) informs on the spatial distribution of these metabolic probes. We used this technology to investigate metabolic heterogeneity of a murine model of TNBC (4T1 tumor line) and normal mammary tissues which have distinctly different metabolic pathways. Mammary tissues relied primarily on oxidative metabolism and showed high levels of glucose and fatty acid uptake across the entire imaging area reflecting a single metabolic phenotype. Though tumors were predominantly glycolytic, they displayed a heterogeneous distribution of nutrient preferences with regions dominated by either fatty acid uptake, glucose uptake or both. Taken together, this work highlights the importance of not only capturing multiple metabolic endpoints but also investigating their spatial relationships to understand heterogeneity in key substrates and metabolic pathways for energy production in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d63dbaa8c689fef7cf1793ea43fde4e36282d40" target='_blank'>
              In vivo imaging of metabolic heterogeneity across three endpoints relevant to aggressive breast cancer
              </a>
            </td>
          <td>
            Victoria W. D’Agostino, Michelle Kwan, Adelle Yong, Kira Grossman, Enakshi D Sunassee, Megan C. Madonna, M. Hirschey, Gregory M Palmer, Nimmi Ramanujam
          </td>
          <td>2026-02-10</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a highly aggressive malignancy with a dismal prognosis, and pronounced interpatient heterogeneity severely limits the efficacy of systemic therapies, underscoring the need for rapid and accurate functional platforms to guide individualized drug selection. Here, we develop a clinically oriented, patient-derived, 3D bioprinted in vitro model for personalized drug sensitivity assessment in ICC. Using a compositionally defined and cost-effective GelMA/HAMA composite hydrogel, we reconstruct a tumor microenvironment that supports rapid self-organization and sustained viability of primary ICC cells. Histological analyses, marker expression profiling, and bright-field imaging demonstrate close similarity to matched patient tumor tissues. Genomic and transcriptomic fidelity are further confirmed by whole-exome and RNA sequencing, revealing preserved driver mutations and transcriptional programs. Drug sensitivity testing was performed on tumor samples from 21 ICC patients using clinically relevant agents. Notably, in patients receiving neoadjuvant therapy, in vitro drug responses were fully consistent with clinical outcomes. Longitudinal follow-up further showed that recurrence occurred exclusively in patients who did not receive the predicted sensitive therapies. Importantly, clinically actionable drug response profiles were generated within 10 days. Collectively, this platform provides a rapid, reproducible, and patient-specific functional drug testing strategy with strong potential for clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/611eb6947060d4dce96e52d121b1b9de9e5572e6" target='_blank'>
              Patient-Derived 3D-Bioprinted Intrahepatic Cholangiocarcinoma Models Recapitulate Tumor Autologous Traits and Predict Personalized Adjuvant Therapy.
              </a>
            </td>
          <td>
            Yuce Lu, Liwei Du, Minghao Sun, Kai Zhang, M. Pang, Shangze Jiang, Jiaxun Dong, Xiyue Liu, Bao Jin, Fu Xu, Hang Sun, Jiangang Zhang, Huiyu Yang, Xiaobo Yang, Xin Lu, Yiyao Xu, Haitao Zhao, Shunda Du, Xinting Sang, Yongchang Zheng, Lei Zhang, Xueshuai Wan, Huayu Yang, Yilei Mao
          </td>
          <td>2026-02-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="PURPOSE
Immune checkpoint blockade (ICB) benefits only a subset of sarcoma patients. Biomarkers of response and resistance are needed to help guide patient selection.


PATIENTS AND METHODS
We analyzed peripheral blood and tumor samples from sarcoma patients treated on five ICB-based clinical trials. Baseline peripheral blood mononuclear cells (PBMCs) underwent 11-color flow cytometry to define T cell immunotypes. Baseline tumor tissue underwent RNA sequencing to classify tumors into four tumor microenvironment (TME) subtypes using consensus clustering of 29 functional gene expression signatures. Associations between immune features and clinical outcomes were assessed. A deep-learning model was applied to baseline hematoxylin and eosin (H&E) slides to detect and quantify lymphoid aggregates in patients with available RNA sequencing.


RESULTS
Among 178 patients with PBMCs available for analysis, a proliferative (PRO) circulating T cell immunotype was associated with inferior overall survival (OS) compared with LAG- or LAG+ immunotypes. RNA sequencing from 67 tumors identified an immune-enriched/non-fibrotic TME subtype associated with higher response rate, longer progression-free survival, and longer OS compared to immune-enriched/fibrotic, immune-depleted, and fibrotic subtypes. Automated analysis of 48 baseline H&E slides identified lymphoid aggregates in five tumors; four were classified as immune-enriched and two responded to ICB.


CONCLUSIONS
Sarcoma patients with a PRO circulating T cell immunotype had inferior outcomes to ICB, while those with an immune-enriched/non-fibrotic TME had superior outcomes. Automated analysis of H&E slides showed promise in identifying patients with an immune-enriched TME. These findings support utilization of a multimodal approach toward identifying predictors of response to immunotherapy in sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20da94b38d82fe9dba8a8ce6c91ac335c7b826c4" target='_blank'>
              Association of Circulating T Cell and Tumor Microenvironment Profiles with Immune Checkpoint Blockade Outcome in Sarcoma.
              </a>
            </td>
          <td>
            E. Rosenbaum, Fiona D. Ehrich, M. Yosofvand, M. Bradic, Jasme Lee, Mathew Adamow, S. Movva, C. Kelly, V. Avutu, Lauren B Banks, J. Chan, P. Chi, M. Dickson, M. Gounder, M. Keohan, Robert G. Maki, Damon R. Reed, Paige Fuentes, Paige Collins, R. Desir, Allison Reiner, Oleg Baranov, Konstantin Chernyshov, N. Kotlov, A. Subramanian, E. Moding, L. Qin, Phillip Wong, W. Tap, C. Antonescu, K. Panageas, Ronglai Shen, S. D’Angelo
          </td>
          <td>2026-02-12</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>139</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) studies have uncovered distinct cancer-associated fibroblast (CAF) populations. While useful as a biological framework, no studies have conclusively defined CAF subtypes with clinical significance. We define restraining (rest) and promoting (pro) CAFs in patient samples that are both prognostic and predictive of therapy response in multiple tumor types. We uncover distinct clinical and spatial interactions between pro- and restCAF subtypes and basal-like and classical tumor subtypes that support tumor-stroma crosstalk. Finally, we find striking differences in the immune contexture of pro- and restCAF tumors where restCAF-dominant tumors are more responsive to immune checkpoint inhibition and proCAF-dominant tumors are more responsive to myeloid inhibition in clinical trials. This work defines CAF subtypes that are clinically robust, prognostic, and predictive of immunotherapy response and provides a single-sample classifier, determination of pro- and restCAF subtypes (DeCAF), which is clinically actionable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917ff4574739090c421afaf6c3e5fea5ef6858c6" target='_blank'>
              DeCAF defines clinical fibroblast subtypes and multidimensional tumor-stroma crosstalk shaping prognosis and immunotherapy response.
              </a>
            </td>
          <td>
            Xianlu Laura Peng, Ian C McCabe, Elena V Kharitonova, Yi Xu, , C. Luan, Joseph F Kearney, Nancy P. Kren, Arthi Hariharan, Jaewon J Lee, J. Su, Priscilla S Chan, Ashley B Morrison, Michelle E. LaBella, Silvia G. Herera Loeza, Ashley Cliff, Hong Jin Kim, Y. Pylayeva-Gupta, Susan Tsai, B. Belt, R. Panni, David Linehan, Shawn M. Gomez, Jeffrey S. Damrauer, Alina C Iuga, William Y. Kim, Naim U Rashid, J. J. Yeh
          </td>
          <td>2026-02-01</td>
          <td>Cell reports. Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Tumorigenesis is a complex and dynamic process in which the tumor microenvironment (TME) plays a central role. In solid tumors, the TME contributes to key mechanisms of tumor progression, including metastasis, immune evasion, and resistance to therapies. One major challenge in preclinical cancer research is the development of reliable three-dimensional (3D) in vitro models, which more accurately replicate the in vivo tumor architecture and microenvironmental conditions, such as hypoxia and extracellular matrix (ECM) organization. However, reproducing functional vascular networks and neo-angiogenesis within these models remains a key challenge. In this study, an advanced 3D tumor model, referred to as angiotumoroids, was developed by co-culturing primary murine breast tumor cells (PTCs) with species-specific adipose-derived microvascular fragments (MVFs). Angiotumoroids were characterized using scanning electron microscopy and immunostaining, and angiogenesis was evaluated through collagen gel sprouting assays. High-resolution proteomic profiling was conducted, focusing on signatures associated with angiogenesis, extracellular matrix (ECM) composition, and tissue remodeling. Additionally, the response and internalization to anticancer drug treatments were evaluated. MVFs are successfully integrated in angiotumoroids, resulting in the formation of vasculature-like structures and demonstrating robust structural organization with dynamic modulation of matrix metalloproteinase 9. Formation of neovasculature was visualized through sprouting and branching, driven by both direct PTC–MVF interactions and PTC-conditioned media, highlighting the roles of juxtacrine and paracrine signaling. Proteomic profiling revealed distinct expression patterns associated with angiogenesis, ECM components (including collagen types I and IV), and active ECM remodeling with elevated MMP expression. Additionally, angiotumoroids showed increased expression of ATP-binding cassette (ABC) transporters, particularly ABCB1 (P-glycoprotein), suggesting potential mechanisms of drug efflux. Functionally, angiotumoroids demonstrated reduced sensitivity to doxorubicin compared to PTC spheroids, maintaining structural integrity and higher cell viability post-treatment. Time-course analysis revealed preferential doxorubicin accumulation in MVF-enriched regions, as confirmed by colocalization with CD31, indicating a spatially regulated distribution of the drug mediated by the vascular compartment. Collectively, these findings establish angiotumoroids as a robust and physiologically relevant in vitro model for studying tumor vascularization, ECM dynamics, and therapeutic response. This platform holds significant promise for predictive cancer research and preclinical drug screening, bridging the gap between traditional in vitro systems and in vivo models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba884bb3dbf2cdd1df613387c3fda90c3178112" target='_blank'>
              A vascularized three-dimensional model integrating primary breast tumor cells and microvascular fragments: mimicking the tumor microenvironment involved in chemoresistance
              </a>
            </td>
          <td>
            Alessandra Lo Cicero, Francesco La Monica, Gabriele Lo Buglio, S. Campora, Federico Gangemi, Paolo Cinà, Monica Salamone, Margot Lo Pinto, S. D. Scilabra, G. Ghersi
          </td>
          <td>2026-01-08</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract Background Primary pulmonary sarcomas (PPS) and pulmonary sarcomatoid carcinoma (PSC) are rare and aggressive diseases that pose significant diagnostic challenges, requiring extensive sampling and comprehensive evaluation. To date, the single‐cell characteristics and distinctions between these two conditions have not been thoroughly investigated. Methods In this study, we employed single‐nucleus RNA sequencing (snRNA‐seq) to characterise the cellular heterogeneity of PSC and PPS. Our analysis included 20 PSC samples, seven PPS samples and two non‐malignant control samples obtained from adjacent normal tissue. Results Our results revealed that the majority of cells in PSC were of epithelial origin, while fibroblasts predominated in PPS. Specifically, AT2 cells, a major source of epithelial cells in PSC, underwent malignant transformation primarily through epithelial–mesenchymal transition, suggesting AT2 cells may serve as the origin of PSC. High Mobility Group AT‐Hook 2 (HMGA2) expression was elevated in malignant AT2 cells of PSC and correlated with an unfavourable prognosis. Moreover, MET‐mutated patients have a significantly higher expression level of HMGA2 (p < .001). In PPS, fibroblasts constituted the majority, only lipofibroblasts exhibited malignant features. A direct comparison between PSC and PPS lipofibroblasts revealed largely similar expression profiles, with the exception of an enrichment in DNA repair pathways specifically observed in PPS lipofibroblasts. Conclusion These findings provide novel insights of PSC and PPS at the single‐cell level. Key points Distinct Cellular Origins: PSC arises primarily from epithelial (AT2) cells via EMT, whereas PPS is predominantly fibroblast‐derived. Prognostic Driver HMGA2: Elevated HMGA2 in malignant AT2 cells correlates with poor prognosis and is significantly higher in MET‐mutated PSC. Pathway Divergence in Malignant Cells: Malignant lipofibroblasts in PPS share a similar profile with PSC but uniquely enrich DNA repair pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/914312002105a075ce979102043e4694713bf91a" target='_blank'>
              Single‐nucleus RNA sequencing reveals the distinct heterogeneity of primary pulmonary sarcomas (PPS) and pulmonary sarcomatoid carcinoma (PSC)
              </a>
            </td>
          <td>
            Jianfei Zhu, Yanlu Xiong, Shouzheng Ma, Jun Wei, Jia-kuan Chen, Wen-chuan Wang, Q. Duan, Qin Zhang, Dongsheng Chen, Wanglong Deng, Tao Jiang, Jie Lei
          </td>
          <td>2025-12-30</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Liquid biopsies can detect actionable mutations and infer broad tumor states from genome-wide cfDNA measurements, but quantitative transcriptome-like phenotyping at single gene resolution still largely requires tissue. Here, we asked whether 6-base whole-genome sequencing that jointly quantifies 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC) could infer gene expression directly from plasma. We applied this framework to plasma from patients enrolled in a phase 2 clinical trial of the PARP inhibitor olaparib plus the PD-L1 inhibitor durvalumab for metastatic castration-resistant prostate cancer. Inferred plasma transcriptomes distinguished adenocarcinoma vs. neuroendocrine phenotypes and identified a noncanonical WNT5A-associated signature linked to poor clinical response. Integrating longitudinal cfDNA methylomic profiles with phylogenetic reconstruction further revealed two resistance trajectories: one featuring high tumor heterogeneity with persistent AR signaling, and another marked by an AR-independent, stem-like program with metabolic reprogramming. These findings demonstrate that ctDNA can inform phenotype-driven tumor biology at gene-level resolution, integrating epigenetic modifications, inferred transcriptional programs, and clonal dynamics as a function of treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/577ed4ef29e6ee2ab69965179663102c2fe760c9" target='_blank'>
              Integrative dual ctDNA 5mC/5hmC methylomics and clonal reconstruction infer tumor transcription and resistance phenotypes in metastatic prostate cancer
              </a>
            </td>
          <td>
            A. Sowalsky, Chennan Li, A. Baj, Clara C Y Seo, N. Terrigino, J. Bright, S. Hennigan, I. King, S. Wilkinson, Tzu-Ting Huang, S. Trostel, William Figg, William L Dahut, David Y. Takeda, Jung-Min Lee, F. Karzai
          </td>
          <td>2026-02-05</td>
          <td>Research Square</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2139ae2049ffda8bfe3d7872db28f8e52809763c" target='_blank'>
              Single-cell mapping of cholesterol metabolism reveals FDPS as a therapeutic vulnerability in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Xupeng Yang, Jiajun Li, Yurong Wang, Qiang Gao, Mao Zhang
          </td>
          <td>2026-02-18</td>
          <td>Cellular Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 The human prostate develops from the urogenital sinus through epithelial bud formation and branching, however the spatial dynamics of progenitor populations during these early stages remain poorly characterized. To addresses this gap, we used a human fetal prostate at 14 weeks of gestation for serial sectioning, yielding approximately 200 sections of 10 µm thickness. Six sections were profiled using spatial transcriptomics with the Visium HD platform, while the remaining sections were H&E-stained for 3D reconstruction of glandular architecture. We mapped transcriptional heterogeneity across developing prostate buds and identified distinct spatial organization of epithelial subdomains with basal-like and luminal-like characteristics. Notably, we also identified a unique cluster of tubule-initiating cells (TIC cells), that have been linked to cancer initiation and androgen-deprivation therapy resistance in adult prostate. This study provides the first high-resolution spatial map of developing human prostate, highlighting the organisation of progenitor populations.



 Emmanouela Perisynaki. 3D spatial mapping of developing human prostate glands [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B059.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9e67e64e1e6f9570a547e8411a3ed76fac26e1" target='_blank'>
              Abstract B059: 3D spatial mapping of developing human prostate glands
              </a>
            </td>
          <td>
            Emmanouela Perisynaki
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5342473ebabd904e74c8bc0079d51547ba07dfc1" target='_blank'>
              Spatial transcriptomics reveals a key role of fibroblast-like vascular smooth muscle cells in human atherosclerotic cell crosstalk and stability.
              </a>
            </td>
          <td>
            Isabel Gonçalves, Mengyu Pan, Pratibha Singh, Wenqi Wang, Jing Zhao, L. Dib, L. Sundius, A. Persson, C. Gialeli, P. Fountas, M. Nitulescu, Jan Nilsson, Stephen Malin, Claudia Monaco, Helle F Jørgensen, Jiangming Sun, A. Edsfeldt
          </td>
          <td>2026-02-13</td>
          <td>European heart journal</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive neuroendocrine disease marked by rapid growth, early metastatic spread, and poor outcomes. The addition of immune checkpoint inhibitors (ICIs) targeting the PD-1/PD-L1 axis to first-line chemotherapy has recently reshaped the treatment landscape for extensive-stage SCLC (ES-SCLC); however, the resulting survival gains remain modest compared with non-small lung cancer (NSCLC). This review explores the molecular features of the SCLC immune landscape that contribute to its predominantly “cold” tumor phenotype, including low MHC class I expression, T-cell exhaustion, and a profoundly immunosuppressive tumor microenvironment (TME). We summarize key clinical findings from landmark trials and examine mechanisms of both primary and acquired resistance against ICIs in SCLC. In addition, we have reviewed the growing role of precision medicine in SCLC, including molecular subtyping (SCLC-A, -N, -P, and -I) and the development of next-generation immunotherapies such as bispecific T-cell engagers (BiTEs), B7-H3, targeted therapy, and antibody–drug conjugates. By combining existing clinical evidence with new molecular insights, this review article presents strategies to overcome the existing therapeutic plateau and enhance personalized immunotherapy approaches in SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b191285a5986b088542ce1486a8d7eb93f1db82a" target='_blank'>
              Immunotherapy in Small Cell Lung Cancer: Advances, Barriers, and Emerging Strategies
              </a>
            </td>
          <td>
            Manish Charan, Tanisha Mukherjee, Krina Patel, Ramesh K. Ganju
          </td>
          <td>2026-02-05</td>
          <td>Onco</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="BACKGROUND
Cancer cells are heterogeneous, each harboring distinct molecular aberrations and being dependent on different genes for their survival and proliferation. While targeted therapies based on driver DNA mutations have shown success, many tumors lack druggable mutations, limiting treatment options. We hypothesize that new precision oncology targets may be identified through "expression-driven dependency," where cancer cells with high expression of specific genes are more vulnerable to the knockout of those same genes.


RESULTS
We developed BEACON, a Bayesian approach to identify expression-driven dependency targets by analyzing global transcriptomic and proteomic profiles alongside genetic dependency data from cancer cell lines across 17 tissue lineages. BEACON successfully identified known druggable genes, including BCL2, ERBB2, EGFR, ESR1, and MYC, while revealing novel targets confirmed by both mRNA and protein-expression driven dependency. The identified genes showed a 3.8-fold enrichment for approved drug targets and a 7 to 10-fold enrichment for druggable oncology targets. Experimental validation demonstrated that depletion of GRHL2, TP63, and PAX5 effectively reduced tumor cell growth and survival in their dependent cells.


CONCLUSIONS
Our approach provides a systematic method to identify precision oncology targets based on expression-driven dependency patterns. By integrating multi-omics data with genetic dependency screens, we've created a comprehensive catalog of potential therapeutic targets that may expand treatment options for cancer patients lacking druggable mutations. This resource offers new opportunities for precision oncology target discovery beyond mutation-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d9df8d8c54b9f6f78829d05c5029fee1f72554" target='_blank'>
              Expression-Driven Genetic Dependency Reveals Targets for Precision Oncology.
              </a>
            </td>
          <td>
            Abdulkadir Elmas, Hillary M. Layden, Jacob D. Ellis, Luke N. Bartlett, Xian Zhao, Reika Kawabata-Iwakawa, Zishan Wang, Hideru Obinata, Scott W. Hiebert, Kuan-lin Huang
          </td>
          <td>2026-01-29</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Predicting spatial gene expression from H&E histology offers a scalable and clinically accessible alternative to sequencing, but realizing clinical impact requires models that generalize across cancer types and capture biologically coherent signals. Prior work is often limited to per-cancer settings and variance-based evaluation, leaving functional relevance underexplored. We introduce HistoPrism, an efficient transformer-based architecture for pan-cancer prediction of gene expression from histology. To evaluate biological meaning, we introduce a pathway-level benchmark, shifting assessment from isolated gene-level variance to coherent functional pathways. HistoPrism not only surpasses prior state-of-the-art models on highly variable genes , but also more importantly, achieves substantial gains on pathway-level prediction, demonstrating its ability to recover biologically coherent transcriptomic patterns. With strong pan-cancer generalization and improved efficiency, HistoPrism establishes a new standard for clinically relevant transcriptomic modeling from routinely available histology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c31313036e644587313c31ba04ae584ee9cd4119" target='_blank'>
              HistoPrism: Unlocking Functional Pathway Analysis from Pan-Cancer Histology via Gene Expression Prediction
              </a>
            </td>
          <td>
            Susu Hu, Qinghe Zeng, Nithya Bhasker, Jakob N Kather, Stefanie Speidel
          </td>
          <td>2026-01-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92e138053e2b61f646d5632a3769e80d536a97a0" target='_blank'>
              Mutation-informed gene pairs to predict melanoma metastasis
              </a>
            </td>
          <td>
            Seongsu Lim, Younggyun Lim, Ju Han Kim
          </td>
          <td>2026-01-29</td>
          <td>Cell Communication and Signaling</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720c9004235c7c3aef4fbef8c74bcb96b3903c12" target='_blank'>
              Multiomics analysis reveals that senescent CXCL16+ macrophages promote lung adenocarcinoma progression through TGF-β signalling.
              </a>
            </td>
          <td>
            Zhenhu Zhang, Jianjie Yin, Wei Li, Han Xiao, Dong Wang, Meng Yang
          </td>
          <td>2026-02-02</td>
          <td>Journal of translational medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad062077323547175d6b721e78e8771a9325ce15" target='_blank'>
              Phenotypic Plasticity and Competition Shape Therapy Sequencing in HER2+/HER2− Breast Cancer: A Mathematical Framework
              </a>
            </td>
          <td>
            Aleksandra Gavrilova, Trachette L. Jackson, Nizhum Rahman
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Colon adenocarcinoma (COAD) is the most common type of colon cancer, posing a significant threat to public health. In the tumor microenvironment (TME), T cells differentiate into terminally exhausted T cells (TEX), but the relationship between TEX and COAD has not been fully elucidated. To identify TEX-related signatures, we integrated transcriptomic data from TCGA and GEO databases (GSE103479, GSE17536). A prognostic model was constructed using GSVA, univariate Cox, LASSO, and random forest algorithms. The tumor immune microenvironment was characterized using CIBERSORTx and GSEA. The functional role of FAT4 was validated in vitro using FAT4-knockdown COAD cell lines assessed by flow cytometry and RT-qPCR. We developed a prognostic signature based on five TEX-related genes (IL21R, FCRL3, TIFAB, TNFSF14, SLAMF1). Patients in the high-risk group showed significantly poorer overall survival and distinct immune cell infiltration patterns, characterized by decreased CD8 + T cells and M1 macrophages. At single-cell resolution, CD8 + TEX cells exhibited high expression of immune checkpoints like LAG3. Furthermore, in vitro experiments demonstrated that knockdown of FAT4, a frequently mutated gene in COAD, promoted apoptosis and induced G0/G1 cell cycle arrest in COAD cells. We proposed a non-invasive prediction method based on TEX-related genes, which effectively predicts survival outcomes and therapeutic responses in COAD patients. Additionally, FAT4 was found to regulate proliferative and apoptotic phenotypes in COAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83927c69818d8de262c593f498367f67a4898a30" target='_blank'>
              T-cell exhaustion indicator characterizes the tumor microenvironment landscape and predicts colon adenocarcinoma prognosis via integrating single-cell RNA-seq and bulk RNA-sequencing
              </a>
            </td>
          <td>
            Haifeng Sun, Ning Hu, Shaowei Ma, Yu Zheng, Ren Niu, Wenya Zhu, Shaofan Qiu
          </td>
          <td>2026-01-14</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Single-cell genomics has emerged as a transformative approach to unravel the complexity of somatic variation in specific cells within the human body. This field has profound implications for understanding the role of somatic mutations in aging, cardiovascular disease, and tissue-specific pathologies. By focusing on circulating cells and cardiac resident cells, including cardiomyocytes, within the heart, this review explores how single-cell genomics provides insights into cellular heterogeneity and clonal evolution. We discuss the implications of somatic variation for cardiovascular health, highlight technological innovation, and future directions in this rapidly evolving field. This review sheds light on national initiatives, such as the National Institutes of Health Somatic Mosaicism Across Human Tissues Network (SMaHT) and the European Somatic Mutations in Vascular-Wall Function and Age-Associated Disease (EU SOMATICART) project, which aim to generate reference atlases of somatic mutations across human tissues. It also explores challenges and future directions in leveraging single-cell approaches to improve diagnostics and therapeutics in cardiovascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ec9fb81424f8866ab25b7b24005bdae8410be68" target='_blank'>
              Single-Cell Genomics and Somatic Variation in Circulating and Cardiac Resident Cells.
              </a>
            </td>
          <td>
            N. Hilal, Maniteja Arava, S. Choudhury
          </td>
          <td>2026-01-02</td>
          <td>Circulation research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Patients with von Hippel-Lindau (VHL) disease develop multiple distinct kidney cysts and clear cell renal cell carcinomas (ccRCCs) throughout their lifetime. Although cysts are typically considered precursors of ccRCC, no molecular evidence of cysts to tumor progression was provided to date, due to the lack of an accurate molecular characterization of these lesions. We performed a comprehensive molecular characterization of four kidney cysts and six ccRCCs obtained from the same VHL-disease patient, thus all sharing the same genetic germline and host environment. We combined whole genome sequencing (WGS) and RNA sequencing (RNA-seq) profiling with pathological examinations. Cysts and tumors exhibited distinct transcriptomic profiles. Tumors showed increased glycolysis and response to hypoxia, while cysts were associated with enrichment in extracellular matrix organization and inflammation and displayed elevated expression of nephron distal portion markers. Deconvolution analysis further revealed different stromal and immune microenvironments with cysts enriched in myofibroblasts and plasma cells signatures whereas tumors were associated with tumor-associated macrophages (TAMs) and tumor vasculature signatures. Genomically, most cysts and tumors were clonally independent, harboring distinct somatic single-nucleotide variants. Despite sharing some somatic mutational signatures, tumors and cysts exhibited divergent somatic copy number alterations. Only tumors displayed chromosome 3p loss, resulting in VHL loss of heterozygosity, without other driver mutations. VHL patients develop multiple kidney cysts and solid tumors throughout their lifetime and their clinical management is challenging. This study presents the first in-depth molecular analysis of kidney cysts and ccRCC in a single VHL patient. We observed distinct molecular profiles between cysts and tumors, suggesting independent origins. While preliminary, these findings challenge the assumption that cysts always serve as precursors of ccRCC in VHL disease and underscore the need for larger studies to improve surveillance and management of renal lesions in patients with VHL disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a5cddad02651b1247d054c02f906a3863c26a3" target='_blank'>
              Distinct genomic, microenvironmental, and nephron signatures in VHL kidney cysts and tumors
              </a>
            </td>
          <td>
            I. Rowe, F. Corea, G. Pipitone, G. Scotti, Roberta Lucianò, M. Sant'angelo, N. Tenace, I. Franco, Annamaria Ferrara, Claudio Doglioni, M. Ponzoni, F. de Cobelli, F. Montorsi, U. Capitanio, Paola Carrera, A. Larcher, Andrea Salonia, Federico Daniela Francesco Giorgia Maria Grazia Annalisa Ra Belladelli Canibus Cei Guazzarotti Patricelli Russ, F. Belladelli, D. Canibus, F. Cei, G. Guazzarotti, M. Patricelli, A. Russo, Chiara Re, L. Salerno
          </td>
          <td>2026-01-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Cell-in-cell (CIC) structures, characterized by one cell engulfing another, play a paradoxical role in cancer metastasis. This review synthesizes their dual nature: while CIC can suppress metastasis by eliminating aberrant cells or activating immune responses, it also drives tumor progression through clonal evolution, metabolic adaptation, and genomic instability. The clinical relevance of CIC varies across cancer types, correlating with both favorable and poor prognoses depending on disease stage and microenvironmental context. Emerging therapeutic strategies aim to harness protective CIC mechanisms or inhibit pro-metastatic pathways, though challenges such as biomarker validation and dynamic tracking persist. Future research integrating advanced technologies will elucidate the spatiotemporal complexity of CIC, enabling tailored interventions to disrupt metastatic cascades. By bridging fundamental biology and clinical translation, CIC represents a promising yet intricate frontier in the battle against metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e6d8a02bb8fcc15958d3d8c8939ab24272dbaf6" target='_blank'>
              Cell-in-cell structures and cancer metastasis.
              </a>
            </td>
          <td>
            Yuchen Cao, Jianing Du, J. Lai, Chunjun Liu
          </td>
          <td>2026-01-23</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Early detection of melanoma, a highly variable and aggressive form of skin cancer, is crucial for improving patient outcomes. It is essential to distinguish malignant cells from normal melanocytes, and therefore, label‐free imaging methods that can do so are needed. Given the genotoxic effect of UV radiation, these mutations are numerous and affect many genes, including NRAS; therefore, therapeutic strategies can be directed toward these recurrent mutations. The aggressive nature of NRAS‐mutant melanoma contributes to poor patient prognosis, highlighting the need for early diagnosis. This study utilizes quantitative phase imaging (QPI) with digital holographic microscopy (DHM) to differentiate the morphology of NRAS‐mutant SK‐MEL‐2 cells from melanocytes using holographic microscopy; dimensionality reduction techniques, including Principal Component Analysis (PCA),t‐distributed stochastic neighbor embedding (t‐SNE); and clustering techniques, including K‐Means Elbow Plots and Hierarchical Clustering Dendrograms. The results demonstrated distinct morphologies and geometries between melanocytes and SK‐MEL‐2 cells, indicating that QPI with DHM can serve as a label‐free tool for identifying optical biomarkers in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c04598128bd69a8ebeb4ac33e2197f4b32bd3b" target='_blank'>
              Morphological and Optical Characterization of NRAS‐Mutant Melanoma Cells and Primary Melanocytes via Quantitative Phase Imaging With Digital Holographic Microscopy
              </a>
            </td>
          <td>
            Ayah A Farhat, Yazan A. Almahdi, Fatima Z Alshuhani, Besa Xhabija
          </td>
          <td>2026-01-01</td>
          <td>Cell Biochemistry and Function</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is an aggressive disease with high mortality and limited treatment options, due to its lack of receptors that have targeted therapies available. The tumor microenvironment (TME) plays a critical role in TNBC progression and therapeutic resistance. In this work, we developed a novel mathematical model to describe key cellular interactions within the TNBC TME, informed by current literature and expert input. Our model consists of a system of ordinary differential equations representing five interacting cell populations: M2 macrophages, cancer-associated fibroblasts, TNBC tumor cells, cytotoxic T lymphocytes, and regulatory T cells. We performed global sensitivity analysis to determine which model parameters most strongly influence tumor burden over a clinically-relevant treatment timeframe. The pathways associated with the most-influential parameters correspond to biological mechanisms that are consistent with known and emerging therapeutic strategies in TNBC, including stromal-mediated tumor support. These results highlight key regulatory interactions within the TNBC TME and provide a quantitative framework for hypothesis generation and future investigation of combination treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/909259cdc855ae38ccb7af1c43dae238c47f3735" target='_blank'>
              Identifying Therapeutic Targets for Triple-Negative Breast Cancer using a Novel Mathematical Model of the Tumor Microenvironment
              </a>
            </td>
          <td>
            K. Adams, Julia Bruner, Salma Ameziane, Ashley N. Brown, M. Gbadamosi, H. Moore
          </td>
          <td>2026-01-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Adamantinomatous craniopharyngioma (ACP), a benign yet highly recurrent and therapy-resistant intracranial tumor, remains a considerable clinical challenge because of its complex pathological structure, infiltrative growth, and limited treatment options. Here, integrated spatially resolved multiomics is employed-including single-cell spatial transcriptomics via CosMx SMI and spatially resolved metabolomics via AFADESI-MSI, accompanied by bulk metabolomics and functional validation-to unravel the driving factors of ACP progression and recurrence. Analysis results reveal three interdependent biological hallmarks: first, the spatial segregation and molecular heterogeneity of 10 distinct tumor epithelial cell subpopulations within the ACP, each of which presents unique transcriptional signatures; second, in tumor regions and recurrent tumor epithelium tissues, stronger transporter-mediated choline/ethanolamine uptake from cystic fluid and significant upregulation of phosphatidylcholine (PC) and phosphatidylethanolamine (PE) synthesis is observed, creating the enhanced "cystic fluid-tumor cell" and "choline/ethanolamine-PC/PE" metabolic axis, and demonstrating the spatial metabolic remodeling of ACP; and third, this metabolic axis directly couples to autophagy activation of corresponding regions in ACP tissue, which is validated by multi-immunohistochemistry for Beclin1 and GABARAP. Together, these findings reveal metabolic remodeling and autophagic activation as critical drivers of ACP progression and recurrence and provide an opportunity for precise biomarker-driven treatment of this intractable tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4bf376edd94fbdf6f86d786c026d6877b062546" target='_blank'>
              Spatially Resolved Multiomics Reveals Metabolic Remodeling and Autophagy Activation in Adamantinomatous Craniopharyngiomas.
              </a>
            </td>
          <td>
            Dongting Chen, Yahui Gao, Yulin Wang, Ting Lei, Zheng Qu, Yuhan An, J. Fu, Xin Li, Fangjun Liu, Yan Li
          </td>
          <td>2026-01-05</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f2b7d842c50e19f140b3dba7c07392f64035b86" target='_blank'>
              Single-nucleus multi-omics delineates distinct epigenetic programs associated with tumor progression in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Yeji Kim, Huiram Kang, Y. Hwang, Kyung Soo Kim, S. Moon, Jeonghan Kim, Seung Joon Kim, Hae-Ock Lee
          </td>
          <td>2026-02-11</td>
          <td>Clinical epigenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Gene-specific expansion patterns were evident among the most frequent CH lesions, with DNMT3A-mutant clones exhibiting impaired hematopoietic differentiation and TET2-mutant clones showing multi-lineage propagation. Notably, identical CH clones were detected in 41% of corresponding lymphomas, displaying distinct clonal dynamics: tumor-promoting CH (expansion in B-NHL; 10/16 clones; mainly TP53) and tumor-infiltrating CH (no expansion; mainly DNMT3A). Moreover, we identified lymphoma-associated mutations in flow-sorted hematopoietic progenitors from patients with indolent but not aggressive B-NHL and observed a stepwise accumulation of mutations along the lymphoid differentiation path. Single-cell genotyping confirmed the presence of mutated progenitors in 3 follicular, 2 mantle cell and 2 marginal zone lymphoma patients, providing direct evidence of a pre-neoplastic state in disease pathogenesis. Our findings offer novel insight into the cellular origin of nodal B-NHLs and highlight a previously underappreciated role for early clonal events involving the stem/progenitor cell compartment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ec5b65b719f8d9e5db3ee2069637056c360c3b" target='_blank'>
              Clonal Hematopoiesis and Lymphoma-Associated Mutations in Hematopoietic Progenitors in B-Cell Non-Hodgkin Lymphoma.
              </a>
            </td>
          <td>
            Laura Wiegand, Patricia Silva, D. Noerenberg, F. Christen, Klara Kopp, B. Locher, P. Löwe, Marlon Tilgner, Robert Altwasser, Vanessa Storzer, Catarina M Stein, F. Briest, C. M. Arends, M. Frick, J. Ihlow, A. Dolnik, N. Ishaque, U. Keller, Il-Kang Na, L. Penter, Lars Bullinger, Raphael Hablesreiter, Frederik Damm
          </td>
          <td>2026-01-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract To understand how the tumor immune microenvironment (TIME) impacts clinical outcomes and treatment response, researchers have been leveraging single-cell protein multiplex imaging techniques. These technologies measure multiple protein markers simultaneously within a tissue sample, providing a more complete assessment of the TIME. However, statistical challenges arise from the over-dispersed and zero-inflated nature of the data and from relationships among different immune cell populations. To address these challenges, we developed a Bayesian hierarchical method using a beta-binomial (BB) distribution to model the abundance of multiple immune cell types simultaneously while incorporating relationships and immune cell differentiation paths. We applied the model to data from three large studies of high-grade serous ovarian tumors (Nurses’ Health Study I/II: N = 321, African American Cancer Epidemiology Study: N = 92, University of Colorado Ovarian Cancer Study: N = 103). We examined associations between cancer stage, age at diagnosis, and debulking status and the abundance of immune cell populations. We compared the multi-cell type model to individual cell type analyses using a Bayesian BB model. The multi-cell type model detected more associations, when present, with narrower credible intervals. To support broader application, we developed an R package, BTIME, with a detailed tutorial. In conclusion, the Bayesian multi-cell type model is flexible in how relationships between cell types are incorporated and can be used for cancer studies that interrogate the TIME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf0b756a4093987a6b750f084e62f9d9891d65d" target='_blank'>
              Bayesian multi-cell type models for the analysis of complex immune cell populations with application to ovarian cancer
              </a>
            </td>
          <td>
            Chase J Sakitis, José Laborde, Julia Wrobel, Alex C. Soupir, Christelle M. Colin-Leitzinger, Benjamin G. Bitler, Mary K Townsend, Andrew B Lawson, J. Schildkraut, Shelley S. Tworoger, K. Terry, Lauren C. Peres, Brooke L. Fridley
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Lung cancer persists as the leading cause of cancer-related mortality globally, largely due to late detection, genomic complexity, and limited durability of existing therapeutic interventions. The integration of CRISPR/Cas9 gene-editing systems with biomarker-driven therapeutic strategies represents a transformative advance in precision oncology. Biomarkers—including genetic mutations, epigenetic alterations, protein signatures, and circulating analytes—enable early detection, patient stratification, and dynamic monitoring of therapeutic response. CRISPR/Cas9 offers a unique opportunity to directly reprogram these biomarkers or the pathways regulating them, enhancing tumor immunogenicity, reversing immune evasion mechanisms, and strengthening anti-tumor immune responses. Preclinical models demonstrate that CRISPR-mediated biomarker editing can restore antigen presentation, augment T-cell cytotoxicity, sensitize resistant tumors to immunotherapy, and improve tumor regression. Early clinical trials further validate the feasibility and safety of CRISPR-engineered immune cells in patients. 
However, major challenges persist, including off-target editing, inefficient delivery to solid tumors, tumor microenvironment–mediated suppression, and ethical considerations linked to genome manipulation. Rapid advancements in editing fidelity, lipid nanoparticle systems, viral vectors, engineered vesicles, high-throughput biomarker discovery, and artificial intelligence–assisted CRISPR design are expected to accelerate clinical translation. This review synthesizes the current landscape, mechanistic underpinnings, emerging applications, and future directions of CRISPR/Cas9-enabled biomarker engineering for lung cancer immunotherapy, positioning this technology as a cornerstone of next-generation personalized oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80a98d71ca868cfed559df1a6401f5e2e8b0702" target='_blank'>
              Precision Reprogramming of Lung Cancer Biomarkers via CRISPR/Cas9: A Paradigm Shift in Personalized Immunotherapy
              </a>
            </td>
          <td>
            M. R. Jameel, Amar Arora, Prem Shankar Mishra, Hina Jameel, M. Makhdoomi, Abid Ali Sheikh
          </td>
          <td>2025-12-30</td>
          <td>Journal of Analytical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastasis remains the leading cause of cancer-related mortality, yet current clinical tools lack sensitivity for early detection of metastatic potential. While gene expression assays have improved prognostic stratification in select cancers, they often overlook dynamic cellular behaviors such as cytoskeletal remodeling and mechanotransduction. We curated a panel of 11 mechanobiology-related genes involved in actin cytoskeleton regulation and evaluated their predictive value using multi-modal data from the DepMap database, including gene expression, copy number variation, CRISPR, and RNAi screens. Analyses were conducted across 351 cell lines from breast, pancreatic, and central nervous system (CNS) tumors, stratified by tumor type and MetMap-derived metastatic potential. Gene expression and CNV emerged as the most informative parameters for distinguishing metastatic potential. Among the candidate genes, FSCN1 consistently demonstrated strong associations across tissues, classification schemes, and correlation with canonical metastasis markers. MYH9, ECM1, ANXA2, and EZR also showed robust performance, particularly in cumulative multi-gene signatures. Correlation analyses revealed significant positive correlations with invasion, EMT, stemness markers (e.g., MMP2, VIM, CD44), and inverse correlations with epithelial markers (e.g., CDH1, EPCAM), reinforcing the biological coherence of the panel. Our study introduces a previously unrecognized, mechanobiologically driven gene panel - FSCN1, MYH9, ECM1, ANXA2, and EZR - as a cross-cutting biomarker signature of metastatic potential, applicable across multiple cancer types. Unlike conventional markers that often reflect tumor-specific traits, this panel captures the mechanical and structural determinants of cell invasiveness. The robustness of these genes across diverse molecular assays and their alignment with core metastatic pathways underscore their translational relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c4b10f1d7534530aa68a76ea1c1843e920c5a7d" target='_blank'>
              Mapping metastatic potential through cytoskeletal remodeling: A mechanobiology-based multi-cancer analysis.
              </a>
            </td>
          <td>
            Ksenia Maksimova, M. Pustovalova, Denis V. Kuzmin, S. Leonov, Y. Merkher
          </td>
          <td>2026-02-14</td>
          <td>Biochemical and biophysical research communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The developmental relationship between nephron progenitors and the renal interstitium remains unresolved, in part due to limitations of existing lineage tracing tools. The widely used transgenic Six2TGC line, which is routinely employed to target the nephron lineage, exhibits mosaic recombination and altered progenitor dynamics. To overcome these shortcomings, we generate a knock-in Six2Cre mouse allele that faithfully recapitulates endogenous Six2 expression, preserves nephron endowment, and achieves near-complete, non-mosaic recombination. Side-by-side lineage tracing with Six2Cre and Six2TGC, combined with RNA velocity analysis of single-cell RNA-sequencing datasets, reveals a brief interval around embryonic day 11 during which Six2-expressing mesenchymal nephron progenitors contribute to the renal interstitium. This contribution is transient and stage-restricted. These findings reveal an early dual potential within nephron progenitors and define a precise developmental window for dissecting mechanisms that coordinate nephron-interstitium integration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d8532b50b850d5681a65c224e04b899e62045e0" target='_blank'>
              A knock‑in Six2Cre line reveals transient interstitial potential in nephron progenitors
              </a>
            </td>
          <td>
            Azadeh Haghighitalab, Fariba Nosrati, Zeinab Dehghani-Ghobadi, Mohammed Sayed, Christopher Ahn, Yueh-Chiang Hu, Eunah Chung, H. Lim, 
          </td>
          <td>2026-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Inflammatory breast cancer (IBC) is a rare, aggressive subtype of breast cancer with unfavorable prognosis. Despite its distinct clinical presentation and molecular features, the immune landscape of IBC and its role in driving the aggressive phenotype remain poorly understood. This study aimed to characterize the spatial immune landscape of IBC using digital image analysis (DIA), compare it with subtype-matched non-inflammatory breast cancer (nIBC), and evaluate prognostic implications of immune cell composition and localization.



 We examined pretreatment tumor samples from 161 IBC and 115 subtype-matched nIBC patients using an AI-assisted DIA pipeline. Immunostainings (H-DAB) were conducted using validated protocols for CD8 (cytotoxic T cells), FOXP3 (Tregs), CD79α (activated B cells), PDL1 (SP124), and CD163 (tumor-associated macrophages, TAMs). Slides were digitized and assessed using VISIOPHARM® software, enabling virtual multiplexing. We quantified DAB+ immune cells and their locations using XY coordinates to calculate density. Spatial colocalization was analyzed using ecological statistics, based on point pattern or quadrant analysis. In point pattern analysis, we counted immune cells within a 30 µm radius around CD8+ cells or tumor cells. For quadrant analysis, we applied the Morisita-Horn Index (MHI) after square tessellation (250×250 µm tiles). The MHI compares cell composition between tile pairs, ranging from 0 to 1, with higher values indicating greater dissimilarity. Associations with clinicopathological features and outcome were assessed using multivariate regression models. Transcriptomic validation was performed using Affymetrix gene expression data and consensusTME deconvolution.



 IBC showed higher CD163+ tumor-associated macrophage (TAM) infiltration than nIBC. Gene expression data confirmed the IHC findings, and pathway analysis linked high TAM density with inflammatory and proliferative pathways. The spatial distribution of immune cells was prognostically relevant, with high CD8+ T-cell (OR: 0.41, 95% CI: 0.22-0.76, P = 0.004) and low CD79α+ B-cell infiltration (OR: 3.19, 95% CI: 1.68-6.03, P < 0.001) correlating with improved overall survival in IBC. The ratio of CD8+ T cells to FOXP3+ regulatory T cells within tumor area was a significant prognostic indicator (OR: 0.34, 95% CI: 0.14-0.83, P = 0.018), whereas absolute densities of CD8+ or FOXP3+ cells alone did not associate with outcome. Beyond density alone, spatial colocalization of B cells with tumor cells emerged as a key prognostic factor. A high tumor colocalization index for CD79α+ B cells was associated with significantly worse survival (OR: 3.59, 95% CI: 1.31-9.84, P = 0.013). Similarly, greater spatial similarity between CD79α+ B cells and tumor cells, assessed via the MHI, was linked to poor outcome (OR: 2.61, 95% CI: 1.19-5.74, P = 0.017). Incorporating spatial B-cell features improved the prognostic model (P = 0.0005).



 This comprehensive spatial immune profiling effort in a large IBC cohort revealed the immunosuppressive nature of the IBC microenvironment and underscored the central role of TAMs in promoting its aggressive behavior. Spatial context, not immune cell abundance, was most predictive of outcome. High colocalization of CD79α+ B cells with tumor cells associated with poor survival, suggesting a tumor-promoting role. These findings demonstrate the value of spatial immunophenotyping beyond conventional quantification and support exploring TAM- and B cell-targeted therapies in IBC.



 C. Van Berckelaer, S. Van Laere, C. Vermeulen, M. Kockx, Y. Waumans, K. Mariën, F. Berditchevski, F. Bertucci, P. Vermeulen, L. Dirix, C. Colpaert, G. R. Devi, P. Van Dam. Spatial Immune Profiling in Inflammatory Breast Cancer Reveals a Central Role for Tumor-Associated Macrophages and a Spatial Context-Dependent Negative Prognostic Impact of B Cells [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS3-12-19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f953a9ca916b4f141aeda25be894325b159bc24" target='_blank'>
              Abstract PS3-12-19: Spatial Immune Profiling in Inflammatory Breast Cancer Reveals a Central Role for Tumor-Associated Macrophages and a Spatial Context-Dependent Negative Prognostic Impact of B Cells
              </a>
            </td>
          <td>
            C. Van Berckelaer, S. Van Laere, C. Vermeulen, M. Kockx, Y. Waumans, K. Mariën, F. Berditchevski, F. Bertucci, P. Vermeulen, L. Dirix, C. Colpaert, G. Devi, P. van Dam
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Crohn’s disease (CD) leads to fistula formation in about 40% of patients due to persistent transmural inflammation. Despite advanced therapies, recurrence remains frequent and the molecular mechanisms remain poorly defined, partly because intact tracts are difficult to profile and highly heterogeneous. We integrated multi-modal spatial transcriptomics, single-cell profiling and matrix imaging to define the cellular and molecular architecture of intestinal fistulae.



 We constructed a comprehensive atlas of 92 human intestinal fistulae and control tissues using Visium spatial transcriptomics, subcellular-resolution Xenium profiling (480 and 5100-plex), single-cell RNA sequencing, multiplexed immunohistochemistry and quantitative collagen imaging. Over 90,000 spatial transcriptomic spots and seven million segmented cells were analysed. Computational integration mapped fistula-linked single-cell states and conserved cellular niches across diverse lesions, including CD and non-CD fistulae. Functional consequences of aberrant fibroblast re-programming was explored via transduction of primary colonic fibroblasts with transcriptional regulators we identified as abnormally induced in fistula tracts.



 We identified a fibroblast population uniquely associated with fistulae, marked by TWIST1, RUNX2, OSR2 and PRRX1, termed fistula-associated stromal (FAS) fibroblasts. These cells expressed developmental morphogens (e.g., WNT5A) and matrix-remodelling mediators (MMP1/3/9, IL11, COL7A1) promoting invasion, propagation and fibrosis. In vitro assays showed that TWIST1 and OSR2 drive distinct functional programs in FAS fibroblasts. Spatial analyses revealed concentric FAS zones around tracts: a proliferative lumen-adjacent zone (FAS-LAZ), a matrix-remodelling active lesion core (FAS-ALC), and a fibrotic outer zone (FAS-FOZ). Collagen mapping demonstrated matching extracellular matrix zonation, from hypodense collagen at proliferative areas to high anisotropy in fibrotic zones. Stromal–immune co-localisation identified interactions between FAS fibroblasts and SPP1+ macrophages driving immune recruitment (CXCL5/9) and matrix remodelling. FAS-like fibroblasts were also found at ulcer bases in non-fistulating CD.



 This study defines fistula-associated stromal and immune states as key mediators of invasion, immune crosstalk and fibrotic remodelling. Aberrant developmental transcriptional programmes in FAS cells lead to abnormal zonation of growth factors and morphogens, reshaping collagen architecture and sustaining fistula persistence over normal healing. These findings provide new mechanistic insights into fistula biology and information to guide model design and preventatives.
 Conflict of interest:
 McGregor, Colleen: No conflict of interest
 Qin, Xiao: No conflict of interest
 Jagielowicz, Marta: No conflict of interest
 Gupta, Tarun: No conflict of interest
 Yin, Zinan: No conflict of interest
 Lentsch, Verena: No conflict of interest
 Fawkner-Corbett, David: No conflict of interest
 Lai, Vy: No conflict of interest
 Gomez Castro, Paula: No conflict of interest
 Bridges, Esther: No conflict of interest
 Lee, Chloe H: No conflict of interest
 Chuang, Huei-Wen: No conflict of interest
 Deng, Lei: No conflict of interest
 Aulicino, Anna: No conflict of interest
 Teague, Renuka: No conflict of interest
 Moradi, Sorayya: No conflict of interest
 Park, Jun Sung: No conflict of interest
 Woo, Jeongmin: No conflict of interest
 Xu, Kexin: No conflict of interest
 Tandon, Ruchi: No conflict of interest
 Cianci, Nicole: No conflict of interest
 Bornschein, Jan: No conflict of interest
 Ho, Ling-Pei: No conflict of interest
 Siejka-Zielinska, Paulina: No conflict of interest
 Christoforidou, Zoe: No conflict of interest
 Hill, Sarah: No conflict of interest
 Lehmann, Johannes: No conflict of interest
 Kujawa, Rhea: No conflict of interest
 Vargas Gutierrez, Paola: No conflict of interest
 Cheng, Carol: No conflict of interest
 Greco, Maria: No conflict of interest
 Baker, Katherine: No conflict of interest
 Bignell, Mark: No conflict of interest
 George, Bruce: No conflict of interest
 Fryer, Eve: None
 Vieth, Michael: No conflict of interest
 Antanaviciute, Agne: No conflict of interest
 Simmons, Alison: No conflict of interest
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abb159aa8756127949b8839929a8382cd786509a" target='_blank'>
              OP24 Spatial fibroblast niches shaping Crohn’s fistula pathogenesis
              </a>
            </td>
          <td>
            C. McGregor, X. Qin, M. Jagielowicz, T. Gupta, Z. Yin, V. Lentsch, D. Fawkner-Corbett, V. Lai, P. G. Castro, E. Bridges, C. H. Lee, H. Chuang, L. Deng, A. Aulicino, R. Teague, S. Moradi, J. S. Park, J. Woo, K. Xu, R. Tandon, N. Cianci, J. Bornschein, L. Ho, P. Siejka-Zielinska, Z. Christoforidou, S. Hill, J. Lehmann, R. Kujawa, P. Vargas Gutierrez, C. Cheng, M. Greco, K. Baker, M. Bignell, B. George, E. Fryer, M. Vieth, A. Antanaviciute, A. Simmons
          </td>
          <td>2026-01-01</td>
          <td>Journal of Crohn’s and Colitis</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Vascular smooth muscle cells (VSMCs) are critical players in atherosclerotic plaque development and progression, but existing isolation methods often yield low cell viability, making them suboptimal for several downstream applications. As such, we aimed to optimize an enzymatic digestion protocol for isolating viable VSMCs from human carotid plaques suitable for cell culture analysis. Carotid plaques were obtained following endarterectomy and subjected to a streamlined enzymatic digestion protocol using a simplified cocktail optimized to minimize cell damage. After brief culture to passage 1, single-cell suspensions were processed using the 10X Genomics Chromium platform. Single-cell RNA sequencing (scRNA-seq) data underwent rigorous quality filtering, Harmony-based integration, clustering, gene ontology (GO) enrichment analysis, and pseudotime trajectory inference to validate cell identity and assess phenotypic transitions. The protocol yielded high-viability single-cell suspensions, with approximately 23,661 cells sequenced after quality filtering from four carotid plaques: two classified as stable and two as unstable, obtained from both male and female patients. scRNA-seq revealed that the majority of isolated cells were VSMCs, confirmed through positive module scores for canonical VSMC markers (TAGLN, ACTA2, MYH11, CNN1, MYL9) and GO enrichment of VSMC-related biological processes. Label transfer analysis using the Tabula Sapiens vasculature reference further confirmed VSMCs as the predominant population. Subcluster analysis identified four transcriptionally distinct VSMC phenotypes. Pseudotime analysis demonstrated a trajectory originating from contractile VSMCs, progressing through intermediate foam cell-like and inflammatory synthetic states, and ultimately transitioning into fibroblast-like phenotypes. This confirms that the entire isolated population was VSMC-derived and undergoing phenotypic switching. In conclusion, this optimized protocol enables the reliable isolation of highly viable VSMCs from human carotid plaques, as validated by single-cell transcriptomic profiling. These findings lay the groundwork for future applications of isolated VSMCs in single-cell studies, disease modeling, and mechanistic investigations of VSMC phenotypic switching and plaque instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e37b4fe829140cd64d821d92561ecddda3a695c7" target='_blank'>
              A standardized approach for the isolation of vascular smooth muscle cells from carotid atherosclerotic plaques
              </a>
            </td>
          <td>
            Jae Hyun Byun, Helena Papacostas Quintanilla, Irem Engin, Rachel Aladeeb, Julia Ho, Ying He, Francisco J Rios, A. Montezano, R. Touyz, Jiannis Ragoussis, Stella S. Daskalopoulou
          </td>
          <td>2026-01-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>129</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2acf989f2f11c299ed63f4ad0a3f33ccaec34cb5" target='_blank'>
              Ecological Dynamics of Pro-tumor and Anti-tumor Teams in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Vaibhav Anand, M. Jolly, Herbert Levine
          </td>
          <td>2026-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Background In 2024, colorectal cancer (CRC) ranked as the third most frequently diagnosed cancer in Canada and remains one of the leading causes of cancer-related deaths. Current experimental models, including animal models and immortalized cell lines, fail to capture the molecular and cellular heterogeneity of patient tumors. These features are essential for understanding disease mechanisms and developing effective therapies. Over the past decade, organoid has emerged as a powerful approach to model human tissues in three-dimensional culture while preserving their genetic complexity. In cancer research, patient-derived organoids (PDOs) can be established directly from tumor samples, providing highly relevant models. However, the use of PDOs remains limited by factors such as cost, reproducibility, and restricted access to patient tissues. Aims To overcome these challenges, we contributed to the establishment of the Canadian National Organoid Network (CNON), which aims to standardize methods and facilitate access to PDO models for researchers across Canada. Methods Within this initiative, the CNON-Sherbrooke node has developed a living biobank of CRC PDOs. Results For each patient, tissue samples were collected from both tumor and adjacent normal mucosa to generate paired organoid lines. Up to eight different culture medium formulations were systematically tested. Each condition supporting organoid growth was maintained as an independent autologous tumoroid line. PDOs and their matched primary tissues were processed for whole-genome and transcriptome profiling to assess molecular concordance and validate model fidelity. With the long-term goal of recapitulating the tumor and its microenvironment, additional autologous tissues were collected, including fibroblasts, PBMCs, and tissue-associated microbiota. Given the critical role of the tumor microenvironment (TME) in cancer development, we implemented an organ-on-a-chip system enabling the co-culture of multiple cell types within a colon-specific chip to better reproduce TME interactions in CRC. Combined with autologous samples, this approach allows the modelling of a patient-specific TME and is expected to (1) improve our understanding of TME involvement in CRC, and (2) enhance drug screening, prognostic assessment, and the development of targeted therapies. In parallel, we are advancing CRC modeling using genome-editing technologies to reproduce the genetic diversity of CRC and support precision medicine approaches. Conclusions In summary, we aim to improve access to standardized CRC PDOs and optimize their culture systems. By integrating organoid biobanking, genome editing, and microfluidic technologies, this initiative will strengthen Canada’s research capacity and contribute to the development of innovative and personalized treatments for colorectal cancer. Funding Agencies Weston Foundation Family">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac45f72764c84f18a2650281955921f479d56dd3" target='_blank'>
              Poster Session I - A168 PATIENT-DERIVED ORGANOIDS TO ACCELERATE COLORECTAL CANCER RESEARCH
              </a>
            </td>
          <td>
            S. Nassari, M. Lecours, J. Zhang, F. Boudreau
          </td>
          <td>2026-02-01</td>
          <td>Journal of the Canadian Association of Gastroenterology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77606e4919065b0b8c161216da3a8b4fee1b4a2c" target='_blank'>
              Deficiency of SMARCB1 drives an immunosuppressive microenvironment in meningioma.
              </a>
            </td>
          <td>
            B. Jin, Hao Hu, Yanhua Lu, Xia Tian, Guanghui Hu, Jingjing Xu, Xingqi Wu, Long Zhang, Juxiang Chen, Miaoxia He
          </td>
          <td>2026-01-24</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="To examine recent advances in understanding breast cancer brain metastases (BCBM), with emphasis on metastatic mechanisms, tumour-microenvironment interactions, receptor discordance between primary breast tumours and brain metastases, diagnostic innovations, and therapeutic strategies. The review sought to clarify how these developments inform precision management and identify priorities for future research. Studies implicate JAK–STAT signalling, homologous recombination deficiency, c-MYC, PI3K/AKT/mTOR, and RET in BCBM progression. Tumour cells adapt via neuronal mimicry, metabolic reprogramming, and crosstalk with astrocytes and microglia. Receptor discordance between primary breast cancers and brain metastases occurs in up to one-third of cases and may alter systemic therapy. Emerging tools including liquid biopsy, spatial transcriptomics, and radiomics offer minimally invasive approaches for molecular profiling, spatial mapping, and imaging-based phenotyping to guide personalised management. Machine learning supports prognostication and imaging interpretation, though external validation is limited. BCBM reflect complex tumour-brain interactions. Advances in molecular understanding and diagnostics are beginning to inform the development of targeted therapies. Future work may benefit from integrating cancer neuroscience with computational modelling, standardise diagnostic platforms, and test the survival impact of receptor reassessment in prospective trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/386a3d55c94174bf7ce57954937bbc0ca6fcc2b0" target='_blank'>
              Breast Cancer Brain Metastases: Current Understanding and Future Directions
              </a>
            </td>
          <td>
            Mohammad Alabdulrahman, Gordon R. Daly, Sindhuja Naidoo, Lea Stuart, Sami Almasri, Ali Benour, Jason McGrath, Minatoullah Habaka, Gavin P. Dowling, C. Hehir, Arnold D K Hill, D. Varešlija, L. Young
          </td>
          <td>2026-02-05</td>
          <td>Current Oncology Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Epstein–Barr virus (EBV)-positive nodal T/NK-cell lymphoma (EBV
 +
 nTNKL) has recently been delineated in the WHO-HAEM5 classification as a distinct and exceptionally rare entity. Its biology and clinical trajectory remain obscure relative to Extranodal NK/T-cell lymphoma (ENKTL).





 We applied spatial transcriptomics and multiplex immunofluorescence to representative ENKTL and EBV
 +
 nTNKL specimens, integrating these data with a retrospective clinical cohort of 14 EBV
 +
 nTNKL patients—constituting one of the largest series described to date.





 Spatial transcriptomics revealed fundamental differences between ENKTL and EBV
 +
 nTNKL. ENKTL, of NK-cell origin, displayed higher malignant cell density, neutrophil enrichment, and an immune-desert phenotype, whereas EBV
 +
 nTNKL, of T-cell origin, showed reduced tumor burden, B-cell enrichment, and an immune-active microenvironment with abundant cytotoxic T cells and PD-1/PD-L1 expression. Intercellular communication analyses further highlighted distinct signaling programs—TGF-β/BMP-driven tumor–neutrophil interactions in ENKTL versus CXCL/CCL–GPCR-mediated macrophage crosstalk in EBV
 +
 nTNKL. In a retrospective cohort of 14 EBV
 +
 nTNKL patients, the disease was frequently complicated by hemophagocytic lymphohistiocytosis and conferred significantly inferior survival, although selected patients achieved durable responses with immune checkpoint inhibitors or CAR-T therapy.





 This study delineates the immunologic and molecular architectures of ENKTL and EBV
 +
 nTNKL, providing rare insights into this understudied lymphoma. Despite limited sampling, these findings underscore the central role of EBV latency programs and tissue context in shaping tumor ecology and suggest avenues for subtype-tailored therapeutic strategies.

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aa714c1440678adf858808bcde35a539ffc4845" target='_blank'>
              Characterizing tumor microenvironment heterogeneity in EBV+ nTNKL vs ENKTL using spatial transcriptomics and MIF
              </a>
            </td>
          <td>
            Siyu Qian, Zeyuan Wang, Yue Zhang, Wanyue Zhao, Honghan Qiao, Xiaoyan Feng, Yukai Duan, Boyuan Su, Shifeng Hao, Zhenzhen Yang, Mingzhi Zhang, Qingjiang Chen, Guan-nan Wang, Shenglei Li, Xudong Zhang
          </td>
          <td>2026-02-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Tumor development and progression reshape the physical properties of the surrounding tumor microenvironment (TME), including its biomechanical traits. This is driven by a prominent cell type in the TME, cancer-associated fibroblasts (CAFs), which increases tissue stiffness via extracellular matrix deposition and remodeling. Currently, it is unclear whether there are also physical changes to CAFs at the cellular level and, if so, how they relate to patient outcome. Here, it is shown that CAFs have distinct morphological and biomechanical features from normal fibroblasts. Matched, patient-derived CAFs and non-malignant prostate fibroblasts (NPFs) from 35 patients with primary prostate cancer are examined. Morphologically, CAFs have more aligned stress fibers and larger and more elongated nuclei, based on quantitative image analysis of confocal microscopy images. In addition, single-cell mechanical measurements using real-time deformability cytometry showed that CAFs are larger and stiffer than NPFs. These changes are consistent across patients and validated with atomic force microscopy. A combined morphomechanical score encompassing these features is significantly associated with patient outcome. In transcriptomic analyses, the score is correlated with microtubule dynamics and a myofibroblast phenotype. Importantly, it is also demonstrated that morphomechanical features of prostate fibroblasts are modified by approved treatments for prostate cancer, such as docetaxel, and other small molecular inhibitors, particularly those targeting FGFR. In summary, changes in cellular morphomechanical properties are a consistent feature of CAFs and are associated with patient outcome. Moreover, cellular morphomechanical properties can be therapeutically targeted, potentially providing a new strategy for manipulating the TME to control cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe160be726a9c222a84b4d1334642ece40944d7" target='_blank'>
              Single-Cell Morphomechanics of Prostate Cancer-Associated Fibroblasts Identifies Distinct Features Associated with Patient Outcome.
              </a>
            </td>
          <td>
            Antje Garside, Angela Jacobi, S. Keerthikumar, Vaibhav Mahajan, M. Richards, B. Niranjan, Linda Teng, Nicholas K Choo, Johannes Low Jun Wei, G. Risbridger, Mitchell G. Lawrence, A. Taubenberger
          </td>
          <td>2026-01-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Background Intratumor heterogeneity (ITH), a critical driver of tumor evolution and immune evasion, remains inadequately characterized at the transcriptomic level in colorectal cancer (CRC), and its clinical implications are not yet fully understood. Methods We integrated transcriptomic datasets from TCGA-COAD/READ and two independent GEO cohorts (GSE40967 and GSE87211) to develop an RNA-seq–based ITH score using the DEPTH2 algorithm and to con0struct an ITH-related gene (ITRG) prognostic model. A unified cutoff value of 0.64 was established to stratify patients into high- and low-ITH groups. Using 52 survival-associated ITRGs, we generated a nine-gene prognostic signature selected from 101 distinct combinations of feature selection techniques and modeling algorithms and validated its performance in two external datasets. The tumor microenvironment and potential responsiveness to immune checkpoint inhibitors were evaluated using ssGSEA, ESTIMATE, and TIDE algorithms. SHAP analysis, together with in vitro and in vivo experiments, was employed to identify and functionally validate key regulatory genes. Results Patients in the high-ITH group had markedly poorer overall survival (OS) than those in the low-ITH group. The ITH score correlated strongly with aggressive clinical features, including T3/4 invasion depth, N1/2 nodal status, and AJCC stage III. The nine-gene prognostic signature demonstrated consistent predictive capability across the TCGA training cohort and both GEO validation cohorts. In TCGA, the model yielded time-dependent AUCs of 0.669, 0.664, and 0.645 for 1-, 3-, and 5-year OS, respectively, and retained its status as an independent prognostic indicator in multivariate Cox regression analysis; in GSE40967 and GSE87211, the corresponding AUCs ranged from 0.544–0.573 and 0.648–0.744. The high-risk subgroup was characterized by a stromal immune phenotype enriched with cancer-associated fibroblasts and macrophages, elevated expression of multiple immune checkpoints, increased TIDE scores, and higher tumor mutational burden. SHAP analysis identified IL20RB as the top risk-associated gene, whose knockdown significantly suppressed CRC cell proliferation, migration, invasion, and tumorigenicity in vitro and in vivo. Conclusion This study introduces and validates a transcriptomic ITH score and a nine-gene ITRG-based prognostic model that delineate the immune landscape and enables effective survival stratification in CRC, complementing the limitations of current staging systems. Additionally, IL20RB is highlighted as a promising therapeutic target, supporting the development of personalized immuno-targeted combination therapies in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/385eed67dbbc79210270a5630b0149ac9e500c8e" target='_blank'>
              Intratumor heterogeneity score reveals immune landscape and survival stratification in colorectal cancer
              </a>
            </td>
          <td>
            Zijing Wang, Liyuan Ma, Jinzhong Cao, Xi Chen, Xin Chang, Ruxue Ma, Hengyi Lv, Zixin Zhang, Hai Li, Tao Jiang
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The integration of immune checkpoint inhibitors (ICIs) with neoadjuvant chemotherapy has significantly improved clinical outcomes for patients with early triple-negative breast cancer (TNBC). However, approximately one-third of patients still fail to achieve a major pathological response, facing a substantially increased risk of recurrence and death. While immune correlates within the tumor microenvironment and peripheral blood have been extensively studied, the immunological dynamics within tumor-draining lymph nodes (TDLNs) - key sites for the initiation and coordination of anti-tumor immunity - remain poorly understood. Here we present our ongoing investigation into the immune architecture of TDLNs following chemoimmunotherapy to uncover predictive signatures of response and mechanisms of resistance.



 Sentinel lymph nodes from early TNBC patients undergoing neoadjuvant chemoimmunotherapy are analyzed using the PhenoCycler multiplex imaging platform, deploying a comprehensive 50-marker panel. This approach enables deep spatial and phenotypic profiling of adaptive immune response, including T-cell priming pathways. We quantify immune cell density, diversity, and spatial organization, with a focus on cellular neighborhood analysis. In addition, we employ an innovative tracking method to identify anti-PD1-bound cells, providing direct insights into the pharmacodynamic effects of PD-1 blockade within TLDNs.



 To date, we have analyzed TDLNs from four stage II TNBC patients - two with complete pathologic response (pCR, residual cancer burden (RCB)=0) and two with residual disease (non-pCR, RCB=1). Preliminary results reveal a trend toward higher frequency of CD8+ (2.7% vs. 1.4% of CD45+ cells) and CD4+ T cells (19.9% vs. 8.5% of CD45+ cells) within the paracortical regions of pCR patients. Furthermore, the composition of CD8+ T cells differed, with increased proportions of precursor-exhausted (Tpex) (16.0% vs. 2.7% of CD8+ cells), and terminally exhausted (Tex) (10.2% vs. 1.1% of CD8+ cells) subsets observed in pCR cases. Ongoing analyses include mapping of key immune populations, such as plasmacytoid dendritic cells (pDC), conventional dendritic cells (cDC1 and cDC2), Th1, Th2, and T follicular helper (Tfh) cells, allowing us to compute cell-to-cell distances and generate individualized neighborhood profiles to identify spatial signatures associated with treatment response.



 Our high dimensionality spatially resolved analysis provides a powerful window into the immunological landscape of TDLNs in TNBC. By preserving tissue architecture and capturing cellular interactions, we aim to unravel how TDLNs contribute to effective anti-tumor immunity or harbor resistance in the context of chemoimmunotherapy. Preliminary findings suggest distinct immune patterns in patients achieving pCR vs. those with residual disease. A larger cohort, including comprehensive spatial and activation marker analysis, will be presented at the conference.



 A. Oseledchyk, P. Caloba, S. Uzun, K. Schaeuble, S. Münst, W. P. Weber, C. Kurzeder, M. Vetter, H. Läubli, M. Binder, A. Zippelius. Immune Landscape in Tumor Draining Lymph Nodes after Neoadjuvant Chemoimmunotherapy in Early Triple Negative Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS3-12-24.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/484ea8a9c681c0b12ed813fd6154065d8766aa2e" target='_blank'>
              Abstract PS3-12-24: Immune Landscape in Tumor Draining Lymph Nodes after Neoadjuvant Chemoimmunotherapy in Early Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            A. Oseledchyk, P. Caloba, S. Uzun, K. Schaeuble, S. Münst, W. Weber, C. Kurzeder, M. Vetter, H. Läubli, M. Binder, A. Zippelius
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b9f75eab0a2dc55023aa28038ca3efa3f689d5e" target='_blank'>
              Tumor ecosystem and microbiome features associated with efficacy and resistance to avelumab plus chemoradiotherapy in head and neck cancer
              </a>
            </td>
          <td>
            N. Riaz, T. Alban, R. Haddad, Michelle Saul, Vladimir Makarov, Yingjie Zhu, E. Cohen, R. Ferris, Peter Mu-Hsin Chang, J. Lin, Amanda Psyrri, P. Parthasarathy, Ardijana Novaj, M. Gawali, Douglas R. Hoen, Phineas Hamilton, Natalie L. Silver, Ivan Juric, Daniel Chawla, Ana Gradissimo, J. Ko, Daniel J. McGrail, Craig B Davis, Nancy Y. Lee, Timothy A. Chan
          </td>
          <td>2026-01-01</td>
          <td>Nature Cancer</td>
          <td>2</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7babb3ccfccbcaf7965ea70428f827ecffbeb95e" target='_blank'>
              Targeting pre-existing club-like cells in prostate cancer potentiates androgen deprivation therapy.
              </a>
            </td>
          <td>
            M. Baurès, Anne-Sophie Vieira Aleixo, E. Pacreau, Aysis Koshy, Vanessa Friedrich, M. Diedisheim, Martin Raigel, Yichao Hua, Charles Dariane, F. Boutillon, Lukas Kenner, J. Marine, Gilles Laverny, Daniel Metzger, Florian Rambow, Jacques-Emmanuel Guidotti, Vincent Goffin
          </td>
          <td>2026-02-02</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Emerging evidence underscores biophysical characteristics of cancer cells as key modulators of cancer progression and metastasis. Herein, we reported a cell-mechanophenotyping screening microfluidic chip (termed LesM) for the high-efficient capture of circulating tumor cells (CTCs) and evaluation of single-cell deformation to reveal the hematogenous metastatic potential of bacteria-infected breast cancer. LesM employs L-shaped traps to capture single cells, leveraging bacteria-infected CTCs with cytoskeletal reorganization traverse narrowed channels while rigid native cells are retained. The platform demonstrates an average single-cell capture efficiency of 95.42% and specificity of 85.34% in discriminating infected versus non-infected breast cancer cells, validated through parallel in vivo metastatic assays. LesM enables high-throughput sensing up to 10,240 cells of mechanical signatures and microbial cargo, correlating with metastatic risk and antibiotic response. By bridging biomechanics and intratumoral microbiota detection, LesM offers a transformative liquid biopsy tool for predicting distant metastasis and guiding antimicrobial therapies in bacteria-infected breast cancers. Emerging evidence underscores biophysical characteristics of cancer cells as key modulators of cancer metastasis. Here, the authors reported a single-cell mechanophenotyping chip that screens deformable CTCs to reveal the hematogenous metastatic potential of bacteria-infected breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/517f57f9e6c335d60cb5a3e0fde4a76539819f12" target='_blank'>
              A Mechanophenotyping chip for high-throughput detection of metastatic bacteria-infected circulating tumor cells
              </a>
            </td>
          <td>
            Wenxin Luo, Yanfeng Gao, Shujun Feng, Bangshun He, Jingjing Li, Jingjing Yang, Yi Yin, Meng Wang, Bin Xue, Yi Cao, Tony Y. Hu, Yujun Song
          </td>
          <td>2026-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Fibroblast-enriched transcriptional signatures in colorectal cancer (CRC) associate with poor prognosis and with mismatch repair–proficient (MMRp) disease, which derives limited benefit from immune checkpoint inhibition (ICI). We hypothesized that fibroblast composition and spatial organization differ between MMRp and mismatch repair–deficient (MMRd) tumors and define microenvironments relevant to ICI resistance.



 Fresh primary tumors from 41 CRC patients (MMRp n=34, MMRd n=7) underwent scRNA-seq. Matched FFPE sections and an independent archival FFPE cohort (MMRp n=50, MMRd n=30) were profiled by multiplex whole-slide imaging (up to 1.2×106 cells/image). Fibroblast subtypes were defined by scRNA-seq analysis and mapped to images using HALO classifiers on eight-marker panels. Spatial profiling quantified cell densities at regular intervals from the tumor surface to the invasive edge and modeled focal aggregates using a density-based clustering algorithm.



 scRNA-seq data resolved 5 fibroblast subtypes conserved across MMRp/MMRd: myofibroblastic cancer-associated fibroblasts (myCAFs; fibrillar ECM genes), inflammatory CAFs (iCAFs; cytokine genes), and 3 normal fibroblast (NF) subtypes—subepithelial myofibroblasts (SEMFs; contractility genes), lamina propria fibroblasts (LPFs; growth factor activity genes), and crypt fibroblasts (CFs; fibrillar and non-fibrillar ECM genes). CAF densities were enriched at the invasive margin directly bordering the tumor core versus distal margin: 4.6-fold in MMRp (p<0.01) and 3.5-fold in MMRd (p<0.01), defining a CAF-enriched niche. In this niche, CAFs formed focal aggregates, within which CAF composition diverged by MMR status. In MMRp tumors, myCAFs were 16-fold more prevalent than iCAFs (p<0.01); in MMRd tumors, iCAFs were 3-fold more prevalent than myCAFs (p<0.03). Higher myCAF:iCAF ratios were correlated with lower CD8+ tumor-infiltrating lymphocyte (TIL) density in the tumor core (p<0.05), but not CD4+ TILs. NFs were collectively as abundant as CAFs in the tumor stroma, with MMR-dependent spatial patterns. In MMRp tumors, LPFs were 3-fold more prevalent within the CAF-niche versus the deeper invasive margin (p<0.01), whereas CF peak densities were observed in deeper invasive margin. In both MMR settings, SEMFs localized closest to tumor cells and in MMRd, SEMFs were 2-fold more prevalent in the niche (p<0.05). CF prevalence was inversely correlated with CD4+ TIL density in the tumor core (p<0.02), but not CD8+ TILs.



 CRC harbors MMR-dependent fibroblast niches characterized by focal CAF aggregates and distinct NF localization. MMRp tumors exhibit myCAF-dominant aggregates at the tumor core border, whereas MMRd tumors show iCAF-dominant aggregates and SEMF enrichment. Our data are consistent with previous reports of immunosuppressive microenvironments in MMRp CRC, and correlate CAF and NF spatial programs to lymphocyte infiltration.



 Debanjan Barua, Tze Guan Tan, Shu Wen Samantha Ho, Michael T. Wong, Iain B. Tan, Jennifer H. Yearley, Su-Yang Liu. MMR-stratified spatial programs of normal and cancer-associated fibroblasts and their association with lymphocyte pervasiveness in CRC [abstract]. In: Proceedings of the AACR Immuno-Oncology Conference (AACR IO): Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2026 Feb 18-21; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2026;14(2 Suppl):Abstract nr A066.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65875961b8404e9726c9dc3a20e2c3991d815d12" target='_blank'>
              Abstract A066: MMR-stratified spatial programs of normal and cancer-associated fibroblasts and their association with lymphocyte pervasiveness in CRC
              </a>
            </td>
          <td>
            Debanjan Barua, T. G. Tan, Shu Wen Samantha Ho, , I. Tan, J. Yearley, Suyun Liu
          </td>
          <td>2026-02-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0896191e666aebe8cdb170bf3f9299b30b002f" target='_blank'>
              The fundamental role of liquid biopsy in lung cancer: a shift in paradigm in the realm of thoracic oncology
              </a>
            </td>
          <td>
            Shaivy Malik, S. Zaheer
          </td>
          <td>2026-01-16</td>
          <td>Surgical and Experimental Pathology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer metastasis is a major contributor to patient morbidity and mortality1, yet the factors that determine the organs where cancers can metastasize are incompletely understood. Here we quantify the absolute levels of 124 metabolites in multiple tissues in mice and investigate how this relates to the ability of breast cancer cells to grow in different organs. We engineered breast cancer cells with broad metastatic potential to be auxotrophic for specific nutrients and assessed their ability to colonize different tissue sites. We then asked how tumour growth in different tissues relates to nutrient availability and tumour biosynthetic activity. We find that single nutrients alone do not define the sites where breast cancer cells can grow as metastases. In addition, we identify purine synthesis as a requirement for tumour growth and metastasis across many tissues and find that this phenotype is independent of tissue nucleotide availability or tumour de novo nucleotide synthesis activity. These data suggest that a complex interplay between multiple nutrients within the microenvironment dictates potential sites of metastatic cancer growth, and highlights the interdependence between extrinsic environmental factors and intrinsic cellular properties in influencing where breast cancer cells can grow as metastases. How the complex interplay between multiple nutrients within the microenvironment dictates potential sites of metastatic cancer growth is explored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255544370c965d4ce200aa6041ef19b9965ad76a" target='_blank'>
              Nutrient requirements of organ-specific metastasis in breast cancer
              </a>
            </td>
          <td>
            Keene L. Abbott, S. Subudhi, Raphael Ferreira, Yetiş Gültekin, Sophie C. Steinbuch, Muhammad Bin Munim, Diya L Ramesh, Sophie E Honeder, Ashwin S. Kumar, Michelle Wu, Jacob A. Hansen, Anna Shevzov-Zebrun, Edrees H. Rashan, K. M. Eghbalian, S. Sivanand, Anna M. Barbeau, Lisa M Riedmayr, Mark Duquette, Ahmed Ali, Nicole Henning, Sonia E Trojan, Millenia Waite, Tenzin Kunchok, Mayu A. Nakano, Florian Gourgue, G. Ferraro, Brian T. Do, Virginia Spanoudaki, Francisco J. Sánchez-Rivera, Xin Jin, George M. Church, Rakesh K Jain, M. V. Vander Heiden
          </td>
          <td>2026-01-07</td>
          <td>Nature</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="Programmed death-ligand 1 (PD-L1) has become a central biomarker and therapeutic target across multiple solid tumors, yet its clinical meaning in prostate cancer (PCa) remains unsettled. PCa is commonly described as an immunologically ‘cold’ malignancy, characterized by limited baseline cytotoxic T-cell infiltration and a tumor microenvironment (TME) shaped by myeloid-driven suppression and low neoantigen load in many cases. Against this background, PD-L1 expression in PCa is typically low in untreated primary tumors but can increase in aggressive variants, advanced stages, and metastatic castration-resistant disease, where therapy pressure and microenvironmental cues may select for immune-evasive phenotypes. The literature is further complicated by major analytic variability, including differences in antibody clones and platforms, scoring algorithms (tumor proportion score, combined positive score, immune-cell scoring), cut-offs, tissue sites and timing, and pre-analytical variables such as fixation and decalcification. Collectively, available studies suggest that higher PD-L1 expression tends to be associated with adverse clinicopathological features and may enrich for responses to immune checkpoint inhibitors in selected settings, but PD-L1 immunohistochemistry alone is insufficient as a stand-alone predictive tool in unselected patients. This review synthesizes the biological drivers of PD-L1 regulation in PCa, dissects key methodological sources of heterogeneity in PD-L1 assessment, summarizes clinicopathological and therapeutic correlations, and outlines emerging biomarkers and approaches (including mismatch repair deficiency/microsatellite instability, tumor mutational burden, gene-expression signatures, liquid biopsies, and neuro-immune interactions) that may enable more actionable patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d357efeaff0e6f43eff4b0c26c198baeace581a" target='_blank'>
              PD-L1 Expression in Prostate Cancer: Anatomopathological Features, Methodological Pitfalls, and Therapeutic Potential
              </a>
            </td>
          <td>
            Ludovica Pepe, C. Pizzimenti, Pietro Tralongo, Valeria Zuccalà, A. Ieni, Pietro Pepe, Gabriele Ricciardi, V. Cianci, Cristina Mondello, Maurizio Martini, Guido Fadda, V. Fiorentino
          </td>
          <td>2026-02-13</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c21032e04b711532f2c1af652faef7052c5a06e" target='_blank'>
              PanCNV-Explorer: Deciphering copy number alterations across human cancers
              </a>
            </td>
          <td>
            Nadine S. Kurz, Kevin Kornrumpf, Anna-Rosa Krüger, Jürgen Dönitz
          </td>
          <td>2026-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/105eb18c0e05b0acf1b6bd49a6cb6163c7165afd" target='_blank'>
              Outlier Detection in Single-Cell Transcriptomics Reveals Disease-Enriched Cytotoxic Immune Populations
              </a>
            </td>
          <td>
            Devanshi Gupta, Arsh Gupta
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Endometriosis is a complex condition affecting 10% of reproductive-age women worldwide, yet its study has long been hindered by the shortcomings of traditional research models. This review aims to propose a novel theoretical framework and strategic roadmap to accelerate precision medicine in endometriosis. We systematically trace the evolution of research models in the field and critically assess the inherent constraints of conventional systems, including immortalized cell lines and primary cell cultures. While immortalized lines often fail to capture patient heterogeneity due to genetic drift, animal models cannot fully replicate the human immune microenvironment. Although primary cells preserve patient-specific traits and support personalized drug testing, their two-dimensional (2D) culture format limits their ability to mimic the three-dimensional (3D) architecture and dynamic cell-cell interactions of native tissues. Recent breakthroughs in organoid technology are reshaping endometriosis research by enabling 3D structural reconstruction, hormone responsiveness, and simulation of lesion progression. This review highlights the transformative potential of organoids in disease modeling, drug screening, and personalized therapy. We also discuss future opportunities for integrating organoids with emerging platforms-such as organ-on-a-chip systems, multi-omics analyses, and digital twin technology-to open new avenues for understanding disease mechanisms and advancing targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2df2be6887144c2a0f3dc1971d0de91004e91ed" target='_blank'>
              The evolution of endometriosis research models: a paradigm shift from immortalized cell lines to organoids†.
              </a>
            </td>
          <td>
            Jie Fang, Yaping Yang, Yujia Xiao, Peng Lü
          </td>
          <td>2026-01-19</td>
          <td>Biology of reproduction</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d837933c654c6e002d876d213914f511b6ba26a2" target='_blank'>
              Ex Vivo Assay for Organ-Specific Cancer Cell Invasion
              </a>
            </td>
          <td>
            François Tyckaert, Paul Frieso Göddertz, Maria Reichhold, Bettina Sarg, K. Faserl, Pere Patón González, F. Eichin, A. Villunger, S. Ormanns, Stefan Redl, J. Hofmann, T. Hautz, Francesco Baschieri
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Despite the high prevalence of prostate cancer (PC), there are relatively few cell lines and they are not sufficient to cover the genetic and phenotypic diversity of the disease. Patient derived xenografts (PDX) provide a useful toolset for studying human PC, but cell line models are still important assets for their ease of use, including genetic engineering and scalability for large-scale screens. We adapted the LuCaP189.4 PDX into a 2D cell line, called LuCaP189.4_CL, and characterized its genetic and phenotypic features and suitability as an experimental model. Of particular note, this line carries natural CDK12 bi-allelic loss (BAL), making it an especially useful for understanding this aggressive disease subtype that accounts for 5-7% of metastatic castration resistant PC.



 LuCaP189.4 PDX tumors were serially passaged to adapt to common cell culture conditions. DNA was analyzed by karyotyping and exome sequencing. Growth rates of the cell line were measured at different passages and RNA was collected for transcriptome analysis. Cells were also grown in 3D culture conditions and re-implanted into mice to generate cell-line derived xenograft tumors (CDX). LNCaP_FGC and NCI-H660 were also grown as cell lines, 3D, or CDX and analyzed by RNA-seq for comparison. LuCaP 189.4_CL was immunoblotted for key tumor suppressors and immunostained for lineage markers. Cells were tested for sensitivity to common PC therapies and additional genomic characteristics were assayed by enzymatic methyl DNA sequencing and S9.6 R-loop chromatin profiling.



 The LuCaP189.4_CL maintained key characteristics of the PDX model, including high AR activity and AR splice variant expression and loss of CDK12. LuCaP189.4_CL cells are smaller than LNCaP_FGC or 22Rv1 and growth rate is partially density dependent. The line is mostly diploid and amenable to lentiviral transduction at roughly one quarter efficiency relative to LNCaP_FGC. LuCaP189.4_CL successfully established CDX tumors when implanted in mice (11/12 take rate). DNA mutations and tandem duplicator phenotype (TDP), a hallmark of CDK12 loss, were matched between the cell line and PDX. LuCaP189.4_CL also exhibits genomic patterns of high DNA methylation and R-loop activity. LuCaP189.4_CL did not respond to enzalutamide or supraphysiologic androgen and showed less sensitivity to carboplatin, olaparib, and docetaxel compared to LNCaP_FGC and 22Rv1. However, the line showed enhanced sensitivity to the RNA Pol2 inhibitor α-amanitin.



 The LuCaP189.4_CL model is a new cell line that retains key characteristics of the PDX and human mCRPC, including high AR activity and CDK12 loss. Transcriptomic analysis of various culture conditions is ongoing. This line is not sensitive to most standard therapies, concordant with observations in patients with CDK12-BAL, and exhibits rare genomic features of DNA methylation, TDP, and high R-loop activity. This cell line adds to the genetic and phenotypic representation of PC cell line models with the potential to enhance future PC research.



 Sander Frank, Dapei Li, Xiulan Yang, Tarana Arman, Galina Semenova, Armand Bankhead, Brian Hanratty, Jessica Maruwan, Emily Distler, Jared Lucas, Ruth Dumpit, Ilsa Coleman, Reza A. Ghodsi, Colm Morrissey, Eva Corey, Michael Haffner, Peter S. Nelson. LuCaP189.4_CL: a novel 2D model of AR -high, CDK12 -mutant mCRPC suitable for studies of genetic and therapeutic vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02fcc72f36d1b5bed1daab9b8ec31ebe9efa8102" target='_blank'>
              Abstract B022: LuCaP189.4_CL: a novel 2D model of
 AR
 -
 high,
 CDK12
 -
 mutant mCRPC suitable for studies of genetic and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Sander Frank, Dapei Li, Xiulan Yang, Tarana Arman, Galina Semenova, Armand Bankhead, B. Hanratty, Jessica Maruwan, Emily Distler, Jared M. Lucas, R. Dumpit, Ilsa M Coleman, Reza A. Ghodsi, C. Morrissey, Eva Corey, M. Haffner, Peter S. Nelson
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/173cc4555c30056ffb452d93c67137c372b260db" target='_blank'>
              Gene duplication is associated with gene diversification and potential neofunctionalization in lung cancer evolution.
              </a>
            </td>
          <td>
            Paul Ashford, A. Frankell, Zofia Piszka, Camilla S. M. Pang, M. Abbasian, Maise Al Bakir, M. Jamal-Hanjani, N. Mcgranahan, C. Swanton, Christine A. Orengo
          </td>
          <td>2026-02-19</td>
          <td>Genome research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3964d291bb3bd085a83a8194679d62bcd5ce5f5e" target='_blank'>
              Substantial genomic and methylation variability between MCF-7 sublines
              </a>
            </td>
          <td>
            H. Atanda, Adam D Ewing
          </td>
          <td>2026-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Immune checkpoint blockade (ICB) has reshaped cancer treatment, yet only a minority of patients respond. This underscores the need for robust biomarkers capable of broadly identifying tumors with high immune activation. Existing immune gene signatures and transcriptomic predictors often lack generalizability across cancer types and independent cohorts. To overcome this limitation, we developed TIME_ACT, an unsupervised 66-gene signature that quantifies tumor immune activation (“hotness”) by integrating signals of immune infiltration, inflammatory activity, and spatial TIL density. TIME_ACT provides a consistent, pan-cancer measure of immune activity and is highly inferable from pathology slides, enabling a low-cost and widely accessible approach for predicting tumor hotness and ICB response using routine histopathology.



 We first evaluated TIME_ACT in 22 publicly available pre-treatment ICB transcriptomic cohorts (n=1,416) across seven cancer types. We then combined TIME_ACT with Path2Omics, a recently published deep-learning model that imputes gene expression from pathology slides, enabling direct prediction of ICB response from pathology. This integrated framework, Path2Omics-TIME_ACT, was tested across nine external treatment cohorts (n=459).



 Using measured gene expression data, TIME_ACT accurately distinguished immune-hot versus immune-cold tumors with AUCs above 0.96 in all 10 evaluated cohorts, and achieved a mean AUC of 0.74 and a mean odds ratio of 5.49 across 22 transcriptomic cohorts, outperforming 26 established ICB response signatures and methods. To assess slide-based prediction, we evaluated Path2Omics performance on TIME_ACT genes. Strikingly, Path2Omics achieved strong predictive fidelity, with a median correlation of 0.61 between actual and imputed gene expression values across 24 cancer types. Using the slide-inferred TIME_ACT scores, Path2Omics-TIME_ACT achieved performance nearly equivalent to that obtained with measured gene expression, with AUCs above 0.94 in all 8 evaluated cohorts for hot/cold classification, and a mean AUC of 0.73 and a mean odds ratio of 5.07 across nine histopathology cohorts for predicting ICB response. Notably, Path2Omics was trained on TCGA data and applied to external treatment cohorts without further fine-tuning, and TIME_ACT is unsupervised learning approach, alleviating concerns about overfitting.



 Path2Omics-TIME_ACT is a robust pan-cancer predictor of tumor immune hotness and ICB response directly from routine pathology slides. With further prospective testing and validation, this framework could significantly broaden access to precision immunotherapy.



 Sumit Mukherjee, Sumeet Patiyal, Lipika R. Pal, Tian-Gen Chang, Sumona Biswas, Saugato Rahman Dhruba, Amos Stemmer, Arashdeep Singh, Tien-Hua Chen, Binbin Wang, Denis Marino, Wonwoo Shon, Yuan Yuan, Mark Faries, Omid Hamid, Karen Reckamp, Keluo Yao, Sabina Signoretti, David A. Braun, Joo Sang Lee, Hyunjeong Joo, Hyungsoo Kim, Jair Bar, Iris Barshack, Chi-Ping Day, Sridhar Hannenhalli, Karine Sargsyan, Andrea B. Apolo, Kenneth Aldape, Muh-Hwa Yang, Michael B. Atkins, Ze’ev A. Ronai, Danh-Tai Hoang, Eytan Ruppin. Path2Omics–TIME_ACT: Pan-cancer prediction of tumor hotness and immune checkpoint blockade response from pathology slides [abstract]. In: Proceedings of the AACR Immuno-Oncology Conference (AACR IO): Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2026 Feb 18-21; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2026;14(2 Suppl):Abstract nr B061.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e3cefd219318bcf543d273143ee2cb32e29ef6f" target='_blank'>
              Abstract B061: Path2Omics–TIME_ACT: Pan-cancer prediction of tumor hotness and immune checkpoint blockade response from pathology slides
              </a>
            </td>
          <td>
            Sumit Mukherjee, Sumeet Patiyal, L.R. Pal, , Sumona Biswas, S. R. Dhruba, Amos Stemmer, Arashdeep Singh, , Binbin Wang, D. Marino, Wonwoo Shon, Yuan Yuan, M. Faries, Omid Hamid, K. Reckamp, Keluo Yao, S. Signoretti, David A. Braun, Joo Sang Lee, Hyunjeong Joo, Hyungsoo Kim, J. Bar, Iris Barshack, Chi-Ping Day, Sridhar Hannenhalli, K. Sargsyan, Andrea Apolo, Kenneth D. Aldape, Muh-Hwa Yang, M. Atkins, Ze’ev A. Ronai, Danh-Tai Hoang, E. Ruppin
          </td>
          <td>2026-02-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>92</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [7, 18],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>