<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-04-28 10:54:10 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Lung squamous cell carcinoma (LUSC) is basal-like subtype of lung cancer with limited treatment options. While prior studies have identified tumor-propagating cell states in squamous tumors, the broader landscape of intra-tumoral heterogeneity within LUSC remains poorly understood. Here, we employ Sox2-driven mouse models, organoid cultures, and single-cell transcriptomic analyses to uncover previously unrecognized levels of cell fate diversity within LUSC. Specifically, we identify a KRT13+ hillock-like population of slower-dividing tumor cells characterized by immunomodulatory gene expression signatures. The tumor hillock-like state is conserved across multiple animal models and is present in the majority of human LUSCs as well as head and neck and esophageal squamous tumors. Our findings shed light on the cellular origins of lung hillock-like states: normal club cells give rise to tumors with luminal hillock-like populations, while basal-like tumor-propagating cells transition into basal hillock-like states, resembling homeostatic cellular responses to lung injury. Mechanistically, we identify KLF4 as a key transcriptional regulator of the hillock-like state, both necessary and sufficient to induce KRT13 expression. Together, these results provide new molecular insights into cell fate plasticity that underlies intra-tumoral heterogeneity in LUSC, offering potential avenues for new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec0519121b0ddc09d45dd5bfdeca6cc730f82cfc" target='_blank'>
              KLF4 promotes a KRT13+ hillock-like state in squamous lung cancer
              </a>
            </td>
          <td>
            Luke T. Izzo, Tony Reyes, Srijan Meesala, A. Ireland, Steven Yang, Hari Shankar Sunil, Xiao Chun Cheng, Nomi Tserentsoodol, Sarah B. Hawgood, Edward F. Patz, Benjamin L. Witt, Darren R Tyson, Kathryn A. O’Donnell, T. Oliver
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="We conducted a spatial analysis of stage III colorectal adenocarcinomas using Hyperion Imaging Mass Cytometry, examining 52 tumors to assess the tumor microenvironment at the single-cell level. This approach identified 10 distinct cell phenotypes in the tumor microenvironment, including stromal and immune cells, with a subset showing a proliferative phenotype. By focusing on spatial neighborhood interactions and tissue niches, particularly regions with tumor-infiltrating lymphocytes, we investigated how cellular organization relates to clinicopathological and molecular features such as microsatellite instability (MSI) and recurrence. We determined that microsatellite stable (MSS) colorectal cancers had an increased risk of recurrence if they had the following features: 1) a low level of stromal tumor-infiltrating lymphocytes, and 2) low interactions between CD4 + T cells and stromal cells. Our results point to the utility of spatial single-cell interaction analysis in defining novel features of the tumor immune microenvironments and providing useful clinical cell-related spatial biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a63f7d3dd84bf14bb59b2621b8ec0ed0c881cf8" target='_blank'>
              The single-cell spatial landscape of stage III colorectal cancers
              </a>
            </td>
          <td>
            Andrew Su, Hojoon Lee, Minh Tran, Richard C Dela Cruz, A. Sathe, Xiangqi Bai, Ignacio A Wichmann, Lance Pflieger, Bryce Moulton, Tyler Barker, D. Haslem, David Jones, Lincoln Nadauld, Quan H Nguyen, , Terence Rhodes
          </td>
          <td>2025-04-07</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Spatial cellular context is crucial in shaping intratumor heterogeneity. However, understanding how each tumor establishes its unique spatial landscape and what factors drive the landscape for tumor fitness remains significantly challenging. Here, we analyzed over 2 million cells from 50 tumor biospecimens using spatial single-cell imaging and single-cell RNA sequencing. We developed a deep learning-based strategy to spatially map tumor cell states and the architecture surrounding them, which we referred to as Spatial Dynamics Network (SDN). We found that different tumor cell states may be organized into distinct clusters, or ‘villages’, each supported by unique SDNs. Notably, tumor cell villages exhibited village-specific molecular co-dependencies between tumor cells and their microenvironment and were associated with patient outcomes. Perturbation of molecular co-dependencies via random spatial shuffling of the microenvironment resulted in destabilization of the corresponding villages. This study provides new insights into understanding tumor spatial landscape and its impact on tumor aggressiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d9a8ffa548a2f4ea069cb9bc5e6205adbb07d1" target='_blank'>
              Tumor cell villages define the co-dependency of tumor and microenvironment in liver cancer
              </a>
            </td>
          <td>
            Meng Liu, M. Hernandez, Darko Castven, Hsin-Pei Lee, Wenqi Wu, Limin Wang, M. Forgues, Jonathan M. Hernandez, Jens U. Marquardt, Lichun Ma
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e484c7784673f4b5393c2f4419927f4ebb20abef" target='_blank'>
              Mapping the spatial architecture of glioblastoma from core to edge delineates niche-specific tumor cell states and intercellular interactions
              </a>
            </td>
          <td>
            Saad M. Khan, Anthony Z. Wang, Rupen Desai, Colin R. McCornack, Rui Sun, Sonika M. Dahiya, Jennifer A. Foltz, Ngima D Sherpa, Lydia Leavitt, T. West, Alexander F. Wang, Aleksandar Krbanjevic, Bryan D. Choi, Eric Leuthardt, B. Patel, Al Charest, Albert H. Kim, Gavin P. Dunn, Allegra A. Petti
          </td>
          <td>2025-04-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer deaths; nonetheless, how tumor cells adapt to vastly different organ contexts is largely unknown. To investigate this question, we generated a transcriptomic atlas of primary tumor and diverse metastatic samples from a patient with pancreatic ductal adenocarcinoma who underwent rapid autopsy. Unsupervised archetype analysis identified both shared and site-specific gene programs, including lipid metabolism and gastrointestinal programs prevalent in peritoneum and stomach wall lesions, respectively. We developed a probabilistic approach for inferring clonal phylogeny from single-cell and matched whole-exome data. Distantly related genetic clones in the peritoneum express the lipid metabolism program, likely due to signaling by the adipocyte-rich peritoneum environment, and cells in most clones express multiple programs, suggesting that transcriptomic plasticity is a prevalent feature of metastatic cells. These deeply annotated analyses using a patient-centric platform provide a model for investigating metastatic mechanisms and plasticity in advanced cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b77f53eedac93ceda8af4b898f6071abded0e4" target='_blank'>
              Transcriptomic plasticity is a hallmark of metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Alejandro Jiménez-Sánchez, S. Persad, Akimasa Hayashi, Shigeaki Umeda, Roshan Sharma, Yubin Xie, A. Mehta, Wungki Park, I. Masilionis, T. Chu, Feiyang Zhu, Jungeui Hong, R. Chaligné, Eileen M. O’Reilly, L. Mazutis, T. Nawy, I. Pe’er, Christine A. Iacobuzio-Donahue, D. Pe’er
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a crucial role in orchestrating tumor cell behavior and cancer progression. Recent advances in spatial profiling technologies have uncovered novel spatial signatures, including univariate distribution patterns, bivariate spatial relationships, and higher-order structures. These signatures have the potential to revolutionize tumor mechanism and treatment. In this review, we summarize the current state of spatial signature research, highlighting computational methods to uncover spatially relevant biological significance. We discuss the impact of these advances on fundamental cancer biology and translational research, address current challenges and future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed05a90f6b1d7a182ae84bb7e554725c91c44235" target='_blank'>
              Quantifying and interpreting biologically meaningful spatial signatures within tumor microenvironments
              </a>
            </td>
          <td>
            Si Jing, He-Qi Wang, Ping Lin, Jiao Yuan, Zhixuan Tang, Hong Li
          </td>
          <td>2025-03-11</td>
          <td>NPJ Precision Oncology</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dada77c78506ee5d9133975164f4c4e0e0e7b027" target='_blank'>
              Prediction of Outcome from Spatial Protein Profiling of Triple-Negative Breast Cancers
              </a>
            </td>
          <td>
            Ali Foroughi pour, Te-Chia Wu, Jan Martinek, S. Sivajothi, Paul Robson, Karolina Palucka, Jeffrey H. Chuang
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL) therapy, recently approved by the FDA for melanoma, is an emerging modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors such as glioblastoma remains limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas using spectral flow cytometry, TCR sequencing, single-cell RNA-seq, Xenium in situ transcriptomics, and CODEX spatial proteomics. Comparative analysis of TIL-generating (TIL⁺) versus non-generating (TIL⁻) tumors revealed that IL7R expression, structured perivascular immune clustering, and tumor-intrinsic metabolic programs such as ACSS3 were associated with successful TIL expansion. In contrast, TIL⁻ tumors were enriched for neuronal lineage signatures, immunosuppressive transcripts including TOX and FERMT1, and tumor-connected macrophages. This study defines spatial and molecular correlates of TIL manufacturing success and establishes a genomics-enabled selection platform for adoptive T cell therapy. The profiling approach is now being prospectively implemented in the GIANT clinical trial (NCT06816927), supporting its translational relevance and scalability across glioblastoma and other immune-excluded cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/945f91b27ecd7bfb796d6743f706c6451c1e2314" target='_blank'>
              A Spatial Multi-Omic Framework Identifies Gliomas Permissive to TIL Expansion
              </a>
            </td>
          <td>
            K. Hotchkiss, Kenan Zhang, Anna M. Corcoran, Elizabeth Owens, P. Noldner, C. Railton, Kyra Van Batavia, Ying Zhou, Jodie Jepson, Kirit Singh, R. McLendon, Kristen Batich, Anoop P Patel, Katayoun Ayasoufi, Michael Brown, Evan Calabrese, Jichun Xie, Jose R. Conejo-Garcia, B. Shaz, John W. Hickey, Mustafa Khasraw
          </td>
          <td>2025-04-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98801c9770ab85a801e0ca53d0db3280fc312fc" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma invasion into healthy brain tissue remains a major barrier to effective treatment, yet current models fail to capture its full complexity in a scalable and patient-specific manner. Here, we introduce GlioTrace, a novel ex vivo imaging and AI-based analytical framework that enables real-time, spatiotemporal tracking of glioblastoma invasion dynamics in patient-derived glioma cell culture xenograft (PDCX) brain slices. By integrating whole-specimen confocal microscopy, vascular counterstaining, and an advanced computational pipeline combining convolutional neural networks and Hidden Markov Models, GlioTrace identifies distinct invasion modes, including dynamic morphological switching, vessel-guided migration, and immune cell interactions and quantifies patient-specific variations in invasion plasticity. Using GlioTrace, we demonstrate that targeted therapies can selectively modulate invasion phenotypes, revealing spatially and temporally distinct drug responses. This scalable platform provides an unprecedented window into glioblastoma progression and treatment response, offering a powerful tool for precision oncology and anti-invasion therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d85bb045bd5b47db33273aef84595c6d869e5e" target='_blank'>
              Reconstructing the Single-Cell Spatiotemporal Dynamics of Glioblastoma Invasion
              </a>
            </td>
          <td>
            H. Mangukiya, Madeleine Skeppås, Soumi Kundu, Maria Berglund, Adam A. Malik, C. Krona, Sven Nelander
          </td>
          <td>2025-03-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tumor subclones refer to distinct cell populations within the same tumor that possess different genetic characteristics. They play a crucial role in understanding tumor heterogeneity, evolution, and therapeutic resistance. The formation of tumor subclones is driven by several key mechanisms, including the inherent genetic instability of tumor cells, which facilitates the accumulation of novel mutations; selective pressures from the tumor microenvironment and therapeutic interventions, which promote the expansion of certain subclones; and epigenetic modifications, such as DNA methylation and histone modifications, which alter gene expression patterns. Major methodologies for studying tumor subclones include single-cell sequencing, liquid biopsy, and spatial transcriptomics, which provide insights into clonal architecture and dynamic evolution. Beyond their direct involvement in tumor growth and invasion, subclones significantly contribute to tumor heterogeneity, immune evasion, and treatment resistance. Thus, an in-depth investigation of tumor subclones not only aids in guiding personalized precision therapy, overcoming drug resistance, and identifying novel therapeutic targets, but also enhances our ability to predict recurrence and metastasis risks while elucidating the mechanisms underlying tumor heterogeneity. The integration of artificial intelligence, big data analytics, and multi-omics technologies is expected to further advance research in tumor subclones, paving the way for novel strategies in cancer diagnosis and treatment. This review aims to provide a comprehensive overview of tumor subclone formation mechanisms, evolutionary models, analytical methods, and clinical implications, offering insights into precision oncology and future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1d4c21265c412fc5d4c0775f9aa0ecd36b6c8eb" target='_blank'>
              Advances in tumor subclone formation and mechanisms of growth and invasion
              </a>
            </td>
          <td>
            Yuhong Zhang, Weidong Wang
          </td>
          <td>2025-04-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc63f157070e0fd70ef6717ad423137ac275d75" target='_blank'>
              Stochastically Emergent Tumors offer in vivo whole genome interrogation of cancer evolution from non-malignant precursors
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel B. Hoelscher, V. Luga, Arun A. Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, C.K. Cornelia Ding, Bradley A. Stohr, Peng Jin, Tejasveeta V. Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, C.L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd408ac5f53831aa4c4d85811f6f5969702d07e" target='_blank'>
              High-Dimensional Protein Analysis Uncovers Distinct Immunological and Stromal Signatures Between Primary and Metastatic Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Emily Greene, Natalie K. Horvat, D. Doxie, Vaunita C. Parihar, Jayden Kim, Cameron J. Herting, Erin E. Grundy, Ayana T. Ruffin, Alyssa Krasinskas, Shishir Maithel, Juan M. Sarmiento, Mihir M. Shah, Mohammad Y Zaidi, Maria Diab, O. Alese, Kavita Dhodapkar, H. Kissick, Bassel F. El-Rayes, C. Paulos, Gregory B. Lesinski
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="In this study, we developed an integrated single cell transcriptomic (scRNAseq) atlas of human breast cancer (BC), the largest resource of its kind, totaling > 600,000 cells across 138 patients. Rigorous integration and annotation of publicly available scRNAseq data enabled a highly resolved characterization of epithelial, immune, and stromal heterogeneity within the tumor microenvironment (TME). Within the immune compartment we were able to characterize heterogeneity of CD4, CD8 T cells and macrophage subpopulations. Within the stromal compartment, subpopulations of endothelial cells (ECs) and cancer associated fibroblasts (CAFs) were resolved. Within the cancer epithelial compartment, we characterized the functional heterogeneity of cells across the axes of stemness, epithelial-mesenchymal plasticity, and canonical cancer pathways. Across all subpopulations observed in the TME, we performed a multi-resolution survival analysis to identify epithelial cell states and immune cell types which conferred a survival advantage in both The Cancer Genome Atlas (TCGA) and METABRIC. We also identified robust associations between TME composition and clinical phenotypes such as tumor subtype and grade that were not discernible when the analysis was limited to individual datasets, highlighting the need for atlas-based analyses. This atlas represents a valuable resource for further high-resolution analyses of TME heterogeneity within BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7db0b505659425c4bfdef92aba55ca31e10f8c9" target='_blank'>
              A highly resolved integrated transcriptomic atlas of human breast cancers
              </a>
            </td>
          <td>
            Andrew Chen, Lina Kroehling, Christina S. Ennis, Gerald V. Denis, S. Monti
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b28a229fd99f6a9c5521983f68e3ec9d09a032a" target='_blank'>
              Abstract 6604: Characterizing the tumor microenvironment using spatially barcoded archival FFPE tissue: converting single-nucleus RNA-seq into spatial transcriptomics
              </a>
            </td>
          <td>
            Cedric R. Uytingco, Julie Wilhelmy, Bryant Ngo, Jonathan Sakkos, Wanxin Wang, Christina Chang, Christina Fan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative breast cancer, characterized by aggressive growth and high intratumor heterogeneity, presents a significant clinical challenge. Here, we use a lineage-tracing system, ClonMapper, which couples heritable clonal identifying tags with single-cell RNA-sequencing (scRNA-seq), to better elucidate the response to doxorubicin in a model of TNBC. We demonstrate that, while there is a dose-dependent reduction in overall clonal diversity, there is no pre-existing resistance signature among surviving clones. Separately, we found the existence of two transcriptomically distinct clonal subpopulations that remain through the course of treatment. Among clones persisting across multiple samples we identified divergent phenotypes, suggesting a response to treament independent of clonal identity. Finally, a subset of clones harbor novel changes in expression following treatment. The clone and sample specific responses to treatment identified herein highlight the need for better personalized treatment strategies to overcome tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88360a99c9dba219540b793cdb094157ffa2ec6a" target='_blank'>
              Lineage Tracing Reveals Clone-Specific Responses to Doxorubicin in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Daylin Morgan, Andrea L. Gardner, Amy Brock
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06a703b5eeba2e901b0564ff4995a24e3b880d79" target='_blank'>
              Single-cell spatial analysis of pediatric high-grade glioma reveals a novel population of SPP1+/GPNMB+ myeloid cells with immunosuppressive and tumor-promoting capabilities
              </a>
            </td>
          <td>
            Thijs J M van den Broek, R. Hoogendijk, M. Kranendonk, J. Lammers, Akshaya L. Krishnamoorthy, Ravian L. van Ineveld, Milo Molleson, Vasily O. Tsvetkov, F. Ringnalda, M. van de Wetering, Yan Su, John I. Bianco, Cristian Ruiz Moreno, Mario G Ries, E. Hoving, J. van der Lugt, Leila Akkari, David P. Schrijver, H. Stunnenberg, A. Rios, D. V. van Vuurden, A. Zomer
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffdda1a300bd205aab897ffa9f139e98ec525b59" target='_blank'>
              Abstract 155: Spatial analysis of tumor-immune-stromal cellular phenotypes and microenvironments in pancreatic cancer
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Elizabeth Sunnucks, Sibyl Drissler, Tiak Ju Tan, Chengxin Yu, Jennifer L. Gorman, Michael J Geuenich, Sheng-Ben Liang, , Golnaz Abazari, Barbara Gruenwald, A. Borgida, , Brendon Seale, Zhen Yuan Lin, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Anne-Claude Gringas, Grianne M. O'Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf4d0e7843bbae99e8106885d03d265c924f51c" target='_blank'>
              A unified eIF1A+ luminal cells-centered hypoxic and “cold” tumor microenvironment promotes PCa progression among different subtypes
              </a>
            </td>
          <td>
            Lilin Wan, Enyao Huang, Zicheng Xu, Bing Zheng, Xuefei Ding, Chunyan Chu, Tiange Wu, Shengrong Chen, Junyong Zhuang, Ming Chen, Rong Na, Shenghong Ju, Dingxiao Zhang, Wenchao Li, Yifei Cheng, Bin Xu
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Objectives: Bone metastasis is a frequent and life-threatening event in advanced cancers, affecting up to 70–85% of prostate cancer patients. Understanding the cellular and molecular mechanisms underlying bone metastasis is essential for developing targeted therapies. This study aimed to systematically characterize the heterogeneity and microenvironmental adaptation of prostate cancer bone metastases using single-cell transcriptomics. Methods: We integrated the largest single-cell transcriptome dataset to date, encompassing 124 samples from primary prostate tumors, various bone metastatic sites, and non-malignant tissues (e.g., benign prostatic hyperplasia, normal bone marrow). After quality control, 602,497 high-quality single-cell transcriptomes were analyzed. We employed unsupervised clustering, gene expression profiling, mutation analysis, and metabolic pathway reconstruction to characterize cancer cell subtypes and tumor microenvironmental remodeling. Results: Cancer epithelial cells dominated the tumor microenvironment but exhibited pronounced heterogeneity, posing challenges for conventional clustering methods. By integrating genetic and metabolic features, we revealed key evolutionary trajectories of epithelial cancer cells during metastasis. Notably, we identified a novel epithelial subpopulation, NEndoCs, characterized by unique differentiation patterns and distinct spatial distribution across metastatic niches. We also observed significant metabolic reprogramming and recurrent mutations linked to prostate-to-bone microenvironmental transitions. Conclusions: This study comprehensively elucidates the mutation patterns, metabolic reprogramming, and microenvironment adaptation mechanisms of bone metastasis in prostate cancer, providing key molecular targets and clinical strategies for the precise treatment of bone metastatic prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64291da16ef528fb561e1af0f4b9ad8a8ddde042" target='_blank'>
              Comprehensive Integrated Analysis Reveals the Spatiotemporal Microevolution of Cancer Cells in Patients with Bone-Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Yinghua Feng, Xiuli Zhang, Guangpeng Wang, Fei-ya Yang, Ruifang Li, Lu Yin, Dong Chen, Wenkuan Wang, Mingshuai Wang, Zhiyuan Hu, Yuan Sh, N. Xing
          </td>
          <td>2025-04-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Background The tumor microenvironment (TME) is a complex and dynamic ecosystem that is known to influence responses to immunotherapy. We leveraged single-cell spatial transcriptomics to systematically dissect the intricate complexity of the TME, in particular the cellular heterogeneity and spatial interactions. Their collective impact on immunotherapy efficacy was studied in the context of a homogeneous group of patients with vulvar high-grade squamous intraepithelial lesions (vHSIL) treated with an immunotherapeutic tumor-specific peptide vaccine. Methods We performed single-cell spatial transcriptomics on 20 pretreatment vHSIL lesions, stratified by clinical response to immunotherapeutic vaccination into complete responders (CR), partial responders (PR) and non-responders (NR). Using a 1,000-gene panel, we mapped over 274,000 single cells in situ, identifying 18 cell clusters and 99 distinct non-epithelial cell states. Findings were validated against public single-cell transcriptomic data sets to assess their broader relevance across tumor types. Results Profound heterogeneity within the TME was detected across the response groups. CR lesions exhibited a higher ratio of immune-supportive to immune-suppressive cells—a pattern mirrored in other solid tumors following neoadjuvant checkpoint blockade. Key immune populations enriched in CRs included CD4+CD161+ effector T cells and chemotactic CD4+ and CD8+ T cells. Conversely, PRs were characterized by increased proportions of T helper 2 cells and CCL18-expressing macrophages, which are associated with the recruitment of type 2 T cells and regulatory T cells. NRs displayed preferential infiltration with immunosuppressive fibroblasts. Distinct spatial immune ecosystems further defined response groups. Although a number of immune cells were detected in all patients, type 1 effector cells dominated interactions in CRs, type 2 cells were prominently interacting in PRs, while NRs lacked organized immune cell interactions. Conclusions This study underscores the dual importance of both cellular composition and spatial organization in steering clinical response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b353470be584af7d95e65b2aa9a5dffe4241b32d" target='_blank'>
              Single-cell spatial transcriptomics unravels cell states and ecosystems associated with clinical response to immunotherapy
              </a>
            </td>
          <td>
            Z. Abdulrahman, R. Slieker, Daniel McGuire, M. J. Welters, M. V. van Poelgeest, S. H. van der Burg
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a1265fddcea3943ae47df40af80fa7b2b32f84" target='_blank'>
              Abstract 2663: Mapping clonal heterogeneity and stromal dynamics in prostate cancer
              </a>
            </td>
          <td>
            S. Figiel, Mengxiao He, Wencheng Yin, Emmanouela Perisynaki, Renuka Teague, T. Anbarasan, Eleanor M O'Roberts, N. Ranasinha, Sophia M. Abusamra, Trishna A. Desai, D. Doultsinos, Andrew Erickson, Srinivasa Rao, Clare Verrill, Richard Colling, P. Gill, T. Mirtti, Ibrahim Kulac, K. Smith-Byrne, Ruth C. Travis, Charlotte Stadler, Richard J. Bryant, Freddie C Hamdy, Dan Woodcock, Ian G. Mills, J. Lundeberg, Alastair D. Lamb
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Lineage tracing is a valuable technique that has greatly facilitated the exploration of cell origins and behavior. With the continuous development of single-cell sequencing technology, lineage tracing technology based on the single-cell level has become an important method to study biological development. Single-cell Lineage tracing technology plays an important role in the hematological system. It can help to answer many important questions, such as the heterogeneity of hematopoietic stem cell function and structure, and the heterogeneity of malignant tumor cells in the hematological system. Many studies have been conducted to explore the field of hematology by applying this technology. This review focuses on the superiority of the emerging single-cell lineage tracing technologies of Integration barcodes, CRISPR barcoding, and base editors, and summarizes their applications in the hematology system. These studies have suggested the vast potential in unraveling complex cellular behaviors and lineage dynamics in both normal and pathological contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84906648ab0b53061a09dee0848b84d693b0c0cb" target='_blank'>
              Single-cell lineage tracing techniques in hematology: unraveling the cellular narrative
              </a>
            </td>
          <td>
            Lu-Han Deng, Mu-Zi Li, Xiaojun Huang, Xiang-Yu Zhao
          </td>
          <td>2025-03-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Glioblastoma, one of the most aggressive and heterogeneous malignant tumors, presents significant challenges for clinical management due to its cellular and metabolic complexity. This review integrates recent advancements in single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to elucidate glioblastoma’s cellular heterogeneity and metabolic reprogramming. Diverse cellular subpopulations, including malignant proliferative cells, stem-like cells, mesenchymal-like cells, and immune-related cells, contribute to tumor progression, treatment resistance, and microenvironmental interactions. Spatial transcriptomics has further revealed distinct spatial distributions of these subpopulations, highlighting differences in metabolic activities between the tumor core and periphery. Key metabolic adaptations, such as enhanced glycolysis, fatty acid oxidation, and glutamine metabolism, play critical roles in supporting tumor growth, immune evasion, and therapeutic resistance. Targeting these metabolic pathways, especially in combination with immunotherapy, represents a promising avenue for glioblastoma treatment. This review emphasizes the importance of integrating single-cell and spatial multi-omics technologies to decode glioblastoma’s metabolic landscape and explore novel therapeutic strategies. By addressing current challenges, such as metabolic redundancy and spatiotemporal dynamics, this work provides insights into advancing precision medicine for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea52e3e92b1a5c4c3caddeca60f128feaaaf952b" target='_blank'>
              Single-cell and spatial transcriptomic insights into glioma cellular heterogeneity and metabolic adaptations
              </a>
            </td>
          <td>
            Yixin Fu, Yong Yi, Yongxiang Shao, Jingcheng Jiang, Qingshan Deng
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="NF2-related Schwannomatosis (NF2 SWN) is a rare disease characterised by the growth of multiple nervous system neoplasms, including bilateral vestibular schwannoma (VS). VS tumours are characterised by extensive leucocyte infiltration. However, the immunological landscape in VS and the spatial determinants within the tumour microenvironment that shape the trajectory of disease are presently unknown. In this study, to elucidate the complex immunological networks across VS, we performed imaging mass cytometry (IMC) on clinically annotated VS samples from NF2 SWN patients. We reveal the heterogeneity in neoplastic cell, myeloid cell and T cell populations that co-exist within VS, and that distinct myeloid cell and Schwann cell populations reside within varied spatial contextures across characteristic Antoni A and B histomorphic niches. Interestingly, T-cell populations co-localise with tumour-associated macrophages (TAMs) in Antoni A regions, seemingly limiting their ability to interact with tumorigenic Schwann cells. This spatial landscape is altered in Antoni B regions, where T-cell populations appear to interact with PD-L1+ Schwann cells. We also demonstrate that prior bevacizumab treatment (VEGF-A antagonist) preferentially reduces alternatively activated-like TAMs, whilst enhancing CD44 expression, in bevacizumab-treated tumours. Together, we describe niche-dependent modes of T-cell regulation in NF2 SWN VS, indicating the potential for microenvironment-altering therapies for VS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e66dea3c7770dc53e6070f848f2e765373a2eab3" target='_blank'>
              Spatial mapping of immune cell environments in NF2-related schwannomatosis vestibular schwannoma
              </a>
            </td>
          <td>
            Adam P. Jones, Michael J Haley, Miriam H Meadows, Grace E. Gregory, C. Hannan, Ana K Simmons, Leoma D. Bere, Daniel Lewis, Pedro Oliveira, Miriam J. Smith, Andrew T King, D. G. R. Evans, P. Paszek, David Brough, Omar N Pathmanaban, Kevin N. Couper
          </td>
          <td>2025-03-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73634c31cfc63813b66fedfb8ecff6a08ba232af" target='_blank'>
              Spatiotemporal organisation of residual disease in mouse and human BRCA1-deficient mammary tumours and breast cancer
              </a>
            </td>
          <td>
            Demeter Túrós, Morgane Decollogny, Astrid Chanfon, Myriam Siffert, L. Romanens, J. Tille, O. Trédan, I. Labidi-Galy, Alberto Valdeolivas, Sven Rottenberg
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Here we present STModule, a Bayesian method developed to identify tissue modules from spatially resolved transcriptomics that reveal spatial components and essential characteristics of tissues. STModule uncovers diverse expression signals in transcriptomic landscapes such as cancer, intraepithelial neoplasia, immune infiltration, outcome-related molecular features and various cell types, which facilitate downstream analysis and provide insights into tumor microenvironments, disease mechanisms, treatment development, and histological organization of tissues. STModule captures a broader spectrum of biological signals compared to other methods and detects novel spatial components. The tissue modules characterized by gene sets demonstrate greater robustness and transferability across different biopsies. STModule: https://github.com/rwang-z/STModule.git. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01441-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61c2f79171ce9194bffe681aecda7bf66be2273" target='_blank'>
              STModule: identifying tissue modules to uncover spatial components and characteristics of transcriptomic landscapes
              </a>
            </td>
          <td>
            Ran Wang, Yan Qian, Xiaojing Guo, Fangda Song, Zhiqiang Xiong, Shirong Cai, Xiuwu Bian, Man-Hon Wong, Qin Cao, Lixin Cheng, Gang Lu, K. Leung
          </td>
          <td>2025-03-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background The heterogeneity of tumors significantly impacts on colorectal cancer (CRC) progression. However, the influence of this heterogeneity on the spatial architecture of CRC remains largely unknown. Methods Spatial transcriptomic (ST) analysis of AOM/DSS-induced colorectal cancer (CRC), integrated with single-cell RNA sequencing, generated a comprehensive spatial atlas of CRC. Pseudotime trajectory, stemness evaluation, and cell-cell communication analyses explored how CD44+ tumor cells at the leading edge remodel the tumor microenvironment (TME). In vitro experiments and immunofluorescence staining of clinical samples validated pleiotrophin (PTN) signaling in promoting cancer-associated fibroblasts (CAFs) phenotypic transition and CRC progression. Results Our findings revealed a distinctive layered ring-like structure within CRC tissues, where CD44+ tumor cells exhibiting high stemness were positioned at the tumor’s leading edge. Inflammatory CAFs (iCAFs)-like, myofibroblastic CAFs (myCAFs)-like cells and pro-tumorigenic neutrophils primarily located at the tumor edge, in proximity to CD44+ tumor cells. CD44+ tumor cells then triggered the phenotypic transition of CAFs into iCAF-like and myCAF-like cells through PTN signaling. Conclusions Our results provide distinctive insights into how tumor heterogeneity reshapes the TME at the leading edge of tumor, thereby promoting CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/562ecaa8f1395c90f98e5b2a88e96773ab95655e" target='_blank'>
              CD44+ cells enhance pro-tumor stroma in the spatial landscape of colorectal cancer leading edge
              </a>
            </td>
          <td>
            F. Tang, Yong-Gui Zhu, Jia Shen, Bowen Yuan, Xiang He, Yuxi Tian, Liang Weng, Lunquan Sun
          </td>
          <td>2025-03-12</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/613002ca1a9de7d4f7ae92b0493c480e7bbe4c62" target='_blank'>
              Abstract 5323: Spatial heterogeneity of tumor and stromal cells in desmoplastic small round cell tumors
              </a>
            </td>
          <td>
            Jiaqian Fan, Adewale A. Adebayo, Diana Shamsutdinova, D. Ingram, Clement Agyemang, R. Cárdenas-Zúñiga, Alexander J. Lazar, Danh D. Truong, Joseph A Ludwig
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have transformed the treatment landscape across multiple cancer types achieving durable responses for a significant number of patients. Despite their success, many patients still fail to respond to ICIs or develop resistance soon after treatment. We sought to identify early treatment features associated with ICI outcome. We leveraged the MC38 syngeneic tumor model because it has variable response to ICI therapy driven by tumor intrinsic heterogeneity. ICI response was assessed based on the level of immune cell infiltration into the tumor – a well-established clinical hallmark of ICI response. We generated a spatial atlas of 48,636 transcriptome-wide spots across 16 tumors using spatial transcriptomics; given the tumors were difficult to profile, we developed an enhanced transcriptome capture protocol yielding high quality spatial data. In total, we identified 8 tumor cell subsets (e.g., proliferative, inflamed, and vascularized) and 4 stroma subsets (e.g., immune and fibroblast). Each tumor had orthogonal histology and bulk-RNA sequencing data, which served to validate and benchmark observations from the spatial data. Our spatial atlas revealed that increased tumor cell cholesterol regulation, synthesis, and transport were associated with a lack of ICI response. Conversely, inflammation and T cell infiltration were associated with response. We further leveraged spatially aware gene expression analysis, to demonstrate that high cholesterol synthesis by tumor cells was associated with cytotoxic CD8 T cell exclusion. Finally, we demonstrate that bulk RNA-sequencing was able to detect immune correlates of response but lacked the sensitivity to detect cholesterol synthesis as a feature of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5d48556d03df2b88d1c06ed57c2bca71cfaf9b2" target='_blank'>
              Spatial genomics reveals cholesterol metabolism as a key factor in colorectal cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Andrew J. Kavran, Yulong Bai, Brian Rabe, Anna Kreshock, Andrew Fisher, Yelena Cheng, Anne Lewin, Chao Dai, Matthew J. Meyer, Konstantinos J. Mavrakis, Anna Lyubetskaya, Eugene Drokhlyansky
          </td>
          <td>2025-03-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc3b5fa458a00b7ac6aa3d0d25d390376657ca" target='_blank'>
              Abstract 5544: Identification, isolation and molecular characterization of drug-resistant sub-populations of pancreatic cancer cells
              </a>
            </td>
          <td>
            S. Choudhury, Shixiong Wang, Y. Akimov, Katarina Willoch, Biswajyoti Sahu, Alfonso Urbanucci, Thomas Fleischer, T. Aittokallio
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="It has been reported that microsatellite instable (MSI) tumors in colorectal cancer (CRC) exhibit stronger anti-tumor responses than microsatellite stable (MSS) tumors. Analyzing scRNA-seq data from 185 CRC patients we found that immune, structural, and cancer cells in MSS tumors with high numbers of tumor-infiltrating CD8 T cells and macrophages (TAMs; CD8hiTAMhi) and CD8lowTAMhi tumors were enriched for pathways associated with anti- and pro-tumor responses, respectively. In CD8hiTAMhi tumors, TAMs expressed an IFN-induced phenotype (e.g. GBP1, CXCL9, IFITM3) and high infiltration of GBP1+ TAMs was associated with better overall survival (n=941). High-resolution spatial transcriptomics (Visium HD) revealed that GBP1+ TAMs clustered with CXCL13+IFNG+PDCD1+ CD8 T cells in tumoral niches, suggesting that GBP1+ TAMs were involved in recruitment and activation of tumor-reactive CD8 T cells. Together, we uncovered coordinated cellular programs across cell types in the microenvironment of MSS tumors, based on simple classification variables, that were strongly associated with prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5dfde3e4f7d39011a34eb5a718ac54f39aa6ee4" target='_blank'>
              Single cell spatial transcriptomics identifies coordinated cellular programs associated with good prognosis in microsatellite stable colorectal cancer
              </a>
            </td>
          <td>
            Victoria T. Karlsen, Umair Majid, Håvard T. Lindholm, S. U. Umu, Karoline Rapp Vander-Elst, A-C Røberg Beitnes, Marianne A. Merok, Kjersti Thorvaldsen Hagen, H. R. Nilsen, Sheraz Yaqub, E. Bækkevold, Diana Domanska, F. Jahnsen
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Objective This study examines the cellular heterogeneity of epithelial cells within pancreatic ductal adenocarcinoma (PDAC) and their contributions to tumor progression, metastasis, and immunosuppressive interactions using single-cell RNA sequencing. Methods Single-cell RNA-sequencing data from two datasets (GSE154778 and GSE158356) were integrated using the Harmony algorithm, followed by quality control, clustering, and differential gene expression analysis. Distinct subpopulations of epithelial cells were identified, and their gene expression profiles were analyzed. To assess the malignancy of these subpopulations, single-cell copy number variation (CNV) analysis and trajectory analysis were conducted. Additionally, intercellular communication was examined using the CellChat platform. Results The analysis revealed pronounced heterogeneity among PDAC epithelial cells, with specific subpopulations exhibiting distinct roles in tumor proliferation, extracellular matrix remodeling, and metastatic dissemination. Subpopulations 4 and 6 were characterized by increased CNV levels and a more malignant phenotype, suggesting an enhanced capacity for metastasis. Single-cell trajectory analysis, along with CellChat, mapped the temporal evolution of epithelial cells, identifying key regulatory genes such as DCBLD2 and JUN. A prognostic model incorporating five key genes, including KLF6, was developed and demonstrated strong predictive accuracy for patient outcomes. Notably, KLF6 emerged as a critical prognostic marker associated with immune modulation, particularly through interactions with M2 macrophages. Conclusion The study highlights the pronounced heterogeneity of epithelial cells in PDAC and their distinct contributions to tumor progression, metastasis, and immune modulation. Through single-cell transcriptomic and CNV analyses, we identified epithelial subpopulations with varying malignant potentials and distinct interactions with the tumor microenvironment. Among these, KLF6 emerged as a key regulator associated with immune modulation and metastasis. Our findings emphasize the significance of epithelial cell heterogeneity in shaping pancreatic cancer progression. These insights provide a foundation for future investigations into novel prognostic markers and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12957-025-03793-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c02e096347df4014f9175b24f19fbd3cae4aa9d0" target='_blank'>
              Deciphering the heterogeneity of epithelial cells in pancreatic ductal adenocarcinoma: implications for metastasis and immune evasion
              </a>
            </td>
          <td>
            Jie Liu, Hui Li, Xiuyun Lin, Jiani Xiong, Guangfeng Wu, Lingyan Ding, Bin Lin
          </td>
          <td>2025-04-16</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4421024c849016285df42c8f390ac35abcedd51e" target='_blank'>
              Pan-cancer spatial characterization of key immune biomarkers in the tumor microenvironment
              </a>
            </td>
          <td>
            James Lindsay, Jennifer Altreuter, J. Alessi, Jason L. Weirather, Anita Giobbie-Hurder, Ian D Dryg, Katharina V. Hoebel, B. Sharma, K. Felt, F. S. Hodi, N. Lindeman, L. Sholl, E. Cerami, Jonathan A. Nowak, Mark M Awad, Scott J Rodig, William E. Lotter
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1db37f2f209f5470571ed3506376b29858f863c2" target='_blank'>
              Tumor cell heterogeneity drives spatial organization of the intratumoral immune response.
              </a>
            </td>
          <td>
            Miho Tanaka, Lotus Lum, Kenneth H Hu, Piyush Chaudhary, Savannah Hughes, Cecilia Ledezma-Soto, B. Samad, Daphne Superville, Kenneth Ng, Arun Chumber, Ciara Benson, Zoe N Adams, Kelly Kersten, Oscar A. Aguilar, Lawrence Fong, Alexis J. Combes, Matthew F. Krummel, Melissa Q Reeves
          </td>
          <td>2025-06-02</td>
          <td>The Journal of experimental medicine</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13f1f42b49de112548e7d274daf610e71f0196dd" target='_blank'>
              Abstract 2067: Single-cell spatial whole transcriptome reveals tumor heterogeneity and stromal dynamics in invasive ductal carcinoma of the breast
              </a>
            </td>
          <td>
            Liang Zhang, Shanshan He, Michael Patrick, Stefan Rogers, Haiyan Zhai, John S Lyssand, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb7304b0f5fc3d1e6eaf91e29f30f258a3c3f94f" target='_blank'>
              Abstract 5263: The spatial landscape of tumor-infiltrating lymphocytes in stage III colorectal cancers by single-cell spatial analysis
              </a>
            </td>
          <td>
            Hojoon Lee, Andrew Su, Minh Tran, Richard Delacruz, A. Sathe, Xiangqi Bai, Ignacio A Wichmann, Lance Pflieger, Bryce Moulton, Tyler Barker, D. Haslem, David Jones, Lincoln Nadauld, Quan H Nguyen, , Terence Rhodes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de704d9a88b35d65b9fbe36b0e36b4cae7473ce" target='_blank'>
              Abstract 161: High-resolution spatial proteomics characterizes colorectal cancer consensus molecular subtypes
              </a>
            </td>
          <td>
            Batuhan Kısakol, Anna Matveeva, Sanghee Cho, Mohammedreza Azimi, A. Lindner, M. Salvucci, E. McDonough, Joanna Fay, Tony O'Grady, N. McCawley, John P Burke, Deborah McNamara, John Graf, Simon McDade, Daniel B Longley, Fiona Ginty, Jochen H M Prehn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7516851ad82e74ba0e5d8528ea22ebc6695c2d0" target='_blank'>
              Abstract 2584: In situ hybridization-based validation of transcriptionally-defined stromal clusters in the prostate cancer microenvironment
              </a>
            </td>
          <td>
            Filippo Pederzoli, H. Pakula, Wikum Dinalankara, Jeanne Jiang, Mohamed Omar, Silvia Rodrigues, P. Nuzzo, Fabio Socciarelli, G. Fanelli, L. Marchionni, Massimo Loda
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178582b0b4cac0c55049a52434a209bd8b7613e6" target='_blank'>
              Quantitative measurement of phenotype dynamics during cancer drug resistance evolution using genetic barcoding
              </a>
            </td>
          <td>
            F. J. Whiting, M. Mossner, C. Gabbutt, C. Kimberley, C. P. Barnes, A. Baker, Erika Yara, A. Sottoriva, Richard A. Nichols, Trevor A. Graham
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b2e8b55d68338e1058f7b78eb076c9fe5aa970" target='_blank'>
              Abstract 166: Spatial subtypes and cellular interactions of cancer-associated fibroblasts revealed by single-cell spatial omics
              </a>
            </td>
          <td>
            Yunhe Liu, Ansam Sinjab, Jimin Min, Guangchun Han, Francesca Paradiso, Yuanyuan Zhang, Yang Liu, Ruiping Wang, Guangsheng Pei, Kyung Serk Cho, Enyu Dai, Akshay V. Basi, Jared K. Burks, K. Rajapakshe, Yanshuo Chu, Jiahui Jiang, Daiwei Zhang, Xinmiao Yan, Paola A. Guerrero, Alejandra G Serrano, Mingyao Li, Tae Hyun Hwang, Andrew Futreal, Jaffer Ajani, Luisa M. Solis Soto, A. Jazaeri, H. Kadara, Anirban Maitra, Linghua Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="A striking characteristic of liver cancer is its extensive heterogeneity, particularly with regard to its varied response to immunotherapy. In this study, we employed multimodal sequencing approaches to explore the various aspects of neoadjuvant nivolumab treatment in liver cancer patients. We used spatially-resolved transcriptomics, single- and bulk-cell transcriptomics, and TCR clonotype analyses to examine the spatiotemporal dynamics of the effects of nivolumab. We observed a significantly higher clonal expansion of T cells in the tumors of patients who responded to the treatment, while lipid accumulation was detected in those of non-responders, likely due to inherent differences in lipid metabolic processes. Furthermore, we found a preferential enrichment of T cells, which was associated with a better drug response. Our results also indicate a functional antagonism between tumor-associated macrophages (TAMs) and CD8 cells and their spatial separation. Notably, we identified a UBASH3B/NR1I2/CEACAM1/HAVCR2 signaling axis, highlighting the intense communication among TAMs, tumor cells, and T-cells that leads to pro-tumorigenic outcomes resulting in poorer nivolumab response. In summary, using integrative multimodal sequencing investigations, combined with the multi-faceted exploration of pre- and post-treatment samples of neoadjuvant nivolumab-treated HCC patients, we identified useful mechanistic determinants of therapeutic response. We also reconstructed the spatiotemporal model that recapitulates the physiological restoration of T cell cytotoxicity by anti-PD1 blockade. Our findings could provide important biomarkers and explain the mechanistic basis differentiating the responders and non-responders. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02314-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18ba5970d9a4d124e0362f78801008f40bb86612" target='_blank'>
              Multimodal sequencing of neoadjuvant nivolumab treatment in hepatocellular carcinoma reveals cellular and molecular immune landscape for drug response
              </a>
            </td>
          <td>
            Fanhong Zeng, Qingyang Zhang, Y. Tsui, Huanhuan Ma, Lu Tian, Abdullah Husain, Jingyi Lu, J. Lee, Vanilla Xin Zhang, Po-Man Li, Gary Cheuk-Hang Cheung, Tan-To Cheung, DanielWai-Hung Ho, I. Ng
          </td>
          <td>2025-04-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae3fe0115c5e85ed3d7113d8af2a6917e00f98fd" target='_blank'>
              Abstract 2091: Integrating single-cell spatial whole transcriptome and histopathology to uncover drivers of tumor heterogeneity in lung adenocarcinoma and squamous cell carcinoma
              </a>
            </td>
          <td>
            Michael Patrick, Liang Zhang, Patrick Danaher, Shanshan He, Sierra Mckinzie, Haiyan Zhai, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Intratumoral heterogeneity is the main cause of tumor treatment failure, varying across disease sites (spatial heterogeneity) and polyclonal properties of tumors that evolve over time (temporal heterogeneity). As our understanding of intratumoral heterogeneity, the formation of which is mainly related to the genomic instability, epigenetic modifications, plastic gene expression, and different microenvironments, plays a substantial role in drug-resistant as far as tumor metastasis and recurrence. Understanding the role of intratumoral heterogeneity, it becomes clear that a single therapeutic agent or regimen may only be effective for subsets of cells with certain features, but not for others. This necessitates a shift from our current, unchanging treatment approach to one that is tailored against the killing patterns of cancer cells in different clones. In this review, we discuss recent evidence concerning global perturbations of intratumoral heterogeneity, associations of specific intratumoral heterogeneity in lung cancer, the underlying mechanisms of intratumoral heterogeneity potentially leading to formation, and how it drives drug resistance. Our findings highlight the most up-to-date progress in intratumoral heterogeneity and its role in mediating tumor drug resistance, which could support the development of future treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d2f0eb0a60c64efe2bd230e8933a5f80d2d405" target='_blank'>
              Intratumoral heterogeneity and drug resistance in cancer
              </a>
            </td>
          <td>
            Yue-Chun Fu, Shao-Bo Liang, Min Luo, Xue-Ping Wang
          </td>
          <td>2025-03-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e2b0637dd6be9b8d44c6621977a5ed5666d298" target='_blank'>
              Abstract 2078: Single-cell spatial transcriptomics in colon adenocarcinoma reveals tumor heterogeneity and immune microenvironmental shifts
              </a>
            </td>
          <td>
            Margaret L Hoang, Yi Cui, Shanshen He, Michael Patrick, Megan Vandenberg, Stefan Rogers, Evelyn Metzger, Haiyan Zhai, Martin Shelton, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC), accounting for 30% of lung cancer cases, lacks adequate research due to limited understanding of its molecular abnormalities. Our study analyzed public LUSC datasets to explore the tumor microenvironment (TME) composition using scRNA-seq from two cohorts. Applying non-negative matrix factorization, we identified unique malignant cell phenotypes, or meta-programs (MPs), based on gene expression patterns. Survival analysis revealed the clinical relevance of these MPs. Findings illuminated a TME landscape enriched with immune cells—CD8 + T, exhausted T, CD4 + T, and naive T cells—and suggested roles for myeloid cells, like cDC1 and pDCs, in LUSC progression. Different MPs highlighted the heterogeneity of malignant cells and their clinical implications. Targeting MP-specific genes may enable personalized therapy, especially for early-stage LUSC. This study offers insights into immune cell function in tumor dynamics, identifies MPs, and paves the way for novel LUSC strategies, enhancing early intervention, personalized treatment, and prognosis, ultimately improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93916-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a302b866a51ae52b8b8d7352c3feab56f45e3d0e" target='_blank'>
              Single-cell RNA sequencing reveals potential therapeutic targets in the tumor microenvironment of lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Junda Fan, Yu Chen, Yue Gong, Hongmei Sun, Rui Hou, Xiaoya Dou, Yanping Zhang, Cheng Huo
          </td>
          <td>2025-03-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1326a43bcf280179de51c8de12b664e06eaa82" target='_blank'>
              Single cell resolution of an epigenetic signature of persister tumor cells
              </a>
            </td>
          <td>
            Mihai Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohamed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cancer cells adapt to treatment, leading to the emergence of clones that are more aggressive and resistant to anti-cancer therapies. We have a limited understanding of the development of treatment resistance as we lack technologies to map the evolution of cancer under the selective pressure of treatment. To address this, we developed a hierarchical, dynamic lineage tracing method called FLARE (Following Lineage Adaptation and Resistance Evolution). We use this technique to track the progression of acute myeloid leukemia (AML) cell lines through exposure to Cytarabine (AraC), a front-line treatment in AML, in vitro and in vivo. We map distinct cellular lineages in murine and human AML cell lines predisposed to AraC persistence and/or resistance via the upregulation of cell adhesion and motility pathways. Additionally, we highlight the heritable expression of immunoproteasome 11S regulatory cap subunits as a potential mechanism aiding AML cell survival, proliferation, and immune escape in vivo. Finally, we validate the clinical relevance of these signatures in the TARGET-AML cohort, with a bisected response in blood and bone marrow. Our findings reveal a broad spectrum of resistance signatures attributed to significant cell transcriptional changes. To our knowledge, this is the first application of dynamic lineage tracing to unravel treatment response and resistance in cancer, and we expect FLARE to be a valuable tool in dissecting the evolution of resistance in a wide range of tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f240aad73d27cb602765176a2877d013637b5963" target='_blank'>
              Hierarchical Lineage Tracing Reveals Diverse Pathways of AML Treatment Resistance
              </a>
            </td>
          <td>
            Rachel Saxe, Hannah Stuart, Abigail Marshall, Fahiima Abdullahi, Zoë Chen, F. Emiliani, Aaron McKenna
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancers present intricate microenvironments comprising heterotypic cellular interactions, yet a comprehensive spatial map remained to be established. Here, we employed the DNA nanoball-based genome-wide in situ sequencing (Stereo-seq) to visualize the geospatial architecture of 30 primary breast tumors and metastatic lymph nodes across different molecular subtypes. This unprecedented high-resolution atlas unveils the fine structure of the tumor vasculature, highlighting heterogeneity in phenotype, spatial distribution, and intercellular communication within both endothelial and perivascular cells. In particular, venular smooth muscle cells are identified as the primary source of CCL21/CCL19 within the microenvironment. In collaboration with ACKR1-positive endothelial cells, they create a chemokine-rich venular niche to synergistically promote lymphocyte extravasation into tumors. High venule density predicts increased immune infiltration and improved clinical outcomes. This study provides a detailed spatial landscape of human breast cancer, offering key insights into the venular regulation of tumor immune infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3853b930721fb6df8f03ce40a70213e79309b5e" target='_blank'>
              Spatially resolved atlas of breast cancer uncovers intercellular machinery of venular niche governing lymphocyte extravasation
              </a>
            </td>
          <td>
            Xin Wang, Zhanyu Wang, Qijun Liao, Pei Yuan, Junpu Mei, Yin Zhang, Chao Wu, Xiyu Kang, Sufei Zheng, Chen-En Yang, Jiaxiang Liu, Qingyao Shang, Jiangtao Li, Bingning Wang, Liangyu Li, Hui Liu, Weining Hu, Zhensheng Dong, Jie Zhao, Linying Wang, Tao Liu, Yusheng Den, Chengrui Wang, Lijuan Han, Qianjun Chen, Huanming Yang, Xun Xu, Jie He, Zhen Yue, Nan Sun, Xiaodong Fang, Jianming Ying
          </td>
          <td>2025-04-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="PURPOSE
To identify the specific intratumoral and microenvironmental heterogeneity of acral (AM) and mucosal melanoma (MM), we aimed to delineate their distinct cellular compositions, evolutionary trajectories, and subtype-specific therapeutic strategies.


EXPERIMENTAL DESIGN
Single-cell transcriptomic and genomic landscapes were analyzed across 42 melanoma samples (28 AM, 11 MM, and 3 non-acral cutaneous melanoma [CM]), supplemented by in vitro and in vivo validation. Tumor and stromal cells were profiled using scRNA-seq, whole-exome sequencing, and functional assays, including transwell migration, co-culture systems, and xenograft models.


RESULTS
Tumor cells exhibited divergent evolutionary routes, with MM dominated by MGP⁺/PCOLCE⁺ subpopulations showing high epithelial-to-mesenchymal transition (EMT) potential. MM displayed elevated neutrophil infiltration and CXCL3⁺ tumor-associated macrophages, while AM was enriched with PI16⁺ cancer-associated fibroblasts (CAFs) promoting tumor proliferation. Molecular classification revealed MM subtypes: an antigen-presenting subtype linked to favorable outcomes and a proliferative subtype associated with recurrence. TIGIT⁺ Treg cells were enriched in AM, suggesting targeted inhibition potential. Genomic analysis connected BRAF/NRAS mutations to ALDOA⁺ stem-like tumor cells and identified PTGDS as a therapeutic target in triple-WT melanomas.


CONCLUSIONS
Our study provides a comprehensive comparison of AM and MM, uncovering subtype-specific stromal-immune interactions and molecular programs. The findings highlight actionable targets (e.g., TIGIT in AM, CXCL3⁺ macrophages in MM) and propose a framework for precision therapies, biomarker-driven trials, and risk stratification to improve outcomes in these aggressive melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d1f1c177fc4d5a9d6abeefba72d4bbfa28d55fc" target='_blank'>
              Single-cell transcriptomic landscape deciphers intratumoral heterogeneity and subtypes of acral and mucosal melanoma.
              </a>
            </td>
          <td>
            Yunyan Li, Ziyang Cui, Xiaole Song, Yeqing Chen, Cang Li, Junfeng Shi, Wenkang Qian, Guoxin Ren, Jiang Zhou, Chunpu Li, Xiaoqing Ma, Yifan Chen, Dongdong Jia, Yongli Zhang, Zhiling Zhang, Ronghao Zhang, Zhaotian Zhang, Yong Chen, Zhixiang Xu, Wantao Chen, X. Miao, Hongmeng Yu, Jianxin Chen, Kai Wang, Colin R. Goding, Zhi Wei, Tao Li, Rutao Cui
          </td>
          <td>2025-04-07</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an exceedingly aggressive primary brain tumor defined by rapid growth, extensive infiltration, and resistance to standard therapies. A central factor driving these malignancies is the subpopulation of glioblastoma stem cells (GSCs), which possess self-renewal capacity, multipotency, and the ability to regenerate tumor heterogeneity. GSCs contribute to key hallmarks of GBM pathobiology, including relentless progression, resistance to chemotherapy and radiotherapy, and inevitable recurrence. GSCs exhibit distinct molecular signatures, enhanced DNA repair, and metabolic adaptations that protect them against conventional treatments. Moreover, they reside within specialized niches—such as perivascular or hypoxic microenvironments—that sustain stemness, promote immunosuppression, and facilitate angiogenesis. Recent discoveries highlight signaling pathways like Notch, Wnt/β-catenin, Hedgehog, STAT3-PARN, and factors such as TFPI2 and HML-2 as critical regulators of GSC maintenance, plasticity, and immune evasion. These findings underscore the complexity of GSC biology and their pivotal role in driving GBM heterogeneity and therapeutic failure. Emerging therapeutic strategies aim to target GSCs through multiple avenues, including surface markers, immunotherapeutics (e.g., CAR T cells), metabolic vulnerabilities, and combination regimens. Advances in patient-derived organoids, single-cell omics, and 3D co-culture models enable more accurate representation of the tumor ecosystem and personalized therapeutic approaches. Ultimately, improved understanding of GSC-specific targets and the tumor microenvironment promises more effective interventions, paving the way toward better clinical outcomes for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e8f7c834c39bbf5240bb0afeda29b66fdae65e" target='_blank'>
              Glioblastoma Stem Cells at the Nexus of Tumor Heterogeneity, Immune Evasion, and Therapeutic Resistance
              </a>
            </td>
          <td>
            Justin Tang, Md Al Amin, J. Campian
          </td>
          <td>2025-04-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Accumulating evidence links pediatric cancers to prenatal transformation events, yet the influence of the developmental stage on oncogenesis remains elusive. We investigated how hematopoietic stem cell developmental stages affect leukemic transformation, disease progression, and therapy response using a novel, humanized model of NUP98::NSD1-driven pediatric acute myeloid leukemia, that is particularly aggressive with WT1 co-mutations. Fetal-derived hematopoietic stem cells readily transform into leukemia, and WT1 mutations further enhance stemness and alter lineage hierarchy. In contrast, stem cells from later developmental stages become progressively resistant to transformation. Single-cell analyses revealed that fetal-origin leukemia stem cells exhibit greater quiescence and reliance on oxidative phosphorylation than their postnatal counterparts. These differences drive distinct therapeutic responses, despite identical oncogenic mutations. In patients, onco-fetal transcriptional programs correlate with worse outcomes. By targeting key vulnerabilities of fetal-origin leukemia cells, we identified combination therapies that significantly reduce aggressiveness, highlighting the critical role of ontogeny in pediatric cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca97e870a1822de5485cb1751fdbfdd64fadf250" target='_blank'>
              Ontogeny Dictates Oncogenic Potential, Lineage Hierarchy, and Therapy Response in Pediatric Leukemia
              </a>
            </td>
          <td>
            Ke Wang, Shayan Saniei, Nikita Poddar, Subrina Autar, Saul Carcamo, Meghana Sreenath, Jack H Peplinski, R. Ries, Isabella G Martinez, Clifford Chao, Anna Huo-Chang Mei, Noshin Rahman, Levan Mekerishvili, Miguel Quijada-álamo, Grace Freed, Mimi Zhang, Katherine Lachman, Zayna Diaz, Manuel M Gonzalez, Jing Zhang, Giang Pham, Dan Filipescu, Mirela Berisa, Tommaso Balestra, J. Reisz, A. D’Alessandro, Daniel J Puleston, Emily Bernstein, Jerry E. Chipuk, M. Wunderlich, S. Tasian, B. Marcellino, Ian A. Glass, Christopher M Sturgeon, D. Landau, Zhihong Chen, E. Papapetrou, Franco Izzo, S. Meshinchi, Dan Hasson, Elvin Wagenblast
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a50452a86dbb7c37a7b8f94f2df738ddd72d94" target='_blank'>
              3D vascularized microtumors unveil aberrant ccRCC vasculature and differential sensitivity to targeted treatments
              </a>
            </td>
          <td>
            Noémie Brassard-Jollive, Y. Atlas, Camille LM Compère, Corinne Ardidie-Robouant, Philippe Mailly, Morad El Bouchtaoui, Virginie Lelarge, Guillaume Blot, Nathalie Josseaume, Stéphanie De Oliveira, C. Hélary, Christophe Leboeuf, Isabelle Cremer, Mathilde Sibony, Guilhem Bousquet, S. Germain, L. Muller, C. Monnot
          </td>
          <td>2025-03-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) often exhibits poor response rates to immune checkpoint inhibitor (ICI) therapies, largely owing to the intricate composition and spatial organization of immune cells within the tumor-immune microenvironment (TIME). The diversity of immune cell populations, their spatial relationships, and dynamic interactions significantly influence the immunosuppressive nature of the TIME, thereby limiting the efficacy of immunotherapy. To address these challenges and enhance the therapeutic potential of ICIs in HNSCC, a comprehensive analysis of the TIME is essential. Spatial transcriptomics (ST), a cutting-edge technology, enables high-resolution mapping of gene expression within the spatial context of the tumor, providing critical insights into the functional roles and interactions of immune cells in the TIME. This review highlights the importance of ST in uncovering the complexities of the TIME in HNSCC and proposes strategies for leveraging these insights to develop more effective immunotherapeutic approaches. By integrating spatial and molecular information, this review aims to pave the way for personalized and precision-based treatments in HNSCC, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef3b95effc04f0811082363795fb3dc9666b0f8" target='_blank'>
              Mapping immunotherapy potential: spatial transcriptomics in the unraveling of tumor-immune microenvironments in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Seo-Won Choi, Jeong Heon Kim, Jisu Hong, Minsu Kwon
          </td>
          <td>2025-04-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd78a6499d1079983203268939cd1b90e4adb3f" target='_blank'>
              Abstract 154: Single-cell spatial transcriptomics unravels cell states and ecosystems associated with clinical response to immunotherapy
              </a>
            </td>
          <td>
            Z. Abdulrahman, Roderick C. Slieker, Daniel McGuire, M. J. Welters, M. V. van Poelgeest, S. H. van der Burg
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinomas (HNSCC) are the seventh most prevalent form of cancer and are associated with human papilloma virus infection (HPV-positive) or with tobacco and alcohol use (HPV-negative). HPV-negative HNSCCs have a high recurrence rate, and individual patients’ responses to treatment vary greatly due to the high level of cellular heterogeneity of the tumor and its microenvironment. Here, we describe a HNSCC single cell atlas, which we created by integrating six publicly available datasets encompassing over 230,000 cells across 54 patients. We contextualized the relationships between existing signatures and cell populations, identified new subpopulations, and show the power of this large-scale resource to robustly identify associations between transcriptional signatures and clinical phenotypes that would not be possible to discover using fewer patients. We reveal a previously undefined myeloid population, sex-associated changes in cell type proportions, and novel interactions between CXCL8-positive fibroblasts and vascular endothelial cells. Beyond our findings, the atlas will serve as a public resource for the high-resolution characterization of tumor heterogeneity of HPV-negative HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe2478d77674f8edfddeb9019aa5fa535133e0f" target='_blank'>
              A highly resolved integrated single-cell atlas of HPV-negative head and neck cancer
              </a>
            </td>
          <td>
            Lina Kroehling, Andrew Chen, Anthony J Spinella, Eric Reed, Maria Kukuruzinka, X. Varelas, S. Monti
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/623dee1630864493156f7c52b0b73a1aacf3c1e1" target='_blank'>
              Abstract 756: Unveiling the spatial landscape of tumor and stroma heterogeneity in colorectal cancer with spatial transcriptomics and deep learning
              </a>
            </td>
          <td>
            Jie Zhou, Todd B Sheridan, Sergii Domanskyi, Stephanie L. Cowles, SungHee Park, Ilham Putra, Olga Anczuków, Jeffrey H. Chuang, Jill C. Rubinstein
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39cd802b620e803fe61dfbfe7d21f2c3b02ece22" target='_blank'>
              Spatial mapping of transcriptomic plasticity in metastatic pancreatic cancer.
              </a>
            </td>
          <td>
            Guangsheng Pei, Jimin Min, K. Rajapakshe, Vittorio Branchi, Yunhe Liu, B. Selvanesan, F. Thege, Dorsay Sadeghian, Daiwei Zhang, Kyung Serk Cho, Yanshuo Chu, Enyu Dai, Guangchun Han, Mingyao Li, Cassian Yee, Kazuki Takahashi, Bharti Garg, Hervé Tiriac, Vincent Bernard, A. Semaan, Jean L. Grem, T. Caffrey, Jared K. Burks, Andrew M Lowy, Andrew J. Aguirre, P. Grandgenett, Michael A. Hollingsworth, Paola A. Guerrero, Linghua Wang, Anirban Maitra
          </td>
          <td>2025-04-23</td>
          <td>Nature</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Motivation Spatial transcriptomics holds great promise for revolutionizing biology and medicine by providing gene expression profiles with spatial information. Until recently, spatial resolution has been limited, but advances in high-throughput in situ imaging technologies now offer new opportunities by covering thousands of genes at a single-cell or even subcellular resolution, necessitating databases dedicated to comprehensive coverage and analysis with user-friendly intefaces. Results We introduce the HISSTA database, which facilitates the archival and analysis of in situ transcriptome data at single-cell resolution from various human tissues. We have collected and annotated spatial transcriptome data generated by MERFISH, CosMx SMI, and Xenium techniques, encompassing 112 samples and 28 million cells across 16 tissue types from 63 studies. To decipher spatial contexts, we have implemented advanced tools for cell type annotation, spatial colocalization, spatial cellular communication, and niche analyses. Notably, all datasets and annotations are interactively accessible through Vitessce, allowing users to focus on regions of interest and examine gene expression in detail. HISSTA is a unique database designed to manage the rapidly growing dataset of in situ transcriptomes at single-cell resolution. Given its comprehensive data content and advanced analysis tools with interactive visualizations, HISSTA is poised to significantly impact cancer diagnosis, precision medicine, and digital pathology. Availability and implementation HISSTA is freely accessible at https://kbds.re.kr/hissta/. The source code is available at https://doi.org/10.5281/zenodo.14904523.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8847bc6bc6ac698124acae8ba2d55144bc394fe" target='_blank'>
              HISSTA: a human in situ single-cell transcriptome atlas
              </a>
            </td>
          <td>
            Jiwon Yu, Jiwoo Moon, Minseo Kim, Gyeol Han, Insu Jang, Jinyoung Lim, Seungmook Lee, Seok-Hwan Yoon, Woong-Yang Park, Byungwook Lee, Sanghyuk Lee
          </td>
          <td>2025-03-31</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9981c24a8ff91461ed8c8211a765a8781b3dfd60" target='_blank'>
              Abstract 6461: The spatial cellular landscape of Gleason pattern 3 and 4 prostate tumors
              </a>
            </td>
          <td>
            Çiğdem Ak, Z. Sayar, Guillaume Thibault, M. J. Kuykendall, George V. Thomas, V. Stavrinides, Young Hwan Chang, E. S. Eksi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee29e34be0729357213c5d8d79a8e1845ecaf78" target='_blank'>
              Multidimensional characterization of cellular ecosystems in Hodgkin lymphoma
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew Lytle, Yifan Yin, Makoto Kishida, Michael Li, Denise Smorra, Laura K. Hilton, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O'Brien, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Patino Rangel, Michael Hong, Shaocheng Wu, Eric Lee, Katsuyoshi Takata, T. Miyata‐Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander M Xu, Akil Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, Ryan D Morin, A. Alizadeh, K. Savage, David W. Scott, Christian Steidl
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89052d3ed2d82ce2288e348ab845b3ae781f2935" target='_blank'>
              Abstract LB265: The spatial landscape of cancer hallmarks reveals patterns of tumor ecological dynamics and drug sensitivity
              </a>
            </td>
          <td>
            E. Porta-Pardo, Mustafa Sibai, Sergi Cervilla, D. Grases, M. Esteller
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2da592cc8a55cacd02ae7c34fe809a09834403" target='_blank'>
              Spatially aligned random partition models on spatially resolved transcriptomics data
              </a>
            </td>
          <td>
            Yunshan Duan, Shuai Guo, Hao Yan, Wenyi Wang, Peter Mueller
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b5519e4d897e90daae110bc4297e9d131ea31b" target='_blank'>
              Biobank of genetically defined murine prostate cancer tumoroids uncovers oncogenic pathways and drug vulnerabilities driven by PTEN-loss
              </a>
            </td>
          <td>
            Jessica Kalla, Thomas E. Dillinger, Zlata Pavlovicova, Reema Jacob, Emine Atas, Anil Baskan, Kristina Draganić, A. Tiefenbacher, T. Limberger, Theresia Mair, Gabriel Wasinger, Ludovica Villanti, S. Kubicek, Lukas Kenner, G. Egger
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is a highly lethal malignant intracranial tumor, distinguished from low-grade glioma by histopathological hallmarks such as pseudopalisading cells around necrosis (PAN) and microvascular proliferation (MVP). To date the spatial organization of the molecular and cellular components of these specific histopathological features has not been fully elucidated. Methods Here, using bulk RNA sequencing, spatial transcriptomic and single cell RNA sequencing (scRNA-seq) data of GBM patients, we identified niche-specific transcriptional programs and characterized the differences in molecular expression and cellular organization between PAN and MVP. Results Notably, we discovered spatially distinct domains within the tumor core and identified niche-specific signatures: NDRG1 and EPAS1, specifically expressed in the PAN and MVP regions. The clustering results showed two distinct phenotypes of endothelial cells (ECs) were enriched in the MVP and PAN regions, respectively. PAN-associated endothelial cells exhibit copy number variations similar to those in GBM cells. Single cell trajectory analysis reveals a pseudotime trajectory, indicating the differentiation of glioblastoma stem cells (GSCs) toward ECs. Conclusions Necrosis cores which are surrounded by hypoxic and perivascular niches and microvascular proliferation area within the glioblastoma tumor microenvironment, have been considered as standardized morphological indicators of aggressive GBM. Our findings provide a cellular and molecular insights into GBM progression. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06185-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75320d960d6bc41d40bd96ad524b1fc28b650a0b" target='_blank'>
              Multi-transcriptomics reveals niche-specific expression programs and endothelial cells in glioblastoma
              </a>
            </td>
          <td>
            Jiukun Hu, Xiaohan Sa, Yue Yang, Yuwen Han, Jie Wu, Minxuan Sun, S. Shafi, Nafees Ahmad, Sami Siraj, Jiao Yang, Yuanshuai Zhou
          </td>
          <td>2025-04-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) represents the most common pathological type of lung cancer, and the combination of neoadjuvant immunotherapy with chemotherapy has emerged as the first-line treatment for NSCLC. Nevertheless, the efficacy of this therapeutic approach remains variable. The present study aims to examine the impact of chemoimmunotherapy in NSCLC patients, with a view to identifying key molecules, critical cell subpopulations, communication patterns and spatial distributions that potentially correlate with therapeutic sensitivity. A total of 16 lung cancer tissue samples were collected from a cohort of 12 NSCLC patients and subjected to single-cell RNA and spatial transcriptome sequencing. Our data demonstrated that the distribution of CD4 + Treg T cells and mCAFs indicated an immunosuppressive tumor microenvironment, while the accumulation of CD4 + Th17 T cells and iCAFs could act as a positive marker for the sensitivity to chemoimmunotherapy. Furthermore, a significant high level of SELENOP-macrophages was observed in tissues from positive responders, and a strong co-localization between SELENOP-macrophages and antigen-presenting cancer associated fibroblasts (CAFs) in the tumor boundaries was identified, indicating the cooperative roles of these two cell types in response to combined therapy. Moreover, SELENOP-macrophages were observed to be accumulated in tertiary lymphoid structures, which further suggested its critical role in recruiting lymphocytes. Furthermore, analysis of cell–cell communication, based on spatial transcriptomics, suggests that the interactions between SELENOP-macrophages, apCAFs, CD4 + and CD8 + T cells were significantly enhanced in responders. In addition, SELENOP-macrophages recruited CD4 + Naïve, Helper and CD8 + Naïve T cells through pathways such as the cholesterol, interleukin, chemokine and HLA when responding to combined therapy. The present study further unveils the dynamic spatial and transcriptional changes in the tumor microenvironment of non-small cell lung cancer in response to combination therapy. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02287-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86da4dd75d392abc980cae7b3d81379fef557c28" target='_blank'>
              Single-cell and spatial transcriptomic analyses revealing tumor microenvironment remodeling after neoadjuvant chemoimmunotherapy in non-small cell lung cancer
              </a>
            </td>
          <td>
            Xiaolu Cui, Siyuan Liu, He Song, Jingjing Xu, Yanbin Sun
          </td>
          <td>2025-04-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4356e915cd9638ea3a28d0766da31c4fd8473bcf" target='_blank'>
              Abstract 3703: Cell identity revealed by precise cell cycle state mapping links data modalities
              </a>
            </td>
          <td>
            Saeed S. Alahmari, Andrew Schultz, Jordan Albrecht, Vural Tagal, Zaid Siddiqui, Sandhya Prabhakaran, I. El Naqa, Alexander Anderson, Laura Heiser, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Hepatoblastoma (HB) is a common paediatric liver malignancy characterised by significant intratumoral heterogeneity and a complex tumour microenvironment (TME). Using single‐cell RNA sequencing (scRNA‐seq), we analysed 43,592 cells from three tumour regions and adjacent normal tissue of an HB patient. Our study revealed distinct cellular compositions and varying degrees of malignancy across different tumour regions, with the T1 region showing the highest malignancy and overexpression of HMGB2 and TOP2A. Survival analysis demonstrated that high HMGB2 expression is associated with poor prognosis and increased recurrence, suggesting its potential as a prognostic marker. Additionally, we identified a diverse immune microenvironment enriched with regulatory T cells (Tregs) and CD8+ effector memory T cells (Tem), indicating potential immune evasion mechanisms. Notably, CTLA‐4 and PD‐1 were highly expressed in Tregs and Tem cells, highlighting their potential as immunotherapy targets. Myeloid cells, including Kupffer cells and dendritic cells, also exhibited distinct functional roles in different tumour regions. This study provides the first comprehensive single‐cell atlas of HB, revealing critical insights into its intratumoral heterogeneity and immune microenvironment. Our findings not only advance the understanding of HB biology but also offer new directions for precision medicine, including the development of targeted therapies and immunotherapeutic strategies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/634b11e52e203c006a13f961efc458d554db20a9" target='_blank'>
              Intratumoral Heterogeneity and Immune Microenvironment in Hepatoblastoma Revealed by Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Mingdi Ma, Chen Jin, Qian Dong
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43efa757ce011d3db682c5cad92c0307f3b656c7" target='_blank'>
              The Role of Phenotypic Plasticity in Adaptation to Treatment and Prospective Plasticity Drivers in Hepatoblastoma
              </a>
            </td>
          <td>
            Emilia Chen, Christie English, Alejandro Allo Anido, Siân Hamer, Stefano Cairo, Alejandra Bruna
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6fee264333ab195499909e9bd27046249a7701" target='_blank'>
              STmiR: A Novel XGBoost-Based Framework for Spatially Resolved miRNA Activity Prediction in Cancer Transcriptomics
              </a>
            </td>
          <td>
            Jiaqi Yuan, Peng Xu, Zheng Ye, Wenbin Liu
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/947de9f123e944f865748b3a75364432746d6f15" target='_blank'>
              Abstract 6257: Spatial and multi-omic profiling of metastatic colorectal cancer reveals key cellular ecosystems and drivers of heterogeneity
              </a>
            </td>
          <td>
            Chien-Wei Peng, Michael D. Iglesia, Adrienne Visani, Xiang Li, Xiangwei Fang, R. Jayasinghe, Jingxian Liu, John M Herndon, Jacqueline L Mudd, Andrew Houston, Preet Lal, Siqi Chen, Ambrose Plante, A. T. D. Targino da Costa, Kapur B. Dhami, J. Ponce, Robert S Fulton, Michael Heinz, M. Chheda, Feng Chen, Ryan C. Fields, Li Ding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background The identification of the complex spatial architecture of immune cell infiltration and its interaction mechanisms within tumor ecosystems provides crucial insights into therapeutic responses to neoadjuvant therapy in muscle-invasive bladder cancer (MIBC). This study aims to characterize the spatial features of distinct cell-type niches within the tumor microenvironment (TME) of patients with varying responses to neoadjuvant therapy. Methods We performed spatial transcriptomic profiling on six MIBC specimens obtained from a registered clinical trial (ChiCTR2000032359), generating whole-transcriptome spatial atlases to map the TME architecture. High-throughput analytical frameworks were employed to deconstruct the TME, and key findings were validated through immunohistochemistry and mouse model experiments. Results Our analysis revealed that tissues from complete responders exhibited greater infiltration of T and B cells, with the formation of tertiary lymphoid structure (TLS). Trajectory analysis identified CCL19/CCL21 as the key signaling molecules driving TLS formation in MIBC. Mouse experiments demonstrated that recombinant CCL19/CCL21 protein injections promoted intratumoral TLS formation and enhance the efficacy of immunotherapy. Furthermore, we observed significant intrinsic heterogeneity within individual tumors, which may contribute to the lack of therapeutic efficacy in MIBC. Conclusions This study underscores the critical role of TLS formation in the response to neoadjuvant therapy in MIBC. We identified CCL19/CCL21 as key drivers of TLS formation within MIBC tumors and potential immune-sensitizing agents. Additionally, the intrinsic heterogeneity of tumor should be considered a significant factor influencing therapeutic efficacy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06358-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d2414e88dc899f0c1cdec00aa234fe0cea16d7" target='_blank'>
              Spatial relationships and interactions of immune cell niches are linked to the pathologic response of muscle-invasive bladder cancer to neoadjuvant therapy
              </a>
            </td>
          <td>
            Wasilijiang Wahafu, Quan Zhou, Xihua Yang, Yongming Yang, Yuanyuan Zhao, Zhu Wang, Xiangpeng Kang, Xiongjun Ye, N. Xing
          </td>
          <td>2025-03-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a powerful tool for characterizing tumor heterogeneity, yet accurately identifying malignant cells remains challenging. Here, we propose scMalignantFinder, a machine learning tool specifically designed to distinguish malignant cells from their normal counterparts using a data- and knowledge-driven strategy. To develop the tool, multiple cancer datasets were collected, and the initially annotated malignant cells were calibrated using nine carefully curated pan-cancer gene signatures, resulting in over 400,000 single-cell transcriptomes for training. The union of differentially expressed genes across datasets was taken as the features for model construction to comprehensively capture tumor transcriptional diversity. scMalignantFinder outperformed existing automated methods across two gold-standard and eleven patient-derived scRNA-seq datasets. The capability to predict malignancy probability empowers scMalignantFinder to capture dynamic characteristics during tumor progression. Furthermore, scMalignantFinder holds the potential to annotate malignant regions in tumor spatial transcriptomics. Overall, we provide an efficient tool for detecting heterogeneous malignant cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a192a3c85f5ce13a8130114121b674f869202937" target='_blank'>
              scMalignantFinder distinguishes malignant cells in single-cell and spatial transcriptomics by leveraging cancer signatures
              </a>
            </td>
          <td>
            Qiaoni Yu, Yuan-Yuan Li, Yunqin Chen
          </td>
          <td>2025-03-27</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4db159a15c79e0c166f89e06a6be6cfce7707f63" target='_blank'>
              Abstract 6470: Spatial analysis of the tumor microenvironment in diffuse large B-cell lymphoma reveals clinically relevant recurrent cellular neighborhoods
              </a>
            </td>
          <td>
            M. Autio, S. Leivonen, L. Meriranta, M. Karjalainen-Lindsberg, T. Pellinen, S. Leppä
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5814a46c3dc77d4dc02305cdfe806d50dc91bd55" target='_blank'>
              DANEELpath: Open-source Digital analysis tools for histopathological research. Applications in NEuroblastoma modELs
              </a>
            </td>
          <td>
            I. Vieco-Martí, A. López-Carrasco, Samuel Navarro, Sofia Granados Aparici, Rosa Noguera
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462e90afb96bda692d274dd22748c2e27453a930" target='_blank'>
              Diving Deep: Profiling Exhausted T Cells in the Tumor Microenvironment Using Spectral Flow Cytometry.
              </a>
            </td>
          <td>
            Karen W Teng, Weng Hua Khoo, Nicholas Ching Wei Ho, S. J. Yang, Douglas C Wilson, Edmond Chua, Shu Wen Samantha Ho
          </td>
          <td>2025-04-03</td>
          <td>Cytometry. Part A : the journal of the International Society for Analytical Cytology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96dda01278fdb738773a58b49aca47aa0308c83d" target='_blank'>
              Abstract 5541: Dissecting chemotherapy resistance development in triple negative breast cancer through single cell multiome and lineage tracing
              </a>
            </td>
          <td>
            Chufan Zhang, Chu Pan, D. Cescon, Mathieu Lupien
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a1ac1da292bdfa7cb083f4691cc613492b11ea" target='_blank'>
              Pan-cancer Analysis Identified Ectopic RUNX1T1 Associated with Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, I. Kim, Su Deng, Ping Mu
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad68c593de3480a301b38845ef8182dbdcc71b6" target='_blank'>
              Abstract 6481: Multiplexed single-cell pharmacotranscriptomics and spatial tumor biology reveals targetable reciprocal cancer-stromal interactions in prostate cancer
              </a>
            </td>
          <td>
            A. Murumägi, Frida Rantanen, Alice Dini, Mariliina Arjama, K. Välimäki, Elina Multamäki, T. Mirtti, Antti S. Rannikko, T. Pellinen, Daniela Ungureanu, Olli Kallioniemi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0444f2eee8d5a4470978d8e6c693faf1ee9e687e" target='_blank'>
              A Phenotype-Driven Multi-Omic Atlas of Glioblastoma Invasion
              </a>
            </td>
          <td>
            C. Krona, Anders Sundström, Emil Rosén, Soumi Kundu, H. Mangukiya, H. Babačić, Irem Uppman, Madeleine Skeppås, Ida Larsson, Ludmila Elfineh, Xiaofang Cao, Rashmi Ramachandra, Ramy Elgendy, Katrin Rut Halldorsdottir, Zankruti Dave, Milena Doroszko, K. Olausson, Johan Wikström, M. Pernemalm, Sven Nelander
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6af959f7143f19a656c32e73ddcf95ebee2a8fbb" target='_blank'>
              Abstract 6463: Interrogating the tumor microenvironment of nasopharyngeal carcinoma by single-cell spatial transcriptomics
              </a>
            </td>
          <td>
            Chee Yit Lim, Yi Ren, W. K. Teo, Chor Hiang Siow, K. Loh, Raymond KY Tsang, Bingcheng Wu, Joshua K. Tay
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddc6a274f48d59a3fed08cd82e2b131cfa0c1209" target='_blank'>
              Abstract 2779: Whole transcriptome spatial exploration of cultured tumor cells at subcellular resolution
              </a>
            </td>
          <td>
            Yi Cui, Margaret L Hoang, Shanshan He, Rachel Liu, Tushar Rane, R. Khafizov, Haiyan Zhai, Christine Kang, Michael Rhodes, Joseph M. Beechem
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Rationale: Advances in cancer therapies have significantly improved patient survival; however, tumors enriched in cancer stem cells (CSCs) have poor treatment responses. CSCs are a key source of tumor heterogeneity, contributing to therapeutic resistance and unfavorable patient outcomes. In the tumor microenvironment (TME), cell-in-cell (CIC) structures, where one cell engulfs another, have been identified as markers of poor prognosis. Despite their clinical relevance, the mechanisms underlying CIC formation across different tumor cell subpopulations remain largely unknown. Elucidating these processes could provide novel insights and therapeutic opportunities to address aggressive, treatment-resistant cancers. Method: Fluorescent mCherry-carrying colorectal cancer stem cells (CRCSCs) were expanded as spheroids in serum-free media and cocultured with either parental cancer cell-expressing Venus fluorescent protein or CFSE dye-stained immune cells (T cells, M1/M2 macrophages, neutrophils, and NK cells) or treated with EGFR- or PD-L1-targeting antibodies to assess the formation of CIC structures. Genes potentially crucial for the formation of CIC structures were knocked down or overexpressed, and their effects on CIC formation were evaluated. The clinical relevance of the in vitro findings was confirmed through analysis of formalin-fixed, paraffin-embedded (FFPE) human colorectal cancer (CRC) specimens. Results: CRCSCs have a strong predilection for serving as the outer cell in a CIC structure and forming homotypic CIC structures predominantly with parental CRC cells. The frequency of CIC structure formation increased when the cells were exposed to anti-PD-L1 antibody treatment. Both the outer CRCSC in a CIC structure and CRCSCs released from a homotypic CIC structure showed enhanced resistance to the cytotoxicity of NK-92MI cells. Restoration of Stathmin1 (STMN1) expression but not RAC1 knockdown in CRCSCs reduced the homotypic CIC frequency, disrupted the outer cell fate in CIC structures, and increased cell susceptibility to NK-92MI cytotoxicity. In CRC patients, CIC structures are associated with poor tumor differentiation, negative STMN1 expression, and poor prognosis. Conclusion: CSCs play a crucial role in informing CIC structures in CRC. CIC structure formation partially depends on low STMN1 expression and confers a survival advantage under NK cytotoxicity. Targeting this pathway may significantly improve immunotherapy's efficacy for CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ab3f42feb06b52b9aecd81de765adfb3a490b01" target='_blank'>
              Colorectal cancer stem cells develop NK cell resistance via homotypic cell-in-cell structures suppressed by Stathmin1
              </a>
            </td>
          <td>
            Yen-Yu Lin, Hsin-Yi Lan, Hao-Wei Teng, Ya-Pei Wang, Wen-Chun Lin, Wei-Lun Hwang
          </td>
          <td>2025-03-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a common and highly lethal type of primary brain tumor in adults. Therapeutic failure is partly attributed to a fraction of Glioblastoma Stem Cells (GSCs) that show high levels of heterogeneity and plasticity. GSCs exist in a transcriptional gradient between two states: Developmental (Dev) and Injury Response (IR) in which IR-GSCs exhibit more invasive behaviors. While previous studies have identified fitness genes in GSCs, the genes required to establish and maintain the Dev and IR states remain poorly defined. To identify the regulators of the IR GSC state, we performed a phenotypic genome-wide CRISPR-Cas9 knockout (KO) screen in patient-derived GSCs based on cell surface expression of the IR marker CD44. Notably, we found that perturbations of the histone acetyltransferase EP300 in IR GSCs led to decreased CD44 cell surface expression, significant downregulation of gene expression signatures associated with the IR state, and to decreased self-renewal and invasion. Furthermore, genetic targeting of Ep300 in a mouse GBM model delayed tumor initiation and/or progression. Collectively, our results establish EP300 as a regulator of the IR state in GSCs and provide a mechanistic basis for its therapeutic targeting in GBM. Significance A genome-wide phenotypic CRISPR-Cas9 screen in a patient-derived Glioblastoma Stem Cell line identified the genes required to maintain the Injury-Response cellular state, with a focus on the histone acetyl transferase gene EP300. This study suggests how therapeutic targeting of cellular state could reduce the aggressiveness of GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72194daa3f94b0757cee54e56b558726a1fc4778" target='_blank'>
              Identification of Injury-Response State Determinants in Glioblastoma Stem Cells
              </a>
            </td>
          <td>
            Fatemeh Molaei, G. Macleod, Shahan Haider, Aleczandria S. Tiffany, Frank M Oteng, Jacob M. Berman, Arianna Skirzynska, Molly S. Shoichet, Daniel Schramek, Peter B. Dirks, Stéphane Angers
          </td>
          <td>2025-04-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6ce39bb3cec413d591d56991415df20b385ea43" target='_blank'>
              Abstract 2705: Spatial profiling of tumor microenvironment uncovers potential novel therapeutic approach in pediatric neuroblastoma
              </a>
            </td>
          <td>
            Cui Tu, Chin Wee Tan, J. Monkman, Ana Clara Almeida, Gabrielle Antonio-Carreon, Natacha Omer, Giaan Hull, Kimberly Chung, Ahmed M Mehdi, Antonietta Salerno, Orazio Vittorio, Aaron Mayer, Cleber Machado de Souza, Fernanda de Almeida Brehm Pinhati, Selene Elifio-Esposito, Wayne Nicholls, Lucia de Noronha, A. Kulasinghe, Fernando Souza-Fonseca-Guimaraes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e024d7323084416b4b6e84dca254aca71e425ab" target='_blank'>
              Learning the cellular origins of cancer using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Bruno Giotti, Wooseung Lee, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract We developed a deep learning Pathomics image analysis workflow to generate spatial Tumor-TIL maps to visualize and quantify the abundance and spatial distribution of tumor infiltrating lymphocytes (TILs) in colon cancer. Colon cancer and lymphocyte detection in hematoxylin and eosin (H&E) stained whole slide images (WSIs) has revealed complex immuno-oncologic interactions that form TIL-rich and TIL-poor tumor habitats, which are unique in each patient sample. We compute Tumor%, total lymphocyte%, and TILs% as the proportion of the colon cancer microenvironment occupied by intratumoral lymphocytes for each WSI. Kaplan-Meier survival analyses and multivariate Cox regression were utilized to evaluate the prognostic significance of TILs% as a Pathomics biomarker. High TILs% was associated with improved overall survival (OS) and progression-free interval (PFI) in localized and metastatic colon cancer and other clinicopathologic variables, supporting the routine use of Pathomics Tumor-TIL mapping in biomedical research, clinical trials, laboratory medicine, and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b4f8d94ce78fc5fa82e89d6b995817192c3e592" target='_blank'>
              Pathomics Image Analysis of Tumor Infiltrating Lymphocytes (TILs) in Colon Cancer
              </a>
            </td>
          <td>
            Yuwei Zhang, Shahira Abousamra, Mahmudul Hasan, Luke A. Torre-Healy, S. Krichevsky, Sampurna Shrestha, Erich Bremer, Derek A. Oldridge, Andrew J. Rech, E. E. Furth, Therese J. Bocklage, Justin S. Levens, Isaac Hands, Erich B. Durbin, D. Samaras, T. Kurç, J. Saltz, Rajarsi Gupta
          </td>
          <td>2025-04-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/527fb3619bec4cdd8432c9c09f5972cd49b69aac" target='_blank'>
              Abstract 5276: Comprehensive spatial mapping uncovers immune ecotypes linked to tumor heterogeneity in HCC lung metastasis
              </a>
            </td>
          <td>
            Jinghan Zhu, Guangzhen Cai, Bin Guo, Xiaoping Chen, Huilan Zhang, Hong Zhu, Wanguang Zhang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Single-cell RNA sequencing (scRNA-seq) has enabled the dissection of complex tumor ecosystems. Recognition of malignant cells as an essential step has a profound impact on downstream interpretation. However, most existing computational strategies are based on prior knowledge of canonical cell-type markers. We have developed a marker-free approach, the Seed-Cluster based Approach for NEoplastic cells Recognition (SCANER), to identify malignant cells based on significant gene expression variations caused by genomic instability. Upon analyzing different cancer types, SCANER achieved superior accuracy and robustness in identifying malignant cells, effectively addressing dropout events and tumor purity variations. Besides, SCANER can significantly detect copy number variations (CNVs) in malignant cells compared to nonmalignant cells, which is further confirmed through the paired whole exome sequencing data. In conclusion, SCANER has the potential to facilitate the biological exploration of the tumor ecosystem by accurately identifying malignant cells and it is applicable across various solid cancer types regardless of prior knowledge. SCANER is available at https://github.com/woolingxiang/SCANER.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12108cdceb40ee3f1cdad70a314795b466e1e854" target='_blank'>
              SCANER: robust and sensitive identification of malignant cells from the scRNA-seq profiled tumor ecosystem
              </a>
            </td>
          <td>
            Peng Xia, Wei Wu, Quanzhong Liu, Bin Huang, Min Wu, Zihan Lin, Mengyan Zhu, Miao Yu, Ying Qu, Kening Li, Lingxiang Wu, Ruohan Zhang, Qianghu Wang
          </td>
          <td>2025-03-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483a65f81564c748a7e6be963230e929d2717f09" target='_blank'>
              Abstract 153: High-resolution profiling of the tumor microenvironment with spatial ecotypes
              </a>
            </td>
          <td>
            Wubing Zhang, , A. Usmani, N. Earland, Hyun Soo Jeon, I. Alahi, N. Semenkovich, Janella C. Schwab, Chloé B. Steen, Chloe M. Sachs, Bilge Gungoren, F. Qaium, P. Harris, Jacqueline L Mudd, Qingyuan Cai, A. Bacchiocchi, Ryan C. Fields, Mario Sznol, R. Halaban, David Y. Chen, A. Chaudhuri, Aaron M. Newman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Characterizing the variety of cell types in the tumor microenvironment (TME) and their organization into cellular communities is vital for elucidating the biological diversity of cancer and informing therapeutic strategies. Methods Here, we employed a machine learning-based algorithm framework, EcoTyper, to analyze single-cell transcriptomes from 139 patients with head and neck squamous cell carcinoma (HNSC)and gene expression profiles from 983 additional HNSC patients, aiming to delineate the fundamental cell states and ecosystems integral to HNSC. Results A diverse landscape of 66 cell states and 9 ecosystems within the HNSC microenvironment was identified, revealing classical cell types while also expanding upon previous immune classifications. Survival analysis revealed that specific cell states and ecotypes (ecosystems) are associated with patient prognosis, underscoring their potential as indicators of clinical outcomes. Moreover, distinct cell states and ecotypes exhibited varying responses to immunotherapy and chemotherapy, with several showing promise as predictive biomarkers for treatment efficacy. Conclusion Our large-scale integrative transcriptome analysis provides high-resolution insights into the cellular states and ecosystems of HNSC, facilitating the discovery of novel biomarkers and supporting the development of precision therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06299-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd5809c9e6b6aaac22c97a1616ff68cb7ac8d0d" target='_blank'>
              High-dimensional deconstruction of HNSC reveals clinically distinct cellular states and ecosystems that are associated with prognosis and therapy response
              </a>
            </td>
          <td>
            Lei Xiao, Zhe Shen, Zhaoyu Pan, Yuanzheng Qiu, Donghai Huang, Yong Liu, Chao Liu, Xin Zhang
          </td>
          <td>2025-03-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcbbcac8a736304b9cab15ef0da632985a22219d" target='_blank'>
              Abstract 2666: Mapping intratumoral heterogeneity in colorectal cancer: Spatially resolved subclones and tumor microenvironment
              </a>
            </td>
          <td>
            YeonJu Lee, Woong-Yang Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous tumor, with significant variation in malignant cells, posing challenges for treatment and prognosis. However, this heterogeneity offers opportunities for personalized therapy. Methods The consensus non-negative matrix factorization algorithm was employed to analyze single-cell transcriptomic data from CRC, which helped identify malignant cell expression programs (MCEPs). Subsequently, a crosstalk network linking MCEPs with immune/stromal cell trajectory development was constructed using Monocle3 and NicheNet. Additionally, bulk RNA-seq data were utilized to systematically explore the relationships between MCEPs, clinical features, and genetic mutations. A prognostic model was then established through Lasso and Cox regression analyses, integrating clinical data into a nomogram for personalized risk prediction. Furthermore, key genes associated with MCEPs and their potential therapeutic targets were identified using protein-protein interaction networks, followed by molecular docking to predict drug-binding affinity. Results We classified CRC malignant cell transcriptional states into eight distinct MCEPs and successfully constructed crosstalk networks between these MCEPs and immune or stromal cells. A prognostic model containing 15 genes was developed, demonstrating an AUC greater than 0.8 for prognostic evaluation over 1 to 10 years when combined with clinical features. A key drug target gene TIMP1 was identified, and several potential targeted drugs were discovered. Conclusion This study demonstrated that characterization of the malignant cell transcriptional programs could effectively reveal the biological features of highly heterogeneous tumors like CRC and exhibit significant potential in tumor prognosis assessment. Our research provides new theoretical and practical directions for CRC prognosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f29ad2e19f93fc33c2a089984de394bfc868d5e" target='_blank'>
              Abstract 3962: Pan-cancer spatial characterization of tertiary lymphoid structures
              </a>
            </td>
          <td>
            Kyung Serk Cho, Yunhe Liu, Guangsheng Pei, Jianfeng Chen, Tieling Zhou, Khalida M Wani, Alejandra G Serrano, Sharia D Hernandez, Akshaya S. Jadhav, Daiwei Zhang, Hannah C. Beird, Jiahui Jiang, Diana Shamsutdinova, Xinmiao Yan, Yibo Dai, Enyu Dai, Ashley L. Wilson, Anne Waldrep, F. Peng, Wei Lu, Yang Liu, Jimin Min, Paola A. Guerrero, Idania C. Julio, Tong Yin, Shin Hsu Shi, Sabitha Prabhakaran, S. Kopetz, D. Maru, Alexander J. Lazar, A. Jazaeri, Michael Green, Mingyao Li, C. Sautès-Fridman, Wolf H. Fridman, Andrew Futreal, L. Soto, H. Kadara, Anirban Maitra, Suet Yi Leung, Jianjun Gao, Linghua Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8315f0147f2ad2f550ec4867f837baf7efd724be" target='_blank'>
              Abstract 156: Tumor microenvironment profiling of multiplexed images in head and neck cancer reveals spatial features associated with immunotherapy response
              </a>
            </td>
          <td>
            E. Markovits, J. Monkman, Aaron Kilgallon, T. Aung, David L Rimm, A. Kulasinghe
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Wilms tumor (WT) is the most common childhood renal malignancy, with recurrence linked to poor prognosis. Identifying the molecular features of tumor phenotypes that drive recurrence and discovering novel targets are crucial for improving treatment strategies and enhancing patient outcomes. Methods Single-nuclei RNA sequencing (snRNA-seq), spatial transcriptomics (ST), bulk RNA-seq, and mutation/copy number data were curated from public databases. The Seurat package was used to process snRNA-seq and ST data. Scissor analysis was applied to identify tumor subpopulations associated with poor relapse-free survival (RFS). Univariate Cox and LASSO analyses were utilized to reduce features. A prognostic ensemble machine learning model was developed. Immunohistochemistry was used to validate the expression of key features in tumor tissues. The CellChat and Commot package was utilized to infer cellular interactions. The PERCEPTION computational pipeline was used to predict the response of tumor cells to chemotherapy and targeted therapies. Results By integrating snRNA-seq and bulk RNA-seq data, we identified a subtype of Scissor+ tumor cells associated with poor RFS, predominantly derived from cap mesenchyme-like blastemal and fibroblast-like tumor subgroups. These cells displayed nephron progenitor signatures and cancer stem cell markers. A prognostic ensemble machine learning model was constructed based on the Scissor+ tumor signature to accurately predict patient RFS. TGFA was identified as the most significant feature in this model and validated by immunohistochemistry. Cellular communication analysis revealed strong associations between Scissor+ tumor cells and cancer-associated fibroblasts (CAFs) through IGF, SLIT, FGF, and PDGF pathways. ST data revealed that Scissor+ tumor cells were primarily located in immune-desert niche surrounded by CAFs. Despite reduced responsiveness to conventional chemotherapy, Scissor+ tumor cells were sensitive to EGFR inhibitors, providing insights into clinical intervention strategies for WT patients at high risk of recurrence. Conclusion This study identified a relapse-associated tumor subtype resembling nephron progenitor cells, residing in immune-desert niches through interactions with CAFs. The proposed prognostic model could accurately identify patients at high risk of relapse, offering a promising method for clinical risk stratification. Targeting these cells with EGFR inhibitors, in combination with conventional chemotherapy, may provide a potential therapeutic strategy for WT patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05daf7364d02794afe753ef756b89d494ba7b356" target='_blank'>
              Integration of single-nuclei and spatial transcriptomics to decipher tumor phenotype predictive of relapse-free survival in Wilms tumor
              </a>
            </td>
          <td>
            Ran Yang, Lulu Xie, Rui Wang, Yi Li, Yifei Lu, Baihui Liu, Shu-yang Dai, Shan Zheng, Kuiran Dong, Rui Dong
          </td>
          <td>2025-03-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Understanding the intricate tumor microenvironment (TME) is crucial for elucidating the mechanisms underlying the progression of cervical squamous cell carcinoma (CSCC) and its response to anti-PD-1 therapy. Methods In this study, we characterized 50,649 cells obtained from the CSCC for single-cell RNA sequencing and integrated bulk sequencing data from The Cancer Genome Atlas (TCGA) and clinical samples to explore their cell composition, metabolic processes, signaling pathways, specific transcription factors, lineage tracking and response to immunotherapy. In vivo experiments were performed to validate the function of key cell subsets. Results We identified ten major cell type and 35 subsets of stromal and immune cells in TME and observed distinct patterns in the metabolic processes and signaling pathways of these cells between tumor and normal tissues. Furthermore, PCNA clamp-associated factor (PCLAF)+ tumor-associated epithelial cell (TAEpis) was negatively correlated with the number of C-X-C motif chemokine ligand 13 (CXCL13)+ CD8+ T cells, overall survival, and response to anti-programmed cell death-1(PD-1) therapy in patients with CSCC. Both in vivo and in vitro experiments demonstrated that PCLAF+ TAEpis promotes the apoptosis of CD8+ T and tumor growth, while also inhibiting T cell infiltration and function. Conclusion Our findings illuminate the heterogeneity of the complex TME in CSCC and offer evidence supporting PCLAF+ TAEpis as a promising therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03725-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9c463795082212654b2fe4f8e1d20c9792788e" target='_blank'>
              Virtual RNA Inference from Spatial Transcriptomics Reveals Histology-Associated Pathways that Stratify Metastasis Risk in Colorectal Cancer
              </a>
            </td>
          <td>
            Gokul Srinivasan, Minh-Khang Le, Zarif L. Azher, Xiaoying Liu, Louis J. Vaickus, Harsimran Kaur, IV FredKolling, Scott M. Palisoul, L. Perreard, Ken S. Lau, Keluo Yao, Joshua Levy
          </td>
          <td>2025-04-23</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) has advanced our understanding of tissue regionalization by enabling the visualization of gene expression within whole-tissue sections, but current approaches remain plagued by the challenge of achieving single-cell resolution without sacrificing whole-genome coverage. Here we present Spotiphy (spot imager with pseudo-single-cell-resolution histology), a computational toolkit that transforms sequencing-based ST data into single-cell-resolved whole-transcriptome images. Spotiphy delivers the most precise cellular proportions in extensive benchmarking evaluations. Spotiphy-derived inferred single-cell profiles reveal astrocyte and disease-associated microglia regional specifications in Alzheimer’s disease and healthy mouse brains. Spotiphy identifies multiple spatial domains and alterations in tumor–tumor microenvironment interactions in human breast ST data. Spotiphy bridges the information gap and enables visualization of cell localization and transcriptomic profiles throughout entire sections, offering highly informative outputs and an innovative spatial analysis pipeline for exploring complex biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/022fdf3eed7c33ba165a376f1b3fb14ad94870ef" target='_blank'>
              Spotiphy enables single-cell spatial whole transcriptomics across an entire section
              </a>
            </td>
          <td>
            Jiyuan Yang, Ziqian Zheng, Yun Jiao, Kaiwen Yu, Sheetal Bhatara, Xu Yang, Sivaraman Natarajan, Jiahui Zhang, Qingfei Pan, John Easton, Koon-Kiu Yan, Junmin Peng, Kaibo Liu, Jiyang Yu
          </td>
          <td>2025-03-12</td>
          <td>Nature Methods</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1bafbb3ba9f42ba8adab0b213f43d4265484aab" target='_blank'>
              Abstract 2100: Integrative spatial analysis reveals tumor heterogeneity and immune colony niche related to clinical-outcomes in small cell lung cancer
              </a>
            </td>
          <td>
            Chaoqiang Deng, Changbin Zhu, Tao Hu, Jing Wang, Haiquan Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/486a09b849707de5f94be47aaa6a9ceac4469f38" target='_blank'>
              Joint Modeling of Spatial Dependencies Across Multiple Subjects in Multiplexed Tissue Imaging
              </a>
            </td>
          <td>
            Joel Eliason, Arvind Rao, Timothy L. Frankel, Michele Peruzzi
          </td>
          <td>2025-04-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2a1c8fd12302b7cec59cd6c236cab65714760b" target='_blank'>
              Abstract 770: ccRCC metastasis rewires tumor microenvironment to increase immunosuppression and fibrosis
              </a>
            </td>
          <td>
            Yufei Wang, Jae-Won Cho, Y. Laimon, Wenxin Xu, A. B. Sheshdeh, Nithyassree Murugan, , Jon Wee, David Braun, Toni K. Choueiri, Catherine J. Wu, S. Signoretti, Gordon Freeman, M. Hemberg, Wayne A. Marasco
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor heterogeneity, characterized by the presence of diverse cell populations within a tumor, is a key feature of the complex nature of cancer. This diversity arises from the emergence of cells with varying genomic, epigenetic, transcriptomic, and phenotypic profiles over the course of the disease. Host factors and the tumor microenvironment play crucial roles in driving both inter-patient and intra-patient heterogeneity. These diverse cell populations can exhibit different behaviors, such as varying rates of proliferation, responses to treatment, and potential for metastasis. Both inter-patient heterogeneity and intra-patient heterogeneity pose significant challenges to cancer therapeutics and management. In retinoblastoma, while heterogeneity at the clinical presentation level has been recognized for some time, recent attention has shifted towards understanding the underlying cellular heterogeneity. This review primarily focuses on retinoblastoma heterogeneity and its implications for therapeutic strategies and disease management, emphasizing the need for further research and exploration in this complex and challenging area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c59ee10750e92caf754d8bc3c66525cdef7d43" target='_blank'>
              Tumor heterogeneity in retinoblastoma: a literature review
              </a>
            </td>
          <td>
            Rani Pallavi, Bihari lal Soni, Gaurab Kumar Jha, Shalini Sanyal, Azima Fatima, S. Kaliki
          </td>
          <td>2025-04-22</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Neurodegenerative diseases and infections can produce lasting effects on brain function, yet the spatial molecular mechanisms underlying these changes remain poorly understood. Here, we present high-resolution spatial transcriptomics of 40 postmortem brain samples from patients with Parkinson’s disease, frontotemporal dementia, dementia with Lewy bodies, and severe COVID-19. Analyzing over 1.5 million spatially resolved cells across dorsolateral prefrontal cortex and anterior cingulate cortex revealed disease-specific transcriptional signatures with pronounced layer-and region-specificity. In Parkinson’s disease, we identified stressed neurons creating distinctive microenvironmental gradients where metabolic and protein degradation pathways are elevated near stress epicenters, while regenerative processes increase with distance. COVID-19 brains displayed extensive peripheral immune cell infiltration, particularly in the subcortical white matter, accompanied by compromised blood-brain barrier and coordinated neuroinflammatory responses from microglia, astrocytes, and endothelial cells. Integration of miRNA sequencing with spatial transcriptomics uncovered layer-specific regulatory patterns, including neuroinflammation-associated miR-155. This atlas provides unprecedented insights into disease pathology and highlights the critical importance of spatial molecular context in understanding brain disorders. Key Messages [1] A high-resolution single-cell spatial transcriptomics atlas of the dorsolateral prefrontal cortex and anterior cingulate cortex across neurodegenerative conditions and severe COVID-19 [2] Region-and layer-specific transcriptional dysregulation across disease comparisons reveals disease-specific differential vulnerability [3] Metabolically stressed cells found selectively in the anterior cingulate cortex of Parkinson’s disease patients but not in dementia with Lewy bodies, along with detailed characterization of their spatial microenvironment [4] Peripheral immune cell clusters identified in the white matter of the cerebral cortex of COVID-19 patients, with detailed characterization of their spatial microenvironment [5] Integration of bulk miRNA sequencing reveals cortical layer-specific miRNA regulatory patterns">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97c071df2a109d09f7501e38a14ec4947fe8e317" target='_blank'>
              Single-cell Spatial Transcriptomics Reveals Disease-specific Microenvironmental Niches in Neurodegeneration and COVID-19
              </a>
            </td>
          <td>
            Simon Graf, N. Ludwig, Matthias Flotho, Annika Engel, I. F. Diks, Paula Zimmer, Wiebke M. Jürgens-Wemheuer, Arne Wrede, Daniela Mirzac, G. González-Escamilla, Victoria Wagner, Vilas Menon, Sergiu Groppa, P. D. De Jager, Lai Yiwei, Miguel A. Esteban, Walter J. Schulz-Schaeffer, Andreas Keller
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8794d489519dce88daef3e8a367a86c1c1372d6" target='_blank'>
              Abstract 6269: Advanced single-cell RNA-seq reveals complex interactions between circulating tumor cells and immune subtypes in prostate cancer
              </a>
            </td>
          <td>
            H. H. Huynh, Tzu-Hao Chang, S. Yeh, Chia-Ling Hsieh, S. Sung
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c0b646451e8f6a513c46aa0a01996017a71aac6" target='_blank'>
              Abstract 5322: Spatial profiling of the tumor microenvironment in colorectal cancer: Insights into immune, tumor, and stromal interactions using high-dimensional protein analysis
              </a>
            </td>
          <td>
            Enrique Velazquez Villarreal, E.-W. Yang, Fernando C. Diaz, Maria Ninova
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a01a4dfe1a0fcf261f5175b5bc5c42fae74ddd1" target='_blank'>
              Abstract 5312: Murine syngeneic tumor model characterization by ultrahigh-plexed immunofluorescence imaging
              </a>
            </td>
          <td>
            Jaimee Wong, Ngan Doan, Andrew Woolley, Anandi Sawant, Shilpa Keerthivasan, Sunitha Bachawal, Jennifer Sahni, Nicholas Wilson, Amanda Mikels Vigdal, Xiaoyun Liao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc383607243031294dc9e768566a894cfedd572c" target='_blank'>
              Glioblastoma stem cell morphotypes convey distinct cell states and clinically relevant functions
              </a>
            </td>
          <td>
            C. Barelli, Matteo Bonfanti, Filippo Mirabella, Dario Ricca, Negin Alizadehmohajer, Viktoria Sokolova, Alberto Campione, G. Sicuri, C. Peano, Stefania Faletti, Roberto Stefini, N. Kalebic
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d545d97bc8751905973e21664bea8353ccb95ea" target='_blank'>
              Abstract 7475: A comprehensive single-cell atlas of gastric cancer reveals malignant cell state dynamics and plasticity in tumor progression and metastasis
              </a>
            </td>
          <td>
            Enyu Dai, Yanshuo Chu, Yang Liu, Ruiping Wang, Yibo Dai, Jaffer Ajani, Linghua Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8623aa3b6277db74a5df00d2e59a3be79b68bfd2" target='_blank'>
              Abstract 2418: CellClique: Dissecting tumor microenvironments at the single cell level using generative AI and spatial transcriptomics
              </a>
            </td>
          <td>
            Sachini Weerasekara, Natasha Darras, Nicolas Fernandez, Melinda Chen, Alina Ainbinder, Colles Price
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca7ecdbac1d09bcfe33d7f9c228aa944ed8e3ef5" target='_blank'>
              Abstract 6484: Single-cell transcriptomic analysis of hepatocellular carcinoma: elucidating intratumoral heterogeneity and immune microenvironment crosstalk
              </a>
            </td>
          <td>
            Qingyang Zhang, Y. Tsui, Vanilla Xin Zhang, Daniel Wai-Hung Ho, Irene Oi-Lin Ng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Gliomas, including both glioblastoma multiforme (GBM) and lower-grade glioma (LGG), present a substantial challenge in neuro-oncology because of genetic heterogeneity and unsatisfactory prognosis. This study aimed to conduct a comprehensive multi-omics analysis of gliomas using various bioinformatics approaches to identify potential therapeutic targets and prognostic markers. A comprehensive analysis was conducted on 1327 sequencing data samples alongside their relevant clinical information sourced from The Cancer Genome Atlas (TCGA) pertaining to glioblastoma (GBM), low-grade glioma (LGG), the Chinese Glioma Genome Atlas (CCGA) and University of California Santa Cruz Xena (UCSC Xena) datasets. These tools were employed for gene expression profiling, survival analysis, and cell communication mapping. Spatial transcriptomics revealed the localization of mesenchymal (MES)-like malignant tumors, and drug sensitivity analysis was performed to evaluate responses to quinpirole and meropenem. Additionally, the Tumor Immune Dysfunction and Exclusion (TIDE) framework was utilized to gauge the responsiveness to immunotherapy. The MES-like malignant and monocyte/macrophage (mono/macro) cell subsets showed high hallmark scores, playing key roles in the tumor microenvironment. MES-like malignant marker gene scores correlated with overall survival across datasets, whereas mono/macro marker gene scores were significant in the TCGA-LGG and CCGA datasets. Key interactions between these cell types were found, especially with CD14-ITGB2, LGALS1-CD69, and APOE-TREM2. The mono/macro cell subset demonstrated better immune therapy responsiveness, as indicated by lower TIDE scores. Spatial transcriptomics revealed that MES-like malignant tumors are predominantly localized in four distinct regions, with the marker genes CHI3L1 and ADM confirming these locations. Drug sensitivity analysis revealed differential responses of the MES-like malignant cell subset to quinpirole and meropenem. Our results offer fresh perspectives on the differential roles of MES-like malignant and monocyte/macrophage cell subsets in tumor progression and immune modulation, providing novel insights into glioma biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ece6067255a1933f97765246f767fae55de1c3a5" target='_blank'>
              A spatial transcriptomics study of MES-like and mono/macro cells in gliomas
              </a>
            </td>
          <td>
            Boyan Zhao, Jianing Wu, Tiehui Zhang, Mingyang Han, Cheng Zhang, Xuan Rong, Ruotian Zhang, Xin Chen, Fei Peng, Jin Jin, Shiya Liu, Xingli Dong, Shiguang Zhao
          </td>
          <td>2025-04-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Recent advances in spatial transcriptomics (ST) technologies have transformed our ability to profile gene expression while retaining the crucial spatial context within tissues. However, existing ST platforms suffer from high costs, long turnaround times, low resolution, limited gene coverage, and small tissue capture areas, which hinder their broad applications. Here we present iSCALE, a method that predicts super-resolution gene expression and automatically annotates cellular-level tissue architecture for large-sized tissues that exceed the capture areas of standard ST platforms. The accuracy of iSCALE were validated by comprehensive evaluations, involving benchmarking experiments, immunohistochemistry staining, and manual annotation by pathologists. When applied to multiple sclerosis human brain samples, iSCALE uncovered lesion associated cellular characteristics that were undetectable by conventional ST experiments. Our results demonstrate iSCALE’s utility in analyzing large-sized tissues with automatic and unbiased tissue annotation, inferring cell type composition, and pinpointing regions of interest for features not discernible through human visual assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca1037b8c19366713347f95c55ebd14edfcd843" target='_blank'>
              Scaling up spatial transcriptomics for large-sized tissues: uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE
              </a>
            </td>
          <td>
            Amelia Schroeder, Melanie Loth, Chunyu Luo, Sicong Yao, Hanying Yan, Daiwei Zhang, Sarbottam Piya, Edward Plowey, Wen-Tuo Hu, Jean R. Clemenceau, Inyeop Jang, Minji Kim, Isabel Barnfather, Su Jing Chan, Taylor L. Reynolds, Thomas Carlile, Patrick Cullen, Ji-Youn Sung, Hui-Hsin Tsai, J. Park, Tae Hyun Hwang, Baohong Zhang, Mingyao Li
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Spatial transcriptomics has revolutionized our understanding of tissue organization by simultaneously capturing gene expression and spatial localization within intact tissues. However, analyzing these increasingly complex datasets requires specialized expertise across computational biology, statistics, and biological context. To address this challenge, we introduce the Spatial Transcriptomics AI Agent (STAgent), an autonomous multimodal agentic AI that integrates multimodal large language models (LLMs) with specialized computational tools to transform weeks-long analysis tasks into minutes of automated processing. Unlike conventional machine learning approaches that are limited to narrow, predefined tasks, STAgent leverages the emergent capabilities of multimodal LLMs – such as flexible reasoning, contextual understanding, and cross-modal integration – which allow it to adapt to novel data, execute multi-step analyses, and generate biologically meaningful insights with minimal human input. STAgent enables autonomous deep research through integrated capabilities, including dynamic code generation for complex analytical workflows, visual reasoning for interpreting spatial patterns, real-time retrieval of relevant peer-reviewd scientific literature, and synthesis of comprehensive, actionable reports. We applied STAgent to investigate the in vivo maturation of human stem cell-derived pancreatic cells (SC-pancreas) transplanted into immunodeficient mice. We generated single-cell spatial transcriptomics data spanning multiple developmental timepoints. STAgent autonomously (1) identified the maturation of initially scattered endocrine cells into well-defined islet-like structures, with predominantly peripheral α-cells surrounding β-cell cores supported by an expanding mesenchymal network; (2) revealed strengthening endocrine-endocrine cell interactions over time and, through context-aware gene set analysis, uncovered spatially resolved biological processes driving maturation; (3) unlike traditional analytical approaches, STAgent offers mechanistic explanations of spatial patterns, contextualizing findings with relevant literatures and developing cohesive insights into human pancreatic development. This agentic approach establishes a new paradigm in spatial transcriptomics analysis by substantially lowering the expertise barrier and reducing analysis time, accelerating biological and biomedical discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c45bfa74d72bc131f7254ae28e5d7ee1abd425f" target='_blank'>
              Spatial transcriptomics AI agent charts hPSC-pancreas maturation in vivo
              </a>
            </td>
          <td>
            Zuwan Lin, Wenbo Wang, Arnau Marin-Llobet, Qiang Li, Samuel D. Pollock, Xin Sui, Almir Aljovic, Jaeyong Lee, Jongmin Baek, Ningyue Liang, Xinhe Zhang, Connie Kangni Wang, Jiahao Huang, Mai Liu, Zihan Gao, Hao Sheng, Jin Du, Stephen J. Lee, Brandon Wang, Yichun He, Jie Ding, Xiao Wang, Juan R. Alvarez-Dominguez, Jia Liu
          </td>
          <td>2025-04-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Prostate cancer (PC) is the most frequently diagnosed malignancy among men and contributes significantly to cancer-related mortality. While recent advances in in vitro PC modeling systems have been made, there remains a lack of robust preclinical models that faithfully recapitulate the genetic and phenotypic characteristics across various PC subtypes—from localized PC (LPC) to castration-resistant PC (CRPC)—along with associated stromal cells. Here, we established human PC assembloids from LPC and CRPC tissues by reconstituting tumor organoids with corresponding cancer-associated fibroblasts (CAFs), thereby incorporating aspects of the tumor microenvironment (TME). Established PC organoids exhibited high concordance in genomic landscape with parental tumors, and the tumor assembloids showed a higher degree of phenotypic similarity to parental tumors compared to tumor organoids without CAFs. PC assembloids displayed increased proliferation and reduced sensitivity to anti-cancer treatments, indicating that PC assembloids are potent tools for understanding PC biology, investigating the interaction between tumor and CAFs, and identifying personalized therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/105fe3ccf7c35aabcf3c81f5242c91ac5e7680b1" target='_blank'>
              Generation of prostate cancer assembloids modeling the patient-specific tumor microenvironment
              </a>
            </td>
          <td>
            Juhee Lee, Yunhee Kim, Cheol Lee, S. Jeon, Hae Seo, Jongwon Lee, Jungmin Choi, Minyong Kang, Eunjee Kim, Kunyoo Shin
          </td>
          <td>2025-03-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99969b328faea200d4bc4333f10cd0e2095990a" target='_blank'>
              Abstract 5302: Mapping the tumor microenvironment at single-cell resolution with Stereo-seq v1.3
              </a>
            </td>
          <td>
            Jiajun Zhang, Xing Liu, Zexian Zeng, Rui Zhang, Tibby Tianbi Duan, Jing Guo, Liang Wu, Sha Liao, Ao Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfaf1ffd1f6785c103e919dc021f81aaaeb67090" target='_blank'>
              Abstract 5316: Multi-platform spatial profiling reveals intra-tumor heterogeneity in intrinsic subtypes and immune markers in breast tumors from Kenyan patients
              </a>
            </td>
          <td>
            Mustapha Abubakar, Shahin Sayed, H. Koka, Scott Lawrence, Karun Mutreja, X. Hua, V. Oluoch, Veronica M Ngundo, Zaitun Ajuoga, Difei Wang, Chad A. Highfill, Petra Lenz, Maria Brown, Aaron M. Rozeboom, Kristine Jones, A. Hutchinson, B. Hicks, Paul S. Albert, Francis W. Makokha, S. Ambs, Jonine Figueroa, Jianxin Shi, Xiaohong R. Yang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195e5c9d321296053fe8059a5c1ae5b1f465a2aa" target='_blank'>
              Abstract 5054: Understanding the genetic heterogeneity within clonal spheroids and its interaction with the tumor microenvironment
              </a>
            </td>
          <td>
            Anshu Yadav, Rupesh Chaturvedi, Rana P. Singh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c5186055b7098bddfe8ece68d797911822c432e" target='_blank'>
              Abstract 6290: sc2DAT: workflow for targeting tumor subpopulations of single cells
              </a>
            </td>
          <td>
            Giacomo B. Marino, Nasheath Ahmed, Daniel J. B. Clarke, Avi Ma’ayan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f3af6173f4bd387223a85f81701f7e3b9e33817" target='_blank'>
              Abstract 1241: Deciphering heterogeneity and tumor evolution in bladder cancer using multi-omics and liquid biopsies from rapid autopsies
              </a>
            </td>
          <td>
            P. Itagi, Samantha L. Schuster, Sonali Arora, Patricia Galipeau, Thomas W Persse, F. Vakar‐López, John K. Lee, P. Grivas, R. Montgomery, Jonathan L. Wright, Hung-Ming Lam, Andrew C. Hsieh, Gavin Ha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Background Gastric cancer is a highly aggressive malignancy characterized by a complex tumor microenvironment (TME). Cancer-associated fibroblasts (CAFs), which are a key component of the TME, exhibit significant heterogeneity and play crucial roles in tumor progression. Therefore, a comprehensive understanding of CAFs is essential for developing novel therapeutic strategies for gastric cancer. Methods This study investigates the characteristics and functional information of CAF subtypes and explores the intercellular communication between CAFs and malignant epithelial cells (ECs) in gastric cancer by analyzing single-cell sequencing data from 24 gastric cancer samples. CellChat was employed to map intercellular communication, and Seurat was used to integrate single-cell sequencing data with spatial transcriptome data to reconstruct a comprehensive single-cell spatial map. The spatial relationship between apCAFs and cancer cells was analyzed using multicolor immunohistochemistry. Results Cells were categorized into nine distinct categories, revealing a positive correlation between the proportions of epithelial cells (ECs) and fibroblasts. Furthermore, six fibroblast subpopulations were identified: inflammatory (iCAFs), pericytes, matrix (mCAFs), antigen-presenting (apCAFs), smooth muscle cells (SMCs), and proliferative CAFs (pCAFs). Each of these subpopulations was linked to various biological processes and immune responses. Malignant ECs exhibited heightened intercellular communication, particularly with CAF subpopulations, through specific ligand-receptor interactions. High-density regions of CAF subpopulations displayed spatial exclusivity, with pericytes serving as a source for iCAFs, mCAFs, and apCAFs. Notably, malignant ECs and apCAFs showed increased interactions, with certain ligand-receptor pairs potentially impacting the prognosis of gastric cancer. Multiplex immunohistochemistry (mIHC) confirmed the close spatial proximity of apCAFs to cancer cells in gastric cancer. Conclusion Our study provided a comprehensive characterization of CAF heterogeneity in gastric cancer and revealed the intricate intercellular networks within the TME. The identified CAF subpopulations and their interactions with malignant cells could serve as potential therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06376-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24163c4c446c34923a67d83b5c03139aa6a3cb4" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics reveal the heterogeneity and intercellular communication of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xijie Zhang, Bo Ren, Bo Liu, Rui Wang, Sen Li, Yuzhou Zhao, Wence Zhou
          </td>
          <td>2025-03-18</td>
          <td>Journal of Translational Medicine</td>
          <td>2</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1870d444cf3f0fec43c774decd3ef3d91c4b6138" target='_blank'>
              Abstract 6622: Dissecting in situ states of invading glioblastoma cells using single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Toshiro Hara
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d379694e8835fc398019d9f1f787e108dd62b53d" target='_blank'>
              Abstract 1896: Spatial proteomics and tumor-immune phenotyping identifies a diverse immune landscape in syngeneic murine tumor models
              </a>
            </td>
          <td>
            Arindam Bose, E. Alonzo, Sophie Struble, Jeremy Fisher, R. Heil-Chapdelaine, Natasha F. Diaz Granados, Vasundhara Agrawal
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dbf2dbcf6c4b1d8a1dbdd8e9dcfc190f14aeaa6" target='_blank'>
              Methylation-Based Deconvolution Unveils Glioblastoma Heterogeneity and Cell-Type Composition Linked to Patient Survival
              </a>
            </td>
          <td>
            Aviel Iluz, Nir Lavi, Hanna Charbit, Mijal Gutreiman, Masha Idelson, Debora Steiner, Etti Ben-Shushan, A. Zick, Amir Eden, A. Mordechai, Moscovici Samuel, Y. Fellig, Alexander Lossos, Joshua Moss, Benjamin E. Reubinoff, I. Lavon
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Cancer cells acquire necessary functional capabilities for malignancy through the influence of the nervous system. We evaluate the extent of neural infiltration within the tumor microenvironment (TME) across multiple cancer types, highlighting its role as a cancer hallmark. We identify cancer-related neural genes using 40 bulk RNA-seq datasets across 10 cancer types, developing a predictive score for cancer-related neural infiltration (C-Neural score). Cancer samples with elevated C-Neural scores exhibit perineural invasion, recurrence, metastasis, higher stage or grade, or poor prognosis. Epithelial cells show the highest C-Neural scores among all cell types in 55 single-cell RNA sequencing datasets. The epithelial cells with high C-Neural scores (epi-highCNs) characterized by increased copy number variation, reduced cell differentiation, higher epithelial–mesenchymal transition scores, and elevated metabolic level. Epi-highCNs frequently communicate with Schwann cells by FN1 signaling pathway. The co-culture experiment indicates that Schwann cells may facilitate cancer progression through upregulation of VDAC1. Moreover, C-Neural scores positively correlate with the infiltration of antitumor immune cells, indicating potential response for immunotherapy. Melanoma patients with high C-Neural scores may benefit from trametinib. These analyses illuminate the extent of neural influence within TME, suggesting potential role as a cancer hallmark and offering implications for effective therapeutic strategies against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/627d476afca905fcf3888b082a1804bf193de61b" target='_blank'>
              Unveiling a novel cancer hallmark by evaluation of neural infiltration in cancer
              </a>
            </td>
          <td>
            Qi Dong, Yingying Guo, Chen Lv, Lingxue Ren, Bo Chen, Yan Wang, Yang Liu, Mingyue Liu, Kaidong Liu, Nan Zhang, Linzhu Wang, Shaocong Sang, Xin Li, Yang Hui, Haihai Liang, Yu Gu
          </td>
          <td>2025-03-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Approximately 15-20% of head and neck cancer squamous cell carcinoma (HNSCC) patients respond favorably to immune checkpoint blockade (ICB). Previous single-cell RNA-Seq (scRNA-Seq) studies identified immune features, including macrophage subset ratios and T-cell subtypes, in HNSCC ICB response. However, the spatial features of HNSCC-infiltrated immune cells in response to ICB treatment need to be better characterized. Methods Here, we perform a systematic evaluation of cell interactions between immune cell types within the tumor microenvironment using spatial omics data using complementary techniques from both 10X Visium spot-based spatial transcriptomics and Nanostring CosMx single-cell spatial omics with RNA gene panel including 435 ligands and receptors. In this study, we used integrated bioinformatics analyses to identify cellular neighborhoods of co-localizing cell types in single-cell spatial transcriptomics and proteomics data. In addition, we used both publicly available scRNA-Seq and in-house spatial RNA-Seq data to identify spatially constrained Ligand-Receptor interactions in Responder patients. Results With 522,399 single cells profiled with both RNA and protein from 26 patients, in addition to spot-resolved spatial RNA-Seq from 8 patients treated with ICB together with bioinformatics analysis of publicly available single-cell and bulk RNA-Seq, we have identified a spatial and cell-type specific context-dependency of myeloid and T cell interaction difference between Responders and Non-Responders. We defined further cellular neighborhood and the sources of chemokine CXCL9/10-CXCR3 interactions in Responders, emerging targets in ICB, as well as CXCL16-CXCR6, CCL4/5-CCR5, and other underappreciated and potential markers and targets for ICB response in HNSCC. In addition, we have contributed a rich data resource of cell-cell Ligand Receptor interactions for the immunotherapy and HNSCC research community. Discussion Our work provides a comprehensive single-cell and spatial atlas of immune cell interactions that correlate with response to ICB in HNSCC. We showcase how integrating multiple technologies and bioinformatics approaches can provide new insights into potential immune-based biomarkers of ICB response. Our results suggested refining future studies using preclinical animal models in a more context-specific manner to elucidate potential underlying mechanisms that lead to improved ICB responses. What is already known on this topic Most cancer patients still do not experience clinical benefits from immune checkpoint blockade (ICB), necessitating the development of response biomarkers and new immunotherapeutic targets. What this study adds Here, we use integrated high-dimensional omics and bioinformatics approaches to identify immune cell-cell interaction markers associated with ICB response in patients with Head and neck squamous cell carcinoma. How this study might affect research, practice or policy We identified spatial and cell-type specificity of Ligand-Receptor interactions between myeloid and T cells in ICB Responder patients that may help inform further mechanistic studies and biomarker development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ece16324064b70206dfb8fd687bfc5db3089e8d7" target='_blank'>
              Integrated single-cell and spatial analysis identifies context-dependent myeloid-T cell interactions in head and neck cancer immune checkpoint blockade response
              </a>
            </td>
          <td>
            Athena E. Golfinos-Owens, Taja Ložar, Parth Khatri, Rong Hu, Paul M. Harari, Paul F. Lambert, Megan B Fitzpatrick, Huy Q. Dinh
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43978659ce38ef19fbe0b48f8cd99b848f7e9b3" target='_blank'>
              Abstract 7490: Uncovering rare cellular populations in esophageal preneoplasia through genetic perturbation tracking and scRNA-seq
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Jae-Il Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9cb5b1581890bc09659329e338f756aca67f340" target='_blank'>
              Abstract 2090: Integrative spatial and single-cell analysis elucidates tumor microenvironment heterogeneity in NSCLC
              </a>
            </td>
          <td>
            Qiming Zhou, Jiadong Zhou, Beixi Wang, Jingwen Shi, Han Yan, Yun Zhang, Peilin Zhang, Zhirong Shen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dfbdf3e1bc24e3e89e9ac8874b0428cc6cc6092" target='_blank'>
              Targeting CAFs-Mediated Stromal Signaling in a Patient-Derived Organotypic Colorectal Tumor Model.
              </a>
            </td>
          <td>
            Astha Lamichhane, Prasiddha Guragain, Jacob Heiss, Pouria Rafsanjani Nejad, Anju Rana Magar, Nicholas Ciavattone, Seema Agarwal, G. Luker, Hossein Tavana
          </td>
          <td>2025-04-14</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf8353686e8bd9d104e11c8e9f95fa9e551e49d0" target='_blank'>
              Abstract 6265: Cross-tissue atlas of human disease identifies tumor-specific components in tumor microenvironment
              </a>
            </td>
          <td>
            Seong-Kun Kim, Sungmin Cheong, Suho Lee, Yeju Kim, Jong-Eun Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab07960659fe1e798a089508fbf87b3452b9a753" target='_blank'>
              Abstract 3116: High-definition spatial transcriptomics reveals microenvironment features associated with resistance to anti-PD1 therapies in lung cancer
              </a>
            </td>
          <td>
            Corinne Ramos, Sandra Delebecq, Richard Ruez, Victor Senechal
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) are pivotal components of the breast cancer (BC) tumor microenvironment (TME), significantly influencing tumor progression and response to therapy. However, the heterogeneity and specific roles of TAM subpopulations in BC remain inadequately understood. Methods We performed an integrated analysis of single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data from BC patients to comprehensively characterize TAM heterogeneity. Utilizing the MetaTiME computational framework and consensus clustering, we identified distinct TAM subtypes and assessed their associations with clinical outcomes and treatment responses. A machine learning-based predictive model was developed to evaluate the prognostic significance of TAM-related gene expression profiles. Results Our analysis revealed three distinct TAM subgroups. Notably, we identified a novel macrophage subtype, M_Macrophage-SPP1-C1Q, characterized by high expression of SPP1 and C1QA, representing an intermediate differentiation state with unique proliferative and oncogenic properties. High infiltration of M_Macrophage-SPP1-C1Q was significantly associated with poor overall survival (OS) and chemotherapy resistance in BC patients. We developed a Random Forest (RF)-based predictive model, Macro.RF, which accurately stratified patients based on survival outcomes and chemotherapy responses, independent of established prognostic parameters. Conclusion This study uncovers a previously unrecognized TAM subtype that drives poor prognosis in BC. The identification of M_Macrophage-SPP1-C1Q enhances our understanding of TAM heterogeneity within the TME and offers a novel prognostic biomarker. The Macro.RF model provides a robust tool for predicting clinical outcomes and guiding personalized treatment strategies in BC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03750-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/287d3599b1ab22b44316058fde7d65a5ef96c17d" target='_blank'>
              Integrated single-cell and bulk transcriptomic analysis identifies a novel macrophage subtype associated with poor prognosis in breast cancer
              </a>
            </td>
          <td>
            Qing Wang, Yushuai Yu, Liqiong Ruan, Mingyao Huang, Wei Chen, Xiaomei Sun, Jun Liu, Zirong Jiang
          </td>
          <td>2025-03-27</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e5ba5dfba618dfefbeb55cda17b746756b7211" target='_blank'>
              Abstract 2570: Deciphering local immunosuppression in pancreatic cancer: targeting CAF-myeloid cell crosstalk to enhance immune response
              </a>
            </td>
          <td>
            Marie C. Hasselluhn, Lukas J. Vlahos, Yuanqing Xue, Dafydd Thomas, Quin T. Waterbury, Allison C. Hess, Tanner C. Dalton, Amanda R. Decker-Farrell, Á. C. García, Stephen A. Sastra, Carmine F. Palermo, Timothy C Wang, Andrea Califano, K. Olive
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73bf7de11a69b6fed18f060c330341fde4d20ec0" target='_blank'>
              Cellular characterization of the peritumoral brain zone of glioblastomas: heterogenous nature and clinical relevance
              </a>
            </td>
          <td>
            Olaya de Dios, J. Romero, M. A. Ramírez-González, B. Herránz, Alicia Avis, A. Ramos, J. Sepúlveda-Sánchez, R. Gargini, B. Meléndez, Pedro González, L. Jimenez-Roldan, Guillermo García-Posadas, A. Hernández-Laín, Á. Pérez-Núñez, P. Sánchez-Gómez
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73dbf54f5b2a0ce96fdfefae2ba812ee9c7ea68b" target='_blank'>
              The 1000+ mouse project: large-scale spatiotemporal parametrization and modeling of preclinical cancer immunotherapies
              </a>
            </td>
          <td>
            Adam L. Kenet, S. Achar, Alka Dwivedi, John Buckley, M. Pouzolles, Haiying Qin, C. Chien, N. Taylor, Grégoire Altan-Bonnet
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3bcf08a317f0fb50f59a7b481b2ba31a08d66c" target='_blank'>
              Abstract 7493: Spatially resolved insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, Padma Kadiyala, Alexander Bray, Jamie N. Mills, Hannah R Watkoske, Arvind Rao, J. Okoye, Atul Deshpande, Jiaqi Shi, E. Fertig, Filip Bednar, Yaqing Zhang, Eileen S. Carpenter, Timothy L. Frankel, M. Pasca di Magliano
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183f97a84615b6dbc6366f5d2cb1c5539c88c0f6" target='_blank'>
              Abstract 7466: Advancing tumor characterization and immune phenotyping through multiomic in situ profiling of RNA and proteins
              </a>
            </td>
          <td>
            Thanutra Zhang, Smritee Dadhwal, Joshua Thao, Anatalia Robles, Cheyenne Christopherson, Suyog Pathare, Kristen Pham, Minkyung Park, Hsin Wen Chang, Janine Hensel, Cedric Espenel, Hayato Ikoma, Gabriele Girelli, Paul Ryvkin, Veronica Gonzalez Muñoz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a32f7726bad7de5bd6ab6b676111c1867233e6" target='_blank'>
              Precancerous Cells Initiate Glioblastoma Evolution and Contribute to Intratumoral Heterogeneity.
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, Jong Hee Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, J. H. Lee
          </td>
          <td>2025-04-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cac8b257d411c793fd0016b4f5f926fd3d98de50" target='_blank'>
              Abstract 5253: Immune landscape analysis of solid tumors via spatially guided cell isolation for enhanced immunotherapeutic development
              </a>
            </td>
          <td>
            Elie Abi Khalil, Yichao Wu, Jingtian Zheng, Joshua Plank, Alexander Lippert, Steve Seung‐Young Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed241f7ab2b6bafb6e0e97061f34d2c422a2e850" target='_blank'>
              Abstract 5076: Unravelling melanoma brain metastasis tumor microenvironment characteristics using spatial transcriptomics
              </a>
            </td>
          <td>
            Clemence J. Belle, S. Brosda, V. Bonazzi, Victor Bulteau, Thomas Stuart, Zherui Xiong, Duncan Lambie, Peter A. Johansson, L. Aoude, Kalpana Patel, Samantha J Stehbens, Mitchell S. Stark, N. Haass, Wen Xu, M. Pinkham, Matthew C. Foote, Sarah Olson, Victoria Atkinson, A. Kulasinghe, Quan H. Nguyen, Andrew Barbour
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are highly heterogeneous cells and important components of the breast tumor microenvironment (TME). However, their role and clinical value in ER-positive breast cancer have not been fully clarified. Our study aims to comprehensively characterize the heterogeneity, potential biological functions, and molecular mechanisms of CAFs in ER-positive breast cancer within the tumor microenvironment using multi-omics data, to provide new strategies for the diagnosis and treatment of ER-positive breast cancer patients. In this study, we found that COL1A2(+) MMP1(+) and COL1A2(+) MMP1(-) CAFs were associated with unfavorable prognosis. The dynamic evolution and cell-cell communications of CAFs were analyzed, revealing that COL1A2(+) MMP1(+/-) CAFs show extensive crosstalk with tumor-associated macrophages (TAMs), contributing to an immunosuppressive TME. Moreover, the somatic mutation of TP53 may be a potential indicator for evaluating the infiltration of COL1A2(+) MMP1(+/-) CAFs. Finally, an MRI-based radiomic model was constructed to estimate the abundance of these CAFs. In conclusion, our findings provide a theoretical basis for targeting CAFs and offer a noninvasive approach to evaluate the infiltration level of COL1A2(+) MMP1(+/-) CAFs. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03705-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9547a039c0fe93efd8fd0787388be5b2aa0012a6" target='_blank'>
              Integrating spatial and single-cell transcriptomes reveals the role of COL1A2(+) MMP1(+/-) cancer-associated fibroblasts in ER-positive breast cancer
              </a>
            </td>
          <td>
            Zhi-Hao Yu, Huan-Ling Xu, Shuo Wang, Yingxi Li, Guixin Wang, Yao Tian, Zhaohui Chen, Wenbin Song, Long He, Xin Wang, Xuchen Cao, Yue Yu
          </td>
          <td>2025-03-07</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Schwannomas are nerve sheath tumors arising at cranial and peripheral nerves, either sporadically or in patients with a schwannomatosis-predisposition syndrome. There is limited understanding of the transcriptional heterogeneity of schwannomas across genetic backgrounds and anatomic locations. Methods Here, we prospectively profile by single-cell full-length transcriptomics tumors from 22 patients with NF2-related schwannomatosis, non-NF2-related schwannomatosis, and sporadic schwannomas, resected from cranial and peripheral nerves. We profiled 11,373 cells (after QC), including neoplastic cells, fibroblasts, T cells, endothelial cells, myeloid cells, and pericytes. Results We characterize the intra-tumoral genetic and transcriptional heterogeneity of schwannoma, identifying six distinct transcriptional metaprograms, with gene signatures related to stress, myelin production, antigen presentation, interferon signaling, glycolysis, and extracellular matrix. We demonstrate the robustness of our findings with analysis of an independent cohort. Conclusions Overall, our atlas describes the spectrum of gene expression across schwannoma entities at the single-cell level and will serve as an important resource for the community. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01462-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a8868eb66d6c72488d55b6daa0e1d4183e3adcb" target='_blank'>
              A single-cell atlas of Schwannoma across genetic backgrounds and anatomic locations
              </a>
            </td>
          <td>
            L. G. Gonzalez Castro, Avishai Gavish, L. Bussema, Christopher W Mount, Cyril Neftel, M. Nomura, E. Chiocca, W. Bi, Omar Arnaout, Fred G. Barker, Justin M. Brown, Justin T Jordan, Tracy T Batchelor, A. Stemmer-Rachamimov, Scott R Plotkin, I. Tirosh, M. Suvà
          </td>
          <td>2025-04-11</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e688452c1e795984dc287acda606201c118b68e6" target='_blank'>
              Abstract 5241: Mapping spatial 3D genome landscape of breast cancer cells in different tumor immune microenvironments using onco-immune co-culture as a model
              </a>
            </td>
          <td>
            Jude Dunne, Huy Nguyen, Shyamtanu Chattoraj
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Solid tumors develop within a complex environment called the tumor microenvironment (TME), which is sculpted by the presence of other cells, such as cancer‐associated fibroblasts (CAFs) and immune cells like macrophages (Mφs). Despite the presence of immune cells, tumor cells orchestrate a tumor‐supportive environment through intricate interaction with the components of the TME. However, the specific mechanism by which this intercellular dialogue is regulated is not fully understood. To that end, the development of an organotypic 3D breast TME‐on‐a‐chip (TMEC) model, integrated with single‐cell RNA sequencing analysis, is reported to mechanistically evaluate the progression of triple‐negative breast cancer (TNBC) cells in the presence of patient‐derived CAFs and Mφs. Extensive functional assays, including invasion and morphometric characterization, reveal the synergistic influence of CAFs and Mφs on tumor cells. Furthermore, gene expression and pathway enrichment analyses identify the involvement of the KYNU gene, suggesting a potential immune evasion mechanism through the kynurenine pathway. Lastly, the pharmacological targeting of the identified pathway is investigated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8de452a3f7784c1593c1ff6c237b52e31623e79" target='_blank'>
              Tumor Microenvironment On‐A‐Chip and Single‐Cell Analysis Reveal Synergistic Stromal–Immune Crosstalk on Breast Cancer Progression
              </a>
            </td>
          <td>
            Kalpana Ravi, Yining Zhang, Lydia Sakala, Twinkle Jina Minette Manoharan, Barbara Pockaj, Joshua LaBaer, Jin G. Park, Mehdi Nikkhah
          </td>
          <td>2025-03-08</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c127048e0d0f262eff5e6fdada1e6625367af0dd" target='_blank'>
              Abstract 5227: Extrachromosomal MYC-paralogs amplification shapes suppressive tumor microenvironment in small cell lung cancer
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Ence Yang, Xiao Li
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/defa96334d149dd89f471b7d8cee1c2ef5d2f853" target='_blank'>
              Multifaceted roles of OCT4 in tumor microenvironment: biology and therapeutic implications.
              </a>
            </td>
          <td>
            Wenjie Chen, Ying-Jie Wang
          </td>
          <td>2025-04-14</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract In solid tumours, malignant cells develop relationships with nonmalignant stromal cells to support tumour growth, tissue structure, and nutrient supply. In a recent issue of this journal, Zheng and Guo catalogue the cellular landscape in chordoma using a combination of single‐cell and spatial transcriptomics. Despite the mesenchymal nature of chordoma, malignant cells retain expression of epithelial markers, in addition to mesenchymal, partial‐EMT, and stem‐cell signatures. Cancer‐associated fibroblasts (CAFs) are among the most abundant stromal cells and the authors define an inflammatory CAF subtype (iCAF), which is associated with poor outcome and tumour invasion. It is proposed that iCAFs arise from normal fibroblasts during malignant tumour progression and play a causative role in driving an invasive poor prognosis tumour phenotype. Recent reports by this and other groups have separately catalogued cell populations, including CAFs and immune cells in chordoma. The next challenge will be to integrate findings from these distinct studies to allow a consensus to be reached regarding cellular heterogeneity within chordoma, and to allow comparison of CAF populations with those found in other tumour types. Comparison of CAF functions in these predominantly mesenchymal tumours with epithelial solid tumours may reveal interesting lessons about the diverse phenotypes CAFs can bring to distinct tumour ecosystems. Understanding the role of CAFs in chordoma progression may also lead to therapeutic opportunities, but separation of correlation and causation in CAF regulation of tumour phenotypes remains a significant challenge. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c222d4cf7dba2a9f6cb176f5eccff6fd31fd816" target='_blank'>
              Cancer‐associated fibroblast heterogeneity in chordoma†
              </a>
            </td>
          <td>
            Jack C Henry, A. Cameron
          </td>
          <td>2025-03-15</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0761133026ec3ba173f2c421844c6435e13b010" target='_blank'>
              Real-Time Tracking of Tumor Invasion Dynamics in Organotypic Brain Slices at Subcellular Resolution
              </a>
            </td>
          <td>
            Jana B. Petr, Meng-Syuan Lin, Dina Hochuli, Andreas Hierlemann, Martin Baumgartner, Mario M. Modena
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eeebf7fbc5adbc5ad13034f3b96cb93919d7660" target='_blank'>
              Abstract 5265: Single cell and spatial analysis unravels the immunosuppressive tumor microenvironment, HPV-associated features, and potential immunotherapy response in vulvar squamous cell carcinoma
              </a>
            </td>
          <td>
            Athena E. Golfinos-Owens, Taja Ložar, Evie H Carchman, Paul F. Lambert, Megan B Fitzpatrick, Huy Q. Dinh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Model organisms allowing for longitudinal examinations of liver disease pathogenesis are pivotal for the development of new therapeutic modalities. The eastern North American woodchuck develops chronic hepatitis and liver cancer after woodchuck hepatitis virus (WHV) infection, mirroring aspects of the natural history of the human hepatitis B virus (HBV). However, the cellular landscape of the woodchuck liver and the cell-level relevance of WHV infection to HBV infection is currently uncharacterized. Methods We employed single-cell RNA sequencing (RNA-seq) to generate an atlas of healthy woodchuck liver (63,389 cells, n=8) and peripheral blood mononuclear cells (PBMCs) (26,972 cells, n=7). Cell-specific and hepatic zonation gene signatures were validated using spatial transcriptomics (n=1). We employed our atlas to examine immune activation in stimulated precision cut-liver slices (PCLS) and disease-related pathway activation in chronic WHV infection (11,797 cells, n=3). We further employed our atlas to examine shared disease pathways between WHV infection and human HBV infection. Results Our atlas revealed woodchuck hepatic cellular diversity comparable to human and murine livers. Applying single-nucleus RNA-seq to PMA/ionomycin-stimulated precision cut liver slices revealed inflammation-associated activation signatures in T cell, myeloid and endothelial cell compartment. Finally, we describe intrahepatic T cells in chronic WHV hepatitis with both exhaustion and activation-associated signatures that resemble intrahepatic T cell genes signatures described in human chronic HBV. Conclusions We present a multi-omic atlas of healthy, diseased and ex vivo stimulated woodchuck liver. By identifying shared pathological processes between WHV and HBV infections, our findings reinforce the value of this preclinical model in translational research. This resource aims to advance studies on HBV pathogenesis and oncogenesis to speed the development of novel therapeutic strategies. Impact and Implications/Lay summary The liver plays important roles in metabolism, detoxification, and immune processes; liver transplantation is often the only treatment option for severe chronic liver diseases. Therefore, developing animal models that reflect human liver disease and can be studied throughout the disease course is crucial for the discovery of new treatment options. The woodchuck is an animal that develops chronic hepatitis and liver cancer after infection with woodchuck hepatitis virus, which models the human hepatitis type B virus infection (HBV) and associated hepatic carcinoma. However, our understanding of the cells that compose the woodchuck liver is limited, making it challenging to design and test cell-based therapeutics. In this study, we atlased the healthy and chronically infected woodchuck liver, found that liver cell types in woodchuck resemble those in humans, and employed the atlas to show similarities between WHV and HBV at the cell level, reinforcing the potential of WHV-infected woodchuck as a model for human HBV disease. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8ef9bc878aedeb55d6c6cc4171d96693eccf30b" target='_blank'>
              A Single-Cell Woodchuck Liver Atlas Identifies Healthy and Disease-related Cellular Programs Conserved in Human
              </a>
            </td>
          <td>
            Zoe A. Clarke, Jawairia Atif, Xinle Wang, Lewis Y. Liu, Lawrence Wood, Damra Camat, Sai W. Chung, Xue-Zhong Ma, Justin Manuel, Si Lok, Timothy Lau, C. Thoeni, Tomasz I. Michalak, Ian Mcgilvray, Gary D. Bader, S. Macparland
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24fe83552a51ec8685227ae6ed241734102c7207" target='_blank'>
              Differential interfacial tension between oncogenic and wild-type populations forms the mechanical basis of tissue-specific oncogenesis in epithelia
              </a>
            </td>
          <td>
            Amrapali Datta, Phanindra Dewan, Aswin Anto, Tanya Chhabra, Tanishq Tejaswi, Sindhu Muthukrishnan, Akshar Rao, Sumantra Sarkar, Medhavi Vishwakarma
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b35fc24142fa9e46cb8827397a0b39028b4c9b9" target='_blank'>
              Abstract 6471: Spatial multiomics reveals T-cell activation and exhaustion states in the tumor microenvironment
              </a>
            </td>
          <td>
            Steve Zhou, Yifan Wang, Debia Wakhloo, A. Dikshit, Li-chong Wang, Maithreyan Srinivasan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ed535cc83be9bb714c00d9bb8580fac8efc104e" target='_blank'>
              Abstract 908: Decoding the role of intrinsic pancreatic cellular signaling in shaping the immunosuppressive tumor microenvironment
              </a>
            </td>
          <td>
            Bushangqing Liu, Ethan S Agritelley, Junping Wei, Gangjun Lei, Tao Wang, Jason McBane, Daniel Nussbaum, Z. Hartman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c001caff1773dd2045b65bb8abc13f8680dd3d4" target='_blank'>
              Pan-Cancer PDOs Preserve Tumor Heterogeneity and Uncover Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Hui-Hsuan Kuo, B. Bhinder, H. Gokozan, Kathryn Gorski, Pooja Chandra, J. Manohar, Daniela Guevara, John Otilano, Jenna E Moyer, Marvel Tranquille, Sarah Ackermann, Jared Capuano, Cynthia Cheung, Thomas Anthony Caiazza, Phoebe L. Reuben, Anastasia Murray Tsomides, Adriana Irizarry, M. Sigouros, David Wilkes, Abigail King, Troy Kane, Majd Al Assaad, Wael Al Zoughbi, Kentaro Ohara, Joonghoon Auh, P. Waltman, Florencia P. Madorsky Rowdo, Enrique Podaza, Valerie Gallegos, John Nguyen, Raehash Shah, Manish Shah, Allyson J. Ocean, Douglas Scherr, Nasser K. Altorki, M. Frey, Ana M. Molina, Lisa Newman, Vivan Bea, E. Chapman-Davis, Marcus D. Goncalves, Ashish Saxena, Parul J. Shukla, Kevin Holcomb, Rachel Simmons, S. Tagawa, J. Zippin, Evelyn Cantillo, R. Chandwani, Melissa B. Davis, Kelly A. Garrett, P. Kasi, Jennifer Marti, David M. Nanus, J. Nauseef, Elizabeth Popa, Joseph Ruggiero, Momin T. Siddiqui, Alicia Alonso, Cora N. Sternberg, B. M. Faltas, O. Elemento, J. Mosquera, A. Sboner, M. L. Martin
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258df61f2a4c963aeb2c62a2f4585ccce0f62b1b" target='_blank'>
              Abstract 6464: Spatial mapping of the tumor microenvironment for combination therapy design
              </a>
            </td>
          <td>
            Arindam Bose, Susan Keezer, E. Alonzo, Sophie Struble, Jeremy Fisher, Matthew Norton, R. Heil-Chapdelaine, Natasha F. Diaz Granados, Vasundhara Agrawal
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Single-cell analysis has refined our understanding of cellular heterogeneity in glioma, yet RNA alternative splicing (AS)—a critical layer of transcriptome regulation—remains underexplored at single-cell resolution. Here, we present a pan-glioma single-cell AS analysis in both tumor and immune cells through integrating seven SMART-seq2 datasets of human gliomas. Our analysis reveals lineage-specific AS across glioma cellular states, with the most divergent AS landscapes between mesenchymal- and neuronal-like glioma cells, exemplified by AS in TCF12 and PTBP2. Comparison between core and peripheral glioma cells highlights AS-redox co-regulation of cytoskeleton organization. Further analysis of glioma-infiltrating immune cells reveals potential isoform-level regulation of protein glycosylation in regulatory T cells and a link between MS4A7 AS in macrophages and clinical response to anti-PD-1 therapy. This study emphasizes the role of AS in glioma cellular heterogeneity, highlighting the importance of an isoform-centric approach to better understand the complex biological processes driving tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad0ab2b92026794b33c50be07f43cb4158d402cd" target='_blank'>
              A Single-Cell Atlas of RNA Alternative Splicing in the Glioma-Immune Ecosystem
              </a>
            </td>
          <td>
            Xiao Song, D. Tiek, Minghui Lu, Xiaozhou Yu, Runxin Wu, Maya Walker, Qiu He, Derek Sisbarro, Bo Hu, Shi-Yuan Cheng
          </td>
          <td>2025-03-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9d10ba962b31f50d67da52456440cdb7f8eb0c" target='_blank'>
              Abstract 3880: Functional dissection of the highly plastic basal cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Kate Ryan, Jung Yun Kim, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G. Christensen, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3bfe2b136df9484a23fdf6a59f1b40397fdc9bc" target='_blank'>
              Abstract 87: Targeting POU5F1-expressing circulating tumor cells to prevent metastasis and recurrence in colorectal cancer
              </a>
            </td>
          <td>
            S. Fujino, Aya Ito, Rie Hayashi, M. Yasui, M. Ohue, N. Miyoshi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Distal cholangiocarcinoma (dCCA) is a highly lethal malignancy that accounts for approximately 40% of patients with primary cholangiocarcinoma. Remarkable cellular heterogeneity and perineural invasion (PNI) are two typical features of dCCA. Deciphering the complex interplay between neoplastic and neural cells is crucial for understanding the mechanisms propelling PNI-positive dCCA progression. Herein, we conduct single-cell RNA sequencing on 24,715 cells from two pairs of PNI-positive dCCA tumors and adjacent tissues, identifying eight unique cell types. Malignant cells exhibit significant inter- and intra-tumor heterogeneity. We delineate the compositional and functional phenotypes of five Schwann cell (SC) subsets in PNI-positive dCCA. Moreover, our analyses reveal two potential cell subtypes critical to forming PNI: NEAT1+ malignant cells characterized by hypoxic propensity and GFAP+ dedifferentiated SCs featuring hypermetabolism. Further bioinformatics uncover extensive cellular interactions between these two subpopulations. Functional experiments confirm that lactate in the hypoxic tumor microenvironment can induce GFAP-dedifferentiation in SCs, which promotes cancer cell invasion and progression through upregulating HMGB1. Taken together, our findings offer a thorough characterization of the transcriptional profile in PNI-positive dCCA and unveil potential therapeutic targets for dCCA PNI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c94bb9ca69e98a394049a5ef694607298253e3ff" target='_blank'>
              Single-cell analysis reveals that GFAP+ dedifferentiated Schwann cells promote tumor progress in PNI-positive distal cholangiocarcinoma via lactate/HMGB1 axis
              </a>
            </td>
          <td>
            Ziyang Zu, Chong Zhang, Jianxiang Shi, Kunlun Chen, Hongwei Tang, Kaizhao Hu, Enchi Liu, Chengyang Ji, Ruo Feng, Xiaojing Shi, Wenlong Zhai
          </td>
          <td>2025-03-27</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Polyploidy (whole-genome multiplication) is a common yet under-surveyed property of tissues across multicellular organisms. Polyploidy plays a critical role during tissue development, following acute stress, and during disease progression. Common methods to reveal polyploidy involve either destroying tissue architecture by cell isolation or by tedious identification of individual nuclei in intact tissue. Therefore, there is a critical need for rapid and high-throughput ploidy quantification using images of nuclei in intact tissues. Here, we present iSPy (Inferring Spatial Ploidy), a new unsupervised learning pipeline that is designed to create a spatial map of nuclear ploidy across a tissue of interest. We demonstrate the use of iSPy in Arabidopsis, Drosophila, and human tissue. iSPy can be adapted for a variety of tissue preparations, including whole mount and sectioned. This high-throughput pipeline will facilitate rapid and sensitive identification of nuclear ploidy in diverse biological contexts and organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57464323a77a94ca4250035fb30d39ccf24ecac9" target='_blank'>
              Spatial ploidy inference using quantitative imaging
              </a>
            </td>
          <td>
            Nicholas J. Russell, Paulo B. Belato, L. S. Oliver, Archan Chakraborty, Adrienne H. K. Roeder, Donald T. Fox, Pau Formosa-Jordan
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db705f60f289f122950733aa7588b5a6d8c6c985" target='_blank'>
              Abstract 5317: Multimodal spatial analysis of colon cancer tissue reveals emergence of an immunosuppressive tumor maintenance mechanism
              </a>
            </td>
          <td>
            Gaurav N Joshi, Nicholas Sciascia, Michael T Yang, B. Falkenstein, R. Yan, S. Quinn, Brenna Fearey, Junya Yoshioka, Arif Burak Tosun, S. Chennubhotla, F. Pullara, Fumiki Yanagawa
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f69de6f588f5dc88e965e9e468e394e888395b6" target='_blank'>
              Single-cell and spatial transcriptomics reveal a tumor-associated macrophage subpopulation that mediates prostate cancer progression and metastasis.
              </a>
            </td>
          <td>
            Shenglin Mei, Hanyu Zhang, Taghreed Hirz, N. Jeffries, Yanxin Xu, Ninib Baryawno, Shulin Wu, Chin-Lee Wu, Akash Patnaik, P. Saylor, David B. Sykes, Douglas M Dahl
          </td>
          <td>2025-03-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cbc83941350f959c8dff2b5403f5812073037ec" target='_blank'>
              Abstract 777: Revealing the transcriptional and proteomic spatial neighborhoods of early prostate cancer and the tumor microenvironment
              </a>
            </td>
          <td>
            A. Lee, M. Hopkins, L. Liao, Vida Talebian, Tamara Jamaspishvili, David M. Berman, Melanie Spears, Jane Bayani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer's notorious heterogeneity poses significant challenges, as each tumor comprises a unique ecosystem. While single-cell and spatial transcriptomics advancements have transformed our understanding of spatial diversity within tumors, the temporal dimension remains underexplored. Tumors are dynamic entities that continuously evolve and adapt, and relying solely on static snapshots obscures the intricate interplay between cancer cells and their microenvironment. Here, we advocate for integrating temporal dynamics into cancer research, emphasizing a fundamental shift from traditional endpoint experiments to data-driven, continuous approaches. This integration involves, for instance, the development of advanced live imaging techniques, innovative temporal omics methodologies, and novel computational tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b0f973c23f03f7770f1a751b49311e191b322a" target='_blank'>
              Time, the final frontier.
              </a>
            </td>
          <td>
            Gautier Follain, Michal Dibus, Omkar Joshi, Guillaume Jacquemet
          </td>
          <td>2025-03-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Different patterns of immune and stromal interactions within the tumor microenvironment significantly impact tumor behavior and response to treatment. Recent research has identified distinct immune subtypes—immune desert, excluded, and inflamed—each associated with specific mechanisms that impede anti-tumor immune responses. In the era of precision medicine, accurately stratifying patients into these subtypes is essential for optimizing cancer management and personalizing immunotherapy. The research team at National Cancer Institute in Lithuania addresses critical gaps in this field by focusing on several key areas. Our first focus is on developing reliable and accessible tools for determining patient immune subtypes. We are integrating transcriptomics and histology data to create a robust subtyping pipeline, which we are validating with real-world data and streamlining for clinical use. Another aspect of our work involves creating translatable preclinical models for cancer immunotherapy through utilizing 3D cell cultures and biocompatible scaffolds. We are developing druggable syngeneic murine tumor models that accurately reflect the three immune subtypes, enhancing the relevance of preclinical studies. Using both pharmaceutical and non-pharmaceutical approaches, we are tailoring immunomodulatory interventions to address the unique characteristics of each immune subtype. Joint efforts in these research areas could generate translatable insights, contributing to more precise cancer therapies and overcoming resistance mechanisms. 
  
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56c8e8d206eced9c976c9ce13db22dd5e4aadd38" target='_blank'>
              Deciphering immune tumor subtypes through microenvironment profiling
              </a>
            </td>
          <td>
            A. Mlynska, Eglė Žymantaitė, Austeja Butkute, N. Dobrovolskienė, J. Krasko, Olha Karaman, Karolina Suveizdė, Beatrice Gudaite, Margarita Žvirblė, Emilija Paberalė, Nijole Matuseviciene, V. Pašukonienė
          </td>
          <td>2025-03-25</td>
          <td>South East European Journal of Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f680cbd3c45701162ce05be4c984e82cf3cb95" target='_blank'>
              Abstract 1258: Deciphering evolutionary dynamics and transcriptomic adaptions under therapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Kaiser, Marcel Schmiel, Christian Müller, Maria Cartolano, M. Peifer, Roman K. Thomas, Julie George
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Advances in tissue labeling, imaging, and automated cell identification now enable the visualization of immune cell types in human tumors. However, a framework for analyzing spatial patterns within the tumor microenvironment (TME) is still lacking. To address this, we develop Spatiopath, a null-hypothesis framework that distinguishes statistically significant immune cell associations from random distributions. Using embedding functions to map cell contours and tumor regions, Spatiopath extends Ripley’s K function to analyze both cell-cell and cell-tumor interactions. We validate the method with synthetic simulations and apply it to multi-color images of lung tumor sections, revealing significant spatial patterns such as mast cells accumulating near T cells and the tumor epithelium. These patterns highlight differences in spatial organization, with mast cells clustering near the epithelium and T cells positioned farther away. Spatiopath enables a better understanding of immune responses and may help identify biomarkers for patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80949e1640c0e5682964239d082757c3241b697f" target='_blank'>
              Statistical analysis of spatial patterns in tumor microenvironment images
              </a>
            </td>
          <td>
            Mohamed M Benimam, V. Meas-Yedid, Suvadip Mukherjee, Astri Frafjord, Alexandre Corthay, T. Lagache, Jean-Christophe Olivo-Marin
          </td>
          <td>2025-03-31</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37bbf9a9279b9ea882191a6ab180a2f1195c66ee" target='_blank'>
              Abstract 1246: Driver and signaling cascade in therapy pressure driven cancer cell plasticity
              </a>
            </td>
          <td>
            Poorvi Subramanian, Sreenidhi Mohanvelu, Sivasubramani Narayanan, Afsana Parveen Jahir Hussain, Sabir Salim, Madhi Oli Ramamurthy, Loganayaki Periyasamy, N. Aravindan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/596ee5bf00c8a656038f1a6564aba0e8dbb21dea" target='_blank'>
              Longitudinal single-cell multiomic atlas of high-risk neuroblastoma reveals chemotherapy-induced tumor microenvironment rewiring.
              </a>
            </td>
          <td>
            Wenbao Yu, Rumeysa Biyik‐Sit, Yasin Uzun, Chia-Hui Chen, Anusha Thadi, Jonathan H Sussman, Minxing Pang, Chi-Yun Wu, Liron D. Grossmann, Peng Gao, David W Wu, Aliza Yousey, Mei Zhang, Christina S. Turn, Zhan Zhang, S. Bandyopadhyay, Jeffrey Huang, Tasleema Patel, Changya Chen, Daniel Martinez, Lea F Surrey, M. Hogarty, K. Bernt, Nancy R. Zhang, John M. Maris, Kai Tan
          </td>
          <td>2025-04-14</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a22b42357df2b30b92286c1a429534a02e045698" target='_blank'>
              Abstract 4596: Spatial transcriptomics reveals subtype heterogeneity within the tumor epithelial compartment of pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            J. Topham, Jenny Chu, Maya Kevorkova, Steve E. Kalloger, J. Karasinska, Andrew Metcalfe, Hassan Ali, Dongxia Gao, Christine Chow, Jonathan Loree, D. Renouf, David F. Schaeffer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Objective Active surveillance (AS) offers a viable alternative to surgical intervention for the management of indolent papillary thyroid carcinoma (PTC), helping to minimize the incidence of unnecessary treatment. However, the broader adoption of AS is hindered by the need for more reliable diagnostic markers. This study aimed to identify the differences between indolent and progressive PTC and find new targets for biomarker development and therapeutic strategies. Methods We used single-cell RNA sequencing (scRNA-seq) to analyze cellular differences in 10 early-stage PTC tumors. Findings were validated in an additional 25 tumors using cell co-culture, migration assays, immunofluorescence staining, flow cytometry, and analysis of data from The Cancer Genome Atlas (TCGA). Results Tumor-infiltrating B cells (TIL-B), particularly germinal center B cells (GC-B), were more abundant in indolent PTC. These cells suppressed thyroid cell proliferation in both indolent and progressive cases, though indolent PTC had a higher capacity to recruit peripheral B cells. In indolent cases, TIL-B cells showed increased proliferation and formed clusters within tertiary lymphoid structures (TLS). PTPRC-CD22 interactions were identified as potential drivers of TIL-B cell proliferation. Markers linked to GC-B cells, such as LMO2, were highlighted as potential diagnostic and prognostic indicators for indolent PTC. Conclusion This study provides insights into the cellular landscape of early-stage PTC, revealing distinct tumor and immune microenvironment features in indolent and progressive cases. These findings advance the understanding of indolent PTC biology and support the development of reliable diagnostic and prognostic biomarkers. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03341-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c059a59d61bc9cea405825a6d5d7499c3788b8e" target='_blank'>
              Single-cell transcriptomics analysis reveals that the tumor-infiltrating B cells determine the indolent fate of papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Chunmei Li, Pei Wang, Zhizhong Dong, Weihan Cao, Yan-Jun Su, Jian-ming Zhang, Shuyan Zhao, Zhiyuan Wang, Zi Lei, Li Shi, Ruochuan Cheng, Wen Liu
          </td>
          <td>2025-03-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Mammalian organs and tissues are composed of heterogeneously distributed cells, which interact with each other and the extracellular matrix surrounding them in a spatially defined way. Therefore, spatially resolved gene expression profiling is crucial for determining the function and phenotypes of these cells. While genome mutations and transcriptome alterations act as drivers of diseases, the proteins that they encode regulate essentially all biological functions and constitute the majority of biomarkers and drug targets for disease diagnostics and treatment. However, unlike transcriptomics, which has a recent explosion in high-throughput spatial technologies with deep coverage, spatial proteomics capable of reaching bulk tissue-level coverage is still rare in the field, due to the non-amplifiable nature of proteins and sensitivity limitation of mass spectrometry (MS). More importantly, due to the limited multiplexing capability of the current proteomics methods, whole-tissue slice mapping with high spatial resolution requires a formidable amount of MS matching time. To achieve spatially resolved, deeply covered proteome mapping for centimeter-sized samples, we developed a sparse sampling strategy for spatial proteomics (S4P) using computationally assisted image reconstruction methods, which is potentially capable of reducing the number of samples by tens to thousands of times depending on the spatial resolution. In this way, we generated the largest spatial proteome to date, mapping more than 9000 proteins in the mouse brain, and discovered potential new regional or cell type markers. Considering its advantage in sensitivity and throughput, we expect that the S4P strategy will be applicable to a wide range of tissues in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/635e78858dc942db5f55ba0ae972708b0c927e55" target='_blank'>
              In-depth and high-throughput spatial proteomics for whole-tissue slice profiling by deep learning-facilitated sparse sampling strategy
              </a>
            </td>
          <td>
            Ritian Qin, Jiacheng Ma, Fuchu He, Weijie Qin
          </td>
          <td>2025-03-11</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d66d16b13f8aa6c453618719a245bd84fb380319" target='_blank'>
              Abstract 759: Spatial transcriptomics identifies unique tumor and microenvironment pathomic programs that are associated with the lung premalignancy and adenocarcinoma continuum
              </a>
            </td>
          <td>
            Yibo Dai, F. Peng, Ansam Sinjab, Sujuan Yang, Minyue Chen, Warapen Treekitkarnmongkol, Lorena I. Gomez Bolanos, Tieling Zhou, Alejandra G Serrano, Jianlong Liao, Guangsheng Pei, Yunhe Liu, Yang Liu, Jiahui Jiang, Kyung Serk Cho, Yanshuo Chu, Kai Yu, Ruiping Wang, Jiping Feng, Zahraa Rahal, Guangchun Han, N. Jimbo, Takuo Hayashi, S. Kishikawa, K. Takamochi, Akshay V. Basi, Avrum E Spira, Stephen Dubinett, Tomokazu Itoh, Takashi Yao, Kenji Suzuki, Luisa M. Solis, S. Swisher, Mingyao Li, Junya Fujimoto, I. Wistuba, Jared K. Burks, Kadir C Akdemir, Hind Refai, K. Rezvani, Jeffrey Myers, H. Kadara, Linghua Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background The identification of molecular features characterizing metastatic disease is a critical area of oncology research, as metastatic foci often exhibit distinct biological behaviors compared to primary tumors. While the focus has largely been on the neoplastic cells themselves, the characterization of the associated stroma remains largely underexplored, with significant implications for understanding metastasis. Main body By employing spatially resolved transcriptomics, we analyzed the transcriptional features of primary breast adenocarcinoma and its associated metastatic foci, on a representative set of microregions. We identified a stromal metastatic (Met) signature, which was subsequently validated across transcriptomic reference human breast cancer (BC) datasets and in spatial transcriptomics of a murine model. Conclusion We discuss the potential of a stromal Met signature to pinpoint metastatic breast cancer, serving as a prognostic tool that can provide a foundation for the exploration of tumor-extrinsic molecular hallmarks of BC metastatic foci. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03353-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cabdeb7c0a8725d1b43bd88dd9faad675928ee91" target='_blank'>
              Extraction of a stromal metastatic gene signature in breast cancer via spatial profiling
              </a>
            </td>
          <td>
            G. Bertolazzi, Valeria Cancila, Davide Vacca, Beatrice Belmonte, D. Lecis, Parsa Sirati Moghaddam, A. Di Napoli, M. P. Colombo, G. Pruneri, Giannino Del Sal, Giorgio Scita, Matteo Fassan, A. Vecchione, Silvio Bicciato, Claudio Tripodo
          </td>
          <td>2025-03-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/339780e1e052f73bd7a155d38cbe11ed7059d53c" target='_blank'>
              Abstract 1253: Single-cell analysis reveals intricate subclonal heterogeneity and evolution in response to standard of care treatments in glioblastoma
              </a>
            </td>
          <td>
            Yueshan Zhao, Xiaoyu Lu, Verah Nyarige, Junfei Zhao, Gonzalo Lopez, Ann Forslund, Maria Ortiz-Estevez, Jorge Benitez-Hernandez, Wei Zhang, C. Fontanillo, Kai Wang, Ha Dang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99aa7af8e7e4a84d84918f1a59bfdf70d970aa43" target='_blank'>
              Abstract 5303: Temporal dynamics of transcriptional and spatial landscape in tumor microenvironment of esophageal squamous cell carcinoma during chemotherapy treatment
              </a>
            </td>
          <td>
            Hiroki Yamashita, Chiaki Mashima, Maiko Narahara, R. Kamata, Kenji Watanabe, R. Nakamura, Sumie Muramatsu, Ayumi Tezuka, Eri Otsuka, Chisa Wada, Yoko Ishimoto, D. Kotani, Takashi Kojima, Naoya Wada, Hiroyuki Sumi, Tomonori Yano, Gensuke Takayama, Akihiro Ohashi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0640112dbbdf90099adbfa03bf4452da14a83a22" target='_blank'>
              Abstract 2664: Spatial and single-cell transcriptomics reveal molecular heterogeneity in pulmonary pleomorphic carcinoma
              </a>
            </td>
          <td>
            Atsushi Matsuoka, K. Shien, Shuta Tomida, Hirofumi Inoue, Kazuya Hisamatsu, Ryota Fujiwara, Kousei Ishimura, Ryunosuke Fujii, Ryo Yoshichika, Tomoaki Higashihara, Naohiro Hayashi, Kazuhiro Okada, Fumiaki Mukohara, Mao Yoshikawa, K. Suzawa, Hidetaka Yamamoto, Shinichi Toyooka
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Programmed cell deaths (PCDs) are crucial for tumor progression. By analyzing 18 PCDs, we generated a robust multigene signature, Combined Cell Death Index (CCDI), comprising necroptosis and autophagy genes for non-small cell lung cancer (NSCLC). The CCDI accurately stratified patients by survival prognosis and predicted immunotherapy responses. We validated CCDI and prioritized CCDI genes using five single-cell RNA sequencing and two spatial transcriptomics datasets. CCDI positively correlates with tumor malignancy, invasiveness, and immunotherapy resistance. Four necroptosis genes (PTGES3, MYO6, CCT6A, and CTSH) may affect cancer cell evolution. In vitro, CTSH overexpression or PTGES3 knockdown inhibited NSCLC cell proliferation and migration while inducing necroptosis with necrosome formation. Moreover, we observed diminished CTSH, heightened PTGES3, and low necroptosis activity in 12 pairs of NSCLC tumors and normal tissues. CTSH overexpression or PTGES3 knockdown induced necroptosis and improved anti-PD1 therapy efficiency in syngeneic cancer mouse models. These findings indicate necroptosis genes as potential therapeutic targets in cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b557366271169d5f50c23b71e8320a685b6fc6f3" target='_blank'>
              Integrating bulk, single-cell, and spatial transcriptomics to identify and functionally validate novel targets to enhance immunotherapy in NSCLC
              </a>
            </td>
          <td>
            Kui Cao, Shenshui Wei, Tianjiao Ma, Xinxin Yang, Yuning Wang, Xiangrong He, Mengdi Lu, Yuwen Bai, Cuicui Qi, Luquan Zhang, Lijuan Li, Hongxue Meng, Jianqun Ma, Jinhong Zhu
          </td>
          <td>2025-04-16</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f10bf47822156d7b368733d148999f0f14fb4c81" target='_blank'>
              Spatial Transcriptomic Landscape of Brain Metastases from Triple-Negative Breast Cancer: Comparison of Primary Tumor and Brain Metastases Using Spatial Analysis.
              </a>
            </td>
          <td>
            Jihwan Yoo, Inho Park, Hyun Jung Kim, Hun Ho Park, Sora Lee, Jee Hung Kim, Yoon Jin Cha
          </td>
          <td>2025-04-15</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6d37955c551700a448a3cd5642f0bb308460197" target='_blank'>
              Abstract 7479: Single-cell OncoTreat: Novel therapeutic candidates by high resolution inference and validation of drug sensitivities in heterogeneous cholangiocarcinoma tumors
              </a>
            </td>
          <td>
            Aleksandar Z Obradovic, Lorenzo Tomassoni, Daoqi You, Charles G. Drake, F. D. Dela Cruz, Andrew L Kung, Andrea Califano
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0adc8a52f0d44cf8d5d6b4f5680ab473639625" target='_blank'>
              Model-based tumour subclonal deconvolution accounting for spatio-temporal sampling biases
              </a>
            </td>
          <td>
            Elena Rivaroli, Elena Buscaroli, Salvatore Milite, Alberto Casagrande, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/700ea58a4c1d36566c271fc146a1aaff828a43d8" target='_blank'>
              Abstract 3652: Celleporter: a foundation model of cell and tissue biology with application to patient stratification and target discovery
              </a>
            </td>
          <td>
            Yubin Xie, Eshed Margalit, Tyler Van Hensbergen, Dexter Antonio, Jake Schmidt, Yu (Phoebe) Guo, Jérémie Decalf, Maxime Dhainaut, Lucas Cavalcante, Hargita Kaplan, Rodney Collins, Francis Fernandez, Rob Schiemann, Eric Siefkas, Michela Meister, Joy Tea, Carl Ebeling, Anastasia Mavropoulos, Nicole Snell, Shafique Virani, Ronald Alfa, Lacey J Padrón, Jacob Rinaldi, Daniel Bear
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d0da13391f739e64143af742f8f9db15f31ce18" target='_blank'>
              Abstract 3904: Clonal analysis of classical subtypes of pancreatic cancer stem cells in ascites
              </a>
            </td>
          <td>
            Wa Xian, Frank McKeon, Paul Daniel, Saeed Khan, Caycel Desales, Ashley Hoffman, Jaffer Ajani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The formation of a blood vessel network is crucial for organ development and regeneration. Over the past three decades, the central molecular mechanisms governing blood vessel growth have been extensively studied. Recent evidence indicates that vascular endothelial cells—the specialized cells lining the inner surface of blood vessels—exhibit significant heterogeneity to meet the specific needs of different organs. This review focuses on the current understanding of endothelial cell heterogeneity, which includes both intra-organ and inter-organ heterogeneity. Intra-organ heterogeneity encompasses arterio-venous and tip-stalk endothelial cell specialization, while inter-organ heterogeneity refers to organ-specific transcriptomic profiles and functions. Advances in single-cell RNA sequencing (scRNA-seq) have enabled the identification of new endothelial subpopulations and the comparison of gene expression patterns across different subsets of endothelial cells. Integrating scRNA-seq with other high-throughput sequencing technologies promises to deepen our understanding of endothelial cell heterogeneity at the epigenetic level and in a spatially resolved context. To further explore human endothelial cell heterogeneity, vascular organoids offer powerful tools for studying gene function in three-dimensional culture systems and for investigating endothelial-tissue interactions using human cells. Developing organ-specific vascular organoids presents unique opportunities to unravel inter-organ endothelial cell heterogeneity and its implications for human disease. Emerging technologies, such as scRNA-seq and vascular organoids, are poised to transform our understanding of endothelial cell heterogeneity and pave the way for innovative therapeutic strategies to address human vascular diseases. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/174f590d400799b5343ccaff1b9f77896aa0b974" target='_blank'>
              Dissecting endothelial cell heterogeneity with new tools
              </a>
            </td>
          <td>
            Jing Zhong, Rong-rong Gao, Xin Zhang, Jia-Xin Yang, Yang Liu, Jinjin Ma, Qi Chen
          </td>
          <td>2025-03-23</td>
          <td>Cell Regeneration</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b881f06a481256ab1b5e06c0de554804f026d184" target='_blank'>
              Abstract 162: Mapping the complex interplay between vasculogenic mimicry and immune response in lung cancer using digital spatial profiling
              </a>
            </td>
          <td>
            , Chun-Hua Hung, Shu-Hsien Wang, Wei-Pang Chung, Yu-Min Yeh, Peng-Chan Lin, 
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs), known for facilitating the progression and metastasis of colorectal cancer (CRC), have become a promising therapeutic target. However, the significant heterogeneity of CAFs and their intricate crosstalk with tumor cells present substantial challenges in the development of precise and effective therapeutic strategies. Methods Single-cell RNA sequencing (scRNA-seq) technology was used to identify various cell subtypes. Spatial transcriptomics (ST) was employed to map the spatial niches and colocalization patterns of these cell subtypes. Cell-cell interactions among these subtypes were analysed via CellChat and NicheNet software. Tumor cell invasion, migration, and proliferation were assessed through wound healing assays, transwell assays, colony formation assays, and xenograft mouse models. Results We identified a significant spatial colocalization between CTHRC1+ CAFs and a distinct subtype of malignant epithelial cells, both residing within the EMT-active spatial niche. Our results demonstrate that CTHRC1+ CAFs, as a major source of WNT5A, promote epithelial-mesenchymal transition (EMT) and enhance tumor cell invasiveness by upregulating MSLN expression in adjacent malignant epithelial cells. This signaling axis contributes significantly to CRC progression and metastasis. Conclusions Targeting the CTHRC1+ CAF-WNT5A-MSLN signaling axis presents a promising therapeutic strategy for advanced CRC patients. Our study provides new insights into the role of CAFs in CRC progression and offers potential avenues for developing targeted therapies to disrupt this pathway. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06236-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efd1362638fd8682aac622f2d9c6d7d60b693242" target='_blank'>
              Single-cell and spatial transcriptome profiling reveal CTHRC1+ fibroblasts promote EMT through WNT5A signaling in colorectal cancer
              </a>
            </td>
          <td>
            Yunfei Lu, Yang Chen, Zhenling Wang, Hengyang Shen, Lei Xu, Changzhi Huang, Ying Tong, Yu Shao, Hongqiang Zhang, Zan Fu
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Pancreatic cancer has a low survival rate with limited treatment options. Perineural invasion (PNI), where cancer cells infiltrate the nerves in the pancreas, is associated with a poor prognosis and local recurrence. Pancreatic cancer is one of the most painful cancers, with PNI contributing to the pain experienced by patients. However, the mechanisms involved in the cancer invasion of nerves are not completely understood. Here, we used spatial transcriptomics to study the expression of both cancer and nerve cells in microscopic regions of PNI and compared it to regions without PNI evidence to better understand the mechanisms involved in PNI. We identified novel mechanisms that may impact cancer growth, PNI and pain experienced by patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1818d7eac1f6857a9dc4b88e4ee2f868f4f36db6" target='_blank'>
              Spatial Transcriptomics Reveals Novel Mechanisms Involved in Perineural Invasion in Pancreatic Ductal Adenocarcinomas
              </a>
            </td>
          <td>
            V. Lakis, Noni L Chan, Ruth J Lyons, Nicola Blackburn, Tam Hong Nguyen, Crystal Chang, Andrew Masel, Nicholas West, Glen M. Boyle, A. Patch, Anthony J. Gill, K. Nones
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Breast cancer is the most prevalent malignancy in women and exhibits significant heterogeneity. The tumor microenvironment (TME) plays a critical role in tumorigenesis, progression, and response to therapy. However, its impact on the prognosis and immunotherapy responses is incompletely understood. Using public databases, we conducted a comprehensive investigation of transcriptome and single-cell sequencing data. After performing immune infiltration analysis, we conducted consensus clustering, weighted gene co-expression network analysis (WGCNA), Cox regression, and least absolute shrinkage and selection operator (Lasso) regression to identify independent prognostic genes in breast cancer. Subsequently, we developed a prognostic model for patients with breast cancer. Tumor Immune Dysfunction and Exclusion (TIDE) values were used to assess patient’s responsiveness to breast cancer. Based on single-cell RNA-sequencing data, we identified various cell types through cluster analysis and investigated the expression of prognostic model genes in each cell type. The drug sensitivity of targeted therapeutic agents for breast cancer treatment was analyzed in different cell types. We identified 12 independent prognostic genes associated with breast cancer and used these genes to construct a prognostic model. The prognostic model accurately discriminated between patients classified as high- and low-risk, providing precise prognostic predictions for individual patients. Additionally, our model exhibited a robust capacity to predict the immunotherapeutic response in breast cancer patients. Our investigation revealed a notable association between the proportion of endothelial cells (ECs) and patient prognosis in breast cancer. A prognostic model for breast cancer was formulated that showed close associations between prognosis and response to immunotherapy. For patients predicted by our model to not respond effectively to immunotherapeutic agents, it may be considered to combine immunotherapeutic agents with targeted therapeutic agents identified through our drug sensitivity analysis, which could potentially enhance treatment efficacy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-97784-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d3bf29828fea11eaaea62401b9c917be2125fbb" target='_blank'>
              Exploring the tumor microenvironment of breast cancer to develop a prognostic model and predict immunotherapy responses
              </a>
            </td>
          <td>
            Ye Tian, Yong Yang, Lei He, Xiaocheng Yu, Hu Zhou, Juan Wang
          </td>
          <td>2025-04-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ed6afc1fbea4684dd9546a568ad33c0ccc1b6d" target='_blank'>
              Mapping molecular landscapes in triple-negative breast cancer: insights from spatial transcriptomics.
              </a>
            </td>
          <td>
            F. Al-Mansour, H. Almasoudi, Ali M Albarrati
          </td>
          <td>2025-03-22</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29faac1f752106215ff7dadafa3504675aecf0f8" target='_blank'>
              Abstract LB076: Spatial transcriptomics reveals differences in the tumor and immune microenvironment of high-grade serous ovarian cancers with differing responses to immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Jing Qian, Lynda D. Roman, Rania Bassiouni, Hanim I Ozkizilkaya, Kathy Kwock, Yuxin Jin, Javier Arias-Stella, Joseph W. Carlson, John D. Carpten
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/722de2170b79b4ce2e94d21e800de77de62534f9" target='_blank'>
              Abstract 772: Spatial analysis of the DLBCL tumor microenvironment via the novel SignalStarTM multiplex immunohistochemistry assay
              </a>
            </td>
          <td>
            J. Ziello, Shuling Zhang, Derek Papalegis, Lily Vu, Gabriella Spang, Jean Loebelenz, Arvin Ruiz, G. Inghirami, Sarah R Klein
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83f7f7a4b14997bd0c54213603c4303a128817de" target='_blank'>
              Abstract 769: Spatial proteomics and transcriptomics reveal an altered immune cell landscape in bladder cancer patients unresponsive to BCG treatment
              </a>
            </td>
          <td>
            T. Strandgaard, T. Andreasen, Tessa Jane Divita, Liina Salminen, Sean Houghton, Kasper Thorsen, I. Nordentoft, I. Okulate, Alexander Schmitz, Søren Riis Paludan, J. Sfakianos, Amir Horowitz, J. B. Jensen, L. Dyrskjøt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149ae335dfeb331ab7d81fc0ac4db62e86f66444" target='_blank'>
              Abstract 4536: Dissecting tumor-immune microenvironment in response and resistance to immune checkpoint blockade in metastatic melanoma
              </a>
            </td>
          <td>
            Xinyu Cui, Giuseppe Tarantino, Yiwen He, Priyanka Solanky, K. Pfaff, A. Thorner, Tyler J. Aprati, Boyang Zhang, Timothy R Blosser, Emily J. Robitschek, Jiajia Chen, Junko Tsuji, Elliot Boblitt, Allison Frangieh, H. Faulkner, Marta M. Holovatska, Aleigha Lawless, M. Manos, K. Helvie, T. Sharova, Dennie T. Frederick, J. L. Fahey, Diego Villamarin, Sachi Krishna, Chanell Mangum, Ajit J. Nirmal, Domenic Abbondanza, Cai McCann, B. Johnson, Alex K. Shalek, E. Van Allen, Xiaowei Zhuang, Ryan J. Sullivan, Barbara Engelhardt, Samouil L. Farhi, Scott J Rodig, F. S. Hodi, Genevieve M. Boland, David Liu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c4c0df9de0d71dd67f18dd27bf1bf7b9c3857af" target='_blank'>
              Abstract 3759: Single cell transcriptomics on a bioprinted 3D neurovascular unit glioblastoma model identifies pharmacological interventions that selectively target tumor cells
              </a>
            </td>
          <td>
            Yen-Ting Tung, M. J. Song, Marc Ferrer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d9f71c97b02d4a1d072a400c199093fb87572c0" target='_blank'>
              Abstract 3458: Antagonism between tumors and lymphocytes drives coevolutionary processes throughout metastasis
              </a>
            </td>
          <td>
            Cort B Breuer, Marcos Labrado, Nathan E. Reticker-Flynn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/931799fecbab71920d4fb53a4b17cf00bc015a8f" target='_blank'>
              Abstract 5073: Whole transcriptome spatial molecular imaging in matched pre and post treatment specimens to identify treatment-associated effects in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Beatrice W. Awasthi, Dennis Gong, N. Lester, Jung Woo Bae, Xunqin Yin, Ashley Heck, Elizabeth Stirling, Maximilian Walter, Megan Vandenberg, Margaret L Hoang, P. Divakar, Joseph M. Beechem, David T. Ting, W. L. Hwang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb216c1ef50675cb1a4983ed2285d35832a20b4e" target='_blank'>
              Abstract 113: Decoding developmental programs driving tumorigenesis and immunotherapy resistance in triple-negative breast cancer
              </a>
            </td>
          <td>
            Rachel Gleyzer, F. Khameneh, Zhen Qi, Wubing Zhang, Chloé B. Steen, M. Kang, Maya Maalouf, Melanis Ghadimi, Makenna Lindsey, S. Bobo, Thomas J. Lomis, F. Dirbas, Michael F. Clarke, Aaron M. Newman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Background Glioblastoma (GB), is an incurable brain tumor characterized by extreme malignancy and invasiveness, and the cellular mechanisms underlying such a severe phenotype are not completely understood. Although calcium (Ca2+) plays an important part in tumor proliferation and infiltration, it remains unclear whether Ca2+ signaling in GB cells is related to its location within the tumor and on the infiltrative potential of the cells. Methods In this study, we developed a stably transfected GL261 cell line that coexpresses a red fluorescent protein for actin cytoskeleton staining and the intracellular Ca2+ sensor, GCaMP6s. By means of intravital 2-photon imaging, we have characterized the morphological and functional properties of cells at different locations within the tumor. Results Our results showed that cells located at the tumor core are densely packed and rounded in shape, contrasting sharply with the polarized morphology observed in the peripheral cells. This anatomical heterogeneity corresponded to notable variations of the physiological phenotype: cells at the tumor core displayed low Ca2+ activity and very limited motility, while peripheral cells displayed intense Ca2+ activity and increased migration rates. Moreover, peripheral cells formed a cellular ensemble characterized by synchronized Ca2+ activity accompanied by a directionally biased collective motility. Conclusions These findings suggest that GB cells manifest activity patterns depending upon their spatial location within the tumor and that these correlate with their migration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f476d5b2005f0aec9da0be185757bb8f7ffc3130" target='_blank'>
              Heterogeneity of intracellular calcium signaling of glioblastoma cells depends on intratumoral location and migration state
              </a>
            </td>
          <td>
            V. Pillai, E. Pracucci, Francesco Trovato, Riccardo Parra, S. Landi, G. Ratto
          </td>
          <td>2025-03-05</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirical testing of all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated five compounds as potent PGCC inhibitors using cell lines and patient-derived models. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0482d6424427dcdf25109ef995747da0a47de141" target='_blank'>
              High-Throughput Empirical and Virtual Screening To Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yuan Zhang, Daniel D Brown, S. Oesterreich, Adrian V Lee, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2025-03-04</td>
          <td>Analytical Chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29fa9279efe44a23f2837544c024f634f88198ac" target='_blank'>
              Abstract 5666: Akos: A novel CAF-targeted therapy to reprogramming the tumor microenvironment to reverse metastatic traits in all solid tumors
              </a>
            </td>
          <td>
            J. Cerda-Infante, Benjamín Prieto, Valentina Cerda, M. Nuñez
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86c2e0fe302cf21b65e16778aa8ab9c33ba4b41f" target='_blank'>
              Abstract 3971: Single-cell transcriptomic insights into tumor and immune dynamics driving resistance to sitravatinib, nivolumab, and ipilimumab in advanced clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Kai Yu, Ying Yuan, Jianfeng Chen, Xinmiao Yan, Menuka Karki, Fei Duan, Rahul A Sheth, Priya Rao, K. Sircar, A. Shah, A. Zurita-Saavedra, Giannincola Genovesa, Min Li, Chih-Chen Yeh, M. Dang, Guangchun Han, Yanshuo Chu, Max Hallin, Peter Olson, Rui Yang, Daniela Slavin, H. Der-Torossian, Curtis Chin, N. Tannir, Linghua Wang, Jianjun Gao, P. Msaouel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most prevalent malignant brain tumor with poor prognosis. Although chromatin intratumoral heterogeneity is a characteristic feature of GBM, most current studies are conducted at a single tumor site. To investigate the GBM-specific 3D genome organization and its heterogeneity, we conducted Hi-C experiments in 21 GBM samples from nine patients, along with three normal brain samples. We identified genome subcompartmentalization and chromatin interactions specific to GBM, as well as extensive intertumoral and intratumoral heterogeneity at these levels. We identified copy number variants (CNVs) and structural variations (SVs) and demonstrated how they disrupted 3D genome structures. SVs could not only induce enhancer hijacking but also cause the loss of enhancers to the same gene, both of which contributed to gene dysregulation. Our findings provide insights into the GBM-specific 3D genome organization and the intratumoral heterogeneity of this organization and open avenues for understanding this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c40cfec55257420ce8df49630fbce7033db9c8" target='_blank'>
              Spatial 3D genome organization reveals intratumor heterogeneity in primary glioblastoma samples
              </a>
            </td>
          <td>
            Qixuan Wang, Juan Wang, Radhika Mathur, Mark Youngblood, Q. Jin, Ye Hou, L. Stasiak, Yu Luan, Hengqiang Zhao, Stephanie Hilz, Chibo Hong, Susan M Chang, Janine M Lupo, Joanna J Phillips, Joseph F Costello, Feng Yue
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background: Intratumor heterogeneity in plantar melanoma orchestrates transcriptional programs that contribute to resistance to target- and immuno-therapies. However, the evolution and spatial distribution of cellular subgroups, as well as their effects on immune environment and patient prognosis, remain unclear. Methods: We analyzed 218,021 cells from 20 plantar melanoma and 6 normal samples using single-cell RNA sequencing to reveal the evolutionary characteristics and communication patterns of tumor subgroups. Spatial transcriptomics and multiplex immunohistochemistry (mIHC) were used to map the spatial distribution of these subgroups, with mIHC scores further evaluating their correlation with patient prognosis. Single-cell multiomics analysis identified key transcription factors associated with chromatin accessibility. In addition, survival analysis was performed using bulk RNA sequencing data from 68 melanoma patients. Results: We identified a continuum of subgroups originating from stem cells via transitional and Schwann cell-like precursor states, ultimately reaching a Schwann cell-like state. This evolution trajectory was supported by integrative evidence, including assessments of stemness, transitional states, RNA velocity, and transcription factors. The histological distribution of these subgroups was validated by spatial transcriptomics and multiple IHC. Notably, Schwann cell-like subgroup, regulated by transcription factor HMGA2, was associated with immune cell dysregulation and a worse prognosis, including increased invasion and lymph node metastasis. Mechanically, inhibition of HMGA2 expression blocked the transition to Schwann-like melanoma fate. Conclusions: This study reveals the unique evolutionary trajectory of plantar melanoma, showing its differentiation towards a Schwann-like fate regulated by HMGA2, leading to a decline in pigment function, enhanced immune tolerance and an increased propensity for lymph node metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/906aa3f7f8f1b89c37180b73edff27a732dad812" target='_blank'>
              A distinct subgroup with Schwann-like differentiation presents immune resistance and poor prognosis in plantar melanoma
              </a>
            </td>
          <td>
            Jie Tian, Lu Zhang, Zhuang Le, Pingping Lin, Shenxi Zhang, Yicen Yan, Yu Yang, Guohong Zhang, Hang Li, Binbin Lai
          </td>
          <td>2025-03-03</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d4da2d95c6bade92c76a9deae19dad9fbd73c0" target='_blank'>
              Abstract 6362: Characterization of therapeutic effects in glioblastoma through integrated spatial single-cell multiome profiling of longitudinal samples: Unveiling clinically relevant subtypes of pathological changes
              </a>
            </td>
          <td>
            M. Vanmechelen, Pouya Nazari, J. Beckervordersandforth, D. Leunissen, B. Cole, B. Decraene, Y. De Visser, Gautam Shankar, M. Verduin, Daniele Pantano, C. B. González-Blas, Chiara Caprioli, Sien Bevers, Tom Moors, Jayesh Telang, Ivey Sebasstian, J. Messiaen, Y. Van Herck, Emma Geens, Danielle Eekers, A. Claeys, M. Derweduwe, A. Z. Hausen, Francesca M Bosisio, Frank Weyns, T. Daenekindt, Peter Van Eyken, Mieke Govers, Ferdinand Mennens, K. Hovinga, S. De Vleeschouwer, Paul Clement, M. Broen, Marc Vooijs, R. Sciot, A. Antoranz, Jon Pey, Ernst-Jan Speel, A. Hoeben, F. De Smet
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfe55329ddb31e27a6a2e90cca8f16e695b75bc6" target='_blank'>
              Abstract 5216: RPS19-C5AR1 axis in tumor-associated macrophages drives glioblastoma progression: Insights from single-cell transcriptomics
              </a>
            </td>
          <td>
            Chan-Chuan Liu, G. Anuraga, Cheng-Lin Wu, J. Chuang, Chih-Yang Wang, Pin-Yuan Chen, K. Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by pervasive copy number alterations (CNAs), significantly contributes to cancer progression and therapeutic resistance. CNAs drive intratumoral genetic heterogeneity, creating distinct subpopulations whose interactions shape tumor evolution through frequency-dependent selection. Here, we introduce, ECO-K (Ecological-Karyotypes), an inverse game theory framework that infers subpopulation interactions from longitudinal single-cell whole genome sequencing data. Applying this approach to serially-passaged, triple-negative breast cancer (TNBC) cell lines and patient-derived xenografts (PDXs), we systematically identified frequency-dependent selection dynamics governed by karyotypic diversity. Notably, in one PDX lineage, we found consistent evidence of karyotypically defined subpopulations acting as interaction hubs, associated specifically with chromosome 1 loss and chromosome 14p gain. Our framework provides testable predictions of intratumoral ecological dynamics, highlighting opportunities to strategically target key subpopulations to disrupt tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffff279b4407d3734303c6d5c0f841102d26d8b" target='_blank'>
              Inverse Game Theory Characterizes Frequency-Dependent Selection Driven by Karyotypic Diversity in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Joel S. Brown, Noemi Andor
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b642d8d5ad00a97352332e89bf3390c538a71e8e" target='_blank'>
              Abstract 1073: Integrated single cell atlas of human breast cancer enables highly-resolved analysis of tumor heterogeneity and metabolic comorbidities
              </a>
            </td>
          <td>
            , Christina S. Ennis, Lina Kroehling, Gerald V. Denis, Stefano Monti
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has significantly advanced our understanding of cellular heterogeneity and the complex interplay within the tumor microenvironment (TME) of colorectal cancer (CRC). However, translating these molecular insights into clinically actionable prognostic biomarkers and therapeutic strategies remains a considerable challenge. In this study, we conducted a comprehensive scRNA-seq analysis of 306 CRC samples comprising 448,255 cells to characterize the TME in depth. By constructing intercellular communication networks based on connection counts and communication probabilities, we identified fibroblasts as central regulatory hubs within the TME. Using Wilcoxon rank-sum tests and univariate survival analyses, we initially identified 23 prognostic fibroblast markers. These were refined to a seven-gene fibroblast-related prognostic signature via an integrated machine learning approach. The signature exhibited robust predictive performance in the The Cancer Genome Atlas - Colon Adenocarcinoma (TCGA-COAD) training cohort (n=351; C-index=0.65) and was successfully validated in the GSE17536 dataset (n=177; C-index=0.63). Functional enrichment analyses revealed that this signature is involved in immune regulation and multiple tumor-associated cellular pathways. Notably, high-risk patients displayed increased macrophage and NK cell infiltration, impaired immune function, and elevated immune rejection scores, while low-risk patients demonstrated heightened sensitivity to camptothecin and irinotecan. Together, our findings underscore the prognostic value of fibroblast-derived signatures in CRC and support their potential utility in risk stratification and the development of personalized therapeutic strategies, contributing to the advancement of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/879f972fe9756d179fa46b174a13c76374f8a7f0" target='_blank'>
              Machine learning integration of single-cell and bulk transcriptomics identifies fibroblast-driven prognostic markers in colorectal cancer.
              </a>
            </td>
          <td>
            Ning Zhang, Ruiyan Liu, Siya Wu, Chenxi Feng, Boxiang Wang, Qiaoqiao Zheng, Linru Jie, Ruihua Kang, Xiaoli Guo, Xiaoyang Wang, Shaokai Zhang, Jiangong Zhang
          </td>
          <td>2025-04-22</td>
          <td>Biomolecules & biomedicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4955ba693d4e35ecf04dfaea41f95d050a589d61" target='_blank'>
              Abstract A029: Highly-resolved integrated single cell atlas of human breast cancer enables hypothesis-driven analysis of tumor heterogeneity
              </a>
            </td>
          <td>
            Andrew Chen, Lina Kroehling, Christina S. Ennis, Gerald V. Denis, Stefano Monti
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eed8fdb3c30cedec701229d2292e4e2a7ab7ba3" target='_blank'>
              Abstract 6356: Multiscale high-content imaging to identify molecular and morphological signatures in drug-resistant cancer cells
              </a>
            </td>
          <td>
            Yaxin Yang, Jianquan Xu, Chaojie Zhang, Hongqiang Ma, Sarah J. Hainer, Yang Liu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c58d3deadeb0eab4e193b1995b6dfa8ecd4238f" target='_blank'>
              Abstract 5014: Integrative molecular and spatial analysis to reveal tumor intrinsic and extrinsic drivers of immunotherapy response in mucosal melanoma
              </a>
            </td>
          <td>
            Yingxiao Shi, Tuulia Vallius, Aikaterini Dedeilia, R. Pelletier, Jia-Ren Lin, M. Leon, Christine Lian, Shishir Pant, Giuseppe Tarantino, Annette Wang, Jiajia Chen, Elizabeth Buchbinder, P. Sorger, David Liu, Genevieve M Boland
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8492f82ea4a9bedb9da34190b6e586980dd59c" target='_blank'>
              Abstract 5313: Spatial transcriptomics reveals clone-specific features of primary and metastatic tumors in triple-negative breast cancer
              </a>
            </td>
          <td>
            Yu-Ting Kang, Tzu-Hung Hsiao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Due to their pivotal roles in tumor progression and therapy resistance, cancer-associated fibroblasts (CAF) are considered key therapeutic targets with loss of stromal androgen receptor (AR) a poorly understood hallmark of aggressive prostate cancer (PCa). A paucity of pre-clinical models however has hampered functional studies of CAF heterogeneity. We demonstrate that our newly-generated CAF biobank contains three FAP+-fibroblast subtypes, each with unique molecular and functional traits. Cultures with an early-activated phenotype expressed the highest levels of AR and exhibited AR-dependent growth. Consistently, stromal cells expressing early-activation markers co-expressed nuclear AR in clinical specimens and were enriched in pre-neoplastic lesions/low-grade PCa. Conversely, myofibroblastic CAF (myCAF), which expressed low AR levels in vitro and in vivo and were proliferatively-insensitive to AR signaling modulation, constituted the predominant CAF subpopulation in stromogenic high-grade PCa and castration-resistant LACP9 patient-derived xenografts. Exacerbation of the myCAF state upon castration of LAPC9-bearing hosts underscored these findings. Mechanistically, AR loss in myCAF was driven by an NFκB-TGFβ1-YAP1 axis, whose combined targeting synergistically repressed myofibroblastic hallmarks and impaired autophagic flux, effects that were potentiated by enzalutamide resulting in myCAF cell death. Collectively, these findings provide a mechanistic rationale for adjuvant targeting of the YAP1-TGFβ signaling axis to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49d403b398b909e0958e9ca61450b155a5c7afec" target='_blank'>
              Unraveling the YAP1-TGFβ1 axis: a key driver of androgen receptor loss in prostate cancer-associated fibroblasts
              </a>
            </td>
          <td>
            Elena Brunner, E. Damisch, Melanie Emma Groninger, Lukas Nommensen, Lucy Neumann, G. Fotakis, Z. Trajanoski, Georg Schäfer, Christian Ploner, S. Karkampouna, F. Bonollo, Marianna Kruithof-de Julio, Natalie Sampson
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Purpose This study aimed to generate the spatial atlas of the human limbus using spatial transcriptomic technology and reveal the deep interaction among the niche microenvironment. Methods The spatial transcriptomic atlas of human limbus was performed using 10× Genomics Space Ranger software platform. Single-cell RNA sequencing data of human limbal epithelial stem cells (LESCs) were downloaded for integrating analysis. Results We profiled more than 400 spots within each sample and spatially located major cell types within the limbus area. LESCs were localized mainly in the basement membrane, and limbal niche cells were situated predominantly within the stromal area. Next, the limbus was divided into four regions based on histological structure, and the differential expressed genes among the four regions were analyzed. Notably, GPHB5 was highly expressed in the epithelium of the middle region and co-staining with deltaNp63 suggested it might be a novel potential biomarker of LESCs. Subsequently, limbal mesenchymal stem cells were found to exhibit the greatest amounts of ligands associated with LESCs. The widespread activity of COL6A2/CD44 signaling among limbal mesenchymal stem cells, melanocytes, immune cells, and LESCs indicate its essential role in mediating bidirectional communication via the collagen pathway. Conclusions This research mapped the spatial positioning of key cells within the limbal niche and detailed interactions between major cell types. These findings provide a foundation for further LESC research and enhance our understanding of corneal biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8512a904bd68ff957e1d665eeda8f3a0e6fd27e1" target='_blank'>
              The Spatial Transcriptomic Atlas of Human Limbus and Vital Niche Microenvironment Regulating the Fate of Limbal Epithelial Stem Cells
              </a>
            </td>
          <td>
            Shiding Li, Hao Sun, Fei Fang, Siyi Zhang, Junzhao Chen, Chunyi Shao, Yao Fu, Liangbo Chen
          </td>
          <td>2025-03-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05724e131a84cae0882ab8214c7527b7f572cfbc" target='_blank'>
              Abstract SY22-02: Cell cycle plasticity drives tumor progression and drug resistance
              </a>
            </td>
          <td>
            Yian Yang, Thu Pham, Janet McLaughlin, S. Oesterreich, Adrian Lee, Katarzyna Kedziora, Wayne Stallaert
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0fc7e6f9b6372a2eebc4b0d3580ed466ef20424" target='_blank'>
              A Tc1- and Th1-T-lymphocyte-rich tumor microenvironment is a hallmark of MSI colorectal cancer.
              </a>
            </td>
          <td>
            Zhihao Huang, T. Mandelkow, N. F. Debatin, M. Lurati, Julia Ebner, J. Raedler, E. Bady, Jan H. Müller, R. Simon, E. Vettorazzi, A. Menz, Katharina Möller, N. Gorbokon, G. Sauter, M. Lennartz, A. Luebke, D. Höflmayer, T. Krech, P. Lebok, C. Fraune, A. Hinsch, Frank Jacobsen, A. Marx, S. Steurer, S. Minner, D. Dum, S. Weidemann, C. Bernreuther, T. Clauditz, E. Burandt, N. Blessin
          </td>
          <td>2025-04-03</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fd0e67d45bf4c5f69a873c6cb3c79e9be60dadd" target='_blank'>
              Abstract 7480: Spatial Groups provide a pan-tumor spatial transcriptomic biomarker of immunotherapy response
              </a>
            </td>
          <td>
            Vivek Behera, Hannah Giba, Ue-yu Pen, A. Di Lello, Benjamin A. Doran, Alessandra Esposito, A. Pezeshk, Christine Bestvina, Justin Kline, M. Garassino, Arjun S. Raman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b71083b48d8d0e95bc226f7370321af11b26e25b" target='_blank'>
              Spatial and single-cell transcriptomic analysis reveals fibroblasts dependent immune environment in colorectal cancer.
              </a>
            </td>
          <td>
            Hang Jia, Xiang Liu, Guimin Wang, Yue Yu, Ning Wang, Tianshuai Zhang, Liqiang Hao, Wei Zhang, Guanyu Yu
          </td>
          <td>2025-03-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Metastasis is a leading cause of cancer-related deaths, with the liver being the most frequent site of metastasis in colorectal cancer. Previous studies have predominantly focused on the influence of the primary tumor itself on metastasis, with relatively limited research examining the changes within target organs. Methods Using an orthotopic mouse model of colorectal cancer, single-cell sequencing was employed to profile the transcriptomic landscape of pre-metastatic and metastatic livers. The analysis focused on identifying cellular and molecular changes within the hepatic microenvironment, with particular emphasis on inflammatory pathways and immune cell populations. Results A neutrophil subpopulation with high Prok2 expression was identified, showing elevated levels in the pre-metastatic and metastatic liver. Increased infiltration of Prok2⁺ neutrophils correlated with poor prognosis in liver metastatic colorectal cancer patients. In the liver metastatic niche (MN), these neutrophils showed high App and Cd274 (PD-L1) expression, suppressing macrophage phagocytosis and promoting T-cell exhaustion. Conclusion A Prok2⁺ neutrophil subpopulation infiltrated both pre-metastatic and macro-metastatic liver environments, potentially driving immunosuppression through macrophage inhibition and T-cell exhaustion. Targeting Prok2⁺ neutrophils could represent a novel therapeutic strategy for preventing liver metastasis in colorectal cancer patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06351-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ae7b5dd668485b7f400a3f84e79d9d667bb33c" target='_blank'>
              Single-cell transcriptomic analysis reveals dynamic changes in the liver microenvironment during colorectal cancer metastatic progression
              </a>
            </td>
          <td>
            Yue Jiang, Guojie Long, Xiaoming Huang, Wenyu Wang, B. Cheng, Weidong Pan
          </td>
          <td>2025-03-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e0df805bc31950b58610c00bf890d8057b9c31" target='_blank'>
              Abstract 165: Spatially resolved mechanostress effects on tumor microenvironments: Tumor cells and fibroblasts in focus
              </a>
            </td>
          <td>
            Wooseok Kim, HyunSeok Kim, Minki Kim, Zhangxian Lan, , Mi Ryung Roh, Joon Kim, Jong-Eun Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The spatial organization of cells in tissues underlies biological function, and recent advances in spatial profiling technologies have enhanced our ability to analyze such arrangements to study biological processes and disease progression. We propose MESA (multiomics and ecological spatial analysis), a framework drawing inspiration from ecological concepts to delineate functional and spatial shifts across tissue states. MESA introduces metrics to systematically quantify spatial diversity and identify hot spots, linking spatial patterns to phenotypic outcomes, including disease progression. Furthermore, MESA integrates spatial and single-cell multiomics data to facilitate an in-depth, molecular understanding of cellular neighborhoods and their spatial interactions within tissue microenvironments. Applying MESA to diverse datasets demonstrates additional insights it brings over prior methods, including newly identified spatial structures and key cell populations linked to disease states. Available as a Python package, MESA offers a versatile framework for quantitative decoding of tissue architectures in spatial omics across health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162d6ff6e3cedba6aedd63bc146790643b33531d" target='_blank'>
              Quantitative characterization of tissue states using multiomics and ecological spatial analysis
              </a>
            </td>
          <td>
            Daisy Yi Ding, Zeyu Tang, Bokai Zhu, Hongyu Ren, Alex K. Shalek, Robert Tibshirani, GP Nolan
          </td>
          <td>2025-04-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7119dba646488771d805798083884f45a999e" target='_blank'>
              Abstract 1243: The pro-metastatic phenotype of chemoresistant breast cancer is stable and transcriptionally defined
              </a>
            </td>
          <td>
            Alberto Dalmasso, Andrea Cossa, Ilaria Servidio, Guido Campani, Elisa Bugani, F. A. Tucci, Lucrezia Stabellini, S. Pece, P. Pelicci
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8ac2ba6fe27ed44bff66e7ee62b13701829278a" target='_blank'>
              Abstract 6437: Mutational patterns in normal tissue hold the key to understanding the cellular origins of cancer
              </a>
            </td>
          <td>
            Kirsten Kübler, R. Karlić, Mendy Miller, C. Stewart, William D. Foulkes, P. Polak, Peter F. Arndt, G. Getz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/653b80a32b949cb3e328b878333ade7e654abf52" target='_blank'>
              Abstract 6283: Performance evaluation of tumor microenvironment deconvolution and gene prediction methods in lung cancer
              </a>
            </td>
          <td>
            Fanying Tang, Dean Lee, Lexiang Ji, Jincheng Wu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2c3bdf68670d6c9bca6109a3e491e1600abb90d" target='_blank'>
              Abstract 2010: A spatial profiling approach to evaluating the prognostic impact of heterogeneity in the triple-negative breast cancer immune microenvironment
              </a>
            </td>
          <td>
            Prerana Sensharma, Huidan Zuo, Melanie Dawe, M. Hopkins, Z. Baskurt, Osvaldo Espin-Garcia, P. Bedard, Melanie Spears, Susan J. Done
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Accumulating evidence has suggested that cancer progression and therapeutic response depend on both tumor epithelium (EPI) and tumor microenvironment (TME). However, the dependency of clinical outcomes on the tumor EPI vs. the TME has neither been clearly defined nor quantified. Methods We classified 2373 colorectal cancer (CRC) tumors into the consensus molecular subtypes (CMS1-4) and generated the 10-gene TMES and the 10-gene EPIS signatures as the serendipitous derivatives of the most (positively vs. negatively) correlated genes of a highly-prognostic, ~ 500-gene signature we previously identified. Distinct TME vs. EPI cellular features of the signature genes were identified by CIBERSORT deconvolution and validated by scRNASEQ in an independent public dataset. Results The TMES signature was strongly associated with the immune/stromal TME-rich CMS1/CMS4 subtypes that portended worse survival, whereas the EPIS signature was predominantly related to the TME-poor, epithelial CMS2/CMS3 classes that portended better survival. Multivariable Cox regression analysis against 29 TME-related signatures revealed that the TMES signature was the most strikingly impacted by the “Cancer-associated fibroblasts” signature (HR: 10.87 vs. 0.13, both P < 0.0001). Moreover, the TMES score was strongly correlated with EMT, SRC activation and MEK inhibitor resistance in 2373 CRC tumors (Spearman r = 0.727, 0.802, 0.824, respectively), which was validated in two independent CRC datasets (n = 626 and n = 566). By contrast, the EPIS score was the dominant force in associating with longer progression free survival in cetuximab-treated metastatic CRC patients derived from two independent clinical trials (Logrank trend P = 0.0005/n = 80; P = 0.0013/n = 44). This finding was further validated in a large real-world clinico-genomics dataset with EGFR inhibitor therapy, which demonstrated that higher EPIS scores were associated with increased overall survival (EGFRi, Logrank trend P < 0.0001/n = 2343) and time on treatment (cetuximab, P = 0.003/n = 953; panitumumab, P < 0.0001/n = 1307). Conclusions Here we identified a pair of new, distinct 10-gene signatures (the EPIS vs. the TMES) capable of distinguishing the cellular contribution of the tumor EPI vs. the TME in determining CRC prognosis and therapeutic outcomes. With targeted approaches emerging to address both tumor epithelial cells and the TME, the EPIS vs. TMES signature scores may have a novel biomarker role to permit optimization of CRC therapy by identifying sensitive vs. resistant subpopulations. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-13829-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7948bcd29ed4c332a5d0dc90dea3c9f39d631a84" target='_blank'>
              Identifying distinct prognostic and predictive contributions of tumor epithelium versus tumor microenvironment in colorectal cancer
              </a>
            </td>
          <td>
            Mingli Yang, M. Nebozhyn, Michael J. Schell, N. Gandhi, Lance Pflieger, Andrey Loboda, W. J. Pledger, Ramani Soundararajan, Michelle M Maurin, Heiman Wang, Jetsen Rodriguez Silva, Ashley J Alden, Domenico Coppola, Andrew Elliott, George W Sledge, M. Khushman, Emil Lou, Sanjay Goel, Timothy J Yeatman
          </td>
          <td>2025-03-12</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The tumor microenvironment (TME) encompasses various cell types, blood and lymphatic vessels, and noncellular constituents like extracellular matrix (ECM) and cytokines. These intricate interactions between cellular and noncellular components contribute to the development of a malignant TME, such as immunosuppressive, desmoplastic, angiogenic conditions, and the formation of a niche for cancer stem cells, but there is limited understanding of the specific subtypes of stromal cells involved in this process. Here, we utilized p16-CreERT2-tdTomato mouse models to investigate the signaling networks established by senescent cancer stromal cells, contributing to the development of a malignant TME. In pancreatic ductal adenocarcinoma (PDAC) allograft models, these senescent cells were found to promote cancer fibrosis, enhance angiogenesis, and suppress cancer immune surveillance. Notably, the selective elimination of senescent cancer stromal cells improves the malignant TME, subsequently reducing tumor progression in PDAC. This highlights the antitumor efficacy of senolytic treatment alone and its synergistic effect when combined with conventional chemotherapy. Taken together, our findings suggest that the signaling crosstalk among senescent cancer stromal cells plays a key role in the progression of PDAC and may be a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06c45a25e4aa78209216dcd2827d3d116f51fb9d" target='_blank'>
              Signaling networks in cancer stromal senescent cells establish malignant microenvironment.
              </a>
            </td>
          <td>
            Yue Zhang, Teh-Wei Wang, Maho Tamatani, Xinyi Zeng, Lindo Nakamura, Satotaka Omori, Kiyoshi Yamaguchi, Seira Hatakeyama, Eigo Shimizu, Satoshi Yamazaki, Yoichi Furukawa, S. Imoto, Yoshikazu Johmura, M. Nakanishi
          </td>
          <td>2025-04-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="ABSTRACT Background To date, many types of immune cells have been identified, but their precise role in cancer immunity remains unclear. Understanding the immune responses involved in cancer and the cancer microenvironment is becoming increasingly important for elucidating disease mechanisms. In recent years, the application of intravital imaging in cancer research has provided new insights into the mechanisms of cancer‐specific immune events, including innate and adaptive immunity. Results In this review, we focus on the emerging role of intravital imaging in cancer research and describe how cancer and immune cells can be observed using intravital imaging in vivo. We also discuss new insights gained by this state‐of‐the‐art technique. Conclusions Intravital imaging is a relatively new field of research that offers significant advantages, including the ability to directly capture cell–cell interactions, pathophysiology, and immune cell dynamics in the cancer microenvironment in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d71051effa98fd53d4b75cd152b365d2bf34ab0" target='_blank'>
              Intravital Imaging of Immune Responses in the Cancer Microenvironment
              </a>
            </td>
          <td>
            Hiroki Hirao, Masaki Honda, Masahiro Tomita, Lian-qian Li, Ahmad Adawy, Weijie Xue, T. Hibi
          </td>
          <td>2025-04-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Intratumour heterogeneity and phenotypic plasticity drive tumour progression and therapy resistance1,2. Oncogene dosage variation contributes to cell-state transitions and phenotypic heterogeneity3, thereby providing a substrate for somatic evolution. Nonetheless, the genetic mechanisms underlying phenotypic heterogeneity are still poorly understood. Here we show that extrachromosomal DNA (ecDNA) is a major source of high-level focal amplification in key oncogenes and a major contributor of MYC heterogeneity in pancreatic ductal adenocarcinoma (PDAC). We demonstrate that ecDNAs drive varying levels of MYC dosage, depending on their regulatory landscape, enabling cancer cells to rapidly and reversibly adapt to microenvironmental changes. In the absence of selective pressure, a high ecDNA copy number imposes a substantial fitness cost on PDAC cells. We also show that MYC dosage affects cell morphology and dependence of cancer cells on stromal niche factors. Our work provides a detailed analysis of ecDNAs in PDAC and describes a new genetic mechanism driving MYC heterogeneity in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>1</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb18bd3e4cb1302dcaf3faa0469e23f342caf15" target='_blank'>
              Abstract 5315: Mapping tumor infiltrating lymphocytes in whole prostatectomy specimens to visualize and quantify the immune microenvironment of prostate cancer
              </a>
            </td>
          <td>
            Yuwei Zhang, Rajarsi Gupta, J. Saltz, T. Kurç, Jakub R. Kaczmarzyk, Erich Bremer, Veronica Wallaengen, Amanda Galvez, Abigail Bartley, Leonard Salcedo, Alan Pollack, S. Punnen, Oleksandr N. Kryvenko, R. Stoyanova, Sandra M Gaston
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11004abebcf7dd56b12c4dc898da1ed0acccbe13" target='_blank'>
              Immunocytes in the tumor microenvironment: recent updates and interconnections
              </a>
            </td>
          <td>
            Jiyao Yu, Li Fu, Rui Wu, Linyi Che, Guodong Liu, Qinwen Ran, Zhiwei Xia, Xisong Liang, Guanjian Zhao
          </td>
          <td>2025-04-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417447ea57e32086f8cc2713f35e946d6685f5c1" target='_blank'>
              Abstract 1250: Genetically-defined organoid models reveal early mechanisms of squamous neoplastic progression and therapeutic vulnerabilities in squamous cell carcinoma
              </a>
            </td>
          <td>
            Hua Zhao, Y. Park, Yueyuan Zheng, Qiong Mao, Luda Lin, Casey Collet, Stephanie Wong, Boyan Hu, Uttam K Sinha, A. Soragni, Dechen Lin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a8e2bdd23ee299c051bf62b1634c5ff6fb1bbaf" target='_blank'>
              SOX2 commands LIM homeobox transcription factors in choroid plexus development and tumorigenesis.
              </a>
            </td>
          <td>
            Lukas J Faltings, Fengjuan Lin, Mariam Zahran, Heena Jalili, Anjali Siluveru, Mahek Chaudry, Leah M Wachsmuth, Navjot Guru, Melanie Schoof, Ping Cao, Yuan-hua Huang, Noreen Mian, Sohyun Moon, Asmaa Zahran, Kristen Green, Siddhi Modi, Maheen Umer, James Q Virga, Ying-Tao Zhao, U. Schüller, Qun Li, Haotian Zhao
          </td>
          <td>2025-03-25</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tumor heterogeneity is predicted to confer inferior clinical outcomes with precision-based strategies, however, modeling heterogeneity in a manner that still represents the tumor of origin remains a formidable challenge. Sequencing technologies are limited in their ability to identify rare subclonal populations and predict response to treatments for patients. Patient-derived organotypic cultures have significantly improved the modeling of cancer biology by faithfully representing the molecular features of primary malignant tissues. Patient-derived cancer organoid (PCO) cultures contain subclonal populations with the potential to recapitulate heterogeneity, although treatment response assessments commonly ignore diversity in the molecular profile or treatment response. Here, we demonstrate the advantage of evaluating individual PCO heterogeneity to enhance the sensitivity of these assays for predicting clinical response. Additionally, organoid subcultures identify subclonal populations with altered treatment response. Finally, dose escalation studies of PCOs to targeted anti-EGFR therapy are utilized which reveal divergent pathway expression when compared to pretreatment cultures. Overall, these studies demonstrate the importance of population-based organoid response assessments, the use of PCOs to identify molecular heterogeneity not observed with bulk tumor sequencing, and PCO heterogeneity for understanding therapeutic resistance mechanisms. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-96204-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d73b21d8a323278ef745932d923e8c41351e22f0" target='_blank'>
              Subclonal response heterogeneity to define cancer organoid therapeutic sensitivity
              </a>
            </td>
          <td>
            J.D. Kratz, Shujah Rehman, Katherine A. Johnson, A. Gillette, Aishwarya Sunil, Peter F Favreau, Cheri A. Pasch, Devon D Miller, Lucas Zarling, Austin H. Yeung, L. Clipson, Samantha J. Anderson, Alyssa K Steimle, Carley M. Sprackling, K. Lemmon, Daniel E Abbott, M. Burkard, Michael F. Bassetti, J. Eickhoff, Eugene F Foley, Charles P Heise, Randall J Kimple, Elise H Lawson, N. LoConte, S. Lubner, Daniel L Mulkerin, Kristina A Matkowskyj, Cristina B. Sanger, Nataliya V. Uboha, Sean J Mcilwain, Irene M Ong, Evie H Carchman, Melissa C Skala, Dustin A. Deming
          </td>
          <td>2025-04-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0577e3703eceb9dd16f950f2b2b372372b328f4f" target='_blank'>
              Abstract 5304: Profiling the tumor- immune landscape with a same-section multiomics assay
              </a>
            </td>
          <td>
            A. Dikshit, Cansaran Saygili Demir, Rose Delvillar, Emerald Doolittle, Alice Comberlato, Arec Manoukian, M. Procopio, Pino Bordignon, S. Brajkovic, Maithreyan Srinivasan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b565f8797f4fa651d27cbd6107a1a5335c33485f" target='_blank'>
              Abstract 2852: Mapping single cell spatial 3D genome landscape of localized and metastatic breast cancer sub-types
              </a>
            </td>
          <td>
            Shyamtanu Chattoraj, Huy Nguyen, Jude Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb874fc1b76423af08fc061b2412437123e6829d" target='_blank'>
              Abstract 6258: Inter- and intra-tumoral heterogeneity of TNBC determined by spatial gene transcriptomics
              </a>
            </td>
          <td>
            Fengyuan Huang, Deepa Bedi, Clayton C. Yates
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive tumor with high metastatic potential which leads to decreased survival rate and resistance to chemotherapy and immunotherapy. Nearly 90% of pancreatic cancer comprises pancreatic ductal adenocarcinoma (PDAC). About 80% of diagnoses takes place at the advanced metastatic stage when it is unresectable, which renders chemotherapy regimens ineffective. There is also a dearth of specific biomarkers for early-stage detection. Advances in next generation sequencing and single cell profiling have identified molecular alterations and signatures that play a role in PDAC progression and subtype plasticity. Most chemotherapy regimens have shown only modest survival benefits, and therefore, translational approaches for immunotherapies and combination therapies are urgently required. In this review, we have examined the immunosuppressive and dense stromal network of tumor immune microenvironment with various metabolic and transcriptional changes that underlie the pro-tumorigenic properties in PDAC in terms of phenotypic heterogeneity, plasticity and subtype co-existence. Moreover, the stromal heterogeneity as well as genetic and epigenetic changes that impact PDAC development is discussed. We also review the PDAC interaction with sequestered cellular and humoral components present in the tumor immune microenvironment that modify the outcome of chemotherapy and radiation therapy. Finally, we discuss different therapeutic interventions targeting the tumor immune microenvironment aimed at better prognosis and improved survival in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e7cd5ce498aa2efbf04ec631daa67571857bc7" target='_blank'>
              Phenotypic heterogeneity and tumor immune microenvironment directed therapeutic strategies in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Remya P. G. Ramesh, Hadida Yasmin, Pretty Ponnachan, Basel Al-Ramadi, Uday Kishore, Ann Mary Joseph
          </td>
          <td>2025-03-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528f741dd4b2dd6cad4ba0b800a0b727e027e98c" target='_blank'>
              Abstract 2583: A T cell engager targeting cancer associated fibroblasts inhibits tumor growth and modulates the tumor microenvironment
              </a>
            </td>
          <td>
            Joshua R Tagore, Robert J Norgard, Pratha Budhani, Sai Charan Penikalapati, Jessica N Egan, Di Feng, Leticia Corrales-Pecino, Anna Bachmayr-Heyda, Claudia Reichel-Voda, Abhishek Kashyap, Youli Xia, Sarah A O'Brien
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Attempts to activate an anti-tumor immune response in glioblastoma (GBM) have been met with many challenges due to its inherently immunosuppressive tumor microenvironment. The degree and mechanisms by which molecularly and phenotypically diverse tumor-propagating glioma stem cells (GSCs) contribute to this state are poorly defined. In this study, our multifaceted approach combining bioinformatics analyses of clinical and experimental datasets, single-cell sequencing, and the molecular and pharmacologic manipulation of patient-derived cells identified GSCs expressing immunosuppressive effectors mimicking regulatory T cells (Tregs). We showed that this immunosuppressive Treg-like (ITL) GSC state is specific to the mesenchymal GSC subset and is associated with and driven specifically by TGFβ type II receptor (TGFBR2) in contrast to TGFBR1. Transgenic TGFBR2 expression in patient-derived GBM neurospheres promoted a mesenchymal transition and induced a six-gene ITL signature consisting of CD274 (PD-L1), NT5E (CD73), ENTPD1 (CD39), LGALS1 (galectin-1), PDCD1LG2 (PD-L2), and TGFB1. This TGFBR2-driven ITL signature was identified in clinical GBM specimens, patient-derived GSCs, and systemic mesenchymal malignancies. TGFBR2high GSCs inhibited CD4+ and CD8+ T cell viability and their capacity to kill GBM cells, effects reversed by pharmacologic and shRNA-based TGFBR2 inhibition. Collectively, our data identify an immunosuppressive GSC state that is TGFBR2-dependent and susceptible to TGFBR2-targeted therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53e8c4d379e1f71697d7c09980cbde8eef065846" target='_blank'>
              Regulatory T Cell Mimicry by a Subset of Mesenchymal GBM Stem Cells Suppresses CD4 and CD8 Cells
              </a>
            </td>
          <td>
            Amanda L Johnson, Harmon S Khela, Jack Korleski, Sophie Sall, Yunqing Li, Weiqiang Zhou, Karen Smith-Connor, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2025-04-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, X. Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a05e53f4a69be4baa72f200b1b7350b5d95d4c02" target='_blank'>
              High-Efficiency Cell-Type Proteomics Strategy Deciphers Cholangiocarcinoma Fibrosis-Associated Pathological Heterogeneity.
              </a>
            </td>
          <td>
            Zhiyang Su, Honghua Zhang, Hongke Hu, Yun Yang, Chuanxi Huang, Chao Liu, Fuchu He, Wendong Chen
          </td>
          <td>2025-03-03</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdedfede7608cb2fb7f1fa7b1c417998bb4fd21" target='_blank'>
              Spatially resolved transcriptomic analysis of the adult human prostate.
              </a>
            </td>
          <td>
            Junyi Hu, Fei Liu, Jing Zhang, Lei Yin, Wanli Cao, Weidong Xu, Yifan Chang, Ye Wang, Jian Wang, Yaxin Hou, Lilong Liu, Sujun Chen, Guanghui Zhu, Junhui Jiang, Zixian Wang, Gong-Hong Wei, H. H. He, Di Gu, Ke Chen, Shancheng Ren
          </td>
          <td>2025-04-01</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Primary liver cancer (PLC) is characterized by significant cellular and molecular heterogeneities. To aid research in this field, we developed LiverSCA, a web-based cell atlas with an intuitive interface that enables users to explore gene expression patterns, cellular compositions, and intercellular communication within liver and PLC tumor microenvironments. The catalog of the latest version of LiverSCA was expanded by including additional datasets, such as intrahepatic cholangiocarcinoma (ICC) and the livers of metabolic dysfunction-associated steatotic liver disease (MASLD/MASH). It now features six phenotypes (normal, HBV-HCC, HCV-HCC, non-viral HCC, ICC, and MASH liver), encompassing data from 63 patients and >248,000 cells. New comparative visualization methods allow users to examine gene expression levels between different phenotypes, making LiverSCA a vital resource for researchers investigating the cellular and molecular landscapes of PLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df69e8507c814680caa9a1cc82d0a28d4c982ee3" target='_blank'>
              LiverSCA 2.0: An Enhanced Comprehensive Cell Atlas for Human Hepatocellular Carcinoma and Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Tina Suoangbaji, Renwen Long, I. Ng, Loey L.Y. Mak, D. Ho
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94a034a47df46d31c3bc02070b210d92025cd351" target='_blank'>
              Abstract 5283: Decoding and disrupting the coupled cellular plasticity and myeloid cell instigation in metastatic prostate cancer
              </a>
            </td>
          <td>
            Tianhe Huang, Yaoyin Li, Jun Wan, Liang Cheng, Xin Lu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6e0f5dce3feb58cd819120465c413a2814ac1e2" target='_blank'>
              Path2Omics: Enhanced transcriptomic and methylation prediction accuracy from tumor histopathology
              </a>
            </td>
          <td>
            Danh-Tai Hoang, E. Shulman, S. R. Dhruba, N. Nair, R. Barman, H. Lalchungnunga, Omkar Singh, MacLean P Nasrallah, Eric A Stone, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff150fb08f88fc51a25fd22d924b89d76aa03ba" target='_blank'>
              Cell Confluency Affects p53 Dynamics in Response to DNA Damage.
              </a>
            </td>
          <td>
            Alina Simerzin, Emily E Ackerman, Kotaro Fujimaki, Rainer H. Kohler, Yoshiko Iwamoto, Mathias S. Heltberg, A. Jambhekar, Ralph Weissleder, G. Lahav
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a60e85c34b84d3ee7bcc4641260225bbd7766477" target='_blank'>
              Abstract 6350: Exploring genetic and non-genetic evolution of glioblastoma by integrating single cell DNA and RNA sequencing
              </a>
            </td>
          <td>
            F. D’Angelo, A. Ko, L. Garofano, Pedro Davila, Luigi Ferraro, Benjamin Currall, S. Williams, S. Gultekin, Ashish Shah, Ricardo Komotar, Michael Ivan, Corneliu Sologon, A. Laurenge, A. Picca, Amel Dridi-Aloulou, B. Mathon, Franck Bielle, Stephan Zuchner, M. D. L. Fuente, M. Eoli, Chul-Kee Park, Michele Ceccarelli, Marc Sanson, A. Lasorella, A. Iavarone
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f43447092efa26d9874b71c88b8fd148bb746d9b" target='_blank'>
              Abstract 5863: Mapping tumor metastasis and EMT-associated immune changes in HNSCC through spatial transcriptomic analysis
              </a>
            </td>
          <td>
            Chih-Hung Chung, Muh-Hwa Yang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d6445efc526771002f7bfac5c1cb947546e45a" target='_blank'>
              Abstract 5456: Development of human antibodies targeting a novel cancer-associated fibroblast protein identified from spatial transcriptomics
              </a>
            </td>
          <td>
            Jinyeoung Choi, Shengjun Li, Minkyu Kim, Yeonjae Jung, Jae Eun Lee, D. Lee, Hongyoon Choi, Hyung-Jun Im
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Tumour‐associated microbiota are integral components of the tumour microenvironment (TME). However, previous studies on intratumoral microbiota primarily rely on bulk tissue analysis, which may obscure their spatial distribution and localized effects. In this study, we applied in situ spatial‐profiling technology to investigate the spatial distribution of intratumoral microbiota in breast cancer and their interactions with the local TME. Using 5R 16S rRNA gene sequencing and RNAscope FISH/CISH on patients’ tissue, we identified significant spatial heterogeneity in intratumoral microbiota, with Fusobacterium nucleatum (F. nucleatum) predominantly localized in tumour cell‐rich areas. GeoMx digital spatial profiling (DSP) revealed that regions colonized by F. nucleatum exhibit significant influence on the expression of RNAs and proteins involved in proliferation, migration and invasion. In vitro studies indicated that co‐culture with F. nucleatum significantly stimulates the proliferation and migration of breast cancer cells. Integrative spatial multi‐omics and co‐culture transcriptomic analyses highlighted the MAPK signalling pathways as key altered pathways. By intersecting these datasets, VEGFD and PAK1 emerged as critical upregulated proteins in F. nucleatum‐positive regions, showing strong positive correlations with MAPK pathway proteins. Moreover, the upregulation of VEGFD and PAK1 by F. nucleatum was confirmed in co‐culture experiments, and their knockdown significantly reduced F. nucleatum‐induced proliferation and migration. In conclusion, intratumoral microbiota in breast cancer exhibit significant spatial heterogeneity, with F. nucleatum colonization markedly altering tumour cell protein expression to promote progression and migration. These findings provide novel perspectives on the role of microbiota in breast cancer, identify potential therapeutic targets, and lay the foundation for future cancer treatments. Key points Intratumoral Fusobacterium nucleatum exhibits significant spatial heterogeneity within breast cancer tissues. F. nucleatum colonization alters the expression of key proteins involved in tumour progression and migration. The MAPK signalling pathway is a critical mediator of F. nucleatum‐induced breast cancer cell proliferation and migration. VEGFD and PAK1 are potential therapeutic targets to mitigate F. nucleatum‐induced tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b181d8c04116c66c673f00453e25d5cfea9b8649" target='_blank'>
              Integrated spatial multi‐omics profiling of Fusobacterium nucleatum in breast cancer unveils its role in tumour microenvironment modulation and cancer progression
              </a>
            </td>
          <td>
            Feng Zhao, Rui An, Yilei Ma, Shaobo Yu, Yuzhen Gao, Yanzhong Wang, Haitao Yu, Xinyou Xie, Jun Zhang
          </td>
          <td>2025-03-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract To unravel the mechanism of immune activation and suppression within tumors, a critical step is to identify transcriptional signals governing cell–cell communication between tumor and immune/stromal cells in the tumor microenvironment. Central to this communication are interactions between secreted ligands and cell-surface receptors, creating a highly connected signaling network among cells. Recent advancements in in situ-omics profiling, particularly spatial transcriptomic (ST) technology, provide unique opportunities to directly characterize ligand–receptor signaling networks that power cell–cell communication. In this paper, we propose a novel statistical method, LRnetST, to characterize the ligand–receptor interaction networks between adjacent tumor and immune/stroma cells based on ST data. LRnetST utilizes a directed acyclic graph model with a novel approach to handle the zero-inflated distributions of ST data. It also leverages existing ligand–receptor regulation databases as prior information, and employs a bootstrap aggregation strategy to achieve robust network estimation. Application of LRnetST to ST data of high-grade serous ovarian tumor samples revealed both common and distinct ligand–receptor regulations across different tumors. Some of these interactions were validated through both a MERFISH dataset and a CosMx SMI dataset of independent ovarian tumor samples. These results cast light on biological processes relating to the communication between tumor and immune/stromal cells in ovarian tumors. An open-source R package of LRnetST is available on GitHub at https://github.com/jie108/LRnetST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db23b179c860dc1d8c3febc0dd28265cf6aa4993" target='_blank'>
              Learning directed acyclic graphs for ligands and receptors based on spatially resolved transcriptomic data of ovarian cancer
              </a>
            </td>
          <td>
            Shrabanti Chowdhury, S. Ferri-Borgogno, Peng Yang, Wenyi Wang, Jie Peng, Samuel C. Mok, Pei Wang
          </td>
          <td>2025-03-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976e22baff59fe9d920ebd8e3c77016a68c58b9" target='_blank'>
              Single-Cell Transcriptomics of Multi-Site Cell Therapy in Osteoarthritis: Tissue-Specific Traits and Treatment Correlations
              </a>
            </td>
          <td>
            P. Chatterjee, H. Stevens, L. Kippner, C. Yeago, H. Drissi, K. Mautner, S. Boden, G. Gibson, K. Roy
          </td>
          <td>2025-04-16</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f05272abf51c9f38dd50672d47db4690dbc1bb3" target='_blank'>
              Spatial Profiling Identifies Regionally Distinct Microenvironments and Targetable Immunosuppressive Mechanisms in Pediatric Osteosarcoma Pulmonary Metastases.
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice S Pereira, Demond Williams, Douglass W Morris-Ii, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael C. Kelly, Amy K LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-04-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a70d8ffb57c8661082ae6d08c258b2d7875bf6f" target='_blank'>
              Abstract 3894: Three-dimensional whole tumor genomic analysis redefine intratumor heterogeneity of gliomas, with implications for early tumorigenesis
              </a>
            </td>
          <td>
            Lee H. Chen, Nicholas O Stevers, Chibo Hong, Ivan V. Smirnov, Nadia R. Grishanina, Joshua S. Wu, Anny Shai, Susan Marina Chang, Mitchell S. Berger, Shawn L. Hervey-Jumper, Joanna J Phillips, Joseph F Costello
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e120967c6039c9b8a8507c569dbb9140d2239a00" target='_blank'>
              Abstract 5320: Integrative spatial omics reveals distinct tumor-promoting multicellular niches and immunosuppressive mechanisms in African American and European American patients with TNBC
              </a>
            </td>
          <td>
            Qian Zhu, Akhila Balasubramanian, Jaya Ruth Asirvatham, Badrajee Piyarathna, Jaspreet Kaur, Nada Mohamed, Ling Wu, Megha Chatterjee, Stacy Wang, Niloufar Pourfarrokh, Uttam Rasaily, Yitian Xu, Junjun Zheng, D. Jebakumar, Arundathi Rao, Shu-Hsia Chen, Yi Li, Eric Chang, Xiaoxian Li, R. Aneja, X. Zhang, Arun Sreekumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background In recent years, immunotherapy has made significant progress. However, the understanding of the heterogeneity and function of T cells, particularly CD8 + T cells, in cervical cancer (CESC) microenvironment remains insufficient. We aim to characterize the heterogeneity, developmental trajectory, regulatory network, and intercellular communication of CD8 + T cells in cervical squamous cell carcinoma and to construct a prognostic risk model based on the transcriptomic characteristics of CD8 + T cells. Methods We integrated single-cell RNA sequencing data from CESC tumor samples with bulk transcriptome data from TCGA and GEO databases. We identified CD8 + T cell subsets in the CESC microenvironment, revealing significant interactions between CD8 + T cells and other cell types through intercellular communication analysis. Pseudotime trajectory analysis revealed dynamic transcriptional regulation during CD8 + T cell differentiation and functional acquisition processes. We constructed a transcriptional regulatory network for CESC CD8 + T cells, identifying key transcription factors. Based on CD8 + T cell-related genes, a prognostic risk model comprising eight core genes was developed and validated using machine learning. Results We identified four distinct CD8 + T cell subsets, namely progenitor, intermediate, proliferative, and terminally differentiated, each exhibiting unique transcriptomic characteristics and functional properties. CD8 + T cell subsets interact with macrophages through different ligand-receptor networks, including the CCL-CCR signaling pathway and costimulatory molecules. Sorafenib was identified as a potential immunotherapeutic drug through drug screening. Experimental validation demonstrated that sorafenib enhances the cytotoxicity of CD8 + T cells by increasing the secretion of IFN-γ and TNF-α, thereby significantly inhibiting the invasiveness and survival of CESC cells. Conclusions Our study provides valuable insights into the heterogeneity and functional diversity of CD8 + T cells in CESC. We demonstrate that a CD8 + T cell-related prognostic signature may serve as a potential tool for risk stratification in patients with CESC. Additionally, our finding suggests that sorafenib could be a promising therapeutic candidate for improving antitumor immunity in this patient population. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-13901-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60059599d76213441d05a5747700b975fa9d0f06" target='_blank'>
              Single-cell transcriptomic analysis reveals CD8 + T cell heterogeneity and identifies a prognostic signature in cervical cancer
              </a>
            </td>
          <td>
            Rongbin Zhou, Yuli Xie, Zuheng Wang, Zige Liu, Wenhao Lu, Xiao Li, Chunmeng Wei, Xing Li, Fubo Wang
          </td>
          <td>2025-03-18</td>
          <td>BMC Cancer</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc725ddd29c73799b619a4c59fece243a3bbe0ad" target='_blank'>
              Abstract 2578: GDF15 reprograms the microenvironment to drive the development of uveal melanoma liver metastases
              </a>
            </td>
          <td>
            S. Sriramareddy, Navya Siddarajappa, Thanh Nguyen, Jeffim Kunetsov, Stefan Kurtenbach, J. Dollar, Jonathan D. Licht, , William J. Harbour, K. Smalley
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8f2b1ddae01e13edf72bbcb5e1a3343151981e" target='_blank'>
              Abstract 7497: Nanopore sequencing of single-cell derived sublines provides insights into melanoma heterogeneity and evolution
              </a>
            </td>
          <td>
            Anton Goretsky, Yuelin Liu, A. Keskus, Tanveer Ahmad, S. Malikić, Glenn Merlino, Chi-Ping Day, Erin K. Molloy, S. C. Sahinalp, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/257016c16ec43ed0b67fd6524a0b1700be98dc49" target='_blank'>
              Bioengineered Tumor-Stroma Prostate Cancer In Vitro Models for Screening Therapeutics.
              </a>
            </td>
          <td>
            M. Monteiro, Filipa Moreira-Silva, Matilde R Lagarto, Luís P. Ferreira, Carlota Ramalhinho, Iola F. Duarte, Carmen Jerónimo, V. Gaspar, João F. Mano
          </td>
          <td>2025-03-13</td>
          <td>Biotechnology and bioengineering</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d80f820ef529b42735b0a0485a4a2c9e277fa1" target='_blank'>
              Abstract 6309: CliPP-on-Web: a dynamic platform for clonal structure identification through penalizing pairwise differences in cancers
              </a>
            </td>
          <td>
            Aaron Wu, Quang Tran, Matthew D. Montierth, Yujie Jiang, Jessica C. Lai, Shuangxi Ji, Wenyi Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Coupling single-cell transcriptomics with tumor-initiating cell enrichment identified IFN response gene expression not previously reported in bulk RNA-sequencing–derived signatures and proposed IFN/STAT1 signaling as a candidate therapeutic target in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49032c731c7f43ad3396d95549c8d805e477fd3" target='_blank'>
              Lineage Tracing and Single-Cell RNA Sequencing Reveal a Common Transcriptional State in Breast Cancer Tumor–Initiating Cells Characterized by IFN/STAT1 Activity
              </a>
            </td>
          <td>
            Eric P Souto, Ping Gong, John D. Landua, R. Srinivasan, Abhinaya Ganesan, L. Dobrolecki, S. C. Purdy, Xingxin Pan, Michael Zeosky, Anna Chung, S. S. Yi, H. Ford, Michael T. Lewis, Connie Cepko, Dr. Kenneth L. Scott, Dr. Michael Davidson
          </td>
          <td>2025-02-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b581086a365c6022e8cfc4df2d851d53411c3c94" target='_blank'>
              494 Multi-Omics Integration and Mapping of Invasion Markers Reveal Tumor-Specific Druggable Targets in Primary Human Glioblastoma
              </a>
            </td>
          <td>
            Paola Suarez-Meade, Jean R. Clemenceau, Isabel Barnfather, Mark Edgar, Aleeshba N. Basil, Tae H. Hwang, A. Quiñones-Hinojosa
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13bcea3579836571808ac96de5e2e45ad09f6441" target='_blank'>
              Abstract 6418: Non-genetic drug adaptation to MAPK pathway inhibitors in melanoma
              </a>
            </td>
          <td>
            Varuna Nangia, Humza Ashraf, Nasreen Marikar, Victor J. Passanisi, Christopher Ryland Ill, Timothy E. Hoffman, Sabrina L. Spencer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/086b2a8f8a0217ab42193f943b683f5fc349b5af" target='_blank'>
              Abstract 2749: Single-cell 3D genome structure analysis reveals clonal evolution and heterogeneity in acute myeloid leukemia
              </a>
            </td>
          <td>
            Yu Luan, Ye Hou, Yutong Lei, Qiushi Jin, Jie Xu, Bei Jia, Siwei Xu, Ping Wang, Alok Swaroop, Yihao Fu, Juan Wang, Qixuan Wang, Hong Zheng, Jing Zhang, Feng Yue
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and lethal malignancy, with a five-year survival rate of just 50% for cases of locally advanced disease. Chromosomal aberrations, particularly the deletion of the short arm of chromosome 3 (3p), have been strongly associated with poor prognosis and more aggressive tumor phenotypes. The tumor microenvironment (TME) plays a pivotal role in tumor progression and resistance to therapy. This study aims to elucidate the impact of 3p deletion on the TME, immune cell infiltration, and treatment response in HNSCC, to identify novel therapeutic targets to improve patient outcomes. We analyzed single-cell RNA sequencing (scRNA-seq) data from the Gene Expression Omnibus (GEO) and bulk transcriptome data from The Cancer Genome Atlas (TCGA). Pseudo-time trajectory and cell-cell communication analyses were performed with the Monocle and CellChat packages. The Wilcoxon test was used to evaluate the differential gene expression between wild-type (wt) and mutant (mut) groups. Prognostic models were developed using the Least Absolute Shrinkage and Selection Operator (LASSO) algorithm and Cox regression analyses to find the genes related to survival, with survival analysis conducted via Kaplan-Meier curves. Gene set enrichment analysis (GSEA) was employed to investigate pathway dysregulation, and immune cell infiltration was assessed using various immune scoring methodologies to explore the differences immune environment. The Tumor Immune Dysfunction and Exclusion (TIDE) database was utilized to predict the potential efficacy of immune checkpoint inhibitors. mRNA and protein expression levels of SPP1 were examined by RT-qPCR and Western blotting, while cell proliferation was assessed using the CCK8 assay. The mut group demonstrated significant alterations in cellular composition, characterized by increased endothelial cells and macrophages and decreased fibroblasts and CD8 + T cells, indicative of an immunosuppressive TME. Differential expression analysis revealed downregulation of immune pathways, including antigen processing and presentation, T cell receptor signaling, and B cell receptor signaling pathways in the mut group, along with enhanced metabolic activity in glycolysis and lipid metabolism. The prognostic model identified nine key genes associated with poor survival in HNSCC. The mut group exhibited poorer overall survival and a more immunosuppressive microenvironment compared to the wt group, which correlated with the outcomes observed in high-risk versus low-risk groups. High-risk patients also showed a diminished response to immunotherapy compared to low-risk patients. Additionally, SPP1 emerged as a critical gene associated with chemotherapy resistance and macrophage M2 polarization. This study demonstrates that 3p deletion significantly reshapes the TME, contributing to poor prognosis in HNSCC by fostering an immunosuppressive environment and enhancing chemoresistance. These findings highlight the potential for developing targeted therapies that address the genetic and immunological landscape of HNSCC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92078-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b962c43e9d02507e6b761c630a0667f1e2bd3ab7" target='_blank'>
              Abstract 5540: Deconstructing the lethality of recurrence in ovarian cancer through multi-modal spatiotemporal analysis
              </a>
            </td>
          <td>
            James M. Whipman, Francis Jacob, Viola Heinzelmann, Bernd Bodenmiller
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39956f5fc30601e3e3ded1b20651b5d5ee0b1a25" target='_blank'>
              Abstract 2454: Spatial profiling of arsenic-responsive gene expression in bladder cancer models via multiplex in situ hybridization
              </a>
            </td>
          <td>
            S. Singhal, K. Gardner, Michael L. Miller, Anushka Dixit, Samarth Singhal, Donald Sens, Sandeep K Singhal
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f6e80235cbbb65b993c05ceb81c0545572bfc88" target='_blank'>
              Robust Spatial Cell-Type Deconvolution with Qualitative Reference for Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Qishi Dong, Yi Yang, Ziye Luo, Haipeng Shen, Xingjie Shi, Jin Liu
          </td>
          <td>2025-03-09</td>
          <td>Small methods</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87978b2579ac3a33bf3369fd95388496b8a42e2" target='_blank'>
              Genomic and Single-Cell Analyses Characterize Patient-Derived Tumor Organoids to Enable Personalized Therapy for Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Jung Hyun Um, Yueyuan Zheng, Qiong Mao, Chehyun Nam, Hua Zhao, Yoon Woo Koh, Su-Jin Shin, Y. Park, 
          </td>
          <td>2025-04-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18fc5411928817db2e4ae82f8b06dabe1ec683eb" target='_blank'>
              Abstract 2672: Eco-evolutionary profiling of the UV-driven clonal dynamics and tumor initiation
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af060e4cc93259c4d19b62b9cc2136ad5699039" target='_blank'>
              ROICellTrack: A deep learning framework for integrating cellular imaging modalities in subcellular spatial transcriptomic profiling of tumor tissues.
              </a>
            </td>
          <td>
            Xiaofei Song, Xiaoqing Yu, Carlos M. Morán-Segura, Hongzhi Xu, Tingyi Li, Joshua T Davis, A. Vosoughi, G. D. Grass, Roger Li, Xuefeng Wang
          </td>
          <td>2025-04-08</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance. Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features. Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations. Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF-mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f549d282ce700f7e5436de47eb2b3446fdb14fc8" target='_blank'>
              Abstract 4250: Tumor-wide RNA splicing aberrations generate immunogenic public neoantigens across cancers
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, T. Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia R. Grishanina, James Woo, Aaron A. Diaz, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, H. Okada
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The tumor microenvironment profoundly influences tumor complexity, particularly in breast cancer, where cancer-associated fibroblasts play pivotal roles in tumor progression and therapy resistance. Extracellular vesicles are involved in mediating communication within the TME, specifically highlighting their role in promoting the transformation of normal fibroblasts into cancer-associated fibroblasts. Recently, we identified an RNA aptamer, namely ex.50.T, that binds with remarkable affinity to extracellular vesicles shed from triple-negative breast cancer cells. Here, through in vitro assays and computational analyses, we demonstrate that the binding of ex.50.T to extracellular vesicles and parental breast cancer cells is mediated by recognition of gremlin-1 (GREM1), a bone morphogenic protein antagonist implicated in breast cancer aggressiveness and metastasis. Functionally, we uncover the role of ex.50.T as an innovative therapeutic agent in the process of tumor microenvironment re-modeling, impeding GREM1 signaling, blocking triple-negative breast cancer extracellular vesicles internalization in recipient cells, and counteracting the transformation of normal fibroblasts into cancer-associated fibroblasts. Altogether, our findings highlight ex.50.T as a novel therapeutical avenue for breast cancer and potentially other GREM1-dependent malignancies, offering insights into disrupting TME dynamics and enhancing cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/938a5e8d747bd6c772f417e0ea77299cf86f62cf" target='_blank'>
              Ex.50.T aptamer impairs tumor–stroma cross-talk in breast cancer by targeting gremlin-1
              </a>
            </td>
          <td>
            C. Quintavalle, Francesco Ingenito, G. Roscigno, Birlipta Pattanayak, C. Esposito, Alessandra Affinito, Danilo Fiore, Gianluca Petrillo, Silvia Nuzzo, B. Della Ventura, Federica D'Aria, C. Giancola, S. Mitola, E. Grillo, Marinella Pirozzi, Greta Donati, F. S. Di Leva, Luciana Marinelli, Zoran Minic, Francesca De Micco, Guglielmo Thomas, M. Berezovski, G. Condorelli
          </td>
          <td>2025-03-11</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f53c3990c3e7c0a0efba69bc39d2581a71cded1" target='_blank'>
              Abstract 1154: SCRUM-Japan MONSTAR-SCREEN-3: A large-scale multi-institutional prospective study platform integrating spatial transcriptomics and multi-omics analysis for comprehensive understanding of tumor microenvironment dynamics
              </a>
            </td>
          <td>
            Mitsuho Imai, Shingo Sakashita, J. Yuda, Shin Kobayashi, Riu Yamashita, Tadayoshi Hashimoto, Takao Fujisawa, T. Shibuki, Shun-Ichiro Kageyama, Yoshikatsu Koga, Michiko Nagamine, Takeshi Kuwata, Hideaki Bando, Y. Nakamura, Woong-Yang Park, Takayuki Yoshino
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0c39aa857bbfd9543aa84644d12df58e1a71f0a" target='_blank'>
              Single-cell and spatial RNA sequencing identify divergent microenvironments and progression signatures in early- versus late-onset prostate cancer.
              </a>
            </td>
          <td>
            Yifei Cheng, Bingxin Liu, Junyi Xin, Xiaobin Wu, Wenchao Li, Jinwei Shang, Jiajin Wu, Zhengdong Zhang, Bin Xu, Mulong Du, Gong Cheng, Meilin Wang
          </td>
          <td>2025-04-10</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The presence of pre-existing or acquired drug-resistant cells within the tumor often leads to tumor relapse and metastasis. Single-cell RNA sequencing (scRNA-seq) enables elucidation of the subtle differences in drug responsiveness among distinct cell subpopulations within tumors. A few methods have employed scRNA-seq data to predict the drug response of individual cells to date, but their performance is far from satisfactory. In this study, we propose SSDA4Drug, a semi-supervised few-shot transfer learning method for inferring drug-resistant cancer cells. SSDA4Drug extracts pharmacogenomic features from both bulk and single-cell transcriptomic data using semi-supervised adversarial domain adaptation. This allows us to transfer knowledge of drug sensitivity from bulk-level cell lines to single cells. We conduct extensive performance evaluation experiments across multiple independent scRNA-seq datasets, demonstrating SSDA4Drug’s superior performance over current state-of-the-art methods. Remarkably, with only one or two labeled target-domain samples, SSDA4Drug significantly boosts the predictive performance of single-cell drug responses. Moreover, SSDA4Drug accurately recapitulates the temporally dynamic changes of drug responses during continuous drug exposure of tumor cells, and successfully identifies reversible drug-responsive states in lung cancer cells, which initially acquire resistance through drug exposure but later restore sensitivity during drug holidays. Also, our predicted drug responses consistently align with the developmental patterns of drug sensitivity observed along the evolutionary trajectory of oral squamous cell carcinoma cells. In addition, our derived SHAP values and integrated gradients effectively pinpoint the key genes involved in drug resistance in prostate cancer cells. These findings highlight the exceptional performance of our method in determining single-cell drug responses. This powerful tool holds the potential for identifying drug-resistant tumor cell subpopulations, paving the way for advancements in precision medicine and novel drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4755a58204169a2b082c843078f1434fb58c7419" target='_blank'>
              Identification of drug-resistant individual cells within tumors by semi-supervised transfer learning from bulk to single-cell transcriptome
              </a>
            </td>
          <td>
            Kaishun Huang, Hui Liu
          </td>
          <td>2025-03-31</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5507e82127e26407851957911f722cad0e285824" target='_blank'>
              Abstract 49: Genetic, genomic, and pharmacologic characterization of a new cohort of ancestrally diverse patient-derived xenograft models of cancer
              </a>
            </td>
          <td>
            Nina M. Dashti-Gibson, Amy Olex, Katarzyna M Tyc, Carson J Walker, Oniya M. Smith, Rachel K. Myrick, Bin Hu, Praveen Bhoopathi, Madhavi Puchalapalli, Li Wang, Yue Zhang, Brydon P. G. Wall, Valentina Robila, Mikhail G. Dozmorov, Jinze Liu, Jennifer E Koblinski, Jose G. Trevino, J. Harrell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Oral squamous cell carcinoma (OSCC) is an increasingly prevalent malignancy worldwide. This study aims to understand molecular alterations associated with lymph node metastasis of OSCC in order to improve treatment strategies. We analysed a cohort of 46 patients with primary OSCC, including 10 with lymph node metastasis and 36 without. Using a comprehensive multi‐omics approach – encompassing genomic, transcriptomic, proteomic, epigenetic, single‐cell, and spatial analyses – we integrated data to delineate the molecular landscape of OSCC in the context of lymph node metastasis. Our genomic analysis identified significant mutations in key genes within the MAPK, TGF‐β and WNT signalling pathways, which are essential for tumour development. The proteogenomic analysis highlighted pathways critical for lymph node dissemination and factors contributing to an immunosuppressive tumour microenvironment. Elevated levels of POSTN were found to reorganise the extracellular matrix (ECM), interact with TGF‐β, disrupt cell cycle regulation and suppress the immune response by reducing VCAM1 activity. Integrated analyses of single‐cell and spatial transcriptome data revealed that cancer‐associated fibroblasts (CAFs) secrete TGF‐β1/2, promoting cancer cell metastasis through epithelial–mesenchymal transition (EMT). Our integrated multi‐omics analysis provides a detailed understanding of molecular mechanisms driving lymph node metastasis of OSCC. These insights could lead to more precise diagnostics and targeted treatments. This article is protected by copyright. All rights reserved">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a9d23bd90d2baf9dbe230266b6f429b3c96696a" target='_blank'>
              Proteogenomic characterisation of primary oral cancer unveils extracellular matrix remodelling and immunosuppressive microenvironment linked to lymph node metastasis
              </a>
            </td>
          <td>
            Yu Liu, Zhenyu Yang, J. Pu, Jie Zhong, U. Khoo, Yu-Xiong Su, Gao Zhang
          </td>
          <td>2025-03-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6271a139874a284ffeac56a9ad3007bdfc0fd11" target='_blank'>
              Abstract 3448: Profiling the urinary bladder cancer composition and inter-cellular proximity patterns using single-cell RNA-sequencing and spatial transcriptomics
              </a>
            </td>
          <td>
            Alexandros Kostakis, Andrea Behanova, I. Kerzeli, Ramy Elgendy, A. Chourlia, Sven Nelander, P. Malmstrom, U. Segersten, Milena Doroszko, Anca Dragomir, Carolina Wahlby, Martin Lord, S. Mangsbo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9d728476d907142dd31d1c87ffc2d2c3f782be" target='_blank'>
              Abstract 1248: Single-cell RNAseq revealed multiple resistance mechanisms in patient-derived xenograft model of rectal cancer during treatment
              </a>
            </td>
          <td>
            Yuwei Song, Regina K. Irwin, Karin M. Hardiman, Zechen Chong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce63b80e10f58d9373f4a7b7bf33e6cdea133649" target='_blank'>
              Abstract 6355: Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy
              </a>
            </td>
          <td>
            Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen E. Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea1a8d6967192526fb7f6ecd11f9a23af479e446" target='_blank'>
              Abstract 7499: Driver mutations propel subclonal evolution and spatial patterns of tumor growth in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Evit R. John, Pooja Shukla, Anish Jain, Alexandria Lau, T. Lesluyes, Huw Ogilvie, M. Tarabichi, J. Roszik, Yinyin Yuan, P. Van Loo, Lawrence Kwong, Yun Shin Chun
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77c67aedf7052778400f6b8258e4d803910f72b0" target='_blank'>
              Abstract 2668: Spatial-temporal diversity of extrachromosomal DNA shapes urothelial carcinoma evolution and tumor-immune microenvironment
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, J. Luebeck, Huiying Tao, Yanfen Zhu, X. Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xiang Xi, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B. Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77cbbc2ef4270ebb636894a741346e8433adaa6" target='_blank'>
              Abstract 2552: TET2-mutant clonal hematopoiesis enrichment in the tumor microenvironment is an actionable driver of treatment resistance in solid tumors: Thyroid cancer as a paradigm
              </a>
            </td>
          <td>
            Pablo Sánchez Vela, Vera Tiedje, Julie L. Yang, B. Untch, Laura Boucai, Aaron J Stonestrom, Alberto Bueno Costa, Sebastià Franch Expósito, Avi Srivastava, M. Kerpelev, Jillian Greenberg, Matthew Wereski, Amanda Kulick, Kevin Chen, Tianyue Qin, Soo-Yeon Im, Anthony R Martinez Benitez, Raquel Pluvinet, Merve Sahin, K. Menghrajani, Gnana P. Krishnamoorthy, E. de Stanchina, A. Zehir, Rahul Satija, J. Knauf, R. Bowman, M. Esteller, Sean M Devlin, Michael F. Berger, R. Koche, Ross L. Levine, James A. Fagin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba2b2e41c178b6dbd66cbfc9ba10d448bf6866f2" target='_blank'>
              Spatial multi-omics reveals cell-type-specific nuclear compartments.
              </a>
            </td>
          <td>
            Yodai Takei, Yujing Yang, Jonathan White, Isabel N. Goronzy, Jina Yun, Meera Prasad, Lincoln J Ombelets, Simon Schindler, Prashant Bhat, Mitchell Guttman, L. Cai
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d17f58baf47231f215e90d6af25b566b44858b3" target='_blank'>
              Abstract 6262: Changes in the spatial architecture of the tumor microenvironment associated with treatment resistance in muscle invasive bladder cancer
              </a>
            </td>
          <td>
            Nikolay Alabi, Nicolas Zheng, Joshua Scurll, Jussi Nikkola, Alberti Contreras-Sanz, Morgan Roberts, A. Bashashati, Peter Black
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63a435551ab128bd453aa26cc898c619184a9dc" target='_blank'>
              Abstract 6238: Interactions between ploidy and resource availability shape clonal interference in glioblastoma
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, Konstantin Maksin, Parag Katira, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f3ab1eb05f9ea4f709c96a50d79b818a9e102da" target='_blank'>
              Single cell-derived spheroids for real-time growth and metabolomic studies in breast cancer
              </a>
            </td>
          <td>
            Margarida Esteves, Francisca Oliveira, Alexandra Teixeira, Arantxa Ibáñez, Andrea Garcia-Lizarribar, I. Madarieta, José Maria Fernandes, Carolina Rodrigues, Alar Ainla, Lorena Diéguez, B. Olalde, Miguel Xavier, S. Abalde‐Cela
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Glioblastomas (GBMs), among the most aggressive and resilient brain tumors, characteristically exhibit high angiogenic potential, leading to the formation of a dense yet aberrant vasculature, both morphologically and functionally. With these premises, numerous expectations were initially placed on anti-angiogenic therapies, soon dashed by their limited efficacy in concretely improving patient outcomes. Neovascularization in GBM soon emerged as a complex, dynamic, and heterogeneous process, hard to manage with the classical standard of care. Growing evidence has revealed the existence of numerous non-canonical strategies of angiogenesis, variously exploited by GBM to meet its ever-increasing metabolic demand and differently involved in tumor progression, recurrence, and escape from treatments. In this review, we provide an accurate description of each neovascularization mode encountered in GBM tumors to date, highlighting the molecular players and signaling cascades primarily involved. We also detail the key architectural and functional aspects characteristic of the GBM vascular compartment because of an intricate crosstalk between the different angiogenic networks. Additionally, we explore the repertoire of emerging therapies against GBM that are currently under study, concluding with a question: faced with such a challenging scenario, could combined therapies, tailored to the patient’s genetic signatures, represent an effective game changer?">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64cd1c95a63b7c6947153915736e2f9dc103a024" target='_blank'>
              Understanding Neovascularization in Glioblastoma: Insights from the Current Literature
              </a>
            </td>
          <td>
            Mariagiovanna Ballato, Emanuela Germanà, Gabriele Ricciardi, Walter Giordano, Pietro Tralongo, Mariachiara Buccarelli, G. Castellani, L. Ricci-Vitiani, Q. D’Alessandris, Giuseppe Giuffrè, Cristina Pizzimenti, V. Fiorentino, Valeria Zuccalà, A. Ieni, M. Caffo, Guido Fadda, M. Martini
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b23788a6f827c0b72ecb2ed6905b6d5b0e6258" target='_blank'>
              Abstract 3763: Genome-wide CRISPR screens identify critical targets to enhance CAR-NK cell antitumor potency
              </a>
            </td>
          <td>
            Alexander Biederstädt, R. Başar, J. Park, Merve Dede, John G. Watts, N. Uprety, R. Shrestha, Francia Reyes-Silva, May Daher, Bin Liu, H. Rafei, P. Banerjee, Ping Li, S. Acharya, , Huihui Fan, Donghai Xiong, Vernikka C. Woods, Paul Lin, Silvia Tiberti, A. N. Nunez Cortes, X. Jiang, Inci Biederstädt, Patrick Zhang, Ye Li, S. Rawal, Enli Liu, Luis Muniz-Feliciano, Gary M. Deyter, Elizabeth J. Shpall, N. Fowlkes, Ken Chen, Katayoun Rezvani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cell death is critical in tumor biology. The common cancer therapies can cause cell death and alleviate tumor, while the cancer cells can develop a resistance to cell death and survive from the therapies. Thus, not only observing the alternative mechanisms of tumor cells resistant to cell death, but also understanding the intricate dynamics of cell death processes within the tumor microenvironment (TME), are essential for tailoring effective therapeutic strategies. High-throughput sequencing technologies have revolutionized cancer research by enabling comprehensive molecular profiling. Recent advances in single cell sequencing have unraveled the heterogeneity of TME components, shedding light on their complex interactions. In this review, we explored the interplay between cell death signaling and the TME, summarised the potential drugs inducing cell death in pre-clinical stage, reviewed some studies applying next-generation sequencing technologies in cancer death research, and discussed the future utilization of updated sequencing platforms in screening novel treatment methods targeted cell death. In conclusion, leveraging multi-omics technologies to dissect cell death signaling in the context of the TME holds great promise for advancing cancer research and therapy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c40c61d7dcc3833421bbc3568f9b1a472c0c4795" target='_blank'>
              Cell death in tumor microenvironment: an insight for exploiting novel therapeutic approaches
              </a>
            </td>
          <td>
            Wenxin Wang, Tong Li, Kui Wu
          </td>
          <td>2025-03-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [8],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>