<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-02 10:01:28 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>103</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e69c4c3aa7ac04b1dca84c51fe6263d52efa060" target='_blank'>
              Breast cancer through the lens of whole transcriptome spatial imaging
              </a>
            </td>
          <td>
            Claire Williams, Yi Cui, Michael Patrick, Giang T Ong, Terence Theisen, Megan Vandenberg, Joseph M. Beechem, Patrick Danaher
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Melanoma outcomes have dramatically improved over the past decade, but some patients still experience disease recurrence, particularly those who present at later stage of disease. Prior studies identified an altered immune microenvironment in the sentinel lymph node (SLN) including the presence of dysfunctional CD8 T cells and CD4 T regulatory cells (Tregs), but the spatial organization of the SLN and the interactions of individual cell types have not been extensively studied. To understand how spatial organization of immune cells in the SLN is related to outcomes, we performed spatial proteomic profiling of Stage I and II melanoma SLN to generate an atlas of cell types. Following multiplexed immunofluorescence imaging, deep learning-based segmentation and clustering, we identified 33 subsets of T cells, B cells and Tumor cells. Lymphoid region analyses established a foundational spatial map of the melanoma SLN, revealing higher regional frequencies of activated and memory CD4 T cells in Stage I SLN and consistent positioning of T cells at functional spatial locations. To better understand cellular interactions, we evaluated the immediate neighbors around each index cell using Effect Size Interaction mapping (ESI-map), a novel computational toolkit for understanding spatial interactions. Nearest-neighbor analyses across biological replicates revealed rewiring in cell-cell interacting pairs across disease progression and patient outcomes, particularly with respect to exhausted TOX+ CD8 T cells. Stage II patients who experienced disease recurrence had notable enrichment of cellular interactions between Tregs and TOX+ CD8 T cells that was associated with reduced expression of granzyme B and interferon-γ that was most pronounced in exhausted CD8 T cells. Together, these data demonstrate that the specific spatial interactions between Tregs and exhausted CD8 T cells in the SLN provide a novel immune signature for understanding melanoma outcomes. One Line Summary Treg-CD8 T cell interactions in the sentinel lymph node predict recurrence outcomes in Stage II SLN+ melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927407a38e9474aa6152c45daf9e0743472dc6ee" target='_blank'>
              Cellular interactions in the sentinel lymph node predict melanoma recurrence
              </a>
            </td>
          <td>
            Sabrina M. Solis, Yang Yang, Yee Hoon Foong, Anushka Dheer, K. Ma, M. S. Farooq, Shira G. Rosenberg, Sophie L. Gray-Gaillard, G. Karakousis, Xiaowei Xu, Rami S. Vanguri, Alexander C. Huang, R. Herati
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d403636ddf059830232f0617d3f6cb6b77d1ff" target='_blank'>
              Genotype-phenotype mapping identifies fetal-like CD55+ immunoregulatory cancer cells as mediators of immune escape in colorectal cancer
              </a>
            </td>
          <td>
            M. Mastel, Aitana Guiseris Martinez, Gabriele Diamante, Jasmin Meier, Luzi Schuchmann, Nikolaos Georgakopoulos, Ioannis Chiotakakos, Laura K. Steffens, Carolin Artmann, Diego Benitez, Michael Günther, Vera Thiel, Roxana Pincheira, A. Trumpp, R. Offringa, S. Ormanns, Rene Jackstadt
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad9383618813c7fb20bfa893bf5368770766ebe" target='_blank'>
              Analysis of intratumoral immune heterogeneity reveals spatial organisation of immunosuppression in breast cancer
              </a>
            </td>
          <td>
            Christina Metoikidou, P. Bonté, Vadim K. Karnaukhov, Laëtitia Lesage, L. Djerroudi, André Nicolas, P. Sirven, A. Vincent-Salomon, Olivier Lantz, Emanuela Romano, Sebastián Amigorena
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d63cfbf90a2e544c2d24c8ab605e5d44e8ba24" target='_blank'>
              A Pan-Cancer Single-Cell Compendium of Intratumoural Heterogeneity
              </a>
            </td>
          <td>
            Ido Nofech-Mozes, Ashton Cook, Tom W. Ouellette, Sara Hafezi-Bakhtiari, Philip Awadalla, Sagi Abelson
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are the main constituents of the tumor microenvironment. Several studies have delineated CAF heterogeneity in different types of tumors, however, it is still unknown how the distinct CAF transcriptional profiles are established during tumor progression. We reanalyzed a previously published single-cell RNA-sequencing dataset of MMTV-PyMT tumors at higher resolution using Seurat , and CytoTRACE to characterize CAF subtypes and their differentiation states. Wilcoxon rank sum test was applied for differential gene expression. Multiplex immunostaining (Akoya PhenoImager HT) was performed on 38 murine mammary tumors from MMTV-PyMT mice to identify the distinct CAF subtypes. Whole-slide imaging and spatial analysis were conducted using QuPath and Cellpose , followed by neighborhood clustering and interaction mapping with CytoMAP . Cellular distances from CAFs to immune, tumor, and endothelial cells were quantified using SPIAT and Wilcoxon tests for comparisons. In parallel, human spatial transcriptomics data from the 10X Genomics Xenium platform were integrated for cross-species validation. Here, by single cell RNA-sequencing and multiplex immunostaining, we identify six CAF substates. Spatial analysis on immunostained murine mammary tumors and human spatial transcriptomics data outlined temporal changes in stromal composition and the existence of distinct functional niches enriched with different CAF substates. Immunomodulatory CAFs co-localized with immune cells while myofibroblastic CAFs formed a shield around the tumor core, thus preventing immune infiltration. Our work supports the idea that distinct spatial locations dictate different CAF transcriptional programs. Targeting specific functional niches will ultimately hinder tumor progression by inhibiting signaling between distinct CAF substates and the surrounding tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d802e2b1323bfcdbf953412180c241803eb33a" target='_blank'>
              Spatial and temporal dynamics of cancer-associated fibroblast niches in breast cancer
              </a>
            </td>
          <td>
            Jessica Pantaleo, J. Sjölund, P. Bolivar, M. Bocci, B. Phung, Maria Malmberg, G. Jönsson, Kristian Pietras
          </td>
          <td>2026-01-11</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="While advances in single-cell genomics have helped to chart the cellular components of tumor ecosystems, it has been more challenging to characterize their specific spatial organization and functional interactions. Here, we combine single-cell RNA-seq, spatial transcriptomics by Slide-seq, and in situ multiplex RNA analysis to create a detailed spatial map of healthy and dysplastic colon cellular ecosystems and their association with disease progression. We profiled inducible genetic CRC mouse models that recapitulate key features of human CRC, assigned cell types and epithelial expression programs to spatial tissue locations in tumors, and computationally used them to identify the regional features spanning different cells in the same spatial niche. We find that tumors were organized in cellular neighborhoods, each with a distinct composition of cell subtypes, expression programs, and local cellular interactions. Comparing to scRNA-seq and bulk RNA-seq data from human CRC, we find that both cell composition and layout features were conserved between the species, with mouse neighborhoods correlating with malignancy and clinical outcome in human patient tumors, highlighting the relevance of our findings to human disease. Our work offers a comprehensive framework that is applicable across various tissues, tumors, and disease conditions, with tools for the extrapolation of findings from experimental mouse models to human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af74f8f0f1a2428bab07b6671a3e436ab63d622f" target='_blank'>
              Spatially defined multicellular functional units in colorectal cancer revealed from single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            I. Avraham-Davidi, Simon Mages, J. Klughammer, Noa Moriel, Shinya Imada, Matan Hofree, Evan Murray, Jonathan Chen, Karin Pelka, Arnav Mehta, Genevieve M Boland, T. Delorey, Leah Caplan, Danielle Dionne, Robert Strasser, Jana Lalakova, Anežka Niesnerová, Hao Xu, Morgane Rouault, I. Tirosh, Hacohen Nir, Fei Chen, Omer Yilmaz, Jatin Roper, O. Rozenblatt-Rosen, Mor Nitzan, A. Regev
          </td>
          <td>2025-12-12</td>
          <td>eLife</td>
          <td>0</td>
          <td>195</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7a3ed4eb95927059b414292cf9cce743a10321" target='_blank'>
              Spatiotemporal Mapping of Tertiary Lymphoid Structure Heterogeneity Shapes Immune Niches and Clinical Outcomes in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Liaoliao Gao, Jie Mei, Libing Hong, Yuzhi Jin, Jinlin Cheng, Xuqi Sun, Chuan Liu, Bin Li, Xiaomeng Dai, Bo Lin, Yajie Sun, Peng Zhao, Minshan Chen, Rongping Guo, Shan Xin, Jingping Yun, Inmaculada Martínez-Reyes, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: In contrast to most solid tumors, high immune cell infiltration in clear cell renal cell carcinoma (ccRCC) is associated with poor patient prognosis. The biological mechanisms underlying this paradox remain unclear, particularly regarding tumor cell-microenvironment interactions promoting local invasion and recurrence. This study aimed to identify spatially resolved tumor, immune, and stromal features that define aggressive phenotypes in localized ccRCC. Methods: Multiplex immunofluorescence was performed using a 33-marker panel on 1,728 multi-region tissue cores from 435 surgically treated patients with localized ccRCC. Samples systematically included tumor centers, invasive borders, and adjacent benign tissue. Single-cell analyses quantified immune, stromal, endothelial, and epithelial cell populations within their spatial context. Results: Spatially resolved profiling uncovered a highly aggressive tumor subtype distinguished by fibroblast activation protein (FAP) expression on tumor epithelial cells, a marker typically associated with stromal cells. Tumor-cell-specific FAP expression characterized an epithelial-to-mesenchymal transition (EMT)-like state and was spatially associated with profound immunosuppression, marked by enrichment of regulatory T cells, exhausted CD8+ T cells, and M2-like macrophages, particularly at the invasive border. Tumor-cell FAP promoted invasion and independently predicted significantly poorer recurrence-free survival (RFS), even in early-stage disease (multivariable Cox p = 0.022 for pT1-2), surpassing established biomarkers such as PD-L1 in capturing aggressive biological features. Conclusions: Tumor epithelial FAP expression identifies an aggressive, immune-rich subtype of localized ccRCC, integrating EMT with spatially organized immunosuppression. These findings establish tumor-cell FAP as a promising biomarker with substantial translational potential for patient risk stratification, targeted imaging (FAPI-PET), and FAP-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42470b1512d22d2033f3b6dc6829a3e316db8519" target='_blank'>
              Tumor cell FAP orchestrates EMT and immune suppression in aggressive localized ccRCC
              </a>
            </td>
          <td>
            T. Pellinen, L. Luomala, K. Mattila, A. Hemmes, K. Välimäki, Mariliina Arjama, Oscar Brück, L. Paavolainen, Elisa Kankkunen, H. Nísen, P. Järvinen, Leticia Castillon, Sakari Vanharanta, P. Vainio, Olli Kallioniemi, Panu M. Jaakkola, T. Mirtti
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The tumor immune microenvironment (TIME) is closely involved in tumor initiation, malignant progression, immune escape, and response to immunotherapy. With the continued development of high-throughput sequencing technologies, transcriptomic approaches have become essential for examining the cellular and molecular features of the TIME. Bulk RNA sequencing offers tissue-level gene expression profiles and allows the estimation of immune cell composition through computational deconvolution. Single-cell RNA sequencing provides finer resolution, revealing cellular heterogeneity, lineage relationships, and functional states. Spatial transcriptomics (ST) retains the native anatomical context, making it possible to localize gene expression patterns and cell–cell interactions within intact tissues. These approaches, when considered together, have shifted TIME research from averaged measurements toward a more detailed and mechanistic understanding. This review summarizes the principles, applications and limitations of bulk, single-cell and spatial transcriptomic methods, highlighting emerging strategies for integrative analysis. Such multi-scale frameworks are increasingly important for studying immune dynamics and may contribute to the development of more precise biotechnological and immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a95188583167d2d5ed84b42177b48c6d850371a9" target='_blank'>
              Multi-Scale Transcriptomics Redefining the Tumor Immune Microenvironment
              </a>
            </td>
          <td>
            Jing Sun, Yingxue Xiao, Lingling Xie, Dan Qin, Yue Zou, Yingying Liu, Yitong Zhai, Minyi Zhang, Tong Li, Youjin Hao, Bo Li
          </td>
          <td>2026-01-15</td>
          <td>BioTech</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="While single-cell technologies have advanced bladder cancer research, the microenvironment of lymph node metastases and its role in progression remain unclear. We aimed to elucidate dynamic tumor-microenvironment interactions across metastatic and primary sites, identify poor-prognosis features, and uncover novel therapeutic targets. We profiled matched single-cell and spatial transcriptomic samples from lymph node-metastasized bladder cancer to construct a high-resolution atlas. Bioinformatic analyses were used to inferred ligand-receptor interactions, regulatory networks, and spatial co-localization. Integration with multiple prognostic cohorts revealed key cell subpopulations linked to tumor biology and poor outcomes. Our atlas included 32 samples, 61 cell subtypes, key ligand signals (e.g., TGFB1, OSM, IFNG), 343 transcription factors, 188 co-localization events, over 1,000 spatially validated interactions, and 21 cell niches. Tumor-driven convergence was observed across distinct microenvironments. Notably, a pericyte-like myofibroblast subtype, myoCAF_PLXDC1, was enriched in malignant tissues, associated with poor prognosis, and exhibited enhanced angiogenic but reduced chemotactic activity compared to myoCAF_CCL21. Its phenotype was linked to intracellular regulators (PRRX1 and MXD4) and extracellular signals (IFNG and TGFB1). Spatially, myoCAF_PLXDC1 co-localized with ITGA8+ endothelial cells and enriched in tumor regions. This study offers a comprehensive resource for lymph node-metastasized bladder cancer and identifies PLXDC1+ fibroblasts co-localized with ITGA8+ ECs as potential components of a tumor-specific vascular niche and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d7c9cdf4d9cefce93639f3970e5da305a8fc5c" target='_blank'>
              Single-cell and spatial atlas unveil tumor-specific microenvironment convergence and a prognosis-associated PLXDC1+ myofibroblast population in metastatic bladder cancer
              </a>
            </td>
          <td>
            Ziwei Wang, Jiaying Miao, Maoyu Wang, Yidie Ying, Zhensheng Zhang, Shuxiong Zeng, Chuanliang Xu
          </td>
          <td>2025-12-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Understanding how tissues remodel in response to perturbations requires computational tools that can untangle condition-specific changes from the conserved tissue architecture. We present Haruka, a spatially aware contrastive learning framework that identifies salient (condition-specific) and background (shared) spatial domains across tissue slices and experimental conditions. Haruka integrates contrastive variational inference with an auxiliary microenvironment reconstruction task, enabling the model to learn spatial-context-informed embeddings that capture both perturbation effects and local neighborhood context. Through benchmarking on simulated and real datasets, Haruka outperforms state-of-the-art methods in detecting spatially heterogeneous responses. Applied to diverse spatial omics platforms, Haruka distinguished immunotherapy responders in melanoma, traced fibrosis progression in human lung tissue, and mapped treatment-resistant microenvironments in KRASG12D-mutated lung cancer. Thus, Haruka provides a generalizable framework for spatial contrastive analysis, enabling systematic dissection of tissue organization, cellular plasticity, and microenvironmental remodeling across disease, development, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa467a4fed6deacca0c759e8099e8c763a30e66" target='_blank'>
              Haruka Resolves Perturbation Response Heterogeneity in Spatial Cell Niches
              </a>
            </td>
          <td>
            Yan Cui, J. Blandin, K. Weiskopf, Na Sun
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5486f6aaee8639d9484243cb88e9c2a7fbf4ba3" target='_blank'>
              Cancer-myeloid cell invasive program in pediatric-type diffuse high-grade glioma
              </a>
            </td>
          <td>
            C. Ruiz-Moreno, R. Collot, T. J. van den Broek, E. J. Wehrens, N. Bessler, Gitanjali Dharmadhikari, B. T. Te Pas, Ignacio L Ibarra, D. Metselaar, M. Kranendonk, E. Hoving, J. van der Lugt, F. Calkoen, A. Zomer, Fabian Theis, E. Hulleman, D. V. van Vuurden, A. Rios, H. Stunnenberg
          </td>
          <td>2026-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Cervical cancer (CC), a leading cause of cancer-related deaths among women worldwide, is primarily driven by high-risk human papillomavirus (HPV) infections and comprises two major histological subtypes: adenocarcinoma (AC) and squamous cell carcinoma (SCC). Despite advances in prevention and treatment, the molecular and cellular heterogeneity of these subtypes poses significant challenges to achieving optimal clinical outcomes. Here, we integrate single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) to dissect the cellular and spatial heterogeneity of AC and SCC, uncovering distinct tumor microenvironment (TME) dynamics that underlie their divergent clinical behaviors. Our scRNA-seq analysis reveals that AC is enriched in epithelial cells, while SCC exhibits a more immunogenic TME with elevated plasma cells and NK/T cells. Spatial transcriptomics further highlights robust interactions between CD8 + T cells and epithelial subtypes in SCC, contrasting with the stromal-rich, immune-cold phenotype of AC. We identify subtype-specific immune and stromal features, including ICOS+ Tregs, IDO1+ cancer-associated fibroblasts (CAFs), and PLVAP+ endothelial cells, which may drive immune evasion, angiogenesis, and metastasis. These findings provide a comprehensive framework for understanding CC heterogeneity and offer actionable insights for developing subtype-specific therapeutic strategies, such as combining immune checkpoint inhibitors with stromal-targeting agents. This study underscores the potential of spatial multi-omics technologies to advance precision oncology and improve outcomes for cervical cancer patients. Single-cell and spatial transcriptomics uncovered a stark contrast in the TMEs of cervical cancer subtypes: squamous cell carcinoma is more immunogenic, whereas adenocarcinoma is stromal-rich and immune-cold.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b0c553f54a8eda4d3dc1abe42f3db69a41cfe1" target='_blank'>
              Single-cell and spatial transcriptomic profiling reveals distinct tumor microenvironment dynamics in cervical adenocarcinoma and squamous cell carcinoma
              </a>
            </td>
          <td>
            Yaqin Wu, Bei Wei, Zhentong Wei, Hongyan Wang, Yunyun Liu, Dongdong Xu, Xiaokang Li, Liyuan Guo, Huaiwu Lu, Huiling Shang
          </td>
          <td>2025-12-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba1256bc5be1c15c5a52165e8dbea8c1e3c75014" target='_blank'>
              Integrative Single-cell and Spatial Transcriptomic Analysis of Osteosarcoma Reveals Conserved and Distinct Ecosystems Across Sites and Species
              </a>
            </td>
          <td>
            Yogesh Budhathoki, Matthew V. Cannon, T. A. McEachron, Anand G. Patel, Matthew J Gust, Jaime F Modiano, D.T. Ammons, Kathryn E Cronise, M. Macchietto, A. Sarver, Ruben Dries, B. Gryder, Daniel P. Regan, Heather L. Gardner, Ryan D. Roberts
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3333bb3dc7aa46ec7dea00023d761c89be2478c2" target='_blank'>
              Large-scale single-cell analysis and in silico perturbation reveal dynamic evolution of HCC: from initiation to therapeutic targeting.
              </a>
            </td>
          <td>
            Peng Xia, Shuang Si, Daosen Fu, Luyao Liu, Dongliang Yang, Yanan Guo, Yixiao Tian, Pengfei Ji, Xinyi Yuan, Yingxia Tian, Rong Shen, Degui Wang
          </td>
          <td>2026-01-30</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/342dd98c58335efcd7b4ec06f157649f394ee9d4" target='_blank'>
              Clinical feasibility of spatial transcriptomics using discarded tissue from diagnostic breast biopsies
              </a>
            </td>
          <td>
            Jeffrey P Sheridan, Quynh Sun, Jiaji G Chen, Yibing Wei, Veronica Jarzabek, Anirudh Jaishankar, Iqra Amin, L. Sultan, Thea Nalan, Sarah B. Mueller, Eric J. Burks, Lubna H. Suaiti, Christopher M. Heaphy, Naomi Y. Ko, Ruben Dries
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafddf4eb7b4f74e9d7699b13e08df2afba97c1e" target='_blank'>
              Pan-cancer tumor classification by a holistic tumor microenvironment atlas
              </a>
            </td>
          <td>
            Shishang Qin, Xiao Du, Jinhu Li, Nan Jiang, Tian Diao, Yufei Bo, Qinhang Gao, Liangtao Zheng, Xinnan Ling, Qianqian Gao, Xiangjie Li, Sen Gao, Fei Tang, Wenjie Zhang, Chenwei Li, Peihong Fang, Linna Zhu, Dongfang Wang, Zemin Zhang
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Understanding the spatial organization of individual cell types within tissue and how this organization is disrupted in disease, is a central question in biology and medicine. Hematoxylin and eosin-stained slides are widely available and provide detailed morphological context, while spatial gene expression profiling offers complementary molecular insights, though it remains costly and limited in accessibility. Predicting gene expression directly from histological images is therefore an attractive goal. However, existing approaches typically rely on small image patches, limiting resolution and the ability to capture fine-grained morphological variation. Here, we introduce a deep learning approach that predicts single-cell gene expression from morphology, matching patch-based methods on spot level prediction tasks. The model recovers biologically meaningful expression patterns across two cancer datasets and distinguishes fine cell populations. This approach enables molecular-level interpretation of standard histological slides at scale, offering new opportunities to study tissue organization and cellular diversity in health and disease. Predicting gene expression from H&E slides is a cost-effective alternative to Spatial Transcriptomics for clinical use. Here the authors introduce a Multiple Instance Learning approach to infer single-cell expression, unlike prior methods which operate at coarser patch level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d903aacfbf3a6dfc2c450d8cb95280e66a5321f" target='_blank'>
              sCellST predicts single-cell gene expression from H& E images
              </a>
            </td>
          <td>
            Loïc Chadoutaud, Marvin Lerousseau, Daniel Herrero-Saboya, Julian Ostermaier, Jacqueline Fontugne, Emmanuel Barillot, Thomas Walter
          </td>
          <td>2026-01-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be57704ac8a67d3c2570360452cb391401b265a" target='_blank'>
              A Comprehensive Treatment-Induced Resistance Atlas of Glioblastoma Reveals a Fibrotic Niche Shielding the Tumor from Immunotherapy
              </a>
            </td>
          <td>
            Fei Wang, Run Huang, Hongyi Ling, Yuhan Bai, Chen Yang, Guozhen Zhao, Xin Wu, Wenqian Cao, Yue Lu, Yining Zhang, Deng-Feng Lu, Youjia Qiu, Juyi Zhang, Bixi Gao, Chao Ma, Hanhan Dang, Yulian Zhang, Yanbo Yang, Ting Sun, Zhouqing Chen, Zhong Wang
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Understanding how prostate cancers evolve toward treatment resistance requires experimental systems that both recreate the phenotypic diversity of the disease and resolve lineage dynamics at high resolution. We developed complementary platforms to meet these challenges. The Prostate Mouse Platform (ProMPt) is a genetically scalable organoid- and allograft-based system encompassing over 150 combinatorial genotypes, enabling reconstruction of the principal phenotypic states of prostate cancer, from androgen-responsive tumors to castration-resistant double-negative, hybrid/mixed, and neuroendocrine-like variants. Through multi-omic and functional analyses, ProMPt models capture the context-specific vulnerabilities that arise during adaptation to androgen deprivation or growth-factor loss. To map and track this evolving phenoscape, we designed XCODE, a universal, multiplatform barcoding technology that enables clonal tracing across sequencing-, cytometry-, and imaging-based modalities (CyTOF, Phenocycler, Xenium). XCODE integrates lineage information across bulk, single-cell, and spatial layers, linking histology, clonal dynamics, phenotype, and microenvironmental context. Together, ProMPt and XCODE provide a powerful preclinical framework to reconstruct, resolve, and perturb tumor evolution within a controlled yet clinically relevant landscape. This integrated approach reveals how defined genetic combinations reshape lineage plasticity, signaling networks, and therapeutic responses, exposing new intervention points to prevent or redirect adaptive evolution in advanced prostate cancer.



 Marco Bezzi. From reconstruction to resolution: platform approaches to engineer the prostate cancer phenoscape and map its evolution [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A002.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3245085c50a80831901ee082ba85bf39b096b337" target='_blank'>
              Abstract A002: From reconstruction to resolution: platform approaches to engineer the prostate cancer phenoscape and map its evolution
              </a>
            </td>
          <td>
            Marco Bezzi
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Although well-differentiated thyroid carcinoma (WDTC) is characterized by a robust treatment response, aggressive subtypes, such as anaplastic thyroid carcinoma (ATC), remain highly lethal. To understand thyroid cancer evolution in both children and adults, we analyzed single-cell transcriptomes of 423,733 cells from 81 samples and spatially resolved key tumor and microenvironment populations across 28 tumors with spatial transcriptomics, including rare and unique composite WDTC/ATC tumors and pediatric diffuse sclerosing thyroid carcinomas. Additionally, we identified gene signatures of stromal cell populations in 5 large thyroid cancer bulk RNA-sequencing cohorts. Through this multi-institutional effort, we defined a population of POSTN+ myofibroblast cancer-associated fibroblasts (myCAFs) that are intimately associated with invasive tumor cells and correlate with poor prognosis, lymph node metastasis, and disease progression in thyroid carcinoma. We also revealed a population of inflammatory CAFs that are distant to tumor cells and are found in the inflammatory stromal microenvironment of autoimmune thyroiditis. Together, our study provides spatial profiling of thyroid cancer evolution in samples with mixed WDTC/ATC histopathology and identifies a prognostic myCAF subtype with potential clinical utility in predicting aggressive disease in both children and adults.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9674219bbd66cbdd5e33b2f92b59cb54d20e94cd" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations.
              </a>
            </td>
          <td>
            M. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey P. Caroland, Megan L. Tigue, H. Hartmann, Jean-Nicolas Gallant, Courtney J. Phifer, Andres A Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H. Belcher, K. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2026-01-09</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27e17b141d07dfd3357f3ee906c1f3a2792dd3c" target='_blank'>
              SigDyn: single-cell mutational signature dynamics
              </a>
            </td>
          <td>
            Sara B. Costa, Joana P. Gonçalves
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ee6608293723ba24e369164c3802a52bef8a361" target='_blank'>
              Redefining the topology of the human bone marrow using augmented spatial transcriptomic analysis
              </a>
            </td>
          <td>
            Rosalin Cooper, Emily Thomas, Muhammad Dawood, H. Ryou, Anna M. Sozanska, C. Pescia, Oliver McCallion, Muskaan Gupta, Renuka Teague, J. Hester, F. Issa, Dan J Woodcock, B. Psaila, Adam J. Mead, Jens Rittscher, D. Royston
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18e77f40b4649c94b742354f1776a0273d7ae6d5" target='_blank'>
              A Hallmark-Integrated, Agent-Based Framework for Intratumor Heterogeneity in Melanoma Evolution
              </a>
            </td>
          <td>
            Khola Jamshad, Trachette L. Jackson
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67c0bdeec747ad1c41e7a933728f867f5f704e68" target='_blank'>
              Spatial pharmaco-multiomics reveals drug distribution, metabolic niches, and spatially constrained resistance in medulloblastoma
              </a>
            </td>
          <td>
            Tuan Vo, , Aitor Benedicto, Yibi Chen, Andrew Causer, Xiao Tan, Eun Ju Kim, Hani Vu, Albert Xiong, Mark Hodson, Yak Deng, Ana Maria Romero Jimenez, Ning Liu, Brett Hamilton, Thomas Robertson, Laura A. Genovesi, Brandon J. Wainwright, Jenny Fung, John Lee, Quan Nguyen
          </td>
          <td>2026-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb72dd83254c43d1127b19ea69b553e5fd51c4d" target='_blank'>
              TMPO promotes cellular dissemination and metastasis in circulating tumor cells
              </a>
            </td>
          <td>
            A. Giacobbe, Aleksandar Z Obradovic, Jinqiu Lu, Soonbum Park, Carlos Pedraz-Valdunciel, G. Fanelli, Aunika Zheng, Jaime Y Kim, Maya Stella Dixon, Jung Seung Nam, Florencia Picech, Caroline J. Laplaca, R. Virk, Matteo Di Bernardo, Alexander Chui, J. M. Arriaga, Stephanie N Afari, Francisca Nunes de Almeida, Min Zou, Helen Garcia, Brian D. Robinson, Hongshan Guo, S. Maheswaran, D. Haber, D. Miyamoto, D. Nanus, S. Tagawa, Tian Zheng, Massimo Loda, I. I. C. Chio, Michael M. Shen, P. Giannakakou, Andrea Califano, Peter A. Sims, Cory Abate-Shen
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71c46aa519f5755b995d1d19efbf9695d85981f7" target='_blank'>
              Molecular Profiling and High-Content Drug Screening of Metastatic Colorectal Cancer Organoids Reveal Evolutionary Mechanisms of Pan-KRAS Inhibitor Resistance
              </a>
            </td>
          <td>
            Gizem Damla Yalcin, Kübra Yılmaz, Raffaella Klima, O. Kurtulan, Duriye Ozer Turkay, O. Aktepe, A. B. Doğrul, Cenk Sokmensuer, V. Oter, E. B. Bostancı, Javier Fernandez-Mateos, Suayib Yalcin, Luca Braga, Ahmet Acar
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), a highly lethal disease often developing in the context of chronic liver disease, is marked by high relapse rates and low 5‐year survival. To investigate the multicellular ecosystem and molecular features of hepatocarcinogenesis and progression, a comprehensive analysis integrating single‐cell and spatial transcriptomics was conducted. Significant alterations were observed in various cell types, notably an increase in liver parenchymal and endothelial cells, which play critical roles in tissue repair and angiogenesis. Scissor+ cells, predominantly hepatocytes, were identified as potentially evolving into malignant cells with poor prognosis and enhanced immune evasion capabilities. Key genes associated with Scissor+ cells exhibited potential as cancer prognostic markers. Additionally, the disruption of cellular communication networks during malignant progression revealed the evolution of hepatocytes into cancerous phenotypes. The enhanced signaling pathways, such as MDK‐SDC4 and COL4A‐SDC, were identified as pivotal in driving the malignant transformation of hepatocytes. This transformative process, wherein Scissor+ cells acquire malignant features, highlights novel mechanisms of cancer progression. These findings provide deeper insights into hepatocarcinogenesis and offer potential avenues for targeted and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea05c9d911e5acfb02ab472396d64afacfb09aa" target='_blank'>
              Spatiotemporal Evolution of Malignant Hepatocyte Clones Unveils Immune Evasion Features and Therapeutic Vulnerabilities in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Shizheng Xiong, Dekang Ren, Jiaming Jin, Linjie Zhao, Chengjun Gong, Jiarui Liu, Wanjie Zheng, Zhiheng He, Haochuan Guo, Li Guo, Tingming Liang
          </td>
          <td>2026-01-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Neoadjuvant PD-1/PD-L1 blockade yields robust efficacy in advanced cutaneous squamous cell carcinoma (cSCC), yet many patients fail to achieve a complete or major pathologic response. The reasons why some patients experience response but others do not are unclear. Methods We profiled cSCC specimens before, after 1 dose, and after 3–4 doses of PD-1/PD-L1 blockade to uncover resistance mechanisms and predict therapeutic response. In total, 27 patients across three cohorts, including two phase II trials, were studied. We created 1.7 mm tissue-core microarrays and performed single-cell spatial transcriptomics, including spatial clustering, gene-set enrichment, and spatial correlation analyses. Results After profiling all samples, six distinct spatial niches emerged, each differentially enriched in responders versus non-responders. A high antigen presentation niche, B/plasma cell enriched niche, and inflammatory keratinocyte niche were more frequent in responders, whereas proliferative keratinocyte, low antigen presentation myeloid, and fibroblast-rich epithelial–mesenchymal transition niches prevailed in non-responders. Notably, spatial niche profiling on pretreatment samples outperformed PD-L1 status in predicting pathologic response. Each niche displayed unique gene coexpression modules, suggesting niche-specific resistance mechanisms. Individual tumor analyses revealed varied immune evasion strategies, including defective interferon-induced antigen presentation, immunosuppressive myeloid environments, and epithelial–mesenchymal transition. Conclusions Our single-cell spatial transcriptomic approach identifies six spatial niches that predict immunotherapy response better than PD-L1 status using only 1.7 mm tissue cores and may inform the development of biomarkers. Our results further underscore the heterogeneity of resistance mechanisms among cSCC patients, highlighting the need for tailored therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf68ddba765459abfda326b7a403a032b5df3da" target='_blank'>
              Single-cell spatial transcriptomics uncovers niches that govern response to PD-1/PD-L1 blockade in cutaneous squamous cell carcinoma
              </a>
            </td>
          <td>
            Maxwell Y Lee, Alistaire R. Sherman, Luis Martinez Ramirez, Mobeen Rahman, Sabrina Zdravkovic, June Ho Shin, Ivan Stepanek, A. D. Colevas, John B. Sunwoo, V. Divi
          </td>
          <td>2026-01-01</td>
          <td>Journal for ImmunoTherapy of Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Lung cancer remains a significant global health challenge, with metastatic progression being the leading driver of mortality. Organoid technology provides a tractable, physiologically relevant platform to model key aspects of lung cancer metastasis in vitro. The present review summarized methodologies for constructing and interrogating these models, covering tissue sources, culture modalities, gene editing and in vivo transplantation; applications in studying metastatic mechanisms, drug screening and capturing intra- and intertumoral heterogeneity are also highlighted. Persistent challenges include standardizing derivation and culture conditions, improving preservation of tumor-microenvironmental interactions, expanding immune-competent and vascularized models, and addressing scalability, cost, and regulatory and ethical considerations for clinical translation. Future directions include integrating multi-omics approaches and spatial profiling, leveraging artificial intelligence for image and response analytics, advancing immune-organoid models and establishing shared standards, reference materials and reporting guidelines to enhance reproducibility and clinical impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9831c4c96b7e8c9d326d8d56192940951debd787" target='_blank'>
              Recapitulating lung cancer metastasis in vitro: Advances in organoid models and challenges in clinical translation (Review)
              </a>
            </td>
          <td>
            Jian Jiang, Guomin Dong, Zhiyun Zhang, Xiaoyuan Lei, Fanshu Huang, Suoni Li, Jiequn Ma, Jie Bai, Qian Ge, Hui Wei, Zheng Zhao
          </td>
          <td>2026-01-20</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb4486dfc086ad91b487a1c5269e719293310a3" target='_blank'>
              Machine perfusion and single-cell spatial transcriptome mapping identifies novel immune escape mechanisms in colorectal cancer liver metastasis
              </a>
            </td>
          <td>
            Peter Kok-Ting Wan, Ranchu Cheng, David Johnson, Carl Lee, Shihong Wu, Areeb Mian, Ahmet Hazini, Sorayya Moradi, Kate Friesen, Flurin Caviezel, Syed Hussain Abbas, Hatem Sadik, A. Lakha, Keaton Jones, Girishkumar Kumaran, M. Bottomley, Rob Jones, Mark C. Coles, R. Bashford-Rogers, Constantin C. Coussios, Len Seymour, Robert Carlisle, Kerry Fisher, Alex Gordon-Weeks
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a19bfda47de17fb8ef4014da55fa6d5d86101b2f" target='_blank'>
              Addressing complex autofluorescence signatures in solid tissue samples to enhance full spectrum flow cytometry of non-immune cells
              </a>
            </td>
          <td>
            Christina Gkantsinikoudi, Manuela Terranova-Barberio, Neil P Dufton
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faf55c0c7d18000c8ba092357d0dd95271e7c35c" target='_blank'>
              Spatial dissection of ADC/RPT targets defines therapeutic opportunities in rhabdoid tumors
              </a>
            </td>
          <td>
            N. Reitsam, V. Fincke, Maria Daniela Hernandez Ramirez, M. Mucha, Eva Sipos, Lisa Siebenhüter, J. Enke, Maurice Loßner, C. Vokuhl, C. Lapa, M. Hasselblatt, Michael C. Frühwald, B. Märkl, Pascal D. Johann
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Rationale: Esophageal squamous cell carcinoma (ESCC) is a highly aggressive malignancy. The metastasis and poor prognosis of ESCC are closely associated with tumor microenvironment (TME) heterogeneity, which is driven by epithelial-mesenchymal transition (EMT). Clinically, how to diagnose and target EMT progression remains a key challenge for ESCC. Methods: Integration of pathological images and bulk RNA sequencing profiles identified a high-risk subtype exhibiting EMT enrichment and immunosuppression. Single-cell and spatial transcriptomics revealed EMT macrostates and their spatial distribution. The role of CACNA1C in programming malignant phenotype was tested in vitro. A pathological image-based deep learning model successfully predicted the spatial expression distribution of CACNA1C, indicating possible clinical utility. Results: EMT progression comprised three macrostates: the early state (high epithelial and metastatic potential), the stable state (hybrid E/M phenotype and high stemness), and the late state (high mesenchymal and invasive propensity). ITGA3 and ITGB4 antagonistically regulate malignant phenotype in the early state. Notably, suppression of CACNA1C induced transdifferentiation from stable/late-state cells to normal epithelium-like cells. Conclusions: This study provides novel insights into the EMT mechanism in ESCC, proposes an intervention strategy, and emphasizes the promising clinical application of pathological images in EMT assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2bd67040364633d082b64fbfe1d8f04e033d05" target='_blank'>
              Pathogenomic analysis reveals clinically relevant epithelial-mesenchymal plasticity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Ruzhen Chen, Chenyi Xie, Ziyu Ning, Meng Yang, Zezhuo Su, Jiahui Chen, Kunheng Du, Yihuai Hu, Chu Han, Shaojun Zhang, Qingling Zhang, Meng Liu, Zaiyi Liu
          </td>
          <td>2026-01-14</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tumor-associated stromal cells (TASCs) are key architects of the tumor microenvironment (TME), playing a vital role in tumor development, metastasis, and therapeutic response. Their spatiotemporal heterogeneity, characterized by dynamic phenotypic plasticity, diverse cellular subtypes, and distinct spatial distributions, offers profound insights into tumor behavior and paves the way for innovative therapy development. In particular, stromal–immune interactions reveal the powerful capacity of TASCs to shape the immune landscape, highlighting their potential as targets in immunotherapy. Despite growing evidence in functional diversity, precise mechanisms underlying the temporal evolution and spatial organization of TASCs remain elusive, impeding clinical translation. This review delved into the molecular signatures and functional states of TASCs, emphasizing their roles in tumor dynamics and therapeutic resistance. We also discussed innovative strategies targeting the plasticity of TASCs to reverse immune evasion and potentiate immune-mediated tumor eradication. Future studies should prioritize identifying spatially resolved and mechanically defined biomarkers with multi-omics and machine learning approaches, enabling a comprehensive understanding of TASCs to bridge the gap from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc16bf824dc473946d5d3958736301da12edf085" target='_blank'>
              The Spatiotemporal Heterogeneity of Tumor-Associated Stromal Cells: Reprogramming Plasticity to Unlock Precision Cancer Immunotherapy
              </a>
            </td>
          <td>
            Yingying Lv, Tingfei Duan, Jinling Song, Shutong Liu, Zhaokai Zhou, Yuhao Ba, Siyuan Weng, Anning Zuo, Hui-Xian Xu, Peng Luo, Quan Cheng, Chuhan Zhang, Jingyuan Ning, Yukang Chen, Yuyuan Zhang, Zaoqu Liu, Xinwei Han
          </td>
          <td>2025-12-12</td>
          <td>Cancer Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the leading causes of cancer-related morbidity and mortality worldwide. Tumor sidedness, distinguishing left- (LCRC) and right-sided (RCRC) cancers, has emerged as a critical clinical determinant, influencing patient prognosis and therapeutic response. However, the cellular and molecular mechanisms underlying these differences remain poorly understood. Recent advances in single-cell RNA sequencing (sc-seq) provide high-resolution insights into CRC heterogeneity, revealing distinct tumor, immune, and stromal cell populations and their context-specific interactions. In this review, we synthesize sc-seq studies that dissect the molecular programs driving progression, therapy resistance, and metastasis in CRC. We highlight malignant subclusters characterized by metabolic reprogramming and spatially organized oncogenic signaling; specialized immune cell states, including macrophage subsets, exhausted T cells, and mast cells, that shape tumor immunity; and stromal elements such as cancer-associated fibroblasts and endothelial tip cells that remodel the extracellular matrix, promote angiogenesis, and foster immune evasion. Importantly, sc-seq demonstrates that LCRC and RCRC represent distinct multicellular ecosystems with differential immune recruitment and stromal signaling, underscoring the need for sidedness-informed therapeutic strategies. We propose that future interventions should target cell-cell communication networks and spatially defined tumor–microenvironment interactions to overcome heterogeneity and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03b9e3bca233f1317bcd2e2e43fb69a5fedfb984" target='_blank'>
              Sidedness matters: single-cell perspectives on left- and right-sided colorectal cancer
              </a>
            </td>
          <td>
            M. Yanova, Diana Maltseva, Alexander Tonevitsky
          </td>
          <td>2025-12-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3083560b7513f7f17aefe85af32208b0613ecab1" target='_blank'>
              Cellular neighborhoods in cancer
              </a>
            </td>
          <td>
            Lichun Ma, Barbara Xiong, Meng Liu, Kai Tan
          </td>
          <td>2026-01-01</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d29acb3217c08dff3be7e7d47281f950232e5a4" target='_blank'>
              Identification of an epigenetically and phenotypically distinct peritumoral glioblastoma cell population linked to inferior patient outcome
              </a>
            </td>
          <td>
            I. Neves, Xi Lu, Veera Jokinen, Francesco Latini, N. Maturi, A. Sundström, Yonglong Dang, Irem Uppman, Tobias Bergström, Pengwei Xing, M. Ryttlefors, Xingqi Chen, F. Swartling, L. Uhrbom
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The omentum, a visceral adipose tissue with critical metabolic, immunological, and stem cell functions is the preferred site for ovarian cancer metastasis. However, its role in maintaining homeostasis and its responses to metastatic colonization remain incompletely understood. Using single-cell transcriptomics, we profile different anatomical regions of the omentum in patients with benign conditions and metastasis. We catalog the benign omentum and observe a stable cell type composition and a preserved stem and progenitor niche. Upon metastatic colonization, we report on increased immune heterogeneity and a concomitant reduction of mesothelial and progenitor cells. The lesser omentum, which is not routinely removed during surgical debulking, is identified as a premetastatic niche characterized by neutrophil infiltration, extracellular trap formation, and the presence of micrometastases. At established metastatic sites, resident cells exhibit cancer-associated phenotypes with regulatory, anti-adipogenic, and immunosuppressive properties. Cellular reprogramming across the omentum is associated with signaling profiles of tumor cells, suggesting potential influences on both proximal and distal tissue regions. This cell atlas illuminates the cellular and molecular determinants of organ homeostasis and reveals a high degree of plasticity and cellular reprogramming promoted by cancer colonization. The omentum is the main metastatic site for ovarian cancer. Here, the authors present a human cell atlas of both benign and omentum metastatic ovarian cancer, providing insights into established metastatic sites but also of the omentum pre-metastatic niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d421861e9c6ef5f5ca4fb4ee26c92b15a98c09c9" target='_blank'>
              Ovarian cancer metastasis to the human omentum disrupts organ homeostasis and induces fundamental tissue reprogramming
              </a>
            </td>
          <td>
            Ulrike Lischetti, Ching-Yeu Liang, Matteo Carrara, R. Coelho, Antoine Hanns, Flavio C. Lombardo, Arthur Dondi, Sandra Goetze, M. Hensler, C. Kurzeder, Céline Montavon, J. Fucikova, Franziska Singer, N. Beerenwinkel, C. Beisel, V. Heinzelmann-Schwarz, Francis Jacob
          </td>
          <td>2025-12-16</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>65</td>
        </tr>

        <tr id="Single-cell sequencing (scRNA-seq) has emerged as a pivotal technology for deciphering the complex cellular heterogeneity and tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC), positioning it as a critical tool for informing novel therapeutic strategies. This review explores how scRNA-seq reveals diverse cellular subpopulations and their functional roles within the PDAC TME, including malignant epithelial cells with transitional phenotypes, heterogeneous cancer-associated fibroblasts (CAFs), functionally distinct immune cells such as tumor-associated neutrophils (TANs) and macrophages (TAMs), and actively participating neural components like Schwann cells. These cellular constituents form specialized functional units that drive tumor progression, immune evasion, neural invasion, and therapy resistance through metabolic reprogramming, immunosuppressive signaling, and cellular plasticity. The review further examines technological advances in single-cell sequencing from 2023 to 2025, focusing on sample preprocessing innovations, multi-omics integration (combining transcriptomics with epigenomics and proteomics), spatial resolution enhancements, and customized computational tools that address PDAC-specific challenges. Clinically, single-cell sequencing enables precise cellular subtyping, identification of novel biomarkers, and development of personalized therapeutic approaches, including combination therapies targeting specific cellular subpopulations and their interactions. Despite these advances, significant challenges remain in standardizing clinical applications such as liquid biopsy for early detection and tumor microenvironment assessment for diagnostic staging, validating biomarkers like CLIC4, GAS2L1, Cytokeratins, Vimentin and N-cadherin in circulating tumor cells, and comprehensively integrating multi-omics data. Future research focusing on both technology refinement and biological validation will be essential for translating single-cell insights into improved diagnostic and therapeutic outcomes for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9576dbb6fe77bb5f04f2ad67b474589557936" target='_blank'>
              Single-Cell Sequencing Unravels Pancreatic Cancer: Novel Technologies Reveal Novel Aspects of Cellular Heterogeneity and Inform Therapeutic Strategies
              </a>
            </td>
          <td>
            Keran Chen, Zeyu Chen, Jinai Wang, Mo Zhou, Yun Liu, Bin Xu, Zhi-Jing Yu, Yiming Li, Guanhu Yang, Tiancheng Xu
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Gastroenteropancreatic neuroendocrine tumors (GEP-NETs) are clinically heterogeneous malignancies whose biology and microenvironmental organization remain poorly understood. Here, we integrated single-nucleus multiomic (snRNA-seq and snATAC-seq) and spatial transcriptomic profiling across 38 well-differentiated pancreatic (pNET) and small-intestinal (siNET) tumors to define conserved malignant programs, their regulatory circuits, and spatial niches. We observed two conserved malignant cell programs spanning a continuous transcriptional spectrum: a neuronal-like program (si-cNMF1/p-cNMF1), and a secretory neuroendocrine program (si-cNMF2/p-cNMF2). Matched chromatin accessibility profiles uncovered distinct, tissue-specific regulatory networks, including MAX::MYC and MITF transcription factor binding motifs in siNETs versus ISL1 and TFAP4 in pNETs, indicating organ-specific epigenetic control. Spatial transcriptomic analyses revealed that si/p-cNMF1-high regions localized to high cell density, immune-rich tumor areas, whereas si/p-cNMF2-high regions occupied stromal and vascularized niches and co-occured with fibroblast and endothelial compartments enriched for TGFB1-ITGB1, VEGFA-FLT1, and LAMA2-ITGA1 signaling. Across both tumor types, the cNMF2 program was enriched in metastatic lesions and was enrichedfor pro-fibrotic and pro-angiogenic gene signatures. Thus, GEP-NETs are organized along a conserved neuronal-to-secretory axis defined by distinct epigenetic programs and spatially coupled to specific microenvironmental niches. This framework unifies NET heterogeneity across organ sites and identifies pathway-specific, microenvironment-linked vulnerabilities for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f5c183623da01cf434288165ef5499fec974dff" target='_blank'>
              Conserved Neuronal-like and Secretory Programs Define the Spatial Architecture of Gastroenteropancreatic Neuroendocrine Tumors
              </a>
            </td>
          <td>
            Julie Karam, Samantha E. Hoffman, Amanda E. Garza, Dan Y Gui, Hannah I. Hoffman, Breanna M. Titchen, Yutaro Tanaka, Erica M Pimenta, Theodora Pappa, Laura Valderrábano, Kevin Bi, R. Gillani, Lauren Brais, Erin Shannon, Jason L. Hornick, Jihye Park, Jennifer Chan, E. V. Van Allen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the leading causes of cancer-related mortality worldwide. Compared with traditional two-dimensional (2D) models, patient-derived CRC organoids more faithfully preserve the genomic, transcriptomic, and architectural features of primary tumors, making them a powerful intermediate platform bridging basic discovery and clinical translation. Over the past several years, organoid systems have rapidly expanded beyond conventional epithelial-only cultures toward increasingly complex architectures, including immune-organoid co-culture models and mini-colon systems that enable long-term, spatially resolved tracking of tumor evolution. These advanced platforms, combined with high-throughput technologies and clustered regularly interspaced short palindromic repeats (CRISPR)-based functional genomics, have substantially enhanced our ability to dissect CRC mechanisms, identify therapeutic vulnerabilities, and evaluate drug responses in a physiologically relevant context. However, current models still face critical limitations, such as the lack of systemic physiology (e.g., gut–liver or gut–brain axes), limited standardization across platforms, and the need for large-scale, prospective clinical validation. These gaps highlight an urgent need for next-generation platforms and computational frameworks. The development of high-throughput multi-omics, CRISPR-based perturbation, drug screening technologies, and artificial intelligence-driven predictive approaches will offer a promising avenue to address these challenges, accelerating mechanistic studies of CRC, enabling personalized therapy, and facilitating clinical translation. In this perspective, we propose a roadmap for CRC organoid research centered on two major technical pillars: advanced organoid platforms, including immune co-culture and mini-colon systems, and mechanistic investigations leveraging multi-omics and CRISPR-based functional genomics. We then discuss translational applications, such as high-throughput drug screening, and highlight emerging computational and translational strategies that may support future clinical validation and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ed98ccf8eb7ee4178076cc0e4368c14e7e399d9" target='_blank'>
              An Integrative Roadmap for Advancing Colorectal Cancer Organoid
              </a>
            </td>
          <td>
            Youqing Zhu, Ke He, Zhi Shi
          </td>
          <td>2026-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immune dysfunction in cancer is enacted by multiple programs, including tumor cell-intrinsic responses to distinct immune subpopulations. A subset of these immune evasion programs can be systematically recapitulated through direct tumor-immune interactions in vitro. Here, we present an integrated, high-throughput single-cell CRISPR screening framework focused on the protein kinome for mapping the tumor-intrinsic regulation of T cell-driven immune pressure in glioblastoma (GBM). We combine pooled CRISPR interference and activation (CRISPRi/a) with immune-matched NY-ESO-1 antigen-specific allogeneic GBM-T cell co-culture and massively multiplexed single-cell transcriptomics to systematically quantify how genetic perturbation reshapes baseline tumor state and adaptive responses across graded effector-to-target ratios. We further leverage deep generative models for analyzing pooled CRISPR screens to decipher the effects of genetic perturbations on the mechanisms of tumor resistance. This framework resolves distinct modules of immune evasion and survival, including the regulation of the antigen-presentation machinery, interferon/NF-κB signaling, oxidative stress resilience, and checkpoint/cytokine programs, while identifying perturbations that reroute the continuous tumor transcriptional trajectory induced by T cell engagement. A secondary chemical screen in patient-derived GBM cultures identified putative kinase targets of immune evasion phenotypes (e.g., EPHA2 and PDGFRA), whose inhibition leads to the blockade of evasive programs and enhances T cell-mediated GBM killing. Together, this workflow provides a scalable blueprint for comprehensive charting of the genetic control of tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e76a058bed6b1c1e4d91cd692a05e66fab6beee" target='_blank'>
              Mapping kinase-dependent tumor immune adaptation with multiplexed single-cell CRISPR screens
              </a>
            </td>
          <td>
            Lingting Shi, Ross M. Giglio, Qingyuan Cai, Mathini Vaikunthan, Justin Hong, Abdullah Naqvi, Marina Milea, Hannah Khanshali, Anna Schoonen, Nicholas Hou, Jonathan Guo, Melanie Fraidenburg, Xumin Shen, Seth Malinowski, K. L. Ligon, Raúl Rabadán, Elham Azizi, José L. McFaline-Figueroa
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a05de91b68f189c5a7a8fad28c43d535bcdc834" target='_blank'>
              A spatial atlas of colorectal cancer reveals the influence of stromal niches on tumour differentiation
              </a>
            </td>
          <td>
            Andrew D. Pattison, Rebekah M. Engel, Wing Hei Chan, Spencer Greatorex, David Nickless, Liam Skinner, Camilla Cohen, Julie Hickey, Christine Georges, Anne L. Fletcher, P. McMurrick, Thierry Jardé, Lochlan Fennell, Helen E Abud
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Lineage tracing at single-cell resolution is vital for mapping cell fate decisions, yet synthetic barcoding faces limitations in precision, diversity, and toxicity-especially in human pluripotent stem cells (hPSCs). Here, we repurpose naturally occurring somatic mutations in mitochondrial transcripts from single-cell RNA sequencing as endogenous genetic barcodes. By enriching mitochondrial reads and applying a robust computational pipeline, we identified clonally informative variants to trace hematopoietic lineage emergence from hPSCs during early embryogenesis. Integrating mitochondrial barcoding with synthetic lineage tracing, we modeled embryonic tissue development and reconstructed the transcriptional logic and regulatory networks driving fate specification using a dynamical systems model. Extending this approach to spatial transcriptomics, we mapped the clonal architecture of human embryonic organoids, revealing spatial zonation orchestrated by NOTCH-mediated crosstalk between stromal cells and hematopoietic progenitors. This multimodal strategy links clonal dynamics with niche-driven fate decisions, offering a scalable, non-invasive method to dissect tissue organization in development and disease. Together, our work establishes a scalable, non-invasive multimodal framework that leverages endogenous mitochondrial DNA variants to reconstruct high-resolution spatiotemporal clonal dynamics and decode niche-driven fate decisions in a human stem cell-derived model. This approach provides a powerful strategy for dissecting tissue self-organization in development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44d3ba922d5d14a9018eca6710873d3d64119d9" target='_blank'>
              Single-Cell Mitochondrial Lineage Tracing Decodes Fate Decision and Spatial Clonal Architecture in Human Hematopoietic Organoids.
              </a>
            </td>
          <td>
            Yan Xue, J. Su, Yiming Chao, Lu Liu, Xinyi Lin, Yang Xiang, Mun Kay Ho, Zezhuo Su, Junyi Chen, Zhuojuan Luo, Chengqi Lin, Ruibang Luo, Theo Aurich, Jianfeng Wu, Kelvin Sin Chi Cheung, Yuanhua Huang, Joshua W. K. Ho, Ryohichi Sugimura
          </td>
          <td>2026-01-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 The tumor microenvironment (TME) plays a pivotal role in shaping cancer progression, immune evasion, and therapeutic response. However, a systematic understanding of immune and stromal cell heterogeneity across cancer types—and their associations with clinical phenotypes—remains lacking due to limited single‐cell data and the associated clinical metadata. Here, we present a comprehensive pan‐cancer TME cell atlas to date, quantifying the composition of 102 immune and stromal subtypes across 30 cancer types and encompassing over 11,000 bulk RNA‐seq samples from TCGA. To enable this unprecedented resolution, we developed DECEPTICONx, a flexible computational framework that integrates single‐cell RNA‐seq data by constructing high‐resolution expression templates, which are then paired with context‐optimized deconvolution strategies, dynamically selecting the best‐matched template‐deconvolution combinations for each task. By leveraging 535,665 single cells spanning T cells, B cells, fibroblasts, macrophages, and NK cells, we constructed high‐resolution expression templates and systematically deconvolved bulk samples. The resulting atlas revealed robust and novel associations between specific cell subtypes and patient prognosis, immunotherapy response, and inter‐cellular coordination within the TME. In summary, the DECEPTICONx‐based atlas not only advances our understanding of tumor‐immune ecology but also offers a foundation for biomarker discovery and therapeutic development in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaaef7a98ed82e86ab1fb8032603b073ca5c5bb6" target='_blank'>
              A Pan‐Cancer Microenvironment Atlas Constructed by DECEPTICONx
              </a>
            </td>
          <td>
            Yida Gu, Fulan Deng, Lianchong Gao, Zhiyu Chen, Jianping Chen, Jiale Wu, Zengqiang Xia, Zhijian Li, Fangkaikai Cheng, Jie Chen, HoiYee Tong, Wantao Chen, Xiaoqing Pan, Xin Zou, Jie Hao
          </td>
          <td>2026-01-17</td>
          <td>iMetaMed</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is considered immunologically “cold,” however, we have previously identified an immunogenic subset of high-risk localized PCa with T-cell tumor infiltration within antigen-presenting-cell (APC) niches. Here, we used high-dimensional single-cell spatial proteomics to dissect tumor-infiltrating-lymphocyte in (TIL)-high (H), TIL-low (L), and TIL-excluded (X) PCas, to understand the tumor-specific and conserved mechanisms of immune dysregulation for targeted therapeutics.



 Treatment-naive prostatectomy specimens of all three tumor immune microenvironment (TIME) phenotypes, were assembled into tissue microarrays representing within-tumor and adjacent foci. TIL-X was exclusively observed in cribriform pattern 4 cases. A 40-plex antibody panel made of phenotyping and functional markers (e.g AR) was applied to the TMAs, enabling single-cell annotation of key immune cell types (e.g CD8+ T cells) and assessment of their functional states within distinct spatial microenvironments.



 In TIL-X PCa, immunosuppressive macrophages and DCs were enriched in the tumor nest whereas T cells accumulated at the periphery with minimal tumor infiltration. Tumor adenosine 2A receptor (A2AR) expression was uniquely enriched within tumor, suggesting a metabolically suppressive microenvironment that reinforces immune exclusion. TIL-H tumors had comparable T cell proportions inside and outside the tumor, accompanied by the enrichment of myeloid cells within the tumor. CD8+ T cells expressing high levels of AR and B7H3 frequently co-localized with tumor and APCs, forming APC niches and tertiary lymphoid-like structures. CD8+ARhi and CD8+B7H3hi subsets with elevated exhaustion scores preferentially infiltrated tumor, whereas CD8+ARlo and CD8+B7H3lo subsets with moderate exhaustion levels remained peripheral; defining TIL-H PCa as tumors harboring antigen-experienced but exhausted CD8 T cells in a TIME that supports immune cell engagement but restrains anti-tumor activity. TIL-L PCa had sparse T cell infiltration and enrichment of myeloid cells with moderate AR expression within tumor, suggesting a role of androgen signaling and myeloid cells in shaping the TIME, marked by high PD-1+CD8+ cells and PD-L1+ tumor cells. Thus, TIL-L PCa shows low immune recruitment, yet strong tumor-mediated inhibitory signaling that constrains anti-tumor immunity.



 TILs in TIL-H PCa are phenotypically, functionally, and spatially distinct from those in TIL-L or TIL-X tumors, reflecting tumor-intrinsic immune programs. High-dimensional spatial proteomics reveal differential molecular modules and signaling pathways that define unique and potentially targetable mechanisms of immunosuppression across the three TIME phenotypes; additional spatial transcriptomics validation and dissection experiments are underway. These findings demonstrate that the PCa TIME is diverse yet spatially organized, and that comprehensive understanding of baseline spatial and transcriptional immune states may guide personalization of immunotherapy strategies in PCa.



 Precious Cramer, Yang Wang, Wenrui Wu, Huaying Qiu, Lindsay Parmelee, Berkay Simsek, Gabriel Roberti. De Oliveira, Seifeldin Awad, Yao Yu Yeo, Stephanie Pei Tung. Yiu, Hendrik Michel, Sabina Signoretti, Yue Sun, Steven Balk, Sizun Jiang, David Einstein Einstein. High-dimensional spatial profiling identifies distinct and targetable tumor-infiltrating lymphocyte phenotypes in localized prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d786dbf212bd5a71f25f7e11f429f8dba252ddc" target='_blank'>
              Abstract PR010: High-dimensional spatial profiling identifies distinct and targetable tumor-infiltrating lymphocyte phenotypes in localized prostate cancer
              </a>
            </td>
          <td>
            Precious Cramer, Yang Wang, Wenrui Wu, Huaying Qiu, Lindsay Parmelee, Berkay Simsek, Gabriel Roberti. De Oliveira, Seifeldin Awad, Yao Yu Yeo, Stephanie Pei Tung. Yiu, Hendrik A. Michel, Sabina Signoretti, Yue Sun, Steven P. Balk, Sizun Jiang, D. Einstein
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Prostate cancer cell invasion into the surrounding muscle capsule, known as extracapsular extension (ECE), is associated with a significantly worse prognosis (5-year survival reduction: 98% to 31%). Using a live ECE mouse xenograft model with DU145 tumor cells, our group previously demonstrated that functional heterogeneity among adhesion subtypes within invasive clusters is essential for muscle invasion. Bulk RNA sequencing from three spatial compartments (pre-invasive, invasive, and post-invasive) revealed distinct gene expression profiles, with hypoxia-related and cell–cell/cell–ECM interaction genes prominently dysregulated in muscle-invasive disease. In this study, we sought to define the single-cell transcriptional heterogeneity across the three ECE compartments and compare these patterns to a human patient-derived xenograft (PDX) model. Spatial transcriptomic analysis (STA) was performed using the Bruker CosMx platform with a 6K gene panel. Data was preprocessed (quality control and dimensional reduction) and analyzed in R 4.4.0 using Seurat v5. Notable downstream analyses included unsupervised Louvain clustering, trajectory inference with Monocle 3, and cluster-specific differential gene expression. In the ECE model, STA resolved at least 11 distinct tumor cell identity clusters previously undetectable by bulk methods and revealed a clear spatial demarcation between pre-invasive and invasive populations crossing the muscle barrier. Trajectory analysis identified a pre-invasive population branching into two invasive fates with unique expression profiles. Dispersion scores, calculated using principal component embeddings, exhibited progressively increasing intracluster heterogeneity along one branch. Strong expression enrichment of the Prolaris 31-gene prognostic panel, a clinically established marker of aggressiveness, was observed at the bifurcation point of tumor evolution. STA of the human PDX model similarly revealed a high-dispersion invasive cluster branching into two fates, one exhibiting decreased heterogeneity consistent with specialization of invasive phenotypes. Together, these findings reveal, for the first time, that tumor invasion of the extracapsular muscle layer induces substantial transcriptional heterogeneity. This dynamic process highlights potential therapeutic vulnerabilities during the transition to extracapsular extension. Targeting tumor cells at this critical juncture may represent a key strategy for preventing ECE and improving prostate cancer outcomes. Supported by P30CA023074, University of Arizona Comprehensive Cancer Center Support Grant.



 Saptarshi Mallick, Rafael Sainz, Jayati Chakrabarti, Kendra D. Marr, Noel A. Warfel, Gregory C. Rogers, Beatrice S. Knudsen, Rashid Sayyid, Kelvin W. Pond, Anne E. Cress. Single cell spatial transcriptomics identifies unique cell populations driving muscle invasive prostate adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcc6a26c8a5d704d47fd40ba9c2b9c0030fcf73d" target='_blank'>
              Abstract B045: Single cell spatial transcriptomics identifies unique cell populations driving muscle invasive prostate adenocarcinoma
              </a>
            </td>
          <td>
            Saptarshi Mallick, Rafael Sainz, Jayati Chakrabarti, Kendra D. Marr, N. Warfel, Gregory C Rogers, Beatrice S. Knudsen, Rashid Sayyid, Kelvin W. Pond, A. Cress
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="CD8+ T cells are an important weapon in the therapeutic armamentarium against cancer. While CD8+CD103+ T cells with a tissue-resident memory T (TRM) cell phenotype are associated with favorable prognoses, the tumor microenvironment also contains dysfunctional exhausted T (TEX) cells that exhibit a variety of TRM-like features. Here we deconvolute TRM and TEX cells across human cancers, ascribing markers and gene signatures that distinguish these populations and enable their functional distinction. Although TRM cells have superior functionality and are associated with long-term survival post-tumor resection, they are not associated with responsiveness to immune checkpoint blockade. Tumor-associated TEX and TRM cells are clonally distinct, with the latter comprising tumor-independent bystanders and tumor-specific cells segregated from cognate antigen. Intratumoral TRM cells can be forced toward an exhausted fate when chronic antigen stimulation occurs, indicating that the presence or absence of continuous antigen exposure within the microenvironment is the key distinction between tumor-associated TEX and TRM populations. These results highlight unique functions for TRM and TEX cells in tumor control, underscoring the need for distinct strategies to harness these populations for cancer therapies. Here the authors show that tissue-resident memory and exhausted T cells in tumors are distinct populations that are shaped by relative presence or absence of TCR signals, suggesting that a tailored therapeutic strategy is needed to target each subset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a712967a1f06be04c88812bdec6b8089c56c0b" target='_blank'>
              Antigen reactivity defines tissue-resident memory and exhausted T cells in tumors
              </a>
            </td>
          <td>
            Thomas N. Burn, Jan Schröder, Luke C. Gandolfo, M. Osman, Elanor N. Wainwright, E. Lam, Keely M. McDonald, Rachel B. Evans, Shihan Li, Daniel Rawlinson, Lachlan Dryburgh, Ali Zaid, Z. Maliga, Dominick Schienstock, Philippa Meiser, H. J. Lee, Hongjin Lai, Marcela L. Moreira, Pirooz Zareie, Louis H-Y. Lee, Lutfi Huq, Susan N. Christo, Justine Seow, Keith A. Ching, Stéphane M. Guillaume, Kathy Knezevic, Simone L. Park, Maximilien Evrard, Jason Waithman, T. Gebhardt, Scott N. Mueller, G. Riddiough, M. V. Perini, S. C. Tsao, Terence P. Speed, P. Sorger, S. Loi, Francis R. Carbone, Stephanie Gras, Timothy S Fisher, Bas J Baaten, M. Dawson, Laura K. Mackay
          </td>
          <td>2025-12-29</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7276a25631d754628d5b48707ca83be4de135812" target='_blank'>
              Biomarker Quantification in Breast Cancer using Xenium In Situ
              </a>
            </td>
          <td>
            Amanda Janesick, Stephanie N. Kravitz, Weston Stauffer, Miriam Valencia, Sarah E. B. Taylor
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1654ebba084e6a42978b44a5cecff88e9f2e8ee" target='_blank'>
              DISCO-seq: 3D single-cell transcriptomics of intact biological systems
              </a>
            </td>
          <td>
            H. Bhatia, Laurent H. A. Simons, L. B. Kuemmerle, Cristin McCabe, S. Jansen, Zhisong He, Mayar Ali, Denise Jeridi, M. Todorov, Luciano Hoeher, S. Padmarasu, Ha Eun Park, Justus Thevis, L. Bartos, Inderjeet Singh, S. Ussar, Kıvanç Görgülü, Hana Algül, Kenny Roberts, O. Bayraktar, Matthias Brendel, F. Theis, Barbara Treutlein, Aviv Regev, Ali Ertürk
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 Colitis-associated colorectal cancer (C-CRC) arises in the context of chronic inflammation, where tumor microenvironment interactions strongly influence progression. Spatial transcriptomics enables characterization of these interactions by retaining tissue architecture and cellular context. We aimed to characterize the cellular and microenvironmental features that underlie tumor development in C-CRC.



 Seven tissue samples from three C-CRC patients were analyzed using consecutive sections processed by 10x Xenium and 10x Visium. Spatial coordinates of Xenium cells were mapped onto Visium spots to enable integrated spatial transcriptomic profiling (Figure 1A).



 Xenium analysis identified 3,506,705 cells with diverse epithelial, fibroblast, and immune cell populations (Figure 1B-C). Tumor cell subclustering revealed two functionally segregated populations: (1) proliferating tumor cells characterized by cell-cycle genes (MKI67, TOP2A) and (2) invasive tumor cells with elevated epithelial mesenchymal transition (EMT)-associated genes (CD44, CXCL1). Cellular neighborhood analysis based on distances between cells identified a distinct microenvironmental niche unique to invasive tumor cells, characterized by spatial proximity of cancer-associated fibroblasts (CAFs) and SPP1+ macrophages (Figure 1D). Pathway enrichment analysis showed selective up-regulation of angiogenesis, hypoxia, and inflammatory pathways in invasive versus proliferating tumor regions (Figure 1E). Receptor–ligand analysis identified TNC–ITGB6 interactions between CAFs and invasive tumor cells (Figure 2A). A differentially expressed gene signature distinguishing invasive from proliferating tumor cells stratified TCGA sporadic CRC samples into invasive-tumor-like and proliferating-tumor-like groups. Invasive-tumor-like tumors predominantly displayed consensus molecular subtype (CMS) 4 characteristics, consistent with the known enrichment of CMS4 in UC-CRC and supporting a role for CAF-mediated interactions in mesenchymal phenotype development (Figure 2B).



 This integrated spatial transcriptomics study identified two distinct tumor populations in C-CRC with segregated microenvironmental contexts. The invasive tumor phenotype is characterized by CAF proximity, SPP1+ macrophages, and TNC–ITGB6-mediated signaling. These findings reveal that tumor-stromal composition and architecture are linked to mesenchymal phenotype development, suggesting the microenvironment as a potential target for biomarker and therapeutic development in C-CRC.
 Conflict of interest:
 Lee, Yoonho: No conflict of interest
 Kim, Yongjae: No conflict of interest
 Baek, Jiwon: No conflict of interest
 Hwang, Sung Wook: no conflicts
 Lee, Jong Lyul: None.
 Park, Sang Hyoung: No conflicts
 Yang, Dong-Hoon: Nothing to declare
 Kim, Jihun: No conflict of interest
 Yang, Suk-Kyun: No conflict of interest
 Han, Buhm: No conflict of interest
 Song, Kyuyoung: No conflict of interest
 Yoon, Yong Sik: No conflict of interest
 Ye, Byong Duk: Byong Duk Ye reports consulting fees from AbbVie Korea, BMS Pharmaceutical Korea Ltd., Celltrion, Chong Kun Dang Pharm, CJ Red BIO, Curacle, Daewoong Pharm, Dong-A ST, Ferring Korea, Hanmi Pharmaceutical, Imscout, IQVIA, Johnson & Johnson, Johnson & Johnson Korea, Jeil Pharmaceutical Co., Kangstem Biotech, Korea Otsuka Pharm, Korea United Pharm, Lilly Korea, Medtronic Korea, NanoEntek, ORGANOIDSCIENCES Ltd., Pfizer Korea, Samsung Bioepis, Takeda, Takeda Korea and Yuhan speaker fees from AbbVie Korea, BMS Pharmaceutical Korea Ltd., Celltrion, Cornerstones Health, Curacle, Daewoong Pharm, Eisai Korea, Ferring Korea, IQVIA, Johnson & Johnson Korea, Pfizer Korea, Samsung Bioepis, and Takeda Korea and research support from Celltrion and Pfizer Korea.
 Lee, Ho-Su: No conflict of interest


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c70793b20108beca09788feb076288ec350d6d1" target='_blank'>
              DOP110 Spatial mapping of tumor-stromal cell interactions defines mesenchymal phenotypes in colitis-associated colorectal cancer
              </a>
            </td>
          <td>
            Y Lee, Y. Kim, J. Baek, S. Hwang, J. Lee, S. Park, D. H. Yang, J. Kim, S. K. Yang, B. Han, K Song, Y S Yoon, B. Ye, H S Lee
          </td>
          <td>2026-01-01</td>
          <td>Journal of Crohn’s and Colitis</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b48b1c55225e03672e68d4c772d4807ab8a0a9" target='_blank'>
              Critical role of cell competition in gliomagenesis
              </a>
            </td>
          <td>
            Ying Jiang, Ryuhjin Ahn, Arthur Huang, Phillippe P. Gonzalez, Jungeun Kim, Guoxin Zhang, Zihao Liu, Zhenqiang He, Lindsey A Dudley, Kunal S. Patel, Godfrey A Dzhivhuho, Sam Crowl, Piotr Przanowski, Luisa Quesada Camacho, Sijie Hao, Jianhao Zeng, S. Hippenmeyer, Mohammad Fallahi-Sichani, Kevin A. Janes, Kristen M. Naegle, M. Hammarskjold, Steven A. Goldman, H. Kornblum, Maojin Yao, Forest White, Hui Zong
          </td>
          <td>2026-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Intratumoral heterogeneity (ITH) is a major driver of mortality in breast cancer (BC) patients and a critical factor in the variable therapeutic outcomes observed in BC treatment. Understanding the mechanisms underlying ITH is essential for advancing both clinical and basic BC research. Chromatin accessibility is critical for regulation of gene expression and cellular identity and plays a central role in shaping ITH and tumor evolution. However, studying chromatin accessibility in situ has been challenging due to the availability of technical platforms. Here, we leveraged the spatial ATAC-seq platform to profile the chromatin accessibility landscape of tumors from six BC patients. Our analyses revealed prominent heterogeneity within tumor regulatory modules and spatial variations in immune cell composition and stromal structures, offering a framework for investigation of the molecular architecture underlying ITH. Moreover, we identified two tumor subclones with potential common origin but distinct immune infiltration conferred by regulatory cascades, suggesting that epigenetic regulation may further contribute to the divergent tumor microenvironments and phenotypic diversity of these subclones. Our study provides novel insights into the molecular mechanisms driving ITH and opens up potential avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b8d2e84c5991207c662ff2b4c48eb4734a447e" target='_blank'>
              Spatial Chromatin Accessibility Analysis of Intratumor Heterogeneity in Breast Cancer.
              </a>
            </td>
          <td>
            Yingying Qian, Miao Zhu, C. Ren, Yeyong Zhou, Jian Xu, Liang Dong, Guangyu Zhang, Cheukfai Li, Jiaoyi Lv, Qiaorui Xing, Guochun Zhang, Guangdun Peng, Ning Liao
          </td>
          <td>2026-01-12</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The human brain contains a milieu of extracellular matrix (ECM) components that promote normal development and physiology. ECM signaling pathways are often dysregulated in brain pathologies including the malignant cancer glioblastoma (GBM). Here, we used single-cell spatial transcriptomic platforms to map the expression of nearly 400 ECM genes in matching non-cancerous brain and GBM samples. At least four different GBM cell populations have been identified that show unique ECM expression profiles and spatial enrichment in distinct intratumor regions. Spatial mapping demonstrates largely non-overlapping expression signatures of ECM components in GBM stromal cell types, particularly in vascular endothelial cells and microglia/macrophages. Comparisons of GBM versus lower grade astrocytoma samples identifies differential expression of key ECM components. Computational analysis reveals novel ECM ligand-receptor networks between GBM and stromal cells. This spatial atlas provides new insights into ECM control of brain tumor initiation and progression and identifies potential targets for therapy in GBM. Using spatial transcriptomics, this study profiled the expression of hundreds of matrisome genes at single-cell resolution in human brain tumors, revealing targetable adhesion and signalling components in glioblastoma and lower-grade gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7358fd5e226c097b2efb1a3758a0fef198cc402b" target='_blank'>
              Single cell spatial profiling of the matrisome identifies region-specific adhesion and signaling networks in glioblastoma
              </a>
            </td>
          <td>
            Arpan De, Santiago A Forero, Ali Pirani, John E. Morales, Marisol De La Fuente-Granada, Sumod Sebastian, Jason T. Huse, Leomar Ballester, J.S. Weinberg, Frederick F. Lang, Kadir C. Akdemir, Joseph H. McCarty
          </td>
          <td>2025-12-09</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Spatial organization of the disease microenvironment informs patient prognosis. Key modalities for studying spatial biology include H&E images (WSIs) for tissue structure and spatial transcriptomics (ST) for transcriptome-level programs. Spatial analysis aims to (1) identify markers linked to clinical outcome, (2) understand functional programs driving these associations, and (3) guide targeted therapies. Current research addresses these topics but offers limited explainability across the full morphology - molecular mechanism - outcome axis. Further, given the abundance of WSIs and limited availability of ST, there is a need for analyses integrating these complementary datasets. We present an AI-driven framework combining foundation-model features, multiple-instance learning, unsupervised clustering, and molecular analyses to identify mechanisms underlying outcome associated patterns. Applied to HER2+ breast cancer, we identify CCND1 and PTK6 signaling in tumor regions linked to trastuzumab resistance, consistent with prior studies. Our approach offers interpretable insights for multi-level resistance mechanisms, tissue-specific drug targeting, and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7a60accd94c10ba89d66c522cd744cd2f973bb5" target='_blank'>
              A multimodal framework to identify molecular mechanisms driving patient group-associated morphology through the integration of spatial transcriptomics and whole slide imaging
              </a>
            </td>
          <td>
            Reva Kulkarni, A. Maddox, Sara Bailey, Arvind Rao
          </td>
          <td>2026-01-16</td>
          <td>Npj Artificial Intelligence</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Recent progress in single-cell RNA sequencing has led to mechanistic and clinically actionable insight into genetic heterogeneity and tumor progression via transcriptome profiling at single-cell level, applied to identification of cell types, gene expression patterns, and signaling pathways involved in cancer development. In this work, we review the use of single-cell RNA sequencing (scRNA-seq) applications to gain insights into tumor molecular and cellular characteristics, such as cellular heterogeneity, rare cell populations, characteristic pathogenic cell populations, cells of the immune tumor microenvironment, and information regarding clonal evolution, none of which can be observed using bulk RNA-seq. We describe how this set of methods facilitates a better understanding of tumor heterogeneity, interactions between the tumor cells and the cells of the tumor microenvironment (TME), and can elucidate potential therapeutic targets. From the applied clinical perspective, we summarise the ability of scRNA-seq data to identify molecular indicators for diagnosis, outcome, and prediction of response to therapy. This is particularly relevant due to the low response rate to therapy of non-small cell lung cancer (NSCLC) and acquired resistance, including in immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d91b19b2652e7118bd597f31b2a5c69fb2aa994e" target='_blank'>
              Tumor heterogeneity assessment using single-cell RNA sequencing (scRNA-seq): applications in lung cancer for diagnosis and treatment
              </a>
            </td>
          <td>
            C. Bica, Oana Zanoaga, L. Pop, C. Ciocan, L. Raduly, A. Nutu, I. Berindan‐Neagoe, Andreas Bender
          </td>
          <td>2026-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e234e293312dfb9093dad9f87bd40ef8d26f2e28" target='_blank'>
              Spatial transcriptomics reveals molecular heterogeneity and subtype-specific therapeutic targets in small cell lung cancer.
              </a>
            </td>
          <td>
            Tongji Xie, Le Tang, G. Fan, Xiaohong Han, Yuankai Shi
          </td>
          <td>2026-01-16</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3490b5722169f26207fe3782f1d2c2b6c0970030" target='_blank'>
              Dual Lineage Tracing Identifies Cellular Mechanisms Underlying Radiation-Associated Changes in Atherosclerotic Lesion Composition
              </a>
            </td>
          <td>
            Rebecca A. Deaton, Tajbir Raihan, Victoria M. Milosek, Fatema Allaham, Alexandra L. Krinsky, Laura S. Shankman, G. F. Alencar, A. Salamon, Nazanin Moradinasab, Subhashis Banerjee, Gary K. Owens
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT Head and neck squamous cell carcinoma (HNSCC) is the sixth most common cancer worldwide. HPV-negative HNSCC, arising in diverse upper airway mucosal niches, is particularly aggressive, with poor 5-y survival and a limited response to immune checkpoint inhibitors. A deeper understanding of the tumor-localized immune landscape is essential to uncover actionable immunotherapeutic targets. Here, we integrated two single-cell RNA sequencing (scRNA-seq) datasets from 29 samples totaling nearly 300,000 immune cells to dissect immune rewiring during tumor progression and lymph node metastasis in HPV-negative HNSCC. We identified distinct shifts in adaptive immune cell populations across 14 peripheral blood mononuclear cell (PBMC) and 21 tumor-infiltrating immune cell (TIC) states. Notably, TICs exhibited enriched interferon response and immunomodulatory gene signatures, in contrast to PBMCs, indicating tumor-specific immune imprinting. Ligand–receptor analysis revealed that immunosuppressive crosstalk between macrophages and cytotoxic cells was associated with advanced disease. To spatially validate these transcriptional states, we conducted multiplexed immunofluorescence profiling on nine locally invasive HPV-negative HNSCCs, all from the ventrolateral tongue mucosa. Spatial proteomics confirmed peritumoral enrichment of activated (CD107a+, ICOS+) NK and CD8+ T cells and intratumoral accumulation of exhausted (PD-1+, PD-L1+) phenotypes, mirroring pseudotime trajectories inferred from scRNA-seq. These findings highlight spatially localized cytotoxic cell exhaustion as a key immune evasion mechanism in HPV-negative HNSCC and underscore the value of integrating spatial and single-cell data to reveal therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e184d6ff5ae8303f9ef6f76673a4a076479b44fb" target='_blank'>
              Single-cell landscape of peripheral and tumor-infiltrating immune cells in HPV-negative HNSCC
              </a>
            </td>
          <td>
            Rômulo Gonçalves Agostinho Galvani, Adolfo Alexis Rojas Hidalgo, Carlos Alberto Biagi-Junior, B. Matuck, Jelte Martinus Maria Krol, Brittany T. Rupp, Nikhil V. Kumar, Khoa L. A. Huynh, Jinze Liu, Siddharth Sheth, Vinicius Maracaja-Coutinho, K. Byrd, Patricia Severino
          </td>
          <td>2025-12-30</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9542f45171a12fa01fc5a868a4f227534778e8" target='_blank'>
              Iron-Associated Mesenchymal Plasticity and Tumor Invasion in Glioblastoma
              </a>
            </td>
          <td>
            C. Petry, A. Boecke, A. Mohammed, N. Ghariwala, V. Jariwala, Mohammad Al Shhab, O. Buchholz, I. Vasilikos, J. M. Nakagawa, Andreas Vlachos, J. Grauvogel, M. Shah, Roland Rölz, Marco Prinz, U. G. Hofmann, Jürgen Beck, Kevin Joseph, V. Ravi
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="This study uses the well-established 4T1.2 and 67NR mammary cancer cell lines as syngeneic immunocompetent implantable murine models of triple negative breast cancer. Using pair-wise analysis, we compare the implantation route of cancer cells by conventional orthotopic injections in the mammary fat pad vs intraductal injection in the mammary ductal tree. We reveal that implantation methodology influences the inherent metastatic potential, tumour progression and the co-evolution of cancer cells and the tumour microenvironment. Intraductal injection accelerates tumour growth, especially in the non-metastatic 67NR model, and better mimics early tumourigenic events that preserve the tissue microenvironment crucial for tumour-stromal interactions. Immune cell composition differences are driven by the tumour model but independent of the implantation method, however, intraductal implantation carries higher stromal proportions at early stages of tumour development. In the 4T1.2 model, intraductal injection increases metastatic burden. We found markedly different systemic inflammation traits unique to the metastatic 4T1.2 scenario consistent with the acquisition of a cancer immunotolerant phenotype during progression to metastatic disease. Metastatic dissemination induced a myeloid-dominated inflammatory milieu, particularly driven by granulocytic myeloid-derived suppressor cells, contrasting with the lymphocyte-rich profile in the non-metastatic 67NR tumours. Immune responses of the metastatic 4T1.2 scenario were further analysed by targeted single-cell RNA sequencing revealing distinct immune activation in primary tumours, emphasising the differential role of immune cells between primary and metastatic lesions. Our findings stress the importance of carefully selecting experimental models to faithfully recapitulate breast cancer progression and metastasis, providing insights for future therapeutic interventions focused on immune modulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/550aca754482f013c49aead621e9ffc825e3ee4c" target='_blank'>
              Modelling the metastatic immune microenvironment in triple-negative breast cancer
              </a>
            </td>
          <td>
            Laura Rodriguez de la Fuente, Andrew M. K. Law, Laura Rangel-Sanchez, J. Venhuizen, Fiona Steffan, Lesley E Castillo, Robin L. Anderson, F. Valdés-Mora, D. Gallego-Ortega
          </td>
          <td>2025-12-10</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8015c113e13d1290a915b03f174abb55236e475" target='_blank'>
              Longitudinal analysis of matched patient biospecimens reveals Neural Reprogramming of Cancer-Associated Fibroblasts Following Chemotherapy in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Aylin Z. Henstridge, Nandini Arya, Ahmed M. Elhossiny, Padma Kadiyala, Georgina Branch, Vaibhav Sahai, Nicole Peterson, Jorge D. Machicado, Richard S. Kwon, Allison Schulman, Erik Wamsteker, George M. Philips, Stacy Menees, Jonathan Xia, Arvind Rao, Jiaqi Shi, Timothy L. Frankel, Filip Bednar, Marina Pasca di Magliano, C. Lyssiotis, M. Truty, Martin Fernandez-Zapico, Eileen S. Carpenter
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal cancer, with liver metastases significantly worsening outcomes. However, features of the tumor microenvironment (TME) that are distinct between primary and metastatic sites remain poorly defined. Cellular neighborhoods within the TME are recognized as functional units that influence tumor behavior. Conventional spatial methods, which assign equal weights to all cells in a region, fail to capture the nuances of cellular interactions. To address this, we report here the development of Functional Cellular Neighborhood (FunCN) quantification, which integrates both the proportion and proximity of surrounding cells. Applying FunCN to PDAC imaging mass cytometry data, we identified neutrophil-enriched interactions in liver metastases compared to primary tumors, correlating with elevated VISTA expression by tumor cells. Additionally, FunCN clusters around CD8⁺ T cells in pancreas and liver were associated with higher TIGIT and LAG3, respectively. These findings demonstrate the importance of spatial immune landscapes in PDAC and identify potential therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87979b18b917426107034a7951a93426ded0148e" target='_blank'>
              Computational modeling of cellular influence delineates functionally relevant and distinct cellular neighborhoods in primary and metastatic pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Yeonju Cho, Jae W Lee, Sarah M. Shin, Alexei G Hernandez, Xuan Yuan, Jowaly Schneider, Jody E. Hooper, Laura D. Wood, E. Jaffee, Atul Deshpande, W.J. Ho
          </td>
          <td>2026-01-23</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The tissue architecture of classic Hodgkin Lymphoma (CHL) is unique among cancers and characterized by rare malignant Hodgkin and Reed-Sternberg cells that co-evolve with a complex ecosystem of immune cells in the tumor microenvironment (TME). The lack of a comprehensive systems-level interrogation has hindered the description of disease heterogeneity and clinically relevant molecular subtypes. Here, we employed an integrative, multimodal approach to characterize CHL tumors using malignant cell sequencing, spatial transcriptomics and imaging mass cytometry. We identified four molecular subtypes (CST, CN913, STB, and CN2P), each characterized by distinct clinical features, mutational patterns, malignant cell gene expression profiles, and spatial architecture involving immune cell populations. Functional modeling of CSF2RB mutations, a characteristic feature of the CST subtype, revealed dysregulated oncogenic signaling and unique TME crosstalk. These findings highlight the significance of multi-dimensional profiling in elucidating patterns of molecular alterations that drive immune ecosystems and underlie therapeutically exploitable vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6324873c99ed0026f9e046294cb9724bf214ad" target='_blank'>
              Multidimensional Characterization of Tumor-Immune Architecture Reveals Clinically Relevant Classic Hodgkin Lymphoma Subtypes.
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew Lytle, Yifan Yin, Makoto Kishida, Michael Yu Li, Cecilia Lee, Denise Smorra, L. Hilton, S. Healy, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O Brien, Jasper Ch Wong, Claudia Cassidy, Manabu Fujisawa, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Rangel Patiño, Michael Hong, Shaocheng Wu, Eric Lee, Ali Sakhdari, Katsuyoshi Takata, T. Miyata-Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander Xu, A. Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, R. Morin, A. Alizadeh, K. Savage, David W Scott, Christian Steidl
          </td>
          <td>2026-01-30</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Quiescence (reversible cell-cycle arrest) and senescence (irreversible arrest) are challenging to distinguish due to a lack of specific biomarkers, yet both arise simultaneously after chemotherapy. While senescence suppresses tumors by limiting proliferation and recruiting the immune system, quiescent cancer cells evade future therapies and may resume proliferation. Here, we pair time-lapse imaging of cell-cycle dynamics with single-cell RNA sequencing after etoposide treatment to differentiate these states, linking heterogeneous cell-cycle phenotypes to the transcriptomic landscape. We identify diverse senescent types (senotypes) and link them to two arrest pathways — a gradual path arising after a standard mitosis-to-G0 transition, and an alternative direct path driven by a mitotic slip. Using pseudotime trajectory analysis, we find that senescent phenotypes begin to manifest early and gradually along the first trajectory, even in shallow quiescent cells. These data support a model wherein, following chemotherapy, quiescence and senescence exist on a continuum of cell-cycle withdrawal at a transcriptome-wide level. Quiescent and senescent cells often arise side by side after stress, yet their molecular relationship remains unclear. Here, the authors combine live-cell imaging and single-cell RNA sequencing to reveal a continuum of cell-cycle withdrawal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba1a5e18a7ba7e39898f97a7ee1e3a1c3c9bb5a2" target='_blank'>
              Single-cell RNA sequencing reveals a quiescence-senescence continuum and distinct senotypes following chemotherapy
              </a>
            </td>
          <td>
            Brianna Fernandez, Victor J. Passanisi, Humza M. Ashraf, S. Spencer
          </td>
          <td>2025-12-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0135229fffb59fe03ba95bebb58be45a4b3470c" target='_blank'>
              De novo H3.3K27M-altered Diffuse Midline Glioma in human brainstem organoids to dissect GD2 CAR T cell function
              </a>
            </td>
          <td>
            N. Bessler, A. Wezenaar, H. Ariese, C. Honhoff, E. J. Wehrens, N. Dommann, C. Ruiz Moreno, T. J. van den Broek, R. Collot, D.J. Kloosterman, F. Keramati, M. Roosen, S. de Blank, Esmée J. van Vliet, M. Barrera Román, L. C. Gatti, Ali Ertürk, J. Kuball, Z. Sebestyén, M. Kool, S. Patrizi, E. Miele, A. Künkele, M. Kranendonk, Annelisa M. Cornel, Stefan Nierkens, Christian Mayer, H. Stunnenberg, A. Alemany, Maria Alieva, A. Rios
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Neoadjuvant immunotherapy has reshaped the treatment paradigm for esophageal squamous cell carcinoma (ESCC), yet mechanisms of resistance in non-responders remain poorly understood. Specifically, the spatial orchestration of the tumor microenvironment that limits effective antitumor immunity is not fully elucidated. Methods Integrating spatial transcriptomic analysis with large-scale tissue pathology, we mapped the spatial architecture of the immune landscape in immunotherapy-resistant ESCC to identify cellular and molecular determinants of treatment failure. Results We revealed a distinct immune exclusion phenotype in non-responders, characterized by peritumoral CD8+ T cell enrichment coupled with intratumoral depletion. At the invasive front, we identified COL11A1+ cancer-associated fibroblasts (CAFs) and SPP1+ tumor-associated macrophages as spatially correlated markers of immune exclusion, demarcating regions characterized by limited T cell infiltration. Mechanistically, a distinct subpopulation of LAMC2+ tumor cells localized at the tumor boundary acts as the master orchestrator of this barrier. These LAMC2+ cells exhibit aberrant lactate metabolism and elevated stemness, driving CAF activation via Semaphorin 3C secretion. Strikingly, this ‘LAMC2+ boundary tumor cell-immune-privileged niche’ axis exhibits pan-cancer correlations with immunotherapy resistance, positioning LAMC2 as a robust predictive biomarker. Conclusion In this study, we identify the tumor invasive margin as a critical determinant of immunotherapy resistance in ESCC. By defining the specific spatial markers and molecular architecture of an immune-privileged niche associated with the immune-exclusion barrier, our findings demonstrate the value of spatially resolved microenvironmental analysis. Moreover, we propose that therapeutic targeting of this boundary niche represents a promising strategy to overcome resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/858298d1227537940c73aee34b8d245e5b4c9498" target='_blank'>
              An Immune-privileged niche mediates immunotherapy resistance in esophageal carcinoma
              </a>
            </td>
          <td>
            Li Wan, Boye Li, Tianyun Ma, Min Wang, Fei Xie, Haotian Ge, Zixuan Xiao, Xinyi Chen, Lingda Meng, Li Zhang, Kunxin Xie, Qilong Wang
          </td>
          <td>2026-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Non-small cell lung cancer is the leading cause of cancer-related mortality worldwide, with lung adenocarcinoma (LUAD) as the most common subtype. Although early-stage disease is often treated surgically, advanced LUAD typically requires chemotherapy, radiation, and/or immunotherapy, largely focused on T cell–mediated responses. Therapeutic efficacy, however, is also shaped by tumor-infiltrating (TI) B cells, whose roles in LUAD remain incompletely understood.



 We performed cytometry by time of flight (CyTOF) using 44 markers on matched tumor, adjacent lung, and peripheral blood samples from 48 LUAD patients to define TI immune landscapes. 66 immune cell subsets were identified, and patients were stratified into four groups based on TI cell composition. Adaptive immune receptor repertoire sequencing (AIRR-seq) of IgM and IgG was conducted on matched samples from 29 patients to assess clonal expansion and affinity maturation. Subisotype-resolved AIRR-seq was additionally performed on tumor samples from 18 patients.



 CyTOF analysis revealed four patient groups with distinct TI immune profiles. AIRR-seq demonstrated increased clonal expansion and affinity maturation in tumors compared to adjacent lung and blood. Tumor-specific IGHV enrichment patterns were observed but were not associated with patient group assignment. Instead, clonal expansion was greatest in tumors with higher lymphocyte proportions. Subisotype-resolved analysis showed enrichment of IGHG2 and IGHG3 in tumors from patients with low TI B cell abundance, whereas IGHG4 was enriched in patients with high TI B cell infiltration and correlated with four CyTOF-defined immune subsets.



 These findings reveal substantial inter-individual variation in TI immune landscapes and highlight distinct B cell repertoire and subisotype features within LUAD tumors. Together, these data suggest that B cell composition and antibody subisotype usage may contribute to immune contexture and could inform the development of more tailored immunotherapeutic strategies in LUAD.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6aa20b3c0324cfd852e979d6ecb7e9b1134c03" target='_blank'>
              B cell phenotypes and antibody signatures associate with interpatient variation in the lung adenocarcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Kamille M. Rasche, David Tieri, Samantha M. Morrissey, Hong Li, Fan Zhang, William S. Gibson, EA Ford, Uddalok Jana, Melissa L. Smith, Jun Yan, Corey T. Watson
          </td>
          <td>2026-01-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="

 The spatial distribution and functional heterogeneity of tumor-infiltrating lymphocytes (TILs) significantly impact patient outcomes in non-small cell lung cancer (NSCLC). While T cell factor 1 (TCF1) expressing TILs have emerged as key players in sustaining anti-tumor immunity, their subset characterization, localization, and clinical significance within the tumor microenvironment remain poorly defined.




 We performed multiplex immunohistochemistry and immunofluorescence to characterize TCF1
 +
 immune cell subsets, in 102 NSCLC tumors, separately analyzing the tumor center (TC) and invasive front (IF). We integrated this data with publicly available single-cell RNA-sequencing datasets and clinical outcome analyses.





 CD4
 +
 T cells and CD79α
 +
 B cells, dominate the TCF1
 +
 landscape, while CD8
 +
 T cells constitute a minority of TCF1
 +
 immune cells, particularly in the TC. We demonstrated the presence of tumor-infiltrating IgG
 +
 /IgA
 +
 plasma cells co-expressing TCF1. PD1
 +
 TCF1
 -
 cells were more frequent than PD1
 +
 TCF1
 +
 cells both in the TC and IF, reflecting that terminally differentiated exhausted TILs predominate within the tumor microenvironment. Survival analyses revealed significantly different prognostic impact of TILs including TCF1-expressing cells based on topography. Multivariate analysis showed that increased CD8
 +
 TCF1
 +
 cells (HR: 2.5; p=0.039) and increased TCF1 expression by cancer cells (HR: 2,7; p=0.041) in the TC and CD4
 +
 TCF1
 +
 cells (HR: 0.4; p=0.043) in the IF emerged as negative and positive independent prognostic markers for Disease-free survival (DFS), respectively. Integrating PD-L1 expression with TILs, PD-L1 immunopositivity was correlated with increased CD8
 +
 and PD1
 +
 TCF1
 -
 cell infiltration and was associated with favorable DFS especially in the TC.





 Our findings support a more refined framework for TCF1
 +
 TIL assessment and TCF1 expression across cellular populations in the tumor microenvironment, with implications for prognostication in operable NSCLC.

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb42a95fd19ed986ce09103139534fde5284c2fa" target='_blank'>
              Immunophenotyping TCF1-expressing TILs: spatial profiling and prognostic value in operable non-small cell lung cancer
              </a>
            </td>
          <td>
            Konstantinos Ntostoglou, Georgios Christodoulopoulos, Katie Stoker, Jean Descarpentrie, A. Xagara, D. Chatzidaki, Vasiliki Anastasopoulou, Ilias P. Nikas, A. Ieronimaki, Lucy Booth, Sophia Tsoka, Ioannis Vamvakaris, A. Klinakis, Eleni Patsea, Sophia N. Karagiannis, Ioannis G. Panayiotides, Teresa Frisan, Vassilis Georgoulias, A. Kotsakis, I. Pateras
          </td>
          <td>2026-01-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Distal cholangiocarcinoma (dCCA) arises from the distal bile duct and is anatomically embedded within the pancreatic head, adjacent to abundant autonomic nerve plexuses. This unique location renders dCCA particularly prone to perineural invasion (PNI), a pathological hallmark that contributes to its dismal prognosis. However, the spatial architecture and molecular drivers that orchestrate PNI remain poorly defined. Here, we applied Xenium subcellular resolution spatial transcriptomics platform to profile resected tumor tissues from dCCA patients stratified by PNI status pathologically. A spatially resolved atlas comprising a total of 20 cell types was generated, uncovering enrichment of Schwann cells, type 2 conventional dendritic cells (cDC2), M2-like macrophages, cancer associated fibroblasts (CAFs) and B/plasma cells in PNI-high tumors, along with depletion of exhausted CD8+ T cells. Heterogeneous malignant cells in PNI-high tumors demonstrated activation of extracellular matrix remodeling and axonogenesis pathways, in line with the initial pathological classification. Spatial mapping further revealed distinct PNI-associated niches, notably matrix-producing CAFs (mCAFs)-macrophage clusters exhibiting coordinated enrichment of inflammatory and fibrotic programs. We further identified the LAMB3-DAG1 axis as a potential mediator of dCCA cells-Schwann cell interaction, while the preferential proximity of arteries to Schwann cells suggested additional microenvironmental support for nerve invasion. Collectively, our study provides a comprehensive subcellular atlas of PNI in dCCA, uncovering coordinated epithelial, stromal, and immune remodeling that drives perineural invasion. The identified biomarkers not only hold promise for patient stratification but may also guide intraoperative navigation and surgical margin determination, offering new avenues for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7ed53173a0ee6bb5d8c5c5791ee85be378c7900" target='_blank'>
              A subcellular spatial atlas illuminates the microenvironmental remodeling of perineural invasion in distal cholangiocarcinoma
              </a>
            </td>
          <td>
            Fansen Ji, Hao Chen, Huan Li, Jiawei Zhang, Sijia Li, Pengfei Wang, Hao Liu, Cui Ge, Bingjun Tang, Hongfang Yin, Xuedong Wang, Jiahong Dong
          </td>
          <td>2026-01-08</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5bdb6dd873fe4b6bf8ca46144bcdfa66e477ac0" target='_blank'>
              Beyond one-size-fits-all: single-cell transcriptomic signatures predict drug efficacy and reveal responder subgroups in endometriosis
              </a>
            </td>
          <td>
            R. Pérez-Moraga, C. Bafligil, S. Harden, Santiago García‐Martín, M. J. Jiménez-Santos, Ioanna Tiniakou, Sophie Ribeiro-Volturo, Line Gies, María Teresa Pérez Zaballos, Cristina Fernández-Molina
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f2b8ad851f4ed7ac71d3b37aff3d0eab8d4c1da" target='_blank'>
              Connecting spatial regions to clinical phenotypes by transferring knowledge from bulk patient data
              </a>
            </td>
          <td>
            Bayarbaatar Amgalan, Ermin Hodzic, Ziynet Nesibe Kesimoglu, M.G. Hirsch, John D. Bridgers, Jan Hoinka, David L Levens, Chi-Ping Day, Teresa M. Przytycka
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Terminally exhausted CD8+ T cells (Ttex) are emerging as clinically relevant immune subsets across solid tumors, marked by sustained inhibitory receptor expression, loss of TCF1, and limited proliferative capacity. Once considered functionally inert, Ttex are now recognized for their residual cytotoxic potential and strong associations with tumor immunogenicity, including microsatellite instability (MSI), high tumor mutational burden (TMB), and neoantigen load. Importantly, the prognostic significance of Ttex is highly tumor-context-dependent, shaped by stromal architecture, mutational burden, and progenitor Tpex availability. This review examines the biology, spatial localization, and prognostic value of Ttex, highlighting the Ttex/CD8+ ratio as a promising biomarker in cancers such as colorectal, lung, and esophageal carcinoma. We summarize recent advances in multiplex imaging, digital pathology, and AI-driven quantification that support the clinical integration of Ttex assessment. In addition, we discuss emerging therapeutic strategies targeting Ttex through immune checkpoint combinations, thymocyte selection-associated high mobility group box protein (TOX) and circRNA-mediated reprogramming, and exhaustion-resistant T cell engineering. Finally, we outline translational priorities including assay harmonization, functional validation, and longitudinal profiling to advance Ttex-based precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af1a0b32b60b15c52819ffc31235e60745d2593b" target='_blank'>
              Terminally exhausted CD8+ T cells in solid tumors: biology, biomarker potential and translational tools for precision oncology
              </a>
            </td>
          <td>
            Xuejun Guo, Shuhan Ma, Jingwen Wang, Yilin Fu, Wenxue Ma
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) enables high-resolution transcriptomic analysis but loses spatial context due to tissue dissociation, thereby limiting insights into spatially regulated cell–cell interactions critical for tissue function and disease, including cancer. Consequently, developing approaches capable of extracting spatially relevant intercellular interaction information from scRNA-seq data is of substantial importance for elucidating mechanisms underlying disease initiation and progression. Here, we present DeepPNCC, a novel deep learning framework that reconstructs pseudo-spatial cell-cell interaction networks (PSCCIs) from scRNA-seq data by leveraging latent spatial cues preserved in undissociated cell aggregates. Built on a variational graph autoencoder (VGAE) enhanced with adversarial regularization, DeepPNCC uniquely integrates local adjacency matrices derived from multiplet data, without relying on prior knowledge such as ligand-receptor pairs, to infer global, spatially informed interaction landscapes. DeepPNCC is available as an open-source Python package. DeepPNCC outperforms existing methods in recovering interactions aligned with spatial transcriptomics across mouse brain and breast cancer datasets. In breast cancer, the inferred cell-cell interaction network reveals a closed-loop signaling axis in triple-negative breast cancer, in which cancer-associated fibroblasts (CAFs) and perivascular-like cells (PVL) promote angiogenesis through the ADM-CALCRL-STAT1-CD40/CCL2/ICAM1 pathway. These findings provide important clues for elucidating the regulatory mechanisms of the tumor microenvironment. DeepPNCC provides an efficient tool for spatially informed analysis of scRNA-seq data, enabling the reconstruction of pseudo-spatial cell–cell interaction networks from datasets lacking explicit spatial information. This approach expands the spatial analytical scope of scRNA-seq and facilitates mechanistic dissection of cellular ecosystems in both health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee882b8d93f6b445af77befa6248cb5501417766" target='_blank'>
              DeepPNCC: reconstructing pseudo-spatial cell-cell interaction landscapes from single-cell data to decipher breast cancer pathogenesis
              </a>
            </td>
          <td>
            Xu-hua Li, Xiao-Ling Gao, Deng-Hui Guo, Jie Shen, Si-Rui Li, Na Wang, Xiao-ling Wen, Jiang-Ying Liang, Hong-jiu Wang, Zhen-Zhen Wang
          </td>
          <td>2025-12-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Osteosarcoma (OS) exhibits substantial genomic complexity and inter-patient heterogeneity, necessitating longitudinal, patient-matched analyses to understand acquired features of tumor evolution. However, most published OS data is limited to primary tumor samples, limiting insight into patient-specific resistance mechanisms. To address this, we characterized the genomic landscape of paired primary and metastatic tumor samples from dogs with spontaneous OS. Whole-genome and single-cell RNA sequencing reveal mutation and gene expression profiles that are predominantly organized by patient identity. Mutational burden and pathway alterations such as those involving PI3K, NOTCH, TP53, MAPK, RAS and epigenetic regulation differ between primary and metastatic samples. Variants present in tumor tissue are readily detectable in paired cfDNA samples, demonstrating the utility of this assay for identifying tumor-specific alterations associated with treatment resistance. Analysis of bulk RNA-seq data to estimate cell-type composition shows greater immune cell representation in metastases, underscoring the importance of immune signaling pathways in OS. These findings exemplify the presence of patient-specific alterations in genomic architecture over the course of tumor progression, linking CNV amplification, pathway reprogramming, and immune evasion in metastatic OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02ad1efaa62d98c7f16dd57c71f8980e8844d4c9" target='_blank'>
              Tumor microenvironment signature associated with morphology in systemic ALK-positive anaplastic large cell lymphoma
              </a>
            </td>
          <td>
            Chloé Bessiere, Loélia Babin, Sandra Dailhau, P. Gaulard, Stéphane Pyronnet, Fabienne Meggetto, Laurence Lamant
          </td>
          <td>2026-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell studies have significantly advanced our understanding of the transcriptional and functional heterogeneity in cancers. Recent studies have identified distinct states of cancer, immune, and stromal cells in the tumor microenvironment (TME), with growing evidence highlighting their clinical significance and therapeutic potential. Here, we present CancerSEA-X, an expanded version of CancerSEA that offers a comprehensive atlas of TME cell states. CancerSEA-X integrates 25 cancer cell states, 105 immune cell states, and 26 stromal cell states from systematically curated publications. Combining 239 single-cell datasets across 32 cancer types, encompassing over 9 million cells from 2120 patients, CancerSEA-X provides functional activity spectra and cancer-specific gene associations for these 156 cell states. These cell state-gene relationships were mapped onto networks, providing a systematic view of the TME. To improve usability, we redesigned the user interface to feature cell state characterization, state-gene correlation analysis, and interactive visualization of cell state-gene networks, enabling researchers to comprehensively explore these states and their functional relevance. Overall, CancerSEA-X serves as a valuable platform for investigating TME cell states, deepening our understanding of cancer heterogeneity, and potentially advancing the design of more effective clinical therapies. CancerSEA-X is freely available at http://biocc.hrbmu.edu.cn/CancerState.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc11040a019ecb4b6c2065cd731c8ff500837ec3" target='_blank'>
              CancerSEA-X: A Single-cell Resource for Tumor Microenvironment Cell States Across over 30 Cancer Types.
              </a>
            </td>
          <td>
            Lantian Deng, Wei Liu, Jinyuan Xu, Bo Zhang, Shengyuan He, Kun Liu, Xinxin Zhang, Huating Yuan, Fei Quan, Yun Xiao
          </td>
          <td>2026-01-16</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cf48b273167de19bd485ee182b5df8f720d9f6b" target='_blank'>
              Spatially distinct chromatin compaction states predict neoadjuvant chemotherapy resistance in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Reut Mealem, T. A. Phillips, Leor Ariel Rose, S. Marcotti, M. Parsons, A. Zaritsky
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Immunological intratumor heterogeneity (ImTH) describes the variability in the types, spatial distribution, and functional states of immune cells within tumors. While evidence suggests that ImTH influences tumor progression and therapeutic response, few studies have provided a quantitative characterization of ImTH. Here, we present Scoring Immunological Intratumor Heterogeneity (ScImTH), a novel algorithm that quantifies ImTH by calculating the Shannon entropy of immune cell type proportions within the tumor microenvironment. Using bulk, single-cell, and spatial transcriptomic datasets, we show that reduced ScImTH scores are associated with unfavorable survival outcomes, tumor progression-related molecular and phenotypic features, immunosuppressive states, and resistance to immunotherapy across multiple cancer types. Compared with existing measures of tumor immunity, such as immune score and B-cell receptor diversity, the ScImTH score demonstrated stronger and more consistent associations with clinicopathological features. Notably, the ScImTH score outperformed established biomarkers, including tumor mutational burden and PD-L1 expression, in predicting immunotherapy response. These findings highlight the clinical potential of the ScImTH score as a biomarker for cancer prognosis and immunotherapy stratification. More broadly, our results support the hypothesis that loss of immune diversity is a hallmark of tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e493ac7b6aab5ae1ab712cb247fef1395ea0b1a" target='_blank'>
              Quantitative profiling of intratumor immune heterogeneity identifies loss of immune diversity as a hallmark of cancer progression
              </a>
            </td>
          <td>
            Qiqi Lu, Jian-qin Luo, Chia-Hao Tung, Xiaosheng Wang, Zhongming Zhao
          </td>
          <td>2025-12-16</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Papillary thyroid cancer (PTC) exhibits highly variable clinical behavior, ranging from indolent growth to aggressive lymph node (LN) metastasis. However, the specific molecular mechanisms involved in PTC evolution, driven by metabolic reprogramming and tumor-microenvironmental interactions, remain poorly understood. This study aims to elucidate the spatial metabolic and transcriptional mechanisms underlying PTC tumorigenesis and LN metastasis at multiple molecular levels. An integrated spatial multi-omics strategy combining spatial metabolomics and spatial transcriptomics was employed to map the distribution of metabolites and gene expression in heterogeneous PTC tissues. Metabolite profiles and transcriptomic data were analyzed to identify dysregulated pathways and to uncover the key molecular features contributing to PTC tumorigenesis and LN metastasis. Genes related to metastasis-driving metabolites were validated using the TCGA dataset, and further substantiated by zebrafish xenograft models. Spatial mapping revealed unique metabolic and transcriptional signatures across heterogeneous regions of PTC tissue. Key metabolic pathways, such as the arginine–polyamine axis, glycolysis and lipid metabolism, were prominently dysregulated in cancer regions. The surrounding stroma also exhibited metabolic adaptations, like polyamine recycling and glucose storage, which supported tumor growth via metabolic crosstalk. Most importantly, greater spatial heterogeneity, spatial evolutionary routes and 5 potential metastasis-driving metabolites, including FA (22:6), PC (36:4), PC (34:1), N-acetylaspartate and ascorbic acid, were discovered in the primary PTC tissue with LN metastasis. Knockdown of key genes NAT8L and SVCT-2 obviously decreased the metastatic ability of PTC cells. The TCGA database further confirmed that high expression of 10 pro-metastatic metabolite-related genes was significantly associated with poorer prognosis in PTC patients. This integrated spatial multi-omics approach provides novel insights into the molecular mechanisms underlying PTC evolution, potentially guiding the development of more effective diagnostic and therapeutic strategies for PTC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4ee960363a0cd08e85ab4a85deecb06d245c95" target='_blank'>
              Integrated spatial metabolomics and transcriptomics reveal the molecular landscape of papillary thyroid cancer and its lymph node metastasis
              </a>
            </td>
          <td>
            Kening Li, Zongfu Pan, Wenqiao Chang, Yingying Gong, Tong Xu, Bin Lu, Jiuming He, Minghua Ge, Zhuo Tan, Ping Huang
          </td>
          <td>2025-12-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b4d870dc4ba763551ef81671814fc477f8046f" target='_blank'>
              CancerSTFormer enables multi-scale analysis of spot-resolution spatial transcriptomes and dissects gene and immune regulatory responses to targeted therapies
              </a>
            </td>
          <td>
            Benjamin S Strope, Dana Varghese, William Z Bowie, Stacy Wang, Qian Zhu
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Breast cancer is genetically and histologically heterogenous, and is influenced by a variety of factors, including the tumor microenvironment (TME). The PAM50 subtypes; Luminal A, Luminal B, Normal-like, Basal-like and Her2-enriched, are associated with different tumor phenotypes and overall survival. The quantity and quality of immune cell infiltration in breast tumors play a key role in cancer development and progression and are associated with survival and treatment response. We used multiplex immunohistochemistry, single-cell RNA sequencing, and two deconvolution algorithms to explore the immune landscape across PAM50 subtypes. Immunostaining of CD3+ T cells, tryptase+ mast cells, CD20+ B cells, CD68+ CD163+ macrophages, and CD66b+ granulocytes revealed marked differences in tumor immune infiltrates according to breast cancer subtypes. Luminal tumors were relatively deprived of T cells and B cells, while exhibiting sparse to moderate amounts of macrophage infiltration. In contrast, Her2-enriched tumors exhibited a moderate immune presence, with sparse to moderate T cell infiltration and moderate infiltration of B cells and macrophages. At the other end of the spectrum, Basal-like tumors stood out for their strikingly rich immune environment and are heavily infiltrated by T cells, B cells, and macrophages. The results from the single-cell and deconvolution analyses confirmed subtype-specific immune microenvironments, which also allowed us to observe increased levels of natural killer (NK) and CD8+ T cells in Her2-enriched and Basal-like subtypes. In conclusion, our findings demonstrate significant differences in the immune tumor microenvironment between the established breast cancer molecular subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c99c23b4dfb7da23c9b0f3a691c05aec9a1b68" target='_blank'>
              Multi-platform analysis of the tumor immune microenvironment associated with breast cancer subtypes
              </a>
            </td>
          <td>
            Lilly Anne Torland, Jürgen Geisler, Y. Azimzade, Astri Frafjord, Hogne Røed Nilsen, Linn Buer, J. Lømo, Ø. Garred, Andliena Tahiri, M. Lyngra, Inger Øynebråten, Alexandre Corthay, Kristine Sahlberg, Vessela N. Kristensen, Xavier Tekpli
          </td>
          <td>2026-01-13</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Remodeling of the tumor microenvironment (TME) under therapeutic pressure is a critical determinant of treatment response and resistance in hepatocellular carcinoma (HCC). Triple-combination therapy integrating targeted agents, immune checkpoint inhibitors, and radiotherapy (T+I+R) has shown potential synergistic effects in intermediate to advanced HCC, particularly in patients with portal vein tumor thrombus (PVTT), yet the spatial and cellular mechanisms underlying its efficacy remain largely unknown. In this retrospective clinical cohort study, we compared T+I+R with targeted therapy plus radiotherapy (T+R) in advanced HCC, and further employed single-cell spatial transcriptomics and spatial proteomics to generate an integrated multi-omics atlas mapping tumor and stromal compartments, cellular compositions, and intercellular interactions with spatial resolution. Clinically, T+I+R achieved superior tumor shrinkage and disease control compared with T+R. Spatial multi-omics revealed marked region-specific remodeling, with myofibroblastic cancer-associated fibroblasts, angiogenic tip endothelial cells, and conventional dendritic cells enriched at the invasive margin and associated with therapeutic resistance, while CD8+ effector T cells were redistributed away from immunosuppressive niches, a spatial configuration correlating with enhanced response. These findings identify spatial segregation between cytotoxic and suppressive immune elements as a potential hallmark of effective therapy, providing a high-resolution spatial framework for understanding T+I+R induced TME remodeling and offering mechanistic insights to guide biomarker discovery and the optimization of combination strategies in advanced HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d6c3edb9469068b18d4bde88276d91593b43ce6" target='_blank'>
              Spatial multi-omics mapping of tumor microanatomy dynamics following radiotherapy combined with targeted-immunotherapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Fansen Ji, Haoming Xia, Ying Xiao, Jiawei Zhang, Hao Chen, Bingjun Tang, Huan Li, Hao Liu, Boyang Wu, Xiaojuan Wang, Shizhong Yang, Jiahong Dong
          </td>
          <td>2026-01-08</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00392ec00640d6a4eac41b780bf20f612e7a3c42" target='_blank'>
              TMEM Doorway Mediated Metastasis in Pancreatic Ductal Adenocarcinoma by Tie2 Signaling
              </a>
            </td>
          <td>
            Erika Pereira Zambalde, Lina A. Ariyan, Francisco Puerta-Martinez, Yu Quin Zhu, Priyanka Patil, Nicole C. Panarelli, Jiufeng Li, Yookyung Jung, Christian Adkisson, Jakeb Petersen, Dianne Cox, Hava Gil-Henn, Robert J. Eddy, Ben Z. Stanger, D. Entenberg, M. Oktay, J. Condeelis, John C. McAuliffe
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c89572959306f18148b46202d05fa7473ebfa54b" target='_blank'>
              Mosaic hotspot PIK3CA mutations cause non-cell-autonomous vascular overgrowth and pan-lineage dysregulation at disease onset
              </a>
            </td>
          <td>
            Hannah Brunsdon, Nuoya Wang, Micha Sam, Brickman Raredon, R. Madsen, R. K. Semple, E. Patton
          </td>
          <td>2025-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Bone sarcomas remain lethal despite multimodal therapy, primarily because the mineralized, immunosuppressive tumor microenvironment (TME) promotes chemo- and immune-resistance. Integrating single-cell and spatial omics across osteosarcoma, Ewing sarcoma and chondrosarcoma delineates subtype-specific TME archetypes dominated by M2 macrophages, exhausted T cells and a stiff extracellular matrix. Mechanistic dissection reveals tractable vulnerabilities, myeloid reprogramming, extracellular matrix modulation and metabolic and epigenetic checkpoints, that can be targeted with bone-selective delivery systems and biomarker-driven combination trials to convert therapeutic failure into durable remission. Therefore, the aim of the present review is to synthesize the latest single-cell, spatial and functional data to map bone-sarcoma TME heterogeneity, dissect resistance mechanisms and propose integrated, biomarker-guided therapeutic strategies that can be translated into treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a42d68e5855e4884a22e5af558a0428be6a50022" target='_blank'>
              Tumor microenvironment in bone sarcomas: Implications for immunotherapy and emerging therapeutic vulnerabilities (Review)
              </a>
            </td>
          <td>
            Wentao Li, Lijun Lv, Yibin Jin, Xin Yuan
          </td>
          <td>2026-01-16</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is a lethal subtype of prostate cancer characterized by its androgen independence, resistance to chemotherapy, and display of neuroendocrine features which can emerge either de novo or via transformation after a prior diagnosis of adenocarcinoma. The poor clinical outcomes in patients with AVPC are associated with its profound molecular heterogeneity. In this study, we analyzed 23 consecutive AVPC cases treated at a dedicated small-cell clinic (2017-2025) using clinicogenomic and transcriptomic profiling. Transformed AVPC exhibited significantly shorter overall survival times than de novo AVPC (11.8 vs 26.0 months, P < 0.001). Integrative genomic analyses identified residual androgen signaling in subsets of cases harboring neuroendocrine lineage programs, highlighting a decoupling of lineage identity and morphology. To facilitate mechanistic and pharmacologic studies, we established NCI-LYM-1, a patient-derived organoid/PDX from an AR-negative, ASCL1+/SYP+ lymph node metastasis, which faithfully recapitulates the donor tumor's molecular and phenotypic features. Short- and long-read whole-genome sequencing combined with optical genome mapping identified biallelic inactivation of PTEN, TP53, RB1 and BRCA2 as potential drivers, demonstrating clonal concordance with circulating tumor DNA from the original patient donor. Pathway and perturbation analyses suggested that NCI-LYM-1 harbored a strong dependency on apoptotic pathways, which was confirmed by in vitro organoid testing with the BCL-2/BCL-xL inhibitor navitoclax (IC50: 0.27 M) and the MCL-1 inhibitor AZD-5991 (IC50: 0.060 M). Overall, NCI&-LYM-1 recapitulates the clinical aggressiveness and heterogeneity of AVPC, providing a tractable platform to identify novel precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e71e79208c8d37fdc48221a6ff9b64fb28fe818c" target='_blank'>
              Molecular decoupling of lineage identity and morphology in aggressive variant prostate cancer
              </a>
            </td>
          <td>
            Chennan Li, JuanJuan Yin, Melissa L Abel, Dana S Vargas Solivan, Kinjal Bhadresha, Sumeyra Kartal, Samantha Nichols, Kanak Parmar, Joseph Twohig, Tri M Truong, C. Chau, Kathleen Kelly, W. Figg, Anish Thomas, A. Sowalsky
          </td>
          <td>2026-01-09</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Nodal involvement constitutes a pivotal prognostic indicator in colorectal carcinoma, yet the transcriptional machinery governing lymphatic dissemination and tumor‐microenvironment crosstalk remains poorly elucidated. Conventional bulk sequencing methodologies lack sufficient resolution to deconvolve functionally distinct malignant subclones that drive the metastatic cascade. Methods We employed an integrative analytical framework combining tissue‐level gene expression profiling from TCGA and GEO repositories with eight single‐cell transcriptomic datasets comprising 266,995 individual cells. A phenotype‐guided computational algorithm was implemented to delineate metastasis‐driving malignant populations through correlating clinical parameters with cellular transcriptional profiles. Gene regulatory networks and transcription factor activity inference systematically decoded the molecular programs underlying metastatic phenotypes. Ligand‐receptor pairing analysis mapped intercellular communication architectures between neoplastic cells and microenvironmental constituents. Experimental validation encompassed genetic perturbation studies, functional characterization assays, and pharmacological response evaluation in preclinical systems. Results We discovered a phenotypically distinct malignant population exhibiting robust associations with lymph node involvement and adverse clinical outcomes across nine independent validation cohorts. Regulatory network dissection identified IRF9 as the master transcriptional orchestrator of this metastatic program through coordination of a discrete gene module. Relative to their nonmetastatic counterparts, these aggressive cells establish markedly expanded intercellular communication networks, characterized by prominent VEGF‐driven angiogenic signaling to endothelial compartments and integrin‐laminin–mediated adhesion with stromal elements. EFNA1 emerged as a key signaling mediator demonstrating preferential enrichment in this metastatic subset. Elevated EFNA1 levels correlated with progressive disease stages and microsatellite‐stable subtypes while showing inverse relationships with PD‐L1 expression and T cell infiltration density—suggesting a unique immunoevasion mechanism. Genetic ablation of Efna1 substantially impaired cellular proliferation, motility, and invasion capabilities, while simultaneously augmenting Linifanib sensitivity, indicating synergistic therapeutic potential. Conclusions Our investigation reveals a transcriptionally defined malignant population under IRF9 control that orchestrates immunosuppressive microenvironmental reprogramming via EFNA1‐mediated signaling networks. The EFNA1–Linifanib combination may represent a potential therapeutic approach to mitigate anti‐angiogenic resistance and restrain metastatic progression in colorectal carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/213ee7511b6e61fa3300d628f006d8273257969a" target='_blank'>
              Integrative Multiscale Analysis Reveals EFNA1‐Driven Immune Remodeling Promotes Colorectal Cancer Lymph Node Metastasis
              </a>
            </td>
          <td>
            Wu Ning, Nan Qiao, Lei Zhou, Zongze Li, Lin Zhang, Xin Song, Shaoqiu Chen
          </td>
          <td>2026-01-01</td>
          <td>Human Mutation</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor evolution involves genetic, transcriptional, and phenotypic alterations that shape cancer cell behavior and interactions with the microenvironment. While single-cell technologies have advanced our understanding of this process, spatial dynamics remain incompletely characterized. Here, whole-exome sequencing (WES), imaging mass cytometry (IMC), and spatial transcriptomics (ST) were integrated to study molecular evolution and immune responses in two lung adenocarcinoma (LUAD) mouse models: a genetically engineered model (129S4/Sv-KrasLSL-G12D, termed 129S4 K) and a carcinogen-induced precancer model (129S4 U). Compared to 129S4 K, 129S4 U tumors exhibited higher mutational, neoantigen but lower copy number variation (CNV) burdens at matched developmental timepoints, consistent with findings of higher mutational burden in human smoking-related LUAD than nonsmoking LUAD. We profiled over 1.4 million spatial single cells from 284 IMC regions of interest and 51,531 spatial transcriptomic spots from 156 lesions across 141 mice. Macrophage abundance increased with tumor progression, while CD8 T-cell and B-cell densities declined in late-stage LUAD. 129S4 U showed greater immune infiltration in both tumor and adjacent normal tissue, higher T-cell cytotoxicity signature score in line with its higher mutational and neoantigen burdens. LUAD progression was marked by early morphological shifts and late-stage changes in cell states and interactions. These data define spatial and genetic landscapes of LUAD development and provide a framework for investigating immune evolution and therapeutic strategies in early carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e7ad7c0953d4cab7fb1f9dcb251447c108ead" target='_blank'>
              Spatial Profiling Reveals Distinct Molecular and Immune Evolution of Mouse Lung Adenocarcinoma Precancers with or Without Carcinogen Exposure.
              </a>
            </td>
          <td>
            Bo Zhu, M. Aminu, Pingjun Chen, Jian-Rong Li, Chuanpeng Dong, Chenyang Li, Yanhua Tian, Shao-Wei Lu, Hong Chen, Chenxi Ma, Xin Hu, Jie-Tong Ye, Andrew Y Liu, Beibei Huang, Frank R. Rojas, Parra Cuentas Edwin Roger, Ou Shi, M. Nilsson, A. Poteete, Khaja B Khan, Wei Lu, Luisa M. Solis Soto, Junya Fujimoto, C. Haymaker, Ignacio I. Wistuba, Zhubo Wei, Linghua Wang, Don L. Gibbons, Ken Chen, Alexandre Reuben, Jason M Schenke, J. Heymach, Chao Cheng, Jia Wu, Jianjun Zhang
          </td>
          <td>2026-01-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cef7edb3493fb7ab6d3516e83b3f06b2dd845210" target='_blank'>
              VAETracer: Mutation-Guided Lineage Reconstruction and Generational State Inference from scRNA-seq
              </a>
            </td>
          <td>
            Linnuo Pan, Kaiyu Wang, Luonan Chen
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The progression of prostate cancer (PCa) to metastatic disease remains a critical clinical challenge, making it essential to gain a detailed understanding of the factors that enable certain subclones to invade and spread. Building on our previous study, which examined genomic variations in benign and malignant prostate tissues by inferring copy number alterations (CNA) from spatial transcriptomics data1, this study aims to trace the spatial evolutionary trajectories of PCa subclones from the primary tumor to matched lymph node metastases and to identify alterations in associated microenvironments and early diagnostic markers. To achieve this, we performed spatial transcriptomics on entire prostate axial disks and patient-matched lymph node metastases from 10 individuals. Standard spatial transcriptomics (55 µm) was used for tissue profiling; from the spatial transcriptomics data, copy number alterations (CNAs) were inferred to identify distinct tumor subclones, and a clone-tree was constructed to describe their evolutionary relationships. Additionally, a high-resolution spatial transcriptomics (2 µm) technology was employed to gain a cellular-level view of the immediate tumor-microenvironment (TME) surrounding specific subclones. Analysis of over 1,000,000 barcoded regions identified many distinct tumor subclones, enabling us to trace their evolutionary trajectories spatially. We observed notable subclonal events within the lymph nodes, including polyclonal colonization, indicating multiple dissemination events during the evolution of the primary disease. Exploration of the immediate TME revealed significant cellular heterogeneity and upregulation of genes related to antigen presentation and inflammatory pathways concentrated near ancestral tumor clones, specifically at the tumor border. By focusing on the inferred CNA profiles of the metastasizing clone across patients, we identified several common features. Additionally, we were able to identify potential lethal disease as early as the diagnostic core needle biopsy stage by detecting features of metastasis and pinpointing the metastasizing clone. In summary, our study provides a detailed spatial map of PCa clonal evolution and dissemination, linking primary tumors to nodal metastases and revealing altered cell composition and gene expression around tumor clone borders. Importantly, it demonstrates subclonal events within lymph nodes, polyclonal colonization, and the potential to identify metastasizing clones as early as the diagnostic phase, which holds strong implications for precision oncology and early intervention.
 1. Erickson, A., He, M., Berglund, E. et al. Spatially resolved clonal copy number alterations in benign and malignant tissue. Nature 608, 360–367 (2022). https://doi.org/10.1038/s41586-022-05023-2



 Mengxiao He, Sandy Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, Ian G. Mills, Joakim Lundeberg, Alastair D. Lamb. Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604426087ba878274c47234a7ce00263acf256a4" target='_blank'>
              Abstract PR013: Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers
              </a>
            </td>
          <td>
            Mengxiao He, S. Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, I. Mills, J. Lundeberg, Alastair D. Lamb
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy characterised by remarkable cellular heterogeneity, which emerges early from the interplay of oncogenic KRAS signalling and inflammatory injury. However, the transcriptional, metabolic and functional properties of these pre-malignant cell states that initiate and drive PDAC progression remain elusive. Objective This study aimed to identify and functionally characterise the critical premalignant cell states that arise from this heterogeneity, to define novel biomarkers and targets for early intervention. Design Public and in-house scRNA-seq data of pancreatic tumour models were analysed to identify key subpopulations in early cellular heterogeneity. Genetic perturbation in KrasG12D-driven models was performed to assess functional impact. Mechanistic studies used TurboID proximity proteomics, epigenetic profiling and metabolic assays. Clinical relevance was validated in human PDAC cohorts. Results We identified LY6D as a marker of a distinct, gastric-like cell state that emerges early and persists throughout tumourigenesis. The LY6D+ population exhibits conserved stemness and a unique, pan-stage dependency on oxidative phosphorylation (OXPHOS). Genetic ablation of Ly6d specifically impaired the gastric lineage and delayed tumourigenesis, while its overexpression enhanced tumourigenic and metastatic potential. Mechanistically, the glycosylphosphatidylinositol (GPI)-anchored LY6D protein scaffolds a lipid raft-associated kinase network that drives FOSL1-dependent epigenetic-transcriptional reprogramming. In human PDAC, LY6D+ cells harbour stemness and Epithelial-Mesenchymal Transition (EMT) signatures, and high LY6D expression is an independent prognostic marker of poor survival. Conclusion Our work defines the LY6D+ gastric-like cell state as a key driver linking early pre-malignant heterogeneity to PDAC initiation and progression. LY6D represents a pan-stage therapeutic target and a candidate biomarker for early detection and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f42670138ad6a4407b9d3c018c7d78a840fd287d" target='_blank'>
              LY6D identifies persistent stem-like cells driving pancreatic tumourigenesis
              </a>
            </td>
          <td>
            Juanjuan Shi, Xian Wang, Yingying Tang, Shixin Meng, Zhengyan Zhang, Ping Lu, Junyi Xu, Feier Yu, Xueni Wang, Zheng Wang, Yongwei Sun, Jing Xue
          </td>
          <td>2026-01-16</td>
          <td>Gut</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Breast tumors have traditionally been categorized into four subtypes. Recent single-cell advancements, however, reveal that tumors can contain multiple subtypes, reflecting intrinsic heterogeneity. Relatedly, whilst the crucial role of cell cycle regulation in cancer development is widely acknowledged, the impact of cell cycle phases on oncogenic signaling within these subtypes remains unresolved. Here, we examine differences in transcriptomic hallmarks across and within cell cycle phases and breast cancer subtypes in two large single-cell cohorts. We show that accounting for cell cycle phases reveals a broader pool of hallmark pathways while preserving those identified when phases are ignored. Moreover, we pinpoint FDA approved drugs that can target hallmarks within specific phase-subtype combinations. This study highlights the critical role of cell cycle phases in advancing our understanding of cancer biology. The distinct mechanisms observed here suggest that targeting these hallmarks may offer a more precise and effective strategy for patient treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/680e0867e66e372634861e3e8e8f0639de30e291" target='_blank'>
              Computational decoding of cell-cycle phase effects on cancer hallmarks across breast cancer subtypes
              </a>
            </td>
          <td>
            Miguel Castresana-Aguirre, A. Matikas, Linda S Lindström, N. Tobin
          </td>
          <td>2025-12-24</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor heterogeneity is one of the key factors leading to diagnostic errors, difficulty in identifying subtle cellular differences, and the occurrence of immune escape. Although single-cell RNA sequencing (scRNA-seq) has elucidated the molecular characteristics and heterogeneity at the single-cell level, it fails to retain spatial location information. In contrast, spatial transcriptomics (ST) technology can simultaneously acquire gene expression information and cellular localization, providing a unique perspective for studying the tumor microenvironment, cell-cell interactions, and other aspects. The integrated application of scRNA-seq and ST technologies enables the analysis of tumor heterogeneity from both molecular and spatial dimensions, offering comprehensive information support for precision medicine. The complementary advantages of these two technologies not only overcome the limitations of traditional methods but also promote the research on complex mechanisms such as tumor heterogeneity and the tumor microenvironment, accelerating the translational process of clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9514d53fae21da5a677d774e552caceeac1e73" target='_blank'>
              Research Progress in Single-Cell RNA Sequencing and Spatial Transcriptomics for Precision Oncology
              </a>
            </td>
          <td>
            Minxin Mo
          </td>
          <td>2025-12-18</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Immune cells critically influence pituitary neuroendocrine tumor (PitNET) progression and therapeutic responses. However, their composition and functional dynamics remain unclear. This study aimed to delineate the immune heterogeneity and intercellular communication networks within PitNETs.


METHODS
This study conducted single-cell RNA sequencing (scRNA-seq) on 22 fresh PitNET samples obtained from surgical patients at Beijing Tiantan Hospital between September 2023 and January 2024, with classification based on the expression of key transcription factors: pituitary-specific transcription factor 1 (PIT1), steroidogenic factor 1 (SF1), and T-box transcription factor 19 (TPIT). Following cell type identification, cell-cell communication analysis revealed specific intercellular interactions, which were validated by multiplex immunohistochemistry (mIHC), flow cytometry, and in vitro co-culture assays.


RESULTS
The scRNA-seq analysis revealed significant cellular heterogeneity and a lineage-specific immune landscape, including cancer-associated fibroblasts (CAFs), neutrophils, T cells, natural killer cells, and myeloid cells. Specifically, CAF subset distribution was highly lineage-dependent. Collagen-expressing CAF3 was significantly enriched in the PIT1 lineage compared to both TPIT and SF1 lineages. In contrast, inflammatory CAF5 was predominantly found in the TPIT lineage relative to the PIT1 and SF1 lineages. Furthermore, CD4+ regulatory T (Treg) and T follicular helper (Tfh) cells were significantly enriched in the PIT1 lineage relative to the TPIT and SF1 lineages, as validated by both flow cytometry and mIHC data. Cellular communication analysis revealed notable interactions between Treg/Tfh cells and macrophages/microglia that were mediated by the cytotoxic T-lymphocyte-associated protein 4 (CTLA4)-CD86 pathway. Subsequent mIHC assays confirmed spatial colocalization of Treg cells with macrophages/microglia. Complementing these findings, in vitro co-culture assays demonstrated functional CTLA4-CD86 signaling specifically between Treg cells and macrophages, providing deeper insights into the complex cellular crosstalk within the PitNET tumor microenvironment (TME).


CONCLUSIONS
The three major PitNET lineages exhibited distinct immune cell frequencies, with the PIT1 lineage notably enriched in Treg and Tfh cells, as well as collagen-producing CAF3. Furthermore, a novel immunosuppressive interaction between Treg cells and macrophages, mediated by the CTLA4-CD86 pathway, was identified, suggesting its potential regulatory role within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d53bbb9dbcd32760a571bdb944453b1d4eebc82" target='_blank'>
              Single-cell transcriptomics reveals the landscape of immune phenotypes in pituitary neuroendocrine tumors.
              </a>
            </td>
          <td>
            Zhe Zhang, Hong Su, Jie Yin, Junhua He, Yanan Hu, Peng Li, Pinan Liu
          </td>
          <td>2026-01-09</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity and plasticity. GBM tumors contain neoplastic cells within a fluid spectrum of diverse transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we develop scFOCAL (Single-Cell Framework for -Omics Connectivity and Analysis via L1000), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identify compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states, and leverage this capability to predict combination synergy. We validate the significance of these findings in vitro, ex vivo, and in vivo, and identify a combination of an OLIG2 inhibitor and Depatux-M for the treatment of GBM. Our studies suggest that scFOCAL identifies cell states that are sensitive and resistant to targeted therapies in GBM using a measure of cell and drug connectivity, which can be applied to identify new synergistic combinations. Glioblastoma is characterised by high levels of intratumoural heterogeneity and plasticity, hindering treatment. Here, the authors develop an analytical framework, scFOCAL, to predict the sensitivity of glioblastoma cell subpopulations to therapies based on reversal of disease transcriptional signatures to identify synergistic therapeutic combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef55e3b53c83b7969434910c5b4838c55ce9672" target='_blank'>
              Drug and single-cell gene expression integration identifies sensitive and resistant glioblastoma cell populations
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, R. Komotar, Macarena I. de la Fuente, Gregory Stein, Alexandre Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, Nagi G. Ayad
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Highlights What are the main findings? We established and characterized a unique pair of isogenic cervical cancer cell lines (AdMer35/AdMer43) from a patient with squamous cell carcinoma of the cervix before and after chemoradiotherapy, revealing that resistance is not caused by a single factor but by complex reorganization of cancer cells. Single-cell analysis identified key resistance hallmarks: transcriptional reprogramming towards embryonic stemness (HOX, POU5F1), a shift from fibrillar to non-fibrillar collagen expression, and activation of interferon/inflammatory pathways. What are the implications of the main findings? The study maps a remodeled tumor ecosystem with a metabolically reprogrammed, pro-tumorigenic senescent-like cell population and a progenitor-like cell reservoir, identifying them as critical therapeutic targets for eliminating resistant cell pools. The identified key pathways, Wnt, TGFβ, BMP, and the specific ligands EREG and SEMA3C, along with the metabolic vulnerabilities of the senescent compartment, provide a robust framework for future therapeutic exploration. Abstract Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V. Semenov, Irina S. Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="A central challenge in cancer research is to identify the secreted factors that sustain tumor cell survival. This is best exemplified in Hodgkin lymphoma, where malignant cells constitute a minor fraction of the tumor and rely on signals from the microenvironment for survival. Using genome-wide transcriptional profiling with spatial and single-cell resolution, we show that the neighborhood around malignant cells forms a distinct niche of 31 non-malignant cell types, enriched in helper T cells and myeloid cells, but depleted of plasma cells. Moreover, our spatial analysis nominates IL13 as a candidate survival factor. Recombinant IL13 augments malignant cell growth in vitro, and genome-wide loss-of-function screens across >1000 human cancer cell lines identify IL4R and IL13RA1, heterodimeric components of the IL13 receptor, as uniquely essential in Hodgkin lymphoma. Importantly, blocking antibodies phenocopy genetic inactivation. Our findings provide a biological rationale for testing IL13-directed therapies, which are already FDA-approved, in Hodgkin lymphoma. Hodgkin Reed Sternberg (HRS) cells and their surrounding microenvironment in Hodgkin lymphoma remain poorly characterized. Here, the authors perform genome-wide transcriptional profiling with spatial and single-cell resolution to explore the cellular and molecular composition of the Hodgkin lymphoma microenvironment and used machine learning to identify IL13 as a potential HRS cell survival factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdc2f4fd014e0601edcc3f78406fd8afaf2448ea" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, D. Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson A. Weir, O. Ashenberg, Geraldine S Pinkus, Scott J. Rodig, Caroline Uhler, Evan Z. Macosko, Margaret Shipp, A. Louissaint, Fei Chen, Todd R Golub
          </td>
          <td>2025-12-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Molecular heterogeneity in head and neck squamous cell carcinoma (HNSCC) is well recognized, yet existing subtype frameworks remain largely descriptive and have not translated into therapeutic decision-making. Here, we establish a mechanistic platform that converts transcriptomic diversity into drug-actionable tumor states. Integrating multi-cohort RNA-seq from 727 tumors across five independent datasets, genome-scale CRISPR dependency maps, and pharmacologic screening, we define distinct tumor survival circuits across HPV-negative HNSCC and nominate subtype-matched therapeutic strategies. These circuits encompass a proliferative axis (MYC, MET/FAK, inflammatory and translational programs), an epithelial-differentiated/adhesion program, an EMT-like state with stromal activation, and mitochondrial/oxidative metabolic states, each mapping to selective liabilities (e.g., mitotic/autophagy control, ERBB/PI3K and cadherin signaling, OXPHOS/mitochondrial translation, and G2/M-integrin-Notch pathways, respectively). We then develop a transcriptomic predictor of EGFR-inhibitor response using machine learning and validate it in prospectively collected, fresh patient-derived 3D microtumors. The resulting 13-gene signature identifies erlotinib-responsive tumors (R = 0.93) and maps biologically to an epithelial-differentiated state, outperforming EGFR expression alone. Our study establishes a subtype-to-dependency-to-therapy framework, enabling precision stratification and providing a clinically feasible path for prospective biomarker deployment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29377070a0d6fb125a75d4b7cf89275cb55faab" target='_blank'>
              Subtype-Specific Dependencies and Drug Vulnerabilities Enable Precision Therapeutics in Head and Neck Cancer
              </a>
            </td>
          <td>
            Joel M Vaz, Songli Zhu, Mateo Useche, Lauren Shih, Emily Marchiano, Slobodan Beronja, Bruce E Clurman, Cristina P. Rodriguez, Brittany Barber, Marina Chan, Taranjit S. Gujral
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Gastric cancer (GC), of which cases with peritoneal metastasis are particularly challenging, retains its position of being highly complex and remarkably resistant to therapy. Understanding the spatial heterogeneity and leveraging recent technologies such as machine learning to uncover explanatory patterns remains critical to truly understanding this disease. Here, we conducted spatial transcriptomics analysis to identify distinct niches within GC tissues. Among these, a niche enriched with fibroblasts and macrophages exhibited a striking spatial co-infiltration pattern with tumor cells dominant niches. Further validation by multiplex immunofluorescence highlighted the coordinated cellular interactions that characterize the TME. Through integration of sc-RNA with bulk RNA sequencing, we identified DAB2⁺ TAMs and ACTA2⁺ myCAFs as the main contributors to this co-infiltration pattern. NicheNet analysis further revealed that the PLAU-PLAUR signaling axis holds a central regulatory role in the communication between macrophages, fibroblasts and tumor cells. Given the prognostic value of this spatial pattern, we additionally applied transfer learning based on an ImageNet pre-trained ResNet-50 model to develop a machine learning framework that can accurately recognize the macrophage-fibroblast-malignant cell co-infiltration pattern, called Gastric-Discovery. Potentially, Gastric-Discovery could be a tool for precise patient stratification and provides novel insights into the dynamic architecture of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ca98a4e613e1447cb2daf8bdbf25c8434e308a3" target='_blank'>
              Multi-dimensional omics integrated machine learning framework identifies macrophage-fibroblast-tumor co-infiltration patterns to predict prognosis in gastric cancer
              </a>
            </td>
          <td>
            Qi Wang, Yuan Ni, Sheng Lu, Benyan Zhang, J. Ji, Qu Cai, Chao Yan, Feng Qi, M. Shi, Jun Zhang
          </td>
          <td>2025-12-09</td>
          <td>NPJ Digital Medicine</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c14f73e77a88fcb7e1149959d53563fad60dc97" target='_blank'>
              Liquid Crystal-like Self-Organization of Glioblastoma Prevents Cell Density Induced Migratory Arrest
              </a>
            </td>
          <td>
            Urszula Hohmann, Chalid Ghadban, J. Prell, Christian Strauss, Faramarz Dehghani, T. Hohmann
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Osteosarcoma (OS), the most common malignant bone tumor in adolescents, exhibits marked genetic and cellular heterogeneity, contributing to its resistance to conventional therapies. Although surgical and chemotherapeutic interventions have advanced, survival rates remain stagnant, especially for patients with metastatic or treatment-refractory disease. Recent multi-omic studies have uncovered distinct molecular subtypes characterized by unique genomic, epigenomic, transcriptomic, and tumor microenvironmental signatures. The integration of bulk and single-cell transcriptomics, spatial profiling, epigenetic mapping, and proteomic analyses has provided unprecedented insights into the pathogenesis of OS and has facilitated the identification of novel biomarkers. This review comprehensively summarizes the current advances in the molecular subtyping of OS, highlighting subtype-specific oncogenic drivers, their implications in tumor progression and therapeutic resistance, and the potential for precision medicine strategies tailored to these molecular profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ade8a8b9df4408ea89e61a0e1bd36268d586bf5" target='_blank'>
              Decoding osteosarcoma from heterogeneity to precision therapy
              </a>
            </td>
          <td>
            Yu Xu, Yanfu Bai, Fan Yang, Binbin Yang, Zhisheng Huang, Lingling Su, Kuntao Peng
          </td>
          <td>2025-12-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9e77e33cbcc04a9a109c08426224fcfb14cc985" target='_blank'>
              Hallmarks of adaptive immunity in a metastatic clear cell renal cell carcinoma (ccRCC) long-term elite survivor
              </a>
            </td>
          <td>
            Felicia Tucci, Edward Drydale, James Bancroft, Sakina Amin, Jane Pernes, Giulia Gardumi, Tinashe Kanyowa, Maria Lima, Nadia Nasreddin, Olivia Lombardi, Simon J. Leedham, A. Rendek, Lisa Browning, Emmanuel Bugarin Estrada, Ruddy Montandon, Santiago Revale, H. Slawinski, David Mole, Andy Protheroe, Shivan Sivakumar, R. Bashford-Rogers
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background: Hepatocellular carcinoma (HCC) harbors a dynamic tumor microenvironment (TME) in which macrophages are highly abundant and plastic. Under physiological conditions, macrophages switch between inflammatory and resolution/tissue-repair programs to maintain homeostasis; however, during hepatocarcinogenesis these programs are reprogrammed into tumor-associated macrophage (TAM) states that foster immune suppression, angiogenesis, and tumor progression. Purpose: To summarize macrophage heterogeneity and polarization mechanisms in HCC, and to highlight omics-informed therapeutic opportunities for targeting TAMs and improving precision immunotherapy. Research Design: This review summarizes physiological macrophage polarization and the mechanistic basis of macrophage reprogramming in the hepatocellular carcinoma immune microenvironment, integrating evidence from recent advances in single-cell sequencing, multi-omics, and spatial transcriptomics, with a focus on macrophage subset diversity, key regulatory pathways governing polarization and function, and emerging macrophage-targeted interventions and biomarkers. Results: Recent single-cell and spatial multi-omics studies reveal substantial TAM heterogeneity and plasticity in HCC. Macrophage-targeted strategies—including TAM depletion, phenotypic reprogramming, and exosome-mediated drug delivery—show encouraging preclinical efficacy. Macrophage-associated prognostic models and biomarkers may support individualized immunotherapeutic approaches. Conclusions: Macrophage polarization in HCC represents a dynamic continuum that is essential for homeostasis but is co-opted by tumors to drive immunosuppression and tissue remodeling. Advances in single-cell and spatial multi-omics are redefining TAM subsets and actionable pathways, enabling more rational macrophage-targeted therapies. However, challenges remain in standardizing TAM definitions, identifying robust predictive biomarkers, minimizing off-target effects, and optimizing combinations with immunotherapy. Integrating longitudinal multi-omics with AI-based modeling may help predict macrophage state transitions, guide patient-specific regimens, and advance precision medicine in HCC. Plain language summary Hepatocellular carcinoma is a highly aggressive cancer. Studies have found that immune cells within tumors—particularly macrophages—play a crucial role in cancer development and therapeutic response. Macrophages have two main “personalities”: the M1 type, which can kill cancer cells, and the M2 type, which supports cancer growth. They shift between these states in response to signals from the tumor microenvironment. Both liver-resident Kupffer cells and infiltrating macrophages are involved in this process. Various molecules, such as cytokines, enzymes, and exosomes, influence their polarization. This polarized state of macrophages, in turn, affects key tumor processes such as stemness, proliferation, metastasis, and angiogenesis. In recent years, scientists have been exploring drugs that specifically target macrophage polarization to suppress liver cancer, offering new directions for future therapy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b9c3d3369daa7c8bd747ff210bf91cca4945570" target='_blank'>
              Macrophage Polarization in the Tumor Microenvironment of Hepatocellular Carcinoma: From Mechanistic Insights to Translational Therapies
              </a>
            </td>
          <td>
            Xiaoqing Fu, Mingquan Pang, Zhixin Wang, Haijiu Wang
          </td>
          <td>2025-11-01</td>
          <td>Cancer Control: Journal of the Moffitt Cancer Center</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Somatic mutations accumulate during the first zygotic division and continue throughout an organism’s lifespan. The characteristics and frequency of these mutations are contingent on developmental timing and tissue type, giving rise to somatic mosaicism, defined as the presence of unique genomic alterations across different cells. They serve as endogenous cellular barcodes, enabling detailed reconstruction of cell lineages and clonal dynamics. Although lineage tracing techniques have advanced from early microscopic observation and dye staining to the introduction of artificial barcodes via gene editing, owing to ethical considerations, such genetic manipulations in human developmental research are unavailable. Therefore, spontaneously arising somatic mutations are the most suitable strategy for tracing human lineages. Current approaches can be broadly categorized into two strategies: (i) high-resolution methods, including single-cell clonal expansion or laser-capture microdissection, which construct precise phylogenetic trees based on shared mutation profiles; and (ii) bulk sequencing methods, which infer lineage proximity by comparing variant allele frequencies across samples. As more lineage-tracing studies are being conducted focusing on a wider variety of organs, the integration of such data will make it possible to discover the general principles governing human development. This review highlights how the concept of somatic mutations has been applied across diverse biological contexts and discusses the insights and common principles that can be drawn from these findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b635d70a77d777444fb98a226730074956b4517" target='_blank'>
              Reconstructing developmental lineages: a retrospective approach using somatic mutations and variant allele frequency
              </a>
            </td>
          <td>
            Mahnoor Sajjad, Seong Gyu Kwon
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Angioimmunoblastic T-cell lymphoma (AITL) represents the second most prevalent subtype of peripheral T-cell lymphoma, characterized by a dismal prognosis. However, a systematic exploration of tumor microenvironment (TME) features and their prognostic significance in AITL remains notably deficient. To address this knowledge gap, we conducted spatial transcriptome sequencing (ST-SEQ) and whole-exome sequencing in four AITLs and two noncancerous lymph nodes for discovery purposes, complemented by immunohistochemistry analyses on 37 AITL cases for validation. We identified 14 ST clusters, including five neoplastic clusters, wherein a global shift in B-cell phenotypes and enrichment of myeloid cells were observed. These findings underscore a hallmark of exacerbated inflammation and immune dysregulation within the neoplastic TME. Among the 4 ST-sequenced AITLs, 3 expressed high CD40-CD40LG activity, accompanied by the upregulation of immune-suppressive-associated genes, such as CCL17 and PDCD1. Conversely, the remaining patient displayed an uncommon absence of CD40-CD40LG activity but harbored a phagocytosis-associated tumor-associated macrophage (TAM)-enriched TME, which correlated with significantly reduced relapse rates and longer event-free survival (EFS), highlighting the critical value of precise TME stratification in tailoring AITL therapeutic strategies. Finally, trajectory analysis unveiled a distinct trajectory of molecular evolution within this TME landscape. Collectively, our findings illuminate the heterogeneity and prognostic implications of the TME in AITL, providing a robust foundation for the rational design of targeted immunotherapeutic approaches. These insights may substantially advance the development of personalized treatment strategies for AITL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47887a6371994feccf99a507b06079fcceafb4cb" target='_blank'>
              Revealing tumor microenvironmental heterogeneity and prognostic value in angioimmunoblastic T-cell lymphoma via spatial transcriptome sequencing
              </a>
            </td>
          <td>
            Xiang Zhang, Yong Sun, Duo Wu, F. Yu, Hanjin Yang, X. Ye, Juying Wei, Xuewu Zhang, Yanan Zhu, Yunfei Lv, Zijing Xu, Yuxiang Chen, Hongyan Tong, Jie Jin, Xiaofei Ye, Wenjuan Yu
          </td>
          <td>2026-01-09</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) enables simultaneous mapping of tissue morphology and spatially resolved gene expression, offering unique opportunities to study tumor microenvironment heterogeneity. Here, we introduce a computational framework that predicts spatial pathway activity directly from hematoxylin-and-eosin-stained histology images at microscale resolution 55 and 100 um. Using image features derived from a computational pathology foundation model, we found that TGFb signaling was the most accurately predicted pathway across three independent breast and lung cancer ST datasets. In 87-88% of reliably predicted cases, the resulting spatial TGFb activity maps reflected the expected contrast between tumor and adjacent non-tumor regions, consistent with the known role of TGFb in regulating interactions within the tumor microenvironment. Notably, linear and nonlinear predictive models performed similarly, suggesting that image features may relate to pathway activity in a predominantly linear fashion or that nonlinear structure is small relative to measurement noise. These findings demonstrate that features extracted from routine histopathology may recover spatially coherent and biologically interpretable pathway patterns, offering a scalable strategy for integrating image-based inference with ST information in tumor microenvironment studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29b8964f22d869e7cd99fbb783574fbfe821242b" target='_blank'>
              Digital Modeling of Spatial Pathway Activity from Histology Reveals Tumor Microenvironment Heterogeneity
              </a>
            </td>
          <td>
            Ling Liao, Changhuei Yang, Maxim N. Artyomov, Mark Watson, Adam Kepecs, Haowen Zhou, Alexey Sergushichev, R. Cote
          </td>
          <td>2025-12-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Prostate cancer is genomically heterogeneous with few shared drivers. We characterized tumor-specific DNA methylation programs that define evolutionary subtypes, remodel the tumor–microenvironment ecosystem, and yield liquid-biopsy biomarkers.



 Illumina EPIC-seq methylation was profiled in matched benign/tumor tissues from the UK Prostate ICGC cohort. A covariance-based DMR caller identified robust tumor-specific regions that were intersected with bulk RNA-seq, network analyses, and single-cell/spatial transcriptomics to localize programs in situ. A FACS-sorted, cell-type methylome atlas (luminal, basal, endothelial, fibroblast, immune, smooth muscle) anchored cell-of-origin assignments. Cell-free DNA was deconvolved against this atlas to quantify luminal tumor DNA and circulating microenvironmental DNA (cmDNA). Prognostic associations used penalized Cox models.



 Tumor-specific DMRs separated cancers into two “mevotypes” mirroring genomic evotypes (ERG-fusion vs SPOP-mutant), indicating genomic–epigenomic interplay in evolutionary commitment. Cell-resolved methylomes attributed most tumor DMRs to luminal epithelium and uncovered neuronal/plasticity pathways in luminal cells, while immune/endothelial-assigned DMRs converged on ECM remodeling, angiogenesis, TGF-β, and Rho-GTPase signaling. Single-cell pseudotime and spatial mapping localized neuronal-enriched luminal programs to late-trajectory regions and a CHGA/CHGB-positive duct-adjacent epithelial circuit interfacing with fibroblasts at tumor–stroma borders. Network analysis highlighted MYC-centric modules and EZH2-linked transcription factor contexts, consistent with epigenetic priming toward quasi-neuroendocrine states in treatment-naïve localized adenocarcinoma. In plasma, deconvolution detected both luminal ctDNA and cmDNA; luminal fractions tracked tumor burden in metastatic disease (ichorCNA-concordant), whereas cmDNA—predominantly attributable to immune compartments—suggested an early, tumor-specific microenvironmental response that can precede and is independent of luminal ctDNA shedding. Methylation signatures derived from neuronal and microenvironmental gene sets associated with inferior relapse-free survival.



 Epigenetic drivers in early prostate cancer encode luminal neuronal plasticity and reorganize the tumor niche. A cell-resolved atlas enables cfDNA deconvolution to reveal luminal ctDNA plus cmDNA, supporting liquid-biopsy biomarker strategies for earlier detection and improved risk stratification in prostate cancer.



 Harveer Dev, Toby Milne-Clark, Henson Lee Yu, CRUK-ICGC Prostate Cancer Group. Epigenetic Drivers of Localised Prostate Cancers Reveal Neuronal and Microenvironmental Reprogramming and inform Liquid Biopsy Biomarkers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2503538f0dcd516b3bf974c3bb6eeaf7a1544a19" target='_blank'>
              Abstract A015: Epigenetic Drivers of Localised Prostate Cancers Reveal Neuronal and Microenvironmental Reprogramming and inform Liquid Biopsy Biomarkers
              </a>
            </td>
          <td>
            H. Dev, Toby Milne-Clark, Henson L. Lee Yu
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="HCC is characterized by extensive ECM remodeling, primarily mediated by stromal cells. While cancer-associated fibroblasts are known contributors to tumor fibrosis, their transcriptional diversity and role in immune modulation within the HCC tumor microenvironment remain poorly resolved. Integrative analysis of six public HCC scRNA-seq datasets was employed to identify ECM-active stromal cells. High-ECM cells were re-clustered and fibroblast subtypes were defined through differential expression, pathway enrichment, lineage scoring and cell-cell interaction modeling. Spatial transcriptomic mapping and tissue-level profiling were performed to validate and prioritize ECM-related genes for experimental analysis. COL1A1 and COL3A1 were selected for siRNA-mediated knockdown in HepG2 and HepB3 cells followed by RT-qPCR, Western blot, proliferation, colony formation and wound healing assays. ECM activity analysis identified fibroblasts as the most ECM-enriched stromal population. Reclustering of ECM-high stromal cells identified eight distinct subtypes and targeted fibroblast clustering revealed five functionally diverse states. GO and spatial transcriptomic analysis confirmed subtype-specific functions and localization. In silico tissue profiling further prioritized COL1A1 and COL3A1 as pan-mesenchymal ECM genes enriched in fibrogenic fibroblast subsets. CellChat analysis revealed myofibroblasts and inflammatory CAFs as dominant signal senders. Knockdown reduced COL1A1 and COL3A1 expression at both mRNA and protein levels, and enhanced HCC cell proliferation, migration and colony formation. This study identifies transcriptionally and functionally distinct fibroblast subtypes in HCC and highlights COL1A1 and COL3A1 as key matrix-regulatory genes expressed in fibrogenic stromal subsets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2fd864648cbb63d6bef938be32a938f167aaf27" target='_blank'>
              Single-cell profiling reveals lineage-specific fibroblast stromal subtypes drive ECM remodeling and immune modulation in the hepatocellular carcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Zhichao Jiang, Haitao Wang, Han Li, Zhiqiang Chen, Baiyu Sun
          </td>
          <td>2026-01-03</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6aac86da3cbb3bd85a8ebe57c0ff220d334a71" target='_blank'>
              Circulating immune profiling reveals impaired monocyte states and trajectories driving immunosuppression in glioblastoma
              </a>
            </td>
          <td>
            A. Scafidi, T. Kaoma, Claudia Cerella, Eleonora Campus, K. Grzyb, Bakhtiyor Nosirov, Frida Lind-Holm Mogensen, E. Klein, Raul da Costa, A. Skupin, Frank Hertel, G. Berchem, Michel Mittelbronn, A. Cosma, Beatrice Melin, Simone P. Niclou, Petr V Nazarov, A. Gołȩbiewska, Aurélie Poli, A. Michelucci
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The various cellular composition of the tumor microenvironment (TME) comprises the fundamental units of tumor tissue. The types of stromal cells in the TME are genetically stable, with reduced risk of tumor recurrence and drug resistance. More and more evidence shows their clinicopathological significance and therapeutic effect in predicting prognosis. Therefore, we performed an integrated analysis of the breast cancer TME, correlating it with genomic landscapes and clinical profiles. In this work, we first conducted unsupervised hierarchical clustering on 830 tumors in the breast cancer cohort. Then, we defined three TME phenotypes and applied principal component analysis to construct a TMEscore for quantifying TME. Analysis revealed that patients stratified into the high TMEscore cohort exhibited superior survival compared to the low‐scoring group. Additionally, a high TMEscore is associated with an improved response to immunotherapy. Through TME gene signature analysis, KIF15 was identified as a pivotal driver of the immunosuppressive microenvironment in breast cancer. KIF15 knockdown may promote dendritic cell infiltration and function, thereby inducing CD8+ T cell recruitment. In summary, the immune microenvironment‐derived TMEscore represents an independent prognostic biomarker in breast cancer, while KIF15 emerges as a crucial molecular determinant of its immunosuppressive niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44851ced88ac81dcb9eaccc6f7fcac4727fd09a" target='_blank'>
              Tumor Microenvironment Characterization Identifies KIF15 as an Immunosuppressive Driver in Breast Cancer
              </a>
            </td>
          <td>
            Bo Zhang, Feiran Wang, Huiwei Huang, Xiang Zhong, Qian Chen, Xiancheng Liu, Sujit Nair
          </td>
          <td>2026-01-01</td>
          <td>Human Mutation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Patient-derived organoids (PDOs) have emerged as transformative preclinical models in non-small cell lung cancer (NSCLC), offering high-fidelity recapitulation of tumor heterogeneity and drug responses. Compared to traditional cell lines and xenografts, PDOs preserve the genetic, phenotypic, and functional features of parental tumors, enabling precise drug sensitivity testing for chemotherapy, targeted therapy, and immunotherapy, particularly through optimized culture protocols, genetic engineering techniques, and cryopreservation methods, have significantly enhanced their scalability and clinical relevance. PDOs have proven instrumental in elucidating key resistance mechanisms such as EGFR-TKI resistance mediated through DCLK1-dependent Wnt signaling activation, while simultaneously identifying novel therapeutic synergies for clinical translatio. However, challenges remain in modeling the tumor immune microenvironment and standardizing clinical translation. This review systematically outlines the advancements and challenges in establishing NSCLC PDOs, highlights the potential of PDOs to guide personalized NSCLC therapy while addressing current limitations to bridge the gap between research and clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a0a3f8f865fb80834ba9f16e69c07a2097e28f2" target='_blank'>
              Patient-derived organoids in non-small cell lung cancer: advances in drug sensitivity testing
              </a>
            </td>
          <td>
            Wanyu Tang, Xudong Tian
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5076c3231fa10b8e1b6c94efd5aa380d1b021cf8" target='_blank'>
              Single-Cell Metabolic Profiling in a Glioblastoma Co-culture Model Using AP-MALDI-based Mass Spectrometry Imaging
              </a>
            </td>
          <td>
            U. Kontrimaite, Kei F. Carver Wong, Sandra Martínez-Jarquín, Phoebe McCrorie, Ruman Rahman, Dong-Hyun Kim
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/283f47aefbdff4ebd6faa24c4a2c93b8e125039d" target='_blank'>
              Translating spatial transcriptomic signatures in adenosquamous carcinoma into bulk prognostic biomarkers in lung adenocarcinoma: a bottom-up approach.
              </a>
            </td>
          <td>
            K. Hatakeyama, T. Kawata, Koji Muramatsu, T. Oishi, Takeshi Nagashima, Ai Sakai, Terumi Nishimura, K. Ohshima, Sumiko Ohnami, S. Ohnami, Y. Shimoda, A. Naruoka, Koji Maruyama, K. Urakami, Y. Akiyama, Y. Ohde, Takashi Sugino, Ken Yamaguchi
          </td>
          <td>2026-01-23</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background/Objectives: Achieving complete resection (R0) during secondary cytoreduction (SCR) is a pivotal prognostic factor for patients with platinum-sensitive relapsed ovarian cancer (PSROC). Methods: By integrating single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and functional assays on primary cells, we identified tumor microenvironment (TME) heterogeneity, particularly the presence of distinct subpopulations of cancer-associated fibroblasts (CAFs), as a critical determinant of surgical outcomes for SCR. Results: We characterized a previously unrecognized CAF-C7 subpopulation that was selectively enriched in tumors from non-R0 patients. Mechanistically, CAF-C7 promoted tumor cell migration, suppressed anoikis, and facilitated angiogenesis via the IGF1-IGF1R signaling axis, thereby contributing to multifocal recurrence and reducing the likelihood of complete resection. Importantly, the inhibition of the IGF1-IGF1R pathway effectively attenuated the pro-tumorigenic functions of CAF-C7. Conclusions: These findings uncover a novel cellular driver of unfavorable surgical outcomes in PSROC and suggest promising biomarkers and therapeutic targets for improving patient stratification and treatment in the context of SCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c69a7316901a4f26ec9560008414f1bd2cba29" target='_blank'>
              Single-Cell Analysis Links C7+ Cancer-Associated Fibroblasts with Incomplete Resection in Platinum-Sensitive Relapsed Ovarian Cancer
              </a>
            </td>
          <td>
            Longxia Li, Shangbing Gao, Yilizhati Maimaiti, Lifeng Lin, Xiaoxia Xing, Wei Wu, Yulian Chen, Mei-Chun Cai, Guanglei Zhuang, Rongyu Zang
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Low-grade serous ovarian carcinoma (LGSOC) is a rare, treatment-resistant subtype of ovarian cancer characterized by peritoneal spread. Pressurized Intraperitoneal Aerosol Chemotherapy (PIPAC) delivers chemotherapy directly into the abdomen during serial laparoscopic surgeries, enabling repeated sampling of tumors for longitudinal analysis. Using single-nuclei RNA sequencing on 15 tumor samples across PIPAC cycles in two patients (a responder and a non-responder), we tracked molecular and cellular changes over time. Non-responder tumors showed upregulation of proliferation pathways (E2F, MYC, G2/M), KRAS signaling, epithelial-mesenchymal transition, and unfolded protein response, correlating with resistance. Responder tumors exhibited downregulation of proliferation and stress pathways but activated alternative survival mechanisms (PI3K, Wnt/β-catenin, Notch). Archetype analysis revealed dynamic shifts in metabolic and immune-related subpopulations in the responder tumors, while immunoprofiling showed greater immunosuppression in non-responder tumors. Despite the small cohort, these exploratory observations highlight tumor adaptation and underscore the need for multi-targeted therapies addressing proliferation, stress, survival, and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03dd1cf18442a46d00407d6ee891f98b166026d8" target='_blank'>
              Divergent oncogenic signaling and immune microenvironment changes in low-grade serous ovarian cancer patients undergoing intraperitoneal chemotherapy
              </a>
            </td>
          <td>
            Isaac Bishara, Patrick A. Cosgrove, Sumana Majumdar, Colt A. Egelston, Oscar Colunga Flores, Brad Nakamura, Nora H. Ruel, Paul Frankel, S. Yost, Sue Chang, Alexander Jung, Mihaela Cristea, Mustafa Raoof, Andrea H. Bild, Aritro Nath, T. Dellinger
          </td>
          <td>2025-12-14</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Aggressive variant prostate cancers (AVPC) are a poorly understood subset of prostate cancers characterized by high mortality and extensive molecular heterogeneity. To address a lack of AVPC representative models for investigating underlying molecular and physiological mechanisms, we established NCI-LYM-1, a novel AVPC organoid/patient derived xenograft (PDX) derived from a patient with metastatic prostate cancer with small cell morphology displaying a unique clinical history and defined genomic features. It was hypothesized that this model would be able to accurately characterize biological underpinnings of the patient’s tumor profile and allow for comparative analysis to similar prostate cancers.



 NCI-LYM-1 was developed from a patient-derived AR-negative, ASCL1/Synaptophysin (SYP)-positive lymph node metastasis with small cell morphology. Short-read whole-genome sequencing (WGS) was used to identify genomic alterations and long-read WGS was used to identify epigenetic modifications. RNA-seq was used to characterize gene expression profiles. The resulting genomic, epigenetic, and transcriptomic data from NCI-LYM-1 was compared to molecular profiles from the LuCaP series of PDX models. In-vitro drug sensitivity assays were also conducted using organoid cultures.



 NCI-LYM-1 harbored biallelic inactivation of known neuroendocrine prostate cancer (NEPC) drivers: PTEN, TP53, RB1 and BRCA2, in concordance with patient diagnostic history. In addition, epigenetic evidence supported a consistent phenotype between NCI-LYM-1 and the donor tumor, including greater gene expression of neuroendocrine marker ASCL1 with decreased AR expression. Comparisons between NCI-LYM-1 and LuCaP models further supported transcriptional similarities between the model and a subset of AVPC cancer subtypes. Drug sensitivity assays were also consistent with the patient’s prior treatment exposure, displaying PARP inhibitor sensitivity but a lack of sensitivity to platinum treatments.



 NCI-LYM-1 faithfully recapitulates the phenotypic and genomic characteristics of aggressive metastatic AVPCs, including the display of neuroendocrine features. NCI-LYM-1 will be a useful model system to study molecular frameworks of understudied cancers like those of AVPC, including aspects such as drug sensitivity, disease progression, and lineage plasticity.



 Ana S. Jaramillo, Chennan Li, JuanJuan Yin, Melissa L. Abel, Dana S. Vargas, Kinjal Bhadresha, Sumeyra Kartal, Joseph Twohig, Tri M. Truong, Anish Thomas, Cindy H. Chau, Kathleen Kelly, William D. Figg, Adam G. Sowalsky. NCI-LYM-1 molecularly and phenotypically recapitulates aggressive variant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483313e79c5e529d10092ffd1001e71580f4403a" target='_blank'>
              Abstract A025: NCI-LYM-1 molecularly and phenotypically recapitulates aggressive variant prostate cancer
              </a>
            </td>
          <td>
            Ana S. Jaramillo, Chennan Li, JuanJuan Yin, Melissa L Abel, Dana S. Vargas, Kinjal Bhadresha, Sumeyra Kartal, Joseph Twohig, Tri M Truong, Anish Thomas, C. Chau, Kathleen Kelly, W. Figg, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Routine histology contains rich prognostic information in stage II/III colorectal cancer, much of which is embedded in complex spatial tissue organisation. We present INSIGHT, a graph neural network that predicts survival directly from routine histology images. Trained and cross-validated on TCGA (n=342) and SURGEN (n=336), INSIGHT produces patient-level spatially resolved risk scores. Large independent validation showed superior prognostic performance compared with pTNM staging (C-index 0.68-0.69 vs 0.44-0.58). INSIGHT spatial risk maps recapitulated canonical prognostic histopathology and identified nuclear solidity and circularity as quantitative risk correlates. Integrating spatial risk with data-driven spatial transcriptomic signatures, spatial proteomics, bulk RNA-seq, and single-cell references revealed an epithelium-immune risk manifold capturing epithelial dedifferentiation and fetal programs, myeloid-driven stromal states including $\mathrm{SPP1}^{+}$ macrophages and $\mathrm{LAMP3}^{+}$ dendritic cells, and adaptive immune dysfunction. This analysis exposed patient-specific epithelial heterogeneity, stratification within MSI-High tumours, and high-risk routes of CDX2/HNF4A loss and CEACAM5/6-associated proliferative programs, highlighting coordinated therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0506faf43ebc888bee635ffd236772260977faab" target='_blank'>
              INSIGHT: Spatially resolved survival modelling from routine histology crosslinked with molecular profiling reveals prognostic epithelial-immune axes in stage II/III colorectal cancer
              </a>
            </td>
          <td>
            Piotr Keller, Mark Eastwood, Zedong Hu, A. Selten, Ruqayya Awan, Gertjan Rasschaert, Sara Verbandt, V. Popovici, Hubert Piessevaux, Hayley T Morris, Petros K Tsantoulis, Thomas Mckee, A. D'Hoore, Cédric Schraepen, X. Sagaert, G. Hertogh, S. Tejpar, Fayyaz Minhas
          </td>
          <td>2025-12-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background: Tertiary Lymphoid Structures (TLS) are organized aggregates of immune cells that form in non-lymphoid tissues in response to chronic inflammation, including cancer. In melanoma, TLS presence within the tumor microenvironment is associated with a pre-existing anti-tumor immunity and linked to improved immunotherapy responses. However, their role in predicting recurrence, particularly in early-stage melanoma, remains unclear.Methods: We conducted a retrospective analysis of archival primary tumor samples from 65 patients with stage I-II melanoma and negative sentinel lymph node (Table 1). Tumor immune infiltrates and TLS were initially assessed using four-marker immunohistochemistry (IHC) panels targeting CD20, CD4, CD8, and Podoplanin (PDPN), a marker of stromal cells. TLS were objectively identified using the HDBSCAN AI-based clustering algorithm. Tumor samples were further analyzed for high-dimensional phenotyping and spatial analysis using a 17 markers panel with hyperplexed immunofluorescence imaging (HIFI). Associations between TLS features and clinical outcomes were evaluated.Results: IHC revealed that 88% of patients had peritumoral T cell infiltration, and TLS were detected in 54% of cases. TLS-positive patients had improved disease-free survival (DFS, p=0.2) and overall survival (OS, p=0.2) (Figure 1A, B). To investigate further, HIFI was performed on 8 patients (4 recurrent, 4 non-recurrent), revealing 13 distinct cell types: stromal (n=6), tumor (n=2), and immune (n=5) (Figure 1C). Spatial analysis identified 9 recurrent spatial modules (SM), each representing a distinct pattern of interaction between cell types. One SM, enriched for PD-L1⁺ stromal cells, was significantly associated with recurrence (p=0.019) (Figure 1D).Conclusions: TLS presence associates with improved DFS and OS in early-stage melanoma. However, beyond their mere presence, TLS spatial organization and local stromal-immune context could influence clinical outcomes. Our data demonstrated that a specific spatial module—defined by the proximity of PD-L1⁺ stromal cells—is associated with disease recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/978d2823a740581a2a5f2a0fdbf9de6949d49e50" target='_blank'>
              13 | Spatial contexture of tertiary lymphoid structures as a predictor of recurrence in early-stage melanoma
              </a>
            </td>
          <td>
            Italian Melanoma Intergroup
          </td>
          <td>2025-12-05</td>
          <td>Dermatology Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains incurable despite advances in treatment options. Although tumor subtypes and specific DNA abnormalities are linked to worse prognosis, the impact of immune dysfunction on disease emergence and/or treatment sensitivity remains unclear. We developed an Immune Atlas of MM by generating profiles of 1,397,272 single cells from the bone marrow (BM) of 337 newly diagnosed participants and characterized immune and hematopoietic cell populations. Cytogenetic risk-based analysis revealed heterogeneous associations with T cells of BM, with 17p13 deletion showing distinct enrichment of a type 1 interferon signature. The disease progression-based analysis revealed the presence of a proinflammatory immune senescence-associated secretory phenotype in rapidly progressing participants. Furthermore, signaling analyses suggested active intercellular communication involving a proliferation-inducing ligand and B cell maturation antigen, potentially promoting tumor growth and survival. Lastly, using independent discovery and validation cohorts, we demonstrated that integrating immune cell signatures with known tumor cytogenetics and individual characteristics significantly improves stratification for the prediction of survival. Pilcher et al. present a single-cell transcriptomics-based immune atlas of participants with newly diagnosed multiple myeloma, reporting on the association of cell types, gene expression and intercellular interactions with disease progression phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f88dd8011aa8686433c4d02b3a0e4dfd59a9255" target='_blank'>
              A single-cell atlas characterizes dysregulation of the bone marrow immune microenvironment associated with outcomes in multiple myeloma
              </a>
            </td>
          <td>
            William C. Pilcher, Lijun Yao, Edgar Gonzalez-Kozlova, Yered H. Pita-Juárez, Dimitra Karagkouni, Chaitanya R. Acharya, Marina E. Michaud, Mark Hamilton, Shivani Nanda, Yizhe Song, Kazuhito Sato, Julia Wang, Sarthak Satpathy, Yuling Ma, Jessica Schulman, D. D’souza, R. Jayasinghe, Denis J Ohlstrom, Katherine E. Ferguson, Giulia Cheloni, Mojtaba Bakhtiari, Nick Pabustan, K. Nie, J. Foltz, Isabella Saldarriaga, Rania Alaaeldin, Eva Lepisto, Rachel Chen, M. Fiala, , April Cook, Junia Vieira Dos Santos, Chiang I-ling, Igor Figueiredo, J. Fortier, Michael Slade, Stephen T. Oh, Michael P. Rettig, Emilie I. Anderson, Ying Li, Surendra Dasari, Michael A. Strausbauch, Vernadette A. Simon, , I-ling Chiang, N. Kalavros, Jennifer Rogers, T. Dawson, Brian H. Lee, G. Kelly, Laura A Walker, Nicolas F. Fernandez, John Leech, Jarod Morgenroth-Rebin, K. Angeliadis, Matthew A. Wyczalkowski, Song Cao, Omar Ibrahim, Roderick Lin, T. Fehniger, Andrew J. Houston, E. Radkevich, , Zhihong Chen, A. Laganà, John F. DiPersio, J. Rosenblatt, S. Kim-Schulze, S. Lonial, Shaji Kumar, Swati S. Bhasin, T. Kourelis, M. Dhodapkar, Ravi Vij, D. Avigan, H. Cho, George Mulligan, Li Ding, Sacha Gnjatic, Ioannis S. Vlachos, Manoj K. Bhasin
          </td>
          <td>2026-01-01</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="The formation of intratumoral spatial niches has been reported for many human malignancies. However, the translational potential of such spatial niches is understudied. Herein, we utilize digital spatial profiling (DSP) to explore the prognostic relevance of spatially defined protein expression in high-risk prostate cancer. A total of 49 patient samples were analyzed for the expression of 46 proteins in 463 regions of interest (ROIs) from the tumor center (n = 198) and the tumor periphery (n = 265) resulting in 21,298 primary data points (mean per patient n = 9.4). Expression data from either the tumor center or the tumor periphery were not found to be prognostic. Protein expression of tumor center and periphery was then integrated into single datapoints by calculating the log2-transformed relative expression between the two niches for each protein and patient. Unsupervised hierarchical clustering of these data yielded two distinct patient subgroups. These clusters did not show a statistically significant correlation with known prognostic parameters yet significantly correlated with progression-free survival (p = 0.014, log-rank, HR 0.43; 95% CI, 0.22–0.86). Our results thus reveal that spatial protein expression contains prognostic information, however, only when expression data from both spatial niches are taken into account. In conclusion, our proof-of-concept study shows that DSP can be exploited for the development of novel prognostic biomarkers that rely on spatially resolved protein expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c6b3a4672d8a49cdbb7af4005dce769ab547c41" target='_blank'>
              Prognostic impact of spatial niches in prostate cancer
              </a>
            </td>
          <td>
            Felix Schneider, Sarah Böning, Beatriz Coelho Antunes, A. Kaczorowski, M. Görtz, V. Schütz, Johannes Huber, Albrecht Stenzinger, Markus Hohenfellner, S. Duensing, A. Duensing
          </td>
          <td>2026-01-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Abstract Background and objectives Recent molecular landscape studies characterizing lung cancer mostly utilize single tumor regions, ignoring tumor heterogeneity and evolutionary patterns. To address this issue, the TRAcking NSCLC Evolution through therapy (TRACERx) Consortium established a multiregion sampling of Western lung cancer to investigate its longitudinal evolutionary dynamics. Due to ancestral differences, there are still major gaps in understanding oriental lung cancer. Methods We have established a prospective cohort of patients with non-small cell lung cancer (NSCLC) named Chinese lung cancer evolution and microenvironment tracking under therapy (CLEVER) study. Discussion By acquiring comprehensive genetic variation data and clonal events from multiple regions of the tumor, we aim to decipher the molecular evolution code of Chinese lung cancer. The analysis of cell composition and spatial structure further characterizes the remodeling effects of tumor evolution and neoadjuvant therapy on the tumor microenvironment, helping to identify important markers that influence treatment effectiveness and recurrence. By monitoring mutations using a personalized minimal residual disease (MRD) panel from tumors at baseline, we can trace evolutionary significance and clonality, aiding in therapy decisionmaking. The CLEVER study significantly contributes to the development of personalized treatment strategies for lung cancer patients, enhancing outcomes and tailoring interventions to individual patient profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75f2f25eef8688466fb037f4fd6a8d16c5aeeb96" target='_blank'>
              Protocol for Chinese lung cancer evolution and microenvironment tracking under therapy study
              </a>
            </td>
          <td>
            Wenxiang Wang, Yue He, Xiaoqiu Yuan, Lin Weng, Jing Bai, Xu Liu, Li Han, Jun Xu, Shuai Wang, Shanshan Lu, Jian Bai, Hui Quan, Qingyun Liu, Xinrui Li, Yuxuan Xiao, Yun Wang, Qingna Zhang, Na Zhou, Yanyan Hou, Hao Li, Y. Li, Fan Yang, Jun Wang, Kezhong Chen
          </td>
          <td>2025-12-01</td>
          <td>Journal of Translational Internal Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Thymic epithelial tumors (TETs) exhibit marked heterogeneity that challenges clinical management and prognostic evaluation. By consensus clustering of immune-related gene expression from public datasets and an in-house validation cohort, we identified two immune subtypes: a lymphocyte-rich subtype (LRS, n = 86) characterized by strong T-cell activity and favorable prognosis, and a myeloid/stromal-rich subtype (MSRS, n = 33) defined by immunosuppressive features, stemness, oncogenic alterations, and poorer outcomes. An APC gene-based classification panel demonstrated robust performance in distinguishing these subtypes, as confirmed by ROC curve analysis and validated in the Ruijin cohort using FFPE-RNA sequencing and multiplex immunofluorescence. Single-cell transcriptomics further delineated subtype-specific tumor microenvironment features and revealed that MSRS tumors suppressed CD8+ T-cell function through MIF-mediated mechanisms. Collectively, these findings establish a clinically relevant immune classification of TETs that integrates molecular, immunological, and clinical features, providing insights into tumor heterogeneity and supporting the development of personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f6f13a57b79f0fdbdce54ed88ac9cc1b39fa61" target='_blank'>
              Immune landscape-driven subtyping reveals distinct microenvironment and prognostic profiles in thymic epithelial tumors
              </a>
            </td>
          <td>
            Yajie Zhang, Tong Lu, Yeke Huang, Xipeng Wang, Xiaoyan Chen, Wei Guo, Youqiong Ye, Hecheng Li
          </td>
          <td>2025-12-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Copy number alterations (CNAs) are hallmarks of cancer, yet investigation of their oncogenic role has been hindered by technical limitations and missing model systems. Here we generated a genome-wide DNA methylation and CNA atlas of 106 genetic mouse models across 31 pediatric tumor types, including 18 new models for pediatric glioma. We demonstrated their epigenetic resemblance to human disease counterparts and identified entity-specific patterns of immune infiltration. We discovered that mouse tumors harbor highly recurrent CNA signatures that occur distinctly based on the tumor subgroup and driving oncogene and showed that these CNAs share syntenic regions with the matching human tumor types, thereby revealing a conserved but previously underappreciated role in subgroup-specific tumorigenesis that can be analyzed using the presented models. Our study provides insights into globally available mouse models for pediatric solid cancers and enables access to functional CNA interrogation, with the potential to unlock new translational targets in pediatric cancers. DNA methylation and copy number variant analyses across a large number of genetic mouse models of pediatric brain tumors reveal subtype-specific molecular alterations shared with the corresponding human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e47a668958565050b674aeca2b9f77b67dc49b4" target='_blank'>
              Investigation of a global mouse methylome atlas reveals subtype-specific copy number alterations in pediatric cancer models
              </a>
            </td>
          <td>
            Melanie Schoof, T. Zheng, M. Sill, R. Imle, Alessia Cais, L. Altendorf, Alicia Fürst, Nina Hofmann, K. Ernst, Dominik Vonficht, Kenneth Chun-Ho Chan, T. Holland-Letz, Andreas Postlmayr, Ryo Shiraishi, Wanchen Wang, A. Morcavallo, Michael Spohn, C. Göbel, J. Niesen, Levke-Sophie Peter, Franck Bourdeaut, Zhi-Yan Han, Yanxin Pei, Najiba Murad, F. Swartling, Jessica Taylor, Monika Yadav, Garrett R Gibson, Richard J Gilbertson, M. Dottermusch, Rajanya Roy, K. Kerl, R. Glass, Jiying Cheng, M. Horstmann, G. Wolters-Eisfeld, Haotian Zhao, D. Sturm, Viveka Nand Yadav, L. Chesler, Simon Haas, William A. Weiss, P. Northcott, Lena M. Kutscher, A. G. Guerreiro Stucklin, O. Ayrault, J. Neumann, Daisuke Kawauchi, David T. W. Jones, K. Pajtler, A. Banito, S. Pfister, U. Schüller, Marc Zuckermann
          </td>
          <td>2025-12-11</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832c2464c80a294ed1bb7a8527b6613a4dd9fa1e" target='_blank'>
              Spatial gene expression analysis reveals drivers of extremely early lymph node metastasis in breast cancer.
              </a>
            </td>
          <td>
            Satoi Nagasawa, Keiko Kajiya, Erina Ishikawa, A. Kanai, Ayako Suzuki, A. Motoyoshi, Tsuguo Iwatani, Manabu Kubota, Masaru Nakamura, T. Onishi, Akiyoshi Hoshino, Ichiro Maeda, Akihiko Morozumi, Kenji Takatsuka, Junki Koike, M. Seki, K. Tsugawa, Yutaka Suzuki
          </td>
          <td>2026-01-23</td>
          <td>NPJ breast cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271f2b42923ccb0c522e0defc7aee243eaf5e39d" target='_blank'>
              Reconstructing clone-resolved transcriptional programs from bulk tumor sequencing
              </a>
            </td>
          <td>
            Jiaying Lai, Yi Yang, Kathleen Noller, YunZhou Liu, Archana Balan, Prathima B. Nagendra, Luciane T. Kagohara, E. Fertig, Laura D. Wood, Rachel Karchin
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="


 PSMA expression is heterogeneous across advanced prostate cancer, limiting the durability of PSMA-radioligand therapy (RLT). To characterize tumor heterogeneity and identify determinants of RLT resistance, we established the SHAPE cohort (Spatial Heterogeneity Atlas of Prostate Cancer Evolution). SHAPE is a multimodal resource integrating PET imaging, clinical treatment history, histopathology, immunohistochemistry, spatial transcriptomics, and genomic profiling across 46 patient tumor samples collected through rapid autopsy, biopsy, and surgical resection. The cohort spans adenocarcinoma, castration-resistant (CRPC), and neuroendocrine (NEPC) prostate cancer states and includes eight xenograft models.



 Twenty-five samples profiled using the 10x Genomics Visium spatial transcriptomics platform, comprising 195,905 spots, were integrated and analyzed using Seurat (v5) in R. Targeted genomic profiling was performed using the Dana-Farber Cancer Institute OncoPanel platform on 36 samples derived from 7 patients and 2 animal models. Among the cohort, seven cases had prior Lu-PSMA exposure, and ten cases were untreated. Eleven clinical and preclinical samples had PSMA PET/CT, including six with both pre- and post-treatment imaging. Data integration connected structural genomic events with spatial lineage programs and imaging features.



 RLT–treated tumors exhibited a copy-number–dominant profile compared with untreated tumors. Recurrent alterations enriched in treated tumors included PTEN deletion (p<0.01), KLLN deletion (p<0.01), AURKA gain (p<0.01), and ZNF217 gain (p<0.01). Treated tumors also showed higher per-sample copy number loss (47.6 vs 37.5, p<0.05) and amplification burden (7.91 vs 2, p<0.05). In contrast, untreated tumors displayed more single nucleotide variants (14.9 vs 6.3, ns) and indels (4.7 vs 1.9, ns). Spatial transcriptomics provided complementary insights. Treated samples with PSMA SUV below the liver reference demonstrated higher copy-number loss burden (p<0.05) and contained unique transcriptomic clusters enriched for glucose metabolism. These clusters displayed variable expression of classical NEPC markers but consistently upregulated genes involved in glucose uptake and utilization. In preclinical studies, head-to-head FDG and Ga-PSMA imaging of WCM12 xenografts demonstrated that FDG, as a surrogate marker of glucose uptake, was significantly increased following relapse after PSMA-RLT (FC, p<0.05).



 The SHAPE cohort integrates genomic, spatial, and imaging data to characterize tumor heterogeneity in advanced prostate cancer. Genomic profiling identified PTEN/KLLN loss and AURKA/ZNF217 gain as recurrent alterations enriched in Lu-PSMA–exposed tumors, while spatial transcriptomics revealed PSMA-low metastases with distinct. clusters enriched for glucose metabolic programs. Together, SHAPE can inform rational strategies for combination or alternative targeted therapies.



 Martin K. Bakht, Jacob Egelberg, Yasutaka Yamada, Louise Clark, Xiao Wei, Varadha Balaji Venkadakrishnan, Anthony P. Belanger, Alice Bernard-Tessier, Sarah J. Hill, Francesca Khani, David J. Einstein, Mary-Ellen Taplin, Eliezer Van Allen, Heather A. Jacene7, Himisha Beltran. Spatial heterogeneity atlas of prostate cancer evolution (SHAPE): Spatial and genomic landscapes of advanced prostate cancer with and without Lu-PSMA therap [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912633affb19287de4d61286123b6004f19890a4" target='_blank'>
              Abstract PR027: Spatial heterogeneity atlas of prostate cancer evolution (SHAPE): Spatial and genomic landscapes of advanced prostate cancer with and without Lu-PSMA therapy
              </a>
            </td>
          <td>
            Martin K. Bakht, Jacob Egelberg, Yasutaka Yamada, Louise Clark, Xiao X Wei, V. B. Venkadakrishnan, Anthony P. Belanger, Alice Bernard-Tessier, Sarah J. Hill, F. Khani, David J Einstein, M. Taplin, E. V. Van Allen, Heather A. Jacene, H. Beltran
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="OBJECTIVE
Glioma recurrence severely impacts patient prognosis, with current treatments showing limited efficacy. Traditional methods struggle to analyze recurrence mechanisms due to challenges in assessing tumor heterogeneity, spatial dynamics, and gene networks. Single-cell combined spatial transcriptomics (ST) offers innovative solutions.


METHODS
We analyzed glioma mRNA data from TCGA and single-cell and ST data from GEO. Following quality control, dimensionality reduction, clustering, and cell annotation of single-cell sequencing data, we identified cell types exhibiting significantly aberrant distributions between primary and recurrent samples by analyzing the deviation degree of Ro/e values. Fibroblasts demonstrating the greatest intergroup differences were subsequently selected as the key cellular population for further investigation. Key differentially expressed genes (DEGs) were identified via random survival forest analysis. Drug sensitivity was assessed using GDSC. Deconvolution algorithms mapped cellular spatial distribution, while PROGENy quantified pathway activity. MISTy modeling revealed cell-cell interactions.


RESULTS
Fibroblasts were the primary recurrence-associated subpopulation, with marker genes enriched in extracellular matrix and adhesion pathways. AEBP1, ZNF708, and TSHZ2 were identified as key genes: AEBP1/TSHZ2 correlated with poor prognosis, while ZNF708 showed an inverse trend. These genes were linked to chemosensitivity (Irinotecan, Carmustine, Vincristine, and Cisplatin). Recurrent tumors exhibited increased plasma cell infiltration, with key genes regulating IL-17, Notch, and Toll-like receptor pathways. Spatial analysis highlighted oligodendrocyte-astrocyte interactions in the tumor microenvironment.


INTERPRETATION
Fibroblasts drive glioma recurrence, with AEBP1, ZNF708, and TSHZ2 predicting recurrence and chemoresistance. These genes promote immune suppression (via plasma cells) and activate recurrence pathways. Oligodendrocyte-astrocyte interactions shape the recurrent microenvironment, suggesting new therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec7c524a9377c361243faed8c72c15c26a0c97d" target='_blank'>
              Unraveling the Molecular Mechanisms of Glioma Recurrence: A Study Integrating Single-Cell and Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Lei Qiu, Yinjiao Fei, Jiaxuan Ding, Kexin Shi, Jinyan Luo, Yuchen Zhu, Xingjian Sun, Gefei Jiang, Yuandong Cao, Weilin Xu, Shu Zhou
          </td>
          <td>2026-01-06</td>
          <td>Annals of clinical and translational neurology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Rising incidence of early-onset prostate and other solid tumors underscores the need for experimental systems that model how normal tissues traverse premalignant states, acquire mutations, and become therapy-responsive malignancies under authentic immune and stromal pressures. A major barrier has been the lack of tractable in vivo platforms that enable genome-wide discovery while preserving continuous tumor evolution without catastrophic chromosomal instability. To address this, the Bose Lab developed Stochastically Emergent Tumors (SETs), an organoid-derived in vivo evolution engine that redefines discovery for early-onset and understudied patient groups. In this system, mismatch-repair deficiency is induced in non-malignant human organoids, which are passaged to accumulate stochastic point mutations and transplanted into mice to permit malignancy to emerge under physiologic selection. SETs evolve primarily through high-resolution point mutations rather than broad copy-number changes, yielding bioinformatically tractable clonal dynamics ideally suited for whole-genome driver discovery and machine learning. Compared with conventional xenografts, SETs display greater intertumoral heterogeneity and reproducible recovery of sensitizing and resistance alleles under therapeutic pressure. As proof of principle in prostate cancer, endocrine therapy applied to SET pools recovered known determinants of androgen-pathway sensitivity and uncovered new drivers. Loss of ZFHX3, typically obscured within a multigene suppressor locus in bulk cohorts, promoted luminal histology and sensitized tumors to androgen-receptor inhibition in vivo, whereas KMT2D or CIC alterations mediated resistance. Consistent with model predictions, ZFHX3 loss in patients correlated with significantly improved survival, a finding comparable in magnitude to the most favorable molecular subtypes of advanced prostate cancer. SETs also quantify evolutionary thresholds: in a Pten-null background, approximately 900 coding mutations accumulated over 208 days were sufficient for malignant transformation in half of grafts. This defines a measurable axis linking mutation burden, genotype, and tumor incidence. Because SETs generate neoantigen-rich point-mutation landscapes, they can be extended to immunocompetent hosts to study tumor–immune coevolution, early T-cell surveillance, macrophage-mediated immune exclusion, and myeloid checkpoints that enable immune escape. In summary, SETs provide a scalable, evolution-aware platform that connects mutational dynamics to therapeutic vulnerability, enabling identification of lineage- and ancestry-associated drivers, immunopreventive targets, and biomarkers of early-onset prostate cancer.



 Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, Valbona Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C.K. Cornelia Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, Charles Sawyers, Felix Feng, Hani Goodarzi, Rohit Bose. Reconstructing prostate evolution with Stochastically Emergent Tumors (SETs) reveals in vivo therapeutic vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23678ab77681ac1f2c46c1e6d9b9824f918c99b" target='_blank'>
              Abstract A008: Reconstructing prostate evolution with Stochastically Emergent Tumors (SETs) reveals in vivo therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, V. Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C. Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R Karthaus, Charles Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive primary malignant tumor of the central nervous system (CNS), characterized by marked molecular heterogeneity, rapid proliferation, extensive invasiveness, and strong resistance to conventional therapies. Despite current standard treatments–surgery, radiotherapy, and temozolomide–median patient survival remains below 15 months. Key contributors to GBM progression include activated glial cells and remodeling of the extracellular matrix. In this study, we investigated the cellular and molecular dynamics of the tumor microenvironment, focusing on astrocytic and microglial responses, and on the modulation of glial markers such as GFAP, Iba1, Connexin 43 (Cx43), and the purinergic receptor P2X4R. Acute and organotypic brain slice cultures were prepared from C57BL/6J mice, into which GL261 glioblastoma cells were injected into the cortex. Tissues were collected 3 and 7 days post-injection to analyze tumor progression. GBM growth was monitored through the proliferation marker Ki67. Histological analysis, confocal immunofluorescence, and western blotting were performed to assess marker expression and localization. GL261 cells engaged with the surrounding neuroglial network, migrating in fascicle-like patterns that extended even into the contralateral hemisphere. This dynamic infiltration was associated with pronounced peritumoral astrocyte activation (increased GFAP) and a robust microglial response, evidenced by morphological changes in Iba1-positive cells. Cx43 expression was altered by tumor invasion, suggesting disrupted intercellular communication, while P2X4R expression dynamics indicated modulation of the local immune response. Consistently, elevated Ki67 levels confirmed the highly proliferative and aggressive behavior of the tumor. Organotypic slice cultures represent a fast and reliable ex vivo platform to investigate GBM invasiveness, migration, and neuroglial interactions, revealing spatiotemporal remodeling of the tumor microenvironment during disease progression. Region-specific glial activation and time-dependent modulation of Cx43 and P2X4R highlight the complex and evolving nature of glial–tumor crosstalk. These findings may help identify potential therapeutic windows and support the development of molecularly targeted treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7731564feaeab85e0322e27b0f7ea6db2c5e59" target='_blank'>
              UNRAVELING GLIAL REMODELING IN SPACE AND TIME DURING GLIOBLASTOMA EVOLUTION
              </a>
            </td>
          <td>

          </td>
          <td>2025-12-12</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma is a lethal brain cancer marked by functional plasticity driven by tumor cell-intrinsic mutations and their interplay with developmental programs. To investigate how canonical glioblastoma mutations promote functional plasticity, we have developed an isogenic human neural stem cell (NSC) model of glioblastoma by sequential addition of TERT promoter, TP53, and PDGFRA point mutations. TP53 loss-of-function increases TERT expression during serial mutagenesis, but only triple mutant NSCs reliably form lethal brain tumors in vivo that recapitulate glioblastoma. Tumor cell evolution triggers stress-related metabolic changes and transitions toward a neuronal progenitor network driven by transcription factor INSM1. INSM1 is highly expressed in human glioblastoma tumors and, during cortical development, in intermediate progenitor cells, which give rise to neurons. Remarkably, INSM1 knockdown in triple mutant NSCs and primary glioblastoma cells disrupts oncogenic gene expression and function and inhibits the in vivo tumorigenicity of triple mutant NSCs, highlighting the functional importance of an intermediate progenitor cell-like cell state in glioblastoma pathogenesis. Glioblastoma (GBM) is characterized by a high degree of heterogeneity and plasticity due to interplay with neural developmental programs. Here, the authors develop a model of GBM by introducing sequential oncogenic mutations in human neural stem cells and using this, identify INSM1 as a driver of a neural progenitor gene network promoting tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ee12cd01ff0dc670471af7bea5ea93e2f7413e0" target='_blank'>
              INSM1 governs a neuronal progenitor state that drives glioblastoma in a human stem cell model
              </a>
            </td>
          <td>
            Patrick A. DeSouza, Matthew Ishahak, Xuan Qu, Colin R. McCornack, Devi Annamalai, Diane D. Mao, Rajanikanth Vangipurapu, Yiwei Fu, Alexandre T. Vessoni, Ryan Cleary, Rowland H Han, Punn Augsornworawat, Timothy Woodiwiss, Darby Agovino, Braxton Sizemore, Jessica Kline, Maryam Borhani-Haghighi, Hao Chen, Sangami Pugazenthi, Hiroko Yano, Ting Wang, Luis F Z Batista, J. Millman, Albert H. Kim
          </td>
          <td>2025-12-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="BACKGROUND
Meningioma brain invasion encumbers surgical resection and increases the risk of tumor recurrence, but the molecular mechanisms underlying this process are poorly understood.


METHODS
To identify molecular and cellular features of brain-invasive meningiomas, we (1) analyzed bulk RNA sequencing data from 199 meningiomas, including 33 brain-invasive tumors, (2) analyzed patient-matched single-cell RNA sequencing data of spatially mapped meningioma samples from the tumor core or brain-tumor interface (BTI), and (3) performed spatial transcriptomic sequencing of brain-invasive meningioma samples. Multiplexed immunofluorescence (IF) was used to validate bioinformatic spatial expression patterns. Functional interactions between meningioma cells and neurons were studied in meningioma/neuron co-cultures using confocal microscopy, multi-electrode array recordings, and live cell calcium imaging.


RESULTS
Transcriptomic analyses showed conserved enrichment of TGM2, S100A11, ZYX, and PDGFRA at the BTI across bulk, single-cell, and spatial RNA sequencing datasets. The expression of these genes at the BTI was confirmed using multiplexed IF, and single-cell bioinformatic and microscopy analyses further demonstrated enrichment of macrophages at the BTI. Co-culture assays showed neuronal hyperexcitability and increased proliferation of meningioma cells, suggesting functional communication between meningioma cells and the tumor microenvironment may contribute to meningioma growth in cases with brain invasion.


CONCLUSIONS
Meningioma brain invasion is defined by molecular remodeling of tumor cells and functional interactions within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4352a0cf42108bac27d918f350130e856ef324b" target='_blank'>
              Meningioma cell reprogramming and microenvironment interactions underlie brain invasion.
              </a>
            </td>
          <td>
            Ayush Aggarwal, Mark W. Youngblood, T. Picart, H. Najem, S. Oten, M. A. Cady, Stephen T. Magill, Craig M Horbinkski, James P. Chandler, A. Heimberger, K. Mirchia, S. Hervey-Jumper, David R. Raleigh
          </td>
          <td>2025-12-31</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4167a37773942f9ca4b809bad0d95f60d49ec113" target='_blank'>
              Impacts of mutation accumulation and order on tumor initiation revealed by engineered murine colorectal cancer organoids
              </a>
            </td>
          <td>
            Yanping Li, Xiaoxin Xie, Daqi Deng, Zhiyuan Sun, Zhaoan Huang, Yisen Tang, Liang Fang, Wei Chen, Qionghua Zhu
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer progression arises from interactions across multiple biological layers, especially beyond morphological and across molecular layers that remain invisible to image-only models. To capture this broader biological landscape, we present EXAONE Path 2.5, a pathology foundation model that jointly models histologic, genomic, epigenetic and transcriptomic modalities, producing an integrated patient representation that reflects tumor biology more comprehensively. Our approach incorporates three key components: (1) multimodal SigLIP loss enabling all-pairwise contrastive learning across heterogeneous modalities, (2) a fragment-aware rotary positional encoding (F-RoPE) module that preserves spatial structure and tissue-fragment topology in WSI, and (3) domain-specialized internal foundation models for both WSI and RNA-seq to provide biologically grounded embeddings for robust multimodal alignment. We evaluate EXAONE Path 2.5 against six leading pathology foundation models across two complementary benchmarks: an internal real-world clinical dataset and the Patho-Bench benchmark covering 80 tasks. Our framework demonstrates high data and parameter efficiency, achieving on-par performance with state-of-the-art foundation models on Patho-Bench while exhibiting the highest adaptability in the internal clinical setting. These results highlight the value of biologically informed multimodal design and underscore the potential of integrated genotype-to-phenotype modeling for next-generation precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7747448149656db4d84a5fb568dfd6adf6c8b244" target='_blank'>
              EXAONE Path 2.5: Pathology Foundation Model with Multi-Omics Alignment
              </a>
            </td>
          <td>
            Juseung Yun, Sunwoo Yu, Sumin Ha, Jonghyun Kim, Janghyeon Lee, Jongseong Jang, Soonyoung Lee
          </td>
          <td>2025-12-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metastatic prostate cancer (mPC) is characterized by molecular and phenotypic heterogeneity. With increasing guideline‐driven use of metastatic biopsies, more mPC specimens are being evaluated in surgical pathology. However, unlike localized prostate cancer, no standardized framework currently exists to guide the diagnostic workup of metastatic biopsies or reliably determine phenotypic subtypes. While many mPCs retain conventional acinar features, a growing subset exhibits phenotypic plasticity – including loss of prostate epithelial identity and emergence of neuroendocrine or other divergent lineages. This phenotypic diversity often occurs in castration‐resistant prostate cancer as a mechanism of resistance to chronic androgen receptor pathway inhibition and is characterized by genomic alterations and epigenetic reprogramming. This review outlines the histologic and molecular spectrum of mPC and proposes a practical, pathology‐informed diagnostic approach integrating morphologic assessment and immunohistochemistry. Adoption of a standardized diagnostic framework and multidisciplinary integration will be useful for employing precision oncology in advanced mPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de38a46f132d3e7836729ee1ba6aae01604d174" target='_blank'>
              Molecular subtypes of metastatic prostate cancer: from pathophysiology to diagnosis
              </a>
            </td>
          <td>
            J. M. Dsouza, Erolcan Sayar, M. Schweizer, Stephanie Harmon, C. Morrissey, H. Beltran, Peter S. Nelson, Liang Cheng, C. C. Ding, Michael C. Haffner
          </td>
          <td>2025-12-12</td>
          <td>Histopathology</td>
          <td>1</td>
          <td>58</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Colorectal cancer (CRC) stands as a significant contributor to cancer-related mortality. Owing to its prognostic and therapeutic implications, intratumoural heterogeneity (ITH) presents a considerable challenge. We have developed an experimental framework integrating single-cell derived spheroids with proteomic profiling to facilitate a molecular, proteomic, and therapeutic characterization of intratumoural heterogeneity during CRC progression. Single cells from the commercially available colorectal cancer cell lines SW480 (primary colorectal adenocarcinoma) and SW620 (locoregional lymph node metastasis of the same donor) were isolated using fluorescence-activated cell sorting (FACS) and subsequently cultured forming spheroids. This platform allowed controlled interrogation of clonal diversity through proliferation and viability assays, alongside deep proteomic characterization using label-free liquid chromatography–mass spectrometry (LC-MS) with data-independent acquisition. To evaluate its utility for therapeutic testing, chemotherapy response was measured after 72 h of incubation with 5-fluorouracil (5-FU). The single-cell derived spheroid system demonstrated significant heterogeneity, as evidenced by variations in morphology, growth dynamics, viability, and proteomic signatures. Protein profiling identified ITH-associated proteins (WDR5, CKB, IPO11, ATP6V1F, DCXR and PCCB) and underscored pathway variations including tumour suppressor and proto-oncogenic signalling, vascularization and metabolic regulation. Furthermore, individual spheroids exhibited differential sensitivities to 5-fluorouracil, demonstrating the platform’s capacity to resolve heterogeneous therapeutic responses. Our study establishes a robust and scalable method that integrates single-cell spheroids with proteomics to model and quantify ITH in CRC. By capturing clinically relevant diversity across morphology, viability, proteomic profiles and drug response, this approach provides a foundation for translating spheroid- and proteomics-based assays into personalized therapeutic testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cc15b9af4c8cdc79c6ae94af0f6f4ec13bc73f7" target='_blank'>
              A single-cell derived spheroid approach to dissect intratumoural heterogeneity in colorectal cancer: cell lines show changes in proteomes and therapeutic response to 5-FU
              </a>
            </td>
          <td>
            Helene Sophia Radloff, Michael Kohl, Thorben Sauer, Sonja Hartwig, Sven Geisler, Stefan Lehr, Timo Gemoll
          </td>
          <td>2026-01-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background

Immune checkpoint inhibitors have transformed the treatment landscape of non-small cell lung cancer (NSCLC); however, primary and acquired resistance remain major clinical challenges. Immune evasion in NSCLC is increasingly recognized as a complex, dynamic process driven by tumor heterogeneity and adaptive remodeling of the tumor microenvironment. Conventional bulk profiling approaches are limited in resolving this complexity, underscoring the need for higher-resolution methodologies.

Objective
This review aims to synthesize current evidence on how single-cell transcriptomic approaches have advanced the understanding of immune evasion mechanisms in treatment-resistant NSCLC, with a focus on tumor-intrinsic programs, immune cell dysfunction, and intercellular communication networks.

Methods

A structured narrative review with systematic elements was conducted, analyzing peer-reviewed studies employing single-cell RNA sequencing and related single-cell technologies in human NSCLC. Studies were evaluated for their insights into immune escape pathways, resistance to immunotherapy, and translational relevance.

Results

Single-cell transcriptomic analyses reveal profound intratumoral and immune heterogeneity in treatment-resistant NSCLC. Resistant tumors harbor distinct malignant cell states characterized by impaired antigen presentation, dysregulated interferon signaling, and lineage plasticity. Within the immune compartment, hierarchical T-cell exhaustion, depletion of progenitor-like exhausted T cells, and expansion of immunosuppressive myeloid populations—particularly tumor-associated macrophages—emerge as dominant features of resistance. Ligand–receptor interaction analyses further demonstrate coordinated immunosuppressive communication networks that sustain immune escape at a systems level. Several single-cell–defined cellular states correlate with poor response to immune checkpoint blockade and adverse clinical outcomes.

Conclusion

Single-cell transcriptomics has fundamentally reshaped the conceptual framework of immune resistance in NSCLC, redefining it as a multicellular, dynamic ecosystem rather than a single-pathway phenomenon. These insights provide a strong biological rationale for developing biomarker strategies and rational combination therapies that target both tumor-intrinsic adaptations and the immunosuppressive microenvironment. Continued integration of single-cell technologies into translational and clinical research is essential to overcome immunotherapy resistance and improve patient outcomes in NSCLC.

Keywords: Non-small cell lung cancer; Single-cell RNA sequencing; Immune evasion; Immunotherapy resistance; Tumor microenvironment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7495ff2bd82ff3331af5929ec1f5ca0f1ed50c9" target='_blank'>
              Single-Cell Transcriptomic Mapping of Immune Evasion Pathways in Treatment-Resistant Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Marco Rossi, Giulia Bianchi, Alessandro Conti
          </td>
          <td>2026-01-19</td>
          <td>Advanced Journal of Biomedicine &amp; Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common type of primary liver cancer. Tumor heterogeneity is a major obstacle to effective treatment and is poorly understood using traditional bulk sequencing methods. This review highlights the transformative role of single-cell and multi-omics technologies in determining the cellular and molecular complexities of HCC. We summarize recent advances in single-cell transcriptomics, epigenomics, multi-omics, and spatial transcriptomics platforms, emphasizing their applications in characterizing tumor subclones, cancer-associated fibroblast-immune interactions, circulating tumor cells, and immune-resistant phenotypes. Spatial approaches have revealed the architecture of cancer stem cell niches and tertiary lymphoid structures, providing unprecedented insights into tumor organization and microenvironmental crosstalk. Although still in their early stages, clinical trials have begun to incorporate these technologies, underscoring their translational potential. Single-cell and spatial omics have reshaped HCC research by enabling high-resolution profiling of tumor ecosystems and driving the discovery of biomarkers, therapeutic targets, and strategies for patient stratification. However, high cost, technical expertise, and limited accessibility, particularly in resource-constrained settings, are major barriers to its widespread adoption. Addressing these challenges is critical for translating these powerful approaches into clinical practice and for advancing precision medicine for the treatment of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0abad77c5fcae33c11d7d851ad3eb337d3881a5" target='_blank'>
              Exploring Single-Cell and Multi-Omics Technologies and Their Role in Unravelling Tumor Heterogeneity of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Charmi Jyotishi, Suresh Prajapati, Mansi Patel, Reeshu Gupta
          </td>
          <td>2025-12-17</td>
          <td>Journal of liver cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ovarian cancer remains a lethal disease marked by profound therapeutic resistance, largely orchestrated by a complex tumor microenvironment (TME) governed by metabolism-immune crosstalk. This review focuses on the spatiotemporal dynamics of the metabolism-immune axis in ovarian cancer progression and resistance, with particular emphasis on how cutting-edge spatial multi-omics technologies reveal previously unrecognized layers of intratumoral heterogeneity and geographic organization that cannot be captured by bulk analyses. Using tools such as MALDI-MSI, GeoMx DSP, and CODEX, these approaches enable high-resolution, spatially resolved mapping of metabolite gradients (e.g., lactate, lipids, kynurenine), immune cell niches, and immunometabolic checkpoints within distinct tumor regions. Such spatial profiling uncovers how metabolic reprogramming-dysregulated glycolysis, lipid metabolism, and glutaminolysis-drives localized immunosuppression and chemoresistance through compartment-specific interactions among tumor cells, cancer-associated fibroblasts (CAFs), adipocytes, and immune populations. These geographically defined insights reshape our understanding of therapeutic failure and highlight precise, location-aware vulnerabilities. Accordingly, we propose spatially informed therapeutic strategies, including regional glycolysis inhibition, glutaminase blockade, lipid pathway interference, and their rational combination with immune checkpoint inhibitors (ICIs), to disrupt pathogenic metabolic-immune circuits and improve immunotherapy outcomes. Looking ahead, advances in vivo spatial imaging, gut microbiota modulation, and AI-powered integrative multi-omics frameworks promise truly personalized treatment of ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc5b7b1f229dc616042558b3b060c33208889755" target='_blank'>
              Spatial profiling of the metabolism-immune axis in ovarian cancer
              </a>
            </td>
          <td>
            Zhi-Bin Wang, Ming-Hui Long, Ping Yu, Ya-Li Wang, Zheng Yang, Ma-Sha Huang
          </td>
          <td>2026-01-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Sinonasal squamous cell carcinoma (SNSCC) is a rare malignancy with poorly understood molecular drivers. Consequently, its cellular composition and tumor microenvironment (TME) remain largely undefined. Here, we performed integrated bulk and single-nucleus multi-omic analyses to map the SNSCC ecosystem. Within the malignant compartment, we identified five distinct populations, with hypoxic (TC1) and proliferative (TC2) subtypes associated with adverse clinical outcomes. Functionally, TC1 cells orchestrate a hypoxia-driven angiogenic program via coordinated secretion of adrenomedullin (ADM), MIF, and VEGFA, promoting endothelial tip cell (EC1) differentiation. Integrative analysis revealed these transcriptional programs are underpinned by tumor-specific chromatin accessibility and DNA hypomethylation, particularly at AP-1-enriched regulatory elements. Mechanistically, in vitro studies confirmed that this response depends on cooperative AP-1 and HIF1A signaling. Furthermore, histological analysis of patient tissues demonstrated spatial co-localization of GLUT1-expressing TC1 cells with DLL4-positive EC1 cells. These findings elucidate the epigenetic landscape underlying tumor-stromal interactions and establish the ADM/VEGFA axis as a critical therapeutic target to disrupt epigenetically controlled angiogenesis in SNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bae1ad73527bcabf41b52f28eb9547020179867e" target='_blank'>
              Single-Nucleus Multi-Omics Reveals Hypoxia-Driven Angiogenic Programs and Their Epigenetic Control in Sinonasal Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, J. Noh, Jaehyun Lee, Geunho Kwon, Myeong S. Yu, Yoo-Sam Chung, Seung-Jun Lee, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2026-01-07</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5e712867c904dd12e6307954591e726635cae8e" target='_blank'>
              Differences in T-cell counts and neighborhood patterns in human colorectal adenomas and sessile serrated lesions
              </a>
            </td>
          <td>
            Souvik Seal, Lauren R. Fanning, Evan Bagley, Christine Bookhout, Elizabeth L. Barry, Elizabeth O’Quinn, Dale C. Snover, David N. Lewin, Silvia Guglietta, A. Kourtidis, M. Shrubsole, John A. Baron, Todd A. Mackenzie, Alexander V. Alekseyenko, Kristin Wallace
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Introduction Understanding CD8+ tissue-resident memory T cells (TRM) spatial characteristics in hepatocellular carcinoma (HCC) is challenging, and clarifying the spatial feature changes following immunotherapy represents an urgent research gap. Methods This study employs a multi-omics approach to analyze the spatial distribution and intercellular interactions of TRM cells in HCC tissues using radiomics, single-cell sequencing, and multiplex immunofluorescence histochemistry (m-IHC). Results Our results show that the number of CD8+ TRM cells in HCC increases following immunotherapy. Furthermore, after dividing tumor tissues into the tumor core (TC), invasion margin (IM), and normal tissue (N), a increase in CD8+ TRM cells from the IM to the TC can be observed. Consistent with the results of single-cell sequencing analysis, this change in spatial characteristics may be associated with the interactions between CD8+ TRM cells and CD68+ cells. Discussion Immunotherapy can modify the spatial characteristics of CD8+ TRM cells via regulating their crosstalk with other immune cells, and the spatial distribution of CD8+ TRM cells in the HCC tumor microenvironment (TME) correlates with immune checkpoint blockade (ICB) therapeutic efficacy. Clarifying the mechanisms of action of immunotherapeutic drugs and developing a non-invasive radiomics model to predict CD8⁺ TRM cell dynamics will facilitate the clinical management of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/384dbc51e0f9efcd6dc85b035d8ba5b7b014c96f" target='_blank'>
              Multi-omics spatial characteristics of CD8+TRM cells in hepatocellular carcinoma and immunotherapy response prediction
              </a>
            </td>
          <td>
            Shipeng Li, Ju Ma, Xinqiang Li, Bingbing Qiao, Liancai Wang, Deyu Li, Hua Zhou, Haixu Xu, Jinzhen Cai
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) remains a leading cause of cancer-related mortality, with limited efficacy of current therapies in advanced cases. As a key risk factor for HCC, liver fibrosis may influence tumor progression and immune responses. However, fibrosis-related therapeutic targets remain poorly defined. This study aimed to identify fibrosis-related genes in HCC tumor microenvironment (TME). Methods Our research integrated single-cell RNA sequencing (GSE149614), spatial transcriptomics (GSE245908), and bulk RNA-seq data to identify fibrosis-related prognostic genes in HCC. The genes were selected via the Random Survival Forest algorithm. Additionally, bioinformatics analyses were conducted to explore gene expression patterns, immune infiltration, and spatial localization. Key genes were further validated through in EDU incorporation assay, Transwell migration assay, and CCK-8 proliferation assay. Results Firstly, single-cell analysis identified endothelial cells as key fibrosis-associated cluster in HCC. Three fibrosis-related prognostic genes, LUC7L3, CREB1, and YIPF4, were further identified and validated to patient survival, immune infiltration, and metabolic activity. In addition, enrichment and drug sensitivity analyses linked key genes to tumor-related pathways and chemotherapy response. Spatial transcriptomics then confirmed the spatial distribution and interactions of these genes. Lastly, cellular assays showed that YIPF4 promoted proliferation and migration of HCC cells. Conclusion In this study, we identified fibrosis-related prognostic genes in HCC, including LUC7L3, CREB1, and YIPF4. The roles of these genes in TME were further explored through relevant analyses, potentially providing clinical evidence to support decision-making in HCC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05045962e54fadb4e8932c963a89c2eda496945e" target='_blank'>
              Integrating single-cell RNA sequencing with spatial transcriptomics reveal the fibrosis-related genes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Wenying Qiao, Lei Li, Ronghua Jin, Caixia Hu
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eca909ba170b3407f8c473dadaa8f67180cd2619" target='_blank'>
              The Evolution of Spatial Omics Technologies Introduces A Novel Avenue for Lung Cancer Research.
              </a>
            </td>
          <td>
            Yue He, Zifan Li, Wenxiang Wang, Xu Liu, Shanshan Lu, Jing Bai, Lin Weng, Qingna Zhang, Jun Wang, Kezhong Chen
          </td>
          <td>2026-01-30</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0b74b19f68c5cff1c8b63cb7ba909481a105a55" target='_blank'>
              Spatial 5mC-seq profiling of embryos and decidua after implantation in mammal
              </a>
            </td>
          <td>
            Xun Shan, Yimin Tang, Jinzhou Hu, Mingyuan Bian, Lei Gao, Jiang Liu
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most lethal brain tumors, characterized by extensive immune evasion and a macrophage-dominated tumor microenvironment (TME). However, the molecular determinants governing tumor-associated macrophage (TAM) states and their immunoregulatory functions remain poorly understood. We integrated bulk- and single-cell transcriptomic datasets (TCGA, CGGA, Ivy GAP, and Brain Immune Atlas) to systematically characterize the expression, prognostic relevance, and immune contexture of the myeloid biomarker membrane-spanning 4-domain A6A, MS4A6A, in GBM. Differential expression, survival, and pathway enrichment analyses were performed. Single-cell mapping and CellChat modeling delineated MS4A6A-associated TAM subpopulations, intercellular communication networks, and ligand–receptor signaling dynamics. Spatial transcriptomic validation and pharmacogenomic modeling were conducted to assess anatomic enrichment and therapeutic vulnerabilities. High MS4A6A expression predicted unfavorable survival and correlated with increased stromal and immune infiltration. Single-cell analyses localized MS4A6A predominantly to TAMs, especially Regulatory- and Ribo-TAM states enriched for antigen presentation, T-cell regulation, and ribosomal biogenesis pathways. CellChat analysis revealed that MS4A6A-high TAMs exhibited markedly enhanced communication with CD4+ T cells and Tregs through upregulated PGE2–PTGER2/PTGER4, PECAM1–CD38, and THBS1–CD36 signaling axes, implicating MS4A6A in prostaglandin-driven immune suppression. Spatial profiling confirmed preferential localization of MS4A6A within perivascular and angiogenic niches. Pharmacogenomic prediction indicated that MS4A6A-high tumors were more sensitive to ERK, mTOR, and CDK4/6 inhibition. MS4A6A defines a macrophage-centered, immunosuppressive ecosystem in GBM, mediated by the activation of the PGE2 signaling axis. These findings position MS4A6A both as a prognostic biomarker and as a potential therapeutic node linking myeloid reprogramming to actionable pathway vulnerabilities in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aace77685200c650b039b4e217d58ff78ecf5af" target='_blank'>
              The Myeloid Biomarker MS4A6A Drives an Immunosuppressive Microenvironment in Glioblastoma via Activation of the PGE2 Signaling Axis
              </a>
            </td>
          <td>
            Jianan Chen, Qiong Wu, Anders E. Berglund, Robert J. Macaulay, James J. Mulé, Arnold B. Etame
          </td>
          <td>2025-12-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7129daca11db4cd5b5b60ff63c51368361b7db2" target='_blank'>
              Multimodal single-cell profiling reveals crosstalk between macrophages and stromal cells in poor prognostic cholangiocarcinoma patients.
              </a>
            </td>
          <td>
            Lara Heij, Sikander Hayat, Konrad Reichel, Sidrah Maryam, C. O'Rourke, X. Tan, Marlous van den Braber, J. Verhoeff, Maurice Halder, Fabian Peisker, G. Wiltberger, Jan Bednarsch, Daniel Heise, Julia Campello Deierl, Sven A Lang, F. Ulmer, Tom Luedde, E. Dahl, Danny Jonigk, Jochen Nolting, Shivan Sivakumar, J. Siveke, Florian Vondran, Flavio G. Rocha, H. Baba, S. Hartmann, Jesper B. Andersen, Zaynab Hobloss, A. Ghallab, J. Hengstler, J. J. G. Vallejo, R. Kramann, Ulf Neumann
          </td>
          <td>2026-01-28</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly invasive and refractory subtype of breast cancer. Despite the promise of immune checkpoint blockade (ICB) therapy, response rates remain limited. The immune resistance driven by the tumor microenvironment has not yet been understood entirely, which hinders the personalized precision treatment of TNBC. We integrated single-cell RNA data from 12 cohorts with TNBC and performed a multi-omics analysis combining spatial transcriptomics (ST), bulk RNA sequencing, and multiplex immunofluorescence (mIF) staining to identify immune-resistant subpopulations. Cell-to-cell communication was explored based on NicheNet and CellChat, and the function of CAF was verified by gene knockdown and overexpression in human mammary fibroblasts, followed by co-culture experiments with TNBC cell lines. ST and mIF data were used to analyze and verify cellular co-localization, while deconvolution was used to examine the relationship between two-cell characteristics and immunotherapy or antibody–drug conjugates (ADC) agent benefit. We identified CA9+cancer-associated fibroblasts (CA9+CAF) as a key subset enriched in non-responders to ICB that promotes immune resistance by establishing a hypoxic and immunosuppressive microenvironment via abnormal angiogenesis and glycolysis. ST and mIF analyses revealed a strong co-localization and interaction between CA9+CAF and SPP1+tumor-associated macrophages (SPP1+TAM), forming a stroma-myeloid axis that promotes immune escape through VEGFA/NRP2 axis in co-localization core region compared to the boundary. In vitro experiments demonstrated that the over-expression of CA9 in fibroblasts enhanced the proliferation, invasion, and migration of TNBC cells, while CA9 knockdown inhibited the tumorigenic effects. The high CA9+CAF/SPP1+TAM profile indicated a poor prognosis, reduced effector T cell infiltration, and attenuated response to immunotherapy, may benefit from TROP2, MUC1, and NECTIN4-based ADC agents. The result was validated in TNBC samples treated with neoadjuvant immunotherapy from our center. This study unveils the critical immunosuppressive axis orchestrated by CA9+CAF and SPP1+TAM in TNBC, offering novel insights into the stromal regulatory mechanisms driving immune resistance. The cell-to-cell interaction signature holds promise as predictor of immunotherapy response and potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9990df240d9cf9afd4d976bdcb6753a231a07876" target='_blank'>
              CA9+ cancer-associated fibroblasts cooperate with SPP1+ tumor-associated macrophages driving immune resistance in triple-negative breast cancer
              </a>
            </td>
          <td>
            Qin Ma, Jing Wang, Qian Jiang
          </td>
          <td>2026-01-09</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a major issue in the sphere of head and neck cancer since it is very heterogeneous, which also leads to the poor treatment results and low survival rates in the advanced stages. Here, the review will discuss how spatial omics methods explore tumor heterogeneity in OSCC, which includes cellular, molecular, and immune microenvironment alterations that occur due to cancer stem cells, stromal interactions, genetic instability, epigenetic reorganization, and metabolic reprogramming. The focus of heterogeneity is given on the contribution of the tumor microenvironment such as immune cells, cancer-associated fibroblasts and extracellular matrix remodeling to the stimulation of progression, metastasis, as well as therapeutic resistance. The introduction of spatial omics technologies, including spatial transcriptomics, proteomics, and metabolomics, has revolutionized the field by preserving tissue architecture, enabling high-resolution mapping of gene expression, protein distribution, and metabolite profiles. Significant developments around the spatial omics technologies are discussed, while how they are used in the identification of ligand-receptor network, signaling pathways, and spatial patterns of heterogeneity in OSCC are described. Integration of multi-omics approaches bridges gaps between transcriptomic, proteomic and metabolic, facilitating the discovery of biomarkers for prognosis, immune evasion mechanisms, and precision therapies targeting epithelial-to-mesenchymal transition and immunosuppressive networks. Despite challenges in data integration, cost, and clinical translation, spatial omics holds promise for personalized oncology, with future directions involving artificial intelligence-driven modeling to enhance diagnostic accuracy and therapeutic efficacy in OSCC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00bf591ef53e5e31062caa9f75b7e97f0abcc8b" target='_blank'>
              Spatial Omics in Decoding Oral Squamous Cell Carcinoma Heterogeneity: Microenvironment Crosstalk and Multi-Omics Integration
              </a>
            </td>
          <td>
            Xiaotong You, Yu Liu
          </td>
          <td>2025-12-31</td>
          <td>GenoMed Connect</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Dogs play a major role in sustaining transmission of Leishmania infantum to people, thus prevention and treatment of canine leishmaniosis (CanL) to reduce transmission represents an unmet public health need. Although advances have been made in understanding how immunopathology correlates to infectiousness as CanL progresses, the immune mechanisms underlying transition between disease stages and terminal decline remain ill-defined. To address this knowledge gap, we generated a comprehensive atlas of peripheral immune cells from control dogs and naturally-infected dogs using single-cell RNA sequencing. The LeishDog Atlas captures the cellular and transcriptional complexity underlying CanL, tracing shifts in immune composition and gene expression across progressive, well-defined disease stages. Notably, we identified distinct myeloid, CD4+ and CD8+ T cell phenotypes and transcriptional states associated with disease progression. This resource provides a valuable framework for understanding systemic immune dysregulation in CanL, including determinants of T cell exhaustion and myeloid cell activation, and establishes a foundation for comparative, translational, and mechanistic studies of CanL immunopathology and transmissibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8412687f28e64e4d6b2558529c494e2ee4440eca" target='_blank'>
              A single-cell transcriptomic atlas of peripheral blood immune cells spanning progressive canine leishmaniosis
              </a>
            </td>
          <td>
            Danielle P. Uhl, Daniel J. Holbrook, Max C Waugh, S. Dey, Najmeeyah Brown, Karen I. Cyndari, Jacob J. Oleson, Paul M. Kaye, Christine A. Petersen
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background: Radiotherapy resistance in breast cancer remains a major clinical challenge. The key molecular determinants and cellular populations driving this resistance are not fully understood. Methods: A radiotherapy resistance (RR) gene panel was identified from TCGA-BRCA and GSE120798 cohorts. Single-cell and spatial transcriptomics characterized RRhigh epithelial cells (RRhighepi). A prognostic model, named SuperPC and StepCox-based Radiotherapy Resistance model (SSRR), was built via machine learning and Mendelian randomization. Functional roles of Prolyl 4-Hydroxylase Subunit Alpha 2 (P4HA2) were validated in vitro. Results: The RR gene panel was upregulated in tumors and enriched for cell cycle pathways. RRhighepi cells exhibited elevated stemness, activated cell cycle and metabolic programs, and enhanced DNA damage repair. RRhighepi represented a developmental origin and communicated with endothelial cells. The SSRR model stratified patients into high-risk groups with poorer survival and distinct therapeutic responses. P4HA2, a key model gene, was upregulated in multiple cancers. P4HA2 knockdown suppressed proliferation, invasion, and colony formation, and synergized with radiotherapy to reduce stemness and enhance DNA damage. WGCNA confirmed co-module membership of P4HA2 and the RR panel. Conclusions: This study, through multi-omics analysis, proposes a potential mechanistic model associated with radiotherapy resistance in breast cancer. P4HA2 is a potential therapeutic target that sensitizes breast cancer to radiotherapy. The RR gene panel and SSRR model provide insights into resistance mechanisms and prognostic stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166d8bb1b89b913023588472f1dc07c1ab7e8bdf" target='_blank'>
              Single-cell and spatial transcriptomics reveal P4HA2-mediated radiotherapy resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Huimin Li, Junzhi Liu, Yuheng Jiao, Fengyu Xu, Shurui Wang, Qiang Tang
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genomic alterations are a hallmark of cancer, and extrachromosomal DNA (ecDNA) has emerged as a key source of oncogene selection, tumor growth, and drug resistance. The intratumor heterogeneity and clonal selection of ecDNA is, however, poorly understood. In this study, we pursue a computational approach that leverages allelic imbalance and outlier expression from standard single-cell RNA sequencing (scRNA-seq) to deconvolve the tumor heterogeneity of ecDNA at the single-cell level (ecSingle). Using this approach, we identify oncogene-carrying ecDNAs in tumor samples at the single-cell level, which we validate using genome sequencing. Moreover, we show the superiority of using single-molecule long-read sequencing in resolving ecDNA. ecDNAs displayed extensive intratumor heterogeneity, including subclonal oncogene-carrying ecDNA in primary tumor cells that segregate with distinct transcriptional cell states. Importantly, we show that a rare ecDNA+ clone in the primary tumor can expand to form dominant clones in relapse tumors. Our study introduces a novel approach to studying ecDNA at the single-cell level, enabling both clonal evolution and transcription cell state analysis. We apply this approach to cancer samples to gain deeper insights into the role of ecDNA in intratumor heterogeneity and cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df14db8ad5c3596e9b413772fed54a722b4bb2b" target='_blank'>
              Resolving clonal evolution and selection of extrachromosomal DNA at single-cell resolution
              </a>
            </td>
          <td>
            J. Hendriksen, A. Locallo, B. Schlotmann, Francisco Germán Rodríguez González, Jane Skjøth-Rasmussen, C. Yde, D. Nørøxe, H. S. Poulsen, U. Lassen, J. Weischenfeldt
          </td>
          <td>2026-01-29</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="While cancer immunotherapies have primarily focused on activation of cytotoxic CD8 cells, CD4 T cell activity is also associated with survival and immunotherapeutic response in numerous cancers. We applied integrated single-cell RNA sequencing and multiplexed protein epitope profiling to breast cancer samples to resolve the complexity of immune cell states within the tumor microenvironment. This approach enhanced phenotypic resolution, identifying three distinct states within the CD4 T follicular helper-like (Tfh) cell cluster. A CXCR4high progenitor state gave rise to two differentiated states: an IGFL2high subset resembling conventional Tfh cells and localised to B cell-rich lymphoid aggregates, and a CD103+ subset, exhibiting features of tissue residency, exhaustion, and cytotoxicity, which co-localised with tumor foci. CD103+ Tfh-like cells were found to interact with CXCL10+ macrophages through production of CCL chemokines and CSF1. A higher CD103+ Tfh to IGFL2high Tfh ratio, together with the selective clonal expansion of the CD103+ subset, was strongly associated with improved tumour immunity and superior responses to anti-PD-1 checkpoint blockade, surpassing the predictive value of exhausted CD8 T cells. These findings integrate Tfh and CD4 with cytotoxic potential in breast cancer, offering new insight into anti-tumor immunity and response to checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/570ecff53156bbe18de4ac45af83e6cec5c103da" target='_blank'>
              Proteotranscriptomic Dissection of Breast Cancer T Cell States Identifies CD103+ Tfh-derived Cytotoxic Cells Linked to Immunotherapy Response
              </a>
            </td>
          <td>
            Alexander Swarbrick, G. Al-Eryani, Sophie van der Leij, Etienne Masle-Farquhar, Chia-Ling Chan, K. Harvey, Sunny Z. Wu, D. Roden, Taopeng Wang, John Reeves, Bertrand Yeung, Christopher Goodnow, Cindy Ma, Charles Perou, N. Hacohen, Aziz Al’Khafaji, Mats Nilsson, J. Lundeberg, Marcel Batten, S. Junankar
          </td>
          <td>2026-01-07</td>
          <td>Research Square</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Clonal hematopoiesis of indeterminate potential (CHIP) is driven by hematopoietic stem cells (HSCs) carrying leukemia-associated mutations that expand in the bone marrow. Several prior studies have revealed that the spatial organization of hematopoietic cells in the bone marrow impacts clonal behaviors. Specifically, leukemic blasts have been found to expand almost exclusively in a subset of marrow cavities that are undergoing active bone remodeling, but whether these cavities also support the expansion of non-malignant mutant clones has never been visualized. Although it is widely appreciated that systemic inflammation promotes the selection of mutant clones, this view has emerged without considering the potential heterogeneity in the inflammatory landscape shaped by local bone remodeling. Leveraging intravital imaging and a murine model of CHIP (Tet2+/-), we demonstrated transcriptional and functional compartmentalization of the marrow microenvironment. Macrophages within non-resorptive cavities are inherently anti-inflammatory, which suppresses disease-initiating Tet2+/- cells while preserving the healthy counterpart. Time-lapse imaging further revealed non-transient association between Tet2+/- clones and CD206+ macrophages. Spatially resolved single-cell transcriptomic profiling and functional assessment revealed that physiological bone remodeling influences CD206+ macrophage plasticity and cytokine secretion which regulate the clonal burden. Additionally, anti-tumor immunity alteration within the microenvironment occurred as early as the formation of initial clones. Suppressing bone remodeling with zoledronate or targeting macrophage-associated niche factors mitigated clonal development. Collectively, our study reveals a previously unrecognized inflammatory landscape shaped by local bone remodeling. The finding presents targetable mechanisms and warrants further studies on the use and precautions of bone-modulating management in clonal blood disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/097a0fcd3f67aa857a852457d51cc39385e5cd32" target='_blank'>
              Compartmentalized inflammatory landscape and macrophage plasticity regulate Tet2+/- mediated clonal hematopoiesis.
              </a>
            </td>
          <td>
            Kevin Lee, Cih-Li Hong, Wimeth Dissanayake, Gulzada Kulzhanova, Alexander Noel Pfeffer, Haiyin Li, Senthil Sivakumar, Zi Yin, Emily R. Quarato, Lauren Benoodt, Jeevisha Bajaj, Chike Cao, Chia-Lung Wu, L. Calvi, Shu-Chi A. Yeh
          </td>
          <td>2026-01-12</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Ovarian cancer(OC) poses a significant clinical challenge due to its frequent peritoneal dissemination and development of chemotherapy resistance, contributing to poor patient outcomes. Chronic inflammation is a pivotal driver of disease initiation and progression. While traditional population-level studies fail to capture cellular heterogeneity, recent advances in single-cell technologies including scRNA-seq, spatial transcriptomics, and multi-omics, have enabled high-resolution analysis of inflammatory regulation within the ovarian cancer microenvironment(OCME). This review synthesizes how single-cell approaches have elucidated inflammation-driven remodeling of OCME components, immune cell signaling pathways, and dynamic inflammatory-immune interactions. Focusing on the “dual microenvironments” of ovarian cancer, we discuss mechanisms of immune suppression, stromal reprogramming, and therapy resistance, along with current progress and challenges in translating these insights into precise diagnostic, targeted therapeutic, and immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b692e5a25b9e7346ecfe0d5da66ab9df1f209d5e" target='_blank'>
              Decoding inflammatory regulation in ovarian cancer at single-cell resolution
              </a>
            </td>
          <td>
            Jianghao Yu, Tingting Zhou, Shuangyu Chen, Jialin Zhu, Lu Xu, Yibo He, Wei Chen
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="CD4 T follicular helper (Tfh) cells coordinate humoral immune responses within germinal centers (GC) of lymphoid tissue. Despite their critical roles in vaccination and autoimmunity, the gene expression programs that define functionally distinct human Tfh states— and the molecular pathways engaged by Tfh positioned within the GC niche—remain incompletely understood. This gap has limited translational efforts to monitor or therapeutically target specific Tfh states for clinical benefit. Here, we delineate human CD4 T cell heterogeneity in tonsils and peripheral blood using trimodal single-cell sequencing and spectral flow cytometry to define epigenomic, transcriptional, and proteomic features of distinct Tfh states. Tfh with a GC-like phenotype exhibited markedly increased chromatin accessibility and both mRNA and protein expression of G protein subunit gamma 4 (GNG4). In tonsil, single-cell spatial transcriptomics defined GNG4 expression as a distinguishing feature of activated Tfh states within spatially demarcated GC compartments, with greater specificity than conventionally GC-associated features such as BCL6, TOX2, and S1PR2. In contrast, GNG4− Tfh primarily localized to nonGC regions and exhibited a resting, Th17-polarized phenotype. Together, these data highlight GNG4 as a central feature of activated, GC-positioned Tfh cell identity in humans. One Sentence Summary GNG4 expression defines activated CD4 T follicular helper cells localized to the germinal center of human lymphoid tissue.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e07a3ba4b7925ada4a1cba1ab2a33fb871cfc9" target='_blank'>
              Multimodal analysis defines GNG4 as a distinguishing feature of germinal center-positioned CD4 T follicular helper cells in humans
              </a>
            </td>
          <td>
            Sam Barnett Dubensky, Yutong Zhu, Molly Gallagher, K. Kumashie, Tianyu Lu, Jonathan Tedesco, Nina De Luna, Katherine Premo, Yi Qi, Suzanna Rachimi, Emylette Cruz Cabrera, Bria Fulmer, Ijeoma C. Meremikwu, Ashley Carter, Sarah E. Henrickson, N. Romberg, Amy E. Baxter, Derek A. Oldridge, Laura A. Vella
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Metastasis is still the leading cause of cancer-related death. It happens when disseminated tumor cells (DTCs) successfully navigate a series of steps and adapt to the unique conditions of distant organs. In this review, key molecular and immune mechanisms that shape metastatic spread, long-term survival, and eventual outgrowth are examined, with a focus on how tumor-intrinsic programs interact with extracellular matrix (ECM) remodeling, angiogenesis, and immune regulation. Gene networks that sustain tumor-cell plasticity and invasion are described, including EMT-linked transcription factors such as SNAIL and TWIST, as well as broader transcriptional regulators like SP1. Also, how epigenetic mechanisms, such as EZH2 activity, DNA methylation, chromatin remodeling, and noncoding RNAs, lock in pro-metastatic states and support adaptation under therapeutic pressure. Finally, proteases and matrix-modifying enzymes that physically and biochemically reshape tissues, including MMPs, uPA, cathepsins, LOX/LOXL2, and heparinase, are discussed for their roles in releasing stored growth signals and building permissive niches that enable seeding and colonization. In parallel, immune-evasion strategies that protect circulating and newly seeded tumor cells are discussed, including platelet-mediated shielding, suppressive myeloid populations, checkpoint signaling, and stromal barriers that exclude effector lymphocytes. A major focus is metastatic dormancy, cellular, angiogenic, and immune-mediated, framed as a reversible survival state regulated by stress signaling, adhesion cues, metabolic rewiring, and niche constraints, and as a key determinant of late relapse. Tumor-specific metastatic programs across mesenchymal malignancies (osteosarcoma, chondrosarcoma, and liposarcoma) and selected high-burden cancers (melanoma, hepatocellular carcinoma, glioblastoma, and breast cancer) are highlighted, emphasizing shared principles and divergent organotropisms. Emerging therapeutic strategies that target both the “seed” and the “soil” are also discussed, including immunotherapy combinations, stromal/ECM normalization, chemokine-axis inhibition, epigenetic reprogramming, and liquid-biopsy-enabled minimal residual disease monitoring, to prevent reactivation and improve durable control of metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a5ad40c8765430fb4e35e8df4878d707b5a6318" target='_blank'>
              Molecular and Immune Mechanisms Governing Cancer Metastasis, Including Dormancy, Microenvironmental Niches, and Tumor-Specific Programs
              </a>
            </td>
          <td>
            Dae Joong Kim
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The regulation of T cell exhaustion within the tumor microenvironment plays a pivotal role in shaping the immune response to cancer and determining the efficacy of immunotherapy. However, the molecular factors governing this process in colon cancer remain poorly understood. This study investigates the expression characteristics and functional significance of the transmembrane protein CD82 in the colon cancer immune microenvironment, with emphasis on its regulatory role in CD8+ T cell exhaustion and clinical outcomes. Publicly available transcriptomic datasets were integrated with multiplex immunohistochemistry on colon cancer tissue microarrays to characterize the cell–type–specific distribution of CD82 and its associations with key markers of T cell dysfunction. CD82 expression was markedly increased in tumor-infiltrating immune and epithelial cells compared with normal tissues, particularly within exhausted CD8+ T cells. Elevated CD82 levels showed strong positive correlations with canonical exhaustion markers such as programmed cell death protein 1 and T cell immunoglobulin and mucin domain-containing protein 3. Multiplex immunohistochemical analysis further revealed that enrichment of CD82-positive epithelial regions and expansion of the CD82+TIM-3+PD-1+CD8+ T cell subset were associated with poor prognosis and were confirmed by multivariate Cox regression as independent risk factors for unfavorable survival. In patients who failed to achieve a complete pathological response following immunotherapy, exhausted CD8+ T cells exhibited significantly higher CD82 expression. Single-cell regulatory network analysis identified BATF and BHLHE40 as potential transcriptional regulators of CD82. Collectively, these findings demonstrate that CD82 promotes CD8+ T cell exhaustion, contributing to tumor progression and immunotherapy resistance in colon cancer. This study provides novel insight into the molecular mechanisms underlying immune dysfunction and offers a potential therapeutic target for reversing immunosuppression and improving immunotherapy efficacy in colon malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254f481971ba1782bc6c638ef396c6a18b14c937" target='_blank'>
              CD82-associated exhausted CD8+ T cells define prognosis and immunotherapy resistance in colon cancer
              </a>
            </td>
          <td>
            Jingfeng Zhang, Hengwei Cui, Shaoxian Wu, Hongwei Shi, Haitao Wang, Jingting Jiang
          </td>
          <td>2026-01-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/126eb3bb80a65b589c6798760093749f010ab57b" target='_blank'>
              Robust characterization and interpretation of rare pathogenic cell populations from spatial omics using GARDEN.
              </a>
            </td>
          <td>
            Xinming Zhang, Zhuohang Yu, Gaoyang Hao, Qi Yao, Yanmei Hu, Fuzhou Wang, Xingjian Chen, Linjing Liu, Ka-chun Wong, Xiangtao Li
          </td>
          <td>2026-01-17</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Urothelial carcinoma (UC), encompassing both bladder cancer and upper tract UC, is a biologically heterogeneous malignancy that has long complicated patient stratification and therapeutic prediction. Recent genomic and transcriptomic analyses, however, have delineated reproducible molecular subtypes across non‐muscle‐invasive and muscle‐invasive disease. These classification frameworks have refined biological understanding and improved both risk stratification and treatment selection. Emerging evidence also indicates that genomic alterations, such as FGFR3 mutations, may confer distinct immunologic phenotypes within specific transcriptomic contexts, thereby challenging prior assumptions. The advent of immune checkpoint inhibitors has transformed UC management, yet variable response rates underscore the complexity of the tumor microenvironment (TME). Single‐cell and spatial analyses have identified immune‐inflamed, ‐desert, and ‐excluded TME subtypes, with cancer‐associated fibroblasts playing a key role in immune exclusion and therapeutic resistance. Liquid biopsy, particularly circulating tumor DNA, has demonstrated utility for real‐time disease monitoring, minimal residual disease detection, and treatment stratification, as highlighted in trials such as IMvigor010 and IMvigor011. Urinary extracellular vesicles also represent promising non‐invasive biomarkers, though further standardization is required. In addition, novel therapeutic strategies—including antibody‐drug conjugates, gene therapies, and sustained‐release delivery systems—are broadening treatment options across disease stages. Collectively, these translational advances mark a paradigm shift toward precision immuno‐oncology in UC. Nonetheless, successful clinical implementation will require prospective validation, optimized analytical platforms, and sustained physician‐scientist engagement to translate these discoveries into improved patient outcomes. This review provides a comprehensive overview of recent progress in UC translational research, with a focus on genomic and transcriptomic insights, the evolving role of immunotherapy, advances in liquid biopsy, and the development of novel therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e44cd39c3a6847a9cce1aefa2b7bb98a79ab335f" target='_blank'>
              Translational Advances in Urothelial Carcinoma: From Bench to Bedside
              </a>
            </td>
          <td>
            T. Iwasawa, T. Tsujino, S. Tsukahara, Hiroshi Fukushima, Daisuke Ito, E. Tomiyama, Fumihiko Urabe
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Urology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The detection of rare cell types in single-cell transcriptomics data is crucial for elucidating disease pathogenesis and tissue development dynamics. However, a critical gap that persists in current methods is their inability to provide an explanation based on genes for each cell they have detected as rare. We identify three primary sources of this deficiency. First, the anomaly detectors often function as"black boxes", designed to detect anomalies but unable to explain why a cell is anomalous. Second, the standard analytical framework hinders interpretability by relying on dimensionality reduction techniques, such as Principal Component Analysis (PCA), which transform meaningful gene expression data into abstract, uninterpretable features. Finally, existing explanation algorithms cannot be readily applied to this domain, as single-cell data is characterized by high dimensionality, noise, and substantial sparsity. To overcome these limitations, we introduce a framework for explainable anomaly detection in single-cell transcriptomics data which not only identifies individual anomalies, but also provides a visual explanation based on genes that makes an instance anomalous. This framework has two key ingredients that are not existed in current methods applied in this domain. First, it eliminates the PCA step which is deemed to be an essential component in previous studies. Second, it employs the state-of-art anomaly detector and explainer as the efficient and effective means to find each rare cell and the relevant gene subspace in order to provide explanations for each rare cell as well as the typical normal cell associated with the rare cell's closest normal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddba44f1e7e5ecb1a732ab848f2143eaae645b95" target='_blank'>
              A New Framework for Explainable Rare Cell Identification in Single-Cell Transcriptomics Data
              </a>
            </td>
          <td>
            Di Su, Kai Ming Ting, Jie Zhang, Xiaorui Zhang, Xinpeng Li
          </td>
          <td>2026-01-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e961f589165d98aa8aff64f0539527c2012d419" target='_blank'>
              Pooled single-cell screen in colorectal cancer defines transcriptional modules linked to oncogenes.
              </a>
            </td>
          <td>
            Viola Hollek, Francisca Böhning, Catalina Florez Vargas, A. Sieber, M. Morkel, Nils Blüthgen
          </td>
          <td>2026-01-19</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 225


 Background:
 Colorectal cancer (CRC) is the second leading cause of cancer-related deaths in the United States. Although immune checkpoint blockade benefits ~15% of patients with high mutational burden and microsatellite instability, the majority of CRC remains refractory, with cytotoxic chemotherapy as the standard of care. Identifying mechanisms that sustain cancer cell survival and proliferation is therefore critical to uncovering vulnerabilities that may enhance therapeutic efficacy. The complement peptide C5a, a potent effector generated by proteolytic activation of the complement cascade, signals through its receptor C5AR1 to drive cell survival, cytokine production, and chemotaxis. While historically regarded as a systemic pathway, recent evidence suggests that complement activation can also occur locally. We hypothesized that CRC tumors locally generate C5a, with cancer cell-intrinsic C5AR1 driving survival and chemoresistance.
 Methods:
 CRC tissue microarrays (TMAs) were generated from de-identified specimens (Hackensack Meridian
 Health
 ,
 N
 = 28; commercial TMA,
 N
 = 17). Preliminary spatial transcriptomics was performed on the commercial TMA using the Nanostring CosMx Spatial Molecular Imager. Publicly available single-cell transcriptomes from two independent CRC cohorts (
 N
 = 63 and
 N
 = 62) were reanalyzed. Multiplexed immunofluorescence (24-plex Cyclic IF) was applied to detect complement fragments (C3, C3b/iC3b, C5a) and C5AR1, and to map their cell type-specific expression in situ. Cancer cell-intrinsic C5AR1 expression was validated in two human CRC cell lines and four patient-derived organoids. Functional relevance of C5AR1 was assessed using in vitro cytotoxicity assays combining chemotherapy with pharmacologic C5AR1 inhibition.
 Results:
 Spatial transcriptomics revealed induction of complement pathway transcripts in CRC tumors, further confirmed by reanalysis of two independent CRC single-cell datasets. Multiplexed CycIF revealed an enriched localization of complement ligands (C3, C3b/iC3b, C5a) to tumor-adjacent regions, whereas C5AR1 was enriched in the tumor core. Furthermore, CycIF demonstrated robust C5AR1 protein expression in cytokeratin-positive cancer cells and infiltrating myeloid phagocytes; cancer cell-specific C5AR1 expression was further validated in two human CRC cell-lines and patient-derived organoids. Functionally, cancer cell-intrinsic C5AR1 proved essential for survival, and its pharmacologic inhibition synergistically enhanced 5-fluorouracil cytotoxicity.
 Conclusions:
 Complement activation is spatially organized in CRC, with cancer cell-intrinsic C5AR1 supporting survival. Pharmacologic C5AR1 inhibition synergizes with chemotherapy, positioning the C5a-C5AR1 axis as a tractable therapeutic vulnerability. Ongoing studies aim to define how C5AR1 supports cancer cell survival and mediates chemoresistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a950393c39eca04ec13c0ab1c5dcbbbb94f802" target='_blank'>
              Role of cell-intrinsic C5AR1 in colorectal cancer survival and chemoresistance.
              </a>
            </td>
          <td>
            Ashley Carlo, Raju Shivarathri, Adela Karuli, Richa Mandrekar, Anastasia Zenkevich, Chen-Hua Ma, Kar Chow, Alvin Makohon-Moore, Kevin Tong, Jigar V. Desai
          </td>
          <td>2026-01-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="

 Bladder cancer (BLCA) is a malignant tumor originating from the urothelial lining, characterized by a complex tumor microenvironment (TME) and heterogeneous tumor mutation burden (TMB). Cancer-associated epithelial cells (EpiCs) exhibit substantial heterogeneity during BLCA initiation and progression. Therefore, elucidating the diversity and functional states of EpiCs is essential for improving future diagnostic and therapeutic strategies.



 We integrated multi-omics datasets, including 13 single-cell RNA-seq samples, 514 bulk transcriptome profiles, and 30 whole-exome sequencing (WES) samples, to comprehensively characterize EpiC subtypes. Nonlinear dimensional reduction (UMAP) and clustering analyses were performed to identify major epithelial subsets, followed by secondary clustering. TMB values calculated from self-generated WES data were incorporated into scAB and Ro/e algorithms to determine the TMB-associated epithelial subset, ultimately identifying the key cluster Epi14. Differentially expressed genes (DEGs) of Epi14 were analyzed, and CellChat was used to infer intercellular communication networks. CytoTRACE and Monocle2 were applied to assess stemness potential and differentiation trajectories. Random survival forest (RSF) combined with DEG analysis was used to identify hub genes. Immune infiltration, drug sensitivity, and functional pathway analyses were subsequently conducted. Spatial transcriptomics were deconvoluted using spacexr, CellChat, and PROGENy to map cellular composition, signaling activity, and pathway nodes. Finally, qPCR and Western blot assays were performed to validate hub gene expression in tumor versus adjacent tissues.



 A total of 77,263 cells and 3,000 highly variable genes were included, yielding 32 annotated cell clusters. Secondary clustering combined with WES-derived TMB identified 14 epithelial subpopulations, among which Epi14 was confirmed as the key TMB-associated subset using the Ro/e algorithm. Integration of DEGs, RSF, and multi-cohort datasets revealed ABRACL and ARPC3 as the pivotal hub genes, from which a risk-score model was constructed. Notably, ABRACL expression showed a strong positive association with tumor TMB and exhibited pronounced enrichment in spatial transcriptomic tumor regions.



 By integrating multi-omics and spatial datasets, this study reveals the epithelial heterogeneity of BLCA and identifies ABRACL and ARPC3 as key TMB-associated hub genes within EpiCs. The established risk-score model and validated functional markers provide valuable insights for future mechanistic studies and potential clinical translation in BLCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3d01e34654acc5d2fc42e0f3045f827529cbe0f" target='_blank'>
              Integrated single-cell and spatial transcriptomics combined with whole-exome sequencing reveal key hub genes and epithelial heterogeneity in bladder cancer
              </a>
            </td>
          <td>
            Lin Li, Qianyue Li, Jiaxin Liu, Yifan Wang, Yanjun Ma, Lei Tang, Yawei Zhao
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2092223941ef46884e67c45fd51c2462a350c8db" target='_blank'>
              Unambiguous signatures of malignancies extracted from images of growing cells
              </a>
            </td>
          <td>
            G. Kalweit, M. Kalweit, W. Checinska, M. Saric, R. Berger, E. Bodurova-Spassova, J. Rawluk, N. Talvard-Balland, A. Klett, M. Follo, S. Kreutmair, J. Duque-Afonso, M. LuÌbbert, R. Zeiser, J. Frank, R. Mertelsmann
          </td>
          <td>2026-01-13</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc440df78b8e3b097a9180b03a9b83b2a7188ec3" target='_blank'>
              Senescence-Linked Fibrosis in the Aging Human Ovary Revealed by p16-Based Histological Profiling and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Mark Watson, P. Devrukhkar, N. Murad, Fan Wu, Moo Joong Kim, Hannah Anvari, Uyen Tran, Nicholas Martin, Tommy Tran, Giuliana Zaza, Kevin Schneider, Bikem Soygur, Elisheva D. Shanes, Denis Wirtz, M. Pavone, Simon Melov, Pei-Hsun Wu, D. Furman, Francesca E. Duncan, B. Schilling
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Myxofibrosarcoma (MFS) is a rare and aggressive soft tissue sarcoma characterized by high genomic instability, resulting in high local recurrence rates and limited effective therapeutic options in advanced stages. Recent progress in cancer immunology research has encouraged investigation into the Tumor Microenvironment (TME) of sarcomas, including MFS, to identify immune-related biomarkers of prognostic and therapeutic relevance. Although data remain limited in MFS, existing evidence suggests a heterogeneous immune landscape, including: i) variable expression of immune checkpoint molecules such as Programmed Cell Death Protein 1 (PD-1) and Programmed Death-Ligand 1 (PD-L1), ii) presence of tumor-infiltrating lymphocytes, iii) alterations in antigen presentation pathways, and iv) a pronounced angiogenic signature. These findings underscore the potential role of immune biomarkers for patients’ clinical stratification and the consequent possibility of developing new immunotherapeutic strategies. This review will focus on the cellular and molecular architecture of immune infiltration, vascular remodeling, and lymphoid neogenesis, assessing their prognostic and predictive value as potential biomarkers. Finally, we will present ongoing clinical trials aimed at modulating the immune-vascular niche to inform innovative therapeutic strategies for this challenging sarcoma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f542fc2b8b6a3c1d5def7187e73b0d930cab87" target='_blank'>
              Unveiling the myxofibrosarcoma tumor microenvironment: implications for immunotherapy
              </a>
            </td>
          <td>
            Cecilia Profumo, Massimiliano Grassi, Valentina Rigo, Matteo Mascherini, Paola Monti, Giorgia Arcovito, Giorgia Anselmi, Laura Damele, G. Timon, Danila Comandini, Michela Croce
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Restoring functional β-cell mass through proliferation is a central goal of diabetes therapy, but progress has been hampered by the inability to isolate live, proliferating β-cells for study. Here, CD168 is identified as a conserved surface marker that specifically enriches for proliferating cells in mouse and human pancreatic islets. Single-cell RNA sequencing reveals a distinct cluster of islet cells with high proliferative activity, low insulin expression, and specific CD168 expression. Flow cytometry and immunostaining confirm CD168+ cells co-localize with Ki67+ proliferating cells, reside in G2/M phase, and comprise ≈0.5% of adult islet cells. Using a new CD168-CreERT2 mouse model for lineage tracing, it is demonstrated that CD168⁺ cells rapidly divide, forming two-cell clones within hours. Unbiased lineage tracing shows 87.4% of clones are uni-β lineage, with smaller proportions of uni-α, uni-δ, and multi-lineage. Integrated lineage tracing with multi-omics maps a ≈60-day maturation trajectory for nascent β-cells, involving progressive epigenome remodeling and shifting transcriptional networks. Notably, CD168 expression is conserved in proliferating cells of human islets and pancreatic neuroendocrine tumors, highlighting its clinical relevance. This work establishes CD168 as a marker for proliferating islet cells, providing a tool to study β-cell proliferation and novel insights into islet cell proliferation and maturation mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba802d29084b63c5d6c432929d0001d8e52e060e" target='_blank'>
              CD168 Identifies Proliferating Pancreatic Islet Cells in Murine and Human.
              </a>
            </td>
          <td>
            Shubo Yuan, Jiafu Li, Min Shao, Haili Bao, Ajun Geng, Jialin Yang, Yu Tao, Xinyi Chen, Tianxiong Xiao, Chunye Liu, Zhiyao Xie, Wenqian Song, Q. Yu, Hongxing Fu, Xu Han, Taochen He, Wenquan Wang, Jianfeng Chen, Sheng Yan, Shaohua Song, Liang Liu, Y. Zeng
          </td>
          <td>2025-12-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) and pancreatic ductal adenocarcinoma (PDAC) are highly lethal cancers marked by profound epigenetic and metabolic reprogramming. Among the candidate biomarkers, the DNA methyltransferase DNMT3A and the guanine monophosphate synthetase (GMPS) have emerged as potential prognostic drivers, yet their roles across tumor contexts remain unclear. Here, we demonstrate the application of KMplotter to interrogated pan-cancer transcriptomic and survival datasets encompassing over 7000 patients, complemented by expression profiling of normal, tumor, and metastatic tissues, and integrated tumor microenvironment (TME) analyses. Elevated DNMT3A and GMPS expression correlated with worse overall survival in HCC, particularly in Asian patients, while in PDAC, high DNMT3A but low GMPS expression predicted favorable outcomes. Both genes were consistently upregulated in tumors relative to normal tissues, with further increases in metastatic HCC. Immune deconvolution revealed that DNMT3A was linked to Th2/Treg-enriched niches, whereas GMPS overexpression coincided with high mutational burden or stromal enrichment, fostering immunosuppressive microenvironments. Comparative analysis of toll-like receptor signatures highlighted divergent antigen-sensing pathways, with HCC reflecting viral-driven immune exhaustion and PDAC showing self-antigen–associated signaling. Collectively, these findings position DNMT3A and GMPS as context-dependent biomarkers that integrate metabolic and immune cues to shape prognosis in liver and pancreatic cancer, offering mechanistic insight and translational relevance for patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c71a88a2f1a7eea4d4abf0ff75d340e4a7fe137" target='_blank'>
              Comparative Tumor Microenvironment Analysis for HCC and PDAC Using KMplotter
              </a>
            </td>
          <td>
            Wen-Han Chang, Drashya Shah, Scott Myers, Michael Potts, Sanjive Qazi, Vuong Trieu
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a988a1774cf7ed3f81f590a732823cd8a46b829" target='_blank'>
              Wnt ligands in breast cancer metastasis: unraveling their dual roles and therapeutic potential
              </a>
            </td>
          <td>
            Azita Asadi, Mehran Molavand, Mostafa Vaghari -Tabari, Bahman Yousefi
          </td>
          <td>2026-01-12</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Sporadic pancreatic neuroendocrine tumors (pNETs) with wild type MEN1 represent a major yet largely ignored subset whose biology and metastatic potential remain poorly understood. Because metastasis can occur despite low histologic grade and modest mutational burden, we hypothesized that metastatic competence in MEN1-wild-type pNETs reflects quantitative reinforcement of shared oncogenic pathways rather than distinct mutational processes. We profiled 75 primary low-grade pNETs by whole-exome and RNA sequencing, including 25 percent with lymph node and/or liver metastasis, and integrated genomic and transcriptomic data to connect pathway lesions with expression state. Metastatic tumors showed a slight increase in mutation frequency but conserved base-substitution spectra relative to non-metastatic cases, and adverse clinicopathologic features were enriched in Grade 2 disease. Aggregating alterations to pathways revealed broad convergence on canonical networks, with transcriptomic analyses demonstrating cohort-wide enrichment of Calcium, WNT, and KRAS/PI3K-AKT programs in metastasis. Intersection of significantly mutated genes with differentially expressed genes identified a focused 29-gene overlap, including RYR1 and ZNF273, that marks these convergent axes and distinguishes metastatic from non-metastatic tumors. Gene set enrichment confirmed preferential activation of Calcium, WNT, and PI3K-AKT signaling in metastatic tumors, consistent with a network-intensity model of progression. Finally, upstream-regulator analysis (iPathwayGuide) and gene-centric perturbation mapping (Gene2Drug) nominated candidate targeted and repurposable agents predicted to reverse the metastatic expression phenotype and flagged drugs unlikely to provide benefit, yielding a prioritized, testable therapeutic shortlist which includes fasudil and spaglumic acid. Convergent, domain-specific mutational patterns in highly mutated genes such as ZNF273 and CLCA1 define a molecular signature that could stratify metastatic risk in low-grade pNETs. Collectively, our data reframe metastasis in MEN1-wild-type low-grade pNETs as a property of pathway state rather than mutation quantity and provide a translational blueprint for biomarker-guided therapy development focused on Calcium, WNT, and KRAS/PI3K hubs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d13ad26cf8fe1a732727d66ce0e811b320f88d0" target='_blank'>
              Genomic and Transcriptomic Landscapes of MEN1-Wild-Type Low-Grade Metastatic Pancreatic NETs Uncover Key Oncogenic Drivers and Targetable Pathways
              </a>
            </td>
          <td>
            Md. Hafiz Uddin, Zaid Mahdi, I. Muqbil, B. Herring, B. Rose, H. Y. Khan, Yiwei Li, A. Aboukameel, Sahar F. Bannoura, Hugo Jimenez, Allan M. Johansen, M. N. Al-Hallak, Ibrahim Azar, Amr Mohamed, T. Hadid, Nitin Vaishampayan, Yang Shi, Yin Wan, Vy Ong, G. Dyson, R. Beydoun, M. Tobon, Eliza W Beal, Herbert Chen, Anthony F. Shields, P. Philip, Jennifer Beebe-Dimmer, R. Mohammad, Boris C. Pasche, B. El-Rayes, A. Azmi
          </td>
          <td>2026-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4d4eea61e2f155251ae235dac3fc5c5c43d18" target='_blank'>
              A pan-cancer single cell landscape reveals heterogeneity and functional diversity of double-negative T cells.
              </a>
            </td>
          <td>
            Qing Hao, Tao Zhou, Huayun Yan, Zhixiang Ren, Wei Mao, Li-Bin Huang, Yue-Yun Chen, Yuelan Wang, Minqin Xiao, Yajiao He, Xuyang Xia, Jing Wu, Wei-Han Zhang, Hai-Ning Chen, Maoxiang Qian, Feng Zhang, Li Yang, Lunzhi Dai, Huiyuan Zhang, Canhua Huang, Yang Shu, Yu Xu, Yang Luo, Yong Peng, Heng Xu
          </td>
          <td>2026-01-03</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) remains a leading cause of cancer-related mortality worldwide, with limited responsiveness to immune checkpoint inhibitors (ICIs). Accumulating evidence indicates that tumor-associated macrophages (TAMs) are central regulators of the immunosuppressive tumor microenvironment (TME) and major contributors to immune escape and therapeutic resistance in HCC. TAMs arise from both circulating monocytes and tissue-resident macrophages and exhibit remarkable plasticity, adopting diverse polarization states in response to microenvironmental cues. Beyond the classical M1/M2 paradigm, single-cell and spatial technologies have revealed a continuum of TAM phenotypes with distinct transcriptional, metabolic, and functional properties. These heterogeneous TAM subsets orchestrate angiogenesis, fibrosis, immune suppression, and resistance to immunotherapy. Consequently, TAMs have emerged as attractive therapeutic targets. Strategies aimed at limiting monocyte recruitment, reprogramming M2-like TAMs toward antitumoral phenotypes, exploiting TAMs as drug delivery vehicles, and combining TAM-targeted interventions with ICIs, radiotherapy, anti-angiogenic agents, or nanobiotechnology have shown promising preclinical and early clinical efficacy. This review summarizes current advances in understanding TAM origin, polarization heterogeneity, and functional roles in HCC, and highlights emerging TAM-centered therapeutic strategies that may improve immunotherapy outcomes and enable more precise, durable treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5cb02bcc1ce59f3581a08c421f69f954a98a78b" target='_blank'>
              Rewiring tumor-associated macrophages in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Xue Han, Rui Jin
          </td>
          <td>2026-01-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive subtype of breast cancer characterized by the absence of estrogen, progesterone, and HER2 receptor expression. This malignancy is often associated with a poor prognosis, early recurrence, and limited treatment options. The tumor microenvironment (TME) in TNBC plays a pivotal role in tumor progression, immune evasion, and therapeutic resistance. In recent years, an increasing body of evidence has highlighted the critical interactions between cancer cells and the components within the TME, including immune cells and soluble components. These interactions influence not only the biological behavior of the tumor but also its response to treatment. Exploring the complex interplay between tumor cells and immune components continues to inform the development of more effective therapeutic approaches. In this study, we provide a synopsis of advancements regarding the TME in TNBC. In light of different cellular compartments, we delineate multiscale interplays within the stroma–tumor symbiosis and highlight their antitumor functions and promising targeting strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11bfc269114236644c5ca2675c1d14983686fe40" target='_blank'>
              Prominent protumoral cellular compartments of the tumor microenvironment in triple-negative breast cancer
              </a>
            </td>
          <td>
            Shuai Sun, Pan Zhao, Changrong Wang, Junjun Du, Tingting Zhang, Xiangyun He, Zhibo Zuo, Nan Li, Rongjing Zhou
          </td>
          <td>2026-01-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c745807b38045a754b3f57a50a75c7b86c914ca" target='_blank'>
              A Scalable Framework for Pan-Cancer Tumor Evolution Analysis Enables Transfer of Progression Mechanisms Across Tumor Entities
              </a>
            </td>
          <td>
            Simon Pfahler, Andreas Lösch, Y. L. Hu, Rudolf Schill, Rainer Spang, T. Wettig
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 The mechanisms by which a small subset of high-risk localized prostate cance becomes immunogenic is not well understood. The objective of this study is to identify the tumor intrinsic and extrinsic mechanisms associated with immunogenicity in this population with the goal of harnessing this knowledge to further immunotherapy in PCa. Methods We performed Visium and Xenium spatial transcriptomic profiling on radical prostatectomy specimens from untreated, high-risk, localized prostate cancer patients obtained from the UCLA biobank with varying level of intra-tumoral T-cell infiltration (TILs). Validation was performed using OPAL multiplex immune-fluorescence (mxIF) staining for CD4, CD8A, CD68, PIGR (marker of club cells), and PSA on 18 specimens from a separate cohort of untreated, intermediate to high-risk localized prostate cancer. Summary Spatial transcriptomic analysis identified cells with club-like features to be associated with global T cell infiltration. Both benign epithelium and PCa can adopt the club-like features by the expression of PIGR protein. Benign glands with club-like features have reduced androgen receptor activity while club-like PCa are enriched with PTEN loss by IHC. Regardless of cell type, expression of PIGR leads to close contact of T cells and club-like cells. Xenium profiling revealed that most T cells are activated CD4+ stem T cells expressing TCF1 protein that are spatially enriched with myeloid dendritic cells and plasmacytoid dendritic cells. Conclusions We defined a subset of immunogenic localized prostate cancer that expresses PIGR either in benign or tumor epithelium. Epithelial expression of PIGR is associated with TILs, suggesting a non-canonical route for immune recruitment and activation.



 Anson Ku, Rong Rong Huang, Sumeyra Kartel, Xingyu Chen, Kayla Heyward, Ross Lake, Rosina Lis, Nikil Pramod, Shana Trostel, Anna Baj, Etyan Ruppin, Scott Wilkinson, John Fenimore, Huihui Ye, Adam Sowalsky. Tumor PTEN-Loss is Associated with Club-Like Cells Phenotype and CD4 T-Cell Infiltration in High-Risk Localized Prostate Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae61f5a94b5b745d520039bbb08b848e6edeb40a" target='_blank'>
              Abstract A035: Tumor PTEN-Loss is Associated with Club-Like Cells Phenotype and CD4 T-Cell Infiltration in High-Risk Localized Prostate Cancer
              </a>
            </td>
          <td>
            A. Ku, R. Huang, Sumeyra Kartel, Xingyu Chen, Kayla Heyward, R. Lake, Rosina T. Lis, Nikil Pramod, S. Trostel, A. Baj, Etyan Ruppin, S. Wilkinson, J. Fenimore, H. Ye, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077d8d4a45150f5dcadd79e304aadcb9aba01396" target='_blank'>
              YAP1 Is a Crucial Nexus in the Tumor Microenvironment.
              </a>
            </td>
          <td>
            Raghavan Narasimhan, Anshula Narayanasamy, Jaya Padmanabhan, Srikumar Chellappan, Durairaj Mohan Kumar
          </td>
          <td>2026-01-13</td>
          <td>Current medical science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background/Objectives: Constructing a comprehensive spatiotemporal map of tumor heterogeneity is essential for understanding tumor evolution, with copy number variation (CNV) being a significant feature. Existing studies often rely on tools originally developed for single-cell data, which fail to utilize spatial information, often leading to an incomplete map of clonal architecture. Our study aims to develop a model that fully leverages spatial omics data to elucidate spatio-temporal changes in tumor evolution. Methods: Here, we introduce SCOIGET (Spatial COpy number Inference by Graph on Evolution of Tumor), a novel framework using graph neural networks with graph attention layers to learn spatial neighborhood features of gene expression and infer copy number variations. This approach integrates spatial multi-omics features to create a comprehensive spatial map of tumor heterogeneity. Results: Notably, SCOIGET achieves a substantial reduction in error metrics (e.g., mean squared error, cosine similarity, and distance measures) and produces superior clustering performance, as indicated by higher Silhouette Scores compared to existing methods, validated by both simulated data with spot-level ground truth and patient cohorts. Our model significantly enhances the accuracy of tumor evolution depiction, capturing detailed spatial and temporal changes within the tumor microenvironment. It is versatile and applicable to various downstream tasks, demonstrating strong generalizability across different spatial omics platforms, including 10× Visium and Visium HD and various cancer types, including colorectal cancer and prostate cancer. This robust performance improves research efficiency and provides valuable insights into tumor progression. Conclusions: SCOIGET offers an innovative solution by integrating multiple features and advanced algorithms, providing a detailed and accurate representation of tumor heterogeneity and evolution, aiding in the development of personalized cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6913fca8817dbe011dd13361f41522851e540c4" target='_blank'>
              Toward Graph-Based Decoding of Tumor Evolution: Spatial Inference of Copy Number Variations
              </a>
            </td>
          <td>
            Yujia Zhang, Yitao Yang, Yan Kong, Bingxu Zhong, Kenta Nakai, Hui Lu
          </td>
          <td>2025-12-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7ef77f1fc622a6df3daeb5f134f560d64366224" target='_blank'>
              A transition zone enriched WIF1⁺ basal cell subtype is associated with benign prostatic hyperplasia
              </a>
            </td>
          <td>
            Rulin Wang, Qizhi Zheng, M. Graham, A. Vaghasia, Jianyong Liu, Jordan Gregg, T. Jones, Anuj Gupta, Nicole Castagna, Yan Zhang, Kornel Schuebel, J. Meyers, Alyza M. Skaist, Dixie Hoyle, Yuhan Yang, William G. Nelson, Angelo M De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Identifying new therapeutic approaches in high-grade serous ovarian cancer (HGSC) requires the development of more accurate preclinical models that replicate the patient-specific tumor and its microenvironment. To address this, we established immunocompetent patient-derived cultures (iPDCs) for HGSC, cultured on a physiologically relevant human omentum-gel matrix. We developed a high-throughput platform that combines drug testing, histological analysis, genomic profiling, single-cell studies, and spatial biomarker discovery. Our results from 47 tumors showed that iPDCs recapitulated the tumor genomic and histological characteristics, while also retaining the intratumoral immune cells. The iPDC treatment responses correlated significantly with the patients' clinical treatment responses. Using iPDCs and scRNAseq, we identified potentially effective therapeutic options for patients with recurrent HGSC linked to distinct tumor cell states and mechanisms of resistance. High-throughput drug response profiling with single cell-imaging identified ataxia telangiectasia and Rad3-related inhibitor (ATRi) combined with an immunotherapy targeting Autotaxin as a promising new combination treatment for HGSC. Using hyperplexed imaging and the spatial analysis, we discovered that ATRi responses were associated with significant increases in both intra- and peritumoral T cell infiltration, particularly in PD1+ CD8+ T cells. Additionally, the ATRi induced reactivation of CD8+ T cells was linked to spatial interactions with replication stress positive tumor cells. Thus, our iPDC platform presents as a representative high-throughput ex vivo model to advance precision oncology in HGSC uncovering ATRi-immunotherapy combination as a potentially effective therapeutic option for clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1f51bba74a8851f75d52e66b786b9875c816a69" target='_blank'>
              Ex Vivo Immuno-Oncology Platform Reveals Spatial T Cell Infiltration Patterns Linked to ATR Inhibition Responses in High-Grade Serous Ovarian Cancer.
              </a>
            </td>
          <td>
            Ashwini Sakrepatna Nagaraj, M. Salko, Aditi Sirsikar, Ziqi Kang, E. Erkan, E. Pietilä, Iga Niemiec, Jie Bao, G. Marchi, Angéla Szabó, María Hincapié-Otero, Anastasia Lundgren, Kirsten Nowlan, S. Pikkusaari, A. Kanerva, J. Tapper, Riitta Koivisto-Korander, H. Lassus, L. Kauppi, S. Hautaniemi, Anna Vähärautio, Jing Tang, E. Kekäläinen, Ulla-Maija Haltia, A. Virtanen, J. Jukonen, T. Salo, Anniina Farkkila
          </td>
          <td>2026-01-21</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent epithelial malignancy. Immune-checkpoint inhibitors have reshaped first-line therapy for recurrent/metastatic disease; yet durable benefit is confined to a subset, reflecting myeloid-centric mechanisms—SPP1+ TAM barriers, cDC1/IL-12 insufficiency, and CXCL8–CXCR1/2–driven neutrophil trafficking—distinct from, and complementary to, classical lymphoid exhaustion. In this review we summarize advances from single-cell RNA and ATAC profiling and spatial transcriptomics that resolve macrophage, dendritic-cell and neutrophil programs, and appraise translational opportunities spanning myeloid reprogramming, innate–adaptive combinations and spatial biomarkers. We also discuss enduring challenges—including HPV-status heterogeneity, limited assay standardization and a scarcity of predictive metrics—that temper implementation. By integrating myeloid-informed readouts (e.g., SPP1–TAM burden, cDC1 competency, serum IL-8) with PD-1–based regimens, EGFR-directed antibodies and myeloid checkpoints (CD47–SIRPα, PI3Kγ, CXCR1/2), emerging strategies aim to restore antigen presentation, improve lymphocyte trafficking and remodel tumor–stroma interfaces. Our synthesis provides an appraisal of the evolving landscape of myeloid-informed precision immuno-oncology in HNSCC and outlines pragmatic standards and avenues for clinical translation. We hope these insights will assist researchers and clinicians as they endeavor to implement more effective, individualized regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1e89f3eb622744a2cb356a1f52faae72322d69" target='_blank'>
              Myeloid-driven immunosuppression in head and neck cancer: single-cell ATAC/RNA and spatial transcriptomic perspectives
              </a>
            </td>
          <td>
            Rui Luo, Jianzheng Yang, Zimeng Cao, Bing Li
          </td>
          <td>2025-12-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 190


 Background:
 Microsatellite stable colorectal cancer (MSS-CRC) in advanced-stage disease remains a major therapeutic challenge due to its molecular heterogeneity. While classification frameworks such as CMS (Consensus Molecular Subtypes) and IMF (Intrinsic CMS, MSI, Fibrosis) have improved understanding of localized CRC, they offer limited insight into the biology of metastatic tumors. To address this gap, we implemented a multi-omic study of a cohort of treatment-naïve MSS-CRC patients with synchronous liver metastases (CRLM) and a large MSS-CRC Real World Dataset (RWD).
 Methods:
 Paired primary CRC and CRLM specimens from 20 patient underwent comprehensive profiling, including Whole Exome, Transcriptome: bulk and single-nucleus RNA sequencing (snRNA-seq) analysis. The intrinsic CMS framework (iCMS) was applied to characterize tumor-intrinsic states across primary CRC and CRLM. iCMS scores were inferred using previously published templates. Key findings were validated in a real-world dataset (TEMPUS MSS-CRC) profiled using multi-omics approaches.
 Results:
 Analysis of 388,018 single nuclei revealed 6 epithelial clusters associated with specific iCMS subtypes, supported by concordant bulk RNA-seq calls. Subtype composition at snRNA-seq level revealed that approximately one-third of tumors exhibited a hybrid phenotype with admixture of iCMS2 and iCMS3 epithelial cells. Tumors that were iCMS-unclassified in bulkRNAseq consistently displayed hybrid composition in snRNA-seq. Similarity metrics (cosine distance) from bulk data correlated with admixture proportions in snRNAseq. Genomic analysis revealed mutations in
 APC
 and
 KRAS
 as the predominant genetic

 events in non-hybrid iCMS2 and iCMS3 tumors, aligned with their differential WNT and MAPK pathway activity, respectively. In contrast, iCMS-hybrid tumors were significantly enriched for clonal co-mutations in WNT and MAPK drivers, with overrepresentation of specific
 KRAS
 variants (G12D, G13D). These iCMS-hybrid tumors exhibited intermediate WNT-MAPK co-activation. In the metastatic setting, iCMS2 and iCMS3 tumors maintained their subtype identity, whereas iCMS-hybrid primary tumors showed a shift toward iCMS2 dominance in liver, suggesting that the liver microenvironment promotes expansion of the iCMS2 state.
 Conclusions:
 Our study identifies a novel iCMS-hybrid phenotype with transcriptomic plasticity and co-activation of WNT and MAPK pathways, offering refined molecular resolution and potential to guide precision therapy in advanced MSS-CRC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7dd62e9da37801a1d7c35e801c417c4239af39d" target='_blank'>
              Genetic configurations and liver niche drive tumor-intrinsic state plasticity in metastatic MSS-CRC.
              </a>
            </td>
          <td>
            Ruben Ferrer Luna, Bogdan-Alexandru Luca, Carlos Mackintosh, Rachel Webster, Micaela Tatman, Nives Rombini, Belen Rivero, Benjamin Chen, S. Tejpar, Gonzalo Lopez, María Ortiz Estévez, Ruslan D Novosiadly
          </td>
          <td>2026-01-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98cf179f4f4e7762fc39a8377027aab26eafcc9e" target='_blank'>
              Image-Based Profiling of Induced Trophoblast Stem Cells Identifies Signatures Associated with Sex, Schizophrenia Genomic Risk and Placental Stress
              </a>
            </td>
          <td>
            Frank J. Piscotta, Jiyoung Kim, Jisu Ha, Bonna Sheehan, Julia Johnston, John Peters, Thomas M. Hyde, Brady J. Maher, Juan Caicedo, D. Weinberger, Evgeny Shlevkov, Gianluca Ursini
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Neoadjuvant therapy (NAT) is increasingly used for pancreatic ductal adenocarcinoma (PDAC); yet most patients only achieve partial response. Pathological treatment response grading focuses on assessing residual tumor burden, often overlooking changes in tumor microenvironment (TME). To address this gap, we compared tumor cells and TME of 13 NAT-naïve and 23 post-NAT PDACs using integrated spatial pathomics and transcriptomics, with validation in an independent single-cell spatial dataset. NAT significantly reduced tumor burden (14.7%-6.2%, p = 0.004), but systemic comparison of 13 cytomorphometric features of tumor cells alone did not reliably distinguish between naïve and NAT cases. In contrast, NAT profoundly remodeled TME by increasing cancer-associated fibroblast (CAF) and CD8+ T cell densities, promoting CD8+ T cell-tumor cell proximity and fibrosis, reducing tumor-associated neutrophils, and redistributing tertiary lymphoid structures (TLSs). Spatial transcriptomics shows NAT induced apoptosis, DNA-damage response, and AGC-kinase (S_TK_X) signaling in tumor cells, and upregulated complement pathway, p53 signaling, and cellular senescence program in TME. Cross-platform single-cell spatial analysis revealed decreased regulatory T cells (Treg) and a shift from myofibroblastic (mCAF) to inflammatory CAF (iCAF). Importantly, post-NAT patients with more fibrosis had longer overall survival (p = 0.02), and higher B-cell density showed a favorable trend (p = 0.06). Together, these results suggest that beyond tumor debulking, NAT induces a coordinated TME remodeling characterized by fibroblast reprogramming, matrix fibrosis, and immune spatial reorganization. Incorporating assessment of NAT-induced stromal and immune changes into TRG may improve prognostication and guide more precise therapy in post-NAT PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc4f312a97dc39d70b704d780122da11f4cc3af9" target='_blank'>
              Neoadjuvant Therapy-Induced Remodeling in Pancreatic Ductal Adenocarcinoma: Multimodal Spatial Analysis and Prognosis.
              </a>
            </td>
          <td>
            Xiaofei Zhang, Ruoxin Lan, Danting Li, Yongjun Liu, Sonu Kalyan, Momin Iqbal, Nancy Liu, Jerry Zhang, Iman Hanna, Mala Gupta, C. Zhao, Weiguo Liu, Jonathan Melamed, Michael Shusterman, Jessica Widmer, John Allendorf, Yao-zhong Liu
          </td>
          <td>2026-01-13</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive tumor type with poor patient outcomes. Most patients present with metastatic disease, which generally has reduced immune infiltration compared to primary tumors. Further work to elucidate the specific cellular features of metastatic PDAC is needed to guide the development of future immunotherapy strategies. Here, we investigated the hypothesis that PDAC tumors harbor distinct immunologic and stromal features depending on their anatomical site. Multiplex immunohistochemistry (mIHC), spatial analysis, and single-cell mass cytometry (CyTOF) uncovered dominant immune and stromal cell populations in tumors derived from 27 primary and 26 liver metastases. Metastatic liver tumors from PDAC patients contained reduced T cell infiltration, fibroblast populations, and collagen accumulation than primary lesions, while CD68+ cells, often co-expressing CCR2, were more abundant. Spatial analyses revealed distinct immune cell communities in primary and metastatic PDAC, whereby CK19+ cells clustered differentially with α-SMA+, CD3+, and CD68+ cells, depending on the tumor site. When comparing tumor-associated regions, the proportion of peritumoral CK19- cells remained consistent, but their composition varied by disease site. CD8+ T cells were significantly less frequent in metastatic tumors, while both CD4+ and CD8+ T cells present in primary tumors expressed more transcription factors (TFs) associated with suppressive properties, including FoxP3 and RORγt. CyTOF revealed that T cells co-expressed multiple inhibitory checkpoint receptors, most notably LAG-3 and PD-1. This report reveals that primary and metastatic tumors from PDAC patients harbor vastly distinct immunologic and stromal features at the protein level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bea286cd97f7f8064af812daefa94a7da6b7d73" target='_blank'>
              High-Dimensional Protein Analysis Uncovers Distinct Immunological and Stromal Features in Primary and Metastatic Pancreatic Ductal Adenocarcinoma.
              </a>
            </td>
          <td>
            E. Greene, N. Horvat, D. Doxie, V. Parihar, Jayden Kim, Cameron J Herting, Erin E. Grundy, Ayana T Ruffin, Alyssa M Krasinskas, Shishir K Maithel, Juan M. Sarmiento, Mihir M. Shah, Mohammad Y Zaidi, Maria Diab, O. Alese, Kavita M. Dhodapkar, H. Kissick, B. El-Rayes, Chrystal M. Paulos, G. Lesinski
          </td>
          <td>2025-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d360e6a196e9de4b8313a49f5e3b66fe3b338c6" target='_blank'>
              Integrative single-cell and spatial transcriptomics with explainable AI reveal lethal prognostic axis in prostate cancer.
              </a>
            </td>
          <td>
            Qintao Ge, Zhenda Wang, Yangyun Wang, Tonghui Chu, Zhongyuan Wang, Wenkai Zhu, Youzhao Zhang, Yonghao Chen, Dingwei Ye, Wenhao Xu, Zhongyuan Wang, Mierxiati Abudurexiti
          </td>
          <td>2026-01-06</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Rationale: Tertiary lymphoid structures (TLSs) play a key role in the adaptive immune response within the local tumor immune microenvironment (TIME) and can predict the clinical outcome of several solid tumors. However, the clinical relevance of TLSs and their formation mechanism in gastrointestinal stromal tumors (GISTs) remain unclear. Methods: Integration analysis was performed on a single-cell RNA sequencing (scRNA-seq) cohort (n = 8), a public transcriptome cohort (n = 65), a public scRNA-seq cohort (n = 7), a tissue cohort (n = 197) and a serum cohort (n = 169) to decode the characteristics of the immunological microenvironment of GIST. The multiplex immunohistochemistry (mIHC) staining, in vitro cell culture, chemotaxis assays and antibody-dependent cellular phagocytosis (ADCP) experiments were used to validate the results of the bioinformatics analysis. Results: Preoperative imatinib targeted therapy significantly enhanced TLS formation in GIST tissues, predicting improved therapeutic efficacy and favorable prognosis. Mechanistically, imatinib remodeled the local microenvironment via tumour-associated macrophages, recruiting B cells via the CXCR4-CXCL12 axis to drive TLS development. Functionally, TLS dominated germinal centre (GC) B-cell differentiation and the formation of IgG+ plasma cells (PCs), which preferentially enhanced the adaptive immune response through the ADCP effect. From a clinical perspective, we identified three distinct GIST immune classes (GICs A-C). GIC-A tumors featured abundant CD20+ B cells and TLS, as well as a favorable prognosis. They were accompanied by enhanced antigen presentation, accumulation of IgG+ PCs, increased immunosuppressive properties and a high frequency of KIT exon 11 mutations. These mutations potentially serve as a predictive biomarker for future targeted and immunotherapies. Furthermore, patients with high serum IgG levels experienced significant therapeutic benefits. Conclusions: Our data show that local adaptive immunity dominated by TLS is a key factor in the efficacy of targeted therapy, and suggest that inducing IgG could be a feasible strategy for improving the prognosis of patients with GIST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/485f7a996f0585ddffcf4dd3ac8a1112abd5a345" target='_blank'>
              Enhanced formation of tertiary lymphoid structures shapes the anti-tumor microenvironment in gastrointestinal stromal tumors after imatinib targeted therapy
              </a>
            </td>
          <td>
            Ping Tao, Jiongyuan Wang, Peidang Fan, Wen-Shuai Liu, Wei-Qi Lu, Yujie Hu, Q. Lu, Lijie Ma, Hanxing Tong, Yong Zhang
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) displays marked cellular heterogeneity and immune microenvironment complexity that fundamentally influence transcatheter arterial chemoembolization (TACE) treatment responses and patient outcomes. Deciphering the molecular architecture underlying therapy resistance remains essential for advancing precision oncology in HCC management. Methods We integrated four single-cell RNA sequencing cohorts with bulk transcriptomic datasets and longitudinal clinical annotations from The Cancer Genome Atlas Liver Hepatocellular Carcinoma database and Gene Expression Omnibus repositories to perform multidimensional analyses. Computational frameworks including single-cell Phenotype Associated Score (scPAS), high-dimensional weighted gene co-expression network analysis (hdWGCNA), and Single-Cell Regulatory Network Inference and Clustering (SCENIC) were deployed to identify resistance-linked cellular subpopulations and pivotal regulatory modules. Functional validation employed adrenomedullin (ADM)-depleted Huh7 cellular models and xenograft tumor-bearing mouse systems, with mechanistic interrogation via Western immunoblotting, quantitative reverse transcription polymerase chain reaction, and Kaplan-Meier survival estimation to confirm ADM biological functions and clinical relevance. Results We identified a TACE-resistant malignant cell subset (scPAS+) characterized by pronounced activation of glycolytic, hypoxic, and epithelial-mesenchymal transition pathways alongside overexpression of resistance-conferring genes including LINC00221, hexokinase 2, and alpha-fetoprotein. This cellular phenotype demonstrated robust associations with TACE non-responsiveness, sorafenib cross-resistance, and abbreviated patient survival. Patient stratification based on scPAS + signature genes delineated two distinct molecular subgroups: the scPAS + -enriched cohort exhibited marked TACE refractoriness, elevated sorafenib failure rates, immunosuppressive microenvironmental architecture, and diminished 5-year survival probability. Mechanistic investigations established ADM as a critical driver orchestrating this resistance phenotype. ADM depletion attenuated Huh7 cell proliferative capacity, migratory potential, and invasive behavior, reduced xenograft tumor burden in murine models, and substantially potentiated sorafenib antitumor efficacy. Conclusion This study delineates an ADM-driven TACE-resistant HCC cellular subtype (scPAS+) that functions simultaneously as a prognostic biomarker and actionable therapeutic target for circumventing treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de9c2505a2f010ea34035f6766d4f925d037d481" target='_blank'>
              Adrenomedullin orchestrates treatment resistance in hepatocellular carcinoma via immune microenvironment remodeling
              </a>
            </td>
          <td>
            Xixi Gao, Yongliang Sun, Jia Huang, Li Xu, Hanchun Huang, Zhiying Yang
          </td>
          <td>2025-12-11</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with limited therapeutic options in part due to a dense stroma and the ability of the tumor cells to defy therapy by creating an immunosuppressive milieu. A histologic feature of PDAC is the abundance of immunosuppressive M2-polarized tumor-associated macrophages (TAMs) that favor cancer growth and metastases. Cancer cells communicate with immune cells of the tumor microenvironment (TME) by several mechanisms including exosomes to induce plasticity of the immune cells favoring pro-cancer instead of anti-cancer phenotypes. We examined a novel strategy to interrupt the cross-communication between cancer and immune cells by reprogramming the TME with a cholecystokinin-B receptor (CCK-BR) antagonist, proglumide. Cross-communication was examined using 3-dimensional human PDAC spheroids that were either co-cultured with macrophages or treated with supernatant spent media from macrophages. RNA was extracted from the spheroids and subjected to RNA sequencing and Reverse Phase Protein Array (RPPA). RNA sequencing revealed that the most upregulated gene in the cancer cells after co-culture was ELANE, which codes for neutrophil elastase. Next, ELANE expression was down regulated by shRNA transfection in PDAC cells and the effect of the decreased expression was examined on the polarization of immune cells in vitro and in vivo in mice. Exosomes were also collected from PDAC spheroids for evaluation by RPPA. Co-culture of spheroids with macrophages increased cancer cell number and induced polarization of M0 macrophages to the M2-phenotype. Co-culture increased oncogenic pathways and genes by RPPA and RNA sequencing, respectively, and these changes were reversed with proglumide. ELANE knockdown in PDAC cells prevented the M2-polarization of macrophages in vitro and decreased tumor growth in vivo. Immunohistochemistry of ELANE knockdown tumors showed a shift in plasticity from M2-polarized TAMs to M1-polarized cells compared to wild-type tumors. Neutrophil elastase in cancer exosomes increased after co-culture and was abolished by treatment with proglumide. ELANE expression in PDAC cells regulates polarization of macrophages through the release of neutrophil-elastase rich exosomes. Proglumide therapy with CCK-B receptor antagonism is a novel method to interrupt cross-communication pathways between PDAC and immune cells in the TME and decrease cancer growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/447ce01b1e9a047c6c9f2d539977e526bf0ccaff" target='_blank'>
              Interruption of cross-communication pathways alters the immune cell signature of pancreatic cancer and decreases tumor growth
              </a>
            </td>
          <td>
            Jill P. Smith, Mahdis Mohit, Joi Kenner, Ajay Nathan, Jack Drda, Martha D. Gay, Hong Cao, Wenqiang Chen, Alison Gomeiz, E. Baldelli, Wenyu Dou, Hong-Bin Fang, N. Shivapurkar, James Davis, Qi Wei, Abdellah Akhrif, M. Pierobon
          </td>
          <td>2026-01-16</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7651b1e6eed642cd6593c6a259d8f8a2bcfd7a5b" target='_blank'>
              DC subsets and states unraveled across human juxtatumoral and malignant tissues
              </a>
            </td>
          <td>
            Kevin Mulder, Margaux Gardet, Wan Ting Kong, Amit Ashok Patel, Anne Calvez, Grégoire Gessain, Carlos de la Calle-Fabregat, Cécile Piot, Quentin Blampey, Elisa Poupaud, Ahmed-Amine Anzali, Garett Dunsmore, A. Bougouin, Guilhem Pupier, Lizhi He, Timothy Wiggins, Jiang He, George Emanuel, A. Goubet, G. Al-Eryani, Alexander Swarbrick, Judith Michels, R. Dress, Marc Deloger, Antonio Bertoletti, V. Thomas de Montpréville, C. Sautès-Fridman, W. Fridman, Laurence Zitvogel, F. Ginhoux, C. Dutertre
          </td>
          <td>2025-12-08</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917b475731c2b6642606d58ea37295a894578bc5" target='_blank'>
              Decoding Neoantigen-encoding Tumor-Specific Transcripts Unveils a Shared Target Reservoir for Immunotherapy in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Peng Lin, Yifan Wen, Jingjing Zhao, Feifei Zhang, Yaoming Su, Hongwu Yu, Qiaojuan Li, Chengye Liu, Zhixiang Hu, Yan Li, Zhuting Fang, Linhui Liang, Shenglin Huang
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Automatic integration of whole slide images (WSIs) and gene expression profiles has demonstrated substantial potential in precision clinical diagnosis and cancer progression studies. However, most existing studies focus on individual gene sequences and slide level classification tasks, with limited attention to spatial transcriptomics and patch level applications. To address this limitation, we propose a multimodal network, BioMorphNet, which automatically integrates tissue morphological features and spatial gene expression to support tissue classification and differential gene analysis. For considering morphological features, BioMorphNet constructs a graph to model the relationships between target patches and their neighbors, and adjusts the response strength based on morphological and molecular level similarity, to better characterize the tumor microenvironment. In terms of multimodal interactions, BioMorphNet derives clinical pathway features from spatial transcriptomic data based on a predefined pathway database, serving as a bridge between tissue morphology and gene expression. In addition, a novel learnable pathway module is designed to automatically simulate the biological pathway formation process, providing a complementary representation to existing clinical pathways. Compared with the latest morphology gene multimodal methods, BioMorphNet's average classification metrics improve by 2.67%, 5.48%, and 6.29% for prostate cancer, colorectal cancer, and breast cancer datasets, respectively. BioMorphNet not only classifies tissue categories within WSIs accurately to support tumor localization, but also analyzes differential gene expression between tissue categories based on prediction confidence, contributing to the discovery of potential tumor biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62d9ab6f28be06b2fcafb69ae7d77db142d962de" target='_blank'>
              Tissue Classification and Whole-Slide Images Analysis via Modeling of the Tumor Microenvironment and Biological Pathways
              </a>
            </td>
          <td>
            Junzhu Liu, Xuemei Du, Daniel Reisenbuchler, Ye Chen, Markus Eckstein, C. Matek, Friedrich Feuerhake, D. Merhof
          </td>
          <td>2026-01-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The Tracking Cancer Evolution Through Therapy (TRACERx) program represents the most comprehensive effort to characterize tumor evolution in real time. Through longitudinal, multiregion, and multiomic profiling of tumors—and particularly of non-small-cell lung cancer and clear cell renal cell carcinoma—TRACERx has illuminated the dynamic interplay between genetic, nongenetic, and (micro)environmental factors that drive cancer progression, immune evasion, and therapeutic resistance. A central insight from TRACERx has been that not all tumor evolution is genomic: Transcriptomic diversity, epigenetic alterations, RNA editing, and changes in cell–cell interactions also drive adaptation. Methodological innovations—including tumor-informed and ultrasensitive circulating tumor DNA assays, representative sequencing, and integrative immune–genomic analyses—have yielded biomarkers resistant to sampling bias and/or predictive of recurrence, metastasis, and treatment response. By demonstrating that intratumor heterogeneity is a key determinant of clinical outcome and revealing its molecular, transcriptional, and ecosystem-level drivers, TRACERx has established a framework for linking evolutionary dynamics to patient care. As both a scientific framework and a clinical paradigm, TRACERx demonstrates how adaptive, iterative research can refine evolutionary models, improve patient risk stratification, and inspire next-generation cancer evolution studies across malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a59431b9259c4928178579f6c460cf8e369f4f1" target='_blank'>
              TRACERx as an Optimized Paradigm for Understanding Cancer Evolution
              </a>
            </td>
          <td>
            Naomi Iris van den Berg, C. Swanton, S. Turajlic
          </td>
          <td>2025-12-24</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="The tumor infiltrating T cells determine whether a patient would respond positively to immunotherapy. This paper has analyzed an mRNA scRNA-seq dataset of melanoma in the context of mapping heterogeneity in T-cells and the question of whether clinical features can shape cell clustering. Unsupervised clustering with k-means based on quality control followed by a description of the major T-cell groups by canonical immune markers. The identification of the differentially expressed genes that were identified was then done by a posteriori information based ranking approach. This study have created a confidence-weighted approach, which is able to merge Spearman and cosine similarity measures with marker-gene rules, to enhance cell-type annotation. Metadata analysis indicated that treatment status is an element that causes the state changes in / T cells pre- and post-checkpoint therapy. Combining graph-based Leiden clustering and k-means, this study refined the definition of the fine-grained T-cell states and proved that, despite the significant heterogeneity of T cells, the strong definition of subclass depends on the combination of several methods of clustering and the use of orthogonal knowledge of experiments. Pseudotime analysis found evidence in gradual transition between naive or early-activated states to cytotoxic or dysfunctional phenotypes that indicate intratumoral T-cell diversity captures a reflection not only of inherent developmental trajectories, but also of the effects due to therapy. Lastly, survival analysis revealed that there was a distinct relationship between clinical response and overall survival. These findings combined narrow the scope of the T-cell picture and offer understanding in the form of transcriptional programs connected with treatment outcomes. Refer to this link for supplementary materials: https://drive.google.com/drive/folders/1UYW-6EMjSUMVOwHxvuYdFSZwREqrGp89">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b245ce41e96da9548625adf2b4b3d69d756bc71f" target='_blank'>
              Single-Cell Transcriptomics Reveals T-Cell State Transitions and Their Association with Immunotherapy Response in the Melanoma Tumor Microenvironment
              </a>
            </td>
          <td>
            Shangyu Ning
          </td>
          <td>2025-12-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/706f238b315ed6aac316adf86f94df56f8c2fd50" target='_blank'>
              An integrated single-cell lung cancer atlas reveals distinct fibroblast phenotypes between adenocarcinoma and squamous cell carcinomas.
              </a>
            </td>
          <td>
            Yuta Hirano, Hironori Suzuki, Jun Nakayama, Tomofumi Yamamoto, Eishu Hirata, Jun Araya, Yu Fujita, Yusuke Yamamoto
          </td>
          <td>2026-01-23</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Background Smooth muscle cells (SMCs) exhibit remarkable plasticity, undergoing extensive phenotypic switching to generate a highly heterogeneous population within atherosclerotic plaques. While recent studies have highlighted the contribution of SMC-derived macrophage-like cells to plaque inflammation, the specific molecular drivers governing the transition to these pathogenic states remain poorly understood. Methods Here, we re-analyzed single-cell RNA sequencing data from lineage-traced mice to dissect SMC heterogeneity during atherogenesis. Trajectory analysis revealed that SMCs transdifferentiate into a distinct pro-inflammatory macrophage-like subpopulation (macrophage 4) via an intermediate “stem–endothelial–monocyte" cell state. Integrated gene regulatory network inference and in silico perturbation modeling identified interferon regulatory factor 7 (IRF7) as a master transcriptional regulator orchestrating this specific pathogenic transition. Results Clinically, IRF7 expression was significantly upregulated in unstable and advanced human atherosclerotic plaques, correlating strongly with inflammatory macrophage burden. In vivo, ApoE−/− mice challenged with a high-fat diet exhibited robust upregulation of IRF7 in aortic plaques, which co-localized with macrophage markers. Crucially, SMC-specific knockdown of Irf7 using an AAV-SM22α-shIRF7 vector significantly attenuated atherosclerotic plaque progression, reduced necrotic core formation, and enhanced fibrous cap stability. Mechanistically, Irf7 silencing preserved the contractile SMC phenotype and inhibited the accumulation of pro-inflammatory SMC-derived macrophage-like cells within the lesion. Conclusions These findings identify IRF7 as a critical checkpoint in maladaptive SMC phenotype switching. We demonstrate that IRF7 drives the transdifferentiation of SMCs into a pro-inflammatory macrophage-like state, thereby fueling plaque instability. Consequently, therapeutic strategies capable of inhibiting IRF7-mediated SMC plasticity may prove effective in stabilizing vulnerable atherosclerotic plaques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc7d29acba6562a1494d454fb457ef77e3152441" target='_blank'>
              IRF7 orchestrates maladaptive smooth muscle cell phenotype switching in atherosclerosis
              </a>
            </td>
          <td>
            Rundong Cai, Xin Chen, Hongxia Zhang, Qi Wang, Wanrong Xie, Xinghua Pan, Chun Liang, Haiying Zhu
          </td>
          <td>2025-12-27</td>
          <td>Precision Clinical Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (iCCA) is a highly metastatic tumor type, and iCCA with intrahepatic metastasis is associated with poor prognosis. Previous studies have confirmed that the tumor microenvironment (TME) is closely related to tumor metastasis. However, the state of the TME during iCCA liver metastasis remains unknown.


APPROACH AND RESULTS
We profiled 28 specimens from 16 patients with iCCA (with and without intrahepatic metastasis) using integrated single-cell and spatial transcriptomics, bulk RNA sequencing, whole-exome sequencing, and in vivo and in vitro functional experiments to characterize the specific TME changes during iCCA liver metastasis. we identified the metastatic tumor cells, COL3A1+ epithelial cells, and revealed that COL3A1+ epithelial cells-endothelial-to-mesenchymal transition cells crosstalk promoted tumor invasion by inducing mesenchymal transformation of endothelial cells, while COL3A1+ epithelial cells-VEGFA+ CCL4+ neutrophils crosstalk promoted tumor colonization by forming neutrophil extracellular traps.


CONCLUSIONS
We revealed the key biological mechanisms involved in the invasion and colonization phases of iCCA liver metastasis. Moreover, we provide valuable data for understanding iCCA liver metastasis and a possible avenue for the treatment of advanced iCCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd684325d487cd1d69724768d37142d24e7c6b41" target='_blank'>
              Spatiotemporal landscape of intrahepatic cholangiocarcinoma liver metastasis at the single-cell level.
              </a>
            </td>
          <td>
            Ziyu Xun, Hao Wang, Zhengfeng Xuan, Kexu Xiong, Na Zhang, J. Long, Huishan Sun, Yiran Li, Chengpei Zhu, M. Piao, Ting Zhang, Longhao Zhang, Shuofeng Li, Chengjie Li, Jiongyuan Li, Boyu Sun, Zi-Qi Zhou, Shanshan Wang, Ziyue Huang, Kai Liu, Yang Tan, Xiaohua Shi, Xiaobo Yang, Hanping Wang, Ling Lu, Haitao Zhao
          </td>
          <td>2025-12-10</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer‐related mortality worldwide, with tumor microenvironment (TME) heterogeneity playing a critical role in disease progression and therapeutic response. Immune escape (IE) mechanisms facilitate tumor evasion from host immune surveillance, yet their characterization at the single‐cell level in CRC is incomplete. This study integrated single‐cell RNA sequencing (scRNA‐seq) and bulk transcriptomic data from multiple public cohorts to systematically explore IE‐related signatures in CRC. We identified major and minor cell populations within the TME and performed differential gene expression analysis. Using high‐dimensional weighted gene coexpression network analysis (hdWGCNA), we identified gene modules correlated with IE activity. Subsequent survival analysis across six independent cohorts revealed Gamma‐glutamylcyclotransferase (GGCT) as a novel prognostic biomarker associated with poor survival. Functional enrichment analysis indicated GGCT′s involvement in critical oncogenic pathways. Furthermore, GGCT expression correlated with altered immune infiltration profiles and stromal components, suggesting its role in modulating the immunosuppressive TME. Additionally, GGCT demonstrated potential predictive value for response to immunotherapy across multiple datasets. Our findings highlight GGCT as a key player in CRC immune evasion and a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fcee0e763a97a07d783d6f8f51f97fc2a1182ca" target='_blank'>
              Integrative Multiomics Nominate GGCT as a Crucial Regulator of Immunosuppression in Colorectal Cancer
              </a>
            </td>
          <td>
            Qichao Niu, Yang Liu, Kejin Huang, Lisheng Nie, Shiming Zhao, Shifeng Yang, Changlei Su, Shuyu Li
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2438dcb543ce3b34986c62da919180266191c9c9" target='_blank'>
              Optimised fluorescence-activated nuclei sorting for epigenomic analysis of cortical cell types
              </a>
            </td>
          <td>
            Barry Chioza, Stefania Policicchio, Joe Burrage, Georgina E T Blake, Rosemary A. Bamford, Alice Franklin, Darren Soanes, Philippa M. Wells, Ann Babtie, Marina Flores Payan, Jonathan P. Davies, Anthony Klokkaris, Emma M. Walker, Joy N. Ismail, Paulina Urbanaviciute, S. Marzi, E. Hannon, J. Mill, E. Dempster
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="(Isocitrate dehydrogenase) IDH-mutant astrocytoma is classified as World Health Organization (WHO) grade 2–4 and is second only to IDH wild-type glioblastoma in the incidence of adult glioma. However, few studies use single-cell and spatial transcriptome sequencing to analyze its malignant progression. Intraoperative navigation and yellow fluorescence visualization were utilized to accurately isolate high-grade (WHO grade 3–4) and low-grade (WHO grade 2) samples of IDH-mutant astrocytoma for single-cell and spatial transcriptome sequencing. By combining single-cell, spatial transcriptome, The Cancer Genome Atlas (TCGA), and The Chinese Glioma Genome Atlas (CGGA) data, analyses of survival, enriched pathways, transcription factors, intercellular communication, differentiation trajectories, and immune response were performed to identify the characteristics of a unique subpopulation of high-grade IDH-mutant astrocytoma. Our single-cell RNA sequencing analysis identified a distinct subpopulation (Cluster 7) present in high-grade IDH-mutant astrocytoma, which was localized to the terminus of the pseudotime trajectory. Importantly, this cluster not only exhibited an immunosuppressive phenotype correlated with poor clinical prognosis, but also demonstrated significant enrichment in Developmental Biology and Calcium Signaling pathways. Furthermore, this subpopulation engaged in prominent ligand-receptor interactions, particularly through PTN_PTPRZ1 and MIF_CD74 pairs. Notably, comparative analysis revealed that high-grade astrocytoma displayed both quantitatively and qualitatively enhanced communication networks when compared to their low-grade counterparts. Our single-cell RNA sequencing analysis identifies a distinct tumor cell subpopulation present in high-grade (WHO grade 3–4) adult IDH-mutant astrocytoma. This cluster, which likely arises from malignant progression in adult astrocytoma, may provide new insights for developing therapeutic strategies against this clinically challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/586fc47b7312366f86c32ecb5b1181c5a720190b" target='_blank'>
              Single-cell and spatial transcriptome sequencing analysis reveals characteristics of a unique subpopulation in high-grade IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            Wei Chen, Min Zheng, Jia-Yin An, Guo-Hao Huang, Jing-Peng Liu, Lin Yang, Peng Ren, Tingting Wang, Jean-Philippe Hugnot, Sheng-Qing Lv
          </td>
          <td>2025-12-29</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d9d9e99d312aee7d3e8959371325610d5ce9af0" target='_blank'>
              MIF-mediated reprogramming of myeloid lineage within the glioma tumor microenvironment impacts the efficacy of immune stimulatory gene therapy
              </a>
            </td>
          <td>
            Ziwen Zhu, Noah C. Kanis, Anthony E. George, Michael Albdewi, Yingxiang Li, Anzar A. Mujeeb, Brandon L. McClellan, Gurveer Singh, Jialin Liu, W. Al-Holou, J. Heth, Joshua D. Welch, Justin D. Lathia, Pedro Lowenstein, Richard Bucala, Maria G. Castro
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43773501df06a12f0f087012a397c0411bf828e5" target='_blank'>
              Differences in regional developmental origins of glioblastomas revealed by integrated molecular profiling
              </a>
            </td>
          <td>
            Ryan Gensler, A. K. Ahmed, Melanie Alfonzo Horowitz, J. Khalifeh, Marvin Li, Negar Sadeghipour, Mostafa Abdulrahim, Mike Glantz, Sonikpreet Aulakh, Theodore Nicolaides, Calixto-Hope G. Lucas Jr., Srinivasan Yegnasubramanian, Jordina Rincon-Torroella, Chetan Bettegowda
          </td>
          <td>2026-01-19</td>
          <td>Journal of Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Simple Summary Clear cell renal cell carcinoma is the most common form of kidney cancer. Recent advances in spatial and long-read sequencing now allow detailed examination of gene activity within tumours. We used non-passaged, patient-derived tumour organoid models which preserve the structure and cellular composition of the original tumours. Long-read spatial transcriptomics was then applied to study gene expression and transcript variants across organoid regions. We also examined how these patterns change following treatment with the experimental medicine NUC-7738. There are distinct spatial patterns in genes related to protein synthesis and energy metabolism, as well as region-specific differences in transcript isoforms. This work provides new insight into the molecular diversity of kidney cancer and demonstrates how advanced sequencing technologies can be used to study treatment effects in physiologically relevant tumour models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11ea98f536949abee4bc7391c3123466548631ed" target='_blank'>
              Long-Read Spatial Transcriptomics of Patient-Derived Clear Cell Renal Cell Carcinoma Organoids Identifies Heterogeneity and Transcriptional Remodelling Following NUC-7738 Treatment
              </a>
            </td>
          <td>
            Hazem Abdullah, Ying Zhang, Kathryn Kirkwood, Alexander Laird, Peter Mullen, David J. Harrison, Mustafa Elshani
          </td>
          <td>2026-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) remains a lethal malignancy lacking reliable biomarkers for precancerous lesion monitoring and intervention. In this study, formalin-fixed, paraffin-embedded archived tissue biopsies from 134 individuals across two cohorts are collected, spanning five stages of ESCC progression. Laser capture microdissection is employed to isolate the epithelial lesion (L) and the adjacent non-lesion (N) tissues. Proteomic profiling reveals a comprehensive spatio-temporal landscape of ESCC progression, encompassing 4461 proteins. Dynamic network biomarker analysis indicates moderate dysplasia (MOD) as the critical turning stage, warranting clinical attention. A seven-protein diagnostic panel (CCDC86, GBP6, PDCD6IP, C19orf53, SF3A3, GMPPB, ARPC5) with an area under the curve (AUC) of 0.956 achieves superior early detection, and the key signatures are validated by immunohistochemistry in an independent cohort. Functional validation using malignantly transformed HET-1A cells, ESCC cells, and mouse xenograft models identifies GBP6 as a novel target: DNA damage-induced progressive loss of GBP6 promotes ESCC progression by accelerating cell cycle and inducing epithelial-mesenchymal transition. Critically, PARP1 inhibition rescues GBP6 loss by suppressing TP63 and prevents ESCC progression. Overall, this study provides a systematic proteomic atlas of ESCC progression, identifies MOD as a pivotal clinical decision point, and proposes PARP1-TP63-GBP6 axis targeting as a novel intervention strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddf5c8a9af6fc57009a93bae9671de888e2c59fc" target='_blank'>
              Spatio-Temporal Proteomic Landscape Reveals Early Warning Signals of Esophageal Squamous Cell Carcinoma Progression.
              </a>
            </td>
          <td>
            Xumiao Li, Jie Yuan, Min Gao, Jibin Liu, Qinqin Wang, Yaqi Zhang, Mingtao Cao, Xiaolin Hu, Hui Yang, Jun Li, Chen Li, Xiaoguang Li, Hui Wang
          </td>
          <td>2026-01-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Brain metastases are frequent and often lethal complications of advanced cancers. Microglia, resident immune cells of the brain, are known to exert both anti-tumor and pro-tumor functions in late-stage metastases; however, their response during the initial outgrowth of metastatic lesions is not well characterized. Understanding how heterogeneous microglial subgroups are regulated in the developing tumor microenvironment could pave the way for therapeutic strategies to eliminate metastatic tumors at an early stage. In this study, we used a combination of in vivo fate map imaging, single-cell RNA sequencing, and a holographic photoconversion-based technique (Opto-omics) to track tumor fate and early microglial responses over time in the same animals during colonization of disseminated tumor cells. The microglial population was transcriptionally and morphologically heterogeneous, comprising both pro- and anti-tumor subsets. Genetic and pharmacological perturbations revealed that microglial phenotypes could be shifted by inhibiting TGF-β signaling or by deleting the tumor cell surface antigens CD24a and CD47. These findings reveal targetable plasticity in early-stage microglial responses to brain metastasis and suggest that harnessing pro-phagocytic microglial states may offer a therapeutic window before systemic immunosuppression becomes dominant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1778e4e2ee1302e3145504143572e313e56713fc" target='_blank'>
              Microglia Display Heterogeneous Initial Responses to Disseminated Tumor Cells.
              </a>
            </td>
          <td>
            T. Tsuji, Haruka Hirose, Daisuke Sugiyama, Mariko Shindo, Rahadian Yudo Hartantyo, Yutaro Saito, Tsuyako Tatematsu, Shouta Sugio, M. Sanbo, Masumi Hirabayashi, Yasuhiro Kojima, Jun Koseki, K. Hosoya, Hiroshi Yoshida, Tatsuya Ogimoto, Y. Yasuda, Kentaro Hashimoto, Hitomi Ajimizu, Yuichi Sakamori, H. Yoshida, Noritaka Sano, Masahiro Tanji, Hiroaki Ito, Kazuhiro Terada, M. Hamaji, T. Menju, Hiroyuki Konishi, S. Kumagai, Cyrus M. Ghajar, Daisuke Kato, Hiroshi Date, Akihiko Yoshizawa, Yoshiki Arakawa, H. Ozasa, A. Moorhouse, Teppei Shimamura, H. Nishikawa, Toyohiro Hirai, Hiroaki Wake
          </td>
          <td>2025-12-10</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b75da29edbdbad60fea18ff4b3049824f5017d" target='_blank'>
              Critical role for a high-plasticity cell state in lung cancer.
              </a>
            </td>
          <td>
            J. Chan, Chun-Hao Pan, Jonathan Rub, Gary Guzman, Klavdija Krause, Emma Brown, Zeda Zhang, Hannah Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2026-01-21</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Intratumoral heterogeneity drives immunotherapy resistance, hindering clinical efficacy. Functional assays that capture heterogeneous responses to immunotherapies remain limited. Here, we present a 3D live imaging assay using patient-derived ovarian cancer organoids to quantify natural killer (NK) cell-induced apoptosis across hundreds of individual organoids. Our assay reveals inter- and intratumor response heterogeneity and detects rare populations unresponsive to NK cytotoxicity, enabling the study of subclonal immune responses and underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c2a1ea73f6fab9f1966b90164d74f94fad3f917" target='_blank'>
              Large scale quantification of natural killer cell-induced apoptosis in patient-derived organoids reveals intratumoral response heterogeneity
              </a>
            </td>
          <td>
            Marisa Mercadante, Armin Scheben, Jacob Estrada, Jan Savas-Carstens, William Sullivan, Nicholas Housel, Maria Sapar, Tatiana Volpari, Jax Hebner, Tomasz Rusielewicz, Frederick J. Monsma, Stefan Semrau, Yinan Wang, Laura A. Martin
          </td>
          <td>2025-12-26</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Introduction Inflammatory breast cancer (IBC) is a rare invasive tumor and characterized by the formation of tumor emboli within dermal lymphatic vessels. The tumor microenvironment (TME) is a key factor in IBC aggressiveness, but its heterogeneity and intercellular role remain incompletely understood. Methods Weighted gene coexpression network analysis (WGCNA) was performed to identify immune infiltration-associated genes. We also depicted cellular communication networks and specific ligand-receptor signaling pathways using the single-cell transcriptomics analysis of IBC and non-IBC samples. Finally, we verified the expression of key proteins by immunohistochemistry. Results Through the intersection of WGCNA module genes and differentially expressed genes between pathological complete response and residual disease samples, GZMB was identified as hub gene which is related to immune infiltration and efficacy of neoadjuvant therapy in IBC. In IBC cohort, patients with high expression of GZMB harbored more immunosuppressive cells thus showed unfavorable prognosis compared with GZMB-low expression group (p<0.05). Subsequent to dimension reduction and clustering, 12 clusters were identified to construct the single-cell atlas between IBC and non-IBC samples. Cellular communication analysis unveiled the heterogeneity of cell communication in IBC. The proportion of immune cells was significantly lower than that of malignant epithelial cells in the cellular composition of IBC. Moreover, it indicated that SPP1 and plasmacytoid dendritic cells were specific in IBC and associated with an immunosuppressive microenvironment in IBC. Immunohistochemical analysis suggested that protein levels of GZMB and SPP1 tended to be higher in the samples from patients with residual disease compared to the patient achieving pathological complete response, though this observation is based on an extremely small sample size. Discussion This study identifies GZMB and SPP1 as potential immunosuppressive-related prognostic biomarkers in IBC patients, reveals the key role of plasmacytoid dendritic cells in remodeling of immunosuppressive microenvironment in IBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/282d00d57fceb6947f3f7237a1980645189f7d26" target='_blank'>
              Integrative analysis of bulk and single-cell transcriptomics identifies factors related to immunosuppressive microenvironment to predict unfavorable prognosis in inflammatory breast cancer
              </a>
            </td>
          <td>
            Kexuan Feng, Xiaoduo Li, Xiaoqian Li, Junjie Liu, Rui Zhang, Heyan Chen, Yunpeng Zhang, Jianjun He, Huimin Zhang
          </td>
          <td>2026-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Sarcomas comprise over 100 mesenchymal malignancies characterised by extreme genomic heterogeneity, ranging from fusion-driven paediatric tumours to highly unstable adult leiomyosarcomas. This genetic complexity shapes tumour behaviour, influences growth and metastasis, and determines how patients respond to therapy. Accurate animal models must reflect not only the molecular characteristics of these tumours, but also their microenvironment and dynamic interaction with the host immune system. The diversity of this group of cancers presents different challenges for animal model selection, particularly as sarcoma-specific resources are very limited. Therefore, we have reviewed in detail several types of models, including: syngeneic (e.g., MCA205 and KRIMS series), chemically induced (e.g., MCA, DMBA), cell-derived xenografts (CDX; e.g., KCS8 and KCS9 osteosarcoma lines), patient-derived xenograft (PDX; e.g., pleomorphic leiomyosarcoma and GIST models), including humanised PDX (huPDX; e.g. HuNOG-EXL), and zebrafish (e.g. tp53M214K PNST and EWS-FLI1 transgenics) - to illustrate their sarcoma-specific use cases and discuss their advantages and limitations. Genetically engineered models and their development are not a subject of this review, as they represent a very broad subject independently and are discussed elsewhere.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f4f4e4d312a546c7a096a58a18c820a9e346cc" target='_blank'>
              Choosing the right animal model for sarcoma research
              </a>
            </td>
          <td>
            Piotr Remiszewski, Eryk Siedlecki, M. Wełniak-Kamińska, M. Mikula, A. Czarnecka
          </td>
          <td>2026-01-10</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Gastric cancer (GC) exhibits molecular heterogeneity and diverse immune cell infiltration patterns closely associated with patient prognosis. However, a comprehensive understanding of the variations in immune cell phenotypes among different patient subgroups still needs to be improved. In this study, we performed a detailed analysis of the tumor microenvironment in GC by integrating 200,466 single cells from 72 patients across six datasets. We classified patients into immune-deserted, B, T, and myeloid cell subtypes. Using genomic and clinical data from TCGA samples, we identified cellular components associated with tumor histology and genotypes. GC patients were stratified into immune-deserted, B cell, T cell, and myeloid cell subtypes, and we described the pathway and transcription factor activity characteristics of different microenvironment subtypes. Integration of bulk RNA-seq data reveals that fibroblasts and endothelial cells were associated with adverse patient outcomes whereas NK and T cells were notably correlated with improved prognosis. Subsequently, we focused on characterizing cancer-associated fibroblasts (CAFs) and discovered that they acquire new functional properties within the tissue microenvironment, providing evidence of CAF plasticity. We constructed a novel four-gene CAF signature including SPARC, EFEMP1, RGS5 and SERPINE1 which may enhance patient stratification and prognostic prediction of GC patients. qPCR analysis revealed that the significant expressions of SPARC, EFEMP1, RGS5 and SERPINE1 were significantly upregulated in gastric cancer tissues compared to the normal tissues. Our study provides insights into the composition of the tumor microenvironment and construction of a four-gene CAF signature associated with clinical prognosis, offering new perspectives for the clinical management of gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6b22dc4e8c3bd98a01915b46ab3333a465c3c7e" target='_blank'>
              Integrative single-cell atlas unveils heterogeneity and prognostic value of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Ziyu Jiang, Xiaomei Zhuang, Junkui Guo, Minyi Zhu, Chunhong Hong, Chunhui Sun, Kaiming Wu, Haofan Yin, Cuncan Deng, P. Jiang
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background The complexities of nucleotide metabolism in breast cancer (BC) cells are not yet fully understood. A deeper exploration of the various tumor subpopulations and their interactions with the tumor microenvironment (TME) could provide important insights into these clinically relevant signaling pathways. Methods We integrated five distinct single-cell enrichment scoring methodologies to conduct a comprehensive enrichment analysis of BC cell populations. The analytical findings underwent subsequent validation using an independent single-cell cohort. Tumor cell clusters were categorized based on their average enrichment scores. Functional analyses were carried out using several tools, including CellChat, Monocle, CopyKAT, SCENIC, and CytoTRACE. The RCTD method was then employed to map the single-cell clusters onto spatial transcriptomics data, facilitating the evaluation of cellular dependencies and pathway activities to differentiate tumor cell subtypes. A prognostic framework was subsequently established using large-scale transcriptomic datasets, enabling prediction of immunotherapy responsiveness. Experimental validation further confirmed expression patterns of pivotal genes implicated in therapeutic outcomes. Results Tumor cells exhibit significantly upregulated nucleotide metabolic activity, enabling their classification into two distinct subgroups: NUhighepi and NUlowepi. Cells within the NUhighepi subgroup demonstrate pronounced malignant phenotypes. Intercellular communication analysis performed with the stLearn platform revealed robust interactions between NUhighepi cells and fibroblasts. Supporting this finding, spatial transcriptomic analysis via MISTy revealed a distinct dependency of NUhighepi on fibroblasts. A robust prognostic model, developed using various machine learning algorithms, was able to predict survival outcomes and responses to immunotherapy. Furthermore, targeted drugs were identified for both the high and low scoring groups. Experimental investigations confirmed the expression of core genes in different breast cancer cells. Discussion In conclusion, we developed a nucleotide metabolism-derived prognostic signature for BC, with DCTPP1 highlighted as a promising biomarker and therapeutic target. These findings provide a valuable framework for early clinical intervention and show promising potential for predicting responses to immunotherapy in BC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502ca564395f021e293ba0e223c1f50472f85541" target='_blank'>
              Spatial transcriptome and single-cell sequencing reveal the role of nucleotide metabolism in breast cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yuejun Pan, Yong Xu, Ke Gong, Songlin Yuan
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Spatial transcriptomics, which deals with tissue architecture in genetic investigation, is an innovative technique for examining cell heterogeneity and tissue organization. This review emphasizes major approaches, include spatially resolved transcriptome methods, immunohistochemistry as well as in situ hybridization, all of which permit the mapping of RNA molecules in their native tissue environment. These methods have proven essential in achieving our understanding of biological events such as tumor evolution, progression of cancer, and cancer tumor stem cell detection. Spatial transcriptomics, the study of patterns of gene expression in space, reveals the intricate nature of the tumor microenvironment (TME) and its effect on cancer biology. Although it delivers insight on the cellular connections that underlie disease, the significance of spatial transcriptomics in multiple organs has expanded.

Although its immense potential, there are still difficulties to be conquered, particularly within the areas of analysis of data, spatial resolution, and integration with other omics data. To be able to fully comprehend the complexities of tissues biology and ailments, this review additionally tackles future potential avenues, including the necessity for greater multiplexing, enhanced resolution, and the combination of functional genomics. With this synthesis, we intend provide an extensive summary of the state of spatial transcriptomics currently and demonstrate that it possesses the potential to improve precision medicine, cancer research, and our understanding of broader biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579b9e172c2769b4ad99d0da42fa24b04f1ee980" target='_blank'>
              Spatial Transcriptomics in Cancer Research: Methods and Applications
              </a>
            </td>
          <td>
            P. K. Kumar, K. M. Rao, Syed Mohiuddin Quadri, Bagadam Aparna, V. U. M. rao
          </td>
          <td>2025-12-18</td>
          <td>International Journal of Current Science Research and Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59359ad678fb5970076eddf100c364fea7a28f75" target='_blank'>
              Adrenergic signaling induces a pro-tumorigenic B cell state in colorectal cancer
              </a>
            </td>
          <td>
            Meike S. Thijssen, Simone L. Schonkeren, Linde Coolkens, Yuchi Zou, Lieve Temmerman, Joëlle de Vaan, M. Idris, Jorunn Vrancken, Kim Wouters, Nathalie Vaes, Marion J. Gijbels, Sharon Scardellato, Erwin Wijnands, Willine J. van de Wetering, F. Verhaegen, L. J. Dubois, Erik Biessen, Carmen López Iglesias, Kim M Smits, A. Bardelli, Stefano Casola, W. Boesmans, V. Melotte
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Simple Summary Diffuse large B-cell lymphoma (DLBCL) is an aggressive cancer with profound heterogeneity. While standard immunochemotherapy is curative for the majority, treatment failure in 30–40% of patients due to refractory or relapsed disease remains a significant clinical challenge, emphasizing the need for better prognostic markers. We investigated the interplay between tumor glycolytic metabolism and the tumor immune microenvironment. We identified four distinct glycolysis-related patterns that were associated with three immune subtypes (known as immune-inflamed, immune-excluded, and immune-desert). Importantly, these patterns stratified clinical outcomes and predicted chemotherapy sensitivity. For example, tumors with an immune-inflamed subtype were more sensitive to doxorubicin. Collectively, our findings propose a metabolism- and immune-based classification framework for DLBCL that may facilitate more precise risk stratification and treatment optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e894f2d6a04501c694424859815514794b12befc" target='_blank'>
              Glycolysis-Driven Immune Subtypes in the Tumor Microenvironment Determine Clinical Outcomes in Diffuse Large B-Cell Lymphoma
              </a>
            </td>
          <td>
            Ying Du, Zhenyu Liu, Chuanyan Liu, Yining Yuan, Mengkun Fang, Tong Zhou, Mengge Pan, Weilu Xu, Xu Liu, Peipei Xu
          </td>
          <td>2025-12-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43834c7632e582a60435566324798a1fd37dbe2b" target='_blank'>
              Fibroblast heterogeneity in 2025
              </a>
            </td>
          <td>
            A. Croft, A. Hackland
          </td>
          <td>2025-12-19</td>
          <td>Nature Reviews Rheumatology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc6a6f856a7083dd2ac5b388edeaab80c7f5a72" target='_blank'>
              A spatially resolved atlas of gastric cancer characterises a lymphocyte-aggregated region.
              </a>
            </td>
          <td>
            Sen Gao, Shishang Qin, Dongfang Wang, Anqiang Wang, Linna Zhu, Yang Li, Qiang Shi, Hongtao Fan, Yufei Bo, Yunshan Zhong, Yuxuan Sun, Kun Dong, Liqin Fu, Ranran Gao, Yan Wu, Ye Liang, Luyujie Huang, Xueda Hu, Xianwen Ren, Z. Bu, Jiafu Ji, Zemin Zhang
          </td>
          <td>2026-01-27</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec774215b12f2be4ebfe5634529efaca54fcef6" target='_blank'>
              Toward Personalized Treatment of Urogenital Cancers: The Role of Patient-Derived Organoids.
              </a>
            </td>
          <td>
            S. Sagliocchi, Michele Musone, Stefano Chianese, A. G. Cicatiello, Silvia Del Mastro, Francesco Del Giudice, M. Dentice, F. Crocetto
          </td>
          <td>2026-01-08</td>
          <td>Oncology and therapy</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 Computational pathology has emerged as an attractive option for improving risk stratification in prostate cancer (PCa), but current approaches either lack interpretability or focus solely on tumor morphology, potentially limiting their utility. We aimed to identify an interpretable, immune microenvironment-derived computational pathology biomarker for PCa.



 We retrospectively identified two independent cohorts (Discovery and Validation) with localized PCa who underwent radical prostatectomy (RP) and had digitized H&E-stained whole-slides images (WSIs) and longitudinal outcomes data. We identified a third cohort from The Cancer Genome Atlas (TCGA) with localized PCa which was treated with RP and had WSIs, bulk RNA sequencing, and whole-exome sequencing. Immune cells were identified from WSIs using a publicly available deep learning model (CellViT), and spatially dense immune clusters were quantified using density-based spatial clustering of applications with noise (DBSCAN). CIBERSORTx was utilized for immune cell deconvolution and TRUST4 for immune receptor repertoire reconstruction from bulk RNA sequencing. Cox proportional hazard regression was used to examine associations between clinicopathologic features and the primary outcome of time to distant metastasis (DM), as well as secondary outcomes of biochemical recurrence (BCR) and overall survival (OS).



 Median follow-up periods for the Discovery (n=272) and Validation (n=218) Cohorts were 12.6 and 8.1 years, respectively. In the Discovery Cohort, median age was 63, 14% had Gleason 8-10 disease, 94% had pT2-T3a disease, and median prostate-specific antigen level at diagnosis was 6.2 ng/mL. Median immune cell proportion was 4.3% (interquartile range 3.0-5.9%), and median immune cluster density was 2.4 per 400 mm2 of tissue (interquartile range 0-7.1). Similar values were observed in the Validation Cohort. In both cohorts, immune cell proportion was not associated with BCR, DM, or OS (P>=0.09). In the Discovery Cohort, while increasing immune cluster was not associated with BCR (P>=0.10), it was independently associated with a decreased risk of DM for Gleason 8-10 (adjusted hazard ratio [AHR] 0.42, 95% confidence interval [CI] 0.19-0.93) but not Gleason 6-7 patients (AHR 1.26, 95% CI 0.77-2.05; Pint=0.020). Similar results were observed in the Validation Cohort, where increasing immune cluster was not associated with BCR (P>=0.10) but was associated with a lower risk of DM in Gleason 8-10 disease (AHR 0.60, 95% CI 0.37-0.98), though not Gleason 6-7 (AHR 1.19, 95% CI 0.74-1.91; Pint=0.043). In TCGA Cohort (n=326), immune cluster was not associated with somatic alterations. For Gleason 8-10 but not Gleason 6-7 disease, high-cluster samples were enriched for CD8+ T cells, activated memory CD4+ T cells, and Tregs (P<=0.037), as well as clonal T cell populations (P<=0.039).



 Our findings nominate immune spatial clustering as a novel, interpretable computational pathology biomarker and provide insights into the unique immune features of high-grade PCa.



 David D. Yang, Aya Abdelnaser, Alexander J. Haas, Jeremiah Wala, Alfred A. Barney, Eddy Saad, Jett P. Crowdis, Cora A. Ricker, Seifeldin Awad, Bora Gurel, Jihye Park, Martin T. King, Paul L. Nguyen, Toni K. Choueiri, David J. Einstein, Steven P. Balk, Alok K. Tewari, Johann S. de Bono6, Keyan Salari, Mary-Ellen Taplin, Chin-Lee Wu, Eliezer M. Van Allen. . Immune spatial organization predicts metastasis risk in aggressive localized prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aace1806fe7d01338e5e9857c054699a561213e" target='_blank'>
              Abstract PR001: Immune spatial organization predicts metastasis risk in aggressive localized prostate cancer
              </a>
            </td>
          <td>
            D. Yang, Aya Abdelnaser, Alexander J. Haas, Jeremiah Wala, Alfred A. Barney, Eddy Saad, Jett Crowdis, Cora A. Ricker, Seifeldin Awad, B. Gurel, Jihye Park, Martin T. King, P. L. Nguyen, T. Choueiri, David J Einstein, Steven P. Balk, A. Tewari, J. D. De Bono, Keyan Salari, M. Taplin, Chin-Lee Wu, E. V. Van Allen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6c25b39df997323e228b3c6d265efc22e9a7958" target='_blank'>
              Multi-omics characterization of IDH-mutant astrocytoma-derived cell lines reveals NOTCH-regulated plastic quiescent astrocyte-like state
              </a>
            </td>
          <td>
            L.Garcia, C.Granotier-Beckers, D.Pineau, D.Sika, S.Hideg, C.Labadie, K.Aguilar Cázarez, J.Kundnani, L.Stuani, L.Gauthier, F.Boussin, M-A.Mouthon, S.Urbach, K.El Koulali, M.Séveno, C.Ripoll, K.Daddi, M.Verreault, A.Idbaih, S.O’Connor, C.Plaisier, L.Zhang, M.Zheng, V.Rigau, H.Duffau, J. Hugnot
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple Negative Breast Cancer (TNBC) is an aggressive subtype of breast cancer with limited treatment options and poor prognosis. Coupled with this, TNBC shows a high rate of metastasis which is known to be aided by tumour vasculature. Endothelial cells that form the lining of the tumour vasculature exhibit distinct genotype and phenotype differences compared with normal tissue vasculature. However, little is known about endothelial signatures that drive metastasis from a primary tumour, particularly in the context of immune infiltration. In this study, we utilized GeoMX Digital Spatial Profiling to investigate spatial proteomics differences in endothelial cells between primary and secondary sites of TNBC. By segmenting tissues using epithelial (PanCK), immune (CD45), and endothelial (CD31) markers, we analysed the microvasculature for a panel of 79 target proteins. In paired primary and secondary TNBC tissues, we identified significant downregulation of fibronectin (− log2(Fold-Change) = − 1.7, p < 0.001) in secondary sites. Specifically in epithelial regions, S100B was found to be downregulated in secondary microvasculature when compared to primary tumours. Additionally, metastasis-free primary tissues exhibited upregulated expression of S100B when compared to primary tissues that metastasized. Our study highlights the potential contributions of microvasculature to metastatic progression in TNBC, presenting new opportunities to explore them as potential biomarkers of TNBC metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9355568219b2a2ea616006baa27f93ab8c621b2" target='_blank'>
              Digital spatial profiling identifies features of primary and locoregional metastatic vasculature in triple negative breast cancer
              </a>
            </td>
          <td>
            Akhilandeshwari Ravichandran, Kyle Upton, Shiva Taheri, Cheng Liu, Kaltin Ferguson, Mark N. Adams, Laura J. Bray
          </td>
          <td>2026-01-19</td>
          <td>Clinical & Experimental Metastasis</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Glioblastoma is the most aggressive adult brain tumour, characterised by resistance to therapy and high recurrence due to diffuse infiltration. We developed a physiologically relevant co-culture model, combining patient-derived glioblastoma cell lines with cortical-like neural spheroids differentiated from human induced pluripotent stem cells. Using high-content imaging, we demonstrate that GBM20 and GBM1 cell lines migrate directionally along axons toward neural spheroids in live imaging assays and infiltrate spheroids extensively in endpoint assays, unlike non-cancerous neural stem cells. A proof-of-principle drug screen identified PF 573228 (FAK inhibitor) and motixafortide (CXCR4 inhibitor) as potent suppressors of GBM20 and GBM1 infiltration, respectively. Bulk RNA sequencing revealed gene expression profiles correlating with invasive behaviour and drug sensitivity. This platform offers a valuable model for studying glioblastoma infiltration along axons and provides proof-of-principle that migration can serve as a measurable and actionable phenotype to screen therapeutic vulnerabilities in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f72fcee02e1f1c7cefb3147af1fbbfca5819be15" target='_blank'>
              A human iPSC-based neural spheroid platform for modelling glioblastoma infiltration using high-content imaging
              </a>
            </td>
          <td>
            V. Tsang, Federica Riccio, Aimee S Wilson, Hannah Nudds, Jason D Coombes, Heiko Wurdak, Harry Bulstrode, Ivo Lieberam, Davide Danovi
          </td>
          <td>2025-12-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction During embryogenesis, specific morphogen gradients are essential for inducing tissue pattern formation. In two-dimensional (2D) human pluripotent stem cell (hPSC) culture, distinct patterns can emerge in hPSC colonies without external morphogen gradients, implying that critical intrinsic factors may induce spatial organization. However, studying the mechanism is challenging due to the lack of efficient spatial labels. Methods We employed the mitochondrial membrane potential (MMP) probe JC-1 to stain and track cells within hPSC colonies. Using this tool, we assessed metabolic patterns under different culture coatings and manipulated pathways using mTOR and ROCK inhibitors. Results We identified JC-1 as a durable spatial tracker, revealing a clear metabolic pattern in hPSC colonies, significantly influenced by coating materials (integrin-stimulating matrices vs. E-cadherin). This metabolic pattern correlated with spatial mesodermal cell fate under BMP4 induction. Modulation of the mTOR pathway altered the metabolic pattern and subsequent mesoderm induction. Conclusion This study reveals that intrinsic metabolic patterns predispose hPSCs to spatial organization of cell fate and highlights JC-1 as a potent spatial marker for studying tissue patterning mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d0ee60c209fd2594c9a8c367a7962559c97908" target='_blank'>
              Metabolic patterns predispose human pluripotent stem cells to spatial organization of cell fate
              </a>
            </td>
          <td>
            Chunhao Deng, Zhaoying Zhang, Xia Xiao, Carlos Godoy-Parejo, Faxiang Xu, Chengcheng Song, Huanyi Lin, Qinru Li, Shicai Fang, Weiwei Liu, Guokai Chen
          </td>
          <td>2026-01-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) exhibits extensive intratumoral heterogeneity on multiple biological scales, contributing to variable clinical outcomes and limiting the effectiveness of conventional TNM staging, which highlights the urgent need for multiscale integrative analytic frameworks. The lipid-deficient de-clear cell differentiated (DCCD) ccRCC subtype, defined by multi-omics analyses, is associated with adverse outcomes even in early-stage disease. Here, we establish a hierarchical cross-scale framework for the preoperative identification of DCCD-ccRCC. At the highest layer, cross-modal mapping transferred molecular signatures to histological and CT phenotypes, establishing a molecular-to-pathology-to-radiology supervisory bridge. Within this framework, each modality-specific model is designed to mirror the inherent hierarchical structure of tumor biology. PathoDCCD captured multi-scale microscopic features, from cellular morphology and tissue architecture to meso-regional organization. RadioDCCD integrated complementary macroscopic information by combining whole-tumor and its habitat-subregions radiomics with a 2D maximal-section heterogeneity metric. These nested models enabled integrated molecular subtype prediction and clinical risk stratification. Across five cohorts totaling 1,659 patients, PathoDCCD reliably recapitulated molecular subtypes, while RadioDCCD provided reliable preoperative prediction. The consistent predictions identified patients with the poorest clinical outcomes. This cross-scale paradigm unifies molecular biology, computational pathology, and quantitative radiology into a biologically grounded strategy for preoperative noninvasive molecular phenotyping of ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2906aadb2798289d6c075f86ae197fcc81c4cc58" target='_blank'>
              Multiscale Cross-Modal Mapping of Molecular, Pathologic, and Radiologic Phenotypes in Lipid-Deficient Clear Cell Renal CellCarcinoma
              </a>
            </td>
          <td>
            Ying Cui, Dongzhe Zheng, Ke Yu, Xiyin Zheng, Xiaorui Wang, Xinxiang Li, Yan Gu, Lin Fu, Xinyi Chen, Wenjie Mei, Xin-gui Peng
          </td>
          <td>2025-12-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a self-renewing population capable of fueling long-term tumor growth. In gastric cancer, the identity of CSC populations remains unclear. In this study, we established a gastric CSC population marked by the water channel protein aquaporin-5 (AQP5), which resides in human and mouse pyloric tumors. Using multiple organoid and mouse models, we found a requirement for AQP5+ CSCs in both initiating and sustaining cancer progression and demonstrated that AQP5 expression also directly promotes tumor growth and invasion in a WNT, PI3K (phosphatidylinositol 3-kinase), and MAPK (mitogen-activated protein kinase)-dependent manner. Beyond primary cancers, AQP5 further enriches for a functional CSC population in metastatic tumors. Together, our findings support a CSC model in gastric tumors that may have application for therapeutic strategies targeting CSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b857f221b4120247f60ae689da7ba8ea312dc26b" target='_blank'>
              AQP5: A functional gastric cancer stem cell marker in mouse and human tumors.
              </a>
            </td>
          <td>
            H. Lim, Swathi Yada, Kazuhiro Murakami, B. Lee, Sowmya Sagiraju, Phyllis Phuah, Tanysha Chi-Ying Chen, Fidelia B Alvina, S. Tan, Kaushal Krishna Kaslikar, Nur Syahirah Binte Ruhazat, Menaka Priyadharsani Rajapakse, Katzrin Bte Ahmed Murad, Snezhina Kancheva, Liang Thing Tan, Seri Mustafah, Jimmy Bok Yan So, Nick Barker
          </td>
          <td>2025-12-18</td>
          <td>Science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits profound cellular heterogeneity, the understanding of which is critical for improving prognosis and therapy. Using single-cell RNA sequencing of 32,247 cells from human HCC samples, we characterized the tumor ecosystem and identified five malignant hepatocyte subpopulations with distinct molecular profiles and stage-specific enrichment. Among these, the S100A6⁺ C1 and S100A9⁺ C4 subpopulations were predominantly associated with advanced tumors and actively remodeled the tumor microenvironment through enhanced signaling pathways such as MDK and MIF. We further identified PGAM2 as a key transcriptional regulator in early-stage tumors, whose activity correlated with sialylation—a process linked to immune evasion. Based on these findings, we developed a prognostic model integrating PGAM2 and sialylation-related genes, which robustly stratified patients into high- and low-risk groups with significantly different survival outcomes, immune contextures, and predicted therapeutic responses. Functional experiments validated AGRN, a component of the signature, as a functional driver of HCC proliferation and invasion. Collectively, our results decode the cellular and molecular heterogeneity of HCC, provide a clinically relevant prognostic tool, and highlight potential targets for further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d560ef3a66a46cb286a615fdcaac30d8c4ef51e5" target='_blank'>
              Machine learning-driven comprehensive profiling of tumor heterogeneity and sialylation in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kaiqiang Tang, Lu Han, Junlin Li, Kang Li
          </td>
          <td>2025-12-17</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Bladder cancer (BCA) shows significant prognostic differences between non-muscle-invasive (NMIBC) and muscle-invasive (MIBC) forms. While NMIBC frequently recurs and can progress to invasive disease, reliable biomarkers to monitor this transition are lacking. Extracellular matrix (ECM) remodeling is a critical factor influencing tumor aggressiveness, yet the key regulators of ECM changes across BCA stages remain unclear. In this study, we investigate the role of COL1A2 in ECM-related tumor biology and its potential as a prognostic biomarker for BCA progression. Methods We utilized a multi-step bioinformatics pipeline, analyzing RNA-seq data from TCGA and GEO datasets to identify molecular differences between NMIBC and MIBC. Prognostic markers were prioritized via differential expression analysis, Cox regression, and Kaplan–Meier survival analysis. The regulatory network was explored using protein-protein interaction analysis, and ECM-related activity was quantified through ssGSEA. Cell-type-specific insights were gained through single-cell RNA-seq analysis, and intercellular communication was deciphered using CellChat. Functional validation was performed through in vitro knockdown experiments in BCA cell lines. Results COL1A2 emerged as a key prognostic ECM-related gene associated with MIBC. Single-cell RNA-seq analysis revealed that COL1A2 and ECM components were predominantly enriched in matrix cancer-associated fibroblasts (CAFs), with PTK2 (FAK, focal adhesion kinase) upregulated in epithelial cells undergoing epithelial-mesenchymal transition (EMT). CellChat analysis uncovered a dominant COL1A2-mediated signaling axis from matrix CAFs to EMT epithelial cells via COL1A1/2–SDC1/4 ligand-receptor interactions. Functional assays confirmed that COL1A2 knockdown significantly impaired MIBC cell invasion and migration by suppressing ECM remodeling and EMT. Conclusion Our results suggest that the COL1A2–ECM–FAK signaling axis plays a critical role in MIBC progression, and COL1A2 could serve as a potential biomarker and therapeutic target for muscle-invasive bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda2a0be0d617f715d6d80984a257431f3977b47" target='_blank'>
              Integrative bulk and single-cell transcriptomic analysis reveals COL1A2-driven ECM remodeling and focal adhesion signaling associated with the transition from non-muscle-invasive to muscle-invasive bladder cancer
              </a>
            </td>
          <td>
            Menglu Li, Xinwei Liu, Yuhui Xue, Yichen Lu, Zhiqiang Chen, Yuwei Zhang, Weiguo Chen, Shan-Chao Zhao, Ke Wang, Ninghan Feng
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Oncology</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Single-cell proteomics (scP) is a crucial complement to transcriptomics, offering deeper insight into cellular heterogeneity, disease mechanisms, and therapeutic vulnerabilities in samples such as 2D cell culture, 3D models, and patient tissue. While transcriptomics enables high-throughput gene expression characterization, RNA levels frequently do not correlate with protein levels even in the same cell. Furthermore, protein isoforms, posttranslational modifications, and complexes are missed by transcriptomic analyses. This review explores modern scP technologies, including flow and mass cytometry, single-cell mass spectrometry, immunohistochemistry, cyclic imaging, and imaging mass cytometry applied to both dissociated and spatially preserved samples. We emphasize applying these techniques to organ-on-a-chip, organoids, spheroids, and intact tissues, highlighting advances in spatial resolution and multiplexing. We also discuss the trade-offs between throughput, spatial fidelity, and protein selectivity across platforms. Finally, we identify key measurement gaps, suggesting future directions toward spatially resolved scP clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70198449e3031642a68cc8bde8c19dcf967028c5" target='_blank'>
              Single-Cell Protein Assays in Context: From 2D to 3D and In Situ Analysis.
              </a>
            </td>
          <td>
            Jocelyn S Baker, Neica I. Joseph, Alison Lao, Julea Vlassakis
          </td>
          <td>2025-12-10</td>
          <td>Annual review of analytical chemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The process of migration and colonization is important in evolution; for example, modern humans experienced multiple waves of migrations out of Africa. However, no data cover the spatio-temporal patterns sufficiently to be truly informative. Metastatic cancer provides a unique in vivo model to study these processes through rapid somatic evolution. Here, we apply the high-resolution sampling technique (Dense 3D Crypt-scale Sampling) to analyze hundreds of spatially mapped micro-samples from the primary colorectal cancer and liver metastases in two representative cases. This would be analogous to recording the "out-of-Africa" events in two repeats. Our results support that liver metastases arise from polyphyletic and polyclonal seeding events where multiple, genetically distinct clones colonize a new site together. Following colonization, these multi-clonal populations can evolve into distinct spatial architectures: segregated territories formed by cells with low motility, or highly intermixed patterns driven by high motility. The colonization (or seeding) process begins within the first third of the primary tumor's progression, creating a large number of widespread but clinically undetectable micrometastatic colonies. These findings support a model where metastatic competence is not an intrinsic trait of a single "winner" clone but an emergent property of multiple concurrent clones. Collectively, our work supports metastasis as a multi-stage process initiated early in tumor development, characterized by continuous polyclonal dissemination and the formation of spatially distinct clonal architectures. This general pattern may echo the ecology of migration and colonization in organismal evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9d8108fc71756559fb44982b7725fb8d93fb854" target='_blank'>
              Full spatio-temporal analyses of migration and colonization in evolution - Dense 3D mapping of cancer metastases provides new insights.
              </a>
            </td>
          <td>
            Qihang Chen, Senmao Li, Xianrui Wu, Qing Xu, Ranran Zhu, Yongsen Ruan, Ao Lan, Zihan Liu, Jiarui Weng, Yanjiang Zhao, Xiying Xu, Xinyue Qi, Jinhong Lai, Leyi Xiao, Ping Lan, Chung-I Wu, Bingjie Chen
          </td>
          <td>2026-01-12</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Epigenetic reprogramming is a major contributor to therapy resistance in prostate cancer (PCa). Indeed, chromatin accessibility profiling classified PCa into distinct subtypes, predictive of clinical outcome. Histone modifications influence nucleosome positioning, ultimately dictating chromatin compactness. The histone methyltransferase G9a, frequently upregulated in advanced PCa, catalyzes a repressive histone modification (di-methylation of histone H3 lysine 9) that signals for gene transcriptional repression. CM-272 is a reversible and highly selective inhibitor against G9a activity, and we have previously shown in cell line models of PCa that it depicted strong anti-tumoral effects. However, fundamental aspects of altered transcriptomic programs remain unexplored. Here, we validated, in vivo, the CM-272 effects in advanced PCa, and explored the molecular mechanisms underscoring G9a inhibition-mediated effects. Specifically, the PCa patient-derived xenograft model BM18 was used to assess the mode of action of CM-272 in vivo. Moreover, RNA sequencing of tumor tissues was performed to further explore the underlying gene expression signature. Finally, chromatin immunoprecipitation followed by qPCR was used to investigate the underlying functional consequence of epigenetic targeting. Our results showed that CM-272 significantly delayed BM18 tumor growth, specifically impairing tumor cell G9a activity and proliferation. Transcriptomic profiling revealed 119 significantly upregulated and 99 downregulated genes in response to G9a inhibition. Notably, by integrating our gene expression dataset with published chromatin accessibility data, an increase in the androgen receptor (AR) signature and a concomitant decrease in the neuroendocrine (NE) features were observed in CM-272-treated samples. Additionally, pathway enrichment and master regulator analysis pinpointed both the androgen receptor and TP53 networks as significantly upregulated in treated tissues. In line with the reduction of the NE score, the MYC/MYCN were the most significantly downregulated transcription factors. Moreover, under the androgen pathway, upregulation of AZGP1, STK39, and DPP4 was observed. At the promoter level, a significant reduction in H3K9me2 levels was detected in treated samples, allowing, in turn, an increase in AR binding. Additionally, upon treatment, higher p53 binding was detected at the promoter of the p53-repressed genes AURKA, CDC25B, and CD44, whose expression was downregulated. Overall, targeting G9a with CM-272 might hold promising therapeutic potential for the management of advanced disease by modulating relevant tumoral transcriptional programs associated with core vulnerabilities in PCa.



 Filipa Moreira-Silva, Mark Zimmerli, Panagiotis Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio. G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5911786a7c980e0c86fa5660d05b013ec96ea309" target='_blank'>
              Abstract A045: G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay
              </a>
            </td>
          <td>
            Filipa Moreira-Silva, Mark Zimmerli, P. Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The cancer stem cell (CSC) hypothesis proposes that rare tumor-initiating cells with stem-like properties drive cancer progression and metastasis. Through comprehensive analysis of prostate cancer patient-derived xenografts (PDX), we demonstrate that all cancer cell types—not exclusively stem-like cells—can initiate tumors in mice, contradicting a core CSC tenet. CD44, commonly used to identify CSCs, proved inconsistent in prostate cancer. However, stem-like cancer cells do exist, characterized by expression of stem cell transcription factors (LIN28A, NANOG, POU5F1, SOX2) and low β-2 microglobulin (scTF⁺B2Mlo). We show that differentiated adenocarcinomas can be experimentally reprogrammed to stem-like small cell carcinomas through scTF expression, while stromal signaling molecules like proenkephalin (PENK) induce differentiation. These findings reveal that prostate cancer progression involves dynamic dedifferentiation due to loss of stromal signaling rather than clonal expansion of rare CSCs. Cancer cells exhibit remarkable plasticity, undergoing reversible differentiation-dedifferentiation cycles independent of mutation burden, suggesting differentiation therapy as a promising treatment strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f552fec4d245302e872d3ce38c426997c64285" target='_blank'>
              The Cancer Stem Cell Concept as Applied to Prostate Cancer
              </a>
            </td>
          <td>
            Liu Alvin Y
          </td>
          <td>2026-01-22</td>
          <td>IgMin Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer is known to be a disease of altered cellular signaling; however, the relationship between mutation-specific changes to signal transduction and the phenotypic consequences produced remains poorly understood. Here, we investigate two common breast cancer driver mutations, the PIK3CAH1047R mutation and the ErbB2 amplification, both of which activate the PI3K-Akt pathway but paradoxically drive distinct cellular outcomes. Indeed, in nontransformed mammary epithelial cells, PI3KH1047R expression induced features of epithelial-mesenchymal transition (EMT), while ErbB2amp cells exhibited a hyperproliferative phenotype. Characterization of PI3K axis signaling revealed that ErbB2amp cells display prolonged, stimulus-dependent PI3K activation, whereas PI3KH1047R cells show constitutive, ligand-independent signaling. To test whether these distinct dynamics contribute to the phenotypic responses, we employed an iLID-based optogenetic system that enables precise, tunable control of endogenous PI3K activity. Using this tool to mimic the mutation-specific dynamics in MCF10A mammary epithelial cells, we found that PI3K signaling patterns alone were sufficient to reproduce key features of the PIK3CA H1047R-associated EMT phenotype but not the ErbB2-associated proliferative phenotype. These findings suggest that the temporal encoding of pathway activity, not merely its magnitude, can drive some phenotypic changes in oncogenic progression, explain how distinct mutations within a common signaling pathway can produce divergent cellular phenotypes, and provide a workflow for interrogating the functional consequences of changes in signaling dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51c6978b325cbac3a14efcc2bf1ac41d165bf7ce" target='_blank'>
              Oncogenic Alterations in PI3K Signaling Emulated Optogenetically Recapitulate Some Phenotypic Changes in Mammary Epithelia.
              </a>
            </td>
          <td>
            Keith A. Gagnon, Veronica W Hui, Terry Ching, Amy E. Stoddard, Esther Koh, J. Eyckmans, Ahmad S. Khalil, Christopher S. Chen
          </td>
          <td>2026-01-19</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Despite advances in cancer therapies, treatment failure from resistance and recurrence remains a major clinical challenge. Therapy-induced senescence (TIS), a state of stable cell cycle arrest with sustained metabolic activity, has emerged as a driver of inflammation, tumor persistence, and relapse. However, the heterogeneity of TIS complicates its detection and targeting. Here, we applied a multi-modal strategy to characterize metabolic alterations in senescent cancer cells induced by doxorubicin or γ-irradiation across three tumor cell lines: MCF7, HeLa, and TPC-1. Mitochondrial dysfunction was assessed using MitoTracker and JC-1 staining, while two-photon excitation fluorescence (TPEF) microscopy enabled label-free visualization of intracellular NAD(P)H and FAD distribution. Lipid remodeling was evaluated by MALDI mass spectrometry imaging, and RNA sequencing was performed on control, senescent, and engulfing-senescent MCF7 cells to identify differentially expressed genes and enriched pathways. Senescent cells displayed mitochondrial dysfunction, with altered NAD(P)H/FAD distribution and decreased membrane potential. TPEF confirmed redistribution of coenzymes, reflecting redox changes. Lipidomics revealed consistent remodeling, notably involving cardiolipin precursors. Transcriptomic profiling showed engulfing-senescent MCF7 cells possess a distinct signature marked by increased lipid metabolism, endocrine signaling, and cell–cell communication. Overall, our findings reveal conserved and cell type–specific metabolic traits of TIS, highlighting metabolic vulnerabilities for senolytic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4f815c5ea5ff8465ffcedac66d5a28bbf1f8878" target='_blank'>
              Metabolic profiling of therapy-induced senescent cancer cells via TPEF, MALDI-MS, and RNA-sequencing
              </a>
            </td>
          <td>
            S. Ghislanzoni, F. Padelli, Matteo Niero, A. Bertolotti, A. Belfiore, Simone Torelli, A. Bresci, Andrea Masella, Silvia Betti, Dario Polli, Luca Agnelli, I. Bongarzone
          </td>
          <td>2025-12-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Primary tracheobronchial tumors (PTBTs) can often be successfully treated by surgery; however, more effective strategies are still warranted for advance tumor cases. The present study aimed to provide a comprehensive understanding of the molecular and cellular features of PTBTs, along with the tumor microenvironment (TME). By performing single-cell RNA, we revealed a distinct TME landscape of PTBTs containing 52,739 single cells. For instance, these three pathological types were found have heterogeneous fibroblast populations, immune cells were more activated in inflammatory myofibroblastic tumor (IMT). Notably, a subpopulation of BIRC5+ cells was identified in primary tracheal squamous cell carcinoma (SCC) and adenoid cystic carcinoma (ACC) samples, which may have similar properties to cancer stem cells (CSCs). Moreover, IMT was characterized by the infiltration of pro-inflammatory immune cells, and might be a benign tumor. Importantly, based on ligand-receptor analysis, the PPIA/CD147 pair was found to be strongly upregulated during tumor and CD8+ T cell interactions. By applying multiplex immunofluorescence for selected markers, we validated the presence of the separate cell entities. Our findings provide deeper insights into the biology of PTBTs, which may be beneficial for advancing the diagnosis and therapy of PTBTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc41a4f5248143a0c3cf35d4c57326e220116bee" target='_blank'>
              A single-cell atlas reveals the tumorigenesis and microenvironment status of primary tracheobronchial tumors
              </a>
            </td>
          <td>
            Chen Shu, Xiyang Tang, Kaifu Zheng, Runze Zhang, Yulong Zhou, Mengchao Li, Weiguang Du, Yuquan Bai, Yang Shen, Yujian Liu, Haitang Yang, Feng Yao, Xiaolong Yan, Nan Ma, Jinbo Zhao
          </td>
          <td>2025-12-13</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0b876781bf210db756c35db260709c192ea56b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution.
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A McIntyre, Akimasa Hayashi, Nicolas Lecomte, Marc Hilmi, Wungki Park, Nan Pang, Eileen M. O’Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2026-01-22</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="We continue to regard stomach tumors as uniform molecular entities while neglecting microanatomical signaling ecologies that genuinely cause resistance, which is a pressing issue that the manuscript highlights. To drive a transition from descriptive atlases to practical, spatially actionable research, this letter adopts a purposefully critical posture, not one that is hostile. The AREG–SOX9 axis is not just “interesting”; it is a perfect example of why many targeted medicines fall short: we have not figured out where the biology is focused. Through the AREG– ERBB2 axis, amphiregulin (AREG) produced by iCAFs is found to contribute to drug resistance and enhance tumor stemness by upregulating SOX9 expression in tumour cells [ 1 ] . Although this claim is based on correlation evidence that confuses proximity with causality, it is mechanistically reasonable. Spatial–functional corroboration is necessary for a genuinely critical perspective: are AREG-high iCAFs close to ERBB2-active clones? Does SOX9 induction decrease, and ERBB2 inhibitor sensitivity return when AREG is blocked in situ ? The axis remains a theory rather than a clinically validated mechanism, given the absence of direct spatial mapping of ligand–receptor activity (e.g., phospho-ERBB2 colocalized with AREG sources). We suggest using ex vivo slice assays with AREG neutralization ± trastuzumab and in situ proximity ligation/phospho-mapping on identified interface zones as an instantaneous, high-yield test to determine causation. The growth of bacteria-induced stomach cancer cells, elevated β - catenin levels, and development of stem cell-like characteristics all depended on SOX9 [ 2 ] . A compelling multi-hit concept (microbiota → stroma → tumor) is introduced, citing">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/079610ca0b7f290f8f651c837134d136fd948c2b" target='_blank'>
              Spatial confinement of the AREG–SOX9 signaling circuit: rethinking tumor–stroma interactions in gastric cancer
              </a>
            </td>
          <td>
            Asra Amjad, Muhammad Talha Usman, Umair Ali, Raghabendra kumar Mahato
          </td>
          <td>2025-12-12</td>
          <td>Annals of Medicine and Surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5407cdb757b46038485ad236efb4e8ad5a8a93" target='_blank'>
              Tumor-associated macrophages promote chemoresistance to Paclitaxel via activating NOTCH2-JAG1 juxtacrine signaling.
              </a>
            </td>
          <td>
            Fazhi Yu, Qin Zhou, Weiqiang Yu, Tong Zhou, Cheng Cao, Yijia Xie, Peng Zhang, Hanyuan Liu, Wei He, Aoxing Cheng, Xiaopeng Ma, Qingfa Wu, Qi Zhao, Jing Guo, Kaiguang Zhang, Ying Zhou, Jue Shi, Zhenye Yang
          </td>
          <td>2026-01-10</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1d3b3474c465409ae2cb832c168d62bf2850c00" target='_blank'>
              Decoding the tumor immune landscape: emerging TIL subsets as prognostic biomarkers and therapeutic targets.
              </a>
            </td>
          <td>
            Rahaman Shaik, Sai Abhistika Royyala, Bhanu Inapanuri, Syeda Fatima Sarwar, Shaheen Mahira, Shaik Azeeza
          </td>
          <td>2026-01-24</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Identifying the cell of origin that harbors an initial driver mutation is key to understanding tumor evolution and for the development of new treatments. For isocitrate dehydrogenase (IDH)-mutant gliomas, the most common malignant primary brain tumor in young adults, the cell of origin is currently poorly understood. We conducted deep sequencing on 142 tissues from 70 individuals comprising tumors, peritumoral cortex or subventricular zones, and blood. Low-level IDH mutations were found in the peritumoral cortex in 37.9% (11 of 29) of patients. Integrating cell-type-specific mutation analysis, the direction of clonal evolution, spatial transcriptomics from patient brains, and a cancer mouse model arising from mutant oligodendrocyte progenitor cell, we determined that glial progenitor cells harboring an initial IDH mutation were responsible for the development of IDH-mutant gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87878ebfd4fd1887849bd103fe178e14f5e98a11" target='_blank'>
              IDH-mutant gliomas arise from glial progenitor cells harboring the initial driver mutation.
              </a>
            </td>
          <td>
            Jung Won Park, Jiehoon Kwak, K. Kim, Saehoon Jung, Chang Hyun Nam, Hyun Jung Kim, Sang Mee Lee, Chanho Choi, Yongjin Ahn, Ji-Hyung Park, Jihwan Yoo, J. Shim, H. Cho, E. Kim, Chungyeul Kim, Sangjeong Ahn, Stefan Pusch, Andreas von Deimling, Jong Hee Chang, Se Hoon Kim, Hoon Kim, Young Seok Ju, S. Kang, J. H. Lee
          </td>
          <td>2026-01-08</td>
          <td>Science</td>
          <td>2</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Background Primary pulmonary sarcomas (PPS) and pulmonary sarcomatoid carcinoma (PSC) are rare and aggressive diseases that pose significant diagnostic challenges, requiring extensive sampling and comprehensive evaluation. To date, the single‐cell characteristics and distinctions between these two conditions have not been thoroughly investigated. Methods In this study, we employed single‐nucleus RNA sequencing (snRNA‐seq) to characterise the cellular heterogeneity of PSC and PPS. Our analysis included 20 PSC samples, seven PPS samples and two non‐malignant control samples obtained from adjacent normal tissue. Results Our results revealed that the majority of cells in PSC were of epithelial origin, while fibroblasts predominated in PPS. Specifically, AT2 cells, a major source of epithelial cells in PSC, underwent malignant transformation primarily through epithelial–mesenchymal transition, suggesting AT2 cells may serve as the origin of PSC. High Mobility Group AT‐Hook 2 (HMGA2) expression was elevated in malignant AT2 cells of PSC and correlated with an unfavourable prognosis. Moreover, MET‐mutated patients have a significantly higher expression level of HMGA2 (p < .001). In PPS, fibroblasts constituted the majority, only lipofibroblasts exhibited malignant features. A direct comparison between PSC and PPS lipofibroblasts revealed largely similar expression profiles, with the exception of an enrichment in DNA repair pathways specifically observed in PPS lipofibroblasts. Conclusion These findings provide novel insights of PSC and PPS at the single‐cell level. Key points Distinct Cellular Origins: PSC arises primarily from epithelial (AT2) cells via EMT, whereas PPS is predominantly fibroblast‐derived. Prognostic Driver HMGA2: Elevated HMGA2 in malignant AT2 cells correlates with poor prognosis and is significantly higher in MET‐mutated PSC. Pathway Divergence in Malignant Cells: Malignant lipofibroblasts in PPS share a similar profile with PSC but uniquely enrich DNA repair pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/914312002105a075ce979102043e4694713bf91a" target='_blank'>
              Single‐nucleus RNA sequencing reveals the distinct heterogeneity of primary pulmonary sarcomas (PPS) and pulmonary sarcomatoid carcinoma (PSC)
              </a>
            </td>
          <td>
            Jianfei Zhu, Yanlu Xiong, Shouzheng Ma, Jun Wei, Jia-kuan Chen, Wen-chuan Wang, Q. Duan, Qin Zhang, Dongsheng Chen, Wanglong Deng, Tao Jiang, Jie Lei
          </td>
          <td>2025-12-30</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 The human prostate develops from the urogenital sinus through epithelial bud formation and branching, however the spatial dynamics of progenitor populations during these early stages remain poorly characterized. To addresses this gap, we used a human fetal prostate at 14 weeks of gestation for serial sectioning, yielding approximately 200 sections of 10 µm thickness. Six sections were profiled using spatial transcriptomics with the Visium HD platform, while the remaining sections were H&E-stained for 3D reconstruction of glandular architecture. We mapped transcriptional heterogeneity across developing prostate buds and identified distinct spatial organization of epithelial subdomains with basal-like and luminal-like characteristics. Notably, we also identified a unique cluster of tubule-initiating cells (TIC cells), that have been linked to cancer initiation and androgen-deprivation therapy resistance in adult prostate. This study provides the first high-resolution spatial map of developing human prostate, highlighting the organisation of progenitor populations.



 Emmanouela Perisynaki. 3D spatial mapping of developing human prostate glands [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B059.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9e67e64e1e6f9570a547e8411a3ed76fac26e1" target='_blank'>
              Abstract B059: 3D spatial mapping of developing human prostate glands
              </a>
            </td>
          <td>
            Emmanouela Perisynaki
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND Breast cancer is one of the most prevalent malignancies affecting women worldwide, with approximately 2.3 million new cases diagnosed annually. Breast cancer stem cells (BCSCs) play pivotal roles in tumor initiation, progression, metastasis, therapeutic resistance, and disease recurrence. Cancer stem cells possess self-renewal capacity, multipotent differentiation potential, and enhanced tumorigenic activity, but their molecular characteristics and regulatory mechanisms require further investigation. AIM To comprehensively characterize the molecular features of BCSCs through multi-omics approaches, construct a prognostic prediction model based on stem cell-related genes, reveal cell-cell communication networks within the tumor microenvironment, and provide theoretical foundation for personalized treatment strategies. METHODS Flow cytometry was employed to detect the expression of BCSC surface markers (CD34, CD45, CD29, CD90, CD105). Transcriptomic analysis was performed to identify differentially expressed genes. Least absolute shrinkage and selection operator regression analysis was utilized to screen key prognostic genes and construct a risk scoring model. Single-cell RNA sequencing and spatial transcriptomics were applied to analyze tumor heterogeneity and spatial gene expression patterns. Cell-cell communication network analysis was conducted to reveal interactions between stem cells and the microenvironment. RESULTS Flow cytometric analysis revealed the highest expression of CD105 (96.30%), followed by CD90 (68.43%) and CD34 (62.64%), while CD29 showed lower expression (7.16%) and CD45 exhibited the lowest expression (1.19%). Transcriptomic analysis identified 3837 significantly differentially expressed genes (1478 upregulated and 2359 downregulated). Least absolute shrinkage and selection operator regression analysis selected 10 key prognostic genes, and the constructed risk scoring model effectively distinguished between high-risk and low-risk patient groups (P < 0.001). Single-cell analysis revealed tumor cellular heterogeneity, and spatial transcriptomics demonstrated distinct spatial expression gradients of stem cell-related genes. MED18 gene showed significantly higher expression in malignant tissues (P < 0.001) and occupied a central position in cell-cell communication networks, exhibiting significant correlations with tumor cells, macrophages, fibroblasts, and endothelial cells. CONCLUSION This study comprehensively characterized the molecular features of BCSCs through multi-omics approaches, identified reliable surface markers and key regulatory genes, and constructed a prognostic prediction model with clinical application value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86e7f7dec5d3e190d876efe5f1afc46eeaf94a9" target='_blank'>
              Breast cancer stem cell activity driven by ME18D gene expression in the tumor microenvironment
              </a>
            </td>
          <td>
            De-Yang Guo, Zhang-Yi Liu, Qian-Chuan Yi
          </td>
          <td>2026-01-26</td>
          <td>World Journal of Stem Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Prostate cancer depends on androgen receptor (AR) signaling for growth, which is why androgen deprivation (castration) therapy is effective at early stages. However, many tumors eventually progress to a lethal form known as castration-resistant prostate cancer (CRPC). A subset of CRPC tumors bypass dependency on AR signaling by acquiring lineage plasticity, where prostate cancer cells transdifferentiate into alternate cellular states through epigenetic reprogramming. Neuroendocrine (NE) prostate cancer represents one well-known lineage plasticity phenotype. Nevertheless, most AR-independent tumors do not exhibit NE features and are defined as AR-negative/NE-negative or “double-negative prostate cancer” (DNPC). In a collaboration with Dr. Ekta Khurana’s computational genomics lab at Weill Cornell Medicine, we recently classified CRPC into four epigenetic subtypes, including the well-established 1) AR and 2) NE, as well as the novel DNPC subgroups 3) WNT and 4) stem cell-like (SCL) (PMID: 35617398). We focused on the SCL subtype as it is the second most common group in CRPC patients and lacks therapeutic targets. Using functional genomic approaches, we found that YAP/TAZ/TEAD cooperates with FOSL1 to drive the SCL lineage and growth of SCL models. We therefore hypothesize that the heightened dependency on the YAP/TAZ/TEAD/FOSL1 transcriptional program represents a therapeutic vulnerability in CRPC-SCL. To test this, we exposed CRPC models to TEAD inhibitors and found robust growth suppression in SCL cells compared to non-SCL cells in vitro. To evaluate whether the TEAD inhibitors are on-target, we performed transcriptomic profiling in SCL models and observed downregulation of YAP/TAZ gene signature as well as FOSL1 expression, which phenocopies the effects of YAP/TAZ double knockdown. To define the cistromes of these factors upon TEAD inhibition, we performed ChIP-seq and observed reduced co-occupancy at consensus sites, suggesting the disruption of the YAP/TAZ/TEAD/FOSL1 transcriptional circuit by the small molecule compound. To determine whether these phenotypes are recapitulated in vivo, we will treat mice harboring CRPC-SCL xenografts with TEAD inhibitors to assess growth response and evaluate epigenetic and transcriptional response using single-nucleus Multiome (ATAC+RNA). These studies will establish whether small molecule inhibition of TEAD is a promising strategy for the treatment of CRPC-SCL and allow high-resolution analysis of cell state transitions, with a focus on loss of SCL-specific signatures and potential emergence of AR/NE programs as adaptive resistance mechanisms.



 Chen Khuan Wong, Dan Li, Hongsu Wang, Marjorie Roskes, Weiling Li, Shipra Shukla, Dana Schoeps, Ekta Khurana, Yu Chen. Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A074.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912fdd784be0fd097e91b40d9e4c6933dd2c1b14" target='_blank'>
              Abstract A074: Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer
              </a>
            </td>
          <td>
            C. K. Wong, Dan Li, Hongsu Wang, M. Roskes, Weiling Li, Shipra Shukla, Dana M. Schoeps, E. Khurana, Yu Chen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba884bb3dbf2cdd1df613387c3fda90c3178112" target='_blank'>
              A vascularized three-dimensional model integrating primary breast tumor cells and microvascular fragments: mimicking the tumor microenvironment involved in chemoresistance.
              </a>
            </td>
          <td>
            Alessandra Lo Cicero, Francesco La Monica, Gabriele Lo Buglio, S. Campora, Federico Gangemi, Paolo Cinà, Monica Salamone, Margot Lo Pinto, S. D. Scilabra, G. Ghersi
          </td>
          <td>2026-01-08</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The survival mechanisms of circulating tumor cells (CTCs) remain poorly understood while these rare cell populations transit through the blood stream and colonize distant organs including the brain. Using single-cell RNA-seq of microfluidically-enriched CTCs and patient-matched brain metastatic tumor cells of lung adenocarcinoma (LUAD), we revealed CTC-selective upregulation of core gene signatures associated with ferroptosis. The malignancy of CTCs was confirmed through comparative genomic instability and copy number variation (CNV) analyses of paired CTCs and metastatic tumor cells using both transcriptomic and whole-exome sequencing. Among the transcription factors elevated in CTCs, Chromobox 3 (CBX3) was the top hit, which was tightly correlated with GPX4 expression. Functionally, CBX3 co-operated with EP300 to protect CTCs from ferroptosis by upregulating GPX4 expression. Genetic depletion of CBX3 triggered ferroptosis and substantially reduced tumor cell survival and invasiveness in two independent cancer cell models. Conversely, CBX3 overexpression was sufficient to promote tumor growth, migration, and invasion in vitro, and metastatic progression in vivo. Clinically, CBX3 expression was significantly correlated with TNM stage and was predictive of both progression-free interval and overall survival in LUAD. In a prospective cohort of LUAD and melanoma patients, CTCs co-expressing CBX3 and GPX4 were selectively elevated in the blood samples of metastatic cases compared to the non-metastatic group, highlighting the clinical association between ferroptosis-resistant CTCs and metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7fe0bbf1d19b9968391a14825a2e7d78f15cc38" target='_blank'>
              CBX3 confers ferroptosis resistance during blood-borne metastasis
              </a>
            </td>
          <td>
            Chun Wu, Xuefei Liu, Boxi Zhao, Mao Zhao, Binyu Zhang, Guanyin Huang, Yixin Cheng, Shuqian Zheng, Jianyang Hu, Ling Guo, Wei-Jie Guo, Jun Tan, Xin Hong
          </td>
          <td>2026-01-15</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Intraductal papillary mucinous neoplasms (IPMNs) are precursor lesions of pancreatic cancer with highly variable malignant potential. Current understanding of their biology remains incomplete, limiting accurate risk stratification and targeted interventions. Objective This study aimed to characterise the molecular and immune features of IPMN across different dysplasia grades and histological subtypes, with a focus on IPMN-associated invasive carcinoma (IPMN-IC). Design Spatial whole-transcriptome profiling using Digital Spatial Profiling was conducted on 12 patients, capturing the full histological and dysplastic spectrum of IPMN and conventional pancreatic ductal adenocarcinoma. A total of 117 epithelial, immune and stromal areas of interest were analysed. An expanded cohort of 43 patients with IPMN was used to validate selected key markers. Results Transcriptomic analysis unveiled stage-specific molecular alterations and identified two distinct subsets of high-grade (HG) IPMN lesions: one resembling indolent lesions (HG) and the other IC (HG+). Key markers associated with divergent biological behaviours were identified, including MUC5AC and TFF1 in indolent lesions, and Claudin-1 in lesions with invasive potential. Immune profiling revealed a trajectory from activation to suppression during IPMN progression. Several characteristic immune checkpoint molecules, including CEACAM1 and CD44, were identified in IPMN-IC. Conclusion This study provides a spatially resolved molecular map of IPMN progression, delineating key transcriptomic and immune signatures. These findings advance the understanding of IPMN biology and highlight potential biomarkers for risk stratification and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde79a67721a5dbb5ada8806ba49e4956d9e4f26" target='_blank'>
              Spatial transcriptomics defines the molecular progression, invasion and immune landscape of IPMN and IPMN-derived pancreatic cancer
              </a>
            </td>
          <td>
            Ming Cui, S. Mo, Jialu Bai, Ammar A. Javed, Joseph R. Habib, Sen Yang, Tianqi Chen, Ruiling Xiao, Wenfei Diao, Decheng Jiang, C. Wolfgang, Xiaoyan Chang, Ya Hu, Yupei Zhao
          </td>
          <td>2025-12-10</td>
          <td>Gut</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad062077323547175d6b721e78e8771a9325ce15" target='_blank'>
              Phenotypic Plasticity and Competition Shape Therapy Sequencing in HER2+/HER2− Breast Cancer: A Mathematical Framework
              </a>
            </td>
          <td>
            Aleksandra Gavrilova, Trachette L. Jackson, Nizhum Rahman
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Most colorectal cancers are resistant to immune checkpoint inhibitors due to an immunosuppressive tumor microenvironment. LINC00673, previously classified as a long non-coding RNA, has been implicated in tumor progression, but its role in antitumor immunity remains poorly understood. We used single-cell RNA sequencing, flow cytometry, and functional assays in Linc00673 knockout and wild-type mouse models treated with programmed death 1 (PD-1) blockade to dissect changes in immune landscape and tumor cell behavior. Linc00673 deletion enhanced the efficacy of PD-1 therapy, reducing tumor burden and prolonging survival. KO tumors showed increased infiltration of cytotoxic CD8⁺ T cells, reduced exhaustion, and improved antigen presentation. The tumor microenvironment shifted toward a pro-inflammatory state, with elevated dendritic cell function, reduced immunosuppressive macrophages, and improved T cell–tumor interactions. Linc00673 also promoted tumor cell migration and immune evasion independently of immune cells, suggesting dual tumor-intrinsic and -extrinsic roles. LINC00673 suppresses antitumor immunity and promotes tumor aggressiveness. Its deletion synergizes with PD-1 blockade, revealing a promising therapeutic target to overcome immune resistance in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9ef0a9a3f2685b8b34be389ae9c671f82a1666" target='_blank'>
              Single-cell RNA sequencing unveils enhanced antitumor immunity in colorectal cancer with PD-1 blockade and LINC00673 deletion
              </a>
            </td>
          <td>
            Yifei Zhu, Yanxi Yao, Yaxian Wang, Zhibing Qiu, Dingpei Zhou, Huixia Huang, Keji Chen, Qingyang Sun, Jiayu Chen, Yuxue Li, Jianqiang Tang, Dawei Li, Ping Wei
          </td>
          <td>2026-01-03</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Clear cell renal cell carcinoma (ccRCC) presents a unique immunological paradox: abundant CD8
 +
 tumor-infiltrating lymphocytes (TILs) correlate with poor prognosis. To clarify their functional status and therapeutic potential, we performed single-cell transcriptomic profiling of TILs from 15 patients with ccRCC and functionally validated dominant T cell receptor (TCR) clonotypes using autologous tumor-derived organoids. Single-cell RNA sequencing revealed dynamic shifts in T cell composition, with effector and progenitor-exhausted CD8
 +
 T cells declining and terminally exhausted CD8
 +
 and regulatory CD4
 +
 T cells enriched in advanced tumors. Despite this exhausted phenotype, in an exploratory analysis with five patients, approximately half of the top 20 TCR clonotypes retained anti-tumor reactivity when re-expressed in non-exhausted T cells, as evidenced by TCR-T cell-mediated cytotoxicity and IFN-γ production against autologous organoids. Transcriptomic signatures enabled the development of a penalized logistic regression classifier that distinguished tumor-reactive from bystander T cells with high accuracy, with AUCs of 0.903 (training) and 0.913 (test). Cross-cancer comparison with pancreatic ductal adenocarcinoma (PDAC) datasets revealed limited generalizability, highlighting the need for cancer type-specific models. Notably, ccRCC-specific TILs exhibited mature, functionally differentiated profiles with limited proliferation, consistent with chronic antigen exposure, whereas PDAC-reactive TILs showed highly proliferative and activated phenotypes indicative of ongoing clonal expansion. Collectively, these findings suggest key features of the immune landscape in ccRCC and provide a preliminary, proof-of-concept transcriptomic framework for prioritizing candidate tumor-reactive TCRs. These insights suggest the feasibility of identifying candidate TCRs for future development of TCR-based adoptive T cell therapies in ccRCC and emphasize the importance of integrating single-cell profiling with functional analyses to refine immunotherapeutic strategies. Given the limited sample size, our results should be considered exploratory and hypothesis-generating, and future studies will be required to validate these findings in larger, independent ccRCC cohorts.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e62c016264580977157f1e59b0fc3cfcc38dd55" target='_blank'>
              Tumor-reactive TCRs within exhausted TILs reveal cancer type-specific immune landscapes in renal cell carcinoma
              </a>
            </td>
          <td>
            Mitsuru Komahashi, S. Horaguchi, Kayoko Tsuji, Daisuke Hoshino, Takeshi Kishida, Kimitsugu Usui, N. Nakaigawa, Shinya Sato, Hiroshi Hamana, Hiroyuki Kishi, Feifei Wei, Yasunobu Mano, T. Kouro, Shuichiro Uehara, T. Sasada
          </td>
          <td>2026-01-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Motivation Single-cell omics analysis has unveiled the heterogeneity of various cell types within tumors. However, no methodology currently reveals how this heterogeneity influences cancer patient survival at single-cell resolution. Here, we introduce scSurv, combining a Cox proportional hazards model with a deep generative model of single-cell transcriptome, to estimate individual cellular contributions to clinical outcomes. Results The accuracy of scSurv was validated using both simulated and real datasets. This method identifies cells associated with favorable or adverse prognoses and extracts genes correlated with their contribution levels. In melanoma, scSurv reproduces known prognostic macrophage classifications and facilitates hazard mapping through spatial transcriptomics in renal cell carcinoma. We also identified genes consistently associated with prognosis across multiple cancers and demonstrated the applicability of this method to infectious diseases. scSurv is a novel framework for quantifying the heterogeneity of individual cellular effects on clinical outcomes. Availability The implementation of scSurv is available on GitHub (https://github.com/3254c/scSurv) and Zenodo (https://doi.org/10.5281/zenodo.17793054).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98bd46ab28d9f09fcd249606230d4459ff93bbef" target='_blank'>
              scSurv: a deep generative model for single-cell survival analysis
              </a>
            </td>
          <td>
            Chikara Mizukoshi, Yasuhiro Kojima, Shuto Hayashi, Ko Abe, Daisuke Kasugai, Teppei Shimamura
          </td>
          <td>2025-12-22</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cervical cancer is a prevalent malignancy among women, yet the involvement of neutrophils in its tumor microenvironment remains insufficiently explored. This study utilized single-cell RNA sequencing (scRNA-seq) to delineate neutrophil subsets and elucidate their roles in disease progression and prognosis. Analysis of 20 cervical biopsy samples across different disease stages identified five neutrophil subsets (N0-N4), among which the N4 subset exhibited a marked increase during disease advancement. Spatial transcriptomics and tissue microarray analyses revealed that N4 neutrophils are enriched in tumor regions and are associated with genes implicated in proliferation, metastasis, and immune evasion. Functional characterization demonstrated that N4 promotes tumor progression via activation of the Wnt signaling pathway and extracellular matrix remodeling. A neutrophil infiltration-based risk model was established and validated through multi-omics approaches, highlighting its potential in prognostic prediction. These findings underscore the pivotal role of N4 neutrophils in cervical cancer and provide valuable insights for the development of targeted immunotherapies and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9eca7bc06f3fa7d3102669c9c6d4eefb56b347" target='_blank'>
              Single-Cell and Multiomic Analysis Reveals Neutrophil Heterogeneity and Prognostic Value in Cervical Lesions.
              </a>
            </td>
          <td>
            Ze Wang, Zhuang Liang, Shiyi Liu, C. Cao, Shimin Chen, Nan Yu, Xiaoyuan Huang, Tao Zhang
          </td>
          <td>2025-12-12</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Gastrointestinal stromal tumors (GISTs) exhibit significant heterogeneity, posing substantial challenges for personalized treatment strategies. Patients often display varied responses to different therapeutic agents and dosages. Currently, the absence of robust and physiologically relevant in vitro models for GISTs impedes accurate prediction of therapeutic efficacy, thereby constraining the advancement of effective treatment strategies. Traditional 2D cell cultures fail to replicate the tumor microenvironment (TME) and lack patient-specific characteristics, limiting their predictive value. In contrast, three-dimensional bioprinting (3DP) technology faithfully recapitulates key histological architecture and molecular features of their parental tumors, enhancing the physiological relevance of in vitro models. We employed patient-derived 3DP-GIST models via 3D bioprinting technology, followed by comprehensive histopathological, genomic, and transcriptomic analyses. Subsequently, we applied clinically approved targeted therapeutic agents to perform drug screening and response prediction on the 3DP-GIST models. The drug sensitivity profiles obtained from these models were then correlated with retrospective clinical data and patient follow-up records to assess the models’ potential in guiding the selection and prediction of effective GIST therapies. In our study, we successfully constructed 12 patient-derived 3DP-GIST models. Histopathological assessments, whole-exome sequencing (WES), and transcriptomic analyses confirmed that these models accurately recapitulate the histological architecture, biomarker expression, and molecular features of their corresponding parental tumors. Transcriptomic profiling further revealed gene expression signatures associated with GIST recurrence risk and imatinib resistance. Importantly, the 3DP-GIST models demonstrated the capacity to provide precise, individualized treatment recommendations within 10 days post-surgery, potentially reducing treatment delays and improving patient outcomes. Overall, the 3DP-GIST model represents a robust and efficient platform for evaluating patient-specific drug sensitivities in vitro, thereby guiding personalized therapeutic strategies for GIST patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492f9dec710186d0bb398ff8467a9fedaf4e7eda" target='_blank'>
              Three-dimensional bioprinting of patient-derived Gastrointestinal stromal tumor: a novel platform for precision oncology and drug response profiling
              </a>
            </td>
          <td>
            Liwei Du, Zicheng Zheng, Yanan Wang, Kai Zhang, Yuce Lu, Minghao Sun, M. Pang, Shangze Jiang, Yixuan He, Shunda Du, Haitao Zhao, Yilei Mao, Huayu Yang, Weiming Kang
          </td>
          <td>2025-12-18</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cell-in-cell (CIC) structures, characterized by one cell engulfing another, play a paradoxical role in cancer metastasis. This review synthesizes their dual nature: while CIC can suppress metastasis by eliminating aberrant cells or activating immune responses, it also drives tumor progression through clonal evolution, metabolic adaptation, and genomic instability. The clinical relevance of CIC varies across cancer types, correlating with both favorable and poor prognoses depending on disease stage and microenvironmental context. Emerging therapeutic strategies aim to harness protective CIC mechanisms or inhibit pro-metastatic pathways, though challenges such as biomarker validation and dynamic tracking persist. Future research integrating advanced technologies will elucidate the spatiotemporal complexity of CIC, enabling tailored interventions to disrupt metastatic cascades. By bridging fundamental biology and clinical translation, CIC represents a promising yet intricate frontier in the battle against metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e6d8a02bb8fcc15958d3d8c8939ab24272dbaf6" target='_blank'>
              Cell-in-cell structures and cancer metastasis.
              </a>
            </td>
          <td>
            Yuchen Cao, Jianing Du, Jianguo Lai, Chunjun Liu
          </td>
          <td>2026-01-23</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor-associated neutrophils (TANs) exhibit highly functional heterogeneity across cancers. Although TANs promote inflammatory responses and contribute to tumor clearance, they frequently undergo context-dependent reprogramming within the tumor microenvironment (TME) into highly immunosuppressive phenotypes that facilitate cancer dissemination and immunotherapy resistance 1,2 . We contend that an underappreciated, upstream determinant of this divergence is the maturation stage of TANs 3,4 . The developmental stage of TANs determines the migration patterns and constrains the functional capacity, and the developmental stage also constrains the extent of TME-driven re-education, together shaping pro-or anti-tumor outcomes 3-5 . In this Perspective, we place maturation at the core of TAN biology and discuss current definitions for TAN developmental stages and the measurable markers that researchers and clinicians can use ( Figure 1 ). In addition, spatial and temporal transitions in TAN maturation stages and the factors that govern these transitions are elucidated. We explain how maturation status shapes TAN function and articulate the key differences between mouse and human TAN maturation systems to highlight the value of human immune system (HIS) mouse models. Based on this framework, functional biomarkers and signatures of TAN maturation are introduced and we show how to embed them into patient stratification and longitudinal monitoring. Finally, we outline immunotherapy strategies targeting TAN maturation, selecting interventions guided by maturation markers to reinforce treatment benefits for">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d662ed678180365badf4da6ebe8288f142877d8a" target='_blank'>
              From bone marrow to the tumor microenvironment: how neutrophil maturation shapes cancer immunity
              </a>
            </td>
          <td>
            Tao Shi, Yiran Cai, Hanbing Wang, Jia Wei
          </td>
          <td>2025-12-15</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="CD24, a glycosylphosphatidylinositol-anchored cell surface protein, is frequently dysregulated in non-small-cell lung cancer (NSCLC) and contributes to tumor proliferation, metastasis, and immune evasion. However, the nanoscale spatial organization of CD24 and its functional relevance to NSCLC proliferation remain incompletely understood. Here, we employed direct stochastic optical reconstruction microscopy (dSTORM) to visualize CD24 clusters on the membranes of NSCLC cell lines, freshly isolated primary NSCLC cells, and clinical NSCLC tissues. Our results revealed that the increased clustering of CD24 promotes proliferation. Using an Asn36 single-point mutant that disrupts CD24 clustering without altering its expression level, we confirmed that CD24 clustering, rather than CD24 expression alone, drives NSCLC proliferation. More importantly, we found that the clustering disruption suppressed the NSCLC cell proliferation by reducing the phosphorylation of PI3K, AKT, and mTOR, which was consistent with the result of treating with PI3K inhibitor. Our findings highlight the role of CD24 nanoscale clustering in promoting NSCLC proliferation via the PI3K/AKT/mTOR axis, providing mechanistic insights into CD24-mediated tumorigenesis and identifying CD24 clustering as a potential prognostic biomarker and therapeutic target for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61d6817b1867b466ef7213c91293e088cd123bad" target='_blank'>
              Super-Resolution Fluorescence Imaging Reveals the Role of CD24 Clusters in NSCLC Proliferation.
              </a>
            </td>
          <td>
            Jincheng Wang, Baofeng Li, Meng Li, Longxiang Li, Xinyang Chen, Hongda Wang, Guangxin Zhang, Jing Gao, Ti Tong
          </td>
          <td>2026-01-14</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Acral melanoma (AM), the predominant melanoma subtype in Asia, responds poorly to immune checkpoint inhibitors, representing a critical unmet medical need. The efficacy of tumor-infiltrating lymphocyte (TIL) therapy in this population is unknown. An Investigator-Initiated Trial evaluates autologous TIL therapy (LM-103) in four Chinese patients with advanced AM, achieving a 75% disease control rate (DCR) and a 25% objective response rate (ORR), including one durable complete response. To define the determinants of response, we performed integrated single-cell RNA and T-cell receptor sequencing on infused TIL products, tumors, and longitudinal peripheral blood. Responders' infused products were significantly enriched for T follicular helper (Tfh) and intermediate exhausted (TEX_int) CD8⁺ T cells, which mediated robust cell-cell signaling networks (e.g., CD40, FASLG). In contrast, the non-responder's product was dominated by terminally exhausted (TEX_term) cells. Clonal tracking revealed that these Tfh and TEX_int subsets possessed higher clonality, and in the complete responder, a dominant clone originating from the TEX_int population persisted systemically by differentiating into a progenitor-like (TEX_prog) state. These findings demonstrate that TIL therapy is clinically active in AM and that durable response is mechanistically linked to the infusion and persistence of Tfh and TEX_int subsets, defining a key cellular and clonal architecture for therapeutic success.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4d4d63db111617fa12c56b70ba0807cf90683e2" target='_blank'>
              Single-Cell Dissection of Tumor-Infiltrating Lymphocytes Reveals Cellular Architecture Predictive of Therapeutic Efficacy in Acral Melanoma.
              </a>
            </td>
          <td>
            Chao Zhang, Wanyi Xiao, Hong-xiu Shen, Fenge Li, Ting Li, Weihong Zhang, Haotian Liu, Ziwei Gao, Hongyu Wang, Xiubao Ren, Kexin Chen, Xiangchun Li, Jilong Yang
          </td>
          <td>2026-01-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ebc0a8908682680600606299cb1bf5354f0c5e" target='_blank'>
              Etiology of polyploid giant cancer cells: a new frontier in cancer biology.
              </a>
            </td>
          <td>
            Mukesh Kumar Manickasamy, Babu Santha Aswani, Ruchira Banerjee, Mohamed Abbas, M. Alqahtani, Gautam Sethi, Le Liu, A. Kunnumakkara
          </td>
          <td>2026-01-19</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a prevalent and aggressive malignancy with a persistently high mortality rate, largely attributable to therapy resistance and tumor recurrence. This review comprehensively explores the critical interplay between epigenetic dysregulation and the tumor microenvironment (TME) in driving OSCC progression. We detail how key epigenetic mechanisms, including DNA methylation, histone modifications, and non-coding RNAs (ncRNAs), intrinsically transform cancer cells and actively orchestrate pro-tumorigenic TME. These alterations substantially contribute to resistance against conventional therapies. Furthermore, we discuss the therapeutic potential of targeting these pathways using epigenetic drugs (epi-drugs), such as DNA methyltransferase (DNMT) inhibitors and histone deacetylase (HDAC) inhibitors, as well as engineered extracellular vesicles (EVs). The primary objective of this review is to synthesize current knowledge on the epigenetic-TME axis, thereby providing a mechanistic foundation for developing novel therapeutic strategies. We emphasize that rational combinations of epigenetic-targeting agents with conventional treatments or immunotherapy hold significant promise for overcoming drug resistance and improving clinical outcomes in OSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96aaa3d5aaab2d3886a4c338f86161bbd9cbefef" target='_blank'>
              Dissecting the role of epigenetic regulation in oral squamous cell carcinoma microenvironment: mechanisms and therapeutics
              </a>
            </td>
          <td>
            Xuechao Li, Yifei Ren, Shenghua Pei, Kai Zhao, Guanyu Chen, Zhenglin He
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAM) exert both pro- and anti-tumoral functions that influence cancer progression and patient prognosis. However, single-cell RNA sequencing (scRNA-seq) studies have revealed that TAM heterogeneity remains incompletely characterized. By performing an unbiased, integrated in silico analyses of publicly available scRNA-seq datasets, comprising samples from blood, tumor, and non-tumoral mammary tissue (NTMT) from both breast cancer (BRCA) patients and healthy individuals, we identified seven transcriptional signatures corresponding to distinct TAM subsets, exhibiting unique functional profiles, including heightened interferon responses, scavenging, and matrix remodeling, the latter two being characteristic of tissue repair. Notably, none of these subsets aligned with the M1/M2 classification of macrophage (Mφ) polarization. Interferon-associated genes were predominantly enriched in blood monocytes, whereas tissue-repair-associated-signatures were more abundant in tissue resident Mφ, suggesting that TAMs bearing these signatures resemble monocyte-derived or tissue-resident Mφs, respectively. Importantly, TAM subsets expressing interferon-associated genes were associated with improved survival compared to tissue-repairing TAMs in a BRCA cohort from The Cancer Genome Atlas (TCGA). Additionally, one signature was heightened in peripheral monocytes from BRCA patients compared with healthy individuals, which was experimentally validated in a pilot study of Mexican BRCA patients. We concluded that these signatures are a closer description of TAM heterogeneity in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769e9d0c6f0718581121627c404c58103ae137e7" target='_blank'>
              Single cell analysis identifies inflammatory and tissue remodeling tumor associated macrophages distinct from M1/M2 paradigm.
              </a>
            </td>
          <td>
            Eric López-Huerta, Diana P Portales-Pérez, M. E. Martínez-Leija, Vianney Ortiz-Navarrete, E. Fuentes-Pananá
          </td>
          <td>2025-12-05</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cancer is one of the leading causes of death worldwide, and the majority of cancer-related deaths are caused by cancer that has spread to other organs. Precision cancer medicine (PCM) holds potential to improve outcomes and relies on molecularly matched therapies based on cancer cell specific molecular alterations. The tumor immune microenvironment plays an important role beyond response to therapy; however, this is generally not considered in current PCM platforms. We established patient-matched organoids and immune cell cultures for drug testing in mono- and co-culture treatment setups using three distinct treatment strategies (pretreatment, co-culture treatment, and T-cell bispecific antibody testing). Response to treatment and impact of immune cells were evaluated by tumor cell viability assays and flow cytometry analysis. Phenotypic analysis showed high heterogeneity of tumor-infiltrating lymphocytes (TILs) across the patients and low immune cell portions of organoids, emphasizing the need for a patient-matched co-culture PCM approach. Our in-depth study of three patients revealed an effect of the patients’ immune cells on drug response and T-cell bispecific antibody treatment in vitro. Here, we illustrate a state-of-the-art co-culture PCM pipeline for patient-matched organoids and immune cells replicating patient response to treatment at the time of biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc0ec2ccde29e016db89d22669a8bf2faa96aa64" target='_blank'>
              Immune Cell Modulation of Patient-Matched Organoid Drug Response in Precision Cancer Medicine Platform
              </a>
            </td>
          <td>
            Silje Kjølle, M. Presti, Jéssica de Pina Roque, Lina Hua Bisgaard, Darío Beceiro Ramos, K. W. Zornhagen, C. Yde, Ane Yde Schmidt, Perrine Verdys, M. Højgaard, U. Lassen, I. M. Svane, K. Rohrberg, Marco Donia, J. T. Erler
          </td>
          <td>2026-01-29</td>
          <td>Cells</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Metabolic rewiring is integral to immune cell activation and tissue inflammation, but its spatial and cell type–specific organization in Crohn’s disease (CD) remains poorly understood. In this study, we aimed to construct a spatially resolved single-cell atlas of immunometabolic states in human ileal CD, spanning active inflammation, remission, and health, to identify activity-linked and persistent metabolic signatures.



 Terminal ileal tissue was obtained from patients with active CD (n = 10), CD in remission (n = 8), and controls (n = 10). Imaging mass cytometry was performed using a 37-marker panel incorporating lineage, structural and 14 metabolic proteins. Data were analyzed at single-cell resolution to define metabolic programs across identified cell clusters and spatial neighborhoods. In parallel, peripheral blood mononuclear cells were profiled by suspension mass cytometry to assess systemic signatures. Comparative and spatial correlation analyses were conducted in R using Phenograph clustering and neighborhood interaction modelling.



 IMC revealed extensive, cell type–resolved metabolic reprogramming in active CD. Inflammation was characterized by reduced mitochondrial and tricarboxylic acid cycle activity (ATP5A, CytC, VDAC1) with a concurrent shift towards the pentose phosphate pathway (G6PD) and dampened mTOR signaling. CD68+ macrophages and neutrophils displayed pronounced PPP activation, whereas B cells showed persistent depression of anaplerosis (GLUD1/2) and lipid uptake (CD36). Spatial neighborhood analysis uncovered compartment-specific metabolic coupling, particularly in remission, where immune-epithelial proximity correlated with restored mitochondrial activity. Peripheral blood exhibited only modest, myeloid-restricted changes, indicating that metabolic rewiring is predominantly tissue-embedded.



 Crohn’s ileitis is defined by spatially organized, cell type–specific metabolic remodeling that persists beyond clinical remission. This immunometabolic atlas provides a framework for identifying tissue-specific therapeutic targets and biomarkers of disease activity in inflammatory bowel disease.
 Conflict of interest:
 Lehmann, Malte: No conflict of interest
 Walling, Sonja: No conflict of interest
 Adrian, Huck: No conflict of interest
 Kunkel, Désirée: No conflict of interest
 Glauben, Rainer: No conflict of interest
 Kühl, Anja A.: No conflict of interest
 Siegmund, Britta: Grant: Pfizer Other: Consultant: Abbvie, Abivax, Boehringer Ingelheim, Bristol Myers Squibb, Dr. Falk Pharma, Eli Lilly, Endpoint Health, Galapagos, Janssen/Johnson & Johnson, Materia Prima, MSD, Pfizer, Takeda, Wedbush Securities. Speaker: Abbvie, AlfaSigma, Bristol Myers Squibb, CED Service GmbH, Dr. Falk Pharma, Eli Lilly, Ferring, Galapagos, Janssen/Johnson & ampJohnson, MD Education, MSD, Pfizer, Tr1xBio.
 Haag, Lea Maxie: No conflict of interest
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55721e835b31a5f5ee95010621f83e8bb19691c1" target='_blank'>
              DOP074 Cell type–specific immunometabolic reprogramming in Crohn’s ileitis
              </a>
            </td>
          <td>
            M. Lehmann, S. Walling, H. Adrian, D. Kunkel, R. Glauben, A. A. Kühl, B. Siegmund, L. Haag
          </td>
          <td>2026-01-01</td>
          <td>Journal of Crohn’s and Colitis</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Tumour heterogeneity, encompassing genetic, epigenetic, and microenvironmental diversity, remains a fundamental obstacle in precision oncology. Traditional bulk sequencing captures only averaged molecular profiles, thereby masking rare yet functionally critical subpopulations that drive malignant progression and therapeutic resistance. Recently, the emergence of single‐cell sequencing technologies has overcome the limitations of bulk approaches, enabling high‐resolution analyses of the genome, transcriptome, epigenome and proteome at the single‐cell level. These advances have enabled detailed mapping of tumour ecosystems, identification of key cellular subtypes, reconstruction of evolutionary trajectories and elucidation of intercellular communication networks within the tumour microenvironment. Accumulating evidence demonstrates that single‐cell technologies elucidate fundamental aspects of tumour biology and reveal potential diagnostic and therapeutic targets. This review systematically summarises the recent advances and applications of single‐cell sequencing in the field of precision oncology, with particular emphasis on its applications in mechanistic discovery, diagnosis, therapy, and prognosis. Furthermore, we discuss current challenges related to technology, data analysis, and clinical translation, and outline future research directions. In summary, single‐cell sequencing has profoundly reshaped our understanding of tumour biology and is propelling oncology into a new era of precision, prediction, and personalisation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11cbf7015a15504645fc0387a8b9ed997f9b6eef" target='_blank'>
              Single‐cell sequencing for cancer precision medicine: From mechanism discovery to diagnosis and therapeutics
              </a>
            </td>
          <td>
            Yue Zhao, Hancong Li, Huanzuo Yang, Gongshuang Zhang, Xinyue Fu, Peiheng Li, Puxing He, Shuang Wu, Han Luo
          </td>
          <td>2026-01-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Early detection of melanoma, a highly variable and aggressive form of skin cancer, is crucial for improving patient outcomes. It is essential to distinguish malignant cells from normal melanocytes, and therefore, label‐free imaging methods that can do so are needed. Given the genotoxic effect of UV radiation, these mutations are numerous and affect many genes, including NRAS; therefore, therapeutic strategies can be directed toward these recurrent mutations. The aggressive nature of NRAS‐mutant melanoma contributes to poor patient prognosis, highlighting the need for early diagnosis. This study utilizes quantitative phase imaging (QPI) with digital holographic microscopy (DHM) to differentiate the morphology of NRAS‐mutant SK‐MEL‐2 cells from melanocytes using holographic microscopy; dimensionality reduction techniques, including Principal Component Analysis (PCA),t‐distributed stochastic neighbor embedding (t‐SNE); and clustering techniques, including K‐Means Elbow Plots and Hierarchical Clustering Dendrograms. The results demonstrated distinct morphologies and geometries between melanocytes and SK‐MEL‐2 cells, indicating that QPI with DHM can serve as a label‐free tool for identifying optical biomarkers in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c04598128bd69a8ebeb4ac33e2197f4b32bd3b" target='_blank'>
              Morphological and Optical Characterization of NRAS‐Mutant Melanoma Cells and Primary Melanocytes via Quantitative Phase Imaging With Digital Holographic Microscopy
              </a>
            </td>
          <td>
            Ayah A Farhat, Yazan A. Almahdi, Fatima Z Alshuhani, Besa Xhabija
          </td>
          <td>2026-01-01</td>
          <td>Cell Biochemistry and Function</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with a poor prognosis and limited response to current therapies. Cancer stem cells (CSCs) contribute to this heterogeneity, driving tumor progression, immune evasion, and therapeutic resistance. Epigenetic regulators have emerged as pivotal modulators of CSC phenotypes. The histone methyltransferase SETDB1, a key stemness factor in both embryonic and adult stem cells, catalyzes H3K9 trimethylation (H3K9me3) and acts as a major oncogenic driver in a wide range of solid and hematological malignancies. Yet, its specific role in maintaining stemness and contributing to therapeutic resistance in HCC remains poorly defined. Integrative analyses of bulk and single-cell transcriptomic datasets from public HCC cohorts were performed. Tumors were stratified by SETDB1 expression, followed by pathway enrichment, immune deconvolution, and stemness scoring. Associations with clinical outcomes, immune phenotypes, drug resistance signatures, and chromatin-binding partners (DNMT3A, TRIM28, HDAC1) were also explored. The role of SETDB1 was then examined in vitro, where a stem-like phenotype was induced in HCC cell lines under hypoxic conditions, and patterns of SETDB1 expression and stemness marker levels were assessed following paclitaxel treatment. SETDB1-low tumors were enriched in well-differentiated, immunologically active subclasses and were associated with increased predicted responsiveness to immune checkpoint blockade (ICB), sorafenib, and transarterial chemoembolization. Conversely, SETDB1-high tumors exhibited transcriptional features of dedifferentiation, elevated stemness scores, and enrichment in aggressive molecular subtypes. These tumors also displayed higher TP53 mutation rates, reduced immune infiltration, immune exclusion signatures, and activation of pathways linked to ferroptosis and ABC transporter dysfunction, consistent with resistance to ICB and conventional therapies. SETDB1 expression was also associated with hypoxia-related pathways, suggesting a role in maintaining CSC niches. In vitro, hypoxia-induced stemness coincided with increased SETDB1 levels, whereas paclitaxel treatment decreased both SETDB1 expression and stemness markers. A chromatin risk score incorporating SETDB1 and its partners predicted poor disease-free survival independently of clinical parameters. SETDB1 defines a stemness-enriched, immune-resistant HCC subtype associated with poor outcomes and therapeutic failure. Correlations between SETDB1 expression and stem-like features in vitro suggest a potential role in maintaining CSC phenotypes, supporting its relevance as a biomarker and candidate for epigenetic- and CSC-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c393716a837e5b1ef0091b6385eb92c64be4ab4" target='_blank'>
              The epigenetic regulator SETDB1 as a key component of cancer stem cells and drug resistance in primary liver cancer
              </a>
            </td>
          <td>
            Maël Padelli, Christophe Desterke, Aurore Devocelle, Georges Uzan, Antoinette Lemoine, Julien Giron-Michel
          </td>
          <td>2026-01-06</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Why and how does cancer start? Building from a Symposium at the 2025 Society of Toxicology meeting, we convened a group of international experts to answer this seemingly simple question. As experimental evidence has evolved, perspectives on cancers' origins have shifted from the accumulation of DNA mutations in single cells to complex processes involving signals from an altered tissue microenvironment which promote tumorigenesis. Carcinogen exposures impact the biology of the microenvironment in complex and tissue-specific ways. These changes can include the infiltration of inflammatory cells that produce growth factors, neo-angiogenesis, morphological changes, and immune tolerance that avoids immune-mediated elimination. In this in-depth review, we discuss the evidence linking chemical-driven microenvironmental changes in the development of a range of solid and liquid tumors. We discuss specific phenotypic alterations, such as selection pressure driving clonal expansion and cellular plasticity and reacquisition of stem cell states, linked to carcinogen-induced changes in the microenvironment. We describe assays and biomarkers which can allow us to experimentally assess links between chemical exposures, the microenvironment, and cancer phenotypes. We end by discussing how understanding the role of the microenvironment and malignancy in toxicology is essential for accurate cancer hazard evaluation, development of next-generation risk assessment frameworks, identifying new strategies for cancer prevention, and improving patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c64691a327a7b8227d277fcb6bbfe9d59f21fba7" target='_blank'>
              Rethinking the Microenvironment's Role in Chemical-Induced Malignancy.
              </a>
            </td>
          <td>
            William H. Bisson, Richard A Currie, Emilia L Lim, Coraline Mlynarczyk, Erik J Tokar, Annamaria Colacci, Justin A. Colacino
          </td>
          <td>2025-12-17</td>
          <td>Toxicological sciences : an official journal of the Society of Toxicology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Angioimmunoblastic T-cell lymphoma (AITL) is a subtype of peripheral T-cell lymphoma originating from follicular helper T cells and characterized by profound clinical, pathological, and molecular heterogeneity. Despite advances in classification and diagnosis, therapeutic outcomes remain suboptimal, particularly in relapsed or refractory settings. In recent years, the evolution of risk stratification models and a deeper understanding of AITL’s molecular pathogenesis, including its unique pattern of clonal evolution, have paved the way for precision medicine. Notably, targeted treatment approaches, such as immunomodulatory agents, epigenetic therapies, kinase inhibitors, and immune checkpoint blockade, are demonstrating promising clinical efficacy, especially in patients harboring specific molecular aberrations. This review comprehensively summarizes the multilayered heterogeneity of AITL, emphasizes the role of immune and molecular profiling in informing therapeutic decisions, and outlines future directions for personalized, multi-agent treatment strategies aimed at overcoming resistance and improving survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1f48636814c0a428af30d1bb9e6df8d06e9d0f" target='_blank'>
              Unraveling the many faces of angioimmunoblastic T-cell lymphoma: clinical, pathological, and molecular heterogeneity
              </a>
            </td>
          <td>
            Jia-Ping Chen, Peng Ke, Xiao Qiu, Qiu-Yuan Xian, Jing-Song Wu, Chun Feng, Hui-Jun Li, Yang Wang, Ji-Hao Zhou
          </td>
          <td>2026-01-01</td>
          <td>Annals of Hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Nodal T-follicular helper cell lymphoma, angioimmunoblastic type (nTFHL-AI, or AITL), is an aggressive peripheral T cell lymphoma without effective treatments. It has been shown that genetic and epigenetic changes lead to the expansion of neoplastic CD4
 +
 T cells originating from T follicular helper (Tfh) cells, which subsequently cause B cell expansion and tumor development. However, it remains unclear if the Tfh-like cell populations contain a subset that drives tumor progression and, if so, whether such a subset may have druggable targets.





 Through single-cell transcriptome analysis of Tfh-like cells isolated from the spontaneously arising AITL-like tumors in

 Roquin
 san/+

 mice, we identified and characterized a tumor-enriched Tfh cell subset highly expressing CXCR6 and IL-18 receptor, termed “Double-Expressor (DE) Tfh” cells. Using genetic or pharmacological approaches, we depleted DE Tfh cells
 in vivo
 and monitored tumor size (by ultrasound imaging) and the status of DE Tfh subset (by flow cytometry). We performed NanoString gene expression profiling of AITL patient samples to identify AITL cases that are relevant to

 Roquin
 san/+

 mouse model.





 DE Tfh cells expressed higher levels of Ki-67 and enhancer of zeste homolog 2 (EZH2) and proliferated more rapidly compared to other Tfh cells in an IL-18 and EZH2 dependent manner. Furthermore, DE Tfh cells engrafted better than non-DE Tfh cells and could cause B cell expansion when adoptively transferred into lymphopenic recipients. On the other hand, depletion of DE Tfh cells via
 Ezh2
 gene deletion, inhibition of EZH2 (using FDA-approved drug, tazemetostat), or anti-CXCR6 mAb led to tumor regression. These findings may be relevant to a subset of human AITL cases since we found that ~20-30% of AITL patient samples have concomitantly elevated expression of
 CXCR6
 ,
 IL-18R1
 , and
 IFNG
 .




 Our study identified a pathogenic Tfh-like subset essential for AITL tumor progression in a mouse model and suggests that identifying and targeting a DE Tfh-like subset in AITL patients might be an effective strategy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e46fc140d0a20e8be95b30a0b36275356db0219" target='_blank'>
              Identification of a neoplastic Tfh-like cellular subset in a mouse model of angioimmunoblastic T cell lymphoma
              </a>
            </td>
          <td>
            Saba Mohammaei, Jiyeon Lee, A. Bouchard, Evany Bernal Ballesteros, Nikoletta Diamantopoulos, Lifei Hou, Eileen Remold, J. Cho, Woong-Kyung Suh
          </td>
          <td>2026-01-30</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b30612d0c0fc6f8fb7f739e86841b70defae6577" target='_blank'>
              Plasticity of extrachromosomal DNA segregation during drug adaptation
              </a>
            </td>
          <td>
            Chikako Shibata, Kenichi Miyata, Kohei Kumegawa, Liying Yang, Ryu-Suke Nozawa, R. Maruyama
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Androgen-targeted therapy, chemotherapy and 177Lu-PSMA radioligand therapy have improved metastatic prostate cancer (mPC) patient survival. However, most patients develop resistance to these therapies. Studying genomic and transcriptomic changes that cause resistance requires repeated sample collection and is not suited to traditional invasive tissue biopsy techniques. We have developed a liquid-biopsy based technique to isolate cell-free DNA (cfDNA) and circulating tumor cell (CTC) RNA from mPC patients enabling genomic and transcriptomic profiling. Transcriptomic profiling of 146 high tumor purity mPC CTC samples from 70 patients identified 4 transcriptional phenotypes driven by changes in luminal, AR and proliferative signaling pathways: Low Proliferation (LP, n= 12, low luminal/AR/proliferation) Luminal A (LumA, n=24, high luminal/AR, low proliferation). Luminal B (LumB, n=31, high luminal/AR/proliferation) and Neuroendocrine Prostate Cancer (NEPC, n=3, low luminal/AR, high proliferation). Compared to LP and LumA, the LumB phenotype is associated with shorter overall survival and early progression on 177Lu-PSMA highlighting an unmet need for treatment options in patients with LumB CTCs. Here, we use mPC CTC transcriptional profiling to identify drivers of proliferation and treatment resistance in the LumB phenotype.



 Blood was collected from patients with mPC receiving standard of care treatment. CTCs were purified using immunomagnetic capture on a microfluidic platform, subjected to RNA-seq analysis and assessed for transcriptional phenotypes and gene expression signatures.



 We compared expression of gene sets related to established tumor suppressors or oncogenic drivers, DNA damage repair by homologous recombination repair (HRR), and cell cycle drivers in LP, LumA and LumB CTC phenotypes. LumB CTC samples display elevated expression of cell cycle genes related to S/G2/M phase progression (CCNE1, CCNA2 and CCNB1), but not G1 phase progression (CCND1 or CCND3), suggesting that increased S/G2/M progression is contributing to increased proliferation. In preclinical models, CCNE1 overexpression causes genomic instability and activation of DNA damage repair, cell cycle checkpoints and a S/G2/M phase transcriptional program mediated by the MYBL2-FOXM1 transcription factor complex. Indeed, we also observe increased expression of HRR, cell cycle checkpoint and MYBL2-FOXM1 transcriptional signatures in LumB subtypes.



 CTC transcriptional profiling has identified an aggressive and treatment resistant LumB phenotype characterized by high expression of genes involved in S/G2/M cell-cycle progression, HRR and cell cycle checkpoints. These findings suggest that elevated cell cycle progression and upregulated DNA repair capability may underlie the aggressive behavior and therapeutic resistance observed in patients with LumB CTCs. Ongoing efforts are focused on developing preclinical models of the mPC CTC transcriptional phenotypes to identify therapeutic vulnerabilities specific to the LumB subtype.



 David Gallo, Jamie M. Sperger, Amy K. Taylor, Kristen Rosche, Viridiana Carreno, Alex H. Chang, Emily Abella, Kaitlin Durnen, Muhammad Dar, Charlotte Linebarger, William M. Stump, Kendra Marr, Kyle T. Helzer, Matthew L. Bootsma, Grace C. Blitzer, John M. Floberg, David Kosoff, Rana R. McKay, Wei X. Xiao, Shuang G. Zhao, Joshua M. Lang, Marina N. Sharifi. Identifying therapeutic vulnerabilities in metastatic prostate cancer transcriptional phenotypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd3122808dac421339edd09df9a3f36d4488905" target='_blank'>
              Abstract B023: Identifying therapeutic vulnerabilities in metastatic prostate cancer transcriptional phenotypes
              </a>
            </td>
          <td>
            David Gallo, J. Sperger, Amy K Taylor, Kristin Rosche, Viridiana Carreno, Alex H Chang, Emily Abella, Kaitlin Durnen, Muhammad Dar, C. Linebarger, William M. Stump, Kendra Marr, K. Helzer, M. Bootsma, G. Blitzer, J. Floberg, D. Kosoff, Rana R. McKay, Wei X. Xiao, Shuang G. Zhao, Joshua M Lang, M. Sharifi
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="AI-driven virtual cell models show the potential to transform the paradigm of life sciences research by integrating multimodal omics data (e.g., single-cell transcriptomics and proteomics) with advanced algorithms such as deep generative models and graph neural networks to enable high-precision predictions of drug responses, gene perturbations, and disease progression. These models enable high-precision predictions of drug responses, gene perturbations, and disease progression. This review outlines the technical pathways and validation mechanisms of virtual cells, emphasizing a closed-loop workflow from computational evaluation to experimental verification using CRISPR assays and organoid platforms. The applications of virtual cells in personalized drug screening and disease modeling are highlighted, showcasing their potential to reduce animal testing and optimize therapy. However, challenges in regulatory acceptance, data privacy, and model interpretability remain. Global policy and standardization trends are driving clinical translation, and future advancements will involve cross-disciplinary integration and greater standardization to enhance the impact of virtual cells in precision medicine and drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5cc4085a09776c1b5f2f07f02af18c81780fd33" target='_blank'>
              AI-driven virtual cell models in preclinical research: technical pathways, validation mechanisms, and clinical translation potential
              </a>
            </td>
          <td>
            Chunyu Ma, Han Zhang, Yiwei Rao, Xinyu Jiang, Boheng Liu, Zhikang Sun, Zhenyu Song, Yuan Gao, Yuhao Cui, Xinyu Liu, Zedong Li
          </td>
          <td>2025-12-11</td>
          <td>NPJ Digital Medicine</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de45c8e1c8c2164d11d2bf078688ab6f987fecf4" target='_blank'>
              Integrated single-cell and bulk RNA sequencing unravels neutrophil heterogeneity and validates SPP1 as a prognostic biomarker in cervical cancer.
              </a>
            </td>
          <td>
            Siting Lin, Junlin Zhong, Shengjin Yuan, Manting Su, Yiwang Zhang, Xinling Zhang
          </td>
          <td>2026-01-30</td>
          <td>BMC cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Improving responses to cancer immunotherapies requires deeper insight into the cellular mechanisms governing T cell-mediated anti-tumor immunity. TMEM33 is an endoplasmic reticulum-resident transmembrane protein enriched across multiple tumor types, with reported functions in anti-viral immunity as well as calcium and lipid homeostasis, yet its role in tumor immunosurveillance remains unknown. Using murine genetic models, we demonstrate that host TMEM33 constrains anti-tumor CD8+ T cell responses. Constitutive Tmem33-/- mice exhibited delayed melanoma tumor growth and increased CD8+ T cell infiltration. Antigen-specific CD8+ compartments in tumors of Tmem33-/- mice showed TCF-1+PD-1+ progenitor-exhausted cell (Tpex) enrichment, elevated effector function and reduced exhaustion, alongside improved effector memory expansion and T-bet expression in draining lymph nodes. We highlight that TMEM33 functions intrinsically within the T cell compartment, as TMEM33 deletion (1) enhanced polyclonal activation of naive CD8+ T cells ex vivo, (2) promoted preferential Tpex accumulation among adoptively transferred naive OT-I cells in B16F10-OVA tumors and draining lymph nodes, and (3) improved the potency of ex vivo-expanded OT-I cells in controlling tumor growth during adoptive cell therapy. Finally, in a large, prospectively recruited metastatic melanoma cohort, lower TMEM33 expression in patient CD8+ T cells significantly correlated with improved survival and elevated TCF-7 (encoding TCF-1). Collectively, our findings define TMEM33 as a formerly unrecognized intrinsic determinant of tumor-directed CD8+ T cell fate that limits Tpex maintenance, and restrains cell therapy responses, suggesting that its modulation may strengthen immunotherapeutic efficacy. One sentence summary TMEM33 intrinsically limits progenitor exhausted CD8+ T cells, scales anti-tumor responses and predicts melanoma patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b5c85fa3ccdffaf374a3f09dd7d217b43c30d4c" target='_blank'>
              TMEM33 deletion potentiates anti-tumor CD8+ T cell immunity
              </a>
            </td>
          <td>
            M. T. Jackson, Tianming Zhao, G. Milotay, I. Pedroza-Pacheco, Yanshu Cai, Claire Willis, S. Phyu, Bruno Beernaert, Jamie TW Kwon, Hala Estephan, Vinnycius Pereira Almeida, Maria Aggelakopoulou, Silvia Panetti, Christian Zierhut, Tim Elliott, Eleni Adamopoulou, J. Rehwinkel, Malcolm JW Sim, Amit Grover, Dmitry I. Gabrilovich, Benjamin P. Fairfax, E. Honoré, D. Withers, John C. Christianson, Eileen E. Parkes
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Focal adhesion kinase (FAK) has long been explored as a therapeutic target in glioblastoma (GBM) based on its overexpression and involvement in invasive signaling. However, clinical trials have consistently failed to show benefit - highlighting a core principle of translational oncology: target presence alone does not imply therapeutic relevance. In contrast, neurofibromatosis type 2 (NF2)-mutant meningiomas present a biologically grounded vulnerability, in which loss of the tumor suppressor moesin-ezrin-radixin-like protein (merlin) creates a synthetic lethal dependency on FAK. This context-specific dependency enables clinically meaningful targeting. Early-phase trials already show promising disease control with favorable safety profiles. This mini review examines the contrasting roles of FAK in GBM and NF2-mutant meningiomas to underscore the importance of biological context in therapeutic decisions. We propose that NF2-mutant meningiomas represent a model for context-specific, synthetic lethal targeting, exemplifying a functional oncogenomics approach to precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/727ca65dac08941664c6173a7e5bd4ba1b46040f" target='_blank'>
              Context-specific targeting of focal adhesion kinase in brain tumors: lessons from glioblastoma and neurofibromatosis type 2-mutant meningioma
              </a>
            </td>
          <td>
            A. Sagerer, I. Eyüpoglu, T. Juratli, Nils Cordes
          </td>
          <td>2025-12-19</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Early recurrence remains a major challenge in the management of hepatocellular carcinoma (HCC), yet its molecular mechanisms are not fully understood. In this study, we applied an integrative multi‐omics strategy at single‐cell resolution to explore potential drivers of early HCC recurrence (recurrence time < 2 years) and to develop a predictive framework. By combining single‐cell RNA sequencing, proteomics, transcriptomics, and clinical feature analysis, we identified 14 relapse‐associated proteins, including CD274, B2M, MYC, and CASP3, as candidate risk factors. Transcriptomic profiling suggested the enrichment of pathways such as MYC‐TARGETS‐V2 and INTERFERON‐GAMMA‐RESPONSE. Single‐cell analysis indicated reduced immune cell infiltration in recurrent tumors, with myeloid cells (particularly cDC2 and macrophages) showing B2M‐associated reprogramming characterized by HLA downregulation and altered GAS6/PROS1 signaling, consistent with tumor‐associated macrophage‐like phenotypes. A LASSO regression model based on cDC2 and macrophage signature genes demonstrated moderate predictive performance in both the training and validation cohorts (AUC > 0.65). Drug sensitivity analyses further suggested that vandetanib may have the potential to inhibit recurrence by targeting B2M‐related pathways. These findings provide evidence that B2M may contribute to remodeling of the immune microenvironment in recurrent HCC. Our integrative single‐cell multi‐omics approach highlights a possible mechanism of early recurrence and offers a preliminary predictive tool with therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea78ab762a4ffddad742cec57b0d57d2bd1edeac" target='_blank'>
              Single‐Cell Multi‐Omics Reveals B2M‐Mediated Myeloid Reprogramming and Constructs a Predictive Model for Early Hepatocellular Carcinoma Recurrence
              </a>
            </td>
          <td>
            Zhenyao Tan, Yezhen Tang, Bendong Chen, Yongxue Lv, Bozhi Zhang, Kejun Liu, Yang Bu
          </td>
          <td>2025-12-22</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77606e4919065b0b8c161216da3a8b4fee1b4a2c" target='_blank'>
              Deficiency of SMARCB1 drives an immunosuppressive microenvironment in meningioma.
              </a>
            </td>
          <td>
            Ben Jin, Hao Hu, Yanhua Lu, Xia Tian, Guanghui Hu, Jingjing Xu, Xingqi Wu, Long Zhang, Juxiang Chen, Miaoxia He
          </td>
          <td>2026-01-24</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Hepatocellular carcinoma (HCC) is characterized by marked intratumoral heterogeneity and poor clinical outcomes. Dysregulated ribosome biogenesis has emerged as a fundamental hallmark of tumor initiation and progression; however, the specific molecular drivers linking this machinery to HCC pathogenesis remain largely undefined. Methods: By integrating multi-omics data from the TCGA and ICGC cohorts, FBLL1 was identified as a key prognostic candidate gene. Its cellular and spatial distribution was analyzed using single-cell RNA sequencing and spatial transcriptomics. Its biological functions in vitro and in vivo were validated through functional experiments, including lentivirus-mediated ectopic expression and siRNA-mediated gene knockdown. Finally, its molecular mechanism was elucidated through transcriptomic analysis and Western blotting. Results: FBLL1 was significantly upregulated in HCC and correlated with poor patient survival. Spatial and single-cell analyses showed that FBLL1 expression was preferentially enriched in malignant hepatocytes within the tumor region. Functionally, knockdown FBLL1 could inhibit the proliferation and clonogenic capacity of HCC cells, while overexpression FBLL1 in non-tumorigenic hepatocytes could promote the tumorigenic phenotype in xenograft models. Transcriptomic analysis indicated that FBLL1 overexpression was associated with the synergistic upregulation of c-Myc and multiple EGFR ligands, as well as decreased expression of hepatocyte functional markers. Consistently, modulation of FBLL1 expression affected the activity of the EGFR–MAPK signaling pathway. Conclusions: Our study identifies FBLL1 as a previously unrecognized regulator associated with malignant state transition in HCC. Rather than acting as a direct regulator of core signaling components, FBLL1 is associated with ligand-dependent activation of the EGFR–MAPK pathway in conjunction with c-Myc upregulation. These findings indicate that FBLL1 represents a promising therapeutic target for disrupting oncogenic signaling programs in liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73452fadba8b76804fe5e945db231c49dcf6ea8b" target='_blank'>
              Integrated Multi-Omics and Spatial Transcriptomics Identify FBLL1 as a Malignant Transformation Driver in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Junye Xie, Shujun Guo, Yujie Xiao, Yibo Zhang, An Hong, Xiaojia Chen
          </td>
          <td>2026-01-27</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer is increasingly recognized not as a fixed genetic condition but as a dynamic and plastic disease state driven by reversible epigenetic, transcriptional and microenvironmental cues. This evolving understanding supports a therapeutic paradigm shift: from eradicating malignant cells to reprogramming them toward quiescence, differentiation or functional normalization. This review explores diverse strategies for redirecting cancer cell fate, including differentiation therapy, epigenetic remodeling, lineage reprogramming, senescence induction and tumor microenvironment resetting. These approaches exploit intrinsic cellular plasticity and contextual adaptability, offering novel avenues to contain malignancy and overcome resistance. By reframing cancer as a potentially reversible phenotype, this therapeutic strategy demands redefinition of clinical endpoints, incorporation of dynamic biomarkers, and development of integrative treatment frameworks. Ultimately, reprogramming‑based oncology holds the promise of transforming aggressive malignancies into manageable conditions while minimizing the collateral damage associated with conventional cytotoxic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99bfdfe1c6957af978b8fb4415d04fd96bdf7b85" target='_blank'>
              Beyond eradication: Therapeutic reprogramming strategies for cancer normalization (Review).
              </a>
            </td>
          <td>
            Aliasgar F. Shahiwala
          </td>
          <td>2025-12-04</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b954d4c894a7ba2af75533050256820b25f647" target='_blank'>
              Heterocellular crosstalk and architecture of the pancreatic tumour microenvironment.
              </a>
            </td>
          <td>
            F. Arnold, Annachiara Del Vecchio, Zainab Hussain, Mara H. Sherman
          </td>
          <td>2026-01-16</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 MB (G3MB) has poor overall survival at <50%, high frequencies of metastases, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling occur frequently in G3MB. Many tumors have no reported mutations, suggesting epigenetic drivers. We here describe novel humanized models for G3MB from human induced pluripotent stem cells (hiPSC). By transducing hiPSC-derived neuroepithelial stem cells (NESC), we determined that: 1) both MYC and TGFβ effectors drove tumors in vivo; 2) MYC/TGFβR1 in combination led to more aggressive tumors and resistance to clinical inhibitors of TGFβ, and 3) NESC-derived tumors clustered with human G3MB. To decipher mechanisms, we integrated RNA-sequencing with CUT&RUN (for MYC genomic localization and post-translational modification of histones). MYC-bound neural developmental genes were repressed in MYC/TGFβR1 co-driven lines. Gene signatures associated with the Polycomb Repressive Complex (PRC) demarcated with H3K27me3; the histone mark directly regulated by PRC. We identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of repressed neural genes. Primary G3MB also showed increased levels of H3K27me3 concurrent with higher expression of JARID1B. Knockdown of JARID1B in human G3MB cell lines reduced growth, supporting potential as a therapeutic target. We conclude that a MYC-TGFβ-JARID1B axis represses target genes to drive G3MB and present new humanized models for G3MB to understand epigenetic dysregulation in G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd58fafdd674bdec3d7247a90b13d164f167e167" target='_blank'>
              Human stem cell models for group 3 medulloblastoma uncover JARID1B as a regulator of the chromatin landscape
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, Wazim Mohammed Ismael, Linyu Wang, Mackenzie O Johnson, Shane Grele, Ronald Phua, Liam D. Hendrikse, Ekin Guney, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael D. Taylor, William A. Weiss
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Tumor development and progression reshape the physical properties of the surrounding tumor microenvironment (TME), including its biomechanical traits. This is driven by a prominent cell type in the TME, cancer-associated fibroblasts (CAFs), which increases tissue stiffness via extracellular matrix deposition and remodeling. Currently, it is unclear whether there are also physical changes to CAFs at the cellular level and, if so, how they relate to patient outcome. Here, it is shown that CAFs have distinct morphological and biomechanical features from normal fibroblasts. Matched, patient-derived CAFs and non-malignant prostate fibroblasts (NPFs) from 35 patients with primary prostate cancer are examined. Morphologically, CAFs have more aligned stress fibers and larger and more elongated nuclei, based on quantitative image analysis of confocal microscopy images. In addition, single-cell mechanical measurements using real-time deformability cytometry showed that CAFs are larger and stiffer than NPFs. These changes are consistent across patients and validated with atomic force microscopy. A combined morphomechanical score encompassing these features is significantly associated with patient outcome. In transcriptomic analyses, the score is correlated with microtubule dynamics and a myofibroblast phenotype. Importantly, it is also demonstrated that morphomechanical features of prostate fibroblasts are modified by approved treatments for prostate cancer, such as docetaxel, and other small molecular inhibitors, particularly those targeting FGFR. In summary, changes in cellular morphomechanical properties are a consistent feature of CAFs and are associated with patient outcome. Moreover, cellular morphomechanical properties can be therapeutically targeted, potentially providing a new strategy for manipulating the TME to control cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe160be726a9c222a84b4d1334642ece40944d7" target='_blank'>
              Single-Cell Morphomechanics of Prostate Cancer-Associated Fibroblasts Identifies Distinct Features Associated with Patient Outcome.
              </a>
            </td>
          <td>
            Antje Garside, Angela Jacobi, S. Keerthikumar, Vaibhav Mahajan, M. Richards, B. Niranjan, Linda Teng, Nicholas K Choo, Johannes Low Jun Wei, G. Risbridger, Mitchell G. Lawrence, A. Taubenberger
          </td>
          <td>2026-01-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Background Intratumor heterogeneity (ITH), a critical driver of tumor evolution and immune evasion, remains inadequately characterized at the transcriptomic level in colorectal cancer (CRC), and its clinical implications are not yet fully understood. Methods We integrated transcriptomic datasets from TCGA-COAD/READ and two independent GEO cohorts (GSE40967 and GSE87211) to develop an RNA-seq–based ITH score using the DEPTH2 algorithm and to con0struct an ITH-related gene (ITRG) prognostic model. A unified cutoff value of 0.64 was established to stratify patients into high- and low-ITH groups. Using 52 survival-associated ITRGs, we generated a nine-gene prognostic signature selected from 101 distinct combinations of feature selection techniques and modeling algorithms and validated its performance in two external datasets. The tumor microenvironment and potential responsiveness to immune checkpoint inhibitors were evaluated using ssGSEA, ESTIMATE, and TIDE algorithms. SHAP analysis, together with in vitro and in vivo experiments, was employed to identify and functionally validate key regulatory genes. Results Patients in the high-ITH group had markedly poorer overall survival (OS) than those in the low-ITH group. The ITH score correlated strongly with aggressive clinical features, including T3/4 invasion depth, N1/2 nodal status, and AJCC stage III. The nine-gene prognostic signature demonstrated consistent predictive capability across the TCGA training cohort and both GEO validation cohorts. In TCGA, the model yielded time-dependent AUCs of 0.669, 0.664, and 0.645 for 1-, 3-, and 5-year OS, respectively, and retained its status as an independent prognostic indicator in multivariate Cox regression analysis; in GSE40967 and GSE87211, the corresponding AUCs ranged from 0.544–0.573 and 0.648–0.744. The high-risk subgroup was characterized by a stromal immune phenotype enriched with cancer-associated fibroblasts and macrophages, elevated expression of multiple immune checkpoints, increased TIDE scores, and higher tumor mutational burden. SHAP analysis identified IL20RB as the top risk-associated gene, whose knockdown significantly suppressed CRC cell proliferation, migration, invasion, and tumorigenicity in vitro and in vivo. Conclusion This study introduces and validates a transcriptomic ITH score and a nine-gene ITRG-based prognostic model that delineate the immune landscape and enables effective survival stratification in CRC, complementing the limitations of current staging systems. Additionally, IL20RB is highlighted as a promising therapeutic target, supporting the development of personalized immuno-targeted combination therapies in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/385eed67dbbc79210270a5630b0149ac9e500c8e" target='_blank'>
              Intratumor heterogeneity score reveals immune landscape and survival stratification in colorectal cancer
              </a>
            </td>
          <td>
            Zijing Wang, Liyuan Ma, Jinzhong Cao, Xi Chen, Xin Chang, Ruxue Ma, Hengyi Lv, Zixin Zhang, Hai Li, Tao Jiang
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The tumor microenvironment (TME) represents a highly dynamic and multifaceted ecosystem composed of both cellular and noncellular components that collectively modulate tumor initiation, progression, and metastatic potential. Within this intricate milieu, neuron-tumor interactions have recently emerged as critical determinants of cancer behavior, especially in malignancies of the nervous system, where neuronal activity appears to sustain and even accelerate tumor growth. Increasing evidence supports the existence of a bidirectional communication between neurons and cancer cells, involving the exchange of soluble factors, neurotransmitters, and vesicular signals that influence cell survival, proliferation, and morphology. To dissect the molecular mechanisms underlying this neuron-tumor crosstalk, we established and optimized a series of in vitro co-culture models. Primary cortical neurons derived from C57BL/6 mice and neurons differentiated from human induced pluripotent stem cells (hiPSCs) were co-cultured with U87 glioblastoma cells to reproduce the neural-tumor interface. Both transwell-based co-culture systems and conditioned media (CM) approaches were implemented to evaluate the impact of glioma-secreted components on neuronal viability and morphology. Neuronal dynamics were monitored in real time using the Incucyte live-cell imaging system equipped with the Neurotrack module. Exposure to glioma-conditioned medium led to pronounced neurotoxic effects in murine cells, including a 36% reduction in neurite length (p=0.0055), a 44% decrease in cell number (p=0.0024), and a significant increase in cytotoxicity after 48 h (p=0.0116). These findings highlight the detrimental influence of glioblastoma-derived soluble factors on neuronal structure and function. Complementary Western blot analyses confirmed this phenotype, revealing a marked decrease in βIII-tubulin (-60%) and synapsin (-80%) expression in CM-treated murine neurons, indicating impaired cytoskeletal stability and synaptic integrity. Current efforts focus on extending these observations to hiPSCderived neuronal cultures and enhancing the physiological relevance of the model by integrating advanced microfluidic co-culture systems. Such platforms will allow spatiotemporal control of cellular interactions, providing a more faithful in vitro representation of the brain tumor microenvironment and enabling the identification of molecular targets capable of disrupting the pro-tumor neuronal support network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47801036babe68c650156cad23ef9951afb3627d" target='_blank'>
              EXPLORING NEURON–GLIOMA SIGNALING THROUGH IN VITRO CO-CULTURES
              </a>
            </td>
          <td>

          </td>
          <td>2025-12-12</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [6, 4],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>